<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/217163-diacylglycerol-acyltransferase-nucleic-acid-sequences-and-associated-products by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:27:02 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 217163:DIACYLGLYCEROL ACYLTRANSFERASE NUCLEIC ACID SEQUENCES AND ASSOCIATED PRODUCTS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">DIACYLGLYCEROL ACYLTRANSFERASE NUCLEIC ACID SEQUENCES AND ASSOCIATED PRODUCTS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention is directed to polypeptides and nucleic sequences related thereto, and methods to purify, ob tain, and use such molecules in genetic engineering applications. More specifically, the present invention relates to polypeptides associated with the production of triacylglycerols in plants and fungi.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>DIACYLGLYCEROL ACYLTRANSFERASE NUCLEIC ACE) SEQUENCES ANDxAl5SOCIATED PRODUCTS<br>
This application claims the benefit of U.S. Provisional Application 60/399,427 filed July 31,2002, which is incorporated herein by reference.<br>
The present invention is directed to polypeptides and nucleic acid sequences related thereto, and methods to purify, obtain, and use such molecules in genetic engineering applications. More specifically, the present invention relates to polypeptides associated with the production of triacylglycerols in plants, fungi, and mammals.<br>
Diacylglycerol acyltransferase (referred to hereinafter as DGAT) is an integral membrane protein that catalyzes the final enzymatic step in the production of triacylglycerols in plants, fungi, and mammals. DGAT has generally been described in Harwood, Biochem. Biophysics. Acta, 13017-13056 (1996); Daum et al, Yeast, 16:1471-1510 (1998); and Coleman et a/., Annu. Rev. Nutr., 20:77-103 (2000). This enzyme is responsible for transferring an acyl group from acyl-coenzyme-A to the sn-3 position of 1,2-diacylglycerol (DAG) to form triacylglycerol (TAG). In plants and fungi, DGAT is associated with the membrane and lipid body fractions, particularly in oilseeds, where it contributes to the storage of carbon used as energy reserves. TAG is believed to be an important chemical for storage of energy in cells. DGAT is known to regulate TAG structure and direct TAG synthesis. Furthermore, it is known that the DGAT reaction is specific for oil synthesis.<br>
In plants, TAG is the primary component of vegetable oil that is used by the seed as a stored form of energy to be used during seed germination. Higher plants are believed to synthesize oils via a similar metabolic pathway commonly referred to as the Kennedy pathway (Kennedy et a/., J. Biol. Chem., 222:193 (1956); Finnlayson et al.,Arch. Biochem. Biophys., 199:179-185 (1980)). Fatty acids are made in plastids from acetyl-CoA through a series of reactions catalyzed by enzymes known collectively as Fatty Acid Synthase (FAS). The fatty acids produced in plastids are exported to the cytosolic compartment of the cell, and are esterified to coenzyme A. These acyl-CoAs are the substrates for glycerolipid synthesis on the endoplasmic reticulum (ER). Glycerolipid synthesis itself is a series of reactions leading first to phosphatidic acid (PA) and DAG. Either of these metabolic intermediates may be directed to membrane phospholipids such as phosphatidylglycerol (PG), phosphatidylethanolamine (PE), or phosphatidylcholine (PC), or they may be directed on to form neutral triacylglycerol (TAG).<br>
DAG is synthesized from glycerol-3-phosphate and fatty acyl-CoAs in two steps catalyzed sequentially by glycerol-3-phosphate acyltransferase (G3PAT), and lysophosphatidic acid acyltransferase (LPAAT) to make PA, and then an additional hydrolytic step catalyzed by phosphatidic acid phosphatase (PAP) to make DAG. In most cells, DAG is used to make membrane phospholipids, the first step being the synthesis of PC catalyzed by CTP-phosphocholine cytidylyltransferase. In cells producing storage oils, DAG is acylated with a third fatty acid in a reaction catalyzed by DGAT.<br>
Two different families of DGAT proteins have been identified. The first family of DGAT proteins (referred to hereinafter as DGAT1) is related to the acyl-coenzyme A:cholesterol acyltransferase (ACAT) and has been described in the literature (see, e.g., U.S. Patents 6,100,077 and 6,344,548). A second family of DGAT proteins (referred to hereinafter as DGAT2), unrelated to the previously identified family of DGAT1 proteins, is described in the present invention. This family of DGAT2 proteins is also described in U.S. Application 10/121,857, filed April 15, 2002.<br>
Obtaining nucleic acid sequences capable of producing a phenotypic result in the incorporation of fatty acids into a glycerol backbone to produce an oil is subject to various obstacles including but not limited to the identification of metabolic factors of interest, choice and characterization of a protein source with useful kinetic properties, purification of the protein of interest to a level which will allow for its arnino acid sequencing, utilizing amino acid sequence data to obtain a nucleic acid sequence capable of use as a probe to retrieve the desired DNA sequence, and the preparation of constructs, transformation and analysis of the resulting plants.<br>
Thus, the identification of enzyme targets and useful tissue sources for nucleic acid sequences of such enzyme targets capable of modifying oil structure and quantity are needed. Ideally, an enzyme target will be amenable to one or more applications alone or in combination with other nucleic acid sequences relating to increased or decreased oil production, TAG structure, the ratio of saturated to unsaturated fatty acids in the fatty acid pool, and/or to other novel oil compositions as a result of the modifications to the fatty acid pool.<br>
SUMMARY OF THE INVENTION<br>
The present invention provides genetic tools that answers the need of both altering the composition of oils produced in a plant as well as the percentage content thereof relative to<br>
other components of a seed, including, for example, the meal content thereof. The present invention includes diacylglycerol acyltransferase (DGAT) polypeptides and polynucleotides encoding these polypeptides. The polypeptides and polynucleotides of the present invention include those derived from plant and fungal sources, including, for example, Mortierella ramanniana, Saccharomyces cerevisiae, and Neurospora crassa.<br>
The present invention further relates to polynucleotides that encode the DGAT proteins, and polynucleotides that include partial or complete DGAT encoding sequences. The present invention also provides polynucleotides that encode the DGAT2 proteins, and polynucleotides that include partial or complete DGAT2 encoding sequences.<br>
The present invention also provides recombinant DNA constructs that can be used for transcription and expression of DGAT2, including constructs that are capable of expressing DGAT2 in plant, and insect host cells.<br>
The present invention also includes an isolated nucleic acid molecule comprising a nucleic acid sequence that encodes a polypeptide molecule comprising a polypeptide sequence selected from the group consisting of SEQ ID NOs: 14,18,20,22, 24, 26, and 28. Preferred such isolated nucleic acid molecules include, for example, SEQ ID NOs: 11,12,13,16,17, 19,21, 23, 25, and 27.<br>
The present invention includes an isolated nucleic acid molecule comprising a nucleic acid sequence that encodes a polypeptide having homology to a diacylglycerol acyltransferase, wherein the nucleic acid molecule is selected from the group consisting of SEQ ID NOs: 11, 12,13,16, 17,19,21, 23, 25, and 27.<br>
The present invention also includes DNA constructs comprising an expression cassette comprising a heterologous promoter that functions in a plant cell operably linked to a nucleic acid molecule that encodes a polypeptide molecule comprising a polypeptide sequence selected from the group consisting of SEQ ID NOs: 14, 18,20,22, and 24. In certain embodiments the DNA construct further comprises a second expression cassette, wherein said second expression cassette comprises a second heterologous promoter that functions in a plant cell operably linked to a nucleic acid that encodes a polypeptide for a diacylglycerol acyltransferase. Preferably, the second heterologous promoter is different or the same from the heterologous promoter used initially; more preferably, the two heterologous promoters are different.<br>
The present invention also includes a DNA construct comprising an expression cassette comprising a heterologous promoter that functions in a plant cell operably linked to a nucleic acid molecule that encodes a polypeptide having homology to a diacylglycerol acyltransferase, wherein the nucleic acid molecule is selected from the group consisting of SEQ ID NOs: 11,12, 13,16,17,19, 21, and 23. In certain embodiments the DNA construct further comprises a second expression cassette wherein said second expression cassette comprises a second heterologous promoter that functions in a plant cell operably linked to a nucleic acid that encodes a polypeptide for a diacylglycerol acyltransferase. Preferably, the second heterologous promoter is different or the same from the heterologous promoter used initially; more preferably, the two heterologous promoters are different.<br>
The present invention also includes a plant or seed comprising the DNA construct comprised of expression cassette comprising a heterologous promoter that functions in a plant cell operably linked to a nucleic acid molecule that encodes a polypeptide having homology to a diacylglycerol acyltransferase, wherein the nucleic acid molecule is selected from the group consisting of SEQ ID NOs: 11,12,13,16,17,19, 21, and 23. Preferably, the plant or seed is one or more of maize, soybean, canola, oil seed rape, cotton, sesame, flax, peanut, sunflower, safflower, olive, and oil palm.<br>
The present invention also includes a plant or seed comprising the DNA construct comprised of an expression cassette comprising a heterologous promoter that functions in a plant cell operably linked to a nucleic acid molecule that encodes a polypeptide molecule comprising a polypeptide sequence selected from the group consisting of SEQ ID NOs: 14, 18,20, 22, and 24. Preferably, the plant or seed is one or more of maize, soybean, canola, oil seed rape, cotton, sesame, flax, peanut, sunflower, safflower, olive, and oil palm.<br>
Preferably, the plant or seed of the present invention is processed. More preferably, the plant or seed is used to produce a product, such as, for example, feed, meal, oil, or protein. The plant or seed used in this context is comprised of a DNA construct that includes a heterologous promoter that functions in a plant cell and a nucleic acid molecule selected from the group consisting of SEQ ID NOs: 11, 12,13, 16,17, 19, 21, and 23.<br>
In another embodiment, the plant or seed of the present invention is comprised of the DNA construct comprising an expression cassette comprising a heterologous promoter that functions in a plant cell operably linked to a nucleic acid molecule that encodes a polypeptide molecule, which polypeptide molecule comprises a polypeptide sequence selected from the<br>
group consisting of SEQ ID NOs: 14, 18, 20, 22, and 24. Preferably, the plant or seed of the invention is processed into a product, which product may be feed, meal, oil, and protein.<br>
The present invention further provides methods for the production of DGAT2 proteins in a host cell or progeny thereof. Recombinant cells containing DGAT2 are also provided.<br>
The present invention provides a method of producing a plant having enhanced oil composition comprising the steps of transforming a plant cell with a DNA construct expressing diacylglycerol acyltransferase and regenerating said plant cell into a fertile plant relative to a plant having a similar genetic background but lacking the introduced nucleic acid molecule. The present invention also includes a fertile plant providing seeds having an increased oil yield relative to a plant having a similar genetic background but lacking the introduced nucleic acid molecule.<br>
In another aspect, the present invention provides a polynucleotide encoding a polypeptide having at least one of the arnino acid motifs: AYVFGYEPHSVXPI (SEQ ED NO: 33) and FXXPXYR (SEQ ID NO: 34), where X represents any amino acid. Such polypeptides include, for example, SEQ ID NOs: 14,18,20, 22, and 24.<br>
In still yet another aspect, the present invention provides a polypeptide, including fragments and proteins, having diacylglycerol acyltransferase activity and which polypeptide comprises at least one of the amino acid motifs: AYVFGYEPHSVXPI (SEQ ED NO: 33) and FXXPXYR (SEQ ED NO: 34), where X represents any amino acid. Such polypeptides include, for example, SEQ ID NOs: 14,18, 20, 22, and 24.<br>
BRIEF DESCRIPTION OF THE SEQUENCE LISTING SEQ ED NO: 1 DNA sequence of Mortierella ramanniana DGAT2A SEQ ID NO: 2 polypeptide sequence of Mortierella ramanniana DGAT2A SEQ ED NO: 3 DNA sequence of Mortierella ramanniana DGAT2B SEQ ED NO: 4 polypeptide sequence of Mortierella ramanniana DGAT2B SEQ ED NO: 5 DNA sequence of Saccharomyces cerevisiae DGAT2B SEQ ID NO: 6 polypeptide sequence of Saccharomyces cerevisiae DGAT2B SEQ ID NO: 7 DNA primer for NcDGAT2 SEQ ID NO: 8 DNA primer for NcDGAT2 SEQ ED NO: 9 DNA primer for NcDGAT2 SEQ ID NO: 10 DNA primer for NcDGAT2 SEQ ED NO: 11 DNA sequence for Mortierella ramaniana DAGT2B.nno<br>
SEQ ID NO: 12 DNA sequence for Neurospora crassa DGAT.nno<br>
SEQ ED NO: 13 DNA sequence for Neurospora crassa DGAT2<br>
SEQ ID NO: 14 polypeptide sequence for Neurospora crassa DGAT2<br>
SEQ ID NO: 15 DNA sequence of Mortierella ramanniana DGAT2A.nno<br>
SEQ ID NO: 16 DNA sequence of Saccharomyces cerevisiae DGAT2.nno<br>
SEQ ID NO: 17 DNA sequence of Hordeum vulgare DGAT2<br>
SEQ ID NO: 18 polypeptide sequence of Hordeum vulgare DGAT2<br>
SEQ ID NO: 19 DNA sequence of Zea mays DGAT2<br>
SEQ ID NO: 20 polypeptide sequence of Zea mays DGAT2<br>
SEQ ID NO: 21 DNA sequence of Glycine max DGAT2<br>
SEQ ID NO: 22 polypeptide sequence of Glycine max DGAT2<br>
SEQ ID NO: 23 DNA sequence of Triticum aestivum DGAT2<br>
SEQ ID NO: 24 polypeptide sequence of Triticum aestivum DGAT2<br>
SEQ ID NO: 25 DNA sequence of Drosophila melanogaster DGAT<br>
SEQ ID NO: 26 polypeptide sequence of Drosophila melanogaster DGAT<br>
SEQ ED NO: 27 DNA sequence of Homo sapiens DGAT<br>
SEQ ED NO: 28 polypeptide sequence of Homo sapiens DGAT<br>
SEQ ED NO: 29 polypeptide sequence of Schizosaccharomyces pombel DGAT2<br>
SEQ ED NO: 30 polypeptide sequence of Schizosaccharomyces pombe2 DGAT2<br>
SEQ ED NO: 31 polypeptide sequence of Candida albicans DGAT2<br>
SEQ ED NO: 32 polypeptide sequence of Arabidopsis thaliana DGAT2<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figures la, Ib, and Ic collectively show the sequence alignment of certain derived DGAT2 polypeptide sequences. Similar sequences between polypeptides can be used to identify like molecules with similar activities. Several computer programs can be used to identify conserved, sequence motifs between related molecules, including but not limited to MEME, or GENE SCAN. Once the sequence motifs are identified, their function can be assayed. For example, motif sequences in DGAT can be used to identify other DGAT polypeptides. Novel motifs can be derived from the polypeptide sequences disclosed in the present invention and used to screen sequence databases or in the design of degenerate nucleic acid probes to isolate novel DNA molecules that encode for DGAT. The amino acid sequences of the predicted DGAT2 polypeptides are aligned using the Clustal multiple sequence alignment program. Totally conserved residues are shaded black, grey shaded is the<br>
consensus of three or more sequences. All sequences are full-length. Residues shown above the alignment are highly conserved signature amino acids found in the motifs D and E of the acyltransferase (Neuwald, Curr. BioL, 7:465-466, (1997)). In this area, DGAT2 and the acyltransferase superfamily sequences co-align, only the shared conserved amino acid residues are shown. Key: MrDGAT2A (SEQ ID NO: 2), MrDGAT2B (SEQ ID NO: 4), ScDGAT2B (SEQ ID NO: 6); NcDGAT2 (SEQ ED NO: 14), ZmDGAT2 (SEQ ID NO: 20), GmDGAT2 (SEQ ID NO: 22), HvDGAT2 (SEQ ID NO: 18), TaDGAT2 (SEQ ID NO: 24), CaDGAT2 (SEQ ID NO: 31), and AtDGAT2 (SEQ ID NO: 32).<br>
Figure 2 shows the phylogenetic tree of DGAT2 family members of Figure 1. The tree is constructed using the DNASTAR software.<br>
Figure 3 is a listing of DNA primer molecules used in the invention.<br>
Figure 4 is a schematic of vector pCGN8832.<br>
Figure 5 is a schematic of vector pMON68762.<br>
Figure 6 is a schematic of vector pMON68654.<br>
Figure 7 is a schematic of vector pMON70904.<br>
Figure 8 is a schematic of vector pMON70920.<br>
Figure 9 is a schematic of vector pMON70923.<br>
Figure 10 is a schematic of vector pMON70925.<br>
Figure 11 is a schematic of vector pMON70927.<br>
Figures 12a and 12b show the sequence alignment of derived DGAT2 polypeptide sequences from certain fungal species, as identified therein. The information for how to read this figure, including the meaning of abbreviations and shadings used, is the same as stated above for Figures la, Ib, and Ic. The conserved regions indicated in shadings and black are usefully employed to identify other DGAT2 polypeptides derived from other species. In particular, analysis of these sequences reveals the following motif: FXXPXYR (SEQ ID NO: 34), where X represents any amino acid, which can be used to identify further DGAT2 sequences, preferably those of fungal origin.<br>
Figure 13 shows the sequence alignment of derived DGAT2 polypeptide sequences from certain plant species, as identified therein. The information for how to read this figure, including the meaning of abbreviations and shadings used, is the same as stated above for Figures la, Ib, and Ic. The conserved regions indicated in shadings and black are usefully employed to identify other DGAT2 polypeptides derived from other species. In particular,<br>
analysis of these sequences reveals the following motif: AYVFGYEPHSVXPI (SEQ ID NO: 33), which can be used to identify further DGAT2 sequences, preferably those of plant origin.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
As used herein, the term triacylglycerol composition means a compound in an organism that includes the water-insoluble, fatty acid triesters of glycerol, i.e., having the chemical formula (CEb - R)s where R is an ester.<br>
As used herein, the term DGAT1 refers to a DGAT protein as described in U.S. Application 09/326,203, filed on June 4, 1999, herein incorporated by reference in its entirety, which is related to the acyl CoA:cholesterol acyltransferase (ACAT) gene family and responsible for transferring an acyl group from acyl-coenzyme-A to the sn-3 position of 1,2-diacylglycerol (DAG) to form triacylglycerol (TAG).<br>
As used herein, the term DGAT2 refers to a non-DGATl protein as defined above where the protein responsible for transferring an acyl group from acyl-coenzyme-A to the sn-3 position of 1,2-diacylglycerol (DAG) to form triacylglycerol (TAG). DGAT2 proteins typically are generally less than 40 kD in weight, and typically in the 33-42 kD range.<br>
As used herein, the term DGAT2A refers to a Mortierella ramanniana DGAT2 that has an amino acid sequence of SEQ ID NO: 2. As used herein, the term DGAT2B refers to a Mortierella ramanniana DGAT2 that has an amino acid sequence of SEQ ID NO: 4.<br>
As used herein, the phrase "oil composition" means the ratio of different fatty acid or oil components within a sample. Such a sample may be a plant or plant part, such as a seed. Such a sample may also be a collection of plant parts.<br>
As used herein, the phrase "percentage content" in a preferred embodiment means the percent by total weight of a particular component, relative to other similar or related components.<br>
As used herein, the phrase "enhanced oil" includes increased oil yield or altered oil composition.<br>
As used herein, a diacylglycerol acyltransferase (DGAT) gene of the present invention includes any nucleic acid sequence encoding amino acids, such as protein, polypeptide, or peptide, obtainable from a cell source, which demonstrates the ability to catalyze the . production of triacylglycerol from 1,2-diacylglycerol and fatty acyl substrates under enzyme reactive conditions. By "enzyme reactive conditions" it is meant that any necessary<br>
conditions are available in an environment (i.e., such factors as temperature, pH, lack of inhibiting substances) that will permit the enzyme to function.<br>
The present invention relates to acyl CoA:diacylglycerol acyltransferase (referred to herein as DGAT) which catalyzes the final step in the production of triacylglycerol (TAG). More particularly, the present invention includes DGAT polypeptides and polynucleotides that encode the DGAT polypeptides. The DGAT polypeptide and polynucleotide molecules of the present invention are isolated from plant and fungal sources. Expression of the cDNAs in insect and plant cells conferred high levels of DGAT activity on the membranes isolated from these cells. The present invention provides a gene family, including members in fungi, plants, and animals, which encode enzymes with DGAT function.<br>
DGAT proteins are isolated from cells of the oleaginous fungus Mortierella ramanniana. Following cell lysis, DGAT activity is associated with the lipid body fraction and detergent solubilization is required to release the membrane-bound proteins to permit their purification using traditional chromatographic techniques. A stimulation of DGAT activity in the homogenate is observed following the addition of the detergent Triton X-100. Using a 5-step protocol, two proteins, 36 kD and 36.5 kD by SDS-PAGE, are identified as being associated with DGAT activity. These proteins are named MrDGAT2A and MrDGAT2B, respectively. Final specific activity recoveries of 1.6% and 4.2%, respectively, are reported for the purest, most active fractions containing each protein. Expression of the cloned cDNAs in insect cells confirmed DGAT. Full-length clones are obtained for several plant and fungal DGAT homologs and the expressed proteins are evaluated in insect cells or plant cells. The homologs tested exhibited some level of DGAT activity demonstrating that the genes in this family are related by function.<br>
The present invention provides a number of agents, for example, nucleic acid molecules and polypeptides associated with enzymes responsible for transferring an acyl group from acyl-coenzyme-A to the sn-3 position of 1,2-diacylglycerol (DAG) to form triacylglycerol (TAG), and provides uses of such agents.<br>
Nucleic Acid Molecules<br>
Agents of the invention include nucleic acid molecules. In a preferred aspect of the present invention the nucleic acid molecule comprises a nucleic acid sequence selected from the group consisting of SEQ ID NOs: 11, 12, 13, 16, 17, 19, 21, 23, 25, and 27.<br>
In a further aspect of the present invention the nucleic acid molecule comprises a nucleic acid sequence encoding an amino acid sequence selected from the group consisting of SEQ ID NOs: 14,18, 20, 22, 24, 26, 28, and fragments thereof.<br>
A preferred embodiment of the present invention relates to the use of motifs, i.e., conserved elements found in the sequences of identified DGAT molecules, for the purpose of identifying other DGAT genes and proteins. Accordingly, one skilled in the art can use a motif, such as, for example SEQ ID NO: 33 or 34, with or without reverse transcribing the motif sequence, and screen for other genes that encode a DGAT or other polypeptides that have DGAT activity.<br>
A first nucleic acid sequence of the present invention displays "substantial identity" to a reference nucleic acid sequence if, when optimally aligned (with appropriate nucleotide insertions or deletions totaling less than 20 % of the reference sequence over the window of comparison) with the other nucleic acid (or its complementary strand), there is at least about 75% nucleotide sequence identity, preferably at least about 80% identity, more preferably at least about 85% identity, yet more preferably at least about 90%, and most preferably at least . about 95% identity over a comparison window of at least 20 nucleotide positions, preferably at least 50 nucleotide positions, more preferably at least 100 nucleotide positions, and most preferably over the entire length of the first nucleic acid. Optimal alignment of sequences for .. aligning a comparison window may be conducted by the local homology algorithm of Smith and Waterman, Adv. Appl. Math., 2:482 (1981); by the homology alignment algorithm of Needleman and Wunsch, J. Mol. Biol., 48:443 (1970); by the search for similarity method of Pearson and Lipman, Proc. Natl. Acad. Sci. (U.S.A.), 85:2444 (1988); preferably by computerized implementations of these algorithms (GAP, BESTFIT, FASTA, and TFASTA) in the Wisconsin Genetics Software Package Release 7.0, Genetics Computer Group, 575 Science Dr., Madison, Wis. The reference nucleic acid may be a full-length molecule or a portion of a longer molecule. Alternatively, two nucleic acids have substantial identity if one hybridizes to the other under stringent conditions.<br>
It is understood that in a further aspect of the present invention, the nucleic acid sequences can encode a protein that differs from any of the proteins in that one or more amino acids have been deleted, substituted, or added without altering the function. For example, it is understood that codons capable of coding for such conservative amino acid substitutions are known in the art.<br>
One subset of the nucleic acid molecules of the present invention is -fragment nucleic acid molecules. Fragment nucleic acid molecules may consist of significant portion(s) of, or indeed most of, the nucleic acid molecules of the present invention, such as those specifically disclosed. Alternatively, the fragments may comprise smaller oligonucleotides (having from about 15 to about 400 nucleotide residues and more preferably, about 15 to about 30 nucleotide residues, or about 50 to about 100 nucleotide residues, or about 100 to about 200 nucleotide residues, or about 200 to about 400 nucleotide residues, or about 275 to about 350 nucleotide residues).<br>
A fragment of one or more of the nucleic acid molecules of the present invention may be a probe and specifically a PCR probe. A PCR probe is a nucleic acid molecule capable of initiating a polymerase activity while in a double-stranded structure with another nucleic acid. Various methods for determining the structure of PCR probes and PCR techniques exist in the art. Computer generated searches using a program such as GeneUp (Pesole et al., BioTechniques, 25:112-123 (1998)), for example, can be used to identify potential PCR primers.<br>
Nucleic acid molecules or fragments thereof of the present invention are capable of specifically hybridizing to other nucleic acid molecules under certain circumstances. Nucleic acid molecules of the present invention include those that specifically hybridize to nucleic acid molecules having a nucleic acid sequence selected from the group consisting of SEQ ID NOs: 11, 12, 13, 16,17,19, 21,23, 25, and 27, and complements thereof. Nucleic acid molecules of the present invention also include those that specifically hybridize to nucleic acid molecules encoding an amino acid sequence selected from SEQ ID NOs: 14, 18, 20, 22, 24, 26,28, and fragments thereof.<br>
A nucleic acid molecule is said to be the "complement" of another nucleic acid molecule if the molecules exhibit complete complementarity. As used herein, molecules are said to exhibit "complete complementarity" when every nucleotide of one of the molecules is complementary to a nucleotide of the other. Two molecules are said to be "minimally complementary" if they can hybridize to one another with sufficient stability to permit them to remain annealed to one another under at least conventional "low-stringency" conditions. Similarly, the molecules are said to be "complementary" if they can hybridize to one another with sufficient stability to permit them to remain annealed to one another under conventional "high-stringency" conditions. Conventional stringency conditions are described by Sambrook<br>
et al, Molecular Cloning, A Laboratory Manual, 2nd Ed., Cold Spring Harbor Press, Cold Spring Harbor, New York (1989), and by Haymes et al., Nucleic Acid Hybridization, A Practical Approach, IRL Press, Washington, DC (1985). Departures from complete complementarity are therefore permissible, as long as such departures do not completely preclude the capacity of the molecules to form a double-stranded structure. Thus, in order for a nucleic acid molecule to serve as a primer or probe it need only be sufficiently complementary in sequence to be able to form a stable double-stranded structure under the particular solvent and salt concentrations employed.<br>
Appropriate stringency conditions which promote DNA hybridization are, for example, 6.0 X sodium chloride/sodium citrate (SSC) at about 45°C, followed by a wash of 2.0 X SSC at 20-25°C, are known to those skilled in the art or can be found in Current Protocols in Molecular Biology, John Wiley &amp; Sons, New York (1989), 6.3.1-6.3.6. For example, the salt concentration in the wash step can be selected from a low stringency of about 2.0 X SSC at 50°C to a high stringency of about 0.2 X SSC at 65°C. In addition, the temperature in the wash step can be increased from low stringency conditions at room temperature, about 22°C, to high stringency conditions at about 65°C. Both temperature and salt may be varied, or either the temperature or the salt concentration may be held constant while the other variable is changed.<br>
In a preferred embodiment, a nucleic acid of the present invention will specifically hybridize to one or more of the nucleic acid molecules set forth in SEQ ID NOs: 11, 12,13, 16,17,19,21, 23, 25, and 27, and complements thereof under moderately stringent conditions, for example at about 2.0 X SSC and about 65°C.<br>
A nucleic acid molecule of the present invention can also encode a homolog polypeptide. As used herein, a homolog polypeptide molecule or fragment thereof is a counterpart protein molecule or fragment thereof in a second species (e.g., corn rubisco small subunit is a homolog of Arabidopsis rubisco small subunit). A homolog can also be generated by molecular evolution or DNA shuffling techniques, so that the molecule retains at least one functional or structure characteristic of the original polypeptide (see, for example, U.S. Patent 5,811,238).<br>
In another embodiment, the homolog is selected from the group consisting of alfalfa, Arabidopsis, barley, Brassica campestris, oilseed rape, broccoli, cabbage, canola, citrus, cotton, garlic, oat, Allium, flax, an ornamental plant, peanut, pepper, potato, rapeseed, rice,<br>
rye, sorghum, strawberry, sugarcane, sugarbeet, tomato, wheat, poplar, pine, fir, eucalyptus, apple, lettuce, lentils, grape, banana, tea, turf grasses, sunflower, soybean, com, Phaseolus, crambe, mustard, castor bean, sesame, cottonseed, linseed, safflower, and oil palm. More particularly, preferred homologs are selected from canola, corn, Brassica campestris, oilseed rape, soybean, crambe, mustard, castor bean, peanut, sesame, cottonseed, linseed, rapeseed, safflower, oil palm, flax, and sunflower. In an even more preferred embodiment, the homolog is selected from the group consisting of canola, rapeseed, corn, Brassica campestris, oilseed rape, soybean, sunflower, safflower, oil palms, and peanut. In a particularly preferred embodiment, the homolog is soybean. In a particularly preferred embodiment, the homolog is canola. In a particularly preferred embodiment, the homolog is oilseed rape.<br>
Due to the degeneracy of the genetic code, different nucleotide codons may be used to code for a particular amino acid. A host cell often displays a preferred pattern of codon usage. Structural nucleic acid sequences are preferably constructed to utilize the codon usage pattern of the particular host cell. This generally enhances the expression of the structural nucleic acid sequence in a transformed host cell. Any disclosed nucleic acid or amino acid sequence may be modified to reflect the codon usage of a host cell or organism in which they are contained. Modification of a structural nucleic acid sequence for codon usage in plants is described, for example in U.S. Patents 5,689,052 and 5,500,365.<br>
In a preferred embodiment, any of the nucleic acid molecules of the present invention can be operably linked to a promoter region that functions in a plant cell to cause the production of an mRNA molecule, where the nucleic acid molecule that is linked to the<br>
promoter is heterologous with respect to that promoter. As used herein, "heterologous" means not naturally occurring together.<br>
Protein and Peptide Molecules<br>
A class of agents includes one or more of the polypeptide molecules encoded by a nucleic acid agent of the present invention. Another class of agents includes one or more polypeptide molecules of the present invention. A particular preferred class of proteins is that having an amino acid sequence selected from the group consisting of SEQ ID NOs: 14, 18, 20, 22, 24,26, and 28, and fragments thereof. Polypeptide agents may have C-terminal or N-terminal amino acid sequence extensions<br>
As used herein, the terms "protein," "peptide molecule," or "polypeptide" includes any molecule that comprises five or more amino acids. It is well known in the art that protein,<br>
peptide, or polypeptide molecules may undergo modification, including post-translational modifications, such as, but not limited to, disulfide bond formation, glycosylation, phosphorylation, or oligomerization. Thus, as used herein, the terms "protein," "peptide molecule," or "polypeptide" includes any protein that is modified by any biological or non-biological process. The terms "amino acid" and "amino acids" refer to all naturally occurring L-amino acids. This definition is meant to include norleucine, norvaline, ornithine, homocysteine, and homoserine.<br>
In a further aspect of the present invention, the DGAT2 proteins of the present invention have been solubilized. "Solubilization" refers to the extraction of the DGAT enzyme from the membranes in such a way that it then behaves in a manner typical of enzymes that are not membrane-associated.<br>
It should also be noted that plant DGAT proteins from a variety of sources can be used to investigate TAG biosynthesis in a wide variety of in vivo applications. Because all plant seeds appear to synthesize lipids via a common metabolic pathway, the study and/or application of one plant DGAT protein to a heterologous plant host may be readily achieved in a variety of species. In other applications, a plant DGAT protein can be used outside the native plant source of the DGAT protein to enhance the production and/or modify the composition of the TAG produced or synthesized in vitro.<br>
The percentage of sequence identity is determined by comparing two optimally aligned sequences over a comparison window, wherein the portion of the polynucleotide or amino acid sequence in the comparison window may comprise additions or deletions (i.e., gaps) as compared to the reference sequence (that does not comprise additions or deletions) for optimal alignment of the two sequences. The percentage is calculated by determining the number of positions at which the identical nucleic acid base or amino acid residue occurs in both sequences to yield the number of matched positions, dividing the number of matched positions by the total number of positions in the window of comparison and multiplying the result by one hundred to yield the percentage of sequence identity.<br>
A polypeptide or polynucleotide molecule can be substantially identical or substantially homologous to related molecules. These homologues with substantial identity to a related molecule generally comprise at least one polypeptide sequence or one polynucleotide sequence that has at least 70% sequence identity compared to other polypeptide sequences or polynucleotide sequences. The Gap program in the WISCONSIN PACKAGE version<br>
10.0-UNIX from Genetics Computer Group, Inc. based on the method of Needleman and Wunsch (J. Mol. Biol., 48:443-453 (1970)) using the set of default parameters for pairwise comparison (for amino acid sequence comparison: Gap Creation Penalty = 8, Gap Extension Penalty = 2; for nucleotide sequence comparison: Gap Creation Penalty = 50; Gap Extension Penalty = 3); or using the TBLASTN program in the BLAST 2.2.1 software suite (Altschul et al, Nucleic Acids Res., 25:3389-3402), using BLOSUM62 matrix (Henikoff and Henikoff, Proc. Natl. Acad. Sci. (U.S.A.), 89:10915-10919 (1992)) and the set of default parameters for pair-wise comparison (gap creation cost = 11, gap extension cost = 1). In BLAST, the E-value, or expectation value, represents the number of different alignments with scores equivalent to or better than the raw alignment score, S, that are expected to occur in a database search by chance. The lower the E value, the more significant the match. Because database size is an element in E-value calculations, E-values obtained by "BLASTing" against public databases, such as GenBank, have generally increased over time for any given query/entry match. Percent identity with respect to proteins refers to the percentage of identically matched amino acid residues that exist along the length of that portion of the sequences which is aligned by the BLAST algorithm.<br>
One aspect of the present invention provides an isolated polynucleic acid molecule comprising a nucleotide sequence or complement thereof, wherein the nucleotide sequence encodes a polypeptide that is substantially homologous to a protein amino acid sequence of the present invention, wherein substantially homologous is defined as at least about 50%, or at least about 60%, or at least about 70%, or at least about 75%, or at least about 80% sequence identity, or at least about 85% or at least about 90% sequence identity, or at least about 95% sequence identity, or at least about 98% sequence identity to a member selected from the group consisting of SEQ ID NOS: 14, 18, 20, 22, and 24.<br>
Agents of the invention include proteins and fragments thereof comprising at least about a contiguous 10 amino acid region preferably comprising at least about a contiguous 20 amino acid region, even more preferably comprising at least a contiguous 25, 35, 50, 75, or 100 amino acid region of a protein of the present invention. In another preferred embodiment, the proteins of the present invention include between about 10 and about 25 contiguous amino acid region, more preferably between about 20 and about 50 contiguous amino acid region, and even more preferably between about 40 and about 80 contiguous amino acid region.<br>
In addition to isolation of other DGAT proteins or genes, genes for other related acyltransferase proteins can also be obtained using the sequence information from the DGAT sequences of the present invention and related nucleic acid or amino acid sequences. For example, Schizosaccharomyces pornbe amino acid sequence homologs of DGAT2 proteins comprising the amino acids SEQ ID NOs: 29 and 30 are disclosed. In another example, other acyltransferase enzymes are involved in plant lipid biosynthesis, including lysophosphosphatidylcholine acyltransferase (LPCAT), lysophosphosphatidylserine acyltransferase (LPSAT), lysophosphosphatidylethanolamine acyltransferase (LPEAT), phosphatidylcholine diacylglycerol acyltransferase (PDAT), and lysophosphosphatidylinositol acyltransferase (LPIAT).<br>
DNA Constructs and Plant Transformants<br>
One or more of the nucleic acid molecules of the present invention may be used in plant transformation or transfection. Exogenous genetic material may be transferred into a plant cell and the plant cell regenerated into a whole, fertile, or sterile plant. Exogenous genetic material is any genetic material, whether naturally occurring or otherwise, from any source that is capable of being inserted into any organism. Bacterial plasmid maintainance elements are components of DNA constructs. These elements comprise antibiotic markers, i.e., the aadA gene (SPC/STR, spectomycin, and streptomycin resistance), Ec.nptll (neomycin phosphotransferase, kanamycin resistance); origins of reph'cation or elements that control plasmid copy number, i.e., Ec.oriV, Ec.ori322, ORI-PUC, and Ec.ROP. Additional information on these elements can be obtained from Sambrook et aL, Molecular Cloning, A Laboratory Manual, 2nd Ed., Cold Spring Harbor Press, Cold Spring Harbor, New York (1989), among others.<br>
In a preferred aspect of the present invention the exogenous genetic material comprises a nucleic acid sequence selected from the group consisting of SEQ ED NOs: 11, 12, 13,16,17, 19, 21, 23, and complements thereof, and fragments of either. In a further aspect of the present invention the exogenous genetic material comprises a nucleic acid sequence encoding an amino acid sequence selected from the group consisting of SEQ ID NOs: 14,18, 20, 22, 24, and fragments thereof.<br>
In an embodiment of the present invention, exogenous genetic material comprised of one or more genes is introduced into a plant. In one embodiment, preferred combinations of genes include, but are not limited to, one or more of the following genes: MrDGAT2A (SEQ<br>
ID NO: 1), M?-DAGAT2A.nno (SEQ ID NO: 15), MrDGAT2B (SEQ ID NO: 3), MrDGAT2B.nno (SEQ ID NO: 11), ScDGAT2 (SEQ ID NO: 5), ScDGAT2.nno (SEQ ID NO: 16), NcDGAT2 (SEQ ID NO: 13), and NcDGAT.nno (SEQ ID NO: 12). In another embodiment, preferred combinations of genes include, but are not limited to, one of the following genes expressed under the control of two separate promoters: MVDGAT2A (SEQ ID NO: 1), MrDAGAT2A.nno (SEQ ID NO: 15), MrDGAT2B (SEQ ID NO: 3), MrDGAT2B.nno (SEQ ID NO: 11), ScDGAT2 (SEQ ID NO: 5), ScDGAT2.nno (SEQ ID NO: 16), NcDGAT2 (SEQ ID NO: 13), and NcDGAT.nno (SEQ ID NO: 12).<br>
In such combinations, one or more of the gene products can be localized in the cytoplasm. Such genes can be introduced, for example, on a single construct in either a monocistronic or polycistronic arrangement, introduced on different constructs but in the same transformation event, or introduced into separate plants followed by one or more crosses to generate the desired combination of genes.<br>
Such genetic material may be transferred into either monocotyledons or dicotyledons including, but not limited to canola, com, soybean, Arabidopsis, Phaseolus, peanut, alfalfa, wheat, rice, oat, sorghum, rapeseed, rye, tritordeum, millet, fescue, perennial ryegrass, sugarcane, cranberry, papaya, banana, safflower, oil palms, flax, muskmelon, apple, cucumber, dendrobium, gladiolus, chrysanthemum, liliacea, cotton, eucalyptus, sunflower, Brassica campestris, oilseed rape, turf grass, sugarbeet, coffee, and dioscorea (Christou, In: Particle Bombardment for Genetic Engineering of Plants, Biotechnology Intelligence Unit, Academic Press, San Diego, California (1996), with canola, corn, Brassica campestris, oilseed rape, rapeseed, soybean, crambe, mustard, castor bean, peanut, sesame, cottonseed, linseed, safflower, oil palm, flax, and sunflower preferred, and canola, rapeseed, corn, Brassica campestris, oilseed rape, soybean, sunflower, safflower, oil palms, and peanut preferred. In a preferred embodiment, the homolog is selected from the group consisting of maize, soybean, canola, cottonseed, sesame, flax, peanut, sunflower, safflower, and oil palm. In a more preferred embodiment, the genetic material is transferred into canola. In another more preferred embodiment, the genetic material is transferred into oilseed rape. In another particularly preferred embodiment, the genetic material is transferred into soybean.<br>
Transfer of a nucleic acid molecule that encodes a protein can result in expression or overexpression of that polypeptide in a transformed cell or transgenic plant. One or more of the proteins or fragments thereof encoded by nucleic acid molecules of the present invention<br>
may be overexpressed in a transformed cell or transformed plant. Such expression or overexpression may be the result of transient or stable transfer of the exogenous genetic material.<br>
In a preferred embodiment, expression or overexpression of a polypeptide of the present invention in a plant provides in that plant, relative to an untransformed plant with a similar genetic background, an association with an increase in DGAT activity.<br>
The levels of products may be increased throughout an organism such as a plant or localized in one or more specific organs or tissues of the organism. For example the levels of products may be increased in one or more of the tissues and organs of a plant including without limitation: roots, tubers, stems, leaves, stalks, fruit, berries, nuts, bark, pods, seeds, and flowers. A preferred organ is a seed.<br>
In a preferred aspect, a similar genetic background is a background where the organisms being compared share about 50% or greater of their nuclear genetic material. In a more preferred aspect a similar genetic background is a background where the organisms being compared share about 75% or greater, even more preferably about 90% or greater of<br>
(<br>
their nuclear genetic material. In another even more preferable aspect, a similar genetic background is a background where the organisms being compared are plants, and the plants are isogenic except for any genetic material originally introduced using plant transformation techniques.<br>
A construct or vector may include a plant promoter to express the polypeptide of choice. In a preferred embodiment, any nucleic acid molecules described herein can be operably linked to a promoter region which functions in a plant cell to cause the production of an mRNA molecule. For example, any promoter that functions in a plant cell to cause the production of an mRNA molecule, such as those promoters described herein, without limitation, can be used. In a preferred embodiment, the promoter is a plant promoter.<br>
Other promoters can also be used to express a polypeptide in specific tissues, such as seeds or fruits. Indeed, in a preferred embodiment, the promoter used is a seed specific promoter. Examples of such promoters include the 5' regulatory regions from such genes as napin (Kridl et al, SeedSci. Res., 1:209:219 (1991)), phaseolin (Bustos et al, Plant Cell, l(9):839-853 (1989)), soybean trypsin inhibitor (Riggs et al, Plant Cell, 1(6):609-621 (1989)), ACP (Baerson et al, Plant MoL Biol., 22(2):255-267 (1993)), stearoyl-ACP desaturase (Slocombe et al, Plant Physiol, 104(4): 167-176 (1994)), soybean a' subunit of<br>
p-conglycinin (P-Gm7S, see, for example, Chen et al, Proc. Natl. Acad. Sci., 83:8560-8564 (1986)), Vidafaba USP (P-Vf.Usp, see, for example, SEQ ID NOs: 1,2, and 3, U.S. Application 10/429,516), and Zea mays L3 oleosin promoter (P-Zm.L3, see, for example, Hong et al, Plant Mol. Biol, 34(3):549-555 (1997)). Also included are the zeins, which are a group of storage proteins found in com endosperm. Genomic clones for zein genes have been isolated (Pedersen et al., Cell, 29:1015-1026 (1982), and Russell et al, Transgenic Res., 6(2):157-168) and the promoters from these clones, including the 15 kD, 16 kD, 19 kD, 22 kD, 27 kD, and genes, could also be used. Other promoters known to function, for example, in com include the promoters for the following genes: waxy, Brittle, Shrunken 2, Branching enzymes I and n, starch synthases, debranching enzymes, oleosins, glutelins, and sucrose synthases. A particularly preferred promoter for corn endosperm expression is the promoter for the glutelin gene from rice, more particularly the Osgt-1 promoter (Zheng et al., Mol. Cell Biol, 13:5829-5842 (1993)). Examples of promoters suitable for expression in wheat include those promoters for the ADPglucose pyrosynthase (ADPGPP) subunits, the granule bound and other starch synthase, the branching and debranching enzymes, the embryogenesis-abundant proteins, the gliadins, and the glutenins. Examples of such promoters in rice include those promoters for the ADPGPP subunits, the granule bound and other starch synthase, the branching enzymes, the debranching enzymes, sucrose synthases, and the glutelins. A particularly preferred promoter is the promoter for rice glutelin, Osgt-1. Examples of such promoters for barley include those for the ADPGPP subunits, the granule bound and other starch synthase, the branching enzymes, the debranching enzymes, sucrose synthases, the hordeins, the embryo globulins, and the aleurone specific proteins. A preferred promoter for expression in the seed is a napin promoter, referred to herein as P-Br.Snap2. Another preferred promoter for expression is an ArcelinS promoter (U.S. Patent Publication 2003/0046727). Yet another preferred promoter is a soybean 7S promoter (P-Gm.7S) and the soybean 7Sa' beta conglycinin promoter (P-Gm.Sphasl).<br>
Promoters, which can cause the overexpression of the polypeptide of the present invention, are generally known in the art, e.g., viral promoters (P-CaMV35S, U.S. Patent 5,352,605; P-FMV35S, and its enhancer element E-FMV35S identified as the 5' portion of the P-FMV35S without the native start of transcription, U.S. Patents 5,378,619 and 5,018,100, and chimeric promoter molecules described in U.S. Patent 6,462,258 as SEQ ED NO: 28 referrerd to in the present invention as E-FMV35S/P-At.Tsfl), and various plant derived<br>
promoters, e.g., plant actin promoters (P-Os.Actl, and genetic elements derived therefrom, U.S. Patents 5,641,876 and 6,429,357).<br>
Additional promoters that may be utilized are described, for example, in U.S. Patents 5,378,619; 5,391,725; 5,428,147; 5,447,858; 5,608,144; 5,608,144; 5,614,399; 5,633,441; 5,633,435; and 4,633,436. In addition, a tissue specific enhancer may be used (Fromm et al., The Plant Cell, 1:977-984 (1989)).<br>
Constructs or vectors may also include, with the coding region of interest, a nucleic acid sequence that acts, in whole or in part, to terminate transcription of that region. A number of such sequences have been isolated, including the Tr7 3' sequence and the NOS 3' sequence (Ingelbrecht et al., Tlie Plant Cell, 1:671-680 (1989); Bevan et al, Nucleic Acids Res., 11:369-385 (1983)). Regulatory transcript termination regions can be provided in plant expression constructs of this present invention as well. Transcript termination regions can be provided by the DNA sequence encoding the gene of interest or a convenient transcription termination region derived from a different gene source, for example, the transcript termination region that is naturally associated with the transcript initiation region. The skilled artisan will recognize that any convenient transcript termination region that is capable of terminating transcription in a plant cell can be employed in the constructs of the present invention.<br>
A vector or construct inay also include additional regulatory elements. Examples of such include the translation leader isolated from Petunia hybrida Hsp70 gene (L-Ph.DnaK, U.S. Patent 5,362,865), the Adh intron 1 (Callis ei al., Genes and Develop., 1:1183-1200 (1987)), the sucrose synthase intron (Vasil et al, Plant Physiol, 91:1575-1579 (1989)), the rice actin intron (I-Os.Actl U.S. Patent 5,641,876), and the TMV omega element (Gallic et al, The Plant Cell, 1:301-311 (1989)).   Transcriptional termination regions, e.g., the 31 untranslated region from the Brassica rapa, T-Br.Snap2. These and other regulatory elements may be included when appropriate,<br>
A vector or construct may also include a selectable marker. Selectable markers may also be used to select for plants or plant cells that contain the exogenous genetic material. Examples of such include, but are not limited to: a nptH gene (Potrykus et al, Mol. Gen. Genet., 199:183-188 (1985)), which codes for kanamycin resistance and can be selected for using kanamycin, Rptll, G41S, hpt etc.; a bar gene which codes for bialaphos resistance; a mutant EPSP synthase gene (Hinchee etal, Bio/Technology, 6:915-922 (1988); Reynaerts et<br>
al, Selectable and Screenable Markers. In Gelvin and Schilperoort; Plant Molecular Biology Manual, Kluwer, Dordrecht (1988); Reynaerts et al, Selectable and Screenable Markers. In Gelvin and Schilperoort; Plant Molecular Biology Manual, Kluwer, Dordrecht (1988); a nitrilase gene which confers resistance to bromoxynil (Stalker et al, J. Biol diem., 263:6310-6314 (19S8)); a mutant acetolactate synthase gene (ALS) which confers imidazolinone or sulphonylurea resistance (European Patent Application 154,204 (Sept. 11, 1985)), ALS (D'Halluin etal, Bio/Technology, 10:309-314 (1992)), and a methotrexate resistant DHFR gene (Thillet et al, J. Biol Chem., 263:12500-12508 (1988)). A particularly preferred marker is glyphosate tolerance, this can be achieved in plants by expressing a glyphosate resistant EPSPS, for example, the aroA-CP4 coding sequence from Agrobacterium tumefaciens (U.S. Patent 5,633,435), herein referred to as AGRtu.aroA. The AGRtu.aroA coding sequence is linked to a chloroplast transit peptide (CTP), for example the CTP2 coding sequence isolated from the Arabidopsis ShkF gene, herein referred to as TS-At.ShkF-CTP2.<br>
In a preferred embodiment of the present invention, a transgenic plant expressing the desired protein is to be produced. Various methods for the introduction of a desired polynucleotide sequence encoding the desired protein into plant cells are available and known to those of skill in the art and include, but are not limited to: (1) physical methods such as microinjection, electroporation, and microprojectile mediated delivery (biolistics or gene gun technology); (2) virus mediated delivery methods; and (3) Agrobacterium-mediated transformation methods.<br>
The most commonly used methods for transformation of plant cells are the Agrobacteri.um-media.ted DNA transfer process and the biolistics or microprojectile bombardment mediated process (i.e., the gene gun). Typically, nuclear transformation is desired but where it is desirable to specifically transform plastids, such as chloroplasts or amyloplasts, plant plastids may be transformed utilizing a microprojectile mediated delivery of the desired polynucleotide.<br>
Agrobacteriwn-mediated transformation is achieved through the use of a genetically engineered soil bacterium belonging to the genus Agrobacterium. A number of wild-type and disarmed strains of Agrobacterium tumefaciens and Agrobacterium rhizogenes harboring Ti or Ri plasmids can be used for gene transfer into plants. Gene transfer is done via the transfer of a specific DNA known as "T-DNA" that can be genetically engineered to carry any desired piece of DNA into many plant species. The transgene(s) are constructed in a DNA plasmid<br>
vector and are usually bordered by an Agrobacterium Ti plasmid right border DNA region (RB) and a left border DNA region (LB). During the process of Agrobacterium mediated transformation the DNA plasmid is nicked by VirDl and VirD2 endonucleases at the right and left border regions and the T-DNA region is inserted into the plant genome.<br>
Agrobacterium-medial&amp;d. genetic transformation of plants involves several steps. The first step, in which the virulent Agrobacterium and plant cells are first brought into contact with each other, is generally called "inoculation." Following the inoculation, the Agrobacterium and plant cells/tissues are permitted to be grown together for a period of several hours to several days or more under conditions suitable for growth and T-DNA transfer. This step is termed "co-culture." Following co-culture and T-DNA delivery, the plant cells are treated with bactericidal or bacteriostatic agents to kill the Agrobacterium remaining in contact with the explant and/or in the vessel containing the explant. If this is done in the absence of any selective agents to promote preferential growth of transgenic versus non-transgenic plant cells, then this is typically referred to as the "delay" step. If done in the presence of selective pressure favoring transgenic plant cells, then it is referred to as a   . "selection" step. When a "delay" is used, it is typically followed by one or more "selection" steps.<br>
With respect to microprojectilebombardment (U.S. Patents 5,550,318; 5,538,880; and: 5,610,042; each of which is specifically incorporated herein by reference in its entirety), particles are coated with nucleic acids and delivered into cells by a propelling force. Exemplary particles include those comprised of tungsten, platinum, and preferably, gold.<br>
Microprojectile bombardment techniques are widely applicable, and may be used to transform virtually any plant species. Examples of species that have been transformed by microprojectile bombardment include monocot species such as maize, barley, wheat (U.S. Patent 5,563,055, incorporated herein by reference in its entirety), rice, oat, rye, sugarcane, and sorghum; as well as a number of dicots including tobacco, soybean (U.S. Patent 5,322,783, specifically incorporated herein by reference in its entirety), sunflower, peanut, cotton, tomato, and legumes in general (U.S. Patent 5,563,055, incorporated herein by reference in its entirety).<br>
To select or score for transformed plant cells regardless of transformation methodology, the DNA introduced into the cell contains a gene that functions in a regenerable plant tissue to produce a compound that confers upon the plant tissue resistance to an<br>
otherwise toxic compound. Genes of interest for use as a selectable, screenable, or scorable marker would include but are not limited to GUS, green fluorescent protein (GFP), luciferase (LUX), antibiotic or herbicide tolerance genes. Examples of antibiotic resistance genes include the penicillins, kanamycin (and neomycin, G418, bleomycin); methotrexate (and trimethoprim); chloramphenicol; kanamycin; and tetracycline.<br>
The regeneration, development, and cultivation of plants from various transformed explants is well documented in the art. This regeneration and growth process typically includes the steps of selecting transformed cells and culturing those individualized cells through the usual stages of embryonic development through the rooted plantlet stage. Transgenic embryos and seeds are similarly regenerated. The resulting transgenic rooted shoots are thereafter planted in an appropriate plant growth medium such as soil. Cells that survive the exposure to the selective agent, or cells that have been scored positive in a screening assay, may be cultured in media that supports regeneration of plants. Developing plantlets are transferred to soil less plant growth mix, and hardened off, prior to transfer to a greenhouse or growth chamber for maturation.<br>
The present invention can be used with any transformable cell or tissue. By transformable as used herein is meant a cell or tissue that is capable of further propagation to give rise to a plant. Those of skill in the ait recognize that a number of plant cells or tissues are transformable in which after insertion of exogenous DNA and appropriate culture conditions the plant cells or tissues can form into a differentiated plant. Tissue suitable for these purposes can include but is not limited to immature embryos, scutellar tissue, suspension cell cultures, immature inflorescence, shoot meristem, nodal explants, callus tissue, hypocotyl tissue, cotyledons, roots, and leaves.<br>
Any suitable plant culture medium can be used. Examples of suitable media would include but are not limited to MS-based media (Murashige and Skoog, Physiol. Plant, 15:473-497 (1962)) or N6-based media (Chu et al., Scientia Sinica, 18:659 (1975)) supplemented with additional plant growth regulators including but not limited to auxins, cytokinins, ABA, and gibberellins. Those of skill in the art are familiar with the variety of tissue culture media, which when supplemented appropriately, support plant tissue growth and development and are suitable for plant transformation and regeneration. These tissue culture media can either be purchased as a commercial preparation, or custom prepared, and modified. Those of skill in the art are aware that media and media supplements such as<br>
nutrients and growth regulators for use in transformation and regeneration and other culture conditions such as light intensity during incubation, pH, and incubation temperatures that can be optimized for the particular variety of interest.<br>
A transgenic plant formed using Agrobacterium transformation methods typically contains a single gene on one chromosome. Such transgenic plants can be referred to as being heterozygous for the added gene. More preferred is a transgenic plant that is homozygous for the added structural gene; i.e., a transgenic plant that contains two added genes, one gene at the same locus on each chromosome of a chromosome pair. A homozygous transgenic plant can be obtained by sexually mating (selfing) an independent segregant, transgenic plant that contains a single added gene, germinating some of the seed produced and analyzing the resulting plants produced for the gene of interest.<br>
It is also to be understood that two different transgenic plants can also be mated to produce offspring that contain two independently segregating, exogenous genes. Selfing of appropriate progeny can produce plants that are homozygous for both added, exogenous genes that encode a polypeptide of interest. Back-crossing to a parental plant and out-crossing with a non-transgenic plant are also contemplated, as is vegetative propagation.<br>
Anti-sense suppression of genes in plants by introducing by transformation of a construct comprising DNA of the gene of interest in an anti-sense orientation is disclosed in U.S. Patents 5,107,065; 5,453,566; 5,759,829; 5,874,269; 5,922,602; 5,973,226; and 6,005,167; all of which are incorporated herein by reference<br>
Co-suppression of genes in a plant by introducing by transformation of a construct for cytoplasmic expression comprising DNA of the gene of interest in a sense orientation is disclosed, for example, in U.S. Patents 5,034,323; 5,231,020; 5,283,184; and 6,271,033, all of which are incorporated herein by reference.<br>
Antisense approaches are a way of preventing or reducing gene function by targeting the genetic material (Mol et al, FEES Lett., 268:427-430 (1990)). The objective of the antisense approach is to use a sequence complementary to the target gene to block its expression and create a mutant cell line or organism in which the level of a single chosen protein is selectively reduced or abolished. Antisense techniques have several advantages over other 'reverse genetic' approaches. The site of inactivation and its developmental effect can be manipulated by the choice of promoter for antisense genes or by the timing of external application or microinjection. Antisense can manipulate its specificity by selecting either<br>
unique regions of the target gene or regions where it shares homology to other related genes (Hiatt et al, In: Genetic Engineering, Setlow (ed.), New York: Plenum 11:49-63 (1989)).<br>
The present invention also provides for parts of the plants, particularly reproductive or storage parts, of the present invention. Plant parts, without limitation, include seed, endosperm, ovule, and pollen. In a particularly preferred embodiment of the present invention, the plant part is a seed. In one embodiment the seed is a constituent of animal feed.<br>
Any of the plants or parts thereof of the present invention that can provide a processed product comprising feed, meal, protein, flour, fiber, extactable nutrients, or oil preparation. A particularly preferred plant part for this purpose is a seed. In a preferred embodiment the processed product is designed for livestock animals or humans, or both. Methods to produce feed, meal, protein, and oil preparations are known in the art. See, for example, U.S. Patents 4,957,748; 5,100,679; 5,219,596; 5,936,069; 6,005,076; 6,146,669; and 6,156,227, herein incorporated by reference. In a preferred embodiment, the protein preparation is a high protein preparation. Such a high protein preparation preferably has a protein content of greater than 5% w/v, more preferably 10% w/v, and even more preferably 15% w/v. In a preferred oil preparation, the oil preparation is a high oil preparation with an oil content derived from a plant or part thereof of the present invention of greater than 5% w/v, more preferably 10% w/v, and even more preferably 15% w/v. In a preferred embodiment the oil preparation is a liquid and of a volume greater than 1, 5,10, or 50 liters. The present invention provides for oil produced from plants of the present invention or generated by a method of the present invention. Such an oil may exhibit enhanced oxidative stability. Also, such oil may be a minor or major component of any resultant product. Moreover, such oil may be blended with other oils. In a preferred embodiment, the oil produced from plants of the present invention or generated by a method of the present invention constitutes greater than 0.5%, 1%, 5%, 10%, 25%, 50%, 75%, or 90% by volume or weight of the oil component of any product. In another embodiment, the oil preparation may be blended and can constitute greater than 10%, 25%, 35%, 50%, or 75% of the blend by volume. Oil produced from a plant of the present invention can be admixed with one or more organic solvents or petroleum distillates.<br>
Plants of the present invention can be part of or generated from a breeding program. The choice of breeding method depends on the mode of plant reproduction, the heritability of the trait(s) being improved, and the type of cultivar used commercially (e.g., F| hybrid<br>
cultivar, pureline cultivar, etc). A breeding program can be enhanced using marker-assisted selection of the progeny of any cross. It is further understood that any commercial and non­commercial cultivars can be utilized in a breeding program. Factors such as, for example, emergence vigor, vegetative vigor, stress tolerance, disease resistance, branching, flowering, seed set, seed size, seed density, standability, and threshability etc. will generally dictate the choice.<br>
Exemplary Uses<br>
Nucleic acid molecules and fragments thereof of the present invention may be employed to obtain other nucleic acid molecules from the same species (nucleic acid molecules from com may be utilized to obtain other nucleic acid molecules from corn). Such nucleic acid molecules include the nucleic acid molecules that encode the complete coding sequence of a protein and promoters and flanking sequences of such molecules. In addition, such nucleic acid molecules include nucleic acid molecules that encode for other isozymes or gene family members. Such molecules can be readily obtained by using the above-described nucleic acid molecules or fragments thereof to screen cDNA or genomic libraries. .Methods for forming such libraries are well known in the art.<br>
Nucleic acid molecules and fragments thereof of the present invention may also be employed to obtain nucleic acid homologs. Such homologs include the nucleic acid molecules of plants and other organisms, including bacteria and fungi, including the nucleic acid molecules that encode, in whole or in part, protein homologs of other plant species or other organisms, sequences of genetic elements, such as promoters and transcriptional regulatory elements. Such molecules can be readily obtained by using the above-described nucleic acid molecules or fragments thereof to screen cDNA or genomic libraries obtained from such plant species. Methods for forming such libraries are well known in the art. Such homolog molecules may differ in their nucleotide sequences from those found in one or more of the sequences selected from the group consisting of SEQ ID NOs: 11, 12,13, 16, 17,19, 21,23, 25, and 27, and complements thereof, because complete complementarity is not needed for stable hybridization. The nucleic acid molecules of the present invention therefore also include molecules that, although capable of specifically hybridizing with the nucleic acid molecules, may lack "complete complementarity."<br>
Promoter sequences and other genetic elements, including but not limited to transcriptional regulatory flanking sequences, associated with one or more of the disclosed<br>
nucleic acid sequences can also be obtained using the disclosed nucleic acid sequence provided herein. In one embodiment, such sequences are obtained by incubating nucleic acid molecules of the present invention with members of genomic libraries and recovering clones that hybridize to such nucleic acid molecules thereof. In a second embodiment, methods of "chromosome walking," or inverse PCR may be used to obtain such sequences (Frohman et al, Proc. Natl. Acad. Sci. (U.S.A.), 85:8998-9002 (1988); Ohara et al, Proc. Natl. Acad. Sci. (U.S.A.), 86:5673-5677 (1989); Pang et al, Biotechniques, 22:1046-1048 (1977); Huang et al., Methods Mol. Biol, 69:89-96 (1997); Huang et al, MetliodMol Biol., 67:287-294 (1997); Benkel et al, Genet. Anal, 13:123-127 (1996); Hartl et al, Methods Mol Biol, 58:293-301 (1996)). The term "chromosome walking" means a process of extending a genetic map by successive hybridization steps.<br>
The nucleic acid molecules of the present invention may be used to isolate promoters of cell-enhanced, cell-specific, tissue-enhanced, tissue-specific, developmentally- or environmentally-regulated expression profiles. Isolation and functional analysis of the 5' flanking promoter sequences of these genes from genomic libraries, for example, using genomic screening methods and PCR techniques would result in the isolation of useful promoters and transcriptional regulatory elements. These methods are known to those of skill in the art and have been described (see, for example, Birren et al, Genome Analysis: Analyzing DNA, 1 (1997), Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York). Promoters obtained utilizing the nucleic acid molecules of the present invention could also be modified to affect their control characteristics. Examples of such modifications would include but are not limited to enhancer sequences. Such genetic elements could be used to enhance gene expression of new and existing traits for crop improvement.<br>
Another subset of the nucleic acid molecules of the present invention includes nucleic acid molecules that are markers. The markers can be used in a number of conventional ways in the field of molecular genetics. Such markers include nucleic acid molecules SEQ ID NOs: 11, 12, 13, 16, 17, 19, 21, 23, 25, and 27, and complements thereof, and fragments of either that can act as markers and other nucleic acid molecules of the present invention that can act as markers.<br>
In an aspect of the present invention, one or more of the nucleic molecules of the present invention are used to determine the level (i.e., the concentration of mRNA in a sample, etc.} in a plant (preferably canola, com, Brassica campestris, oilseed rape, rapeseed,<br>
soybean, crambe, mustard, castor bean, peanut, sesame, cottonseed, linseed, safflower, oil palm, flax, or sunflower) or pattern (i.e., the kinetics of expression, rate of decomposition, stability profile, etc.) of the expression of a protein encoded in part or whole by one or more of the nucleic acid molecule of the present invention.<br>
A number of methods can be used to compare the expression between two or more samples of cells or tissue. These methods include hybridization assays, such as Northerns, RNAse protection assays, and in situ hybridization. Alternatively, the methods include PCR-type assays. In a preferred method, the expression is compared by hybridizing nucleic acids from the two or more samples to an array of nucleic acids. The array contains a plurality of suspected sequences known or suspected of being present in the cells or tissue of the samples.<br>
The present invention now being generally described, it will be more readily understood by reference to the following examples, which are included for purposes of illustration only and are not intended to limit the present invention.<br>
EXAMPLES EXAMPLE 1 Isolation of DGAT2 Nucleic Acid Sequences and Confirmation of DGAT Activity.<br>
Mortierella ramanniana was cultured as described by Kamisaka, Y. et al., Lipids, 28:583-587 (1993). Cells were harvested by passing 10-13 day old cultures through Miracloth and removing excess liquid by hand-wringing. Wet packed cells were stored at -70°C. Purification of DGAT2 proteins from Mortierella ramanniana was performed as follows. Lipid bodies were isolated from 70-75g of wet packed cells. Immediately prior to use, cells were thawed on ice and resuspended in 200 mL of Buffer D (10 mM potassium phosphate (pH 7.0), 1 M KCI, 0.5 M sucrose, 1 mM EDTA). Samples were lysed with an equal volume of 0.5 mm glass beads in a cell disrupter (Bead-Beater, Biospec Products, Bartlesville, OK) set on 'Homogenize' for 45-90 seconds. The cell slurry containing glass beads was centrifuged at 500 x g, the supernatant was removed, and the pellets were washed with another 5 mL of Buffer D. Following centrifugation, the supernatants from both centrifugations were combined. It was divided into six ultracentrifuge tubes (25 x 89 mm) and each was overlaid with 5 mL of Buffer E (10 mM potassium phosphate, pH 7.0, 1 M KCI, and 0.3 M sucrose). Samples were centrifuged at 100,000 x g at 4°C for 3 hours. The lipid body fractions, floating on top of the overlays, were combined and solubilized in 50 mL of<br>
Buffer F (10 mM potassium phosphate (pH 7.0), 75 mM KC1, 0.5 M Sucrose and 1.5% Triton X-100). Non-solubilized material was removed by ultracentrifugation (90,000 x g for 1.8 hours). The floating lipid layer was discarded and the supernatant containing the solubilized fraction (Triton X-100 extract) was retained for column purification. DGAT activity was measured as the production of 14C triacylglycerol from [l-14C]oleoyl-CoA and unlabeled dioleoyl-DAG. For non-solubilized samples the reaction mixture (0.1 mL) consisted of enzyme extract, 3.67 uM [l-14C]oleoyl-CoA, and 1.5 mM 1,2-18:1 diacylglycerol in a buffer containing 10 mM potassium phosphate (pH 7.0), 100-150 mM KC1, and 0.1% Triton x-100 (w/v). Assay mixtures were incubated at 25°C for 5 minutes and reactions were terminated by adding 1.5 mL of heptane:isopropanol:0.5 M H2SO4 (10:40:1, v/v/v). For solubilized samples 1,2-18:1 DAG was reduced to 0.5 mM, Triton X-100 was increased to 0.2%, and SOOuM L-a-phosphatidic acid was included. The L-a-phosphatidic acid was required to recover activity following solubilization with detergent as described by Kamiska et al., J. Biochem., 119:520-523 (1996) except, 300 uM phosphatidic acid was used rather than 500 uM. This resulted in a greater stimulation of activity.<br>
Following solubilization, product formation was dependent on the addition of exogenous DAG. Under these conditions the reaction rate was linear with respect to time for up to 10 minutes. After the assay was stopped, radiolabeled glycerolipids were isolated by adding 0.1 mL of 1 M NaHCO3 and 1 mL of heptane containing 15 nmoles/mL triolein as a carrier. The mixture was vortexed and the upper organic phase was removed to a new glass vial. The organic extract was back-extracted with 1 mL of 1 M NaCl. Forty percent of the final organic phase was removed for liquid scintillation counting and the remaining organic phase evaporated to dryness under nitrogen gas. The residue was resuspended in hexane and subjected to TLC on silica gel-G with a preadsorbent loading zone (Analtech #31011, Newark, Delaware). The TLC plate was developed in hexanerdiethyl ethenacetic acid (50:50:1, v/v/v), before drying and scanning by a radio-image analyzer (AMBIS 3000, AMBIS, Inc., San Diego, California) to determine the portion of radioactivity incorporated into TAG. Confirmation of TAG activity on the TLC plate was determined by co-migration of the unlabeled triolein earner and the [' C]TAG following exposure to iodine vapor.<br>
DGAT activity in the Triton X-100 extract was further purified by dye-binding chromatography on a Yellow 86-Agarose column (2.5 cm x 6.4 cm) equilibrated with 75 mM KC1 in Buffer G (10 mM potassium phosphate (pH 7.0), 0.1% (w/v) Triton X-100,10% (w/v) glycerol). The column was washed with 5 volumes of equilibration buffer at 2 mLper<br>
minute, then the activity was eluted with 500 mM KC1 in Buffer G. DGAT activity is stable to freeze/thaw at this stage of purification, so eluted fractions were assayed immediately and active fractions were stored at -70°C. In order to maintain maximal activity, subsequent chromatography was performed and fractions were assayed on the same day. Four preparations of Yellow 86-Agarose-purified activity were combined and concentrated 12-fold by ultrafiltration (YM-30 membrane, Amicon, Beverly, MA). The activity was further purified by hydroxyapatite chromatography on a 1.0 cm x 25.5 cm column equilibrated with 500 mM KC1 in Buffer G. The column was washed with 40 mL of equilibration buffer before bound proteins were eluted with a step gradient to 100 mM di-potassium phosphate in the equilibration buffer. Fractions from the flow-through containing DGAT activity were pooled and diluted 1:3.3 in Buffer G to reduce the KC1 concentration from 500 to 150 mM. The diluted sample was applied to a heparin column CL-6B (0.55 x 4.7 cm) equilibrated with 150 mM KC1 in Buffer G. The column was washed with 5 volumes of equilibration buffer at 0.5 mL/minute and bound proteins were eluted in a 10 mL linear gradient of 150-500 mM KC1 followed by 10 mL of 500 mM KC1 in Buffer G at 0.25 mL/minute. Fractions of 1.1 mL were collected. Two activity peaks were eluted from the heparin column (fnx 22 and fxn 28). A summary of the protein purification scheme is shown in Table 1. A lipid body fraction isolated from 300g of M. ramanniana cell paste was used for the preparation. Recovery values for Mr-DGAT2A (Heparin fxn 28) and Mr-DGAT2B (Heparin fxn 22) are reported separately in the last chromatographic step.<br>
(Table Remove)<br>
Polyacrylamide gradient gel electrophoresis (10-13%) was carried out according to the method of Laemmli, Nature, 227680-227685 (1970) with some of the modifications of Delepelaire, Proc. Nat. Acad. Sci., 76:115-115 (1979). The resolving gel contained a 10-13% linear gradient of acrylamide stock stabilized by a 0-10% linear gradient of sucrose. Proteins<br>
were visualized by staining with silver according to the method of Blum et al, Electrophoresis, 8:93-99 (1987), or with Coomassie Blue (0.1% Coomassie Blue R-250, 50% methanol (v/v), 10% acetic acid (v/v)).<br>
Several protein bands (36.5 kD, 36 kD, 35 kD, and 34 kD) were associated with the first peak of activity (fxn 22). The 34 kD band did not correlate with DGAT activity in all chromatographic steps, so it was eliminated (i.e., data not shown). The second peak (fxn 28) had a higher specific activity (Table 2) and contained a major protein band at 36 kD by SDS-PAGE. Three proteins (36.5 kD, 36 kD, and 35 kD) were identified from the purification as potential DGAT candidates.<br>
Degenerate primers designed from the amino acid sequences generated from the 36 kD peptide, were constructed in both sense and antisense orientations. These primers were employed in different combinations to amplify cDNA produced from Mortierella ramanniana total RNA. Total RNA was prepared from wet packed cells essentially as described by Jones et al, The Plant Cell, 7:359-371 (1995). cDNA was synthesized from the RNA using the Marathon cDNA Amplification Kit (BD Biosciences Clontech, Inc. Palo Alto, California). The amplification mixture consisted of template, polymerase chain reaction buffer, 200-300 ng of each primer, 2.5 mM dNTP, and 1 unit of AmpliTaq Gold polymerase (Perkin Elmer, Norwalk, CT) in 50 u,L. The amplification program consisted of one 10-minute hold at 95°C, and 30 cycles of denaturation (94°C, 30 seconds), annealing (62°C, 10 seconds, 10% ramp to 50°C, 15 seconds), and primer extension (72°C, 2 minutes). Products of the reaction were separated on a 0.7% agarose gel, excised, and purified according to the QIAPREP DNA extraction handbook (Qiagen, Santa Clara, California). The purified products were cloned into the pCR2.1TOPO vector (Invitrogen, Carlsbad, California) and analyzed by DNA sequencing. Comparisons between peptide sequences obtained by Edman degradation that were not used to design the primers and the deduced amino acid sequences of PCR products were used to confirm the identity of the fragments.<br>
RACE reactions (Marathon cDNA Amplification Kit) using primers specific to these fragments were performed to yield a 1312 base pair (bp) long cDNA that was cloned into the pCR2.1-TOPO vector. The most 5' ATG codon of this reading frame was located at bp 76, allowing for the translation of a polypeptide of 355 amino acids in length (Figure 1, MrDGAT2A).<br>
Genbank searches showed that these polypeptides are not sequence-related to the known DGAT1 or any other acyl transferases, but are members of a previously unannotated gene family present in major phyla of eukaryotes, in particular fungi, plants, animal, and basal eukaryotes.<br>
The commercial BAC-to-BAC Baculovirus Expression System (Life Technologies, Inc., Gaithersburg, MD) was used to express full-length proteins of Mortierella ramanniana DGAT2A and DGAT2B in cultured insect (sf9) cells. Full-length DGAT2 open reading frames were amplified by PCR employing primers containing restriction sites at the 5' ends (NotI and Spel to the sense primers and PstI to the antisense primers). The PCR products were cloned into the pCR2.1TOPO vector and sequenced to confirm the fidelity of the constructs. Full-length cDNAs in pCR2.1-TOPO vectors were digested with NotI and PstI and cloned into the NotI and PstI restriction sites of the pFASTBACl vector (Life Technologies, Inc.). The baculovirus expression system can be used to express the full length cDNA encoding the polypeptides that are set forth in SEQ ID NOs: 18,20, 22, 24,26, and 28 to determine DGAT activity.<br>
Insect cells (1 x 106 cells/mL) were infected at a multiplicity of infection (MOI) of 0.05-0.1 and harvested after 5 days at 27°C by centrifugation. Pelleted cells were re-suspended in Buffer H (100 mM Tricine-NaOH, pH 7.8,10% glycerol, 100 mM NaCl) and lysed by sonication (2 x 10 seconds). Cell walls and other debris were pelleted by centrifugation and discarded. Membranes were harvested by centrifugation of the supernatant fraction (100,000 x g for one hour) and pellets were resuspended in Buffer H for enzyme assay. DGAT activity in insect cell membranes was measured as the production of 14C triacylglycerol from [l-14C]oleoyl-CoA and unlabeled dioleoyl-DAG. The reaction mixture (0.1 mL) consisted of isolated membranes, 3.5 uM [l-14C]oleoyl-CoA, 21.5 uM oleoyl-CoA and 200 uM 1,2-18:1 diacylglycerol in a buffer containing 25-30 mM Tricine (pH 7.8), 50-60 mM NaCl, and 0.06% CHAPS (w/v). Assay mixtures were incubated at 25°C for 5-10 minutes and reactions were terminated by adding 1.5 mL of heptane:isopropanol:0.5 M (10:40:1, v/v/v). Samples were processed as described above. Assays were linear with respect to protein and time.<br>
A significant elevation in DGAT activity was detected relative to untransformed sf9 cells for both Mortierella ramaimiana DGAT2A (94-fold) and DGAT2B proteins (37-fold) (Table 2).<br>
(Table Remove)<br>
Enzymological properties of the expressed Mortierella ramanniana DGAT2A and DGAT2B genes were also investigated. The effect of pH on DGAT activity was evaluated over a range of 4.0 to 11.0. The pH optimum for both enzymes was observed at 6.8. No differences were detected between the two polypeptides with respect to pH. A difference was observed in their response to temperature. The temperature optimum for DGAT2A was 37°C whereas DGAT2B does not demonstrate an optimum temperature<br>
EXAMPLE 2<br>
Isolation of Neurospora crassa DGAT2 Nucleic Acid Sequence and Confirmation of DGAT Activity.<br>
The following protocol was used to obtain the entire coding region corresponding to the Neurospora crassa DGAT2 protein (NcDGAT2). RNA was isolated from Neurospora crassa mating type A (Fungal Genetics Stock Center, Kansas City, Kansas) mycelium using Tri-Reagent (Sigma, St. Louis, MO) according to the manufacturer's protocol. First-strand cDNA synthesis was completed using the SMART cDNA Amplification kit (Clontech, California). Based on sequence comparisons to the Neurospora crassa genomic sequences, gene specific primers were designed to amplify the full-length coding regions of the NcDGAT2 sequence. Additional restriction sites were introduced to facilitate cloning (Hindm and Rsrll), using the primers designated SEQ ID NO: 7 and SEQ ID NO: 8 (Figure 3). The PCR product was cloned into plasmid pCR2.1 according to the manufacturer's protocol (Invitrogen) to yield plasmid pMON69834. Double-stranded DNA sequencing was done to verify the sequence (SEQ ID NO: 13). For expression of the NcDGAT2 protein in insect cells using a baculovirus expression system, the Rsrll -Hindlll fragment of pMON69834 was cloned into Rsi-n-HindlH-digested plasmid pFASTBACl (Gibco-BRL,<br>
Gaithersburg, MD). The resulting plasmid, pMON69839, was transformed into E. coli DH10BAC and the protein was expressed using the BAC-to-BAC Baculovirus Expression System (Gibco-BRL) according to the manufacturers directions, excepting that harvesting of recombinant viruses was done 5 days post-transfection. The supernatant from the transfection mixture was used for generating virus stock, which in turn was used for infecting Sf9 cells for use in the assay. DGAT activity was measured as the production of UC triacylglycerol from [l-14C]oleoyl-CoA and unlabeled dioleoyl-DAG. The reaction mixture (0.1 mL) consisted of isolated membranes, 3.5 uM [l-14C]oleoyl-CoA, 21.5 uM oleoyl-CoA and 200 uM 1,2-18:1 diacylglycerol in a buffer containing 25-30 mM Tricine (pH 7.8), 50-60 mM NaCl, and 0.06% CHAPS (w/v). Assay mixtures were incubated at 25°C for 5-10 minutes and reactions were terminated by adding 1.5 mL of heptane:isopropanol:0.5 M HaSC^ (10:40:1, v/v/v). Samples were processed as described in Example 1. DGAT activity was increased 6-fold in the cells transformed with plasmid pMON69839 relative to activity in untransformed (sf9) cells.<br>
For expression of the NcDGAT2 sequence in plants, the gene was PCRamplified from pMON69S34 in order to introduce NotI and Sse8387I cloning sites using primers ohgoDB#19911 (SEQ ID NO: 9) and oligoDB#19912 (SEQ ID NO: 10) (Figure 3). The PCR product was digested with NotI-SseS387I and the 107 Ibp fragment was ligated with the NotI-Sse8387I-digested vector from pMON67164 to form pMON68762. In this plasmid the gene is under control of a napin promoter. Plasmid pMON68762 was introduced into Agrobacterium tumefaciens ABI strain, which was used to transform soybean as described in Martinell etal, U.S. Patent 6,384,301.<br>
EXAMPLE 3<br>
Preparation and Transformation of Resynthesized DGAT2 Genes<br>
A codon usage table was constructed from 8 highly expressed seed specific proteins from soybean namely conglycinin (GenBank Accession # AB008678, AB008679, AB008680), glycinin (AB003680, AB004062), and globulin (D16107, U59425), and 14 highly expressed seed specific proteins from canola namely cuciferin, (GenBank Accession # 167133,167135, 17800; 17804,17810, 21117), and napin (AA349403,167176,167178, 167174, 167154, 17836,17834, 17832). TheMrDGAT2B and ScDGAT2 amino acid sequences (SEQ ID NO: 4 and SEQ ID NO: 6, respectively), along with the codon usage table described above, were sent to Blue Heron Biotechnology Inc., (Bothell, WA), who then utilized a proprietary algorithm to generate the final codon-optimized nucleotide sequence<br>
with the lowest free energy-of-forming RNA secondary structures. The codon-optimized sequence of MrDGAT2B was synthesized by Blue Heron Biotechnology, Inc., and named MrDGAT2B.nno (SEQ ID NO: 11). The codon-optimized sequence of ScDGAT2 was synthesized by Midland Certified Reagent Company (Midland, TX) and named ScDGAT2.nno (SEQ ID NO: 16).<br>
Plasmid pMON70924, containing MrDGAT2B.nno in an E. coli expression vector, was sequenced to confirm DNA as reported by Blue Heron Biotechnology. Plasmid DNA was digested with Xhol and filled to make a blunt end and then was digested with Sse8387I. The 1068bp fragment was ligated with the blunt/Sse8387I-digested vector pMON7091S to form pMON70925. In this plasmid the gene is under control of a napin promoter.<br>
Plasmid pMON70917, containing ScDGAT2.nno, was sequenced to confirm DNA as reported by Midland Certified Reagent Company. Plasmid DNA was digested with NotI-Sse8387I and the I269bp fragment was gel purified. The fragment was ligated to NotI-Sse8387I-digested pMON70918 to form pMON70920. In this plasmid the gene is under control of a napin promoter. ScDGAT2.nno was cloned into another expression vector, using similar techniques, so that the gene was expressed under control of the USP88 promoter (pMON70923). Plasmids pMON70925, pMON70923, and pMON70920 were introduced into Agrobacterium tumefaciens ABI strain, and each were used to transform soybean as described in Martinell et al., U.S. Patent 6,384,301.<br>
Similarly, the NcDGAT2 amino acid sequence (SEQ ID NO: 14) and the codon usage table described above were sent to Blue Heron Biotechnology, Inc., where a codon-optimized nucleotide sequence with the lowest free energy-of-forming RNA secondary structures was generated. The codon-optimized sequence of NcDGAT2 is synthesized by Blue Heron Technology and is named NcDGAT2.nno (SEQ ID NO: 12). The resynthesized NcDGAT2.nno is sequenced to confirm DNA as reported by Blue Heron Biotechnology. Plasmid DNA is digested with NotI-Sse8387I and the fragment is gel purified. The fragment is ligated to NotI-SseS3S7I digested pMON67164 to create a plasmid where the gene is under control of a napin promoter.<br>
Vectors are constructed that express a sequence set forth in SEQ ID NOs: 17,19,21, and 23, in the genome of a plant host to obtain transcription or transcription and translation of the sequence to effect phenotypic change. Transgenic soybean plants can be obtained by<br>
Agrobacterium-mediated transformation as described by Martinell et aL, U.S. Patent 6,384,301.<br>
EXAMPLE 4<br>
Expression of DGAT2 in Plants<br>
A resynthesized Mortierella ramanniana DGAT2A gene, MrDGAT2A.nno, (SEQ ED NO: 15) was expressed in soybean under control of soybean 7S promoter sequence (pCGN8832). Plants were transformed by particle bombardment and enzyme assays were performed on pooled, developing ri seed. Several plants exhibited significant increases (5-20 fold, Students t Test, alpha = 0.05) in DGAT activity relative to untransformed plants and are shown in Table 3.<br>
(Table Remove)<br>
DGAT activity in plants was assayed as follows. Developing embryos were ground in liquid nitrogen using a mortar and pestle. A portion of the sample was reconstituted with Tricine buffer (lOOmM Tricine, pH7.5,280 mM NaCl, 10% glycerol) and protein concentration was determined using Bradford reagent (Sambrook et al., Molecular Cloning, A Laboratory Manual, 2nd Ed., Cold Spring Harbor Press, Cold Spring Harbor, New York,<br>
1989). Samples were diluted to Img/ml and 10 p.1 were used in the assay, DGAT activity was measured as the production of UC triacylglycerol from [1- C]oleoyl-CoA and unlabeled dioleoyl-DAG. The reaction mixture (0.1 mL) consisted of protein homogenates, 3.5 uM [l-wC]oleoyl-CoA, 10 uJvl oleoyl-CoA and 1.5 mM 1,2-18:1 diacylglycerol in a buffer containing 25 mM Tricine (pH 7.8), 28 mM NaCl, and 0.06% CHAPS (w/v). Assay mixtures were incubated at 25°C for 10 minutes and reactions were terminated by adding 1.5 mL of heptane:isopropanol:0.5 M HaSO* (10:40:1, v/v/v). Samples were processed as described in Example 1.<br>
ri seed from plants expressing the MrDGAT2A.nno gene were advanced to the next generation (R2). Oil and protein levels were determined by Near-Infra-Red (NIR) analysis of    • mature R2 seed. NIR spectra of pooled seed samples harvested from individual plants are measured, and oil levels are calculated based on regression analysis using a standard curve generated from analysis of soybean seed with varying oil levels as determined gravimetrically following accelerated solvent extraction (Better Solutions for Food and Beverage Analysis, 2nd Edition, Dionex Corporation, Sunnyvale, California (1997)). A statistically significant      ; increase of 1.7% was observed between the oil mean of seeds homozygous for MrDGAT2A.nno compared to the oil mean of seeds that did not contain the transgene (nulls) (Students T test, alpha = 0.05). A statistical evaluation of the protein data showed there was no difference in the means (Students T test, alpha = 0.05).<br>
For expression of the resynthesized MrDGAT2A sequence in plants under the control of the napin promoter, the Notl-Sse 83871 fragment was ligated with the NotI-Sse8387I-digested binary vector pMON67164 to yield plasmid pMON70904. Plasmid pMON70904 was introduced into the Agrobacterium tumefadens ABI strain, which was then used to transform soybean. Developing Rl seed was harvested from the RO plant and assayed for DGAT activity. A selected number of events with elevated activity were advanced one generation (R2 seed). Oil levels and protein levels in mature second generation seed were determined by Near Infrared Transmittance (NIT) spectroscopy, whereby NIT spectra of pooled seed samples harvested from individual plants are measured, and oil and protein levels are calculated based on regression analysis using a standard curve generated from analysis of soybean seed with varying oil or protein levels, as determined gravimetrically following accelerated solvent extraction or elemental (%N) analysis, respectively. A statistically significant increase of 2.6% was observed between the oil mean of seeds homozygous for MrDGAT2A.nno compared to the oil mean of seeds that did not contain the transgene (nulls)<br>
(Students t Test, alpha = 0.05). A statistical evaluation of the protein data showed there was no difference in the means (Students t Test, alpha = 0.05).<br>
EXAMPLE 5<br>
Expression of DGAT2 from Multiple Promoters<br>
Two proteins exhibiting DGAT2 activity were identified in Monierella ramanniana (MrDGAT2A and MrDGAT2B). To construct a plasmid capable of expressing two DGAT genes, 2 genes were cloned into the same plasmid on a single t-DNA. Plasmid pMON70927 contained MrDGAT2A.nno (SEQ ID NO: 15) under control of the 7Sa' promotor and MrDGAT2B.nno (SEQ ID NO: 11) under control of the napin promoter. The cloning was as follows: pMON70900, containing MrDGAT2A.nno under control of the 7Sa' promoter, was digested with EcoRV and filled to make blunt ends.  The DNA was then cut with NotI and the 7Sa':MrDGAT2A.nno fragment was gel purified. The fragment was ligated to the blunt/Notl-digested plant expression vector pMON63689 to form pMON70912. To obtain MrDGAT2B.nno, pMON70924 was digested with Xhol and EcoRI and the ends were filled to create a blunt/blunt fragment that was 107Ibp long. The fragment was gel purified and then ligated to blunt/blunt pCGN7770 (an E. coli expression vector containing the napin promoter and 3' UTR) to form pMON70926. This plasmid containing MrDGAT2B.nno in the napin expression cassette was digested with NotI and the fragment was ligated to Notl-digested pMON70912, described above, to form pMON70927. pMON70927 was introduced into A. tumefaciens ABI strain, which was used to transform soybean as described in Martinell el al, U.S. Patent 6,384,301.<br>
Other DGAT2 genes, including, but not limited to, MrDGAT2A (SEQ ED NO: 1), MrDGAT2B (SEQ ID NO: 3), ScDAGT2 (SEQ ID NO: 5), NcDGAT2 (SEQ ID NO: 13), NcDGAT2.nno (SEQ ED NO: 12), and ScDGAT2.nno (SEQ ID NO: 16) are cloned in a similar manner, either in pans or in duplicate. The promoters that are used control the expression with respect to time and/or strength.<br>
EXAMPLE 6<br>
Expression of MrDGAT2A in Corn Germ<br>
An expression vector was prepared to engineer germ-targeted expression of the resynthesized Mortierella ramanniana DGAT2A (SEQ ID NO: 15) gene in com. Specifically, the full length MrDGAT2A.nno gene (SEQ ID NO: 15) contained in a 1076 base<br>
pair Notl/Ss=8387I fragment was c.oned into the Bspl20I/Sse8387I sites of pMON72021 directiy 3' of the Z« mays 13 oleosin promoter followed by the rice actin intron and 5' of the globulin 1 3' UTR to produce PMON68654 (Figure 6).<br>
The construct pMON68654 was transformed into the elite maize line LH59 by Agrobaaerium tumefaciens ABI-mediated transformation. Events resulting from ttas transformation demonstrate an increase in oil.<br><br><br>
claimed<br>
1.        An isolated nucleic acid molecule comprising a nucleic acid sequence that encodes a polypeptide that is at least 6bout 50% identical to one selected from the group consisting of  20, 22, and<br><br>
The isolated nucleic acid molecule of claim 1, wherein the polypeptide is<br>
selected from the group consisting o| SEQ ID NOs: 14, 18, 20, 22, and 24?<br>
An isolated nucleic acid molecule comprising a nucleic acid sequence that is at   <br>
least ah0uf75% identical to one selected from the group consisting OF SEQ ID NOs: 1 1, 12)<br>
37 6, 17, 19, 21, and 23<br>
The isolated nucleic acid molecule of claim 3, wherein the nucleic acid<br>
sequence is selected from the group consisting of/SEQ ID NOs: 11, 12, 13, 16, 17, 19, 21, and<br>
A DNA construct comprising an expression cassette comprising a heterologous<br>
promoter that functions in a plant cell operably linked to a nucleic acid molecule of claim 1 or V<br><br>
6.	The DNA construct/of)claim 5, further comprising a second expression cassette I<br>
wherein said second expression cassette comprises a second heterologous promoter that functions in a plant cell operably linked to a nucleic acid that encodes a polypeptide for a diacylglyderol acyltransferase.<br>
7.	A plant or seed comprising the DNA construct of claim 5.<br>
8.	 The plant or seed of claim 7, wherein the plant or seed is selected from the<br>
group consisting of maize, soybean, canola, oil seed rape, cotton, sesame, flax, peanut,<br>
sunflower, saf flower, olive, and oil palm.<br>
The plant or seed of claim 7, wherein the plant or seed is processed.<br>
The plant or seed of claim 9, wherein the plant or seed is used to produce a<br>
product selected from the group consisting of feed, meal, oil, and protein.<br>
A plant or seed comprising the DNA construct of claim 6.<br>
The plant or seed of claim 11, wherein the plant or seed is selected from the<br>
group consisting of maize, soybean, canola, oil seed rape, cotton, sesame, flax, peanut,<br>
I sunflower, safflower, olive, and oil palm.<br>
The plant or seed of claim 11, wherein the plant or seed is processed.<br>
The plant or seed of claim 13, wherein the plant or seed is used to produce a<br>
product selected from the group consisting of feed, meal, oil, and protein.<br>
A method of producing a plant having comprising the steps of: (A)<br>
transforming a plant cell with a DNA construct of claim 5 or claim 6; and, (B) regenerating<br>
said plant cell into a fertile plant, wherein the fertile plant has enhanced oil relative to seed<br>
from a plant having a similar genetic background but lacking the DNA construct.<br>
The method of claim 15, wherein said plant provides seed having an increased<br>
oil yield relative to seed from a plant having a similar genetic background but lacking the<br>
DNA construct.<br>
An isolated polynucleotide that encodes a polypeptide having at least one<br>
amino acid motif selected from the group consisting of: AYVFGYEPHSVXPI (SEQ ID: 33) and FXXPXYR (SEQ ED NO: 34).<br>
The isolated polynucleotide of claim 17, wherein the polypeptide has<br>
diacylglycerol acyltransferase activity.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMwLWRlbG5wLTIwMDUtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">130-delnp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMwLURFTE5QLTIwMDUtQXNzaWdubWVudC0oMDMtMDMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">130-DELNP-2005-Assignment-(03-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMwLURFTE5QLTIwMDUtQ2xhaW1zLSgwMy0wMy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">130-DELNP-2005-Claims-(03-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMwLWRlbG5wLTIwMDUtY2xhaW1zLSgxMS0wMy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">130-delnp-2005-claims-(11-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMwLWRlbG5wLTIwMDUtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">130-delnp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMwLURFTE5QLTIwMDUtQ29ycmVzcG9uZGVuY2UtT3RoZXJzLSgwMy0wMy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">130-DELNP-2005-Correspondence-Others-(03-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMwLURFTE5QLTIwMDUtQ29ycmVzcG9uZGVuY2UtT3RoZXJzLSgyMC0wNy0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">130-DELNP-2005-Correspondence-Others-(20-07-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMwLWRlbG5wLTIwMDUtY29ycmVzcG9uZGVuY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">130-delnp-2005-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMwLWRlbG5wLTIwMDUtZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">130-delnp-2005-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMwLURFTE5QLTIwMDUtRHJhd2luZ3MtKDAzLTAzLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">130-DELNP-2005-Drawings-(03-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMwLWRlbG5wLTIwMDUtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">130-delnp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMwLWRlbG5wLTIwMDUtZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">130-delnp-2005-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMwLWRlbG5wLTIwMDUtZm9ybS0xOC5wZGY=" target="_blank" style="word-wrap:break-word;">130-delnp-2005-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMwLURFTE5QLTIwMDUtRm9ybS0yLSgwMy0wMy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">130-DELNP-2005-Form-2-(03-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMwLWRlbG5wLTIwMDUtZm9ybS0yLnBkZg==" target="_blank" style="word-wrap:break-word;">130-delnp-2005-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMwLWRlbG5wLTIwMDUtZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">130-delnp-2005-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMwLWRlbG5wLTIwMDUtZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">130-delnp-2005-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMwLURFTE5QLTIwMDUtR1BBLSgwMy0wMy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">130-DELNP-2005-GPA-(03-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMwLWRlbG5wLTIwMDUtZ3BhLnBkZg==" target="_blank" style="word-wrap:break-word;">130-delnp-2005-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMwLURFTE5QLTIwMDUtT3RoZXJzLSgwMy0wMy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">130-DELNP-2005-Others-(03-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMwLURFTE5QLTIwMDUtT3RoZXJzLSgyMC0wNy0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">130-DELNP-2005-Others-(20-07-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMwLWRlbG5wLTIwMDUtcGN0LTMwNC5wZGY=" target="_blank" style="word-wrap:break-word;">130-delnp-2005-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMwLURFTE5QLTIwMDUtUENULTQwOS0oMDMtMDMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">130-DELNP-2005-PCT-409-(03-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMwLWRlbG5wLTIwMDUtcGN0LXNlYXJjaCByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">130-delnp-2005-pct-search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMwLURFTE5QLTIwMDUtUGV0aXRpb24tMTM3LSgwMy0wMy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">130-DELNP-2005-Petition-137-(03-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMwLURFTE5QLTIwMDUtUGV0aXRpb24tMTM4LSgwMy0wMy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">130-DELNP-2005-Petition-138-(03-03-2008).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="217162-a-system-for-enabling-access-to-travel-services.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="217164-a-method-for-securing-software-updates-of-a-plurality-of-pay-television-decoders.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>217163</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>130/DELNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>31-Mar-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>25-Mar-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>13-Jan-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MONSANTO TECHNOLOGY LLC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>MAILZONE E2NA, 800 N.LINDBERGH BOULEVARD, ST. LOUIS, MISSOURI 63167, USA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>LARDIZABAL KATHRYN D</td>
											<td>1546 OWENS VALLEY DRIVE, WOODLAND, CALIFORNIA 95776, USA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>WAGNER NICHOLAS W</td>
											<td>60 AINGER CIRCLE, SACRAMENTO, CALIFORNIA 95835, USA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>BENNETT KRISTEN A</td>
											<td>4130 HACKBERRY PLACE, DAVIS, CALIFORNIA 95616, USA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2003/024822</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-07-31</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/399,427</td>
									<td>2002-07-31</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>10/631,581</td>
									<td>2003-07-31</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/217163-diacylglycerol-acyltransferase-nucleic-acid-sequences-and-associated-products by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:27:03 GMT -->
</html>
