<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/217283-a-method-of-determining-the-proteome-pattern-of-urine-sample-and-a-dianostic-kit-for-diagnosis-of-renal-disorders by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:35:02 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 217283:A METHOD OF DETERMINING THE PROTEOME PATTERN OF URINE SAMPLE AND A DIANOSTIC KIT FOR DIAGNOSIS OF RENAL DISORDERS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD OF DETERMINING THE PROTEOME PATTERN OF URINE SAMPLE AND A DIANOSTIC KIT FOR DIAGNOSIS OF RENAL DISORDERS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention relates to a novel method of determining the proteome pattern of urine sample and a diagnostic kit for diagnosis of renal disorders. The method involving separation of proteins using two- dimensional gel electrophoresis and identification of individual proteins thus separated, by MALDI-TOF mass spectrometry, is a well established and widely used technology, allowing &#x27;proteomic&#x27; analysis of biological systems such as cells, organs and biological fluids. This proteomics analysis of protein mixtures provides a single, yet nearly complete and unique &#x27;map&#x27; of the protein content of the mixture thereby also enabling quantitative evaluation of individual protein levels. this feature of the proteomics approach has been applied it to the analysis of protein content of the human urine, with suitable modification.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Prior art<br>
Human kidneys perform the important function of maintaining the internal milieu and composition of the blood. While glomerular filtration in the kidneys removes ions and small molecular weight proteins out of the blood stream, the process of tubular re-absorption prevents excess loss of ions, sugars and certain low molecular weight proteins. A careful balance of these two counteracting processes characterizes normal functioning of the kidney (1).<br>
As a result of the efficient filtration and re-absorption process in the nephron, urine of a normal person shows very small amount of protein. Large molecular weight proteins are effectively retained in the blood stream. Relatively low molecular weight proteins that do escape through the basement membrane are subsequently reabsorbed by the epithelial cells lining the proximal tubules by receptor-mediated endocytosis. Any damage to the ECM around the glomeruli results in the release of proteins into the tubules resulting in the appearance of serum proteins in the urine. Likewise, any perturbation of the epithelial cells of the proximal tubules can cause inefficient re-absorption of urinary proteins leading to the loss of proteins in the urine. These conditions are clinically referred to as proteinuria and are well-accepted indicators of glomerular or tubular damage to the kidneys (1,2).<br><br>
The presence of detectable quantities of proteins in the urine of individuals is now widely accepted as an indicator of renal damage with the amount of protein being proportional to the extent of damage. The routine urine analysis performed in medical laboratories involves quantitative measurement of protein in the urine by the dipstick method. Depending on the amount of protein found in the urine the disease is graded and classified into sub-categories. A relatively more advanced test that is often employed is the examination of microalbuminuria. This is a radioimmunoassay-based quantitation of albumin in the urine that routine tests are unable to detect. While these methods of estimating extent of proteinuria can identify individuals with increased risk of renal failure, they fail to provide any information about the site of damage in the kidneys.<br>
A large number of the proteins found in the urine have been identified (Table-1). Qualitative differences in the kinds of proteins found in the urine has been correlated with the type of renal damage. For instance the presence of predominantly low molecular weight proteins such as Retinol-binding protein, alpha-1-microglobulin, beta-2-microglobulin, alpha-1-macroglobulin, and N-acetyl glucosaminidase are believed to indicate damage to renal tubules resulting in defective reabsorption of these  proteins  from  tubular  lumen.   Similarly  presence  of higher<br><br>
molecular weight proteins such as albumin, immunoglobulin, transferrin, etc. is believed to indicate glomerular damage leading to defective filtration. There are several reports in the literature describing the detection of specific proteins in the urine and linking it with specific disorders. Such studies have mostly examined the levels of a chosen set of one or more candidate proteins in normal versus clinical conditions. The approach used in the detection of specific proteins in the urine has mostly been radioimmunoassays and ELISAs using specific antibodies to the protein under investigation (5, 6, 7, 8). While these methods are highly sensitive and quantitative they however allow examination of one protein a time only and are expensive and time consuming. Table-I<br><br><br>
8	alpha-1 -microglobulin	33	5.3	Tubular damage	6, 8,17<br>
9	Retinol-binding protein	21	5.4	Tubular damage	9,17<br>
10	beta-2-microglobulin	11.7	6.07	Tubular damage	16,17<br>
Introduction to Proteomics;<br>
Proteomics refers to the study of the total protein content of a biological mixture. The most common interpretation of this term however entails a study of all proteins expressed by the genome of cells under study. A proteomic approach towards studying biological systems has become an essential part of most genome wide investigations since it overcomes the various shortcomings of the genomics approach. For instance information on post-translational modifications of proteins, discrepancies between mRNA levels and corresponding protein levels are some of the issues that genomic analyses cannot address and proteomics can.<br>
A proteomics analysis basically involves separation of proteins from a mixture followed by identification of these proteins. Two-dimensional gel electrophoresis is the preferred method of separation of proteins since its combined use of isoelectric point as well as molecular weight as basis of separation leads to efficient separation of proteins with each spot on a 2D gel mostly representing a single protein species. In-gel proteolytic digestion of protein spots from a 2D gel and mass spectrometric analysis<br><br>
of peptides thus generated is a popular method of protein identification not in the least because it is very sensitive and can be used with pico-femto molar concentrations of proteins. Subsequent Tandem mass spectrometry of specific peptides generates amino acid sequence information leading to unambiguous identification of a protein. Currently automated systems that integrate both the above methods-1) separation and 2) identification, have made high throughput analysis of complex biological mixtures possible, reducing the time and effort required for large scale proteomic analyses. Proteomic analysis can be used to monitor changes in levels of proteins as well as any modifications on individual proteins that result in a change in its molecular weight or pi. Reference is made to the study by Song EJ and Lee KJ published in "Experimental and Molecular medicine, 2001 Apr 21m 33; 5-18" entitled "Identification of proteomes using 2-dimentional gel electrophorosis and Maldi-dof which focuses on the strategies of proteome analysis using sample preparation, 2-dimentional gel electrophoresis, processing of protein spots and identification of proteins, protein-protein interaction and posttranslational modification using MALDI-TOF-MS. Summary of inventions;<br>
1)  Concentrating urinarv proteins by a precipitation step: One of the problems with analyzing urinary proteins  is the  low concentrations of individual proteins, making it difficult to detect them by<br><br>
conventional methods of staining gels such as coomassie blue dye-binding or silver staining. Our method employs a precipitation step to concentrate the protein mixture. Addition of Trichloro Acetic Acid (TCA) or acetone to the urinary sample leads to precipitation of proteins, which can be collected by centrifugation. The concentrated protein pellet can be diluted in desired volume of buffer. This step therefore allows us to analyse equal amounts of protein from urine samples of differing protein concentrations. This becomes especially useful in patients with very low levels of proteinuria such as microalbuminuria. In addition, our method of precipitation of proteins excludes the possibility of changes in soluble urinary protein composition as a result of freezing of the sample as has been reported (19).<br>
2) Use of two dimensional gel electrophoresis:<br>
A two-dimensional gel electrophoresis analysis of urinary proteins can provide in a single step a protein map of the urine. Both quantitative and qualitative differences in the urinary protein profile of patients can be visualized in this "urinary protein map". Identification of the proteins in this map using MALDI-TOF mass spectrometry can allow construction of a database of protein profiles in specific renal disorders. Such a database can then be used for rapid classification of the renal histopathology.<br><br>
In order to demonstrate the efficacy of this method we have applied this technique to examining the urinary proteome of patients with clinically defined proteinuria. We chose three groups of patients- 1) patients with renal failure, 2) Nephrotic syndrome and 3) Diabetics with microalbuminuria. We generated the urinary protein maps of these patients and upon examination we found distinct pattern of proteins spots in each case. More importantly we found that the protein profile was reflective upon gross examination of the kind of renal damage these patients were know to have. For instance patients with renal failure had mostly lower molecular proteins indicative of a predominantly tubular damage while nephritic syndrome patients displayed mostly higher molecular weight proteins indicative of a predominantly glomerular damage. We also identified two proteins found in kidney failure patients, four from nephrotic syndome patients and three from microalbuminuria patients using in-gel protease digestions and MALDI-TOF technology. These were previously reported urinary proteins albumin, alpha-1-antitrypsin in renal failure patients and alpha-1-antitrypsin, Zn-alpha2-glycoprotein, alpha-1 microglobulin, alpha-1-acid glycoprotein 2 in nephrotic syndrome patients.<br><br>
Among the limited number of proteins seen in microalbuminuria patients Zn-alpha- 2-glycoprotein, alpha-1-microglobulin and alpha-1-acid glycoprotein 2 were common with the nephrotic syndrome patients.<br>
Detailed description of mvention:<br>
Two-dimensional Gel Electrophoresis:<br>
5ml of human urine was collected and stored at 4°C. Proteins from 0.5 ml of urine was precipitated by adding equal volume of 20% TCA or two volumes of ice cold acetone and incubating for 30 minutes at 4*"C. The protein precipitate was pelleted by centrifuging at 5000g for lOmin. The pellet was dried, resuspended in 0,1ml of PBS and protein concentration determined using Bradford"s method (18). Ten micrograms of protein from each urine sample was subjected to 2-DE as described by O"Farrel (10) using ampholines of pI range 3.5-9.5 (Amersham-Pharmacia, Uppsala, Sweden). Following 2-DE the proteins were detected by coomassie blue staining. The gels were scanned on a laser scanner and images were analysed using Melanie 2-D gel analysis software. The various protein spots were quantitated using Melanie and protein amounts represented as arbitrary units.<br><br>
Protein identification by MALDI-TOF mass spectrometry: For in-gel digestion, large scale 2-DE was as described by Cells (17). Coomassie stained spots of interest were cut out from at least three 2-D gels, pooled and cut further into small pieces ~2-3 mm in diameter. A blank gel piece with no protein spots was also separately processed as a control. The gel pieces were vortexed in 25mM ammonium bicarbonate in 50% Acetonitrile 3-4 times. The gel pieces were then dried under vacuum, and incubated with lOmM DTT in 25Mm ammonium bicarbonate at 56°C for Ih. Reduced cysteines were modified with 55mM lodoacetamide in 25mM ammonium bicarbonate for 45 min at room temperature in the dark. The gel pieces were washed 3-4 times in 25mM NH4HC03 in 50% Acetonitrile and then completely dried. Trypsin (0.1 mg/ml) or Endoproteinase Glu-C (V8 protease) (0.1 mg/ml) was added just enough to cover the gels and digestion carried out for 12h at 37°C. The peptides were extracted from the gels with first 0.1ml water and followed by two extractions with 0.5% TFA in 50% Acetonitrile. The peptide solution was then concentrated in a speed-vac and used for MALDI-TOF analysis (Kratos Analytical, SHIMADZU corp. Maryland, USA). The matrix used for MALDI-TOF was alpha-cyano cinnamic acid. The peptide masses obtained were used to identify the protein using at least 3 search programs, MS-FIT, Peptident and Propound. Protein identification was based on best matches using molecular weight, pl and<br><br>
peptide masses as parameters. Peptide peaks arising from the matrix or from auto-proteolytic digestion of proteases were determined from the control and discounted from the search.<br>
Clinical subjects:<br>
A total of twenty-five individuals diagnosed with various renal disorders were identified. They were categorized into three broad groups- Renal failure, Nephrotic syndrome and Microalbuminuria. Estimation of serum creatinine Glomerular filteration rate, proteinuria (0.1-0.4 mg/ml), with renal failure were the criteria for identifying renal failure patients from department of Pediatrics, St. John"s Medical College Hospital, Bangalore. Nephrotic syndrome patients with high proteinuria (0.5 - 4.0mg/ml), but normal serum creatinine were from Manipal Institute of Nephrology and Urology, Manipal Hospital, Bangalore. Patients with type 2 diabetes from department of Endocrinology, M.S. Ramaiah Medical College Hospital, Bangalore, who tested positive for microalbuminuria (20-100 ug/min/L) were investigated in the third category,. Overnight urine samples were collected from the patients and used for the investigation.<br><br>
Results:<br>
1. Renal failure patients show the presence of albumin and other low<br>
molecular weight urinary proteins.<br>
To study the pattern of proteins excreted under different clinical conditions we first chose to examine 2-DE profiles of urinary proteins fi-om kidney failure patients. About 5 ml of urine was collected from 10 different patients on dialysis. The urine samples were acetone precipitated and aliquots corresponding to lO^g protein were analyzed by 2-DE and coomassie blue staining as described under methods. The 2-DE profile of proteins seen in all the kidney failure patients was quite similar. A large spot corresponding to albumin could be identified based on its known pl/molecular weight and also by MALDI-TOF analysis of its tryptic peptides. In addition 5 other spots of molecular weights either equal or smaller than albumin could also be seen. These were numbered as RFl to RP5. The 2-DE patterns of urinary proteins fi"om all the renal failure patients were qualitatively similar but the relative quantities of individual spots were different in different patients.<br>
To identify the low molecular weight protein spots on the 2-D gels, we subjected different spots to in-gel digestions with trypsin or V8 protease and analyzed the peptides by MALDI-TOF as described under the methods. The identity of predicted albumin spot was confirmed by this<br><br>
approach and in addition we could identify spot RF2 as alpha-1-antitrypsin (Figure lA and IB). Both are previously reported urinary proteins (see table 1). The identity of spot 3, 4 and 5 is under investigation. The analysis indicated that urinary proteins from renal failure patients are characterized by the presence of relatively low molecular weight proteins commonly used as markers of tubular injury. Figure-1<br><br><br>
2.    Nephrotic syndrome patients show relatively high molecular weight urinary proteins.<br>
Nephrotic syndrome is another example of renal abnormality characterized by a large amount of urinary protein excretion. Using an approach similar to that in Figure 1 we analyzed the 2-DE profiles of urinary proteins from 10 different nephrotic syndrome patients. All the patients showed a fairly similar profile of protein spots. Apart from the common presence of albumin the pattern was quite distinct from that seen in renal failure patients. Unlike renal failure the urinary proteins of nephrotic syndrome patients mostly showed the presence of larger molecular weight proteins clustered in the acidic part of the 2-D gels. Eight distinct spots, in addition to the spot corresponding to albumin, could be easily identified. These were numbered as spots NSl to NS8. Despite the profiles being similar there were quantitative differences in the spots among various patients. Some spots such as NS2, NS3, NS6 and NS7 were absent in patient 1 while NS8 was undetectable in patient 5. It is likely that the levels of these spots were too low to be detected by our staining procedure. A subset of nephrotic syndrome patients also showed<br><br>
some low molecular weight protein spots that were common with the renal failure samples.<br>
Selected spots from these gels were subjected to in-gel tryptic digestion and MALDI-TOF analysis. Based on MALDI-TOF analysis and comparison with the pI/Molecular weights of known urinary proteins we could identify spot NS4 to be alpha-1-antitrypsin (Figure 2A), NS5 to be Zn-alpha-2-glycoprotein (Figure 2B) and NS8 as alpha-1-acid glycoprotein 2 (Figure 2C).<br><br><br>
3. Microalbuminuria patients show urinary protein spots in addition to serum albumin.<br>
Diabetic patients are predisposed to renal damage as a result of accumulation of advanced glycation end products in the blood. These patients are advised to undergo microalbuminuria test for timely detection of traces of albumin in the urine suggesting renal damage. We examined if the microalbuminuria positive patients would show presence of urinary proteins other than albumin. We analyzed urinary proteins from ten different patients tested positive for microalbuminuria by 2-DE. Although the amount of albumin in these samples was very low, the total protein was relatively higher indicating that proteins other than albumin were present (not shown). In addition to the presence albumin, at least three other protein spots were obvious in four different patients. In some patients the spots MAI and MA2 are nearly as distinct as the albumin spot.<br>
As above we subjected spots MAI, MA2 and MA3 to in-gel tryptic digestion followed by MALDI-TOF analysis. The analysis of spectra (see Figure 3A, B, C) and the total urinary protein database indicated that spot MAI was Zn-alpha-2 glycoprotein, spot MA2 was alpha-1-microglobulin and MA3 was alpha-1-acid glycoprotein 2. These spots were same as spots NS4, NS5 and NS8 seen in nephrotic syndrome patients.<br><br><br>
In summary, we have used a proteomic approach to examine the kinds of urinary proteins excreted under different clinical conditions of proteinuria. Three group of patients namely a) renal failure b) nephrotic syndrome and c) microalbuminuria were randomly selected for the study. The 2D urinary protein profiles observed in these different groups of patients are schematically represented and summarized in figure-4. The spots are colour coded. Red shows profile for renal failure patients, green shows profile for nephrotic syndrome patients and clear spots denote profile  seen  in microalbuminuria patients.  Overlapping  spots  with<br><br>
different colour-codes indicate identical protein found in two different conditions. Proteins identified by MALDI-TOF analysis are labeled.<br><br>
can support existing tools available to study proteinuria in more accurately defining the stage of renal injury. Distinct protein patterns observed under different renal conditions emphasize the need to develop specific urinary protein database for specific renal disorders. The approach may make it possible to use protein markers to define and categorize renal pathology. In microalbuminuria patients, this method provides great potential in identification of novel urinary markers for renal damage other than albumin.<br><br>
References:<br>
1.	Guyton., Hall (1996). Textbook of medical physiology. 9th Ed, pl53-421.<br>
2.	Mogensen CE. (1994). J. Diabetic. Compl. 8, 135-36.<br><br>
3.	Mogensen CE (1995). The Lancet. 346, 1080-1084.<br>
4.	Airoldi G and Campanini M. (1993). Clin Nephrol 84, 210-24.<br>
5.	Tolkoff-Rubin NE, Rubin NH, Bonventre JV (1988;. Clin Lab. Med. 8, 507-526.0<br>
6.	Guderwg and Hoffman (1992;. Clin Nephrol. 38, S3-7.<br>
7.	Price RG (1992). Clin Nephrol 38, S14-29.<br>
8.	Weber MH and Verwiebe R (1992). Eur J Clin Chem Clin Biochem. 30.<br>
9.	Mastroianni Kirsztain G. (2000). Nephron. 86,109-114.<br>
10.	OTarrel, P. M., (1975). J Biol Chem. 250,4807-4821.<br>
11.	Cells, J. E., Wolf, H., Ostergaard, M., Electrophoresis 2000, 21, 2115-2121.<br>
12.	Rasmussen, H. H., Omtoft, T. F., Wolf, H., Cells, J. E.,(l996). J. Urol, 155, 2113-2119<br>
13.	Spahr, S. S., Davis, M. T., McGinley, M. D., Robinson, J. H., Bures, E. J., Belerle, J., Mort, J., Courchesne, P. L., Chen, K., Wahl, R. C, Yu, W., Luethy, R. and Patterson, S. D. (2000). Proteomics 2001,1, 93-107.<br><br>
14.	Cells, J. E., (1998). Cell Biology: A laboratory handbook. Second edition, Vol4. P375-385.<br>
15.	Price, R. G, (1992). Clin Nephrology. 38, suppl-1, sl4-19.<br>
16.	Steinhoff, J., Feddersen, A., Wood, W. G., Hoyer J. and Sack K 991). Clin Nephrol. 35 (6), 255-62.<br>
17.	Tomlinson, P. A., Dalton, R. Neurospora, Hartley, B., Haycock, G. B. and Chantler C (1997). Pediatr Nephrol 11 (3), 285-90.<br>
18.	Bradford, M. M., (1976). Anal. Biochem. 72, 248-254.<br>
19.	Schultz, C. J., Dalton, R. N., Turner, C, Neil, H. A. W., and Dunger, D. B. (2000). Diabetic Mdecine. 17. 7-14.<br><br><br>
I Claim;<br>
1.	A method of determining the proteome pattern of urine sample<br>
comprising of the following steps:<br>
a.	Precipitation and concentration of the proteins from urine sample;<br>
b.	Two dimensional electrophoresis to generate a "protein map" of<br>
the urine sample;<br>
c.	In-gel proteolytic digestion of protein spots of interest to generate<br>
peptide fragments;<br>
d.	Identification of accurate masses of peptide fragments by MALDI-<br>
TOF mass spectrometry; and<br>
e.	Determination of proteome pattern of the urine sample.<br>
2.	A method as claimed in claim 1 wherein the proteins from urine sample are precipitated with trichloroacetic acid or acetone and concentrated to desired concentration.<br>
3.	A method as claimed in claim 2 wherein the protein sample is analysed by two dimensional electrophoresis comprising of isoelectric focusing using ampholines of pi range 3.5-9.5 and quantitation of coomassie blue stained protein spots using suitable software such as Melanie 2D gel analysis software.<br>
4.	A method of diagnosing renal disorders as claimed in claim 2 wherein the protein spots generated by 2D gel elecfrophoresis are subjected to<br><br>
in-gel digestion using proteases such as trypsin, endoproteinase Glu C (V8 protease) at optimal conditions.<br>
5.	A method as claimed in claim 2 wherein the peptide fragments<br>
obtained from in-gel digestion are analysed by MALDI-TOF mass<br>
spectrometry to generate an identity of the protein map.<br>
6.	A diagnostic kit for diagnosis of renal disorders comprising of:<br>
a.	Solution of trichloro acetic acid or acetone.<br>
b.	Proteases such as trypsin, endoproteinase Glu C (V8 protease).<br>
c.	Phosphate buffered saline pH 7.4.<br>
d.	25 mM ammonium bicarbonate containing 10mM Dithiothreitol<br>
(DTT).<br>
e.	2D gels prepared using ampholines of pI range 3.5-9.5.<br>
f.	Known "proteome pattern" generated from renal disorder samples<br>
for comparison.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDY3Ny1tYXMtMjAwMSBhYnN0cmFjdC1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">0677-mas-2001 abstract-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDY3Ny1tYXMtMjAwMSBhYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">0677-mas-2001 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDY3Ny1tYXMtMjAwMSBjbGFpbXMtZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">0677-mas-2001 claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDY3Ny1tYXMtMjAwMSBjb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">0677-mas-2001 correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDY3Ny1tYXMtMjAwMSBjb3JyZXNwb25kZW5jZS1wby5wZGY=" target="_blank" style="word-wrap:break-word;">0677-mas-2001 correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDY3Ny1tYXMtMjAwMSBkZXNjcmlwdGlvbiAoY29tcGxldGUpLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">0677-mas-2001 description (complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDY3Ny1tYXMtMjAwMSBkZXNjcmlwdGlvbiAocHJvdmlzaW9uYWwpLnBkZg==" target="_blank" style="word-wrap:break-word;">0677-mas-2001 description (provisional).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDY3Ny1tYXMtMjAwMSBmb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">0677-mas-2001 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDY3Ny1tYXMtMjAwMSBmb3JtLTE5LnBkZg==" target="_blank" style="word-wrap:break-word;">0677-mas-2001 form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDY3Ny1tYXMtMjAwMSBmb3JtLTI2LnBkZg==" target="_blank" style="word-wrap:break-word;">0677-mas-2001 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDY3Ny1tYXMtMjAwMSBmb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">0677-mas-2001 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDY3Ny1tYXMtMjAwMSBmb3JtLTQucGRm" target="_blank" style="word-wrap:break-word;">0677-mas-2001 form-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDY3Ny1tYXMtMjAwMSBmb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">0677-mas-2001 form-5.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="217282-a-chiral-copper-complex-catalyst-composition.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="217284-method-for-limiting-the-number-of-simultaneous-call-forwarding-attempts-in-a-cellular-communication-system.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>217283</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>677/MAS/2001</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>21/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>23-May-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>26-Mar-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>17-Aug-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>UTPAL TATU</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>21, RAMANUJAN APT, IISC CAMPUS, BANGALORE - 560 012,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>UTPAL TATU</td>
											<td>21, RAMANUJAN APT, IISC CAMPUS, BANGALORE - 560 012,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>G01N 33/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/217283-a-method-of-determining-the-proteome-pattern-of-urine-sample-and-a-dianostic-kit-for-diagnosis-of-renal-disorders by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:35:03 GMT -->
</html>
