<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/217407-a-method-of-expressing-a-lim-mineralization-protein-in-a-non-osseous-mammalian-cell by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:43:16 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 217407:A METHOD OF EXPRESSING A LIM MINERALIZATION PROTEIN IN A NON-OSSEOUS MAMMALIAN CELL&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD OF EXPRESSING A LIM MINERALIZATION PROTEIN IN A NON-OSSEOUS MAMMALIAN CELL&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Methods of expressing LIM mineralization protein in non-osseous mammalian cells, such as stem cells or intervertebral disc cells (e.g., cells of the annulus fibrosus, or cells of the nucleus pulposus) are described. The methods involve transfecting the cells with an isolated nucleic acid comprising a nucleotide sequence encoding a LIM mineralization protein operably linked to a promoter. Transfection may be accomplished ex vivo or in vivo by direct injection of virus or naked DNA, or by a nonviral vector such as a plasmid. Expression of the LIM mineralization protein can stimulate proteoglycan and/or collagen production in cells capable of producing proteoglycyan and/or collagen. Methods for treating disc disease associated with trauma or disc degeneration are also described. (FIG.)1</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>TITLE OF THE INVENTION<br>
A METHOD OF EXPRESSING A LIM MINERALIZATION<br>
PROTEIN IN A NON-OSSEOUS MAMMALIAN CELL<br>
This application claims priority from U.S. Provisional Application Serial<br>
No. 60/331,321 filed November 14, 2001. The entirety of that provisional<br>
application is incorporated herein by reference.<br>
This application is related to U.S. Patent Application Serial<br>
No. 09/124,238, filed My 29, 1988, now U.S. Patent No. 6,300,127, and U.S.<br>
Patent Application Serial No. 09/959,578, filed April 28, 2000, pending. Each of<br>
these applications is incorporated by reference herein in its entirety.<br>
BACKGROUND OF THE INVENTION<br>
Field of the Invention<br>
The field of the invention relates generally to methods for expressing LIM<br>
mineralization proteins in non-osseous cells such as intervertebral disc cells or<br>
cells of the nucleus pulposus. More specifically, the field of the invention relates<br>
to transfecting non-osseous cells such as intervertebral disc cells with a nucleic<br>
acid encoding a LIM mineralization protein.<br>
Background of the Technology<br>
Osteoblasts are thought to differentiate from pluripotent mesenchymal stem<br>
cells. The maturation of an osteoblast results in the secretion of an extracellular<br>
matrix which can mineralize and form bone. The regulation of this complex<br>
process is not well understood but is thought to involve a group of signaling<br>
glycoproteins known as bone morphogenetic proteins (BMPs). These proteins<br>
have been shown to be involved with embryonic dorsal-ventral patterning, limb<br>
bud development, and fracture repair in adult animals. B. L. Hopan. Genes &amp;<br>
Develop., 10, 1580 (1996). This group of transforming growth factor-beta<br>
superfamily secreted proteins has a spectrum of activities in a variety of cell types<br><br>
at different stages of differentiation; differences in physiological activity between<br>
these closely related molecules have not been clarified. D. M. Kingsley, Trends<br>
Genet, 10, 16(1994).<br>
To better discern the unique physiological role of different BMP signaling<br>
proteins, we recently compared the potency of BMP-6 with that of BMP-2 and<br>
BMP-4, for inducing rat calvarial osteoblast differentiation. Boden. et al..<br>
Endocrinology, 137, 3401 (1996). We studied this process in first passage<br>
(secondary) cultures of fetal rat calvaria that require BMP or glucocorticoid for<br>
initiation of differentiation. In this model of membranous bone formation,<br>
glucocorticoid (GC) or a BMP will initiate differentiation to mineralized bone<br>
nodules capable of secreting osteocalcin, the osteoblast-specific protein. This<br>
secondary culture system is distinct from primary rat osteoblast cultures which<br>
undergo spontaneous differentiation. In this secondary system, glucocorticoid<br>
resulted in a ten-fold induction of BMP-6 mRNA and protein expression which<br>
was responsible for the enhancement of osteoblast differentiation. Boden. et al..<br>
Endocrinology, 138, 2920 (1997).<br>
In addition to extracellular signals, such as the BMPs, intracellular signals<br>
or regulatory molecules may also play a role in the cascade of events leading to<br>
formation of new bone. One broad class of intracellular regulatory molecules are<br>
the LIM proteins, which are so named because they possess a characteristic<br>
structural motif known as the LIM domain. The LIM domain is a cysteine-rich<br>
structural motif composed of two special zinc fingers that are joined by a 2-amino<br>
acid spacer. Some proteins have only LIM domains, while others contain a variety<br>
of additional functional domains. LIM proteins form a diverse group, which<br>
includes transcription factors and cytoskeletal proteins. The primary role of LIM<br>
domains appears to be in mediating protein-protein interactions, through the<br>
formation of dimers with identical or different LIM domains, or by binding distinct<br>
proteins.<br>
In LIM homeodomain proteins, that is, proteins having both LIM domains<br>
and a homeodomain sequence, the LIM domains function as negative regulatory<br>
elements. LIM homeodomain proteins are involved in the control of cell lineage<br>
determination and the regulation of differentiation, although LIM-only proteins<br>
may have similar roles. LIM-only proteins are also implicated in the control of cell<br>
proliferation since several genes encoding such proteins are associated with<br>
oncogenic chromosome translocations.<br>
Humans and other mammalian species are prone to diseases or injuries that<br>
require the processes of bone repair and/or regeneration. For example, treatment of<br>
fractures would be improved by new treatment regimens that could stimulate the<br>
natural bone repair mechanisms, thereby reducing the time required for the<br>
fractured bone to heal. In another example, individuals afflicted with systemic<br>
bone disorders, such as osteoporosis, would benefit from treatment regimens that<br>
would results in systemic formation of new bone. Such treatment regimens would<br>
reduce the incidence of fractures arising from the loss of bone mass that is a<br>
characteristic of this disease.<br>
For at least these reasons, extracellular factors, such as the BMPs, have<br>
been investigated for the purpose of using them to stimulate formation of new bone<br>
in vivo. Despite the early successes achieved with BMPs and other extracellular<br>
signalling molecules, their use entails a number of disadvantages. For example,<br>
relatively large doses of purified BMPs are required to enhance the production of<br>
new bone, thereby increasing the expense of such treatment methods.<br>
Furthermore, extracellular proteins are susceptible to degradation following their<br>
introduction into a host animal. In addition, because they are typically<br>
immunogenic, the possibility of stimulating an immune response to the<br>
administered proteins is ever present.<br>
Due to such concerns, it would be desirable to have available treatment<br>
regimens that use an intracellular signaling molecule to induce new bone<br>
formation. Advances in the field of gene therapy now make it possible to<br>
introduce into osteogenic precursor cells, that is, cells involved in bone formation,<br>
or peripheral blood leukocytes, nucleotide fragments encoding intracellular signals<br>
that form part of the bone formation process. Gene therapy for bone formation<br>
offers a number of potential advantages: (1) lower production costs; (2) greater<br>
efficacy, compared to extracellular treatment regiments, due to the ability to<br>
achieve prolonged expression of the intracellular signal; (3) it would by-pass the<br>
possibility that treatment with extracellular signals might be hampered due to the<br>
presence of limiting numbers of receptors for those signals; (4) it permits the<br>
delivery of transfected potential osteoprogenitor cells directly to the site where<br>
localized bone formation is required; and (5) it would permit systemic bone<br>
formation, thereby providing a treatment regimen for osteoporosis and other<br>
metabolic bone diseases.<br>
In addition to diseases of the bone, humans and other mammalian species<br>
are also subject to intervertebral disc degeneration, which is associated with,<br>
among other things, low back pain, disc herniations, and spinal stenosis. Disc<br>
degeneration is associated with a progressive loss of proteoglycan matrix. This<br>
may cause the disc to be more susceptible to bio-mechanical injury and<br>
degeneration. Accordingly, it would be desirable to have a method of stimulating<br>
proteoglycan and/or collagen synthesis by the appropriate cells, such as, for<br>
example, cells of the nucleous pulposus, cells of the annulus fibrosus, and cells of<br>
the intervertebral disc.<br>
SUMMARY OF THE INVENTION<br>
According to a first aspect of the invention, a method of expressing a LIM<br>
mineralization protein in a non-osseous mammalian cell is provided. According to<br>
this aspect of the invention, the method comprises transfecting the cell with an<br>
isolated nucleic acid comprising a nucleotide sequence encoding the LIM<br>
mineralization protein operably linked to a promoter. The cell can be a cell<br>
capable of producing proteoglycan and/or collagen such that the expression of the<br>
LIM mineralization protein stimulates proteoglycan and/or collagen synthesis in<br>
the cell. The isolated nucleic acid according to this aspect of the invention can be a<br>
nucleic acid which can hybridize under standard conditions to a nucleic acid<br>
molecule complementary to the full length of SEQ. ID NO: 25; and/or a nucleic<br>
acid molecule which can hybridize under highly stringent conditions to a nucleic<br>
acid molecule complementary to the full length of SEQ. ID NO: 26. The cell can<br>
be a stem cell, an intervertebral disc cell, a cell of the annulus fibrosus, or a cell of<br>
the nucleus pulposus.<br>
According to a second aspect of the invention, a non-osseous mammalian<br>
cell comprising an isolated nucleic acid sequence encoding a LIM mineralization<br>
protein is provided. According to this aspect of the invention, the cell can be a<br>
stem cell, a cell of the nucleus pulposus, a cell of the annulus fibrosus, or an<br>
intervertebral disc cell.<br>
According to a third aspect of the invention, a method of treating<br>
intervertebral disc injury or disease is provided. According to this aspect of the<br>
invention, the method comprises transfecting an isolated nucleic acid into a<br>
mammalian cell capable of producing proteoglycan and/or collagen. The isolated<br>
nucleic acid comprises a nucleotide sequence encoding a LIM mineralization<br>
protein operably linked to a promoter. The LIM mineralization protein stimulates<br>
proteoglycan and/or collagen synthesis in the cell.<br>
According to a fourth aspect of the invention, an intervertebral disc implant<br>
is provided. According to this aspect of the invention, the implant comprises a<br>
carrier material and a plurality of mammalian cells comprising an isolated nucleic<br>
acid sequence encoding a LIM mineralization protein. Also according to this<br>
aspect of the invention, the carrier material comprises a porous matrix of<br>
biocompatible material and the mammalian cells are incorporated into the carrier<br>
material.<br>
BRIEF DESCRIPTION OF ACCOMPANYING THE DRAWINGS<br>
The present invention may be better understood with reference to the<br>
accompanying drawings in which:<br>
FIG. 1 is a graph showing the production of sulfated glycosaminoglycan<br>
(sGAG) after expression of HLMP-1 by rat intervertebral disc cells transfected<br>
with different MOIs;<br>
FIG. 2 is a chart showing the dose response of rat intervertebral disc cells<br>
six days after infection with different MOI of AdHLMP-1;<br>
FIG. 3 is a chart showing the expression of Aggrecan and BMP-2 mRNA<br>
by AdHLMP-1 transfected rat intervertebral disc cells six days following<br>
transfection with an MOI of 250 virions/cell;<br>
FIG. 4A is a chart showing HLMP-1 mRNA expression 12 hours after<br>
infection with Ad-hLMP-1 at different MOIs;<br>
FIG. 4B is a chart showing the production of sGAG in medium from 3 to 6<br>
days after infection;<br>
FIG. 5 is a chart showing time course changes of the production of sGAG;<br>
FIG. 6A is a chart showing gene response to LMP-1 over-expression in rat<br>
annulus fibrosus cells for aggrecan<br>
FIG. 6B is a chart showing gene response to LMP-1 over-expression in rat<br>
annulus fibrosus cells for BMP-2;<br>
FIG 7 is a graph showing the time course of HLMP-1 mRNA levels in rat<br>
annulus fibrosus cells after infection with AdLMP-1 at MOI of 25;<br>
FIG. 8 is a chart showing changes in mRNA levels of BMPs and aggrecan<br>
in response to HLMP-1 over-expression;<br>
FIG. 9 is a graph showing the time course of sGAG production<br>
enhancement in response to HLMP-1 expression;<br>
FIG. 10 is a chart showing that the LMP-1 mediated increase in sGAG<br>
production is blocked by noggin; and<br>
FIG. 11 is a graph showing the effect of LMP-1 on sGAG in media after<br>
day 6 of culture in monolayer.<br>
DETAILED DESCRIPTION OF THE PREFERRED EMBODIMENTS<br>
The present invention relates to the transfection of non-osseous cells with<br>
nucleic acids encoding LIM mineralization proteins. The present inventors have<br>
discovered that transfection of non-osseous cells such as intervertebral disc cells<br>
with nucleic acids encoding LIM mineralization proteins can result in the increased<br>
synthesis of proteoglycan, collagen and other intervertebral disc components and<br>
tissue. The present invention also provides a method for treating intervertebral<br>
disc disease associated with the loss of proteoglycan, collagen, or other<br>
intervertebral disc components.<br>
It is to be understood that both the foregoing general description and the<br>
following detailed description are exemplary and explanatory and are intended to<br>
provide further explanation of the invention as claimed.<br>
ABBREVIATIONS AND DEFINITIONS<br>
BMP Bone Morphogenetic Protein<br>
HLMP-1 Human LMP-1, also designated as Human LIM<br>
Protein or HLMP<br>
HLMP-ls Human LMP-1 Short (truncated) protein<br>
HLMPU Human LIM Protein Unique Region<br>
LMP LIM mineralization protein<br>
MEM Minimal essential medium<br>
Trm Triamcinolone<br>
?-GlyP Beta-glycerolphosphate<br>
RACE Rapid Amplification of cDN A Ends<br>
RLMP Rat LIM mineralization protein, also designated<br>
asRLMP-1<br>
RLMPU Rat LIM Protein Unique Region<br>
RNAsin RNase inhibitor<br>
ROB Rat Osteoblast<br>
10-4 Clone containing cDNA sequence for RLMP<br>
(SEQIDNO:2)<br>
UTR Untranslated Region<br>
HLMP-2 Human LMP Splice Variant 2<br>
HLMP-3 Human LMP Splice Variant 3<br>
MOI multiplicity of infection<br>
sGAG sulfated glycosaminoglycan<br>
AdHLMP-1 Recombinant Type 5 Adenovirus comprising<br>
nucleotide sequence encoding HLMP-1<br>
A LIM gene (10-4/RLMP) has been isolated from stimulated rat calvarial<br>
osteoblast cultures (SEQ. ID NO: 1, SEQ. ID NO: 2). See U.S. Patent<br>
No. 6,300,127. This gene has been cloned, sequenced and assayed for its ability to<br>
enhance the efficacy of bone mineralization in vitro. The protein RLMP has been<br>
found to affect the mineralization of bone matrix as well as the differentiation of<br>
cells into the osteoblast lineage. Unlike other known cytokines (e.g., BMPs),<br>
RLMP is not a secreted protein, but is instead an intracellular signaling molecule.<br>
This feature has the advantage of providing intracellular signaling amplification as<br>
well as easier assessment of transfected cells. It is also suitable for more efficient<br>
and specific in vivo applications. Suitable clinical applications include<br>
enhancement of bone repair in fractures, bone defects, bone grafting, and normal<br>
homeostasis in patients presenting with osteoporosis.<br>
The amino acid sequence of a corresponding human protein, named human<br>
LMP-1 ("HLMP1"), has also been cloned, sequenced and deduced. See U.S.<br>
Patent No. 6,300,127. The human protein has been found to demonstrate enhanced<br>
efficacy of bone mineralization in vitro and in vivo.<br>
Additionally, a truncated (short) version of HLMP-1, termed HLMP-ls, has<br>
been characterized. See U.S. Patent No. 6,300,127. This short version resulted<br>
from a point mutation in one source of a cDNA clone, providing a stop codon<br>
which truncates the protein. HLMP-ls has been found to be fully functional when<br>
expressed in cell culture and in vivo.<br>
Using PCR analysis of human heart cDNA library, two alternative splice<br>
variants (referred to as HLMP-2 and HLMP-3) have been identified that differ<br>
from HLMP-1 in a region between base pairs 325 and 444 in the nucleotide<br>
sequence encoding HLMP-1. See U.S. Patent Application Serial No. 09/959,578,<br>
filed April 28, 2000, pending. The HLMP-2 sequence has a 119 base pair deletion<br>
and an insertion of 17 base pairs in this region. Compared to HLMP-1, the<br>
nucleotide sequence encoding HLMP-3 has no deletions, but it does have the same<br>
17 base pairs as HLMP-2, which are inserted at position 444 in the HLMP-1<br>
sequence.<br>
LMP is a pluripotent molecule, which regulates or influences a number of<br>
biological processes. The different splice variants of LMP are expected to have<br>
different biological functions in mammals. They may play a role in the growth,<br>
differentiation, and/or regeneration of various tissues. For example, some form of<br>
LMP is expressed not only in bone, but also in muscle, tendons, ligaments, spinal<br>
cord, peripheral nerves, and cartilage.<br>
According to one aspect, the present invention relates to a method of<br>
stimulating proteoglycan and/or collagen synthesis in a mammalian cell by<br>
providing an isolated nucleic acid comprising a nucleotide sequence encoding LIM<br>
mineralization protein operably linked to a promoter; transfecting said isolated<br>
nucleic acid sequence into a mammalian cell capable of producing proteoglycan;<br>
and expressing said nucleotide sequence encoding LIM mineralization protein,<br>
whereby proteoglycan synthesis is stimulated. The mammalian cell may be a non-<br>
osseous cell, such as an intervertebral disc cell, a cell of the annulus fibrosus, or a<br>
cell of the nucleus pulposus. Transfection may occur either ex vivo or in vivo by<br>
direct injection of virus or naked DNA, such as, for example, a plasmid. In certain<br>
embodiments, the virus is a recombinant adenovirus, preferably AdHLMP-1.<br>
Another embodiment of the invention comprises a non-osseous mammalian<br>
cell comprising an isolated nucleic acid sequence encoding a LIM mineralization<br>
protein. The non-osseous mammalian cell may be a stem cell (e.g., a pluripotential<br>
stem cell or a mesenchymal stem cell) or an intervertebral disc cell, preferably a<br>
cell of the nucleus pulposus or a cell of the annulus fibrosus.<br>
In a different aspect, the invention is directed to a method of expressing an<br>
isolated nucleotide sequence encoding LIM mineralization protein in a non-<br>
osseous mammalian cell, comprising providing an isolated nucleic acid comprising<br>
a nucleotide sequence encoding LIM mineralization protein operably linked to a<br>
promoter; transfecting said isolated nucleic acid sequence into a non-osseous<br>
mammalian cell; and expressing said nucleotide sequence encoding LIM<br>
mineralization protein. The non-osseous mammalian cell may be a stem cell or an<br>
intervertebral disc cell (e.g., a cell of the nucleus pulposus or annulus fibrosus).<br>
Transfection may occur either ex vivo or in vivo by direct injection of virus or<br>
naked DNA, such as, for example, a plasmid. The virus can be a recombinant<br>
adenovirus, preferably AdHLMP-1.<br>
In yet another embodiment, the invention is directed to a method of treating<br>
intervertebral disc disease by reversing, retarding or slowing disc degeneration,<br>
comprising providing an isolated nucleic acid comprising a nucleotide sequence<br>
encoding LIM mineralization protein operably linked to a promoter; transfecting<br>
said isolated nucleic acid sequence into a mammalian cell capable of producing<br>
proteoglycan; and stimulating proteoglycan synthesis in said cell by expressing<br>
said nucleotide sequence encoding LIM mineralization protein, whereby disc<br>
degeneration is reversed, halted or slowed. The disc disease may involve lower<br>
back pain, disc herniation, or spinal stenosis. The mammalian cell may be a non-<br>
osseous cell, such as a stem cell or an intervertebral disc cell (e.g., a cell of the<br>
annulus fibrosus, or a cell of the nucleus pulposus).<br>
Transfection may occur either ex vivo or in vivo by direct injection of virus<br>
or naked DNA, such as, for example, a plasmid. In certain embodiments, the virus<br>
is a recombinant adenovirus, preferably AdHLMP-1.<br>
The present invention relates to novel mammalian LIM proteins, herein<br>
designated LIM mineralization proteins, or LMPs. The invention relates more<br>
particularly to human LMP, known as HLMP or HLMP-1, or alternative splice<br>
variants of human LMP, which are known as HLMP-2 or HLMP-3. The<br>
Applicants have discovered that these proteins enhance bone mineralization in<br>
mammalian cells grown in vitro. When produced in mammals, LMP also induces<br>
bone formation in vivo.<br>
Ex vivo transfection of bone marrow cells, osteogenic precursor cells,<br>
peripheral blood cells, and stem cells (e.g., pluripotential stem cells or<br>
mesenchymal stem cells) with nucleic acid that encodes a LIM mineralization<br>
protein (e.g., LMP or HLMP), followed by reimplantation of the transfected cells<br>
in the donor, is suitable for treating a variety of bone-related disorders or injuries.<br>
For example, one can use this method to: augment long bone fracture repair;<br>
generate bone in segmental defects; provide a bone graft substitute for fractures;<br>
facilitate tumor reconstruction or spine fusion; and provide a local treatment (by<br>
injection) for weak or osteoporotic bone, such as in osteoporosis of the hip,<br>
vertebrae, or wrist. Transfection with LMP or HLMP-encoding nucleic acid is also<br>
useful in: the percutaneous injection of transfected marrow cells to accelerate the<br>
repair of fractured long bones; treatment of delayed union or non-unions of long<br>
bone fractures or pseudoarthrosis of spine fusions; and for inducing new bone<br>
formation in avascular necrosis of the hip or knee.<br>
In addition to ex vivo methods of gene therapy, transfection of a<br>
recombinant DNA vector comprising a nucleic acid sequence that encodes LMP or<br>
HLMP can be accomplished in vivo. When a DNA fragment that encodes LMP or<br>
HLMP is inserted into an appropriate viral vector, for example, an adenovirus<br>
vector, the viral construct can be injected directly into a body site were<br>
endochondral bone formation is desired. By using a direct, percutaneous injection<br>
to introduce the LMP or HLMP sequence stimulation of bone formation can be<br>
accomplished without the need for surgical intervention either to obtain bone<br>
marrow cells (to transfect ex vivo) or to reimplant them into the patient at the site<br>
where new bone is required. Alden, et al., Neurosurgical Focus (1998), have<br>
demonstrated the utility of a direct injection method of gene therapy using a cDNA<br>
that encodes BMP-2, which was cloned into an adenovirus vector.<br>
It is also possible to carry out in vivo gene therapy by directly injecting into<br>
an appropriate body site, a naked, that is, unencapsulated, recombinant plasmid<br>
comprising a nucleic acid sequence that encodes HLMP. In this embodiment of<br>
the invention, transfection occurs when the naked plasmid DNA is taken up, or<br>
internalized, by the appropriate target cells, which have been described. As in the<br>
case of in vivo gene therapy using a viral construct, direct injection of naked<br>
plasmid DNA offers the advantage that little or no surgical intervention is required.<br><br>
Direct gene therapy, using naked plasmid DNA that encodes the endothelial cell<br>
mitogen VEGF (vascular endothelial growth factor), has been successfully<br>
demonstrated in human patients. Baumeartner. et al., Circulation, 97, 12, 1114-<br>
1123(1998).<br>
For intervertebral disc applications, ex vivo transfection may be<br>
accomplished by harvesting cells from an intervertebral disc, transfecting the cells<br>
with nucleic acid encoding LMP in vitro, followed by introduction of the cells into<br>
an intervertebral disc. The cells may be harvested from or introduced back into the<br>
intervertebral disc using any means known to those of skill in the art, such as, for<br>
example, any surgical techniques appropriate for use on the spine. In one<br>
embodiment, the cells are introduced into the intervertebral disc by injection.<br>
Also according to the invention, stem cells (e.g., pluripotential stem cells or<br>
mesenchymal stem cells) can be transfected with nucleic acid encoding a LIM<br>
Mineralization Protein ex vivo and introduced into the intervertebral disc (e.g., by<br>
injection).<br>
The cells transfected ex vivo can also be combined with a carrier to form an<br>
intervertebral disc implant. The carrier comprising the transfected cells can then be<br>
implanted into the intervertebral disc of a subject. Suitable carrier materials are<br>
disclosed in Helm, et al., "Bone Graft Substitutes for the Promotion of Spinal<br>
Arthrodesis", Neurosurg Focus, Vol. 10 (4): April 2001. The carrier preferably<br>
comprises a biocompatible porous matrix such as a demineralized bone matrix<br>
(DBM), a biocompatible synthetic polymer matrix or a protein matrix. Suitable<br>
proteins include extracellular matrix proteins such as collagen. The cells<br>
transfected with the LMP ex vivo can be incorporated into the carrier (i.e., into the<br>
pores of the porous matrix) prior to implantation.<br>
Similarly, for intervertebral disc applications where the cells are transfected<br>
in vivo, the DNA may be introduced into the intevertebral disc using any suitable<br>
method known to those of skill in the art. In one embodiment, the nucleic acid is<br>
directly injected into the intervertebral space.<br>
By using an adenovirus vector to deliver LMP into osteogenic cells,<br>
transient expression of LMP is achieved. This occurs because adenovirus does not<br>
incorporate into the genome of target cells that are transfected. Transient<br>
expression of LMP, that is, expression that occurs during the lifetime of the<br>
transfected target cells, is sufficient to achieve the objects of the invention. Stable<br>
expression of LMP, however, can occur when a vector that incorporates into the<br>
genome of the target cell is used as a delivery vehicle. Retrovirus-based vectors,<br>
for example, are suitable for this purpose.<br>
Stable expression of LMP is particularly useful for treating various<br>
systemic bone-related disorders, such as osteoporosis and osteogenesis imperfecta.<br>
For this embodiment of the invention, in addition to using a vector that integrates<br>
into the genome of the target cell to deliver an LMP-encoding nucleotide sequence<br>
into target cells, LMP expression can be placed under the control of a regulatable<br>
promoter. For example, a promoter that is turned on by exposure to an exogenous<br>
inducing agent, such as tetracycline, is suitable.<br>
Using this approach, one can stimulate formation of new bone on a<br>
systemic basis by administering an effective amount of the exogenous inducing<br>
agent. Once a sufficient quantity of bone mass is achieved, administration of the<br>
exogenous inducing agent can be discontinued. This process may be repeated as<br>
needed to replace bone mass lost, for example, as a consequence of osteoporosis.<br>
Antibodies specific for HLMP are particularly suitable for use in methods for<br>
assaying the osteoinductive, that is, bone-forming, potential of patient cells. In this<br>
way one can identify patients at risk for slow or poor healing of bone repair. Also,<br>
HLMP-specific antibodies are suitable for use in marker assays to identify risk<br>
factors in bone degenerative diseases, such as, for example, osteoporosis.<br>
Following well known and conventional methods, the genes of the present<br>
invention are prepared by ligation of nucleic acid segments that encode LMP to<br>
other nucleic acid sequences, such as cloning and/or expression vectors. Methods<br>
needed to construct and analyze these recombinant vectors, for example, restriction<br>
endonuclease digests, cloning protocols, mutagenesis, organic synthesis of<br>
oligonucleotides and DNA sequencing, have been described. For DNA sequencing<br>
DNA, the dieoxyterminator method is the preferred.<br><br>
Many treatises on recombinant DNA methods have been published,<br>
including Sambrook. et al., Molecular Cloning: A Laboratory Manual, 2nd edition,<br>
Cold Spring Harbor Press, (1988), Davis, et al., Basic Methods in Molecular<br>
Biology, Elsevier (1986), and Ausubel. et al., Current Protocols in Molecular<br>
Biology, Wiley Interscience (1988). These reference manuals are specifically<br>
incorporated by reference herein.<br>
Primer-directed amplification of DNA or cDNA is a common step in the<br>
expression of the genes of this invention. It is typically performed by the<br>
polymerase chain reaction (PCR). PCR is described in U.S. Patent No. 4,800, 159<br>
to Mullis, et al. and other published sources. The basic principle of PCR is the<br>
exponential replication of a DNA sequence by successive cycles of primer<br>
extension. The extension products of one primer, when hybridized to another<br>
primer, becomes a template for the synthesis of another nucleic acid molecule. The<br>
primer-template complexes act as substrate for DNA polymerase, which in<br>
performing its replication function, extends the primers. The conventional enzyme<br>
for PCR applications is the thermostable DNA polymerase isolated from Thermus<br>
aquaticus, or Taq DNA polymerase.<br>
Numerous variations of the basic PCR method exist, and a particular<br>
procedure of choice in any given step needed to construct the recombinant vectors<br>
of this invention is readily performed by a skilled artisan. For example, to measure<br>
cellular expression of 10-4/RLMP, RNA is extracted and reverse transcribed under<br>
standard and well known procedures. The resulting cDNA is then analyzed for the<br>
appropriate mRNA sequence by PCR.<br>
The gene encoding the LIM mineralization protein is expressed in an<br>
expression vector in a recombinant expression system. Of course, the constructed<br>
sequence need not be the same as the original, or its complimentary sequence, but<br>
instead may be any sequence determined by the degeneracy of the DNA code that<br>
nonetheless expresses an LMP having bone forming activity. Conservative amino<br>
acid substitutions, or other modifications, such as the occurrence of an amino-<br>
terminal methionine residue, may also be employed.<br><br>
A ribosome binding site active in the host expression system of choice is<br>
ligated to the 5" end of the chimeric LMP coding sequence, forming a synthetic<br>
gene. The synthetic gene can be inserted into any one of a large variety of vectors<br>
for expression by ligating to an appropriately linearized plasmid. A regulatable<br>
promoter, for example, the E. coli lac promoter, is also suitable for the expression<br>
of the chimeric coding sequences. Other suitable regulatable promoters include<br>
trp, tac, recA, T7 and lambda promoters.<br>
DNA encoding LMP is transfected into recipient cells by one of several<br>
standard published procedures, for example, calcium phosphate precipitation,<br>
DEAE-Dextran, electroporation or protoplast fusion, to form stable transformants.<br>
Calcium phosphate precipitation is preferred, particularly when performed as<br>
follows.<br>
DNAs are coprecipitated with calcium phosphate according to the method<br>
of Graham and Van Per. Virology, 52, 456 (1973), before transfer into cells. An<br>
aliquot of 40-50 µg of DNA, with salmon sperm or calf thymus DNA as a carrier,<br>
is used for 0.5 x 106 cells plated on a 100 mm dish. The DNA is mixed with 0.5 ml<br>
of 2X Hepes solution (280 mM NaCl, 50 mM Hepes and 1.5 mM Na2HPO4,<br>
pH 7.0), to which an equal volume of 2x CaCl2 (250 mM CaCl2 and 10 mM Hepes,<br>
pH 7.0) is added. A white granular precipitate, appearing after 30-40 minutes, is<br>
evenly distributed dropwise on the cells, which are allowed to incubate for 4-16<br>
hours at 37°C. The medium is removed and the cells shocked with 15% glycerol<br>
in PBS for 3 minutes. After removing the glycerol, the cells are fed with<br>
Dulbecco"s Minimal Essential Medium (DMEM) containing 10% fetal bovine<br>
serum.<br>
DNA can also be transfected using: the DEAE-Dextran methods of<br>
Kimura, et al., Virology, 49:394 (1972) and Sompayrac, et al., Proc. Natl. Acad.<br>
Sci. USA, 78, 7575 (1981); the electroporation method of Potter. Proc. Natl. Acad.<br>
Sci. USA, 81, 7161 (1984); and the protoplast fusion method of Sandri-<br>
Goddin, et al., Molec. Cell. Biol., 1, 743 (1981).<br>
Phosphoramidite chemistry in solid phase is the preferred method for the<br>
organic synthesis of oligodeoxynucleotides and polydeoxynucleotides. In addition,<br><br>
many other organic synthesis methods are available. Those methods are readily<br>
adapted by those skilled in the art to the particular sequences of the invention.<br>
The present invention also includes nucleic acid molecules that hybridize<br>
under standard conditions to any of the nucleic acid sequences encoding the LIM<br>
mineralization proteins of the invention. "Standard hybridization conditions" will<br>
vary with the size of the probe, the background and the concentration of the nucleic<br>
acid reagents, as well as the type of hybridization, for example, in situ, Southern<br>
blot, or hybrization of DNA-RNA hybrids (Northern blot). The determination of<br>
"standard hybridization conditions" is within the level of skill in the art. For<br>
example, see U.S. Patent 5,580,775 to Fremeau. et al., herein incorporated by<br>
reference for this purpose. See also. Southern. J. Mol. Biol., 98:503 (1975),<br>
Alwine. et al., Meth. Enzymol., 68:220 (1979), and Sambrook. et al., Molecular<br>
Cloning: A Laboratory Manual, 2nd edition, Cold Spring Harbor Press, 7.19-7.50<br>
(1989).<br>
One preferred set of standard hybrization conditions involves a blot that is<br>
prehybridized at 42°C for 2 hours in 50% formamide, 5X SSPE (150 nM NaCl,<br>
10 mM Na H2PO4 [pH 7.4], 1 mM EDTA [pH 8.0])l 5X Denhardt"s solution<br>
(20 mg Ficoll, 20 mg polyvinylpyrrolidone and 20 mg BSA per 100 ml water),<br>
10% dextran sulphate, 1% SDS and 100 µg/ml salmon sperm DNA. A 32P- labeled<br>
cDNA probe is added, and hybridization is continued for 14 hours. Afterward, the<br>
blot is washed twice with 2X SSPE, 0.1 % SDS for 20 minutes at 22°C, followed<br>
by a 1 hour wash at 65°C in 0.1X SSPE, 0.1 %SDS. The blot is then dried and<br>
exposed to x-ray film for 5 days in the presence of an intensifying screen.<br>
Under "highly stringent conditions," a probe will hybridize to its target<br>
sequence if those two sequences are substantially identical. As in the case of<br>
standard hybridization conditions, one of skill in the art can, given the level of skill<br>
in the art and the nature of the particular experiment, determine the conditions<br>
under which only substantially identical sequences will hybridize.<br>
According to one aspect of the present invention, an isolated nucleic acid<br>
molecule comprising a nucleic acid sequence encoding a LIM mineralization<br>
protein is provided. The nucleic acid molecule according to the invention can be a<br><br>
molecule which hybridizes under standard conditions to a nucleic acid molecule<br>
complementary to the full length of SEQ. ID NO: 25 and/or which hybridizes<br>
under highly stringent conditions to a nucleic acid molecule complementary to the<br>
full length of SEQ. ID NO: 26. More specifically, the isolated nucleic acid<br>
molecule according to the invention can encode HLMP-1, HLMP-1s, RLMP,<br>
HLMP-2, or HLMP-3.<br>
Another aspect of the invention includes the proteins encoded by the<br>
nucleic acid sequences. In still another embodiment, the invention relates to the<br>
identification of such proteins based on anti-LMP antibodies. In this embodiment,<br>
protein samples are prepared for Western blot analysis by lysing cells and<br>
separating the proteins by SDS-PAGE. The proteins are transferred to<br>
nitrocellulose by electrobloffing as described by Ausubel, et al., Current Protocols<br>
in Molecular Biology, John Wiley and Sons (1987). After blocking the filter with<br>
instant nonfat dry milk (1 gm in 100 ml PBS), anti-LMP antibody is added to the<br>
filter and incubated for 1 hour at room temperature. The filter is washed<br>
thoroughly with phosphate buffered saline (PBS) and incubated with horseradish<br>
peroxidase (HRPO)-antibody conjugate for 1 hour at room temperature. The filter<br>
is again washed thoroughly with PBS and the antigen bands are identified by<br>
adding diaminobenzidine (DAB).<br>
Monospecific antibodies are the reagent of choice in the present invention,<br>
and are specifically used to analyze patient cells for specific characteristics<br>
associated with the expression of LMP. "Monospecific antibody" as used herein is<br>
defined as a single antibody species or multiple antibody species with homogenous<br>
binding characteristics for LMP. "Homogeneous binding" as used herein refers to<br>
the ability of the antibody species to bind to a specific antigen or epitope, such as<br>
those associated with LMP, as described above. Monospecific antibodies to LMP<br>
are purified from mammalian antisera containing antibodies reactive against LMP<br>
or are prepared as monoclonal antibodies reactive with LMP using the technique of<br>
Kohler and Milstein. Kohler, et al., Nature. 256, 495-497 (1975). The LMP<br>
specific antibodies are raised by immunizing animals such as, for example, mice,<br><br>
rats, guinea pigs, rabbits, goats or horses, with an appropriate concentration of<br>
LMP either with or without an immune adjuvant.<br>
In this process, pre-immune serum is collected prior to the first<br>
immunization. Each animal receives between about 0.1 mg and about 1000 mg of<br>
LMP associated with an acceptable immune adjuvant, if desired. Such acceptable<br>
adjuvants include, but are not limited to, Freund"s complete, Freund"s incomplete,<br>
alum-precipitate, water in oil emulsion containing Corynebacterium parvum and<br>
tRNA adjuvants. The initial immunization consists of LMP in, preferably,<br>
Freund"s complete adjuvant injected at multiple sites either subcutaneously (SC),<br>
intraperitoneally (IP) or both. Each animal is bled at regular intervals, preferably<br>
weekly, to determine antibody titer. The animals may or may not receive booster<br>
injections following the initial immunization. Those animals receiving booster<br>
injections are generally given an equal amount of the antigen in Freund"s<br>
incomplete adjuvant by the same route. Booster injections are given at about three<br>
week intervals until maximal titers are obtained. At about 7 days after each<br>
booster immunization or about weekly after a single immunization, the animals are<br>
bled, the serum collected, and aliquots are stored at about -20°C.<br>
Monoclonal antibodies (mAb) reactive with LMP are prepared by<br>
immunizing inbred mice, preferably Balb/c mice, with LMP. The mice are<br>
immunized by the IP or SC route with about 0.1 mg to about 10 mg, preferably<br>
about 1 mg, of LMP in about 0.5 ml buffer or saline incorporated in an equal<br>
volume of an acceptable adjuvant, as discussed above. Freund"s complete adjuvant<br>
is preferred. The mice receive an initial immunization on day 0 and are rested for<br>
about 3-30 weeks. Immunized mice are given one or more booster immunizations<br>
of about 0.1 to about 10 mg of LMP in a buffer solution such as phosphate<br>
buffered saline by the intravenous (IV) route. Lymphocytes from antibody-<br>
positive mice, preferably splenic lymphocytes, are obtained by removing the<br>
spleens from immunized mice by standard procedures known in the art.<br>
Hybridoma cells are produced by mixing the splenic lymphocytes with an<br>
appropriate fusion partner, preferably myeloma cells, under conditions which will<br>
allow the formation of stable hybridomas. Fusion partners may include, but are not<br><br>
limited to: mouse myelomas P3/NSl/Ag 4-1; MPC-11; S-194 and Sp 2/0, with Sp<br>
2/0 being preferred. The antibody producing cells and myeloma cells are fused in<br>
polyethylene glycol, about 1,000 mol. wt., at concentrations from about 30% to<br>
about 50%. Fused hybridoma cells are selected by growth in hypoxanthine,<br>
thymidine and aminopterin in supplemented Dulbecco"s Modified Eagles Medium<br>
(DMEM) by procedures known in the art. Supernatant fluids are collected from<br>
growth positive wells on about days 14, 18, and 21, and are screened for antibody<br>
production by an immunoassay such as solid phase immunoradioassay (SPIRA)<br>
using LMP as the antigen. The culture fluids are also tested in the Ouchterlony<br>
precipitation assay to determine the isotype of the mAb. Hybridoma cells from<br>
antibody positive wells are cloned by a technique such as the soft agar technique of<br>
MacPherson. "Soft Agar Techniques: Tissue Culture Methods and Applications",<br>
Kruse and Paterson (eds.), Academic Press (1973). See, also, Harlow, et al.,<br>
Antibodies: A Laboratory Manual, Cold Spring Laboratory (1988).<br>
Monoclonal antibodies may also be produced in vivo by injection of<br>
pristane- primed Balb/c mice, approximately 0.5 ml per mouse, with about 2x106 to<br>
about 6x106 hybridoma cells about 4 days after priming. Ascites fluid is collected<br>
at approximately 8-12 days after cell transfer and the monoclonal antibodies are<br>
purified by techniques known in the art.<br>
In vitro production in anti-LMP mAb is carried out by growing the<br>
hydridoma cell line in DMEM containing about 2% fetal calf serum to obtain<br>
sufficient quantities of the specific mAb. The mAb are purified by techniques<br>
known in the art.<br>
Antibody titers of ascites or hybridoma culture fluids are determined by<br>
various serological or immunological assays, which include, but are not limited to,<br>
precipitation, passive agglutination, enzyme-linked immunosorbent antibody<br>
(ELISA) technique and radioimmunoassay (RIA) techniques. Similar assays are<br>
used to detect the presence of the LMP in body fluids or tissue and cell extracts.<br>
It is readily apparent to those skilled in the art that the above described<br>
methods for producing monospecific antibodies may be utilized to produce<br><br>
antibodies specific for polypeptide fragments of LMP, full-length nascent LMP<br>
polypeptide, or variants or alleles thereof.<br>
In another embodiment, the invention is directed to alternative splice<br>
variants of HLMP-1. PCR analysis of human heart cDNA revealed mRNA for two<br>
HLMP alternative splice variants, named HLMP-2 and HLMP-3, that differ from<br>
HLMP-1 in a region between base pairs 325 and 444 in the HLMP-1 sequence.<br>
The HLMP-2 sequence has a 119 base pair deletion and an insertion of 17 base<br>
pairs in this region. These changes preserve the reading frame, resulting in a 423<br>
amino acid protein, which compared to HLMP-1, has a net loss of 34 amino acids<br>
(40 amino acids deleted plus 6 inserted amino acids). HLMP-2 contains the c-<br>
terminal LIM domains that are present in HLMP-1.<br>
Compared to HLMP-1, HLMP-3 has no deletions, but it does have the same<br>
17 base pair insertion at position 444. This insertion shifts the reading frame,<br>
causing a stop codon at base pairs 459-461. As a result, HLMP-3 encodes a<br>
protein of 153 amino acids. This protein lacks the c-terminal LIM domains that are<br>
present in HLMP-1 and HLMP-2. The predicted size of the proteins encoded by<br>
HLMP-2 and HLMP-3 was confirmed by western blot analysis.<br>
PCR analysis of the tissue distribution of the three splice variants revealed<br>
that they are differentially expressed, with specific isoforms predominating in<br>
different tissues. HLMP-1 is apparently the predominant form expressed in<br>
leukocytes, spleen, lung, placenta, and fetal liver. HLMP-2 appears to be the<br>
predominant isoform in skeletal muscle, bone marrow, and heart tissue. HLMP-3,<br>
however, was not the predominant isoform in any tissue examined.<br>
Over-expression of HLMP-3 in secondary rat osteoblast cultures induced<br>
bone nodule formation (287±56) similar to the effect seen for glucicorticoid<br>
(272±7) and HLMP-1 (232±200). Since HLMP-3 lacks the C-terminal LIM<br>
domains, there regions are not required for osteoinductive activity.<br>
Over-expression of HLMP-2, however, did not induce nodule formation<br>
(11±3). These data suggest that the amino acids encoded by the deleted 119 base<br>
pairs are necessary for osteoinduction. The data also suggest that the distribution<br>
of HLMP splice variants may be important for tissue-specific function.<br><br>
Surprisingly, we have shown that HLMP-2 inhibits steroid-induced osteoblast<br>
formation in secondary rat osteoblast cultures. Therefore, HLMP-2 may have<br>
therapeutic utility in clinical situations where bone formation is not desirable.<br>
On July 22, 1997, a sample of 10-4/RLMP in a vector designated<br>
pCMV2/RLMP (which is vector pRc/CMV2 with insert 10-4 clone/RLMP) was<br>
deposited with the American Type Culture Collection (ATCC), 12301 Parklawn<br>
Drive, Rockville, MD 20852. The culture accession number for that deposit is<br>
209153. On March 19, 1998, a sample of the vector pHis-A with insert HLPM-ls<br>
was deposited at the American Type Culture Collection ("ATCC"). The culture<br>
accession number for that deposit is 209698. On April 14, 2000, samples of<br>
plasmids pHAhLMP-2 (vector pHisA with cDNA insert derived from human heart<br>
muscle cDNA with HLMP-2) and pHAhLMP-3 (vector pHisA with cDNA insert<br>
derived from human heart muscle cDNA with HLMP-3) were deposited with the<br>
ATCC, 10801 University Blvd., Manassas, VA, 20110-2209, USA, under the<br>
conditions of the Budapest treaty. The accession numbers for these deposits are<br>
PTA-1698 and PTA-1699, respectively. These deposits, as required by the<br>
Budapest Treaty, will be maintained in the ATCC for at least 30 years and will be<br>
made available to the public upon the grant of a patent disclosing them. It should<br>
be understood that the availability of a deposit does not constitute a license to<br>
practice the subject invention in derogation of patent rights granted by government<br>
action.<br>
In assessing the nucleic acids, proteins, or antibodies of the invention,<br>
enzyme assays, protein purification, and other conventional biochemical methods<br>
are employed. DNA and RNA are analyzed by Southern blofting and Northern<br>
blotting techniques, respectively. Typically, the samples analyzed are size<br>
fractionated by gel electrophoresis. The DNA or RNA in the gels are then<br>
transferred to nitrocellulose or nylon membranes. The blots, which are replicas of<br>
sample patterns in the gels, were then hybridized with probes. Typically, the<br>
probes are radio-labeled, preferably with 32P , although one could label the probes<br>
with other signal-generating molecules known to those in the art. Specific bands of<br>
interest can then be visualized by detection systems, such as autoradiography.<br><br>
For purposes of illustrating preferred embodiments of the present invention,<br>
the following, non-limiting examples are included. These results demonstrate the<br>
feasibility of inducing or enhancing the formation of bone using the LIM<br>
mineralization proteins of the invention, and the isolated nucleic acid molecules<br>
encoding those proteins.<br>
EXAMPLE 1: Calvarial Cell Culture<br>
Rat calvarial cells, also known as rat osteoblasts ("ROB"), were obtained<br>
from 20-day pre-parturition rats as previously described. Boden, et al.,<br>
Endocrinology, 137, 8, 3401-3407 (1996). Primary cultures were grown to<br>
confluence (7 days), trypsinized, and passed into 6-well plates (1 x 105 cells/35 mm<br>
well) as first subculture cells. The subculture cells, which were confluent at day 0,<br>
were grown for an additional 7 days. Beginning on day 0, media were changed and<br>
treatments (Trm and/or BMPs) were applied, under a laminar flow hood, every 3 or<br>
4 days. The standard culture protocol was as follows: days 1-7, MEM, 10% FBS,<br>
50 µg/ml ascorbic acid, ± stimulus; days 8-14, BGJb medium, 10% FBS, 5mM ?-<br>
GlyP (as a source of inorganic phosphate to permit mineralization). Endpoint<br>
analysis of bone nodule formation and osteocalcin secretion was performed at<br>
day 14. The dose of BMP was chosen as 50 ng/ml based on pilot experiments in<br>
this system that demonstrated a mid-range effect on the dose-response curve for all<br>
BMPs studied.<br>
EXAMPLE 2: Antisense Treatment And Cell Culture<br>
To explore the potential functional role of LMP-1 during membranous bone<br>
formation, we synthesized an antisense oligonucleotide to block LMP-1 mRNA<br>
translation and treated secondary osteoblast cultures that were undergoing<br>
differentiation initiated by glucocorticoid. Inhibition of RLMP expression was<br>
accomplished with a highly specific antisense oligonucleotide (having no<br>
significant homologies to known rat sequences) corresponding to a 25 bp sequence<br>
spanning the putative translational start site (SEQ. ID NO: 42). Control cultures<br>
either did not receive oligonucleotide or they received sense oligonucleotide.<br><br>
Experiments were performed in the presence (preincubation) and absence of<br>
lipofectamine. Briefly, 22 µg of sense or antisense RLMP oligonucleotide was<br>
incubated in MEM for 45 minutes at room temperature. Following that incubation,<br>
either more MEM or pre-incubated lipofectamine/MEM (7% v/v; incubated 45<br>
minutes at room temperature) was added to achieve an oligonucleotide<br>
concentration of 0.2 µM. The resulting mixture was incubated for 15 minutes at<br>
room temperature. Oligonucleotide mixtures were then mixed with the appropriate<br>
medium, that is, MEM/Ascorbate/±Trm, to achieve a final oligonucleotide<br>
concentration of 0.1 µM.<br>
Cells were incubated with the appropriate medium (±stimulus) in the<br>
presence or absence of the appropriate oligonucleotides. Cultures originally<br>
incubated with lipofectamine were re-fed after 4 hours of incubation (37°C; 5%<br>
CO2) with media containing neither lipofectamine nor oligonucleotide. All<br>
cultures, especially cultures receiving oligonucleotide, were re-fed every 24 hours<br>
to maintain oligonucleotide levels.<br>
LMP-1 antisense oligonucleotide inhibited mineralized nodule formation<br>
and osteocalcin secretion in a dose-dependent manner, similar to the effect of<br>
BMP-6 oligonucleotide. The LMP-1 antisense block in osteoblast differentiation<br>
could not be rescued by addition of exogenous BMP-6, while the BMP-6 antisense<br>
oligonucleotide inhibition was reversed with addition of BMP-6. This experiment<br>
further confirmed the upstream position of LMP-1 relative to BMP-6 in the<br>
osteoblast differentiation pathway. LMP-1 antisense oligonucleotide also inhibited<br>
spontaneous osteoblast differentiation in primary rat osteoblast cultures.<br>
EXAMPLE 3: Quantitation of Mineralized Bone Nodule Formation<br>
Cultures of ROBs prepared according to Examples 1 and 2 were fixed<br>
overnight in 70% ethanol and stained with von Kossa silver stain. A semi-<br>
automated computerized video image analysis system was used to quantitate<br>
nodule count and nodule area in each well. Boden, et al., Endocrinology, 137, 8,<br>
3401-3407 (1996). These values were then divided to calculate the area per nodule<br>
values. This automated process was validated against a manual counting technique<br>
and demonstrated a correlation coefficient of 0.92 (p 
<br>
expressed as the mean ± standard error of the mean (S.E.M.) calculated from 5 or 6<br>
wells at each condition. Each experiment was confirmed at least twice using cells<br>
from different calvarial preparations.<br>
EXAMPLE 4: Quantitation of Osteocalcin Secretion<br>
Osteocalcin levels in the culture media were measured using a competitive<br>
radioimmunoassay with a monospecific polygonal antibody (Pab) raised in our<br>
laboratory against the C-terminal nonapeptide of rat osteocalcin as described in<br>
Nanes, et al., Endocrinology. 127:588(1990). Briefly, 1 µg of nonapeptide was<br>
iodinated with 1 mCi 125I-Na by the lactoperoxidase method. Tubes containing 200<br>
gl of assay buffer (0.02 M sodium phosphate, 1 mM EDTA, 0.001 % thimerosal,<br>
0.025% BSA) received media taken from cell cultures or osteocalcin standards<br>
(0 - 12,000 frnole) at 100 gl/tube in assay buffer. The Pab (1:40,000; 100 µl) was<br>
then added, followed by the iodinated peptide (12,000 cpm; 100 µl). Samples<br>
tested for non-specific binding were prepared similarly but contained no antibody.<br>
Bound and free PAbs were separated by the addition of 700 µl goat<br>
antirabbit IgG, followed by incubation for 18 hours at 4°C. After samples were<br>
centrifuged at 1200 rpm for 45 minutes, the supernatants were decanted and the<br>
precipitates counted in a gamma counter. Osteocalcin values were reported in<br>
fmole/100 µl, which was then converted to pmole/ml medium (3-day production)<br>
by dividing those values by 100. Values were expressed as the mean ± S.E.M. of<br>
triplicate determinations for 5-6 wells for each condition. Each experiment was<br>
confirmed at least two times using cells from different calvarial preparations.<br>
EXAMPLE 5: Effect of Trm and RLMP on Mineralization In Vitro<br>
There was little apparent effect of either the sense or antisense<br>
oligonucleotides on the overall production of bone nodules in the non-stimulated<br>
cell culture system. When ROBs were stimulated with Trm, however, the<br>
antisense oligonucleotide to RLMP inhibited mineralization of nodules by &gt; 95%.<br>
The addition of exogenous BMP-6 to the oligonucleotide-treated cultures did not<br>
rescue the mineralization of RLMP-antisense-treated nodules.<br><br>
Osteocalcin has long been synonymous with bone mineralization, and<br>
osteocalcin levels have been correlated with nodule production and mineralization.<br>
The RLMP-antisense oligonucleotide significantly decreases osteocalcin<br>
production, but the nodule count in antisense-treated cultures does not change<br>
significantly. In this case, the addition of exogenous BMP-6 only rescued the<br>
production of osteocalcin in RLMP-antisense-treated cultures by 10-15%. This<br>
suggests that the action of RLMP is downstream of, and more specific than,<br>
BMP-6.<br>
EXAMPLE 6: Harvest and Purification of RNA<br>
Cellular RNA from duplicate wells of ROBs (prepared according to<br>
Examples 1 and 2 in 6-well culture dishes) was harvested using 4M guanidine<br>
isothiocyanate (GIT) solution to yield statistical triplicates. Briefly, culture<br>
supernatant was aspirated from the wells, which were then overlayed with 0.6 ml<br>
of GIT solution per duplicate well harvest. After adding the GIT solution, the<br>
plates were swirled for 5-10 seconds (being as consistent as possible). Samples<br>
were saved at -70 °C for up to 7 days before further processing.<br>
RNA was purified by a slight modification of standard methods according<br>
to Sambrook, et al. Molecular Cloning: a Laboratory Manual, Chapter 7.19, 2nd<br>
Edition, Cold Spring Harbor Press (1989). Briefly, thawed samples received 60 µl<br>
2.0 M sodium acetate (pH 4.0), 550 µl phenol (water saturated) and 150 µl<br>
chlorofornrisoamyl alcohol (49:1). After vortexing, the samples were centrifuged<br>
(10000 x g; 20 minutes; 4 °C), the aqueous phase transferred to a fresh tube, 600<br>
ul isopropanol was added and the RNA precipitated overnight at -20°C.<br>
Following the overnight incubation, the samples were centrifuged<br>
(10000 x g; 20 minutes) and the supernatant was aspirated gently. The pellets were<br>
resuspended in 400 µl DEPC-treated water, extracted once with phenol:hloroform<br>
(1:1), extracted with chloroform:isoamyl alcohol (24:1) and precipitated overnight<br>
at -20°C after addition of 40 µl sodium acetate (3.0 M; pH 5.2) and 1.0 ml absolute<br>
ethanol. To recover the cellular RNA, the samples were centrifuged<br>
(10000 x g; 20 min), washed once with 70% ethanol, air dried for 5-10 minutes and<br><br>
resuspended in 20 µl of DEPC-treated water. RNA concentrations were calculated<br>
from optical densities that were determined with a spectrophotometer.<br>
EXAMPLE 7: Reverse Transcription-Polvmerase Chain Reaction 25<br>
Heated total RNA (5 µg in 10.5 ul total volume DEPC-H2O at 65°C for<br>
5 minutes) was added to tubes containing 4 µl 5X MMLV-RT buffer, 2 µl dNTPs,<br>
2 µl dT17 primer (10 pmol/ml), 0.5 µl RNAsin (40 U/ml) and 1 µl MMLV-RT<br>
(200 units/µl). The samples were incubated at 37°C for 1 hour, then at 95°C for<br>
5 minutes to inactivate the MMLV-RT. The samples were diluted by addition of<br>
80 µl of water.<br>
Reverse-transcribed samples (5 µl) were subjected to polymerase-chain<br>
reaction using standard methodologies (50 µl total volume). Briefly, samples were<br>
added to tubes containing water and appropriate amounts of PCR buffer, 25 mM<br>
MgCl2, dNTPs, forward and reverse primers for glyceraldehyde 3-phosphate<br>
dehydrogenase (GAP, a housekeeping gene) and/or BMP-6,32P-dCTP, and Taq<br>
polymerase. Unless otherwise noted, primers were standardized to run consistently<br>
at 22 cycles (94°C, 30"; 58°C, 30"; 72°C, 20").<br>
EXAMPLE 8: Quantitation of RT-PCR Products bv Polvacrylamide Gel<br>
Electrophoresis (PAGE) and PhosphorImager Analysis<br>
RT-PCR products received 5 µl/tube loading dye, were mixed, heated at<br>
65 °C for 10 min and centrifuged. Ten ul of each reaction was subjected to PAGE<br>
(12% polyacrylamide:bis; 15 V/well; constant current) under standard conditions.<br>
Gels were then incubated in gel preserving buffer (10% v/v glycerol, 7% v/v acetic<br>
acid, 40% v/v methanol, 43% deionized water) for 30 minutes, dried (80°C) in<br>
vacuo for 1-2 hours and developed with an electronically-enhanced<br>
phosphoresence imaging system for 6-24 hours. Visualized bands were analyzed.<br>
Counts per band were plotted graphically.<br><br>
EXAMPLE 9: Differential Display PCR<br>
RNA was extracted from cells stimulated with glucocorticoid (Trm, 1 nM).<br>
Heated, DNase-treated total RNA (5 µg in 10.5 µl total volume in DEPC- H2O at<br>
65 °C for 5 minutes) was reverse transcribed as described in Example 7, but H-T11<br>
M (SEQ. ID. NO: 4) was used as the MMLV-RT primer. The resulting cDNAs<br>
were PCR-amplified as described above, but with various commercial primer sets<br>
(for example, H-T11G (SEQ. ID NO: 4) and H-AP-10 (SEQ. ID NO: 5); GenHunter<br>
Corp, Nashville, TN). Radio-labeled PCR products were fractionated by gel<br>
electrophoresis on a DNA sequencing gel. After electrophoresis, the resulting gels<br>
were dried in vacuo and autoradiographs were exposed overnight. Bands<br>
representing differentially-expressed cDNAs were excised from the gel and<br>
reamplified by PCR using the method of Conner, et al., Proc. Natl. Acad. Sci.<br>
USA, 88, 278 (1983). The products of PCR reamplification were cloned into the<br>
vector PCR-11 (TA cloning kit; InVitrogen, Carlsbad, CA).<br>
EXAMPLE 10: Screening of a UMR 106 Rat Osteosarcoma Cell cDNA Library<br>
A UMR 106 library (2.5 x 1010 pfu/ml) was plated at 5 x 104 pfu/ml onto<br>
agar plates (LB bottom agar) and the plates were incubated overnight at 37°C.<br>
Filter membranes were overlaid onto plates for two minutes. Once removed, the<br>
filters were denatured, rinsed, dried and UV cross-linked. The filters were then<br>
incubated in pre-hyridization buffer (2X PIPES [pH 6.5], 5% formamide, 1% SDS<br>
and 100 µg/ml denatured salmon sperm DNA) for 2 h at 42°C. A 260 base-pair<br>
radio-labeled probe (SEQ. ID NO: 3; 32P labeled by random priming) was added to<br>
the entire hybridization mix/filters, followed by hybridization for 18 hours at 42°C.<br>
The membranes were washed once at room temperature (10 min, 1 x SSC, 0.1%<br>
SDS) and three times at 55°C (15 min, 0.1 x SSC, 0.1 % SDS).<br>
After they were washed, the membranes were analyzed by autoradiography<br>
as described above. Positive clones were plaque purified. The procedure was<br>
repeated with a second filter for four minutes to minimize spurious positives.<br>
Plaque-purified clones were rescued as lambda SK(-) phagemids. Cloned cDNAs<br>
were sequenced as described below.<br><br>
EXAMPLE 11: Sequencing of Clones<br>
Cloned cDNA inserts were sequenced by standard methods. Ausubel. et al.,<br>
Current Protocols in Molecular Biology, Wiley Interscience (1988). Briefly,<br>
appropriate concentrations of termination mixture, template and reaction mixture<br>
were subjected to an appropriate cycling protocol (95°C, 30 s; 68°C, 30 s; 72°C,<br>
60 s; x 25). Stop mixture was added to terminate the sequencing reactions. After<br>
heating at 92 °C for 3 minutes, the samples were loaded onto a denaturing 6%<br>
polyacrylamide sequencing gel (29:1 acrylamide:bisacrylamide). Samples were<br>
electrophoresed for about 4 hours at 60 volts, constant current. After<br>
electrophoresis, the gels were dried in vacuo and autoradiographed.<br>
The autoradiographs were analyzed manually. The resulting sequences<br>
were screened against the databases maintained by the National Center for<br>
Biotechnology Information (NIH, Bethesda, MD; hftp://www.ncbi.nlm.nih.gov/)<br>
using the BLASTN program set with default parameters. Based on the sequence<br>
data, new sequencing primers were prepared and the process was repeated until the<br>
entire gene had been sequenced. All sequences were confirmed a minimum of<br>
three times in both orientations.<br>
Nucleotide and amino acid sequences were also analyzed using the<br>
PCGENE software package (version 16.0). Percent homology values for<br>
nucleotide sequences were calculated by the program NALIGN, using the<br>
following parameters: weight of non-matching nucleotides, 10; weight of non-<br>
matching gaps, 10; maximum number of nucleotides considered, 50; and minimum<br>
number of nucleotides considered, 50.<br>
For amino acid sequences, percent homology values were calculated using<br>
PALIGN. A value of 10 was selected for both the open gap cost and the unit gap<br>
cost.<br>
EXAMPLE 12: Cloning of RLMP cDNA<br>
The differential display PCR amplification products described in Example 9<br>
contained a major band of approximately 260 base pairs. This sequence was used<br>
to screen a rat osteosarcoma (UMR 106) cDNA library. Positive clones were<br><br>
subjected to nested primer analysis to obtain the primer sequences necessary for<br>
amplifying the full length cDNA. (SEQ. ID NOs: 11, 12, 29, 30 and 31). One of<br>
those positive clones selected for further study was designated clone 10-4.<br>
Sequence analysis of the full-length cDNA in clone 10-4, determined by<br>
nested primer analysis, showed that clone 10-4 contained the original 260 base-pair<br>
fragment identified by differential display PCR. Clone 10-4 (1696 base pairs;<br>
SEQ ID NO: 2) contains an open reading frame of 1371 base pairs encoding a<br>
protein having 457 amino acids (SEQ. ID NO: 1). The termination codon, TGA,<br>
occurs at nucleotides 1444-1446. The polyadenylation signal at nucleotides 1675-<br>
1680, and adjacent poly(A)+ tail, was present in the 3" noncoding region. There<br>
were two potential N-glycosylation sites, Asn-Lys-Thr and Asn-Arg-Thr, at amino<br>
acid positions 113-116 and 257-259 in SEQ. ID NO: 1, respectively. Two<br>
potential cAMP- and cGMP-dependent protein kinase phosphorylation sites, Ser<br>
and Thr, were found at amino acid positions 191 and 349, respectively. There were<br>
five potential protein kinase C phosphorylation sites, Ser or Thr, at amino acid<br>
positions 3, 115, 166, 219, 442. One potential ATP/GTP binding site motif A (P-<br>
loop), Gly-Gly-Ser-Asn-Asn-Gly-Lys-Thr, was determined at amino acid positions<br>
272-279.<br>
In addition, two highly conserved putative LIM domains were found at<br>
amino acid positions 341-391 and 400-451. The putative LIM domains in this<br>
newly identified rat cDNA clone showed considerable homology with the LIM<br>
domains of other known LIM proteins. However, the overall homology with other<br>
rat LIM proteins was less than 25%. RLMP (also designated 10-4) has 78.5%<br>
amino acid homology to the human enigma protein (see U.S. Patent<br>
No. 5,504,192), but only 24.5% and 22.7% amino acid homology to its closest rat<br>
homologs, CLP-36 and RIT-18, respectively.<br>
EXAMPLE 13: Northern Blot Analysis of RLMP Expression<br>
Thirty ug of total RNA from ROBs, prepared according to Examples 1 and<br>
2, was size fractionated by formaldehyde gel electrophoresis in 1 % agarose flatbed<br>
gels and osmotically transblotted to nylon membranes. The blot was probed with a<br><br>
600 base pair EcoRl fragment of full-length 10-4 cDNA labeled with 32P-dCTP by<br>
random priming.<br>
Northern blot analysis showed a 1.7 kb mRNA species that hybridized with<br>
the RLMP probe. RLMP mRNA was up-regulated approximately 3.7-fold in<br>
ROBs after 24 hours exposure to BMP-6. No up-regulation of RMLP expression<br>
was seen in BMP-2 or BMP-4-stimulated ROBs at 24 hours.<br>
EXAMPLE 14: Statistical Methods<br>
For each reported nodule/osteocalcin result, data from 5-6 wells from a<br>
representative experiment were used to calculate the mean ± S.E.M. Graphs may<br>
be shown with data normalized to the maximum value for each parameter to allow<br>
simultaneous graphing of nodule counts, mineralized areas and osteocalcin.<br>
For each reported RT-PCR, RNase protection assay or Western blot<br>
analysis, data from triplicate samples of representative experiments, were used to<br>
determine the mean ± S.E.M. Graphs may be shown normalized to either day 0 or<br>
negative controls and expressed as fold-increase above control values.<br>
Statistical significance was evaluated using a one-way analysis of variance<br>
with post-hoc multiple comparison corrections of Bonferroni as appropriate.<br>
P. V. Huntsberger, "The Analysis of Variance", Elements of Statistical Variance,<br>
P. Billingsley (ed.), Allyn &amp; Bacon Inc., Boston, MA, 298-330 (1977) and<br>
SigmaStat, Jandel Scientific, Corte Madera, CA. Alpha levels for significance<br>
were defined as p 
EXAMPLE 15: Detection of Rat LIM Mineralization Protein bv Western Blot<br>
Analysis<br>
Polyclonal antibodies were prepared according to the methods of<br>
England, et al., Biochim.Biophys. Acta, 623, 171 (1980) and Timmer, et al., J.<br>
Biol. Chem., 268, 24863 (1993).<br>
HeLa cells were transfected with pCMV2/RLMP. Protein was harvested<br>
from the transfected cells according to the method of Hair, et al., Leukemia<br><br>
Research, 20, 1 (1996). Western Blot Analysis of native RLMP was performed as<br>
described by Towbin, et al., Proc. Natl. Acad. Sci. USA, 76:4350 (1979).<br>
EXAMPLE 16: Synthesis of the Rat LMP-Uniaue (RLMPU) derived Human<br>
PCR product<br>
Based on the sequence of the rat LMP-1 cDNA, forward and reverse PCR<br>
primers (SEQ. ID NOS: 15 and 16) were synthesized and a unique 223 base-pair<br>
sequence was PCR amplified from the rat LMP-1 cDNA. A similar PCR product<br>
was isolated from human MG63 osteosarcoma cell cDNA with the same PCR<br>
primers.<br>
RNA was harvested from MG63 osteosarcoma cells grown in T-75 flasks.<br>
Culture supernatant was removed by aspiration and the flasks were overlayed with<br>
3.0 ml of GIT solution per duplicate, swirled for 5-10 seconds, and the resulting<br>
solution was transferred to 1.5 ml eppendorf tubes (6 tubes with 0.6 ml/tube).<br>
RNA was purified by a slight modification of standard methods, for example, see<br>
Sambrook. et al.. Molecular Cloning: A Laboratory Manual, Chapter 7, page 19,<br>
Cold Spring Harbor Laboratory Press (1989) and Boden. et al.. Endocrinology,<br>
138, 2820-2828 (1997). Briefly, the 0.6 ml samples received 60 µl 2.0 M sodium<br>
acetate (pH 4.0), 550 ul water saturated phenol and 150 µl chloroform:isoamyl<br>
alcohol (49:1). After addition of those reagents, the samples were vortexed,<br>
centrifuged (10000 x g; 20 min; 4C) and the aqueous phase transferred to a fresh<br>
tube. Isopropanol (600 µl) was added and the RNA was precipitated overnight at<br>
-20°C. The samples were centrifuged (10000 x g; 20 minutes) and the supernatant<br>
was aspirated gently. The pellets were resuspended in 400 µl of DEPC-treated<br>
water, extracted once with phenolxhloroform (1:1), extracted with<br>
chloroform:isoamyl alcohol (24:1) and precipitated overnight at -20 °C in 40 ul<br>
sodium acetate (3.0 M; pH 5.2) and 1.0 ml absolute ethanol. After precipitation,<br>
the samples were centrifuged (10000 x g; 20 min), washed once with 70% ethanol,<br>
air dried for 5-10 minutes and resuspended in 20 µl of DEPC-treated water. RNA<br>
concentrations were derived from optical densities.<br><br>
Total RNA (5 µg in 10.5 µl total volume in DEPC-H2O) was heated at<br>
65 °C for 5 minutes, and then added to tubes containing 4 µl 5X MMLV-RT buffer,<br>
2 µl dNTPS, 2 µl dT17 primer (10 pmol/ml), 0.5 µl RNAsin (40 U/ml) and 1 µl<br>
MMLV-RT (200 units/µl). The reactions were incubated at 37°C for 1 hour.<br>
Afterward, the MMLV-RT was inactivated by heating at 95 °C for 5 minutes. The<br>
samples were diluted by addition of 80 µl water.<br>
Transcribed samples (5 µl) were subjected to polymerase-chain reaction<br>
using standard methodologies (50 µl total volume). Boden, et al., Endocrinology,<br>
138, 2820-2828 (1997); Ausubel, et al., "Quantitation of Rare DNAs by the<br>
Polymerase Chain Reaction", Current Protocols in Molecular Biology,<br>
Chapter 15.31-1, Wiley &amp; Sons, Trenton, NJ (1990). Briefly, samples were added<br>
to tubes containing water and appropriate amounts of PCR buffer (25 mM MgCl2,<br>
dNTPs, forward and reverse primers (for RLMPU; SEQ. ID NOS: 15 and 16),<br>
32P-dCTP, and DNA polymerase. Primers were designed to run consistently at 22<br>
cycles for radioactive band detection and 33 cycles for amplification of PCR<br>
product for use as a screening probe (94°C, 30 sec, 58°C, 30 sec; 72°C, 20 sec).<br>
Sequencing of the agarose gel-purified MG63 osteosarcoma-derived PCR<br>
product gave a sequence more than 95% homologous to the RLMPU PCR product.<br>
That sequence is designated HLMP unique region (HLMPU; SEQ. ID NO: 6).<br>
EXAMPLE 17: Screening of reverse-transcriptase-derived MG63 cDNA<br>
Screening was performed with PCR using specific primers (SEQ. ID<br>
NOS:16 and 17) as described in Example 7. A 717 base-pair MG63 PCR product<br>
was agarose gel purified and sequenced with the given primers (SEQ. ID NOs: 12,<br>
15, 16, 17, 18, 27 and 28). Sequences were confirmed a minimum of two times in<br>
both directions. The MG63 sequences were aligned against each other and then<br>
against the full-length rat LMP cDNA sequence to obtain a partial human LMP<br>
cDNA sequence (SEQ. ID NO: 7).<br><br>
EXAMPLE 18: Screening of a Human Heart cDNA Library<br>
Based on Northern blot experiments, it was determined that LMP-1 is<br>
expressed at different levels by several different tissues, including human heart<br>
muscle. A human heart cDNA library was therefore examined. The library was<br>
plated at 5 x 104 pfu/ml onto agar plates (LB bottom agar) and plates were grown<br>
overnight at 37°C. Filter membranes were overlaid onto the plates for two minutes.<br>
Afterward, the filters denatured, rinsed, dried, UV cross-linked and incubated in<br>
pre-hyridization buffer (2X PIPES [pH 6.5]; 5% formamide, 1 % SDS, 100 g/ml<br>
denatured salmon sperm DNA) for 2 h at 42 °C. A radio-labeled, LMP-unique,<br>
223 base-pair probe (32P, random primer labeling; SEQ ID NO: 6) was added and<br>
hybridized for 18 h at 42 °C. Following hybridization, the membranes were<br>
washed once at room temperature (10 min, 1 x SSC, 0.1 % SDS) and three times at<br>
55 °C (15 min, 0.1 x SSC, 0.1 % SDS). Double-positive plaque-purified heart<br>
library clones, identified by autoradiography, were rescued as lambda phagemids<br>
according to the manufacturers" protocols (Stratagene, La Jolla, CA).<br>
Restriction digests of positive clones yielded cDNA inserts of varying sizes.<br>
Inserts greater than 600 base-pairs in length were selected for initial screening by<br>
sequencing. Those inserts were sequenced by standard methods as described in<br>
Example 11.<br>
One clone, number 7, was also subjected to automated sequence analysis<br>
using primers corresponding to SEQ. ID NOS: 11-14, 16 and 27. The sequences<br>
obtained by these methods were routinely 97-100% homologous. Clone 7 (Partial<br>
Human LMP-1 cDNA from a heart library; SEQ. ID NO: 8) contained a sequence<br>
that was more than 87% homologous to the rat LMP cDNA sequence in the<br>
translated region.<br>
EXAMPLE 19: Determination of Full-Length Human LMP-1 cDNA<br>
Overlapping regions of the MG63 human osteosarcoma cell cDNA<br>
sequence and the human heart cDNA clone 7 sequence were used to align those<br>
two sequences and derive a complete human cDNA sequence of 1644 base-pairs.<br>
NALIGN, a program in the PCGENE software package, was used to align the two<br><br>
sequences. The overlapping regions of the two sequences constituted<br>
approximately 360 base-pairs having complete homology except for a single<br>
nucleotide substitution at nucleotide 672 in the MG63 cDNA (SEQ. ID NO: 7)<br>
with clone 7 having an "A" instead of a "G" at the corresponding nucleotide 516<br>
(SEQ. ID NO: 8).<br>
The two aligned sequences were joined using SEQIN, another subprogram<br>
of PCGENE, using the "G" substitution of the MG63 osteosarcoma cDNA clone.<br>
The resulting sequence is shown in SEQ. ID NO: 9. Alignment of the novel<br>
human-derived sequence with the rat LMP-1 cDNA was accomplished with<br>
NALIGN. The full-length human LMP-1 cDNA sequence (SEQ. ID NO: 9) is<br>
87.3% homologous to the translated portion of rat LMP-1 cDNA sequence.<br>
EXAMPLE 20: Determination of Amino Acid Sequence of Human LMP-1<br>
The putative amino acid sequence of human LMP-1 was determined with<br>
the PCGENE subprogram TRANSL. The open reading frame in SEQ. ID NO: 9<br>
encodes a protein comprising 457 amino acids (SEQ. ID NO: 10). Using the<br>
PCGENE subprogram Palign, the human LMP-1 amino acid sequence was found<br>
to be 94.1% homologous to the rat LMP-1 amino acid sequence.<br>
EXAMPLE 21: Determination of the 5 Prime Untranslated Region of the<br>
Human LMP cDNA<br>
MG63 5" cDNA was amplified by nested RT-PCR of MG63 total RNA<br>
using a 5" rapid amplification of cDNA ends (5" RACE) protocol. This method<br>
included first strand cDNA synthesis using a lock-docking oligo (dT) primer with<br>
two degenerate nucleotide positions at the 3" end (Chenchik, et al.,<br>
CLONTECHniques, X:5 (1995); Borson, et al., PC Methods Applic, 2, 144<br>
(1993)). Second-strand synthesis is performed according to the method of<br>
Gubler. et al.. Gene, 2, 263 (1983), with a cocktail of Escherichia coli DNA<br>
polymerase 1, RNase H, and E. coli DNA ligase. After creation of blunt ends with<br>
T4 DNA polymerase, double-stranded cDNA was ligated to the fragment (5"-<br>
CTAATACGACTCACTATAGGGCTCGAGCGGCCGCCCGGGCAGGT-3")<br><br>
(SEQ. ID NO: 19). Prior to RACE, the adaptor-ligated cDNA was diluted to a<br>
concentration suitable for Marathon RACE reactions (1:50). Adaptor-ligated<br>
double-stranded cDNA was then ready to be specifically cloned.<br>
First-round PCR was performed with the adaptor-specific oligonucleotide,<br>
5"-CCATCCTAATACGACTCACTATAGGGC- 3" (API) (SEQ. ID NO: 20) as<br>
sense primer and a Gene Specific Primer (GSP) from the unique region described<br>
in Example 16 (HLMPU). The second round of PCR was performed using a<br>
nested primers GSP1-HLMPU (antisense/reverse primer) (SEQ. ID NO: 23) and<br>
GSP2-HLMPUF (SEQ. ID NO: 24) (see Example 16; sense/forward primer). PCR<br>
was performed using a commercial kit (Advantage cDNA PCR core kit;<br>
CloneTech Laboratories Inc., Palo Alto, CA) that utilizes an antibody-mediated,<br>
but otherwise standard, hot-start protocol. PCR conditions for MG63 cDNA<br>
included an initial hot-start denaturation (94°C, 60 sec) followed by: 94°C, 30 sec;<br>
60°C, 30 sec; 68°C, 4 min; 30 cycles. The flrstround PCR product was<br>
approximately 750 base-pairs in length whereas the nested PCR product was<br>
approximately 230 base-pairs. The first-round PCR product was cloned into<br>
linearized pCR 2.1 vector (3.9 Kb). The inserts were sequenced in both directions<br>
using M13 Forward and Reverse primers (SEQ. ID NO: 11; SEQ. ID NO: 12).<br>
EXAMPLE 22: Determination of Full-length Human LMP-1 cDNA with<br>
5 Prime UTR<br>
Overlapping MG63 human osteosarcoma cell cDNA 5"-UTR sequence<br>
(SEQ. ID NO: 21), MG63 717 base-pair sequence (Example 17; SEQ. ID NO: 8)<br>
and human heart cDNA clone 7 sequence (Example 18) were aligned to derive a<br>
novel human cDNA sequence of 1704 base-pairs (SEQ. ID NO: 22). The<br>
alignment was accomplished with NALIGN, (both PCGENE and Omiga 1.0;<br>
Intelligenetics). Over-lapping sequences constituted nearly the entire 717 base-pair<br>
region (Example 17) with 100% homology. Joining of the aligned sequences was<br>
accomplished with SEQIN.<br><br>
EXAMPLE 23: Construction of LIM Protein Expression Vector<br>
The construction of pHIS-5ATG LMP-ls expression vector was carried out<br>
with the sequences described in Examples 17 and 18. The 717 base-pair clone<br>
(Example 17; SEQ. ID NO: 7) was digested with Clal and EcoRV. A small<br>
fragment (-r250 base-pairs) was gel purified. Clone 7 (Example 18; SEQ. ID<br>
NO: 8) was digested with ClaI and XbaI and a 1400 base-pair fragment was gel<br>
purified. The isolated 250 base-pair and 1400 base-pair restriction fragments were<br>
ligated to form a fragment of ~1650 base-pairs.<br>
Due to the single nucleotide substitution in Clone 7 (relative to the 717<br>
base-pair PCR sequence and the original rat sequence) a stop codon at translated<br>
base-pair 672 resulted. Because of this stop codon, a truncated (short) protein was<br>
encoded, hence the name LMP-ls. This was the construct used in the expression<br>
vector (SEQ. ID NO: 32). The full length cDNA sequence with 5" UTR (SEQ. ID<br>
NO: 33) was created by alignment of SEQ. ID NO: 32 with the 5" RACE sequence<br>
(SEQ. ID NO: 21). The amino acid sequence of LMP-ls (SEQ. ID NO: 34) was<br>
then deduced as a 223 amino acid protein and confirmed by Western blot (as in<br>
Example 15) to run at the predicted molecular weight of -23.7 kD.<br>
The pHis-ATG vector (InVitrogen, Carlsbad, CA) was digested with<br>
EcoRV and Xbal. The vector was recovered and the 650 base-pair restriction<br>
fragment was then ligated into the linearized pHis-ATG. The ligated product was<br>
cloned and amplified. The pHis-ATG-LMP-ls Expression vector, also designated<br>
pHIS-A with insert HLMP-ls, was purified by standard methods.<br>
EXAMPLE 24: Induction of Bone Nodule Formation and Mineralization In vitro<br>
with LMP Expression Vector<br>
Rat Calvarial cells were isolated and grown in secondary culture according<br>
to Example 1. Cultures were either unstimulated or stimulated with glucocorticoid<br>
(GC) as described in Example 1. A modification of the Superfect Reagent<br>
(Qiagen, Valencia, CA) transfection protocol was used to transfect 3 µg/well of<br>
each vector into secondary rat calvarial osteoblast cultures according to<br>
Example 25.<br><br>
Mineralized nodules were visualized by Von Kossa staining, as described in<br>
Example 3. Human LMP-ls gene product over expression alone induced bone<br>
nodule formation (~203 nodules/well) in vitro. Levels of nodules were<br>
approximately 50% of those induced by the GC positive control<br>
(~412 nodules/well). Other positive controls included the pHisA-LMP-Rat<br>
expression vector (~152 nodules/well) and the pCMV2/LMP-Rat-Fwd Expression<br>
vector (~206 nodules/well), whereas the negative controls included the<br>
pCMV2/LMP-Rat-Rev. expression vector (~2 nodules/well) and untreated (NT)<br>
plates (~4 nodules/well). These data demonstrate that the human cDNA was at<br>
least as osteoinductive as the rat cDNA. The effect was less than that observed<br>
with GC stimulation, most likely due to sub-optimal doses of Expression vector.<br>
EXAMPLE 25: LMP-Induced Cell Differentiation In Vitro and In Vivo<br>
The rat LMP cDNA in clone 10-4 (see Example 12) was excised from the<br>
vector by double-digesting the clone with NotI and Apal overnight at 37°C.<br>
Vector pCMV2 MCS (InVitrogen, Carlsbad, CA) was digested with the same<br>
restriction enzymes. Both the linear cDNA fragment from clone 10-4 and pCMV2<br>
were gel purified, extracted and ligated with T4 ligase. The ligated DNA was gel<br>
purified, extracted and used to transform E. coli JM109 cells for amplification.<br>
Positive agar colonies were picked, digested with NotI and Apal and the restriction<br>
digests were examined by gel electrophoresis. Stock cultures were prepared of<br>
positive clones.<br>
A reverse vector was prepared in analogous fashion except that the<br>
restriction enzymes used were Xbal and Hindlll. Because these restriction<br>
enzymes were used, the LMP cDNA fragment from clone 10-4 was inserted into<br>
pRc/CMV2 in the reverse (that is, non-translatable) orientation. The recombinant<br>
vector produced is designated pCMV2/RLMP.<br>
An appropriate volume of pCMV10-4 (60 nM final concentration is optimal<br>
[3µg]; for this experiment a range of 0-600 nM/well [0-30 µg/well] final<br>
concentration is preferred) was resuspended in Minimal Eagle Media (MEM) to<br>
450 µl final volume and vortexed for 10 seconds. Superfect was added (7.5 µl/ml<br><br>
final solution), the solution was vortexed for 10 seconds and then incubated at<br>
room temperature for 10 minutes. Following this incubation, MEM supplemented<br>
with 10% FBS (1 ml/well; 6 ml/plate) was added and mixed by pipetting.<br>
The resulting solution was then promptly pipetted (1 ml/well) onto washed<br>
ROB cultures. The cultures were incubated for 2 hours at 37 °C in a humidified<br>
atmosphere containing 5% CO2. Afterward, the cells were gently washed once<br>
with sterile PBS and the appropriate normal incubation medium was added.<br>
Results demonstrated significant bone nodule formation in all rat cell<br>
cultures which were induced with pCMV10-4. For example, pCMV10-4<br>
transfected cells produced 429 nodules/well. Positive control cultures, which were<br>
exposed to Trm, produced 460 nodules/well. In contrast, negative controls, which<br>
received no treatment, produced 1 nodule/well. Similarly, when cultures were<br>
transfected with pCMV10-4 (reverse), no nodules were observed.<br>
For demonstrating de novo bone formation in vivo, marrow was aspirated<br>
from the hind limbs of 4-5 week old normal rats (rnu/+; heterozygous for recessive<br>
athymic condition). The aspirated marrow cells were washed in alpha MEM,<br>
centrifuged, and RBCs were lysed by resuspending the pellet in 0.83% NH4Cl in<br>
10 mM Tris (pH 7.4). The remaining marrow cells were washed 3x with MEM and<br>
transfected for 2 hours with 9 µg of pCMV-LMP-ls (forward or reverse<br>
orientation) per 3 x 106 cells. The transfected cells were then washed 2X with<br>
MEM and resuspended at a concentration of 3 x 107 cells/ml.<br>
The cell suspension (100 µl) was applied via sterile pipette to a sterile 2x5<br>
mm type I bovine collagen disc (Sulzer Orthopaedics, Wheat Ridge, CO). The<br>
discs were surgically implanted subcutaneously on the skull, chest, abdomen or<br>
dorsal spine of 4-5 week old athymic rats (rnu/rnu). The animals were scarified at<br>
3-4 weeks, at which time the discs or surgical areas were excised and fixed in 70%<br>
ethanol. The fixed specimens were analyzed by radiography and undecalcified<br>
histologic examination was performed on 5 urn thick sections stained with Goldner<br>
Trichrome. Experiments were also performed using devitalized (guanidine<br>
extracted) demineralized bone matrix (Osteotech, Shrewsbury, NJ) in place of<br>
collagen discs.<br><br>
Radiography revealed a high level of mineralized bone formation that<br>
conformed to the form of the original collagen disc containing LMP-ls transfected<br>
marrow cells. No mineralized bone formation was observed in the negative control<br>
(cells transfected with a reverse-oriented version of the LMP-ls cDNA that did not<br>
code for a translated protein), and absorption of the carrier appeared to be well<br>
underway.<br>
Histology revealed new bone trabeculae lined with osteoblasts in the<br>
LMP-ls transfected implants. No bone was seen along with partial resorption of<br>
the carrier in the negative controls.<br>
Radiography of a further experiment in which 18 sets (9 negative control<br>
pCMV-LMP-REV &amp; 9 experimental pCMV-LMP-ls) of implants were added to<br>
sites alternating between lumbar and thoracic spine in athymic rats demonstrated<br>
0/9 negative control implants exhibiting bone formation (spine fusion) between<br>
vertebrae. All nine of the pCMV-LMP-ls treated implants exhibited solid bone<br>
fusions between vertebrae.<br>
EXAMPLE 26: The Synthesis of pHIS-5" ATG LMP-ls Expression Vector from<br>
the sequences Demonstrated in Examples 2 and 3<br>
The 717 base-pair clone (Example 17) was digested with Clal and EcoRV<br>
(New England Biologicals, city, MA). A small fragment (~250 base pairs) was gel<br>
purified. Clone No. 7 (Example 18) was digested with Clal and Xbal. A 1400<br>
base-pair fragment was gel purified from that digest. The isolated 250 base-pair<br>
and 1400 base-pair cDNA fragments were ligated by standard methods to form a<br>
fragment of ~1650 bp. The pHis-A vector (InVitrogen) was digested with EcoRV<br>
and Xbal. The linearized vector was recovered and ligated to the chimeric 1650<br>
base-pair cDNA fragment. The ligated product was cloned and amplified by<br>
standard methods, and the phis-A-5" ATG LMP-ls expression vector, also<br>
denominated as the vector pHis-A with insert HLMP-ls, was deposited at the<br>
ATCC as previously described.<br><br>
EXAMPLE 27: The Induction of Bone Nodule Formation and Mineralization In<br>
Vitro With pHis-5" ATG LMP-ls Expression Vector<br>
Rat calvarial cells were isolated and grown in secondary culture according<br>
to Example 1. Cultures were either unstimulated or stimulated with glucocorticoid<br>
(GC) according to Example 1. The cultures were transfected with 3 µg of<br>
recombinant pHis-A vector DNA/well as described in Example 25. Mineralized<br>
nodules were visualized by Von Kossa staining according to Example 3.<br>
Human LMP-ls gene product overexpression alone (i.e., without GC<br>
stimulation) induced significant bone nodule formation (~203 nodules/well) in<br>
vitro. This is approximately 50% of the amount of nodules produced by cells lo<br>
exposed to the GC positive control (~412 nodules/well). Similar results were<br>
obtained with cultures transfected with pHisA-LMP-Rat Expression vector (~152<br>
nodules/well) and pCMV2/LMP-Rat-Fwd (~206 nodules/well). In contrast, the<br>
negative control pCMV2/LMP-Rat-Rev yielded (~2 nodules/well), while<br>
approximately 4 nodules/well were seen in the untreated plates. These data<br>
demonstrate that the human LMP-1 cDNA was at least as osteoinductive as the rat<br>
LMP-1 cDNA in this model system. The effect in this experiment was less than<br>
that observed with GC stimulation; but in some the effect was comparable.<br>
EXAMPLE 28: LMP Induces Secretion of a Soluble Osteoinductive Factor<br>
Overexpression of RLMP-1 or HLMP-ls in rat calvarial osteoblast cultures<br>
as described in Example 24 resulted in significantly greater nodule formation than<br>
was observed in the negative control. To study the mechanism of action of LIM<br>
mineralization protein conditioned medium was harvested at different time points,<br>
concentrated to 10 X, sterile filtered, diluted to its original concentration in<br>
medium containing fresh serum, and applied for four days to untransfected cells.<br>
Conditioned media harvested from cells transfected with RLMP-1 or<br>
HLMP-ls at day 4 was approximately as effective in inducing nodule formation as<br>
direct overexpression of RLMP-1 in transfected cells. Conditioned media from<br>
cells transfected with RLMP-1 or HLMP-1 in the reverse orientation had no<br>
apparent effect on nodule formation. Nor did.conditioned media harvested from<br><br>
LMP-1 transfected cultures before day 4 induce nodule formation. These data<br>
suggest that expression of LMP-1 caused the synthesis and/or secretion of a soluble<br>
factor, which did not appear in culture medium in effective amounts until 4 days<br>
post transfection.<br>
Since overexpression of rLMP-1 resulted in the secretion of an<br>
osteoinductive factor into the medium, Western blot analysis was used to<br>
determine if LMP-1 protein was present in the medium. The presence of RLMP-1<br>
protein was assessed using antibody specific for LMP-1 (QDPDEE) and detected<br>
by conventional means. LMP-1 protein was found only in the cell layer of the<br>
culture and not detected in the medium.<br>
Partial purification of the osteoinductive soluble factor was accomplished<br>
by standard 25% and 100% ammonium sulfate cuts followed by DE-52 anion<br>
exchange batch chromatography (100 tnM or 500 mM NAC1). All activity was<br>
observed in the high ammonium sulfate, high NaCl fractions. Such localization is<br>
consistent with the possibility of a single factor being responsible for conditioning<br>
the medium.<br>
EXAMPLE 29: Gene Therapy In Lumbar Spine Fusion Mediated by Low Dose<br>
Adenovirus<br>
This study determined the optimal dose of adenoviral delivery of the LMP-<br>
1 cDNA (SEQ. ID NO: 2) to promote spine fusion in normal, that is, immune<br>
competent, rabbits.<br>
A replication-deficient human recombinant adenovirus was constructed<br>
with the LMP-1 cDNA (SEQ. ID NO: 2) driven by a CMV promoter using the<br>
Adeno-Quest™ Kit (Quantum Biotechnologies, Inc., Montreal). A commercially<br>
available (Quantum Biotechnologies, Inc., Montreal) recombinant adenovirus<br>
containing the beta-galactosidase gene was used as a control.<br>
Initially, an in vitro dose response experiment was performed to determine<br>
the optimal concentration of adenoyirus-delivered LMP-1 ("AdV-LMP-1") to<br>
induce bone differentiation in rat calvarial osteoblast cultures using a 60-minute<br>
transduction with a multiplicity of infection ("MOI") of 0.025, 0.25, 2.5, or 25<br><br>
plaque-forming units (pfu) of virus per cell. Positive control cultures were<br>
differentiated by a 7-day exposure to 109 M glucocorticoid ("GC"). Negative<br>
control cultures were left untreated. On day 14, the number of mineralized bone<br>
nodules was counted after von Kossa staining of the cultures, and the level of<br>
osteocalcin secreted into the medium (pmol/mL) was measured by<br>
radioimmunoassay (mean ± SEM).<br>
The results of this experiment are shown in Table 1. Essentially no<br>
spontaneous nodules formed in the untreated negative control cultures. The data<br>
show that a MOI equal to 0.25 pfu/cell is most effective for osteoinducing bone<br>
nodules, achieving a level comparable to the positive control (GC). Lower and<br>
higher doses of adenovirus were less effective.<br>
TABLE I<br><br>
In vivo experiments were then performed to determine if the optimal in<br>
vitro dose was capable of promoting intertransverse process spine fusions in<br>
skeletally mature New Zealand white rabbits. Nine rabbits were anesthetized and<br>
3 cc of bone marrow was aspirated from the distal femur through the intercondylar<br>
notch using an 18 gauge needle. The buffy coat was then isolated, a 10-minute<br>
transduction with AdV-LMP-1 was performed, and the cells were returned to the<br>
operating room for implantation. Single level posterolateral lumbar spine<br>
arthrodesis was performed with decortication of transverse processes and insertion<br>
of carrier (either rabbit devitalized bone matrix or a collagen sponge) containing 8-<br>
15 million autologous nucleated buffy coat cells transduced with either AdV-LMP-<br>
1 (MOI = 0.4) or AdV-BGal (MOI = 0.4). Rabbits were euthanized after 5 weeks<br><br>
and spine fusions were assessed by manual palpation, plain x-rays, CT scans, and<br>
undecalcified histology.<br>
The spine fusion sites that received AdV-LMP-1 induced solid, continuous<br>
spine fusion masses in all nine rabbits. In contrast, the sites receiving AdV-BGal,<br>
or a lower dose of AdV-LMP-1 (MOI = 0.04) made little or no bone and resulted<br>
in spine fusion at a rate comparable to the carrier alone (
were consistent as evaluated by manual palpation, CT scan, and histology. Plain<br>
radiographs, however, sometimes overestimated the amount of bone that was<br>
present, especially in the control sites. LMP-1 cDNA delivery and bone induction<br>
was successful with both of the carrier materials tested. There was no evidence of<br>
systemic or local immune response to the adenovirus vector.<br>
These data demonstrate consistent bone induction in a previously validated<br>
rabbit spine fusion model which is quite challenging. Furthermore, the protocol of<br>
using autogenous bone marrow cells with intraoperative ex vivo gene transduction<br>
(10 minutes) is a more clinically feasible procedure than other methods that call for<br>
overnight transduction or cell expansion for weeks in culture. In addition, the most<br>
effective dose of recombinant adenovirus (MOI=0.25) was substantially lower than<br>
doses reported in other gene therapy applications (MOI 40-500). We believe this is<br>
due to the fact that LMP-1 is an intracellular signaling molecule and may have<br>
powerful signal amplification cascades. Moreover, the observation that the same<br>
concentration of AdV-LMP-1 that induced bone in cell culture was effective in<br>
vivo was also surprising given the usual required increase in dose of other growth<br>
factors when translating from cell culture to animal experiments. Taken together,<br>
these observations indicate that local gene therapy using adenovirus to deliver the<br>
LMP-1 cDNA is possible and the low dose required will likely minimize the<br>
negative effects of immune response to the adenovirus vector.<br>
EXAMPLE 30: Use of Peripheral Venous Blood Nucleated Cells rBuffv Coat) for<br>
Gene Therapy With LMP-1 cDNA To Make Bone<br>
In four rabbits we performed spine fusion surgery as above (Example 29)<br>
except the transduced cells were the buffy coat from venous blood rather than bone<br><br>
marrow. These cells were transfected with Adeno-LMP or pHIS-LMP plasmid and<br>
had equivalent successful results as when bone marrow cells were used. This<br>
discovery of using ordinary venous blood cells for gene delivery makes gene<br>
therapy more feasible clinically since it avoids painful marrow harvest under<br>
general anesthesia and yields two times more cells per mL of starting material.<br>
EXAMPLE 31: Isolation of Human LMP-1 Splice Variants<br>
Intron/Exon mRNA transcript splice variants are a relatively common<br>
regulatory mechanism in signal-transduction and cellular/tissue development.<br>
Splice variants of various genes have been shown to alter protein-protein, protein-<br>
DNA, protein-RNA, and protein-substrate interactions. Splice variants may also<br>
control tissue specificity for gene expression allowing different forms (and<br>
therefore functions) to be expressed in various tissues. Splice variants are a<br>
common regulatory phenomenon in cells. It is possible that the LMP splice<br>
variants may result in effects in other tissues such as nerve regeneration, muscle<br>
regeneration, or development of other tissues.<br>
To screen a human heart cDNA library for splice variants of the HLMP-1<br>
sequence, a pair of PCR primer corresponding to sections of SEQ. ID NO: 22 was<br>
prepared. The forward PCR primer, which was synthesized using standard<br>
techniques, corresponds to nucleotides 35-54 of SEQ. ID NO: 22. It has the<br>
following sequence:<br>
5" GAGCCGGCATCATGGATTCC 31 (SEQ. ID NO: 35)<br>
The reverse PCR primer, which is the reverse complement of nucleotides<br>
820-839 in SEQ. ID NO: 22, has the following sequence:<br>
5" GCTGCCTGCACAATGGAGGT 3" (SEQ. ID NO: 36)<br>
The forward and reverse PCR primers were used to screen human heart<br>
cDNA (ClonTech, Cat No. 7404-1) for sequences similar to HLMP-1 by standard<br><br>
techniques, using a cycling protocol of 94°C for 30 seconds, 64°C for 30 seconds,<br>
and 72°C for 1 minute, repeated 30 times and followed by a 10 minute incubation<br>
at 72°C. The amplification cDNA sequences were gel-purified and submitted to<br>
the Emory DNA Sequence Core Facility for sequencing. The clones were<br>
sequenced using standard techniques and the sequences were examined with<br>
PCGENE (intelligenetics; Programs SEQUIN and NALIGN) to determine<br>
homology to SEQ. ID NO: 22. Two homologous nucleotide sequences with<br>
putative alternative splice sites compared to SEQ. ID NO: 22 were then translated<br>
to their respective protein products with Intelligenetic"s program TRANSL.<br>
One of these two novel human cDNA sequences (SEQ. ID NO: 37)<br>
comprises 1456 bp:<br><br><br>
CATTTCTGGG TAGGGCTGGC AATGGTTGCC TTAACCCTGG CTCCTGGCCC GAGCCTGGGC 14 40<br>
TCCCGGGCCC TGCCCA 1456<br>
Reading frame shifts caused by the deletion of a 119 bp fragment (between<br>
X) and the addition of a 17 bp fragment (underlined) results in a truncated gene<br>
product having the following derived amino acid sequence (SEQ. ID NO: 38):<br><br><br><br>
This 423 amino acid protein demonstrates 100% homology to the protein<br>
shown in SEQ. ID NO. 10, except for the sequence in the highlighted area (amino<br>
acids 94-99), which are due to the nucleotide changes depicted above.<br>
The second novel human heart cDNA sequence (SEQ. ID NO: 39)<br>
comprises 1575 bp:<br><br><br><br>
Reading frame shifts caused by the addition of a 17 bp fragment (bolded,<br>
italicized and underlined) results in an early translation stop codon at position 565-<br>
567 (underlined).<br>
The derived amino acid sequence (SEQ. ID NO: 40) consists of 153 amino<br>
acids:<br><br>
This protein demonstrates 100% homology to SEQ. ID NO: 10 until amino<br>
acid 94, where the addition of the 17 bp fragment depicted in the nucleotide<br>
sequence results in a frame shift. Over amino acids 94-153, the protein is not<br>
homologous to SEQ. ID NO: 10. Amino acids 154-457 in SEQ. ID NO: 10 are not<br>
present due to the early stop codon depicted in nucleotide sequence.<br><br>
EXAMPLE 32: Genomic HLMP-1 Nucleotide Sequence<br>
Applicants have identified the genomic DNA sequence encoding HLMP-1,<br>
including putative regulatory elements associated with HLMP-1 expression. The<br>
entire genomic sequence is shown in SEQ. ID. NO: 41. This sequence was derived<br>
from AC023788 (clone RP11-564G9), Genome Sequencing Center, Washington<br>
University School of Medicine, St. Louis, MO.<br>
The putative promoter region for HLMP-1 spans nucleotides 2,660-8,733 in<br>
SEQ. ID NO: 41. This region comprises, among other things, at least ten potential<br>
glucocorticoid response elements ("GREs") (nucleotides 6148-6153, 6226-6231,<br>
6247-6252, 6336-6341, 6510-6515, 6552-6557, 6727-6732, 6752-6757, 7738-<br>
7743, and 8255-8260), twelve potential Sma-2 homologues to Mothers against<br>
Drosophilla decapentaplegic ("SMAD") binding element sites (nucleotides 3569-<br>
3575, 4552-4558, 4582-4588, 5226-5232, 6228-6234, 6649-6655, 6725-6731,<br>
6930-6936, 7379-7384, 7738-7742, 8073-8079, and 8378-8384), and three TATA<br>
boxes (nucleotides 5910-5913, 6932-6935, and 7380-7383). The three TATA<br>
boxes, all of the GREs, and eight of the SMAD binding elements ("SBEs") are<br>
grouped in the region spanning nucleotides 5,841-8,733 in SEQ. ID NO: 41. These<br>
regulatory elements can be used, for example, to regulate expression of exogenous<br>
nucleotide sequences encoding proteins involved in the process of bone formation.<br>
This would permit systemic administration of therapeutic factors or genes relating<br>
to bone formation and repair, as well as factors or genes associated with tissue<br>
differentiation and development.<br>
In addition to the putative regulatory elements, 13 exons corresponding<br>
to the nucleotide sequence encoding HLMP-1 have been identified. These exons<br>
span the following nucleotides in SEQ. ID NO: 41:<br><br><br><br>
In HLMP-2 there is another exon (Exon 5A), which spans nucleotides<br>
14887-14904.<br>
EXAMPLE 33: Expression of HLMP-1 in Intervertebral Disc Cells<br>
LIM mineralization protein-1 (LMP-1) is an intracellular protein that can<br>
direct cellular differentiation in osseous and non-osseous tissues. This example<br>
demonstrates that expressing human LMP-1 ("HLMP-1") in intervertebral disc<br>
cells increases proteoglycan synthesis and promotes a more chondrocytic<br>
phenotype. In addition, the effect of HLMP-1 expression on cellular gene<br>
expression was demonstrated by measuring Aggrecan and BMP-2 gene expression.<br>
Lumbar intervertebral disc cells were harvested from Sprague-Dawley rats by<br>
gentle enzymatic digestion and cultured in monolayer in DMEM/F12<br>
supplemented with 10% FBS. These cells were then split into 6 well plates at<br>
approximately 200,000 cells per well and cultured for about 6 days until the cells<br>
reached approximately 300,000 cells per well. The culture media was changed to 1<br>
% FBS DMEM/F12 and this was considered Day 0.<br>
Replication deficient Type 5 adenovirus comprising a HLMP-1 cDNA<br>
operably linked to a cytomegalovirus ("CMV") promoter has been previously<br>
described, for example, in U.S. Patent No. 6,300,127. The negative control<br>
adenovirus was identical except the HLMP-1 cDNA was replaced by LacZ cDNA.<br>
For a positive control, uninfected cultures were incubated in the continuous<br>
presence of BMP-2 at a concentration of 100 nanograms/milliliter.<br><br>
On Day 0, the cultures were infected with adenovirus for 30 minutes at<br>
37°C in 300 microliters of media containing 1 % FBS. Fluorescence Activated<br>
Cell Sorter ("FACS") analysis of cells treated with adenovirus containing the green<br>
fluorescent protein ("GFP") gene ("AdGFP") was performed to determine the<br>
optimal dose range for expression of transgene. The cells were treated with<br>
adenovirus containing the human LMP-1 cDNA (AdHLMP-1) (at MOIs of 0, 100,<br>
300,1000, or 3000) or with adenovirus containing the LacZ marker gene (AdLacZ<br>
MOI of 1000) (negative control). The culture media was changed at day 3 and day<br>
6 after infection.<br>
Proteoglycan production was estimated by measuring the sulfated<br>
glycosaminoglycans (sGAG) present in the culture media (at day 0, 3, and 6) using<br>
a di-methyl-methylene blue ("DMMB") calorimetric assay.<br>
For quantification of Aggrecan and BMP-2 mRNA, cells were harvested at<br>
day 6 and the mRNA extracted by the Trizol technique. The mRNA was converted<br>
to cDNA using reverse-transcriptase and used for real-time PCR, which allowed<br>
the relative abundance of Aggrecan and BMP-2 message to be determined. Real<br>
time primers were designed and tested for Aggrecan and BMP-2 in previous<br>
experiments. The Cybergreen technique was used. Standardization curves were<br>
used to quantitate mRNA abundance.<br>
For transfected cells, cell morphology was documented with a light<br>
microscope. Cells became more rounded with AdHLMP-1 (MOI 1000) treatment,<br>
but not with AdLacZ treatment. AdLacZ infection did not significantly change cell<br>
morphology.<br>
FACS analysis of rat disc cells infected with ADGFP at MOI of 1000<br>
showed the highest percentage cells infected (45%).<br>
There was a dose dependent increase between sGAG production and<br>
AdhLMP-1 MOI. These data are seen in FIG. 1, which shows the production of<br>
sGAG after over-expressing HLMP-1 at different MOIs in rat disc cells in<br>
monolayer cultures. The results have been normalized to day 0 untreated cells.<br>
Error bars represent the standard error of the mean. As shown in FIG. 1, the sGAG<br>
production observed at day 3 was relatively minor, indicating a lag time between<br><br>
transfection and cellular production of GAG. Treatment with AdLacZ did not<br>
significantly change the sGAG production. As also shown in FIG. 1, the optimal<br>
dose of AdhLMP-1 was at a MOI of 1000, resulting in a 260% enhancement of<br>
sGAG production over the untreated controls at day 6. Higher or lower doses of<br>
AdhLMP-1 lead to a diminished response.<br>
The effect of AdhLMP-1 dosage (MOI) on sGAG production is further<br>
illustrated in FIG. 2. FIG. 2 is a chart showing rat disc sGAG levels at day 6 after<br>
treatment with AdhLMP-1 at different MOIs. As can be seen from FIG. 2, the<br>
optimal dose of AdhLMP-1 was at a MOI of 1000.<br>
Aggrecan and BMP-2 mRNA production is seen in FIG. 3. This figure<br>
demonstrates the increase in Aggrecan and BMP-2 mRNA after over-expression of<br>
HLMP-1. Real-time PCR of mRNA extracted from rat disc cells at day 6 was<br>
performed comparing the no-treatment ("NT") cells with cells treated with<br>
ADhLMP-1 at a MOI of 250. The data in FIG. 3 are represented as a percentage<br>
increase over the untreated sample. As illustrated in FIG. 3, a significant increase<br>
in Aggrecan and BMP-2 mRNA was noted following AdhLMP-1 treatment. The<br>
increase in BMP-2 expression suggests that BMP-2 is a down-stream gene that<br>
mediates HLMP-1 stimulation of proteoglycan synthesis.<br>
These data demonstrate that transfection with AdhLMP-1 is effective in<br>
increasing proteoglycan synthesis of intervertebral disc cells. The dose of virus<br>
leading to the highest transgene expression (MOI 1000) also leads to the highest<br>
induction of sGAG, suggesting a correlation between HLMP-1 expression and<br>
sGAG induction. These data indicate that HLMP-1 gene therapy is a method of<br>
increasing proteoglycan synthesis in the intervertebral disc, and that HLMP-1 is a<br>
agent for treating disc disease.<br>
FIG. 4A is a chart showing HLMP-1 mRNA expression 12 hours after<br>
infection with Ad-hLMP-1 at different MOIs. In FIG. 4A, exogenous LMP-1<br>
expression was induced with different doses (MOI) of the Ad-hLMP-1 virus and<br>
quantitated with realtime PCR. The data is normalized to HLMP-1 mRNA levels<br>
from Ad-LMP-1 MOI 5 for comparison purposes. No HLMP-1 was detected in<br>
negative control groups, the no-treatment ("NT") or Ad-LacZ treatment ("LacZ").<br><br>
HLMP-1 mRNA levels in a dose dependent fashion to reach a plateau of<br>
approximately 8 fold with a MOI of 25 and 50.<br>
FIG. 4B is a chart showing the production of sGAG in medium from 3 to 6<br>
days after infection. DMMB assay was used to quantitate total sGAG production<br>
between days 3 to 6 after infection. The data in FIG. 4B is normalized to the<br>
control (i.e., no treatment) group. As can be seen from FIG. 4B, there was a dose<br>
dependent increase in sGAG. with the peak of approximately three fold increase<br>
above control reached with a MOI of 25 and 50. The negative control, Ad-LacZ at<br>
a MOI of 25, lead to no increase in sGAG. In FIG. 4B, each result is expressed as<br>
mean with SD for three samples.<br>
FIG. 5 is a chart showing time course changes of the production of sGAG.<br>
As can be seen from FIG. 5, on day 3 sGAG production increased significantly at a<br>
MOI of 25 and 50. On day 6 there was a dose dependent increase in sGAG<br>
production in response to AdLMP-1. The plateau level of sGAG increase was<br>
achieved at a MOI of 25. As can also be seen from FIG. 5, treatment with AdLacZ<br>
("LacZ") did not significantly change the sGAG production. Each result is<br>
expressed as mean with SD for six to nine samples. In FIG. 5, "**" indicates data<br>
points for which the P value is 
FIGS. 6A and 6B are charts showing gene response to LMP-1 over-<br>
expression in rat annulus fibrosus cells for aggrecan and BMP-2, respectively.<br>
Quantitative realtime PCR was performed on day 3 after infection with Ad-LMP-1<br>
("LMP-1") at a MOI of 25. As can be seen from FIGS. 6A and 6B, the gene<br>
expression of aggrecan and BMP-2 increased significantly after infection with Ad-<br>
LMP-1 compared to the untreated control ("NT"). Further, treatment with AdLacZ<br>
("LacZ") at a MOI of 25 did not significantly change the gene expression of either<br>
aggrecan or BMP-2 compared to the untreated control. In FIGS. 6A and 6B, each<br>
result is expressed as mean with SD for six samples. In FIGS. 6A and 6B, "**"<br>
indicates data points for which the P value is P 
FIG. 7 is a graph showing the time course of HLMP-1 mRNA levels in rat<br>
annulus fibrosus cells after infection with AdLMP-1 at a MOI of 25. The data is<br>
expressed as a fold increase above a MOI of 5 of AdLMP-1 after standardization<br><br>
using 18S and replication coefficient of over-expression LMP-1 primer. As can be<br>
seen from FIG 7, HLMP-1 mRNA was upregulated significantly as early as 12<br>
hours after infection. Further, there was a marked increase of expression levels<br>
between day 1 and day 3. Each result in FIG 7 is expressed as mean with SD for<br>
six samples.<br>
FIG 8 is a chart showing changes in mRNA levels of BMPs and aggrecan<br>
in response to HLMP-1 over-expression. The mRNA levels of BMP-2, BMP-4,<br>
BMP-6, BMP7, and aggrecan were determined with realtime-PCR at different time<br>
points after infection with Ad-hLMP-1 at a MOI of 25. As can be seen from<br>
FIG 8, BMP-2 mRNA was upregulated significantly as early as 12 hours after<br>
infection with AdLMP-1. On the other hand, Aggrecan mRNA was not<br>
upregulated until 3 day after infection. Each result is expressed as mean with SD<br>
for six samples. In FIG. 8, "**" indicates data points for which the P value is 
0.01 for infection with AdLMP-1 versus an untreated control.<br>
FIG 9 is a graph showing the time course of sGAG production<br>
enhancement in response to HLMP-1 expression. For the data in FIG 9, rat<br>
annulus cells were infected with Ad-hLMP-1 at a MOI of 25. The media was<br>
changed every three days after infection and assayed for sGAG with the DMMB<br>
assay. This data shows that sGAG production reaches a plateau at day 6 and is<br>
substantially maintained at day 9.<br>
FIG. 10 is a chart showing the effect of noggin (a BMP antagonist) on<br>
LMP-1 mediated increase in sGAG production. As seen in FIG. 10, infection of rat<br>
annulus cells with Ad-LMP-1 at a MOI of 25 led to a three fold increase in sGAG<br>
produced between day 3 and day 6. This increase was blocked by the addition of<br>
noggin (a BMP antagonist) at concentration of 3200 ng/ml and 800 ng/m. As<br>
shown in FIG. 10, however, noggin did not significantly alter sGAG production in<br>
uninfected cells. As can also be seen in FIG. 10, stimulation with rhBMP-2 at 100<br>
ng/ml led to a 3 fold increase in sGAG production between day 3 and day 6 after<br>
addition of BMP-2. Noggin at 800 ng/ml also blocked this increase.<br>
FIG. 11 is a chart showing the effect of LMP-1 on sGAG in media after day<br>
6 of culture in monolayer. The data points are represented as fold increase above<br><br>
untreated cells. As shown in FIG. 11, LMP-1 with the CMV promoter when<br>
delivered by the AAV vector is also effective in stimulating glycosaminoglycan<br>
synthesis by rat disc cells in monolayer.<br>
TABLE 2: Primer Sequences for RT-PCR &amp; Real-time PCR of SYBR Green<br><br><br>
TaqMan® Ribosomal RNA Control Reagents (Part number 4308329, Applied<br>
Biosystems, Foster City, CA, U.S.A.) were used for the forward primer, reverse<br>
primer and probe of 18S ribosomal RNA (rRNA) gene.<br>
All cited publications and patents are hereby incorporated by reference in<br>
their entirety.<br>
While the foregoing specification teaches the principles of the present<br>
invention, with examples provided for the purpose of illustration, it will be<br>
appreciated by one skilled in the art from reading this disclosure that various<br>
changes in form and detail can be made without departing from the true scope of<br>
the invention.<br><br>
WE CLAIM:<br>
1. A method of expressing a LIM mineralization protein in a non-osseous<br>
mammalian cell, comprising:<br>
transferring the cell with an isolated nucleic acid comprising a nucleotide<br>
sequence encoding the LIM mineralization protein operably linked to a promoter.<br>
2. The method as claimed in claim 1, wherein the cell is capable of producing<br>
proteoglycan and/or collagen and wherein expression of the LIM mineralization protein<br>
stimulates proteoglycan and/or collagen synthesis in the cell.<br>
3. The method as claimed in claim 2, wherein the isolated nucleic acid :<br>
hybridizes under standard conditions to a nucleic acid molecule complementary to<br>
the full length of SEQ. ID NO: 25; and/or<br>
hybridizes under highly stringent conditions to a nucleic acid molecule<br>
complementary to the full length of SEQ. ID NO: 26.<br>
4. The method as claimed in claim 2, wherein the cell is a stem cell, an intervertebral<br>
disc cell, a cell of the nucleus pulposus, or a cell of the annulus fibrosus.<br>
5. The method as claimed in claim 1, wherein the cell is trasfected ex vivo.<br>
6. The method as claimed in claim 1, wherein the cell is transfected in vivo.<br>
7. The method as claimed in claim 2, wherein the cell is trasfected in vivo by direct<br>
injection of the nucleic acid into an intervertebral disc of a mammal.<br>
8. The method as claimed in claim 1, wherein the nucleic acid is in a vector.<br>
9. The method as claimed in claim 8, wherein the vector is an expression vector.<br>
10. The method as claimed in claim 9, wherein the expression vector is a plasmid.<br>
11. The method as claimed in claim 8, wherein the vector is a virus.<br>
12. The method as claimed in claim 11, wherein the virus is an adenovirus.<br>
13. The method as claimed in claim 11, wherein the virus is a retro virus.<br>
14. The method as claimed in claim 12, wherein the adenovirus is AdHLMP-1.<br>
15. The method as claimed in claim 1, wherein the promoter is a cytomegalovirus<br>
promoter.<br>
16. The method as claimed in claim 1, wherein the proteoglycan is sulfated<br>
glycosaminoglycan.<br><br>
17. The method as claimed in claim 1, wherein the LIM mineralization protein is<br>
RLMP, HLMP-1, HLMP-ls, HLMP-2, or HLMP-3.<br>
18. The method as claimed in claim 1, wherein the LIM mineralization protein is<br>
HLMP-1.<br>
19. The method as claimed in claim 1, wherein expression of the LIM mineralization<br>
protein increases the expression of one or more bone morphogenetic proteins in the cell.<br>
20. A pharmaceutical composition comprising a recombinant non-osseous<br>
mammalian cell transfected with an isolated nucleic acid encoding a LIM mineralization<br>
protein and a carrier.<br>
21. The composition as claimed in claim 20, wherein the non-osseous mammalian cell<br>
is a stem cell, a cell of the annulus fibrosus, a cell of the nucleus pulposus or an<br>
intervertebral disc cell.<br>
22. The composition as claimed in claim 20, wherein the non-osseous mammalian cell<br>
is capable of producing proteoglycan and/or collagen.<br>
23. The composition as claimed in claim 20, wherein the nucleotide sequence is<br>
operably linked to a promoter<br>
24. The composition as claimed in claim 22, wherein the expression of the LIM<br>
mineralization protein stimulates proteoglycan and/or collagen synthesis in the cell.<br>
25. The composition as claimed in claim 20, wherein the composition is for treating<br>
intervertebral disc injury or disease in a mammal.<br>
26. The composition as claimed in claim 20, wherein the composition reverses,<br>
prevents or retards disc degeneration.<br>
27. The composition as claimed in claim 25, wherein the disc disease is degenerative<br>
disc disease, lower back pain, disc herniation, or spinal stenosis.<br>
28. The composition as claimed in claim 20, wherein the isolated nucleic acid is in a<br>
vector.<br>
29. The composition as claimed in claim 20, wherein the isolated nucleic acid is in an<br>
expression vector.<br>
30. The composition as claimed in claim 20, wherein the isolated nucleic acid is in a<br>
plasmid.<br><br>
31. The composition as claimed in claim 20, wherein the isolated nucleic acid is in a<br>
virus.<br>
32. The composition as claimed in claim 20, wherein the isolated nucleic acid is in an<br>
adenovirus.<br>
33. The composition as claimed in claim 20, wherein the isolated nucleic acid is in a<br>
retrovirus.<br>
34. The composition as claimed in claim 32, wherein the adenovirus is AdHLMP-1.<br>
35. The composition as claimed in claim 23, wherein the promoter is a<br>
cytomegalovirus promoter.<br>
36. The composition as claimed in claim 20, wherein the LIM mineralization protein<br>
is RLMP, HLMP-1, HLMP-1s, HLMP-2, or HLMP-3.<br>
37. The composition as claimed in claim 20, wherein the LIM mineralization protein<br>
is HLMP-1.<br>
38. The composition as claimed in claim 20, wherein the isolated nucleic acid<br>
hybridizes under standard conditions to a nucleic acid molecule complementary to the full<br>
length of SEQ.ID NO: 25; and/or hybridizes under highly stringent conditions to a nucleic<br>
acid molecule complementary to the full length of SEQ.ID NO: 26.<br>
39. The composition as claimed in claim 20, wherein the composition is to be<br>
implanted into an intervertebral disc.<br>
40. The composition as claimed in claim 20, wherein the carrier comprises a porous<br>
matrix.<br>
41. The composition as claimed in claim 20, wherein the carrier comprises a synthetic<br>
polymer or collagen matrix.<br>
42. An artificial intervertebral disc implant comprising: a carrier material; and a<br>
plurality of mammalian cells comprising an isolated nucleic acid sequence encoding a<br>
LIM mineralization protein; wherein the carrier material comprises a porous matrix of<br>
biocompatible material and wherein the mammalian cells are incorporated into the<br>
carrier.<br><br>
43. The implant as claimed in claim 42, wherein the mammalian cells are selected<br>
from the group consisting of stem cells, cells of the annulus fibrosus, cells of the nucleus<br>
pulposus, intervertebral disc cells and combinations thereof.<br>
44. The implant as claimed in claim 42, wherein the biocompatible material<br>
comprises a synthetic polymer or a protein.<br>
45. The implant as claimed in claim 42, wherein the biocompatible material comprises<br>
collagen.<br>
SEQUENCE LISTING<br> McKay, William F.<br>
Boden M.D., Scott D<br>
Yoon, Sangwook T.<br> Methods of Expressing LIM Mineralization Protein in<br>
Intervertebral Disc Cells<br> 3819-002-53<br><br><br> US 60/331,321<br> 2001-11-14<br> 42<br> PatentIn Ver. 2.1<br> 1<br> 457<br> PRT<br> Rattus norvegicus<br><br><br><br> 2<br> 1696<br> DNA<br> Rattus norvegicus<br> 2<br>
gcacgaggat cccagcgcgg ctcctggagg ccgccaggca gccgcccagc cgggcattca 60<br>
ggagcaggta ccatggattc cttcaaggta gtgctggagg gacctgcccc ttggggcttc 120<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
The invention relates generally to methods for expressing LIM mineralization proteins in non-osseous<br>
cells to treat intervertebral disk degeneration. The methods involve transfecting the cells with<br>
an isolated nucleic acid comprising a nucleotide sequence encoding a LIM mineralization protein<br>
operably linked to a promoter. Transfection may be accomplished ex vivo or in vivo by direct injection<br>
of virus or naked DNA, or by a nonviral vector such as a plasmid. Expression of the LIM mineralization<br>
protein can stimulate proteoglycan and/or collagen production in cells capable of producing<br>
proteoglycan and/or collagen. The invention also provides a pharmaceutical composition for treating<br>
intervertebral disc injury or disease in a mammal comprising a recombinant non-osseous mammalian<br>
cell transfected with an isolated nucleic acid encoding a LIM mineralization protein and an artificial<br>
intervertebral disc implant comprising a carrier material and mammalian cells including an isolated<br>
nucleic acid sequence encoding a LIM mineralization protein.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzYzLWtvbG5wLTIwMDQtZ3JhbnRlZC1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">763-kolnp-2004-granted-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzYzLWtvbG5wLTIwMDQtZ3JhbnRlZC1hc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">763-kolnp-2004-granted-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzYzLWtvbG5wLTIwMDQtZ3JhbnRlZC1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">763-kolnp-2004-granted-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzYzLWtvbG5wLTIwMDQtZ3JhbnRlZC1jb3JyZXNwb25kZW5jZS5wZGY=" target="_blank" style="word-wrap:break-word;">763-kolnp-2004-granted-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzYzLWtvbG5wLTIwMDQtZ3JhbnRlZC1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">763-kolnp-2004-granted-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzYzLWtvbG5wLTIwMDQtZ3JhbnRlZC1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">763-kolnp-2004-granted-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzYzLWtvbG5wLTIwMDQtZ3JhbnRlZC1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">763-kolnp-2004-granted-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzYzLWtvbG5wLTIwMDQtZ3JhbnRlZC1mb3JtIDE4LnBkZg==" target="_blank" style="word-wrap:break-word;">763-kolnp-2004-granted-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzYzLWtvbG5wLTIwMDQtZ3JhbnRlZC1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">763-kolnp-2004-granted-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzYzLWtvbG5wLTIwMDQtZ3JhbnRlZC1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">763-kolnp-2004-granted-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzYzLWtvbG5wLTIwMDQtZ3JhbnRlZC1mb3JtIDYucGRm" target="_blank" style="word-wrap:break-word;">763-kolnp-2004-granted-form 6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzYzLWtvbG5wLTIwMDQtZ3JhbnRlZC1ncGEucGRm" target="_blank" style="word-wrap:break-word;">763-kolnp-2004-granted-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzYzLWtvbG5wLTIwMDQtZ3JhbnRlZC1sZXR0ZXIgcGF0ZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">763-kolnp-2004-granted-letter patent.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzYzLWtvbG5wLTIwMDQtZ3JhbnRlZC1yZXBseSB0byBleGFtaW5hdGlvbiByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">763-kolnp-2004-granted-reply to examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzYzLWtvbG5wLTIwMDQtZ3JhbnRlZC1zZXF1ZW5jZSBsaXN0aW5nLnBkZg==" target="_blank" style="word-wrap:break-word;">763-kolnp-2004-granted-sequence listing.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzYzLWtvbG5wLTIwMDQtZ3JhbnRlZC1zcGVjaWZpY2F0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">763-kolnp-2004-granted-specification.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzYzLWtvbG5wLTIwMDQtZ3JhbnRlZC10cmFuc2xhdGVkIGNvcHkgb2YgcHJpb3JpdHkgZG9jdW1lbnQucGRm" target="_blank" style="word-wrap:break-word;">763-kolnp-2004-granted-translated copy of priority document.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="217406-process-for-preparing-palladium-0-compounds.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="217408-nicotinamide-derivatives-useful-as-p38-inhibitors.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>217407</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>00763/KOLNP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>28-Mar-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>26-Mar-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>04-Jun-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>WARSAW ORTHOPEDIC,INC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>2500 SILVEUS CROSSING WARSAW INDIANA 46581 USA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>MCKAY WILLIAM F</td>
											<td>3870 MCELRIE COVE MEMPHIS TN 38133 USA.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>BODEN SCOTT D</td>
											<td>2842 CRAVEY DRIVE ATLANTA TN30345 USA.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>YOON SAN GWOOK T</td>
											<td>2431 VALHALLA DRIVE, ATLANTA GA 30345 USA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US02/36465</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-11-14</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/331, 321</td>
									<td>2001-11-14</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/217407-a-method-of-expressing-a-lim-mineralization-protein-in-a-non-osseous-mammalian-cell by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:43:17 GMT -->
</html>
