<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/217673-a-method-of-isolating-nucleic-acid-ligands-to-tenascin-c by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:00:14 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 217673:&quot;A METHOD OF ISOLATING NUCLEIC ACID LIGANDS TO TENASCIN-C&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A METHOD OF ISOLATING NUCLEIC ACID LIGANDS TO TENASCIN-C&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>method of isolating nucleic acid ligands to tenascin-C, the method comprising: a) contacting a candidate mixture of nucleic acids with tenascin-C, wherein nucleic acids having an increased affinity to tenascin-C relative to the candidate mixture may be partitioned from the remainder of the candidate mixture; b) partitioning the increased affinity nucleic acids from the remainder of the candidate mixture; c) amplifying the increased affinity nucleic acids to yield a mixture of nucleic acids enriched for nucleic acids with relatively higher affinity and specificity for binding to tenascin-C, whereby a nucleic acid ligand of tenascin-C may be identified; and d) repeating steps a), b)and c).</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>TENASCIN-C NUCLEIC ACID LIGANDS<br>
FIELD OF THE INVENTION<br>
Described herein are high affinity nucleic acid ligands to tenascin-C. Also described herein are methods for identifying and preparing high affinity nucleic acid ligands to tenascin-C. The method used herein for identifying such nucleic acid ligands is called SELEX, an acronym for Systematic Evolution of Ligands by Exponential enrichment. Further disclosed are high affinity nucleic acid ligands to tenascin-C. Further disclosed are RNA ligands to tenascin-C. Also included are oligonucleotides containing nucleotide derivatives chemically modified at the 2'-positions of the purines and pyrimidines. Additionally disclosed are RNA ligands to tenascin-C containing 2'-F and 2'OMe modifications. The oligonucleotides of the present invention are useful as diagnostic and/or therapeutic agents.<br>
BACKGROUND OF THE INVENTION<br>
Tenascin-C is a 1.1-1.5 million Da, hexameric glycoprotein that is located primarily in the extracellular matrix. Tenascin-C is expressed during embryogenesis, wound healing, and neoplasia, suggesting a role for this protein in tissue remodeling (Erickson and Bourdon (1989) Ann Rev Cell Biol 5:71-92). Neoplastic processes also involve tissue remodeling, and tenascin-C is over-expressed in many tumor types including carcinomas of the lung, breast, prostate, and colon, astrocytomas, glioblastomas, melanomas, and sarcomas (Soini et al. (1993) Am J Clin Pathol 100(2): 145-50: Koukoulis et al. (1991) Hum Pathol 22(7):636-43: Borsi etal. (1992) Int J Cancer 52(5):688-92; Koukoulis et al. (1993) J Submicrosc Cytol Pathol 25(2):285-95; Ibrahim et al. (1993) Hum Pathol 24(9^:982-9: Riedl et al. (1998) Dis Colon Rectum 41(11:86-92: Tuominen and Kallioinen (1994) J Cutan Pathol 21(5):424-9: Natali et al. (1990) Int J Cancer 46(4):586-90; Zagzag et al. (1995) Cancer Res 55(4):907-14; Hasegawa et al. (1997) Acta Neuropathol (Bed) 93(5):431-7; Saxon et al. (1997) Pediatr Pathol Lab Med 17(2):259-66: Hasegawa et al. (1995) Hum Pathol 26(8):838-45). In addition, tenascin-C is overexpressed in hyperproliferative skin diseases, e.g. psoriasis (Schalkwijk et al. (1991) Br J Dermatol 124(1): 13-20). and in atherosclerotic lesions (Fukumoto et al. (1998) J Atheroscler Thromb 5(1):29-35; Wallner et al. (1999) Circulation 99(10): 1284-9). Radiolabeled antibodies that bind tenascin-C are used for imaging and therapy of tumors in clinical settings (Paganelli et al. (1999) Eur J Nucl Med 26(4):348-57; Paganelli et al. (1994) Eur J Nucl Med 21(4):314-21; Bigner et al. (1998) J Clin Oncol<br>
'6):2202-12; Merlo et al. (1997) Int J Cancer 71(5):810-6).<br>
Aptamers against tenascin-C have potential utility for cancer diagnosis and therapy, as well as for diagnosis and therapy of atheroslerosis and therapy of psoriasis. Relative to antibodies, aptamers are small (7-20 kDa), clear very rapidly from blood, and are chemically synthesized. Rapid blood clearance is important for in vivo diagnostic imaging, where blood levels are a primary determinant of background that obscures an image. Rapid blood clearance may also be important in therapy, where blood levels may contribute to toxicity. SELEX technology allows rapid aptamer isolation, and chemical synthesis enables facile and site-specific conjugation of aptamers to a variety of inert and bioactive molecules. An aptamer to tenascin-C would therefore be useful for tumor therapy or in vivo or ex vivo diagnostic imaging and/or for delivering a variety of therapeutic agents complexed with the tenascin-C nucleic acid ligand for treatment of disease conditions in which tenascin-C is expressed.<br>
The dogma for many years was that nucleic acids had primarily an informational role. Through a method known as Systematic Evolution of Ligands by Exponential enrichment, termed the SELEX process, it has become clear that nucleic acids have three dimensional structural diversity not unlike proteins. The SELEX process is a method for the in vitro evolution of nucleic acid molecules with highly specific binding to target molecules and is described in United States Patent Application Serial No. 07/536.428. filed June 11, 1990, entitled "Systematic Evolution of Ligands by Exponential Enrichment," now abandoned. United States Patent No. 5,475.096 entitled "Nucleic Acid Ligands," United States Patent No. 5,270,163 (see also WO 91/19813) entitled "Methods for Identifying Nucleic Acid Ligands," each of which is specifically incorporated by reference herein in its entirety. Each of these applications, collectively referred to herein as the SELEX Patent Applications, describes a fundamentally novel method for making a nucleic acid ligand to any desired target molecule. The SELEX process provides a class of products which are referred to as nucleic acid ligands or aptamers. each having a unique sequence, and which have the property of binding specifically to a desired target compound or molecule. Each SELEX-identified nucleic acid ligand is a specific ligand of a given target compound or molecule. The SELEX process is based on the unique insight that nucleic acids have sufficient capacity for forming a variety of two- and three-dimensional structures and sufficient chemical versatility available within their monomers to act as ligands (form specific binding pairs) with virtually any chemical compound, whether monomeric or polymeric. Molecules of any size or composition can<br>
ve as targets in the SELEX method. The SELEX method applied to the application of<br>
high affinity binding involves selection from a mixture of candidate oligonucleotides and step-wise iterations of binding, partitioning and amplification, using the same general selection scheme, to achieve virtually any desired criterion of binding affinity and selectivity. Starting from a mixture of nucleic acids, preferably comprising a segment of randomized sequence, the SELEX method includes steps of contacting the mixture with the target under conditions favorable for binding, partitioning unbound nucleic acids from those nucleic acids which have bound specifically to target molecules, dissociating the nucleic acid-target complexes, amplifying the nucleic acids dissociated from the nucleic acid-target complexes to yield a ligand-enriched mixture of nucleic acids, then reiterating the steps of binding, partitioning, dissociating and amplifying through as many cycles as desired to yield highly specific high affinity nucleic acid ligands to the target molecule.<br>
It has been recognized by the present inventors that the SELEX method demonstrates that nucleic acids as chemical compounds can form a wide array of shapes, sizes and configurations, and are capable of a far broader repertoire of binding and other functions than those displayed by nucleic acids in biological systems.<br>
The basic SELEX method has been modified to achieve a number of specific objectives. For example, United States Patent Application Serial No. 07/960,093, filed October 14, 1992, now abandoned, and United States Patent No. 5,707,796, both entitled "Method for Selecting Nucleic Acids on the Basis of Structure," describe the use of the SELEX process in conjunction with gel electrophoresis to select nucleic acid molecules with specific structural characteristics, such as bent DNA. United States Patent Application Serial No. 08/123,935, filed September 17, 1993, entitled "Photoselection of Nucleic Acid Ligands," now abandoned, United States Patent No. 5,763,177, entitled "Systematic Evolution of Ligands by Exponential Enrichment: Photoselection of Nucleic Acid Ligands and Solution SELEX" and United States Patent Application Serial No. 09/093,293, filed June 8, 1998, entitled "Systematic Evolution of Ligands by Exponential Enrichment: Photoselection of Nucleic Acid Ligands and Solution SELEX," describe a SELEX based method for selecting nucleic acid ligands containing photoreactive groups capable of binding and/or photocrosslinking to and/or photoinactivating a target molecule. United States Patent No. 5,580,737, entitled "High-Affinity Nucleic Acid Ligands That Discriminate Between Theophylline and Caffeine," describes a method for identifying highly specific nucleic acid ligands able to discriminate between closely related molecules, which can be non-peptidic,<br>
jned Counter-SELEX. United States Patent No. 5,567,588, entitled "Systematic Evolution<br>
of Ligands by Exponential Enrichment: Solution SELEX," describes a SELEX-based method which achieves highly efficient partitioning between oligonucleotides having high and low affinity for a target molecule.<br>
The SELEX method encompasses the identification of high-affinity nucleic acid ligands containing modified nucleotides conferring improved characteristics on the ligand, such as improved in vivo stability or improved delivery characteristics. Examples of such modifications include chemical substitutions at the ribose and/or phosphate and/or base positions. SELEX process-identified nucleic acid ligands containing modified nucleotides are described in United States Patent No. 5,660,985, entitled "High Affinity Nucleic Acid Ligands Containing Modified Nucleotides," that describes oligonucleotides containing nucleotide derivatives chemically modified at the 5- and 2'-positions of pyrimidines. United States Patent No. 5,580,737, supra, describes highly specific nucleic acid ligands containing one or more nucleotides modified with 2'-amino (2'-NH2), 2'-fluoro (2'-F), and/or 2'-O-methyl (2'-OMe). United States Patent Application Serial No. 08/264,029, filed June 22, 1994, entitled "Novel Method of Preparation of Known and Novel 2' Modified Nucleosides by Intramolecular Nucleophilic Displacement," now abandoned, describes oligonucleotides containing various 2'-modified pyrimidines.<br>
The SELEX method encompasses combining selected oligonucleotides with other selected oligonucleotides and non-oligonucleotide functional units as described in United States Patent No. 5,637,459, entitled "Systematic Evolution of Ligands by Exponential Enrichment: Chimeric SELEX," and United States Patent No. 5.683,867, entitled "Systematic Evolution of Ligands by Exponential Enrichment: Blended SELEX," respectively. These applications allow the combination of the broad array of shapes and other properties, and the efficient amplification and replication properties, of oligonucleotides with the desirable properties of other molecules.<br>
The SELEX method further encompasses combining selected nucleic acid ligands with lipophilic compounds or non-immunogenic, high molecular weight compounds in a diagnostic or therapeutic complex as described in United States Patent Application Serial No. 08/434,465. filed May 4,1995, entitled "Nucleic Acid Ligand Complexes". Each of the above described patents and applications which describe modifications of the basic SELEX procedure are specifically incorporated by reference herein in their entirety.<br>
SUMMARY OF THE INVENTION<br>
The present invention describes a method for isolating nucleic acid ligands that bind to tenascin-C with high specificity. Further described herein are nucleic acid ligands to tenascin-C. Also described herein are high affinity RNA ligands to tenascin-C. Further described are 2'fluoro-modified pyrimidine and 2'OMe-modified purine RNA ligands to tenascin-C. The method utilized herein for identifying such nucleic acid ligands is called SELEX, an acronym for Systematic Evolution of Ligands by Exponential enrichment. Included herein are the ligands that are shown in Tables 3 and 4 and Figure 10.<br>
Further included in this invention is a method for detecting the presence of a disease that is expressing tenascin-C in a biological tissue that may contain the disease. Still further included in this invention is a method for detecting the presence of a tumor that is expressing tenascin-C in a biological tissue that may contain the tumor. Further included in this invention is a complex for use in in vivo or ex vivo diagnostics. Still further included in this invention is a method for delivering therapeutic agents for the treatment or prophylaxis of diseased tissues that express tenascin-C. Still further included in this invention is a complex for use in delivering therapeutic agents for treatment or prophylaxis of diseased tissues that express tenascin-C.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figure 1 shows binding of Cell SELEX RNA pools to U251 cells.<br>
Figure 2 shows proposed secondary structure of aptamers TTA1 and TTA1 .NB. Included in the figure is the conjugation of the aptamers with Tc-99m chelator. All A's are 2'OMe modified. All G's, except as indicated, are 2'OMe modified. All C's and U's are 2'F modified.<br>
Figure 3 shows images of U251 tumor xenografts in mice, obtained using Tc-99m-labeled TTA1 and TTA1.NB, three hours post-injection.<br>
Figure 4 shows fluorescence microscopy of a U251 glioblastoma tumor section, taken three hours after i.v. injection of Rhodamine-Red-X-labeled TTA1.<br>
Figure 5 shows the way in which the Tc-99m and linker is bound through the 5'G of TTA1.<br>
Figure 6 describes TTA1/GS7641 uptake at 3 hours into various human tumor xenografts in mouse, compared to uptake of the non-binding control aptamer. ID/g = injected dose/gram.<br>
Figure 7 displays the biodistribution of In-111 labeled TTA1/GS7641 using either<br>
DOTA or DTPA as the radiometal chelator, 3 hours after injection. ID/g = injected dose/gram.<br>
Figure 8 shows the conjugation of the aptamer to DTPA. The ' 'in is shown as chelated by DTPA.<br>
Figure 9 shows the conjugation of the aptamer to DOTA. The ' 'in is shown as chelated by DOTA.<br>
Figure 10 shows the proposed secondary structure of aptamer TTA1. Included in the figure is the conjugation of the aptamer with Tc-99m chelator. The aptamer is shown in its Tc-99m labled form. All A's are 2'OMe modified. All G's, except as indicated, are 2'OMe modified. All C's and U's are 2'F modified.<br>
DETAILED DESCRIPTION OF THE PREFERRED EMBODIMENTS<br>
The central method utilized herein for identifying nucleic acid ligands to tenascin-C is called the SELEX process, an acronym for Systematic Evolution of Ligands by Exponential enrichment and involves (a) contacting the candidate mixture of nucleic acids with tenascin-C (b) partitioning between members of said candidate mixture on the basis of affinity to tenascin-C, and c) amplifying the selected molecules to yield a mixture of nucleic acids enriched for nucleic acid sequences with a relatively higher affinity for binding to tenascin-C. The invention includes RNA ligands to tenascin-C. This invention further includes the specific RNA ligands to tenascin-C shown in Tables 3 and 4 and Figure 10. More specifically, this invention includes nucleic acid sequences that are substantially homologous to and that have substantially the same ability to bind tenascin-C as the specific nucleic acid ligands shown in Tables 3 and 4 and Figure 10. By substantially homologous it is meant a degree of primary sequence homology in excess of 70%, most preferably in excess of 80%, and even more preferably in excess of 90%, 95%. or 99%. The percentage of homology as described herein is calculated as the percentage of nucleotides found in the smaller of the two sequences which align with identical nucleotide residues in the sequence being compared when 1 gap in a length of 10 nucleotides may be introduced to assist in that alignment. Substantially the same ability to bind tenascin-C means that the affinity is within one or two orders of magnitude of the affinity of the ligands described herein. It is well within the skill of those of ordinary skill in the art to determine whether a given sequence - substantially homologous to those specifically described herein - has the same ability to bind tenascin-C.<br>
A review of the sequence homologies of the nucleic acid ligands of tenascin-C shown<br>
in Tables 3 and 4 and Figure 10 shows that sequences with little or no primary homology may have substantially the same ability to bind tenascin-C. For these reasons, this invention also includes nucleic acid ligands that have substantially the same postulated structure or structural motifs and ability to bind tenascin-C as the nucleic acid ligands shown in Tables 3 and 4 and Figure 10. Substantially the same structure or structural motifs can be postulated by sequence alignment using the Zukerfold program (see Zuker (1989) Science 244:48-521 As would be known in the art, other computer programs can be used for predicting secondary structure and structural motifs. Substantially the same structure or structural motif of nucleic acid ligands in solution or as a bound structure can also be postulated using NMR or other techniques as would be known in the art.<br>
Further included in this invention is a method for detecting the presence of a disease that is expressing tenascin-C in a biological tissue which may contain the disease by the method of (a) identifying a nucleic acid ligand from a candidate mixture of nucleic acids, the nucleic acid ligand being a ligand of tenascin-C, by the method comprising (i) contacting a candidate mixture of nucleic acids with tenascin-C, wherein nucleic acids having an increased affinity to tenascin-C relative to the candidate mixture may be partitioned from the remainder of the candidate mixture; (ii) partitioning the increased affinity nucleic acids from the remainder of the candidate mixture;<br>
(iii) amplifying the increased affinity nucleic acids to yield a mixture of nucleic acids with relatively higher affinity and specificity for binding to tenascin-C, whereby a nucleic acid ligand of tenascin-C is identified; (b) attaching a marker that can be used in in vivo or ex vivo diagnostics to the nucleic acid ligand identified in step (iii) to form a marker-nucleic acid ligand complex; (c) exposing a tissue which may contain the disease to the marker-nucleic acid ligand complex; and (d) detecting the presence of the marker-nucleic acid ligand in the tissue, whereby a disease expressing tenascin-C is identified.<br>
It is a further object of the present invention to provide a complex for use in in vivo or ex vivo diagnostics comprising one or more tenascin-C nucleic acid ligands and one or more markers. Still further included in this invention is a method for delivering therapeutic agents for the treatment or prophylaxis of disease conditions in which tenascin-C is expressed. Still further included in this invention is a complex for use in delivering therapeutic agents for treatment or prophylaxis of disease conditions in which tenascin-C is expressed.<br>
Various terms are used herein to refer to aspects of the present invention. To aid in the clarification of the description of the components of this invention, the following definitions are provided:<br>
As used herein, "nucleic acid ligand" is a non-naturally occurring nucleic acid having a desirable action on a target. Nucleic acid ligands are often referred to as "aptamers." The target of the present invention is tenascin-C, hence the term tenascin-C nucleic acid ligand. A desirable action includes, but is not limited to, binding of the target, catalytically changing the target, reacting with the target in a way which modifies/alters the target or the functional activity of the target, covalently attaching to the target as in a suicide inhibitor, facilitating the reaction between the target and another molecule. In the preferred embodiment, the action is specific binding affinity for a target molecule, such target molecule being a three dimensional chemical structure other than a polynucleotide that binds to the nucleic acid ligand through a mechanism which predominantly depends on Watson/Crick base pairing or triple helix binding, wherein the nucleic acid ligand is not a nucleic acid having the known physiological function of being bound by the target molecule. Nucleic acid ligands are identified from a candidate mixture of nucleic acids, said nucleic acid ligand being a ligand of a tenascin-C, by the method comprising: a) contacting the candidate mixture with tenascin-C, wherein nucleic acids having an increased affinity to tenascin-C relative to the candidate mixture may be partitioned from the remainder of the candidate mixture; b) partitioning the increased affinity nucleic acids from the remainder of the candidate mixture: and c) amplifying the increased affinity nucleic acids to yield a ligand-enriched mixture of nucleic acids (see U.S. Patent Application No. 08/434,425, filed May 3,1995, now United States Patent No. 5.789,157, which is hereby incorporated herein by reference).<br>
As used herein, "candidate mixture" is a mixture of nucleic acids of differing sequence from which to select a desired ligand. The source of a candidate mixture can be from naturally-occurring nucleic acids or fragments thereof, chemically synthesized nucleic acids, enzymatically synthesized nucleic acids or nucleic acids made by a combination of the foregoing techniques. In a preferred embodiment, each nucleic acid has fixed sequences surrounding a randomized region to facilitate the amplification process.<br>
As used herein, "nucleic acid" means either DNA, RNA, single-stranded or double-stranded, and any chemical modifications thereof. Modifications include, but are not limited to, those which provide other chemical groups that incorporate additional charge,<br>
poizability, hydrogen bonding, electrostatic interaction, and fluxionality to the nucleic acid<br>
ligand bases or to the nucleic acid ligand as a whole. Such modifications include, but are not limited to, 2'-position sugar modifications, 5-position pyrimidine modifications, 8-position purine modifications, modifications at exocyclic amines, substitution of 4-thiouridine, substitution of 5-bromo or 5-iodo-uracil; backbone modifications, methylations, unusual base-pairing combinations such as the isobases isocytidine and isoguanidine and the like. Modifications can also include 3' and 5' modifications such as capping.<br>
"SELEX" methodology involves the combination of selection of nucleic acid ligands which interact with a target in a desirable manner, for example binding to a protein, with amplification of those selected nucleic acids. Optional iterative cycling of the selection/amplification steps allows selection of one or a small number of nucleic acids which interact most strongly with the target from a pool which contains a very large number of nucleic acids. Cycling of the selection/amplification procedure is continued until a selected goal is achieved. In the present invention, the SELEX methodology is employed to obtain nucleic acid ligands to tenascin-C.<br>
The SELEX methodology is described in the SELEX Patent Applications.<br>
"SELEX target" or "target" means any compound or molecule of interest for which a ligand is desired. A target can be a protein, peptide, carbohydrate, polysaccharide, glycoprotein, hormone, receptor, antigen, antibody, virus, substrate, metabolite, transition state analog, cofactor, inhibitor, drug, dye, nutrient, growth factor, etc. without limitation. In this application, the SELEX target is tenascin-C.<br>
"Complex" as used herein means the molecular entity formed by the covalent linking of one or more tenascin-C nucleic acid ligands with one or more markers. In certain embodiments of the present invention, the complex is depicted as A-B-Y, wherein A is a marker; B is optional, and comprises a linker; and Y is a tenascin-C nucleic acid ligand.<br>
"Marker" as used herein is a molecular entity or entities that when complexed with the tenascin-C nucleic acid ligand, either directly or through a linker(s) or spacer(s), allows the detection of the complex in an in vivo or ex vivo setting through visual or chemical means. Examples of markers include, but are not limited to radionuclides. including Tc-99m, Re-188, Cu-64, Cu-67, F-18, I25I, I31I, mm,32P, 186Re; all fluorophores, including fluorescein, rhodamine, Texas Red; derivatives of the above fluorophores, including Rhodamine-Red-X; magnetic compounds; and biotin.<br>
As used herein, "linker" is a molecular entity that connects two or more molecular<br>
entities through covalent bond or non-covalent interactions, and can allow spatial separation of the molecular entities in a manner that preserves the functional properties of one or more of the molecular entities. A linker can also be known as a spacer. Examples of a linker include, but are not limited to, the (CH2CH20)6 and hexylamine structures shown in Figure 2.<br>
"Therapeutic" as used herein, includes treatment and/or prophylaxis. When used, therapeutic refers to humans and other animals.<br>
"Covalent Bond" is the chemical bond formed by the sharing of electrons. "Non-covalent interactions" are means by which molecular entities are held together by interactions other than Covalent Bonds including ionic interactions and hydrogen bonds.<br>
In the preferred embodiment, the nucleic acid ligands of the present invention are derived from the SELEX methodology. The SELEX process is described in United States Patent Application Serial No. 07/536,428, entitled "Systematic Evolution of Ligands by Exponential Enrichment," now abandoned, United States Patent No. 5,475,096, entitled "Nucleic Acid Ligands" and United States Patent No. 5,270,163, (see also WO 91/19813), entitled "Methods for Identifying Nucleic Acid Ligands." These applications, each specifically incorporated herein by reference, are collectively called the SELEX Patent Applications.<br>
The SELEX process provides a class of products which are nucleic acid molecules, each having a unique sequence, and each of which has the property of binding specifically to a desired target compound or molecule. Target molecules are preferably proteins, but can also include among others carbohydrates, peptidoglycans and a variety of small molecules. SELEX methodology can also be used to target biological structures, such as cell surfaces or viruses, through specific interaction with a molecule that is an integral part of that biological structure.<br>
In its most basic form, the SELEX process may be defined by the following series of<br>
steps:<br>
1) A candidate mixture of nucleic acids of differing sequence is prepared. The candidate mixture generally includes regions of fixed sequences (i.e., each of the members of the candidate mixture contains the same sequences in the same location) and regions of randomized sequences. The fixed sequence regions are selected either: (a) to assist in the amplification steps described below, (b) to mimic a sequence known to bind to the target, or<br>
enhance the concentration of a given structural arrangement of the nucleic acids in the candidate mixture. The randomized sequences can be totally randomized (i.e., the probability of finding a base at any position being one in four) or only partially randomized (e.g., the probability of finding a base at any location can be selected at any level between 0 and 100 percent).<br>
2)	The candidate mixture is contacted with the selected target under conditions<br>
favorable for binding between the target and members of the candidate mixture. Under these<br>
circumstances, the interaction between the target and the nucleic acids of the candidate<br>
mixture can be considered as forming nucleic acid-target pairs between the target and those<br>
nucleic acids having the strongest affinity for the target.<br>
3)	The nucleic acids with the highest affinity for the target are partitioned from those<br>
nucleic acids with lesser affinity to the target. Because only an extremely small number of<br>
sequences (and possibly only one molecule of nucleic acid) corresponding to the highest<br>
affinity nucleic acids exist in the candidate mixture, it is generally desirable to set the<br>
partitioning criteria so that a significant amount of the nucleic acids in the candidate mixture<br>
(approximately 5-50%) are retained during partitioning.<br>
4)	Those nucleic acids selected during partitioning as having the relatively higher<br>
affinity for the target are then amplified to create a new candidate mixture that is enriched in<br>
nucleic acids having a relatively higher affinity for the target.<br>
5)	By repeating the partitioning and amplifying steps above, the newly formed<br>
candidate mixture contains fewer and fewer unique sequences, and the average degree of<br>
affinity of the nucleic acids to the target will generally increase. Taken to its extreme, the<br>
SELEX process will yield a candidate mixture containing one or a small number of unique<br>
nucleic acids representing those nucleic acids from the original candidate mixture having the<br>
highest affinity to the target molecule.<br>
The basic SELEX method has been modified to achieve a number of specific objectives. For example. United States Patent Application Serial No. 07/960.093, filed October 14, 1992, now abandoned, and United States Patent No. 5,707,796, both entitled "Method for Selecting Nucleic Acids on the Basis of Structure," describe the use of the SELEX process in conjunction with gel electrophoresis to select nucleic acid molecules with specific structural characteristics, such as bent DNA. United States Patent Application Serial No. 08/123,935, filed September 17, 1993, entitled "Photoselection of Nucleic Acid Ligands," now abandoned. United States Patent No. 5,763,177, entitled "Systematic<br>
lution of Ligands by Exponential Enrichment: Photoselection of Nucleic Acid Ligands<br>
ana Solution SELEX" and United States Patent Application Serial No. 09/093,293, filed June 8 1998, entitled "Systematic Evolution of Ligands by Exponential Enrichment: Photoselection of Nucleic Acid Ligands and Solution SELEX," all describe a SELEX based method for selecting nucleic acid ligands containing photoreactive groups capable of binding and/or photocrosslinking to and/or photoinactivating a target molecule. United States Patent No. 5,580,737, entitled "High-Affinity Nucleic Acid Ligands That Discriminate Between Theophylline and Caffeine," describes a method for identifying highly specific nucleic acid ligands able to discriminate between closely related molecules, termed Counter-SELEX. United States Patent No. 5,567,588, entitled "Systematic Evolution of Ligands by Exponential Enrichment: Solution SELEX," describes a SELEX-based method which achieves highly efficient partitioning between oligonucleotides having high and low affinity for a target molecule. United States Patent No. 5,496,938, entitled "Nucleic Acid Ligands to HIV-RT and HIV-1 Rev," describes methods for obtaining improved nucleic acid ligands after SELEX has been performed. United States Patent No. 5,705,337 entitled "Systematic Evolution of Ligands by Exponential Enrichment: Chemi-SELEX," describes methods for covalently linking a ligand to its target.<br>
The SELEX method encompasses the identification of high-affinity nucleic acid ligands containing modified nucleotides conferring improved characteristics on the ligand, such as improved in vivo stability or improved delivery characteristics. Examples of such modifications include chemical substitutions at the ribose and/or phosphate and/or base positions. SELEX-identified nucleic acid ligands containing modified nucleotides are described in United States Patent No. 5,660,985, entitled "High Affinity Nucleic Acid Ligands Containing Modified Nucleotides," that describes oligonucleotides containing nucleotide derivatives chemically modified at the 5- and 2'-positions of pyrimidines. United States Patent No. 5,637,459, supra, describes highly specific nucleic acid ligands containing one or more nucleotides modified with 2'-amino (2'-NH2), 2'-fluoro (2'-F). and/or 2'-O-methyl (2'-OMe). United States Patent Application Serial No. 08/264,029, filed June 22, 1994, entitled "Novel Method of Preparation of Known and Novel 2' Modified Nucleosides by Intramolecular Nucleophilic Displacement," describes oligonucleotides containing various 2'-modified pyrimidines.<br>
The SELEX method encompasses combining selected oligonucleotides with other selected oligonucleotides and non-oligonucleotide functional units as described in United<br>
Patent No. 5,637,459, entitled "Systematic Evolution of Ligands by Exponential<br>
Enrichment: Chimeric SELEX," and United States Patent No. 5,683,867, entitled "Systematic Evolution of Ligands by Exponential Enrichment: Blended SELEX," respectively. These applications allow the combination of the broad array of shapes and other properties, and the efficient amplification and replication properties, of oligonucleotides with the desirable properties of other molecules.<br>
In United States Patent No. 5,496,938 methods are described for obtaining improved Nucleic Acid Ligands after the SELEX process has been performed. This patent, entitled "Nucleic Acid Ligands to HIV-RT and HIV-1 Rev," is specifically incorporated herein by reference.<br>
United States Patent Application No. 08/434,425, entitled "Systematic Evolution of Ligands by Exponential Enrichment: Tissue SELEX," filed May 3, 1995, now United States Patent No. 5,789,157, describes methods for identifying a nucleic acid ligands to a macromolecular component of a tissue, including cancer cells, and the nucleic acid ligands so identified. This patent is specifically incorporated herein by reference.<br>
One potential problem encountered in the diagnostic or therapeutic use of nucleic acids is that oligonucleotides in their phosphodiester form may be quickly degraded in body fluids by intracellular and extracellular enzymes such as endonucleases and exonucleases before the desired effect is manifest. Certain chemical modifications of the nucleic acid ligand can be made to increase the in vivo stability of the nucleic acid ligand or to enhance or to mediate the delivery of the nucleic acid ligand. See, e.g., U.S. Patent Application Serial No. 08/117,991. filed September 8. 1993, now abandoned, and United States Patent No. 5,660,985, both entitled "High Affinity Nucleic Acid Ligands Containing Modified Nucleotides," which is specifically incorporated herein by reference. Modifications of the nucleic acid ligands contemplated in this invention include, but are not limited to, those which provide other chemical groups that incorporate additional charge, polarizability, hydrophobicity, hydrogen bonding, electrostatic interaction, and fluxionality to the nucleic acid ligand bases or to the nucleic acid ligand as a whole. Such modifications include, but are not limited to, 2'-position sugar modifications, 5-position pyrimidine modifications, 8-position purine modifications, modifications at exocyclic amines, substitution of 4-thiouridine, substitution of 5-bromo or 5-iodo-uracil; backbone modifications, phosphorothioate or alkyl phosphate modifications, methylations, unusual base-pairing combinations such as the isobases isocytidine and isoguanidine and the like. Modifications<br>
T pnp also include 3' and 5' modifications such as capping. In preferred embodiments of the<br>
instant invention, the nucleic acid ligands are RNA molecules that are 2'-fluoro (2'-F) modified on the sugar moiety of pyrimidine residues.<br>
The modifications can be pre- or post-SELEX process modifications. Pre-SELEX process modifications yield nucleic acid ligands with both specificity for their SELEX target and improved in vivo stability. Post-SELEX process modifications made to 2'-OH nucleic acid ligands can result in improved in vivo stability without adversely affecting the binding capacity of the nucleic acid ligand.<br>
Other modifications are known to one of ordinary skill in the art. Such modifications may be made post-SELEX process (modification of previously identified unmodified ligands) or by incorporation into the SELEX process.<br>
The nucleic acid ligands of the invention are prepared through the SELEX methodology that is outlined above and thoroughly enabled in the SELEX applications incorporated herein by reference in their entirety.<br>
The tenascin-C aptamers of the invention bind to the heparin binding site of the tenascin-C COOH terminus.<br>
In certain embodiments of the present invention, the nucleic acid ligands to tenascin-C described herein are useful for diagnostic purposes and can be used to image pathological conditions (such as human tumor imaging). In addition to diagnosis, the tenascin-C nucleic acid ligands are useful in the prognosis and monitoring of disease conditions in which tenascin-C is expressed.<br>
Diagnostic agents need only be able to allow the user to identify the presence of a given target at a particular locale or concentration. Simply the ability to form binding pairs with the target may be sufficient to trigger a positive signal for diagnostic purposes. Those skilled in the art would be able to adapt any tenascin-C nucleic acid ligand by procedures known in the art to incorporate a marker in order to track the presence of the nucleic acid ligand. Such a marker could be used in a number of diagnostic procedures, such as detection of primary and metastatic tumors and athersclerotic lesions. The labeling markers exemplified herein are technetium-99m and mln: however, other markers such as additional radionuclides. magnetic compounds, fluorophores. biotin, and the like can be conjugated to the tenascin-C nucleic acid ligand for imaging in an in vivo or ex vivo setting disease conditions in which tenascin-C is expressed (e.g., cancer, atherosclerosis, and psoriasis). The marker may be covalently bound to a variety of positions on the tenascin-C nucleic acid<br>
such as to an exocyclic amino group on the base, the 5-position of a pyrimidine<br>
nucleotide, the 8-position of a purine nucleotide, the hydroxyl group of the phosphate, or a hydroxyl group or other group at the 5' or 3' terminus of the tenascin-C nucleic acid ligand. In embodiments where the marker is technetium-99m of 1HIn, preferably it is bonded to the 5' or 3' hydroxyl of the phosphate group thereof or to the 5 position of a modified pyrimidine. In the most preferred embodiment, the marker is bonded to the 5' hydroxyl of the phosphate group of the nucleic acid ligand with or without a linker. In another embodiment, the marker is conjugated to the nucleic acid ligand by incorporating a pyrimidine containing a primary amine at the 5 position, and use of the amine for conjugation to the marker. Attachment of the marker can be done directly or with the utilization of a linker. In the embodiment where technetium-99m or " 'in is used as the marker, the preferred linker is a hexylamine linker as shown in Figure 10.<br>
In other embodiments, the tenascin-C nucleic acid ligands are useful for the delivery of therapeutic compounds (including, but not limited to, cytotoxic compounds, immune enhancing substances and therapeutic radionuclides) to tissues or organs expressing tenascin-C. Disease conditions in which tenascin-C may be expressed include, but are not limited to, cancer, atherosclerosis, and psoriasis. Those skilled in the art would be able to adapt any tenascin-C nucleic acid ligand by procedures known in the art to incorporate a therapeutic compound in a complex. The therapeutic compound may be covalently bound to a variety of positions on the tenascin-C nucleic acid ligand, such as to an exocyclic amino group on the base, the 5-position of a pyrimidine nucleotide, the 8-position of a purine nucleotide, the hydroxyl group of the phosphate, or a hydroxyl group or other group at the 5' or 3' terminus of the tenascin-C nucleic acid ligand. In the preferred embodiment, the therapeutic agent is bonded to the 5' amine of the nucleic acid ligand. Attachment of the therapeutic agent can be done directly or with the utilization of a linker. In embodiments in which cancer is the targeted disease, 5-fluorodeoxyuracil or other nucleotide analogs known to be active against tumors can be incorporated internally into existing U's within the tenascin-C nucleic acid ligand or can be added internally or conjugated to either terminus either directly or through a linker. In addition, both pyrimidine analogues 2'2'-difluorocytidine and purine analogues (deoxycoformycin) can be incorporated. In addition. United States Application Serial No. 08/993,765. filed December 18, 1997, incorporated herein by reference in its entirety, describes, inter alia, nucleotide-based prodrugs comprising nucleic acid ligands directed to a<br>
%inor, for example tenascin-C, for precisely localizing chemoradiosensitizers, and<br>
radiosensitizers and radionuclides and other radiotherapeutic agents to the tumor.<br>
It is also contemplated that both the marker and therapeutic agent may be associated with the tenascin-C nucleic acid ligand, such that detection of the disease condition and delivery of the therapeutic agent is accomplished together in one aptamer or as a mixture of two or more different modified versions of the same aptamer. It is also contemplated that either or both the marker and/or the therapeutic agent may be associated with a non-immunogenic, high molecular weight compound or lipophilic compound, such as a liposome. Methods for conjugating nucleic acid ligands with lipophilic compounds or non-immunogenic compounds in a diagnostic or therapeutic complex are described in United States Patent Application Serial No. 08/434,465, filed May 4,1995, entitled "Nucleic Acid Ligand Complexes," which is incorporated herein in its entirety.<br>
The therapeutic or diagnostic compositions described herein may be administered parenterally by injection (e.g., intravenous, subcutaneous, intradermal, intralesional), although other effective administration forms, such as intraarticular injection, inhalant mists, orally active formulations, transdermal iontophoresis or suppositories, are also envisioned. They may also be applied locally by direct injection, can be released from devices, such as implanted stents or catheters, or delivered directly to the site by an infusion pump. One preferred carrier is physiological saline solution, but it is contemplated that other pharmaceutically acceptable carriers may also be used. In one embodiment, it is envisioned that the carrier and the tenascin-C nucleic acid ligand complexed with a therapeutic compound constitute a physiologically-compatible, slow release formulation. The primary solvent in such a carrier may be either aqueous or non-aqueous in nature. In addition, the carrier may contain other pharmacologically acceptable excipients for modifying or maintaining the pH, osmolarity, viscosity, clarity, color, sterility, stability, rate of dissolution, or odor of the formulation. Similarly, the carrier may contain still other pharmacologically-acceptable excipients for modifying or maintaining the stability, rate of dissolution, release, or absorption of the tenascin-C nucleic acid ligand. Such excipients are those substances usually and customarily employed to formulate dosages for parental administration in either unit dose or multi-dose form.<br>
Once the therapeutic or diagnostic composition has been formulated, it may be stored in sterile vials as a solution, suspension, gel, emulsion, solid, or dehydrated or lyophilized powder. Such formulations may be stored either in ready to use form or requiring reconstitution immediately prior to administration. The manner of administering formulations containing<br>
taa&amp;cin-C nucleic acid ligands for systemic delivery may be via subcutaneous, intramuscular,<br>
intravenous, intraarterial, intranasal or vaginal or rectal suppository.<br>
The following examples are provided to explain and illustrate the present invention and are not to be taken as limiting of the invention. Example 1 describes the materials and experimental procedures used in Example 2 for the generation of RNA ligands to tenascin-C. Example 2 describes the RNA ligands to tenascin-C and the predicted secondary structure of a selected nucleic acid ligand. Example 3 describes the determination of minimal size necessary for high affinity binding of a selected nucleic acid ligand, and substitution of 2'-OH purines with 2'-OMe purines. Example 4 describes the biodistribution of Tc-99m labeled tenascin-C nucleic acid ligands in tumor-bearing mice. Example 5 describes the use of a fluorescently labeled tenascin-C nucleic acid ligand to localize tenascin-c within tumor tissue. Example 6 describes detection of tumors in vivo by Aptamer TTA1 (also known as GS764I). Example 7 describes alternative labeling using inln.<br>
EXAMPLES<br>
Example 1. Use of SELEX to obtain nucleic acid ligands to tenascin-C and to U251 glioblastoma cells.<br>
Materials and Methods<br>
Tenascin-C was purchased from Chemicon (Temecula, CA). Single-stranded DNA-primers and templates were synthesized by Operon Technologies Inc. (Alameda. CA).<br>
The SELEX-process has been described in detail in the SELEX Patent Applications. In brief, double-stranded transcription templates were prepared by Klenow fragment extension of 40N7a ssDNA:<br>
5'- TCGCGCGAGTCGTCTG [4 ON] CCGCATCGTCCTCCC 3' (SEQ ID NO:1) using the 5N7 primer:<br>
5'-TAATACGACTCACTATAGGGAGGACGATGCGG-3' (SEQ ID NO:2) which contains the T7 polymerase promoter (underlined). RNA was prepared with T7 RNA polymerase as described previously in Fitzwater and Polisky (1996) Methods Enzymol. 267: 275-301, incorporated herein by reference in its entirety. All transcription reactions were performed in the presence of pyrimidine nucleotides that were 2'-fluoro (2'-F) modified on the sugar moiety. This substitution confers enhanced resistance to ribonucleases that utilize the 2'-hydroxyl moiety for cleavage of the phosphodiester bond. Specifically, each transcription mixture contained 3.3 mM 2'-F UTP and 3.3 mM 2'-F CTP along with 1 mM<br>
4jg^P and ATP. The initial randomized RNA library thus produced comprised 3 x 1014<br>
molecules. The affinities of individual ligands for tenascin-C were determined by standard methods using nitrocellulose filter partitioning (Tuerk and Gold (1990) Science 249[4968):505-10).<br>
For each round of SELEX, Lumino plates (Labsystems, Needham Heights, MA) were coated for 2 hours at room temperature with 200 ^1 Dulbecco's PBS containing tenascin-C concentrations as shown in Table 1. After coating, wells were blocked using HBSMC+ buffer [20 mM Hepes, pH 7.4,137 mM NaCl, 1 mM CaCl2, 1 mM MgCl2 and 1 g/liter human serum albumin (Sigma, fraction V) for rounds 1 to 6 while for rounds 7 and 8 wells were blocked HBSMC+ buffer containing 1 g/liter casein (I-block; Tropix). Binding and wash buffer consisted of HBSMC+ buffer containing 0.05% Tween 20. For each SELEX round, RNA was diluted into 100 ul of binding buffer and allowed to incubate for 2 hours at 37C in the protein coated wells that were pre-washed with binding buffer. After binding, six washes of 200 ul each were performed. Following the wash step, the dry well was placed on top of a 95C heat block for 5 minutes. Standard AMV reverse transcriptase reactions (50 ul) were performed at 48C directly in the well and the reaction products utilized for standard PCR and transcription reactions. Two synthetic primers 5N7 (see above) and 3N7a: 5'-TCGCGCGAGTCGTCTG-3' (SEQ ID NO:3) were used for these template amplification and reverse transcription steps.<br>
For cell SELEX, U251 human glioblastoma cells (Hum. Hered. (1971) 2J.:238) were grown to confluence in Dulbecco's Modified Eagle's Medium supplemented with 10% fetal calf serum (GIBCO BRL. Gaithersburg. MD) on six-well tissue culture plates (Becton Dickinson Labware, Lincoln Park, NJ) and washed three times using Dulbecco's PBS supplemented with CaCl2 (DPBS, GIBCO BRL) buffer. RNA labeled internally by transcription (Fitzwater (1996) supra) was incubated with the cells at 37C for one hour. The labeled RNA was then removed, and the cells were washed six times for ten minutes each at 37C with DPBS. DPBS containing 5 mM EDTA was then added and incubated with the cells for 30 minutes to elute bound RNAs that remained after the washing steps. This RNA was quantitated by a standard liquid scintillation counting protocol and amplified using RT-PCR.<br>
Binding assays for the U251 cells. Internally labeled RNA was incubated at increasing concentrations with confluent U251 cells in six-well tissue culture plates (Becton Dickinson Labware, Lincoln Park, NJ) at 37C for 60 min. Unbound RNA was washed away<br>
usual; three 10 minute washes with DPBS+ CaCl2 at 37C, and bound RNA was collected by<br>
disrupting the cells using Trizol (Gibco BRL, Gaithersburg, MD). Bound RNA was quantitated by liquid scintillation counting.<br>
Cloning and Sequencing. Amplified affinity enriched oligonucleotide pools were purified on an 8% polyacrylamide gel, reverse transcribed into ssDNA and the DNA amplified by the polymerase chain reaction (PCR) using primers containing BamHl and Hindlll restriction endonuclease sites. PCR fragments were cloned, plasmids prepared and sequence analyses performed according to standard techniques (Sambrook et al. (1989) Molecular Cloning: A Laboratory Manual. 2nd Ed. 3 vols., Cold Spring Harbor Laboratory Press, Cold Spring Harbor).<br>
Example 2. RNA ligands to tenascin-C.<br>
Nucleic Acid Ligands to U251 cells were obtained by the SELEX process and are described in United States Patent Application Serial No. 08/434,425, entitled "Systematic Evolution of Ligands by Exponential Enrichment: Tissue SELEX," filed May 3,1995, now United States Patent No. 5,789,157. Subsequently it was determined that the ligands that were obtained were tenascin-C nucleic acid ligands.<br>
To obtain oligonucleotide ligands against human tenascin-C, eight rounds of SELEX were performed using the randomized nucleotide library as described above in Materials and Methods. RNA and protein input into each round is shown in Table 1. After 8 rounds of SELEX, the affinity of the oligonucleotide pool for tenascin-C was 10 nM, and this affinity did not increase with additional SELEX rounds.<br>
To obtain ligands to U251 glioblastoma cells, nine rounds of SELEX were performed using the randomized nucleotide library. After nine rounds of binding to U251 cells and EDTA elution, rounds 3, 5 and 9 were tested for their ability to bind to U251 cells. Figure 1 shows that as the number of SELEX rounds increases, the amount of bound RNA also increases at a particular concentration. Because of the complexity of the target tissue, it was not possible to estimate the affinity of the oligonucleotide pools for the unknown target molecules(s) on these cells.<br>
The E9 pool (nine rounds of binding and EDTA elution from U251 cells) was then used as a starting point for a SELEX against purified tenascin-C. Two rounds of SELEX using purified tenascin-C were performed as described above. Input protein and RNA concentrations for two rounds of SELEX (E9P1 and E9P2) are described in Table 2.<br>
In summary, three different SELEX experiments were performed: an experiment<br>
using purified tenascin-C as the target, an experiment using U251 glioblastoma cells as the target, and an experiment in which the SELEX pool from the U251 glioblastoma cells was used to initiate a SELEX experiment using purified tenascin-C as the target.<br>
All three SELEX experiments were analyzed by cloning and sequencing ligands from round 8 of the purified tenascin-C SELEX ("TN" sequences), from round 9 of the U251 cell SELEX ("E9" sequences), and from round 2 of the U251/tenascin-C hybrid SELEX ("E9P2" sequences). The sequences of 34 unique clones are shown in Table 3, and are divided into two major groups: tenascin-C ligands ("TN" and "E9P2" sequences) and U251 cell ligands ("E9" ligands). Among the tenascin-C ligands, the majority of the clones (65 total) represent one of two distinct sequence classes designated Family I and Family II (Figure 1). Examination of the variable region of the 12 clones in Family I revealed 7 unique sequences that are related through the consensus sequence GACNYUUCCNGCYAC (SEQ ID NO: 12). Examination of the variable region of the 18 clones in Family II revealed sequences that share a consensus sequence CGUCGCC (Table 3). The E9 sequences could be grouped into a related set by virtue of conserved GAY and CAU sequences within the variable regions. The remaining sequences did not appear related to other sequences and were classified as orphans. Three sequences predominate, with E9P2-1, E9P2-2, and TN9 represented 14,16, and 10 times respectively. In the "Orphan" category, one sequence, TNI8, was represented twice. Overall, these data represent a highly enriched sequence pool.<br>
Most individuals displayed low nanomolar dissociation constants, with the three most prevalent sequences, TN9 and E9P2-1 and -2. having the highest affinities at 5 nM 2nM. and 8 nM. (Table 3). These results indicate that the U251 cell SELEX is a repository for aptamers against tenascin-C, and that only two rounds of SELEX were required to isolate the tenascin-specific ligands from the cell SELEX pool. Oligonucleotide ligands against other proteins can be similarly isolated from the E9 pool using purified protein targets.<br>
Example 3. Determination of minimal size of TN9, and substitution of 2'-OH purines with 2'-OMe purines: synthesis of aptamer TTA1.<br>
Oligonucleotide synthesis procedures were standard for those skilled in the art (Green et al. (1995) Chem Biol 2(101:683-951. 2'-fluoro pyrimidine phosphoramidite monomers were obtained from JBL Scientific (San Luis Obispo, CA); 2'-OMe purine, 2'-OH purine, hexylamine. and (CH2CH2O)6 monomers, along with the dT polystyrene solid support, were<br><br>
obtained from Glen Research (Sterling, VA). Aptamer affinities were determined using<br>
nitrocellulose filter partitioning (Green et al., supra).<br>
TN9 was chosen for further analysis based on its high affinity for tenascin-C. We first searched for a minimal sequence necessary for high affinity binding. Using standard techniques (Green et al, supra), it was discovered that nucleotides 3'of nucleotide 55 were required for binding to tenascin-C, while no nucleotides could be removed from the 5' end without loss of affinity. To further decrease the TN9's length from 55 nucleotides and retain high affinity binding, we then attempted to define internal deletions of TN9. The first 55 nucleotides of TN9, along with the first 55 nucleotides of related family II ligands TN7, TN21, and TN41, were input into a computer algorithm to determine possible RNA secondary structure foldings (mfold 3.0, accessed at http://www.ibc.wustl.edu/~zuker/. M. Zuker, D.H. Mathews &amp; D.H. Turner. Algorithms and Thermodynamics for RNA Secondary Structure Prediction: A Practical Guide. In: RNA Biochemistry and Biotechnology, J. Barciszewski &amp; B.F.C. Clark, eds., NATO ASI Series, Kluwer Academic Publishers, (1999)). Among many potential RNA foldings predicted by the algorithm, a structure common to each oligonucleotide was found. This structure, represented by oligonucleotide TTA1 in Figure 2, contains three stems that meet at a single junction, a so-called 3-stem junction. This folding places the most highly conserved nucleotides of family II oligonucleotides at the junction area. In comparing TN9, TN7, TN21, and TN41, the second stem was of variable length and sequence, suggesting that extension of the second stem is not required for binding to tenascin-C. Testing this hypothesis on TN9. we found that nucleotides 10-26 could be replaced with an ethylene glycol linker, (CH2CH2O)6- The linker serves as a substitute loop and decreases the size of the aptamer. Additionally, four-nucleotide loops (CACU or GAGA) that replace nucleotides 10-26 produce sequences with high affinity for tenascin-C. It would be well within one skilled in the art to determine other nucleotide loops or other spacers that could replace nucleotides 10-26 to produce sequences with high affinity for tenascin-C.<br>
To increase protection against nuclease activity, purine positions that could be substituted with the corresponding 2'-OMe purines were located. The oligonucleotide was arbitrarily divided into five sectors and all purines within each sector were substituted by the corresponding 2'-OMe purine nucleotide. a total of five oligonucleotides (Table 4, Phase I syntheses). The affinity of each oligonucleotide for tenascin-C was determined, and it was found that all purines within sectors 1.3 and 5 could be substituted without appreciable loss in<br>
Within sectors 2 and 4, individual purines were then substituted with 2'-OMe<br>
purines and the effect of affinity was measured (Table 4, Phase III syntheses). From these experiments, it was deduced that substitution of nucleotides G9, G28, G31, and G34, with 2'-OMe G causes loss in affinity for tenascin-C. Therefore these nucleotides remain as 2'-OH purines in the aptamer TTA1.<br>
The aptamer TTA1 (Table 4) was then synthesized with the (CH2CH2O)6 (Spacer 18) linker, a 3'-3' dT cap for exonuclease protection, a 5' hexylamine (Table 4), and all purines as 2'-OMe except the 5 Gs indicated in Table 4. A non-binding control aptamer, TTA1 .NB, was generated by deleting 5 nucleotides at the 3' end to produce TTA1.NB. TTA1 binds to tenascin-C with an equilibrium dissociation constant (Kd) of 5 nM, while TTA1.NB has a Kd of &gt; 5 uM for tenascin-C.<br>
Nucleotides 10-26 can be replaced by a non-nucleotide ethylene glycol linker. It is therefore likely that TTA1 can be synthesized in two separate pieces, where a break is introduced at the position of the ethylene glycol linker and new 5' and 3' ends are introduced. Subsequent to synthesis, the two molecules will incubated together to allow hybrid formation. This method allows introduction of additional amine groups as well as nucleotides at the new 5' and 3' ends. The new functionalities could be used for bioconjugation. In addition, two-piece synthesis results in increased chemical synthetic yield due to shortening the length of the molecules.<br>
Example 4. Redistribution of Tc-99m labeled aptamers in tumor-bearing mice.<br>
Aptamer biodistribution was tested by conjugating a Tc-99m chelator (Hi 15; Hilger ci al. (1998) Tet Lett 39:9403-9406) to the 5' end of the oligonucleotide as shown in Figure 2, and radiolabeling the aptamer with Tc-99m. The aptamer in its Tc-99m labeled form is shown in Figure 10. TTA1 and TTA1.NB were conjugated to Hi|5 at 50 mg/ml aptamer in 30% dimethylformamide with 5 molar equivalents of Hiis-N-hydoxysuccinirnide, buffered in 100 mM Na Borate pH 9.3, for 30 minutes at room temperature. The aptamer in its Tc-99m labeled form is shown in Figure 10. Reversed phase HPLC purification yielded Hiis-TTAl and Hiis-TTAl.NB. The oligonucleotides were then labeled with Tc-99m in the following manner: to 1 nmole Hil5-aptamer was added 200 uL of 100 mM sodium phosphate buffer, pH 8.5, 23 mg/mL NaTartrate, and 50 uL Tc-99m pertechnetate (5.0 mCi) eluted from a Mo-99 column (Syncor, Denver) within 12 hours of use. The labeling reaction was initiated by<br>
the addition of 10 uL 5 mg/mL SnCl2. The reaction mixture was incubated for 15 minutes at<br>
90 C. The reaction was separated from unreacted Tc-99m by spin dialysis through a 30,000 MW cut-off membrane (Centrex, Schleicher &amp; Scheull) with two 300 uL washes. This labeling protocol results in 30-50% of the added 99mTc being incorporated with a specific activity of 2-3 mCi/nmole RNA. The Tc-99m is bound through the 5'G as shown in Figure 5.<br>
For biodistribution experiments, U251 xenograft tumors were prepared as follows: U251 cells were cultured in Dulbeccos' Modified Eagle's Medium supplemented with 10% v/v fetal calf serum (Gibco BRL, Gaithersburg, MD). Athymic mice (Harlan Sprague Dawley, Indianapolis, IN) were injected subcutaneous! y with 1x106 U251 cells. When the tumors reached a size of 200-300 mg (1-2 weeks), Tc-99m labeled aptamer was injected intravenously at 3.25 mg/kg. At indicated times, animals were anesthetized using isoflurane (Fort Dodge Animal Health, Fort Dodge, IA), blood was collected by cardiac puncture, and the animal was sacrificed and tissues were harvested. Tc-99m levels were counted using a gamma counter (Wallac Oy, Turku, Finland). Aptamer uptake into tissues was measured as the % of injected dose per gram of tissue (%ID/g).<br>
Images of mice were obtained using a gamma camera. Mice were placed onto the camera (Siemens, LEM+) under anesthesia (isoflurane). Data were collected (30 sec to 10 minutes) and analyzed using Nuclear MAC software version 3.22.2 (Scientific Imaging. CA) on a Power MAC G3 (Apple Computer, CA).<br>
Biodistribution experiments. Table 5, indicated rapid and specific uptake of the aptamer into tumor tissue; the non-binding aptamer does not remain in the tumor. Blood levels of Tc-99m also cleared rapidly. After three hours, Tc-99m levels brought into the tumor using Hii5-TTAl had a very long half life (&gt; 18 hrs). This indicates that once the aptamer penetrates the tumor, the radiolabel carried with it remains in the tumor for long periods of time. Such data indicate that cytotoxic agents, including radionuclides and non-radioactive agents, conjugated to the aptamer will also remain in the tumor with long hall lives.<br>
Tc-99m radioactivity also appears in other tissues, notably the small and large intestines. The hepatobiliary clearance pattern seen here can be readily altered by those skilled in the art, for example by altering the hydrophilicity of the Tc-99m chelator. changing the chelator, or changing the radiometal/chelator pair altogether.<br>
Whole animal images were obtained using Tc-99m labeled Hiis-TTAl and at 3 hours<br>
post-injection. Images obtained from mice injected with Hi15-TTAl, but not from mice injected with Hi15-TTAl.NB, clearly show the tumor (Figure 3). Additional radioactivity is evident in gastrointestinal tract, as predicted by the biodistribution experiments.<br>
Example 5. Use of fluorescently labeled TTA1 to localize tenascin-C within tumor tissue.<br>
Materials and Methods.<br>
TTA1 and TTA1.NB were synthesized as described above. Succinimdyl Rhodamine-Red-X (Molecular Probes, Eugene, OR) was conjugated to the 5' amine of the aptamers as described above for H15-NHS conjugation. The Rhodamine-Red-X-conjugated aptamers, TTAl-Red and TTAl.NB-Red, were purified by reversed phase HPLC. U251 cell culture and tumor growth in nude mice were as described above. Five nmol of TTA1 -Red or TTAl.NB-Red were injected intravenously into nude mice and at the desired time the animal was placed under anesthesia, perfused with 0.9% NaCl, and sacrificed. The tumor was excised and placed in formalin. After 24 hr in formalin, 10 uM sections were cut and Rhodamine-Red-X was detected using a fluorescence microscope (Eclipse E800, Nikon, Japan).<br>
Results: TTAl-Red has identical affinity for tenascin-C as the unconjugated parent aptamer, TTA1, at 5 nM. We compared tumor fluorescence levels of TTAl-Red and TTAl.NB-Red 10 min post-injection. The binding aptamer, TTAl-Red. strongly stains the tumor but not adjacent tissue (Figure 4). In contrast, only tissue auto-fluorescence is detected with TTA1 .NB-Red. These results demonstrate the utility of the aptamer in fluorescent detection of tenascin-C in vivo, and the aptamer may be similarly used for staining tissues sections ex vivo.<br>
Example 6. Detection of Tumors In Vivo by Aptamer TTA1 (now also known as GS7641): additional tumor types.<br>
Aptamer labeling, biodistribution. and nude mouse tumor xenografts were performed as described in Example 4.<br>
Many human tumor types are known to express tenascin-C. To assess the ability of TTA1/GS7641 to target tumor types in addition to glioblastomas, human tumor cell lines were grown as tumors in nude mice. Tumor tissue was tested for expression of human<br>
nascin-C, and those tumors expressing human tenascin-C were tested for aptamer uptake.<br>
Figure 6 demonstrates aptamer uptake in several tumors, including glioblastoma, breast, colorectal, and rhabdomyosarcoma. Specific uptake into the tumor is demonstrated by the comparison between binding (TTA1/GS7641) and non-binding aptamer (TTA1 .NB). Note that KB, a xenograft that expresses mouse but not human TN-C, does not show tumor uptake. This experiment extends the observation of glioblastoma uptake into additional carcinomas and sarcomas, and further indicates that all tumors expressing human tenascin-C show uptake ofTTAl/GS7641.<br>
Example 7. Alternative Labeling Using In-Ill.<br>
Tumor xenograft and biodistribution studies were performed as described in Example 4. To couple DTPA and DOTA to TTA1/GS764L the cyclic anhydride of each was incubated with the amine-containing TTA1/GS7641 under neutral pH conditions using standard methods. The structures of the DTPA and conjugates are shown in Figure 8 and Figure 9, respectively, where each has been labeled with In-111. The DOTA conjugate has the identical linkers as for DTPA. DOTA- and DTPA-conjugates were labeled with In-111 by incubation at 95C in 0.5 M NaOAc, pH 5.5 for 30 min. After removal of unincorporated radiolabel by spin dialysis over a 30K cut-off membrane, radiolabeled aptamer was transferred into phosphate-buffered saline for injection into tumor-bearing mice.<br>
The biodistribution of In-11 labeled aptamer is markedly different from the Tc-labeled formulation described in Example 4. Figure 7 shows that, relative to Tc-99m-labeled TTA1/GS7641, In-111-labeled TTA1/GS7641 radioactivity in the intestines is greatly reduced, with a concomitant increase in liver and kidney uptake. This experiment indicates that the chemical properties of the chelator have a large effect on distribution of the radiolabel of TTA1/GS7641 within a living animal. Biodstribution patterns that are different from that of Hil5-TTAl/GS7641 may useful for targeting tumors under certain clinical conditions where hepatobiliary clearance is undesired. Such conditions include but are not limited to radiotherapy applications as well as imaging of intestines, prostate and other abdominal regions.<br>
Tenascin-C SELEX RNA and protein input.      <br>
(Table Removed)<br>
Table 2. Cell SELEX/tenascin-C SELEX RNA and protein input<br><br>
(Table Removed)<br>
Table 3: Tenascin-C	Sequences: purified protein SELEX (tenascin sequences) and U251 cell SELEX+purified protein SELEX (E9P2 sequence)<br>
 (Table Removed)<br>
Table 4: 2'-OMe Substitutions, Internal Deletions, TTA1, and TTA1.NB <br>
(Table Removed)<br>
Table 5: Biodistribution of Tc-99m-TTAl and -TTA1.NB <br>
(Table Removed)<br><br><br><br><br>
What is claimed is:<br>
1. A nucleic acid ligand to tenascin-C identified according to the method comprising:<br>
a)	contacting a candidate mixture of nucleic acids with tenascin-C, wherein nucleic<br>
acids having an increased affinity to tenascin-C relative to the candidate mixture may be<br>
partitioned from the remainder of the candidate mixture;<br>
b)	partitioning the increased affinity nucleic acids from the remainder of the candidate<br>
mixture;<br>
c)	amplifying the increased affinity nucleic acids to yield a mixture of nucleic acids<br>
enriched for nucleic acids with relatively higher affinity and specificity for binding to<br>
tenascin-C, whereby a nucleic acid ligand of tenascin-C may be identified.<br><br>
2.	The nucleic acid ligand of claim 1 wherein said candidate mixture of nucleic acids<br>
is comprised of single stranded nucleic acids.<br>
3.	The nucleic acid ligand of claim 2 wherein said single stranded nucleic acids are<br>
ribonucleic acids.<br>
4.	The nucleic acid ligand of claim 3 wherein said candidate mixture of nucleic acids<br>
comprises 2'-F (2'-fluoro) modified ribonucleic acids.<br>
5.	A purified and isolated non-naturally occurring nucleic acid ligand to tenascin-C.<br>
6.	The purified and isolated non-naturally occurring nucleic acid ligand of claim 5<br>
wherein said nucleic acid ligand is single stranded.<br>
7.	The purified and isolated non-naturally occurring nucleic acid ligand of claim 6<br>
wherein said nucleic acid ligand is RNA.<br>
8.	The purified and isolated non-naturally occurring RNA ligand of claim 7 wherein<br>
said ligand is comprised of 2'-fluoro (2'-F) modified nucleotides.<br>
9.	The purified and non-naturally occurring RNA ligand of claim 8 wherein said<br>
ligand is selected from the group consisting of the sequences as set forth in Tables 3 and 4<br>
and Figure 2.<br>
10.	A method of identifying nucleic acid ligands to tenascin-C. the method<br>
comprising:<br>
a)	contacting a candidate mixture of nucleic acids with tenascin-C, wherein nucleic<br>
acids having an increased affinity to tenascin-C relative to the candidate mixture may be<br>
partitioned from the remainder of the candidate mixture;<br>
b)	partitioning the increased affinity nucleic acids from the remainder of the candidate<br>
mixture;<br><br>
c) amplifying the increased affinity nucleic acids to yield a mixture of nucleic acids<br>
enriched for nucleic acids with relatively higher affinity and specificity for binding to tenascin-C, whereby a nucleic acid ligand of tenascin-C may be identified.<br>
11.	The method of claim 10 further comprising:<br>
d) repeating steps a), b), and c).<br>
12.	The method of claim 10 wherein said candidate mixture of nucleic acids is<br>
comprised of single stranded nucleic acids.<br>
13.	The method of claim 12 wherein said single stranded nucleic acids are<br>
ribonucleic acids.<br>
14.	The method of claim 13 wherein said nucleic acids are 2'-F (21- fluoro) modified<br>
ribonucleic acids.<br>
15.	A complex for use in in vivo diagnostics comprising a tenascin-C nucleic acid<br>
ligand and a marker.<br>
16.	The complex of claim 15 wherein said tenascin-C nucleic acid ligand is identified<br>
by the method comprising:<br><br>
a)	contacting a candidate mixture of nucleic acids with tenascin-C, wherein nucleic<br>
acids having an increased affinity to tenascin-C relative to the candidate mixture may be<br>
partitioned from the remainder of the candidate mixture;<br>
b)	partitioning the increased affinity nucleic acids from the remainder of the candidate<br>
mixture;<br>
c)	amplifying the increased affinity nucleic acids to yield a mixture of nucleic acids<br>
enriched for nucleic acids with relatively higher affinity and specificity for binding to<br>
tenascin-C, whereby a nucleic acid ligand of tenascin-C may be identified.<br><br>
17.	The complex of claim 16 wherein said candidate mixture of nucleic acids is<br>
comprised of single stranded nucleic acids.<br>
18.	The complex of claim 17 wherein said single stranded nucleic acids are<br>
ribonucleic acids.<br>
19.	The complex of claim 18 wherein said candidate mixture of nucleic acids<br>
comprises 2'-F (2'-fluoro) modified ribonucleic acids.<br>
20.	The complex of claim 15 wherein said marker is selected from the group<br>
consisting of radionuclides, fluorophores, magnetic compounds, and biotin.<br>
21. The complex of claim 20 wherein said radionuclide is selected from the group consisting of technetium-99m (Tc-99m), Re-188, Cu-64, Cu-67, F-18,1251,131I, H1In,32P, and<br>
22.	The complex of claim 21 wherein said marker is technetium-99m.<br>
23.	The complex of claim 22 further comprising a linker.<br>
24.	The complex of claim 23, wherein said linker has the structure<br>
(Formula Removed)<br>
25. The complex of claim 24, wherein said complex is <br>
(Formula Removed)<br>
Hexylamine linker<br>
AllA's = 2'-OMe<br>
All Gs, except as indicated, are 2'-OMe modified<br>
AllCsare2'-F modified<br>
AHUsare2'-F modified<br>
26.	A method for the preparation of a complex comprised of a tenascin-C nucleic<br>
acid ligand and a marker, said method comprising:<br>
a)	contacting a candidate mixture of nucleic acids with tenascin-C, wherein nucleic<br>
acids having an increased affinity to tenascin-C relative to the candidate mixture may be<br>
partitioned from the remainder of the candidate mixture;<br>
b)	partitioning the increased affinity nucleic acids from the remainder of the candidate<br>
mixture;<br>
c)	amplifying the increased affinity nucleic acids to yield a mixture of nucleic acids<br>
enriched for nucleic acids with relatively higher affinity and specificity for binding to<br>
tenascin-C; and<br>
d)	covalently linking said identified tenascin-C nucleic acid ligand with a marker.<br>
27.	The method of claim 26 wherein said marker is selected from the group<br>
consisting of radionuclides, fluorophores, magnetic compounds, and biotin.<br>
28.	The method of claim 27 wherein said radionuclide is selected from the group<br>
consisting of Tc-99m, Re-188, Cu-64, Cu-67, F-18, I251,131I, mm,32P, and 186Re.<br>
29.	The method of claim 28 wherein said marker is Tc-99m.<br>
30.	The method of claim 29 further comprising a linker.<br>
31.	The complex of claim 30, wherein said linker has the structure<br>
(Formula Removed)<br>
The complex of claim 31, wherein said complex is<br>
(Formula Removed) <br>
Hexylamine linker '<br><br>
AllA's = 2'-OMe<br>
All Gs, except as indicated, are 2'-OMe modified<br>
AllCsare2'-F modified<br>
All Us are 2 '-F modified<br>
33. A method for detecting the presence of a disease that is expressing tenascin-C in a biological tissue which may contain said disease comprising;<br>
a)	identifying a nucleic acid ligand from a candidate mixture of nucleic acids, said<br>
nucleic acid ligand being a ligand of tenascin-C, by the method comprising<br>
i) contacting a candidate mixture of nucleic acids with tenascin-C. wherein nucleic acids having an increased affinity to tenascin-C relative to the candidate mixture may be partitioned from the remainder of the candidate mixture;<br>
ii) partitioning the increased affinity nucleic acids from the remainder of the candidate mixture;<br>
iii) amplifying the increased affinity nucleic acids to yield a mixture of nucleic acids with relatively higher affinity and specificity for binding to tenascin-C, whereby a nucleic acid ligand of tenascin-C is identified;<br>
b)	attaching a marker that can be used in in vivo diagnostics to said nucleic acid<br>
ligand identified in step iii) to form a marker-nucleic acid ligand complex;<br>
c)	exposing a tissue which may contain said tumor to said marker-nucleic acid ligand<br>
complex; and<br>
d) detecting the presence of said marker-nucleic acid ligand in said tissue, whereby i<br>
disease expressing tenascin-C is identified.<br>
34.	The method of claim 33 wherein said marker-nucleic acid ligand further<br>
comprises a linker.<br>
35.	The method of claim 34 wherein said marker is selected from the group<br>
consisting of radionuclides, fluorophores, magnetic compounds, and biotin.<br>
36.	The method of claim 35 wherein said radionuclide is selected from the group<br>
consisting of Tc-99m, Re-188, Cu64, Cu67, F-18,1251,13II, H1In,32P, and l86Re.<br>
37.	The method of claim 36 wherein said marker is technetium-99m.<br>
38.	The method of claim 37 wherein said linker<br>
(Formula Removed)<br>
39.	The method of claim 38 wherein said marker-nucleic acid ligand is<br>
(CH2CH2O)S linker<br>
(Formula Removed)<br>
Hexylamine linker'<br>
AllA's = 2'-OMe<br>
All Gs, except as indicated, are 2'-OMe modified<br>
All Cs are 2'-F modified<br>
AHUsare2'-F modified<br>
40.	The method of claim 33 further comprising attaching a therapeutic or diagnostic<br>
agent to said complex.<br>
41.	The method of claim 33 wherein said disease is selected from the group<br>
consisting of cancer, psoriasis, and atherosclerosis.<br>
42.	The method of claim 41 wherein said disease is cancer.<br>
43.	A method for delivering a therapeutic agent to a disease that is expressing<br>
tenascin-C comprising:<br>
covalently linking a tenascin-C nucleic acid ligand with a therapeutic agent to form a complex, and administering said complex to a patient.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTA0LWRlbC1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-104-del-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTA0LWRlbC1hc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-104-del-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTA0LWRlbC1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-104-del-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTA0LWRlbC1jb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-104-del-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTA0LWRlbC1jb3JyZXNwb25kZW5jZS1wby5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-104-del-correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTA0LWRlbC1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-104-del-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTA0LWRlbC1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-104-del-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTA0LWRlbC1mb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-104-del-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTA0LWRlbC1mb3JtLTE5LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-104-del-form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTA0LWRlbC1mb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-104-del-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTA0LWRlbC1mb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-104-del-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTA0LWRlbC1mb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-104-del-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTA0LWRlbC1ncGEucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-104-del-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTA0LWRlbC1wY3QtMzA0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-104-del-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTA0LWRlbC1wY3QtMzA2LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-104-del-pct-306.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTA0LWRlbC1wY3QtNDAxLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-104-del-pct-401.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTA0LWRlbC1wY3QtNDAyLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-104-del-pct-402.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTA0LWRlbC1wY3QtNDA5LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-104-del-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTA0LWRlbC1wY3QtNDE2LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-104-del-pct-416.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTA0LWRlbC1wZXRpdGlvbi0xMzcucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-104-del-petition-137.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTA0LWRlbC1wZXRpdGlvbi0xMzgucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-104-del-petition-138.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="217672-a-method-of-manufacturing-a-contact-lens-having-improved-visual-acvity.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="217674-an-improved-process-for-preparation-of-super-concentra-of-iron-fines-useful-for-making-ferrites.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>217673</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2002/00104/DEL</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>17/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>25-Apr-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>28-Mar-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>28-Jan-2002</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>GILEAD SCIENCES, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>333, LAKESIDE DRIVE, FOSTER CITY, CALIFORNIA 94404, U.S.A.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>BRIAN HICKE</td>
											<td>3255 6TH STREET, BOULDER, COLORADO 80304, U.S.A.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>STEPHEN WARREN</td>
											<td>1095 UTTICA AVENUE, BOULDER, COLORADO 80304, U.S.A.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>DAVID PARMA</td>
											<td>6140 HABITAT DRIVE, BOULDER, COLORADO 80301, U.S.A.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>LARRY GOLD</td>
											<td>1033 5TH STREET, BOULDER, COLORADO 80302, U.S.A.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US00/02167</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-01-28</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>09/364,902</td>
									<td>1999-07-29</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/217673-a-method-of-isolating-nucleic-acid-ligands-to-tenascin-c by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:00:15 GMT -->
</html>
