<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/217807-a-method-for-assaying-sequence-specific-hybridization by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:09:10 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 217807:&quot;A METHOD FOR ASSAYING SEQUENCE SPECIFIC HYBRIDIZATION &quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A METHOD FOR ASSAYING SEQUENCE SPECIFIC HYBRIDIZATION &quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention provides homogeneous assay methods for nucleic acid hybridization, detection and evaluation. The assay includes obtaining signals from a test sample both before and during the application of a voltage to the test sample and correlating the signals, each of which is indicative of the binding affinity of the probe and the target to each other. The assay enables determining an extent of matching between the probe and the target, as the voltage can be calibrated so as to destabilize significantly any hybridization except perfectly complementary hybridization. The signals whose magnitude is correlated with binding affinity can be electrical conductance and/or fluorescent intensity. Preferably, both signal pairs are measured and com...</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Field of Invention<br>
The invention relates to methods of for assaying sequence-specific hybridization.<br>
Description of Related Art<br>
It has been understood for a number of years that biological molecules can be isolated and characterized through the application of an electric field to a sample.<br>
Electrophoresis is perhaps the most well-known example of an isolation and characterization technique based on the influence of electric fields on biological molecules. In gel electrophoresis, a uniform matrix or gel is formed of, for example, polyacrylamide, to which an electric field is applied. Mixtures applied to one end of the gel will migrate through the gel according to their size and interaction with the electric field. Mobility is dependent upon the unique characteristics of the substance such as conformation, size and charge. Mobilities can be influenced by altering pore sizes of the gel, such as by formation of a concentration or pH gradient, or by altering the composition of the buffer (pH, SDS, DOC, glycine, salt). One- and two-dimensional gel electrophoresis are fairly routine procedures in most research laboratories. Target substances can be purified by passage through and/or physical extraction from the gel.<br>
A more recently developed process in which an electric field is applied to a biological sample is disclosed in U.S. Patent No. 5,824,477 to Stanley. The Stanley patent discloses a process for detecting the presence or absence of a predetermined nucleic acid sequence in a sample. The process comprises: (a) denaturing a sample double-stranded nucleic acid by means of a voltage applied to the sample in a solution by means of an electrode; (b) hybridizing the denatured nucleic acid with an oligonucleotide probe for the sequence; and (c) determining whether the hybridization has occurred. The Stanley patent discloses the application of an electric field to the sample to be assayed for the limited purpose of denaturing the target sequence.<br>
A more well-known type of hybridization assay is based on the use of fluorescent marking agents. In their most basic form, fluorescent intensity-based assays have typically comprised contacting a target with a fluorophore-containing probe, removing any unbound probe from bound probe, and detecting fluorescence in the washed sample. Homogeneous assays improve upon such basic assays, in that the former do not require a washing step or the provision of a non-liquid phase support.<br>
Some assays have employed intercalating fluorophores to detect nucleic acid hybridization, based on the ability of such fluorophores to bind between strands of nucleic acid in a hybridization complex.<br>
For example, U.S. Patent No. 5,824,557 to Burke et al. discloses a method and kit for detecting and quantitating nucleic acid molecules. A preferred embodiment relies on the intercalation of a dye into a double-stranded nucleic acid helix or single-stranded nucleic acid. The dye fluoresces after intercalation and<br>
the intensity is a direct measurement of the amount of nucleic acid present in the sample. While the method of Burke et al. is purported to be useful for measuring the amount of nucleic acid in a sample, the non-specific binding between intercalator and nucleic acid upon which the method is based renders the method impractical for detecting specific binding, particularly under conditions where non-target nucleic acid duplexes are present.<br>
U.S. Patent No. 5,814,447 to Ishiguro et al. discloses an assay which is purported to improve upon assays that rely on non-specific interaction between intercalating agents and nucleic acid duplexes, such as Burke et al. and an earlier assay described by Ishiguro et al. in Japanese Patent Public Disclosure No. 237000/1993. The earlier development comprised adding an intercalating fluorochrome having a tendency to exhibit increased intensity of fluorescence when intercalated to a sample solution before a specific region of a target nucleic acid was amplified by PCR, and measuring the intensity of fluorescence from the reaction solution at given time intervals to detect and quantitate the target nucleic acid before amplification. The '447 patent attempted to improve upon the earlier development by providing an assay having improved specificity, characterized in that the probe is a single-stranded oligonucleotide labeled with an intercalating fluorochrome which is to be intercalated into a complementary binding portion between a target nucleic acid and a single-stranded oligonucleotide probe.<br>
In the ongoing search for more sensitive, accurate and rapid assay techniques, one research group developed an assay comprising analyzing the effects of an electric field on the fluorescent; intensity of nucleic acid hybridization duplexes. See U.S. Patent Application Nos.<br>
08/807,901 and 08/870,370, respectively filed February 27, 1997 and June 6, 1997. The researchers indicated that the fluorescesnt intensity of a one base-pair mismatched duplex differed from that of a perfectly matched duplex. Thus, the applications purport to disclose a method for detecting a nucleotide sequence, wherein an electric field is applied to a liquid medium prior to or concurrently with a detecting step, and a change in an intensity of a fluorescent emission as a function of the electric field is detected as an indication of whether the probe is hybridized to a completely complementary nucleotide sequence or an incompletely complementary nucleotide sequence.<br>
Despite the foregoing developments, a need has continued to exist in the art for a simple, highly sensitive, effective and rapid method for analyzing interaction between nucleic acids and/or nucleic acid analogs.<br>
All references cited herein are incorporated herein by reference in their entireties.<br>
SUMMARY OP THE INVENTION<br>
The invention provides a method for assaying sequence-specific hybridization, said method comprising:<br>
providing a target comprising at least one nucleic acid sequence;<br>
providing a probe comprising a nucleic acid or nucleic acid analog sequence;<br>
adding said probe and said target to a hybridization medium to provide a test sample;<br>
applying a first stimulus to said test sample to provide a first stimulated test sample;<br>
detecting a first signal from said first stimulated test sample,  wherein said first signal is<br>
correlated with a binding affinity between said probe and said target;<br>
calibrating said first signal against a reference signal exhibited by a reference sample comprising at least one reference probe combined with said target, wherein relative to said target, each of said probe and said at least one reference probe is a different member selected from the group consisting of a perfect match, a one-base mismatch, a two-base mismatch, a three-base mismatch, a one-base deletion, a two-base deletion and a three-base deletion; and<br>
determining from said calibrating a first determination' of an extent of matching between said probe and said target;<br>
applying a second stimulus to said first stimulated test sample to provide a second stimulated test sample; and<br>
detecting a second signal from said second stimulated test sample, wherein said second signal is correlated with said binding affinity between said probe and said target;<br>
determining from said detecting of said second signal a second determination of said extent of matching between said probe and said target; and<br>
comparing said first determination and said second determination.<br>
Also provided is another method for assaying hybridization, said method comprising:<br>
providing a target comprising at least one nucleic acid sequence;<br>
providing a probe comprising a nucleic acid or nucleic acid analog sequence;<br>
adding said probe and said target to a hybridization medium to provide a test sample;<br>
measuring a first signal of a first condition of said test sample to provide a primary determination regarding hybridization between said probe and said target, wherein said first signal is correlated with hybridization between said probe and said target;<br>
measuring a second signal of a second condition of said test sample to provide a secondary determination regarding hybridization between said probe and said target, wherein said second signal is correlated with hybridization between said probe and said target, provided that when said first condition and said second condition are alike, a stimulus is applied to said test sample after measuring said first signal and before measuring said second signal, wherein said stimulus significantly affects imperfectly complementary hybridization between said probe and said target and does not significantly affect perfectly complementary hybridization between said probe and said target; and<br>
comparing said primary determination and said secondary determination to evaluate whether any inconsistency therebetween warrants retesting.<br>
In addition, the invention provides still another hybridization assay method comprising:<br>
providing a target comprising at least one nucleic acid sequence;<br>
providing a probe comprising a nucleic acid or nucleic acid analog sequence;<br>
adding said probe and said target to a hybridization medium to provides a test sample;<br>
measmring a pre-electrification fluorescent intensity of said test sample to provide a primary determination regarding hybridization between said probe and said target, wherein said pre-electrification fluorescent intensity is correlated with hybridization between said probe and said target;<br>
applying a voltage to said test sample;<br>
measuring a post-electrification fluorescent intensity of said test sample, during or after said voltage applying, to provide a secondary determination' regarding hybridization between said probe and said target, wherein said post-electrification fluorescent intensity is correlated with hybridization between said probe and said target; and<br>
comparing said primary determination and said secondary determination to evaluate whether any inconsistency therebetween warrants retesting.<br>
Also provided is a method for assaying sequence-specific hybridization, said method comprising:<br>
providing a target comprising at least one nucleic acid sequence;<br>
providing a probe comprising a nucleic acid or nucleic acid analog sequence;<br>
adding said probe and said target to a hybridization medium to provide a test sample;<br>
applying an electrical voltage to -said test sample;<br>
detecting a signal of said test sample during or after said applying of said electrical voltage,<br>
wherein said signal is correlated with a binding affinity between said probe and said target;<br>
calibrating said signal aqainst a reference signal exhibited by a reference sample comprising at least one reference probe combined with said target, wherein relative to said target, each of said probe and said at least one reference probe is a different member selected from the group consisting of a perfect match, a one-base mismatch, a two-base mismatch, a three-base mismatch, a one-base deletion, a two-base deletion and a three-base deletion; and<br>
determining from said calibrating an extent of matching between said probe and said target.<br>
This invention therefore provides a method for assaying sequence-specific hybridization, said method comprising:<br>
providing a target comprising at least one nucleic acid sequence;<br>
providing a probe comprising a nucleic acid or nucleic acid analog<br>
sequence;<br>
adding said probe, said target and intercalating agent to a hybridization medium as herein described to provide a test sample;<br>
applying a first stimulus as herein described to said test sample to provide a first stimulated test sample;<br>
detecting a first signal from said first stimulated test sample. Wherein said first signal is correlated with a binding affinity between said probe and said target;<br>
calibrating said first signal against a reference signal exhibited by a reference sample comprising at least one reference probe combined with said target, wherein relative to said target, each of said probe and said at least one reference probe is a different member selected from the group consisting of a perfect match, a one-base mismatch, a two-base mismatch, a three-base mismatch, a one-base deletion, a two-base deletion and a three-base deletion;<br>
determining from said calibrating a first determination of an extent of matching between said probe and said target;<br>
applying a second stimulus as herein described to said first stimulated test sample to provide a second stimulated test sample;<br>
detecting a second signal from said second stimulated test sample, wherein said second signal is correlated with said binding affinity between said probe and said target;<br>
determining from said detecting of said second signal a second determination of said extent of matching between said probe and said target; and comparing said first determination and said second determination, and<br>
optionally including the step of quantifying said binding affinity.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
The invention will be described in conjunction with the following drawings in which like reference numerals designate like elements and wherein:<br>
Figs. 1A and 1B are graphs of current as a function of time and complementarity;<br>
Figs. 1C and 1D are graphs of current as a function of temperature and complementarity;<br>
Figs. 2A, 2B, 2C, 3A and 3B are graphs of current as a function of temperature, complementarity and additional factors;<br>
Fig. 4  is a graph of current as a function of time and complementarity; and<br>
Figs. 5A, 5B, 5C and 6 are fluorescent intensity spectra.<br>
DETAILED DESCRIPTION OF PREFERRED EMBODIMENTS<br>
The invention provides a rapid, sensitive, environmentally friendly, and safe method for assaying binding between a target and a probe, wherein the target<br>
comprises a nucleic acid sequence or a nucleic acid analog sequence and the probe comprises a nucleic acid sequence or a nucleic acid analog sequence.<br>
Unlike certain prior art assays, the invention not only detects the presence of hybridization, but also provides qualitative and quantitative information regarding the nature of hybridization between a probe and target. Thus, the invention enables the practitioner to distinguish among a perfect match, a one base pair mismatch, a two base pair mismatch, a three base pair mismatch, a one base pair deletion, a two base pair deletion and a three base pair deletion.<br>
Embodiments of the invention comprise calibrating the measured signal (e.g., electric current and/or fluorescent intensity) for a first probe-target mixture against the same type of signal exhibited by other probes combined with the same target, wherein each of the other probes differs from the first probe by at least one base.<br>
In certain embodiments, a low voltage is applied to the sample prior to or concurrent with measuring said signal. Generally, the voltage .is selected such that it is high enough to destabilize imperfectly matched hybridization partners but not so high as to destabilize perfectly matched hybridization partners. In certain preferred embodiments, the voltage is about IV to about 20V.<br>
A calibration curve can be generated, wherein the magnitude of the measured signal (e.g., electric current and/or fluorescent intensity) is a function of the binding affinity between the target and probe. As the binding affinity between the target and a plurality of different probes varies with the number of mismatched bases, the nature of the mismatch (A-G vs. A-C vs. T-G vs. T-C,<br>
etc.), the location of the mismatch(es) within the hybridization complex, etc., the assay of the invention can be used to sequence the target.<br>
The signal measured can be, e.g., electrical conductance. In such embodiments, the binding affinity between the probe and target is directly correlated with the magnitude of the signal. That is, the electrical conductance increases along with the extent of matching between the probe and target, preferably over a range inclusive of 0-2 mismatches and/or deletions, more preferably over a range inclusive of 0-3 mismatches and/or deletions.<br>
In other embodiments, the signal measured can be the fluorescent intensity of a fluorophore included in the test sample. In such embodiments, the binding affinity between the probe and target can be directly or inversely correlated with the intensity, depending on whether the fluorophore signals hybridization through signal quenching or signal amplification. Thus, the fluorescent intensity generated by intercalating agents is directly correlated with probe-target binding affinity, whereas the intensity of embodiments employing non-intercalating fluorophores covalently bound to the probe is inversely correlated with probe-target binding affinity. The fluorescent intensity increases (or decreases for non-intercalators) along with the extent of matching between the probe and target, preferably over a range inclusive of 0-2 mismatches and/or deletions, more preferably over a range inclusive of 0-3 mismatches and/or deletions.<br>
Although the inventors have previously disclosed the advantages  of  fluorescent  intensity  assays  for hybridization (see U.S. Patent Application No. 09/468,679, filed December 21, 1999), the application of an electric<br>
field to the sample appears to increase the resolution of the assay, as shown in Example 6 below.<br>
Moreover, in particularly preferred embodiments of the invention, the assay comprises measuring at least two signals of the sample. The first signal is preferably fluorescent intensity and the second signal is preferably selected from several electrical conductance measurements (or vice versa).<br>
In the preferred multiple measurement embodiments, the first signal can be the same as or different from the second signal. When the first and second signals measured are the same, the second signal can be calibrated against the first signal and/or against the same reference signal(s) used to calibrate the first signal. In addition, a condition-altering stimulus is preferably applied to the test sample after the first signal is measured and before the second signal is measured. The stimulus is preferably sufficient to significantly affect imperfectly complementary hybridization between the probe and the target and insufficient to significantly affect perfectly complementary hybridization between the probe and the target.<br>
For example, in a particularly preferred embodiment of the invention, the first signal measured is pre-electrification fluorescent intensity (i.e., intensity measured before a condition-altering voltage is applied to the test sample) and the second signal measured is post-electrification fluorescent intensity (i.e., intensity measured during or after the condition-altering voltage has been applied to the test sample) .<br>
The additional measurements in the foregoing embodiments increase the reliability of the assay and enable immediately retesting suspect results.<br><br>
Inconsistent  results  achieved by the at  least  two measurements will typically warrant retesting.<br>
The invention enables quantifying the binding affinity between probe and target. Such information can be valuable for a variety of uses, including designing antisense drugs with optimized binding characteristics.<br>
Unlike prior art methods, the assay of cne invention is preferably homogeneous.  The assay can be conducted without separating the probe-target complex from the free probe and target prior to detecting the magnitude of the measured signal.   The assay does not require a gel separation step, thereby allowing a great increase in testing throughput.  Quantitative analyses are simple and accurate.  Consequently the binding assay saves a lot of time  and  expense,  and  can  be  easily  automated. Furthermore, it enables binding variables such as buffer, pH, ionic concentration, temperature, incubation time, relative concentrations of probe and target sequences, intercalator concentration, length of target sequences, length  of  probe  sequences,  and possible  cofactor requirements to be rapidly determined.<br>
The assay can be conducted in e.g., a solution within a well, on an impermeable surface or on a biochip.<br>
Moreover, the inventive assay is preferably conducted without providing a signal quenching agent on the target or on the probe.<br>
Preferred embodiments of the invention specifically detect triplex hybridization between the probe and the double-stranded target,  thus obviating the need to denature the target.  While PNA probes have been known to form triplexes with certain classes of targets (see, e.g., Egholm et al., 365 Nature 566 (1993), and Tomac et al., 118 J.Am.Chem.Soc.  5544  (1996)),  the inventors were<br>
surprised that they were able to specifically assay triplexes formed between single-stranded nucleic acid (e.g., ssDNA and RNA) probes and double-stranded nucleic acid (e.g., dsDNA) targets. Triplex formation and/or stabilization is enhanced by the presence of an intercalating agent in the sample being tested.<br>
Suitable probes for use in the inventive assay include, e.g., ssDNA, RNA, PNA and other nucleic acid analogs having uncharged or partially-charged backbones. Although antiparallel probes are preferred in certain embodiments, PNA probes can also be parallel. Probe sequences having any length from 8 to 20 bases are preferred since this is the range within which the smallest unique DNA  sequences of prokaryotes and eukaryotes are found. Probes of 12 to 18 bases are particularly preferred since this is the length of the smallest unique sequences in the human genome. In embodiments, probes of 6 to 30 bases are most preferred. However, a plurality of shorter probes can be used to detect a nucleotide sequence having a plurality of non-unique target sequences therein, which combine to uniquely identify the nucleotide sequence. The length of the probe can be selected to match the length of the target.<br>
The invention does not require the use of radioactive probes, which are hazardous, tedious and time-consuming to use, and need to be constantly regenerated. Probes of the invention are preferably safe to use and stable for years. Accordingly, probes can be made or ordered in large quantities and stored.<br>
It is preferred that the probe and target be unlabeled, but in alternative embodiments, there is an intercalating agent covalently bound to the probe. In<br>
such embodiments, the intercalating agent is preferably bound to the probe at either end.<br>
In other embodiments, the intercalating agent is not covalently bound to the probe, although it can insert itself between the probe and target during the assay, in a sense bonding to the probe in a non-covalent fashion.<br>
Preferred intercalating agents for use in the invention include, e.g., YOYO-1, TOTO-1, ethidium bromide, ethidium homodimer-1, ethidium homodimer-2 and acridine. In general, the intercalating agent is a moiety that is able to intercalate between strands of a duplex and/or a triplex nucleic acid complex. In preferred embodiments, the intercalating agent (or a component thereof) is essentially non-fluorescent in the absence of nucleic acids and fluoresces when intercalated and excited by radiation of an appropriate wavelength, exhibiting a 100-fold to 10,000-fold enhancement of fluorescence when intercalated within a duplex or triplex nucleic acid complex.<br>
In alternative embodiments, the intercalating agent may exhibit a shift in fluorescent wavelength upon intercalation and excitation by radiation of an appropriate wavelength. The exact fluorescent wavelength may depend on the structure of the nucleic acid that is intercalated, for example, DNA vs. RNA, duplex vs. triplex, etc.<br>
The excitation wavelength is selected (by routine experimentation and/or conventional knowledge) to correspond to this excitation maximum for the fluorophore being used, and is preferably 200 to 1000 nm. Intercalating agents are preferably selected to have an emission wavelength of 200 to 1000 nm. In preferred embodiments, an argon ion laser is used to irradiate the<br>
fluorophore with light having a wavelength in a range of 400 to 540 nm, and fluorescent emission is detected in a range of 500 to 750 nm.<br>
The assay of the invention can be performed over a wide variety of temperatures, such as, e.g., from 5 to 85°C. Certain prior art assays require elevated temperatures, adding cost and delay to the assay. On the other hand, the invention can be conducted at room temperature or below (e.g., at a temperature below 25°C).<br>
The inventive assay is extremely sensitive, thereby obviating the need to conduct PCR amplification of the target. For example, in at least the fluorescent intensity embodiments, it is possible to assay a test sample having a volume' of about 20 microliters, which contains about 10 femtomoles of target and about 10 femtomoles of probe. Embodiments of the invention are sensitive enough to assay targets at a concentration of 5 X 10"9 M, preferably at a concentration of not more than 5 x 10~10 M. Embodiments of the invention are sensitive enough to employ probes at a concentration of 5 X 10"9 M, preferably at a concentration of not more than 5 x 10"10 M.<br>
Conductivity measurements can distinguish samples having as little as about 1 pmole of probe and 1 pmole of target in 40 microliters. Decreasing the sample volume would permit the use of even smaller amounts of probe and target.<br>
It should go without saying that the foregoing values are not intended to suggest that the method cannot detect higher concentrations.<br>
A wide range of intercalator concentrations are tolerated at each concentration of probe and target tested. For example, when 5 X 10'10 M probe and 5 X 10"10 M target are hybridized, the optimal concentration of the<br>
intercalates YOYO-1 ranges from 25 nM to 2.5 nM. At a 5 X 10"B M concentration of both probe and target, the preferred YOYO-1 concentration range is 1000 nM to 100 nM.<br>
The assay is sufficiently sensitive to distinguish a one base-pair mismatched probe-target complex from a two base-pair mismatched probe-target complex, and preferably a two base-pair mismatched probe-target complex from a three base-pair mismatched probe-target complex. Of course, the assay is sufficiently sensitive to distinguish a perfectly matched probe-target complex from any of the above mismatched complexes.<br>
The hybridization medium can be any conventional medium known to be suitable for preserving nucleotides. See, e.g., Sambrook et al., "Molecular Cloning: A Lab Manual," Vol. 2 (1989). For example, the liquid medium can comprise nucleotides, water, buffers and standard salt concentrations.<br>
Hybridization between complementary bases occurs under a wide variety of conditions having variations in temperature, salt concentration, electrostatic strength, and buffer composition. Examples of these conditions and methods for applying them are known in the art.<br>
It is preferred that hybridization complexes be formed at a temperature of about 15 °C to about 25°C for about 1 minute to about 5 minutes. Longer reaction times are not required, but incubation for several hours will not adversely affect the hybridization complexes.<br>
It is possible (although unnecessary, particularly for embodiments containing an intercalating agent) to facilitate hybridization in solution by using certain reagents". Preferred examples of these reagents include single stranded binding proteins such as Rec A protein, T4 gene 32 protein, E. coli single stranded binding protein,<br>
major or minor nucleic acid groove binding proteins, divalent ions, polyvalent ions, viologen and intercalating substances such as ethidium bromide,  actinomycin D, psoralen, and angelicin.  Such facilitating reagents may prove useful in extreme operating conditions, for example, under abnormal pH levels or extremely high temperatures.<br>
The inventive assay can be used to, e.g., identify accessible regions in folded nucleotide sequences, to determine the number of mismatched base pairs in a hybridization complex, and to map genomes.<br>
In embodiments wherein fluorescent intensity is detected using an intercalating agent, intensity increases with increasing binding affinity between the probe and target. In embodiments "wherein fluorescent intensity is detected using a non-intercalating fluorophore, intensity decreases as binding affinity increases between the probe and target. Regardless of whether the fluorophore intercalates or not, the instant method does not require the measurement of the polarization of fluorescence, unlike fluorescent anisotropy methods.<br>
The invention will be illustrated in more detail with reference to the following Examples, but it should be understood that the present invention is not deemed to be limited thereto.<br>
EXAMPLES Example 1<br>
Sense and antisense 50-mer ssDNA target sequences, derived from exon 10 of the human cystic fibrosia aene (Nature 380. 207 (1996)} were synthesized on a DNA synthesizer (Expedite 8909, PerSeptive Biosystems) and purified by HPLC. Equimolar amounts of complementary oligonucleotides were denatured at 95°C for 10 min and allowed to anneal gradually as the temperature cooled to<br>
21°C over 1.5 hours. Double stranded DMA (dsDNA) oligonucleotides were dissolved in ddH20 at a concentration of 1 pmole/µ1.<br>
Sequence for the sense strand of the wild-type target DNA (SEQ ID N0:l): 5'-TGG CAC CAT TAA AGA AAA TAT CAT CTT TGG TGT TTC CTA TGA TGA ATA TA-31.<br>
Sequence for the antisense strand of the wild-type target DNA (SEQ ID NO:1): 5'-TAT ATT CAT CAT AGG AAA CAC CAA AGA TGA TAT TTT CTT TAA TGG TGC CA-3 ' .<br>
The predicted melting temperature (Tm) of dsDNA (SEQ ID N0:l) is 65.2°C.<br>
SEQ ID NO:2 was a 50-mer mutant dsDNA target sequence identical to wild-type target DNA (SEQ ID N0:l) except for a one base pair mutation (underlined) at amino acid position 507 at which the wild-type sequence CAT was changed to CGT.<br>
Sequence for the sense strand of SEQ ID NO:2: 5'-TGG CAC CAT TAA AGA AAA TAT CGT CTT TGG TGT TTC CTA TGA TGA ATA TA-3' .<br>
Sequence for the antisense strand of SEQ ID NO:2: 5'-TAT ATT CAT CAT AGG AAA CAC CAA AGA CGA TAT TTT CTT TAA TGG TGC CA-3 ' .<br>
The predicted melting temperature (T,,,) of dsDNA (SEQ ID NO:2) is 66.0°C.<br>
SEQ ID NO:3 was a 50-mer mutant dsDNA target sequence identical to wild-type target DNA (SEQ ID NO:1) except for a consecutive two base pair mutation (underlined) at amino acid positions 506 and 507 at which the wild-type sequence CAT was changed to ACT.<br>
Sequence for the sense strand of SEQ ID NO:3: 5'-TGG CAC CAT TAA AGA AAA TAT ACT CTT TGG TGT TTC CTA TGA TGA ATA TA-3 ' <br>
Sequence for the antisense strand of SEQ ID NO:3: 5'-TAT ATT CAT CAT AGG AAA CAC CAA AGA GTA TAT TTT CTT TAA TGG TGC CA-31.<br>
The predicted melting temperature (Tm) of dsDNA (SEQ ID NO:3) is 65.2°C.<br>
The PNA probes used in the Examples were synthesized, HPLC purified and confirmed by mass spectroscopy by Commonwealth Biotechnologies, Inc. (Richmond, VA, USA) . PNA probes were first dissolved in 0.1% TFA (trifluoroacetic acid) to a concentration of 10 mg/ml, and then diluted to 1 mg/ml by the addition of ddH20. Final PNA stock solutions were prepared in ddH2O at a concentration of 1 pmole/µl.<br>
Probe No. 1 was a 15-mer antiparallel PNA probe designed to be completely complementary to a 15 nucleotide segment of the sense strand of the 50-mer wild-type target DNA (SEQ ID N0:l), overlapping amino acid positions 505 to 510 (Nature 380. 207 (1996)) . The probe had the following structure {SEQ ID NO:8):<br>
5'-H-CAC  CAA  AGA  TGA TAT-Lys-CONH2-3'<br>
The hybridization reaction mixture (80µ1) contained the following: 2 pmoles of target dsDNA, 2 pmoles of PNA probe, 0.5X TBE and 250 nM of the DNA intercalator YOYO-1 (Molecular Probes, Eugene, OR, USA). Samples were placed into a 3 mm quartz cuvette and were subjected to 1 or 5 volts DC (V) electrification for 15 seconds. The amperometric assay consisted of the monitoring of current while the voltage was being applied to the solution. A temperature probe was placed in each solution to measure temperature at the time of amperometric assessment. At 1 volt, a current peak was observed during the first 2 seconds of electrification. The current declined sharply over the following 13 seconds. Experiments applying 5<br>
volts gave rise to currents that remained relatively stable over the entire electrification period (15 seconds) .<br>
A series of experiments were carried out where the conductance values were observed when no DNA or PNA was present (control), or when wild-type SEQ ID NO:1, mutant SEQ ID NO:2 or mutant SEQ ID NO:3 were reacted with antiparallel PNA Probe No. 1. Figures 1A and IB plot the data obtained for conductance in the individual experiments. Figure 1A displays the results of the application of IV electrification and Figure IB the application of 5V. Double stranded DNA:PNA hybrid triplexes consisting of perfectly complementary sequences<br>
(SEQ ID NO:1 + Probe No. 1) allowed maximum intercalation of YOYO-1,  yielding the highest conductance values<br>
(depicted on the figures as negative current values) throughout the entire 15 seconds of IV application. The normalized peak conductance for the triplex hybridization of the antiparallel PNA probe with a 1 bp mismatched dsDNA<br>
(SEQ ID NO:2 + Probe No. 1) and with the 2 bp mismatched dsDNA (SEQ ID NO:3 + Probe No. 1) were respectively 79% and 96% lower than that observed with the perfectly matched dsDNA:PNA triplex hybrid (SEQ ID NO:1 + Probe No. 1) during the first second of voltage application (Fig. 1A) . Similar percent decreases in conductance between perfectly complementary triplexes and triplexes containing base pair mismatches were obtained when the conductance values over the entire 15 seconds of voltage application were averaged. In Fig. 1A the 1 bp and 2 bp mismatched dsDNA:PNA hybrids resulted in average conductance values that were 65% and 91% lower, respectively, than those for the perfectly matched dsDNA:PNA hybrid. All experiments expressed in Figure 1A were carried out at room<br>
temperature (23°C). As the degree of mismatch between the probe and the double stranded target increased', the level of intercalation by YOYO-1 diminished and the level of conductance decreased. These relationships were also observed when the experiments referred to above were repeated and a higher voltage (5V) was applied. During the 5V application the normalized average conductance values for the 1 bp mismatched dsDNA:PNA triplex (SEQ ID NO:2 + Probe No. 1) and the 2 bp mismatched dsDNA:PNA triplex (SEQ ID NO:3 + Probe No. 1) were respectively 52% and 67% lower than that observed for the perfectly matched dsDNA:PNA triplex (SEQ ID NO:3 + Probe No. 1) (Fig. IB). Experiments expressed in Figure 1B were performed at room temperature (23°C). '<br>
When the experiments were repeated with the temperature increased to 50°C and 65°C, similar amperometric values were observed. At 50°C, the application of IV for 15 seconds to the perfectly matched dsDNArPNA triplex (SEQ ID NO:1 + Probe No. 1) produced an average current of -0.25 µAmp as compared to values of -0.15 µAmp (a 40% reduction) and -0.06 µAmp (a 76% reduction) for the 1 bp mismatched dsDNAtPNA triplex (SEQ ID NO:2 + Probe No. 1) and the 2 bp mismatched dsDHA:PNA triplex (SEQ ID NO:3 + Probe No. 1) , respectively (Fig. 1C) . At 65°C, similar observations were noted when IV of electricity was applied for 15 seconds. Perfectly matched nucleic acid hybrids produced an average current of -0.37 µAmp compared with -0.16 µzAmp (a 57% reduction) and -0.01 µAmp (a 97% reduction) for 1 bp and 2 bp mismatched hybrids, respectively (Fig. 1C) . The application of 5 volts at high temperatures produced analogous results. While experiments performed at 50°C generated average currents of -0.27 mAmp, -0.13 mAmp (a 52% reduction), and<br>
-0.08 mAmp (a 70% reduction), for perfectly matched hybrids, I bp mismatched hybrids, and 2 bp mismatched hybrids, respectively, experiments performed at 65°C resulted in average current values of -0.31 mAmp, -0.14 mAmp (a 55% reduction), and -0.10 mAmp (a 68% reduction) for the same three respective groups (Fig. ID). For all of the foregoing experiments, dsDNA was not denatured prior to triplex hybridization with the antiparallel PNA Probe No. 1.<br>
Similar experiments were done at varying temperatures after the hybridization mixes had been heated to 65°C and immediately allowed to cool. After cooling to room temperature (23 °C), applying IV for 15 seconds to the perfectly matched sample (SEQ ID N0:l + Probe No. 1) produced an average current of -0.18 µAmp. By comparison, values of -0.06 µAmp (a 67% reduction) and -0.05 µAmp (a 72% reduction) for the 1 bp mismatched dsDNA:PNA triplex hybrid {SEQ ID NO:2 + Probe No. 1) and the 2 bp mismatched dsDNA:PNA triplex hybrid (SEQ ID NO:3 + Probe No. 1), were respectively observed (data not shown) . When the samples were cooled from 65°C to 50°C, similar observations were noted when IV was subsequently applied for 15 seconds. The perfectly matched sample (SEQ ID N0:l + Probe No. 1) produced an average current of -0.23 µAmp compared with -0.11 µAmp (a 52% reduction) and -0.01 µzAmp (a 96% reduction) observed for the 1 bp and 2 bp mismatched samples, respectively (data not shown) . When 5V was applied after cooling to 23°C or 50°C, the average current generated in the perfectly matched triplex hybrid (SEQ ID NO:1 + Probe No. 1), the 1 bp mismatched triplex hybrid (SEQ ID NO:2 + Probe No. 1), and the 2 bp mismatched triplex hybrid (SEQ ID NO:3 + Probe No. 1) were: -0.15 mAmp, -0.09 mAmp (a 40% reduction), and -0.07 mAmp (a 53%<br>
reduction), respectively at 23°C, and -0.23 mAmp, -0.09 mAmp (a 61% reduction), and -0.09 mAmp (a 61% reduction), respectively at 50°C (data not shown).<br>
Pretreatment of hybridization mixes at 65°C (the Tm of the 50-mer dsDNA sequences) followed by cooling did not significantly affect the difference in conductance observed between perfectly complementary dsDNA:PNA triplexes and those containing 1 or 2 bp mismatches when measured directly at 23°C or 50°C (without preheating at 65°C) when an antiparallel PNA probe was used. Clearly, the antiparallel PNA probe in the presence of the DNA intercalator YOYO-1 was able to form triplex structures with the dsDNA targets. Application of low levels of electricity (such as IV or 5V) allowed the perfectly matched dsDNA:PNA triplex sequences to be distinguished from those containing 1 bp or 2 bp mutations, without prior denaturation of sequences. Example 2<br>
Figure 2 demonstrates that the amperometric assay of the invention can also discriminate between perfectly matched dsDNArPNA triplex hybrids and those containing 1 bp or 2 bp mismatches when the PNA probe used is in a parallel orientation with respect to the target DNA sequence. Probe No. 2 was a 15-mer PNA probe identical in sequence to Probe No. 1, but was synthesized to match the parallel orientation of the target DNA, instead of the conventional anti-parallel orientation. Probe No. 2 had the following structure (SEQ ID NO:9):<br>
5'-H-TAT AGT AGA AAC CAC-Lys-CONH2-3'<br>
Experiments with assay conditions identical to those described in Example 1 were carried out with the sole difference that Probe No. 2 was used in place of Probe No. 1. When 1 volt was applied, the average current for a 1<br>
bp mismatched dsDNA:PNA triplex (SEQ ID NO:2 + Probe No. 2), and a consecutive 2 bp mismatched dsDNA:PNA triplex<br>
(SEQ ID NO:3 + Probe No. 2) , were respectively 25% and 32% lower at 23°C, respectively 30% and 23% lower at 50°C, and respectively 28% and 53% lower at 65°C than that observed with the perfectly matched dsDNA:PNA triplex (SEQ ID NO:1 + Probe No. 2) at matching temperatures (Fig. 2A) .<br>
Similar results were obtained when 5V (instead of IV) was applied for 15 seconds.  Perfectly matched dsDNA:PNA hybrids at 23°C, 50°C and 65°C generated average currents of -0.15 mAmp, -0.24 mAmp and -0.17 mAmp, respectively<br>
(Fig. 2B) . Incompletely complementary triplexes with a 1 bp mismatch and a 2 bp mismatch produced average currents that were 27% less (-0.11 mAmp) and 53% less (-0.07 mAmp), respectively at 23°C, 21% less (-0.19 mAmp) and 46% less<br>
(-0.13  mAmp),  respectively at  50°C,  and unchanged<br>
(-0.17mAmp) and 18% less (-0.14 mAmp), respectively at 65°C, than that achieved by the perfectly matched hybrid samples (Fig. 2B) .<br>
The results illustrated in Figures 2A and 2B indicated that when the parallel PNA Probe No. 2 was used, the differences in conductivity obtained between perfectly matched dsDNA:PNA triplexes and those containing 1 bp or 2 bp mismatches were less dramatic than that achieved with the antiparallel PNA Probe No. 1 (Fig. 1).<br>
However, experiments involving parallel Probe No. 2 and the application of 5V after the samples have been heated to 65°C and immediately allowed to cool disclosed amperotnetric measurements which demonstrated enhanced signaling differences between perfectly matched dsDNA:PNA triplexes and the 1 bp or 2 bp mismatched dsDNA:PNA triplexes (Fig. 2C). The perfectly matched hybrids (SEQ ID NO:1 + Probe No. 2), the 1 bp mismatched hybrids (SEQ<br>
ID NO:2 + Probe No. 2) and the 2 bp mismatched hybrids (SEQ ID NO:3 + Probe No. 2) yielded average conductance values of -0.19 mAmps, -0.08 mAmps and -0.06 mAmps, respectively at 23°C, -0.17 mAmps, -0.09 mAmps and -0.07 mAmps, respectively at 50°C, and -0.23 mAmps, -0.13 mAmps and -0.08 mAmps, respectively at 65°C. This translated to reductions in conductivity of 58% and 68% at 23°C, 47% and 59% at 50°C, and 43% and 65% at 65°C for the 1 bp and 2 bp mismatched samples, respectively, when compared to the values achieved by the perfectly complementary samples (Fig. 2C).<br>
Therefore, both antiparallel and parallel PNA probes in the amperometric assay are capable of discriminating between perfectly complementary dsDNA targets and incompletely complementary dsDNA targets containing 1 bp or 2 bp mutations. Example 3<br>
Probe No. 3 was a 15-mer ssDNA probe identical in sequence and orientation to the 15-mer antiparallel PNA Probe No. 1 (SEQ ID NO:8) . Probe No. 3 had the following structure:<br>
5'-CAC  CAA AGA TGA  TAT-3 '<br>
The specificity of the amperometric assay was further investigated by reacting ssDNA Probe No. 3 with the 50-mer wild-type and mutant dsDNA target sequences in the absence of prior denaturation. The assay conditions were identical to that described in Example 1.<br>
Enhanced by the DNA intercalator YOYO-1, dsDNArssDNA triplexes were formed between 30°C and 65°C. Upon 1 volt treatment, the perfectly matched DNA triplex, consisting of SEQ ID N0:l + Probe No. 3, yielded the highest conductivity values (Fig. 3A) . In contrast, incompletely complementary probe and target combinations generating a<br>
1 bp mismatch (SEQ ID NO:2 + Probe No. 3) , and a consecutive 2 bp mismatch (SEQ ID NO:3 + Probe No. 3) , resulted in average conductance values that were 14% and 64% lower at 23°C, 30% and 70% lower at 50°C, and 25% and 72% lower at 65°C, respectively, than that observed with the perfectly complementary sequences at matching temperatures (Fig. 3A) . The application of a higher voltage (5V) to these samples resulted in greater amperometric differences observed between perfectly matched and mismatched samples, than that obtained at IV, particularly at lower temperatures. After a 5V treatment for 15 seconds, the average currents for the 1 bp mismatched DNA triplex and the 2 bp mismatched DNA triplex were 54% and 78% lower, 'respectively at 23°C, 68% and 70% lower, respectively at 50°C, and 33% and 61% lower, respectively at 65°C, than that observed with the perfectly matched DNA triplex at matching temperatures (Fig. 3B).<br>
In similar electricity experiments, the hybridization mixes were heated to 65°C and were either maintained at this temperature or immediately allowed to cool to 50°C or 23 °C prior to application of 1V or 5V. A 1V treatment for 15 seconds to the perfectly matched DNA triplex sequences (SEQ ID NO:1 + Probe No. 3) produced the highest conductance values at 23°C, 50°C and 65°C (Fig. 3A) . The DNA triplexes containing a 1 bp mismatch (SEQ ID NO:2 + Probe No. 3) or a 2 bp mismatch (SEQ ID NO:3 + Probe No. 3) were less conductive by 21% and 63%, respectively at 23°C, by 18% and 74%, respectively at 50°C, and by 12% and 106%, respectively at 65°C (Fig. 3A) . Similarly, when 5V were applied for 15 seconds to pre-heated samples, the average conductance values for the 1 bp mismatched DNA triplexes and the 2 bp mismatched DNA triplexes were<br>
reduced by 24% and 104%, respectively at 23°C, by 42% and 44%,  respectively  at  50°C,  and  by  38%  and  102%, respectively at 65 °C,  when compared to the average conductance values generated by the perfectly matched DNA triplexes (Fig. 3B) .<br>
The observation that the antiparallel PNA probe (Fig. 1) and ssDNA probe (Fig. 3) behaved in a similar fashion in the amperometric assay, suggested that the backbone of the nucleic acid entity used as the probe was not particularly important. The presence of YOYO-1 allowed the dsDNA targets and the ssDNA probe to form a triple helix conformation capable of generating different electrical charges depending on the level of sequence complementarity betweeri the target and the probe in solution. As the degree of mismatch between the probe and the target increased, the level of conductance decreased, proving the reliability of the amperometric assay when a natural DNA probe was used in the absence of prior denaturation. Example 4<br>
In the amperometric assays illustrated in Examples 1 to 3, the DNA intercalator YOYO-1 was added to the solution containing the hybridization mixes. Intercalation by YOYO-1 facilitated the formation of the dsDNArPNA triplexes and dsDNA:ssDNA triplexes. The possibility of utilizing an intercalator moiety covalently tethered to a ssDNA probe in the amperometric assay was evaluated in Example 4.<br>
Acridine is an alternative dsDNA intercalator, that also possesses the ability to intercalate into triplex nucleic acid structures, thereby stabilizing the triple helix formation. See, e.g., Kukreti et al., "Extension of the range of DNA sequences available for triple helix<br>
formation: stabilization of mismatched triplexes by acridine-containing oligonucleotides." 25 Nucleic Acids Research 4264-4270 (1997). A ssDNA probe containing an acridine molecule (Glen Research, Sterling, VA, USA) covalently attached at the 3' end was synthesized on a DNA synthesizer (Expedite 8909, PerSeptive Biosystems) and purified by HPLC.<br>
Probe No. 4 was a 15-mer ssDNA probe identical in sequence and orientation to the 15-mer Probe No. 3 (and thus also identical in sequence and orientation to the 15-mer antiparallel PNA Probe No. 1 (SEQ ID NO: 8)) but with the addition of an acridine moiety at the 3' position. The probe had the following structure:<br>
5'-CAC  CAA AGA TGA TAT-acridine-3'<br>
The hybridization reaction mixture (80 µl) contained the following: 2 pmoles of target dsDNA, 2 pmoles of ssDNA Probe No. 4 and 0.5X TBE. Samples were placed into a 3 mm quartz cuvette and were subjected to 5V DC electrification for 11 seconds at 23°C. The current and temperature were monitored as described in Example 1.<br>
As shown in Pig. 4, the ssDNA Probe No. 4 was able to hybridize with the 50-mer perfectly matched dsDNA target (SEQ ID NO:1) as a result of the stable intercalation of the covalently tethered acridine moiety, generating an average current of -0.53 mAmp. By comparison, the less stable DNA triplexes containing a 1 bp mismatch (SEQ ID NO:2 + Probe No. 4) or a 2 bp mismatch (SEQ ID NO.-3 + Probe No. 4) produced average currents that were 52% and 66% lower, respectively, than that achieved by the perfectly matched DNA triplex, when normalized against the control (Probe No. 4 without target DNA) (Fig. 4) .<br>
Therefore, the acridine attached to a ssDNA probe was equally as efficient as untethered YOYO-1 in forming<br>
triple DNA helices that generated different electrical currents   depending   on  the   level   of   sequence complementarity between the target and the probe in the amperometric assay. Example 5<br>
Sense and antisense 15-mer ssDNA target sequences, derived from exon 10 of the human cystic fibrosis gene, were synthesized, purified and annealed as described in Example 1. DsDNA oligonucleotides were dissolved in ddH20 at a concentration of 1 pmole/µl.<br>
SEQ ID NO: 4 was a 15-mer dsDNA target sequence derived from SEQ ID N0:l, designed to be completely complementary to Probe No. 1.<br>
Sequence for the sense strand of the wild-type target DNA (SEQ ID NO:4) : 5 '-ATA TCA TCT TTG GTG-3 ' .<br>
Sequence for the antisense strand of the wild-type target DNA (SEQ ID NO:4) : 5'-CAC CAA AGA TGA TAT-3' .<br>
The predicted melting temperature (Tm) of dsDNA (SEQ ID NO:4) is 40.0°C.<br>
SEQ ID NO:5 was a 15-mer mutant dsDNA target sequence identical to wild-type target DNA (SEQ ID NO:4) except for a one base pair mutation (underlined) , at which the sequence TTT was changed to TAT.<br>
Sequence for the sense strand of the mutant target DNA (SEQ ID NO:.5) :<br>
5'-ATA  TCA TCT  ATG  GTG-31.<br>
Sequence for the antisense strand of the mutant target DNA (SEQ ID NO:5):<br>
5'-CAC  CAT AGA  TGA  TAT-31.<br>
The predicted melting temperature (Tm) of dsDNA (SEQ ID NO.-5) is 40.0°C.<br>
SEQ ID NO:6 was a 15-mer mutant dsDNA target sequence identical to wild-type target DNA (SEQ ID NO:4) except for<br>
a consecutive two base pair mutation (underlined) , at which the sequence ATC was changed to GGC.<br>
Sequence for the sense strand of the mutant target DMA (SEQ ID NO:6):<br>
5'-ATA  TCG  GCT  TTG  GTG-31.<br>
Sequence for the antisense strand of the mutant target DNA (SEQ ID NO: 6):<br>
5'-CAC  CAA  AGC  CGA  TAT-31.<br>
The predicted melting temperature (TM) of dsDNA (SEQ ID N0:6) is 44.0°C.<br>
SEQ ID NO:7 was a 15-mer mutant dsDNA target sequence identical to wild-type target DNA (SEQ ID NO:4) except for a separated three base pair mutation (underlined), wherein three 1 bp mutations were separated by 3 base pairs each. The sequences ATC, TCT and TGG were changed to ACC, TAT and TAG, respectively.<br>
Sequence for the sense strand of the mutant target DNA (SEQ ID NO: 7) : 5'-ATA CCA TAT TTA GTG-31.<br>
Sequence for the antisense strand of the mutant target DNA (SEQ ID NO:7) : 5'-CAC TAA ATA TGG TAT-31.<br>
The predicted melting temperature (Tn) of dsDNA (SEQ ID N0:7) is 38.0°C.<br>
The hybridization reaction mixture (80 µl) contained the following: 2 pinoles of target dsDNA, 2 pmoles of parallel PNA Probe No. 2, 0.5X TBE and 250 nM of the DNA intercalator YOYO-1. The reaction mixtures were incubated at 95°C for 5-10 minutes to allow denaturation, and then maintained at 65°C until assayed. Samples were placed into a quartz cuvette, irradiated with an argon ion laser beam having a wavelength of 488 nm and monitored for fluorescent emission at 65°C. Concurrent temperature measurements were achieved by a software-controlled temperature probe placed directly into each sample. The<br>
maximum fluorescent intensity occurred at a wavelength of 536 nm, indicative of intercalation of YOYO-1 in the PNA.-DNA hybrids. As a second assay, following the initial laser irradiation of each sample, the same samples were subjected to IV DC electrification for 4 seconds. During the final second of electrification the samples were irradiated a second time with the argon ion laser and monitored for fluorescent emission at 65°C. Fluorescent intensities were plotted as a function of wavelength for each sample analyzed.<br>
SsDNA:PNA hybrids consisting of perfectly complementary sequences (SEQ ID NO: 4 + Probe No. 2) allowed maximum intercalation of YOYO-1, yielding the highest fluorescent intensities (Fig. 5A) . The fluorescent intensities for a 1 bp mismatched ssDNA:PNA hybrid (SEQ ID NO:5 + Probe No. 2) , a consecutive 2 bp mismatched ssDNA:PNA hybrid (SEQ ID NO:6 + Probe No. 2), and a separated 3 bp mismatched ssDNA:PNA hybrid (SEQ ID NO:7 + Probe No. 2) were all lower than that observed with the perfectly matched ssDNA:PNA hybrid at 65°C (Fig. 5 and data not shown) . As the degree of mismatch between the probe and the target increased, the level of intercalation by YOYO-1 diminished and hence the level of fluorescent intensity decreased. Only background levels of fluorescence were observed when no DNA or PNA was present<br>
(YOYO-1 alone) (Fig. 5A).<br>
When the perfectly matched ssDNA:PNA hybrids were subjected to 1V of electricity for 4 seconds at 65°C, the fluorescent intensity remained relatively constant, decreasing by only 2% (Fig. 5A) . In contrast, application of 1V to the incompletely complementary duplexes containing a l bp mismatch (Fig. 5B) , a 2 bp mismatch<br>
(Fig. 5C) and a 3 bp mismatch (data not shown) produced<br>
fluorescent intensities that were 18%, 39% and 71% lower, respectively, than that achieved with the same samples irradiated in the absence of electricity. Treatment with low levels of electricity (such as IV) further diminished the stability of the ssDNA:PNA hybrids containing bp mismatches. As the degree of sequence complementarity between the probe and the target decreased, the level of fluorescent intensity diminished dramatically in the presence of electricity, providing a highly reliable and accurate second assay to differentiate between perfectly matched sequences and those containing 1 bp, 2 bp or 3 bp mutations. Example 6<br>
The hybridization assay in Example 5 was performed after denaturation of the dsDNA target sequences and measured ssDNA:PNA hybrid formation at a temperature above the melting point (TM) of the dsDNA targets. Example 6 will demonstrate the reliability of the fluorescent intensity assay in the absence and presence of applied electricity to differentiate between perfect matches and base pair mismatches without the requirement for prior denaturation.<br>
The hybridization reaction mixture (80 µl) contained the following: 4 pmoles of target dsDNA, 4 pmoles of antiparallel PNA Probe No. 1, 0.5X TBE and 250 nM of the DNA intercalator YOYO-1. Samples were placed into a quartz cuvette, irradiated with an argon ion laser beam having a wavelength of 488 nm for 80 msec and monitored for fluorescent emission at 23°C. Concurrent temperature measurements were achieved by a software-controlled temperature probe placed directly into each sample. The maximum fluorescent intensity occurred at a wavelength of 536 nm, indicative of intercalation of YOYO-1 in the<br>
PNArDNA hybrids. As a second assay, following the initial laser irradiation of each sample, the same samples were subjected to 20V DC electrification for 4 seconds. Immediately after 3 seconds of electrification the samples were irradiated a second time with the argon ion laser for 80 msec and monitored for fluorescent emission at 23°C. Fluorescent intensities were plotted as a function of wavelength for each sample analyzed.<br>
Enhanced by the intercalator YOYO-1, dsDNA:PNA triplexes were formed at 23°C. The highest fluorescent intensity was achieved when the wild-type 50-mer dsDNA target sequence (SEQ ID N0:l) was hybridized with the 15-mer antiparallel PNA Probe No. 1 (Fig. 6} . By comparison, the fluorescent intensities for a 1 bp mismatched dsDNArPNA triplex (SEQ ID NO:2 + Probe No. 1} and a consecutive 2 bp mismatched dsDNArPNA triplex (SEQ ID NO:3 + Probe No. 1) were 60% and 91% lower, respectively, than that observed with the perfectly matched dsDNA:PNA triplex at 23°C (Fig. 6). When no DNA or PNA was present in the reaction mixture containing YOYO-1, only background levels of fluorescence were observed.<br>
The difference in fluorescent intensities obtained by the perfectly complementary triplexes and those containing 1 bp or 2 bp mismatches were significantly greater than that achieved between perfectly matched duplexes and incompletely complementary duplexes (compare Figs. 5 and 6) . Clearly the fluorescent intensity assay of triplex formation possessed enhanced discriminatory ability to detect base pair mismatches.<br>
Moreover, even further discrimination between wild-type and mutated sequences was possible with the secondary application of electricity. A 20V treatment for 3 seconds to the perfectly matched dsDNA:PNA triplexes produced a<br>
fluorescent intensity spectrum virtually identical to that achieved by the same sample not subjected to electricity (Fig. 6} . However, application of 20V for 3 seconds to the incompletely complementary triplexes containing a 1 bp mismatch and a 2 bp mismatch produced fluorescent intensities that were 23% and 71% lower, respectively, than that obtained with the same samples irradiated in the absence of electricity (Fig. 6) . The 20V treatment of electricity did not affect the stability of the perfectly complementary triplexes, but weakened the stability of the dsDNA:PNA triplexes containing base pair mismatches at a level dependent on the degree of sequence complementarity between the probe and the target. Therefore, the application of electricity to the fluorescent intensity assay provided an even more highly reliable assay to distinguish between wild-type sequences and those containing 1 bp or 2 bp mutations, without prior denaturation of sequences.<br>
While the invention has been described in detail and with reference to specific examples thereof, it will be apparent to one skilled in the art that various changes and modifications can be made therein without departing from the spirit and scope thereof.<br><br><br><br><br>
We claim:<br>
1.       A method for assaying sequence-specific hybridization, said method comprising:<br>
providing a target comprising at least one nucleic acid sequence;<br>
providing a probe comprising a nucleic acid or nucleic acid analog sequence;<br>
adding said probe, said target and intercalating agent to a hybridization medium as herein described to provide a test sample;<br>
applying a first stimulus as herein described to said test sample to provide a first stimulated test sample;<br>
detecting a first signal from said first stimulated test sample. Wherein said first signal is correlated with a binding affinity between said probe and said target;<br>
calibrating said first signal against a reference signal exhibited by a reference sample comprising at least one reference probe combined with said target, wherein relative to said target, each of said probe and said at least one reference probe is a different member selected from the group consisting of a perfect match, a one-base mismatch, a two-base mismatch, a three-base mismatch, a one-base deletion, a two-base deletion and a three-base deletion;<br>
determining from said calibrating a first determination of an extent of matching between said probe and said target;<br>
applying a second stimulus as herein described to said first stimulated test sample to provide a second stimulated test sample;<br>
detecting a second signal from said second stimulated test sample, wherein said second signal is correlated with said binding affinity between said probe and said target;<br>
determining from said detecting of said second signal a second determination of said extent of matching between said probe and said<br>
target;   and   comparing   said   first   determination   and   said   second determination, and<br>
optionally including the step of quantifying said binding affinity.<br>
2.	The method as claimed in 1, wherein said test sample comprises of an<br>
intercalating agent, said target is dsDNA and said probe hybridizes<br>
specifically with said target to form a triplex.<br>
3.	The method as claimed in 1, wherein said probe is ssDNA, RNA and<br>
ssPNA sequence.<br>
4.	The method as claimed in 1, wherein said probe has a partially charged<br>
backbone.<br>
5.	The method as claimed in 1, wherein said probe has an uncharged<br>
backbone.<br>
6.	The method as claimed in 1, wherein said probe and said target are of<br>
identical length.<br>
7.	The method as claimed in 1, wherein said probe is 6 to 30 nucleotides<br>
long.<br>
8.	The method as claimed in 1, conducted in a solution within a well or on<br>
an impermeable surface.<br>
9.	The method as claimed in 1, conducted on a biochip.<br><br>
10.	The method as claimed in 1, optionaly comprises the step of varying a<br>
condition of said first stimulated test sample prior to said applying of<br>
said second stimulus, wherein said varying is sufficient to significantly<br>
affect imperfectly complementary hybridization between said probe and<br>
said     target    and    insufficient    to     significantly    affect    perfectly<br>
complementary hybridization between said probe and said target.<br>
11.	The method as claimed in 10, wherein said varying comprises applying<br>
an electrical voltage to said first stimulated test sample, and said binding<br>
affinity is directly correlated with a difference between said first signal<br>
and said second signal.<br>
12.	The method as claimed in 11, wherein said electrical voltage is about 1<br>
volt to about 20 volts.<br>
13.	The method as claimed in 11, wherein said electrical voltage is either a<br>
direct current or an alternating current.<br>
14.	The method as claimed in 1, wherein said first signal or said second<br>
signal is an electrical conductance.<br>
15.	The method as claimed in  1, wherein said test sample  comprises<br>
providing a fluorophore which is an intercalating agent in said test<br>
sample; and calibrating said second signal against a second reference<br>
signal,<br>
wherein (a) said first stimulus is electrical voltage, said first signal is electrical conductance, said second stimulus is exciting second signal is fluorescent intensity; or (b) said first radiation and said first stimulus is exciting radiation, said first signal is fluorescent intensity, said second stimulus is electrical voltage and said second signal is electrical conductance.<br>
16.	The method as claimed in 15, wherein said fluorophore is covalently<br>
attached to said probe such that said fluorophore does not intercalate<br>
between adjacent bases, and said intensity decreases as said extent of<br>
matching between said probe and said target increases.<br>
17.	The   method   as   claimed   in   15,   wherein   said   fluorophore   is   an<br>
intercalating agent and said intensity increases along with said extent of<br>
matching between said probe and said target.<br>
18.	The   method   as   claimed   in   15,   wherein   said   fluorophore   is   an<br>
intercalating agent added to said hybridization medium in a form free of<br>
said probe and free of said target.<br>
19.	The method as claimed in 15, including providing a fluorophore in said<br>
test sample, wherein said varying comprises applying an electrical voltage<br>
to said first stimulated test sample, said first stimulus and said second<br>
stimulus are exciting radiation of 200nm to l000nm wavelength, and said<br>
first signal and said second signal are fluorescent intensity.<br>
20.	The method as claimed in 19, including the step of calibrating said<br>
second signal against said first signal to determine an extent of matching<br>
between said probe and said target.<br>
21.	The   method   as   claimed   in   19,   wherein   said   fluorophore   is   an<br>
intercalating agent selected from the group consisting of YOYO-1,<br>
TOTO-1,    ethidium    bromide,    ethidium    homodimer-1,    ethidium<br>
homodimer-2 and acridine.<br>
22.	The method as claimed in 19, conducted at temperatures within a range<br>
of 5 to 85°C.<br>
23.	The method as claimed in 19, conducted at temperatures below 25°C.<br>
24.	The method as claimed in 19, wherein a concentration of said target in<br>
said sample is not more than 5 X 10 10 M.<br>
25.	The method as claimed in 19, wherein a concentration of said probe in<br>
said sample is not more than 5 X 10 10 M.<br>
26.	A method for assaying sequence-specific hybridization substantially as<br>
herein described with reference to foregoing examples and as illustrated<br>
by the accompanying drawings.<br><br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA3MjEtZGVsLWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-00721-del-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA3MjEtZGVsLWFzc2lnbm1lbnQucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00721-del-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA3MjEtZGVsLWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00721-del-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA3MjEtZGVsLWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00721-del-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA3MjEtZGVsLWNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-00721-del-correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA3MjEtZGVsLWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00721-del-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA3MjEtZGVsLWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-00721-del-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA3MjEtZGVsLWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00721-del-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA3MjEtZGVsLWZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00721-del-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA3MjEtZGVsLWZvcm0tMTkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00721-del-form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA3MjEtZGVsLWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00721-del-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA3MjEtZGVsLWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00721-del-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA3MjEtZGVsLWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00721-del-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA3MjEtZGVsLWZvcm0tNi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00721-del-form-6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA3MjEtZGVsLWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00721-del-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA3MjEtZGVsLXBhLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-00721-del-pa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA3MjEtZGVsLXBjdC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00721-del-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA3MjEtZGVsLXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00721-del-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA3MjEtZGVsLXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00721-del-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA3MjEtZGVsLXBjdC0zMDgucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00721-del-pct-308.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA3MjEtZGVsLXBjdC00MDgucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00721-del-pct-408.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA3MjEtZGVsLXBjdC00MDkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00721-del-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA3MjEtZGVsLXBjdC00MTYucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00721-del-pct-416.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA3MjEtZGVsLXBldGl0aW9uLTEzNy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00721-del-petition-137.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA3MjEtZGVsLXBldGl0aW9uLTEzOC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00721-del-petition-138.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="217806-internal-combustion-engine.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="217808-a-method-of-forming-an-adhesive-in-situ-in-a-microcapsule.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>217807</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2002/00721/DEL</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>17/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>25-Apr-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>28-Mar-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>23-Jul-2002</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>INGENEUS CORPORATION</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>CHANCERY HOUSE, HIGH STREET, BRIDGETOWN, BARBADOS.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ERIKSON GLEN H.</td>
											<td>P.O.BOX 799, PROVIDENCIALES, TURKS &amp; CAICOS, ISLANDS.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>PICARD PIERRE,</td>
											<td>1868 PIXLEY, OTTAWA, ONTARIO K1G 4C5, CANADA.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>DAKSIS JASMINE L.</td>
											<td>36 CARTIER CRESCENT, RICHMOND HILL, ONTARIO L4C 2N2, CANADA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IB01/00077</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-01-23</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>09/490,273</td>
									<td>2000-01-24</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/217807-a-method-for-assaying-sequence-specific-hybridization by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:09:11 GMT -->
</html>
