<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/218062-a-method-for-creating-a-nucleic-acid-multiplex by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:25:56 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 218062:&quot;A METHOD FOR CREATING A NUCLEIC ACID MULTIPLEX.&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A METHOD FOR CREATING A NUCLEIC ACID MULTIPLEX.&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Heteropolymeric triplexes and quadruplexes and methods for making them; the use of accelerator agents such as cations to create them; the use of fluorescent intercalators and fluorescent probe-bound non-intercalators to detect them.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>NJCICLEIC  ACID MULTIPLEX  FORMATION<br>
SPECIFICATION BACKGROUND  OF  THE   INVENTION<br>
1.	FIELD OF INVENTION<br>
The invention relates to nucleic acid multiplexes, and more particularly to methods of creating them as triplexes and quadruplexes, and furthermore employing them in assays to detect specific nucleic acids.<br>
2.	DESCRIPTION OF RELATED ART<br>
The ability of two single-stranded nucleic acid molecules of complementary base sequence to bind specifically to each other has provided the basis for both, powerful research and powerful diagnostic tools. Less fully explored than such "conventional hybridization" has been the ability of single stranded molecules to bind to double-stranded targets and the ability of double-stranded molecules to bind to double-stranded targets. The ability to bind to double-stranded targets potentially has advantages over conventional hybridization. These could stem in part from the fact that the double-stranded target would not be denatured, allowing "milder" hybridization conditions and providing a target less prone to becoming a totally random coil. They could also stem in part from the fact that the base-pairing mechanisms would be at least partially different than in conventional hybridization, allowing the possibility for more favorable kinetics and a reduction in the amount of probe needed in the hybridization reaction mixture. Prior work on creating multiplexes have included:<br>
1)	The formation of triplexes as part of the homologous recombination process, a process mediated by the bacterial protein RecA and proteins of similar function in other organisms;<br>
2)	The creation of 3-stranded structures during in situ hybridization (e.g., U.S. patent 5,707,801 of Bresser et al.); and<br>
3)3-stranded  or  4-stranded  complexes  that  rely  on Hoogstein-type bonding.<br>
This prior work does not fully exploit the potential for forming multiplexes. The RecA-mediated process requires a protein. The in situ hybridization processes are based on the principle that the double-stranded intracellular target will locally open its double-stranded structures, providing a single-stranded target that will hybridize according to conventional hybridization principles. Complexes reported to rely on Hoogstein-type polymers are limited to structures that are not true heteropolymers. Rather they require that a given strand be a polypurine or polypyrimidine or very close thereto. See, e.g., Floris et al. , "Effect of cations on purine-purine-pyrimidine triple helix formation in mixed-valence salt solutions," 260 Eur. J. Biochem. 801-809 (1999) .<br>
As was the case with triplex nucleic acids, the conventional wisdom regarding quadruplex nucleic acids has been that such peculiar structures only exist under relatively extreme conditions for a relatively narrow class of nucleic acids. In particular, Sen et al. (Nature 334:364-366 (1988)) disclosed that guanine-rich oligonucleotides can spontaneously self-assemble into four-stranded helices in vitro. Sen et al. (Biochemistry 31:65-70 (1992)) disclosed that these four-stranded complexes can further associate into superstructures composed of 8, 12, or 16 oligomers.<br>
Marsh et al. (Biochemistry 33:10718-10724 (1994), and Nucleic Acids Research 23:696-700 (1995)) disclosed that some guanine-rich oligonucleotides can also assemble in an offset, parallel alignment, forming long "G-wires". These higher-order structures are stabilized by G-quartets that consist of four guanosine residues arranged in a plane and held together through Hoogsteen base pairings. According to Sen et al. (Biochemistry 31:65-70 (1992)), at least three contiguous guanines within the oligomer are critical for the formation of these higher order structures.<br>
It has been suggested that four-stranded DNAs play a role in a variety of biological processes, such as inhibition of HIV-1 integrase (Mazumder et al., Biochemistry 35:13762-13771 (1996)), formation of synapsis during meiosis (Sen et al., Nature 334 :364-366 (1988)), and telomere maintenance (Williamson et al., Cell 59:871-880 (1989)); Baran et al., Nucleic Acids Research 25:297-303   (1997)).<br>
It has been further suggested that controlling the production of guanine-rich quadruplexes might be the key to controlling such biological processes. For example, U.S. Patent No. 6,017,709 to Hardin et al. suggests that telomerase activity might be controlled through drugs that inhibit the formation of guanine quartets.<br>
U.S. Patent No. 5,888,739 to Pitner et al. discloses that G-quartet based quadruplexes can be employed in an assay for detecting nucleic acids. Upon hybridization to a complementary oligonucleotide, the G-quartet structure unfolds or linearizes, thereby increasing the distance between a donor and an acceptor on different parts of the G-quartet structure, resulting in a decrease in their interaction and a detectable change in a signal (e.g., fluorescence) emitted from the structure.<br>
U.S. 'Patent No. 5,912,332 to Agrawal et al. discloses a method for the purification of synthetic oligonucleotides, wherein the synthetic oligonucleotides hybridize specifically with a desired, full-length oligonucleotide and concomitantly form a multimer aggregate, such as quadruplex DNA. The multimer aggregate containing the oligonucleotide to be purified is then isolated using size-exclusion techniques.<br>
Despite the foregoing developments, the full potential of quadruplex nucleic acid has neither been fully appreciated nor fully exploited.<br>
Related to the problem of performing hybridization-type experiments with double-stranded targets is the means of detecting them. Fluorescent dyes have been used to detect and guantitate nucleic acids for decades. In their most basic form,<br>
fluorescent intensity-based assays have typically comprised contacting a target with a fluorophore-containing. probe, removing any unbound probe from bound probe, and detecting fluorescence in the washed sample. Homogeneous assays improve upon such basic assays, in that the former do not require a washing step or the provision of a non-liquid phase support.<br>
For example, U.S. Patents Nos. 5,538,848 to Livak et al. and 4,220,450 to Maggio disclose homogeneous fluorescence-based assays of nucleotide sequences using oligonucleotide probes in solution. However, these patents require the use of a quenching agent in combination with a reporting agent, so as to distinguish between the signals generated by hybridized probes and unhybridized probes. Livak et al. also requires the use of enzymes in its disclosed method. Quenching agents and enzymes add complexity and expense to the methods.<br>
U.S. Patent No. 5,332,659 to Kidwell discloses a method for detecting nucleotide sequences in solution using probes comprising at least two fluorophore moieties. The fluorophores must be selected to electronically interact with each other when close enough to vary the wavelength dependence of their spectra. Unhybridized probes are much more flexible than probes hybridized to the target sequence, and consequently the two fluorophore moieties on each probe are more likely to be close to each other when the probe is unhybridized than when the probe is hybridized. Thus, a change in emission wavelength correlated with free probe can be monitored as an indication of the amount of free probe in the sample.<br>
U.S. Patent No. 5,846,729 to Wu et al. also discloses homogeneous fluorescence-based assays for detecting nucleic acid.<br>
In addition to the aforementioned developments which detect fluorescent intensity, some have touted the advantages of fluorescent polarization assays. However, there are significant drawbacks to polarization-based assays. The degree of change in polarization as a function of binding can be unpredictable, and<br>
interpretation of data to conform inconsistent data to theoretical expectations can require more effort than is desirable in an analytical method, particularly when the method is to be automated. There are as well constraints arising from the molecular weight of the molecules whose motion is being evaluated in a fluorescent polarization assay.<br>
The present inventions will be seen, in various important embodiments to take advantage of the "properties of fluorescent molecules for purposes of detecting triplexes and quadruplexes.<br>
All references cited herein are incorporated herein by reference in their entireties.<br>
The present invention, therefore, provide a method of creating a nucleic acid multiplex, said method comprising the steps of:<br>
(a)	creating a mixture comprising water, a Watson-Crick duplex, a sufficient number of single-stranded mixed base sequence molecules to form a multiplex that includes the Watson Crick duplex, and an accelerator agent that increases a rate or amount of multiplex formation, said multiplex being a triplex or quadruplex, wherein said single-stranded molecule or molecules are selected so that, if in a multiplex, they would each be related to all other strands of the multiplex by adherence to base pairing rules, said rules being either Watson-Crick base-pairing rules or homologous binding base-pairing rules; and<br>
(b)	incubating said mixture to allow the multiplex to form, each strand of said multiplex related to all other strands of the multiplex by adherence to base-pairing rules; provided that, within the multiplex, the Watson-Crick duplex added in step (a) is heteropolymeric with a G-C content between 10% and 90%.<br>
BRIEF SUMMARY OF THE INVENTION<br>
Methods of creating multiplexes<br>
In one general aspect, the invention is a method of creating a nucleic acid multiplex, said method comprising the steps of:<br>
1)	creating a mixture comprising water, a Watson-Crick<br>
duplex, a sufficient number of single-stranded mixed base<br>
sequence molecules to form a multiplex that includes the Watson-<br>
Crick duplex, and an accelerator agent that increases a rate or<br>
amount of multiplex formation, said multiplex being a triplex or<br>
quadruplex, wherein said single-stranded molecule or molecules<br>
are selected so that, if in a multiplex, they would each be<br>
related to all other strands of the multiplex by adherence to<br>
base pairing rules, said rules being either Watson-Crick base-<br>
pairing rules or homologous binding base-pairing rules; and<br>
2)	incubating said mixture to allow the multiplex to form,<br>
each strand of said multiplex related to all other strands of<br>
the multiplex by adherence to base-pairing rules;<br>
provided that, within the multiplex, the Watson-Crick duplex added in step (1) is heteropolymeric with a G-C content between 10% and 90%.<br>
In one particular aspect of the method, the multiplex created is a triplex, in step (1) the sufficient number of single-stranded molecules is 1, and in step (2) the triplex ie<br>
formed. In a particular embodiment of the method, in the triplex, the single-stranded molecule is related to one strand of the duplex by Watson-Crick base-pairing rules and to the second strand of the duplex by homologous binding base-pairing rules. In a further particular embodiment, the duplex substantially retains its double-helical structure and the single-stranded molecule resides in a groove of that double-helical structure. All such triplexes are also aspects of the invention.<br>
In another particular aspect of the method, the multiplex created is a quadruplex, in step (1) the Watson-Crick duplex is a first Watson-Crick duplex, and in step (1) the sufficient number of single-stranded molecules is 2, those single-stranded molecules are in a second Watson-Crick duplex, and in step (2) the quadruplex is formed from said first and second duplexes. Preferably step  (1)  is done with the two single-stranded molecules already in the second Watson-Crick duplex. Methods of detecting a triplex<br>
The method of creating a triplex can be adapted to be a. method for detecting a triplex by adding an additional step (3) in which the triplex is detected. Methods of detecting a quadruplex<br>
The method of creating a quadruplex can be adapted to be a method for detecting a quadruplex by adding an additional step (3) in which the quadruplex is detected. Triplex<br>
In another general aspect, the invention is a triplex ( a triplex complex) comprising a single-stranded probe bound to a double-stranded nucleic acid target, wherein said probe comprises a heteropolymeric nucleic acid or a heteropolymeric nucleic acid analog, and all base triplets of said triplex are members selected from the group consisting of A-T-A, T-A-T, U-A-T, T-A-U, A-U-A, U-A-U, G-C-G and C-G-C.<br>
Ouadruplex:<br>
In another aspect, the invention is a multiplex structure that is a quadruplex, the quadruplex comprising:<br>
a first strand containing a first sequence of nucleobases;<br>
a second strand containing a second sequence of nucleobases, wherein said second strand is associated with said first strand by Watson-Crick bonding;<br>
a third strand containing a third sequence of nucleobases; and<br>
a fourth strand containing a fourth sequence of nucleobases, wherein said fourth strand is associated with said second strand and said third strand by Watson-Crick bonding.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
The invention will be described in conjunction with the following drawings in which like reference numerals designate like elements and wherein:<br>
Figs. 1,2A and 2B show the intensity of fluorescence as a function of temperature, GC content, and extent of base pair matching.<br>
Figs. 3, 4, 5A, 5B, 5C and 5D show the intensity of fluorescence as a function of wavelength, extent of base pair matching and cation.<br>
Figs. 6A,6B, and 6C show intensity of fluorescence as a function of lasing protocol, and cation.<br>
Figs. 7A, 7B, 7C, 8A, 8B, 9A, 9B, 9C, and 10 show, as regards quadruplexes, the intensity of fluorescence as a function of the extent of base pair matching and cation.<br>
Fig. 11 shows the intensity of fluorescence as a function of the extent of base pair matching in a solution containing ethanol.<br>
Figs. 12A, 12B, 12C and 12D show the intensity of fluorescence as a function of wavelength for perfect and imperfect base pair matches in dsDNA:ssDNA complexes when the cationic DNA intercalator YOYO-1 is present.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
GLOSSARY AND DEFINITIONS<br>
The one-letter codes for the bases that form part of their respective nucleotides are: A: adenine; T: thymine; G: guanine; C: cytosine; U: uracil. These letters are also used to represent their respective nucleotides.<br>
The G-C content of a duplex is the 100 times the number of G-c base pairs divided by the sum of the number of G-C base pairs plus the number of A-T (or A-U) base pairs and is expressed as a percentage (e.g., 20 %) .<br>
An accelerator agent is understood here as one that "increases a rate or amount of said triplex or quadruplex formation." A rate can be obtained with as few as two measurements, each at different time points. Amounts refer to the number of triplexes or quadruplexes formed.<br>
The term "accelerator agent" is used interchangeably with the terms "promoter" and "promoter agent" except where the promoter specifically refers to a gene promoter.<br>
The terms nucleic acid, triplex, quadruplex, and the like are intended to refer to molecules that comprise DNA, RNA, and analogues thereof capable of forming similar structures such as Watson-Crick duplexes, and the triplexes and quadruplexes formed herein.<br>
"Nucleobase" refers to the bases A, U, G, C, T, and those analogs that can conform to Watson-Crick base-pairing rules.<br>
Analogues of A,U,G,C, and T are those analogues that can conform to Watson-Crick base pairing rules.<br>
A Watson-Crick duplex is a 2-stranded molecule or molecular segment in which the two strands are anti-parallel, their 5'-&gt;3' directions being opposite. The overall structure of the duplex is that of a double helix. The strands are held together by hydrogen bonds and hydrophobic interactions. There is base pair complementarity, A is paired with T by two hydrogen bonds (or, in the case of RNA, A is paired with U) and G is paired with C<br>
by three hydrogen bonds. As a result, the base-pairing rules are: A is paired with T, A is paired with U, and G is paired with C. A 3-stranded nucleic acid molecule is not necessarily a triplex and it is not necessary that any segment of a 3-stranded molecule be a triplex. It is possible that, at no region within any strand, is that strand bonded to more than one other strand. A simple example would be a Y-shaped molecule where Strands 1 and 2 form the stem, Strands 1 and 3 form the left upper branch, and Strand 2 forms a single-stranded right upper branch.<br>
An example of a triplex is a 3-stranded nucleic acid molecule or molecular segment in which one strand (arbitrarily named Strand 1) follows the Watson-Crick base-pairing rules (A-T, A-U, and G-C) with both Strand 2 and Strand 3. Strands 1 and 2 form a structure that is, or is close to, that of a Watson-Crick duplex. Strand 3 resides in the major groove of that duplex. An example of such a triplex is the one elaborated by V.B. Zhurkin et al, J. Mol. Biol., (1994) vol. 239, 181-200 (See especially Figure 2 of that reference), which article is incorporated herein by reference. As part of the stabilization of such a triplex, Strand 3 is bonded to the other two strands by base pairing rules as follows:<br>
An A on Strand 3 is paired with both an A on Strand 1 and a T on Strand 2;<br>
a G on Strand 3 is paired with both a G on Strand 1 and a C on Strand 2;<br>
a C on Strand 3 is paired with both a C on Strand 1 and a G on strand 2; and<br>
a T on strand 3 is paired with both a T on Strand 1 and an A on strand 2<br>
These base pairing rules are satisfied regardless of which variant (C or C+ or C' T or T') of the Zhurkin model is considered.<br>
The term "base-pairing rules" are those that define the specificity between one nucleic acid molecule and another nucleic acid molecule when the two bind to each other with specificity.<br>
Examples are Watson-Crick base pairing rules (G-C, and A-T or A-U) and homologous binding base-pairing rules (A-A, T-T, G-G, C-C, U-U) .<br>
"Decondensation" of a duplex is defined as an increase in the overall helical repeat length of the duplex. For example, the B conformation of the duplex has an overall helical repeat length of 10 base pairs; a decondensation of 106° results in that repeat length being 13 base pairs.<br>
PNA stands for polyamide analogs of DNA and RNA (see e.g.,<br>
U.S. patent No. 5,539,082 to Nielsen et al.)<br>
Specific embodiments  and alternative  formulations  of  the inventions<br>
The inventions as described in the Summary of The Invention section have specific embodiments and preferred embodiments of interest. These embodiments are described throughout this application. However, for convenience, many of them are summarized in this section. Similarly, the inventions as summarized in the Summary of Invention section can be phrased in alternative fashions expressing some variation of the invention but retaining substantial overlap as to the essential invention. Such alternative formulations of the invention are also included in this section. (A) Methods of making the multiplexes<br>
The methods of forming the triplex or quadruplex, described generally in the Summary of the Invention section, can optionally be performed by incorporating one or more of the following into the method:<br>
within the multiplex, the Watson-Crick duplex added in step (1) is heteropolymeric with a G-C content between 25% and 75%;<br>
within the multiplex, the Watson-Crick duplex added in step (1) is heteropolymeric with a G-C content between 10% and 90%, and furthermore the combined frequencies therein of purine-pyrimidine dimers and pyrimidine-purine dimers exceeds 25% (dimers are identified starting at the 5' end of the sequence and progressing one base at a time until the 3' end is reached; for<br>
-example, the sequence 5'-AAAGGGT has one purine-pyrimidine dimer (GT) and no pyrimidine-purine dimers - their combined frequencies equal 1/6);<br>
performing steps (1) and/or (2) with the nucleic acid strands and/or duplexes not in a cell (and not in a virus) ;<br>
performing step (2) without the assistance of a protein (such as recA or protein of similar function) ;<br>
in step (1), adding the water so that it accounts, on a volume basis, for at least 50 percent of the final volume of the mixture (more preferably for at least 80% of the final volume, most preferably in step (1) water is the only liquid added to the mixture);<br>
in step (1), not adding any protein;<br>
performing step (2) at a temperature or temperatures above the freezing temperature of the aqueous solution and at not more than 85 °C (more preferably between 5 °C and' 30 °C, most preferably between 15 °C and 25 °C)<br>
in step (1), adding an anion or, more preferably, a cation as the accelerator agent (monovalent, divalent or multivalent; for example, a metallic cation or a cat ionic peptide) ,-wherein said cation is at least one member selected from the group consisting of alkali metal cations, alkaline earth metal cations, transition metal cations, Co(NH3)6+3, trivalent spermidine and tetravalent spermine;<br>
wherein said cation is Na+ provided at a concentration of 50mM to 125mM;<br>
wherein said cation is selected from the group consisting of Mn+2 provided at a concentration of lOmM to 45mM, Mg+2 provided at a concentration of lOmM to 45mM, and Ni+2 provided at a concentration of 20mM;<br>
in step (1) adding an intercalator as an accelerator agent (especially a fluorescent intercalator; preferably a bis-intercalator);<br>
in step (1) said accelerator agent is an intercalating fluorophore selected from the group consisting of YOYO-1, TOTO-1,<br>
YOYO-3, TOTO-3, POPO-1, BOBO-1, POPO-3, BOBO-3, LOLO-1, JOJO-1, cyanine dimers, YO-PRO-1, TO-PRO-1, YO-PRO-3, TO-PRO-3, TO-PRO-5, PO-PRO-1, BO-PRO-1, PO-PRO-3, BO-PRO-3, LO-PRO-1, JO-PRO-1, cyanine monomers, ethidium bromide, ethidium homodimer-1, ethidium homodimer-2, ethidium derivatives, acridine, acridine" orange, acridine derivatives, ethidium-acridine heterodimer, ethidium monoazide, propidium iodide, SYTO dyes, SYBR Green 1, SYBR dyes, Pico Green, SYTOX dyes, and 7-aminoactinomycin D.<br>
in step (1) said accelerator agent is a non-intercalating fluorophore (especially one selected from the group consisting of biotin, rhodamine, Alexa dyes, BODIPY dyes, biotin conjugates, thiol-reactive probes, fluorescein and derivatives (including the "caged" probes), Oregon Green, Rhodamine Green, QSY dyes)) and said intensity is inversely correlated with formation of the triplex or quadruplex;<br>
in step (1 ) said accelerator agent is tethered to at least one of said first strand, said second strand, said third strand and said fourth strand;<br>
in step (1) adding an accelerator agent that is an intercalator that binds to the minor groove of the Watson-Crick duplex (or at least one of the two "Watson-Crick duplexes) ;<br>
in step (1) adding an accelerator agent (especially an organic liquid soluble in water, such as dimethyl formamide, ethanol, and glycerol) that at 25 °C is a liquid;<br>
in step (1) adding more than one accelerator agent;<br>
in step (1) adding an accelerator agent that is a condensation or decondensation agent as regards the Watson-Crick duplex;<br>
in step (1) adding an accelerator agent that is an analog of A, T, U, C, or G;<br>
in step (1) adding an accelerator agent selected from the group consisting of  lectins and polysaccharides;<br>
in steps (1) and (2 ) , buffering the mixture with a pH of about 5 to about 9;<br>
one cytosine in at least one C-G-C or G-C-G base triplet is positively charged;<br>
one cytosine in each C-G-C and G-C-G base triplet is positively charged;<br>
in step (2) the incubation time is not more than about two hours (more preferably not more than 1 hour;  and within either" time frame, prefererably at least 25%, more preferably at least 5 0%, of the possible multiplexes have been formed);<br>
said at least one accelerator agent is a minor groove nucleic acid binding molecule, which binds in a non-intercalating manner and binds with an association constant of at least 103 M-1;<br>
wherein the multiplex is part of an electrical circuit. (Alternatively, the invention is an electrical circuit comprising the multiplex structure.)<br>
It will be apparent to someone of ordinary skill in the art that the foregoing specific conditions also apply to the following method for making a quadruplex.<br>
In an alternatively phrased version, the method of forming the quadruplex comprises:<br>
(1)	providing a hybridization medium comprising a first<br>
strand, a second strand, a third strand, a fourth strand, water, a buffer and at least one accelerator agent; and<br>
(2)	incubating said hybridization medium for an incubation<br>
time effective to hybridize said second strand to said fourth strand to provide said multiplex structure wherein said multiplex structure comprises:<br>
a first strand containing a first sequence of nucleobases;<br>
a second strand containing . a second sequence of nucleobases, wherein said second strand is associated with said first strand by Watson-Crick bonding;<br>
a third strand containing a third sequence of nucleobases; and<br>
a fourth strand containing a fourth sequence of nucleobases, wherein said fourth strand is associated with said second strand and said third strand by Watson-Crick bonding.<br>
In particular embodiments of the methods of forming a quadruplex, the following apply alone or in combination (the descriptions that follow utilize the terminology of the method of making a quadruplex that specifies 4 separate strands but they are also applicable to the method that specifies a Watson-Crick duplex as one or two of the starting materials):<br>
at least one of said first strand and said second strand further comprises a pharmaceutical agent, and hybridization of said second strand to said fourth strand places said pharmaceutical agent an effective distance from a target on said third strand, said fourth strand or on another molecule associated with at least one of said third strand and said fourth strand;<br>
said pharmaceutical agent is a member selected from the group consisting of nucleic acids designed to bind gene promoter sequences of clinically relevant genes, nucleic acids designed to bind clinically relevant genes, and nucleic acids designed to bind origin-of- replication sites of pathogens;<br>
said third strand and said fourth strand are provided in said hybridization medium before said first strand and said second strand, and said first strand and said second strand are provided in dehydrated form prior to rehydration by contact with said hybridization medium;<br>
at least one of said first strand and said second strand is covalently labeled with a non-intercalating fluorophore and saic intensity is inversely correlated with said binding affinity;<br>
at least one accelerator agent is an intercalating fluorophore, and a fluorescent intensity of a test medium containing said multiplex structure is directly correlated with a binding affinity of said second strand for said fourth strand;<br>
hybridization, of said second strand to said fourth strand is detected as a change in a fluorescent, chemiluminescent, electrochemiluminescent or electrical signal;<br>
an intensity of said signal is correlated with a binding affinity between said second strand and said fourth strand;<br>
hybridization of said second strand to said fourth strand inactivates an activity associated with at least one of said third strand and said fourth strand. (B) Methods of detecting triplexes<br>
The methods of detecting a triplex, described generally in the Summary of the Invention section, can optionally be performed by incorporating one or more of the following into the method:<br>
carrying out the method as a homogenous- assay such that, during or prior to step (3) , single-stranded molecules that are not part of the triplex are not placed in a vessel or container separate from that containing the triplex;<br>
using the detection method to discriminate between a perfect base-pairing-rules match, a one-base mismatch (or deletion), and a 2-base mismatch (or deletion), between the duplex and the single-stranded molecule in the triplex (the method preferably comprising calibrating the method with molecules comprising known mismatches);<br>
using the extent of binding of an intercalator (e.g., as indicated by increased fluorescence) as an indication of the formation of the triplex;<br>
such that a wavelength at which said intercalating fluorophore fluoresces shifts to a second wavelength upon intercalation, a difference between said wavelength and said second wavelength indicating whether a complex between said probe and said target is a duplex or a triplex and whether said target is DNA or RNA;<br>
the probe is covalently labeled with a non-intercalating fluorophore and said intensity is inversely correlated with said binding affinity  (especially wherein said non-intercalating<br>
fluorophore is a member selected from the group consisting of biotin, rhodamine and fluorescein);<br>
using a fluorophore-labeled single stranded molecule as the single-stranded molecule;<br>
the method is a homogeneous assay conducted without providing a signal quenching agent on said target sequence (i.e., in the duplex) or on said probe (i.e., the single-stranded molecule) and/or without prior denaturation of said target sequence and/or without PCR amplification of said target sequence;<br>
said method is a homogeneous assay conducted without providing a signal quenching agent on said target sequence or on said probe;<br>
the probe has a partially charged or uncharged backbone;<br>
the probe comprises a PNA sequence and/or is ssPNA prepared by parallel synthesis;<br>
the probe and said target sequence are the same length;<br>
the probe is 5 to 30 nucleotides long;<br>
the fluorescence-exciting radiation is emitted from an argon ion laser at a wavelength from about 200 nm to about 1000 nm;<br>
the test sample has a volume of about 20 microliters containing about 10 femtomoles of target sequence and about 10 femtomoles of probe;<br>
the concentration of the target sequence in said sample is not more than 5 x 10-10 M;<br>
the concentration of the probe in the sample is not more than 5 x 10-10 M;<br>
the method is conducted on a biochip; the intercalating fluorophore is added to the medium in a form free of said probe and free of said target sequence;<br>
the intercalating fluorophore is a member selected from the, group consisting of YOYO-1, TOTO-1, ethidium bromide, ethidium homodimer-1, ethidium homodimer-2 and acridine.<br>
In an alteratively phrased aspect, the invention is a detection method comprising:<br>
providing a target double-stranded nucleic acid or nucleic<br>
acid analogue comprising a target sequence, wherein<br>
said target sequence contains at least one purine base<br>
and at least one pyrimidine base;<br>
providing a probe comprising a nucleic acid sequence or a<br>
nucleic acid analog sequence; providing an accelerator agent;<br>
adding said probe, said target sequence and said accelerator agent to a medium to provide a test sample containing a triplex complex comprising said probe bound to said target sequence, wherein all base triplets of said complex are members selected from the group consisting of A-T-A, T-A-T, U-A-T, T-A-U, A-U-A, U-A-U, G-C-G and C-G-C; irradiating said test sample with exciting radiation to<br>
cause the test sample to emit fluorescent radiation; detecting an intensity of said fluorescent radiation, wherein said intensity is correlated with a binding affinity between said probe and said target sequence; and determining from said intensity an extent of matching<br>
between said probe and said target sequence. In another alternatively phrased related aspect, the method comprises:<br>
providing a target nucleic acid or nucleic acid analogue<br>
having a target sequence, wherein said target sequence<br>
contains at least one purine base and at least one<br>
pyrimidine base;<br>
providing a double-stranded probe comprising a nucleic acid<br>
sequence or a nucleic acid analog sequence; providing a hybridization accelerator agent; adding said probe, said target and said hybridization accelerator agent to a medium to provide a test sample<br>
containing a Watson-Crick triplex comprising said probe bound to said target sequence; irradiating said test sample with exciting radiation to<br>
cause test sample to emit fluorescent radiation; detecting an intensity of said fluorescent radiation, wherein said intensity is correlated with a binding affinity between said probe and said target sequence; and determining from said intensity an extent of matching<br>
between said probe and said target sequence wherein said method is a homogeneous assay conducted without providing a signal quenching agent on said target sequence or on said probe; (C) Methods of detecting quadruplexes<br>
The methods of detecting a quadruplex, described generally in the Summary of the Invention section, can optionally be performed by incorporating one or more of the following into the method:<br>
carrying out the method as a homogenous assay, such that during or prior to step (3)  nucleic acid molecules that are not part of the quadruplex are not placed in a vessel or container, separate from that containing the quadruplex;<br>
the method is a homogeneous assay conducted without providing a signal quenching agent on said target sequence or on said probe (i.e., the second Watson-Crick duplex) and/or without prior denaturation of said target sequence and/or without PCR amplification of said target sequence;<br>
using the detection method to discriminate between a perfect base-pairing-rules match, a one-base mismatch (or deletion), and a 2-base mismatch (or deletion), between the first and second Watson-Crick duplexes (preferably by calibrating the method with molecules comprising known mismatches) ;<br>
using the extent of binding of an intercalator as an indication of the formation of the quadruplex (especially by<br>
using a fluorescent intercalator and using increased fluorescence as an indicator);<br>
the intercalating fluorophore is added to the medium in a form free of said probe and free of said target sequence;<br>
the intercalating fluorophore is a member selected from the group consisting of YOYO-1, TOTO-1, ethidium bromide, ethidium homodimer-l, ethidium homodimer-2 and acridine;<br>
a wavelength at which said intercalating fluorophore fluoresces shifts to a second wavelength upon intercalation, a difference between said wavelength and said second wavelength indicating whether a complex between said probe and said target is a duplex or a triplex and whether said target is DNA or RNA; the probe is covalently labeled with a non-intercalating fluorophore and said intensity is inversely correlated with said binding affinity (especially wherein said non-intercalating fluorophore is a member selected from the group consisting of biotin, rhodamine and fluorescein);<br>
using a fluorophore-labeled single stranded molecule for part of the second Watson-Crick duplex;<br>
the method further comprises quantifying the binding affinity;<br>
the probe has a partially charged or uncharged backbone; the probe comprises a PNA sequence and/or is ssPNA prepared by parallel synthesis;<br>
the probe and said target sequence are the same length; the probe is 5 to 30 nucleotides (or base pairs) long; the fluorescence-exciting radiation is emitted from an argon ion laser at a wavelength from about 200 nm to about 1000 nm;<br>
the test sample has a volume of about 20 microliters containing about 10 femtomoles of target sequence and about 10 femtomoles of probe;<br>
the concentration of the target sequence in said sample is not more than 5 x 10"xo M;<br>
the concentration of the probe in the sample is not more than 5 X 10-10 M;<br>
a ratio of said first strand and said second strand to said third strand and said fourth strand is about 10:1;<br>
concentrations of each of said first strand, said second strand, said third strand and said fourth strand are not more than 5 x 10-10 M;<br>
the method is conducted on a biochip.<br>
In an alternatively phrased related aspect, the method comprises:<br>
providing a target nucleic acid or nucleic acid analogue<br>
having a target sequence, wherein said target sequence<br>
contains at least one purine base and at least one<br>
pyrimidine base;<br>
providing a double-stranded probe comprising a nucleic acid<br>
sequence or a nucleic acid analog sequence; providing a hybridization accelerator agent; adding said probe,  said target and said hybridization accelerator agent to a medium to provide a test sample containing a Watson-Crick quadruplex comprising said probe bound to said target sequence; irradiating said test sample with exciting radiation to<br>
cause test sample to emit fluorescent radiation; detecting an intensity of said fluorescent radiation, wherein said intensity is correlated with a binding affinity between said probe and said target sequence; and determining from said intensity an extent of matching<br>
between said probe and said target sequence wherein said method is a homogeneous assay conducted without providing a signal quenching agent on said target sequence or on said probe. (D) The triplexes<br>
The triplex described generally in the Summary of the Invention section, can optionally have one or more of the following features:<br>
each strand is heteropolymeric with a G-C content between 25% and 75%;<br>
each strand is heteropolymeric with a G-C content between 10% and 90%, and furthermore the combined frequencies therein of purine-pyrimidine dimers and pyrimidine-purine dimers exceeds 25%;<br>
it is not in a cell (and not in a virus);<br>
it is stable at pH greater than 7.6 (but less than pH 9);<br>
it is in a medium at a pH greater than 7.6 (and preferably less than pH 9) ;<br>
the single-stranded nucleic acid or nucleic acid analog is 5 to 3 0 bases long and the double-stranded nucleic acid target is 8 to 3.3 X 109 base pairs long;<br>
the target sequence is heteropolymeric and contains 25% to 75% purine bases and 75% to 25% pyrimidine bases in any order, (preferably wherein the frequency of purine-pyrimidine dimers plus the frequency of pyrimidine-purine dimers exceeds 25%) ;<br>
the probe (i.e, the single-stranded molecule) is covalently bound to a double-stranded nucleic acid cleaving agent;<br>
the probe is covalently bound to a chemotherapeutic agent;<br>
the probe is covalently bound to a label (for example, a multi-molecule signaling complex, a redox pair, a chemi luminescent agent, an electrochemiluminescent agent, or in a preferred embodiment, a fluorophore, especially such that the fluorescent intensity of the complex is correlated with a binding affinity between the probe and the target sequence);<br>
the base pairing rules for the single-stranded nucleic acid molecule are, as regards one strand of the duplex, the Watson-Crick base-pairing rules, G-C and either A- T or A-U, and, as regards the other strand of the duplex are A-A and either T-T or U-U;<br>
the duplex substantially retains its Watson-Crick double helical structure, and the single-stranded molecule resides in a groove of the double helix;<br>
the accelerator agent forms a bond between part of the duplex and part of the single-stranded nucleic acid molecule;<br>
the accelerator agent is covalently linked to the single-stranded nucleic acid molecule;<br>
both strands of the Watson-Crick duplex are DNA (especially where all strands of the triplex are DNA) ;<br>
the accelerator reagent binds to a base in the Watson-Crick duplex, said base being one to which a base in the single-stranded nucleic acid molecule binds;<br>
the accelerator reagent binds to a base in the Watson-Crick duplex, said base not being one in the triplex;<br>
the accelerator agent binds to the phosphate backbone of the Watson-Crick duplex;<br>
the accelerator agent binds to more than one site on the Watson-Crick duplex, each site either on a base or a place on a phosphate backbone of said duplex;<br>
the accelerator agent binds to one site on the Watson-Crick duplex, said site either on a base or on a phosphate backbone of said duplex;<br>
the accelerator agent binds to a base in the Watson-Crick duplex and to a base in the single-stranded nucleic acid^ molecule;<br>
the accelerator agent binds to a base in the single-stranded nucleic acid molecule. (E) The Quadruplexes<br>
The quadruplex described generally in the Summary of the Invention section, can optionally have one or more of the following features:<br>
each of said four strands is heteropolymeric with a G-C content between 10% and 90%;<br>
the second and fourth strands are aligned in a parallel 31 to 5' direction and binding between those 2 strands is according to homologous base-pairing rules;<br>
the first and third strands are aligned in a parallel 5' to 3' direction and binding between those 2 strands is according to homologous base-pairing rules;<br>
the second and fourth strands are aligned in a parallel 3' to 5' direction and binding between said second and fourth strands is according to homologous base-pairing rules and furthermore the first and third strands are aligned in a parallel 5' to 3' direction and binding between said first and third strands is according to homologous base-pairing rules;<br>
the second and fourth strands are aligned in a parallel 3 ' to 5' direction and binding between those 2 strands is according. to Watson-Crick base-pairing rules;<br>
the first and third strands are aligned in a parallel 5 ' to 3' direction and binding between those 2 strands is according to Watson-Crick base-pairing rules;<br>
the second and fourth strands are aligned in a parallel 3' to 5' direction and binding between said second and fourth strands is according to Watson-Crick base-pairing rules and furthermore the first and third strands are aligned in a parallel 5' to 3' direction and binding between said first and third strands is according to Watson-Crick base-pairing rules;<br>
the first and fourth strands are aligned in anti-parallel 5" to 3' and 3' to 5' directions, respectively, and binding between the 2 strands is according to Watson-Crick base-pairing rules,-<br>
the second and third strands are aligned in anti-parallel 3' to 5' and 5' to 3' directions, respectively, and binding between those 2 strands is according to Watson-Crick base-pairing rules; the first and fourth strands are aligned in anti-parallel 5' to 31 and 3' to 5' directions, respectively, and binding between said first and fourth strands is according to Watson-Crick base-pairing rules and furthermore the second and third strands are aligned in anti-parallel 3' to 5' and 5' to 3' directions, respectively, and binding between said second and third strands is according to Watson-Crick base-pairing rules,-<br>
the first and fourth strands are aligned in anti-parallel 5 *, to 31 and 3' to 5' directions, respectively, and binding between those 2 strands is according to homologous base-pairing rules;<br>
the second and third strands are aligned in anti-parallel 31 to 5' and 5' to 3' directions, respectively, and binding between those 2 strands is according to homologous base-pairing rules;<br>
the first and fourth strands are aligned in anti-parallel 5' to 3' and 3' to 5' directions, respectively, and binding between said first and fourth strands is according to homologous base-pairing rules and furthermore the second and third strands are aligned in anti-parallel 3' to 5' and 5' to 31 directions, respectively, and binding between said second and third strands is according to homologus base-pairing rules;<br>
each interacting base of the said first strand interacts specifically with both the adjacent base on the said third strand and with the base on the said fourth strand, the base to which the said third strand base is bound;<br>
each interacting base of the said second strand interacts specifically with both the adjacent base on the said fourth strand and the base on the said third strand, the base to which the said fourth strand base is bound;<br>
it is an isolated, purified, artificial or synthetic quadruplex;<br>
each strand is heteropoly<br>
meric with a G-C content between 2 5% and 75%;<br>
each strand is heteropolymeric with a G-C content between 10% and 90%, and furthermore the combined frequencies therein of purine-pyrimidine dimers and pyrimidine-purine dimers exceeds 25%;<br>
it is not in a cell (and not in a virus);<br>
each said strand independently comprises a heteropolymeric nucleic acid or a heteropolymeric nucleic acid analogue; each said strand independently comprises DNA or RNA;<br>
each said strand independently comprises a heteropolymeric nucleic acid analogue containing an uncharged or partially charged backbone;<br>
one of said second strand or said fourth strand comprises DNA and the other of said second strand or said fourth strand comprises RNA, mRNA, hnRNA, rRNA, tRNA or cDNA;<br>
the second strand and said fourth strand are parallel homologous to each other;<br>
a major groove of said first strand and said second strand is placed in a minor groove of said third strand and said fourth strand;<br>
the second strand and said fourth strand are parallel complementary to each other;<br>
a major groove of said first strand and said second strand is placed in a minor groove of said third strand and said fourth strand;<br>
each nucleobase binds to no more than two other nucleobases;<br>
no strand is contiguous with another strand;<br>
the multiplex structure is substantially free of Hoogsteen bonding;<br>
the multiplex structure is substantially free of G-G quartets;<br>
the first strand and said second strand are 5 to 50 base pairs long;<br>
the third strand and said fourth strand are genomic DNA;<br>
the third strand and said fourth strand include a haplotype in genomic DNA;<br>
the third strand and fourth strand are PCR amplified products;<br>
wherein said multiplex structure is free of solid support; the multiplex structure is bound to a solid support (where the solid support is either electrically conductive or is not electrically conductive) ;<br>
wherein the multiplex structure further comprises a therapeutic, prophylactic or diagnostic agent bound to at least<br>
one of said first strand, said second strand, said third strand and said fourth strand;<br>
wherein the first strand and said second strand are each 5 to 30 bases long and said third strand and said fourth strand are each 8 to 3.3 X 109 base pairs long;<br>
wherein the fourth sequence contains 25% to 75% purine bases and 75% to 25% pyrimidine bases in any order (preferably wherein the frequency of purine-pyrimidine dimers plus the frequency of pyrimidine-purine dimers exceeds 25%);<br>
the first and second Watson-Crick duplexes (see the method of making a quadruplex in the Summary of the Invention section) each have a G-C content between 30 and 70%;<br>
both strands of the first Watson-Crick duplex are DNA (especially where all strands of the quadruplex are DNA);<br>
the accelerator reagent binds to a base in the first Watson-Crick duplex, said base being one to which a base in the second duplex binds;<br>
the accelerator reagent binds to a base in the first or second Watson-Crick duplexes, said base not being part of the quadruplex;<br>
the accelerator reagent binds to a phosphate backbone of the first or second Watson-Crick duplex;<br>
the accelerator reagent binds to more than one site on one of the first or second Watson-Crick duplexes, each site either on a base or on a phosphate backbone;<br>
the accelerator agent binds to one site on the Watson-Crick duplex, said site either on a base or on a phosphate backbone;<br>
the accelerator agent binds to a base in the first Watson-Crick duplex and to a base on the second Watson-Crick duplex;<br>
the accelerator agent binds to the minor groove of the first and/or second Watson-Crick duplex;<br>
the accelerator agent forms a bond between part of the first Watson-Crick duplex and part of the second Watson-Crick duplex.<br>
Additional aspects of the invention<br>
The invention provides triplex complexes comprising a single-stranded probe bound to a double-stranded nucleic acid target, wherein the probe comprises a heteropolymeric nucleic acid or a heteropolymeric nucleic acid analog, and all base triplets of the complex are members selected from the group consisting of A-T-A, T-A-T, U-A-T, T-A-U, A-U-A, U-A-U, G-C-G and C-G-C.<br>
Unlike certain Hoogsteen triplexes disclosed by the prior art, the triplexes of the invention are stable at pH values greater than 7.6. Moreover, the inventive triplexes do not require the presence of homopyrimidine sequences or homopurine sequences, as in certain prior art triplexes. For example, the target sequence can contain 25% to 75% purine bases and 75% to 25% pyrimidine bases in any order.<br>
Preferably the single-stranded nucleic acid or nucleic acid analog of  the  triplex  is  5  to 30  bases  long and  the double-stranded nucleic acid target is 8 to 3.3 X 109 base pairs-long.<br>
Triplex formation according to the invention is suitable for a variety of uses. For example, probes covalently bound to a double-stranded nucleic acid cleaving agent can be used to specifically cleave target sequences of double-stranded nucleic acids. Probes covalently bound to a chemotherapeutic agent can be used to specifically treat target sequences of double-stranded nucleic acids.<br>
In preferred embodiments, the invention provides a rapid, sensitive, environmentally friendly, and safe method for assaying binding between a double-stranded target and a single-stranded probe, wherein the target comprises a nucleic acid sequence or a nucleic acid analog sequence and the probe comprises a nucleic acid sequence or a nucleic acid analog sequence.<br>
Unlike certain prior art assays, the invention not only detects the presence of specific probe-target binding, but also provides qualitative and quantitative information regarding the<br>
nature of interaction between a probe and target. Thus, the invention enables the practitioner to distinguish among a perfect match, a one base pair mismatch, a two base pair mismatch, a three base pair mismatch, a one base pair deletion, a two base pair deletion and a three base pair deletion arising between a base sequence in the probe and in a strand of the double-stranded target.<br>
Embodiments of the invention comprise calibrating the, measured signal (e.g., fluorescent intensity) for a first probe-target mixture against the same type of signal exhibited by other probes combined with the same target, wherein each of the other probes differs from the first probe by at least one base.<br>
A calibration curve can be generated, wherein the magnitude of the measured signal (e.g., fluorescent intensity) is a function of the binding affinity between the target and probe. As the binding affinity between the target and a plurality of different probes varies with the number of mismatched bases, the nature of the mismatch(es) (A-G vs. A-C vs. T-G vs. T-C, etc.), the location of the mismatch(es) within the triplex, etc., the assay of the invention can be used to sequence the target.<br>
In embodiments, the signal measured can be the fluorescent intensity of a fluorophore included in the test sample. In such embodiments, the binding affinity between the probe and target can be directly or inversely correlated with the intensity, depending on whether the fluorophore signals hybridization through signal quenching or signal amplification. Under selected conditions, the fluorescent intensity generated by intercalating agents can be directly correlated with probe-target binding affinity, whereas the intensity of preferred embodiments employing a non-intercalating fluorophore covalently bound to the probe can be inversely correlated with probe-target binding affinity. The fluorescent intensity decreases for non-intercalating fluorophores as the extent of matching between the probe and target increases, preferably over a range inclusive of<br>
0-2 mismatches and/or deletions, more preferably over a range inclusive of 0-3 mismatches and/or deletions.<br>
The invention enables quantifying the binding affinity between probe and target. Such information can. be valuable for a variety of uses, including designing antisense drugs with optimized binding characteristics.<br>
Unlike prior art methods, the assay of the invention is preferably homogeneous. The assay can be conducted without separating the probe-target complex from the free probe and target prior to detecting the magnitude of the measured signal. The assay does not require a gel separation step, thereby allowing a great increase in testing throughput. Quantitative analyses are simple and accurate. Consequently the binding assay saves a lot of time and expense, and can be easily automated. Furthermore, it enables binding variables such as buffer, pH, ionic concentration, temperature, incubation time, relative concentrations of probe and target sequences, intercalator concentration, length of target sequences, length of probe sequences, and possible cofactor requirements to be rapidly determined.<br>
The assay can be conducted in, e.g., a solution within a well, on an impermeable surface or on a biochip.<br>
Moreover, the inventive assay is preferably conducted without providing a signal quenching agent on the target or on the probe.<br>
Although the inventors have previously disclosed the advantages of fluorescent intensity assays for hybridization (see, e.g., U.S. Patent Application No. 09/224,505, filed December 31, 1998), assays according to the present invention specifically detect triplexes of the probe and the double-stranded target, thus obviating the need to denature the target. While nucleic acid (and nucleic acid analog) probes have been known to form triplexes with certain limited classes of targets (see, e.g., Floris et al., supra, Dervan et al. , supra, Egholm et al., 365 Nature 566 (1993), and Tomac et al. , 118<br>
J.Am.Chem.Soc. 5544 (1996)), it is surprising that the inventors have been able to specifically assay triplexes formed between single-stranded nucleic acid (e.g., ssDNA and RNA) probes and double-stranded nucleic acid (e.g., dsDNA) targets, wherein the interaction between the probes and targets is based on Watson-Crick base pairing (at least in the sense that A binds to T (or U, in the case of RNA) and G binds to C) , rather than the very-limited Hoogsteen model of triplex hybridization of, e.g., Dervan et al. The term "Watson-Crick triplex," which is employed herein, is intended to crystallize these differences by limiting, the nature of base pairing between the single-stranded probe and the double-stranded target to A-T-A, T-A-T, U-A-T, T-A-U, A-U-A, U-A-U, G-C-G and/or C-G-C (including C+-G-C, and/or any other ionized species of base). These three-member groups are hereinafter denoted Watson-Crick base triplets and the resulting structures denoted Watson-Crick triplexes.<br>
Suitable probes for use in the inventive assay include, e.g., ssDNA, RNA, PNA and other nucleic acid analogs having uncharged or partially-charged backbones. Probe sequences having any length from 8 to 20 bases are preferred since this is the range within which the smallest  unique DNA sequences  of prokaryotes and eukaryotes are found. Probes of 12 to 18 bases are particularly preferred since this is the length of the smallest unique sequences in the human genome.  In embodiments, probes of 5 to 30 bases are most preferred. However, a plurality of shorter probes can be used to detect a nucleotide sequence' having a plurality of non-unique target sequences therein, which combine to uniquely identify the nucleotide sequence. The length of the probe can be selected to match the length of the target.<br>
The inventors have discovered the surprising development that they were able to specifically assay a wide-variety of triplexes formed in a Watson-Crick base-pair dependent manner between single-stranded nucleic acid (e.g., ssDNA, RNA, ssPNA and other analogs of DNA or RNA) probes and double-stranded nucleic<br><br>
acid (e.g., dsDNA) targets. The inventors have discovered that triplex formation and/or stabilization is enhanced by the presence of an intercalating agent in the sample being tested.<br>
The inventors have discovered that Watson-Crick triplex formation and/or stabilization is enhanced by the presence of cations in the sample being tested.  Suitable cations include, e.g.,  monovalent  cations,  such  as  Na+  (preferably  at  a concentration of 50mM to 125mM), K+, and other alkali metal ions; divalent cations, such as alkaline earth metal ions (e.g., Mg+2 and Ca+2) and divalent transition metal ions (e.g., Mn+2, Ni+2, Cd+2, Co*2 and Zn+2) ; and cations having a positive charge of at least  three,  such  as  Co(NH3)6+3,  trivalent  spermidine  and tetravalent  spermine.    Mn+2  is  preferably provided  at  a concentration of lOmM to 30mM.  Mg+2 is preferably provided at a concentration of 15mM to 20mM.  Ni+2 is preferably provided at a concentration of about 20mM.  In embodiments, Mg+2 and Mn+2 are' provided in combination at a concentration of lOmM each, 15mM each or 20mM each (i.e., 10-20 mM each).<br>
The amount of cation added to the medium in which the triplex forms depends on a number of factors, including the nature of the cation, the concentration of probe, the concentration of target, the presence of additional cations and the base content of the probe and target. The preferred cation concentrations and mixtures can routinely be discovered experimentally.<br>
The instant invention does not require the use of radioactive probes, which are hazardous, tedious and time-consuming to use, and need to be constantly regenerated. Probes of the invention are preferably safe to use and stable for years. Accordingly, probes can be made or ordered in large quantities and stored.<br>
In embodiments, the probe is labeled with a multi-molecule signaling complex or a redox pair, or with a label that elicits chemiluminescent or electrochemiluminescent properties.<br>
It is preferred that the probe or target (preferably the probe) have a fluorescent label covalently bound thereto. The label is preferably a non-intercalating fluorophore. In such embodiments, the fluorophore is preferably bound to the probe at either end. Preferred fluorescent markers include biotin, rhodamine and fluorescein, and other markers that fluoresce when' irradiated with exciting energy.<br>
The excitation wavelength is selected (by routine experimentation and/or conventional knowledge) to correspond to this excitation maximum for the fluorophore being used, and is preferably 200 to 1000 nm. Fluorophores are preferably selected to have an emission wavelength of 200 to 1000 nm. In preferred embodiments, an argon ion laser is used to irradiate the fluorophore with light having a wavelength in a range of 400 to 540 nm, and fluorescent emission is detected in a range of 500 to 750 nm.<br>
The assay of the invention can be performed over a wide variety of temperatures, such as, e.g., from 5 to 85°C. Certain prior art assays require elevated temperatures, adding cost and delay to the assay. On the other hand, the invention can be conducted at room temperature or below (e.g., at a temperature below 25°C).<br>
The reliability of the invention is independent of guanine and cytosine content in said target. Since G-C base pairs form three hydrogen bonds, while A-T base pairs form only two hydrogen bonds, target and probe sequences with a higher G or C content are more stable, possessing higher melting temperatures. Consequently, base pair mismatches that increase the GC content of the hybridized probe and target region above that present in perfectly matched hybrids may offset the binding weakness associated with a mismatched probe. Triplexes containing every possible base pair mismatch between the probe and the target proved to be more unstable than perfectly matched triplexes, always resulting in lower fluorescent intensities than did<br>
perfectly  complementary  hybrids,  when  an  intercalating fluorophore was used.<br>
The inventive assay is extremely sensitive, thereby obviating the need to conduct PCR amplification of the target. For example, it is possible to assay a test sample having a volume of about 20 microliters, which contains about 10 femtomoles of target and about 10 femtomoles of probe. Embodiments of the invention are sensitive enough to assay targets at a concentration of 5 X 10-9 M, preferably at a concentration of not more than 5 x 10"10 M. Embodiments of the invention are sensitive enough to employ probes at a concentration of 5 X 10-9 M, preferably at a concentration of not more than 5 x 10"10 M. It should go without saying that the foregoing values are not intended to suggest that the method cannot detect higher concentrations.<br>
The medium in which triplexes form can be any conventional medium known to be suitable for preserving nucleotides. See, e.g., Sambrook et al., "Molecular Cloning: A Lab Manual," Vol. 2 (1989). For example, the liquid medium can comprise nucleotides, water, buffers and standard salt concentrations. When divalent cations are used exclusively to promote triplex formation, chelators such as EDTA or EGTA should not be included in the reaction mixtures.<br>
Specific binding between complementary bases occurs under a wide variety of conditions having variations in temperature, salt concentration, electrostatic strength, and buffer composition. Examples of these conditions and methods for applying them are*-known in the art.<br>
Unlike many Hoogsteen-type triplexes, which are unstable or non-existent at pH levels above about 7.6, the Watson-Crick triplexes of the invention are stable over a wide range of pH levels, preferably from about pH 5 to about pH 9.<br>
It is preferred that triplexes be formed at a temperature of about 5°C to about 25°C for about one hour or less. Longer reaction times are not required, but incubation for up to 24<br>
hours in most cases did not adversely affect the triplexes. The fast binding times of Watson-Crick triplexes of the invention contrast with the much longer binding times for Hoogsteen triplex-based assays.<br>
Although not required, it is possible to facilitate triplex formation in solution by using certain reagents in addition to cations. Preferred examples of these reagents include single stranded binding proteins such as Rec A protein, T4 gene 32 protein, E. coli single stranded binding protein, major or minor nucleic acid groove binding proteins, viologen and intercalating substances such as ethidium bromide, actinomycin D, psoralen, and angelicin. Such facilitating reagents may prove useful in extreme operating conditions, for example, under abnormal pH levels or extremely high temperatures.<br>
The inventive assay can be used to, e.g., identify accessible regions in folded nucleotide sequences, to determine the number of mismatched base pairs in a hybridization complex, and to map genomes.<br>
The inventors may sometimes herein suggest that Watson-Crick triplexes result from hybridization of the probe to duplex target. While fluorophores tethered to the probe produced quenched fluorescent emissions upon being exposed to duplex targets containing a strand of Watson-Crick complementary bases, which indicates the occurrence of some kind of binding event, the inventors are not sure that what occurs in the Watson-Crick triplex is best described as hybridization in the sense traditionally associated with Watson-Crick duplex formation. While the formation of a Watson-Crick triplex may sometimes be referred to as a hybridization event herein, that is merely for convenience and is not intended to limit the scope of the invention with respect to how the formation of a Watson-Crick triplex can be best characterized.<br>
Unlike the quadruplexes discussed in the Background Section above, the preferred multiplex structures of the invention<br>
contain at least four strands of nucleic acid bonded together according to traditional Watson-Crick bonding rules.<br>
As used herein, the term "Watson-Crick bonding" is intended to define specific association between opposing pairs of nucleic acid (and/or nucleic acid analogue) strands via matched, opposing bases. While the formation of a Watson-Crick quadruplex may sometimes be referred to as a hybridization event herein, that is merely for convenience and is not intended to limit the scope of the invention with respect to how the formation of a Watson-Crick quadruplex can be best characterized.<br>
" The multiplex structures of the invention are preferably quadruplexes. Each strand of the multiplex independently comprises a nucleic acid or a nucleic acid analogue. Suitable nucleic acids include, e.g., DNA or RNA. Preferred nucleic acid analogues contain an uncharged or partially charged backbone (i.e., a backbone having a charge that is not as negative as a native DNA backbone).<br>
In certain embodiments, one of the second and fourth strands of the four-stranded quadruplex comprises DNA and the other of the second and fourth strands comprises RNA, mRNA, hnRNA, rRNA, tRNA or cDNA.<br>
In certain embodiments, the second strand and the fourth strand are parallel homologous to each other. In these embodiments, a major groove of the first and second strands is placed in a minor groove of the third and fourth strands.<br>
In other embodiments, the second and fourth strands are parallel complementary to each other.  In these embodiments, which possess "nested complementarity," a major groove of the first and second strands is placed in a minor groove of the third and fourth strands.<br>
In certain embodiments, each nucleobase binds to no more than two other nucleobases. In some of these embodiments, the bases of the second strand specifically bond (via Watson-Crick rules) to the matching bases of the first strand and to the matching bases of the fourth strand, and the bases of the fourth<br>
strand specifically bond (via Watson-Crick rules) to the matching bases of the third strand and to the matching bases of the second strand, wherein the bases of the first and third strands bind to no more than one other base each. Thus, in addition to the traditional Watson-Crick base pairs, such embodiments include the following Watson-Crick base triplets: A-T-A, T-A-T, U-A-T, T-A-U, A-U-A, U-A-U, G-C-G and/or C-G-C (including C+-G-C, and/or any other ionized species of base).<br>
In certain embodiments, it is believed that opposing bases of the first and third strands also bind to each other, in addition to: (a) the binding between opposing bases of the first and second strands; (b) the binding between opposing bases of the third and fourth strands; and (c) the binding between opposing, bases of the second and fourth strands.<br>
In certain embodiments of the multiplex structure of the invention, no strand is contiguous with another strand. That is, there are at least four separate strands. Although folded conformations and the like (e.g., hairpin turns, etc.) are within the scope of the invention, folded portions of a single strand do not make the strand count more than once toward the minimum of four separate strands.<br>
Multiplex structures of the invention preferably do not rely on Hoogsteen bonding or G-G quartets for maintenance of the multiplex structure, although insignificant amounts of Hoogsteen bonding and/or G-G quartets may be present. That is, multiplex structures of the invention are preferably substantially free of Hoogsteen bonding, and substantially free of G-G quartets.<br>
In certain embodiments, the first and second strands of the multiplex are 5 to 50 bases long (more preferably 5 to 3 0 bases long) and the third and fourth strands are 8 to 3.3 X 109 base pairs long. For example, the first and second strands can constitute a double-stranded probe and the third and fourth strands can constitute a double-stranded target, such as genomic DNA, which can contain a haplotype.<br>
In embodiments, the third strand and the fourth strand are PCR amplified products.<br>
The multiplexes of the invention can be present in solution, . on a solid support, in vitro  or in vivo.     The solid support can be   electrically  conductive   (e.g.,   an   electrode)   or non-conductive.<br>
Quadruplex formation according to the invention is suitable for a variety of uses. For example, double-stranded probes covalently bound to a double-stranded nucleic acid cleaving agent can be used to specifically cleave target sequences of double-stranded nucleic acids. Double-stranded probes covalently bound to a chemotherapeutic agent can be used to specifically treat target sequences of double-stranded nucleic acids. Thus, the invention encompasses multiplex structures further comprising a therapeutic, prophylactic or diagnostic agent bound to at least one of the first, second, third and fourth strands.<br>
In addition, multiplexes of the invention are suitable for use in nanoengineering, such as to provide electrical circuitry on a molecular (i.e., nanoscale) level. Further details regarding nanoengineering with nucleic acids can be found in U.S. Patent No. 5,948,897 to Sen et al. arid the references cited therein.<br>
Multiplex structures of the invention can be provided by a method comprising: providing a hybridization medium comprising the first strand, the second strand, the third strand, the fourth strand, water, a buffer and a promoter; and incubating the hybridization medium for an incubation time effective to hybridize the second strand to the fourth strand.<br>
The hybridization medium can include any conventional medium known to be suitable for preserving nucleotides. See, e.g., Sambrook et al. , "Molecular Cloning: A Lab Manual," Vol. 2 (1989) . For example, the medium can comprise nucleotides, water, buffers and standard salt concentrations. When divalent cations are used exclusively to promote quadruplex formation, chelators<br>
such as EDTA or EGTA should not be included in the reaction mixtures.<br>
Specific binding between complementary bases occurs under a wide variety of conditions having variations in temperature, salt concentration, electrostatic strength, and buffer composition. Examples of these conditions and methods for applying them are known in the art.<br>
Unlike many Hoogsteen-type multiplexes, which are unstable or non-existent at pH levels above about 7.6, the Watson-Crick' multiplexes of the invention are stable over a wide range of pH levels, preferably from about pH 5 to about pH 9.<br>
Moreover, the inventive multiplexes do not require the presence of homopyrimidine sequences or homopurine sequences, as in certain prior art quadruplexes. For example, the target sequence can contain 25% to 75% purine bases and 75% to 25% pyrimidine bases in any order.<br>
It is preferred that multiplexes be formed at a temperature of about 5°C to about 25°C for about two hours or less. The incubation time is preferably less than five minutes, even at room temperature. Longer reaction times are not required, but incubation for up to 24 hours in most cases did not adversely affect the quadruplexes. The fast binding times of Watson-Crick quadruplexes of the invention contrast with the much longer binding times for Hoogsteen quadruplexes.<br>
The promoter in the hybridization medium is preferably an intercalating agent or a cation. The intercalating agent can be, e.g., a fluorophore, such as a member selected from the group consisting  of  Y0Y0-1,  T0T0-1,  ethidium bromide,  ethidium homodimer-1, ethidium homodimer-2 and acridine.<br>
Suitable cations include, e.g., monovalent cations, such as Na+ (preferably at a concentration of 50mM to 125mM) , K+, and other alkali metal ions; divalent cations, such as alkaline earth metal ions (e.g., Mg+2 and Ca+2) and divalent transition metal ions (e.g., Mn+2, Ni+2, Cd+2, Co+2 and Zn+2) ; and cations having a positive charge of at least three, such as Co(NH3)6+3, trivalent<br>
spermidine and tetravalent spermine. Mn+2 is preferably provided at a concentration of lOmM to 45mM. Mg+2 is preferably provided at a concentration of lOmM to 45mM. Ni+2 is preferably provided at a concentration of about 20mM. In embodiments, Mg+a and Mn+2 are provided in combination at a concentration of lOmM each, 15mM each, 20mM each, 25mM each, 30mM each, 35 mM each, or 40 mM each (i.e., 10-40 mM each).<br>
The amount of cation added to the medium in which the multiplex forms depends on a number of factors, including the nature of the cation, the concentration of probe, the concentration of target, the presence of additional cations and the base content of the probe and target. The preferred cation concentrations and mixtures can routinely be discovered experimentally.<br>
Although not required, other promoters include, e.g. , single stranded binding proteins such as Rec A protein, T4 gene 32 protein, E. coli single stranded binding protein, major or minor nucleic acid groove binding proteins, viologen and additional intercalating substances such as actinomycin D, psoralen, and angelicin. Such facilitating reagents may prove useful in extreme operating conditions, for example, under abnormal pH levels or extremely high temperatures.<br>
The invention also enables a method in which hybridization of the second strand to the fourth strand inactivates an activity associated with at least one of the third strand and the fourth strand. Thus, at least one of the first strand and the second strand further comprises a pharmaceutical agent, wherein hybridization of the second strand to the fourth strand places the pharmaceutical agent an effective distance from a target on the third strand, the fourth strand or on another molecule associated with at least one of the third strand and the fourth strand. The pharmaceutical agent is preferably a member selected from the group consisting of nucleic acids designed to bind promoter sequences of clinically relevant genes, nucleic acids<br>
designed to bind clinically relevant genes, or nucleic acids designed to bind origin of replication sites of pathogens.<br>
In preferred embodiments, the invention provides a rapid, sensitive, environmentally friendly, and safe method for assaying binding between a single-stranded or double-stranded target and a double-stranded probe, wherein the target comprises a nucleic acid sequence or a nucleic acid analogue sequence and the probe comprises a nucleic acid sequence or a nucleic acid analogue sequence.<br>
The inventive assay can be used to, e.g., identify accessible regions in folded nucleotide sequences, to determine the number of mismatched base pairs in a hybridization complex, and to map genomes.<br>
The invention not only detects the presence of specific probe-target binding, but also provides qualitative and quantitative information regarding the nature of interaction between a probe and target. Thus, the invention enables the practitioner to distinguish among a perfect match, a one base pair mismatch, a two base pair mismatch, a three base pair mismatch, a one base pair deletion, a two base pair deletion and a three base pair deletion arising between a sequence in the double-stranded probe and in a sequence in the double-stranded target.<br>
Embodiments of the invention comprise calibrating the measured signal (e.g., fluorescence, chemiluminescence, electrochemiluminescence or electrical properties) for a first probe-target mixture against the same type of signal exhibited by other probes combined with the same target, wherein each of the other probes differs from the first probe by at least one base.<br>
A calibration curve can be generated, wherein the magnitude of the measured signal (e.g., fluorescent intensity) is a function of the binding affinity between the target and probe. As the binding affinity between the target and a plurality of different probes varies with the number of mismatched bases, the nature of the mismatch(es) (A-G vs. A-C vs.   T-G vs. T-C,   etc.),<br>
the location of the mismatch(es) within the guadruplex, etc., the assay of the invention can be used to sequence the target.<br>
In embodiments, the signal measured can be the fluorescent emission intensity of a fluorophore included in the test sample. In such embodiments, the binding affinity between the probe and target can be directly or inversely correlated with the emission intensity, depending on whether the fluorophore signals hybridization through signal quenching or signal amplification. Under selected conditions, the fluorescent emission intensity generated by intercalating agents can be directly correlated with probe-target binding affinity, whereas the emission intensity of preferred embodiments employing a non-intercalating fluorophore covalently bound to the probe can be inversely correlated with probe-target binding affinity. The fluorescent emission intensity decreases for non-intercalating fluorophores as the extent of matching between the probe and target increases, preferably over a range inclusive of 0-2 mismatches and/or deletions, more preferably over a range inclusive of 0-3 mismatches and/or deletions.<br>
The invention enables quantifying the binding affinity between probe and target. Such information can be valuable for a variety of uses, including designing antisense drugs with optimized binding characteristics.<br>
The assay of the invention is preferably homogeneous. The assay can be conducted without separating the probe-target complex from the free probe and free target prior to detecting the magnitude of the measured signal. The assay does not require a gel separation step, thereby allowing a great increase in testing throughput. Quantitative analyses are simple and accurate. Consequently the binding assay saves a lot of time and expense, and can be easily automated. Furthermore, it enables binding variables such as buffer, pH, ionic concentration, temperature, incubation time, relative concentrations of probe and target sequences, intercalator concentration, length of target sequences, length of probe sequences, and possible cofactor (i.e., promoter) requirements to be rapidly determined. The assay can be conducted in, e.g., a solution within a well or microchannel, on an impermeable surface or on a biochip. In certain embodiments, the third and fourth strands are provided in the hybridization medium before the first and second strands, and the first and second strands are provided in dehydrated form prior to rehydration by contact with the hybridization medium.<br>
Moreover, the inventive assay is preferably conducted without providing a signal quenching agent on the target or on the probe.<br>
Although the inventors have previously disclosed the advantages of fluorescent intensity assays for hybridization (see, e.g., U.S. Patent Application No. 09/224,505, filed December 31, 1998), certain embodiments of the inventive assay specifically detect quadruplexes of the probe and the' double-stranded target, thus obviating the need to denature the target. It is surprising that the inventors have been able to specifically assay quadruplexes formed between double-stranded probes and double-stranded targets, wherein the interaction between the probes and targets is based on Watson-Crick base pairing (at least in the sense that A binds to T (or U, in the case of RNA) and G binds to C) , rather than the very limited Hoogsteen model of quadruplex hybridization of, e.g., Pitner et al., supra.<br>
Suitable probes for use in the inventive assay include, e.g., dsDNA, dsRNA, DNA:RNA hybrids, dsPNA, PNA:DNA hybrids and other double-stranded nucleic acid analogues having uncharged or partially-charged backbones. Probe sequences having any length from 8 to 2 0 bases are preferred since this is the range within which the smallest unique DNA sequences of prokaryotes and eukaryotes are found. Probes of 12 to 18 bases are particularly preferred since this is the length of the smallest unique sequences in the human genome. In embodiments, probes of 5 to 30 bases are most preferred. However, a plurality of shorter probes<br>
can be used to detect a nucleotide sequence having a plurality of non-unique target sequences therein, which combine to uniquely identify the nucleotide sequence. The length of the probe can be selected to match the length of the target.<br>
The instant invention does not require the use of radioactive probes, which are hazardous, tedious and time-consuming to use, and need to be constantly regenerated. Probes of the invention are preferably safe to use and stable for years. Accordingly, probes can be made or ordered in large quantities and stored.<br>
In embodiments, the probe is labeled with a multi-molecule signaling complex or a redox pair, or with a label that elicits chemiluminescent or electrochemiluminescent properties.<br>
When a fluorescent intercalator is not present in the hybridization medium, it is preferred that the probe or target (preferably the probe) have a fluorescent label covalently bound thereto. The label is preferably a non-intercalating fluorophore or an intercalating fluorophore. In such embodiments, the fluorophore is preferably bound to the probe at either end. Preferred fluorescent markers include biotin, rhodamine, acridine and fluorescein, and other markers that fluoresce when irradiated with exciting energy.<br>
The excitation wavelength is selected (by routine experimentation and/or conventional knowledge) to correspond to this excitation maximum for the fluorophore being used, and is preferably 200 to 1000 nm. Fluorophores are preferably selected to have an emission wavelength of 200 to 1000 nm. In preferred embodiments, an argon ion laser is used to irradiate the fluorophore with light having a wavelength in a range of 400 to 540 nm, and fluorescent emission is detected in a range of 500 to 750 nm.<br>
The assay of the invention can be performed over a wide variety of temperatures, such as, e.g., from 5 to 850C. Certain prior art assays require elevated temperatures, adding cost and delay to the assay.  On the other hand, the invention can be<br>
conducted at room, temperature or below (e.g., at a temperature below 25°C).<br>
The reliability of the invention is independent of guanine and cytosine content in said target. Since G-C base pairs form three hydrogen bonds, while A-T base pairs form only two hydrogen bonds, target and probe sequences with a higher G or C content are more stable, possessing higher melting temperatures. Consequently, base pair mismatches that increase the GC content of the hybridized probe and target region above that present in perfectly matched hybrids may offset the binding weakness associated with a mismatched probe.<br>
The inventive assay is extremely sensitive, thereby obviating the need to conduct PCR amplification of the target. For example, it is possible to assay a test sample having a volume of about 20 microliters, which contains about 10 femtomoles of target and about 10 femtomoles of probe. Embodiments of the invention are sensitive enough to assay targets at a concentration of 5xl0"9 M, preferably at a concentration of not more than 5xl0"10 M. Embodiments of the invention are sensitive enough to employ probes at a concentration of 5xl0"9 M, preferably at a concentration of not more than 5xl0"10 M. It should go without saying that the foregoing values are not intended to suggest that the method cannot detect higher concentrations.<br>
The ratio of probe (e.g., first and second strands) to target (e.g., third and fourth strands) is 30:1 to 1:1, preferably about 10:1.<br>
The Examples will show that YOYO-1, a known minor groove inhabiting intercalator of duplex DNA, can facilitate and signal triplex association or binding of a DNA oligo with a duplex target, which is indicative of the degree of complementarity, ascertained on the basis of Watson-Crick base pair recognition between the bases on the oligo and the bases of the complementary sequence in the duplex target.<br>
The triplex binding observed can not be a version of the homopyrimidine triplex motif, well characterized in the literature, due to that complexes' requirement for high acid conditions. Similarly, if our typical wild type 33% GC content 15-mer oligo was evaluated as a binding partner to the "purine rich strand" in the duplex target, as is the requirement for the homopurine triplex motif, our oligo would be mismatched by 9 bases in either the oligo sequence or the putative binding site on the purine rich strand of the target duplex.<br>
The YOYO-l intercalator has been reported by Johansen and Jacobsen (1998) as inhabiting the minor groove of duplex DNA based upon NMR spectroscopic studies. It is also reported that YOYO-l acted to locally decondense the B conformation of the duplex DNA by 106° resulting in an overall helical repeat of 13 base pairs rather than the 10 base pairs usually found in a B conformation helical repeat.<br>
As a fluorophore, YOYO-l's emission was greatly enhanced by. the addition of complementary oligo to the medium bearing the duplex target. The enhanced emission associated with the interaction of the oligo and the duplex can be credited either to the YOYO-l beirig present in the duplex minor groove and emitting light more energetically upon the contacting of the oligo with the duplex or to the YOYO-l finding a second hospitable place to locate itself in the groove created by the oligo and the complementary strand in the duplex when the respective bases thereof contact one another. It is also possible that both events are occurring. Further studies, such as X-ray crystallography and NMR spectroscopy will establish the possible locations and interactions of YOYO-l with the Natural Triplex.<br>
Johansen and Jacobsen (1998) concluded that the decondensation of the duplex DNA by 106° is caused by the bis-intercalation of the YOYO-l chromophore while the polypropylene amine linker chain remains in the minor groove of the duplex. This may be part of the mechanism whereby the target is decondensed 106°,   a relaxation which will be expressed out from<br>
the site of the YOYO-1 interaction in both directions. It may also be possible that the +4 cationic nature of YOYO-1 located in the minor groove between the backbone of the duplex results in relaxation of the repulsion between the phosphate anionic charges in the immediate vicinity of the YOYO-1 binding site, causing the backbone strands to move closer to one another in the vicinity of the YOYO-1. Should a second YOYO-1 locate itself near a YOYO-1. present in the duplex minor groove, by inhabiting the groove created by the oligo and the complementary strand when contacting one another in the minor groove, that YOYO-1 would also act in reducing the anionic repulsion between the oligo backbone and the complementary sequence's region of the backbone.<br>
We also observed that reagents which condense duplex DNA have the ability to facilitate triplex formation. In addition to the well known ability of monovalent, divalent, and multivalent cations to condense duplex DNA, our results suggest that cations form bridges which make possible binding between bases in the oligo and bases in the complementary strand of the duplex. We showed this by first forming triplexes facilitated by cations and then destroying the bridging structures. The result of continued lasing of the medium in which the cations and triplex were present was the rapid disappearance of all triplex structures.<br>
Our experience with triplexes facilitated by YOYO-1 is that' they are readily formed at room temperature and persist for many hours when formed. Over time, mismatched oligos continue the same lower level of fluorophore emission indicating the same level of triplex formation. Cations on the other hand appear to result in transitory conditions which are conducive to triplex or quadruplex formations. The cations included intercalators such as YOYO-1, metal center cations such as MgCl2, or cationic peptides such as spermidine, all of which act on the conformation by interacting with anionic charges to result in the modification of the conformation of the duplex DNA target. The results varied with cation species, concentration or the presence of several species at varying concentrations.<br>
We have shown that appropriate concentrations of organic solvents allow modification of the target duplex DNA, which allows triplexes to occur.<br>
The transitory nature of much of triplex and quadruplex formation is congruent with the organism's requirement that many processes involving the accessing of sequence information from duplex DNA be easily reversible, that is activated and terminated. Nevertheless, such formations are sufficiently stable so that diagnostic and other assays can be based upon them.<br>
We have therefore discovered that it is condensation or decondensation of the duplex DNA by whatever means which makes possible recognition between the oligo and the complementary sequence sufficient to achieve a contacting of at least one base-on the oligo with a base in the complementary strand of the duplex target. There are probably a number of departures from the position usually enjoyed by adjacent base pairs in a sequence when the size of the helical repeat is modified by condensation or decondensation. This realignment of stacked base pairs in the modified helix most likely allows for a sufficient change in electron orbits from that present within the base stack of an unmodified double helix, to allow binding of the third strand to the complementary strand to commence.<br>
Duplex modification by either condensation or decondensation may occur in a rather homogeneous fashion with all anionic charges in the dsDNA and vicinity affected at generally the same time or the modification may be localized. In the latter case, a series of greatly varying modifications will occur in the DNA base pairs and backbones progressively distant from the locus of modification. In either case conditions can be created to allow a binding event between a base on the oligo and a base in the complementary strand of the duplex to generate triplex pairing. Once such a binding has been initiated it can be understood how the two flanking sequences of bases on the oligo might be swung into position to bind to bases in the flanking sequences of the complementary strand of the duplex.<br>
While our experiments show great stability in the triplexes facilitated by Y0Y0-1, many other reagents have effects on the conformation of the duplex DNA which result in Natural Triplexes • being formed transiently. These triple helix structures continue to progress into conformations which are less favorable to triplex maintenance, resulting in loss of the triplex structure. Any agent capable of acting to modify the conformation of target dsDNA must be monitored as to its effects over time to establish concentrations and incubation times suitable in the conditions of temperature, pH, etc. selected. This application teaches many methods of evaluating such reagents which can be used or modified by those skilled in the art so as to practice what is herein taught.<br>
The invention will be illustrated in more detail with reference to the Examples that follow, but it should be understood that the present invention is not deemed to be limited thereto.<br>
EXAMPLES The invention will be illustrated in more detail with reference to the following Examples, but it should be understood that the present invention is not deemed to be limited thereto. Example 1<br>
Example 1 demonstrates that the assay of the invention can discriminate between perfectly complementary dsDNA:ssDNA complexes and dsDNA.:ssDNA complexes containing 1 bp, 2 bp and 3 bp mismatches or deletions when a cationic DNA intercalator, Y0Y0-1 (Molecular Probes, Eugene, OR, USA), is present. NMR spectroscopic analyses of the mechanism of interaction between YOYO-1 and dsDNA have shown that the intercalation of Y0Y0-1 results in a localized decondensation of the dsDNA helix by 106° generating an overall helical repeat of 13 base pairs as opposed to the normal 10 base pair helical repeat in non-condensed B conformation dsDNA [J. Biomolec. Struct, and Dynamics 16, 205-222 (1998)] .<br>
Complementary sense and antisense 50-mer ssDNA target sequences, derived from exon 10 of the human cystic fibrosis gene [Nature 380, 207 (1996)] were synthesized on a DNA synthesizer (Expedite 8909, PerSeptive Biosystems) and purified by HPLC. Equimolar amounts of the complementary oligonucleotides were heated at 95°C for 10 min and allowed to anneal gradually as the temperature cooled to 21°C over 1.5 hours. DsDNA oligonucleotides were dissolved in ddH2O at a concentration of 1 pmole/µl.<br>
Sequence for the sense strand of the wild-type target DNA (SEQ ID NO:l) was:<br>
5' -TGG  CAC  CAT  TAA  AGA  AAA  TAT  CAT  CTT  TGG  TGT  TTC CTA  TGA  TGA  ATA  TA-3 '<br>
Sequence for the antisense strand of the wild-type target DNA (SEQ ID NO:l) was:<br>
5' -TAT  ATT  CAT  CAT  AGG  AAA  CAC  CAA  AGA  TGA  TAT  TTT CTT  TAA  TGG  TGC  CA-3' <br>
SEQ ID NO:2 was a 50-mer mutant dsDNA target sequence identical to wild-type target DNA (SEQ ID NO:l) except for a one base pair mutation (underlined) at amino acid position 507 at which the wild-type sense strand sequence CAT was changed to CGT.<br>
Sequence for the sense strand of SEQ ID NO:2 was: 5' -TGG  CAC  CAT  TAA  AGA  AAA  TAT  CGT  CTT  TGG  TGT  TTC CTA  TGA  TGA  ATA  TA-3'<br>
Sequence for the antisense strand of SEQ ID NO:2 was: 5' -TAT  ATT  CAT  CAT  AGG  AAA  CAC  CAA  AGA  CGA  TAT  TTT CTT  TAA  TGG  TGC  CA-3'<br>
SEQ ID NO:3 was "a 50-mer -mutant dsDNA target sequence identical to wild-type target DNA (SEQ ID NO:l) except for a consecutive two base pair mutation (underlined) at amino acid positions 506 and 507 at which the wild-type sense strand sequence CAT was changed to ACT.<br>
Sequence for the sense strand of SEQ ID NO:3 was: 5' -TGG  CAC  CAT  TAA  AGA  AAA  TAT  ACT  CTT  TGG  TGT  TTC CTA  TGA  TGA  ATA  TA-3'<br>
Sequence for the antisense strand of SEQ ID NO:3 was: 5 ' -TAT  ATT  CAT  CAT  AGG  AAA  CAC  CAA  AGA  GTA  TAT  TTT CTT  TAA  TGG  TGC  CA-3'<br>
SEQ ID NO: 4 was a 50-mer mutant dsDNA target sequence identical to wild-type target DNA (SEQ ID N0:1) except for a consecutive three base pair mutation (underlined) at amino acid positions 506 and 507 at which the wild-type sense strand sequence CAT was changed to ACG.<br>
Sequence for the sense strand of SEQ ID NO:4 was: 5 ' -TGG  CAC  CAT  TAA  AGA  AAA  TAT  ACG  CTT  TGG  TGT  TTC CTA  TGA  TGA ATA  TA-3'<br>
Sequence for the antisense strand of SEQ ID NO:4 was: 5' -TAT  ATT  CAT  CAT  AGG  AAA  CAC  CAA  AGC  GTA  TAT  TTT CTT  TAA  TGG  TGC  CA-3 '<br>
SEQ ID NO: 5 was a 47-mer mutant dsDNA target sequence identical to wild-type target DNA (SEQ ID NO: 1) except for a consecutive three base pair deletion (indicated by three dots) at amino acid positions 507 and 508 at which the wild-type sense strand sequence CTT is deleted.<br>
Sequence for the sense strand of SEQ ID NO: 5 was: 5' -TGG  CAC  CAT  TAA AGA AAA TAT  CAT  . . .  TGG  TGT  TTC CTA  TGA  TGA ATA  TA-3'<br>
Sequence for the antisense strand of SEQ ID NO:5 was: 5' -TAT  ATT  CAT  CAT AGG  AAA  CAC  CA .  . . A  TGA  TAT  TTT CTT  TAA  TGG  TGC  CA-3 '<br>
Probe No. 1 was a 15-mer ssDNA probe designed to be completely complementary to a 15 nucleotide segment of the sense strand of the 50-mer wild-type target DNA (SEQ ID NO:l), overlapping amino acid positions 505 to 510 [Nature 380. 207 (1996)]. The chirality of the probe was opposite or antiparallel• to that of the sense strand in the target. Probe No. 1 was synthesized on a DNA synthesizer, purified by HPLC, and dissolved in ddH20 at a concentration of 1 pmole/µl -<br>
Sequence for Probe No. 1 was: 5'-CAC  CAA  AGA  TGA  TAT-3 '<br>
The hybridization reaction mixture (120 µl) contained the following: 6 pmoles of target dsDNA, 6 pmoles of ssDNA probe, 0.5 x TBE and 500 nM of the DNA intercalator YOYO-1 (Molecular Probes, Eugene, OR, USA). The reaction mixtures were incubated at room temperature (21°C) for 5 minutes, placed into a quartz cuvette, irradiated with an argon ion laser beam having a wavelength of 488 nm and monitored repeatedly for fluorescent emission as the temperature increased with time, in a heated chamber. Concurrent temperature measurements of the samples were achieved by a software-controlled temperature probe placed directly into each sample. Maximum fluorescent intensities were plotted as a function of temperature for each sample analyzed.<br>
Figure  1  illustrates  that  the  highest  fluorescent intensities were achieved when the wild-type 50-mer non-denatured dsDNA target sequence (SEQ ID NO:l) was reacted with the 15-mer ssDNA Probe No. 1 from 30°C to 85°C. At temperatures below 65°C, the Tm of the 50-mer wild-type dsDNA, dsDNA:ssDNA complexes were formed,  enhanced by the DNA intercalator YOYO-1.   As the temperature increased above 65°C,  the dsDNA:ssDNA complexes converted to ssDNA:ssDNA complexes.  Clearly YOYO-1 was able to • intercalate and fluoresce efficiently in both types of complexes. In contrast, incompletely complementary probe and target combinations generating a 1 bp mismatch (SEQ ID NO:2 + Probe No. 1), a consecutive 2 bp mismatch (SEQ ID NO:3 + Probe No. 1), a consecutive 3 bp mismatch (SEQ ID NO:4 + Probe No. 1) and a 3 bp deletion (SEQ ID NO: 5 + Probe No. 1) resulted in fluorescent intensities that were 57%, 94%, 97% and 98% lower at 30°C, and 47%, 79%, 92% and 91% lower at 65°C, respectively, than that observed with the perfectly matched sequences (Fig. 1).  Control samples comprising 50-mer dsDNA targets plus 500 nM YOYO-1 exhibited levels of fluorescence which were at or below the level of fluorescence observed with 3 bp mismatched complexes at 30°C (data not shown) . The level of fluorescence emitted by the ssDNA Probe No. 1 plus 500 nM YOYO-1 sample was identical to that emitted by YOYO-1 alone (data not shown).  As the temperature<br>
increased above 65°C, the degree of discrimination between perfect match and the base pair mismatches decreased, indicative of the gradual breakdown of the dsDNA:ssDNA structure.  By 85°C, the fluorescent intensities achieved by a 1 bp mismatch, a 2 bp mismatch, a 3 bp mismatch and a 3 bp deletion were 40%, 63%, 92% and 83% lower than that obtained by the perfect match (Fig. 1). At  any  given  temperature,  the  characteristic  level  of fluorescence emitted by each complex was monitored over time and was stable between 5 minutes and 24 hours. The presence of the DNA decondensing agent, YOYO-1, allowed a ssDNA probe to be used to differentiate between perfectly' complementary dsDNA:ssDNA complexes and those containing 1 bp, 2 bp or 3 bp mismatches or deletions, without the requirement for prior denaturation of dsDNA targets.    Example 2<br>
To ensure that the fluorescent intensity assay using a DNA decondensing agent, ssDNA probes and non-denatured dsDNA targets, would apply to probe and target DNAs possessing dramatically different percent GC contents (and potentially different annealing temperatures) , new 15-mer ssDNA probes and 50-mer dsDNA target sequences were synthesized, purified and annealed as above. Both ssDNA probes and dsDNA targets were dissolved in ddH2O at a concentration of 1 pmole/µl.<br>
SEQ ID NO: 6 was a 50-mer dsDNA target sequence modified from SEQ ID NO:l, wherein the percent GC content was changed from 30% to 52%.<br>
Sequence for the sense strand of the wild-type target DNA (SEQ ID NO:6) was:<br>
5 ' -GAG  CAC  CAT  GAC  AGA  CAC  TGT  CAT  CTC  TGG  TGT  GTC CTA  CGA  TGA  CTC  TG-3 '<br>
Sequence for the antisense strand of the wild-type target DNA (SEQ ID NO:6) was:<br>
5'-CAG  AGT  CAT  CGT  AGG  ACA  CAC  CAG  AGA  TGA  CAG  TGT CTG  TCA  TGG  TGC  TC-3'<br>
Sequence for the antisense strand of the wild-type target DNA (SEQ ID NO:6) was:<br>
5'-CAG  AGT  CAT  CGT  AGG  ACA  CAC  CAG  AGA  TGA  CAG  TGT CTG  TCA  TGG  TGC  TC-3'<br>
SEQ ID NO:7 was a 50-mer mutant dsDNA target sequence identical to SEQ ID NO:6, except for a one base pair mutation (underlined), at which the sense strand sequence CTC was changed to CTT.<br>
Sequence for the sense strand of mutant SEQ ID NO:7 was: 5' -GAG  CAC  CAT  GAC  AGA  CAC  TGT  CAT  CTT  TGG  TGT  GTC CTA  CGA  TGA  CTC  TG-3 '<br>
Sequence for the antisense strand of mutant SEQ ID NO: 7 was: 5' -CAG AGT CAT CGT AGG ACA CAC CAA AGA TGA CAG TGT CTG  TCA  TGG  TGC  TC-3 '<br>
SEQ ID NO: 8 was a 50-mer mutant dsDNA target sequence identical to SEQ ID NO: 6, except for a one base pair mutation (underlined) , at which the sense strand sequence CAT was changed to CGT.<br>
Sequence for the sense strand of mutant SEQ ID NO:8 was: 5 ' -GAG  CAC  CAT  GAC  AGA  CAC  TGT  CGT  CTC  TGG  TGT  GTC' CTA  CGA  TGA  CTC  TG-3 •<br>
Sequence for the antisense strand of mutant SEQ ID NO:8 was: 5 ' -CAG AGT CAT CGT AGG ACA CAC CAG AGA CGA CAG TGT CTG  TCA  TGG  TGC  TC-31<br>
SEQ ID NO: 9 was a 50-mer mutant dsDNA target sequence identical to SEQ ID NO: 6, except for a one base pair mutation (underlined), at which the sense strand sequence CAT was changed to CTT.<br>
Sequence for the sense strand of mutant SEQ ID NO:9 was: 5' -GAG  CAC  CAT  GAC  AGA  CAC  TGT  CTT  CTC  TGG  TGT  GTC CTA  CGA  TGA  CTC  TG-3'<br>
Sequence for the antisense strand of mutant SEQ ID NO: 9 was: 5* -CAG AGT CAT CGT AGG ACA CAC CAG AGA AGA CAG TGT CTG  TCA  TGG  TGC  TC-3'<br>
SEQ ID NO:10 was a 50-mer mutant dsDNA target sequence identical to SEQ ID NO: 6, except for a one base pair mutation (underlined), at which the sense strand sequence CTC was changed to CCC.<br>
Sequence for the sense strand of mutant SEQ ID NO:10 was:<br>
5' -GAG  CAC  CAT  GAC  AGA  CAC  TGT  CAT  C£C  TGG  TGT  GTC CTA  CGA  TGA  CTC  TG-3 '<br>
Sequence for the antisense strand of mutant SEQ ID NO: 10 was:<br>
5' -CAG  AGT  CAT  CGT  AGG  ACA  CAC  CAG  GGA  TGA  CAG  TGT CTG  TCA  TGG  TGC  TC-3'<br>
SEQ ID NO:11 was a 50-mer mutant dsDNA target sequence' identical to SEQ ID NO:6, except for a one base pair mutation (underlined), at which the sense strand sequence CTC was changed to CGC.<br>
Sequence for the sense strand of mutant SEQ ID NO-.11 was: 5' -GAG  CAC  CAT  GAC  AGA  CAC  TGT  CAT  CGC  TGG  TGT  GTC CTA  CGA  TGA  CTC  TG-3'<br>
Sequence for the antisense strand of mutant SEQ ID NO: 11 was:<br>
5' -CAG  AGT  CAT  CGT  AGG  ACA  CAC  CAG  CGA  TGA  CAG  TGT CTG  TCA  TGG  TGC  TC-3'<br>
SEQ ID NO:12 was a 50-mer mutant dsDNA target sequence identical to SEQ ID NO:6, except for a consecutive two base pair mutation (underlined), at which the sense strand sequence CAT was changed to ACT.<br>
Sequence for the sense strand of mutant SEQ ID NO:12 was: 5 ' -GAG  CAC  CAT  GAC  AGA  CAC  TGT  ACT  CTC  TGG  TGT  GTC CTA  CGA  TGA  CTC  TG-3'<br>
Sequence for the antisense strand of mutant SEQ ID NO: 12 was:<br>
5 ' -CAG  AGT  CAT  CGT  AGG  ACA  CAC  CAG  AGA  GTA  CAG  TGT CTG  TCA  TGG  TGC  TC-3'<br>
SEQ ID NO:13 was a 50-mer dsDNA target sequence modified from SEQ ID NO:l, wherein the percent GC content was changed from 30% to 72%.<br>
Sequence for the sense strand of the wild-type target DNA (SEQ ID NO:13) was:<br>
5' -GAG  CAC  CCT  CCC  AGG  CAC  GGT  CGT  CCC  TGG  TGC  GAC CTC  CGA  CGA  GCG  TG-3'<br>
Sequence for the antisense strand of the wild-type target DNA (SEQ ID NO:13) was:<br>
5'-CAC  GCT  CGT  CGG  AGG  TCG  CAC  CAG  GGA  CGA  CCG  TGC CTG  GGA  GGG  TGC  TC-3 '<br>
SEQ ID NO:14 was a 50-mer mutant dsDNA target sequence identical to SEQ ID NO:13, except for a one base pair mutation (underlined), at which the sense strand sequence CGT was changed to CAT.<br>
Sequence for the sense strand of mutant SEQ ID NO:14 was: 5 ' -GAG  CAC  CCT  CCC  AGG  CAC  GGT  CAT  CCC  TGG  TGC  GAC CTC  CGA  CGA  GCG  TG-3'<br>
Sequence for the antisense strand of mutant SEQ ID NO: 14 was:<br>
5 ' -CAC  GCT  CGT  CGG  AGG  TCG  CAC  CAG  GGA  TGA  CCG  TGC CTG  GGA  GGG  TGC  TC-3'<br>
SEQ ID NO: 15 was a 50-mer' mutant dsDNA target sequence identical to SEQ ID NO: 13, except for a consecutive two base pair mutation (underlined) , at which the sense strand sequence CGT was changed to ATT.<br>
Sequence for the sense strand of mutant SEQ ID NO:15 was: 5' -GAG  CAC  CCT  CCC  AGG  CAC  GGT  ATT  CCC  TGG  TGC  GAC CTC  CGA  CGA  GCG  TG-3 '<br>
Sequence for the antisense strand of mutant SEQ ID NO: 15 was:<br>
5 ' -CAC  GCT  CGT  CGG  AGG  TCG  CAC  CAG  GGA  ATA  CCG  TGC CTG  GGA  GGG  TGC  TC-3"<br>
Probe No.  2 was a 15-mer ssDNA probe designed to be completely complementary to a 15 nucleotide segment of the sense-strand of the 50-mer wild-type target DNA (SEQ ID NO: 6) .  The chirality of the probe was opposite or antiparallel to that of the sense strand in the target.<br>
Sequence for Probe No. 2 was: 5'-CAC  CAG  AGA  TGA  CAG-3 '<br>
Probe No. 3 was a 15-meir ssDNA probe designed to be completely complementary to a 15 nucleotide segment of the sense<br>
strand of the 50-mer wild-type target DMA (SEQ ID NO:13). The chirality of the probe was opposite or antiparallel to that of the sense strand in the target.<br>
Sequence for Probe No. 3 was: 5' -CAC  CAG  GGA  CGA  CCG-3'<br>
The hybridization assay conditions were identical to that described in Example 1.<br>
When the ssDNA Probe No. 2 (with a 53% GC content) was reacted with the 50-mer wild-type dsDNA target (SEQ ID NO:6) and mutant dsDNA targets (SEQ ID N0:8 and SEQ ID N0:12) , dsDNA:ssDNA complexes were formed at low temperatures under non-denaturing conditions (Fig. 2A) . While perfectly matched DNA complexes achieved the highest fluorescent intensities, incompletely complementary complexes with a 1 bp mismatch (SEQ ID NO: 8 + Probe No. 2) and a consecutive 2 bp mismatch (SEQ ID NO:12 + Probe No. 2) produced fluorescent intensities that were 63% and 95% lower, respectively, than that observed with the perfectly matched sequences at 30°C (Fig. 2A) . As the temperature increased, the-gradual breakdown of the dsDNA:ssDNA complex occurred, resulting in diminished fluorescent intensities and less discrimination between perfect match and the base pair mismatches. By 85°C, very little difference in fluorescence was seen between perfectly matched sequences and those containing base pair mismatches (Fig. 2A) .<br>
Similarly, in the presence of Y0Y0-1, dsDNA:ssDNA complexes were formed when the ssDNA Probe No. 3 (possessing a 73% GC content) was reacted with the corresponding 50-mer wild-type dsDNA target (SEQ ID NO: 13) and mutant dsDNA targets (SEQ ID NO:14 and SEQ ID NO:15). The fluorescent intensities for a 1 bp mismatched DNA complex (SEQ ID NO.-14 + Probe No. 3) and a consecutive 2 bp mismatched DNA complex (SEQ ID NO: 15 + Probe No. 3) were 48% and 64% lower, respectively, than that obtained by the perfectly matched sequences at 30°C (Fig. 2B). Fluorescence of all samples decreased as the temperature increased from 3 0°C<br>
to 85°C, indicative of diminished YOYO-1 binding and dsDNA:ssDNA complex breakdown.<br>
Regardless of the percent GC content of the ssDNA probes and dsDNA targets, YOYO-1 was able to facilitate dsDNA:ssDNA complex formation under non-denaturing conditions, to allow accurate discrimination between perfectly complementary sequences and those containing 1 or 2 bp mutations. Example 3<br>
The next examples will demonstrate the specificity of the assay utilizing different DNA condensing agents to promote and stabilize complex formation with non-denatured dsDNA targets and ssDNA-F probes.<br>
Probe No. 4 was a 15-mer antiparallel ssDNA probe identical to Probe No. 1 except it had an attached fluorescein moiety at the 5' position. Probe No. 4 was synthesized on a DNA synthesizer, purified by HPLC, and dissolved in ddH20 at a concentration of 1 pmole//il.<br>
Sequence for Probe No. 4 was: 5'-Flu-CAC  CAA AGA  TGA  TAT-3 '<br>
The hybridization reaction mixture (40 µl)    contained the following:   0.4 pmoles of target dsDNA,  4 pmoles of 5'-fluorescein labeled ssDNA Probe No. 4, 10 mM Tris-HCl, pH 7.5, and 10 mM to 125 mM NaCl.  The reaction mixtures were incubated at room temperature (21°C) for 1 hour, without prior denaturation of dsDNA targets.  Samples were placed into a quartz cuvette, irradiated with an argon ion laser beam having a wavelength of 488 nm and monitored for fluorescent emission.   The maximum fluorescent intensities occurred at a wavelength of 525 nm, the emission  wavelength  for  fluorescein.    The  intensity  of fluorescent emission was plotted as a function of wavelength for each sample analyzed.<br>
In the absence of NaCl or presence of 10 mM or 25 mM NaCl, no binding between the dsDNA targets and the antiparallel ssDNA-F probe was detected (data not shown).<br>
After a 1 hour incubation in the presence of 50 mM NaCl, dsDNA: ssDNA-F complexes consisting of perfectly complementary sequences (SEQ ID NO:l + Probe No. 4) formed readily, resulting in a 49% decrease in fluorescent emission intensity compared to that emitted by the control Probe No. 4 (labeled ssDNA-F) (Fig. 3).   By contrast,  incompletely complementary dsDNA:ssDNA-F complexes containing a 1 bp G-T mismatch (SEQ ID NO:2 + Probe No. 4) yielded a 11% decrease in fluorescent emission intensity compared to that exhibited by the Probe No. 4 control sample.<br>
The presence of 75 mM, 100 mM and 125 mM NaCl in the reaction mixture also resulted in fluorescent emission quenching consistent with significant amounts of complex formation between the perfectly matched SEQ ID NO:l target and antiparallel Probe No. 4, and significantly less quenching when the 1 bp G-T mismatched SEQ ID NO,:2 target and Probe No. 4 were present, producing similar fluorescent intensities to that observed in the presence of 50 mM NaCl (data not shown).<br>
Use of monovalent cations, which are known DNA condensing agents, facilitated DNA complex formation between non-denatured dsDNA targets and f luorescently labeled antiparallel ssDNA probes, to allow reliable differentiation between perfectly complementary DNA sequences and those containing a single 1 bp mismatch. The reaction occurred at room temperature within 1 hour of incubation at a ratio of probe to target of 10 to 1. The dsDNA targets and ssDNA probe used in this example contained a 33% GC content, and did not contain homopurine or homopyrimidine stretches of DNA. Despite the presence of 6 pyrimidine bases interspersed within the 15 nucleotide ssDNA probe, dsDNA:ssDNA complexes formed readily in a sequence specific manner. Example 4<br>
To ensure that the fluorescent intensity assay, which used 5'-fluorescein labeled ssDNA probes and non-denatured dsDNA targets in the presence of DNA condensing agents such as cations, would apply to probe and target DNAs possessing dramatically different  percent GC     contents  (and  potentially  different<br>
annealing preferences) , 15-mer ssDNA-F probes and 50-mer dsDNA target sequences (with varying percent GC contents) were synthesized, purified and annealed as above. Both ssDNA-F probes and dsDNA targets were dissolved in ddH20 at a concentration of 1 pmole/µl.<br>
Probe No. 5 was a 15-mer antiparallel ssDNA probe identical to Probe No. 2 except it had an attached fluorescein moiety at the 5' position.<br>
Sequence for Probe No. 5 was: 5'-Flu-CAC  CAG  AGA  TGA  CAG-3'<br>
Probe No. 6 was a 15-mer antiparallel ssDNA probe identical to Probe No. 3 except it had an attached fluorescein moiety at the 5' position.<br>
Sequence for Probe No. 6 was: 5 ' -Flu-CAC  CAG  GGA  CGA  CCG-3 '<br>
The assays performed in Example 3 were facilitated by the addition of monovalent cations in the reaction mixtures. The specificity of the assay was further examined utilizing divalent cations (instead of monovalent cations) to promote complex formation with dsDNA targets and ssDNA-F probes possessing various percent GC contents.<br>
The hybridization reaction mixture (40 µl) contained the following: 0.4 pmoles of target dsDNA, 4 pmoles of 5'-fluorescein labeled ssDNA probe, 10 mM Tris-HCl, pH 7.5, and 5 mM to 30 mM MnCl2 or 5 mM to 30 mM MgCl2 or 5 mM to 30 mM NiCls. The reaction mixtures were incubated at room temperature (21°C) for 1 hour, without prior denaturation of dsDNA targets. Samples were placed into a quartz cuvette, irradiated with an argon ion laser beam having a wavelength of 488 nm and monitored for fluorescent emission. The maximum fluorescent intensities occurred at a wavelength of 525 nm, the emission wavelength for fluorescein. The intensity of fluorescent emission was plotted as a function of wavelength for each sample analyzed.<br>
When the ssDNA-F Probe No. 5 (with a 53% GC content) was incubated with the 50-mer wild-type dsDNA target (SEQ ID N0:6)<br><br>
and mutant dsDNA targets (SEQ ID NO:7 to SEQ ID NO: 12) in the presence of 10 mM MnCl2, dsDNA: ssDNA-F complexes were formed at room temperature under non-denaturing conditions. While-perfectly matched DNA complexes yielded the maximum decrease in fluorescent intensity (a 43% decrease after a 1 hour incubation) , the less stable dsDNA:ssDNA-F complexes containing a 1 bp T-G mismatch (SEQ ID NO:7 + Probe No. 5) produced a fluorescent intensity that was 20% lower than that observed with Probe No. 5 alone after a 1 hour incubation (Fig. 4). dsDNA: ssDNA-F complexes that resulted in a 1 bp G-T mismatch (SEQ ID NO: 8 + Probe No. 5), a 1 bp T-T mismatch (SEQ ID NO:9 + Probe No. 5), a<br>
1	bp C-A mismatch (SEQ ID NO: 10 + Probe No. 5) and a consecutive<br>
2	bp A-G and C-T mismatch (SEQ ID NO: 12 + Probe No. 5) were all less stable than the perfectly matched dsDNA:ssDNA-F complex (SEQ ID NO: 6 + Probe No. 5) yielding fluorescent intensities in between that observed for Probe No. 5 alone and that observed for the perfectly matched DNA complex (data not shown). Except for the 1 bp T-T mismatched dsDNA: ssDNA-F complex, which was the least stable (resulting in only a 5% decrease in fluorescent intensity after 1 hour) , all of the other mismatched DNA complexes generated very similar fluorescent intensities. Only the dsDNA:ssDNA-F complex that contained a 1 bp G-A mismatch (SEQ ID NO:11 + Probe No. 5) yielded a fluorescent intensity lower than that produced by the perfectly matched dsDNA:ssDNA-F complex<br>
(data not shown).<br>
The inclusion of 20 mM MgCl2 or 20 mM MnCl2 or 20 mM NiCl2 also facilitated dsDNA:ssDNA-F complex formation when the ssDNA-F Probe No. 6 (possessing a 73% GC content) was reacted with the corresponding 50-mer wild-type dsDNA target (SEQ ID NO:13) and mutant dsDNA target (SEQ ID NO: 14) for 1 hour (data not shown) . As expected, the perfectly matched dsDNA:ssDNA-F complexes generated the maximum decreases in fluorescent intensity, while the less stable 1 bp A-C mismatched dsDNA:ssDNA-F complexes (SEQ ID NO: 14 + Probe No. 6) produced intermediate levels of fluorescence (data not shown).<br>
Perfectly matched dsDNA:ssDNA-F complexes (possessing a 33% GC content) (SEQ ID NO:l + Probe No. 4) formed readily within 1 hour in the presence of 10 mM MnCl2, resulting in a 57% decrease in fluorescent intensity compared to that emitted by Probe No. 4 alone (data not shown). These reaction conditions were highly unfavorable for dsDNA:ssDNA-F complexes that contained a 1 bp G-T mismatch (SEQ ID N0:2 + Probe No. 4), resulting in an increased fluorescence compared to that observed by Probe No. 4 alone (data not shown).<br>
Regardless of the percent GC content of the dsDNA targets and ssDNA probes, the addition of divalent cations such as Mn*2, Mg+2 or Ni+2 promoted dsDNArssDNA complex formation under non-denaturing conditions, to allow accurate discrimination between perfectly complementary sequences and those containing 1 bp mutations. Example 5<br>
The dsDNA:ssDNA complex formation assays in Examples 3 and 4 were performed in the presence of one type of monovalent or divalent cation. The next Examples will demonstrate the reliability of the assay of the invention to differentiate between perfect matches and 1 bp mismatches in dsDNArssDNA complexes when combinations of divalent cations were used as promoting agents for complex formation.<br>
The hybridization reaction mixture (40 µl) contained the. following: 0.4 pmoles of target dsDNA, 4 pmoles of 5'-fluorescein labeled ssDNA probe, 10 mM Tris-HCl, pH 7.5, and 5 mM to 2 0 mM each of MgCl2 and MnCl2. The reaction mixtures were incubated at room temperature (21 °C) for 1 hour, without prior denaturation of dsDNA targets. Samples were placed into a quartz cuvette, irradiated with an argon ion laser beam having a wavelength of 488 nm and monitored for fluorescent emission. The intensity of fluorescence was plotted as a function of wavelength for each sample analyzed.<br>
When the antiparallel ssDNA-F Probe No. 6 (with a 73% GC content) was incubated for 1 hour with the 50-mer wild-type dsDNA<br>
target (SEQ ID NO:13) in the presence of 20 mM MgCl2 and 20 mM MnCl2, perfectly complementary dsDNA:ssDNA-F complexes were formed efficiently, generating a 46% decrease in fluorescence compared to that emitted by Probe No. 6 alone (Fig. 5A) . These reaction conditions were highly unfavorable for dsDNA:ssDNA-F complexes that contained a 1 bp A-C mismatch (SEQ ID NO: 14 + Probe No. 6) , resulting in a 3% reduction in fluorescence compared to that observed with Probe No. 6 alone (Fig. 5A) . Very similar results were obtained when the same samples were incubated for 1 hour in the presence of 10 mM MgCl2 and 10 mM MnCl2, or 15 mM MgCl2 and 15 mM MnCl2 (data not shown) . The, addition of 5 mM MgCl2 and 5 mM MnCl2 was insufficient to allow complex formation between the antiparallel ssDNA-F Probe No. 6 and all dsDNA targets tested following a 1 hour incubation (data not shown).<br>
When the antiparallel ssDNA-F Probe No. 4 (with a 33% GC content) was incubated with the wild-type dsDNA target (SEQ ID NO:l) or mutant dsDNA targets (SEQ ID NO:2 and SEQ ID NO:3), in the presence of 10 mM MgCl2 and 10 mM MnCl2, minimal DNA complex formation was observed (data not shown) . However, incubation in the presence of 15 mM MgCl2 and 15 mM MnCl2 for 1 hour facilitated perfectly matched dsDNA:ssDNA-F (SEQ ID N0:1 + Probe No. 4) complex  formation,  as  evidenced  by  the  49%  decrease  in fluorescent intensity observed, compared to that obtained by Probe No. 4 (Fig. 5B) .  dsDNA:ssDNA-F complexes that resulted in a 1 bp G-T mismatch (SEQ ID NO:2 + Probe No. 4) or a 3 bp deletion (SEQ ID NO:3 + Probe No. 4) were very unstable in the presence of 15 mM MgCl2 and 15 mM MnCl2, yielding a 2% decrease in fluorescence and a 5% increase in fluorescence, respectively, compared to that emitted by Probe No.  4 alone  (Fig.  5B) . Treatment with 20 mM MgCl2 and 20 mM MnCl2 for 1 hour, resulted in a 68%, a 48% and a 6% reduction in fluorescence for perfectly matched dsDNA:ssDNA-F complexes, and for dsDNA:ssDNA-F complexes containing a 1 bp G-T mismatch or a 3 bp deletion, respectively, compared to that observed with Probe No. 4 alone (data not shown).<br>
As illustrated in Fig. 5C, in the presence of 10 mM MgCl2 and 10 mM MnCl2, the dsDNA:ssDNA-F complexes possessing a 53% GC content and containing perfectly complementary sequences (SEQ ID NO:6 + Probe No. 5) or a 1 bp T-G mismatch (SEQ ID NO:7 + Probe No. 5) generated fluorescent intensities after an 1 hour incubation that were 68% and 2 0% lower, respectively, than that emitted by the antiparallel Probe No. 5 alone.<br>
When the antiparallel ssDNA-F Probe No. 5 (with a 53% GC<br>
content) was incubated for 1 hour with the 50-mer wild-type dsDNA<br>
target (SEQ ID NO: 6) in the presence of 15 mM MgCl2 and 15 mM<br>
MnCl2, perfectly complementary dsDNA:ssDNA-F complexes were formed<br>
very efficiently, generating a 74% decrease in fluorescence<br>
compared to that achieved by Probe No. 5 alone (Fig. 5D) .  By<br>
contrast, dsDNA:ssDNA-F complexes that contained a 1 bp T-G<br>
mismatch (SEQ ID NO:7 + Probe No. 5) were much less stable in the<br>
presence of 15 mM MgCl2 and 15 mM MnCl2, yielding a 15% decrease<br>
in fluorescence compared to that emitted by Probe No. 5 alone<br>
after a 1 hour incubation (Fig. 5D) .  Similarly, dsDNA:ssDNA-F<br>
complexes that resulted in a 1 bp G-T mismatch (SEQ ID NO: 8 +<br>
Probe No. 5), a 1 bp C-A mismatch (SEQ ID NO:10 + Probe No. 5) , •<br>
a 1 bp G-A mismatch  (SEQ ID NO: 11 + Probe No.  5)  and a<br>
consecutive 2 bp A-G and C-T mismatch (SEQ ID NO: 12 + Probe No.<br>
5) were all much less stable than the perfectly matched DNA<br>
complex (data not shown) . When Probe No. 5 (containing a 53% GC<br>
content)  was reacted with the dsDNA target  SEQ  ID NO-. 3<br>
(containing a 33% GC content) , a 3% increase in fluorescence was<br>
observed compared to that obtained by Probe No. 5 alone (Fig.<br>
5D) , indicative of no DNA complex formation.  This result was<br>
expected considering this probe and target combination would<br>
result in a 5 bp mismatch.<br>
Collectively, Examples 3, 4 and 5 demonstrated that the addition of condensing agents such as monovalent cations or divalent cations (on their own or in combination), promoted DNA complex formation between dsDNA targets and fluorescently-labeled ssDNA probes,  possessing dramatically different percent GC<br>
contents, to allow accurate and reliable discrimination between perfectly complementary sequences and those containing various 1 bp mutations. Example 6<br>
DsDNA:ssDNA complexes facilitated by YOYO-1 readily form at room temperature within 5 minutes of incubation and generate fluorescent emissions at the same level of intensity for hours. Complexes containing base pair mismatches similarly emit fluorescent signals which persist, indicating the same level of complex formation over time. To examine the rate of formation, • stability and rate of disassociation of dsDNA-.ssDNA complexes formed in the presence of condensing agents such as cations, time course experiments were performed.<br>
The hybridization reaction mixture (4 0 µl) contained the following: 0.4 pmoles of non-denatured target dsDNA, 4 pmoles of 5'-fluorescein labeled ssDNA probe, 10 mM Tris-HCl, pH 7.5, and 10 mM MgCl2 and 10 mM MnCl2. The reaction mixtures were incubated at room temperature (21°C) for various periods ranging from 1 minute to 2 hours. Following incubation, samples were placed into a quartz cuvette, irradiated once with an argon ion laser beam having a wavelength of 488 nm and monitored for fluorescent emission. Further fluorescent measurements were taken of the same samples after subsequent multiple laser irradiation, at the indicated times (Fig. 6) . The intensity of fluorescence was plotted as a function of time for each sample analyzed.<br>
  The fluorescence emitted by control samples comprising 4 pmoles of Probe No. 5 plus 10 mM MgCl2 and 10 mM MnCl2, in the absence of target dsDNA, dramatically decreased 3-fold within just 5 minutes of incubation (data not shown) , and then steadily declined at a much slower rate within the next few hours (Fig. 6A) . This effect we refer to as "Cationic Quench". This inhibition of fluorescence, associated with increased incubation periods of ssDNA-F probes with specific cations,  occurred<br><br><br>
routinely in the presence of divalent cations, but not in the presence of monovalent cations (data not shown). This observation makes evident the importance of incubating the control sample in an experiment under exactly the same conditions that the test samples of an experiment are reacted. Multiple lasing of each ssDNA-F control sample after varying periods of incubation inhibited further quenching of the fluorophore, resulting in a steady level of fluorescence thereafter (Fig. 6A) . This result was entirely unanticipated.<br>
When the antiparallel ssDNA-F Probe No. 5 was incubated with the 50-mer wild-type dsDNA target (SEQ ID NO:6) in the presence of 10 mM MgCl2 and 10 mM MnCl2/ dsDNA:ssDNA-F complex formation was evident after 15 minutes of incubation resulting in a decrease in fluorescence, which was 6% greater than the progressive cationic quench of the control Probe No. 5 (Fig. 6B). Complex formation was greatly indicated after 30 and 60 minutes-of incubation of SEQ ID NO:6 with Probe No. 5 in the presence of 10 mM MgCl2 and 10 mM MnCl2; generating a 76% and 61% decrease in fluorescence, respectively, compared to that achieved by the cationically quenched Probe No. 5 alone (Fig. 6B). After 90 and 12 0 minutes of incubation in the presence of 10 mM MgCl2 and 10 mM MnCl2, no complex formation was being signaled (Fig. 6B) . The level of fluorescent emission seen at 90 and 120 minutes was wholly attributable to the cationic quench effect (compare Figs. 6A and 6B).<br>
By contrast, dsDNA:ssDNA-F complexes that contained a 1 bp T-G mismatch (SEQ ID NO:7 + Probe No. 5) formed at a slower rate and were much less stable once formed in the presence of 10 mM MgCl2 and 10 mM MnCl2. The 1 bp T-G mismatched complex was first observed after 30 minutes of incubation, and appeared to have been eliminated after 60 minutes of incubation (Fig. 6C). Once again, the probe was antiparallel to the complementary strand in the duplex (Fig. 6C).<br>
Multiple laser irradiation of perfectly complementary dsDNA:ssDNA-F complexes (SEQ ID NO:6 + Probe No. 5) formed after<br>
3 0 minutes or 60 minutes of incubation in the presence of 10 mM MgCl2 and 10 mM MnCl2 resulted in fluorescent emissions consistent with the destruction of these complexes at a rate characteristic for DNA complexes containing an antiparallel ssDNA probe (Fig. 6B) . When a subsequent measurement was made at 45 minutes after lasing of the perfectly complementary complex at 3 0 minutes, the emission intensity level was 1869, testimony to the rapidity with which the complex was destroyed (data not shown) . The level of fluorescent emission, after multiple lasing, returned to the cationically quenched values observed by the uncomplexed Probe No. 5 alone control (compare Figs. 6A and 6B) . The only exception was the perfectly matched complexes formed after 15 minutes of incubation and repeated irradiated thereafter (Fig. 6B) . In this case the fluorescent emission was not consistent with the destruction of the complexes (Fig. 6B) , even though further cationic quench of Probe No. 5, when multiply irradiated after a 15 minute incubation, was totally inhibited (Fig. 6A) . DsDNA: ssDNA-F complexes containing a 1 bp T-G mismatch (SEQ ID NO: 7 + Probe No. 5) were similarly apparently destroyed by multiple lasing (Fig. 6C).<br>
An experiment was performed to determine the basis for the<br>
effect of multiple lasing on the complexes.  It was found that<br>
when fresh cations were added to the reaction mixture which had<br>
been  lased  twice,  the  inhibition  of  cationic  quench  in<br>
fluorescence emitted by the ssDNA-F probe could not be reversed<br>
and  further  cationic  quench did  not  occur  upon  further<br>
incubation,  strongly suggesting that the ssDNA-F probe was<br>
inactivated by multiple irradiation, by a yet unknown mechanism<br>
(data not shown) .   Similarly, when fresh ssDNA-F probes were<br>
added to the reaction mixture which had been lased twice, after<br>
normalizing for the increased fluorescent emission of the fresh<br>
probe, no subsequent progressive cationic quenching was observed<br>
upon further incubation of  the reaction mixture,  strongly<br>
suggesting that the lased cations were somehow disabled (data not<br>
shown)Examples 1-6 demonstrated dsDNA:ssDNA complex formation in a sequence specific manner between dsDNA targets and ssDNA probes facilitated by either DNA decondensing agents such as Y0Y0-1 or by DNA condensing agents such as monovalent or divalent cations. The next examples will examine the rate of formation, stability and rate of disassociation of dsDNA:dsDNA complexes formed in the presence of various cations. These examples will show how the species of cation, the concentration of each cation, and the combination of different cations at different concentrations influence the rate of formation, stability and rate of disassociation of dsDNA:dsDNA complexes formed.<br>
Complementary sense and antisense 15-mer ssDNA probe sequences were synthesized on a DNA synthesizer and purified by HPLC. Equimolar amounts of the complementary oligonucleotides were heated at 95 °C for 10 min and allowed to anneal gradually as the temperature cooled to 21°C over 1.5 hours. DsDNA oligonucleotides were dissolved in ddH20 at a concentration of 1 pmole/µl.<br>
Probe No. 7 was a 15 bp dsDNA probe with an attached<br>
fluorescein moiety at each 5' position, designed to be completely<br>
homologous to a 15 bp segment of the 50-mer wild-type target DNA<br>
(SEQ ID NO:6) .     The antisense strand of Probe No. 7 was identical<br>
to Probe No. 5.<br>
Sequence for the sense strand of Probe No. 7 was:<br>
5'-Flu-CTG  TCA TCT  CTG  GTG-3'<br>
Sequence for the antisense strand of Probe No. 7 was : 5'-Flu-CAC  CAG  AGA  TGA  CAG-3'<br>
The hybridization reaction mixture (40 µl) contained the following: 0.4 pmoles of ,non-denatured target dsDNA, 4 pmoles of 5'-fluorescein labeled dsDNA probe, 10 mM Tris-HCl, pH 7.5, 70 mM to 90 mM KCl and 0 mM to 20 mM NaCl. The reaction mixtures were incubated at room temperature (21°C) for various periods ranging from 1 minute to 2 hours.  Following incubation, samples<br><br>
were placed into a quartz cuvette, irradiated once with an argon ion laser beam having a wavelength of 488 ran and monitored for fluorescent emission. The intensity of fluorescence was plotted as a function of time for each sample analyzed.<br>
The fluorescence emitted by control samples, comprising 4 pmoles of Probe No. 7 plus 90 mM KC1 in the absence of target dsDNA, remained relatively constant throughout the 120 minute incubation, indicating that no cationic quenching of the dsDNA-F' probe was occurring in the presence of the monovalent KCl (Fig. 7A). When the dsDNA-F Probe No. 7 was incubated with the 50-mer wild-type dsDNA target (SEQ ID NO:6) and mutant dsDNA target (SEQ ID NO: 7) in the presence of 90 mM KCl, dsDNA:dsDNA-F complexes were formed at room temperature under non-denaturing conditions within just 15 minutes of incubation and persisted for at least 12 0 minutes (Fig. 7A) . Maximum discrimination between perfectly matched and 1 bp mismatched dsDNA:dsDNA-F complexes was observed after 30 and 45 minutes of incubation in the presence of 90 mM KCl, generating fluorescent intensities that were 27% and 3% lower, respectively after 30 minutes, and 34% and 15% lower, respectively after 45 minutes, than that emitted by Probe No. 7 alone (Fig. 7A).<br>
In the presence of 70 mM KCl and 20 mM NaCl, the dsDNA:dsDNA-F complexes containing perfectly matched sequences (SEQ ID NO: 6 + Probe No. 7) or a 1 bp mismatch (SEQ ID NO: 7 + Probe No. 7) produced fluorescent intensities that were 47% and 19% lower, respectively after 30 minutes, and 35% and 14% lower, respectively after 45 minutes, than that achieved by Probe No. 7 alone (Fig. 7B) . A small amount of cationic quench of the dsDNA-F Probe No. 7 control sample was observed over the 120 minute incubation in the presence of 70 mM KCl and 20 mM NaCl (Fig. 7B) . This minimal cationic quench was caused by.the inclusion of NaCl, which when present alone causes a similar progressive fluorescence quench of Probe No. 7 (data not shown).<br>
The presence of 80 mM KCl and 10 mM NaCl, preferentially promoted perfectly matched dsDNA:dsDNA-F complex formation<br>
between the wild-type dsDNA target (SEQ ID NO:6) and dsDNA-F Probe No. 7, resulting in a decrease in fluorescent emission of 18%, 48% and 34% after a 30 minute, 45 minute and 60 minute incubation, respectively, compared to the fluorescence emitted by the control Probe No. 7 sample at these times (Fig. 7C) . By contrast, formation of the 1 bp mismatched dsDNA: dsDNA-F complexes (SEQ ID NO:7 + Probe No. 7) was very inefficient throughout the entire 120 minute incubation in the presence of 80 mM KCl and 10 mM NaCl, as evidenced by the low 1% to 5% reduction in fluorescence observed with these mismatched complexes compared to that exhibited by Probe No. 7 (Fig. 7C).<br>
Therefore the inclusion of monovalent cations such as KCl and NaCl at close to physiological concentrations promoted DNA complex formation between non-denatured dsDNA targets and fluorescently-labeled dsDNA probes. Complex formation occurred on the basis of homologous base pair recognition, with a measurable and significantly greater amount of complex formation between fully matched homologous duplex strands. The reaction occurred at room temperature within relatively short incubation periods of 15 minutes to 60 minutes at a ratio of probe to target of 10 to 1. The dsDNA targets and dsDNA probe used in this example were homologous, contained 53% GC content, and did not contain homopurine or homopyrimidine stretches on any DNA strand. The assay of the invention was able to identify perfectly matched dsDNA sequences and those containing a pair of mismatched bases, using dsDNA probes. Example 8<br>
The assays performed in Example 7 were facilitated by the addition of monovalent cations in the reaction mixtures. This example will demonstrate the rate of formation, stability and rate of dissassociation of dsDNA:dsDNA complexes formed in the presence of divalent cations. The reaction conditions were identical to that described in Example 7, except that KCl and NaCl was replaced with 30 mM to 40 mM each of MgCl2 and MnCl2.<br>
The control .dsDNA-F Probe No. 7 sample exhibited a progressive reduction in fluorescence with increased time of incubation in the presence of 30 mM MgCl2 and 3 0 mM MnCl2 (Fig. 8A) or 40 mM MgCl2 and 40 mM MnCl2 (Fig. 8B) . This cationic quench was similar to that observed with ssDNA-F Probe No. 5 in the presence of 10 mM MgCl2 and 10 mM MnCl2 (Fig. 6A) .<br>
When the dsDNA-F Probe No. 7 was incubated with the 50-mer<br>
wild-type dsDNA target (SEQ ID NO: 6) in the presence of 30 mM<br>
MgCl2 and 3 0 mM MnCl2, dsDNA:dsDNA-F complex formation was evident<br>
after 60 minutes of incubation resulting in a decrease in<br>
fluorescence, which was 13% greater than the progressive cationic<br>
quench of the control Probe No. 7 (Fig. 8A) .  Complex formation<br>
was greatly indicated after 75 minutes of incubation of SEQ ID<br>
NO:6 with Probe No. 7 in the presence of 30 mM MgCl2 and 30 mM<br>
MnCl2, generating an 81% decrease in fluorescence, compared to<br>
that achieved by the cationically quenched Probe No. 7 alone<br>
(Fig.  8A) .   DsDNA:dsDNA-F complexes that contained a 1 bp<br>
mismatch (SEQ ID N0:7 + Probe No. 7) also formed after 60 minutes<br>
and 75 minutes of incubation in the presence of 30 mM MgCl2 and.<br>
3 0 mM MnCl2, generating a 16% and 30% decrease in fluorescence,<br>
respectively, compared to that achieved by the cationically<br>
quenched Probe No. 7 alone (Fig. 8A).  After 90 and 120 minutes<br>
of incubation in the presence of 30 mM MgCl2 and 3 0 mM MnCl2, no<br>
complex formation was being signaled (Fig. 8A) .    After 90<br>
minutes,  the  level  of  fluorescence  observed  was  wholly<br>
attributable to the cationic quench effect (Fig. 8A) .<br>
In the presence of 40 mM MgCl2 and 40 mM MnCl2, the dsDNA:dsDNA-F complexes containing perfectly matched sequences (SEQ ID NO:6 + Probe No. 7) or a 1 bp mismatch (SEQ ID NO: 7 + Probe No. 7) produced fluorescent intensities that were 17% lower and 4% higher, respectively after 60 minutes, 36% and 22% lower, respectively after 75 minutes, and 57% lower and 0.2 % higher, respectively after 90 minutes, than that emitted by the cationically quenched Probe No. 7 alone (Fig. 8B) .  After 12 0<br>
minutes of incubation in the presence of 40 mM MgCl2 and 40 mM MnCl2, no complex formation was being signaled (Fig. 8B).<br>
The addition of divalent cations, such as MgCl2 and MnCl2, facilitated DNA complex formation between non-denatured dsDNA targets and fluorescently-labeled dsDNA probes, to allow accurate discrimination between perfectly matched homologous sequences and those containing 1 bp mutations. Approximately double the concentration of both MgCl2 and MnCl2 was required for the formation of dsDNA:dsDNA complexes compared to that required for the formation of dsDNA:ssDNA complexes. The rate of formation of the dsDNA:dsDNA complexes was slower in the presence of divalent cations than in the presence of monovalent cations. Once formed the divalent cation induced dsDNA:dsDNA complexes seemed to be stable over a shorter time period. Example 9<br>
The rate of formation, stability and rate of dissassociation of dsDNA.-dsDNA complexes formed in the presence of monovalent and divalent cations was examined next. The hybridization reaction mixture (40µl) contained the following: 0.4 pmoles of non-denatured target dsDNA, 4 pmoles of 5'-fluorescein labeled dsDNA probe, 10 mM Tris-HCl, pH 7.5, 60 mM to 80 mM KCl, 10 mM to 2 0 mM NaCl, 30 mM MgCl2 and 30 mM MnCl2. The reaction mixtures were incubated at room temperature (21°C) for various periods ranging from 1 minute to 150 minutes. Following incubation, samples were placed into a quartz cuvette, irradiated once with an argon ion-laser beam having a wavelength of 488 nm and monitored for fluorescent emission. The intensity of fluorescence was plotted as a function of time for each sample analyzed.<br>
The presence of various KCl and NaCl concentrations together with 3 0 mM MgCl2 and 30 mM MnCl2 resulted in a cat ionic quench in fluorescence of control dsDNA-F Probe No. 7 (Fig. 9) that was very similar to that observed in the presence of only the divalent cations (Fig. 8) . It seems that the inclusion of the monovalent cations did not affect the cationic quench of the dsDNA-F probe by the divalent cations.<br>
When the dsDNA-F Probe No. 7 was incubated with the 50-mer wild-type dsDNA target (SEQ ID NO:6) and mutant dsDNA target (SEQ ID NO: 7) in the presence of 60 mM KCl, 20 mM NaCl, 3 0 mM MgCl2 and 3 0 mM MnCl2, dsDNA: dsDNA-F complexes were formed at room temperature under non- denaturing conditions within just 15 minutes of incubation and persisted for at least 150 minutes (Fig. 9A) . Maximum discrimination between perfectly matched and 1 bp mismatched dsDNA:dsDNA-F complexes was observed after 30, 45 and 150 minutes of incubation, generating fluorescent intensities that were 43% and 16% lower, respectively after 30 minutes, 38% and 16% lower, respectively after 45 minutes, and 65% and 1% lower, respectively after 150 minutes, than that emitted by the cationically quenched Probe No. 7 alone (Fig. 9A) .<br>
In the presence of 70 mM KCl, 20 mM NaCl, 30 mM MgCl2 and 3 0 mM MnCl2, the dsDNA:dsDNA-F complexes containing perfectly matched-sequences (SEQ ID NO: 6 + Probe No. 7) or a 1 bp mismatch (SEQ ID NO:7 + Probe No. 7) produced fluorescent intensities that were 17% and 1% lower, respectively after 60 minutes, 48% and 11% lower, respectively after 75 minutes, 22% and 3% lower, respectively after 90 minutes, and 34% and 2% lower, respectively after 120 minutes, than that achieved by the cationically quenched Probe No. 7 alone (Fig. 9B).<br>
The presence of 80 mM KCl, 10 mM NaCl, 3 0 mM MgCl2 and 3 0 mM MnCl2, preferentially promoted perfectly matched dsDNA:dsDNA-F complex formation between the wild-type dsDNA target (SEQ ID NO: 6) and dsDNA-F Probe No. 7, after 30 minutes to 120 minutes of incubation. The perfectly matched dsDNA:dsDNA-F complexes generated a decrease in fluorescent emission of 27%, 43%, 29% and 52% after a 60 minute, 75 minute, 90 minute and 120 minute incubation, respectively, compared to the fluorescence emitted by the cationically quenched Probe No. 7 sample at these times (Fig. 9C) . By contrast, formation of the 1 bp mismatched dsDNA:dsDNA-F complexes (SEQ ID NO: 7 + Probe No. 7) was very inefficient following a 45 minute to 90 minute incubation in the presence of 8 0 mM KCl, 10 mM NaCl, 30 mM MgCl2 and 30 mM MnCl2, as evidenced<br>
py the low 2% to 3% reduction in fluorescence observed with these mismatched complexes compared to that exhibited by the cationically quenched Probe No. 7 (Fig. 9C) . Minimal 1 bp mismatched dsDNA:dsDNA-F complex formation occurred after 12 0 minutes of incubation,, resulting in a 14% decrease in-fluorescence compared to that emitted by the cationically quenched Probe No. 7.<br>
The presence of both monovalent and divalent cations promoted dsDNA.-dsDNA complex formation at a faster rate than did divalent cations alone. Once formed the complexes facilitated by both types of cations persisted for a longer time than did the complexes promoted by either monovalent cations or divalent cations separately. Maximum discrimination between perfectly matched and 1 bp mismatched dsDNA: dsDNA-F complexes was observed for a longer time interval in the presence of both monovalent and divalent cations. Therefore physiological concentrations of KC1, NaCl and MgCl2 preferentially facilitated complex formation between fully matched homologous duplex strands, on the basis of homologous base pair recognition. Example 10<br>
Examples 3-6 demonstrated dsDNA:ssDNA complex formation in a sequence specific manner between dsDNA targets and ssDNA probes facilitated by DNA condensing agents such as monovalent or divalent cations. The next example will examine the rate of formation, stability and rate of disassociation of dsDNA:ssDNA complexes formed in the presence of the multivalent cation, spermidine (possessing a charge of +3) , which is also capable of condensing DNA.<br>
The hybridization reaction mixture (40 µl) contained the following: 0.2 pmoles of non-denatured target dsDNA, 2 pmoles of 5'-fluorescein labeled ssDNA probe, 10 mM Tris-HCl, pH 7.5, and 1 mM spermidine. The reaction mixtures were incubated at room temperature (21°C) for various periods ranging from 1 minute to 2 hours.  Following incubation, samples were placed into a<br>
quartz cuvette, irradiated once with an argon ion laser beam having a wavelength of 488 nm and monitored for fluorescent emission. The intensity of fluorescence was plotted as a function of time for each sample analyzed.<br>
The control ssDNA-F Probe No. 5 exhibited a progressive reduction in fluorescence with increased incubation time in the presence of 1 mM spermidine (Fig. 10). This cationic quench was not as pronounced as that observed when the same probe was incubated in the presence of 10 mM MgCl2 and 10 mM MnCl2, especially within the first 5 minutes of incubation (Fig. 6A) .<br>
Perfectly complementary dsDNA:ssDNA-F complex formation between the dsDNA target (SEQ ID NO: 6) and the ssDNA-F Probe No. ' 5 was indicated after just 15 and 3 0 minutes of incubation in the presence of 1 mM spermidine, generating a 11% and 20% decrease in fluorescence, respectively, compared to that achieved by the cationically quenched Probe No. 5 alone (Fig. 10) . After 45 minutes of incubation in the presence of 1 mM spermidine, minimal complex formation was being signaled (Fig. 10).<br>
The inclusion of 1 mM spermidine was highly unfavourable for dsDNA:ssDNA-F complex formation that contained a 1 bp T-G mismatch (SEQ ID NO: 7 + Probe No. 5), as evidenced by an increased fluorescence of 11% and 7% compared to that observed by the cationically quenched Probe No. 5 after 15 minutes and 30 minutes of incubation, respectively (Fig. 10). Minimal complex formation involving incompletely complementary sequences appeared only after 75 minutes of incubation in the presence of 1 mM spermidine and disappeared by 120 minutes.<br>
The addition of the DNA condensing agent,  spermidine, facilitated rapid dsDNA:ssDNA complex formation between perfectly matched sequences under non-denaturing conditions, to allow differentiation between perfectly complementary sequences and those containing a 1 bp mutation. Example 11<br>
Many different agents besides cations are capable of condensing dsDNA. Example 11 demonstrates the rate of formation,• stability and rate of disassociation of dsDNA:ssDNA complexes formed in the presence of ethanol, another type of DNA condensing agent.<br>
The hybridization reaction mixture (40 µl) contained the following: 0.2 pmoles of non-denatured target dsDNA, 2 pmoles of 5'-fluorescein labeled ssDNA probe, 10 mM Tris-HCl, pH 7.5, and 10% ethanol. The reaction mixtures were incubated at room temperature (21°C) for various periods ranging from 1 minute to 90 minutes. Following incubation, samples were placed into a quartz cuvette, irradiated once with an argon ion laser beam having a wavelength of 488 nm and monitored for fluorescent emission. The intensity of fluorescence was plotted as a function of time for each sample analyzed.<br>
Unlike divalent or multivalent cations, the addition of 10% ethanol did not result in a quench in fluorescence emitted by the control ssDNA-F Probe No. 5, but actually caused a slight increase in fluorescence of the probe over time (Fig. 11).<br>
The presence of 10% ethanol preferentially promoted perfectly complementary dsDNA:ssDNA-F complex formation between the wild-type dsDNA target (SEQ ID NO:6) and ssDNA-F Probe No. 5 following a 15 minute to 60 minute incubation (Fig. 11) . By contrast, formation of the 1 bp T-G mismatched dsDNA:ssDNA-F complex (SEQ ID NO:7 + Probe No. 5) was very inefficient during this incubation time in the presence of 10% ethanol. The fluorescent intensities produced by the perfectly matched complexes and 1 bp mismatched complexes were 11% and 2% lower, respectively after 15 minutes, 12% and 1% lower, respectively after 30 minutes, 25% and 2% lower, respectively after 45 minutes, and 13% and 7% lower, respectively after 60 minutes, than that emitted by the Probe No. 5 control (Fig. 11) . No significant dsDNA:ssDNA-F complex formation was observed after a1" 75 minute incubation in the presence of 10% ethanol.<br>
The addition of low concentrations of ethanol, facilitated DNA complex formation between non-denatured dsDNA targets and fluorescently-labeled ssDNA probes, to allow discrimination between perfectly complementary sequences and those containing a 1 bp mutation. Example 12<br>
Example 12 demonstrates that the assay of the invention can-discriminate  between  perfectly  complementary  dsDNA:ssDNA complexes and dsDNA:ssDNA complexes containing every type of 1 bp mismatch possible, when the cationic DNA intercalator YOYO-1 is present.<br>
Complementary sense and antisense 15-mer ssDNA probes were synthesized on a DNA synthesizer, purified by HPLC, and dissolved in ddH20 at a concentration of 1 pmole/µl. Probe No. 1 was a 15-mer ssDNA probe designed to be completely complementary to a 15 nucleotide segment of the sense strand of the 50-mer wild-type target DNA (SEQ ID NO.-l), overlapping amino acid positions 505 to 510 [Nature 380, 207 (1996)]. The chirality of the probe was opposite or antiparallel to that of the sense strand in the target. Mutant probes (Probe No. 8 to Probe No. 16) were identical in sequence to Probe No. 1 except for a 1 base mutation (underlined) .<br>
Sequence for Probe No. 1 was: 5'-CAC  CAA  AGA  TGA  TAT-3'<br>
Sequence for Probe No. 8 was: 5'-CAC  CAA  AGA  AGA  TAT-31<br>
Sequence for Probe No. 9 was: 5'-CAC  GAA  AGA  TGA  TAT-3'<br>
Sequence for Probe No. 10 was: 5' -CAC  CAA  ACA  TGA TAT-3"<br>
Sequence for Probe No. 11 was: 5'-CAC  CAT  AGA  TGA  TAT-3'<br>
Sequence for Probe No. 12 was: 5'-CAC  CAG  AGA  TGA  TAT-3'<br>
Sequence for Probe No. 13 was: 5'-CAC  CAC.  AGA  TGA  TAT-3 '<br>
Sequence for Probe No. 14 was: 5 ' -CAC  CAA  AGA  C.GA  TAT-3 '<br>
Sequence for Probe No. 15 was: 5' -CAC  CAA  AAA  TGA  TAT-3'<br>
Sequence for Probe No. 16 was: 5' -CAC  AAA AGA TGA  TAT-3'<br>
Probe No. 17 was a 15-mer ssDNA probe designed to be completely complementary to a 15 nucleotide segment of the antisense strand of the 50-mer wild-type target DNA (SEQ ID NO:l), overlapping amino acid positions 505 to 510 [Nature 380, 207 (1996)] . The chirality of the probe was opposite or. antiparallel to that of the antisense strand in the target. Mutant probes (Probe No. 18 to Probe No. 26) were identical in sequence to Probe No. 17 except for a 1 base mutation (underlined).<br>
Sequence for Probe No. 17 was: 5'-ATA  TCA  TCT  TTG  GTG-3'<br>
Sequence for Probe No. 18 was: 5'-ATA  TCT  TCT  TTG  GTG-3'<br>
Sequence for Probe No. 19 was: 5'-ATA  TCA  TCT  TTC  GTG-3'<br>
Sequence for Probe No. 20 was: 5'-ATA  TCA  TGT  TTG  GTG-3'<br>
Sequence for Probe No. 21 was: 5'-ATA  TCA  TCT  ATG  GTG-3'<br>
Sequence for Probe No. 22 was: 5'-ATA  TCA  TCT  CTG  GTG-3'<br>
Sequence for Probe No. 23 was: 5'-ATA  TCA  TCT  GTG  GTG-3'<br>
Sequence for Probe No. 24 was: 5'-ATA  TCG  TCT  TTG  GTG-3'<br>
Sequence for Probe No. 25 was: 5'-ATA  TCA  TTT  TTG  GTG-3'<br>
Sequence for Probe No. 26 wa@: 5'-ATA  TCA  TCT  TTT  GTG-3'<br>
The hybridization reaction mixture (40 µl) contained the. following: 2 pmoles of target dsDNA, 2 pmoles of ssDNA probe, 0.5 x TBE and 500 nM of the DNA intercalator YOYO-I. The reaction mixtures were incubated at room temperature (21°C) for 5 minutes, without prior denaturation of dsDNA targets. Samples were placed into a quartz cuvette, irradiated with an argon ion laser beam having a wavelength of 488 nm and monitored for fluorescent emission. The intensity of fluorescent emission was plotted as a function of wavelength for each sample analyzed.<br>
The 50-mer wild-type non-denatured dsDNA target (SEQ ID NO:l) was reacted with the 15-mer wild-type antisense ssDNA Probe No. 1 and various 15-mer 1 base mutated antisense ssDNA probes (Probe No. 8 to Probe No. 16), tha.t would generate every type of 1 bp mismatch possible.  As expected, the highest fluorescent emission intensities were generated by the dsDNA:ssDNA complexes consisting of perfectly complementary sequences (SEQ ID N0:1 + Probe No. 1) (Figures 12A and 12B).  DsDNA:ssDNA complexes that' resulted in a 1 bp A-A mismatch (SEQ ID NO:l + Probe No. 8), a 1 bp G-G mismatch (SEQ ID NO:l + Probe No. 9), a 1 bp C-C mismatch (SEQ ID N0:1 + Probe No. 10), a 1 bp T-T mismatch (SEQ ID NO:l + Probe No. 11), a 1 bp T-G mismatch (SEQ ID N0:1 + Probe No. 12), a 1 bp T-C mismatch (SEQ ID NO-.l + Probe No. 13), a 1 bp A-C mismatch (SEQ ID N0:1 + Probe No. 14), a 1 bp C-A mismatch (SEQ ID NO:l + Probe No. 15), and a 1 bp G-A mismatch (SEQ ID NO:l + Probe No. 16) produced fluorescent emission intensities that were 63%, 66%,      50%,  47%,  49%,  79%,  57%,  51%  and  71%  lower, respectively,  than that  emitted by the perfectly matched dsDNA:ssDNA complexes (Figures 12A and 12B).<br>
The triple strand assay was also evaluated by reacting the 50-mer wild-type non-denatured dsDNA target (SEQ ID NO:l) with the 15-mer wild-type sense ssDNA Probe No. 17 and various 15-mer 1 base mutated sense ssDNA probes (Probe No. 18 to Probe No. 26) , , that would generate every type of 1 bp mismatch possible.  The<br>
perfectly complementary dsDNA:ssDNA triple strand complexes (SEQ ID N0.-1 + Probe No. 17) containing a sense strand probe (Figures 12C and 12D) formed with similar efficacy as the perfectly complementary triple strand complexes (SEQ ID N0:1 + Probe No. 1) containing an antisense strand probe (Figures 12A and 12B) . DsDNA:ssDNA complexes that resulted in a 1 bp T-T mismatch (SEQ ID N0:1 + Probe No. 18), a 1 bp C-C mismatch (SEQ ID N0:1 + Probe No. 19), a 1 bp G-G mismatch (SEQ ID N0:1 + Probe No. 20), a 1 bp A-A mismatch (SEQ ID NO:l + Probe No. 21), a 1 bp C-A mismatch (SEQ ID NO:l + Probe No. 22), a 1 bp G-A mismatch (SEQ ID NO:l + Probe No. 23), a 1 bp G-T mismatch (SEQ ID NO:1 + Probe No. 24), a 1 bp T-G mismatch (SEQ ID NO:l + Probe No. 25), and a 1 bp T-C mismatch (SEQ ID NO.-l + Probe No. 26) produced fluorescent emission intensities that were 63%, 67%, 76%, 59%, 54%, 57%, 59%, 56% and 82% lower, respectively, than that emitted by the perfectly matched dsDNA:ssDNA complexes (Figures 12C and 12D).<br>
The variability in the fluorescent emission intensities observed between the various 1 bp mismatches depended more on the particular base pair mismatch than the change in percent GC content of the mutant triple strand sequences (Fig. 12) . The results of Figure 12 confirmed the reliability of the triple strand assay to identify all possible 1 bp mismatches with great accuracy when antisense or sense ssDNA probes were reacted with non-denatured dsDNA targets in the presence of the DNA decondensing agent YOYO-1.<br>
While the invention has been described in detail and with reference to specific examples thereof, it will be apparent to one skilled in the art that various changes and modifications can be made therein without departing from the spirit and scope thereof.<br><br><br>
We claim:<br>
1.   A method of creating a nucleic acid multiplex, said method comprising the steps of:<br>
(a)	creating a mixture comprising water, a Watson-Crick duplex, a sufficient number of single-stranded mixed base sequence molecules to form a multiplex that includes the Watson Crick duplex, and an accelerator agent that increases a rate or amount of multiplex formation, said multiplex being a' triplex or quadruplex, wherein said single-stranded molecule or molecules are selected so that, if in a multiplex, they would each be related to all other strands of the multiplex by adherence to base pairing rules, said rules being either Watson-Crick base-pairing rules or homologous binding base-pairing rules; and<br>
(b)	incubating said mixture to allow the multiplex to form, each strand of said multiplex related to all other strands of the multiplex by adherence to base-pairing rules; provided that, within the multiplex, the Watson-Crick duplex added in step (a) is heteropolymeric with a G-C content between 10% and 90%.<br><br>
2.	A method as claimed in claim 1 wherein the single-stranded molecule is related to one strand of the duplex by Watson-Crick base-pairing rules and to the second strand of the duplex by homologous binding base-pairing rules.<br>
3.	A method as claimed in claim 1 or 2 wherein the duplex substantially retains its double-helical stnicture and the single-stranded molecule resides in a groove of that double-helical structure.<br>
4.	A method as claimed in claim 1 where, within the multiplex, the Watson-Crick duplex added in step (a) is heteropolymeric with a G-C content between 10% and 90%, and furthermore the combined frequencies therein of purme-pynmidine  dimers  and  pyrimidmepurine dimers  at least  25%.<br>
5.	A method as claimed in any preceding claim wherein steps (a) and (b) are<br>
performed with the nucleic acid strands and/or duplexes not in a cell.<br>
6.	A method as claimed in any preceding claim wherein step (b) is performed without the assistance of a protein.<br>
7.	A method as claimed in any preceding claim wherein in step (a) , the water is added so that it accounts, on a volume basis, for at least 50 percent of the final volume of the mixture.<br>
8.	A method as claimed in any preceding claim wherein step (b) is performed at a temperature or temperatures above the freezing temperature of the aqueous solution and not more than 85 °C.<br>
9.	A method as claimed in claim 8 wherein step (b) is performed at a temperature or temperatures between 5 °C to 30 °C.<br><br>
10.	A method as claimed in any of claims 1 to 9 wherein in step (a), if desired, a cation is added as the accelerator agent.<br>
11.	A method as claimed in claim 10 wherein said cation is selected from the group consisting of Na+ at a concentration of 50mM to 125mM, Mn-2 at a concentration of l0mM to 45mM, Mg-2 is at a concentration of 10mM to 45mM, and Ni+2 is provided at a concentration of 20mM.<br>
12.	A method as claimed in any of claims 1 to 9 wherein in step (a), if desired, an intercalator is added as an accelerator agent. .<br>
13.	A method as claimed in claim 12 wherein the intercalator is a fluorescent intercalator.<br>
14.	A method as claimed in claim 13 wherein the fluorescent intercalator is selected from the group consisting of YOYO-1, TOTO-I, YOYO-3, TOTO-3, POPO-1,"BOBO-1, POPO-31 BOBO-3, LOLO-11 IOjO-1, cvanine dimers, YO-PRO-1, TO-PRO-1, YO-PRO-3, TO-PRO-3, TO-PRO-5, PO-PRO-1, BO-PRO-1, PO-PRO-3, BO-PRO-3, LO-PRO-1, lOPRO-1, cyanine monomers, ethidium bromide, ethidium, homodimer-1, ethidium, homodimer-2, ethidium derivatives, acridine, acridine orange, acridine derivatives, ethidium-acridine heterodimer, ethidium monoa/ide, propidium. iodide, SYTO dyes, S7BR CI re en 1. SYBR dyes, Pico Green, S7TOX dves, and 7-aminoactinomycin D.<br>
15.	The method as claimed in any of the claims 1   to 9 wherein the accelerator<br>
agent is a non-intercalating fluorophore.<br>
16.	A method as claimed in claim 15 wherein the non-intercalating fluorophore is selected from the group consisting of biotin, rhodamine, Alexa dyes, BODIPY dyes, biotin conjugates, thiolreactive probes, fluorescein and derivatives including hut.<br>
17.	A method as claimed in any of claims 1 to 9 wherein in step (a) the accelerator agent is an intercalator that binds to the minor and/or major groove of the Watson-Crick duplex.<br>
18.	The method as claimed in any of claims 1 to 9 wherein in step (a) optionally an accelerator agent that is a condensation agent as regards the Watson-Crick duplex is added.<br><br>
19.	The method as claimed in any of claims 1 to 9 wherein in step (a) optionally an accelerator agent that is a decondensation agent as regards the Watson-Crick duplex is added.<br>
20.	The method as claimed in claim 1 wherein in step (a) the accelerator agent is a liquid at 25 °C.<br>
21.	A method as claimed in claim 20 wherein in step (a) the accelerator agent is an organic liquid soluble in water<br>
22.	A method as claimed in any preceding claim wherein the multiplex created is a triplex in step (a) the sufficient number of single stranded molecules is 1 and in step (b) the triplex is formed<br>
23.	A method of detecting a triplex, said method comprising the steps of<br>
creating a triplex as claimed in claim 22 and optionally comprising an<br>
additional step (c) in which the triplex is detected.<br>
24.	A method as claimed in any one of claim 1 to 20 wherein the multiplex<br>
created is a quadruplex, in step (a) the Watson-Crick duplex is a first<br>
Watson-Crick duplex, and in step (a) the sufficient number of single-<br>
stranded molecules is 2, those single-stranded molecules are in a second<br>
Watson-Crick duplex, and in step (b) the quadruplex is formed from said<br>
first and second duplexes.<br>
25.	A method of detecting a quadruplex, comprising the steps of creating a quadruplex as claimed in claim 24 and comprising an additional step (c) in which the quadruplex is detected.<br>
26.	A method of detecting a triplex, said method comprising: providing a target double-stranded nucleic acid or nucleic acid analogue comprising:<br><br>
-	a target sequence, wherein said target sequence contains at least one purine base and at least one pyrimidine base;<br>
-	providing a probe comprising a nucleic acid sequence or a nucleic acid analog sequence; providing an accelerator agent; adding said probe, said target sequence and said accelerator agent to a medium to provide a test sample containing a triplex complex comprising said probe bound to said target sequence, wherein all base triplets of said complex arc members selected from the group consisting of A-T-A, T-A-T, U-A-T, T-A-l". A-l'-A, U-A-U, G-C-G and C-G-C; irradiating said test sample with exciting radiation to cause the test sample to emit fluorescent radiation; detecting an intensity of said fluorescent radiation, wherein said intensity is correlated with a binding affinity between said probe and said target sequence; and determining from said intensity an extent of matching between said probe and said target sequence; wherein said method is a homogeneous assay conducted without providing a signal quenching agent on said target sequence or on said probe.<br>
27. A method of detecting a triplex, said method comprising: providing a target nucleic acid or nucleic acid analogue having a target sequence, wherein said target sequence contains at least one purine base and at least one pyrimidine base; providing a double-stranded probe comprising a nucleic acid sequence or a nucleic acid analog sequence; providing a hybridization accelerator agent; adding said probe, said target and said hybridization accelerator agent to a medium to provide a test sample containing a Watson-Crick triplex comprising said probe bound to said target sequence; irradiating said test sample with exciting radiation to cause test sample to emit fluorescent radiation; detecting an intensity of said fluorescent radiation, wherein said intensity is correlated with a binding affinity between said probe and said target sequence; and determining from said intensity an extent of matching between  said probe and  said  target  sequence wherein  said  method  is  a homogeneous assay conducted without providing a signal quenching agent on said target sequence or on said probe.<br>
28. A method of detecting a quadmplex, said method comprising, providing a target nucleic acid or nucleic acid analogue having a target sequence, wherein said target sequence contains at least one purine base and at least one pyrimidine base; providing a double-stranded probe comprising a nucleic acid sequence or a nucleic acid analog sequence; providing a hybridization accelerator agent; adding said probe, said target and said hybridization accelerator agent to a medium to provide a test sample containing a Watson-Crick quadmplex comprising said probe bound to said target sequence; irradiating said test sample with exciting radiation to cause test sample to emit fluorescent radiation; detecting an intensity of said fluorescent radiation, wherein said intensity is correlated with a binding affinity between said probe and said target sequence; and determining from said intensity an extent of matching between said probe and said target sequence wherein said method is a homogeneous assay conducted without providing-a signal quenching agent on said target sequence or on said probe.<br>
29.	A   method  of creating a nucleic  acid multiplex  substantially  as   herein described with reference to the foregoing examples and drawings.<br>
30.	A method of detecting a triplex substantially as herein described with reference to the foregoing examples and drawings.<br>
31.	A method of detecting a quadmplex substantially as herein described with reference to the foregoing examples and drawings.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxMzItZGVsbnAtMjAwNC1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">00132-delnp-2004-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxMzItZGVsbnAtMjAwNC1hc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">00132-delnp-2004-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxMzItZGVsbnAtMjAwNC1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">00132-delnp-2004-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxMzItZGVsbnAtMjAwNC1jb21wbGV0ZSBzcGVjaWZpY2F0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">00132-delnp-2004-complete specification.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxMzItZGVsbnAtMjAwNC1jb3JyZXJzcG9uZGVuY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">00132-delnp-2004-correrspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxMzItREVMTlAtMjAwNC1Db3JyZXNwb25kZW5jZS1PdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">00132-DELNP-2004-Correspondence-Others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxMzItZGVsbnAtMjAwNC1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">00132-delnp-2004-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxMzItZGVsbnAtMjAwNC1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">00132-delnp-2004-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxMzItZGVsbnAtMjAwNC1mb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">00132-delnp-2004-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxMzItZGVsbnAtMjAwNC1mb3JtLTEzLnBkZg==" target="_blank" style="word-wrap:break-word;">00132-delnp-2004-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxMzItZGVsbnAtMjAwNC1mb3JtLTE5LnBkZg==" target="_blank" style="word-wrap:break-word;">00132-delnp-2004-form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxMzItZGVsbnAtMjAwNC1mb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">00132-delnp-2004-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxMzItREVMTlAtMjAwNC1Gb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">00132-DELNP-2004-Form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxMzItZGVsbnAtMjAwNC1mb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">00132-delnp-2004-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxMzItZGVsbnAtMjAwNC1mb3JtLTYucGRm" target="_blank" style="word-wrap:break-word;">00132-delnp-2004-form-6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxMzItZGVsbnAtMjAwNC1ncGEucGRm" target="_blank" style="word-wrap:break-word;">00132-delnp-2004-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxMzItZGVsbnAtMjAwNC1wY3QtMTAxLnBkZg==" target="_blank" style="word-wrap:break-word;">00132-delnp-2004-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxMzItZGVsbnAtMjAwNC1wY3QtMjEwLnBkZg==" target="_blank" style="word-wrap:break-word;">00132-delnp-2004-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxMzItZGVsbnAtMjAwNC1wY3QtMjIwLnBkZg==" target="_blank" style="word-wrap:break-word;">00132-delnp-2004-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxMzItZGVsbnAtMjAwNC1wY3QtNDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">00132-delnp-2004-pct-408.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxMzItZGVsbnAtMjAwNC1wY3QtNDA5LnBkZg==" target="_blank" style="word-wrap:break-word;">00132-delnp-2004-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxMzItZGVsbnAtMjAwNC1wY3QtNDE2LnBkZg==" target="_blank" style="word-wrap:break-word;">00132-delnp-2004-pct-416.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="218061-communication-device-and-method-for-providing-localized-information-to-a-communication-device-in-a-wide-area-communication-system.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="218063-device-for-detecting-force-acting-on-a-tire.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>218062</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>00132/DELNP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>37/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>12-Sep-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>31-Mar-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>19-Jan-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>INGENEUS CORPORATION</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>CHANCERY HOUSE HIGH STREET BRIDGETOWN BARBADOS.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ERIKSON GLEN H.,</td>
											<td>P.O. BOX N-7755, NASSAU, BAHAMAS.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IB02/01972</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-05-31</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0985,731</td>
									<td>2001-06-20</td>
								    <td>Bahamas</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/218062-a-method-for-creating-a-nucleic-acid-multiplex by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:25:57 GMT -->
</html>
