<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/218210-a-method-for-binding-a-ligand by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:35:48 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 218210:&quot;A METHOD FOR BINDING A LIGAND&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A METHOD FOR BINDING A LIGAND&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>An aptamer contains at least two parallel or antiparallel heteropolymeric nucleobase-containing sequences bonded together by Watson-Crick complementary base interaction or by homologous base interaction, provided that: (a) when the aptamer is single-stranded, the at least two sequences are bonded together by homologous base interaction; and (b) when the aptamer is a duplex and the at least two sequences are antiparallel to each other, the at least two sequences are bonded together by homologous base interaction. The aptamer can be used to bind ligands or to catalyze reactions when functioning as an aptazyme.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td><br><br><br><br><br><br><br>
SPECIFICATION     <br>
	  1-   FIELD OF INVENTION<br>
The invention relates to aptamers, and more particularly to aptamers comprising ('ait leas^ two nucleobase-containing sequences, which are parallel or antiparallel to each other, Land bound by Watson-Crick or homologous binding preferences. 2.   DESCRIPTION OF RELATED ART<br>
Protein-nucleic acid complexes are known to play an important role in a variety of biological processes. See, e.g., 'Hill et al. "Fluorescence Approaches to Study of Protein-Nucleic Acid Complexation," 278 Methods in Enzymology 390 (1997). For example, DNA-binding proteins are known to play an important role in gene regulation. Genes are typically regulated at the transcriptional level by DNA-binding proteins, which are referred tcr as transcription factors. Transcription factors regulate gene expression by specifically binding to a target nucleic acid sequence in promoter DNA.<br>
application  Serial  <br>
Due to the biological importance of protein-nucleic acid interaction, a variety of methods for studying protein-nucleic acid binding characteristics have been proposed. See, e.g., Hill et al. and the references cited therein. See also, the inventors' prior ^09/224,5^05.<br>
Aptamers can be designed to interact specifically with non-nucleic acid substances, such as proteins or other bodily substances. Aptamers can function as high affinity receptors for small molecule Ijgands or can tightly interact with target proteins for therapeutic or diagnostic purposes. The folding<br>
of an initially unstructured molecule around its ligand and forming a hydrogen-bond network with its ligand facilitate this binding. Marshall et al. VA biopolymer by any other name would bind as well: a comparison of the ligand-binding pockets of nucleic acids and proteins.", 5(6) Structure 729-734 (1997) . These aptamers ca^^e_J^i^ands used to^ screen_for other molecules or_thjy_canwb^e__c^jtalYtic. Aptamers that are catalytic are considered approximate ribozymes, or aptazymes. To date, agtamers have been almost exclusively of single-s^randed RNA. Aptamers can as well be designed to interact specifically with nucleic acid substances, other than to simply bind them on the basis of Watson-Crick base pairing ibetween bases in nucleic acid sequences of opposite orientation. Such aptamers, if catalytic, may be fairly called aptazymes. Such specific action can be sought for therapeutic or diagnostic purposes.<br>
A small number of RNA molecules are known to be active as catalysts and do not merely serve as the means by which information is moved out of the nucleus. Ribozymes can be self-cleaving or can cleave other RNA.   This activity is understood to be dependent on the RNA's secondary structure, which can be dependent on factors such as base sequence and the inclusion of metallocations.  In the past, there has been a large effort directed at building novel or improved ribozymes.  Ribozymes  have great utility  in artificially controlling gene expression. Developers have sought to take advantage of the very specific charge patterns of nucleic acids, their bases and backbones and DNA's ability to form predictable secondary structure, based upon base sequence and predictable Watson-Crick base pairing.   Nucleic acid's small dimensions and flexible nature make  it well suited for constructing complexes capable of recognizing and specifically binding to features on other substances, such as proteins, and perhaps thereupon adopting tertiary structure.<br>
Through SELEJL-driven screening (U.S. Patent No. 5,567,588<br>
to Gold et al . ) , which depends upon binding to single-stranded<br>
nucleic acids mounted on biochips, researchers have discovered ribozymes, which are 100 or even 1000 fold more active catalytically.<br>
Fernandez et al. "Pulling on Hair(pins)," 292 Science 653. (27 April 2001), reports data collected from a single molecule conformational change in a ribozyme. Fernandez et al . also reports that such essentially duplex nucleic acid structures undergo "all or none" discrete transitions in conformation, not the progressive pair by pair binding one would expect .<br>
Researchers have disclosed a circular RNA that has<br>
enzymatic activity to cleave a separate RNA molecule at a<br>
cleavage site and RNA molecules capable of conferring<br>
stability to RNA in vivo through an endogenous ribozyme<br>
binding protein. See ^_J?atent^JlOj^	Been et al.<br>
and U_- s . Pat ent _No .5 ,.<br>
No -• 5&gt; 840 * -Sfi.7— to Tpole discloses methods for making aptamers and aptamers that bind to biomolecules . These aptamers can be used to interfere with the normal biological function of the biomolecules, as a separation tool, a diagnostic or a therapeutic. The aptamers can be single chain or duplex RNA or DNA. However, this patent only discloses intramolecular or intermolecular Watson-Crick binding of the antiparallel variety.<br>
Researchers have applied single-stranded RNA aptamers directed against Syrian golden hamster prion protein and the aptamers were able to recognize their specific target within a mixture of hundreds of different proteins contained in tissue homogenates thereby tending to validate the utility of aptamers. Korth et al. "Prion (PrPSc) -specif ic epitope defined by a monoclonal antibody." Nature 390:74-77 (1997).<br>
U.S. Patent No. 6,207,388 to Grossman is directed to<br>
methods, compositions, kits and apparatus to identify and detect the presence or absence of target analytes. The compositions comprise an RNA molecule that can be an aptamer that binds to a target molecule. However, Grossman only teaches Watson-Crick antiparallel -binding of nucleobases.<br>
U.S. Patent No. 5,858,774 to Malbon et al. provides a method of regulating a gene by, introducing into a cell an antisense DNA construct. However, this patent does not teach using a nucleic acid to bind to a non-nucleic acid.<br>
Aptamers have been used to identify and evaluate new substances, or drugs, that have a specific binding activity, or that predictably alter the binding characteristics of other binding pairs/complexes. For example, researchers have found a single-stranded DNA aptamer that binds the active site of thrombin, (a protein involved in blood coagulation) , and exhibits anti-coagulation effects in vivo. Davis, "Kinetic characterization of Thrombin-Aptamer interactions." Pharmacia Biosensor Application Note 305, 1994.<br>
Despite the foregoing developments, there is still room in the art for aptamers of novel design with unique binding properties, and for novel uses of such aptamers.<br>
BRIEF SUMMARY OF THE INVENTION<br>
The invention provides an aptamer comprising at least two parallel or antiparallel heteropolymeric nucleobase-containing sequences bonded together by Watsonj^CricJk complementary base interaction or by homologous base interaction, provided that: (a) when said aptamer is single-stranded, said at least two sequences are bonded together by homologous base interaction; and (b) when said aptamer is a duplex and said at least two sequences are antiparallel to each other, said at least two sequences are bonded together by homologous base interaction.<br>
Also provided is a method for binding a ligand. The method comprises contacting the ligand with an aptamer of the invention.<br>
Still further provided is a method for catalyzing a reaction with a catalytic aptamer (aptazyme) of the invention.<br>
BRIEF DESCRIPTION OF SEVERAL VIEWS OF THE DRAWINGS<br>
The invention will be described in conjunction with the<br>
following drawings in which like reference numerals designate<br>
like elements and wherein Figs. 1A, IB, 1C, ID, 2A, 2B, 3A,<br>
3B, 4A, 4B, 5A, SB, 6A, 6B, 7A and 7B are fluorescent spectra.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The invention flows from our elucidation of the specific<br>
binding properties of heteropolymeric nucleobase sequences.<br>
Our prior patent applications disclosed the specific binding<br>
of a heteropolymeric strand to duplex nucleic acid and the<br>
specific binding of duplex nucleic acid to other duplex<br>
nucleic acid.   Our prior patent applications have also<br>
disclosed  that  heteropolymeric  sequences  of  nucleobases<br>
(and/or their analogues) can specifically bind to each other<br>
by homologous base bonding as well as by Watson-Crick base<br>
interaction,  and that such specific base bonding is not<br>
limited  to  sequences  on  strands  having  antiparallel<br>
directionality relative to each other or to duplex formation.<br>
Thus, heteropolymeric base sequences (and/or their analogues)<br>
can  specifically bind  to each other  with parallel  or<br>
antiparallel  directionality,  wherein  the  bases  bond  by<br>
homologous base bonding and/or Watson-Crick base bonding<br>
rules, whether present on the same nucleic acid strand or on<br>
different strands.<br>
The invention is more than merely the disclosure of unorthodox, but specific, binding properties of nucleic acids. The invention encompasses novel aptamers, methods for making^ the  aptamers,  and methods  for using  the  aptamers  for / therapeutic, diagnostic, prophylactic, engineering or other j purpose<br>
Aptamers are nucleic acids typically designed to bind specifically  with non-nucleic  acid  substances,  such  as<br>
proteins or other biomolecules (e.g., carbohydrates, lipids,<br>
etc.) although they may also bind nucleic acids for certain purposes. The term Maptazymes" as used herein means an aptamer catalyst. Aptazymes can specifically bind to nucleic acids (particularly to catalyze cleavage thereof), as well as to proteins and other biomolecules. "Binding" in terms of aptamer and target as used herein refers to an interaction or complexation between a target and an aptamer, resulting in a<br>
sufficiently stable complex so as accomplishatherapeutic purpose or to permit detection^gf the complexes or separation of aptamer:target complexes from uncomplexed structures under given conditions. Aptamers can perform many of the same functions as protein molecules, including undergoing specific changes in their 2 or 3-dimensional structure in response to ligand-binding. Aptamers can be functionalized by including unpaired nucleobases or non-nucleobase molecules.<br>
We disclose aptamers that employ our discovery that mixed nucleobase sequences can bind specifically in either the Watson-Crick complementary binding motif or in the homologous binding motif. Our previous elucidation of the mixed nucleobase sequence triplex and quadruplex demonstrate that the specificity of nucleobase binding is not restricted to a specific plane or surface of the nucleobase, and that a nucleobase is capable of binding specifically to two or more nucleobases at one time, and, most remarkably, that a nucleobase can bind one nucleobase in the Watson-Crick complementary manner while specifically binding complementarily or homologously to another nucleobase. All of these discoveries make possible a great expansion of aptamer design.<br>
Without wishing to be bound to any particular theory, we believe that nucleobase sequences are inherently ambivalent as to binding motif and that binding is a function of the opportunities to bind presented by their environment.<br>
We believe as well that binding motif specificity is a property of a sequence of bases which is enforced upon nucleation having been achieved.• Accordingly, we believe the binding preference of a nucleated strand is the result of base stacking, electrostatic forces or, the like operating within the strand of nucleobases.  Once such a motif preference has been established for a strand, ,it was possible to observe Markovian "all or none kinetics" in connection with the forced disassociation of  the  bound bases  and their  "two-state" reassociation which does not exhibit the expected step-wise sequential "zipping up" of the re-annealing bases.  We propose that the behavior observed by Fernandez, supra, is the product of "motif preference" or "motif memory" by the nucleobase strand.  Fernandez suggests experiments examining refolding by the autonomous units of proteins will be carried out soon. Similar "memory" may be observed as regards refolding protein. We further believe it useful to connect binding motif preference and the phenomenon of mismatch instability.  We are much impressed by the great instability introduced locally by an incompatible pair of nucleobases, even nucleobases whose geometry suggests they ought not be destabilizing to their bound  neighbors.     We  connect  this  instability  with contradiction  of  the  imperative  inherent  in  the  motif preference of the bound sequences.<br>
We find it useful to connect binding motif preference to certain facts relating to protein-DNA interaction, such as translation by Rec BCD along duplex DNA. Dohoney, Nature 2001 Jan 18; 409(6818): 370-374, reports the protein's movement and attendant DNA unwinding at a constant and exceptionally fast velocity. We imagine that duplex DNA stability can be undermined in an allosteric manner, just as we imagine that nucleation is allosteric and creates or enforces a binding preference on adjacent sequences of nucleobases.<br>
Our prior applications have shown that motif preference<br>
can result in homologous binding, which is specific between mixed base sequences on antiparallel strands. As backbone deformation, which was previously condsidered to be a barrier to such pairing, is an unlikely concomitant of such binding, it is likely that unexpected facts will be learned when such duplexes are observed by NMR scanning or other techniques.<br>
Our prior applications have also shown that parallel strands of mixed sequence nucleobases can bind specifically under either motif to form a duplex or that parallel or antiparallel mixed base sequences can bind specifically to previously formed duplex. Most remarkably, we have shown that duplex-bound bases remain reactive specifically as regards other proximal duplex-bound bases, either pursuant to the Watson-Crick complementary binding motif or the Homologous binding motif. Accordingly, our inventions relating to aptamers depend upon our many remarkable discoveries relating to nucleic acid binding, behavior and characteristics, which can greatly extend the field of aptamer design and use.<br>
It is not surprising that complementary base pairing in duplex DNA as suggested by Watson and Crick in 1953 should have carried all before it when proposed and greatly inhibited thinking and experimentation about nucleic acid binding.<br>
In 1940 Linus Pauling and Max Delbruck had expressed the view that molecular complementarity was the basis of biological specificity and the "secret of life". The views of the paper, The Nature of Intermolecular Forces Operative in Biological Processes" prepared the ground for the expectation that such complementary binding would be by different moieties and not by the same moieties. Hence, the readiness of Watson and Crick and others to discard the concept of homologous binding by nucleic acids.<br>
Nucleic acid strands have inherent directionality. The conventional wisdom holds that strands of opposite directionality, i.e., which are antiparallel in their<br>
orientation to one another, can form a duplex through Watson-Crick complementary binding of their respective base sequences.<br>
Certain duplex aptamers according to the invention, on the other hand, comprise two strands of nucleic acid (and/or nucleic acid analogues) hybridized in parallel relation to one another, wherein specific binding is either through homologous base pairing or Watson-Crick base pairing. Conventional wisdom holds that such duplexes do not exist, or at least would be extremely unstable due to, e.g., backbone irregularities necessitated by the conformational requirements of parallel base bonding. Even more surprising is our discovery that under appropriate mild hybridization conditions, parallel homologous duplex bonding demonstrates specificity and stability rivaling or exceeding that of Watson-Crick complementary antiparallel duplex.<br>
The invention also encompasses duplex aptamers containing two strands of nucleic acid (and/or nucleic acid analogues) hybridized in antiparallel relation to one another, wherein specific binding, remarkably, is through homologous base pairing.<br>
As used herein, the terms "Watson-Crick base pairing", "complementary base pairing" and the like are intended to define specific association between opposing or adjacent pairs of nucleic acid and/or nucleic acid analogue strands via matched bases (e.g., A:T; G:C and/or A:U). In the context of the "non-canonical" complexes described herein, including parallel duplexes, parallel and antiparallel triplexes, and parallel quadruplexes, terms like "Watson-Crick base bonding" and "complementary base bonding" are intended to denote bonding between A and T, A and U and/or G and C, but not necessarily in the edgewise, opposed planar conformation first suggested by Watson and Crick.<br>
In addition to the conventional binding motif first proposed by Watson and Crick (the "W-C motif"), and conformational variants thereof encompassed by the foregoing definition of Watson-Crick base bonding, the present invention encompasses aptamers formed by homologous base bonding. In homologous base bonding, bases bond specifically with identical bases rather than complementary bases but not necessarily in a manner similar to the edgewise opposed planar conformation first suggested by Watson and Crick. Thus, in the "Homologous motif", homologous base pairs include A: A, G:G, C:C, T:T and U:U. References to either binding "motif" comprise not only specific binding by nucleobases opposite one another which interact edgewise, as in an antiparallel Watson-Crick bound duplex, but also nucleobases in sequences which are sufficiently proximal to one another whether stably bound in antiparallel duplexes or not. A nucleobase can specifically bind a base previously bound in a duplex in accordance with the Watson-Crick motif and simultaneously, bind a second nucleobase on the basis of the Homologous motif.<br>
The binding by the bases of nucleic acid strands is affected or conditioned by a number of factors, particularly the binding potential of the strands pursuant to either the W-C motif or Homologous motif, and ionic conditions (e.g., salt concentration and/or type). Salty conditions tend to favor the formation of Watson-Crick bonding over homologous bonding. Homologous motif quadruplexes are favored over W-C motif quadruplexes under identical buffer conditions probably because the localized environment can become relatively low-salt, based on the presence of the charged backbones of the two duplex nucleic acids.<br>
An aptamer of the invention can comprise one or more sequences of nucleobases and/or nucleobase analogues, provided the nucleobases are related to the nucleobases to which they are to specifically bind by either the W-C motif or the<br><br><br>
Homologous motif. Contrary to certain teachings of the prior art, the binding nucleobases of the aptamer need not be homopolymeric to achieve binding, in the case of triplex or quadruplex formation. Thus, in certain embodiments, the nucleobases of a first binding .sequence are arranged in a heteropolymeric sequence of interspersed purines and pyrimidines, and the nucleobases,of a second, binding sequence are arranged in a heteropolymeric sequence of interspersed purines and pyrimidines, and are at least partially complementary or partially homologous to the first sequence. For example, the binding sequence of a strand can contain 25% to 75% purine bases and 75% to 25% pyrimidine bases in any order. Aptamers of the invention can form from heteropolymeric sequences, which as defined herein, means sequences containing at least one purine nucleobase or purine analogue and at least one pyrimidine nucleobase or pyrimidine analogue in at least their binding segments. Heteropolymeric sequences preferably lack homopolymeric fragments greater than 5 bases long. Other nucleobases are also suitable for use in the invention, such as, e.g., synthetic analogues of naturally occurring bases which have specific Watson-Crick and/or homologous binding affinities to other bases..<br>
In addition to self-binding nucleic acids and duplexes based on homologous binding, aptamers of the invention also include triplex and quadruplex nucleic acids wherein opposing heteropolymeric strands are linked by Watson-Crick complementary bases or by homologous bases, and the relative directionality of the bound sequences is parallel or antiparallel to one another.<br>
A first sequence of nucleobases can specifically bind in the major or minor groove of a double-stranded nucleic acid complex. Further, the bases can simultaneously interact specifically with bases on one or both strands of a double-stranded nucleic acid complex, with which the first sequence<br>
is bound. Similarly, the bases of each strand of a double-stranded complex can interact specifically with bases on one or both strands of a double-stranded complex in quadruplex aptamers of the invention.<br>
In certain triplex and quadruplex embodiments, each nucleobase binds to one or two other nucleobases. Thus, in addition to the traditional duplex Watson-Crick base pairs and the duplex homologous base pairs described above, such embodiments include the following Watson-Crick base binding triplets: A:T:A, T:A:T, U:A:T, T:A:U, A:U:A, U:A:U, G:C:G and/or C:G:C (including C*:G:C, and/or any other ionized species of base), and/or the following homologous base triplets: A:A:T, T:T:A, U:U:A, T:U:A, A:A:U, U:T:A, G:G:C and/or C:C:G (including C:C+:G, and/or any other ionized species of base).<br>
Thus, in certain quadruplex embodiments wherein the aptamer comprises first, second, third and fourth strands, it is believed that the bases of the first and third strands bind to each other, in addition to: (a) the binding between opposing bases of the first and second strands; (b) the binding between opposing bases of the third and fourth strands; and (c) the binding between opposing bases of the second and fourth strands.<br>
In certain embodiments of the triplex and quadruplex aptamers of the invention, no binding sequence of bases is contiguous with another binding sequence of bases. That is, there are at least three separate strands. Although folded conformations and the like (e.g., hairpin turns, etc.) are within the scope of the invention (particularly when the aptamer is cotnplexed with target molecules, such as proteins) , folded portions of a single strand do not make the strand count more than once in our descriptions of the invention. Aptamers of the invention preferably do not rely on Hoogsteen bonding or G-G quartets for maintenance of the<br>
complex structure, although Hoogsteen bonding and/or G-G quartets may be present. That is, aptamers of the invention are preferably substantially;free of Hoogsteen bonding, and substantially free of G-G quartets.<br>
Each strand of the aptamer independently comprises a nucleic acid having a deoxyribose phosphate or ribose phosphate backbone (e.g., DNA, RNA, mRNA, hnRNA, rRNA, tRNA or cDNA) or a nucleic acid analogue thereof. Preferred nucleic acid analogues contain an uncharged or partially negatively charged backbone (i.e., a backbone having a charge that is not as negative as a native DNA backbone), and include, e.g., PNA and LNA. Certain embodiments are free of PNA. Nucleic acid analogues of the invention can also comprise partially positively charged backbones.<br>
At least a portion of the aptamer may be isolated, purified, artificial or synthetic.<br>
In embodiments, a portion of the aptamer is a PCR amp1i f i ed produc t.<br>
The aptamers of the invention can be present in solution, on a solid support, in vitro, in vivo or in silica. The solid support can be electrically conductive (e.g., an electrode) or non-conductive. In addition, the complexes can be optically mapped or sequenced after being elongated, as taught in U.S. Patents Nos. 6,147,198 and 5,720,928 to Schwartz.<br>
Aptamers of the invention can be provided by a method comprising: (a) providing a hybridization mixture comprising a first single-stranded or double-stranded moiety containing a first heteropolymeric sequence of nucleic acids or nucleic acid analogues, a second single-stranded or double-stranded moiety containing a second heteropolymeric sequence of nucleic acids or nucleic acid analogues, water, and a buffer; and (b) incubating said hybridization mixture for an incubation time effective to hybridize said first heteropolymeric sequence to said second heteropolymeric sequence to provide the aptamer.<br>
The hybridization mixture can include any conventional medium known to be suitable for preserving nucleotides. See, e.g., Sambrook et al., "Molecu-lar Cloning: A Lab Manual," Vol. 2 (1989). For example, the medium can comprise nucleotides, water, buffers and standard salt concentrations. When divalent cations are used exclusively to promote triplex or quadruplex formation, chelatorp such as EDTA or EGTA should not be included in the reaction mixtures.<br>
Specific binding between bases occurs under a wide variety of conditions having variations in temperature, salt concentration, electrostatic strength, and buffer composition. Examples of these conditions and methods for applying them are known in the art. Our copending U.S. Patent Application No. 09/885,731, filed June 20, 2001, discloses conditions particularly suited for use in this invention.<br>
Unlike many Hoogsteen-type complexes, which are unstable or non-existent at pH levels above about 7.6, the complexes of the invention are stable over a wide range of pH levels, preferably from about pH 5 to about pH 9.<br>
Aptamers of the invention can be provided for analytic, diagnostic, prophylactic, therapeutic and/or engineering purposes. The complexes can be used to analyze, diagnose, prevent and/or treat conditions associated with infection by an organism or virus. The organism or virus can be quantitated, if desired.<br>
An aptamer of the invention can be used as a separation tool for retrieving targets to which it specifically binds. In this situation, the aptamer is functioning much like a monoclonal antibody in both its specificity and function. By coupling such an aptamer containing the specifically binding sequence to a solid support, desired target substances can be recovered. This is particularly useful in research or manufacturing in effecting the isolation and purification of substances to which they bind.<br>
In diagnostic applications, the inventive aptamers can be employed in specific binding assays for target substances. They can be labeled using methods and labels known in the art including, but not limited to, detectable moieties such as fluorophores and radioisotopes,.. and then used for in vivo imaging or histological analysis. Because of their high specificity, one application of the aptamers is detecting differences in the type and level of post-translational protein modifications, and even the presence of mutant proteins.<br>
Therapeutically, the aptamer can specifically bind to biologically active sites on the target molecule and affect biological activity. "Biological activity" is used herein to describe any activity that the target possesses in the normal context of its metabolic or other in vivo function in the organism. Without limiting the invention, this can include the catalytic function of an enzyme, or ribozyme, the regulatory function of a hormone, or the recognition function of a cell surface molecule.<br>
The aptamers can be formulated for a variety of modes of administration, including systemic and topical or localized administration. For systemic administration the aptamer can be given via inhalation or injection, including intramuscular, intravenous, intraperitoneal and subcutaneous. Administration can also be transmucosal or transdermal as well as orally.<br>
The aptamers can also be used in expression systems, for example in applying gene therapy.<br>
In certain embodiments, the aptamer can be a drug or can be formed as an anti-cancer agent, autopathogen agent or to effect cellular regulation or transcription as well as gene expression. The aptamer can stimulate an immune response or apoptosis.<br>
This invention enables aptamer binding in a living organism or virus, or in a cell. The complex can be formed in<br>
solution, attached to a surface or substrate, a partition, a bead or an electrode or biochip. The utility of these biochips can be found in, but not limited to, forming molecular fingerprints of tissue samples, analysis of molecular response to viral infection, analysis of inflammation response and analysis of biochemical pathways.<br>
Aptamers of the invention can be formed under conventional hybridization conditions, under triplex hybridization conditions, under quadruplex hybridization conditions or under conditions of in situ hybridization. It is preferred that complexes be formed at a temperature of about 2°C to about 55°C for about two hours or less. In certain embodiments, the incubation time is preferably less than five minutes, even at room temperature. Longer reaction times may not be required, but incubation for up to 24 hours in many cases may not adversely affect the complexes. The fast binding times of the complexes of the invention contrast with the much longer binding times necessary for the formation of Hoogsteen bound complexes. Portions of the aptamers may be crosslinked by the many means of cross-linking known in the art. Aptamers may comprise unpaired nucleobases and non-nucleobase molecules.<br>
The promoter in the hybridization medium is preferably an intercalating agent or a cation, as disclosed in U.S. Patent Application No. 09/613,263, filed July 10, 2000. The intercalators are optionally fluorescent. The intercalating agent can be, e.g., a fluorophore, such as a member selected from the group consisting of YOYO-1, TOTO-1, YOYO-3, TOTO-3, POPO-1, BOBO-1, POPO-3, BOBO-3, LOLO-1, JOJO-1, cyanine dimers, YO-PRO-1, TO-PRO-1, YO-PRO-3, TO-PRO-3, TO-PRO-5, PO-PRO-1, BO-PRO-1, PO-PRO-3, BO-PRO-3, LO-PRO-1, JO-PRO-1, cyanine monomers, ethidium bromide, ethidium homodimer-1, ethidium homodimer-2, ethidium derivatives, acridine, acridine orange, acridine derivatives, ethidium-acridine heterodimer,<br>
ethidium monoazide, propidium iodide, SYTO dyes, SYBR Green 1, SYBR dyes, Pico Green, SYTOX dyes and 7-aminoact inotnyc in D.<br>
Suitable cations include;, -e.g., monovalent cations, such as Na* (preferably at a concentration of 40 mM to 200 mM) , K* (preferably at a concentration of 40 mM to 200 mM) , and other alkali metal ions; divalent cations, such as alkaline earth metal ions (e.g., Mg*2 and Ca*2) and divalent transition metal ions (e.g., Mn*2, Ni*2, Cd*2, Co*2 and Zn*2) ; and cations having a positive charge of at least three, such as Co(NH3)6*3, trivalent spermidine and tetravalent spermine. Mn*2 is preferably provided at a concentration of lOmM to 45mM. Mg*2 is preferably provided at a concentration of lOmM to 45mM. Ni*2 is preferably provided at a concentration of about 20mM. In embodiments, Mg*2 and Mn*2 are provided in combination at a concentration of ImM each, 2mM each, 3mM each . . . 40mM each (i.e., 1-40 mM each).<br>
The amount of cation added to the medium in which the complex forms depends on a number of factors, including the nature of the binding to occur, the nature of the cation, the concentration of binding strands, the presence of additional cations and the base content of the probe and target. The preferred cation concentrations and mixtures can routinely be discovered experimentally. For triplexes, it is preferred to add cation(s) to the medium in the following amounts: (a) 10mM-30mM Mn*2; (b) 10mM-20mM Mg*2; (c) 20mM Ni*2; or (d) ImM-30mM of each of Mn*2 and Mg*2. For quadruplexes, it is preferred to add cat ion (s) to the medium in the following amounts: (a) 10mM-45mM Mn*2; (b) 10mM-45mM Mg*2; or (c) lOmM-40mM of each of Mn*2 and Mg*2.<br>
Although not required, other binding promoters include, e.g., single stranded binding proteins such as Rec A protein, T4 gene 32 protein, E. coli single stranded binding protein, major or minor nucleic acid groove binding proteins, viologen and additional intercalating substances such as actinomycin D,<br>
psoralen, and angelicin. Such facilitating reagents may prove useful in extreme operating conditions, for example, under abnormal pH levels or extremely high temperatures. Certain methods for providing complexes of the invention are conducted in the absence of protein promoters, such as Rec A and/or other recombination proteins.<br>
In addition to providing novel aptamers, the invention also provides a rapid, sensitive, environmentally friendly, and safe method for assaying binding between aptamers and targets.<br>
Embodiments of the invention comprise calibrating the measured signal (e.g., optical, fluorescence, chemiluminescence, electrochemiluminescence, electrical or electromechanical properties) for a first aptamer-target mixture against the same type of signal exhibited by other aptamers combined with the same target, wherein each of the other aptamers differs from the first aptamer by at least one nucleobase.<br>
A calibration curve can be generated, wherein the magnitude of the measured signal (e.g., fluorescent intensity) is a function of the binding affinity between the target and aptamer.<br>
In embodiments, the signal measured can be the fluorescent intensity of a fluorophore included in the test sample. In such embodiments, the binding affinity between the aptamer and target can be directly or inversely correlated with the intensity, depending on whether the fluorophore signals hybridization through signal quenching or signal amplification. Under selected conditions, the fluorescent intensity generated by intercalating agents can be directly correlated with aptamer-target binding affinity, whereas the intensity of preferred embodiments employing a non-intercalating fluorophore covalently bound to the aptamer<br>
or target can be inversely correlated with aptamer-target binding affinity.<br>
The invention enables quantifying the binding affinity between aptamer and target. Such information can be valuable for a variety of uses, including designing drugs with optimized binding characteristics.<br>
The assay of the invention, is preferably homogeneous. The assay can be conducted without separating free aptamer and free target from the aptamer-target complex prior to detecting the magnitude of the measured signal. The assay does not require a gel separation step, thereby allowing a great increase in testing throughput. Quantitative analyses are simple and accurate. Consequently the binding assay saves a lot of time and expense, and can be easily automated. Furthermore, it enables binding variables such as buffer, pH, ionic concentration, temperature, incubation time, relative concentrations of aptamer and target sequences, intercalator concentration, length of target sequences, length of aptamer sequences, and possible cofactor (i.e., promoter) requirements to be rapidly determined.<br>
The assay can be conducted in, e.g., a solution within a well or microchannel, on an impermeable surface or on a biochipl In certain embodiments, the target is provided in the hybridization medium before the aptamer, and the aptamer is provided in dehydrated form prior to rehydration by contact with the hybridization medium.<br>
In certain embodiments, the inventive assay is conducted without providing a signal quenching agent on the target or on the aptamer.<br>
Aptamers of the invention are preferably 2 to 100 bases long (more preferably 5 to 45 bases long), and comprise at least one nucleobase-containing strand. As used herein, the term "nucleobase-containing strand(s)" denotes, e.g., ssDNA, RNA, ssPNA, LNA, dsDNA, dsRNA, DNArRNA hybrids, dsPNA, PNA:DNA<br>
hybrids and other single and double-stranded nucleic acids and nucleic acid analogues having uncharged, partially negatively charged, sugar phosphate and/pr, peptide backbones. It also denotes nucleobase strands having positively charged or partially positively charged backbones.<br>
The assay of the invention does not require the use of radioactive probes, which are hazardous, tedious and<br>
i<br>
time-consuming to use, and need to be constantly regenerated. Aptamers of the invention are preferably safe to use and stable for years.<br>
Targets of the invention are moieties that are substantially free of nucleobases. Preferred targets include, e.g., proteins, peptides (e.g., peptides, dipeptides, tripeptides, etc.), polypeptides, proteins, multi-protein complexes, hormones, lipids, etc.<br>
A variety of aptamer-target complexes can be assayed with the method of the invention. The invention can be used to analyze binding characteristics (including the presence or absence of binding, and the binding affinity) between an aptamer and, e.g., a peptide, a protein, or a multi-protein complex. Suitable proteins for analysis include, e.g., wild-type, mutant, isolated, in vitro translated, and/or synthesized. The invention is particularly suitable for analyzing binding of DNA-binding protein. Test samples need not be 100% pure, but rather, can comprise, e.g., a purified preparation, a synthesized preparation, a semi-purified protein extract, a crude protein extract, or an in vitro translated preparation.<br>
The aptamer-target complex is preferably detected by a change in at least one label. The at least one label can be attached to the aptamer and/or the target, and/or can be free in the test medium. The at least one label can comprise at least two moieties.<br>
The label is preferably at least one member selected from the group consisting of a spin label, a fluorophore, a chromophore, a chemiluminescent agent, an electro-chemiluminescent agent, a radioisotope, an enzyme, a hapten, an antibody and a labeled antibody. Preferably, the complex is detected by at least one emission from the label or by monitoring an electronic characteristic of the complex.<br>
The aptamer-target complex can be detected under at least one varied condition, such as disclosed in U.S. Patent No. 6,265,170 to Picard et al. Suitable varied conditions include, e.g., (a) a change in nonaqueous components of the test medium, (b) a change in a pH of the test medium, (c) a change in a salt concentration of the test medium, (d) a change of an organic solvent content of the test medium, (e) a change in a formamide content of the test medium, (f) a change in a temperature of the test medium, and (g) a change in chaotropic salt concentration in the test medium. In addition, the varied condition can be the application of a stimulus, such as, e.g., electric current (DC and/or AC), photon radiation (e.g., laser light), or electromagnetic force. The stimulus can be applied constantly or pulsed. Detection can be accomplished through the use of a single varied condition, or through a combination of conditions varied serially.<br>
The response of a characteristic of the aptamer-target complex in the test medium to the varied condition or stimulus can be monitored to detect the complex. The characteristic can be, e.g., electrical conductance or Q (a resonant structure of a transmission line or changes in phase or amplitude of a signal propagated in the transmission line in the test medium).<br>
In embodiments, the detection method comprises:   (a) detecting a signal from a label,  wherein the signal is correlated to a binding affinity between the aptamer and the target;  (b)  varying a condition of a test medium;  (c)<br>
detecting a subsequent signal; and (d) comparing the signal and the subsequent signal. The varying and the detecting can be repeated at least once or performed only once.<br>
The label is preferably a fluorophore. Both intercalating and non-intercalating fluorophores are suitable for use in the invention. The fluorophore can be free in solution, covalently bound to the aptamer and/or covalently bound to the target. When the fluorophore is covalently bound to the aptamer, it is preferably bound to an end thereof. Preferred fluorescent markers include biotin, rhodamine, acridine and fluorescein, and other markers that fluoresce when irradiated with exciting energy. Suitable non-intercalating fluorophores include, e.g., alexa dyes, BODIPY dyes, biotin conjugates, thiol reactive probes, fluorescein and its derivatives (including the "caged probes"), Oregon Green, Rhodamine Green and QSY dyes (which quench the fluorescence of visible light excited fluorophores).<br>
The excitation wavelength is selected (by routine experimentation and/or conventional knowledge) to correspond to this excitation maximum for the fluorophore being used, and is preferably 200 to 1000 nm. Fluorophores are preferably selected to have an emission wavelength of 200 to 1000 nm. In preferred embodiments, an argon ion laser is used to irradiate the fluorophore with light having a wavelength in a range of 400 to 540 nm, and fluorescent emission is detected in a range of 500 to 750 nm.<br>
The assay of the invention can be performed over a wide variety of temperatures, such as, e.g., from about 2 to about 60°C. Certain prior art assays require elevated temperatures, adding cost and delay to the assay. On the other hand, the invention can be conducted at room temperature or below (e.g., at a temperature below 25°C).<br>
The invention will be illustrated in more detail with reference  to  the  following  Examples,  but  it  should be<br>
understood that the present invention is not deemed to be limited thereto. Examples<br>
The Examples demonstrate the binding of three different classes of DMA-binding proteins to their respective DNA recognition sites, and the detection of the complex formed. The three representative proteips selected for the Examples are c-JUN (Examples 1, 2 and 4), Spl (Example 3) and Oct-1 (Examples 5-6). Example 1<br>
c-JUN is a member of the AP-1 family of transcription factors  that  bind and  regulate  AP-1  DMA-binding  sites naturally present in promoter or enhancer sequences of many cellular and viral genes.  See, e.g., Bohmann et al., "Human proto-oncogene c-jun encodes a DNA binding protein with structural and functional properties of transcription factor AP-1." 238 Science 1386-1392 (1987).  Furthermore, the human c-JUN protein belongs to a class of proteins (that include c-FOS and c-MYC), designated proto-oncoproteins,  which when deregulated and activated, cause tumorigenesis and cancer,  c-JUN, c-FOS and c-MYC constitute a specific group of DNA-binding proteins,  whose DNA-binding domain consists of a region rich in basic amino acids (commonly called the "basic region" or "basic domain") that lies immediately adjacent to-a structural domain, designated the "leucine zipper".   The leucine zipper consists of 4 to 5 leucine residues (c-JUN has 5), that are separated at regular intervals of 7 amino acids, which form bimolecular coiled-coiled structures.   Specific contact with its palindromic DNA sequence occurs primarily via the basic region.  The leucine zipper allows dimerization of c-JUN to itself, forming c-JUN:c-JUN homodimers, or to c-FOS forming c-JUN:c-FOS heterodimers.   Homodimers of c-JUN bend duplex DNA 79° inward in the minor groove of a DNA helix, while c-JUN: c-FOS heterodimers bend duplex DNA 94° in the<br>
opposite orientation, inward in the major groove. A fully functional DNA-binding domain requires both the basic region and the leucine zipper. As pure human c-JUN protein is used in the following assays, the examples show binding of c-JUN:c-JUN homodimers to a single AP-1 site (JD1F/2F).<br>
A fluorescein labeled wild-type dsDNA oligonucleotide, JD1F/2F, containing a consensus, 7 bp AP-1 DNA binding site, was derived from the promoter sequence of the human collagenase gene. Complementary 5'-fluorescein labeled ssDNA 17-mers JD1F and JD2F, having 5 nucleotides flanking both ends of the consensus AP-1 site, were synthesized on a PerSeptive Biosystems Expedite nucleic acid synthesizer and purified by HPLC. Equimolar amounts of JD1F and JD2F oligos were annealed in 10 mM Tris, pH 7.5, 100 mM NaCl, 1 mM EDTA by denaturation at 95°C for 5 minutes, followed by incubation at 42°C, 35°C and 21°C for 40 minutes each. Annealed oligos were ethanol precipitated for 2 hours at -20°C, pelleted by centrifugation at 14K rpm for 20 minutes at 0°C, washed with 100% ethanol, repelleted at 14K rpm for 20 minutes at 0°C, dried and dissolved in ddH2O at a final concentration of 100 ng/pl. The dsDNA oligos formed had a single fluorescein molecule on both 5' ends.<br>
Sequence for wild-type JD1F (SEQ ID NO:1):<br>
B'-Flu-GTG TCT GAC TCA TGC TT-3' Sequence for wild-type JD2F (SEQ ID NO:2):<br>
S'-Flu-AAG CAT GAG TCA GAC AC-3'<br>
Mutant dsDNA 17-mer JD3F/4F was identical in sequence to wild-type JD1F/2F, except for a single base pair change (underlined) from GC to TA within the wild-type AP-1 consensus DNA-binding site.<br>
Sequence for mutant JD3F (SEQ ID NO:3):<br>
B'-Flu-GTG TCT TAG TCA TGC TT-3' Sequence for mutant JD4F (SEQ ID NO:4): S'-Flu-AAG CAT GAG TAA GAC AC-3'<br>
The c-JON:DNA binding reaction mixture (30pl) contained the following: 9.25 tnM HEPES, pH 7.9, 2.23 tnM MgCl2, 0.03 raM EDTA, 50 mM NaCl, 5.0 mM DTT,/3:75% (v/v) glycerol, 0.15 pg/pl bovine serum albumin (BSA) , 0 - 2.0 ug pure c-JUN protein (Promega, Madison, Wisconsin) or 0 - 400 ng pure c-JUN peptide, and 0.075 pmole 5'-fluorescein labeled dsDNA oligonucleotide. When full-length c-JUN was used, 3 ng/ul poly(dl) - poly(dC) was included in the reaction mix, and added before the addition of protein and fluorescein-labeled DNA. The examples in Figs. IB and ID contained 50 mM KC1 in lieu of 50 mM NaCl. Wild-type and mutant c-JUN DNA-binding domain peptides were generously supplied by Dr. Dirk Bohmann (European Molecular Biology Laboratory, Heidelberg, Germany). The reaction mixtures were incubated at 21°C for 30 minutes, placed into a quartz cuvette, irradiated with an argon ion laser beam having a wavelength of 488 nm and monitored for fluorescent emission.<br>
The wild-type c-JUN DNA-binding domain peptide consisted of the C-terminal 132 amino acid residues of c-JUN (from Gin 209 to Phe 340) . The c-JUN mutant 14 DNA-binding domain peptide was identical in sequence to the wild-type peptide, except for a two amino acid substitution, (underlined) that converted lysine to isoleucine at position 277 and cysteine to aspartic acid at position 278, within the central basic domain.<br>
RLEEKVKTLKAQNSELASTANMLREQVAQLKQKVMNHV<br>
QPQQQQQPPHHLPQQMPVQHPRLQALKEEPQTVPEMPGE<br>
250	260	270	280<br>
TPPLSPIDMESQERIKAERKRMRNRIAASIDRKRKLERIA<br>
290	300	310	320<br>
RLEEKVKTLKAQNSELASTANMLREQVAQLKQKVMNHV<br>
330	340<br>
NSGCQLMLTQQLQTF<br>
The fluorescence spectra obtained for the binding of 2 pg, 1 pg or 0.05 ug full length c-JUN to 0.075 pmole wild-type JD1F/2F or 0.075 pmole mutant JD3F/4P are shown in Figs. 1A-1D.  The DNA concentration was kept constant at 2.5 fmole/pi for every sample tested.  All samples, whether DNA alone, or in the presence of c-JUN, were tested under identical reaction conditions.  The maximum fluorescent intensity occurred at 525 nm, since the fluorophore used was fluorescein.  The maximum intensity observed when 1 ug or 0.05 pg c-JUN was bound to JD1F/2F was 54% and 49% lower,  respectively,  than that observed with JD1F/2F alone (Fig. 1A) .  A 55% decrease in intensity resulted when 2 ug c-JUN was bound to wild-type JD1F/2F (data not shown).  The similar decreases in intensity obtained  with both 1  pg and  2  pg  c-JUN,  suggest  that saturation levels of binding were achieved by addition of 1 pg<br>
protein.<br>
To test c-JUN's preference for binding duplex DNA under different salt conditions, the-above experiment was performed simultaneously in a reaction buffer containing 50 mM KC1 instead of 50 mM NaCl (Fig. IB).. When 2 ug c-JUN was bound to wild-type JD1F/2F in the KC1 reaction buffer, a 57% decrease in intensity was observed, compared to the level achieved with DNA alone. I \ig and 0.5 ug c-JUN bound to wild-type JD1P/2F in the 50 mM KCl buffer, yielded a 40% and 34% decrease, respectively, suggesting below saturation levels of binding. Therefore, c-JUN binds to its AP-1 site with higher binding affinity in a 50 mM NaCl reaction mix than in a 50 mM KCl reaction mix. Thus, the laser binding assay according to the invention could not only reliably detect c-JUN:DNA binding, but could also identify preferential binding conditions.<br>
During the same experiment, when the exact same amounts of c-JUN were reacted with 0.075 pmole mutant JD3F/4F in the 50 mM NaCl reaction mix (Fig. 1C) or the 50 mM KCl reaction mix (Fig. ID), no decrease in fluorescent intensity was observed in every sample, indicating non-binding of protein to the mutated DNA sequence. These mutant DNA binding studies confirm the specificity of both the c-JUN:wild-type DNA binding conditions and the laser detection method.<br>
Identical  results  were  obtained  when  the  emitted fluorescent  intensities  were measured  at  three different integration times (data not shown), demonstrating consistent results irrespective of the integration time. Example 2<br>
Full-length c-JUN protein is 40 KDa or 340 amino acids in size. The DNA-binding domain of c-JUN is localized to the C-terminal 132 amino acid residues of c-JUN (from glutamine at residue 209 to phenylalanine at residue 340) , and is able to bind duplex DNA with similar binding affinity as the full-length protein.<br>
Figs. 2A-2B demonstrate that binding and detection of duplex DNA binding to such a peptide, consisting of only this 132 ami no acid DNA-binding domain. 20 ng, 100 ng and 200 ng of wild-type c-JUN DNA-binding domain peptide bound to 0.075 pmole wild-type JD1F/2F in the. 50 mM NaCl reaction mix, resulted in a 13%, 28% arid 43% decrease in fluorescent intensity, respectively, compared to the intensity emitted by JD1F/2F alone (Fig. 2A). The fact that the binding of just 20 ng of c-JUN peptide to 0.075 pmole DNA could be reliably detected, demonstrates the high sensitivity of the laser assay. Moreover, the peptide:DNA binding assay is quantitative since increasing amounts of c-JUN peptide resulted in progressively more binding to wild-type DNA.<br>
By contrast, 20 ng, 100 ng and 200 ng wild-type c-JUN peptide did not bind mutant JD3F/4F, resulting in minor increases in fluorescent intensity above that observed with mutant DNA alone (Fig. 2B), confirming the specificity of the laser binding assay.<br>
The 43% decrease in fluorescent intensity observed for 200 ng of c-JUN peptide bound to JD1F/2F, is less than the 54% and 49% decreases observed for 1 ug and 0.5 ug full length c-JUN protein, respectively, as predicted. One would expect to get less static quenching occurring with a peptide, than with a full length protein, since less mass of protein would absorb the emitted fluorescent light in a peptide. Example 3<br>
Spl belongs to a significant class of duplex DNA-binding proteins designated zinc finger DNA-binding proteins. See, e.g., Kadonaga et al., "Isolation of cDNA encoding "J transcription factor Spl and functional analysis of the DNA binding domain." 51 Cell 1079-1090 (1987). Spl controls the transcription of a large number of viral and cellular promoters or enhancers, including the HIV-I long terminal repeat (LTR). The number, spacing, orientation and sequence<br>
of Spl binding sites vary widely between promoters, resulting in high, medium or low affinity binding sites. Although Spl is a relatively large protein- (95 KDa and 105 KDa in its glycosylated and phosphorylated form), its DNA-binding activity is localized near the C-.terminus of the protein (from cysteine at residue 539 to histidine at residue 619) . This region contains three contiguous Zn(II) finger motifs, which are metalloprotein structures that interact with DNA. Sequence specificity of DNA binding is conferred entirely by the three Zn(II) fingers. Finger 3 is the most critical finger (with respect to binding affinity), followed by finger 2 and lastly finger 1. Two cysteine and two histidine residues bind a Zn(II) ion to form each finger. Removal of zinc collapses the secondary structure of the three zinc fingers. The fingers in this class of duplex DNA-binding proteins have a consensus sequence of Cys-X2,4-Cys-X3-Phe-X5-Leu-X2-His-X3-His, referred to as Cysa/His2 fingers. A second type of Zn(II) finger motif, referred to as Cys2/Cys2 fingers with the form of Cys-X2-Cys-Xi3-Cys-X2-Cys, are found in other DNA-binding proteins, such as many hormone receptors.<br>
A wild-type fluorescein labeled dsDNA oligonucleotide, JD11F/12F, containing a single consensus 10-bp Spl DNA binding site, was derived from the promoter sequence of the human metallothionein-IlA gene. Complementary 5'-fluorescein labeled ssDNA 20-mers JD11F and JD12F were synthesized, purified and annealed as above.<br>
Sequence for wild-type JD11F (SEQ ID N0:7): 5'-Flu-CCG GCC GGG GCG GGG CTT TT-3'<br>
Sequence for wild-type JD12F (SEQ ID NO:8): 5'-Flu-AAA AGC CCC GCC CCG GCC GG-3'<br>
Mutant dsDNA 20-mer JD13F/14F was identical in sequence to wild-type JD11F/12F, except for a 6 bp change (underlined) which converted the consensus Spl binding site GGG GCG GGG C to TAA ATA GGG C.Sequence for mutant JD13F (SEQ ID N0:9):<br>
5'-Flu-CCG GCC TAA ATA GGG CTT TT-31<br>
Sequence for mutant JD14F CSEQ ID NO:10):<br>
S'-Flu-AAA AGC CCT ATT TAG GCC GG-31<br>
The Spl:DNA binding reaction.mixture (30pl) contained the following: 25 mM HEPES, pH 7.8, 100 mM KC1, 100 pM ZnSO4, 1 mM DTT, 20% (v/v) glycerol, 0.05 ug/ul BSA, 0 - 200 ng pure Spl protein (Promega) and 0.1 pmole 5'-fluorescein labeled dsDNA oligonucleotide. The reaction mixes were incubated at 0°C for 15 minutes, placed into a quartz cuvette, irradiated with an argon ion laser beam having a wavelength of 488 nm and monitored for fluorescent emission.<br>
Fig. 3 illustrates the binding of the zinc finger DNA-binding  protein  Spl  to  wild-type  JD11F/12F  or  mutant JD13F/14F.  When 200 ng Spl was bound to 0.1 pmole JD11F/12F, a 44% decrease in fluorescent intensity was observed, compared to the intensity level achieved with JD11F/12F alone (Fig. 3A) .  Furthermore, the binding of 25 ng of full length Spl protein  could  be  reliably  detected  (data  not  shown), demonstrating the high sensitivity of the laser assay.  Since Spl is a relatively large protein (95 KDa) , while c-JUN is only 40 KDa in size, a lesser amount of protein was required to achieve a 44% reduction in fluorescent intensity for Spl-bound DNA than c-JUN-bound DNA, due to greater absorption and retention of emitted fluorescent light by the larger protein.<br>
When 200 ng Spl was reacted with 0.1 pmole mutant JD13F/14F, no decrease in fluorescent intensity was observed (Fig. 3B) , indicating non-binding of protein to the mutated DNA sequence.  These studies confirmed the specificity of the laser detection assay for a completely different class of DNA-binding proteins. Example 4 This example illustrates the ability of the method of the<br>
invention to study the binding of an antibody directed to a specific protein, which is directly bound to the labeled DNA sequence. Addition of specific antibodies to protein:DNA complexes (especially multi-protein:DNA complexes) is a technique used to identify the presence of unknown proteins in protein:DNA complexes. The binding of the antibody will either inhibit or totally prevent; the protein:DNA complex from forming (resulting in a minimal decrease or no change in fluorescent intensity when compared to free DNA) or will result in an antibody:protein:DNA complex that decreases the intensity of fluorescence even more than the protein: DNA complex.<br>
1 ug, 500 ng and 250 ng c-JUN were reacted with 0.075 pmole of wild-type JD1F/2F in the 50 mM NaCl or 50 mM KC1 reaction mix as previously described. After a 15 minute incubation at 21°C, variable amounts of the monoclonal IgGi antibody, c-JUN (KM-1) (from Santa Cruz Biotechnology, Santa Cruz, California), raised against a peptide corresponding to amino acids 56 to 69 of human c-JUN, was added to some of the c-JUN:DNA mixtures. The reaction mixtures were incubated for an additional 40 minutes at 21°C, placed into a quartz cuvette, irradiated with an argon ion laser beam having a wavelength of 488 nm and monitored for fluorescent emission.<br>
Figs. 4A and 4B show the binding of 1 pg or 250 ng c-JUN to JD1F/2F, respectively, in the 50 mM NaCl reaction mix. When 1 ug or 250 ng c-JUN was bound to JD1F/2F, a 25% and 11% decrease in intensity, respectively, was observed, compared to the level achieved with DNA alone. Addition of 5 ug or 1 ug of c-JUN antibody to 1 pg c-JUN resulted in a 42% and 37% decrease, respectively (i.e., a further decrease of 17% and 12%), indicative of IgG:c-JUN:DNA complex formation (Fig. 4A) . Identical decreases in intensity were observed when c-JUN antibody was bound to 1 ug c-JUN bound to JD1F/2F in the 50 mM KC1 reaction mix (data not shown). Similarly, addition of 750<br>
ng of c-JUN antibody to 250 ng c-JUN bound to JD1F/2F, yielded a 27% decrease in intensity, a further decrease of 16% from the level achieved from the protein:DNA complex alone (Fig. 4B) . IgG:c-JUN complexes did not bind to mutant DNA JD3F/4F (data not shown) , confirming the specificity of the laser assay.<br>
This example demonstrates that the laser detection method can differentiate between an antibody:protein:DNA complex and a protein:DNA complex. Moreover, it establishes the ability of the invention to reliably detect heterologous multi-protein complexes bound to DNA and not just monomers or homodimers of protein bound to DNA. Only one of the proteins in the multi-protein:DNA complex needs to be bound to DNA. Multi-protein: DNA complexes, where more than one protein interacts with DNA can also be assayed by the method of the invention.<br>
Example 5<br>
The ubiquitous cellular octamer-binding protein (Oct-1) binds DNA directly by a characteristic DNA-binding domain, which is completely different than the DNA-binding domains of c-JUN or Spl. Oct-1 is a member of the POU domain DNA-binding proteins, which regulate cell-specific transcription and development. See, e.g., Sturm et al., "The ubiquitous octamer-binding protein Oct-1 contains a POU domain with a homeo box subdomain." 2 Genes and Development 1582-1599 (1988). The structure of the POU domain is unique among DNA-binding domains, because it contains two structurally independent domains that cooperate functionally as a single DNA-binding unit. Oct-1 binds to DNA via this POU domain, composed of a 75 amino acid POU-specific (POUS) domain, a short linker region of 24 amino acids, and a 60 amino acid POU-type homeo (POUH) domain. Both the POUS domain and the POUH domain contain helix-turn-helix (HTH) structures.<br>
Unlike Examples 1-4, which used purified protein, this<br>
example uses HeLa cell nuclear extracts (from Promega, Madison, Wisconsin) as the source for Oct-1. The use of HeLa cell nuclear extracts, which- contain a vast multitude of various DNA-binding proteins and transcription factors, shows the feasibility of using crude protein extracts to detect sequence-specific protein:DNA binding by the laser assay of the invention.<br>
A wild-type fluorescein labeled dsDNA oligonucleotide, JD49F/50F, containing a single consensus 8 bp Oct-1 DNA binding site, was derived from the human immunoglobulin heavy chain promoter. Complementary 5'-fluorescein labeled ssDNA 18-mers JD49F and JD50F were synthesized, purified and annealed as above.<br>
Sequence for wild-type JD49F (SEQ ID NO:11):<br>
S'-Flu-GAG TAT GCA AAT CAT GTG-3' Sequence for wild-type JD50F (SEQ ID NO:12):<br>
5'-Flu-CAC ATG ATT TGC ATA CTC-3'<br>
Mutant dsDNA 18-mer JD51F/52F was identical in sequence to wild-type JD49F/50F, except for a double point mutation (AjT2 -&gt; CG) (underlined) that inactivated the POUS binding site, and a second double point mutation (AeA7 -. CC) (underlined) that inactivated the POUH binding site, thereby converting the consensus Oct-1 binding site ATGCAAAT to CGGCACCT.<br>
Sequence for mutant JD51F (SEQ ID NO:13):<br>
S'-Flu-GAG TCG GCA CCT CAT GTG-31 Sequence for mutant JD52F (SEQ ID NO:14): B'-Flu-CAC ATG AGG TGC CGA CTC-31<br>
The Oct-1:DNA binding reaction mixture (30 ul) contained the following: 9.25 mM HEPES, pH 7.9, 2.23 mM MgCl2, 0.03 mM EDTA, 63 mM NaCl, 1.0 mM DTT, 3.75% (v/v) glycerol, 0.10 mg/ml BSA, 0.01 mM PMSF, 67 ug/ml poly(dl)-poly(dC), 67 pg/ml poly (dG-dC)-poly (dG-dC) , 0 - 15 pg HeLa cell nuclear extract (Promega) and 0.05 pmole 5'-fluorescein labeled dsDNA<br>
oligonucleotide. The relatively high concentrations of poly(dl)-poly(dC) and poly(dG-dC)-poly(dG-dC) are required to ensure sequence specific protein:DNA binding, when using crude nuclear protein extracts. The reaction mixtures were incubated at 21°C for 30 minutes, placed into a quartz cuvette, irradiated with an argon ion laser beam having a wavelength of 488 nm and monitored for fluorescent emission.<br>
The fluorescent spectra obtained when 10 pg of HeLa cell nuclear extract was reacted with 0.05 pmole wild-type JD49F/50F or 0.05 pmole mutant JD51F/52F are shown in Figs. 5A and 5B, respectively. The Oct-1 protein present in the HeLa cell nuclear extract, bound specifically to the wild-type high affinity Oct-1 binding site, resulting in a 22% decrease in fluorescent intensity compared to the level observed with JD49F/50F alone (Fig. 5A). By contrast, Oct-1 did not bind to mutant JD51F/52F, as indicated by the increase in fluorescent intensity above that observed with mutant DNA alone (Fig. 5B) , confirming the sequence specificity of the assay. These experiments demonstrated the specific detection of another completely different class of DNA-binding proteins.<br>
Moreover, this example confirmed that specific protein:DNA binding may be reliably measured by the invention even when using crude HeLa cell nuclear extracts, that contain hundreds of other DNA-binding proteins. Specificity is conferred by the selection of the appropriately labeled DNA sequence, that recognizes the particular DNA-binding protein to be studied. Example 6<br>
This Example clearly demonstrates that the method of the invention can measure binding of a multi-protein complex (consisting of two or more different proteins) to one (or more) binding sites on a DNA sequence. Studies were conducted on the binding of the human cellular proteins octamer-binding protein (Oct-1) and host cellular factor (HCF -- see, e.g.,<br>
Wilson et al., "The VP16 accessory protein HCF is a family of polypeptides processed from a large precursor protein." 74 Cell 115-125 (1993)) with the -herpes simplex virus type 1 (HSV-1) protein VP16 (or Vmw65) to the DNA sequence TAATGARAT (where R is a purine). This multi-protein:DNA complex is called the immediate early complex (IEC) or VP16-induced complex. Although VP16 is the mpst potent trans-activator of genes ever identified, it cannot bind DNA efficiently on its own. Instead, it interacts specifically with Oct-1 and HCF to induce genes. VP16 binds to Oct-1 and HCF via its amino terminal 411 amino acids. The C-terminal highly acidic domain of VP16, defined by amino acids 411 to 490, functions as the potent transcriptional activating region. See, e.g., Dalrymple et al., "DNA sequence of the herpes simplex virus type 1 gene whose product is responsible for transcriptional activation of immediate early promoters." 13 Nucleic Acids Research 7865-7879 (1985).<br>
Oct-1 binds to DNA via its bipartite POU domain, which is capable of displaying exceptional DNA sequence recognition flexibility. The Oct-1 POU domain binds to the octamer sequence ATGCAAAT as a monomer, with the POUS domain contacting the 5' half of this site (ATGC) and the POUH domain interacting with the 3' half of this site (AAAT) on opposite sides of the DNA. When Oct-1 is bound to the high affinity ATGCAAAT binding site, it is incapable of interacting with VP16.<br>
Oct-1 also binds to DNA sites that bear little resemblance to the octamer consensus. For example, Oct-1 by itself or in association with HCF and VP16 can bind the DNA sequence TAATGARAT, which bears as little as a 4 of 8 bp match to the octamer consensus site. Two forms of the TAATGARAT site are found in the promoter sequences of the herpes simplex virus immediate early (HSV IE) genes. The first, designated the (OCT A*) TAATGARAT motif, contains an overlapping<br>
octamer/TAATGARAT sequence, which binds Oct-l with high affinity. The second, called (OCTA') TAATGARAT, lacks an overlapping octamer sequence and binds Oct-l with relatively low affinity. The POUH domain of Oct-l binds the 5' TAAT sequence, while the POUS domain binds the GARAT sequence on the (OCTA")TAATGARAT site. On the (OCTA*) TAATGARAT binding site, the POUH domain remains fixed to the TAAT sequence, while the POUS domain can bind either the 5' ATGC sequence or the 3" GARAT element. The Oct-l POUH domain is sufficient for interacting with VP16.<br>
The HCF is required to stabilize the association of Oct-l with VP16 on a TAATGARAT site, by first forming a stable complex with VP16 independent of Oct-l or the TAATGARAT element. The exact mechanism by which HCF stabilizes VP16 association with Oct-l is unknown. The HCF may induce a conformational change within VP16, which primes VP16 to interact with Oct-l and the GARAT element of the TAATGARAT site. Alternatively, within the IEC complex, the HCF may contact Oct-l or the DNA, and thus confer greater stability to the complex.<br>
A wild-type fluorescein labeled dsDNA oligonucleotide, JD41F/42F, containing an (OCTA") TAATGARAT site was derived from a 20 bp region (-343 to -324) from the HSV-1 IE gene 4/5 promoter. Complementary 5 '-fluorescein labeled ssDNA 20-mers JD41F and JD42F were synthesized, purified and annealed as above.<br>
Sequence for wild-type JD41F (SEQ ID NO:15): S'-Flu-GGC GGT AAT GAG ATA CGA GC-31<br>
Sequence for wild-type JD42F (SEQ ID NO:16): 5'-Flu-GCT CGT ATC TCA TTA CCG CC-3'<br>
Mutant dsDNA 20-tner JD43F/44F was identical in sequence to wild-type JD41F/42F, except for a double point mutation (A2A3 - CC) (underlined) that inactivated the POUH binding site, and a second double point mutation (A8T9 -» CG)<br>
 (underlined) that inactivated the POUS binding site, thereby converting the Oct-1 binding site TAATGAGAT to TCCTGAGCG.<br>
Sequence for mutant JD43F (SEQ ID NO:17): S'-Flu-GGC GGT CCT GAG CGA CGA GC-3'<br>
Sequence for mutant JD44F (SEQ ID NO:18): B'-Flu-GCT CGT CGC TCA GGA CCG CC-3'<br>
A wild-type fluorescein labeled dsDNA oligonucleotide, JD45F/46F, containing a (OCTA*) TAATGARAT site was derived from a 23 bp region (-170 to -148) from the HSV-1 IE gene 1 promoter. Complementary 5'-fluorescein labeled ssDNA 23-mers JD45F and JD46F were synthesized, purified and annealed as above.<br>
Sequence for wild-type JD45F (SEQ ID NO:19): S'-Flu-GTG CAT GCT AAT GAT ATT CTT TG-3'<br>
Sequence for wild-type JD46F (SEQ ID NO:20): S'-Flu-CAA AGA ATA TCA TTA GCA TGC AC-3'<br>
Mutant dsDNA 23-mer JD47F/48F was identical in sequence to wild-type JD45F/46F, except for a double point mutation (A6A7 -. CC) (underlined) that inactivated the POUH binding site, and two additional double point mutations (AiT2 -. CG) and (Ai2T13 -. CG) (underlined) that inactivated the two POUS binding sites, thereby converting the Oct-1 binding site ATGCTAATGATAT to CGGCTCCTGATCG.<br>
Sequence for mutant JD47F (SEQ ID N0:21):<br>
S'-Flu-GTG CCG GCT CCT GAT CGT CTT TG-31<br>
Sequence for mutant JD48F (SEQ ID NO:22):<br>
S'-Flu-CAA AGA CGA TCA GGA GCC GGC AC-31<br>
The Oct-l:HCF:VP16:DNA binding reaction mixture  (30ul) contained the following:   9.25 mM HEPES, pH 7.9, 2.23 mM MgCl2, 0.03 mM EDTA, 63 mM NaCl, 1.0 mM DTT,  3.75%  (v/v) glycerol, 0.10 mg/ml BSA, 0.01 mM PMSF, 133 pg/ml poly(dl)-poly(dC), 67 ug/ml poly(dG-dC)-poly(dG-dC),  0 - 25 ug HeLa cell nuclear extract (Promega),  0 - 0.1 pg HSV-1 virion extract  and  0.025  pmole  5'-fluorescein  labeled  dsDNA<br><br>
oligonucleotide.  The HSV-1 virion extract containing 80% pure VP16 was kindly provided by Dr. Chris Preston (MRC Institute of Virology, Glasgow, Scotland).- HeLa cell nuclear extracts served as the source for Oct-1 and HCF. All components except the DNA and the virion extract were incubated at 21°C for 10 minutes.  DNA was then added, followed by the addition of HSV-1 virion extract (where appropriate).  Reaction mixtures were incubated for an additional 30 minutes at 21°C, placed into a quartz cuvette, irradiated with an argon ion laser beam having a wavelength of 488 nm and monitored for fluorescent emission. The Oct-1 protein, present in 10 pg and 20 ]ig of HeLa cell nuclear extract, bound specifically to 0.025 pmole wild-type JD41F/42F,  resulting in a 10% and a 43% decrease, respectively, in fluorescent intensity compared to the level achieved with JD41F/42F alone (Fig. 6A) .  The low DNA amount of 0.025 pmole was in molar excess to the amount of Oct-1 present in the HeLa cell nuclear extract.  The observation that 10 ug of HeLa cell nuclear extract produced a 22% decrease in fluorescent intensity when Oct-1 was bound to 0.05 pmole of its high affinity JD49F/50F binding site (in Example 5) , whereas the same amount of HeLa cell nuclear extract resulted in only a 10% decrease in fluorescent intensity when Oct-1 was bound to 0.025 pmole of its low affinity JD41F/4'2F binding site (which is in molar excess to the amount of Oct-1 present), verified the ability of the laser binding assay to discriminate between high affinity and low affinity DNA binding sites for the same protein.<br>
When 0.1 ug of VP16 was added to the Oct-1: JD41F/42F reaction mix, a 20% decrease in fluorescent intensity was observed, representing a further decrease of 10% from the level achieved from the Oct-1:JD41F/42F complex alone (Fig. 6A). This additional decrease arose from the multi-protein Oct-1:HCF:VP16:JD41F/42F complex formation, which was able to absorb and retain more emitted fluorescent light than the<br>
single protein Oct-1:JD41F/42F complex.<br>
No decrease in fluorescent intensity was observed when 10 ]ig or 20 pg of HeLa cell nuclear extract, in the absence or presence of VP16, was reacted with 0.025 pinole mutant JD43F/44F, indicating non-binding, of Oct-1 or Oct-1:HCF:VP16 complex to the mutated DNA sequence (Fig. 6B). These mutant DMA binding studies confirmed the specificity of the laser detection method for measuring specific multi-protein:DNA complex formation using crude nuclear extracts.<br>
When 10 ug of HeLa cell nuclear extract was reacted with 0.025 pmole of wild-type JD45F/46F, a 32% decrease in fluorescent intensity occurred, compared to the fluorescent intensity observed with JD45F/46F alone (Fig. 7A) . This relatively large decrease in intensity is a function of Oct-1's ability to bind with high affinity to the (OCTA*) TAATGARAT site.<br>
Addition of 0.1 pg of VP16 to 10 ug HeLa cell nuclear extract and 0.025 pmole wild-type JD45F/46F, resulted in a 69% decrease in fluorescent intensity, representing a further decrease of 37% from the intensity level obtained from the Oct-l:JD45F/46F complex alone (Fig. 7A). Since Oct-1, HCF and VP16 are 110 KDa, -300 KDa and 65 KDa in size, respectively, the huge 69% decrease is a direct result of highly efficient multi-protein Oct-l:HCF:VP16 binding to the (OCTA*)TAATGARAT site present in JD45F/46F.<br>
By contrast, no decrease in fluorescent intensity was observed when 10 ug of HeLa cell nuclear extract, in the absence or presence of 0.1 ug VP16, was reacted with 0.025 pmole mutant JD47F/48F (Fig. 7B) , clearly indicating disruption of DNA binding to the mutated DNA sequence, and further proving the specificity of the laser binding assay.<br>
This example clearly demonstrates that the method of the invention can reproducibly measure specific binding of a multi-protein complex (consisting of two or more different<br>
proteins) to one (or more) binding sites on a DNA sequence, when using crude nuclear cell extracts. Furthermore, the laser binding assay can evaluate the affinity of a specific protein or multi-protein complex to any given DNA sequence.<br>
As demonstrated by the Examples, the invention is applicable to all classes of DNA-binding proteins. For example, when the oncoprotein c-JUN binds to its specific DNA recognition site, a 55% decrease in measurable units is observed, compared to the level achieved by unbound DNA (Figs. 1A and IB) . No decrease is observed when c-JUN is reacted with a mutant DNA sequence (Figs. 1C and ID), indicating non-binding and confirming the specificity of the detection method.<br>
Furthermore, specific binding of peptides containing just the DNA-binding domain of the protein can be detected in a quantitative manner. For example, 20 ng, 100 ng and 200 ng of c-JUN peptide bound to wild-type DNA results in 13%, 28% and 43% decreases, respectively, compared to the level observed for free DNA (Fig. 2A). The fact that the binding of just 20 ng of c-JUN peptide can be reliably measured, demonstrates the high sensitivity of the detection assay. By contrast, 20 ng, 100 ng and 200 ng of c-JUN peptide do not bind mutant DNA, resulting in minor increases above the level observed with mutant DNA alone (Fig. 2B) . Binding of peptides in lieu of full length proteins may be of -particular interest to designing and/or screening Pharmaceuticals.<br>
Figs. 3A and 3B illustrate the binding of the zinc finger DNA-binding protein Spl to wild-type or mutant DNA binding sites, respectively. When 200 ng of Spl is bound to wild-type DNA, a 44% decrease is observed, compared to the level measured for DNA alone. Non-binding of 200 ng Spl is observed for the mutated DNA sequence.<br>
The assay of the invention can differentiate between an antibody:protein:DNA complex and a protein-.DNA complex. For<br>
example, a 42% and 37% decrease in fluorescent intensity was observed when 5 ug or 1 \ig of c-JUN antibody, respectively, was bound to 1 ug c-JUN complexed to wild-type DNA, compared to the 25% decrease obtained for c-JUN:DNA complexes (Fig. 4A) . IgG:c-JUN complexes did - not bind to mutant DNA sequences.<br>
Figs. 5, 6 and 7 illustrate ,the binding of the bipartite POU domain DNA-binding protein Oct-1 to three different DNA sequence recognition sites, with different binding affinities. Moreover, Examples 5 and 6 prove the feasibility of using crude nuclear protein extracts as a source of DNA-binding proteins, while still retaining highly specific protein-DNA binding. Depending on the binding affinity of each DNA site, 10 ug of HeLa cell nuclear extracts bound wild-type Oct-1 DNA binding sites with a 10%, 22% or 32% decrease, compared to levels achieved for unbound DNA.<br>
Significantly, the method of the invention can reliably measure the binding of multi-protein complexes (consisting of two or more different proteins) to one (or more) DNA binding sites, whether pure proteins or crude nuclear extracts are used. For example, Oct-1:HCF:VP16:DNA complexes yielded a 69% and 20% decrease in fluorescent intensity when bound to a high affinity (OCTA*) TAATGARAT site or a low affinity (OCTA' )TAATGARAT site, respectively (Figs. 7 and 6). Non-binding of Oct-1 protein or Oct-1:HCF:VP16 protein complex is observed for all of the mutated DNA sequences.<br>
Multi-protein:DNA complexes are much more prevalent in nature and biologically significant than single protein:DNA complexes. The ability of the method of the invention to employ crude nuclear protein extracts to assay single or multi-protein binding to DNA in a highly specific manner is of major clinical relevance.<br>
While the invention has been described in detail and with reference to specific examples thereof, it will be apparent to<br>
one skilled in the art that various changes and modifications can be made therein without departing from the spirit and scope thereof.<br><br><br><br><br>
We claim:<br>
1.	A method for binding a ligand said method comprising:<br>
providing an aptamer comprising at least two parallel or antiparallel heteropolymeric nucleobase-containing sequences bonded together by Watson-Crick complementary base interaction or by homologous base interaction, wherein<br>
a)	said aptamer comprises a duplex, and said at least two<br>
sequences    are   bonded   together   by    Watson-Crick<br>
complementary base interaction with parallel directionality; or<br>
b)	said aptamer comprises a duplex, and said at least two<br>
sequences  are  bonded together  by  homologous  base<br>
interaction with parallel or antiparallel directionality; or<br>
c)	said aptamer comprises a triplex , and said at least two<br>
sequences    are    bonded   together    by    Watson-Crick<br>
complementary base interaction with parallel or antiparallel<br>
directionality; or<br>
d)	said aptamer comprises a triplex, and said at least two<br>
sequences  are  bonded together  by  homologous  base<br>
interaction with parallel or antiparallel directionality; or<br>
e)	said aptamer comprises a quadruplex, and said at least two<br>
sequences    are    bonded   together    by    Watson-Crick<br>
complementary base interaction with parallel or antiparallel<br>
directionality; or<br>
f)	said aptamer comprises a quadruplex, and said at least two<br>
sequences  are  bonded together  by  homologous  base<br>
interaction with parallel or antiparallel directionality; and<br>
contacting such as herein described said aptamer with said ligand to bind said ligand to said aptamer by interactin other than by Watson-Crick base pairing of nucleobase to nucleobase<br>
2.	The method as claimed in claim 1 wherein said aptamer is administered<br>
in an amount effective to alter a biological activity of said ligand, wherein<br>
the said administration is not in vivo.<br>
3.	The method as claimed in claim 1 wherein said aptamer is labeled and is<br>
administered in an amount effective to detect said ligand or said ligand<br>
and a  second ligand bound to  said ligand,  wherein the  said<br>
administration is not in vivo.<br>
4.	The method as claimed hi claim 1 wherein said providing comprises<br>
placing said aptamer in a test medium, and a presence or an absence of<br>
said ligand in said test medium is detected.<br>
5.	The method as claimed in claim 1 - 4 wherein said aptamer comprises a<br>
duplex, and said at least two sequences are bonded together by Watson-<br>
Crick complementary base interaction with parallel directionality.<br>
6.	The method as claimed in claim 1 - 4 wherein said aptamer comprises a<br>
duplex, and said at least two sequences are bonded together by<br>
homologous base interaction with parallel or antiparallel directionality.<br>
7.	The method as claimed in claim 1-4 wherein said aptamer comprises a<br>
triplex, and said at least two sequences are bonded together by Watson-<br>
Crick complementary base interaction with parallel or antiparallel<br>
directionality.<br>
8.	The method as claimed in claim 1-4 wherein said aptamer comprises a<br>
triplex, and said at least two sequences are bonded together by<br>
homologous interaction with parallel and antiparallel directionality.<br>
9.	The method as claimed in claim 1-4 wherein said aptamer comprises a<br>
quadruplex, and said at least two sequences are bonded together by<br>
Watson-Crick complementary base interaction with parallel and<br>
antiparallel directionality.<br>
10.	The method as claimed in claim 1-4 wherein said aptamer comprises a<br>
quadruplex, and said at least two sequences are bonded together by<br>
homologous base interaction with parallel and antiparallel directionality.<br>
11.	The method as claimed in any preceding claims wherein said ligand<br>
comprises a protein or peptide.<br>
12.	The method as claimed in any preceding claims wherein said ligand is<br>
free of nucleobases.<br>
13.	The method as claimed in any one of claims 1 to 11 wherein some<br>
nucleobases in said at least two sequences are not paired.<br>
14.	The method as claimed in any one of claims 1 to 11 &amp; 13 wherein said<br>
at least two sequences are contained in at least two nucleic acid strands<br>
that are cross-linked.<br>
15.	The method as claimed in any preceding claims wherein the aptamer is<br>
an aptazyme and the method optionally comprises catalyzing a reaction<br>
involving said ligand.<br>
16.	The method as claimed in claim 17 wherein said ligand contains at least<br>
one of any amino acid sequence, a nucleic acid sequence, a carbohydrate<br>
and a lipid.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzcwLWRlbG5wLTIwMDQtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">770-delnp-2004-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzcwLWRlbG5wLTIwMDQtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">770-delnp-2004-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzcwLWRlbG5wLTIwMDQtY29ycmVzcG9uZGVuY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">770-delnp-2004-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzcwLWRlbG5wLTIwMDQtY29ycmVzcG9uZGVuY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">770-delnp-2004-correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzcwLWRlbG5wLTIwMDQtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">770-delnp-2004-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzcwLWRlbG5wLTIwMDQtZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">770-delnp-2004-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzcwLWRlbG5wLTIwMDQtZm9ybS0xMy5wZGY=" target="_blank" style="word-wrap:break-word;">770-delnp-2004-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzcwLWRlbG5wLTIwMDQtZm9ybS0xOS5wZGY=" target="_blank" style="word-wrap:break-word;">770-delnp-2004-form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzcwLWRlbG5wLTIwMDQtZm9ybS0yLnBkZg==" target="_blank" style="word-wrap:break-word;">770-delnp-2004-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzcwLWRlbG5wLTIwMDQtZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">770-delnp-2004-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzcwLWRlbG5wLTIwMDQtZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">770-delnp-2004-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzcwLWRlbG5wLTIwMDQtZ3BhLnBkZg==" target="_blank" style="word-wrap:break-word;">770-delnp-2004-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzcwLWRlbG5wLTIwMDQtcGN0LTIxMC5wZGY=" target="_blank" style="word-wrap:break-word;">770-delnp-2004-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzcwLWRlbG5wLTIwMDQtcGN0LTMwNC5wZGY=" target="_blank" style="word-wrap:break-word;">770-delnp-2004-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzcwLWRlbG5wLTIwMDQtcGV0aXRpb24tMTM3LnBkZg==" target="_blank" style="word-wrap:break-word;">770-delnp-2004-petition-137.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="218209-a-rotor-for-food-pulping-and-straining-machiens.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="218211-internal-combustion-engine-with-at-least-two-intake-valves-per-cylinder.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>218210</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>770/DELNP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>22/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>30-May-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>31-Mar-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>24-Mar-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>INGENEUS CORPORATION</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>CHANCERY HOUSE, HIGH STREET, BRIDGETOWN, BARBADOS.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ERIKSON GLEN H.</td>
											<td>P.O.BOX 3001,NASSAU,THE BAHAMAS</td>
										</tr>
										<tr>
											<td>2</td>
											<td>DAKSIS JASMINE I</td>
											<td>36 CARTIER CRESCENT,RICHMOND HILL,ONTARIO L4C 2N2,CANADA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IB02/03827</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-09-16</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>09/961,865</td>
									<td>2001-09-24</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/218210-a-method-for-binding-a-ligand by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:35:49 GMT -->
</html>
