<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/218516-hepatitis-c-virus-vaccine by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:51:48 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 218516:HEPATITIS C VIRUS VACCINE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">HEPATITIS C VIRUS VACCINE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention features Ad6 vectors and a nucleic acid encoding a Met^NS3-NS4A-NS4B&quot;NS5A-NS5B polypeptide containing an inactive NS5B RNA-dependent RNA polymerase region. The nucleic acid is particularly useful as a component of an adenovector or DNA plasmid vaccine providing a broad range of antigens for generating an HCV specific cell mediated immune (CMI) response against HCV.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
This invention relates to hepatitis C virus vaccine<br>
BACKGROUND OF THE INVENTION<br>
About 3% of the world's population are infected with the Hepatitis C virus (HCV). (Wasley et al, Semin. Liver Dis, 20, 1-16, 2000.) Exposure to HCV results in an overt acute disease in a small percentage of cases, while in most instances the virus establishes a chronic infection causing liver inflammation and slowly progresses into liver failure and cirrhosis. (Iwarson, FEMS Microbiol. Rev. 14,201-204, 1994.) In addition, epidemiological surveys indicate an important role of HCV in the pathogenesis of hepatocellular carcinoma. (Kew, FEMS Microbiol. Rev. 14, 211-220, 1994, Alter, Blood 55, 1681-1695, 1995.)<br>
Prior to the implementation of routine blood screening for HCV in 1992, most infections were contracted by inadvertent exposure to contaminated blood, blood products or transplanted organs. In those areas where blood screening of HCV is carried out, HCV is primarily contracted through direct percutaneous exposure to infected blood, i.e., intravenous drug use. Less frequent methods of transmission include perinatal exposure, hemodialysis, and sexual contact with an HCV infected person. (Alter et al, N. Engl. J. Med 341(8), 556-562, 1999, Alter, J. Hepatol. 31 Suppl. 88-91, 1999. Semin. Liver. Dis. 201, 1-16, 2000.)<br>
The HCV genome consists of a single strand RNA about 9.5 kb encoding a precursor polyprotein of about 3000 amino acids. (Choo et al, Science 244, 362-364, 1989, Choo et al, Science 244, 359-362, 1989, Takamizawa et al, J. Virol 65, 1105-1113, 1991.) The HCV polyprotein contains the viral proteins in the order: C-El-E2-p7-NS2-NS3-NS4A-NS4B-NS5A-NS5B.<br>
Individual viral proteins are produced by proteolysis of the HCV polyprotein. Host cell proteases release the putative structural proteins C, El, E2, and p7, and create the N-terminus of NS2 at amino acid 810. (Mizushima et al, J. Virol 68, 2731-2734, 1994, Hijikatae/a/., PMAS USA 90, 10773-10777, 1993.)<br>
The non-structural proteins NS3, NS4A, NS4B, NS5A and NS5B presumably form the virus replication machinery and are released from the polyprotein. A zinc-dependent protease associated with NS2 and the N-terminus of NS3  is<br><br>
responsible for cleavage between NS2 and NS3. (Grakoui et al, J, Virol. 67, 1385-1395, 1993, Hijikatsi et al., PMA.S. USA 90,10773-10777, 1993.) A distinct serine protease located in the N-terminal domain of NS3 is responsible for proteolytic cleavages at the NS3/NS4A, NS4A/NS4B, NS4B/NS5A and NS5A/NS5B junctions. (Bartenschlager et al., J. Virol. 67, 3835-3844, 1993, Grakoui et al, Proc. Natl Acad. Sci. USA 90, 10583-10587, 1993, Tomei et al, J. Virol 67, 4017-4026, 1993.) NS4A provides a cofactor for NS3 activity. (Failla et al, J. Virol 68, 3753-3760, 1994, De Francesco et al, U.S. Patent No. 5,739,002.)<br>
NS5A is a highly phosphorylated protein conferring interferon resistance. (De Francesco et al, Semin. Liver Dis., 20(1), 69-83, 2000, Pawlotsky, Viral Hepat. Suppl 1, 47-48, 1999.)<br>
NS5B provides an RNA-dependent RNA polymerase. (De Francesco et al, International Publication Number WO 96/37619, Behrens et al, EMBO 15, 12-22, 1996,Lohmann et al., Virology 249, 108-118, 1998.)<br>
SUMMARY OF THE INVENTION<br>
The present invention features Ad6 vectors and a nucleic acid encoding a Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptide containing an inactive NS5B RNA-dependent RNA polymerase region. The nucleic acid is particularly useful as a component of an adenovector or DNA plasmid vaccine providing a broad range of antigens for generating an HCV specific cell mediated immune (CMI) response against HCV.<br>
A HCV specific CMI response refers to the production of cytotoxic T lymphocytes and T helper cells that recognize an HCV antigen. The CMI response may also include non-HCV specific immune effects.<br>
Preferred nucleic acids encode a Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptide that is substantially similar to SEQ. ID. NO. 1 and has sufficient protease activity to process itself to produce at least a polypeptide substantially similar to the NS5B region present in SEQ. ID. NO. 1. The produced polypeptide corresponding to NS5B is enzymatically inactive. More preferably, the HCV polypeptide has sufficient<br><br>
protease activity to produce polypeptides substantially similar to the NS3, NS4A, NS4B, NS5A, and NS5B regions present in SEQ. ID. NO. 1.<br>
Reference to a "substantially similar sequence" indicates an identity of at least about 65% to a reference sequence. Thus, for example, polypeptides having an amino acid sequence substantially similar to SEQ. ID. NO. 1 have an overall amino acid identity of at least about 65% to SEQ. ID. NO. 1.<br>
Polypeptides corresponding to NS3, NS4A, NS4B, NS5A, and NS5B have an amino acid sequence identity of at least about 65% to the corresponding region in SEQ. ID. NO. 1. Such corresponding polypeptides are also referred to herein as NS3, NS4A, NS4B, NS5A, and NS5B polypeptides.<br>
Thus, a first aspect of the present invention describes a nucleic acid comprising a nucleotide sequence encoding a Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptide substantially similar to SEQ. ID. NO. 1. The encoded polypeptide has sufficient protease activity to process itself to produce an NS5B polypeptide that is enzymatically inactive.<br>
In a preferred embodiment, the nucleic acid is an expression vector capable of expressing the Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptide in a desired human cell. Expression inside a human cell has therapeutic applications for actively treating an HCV infection and for prophylactically treating against an HCV infection.<br>
An expression vector contains a nucleotide sequence encoding a polypeptide along with regulatory elements for proper transcription and processing. The regulatory elements that may be present include those naturally associated with the nucleotide sequence encoding the polypeptide and exogenous regulatory elements not naturally associated with the nucleotide sequence. Exogenous regulatory elements such as an exogenous promoter can be useful for expression in a particular host, such as in a human cell. Examples of regulatory elements useful for functional expression include a promoter, a terminator, a ribosome binding site, and a polyadenylation signal.<br>
Another aspect of the present invention describes a nucleic acid comprising a gene expression cassette able to express in a human cell a Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptide substantially similar to SEQ. ID. NO. 1. The polypeptide can process itself to produce an enzymatically inactive NS5B protein. The gene expression cassette contains at least the following:<br><br>
a)	a  promoter  transcriptionally  coupled  to  a  nucleotide  sequence<br>
encoding a polypeptide;<br>
b)	a 5' ribosome binding site functionally coupled to the nucleotide<br>
sequence,<br>
c)	a terminator joined to the 3' end of the nucleotide sequence, and<br>
d)	a 3' polyadenylation signal functionally coupled to the nucleotide<br>
sequence.<br>
Reference to "transcriptionally coupled" indicates that the promoter is positioned such that transcription of the nucleotide sequence can be brought about by RNA polymerase binding at the promoter. Transcriptionally coupled does not require that the sequence being transcribed is adjacent to the promoter.<br>
Reference to "functionally coupled" indicates the ability to mediate an effect on the nucleotide sequence. Functionally coupled does not require that the coupled sequences be adjacent to each other. A 3' polyadenylation signal functionally coupled to the nucleotide sequence facilitates cleavage and polyadenylation of the transcribed RNA. A 5' ribosome binding site functionally coupled to the nucleotide sequence facilitates ribosome binding.<br>
In preferred embodiments the nucleic acid is a DNA plasmid vector or an adenovector suitable for either therapeutic application in treating HCV or as an intermediate in the production of a therapeutic vector. Treating HCV includes actively treating an HCV infection and prophylactically treating against an HCV infection.<br>
Another aspect of the present invention describes an adenovector comprising a Met-NS3-NS4A-NS4B-NS5A-NS5B expression cassette able to express a polypeptide substantially similar to SEQ. ID. NO. 1 that is produced by a process involving (a) homologous recombination and (b) adenovector rescue. The homologous recombinant step produces an adenovirus genome plasmid. The adenovector rescue step produces the adenovector from the adenogenome plasmid.<br>
Adenovirus genome plasmids described herein contain a recombinant adenovirus genome having a deletion in the El region and optionally in the E3 region and a gene expression cassette inserted into one of the deleted regions. The recombinant adenovirus genome is made of regions substantially similar to one or more adenovirus serotypes.<br><br>
Another aspect of the present invention describes an adenovector consisting of the nucleic acid sequence of SEQ. ID. NO. 4 or a derivative thereof, wherein said derivative thereof has the HCV polyprotein encoding sequence present in SEQ. ID. NO. 4 replaced with the HCV polyprotein encoding sequence of either SEQ. ID. NO. 3, SEQ. ID. NO. 10 or SEQ. ID. NO. 11.<br>
Another aspect of the present invention describes a cultured recombinant cell comprising a nucleic acid containing a sequence encoding a Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptide substantially similar to SEQ. ID. NO. 1. The recombinant cell has a variety of uses such as being used to replicate nucleic acid encoding the polypeptide in vector construction methods.<br>
Another aspect of the present invention describes a method of making an adenovector comprising a Met-NS3-NS4A-NS4B-NS5A-NS5B expression cassette able to express a polypeptide substantially similar to SEQ. ID. NO, 1. The method involves the steps of (a) producing an adenovirus genome plasmid containing a recombinant adenovirus genome with deletions in the El and E3 regions and a gene expression cassette inserted into one of the deleted regions and (b) rescuing the adenovector from the adenovirus genome plasmid.<br>
Another aspect of the present invention describes a pharmaceutical composition comprising a vector for expressing a Met-NS3-NS4A"NS4B-NS5A-NS5B polypeptide substantially similar to SEQ. ID. NO. 1 and a pharmaceutically acceptable carrier. The vector is suitable for administration and polypeptide expression in a patient.<br>
A "patient" refers to a mammal capable of being infected with HCV. A patient may or may not be infected with HCV. Examples of patients are humans and chimpanzees.<br>
Another aspect of the present invention describes a method of treating a patient comprising the step of administering to the patient an effective amount of a vector expressing a Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptide substantially similar to SEQ. ID. NO. 1. The vector is suitable for administration and polypeptide expression in the patient.<br>
The patient undergoing treatment may or may not be infected with HCV. For a patient infected with HCV, an effective amount is sufficient to achieve one or<br><br>
more ot the following effects: reduce the ability ot HCV to replicate, reduce HUV load, increase viral clearance, and increase one or more HCV specific CMI responses. For a patient not infected with HCV, an effective amount is sufficient to achieve one or more of the following: an increased ability to produce one or more components of a HCV specific CMI response to a HCV infection, a reduced susceptibility to HCV infection, and a reduced ability of the infecting virus to establish persistent infection for chronic disease.<br>
Another aspect of the present invention features a recombinant nucleic acid comprising an Ad6 region and a region not present in Ad6. Reference to "recombinant" nucleic acid indicates the presence of two or more nucleic acid regions not naturally associated with each other. Preferably, the Ad6 recombinant nucleic acid contains Ad6 regions and a gene expression cassette coding for a polypeptide heterologous to Ad6.<br>
Other features and advantages of the present invention are apparent from the additional descriptions provided herein including the different examples. The provided examples illustrate different components and methodology useful in practicing the present invention. The examples do not limit the claimed invention. Based on the present disclosure the skilled artisan can identify and employ other components and methodology usefiil for practicing the present invention.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figures lA and IB illustrate SEQ. ID. NO. 1.<br>
Figures 2A, 2B, 2C, and 2D illustrate SEQ. ID. NO. 2. SEQ. ID. NO. 2 provides a nucleotide sequence coding for SEQ. ID. NO. 1 along with an optimized internal ribosome entry site and TAAA termination. Nucleotides 1-6 provides an optimized internal ribosome entry site. Nucleotides 7-5961 code for a HCV Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptide with nucleotides in positions 5137 to 5145 providing a AlaAlaGly sequence in amino acid positions 1711 to 1713 that renders NS5B inactive. Nucleotides 5962-5965 provide a TAAA termination.<br>
Figures 3A, 3B, 3C, and 3D illustrate SEQ. ID. NO. 3. SEQ. ID. NO. 3 is a codon optimized version of SEQ. ID. NO. 2. Nucleotides 7-5961 encode a HCV Met-NS3-NS4A-NS4B.NS5A-NS5B polypeptide.<br><br>
Figures 4A-4M illustrate MRKAd6-NSmut (SEQ. ID. NO. 4). SEQ. ID. NO. 4 is an adenovector containing an expression cassette where the polypeptide of SEQ. ID. NO. I is encoded by SEQ. ID. NO. 2. Base pairs 1-450 correspond to the Ad5 bp 1 to 450; base pairs 462 to 1252 correspond to the human CMV promoter; base pairs 1258 to 1267 correspond to the Kozak sequence; base pairs 1264 to 7222 correspond to the NS genes; base pairs 7231 to 7451 correspond to the BGH polyadenylation signal; base pairs 7469 to 9506 correspond to Ad5 base pairs 3511 to 5548; base pairs 9507 to 32121 correspond to Ad6 base pairs 5542 to 28156; base pairs 32122 to 35117 correspond to Ad6 base pairs 30789 to 33784; and base pairs 35118 to 37089 correspond to Ad5 base pairs 33967 to 35935.<br>
Figures 5A-50 illustrate SEQ. ID. NOs. 5 and 6. SEQ. ID. NO. 5 encodes a HCV Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptide with an active RNA dependent RNA polymerase. SEQ. ID. NO. 6 provides the amino acid sequence for the polypeptide.<br>
Figures 6A-6C provide the nucleic acid sequence for pVlJnsA (SEQ. ID. NO. 7).<br>
Figures 7A-7N provide the nucleic acid sequence for the Ad6 genome (SEQ. ID. NO. 8).<br>
Figures 8A-8K provide the nucleic acid sequence for the Ad5 genome (SEQ. ID. NO. 9).<br>
Figure 9 illustrates different regions of the Ad6 genome. The linear (35759 bp) ds DNA genome is indicated by two parallel lines and is divided into 100 map units. Transcription units are shown relative to their position and orientation in the genome. Early genes (El A, ElB, E2A/B, E3 and E4 are indicated by gray arrows. Late genes (LI to L5) , indicated by black arrows, are produced by alternative splicing of a transcript produced from the major late promoter (MLP) and all contain the tripartite leader (1, 2, 3) at their 5' ends. The El region is located from approximately 1.0 to 11.5 map units, the E2 region from 75.0 to 11.5 map units, E3 from 76.1 to 86.7 map units, and E4 from 99.5 to 91.2 map units. The major late transcription unit is located between 16.0 and 91.2 map units.<br>
Figure 10 illustrates homologous recombination to recover pAdEl-E3+ containing Ad6 and Ad5 regions.<br><br>
Figure 11 illustrates homologous recombinant to recover a pAdEl-E3+ containing Ad6 regions.<br>
Figure 12 illustrates a western blot on whole-cell extracts from 293 cells transfected with plasmid DNA expressing different HCV NS cassettes. Mature NS3 and NS5A products were detected with specific antibodies. "pVlJns-NS" refers to a pVl JnsA plasmid where a Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptide is encoded by SEQ. ID. NO. 5, and SEQ. ID. NO. 5 is inserted between bases 1881 and 1912 of SEQ. ID. NO. 7. "pVlJns-NSmut" refers to a pVlJnsA plasmid where SEQ. ID. NO. 2 is inserted between bases 1882 and 1925 of SEQ, ID. NO. 7. "pVIJns-NSOPTmut" refers to a pVlJnsA plasmid where SEQ. ID. NO. 3 is inserted between bases 1881 and 1905ofSEQ.ID.NO.7.<br>
Figures 13A and 13B illustrate T cell responses by IFND ELIspot induced in C57black6 mice (A) and BalbC mice (B) by two injections of 25Dg and 50Dg, respectively, of plasmid DNA encoding the different HCV NS cassettes with Gene Electro-Transfer (GET).<br>
Figure 14 illustrates protein expression from different adenovectors upon infection of HeLa cells. MRKAdS-NSmut is an adenovector based on an Ad5 sequence (SEQ. ID. NO. 9), where the Ad5 genome has an El deletion of base pairs 451 to 3510, an E3 deletion of base pairs 28134 to 30817, and has the NS3-NS4A-NS4B-NS5A-NS5B expression cassette as provided in base pairs 451 to 7468 of SEQ. ID. NO. 4 inserted between positions 450 and 3511. Ad5-NS is an adenovector based on an Ad5 backbone with an El deletion of base pairs 342 to 3523, and E3 deletion of base pairs 28134 to 30817 and containing an expression cassette encoding a NS3-NS4A-NS4B-NS5A-NS5B from SEQ. ID. NO. 5. "MRKAd6-NSOPTmut" refers to an adenovector having a modified SEQ. ID. NO. 4 sequence, wherein base pairs 1258 to 7222 of SEQ. ID. NO. 4 is replaced with SEQ. ID. NO. 3.<br>
Figure 15 illustrates T cell responses by IFND ELIspot induced in C57black6 mice by two injections of 109 vp of adenovectors containing different HCV non-structural gene cassettes.<br>
Figures 16A-16D illustrate T cell responses by IFND ELIspot induced in Rhesus monkeys by one or two injections of 1010 vp (A) or 1011 vp (B) of adenovectors containing different HCV non-structural eene cassettes.<br><br>
Figures 17A and 17B illustrates CD8+ T cell responsW by IFND ICS induced in Rhesus monkeys by two injections of 10'° vp (A) or 1011 vp (B) of adenovectors encoding the different HCV non-structural gene cassettes.<br>
Figures 18A-18F illustrate T cell responses by bulk CTL assay induced in Rhesus monkeys by two injections of 1011 vp of Ad5-NS (A), MRKAdS-NSmut (B), or MRKAd6-NSmut (C).<br>
Figure 19 illustrates the plasmid pE2.<br>
Figures 20A-D illustrates the partial codon optimized sequence NSsuboptmut (SEQ. ID. NO. 10). Coding sequence for the Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptide is from base 7 to 5961.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The present invention features Ad6 vectors and nucleic acid encoding a Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptide that contains an inactive NS5B region. Providing an inactive NS5B region supplies NS5B antigens while reducing the possibility of adverse side effects due to an active viral RNA polymerase. Uses of the featured nucleic acid include use as a vaccine component to introduce into a cell an HCV polypeptide that provides a broad range of antigens for generating a CMI response against HCV, and as an intermediate for producing such a vaccine component.<br>
The adaptive cellular immune response can function to recognize viral antigens in HCV infected cells throughout the body due to the ubiquitous distribution of major histocompatibility complex (MHC) class I and II expression, to induce immunological memory, and to maintain immunological memory. These functions are attributed to antigen-specific CD4+ T helper (Th) and CD8+ cytotoxic T cells (CTL).<br>
Upon activation via their specific T cell receptors, HCV specific Th cells fulfill a variety of immunoregulatory functions, most of them mediated by Thl and Th2 cytokines. HCV specific Th cells assist in the activation and differentiation of B cells and induction and stimulation of virus-specific cytotoxic T cells. Together with CTL, Th cells may also secrete IFN-n and TNF-D that inhibit replication and gene expression of several viruses. Additionally, Th cells and CTL, the main effector cells, can induce apoptosis and lysis of virus infected cells.<br><br>
HCV specific CTL are generated from antigens processed by professional antigen presenting cells (pAPCs). Antigens can be either synthesized within or introduced into pAPCs. Antigen synthesis in a pAPC can be brought about by introducing into the cell an expression cassette encoding the antigen.<br>
A preferred route of nucleic acid vaccine administration is an intramuscular route. Intramuscular administration appears to result in the introduction and expression of nucleic acid into somatic cells and pAPCs. HCV antigens produced in the somatic cells can be transferred to pAPCs for presentation in the context of MHC class 1 molecules.     (Donnelly et al, Annu.  Rev.  Immunol,   75:617-648,   1997.)<br>
pAPCs process longer length antigens into smaller peptide antigens in the proteasome complex. The antigen is translocated into the endoplasmic reticulum/Golgi complex secretory pathway for association with MHC class I proteins. CD8+ T lymphocytes recognize antigen associated with class I MHC via the T cell receptor (TCR) and the CDS cell surface protein.<br>
Using a nucleic acid encoding a Met-NS3-NS4A-NS4B"NS5A-NS5B polypeptide as a vaccine component allows for production of a broad range of antigens capable of generating CMI responses from a single vector. The polypeptide should be able to process itself sufficiently to produce at least a region corresponding to NS5B. Preferred nucleic acids encode an amino acid sequence substantially similar to SEQ. ID. NO. 1 that has sufficient protease activity to process itself to produce individual HCV polypeptides substantially similar to the NS3, NS4A, NS4B, NS5A, and NS5B regions present in SEQ. ID. NO. 1.<br>
A polypeptide substantially similar to SEQ. ID. NO. 1 with sufficient protease activity to process itself in a cell provides the cell with T cell epitopes that are present in several different HCV strains. Protease activity is provided by NS3 and NS3/NS4A proteins digesting the Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptide at the appropriate cleavage sites to release polypeptides corresponding to NS3, NS4A, NS4B, NS5A, and NS5B. Self- processing of the Met-NS3-NS4A-NS4B-NS5A-NS5B generates polypeptides that approximate naturally occurring HCV polypeptides.<br>
Based on the guidance provided herein a sufficiently strong immune response can be generated to achieve beneficial effects in a patient.   The provided<br><br>
guidance includes information concerning HCV sequence selection, vector selection, vector production, combination treatment, and administration.<br>
I. HCV SEQUENCES<br>
A variety of different nucleic acid sequences can be used as a vaccine component to supply a HCV Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptide to a cell or as an intermediate to produce vaccine components. The starting point for obtaining suitable nucleic acid sequences are preferably naturally occurring NS3-NS4A-NS4B-NS5A-NS5B polypeptide sequences modified to produce an inactive NS5B.<br>
The use of a HCV nucleic acid sequence providing HCV non-structural antigens to generate a CMI response is mentioned by Cho et aL, Vaccine 7 7:1136-1144, 1999, Paliard et aL, International Publication Number WO 01/30812 (not admitted to be prior art to the claimed invention), and Coit et aL, International Publication Number WO 01/38360 (not admitted to be prior art to the claimed invention). Such references fail to describe, for example, a polypeptide that processes itself to produce an inactive NS5B, and the particular combinations of HCV sequences and delivery vehicles employed herein.<br>
Modifications to a HCV Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptide sequence can be produced by altering the encoding nucleic acid. Alterations can be performed to create deletions, insertions and substitutions.<br>
Small modifications can be made in NS5B to produce an inactive polymerase by targeting motifs essentially for replication. Examples of motifs critical for NS5B activity and modifications that can be made to produce an inactive NS5B are described by Lohmann et aL, Journal of Virology 77:8416-8426, 1997, and Kolykhalov et aL, Journal of Virology 74:2046-2051, 2000.<br>
Additional factors to take into account when producing modifications to a HCV Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptide include maintaining the ability to self-process and maintaining T cell antigens. The ability of the HCV polypeptide to process itself is determined to a large extent by a functional NS3 protease. Modifications that maintain NS3 activity protease activity can be obtained by taking into account the NS3 protein, NS4A which serves as a cofactor for NS3, and<br><br>
NS3 protease recognition sites present within the NS3-NS4A-NS4B-NS5A-NS5B polypeptide.<br>
Different modifications can be made to naturally occurring NS3-NS4A-NS4B-NS5A-NS5B polypeptide sequences to produce polypeptides able to elicit a broad range of T cell responses. Factors influencing the ability of a polypeptide to elicit a broad T cell response include the preservation or introduction of HCV specific T cell antigen regions and prevalence of different T cell antigen regions in different HCV isolates.<br>
Numerous examples of naturally occurring HCV isolates are well known in the art. HCV isolates can be classified into the following six major genotypes comprising one or more subtypes: HCV-l/(la,lb,lc), HCV-2/(2a,2b,2c), HCV-3/(3a,3b,10a), HCV-4/(4a), HCV-5/(5a) and HCV-6/(6a,6b,7b,8b,9a,lla). (Simmonds, J. Gen. Virol, 693-712, 2001.) Examples of particular HCV sequences such as HCV-BK, HCV-J, HCV-N, HCV-H, have been deposited in GenBank and described in various publications. (See, for example, Chamberlain et, al., J.Gew. ViroL, 1341-1347, 1997.)<br>
HCV T cell antigens can be identified by, for example, empirical experimentation. One way of identifying T cell antigens involves generating a series of overlapping short peptides from a longer length polypeptide and then screening the T-cell populations from infected patients for positive clones. Positive clones are activated/primed by a particular peptide. Techniques such as IFND-ELISPOT, IFNG-Intracellular staining and bulk CTL assays can be used to measure peptide activity. Peptides thus identified can be considered to represent T-cell epitopes of the respective pathogen.<br>
HCV T cell antigen regions from different HCV isolates can be introduced into a single sequence by, for example, producing a hybrid NS3-NS4A-NS4B-NS5A-NS5B polypeptide containing regions from two or more naturally occurring sequences. Such a hybrid can contain additional modifications, which preferably do not reduce the ability of the polypeptide to produce an HCV CMI response.<br>
The ability of a modified Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptide to process itself and produce a CMI response can be determined using<br><br>
techniques described herein or well known in the art. Such techniques include the use of IFNn-ELISPOT, DFND-Intracellular staining and bulk CTL assays to measure a HCV specific CMI response.<br>
A. Met-NS3-NS4A-NS4B-NS5A-NS5B Sequences<br>
SEQ. ID. NO. 1 provides a preferred Met-NS3-NS4A-NS4B-NS5A-NS5B sequence. SEQ. ID. NO. 1 contains a large number of HCV specific T cell antigens that are present in several different HCV isolates. SEQ. ID. NO. 1 is similar to the NS3-NS4A-NS4B-NS5A-NS5B portion of the HCV BK strain nucleotide sequence (GenBank accession number M58335).<br>
In SEQ. ID. NO. 1 anchor positions important for recognition by MHC class I molecules are conserved or represent conservative substitutions for 18 out of 20 known T-cell epitopes in the NS3-NS4A-NS4B-NS5A-NS5B portion of HCV polyproteins. With respect to the remaining two known T-cell epitopes, one has a non-conservative anchor substitution in SEQ. ID. NO. 1 that may still be recognized by a different HLA supertype and one epitope has one anchor residue not conserved. HCV T-cell epitopes are described in Chisari et al., Curr. Top, Microbiol Immunol, 242:299-325, 2000, and Lechner et al. 7. Exp. Med 9:1499-1512, 2000.<br>
Differences between the HCV-BK NS3-NS4A-NS4B-NS5A-NS5B nucleotide sequence and SEQ. ID. NO. 1 include the introduction of a methionine at the 5' end and the presence of modified NS5B active site residues in SEQ. ID. NO. 1. The modification replaces GlyAspAsp with AlaAlaGly (residues 1711-1713) to inactivate NS5B.<br>
The encoded HCV Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptide preferably has an amino acid sequence substantially similar to SEQ. ID. NO. 1. In different embodiments, the encoded HCV Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptide has an amino acid identify to SEQ. ID. NO. 1 of at least 65%, at least 75%, at least 85%, at least 95%, at least 99% or 100%; or differs from SEQ. ID. NO. 1 by 1-2, 1-3, 1-4, 1-5, 1-6, 1-7, 1-8, 1-9, 1-10, 1-11, 1-12, 1-13, 1-14, 1-15, 1-16, 1-17, 1-18, 1-19, or 1-20 amino acids.<br>
Amino acid differences between a Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptide and SEQ. ID. NO. 1 are calculated by determining the minimum number of<br><br>
amino acid modifications in which the two sequences differ. Amino acid modifications can be deletions, additions, substitutions or any combination thereof<br>
Amino acid sequence identity is determined by methods well known in the art that compare the amino acid sequence of one polypeptide to the amino acid sequence of a second polypeptide and generate a sequence alignment. Amino acid identity is calculated from the alignment by counting the number of aligned residue pairs that have identical amino acids.<br>
Methods for determining sequence identity include those described by Schuler, G.D. in Bioinformatics: A Practical Guide to the Analysis of Genes and Proteins, Baxevanis, A.D. and Ouelette, B.F.F., eds., John Wiley &amp; Sons, Inc, 2001; Yona, et ai, in Bioinformatics: Sequence, structure and databanks, Higgins, D. and Taylor, W. eds, Oxford University Press, 2000; and Bioinformatics: Sequence and Genome Analysis, Mount, D.W., ed., Cold Spring Harbor Laboratory Press, 2001). Methods to determine amino acid sequence identity are codified in publicly available computer programs such as GAP (Wisconsin Package Version 10.2, Genetics Computer Group (GCG), Madison, Wise), BLAST (Altschul et aL, J. Mol Biol 275(3;:403-10, 1990), and FASTA (Pearson, Methods in Enzymology 753:63-98, 1990, R.F, Doolittle, ed.).<br>
In an embodiment of the present invention sequence identity between two polypeptides is determined using the GAP program (Wisconsin Package Version 10.2, Genetics Computer Group (GCG), Madison, Wise). GAP uses the alignment method of Needleman and Wunsch. (Needleman, et al., J. Mol. Biol. 4S:443-453, 1970.) GAP considers all possible alignments and gap positions between two sequences and creates a global alignment that maximizes the number of matched residues and minimizes the number and size of gaps. A scoring matrix is used to assign values for symbol matches. In addition, a gap creation penalty and a gap extension penalty are required to limit the insertion of gaps into the alignment. Default program parameters for polypeptide comparisons using GAP are the BLOSUM62 (Henikoff et al, Proc. Natl Acad. ScL USA, S9:10915-10919, 1992) amino acid scoring matrix (MATrix=blosum62.cmp), a gap creation parameter (GAPweight=8) and a gap extension pararameter (LENgthweight=2).<br><br>
More preferred HCV Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptides in addition to being substantially similar to SEQ. ID. NO. 1 across their entire length produce individual NS3, NS4A, NS4B, NS5A and NS5B regions that are substantially similar to the corresponding regions present in SEQ. ID. NO. 1. The corresponding regions in SEQ. ID. NO. 1 are provided as follows: Met-NS3 amino acids 1-632; NS4A amino acids 633-686; NS4B amino acids 687-947; NS5A amino acids 948-1394; and NS5B amino acids 1395-1985.<br>
In different embodiments a NS3, NS4A, NS4B, NS5A and/or NS5B region has an amino acid identity to the corresponding region in SEQ. ID. NO. I of at least 65%, at least 75%, at least 85%, at least 95%, at least 99%, or 100%; or an amino acid difference of 1-2, 1-3, 1-4, 1-5, 1-6, 1-7, 1-8, 1-9, 1-10, 1-11, 1-12, 1-13, 1-14, 1-15, 1-16, 1-17, 1-18, 1-19, or 1-20 amino acids.<br>
Amino acid modifications to SEQ. ID. NO. 1 preferably maintain all or most of the T-cell antigen regions. Differences in naturally occurring amino acids are due to different amino acid side chains (R groups). An R group affects different properties of the amino acid such as physical size, charge, and hydrophobicity. Amino acids can be divided into different groups as follows: neutral and hydrophobic (alanine, valine, leucine, isoleucine, proline, tyrptophan, phenylalanine, and methionine); neutral and polar (glycine, serine, threonine, tryosine, cysteine, asparagine, and glutamine); basic (lysine, arginine, and histidine); and acidic (aspartic acid and glutamic acid).<br>
Generally, in substituting different amino acids it is preferable to exchange amino acids having similar properties. Substituting different amino acids within a particular group, such as substituting valine for leucine, arginine for lysine, and asparagine for glutamine are good candidates for not causing a change in polypeptide tertiary structure.<br>
Starting with a particular amino acid sequence and the known degeneracy of the genetic code, a large number of different encoding nucleic acid sequences can be obtained. The degeneracy of the genetic code arises because almost all amino acids are encoded by different combinations of nucleotide triplets or "codons". The translation of a particular codon into a particular amino acid is well known in the art (see, e.g., Lewin GENES IV, p. 119, Oxford University Press, 1990). Amino acids are encoded by codons as follows:<br><br>
A=Ala=Alanine: codons GCA, GCC, GCG, GCU C=Cys=Cysteine: codons UGC, UGU D=Asp=Aspartic acid: codons GAC, GAU E=Glu=Glutamic acid: codons GAA, GAG F=Phe=Phenylalanine: codons UUC, UUU G=Gly=Glycine: codons GGA, GGC, GGG, GGU H=His=Histidine: codons CAC, CAU I=Ile=Isoleucine: codons AUA, AUG, AUU K=Lys=Lysine: codons AAA, AAG<br>
L=Leu=Leucine: codons UUA, UUG, CUA, CUC, CUG, CUU M=Met=Methionine: codon AUG N=Asn=Asparagine: codons AAC, AAU P=Pro=Proline: codons CCA, CCC, CCG, CCU Q=Gln=Glutamine: codons CAA, CAG<br>
R=Arg=Arginine: codons AGA, AGG, CGA, CGC, CGG, CGU S=Ser-Serine: codons AGC, AGU, UCA, UCC, UCG, UCU T=Thr=Threonine: codons ACA, ACC, ACG, ACU V-Val=Valine: codons GUA, GUC, GUG, GUU W=Trp=Tryptophan: codon UGG Y=Tyr=Tyrosine: codons UAC, UAU.<br>
Nucleic acid sequences can be optimized in an effort to enhance expression in a host. Factors to be considered include C:G content, preferred codons, and the avoidance of inhibitory secondary structure. These factors can be combined in different ways in an attempt to obtain nucleic acid sequences having enhanced expression in a particular host. (See, for example, Donnelly et ai, International Publication Number WO 97/47358.)<br>
The ability of a particular sequence to have enhanced expression in a particular host involves some empirical experimentation. Such experimentation involves measuring expression of a prospective nucleic acid sequence and, if needed, altering the sequence.<br>
B. Encoding Nucleotide Sequences<br><br>
SEQ. ID. NOs. 2 and 3 provide two examples of nucleotide sequences encoding a Met-NS3-NS4A-NS4B-NS5A-NS5B sequence. The coding sequence of SEQ. ID. NO. 2 is similar (99.4% nucleotide sequence identity) to the NS3-NS4A-NS4B-NS5A-NS5B region of the naturally occurring HCV-BK sequence (GenBank accession number M58335). SEQ. ID. NO. 3 is a codon-optimized version of SEQ. ID. NO. 2. SEQ. ID. NOs. 2 and 3 have a nucleotide sequence identity of 78.3%.<br>
Differences between the HCV-BK NS3-NS4A-NS4B-NS5A-NS5B nucleotide (GenBank accession number M58335) and SEQ. ID. NO. 2, include SEQ. ID. NO. 2 having a ribosome binding site, an ATG methionine codon, a region coding for a modified NS5B catalytic domain, a TAAA stop signal and an additional 30 nucleotide differences. The modified catalytic domain codes for a AlaAlaGly (residues 171M7I3) instead of GlyAspAsp to inactivate NS5B.<br>
A nucleotide sequence encoding a HCV Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptide is preferably substantially similar to the SEQ. ID. NO. 2 coding region. In different embodiments, the nucleotide sequence encoding a HCV Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptide has a nucleotide sequence identify to the SEQ. ID. NO. 2 coding region of at least 65%, at least 75%, at least 85%, at least 95%, at least 99%, or 100%; or differs from SEQ. ID. NO. 2 by 1-2, 1-3, 1-4, 1-5, 1-6, 1-7, 1-8, 1-9, 1-10, 1-11, 1-12, 1-13, 1-14, 1-15, 1-16, 1-17, 1-18, 1-19, 1-20, 1-25, 1-30, 1-35, 1-40, 1-45, or 1-50 nucleotides.<br>
Nucleotide differences between a sequence coding Met-NS3-NS4A-NS4B-NS5A-NS5B and the SEQ. ID. NO. 2 coding region are calculated by determining the minimum number of nucleotide modifications in which the two sequences differ. Nucleotide modifications can be deletions, additions, substitutions or any combination thereof<br>
Nucleotide sequence identity is determined by methods well known in the art that compare the nucleotide sequence of one sequence to the nucleotide sequence of a second sequence and generate a sequence alignment. Sequence identity is determined from the alignment by counting the number of aligned positions having identical nucleotides.<br>
Methods for determining nucleotide sequence identity between two polynucleotides include those described by Schuler, in Bioinformatics: A Practical<br><br>
Guide to the Analysis of Genes and Proteins, Baxevanis, A.D. and Ouelette, B.F.F., eds., John Wiley &amp; Sons, Inc, 2001; Yona et al.,. in Bioinformatics: Sequence, structure and databanks, Higgins, D. and Taylor, W. eds, Oxford University Press, 2000; and Bioinformatics: Sequence and Genome Analysis, Mount, D.W., ed., Cold Spring Harbor Laboratory Press, 2001). Methods to determine nucleotide sequence identity are codified in publicly available computer programs such as GAP (Wisconsin Package Version 10.2, Genetics Computer Group (GCG), Madison, Wise), BLAST (Altschul et aL, 1 Mol Biol 2/5f3;:403-10, 1990), and FASTA (Pearson, W.R., Methods in Enzymology 7SJ:63-98, 1990, R.F. Doolittle, ed.).<br>
In an embodiment of the present ivnention, sequence identity between two polynucleotides is determined by application of GAP (Wisconsin Package Version 10.2, Genetics Computer Group (GCG), Madison, Wise). GAP uses the alignment method of Needleman and Wunsch. (Needleman et al, J. Mol BioL 45:443-453, 1970.) GAP considers all possible alignments and gap positions between two sequences and creates a global alignment that maximizes the number of matched residues and minimizes the number and size of gaps. A scoring matrix is used to assign values for symbol matches. In addition, a gap creation penalty and a gap extension penalty are required to limit the insertion of gaps into the alignment. Default program parameters for polynucleotide comparisons using GAP are the nwsgapdna.cmp scoring matrix (MATrix=nwsgapdna.cmp), a gap creation parameter (GAPweight=50) and a gap extension pararameter (LENgthweight=3).<br>
More preferred HCV Met-NS3-NS4A-NS4B-NS5A-NS5B nucleotide sequences in addition to being substantially similar across its entire length, produce individual NS3, NS4A, NS4B, NS5A and NS5B regions that are substantially similar to the corresponding regions present in SEQ. ID. NO. 2. The corresponding coding regions in SEQ. ID. NO. 2 are provided as follows: Met-NS3, nucleotides 7-1902; NS4A nucleotides 1903-2064; NS4B nucleotides 2065-2847; NS5A nucleotides 2848-4188: NS5B nucleotides 4189-5661.<br>
In different embodiments a NS3, NS4A, NS4B, NS5A and/or NS5B encoding region has a nucleotide sequence identity to the corresponding region in SEQ. ID. NO. 2 of at least 65%, at least 75%, at least 85%, at least 95%, at least 99% or 100%; or a nucleotide difference to SEQ. ID. NO. 2 of 1-2, 1-3, 1-4, 1-5, 1-6, 1-7, 1-8,<br><br>
1-9, 1-10, 1-11, 1-12, 1-13, 1-14, 1-15, 1-16, 1-17, 1-18, 1-19, 1-20, 1-25, 1-30, 1-35, 1-40, 1-45, or 1-50 nucleotides.<br>
C. Gene Expression Cassettes<br>
A gene expression cassette contains elements needed for polypeptide expression. Reference to "polypeptide" does not provide a size limitation and includes protein. Regulatory elements present in a gene expression cassette generally include: (a) a promoter transcriptionally coupled to a nucleotide sequence encoding the polypeptide, (b) a 5' ribosome binding site functionally coupled to the nucleotide sequence, (c) a terminator joined to the 3' end of the nucleotide sequence, and (d) a 3' polyadenylation signal functionally coupled to the nucleotide sequence. Additional regulatory elements useful for enhancing or regulating gene expression or polypeptide processing may also be present.<br>
Promoters are genetic elements that are recognized by an RNA polymerase and mediate transcription of downstream regions. Preferred promoters are strong promoters that provide for increased levels of transcription. Examples of strong promoters are the immediate early human cytomegalovirus promoter (CMV), and CMV with intron A. (Chapman et al, Nucl Acids Res. 19:3979-3986, 1991.) Additional examples of promoters include naturally occurring promoters such as the EFl alpha promoter, the murine CMV promoter, Rous sarcoma virus promoter, and SV40 early/late promoters and the P-actin promoter; and artificial promoters such as a synthetic muscle specific promoter and a chimeric muscle-specific/CMV promoter (Li etaL Nat Biotechnol 77:241-245, 1999, Hagstrom et al., Blood 95:2536-2542, 2000).<br>
The ribosome binding site is located at or near the initiation codon. Examples of preferred ribosome binding sites include CCACCAUGG, CCGCCAUGG, and ACCAUGG, where AUG is the initiation codon. (Kozak, Cell 44:2»3-292, 1986). Another example of a ribosome binding site is GCCACCAUGG (SEQ. ID. NO. 12).<br>
The polyadenylation signal is responsible for cleaving the transcribed RNA and the addition of a poly (A) tail to the RNA. The polyadenylation signal in higher eukaryotes contains an AAUAAA sequence about 11-30 nucleotides from the polyadenylation addition site.  The AAUAAA sequence is involved in signaling RNA<br><br>
cleavage. (Lewin, Genes IV, Oxford University Press, NY, 1990.) The poly (A) tail is important for the mRNA processing.<br>
Polyadenylation signals that can be used as part of a gene expression<br>
cassette include the minimal rabbit β -globin polyadenylation signal and the bovine<br>
growth hormone polyadenylation (BGH). (Xu et al, Gene 272:149-156, 2001, Post et<br>
al, U.S. Patent U. S. 5,122,458.) Additional examples include the Synthetic<br>
Polyadenylation Signal (SPA) and SV40 polyadenylation signal. The SPA sequence is<br>
as	follows:	AAUAAAAGAUCUUUAUUUUCAUUAGAUCUGUGUG<br>
UUGGUUUUUUGUGUG (SEQ. ID. NO. 13).<br>
Examples of additional regulatory elements useful for enhancing or regulating gene expression or polypeptide processing that may be present include an enhancer, a leader sequence and an operator. An enhancer region increases transcription. Examples of enhancer regions include the CMV enhancer and the SV40 enhancer. (Hitt et ai, Methods in Molecular Genetics 7:13-30, 1995, Xu, et al., Gene 272:149-156, 2001.) An enhancer region can be associated with a promoter.<br>
A leader sequence is an amino acid region on a polypeptide that directs the polypeptide into the proteasome. Nucleic acid encoding the leader sequence is 5' of a structural gene and is transcribed along the structural gene. An example of a leader sequences is tPA.<br>
An operator sequence can be used to regulate gene expression. For example, the Tet operator sequence can be used to repress gene expression.<br>
II. THERAPEUTIC VECTORS<br>
Nucleic acid encoding a Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptide can be introduced into a patient using vectors suitable for therapeutic administration. Suitable vectors can deliver nucleic acid into a target cell without causing an unacceptable side effect.<br>
Cellular expression is achieved using a gene expression cassette encoding a Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptide. The gene expression cassette contains regulatory elements for producing and processing a sufficient amount of nucleic acid inside a target cell to achieve a beneficial effect.<br><br>
Examples of vectors that can be used for therapeutic applications include first and second generation adenovectors, helper dependent adenovectors, adeno-associated viral vectors, retroviral vectors, alpha virus vectors, Venezuelan Equine Encephalitis virus vector, and plasmid vectors. (Hitt, et al., Advances in Pharmacology 40:137-206, 1997, Johnston et al, U.S. Patent No. 6,156,588, and Johnston et al, International Publication Number WO 95/32733.) Preferred vectors for introducing a Met-NS3"NS4A-NS4B-NS5A-NS5B polypeptide into a subject are first generation adenoviral vectors and plasmid DNA vectors.<br>
A. First Generation Adenovectors<br>
First generation adenovector for expressing a gene expression cassette contain the expression cassette in an El and optionally E3 deleted recombinant adenovirus genome. The deletion in the El region is sufficiently large to remove elements needed for adenoviral replication.<br>
First generation adenovectors for expressing a Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptide contain a El and E3 deleted recombinant adenovirus genome. The deletion in the El region is sufficiently large to remove elements needed for adenoviral replication. The combinations of deletions of the El and E3 regions are sufficiently large to accommodate a gene expression cassette encoding a Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptide.<br>
The adenovirus has a double-stranded linear genome with inverted terminal repeats at both ends. During viral replication, the genome is packaged inside a viral capsid to form a virion. The virus enters its target cell through viral attachment followed by internalization. (Hitt et al, Advances in Pharmacology 40:137-206, 1997.)<br>
Adenovectors can be based on different adenovirus serotypes such as those found in humans or animals. Examples of animal adenoviruses include bovine, porcine, chimp, murine, canine, and avian (CELO). Preferred adenovectors are based on human serotypes, more preferably Group B, C, or D serotypes. Examples of human adenovirus Group B, C, D, or E serotypes include types 2 ("Ad2"), 4 ("Ad4"), 5 ("Ad5"), 6 ("Ad6"), 24 ("Ad24"), 26 ("Ad26"), 34 ("Ad34") and 35 ("Ad35"). Adenovectors can contain regions from a single adenovirus or from two or more adenovirus.<br><br>
In different embodiments adenovectors are based on Ad5, Ad6, or a combination thereof Ad5 is described by Chroboczek, et al., J. Virology 7Sd:280-285, 1992. Ad6 is described in Figures 7A-7N. An Ad6 based vector containing Ad5 regions is described in the Example section provided below.<br>
Adenovectors do not need to have their El and E3 regions completely removed. Rather, a sufficient amount the El region is removed to render the vector replication incompetent in the absence of the El proteins being supplied in trans; and the El deletion or the combination of the El and E3 deletions are sufficiently large enough to accommodate a gene expression cassette.<br>
El deletions can be obtained starting at about base pair 342 going up to about base pair 3523 of Ad5, or a corresponding region from other adenoviruses. Preferably, the deleted region involves removing a region from about base pair 450 to about base pair 3511 of Ad5, or a corresponding region from other adenoviruses. Larger El region deletions starting at about base pair 341 removes elements that facilitate virus packaging.<br>
E3 deletions can be obtained starting at about base pair 27865 to about base pair 30995 of Ad5, or the corresponding region of other adenovectors. Preferably the deletion region involves removing a region from about base pair 28134 up to about base pair 30817 of Ad5, or the corresponding region of other adenovectors.<br>
The combination of deletions to the El region and optionally the E3 region should be sufficiently large so that the overall size of the recombinant genome containing the gene expression cassette does not exceed about 105% of the wild type adenovirus genome. For example, as recombinant adenovirus Ad5 genomes increase size above about 105% the genome becomes unstable. (Bett et al,, Journal of Virology 67:5911-5921,1993.)<br>
Preferably, the size of the recombinant adenovirus genome containing the gene expression cassette is about 85% to about 105% the size of the wild type adenovirus genome. In different embodiments, the size of the recombinant adenovirus genome containing the expression cassette is about 100% to about 105.2%, or about 100%, the size of the wild type genome.<br><br>
Approximately 7,500 kb can be inserted into an adenovirus genome with a El and E3 deletion. Without any deletion, the Ad5 genome is 35,935 base pairs and the Ad6 genome is 35,759 base pairs.<br>
Replication of first generation adenovectors can be performed by supplying the El gene products in trans. The El gene product can be supplied in trans^ for example, by using cell lines that have been transformed with the adenovirus El region. Examples of cells and cells lines transformed with the adenovirus El region are HEK 293 cells, 911 cells, PERC.6'''^ cells, and transfected primary human aminocytes cells. (Graham et aly Journal of Virology 36:59-72, 1977, Schiedner et aL, Human Gene Therapy 77:2105-2116, 2000, Fallaux et al. Human Gene Therapy 9:1909-1917, 1998, Bout et al, U.S. Patent No. 6,033,908.)<br>
A Met-NS3-NS4A-NS4B-NS5A-NS5B expression cassette should be inserted into a recombinant adenovirus genome in the region corresponding to the deleted El region or the deleted E3 region. The expression cassette can have a parallel or anti-parallel orientation. In a parallel orientation the transcription direction of the inserted gene is the same direction as the deleted El or E3 gene. In an anti-parallel orientation transcription the opposite strand serves as a template and the transcription direction is in the opposite direction.<br>
In an embodiment of the present invention the adenovector has a gene expression cassette inserted in the El deleted region. The vector contains:<br>
a)	a first adenovirus region from about base pair 1 to about base pair 450 corresponding to either Ad5 or Ad6;<br>
b)	a gene expression cassette in a HI parallel or El anti-parallel orientation joined to the first region;<br>
c)	a second adenovirus region from about base pair 3511 to about base pair 5548 corresponding to Ad5 or from about base pair 3508 to about base pair 5541 corresponding to Ad6, joined to the expression cassette;<br>
d)	a third adenovirus region from about base pair 5549 to about base pair 28133 corresponding to Ad5 or from about base pair 5542 to about base pair 28156 corresponding to Ad6, joined to the second region;<br><br>
e)	a fourth adenovirus region fi-om about base pair 30818 to about base pair 33966 corresponding to Ad5 or from about base pair 30789 to about base pair 33784 corresponding to Ad6, joined to the third region; and<br>
f)	a fifth adenovirus region from about base pair 33967 to about base pair 35935 corresponding to Ad5 or from about base pair 33785 to about base pair 35759 corresponding to Ad6 joined to the fourth region.<br>
In another embodiment of the present invention the adenovector has an expression cassette inserted in the E3 deleted region. The vector contains:<br>
a)	a first adenovirus region from about base pair 1 to about base pair 450 corresponding to either Ad5 or Ad6;<br>
b)	a second adenovirus region from about base pair 3511 to about base pair 5548 corresponding to Ad5 or from about base pair 3508 to about base pair 5541 corresponding to Ad6, joined to the first region;<br>
c)	a third adenovirus region from about base pair 5549 to about base pair 28133 corresponding to Ad5 or from about base pair 5542 to about base pair 28156 corresponding to Ad6, joined to the second region;<br>
d)	a gene expression cassette in a E3 parallel or E3 anti-parallel orientation joined to the third region;<br>
e)	a fourth adenovirus region from about base pair 30818 to about base pair 33966 corresponding to Ad5 or from about base pair 30789 to about base pair 33784 corresponding to Ad6, joined to the gene expression cassette; and<br>
f)	a fifth adenovirus region from about base pair 33967 to about base pair 35935 corresponding to Ad5 or from about base pair 33785 to about base pair 35759 corresponding to Ad6, joined to the fourth region.<br>
In preferred different embodiments concerning adenovirus regions that are present: (1) the first, second, third, fourth, and fifth region corresponds to Ad5; (2) the first, second, third, fourth, and fifth region corresponds to Ad6; and (3) the first region corresponds to Ad5, the second region corresponds to Ad5, the third region corresponds to Ad6, the fourth region corresponds to Ad6, and the fifth region corresponds to Ad5.<br>
B. DNA Plasmid Vectors<br><br>
DNA vaccine plasmid vectors contain a gene expression cassette along with elements facilitating replication and preferably vector selection. Preferred elements provide for replication in non-mammalian cells and a selectable marker. The vectors should not contain elements providing for replication in human cells or for integration into human nucleic acid.<br>
The selectable marker facilitates selection of nucleic acids containing the marker. Preferred selectable markers are those that confer antibiotic resistance. Examples of antibiotic selection genes include nucleic acid encoding resistance to ampicillin, neomycin, and kanamycin.<br>
Suitable DNA vaccine vectors can be produced starting with a plasmid containing a bacterial origin of replication and a selectable marker. Examples of bacterial origins of replication providing for higher yields include the ColEl plasmid-derived bacterial origin of replication. (Donnelly et al, Annu. Rev. Immunol 15:617-648, 1997.)<br>
The presence of the bacterial origin of replication and selectable marker allows for the production of the DNA vector in a bacterial strain such as E. coli. The selectable marker is used to eliminate bacteria not containing the DNA vector.<br>
III. AD6 RECOMBINANT NUCLEIC ACID<br>
Ad6 recombinant nucleic acid comprises an Ad6 region substantially similar to an Ad6 region found in SEQ. ID. NO. 8, and a region not present in Ad6 nucleic acid. Recombinant nucleic acid comprising Ad6 regions have different uses such as in producing different Ad6 regions, as intermediates in the production of Ad6 based vectors, and as a vector for delivering a recombinant gene.   .<br>
As depicted in Figure 9, the genomic organization of Ad6 is very similar to the genomic organization of Ad5. The homology between Ad5 and Ad6 is approximately 98%.<br>
In different embodiments, the Ad6 recombinant nucleic acid comprises a nucleotide region substantially similar to ElA, ElB, E2B, E2A, E3, E4, LI, L2, L3, or L4, or any combination thereof A substantially similar nucleic acid region to an Ad6 region has a nucleotide sequence identity of at least 65%, at least 75%, at least 85%, at least 95%, at least 99% or 100%; or a nucleotide difference of 1-2,1-3, 1-4, 1-5, 1-6,1-<br><br>
7, 1-8, 1-9, 1-10, 1-11, 1-12, 1-13, 1-14, 1-15, 1-16, 1-17, 1-18, 1-19, 1-20, 1-25, 1-30, 1-35, 1-40, 1-45, or 1-50 nucleotides. Techniques and embodiments for determining substantially similar nucleic acid sequences are described in Section LB. supra.<br>
Preferably, the recombinant Ad6 nucleic acid contains an expression cassette coding for a polypeptide not found in Ad6. Examples of expression cassettes include those coding for HCV regions and those coding for other types of polypeptides.<br>
Different types of adenoviral vectors can be produced incorporating different amounts of Ad6, such as first and second generation adenovectors. As noted in Section 11.A. supra, first generation adenovectors are defective in El and can replicate when El is supplied in trans.<br>
Second generation adenovectors contain less adenoviral genome than first generation vectors and can be used in conjugation with complementing cell lines and/or helper vectors supplying adenoviral proteins. Second generation adenovectors are described in different references such as Russell, Journal of General Virology 87:2573-2604, 2000; Hitt et al, 1997, Human Ad vectors for Gene Transfer, Advances in Pharmacology, Vol 40 Academic Press.<br>
In an embodiment of the present invention, the Ad6 recombinant nucleic acid is an adenovirus vector defective in El that is able to replicate when El is supplied in trans. Expression cassettes can be inserted into a deleted El region and/or a deleted E3 region.<br>
An example of an Ad6 based adenoviral vector with an expression cassette provided in a deleted El region comprises or consists of:<br>
a)	a first adenovirus region from about base pair 1 to about base pair 450 corresponding to either Ad5 or Ad6;<br>
b)	a gene expression cassette in a El parallel or El anti-parallel orientation joined to the first region;<br>
c)	a second adenovirus region from about base pair 3511 to about base pair 5548 corresponding to Ad5 or from about base pair 3508 to about base pair 5541 corresponding to Ad6, joined to the expression cassette;<br>
d)	a third adenovirus region from about base pair 5549 to about base pair 28133 corresponding to Ad5 or from about base pair 5542 to about base pair 28156 corresponding to Ad6, joined to the second region;<br><br>
e)	an optionally present fourth region from about base pair 28134 to about base pair 30817 corresponding to Ad5, or from about base pair 28157 to about base pair 30788 corresponding to Ad6, joined to the third region;<br>
f)	a fifth adenovirus region from about base pair 30818 to about base pair 33966 corresponding to Ad5 or from about base pair 30789 to about base pair 33784 corresponding to Ad6, wherein the fifth region is joined to the fourth region if the fourth region is present, or the fifth is joined to the third region if the fourth region is not present; and<br>
g)	a sixth adenovirus region from about base pair 33967 to about base pair 35935 corresponding to Ad5 or from about base pair 33785 to about base pair 35759 corresponding to Ad6, joined to the fifth region;<br>
wherein at least one Ad6 region is present.<br>
In different embodiments of the invention, all of the regions are from Ad6; all of the regions expect for the first and second are from Ad6; and 1, 2, 3, or 4 regions selected from the second, third, fourth, and fifth regions are from Ad6.<br>
An example of an Ad6 based adenoviral vector with an expression cassette provided in a deleted E3 region comprises or consists of:<br>
a)	a first adenovirus region from about base pair 1 to about base pair 450 corresponding to either Ad5 or Ad6;<br>
b)	a second adenovirus region from about base pair 3511 to about base pair 5548 corresponding to Ad5 or from about base pair 3508 to about base pair 5541 corresponding to Ad6, joined to the first region;<br>
c)	a third adenovirus region from about base pair 5549 to about base pair 28133 corresponding to Ad5 or from about base pair 5542 to about base pair 28156 corresponding to Ad6, joined to the second region;<br>
d)	a gene expression cassette in a E3 parallel or E3 anti-parallel orientation joined to the third region;<br>
e)	a fourth adenovirus region from about base pair 30818 to about base pair 33966 corresponding to Ad5 or from about base pair 30789 to about base pair 33784 corresponding to Ad6, joined to the gene expression cassette; and<br><br>
f) a fifth adenovirus region from about base pair 33967 to about base pair 35935 corresponding to Ad5 or from about base pair 33785 to about base pair 35759 corresponding to Ad6, joined to the fourth region;<br>
wherein at least one Ad6 region is present.<br>
In different embodiment of the invention, all of the regions are from Ad6; all of the regions expect for the first and second are from Ad6; and 1, 2, 3, or 4 regions selected from the second, third, fourth and fifth regions are from Ad6.<br>
IV. VECTOR PRODUCTION<br>
Vectors can be produced using recombinant nucleic acid techniques such as those involving the use of restriction enzymes, nucleic acid ligation, and homologous recombination. Recombinant nucleic acid techniques are well known in the art. (Ausubel, Current Protocols in Molecular Biology, John Wiley, 1987-1998, and Sambrook et al. Molecular Cloning, A Laboratory Manual 2nd Edition, Cold Spring Harbor Laboratory Press, 1989.)<br>
Intermediate vectors are used to derive a therapeutic vector or to transfer an expression cassette or portion thereof from one vector to another vector. Examples of intermediate vectors include adenovirus genome plasmids and shuttle vectors.<br>
Useful elements in an intermediate vector include an origin of replication, a selectable marker, homologous recombination regions, and convenient restriction sites. Convenient restriction sites can be used to facilitate cloning or release of a nucleic acid sequence.<br>
Homologous recombination regions provide nucleic acid sequence regions that are homologous to a target region in another nucleic acid molecule. The homologous regions flank the nucleic acid sequence that is being inserted into the target region. In different embodiments homologous regions are preferably about 150 to 600 nucleotides in length, or about 100 to 500 nucleotides in length.<br>
An embodiment of the present invention describes a shuttle vector containing a Met-NS3-NS4A-NS4B-NS5A-NS5B expression cassette, a selectable marker, a bacterial origin of replication, a first adenovirus homology region and a second adenovirus homologous region that target the expression cassette to insert in or replace an El region. The first and second homology regions flank the expression<br><br>
cassette. The first homology region contains at least about 100 base pairs substantially homologous to at least the right end (3' end) of a wild-type adenovirus region from about base pairs 4-450. The second homology contains at least about 100 base pairs substantially homologous to at least the left end (5' end) of Ad5 from about base pairs 3511-5792, or the corresponding region from another adenovirus.<br>
Reference to "substantially homologous" indicates a sufficient degree of homology to specifically recombine with a target region. In different embodiments substantially homologous refers to at least 85%, at least 95%, or 100% sequence identity. Sequence identity can be calculated as described in Section LB. supra.<br>
One method of producing adenovectors is through the creation of an adenovirus genome plasmid containing an expression cassette. The pre-Adenovirus plasmid contains all the adenovirus sequences needed for replication in the desired complimenting cell line. The pre-Adenovirus plasmid is then digested with a restriction enzyme to release the viral ITR's and transfected into the complementing cell line for virus rescue. The ITR's must be released from plasmid sequences to allow replication to occur. Adenovector rescue results in the production on an adenovector containing the expression cassette.<br>
A. Adenovirus Genome Plasmids<br>
Adenovirus genome plasmids contain an adenovector sequence inside a longer-length plasmid (which may be a cosmid). The longer-length plasmid may contain additional elements such as those facilitating growth and selection in eukaryotic or bacterial cells depending upon the procedures employed to produce and maintain the plasmid. Techniques for producing adenovirus genome plasmids include those involving the use of shuttle vectors and homologous recombination, and those involving the insertion of a gene expression cassette into an adenovirus cosmid. (Hitt et al, Methods in Molecular Genetics 7:13-30, 1995, Danthinne et al, Gene Therapy 7:1707-1714,2000.)<br>
Adenovirus genome plasmids preferably have a gene expression cassette inserted into a El or E3 deleted region. In an embodiment of the present invention, the adenovirus genome plasmid contains a gene expression cassette inserted in the El<br><br>
deleted region, an origin of replication, a selectable marker, and the recombinant adenovirus region is made up of:<br>
a)	a first adenovirus region from about base pair 1 to about base 450 corresponding to either Ad5 or Ad6;<br>
b)	a gene expression cassette in a El parallel or El anti-parallel orientation joined to the first region;<br>
c)	a second adenovirus region from about base pair 3511 to about base pair 5548 corresponding to Ad5 or from about base pair 3508 to about base pair 5541 corresponding to Ad6, joined to the expression cassette;<br>
d)	a third adenovirus region from about base pair 5549 to about base pair 28133 corresponding to Ad5 or from about base pair 5542 to about base pair 28156 corresponding to Ad6, joined to the second region;<br>
e)	a fourth adenovirus region from about base pair 30818 to about base pair 33966 corresponding to Ad5 or from about base pair 30789 to about base pair 33784 corresponding to Ad6, joined to the third region;<br>
f)	a fifth adenovirus region from about base pair 33967 to about base pair 35935 corresponding to Ad5 or from about base pair 33785 to about base pair 35759 corresponding to Ad6, joined to the fourth region, and<br>
g)	an optionally present E3 region corresponding to all or part of the E3 region present in Ad5 or Ad6, which may be present for smaller inserts taking into account the overall size of the desired adenovector.<br>
In another embodiment of the present invention the recombinant<br>
adenovirus genome plasmid has the gene expression cassette inserted in the E3 deleted<br>
region. The vector contains an origin of replication, a selectable marker, and the<br>
following: a)	a first adenovirus region from about base pair 1 to about base<br>
pair 450 corresponding to either Ad5 or Ad6;<br>
b)	a second adenovirus region from about base pair 3511 to about<br>
base pair 5548 corresponding to Ad5 or from about base pair 3508 to about base pair<br>
5541 corresponding to Ad6, joined to the expression cassette;<br><br>
c)	a third adenovirus region from about base pair 5549 to about base pair 28133 corresponding to Ad5 or from about base pair 5542 to about base pair 28156 corresponding to Ad6, joined to the second region;<br>
d)	the gene expression cassette in a E3 parallel or E3 anti-parallel orientation joined to the third region;<br>
e)	a fourth adenovirus region from about base pair 30818 to about base pair 33966 corresponding to Ad5 or from about base pair 30789 to about base pair 33784 corresponding to Ad6, joined to the gene expression cassette; and<br>
f)	a fifth adenovirus region from about base pair 33967 to about base pair 35935 corresponding to Ad5 or from about base pair 33785 to about base pair 35759 corresponding to Ad6, joined to the fourth region.<br>
In different embodiments concerning adenovirus regions that are present: (1) the first, second, third, fourth, and fifth region corresponds to Ad5; (2) the first, second, third, fourth, and fifth region corresponds to Ad6; and (3) the first region corresponds to Ad5, the second region corresponds to Ad5, the third region corresponds to Ad6, the fourth region corresponds to Ad6, and the fifth region corresponds to Ad5.<br>
An embodiment of the present invention describes a method of making an adenovector involving a homologous recombination step to produce a adenovirus genome plasmid and an adenovirus rescue step. The homologous recombination step involves the use of a shuttle vector containing a Met-NS3-NS4A-NS4B-NS5A-NS5B expression cassette flanked by adenovirus homology regions. The adenovirus homology regions target the expression cassette into either the El or E3 deleted region.<br>
In an embodiment of the present invenfion concerning the production of an adenovirus genome plasmid, the gene expression cassette is inserted into a vector comprising: a first adenovirus region from about base pair 1 to about base pair 450 corresponding to either Ad5 or Ad6; a second adenovirus region from about base pair 3511 to about base pair 5548 corresponding to Ad5 or from about base pair 3508 to about base pair 5541 corresponding to Ad6, joined to the second region; a third adenovirus region from about base pair 5549 to about base pair 28133 corresponding to Ad5 or from about base pair 5542 to about base pair 28156 corresponding to Ad6, joined to the second region; a fourth adenovirus region from about base pair 30818 to about base pair 33966 corresponding to Ad5 or from about base pair 30789 to about<br><br>
base pair 33784 corresponding to Ad6, joined to the third region; and a fifth adenovirus region from about 33967 to about 35935 corresponding to Ad5 or from about base pair 33785 to about base pair 35759 corresponding to Ad6, joined to the fourth region. The adenovirus genome plasmid should contain an origin of replication and a selectable marker, and may contain all or part of the Ad5 or Ad6 E3 region.<br>
In different embodiments concerning adenovirus regions that are present: (1) the first, second, third, fourth, and fifth region corresponds to Ad5; (2) the first, second, third, fourth, and fifth region corresponds to Ad6; and (3) the first region corresponds to Ad5, the second region corresponds to Ad5, the third region corresponds to Ad6, the fourth region corresponds to Ad6, and the fifth region corresponds to Ad5.<br>
B. Adenovector Rescue<br>
An adenovector can be rescued from a recombinant adenovirus genome plasmid using techniques known in the art or described herein. Examples of techniques for adenovirus rescue well known in the art are provided by Hitt et aL, Methods in Molecular Genetics 7:13-30, 1995, and Danthinne et ai, Gene Therapy 7:1707-1714, 2000.<br>
A preferred method of rescuing an adenovector described herein involves boosting adenoviral replication. Boosting adenoviral replication can be performed, for example, by supplying adenoviral fijnctions such as E2 proteins (polymerase, pre-terminal protein and DNA binding protein) as well as E4 orf6 on a separate plasmid. Example 10 infra, illustrates the boosting of adenoviral replication to rescue an adenovector containing a codon optimized Met-NS3-NS4A-NS4B-NS5A-NS5B expression cassette.<br>
V. PARTIAL-OPITIMIZED HCV ENCODING SEQUENCES<br>
Partial optimization of HCV polyprotein encoding nucleic acid provides for a lesser amount of codons optimized for expression in a human than complete optimization. The overall objective is to provide the benefits of increased expression due to codon optimization, while facilitating the production of an adenovector containing HCV polyprotein encoding nucleic acid having optimized codons.<br><br>
Complete optimization of an HCV polyprotein encoding sequence provides the most frequently observed human codon for each amino acid. Complete optimization can be performed using codon frequency tables well known in the art and using programs such as the BACKTRANSLATE program (Wisconsin Package version 10, Genetics Computer Group, GCG, Madison, Wise).<br>
Partial optimization can be preformed on an entire HCV polyprotein encoding sequence that is present (e.g., NS3-NS5B), or one or more local regions that are present. In different embodiments the GC content for the entire HCV encoded polyprotein that is present is no greater than at least about 65%; and the GC content for one or more local regions is no greater than about 70%.<br>
Local regions are regions present in HCV encoding nucleic acid, and can vary in size. For example, local regions can be about 60, about 70, about 80, about 90 or about 100 nucleotides in length.<br>
Partial optimization can be achieved by initially constructing an HCV encoding polyprotein sequence to be partially optimized based on a naturally ocurring sequence. Alternatively, an optimized HCV encoding sequence can be used as basis of comparison to produce a partial optimized sequence.<br>
VI. HCV COMBINATION TREATMENT<br>
The HCV Met-NS3-NS4A-NS4B-NS5A-NS5B vaccine can be used by itself to treat a patient, can be used in conjunction with other HCV therapeutics, and can be used with agents targeting other types of diseases. Additional therapeutics include additional therapeutic agents to treat HCV and diseases having a high prevalence in HCV infected persons. Agents targeting other types of disease include vaccines directed against HIV and HBV.<br>
Additional therapeutics for treating HCV include vaccines and non-vaccine agents. (Zein, Expert Opin. Investig. Drugs 70:1457-1469, 2001.) Examples of additional HCV vaccines include vaccines designed to elicit an immune response against an HCV core antigen and the HCV El, E2 or p7 region. Vaccine components can be naturally occurring HCV polypeptides, HCV mimotope polypeptides or nucleic acid encoding such polypeptides.<br><br>
HCV mimotope polypeptides contain HCV epitopes, but have a different sequence than a naturally occurring HCV antigen. A HCV mimotope can be fiised to a naturally occurring HCV antigen. References describing techniques for producing mimotopes in general and describing different HCV mimotopes are provided in Felici et al. U.S. Patent No. 5,994,083 and Nicosia et al, International Application Number WO 99/60132.<br>
Vll. PHARMACEUTICAL ADMINISTRATION<br>
HCV vaccines can be formulated and administered to a patient using the guidance provided herein along with techniques well known in the art. Guidelines for pharmaceutical administration in general are provided in, for example. Modern Vaccinology, Ed. Kurstak, Plenum Med. Co. 1994; Remington's Pharmaceutical Sciences 18th Edition, Ed. Gennaro, Mack Publishing, 1990; and Modern Pharmaceutics 2nd Edition, Eds. Banker and Rhodes, Marcel Dekker, Inc., 1990, each of which are hereby incorporated by reference herein.<br>
HCV vaccines can be administered by different routes such intravenous, intraperitoneal, subcutaneous, intramuscular, intradermal, impression through the skin, or nasal. A preferred route is intramuscular.<br>
Intramuscular administration can be preformed using different techniques such as by injection with or without one or more electric pulses. Electric mediated transfer can assist genetic immunization by stimulating both humoral and cellular immune responses.<br>
Vaccine injection can be performed using different techniques, such as by employing a needle or a needless injection system. An example of a needless injection system is a jet injection device. (Donnelly et al., International Publication Number WO 99/52463.)<br>
A. Electrically Mediated Transfer<br>
Electrically mediated transfer or Gene Electro-Transfer (GET) can be performed by delivering suitable electric pulses after nucleic acid injection. (See Mathiesen, International Publication Number WO 98/43702). Plasmid injection and electroporation can be performed using stainless needles.   Needles can be used in<br><br>
couples, triplets or more complex patterns. In one configuration the needles are soldered on a printed circuit board that is a mechanical support and connects the needles to the electrical field generator by means of suitable cables.<br>
The electrical stimulus is given in the form of electrical pulses. Pulses can be of different forms (square, sinusoidal, triangular, exponential decay) and different polarity (monopolar of positive or negative polarity, bipolar). Pulses can be delivered either at constant voltage or constant current modality.<br>
Different patterns of electric treatment can be used to introduce nucleic acid vaccines including HCV and other nucleic acid vaccines into a patient. Possible patterns of electric treatment include the following:<br>
Treatment 1: 10 trains of 1000 square bipolar pulses delivered every other second, pulse length 0.2 msec/phase, frequency 1000 Hz, constant voltage mode, 45 Volts/phase, floating current.<br>
Treatment 2: 2 trains of 100 square bipolar pulses delivered every other second, pulse length 2 msec/phase, frequency 100 Hz, constant current mode, 100 mA/phase, floating voltage.<br>
Treatment 3: 2 trains of bipolar pulses at a pulse length of about 2 msec/phase, for a total length of about 3 seconds, where the actual current going through the tissue is fixed at about 50 mA.<br>
Electric pulses are delivered through an electric field generator. A suitable generator can be composed of three independent hardware elements assembled in a common chassis and driven by a portable PC which runs the driving program. The software manages both basic and accessory functions. The elements of the device are: (1) signal generator driven by a microprocessor, (2) power amplifier and (3) digital oscilloscope.<br>
The signal generator delivers signals having arbitrary frequency and shape in a given range under software control. The same software has an interactive editor for the waveform to be delivered. The generator features a digitally controlled current limiting device (a safety feature to control the maximal current output). The power amplifier can amplify the signal generated up to -f/- 150 V. The oscilloscope is digital and is able to sample both the voltage and the current being delivered by the amplifier.<br><br>
B.	Pharmaceutical Carriers<br>
Pharmaceutically acceptable carriers facilitate storage and administration of a vaccine to a subject. Examples of pharmaceutically acceptable carriers are described herein. Additional pharmaceutical acceptable carriers are well known in the art.<br>
Pharmaceutically acceptable carriers may contain different components such a buffer, normal saline or phosphate buffered saline, sucrose, salts and polysorbate. An example of a pharmaceutically acceptable carrier is follows: 2.5-10 mM TRIS buffer, preferably about 5 mM TRIS buffer; 25-100 mM NaCl, preferably about 75 mM NaCl; 2.5-10% sucrose, preferably about 5% sucrose; 0.01 -2 mM MgCl2; and 0.001%-0.01% polysorbate 80 (plant derived). The pH is preferably from about 7.0-9.0, more preferably about 8.0. A specific example of a carrier contains 5 mM TRIS, 75 mM NaCl, 5% sucrose, 1 mM MgCb, 0.005% polysorbate 80 at pH 8.0.<br>
C.	Dosing Regimes<br>
Suitable dosing regimens can be determined taking into account the efficacy of a particular vaccine and factors such as age, weight, sex and medical condition of a patient; the route of administration; the desired effect; and the number of doses. The efficacy of a particular vaccine depends on different factors such as the ability of a particular vaccine to produce polypeptide that is expressed and processed in a cell and presented in the context of MHC class I and II complexes.<br>
HCV encoding nucleic acid administered to a patient can be part of different types of vectors including viral vectors such as adenovector, and DNA plasmid vaccines. In different embodiments concerning administration of a DNA plasmid, about 0.1 to 10 mg of plasmid is administered to a patient, and about I to 5 mg of plasmid is administered to a patient.    In different embodiments concerning<br>
administration of a viral vector, preferably an adenoviral vector, about 105 to 1101 viral particles are administered to a patient, and about 107 to 1010 viral particles are administered to a patient.<br>
Viral vector vaccines and DNA plasmid vaccines may be administered alone, or may be part of a prime and boost administration regimen. A mixed modality<br><br>
priming and booster inoculation involves either priming with a DNA vaccine and boosting with viral vector vaccine, or priming with a viral vector vaccine and boosting with a DNA vaccine.<br>
Multiple priming, for example, about to 2-4 or more may be used. The length of time between priming and boost may typically vary from about four months to a year, but other time frames may be used. The use of a priming regimen with a DNA vaccine may be preferred in situations where a person has a pre-existing anti-adenovirus immune response.<br>
In an embodiment of the present invention, 1x10 to 1x10 particles and preferably about lx1010 to 1x1011 particles of adenovector is administered directly into muscle tissue. Following initial vaccination a boost is performed with an adenovector or DNA vaccine.<br>
In another embodiment of the present invention initial vaccination is performed with a DNA vaccine directly into muscle tissue. Following initial vaccination a boost is performed with an adenovector or DNA vaccine.<br>
Agents such as interleukin-12, GM-CSF, B7-1, B7-2, IPIO, Mig-1 can be coadministered to boost the immune response. The agents can be coadministered as proteins or through use of nucleic acid vectors.<br>
D. Heterologous Prime-Boost<br>
Heterologous prime-boost is a mixed modality involving the use of one type of viral vector for priming and another type of viral vector for boosting. The heterologous prime-boost can involve related vectors such as vectors based on different adenovirus serotypes and more distantly related viruses such adenovirus and poxvirus. The use of poxvirus and adenovirus vectors to protect mice against malaria is illustrated by Gilbert et aL, Vaccine 20:1039-1045, 2002.<br>
Different embodiments concerning priming and boosting involve the following types of vectors expressing desired antigens such as Met-NS3-NS4A-NS4B-NS5A-NS5B: Ad5 vector followed by Ad6 vector; Ad6 vector followed by Ad5 vector; Ad5 vector followed by poxvirus vector; poxvirus vector followed by Ad5 vector; Ad6 vector followed by poxvirus vector; and poxvirus vector followed by Ad6 vector.<br><br>
The length of time between priming and boosting typically varies from about four months to a year, but other time frames may be used. The minimum time frame should be sufficient to allow for an immunological rest. In an embodiment, this rest is for a period of at least 6 months. Priming may involve multiple priming with one type of vector, such as 2-4 primings.<br>
Expression cassettes present in a poxvirus vector should contain a promoter either native to, or derived from, the poxvirus of interest or another poxvirus member. Different strategies for constructing and employing different types of poxvirus based vectors including those based on vaccinia virus, modified vaccinia virus, avipoxvirus, raccoon poxvirus, modified vaccinia virus Ankara, canarypoxviruses (such as ALVAC), fowlpoxviruses, cowpoxviruses, and NYVAC are well known in the art. (Moss, Current Topics in Microbiology and Immunology 75S:25-38, 1982; Earl et al.. In Current Protocols in Molecular Biology, Ausubel et al. eds., New York: Greene Publishing Associates &amp; Wiley Interscience; 1991:16.16.1-16.16.7, Child et al, Virology 174(2):625-9, 1990; Tartagliae et al., Virology 188:217-232, 1992; U.S. Patent Nos., 4,603,112, 4,722,848, 4,769,330, 5,110,587, 5,174,993, 5,185,146, 5,266,313, 5,505,941, 5,863,542, and 5,942,235.<br>
E. Adjuvants<br>
HCV vaccines can be formulated with an adjuvant. Adjuvants are particularly useful for DNA plasmid vaccines. Examples of adjuvants are alum, AIPO4, alhydrogel, Lipid-A and derivatives or variants thereof, Freund's incomplete<br>
adjuvant, neutral liposomes, liposomes containing the vaccine and cytokines, non-ionic block copolymers, and chemokines.<br>
Non-ionic block polymers containing polyoxyethylene (POE) and polyxylpropylene (POP), such as POE-POP-POE block copolymers may be used as an adjuvant. (Newman et al., Critical Reviews in Therapeutic Drug Carrier Systems 75:89-142, 1998.) The immune response of a nucleic acid can be enhanced using a non-ionic block copolymer combined with an anionic surfactant.<br>
A specific example of an adjuvant formulation is one containing CRL-1005 (CytRx Research Laboratories), DNA, and benzylalkonium chloride (BAK). The formulation can be prepared by adding pure polymer to a cold   (
 <br>
plasmid DNA in PBS using a positive displacement pipette. The solution is then vortexed to solubilize the polymer. After complete solubilization of the polymer a clear solution is obtained at temperatures below the cloud point of the polymer ('-6-7°C). Approximately 4 mM BAK is then added to the DNA/CRL-1005 solution in PBS, by slow addition of a dilute solution of BAK dissolved in PBS. The initial DNA concentration is approximately 6 mg/mL before the addition of polymer and BAK, and the final DNA concentration is about 5 mg/mL. After BAK addition the formulation is vortexed extensively, while the temperature is allowed to increase from - 2°C to above the cloud point. The formulation is then placed on ice to decrease the temperature below the cloud point. Then, the formulation is vortexed while the temperature is allowed to increase from --l°C to above the cloud point. Cooling and mixing while the temperature is allowed to increase from -2°C to above the cloud point is repeated several times, until the particle size of the formulation is about 200-500 nm, as measured by dynamic light scattering. The formulation is then stored on ice until the solution is clear, then placed in storage at -70°C. Before use, the formulation is allowed to thaw at room temperature.<br>
F. Vaccine Storage<br>
Adenovector and DNA vaccines can be stored using different types of buffers. For example, buffer A105 described in Example 9 infra, can be used to for vector storage.<br>
Storage of DNA can be enhanced by removal or chelation of trace metal ions. Reagents such as succinic or malic acid, and chelators can be used to enhance DNA vaccine stability. Examples of chelators include multiple phosphate ligands and EDTA. The inclusion of non-reducing free radical scavengers, such as ethanol or glycerol, can also be useful to prevent damage of DNA plasmid from free radical production. Furthermore, the buffer type, pH, salt concentration, light exposure, as well as the type of sterilization process used to prepare the vials, may be controlled in the formulation to optimize the stability of the DNA vaccine.<br>
Vn. EXAMPLES<br><br>
Examples are provided below to further illustrate different features of the present invention. The examples also illustrate useful methodology for practicing the invention. These examples do not limit the claimed invention.<br>
Example 1: Met-NS3-NS4A-NS4B-NS5A-NS5B Expression Cassettes<br>
Different gene expression cassettes encoding HCV NS3-NS4A-NS4B-NS5A-NS5B were constructed based on a lb subtype HCV BK strain. The encoded sequences had either (1) an active NS5B sequence ("NS"), (2) an inactive NS5B sequence ("NSmut"), (3) a codon optimized sequence with an inactive NS5B sequence ("NSOPTmut"). The expression cassettes also contained a CMV promoter/enhancer and the BGH polyadenylation signal.<br>
The NS nucleotide sequence (SEQ. ID. NO. 5) differs from HCV BK strain GenBank accession number M58335 by 30 out of 5952 nucleotides. The NS amino acid sequence (SEQ. ID. NO. 6) differs from the corresponding lb genotype HCV BK strain by 7 out of 1984 amino acids. To allow for initiation of translation an ATG codon is present at the 5' end of the NS sequence. A TGA termination sequence is present at the 3' end of the NS sequence.<br>
The NSmut nucleotide sequence (SEQ. ID, NO. 2, Figure 2), is similar to the NS sequence. The differences between NSmut and NS include NSmut having an altered NS5B catalytic site; an optimal ribosome binding site at the 5' end; and a TAAA termination sequence at the 3' end. The alterations in NS5B comprise bases 5138 to 5146, which encode amino acids 1711 to 1713. The alterations result in a change of amino acids GlyAspAsp into AlaAlaGly and creates an inactive form of the NS5B RNA-dependent RNA-polymerase NS5B.<br>
The NSOPTmut sequence (SEQ. ID. NO. 3, Figure 3) was designed based on the amino acid sequence encoded by NSmut. The NSmut amino acid sequence was back translated into a nucleotide sequence with the GCG (Wisconsin Package version 10, Genetics Computer Group, GCG, Madison, Wise.) BACKTRANSLATE program. To generate a NSOPTmut nucleotide sequence where each amino acid is coded for by the corresponding most frequently observed human codon, the program was run choosing as parameter the generation of the most probable nucleotide sequence and specifying the codon frequency table of highly expressed<br><br>
human genes (human_high.cod) available within the GCG Package as translation scheme.<br>
Example 2: Generation pVlJns plasmid with NS. NSmut or NSOPTmut Sequences pVlJNns plasmids containing either the NS sequence, NSmut sequence or NSOPTmut sequences were generated and characterised as follows:<br>
pVlJns Plasmid with the NS Sequence<br>
The coding region Met-NS3-NS4A-NS4B-NS5A and the coding region Met-NS3-NS4A-NS4B-NS5A-NS5B from a HCV BK type strain (Tomei et ai, J. Virol. 67:4017-4026, 1993) were cloned into pcDNA3 plasmid (Invitrogen), generating pcD3-5a and pcD3-5b vectors, respectively. PcD3-5A was digested with Hind III, blunt-ended with Klenow fill-in and subsequently digested with Xba I, to generate a fragment corresponding to the coding region of Met-NS3-NS4A-NS4B-NS5A. The fragment was cloned into pVlJns-poly, digested with Bgl II blunt-ended with Klenow fill-in and subsequently digested with Xba I, generating pVl JnsNS3-5A.<br>
pVlJns-poly is a derivative of pVlJnsA plasmid (Montgomery et al., DNA and Cell Biol. 72:777-783, 1993), modified by insertion of a polylinker containing recognition sites for Xbal, Pmel, Pad into the unique Bglll and NotI restriction sites. The pVlJns plasmid with the NS sequence (pVlJnsNS3-5B) was obtained by homologous recombination into the bacterial strain BJ5183, co-transforming pVlJNS3-5A linearized with Xbal and NotI digestion and a PCR fragment containing approximately 200 bp of NS5A, NS5B coding sequence and approximately 60 bp of the BGH polyadenylation signal. The resulting plasmid represents pVlJns-NS.<br>
pVlJns-NS can be summarized as follows:<br>
Bases	1 to 1881 ofpVlJnsA<br>
an additional	AGCTT<br>
then the	Met-NS3-NS5B sequence (SEQ. ID. NO. 5)<br>
then the	wt TGA stop<br>
an additional	TCTAGAGCGTTTAAACCCTTAATTAAGG    (SEQ. ID. NO.<br>
14)<br>
Bases	1912 to 4909 of pVl JnsA<br><br>
pVlJns Plasmid-with the NSmut Sequence<br>
The VlJnsNS3-5A plasmid was modified at the 5' of the NS3 coding sequence by addition of a full Kozak sequence. The plasmid (VlJNS3-5Akozak) was obtained by homologous recombination into the bacterial strain BJ5183, co-transforming VIJNS3-5A linearized by AfII digestion and a PCR fragment containing the proximal part of Intron A, the restriction site Bglll, a full Kozak translation initiation sequence and part of the NS3 coding sequence.<br>
The resulting plasmid (VlJNS3-5Akozak) was linearized with Xba I digestion and co-transformed into the bacterial strain BJ5183 with a PCR fragment, containing approximately 200 bp of NS5A, the NS5B mutated sequence, the strong translation termination TAAA and approximately 60 bp of the BGH polyadenylation signal. The PCR fragment was obtained by assembling two 22bp-overlapping fragments where mutations were introduced by the oligonucleotides used for their amplification. The resulting plasmid represents pVlJns-NSmut.<br>
pVlJns-NSmut can be summarized as follows:<br>
Bases	1 to 1882 of pVlJnsA<br>
then the	kozak Met-NS3-NS5B(mut) TAAA sequence (SEQ. ID. NO. 2)<br>
an additional   TCTAGA<br>
Bases	1925 to 4909 of pVlJnsA<br>
pVlJns Plasmid with the NSOPTmut Sequence<br>
The human codon-optimized synthetic gene (NSOPTmut) with mutated NS5B to abrogate enzymatic activity, full Kozak translation initiation sequence and a strong translation termination was digested with BamHI and Sail restriction sites present at the 5' and 3' end of the gene. The gene was then cloned into the Bglll and Sail restriction sites present in the polylinker of pVlJnsA plasmid, generating pVlJns-NSOPTmut.<br>
pVlJns-NSOPTmut can be summarized as follows:<br>
Bases	1 to 1881 of pVlJnsA<br>
an additional   C<br>
then	kozak Met-NS3-NS5B(optmut) TAAA sequence (SEQ. ID. NO. 3)<br><br>
an additional   TTTAAATGTTTAAAC (SEQ. ID. NO. 15)<br>
Bases	1905 to 4909 of pVlJnsA<br>
Plasmids Characterization<br>
Expression of HCV NS proteins was tested by transfection of HEK 293 cells, grown in 10% FCS/DMEM supplemented by L-glutamine (final 4 mM). Twenty-four hours before transfection, cells were plated in 6-well 35 mm diameter, to reach 90-95% confluence on the day of transfection. Forty nanograms of plasmid DNA (previously assessed as a non-saturating DNA amount) were co-transfected with 100 ng of pRSV-Luc plasmid containing the luciferase reporter gene under the control of Rous sarcoma virus promoter, using the LIPOFECTAMINE 2000 reagent. Cells were kept in a CO2 incubator for 48 hours at 37 °C.<br>
Cell extracts were prepared in 1% Triton/TEN buffer. The extracts were normalized for Luciferase activity, and run in serial dilution on 10% SDS-acrylamide gel. Proteins were transferred on nitrocellulose and assayed with antibodies directed against NS3, NS5A and NS5B to assess strength of expression and correct proteolytic cleavage. Mock-transfected cells were used as a negative control. Results from representative experiments testing pVlJnsNS, pVlJnsNSmut and pVlJnsNSOPTmut are shown in Figure 12.<br>
Example 3: Mice Immunization with Plasmid DNA Vectors<br>
The DNA plasmids pV 1 Jns-NS, p V1 Jns-NSmut and p V1 Jns-NSOPTmut were injected in different mice strains to evaluate their potential to elicit anti-HCV immune responses. Two different strains (Balb/C and C57Black6, N=9-10) were injected intramuscularly with 25 or 50 Dg of DNA followed by electrical pluses. Each animal received two doses at three weeks interval.<br>
Humoral immune response elicited in C57Black6 mice against the NS3 protein was measured in post dose two sera by ELISA on bacterially expressed NS3 protease domain. Antibodies specific for the tested antigen were detected in animals immunized with all three vectors with geometric mean titers (GMT) ranging from 94000 to 133000 (Tables 1-3).<br><br>
TABLE 1: PVIJNS-NS<br><br>
	GMT<br>
Mic e n.	1	2	3	4	5	6	7	8	9	<br>
Tite r	10546 6	89198 0	7879 9	3949 6	54354 2	18213 9	3235 1	9502 8	6780 0	94553<br>
TABLE 2: PVIJNS-NSMUT<br><br>
	GM T<br>
Mic en.	11	12	13	14	15	16	17	18	19	20	<br>
Tite r	20298 1	5567 0	1307 86	4974 8	1767 2	1749 58	4430 4	3733 7	78182	1936 95	7508 3<br>
TABLE 3: PVIJNS-NSOPTMUT<br><br>
	GMT<br>
Mic en.	21	22	23	24	25	26	27	28	29	30	<br>
Tite r	31034 9	4364 5	6349 6	8217 4	6307 78	2972 59	6686 1	146735	1735 06	7773 2	1331 65<br>
A T cell response was measured in C57Black6 mice immunized with two intramuscular injections at three weeks interval with 25 Dg of plasmid DNA. Quantitative ELIspot assay was performed to determine the number of IFND secreting<br><br>
T cells in response to five pools of 20mer peptides overlapping by ten residues encompassing the NS3-NS5B sequence. Specific CD8+ response was analyzed by the same assay using a 20mer peptide encompassing a CD8+ epitope for C57Black6 mice (pepl480).<br>
Cells secreting IFND in an antigen specific-manner were detected using a standard ELIspot assay. T cell response in C57Black6 mice immunized with two intramuscular injections at three weeks interval with 50 Dg of plasmid DNA, was analyzed by the same ELIspot assay measuring the number of IFND secreting T cells in response to five pools of 20mer peptides overlapping by ten residues encompassing the NS3-NS5B sequence.<br>
Spleen cells were prepared from immunized mice and re-suspended in RIO medium (RPMI1640 supplemented with 10% PCS, 2 mM L-Glutamine, 50 U/ml-50^g/ml Penicillin/Streptomycin, 10 mM Hepes, 50 μM 2-mercapto-ethanol). Multiscreen 96-well Filtration Plates (Millipore, Cat. No. MAIPS4510, Millipore Corporation, 80 Ashby Road Bedford, MA) were coated with purified rat anti-mouse INFO antibody (PharMingen, Cat. No. 1818ID, PharmiMingen, 10975 Torreyana Road, San Diego, California 92121-1111 USA). After overnight incubation, plates were washed with PBS lX/0.005% Tween and blocked with 250 ^il/well of RIO medium.<br>
Splenocytes from immunized mice were prepared and incubated for twenty-four hours in the presence or absence of 10μM peptide at a density of 2.5 X 105/well or 5 X lOVwell. After extensive washing (PBS lX/0.005% Tween), biotinylated rat anti-mouse IFND antibody (PharMingen, Cat. No. 18112D, PharMingen, 10975 Torreyana Road, San Diego, California 92121-1111 USA) was added and incubated overnight at 4° C. For development, streptavldin-AKP (PharMingen, Cat. No. 13043E, PharMingen, 10975 Torreyana Road, San Diego, California 92121-1111 USA) and 1-Step™ NBT-BCIP development solution (Pierce, Cat. No. 34042, Pierce, P.O. Box 117, Rockford, IL 61105 USA) were added.<br>
Pools of 20mer overlapping peptides encompassing the entire sequence of the HCV BK strain NS3 to NS5B were used to reveal HCV-specific IFND-secreting T cells. Similarly a single 20mer peptide encompassing a CD8+ epitope for C57Black6 mice was used to detect CD8 response. Representative data from groups of C57Black6<br><br>
and Balb/C mice (N=9-10) immunized with two injections of 25 or 50 Dg of plasmid vectors pVlJns-NS, pVlJns-NSmut and pVlJns-NSOPTmut are shown in Figures 13A andl3B.<br>
Example 4: Immunization of Rhesus Macaques<br>
Rhesus macaques (N=3) were immunized by intramuscular injection with 5mg of plasmid pVlJns-NSOPTmut in 7.5mg/ml CRL1005, Benzalkonium chloride 0.6 mM. Each animal received two doses in the deltoid muscle at 0, and 4 weeks.<br>
CMI was measured at different time points by IFN-D ELISPOT. This assay measures HCV antigen-specific CD8+ and CD44- T lymphocyte responses, and can be used for a variety of mammals, such as humans, rhesus monkeys, mice, and rats.<br>
The use of a specific peptide or a pool of peptides can simplify antigen presentation in CTL cytotoxicity assays, interferon-gamma ELISPOT assays and interferon-gamma intracellular staining assays. Peptides based on the amino acid sequence of various HCV proteins (core, E2, NS3, NS4A, NS4B, NS5A, NS5B) were prepared for use in these assays to measure immune responses in HCV DNA and adenovirus vector vaccinated rhesus monkeys, as well as in HCV-infected humans. The individual peptides are overlapping 20-mers, offset by 10 amino acids. Large pools of peptides can be used to detect an overall response to HCV proteins while smaller pools and individual peptides may be used to define the epitope specificity of a response.<br>
IFND ELISPOT<br>
The IFND-ELISPOT assay provides a quantitative determination of HCV-specific T lymphocyte responses. PBMC are serially diluted and placed in microplate wells coated with anti-rhesus IFN-D antibody (MD-1 U-Cytech). They are cultured with a HCV peptide pool for 20 hours, resulting in the restimulation of the precursor cells and secretion of IFN-D. The cells are washed away, leaving the secreted IFN bound to the antibody-coated wells in concentrated areas where the cells were sitting. The captured IFN is detected with biotinylated anti-rhesus IFN antibody (detector Ab U-Cytech) followed by alkaline phosphatase-conjugated streptavidin (Pharmingen 13043E). The addition of insoluble alkaline phosphatase substrate results<br><br>
in dark spots in the wells at the sites where the cells were located, leaving one spot for each T cell that secreted IFN-D.<br>
The number of spots per well is directly related to the precursor frequency of antigen-specific T cells. Gamma interferon was selected as the cytokine visualized in this assay (using species specific anti-gamma interferon monoclonal antibodies) because it is the most common, and one of the most abundant cytokines synthesized and secreted by activated T lymphocytes. For this assay, the number of spot forming cells (SFC) per million PBMCs is determined for samples in the presence and absence (media control) of peptide antigens. Data from Rhesus macaques on PBMC from post dose two material are shown in Table 4.<br>
Table 4<br><br>
INFyELISPOT on PBMC from Rhesus monkeys immunized with two injections of 5 mg DNA/dose in OPTIVAX/BAK of plasmid pVlJns-NSOPTmut. Data are expressed<br>
asSFC7 106pBMC.<br>
Example 5: Construction of Ad6 Pre-Adenovirus Plasmids<br>
Ad6 pre-adenovirus plasmids were obtained as follows:<br>
Construction of pAd6 El-ES-- Pre-adenovirus Plasmid<br><br>
An Ad6 based pre-adenovirus plasmid which can be used to generate first generation Ad6 vectors was constructed either taking advantage of the extensive sequence identity (approx. 98%) between Ad5 and Ad6 or containing only Ad6 regions. Homologous recombination was used to clone wtAd6 sequences into a bacterial plasmid.<br>
A general strategy used to recover pAd6El-E3+ as a bacterial plasmid containing Ad5 and Ad6 regions is illustrated in Figure 10. Cotransformation of BJ 5183 bacteria with purified wt Ad6 viral DNA and a second DNA fragment termed the Ad5 ITR cassette resulted in the circularization of the viral genome by homologous recombination. The ITR cassette contains sequences from the right (bp 33798 to 35935) and left (bp 1 to 341 and bp 3525 to 5767) end of the Ad5 genome separated by plasmid sequences containing a bacterial origin of replication and an ampicillin resistance gene. The ITR cassette contains a deletion of El sequences from Ad5 342 to 3524. The Ad5 sequences in the ITR cassette provide regions of homology with the purified Ad6 viral DNA in which recombination can occur.<br>
Potential clones were screened by restriction analysis and one clone was selected as pAd6El-E3+. This clone was then sequenced in it entirety. pAd6El-E3+ contains Ad5 sequences from bp 1 to 341 and from bp 3525 to 5548, Ad6 bp 5542 to 33784, and Ad5 bp 33967 to 35935 (bp numbers refer to the wt sequence for both Ad5 and Ad6). pAd6El-E3+ contains the coding sequences for all Ad6 virion structural proteins which constitute its serotype specificity.<br>
A general strategy used to recover pAd6El-E3+ as a bacterial plasmid containing Ad6 regions is illustrated in Figure 11. Cotransformation of BJ 5183 bacteria with purified wt Ad6 viral DNA and a second DNA fragment termed the Ad6 ITR cassette resulted in the circularization of the viral genome by homologous recombination. The ITR cassette contains sequences from the right (bp 35460 to 35759) and left (bp 1 to 450 and bp 3508 to 3807) end of the Ad6 genome separated by plasmid sequences containing a bacterial origin of replication and an ampicillin resistance gene. These three segments were generated by PCR and cloned sequentially into pNEB193, generating pNEBAd6-3 (the ITR cassette). The ITR cassette contains a deletion of El sequences from Ad5 451 to 3507.   The Ad6 sequences in the ITR cassette provide<br><br>
regions of homology with the purified Ad6 viral DNA in which recombination can occur.<br>
Construction of pAd6 E1-E3- pre-adenovirus plasmids<br>
Ad6 based vectors containing A5 regions and deleted in the E3 region were constructed starting with pAd6El-E3+ containing Ad5 regions. A 5322 bp subfragment of pAd6El-E3+ containing the E3 region (Ad6 bp 25871 to 31192) was subcloned into pABS.3 generating pABSAd6E3. Three E3 deletions were then made in this plasmid generating three new plasmids pABSAd6E3(1.8Kb) (deleted for Ad6 bp 28602 to 30440), pABSAd6E3(2.3Kb) (deleted for Ad6 bp 28157 to 30437) and pABSAd6E3(2.6Kb) (deleted for Ad6 bp 28157 to 30788). Bacterial recombination was then used to substitute the three E3 deletions back into pAd6El-E3+ generating the Ad6 genome plasmids pAd6El-E3-1.8Kb, pAd6El-E3-2.3Kb and pAd6El-E3-2.6Kb.<br>
Example 6: Generation of Ad5 Genome Plasmid with the NS Sequence<br>
A pcDNA3 plasmid (Invitrogen) containing the coding region NS3-NS4A-NS4B-NS5A was digested with Xmnl and Nrul restriction sites and the DNA fragment containing the CMV promoter, the NS3-NS4A-NS4B-NS5A coding sequence and the Bovine Growth Hormone (BGH) polyadenylation signal was cloned into the unique EcorW restriction site of the shuttle vector pDelElSpa, generating the Sva3-5A vector.<br>
A pcDNA3 plasmid containing the coding region NS3-NS4A-NS4B-NS5A-NS5B was digested with Xmnl and Ecorl (partial digestion), and the DNA fragment containing part of NS5A, NS5B gene and the BGH polyadenylation signal was cloned into the Sva3-5A vector, digested Ecorl and Bglll blunted with Klenow, generating the Sva3-5B vector.<br>
The Sva3-5B vector was finally digested Sspl and 5^/11071 restriction sites and the DNA fragment containing the expression cassette (CMV promoter, NS3-NS4A-NS4B-NS5A-NS5B coding sequence and the BGH polyadenylation signal) flanked by adenovirus sequences was co-transformed with pAd5HVO (E1-,E3-) Clal<br><br>
linearized genome plasmid into the bacterial strain BJ5183, to generate pAd5HV0NS, pAd5HV0 contains Ad5 bp 1 to 341, bp 3525 to 28133 and bp 30818 to 35935.<br>
Example 7: Generation of Adenovirus Genome Plasmids with the NSmut Sequence<br>
Adenovirus genome plasmids containing an NS-mut sequence were generated in an Ad5 or Ad6 background. The Ad6 background contained Ad5 regions at bases 1 to 450, 3511 to 5548 and 33967 to 35935.<br>
pVlJNS3-5Akozak was digested with Bglll and Xbal restriction enzymes and the DNA fragment containing the Kozak sequence and the sequence coding NS3-NS4A-NS4B-NS5A was cloned into a Bglll and Xbal digested polypMRKpdelEl shuttle vector. The resulting vector was designated shNS3-5Akozak.<br>
PolypMRKpdelEl is a derivative of RKpdelEl(Pac/pIX/pack450) + CMVmin+BGHpA(str.) modified by the insertion of a polylinker containing recognition sites for Bglll, Pmel, Swal, Xbal, Sail, into the unique Bglll restriction site present downstream the CMV promoter. MRKpdelEl(Pac/pIX/pack450) + CMVmin + BGHpA(str.) contains Ad5 sequences from bp 1 to 5792 with a deletion of El sequences from bp 451 to 3510. The human CMV promoter and BGH polyadenylation signal were inserted into the El deletion in an El parallel orientation with a unique Bglll site separating them.<br>
The NS5B fragment, mutated to abrogate enzymatic activity and with a strong translation termination at the 3' end, was obtained by assembly PCR and inserted into the shNS3-5Akozak vector via homologous recombination, generating polypMRXpdelElNSmut. In polypMRKpdelEl NSmut the NS-mut coding sequence is under the control of CMV promoter and the BGH polyadenylation signal is present downstream.<br>
The gene expression cassette and the flanking regions which contain adenovirus sequences allowing homologous recombination were excised by digestion with Pad and BstWOll restriction enzymes and co-transformed with either pAd5HV0 (E1-,E3-) or pAd6El-E3-2.6Kb Clal linearized genome plasmids into the bacterial strain BJ5183, to generate pAd5HV0NSmut and pAd6El-,E3-NSmut, respectively.<br>
pAd6El-E3-2.6Kb contains Ad5 bp 1 to 341 and from bp 3525 to 5548, Ad6 bp 5542 to 28157 and from bp 30788 to 33784, and Ad5 bp 33967 to 35935 (bp<br><br>
numbers refer to the wt sequence for both Ad5 and Ad6). In both plasmids the viral ITR's are joined by plasmid sequences that contain the bacterial origin of replication and an ampicillin resistance gene.<br>
Example 8: Generation of Adenovirus Genome Plasmids with the NSOPTmut<br>
The human codon-optimized synthetic gene (NSOPTmut) provided by<br>
SEQ. ID. NO. 3 cloned into a pCRBlunt vector (Invitrogen) was digested with BamHX<br>
and SalV restriction enzymes and cloned into BglU and SalV restriction sites present in<br>
the       shuttle       vector       polypMRKpdelEl.	The       resuUing       clone<br>
(polypMRKpdelElNSOPTmut) was digested with Pad and 5^/11071 restriction enzymes and co-transformed with either pAdSHVO (E1-,E3-) or pAd6El-E3-2.6Kb Cla\ linearized genome plasmids, into the bacterial strain BJ5183, to generate pAdSHVONSOPTmut and pAd6El-,E3-NSOPTmut, respectively.<br>
Example 9: Rescue and Amplification of Adenovirus Vectors<br>
Adenovectors were rescued in Per.6 cells. Per.C6 were grown in 10% PCS / DMEM supplemented by L-glutamine (final 4mM), penicillin/streptomycin (final 100 lU/ml) and 10 mM MgCl2.  After infection, cells were kept in the same medium<br>
supplemented by 5% horse serum (HS). For viral rescue, 2.5 X 106 Per.C6 were plated in 6 cm 0 Petri dishes.<br>
Twenty-four hours after plating, cells were transfected by calcium phosphate method with 10 Dg of the Pac I linearized adenoviral DNA. The DNA precipitate was left on the cells for 4 hours. The medium was removed and 5% HS/DMEM was added.<br>
Cells were kept in a CO2 incubator until a cytopathic effect was visible (1 week). Cells and supernatant were recovered and subjected to 3X freeze/thawing cycles (liquid nitrogen / water bath at 37oC). The lysate was centrifuged at 3000 rpm at - 4°C for 20 minutes and the recovered supernatant (corresponding to a cell lysate containing virus passed on cells only once; PI) was used, in the amount of 1 ml/ dish, to infect 80-90% confluent Per.C6 in 10 cm 0 Petri dishes. The infected cells were incubated until a cytopathic effect was visible, cells and supernatant recovered and the lysate prepared as described above (P2).<br><br>
P2 lysate (4 ml) were used to infect 2 X 15 cm 0 Petri dishes. The lysate recovered from this infection (P3) was kept in aliquots at -80°C as a stock of virus to be used as starting point for big viral preparations. In this case, 1 ml of the stock was enough to infect 2 X 15 cm 0 Petri dishes and resulting lysate (P4) was used for the infection of the Petri dishes devoted to the large scale infection.<br>
Further amplification was obtained from the P4 lysate which was diluted in medium without PCS and used to infect 30 X 15 cm 0 Petri dishes (with Per.C6 80%-90% confluent) in the amount of 10 ml/dish. Cells were incubated 1 hour in the CO2 incubator, mixing gently every 20 minutes. 12 ml / dish of 5% HS / DMEM was added and cells were incubated until a cytopathic effect was visible (about 48 hours).<br>
Cells and supernatant were collected and centrifuged at 2K rpm for 20 minutes at 4°C. The pellet was resuspended in 15 ml of 0.1 M Tris pH=8.0. Cells were lysed by 3X freeze/thawing cycles (liquid nitrogen / water bath at 37oC). 150μl of 2 M MgCl2 and 75 ^ll of DNAse (10 mg of bovine pancreatic deoxyribonuclease I in 10 ml<br>
of 20 mM Tris-HCl pH= 7.4, 50 mM NaCl, 1 mM dithiothreitol, 0.1 mg/ml bovine serum albumin, 50% glycerol) were added. After a 1 hour incubation at 37oC in a water bath (vortex every 15 minutes) the lysate was centrifuged at 4K rpm for 15 minutes at 4oC. The recovered supernatant was ready to be applied on CsCl gradient.<br>
The CsCl gradients were prepared in S W40 ultra-clear tubes as follows: 0.5mlofl.5dCsCl 3mlofl.35dCsCl 3mlofl.25dCsCl 5-ml/ tube of viral supernatant was applied.<br>
If necessary, the tubes were topped up with 0.1 M tris-Cl pH=8.0. Tubes were centrifuged at 35K rpm for 1 hour at -10oC with rotor SW40. The viral bands (located at the 1.25/1.35 interface) were collected using a syringe.<br>
The virus was transferred into a new SW40 ultraclear tube and 1.35d CsCl was added to top the tube up. After centrifugation at 35K rpm for 24 hours at 10oC in the rotor SW40, the virus was collected in the smallest possible volume and dialyzed extensively against buffer A105 (5 mM Tris, 5% sucrose, 75 mM NaCl, 1 mM MgCl2, 0.005% polysorbate 80 pH=8.0).  After dialysis, glycerol was added to final<br>
10% and the virus was stored in aliquots at - 80oC.<br><br>
Example 10: Enhanced Adenovector Rescue<br>
First generation Ad5 and Ad6 vectors carrying HCV NSOPTmut transgene were found to be difficult to rescue. A possible block in the rescue process might be attributed to an inefficient replication of plasmid DNA that is a sub-optimal template for the replication machinery of adenovirus. The absence of the terminal protein linked to the 5'ends of the DNA (normally present in the viral DNA), associated with the very high G-C content of the transgene inserted in the El region of the vector, may be causing a substantial reduction in replication rate of the plasmid-derived adenovirus.<br>
To set up a more efficient and reproducible procedure for rescuing Ad vectors, an expression vector (pE2; Figure 19) containing all E2 proteins (polymerase, pre-terminal protein and DNA binding protein) as well as E4 orf6 under the control of tet-inducible promoter was employed. The transfection of pE2 in combination with a normal preadeno plasmid in PerC6 and in 293 leads to a strong increase of Ad DNA replication and to a more efficient production of complete infectious adenovirus particles.<br>
Plasmid Construction<br>
pE2 is based on the cloning vector pBI (CLONTECH) with the addition of two elements to allow episomal replication and selection in cell culture: (1) the EBV-OriP (EBV [nt] 7421-8042) region permitting plasmid replication in synchrony with the cell cycle when EBNA-1 is expressed and (2) the hygromycin-B phosphotransferase (HPH)-resistance gene allowing a positive selection of transformed cells. The two transcriptional units for the adenoviral genes E2 a and b and E4-Orf6 were constructed and assembled in pE2 as described below.<br>
The Ad5-Polymerase Clal/Sphl fragment and the Ad5-pTP Jcc65/EcoRC fragment were obtained from pVac-Pol and pVac-pTP (Stunnemberg et al NAR 76:2431-2444, 1988). Both fragments were filled with Klenow and cloned into the 5a//(filled) and EcoRVsites of pBI, respectively obtaining pBI-Pol/pTP.<br>
EBV-OriP element from pCEP4 (Invitrogen) was first inserted within two chicken D-globin insulator dimers by cloning it into BamHI site of pJC13-l<br><br>
(Chung et ai. Cell 74(3);5055AA, 1993). HS4-0riP fragment from pJC13-0riP was then cloned inside pSAlmv (a plasmid containing tk-Hygro-B resistance gene expression cassette as well as Ad5 replication origin), the ITR's arranged as head-to-tail junction, obtained by PCR from pFG140 (Graham, EMBOJ. 5:2917-2922, 1984) using the following primers: 5'-TCGAATCGATACGCGAACCTACGC-3' (SEQ. ID. NO. 16) and 5'-TCGACGTGTCGACTTCGAAGCGCACACCAAAAACGTC-3' (SEQ. ID. NO. 17), thus generating pMVHS40rip. A DNA fragment from pMVHS40rip, containing the insulated OriP, Ad5 ITR junction and tk-HygroB cassette, was then inserted into pBI-Pol/pTP vector restricted Asel/AatII generating pBI-Pol/pTPHS4 .<br>
To construct the second transcriptional unit expressing Ad5-Orf6 as well as Ad5-DBP, E4orf6 (Ad 5 [nt] 33193-34077) obtained by PCR was first inserted into pBI vector, generating pBI-0rf6. Subsequently, DBP coding DNA sequence (Ad 5 [nt] 22443-24032) was inserted into pBI-Orf6 obtaining the second bi-directional Tet-reguiated expression vector (pBI-DBP/E4orf6). The original polyA signals present in pBI were substituted with BGH and SV40 polyA.<br>
pBI-DBP/E4orf6 was then modified by inserting a DNA fragment containing the Adeno5-ITRs arranged in head-to-tail junction plus the hygromicin B resistance gene obtained from plasmid pSA-lmv. The new plasmid pBI-DBP/E4orf6shuttle was then used as donor plasmid to insert the second tet-regulated transcriptional unit into pBI-Pol/pTPHS4 by homologous recombination using E, coli strain BJ5183 obtaining pE2.<br>
Cell lines, Transfections and Virus Amplification<br>
PerC6 cells were cultured in Dulbecco's modified Eagle's Medium (DMEM) plus 10% fetal bovine serum (FBS), 10 mM MgCl2, penicillin (100 U/ml), streptomycin (100 g/ml) and 2 mM glutamine.<br>
All transient transfections were performed using Lipofectamine2000 (Invitrogen) as described by the manufacturer. 90% confluent PERC.6'TM planted in 6-cm plates were transfected with 3.5 Dg of Ad5/6NS0PTmut pre-adeno plasmids, digested with Pad, alone or in combination with 5 Dg pE2 plus 1 Dg pUHD52.1. pUHD52.1 is the expression vector for the reverse tet transactivator 2 (rtTA2) (Urlinger et ah, Proa NatL Acad ScL U.S.A. 97(14):7963-196i, 2000). Upon transfection, cells<br><br>
were cultivated in the presence of 1 Dg/ml of doxycycline to activate pE2 expression. 7 days post-transfection cells were harvested and cell lysate was obtained by three cycles of freeze-thaw. Two ml of cell lysate were used to infect a second 6-cm dish of PerC6. Infected cells were cultivated until a full CPE was observed then harvested. The virus was serially passaged five times as described above, then purified on CsCl gradient. The DNA structure of the purified virus was controlled by endonuclease digestion and agarose gel electrophoresis analysis and compared to the original pre-adeno plasmid restriction pattern.<br>
Example 11: Partial Optimizeation of HCV Polvprotein Encoding Nucleic acid<br>
Partial optimization of HCV polyprotein encoding nucleic acid was performed to facilitate the production of adenovectors containing codons optimized for expression in a human host. The overall objective was to provide for increased expression due to codon optimization, while facilitating the production of an adenovector encoding HCV polyprotein.<br>
Several difficulties were encountered in producing an adenovector encoding HCV polyprotein with codons optimized for expression in a human host. An adenovector containing an optimized sequence (SEQ. ID. NO. 3) was found to be more difficult to synthesize and rescue than an adenovector containing a non-optimized sequence (SEQ. ID. NO. 2).<br>
The difficulties in producing an adenovector containing SEQ. ID. NO. 3 were attributed to a high GC content. A particularly problemetic region was the region at about position 3900 of NSOPTmut (SEQ. ID. NO. 3).<br>
Alternative versions of optimized HCV encoding nucleic acid sequence were designed to facilitate its use in an adenovector. The alternative versions, compared to NSOPTmut, were designed to have a lower overall GC content, to reduce/avoid the presence of potentially problematic motifis of consecutive G's or C's, while maintaining a high level of codon optimization to allow improved expression of the encoded polyprotein and the individual cleavage products.<br>
A starting point for the generation of a suboptimally codon-optimized sequence is the coding region of the NSOPTmut nucleotide sequence (bases 7 to 5961 of SEQ. ID. NO. 3). Values for codon usage frequencies (normalized to a total of 1.0<br><br>
for each amino acid) were taken from the file human_high.cod available in the Wisconsin Package Version 10.3 (Accelrys Inc., a wholly owned subsidiary of Pharmacopeia, Inc).<br>
To reduce the local and overall GC content a table defining preferred codon substitutions for each amino acid was manually generated. For each amino acid the codon having 1) a lower GC content as compared to the most frequent codon and 2) a relativly high observed codon usage frequency (as defined in human_high.cod) was choosen as the replacement codon. For example for Arg the codon with the highest frequency is CGC. Out of the other five alternative codons encoding Arg (CGG, AGG, AGA, CGT, CGA) three (AGG, CGT, CGA) reduce the GC content by 1 base, one (AGA) by two bases and one (CGG) by 0 bases. Since the AGA codon is listed in human_high.cod as having a relatively low usage frequency (0.1), the codon substituting CGC was therefore choosen to be AGG with a relative frequency of 0.18. Similar criteria were applied in order to establish codon replacements for the other amino acids resulting in the list shown in Table 5. Parameters applied in the following optimization procedure were determined empirically such that the resulting sequence maintained a considerably improved codon usage (for each amino acid) and the GC content (overall and in form of local stretches of consecutive G's and/or C's) was decreased.<br>
Two examples of partial optimized HCV encoding sequences are provided by SEQ. ID. NO. 10 and SEQ. ID. NO. 11. SEQ. ID. NO. 10 provides a HCV encoding sequence that is partially optimized throughout. SEQ. ID. NO. 11 provides an HCV encoding sequence fully optimized for codon usage with the exception of a region that was partially optimized.<br>
Codon optimization was performed using the following procedure:<br>
Step 1) The coding region of the input fully optimized NSOPTmut sequence was analyzed using a sliding window of 3 codons (9 bases) shifting the window by one codon after each cycle. Whenever a stretch containing 5 or more consecutive C's and/or G's was detected in the window the following replacement rule was applied: Let N indicate the number of codon replacements previously performed. If N is odd replace the middle codon in the window with the codon specified in Table 5, if N is even replace the third terminal codon in the window with the codon specified in a<br><br>
codon optimization table such as human_high.cod. If Leu or Val is present at the second or third codon do not apply any replacement in order not to introduce Leu or Val codons with very low relative codon usage frequency (see, for example, human_high.cod). In the following cycle analysis of the shifted window was then applied to a sequence containing the replacements of the previous cycle.<br>
The alternating replacement of the middle and terminal codon in the 3 codon window was found empirically to give a more satisfying overall maintenance of optimized codon usage while also reducing GC content (as judged from the final sequence after the procedure). In general, however, the precise replacement strategy depends on the amino acid sequence encoded by the nucelotide sequence under analysis and will have to be determined empirically.<br>
Step 2) The sequence containing all the codon replacements performed during step 1) was then subjected to an additional analysis using a sliding window of 21 codons (63 bases) in length: according to an adjustable parameter the overall GC content in the window was determined. If the GC content in the window was higher than 70% the following codon replacement strategy was applied: In the window replace the codons for the amino acids Asn, Asp, Cys, Glu, His, He, Lys, Phe, Tyr by the codons given in Table 5. Restriction of the replacement to this set of amino acids was motivated by the fact that a) the replacement codon still has an accetably high frequency of usage in human_high.cod and b) the average overall human codon usage in CUTG for the replacement codon is nearly as high as the most frequent codon. In the following cycle analysis of the shifted window is then applied to a sequence containing the replacements of the previous cycle.<br>
The threshold 70% was determined empirically by compromising between an overall reduction in GC content and maintenance of a high codon optimization for the individual amino acids. As in step 1) the precise replacement strategy (choice of amino acids and GC content threshold value) will again depend on the amino acid sequence encoded by the nucleotide sequence under analysis and will have to be determined empirically.<br>
Step 3) The sequence generated by steps 1) and 2) was then manually edited and additional codons were changed according to the following criteria: Regions<br><br>
still having a GC content higher than 70% over a window of 21 codons were examined manually and a few codons were replaced again following the scheme given in Table 5.<br>
Subsequent steps were performed to provide for useful restriction sites, remove possible open reading frames on the complementary strand, to add homologous recombinant regions, to add a Kozac signal, and to add a terminator. These steps are numbered 4-7<br>
Step 4) The sequence generated in step 3 was examined for the absence of certain restriction sites (Bglll, Pmel and Xbal) and presence of only 1 StuI site to allow a subsequent cloning strategy using a subset of restriction enzymes. Two sites (one for Bglll and one for StuI) were removed from the sequence by replacing codons that were part of the respective recognition sites.<br>
Step 5) The sequence generated by steps 1) through 4) was then modified according to allow subsequent generation of a modified NSOPTmut sequence (by homologous recombination). In the sequence obtained from steps 1) through 4) the segment comprising base 3556 to 3755 and the segment comprising base 4456 to 4656 were replaced by the corresponding segments from NSOPTmut. The segment comprising bases 3556 to 4656 of SEQ. ID. NO. 10 can be used to replace the problematic region in NSOPTmut (around position 3900) by homologous recombination thus creating the variant of NSOPTmut having the sequence of SEQ. ID. NO. 11.<br>
Step 6) Analysis of the sequence generated through steps 1) to 5) revealed a potential open reading frame spanning nearly the complete fragment on the complementary strand. Removal of all codons CTA and TTA (Leu) and TCA (Ser) from the sense strand effectively removed all stop codons in one of the reading frames on the complementary strand. Although the likelyhood for transcription of this complementary strand open reading frame and subsequent translation into protein is very small, in order to exclude a potential interference with the transcription and subsequent translation of the sequence encoded on the sense strand, TCA codons for Ser were introduced on the sense approximately every 500 bases. No changes were introduced in the segments introduced during step 5) to allow homologous recombination. The TCA codon for Ser was preferred over the CTA and TTA codons for Leu because of the higher relative frequency for TCA (0.05) as compared to CTA<br><br>
1;<br>
(0.02) and TTA (0.03) in human_high.cod. In addition, the average human codon usage from CUTG favored TCA (0.14 against 0.07 for CTA and TTA).<br>
Step 7) In a final step GCCACC was added at the 5' end of the sequence to generate an optimized internal ribosome entry site (Kozak signal) and a TAAA stop sgnal was added at the 3'. To maintain the initiation of translation properties of NSsuboptmut the first 8 codons of the coding region were kept identical to the NSOPTmut sequence. The resulting sequence was again checked for the absence of Bglll, Pmel and Xbal recognition sites and the presence of only 1 StuI site.<br>
The NSsuboptmut sequence (SEQ. ID. NO. 10) has an overall reduced GC content (63.5%) as compared to NSOPTmut (70.3%) and maintains a well optimized level of codon usage optimization. Nucleotide sequence identity of NSsuboptmut is 77.2% with respect to NSmut.<br>
Table 5: Definition of codon replacements performed during steps 1) and 2).<br><br><br><br>
Example 12: Virus Characterization<br>
Adenovectors were characterized by: (a) measuring the physical particles/ml; (b) running a TaqMan PCR assay; and (c) checking protein expression after infection of HeLa cells.<br>
a)	Physical Particles Determination<br>
CsCl purified virus was diluted 1/10 and 1/100 in 0.1% SDS PBS. As a control, buffer A105 was used. These dilutions were incubated 10 minutes at 55°C. After spinning the tubes briefly, O.D. at 260 nm was measured. The amount of viral particles was calculated as follows: 1 OD 260 nm = 1.1 X 1012 physical particles/ml. The results were typically between 5X1011 and 1 X 1012 physical particles /ml.<br>
b)	TaqMan PCR Assay<br>
TaqMan PCR assay was used for adenovectors genome quantification (Q-PCR particles/ml). TaqMan PCR assay was performed using the ABI Prism 7700-sequence detector. The reaction was performed in a final 50 μl volume in the presence of oligonucleotides (at final 200 nM) and probe (at final 200 μM) specific for the adenoviral backbone. The virus was diluted 1/10 in 0.1% SDS PBS and incubated 10 minutes at 55oC.  After spinning the tube briefly, serial 1/10 dilutions (in water) were<br><br>
prepared. 10 nl the 10" , 10'   and 10"  dilutions were used as templates in the PCR assay.<br>
The amount of particles present in each sample was calculated on the basis of a standard curve run in the same experiment. Typically results were between 1 X 1012 and 3 X 1012 Q-PCR particles /ml.<br>
c) Expression of HCVNon-Structural Proteins<br>
Expression of HCV NS proteins was tested by infection of HeLa cells. Cells were plated the day before the infection at 1.5 X 106 cells/dish (10 cm 0 Petri dishes). Different amounts of CsCl purified virus corresponding to m.o.i. of 50, 250 and 1250 pp/cell were diluted in medium (PCS free) up to a final volume of 5 ml. The diluted virus was added on the cells and incubated for 1 hour at 37oC   in a CO2 incubator (gently mixing every 20 minutes). 5 ml of 5% HS-DMEM was added and the cells were incubated at 37oC for 48 hours.<br>
Cell extracts were prepared in 1% Triton/TEN buffer. The extracts were run on 10% SDS-acrylamide gel, blotted on nitrocellulose and assayed with antibodies directed against NS3, NS5a and NS5b in order to check the correct polyprotein cleavage. Mock-infected cells were used as a negative control. Results from representative experiments testing the Ad5-NS, MRKAd5-NSmut, MRKAd6-NSmut and MRKAd6-NS0PTmut are shown in Figure 14.<br>
Example 13: Mice Immunization with Adenovectors Encodine Different NS Cassettes The adenovectors Ad5-NS, MRKAd5-NSmut, MRKAd6-NSmut and MRKAd6-NSOPTmut were injected in C57Black6 mice strains to evaluate their potential to elicit anti-HCV immune responses.   Groups of animals (N=9-10) were<br>
injected intramuscularly with 109 pp of CsCl purified virus. Each animal received two doses at three weeks interval.<br>
Humoral immune response against the NS3 protein was measured in post dose two sera from C57Black6 immunized mice by ELISA on bacterially expressed NS3 protease domain. Antibodies specific for the tested antigen were detected with geometric mean titers (GMT) ranging from 100 to 46000 (Tables 6, 7, 8 and 9).<br><br>
TABLE 6: AD5-NS<br><br><br>
T cell response in C57Biack6 mice was analyzed by the quantitative ELISPOT assay measuring the number of IFNn secreting T cells in response to five pools (named fi-om F to L+M) of 20mer peptides overlapping by ten residues encompassing the NS3-NS5B sequence. Specific CD8+ response induced in C57Black6 mice was analyzed by the same assay using a 20mer peptide encompassing a CD8-+ epitope for C57Black6 mice (pepl480). Cells secreting IFND in an antigen specific-manner were detected using a standard ELIspot assay.<br>
Spleen cells, splenocytes and peptides were produced and treated as described in Example 3, supra. Representative data from groups of C57Black6 mice (N=9-10) immunized with two injections of 109 viral particles of vectors Ad5-NS, MRKAdS-NSmut and MRKAd6-NSmut are shown in Figure 15.<br>
Example 14: Immunization of Rhesus macaques with Adenovectors<br>
Rhesus macaques (N=3-4) were immunized by intramuscular injection of CsCl purified Ad5-NS, MRKAdS-NSmut, MRKAd6-NSmut or MRKAd6-NSOPTmut virus. Each animal received two doses of 1011 or 1010 vp in the deltoid muscle at 0, and 4 weeks.<br>
CMI was measured at different time points by a) IFN-D ELISPOT (see Example 3, supra), b) IFN-D ICS and c) bulk CTL assays. These assays measure HCV antigen-specific CD8+ and CD4+ T lymphocyte responses, and can be used for a variety of mammals, such as humans, rhesus monkeys, mice, and rats.<br>
The use of a specific peptide or a pool of peptides can simplify antigen presentation in CTL cytotoxicity assays, interferon-gamma ELISPOT assays and interferon-gamma intracellular staining assays. Peptides based on the amino acid sequence of various HCV proteins (core, E2, NS3, NS4A, NS4B, NS5a, NS5b) were prepared for use in these assays to measure immune responses in HCV DNA and adenovirus vector vaccinated rhesus monkeys, as well as in HCV-infected humans. The individual peptides are overlapping 20-mers, offset by 10 amino acids. Large pools of peptides can be used to detect an overall response to HCV proteins while smaller pools and individual peptides may be used to define the epitope specificity of a response.<br>
IFN-U ICS<br><br>
For IFN-D ICS, 2 x 106 PBMC in 1 ml R10 (RPMI medium, supplemented with 10% PCS) were stimulated with peptide pool antigens. Final concentration of each peptide was 2 g/ml. Cells were incubated for 1 hour in a CO2 incubator at 37oC and then Brefeldin A was added to a final concentration of 10g /ml to inhibit the secretion of soluble cytokines. Cells were incubated for additional 14-16 hours at 37°C.<br>
Stimulation was done in the presence of co-stimulatory antibodies: CD28 and CD49d (anti-humanCD28 BD340975 and anti-humanCD49d BD340976). After incubation, cells were stained with fluorochrome-conjugated antibodies for surface antigens: anti-CD3, anti-CD4, anti-CD8 (CD3-APC Biosource APS0301, CD4-PE BD345769, CD8-PerCP BD345774).<br>
To detect intracellular cytokines, cells were treated with FACS permeabilization buffer 2 (BD340973), 2x final concentration. Once fixed and permeabilized, cells were incubated with an antibody against human IFN- IFN-FITC (Biosource AHC4338).<br>
Cells were resuspended in 1% formaldehyde in PBS and analyzed at FACS within 24 hours. Four color FACS analysis was performed on a FACSCalibur instrument (Becton Dickinson) equipped with two lasers. Acquisition was done gating on the lymphocyte population in the Forward versus Side Scatter plot coupled with the CD3, CD8 positive populations. At least 30,000 events of the gate were taken. The positive cells are expressed as number of IFN- expressing cells over 10^ lymphocytes.<br>
IFN- ELISPOT and IFN- ICS data from immunized monkeys after one or two injections of 1010 or 1011 vp of the different adenovectors are reported in Figures 16A-16D, 17A, and 17B.<br>
Bulk CTL Assays<br>
A distinguishing effector function of T lymphocytes is the ability of subsets of this cell population to directly lyse cells exhibiting appropriate MHC-associated antigenic peptides. This cytotoxic activity is most often associated with CD8+ T lymphocytes.<br>
PBMC samples were infected with recombinant vaccine viruses expressing HCV antigens in vitro for approximately 14 days to provide antigen<br><br>
restimulation and expansion of memory T cells. Cytotoxicity against autologous B cell lines treated with peptide antigen pools was tested.<br>
The lytic function of the culture is measured as a percentage of specific lysis resulted from chromium released from target cells during 4 hours incubation with CTL effector cells. Specific cytotoxicity is measured and compared to irrelevant antigen or excipient-treated B cell lines. This assay is semi-quantitative and is the preferred means for determining whether CTL responses were elicited by the vaccine. Data after two injections from monkeys immunized with 1011 vp/dose with adenovectors Ad5-NS, MRKAdS-NSmut and MRKAd6-NSmut are reported in Figures 18A-18F.<br>
Other embodiments are within the following claims. While several embodiments have been shown and described, various modifications may be made without departing from the spirit and scope of the present invention.<br><br><br><br><br>
WE CLAIM:<br>
1.	A nucleic acid comprising a nucleotide sequence encoding a<br>
Met-NS3-NS4A-NS4B-NS5A-NS5B polypeptide substantially similar to SEQ ID NO:<br>
1, provided that said polypeptide has sufficient protease activity to process itself to<br>
produce an enzymatically inactive NS5B protein, wherein said substantially similar to<br>
SEQ ID No. 1 has at least 95% overall amino acid identity with SEQ ID No. 1.<br>
2.	The nucleic acid as claimed in claim 1, wherein said polypeptide<br>
substantially similar to SEQ ID NO: 1 differs from SEQ ID NO: 1 by 1-10 amino acids<br>
preferably differs from SEQ ID NO: 1 by 1-20 amino acids more preferably<br>
substantially similar to SEQ ID NO: 1 consists of SEQ ID NO: 1.<br>
3.	The nucleic acid as claimed in claim 1 claim 1, wherein said<br>
nucleotide sequence is the coding sequence of either SEQ ID NO: 10 or SEQ ID NO:<br>
11.<br>
4.	The nucleic acid as claimed in claim 1 claim 1, wherein said nucleotide sequence is the coding sequence of either SEQ ID NO: 2 or SEQ ID NO: 3.<br>
5.	The nucleic acid as claimed in preceding claims which is an expression vector capable of expressing said polypeptide from said nucleotide sequence in a human cell.<br>
6.	The nucleic acid as claimed in claim 1 any one of claims 1-6, wherein<br>
said nucleotide sequence is present in a expression cassette comprising:<br>
a)	a promoter transcriptionally coupled to a nucleotide sequence encoding said polypeptide;<br>
b)	a 5' ribosome binding site functionally coupled to said nucleotide sequence,<br>
c)	a terminator joined to the 3' end of said nucleotide sequence, and<br>
d)	a 3' polyadenylation signal functionally coupled to said nucleotide sequence.<br><br>
9.	The nucleic acid as claimed in claim 8, wherein said promoter is the human intermediate early cytomegalovirus promoter (intron A), said 5' ribosome binding site consists of SEQ ID NO: 12, and said 3' polyadenylation is the bovine growth hormone (BGH) polyadenylation signal.<br>
10.	The nucleic acid as claimed in claim 8, wherein said nucleic acid is a shuttle vector further comprising a selectable marker, an origin of replication, a first adenovirus homology region and a second adenovirus homology region flanking said expression cassette, wherein said first homology region has at least about 100 base pairs substantially homologous to at least right end of a wild-type adenovirus region from about base pairs 1-425, and said second homology region has at least about 100 base pairs substantially homologous to at least the left end of a wild-type adenovirus region from about base pairs 3511-5792 of Ad5 or corresponding region of another adenovirus.<br>
11.	The nucleic acid as claimed in claim 8, wherein said nucleic<br>
acid is a DNA plasmid suitable for administration into a human and fiarther comprises a<br>
prokaryotic origin of replication and a gene coding for a selectable marker.<br>
12.	The nucleic acid as claimed in claim 8, wherein said nucleic acid is a adenovirus genome plasmid comprising a selectable marker, an origin of replication, and a recombinant adenovector genome containing an El deletion, an E3 deletion, and said expression cassette.<br>
13.	The nucleic acid as claimed in claim 1 claim 12, wherein said nucleic acid is a adenovirus genome plasmid comprising a selectable marker, an origin of replication, and<br><br>
a)	a first adenovirus region from about base pair 1 to about base pair 450 corresponding to either Ad5 or Ad6;<br>
b)	said gene expression cassette in a El parallel or El anti-parallel orientation joined to said first region;<br><br>
c)	a second adenovirus region from about base pair 3511 to about base pair 5548 corresponding to Ad5 or from about base pair 3508 to about base pair 5541 corresponding to Ad6, joined to said expression cassette;<br>
d)	a third adenovirus region from about base pair 5549 to about base pair 28133 corresponding to Ad5 or from about base pair 5542 to about base pair 28156 corresponding to Ad6, joined to said second region;<br>
e)	a fourth adenovirus region from about base pair 30818 to about base pair 33966 corresponding to Ad5 or from about base pair 30789 to about base pair 33784 corresponding to Ad6, joined to said third region; and<br>
f)	a fifth adenovirus region from about base pair 33967 to about base pair 35935 corresponding to Ad5 or from about base pair 33785 to about base pair 35759 corresponding to Ad6, joined to said fourth region,<br><br>
14.	The nucleic acid as claimed in claim 13, wherein said promoter is also the human intermediate early cytomegalovirus promoter, said 5' ribosome binding site consists of SEQ ID NO: 12, and said 3' polyadenylation is the BGH polyadenylation signal.<br>
15.	The nucleic acid as claimed in claim 1 claim 12, wherein said nucleic acid is a adenovirus genome plasmid comprising an origin of replication, a selectable marker, and:<br><br>
a)	a first adenovirus region from about base pair 1 to about base pair 450 corresponding to either Ad5 or Ad6;<br>
b)	a second adenovirus region from about base pair 3511 to about base pair 5548 corresponding to Ad5 or from about base pair 3508 to about base pair 5541 corresponding to Ad6, joined to said first region;<br>
c)	a third adenovirus region from about base pair 5549 to about base pair 28133 corresponding to Ad5 or from about base pair 5542 to about base pair 28156 corresponding to Ad6, joined to said second region;<br>
d)	said gene expression cassette in a E3 parallel or E3 anti-parallel orientation joined to said third region;<br><br>
e)	a fourth adenovirus region from about base pair 30818 to about base pair 33966 corresponding to Ad5 or from about base pair 30789 to about base pair 33784 corresponding to Ad6, joined to said gene expression cassette; and<br>
f)	a fifth adenovirus region from about base pair 33967 to about base pair 35935 corresponding to Ad5 or from about base pair 33785 to about base pair 35759 corresponding to Ad6, joined to said fourth region.<br><br>
16.	The nucleic acid as claimed in claim 1 claim 1, wherein said nucleic acid is a adenovector consisting of a nucleotide sequence of SEQ ID NO. 4 or a derivative thereof, wherein said derivative thereof has the HCV polyprotein encoding sequence present in SEQ ID NO: 4 replaced with the HCV polyprotein encoding sequence of either SEQ ID NO: 3, SEQ ID NO: 10 or SEQ ID NO: 11.<br>
17.	An adenovector produced by a process comprising the steps of:<br>
a)	producing an adenovirus genome plasmid by homologous<br>
recombination between the shuttle vector of claim 10 and a nucleic acid comprising;<br>
a first adenovirus region from about base pair 1 to about base pair 450 corresponding to either Ad5 or Ad6;<br>
a second adenovirus region from about base pair 3511 to about base pair 5548 corresponding to Ad5 or from about base pair 3508 to about base pair 5541 corresponding to Ad6, joined to said first region;<br>
a third adenovirus region from about base pair 5549 to about base pair 28133 corresponding to Ad5 or from about base pair 5542 to about base pair 28156 corresponding to Ad6, joined to said second region;<br>
a fourth adenovirus region from about base pair 30818 to about base pair 33966 corresponding to Ad5 or from about base pair 30789 to about base pair 33784 corresponding to Ad6 Joined to said third region; and<br>
a fifth adenovirus region from about base pair 33967 to about base pair 35935 corresponding to Ad5 or from about base pair 33785 to about base pair 35759 corresponding to Ad6, joined to said fourth region; and<br>
b)	rescuing said adenovector from said adenovirus plasmid.<br><br>
18.	A method of making an adenovector comprising the steps of:<br>
a)	producing an adenovirus genome plasmid comprising a gene<br>
expression cassette by homologous recombination between the nucleic acid of claim 10<br>
and a nucleic acid comprising;<br>
a first adenovirus region from about base pair 1 to about base pair 450 corresponding to either Ad5 or Ad6;<br>
a second adenovirus region from about base pair 3511 to about base pair 5548 corresponding to Ad5 or from about base pair 3508 to about base pair 5541 corresponding to Ad6, joined to said first region;<br>
a third adenovirus region from about base pair 5549 to about base pair 28133 corresponding to Ad5 or from about base pair 5542 to about base pair 28156 corresponding to Ad6 Joined to said second region;<br>
a fourth adenovirus region from about base pair 30818 to about base pair 33966 corresponding to Ad5 or from about base pair 30789 to about base pair 33784 corresponding to Ad6, joined to said third region; and<br>
a fifth adenovirus region from about base pair 33967 to about base pair 35935 corresponding to Ad5 or from about base pair 33785 to about base pair 35759 corresponding to Ad6, joined to the fourth region; and<br>
b)	rescuing said recombinant adenovirus from said recombinant<br>
adenovirus plasmid.<br>
19.	A pharmaceutical composition comprising the nucleic acid as claimed in preceding claims and conventional pharmaceutically acceptable carrier such as herein described.<br>
20.	A nucleic acid as claimed in preceding claims and a pharmaceutical composition thereof is useful for treating or preventing HCV infected subjects.<br><br>
21.      A   nucleic  acid   and   a  pharmaceutical   composition  thereof substantially as herein described with reference to examples, drawings and sequence.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDc0MC1jaGVucC0yMDA0IGNsYWltcyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">0740-chenp-2004 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDc0MC1jaGVucC0yMDA0IGRlc2NyaXB0aW9uIChjb21wbGV0ZSkgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">0740-chenp-2004 description (complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzQwLWNoZW5wLTIwMDQtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">740-chenp-2004-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzQwLWNoZW5wLTIwMDQtYXNzaWduZW1lbnQucGRm" target="_blank" style="word-wrap:break-word;">740-chenp-2004-assignement.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzQwLWNoZW5wLTIwMDQtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">740-chenp-2004-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzQwLWNoZW5wLTIwMDQtY29ycmVzcG9uZG5lY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">740-chenp-2004-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzQwLWNoZW5wLTIwMDQtY29ycmVzcG9uZG5lY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">740-chenp-2004-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzQwLWNoZW5wLTIwMDQtZGVzY3JpcHRpb24oY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">740-chenp-2004-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzQwLWNoZW5wLTIwMDQtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">740-chenp-2004-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzQwLWNoZW5wLTIwMDQtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">740-chenp-2004-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzQwLWNoZW5wLTIwMDQtZm9ybSAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">740-chenp-2004-form 13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzQwLWNoZW5wLTIwMDQtZm9ybSAyNi5wZGY=" target="_blank" style="word-wrap:break-word;">740-chenp-2004-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzQwLWNoZW5wLTIwMDQtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">740-chenp-2004-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzQwLWNoZW5wLTIwMDQtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">740-chenp-2004-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzQwLWNoZW5wLTIwMDQtcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">740-chenp-2004-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="218515-4-methanesulfonyl-biphenyl-derivatives-of-formula-i.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="218517-a-method-and-image-signal-correcting-apparatus-and-a-pickup-and-reproduction-system-thereof.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>218516</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>740/CHENP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>21/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>23-May-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>02-Apr-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>08-Apr-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MERCK &amp; CO., INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>126 East Lincoln Avenue, Rahway, NJ 07065-0907 (US)</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>EMINI, Emilio</td>
											<td>126 East Lincoln Avenue, Rahway, NJ 07065 - 0907,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SHIVER, John, W</td>
											<td>VIA PONTINA KM 30.600, I-00040 POMEZIA,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>CORTESE, Riccardo</td>
											<td>VIA PONTINA KM 30.600, I-00040 POMEZIA,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>KASLOW, David, C</td>
											<td>126 East Lincoln Avenue, Rahway, NJ 07065 - 0907,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>BETT, Andrew, J</td>
											<td>126 East Lincoln Avenue, Rahway, NJ 07065 - 0907,</td>
										</tr>
										<tr>
											<td>6</td>
											<td>NICOSIA, Alfredo</td>
											<td>VIA PONTINA KM 30.600, I-00040 POMEZIA,</td>
										</tr>
										<tr>
											<td>7</td>
											<td>LAHM, Armin</td>
											<td>VIA PONTINA KM 30.600, I-00040 POMEZIA,</td>
										</tr>
										<tr>
											<td>8</td>
											<td>LUZZAGO, Alessandra</td>
											<td>VIA PONTINA KM 30.600, I-00040 POMEZIA,</td>
										</tr>
										<tr>
											<td>9</td>
											<td>COLLOCA, Stefano</td>
											<td>VIA PONTINA KM 30.600, I-00040 POMEZIA,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 14/18</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2002/032512</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-10-10</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/363,774</td>
									<td>2002-03-13</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/328,655</td>
									<td>2001-10-11</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/218516-hepatitis-c-virus-vaccine by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:51:49 GMT -->
</html>
