<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/218785-gene-coding-for-acetolactate-synthase by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:34:10 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 218785:GENE CODING FOR ACETOLACTATE SYNTHASE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">GENE CODING FOR ACETOLACTATE SYNTHASE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention provides a gene coding for the following protein (a) or (b) showing a high level of resistance to PC herbicides or sulfonylurea herbicides: (a) a protein which consists of an amino acid sequence of any one of SEQ ID NOS: 2, 4, 6 and 8; (b) a protein which consists of an amino acid sequence derived from the amino acid sequence of any one of SEQ ID NOS: 2, 4, 6 and 8 by substitution, deletion or addition of at least one or more amino acids, has resistance to a pyrimidinyl carboxy herbicide, and has acetolactate synthase activity.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FIELD OF THE INVENTION<br>
The present invention relates to a gene coding for acetolactate synthase which is a rate-limiting enzyme in the branched-chain amino acid biosynthetic pathway.<br>
BACKGROUND OF THE INVENTION<br>
Acetolactate synthase (hereinafter referred to as "ALS") is a rate-limiting enzyme in the biosynthetic pathway of branched chain amino acids, such as leucine  valine and isoleucine, and is known as an essential enzyme for the growth of plants. ALS is also known to be present in a wide variety of higher plants. In addition, ALS is found in various microorganisms, such as yeast {Saccharomyces cerevisiae), Escherichia coli, and Salmonella typhimurium.<br>
Three types of isoenzymes of ALS are known to be present in Escherichia coli and Salmonella typhimurium. Each of these isoenzymes is a hetero-oligomer consisting of (catalytic subunits) with a large molecular weight that govern catalytic activity of the enzyme and (regulatory subunit^ with a small molecular weight that function as feedback inhibitors by binding of branched-chain amino acids (Chipman et al., Biochim. Biophys. Acta. 1385, 401-419, 1998). Catalytic subunits are located at Ilv IH, IIv GM and Ilv BN operons, respectively. On the other hand, ALS in yeast is a single enzyme, which comprises a catalytic subunit and a regulatory subunit, as is the case in bacteria (Pang et al., Biochemistry, 38, 5222-5231, 1999). The catalytic protein subunit is located at the locus ILV2.<br>
In plants, ALS is known to consist catalytic subunit(s) and regulatory subunit(s) as, is the case in the above microorganisms (Hershey et al.. Plant Molecular Biology. 40, 795-806, 1999). For example, the catalytic subunit of ALS in tobacco (dicotyledon) is coded by two gene loci, SuRA and SuRB (Lee<br><br>
et al., EMBO J. 7, 1241-1248, 1988); and that in maize is coded by two gene loci, als 1 and als 2 (Burr et al.. Trends in Genetics 7, 55-61, 1991; Lawrence et al., Plant Mo^l. Biol. 18, 1185-1187, 1992). The nucleotide sequences of genes coding for a catalytic subunit have been completely determined for dicotyledonous plants including tobacco, Arabid^opsis, rapeseed, cotton, Xanthium, Amaranthus and Kochia (see Chipman et al., Biochim. B^iophys. Acta. 1385, 401-419, 1998 and domestic re-publication of PCT internatjona] publication for patent applications WO97/08327). However, maize and rice (Kaku et al., the 26"" Conference of Pesticide Science Society of Japan, Lecture Abstracts, plOl, 2001) are the only monocotyledonous plants whose nucleotide sequences have been completely determined.<br>
Meanwhile, herbicides, for example, sulfonylurea herbicides, imidazolinon herbicides, ^triazolopyrimidine herbicides and pyrimidinyl carboxy herbicides (hereinafter referred to as t^^^^^^^^^^^PC herbiddsa"!)! are known to suppress the growth of a plant by inhibiting ALS (Ray, Plant Physiol. 75, 827-831, 1984; Shaner et al., Plant Physiol.76, 545-546, 1984; Subramanian et al., Plant Physiol. 96, 310-313, 1991; Shimizu et al., J. Pestic. Sci.l9, 59-67, 1994).<br>
As shown in Tables 1 and 2, known plants having resistance to these herbicides contain a gene coding for ALS that includes substitution of one or two nucleotides which induces substitution of one or two amino acids in a region conserved among different species.<br>
 ^<br>
.J<br><br><br><br>
Table 2<br>
Mu^     tation in plant ALS which imparts resistance to ALS-inhibiting type<br>
herbicide (2)<br>
Plant species      Mutation  Corresponding rice ALS amino acid<br>
Kochia scoparia	A5p260Gly	SU      Asp242Gly<br>
Kochia scoparia	Trp487Arg	SU      Try465Arg<br>
Kochia scoparia	Asn561Ser	SU      Asn5395er<br>
Kochia scoparia	Trp570Leu	Trp548Leu<br>
Gossypium hirsutum L.	Trp563Cys	SU ?    Try548Cys<br>
Gossypium hirsutum L.. Trp563Ser	SU ?	Try548Ser<br>
Brassica napus	Trp557Leu	Try548Leu<br>
Zea  mays L.	Trp552Leu	IM      Try548Leu<br>
Nicotine tabacum L.	Trp537Leu	SU      Try548Le//**520u<br>
Arabidopsis thaliona	Trp574Leu	Try548Leu<br>
Arabidopsis thaliona	Trp574Ser	Try548Ser<br>
Arabidopsis thaliano	Trp574deletion	Try548deletion<br>
Xantium strumarium	Trp552Leu	IM      Try548Leu<br>
Oryza sativa.	Trp548Leu	PC      Try548Leu<br>
Amaranthus sp.	Trp569Leu	.Try548Leu<br>
Amaransus rudis	Trp569Leu	IMI     Try548Leu<br>
Sisymbrium orientole	Trp—»Leu	Try548Leu<br>
Zea mays	Ser621Asp	IM      Ser627Asp<br>
Zea  mays	Ser621Asn	IM      Ser627Asn<br>
Arabidopsis thaliana	Ser653Asn	IM      Ser627Asn<br>
Arabidopsis thaliano	Ser653Thr	Ser627Thr<br>
Arabidopsis thaliana	Ser653Phe	Ser627Phe<br>
Arabidopsis thaliano	Ser653delition	Ser627deletion<br>
Oryza sativa	Ser627Ile	PC      Ser627Ile<br>
Kochia scoparia	Val276Glu	SU<br>
Examples of such a gene include a gene coding for ALS having resistance specific to sulfonylurea herbicides (see Kathleen et al., EMBO J. 7, 1241-1248, 1988; Mourad et al., Planta, 188, 491-497, 1992; Guttieri et al., Weed Sci. 43, 175-178, 1995; Bernasconi et al., J. Biol. Chem. 270, 17381-17385, 1995; and JP Patent Publication (Unexamined Application) No. 63-71184); a gene coding for ALS having resistance specific to imidazolinon herbicides (see Mourad et al., Planta, 188, 491-497, 1992; Lee et al., FEES Lett. 452, 341-345, 1999; and JP Patent Publication (Unexamined Application) No. 5-227964); a gene coding for ALS having resistance to b/852014droth sulfonylurea and imidazolinon herbicides (see Kathleen et a]., EMBO J. 7,  1241-1248, 1988;<br><br>
Bernasconi et al., J. Biol. Chem. 270, 17381-17385, 1995; Hattori et al., Mol. Gen. Genet. 246, 419-425, 1995; Alison et al., Plant Physiol. Ill, 1353, 1996wr55; Rajasekarau et al.. Plant Sci. 119, 115-124, 1996; JP Patent Publication (Unexamined Application) No. 63-71184; JP Patent Publication (Unexamined Application) No. 4-311392; and Bernasconi et al., US Patent 5633437, 1997); and a gene coding for ALS having a high level of resistance to PC herbicides (Kaku et al., the 26"^ Conference of Pesticide Science Society of Japan, Lecture Abstracts, plOl, 2001). The production of a plant body showing resistance to both sulfonylurea and imidazolinon herbicides has been attempted by crossing a plant having ALS showing resistance specific to sulfonylurea herbicides with a plant having ALS showing resistance specific to imidazolinon herbicides (Mourad et al., Mol. Gen. Genet, 243, 178-184, 1994). Furthermore, artificial alteration of a gene coding for ALS into a herbicide resistance gene has been attempted (see Ott et al., J. Mol. Biol. 263, 359-368, 1996, JP356rf22 Patent Publication (Unexamined Application) No. 63-71184, JP Patent Publication (Unexamined Application) No. 5-227964, JP Patent Publication (PCT Translation) No. 11-504213), such that it has been found that a (single amino acid deletion causes ALS to show resistance to both sulfonylurea and imidazolinon herbicide (see JP Patent Publication (Unexamined Application) No. 5-227964).<br>
As described above, ALSs having resistance to herbicides, and genes<br>
r coding for ALS have been aggressively studied.    However,, only a few cases<br>
U-	;<br>
have been reported concerning a mutant ALS gene having resistance specific to<br>
a PC herbicide using resistance to PC herbicides as an indicatpr.h Moreover,<br>
there have been also only a few cases reported concerning the study of the<br>
resistance to PC herbicides and othd344er herbicides.<br>
SUMMARY OF THE INVENTION<br>
The purpose of the present invention is to provide a gene coding for an<br><br>
ALS protein showing extremely high level of resistance to PC herbicides or to sulfonylurea herbicides, an ALS protein coded by the gene, a recombinant vector having the gene, a transformant having the recombinant vector, a plant having the gene, a method for rearing the plant, and a method for selecting a transformant cell using the gene as a selection marker.<br>
As a result of thorough studies to achieve the above purpose, we have completed the present invention by finding that a mutant ALS which is derived from the wild type ALS by substituting a certain arnino acid residue of the wild type ALS with a certain amino acid shows extremely high resistance to PC herbicides.<br>
(1)	Specifically, the present invention is a gene which codes for the following<br>
protein (a) or (b):<br>
(a)	a protein consisting of an amino acid sequence of any one of SEQ ID NOS: 2, 4, 6 and 8;<br>
(b)	a protein consisting of an amino acid sequence derived from the amino acid se33u867-+2 quence of any one of SEQ ID NOS: 2, 4, 6 and 8 by substitution, deletion or addition of at least one or more amino acids, which has resistance to PC herbicides and has acetolactate synthase activity.<br><br>
(2)	Further, the present invention is an acetolactate synthase protein, which is coded by the gene of (1).<br>
(3)	Furthermore, the present invention is a recombinant vector, which has the gene of (1).<br>
(4)	Further, the present invention is a transformant, which has the recombinant vector of (3).<br>
(5)	Moreover, the present invention is a p***lant, which has the gene of (1) and has resistance to PC herbicides.<br>
(6)	Further, the present invention is a method for cultivating the plant of (5) which comprises cultivating the plant in the presence of a PC herbicide.<br><br>
(7) Still further, the present invention is a method for selecting a transformant cell having the gene of (1), which uses this gene as a selection marker.<br>
Hereunder, a more detailed explanation will be given of the present invention.<br>
The gene coding for the acetolactate synthase of the present invention (hereinafter referred to as "mutant ALS gene") codes for an acetolactate synthase protein (hereinafter referred to as "mutant ALS protein") having an amino acid sequence that is different from that of a wild type acetolactate synthase protein (hereinafter, referred to as "wild type ALS protein"). The mutant ALS protein can be obtained by mutating a certain site in a wild type ALS protein expressed in a rice plant. The mutant ALS protein of the present invention consists of the amino acid sequence of any one of SEQ ID NOS: 2, 4, 6, and 8.<br>
The amino acid sequence of SEQ ID NO: 2 is derived from the amino<br>
acid sequence of the wild type ALS protein by substitution of proline 171 with<br>
histidine and substitution of arginine 172 with serine.)   A mutant ALS protein /<br>
containing  the  amino  acid  sequence  of SEQ  ID  NO:  2  is referred  to as<br>
"P171H/R172S mutant ALS protein," or\:"P171H/R172S mutant." ^	-^<br>
The amino acid sequence of SEQ ID NO: 4 is derived from the amino acid sequence of the wild type ALS protein by^"substitution of proline 171 with histidine and substitution of tryptophan 548 with leucine^ A mutant ALS protein containing the amino acid sequence of SEQ ID NO: 4 is referred to as "P171H/W548L mutant ALS protein," or"(T^71H/W548L mutant.".<br>
The amino acid sequence of SEQ ID NO: 6 is derived from the amino acid sequence of the wild type ALS protein by/substitution of proline 171 with histidine, and substitution of serine 627 with isoleucine. A mutant ALS protein containing the amino acid sequence of SEQ ID NO: 6 is referred to as "P171H/S627I mutant ALS protein," oi{"P171H/S627I mutant."<br><br>
The amino acid sequence of SEQ ID NO: 8 is derived from the amino acid sequence of the wild type ALS protein by substitution of proline 171 with histidine, substitution of tryptophan 548 with leucine, and substitution of serine 627 with isoleuciney A mutant ALS protein containing the amino acid sequence of SEQ ID NO: 8 is referred to as "P171H/W548L/S627I mutant ALS protein," orv"P171H/W548L/S627I mutant."<br>
Figs. lA and B show the results of comparisons among the amino acid sequences of these 4 types of mutant ALS proteins and the amino acid sequence of the wild type ALS protein. Further, in Figs. lA and B, the amino acid sequence in the 1^" row represents the wild type ALS protein, the amino acid sequence in the 2"*" row represents P171H/R172S mutant ALS protein, the amino acid sequence in the 3^** row represents P171H/W548L mutant ALS protein, the amino acid sequence in the 4"^ row represents P171H/S627I mutant ALS protein, and the amino acid sequence in the 5*"^ row represents P171H/W548L/S627I mutant ALS protein.<br>
Compared to the wild type ALS protein, these mutant ALS proteins possess good resistance not only to PC herbicides, but also to sulfonylurea and imidazolinon herbicides, l^ This can be determined by subcloning a gene coding for the mutant ALS protein into pGEX 2T, transforming E. coli or the like with the pGEX 2T, and then examining the sensitivity of the expressed mutant ALS protein to herbicides.   )<br><br>
Examples of a PC herbicide include bispyribac-sodium, pyrithiobac/////-sodium and pyriminobac, as represented by the following chemical formula 1.<br><br><br>
.^^COOH      pCHs<br>
N-<br>
OChfe<br>
pyrithiobac-sodium<br>
An example of a sulfonylurea herbicide is chlorsulfuron, as represented by the following chemical formula 2.<br>
pyriminabac<br>
//    \	OCH3<br>
f        &gt;—COONa<br>
0CH3<br>
bispyribac-sodium<br><br>
COONa<br>
yOCH3<br>
N—OCH3<br><br>
OCH3 / N<br>
HsGs^<br><br>
OCH3<br>
CI<br>
=
Cx"<br><br>
N<br>
chlorsulfuron<br><br>
An example of an imidazolinon herbicide is imazaquin, as represented by the following chemical formula 3.<br>
COOH<br>
OX"<br>
r-^<br>
H<br>
imazaquui<br>
In particular, P171H/R172S mutant ALS protein shows resistance to a<br><br>
certain herbicide at a level not only better than that of a mutant ALS protein independently having P171H or R172S, but also superior to the combined resistance predicted from the mutant ALS proteins independently having P171H or R172S. ] Further, the mutant ALS protein independently having R172S does not show resistance to any herbicides, therefore the R172S mutation is a silent mutation. In other words, in P171H/R172S mutant ALS protein, R172S mutation, which is a silent mutation by itself, improves the resistance of P171H mutant ALS protein.<br>
Further, P171H/W548L mutant protein shows resistance to a certain herbicide at a level not only better than that of a mutant ALS protein independently having P171H or W548L, but also better than the combined resistance predicted from the mutant ALS proteins independently having P171H or W548L. In other words, P171H/W548L mutant protein shows resistance which is far greater than the synergistic effect predicted from the resistances of both P171H mutant protein and W548L mutant protein.<br>
Further, in particular, P171H/S627I mutant protein shows resistance to a certain herbicide at a level not only better than that of a mutant ALS protein independently having P171H or S627I, but also better than the combined resistance predicted from the mutant ALS proteins independently having PI71H or S627I. In other words, P171H/S627I mutant protein shows resistance which is far greater than the synergistic effect predicted from the resistances of both P171H mutant protein and S627I mutant protein.<br>
Still further, in particular, P171H/W548L/S627I mutant protein shows resistance to a certain herbicide better than that of a mutant ALS protein independently having P171H, W548L or S627I.<br>
Adoreover, the mutant ALS protein of the present invention may consist of any amino acid sequence derived from the amino acid sequence of any one of SEQ ID NOS: 2, 4, 6 and 8 by substitution, deletion or addition of at least one or more amino acids, as long as the sequence has resistance to a PC herbicide<br><br>
and has acetolactate synthase activity. Here, "one or more amino acids" preferably refers to 1 to 30 amino acids, more preferably 1 to 20 amino acids, and more preferably 1 to 10 amino acids.<br>
Particularly, in the amino acid sequence of SEQ ID NO: 2, "at least one<br>
or more amino acids" are preferably (an) amino acids other than the 171*" and<br>
172""" amino acids. In the amino acid sequence of SEQ ID NO: 4, "at least one<br>
or more amino acids" are preferably (an) amino acids other than the 171^" and<br>
548"^ amino acids. In the amino acid sequence of SEQ ID NO: 6, "at least one<br>
or more amino acids" are preferably (an) amino acids other than the 171*" and<br>
627"^ amino acids. In the amino acid sequence of SEQ ID NO: 8, "at least one<br>
or more amino acids" are preferably (an) amino acids other than the 171*", 627"^,<br>
and 548"" amino acids.<br>
V	The mutant ALS  gene  of the present invention  is not  specifically<br>
1 limited, as long as it has a nucleotide sequence coding for the above-described V mutant ALS protein. Examples of the nucleotide sequence include the nucleotide sequence of any one of SEQ ID NOS: 1, 3, 5 and 7. The nucleotide sequence of SEQ ID NO: 1 codes for the amino acid sequence of SEQ ID NO: 2, the nucleotide sequence of SEQ ID NO: 3 codes for the amino acid sequence of SEQ ID NO: 4, the nucleotide sequence of SEQ ID NO: 5 codes for the amino acid sequence of SEQ ID NO: 6, and the nucleotide sequence of SEQ ID NO: 7 codes for the amino acid sequence of SEQ ID NO: 8. The mutant AT.S gene may have a nucleotide sequence derived from the nucleotide sequence of any one of SEQ ID NOS: 1, 3, 5 and 7 by substitution of a codon coding for a certain amino acid with a Regenerate codon.<br>
Figs. 2A, B, C and D show the results of comparisons among the nucleotide sequences coding for these 4 types of mutant ALS proteins and the nucleotide sequence coding for a wild type ALS protein. In Figs. 2A, B, C and D, the nucleotide sequence in the 1*" row represents the wild type ALS protein, the nucleotide sequence in the 2"** row represents P171H/R172S mutant ALS<br><br>
protein, the nucleotide sequence in the 3"^"" row represents P171H/W548L mutant ALS protein, the nucleotide sequence in the 4"^ row represents P171H/S627I mutant ALS protein, and the nucleotide sequence in the 5"^ row represents P171H/W548L/S627I mutant ALS protein.<br>
Moreover, the mutant ALS gene of the present invention may consist of a nucleotide sequence which can hybridize under stringent conditions to a nucleotide sequence complementary to the nucleotide sequence of any one of SEQ ID NOS: 1,3,5 and 7, and codes for an amino acid sequence having acetolactate synthase activity, (^tringent conditions"" refers to conditions wherein a so-called specific hybrid is formed and a non-specific hybrid is not formed. Examples of such stringent conditions include conditions whereby DNAs having high homology to each other (for example, DNAs having 50% or more homology to each other) hybridize and DNAs having low homology to each other do not hybridize. Specific examples of the stringent conditions, under which hybridization is possible, include conditions for Washing in the normal Southern hybridization of 60°C, and a salt concentration corresponding to IxSSC, 0.1% SDS, or preferably, O.lxSSC, 0.1% SDS.l<br>
Genes coding for these mutant ALS proteins can be obtained by introducing a mutation as described above into a gene coding for a wild type ALS protein which is present in the genomic DNA of japonica type rice variety, Kinmaze. } To introduce mutations, any known techniques can be employed. For example, site-directed mutagenesis can be used. Site-directed mutagenesis can be performed using a commercial kit, e.g., Mutan-K (Takara Shuzo), Gene Editor (Promega) or ExSite (Stratagene). ^<br>
In addition, a gene coding for the mutant ALS protein can be obtained by culturing wild type culture cells sensitive to a PC herbicide in the presence of the PC herbicide and then obtaining the gene from mutant culture cells that appear and show resistance to the PC herbicide. \Then, a gene coding for ALS protein having a new combination of mutations can be synthesized based on the<br><br>
thus found mutations by the PCR method and SPR (self polymerase reaction) method using enzymes. -,<br>
Specifically, first, mRNAs are prepared from mutant culture cells resistant to a PC herbicide, cDNAs are synthesized, and then a cDNA library of A,gt 11 phage is constructed. Then, the library is screened using a nucleic acid probe containing part of a gene coding for the wild type ALS protein. Next, the insert DNA of the resulting positive clone is subcloned into pBluescript II SK+, to determine the nucleotide sequence. For cDNA inserts that have been shown to have mutations, fragments containing the mutation are synthesized by the PCR and SPR methods using as a template pBluescript II SK+ retaining the insert DNA, and primers designed based on the wild type rice ALS gene. Meanwhile, genomic DNAs are prepared from PC-herbicide-resistant rice culture cells, and various primers are designed based on rice ALS genes. Then, primer walking is performed, so that the sequences of ALS genes present in the prepared genomic DNAs are determined, and mutations sites are found. When mutations are found, fragments containing the mutations are synthesized by the PCR and SPR methods. Fragments containing mutations synthesized from mutant ALS cDNA cloned into pBluescript II SK+ (including the fragments containing these mutations) are subcloned into pGEX 2T, and then E. coli is transformed using the vector.<br>
Clones having the insert DNAs coding for the amino acid sequences represented by SEQ ID NOS: 2, 4, 6 or 8 are then selected, so that genes coding for mutant ALS proteins can be obtained. In addition, the thus obtained plasmid in which a gene coding for a mutant ALS protein containing the amino acid sequence represented by SEQ ID NO: 2 had been incorporated in pGEX 2T was deposited as Rice Mutant ALS cDNA 1 (PERM BP-7944), the plasmid in which a gene coding for a mutant ALS protein containing the amino acid sequence represented by SEQ ID NO: 4 had been incorporated in pGEX 2T was deposited as Rice Mutant ALS cDNA 2 (PERM BP-7945), the plasmid in which<br><br>
a gene coding for a mutant ALS protein containing the amino acid sequence represented by SEQ ID NO: 6 had been incorporated in pGEX 2T was deposited as Rice Mutant ALS cDNA 3 (FERM BP-7946), and the plasmid in which a gene coding for a mutant ALS protein containing the amino acid sequence represented by SEQ ID NO: 8 had been incorporated in pGEX 2T was deposited as Rice Mutant ALS cDNA 4 (FERM BP-7947) with the Patent and Bio-Resource Center, National Institute of Advanced Industrial Science and Technology (Chuo-6, 1-1-1, Higashi, Tsukuba-shi, Ibaraki, JAPAN) on March 8, 2002 under the Budapest Treaty.<br>
On the other hand, transformation of a target plant using a gene coding for the mutant ALS protein can impart resistance to various herbicides, such as PC herbicides, to the plant. Any known technique can be used for transformation of a plant. For example, a foreign gene can be introduced into a target plant cell using Agrobacterium tumefaciens.<br>
More specifically, a gene coding for the mutant ALS protein is inserted into a binary vector containing T-DNA sequence of a Ti plasmid of Agrobacterium. The Ti plasmid is transformed into E. coli and the like. Then, the binary vectors retaining the gene coding for the mutant ALS protein replicated by, e.g., E. coli are transformed into Agrobacteria which contain helper plasmids. Target plants are infected with the Agrobacteria, and then the transformed plants are identified. When the identified transformed plant is a culture cell, the plant cell can be regenerated into a complete plant by any known technique.<br>
To transform a target plant with a gene coding for the mutant ALS protein, the   gene   can   be   directly   introduced   using   known   standard   techniques. \ Examples of a method which transforms an expression vector containing a gene coding for the mutant ALS protein include the polyethylene glycol method, electroporation, and the particle gun method."<br>
A gene coding for the mutant ALS protein may be transformed into any<br><br>
l^<br><br>
type of plant, such as monocotyledonous and dicotyledonous plants. Examples of a target crop into which a gene coding for the mutant ALS protein is transformed include rice, maize, wheat, barley, soybean, cotton, rapeseeds, sugar beet and tobacco. In addition, turf grass, trees )and the like can be transformed by introducing a gene coding for the mutant ALS protein.<br>
In any of the above cases, transformation of a plant using a gene coding for the mutant ALS protein can impart resistance to PC herbicides, sulfonylurea herbicides, and imidazolinon herbicides to the plant.<br>
r Moreover, a gene coding for the mutant ALS protein can also be used as a selection marker in an experiment for transformation of a plant. For example, to transform a plant cell using a target gene, a vector which has a gene coding for the mutant ALS protein and a target gene is introduced into the plant cell, followed by culturing of the plant cell under the presence of a PC herbicide or the like. If a plant cell survives in the presence of the herbicide, it indicates that the plant cell contains a gene coding for the mutant ALS protein and the gene of interest introduced therein. Further, whether a target gene and a gene coding for the mutant ALS protein are incorporated into the chromosome of a plant cell can be confirmed by observing the phenotype of the plant and then examining the presence of these genes on the genome, by genome southern hybridization or PCR.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figure lA shows an amino acid sequence comparison between the mutant ALS proteins and the wild type ALS protein.<br>
Figure IB is a continuation from Fig. lA, and shows an amino acid sequence comparison between the mutant ALS proteins and the wild type ALS protein.<br>
Figure 2A shows a nucleotide sequence comparison between the mutant ALS genes and the wild type ALS gene.<br><br>
Figure 2B is a continuation from Fig. 2A, and shows a nucleotide sequence comparison between the mutant ALS genes and the wild type ALS gene.<br>
Figure 2C is a continuation from Fig. 2B, and shows a nucleotide sequence comparison between the mutant ALS genes and the wild type ALS gene.<br>
Figure 2D is a continuation from Fig. 2C, and shows a nucleotide sequence comparison between the mutant ALS genes and the wild type ALS gene.<br>
Figure 3 is a characteristic figure showing sensitivity of Rb line to bispyribac-sodium.<br>
Figure 4 is a characteristic figure showing sensitivity of Sr line to bispyribac-sodium.<br>
Figure 5 is a characteristic figure showing sensitivity of Ga line to bispyribac-sodium.<br>
Figure 6 is a characteristic figure showing sensitivity of Vg line to bispyribac-sodium.<br>
Figure 7 is a characteristic figure showing sensitivity of the wild type to bispyribac-sodium.<br>
Figure 8 is a characteristic figure showing sensitivity of the wild type to chlorsulfuron.<br>
Figure 9 is a characteristic figure showing sensitivity of Rb line to chlorsulfuron.<br>
Figure 10 is a characteristic figure showing sensitivity of Sr line to chlorsulfuron.<br>
Figure 11 is a characteristic figure showing sensitivity of Ga line to chlorsulfuron.<br>
Figure 12 is a characteristic figure showing sensitivity of Vg line to chlorsulfuron.<br><br>
Figure 13 is a characteristic figure showing the relation between the fraction number and absorbance at OD 525 nm in anion exchange column chromatography performed for the purpose of separating the ALS protein of the resistant mutant.<br>
Figure 14 is a characteristic figure showing the relation between the fraction number and absorbance at OD 525 nm in anion exchange column chromatography performed for the purpose of separating the wild type ALS protein.<br>
Figure 15 is a characteristic figure showing sensitivity of the wild type ALS protein and the mutant ALS protein to bispyribac-sodium.<br>
Figure 16 is a characteristic figure showing sensitivity of the wild type ALS protein and the mutant ALS protein to chlorsulfuron.<br>
Figure 17 is a characteristic figure showing sensitivity of the wild type ALS protein and the mutant ALS protein to imazaquin.<br>
Figure 18A shows a nucleotide sequence comparison between Nippon-bare EST and maize ALS gene.<br>
Figure 18B is a continuation from Fig. 18A and shows a nucleotide sequence comparison between Nippon-bare EST and maize ALS gene.<br>
Figure 19A is a nucleotide sequence comparison between the full-length cDNA derived from Sr line and wild type cDNA 1.<br>
Figure 19B is a continuation from Fig. 19A, and shows a nucleotide sequence comparison between the full-length cDNA derived from Sr line and wild type cDNA 1.<br>
Figure 19C is a continuation from Fig. 19B, and shows a nucleotide sequence comparison between the full-length cDNA derived from Sr line and wild type cDNA 1.<br>
Figure 20 shows processes for synthesizing ALS cDNAs independently having G1643T (W548L) mutation or G1880T (S627I) mutation, and for constructing pGEX 2T retaining the ALS cDNA.    Arrows denote primers, and<br><br>
asterisks denote mutated points.<br>
Figure 21 shows a process for preparing C512A (P171H) mutant DNA fragment and C514A (R172S) mutant DNA fragment. Arrows denote primers, and asterisks denote mutated points.<br>
Figure 22 shows processes for synthesizing ALS cDNAs independently having C512A (P171H) mutation or C514A (R172S) mutation, and for constructing pGEX 2T retaining the ALS cDNA. Asterisks denote mutated points.<br>
Figure 23 shows a process for preparing a DNA fragment having C512A(P171H)/C514A(R172S). Arrows denote primers, and asterisks denote mutated points.<br>
Figure 24 shows processes for synthesizing P171H/W548L mutant ALS cDNA and P171H/S627I mutant ALS cDNA and for constructing pGEX 2T retaining the ALS cDNA.    Asterisks denote mutated points.<br>
Figure 25 shows processes for synthesizing P171H/W548L/S627I mutant ALS cDNA and for constructing pGEX 2T retaining the ALS cDNA. Asterisks denote mutated points.<br>
Figure 26 shows a comparison of sensitivity to bispyribac-sodium between the mutant ALS protein coded by 1-point mutant ALS gene and the wild type ALS protein.<br>
Figure 27 shows a comparison of sensitivity to bispyribac-sodium among the mutant ALS proteins coded by 2-point and 3-point mutant ALS genes and the wild type.<br>
I BEST MODE FOR CARRYING OUT THE INVENTION<br>
Now, the present invention will be further described by the following examples, but the technical scope of the invention is not limited by these examples. ^<br><br>
[Example   1]  Production of rice  (Kinmaze) culture cells resistant to  a PC herbicide<br>
Chaff was removed from rice seeds (variety: Kinmaze, scientific name: Oryza sativa var. Kinmaze). The seeds were immersed in 70% ethanol for 5 minutes, and then immersed in about 5% antiformin for 20 minutes, followed by washing several times with sterile distilled water. Then, the seeds were static-cultured on a medium with a composition as shown in Table 3.<br>
Table 3<br><br>
Inorganic    salt    (mixed     saline    for Murashige-Skoog medium)	I pack<br>
Thiamin-HCl (0.1 g/1)	1 ml<br>
Nicotinic acid (0.5 g/1)	1 ml<br>
Pyridoxine-HCl (0.5 g/1)	1 ml<br>
Glycine (2 g/1)	1 ml<br>
myo-inositol (50 g/1)	2 ml<br>
2,4-D (200 ppm)	10 ml<br>
Sucrose	30 g<br>
Gelrite	3g<br><br>
Prepare the medium to  1000 ml with distilled water, and adjust pH to 5.7.	<br>
In the above medium composition, 2,4-D is synthesized auxin. To prepare the medium, first, a medium with the above composition was placed in a II beaker, and distilled water was added to the beaker to 1000 ml. Next, the solution was adjusted to pH 5.7, and supplemented with 3 g of Gelrite. The Gelrite was dissolved well by heating with a microwave oven, and then the mixture was added 30 ml at a time to culture flasks using a pipetter. Next, three sheets of aluminum foil were laid over the culture flask, followed by heating for sterilization in an autoclave at 121 °C for 15 to 20 minutes. Finally the solution was cooled to room temperature so that the media for static culture of the above seeds were prepared.<br><br>
Next, endosperm .portions were removed from the callus induced on the medium, and then subculture was performed. Then, part of the obtained calli was sub-cultured, that is, cultured successively once per two weeks in a liquid medium (the composition is the same as in that shown in Table 3, but not supplemented with Gelrite) supplemented with 1 )LIM bispyribac-sodium (one type of PC herbicides). Two to 6 weeks later the culture cells started to wither. About 2 months later, a plurality of non-discolored cell masses that were thought to be conducting cell division were obtained from among culture cell populations most of which had died and became discolored brown. These cell masses were isolated and cultured, so that a plurality of cell lines that can proliferate in the presence of 2 ^M bispyribac-sodium were obtained. The obtained cell lines were named Rb line, Sr line, Ga line and Vg line, respectively.<br>
Subsequently, the resulting plurality of cell lines were cultured while elevating the concentration of bispyribac-sodium in an orderly manner. As a result, cell lines that can proliferate in the presence of 100 )iM bispyribac-sodium were obtained. The bispyribac-sodium resistant culture cells (hereinafter referred to as "resistant mutant") were sub-cultured on MS-2,4-D solid media supplemented with 1 to 10 )j,M bispyribac-sodium. Part of the sub-cultured resistant mutant was sampled, added into MS-2,4-D liquid media not supplemented with bispyribac-sodium, and then subjected to suspended cell culture at a cycle of 8 to 10 days.<br>
Approximately 1.5 g (wet weight) of the resistant mutant was transplanted into a 200 ml Erlenmeyer flask supplemented with 50 ml of a MS-2,4-D liquid medium and bispyribac-sodium at a Icertain concentration) followed by culturing at approximately 27°C for a certain period. The wet weight of the callus was measured periodically. The relative amount of increase was determined based on the wet weight of the transplanted resistant mutant.    In addition, the experiment was performed three times with different<br><br>
bispyribac-sodium concentrations, and the standard error was calculated. Figures 3 to 6 show the relation between changes in bispyribac-sodium concentration and the relative weight on day 8 or 12 in the resistant mutant. As a control, a similar experiment was conducted using the wild type (Kinmaze). Figure 7 shows the result of measuring the relation between bispyribac-sodium concentration and relative weight on day 8 in the wild type (Kinmaze).<br>
As shown in Fig. 7, the growth of the wild type was not inhibited in a group supplemented with 1 nM bispyribac-sodium, but was inhibited in a group supplemented with 10 nM or more bispyribac-sodium. On the other hand, as shown in Figs. 3 to 6, almost none of the growth of the resistant mutants (Rb line, Sr line, Ga line, and Vg line) other than Vg line was affected even in a group supplemented with 10 |J.M bispyribac-sodium. Even in Vg line, it was shown that the effect of bispyribac-sodium on the growth was smaller than that in the wild type.<br>
Also in the case of using ch]orsu]furon instead of bispyrib^ac^^spdium, the growth rates of the wild type and the resistant mutants were measured as described above. Figure 8 shows the relation between changes in chlorsulfuron concentration and relative weight on day 9 in the wild type. Further, Figs. 9 to 12 show the relation between changes in chlorsulfuron concentration and relative weight on day 8 or 10 in the resistant mutants, that is, Rb line, Sr line, Ga line and Vg line.<br>
As shown in Fig. 8, the growth of the wild type was inhibited by addition of 1 nM chlorsulfuron, showing that the wild type has higher sensitivity to chlorsulfuron than to bispyribac-sodium. However, as shown in Figs. 9 to 12, Rb line, Sr line, Ga line and Vg line differed in sensitivity, but the growth was not inhibited so much by addition of chlorsulfuron, showing their resistance to chlorsulfuron. Sensitivity to bispyribac-sodium and chlorsulfuron remained almost unchanged in both the wild type and the resistant mutants, even with longer culture duration.    The growth rate was almost the same in the wild type<br><br>
and the resistant mutants.<br>
These results revealed that the resistant mutants possess high resistance to bispyribac-sodium. Moreover, the resistant mutants were shown to have improved resistance to chlorsulfuron compared to the wild type.<br>
[Example 2] Herbicide sensitivity of ALS protein partially purified fi-om the resistant mutant<br>
In this example, mutant ALS protein was partially purified fi-om the resistant mutants obtained in Example 1 (Rb line, Sr line and Vg line, with Ga line excluded), and then herbicide sensitivity of the obtained mutant ALS protein was examined. The mutant ALS protein was partially purified as follows.<br>
First, 200 g or more of resistant mutant was prepared by a liquid culture method (no supplementation with bispyribac-sodium), using a composition as shown in Table 3 excluding Gelrite. Then, about 150 g of the resistant mutant was homogenized using Hiscotron in a volume of buffer-1 [100 mM potassium phosphate buffer (pH 7.5) containing 20% (v/v) glycerol, 0.5 mM thiamin pyrophosphate (TPP), 10 \xM flavin adenine dinucleotide (FAD), 0.5 mM MgCb, and a volume of polyvinyl polypyrrolidone one-tenth that of tissue volume] 3-fold greater than tissue volume. The homogenate was filtered through nylon gauze, and then centrifuged at 15000 x g for 20 minutes. Ammonium sulfate was added to the centrifuged supernatant to 50 % saturation, and then allowed to stand in ice for approximately 1 hour. The mixture was again centrifuged at 15000 x g for 20 minutes, and then the precipitated fraction was dissolved in approximately 30 ml of buffer-2 [10 mM Tris hydrochloric acid buffer (pH 7.5) containing 20 % (v/v) glycerol, 0.5 mM TPP and 0.5 mM MgCh]. The mixture was again centrifuged at 15000 x g for 20 minutes, and then the supernatant fraction was applied to a Sephadex G-25 (Amersham Bioscience). About 40 ml of the fraction that had passed through the column was collected as<br><br>
a crude enzyme solution.<br>
Next, the protein concentration of the crude enzyme solution was measured by the Bradford method according to the manual of Bio-Rad Protein Assay. The crude enzyme solution was then filtered through a Whatman filter (Whatman), and then the crude enzyme solution in an appropriate protein amount (10 to 15 ml) was applied to three vertically-connected HiTrap Q columns (Amersham Bioscience) using a FPLC device (Amersham Bioscience). After protein component was adsorbed using HiTrap Q, unadsorbed fractions were washed out using buffer-2 having a volume 3 to 5 fold greater than the bed volume. Then, the adsorbed protein component was eluted using an eluate having a volume 10 fold greater than the bed volume (150 ml). Here, the eluate was prepared by dissolving KCI with a linear concentration gradient (0 to 0.4 M) into buffer-2. The eluate containing the eluted protein component was apportioned, 5 ml each, into a plurality of test tubes for apportioning. Further, to stabilize ALS protein contained in the eluted protein component, 0.5 ml of buffer-2 containing 20 mM sodium pyruvate had been previously added to each test tube for apportioning.<br>
ALS activity resulting from the mutant ALS protein contained in the eluted fractions apportioned into each test tube for apportioning was measured as follows. A reaction solution to be used in a measurement reaction was prepared by mixing an eluted fraction to be measured v/ith a solution comprising 20 mM sodium pyruvate, 0.5 mM TPP, 0.5 mM MgCb, 10 }aM FAD and 20 mM potassium phosphate buffer (pH 7.5). One ml of this reaction solution was used. After the eluted fraction to be measured was added, the measurement reaction was performed at 30°C for 40 to 60 minutes. Then, the reaction was stopped by addition of 0.1 ml of 6N sulfuric acid (or 0.25 N sodium hydroxide).<br>
After the reaction was stopped, the reaction solution was incubated at 60°C for 10 minutes, thereby converting acetolactate contained in the reaction<br><br>
solution to acetoin.<br>
Then, to quantify acetoin contained in the reaction solution, 1 ml of 0.5 % (w/v) creatine and 1 ml of 5 % (w/v) a-naphthol dissolved in 2.5 N sodium hydroxide was added to the reaction solution, followed by incubation at 37°C for 10 minutes. Acetoin was then quantified by color comparison of the absorbance (at 525 nm) of the reaction solution, thereby evaluating ALS activity. In addition, since the reaction solution contained a small amount of sodium pyruvate, reaction time 0 was used as control.<br>
As a result, absorbance at OD525 nm was as high as approximately 7 per 0.2 ml of the reaction solution. /However, when the above measurement reaction was ceased with sodium hydroxide, and acetoin generation activity due to activity other than ALS activity was examined, nearly 80 % of the apparent ALS activity resulted from direct acetoin generation activity which was not due to activity of the mutant ALS protein. Accordingly, the mutant ALS protein and the other proteins were separated for acetoin generation activity by FPLC using anion exchange resin. | Figure 13 shows the result in the case of using Sr line as a resistant mutant. As a result, three activity peaks were detected as shown in Fig. 13.<br>
To determine which one of the three activity peaks corresponded to the mutant ALS protein, acetoin generation activity was examined for each peak. Thus it was found that a fraction shown by the peak of initial elution corresponded to the mutant ALS protein.<br>
Using the enzyme solution containing the mutant ALS protein, sensitivity of the mutant ALS protein to bispyribac-sodium, chlorsulfuron and imazaquin was examined. Sensitivity to each of these herbicides was evaluated by measuring ALS activity in the same manner as in the above measurement reaction, except that a herbicide was added to a certain concentration before addition of the enzyme solution. For comparison, the wild type ALS protein was separated and purified (Fig.  14) in the same manner and used for the<br><br>
experiment. In addition, bispyribac-sodium was prepared as an aqueous solution, and chlorsulfuron and imazaquin were prepared as acetone solutions. The final concentration of acetone in the reaction mixture was 1 %.<br>
Figure 15 shows the relation between ALS activity inhibition rate and bispyribac-sodium concentration. Figure 16 shows the relation between ALS activity inhibition rate and chlorsulfuron concentration. Figure 17 shows the relation between ALS activity inhibition rate and imazaquin concentration. In these Figs. 15 to 17, a dotted line denotes the wild type ALS protein, a long dashed double-dotted line denotes Sr line of the mutant ALS protein, a solid line denotes Rb line of the mutant ALS protein, and a long dashed dotted line denotes Vg line of the mutant ALS protein.<br>
A herbicide concentration which inhibits 50 % of ALS activity (150) was found from calculation according to probit analysis, thereby calculating the ratio of 150 for the mutant ALS protein vs. 150 for the wild type ALS protein. Table 4 shows the results.<br><br>
Bispyribac-sodium<br>
Chlorsulfuron<br>
Imazaquin<br><br>
Table 4				<br>
Herbicide		Iso (nM)		<br><br>
	Wild type	Vg	Sr	Rb<br><br>
5.63	97.2	421	247000<br>
17.3	495	92.8	32000<br>
1480	44100	16700	609000<br><br>
Further, based on the results in Table 4, 150 of the resistant mutant against each herbicide was divided by 150 of the wild type to work out RS.    The results are shown in Table 5.<br><br>
Table 5<br><br>
		RS ratio	<br>
mei Diciue	Vg	Sr	Rb<br>
Bispyribac-sodium	17.3	74.8	43900<br>
Chlorsulfuron	28.6	5.36	1850<br>
Imazaquin	29.8	11.3	411<br>
As shown in Figs. 15 to 17 and Tables 4 and 5, the mutant ALS protein showed a relatively high ALS activity even in the presence of the herbicide, when compared to the wild type ALS protein. In particular, the mutant ALS proteins derived from Rb line and Sr line were shown to have sensitivity to bispyribac-sodium which was significantly superior to sensitivities to other herbicides. That is, Rb and Sr lines possess good resistance to bispyribac-sodium in particular.<br>
[Example 3] Cloning of wild type and mutant ALS genes<br>
In this example, a gene (wild type ALS gene) coding for the wild type ALS protein was cloned from the wild type, while a gene (mutant ALS gene) coding for the mutant ALS protein was cloned from the resistant mutant.<br>
Probes used for cloning the wild type ALS gene and the mutant ALS gene were prepared as follows. The partial cDNA derived from rice (Nippon-bare) showing high homology with the ALS gene of maize was used as a probe in this example.<br>
(1) Determination of the nucleotide sequence of a partial cDNA derived from rice (Nippon-bare) showing high homology with the ALS gene of maize<br>
As a part of the Rice Genome Project conducted by the Society for Techno-innovation of Agriculture, Forestry and Fisheries, and the National Institute of Agrobiological Sciences, partial nucleotide sequences of cDNAs of rice (Nippon-"^"^"""*"* li^H  hpen Hptermined  and  a partial nucleotide sequence<br><br>
database of cDNAs had already been established. A cDNA clone (Accession No. C72411) which is known as a nucleotide sequence of approximately 350 bp contained in this database showed high homology to the ALS gene of maize. The ALS gene of maize had been completely sequenced.<br>
This cDNA clone (Accession No. C72411) was obtained from the National Institute of Agrobiological Sciences, and the nucleotide sequence was determined as follows. Here, the cDNA clone comprised an ALS homolog gene inserted within pBluescript II SK+, and it was capable of autonomous replication in E. coli.<br>
First, an ALS homolog-retaining plasmid vector was transformed into E. coli (DH5a). White colonies obtained from a plate were cultured in liquid, and then plasmids were extracted from the cells by standard techniques. Since the insert DNA had been inserted between Sal I and Not I (restriction enzymes of multi-cloning sites injhe plasmid vector), the vector was digested with the two enzymes. The insert was confirmed by agarose electrophoresis. Then, the obtained ALS homolog-retaining plasmid vector was purified by standard techniques using, e.g., RNaseA, PEG and LiCl, followed by sequencing reaction using primers and an ABI BigDyeTerminator Cycle Sequencing Kit. I Conditions for PCR reaction followed the manufacturer"s protocols. Primers used herein were Ml 3 primers and synthesized primers designed from the determined nucleotide sequence. The resulting PCR product VA^S purified by ethanol precipitation, and then the nucleotide sequence thereof was determined by an ABI PRISM 310 sequencer.<br>
The ALS homolog-retaining plasmid vector is known to contain an insert DNA with a length of 1.6 kb. The obtained ALS homolog-retaining plasmid vector was digested with restriction enzymes Sal I and Not I, and then subjected to electrophoresis. As a result, a band of approximately 3 kbp corresponding to pBluescript II SK+ and a band of approximately 1.6 kbp corresponding to the insert DNA fragment were detected (data not shown).    The entire nucleotide<br><br>
sequence of the insert DNA portion was determined, and its homology to the nucleotide sequence of maize was searched. As shown in Figs. 18A and B, 84.7 % homology was found. (Since the ALS homolog was determined to be a partial cDNA of the ALS gene of the var. Nippon-bare, the insert DNA excised after digestion with Sal I and Not I was used as a probe/^ Further in Figs. 18A and B, the first row is a nucleotide sequence of the cDNA of the ALS gene of the var. Nippon-bare; the second row is that of the ALS gene of maize.<br>
(2) Preparation of mRNA from resistant mutant and wild type<br>
First, the resistant mutant frozen with liquid nitrgg^en was crushed with a mortar and pestle, and then finely crushed with a mixer for 30 seconds.    The crushed powder was suspended in an extraction buffer [(100 mM Tris-HCl pH 9.0, 100 mM NaCl, 1 weight% SDS, 5 mM EDTA) : (p-mercaptoethanol) : (Tris saturated phenol) = 15:3:20], and then stirred thoroughly.      This solution was centrifuged at 12000 x g for 15 minutes, and then the supernatant was collected. Two    hundred    ml    of   PCI    [(Tris    saturated    phenol)    :    (chloroform)    : (isoaniylalcohol) = 25:24:1] was added to the supernatant, shaken at 4°C for 10 minutes, centrifuged at 12000 x g for 15 minutes, and then the supernatant was collected.    The procedure was repeated twice.    A 1/20 volume of 5 M NaCl and a 2.2-fold volume of ethanol were added to the obtained supernatant, and then the mixture was allowed to stand at -80°C for 30 minutes.    The precipitate was collected by centrifugation at 12000 x g for 5 minutes.    The precipitate was   washed   with   70%   ethanol,   dried,   and   then   dissolved   in   10   mM P-mercaptoethanol solution.    Next, the solution was centrifuged at 27000 x g for 10 minutes to remove insoluble fraction.    A 1/4 volume of 10 M LiCl was added to the solution, which was then allowed to stand on ice for 1 hour. Further, the solution was centrifuged at 27000 x g for 10 minutes to collect precipitate, dissolved in 4 ml of H2O, and then absorbance at 260 nm was measured to find the concentration of RNA.    A 1/20 volume of 5 M NaCl and a<br><br>
2.2-fold volume of ethanol were added to the solution, which was then allowed<br>
to stand at -80°C for 30 minutes.    Subsequently the solution was centrifuged at<br>
27000 X g for 10 minutes to collect the precipitate, followed by washing with<br>
70 % ethanol, and drying. The resulting product was dissolved in an appropriate<br>
amount of H2O to obtain a total RNA solution.    Here,  centrifugation was<br>
performed at 4°C.<br>
mRNA was separated and purified from total RNA by the following<br>
r	<br>
method..  A 2x binding buffer (20 mM Tris-HCl (pH 7.5), 10 mM EDTA, 1 M<br>
NaCl) in a volume equivalent to that of the extracted total RNA solution was<br>
added to the extracted total RNA solution.    A column filled with 0.1 g of oligo<br>
dT cellulose (Amersham Bioscience) was washed with a Ix binding buffer, and<br>
then the total RNA solution was applied to the column.    After the column was<br>
washed with a Ix binding buffer, an elution buffer (10 mM Tris-HCl (pH 7.5), 5<br>
mM EDTA) was applied, and the eluate collected 0.5 ml at a time.    Fractions<br>
that had passed through the column were applied to another oligo dT cellulose<br>
(Amersham Bioscience) column, and treated in the same manner.    After the<br>
concentration of eluted mRNA was calculated based on the absorbance of each<br>
fraction, a 1/10 volume of 10 M LiCl and a 2.5-fold volume of ethanol were<br>
added to the products, and then the mixtures were allowed to stand at -80°C for<br>
30 minutes.    Next, the mixtures were centrifuged and the precipitated fractions<br>
were dried, and dissolved in 100 fil of H2O.    The thus obtained mRNA was<br>
subjected to size fractionation by sucrose density gradient centrifugation.<br>
The separated and purified mRNA was applied to a centrifuge tube with<br>
density gradient given by a 25 % sucrose solution and 5 % sucrose solution, and<br>
then ultraceiitrifuged at 27000 rpm for 15 hours at 4°C using a swing rotor.<br>
After centrifugation, 0.5 ml of each fraction was collected in order of density<br>
gradient.    Absorbance of each fraction was measured, the concentration of the<br>
/ collected   mRNA   was   calculated,   and (^the  presence   of ALS   mRNA   was<br>
confirmed by hybridization using an ECL kit (ECL direct nucleic acid labeling<br><br>
and detection system, Aniersham Bioscience). Hybridization was performed using a probe prepared in (1) above at 42°C for 16 hours. After hybridization, washing at 42°C for 5 minutes was performed twice using a primary washing buffer provided with the kit, and then washing at 42°C for 5 minutes was performed once using 2 x SSC solution. The washed film was wrapped with a transparent plastic film to keep it immersed in an attached luminous reagent provided with the kit, and then exposed to an X-ray film.<br>
When Sr line was used as the resistant mutant, approximately 35 mg of total RNA and approximately 4 mg of mRNA could be extracted by the above procedures. Further, in sucrose density gradient centrifugation, a hybridization-positive spot was found for a fraction expected to be positive.<br>
When the wild type was used, approximately 95 mg of total RNA was extracted in addition to approximately 7 mg of mRNA. When mRNA was extracted from the wild type, the above method was applied except that the wild type was used instead of the resistant mutant.<br>
(3) Construction of cDNA libraries derived from resistant mutant and wild type Using 2 |j,g of mRNA purified in (2) above and a cDNA synthesis kit (Amersham  Bioscience),  cDNA  was  synthesized,   so  that  a  cDNA  library derived from the resistant mutant was constructed.<br>
First, RTase provided with the kit was used for a reverse transcription reaction; and T4 DNA polymerase provided with the kit was used for a subsequent complementary chain elongation reaction. At the time of complementary chain elongation reaction, P-dCTP was added to calculate the yield of cDNA synthesis. After an adaptor was added, the synthesized cDNA was incorporated into A, phage by in vitro packaging method.<br>
The adaptor added to cDNA was an Eco Kl-Not l-Bam HI adaptor (Takara<br>
f-Shuzo).    Adapters with a molar concentration \50-fold greater than that of<br>
cDNA were added to a solution containing cDNA."  Then, T4 DNA Ligase<br><br>
(Pharmacia) was added to the mixture followed by ligation reaction at 4°C overnight. The reaction solution was applied to HPLC using an AsahiPak GS 710 column (Asahi Chemical Industry Co., Ltd.), followed by monitoring of the eluate with ultraviolet rays at a wavelength of 260 nm. The eluate was fractionated into 25 fractions of 0.5 ml each. Each fraction was measured with a Cerenkov counter, and 3 to 4 fractions with a high count were collected. The 5" terminus of the adaptor contained in the fraction was phosphorylated using T4 polynucleotide kinase (Takara Shuzo), and then A,gt 11 Eco RI arm was added to perform ligation. GigaPack Gold III (Stratagene) was added to the solution, and then ligation reaction was performed at room temperature for 2 hours. After reaction, 200 pi of an SM buffer and 8 pi of chloroform were added to the reaction solution, thereby preparing a phage solution. This phage solution was diluted 10-fold. One pi of the diluted solution was infected with E. coli (Y-1088), to which 0.7 % top agar was added, and then the solution was inoculated over an LB plate. The number of plaques that had appeared on the plate 4 to 8 hours later was counted, thereby measuring the titer.<br>
Synthesis of approximately 74 ng of cDNA derived from Sr line was confirmed by the result of DE 81 pa£er and Cerenkov counting. The result of Cerenkov counting after ligation of a vector with an adaptor added thereto revealed that approximately 22 ng of A.DNA contained the insert was obtained for Sr line. The A,DNA was packaged into the phage, thereby preparing a cDNA library derived from the cells of the resistant mutant. The titer of the library solution was 16600 pfu/|al.<br>
When a cDNA library was constructed using mRNA extracted from the wild type according to the above-described method, it was shown that approximately 38 ng of cDNA derived from the wild type had been synthesized. Further, approximately 5 ng of ?iDNA contained the insert was obtained for the wild type. Furthermore, the titer of the cDNA library solution derived from the wild type was 18160 pfu/pl.<br><br>
(4) Screening of cDNA containing the ALS gene<br>
To form about 20,000 plaques on plates, the library solution prepared in (3) above was diluted, and then phages derived from the wild type and those derived from Sr line were separately inoculated over 10 plates, respectively. Plaques were then transferred to a nitrocellulose membrane (Schleicher &amp; Schnell, PROTORAN BA85, pore size 0.45 ^m), and the nitrocellulose membrane was immersed in a denaturation solution (0.5 M NaOH, 1.5 M NaCl), and then in a neutralization solution (1.5 M NaCl, 0.5 M Tris-HCl (pH 7.5), 1 mM EDTA) for approximately 20 seconds. Excess water was removed from the nitrocellulose membrane using a filter paper, and then the nitrocellulose membrane was baked at 80°C for 2 hours. Here, the baking step was omitted when Hybond-N+ (Amersham Biotech) was used instead of a nitrocellulose membrane, and immobilization was performed with 0.4 M NaOH for 20 minutes.<br>
The insert DNA prepared in (1) above was labeled by two types of method, RI and non-RI, and then used as a probe DNA. Labeling with RI and hybridization were performed by the following method. First, approximately 200 to 500 ng of probe DNA was thermally denatured, and then labeled using a BcaBEST DNA labeling kit (Takara Shuzo). At the time of this labeling reaction, a buffer, random primers and P-dCTP provided with the kit were added. Next, BcaBEST was added, followed by incubation at 65°C for 30 minutes. Subsequently, EDTA was added to stop the reaction. The reaction solution was applied to nitrocellulose membranes, so that 8 of the membranes contained approximately 100 ng of probes. Hybridization was performed at 42°C overnight with weak shaking. After hybridization, the membranes were washed three times with 2 x SSC, 0.1 % SDS solution, followed by exposure for about 1 hour to an imaging plate of a BAS 2000 imaging analyzer (Fuji Photo Film).    Following exposure, positive clones were detected using the imaging<br><br>
analyzer.<br>
Labeling with non-RI was performed by the following method. Following thermal denaturation of approximately 200 to 500 ng of probe DNA, DNA labeling reagent (peroxidase) and glutaraldehyde which were provided with an ECL direct DNA/RNA labeling and detection system (Amersham Bioscience) were added, followed by incubation at 37°C. In this case, the labeled probe DNA was applied to nitrocellulose membranes, so that 8 of the membranes contained approximately 100 ng of the labeled probe DNA. Hybridization was performed at 42°C overnight with weak shaking. After hybridization, the membrane was washed three times with a primary washing buffer at room temperature for 10 minutes, and then once with 2 x SSC at room temperature for 10 minutes. The membrane was immersed in a luminous solution provided with the ECL kit, and then exposed to an X-ray film for 30 minutes to 3 hours.<br>
Positive phages obtained by hybridization (primary screening) were scraped off together with top agar using a sterile toothpick, and then suspended in 200 ^1 of SM buffer, thereby obtaining a phage solution. Phage solutions of each clone were appropriately diluted, infected with E. coli strain Y-1088, and then inoculated over LB plates. Using these newly prepared plates, hybridization (secondary screening) was performed similarly. Positive phages were suspended in 200 fil of a SM buffer, thereby obtaining single phages. If no single phage was isolated by secondary screening, another dilution was performed, followed by inoculation over LB plates. Subsequently, hybridization (the third screening) was performed, so that single phages were obtained.<br>
Next, ^DNA was prepared from the single phages by the following methods. 1 phages collected with a bamboo brochette or a toothpick from plaques of positive clones were inoculated in 200 \i\ of a 2xYT medium (containing 10 mM MgCb and 0.2% maltose) containing 5 |.il of a suspension of<br><br>
fresh host E. coli (Y1088).    The product was allowed to stand and incubated at 42°C overnight.    Then, the medium was inoculated again in 1 ml of a 2xYT medium (containing 10 mM MgCl2 and 0.2% maltose) containing 25 jil of a suspension of host E. coli (Y1088), and then shake-cultured overnight (these steps compose a pre-culturing process).    The pre-cultured solution (10 to 50 \x\) was inoculated in 12 ml of 2xYT medium containing 10 mM MgCli and 0.5 ml of E.  coli Y1088 suspension.    Then, incubation was performed at 42°C overnight with relatively strong shaking, until turbidity increased after lysis. After culturing, 50 \\\ of chloroform and 1.2 ml of 5 M NaCl were added, and then incubation was performed at 42°C for 10 minutes while shaking.    The product was centrifuged at 27000 x g for 10 minutes, and then the supernatant was newly transferred to a centrifugation tube.    Five ml of 50 % PEG was added to the supernatant, and then incubated on ice for 1 hour or more.    The product was centrifuged at 27000 x g for 10 minutes, and then the supernatant was discarded.    Next, another centrifugation was performed at 27000 x g, and then the liquid portion was discarded.    The precipitated fraction was suspended in 300 \i\ of a 30 mM Tris hydrochloric acid buffer (pH 7.5) containing 4 \xg of DNase  I,  20   \ig  of RNase  A  and   10 niM  MgCli.     The  suspension  was transferred to a 1.5 ml tube.    After incubation of the suspension at 37°C for 30 minutes, 7.5 |al of 20 % SDS, 3|.il of proteinase K (10 mg/ml), and 12 ^il of 0.5 M EDTA were added to the su.spen.sion, followed by further incubation at 55°C for 15 minutes.    Subsequently, 150 \x\ of phenol was added to the product, and then stirred vigorously.    Then the mixture was centrifuged at 15000 rpm for 3 minutes using a TOMY Microcentrifuge MR-150 (TOMY DIGITAL BIOLOGY), and an aqueous layer was collected.    800 |al of ethyl ether (to which distilled water had been added to remove peroxide) was added to the collected aqueous layer.    The mixture was stirred vigorously, and then centrifuged at 15000 rpm for 10 seconds and the ether layer was discarded.    After the ether extraction step was repeated, ether remaining in the aqueous layer was removed with<br><br>
nitrogen gas. Thirty |.il of 5 M NaCl and 875 |al of ethanol were added to the aqueous layer, so that precipitated XDNA was rapidly collected. The collected X.DNA was rinsed with approximately 1 ml of 70 % ethanol, and then dried under reduced pressure for approximately 1 minute, thereby removing ethanol. The product was dissolved in 20 pi to 50 p.1 of a TE buffer (pH 8.0), thereby preparing a XDNA solution.<br>
Subcloning and sequencing of the insert DNA in the obtained ?tDNA were performed by the following method.    The obtained X.DNA solution (1 pi) was digested with Not I so as to excise the insert DNA.    The composition of a reaction solution (for cleavage reaction) followed the procedure in the manual attached to the restriction enzyme.    After reaction at 37°C for approximately 2 hours, the insert size was confirmed by electrophoresis using 1 % agarose gel. A,DNA (10 pi to 20 pi) containing the insert DNA was digested with Not I, so as to excise the insert DNA.    The insert DNA was separated using agarose gel for apportioning, the corresponding band was cleaved from the gel, and then the insert DNA was purified by standard techniques.    The insert DNA was mixed with  a  vector  following  BAP  treatment  (dephosphorylation  using  alkaline phosphatase derived from a shrimp) at molar ratio of 1:1, followed by ligation reaction with T4 DNA ligase at 16°C for 2 hours or more.    Here, since the insert DNA cleaved with Not I was used as material, BAP treatment was performed for vectors cleaved with Not I.    Following ligation, part of the solution was mixed with competent cells (DH5a), and then allowed to stand on ice for 30 minutes.    Next, the mixture was subjected to heat shock at 42°C for 30 seconds, and then allowed to stand on ice again for 2 minutes.    Then, SOC was added to the mixture, incubated at 37°C for 1 hour, inoculated over a LB medium plate on which a mixture of 100 pi of 2xYT (containing 50 pg/ml ampicillin), 30 pi of 3 % X-Gal and 3 pi of 1 M IPTG had been previously added  uniformly,  and then cultured  at 37°C  for  10 hours  or more.     The transformed white colonies were each inoculated on 2 ml of an LB medium<br><br>
containing ampicillin or a 2 x YT medium, and then cultured at 37°C overnight. From the culture solution, plasmids were prepared by standard techniques and dissolved in H2O. The DNA concentration thereof was quantified, and then the plasmids were subjected to PCR reaction for sequencing. PCR reaction and sequencing were performed by methods described above.<br>
As a result of the above experiment, the ALS cDNA with an incomplete length of approximately 2.2 kb was obtained from the culture cells of each wild type and Sr line.    Since an Sma I site was present at a position approximately 250 bp from the 5" side of the DNA, a new probe was prepared by the following method.    pBluescript II SK+ retaining the ALS cDNA with an incomplete length of approximately 2.2 kbp derived from the wild type was amplified with host E.  coll JM109, and then plasmids were extracted using an automated isolation system (KURABO PI-100).    The plasmid was directly digested with Sj-na I.    The generated fragment of approximately 250 bp was separated and purified  by   1   %  agarose  electrophoresis,  and  then  the  concentration was calculated,  thereby preparing  a probe.     Using the probe,  the  library  was screened again by the above method employing the above RI.    A,DNA was prepared from the thus obtained single phages, the XDNA solution (1 |al) was digested with Eco RI, and then size was confirmed by electrophoresis, followed by immobilization onto a nitrocellulose membrane.    Following electrophoresis, the gel was immersed in 0.5 M NaOH solution containing 1.5 M NaCl, and then shaken lightly for 15 minutes.    The gel was then washed with water, immersed in 0.5 M Tris-HCl (pH 7.5) containing 3 M NaCl, and then neutralized while shaking for approximately 15 minutes.    Approximately 5 thick, industrial filter papers were piled up to make a base.    The base was placed in 20xSSC spread over  a  stainless  bat.     Subsequently,   the  neutralized   gel,   a   nitrocellulose membrane (which had been cut into a certain size, immersed in distilled water and then immersed in 20xSSC for another 10 minutes), and two-ply filter papers were placed in order on the base, on which a paper towel with a thickness of 3<br><br>
cm to 4 cm was further placed. A glass plate and then a light weight were placed on the product, followed by blotting for approximately 5 minutes. After confirming that no bubbles were entrapped between the gel and the membrane, blotting was performed for approximately 10 minutes. Following blotting, the membrane was subjected to UV treatment with a trans-illuminator, and then baked at 80°C for approximately 15 minutes to 30 minutes. After baking, hybridization (hybridization buffer composition: SxSSPE, 0.5% SDS, 5x Denharlts, solum sperm DNA, 50% formamide) was performed with the above 250 bp probe DNA labeled with P. Radiation of the hybridized band was transferred to an imaging plate, and the result was analyzed with BAS-2000. Among inserts positive in hybridization, those showing a relatively large size were prepared in large quantity, and then sub-cloned into pBluescript II SK+ that had been digested with Eco RI and then treated with BAP by the above method. The product was transformed into E. coli (JM 105). The obtained transformants were subjected to liquid culture, and then plasmids were prepared by standard techniques. Thus, the nucleotide sequence was determined by the above methods.<br>
As a result, the full-length ALS cDNA gene could be obtained from the culture cells of each wild type and Sr line. The results of homology comparisons between the wild type and the mutant ALS genes are shown in Figs. 19A, B and C. As shown in Figs. I9A, B, and C, compared to the wild type, 2-point mutations were observed in Sr line at 2 points, the 1643rd and 1880"\ from the first base A as the starting point of the transcription initiation codon ATG. In Sr line, the 1643rd G in the wild type was mutated to T (denoted as G1643T), and the 1880th G in the wild type was mutated to T (denoted as Gl 880T). When converted into amino acids, these mutations indicated that the mutant ALS protein of Sr line had an amino acid sequence wherein the 548th tryptophan in the wild type ALS protein was mutated to leucine (that is, "W548L mutation"), and the 627th serine in the wild type ALS protein was<br><br>
mutated to isoleucine (that is, "S627I mutation").<br>
(5) Subcloning of the wild type ALS cDNA cloned into pBluescript II SK+ into pGEX 2T<br>
After the pBluescript II SK+ plasmid having the full-length wild type ALS cDNA obtained in (4) above incorporated therein was digested with Eco RI, cDNA containing the wild type ALS gene was excised. Then, the cDNA was incorporated into Eco RI site of pGEX-2T (Amersham Bioscience), which is an E. coli expression vector. Hereinafter, an expression vector having the full-length wild type ALS cDNA incorporated into the Eco RI site of pGEX-2T is referred to as "pGEX-2T(ALS-wild)." pGEX-2T(ALS-wild) was transformed into E. coli (JM 109). Colonies obtained by transformation were liquid-cultured, plasmids were extracted, and then the insertion direction of insert DNA was confirmed by sequencing. Thus, E. coli (JM109) transformed with pGEX-2T(ALS-wild) was prepared.<br>
[Example 4] Elucidation of mutation sites in ALS gene of PC herbicide resistant rice culture cell<br>
(1)	Extraction of genomic DNA from resistant mutant (strains of Sr, Rb, Vg,<br>
and Ga lines)<br>
Using a plant DNA extraction kit ISOPLANT II (Nippon Gene), genomic DNA was extracted from 0.1 g of cultured cells of each of Sr, Rb, Vg and Ga lines according to the protocols attached to the kit. After genomic DNA was extracted using the above kit, RNA was denatured and removed using RNase A. Then, agarose gel electrophoresis was performed, thereby confirming the genomic DNA.<br>
(2)	PCR of ALS gene using genomic DNA as template<br>
PCR was performed using each genomic DNA as a template, and a primer  "ALS-Rsp3"  and  a  primer  "4-83-3,"  as  shown below.     PCR  was<br><br>
performed using Ready to Go PCR Beads (Amersham Bioscience) at a final volume of 25 |al. The reaction was performed for 40 cycles, each cycle condition consisting of an initial denaturation step at 94°C for 5 minutes, followed by a denaturation step at 94°C for 30 seconds, annealing step at 55°C for 1 minute, and elongation step at 72°C for 2 minutes, i In addition, the elongation step in the final cycle was performed at 72°C for 9 minutes.<br>
Next, the PCR reaction solution was subjected to 2% agarose gel electrophoresis (lOOV, 1 X TBE buffer). Gels containing PCR products were excised, and then excised gels were cut into small fragments. The obtained gel fragments were rinsed twice or three times with sterile ion exchanged water, 500 |j,l of sterile ion exchanged water was added, and then freezing and dissolving was repeated three times. Thus, the PCR product could be eluted in water.<br>
Next, PCR was performed again using the eluate in which the PCR product had been dissolved. Specifically, this PCR was performed at a final volume of 100 ^il using the PCR product contained in the solution as a template, and the same primer set or nested primers. After reaction, the reaction solution was subjected to agarose gel electrophoresis (1%) for apportioning. Gels containing target bands were excised, and then purified using a GFX PCR DNA &amp; Gel Band Purification Kit (Amersham Bioscience). Finally, the PCR product was eluted using 75 fil of sterile deionized water. (3) Sequencing<br>
Sequence reaction was performed using the DNA fragment amplified by PCR as a template and ABI PRISM BigDye ver.2 (Applied Biosystem). For sequence reaction, 11 ^1 of the template DNA, 1 |al of the primer (3.2 pmol/|il) and 8 \i\ of pre-mix was mixed, therefore the total volume was 20 \x\. The sequence reaction was performed for 40 cycles, each cycle condition consisting of an initial denaturation step at 96°C for 5 minutes, followed by a denaturation step at 96°C for 5 seconds, annealing step at 50°C for 5 seconds, and elongation<br><br>
step at 60°C for 4 minutes.    In addition, the elongation step of the final cycle was performed at 60°C for 9 minutes.    After sequence reaction, fluorescent nucleotides  in  the  reaction  solution  were  removed by  gel  filtration  using. AutoSeq G-50 column (Amersham Biotech).    Then the nucleotide sequences were read using ABI PRISM 310 DNA sequencer. (4) Names of primers and nucleotide sequences used herein<br>
Names, nucleotide sequences and the like of primers used in (2) above and of primers used in the following examples are listed in Table 6.<br><br>
Table 6<br><br>
Name<br><br>
Nucleotide sequence<br><br>
Corresponding Number Direction       ALS site      of bases<br><br><br><br>
ALS-Rsp]   5"-GCTCTGCTACAACAGAGCACA-3"<br>
ALS-Rsp2  5"-AGTCCTGCCATCACCATCCAG-3"<br>
ALS-Rsp3   5"-CTGGGACACCTCGATGAAT-3"<br>
ALS-Rsp4  5"-CAACAAACCAGCGCAATTCGTCACC-3"<br>
ALS-Rsp6  5 "-CATCACCAACCACCTCTT-3"<br>
ALS-Rsp7  5"-AACTGGGATACCAGTCAGCTC-3"<br>
ALS-RspA 5"-TGTGCTTGGTGATGGA-3"<br>
ALS-RspB 5"-TCAAGGACATGATCCTGGATGG-3"<br>
ALS-RspC 5"-CAGCGACGTGTTCGCCTA-3"<br>
ALS-RspD 5"-CCACCGACATAGAGAATC-3"<br>
ALS-RspF  5"-ACACGGACTGCAGGAATA-3"<br>
ALS-RspE 5"-TTACAAGGCGAATAGGGC-3"<br>
3-1-1	5"-GCATCTTCTTGATGGCG-3"<br>
3-1-2	5"-ATGCATGGCACGGTGTAC-3"<br>
3-1-3	5"-GATTGCCTCACCTTTCG-3"<br>
3-1-4	5"-AGGTGTCACAGTTGTTG-3"<br>
4-83-1	5"-AGAGGTGGTTGGTGATG-3"<br>
4-83-3	5"-GCTTTGCCAACATACAG-3"<br>
4-83-10       5"-CAGCCCAAATCCCATTG-3" 4-83-15       5"-ATGTACCCTGGTAGATTC-3" ALS-DG7   5"-GTITT(CT)GClTA(CT)CCIGG(ACGT)GG-3"<br><br><br>
sense	1192-1212	21 mer<br>
antisense	1906-1926	21 mer<br>
sense	720-738	19 mer<br>
antisense	862-886	25 mer<br>
sense	327-344	18 mer<br>
antisense	886-906	2] mer<br>
antisense	571-586	] 6 mer<br>
sense	1913-1944	]6 mer<br>
sense	258-275	16 mer<br>
antisense	828-845	18 mer<br>
antisense	1749-1766	18 mer<br>
sense	1656-1673	18 mer<br>
antisense	1791-1807	17 mer<br>
sense	973-990	18 mer<br>
antisense	1346-1362	17 mer<br>
sense	1506-1522	17 mer<br>
antisense	327-343	17 mer<br>
antisense	1944-1960	17 mer<br>
antisense	1457-1473	17 mer<br>
antisense	735-752	18 mer<br>
sense	265-284	20 mer<br><br>
In   Table   6,   the   corresponding   ALS   site   is   the   number   of   a corresponding base when a transcription initiation codon (ATG) is the starting<br><br>
point.    In addition, the nucleotide sequence of ALS-Rspl is shown in SEQ ID NO: 9, the nucleotide sequence of ALS-Rsp2 is shown in SEQ ID NO: 10, the nucleotide sequence of ALS-Rsp3 is shown in SEQ ID NO: 11, the nucleotide sequence of ALS-Rsp4 is shown in SEQ ID NO: 12, the nucleotide sequence of ALS-Rsp6 is shown in SEQ ID NO: 13, the nucleotide sequence of ALS-Rsp7 is shown in SEQ ID NO: 14, the nucleotide sequence of ALS-RspA is shown in SEQ ID NO: 15, the nucleotide sequence of ALS-RspB is shown in SEQ ID NO: 16, the nucleotide sequence of ALS-RspC is shown in SEQ ID NO: 17, the nucleotide sequence of ALS-RspD is shown in SEQ ID NO: 18, the nucleotide sequence of ALS-RspF is shown in SEQ ID NO: 19, the nucleotide sequence of ALS-RspE is shown in SEQ ID NO: 20, the nucleotide sequence of 3-1-1 is shown in SEQ ID NO: 21, the nucleotide sequence of 3-1-2 is shown in SEQ ID NO: 22, the nucleotide sequence of 3-1-3 is shown in SEQ ID NO: 23, the nucleotide sequence of 3-1-4 is shown in SEQ ID NO: 24, the nucleotide sequence of 4-83-1 is shown in SEQ ID NO: 25, the nucleotide sequence of 4-83-3 is shown in SEQ ID NO: 26, the nucleotide sequence of 4-83-10 is shown in SEQ ID NO: 27, the nucleotide sequence of 4-83-15 is shown in SEQ ID NO: 28, and the nucleotide sequence of ALS-DG7 is shown in SEQ ID NO: 29. (5) Mutations in each line revealed as a result of sequencing<br>
As a result of analysis of nucleotide sequences determined in (3) above, mutations in Rb, Vg, Ga, and Sr lines were revealed. The mutated points of each line are listed in Table 7.<br><br>
Table 7<br><br>
Mutant base<br>
Mutant amino<br>
acid	C512A P171H	C514A R172S	G1643T W548L	G1880T S627I<br>
Rb line	ohomo		o hetero	<br>
Vg line			0 hetero	<br>
Ga line	o homo or hetero	0<br>
homo or hetero	0 hetero	<br>
Sr line			o hetero	o hetero<br>
As shown in Table 7, in the nucleotide sequence of Rb line strain, the<br>
rd<br>
512"" C was mutated to A (homo), and the 1643"° G was mutated to T (hetero). This mean^ that at the amino acid level, the 171^" proline and the 548""" tryptophan (W) were mutated to histidine (H) and leucine (L), respectively. In the nucleotide sequence of Vg line strain, the 1643^"^ G was mutated to T (hetero), suggesting that at the amino acid level, the 548"^ tryptophan (W) was mutated to leucine (L). In the nucleotide sequence of Ga line strain, the 512""" and 514"^ C were mutated to A (homo or hetero) (these types differed depending on the PCR product obtained), and the 1643^"" G was mutated to T (hetero). This means that at the amino acid level, the 171^" proline (P), 172"*^ arginine (R) and 548"" tryptophan (W) were mutated to histidine (H), serine (S) and leucine (L), respectively. Further, in the nucleotide sequence of Sr line strain, the 1643"* and 1880"" G were mutated to T (hetero).<br>
When ALS genes were screened and isolated from the cDNA library of Sr line strain by the above method, not only a 2-point mutant gene, but also a gene of the wild type was isolated. Thus, it was assumed that at the genomic DNA level, heterologous mutation had occurred, and the results obtained by genome PCR also supported this assumption.<br>
As described above, in all the resistant mutants, the 548"" tryptophan (W) was mutated to leucine (L) (hetero), and Vg line had this mutation only. As   described   above,   Vg   line   strain   showed   sensitivity   up   to   10   \xM<br><br>
bispyribac-sodium, and Sr, Rb and Ga line strains showed the same up to 100 \xM bispyribac-sodium. Accordingly, it was suggested that the acquisition of resistance started from Vg line and branched into other lines and mutated, as the intensity of the selection pressure increased.<br>
[Example 5] Synthesis of ALS cDNAs independently having G1643T(W548L) mutation or G1880T(S627I) mutation, construction of pGEX 2T retaining the ALS cDNAs, and transformation of E. coli using the vector<br>
First, synthesis of ALS cDNAs independently having G1643T(W548L) mutation or Gl880T(S627I) mutation, and construction of pGEX 2T retaining the ALS cDNAs are described using Fig. 20.<br>
PCR was performed at a final reaction volume of 100 p,l using 1 ^1 (585 ng/fil and 554 ng/fil, respectively) of pBluescript JJ SK+(ALS-2 point mutant) or pBluescript II SK+(ALS-wild) as a template, and 1 ^il of LA Taq DNA polymerase (Takara). The reaction was performed for 25 cycles, each cycle condition consisting of 95°C for 30 seconds, 55°C for 30 seconds and 72°C for 2 minutes. Further, pBluescript II SK+(ALS-2 point mutant) contained 2-point mutant ALS gene, G1643T(W548L) and G1880T(S627I). pBluescript II SK+(ALS-wild) contained the wild type ALS gene having no mutation. For the PCR, a combination of ALS-Rsp6 and ALS-RspF primers and a combination of ALS-RspE and M13R primers were used. Names of fragments amplified using ALS genes as a template and the given combination of primers are listed in Table 8. In addition, primer M13R is an antisense primer in the vicinity of T3 promoter of pBluescript II SK+. Further, the nucleotide sequence of M13R is 5"-GGAAACAGCTATGACCATG-3" (SEQ ID NO: 30).<br><br>
Table 8<br><br>
	pBluescript II SK+(ALS-2 point mutant)	pBluescript II SK+(ALS-wild)<br>
ALS-Rsp6 ALS-RspF	PCR-1	PCR-3<br>
ALS-RspE M13R	PCR-2	PCR-4<br>
PCR-1, PCR-2, PCR-3 and PCR-4 obtained by PCR were respectively subjected to agarose gel electrophoresis for separation, and then the products were collected in a manner similar to the above method from the agarose gel, and then the products were eluted with 50 p.1 of sterilized water.<br>
Next, a set of PCR-1 and PCR-4, and a set of PCR-2 and PCR-3 were subjected to SPR (self polymerase reaction). SPR was performed by adding 23.5 ^1 of the set of PCR-1 and PCR-4, or the set of PCR-2 and PCR-3 and 1 ).il of LA Taq DNA polymerase to a final volume of 75 \x\, and by performing 25 times a cycle consisting of a denaturation step at 95 °C for 1 minute, annealing step at 55°C for 30 seconds, and elongation step at 72°C for 2 minutes. DNA fragments obtained by SPR using the set of PCR-1 and PCR-4 was regarded as SPR-1, and DNA fragments obtained by SPR using the set of PCR-2 and PCR-3 as SPR-2.<br>
Further, in this example, to secure a sufficient amount of SPR-1 and of SPR-2, PCR was respectively performed at a final reaction volume of 100 ^1 using purified SPR-1 or SPR-2 as a template, ALS-Rsp6 and M13R, and LA Taq DNA polymerase again. PCR in this case was performed by repeating 25 times a cycle consisting of a denaturation step at 95°C for 30 seconds, annealing step at 55°C for 30 seconds and elongation step at 72°C for 2 minutes. After PCR, the reaction solution was subjected to agarose gel electrophoresis. An approximately 2 kbp single band (PCR product) was collected from agarose gel, and then eluted with 100 pi of sterilized water.<br>
Next, SPR-1 and SPR-2 amplified by PCR were respectively digested<br><br>
with Ace \ and Eco RI, thereby obtaining SPR-1 (Ace \IEco Rl-digested fragment) and SPR-2 (Ace l/Eco Rl-digested fragment). Specifically, 50 p,l of the sterilized water (100 |.il in total) containing PCR product dissolved therein was mixed with 1 )-il of Aec I (12 u/|j,l) and 1 ij.1 of £co RI (12 u/jal) in the presence of 10 x M buffer (Takara), followed by incubation at a final volume of 60 111 at 37°C for 1 hour. Afterwards, the total volume of the reaction solution was subjected to agarose gel electrophoresis, and then a target 1.5 kbp fragment was collected using a GFX PCR and Gel Purification Kit. The collected 1.5 kbp fragment was eluted with 50 p,l of sterilized water, so that a solution containing SPR-1 (Ace l/Eco Rl-digested fragment) and a solution containing SPR-2 (Aec l/Eco Rl-digested fragment) were prepared.<br>
Meanwhile, 150 |J.1 of a protein expression vector having the wild type ALS gene incorporated therein, pGEX-2T(ALS-wild) plasmid (concentration of approximately 50 ng/nl), was mixed with 1 |al of ^cc I (12 u/^il, Takara) in the presence of 10 X M buffer, followed by incubation at 37°C for 2 hours.    After reaction,   a   linear   7.2   kbp   band   was   confirmed   by    1%    agarose   gel electrophoresis.    According to the protocols of GFX PCR and Gel Purification Kit, DNA corresponding to the 7.2 kbp band was collected from the agarose gel, and then the product was eluted with 180 \i\ of sterilized water.    89 ^1 of the eluted product was mixed with 10 |al of 10 x H buffer (Takai-a) and 1 pi of Eco RI (12 u/^1), and then allowed to react at 37""C for 1 minute, thereby partially digesting the thus collected DNA with Eco RI.    After reaction, 10 x loading buffer was added, and then 1.5% agarose gel electrophoresis was performed. 4.9 kbp, 5.7 kbp, and 6.5 kbp bands, and a 7.2 kbp band that was not cleaved at all appeared separately, and then the target 5.7 kbp band was excised from the gel.    An approximately 5.7 kbp DNA fragment contained in the excised gel was collected using GFX PCR and Gel Purification Kit, and then the product was eluted with 50 \x\ of sterilized water.<br>
Subsequently, 3  f.il  of fragments digested with Ace I and partially<br><br>
digested with Eco RI of the thus obtained pGEX-2T(ALS-wild) and 3 ^xl of SPR-1 {Ace \IEco Rl-digested fragment) or SPR-2 {Ace l/Eco Rl-digested fragment) were respectively allowed to react in 6 |.il of Takara ligation buffer (ver.2, solution I) at 16°C overnight.<br>
Then, the reaction solution was transformed into E. coli competent cells (strain JM109, Takara) according to the protocols attached thereto.    The cells were inoculated on LB medium containing 50 ppm of ampicillin, and then incubated at 37°C overnight.    As a result, several of the colonies that appeared were selected.    PCR was directly performed using the colonies as a template, and    the     set     of    ALS-RspE     described     in     Table     6     and     PGEX-3 (5"-CCGGGAGCTGCATGTGTCAGAGG-3":   SEQ   ID   NO:   31),   the   set   of PGEX-5  (5"-GGGCTGGCAAGCCACGTTTGGTG-3":  SEQ ID NO:  32) and PGEX-3, and the set of PGEX-5 and ALS-RspA described in Table 6.    In addition, PGEX-3 had a sequence the same as a part of an antisense strand located on the 3" side of pGEX-2T used as a vector.    PGEX-5 had a sequence the same as a part of a sense strand located on the 5" side of pGEX-2T used as a vector.    As the reaction condition for the ALS-RspE/PGEX-3 set, each 1   yu M primer and 1 PCR bead were dissolved in a total volume of 25   ii 1, and reaction was performed by repeating 40 times a cycle consisting of a denaturation step at 95 °C for 30 seconds, annealing step at 55""C for 1 minute, and elongation step at 72°C   for   2   minutes.       In   the   case   of   the   PGEX-5/PGEX-3    set   and PGEX-5/ALS-RspA set, DMSO with a final concentration of 5% was further added to the above solution, because of the presence, at an upstream portion, of a region having approximately 75% of GC content.    As a result of this PCR, insertion of a desired insert was confirmed.<br>
A colony for which the insertion of a desired insert had been confirmed was picked up, and then shake-cultured in LB liquid medium (3 ml each, 10 medias) containing 50 ppm of ampicillin at 37°C for 12 hours.    After culturing, plasmids were extracted (400 to 500  |.il) from the media using a plasmid<br><br>
extraction system (TOMY, DP-480), and then concentrated to approximately 200 jil by centrifugation. Then, the concentrate was purified and desalted using GFX PCR and Gel Purification Kit, and then finally eluted with approximately 130 i-il of sterilized water.<br>
Sequence reaction was performed using ABI PRISM BigDye ver. 2 for these plasmids, so that the nucleotide sequence of the insert in the plasmid was analyzed. For sequence reaction, the reaction solution was prepared to have a total volume of 20 ^1 by mixing 11 \xl of template DNA, 1 )j,l of primer (3.2 pmol/jil) and 8 |al of pre-mix. The sequence reaction was performed for 40 cycles, each cycle condition consisting of an initial denaturation step at 96°C for 5 minutes, denaturation step at 96°C for 5 seconds, annealing step at 50°C for 5 seconds, and elongation step at 60°C for 4 minutes, and the elongation step of the final cycle was performed at 60° for 9 minutes. After sequence reaction, fluorescent nucleotides in the reaction solution were removed by gel filtration using AutoSeq G-50 column, and then the nucleotide sequence was determined using ABI PRISM 310 DNA sequencer.<br>
In addition, for sequence reaction, of the primers described in Table 6, PGEX-5, ALS-RspC, ALS-Rsp3, ALS-Rspl, 3-1-4 and ALS-RspB were used as sense primers, and 4-83-3, PGEX-3, ALSRsp2, 4-83-10 and ALS-Rsp7 were used as antisense primers.<br>
As a result of analysis, it was confirmed that pGEX 2T vector comprising the mutant ALS gene with W548L mutation (described as "pGEX 2T(ALS-W548L mutant)" in Fig. 20) and pGEX 2T vector comprising the mutant ALS gene with S627I mutation (described as "pGEX 2T(ALS-S627I mutant)" in Fig. 20) were obtained. Subsequently, E. coli was transformed with these pGEX 2T(ALS-W548L mutant) and pGEX 2T(ALS-S627I mutant).<br>
[Example 6]    Synthesis of ALS cDNAs independently having C512A (P171H) mutation found by genome PCR for Rb line or C514A (R172S) mutation found<br><br>
by genome PCR for Ga line, construction of pGEX 2T retaining the ALS cDNAs, and transformation of E. coli with this vector<br>
First, the synthesis of ALS cDNAs independently having C512A (P171H) mutation and C514A (R172S) mutation, and construction of pGEX 2T retaining the ALS cDNAs are described using Figs. 21 and 22.<br>
To obtain C512A (P171H) mutant DNA fragment, PCR was performed using the genomic DNA of Rb line as a template and a primer set of ALS-Rsp6 and ALS-Rsp4 described in Table 6. Specifically, PCR was performed using Ready to Go PCR Beads by adding 5 ^il of the template genomic DNA and 1 ]i\ of each primer (25 pmol/|il) to a final volume of 25 |al. The reaction condition consisted of an initial denaturation step at 95°C for 5 minutes, followed by a cycle (repeated 40 times) of a denaturation step at 95°C for 30 seconds, annealing step at 55°C for 1 minute, and elongation step at 72°C for 2 minutes. In addition, the elongation step of the final cycle was performed at 72°C for 9 minutes.<br>
After PCR reaction, the reaction solution was subjected to 2% agarose gel electrophoresis, a band of the PCR product (described as "PCR-5" in Fig. 21) was excised from agarose gel, and then purified using GFX PCR DNA &amp; Gel Band Purification Kit. Next, the purified PCR-5 was incorporated into pT7Blue T-vector (Novagen), the vector (TA cloning vector) for cloning PCR product. Specifically, 1 )al of the purified PCR product was mixed with 1 pi of pT7 Blue T-vector (50 ng/|J.l), 3 |al of sterile deionized water and 5 pi of ligation buffer (ver 2, solution I, Takara Shuzo), and then allowed to react overnight at 16°C.<br>
After reaction, the total volume of the reaction solution was transformed into E. coli (strain JM109) according to standard methods. After culturing of E. coli on LB solid medium containing 50 ppm of ampicillin, the colonies having a target sequence was selected from the single colonies that appeared on the medium in a manner similar to Example 5.    The selected<br><br>
single colonies were shake-cultured in LB liquid culture solution (3 ml, 10 media) containing 50 ppni of anipicillin at 37°C for 12 hours. After culturing, plasmids were extracted (400 to 500 fil) using a plasmid extraction system (TOMY, DP-480). The plasmids were concentrated to approximately 200 p,I by centrifugation, purified and desalted using GFX PCR and Gel Purification Kit, and then eluted with approximately 80 )al of sterilized water.<br>
Fifty |j.l of the eluate was mixed with 1 jxl of Ace I (12 u/jil) and 1 \x] of Sma I (10 u/^1) in the presence of 10 ^1 of 10 X T buffer and 10 jil of 0.1% BSA to bring to a total volume of 100 fil, and then the mixture was incubated at 37°C for 2 hours. After reaction, the reaction solution was subjected to agarose gel electrophoresis, a target band was excised and collected, and then a DNA fragment was collected according to the protocols of GFX PCR and Gel Purification Kit. Thus, C512A (P171H) mutant DNA fragment having Sma I site and Ace I site on its termini was obtained.<br>
On the other hand, since C514A and C512A mutations are close to each<br>
other, a DNA fragment having C5I4A (R172S) mutation only cannot be<br>
obtained by PCR using the genomic DNA extracted from Gb line as a template.<br>
Thus, as shown in Fig. 21, a DNA fragment having C514A (R172S) mutation<br>
only was prepared using a pair of primers to which mutated points had been<br>
previously introduced. That is, PCR was respectively performed using as<br>
primers having mutated points introduced therein ALS-Ml<br>
(5"-CCCCAGCCGCATGATCGGCACCGACGCCTT-3": SEQ ID NO: 33,<br>
underlined	A	is	a	mutated	point)	and	ALS-M2<br>
(5"-CGGTGCCGATCATGCGGCTGGGGACCT-3": SEQ ID NO: 34, underlined T is a mutated point) and as a template pBluescript II SK+ having the wild type ALS cDNA incorporated therein; and using a primer set of ALS-Rsp6 and ALS-M2; and using a primer set of ALS-Ml and ALS-Rsp4. In addition, complementary portions are the nucleotide sequence (1^" to 23"* nucleotides) of ALS-Ml and that (1"" to 23"^ nucleotides) of ALS-M2.    When the primer set of<br><br>
ALS-Rsp6 and ALS-M2 were used, a DNA fragment described as "PCR-6" in Fig. 21 was amplified, and when the primer set of ALS-Ml and ALS-Rsp4 was used, a DNA fragment described as "PCR-7" in Fig. 21 was amplified.<br>
The reaction solution was prepared at the time of PCR by dissolving 1 \.i\ of LA Taq DNA polymerase (5 units/pl, TAKARA), 10 ^1 of 10 X LA buffer, 10 )il of 25 mM MgCla, 16 yi\ of dNTPs (consisting of 25 niM of dATP, dGTP, dCTP and dTTP, respectively), 1 \x\ of template DNA, and 4 ^il each of sense and antisense primers (25 pmol/^i], respectively) to a total volume of 100 /il. The reaction was performed by repeating 25 times a cycle consisting of an initial denaturation step at 95°C for 5 minutes, a denaturation step at 95°C for 30 seconds, annealing step at 55°C for 1 minute, and elongation step at 72°C for 2 minutes, and the elongation step in the final cycle was performed at 72°C for 9 minutes.<br>
After reaction, the reaction solution was subjected to 1.5% agarose gel electrophoresis for apportioning, target 213 bp (PCR-6) and 377 bp (PCR-7) bands were excised and purified using GFX PCR DNA &amp; Gel Band Purification Kit, and then the thus generated DNA fragments were respectively eluted with 100 ^1 of sterile deionized water.<br>
Next, SPR was performed using the thus obtained PCR-6 and PCR-7. At the time of SPR, a reaction solution was prepared to a total volume of 100 jal by mixing 30 fil of the thus obtained eluate with 1 \i] of LA- Taq DNA polymerase (5 units/|il), 10 pi of 10 X LA buffer, 10 \i] of 25 mM MgCl2, and 16 ii\ of dNTPs (consisting of 25 mM of dATP, dGTP, dCTP and dTTP, respectively). SPR was performed by repeating 40 times a cycle consisting of an initial denaturation step at 95°C for 5 minutes, a denaturation step at 95°C for 30 seconds, annealing step at 55°C for 1 minute, and elongation step at 72°C for 2 minutes, and the elongation step in the final cycle was performed at 72°C for 9 minutes.<br>
After reaction,  the reaction  solution was  subjected to  agarose  gel<br><br>
(1.5%) electrophoresis for apportioning, a target 560 bp band (described as "SPR-3" in Fig. 21) was excised and purified using GFX PCR DNA &amp; Gel Band Purification Kit, and then the generated DNA fragment (SPR-3) was eluted with 100 \i\ of sterile deionized water. In a manner similar to the above method, the eluted fragment was incorporated into pT7Blue T-vector and then transformed into E. coli (JM109). The E. coli was cultured, and then the thus extracted plasmid was digested with Ace I and Sma I, thereby obtaining C514A (R172S) mutant DNA fragment having Sma I site and Ace I site at its termini.<br>
Meanwhile, E. coli (strain JM109) transformed with pGEX-2T(ALS-wild), the plasmid having the wild type ALS gene incorporated therein, was shake-cultured in LB liquid medium containing 50 ppm of ampicillin (2 ml x 15 media) overnight at 37°C. After the plasmid was extracted using a plasmid extraction system (DP-480), the extract (approximately 750 jil) was concentrated to approximately 200 \x\ using a vacuum centrifugation concentrator. Then, the concentrate was desalted using GFX PCR DNA &amp; Gel Band Purification Kit, and then the plasmid was finally eluted with 200 ^il of sterile deionized water.<br>
Next, the thus obtained plasmid, pGEX-2T(ALS-wild), was digested with Ace \. Specifically, 75 pi of the eluate was mixed with 9 |-il of 10 X M buffer, 3pl of Ace I (12u/pl), and 3 \x\ of sterile deionized water, and then the mixture was allowed to react at 37°C for 3 hours. After reaction, the reaction solution was subjected to 1.5% agarose gel electrophoresis for apportioning, the target band was excised and collected, and then purified using GFX PCR DNA &amp; Gel Band Purification Kit, and then a DNA fragment was finally eluted with 100 i-il of sterile deionized water.<br>
Subsequently, pGEX-2T(ALS-wild) digested with Ace I was partially digested with Sma \. Specifically, 79 pi of the eluate was mixed with 10 pi of 10 X T buffer, 10 ^il of 0.1% BSA, and 1 pi of Sma I (lOu/pl) to a total volume of 100 p], and then the mixture was incubated at 30°C for 1  minute.    In<br><br>
addition, since pGEX-2T(ALS-wi]d) contained Sma I recognition sequences (on the multicloning site adjacent to Thrombin cleavage site of pGEX-2T, 276"" and 43O"" sequences of ALS gene) located at three positions separately, partial digestion was performed in a short time. After reaction, the reaction solution was subjected to agarose gel electrophoresis, a band corresponding to the plasmid wherein only the 430"" Sma I recognition sequence of ALS gene had been digested was excised and collected, and then purified using GFX PCR DNA &amp; Gel Band Purification Kit to remove enzyme and protein. Finally, the purified product was eluted with 50 i^l of sterile deionized water. This Ace l-digested/Sma I partially-digested pGEX-2T-wild type ALS cDNA fragment, C512A(P171H) mutant DNA fragment having Srita I site and Ace 1 site on its termini obtained by the above method, and C514A(R172S) mutant DNA fragment were ligated by a standard method. In Fig. 22, a plasmid containing a mutant ALS gene independently having only C512A(P171H) mutation obtained by the method is described as "pGEX-2T(ALS P171H mutant)," and a plasmid containing a mutant ALS gene independently having only C514A(R172S) mutation is described as "pGEX-2T(ALS R172S mutant)."<br>
After that, E. coli (strain JM 109) was transformed using a total volume of the reaction solution. Single colonies that appeared on LB media containing ampicillin were screened by PCR in a manner similar to the above method, so that E. coli transformed with pGEX-2T(ALS P171H mutant) and E. coli transformed with pGEX-2T(ALS R172S mutant) were selected.<br>
[Example 7] Synthesis of 2-point mutant (C512A(P171H)/C514A(RI72S))ALS cDNA, construction of pGEX-2T retaining the ALS cDNA, and transformation of E. coli using this vector<br>
Synthesis of 2-point mutant (C512A(P171H)/C514A(R172S))ALS cDNA, and construction of pGEX-2T retaining the ALS cDNA are described using Fig. 23.<br><br>
2-point mutant (C512A(P171H)/C514A(R172S))ALS cDNA was synthesized by PCR using as a template the genomic DNA extracted from Ga line, according to the method described in Example 6 above. Specifically, PCR was performed using as a template the genomic DNA extracted from Ga line, and a primer set of ALS-Rsp6 and ALS-Rsp4, thereby amplifying a DNA fragment described as "PCR-8" in Fig. 23. Then, the amplified DNA fragment was ligated into pT7Blue T-vector, followed by digestion with Ace I and Sma I, thereby obtaining C512A(P17]H)/C514A(A172S) mutant DNA fragment. Next, as shown in Fig. 22, Ace 1-digested/Sma I partially-digested pGEX-2T-wild type ALS cDNA fragment and C512A(P171H)/C514A(R172S) mutant DNA were ligated by a standard method. Thus, pGEX-2T(ALS P171H, R172S mutant) was constructed. Further, similar to Example 6, E. coli transformed with pGEX-2T(ALS P171H, R172S mutant) was prepared.<br>
[Example 8] Synthesis of 2-point mutant (C512A(P171H)/G1643T(W548L) and C512A(P17IH)/G1880T(S627I))ALS cDNA, construction of pGEX-2T retaining the ALS cDNA, and transformation of £■. coli with this vector<br>
Synthesis of 2-point mutant (C512A(PI71H)/GI643T(W548L) and C512A(P171H)/G1880T(S627I))ALS cDNA, and construction of pGEX-2T retaining the ALS cDNA are described using Fig. 24.<br>
First, pGEX 2T(ALS-W548L mutant) obtained in Example 5 v/as digested with Ace I and then partially digested with Sma I according to the method of Example 6, so as to cause deletion of a portion from the 43 O"" Sma I recognition sequence to Aec I recognition sequence of ALS gene. Next, this product and C512A(P171H) mutant fragment prepared in Example 6 were ligated, so that a plasmid (described as pGEX-2T(ALS-P171H, W548L mutant) in Fig. 24), containing 2-point mutant (C512A(P171H)/G1643T(W548L)) ALS cDNA was constructed.<br>
Meanwhile, using pGEX 2T(ALS-S627I mutant) obtained in Example 5,<br><br>
instead of pGEX 2T(ALS-W548L mutant), a plasmid (described as "pGEX-2T(ALS-P171H, S627I mutant)" in Fig. 24) containing 2-point mutant (C512A(P171H)/G1880T(S627I)) ALS cDNA was constructed similarly.<br>
Further, in a manner similar to the method of Example 6, E. coli was transformed using these pGEX-2T(ALS-P171H, W548L mutant) and pGEX-2T(ALS-P171H, S627I mutant).<br>
[Example	9]	Synthesis	of	3-point	mutant<br>
(C512A(P171H)/G1643T(W548L)/G1880T(S627I)) ALS cDNA, construction of pGEX-2T retaining the ALS cDNA, and transformation of E. coli with this vector<br>
Synthesis	of	3-point	mutant<br>
(C512A(P171H)/G1643T(W548L)/G1880T(S627I)) ALS cDNA, and construction of pGEX-2T retaining this cDNA are described using Fig. 25.<br>
First, after pGEX 2T(ALS-S627I mutant) obtained in Example 5 was digested with Xho I, BAP treatment was performed according to a standard method. Next, according to the above method, a target gene fragment (on the vector side) was separated and purified from agarose gel. Further, pGEX 2T(ALS-W548L mutant) obtained in Example 5 was digested with Xho I, and then a fragment containing the mutation was separated and purified from agarose gel according to the above method.<br>
Next, to construct "pGEX-2T(ALS-W548L, S627I mutant)" having 2-point mutation, G1880T(S627I) and G1643T(W548L), the obtained DNA fragments were respectively subjected to ligation reaction. After reaction, the total volume of the reaction solution was transformed into E. coli (strain JM109). Single colonies that appeared on LB media containing ampicillin were screened by PCR according to the above method, and then E. coli having a target plasmid (pGEX-2T(ALS-W548L, S627I mutant)) was selected.<br>
After culturing the  selected E.   coli, pGEX-2T(ALS-W548L,  S627I<br><br>
mutant) was constructed according to the above method. pGEX-2T(ALS-W548L, S627I mutant) was digested with Aee 1, and then partially digested with Sma I, thereby constructing pGEX-2T(ALS-W548L, S627I mutant) wherein a portion from the 43O"^ Sma I recognition sequence to Ace I recognition sequence in ALS gene had been deleted. Subsequently, ligation of this pGEX-2T and C512A(P171H) mutant fragment prepared in Example 6 was performed, thereby constructing a plasmid containing 3-point mutant (C512A(P171H)/G1643T(W548L)/G1880T(S627I)) ALS cDNA (described as "pGEX-2T(ALS-P171H, W548L, S627I mutant" in Fig. 25).<br>
Further, E. coli was transformed using pGEX-2T(ALS-P171H, W548L, S627I mutant) in a manner similar to the method of Example 6.<br>
[Example 10] Expression of mutant ALS protein<br>
E. coli transformed with pGEX-2T(ALS-wiId) constructed in Example 3(5), E. coli transformed with pGEX-2T(ALS-W548L mutant) constructed in Example 5, E. coli transformed with pGEX-2T(ALS-S627I mutant) constructed in   Example   5,   E.   coli   transformed   with   pGEX-2T(ALS   P171H   mutant) constructed in Example 6, E.  coli transformed with pGEX-2T(ALS R172S mutant) constructed in Example 6, E.  coli transformed with pGEX-2T(ALS P171H, R172S mutant) constructed in Example 7, E.  coli transformed with pGEX-2T(ALS-P171H,   W548L  mutant)  constructed  in  Example  8,  E.   coli transformed with pGEX-2T(ALS-P171H, S627I mutant) constructed in Example 8, and E. coli transformed with pGEX-2T(ALS-P171H, W548L, S627I mutant) constructed in Example 9 were respectively shake-cultured (pre-culture) at 27°C in 2 ml of LB liquid medium containing ampicillin.    These types of E. coli were respectively cultured in 250 ml of LB liquid medium containing ampicillin using 1 ml of the pre-culture solution.    After culturing overnight, 1 mM IPTG was added to the media, and then culturing was performed for a further 3 to 4 hours, so that the expression of GST fusion protein was induced.    In addition.<br><br>
the cells were stored at -8G°C after washing.<br>
Preparation and purification of ALS from E. coli were performed by the following method. First, the pellet of the transformant E. coli stored at -80°C was suspended in ALS extraction buffer (potassium phosphate buffer (pH 7.5) containing 30 % glycerol and 0.5 niM MgCb). Specifically, 2.5 ml of the buffer was added to the pellet obtained from 50 ml of the culture solution. The suspension was subjected to ultrasonication (Heat Systems-Ultrasonics, Sonicator W-225R, micro chip, output control 8, interval of approximately 1 second, twice (40 seconds each)), and subjected to centrifugation at 15000 x g, 4°C for 20 minutes, thereby obtaining the supernatant as a crude enzyme solution.<br>
Thus, 9 types of crude enzyme solutions containing any one of GST fusion wild type ALS protein, GST fusion W548L mutant ALS protein, GST fusion S627I mutant ALS protein, GST fusion P171H mutant ALS protein, GST fusion R172S mutant ALS protein, GST fusion P171H/R172S mutant ALS protein, GST fusion P171H/W548L mutant ALS protein, GST fusion P171H/S 6271 mutant ALS protein and GST fusion P171H/W548L/S627I mutant ALS protein were prepared.<br>
[Example 11] Herbicide sensitivity of mutant ALS protein<br>
Herbicide sensitivity of the wild type ALS protein and that of mutant<br>
ALS protein were examined using the 9 types of crude enzyme solutions<br>
obtained in Example 10. Herbicide sensitivity test was performed according to<br>
procedures almost the same as those in Example 2. However, in this example,<br>
reaction temperature was 37°C, reaction time was 30 minutes, and 10 mM<br>
valine was added to the reaction solution to inhibit ALS activity derived from E.<br>
coli.	Further,      three      types      of     herbicides,      bispyribac-sodium,<br>
pyrithiobac-sodium,    and    pyriminobac,    were    used    as    PC    herbicides; chlorsulfuron was used as a sulfonylurea herbicide; and imazaquin was used as<br><br>
an imidazolinon herbicide. Before the addition of mutant ALS protein, the solutions of these herbicides (aqueous solutions for bispyribac-sodium and pyrithiobac-sodium, and acetone solutions for other herbicides) at a certain concentration were added into the reaction solutions. The final concentration of acetone was 1%.<br>
For the 9 types of crude enzyme solutions, inhibition activity by bispyribac-sodium is shown in Figs. 26 and 27, and Table 9, inhibition activity by pyrithiobac-sodium is shown in Table 10, inhibition activity by pyriminobac is shown in Table 11, inhibition activity by chlorsulfuron is shown in Table 12, and inhibition activity by imazaquin is shown in Table 13.<br>
In Tables 9 to 13, inhibition activity by each herbicide is represented by a  herbicide  concentration  (150)  which   causes   50%  inhibition,   when   50% inhibition   is   obtained   at   a   concentration   tested,   and   is   represented   by inhibition % at the highest concentration among the concentrations tested, when 50%) inhibition could not be obtained.    Further, in Tables 9 to 13, predicted RS ratio refers to the RS ratio of a mutant ALS protein having multiple mutations, which is a combined RS ratio normally predicted from each RS ratio of mutant ALS proteins independently having a mutation.    That is, the predicted RS ratio refers to a synergistic effect normally predicted from a combined RS ratio of mutant  ALS   proteins  independently  having   a  mutation.     Specifically,  the predicted RS ratio of a mutant ALS protein having multiple mutations v,?as calculated by selecting RS ratios (for all the mutations corresponding to the multiple mutations of this protein) of mutant ALS proteins respectively having only one of the mutations, and then multiplying the selected RS ratios.    When an actual RS ratio exceeds the predicted RS ratio of a mutant ALS protein having multiple mutations, this protein has resistance exceeding the synergistic effect (resistance) predicted from a combined resistance of mutant ALS proteins independently having a mutation.<br><br>
Table 9<br><br>
ALS protein type	150 (i-iM)	RS ratio	Predicted RS ratio	RS ratio/predicted RS ratio<br>
Wild type	0.0063			<br>
PI 71H mutant	0.055	8.7		<br>
R172S mutant	0.0062	0.98		<br>
W548L mutant	3.3	520		<br>
S627I mutant	0.26	41		<br>
PI71H/R172S mutant	0.048	7.6	8.5	0.89<br>
PI71H/W548L mutant	5.5% in IOOMM	&gt;15000	4500	&gt;3.3<br>
PI 7IH/S 62 71 mutant	23	3700	360	10<br>
P171H/W548L/S627I	1.1% in lOO^M	&gt;16000	190000	&gt;0.084<br>
mutant				<br>
Table  10				<br>
ALS protein type	150 (laM)	RS ratio	Predicted RS ratio	RS ratio/predicted RS ratio<br>
Wild type	0.011			<br>
P171H mutant	0.037	3.4		<br>
R172S mutant	0.011	1		<br>
W548L mutant	41% in lOO^M	&gt;9100		<br>
S627I mutant	2.2	200		<br>
P171H/R172Smutant	0.14	13	3.4	3.8<br>
P171H/W548Lmutant	20% in 1 OOfiM	&gt;9100	&gt;31000	<br>
P171H/S627Imutant	9.4	850	680	1.3<br><br>
Table  11<br><br>
ALS protein type	150 (nM)	RS ratio	Predicted RS ratio	RS ratio/predicted RS ratio<br>
Wild type	0.008			<br>
PI71H mutant	0.04	5		<br>
R172S mutant	0.0092	1.2		<br>
W548L mutant	36	4500		<br>
S627I mutant	22	2800		<br>
P171H/R172Smutant	0.041	5.1	6	0.85<br>
PI71H/W548L mutant	11% in lOOpiM	&gt;13000	23000	&gt;0.57<br>
PI 7IH/S 62 71 mutant	21% in lOOiLiM	&gt;13000	14000	&gt;0.93<br>
Table  12				<br>
ALS protein type	150 (laM)	RS ratio	Predicted RS ratio	RS<br>
ratio/predicted RS ratio<br>
Wild type	0.013			<br>
PI7IH mutant	1.1	85		<br>
R172S mutant	0.011	0.85		<br>
W548L mutant	9.9	760		<br>
S627I mutant	0.031	2.4		<br>
P171H/R172Smutant	5.5	420	72	5.8<br>
P171H/W548L mutant	16%inl00^iM      &gt;7700	65000	&gt;0.18<br>
PI 7IH/S 62 71 mutant	9.9	760	200	3.8<br>
P171H/W548L/S627I	30% in 500^M     &gt;38000	160000	&gt;0.24<br>
mutant				<br><br>
Table  13<br><br>
ALS protein type	150 (pM)	RS ratio	Predicted RS ratio	RS<br>
ratio/predicted RS ratio<br>
Wild type	2.2			<br>
PI71H mutant	3.4	1.5		<br>
R172S mutant	2.3	1		<br>
W548L mutant	16% in lOOiaM	&gt;45		<br>
S627I mutant	15	6.8		<br>
P171H/R172Smutant	3.9	1.8	1.5	1.2<br>
P171H/W548Lmutant	13% in 100)iM	&gt;45	&gt;68	<br>
P171H/S627I mutant	71	32	10	3.2<br>
P171H/W548L/S627I	15% in lOO^M	&gt;45	&gt;460	<br>
mutant				<br>
Data of the above Tables 9 to 13 are described below in order.<br>
First,   data   of  inhibition   activity   by   bispyribac-sodium   (Table   9) revealed the following:<br>
Among mutant ALS protein coded by the 1-point mutant genes (P171H, R172S, W548L and S627I), W548L mutant ALS protein showed the highest resistance to bispyribac-sodium (RS ratio: 520). S627I mutant ALS protein or P171H mutant ALS protein also showed high resistance (RS ratio: 41 and 8.7, respectively), but R172S mutant ALS protein showed resistance only equivalent to that of wild type ALS protein (RS ratio: 0.98). These results revealed that P171H mutation, W548L mutation and S627I mutation in ALS protein are mutations effective in enhancing resistance to bispyribac-sodium. Further, R172S mutation in ALS protein was shown to be a silent mutation.<br>
On the other hand, among mutant ALS proteins coded by the 2-point mutant genes, P171H/W548L mutant ALS protein showed the strongest resistance to bispyribac-sodium (5.5% inhibition in 100 [iM, and RS ratio: &gt;15000). P171H/S627I mutant ALS protein also showed strong resistance to bispyribac-sodium (RS ratio: 3700). The degree of resistance of P171H/R172S mutant ALS protein was approximately the same as P171H mutant ALS protein.<br><br>
Further, P171H/W548L/S6271 mutant ALS protein coded by the 3-point mutant gene also imparted strong resistance to bispyribac-sodium (1.1% inhibition when lOO^iM, and RS ratio: &gt;16000). In addition, actual results of herbicide dose-response on which these results were based are shown in Figs. 26 and 27.<br>
For the 2-point and 3-point mutations, the predicted RS ratios and actual RS ratios were compared. RS ratios of P171H/W548L mutant ALS protein and P171H/S627I mutant ALS protein were significantly higher than the predicted RS ratios (the ratio of the RS ratio to the predicted RS ratio was remarkably larger than 1). These results revealed that these two 2-point mutant genes (the gene coding for P171H/W548L mutant ALS protein, and the gene coding for P171H/S627I mutant ALS protein) impart resistance against bispyribac-sodium to ALS protein which is stronger than an additive effect predicted from the degree of each resistance of the 1-point mutant gene.<br>
Next, inhibition activity by pyrithiobac-sodium (Table 10) revealed the following:<br>
Among mutant ALS proteins (P171H, R172S, W548L and S627I) coded by 1-point mutant genes, W548L mutant ALS protein showed the strongest resistance to pyrithiobac-sodium (41% in 100 \xM, and RS ratio: &gt;9100). S627I mutant ALS protein also showed resistance (RS ratio: 200), but the degree of the resistance of P171H mutant ALS protein was low (RS ratio: 3.4). R172S mutant ALS protein showed resistance only equivalent to that of the wild type ALS protein (RS ratio: 0.85). These results revealed that P171|i mutation, W548L mutation and S627I mutation in ALS proteins are effective mutations in enhancing resistance to pyrithiobac-sodium. Further, R172S mutation in ALS protein was shown to be a silent mutation.<br>
On the other hand, among the mutant ALS proteins coded by 2-point mutant genes, P171H/W548L mutant ALS protein imparted the strongest resistance (20% inhibition in 100 )aM, and RS ratio: &gt;9100), followed by P171H/S6271  mutant  ALS  protein  (RS   ratio:   850).     Unlike  the   data  of<br><br>
inhibition activity by bispyribac-sodium shown in Table 9, in the case of pyrithiobac-sodium, P17IH/R172S mutant ALS protein showed a degree of resistance higher than that of P171H mutant ALS protein (RS ratio: 13). Thus, it was clarified that R172S mutation, which is a silent mutation by itself, enhances the degree of resistance of P171H mutant ALS protein.<br>
Further, for 2-point mutant ALS proteins, when a combined RS ratio predicted from each RS ratio of 1-point mutant ALS proteins and the actual RS ratio were compared, it was found that the RS ratio of P171H/R172S mutant ALS protein was significantly higher than that of the predicted RS ratio (the ratio of the actual RS ratio to the predicted RS ratio was remarkably larger than 1). These results revealed that P171H/R172S mutant ALS protein showed resistance to pyrithiobac-sodium stronger than that predicted from the degrees of resistances of the 1-point mutant genes.<br>
Next, inhibition activity by pyriminobac (Table 11) revealed the following:<br>
Among mutant ALS proteins coded by 1-point mutant genes (P171H, R172S, W548L and S627I), W548L mutant ALS protein showed the strongest resistance to pyriminobac (RS ratio: 4500). S627I mutant ALS protein also imparted strong resistance (RS ratio: 2800), but the degree of resistance of P171H mutant ALS protein was low (RS ratio: 5). R172S mutant ALS protein showed resistance only equivalent to that of the wild type ALS protein (RS ratio: 1.2). These results revealed that P171H mutation, W548L mutation and S627I mutation in ALS proteins are mutations effective in enhancing resistance to pyriminobac. Further, R172S mutation in ALS protein was shown to be a silent mutation.<br>
Among the mutant ALS proteins coded by the 2-point mutant genes, P171H/W548L mutant ALS protein imparted the strongest resistance (11% inhibition in 100 jiM, and RS ratio: &gt;13000), followed by P171H/S627I mutant ALS protein (21% inhibition when 100 }.iM, and RS ratio: &gt;13000).    For these<br><br>
P171H/W548L mutant ALS and P171H/S627I mutant ALS proteins, predicted RS ratios and actual RS ratios were compared. However, it could not be clarified whether resistance stronger than the resistance predicted from the degrees of resistances of each 1-point mutant gene is shown.<br>
Next,  inhibition  activity  by   chlorsulfuron  (Table   12)   revealed  the following:<br>
Among the mutant ALS proteins coded by 1-point mutant genes (P171H, R172S, W548L and S627I), W548L mutant ALS protein showed the strongest resistance to chlorsulfuron (RS ratio: 760). P171H mutant ALS protein showed relatively strong resistance (RS ratio: 85), but the degree of resistance of S627I mutant ALS protein was low (RS ratio: 2.4). R172S mutant ALS protein showed resistance only equivalent to that of the wild type ALS protein (RS ratio: 0.85). These results revealed that P171H mutation and W548L mutation in ALS protein are mutations effective in enhancing resistance to chlorsulfuron. Further, R172S mutation in ALS protein was shown to be a silent mutation.<br>
Among the mutant ALS proteins coded by 2-point mutant genes, P171H/W548L mutant ALS protein imparted the strongest resistance (16% inhibition in 100 |aM, and RS ratio: &gt;7700), followed by P171H/S627I mutant ALS protein (RS ratio: 760). Unlike the data of inhibition activity by bispyribac-sodium shown in Table 9, in the case of chlorsulfuron, P171H/R172S mutant ALS protein showed a degree of resistance (RS ratio: 420) higher than that of PI71H mutant ALS protein. Thus, it was clarified that R172S mutation, which is a silent mutation by itself, enhances the degree of resistance of PI71H mutant ALS protein. Further, P171H/W548L/S627I mutant ALS protein also imparted strong resistance (30% inhibition in 500 p,M, and RS ratio: &gt;38000).<br>
For P171H/R172S mutant ALS and P171H/S627I mutant ALS proteins, predicted RS ratios and actual RS ratios were compared. For both proteins, the actual RS ratios were significantly higher than the predicted RS ratios. These results revealed that  P171H/R172S  mutant ALS  protein  and P171H/S627I<br><br>
mutant ALS  protein  showed resistance to  chlorsulfuron  stronger than that predicted from the degrees of resistances of each 1-point mutant gene.<br>
Next, data of inhibition activity by Imazaquin (Table 13) revealed the following:<br>
Among the mutant ALS proteins coded by 1-point mutant genes (P171H, R172S, W548L and S627I), W548L mutant ALS protein showed the strongest resistance to imazaquin (16% in 100 |aM, and RS ratio: &gt;45). S627I mutant ALS protein also showed resistance (RS ratio: 6.8), but P171H mutant ALS protein showed almost no resistance (RS ratio: 1.5). R172S mutant ALS protein showed resistance only equivalent to that of the wild type ALS protein (RS ratio: 1.0). These results revealed that W548L mutation and S627I mutation in ALS protein are mutations effective in enhancing resistance to imazaquin. Further, P171H mutation and R172S mutation in ALS protein were shown to be silent mutations against imazaquin.<br>
Among the 2-point mutant genes, P171H/W548L mutant ALS protein imparted the strongest resistance (13% inhibition in 100 )aM, and RS ratio: &gt;45), followed by P171H/S627I mutant ALS protein (RS ratio: 32). The degree of resistance of P171H/R172S mutant ALS protein was almost the same as that of pl71H 1-point mutant gene. Further, P171H/W548L/S627I mutant ALS protein also imparted strong resistance (15%) inhibition in 100 )iM, and RS ratio: &gt;45).<br>
For these 2-point ALS mutant proteins and 3-point ALS mutant protein, predicted RS ratios and actual RS ratios were compared. The RS ratio of P171H/S627I mutant ALS protein was significantly higher than the predicted RS ratio (the ratio of the actual RS ratio to the predicted RS ratio was clearly larger than I). These results revealed that P171H/S627I mutant ALS protein showed resistance to imazaquin stronger than that predicted from the degrees of resistances of each 1-point mutant gene. Industrial Applicability<br><br>
As described in detail above, the present invention can provide a gene coding for acetolactate synthase showing good resistance to various herbicides, an acetolactate synthase protein coded by the gene, a recombinant vector having the gene, a transformant having the recombinant vector, a plant having the gene, a method for rearing the plant, and a method for selecting a transformant cell using the gene as a selection marker.<br>
Sequence Listing Free Text<br>
SEQ ID NOS: 9 to 34 represent primers.<br>
The 15"" n in SEQ ID NO: 29 represents a, c, g or t.<br><br><br>
WE CLAIM:<br>
1.	A mutant ALS gene, which codes for the following protein (a) or (b):<br>
(a)	a protein consisting of an amino acid sequence of any one of SEQ ID NOS: 2, 4, 6 and 8;<br>
(b)	a protein consisting of an amino acid sequence derived from the amino acid sequence of any one of SEQ ID NOS: 2, 4, 6 and 8 by substitution, deletion or addition of at least one or more amino acids having resistance to a bispyribac sodium herbicide, a pyrithiobac sodium herbicide, and a pyriminobac herbicide, and has acetolactate synthase activity <br><br>
2.	An acetolactate synthase protein, which is coded by the gene as claimed in claim 1.<br>
3.	A recombinant vector, which has the gene as claimed in claim 1.<br>
4.	A microbial transformant, which has the recombinant vector as claimed in claim 3.<br><br>
5.	A method for selecting a transformant cell having the gene as claimed in claim 1, which comprises step of selecting a transformat cell that can live in the presence of a pyrimidinyl carboxy herbicide.<br>
6.	A method for selecting a transformant cell having as gene coding acetolactate synthase as a selection marker, which comprises cultivating the cell in the presence of a pyrithiobac sodium herbicide and/or pyriminobac herbicide, wherein the acetolactate synthase has an amino acid sequence in which a serine corresponding to serine at position 627 of a wild type rice acetolactate synthase, is replaced by isoleucine.<br>
7.	A mutant gene, which codes for a protein, which has an amino acid sequence in which<br>
a serine corresponding to serine at position 627 of a wild type rice acetolactate synthase, is<br>
replaced by isoleucine, and which exhibits resistance to a bispyribac sodium herbicide, a<br>
pyrithiobac sodium herbicide, and a pyriminobac herbicide.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQzMi1jaGVucC0yMDA0IGFic3RyYWN0LWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">2432-chenp-2004 abstract-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQzMi1jaGVucC0yMDA0IGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2432-chenp-2004 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQzMi1jaGVucC0yMDA0IGNsYWltcy1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">2432-chenp-2004 claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQzMi1jaGVucC0yMDA0IGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2432-chenp-2004 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQzMi1jaGVucC0yMDA0IGNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2432-chenp-2004 correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQzMi1jaGVucC0yMDA0IGNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">2432-chenp-2004 correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQzMi1jaGVucC0yMDA0IGRlc2NyaXB0aW9uIChjb21wbGV0ZSktZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">2432-chenp-2004 description (complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQzMi1jaGVucC0yMDA0IGRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">2432-chenp-2004 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQzMi1jaGVucC0yMDA0IGRyYXdpbmdzLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">2432-chenp-2004 drawings-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQzMi1jaGVucC0yMDA0IGRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">2432-chenp-2004 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQzMi1jaGVucC0yMDA0IGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">2432-chenp-2004 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQzMi1jaGVucC0yMDA0IGZvcm0tMTkucGRm" target="_blank" style="word-wrap:break-word;">2432-chenp-2004 form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQzMi1jaGVucC0yMDA0IGZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">2432-chenp-2004 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQzMi1jaGVucC0yMDA0IGZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">2432-chenp-2004 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQzMi1jaGVucC0yMDA0IGZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">2432-chenp-2004 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQzMi1jaGVucC0yMDA0IHBjdCBzZWFyY2ggcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">2432-chenp-2004 pct search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQzMi1jaGVucC0yMDA0IHBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">2432-chenp-2004 pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="218784-a-shape-memory-alloy-motor.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="218786-system-and-method-for-using-equipment-identity-information-in-providing-location-services-to-a-wireless-communication-device.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>218785</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2432/CHENP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>21/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>23-May-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>16-Apr-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>27-Oct-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NATIONAL INSTITUTE OF AGROBIOLOGICAL SCIENCES</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1-2, Kannondai 2-chome, Tsukuba-shi, Ibaraki 305-8602</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>KAKU, Koichiro</td>
											<td>276 - 1, Sunheights Hiruike C, Fukudecho, Iwata-gun, Shizuoka 437 - 1207,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SHIMIZU, Tsutomu</td>
											<td>114, Hirao, Kikugawa-cho, Ogasa-gun, Shizuoka 439 - 0035,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>KAWAI, Kiyoshi</td>
											<td>1570, Aino, Fukuroi-shi, Shizuoka 437 - 0031,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>FUKUDA, Atsunori</td>
											<td>2-11-802-404, Azuma, Tsukuba-shi, Ibaraki 305 - 0031,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>NAGAYAMA, Kozo</td>
											<td>2384 - 4, Yasaka, Kakegawa-shi, Shizuoka 436 - 0004,</td>
										</tr>
										<tr>
											<td>6</td>
											<td>TANAKA, Yoshiyuki</td>
											<td>2-3-2-708-305, Azuma, Tsukuba-shi, Ibaraki 305 - 0031,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/29</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/JP2003/001917</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-02-21</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2002/95721</td>
									<td>2002-03-29</td>
								    <td>Japan</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/218785-gene-coding-for-acetolactate-synthase by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:34:11 GMT -->
</html>
