<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/218809-a-recombinant-dna-coding-for-enone-reductase-and-microbial-production-of-levodione by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:32:34 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 218809:A RECOMBINANT DNA CODING FOR ENONE REDUCTASE AND MICROBIAL PRODUCTION OF LEVODIONE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A RECOMBINANT DNA CODING FOR ENONE REDUCTASE AND MICROBIAL PRODUCTION OF LEVODIONE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a recombinant DNA comprising a nucleotide sequence coding for an enzyme having enone reductase activity wherein the enzyme is characterized by the following physico-chemical properties: (a) molecular mass: 61,300 ± 5,000 Da (estimated using gel filtration, consisting of one subunit); (b) co-factor: NADPH and NADH; (c) substrate specificity: active on &amp;agr;,ß-unsaturated ketons; (d) optimum temperature: 55-60°C at pH 7.4; and (e) optimum pH: pH 4.5-8.5; and wherein said nucleotide sequence is selected from the group consisting of: (i) a nucleotide sequence represented in SEQ ID NO:1; (ii) a nucleotide sequence coding for an enone reductase having the amino acid sequence encoded by the nucleotide sequence shown in SEQ ID NO:1; (iii) a nucleotide sequencewhich hybridizes to the complement of the nucleotide sequence of (i)or (ii) under stringent hybridizing conditions; and (iv) a nucleotide sequence which is att least 80% identical to the nucleotide sequence of (i).</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present invention relates to a recombinant DNA encoding an enone reductase, an expression vector comprising the DNA, a microorganism into which the DNA has been introduced, and a method for producing (6R)-2,2,6-trim6thyl-l,4-cyclohexanedione (hereinafter referred to as levodione) from 2,6,6-trimethly-2-cyclohexene-l,4-dione (hereinafter referred to as ketoisophorone) using the microorganism.<br>
Levodione is a useful intermediate in the synthesis of optically active carotenoids such as zeaxanthin. A microbiological process of producing levodione from ketoisophorone is known (US 4,156,100). Enone reductase that acts on ketoisophorone to produce levodione, which was isolated from Candida kefyr was described in (European Patent Application No. 02003967.3 filed on Feb. 22,2002). This enzyme is characterized by the following physico-chemical properties:<br>
(a)	molecular mass: 61,300 ± 5,000 Da<br>
(Estimated using gel filtration. Consisting of one subunit.)<br>
(b)	Co-factor: NADPH and NADH<br>
(c)	Substrate specificity: active on a,P-unsaturated ketons<br>
(d)	Optimum temperature: 55-60°C at pH 7.4 (e) Optimum pH:pH 4.5-8.5<br>
As used herein, the term "enone reductase" encompasses proteins catalyzing the enzymatic reduction of carbonyl activated double bonds according to the Enzyme Nomenclature provided by the Nomenclature Committee of the International Union of Biochemistry and Molecular Biology (NC-IUBMB). It also relates to proteins having the above mentioned activities of an enone reductase, which proteins preferably catalyse the conversion of ketoisophorone into levodione. The<br><br>
gene for an enone reductase involved in the biosynthesis of levodione would be very useful for improvement of levodione productivity by a microorganism.<br>
The present invention provides to an recombinant DNA sequence encoding enone reductase.<br>
The recombinant DNA sequence may be more specifically characterized in that (a) it codes for the enzyme having the amino acid sequence described in SEQ ID N0;2, or (b) it codes for a variant of the enzyme selected from (i) an allelic variant, and (ii) an enzyme having one or more amino acid addition, insertion, deletion and/or substitution and having the stated enzyme activity.<br>
More particularly, the present invention provides an isolated DNA sequence derived from a gene of Candida kefyr (Candida macedoniensis) IFO 0960 and is selected from (i) the DNA sequence represented in SEQ ID NO;l, (ii) an isocoding or an allelic variant of the DNA sequence represented in SEQ ID NO:l, (iii) a derivative of the DNA sequence represented in SEQ ID NO:l, with addition, insertion, deletion and/or substitution of one or more nucleotide(s) and coding for a polypeptide having the enzyme activity, (iv) the DNA sequence which hybridizes to the complement of the nucleotide sequence of (i) or (Ii) under stringent hybridizing conditions and coding for a polypeptide having the enzyme activity, and (v) the DNA sequence which is at least 80% identical to the nucleotide sequence of (!) and coding for a polypeptide having the enzyme activity.<br><br>
The strain Candida kefyr (Candida macedoniensis) IFO 0960 is publicly available from the Institute for Fermentation Osaka (IFO), 17-85 Juso-honmachi 2-chome, Yodogawa-ku, Osaka, 532-8686, Japan.<br>
Instructions for identifying DNA sequences by means of hybridization are well-known to a person skilled in the art. The hybridization may take place under stringent conditions wherein only hybrids in which the probe and target sequence, i.e. the polynucleotides treated with the probe and are at least 70% identical, are formed. It is known that the stringency of the hybridization, including the washing steps, is influenced or determined by varying the buffer composition, the temperature and the salt concentration. The hybridization reaction is preferably carried out under a relatively low stringency compared with the washing steps. A 5 X SSC buffer at a temperature of approx. 50-68°C, for example, can be employed for the hybridization reaction. Probes can also hybridize here with polynucleotides that are less than 70% identical to the sequence of the probe. Such hybrids are less stable and are removed by washing under stringent conditions. This can be achieved, for example, by lowering the salt concentration to 2 x SSC and subsequently 0.5 x SSC at a temperature of approx. 50-68°C being established. It is optionally possible to lower the salt concentration to 0.1 x SSC. Polynucleotide fragment, for example, at least 70% or at least 80%, or at least 90% to 95% identical to the sequence of the probe employed can be isolated by increasing the hybridization temperature stepwise in steps of approx. 1-2°C. "Stringent conditions" in the context of this invention mean hybridization in a buffer, for example, consisting of 5 x SSC, 0.1% (w/v) N-lauroylsarcosine,<br><br>
0.02%(w/v) SDS, 1% blocking reagent (Roche Diagnostics, Cat. No. 1096 176) at 50°C overnight and two times of washing with 2 x SSC, 0.1% (w/v) SDS for 5 min. at room temperature and following two times of washing with 0.1 x SSC, 0.1% (w/v) SDS for 15 min. at 68°C in the washing step of hybridization. The DNA sequence may be cloned from a strain of C. kefyr (C macedoniensis) IFO 0960, or another or related organism and thus, for example, may be an allelic or species variant of an enone reductase encoding region of the DNA sequence. Also included within the scope of the present invention is a derivative of the DNA sequence with addition, insertion, deletion and/or substitution of different nucleotides resulting in a polypeptide that encodes the same or a functionally equivalent levodione reductase. The encoded protein may also contain addition, deletion, insertion and/or substitution of amino acid residues, which produce a silent change and result in a functionally equivalent enone reductase. The DNA of the present invention also means a genomic DNA that contains regulatory sequences such as a promoter and a terminator, which are involved in the expression of the gene of interest, and also a cDNA that contains only open reading frame flanked between the short fragments in its 5"- and 3"- untranslated region.<br>
The enone reductase gene, the recombinant expression vector, and the recombinant organisms utilized in the present invention may be obtained by the following steps: - Isolating chromosomal DNA from a microorganism that can provide enone reductase of the present invention and constructing the gene library with the chromosomal DNA.<br><br>
-	Cloning an enone reductase gene from the chromosomal DNA by colony- or plaque-hybridization, PCR cloning. Southern-blot hybridization and so on.<br>
-	Determining nucleotide sequence of the enone reductase gene obtained as above by usual methods and constructing recombinant expression vectors which contain and express the enone reductase gene efficiently.<br>
-	Constructing recombinant organisms carrying the enone reductase gene on<br>
recombinant expression vectors or on chromosomes by transformation,<br>
transduction, transconjugation or electroporation.<br>
The techniques used to isolate or clone a DNA encoding enone reductase of the present invention are known in the art and include isolation from genomic DNA. The cloning of the DNA sequence of the present invention from such genomic DNA can be effected by using the degenerate polymerase chain reaction (hereinafter referred to as PCR).<br>
On the basis of information on the partial amino acid sequence oligonucleotides as primers for PCR may be synthesized. The primers used for cloning of the enone reductase gene by PCR may be based on the amino acid sequence of the peptide fragments of the purified enone reductase from the genera including, but not to limited to, Candida and Zygosaccharomyces, and in the most preferred embodiment, from C kefyr (C, macedoniensis) IFO 0960. A DNA fragment (a partial DNA sequence) of enone reductase is generated by PCR amplification with the primers and the template of, e.g., C kefyr chromosomal DNA. The amplified DNA fragment can be used as the probe to clone a genomic fragment coding for the whole enone reductase. An entire gene containing its coding region as well as<br><br>
its regulation region such as a promoter or terminator can be cloned from a<br>
chromosome, for example, by inverse PCR method using primers based on part of<br>
sequence of the obtained DNA fragment after it was sequenced, or screening of<br>
genomic library which is constructed in phage vector or plasmid vector in an<br>
appropriate host, by using a partial DNA fragment obtained by PCR as described<br>
above as a probe after it was labeled.<br>
Generally, E. coli as a host strain and E. coli vector, a phage vector such as X<br>
phage vector, a plasmid vector, or a yeast vector is often used in the construction of<br>
library and a following genetic manipulation such as sequencing, restriction<br>
digestion, ligation and so on. After the isolation of all necessary parts of the entire<br>
gene containing its coding region as well as its regulation region, obtained<br>
fragments were subcloned into an appropriate plasmid vector, which can be<br>
conveniently used for sequencing and construction of the entire gene of the enone<br>
reductase.   In this invention, the insert fragments were subcloned into pUC18<br>
vector. Nucleotide sequence can be determined by a well-known method such as<br>
dideoxy chain-termination method.<br>
The isolated DNA sequence of the present invention may be used to identify and<br>
clone DNA encoding a polypeptide having enone reductase activity from other<br>
strains of different genera or species according to methods well known in the art.<br><br>
 The present invention also relates to a recombinant DNA, preferably a vector<br>
and/or  plasmid   comprising   a   sequence   coding   for  enone  reductase.      The<br>
recombinant DNA vector and/or plasmid may comprise the regulatory regions such<br>
as promoters and terminators as well as open reading frames of a enone reductase<br><br>
gene. Methods which are well known to those skilled in the art may be used to construct expression vectors containing a nucleotide sequence encoding enone reductase and appropriate transcriptional and translational regulatory elements including all components which are necessary or advantageous for expression of the coding sequence of the nucleotide sequence. Specific initiation and termi¬nation signals may also be used to achieve more efficient translation of sequences encoding enone reductase. An isolated DNA sequence encoding enone reductase may be manipulated in a variety of ways to provide for expression of the polypeptide. Manipulation of the nucleotide sequence encoding enone reductase prior to its insertion into a vector may be desirable or necessary depending on the expression vector. The techniques for modifying nucleotide sequences utilizing cloning methods are well known in the art. A variety of expression vector/host systems may be utilized to contain and express sequences encoding enone reductase.<br>
The present invention also provides the use of the recombinant DNA to transform a host organism. A convenient form of the recombinant DNA may be a vector. The host organism transformed with the recombinant DNA may be useful in the production of a polypeptide of enone reductase and also useful in the improvement of the production process of levodione. Thus, the present invention also provides such a transformed host cell (recombinant microorganism) and a polypeptide encoded by the recombinant DNA.<br>
The present invention also provides a process for the production of the polypeptide encoded by the recombinant DNA, which comprises culturing the transformed host<br><br>
cell under the conditions suitable for the expression of the enzyme and recovery of the polypeptide from the cell culture. Cultivation of the recombinant microorganism can be carried out aerobically or anaerobically at pH values from 4.0 to 9.0, at a temperature in the range of from 10 to 60°C, for 15 minutes to 72 hours, preferably, at pH values from 5.0 to 8.0, at a temperature in the range of from 20 to 40°C for 30 minutes to 48 hours. The enone reductase produced by the recombinant cell may be secreted or contained intracellularly depending on the sequence and/or the vector used. The enone reductase may then be isolated from the culture medium or the recombinant cell by conventional procedures. The present invention further provides a process for the production of levodione, which comprises contacting ketoisophorone with the polypeptide enone reductase. The enone reductase of the present invention catalyzes the reduction of ketoisophorone to levodione in the presence of a co-factor, NADH or NADPH, according to the following formula:<br>
Ketoisophorone + NADH (NADPH)             Levodione + NAD (NAD?)<br>
The reaction can be conducted in a solvent such as Tris-HCl buffer and phosphate<br>
buffer.<br>
Preferable conditions for the reaction are pH values from 4.5 to 8.5, more<br>
preferably from 5.0 to 8.0, a temperature range of from  10 to 60°C, more<br>
preferably from 20 to 60°C, for a period of 5 minutes to 72 hours, more preferably<br>
for 15 minutes to 48 hours.<br><br>
The present invention also provides a method for the biological production of levodione, which comprises contacting ketoisophorone with a recombinant microorganism as described above, including cultivation of the recombinant microorganism in the presence of ketoisophorone as a substrate, under conditions suitable for the production of levodione, and isolating the resulting levodione from the reaction mixture.<br>
Either a growing or a resting cell culture or immobilized cells or a cell-free extract, or the like, of the recombinant microorganism may be used for the production of levodione.<br>
The growing cell culture can be obtained by culturing the recombinant microorganism in a nutrient medium containing saccharides such as glucose or sucrose, alcohols, such as ethanol or glycerol, fatty acids, such as oleic acid and stearic acid or esters thereof, or oils, such as rapeseed oil or soybean oil, as carbon sources; ammonium sulfate, sodium nitrate, peptone, amino acids, com steep liquor, bran, yeast extract and so on, as nitrogen sources; magnesium sulfate, sodium chloride, calcium carbonate, potassium monohydrogen phosphate, potassium dihydrogen phosphate, and so on, as inorganic salt sources; and malt ex¬tract, meat extract, and so on, as other nutrient sources. Cultivation of the recombinant microorganism can be carried out aerobically or anaerobically at pH values from 4.0 to 9.0, at a temperature in the range of from 10 to 60°C for 15 minutes to 72 hours, preferably, at pH values from 5.0 to 8.0, at a temperature in the range of from 20 to 40°C for 30 minutes to 48 hours.  Appropriate mixing of<br><br><br>
the culture during the cultivation will be preferable for the cell growth or the<br>
reaction.<br>
Using the growing cell culture thus obtained, a resting cell culture or immobilized<br>
cells or a cell-free extract may be prepared by any means generally known in the<br>
art.<br>
Preferable conditions for the production of levodione are pH values from 4.0 to 9.0<br>
and a temperature range of from 10 to 60°C for a period of 15 minutes to 72 hours.<br>
More preferable conditions for the production of levodione are pH values from 5.0<br>
to 8.0 and a temperature range of from 20 to 60°C for a period of 30 minutes to 48<br>
hours.<br>
The concentration of ketoisophorone in a reaction mixture can vary depending on<br>
other reaction conditions, but, in general, is between 0.1 g/1 and 300 g/1, preferably<br>
between 1 g/1 and 30 g/1.<br>
Levodione produced enzymatically or biologically in a reaction mixture as<br>
described above may be exfracted by an organic solvent such as ethyl acetate, n-<br>
hexane, toluene, or n-butyl.  The extract may be analyzed by known method such<br>
as gas chromatography, high performance liquid chromatography, thin layer<br>
chromatography   or   paper   chromatography,   or  the   like.      In   case   of gas<br>
chromatography, the following condition can be applied as an example: Column:<br>
ULBON HR-20M (Shinwa, Japan) 0.25 mm x 30m; Column temperature: 160°C<br>
(constant); Injector temperature: 250°C; Carrier gas: He (ca. Iml/min)<br>
After the reaction, levodione in the reaction mixture may be recovered, for<br>
example, by extraction with a water-immiscible organic solvent, which readily<br><br>
solubilizes levodione, such as ethyl acetate, n-hexane, toluene, or n-butyl acetate.<br>
Further purification of levodione can be effected by concentrating the extract to<br>
directly   crystallize   levodione   or  by  the   combination  of various  kinds   of<br>
chromatography,     for    example,    thin    layer    chromatography,     adsorption<br>
chromatography, ion-exchange chromatography, gel filtration chromatography or<br>
high performance liquid chromatography.<br>
The following Examples further illustrate the present invention.<br>
Example 1: Partial amino acid sequence of enone reductase of C. kefyr (C<br>
macedoniensis) IFO 0960 The freeze-dried purified enone reductase (as described in European Patent Application No. 02003967.3 filed on Feb. 22, 2002) of C. kefyr was digested with lysyl endopeptidase, and the resulting digest was separated by the Smart system, i.e., one nmol of the purified enzyme was dissolved in 25 \x\ of 50mM Tris-HCl buffer (pH 8.6) containing 8 M urea, and incubated at 37°C for I hour. After this, 25 l of 50 mM Tris-HCl buffer (pH 8.6) were added to make the concentration of urea 4 M. Then, 0.5 1 of 12 nmol/ml lysyl endopeptidase (Wako, Japan, 0.006 nmol, E/S=l/167) was added, and incubated at 30°C for 6 hours. The resulting peptides were separated by the Smart system using the following conditions: Column: µRPC C2/C18 SC2.1/10 (Amersham Bioscience / Buckinghamshire, England); Flow rate: 100 µl/min; Liquid A: 0.l % TFA; Liquid B: 0.1 % TFA+ 80 % CH3CN; Gradient: 100 % A (0-15 min); 100 % A &gt; 100 % B (15-75 min); Column temperature: Room temperature; Detection: 214 nm, 280 nm.<br><br>
The peptides (K-I5, K-25.1, K-6.i, K-6.2, K30, K13.2, K-1.1, K-1.2, K-33, K-25.2, K-20, K-17, K-22, K-4.1, K-13.1, and K-9) were isolated, and the amino acid sequences of these peptides were analyzed with a protein sequencer, i.e. by automated Edman degradation with a model 491HT pulsed liquid protein sequencer (Applied Biosystems, Foster City, California) to be Peptide K-15: SEQ ID NO: 3; PeptideK-25.1: SEQ ID NO: 4; Peptide K-6.1: SEQ ID NO: 5; Peptide K-6.2: SEQ ID NO: 6; Peptide K-30: SEQ ID NO: 7; Peptide K-13.2: SEQ ID NO: 8; Peptide K-1.1: SEQ ID NO: 9; Peptide K-1.2: SEQ ID NO: 10; Peptide K-33: SEQ ID NO: II; Peptide K-25.2: SEQ ID NO: 12; Peptide K-20: SEQ ID NO: 13; Peptide K-17: SEQ ID NO: 14; Peptide K-22: SEQ ID NO: 15; Peptide K-4.1: SEQ ID NO: 16; Peptide K-13.1: SEQ ID NO: 17; and Peptide K-9: SEQ ID NO: 18.<br>
The partial amino acid sequence obtained was compared with the sequences of proteins stored in the SWISS-PROT (release 37.0+/06-14, June 99), PIR (release 60.0, March 99), and PRF (release 99-05, May 99) protein databases. Sequence alignment was performed by using Blast (J. Mol. Biol., 215, 403-410, 1990) and Fasta (Proc. Natl. Acad. Sci. USA, 85, 2444-2448, 1988) programs. As a result, high homology with known Old Yellow Enzymes was found. Example 2: Preparation of chromosomal DNA of C. kefyr (C. macedoniensis)<br>
IFO 0960 The cells of C. kefyr (C macedoniensis) IFO 0960 were cultivated in 200 ml medium.   Cells were collected by centrifugation and suspended in 10 ml TES buffer.    3 ml of 0.5 M EDTA, 0.5 ml of Zymolyase solution, and 0.5 ml of<br><br>
Proteinase K solution were added to the cell suspension. After incubation at 37°C for 0.5 hour with gently mixing, 2 ml of 10 % SDS was added and mixed. After addition of H2O to make the volume 20 ml, 10 ml of TE-saturated phenol and 10 ml of chloroform were added and mixed. The upper layer was collected after centrifugation, the same volume of phenol/chloroform was added and mixed. After centrifugation, upper layer was collected and added with 0.1 x volume of 3M sodium acetate and 2.5 x volume of ethanol. Using a winding glass rod, DNA precipitate was collected, rinsed with 70 %, 80 %, and 90 % ethanol, dried and resuspended in 5 ml of TE buffer containing 10 µl of 5 mg/ml RNase A. DNA was completely dissolved by gently mixing at 4°C over night. 10 µl of 5 mg/ml RNase A were added again, and the DNA solution was incubated at 37°C for 2 hours. After treatment with phenol/chloroform, water layer was recovered and the DNA was ethanol precipitated, followed by centrifugation. The pellet was resuspended in 50 ml of TE buffer. Concentration of thus obtained genomic DNA was 88 ng/µl. Example 3: Cloning   of   partial   enone   reductase   gene   of   C.   kefyr   (C.<br>
macedoniensis) IFO 0960 Using the prepared genomic DNA as a template, a partial sequence for the enone reductase gene was obtained by degenerate PCR amplification using a thermal cycler (Perkin-Elmer Cetus Instruments, USA). The degenerate PCR primers were designed based on the partial amino acid sequences (K-15, K-13.1, and K-9) obtained in Example 1, and were as follows: Sense1 SEQ ID NO: 19)<br><br><br>
The PCR reaction (50 jxl) was carried out using 176 ng of chromosomal DNA (obtained in Example 2) as a template, 150 pmol each of degenerate primer, 2.5 nmol each of dATP, dCTP, dGTP, and dTTP, 1.5 units of Ex Taq polymerase (Takara Shuzo, Kyoto, Japan), and 5 µl of EX Taq buffer (Takara Shuzo). The initial template denaturation step consisted of 4 min at 94°C. An amplification cycle of 1 min at 94°C, 1 min at 50°C, and 1.5 min at 72°C was repeated for 35 times. After additional 10 min reaction at 72°C, a DNA fragment containing a partial enone reductase gene (approx. 1 kb) was amplified. This fragment was cloned on a sequencing vector, and DNA sequence was determined by the dideoxy chain-termination method. A Taq dye primer sequencing kit was used with an autosequencer (DNA Sequencer 373A, Applied Biosystems).   The partial DNA<br><br>
sequence thus obtained for the enone reductase and deduced amino acid sequence are as illustrated in SEQ ID N0:21 and SEQ ID NO:22, respectively. Example 4: Cloning   of  complete   enone   reductase   gene   of  C.   kefyr  (C macedoniensis) IFO 0960<br>
The inverse PCR was used to clone both upstream and downstream sequence flanking the partial enone reductase DNA sequence obtained in Example 3. 1 µg of the genomic DNA of C, kefyr (C. macedoniensis) IFO 0960 {obtained in Example 2) was digested with 10 units of Nco I (Takara Shuzo, Kyoto, Japan) in 50 \i\ of K-buffer containing 0.01 % BSA. After overnight reaction at 37°C, the reaction mixture was treated with phenol/chloroform, the water layer was recovered and the DNA was ethanol precipitated, followed by centrifugation. The DNA pellet was resuspended in 1 ml of T4 DNA ligase buffer containing 700 units of T4 DNA ligase (Takara Shuzo). After overnight reaction at 15°C, the reaction mixture was treated with phenol/chloroform, the water layer was recovered and the DNA was ethanol precipitated, followed by centrifugation. The DNA pellet was resuspended in TE buffer and used as a template for PCR. The PCR primers were designed based on the partial enone reductase gene sequence obtained in Example 3, and were as follows: IAl (antisense primer for upstream region) = SEQ ID NO:23 and ISl (sense primer for downstream region) = SEQ ID NO: 24. The PCR reaction (50µl) was carried out using 250 ng of the template DNA, 5 pmol each of primer, 2.5 nmol each of dATP, dCTP, dGTP, and dTTP, 2.5 units of Ex Taq polymerase (Takara Shuzo), and 5 \il of EX Taq buffer (Takara Shuzo). The initial template denaturation step consisted of 4 min at 94°C. An amplification<br><br>
cycle of 1 min at 94°C, 1 min at 60°C, and 4 min at 12°C was repeated for 30 times. After additional 10 min reaction at 72°C, a DNA fragment (approx. 4 kb) containing the upstream and downstream sequence of the enone reductase gene was amplified. This fragment was cloned on a sequencing vector, and DNA sequence was determined.<br>
By combining the thus obtained sequence with the partial enone reductase DNA sequence obtained in Example 3, an estimated entire gene sequence containing its coding region as well as its regulatory region such as a promoter or a terminator was obtained. The estimated entire DNA sequence thus obtained for the enone reductase is illustrated in SEQ ID NO: 25 containing the coding region as well as its flanking upstream and downstream region {estimated ORE is 148-1359). Next, the actual entire sequence of the enone reductase gene containing its coding region as well as its flanking upstream and downstream region was obtained by PCR as follows. The genomic DNA of C. kefyr (C. macedoniensis) IFO 0960 (obtained in Example 2) was used as a template. The PCR primers were designed based on the estimated enone reductase gene sequence obtained above (SEQ ID NO: 25), and are illustrated in SEQ ID NO: 26 (Sense) and SEQ ID NO: 27 (Antisense).<br>
The PCR reaction (50 µl) was carried out using 900 ng of the template DNA, 10 pmol each of primer, 2.5 nmol each of dATP, dCTP, dGTP, and dTTP, 100 nmol of MgC1, 1.5 units of LA Taq polymerase (Takara Shuzo), and 5 µl of LA Taq buffer (Takara Shuzo). The initial template denaturation step consisted of 2 min at 94°C.   An amplification cycle of 1 min at 94°C, 1 min at 6D°C, and 1.5 min at<br><br>
74°C was repeated for 23 times.  After additional 7 min reaction at 74°C, a DNA<br>
fragment (approx. 1.3 kb) containing the entire sequence of the enone reductase<br>
gene was amplified.   This fragment was cloned on a sequencing vector, and the<br>
DNA sequence was determined.<br>
The entire DNA sequence thus obtained for the enone reductase containing its<br>
coding region as well as its flanking upstream and downstream region is illustrated<br>
in SEQ ID NO:28 (ORF is 55-1266).<br>
Example 5: Expression of the enone reductase gene and levodione production<br>
using E.coli having the enone reductase gene of C kefyr A DNA fragment containing just the ORF of the enone reductase gene (1212 bp) was obtained by PCR amplification.  The PCR was performed with primers, ExS (SEQ ID NO:29) and ExA (SEQ ID NO:30).<br>
The vector carrying the entire sequence of the enone reductase gene (obtained in Example 4) was used as a template. The PCR reaction (50 µl) was carried out using 250 ng of the template DNA, 10 pmol each of primer, 2.5 nmol each of dATP, dCTP, dGTP, and dTTP, 1.5 units of Pyrobest DNA polymerase (Takara Shuzo), and 5 µl of Pyrobest buffer (Takara Shuzo). The initial template denaturation step consisted of 1 min at 94°C. An amplification cycle of 0.5 min at 94°C, 1 min at 60°C, and 1.5 min at 75°C was repeated for 15 times. After additional 5 min reaction at 75°C, a DNA fragment (approx. 1.2 kb) containing just the ORF of enone reductase gene was amplified.<br>
This amplified fragment of the enone reductase gene was cloned on a vector, pET101/D-TOPO, using a pET Directional TOPO® Expression Kits (Invitrogen<br><br>
Corporation, USA) according to an instruction manual prepared by the manufacturer. The vector carrying the enone reductase gene thus obtained (pET101/D-TOPO-ER) was introduced into E.coli BL21 (DE3), and several clones were selected for sequence analysis using an automatic sequence analyzer (DNA Sequencer 373A, Applied Biosystems). One of the clones, E.coli BL21 (DE3)[ pET101/D-TOPO-ER ], that showed completely the same sequence as the enone reductase sequence of C. kefyr was selected for further experiments. The strain, E.coli BL21 (DE3)[pET101/D-TOPO] was also prepared as a control. Each of the strains, E.coli BL21 (DE3)[ pET101/D-TOPO-ER ] and E.coli BL21 (DE3)[pET101/D-TOPO], was inoculated into the M9 minimum medium (5 ml in tube) containing 0.05 mg/ml of ampicillin and 2 % (W/V) of casamino acids (Difco laboratories, USA) and cultivated at 37°C. When the optical density at 610 nm reached 0.4, IPTG (isopropyl beta-D-thiogalactopyranoside) was added to the medium to make the concentration 0.01 mM and cultivation was continued for further 8-10 hours. Then the cells were collected by centrifugation, and a portion of the cells was used for SDS-PAGE analysis. As a result, an IPTG-induced protein band estimated as 45 kDa was observed only when the recombinant strain, E.coli BL21 (DE3)[pET101/D-TOPO-ER] was used.<br>
The rest of the collected cells was resuspended into 2 ml of 100 mM potassium phosphate buffer (pH 7.0). The suspension was used for confirming an activity to produce levodione from ketoisophorone. This suspension was divided into two portions (I ml each), and the reaction was started by adding 33 mM (final concentration, hereinafter abbreviated as f.c.) of ketoisophorone and 280 mM (f c.)<br><br>
of D-glucose with or without 0.37 mM (f.c.) of NAD*, 15 units / ml (f.c.) of glucose dehydrogenase. The reaction was carried out at 30°C overnight. The reaction mixture was extracted with ethylacetate to recover levodione in the ethylacetate layer. The extract was analyzed by gas chromatography. As a result, levodione was detected only when the recombinant strain, E.coli BL21 (DE3)[pET101/D-TOPO-ER] was used.<br><br><br><br><br><br>
WE CLAIM:<br>
1.	A recombinant DNA comprising a nucleotide sequence coding for an enzyme<br>
having enone reductase activity and wherein said nucleotide sequence is selected from the group consisting of: (i) a nucleotide sequence represented in SEQ ID NO:1;<br>
(ii) a nucleotide sequence coding for an enone reductase having the amino acid sequence encoded by the nucleotide sequence shown in SEQ ID NO: 1; (iii) a nucleotide sequence which hybridizes to the complement of the nucleotide sequence of (i) or (ii) under stringent hybridizing conditions; and (iv) a nucleotide sequence which is at least 80% identical to the nucleotide sequence of (i).<br>
2.	The recombinant DNA as claimed in claim 1, wherein said nucleotide<br>
sequence is selected from the group consisting of:<br>
(a)	a nucleotide sequence coding for a polypeptide having the amino acid sequence shown in SEQ ID No: 2;<br>
(b)	a nucleotide sequence coding for an allelic variant of the polypeptide having the amino acid sequence shown in SEQ ID NO:2; and<br>
(c)	a nucleotide sequence coding for a polypeptide having the amino acid<br>
sequence shown in SEQ ID NO: 2, in which one or more amino acids are added,<br>
inserted, deleted and/or substituted but having the enone reductase activity.<br>
3.	A vector or a plasmid comprising the recombinant DNA as claimed in claim 1.<br><br><br>
4.	A host cell such as E.coli, transformed or transfected by the recombinant DNA<br>
as claimed in claim 1, or the vector or the plasmid as claimed in claim 3.<br>
5.	A polypeptide encoded by the recombinant DNA as claimed in claim 1.<br>
6.	A process for the production of levodione, which comprises contacting ketoisophorone with the polypeptide as claimed in claim 5, under conditions suitable for the production of levodione, e.g. at pH values of from 4.5 to 8.5 and at a temperature range from 10 to 60 °C for 5 minutes to 72 hours, or at pH values of from 5.0 to 8.0 and at a temperature range from 20 to 60 °C for 15 minutes to 48 hours.<br>
7.	A process for the production of levodione, which comprises contacting ketoisophorone with the host cell as claimed in claim 5, or a cell-free extract thereof under the conditions suitable for the production of levodione, e.g. at pH values of from 4.0 to 9.0 and at a temperature range from 10 to 60 °C for 15 minutes to 72 hours, or at pH values of from 5.0 to 8.0 and at a temperature range from 20 to 60 °C for 30 minutes to 48 hours.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDMwLWNoZW5wLTIwMDUgYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">430-chenp-2005 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDMwLWNoZW5wLTIwMDUgY2xhaW1zIGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">430-chenp-2005 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDMwLWNoZW5wLTIwMDUgY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">430-chenp-2005 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDMwLWNoZW5wLTIwMDUgY29ycmVzcG9uZGVuY2Ugb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">430-chenp-2005 correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDMwLWNoZW5wLTIwMDUgY29ycmVzcG9uZGVuY2UgcG8ucGRm" target="_blank" style="word-wrap:break-word;">430-chenp-2005 correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDMwLWNoZW5wLTIwMDUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKSBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">430-chenp-2005 description (complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDMwLWNoZW5wLTIwMDUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">430-chenp-2005 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDMwLWNoZW5wLTIwMDUgZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">430-chenp-2005 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDMwLWNoZW5wLTIwMDUgZm9ybS0xOC5wZGY=" target="_blank" style="word-wrap:break-word;">430-chenp-2005 form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDMwLWNoZW5wLTIwMDUgZm9ybS0yNi5wZGY=" target="_blank" style="word-wrap:break-word;">430-chenp-2005 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDMwLWNoZW5wLTIwMDUgZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">430-chenp-2005 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDMwLWNoZW5wLTIwMDUgZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">430-chenp-2005 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDMwLWNoZW5wLTIwMDUgb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">430-chenp-2005 others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDMwLWNoZW5wLTIwMDUgcGN0IHNlYXJjaCByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">430-chenp-2005 pct search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDMwLWNoZW5wLTIwMDUgcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">430-chenp-2005 pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDMwLWNoZW5wLTIwMDUgcGV0aXRpb24ucGRm" target="_blank" style="word-wrap:break-word;">430-chenp-2005 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="218808-method-of-vapor-phase-catalytic-oxidation-using-multitubular-reactor.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="218810-a-continuously-operated-distillative-purification-of-1-2-propylene-glycol-formed-in-the-coproduct-free-synthesis-of-propylene-oxide.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>218809</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>430/CHENP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>21/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>23-May-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>16-Apr-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>18-Mar-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>DSM IP ASSETS B.V</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>Het Overloon 1, 6411 TE Heerlen,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>KATAOKA, Michihiko</td>
											<td>24-308 Okazaki-Irie-cho, Sakyo-ku, Kyoto-shi, Kyoto-fu, Kyoto 606-8322,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SHIMIZU, Sakayu</td>
											<td>6-9 Yamashita-cho, Tokiwa, Ukyo-ku, Kyoto-shi, Kyoto-fu, Kyoto 616-8212,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/53</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP03/10473</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-09-19</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>02021098.5</td>
									<td>2002-09-23</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/218809-a-recombinant-dna-coding-for-enone-reductase-and-microbial-production-of-levodione by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:32:35 GMT -->
</html>
