<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/218965-a-process-for-preparing-a-lignocellulose-based-product-and-the-product-obtained-by-the-process by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:22:11 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 218965:A PROCESS FOR PREPARING A LIGNOCELLULOSE-BASED PRODUCT, AND THE PRODUCT OBTAINED BY THE PROCESS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A PROCESS FOR PREPARING A LIGNOCELLULOSE-BASED PRODUCT, AND THE PRODUCT OBTAINED BY THE PROCESS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>ABSTRACT IN/PCT/2002/00857/CHE &quot;A process for preparing a lignocellulose-based product, and the product obtained by the process&quot; This invention relates to a process for the manufacture of a product, the process comprising the step of mixing in a reaction medium (i) a phenolic polymer being substituted with a phenolic hydroxyl group; (ii) a lignocellulose containing material having immobilized to a cellulosic fraction thereof a fusion polypeptide, the fusion polypeptide including an enzyme being capable of catalyzing the oxidation of phenolic groups and a cellulose binding peptide and (iii) an oxidizing agent. A composition of matter for use in the process and a lignocellulose product obtainable by the process are also disclosed.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
PROCESS AND COMPOSITION FOR PREPARING A<br>
LIGN0CELLUL0SE-BA3ED PRODUCT, AND THE Product<br>
OBTAINED BY THE PROCESS<br>
FIELD AND BACKGROUND OF THE INVENTION<br>
The present invention provides a process and compositions for producing a lignocellulose based product, e.g., fiber board, such as hardboard or medium-density fiber board ("MDF"), particle board, plywood, paper or paperboard (such as cardboard and linerboard), from an appropriate lignocellulosic starting material, such as wood fiber or vegetable fiber, having an enzyme adhered thereto via a cellulose binding peptide, which enzyme is capable of catalyzing the oxidation of phenolic groups of a phenolic polymer which may form an integral part of the lignocellulosic starting material, e.g., lignin, in the presence of an oxidizing agent and optionally in the presence of additional lignocellulosic starting material devoid of the enzyme, e.g., recycled fibers.<br>
The use of the process of the invention confers improved mechanical properties on lignocellulose-based products prepared thereby, especially paper products such as liner board, cardboard and corrugated board.<br>
Lignocellulose-based products prepared from lignocellulosic starting materials, notably products manufactured starting from vegetable fiber or wood fiber prepared by mechanical or mechanical/chemical procedures (the latter often being denoted "semi-chemical" procedures), or by a chemical procedure without bleaching, or from wood particles (wood "chips", flakes and the like), are indispensable everyday materials.<br>
Some of the most familiar types of such products include paper for writing or printing, cardboard, corrugated cardboard, fiber board (e.g. "hardboard"), and particle board.<br>
Virtually all grades of paper, cardboard and the like are produced from aqueous pulp slurry. Typically, the pulp is suspended in water, mixed with various additives and then passed to equipment in which the paper, cardboard etc. is formed, pressed and dried, irrespective of whether mechanically produced pulp (hereafter denoted "mechanical pulp"), semi-chemically produced pulp (hereafter denoted "semi-chemical pulp"), unbleached chemical pulp or pulp made from recycled fibers (i.e., pulp prepared from recycled fibers, rags and the like) is employed, it is often necessary to add various strengthening agents to the pulp in order to obtain an end product having adequate mechanical properties.<br><br>
In the case of paper and board for use in packaging and the like, the tensile strength and tear strength under dry and wet conditions are of primary importance; moreover, notably in the case of certain grades of cardboard (e.g., so-called unbleached board for the manufacture of corrugated cardboard boxes for packaging, transport and the like), the compression strength of the material is often also an important factor. Among the strengthening agents used today there are a number of environmentally undesirable substances which it would be desirable to replace by more environmentally acceptable materials. As examples hereof may be mentioned epichlorohydrin, urea-formaldehyde and melamine-formaldehyde.<br>
In the case of "traditional" lignocellulose-based composites for use in building construction, flooring, cladding, furniture, packaging and the like, such as hardboard (which is normally made from wood fibers produced by mechanical or semi-chemical means or by so-called "steam explosion") and particle board (which is made from relatively coarse wood particles, fragments or "chips"), binding of the wood fibers or particles to give a coherent mass exhibiting satisfactory strength properties can be achieved using a process in which the fibers/particles are treated - optionally in a mixture with one or more "extenders", such as lignosulfonates and/or kraft lignin - with synthetic adhesives (typically adhesives of the urea-formaldehyde, phenol-formaldehyde or isocyanate type) and then pressed into the desired form (boards, sheets, panels etc.) with the application of heat.<br>
The use of synthetic adhesives of the above-mentioned types in the production of wood products is, however, generally undesirable from an environmental and/or safety point of view, since many such adhesives are directly toxic - and therefore require special handling precautions - and/or can at a later stage give rise to release of toxic and/or environmentally harmful substances; thus, for example, the release of formaldehyde from certain cured formaldehyde-based adhesives (used as binders in, e.g., particle board and the like) has been demonstrated.<br>
In the light of the drawbacks associated with the use of synthetic adhesives as binders in the manufacture of lignocellulose-based products, considerable effort has been devoted in recent years to the development of binder systems and binding processes which are more acceptable from an environmental and toxicity point of view, and relevant patent literature in this respect includes the following:<br><br>
3<br>
EP 0 433 258 Al discloses a procedure for the production of mechanical pulp from a fibrous product using a chemical and/or enzymatic treatment in which a "binding agent" is linked with the lignin in the fibrous product via the formation of radicals on the lignin part of the fibrous product. This document mentions "hydrocarbonates", such as cationic starch, and/or proteins as examples of suitable binding agents. As examples of suitable enxymes are mentioned laccase, lignin peroxidase and manganese peroxidase, and as examples of suitable chemical agents are mentioned hydrogen peroxide with ferro ions, chlorine dioxide, ozone, and mixtures thereof.<br>
EP 0 565 109 A1 discloses a method for achieving binding of mechanically produced wood fragments via activation of the lignin in the middle lamella of the wood cells by incubation with phenol-oxidizing enzymes. The use of a separate binder is thus avoided by this method.<br>
U.S. Pat. No. 4,432,921 describes a process for producing a binder for wood products from a phenolic compound having phenolic groups, and the process in question involves treating the phenolic compound with enzymes to activate and oxidatively polymerize the phenolic compound, thereby converting it into the binder. The only phenolic compounds which are specifically mentioned in this document, or employed in the working examples given therein, are lignin sulfonates, and a main purpose of the invention described in U.S. Pat. No. 4,432,921 is the economic exploitation of so-called "sulfite spent liquor", which is a liquid waste product produced in large quantities through the operation of the widely-used sulfite process for the production of chemical pulp, and which contains lignin sulfonates.<br>
With respect to the use of lignin sulfonates - in particular in the form of sulfite spent liquor - as phenolic polymers in systems/processes for binding wood products (as described in U.S. Pat. No. 4,432,921), the following comments are appropriate: (i) subsequent work (see H. H. Nimz in Wood Adhesives, Chemistry and Technology, Marcel Dekker, New York and Basel 1983, pp. 247-288), and A Haars et al. in Adhesives from Renewable Resources, ACS Symposium Series 385, American Chemical Society 1989, pp. 126-134) has demonstrated that by comparison with the amounts of "traditional" synthetic adhesives which are required in the manufacture of wood-based boards, very large amounts of lignin sulfonates are required in order to achieve comparable strength properties; (ii) the pressing time required when pressing wood-based board products prepared using lignin sulfonate binders has been found to be very long, see E.<br><br>
Roffael and B. Dix, Holz als Roh- und Werkstoff 49 (1991) 199-205; (iii) lignin sulfonates available on a commercial scale are generally very impure and of very variable quality, see J. L. Philippou, Journal of Wood Chemistry and Technology 1(2) (1981) 199-227; (iv) the very dark color of spent sulfite liquor renders it unsuited as a source of lignin sulfonates for the production of, e.g., paper products (such as packaging paper, linerboard or unbleached board for cardboard boxes and the like) having acceptable color properties.<br>
U.S. Pat. No. 5,846,788, from which the above background information is derived, and which is incorporated by reference as if fully set forth herein, teaches that binding of lignocellulosic materials (vegetable fibers, wood chips, etc.) using a combination of a polysaccharide having at least substituents containing a phenolic hydroxy group (in the following often simply denoted a "phenolic polysaccharide"), an oxidizing agent and an enzyme capable of catalyzing the oxidation of phenolic groups by the oxidizing agent can be employed in the manufacture of lignocellulose-based products exhibiting strength properties at least comparable to, and often significantly better than, those achievable using previously known processes which have attempted to reduce or avoid the use of toxic and/or otherwise harmftil substances, such as the processes described in EP 0 433 258 Al, EP 0 565 109 Al and U.S. Pat. No. 4,432,921. Thus, for example, the amount of binder required to prepare lignocellulose-based products of very satisfactory strength by the process described in U.S. Pat. No. 5,846,788 is generally much lower typically by a factor of about three or more - than the level of binder (based on lignin sulfonate) required to obtain comparable strength properties using the process according to U.S. Pat. No. 4,432,921. The process according to U.S. Pat. No. 5,846,788 can thus not only provide an environmentally attractive alternative to more traditional binding processes employing synthetic adhesives, but it can probably also compete economically with such processes.<br>
However, the process described in U.S. Pat. No. 5,846,788, requires the use of purified enzymes which are expensive materials as is compared to other raw materials and reagents used in the process of manufacturing lignocellulose-based products.<br>
There is thus a widely recognized need for, and it would be highly advantageous to have, a process for producing a lignocellulose-based product, e.g. fiber board, such as hardboard or medium-density fiber board ("MDF"), particle board, plywood, paper or paperboard (such as cardboard<br><br>
and linerboard), from an appropriate lignoceliuiosic starting material devoid of the above limitation.<br>
SUMMARY OF THE INVENTION<br>
According to one aspect of the present invention there is provided a process for the manufacture of a lignocellulose product, the process comprising the step of mixing in a reaction medium (i) a phenolic polymer being substituted with a phenolic hydroxy group; (ii) a lignocellulose containing material having immobilized to a cellulosic fraction thereof a fusion polypeptide, the fusion polypeptide including an enzyme being capable of catalyzing the oxidation of phenolic groups and a cellulose binding peptide; and (iii) an oxidizing agent.<br>
According to further features in preferred embodiments of the invention described below, the lignocellulose product is selected from the group consisting of fiber board, particle board, flakeboard, plywood and molded composites.<br>
According to still further features in the described preferred embodiments the lignocellulose product is selected from the group consisting of paper and paperboard.<br>
According to still further features in the described preferred embodiments the lignocellulose containing material is a cell wall preparation derived from a genetically modified or virus infected plant or cultured plant cells expressing the fusion protein.<br>
According to still further features in the described preferred embodiments the lignocellulose containing material is selected from the group consisting of vegetable fiber and wood fiber derived from a genetically modified or virus infected plant expressing the fusion polypeptide.<br>
According to still further features in the described preferred embodiments the lignocellulose containing material is selected from the group consisting of vegetable fiber and wood fiber that has previously made contact with an oxidising enzyme fused to a cellulose binding peptid.<br>
According to still further features in the described preferred embodiments the phenolic substituent is selected from the group consisting of p-coumaric acid, p-coumaryl alcohol, coniferyl alcohol, sinapyl alcohol, ferulic acid p-hydroxybenzoic acid and any other phenolic group that can be oxidized.<br><br>
According to still further features in the described preferred embodiments the phenolic polymer forms an integral part of the lignocellulose containing material.<br>
According to still further features in the described preferred embodiments the phenolic polymer is lignin.<br>
According to still further features in the described preferred embodiments the phenolic polymer is a phenolic polysaccharide.<br>
According to still fiirther features in the described preferred embodiments the polysaccharide portion of the phenolic polysaccharide is selected from the group consisting of modified and unmodified starches, modified and unmodified cellulose, and modified and unmodified hemicelluloses.<br>
According to still further features in the described preferred embodiments the phenolic polysaccharide is selected from the group consisting of ferulylated arabinoxylans and feruiylated pectins.<br>
According to still further features in the described preferred embodiments the reaction medium is incubated for a period of from 1 minute to 10 hours.<br>
According to still further features in the described preferred embodiments the fusion polypeptide is incubated in the presence of the oxidizing agent for a period of from 1 minute to 10 hours.<br>
According to still flirther features in the described preferred embodiments the enzyme is selected from the group consisting of oxidases and peroxidases.<br>
According to still further features in the described preferred embodiments the enzyme is an oxidase selected from the group consisting of laccases (EC 1.10.3.2), catechol oxidases (EC 1.10.3.1) and bilirubin oxidases (EC 1.3.3.5), and the oxidizing agent is oxygen.<br>
According to still further features in the described preferred embodiments the enzyme is a laccase and is present in an amount in the range of 0.02-2000 LACU per g of dry lignocellulose.<br>
According to still further features in the described preferred embodiments the reaction medium is aerated.<br>
According to still further features in the described preferred embodiments the enzyme is a laccase encoded by a polynucleotide obtained from a fungus of the genus Botrytis, Myceliophthora, or Trametes.<br>
According to still further features in the described preferred embodiments the funaus is Trametes versicolor or Trametes villosa.<br><br>
According to still fiirther features in the described preferred embodiments the enzyme is a laccase from Acerpseudoplanus.<br>
According to still further features in the described preferred embodiments the enzyme is a peroxidase and the oxidizing agent is hydrogen peroxide.<br>
According to still fiirther features in the described preferred embodiments the peroxidase is present in an amount in the range of 0.02-2000 PODU per g of dry lignocellulose, and the initial concentration of hydrogen peroxide in the reaction medium is in the range of 0.01-100 mM.<br>
According to still further features in the described preferred embodiments the amount of lignocellulose employed corresponds to 0.1-90 % by weight of the reaction medium, calculated as dry lignocellulose.<br>
According to still further features in the described preferred embodiments the temperature of the reaction medium is in the range of 10 ° -120°C.<br>
According to still fiirther features in the described preferred embodiments the temperature of the reaction medium is in the range of 15 ° - 90 ° C.<br>
According to still further features in the described preferred embodiments an amount of the phenolic polysaccharide in the range of 0.1 %- 10% by weight.<br>
According to still fiirther features in the described preferred embodiments the pH in the reaction medium is in the range of 3-10.<br>
According to still further features in the described preferred embodiments the pH in the reaction medium is in the range of 4-9.<br>
According to still fiirther features in the described preferred embodiments the reaction medium further comprising a lignocellulose containing material devoid of the fusion protein.<br>
According to still further features in the described preferred embodiments the lignocellulose containing material devoid of the fiision protein is selected from the group consisting of vegetable fiber, wood fiber, wood chips, wood flakes, wood veneer and recycled fibers.<br>
■ Further according to the present invention there  is provided a lignocellulose product obtainable by the process described herein.<br>
According to another aspect of the present invention there is provided a genetically modified or viral infected plant or cultured plant cells expressing a fiision protein including an enzyme being capable of catalyzing the oxidation of phenolic groups and a cellulose binding peptide.<br><br>
According to still further features in the described preferred embodiments the fusion protein being compartmentalized within cells of the plant or cultured plant cells, so as to be sequestered from cell walls of the cells of the plant or cultured plant cells.<br>
According to still further features in the described preferred embodiments expression of the fusion protein is under a control of a constitutive or tissue specific plant promoter.<br>
According to still further features in the described preferred embodiments the fusion protein is compartmentalized within a cellular compartment selected from the group consisting of cytoplasm, endoplasmic reticulum, golgi apparatus, oil bodies, starch bodies, chloroplastids, chloroplasts, chromoplastids, chromoplasts, vacuole, lysosomes, mitochondria, and nucleus.<br>
According to still another aspect of the present invention there is provided a composition of matter comprising a cell wall preparation derived from a genetically modified or virus infected plant or cultured plant cells expressing a fusion protein including an enzyme being capable of catalyzing the oxidation of phenolic groups and a cellulose binding peptide, the fusion protein being immobilized to cellulose in the cell wall preparation via the cellulose binding peptide.<br>
According to still another aspect of the present invention there is provided a nucleic acid molecule comprising (a) a promoter sequence for directing protein expression in plant cells; and (b) a heterologous nucleic acid sequence including (i) a first sequence encoding a cellulose binding peptide; and (ii) a second sequence encoding an enzyme being capable of catalyzing the oxidation of phenolic groups, wherein the first and second sequences are joined together in frame.<br>
According to still further features in the described preferred embodiments the nucleic acid molecule further comprising a sequence element selected from the group consisting of an origin of replication for propagation in bacterial cells, at least one sequence element for integration into a plant's genome, a polyadenylation recognition sequence, a transcription termination signal, a sequence encoding a translation start site, a sequence encoding a translation stop site, plant RNA virus derived sequences, plant DNA virus derived sequences, tumor inducing (Ti) plasmid derived sequences, a transposable element derived sequence and a plant operative signal peptide for directing a protein to a cellular compartment of a plant cell.<br><br>
According to still further features in the described preferred embodiments the cellular compartment is selected from the group consisting of cytoplasm, endoplasmic reticulum, golgi apparatus, oil bodies, starch bodies, chloroplastids, chloroplasts, chromoplastids, chromoplasts, vacuole, lysosomes, mitochondria, and nucleus.<br>
The present invention successfiiUy addresses the shortcomings of the presently known configurations by providing a process, and compositions for producing a lignocellulose-based product which obviates the need for purified enzymes which are expensive materials as is compared to other raw materials and reagents described in the prior art for use in the process of manufacturing lignocellulose-based products.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The present invention is of a process and composition of matter for the manufacture of a lignocellulose-based product from a lignocellulosic material, which process obviates the need for using purified enzymes.<br>
The principles and operation of a process according to the present invention may be better understood with reference to the accompanying descriptions.<br>
Before explaining at least one embodiment of the invention in detail, it is to be understood that the invention is not limited in its application to the details of steps and components set forth in the following description. The invention is capable of other embodiments or of being practiced or carried out in various ways. Also, it is to be understood that the phraseology and terminology employed herein is for the purpose of description and should not be regarded as limiting.<br>
The present invention thus provides a process for the manufacture of a lignocellulose-based product from a lignocellulosic material. The process according to the present invention is effected by mixing in a reaction medium (i) a phenolic polymer substituted with a phenolic hydroxy group (e.g., lignin or a polysaccharide which is substituted with at least substituents containing a phenolic hydroxy group); (ii) a lignocellulose containing material having immobilized to a cellulosic fraction thereof a fiision polypeptide, the fusion polypeptide including an enzyme being capable of catalyzing the oxidation of phenolic groups and a cellulose binding peptide; and (iii) an oxidizing agent.<br>
The order of mixing/contacting the three components is unimportant as long as the process set-up ensures that the activated lignocellulosic<br><br>
material and the activated phenolic polysaccharide are brought together in a way that enables them to react in the desired manner. Thus, for example, the oxidizing agent may be mixed with the lignocellulose containing material before or after being mixed with the phenolic polymer.<br>
As is further detailed hereinunder, the lignocellulose containing material is preferably a cell wall preparation derived from a genetically modified or virus infected plant or cultured plant cells expressing the fusion protein. As such,.the phenolic polymer may form an integral part of the lignocellulose containing material because cell walls of plants contain lignin which is a phenolic polymer and thus the cell wall preparation can be made to contain lignin. In this case, and in order to prevent from the enzyme to exert its catalytic activity ahead of time, the cell wall preparation may be kept under a non-oxidizing atmosphere, such as an N2 atmosphere.<br>
Jt will generally be appropriate to incubate the reaction medium containing the three components for a period of at least a few minutes. An incubation time of from 1 minute to 10 hours will generally be suitable, although a period of from 1 minute to 10 hours is preferable.<br>
As already indicated, the process of the invention is well suited to the production of all types of lignocellulose-based products, e.g., various types of fiber board (such as hardboard), particle board, flakeboard, such as oriented-strand board (OSB), plywood, molded composites (e.g., shaped articles based on wood particles, often in combination with other, non-lignocellulosic materials, e.g., certain plastics), paper and paperboard (such as cardboard, linerboard and the like).<br>
Lignocellulose containing material:<br>
The lignocellulose containing material employed in the method of the invention can be in any appropriate form, e.g., in the form of vegetable fiber (such as wood fiber) with the provision that it is derived from a genetically modified or virus infected plant expressing the fusion polypeptide.<br>
If appropriate, a lignocellulosic material can be used in combination with a non-lignocellulosic material having phenolic hydroxy functionalities. Using the process of the invention, intermolecular linkages between the lignocellulosic material and the non-lignocellulosic material, respectively, jD may then be formed (i.e., in a manner analogous to that in which intermolecular linkages are formed when lignocellulosic materials alone are employed in the process), resulting in a composite product. Besides ftjnctioning as a good adhesive/binder, the phenolic polysaccharide also<br><br>
serves as a good "gap-filler", which is a big advantage when producing, e.g., particle boards from large wood particles.<br>
It will normally be appropriate to employ the lignocellulosic material in question in an amount corresponding to a weight percentage of dry lignocellulosic material [dry substance (DS)] in the reaction medium in the range of 0.1-90%.<br>
The temperature of the reaction mixture in the process of the invention may suitably be in the range of 10 °C - 120 °C., as appropriate; however, a temperature in the range of 15 °C - 90 °C is generally to be preferred. As illustrated by the working examples described in U.S. Pat. No. 5,846,788, it is anticipated that the reactions involved in a process of the invention may take place very satisfactorily at ambient temperatures around 20 °C.<br>
In addition to lignocellulose containing material to which the fusion protein is immobilized, the reaction medium according to the present invention may include a lignocellulose containing material devoid of such fusion protein, such as, but not limited to, vegetable fiber, wood fiber, wood chips, wood flakes, wood veneer and recycled fibers.<br>
Phenolic Polymers:<br>
The phenolic polymers employed in the process of the invention may suitably be materials obtainable from natural sources or polymers which have been chemically modified by the introduction of substituents having phenolic hydroxy groups. Examples of the latter category are modified starches containing phenolic substituents, e.g., acyl-type substituents derived from hydroxy-substituted benzoic acids (such as, e.g., 2-, 3- or 4-hydroxybenzoic acid).<br>
The phenolic substituent{s) in phenolic polysaccharides suited for use in the context of the present invention may suitably be linked to the polymer species by, e.g., ester linkages or ether linkages.<br>
Very suitable phenolic polymers are phenolic polysaccharides in which the phenolic substituent of the phenolic polysaccharide is a substituent derived from a phenolic compound which occurs in at least one of the following plant-biosynthetic pathways: from p-coumaric acid to p-coumaryl alcohol, from p-coumaric acid to coniferyl alcohol and from p-coumaric acid to sinapyl alcohol; p-coumaric acid itself and the three mentioned "end products" of the latter three biosynthetic pathways are also relevant compounds in this respect. Examples of relevant "intermediate" compounds formed in these biosynthetic pathways include caffeic acid.<br><br>
ferulic acid (i.e., 4-hydroxy-3-methoxycinnamic acid), 5-hydroxy-ferulic acid and sinapic acid.<br>
Particularly suitable phenolic polysaccharides are those which exhibit good solubility in water, and thereby in aqueous media in the context of the invention. In this and other respects, a number of types of phenolic polysaccharides which are readily obtainable in uniform quality from vegetable sources have been found to be particularly well-suited for use in the process of the present invention. These include, but are in no way limited to, phenolic arabino and heteroxylans, and phenolic pectins. Ver}' suitable examples thereof are ferulylated arabinoxylans (obtainable, e.g., from wheat bran or maize bran) and ferulylated pectins (obtainable from, e.g., beet pulp), i.e., arabinoxylans and pectins containing ferulyl substituents attached via ester linkages to the polysaccharide molecules.<br>
The amount of phenolic polysaccharide or other phenolic polymers, such as lignin, employed in the process of the invention will generally be in the range of 0.01-10 weight percent, based on the weight of lignocellulosic material (calculated as dry lignocellulosic material), and amounts in the range of about 0.02-6 weight per cent (calculated in this manner) will often be very suitable.<br>
Enzymes and polynucleotides encoding same In principle, any type of enzyme capable of catalyzing oxidation of phenolic groups may be employed in the process of the invention, with the provision that a polynucleotide encoding same has been isolated or is readily isolateable using conventional genetic engineering isolation techniques and which can therefore be expressed as a part of a fusion polypeptide.<br>
Preferred enzymes are, however, oxidases, e.g., laccases (EC 1.10.3.2), catechol oxidases (EC 1.10.3.1) and bilirubin oxidases (EC 1.3.3.5) and peroxidases (EC 1.11.1.7). In some cases it may be appropriate to employ two or more different enzymes in the process of the invention.<br>
Among types of oxidases (in combination with which oxygen - e.g., atmospheric oxygen - is an excellent oxidizing agent), laccases have proved to be well suited for use in the method of the invention.<br>
Polynucleotides encoding laccases have been or are readily isolateable from a variety of plant and microbial sources, notably bacteria and fungi (including filamentous fungi and yeasts), see, for example, U.S. Pat. Nos. 5,843,745; 5,795,760; 5,770,418; and 5,750,388, which are incorporated herein by reference.     Suitable examples of polynucleotides<br><br>
encoding laccases include those obtained or obtainable from strains of Aspergillus, Neurospora (e.g., N. crassa), Podospora, Botrytis, Collybia, Fames, Lentinus, Pleurotus, Trametes - some species/strains of which are known by various names and/or have previously been classified within other genera; e.g. Trametes villosa = T. pinsitus = Polyporuspinsitis (also known as P. pinsitus or P. villosus) - Coriolus pinsitits, Polyporus, Rhizoctonia (e.g., R. solani), Coprinus (e.g., C. pUcatilis), Psatyrella, Myceliophthora (e.g., M thermophila), Schytalidium, Phlebia (e.g. P. radita; see WO 92/01046), or Coriolus (e.g., C. hirsutus; see JP 2-238885,).<br>
A preferred laccase in the context of the invention is that obtainable from Trametes villosa or Acer pseudoplanus.<br>
Polynucleotides encoding peroxidase enzymes (EC 1.11.1) employed in the method of the invention are preferably those obtained or obtainable from plants (e.g., horseradish peroxidase or soy bean peroxidase) or from microorganisms, such as fungi or bacteria. In this respect, some preferred fungi include strains belonging to the sub-division Deuteromycotina, class Hyphomycetes, e.g., Fusarium, Humicola, Tricoderma, Myrothecium, Verticillum, Arthromyces, Caidariomyces, Ulocladium, Embellisia, Cladosporium or Dreschlera, in particular Fusarium oxysporum (DSM 2672,), Humicola insolens, Trichoderma resii, Myrothecium verrucana (IFO 6113,), Verticillum alboatrum, Verticillum dahlie, Arthrorriyces ramosus (PERM P-7754), Caidariomyces ftimago, Ulocladium chartarum, Embellisia alii or Dreschlera halodes.<br>
Other preferred fungi include strains belonging to the sub-division Basidiomycotina, class Basidiomycetes, e.g., Coprinus, Phanerochaete, Coriolus or Trametes, in particular Coprinus cinereus f. microspores (IFO 8371), Coprinus macrorhizus, Phanerochaete chrysosporium (e.g., NA-12) or Trametes versicolor (e.g. PR4 28-A).<br>
Further preferred fungi include strains belonging to the sub-division Zygomycotina, class Mycoraceae, e.g., Rhizopus or Mucor, in particular Mucor hiemalis.<br>
Some preferred bacteria include strains of the order Actinomycetales, e.g., Streptomyces spheroides (ATTC 23965), Streptomyces thermoviolaceus (IFO 12382) or Streptoverticillum verticillium ssp. verticillium.<br>
Other preferred bacteria include Bacillus pumilus (ATCC 12905), Bacillus    stearothermophilus,    Rhodobacter    sphaeroides,    Rhodomonas<br><br>
palustri. Streptococcus lactis, Pseudomonas purrocinia (ATCC 15958) or Pseudomonas fluorescens (NRRL B-11).<br>
Further preferred bacteria include strains belonging to Myxococcus, e.g., M. virescens.<br>
Other potential sources of useful sources for polynucleotides encoding peroxidases are listed in B. C. Saunders et al., Peroxidase, London 1964, pp. 41-43.<br>
Cellulose binding peptides:<br>
As used herein in the specification and in the claims section below, the phrase "cellulose binding peptide" mcludes peptides e.g., proteins and domains (portions) thereof, which are capable of, when expressed in plant cells, affinity binding to a plant derived cellulosic (e.g., lignocellulosic) matter, e.g., following homogenization and cell rupture or during plant growth and development. The phrase thus includes, for example, peptides which were screened for their cellulose binding activity out of a library, such as a peptide library or a DNA library (e.g., a cDNA expression library or a display library) and the genes encoding such peptides isolated and are expressible in plants. Yet, the phrase also includes peptides designed and engineered to be capable of binding to cellulose and/or units thereof<br>
Such peptides include amino acid sequences expressible in plants that are originally derived from a cellulose binding region of, e.g., a cellulose binding protein (CEP) or a cellulose binding domain (CBD). The cellulose binding peptide according to the present invention can include any amino acid sequence expressible in plants which binds to a cellulose polymer. For example, the cellulose binding domain or protein can be derived from a cellulase, a binding domain of a cellulose binding protein or a protein screened for, and isolated from, a peptide library, or a protein designed and engineered to be capable of binding to cellulose or to saccharide units thereof, and which is expressible in plants. The cellulose binding domain or protein can be naturally occurring or synthetic, as long as it is expressible in plants. Suitable polysaccharidases from which a cellulose binding domain or protein expressible in plants may be obtained include p-l,4-glucanases. In a preferred embodiment, a cellulose binding domain or protein fi-om a cellulase or scaffoldin is used. Typically, the amino acid sequence of the cellulose binding peptide expressed in plants according to the present invention is essentially lacking in the hydrolytic activity of a polysaccharidase (e.g., cellulase, chitinase), but retains the cellulose binding activit}'. The amino acid sequence preferably has less than<br><br>
about 10 % of the hydrolytic activity of the native polysaccharidase; more preferably less than about 5 %, and most preferably less than about 1 % of the hydrolytic activity of the native polysaccharidase, ideally no activity altogether.<br>
The cellulose binding domain or protein can be obtained from a variety of sources, including enzymes and other proteins which bind to cellulose which find use in the subject invention.<br>
In Table 4 below are listed those binding domains which bind to one or more soluble/insoluble polysaccharides including all binding domains with affinity for soluble glucans (a, p, and/or mixed linkages). The Nl cellulose-binding domain from endoglucanase CenC of C. fimi is the only protein known to bind soluble cellosaccharides and one of a small set of proteins which are known to bind any soluble polysaccharides. Also, listed in Tables 1 to 3 are examples of proteins containing putative p-l,3-glucan-binding domains (Table 1); proteins containing Streptococcal glucan-binding repeats (Cpl superfamily) (Table 2); and enzymes with chitin-binding domains, which may also bind cellulose (Table 3). The genes encoding each one of the peptides listed in Tables 1-4 are either isolated or can be isolated as further detailed hereinunder, and therefore, such peptides are expressible in plants. Scaffoldin proteins or portions thereof, which include a cellulose binding domain, such as that produced by Clostridium cellulovorans (Shoseyov et al, PCT/US94/04132) can also be used as the cellulose binding peptide expressible in plants according to the present invention. Several fungi, including Trichoderma species and others, also produce polysaccharidases from which polysaccharide binding domains or proteins expressible in plants can be isolated. Additional examples can be found in, for example. Microbial Hydrolysis of Polysaccharides, R. A. J. Warren, Annu. Rev. Microbiol. 1996, 50:183-212; and "Advances in Microbial Physiology" R. K. Poole, Ed., 1995, Academic Press Limited, both are incorporated by reference as if fully set forth herein.<br><br>
Table 1 Overview of proteins containing putative fhl,3 glucan-binding domains<br><br>
Source (strain)<br><br>
Protein<br><br>
accession No.<br><br>
Ref^l<br><br>
Type I<br><br><br>
B.  circiilans (WL-\2)	GLCAl<br>
B. circulans (lAM 1165)	BglH<br>
Type II	<br>
Actinomadura sp. (FC7)	Xynll<br>
Arthrobacter sp. (YCWD3)	GLCl<br>
O. xanthineolytica	GLC<br>
R. faecitabidus (YLM-50)	RPJ<br>
R. communis	Rictn<br>
S. lividans (1326)	XInA<br>
T. tridentalus	FactorGa<br><br>
P23903/M34503/JQ0420   1 JN0772/D17519/S67033   2<br>
U08894	3<br>
D23668	9<br>
P22222/M60826/A39094	4<br>
O05308/A45053/DI0753	5a,b<br>
A12892	6<br>
P265I4/M64551/JS07986	7<br>
D16622	8<br><br>
B.   : Bacillus, O.   : Oerskovia, R. faecitabidus : Rarobacter faecitabidus, R.   communis: Ricinus communis, S.  : Streptomyces, T.  : Tachypleus (Horseshoe Crab)<br>
' References:<br>
1)	Yahata et al.  (1990) Gene 86, 113-117<br>
2)	Yamamoto e/a/. {\993) Biosci. Biotechnol Biochem. 57,1518-1525<br>
3)	Harpin et al. (1994) EMBL Data Library 4)SheneJa/. (1991)^. Biol. Chem. 266,1058-1063<br>
5a) Shimoi ei a/. (1992)7. Biol. Chem. 267,25189-25195 5b) Shimoi et al. (1992) J. Biochem 110, 608-613<br>
6)	Horn et al (1989) Patent A12892<br>
7)	Shareck et al. (1991) Gene 107, 75-82 8)Sekie/fl/. (1994) J. Biol. Chem. 269,1370-1374 9) Watanabe et al. (1993) EMBL Data Library<br>
Table 2 Overview of proteins containing Streptococcal glucan-binding repeats<br>
(Cpl superfamily)<br><br>
Source<br><br>
Protein<br><br>
Accession No.<br><br>
Ref.^<br><br><br><br>
S.	downei(sobrinus) {0MZ]76)	GTF-I<br>
S.	downei (sobrinus) {MFtlS)	GTF-1<br>
5.	downei (sobrimisj (MTelS)	GTF-S<br>
S.	downei (sobrinus) {67\5)	GTF-1<br>
S.	downei (sobrinvs)	DEI<br>
S.	mutants ilngbun)	GBP<br>
5.	mutants (GS-5)	GTF-B<br><br>
D13858	1<br>
P11001/M17391	2<br>
P29336/M30943/A41483	3<br>
P27470/D90216/A38175	4<br>
L34406	5<br>
M30945/A37184	6<br>
A33128	7<br><br>
Table 2 (Continued)<br><br>
s. s. s. s. s.	mutants (GS-5) mvtants mutants (GS-5) mutants (GS-5) mutants (GS-5)	GTF-B GXF-B3'-0RF<br>
GTF-C GTF-C GTF-D<br>
s.	salivarius	GTF-J<br><br>
S'. salivarius	GTF-K<br>
S. salivarius (ATCC25975)	GTF-L<br>
S salivarius (ATCC25975)	GTF-M<br>
S. pneumoniae R6	LytA<br>
S. pneumoniae	PspA<br>
Phage HB-3	HBL<br>
Phage Cp-1	CPL-1<br>
Phage Cp-9	CPL-9<br>
Phase EJ-1	EJL<br><br>
P08987/M)7361/B33135 8<br>
P05427/C33135	8<br>
P13470/Mi7361/M22054 9<br>
not available	10<br>
M29296/A45866	11<br>
A448II/S22726/S28809	12<br>
Z1I873/M64]11<br>
S22737/S22727/Z11872	13<br>
L35495	14<br>
L35928	14<br>
P06653/A25634/M13812 15<br>
A41971/M74122	16<br>
P32762/M34652	17<br>
P15057/J03586/A31086 18<br>
P19386/M34780/JQ0438 19<br>
A42936	20<br><br><br><br>
C difficile iV?\ 10463)	ToxA<br>
C difficile (BARTS V^\)	ToxA<br>
C. a'/^c;7e(VPl 10463)	ToxB<br>
C  difficile(\A70)	ToxB<br>
C. novyi	a-loxin<br>
C. novyi	a-toxin<br>
C ace/ofc«fv//cM;)i(NCIB8052)	CspA<br>
C. acetobuty'licum (UCIBS052)	CspB<br>
C. aceiobutylicum (NCIBS052)	CspC<br>
C. acetobufylicum (HCIB^OSI)	CspD<br><br><br>
P16154/A37052/M30307	21<br>
X51797/S08638	<br>
A60991/X17194	22<br>
PI8177/X53I38/X60984	23,24<br>
S10317	<br>
S44271/Z23277	25,26<br>
S44272/Z23280	27<br>
Z48636	28<br>
S49255/Z37723	29<br>
Z50008	30<br>
Z50033	30<br>
Z50009	30<br><br>
'References:<br>
1)	Sato et al. (1993) DNA sequence 4.19-27<br>
2)	Ferreti e/a/. (1987)/ Bacieriol.  169,4271-4278<br>
3)	Gilmore e/fl/. (1990)7. Infect. Immvn. 58,2452-2458<br>
4)	Abo era/. (1991) J. Bacterial  173,989-996<br>
5)	Sun e/a/.  (1994)7.  Bacterioi  176,7213-7222 6)Banase/a/. (1990) J. Infect. Immun. 58,667-673<br><br>
7)	Shiroza et al. (1990) Protein Sequence Database<br>
8)	Shiroza e/o/. (1987) J. Bacterioi  169,4263-4270 9)Uedae/a/. (1988) Gene 69, 101-109<br>
.10)Russei(!990)^/-c/7. Oral Biol 35,53-58 ll)Hondae/a/. (1990)7. Gen. Microbiol  136,2099-2105<br>
12)	Giffard e/a/. (1991)7 Gen. Microbiol 137,2577-2593<br>
13)	Jacques (1992) EMBL Data Library<br>
14)	Simpson e/a/. (1995)7 Infect. Immun. 63,609-621<br>
15)	Gargia et al. (1986) Gewe 43,265-272 ]6)Yothere/a/.  (1992)7. Bacterioi   174,601-609<br>
17) Romero e/a/. (1990)7 Bacterioi. 172,5064-5070<br><br>
18)	Garcia e/a/, (1988) Froc. Natl. Acad. .7a", L'S,-( 85, 914-918<br>
19)	Garcia et at. (1990) Gene 86, 81 -88<br>
20)	Diaz e/a/. (1992) J. Bacteriol.  174,5516-5525 21)Dovee/fl/. (1990) J. Infect. Immun. 58,480-488<br><br>
22)	Wren e/a/. (]99Q) FEMS Microbiol. Lett. 70,1-6<br>
23)	Barroso eta. (1990) Nucleic Acids Res.  18, 4004-4004<br>
24)	von Eichel-Streiber e/a/. (1992) A/o/. Gen. Genet. 233.260-268<br>
25)	Sartinger et al. (1993) EMBL Data Library<br>
26)	von Eichel-Streiber e/fl/. (1995) A/o/. Microbiol. In Press<br>
27)	Hofmann et al. (1993) EMBL Data Library' 28)HoftTianne/fl/. (1995)/Wo/. Cen. Genet.  In Press<br><br>
29)	Sanchez el al. (1994) EMBL Data Library<br>
30)	Sanchez et ai (1995) EMBL Data Library<br>
New cellulose binding peptides with interesting binding characteristics and specificities can be identified and screened for and the genes encoding same isolated using well known molecular biology approaches combined with a variety of other procedures including, for example, spectroscopic (titration) methods such as: NMR spectroscopy (Zhu et al. Biochemistry (1995) 34:13196-13202, Gehring et al. Biochemistry (1991) 30:5524-5531), UV difference spectroscopy (Belshaw et al. Eur. J. Biochem. (1993) 211:717-724), fluorescence (titration) spectroscopy (Miller e/fl/. J. Biol. Chem. (1983) 258:13665-13672), UV or fluorescence stopped flow analysis (De Boeck et al. Eur. J. Biochem. (1985) 149:141-415), affinity methods such as affinity electrophoresis (Mimura et al. J. chromatography (1992) 597:345-350) or affinity chromatography on immobilized mono or oligosaccharides, precipitation or agglutination analysis including turbidimetric or nephelometric analysis (Knibbs et al. J. Biol. Chem. (1993) 14940-14947), competitive inhibition assays (with or without quantitative IC50 determination) and various physical or physico-chemical methods including differential scanning or isothermal titration calorimetry (Sigurskjold et al. J. Biol Chem. (1992) 267:8371-8376; Sigurskjold e/fl/. Eur. J. Biol. (1994) 225:133-141) or comparative protein stability assays (melts) in the absence or presence of oligo saccharides using thennal CD or fluorescence spectroscopy.<br>
The Ka for binding of the cellulose binding domains or proteins to cellulose is at least in the range of weak antibody-antigen extractions, i.e., &gt; 10^, preferably lO'^, most preferably 10^ M'^. If the binding of the cellulose binding domain or protein to cellulose is exothermic or endothermic, then binding will increase or decrease, respectively, at lower<br><br>
temperatures, providing a means tor temperature modulation of the binding step.<br>
Table 3 Overview of enzymes with chitin-binding domains<br><br>
Source (strain)<br><br>
Enzyme<br><br>
Accession No.<br><br>
Ref-<br><br>
Bacterial enzymes<br>
Typel<br>
Aeromonas sp. (NolOS-24)	Chi	D31818	1<br>
Bacillus circulans (WL-12)	ChiAl	P20533/M57601/A38368	2<br>
Bacillus circulans (WL-12)	ChiD	P27050/D10594	3<br>
Janthinobactehiim lividum	Chi69	U07025	4<br>
Streptomyces griseus	Protease C	A53669	5<br>
Type il			<br><br>
A eromonas cavia (K1)	Chi<br>
Alteromonas sp (0-7)	Chi85<br>
Aufographa califomica (C6)	NPH-128'<br>
Serralia marcescens	ChiA<br><br>
U09139	6<br>
A40633/P32823/D13762	7<br>
P41684/L22858	8<br>
A25090/X03657/L014'55/P07254	9<br><br>
Saccharomyces cerevisiae<br>
Sacchordniyces cerevisiae Chil<br>
(DBY939)'<br>
Saccharomyces cerevisiae Chi2<br>
(DBY918)<br>
Plant enzymes<br>
H eve in superfamiiy<br>
Allium sativum<br>
Amaranthus caudatus<br>
Amaranlhus caudatus<br>
Arabidopsis thaliana<br>
(cv. Colombia)<br>
Arabidopsis thaliana<br>
Brassica napus<br>
Brassica napus<br>
Hevea brasiliensis<br>
Hordeum vulgare<br>
Lycopersicon esculentum<br>
Nicotiana labacum<br>
Nicotiana tabacum<br>
Nicotiana tabacum (cv. Havana)<br>
Nicotiana tabacum (FB7-1)<br>
Type 111<br>
Rhizopus oligosporus (1F08631)    Chi 1 Rhizopus oligosporus (1F08631)    Chi2<br>
P29026/A47022/D10157/S27418	10<br>
P29027/B47022/D10158/S274I9	10<br>
Chi<br>
S50371/U17243	H<br>
P29028/M74069	12<br>
P29029/M7407/B41035	12<br><br>
Chi	M94105	13<br>
AMP-]'	P27275/A40240	14, 15<br>
AMP-2b	S37381/A40240	14,15<br>
ChiB	P19171/M38240/B45511	16<br>
PHPC	U01880	17<br>
Chi	mi 848	18<br>
Chi2	Q09023/TV195835	19<br>
Hevl^	P02877/M36986/A03770/A38288	20,21<br>
Chi33	L342I1	22<br>
Chi9	Q05538/Z15140/S37344	23<br>
CBP20e	S72424	24<br>
Chi	A21091	25<br>
Chi	A29074/M15173/S20981 /S19855	26<br>
Chi	JO0993/S0828	27<br><br>
		Table 3 (Continued)	<br>
Nicotiana tabacvm (cv.	Samsun)	Chi	A161I9	28<br>
Nicoliana tabaciim (cv.	Havana)	Chi	P08252/X16939/S08627	27<br>
Nicotiana tabacum (cv.	BY4)	Chi	P24091/X51599/X64519//S13322	26,27,29<br>
Nicotiana tabacum (cv.	Havana)	Chi	P29059/X64518/S20982	26<br>
Oryza sativum (IR36)		ChiA	L37289	30<br>
Oryza sativum		ChiB	JC2253/S42829/Z29962	31<br>
Oryza sativum		Chi	S39979/S40414/X56787	32<br>
Oryza sativum (cv. Japonicum)	Chi	X56063	33<br>
Oryza sativum ic\. Japonicum)	Chi I	P24626/X54367/S14948	34<br>
Oryza sativum		Chi2	P25765/S15997	35<br>
Oryza sativum {c\. Japonicum)	Chi3	D16223	<br>
Oryza sativum		ChiA	JC2252/S42828	30<br>
Oryza sativum		Chil	D16221	32<br>
Oryza sativum (1R58)		Chi	U02286	36<br>
Oryza sativum		Chi	X87109	37<br>
Pisum sativum (cv. Birte)	Chi	P36907/X63899	38<br>
Pisum sativum (cv. Aican)	Chi2	L37876	39<br>
Populus trichocarpa		Chi	S18750/S187 51 /X59995/P29032	40<br>
Populus trichocarpa (H11-11)	Chi	U01660	41<br>
Phaseolus vulgaris (c\i.	Saxa)	Chi	A24215/S43926/Jq0965/P36361	42<br>
Phaseofus vulgaris (cv.	Saxa)	Chi	P06215/M13968/M19052/A25898 43,44,45<br>
Sambucus nigra		PR-3
Secale cereale		Chi	JC2071	47<br>
Solanum tuberosum		ChiBl	U02605	48<br>
Solatium tuberosum		ChiB2	U02606	48<br>
Solanum tuberosum		ChiB3	U02607/S43317	48<br>
Solanum tuberosum		ChiB4	U02608	48<br>
Solanum tuberosum		WlN-lS	P09761/X13497/S04926	49<br>
(cv. Maris Piper)				<br>
Solanum tuberosum		WrN-2S	P09762/X13497/504927	49<br>
(cv. Maris Piper)				<br>
Triticum aestivum		Chi	S38670/X76041	50<br>
Triticum aestivum		WGA-lK	P10968/M25536/S09623/S07289	51,52<br>
Triticum aestivum		WGA-2h	P02876/M25537/S09624	51,53<br>
Triticum aestivum		WGA-3	P10969/J02961 /S10045/A28401	54<br>
Vlmus americana (NPS3-487)	Chi	L22032	55<br>
Urtica dioica		AGL'	M87302	56<br>
Vigna unguiculata		Chil	X88800	57<br>
(cv. Red caloona)				<br>
^NHP : nuclear polyhedrosis virus endochitinase like sequence; Chi : chitinase, "anti-microbial peptide, ^qjre-hevein like protein, "hevein, ^chitin-binding protein, 'pathogenesis related protein, Swound-induced protein, "wheat germ agglutinin, 'agglutinin (lectin).<br>
^References:<br>
l)Udeae/o/. (1994) J. Ferment. Bioeng.  78,205-211<br>
2)	Watanabe er a/. (1990) J. Biol Chem. 265,15659-16565<br>
3)	Watanabe e/a/. (1992)/ Bacteriol. 174,408-414<br>
4)	Gleave et al. (1994) EMBL Data Library 5)Sidhue/a/. (1994) J. Biol. Chem. 269^20167-20171 6)ionesetai {\9%6)EMBOJ. 5,467-473<br><br>
7)	Sitrit et al (1994) EMBL Data Library<br>
8)	Genbank entry only<br>
9)	Tsujibo er a/. (1993) J. Bacteriol  175,176-181<br><br>
)0)Yanaie/o/. (1992) J Bacterial 174,7398-7406<br>
11) Pauley (1994) EMBL Data Library<br>
12)Kurandaera/. (199!) J. Biol Chem. 266,19758-19767<br>
13)	van Damme et al. (1992) EMBL Data Library<br>
14)	Broekaert et al.  (1992) Biochemistry 31, 4308-4314 15)deBoliee/G/. {\993) Plant Mol. Physiol 22,1187-1190 16) Samac et al. (1990) Plant Physiol. 93, 907-914 l7)Ponere/a/.  (1993) A/o/. Plant Microbe Interact. 6,680-685 18) Buchanan-Wollaston (1995) EMBL Data Library 19)Hamele/a/. {]993) Plant Physiol 101,1403-1403<br>
20) Broekaert g/fl/.  (1990) Proc. Natl. Acad Sci.  USA 87,1633-1631 2\)Ltt etai  (1991) J. Biol. Chem. 266,15944-15948<br>
22)	Leah era/. (1994) Plant Physiol 6,579-589<br>
23)	Danhash e/a/. (1993) Plant Mol. Biol. 22 1017-1029<br>
24)	Ponstein et al (1994) Plant Physiol 104, 109-118<br>
25)	Meins et al. (1991) Patent EP0418695-A1<br>
26)	van Buuren e/a/. (1992) Mol. Gen. Genet. 232,460-469<br>
27)	Shinshi e/a/. (1990) Plant Mol Biol.  14,357-368<br>
28)	Comellisen et al. (1991) Patent EP0440304-A2<br>
29)	Fukuda et al. (1991) Plant Mol. Biol.  16,1-10<br>
30)	Yun et al. (1994) EMBL Data Library<br>
3\)K\m etal. (\994) Biosci. Biotechnol  Biochem. 58,1164-1166<br>
32)	Nishizawa e/a/. (1993) A/o/. Gen. Genet. 241,1-10<br>
33)	Nishizawa et al. (1991) Plant Sci 76, 211-218<br>
34)	Huang e/o/. {\99\) Plant Mol Biol  16,479-480 35)Zhue/fl/. (1991)AYo/. Gen. Genet. 226,289-296<br><br>
36)	Muthukrishhnan et al. (1993) EMBL Data Library<br>
37)	Xu (1995) EMBL Data Library<br>
38)	Vad et al. (1993) Plant Sci 92, 69-79<br>
39)	Chang et al. (1994) EMBL Data Library<br>
40)	Davis e/0/. {\99\) Plant Mol. Bioi  17,631-639<br>
41)	Clarke e/a/. {1994) Plant Mol Biol 25,799-815<br>
42)	Broglie el al. (1989) Plant Cell 1, 599-607<br>
43)	Broglie er o/. (1986) Proc. Natl, acad Sci.  t/S/l 83, 6820-6824<br>
44)	Lucas e/a/. {\985) FEBS Lett.  193,208-210<br>
45)	Hedrick et al. (1988) Plant Physiol 86, 182-186<br>
46)	Roberts et al. (1994) EMBL Data Library]<br>
47)	Vzmagam'i el al. (1994) Biosci. Biotechnol. Biochem. 58,322-329<br>
48)	Beerhues e/a/. (1994) Plant Mol Biol. 24,353-367<br>
49)	Stanford er a/. (\9S9) Mol. Gen. Genet. 215,200-208<br>
50)	Liao et al. (1993) EMBL Data Library<br>
51)	Smith era/. (19Z9) Plant Mol. Biol  13,601-603<br>
52)	Wright er a/. (1989) J. Mol. Evol. 28,327-336<br><br>
53)	Wright et al. (1984) Biochemistry 23, 280-287<br>
54)	Raikhel er fl/. (1987) Proc. Natl, acad Sci. USA 84, 6145-6149<br>
55)	Hajela et al. (1993) EMBL Data Library<br>
56)	Lemer e/fl/. (1992) J. Biol, Chem. 267,11085-11091 51) \ octal  (\995) EMBL Data Library<br><br>
Table 4 Sources of polysaccharide binding domains<br><br>
Binding Domain<br><br>
Proteins Where Binding Domain is Found<br><br><br><br>
Cellulose Binding Domains'<br>
Starch/Maltodextrin Binding Domains<br>
Dextran Bindin" Domains<br>
P-Glucan Binding Domains<br>
Chitin Binding Domains<br><br>
P-glucanases (avicelases, CMCases,<br>
cellodextrinases)<br>
exoglucanses or celiobiohydrolases<br>
cellulose binding proteins<br>
xylanases<br>
mixed xylanases/giucanases<br>
esterases<br>
chitinases<br>
p-l,3-glucanases<br>
P-1,3-(P-1,4)-glucanases<br>
(P-&gt;mannanases<br>
P-glucosidases/galactosidases<br>
cellulose synthases (unconfirmed)<br>
a-amylases-'3 P-amylases''^ puilulanases glucoamylases°' cyclodextrin glucotransferases^"'" (cyclomaltodextringlucanotransferases) maltodextrin binding proteins'<br>
(Streptococcal) glycosyl transferases'-dextran sucrases (unconfirmed) Clostridial toxins' -^ &gt;' ^ glucoamylases" dextran binding proteins<br>
P-1,3-glucanases' ^' ^<br>
P-l,3-(p-l ,4)-glucanases (unconfirmed)<br>
P-I,3-gIucan binding protein<br>
chitinases<br>
chitobiases<br>
chitin binding proteins<br>
(see also cellulose binding domains)<br>
Heivein<br><br>
' Gilkes el al. Adv. Microbiol Review's. (1991) 303-315. 2s?gaard e/a/., / Biol. Chem. (1993)268:22480. ^Weselake et al. Cereal Chem. (1983) 60:98. "^Svensson cr fl/., J. (1989)264:309. 5jespersen et al., J. (1991) 280:51. %elshaw er a/., £ur. J. Biochem. (1993)211:717. '^Sigurskjold er a/., £ur. J. Biochem. (1994)225:133. ^VWi^nt et al.. Biotechnol Appl. Biochem.  (1992)16:57. ^fMV.ada.elal., Biosci. Biotechnol.  Biochem. (1992)56:556.<br><br>
^^Lawson etal.. J. Mo/. Biol (1994)236:590. '^vonEichel-Streibere/fl/., Mo/. Gen. Genet. (1992)233:260. '5Kleble/a/., J. Bacterial. (1989)171:6259. '6watanabeera/..7, Bactehol (1992)174:186. l^Duvice/fl/., J. Biol Chem. (1990):9327.<br>
Thus, and as already stated, the phrase "polysaccharide binding peptide" includes an amino acid sequence which comprises at least a functional portion of a polysaccharide binding region (domain) of a polysaccharidase or a polysaccharide binding protein. The phrase further relates to a polypeptide screened for its cellulose binding activity out of a library, such as a peptide library or a DNA library (e.g., a cDNA library or a display library). By "functional portion" is intended an amino acid sequence which binds to cellulose.<br>
The techniques used in isolating polysaccharidase genes, such as cellulase genes, and genes for cellulose binding proteins are known in the art, including synthesis, isolation from genomic DNA, preparation from cDNA, or combinations thereof. (See, U.S. Pat. Nos. 5,137,819; 5,202,247; 5,340,731; 5,496,934; and 5,837,814). The sequences for several binding domains, which bind to soluble oligosaccharides are known (See, Figure 1 of PCT/CA97/00033, WO 97/26358). The DNAs coding for a variety of polysaccharidases and polysaccharide binding proteins are also known. Various techniques for manipulation of genes are well known, and include restriction, digestion, resection, ligation, in vitro mutagenesis, primer repair, employing linkers and adapters, and the like (see Sambrook et al. Molecular Cloning - A Laboratory Manual, Cold Spring Harbor Laboratory, Cold Spring Harbor, New York, 1989, which is incorporated herein by reference).<br>
The amino acid sequence of a polysaccharidase can be used to design a probe to screen a cDNA or a genomic library prepared from mRNA or DNA from cells of interest as donor cells for a polysaccharidase gene or a polysaccharide binding protein gene. By using the polysaccharidase cDNA or binding protein cDNA or a fragment thereof as a hybridization probe, structurally related genes found in other species can be easily cloned and provide a cellulose binding peptide which is expressible in plants according to the present invention. Particularly contemplated is the isolation of genes from organisms that express polysaccharidase activity using oligonucleotide probes based on the nucleotide sequences of genes obtainable from an organism wherein the catalytic and binding domains of the polysaccharidase<br><br>
are discrete, although other polysaccharide binding proteins also can be used (see, for example, Shoseyov, et al., Proc.  Nat'l.  Acad.   Sci.   (USA)<br>
(1992)	89:3483-3487).<br>
Probes developed using consensus sequences for the binding domain of a polysaccharidase or polysaccharide-binding protein are of particular interest. The P-l,4-glycanases from C. fimi characterized to date are endoglucanases A, B, C and D (CenA, CenB, CenC and CenD, respectively), exocellobiohydrolases A and B (CbhA and CbhB, respectively), and xylanases A and D (Cex and XylD, respectively) (see Wong et al. (1986) Gene, 44:315; Meinke er o/. (1991) J. Bacteriol., 173:308; Coutinho et al., (1991) Mol.   Microbiol.   5:1221; Meinke et al.,<br>
(1993)	Bacteriol., 175:1910: Meinke e/o/., (1994) Mol. Microbiol., 12:413;<br>
Shen et al., Biochem. J., in press; O'Neill et al., (1986) Gene, 44:325; and<br>
Millward-Sadler et al, (1994) Mol. Microbiol., 11:375). All are modular<br>
proteins of varying degrees of complexity, but with two features in<br>
common: a catalytic domain (CD) and a cellulose-binding domain (CBD)<br>
which can function independently (see Millward-Sadler et al, (1994) Mol.<br>
Microbiol., 11:375; Gilkes et al, (1988) J. Biol. Chem., 263:10401;<br>
Meinke et al, (1991) J. Bacteriol., 173:7126; and Coutinho et al, (1992)<br>
Mol. Microbiol., 6:1242). In four of the enzymes, CenB, CenD, CbhA and<br>
CbhB, fibronectin type III (Fn3) repeats separate the N-terminal CD from<br>
the C-terminal CBD. The CDs of the enzymes come from six of the<br>
families of glycoside hydrolases (see Henrissat (1991) Biochem. J.,<br>
280:309; and Henrissat et al, (1993) Biochem. J., 293:781); all of the<br>
enzymes have an N- or C-teraiinal CBD or CBDs (see Tomme et al. Adv.<br>
Microb. Physiol., in press); CenC has tandem CBDs from family IV at its<br>
N-terminus; CenB and XylD each have a second, internal CBD from<br>
families III and II, respectively. Cex and XylD are clearly xylanases;<br>
however, Cex, but not XylD, has low activity on cellulose. Nonetheless,<br>
like several other bacterial xylanases (see Gilbert et al, (1993) J. Gen.<br>
Microbiol., 139:187), they have CBDs. C. fimi probably produces other p-<br>
1,4-glycanases. Similar systems are produced by related bacteria (see<br>
Wilson (1992) Crit. Rev. Biotechnol., 12:45; and Hazlewood e/a/., (1992)<br>
J. Appl. Bacteriol., 72:244). Unrelated bacteria also produce glycanases;<br>
Clostridium thermocellum, for example, produces twenty or more P-1,4-<br>
glycanases (see Beguin et al, (1992) FEMS Microbiol. Lett., 100:523).<br>
The CBD derived from C.  fimi endoglucanase C Nl, is the only protein<br><br>
known to bind soluble cellosaccharides and one of a small set of proteins that are known to bind any soluble polysaccharides.<br>
Examples of suitable binding domains are shown in Figure 1 of PCT/CA97/00033 (WO 97/26358), which presents an aligmnent of binding domains from various enzymes that bind to polysaccharides and identifies amino acid residues that are conserved among most or all of the enzymes. This information can be used to derive a suitable oligonucleotide probe using methods known to those of skill in the art.   The probes can be considerably shorter than the entire sequence but should at least be 10, preferably at least 14, nucleotides in length.   Longer oligonucleotides are usefiil, up to the full length of the gene, preferably no more than 500, more preferably no more than 250, nucleotides in length.  RNA or DNA probes can be used. In use, the probes are typically labeled in a detectable manner, for example, with ^^p^ 3H, biotin, avidin or other detectable reagents, and are incubated with single-stranded DNA or RNA from the organism in which a gene is being sought.   Hybridization is detected by means of the label after the unhybridized probe has been separated from the hybridized probe. The hybridized probe is typically immobilized on a solid matrix such as nitrocellulose paper.    Hybridization techniques suitable for use with oligonucleotides are well known to those skilled in the art. Although probes are normally used with a detectable label that allows easy identification, unlabeled oligonucleotides are also useful, both as precursors of labeled probes and for use in methods that provide for direct detection of double-stranded DNA (or DNA/RNA).    Accordingly, the term "oligonucleotide probe" refers to both labeled and unlabeled forms.<br>
Generally, the binding domains identified by probing nucleic acids from an organism of interest will show at least about 40 % identity (including as appropriate allowances for conservative substitutions, gaps for better alignment and the like) to the binding region or regions from which the probe was derived and will bind to a soluble (3-1,4 glucan with a Ka of ^ 10^ M"l. More preferably, the binding domains will be at least about 60 % identical, and most preferably at least about 70 % identical to the binding region used to derive the probe.  The percentage of identity will be greater among those amino acids that are conserved among polysaccharidase binding domains.   Analyses of amino acid sequence comparisons can be performed using programs in PC/Gene (IntelliGenetics, Inc.). PCLUSTAL can be used for multiple sequence alignment and generation of phylogenetic trees.<br><br>
In order to isolate the polysaccharide binding protein or a polysaccharide binding domain from an enzyme or a cluster of enzymes that binds to a polysaccharide, several genetic approaches can be used. One method uses restriction enzymes to remove a portion of the gene that codes for portions of the protein other than the binding portion thereof The remaining gene fragments are fused with expression control sequences to obtain a mutated gene that encodes a truncated protein. Another method involves the use of exonucleases such as BaB 1 to systematically delete nucleotides either externally from the 5' and the 3' ends of the DNA or internally from a restricted gap within the gene.. These gene deletion methods result in a mutated gene encoding a shortened protein molecule which can then be evaluated for substrate or polysaccharide binding ability.<br>
Any cellulose binding protein or cellulose binding domain may be used in the present invention. The term "cellulose binding protein" ("CBP") refers to any protein or polypeptide which specifically binds to cellulose. The cellulose binding protein may or may not have cellulose or cellulolytic activity. The term "cellulose binding domain" ("CBD") refers to any protein or polypeptide which is a region or portion of a larger protein, said region or portion binds specifically to cellulose. The cellulose binding domain (CBD) may be a part or portion of a cellulase, xylanase or other polysaccharidase, e.g., a chitinase, etc., a sugar binding protein such as maltose binding protein, or scaffoldin such as CbpA of Clostridium celluvorans, etc.  Many cellulases and hemicellulases (e.g., xylanases and mannases) have the ability to associate with cellulose. These enzymes typically have a catalytic domain   containing   the   active   site   for   substrate   hydrolysis   and   a carbohydrate-binding domain or cellulose-binding domain  for binding cellulose.    The CBD may also be from a non-catalytic polysaccharide binding protein. To date, more than one hundred cellulose-binding domains (CBDs) have been classified into at least thirteen families designated I-XIII (Tomme et al.   (1995)  "CelluloseBinding Domains:  Classification  and Properties", in ACS Symposium Series 618 Enzymatic Degradation and Insoluble Carbohydrates, pp.   142-161, Saddler and Penner eds., American Chemical Society, Washington, D.C.    (Tomme I) ; Tomme et al. Adv. Microb.   Physiol. (1995) 37:1 (Tomme II); and Smant et al, Proc.   Natl. Acad.    Sci U.S.A. (1998) 95:4906,-4911, all of which are incorporated herein by reference). Any of the CBDs described in Tomme I or II or any variants thereof, any other presently known CBDs or any new CBDs which may be identified can be used in the present invention.  As an illustrative,<br><br>
but in no way limiting example, the CBP or CBD can be from a bacterial, fungal, slime mold, or nematode protein or polypeptide.    For a more particular illustrative example, the CBD is obtainable from Clostridium cellulovorans, Clostridium cellulovorans, or Cellulomonas fimi (e.g., CenA, CenB, CenD, Cex).   In addition, the CBD may be selected from a phage display peptide or peptidomimetic library, random or otherwise, using e.g., cellulose as a screening agent.  (See Smith Science (1985) 228:1315-1317 and Lam, Nature (1991) 354:82-84).       Furthermore,  the  CBD  may  be derived by mutation of a portion of a protein or polypeptide which binds to a polysaccharide other than cellulose (or hemicellulose) but also binds cellulose, such as a chitinase, which specifically binds chitin, or a sugar binding protein such as maltose binding protein, rendering said portion capable of binding to cellulose.  In any event, the CBD binds cellulose or hemicellulose.   Shoseyov and Doi (Proc.   Natl.   Acad.   Sci. USA (1990) 87:2192-2195) isolated a unique cellulose-binding protein (CbpA) from the cellulose "complex" of the cellulolytic bacterium Clostridium cellulovorans. This major subunit of the cellulose complex was found to bind to cellulose, but had no hydrolytic activity, and was essential for the degradation of crystalline cellulose.    The CbpA gene has been cloned and sequenced (Shoseyov et al. Proc.   Natl.   Acad.   Sci.   USA (1992) 89:3483-3487). Using PCR primers flanking the cellulose-binding domain of CbpA, the latter was successfully cloned into an overexpression vector that enabled overproduction of the approximately 17 kDa CBD in Escherichia coli. The recombinant CBD exhibits very strong affinity to cellulose and chitin (U.S. Pat.  No.  5,496,934; Goldstein et al., J.  Bacteriol. (1993) 175:5762; PCT International Publication WO 94/24158, all are incorporated by reference as if ftilly set forth herein).<br>
In recent years, several CBDs have been isolated from different sources. Most of these have been isolated from proteins that have separate catalytic, i.e., cellulose and cellulose binding domains, and only two have been isolated from proteins that have no apparent hydrolytic activity but possess cellulose-binding activity (Goldstein et al. J. Bacteriol. (1993) 175:5762-5768; Morag et al. Appl. (1995) Environ. Microbiol. 61:1980-1986).<br>
Cellulose binding peptide-recombinant protein fusions: The fusion of two proteins for which genes has been isolated, such as a cellulose binding peptide and an oxidase, such as a laccase, is well known and regularly practiced in the art. Such fusion involves the joining together<br><br>
ot neteroiogous nucieic acia sequences, m namt, au»^ii uiai. LIUIUIUI-IUJIJ thereof results in the generation of a fused protein product or a fusion proteins. Methods, such as the polymerase chain reaction (PCR), restriction, nuclease digestion, ligation, synthetic oligonucleotides synthesis and the like are typically employed in various combinations in the process of generating fusion gene constructs. One ordinarily skilled in the art can readily form such constructs for any pair or more of individual proteins. Interestingly, in most cases where such fusion or chimera proteins are produced, and in all cases where one of the proteins was a cellulose binding peptide, both the former and the latter retained their catal3^ic activity or fimction. In any case, an in frame spacer can be included. The length thereof may range, for example, from several to several dozens of amino acids. Such a spacer may also function to reduce mobilization constraints.<br>
For example, Greenwood et al. (1989, FEES Lett. 224:127-131) fused the cellulose binding region of Cellulomonas fimi endoglucanase to the enzyme alkaline phosphatase. The recombinant fusion protein retained both its phosphatase activity and the ability to bind to cellulose. For more descriptions of cellulose binding flision proteins, see U.S. Patent No. 5,137,819 issued to Kilbum et al, and U.S. Patent No. 5,719,044 issued to Shoseyov et al. both incorporated by reference herein. See also U.S. Pat. No. 5,474,925. All of which are incorporated herein by reference.<br>
Thus, according to the present invention there is provided a nucleic acid molecule comprising a promoter sequence for directing protein expression in plant cells and a heterologous nucleic acid sequence including a first sequence encoding a cellulose binding peptide; and a second sequence encoding an enzyme being capable of catalyzing the oxidation of phenolic groups, wherein the first and second sequences are joined together in frame.<br>
According to a preferred embodiment of the invention the nucleic acid molecule further comprising a sequence element selected from the group consisting of an origin of replication for propagation in bacterial cells, at least one sequence element for integration into a plant's genome, a polyadenylation recognition sequence, a transcription termination signal, a sequence encoding a translation start site, a sequence encoding a translation stop site, plant RNA virus derived sequences, plant DNA virus derived sequences, tumor inducing (Ti) plasmid derived sequences, a transposable element derived sequence and a plant operative signal peptide for directing a protein to a cellular compartment of a plant cell.<br><br>
According to still a preferred embodiment, the cellular compartment is selected from the group consisting of cytoplasm, endoplasmic reticulum, golgi apparatus, oil bodies, starch bodies, chloroplastids, chloropiasts, chromoplastids, chromoplasts, vacuole, lysosomes, mitochondria, and nucleus.<br>
Genetically modified plant material:<br>
The present invention employs recombinant nucleic acid molecules. Such a molecule includes, for example, a promoter sequence for directing protein expression in plant cells; and a heterologous nucleic acid sequence as further detailed herein, wherein, the heterologous nucleic acid sequence is down stream the promoter sequence, such that expression of the heterologous nucleic acid sequence is effectable by the promoter sequence. Such a nucleic acid molecule needs to be effectively introduced into plant cells, so as to genetically modify the plant.<br>
There are various methods of introducing foreign genes into both monocotyledonous and dicotyledenous plants (Potrykus, I., Armu. Rev. Plant. Physiol., Plant. Mol. Biol. (1991) 42:205-225; Shimamoto et al. Nature (1989) 338:274-276). The principle methods of causing stable integration of exogenous DNA into plant genomic DNA include two main approaches:<br>
(i) Agrobacterium-mtdXZiXtd. gene transfer: Klee et al. (1987) Annu. Rev. Plant Physiol. 38:467-486; Klee and Rogers in Cell Culture and Somatic Cell Genetics of Plants, Vol. 6, Molecular Biology of Plant Nuclear Genes, eds. Schell, J., and Vasil, L. K., Academic Publishers, San Diego, Calif (1989) p. 2-25; Gatenby, in Plant Biotechnology, eds. Kung, S. and Amtzen, C. J., Butterworth Publishers, Boston, Mass. (1989) p. 93-112.<br>
(ii) direct DNA uptake: Paszkowski et al, in Cell Culture and Somatic Cell Genetics of Plants, Vol. 6, Molecular Biology of Plant Nuclear Genes eds. Schell, J., and Vasil, L. K., Academic Publishers, San Diego, Calif (1989) p. 52-68; including methods for direct uptake of DNA into protoplasts, Toriyama, K. et al. (1988) Bio/Technology 6:1072-1074. DNA uptake induced by brief electric shock of plant cells: Zhang et al Plant Cell Rep. (1988) 7:379-384. Fromm et al Nature (1986) 319:791-793. DNA injection into plant cells or tissues by particle bombardment, Klein et al Bio/Technology (1988) 6:559-563; McCabe et al Bio/Technology (1988) 6:923-926; Sanford, Physiol. Plant. (1990) 79:206-209; by the use of micropipette systems: Neuhaus et al, Theor.   Appl.<br><br>
Genet. (1987) 75:30-36; Neuhaus and Spangenberg, Physiol. Plant. (1990) 79:213-217; or by the direct incubation of DNA with germinating pollen, DeWet et al. in Experimental Manipulation of Ovule Tissue, eds. Chapman, G. P. and Mantell, S. H. and Daniels, W. Longman, London, (1985) p. 197-209; and Ohta,Proc. Natl. Acad. Sci. USA (1986) 83:715-719.<br>
The Agrobacterium system includes the use of plasmid vectors that contain defined DNA segments that integrate into the plant genomic DNA. Methods of inoculation of the plant tissue vary depending upon the plant species and the Agrobacterium delivery system. A widely used approach is the leaf disc procedure which can be performed with any tissue explant that provides a good source for initiation of whole plant differentiation. Horsch et al. in Plant Molecular Biology Manual A5, Kluwer Academic Publishers, Dordrecht (1988) p. 1-9. The Agrobacterium system is especially viable in the creation of transgenic dicotyledenous plants.<br>
There are various methods of direct DNA transfer into plant cells. In electroporation, the protoplasts are briefly exposed to a strong electric field. In microinjection, the DNA is mechanically injected directly into the cells using very small micropipettes. In microparticle bombardment, the DNA is adsorbed on microprojectiles such as magnesium sulfate crystals or tungsten particles, and the microprojectiles are physically accelerated into cells or plant tissues.<br>
Following transformation plant propagation is exercised. The most common method of plant propagation is by seed. Regeneration by seed propagation, however, has the deficiency that due to heterozygosity there is a lack of uniformity in the crop, since seeds are produced by plants according to the genetic variances governed by Mendelian rules. Basically, each seed is genetically different and each will grow with its own specific traits. Therefore, it is preferred that the transgenic plant be produced such that the regenerated plant has the identical traits and characteristics of the parent transgenic plant, e.g., a reproduction of the ftision protein. Therefore, it is preferred that the transgenic plant be regenerated by micropropagation which provides a rapid, consistent reproduction of the transgenic plants.<br>
Micropropagation is a process of growing new generation plants from a single piece of tissue that has been excised from a selected parent plant or cultivar. This process permits the mass reproduction of plants having  the preferred  tissue  expressing  the  fijsion  protein.     The  new<br><br>
generation plants which are produced are genetically identical to, and have all of the characteristics of, the original plant. Micropropagation allows mass production of quality plant material in a short period of time and offers a rapid multiplication of selected cultivars in the preservation of the characteristics of the original transgenic or transformed plant. The advantages of cloning plants are the speed of plant multiplication and the quality and uniformity of plants produced.<br>
Micropropagation is a multi-stage procedure that requires alteration of culture medium or growth conditions between stages. Thus, the micropropagation process involves four basic stages: Stage one, initial tissue culturing; stage two, tissue culture multiplication; stage three, differentiation and plant formation; and stage four, greenhouse culturing and hardening. During stage one, initial tissue culturing, the tissue culture is established and certified contaminant-free. During stage two, the initial tissue culture is multiplied until a sufficient number of tissue samples are produced to meet production goals. During stage three, the tissue samples grown in stage two are divided and grown into individual plantlets. At stage four, the transgenic plantlets are transferred to a greenhouse for hardening where the plants' tolerance to light is gradually increased so that it can be grown in the natural environment.<br>
The basic bacterial/plant vector construct will preferably provide a broad host range prokaryote replication origin; a prokaryote selectable marker; and, for Agrobacterium transformations, T DNA sequences for Agrobacterium-medxzitd transfer to plant chromosomes. Where the heterologous sequence is not readily amenable to detection, the construct will preferably also have a selectable marker gene suitable for determining if a plant cell has been transformed. A general review of suitable markers for the members of the grass family is found in Wilmink and Dons, Plant Mol. Biol. Reptr. (1993)11:165-185.<br>
Sequences suitable for permitting integration of the heterologous sequence into the plant genome are also recommended. These might include transposon sequences and the like for homologous recombination as well as Ti sequences which permit random insertion of a heterologous expression cassette into a plant genome.<br>
Suitable prokaryote selectable markers include resistance toward antibiotics such as ampicillin or tetracycline. Other DNA sequences encoding additional functions may also be present in the vector, as is known in the art.<br><br>
The constructs of the subject invention will include an expression<br>
cassette for expression of the fusion protein of interest.  Usually, there will<br>
be only one expression cassette, although two or more are feasible.   The<br>
recombinant expression cassette will contain in addition to the heterologous<br>
sequence one or more of the following sequence elements, a promoter<br>
region, plant 5' untranslated sequences, initiation codon depending upon<br>
vhether or not the structural gene comes  equipped with one, and a<br>
ranscription and translation termination  sequence.     Unique restriction<br>
inzyme sites at the 5' and 3' ends of the cassette allow for easy insertion into<br>
I pre-existing vector.<br>
Viral infected plant material:<br>
Viruses are a unique class of infectious agents whose distinctive features are their simple organization and their mechanism of replication, [n fact, a complete viral particle, or virion, may be regarded mainly as a t&gt;lock of genetic material (either DNA or RNA) capable of autonomous replication, surrounded by a protein coat and sometimes by an additional membranous envelope such as in the case of alpha viruses. The coat protects the virus from the environment and serves as a vehicle for transmission from one host cell to another.<br>
Viruses that have been shown to be useful for the transformation of plant hosts include CaV, TMV and BV. Transformation of plants using plant viruses is described in U.S. Pat. No. 4,855,237 (BGV), EP-A 67,553 (TMV), Japanese Published Application No. 63-14693 (TMV), EPA 194,809 (BV), EPA 278,667 (BV); and Gluzman, Y. et al. Communications in Molecular Biology: Viral Vectors, Cold Spring Harbor Laboratory, New York, pp. 172-189 (1988). Pseudovirus particles for use in expressing foreign DNA in many hosts, including plants, is described in WO 87/06261.<br>
Construction of plant RNA viruses for the introduction and expression of non-viral foreign genes in plants is demonstrated by the above references as well as by Dawson, W. O. et al. Virology (1989) 172:285-292; Takamatsu et al. EMBO J. (1987) 6:307-311; French et al Science (1986) 231:1294-1297; and Takamatsu et al FEBS Letters (1990) 269:73-76.<br>
When the virus is a DNA virus, the constructions can be made to the virus itself. Altematively, the virus can first be cloned into a bacterial plasmid for ease of constructing the desired viral vector with the foreign DNA.   The virus can then be excised from the plasmid.   If the virus is a<br><br>
DNA virus, a bacterial origin of replication can be attached to the viral DNA, which is then replicated by the bacteria. Transcription and translation of this DNA will produce the coat protein which will encapsidate the viral DNA. If the virus is an RNA virus, the virus is generally cloned as a cDNA and inserted into a plasmid. The plasmid is then used to make all of the constructions. The RNA virus is then produced by transcribing the viral sequence of the plasmid and translation of the viral genes to produce the coat protein(s) which encapsidate the viral RNA.<br>
Construction of plant RNA viruses for the introduction and expression of non-viral foreign genes in plants is demonstrated by the above references as well as in U.S. Pat. No. 5,316,931<br>
In one embodiment, a plant viral nucleic acid is provided in which the native coat protein coding sequence has been deleted from a viral nucleic acid, a non-native plant viral coat protein coding sequence and a non-native promoter, preferably the subgenomic promoter of the non-native coat protein coding sequence, capable of expression in the plant host, packaging of the recombinant plant viral nucleic acid, and ensuring a systemic infection of the host by the recombinant plant viral nucleic acid, has been inserted. Alternatively, the coat protein gene may be inactivated by insertion of the non-native nucleic acid sequence within it, such that a fusion protein is produced. The recombinant plant viral nucleic acid may contain one or more additional non-native subgenomic promoters. Each non-native subgenomic promoter is capable of transcribing or expressing adjacent genes or nucleic acid sequences in the plant host and incapable of recombination with each other and with native subgenomic promoters. Non-native (foreign) nucleic acid sequences may be inserted adjacent the native plant viral subgenomic promoter or the native and a non-native plant viral subgenomic promoters if more than one nucleic acid sequence is included. The non-native nucleic acid sequences are transcribed or expressed in the host plant under control of the subgenomic promoter to produce the desired products.<br>
In a second embodiment, a recombinant plant viral nucleic acid is provided as in the first embodiment except that the native coat protein coding sequence is placed adjacent one of the non-native coat protein subgenomic promoters instead of a non-native coat protein coding sequence.<br>
In a third embodiment, a recombinant plant viral nucleic acid is provided in which the native coat protein gene is adjacent its subgenomic<br><br>
promoter and one or more non-native subgenomic promoters have been inserted into the viral nucleic acid. The inserted non-native subgenomic promoters are capable of transcribing or expressing adjacent genes in a plant host and are incapable of recombination with each other and with native subgenomic promoters. Non-native nucleic acid sequences may be inserted adjacent the non-native subgenomic plant viral promoters such that said sequences are transcribed or expressed in the host plant under control of the subgenomic promoters to produce the desired product.<br>
In a fourth embodiment, a recombinant plant viral nucleic acid is provided as in the third embodiment except that the native coat protein coding sequence is replaced by a non-native coat protein coding sequence.<br>
The viral vectors are encapsidated by the coat proteins encoded by the recombinant plant viral nucleic acid to produce a recombinant plant virus. The recombinant plant viral nucleic acid or recombinant plant virus is used to infect appropriate host plants. The recombinant plant viral nucleic acid is capable of replication in the host, systemic spread in the host, and transcription or expression of foreign gene(s) in the host to produce the desired fiision protein.<br>
Fusion protein comparimentalizatiou - signal peptides:<br>
As already mentioned hereinabove, compartmentalization of the fusion protein is an important feature of the present invention because it allows undisturbed plant growth. Thus, according to one aspect of the present invention, the ftision protein is compartmentalized within cells of the plant or cultured plant cells, so as to be sequestered from cell walls of the cells of the plant or cultured plant cells.<br>
The fusion protein can be compartmentalized within a cellular compartment, such as, for example, the cytoplasm, endoplasmic reticulum, golgi apparatus, oil bodies, starch bodies, chloroplastids, chloroplasts, chromoplastids, chromoplasts, vacuole, lysosomes, mitochondria or the nucleus.<br>
Accordingly, the heterologous sequence used while implementing the process according to this aspect of the present invention includes (i) a first sequence encoding a cellulose binding peptide; (ii) a second sequence encoding a recombinant protein, wherein the first and second sequences are joined together in frame; and (iii) a third sequence encoding a signal peptide for directing a protein to a cellular compartment, the third sequence being upstream and in frame with the first and second sequences.<br><br>
The following provides description of signal peptides which can be jsed to direct the fusion protein according to the present invention to jpecific cell compartments.<br>
It   is   well-known   that   signal   peptides   serve   the   function   of translocation   of  produced   protein   across   the   endoplasmic   reticulum membrane.     Similarly, transmembrane segments halt translocation and provide anchoring of the protein to the plasma membrane, see, Johnson et al The Plant Cell (1990) 2:525-532; Sauer et al.  EMBO J. (1990) 9:3045-3050; Mueckler et al. Science (1985) 229:941-945. Mitochondrial, nuclear, chloroplast, or vacuolar signals target expressed protein correctly into the corresponding organelle through the secretory pathway, see, Von Heijne, Eur.   J.   Biochem. (1983) 133:17-21; Yon Heijne, J.   Mol.   Biol. (1986) 189:239-242; Iturriaga et al. The Plant Cell (1989) 1:381-390; McKnight et al, Nucl. Acid Res. (1990) 18:4939-4943; Matsuoka and Nakamura, Proc. Natl. Acad. Sci. USA (1991) 88:834-838. A recent book by Cunningham and Porter (Recombinant proteins from plants, Eds.  C.   Cunningham and A.J.R.  Porter, 1998 Humana Press Totowa, N.J.) describe methods for the production of recombinant proteins in plants and methods for targeting the proteins to different compartments in the plant cell.    In particular, two chapters therein  (14  and  15) describe different methods to introduce targeting sequences that results in accumulation of recombinant proteins in compartments such as ER, vacuole, plastid, nucleus and cytoplasm.   The book by Cunningham and Porter is incorporated herein by reference. Presently, the preferred site of accumulation of the fusion protein according to the present invention is the ER using signal peptide such as Cel 1 or the rice amylase signal peptide at the N-terminus and an ER retaining peptide (HDEL or KDEL) at the C-terminus.<br>
Promoters and control of expression:<br>
Any promoter which can direct the expression of the fusion protein according to the present invention can be utilized to implement the process of the instant invention, both constitutive and tissue specific promoters. According to presently preferred embodiment the promoter selected is constitutive, because such a promoter can direct the expression of higher levels of the fusion protein. In this respect the present invention offers a major advantage over the teachings of U.S. Pat. No. 5,474,925 in which only tissue specific and weak promoters can be employed because of the deleterious effect of the fusion protein described therein on cell wall development. The reason for which the present invention can utilize strong<br><br>
and constitutive promoters relies in the compartmentalization and sequestering approach which prohibits contact between the expressed fusion protein and the plant cell walls which such walls are developing.<br>
Constitutive and tissue specific promoters, CaMV35S promoter (Odell et al. Nature (1985) 313:810-812) and ubiquitin promoter (Christensen and Quail, Transgenic research (1996) 5:213-218) are the most commonly used constitutive promoters in plant transformations and are the preferred promoters of choice while implementing the present invention.<br>
In com, within the kernel, proteins under the ubiquitin promoters, are preferentially accumulated in the germ (Kusnadi et al., Biotechnol. Bioeng. (1998) 60:44-52).     The amylose-extender (Ae) gene encoding starch-branching enzyme lib (SBEIIb) in maize is predominantly expressed in endosperm and embryos during kernel development (Kim ei al. Plant. Mol. Biol. (1998) 38:945-956).    A starch branching enzyme (SBE) showed promoter activity after it was introduced into maize endosperm suspension cells by particle bombardment (Kim e/a/.   Gene (1998) 216:233-243).   In transgenic wheat it has been shown that a native HMW-GS gene promoter can be used to obtain high levels of expression of seed storage and, potentially, other proteins in the endosperm (Blechl and Anderson, Nat. Biotechnol. (1996) 14:875-9). Polygalacturonase (PG) promoter was shown to confer high levels of ripening-specific gene expression in tomato (Nicholass et al. Plant. Mol. Biol. (1995) 28:423-435). The ACC oxidase promoter (Blume and Grierson, Plant.   J. (1997) 12:731-746) represents a promoter from the ethylene pathway and shows increased expression during fruit ripening and senescence in tomato.    The promoter for tomato 3-hydroxy-3-methylglutaryl coenzyme A reductase gene accumulates to high level during fioiit ripening (Daraselia et al. Plant. Physiol. (1996) 112:727-733).   Specific protein expression in potato tubers can be mediated by the patatin promoter (Sweetlove et al.   Biochem.   J.   (1996) 320:487-492). Protein linked to a chloroplast transit peptide changed the protein content in transgenic soybean and canola seeds when expressed from a seed-specific promoter (Falco et al. Biotechnology (NY) (1995) 13:577-82).   The seed specific bean phaseolin and soybean beta-conglycinin promoters are also suitable for the latter example (Keeler et al.   Plant.   Mol.   Biol. (1997) 34:15-29).   Promoters that are expressed in plastids are also suitable in conjunction with plastid transformation.<br>
Each of these promoters can be used to implement the process according to the present invention.<br><br>
Thus, the plant promoter employed can a constitutive promoter, a tissue specific promoter, an inducible promoter or a chimeric promoter.<br>
Examples of constitutive plant promoters include, without being limited to, CaMV35S and CaMV19S promoters, FMV34S promoter, sugarcane bacilliform badnavirus promoter, CsVMV promoter, Arabidopsis ACT2/ACT8 actin promoter, Arabidopsis ubiquitin UBQl promoter, barley leaf thionin BTH6 promoter, and rice actin promoter.<br>
Examples of tissue specific promoters include, without being limited to, bean phaseolin storage protein promoter, DLEC promoter, PHSp promoter, zein storage protein promoter, conglutin gamma promoter from soybean, AT2SI gene promoter, ACTll actin promoter from Arabidopsis, napA promoter from Brassica napus and potato patatin gene promoter.<br>
The inducible promoter is a promoter induced by a specific stimuli such as stress conditions comprising, for example, light, temperature, chemicals, drought, high salinity, osmotic shock, oxidant conditions or in case of pathogenicity and include, without being limited to, the light-inducible promoter derived from the pea rbcS gene, the promoter from the alfalfa rbcS gene, the promoters DRE, MYC and MYB active in drought; the promoters INT, INPS, prxEa, Ha hspl7.7G4 and RD21 active in high salinity and osmotic stress, and the promoters hsr303J and str246C active in pathogenic stress.<br>
Expression follow up:<br>
Expression of the fusion protein can be monitored by a variety of methods. For example, ELISA or western blot analysis using antibodies specifically recognizing the recombinant protein or its cellulose binding peptide counterpart can be employed to qualitatively and/or quantitatively monitor the expression of the fusion protein in the plant. Alternatively, the fusion protein can be monitored by SDS-PAGE analysis using different staining techniques, such as, but not limited to, coomasie blue or silver staining. Other methods can be used to monitor the expression level of the RNA encoding for the fusion protein. Such methods include RNA hybridization methods, e.g., Northern blots and RNA dot blots.<br>
Thus, according to the present invention there is provided a genetically modified or viral infected plant or cultured plant cells expressing a fusion protein including an enzyme being capable of catalyzing the oxidation of phenolic groups and a cellulose binding peptide.<br>
According to a preferred embodiment of the present invention the fusion protein is compartmentalized within cells of said plant or cultured<br><br>
plant cells, so as to be sequestered from cell walls of said cells of said plant or cultured plant cells, so as not to hamper development and to allow higher expression, if so required. According to a preferred embodiment the fusion protein is compartmentalized within a cellular compartment selected from the group consisting of cytoplasm, endoplasmic reticulum, golgi apparatus, oil  bodies,   starch  bodies,   chloroplastids,   chloroplasts,   chromoplastids, chromoplasts, vacuole, lysosomes, mitochondria, and nucleus. Determination of Oxidase and Peroxidase Activity: When employing a polynucleotide encoding a laccase in the process of the invention, an amount of laccase in the range of 0.02-2000 laccase units (LACU) per gram of dry lignocellulosic material will generally be suitable; when employing peroxidases, an amount thereof in the range of 1.02-2000 peroxidase units (PODU) per gram of dry lignocellulosic material vill generally be suitable.<br>
The determination of oxidase (e.g., laccase) activity is based on the )xidation of syringaldazin to tetramethoxy azo bis-methylene quinone under lerobic conditions, and 1 LACU is the amount of enzyme which converts 1 xM of syringaldazin per minute under the following conditions: 19 |iM ;yringaldazin, 23.2 mM acetate buffer, 30 °C., pH 5.5, reaction time 1 Tiinute, shaking; the reaction is monitored spectrophotometrically at 530 im.<br>
With respect to peroxidase activity, 1 PODU is the amount of enzyme which catalyses the conversion of 1 jimol of hydrogen peroxide per minute under the following conditions: 0.88 mM hydrogen peroxide, 1.67 mM 2,2'-azinobis(3-ethylbenzothiazoline-6-sulfonate), 0.1 M phosphate buffer, pH 7.0, incubation at 30 °C.; the reaction is monitored photometrically at 418 nm.<br>
Binding of the fusion protein to the plant derived cellulosic matter: When sufficient expression has been detected, binding of the fusion protein to the plant derived cellulosic matter is effected. Such binding can be achieved, for example, as follows. Whole plants, plant derived tissue or cultured plant cells are homogenized by mechanical method in the presence or absence of a buffer, such as, but not limited to, PBS. The fusion protein is therefore given the opportunity to bind to the plant derived cellulosic matter. Buffers that may include salts and/or detergents at optimal concentrations may be used to wash non specific proteins from the cellulosic matter.<br><br>
Thus, further according to the present invention there is provided a composition of matter comprising a cell wall preparation derived from a genetically modified or virus infected plant or cultured plant cells expressing a fusion protein including an enzyme being capable of catalyzing the oxidation of phenolic groups and a cellulose binding peptide, said fusion protein being immobilized to cellulose in said cell wall preparation via said cellulose binding peptide.<br>
Oxidizing agents:<br>
The enzyme(s) and oxidizing agent(s) used in the process of the invention should clearly be matched to one another, and it is clearly preferable that the oxidizing agent(s) in question participate(s) only in the oxidative reaction involved in the binding process, and does/do not otherwise exert any deleterious effect on the substances/materials involved in the process.<br>
Oxidases, e.g. laccases, are, among other reasons, well suited in the context of the invention since they catalyze oxidation by molecular oxygen. Thus, reactions taking place in vessels open to the atmosphere and involving an oxidase as enzyme will be able to utilize atmospheric oxygen as oxidant; it may, however, be desirable to forcibly aerate the reaction medium during the reaction to ensure an adequate supply of oxygen.<br>
In the case of peroxidases, hydrogen peroxide is a preferred peroxide in the context of the invention and is suitably employed in a concentration (in the reaction medium) in the range of 0.01-100 mM.<br>
pH in the Reaction Medium:<br>
Depending, inter alia, on the characteristics of the enzyme(s) employed, the pH in the aqueous medium (reaction medium) in which the process of the invention takes place will be in the range of 3-10, preferably in the range 4-9.<br>
General Procedures:<br>
Generally, the nomenclature used herein and the laboratory procedures utilized when practicing the present invention include molecular, biochemical, microbiological and recombinant DNA techniques. Such techniques are thoroughly explained in the literature. See, for example, "Molecular Cloning: A laboratory Manual" Sambrook et al., (1989); "Current Protocols in Molecular Biology" Volumes I-III Ausubel, R. M., ed. (1994); Ausubel et al., "Current Protocols in Molecular Biology", John Wiley and Sons, Baltimore, Maryland (1989); Perbal, "A Practical Guide to Molecular Cloning", John Wiley &amp; Sons, New York (1988); Watson et al.,<br><br>
"Recombinant DNA", Scientific American Books, New York; Birren et al. (eds) "Genome Analysis: A Laboratory Manual Series", Vols. 1-4, Cold Spring Harbor Laboratory Press, New York (1998); methodologies as set forth in U.S. Pat. Nos. 4,666,828; 4,683,202; 4,801,531; 5,192,659 and 5,272,057; "Cell Biology: A Laboratory Handbook", Volumes I-III Cellis, J. E., ed. (1994); "Current Protocols in Immunology" Volumes MIT Coligan J. E., ed. (1994); Stites et al. (eds), "Basic and Clinical Immunology" (8th Edition), Appleton &amp; Lange, Norwalk, CT (1994); Mishell and Shiigi (eds), "Selected Methods in Cellular Immunology", W. H. Freeman and Co., New York (1980); available immunoassays are extensively described in the patent and scientific literature, see, for example, U.S. Pat. Nos. 3,791,932; 3,839,153;    3,850,752;   3,850,578;   3,853,987;   3,867,517;    3,879,262; 3,901,654;    3,935,074;   3,984,533;   3,996,345;   4,034,074;    4,098,876; 4,879,219; 5,011,771 and 5,281,521; "Oligonucleotide Synthesis" Gait, M. J., ed. (1984); "Nucleic Acid Hybridization" Hames, B. D., and Higgins S. J., eds. (1985); "Transcription and Translation" Hames. B. D., and Higgins S. J., eds. (1984); "Animal Cell Culture" Freshney, R. I., ed. (1986); "Immobilized Cells and Enzymes" IRL Press, (1986); "A Practical Guide to Molecular Cloning" Perbal, B., (1984) and "Methods in Enzymology" Vol. 1-317, Academic Press; "PCR Protocols:  A Guide To Methods And Applications", Academic Press, San Diego, CA (1990); Marshak et al., "Strategies for Protein Purification and Characterization - A Laboratory Course Manual" CSHL Press (1996); all of which are incorporated by reference as if fully set forth herein. Other general references are provided throughout this document.  The procedures therein are believed to be well known in the art and are provided for the convenience of the reader. All the information contained therein is incorporated herein by reference.<br>
Product by Process:<br>
The present invention also relates to a lignocellulose-based product obtainable by a process according to the invention as disclosed herein.<br>
Although the invention has been described in conjunction with specific embodiments thereof, it is evident that many alternatives, modifications and variations will be apparent to those skilled in the art. Accordingly, it is intended to embrace all such alternatives, modifications and variations that fall within the spirit and broad scope of the appended claims. All publications cited herein are incorporated by reference in their entirety.<br><br><br>
WE CLAIM:<br>
1.	A process for the manufacture of a lignocellulose product such as herein<br>
described, the process comprising the step of mixing in a reaction medium:<br>
(i)       a phenolic polymer being substituted with a phenolic hydroxy group <br>
(ii) a lignocellulose containing material having immobilized to a cellulosic fraction thereof a fusion polypeptide, said fusion polypeptide having an enzyme being capable of catalyzing the oxidation of phenolic groups and a cellulose binding peptide; and<br>
(iii)     An oxidizing agent.<br>
2.	The process as claimed in claim 1, wherein said lignocellulose product is selected from the group consisting of fiber board, particle board, flakeboard, plywood and molded composites.<br>
3.	The process as claimed in claim 1, wherein said Hgnocellulose product is selected from the group consisting of paper and paperboard.<br>
4.	The process as claimed in claim 1, wherein said lignocellulose containing material is a cell wall preparation derived from a genetically modified or virus infected plant or cultured plant cells expressing said fusion protein.<br>
5.	The process as claimed in claim 1, wherein said lignocellulose containing material is selected from the group consisting of vegetable fiber and wood fiber derived from a genetically modified or virus infected plant expressing said fusion polypeptide.<br><br>
6.	The process as claimed in claim 1, wherein the phenolic substituent is selected from the group consisting of p-coumaric acid, p-coumaryl alcohol, coniferyl alcohol, sinapyl alcohol, ferulic acid and p-hydroxybenzoic acid.<br>
7.	The process as claimed in claim 1, wherein said phenolic polymer forms an integral part of said lignocellulose containing material.<br>
8.	The process as claimed in claim 7, wherein said phenolic polymer is lignin.<br>
9.	The process as claimed in claim 1, wherein said phenolic polymer is a phenolic polysaccharide.<br>
10.	The process as claimed in claim 9, wherein the polysaccharide portion of the phenolic polysaccharide is selected from the group consisting of modified and unmodified starches, modified and unmodified cellulose, and modified and unmodified hemicelluloses.<br>
11.	The process as claimed in claim 9, wherein said phenolic polysaccharide is selected from the group consisting of ferulylated arabinoxylans and ferulylated pectins.<br>
12.	The process as claimed in claim 1, wherein said reaction medium is incubated for a period of from 1 minute to 10 hours.<br>
13.	The process as claimed in claim 12, wherein said fusion polypeptide is incubated in the presence of said oxidizing agent for a period of from 1 minute to 10 hours.<br><br>
14.	The process as claimed in claim 1, wherein said enzyme is selected from the group consisting of oxidases and peroxidases.<br>
15.	The process as claimed in claim 1, wherein said enzyme is an oxidase selected from the group consisting of laccases (EC 1.10.3.2), catechol oxidases (EC 1.10.3.1) and bilirubin oxidases (EC 1.3.3.5), and said oxidizing agent is oxygen.<br>
16.	The process as claimed in claim 15, wherein said enzyme is a laccase and is present in an amount in the range of 0.02-2000 LACU per g of dry lignocellulose.<br>
17.	The process as claimed in claim 15, wherein said reaction medium is aerated.<br>
18.	The process as claimed in claim 15, wherein said enzyme is a laccase encoded by a polynucleotide obtained from a fungus of the genus Botrytis, Myceliophthora, Trametes or the plant Acer pseudoplanus.<br>
19.	The process as claimed in claim 18, wherein the fungus is Trametes versicolor or Trametes villosa.<br>
20.	The process as claimed in claim 1, wherein said enzyme is a peroxidase and said oxidizing agent is hydrogen peroxide.<br>
21.	The process as claimed in claim 20, wherein said peroxidase is present in an amount in the range of 0.02-2000 PODU per g of dry lignocellulose, and the initial concentration of hydrogen peroxide in the reaction medium is in the range of 0.01-100 mM.<br><br>
22.	The process as claimed in claim 1, wherein the amount of lignocellulose<br>
employed corresponds to 0.1-90 % by weight of the reaction medium,<br>
calculated as dry lignocellulose.<br>
23.	The process as claimed in claim 1, wherein the temperature of the reaction<br>
medium is in the range of 10° - 120°C.<br>
24.	The process as claimed in claim 23, wherein the temperature of the reaction<br>
medium is in the range of 15° - 90°C.<br>
25.	The process as claimed in claim 1, wherein an amount of said phenolic polymer<br>
in the range of 0.1 % - 10 % by weight.<br>
26.	The process as claimed in claim 1, wherein the pH in the reaction medium is in<br>
the range of 3-10.<br>
27.	The process as claimed in claim 26, wherein the pH in the reaction medium is<br>
in the range of 4-9.<br>
28.	A lignocellulose product obtained by the process as claimed in claim 1.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDg1Ny1jaGUgYWJzdHJhY3QtZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0857-che abstract-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDg1Ny1jaGUgY2xhaW1zLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-0857-che claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDg1Ny1jaGUgZGVzY3JpcHRpb24oY29tcGxldGUpLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-0857-che description(complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDg1Ny1jaGUgZHJhd2luZ3MtZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0857-che drawings-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDg1Ny1jaGUgZm9ybS0xOS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-0857-che form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDg1Ny1jaGUgcGN0IHNlYXJjaCByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-0857-che pct search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDg1Ny1jaGUgcGV0aXRpb24ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-0857-che petition.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItODU3LWNoZS0gYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-857-che- abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItODU3LWNoZS0gYXNzaWdubWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-857-che- assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItODU3LWNoZS0gY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-857-che- claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItODU3LWNoZS0gY29ycmVzcG9uZGVuY2Ugb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-857-che- correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItODU3LWNoZS0gY29ycmVzcG9uZGVuY2UgcG8ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-857-che- correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItODU3LWNoZS0gZGVzY3JpcGl0aW9uIGNvbXBsZXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-857-che- descripition complete.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItODU3LWNoZS0gZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-857-che- form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItODU3LWNoZS0gZm9ybSAyNi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-857-che- form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItODU3LWNoZS0gZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-857-che- form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItODU3LWNoZS0gZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-857-che- form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItODU3LWNoZS0gZm9ybSA2LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-857-che- form 6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItODU3LWNoZS0gcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-857-che- pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="218964-electrochemical-test-strip-for-use-in-analyte-determination.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="218966-new-process-for-pre-treating-cellulosic-fibers-and-cellulosic-fiber-blends.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>218965</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2002/857/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>23/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>06-Jun-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>16-Apr-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>07-Jun-2002</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>YISSUM RESEARCH DEVELOPMENT COMPANY</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SHOSEYOV, Oded</td>
											<td></td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>D21C9/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IL2000/000665</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-10-19</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/166, 389</td>
									<td>1999-11-18</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/164, 140</td>
									<td>1999-11-08</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/218965-a-process-for-preparing-a-lignocellulose-based-product-and-the-product-obtained-by-the-process by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:22:12 GMT -->
</html>
