<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/219122-a-method-for-identifying-a-target-base-sequence-of-rna-interference-and-an-apparatus-used-therein by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:11:52 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 219122:A METHOD FOR IDENTIFYING A TARGET BASE SEQUENCE OF RNA INTERFERENCE AND AN APPARATUS USED THEREIN</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR IDENTIFYING A TARGET BASE SEQUENCE OF RNA INTERFERENCE AND AN APPARATUS USED THEREIN</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>In the present invention, a sequence segment conforming to the following rules (a) to (d) is searched from the base sequences of a target gene of RNA interference and, based on the search results, siRNA capable of causing RNAi is designed, synthesized, etc.: (a) The 3&#x27; end base is adenine, thymine, or uracil, (b) The 5&#x27; end base is guanine or cytosine, (c) A 7-base sequence from the 3&#x27; end is rich in one or more types of bases selected from the group consisting of adenine, thymine, and uracil, and (d) The number of &quot;bases is within a range that allows RNA interference to occur without causing cytotoxicity.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
SPECIFICATION<br>
METHOD FOR SEARCHING TARGET BASE SEQUENCE OF RNA INTERFERENCE, METHOD FOR DESIGNING BASE SEQUENCE<br>
OF POLYNUCLEOTIDE FOR CAUSING RNA INTERFERENCE,<br>
METHOD FOR PRODUCING DOUBLE-STRANDED POLYNUCLEOTIDE,<br>
METHOD FOR INHIBITING GENE EXPRESSION, BASE SEQUENCE PROCESSING APPARATUS, PROGRAM FOR RUNNING BASE<br>
SEQUENCE PROCESSING METHOD ON COMPUTER, RECORDING<br>
MEDIUM, AND BASE SEQUENCE PROCESSING SYSTEM<br>
TECHNICAL FIELD<br>
The present invention relates to RNA interference and more particularly, for example, to a method for designing sequences of polynucleotides for causing RNA interference, the method improving efficiency in testing, manufacturing, etc., in which RNA interference is used.  Hereinafter, RNA interference may also be referred to as "RNAi".<br>
The present invention further relates to a base sequence processing apparatus, a program for running a base sequence processing method on a computer, a recording medium, and a base sequence processing system.  In particular, the invention relates to a base sequence processing apparatus capable of efficiently selecting a base sequence from the base sequences of a target gene, which causes RNA interference in a target gene, a program for running a base sequence processing method on a computer, a recording medium, and a base sequence processing system.<br>
BACKGROUND ART<br><br>
RNA interference is a phenomenon of gene destruction wherein double-stranded RNA comprising sense RNA and anti-sense RNA (hereinafter also referred to as "dsRNA") homologous to a specific region of a gene to be functionally inhibited, destructs the target gene by causing interference in the homologous portion of mRNA which is a transcript of the target gene.  RNA interference was first proposed in 1998 following an experiment using nematodes.  However, in mammals, when long dsRNA with about 30 or more base pairs is introduced into cells, an interferon response is induced, and cell death occurs due to apoptosis.  Therefore, it was difficult to apply the RNAi method to mammals.<br>
On the other hand, it was demonstrated that RNA interference could occur in early stage mouse embryos and cultured mammalian cells, and it was found that the induction mechanism of RNA interference also existed in the mammalian cells.  At present,, it has been demonstrated that short double-stranded RNA with about 21 to 23 base pairs (short interfering RNA, siRNA) can induce RNA interference without exhibiting cytotoxicity even in the mammalian cell system, and it has become possible to apply the RNAi method to mammals.<br>
DISCLOSURE OF INVENTION<br>
The RNAi method is a technique which is expected to have various applications. However, while dsRNA or siRNA that is homologous to a specific region of a gene, exhibits<br><br>
an RNA interference effect in most of the sequences in drosophila and nematodes, 70% to 80% of randomly selected (21 base) siRNA do not exhibit an RNA interference effect in mammals.  This poses a great problem when gene functional analysis is carried out using the RNAi method in mammals.<br>
Conventional designing of siRNA has greatly depended on the experiences and sensory perceptions of the researcher or the like, and it has been difficult to design siRNA actually exhibiting an RNA interference effect with high probability.  Other factors that prevent further research being conducted on RNA interference and its various applications are high costs and time consuming procedures required for carrying out an RNA synthesis resulting in part from the unwanted synthesis of siRNA.<br>
Under such circumstances, it is an object of the present invention to provide a more efficient and simplified means for the RNAi method.<br>
In order to achieve the above object, the present inventors have studied a technique for easily obtaining siRNA, which is one of the steps requiring the greatest effort, time, and cost when the RNAi method is used.  In view of the fact that preparation of siRNA is a problem especially in mammals, the present inventors have attempted to identify the sequence regularity of siRNA effective for , RNA interference using mammalian cultured cell systems.  As a result, it has been found that effective siRNA sequences have certain regularity, and thereby, the present invention<br><br>
has been completed.  Namely, the present invention is as described below.<br>
[1] A method for searching a target base sequence of RNA interference comprising: searching a sequence segment, conforming to the following rules (a) to (d), from the base ' sequences of a target gene for RNA interference:<br>
(a)	The 3' end base is adenine, thymine, or uracil,<br>
(b)	The 5' end base is guanine or cytosine,<br>
(c)	A 7-base sequence from the 3! end is rich in one or more types of bases selected from the group consisting of adenine, thymine, and uracil, and<br>
(d)	The number of bases is within a range that allows RNA interference to occur without causing cytotoxicity.<br>
[2] The method for searching the target base sequence<br>
according to item [1], wherein, in the rule (c), at least<br>
three bases among the seven bases are one or more types of<br>
bases selected from the group consisting of adenine,<br>
thymine, and uracil.<br>
[3] The method for searching the target base sequence<br>
according to item [1] or [2], wherein, in the rule (d), the<br>
number of bases is 13 to 28.<br>
[4] A method for designing a base sequence of a<br>
polynucleotide for causing RNA interference comprising:<br>
searching a base sequence, conforming to the rules (a) to<br>
(d) below, from the base sequences of a target gene and<br>
designing a base sequence homologous to the searched base<br>
sequence:<br>
(a) The 3' end base is adenine, thymine, or uracil.<br><br>
(b)	The 5' end base is guanine or cytosine,<br>
(c)	A 7-base sequence from the 3' end is rich in one or more types of bases selected from the group consisting of adenine, thymine, and uracil, and<br>
(d)	The number of bases is within a range that allows RNA interference to occur without causing cytotoxicity.<br>
[5] The method for designing the base sequence according to item [4], wherein, in the rule (c), at least three bases among the seven bases are one or more types of bases selected from the group consisting of adenine, thymine, and uracil.<br>
[6] The method for designing the base sequence according to item [4] or [5], wherein the number of bases in the homologous base sequence designed is 13 to 28. [7] The method for designing the base sequence according to any one of items [4] to [6], wherein designing is performed so that at least 80% of bases in the homologous base sequence designed corresponds; to the base sequence searched. [8] The method for designing the base sequence according to any one of items [4] to [7], wherein the 3' end base of the base sequence searched is the same as the 3' end base of the base sequence designed, and the 5'end base of the base sequence searched is the same as the 5' end base of the base sequence designed.<br>
[9] The method for designing the base sequence according to any one of items [4] to [8], wherein an overhanging portion is added to the 3' end of the polynucleotide. [10] A method for producing a double-stranded<br><br>
polynucleotide comprising: forming one strand by providing an overhanging portion to the 3' end of a base sequence homologous to a prescribed sequence which is contained in the base sequences of a target gene and which conforms to the following rules (a) to (d), and forming the other strand by providing an overhanging portion to the 3! end of a base sequence complementary to the base sequence homologous to the prescribed sequence, wherein the number of bases in each strand is 15 to 30:<br>
(a)	The 3' end base is adenine, thymine, or uracil,<br>
(b)	The 5' end base is guanine or cytosine,<br>
(c)	A 7-base sequence from the 3' end is rich in one or more types of bases selected from the group consisting of adenine, thymine, and uracil, and<br>
(d)	The number of bases is within a range that allows RNA interference to occur without causing cytotoxicity.<br>
[11] A double-stranded polynucleotide synthesized by searching a sequence segment having 13 to 28 bases, conforming to the following rules (a) to (d), from the base sequences of a target gene for RNA interference, forming one strand by providing an overhanging portion to the 3' end of a base sequence homologous to a prescribed sequence which is contained in the base sequences of the target gene and which conforms to the following rules (a) to (d), and forming the other strand by providing an overhanging portion to the 3' end of a base sequence complementary to the base sequence homologous to the prescribed sequence, wherein the number of bases in each strand is 15 to 30:<br><br>
(a)	The 3' end base is adenine, thymine, or uracil,<br>
(b)	The 5' end base is guanine or cytosine,<br>
(c)	A 7-base sequence from the 3'end is rich in one or more types of bases selected from the group consisting of adenine, thymine, and uracil, and<br>
(d)	The number of bases is within a range that allows RNA<br>
"interference to occur without causing cytotoxicity.<br>
[12] A method for inhibiting gene expression comprising the steps of searching a sequence segment having 13 to 28 bases, conforming to the following rules (a) to (d), from the base sequences of a target gene for RNA interference, synthesizing a double-stranded polynucleotide such that one strand is formed by providing an overhanging portion to the 3' end of a base sequence homologous to a prescribed' sequence which is contained in the base sequences of the target gene and which conforms to the following rules (a) to (d), the other strand is formed by providing an overhanging portion to the 3' end of a base sequence complementary to the base sequence homologous to the prescribed sequence, and the number of bases in each strand is 15 to 30, and adding the synthesized double-stranded polynucleotide to an expression system of the target gene of which expression is to be inhibited to inhibit the expression of the target gene:<br>
(a)	The 3*.end base is adenine, thymine, or uracil,<br>
(b)	The 51 end base is guanine or cytosine,<br>
(c)	A 7-base sequence from the 3f end is rich in one or more types of bases selected from the group consisting of<br><br>
adenine, thymine, and uracil, and<br>
(d) The number of bases is within a range that allows RNA interference to occur without causing cytotoxicity. [13] A base sequence processing apparatus characterized in that it comprises partial base sequence creation means for acquiring base sequence information of a target gene for RNA interference and creating partial base sequence information corresponding to a sequence segment having a predetermined number of bases in the base sequence information; 3' end base determination means for determining whether the 3' end base in the partial base sequence information created by the partial base sequence creation means'is adenine, thymine, or uracil; 5' end base determination means for determining whether the 5' end base in the partial base sequence information created by the partial base sequence creation means is guanine or cytosine; predetermined base inclusion determination means for determining whether base sequence information comprising 7 bases at the 3' end in the partial base sequence information created by the partial base sequence creation means is rich in one or more types of bases selected from the group consisting of adenine, thymine, and uracil; and prescribed sequence selection means for selecting prescribed sequence information which specifically causes RNA interference in the target gene from the partial base sequence information created by the partial base sequence creation means, based on the results determined by the 3' base determination means, the 5' end<br><br>
base determination means, and the predetermined base inclusion determination means.<br>
[14] The base sequence processing apparatus according to item [13], characterized in that the partial base sequence creation means further comprises region-specific base sequence creation means for creating the partial base sequence information having the predetermined number of bases from a segment corresponding to a coding region or transcription region of the target gene in the base sequence information.<br>
[15] The base sequence processing apparatus according to item [13] or [14], characterized in that the partial base sequence creation means further -comprises common base sequence creation means for creating the partial base sequence information having the predetermined number of bases which is common in a plurality of base sequence information derived from different organisms. [16] The base sequence processing apparatus according to any one of items [13] to [15], characterized in that the base sequence information that is rich corresponds to base sequence information comprising the 7 bases containing at least 3 bases which are one or more types of bases selected from the group consisting of adenine, thymine, and uracil. [17] The base sequence processing apparatus according to any one of items [13] to [16], wherein the predetermined number of bases is 13 to 28.<br>
[18] The base sequence processing apparatus according to any one of iter.: .2}    c.    ;.'/'], characterise.:, „: -..-.-<br><br>
partial base sequence creation means further comprises overhanging portion-containing base sequence creation means for creating the partial base sequence information containing an overhanging portion.<br>
[19] The base sequence processing apparatus according to any one of items [13] to [17], characterized in that it comprises overhanging-portion addition means for adding an overhanging portion to at least one end of the prescribed sequence information.<br>
[20] The base sequence processing apparatus according to item [18] or [19], wherein the number of bases in the overhanging portion is 2.<br>
[21] The base sequence processing apparatus according to any one of items [13] to [20], characterized in that it comprises identical/similar base sequence search means for searching base sequence information, identical or similar to the prescribed sequence information, from other base sequence information, and unrelated gene target evaluation means for evaluating whether the prescribed sequence information targets genes unrelated to the target gene based on the identical or similar base sequence information searched by the identical/similar base sequence search means.<br>
[22] The base sequence processing apparatus according to item [21], characterized in that the unrelated gene target evaluation means further comprises total sum calculation means for calculating the total sum of reciprocals of the values showing t&gt;. r ^e:rffj-' ' ~ : *■-:-":"_-■■- ^r -■---■-■--'   ■ -~<br><br>
on the total amount of base sequence information on the genes unrelated to the target gene in the identical or similar base sequence information searched by the identical/similar base sequence search means and the values showing the degree of identity or similarity attached to the base sequence information on the genes unrelated to the target gene, and total sum-based target evaluation means for evaluating whether the prescribed sequence information targets the genes unrelated to the target gene based on the total sum calculated by the total sum calculation means. [23] A program for running base sequence processing method on a computer, characterized in that it comprises a partial base sequence creation step of acquiring base sequence information of a target gene for RNA interference and creating partial base sequence information corresponding to a sequence segment having a predetermined number of bases in the base sequence information; a 3' end base determination step of determining whether the 3' end base in the partial base sequence information created in the partial base sequence creation step is adenine, thymine, or uracil; a 5' end base determination step of determining whether the 5' end base in the partial base sequence information created in the partial base sequence creation step is guanine or cytosine; a predetermined base inclusion determination step of determining whether base sequence information comprising 7 bases at the 3' end in the partial base sequence information created in the partial base sequence creation  ~ ~z-   ::. ■: .:i '" - "   ?; c-r ■*.:.: -.<br><br>
bases selected from the group consisting of adenine, thymine, and uracil; and a prescribed sequence selection step of selecting, based on the results determined in the 3' base determination step, the 5' end base determination   step, and the predetermined base inclusion determination step, 'prescribed sequence information which specifically causes RNA interference in the target gene from the partial base sequence information created in the partial base sequence creation step.<br>
[24] A computer-readable recording medium characterized in that the program according to item [23] is recorded in the medium.<br>
[25] A base sequence processing system which comprises a base sequence processing apparatus which processing base sequence information of a target gene for RNA interference and a client apparatus, the base sequence processing apparatus and the client apparatus being connected to each other via a network in a communicable manner, characterized in that the client apparatus comprises base sequence transmission means for transmitting a name of the target gene or the base sequence information to the base sequence processing apparatus, and prescribed sequence acquisition means for acquiring prescribed sequence information which is transmitted from the base sequence processing apparatus and which specifically causes RNA interference in the target gene, and the base sequence processing apparatus comprises partial base sequence creation means for<br>
■--■'■•'-      '	■    -\ r-*   "..	'-    —    -■--    - ■	, .<br>
- » . * ^&gt; ■ _ ■    ,..---.-.■		- _. j ..<br><br>
name of the target gene or the base sequence information transmitted from the client apparatus and creating partial base sequence information corresponding to a sequence segment having a predetermined number of bases in the base sequence information; 3' end base determination means for determining whether the 3' end base in the partial base sequence information created by the partial base sequence creation means is adenine, thymine, or uracil; 5' end base determination means for determining whether the 5' end base in the partial base sequence information created by the partial base sequence creation means is guanine or cytosine; predetermined base inclusion determination means for determining whether base sequence information comprising 7 bases at the 3? end in the partial base sequence information created by the partial base sequence creation means is rich in one or more types of bases selected from the group consisting of adenine, thymine, and uracil; prescribed sequence selection means for selecting the prescribed sequence information from the partial base sequence information created by the partial base sequence creation means, based on the results determined by the 3' base determination means, the 5'end base determination means, and the predetermined base inclusion determination means; and prescribed sequence transmission means for transmitting the prescribed sequence information selected by the prescribed sequence selection means to the client apparatus.<br><br>
that it comprises a partial base sequence creation step of acquiring base sequence information of a target gene for RNA interference and creating partial base sequence information corresponding to a sequence segment having a predetermined number of bases in the base sequence information; a 3' end base determination step of determining whether the 3' end base in the partial base sequence information created in the partial base sequence creation step is adenine, thymine, or uracil; a 5' end base determination step of determining whether the 5' end base in the partial base sequence information created in the partial base sequence creation step is guanine or cytosine; a predetermined base inclusion determination step of determining whether base sequence information comprising 7 bases at the 3' end in the partial base sequence information created in the partial base sequence creation step is rich in one or more types of bases selected from the group consisting of adenine, thymine, and uracil; and a prescribed sequence selection step of selecting, based on the results determined in the 3f base determination step, the 5! end base determination step, and the predetermined base inclusion determination step, prescribed sequence information which specifically causes RNA interference in the target gene from the partial base sequence information created in the partial base sequence creation step. [27] The base sequence processing method according to item [26], characterized in that the partial base sequence creation step further comprises a regicr.-specific base<br><br>
sequence creation step of creating the partial base sequence information having the predetermined number of bases from a segment corresponding to a coding region or transcription region of the target gene in the base sequence information.<br>
[28] The base sequence processing method according to item [26] or [27], characterized in that the partial base sequence creation step further comprises a common base sequence creation step for creating the partial base sequence information having the predetermined number of bases which is common in a plurality of base sequence information derived from different organisms. [29] The base sequence processing method according to any one of items [26] to [28], characterized in that the base sequence information that is rich corresponds to base sequence information comprising the 7 bases containing at least 3 bases which are one or more types of bases selected from from the group consisting of adenine, thymine, and uracil.<br>
[30] The base sequence processing method according to any one of items [26] to [29], wherein the predetermined number of bases is 13 to 28.<br>
[31] The base sequence processing method according to any one of items [26] to [30], characterized in that the partial base sequenpe creation step further comprises an overhanging portion-containing base sequence creation step of creating the partial base sequence information containing an overhanging portion.<br><br>
[32] The base sequence processing method according to any one of items [26] to [30], characterized in that it comprises an overhanging-portion addition step of adding an overhanging portion to at least one end of the prescribed sequence information.<br>
[33] The base sequence processing method according to item [31] or [32], wherein the number of bases in the overhanging portion is 2.<br>
[34] The base sequence processing method according to any one of items [26] to [33], characterized in that it comprises an identical/similar base sequence search step of searching base sequence information identical or similar to the prescribed sequence information from other base sequence information, and unrelated gene target evaluation step of evaluating whether the prescribed sequence information targets genes unrelated to the target gene based on the identical or similar base sequence information searched in the identical/similar base sequence search step. [35] The base sequence processing method according to item [34], characterized in that the unrelated gene target evaluation step further comprises a total sum calculation step of calculating the total sum of reciprocals of the values showing the degree of identity or similarity based on the total amount of base sequence information on the genes unrelated to the target gene in the identical or similar base sequence information searched in the identical/similar base sequence search step and the values<br><br>
the base sequence information on the genes unrelated to the target gene, and a total sum-based target evaluation step of evaluating whether the prescribed sequence information targets the genes unrelated to the target gene based on the total sum calculated in the total sum calculation step. [36] The program according to item [23], characterized in that the partial base sequence creation step further comprises a region-specific base sequence creation step of creating the partial base sequence information having the predetermined number of bases from a segment corresponding to a coding region or transcription region of the target gene in the base sequence information. [37] The program according to item [23].or [36], characterized in that the partial base sequence creation step further comprises a common base sequence creation step of creating the partial base sequence information having the predetermined number of bases which is common in a plurality of base sequence information derived from different organisms.<br>
[38] The program according to any one of items [23], [36], and [37], characterized in that the base sequence information that is rich corresponds to base sequence information comprising the 7 bases containing at least 3 bases which are one or more types of bases selected from the group consisting of adenine, thymine, and uracil, [39] The program according to any one of items [23], [36], [37], and [38], wherein the predetermined number of bases<br><br>
[40] The program according to any one of items [23], [36], [37], [38], and [39], characterized in that the partial base sequence creation step further comprises an overhanging portion-containing base sequence creation step of creating the partial base sequence information containing an overhanging portion.<br>
[41] The program according to any one of items [23], [36], [37], [38], and [39], characterized in that it comprises an overhanging-portion addition step of adding an overhanging portion to at least one end of the prescribed sequence information,<br>
[42] The program according to item [40] or [41], wherein the number of bases in the overhanging portion is 2. [43] The program according to any one of items [23], [36], [37], [38], [39], [40], [41], and [42], characterized in that it comprises an idential/similar base sequence search step of searching base sequence information identical or similar to the prescribed sequence information from other base sequence information, and an unrelated gene target evaluation step of evaluating whether the prescribed sequence information targets genes unrelated to the target gene based on the identical or similar base sequence information searched in the identical/similar base sequence search step.<br>
[44] The program according to item [43], characterized in that the unrelated gene target evaluation step further comprises a total sum calculation step of calculating the<br><br>
of identity or similarity based on the total amount of base sequence information on the genes unrelated to the target gene in the identical or similar base sequence information searched in the identical/similar base sequence search step and the values showing the degree of identity or similarity attached to the base sequence information on the genes unrelated to the target gene, and a total sum-based target evaluation step of evaluating whether the prescribed sequence information targets the genes unrelated to the target gene based on the total sum calculated in the total sum calculation step.<br>
[45] A computer-readable recording medium characterized in that the program according to any one of items [23] and [36] to [44] is recorded in the medium.<br>
[46] The base sequence processing system according to item [25], characterized in that, in the base sequence processing apparatus, the partial base sequence creation means further comprises region-specific base sequence creation means for creating the partial base sequence information having the predetermined number of bases from a segment corresponding to a coding region or transcription region of the target gene in the base sequence information. [47] The base sequence processing system according to item [25] or [46], characterized in that, in the base sequence processing apparatus, the partial base sequence creation means further comprises common base sequence creation means for creating the partial base sequence information having<br><br>
plurality of base sequence information derived from different organisms.<br>
[48] The base sequence processing system according to any one of items [25], [46], and [47], characterized in that, in the base sequence processing apparatus, the base sequence information that is rich corresponds to base sequence information comprising the 7 bases containing at least 3 bases which are one or more types of bases selected from the group consisting of adenine, thymine, and uracil. [49] The base sequence processing system according to any one of items [25], [46], [47], and [48], wherein, in the base sequence processing apparatus, the predetermined number of bases is 13 to 28.<br>
[50] The base sequence processing system according to any one of items [25], [46], [47], [48], and [49], characterized in that, in the base sequence processing apparatus, the partial base sequence creation means further comprises overhanging portion-containing base sequence creation means for creating the partial base sequence information containing an overhanging portion.<br>
[51] The base sequence processing system according to any one of items [25], [46], [47], [48], and [49], characterized in that the base sequence processing apparatus comprises overhanging-portion addition means for adding an overhanging portion to at Least one end of the prescribed sequence information.<br>
[52] The base sequence processing system according to item<br><br>
apparatus, the number of bases in the overhanging portion is 2.<br>
[53] The base sequence processing system according to any one of items [25], [46], [47], [48], [49], [50], [51], and [52], characterized in that the base sequence processing apparatus comprises identical/similar base sequence search means for searching base sequence information identical or similar to the prescribed sequence information from other base sequence information, and unrelated gene target evaluation means for evaluating whether the prescribed sequence information targets genes unrelated to the target gene based on the identical or similar base sequence information searched by the identical/similar base sequence search means -<br>
[54] The base sequence processing system according to item [53], characterized in that, in the base sequence processing apparatus, the unrelated gene target evaluation means further comprises total sum calculation means for calculating the total sum of reciprocals of the values showing the degree of identity or similarity based on the total amount of base sequence information on the genes unrelated to the target gene in the identical or similar base sequence information searched by the identical/similar base sequence search means and the values showing the degree of identity or similarity attached to the base sequence information on the genes unrelated to the target gene, and total sum-based target evaluation means for<br><br>
targets the genes unrelated to the target gene based on the total sum calculated by the total sum calculation means.<br>
Brief Description of the Drawings<br>
Fig. 1 is a diagram which shows the designing of siRNA corresponding to sequences common to human and mice. Fig. 2 is a diagram which shows the regularity of siRNA exhibiting an RNAi effect.  Fig. 3 is a diagram which shows common segments (shown in bold letters) having prescribed sequences in the base sequences of human FBP1 and mouse Fbpl.  Fig. 4 is a diagram listing prescribed sequences common to human FBP1 and mouse Fbpl.  Fig. 5 is a diagram in which the prescribed sequences common to human FBP1 and mouse Fbpl are scored.  Fig. 6 is a diagram showing the results of BLAST searches on one of the prescribed sequences performed so that genes other than the target are not knocked out.  Fig. 7 is a diagram showing the results of BLAST searches on one of the prescribed sequences performed so that genes other than the target are not knocked out.  Fig. 8 is a diagram showing an output result of a program.  Fig. 9 is a diagram which shows the designing of RNA fragments (a to p).  Fig. 10 is a diagram showing the results of testing whether siRNA a to p exhibited an RNAi effect, in which "B" shows the results in drosophila cultured cells, and "C" shows the results in human cultured cells.  Fig. 11 is a diagram showing the analysis results concerning the characteristics of<br><br>
showing the basic principle of the present invention.  Fig. 13 is a block diagram which shows an example of the configuration of a base sequence processing apparatus 100 of the system to which the present invention is applied. Fig. 14 is a diagram which shows an example of information stored in a target gene base sequence file 106a.  Fig, 15 is a diagram which shows an example of information stored in a partial base sequence file 106b.  Fig. 16 is a diagram which shows an example of information stored in a determination result file 106c.  Fig. 17 is a diagram which shows an example of information stored in a prescribed sequence file 106d.  Fig. 18 is a diagram which shows an example of information stored in a reference sequence database 106e.  Fig. 19 is a diagram which shows an example of information stored in a degree of identity or similarity file 106f.  Fig. 20 is a diagram which shows an example of information stored in an evaluation result file 106g. Fig. 21 is a block diagram which shows an example of the structure of a partial base sequence creation part 102a of the system to which the present invention is applied. Fig. 22 is a block diagram which shows an example of the structure of an unrelated gene target evaluation part 102h of the system to which the present invention is applied. Fig. 23 is a flowchart which shows an example of the main processing of the system in^the embodiment.  Fig. 24 is a flowchart which shows an example of the unrelated gene evaluation process of the system in the embodiment.<br><br>
expression vector pTREC.  Fig. 26 is a diagram which shows the results of PCR in which one of the primers in Example 2, 2. (2) is designed such that no intron is inserted. Fig. 27 is a diagram which shows the results of PCR in which one of the primers in Example 2, 2. (2) is designed such that an intron is inserted.  Fig. 28 is a diagram which shows the sequence and structure of siRNA; siVIM35. Fig. 29 is a diagram which shows the sequence and structure of siRNA; siVIM812.  Fig. 30 is a diagram which shows the sequence and structure of siRNA; siControl.  Fig. 31 is a diagram which shows the results of assay of RNAi activity of siVIM812 and siVIM35.  Fig. 32 is a diagram which shows RNAi activity of siControl, siVIM812, and siVIM35 against vimentin.  Fig. 33 is a diagram which shows the results of antibody staining.  Fig. 34 is a diagram which shows the assay results of RNAi activity of siRNA designed by the program against the luciferase gene.  Fig. 3 5 is a diagram which shows the assay results of RNAi activity of siRNA designed by the program against the sequences of SARS virus.<br>
BEST MODE FOR CARRYING OUT THE INVENTION<br>
The embodiments of the present invention will be described below in the order of the columns  to .  Method for searching target base sequence of RNA interference<br> Method for designing base sequence of polynucleotide for causing RNA interference<br><br> Method for inhibiting gene expression<br> siRNA sequence design program<br> siRNA sequence design business model system<br> Base sequence processing apparatus for running siRNA<br>
sequence design program, etc.<br> Method for searching target base sequence of RNA interference<br>
The search method of the present invention is a method for searching a base sequence, which causes RNA interference, from the base sequences of a target gene. Specifically, in the search method of the present invention, a sequence segment conforming to the following rules (a) to (d) is searched from the base sequences of a target gene for RNA interference.<br>
(a)	The 3' end base is adenine, thymine, or uracil.<br>
(b)	The 5' end base is guanine or cytosine.<br>
(c)	A 7-base sequence from the 3' end is rich in one or more types of bases selected from the group consisting of adenine, thymine, and uracil.<br>
(d)	The number of bases is within a range that allows RNA interference to occur without causing cytotoxicity.<br>
The term "gene" in the term "target gene" means a medium which codes for genetic information.  The "gene" consists of a substance, such as DNA, RNA, or a complex*of DNA and RNA, which codes for genetic information.  As the genetic information, instead of the substance itself, * r - — -   ■"" :  jf base sequences can ~,~ ; . .<br><br>
computer or the like.  The "target gene" may be set as one coding region, a plurality of coding regions, or all the polynucleotides whose sequences have been revealed.  When a gene with a particular function is desired to be searched, by setting only the particular gene as the target, it is possible to efficiently search the base sequences which cause RNA interference specifically in the particular gene. Namely, RNA interference is known as a phenomenon which destructs mRNA by interference, and by selecting a particular coding region, search load can be reduced. Moreover, a group of transcription regions may be treated as the target region to be searched.  Additionally, in the present specification, base sequences are shown on the basis of sense strands, i.e., sequences of mRNA, unless otherwise described.  Furthermore, in the present specification, a base sequence which satisfies the rules (a) to (d) is referred to as a "prescribed sequence".  In the rules, thymine corresponds to a DNA base sequence, and uracil corresponds to an RNA base sequence.<br>
The rule (c) regulates so that a sequence in the vicinity of the 3' end contains a rich amount of type(s) of base(s) selected from the group consisting of adenine, thymine, and uracil, and more specifically, as an index for search, regulates so that a 7-base sequence from the 3' end is rich in one or more types of bases selected from adenine, thymine, and uracil.<br>
In the rule (c), the phrase "sequence rich in" means '."*.".t ;:'.:a frequ-..cy of a _;;/-'^._. case appearing _LS high, and<br><br>
schematically, a 5 to 10-base sequence, preferably a 7-base sequence, from the 3' end in the prescribed sequence contains one or more types of bases selected from adenine, thymine, and uracil in an amount of preferably at least 40% or more, and more preferably at least 50%.  More specifically, for example, in a prescribed sequence of about 19 bases, among 7 bases from the 3' end, preferably at least 3 bases, more preferably at least 4 bases, and particularly preferably at least 5 bases, are one or more types of bases selected from the group consisting of adenine, thymine, and uracil.<br>
The means for confirming the correspondence to the rule (c) is not particularly limited as long as it can be confirmed that preferably at least 3 bases, more preferably at least 4 bases, and particularly preferably at least 5 bases, among 7 bases are adenine, thymine, or uracil.  For example, a case, wherein inclusion of 3 or more bases which correspond to one or more types of bases selected from the group consisting of adenine, thymine, and uracil in a 7-base sequence from the 3' end is defined as being rich, will be described below.  Whether the base is any one of the three types of bases is checked from the first base at the 3' end one after another, and when three corresponding bases appear by the seventh base, conformation to the rule (c) is determined.  For example, if three corresponding bases appear by the third base, checking of three bases is sufficient.  That is, in the search with respect to the rule (c), it is not always necessary to check all of the<br><br>
seven bases at the 3' end.  Conversely, non-appearance of three or more corresponding bases by the seventh base means being not rich, thus being determined that the rule (c) is not satisfied.<br>
In a double-stranded polynucleotide, it is well-known that adenine complementarily forms hydrogen-bonds to thymine or uracil.  In the complementary hydrogen bond between guanine and cytosine (G-C hydrogen bond), three hydrogen bonding sites are formed.  On the other hand, the complementary hydrogen bond between adenine and thymine or uracil (A-(T/U) hydrogen bond) includes two hydrogen bonding sites.  Generally speaking, the bonding strength of the A-(T/U) hydrogen bond is weaker than that of the G-C hydrogen bond.<br>
In the rule (d), the number of bases of the base sequence to be searched is regulated.  The number of bases of the base sequence to be searched corresponds to the number of bases capable of causing RNA interference. Depending on the conditions, for example the species of an organism, in cases of siRNA having an excessively large number of bases, cytotoxicity is known to occur.  The upper limit of the number of bases varies depending on the species of organism to which RNA interference is desired to be caused.  The number of bases of the single strand constituting siRNA is preferably 30 or less regardless of the species.  Furthermore, in mammals, the number of bases is preferably 24 or less, and more preferably 22 or less. The lower limit, which is not particularly ii~.:.:■..     -^ng<br><br>
as RNA interference is caused, is preferably at least 15, more preferably at least 18, and still more preferably at least 20.  With respect to the number of bases as a single strand constituting siRNA, searching with a number of 21 is particularly preferable.<br>
Furthermore, although a description will be made below, in siRNA, an overhanging portion is provided at the 3! end of the prescribed sequence.  The number of bases in the overhanging portion is preferably 2.  Consequently, the upper limit of the number of bases in the prescribed sequence only, excluding the overhanging portion, is preferably 28 or less, more preferably 22 or less, and still more preferably 20 or less, and the lower limit is preferably at least 13, more preferably at least 16, and still more preferably at least 18.  In the prescribed sequence, the most preferable number of bases is 19.  The target base sequence for RNAi may be searched either including or excluding the overhanging portion.<br>
Base sequences conforming to the prescribed sequence have an extremely high probability of causing RNA interference.  Consequently, in accordance with the search method of the present invention, it is possible to search sequences that cause RNA interference with extremely high probability, and designing of polynucleotides which cause RNA interference can be simplified.<br>
In another preferred example, the prescribed sequence does not contain a sequence in which 7 or more bases of guanine (G) and/or cytosine (C) are cc-:.:..<br><br>
Examples of the sequence in which 7 or more bases of guanine and/or cytosine are continuously present include a sequence in which either guanine or cytosine is continuously present as well as a sequence in which a mixed sequence of guanine and cytosine is present.  More specific examples include GGGGGGG, CCCCCCC, and a mixed sequence of GCGGCCC.<br>
Furthermore, in the search of the prescribed sequence, detection can be efficiently performed by using a computer installed with a program which allows a search of segments conforming to the rules (a) to (a), etc., after determining the number of bases.  More specific embodiments will be described below in the columns  siRNA sequence design program and  Base sequence processing apparatus for running siRNA sequence design program.<br> Method for designing base sequence of polynucleotide for causing RNA interference<br>
In the method for designing a base sequence in accordance with the present invention, a base sequence of polynucleotide which causes RNA interference (siRNA) is designed on the basis of the base sequence searched by the search method described above.  siRNA is mainly composed of RNA.  siRNA which partially contains DNA, i.e., a hybrid polynucleotide, is also included in the examples of siRNA, In the method for designing a base sequence in accordance with the present invention, a base sequence conforming to the rules (a) to (d) is searched from the base sequences of --, *::."-■'* gene, and a base seque:/:?. homologous t<br><br>
searched base sequence is designed.  In another preferred design example, it may be possible to take into consideration a case in which the prescribed sequence does not contain a sequence in which 7 or more bases of guanine (G) and/or cytosine (C) are continuously present,  The rules (a) to (d) and the search method are the same as those described above regarding the search method of the present invention.<br>
The term "homologous sequence" refers to the same sequence and a sequence in which mutations, such as deletions, substitutions, and additions, have occurred to the same sequence to an extent that the function of causing the RNA interference has not been lost.  Although depending on the conditions, such as the type and sequence of the target gene, the range of the allowable mutation, in terms of homology, is preferably 80% or more, more preferably 90% or more, and still more preferably 9 5% or more.  When homology in the range of the allowable mutation is calculated, desirably, the numerical values calculated using the same search algorithm are compared.  The search algorithm is not particularly limited.  A search algorithm suitable for searching for local sequences is preferable. More specifically, BLAST, ssearch, or the like is preferably used.<br>
As described above, although slight modification of the searched sequence is allowable, it is particularly preferred that the number of bases in the base sequence to be designed be the same as that of the searched sequence.<br><br>
For example, with respect to the allowance for change under the same number of bases, the bases of the base sequence to be designed correspond to those of the sequence searched at a rate of preferably 80% or more, more preferably 90% or more, and particularly preferably 95% or more.  For example, when a base sequence having 19 bases is designed, preferably 16 or more bases, more preferably 18 or more bases, correspond to those of the searched base sequence. Furthermore, when a sequence homologous to the searched base sequence is designed, desirably, the 3' end base of the base sequence searched is the same as the 3' end base of the base sequence designed, and also desirably, the 5' end base of the base sequence searched is the same as the 5'■ end base of the base sequenced designed.<br>
An overhanging portion is usually provided on a siRNA molecule.  The overhanging portion is a protrusion provided on the 3f end of each strand in a double-stranded RNA molecule.  Although depending on the species of organism, the number of bases in the overhanging portion is preferably 2.  Basically, any base sequence is acceptable in the overhanging portion.  In some cases, the same base sequence as that of the target gene to be searched, TT, UU, or the like may be preferably used.  As described above, by providing the overhanging portion at the 3' end of the prescribed sequence which has been designed so as to be homologous to the base sequence searched, a sense strand constituting siRNA is designed.<br>
Alternatively, it may be possible to search the<br><br>
prescribed sequence with the overhanging portion being included from the start to perform designing.  The preferred number of bases in the overhanging portion is 2. Consequently, for example, in order to design a single strand constituting siRNA including a prescribed sequence having 19 bases and an overhanging portion having 2 bases, as the number of bases of siRNA including the overhanging portion, a sequence of 21 bases is searched from the target gene.  Furthermore, when a double-stranded state is searched, a sequence of 23 bases may be searched.<br>
In the method for designing a base sequence in accordance with the present invention, as described above, a given sequence is searched from a desired target gene. The target to which RNA interference is intended to be caused does not necessarily correspond to the origin of the target gene, and is also applicable to an analogous species, etc.  For example, it is possible to design siRNA used for a second species that is analogous to a first species using a gene isolated from the first species as a target gene. Furthermore, it is possible to design siRNA that can be widely applied to mammals, for example, by searching a common sequence from two or more species of mammals and searching a prescribed sequence from the common sequence to perform designing.  The reason for this is that it is highly probable that the sequence common to two or more       . -mammals exists in other mammals.<br>
In order to prevent RNA interference from occurring in genes not related to the target gene, preferably, a<br><br>
search is made to determine whether a sequence that is identical or similar to the designed sequence is included in the other genes.  A search for the sequence that is identical or similar to the designed sequence may be performed using software capable of performing a general homology search, etc.  By excluding such an identical/similar sequence, it is possible to design a sequence which causes RNA interference specifically to the target gene only.<br>
In the design method of the present invention, RNA molecules that cause RNA interference can be easily designed with high probability.  Although synthesis of RNA still requires effort, time, and cost, the design method of the present invention can greatly minimize them.  Method for producing double-stranded polynucleotide<br>
By the method for producing a double-stranded polynucleotide in accordance with the present invention, a double-stranded polynucleotide that has a high probability of causing RNA interference can be produced.  For the double-stranded polynucleotide of the present invention, a base sequence of the polynucleotide is designed in accordance with the method for designing the base sequence of the present invention described above, and a double-stranded polynucleotide is synthesized so as to follow the sequence design.  Preferred embodiments in the sequence design are the same as those described above regarding the method for designing the base sequence.<br>
The double-stranded polynucleotide synthesized causes<br><br>
RNA interference, and siRNA is known as such a double-stranded polynucleotide.  Additionally, the double-stranded polynucleotide produced by the production method of the present invention is preferably composed of RNA, but a hybrid polynucleotide which partially includes DNA may be acceptable.  In this specification, double-stranded polynucleotides partially including DNA are also contained in the concept of siRNA.  According to the research conducted by the present inventors, siRNA tends to have structural and functional asymmetry, and in view of the object of causing RNA interference, a half of the sense strand at the 5' end side and a half of the antisense strand at the 3' end side are desirably composed of RNA. In a double-stranded polynucleotide, one strand is formed by providing an overhanging portion to the 3f end of a base sequence homologous to the prescribed sequence conforming to the rules (a) to (d) contained in the base sequence of the target gene, and the other strand is formed by providing an overhanging portion to the 3' end of a base sequence complementary to the base sequence homologous to the prescribed sequence.  The number of bases in each strand, including the overhanging portion, is 18 to 24, more preferably 20 to 22, and particularly preferably 21. The number of bases in the overhanging portion is preferably 2.  siRNA having 21 bases in total in which the overhanging portion is composed of 2 bases is suitable for causing RNA interference with high probability without causing cytotoxicity even in mammals.<br><br>
RNA may be synthesized, for example, by chemical synthesis or by standard biotechnology.  In one technique, a DNA strand having a predetermined sequence is produced, single-stranded RNA is synthesized using the produced DNA strand as a template in the presence of a transcriptase, and the synthesized single-stranded RNA is formed into double-stranded RNA,<br>
With respect to the basic technique for molecular biology, there are many standard, experimental manuals, for example, BASIC METHODS IN MOLECULAR BIOLOGY (1986); Sambrook et al., MOLECULAR CLONING; A LABORATORY MANUAL, Second Edition, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y. (1989); Saibo-Kogaku Handbook (Handbook for cell engineering), edited by Toshio Kuroki et al., Yodosha (1992); and Shin-Idenshi-Kogaku Handbook (New handbook for genetic engineering), edited by Muramatsu et al., Yodosha (1999) .<br>
One preferred embodiment of polynucleotide produced by the production method of the present invention is a double-stranded polynucleotide produced by a method in which a sequence segment including 13 to 28 bases conforming to the rules (a) to (d) is searched from a base sequence of a target gene for RNA interference, one strand is formed by providing an overhanging portion at the 3' end of a base sequence homologous to the prescribed sequence following the rules (a) to (d), the other strand is formed by providing an overhanging portion at the 3' end of a sequence complementary to the base sequence homologous to<br><br>
the prescribed sequence, and synthesis is performed so that the number of bases in each strand is 15 to 30.  The resulting polynucleotide has a high probability of causing RNA interference.<br>
It is also possible to prepare an expression vector which expresses siRNA.  By placing a vector which expresses a sequence containing the prescribed sequence under a condition of a cell line or cell-free system in which expression is allowed to occur, it is possible to supply predetermined siRNA using the expression vector.<br>
Since conventional designing of siRNA has depended on the experiences and intuition of the researcher, trial and error have often been repeated.  However, by the double-stranded polynucleotide production method in accordance with the present invention, it is possible to produce a double-stranded polynucleotide which causes RNA interference with high probability.  In accordance with the search method, sequence design method, or polynucleotide production method of the present invention, it is possible to greatly reduce effort, time, and cost required for various experiments, manufacturing, etc., which use RNA interference.  Namely, the present invention greatly simplifies various experiments, research, development, manufacturing, etc., in which RNA interference is used, such as gene analysis, search for targets for new drug development, development of new drugs, gene therapy, and research on differences between species, and thus efficiency can be improved.<br><br> Method for inhibiting gene expression<br>
The method for inhibiting gene expression in accordance with the present invention includes a step of searching a predetermined base sequence, a step of designing and synthesizing a base sequence of siRNA based on the searched base sequence, and a step of introducing the resulting siRNA into an expression system containing a target gene.<br>
The step of searching the predetermined base sequence follows the method for searching the target base sequence for RNA interference described above.  Preferred embodiments are the same as those described above.  The step of designing and synthesizing the base sequence of siRNA based on the searched base sequence can be carried out in accordance with the method for designing the base sequence of the polynuclecrtide for causing RNA interference and the method for producing the double-stranded polynucleotide described above.  Preferred embodiments are the same as those described above.<br>
The resulting double-stranded polynucleotide is added to an expression system for ei  target gene to inhibit the expression of the target gene.  The expression system for a target gene means a system in which the target gene is expressed, and more specifically, a system provided with a reaction system in which at least mRNA of the target gene is formed.  Examples of the expression system for the target gene include both in  vitro  and in  vivo  systems.  In addition to cultured cells, cultured tissues, and living<br><br>
bodies, cell-free systems can also be used as the expression system for the target genes.  The target gene of which expression is intended to be inhibited (inhibition target gene) is not necessarily a gene of a species corresponding to the origin of the searched sequence. However, as the relationship between the origin of the search target gene and the origin of the inhibition target gene becomes closer, a predetermined gene can be more specifically and effectively inhibited.<br>
Introduction into an expression system means incorporation into the expression reaction system for the target gene.  For example, in one method, a double-stranded nucleotide is transfected to a cultured cell including a target gene and incorporated into the cell.  In another method, an expression vector having a base sequence comprising a prescribed sequence and an overhanging portion is formed, and the expression vector is introduced into a cell having a target gene.<br>
In accordance with the gene inhibition method of the present invention, since polynucleotides which cause RNA interference can be efficiently produced, it is possible to inhibit genes efficiently and simply.  siRNA sequence design program<br>
Embodiments of the siRNA sequence design program will be described below.	(5-1) Outline of the program<br>
When species whose genomes are not sequenced, for example, horse and swine, are subjected to RNA interference,<br><br>
this program calculates a sequence of siRNA usable in the target species based on published sequence information regarding human beings and mice.  If siRNA is designed using this program, RNA interference can be carried out rapidly without sequencing the target gene.  In the design (calculation) of siRNA, sequences having RNAi activity with high probability are selected in consideration of the rules of allocation of G or C (the rules (a) to (d) described above), and checking is performed by homology search so that RNA interference does not occur in genes that are not related to the target gene.  In this specification, "G or C" may also be written as "G/C", and "A or T" may also be written as "A/T".  Furthermore, "T(U)H in ,,A/T(U)" means T (thymine) in the case of sequences of deoxyribonucleic acid and U (uracil) in the case of sequences of ribonucleic acid.<br>
(5-2) Policy of siRNA design<br>
Sequences of human gene X and mouse gene X which are homologous to the human gene are assumed to be known.  This program reads the sequences and searches completely common sequences each having 23 or more bases from the coding regions (CDS).  By designing siRNA from the common portions, the resulting siRNA can target both human and mouse gene X (Fig. 1).<br>
Since the portions completely common to human beings and mice are believed to also exist in other mammals with high probability, the siRNA is expected to act not only on gene X of human beings and mice but also on gene X of other mammals.  Namely, even if in an animal species in which the<br><br>
sequence of a target gene is not known, if sequence information is known regarding the corresponding homologues of human beings and mice, it is possible to design siRNA using this program.<br>
Furthermore, in mammals, it is known that sequences of effective siRNA have regularity (Fig. 2).  In this program, only sequences conforming to the rules are selected.  Fig. 2 is a diagram which shows regularity of siRNA sequences exhibiting an RNAi effect (rules of G/C allocation of siRNA).  In Fig. 2, with respect to siRNA in which two RNA strands, each having a length of 21 bases and having an overhang of 2 bases on- the 3' side, form base pairs between 19 bases at the 5' side of the two strands, the sequence in the coding side among the 19 bases forming the base pairs must satisfy the following conditions: 1) The 3! end is A/U; 2) the 5' end is G/C, and 3) 7 characters on the 3' side has a high ratio of A/U.  In particular, the conditions 1) and 2) are important.<br>
(5-3) Structure of program<br>
This program consists of three parts, i.e., (5-3-1) a part which searches sequences of sites common to human beings and mice (partial sequences), (5-3-2) a part which scores the sequences according to the rules of G/C allocation, and (5-3-3) a part which performs checking by homology search so that unrelated genes are not targeted.<br>
(5-3-1) Part which searches common sequences<br>
This part reads a plurality of base sequence files (file 1, file 2, file 3, ...) and finds all sequences of 23<br><br>
characters that commonly appear in all the files.<br>
(Calculation Example)<br>
As file 1, sequences of human gene FBP1 (KM_000507: Homo sapiens fructose-1,6-bisphosphatase 1) and, as file 2, sequences of mouse gene Fbpl (NM_019395: Mus musculus fructose*bisphosphatase 1) were inputted into the program. As a result, from the sequences of the two (Fig. 3), 15 sequences, each having 23 characters, that were common to the two (sequences common to human FBP1 and mouse Fbpl) were found (Fig. 4).<br>
(5-3-2) Part which scores sequences<br>
This part scores the sequences each having 23 characters in order to only select the sequences conforming to the rules of G/C allocation.<br>
(Method)<br>
The sequences each having 2 3 characters are scored in the following manner.<br>
Score 1: Is the 21st character from the head A/U?<br>
[no = 0, yes = 1]<br>
Score 2: Is the third character from the head G/C?<br>
[no = 0, yes = 1]<br>
Score 3: The number of A/U among 7 characters between the 15th character and 21st character from the head<br>
[0 to 7]<br>
Total score: Product of scores 1 to 3.  However, if the product is 3 or less, the total score is considered as zero.<br>
(Calculation Example)<br><br>
With respect to 15 sequences in Fig. 4, the results of calculation are shown in Fig. 5.  Fig. 5 is a diagram in which the sequences common to human FBP1 and mouse Fbpl are scored.  Furthermore, score 1, score 2, score 3, and total score are described in this order after the sequences shown in Fig. 5.<br>
(5-3-3) Part which performs checking so that unrelated genes are not targeted<br>
In order to prevent the designed siRNA from acting on genes unrelated to the target gene, homology search is performed against all the published mRNA of human beings and mice, and the degree of unrelated genes being hit is evaluated.  Various search algorithms can be used in the homology search.  Herein, an example in which BLAST is used will be described.  Additionally, when BLAST is used, in view that the sequences to be searched are as short as 23 bases, it is desirable that Word Size be decreased sufficiently.<br>
After the Blast search, among the hits with an E-value of 10.0 or less, with respect to all the hits other than the target gene, the total sum of the reciprocals of the E-values are calculated (hereinafter, the value is referred to as a homology score).  Namely, the homology score (X) is found in accordance with the following expression.<br>
all hits Note: A lower E value of the hit indicates higher<br><br>
homology to 23 characters of the query and higher risk of being targeted by siRNA.  A larger number of hits indicates a higher probability that more unrelated genes are targeted. In consideration of these two respects, the risk that siRNA targets genes unrelated to the target gene is evaluated using the above expression.<br>
(Calculation Example)<br>
The results of homology search against the sequences each having 23 characters and the homology scores are shown (Figs. 6 and 7).  Fig. 6 shows the results of BLAST searches of a sequence common to human FBP1 and mouse Fbpl, i.e., "caccctgacccgcttcgtcatggn, and the first two lines are the results in which both mouse Fbpl and human FBP1 are hit.  The homology score is 5.9, and this is an example of a small number of hits.  The risk that siRNA of this sequence targets the other genes is low.  Furthermore, Fig, 7 shows the results of BLAST searches of a sequence common to human FBP1 and mouse Fbpl, i.e., "gccttctgagaaggatgctctgc".  This is an example of a large number of hits, and the homology score is 170.8.  Since the risk of targeting other genes is high, the sequence is not suitable as siRNA.<br>
In practice, the parts (5-3-1), (5-3-2), and (5-3-3) may be integrated, and when the sequences of human beings and mice shown in Fig. 3 are inputted, an output as shown • in Fig. 8 is directly obtained.  Herein, after the sequences shown in Fig. 8, score 1, score 2, score 3, total score, and the tenfold value of homology score are<br><br>
described in this order.  Additionally, in order to save processing time, the program may be designed so that the homology score is not calculated when the total score is zero.  As a result, it is evident that the segment "36 caccctgacccgcttcgtcatgg" can be used as siRNA.  Furthermore, one of the parts (5-3-1), (5-3-2), and (5-3-3) may be used independently.<br>
(5-4) Actual calculation<br>
With respect to about 6,400 gene pairs among the homologues between human beings and mice, siRNA was actually designed using this program.  As a result, regarding about 70% thereof, it was possible to design siRNA which had a sequence common to human beings and mice and which satisfied the rules of effective siRNA sequence regularity so that unrelated genes were not targeted.<br>
These siRNA sequences are expected to effectively inhibit target genes not only in human beings and mice but also in a wide range of mammcils, and are believed to have a high industrial value, such as applications to livestock and pet animals.  Moreover, it is possible to design siRNA which simultaneously targets two or more genes of the same species, e.g., eIF2Cl and eIF2C2t using this program.  Thus, the method for designing siRNA provided by this program has a wide range of application and is extremely strong.  In further application,vby designing a PCR primer using a sequence segment common to human beings and mice, target genes can be amplified in a wide range of mammals.<br>
Additionally, embodiments of the apparatus which runs<br><br>
the siRNA sequence design program will be described in detail below in the column  Base sequence processing apparatus for running siRNA sequence design program.  siRNA sequence design business model system<br>
In the siRNA sequence design business model system of the present invention, when the siRNA sequence design program is applied, the system refers to a genome database, an EST database, and a phylogenetic tree database, alone or in combination, according to the logic of this program, and effective siRNA in response to availability of gene sequence information is proposed to the client.  The term "availability" means a state in which information is available.<br>
(1)	In a case in which it is difficult to specify an ORF although genome information is available, siRNA candidates effective against assumed exon sites are extracted based on EST information, etc., and siRNA sequences in consideration of splicing variants and evaluation results thereof are displayed.<br>
(2)	In a case in which a gene sequence and a gene name are known, after the input of the gene sequence or the gene name, effective siRNA candidates are extracted, and siRNA sequences and evaluation results thereof are displayed.<br>
(3)	In a case in which genome information is not available, using the gene sequences of a related species storing the same type of gene functions (congeneric or having the same origin) or gene sequences of two or more species which have a short distance in phylogenetic trees and of which genome<br><br>
sequences are available, effective siRNA candidates are extracted, and siRNA sequences and evaluation results thereof are displayed.<br>
(4) In order to analyze functions of genes relating infectious diseases and search for targets for new drug development, a technique is effective in which the genome database and phylogenetic tree database of microorganisms are further combined with apoptosis induction site information and function expression site information of microorganisms to obtain exhaustive siRNA candidate sequences.<br> Base sequence processing apparatus for running siRNA sequence design program, etc.<br>
Embodiments of the base sequence processing apparatus which is an apparatus for running the siRNA sequence design program described above, the program for running a base sequence processing method on a computer, the recording medium, and the base sequence processing system in accordance with the present invention will be described in detail below with reference to the drawings.  However, it is to be understood that the present invention is not restricted by the embodiments. [Summary of the present invention]<br>
The summary of the present invention will be described below, and then the constitution, processing, etc., of the present invention will be described in detail. Fig. 12 is a principle diagram showing the basic principle of the present invention.<br><br>
Overall, the present invention has the following basic features.  That is, in the present invention, base sequence information of a target gene for RNA interference is obtained, and partial base sequence information corresponding to a sequence segment having a predetermined number of bases in the base sequence information is created (step S-l).<br>
In step S-l, partial base sequence information having a predetermined number of bases may be created from a segment corresponding to a coding region or transcription region of the target gene in the base sequence information. Furthermore, partial base sequence information having a predetermined number of bases which is common in a plurality of base sequence information derived from different organisms (e.g., human base sequence information and mouse base sequence information) may be created. Furthermore, partial base sequence information having a predetermined number of bases which is common in a plurality of analogous base sequence information in the same species may be created.  Furthermore, common partial base sequence information having a predetermined number of bases may be created from segments corresponding to coding regions or transcription regions of the target gene in a plurality of base sequence information derived from different species.  Furthermore, common partial base sequence information having a predetermined number of bases may be created from segments corresponding to coding regions or transcription regions of the target gene in a<br><br>
plurality of analogous base sequence information in the same species.  Consequently, a prescribed sequence which specifically causes RNA interference in the target gene can be efficiently selected, and calculation load can be reduced.<br>
Furthermore, in step S-l, partial base sequence information including an overhanging portion may be created. Specifically, for example, partial base sequence information to which overhanging portion inclusion information, which shows that an overhanging portion is included, is added may be created.  Namely, partial base sequence information and overhanging portion inclusion information may be correlated with each other.  Thereby, it becomes possible to select the prescribed sequence with the overhanging portion being included from the start to perform designing.<br>
The upper limit of the predetermined number of bases is, in the case of not including the overhanging portion, preferably 28 or less, more preferably 22 or less, and still more preferably 20 or less, and in the case of including the overhanging portion, preferably 32 or less, more preferably 26 or less, and still more preferably 24 or less.  The lower limit of the predetermined number of bases is, in the case of not including the overhanging portion, preferably at least 13, more preferably at le.ast 16, and still more preferably at least 18, and in the case of including the overhanging portion, preferably at least 17, more preferably at least 20, and still more preferably at<br><br>
least 22.  Most preferably, the predetermined number of bases is, in the case of not including the overhanging portion, 19, and in the case of including the overhanging portion, 23.  Thereby, it is possible to efficiently select the prescribed sequence which causes RNA interference without causing cytotoxicity even in mammals.<br>
Subsequently, it is determined whether the 31 end base in the partial base sequence information created in step S-l is adenine, thymine, or uracil (step S-2). Specifically, for example, when the 3' end base is adenine, thymine, or uracil, "1" may be outputted as the determination result, and when it is not, "0" may be outputted.<br>
Subsequently, it is determined whether the 51 end base in the partial base sequence information created in step S-l is guanine or cytosine (step S-3).  Specifically, for example, when the 51 end base is guanine or cytosine, "1" may be outputted as the determination result, and when it is not, "0" may be outputted.<br>
Subsequently, it is determined whether base sequence information comprising 7 bases at the 31 end in the partial base sequence information created in step S-l is rich in one or more types of bases selected from the group consisting of adenine, thymine, and uracil (step S-4). Specifically, for example, the number of bases of one or more types of bases selected from the group consisting of adenine, thymine, and uracil contained in the base sequence information comprising 7 bases at the 3' end may be<br><br>
outputted as the determination result.  The rule of determination in step S-4 regulates that base sequence information in the vicinity of the 3! end of the partial base sequence information created in step S-l contains a rich amount of one or more types of bases selected from the group consisting of adenine, thymine, and uracil, and more specifically, as an index for search, regulates that the base sequence information in the range from the 3' end base to the seventh base from the 3! end is rich in one or more types of bases selected from the group consisting of adenine, thymine, and uracil.<br>
In step S-4, the phrase "base sequence information rich in" corresponds to the phrase "sequence rich in" described in the column  Method for searching target base sequence for RNA interference.  Specifically, for example, when the partial base sequence information created in step S-l comprises about 19 bases, in the base sequence information comprising 7 bases in the partial base sequence information, preferably at least 3 bases, more preferably at least 4 bases, and particularly preferably at least 5 bases, are one or more types of bases selected from the group consisting of adenine, thymine, and uracil.<br>
Furthermore, in steps S-2 to S-4, when partial base sequence information including the overhanging portion is determined, the sequence segment excluding the overhanging portion in the partial base sequence information is considered as the determination target.<br>
Subsequently, based on the determination results in<br><br>
steps S-2, S-3, and S-4, prescribed sequence information which specifically causes RNA interference in the target gene is selected from the partial base sequence information created in step S-l (Step S-5).<br>
Specifically, for example, partial base sequence information'in which the 3! end base has been determined as adenine, thymine, or uracil in step S-2, the 51 end base has been determined as guanine or cytosine in step S-3, and base sequence information comprising 7 bases at the 3' end in the partial base sequence information has been determined as being rich in one or more types of bases selected from the group consisting of adenine, thymine, and uracil is selected as prescribed sequence information. Specifically, for example, a product of the values outputted in steps S-2, S-3, and S-4 may be calculated, and based on the product, prescribed sequence information may be selected from the partial base sequence information created in step S-l,<br>
Consequently, it is possible to efficiently and easily produce a siRNA sequence which has an extremely high probability of causing RNA interference, i.e., which is effective for RNA interference, in mammals, etc.<br>
Here, an overhanging portion may be added to at least one end of the prescribed sequence information selected in step S-5.  Additionally, for example, when a target is searched, the overhanging portion may be added to both ends of the prescribed sequence information.  Consequently, designing of a polynucleotide which causes RNA interference<br><br>
can be simplified.<br>
Additionally, the number of bases in the overhanging portion corresponds to the number of bases described in the column  Method for designing base sequence of polynucleotide for causing RNA interference.  Specifically, for example, 2 is particularly suitable as the number of bases.<br>
Furthermore, base sequence information that is identical or similar to the prescribed sequence information selected in step S-5 may be searched from other base sequence information (e.g., base sequence information published in a public database, such as RefSeq (Reference Sequence project) of NCBI) using a known homology search method, such as BLAST, FASTA, or ssearch, and based on the searched identical or similar base sequence information, evaluation may be made whether the prescribed sequence information targets genes unarelated to the target gene.<br>
Specifically, for example, base sequence information that is identical or similar to the prescribed sequence information selected in step S-5 is searched from other base sequence information (e.g., base sequence information published in a public database, such as RefSeq of NCBI) using a known homology search method, such as BLAST, FASTA, or ssearch.  Based on the total amount of base sequence information on the genes unrelated to the target gene in the searched identical or similar base sequence information and the values showing the degree of identity or similarity (e.g., "E value" in BLAST, FASTA, or ssearch) attached to<br><br>
the base sequence information on the genes unrelated to the target gene, the total sum of the reciprocals of the values showing the degree of identity or similarity is calculated, and based on the calculated total sum (e.g., based on the size of the total sum calculated), evaluation may be made whether the prescribed sequence information targets genes unrelated to the target gene.<br>
Consequently, it is possible to select a sequence which specifically causes RNA interference only to the target gene.<br>
If RNA is synthesized based on the prescribed sequence information which is selected in accordance with the present invention and which does not cause RNA interference in genes unrelated to the target gene, it is possible to greatly reduce effort, time, and cost required compared with conventional techniques. [System configuration]<br>
First, the configuration of this system will be described.  Fig. 13 is a block diagram which shows an example of the system to which the present invention is applied and which conceptually shows only the parts related to the present invention.<br>
Schematically, in this system, a base sequence processing apparatus 100 which processes base sequence information of a target gene for RNA interference and an external system 200 which provides external databases regarding sequence information, structural information, etc., and external programs, such as homology search, are<br><br>
connected to each other via a network 300 in a communicable manner.<br>
In Fig. 13, the network 300 has a function of interconnecting between the base sequence processing apparatus 100 and the external system 200, and is, for example, the Internet.<br>
In Fig. 13, the external system 200 is connected to the base sequence processing apparatus 100 via the network 300, and has a function of providing the user with the external databases regarding sequence information, structural information, etc., and Web sites which execute external programs, such as homology search and motif search.<br>
The external system 200 may be constructed as a WEB server, ASP server, or the like, and the hardware structure thereof may include a commercially available information processing apparatus, such as a workstation or a personal computer, and its accessories.  Individual functions of the external system 200 are implemented by a CPU, a disk drive, a memory unit, an input unit, an output unit, a communication control unit, etc., and programs for controlling them in the hardware structure of the external system 200.<br>
In Fig. 13, the base sequence processing apparatus 100 schematically includes a controller 102, such as a CPU, which controls the base sequence processing apparatus 100 overall; a communication control interface 104 which is connected to a communication device (not shown in the drawing), such as a router, connected to a communication<br><br>
line or the like; an input-output control interface 108 connected to an input unit 112 and an output unit 114; and a memory 106 which stores various databases and tables. These parts are connected via given communication channels in a communicable manner.  Furthermore, the base sequence processing apparatus 100 is connected to the network 300 in a communicable manner via a communication device, such as a router, and a wired or radio communication line.<br>
Various databases and tables (a target gene base sequence file 106a - a target gene annotation database 106h) which are stored in the memory 106 are storage means, such as fixed disk drives, for storing various programs used for various processes, tables, files, databases, files for web pages, etc.<br>
Among these components of the memory 106, the target gene base sequence file 106a is target gene base sequence storage means for storing base sequence information of the target gene for RNA interference.  Fig. 14 is a diagram which shows an example of information stored in the target gene base sequence file 106a.<br>
As shown in Fig. 14, the information stored in the target gene base sequence file 106a consists of base sequence identification information which uniquely identifies base sequence information of the target gene for RNA interference (e.g., "NM_000507" in Fig. 14) and base sequence information (e.g., "ATGGCTGA ... AGTGA" in Fig. 14), the base sequence identification information and the base sequence information being associated with each other.<br><br>
f /<br>
Furthermore, a partial base sequence file 106b is partial base sequence storage means for storing partial base sequence information, i.e., a sequence segment having a predetermined number of bases in base sequence 5  information of the target gene for RNA interference.<br>
Fig. 15 is a diagram which shows an example of information stored in the partial base sequence file 106b.<br>
As shown in Fig. 15, the information stored in the partial base sequence file 106b consists of partial base 10  sequence identification information which uniquely identifies partial base sequence information (e.g., "NM_000507:36" in Fig. 15), partial base sequence information (e.g., "caccct ... tcatgg" in Fig. 15), and information on inclusion of an overhanging portion which shows the inclusion of the overhanging portion (e.g., "included" in Fig. 15), the partial base sequence identification information, the partial base sequence information, and the information on inclusion of the overhanging portion being associated with each other.<br>
A determination result file 106c is determination result storage means for storing the results determined by a 31 end base determination part 102b, a 51 end base determination part 102c, and a predetermined base inclusion determination part 102d, which will be described below. Fig. 16 is a diagram which shows an example of information stored in the determination result file 106c.<br>
As shown in Fig. 16, the information stored in the determination result file 106c consists of partial base<br><br>
sequence identification information (e.g., "NM_000507:36" in Fig. 16), determination result on 3'   end base corresponding to a result determined by the 3' end base determination part 102b (e.g., "1" in Fig. 16), determination result on 5' end base corresponding to a result determined by the 5! end base determination part 102c (e.g., "1" in Fig. 16), determination result on inclusion of predetermined base corresponding to a result determined by the predetermined base inclusion determination part 102d (e.g., "4" in Fig. 16), and comprehensive determination result corresponding to a result obtained by putting together the results determined by the 31 end base determination part 102b, the 51 end base determination part 102c, and the predetermined base inclusion determination part 102d (e.g., "4" in Fig. 16), the partial base sequence identification information, the determination result on 3' end base, the determination result on 5' end base, the determination result on inclusion of predetermined base, and the comprehensive determination result being associated with each other.<br>
Additionally, Fig. 16 shows an example of the case in which, with respect to the determination result on 31 end base and the determination result on 5' end base, "1" is set when determined as being "included" by each of the 31 end base determination part 102b and the 5' end base determination part 102c and "0" is set when determined as being "not included".  Furthermore, Fig. 16 shows an example of the case in which the determination result on<br><br>
inclusion of predetermined base is set as the number of bases corresponding to one or more types of bases selected from the group consisting of adenine, thymine, and uracil contained in the base sequence information comprising 7 bases at the 3' end in the partial base sequence information.  Furthermore, Fig. 16 shows an example of the case in which the comprehensive determination result is set as the product of the determination result on 31 end base, the determination result on 5' end base, and the determination result on inclusion of predetermined base. Specifically, for example, when the product is 3 or less, "0" may be set.<br>
Furthermore, a prescribed sequence file 106d is prescribed sequence storage means for storing prescribed sequence information corresponding to partial base sequence information which specifically causes RNA interference in the target gene.  Fig. 17 is a diagram which shows an example of information stored in the prescribed sequence file 106d.<br>
As shown in Fig. 17, the information stored in the prescribed sequence file 106d consists of partial base sequence identification information (e.g., "NM__000507:36" in Fig. 17) and prescribed sequence information corresponding to partial base sequence information which specifically causes RNA interference in the target gene (e.g., caccct ... tcatgg" in Fig. 17), the partial base sequence identification information and the prescribed sequence information being associated with each other.<br><br>
Furthermore, a reference sequence database 106e is a database which stores reference base sequence information corresponding to base sequence information to which reference is made to search base sequence information identical or similar to the prescribed sequence information by an identical/similar base sequence search part 102g, which will be described below.  The reference sequence database 106e may be an external base sequence information database accessed via the Internet or may be an in-house database created by copying such a database, storing the original sequence information, or further adding unique annotation information to such a database.  Fig. 18 is a diagram which shows an example of information stored in the reference sequence database 106e.<br>
As shown in Fig. 18, the information stored in the reference sequence database 106e consists of reference sequence identification information (e.g., "ref|NM_J)15820. l|" in Fig. 18) and reference base sequence information (e.g., "caccct ... gcatgg" in Fig. 18), the reference sequence identification information and the reference base sequence information being associated with each other.<br>
Furthermore, a degree of identity or similarity file 106f is degree of identity or similarity storage means for storing the degree of identity or similarity *corresponding to a degree of identity or similarity of identical or similar base sequence information searched by an identical/similar base sequence search part 102g, which<br><br>
will be described below. F:Lg. 19 is a diagram which shows an example of information stored in the degree of identity or similarity file 106f.<br>
As shown in Fig. 19, the information stored in the degree of identity or similarity file 106f consists of partial base sequence identification information (e.g., "NM_J)00507:36" in Fig. 19), reference sequence identification information (e.g., "ref|NM__015820 . l|" and "ref|NM_003837.l|" in Fig. 19), and degree of identity or similarity (e.g., "0.52" in Fig. 19), the partial base sequence identification information, the reference sequence identification information, and the degree of identity or similarity being associated with each other.<br>
Furthermore, an evaluation result file 106g is evaluation result storage means for storing the result of cvaiuanuu un wnetner genes unrelated to the target gene are targeted by an unrelated gene target evaluation part 102h, which will be described below.  Fig. 20 is a diagram which shows an example of information stored in the evaluation result file 106g.<br>
As shown in Fig. 20, the information stored in the evaluation result file 106g consists of partial base sequence identification information (e.g., "NM_000507:36" and "NM_000507:441" in Fig. 20), total sum calculated by a total^ sum calculation part 102m, which will be described below, (e.g., "5.9" and "170•8" in Fig. 20), and evaluation result (e.g., "nontarget" and "target" in Fig. 20), the partial base sequence identification information, the total<br><br>
sum, and the evaluation result being associated with each other.  Additionally, in Fig. 20, "ncntarget" means that the prescribed sequence information does not target genes unrelated to the target gene, and "target" means that the prescribed sequence information targets genes unrelated to the target gene.<br>
A target gene annotation database 106h is target gene annotation storage means for storing annotation information regarding the target gene.  The target gene annotation database 106h may be an external annotation database which stores annotation information regarding genes and which is accessed via the Internet or may be an in-house database created by copying such a database, storing the original sequence information, or further adding unique annotation information to such a database.<br>
The information stored in the target gene annotation database 106h consists of target gene identification information which identifies the target gene (e.g., the name of a gene to be targeted, and Accession number (e.g., "NM__000507" and "FBP1" described on the top in Fig, 3)) and simplified information on the target gene (e.g., "Homo sapiens fructose-1,6-bisphosphatase 1" describe on the top in Fig. 3), the target gene identification information and the simplified information being associated with each other.<br>
In Fig. 13, the communication control interface 104 controls communication between the base sequence processing apparatus 100 and the network 300 (or a communication<br><br>
control interface 104 performs data communication with other terminals via communication lines.<br>
In Fig. 13, the input-output control interface 108 controls the input unit 112 and the output unit 114.  Here, as the output unit 114, in addition to a monitor (including a home television), a speaker may be used (hereinafter, the output unit 114 may also be described as a monitor).  As the input unit 112, a keyboard, a mouse, a microphone, or the like may be used.  The monitor cooperates with a mouse to implement a pointing device function.<br>
In Fig. 13, the controller 102 includes control programs, such as OS (Operating System), programs regulating various processing procedures, etc., and internal memories for storing required data, and performs information processing for implementing various processes using the programs, etc.  The controller 102 functionally includes a partial base sequence creation part 102a, a 3' end base determination part 102b, a 5! end base determination part 102c, a predetermined base inclusion determination part 102d, a prescribed sequence selection part 102e, an overhanging portion-adding part 102f, an identical/similar base sequence search part 102g, and an unrelated gene target evaluation part 102h.<br>
Among them, the partial base sequence creation part 102a is partial base sequence creation means for acquiring base sequence information of a target gene for RNA interference and creating partial base sequence information<br><br>
number of bases in the base sequence information.  As shown in Fig. 21, the partial base sequence creation part 102a includes a region-specific base sequence creation part 1021, a common base sequence creation part 102j, and an overhanging portion-containing base sequence creation part 102k.<br>
Fig. 21 is a block diagram which shows an example of the structure of the partial base sequence creation part 102a of the system to which the present invention is applied and which shows only the parts related to the present invention.<br>
In Fig. 21, the region-specific base sequence creation part 102i is region-specific base sequence creation means for creating partial base sequence information having a predetermined number of bases from a segment corresponding to a coding region or transcription region of the target gene in the base sequence information.<br>
The common base sequence creation part 102j is common base sequence creation means for creating partial base sequence information having a predetermined number of bases which is common in a plurality of base sequence information derived from different organisms.<br>
The overhanging portion-containing base sequence creation part 102k is overhanging portion-containing base 25  sequence creation means for creating partial base sequence information containing an overhanging portion.<br>
Referring back to Fig. 13, the 3' end base determina4""\.i?^. -^^-- * ^~*- •.-■--.--..<br><br>
for determining whether the 31 end base in the partial base sequence information is adenine, thymine, or uracil.<br>
Furthermore, the 5' end base determination parr 102c is 5! end base determination means for determining whether the 51 end base in the partial base sequence information is guanine or cytosine.<br>
Furthermore, the predetermined base inclusion determination part 102d is predetermined base inclusion determination means for determining whether the base sequence information comprising 7 bases at the 3! end in the partial base sequence information is rich in one or more types of bases selected from the group consisting of adenine, thymine, and uracil..<br>
Furthermore, the prescribed sequence selection part 102e is prescribed sequence selection means for selectina prescribed sequence information, which specifically causes RNA interference in the target gene, from the partial base sequence information based on the results determined by the 31 end base determination part 102b, the 5f end base determination part 102c, and the predetermined base inclusion determination part 102cL-<br>
Furthermore, the overhanging portion-adding part 102f is overhanging portion addition means for adding an overhanging portion to at least one end of the prescribed sequence information.<br>
Furthermore, the identical/similar base sequence search part 102g is identical/similar base sequence search<br><br>
similar to the prescribed sequence information, from other base sequence information.<br>
Furthermore, the unrelated gene target evaluation part 102h is unrelated gene target evaluation means for evaluating whether the prescribed sequence information targets genes unrelated to the target gene based on the identical or similar base sequence information.  As shown in Fig. 22, the unrelated gene target evaluation part 102h further includes a total sum calculation part 102m and a total sum-based evaluation part 102n.<br>
Fig. 22 is a block diagram which shows an example of the structure of the unrelated gene target evaluation part 102h of the system to which the present invention is applied and which schematically shows only the parts related to the present invention.<br>
In Fig- 22, the total sum calculation part 102m is total sum calculation means for calculating the total sum of reciprocals of the values showing the degree of identity or similarity based on the total amount of base sequence information on the genes unrelated to the target gene in identical or similar base sequence information and the values showing the degree of identity or similarity attached to the base sequence information on the genes unrelated to the target gene (identity or similarity).<br>
Furthermore, the total sum-based evaluation part 102n is total sum-based target evaluation means for evaluating whether the prescribed sequence information targets genes<br><br>
calculated by the total sum calculation part 102m.<br>
The details of processing of each part will be described later. [Processing of the system]<br>
An example of processing of the system having the configuration described above in this embodiment will be described in detail with reference to Figs. 23 and 24. [Main processing]<br>
First, the details of the main processing will be described with reference to Fig. 23, etc.  Fig. 23 is a flowchart which shows an example of the main processing of the system in this embodiment.<br>
The base sequence processing apparatus 100 acquires base sequence information of a target gene for RNA interference by the partial base sequence creation process performed by the partial base sequence creation part 102a, stores it in a predetermined memory region of the target gene base sequence file 106a# creates partial base sequence information corresponding to a sequence segment having a predetermined number of bases in the base sequence information, and stores the created partial base sequence information in a predetermined memory region of the partial base sequence file 106b (step SA-1).<br>
In step SA-1, the partial base sequence creation part 102a may create partial base sequence information having aT predetermined number of bases from a segment corresponding to a coding region or transcription region of the target<br><br>
the region-specific base sequence creation part 102i and may store the created partial base sequence information in a predetermined memory region of the partial base sequence file 106b.<br>
In step SA-1, the partial base sequence creation part 102a may create partial base sequence information having a predetermined number of bases which is common in a plurality of base sequence information derived from different organisms (e.g., human base sequence information and mouse base sequence information) by the processing of the common base sequence creation part 102j and may store the created partial base sequence information in a predetermined memory region of the partial base sequence file 106b.  Furthermore, common partial base sequence information having a predetermined number of bases which is common in a plurality of analogous base sequence information in the same species may be created.<br>
In step SA-1, the partial base sequence creation part 102a may create partial base sequence information having a predetermined number of bases from segments corresponding to coding regions or transcription regions of the target gene in a plurality of base sequence information derived from different species by the processing of the region-specific base sequence creation part 102i and the common base sequence creation part 102j and may store the created partial base sequence information in a predetermined memory region of the partial base sequence file 106b,  Furthermore,<br><br>
predetermined number of bases may be created from segments corresponding to coding regions or transcription regions of the target gene in a plurality of analogous base sequence information in the same species.<br>
Furthermore, in step SA-1, the partial base sequence creation part 102a may create partial base sequence information containing an overhanging portion by the processing of the overhanging portion-containing base sequence creation part 102k.  Specifically, for example, &gt;  the partial base sequence creation part 102a may create partial base sequence information to which the overhanging portion inclusion information which shows the inclusion of the overhanging portion by the processing of the overhanging portion-containing base sequence creation part 102k and may store the created partial base seauence information and the overhanging portion inclusion information so as to be associated with each other in a predetermined memory region of the partial base sequence file 106b.<br>
The upper limit of the predetermined number of bases is, in the case of not including the overhanging portion, preferably 28 or less, more preferably 22 or less, and still more preferably 20 or less, and in the case of including the overhanging portion, preferably 32 or less, more preferably 26 or less, and still more preferably 24 or less.  The lower limit of the predetermined number of bases is, in the case of not including the overhanging portion,<br><br>
still more preferably at least 18, and in the case of including the overhanging portion, preferably at least 17, more preferably at least 20, and still more preferably at least 22.  Most preferably, the predetermined number of bases is, in the case of not including the overhanging portion, 19, and in the case of including the overhanging portion, 23.<br>
Subsequently, the base sequence processing apparatus 100 determines whether the 3' end base in the partial base sequence information created in step SA-1 is adenine, thymine, or uracil by the processing of the 31 end base determination part 102b and stores the determination result in a predetermined memory region of the determination result file 106c (step SA-2).  Specifically, for example, the base sequence processina apparatus 100 may store "1" when the 3r end base in the partial base sequence information created in step SA-1 is adenine, thymine, or uracil, by the processing of the 3' end base determination part 102b, and "0" when it is not, in a predetermined memory region of the determination result file 106c.<br>
Subsequently, the base sequence processing apparatus 100 determines whether the 5' end base in the partial base sequence information created in step SA-1 is guanine or cytosine by the processing of the 5! end base determination part 102c and stores the determination result in a predetermined memory region of the determination result file 106c (step SA-3).  Specifically, for example, the base<br><br>
end base in the partial base sequence information created in step SA-1 is guanine or cytosine, by the processing of the 5' end base determination part 102c, and "0" when it is not, in a predetermined memory region of the determination result file 106c.<br>
Subsequently, the base sequence processing apparatus 100 determines whether the base sequence information comprising 7 bases at the 31 end in the partial base sequence information created in step SA-1 is rich in one or more types of bases selected from the group consisting of adenine, thymine, and uracil by the processing of the predetermined base inclusion determination part 102d and stores the determination result in a predetermined memory region of the determination result file 106c (step SA-4). Specifically, for example, the base sequence processing apparatus 100, by the processing of the predetermined base inclusion determination part 102d, may store the number of bases corresponding to one or more types of bases selected from the group consisting of adenine, thymine, and uracil contained in the base sequence information comprising 7 bases at the 3' end in the partial base sequence information created in step SA-1 in a predetermined memory region of the determination result file 106c.  The rule of determination in step SA-4 regulates that base sequence information in the vicinity of the 3' end of the partial base sequence information created in step SA-1 contains a rich amount of one or more types of bases selected from the<br><br>
specifically, as an index for search, regulates that the base sequence information in the range from the 31 end base to the seventh base from the 3' end is rich in one or more types of bases selected from the group consisting of adenine, thymine, and uracil.<br>
In step SA-4, the phrase "base sequence information rich in" corresponds to the phrase "sequence rich in1' described in the column  Method for searching target base sequence for RNA interference.  Specifically, for example, when the partial base sequence information created in step SA-1 comprises about 19 bases, in the base sequence information comprising 7 bases at the" 3 ' end in the partial base sequence information, preferably at least 3 bases, more preferably at least 4 bases, and particularly preferably at least 5 bases, are one  or more types of bases seiectea rrom the group consisting of adenine, thymine, and uracil.<br>
Furthermore, in steps SA-2 to SA-4, when partial base sequence information including the overhanging portion is determined, the sequence segment excluding the overhanging portion in the partial base sequence information is considered as the determination target.<br>
Subsequently, based on the determination results in steps SA-2, SA-3, and SA-4, the base sequence processing apparatus 100, by the processing of the prescribed sequence selection part 102e, selects prescribed sequence information which specifically causes RNA interference in<br><br>
created in step SA-1 and stores it in a predetermined memory region of the prescribed sequence file 106d (Step SA-5).<br>
Specifically, for example, the base sequence processing apparatus 100, by the processing of the prescribed sequence selection part 102e, selects partial base sequence information, in which the 3' end base has been determined as adenine, thymine, or uracil in step SA-2, the 51 end base has been determined as guanine or cytosine in step SA-3, and base sequence information comprising 7 bases at the 31 end in the partial base sequence information has been determined as being rich in one or more types of bases selected from the group consisting of adenine, thymine, and uracil, as prescribed sequence information, and stores it in a predetermined memory region of the prescribed sequence file 106d.  Specifically, for example, the base sequence processing apparatus 100, by the processing of the prescribed sequence selection part 102e, may calculate a product of the values outputted in steps SA-2, SA-3, and SA-4 and, based on the product, select prescribed sequence information from the partial base sequence information created in step SA-1.<br>
Here, the base sequence processing apparatus 100 may add an overhanging portion to at least one end of the prescribed sequence information selected in step SA-5 by the processing of the overhanging portion-adding part 102f, and may store it in a predetermined memory region of the<br><br>
by the processing of the overhanging portion-adding part 102f, the base sequence processing apparatus 100 may change the prescribed sequence information stored in the prescribed sequence information section in the prescribed sequence file 106d to prescribed sequence information in which an overhanging portion is added to at least one end. Additionally, for example, when a target is searched, the overhanging portion may be added to both ends of the prescribed sequence information.<br>
Additionally, the number of bases in the overhanging portion corresponds to the number of bases described in the column  Method for designing base sequence of polynucleotide for causing RNA interference.  Specifically, for example, 2 is particularly suitable as the number of bases.<br>
Furthermore, the base sequence processing apparatus 100, by the processing of the identical/similar base sequence search part 102g, may search base sequence information that is identical or similar to the prescribed sequence information selected in step SA-5 from other base sequence information (e.g., base sequence information published in a public database, such as RefSeq of NCBI) using a known homology search method, such as BLAST, FASTA, or ssearch, and based on the searched identical or similar base sequence information, by the unrelated gene target evaluation process performed by the unrelated gene target evaluation part 102h, may evaluate whether the prescribed<br><br>
gene.<br>
Specifically, for example, the base sequence processing apparatus 100, by the processing of the identical/similar base sequence search part 102g, may search base sequence information that is identical or similar to the prescribed sequence information selected in step SA-5 from other base sequence information (e.g., base sequence information published in a public database, such as RefSeq of NCBI) using a known homology search method, such as BLAST, FASTA, or ssearch.  The unrelated gene target evaluation part 102h, by the processing of the total sum calculation part 102m, may calculate the total sum of the reciprocals of the values showing the degree of identity or similarity based on the total amount of base sequence information on the genes unrelated to the target gene ±n the searched identical or similar base sequence information and the values showing the degree of identity or similarity (e.g., "E value" in BLAST, FASTA, or ssearch) attached to the base sequence information on the genes unrelated to the target gene.  The unrelated gene target evaluation part 102h, by the processing of the total sum-based evaluation part 102n, may evaluate whether the prescribed sequence information targets genes unrelated to the target gene based on the calculated total sum.<br>
Here, the details of the unrelated gene target evaluation process performed by the unrelated gene target evaluation part 102h will be described with reference to<br><br>
Fig. 2 4 is a flowchart which shows an example of the unrelated gene evaluation process of the system in this embodiment.<br>
First, the base sequence processing apparatus 100, by the processing of the identical/similar base sequence search part 102g, searches base sequence information that is identical or similar to the prescribed sequence information selected in step SA-5 from other base sequence information (e.g., base sequence information published in a public database, such as RefSeq of NCBI) using a known homology search method, such as BLAST, FASTA, or ssearch, and stores identification information of the prescribed sequence information ("partial base sequence identification information" in Fig. 19), identification information of the searched identical or similar base sequence information V reference sequence identification information" in Fig. 19), and the value showing the degree of identity or similarity (e.g., "E value" in BLAST, FASTA, or ssearch) ("degree of identity or similarity" in Fig, 19) attached to the searched identical or similar base sequence information so as to be associated with each other in a predetermined memory region of the degree of identity or similarity file 106f.<br>
Subsequently, the unrelated gene target evaluation part 102h, by the processing of the total sum calculation part 102m, calculates the total sum of reciprocals of the values showing the degree of identity or similarity based<br><br>
genes unrelated to the target gene in the searched identical or similar base sequence information and the values showing the degree of identity or similarity (e.g., "E value" in BLAST, FASTA, or ssearch) attached to the base sequence information on the genes unrelated to the target gene, and stores identification information of the prescribed sequence information ("partial base sequence identification information" in Fig. 20) and the calculated total sum ("total sum" in Fig. 20) so as to be associated with each other in a predetermined memory region of the evaluation result file 106g (step SB-1).<br>
Subsequently, the unrelated gene target evaluation part 102h, by the processing of the total sum-based evaluation part 102n, evaluates whether the prescribed sequence information targets genes unrelated to the target gene based on the total sum calculated in step SB-1 (e.g., based on the size of the total sum calculated in step SB-1), and stores the evaluation results ("nontarget" and "target" in Fig. 20) in a predetermined memory region of the evaluation result file 106g (Step SB-2).<br>
The main process is thereby completed. [Other embodiments]<br>
One preferred embodiment of the present invention has been described above.  However, it is to be understood that the present invention can be carried out in various embodiments other than the embodiment described above within the scope of the technical idea described in the<br><br>
For example, although the case in which the base sequence processing apparatus 100 performs processing on a stand-alone mode has been described, construction may be made such that processing is performed in accordance with the request from a client terminal which is constructed separately from the base sequence processing apparatus 100, and the processing results are sent back to the client terminal.  Specifically, for example, the client terminal transmits a name of the target gene for RNA interference (e.g., gene name or accession number) or base sequence information regarding the target gene to the base sequence processing apparatus 100, and the base sequence processing apparatus 100 performs the processes described above in the controller 102 on base sequence information corresponding to the name or the base sequence information transmitted from the client terminal to select prescribed sequence information which specifically causes RNA interference in the target gene and transmits it to the client terminal. In such a case, for example, by acquiring sequence information from a public database, siRNA against the gene in query may be selected.  Alternatively, for example, siRNA for all the genes may be calculated and stored preliminarily, and siRNA may be immediately selected in response to the request from the client terminal (e.g., gene name or accession number) and the selected siRNA may be sent back to the client terminal.<br>
Furthermore, the base sequence processing apparatus<br><br>
information with respect to genes unrelated to the target gene.  Thereby, it is possible to select prescribed sequence information which specifically causes RNA interference only in the target gene.<br>
Furthermore, in the system comprising a client terminal and the base sequence processing apparatus 100, an interface function may be introduced in which, for example, the results of RNA interference effect of siRNA (e.g., "effective" or "not effective") are fed back from the Web page users on the Web, and the experimental results fed back from the users are accumulated in the base sequence processing apparatus 100 so that the sequence regularity of siRNA effective for RNA interference is improved.<br>
Furthermore, the base sequence processing apparatus 100 may calculate base sequence information of a sense strand of siRNA and base sequence information of an antisense strand complementary to the sense strand from the prescribed sequence information.  Specifically, for example, when "caccctgacccgcttcgtcatgg" is selected as 23-base sequence information wherein 2-base overhanging portions are added to both ends of the prescribed sequence as a result of the processes described above, the base sequence processing apparatus 100 calculates the base sequence information of a sense strand "5'-CCCUGACCCGCUUCGUCAUGG-3'" and the base sequence information of an antisense strand "5'-AUGACGAAGCGGGUCAGGGUG-3'".  Consequently, it is not necessary to manually arrange the sense strand and the<br><br>
improving convenience.<br>
Furthermore, in the processes described in the embodiment, the processes described as being automatically performed may be entirely or partially performed manually, or the processes described as being manually performed may be entirely or partially performed automatically by a known method.<br>
In addition, processing procedures, control procedures, specific names, information including various registration data and parameters, such as search conditions, examples of display screen, and database structures may be changed in any manner except when otherwise described.<br>
Furthermore, with respect to the base sequence processing apparatus 100, the components are shown in the drawings only based on the functional concept, and it is not always necessary to physically construct the components as shown in the drawings.<br>
For example, the process functions of the individual parts or individual units of the base sequence processing apparatus 100, in particular, the process functions performed in the controller 102, may be entirely or partially carried out by a CPU (Central Processing Unit) or programs which are interpreted and executed by the CPU. Alternatively, it may be possible to realize the functions based on hardware according to a wired logic.  Additionally, ^ the program is recorded in a recording medium which will be described below and is mechanically read by the base<br><br>
Namely, the memory 106, such as a ROM or HD, records a computer program which, together with OS (Operating System), gives orders to the CPU to perform various types of processing.  The computer program is executed by being loaded into a RAM or the like, and, together with the CPU, constitutes the controller 102.  Furthermore, the computer program may be recorded in an application program server which is connected to the base sequence processing apparatus 100 via any network 300, and may be entirely or partially downloaded as required.<br>
The program of the present invention may be stored in a computer-readable recording medium.  Here, examples of the "recording medium" include any "portable physical medium", such as a flexible disk, an optomagnetic disk, a ROM, an EPROM, an EEPROM, a CD-ROM, a MO, a DVD, or a flash ~dl~sk; any "fixed physical medium", such as a ROM, a RAM, or a HD which is incorporated into various types of computer system; and a "communication medium" which holds the program for a short period of time, such as a communication line or carrier wave, in the case when the program is transmitted via a network, such as a LAN, a WAN, or Internet.<br>
Furthermore, the "program" means a data processing method described in any language or by any description method, and the program may have any format Ce.g., source code or binary code).  The "program" is not always limited to the one having a single system configuration, and may<br><br>
plurality of modules or libraries, or may achieve its function together with another program, such as OS (Operating System).  With respect to specific configurations and procedures for reading the recording medium in the individual units shown in the embodiment, or installation procedures after reading, etc., known configurations and procedures may be employed.<br>
The various types of databases, etc. (target gene base sequence file 106a - target gene annotation database 106h) stored in the memory 106 are storage means, such as memories (e.g., RAMs and ROMs), fixed disk drives (e.g., hard disks), flexible disks, and optical disks, which store various types of programs used for various processes and Web site provision, tables, files, databases, files for Web pages, etc.<br>
rurtnermore, the base sequence processing apparatus 100 may be produced by connecting peripheral apparatuses, such as a printer, a monitor, and an image scanner, to a known information processing apparatus, for example, an information processing terminal, such as a personal computer or a workstation, and installing software (including programs, data, etc.) which implements the method of the present invention into the information processing apparatus.<br>
Furthermore, specific modes of distribution/integration of the base sequence processing apparatus 100, etc, are not limited to those shown in the<br><br>
processing apparatus 100, etc., may be entirely or partially distributed/integrated functionally or physically in any unit corresponding to various types of loading, etc. (e.g., grid computing).  For example, the individual databases may be independently constructed as independent database units, or processing may be partially performed using CGI (Common Gateway Interface).<br>
Furthermore, the network 300 has a function of interconnecting between the base sequence processing apparatus 100 and the external system 200, and for example, may include any one of the Internet, intranets, LANs (including both wired and radio), VANs, personal computer communication networks, public telephone networks (including both analog and digital), dedicated line networks (including both analog and digital), CATV networks, portable line exchange networks/portable packet exchange networks of the IMT2000 system, CSM system, or PDC/PDC-P system, radio paging networks, local radio networks, such as the Bluetooth, PHS networks, and satellite communication networks, such as CS, BS, and ISDB.  Namely, the present system can transmit and receive various types of data via any network regardless of wired or radio.<br>
EXAMPLES<br>
The present invention will be described in more detail with reference to the examples.  However, it is to be understood that the present invention is not restricted<br><br>
[Example 1]<br> Gene for measuring RNAi effect and expression vector<br>
As a target gene for measuring an RNAi effect by siRNA, a firefly (Photinus pyralis, P. pyralis) luciferase (luc) gene (P. pyralis luc gene: accession number: U47296) was used, and as an expression vector containing this gene, a pGL3-Control Vector (manufactured by Promega Corporation) was used.  The segment of the P. pyralis luc gene is located between an SV40 promoter and a poly A signal within the vector.  As an internal control gene, a luc gene of sea pansy (Renilla reniformis, R. reniformis) was used, and as an expression vector containing this gene, pRL-TK (manufactured by Promega Corporation) was used.  Synthesis of 21-base double-stranded RNA (siRNA)<br>
Synthesis of 21-base sense strand and 21-base auij-aense sirana KNA (located as shown in Fig. 9; a to p) was entrusted to Genset Corporation through Hitachi Instrument Service Co., Ltd.<br>
The double-stranded RNA used for inhibiting expression of the P. pyralis luc gene was prepared by associating sense and antisense strands.  In the association process, the sense strand RNA and the antisense strand RNA were heated for 3 minutes in a reaction liquid of 10 mM Tris-HCl (pH 7.5) and 20 mM NaCl, incubated for one hour at 37°C, and left to stand until the temperature reached room temperature.  Formation of double-stranded polynucleotides was assayed by electrophoresis on 2%<br><br>
almost all the single-stranded polynucleotides were associated to form double-stranded polynucleotides.  Mammalian cell cultivation<br>
As mammalian cultured cells, human HeLa cells and HEK293 cells and Chinese hamster CHO-KI cells (RIKEN Cell bank) were used.  As a medium, Dulbecco's modified Eagle!s medium (manufactured by Gibco BRL) to which a 10% inactivated fetal bovine serum (manufactured by Mitsubishi Kasei) and as antibiotics, 10 units/ml of penicillin (manufactured by Meiji) and 50 fig/ml of streptomycin (manufactured by Meiji) had been added was used. Cultivation was performed at 37°C in the presence of 5% C02.  Transfection of target gene, internal control gene, and siRNA into mammalian cultured cells<br>
The mammalian cells were seeded at a concentration of 0.2 to 0.3 x 106 cells/ml into a 24-well plate, and after one day, using a Ca-phosphate precipitation method (Saibo-Kogaku Handbook (Handbook for cell engineering), edited by Toshio Kuroki et al., Yodosha (1992)), 1.0 \xg  of pGL3-Control DNA, 0.5 or 1.0 [ig  of pRL-TK DNA, and 0.01, 0.1, 1, 10 or 100 nM of siRNA were introduced.  Drosophila cell cultivation<br>
As drosophila cultured cells, S2 cells (Schneider, I., et al., J. Embryol. Exp. Morph., 27, 353-365 (1972)) were used.  As a medium, Schneider's Drosophila medium (manufactured by Gibco BRL) to which a 10% inactivated fetal bovine serum (manufactured by Mitsubishi Kasei) and<br><br>
Meiji) and 50 [ig/ml of streptomycin (manufactured by Meiji) had been added was used. Cultivation was performed at 25°C in the presence of 5% C02.<br> Transfection of target gene, internal control gene, and siRNA into drosophila cultured cells<br>
The S2 cells were seeded at a concentration of 1.0 x 106 cells/ml into a 24-well plate, and after one day, using a Ca-phosphate precipitation method (Saibo-Kogaku Handbook (Handbook for cell engineering), edited by Toshio Kuroki et al., Yodosha (1992)), 1.0 \ig  of pGL3-Control DNA, 0.1 Ltg of pRL-TK DNA, and 0."oi, 0.1, 1, 10 or 100 nM of siRNA were introduced.  Measurement of RNAi effect<br>
The cells transfected with siRNA were recovered 20 hours after transfection, and using a Dual-Luciferase Reporter Assay System (manufactured by Promega Corporation), the levels of expression (luciferase activities) of two types of luciferase (P. pyralis luc and reniformis luc) protein were measured.  The amount of luminescence was measured using a Lumat LB9507 luminometer (EG&amp;G Berthold) .  Results<br>
The measurement results on the luciferase activities are shown in Fig. 10.  Furthermore, the results of study on correspondence between the luciferase activities and the individual base sequences are shown in Fig. 11.<br>
In Fig, 10, the graph represented by B shows the results in the drosophila cells, and the graph represented<br><br>
10, in the drosophila cells,, by creating RNA with a base number of 21, it was possible to inhibit the luciferase activities in almost all the sequences.  On the other hand, in the human cells, it was evident that it was difficult to obtain sequences which could inhibit the luciferase activities simply by setting the base number at 21.<br>
Analysis was then conducted on the regularity of base sequence with respect to RNA a to p.  As shown in Fig. 11, with respect to 5 points of the double-stranded RNA, the base sequence was analyzed.  With respect to siRNA a in the top row of the table shown in Fig. 11, the relative luciferase activity (RLA) is 0.03.  In the antisense strand, from the 3f end, the base sequence of the overhanging portion (OH) is UC; the G/C content (content of guanine or cytosine) in the subsequent 7 bases (31- T in Fig. 11) is 57%; the G/C content in the further subsequent 5 bases (M in Fig. 11) is 20; the G/C content in the further subsequent 7 bases (5'-T in Fig. 11) is 14%; the 51 end is U; and the G/C content in total is 32%.  In the table, a lower RLA value indicates lower RLA activity, i.e., inhibition of the expression of luciferase.<br>
As is evident from the results, in the base sequence of polynucleotides for causing RNA interference, it is highly probable that the 3' end is adenine or uracil and that the 51 end is guanine or cytosine.  Furthermore, it has become clear that the 7-base sequence from the 3f end is rich in adenine or uracil.<br><br>
1. Construction of target expression vector pTRSC<br>
A target expression vector was constructed as follows. A target expression molecule is a molecule which allows expression of RNA having a sequence to be targeted by RNAi (hereinafter, also referred to as a "target sequence").<br>
A target mRNA sequence was constructed downstream of the CMV enhancer/promoter of pCI-neo (GenBank Accession No. U47120, manufactured by Promega Corporation) (Fig. 25). That is, the following double-stranded oligomer was synthesized, the oligomer including a Kozak sequence (Kozak), an ATG sequence, a cloning site having a 23 base-pair sequence to be targeted (target), and an identification sequence for restriction enzyme (Nhel, EcoRI, Xhol) for recombination.  The double-stranded oligomer consists of a sequence shown in SEQ ID NO: 1 in the sequence listing and its complementary sequence.  The synthesized double-stranded oligomer was inserted into the Nhel/Xbal site of the pCI-neo to construct a target expression vector pTREC (Fig. 25).  With respect to the intron, the intron site derived from |3-globin originally incorporated in the pCI-neo was used. 5!-gctagccaccatggaattcacgcgtctcgagtctaga-3'<br>
(SEQ ID NO: 1) The pTREC shown in Fig. 25 is provided with a promoter and an enhancer (pro/enh) and regions PAR(F) 1 and PAR(R) 1 corresponding to the PCR primers.  An intron (Intron) is inserted into PAR(F) 1, and the expression<br><br>
does not become a template of PCR.  After transcription of RNA, in an environment in which splicing is performed in eukaryotic cultured cells or the like, the intron site of the pTREC is removed to join two neighboring PAR(F) l's. RNA produced from the pTREC can be amplified by RT-PCR. With respect to the intron, the intron site derived from (3-globin originally incorporated in the pCI-neo was used. The pTREC is incorporated with a neomycin-resistant gene (neo) as a control, and by preparing PCR primers corresponding to a part of the sequence in the neomycin-resistant gene and by subjecting the part of the neomycin-resistant gene to RT-PCR, the neomycin-resistant gene can be used as an internal standcird control (internal control). PAR(F) 2 and PAR(R) 2 represent the regions corresponding to the PCR primers in the neomycin-resistant gene. Although not shown in the example of Fig. 25, an intron may be inserted into at least one of PAR(F) 2 and PAR(R) 2. 2. Effect of primer for detecting target mRNA<br>
(1)	Transfection into cultured cells<br>
HeLa cells were seeded at 0.2 to 0.3 x 106 cells per well of a 24-well plate, and after one day, using Lipofectamine 2000 (manufactured by Invitrogen Corp.), 0.5 ug of pTREC vector was transfected according to the manual.<br>
(2)	Recovery of cells and quantification of mRNA<br>
One day after the transfection, the cells were recovered and total RNA was extracted with Trizol (manufactured by Invitrogen Corp.).  One hundred nanograms<br><br>
II RT (manufactured by Invitrogen Corp.), using oligo (dT) primers, to synthesize cDNA.  A control to which no reverse transcriptase was added was prepared.  Using one three hundred and twentieth of the amount of the resulting cDNA as a PCR template, quantitative PCR was carried out in a 50-ul reaction system using SYBR Green PCR Master Mix (manufactured by Applied Biosystems Corp.) to quantify target mRNA (referred to as mRNA (T)) and, as an internal control, mRNA derived from the neomycin-resistant gene in the pTREC (referred to as mRNA (C)).  A real-time monitoring apparatus ABI PRIZM7000 (manufactured by Applied Biosystems) was used for the quantitative PCR.  A primer pair T (SEQ ID NOs: 2 and 3 in the sequence listing) and a primer pair C (SEQ ID NOs: 4 and 5 in the sequence listing) were used for the quantification of mRNA (T) and mRNA (C), respectively. Primer pair T:<br>
aggcactgggcaggtgtc	(SEQ ID NO: 2)<br>
tgctcgaagcattaaccctcacta     (SEQ ID NO: 3) Primer pair C<br>
atcaggatgatctggacgaag  (SEQ ID NO: 4)<br>
ctcttcagcaatatcacgggt  (SEQ ID NO: 5)<br>
Figs. 26 and 27 show the results of PCR.  Each of 25  Figs. 26 and 27 is a graph in which the PCR product is<br>
y-<br>
taken on the axis of ordinate and the number of cycles of PCR is taken on the axis of abscissa.  In the neomycin-<br><br>
amplification of the PCR product between the case in which cDNA was synthesized by the reverse transcriptase (+RT) and the control case which no reverse transcriptase was added (-RT) (Fig. 26).  This indicates that not only cDNA but also the vector remaining in the cells also acted as a template and was amplified.  On the other hand, in target sequence mRNA, there is a large difference between the case in which the reverse transcriptase was added (+RT) and the case in which no transcriptase was added (-RT) (Fig. 27). This result indicates that since one member of the primer pair T is designed so as to sandwich the intron, cDNA derived from intron-free mRNA is efficiently amplified, while the remaining vector having the intron does not easily become a template.<br>
3. Inhibition of expression of target mRNA by siRNA (1) Cloning of evaluation sequence to target expression vector<br>
Sequences corresponding to the coding regions 812-834 and 35-57 of a human vimentin (VIM) gene (RefSeq ID: NM_003380) were targeted for evaluation.  The following synthetic oligonucleotides (evaluation sequence fragments) of SEQ ID NOs: 6 and 7 in the sequence listing were produced, the synthetic oligonucleotides including these sequences and identification sequences for EcoRI and Xhol. Evaluation sequence VIM35 (corresponding to 35-57 of VIM) 5'-gaattcgcaggatgttcggcggcccgggcctcgag-3'<br>
(SEQ ID NO: 6)<br><br>
VIM)<br>
5'-gaattcacgtacgtcagcaatatgaaagtctcgag-3!<br>
(SEQ ID NO: 7) Using the EcoRI and Xhol sites located on both ends of each of the evaluation sequence fragments, each fragment was cloned as a new target sequence between the EcoRI and Xhol sites of the pTREC, and thereby OTREC-VIM35 and pTREC-VIM812 were constructed.<br>
(2)	Production of siRNA<br>
siRNA fragments corresponding to the evaluation sequence VIM35 (SEQ ID NO: 8 in the sequence list. Fig. 28), the evaluation sequence VIM812 (SEQ ID NO: 9, Fig. 29), and a control sequence (siCont^leei, SEQ ID NO: 10, Fig. 30) were synthesized, followed by annealing.  Each of the following siRNA sequences is provided with an overhanging portion on the 31 end.<br>
siVIM35	5!-aggauguucggcggcccgggc-3 f<br>
(SEQ ID NO: 8)<br>
siVIM812	5!-guacgucagcaauaugaaagu-3'<br>
(SEQ ID NO: 9) As a control, siRNA for the luciferase gene was used. siControl        5!-cauucuauccgcuggaagaug-3!<br>
(SEQ ID NO: 10)<br>
(3)	Transfection into cultured cells<br>
HeLa cells were seeded at 0.2 to 0.3 .x 106 cells per well of a 24-well plate, and after one  day, using Lipofectamine 2000 (manufactured by Invitrogen Corp.),<br><br>
corresponding to the sequence derived from each VIM (siVIM35, siVIM812) were simultaneously transfected according to the manual.  Into the control cells, 0.5 ug of pTREC-VIM35 or pTREC-VIM812 and 100 nM of siRNA for the luciferase gene (siControl) were simultaneously transfected. (4) Recovery of cells and quantification of mRNA<br>
One day after the transfection, the cells were recovered and total RNA was extracted with Trizol (Invitrogen).  One hundred nanograms of the resulting RNA was reverse transcribed by Superscript II RT (manufactured by Invitrogen Corp.)/ using oligo (dT) primers, to synthesize cDNA.  Using one three hundred and twentieth of the amount of the resulting cDNA as a PCR template, quantitative PCR was carried out in a 50-ul reaction system using SYBR Green PCR Master Mix (manufactured by Applied Biosystems Corp.) to quantify mRNA (referred to as mRNA (T)) including the sequence derived from VIM to be evaluated and, as an internal control, mRNA derived from the neomycin-resistant gene in the pTREC (referred to as mRNA (C)).<br>
A real-time monitoring apparatus ABI PRIZM7000 (manufactured by Applied Biosystems) was used for the quantitative PCR.  The primer pair T (SEQ ID NOs: 2 and 3 in the sequence listing) and the primer pair C (SEQ ID NOs: 4 and 5 in the sequence listing) were used for the quantification of mRNA (T) and mRNA (C), respectively.  The ratio (T/C) of the resulting values of mRNA was taken on<br><br>
in a graph (Fig. 31).<br>
In the control cells, since siRNA for the luciferase gene does not affect target mRNA, the ratio T/C is substantially 1.  In VIM812 siRNA, the ratio T/C is extremely decreased.  The reason for this is that VIM812 siRNA cut mRNA having the corresponding sequence, and it was shown that VIM812 siRNA has the RNAi effect.  On the other hand, in VIM35 siRNA, the T/C ratio.was substantially the same as that of the control, and thus it was shown that the sequence of VIM35 does not substantially have the RNAi effect. [Example 3] 1. Inhibition of expression of endogenous vimentin by siRNA<br>
(1)	Transfection into cultured cells<br>
HeLa cells were seeded at 0.2 to 0.3 x 106 cells per well of a 24-well plate, and after one day, using Lipofectamine 2000 (manufactured by Invitrogen Corp.), 100 nM of siRNA for VIM (siVIM35 or siVIM812) or control siRNA (siControl) and, as a control for transfection efficiency, 0.5 ug of pEGFP (manufactured by Clontech) were simultaneously transfected according to the manual.  pEGFP is incorporated with EGFP.<br>
(2)	Assay of endogenous vimentin mRNA<br>
Three days after the transfection, the cells were recovered and total RNA was extracted with Trizol (manufactured by Invitrogen Corp.).  One hundred nanograms of the resulting RNA was reverse transcribed by Superscript<br><br>
primers, to synthesize cDNA.  PCR was carried out using the cDNA product as a template and using primers for vimentin, VIM-F3-84 and VIM-R3-274 (SEQ ID NOs: 11 and 12). VIM-F3-84;  gagctacgtgactacgtcca  (SEQ ID NO: 11) VIM-R3-274;  gttcttgaactcggtgttgat  (SEQ ID NO: 12)<br>
Furthermore, as a control, PCR was carried out using p-actin primers ACTB-F2-481 and ACTB-R2-664 (SEQ ID NOs: 13 and 14).  The level of expression of vimentin was evaluated under the common quantitative value of p-actin for each sample.<br>
ACTB-F2-481;  cacactgtgcccatctacga  (SEQ ID NO: 13) ACTB-R2-664;  gccatctcttgctcgaagtc  (SEQ ID NO: 14)<br>
The results are shown in Fig. 32.  In Fig. 32, the case in which siControl (i.e., the sequence unrelated to the target) is incorporated is considered as 100% for comparison, and the degree of decrease in mRNA of VIM when siRNA is incorporated into VIM is shown.  siVIM-812 was able to effectively inhibit VIM mRNA.  In contrast, use of siVIM-35 did not substantially exhibit the RNAi effect. (3) Antibody staining of cells<br>
Three days after the transfection, the cells were fixed with 3.7% formaldehyde, and blocking was performed in accordance with a conventional method.  Subsequently, a rabbit anti-vimentin antibody (a-VIM) or, as an internal control, a rabbit anti-Yes antibody (a-Yes) was added thereto, and reaction was carried out at room temperature. Subsequently, the surfaces of the cells were washed with<br><br>
antibody, a fluorescently-labeled anti-rabbit IgG antibody was added thereto.  Reaction was carried out at room temperature.  After the surfaces of the cells were washed with PBS, observation was performed using a fluorescence microscope.<br>
The fluorescence microscope observation results are shown in Fig. 33.  In the nine frames of Fig. 33, the parts appearing white correspond to fluorescent portions.  In EGFP and Yes, substantially the same expression was confirmed in all the cells.  In the cells into which siControl and siVIM35 were introduced, fluorescence due to antibody staining of vimentin was observed, and the presence of endogenous vimentin was confirmed.  On the other hand, in the cells into which siVIM812 was introduced, fluorescence was significantly weaker than that of the cells into which siControl and siVIM35 were introduced. The results show that endogenous vimentin mRNA was interfered by siVIM812, and consequently, the level of expression of vimentin protein was decreased.  It has become evident that siVIM812 also has the RNAi effect against endogenous vimentin mRNA.<br>
The results obtained in the assay system of the present invention [Example 2] matched well with the results obtained in the cases in which endogenous genes were actually treated with corresponding siRNA [Example 3]. Consequently, it has been confirmed that the assay system is effective as a method for evaluating the RNAi activity<br><br>
[Example 4]<br>
Base sequences were designed based on the predetermined rules (a) to (d).  The base sequences were designed by a base sequence processing apparatus which runs the siRNA sequence design program.  As the base sequences, 15 sequences (SEQ ID NOs: 15 to 29) which were expected to have RNAi activity and 5 sequences (SEQ ID NOs: 30 to 34) which were not expected to have RNAi activity were prepared.<br>
RNAi activity was evaluated by measuring the luciferase activity as in Example 1 except that the target sequence and siRNA to be evaluated were prepared based on each of the designed sequences.  The results are shown in Fig. 34.  A low luciferase relative activity value indicates an effective state, i.e., siRNA provided with RNAi activity.  All of the siRNA which was expected to have RNAi activity by the program effectively inhibited the expression of luciferase.<br>
[Sequences which exhibited RNAi activity; prescribed sequence portions, excluding overhanging portions] 5,gacgccaaaaacataaaga (SEQ ID NO: 15) 184,gttggcagaagctatgaaa (SEQ ID NO: 16) 272,gtgttgggcgcgttattta (SEQ ID NO: 17) 309/ccgcgaacgacatttataa (SEQ ID NO: 18) 428,ccaatcatccaaaaaatta (SEQ ID NO: 19) 515,cctcccggttttaatgaat (SEQ ID NO: 20) 658,gcatgccagagatcctatt (SEQ ID NO: 21) 695,ccggatactgcgattttaa (SEQ ID NO: 22)<br><br>
774,gatttcgagtcgtcttaat (SEQ ID NO: 24) 891,gcactctgattgacaaata (SEQ ID-NO: 25) 904,caaatacgatttatctaat (SEQ ID NO: 26) 1186,gattatgtccggttatgta (SEQ ID NO: 27) 1306,ccgcctgaagtctctgatt (SEQ ID NO: 28) 1586,ctcgacgcaagaaaaatca (SEQ ID NO: 29)<br>
[Sequences which did not exhibit RNAi activity; prescribed sequence portions, excluding overhanging portions] 14,aacataaagaaaggcccgg (SEQ ID NO: 30) 265,tatgccggtgttgggcgcg (SEQ ID NO: 31) 295,agttgcagttgcgcccgcg (SEQ ID NO:   32) 411,acgtgcaaaaaaagctccc (SEQ ID NO: 33) 1044,ttctgattacacccgaggg (SEQ ID NO: 34) [Example 5]<br>
siRNA sequences for the SARS virus were designed and the RNAi activities thereof were investigated.  The RNAi activity was evaluated by the same assay as used in Example 2 except that the target sequences and the sequences to be evaluated were changed.<br>
The siRNA sequences were designed with respect to 3CL-PRO, RdRp, Spike glycoprotein, Small envelope E protein, Membrane glycoprotein M, Nucleocapsid protein, and s2m motif from the genome of the SARS virus, using the siRNA sequence design program, so as to conform to the predetermined regularity.<br>
As a result of the assay shown in Fig. 35, 11 siRNA sequences designed so as to conform to the regularity<br><br>
sequences were incorporated as targets.  The case in which<br>
siControl (the sequence unrelated to SARS) is incorporated<br>
is considered as being 100%, and the relative amount of<br>
target mRNA in the case in which each siRNA sequence of<br>
SARS is incorporated is shown.  When each siRNA sequence is<br>
incorporated, the amount of target RNA was decreased to<br>
about 10% or less, and the presence of the RNAi activity<br>
was confirmed.<br>
[siRNA sequences designed (prescribed sequence portions,<br>
excluding overhanging portions)]<br>
siControl;gggcgcggtcggtaaagtt (SEQ ID NO: 35)<br>
3CL-PRO;SARS-10754;ggaattgccgtcttagata (SEQ ID NO: 36)<br>
3CL-PRO;SARS-10810;gaatggtcgtactatcctt (SEQ ID NO: 37)<br>
RdRp;SARS-14841;ccaagtaatcgttaacaat (SEQ ID NO: 38)<br>
Spike glycoprotein;SARS-23341;gcttggcgcatatattcta (SEQ ID<br>
NO: 39)<br>
Spike glycoprotein;SARS-24375;cctttcgcgacttgataaa (SEQ ID<br>
NO: 40)<br>
Small envelope E protein;SARS-26233;gtgcgtactgctgcaatat<br>
(SEQ ID NO: 41)<br>
Small envelope E protein;SARS-26288;ctactcgcgtgttaaaaat<br>
(SEQ ID NO: 42)<br>
Membrane glycoprotein M;SARS-26399;gcagacaacggtactatta (SEQ<br>
ID NO: 43)<br>
Membrane glycoprotein M;SARS-27024;ccggtagcaacgacaatat (SEQ<br>
ID NO: 44)<br>
Nucleocapsid protein;SARS-28685;cgtagtcgcggtaattcaa (SEQ ID<br><br>
s2m motif;SARS-29606;gatcgagggtacagtgaat (SEQ ID NO: 46) [Example 6]<br>
The following siRNA sequences were designed in accordance with the columns " siRNA sequence design program" and " Base sequence processing apparatus for running siRNA sequence design program, etc.".  The designed siRNA sequences are shown under SEQ ID NOs: 47 to 892 in the sequence listing. (Target gene of RNAi)<br>
NM_000604, Homo sapiens fibroblast growth factor receptor 1 (fms-related tyrosine kinase 2, Pfeiffer syndrome) (FGFR1). (Target sequences)<br>
NM_000604-807igtagcaacgtggagttcat (SEQ ID-NO: 47) NM_000604-806,ggtagcaacgtggagttca (SEQ ID NO: 48) NM_000604-811,caacgtggagttcatgtgt (SEQ ID NO: 49) NM_000604-880,ggtgaatgggagcaagatt (SEQ ID .NO: 50) NM_000604-891,gcaagattggcccagacaa (SEQ ID NO: 51) (Target sequence effective for mouse homolog) NMJ)00604-818.gagttcatgtgtaaggtgt (SEQ ID NO: 52) (Target gene of RNAi)<br>
NM_000141, Homo sapiens fibroblast growth factor receptor 2 (bacteria-expressed kinase, keratinocyte growth factor receptor, craniofacial dysostosis 1, Crouzon syndrome, Pfeiffer syndrome, Jackson-Weiss syndrome) (FGFR2). (Target sequences)<br>
NM__000141-612,gaggctacaaggtacgaaa (SEQ ID NO: 53) NM_000141-615,gctacaaggtacgaaacca (SEQ ID NO: 54)<br>
\*~\   -T	f&gt;-     -*\ ;"**  **   .( ".	^     "&gt; ""'       -  ■'-    -  -•   -  --  — —t   -^    -	-	'—	......	"        ' '        •<br><br>
NM_000141-574,gaaaaacgggaaggagttt (SEQ ID NO: 56) (Target sequences effective for mouse homolog) NM_000141-595,gcaggagcatcgcattgga (SEQ ID NO: 57) NM_000141-69,ccttcagtttagttgagga (SEQ ID NO: 58) NMJD00141-7Q,cttcagtttagttga.ggat (SEQ ID NO: 59) (Target gene of RNAi)<br>
NM_000142, Homo sapiens fibroblast growth factor receptor 3 (achondroplasia, thanatophoric dwarfism) (FGFR3). (Target sequences)<br>
NM_000142-899,gacggcacaccctacgtta (SEQ ID NO: 60) NM__000142-1925,cacaacctcgactactaca (SEQ ID NO: 61) NM_000142-2154,gcacacacgacctgtacat (SEQ ID NO: 62) NM__000142-678,cctgcgtcgtggagaacaa (SEQ ID NO: 63) NM__000142-2157,cacacgacctgtacatgat (SEQ ID NO: 64) (Target sequence effective for mouse homolog) "N!^"000r42"-8T2Tgagttccactgcaaggtgt (SEQ ID NO: 65) (Target gene of RNAi)<br>
NM_004448, Homo sapiens v-erb-b2 erythroblastic leukemia viral oncogene homolog 2, neuro/glioblastoma derived oncogene homolog (avian) (ERBB2). (Target sequences)<br>
NM_004448-356,ggagacccgctgaacaata (SEQ ID NO: 66) NM_004448-3645,ccttcgacaacctctatta (SEQ ID NO: 67) NM_004448-3237,gggctggctccgatgtatt (SEQ ID NO: 68) NM__004448-3238,ggctggctccgatgtattt (SEQ ID NO: 69) NM_004448-3240,ctggctccgatgtatttga (SEQ ID NO: 70)<br><br>
NM_001982, Homo sapiens v-erb-b2 erythroblastic leukemia viral oncogene homolog 3 (avian) (ERBB3). (Target sequences)<br>
NM_001982-1347,gtgctgggcgtatctatat (SEQ ID NO: 71) NM_001982-1349,gctgggcgtatctatataa (SEQ ID NO: 72) NM_001982-1548,gcttgtcctgtcgaaatta (SEQ ID NO: 73) NM_001982-1549,cttgtcctgtcgaaattat (SEQ ID NO: 74) NM_001982-2857,cattcgcccaacctttaaa (SEQ ID NO: 75) (Target gene of RNAi)<br>
NM_005235, Homo sapiens v-erb-a erythroblastic leukemia viral oncogene homolog 4 (avian) (ERBB4). (Target sequences)<br>
NM_005235-295,ggagaatttacgcattatt (SEQ ID NO: 76) NM_005235-2120,gctcaacttcgtattttga (SEQ ID NO: 77) NM_005235-2940,ctcaaagatacctagttat (SEQ ID NO: 78) NM_005235-2121,ctcaacttcgtattttgaa (SEQ ID NO: 79) NM_005235-2880,ctgacagtagacctaaatt (SEQ ID NO: 80) (Target gene of RNAi)<br>
NM_002227, Homo sapiens Janus kinase 1 (a protein tyrosine kinase) (JAK1). (Target sequences)<br>
NM_002227-441,ctcagggacagtatgattt (SEQ ID NO: 81) NM_002227-1299,cagaatacgccatcaataa (SEQ ID NO: 82) NM_002227-673,gatgcggataaataatgtt (SEQ ID NO: 83) NM_002227-672,ggatgcggataaataatgt (SEQ ID NO: 84) NM_002227-3385,ctttcagaaccttattgaa (SEQ ID NO: 85) (Target sequences effective for mouse homolog)<br>
vig      r*- p- n *~ —   - i-. -^   _ _ _- .        „ ... ,-.f ^, .	■ "    —7*, ^Jfi  ♦   0 Z  '<br><br>
NM_002227-3042,caattgaaaccgataagga (SEQ ID NO: 87) NM_002227-2944,gggttctcggcaatacgtt (SEQ ID NO: 88) (Target gene of RNAi)<br>
NM_004972, Homo sapiens Janus kinase 2 (a protein tyrosine kinase) (JAK2), (Target sequences)<br>
NM_004972-2757,ctggtcggcgtaatctaaa (SEQ ID NO: 89) NM_004972-2759,ggtcggcgtaatctaaaat (SEQ ID NO: 90) NM_004972-2760,gtcggcgtaatctaaaatt (SEQ ID NO: 91) NM__004972-3175,ggaatttatgcgtatgatt (SEQ ID NO: 92) NM_004972-1452,ctgttcgctcagacaatat (SEQ ID NO: 93) (Target sequences effective for mouse homolog) NM_004972-872,ggaaacggtggaattcagt (SEQ ID NO: 94) NM_004972-870,ctggaaacggtggaattca (SEQ ID NO: 95) NM_004972-847,gatttttgcaaccattata (SEQ ID NO: 96)_ (Target gene of RNAi)<br>
NM__000215, Homo sapiens Janus kinase 3 (a protein tyrosine kinase, leukocyte) (JAK3). (Target sequences)<br>
NM_000215~2315,gtcattcgtgacctcaata (SEQ ID NO: 97) NM_000215-2522,gacccgctagcccacaata (SEQ ID NO: 98) NM_000215-2524,cccgctagcccacaataca (SEQ ID NO: 99) NM_000215-1788,ccatggtgcaggaatttgt (SEQ ID NO: 100) NM_000215-1825,catgtatctgcgaaaacgt (SEQ ID NO: 101) (Target gene of RNAi) NM_003331, Homo sapiens tyrosine kinase 2 (TYK2).<br>
(Target sequences)<br>
NZt_003331-3213,gccrgaaggagtataagtt   (SEQ  ID  NO:   102)<br><br>
NM_003331-2658,cggaccctacggttttcca (SEQ ID NO: 103) NM_003331-299,ctatatttccgcataaggt (SEQ ID NO: 104) (Target sequences effective for mouse homolog) NM_003331-2674,ccacaagcgctatttgaaa (SEQ ID NO: 105) NM_003331-2675,cacaagcgctatttgaaaa (SEQ ID NO: 106) NM_003331-328,gaactggcatggcatgaat (SEQ ID NO: 107) (Target gene of RNAi)<br>
NM_001079, Homo sapiens zeta-chain (TCR) associated protein kinase 70kDa (ZAP70). (Target sequences)<br>
NM_001079-512,gaggccgagcgcaaacttt (SEQ ID NO: 108) NM_001079-1512,ggtacgcacccgaatgcat (SEQ ID NO: 109) NM_001079-242,gagctctgcgagttctact (SEQ ID NO: 110) NM_00I079-929,gacacgagcgtgtatgaga (SEQ ID NO: 111) NM_001079-1412,cggcactacgccaagatca (SEQ ID NO: 112) (Target sequence effective for mouse homolog) NM_001079-1566,ggagctatggggtcaccat (SEQ ID NO: 113) (Target gene of RNAi)<br>
NM_005417, Homo sapiens v-src sarcoma (Schmidt-Ruppin A-2) viral oncogene homolog (avian) (SRC). (Target sequences)<br>
NML005417-185,ctgttcggaggcttcaact (SEQ ID NO: 114) NM_005417-685/ggtggcctactactccaaa (SEQ ID NO: 115) NM__005417-474,gggagtcagagcggttact (SEQ ID NO: 116) NM_005417-480,cagagcggttactgctcaa (SEQ ID NO: 117) NM__005417-567,cagtgtctgacttcgacaa (SEQ ID NO: 118) (Target sequence effective for mouse homolog) NM_Q05417-651,cctcccgcacccagttcaa (SEQ ID NO: 119)<br><br>
(Target gene of RNAi)<br>
NM_002350, Homo sapiens v-yes-1 Yamaguchi sarcoma viral related oncogene homolog (LYN). (Target sequences)<br>
NM„002350-610,cagcgacatgattaaacat (SEQ ID NO: 120) NM_002350-533,gttattaagcactacaaaa (SEQ ID NO: 121) NM_002350-606,gtatcagcgacatgattaa (SEQ ID NO: 122) (Target sequences effective for mouse homolog) NM_002350-783,ggatgggttactataacaa (SEQ ID NO: 123) NM_002350-694,gaagccatgggataaagat (SEQ ID NO: 124) NM_002350-541fgcactacaaaattagaagt (SEQ ID NO: 125) (Target gene of RNAi)<br>
NM_005157, Homo sapiens v-abl Abelson murine leukemia viral oncogene homolog 1 (ABL1). (Target sequences)<br>
NM_005157-232,cactctaagcataactaaa (SEQ ID NO: 126)<br>
NM_005157-770,gagggcgtgtggaagaaat (SEQ ID NO: 127)<br>
NML005157-262,ccgggtcttaggctataat (SEQ ID NO: 128)<br>
NM„005157-264,gggtcttaggctataatca (SEQ ID NO: 129)<br>
NM_0G5157-484,catctcgctgagatacgaa (SEQ ID NO: 130)<br>
(Target sequences effective for mouse homolog)<br>
NM_005157-217,ggccagtggagataacact (SEQ ID NO: 131)<br>
NM„005157-1227,gcctggcctacaacaagtt (SEQ ID NO: 132)<br>
NM_005157-680,gtgtcccccaactacgaca (SEQ ID NO: 133)<br>
(Target gene of RNAi)	, .<br>
NM„005158, Homo sapiens v-abl Abelson murine leukemia viral oncogene homolog 2 (arg, Abelson-related gene) (ABL2). (Target sequences)<br><br>
NM_005158-3273,ctcaaactcgcaacaaatt (SEQ ID NO: 134) NM_005158-3272,cctcaaactcgcaacaaat (SEQ ID NO: 135) NM_005158-1425,ctaaggtttatgaacttat (SEQ ID NO: 136) NM_005158-448,gctcagcagtctaa.tcaat (SEQ ID NO: 137) NM_005158~3110,caggccgctgagaaaatct (SEQ ID NO: 138) (Target gene of RNAi)<br>
NM_004071, Homo sapiens CDC-like kinase 1 (CLK1). (Target sequences)<br>
NM_004071-1215,ccaggaaacgtaaatattt (SEQ ID NO: 139) NM_004071-774,catttcgactggatcatat (SEQ ID NO: 140) NM__004071-1216,caggaaacgtaaat.atttt (SEQ ID NO: 141) NM_004071-973,ctttggtagtgcaacatat (SEQ ID NO: 142) NM_004071-463,cgtactaagtgcaagatat (SEQ ID NO: 143) (Target gene of RNAi)<br>
NM_001291, Homo sapiens CDC-like kinase 2 (CLK2) (Target sequences)<br>
NM_001291-202,gtatgaccggcgatactgt (SEQ ID NO: 144) NM_001291-225,gctacagacgcaacgatta (SEQ ID NO: 145) NM_001291-226,ctacagacgcaacgat:tat (SEQ ID NO: 14 6) NM_001291-45,ggagttaccgtgaacacta (SEQ ID NO: 147) NM_001291-46,gagttaccgtgaacactat (SEQ ID NO: 148) (Target gene of RNAi)<br>
NM_001292, Homo sapiens CDC-like kinase 3 (CLK3). (Target sequences)<br>
NM_001292-189,gccgtgacagcgatacata (SEQ ID NO: 149)<br>
NM_001292-72,cctacagtcgggaacatga (SEQ ID NO: 150)<br>
NM_001292-73,ctacagtcgggaacatgaa (SEQ ID NO: 151)<br>
NM_001291-".3 3 , c~-~~   ^gec".;: . '".	"<br><br>
NM_001292-121,gcctcccccacgaagatct (SEQ ID NO: 153) (Target sequence effective for mouse homolog) NM_001292-388,ggtgaaggcacctttggca (SEQ ID NO: 154) (Target gene of RNAi)<br>
NM_020666, Homo sapiens CDC-like kinase 4 (CLK4). (Target sequences)<br>
NM_020666-617,gtattagagcacttaaata (SEQ ID NO: 155) NM_020666-1212,gaaaacgcaagta':tttca (SEQ ID NO: 156) NM_020666-1348,cctggttcgaagaatgtta (SEQ ID NO: 157) NM_020666-181,cttgaatgagcgagattat (SEQ ID NO: 158) NM_020666-803,cagatctgccagtcaataa (SEQ ID NO: 159) (Target sequences effective for mouse homolog) NM_J)20666-457,cgttctaagagcaagatat (SEQ ID NO: 160) NM__020666-446,caaagtggagacgttctaa (SEQ ID NO: 161) NM_020666~461,ctaagagcaagatatgaaa (SEQ ID NO: 162) (Target gene of RNAi)<br>
NM_002093, Homo sapiens glycogen synthase kinase 3 beta (GSK3B).<br>
(Target sequences)<br>
NM__002093-326,gtccgattgcgttatttct (SEQ ID NO: 163) NM_002093-307,gctagatcactgtaacata (SEQ ID NO: 164) NM_002093-451,gacgctccctgtgatttat (SEQ ID NO: 165) NM_002093-632,cccaatgtttcgtatatct (SEQ ID NO: 166) NM_0G2093-623,cgaggagaacccaatgttt (SEQ ID NO: 167) (Target sequences effective for mouse homolog) NM„002093-206,gtatatcaagccaaacttt (SEQ ID NO: 168) NM_002093-195,catttggtgtggtatatca (SEQ ID NO: 169)<br><br>
(Target gene of RNAi)<br>
NM_1826?1, Homo sapiens SFRS protein kinase 2 (SRPK2).<br>
(Target sequences)<br>
NM_182691-1312,gccaaatggacgacataaa (SEQ ID NO: 171)<br>
NM_182691-1313,ccaaatggacgacataaaa (SEQ ID NO: 172)<br>
NM__182691-1314,caaatggacgacataaaat (SEQ ID NO: 173)<br>
NM__182691-1985,ctgatcccgatgttagaaa (SEQ ID NO: 174)<br>
NM_182691-233,ggccggtatcatgttatta (SEQ ID NO: 175)<br>
(Target gene of RNAi)<br>
NM_005430, Homo sapiens wingless-type MMTV integration site<br>
family, member 1 (WNT1).<br>
(Target sequences)<br>
NM_005430-614,ggccgtacgaccgtattct (SEQ ID NO: 176)<br>
NM_005430-205,gcgtctgatacgccaaaat (SEQ ID NO: 177)<br>
NM_005430-855,cccacgacctcgtctactt (SEQ ID NO: 178)<br>
NM_005430-196,caaacagcggcgtctgata (SEQ ID NO: 179)<br>
(Target sequences effective for mouse homolog)<br>
NM_005430-875,gagaaatcgcccaacttct (SEQ ID NO: 180)<br>
NM_005430-863,ctcgtctacttcgagaaat (SEQ ID NO: 181)<br>
NM_005430-860,gacctcgtctacttcgaga (SEQ ID NO: 182)<br>
(Target gene of RNAi)<br>
NM_003391., Homo sapiens wingless-type MMTV integration site<br>
family member 2 (WNT2).<br>
(Target sequences)<br>
NM„003391-lll,gggtgatgtgcgataatgt (SEQ ID NO: 183)<br>
NM_003391-681/ggaaaacgggcgattatct (SEQ ID NO: 184)<br>
NM_003391-764,gctaacgagaggtttaaga (SEQ ID NO: 185)<br><br>
NM_003391-295,ggtcctactccgaagtagt (SEQ ID NO: 187) (Target sequences effective for mouse homolog) NM_003391-797,gacctcgtgtattttgaga (SEQ ID NO: 188) NM__003391-790,gaaaaatgacctcgtgtat (SEQ ID NO: 189) NM__003391-789,cgaaaaatgacctcgtgta (SEQ ID NO: 190) (Target gene of RNAi)<br>
NM_004625, Homo sapiens wingless-type MMTV integration site family, member 7A (WNT7A). (Target sequences)<br>
NM_004625-92,ctgggcgcaagcatcatct (SEQ ID NO: 191) NM_004625-313,gttcacctacgccatcatt (SEQ ID NO: 192) NM_004625-524,gcccggactctcatgaact (SEQ ID NO: 193) NM_004625-480,gcttcgccaaggtctttgt (SEQ ID NO: 194) (Target sequences effective for mouse homolog) NM_004625-205,cctggacgagtgtcagttt (SEQ ID NO: 195) NMZ004625-209/gacgagtgtcagtttcagt (SEQ ID NO: 196) NM_004625-172,catcatcgtcataggagaa (SEQ ID NO: 197) (Target gene of RNAi)<br>
NM_004626, Homo sapiens wingless-type MMTV integration site family, member 11 (WNT11). (Target sequences)<br>
NM_004626-543,gatcccaagccaataaact (SEQ ID NO: 198) NM_004626-917,gacagctgcgaccttatgt (SEQ ID NO: 199) NM_004626-915,gcgacagctgcgaccttat (SEQ ID NO: 200) NM_004626-54,ccggcgtgtgctatggcat (SEQ ID NO: 201) (Target sequences effective for mouse homolog) NM_004626-59,gtgtgctatggcatcaagt (SEQ ID NO: 202) NM_0G4626-560,crgacgcgtctacacaaca (SEQ ID NO: 203)<br><br>
NM_004626-562,gatgcgtctacacaacagt (SEQ ID NO: 204) (Target gene of RNAi)<br>
NM_030753, Homo sapiens wingless-type MMTV integration site family, member 3 (WNT3). (Target sequences)<br>
NM_030753-417,gctgtgactcgcatcataa (SEQ ID NO: 205) NM__030753-483,ctgacttcggcgtgttagt (SEQ ID NO: 206) NM_030753-485,gacttcggcgtgttagtgt (SEQ ID NO: 207) (Target sequences effective for mouse homolog) NM_030753-887,gaccggacttgcaatgtca (SEQ ID NO: 208) NM_030753-56,ctcgctggctacccaattt (SEQ ID NO: 209) NM_030753-59,gctggctacccaatttggt (SEQ ID NO: 210) (Target gene of RNAi)<br>
NM_033131, Homo sapiens wingless-type MMTV integration site family, member 3A (WNT3A). (Target sequences)<br>
NM_033131-2,gccccactcggatacttct (SEQ ID NO: 211) NM_033131-3,ccccactcggatacttctt (SEQ ID NO: 212) NM_033131-4,cccactcggatacttctta (SEQ ID NO: 213) NM„033131-77,gctgttgggccacagtatt (SEQ ID NO: 214) NM_033131-821,gaggcctcgcccaacttct (SEQ ID NO: 215) (Target sequences effective for mouse homolog) NM_033131-168,ggaactacgtggagatcat (SEQ ID NO: 216) NM„033131-50,ggcagctacccgatctggt (SEQ ID NO: 217) NM_033131-165,gcaggaactacgtggagat (SEQ ID NO: 218) (Target gene of RNAi)<br>
NM„003392, Homo sapiens wingless-type MMTV integration site family, member 5A (WNT5A).<br><br>
(Target sequences)<br>
NM_003392-91,gtggtcgctaggtatgaat (SEQ ID NO: 219) NM_003392-93,ggtcgctaggtatgaataa (SEQ ID NO: 220) NM_003392-307,ggataacacctctgttttt (SEQ ID NO: 221) NM_003392-57,ccttcgcccaggttgtaat (SEQ ID NO: 222) NM_003392-87,cttggtggtcgctaggtat (SEQ ID NO: 223) (Target sequences effective for mouse homolog) NM_003392-163,ccaactggcaggactttct (SEQ ID NO: 224) NM_003392-116,gttcagatgtcagaagtat (SEQ ID NO: 225) NM__003392-102,gtatgaataaccctgttca (SEQ ID NO: 226) (Target gene of RNAi)<br>
NM_004196, Homo sapiens cyclin-dependent kinase-like 1 (CDC2-related kinase) (CDKL1). (Target sequences)<br>
NM_004196-405,cgaaacattccgtgattaa (SEQ ID NO: 227) NM_004196-305,ctcgtgaagagcataactt (SEQ ID NO: 228) NM_004196-458,ggaccgagtgactactata (SEQ ID NO: 229) NM_004196-844,gttgcatcacccatatttt (SEQ ID NO: 230) NM_004196-330,cactgcaagctgtaaattt (SEQ ID NO: 231) (Target sequence effective for mouse homolog) NM__004196-119,gatgaccctgtcataaaga (SEQ ID NO: 232) (Target gene of RNAi)<br>
NM_003948, Homo sapiens cyclin-dependent kinase-like 2 (CDC2-related kinase) (CDKL2). (Target sequences)<br>
NM__003948-623,gatcagctatatcatatta (SEQ ID NO': 233) NM_003948-1379,ccatcaggcatttataaca (SEQ ID NO: 234) NM_003948-1380,catcaggcatttataacat (SEQ ID NO: 235)<br><br>
NM_003948-768,ctgaagtggtgatagattt (SEQ ID NO: 236) NM_003948-626,cagctatatcatattatga (SEQ ID NO: 237) (Target sequences effective for mouse homolog) NM_003948-325,gattattaatggaattgga (SEQ ID NO: 238) NM_003948-1012,ggtacaggataccaatgct (SEQ ID NO: 239) (Target gene of RNAi)<br>
NM_016508, Homo sapiens cyclin-dependent kinase-like 3 (CDKL3).<br>
(Target sequences)<br>
NM_016508-498,gagctcccgaattagtatt (SEQ ID NO: 240) NM_016508-500,gctcccgaattagtattaa (SEQ ID NO: 241) NM_016508-1290,cacccatcaatctaactaa (SEQ ID NO: 242) NM_016508-1301,ctaactaacagtaatttga (SEQ ID NO: 243) NM_016508-501,ctcccgaattagtattaaa (SEQ ID NO: 244) (Target sequences effective for mouse homolog) NM_016508-785,gttcatgcttgtttacaaa (SEQ ID NO: 245) NM_016508-555,ctttgggctgtatgatcat (SEQ ID NO: 246) NM_016508-776,gcagatatagttcatgctt (SEQ ID NO: 247) (Target gene of RNAi)<br>
NM_002745, Homo sapiens mitogen-activated protein kinase 1 (MAPK1).<br>
(Target sequences)<br>
NM_002745-746,gaagacctgaattgtataa (SEQ ID NO: 248) NM_002745-276,caaccatcgagcaaatgaa (SEQ ID NO: 249) NM_002745-849,ccaaagctctggacttatt (SEQ ID NO: 250) NM_002745-749,gacctgaattgtataataa (SEQ ID NO: 251) NM_002745-113,gtgtgctctgcttatgata (SEQ ID NO: 252) (Target sequences effective for mouse homolog)<br><br>
NM_002745-220,cttactgcgcttcagacat (SEQ ID NO: 253) NM_002745-228,gcttcagacatgagaacat (SEQ ID NO: 254) NM_002745-224,ctgcgcttcagacatgaga (SEQ ID NO: 255) (Target gene of RNAi) 5  NM_016231, Homo sapiens nemo-like kinase (NLK). (Target sequences)<br>
NM__016231-450,gagtagcgctcaaaaagat (SEQ ID NO: 256) NM_016231-1074,gcgctaaggcacatatact (SEQ ID NO: 257) NM_016231-962/ctactaggacgaagaatat (SEQ ID NO: 258)<br>
10  NM_016231-579,ctccacacattgactattt (SEQ ID NO: 259) (Target sequences effective for mouse homolog) NM_016231-703,gattttgcgaggtttgaaa (SEQ ID NO: 260) NM__016231-1382,gtccgacaggttaaagaaa (SEQ ID NO: 261) NM„016231-1384,ccgacaggttaaagaaatt (SEQ ID NO: 262)<br>
15  (Target gene of RNAi)<br>
NM_001315, Homo sapiens mitogen-activated protein kinase 14<br>
(MAPK14).<br>
(Target sequences)<br>
NM_001315-401,ctccgaggtctaaagtata (SEQ ID NO: 263)<br>
20 NM_001315-403,ccgaggtctaaagtatata (SEQ ID NO: 264) NM_001315-251,ggtctgttggacgttttta (SEQ ID NO: 265) NM_001315-212,ctgcggttacttaaacata (SEQ ID NO: 266) NMJ)01315-405,gaggtctaaagtatataca (SEQ ID NO: 267) (Target sequence effective for mouse homolog)<br>
25  NM_001315-664,gtttcctggtacagaccat (SEQ ID NO: 268) . . (Target gene of RNAi)<br>
NM__002751, Homo sapiens mitogen-activated protein kinase 11 (MAPK11).<br><br>
w^	3^«ui. \&amp;r*\i   ±v   WU: 211)<br>
15  NM„002969-1013,gaatggaagcgtgttactt (SEQ ID NO: 2J78) (Target gene of RNAi)<br>
NM_002754, Homo sapiens mitogen-activated protein kinase 13 (MAPK13).<br>
(Target sequences) 20 NM_002754-164,ctgagccgaccctttcagt (SEQ ID NO: 279) NM_002754-174,cctttcagtccgagatctt (SEQ ID NO: 280) NM_002754-978,ccttagaacacgagaaact (SEQ ID NO: 281) NM_002754-285,ccctgcgcaacttctatga (SEQ ID NO: 282) NM__002754-287,ctgcgcaacttctatgact (SEQ ID NO: 283) 25  (Target gene of RNAi)<br>
NM_139049, Homo sapiens mitogen-activated protein kinase 8<br>
(MAPK8).<br>
(Target sequences)<br><br>
NM_139049-449,gacttaaagcccagtaata (SSQ ID NO: 284) NM_139049-213,gagagctagttcttatgaa (SSQ ID NO: 285) NM_139049-451,cttaaagcccagtaatata (SSQ ID NO: 286) (Target sequences effective for mouse homolog) NM_139049-525,caggaacgagttttatgat (SSQ ID NO: 287) NM_139049-524,gcaggaacgagttttatga (SSQ ID NO: 288) NM_139049-283,gaaatccctagaagaattt (SSQ ID NO: 289) (Target gene of RNAi)<br>
NM_002752, Homo sapiens mitogen-activated protein kinase 9 (MAPK9).<br>
(Target sequences)<br>
NM_002752-116,gtttgtgctgcatttgata (SHQ ID NO: 290) NM_002752-204,gagcttatcgtgaacttgt (SEQ ID NO: 291) (Target sequences effective for mouse homolog)<br>
NM_002752-878,gccagagatctgttatcaa (SEQ JD NO: 292J	<br>
NM_002752-879,ccagagatctgttatcaaa (SEQ ID NO: 293) NM_002752-880,cagagatctgttatcaaaa (SEQ ID NO: 294) (Target gene of RNAi)<br>
NM_002753, Homo sapiens mitogen-activated protein kinase 10 (MAPK10).<br>
(Target sequences)<br>
NM_002753-668,gtggtgacacgttattaca (SEQ ID NO: 295) NM_002753-957,cggactccgagcacaataa (SEQ ID NO: 296) NM_002753-958,ggactccgagcacaataaa (SEQ ID NO: 297) NM_002753-811#gtggaataaggtaattgaa (SEQ ID NO: 298) NM_002753-1212,ctaaaaatggtgtagtaaa (SEQ ID NO: 299) (Target sequences effective for mouse homolog) NM_002753-1167,ggaaagaacttatctacaa (SEQ ID NO: 300)<br><br>
NM_002753-584,gtagtcaagtctgattgca (SEQ ID NO: 301) NM_002753-761,gaaatggttcgccacaaaa (SEQ ID NO: 302) (Target gene of RNAi)<br>
NM_001786, Homo sapiens cell division cycle 2, Gl to S and G2 to M (CDC2). (Target sequences)<br>
NM_001786-782,gatttgctctcgaa£.atgt (SEQ ID NO: 303) NM__001786-788,ctctcgaaaatgttaatct (SEQ ID NO: 304) NM_001786-658,gggcactcccaataatgaa (SEQ ID NO: 305) NM_001786-696,ctttacaggactataagaa (SEQ ID NO: 306) NM_001786-562,gagtataggcaccatattt (SEQ ID NO: 307) (Target sequence effective for mouse homolog) NM_001786-869,gacaatcagattaagaaga (SEQ ID NO: 308) (Target gene of RNAi)<br>
NM__001798, Homo sapiens cyclin-dependent kinase 2 (CDK2) (Target sequences)<br>
NM_001798-224,ctctacctggtttttgaat (SEQ ID NO: 309) NM_001798-690,cttctatgcctgattacaa (SEQ ID NO: 310) NM_001798-770,gatggacggagcttgttat (SEQ ID NO: 311) NM„001798-226,ctacctggtttttgaattt (SEQ ID NO: 312) NM_001798-36,gcacgtacggagttgtgta (SEQ ID NO: 313) (Target gene of RNAi)<br>
NM__000075, Homo sapiens cyclin-dependent kinase 4 (CDK4) (Target sequences)<br>
NM__000075-45,cctatgggacagtgtacaa (SEQ ID „NQ: 314) NM__000075-616#gatgtttcgtcgaaagcct (SEQ ID NO: 315) NM__000075-161/cgtgaggtggctttactga (SEQ ID NO: 316) NM_000075-35,ggtgtcggtgcctatggga (SEQ ID NO: 317)<br><br>
NM_000075-242,cgaactgaccgggagatca (SEQ ID NO: 318) (Target gene of RNAi)<br>
NM_052984, Homo sapiens cyclin-dependent kinase 4 (CDK4), transcript variant 2, mRNA.,228..563 , 0 (Target sequences)<br>
NM„052984-248,gaccgggagatcaagagat (SEQ ID NO: 319) NM_052984-251,cgggagatcaagagatgtt (SEQ ID NO: 320) (Target gene of RNAi)<br>
NM_001799, Homo sapiens cyclin-dependent kinase 7 (M015 homolog, Xenopus laevis, cdk-activating kinase) (CDK7). (Target sequences)<br>
NM_001799-242,ggacataaatctaatatta (SEQ ID NO: 321) NM_001799-104/caaattgtcgccattaaga (SEQ ID NO: 322) NM_001799-490,ccccaatagagcttataca (SEQ ID NO: 323) NM_001799-20#cgggcaaagcgttatgaga (SEQ ID NO: 324) NM_001799-21,gggcaaagcgttatgagaa (SEQ ID NO: 325) (Target sequence effective for mouse homolog) NM__001799-345,cctacatgttgatgactct (SEQ ID NO: 326) (Target gene of RNAi)<br>
NM_000455, Homo sapiens serine/threonine kinase 11 (Peutz-Jeghers syndrome) (STK11). (Target sequences)<br>
NM_000455-306,ggaggttacggcacaaaaa (SEQ ID NO: 327) NM_000455-307,gaggttacggcacaaaaat (SEQ ID NO: 328) NM_000455-309,ggttacggcaoaaaaatgt (SEQ ID NO: 329) NM_000455-1157,cccaaggccgtgtgtatga (SEQ ID NO: 330) NM_000455-1158,ccaaggccgtgtgtatgaa (SEQ ID NO: 331) (Target sequence effective for mouse homolog)<br><br>
NMJ)00455-916,cagctggttccggaagaaa (SEQ ID NO: 332) (Target gene of RNAi)<br>
NM_001274, Homo sapiens CHK1 checkpoint homolog (S. pombe) (CHEK1). i  (Target sequences)<br>
NM_001274-456,cagtatttcggtataataa (SEQ ID NO: 333) NM_001274-361,gcatggtattggaataact (SEQ ID NO: 334) NM_001274-990,gcccctcatacattgataa (SEQ ID NO: 335) NM_J301274-1038,ccacatgtcctgatcatat (SEQ ID NO: 336) NM__001274-227,ggcaatatccaatatttat (SEQ ID NO: 337) (Target sequences effective for mouse homolog) NM_001274-573,ggtcctgtggaatagtact (SEQ ID NO: 338) NM_001274-416,gaaagggataacctcaaaa (SEQ ID NO: 339) NM_001274-577,ctgtggaatagtacttact (SEQ ID NO: 340) (Target gene of RNAi)<br>
NM—002648, Homo sapiens pim-1 oncogene (PIM1). (Target sequences)<br>
NM_002648-831,ggccaaccttcgaagaaat (SEQ ID NO: 341) NM_002648-601,cgatgggacccgagtgtat (SEQ ID NO: 342) NM_002648-602,gatgggacccgagtgt:ata (SEQ ID NO: 343) NM__002648-293,ggtttctccggcgtcatta (SEQ-ID NO: 344) NM_002648-834,caaccttcgaag.aaatcca (SEQ ID NO: 345) (Target sequences effective for mouse homolog) NM_002648-96,ccctggagtcgcagtacca (SEQ ID NO: 346) NM_002648-203,gtggagaaggaccggattt (SEQ ID NO: 347) (Target gene of RNAi)<br>
NM_006875/ Homo sapiens pim-2 oncogene (PIM2). (Target sequences)<br><br>
NM_006875-698,ggggacattccctttgaga (SHQ ID NO: 348) NM_006875-242,ctcgaagtcgcactgctat (SHQ ID NO: 349) NM_006875-245,gaagtcgcactgctatgga (SEQ ID NO: 350) NM_006875-499,gaacatcctgatagaccta (SEQ ID NO: 351) NM_006875-468,gtggagttgtccatcgtga (SEQ ID NO: 352) (Target gene of RNAi)<br>
NM_021643, Homo sapiens tribbles homolog 2 (TRB2). (Target sequences)<br>
NM_021643-174,cttgtatcgggaaatactt (SEQ ID NO: 353) NM_021643-71,gaagagttgtcgtctataa (SEQ ID NO: 354)<br>
NM_021643-177,gtatcgggaaatacttatt (SEQ ID NO: 355) NM_021643-524,ctcaagctgcggaaattca (SEQ ID NO: 356)<br>
(Target sequences effective for mouse homolog) NM_021643-41,gggagatcgcggaacaaaa (SEQ ID NO: 357) NM_021643-382,gttctttgagcgaagctat (SEQ ID NO: 358) NM_021643-143,cccgagactccgaacttgt (SEQ ID NO: 359)<br>
(Target gene of RNAi)<br>
NM_007118, Homo sapiens triple functional domain (PTPRF interacting) (TRIO). (Target sequences)<br>
NM_007118-1684,caccaatgcggataaatta (SEQ	ID NO:	360)<br>
NM_007118-1686,ccaatgcggataaattact (SEQ	ID NO:	361)<br>
NM_007118-3857,gaaatctacgaatttcata (SEQ	ID NO:	362)<br>
NM_007118-6395,gagcagatcgtcatattca (SEQ	ID NO:	363)<br>
NM_007118-8531,cctatccgtagcattaaaa (SEQ	ID NO:	364)<br>
(Target gene of RNAi)<br><br>
NM_004938, Homo sapiens death-associated protein kinase 1 (DAPK1).<br>
(Target sequences)<br>
NM_004938-917,caatccgttcgcttgatat (SEQ ID NO: 365) NM_004938-1701,ggtgtttcgtcgattatca (SEQ ID NO: 366) NM_004938-1702,gtgtttcgtcgattatcaa (SEQ ID NO: 367) NM_004938-2824,gaaggtacttcgaaatcat (SEQ ID NO: 368) NM_004938-668,gaaacgttagcaaatgtat (SEQ ID NO: 369) (Target sequences effective for mouse homolog) NM_004938-609,gggtaataacctatatcct (SEQ ID NO: 370) NM_004938-2697,gaggcgagtttggatatga (SEQ ID NO: 371) NM_004938-490,ggcccataaaattgacttt (SEQ ID NO: 372) (Target gene of RNAi)<br>
NM_006252, Homo sapiens protein kinase, AMP-activated, alpha 2 catalytic subunit (PRKAA2). (Target sequences)<br>
NM_006252-760,gaaacgagcaactatcaaa (SEQ ID NO: 373) NM_006252-148,gaagattcgcagtttagat (SEQ ID NO: 374) NM_006252-1227,gcaaaccgtatgacc.ttat (SEQ ID NO: 375) NM_006252-1338,ctggcaattacgtga.aaat (SEQ ID NO: 376) NM_006252-1340,ggcaattacgtgaaaatga (SEQ ID NO: 377) (Target gene of RNAi)<br>
NM_002742, Homo sapiens protein kinase C, mu (PRKCM). (Target sequences)<br>
NM_002742-508iggtacgtcaaggtcttaaa (SEQ ID NO: 378) NM_002742-I332,gattggatagcaaatgtat (SEQ ID NO: 379) NM_002742-509,gtacgtcaaggtcttaaat (SEQ ID NO: 380) NM_002742-370,ggaaggcgatcttattgaa (SEQ ID NO: 381)<br><br>
(Target sequences effective for mouse homolog) NM_002742-1913,caccctggtgttgtaaatt (SEQ ID NO: 382) NM_002742-2041,cataacgaagtttt:taatt (SSQ ID NO: 383) NM_002742-2521,ctatcagacctggttagat (SEQ ID NO: 384) (Target gene of RNAi)<br>
NM_003684, Homo sapiens MAP kinase-interacting serine/threonine kinase 1 (MKNK1). (Target sequences)<br>
NM_003684-218,gagtatgccgtcaaaatca (SEQ ID NO: 385) NM_003684-229/caaaatcatcgagaaacaa (SEQ ID NO: 386) NM_003684-344,gatgacacaaggttttact (SEQ ID NO: 387) NM_003684-192,gtgccgtgagcctacagaa (SSQ ID NO: 388) NM__003684-379,gcaaggaggttccatctta (SEQ ID NO: 389) (Target gene of RNAi)<br>
NM_004759, Homo sapiens mitogen-activated protein kinase-activated protein kinase 2 (MAPKAPK2). (Target sequences)<br>
NM_004759-942/ccatcaccgagtttai:gaa (SEQ ID NO: 390) NM_004759-836,cgaatgggccagtatgaat (SEQ ID NO: 391) NM_004759-563,cctgagaatctcttataca (SEQ ID NO: 392) NM_004759-669,gttatacaccgtactatgt (SEQ ID NO: 393) NM_004759-362,gatgtgtacgagaatctgt (SEQ ID NO: 394) (Target gene of RNAi)<br>
NM__172171, Homo sapiens calcium/calmodulin-dependent protein kinase (CaM kinase) II gamma (CAMK2G).<br>
(Target sequences) NM_172171-113,gagtacgcagcaaaaatca (SEQ ID NO: 395)<br><br>
NM_172171-422,ctgctgctggcgagtaaat (SEQ ID NO: 396) NM_172171-1075,ggtacacaacgctacagat (SEQ ID NO: 397) NM_172171-474,gcctagccatcgaagtaca (SEQ ID NO: 398) (Target sequences effective for mouse homolog) NM_172171-425,ctgctggcgagtaaatgca (SEQ ID NO: 399) NM_172171-260,ctcgtgtttgaccti:gtta (SEQ ID NO: 400) NM_172171~597,gcggggtcatcctgt:atat (SEQ ID NO: 401) (Target gene of RNAi)<br>
NM__015981 , Homo sapiens calcium/calmodulin-dependent protein kinase (CaM kinase) II alpha (CAMK2A). (Target sequences)<br>
NM_015981-1213,ccatcgattctattttgaa (SEQ ID NO: 402) NM__015981-1210,cttccatcgattctatttt (SEQ ID NO: 403) NM_015981-1067,cggaaacaggaaattataa (SEQ ID NO: 404) NM__015981-1066,gcggaaacaggaaattata (SEQ ID NO: 405J NM_015981-754,gaccattaacccatccaaa (SEQ ID NO: 406) (Target sequences effective for mouse homolog) NM_015981-1130,gagtcctacacgaagatgt (SEQ ID NO: 407) NM_015981-1416/ggcagatcgtccacttcca (SEQ ID NO: 408) NM_015981-1418,cagatcgtccacttccaca (SEQ ID NO: 409) (Target gene of RNAi)<br>
NM_020439, Homo sapiens calcium/calmodulin-dependent protein kinase IG (CAMK1G). (Target sequences)<br>
NM_020439-1354/ggtcatggtaccagttaaa (SEQ ID' NO: 410) NM_020439-1409/ggagtctgtctcattatgt (SEQ ID NO: 411) NM_020439-639/gtggataccccccattcta (SEQ ID NO: 412) NM_020439-823,ctggattgacggaaacaca (SEQ ID NO: 413)<br><br>
NM_020439-662,gaaacggagtctaagcttt (SEQ ID NO: 414) (Target sequences effective for mouse homolog) NM_020439-85,gggatcaggagctttctca (SEQ ID NO: 415) NM_020439-903,gcaagtggaggcaagcctt (SEQ ID NO: 416) (Target gene of RNAi)<br>
NM_007194, Homo sapiens CHK2 checkpoint homolog (S. pombe) (CHEK2).<br>
(Target sequences)<br>
NM_007194-460,ctcttacattgcats.cata (SEQ ID NO: 417) NM_007194-201,ctcaggaactctattctat (SEQ ID NO: 418) NM_007194-1233,gtttaggagttattctttt (SEQ ID NO: 419) NM_007194-398,gataaataccgaacataca (SEQ ID NO: 420) NM_007194-396,cagataaataccgaacata (SEQ ID NO: 421) (Target sequences effective for mouse homolog) NM_007194-614,gtagatgatcagtcagttt (SEQ ID NO: 422) NM_007194-620,gatcagtcagtttatccta (SEQ ID NO: 423) NM_O07194-612,ctgtagatgatcagtcagt (SEQ ID NO: 424) (Target gene of RNAi)<br>
NM_002610, Homo sapiens pyruvate dehydrogenase kinase, isoenzyme 1 (PDK1). (Target sequences)<br>
NM_002610-1194,gactcccagtgtataacaa (SEQ ID NO: 425) NM_002610-553,catgagtcgcatttce.att (SEQ ID NO: 426) NM_002610-306,ggacaccatccgttca.att (SEQ ID NO: 427) NM_Q02610-1086#gtctttacgcacaatactt (SEQ ID NO: 428) NM_002610-388/ggatgctaaagctatttat (SEQ ID NO: 429) (Target gene of RNAi) NM_001619, Homo sapiens adrenergic, beta, receptor kinase 1<br><br>
(ADRBK1).<br>
(Target sequences)<br>
NM_001619-474,gggacgtgttccagaaatt (SEQ ID NO: 430)<br>
NM_001619~317,gagatcttcgactcataca (SEQ ID NO: 431)<br>
NM_001619-665,gacaaaaagcgcatcaaga (SEQ ID NO: 432)<br>
NM_001619-439,gccatacatcgaagagatt (SEQ ID NO: 433)<br>
NM_001619-476,gacgtgttccagaaattca (SEQ ID NO: 434)<br>
(Target sequences effective for mouse homolog)<br>
NM_001619-1476,caaaaggaatcaagttact (SEQ ID NO: 435)<br>
NM_001619-1474,cacaaaaggaatcaagtta (SEQ ID NO: 436)<br>
NM_001619-1171,ccggcagcacaagaccaaa (SEQ ID NO: 437)<br>
(Target gene of RNAi)<br>
NM_005160, Homo sapiens adrenergic, beta, receptor kinase<br>
(ADRBK2).<br>
(Target sequences)<br>
NM_005160-1779,gagagtcccggcaaaattt (SEQ ID NO: 438)<br>
NM_005160-1778,ggagagtcccggcaaaatt (SEQ ID NO: 439)<br>
NM_005160-1373,cagcatgtctacttacaaa (SEQ ID NO: 440)<br>
NM_005160-307,cagaagtcgacaaatttat (SEQ ID NO: 441)<br>
NM__005160-306,gcagaagtcgacaaattta (SEQ ID NO: 442)<br>
(Target gene of RNAi)<br>
NM_003161, Homo sapiens ribosomal protein S6 kinase, 70kDa,<br>
polypeptide 1 (RPS6KB1).<br>
(Target sequences)<br>
NM„003161-1294,ccgatcacctcgaagattt (SEQ ID NO: 443)<br>
NM_003161-1556,cacctgcgtatgaatctat (SEQ ID NO: 444)<br>
NM_003161-1296,gatcacctcgaagatttat (SEQ ID NO: 445)<br>
NM_003161-831,gtttgggagcattaatgta (SEQ ID NO: 446)<br><br>
NM_003161-1295,cgatcacctcgaagattta (SEQ ID NO: 447) (Target gene of RNAi)<br>
NM_014496# Homo sapiens ribosomal protein S6 kinase, 90kDa, polypeptide 6 (RPS6KA6). (Target sequences)<br>
NM_014496-682,gaaggcttactcattttgt (SEQ ID NO: 448) NM_014496-1552,ggaggctagtgatatacta (SEQ ID NO: 449) NM_014496-1553,gaggctagtgatatactat (SEQ ID NO: 450) NM_014496-1551,gggaggctagtgatatact (SEQ ID NO: 451) NM_014496-1481,cttgttacggatttaatga (SEQ ID NO: 452) (Target sequences effective for mouse homolog) NM_014496-831,gaaatgagaccatgaatat (SEQ ID NO: 453)<br>
NM_014496-1411,gatgcgctatggacaacat (SEQ ID NO: 454) NM_014496-927,ggaatccagcaaatagatt (SEQ ID NO: 455) (Target gene of RNAi)<br>
NM_002953, Homo sapiens ribosomal protein S6 kinase, 9 0kDa, polypeptide 1 (RPS6KA1). (Target sequences)<br>
NM_002953-739,ctatggggtgttgatgttt (SEQ ID NO: 456) NM_002953-1331,gctgtcaaggtcattgata (SEQ ID NO: 457) NM_002953-1332,ctgtcaaggtcattgataa (SEQ ID NO: 458) NM_002953-735,ggtcctatggggtgttgat (SEQ ID NO: 459) NM_002953-738,cctatggggtgttgatgtt (SEQ ID NO: 460) (Target sequences effective for -mouse homolog) NM_002953-666,gcgggacagtggagtacat (SEQ ID NO: 461) NM_002953-832,gctaggcatgccccagttt (SEQ ID NO: 462) NM_002953-1315,caccaacatggagtatgct (SEQ ID NO: 463)<br><br>
(Target gene of RNAi)<br>
NM__001626, Homo sapiens v-akt murine thymoma viral oncogene homolog 2 (AKT2). (Target sequences)<br>
NM_001626-141,ctctaccccccttaaacaa (SEQ ID NO: 464) NM_001626-35,cacaagcgtggtgaataca (SEQ ID NO: 465) NM_001626-143,ctaccccccttaaacaact (SEQ ID NO: 466) NM_001626-41,cgtggtgaatacatcaaga (SEQ ID NO: 467) NM_001626-420,gcaaggcacgggctaaagt (SEQ ID NO: 468) (Target gene of RNAi)<br>
NM_005163, Homo sapiens v-akt murine thymoma viral oncogene homolog 1 (AKT1). (Target sequences)<br>
NM_005163-1294,gactgacaccaggtatttt (SEQ ID NO: 469) NM_005163-1296,ctgacaccaggtattttga (SEQ ID NO: 470) NM_005163-1292,gagactgacaccaggtatt (SEQ ID NO: 471) NM_005163-751,cttctatggcgctgagatt (SEQ ID NO: 472) NM_005163-630,cagccctgaagtactcttt (SEQ ID NO: 473) (Target gene of RNAi)<br>
NM_005465, Homo sapiens v-akt murine thymoma viral oncogene homolog 3 (protein kinase B, gamma) (AKT3). (Target sequences)<br>
NM_005465-229,ccagtggactactgttata (SEQ ID NO: 474) NM_005465-99,cattcataggatataaaga (SEQ ID NO: 475) NM_005465-402,cctctacaacccatcataa (SEQ ID NO: 476) NM_005465-1283,gagacagatactagatatt (SEQ ID NO: 477) (Target sequences effective for mouse homolog) NM__005465-733,ggaccgcacacgtttctat (SEQ ID NO: 478)<br><br>
NM_005465-1317,cagctcagactattacaat (SSQ ID NO: 479)<br>
NM_005465-1319,gctcagactattacaataa (SZQ ID NO: 480) (Target gene of RNAi)<br>
NM__005627, Homo sapiens serum/glucocorticoid regulated kinase (SGK). (Target sequences)<br>
NM__005627-875,ggcctgccgcctttttata (SEQ ID NO: 481) NM_005627-97,gggtctgaacgactttatt (SEQ ID NO: 482) NM__005627-99,gtctgaacgactttattca (SEQ ID NO: 483) NM_005627-190,ggagcctgagcttatgaat (SEQ ID NO: 484) NM_005627-413/gaggagaagcatattatgt (SEQ ID NO: 485) (Target sequences effective for mouse homolog) NM_005627-649,catcgtttatagagactta (SEQ ID NO: 486) NM_005627-367,ctatgcagtcaaagtttta (SEQ ID NO: 487) NM_005627-307,gatcggaaagggcagtttt (SEQ ID NO: 488) (Target gene of RNAi)<br>
NM_170693, Homo sapiens serum/glucocorticoid regulated kinase 2 (SGK2). (Target sequences)<br>
NM_170693-163,gtctgatggggcgttctat (SEQ ID NO: 489) NM_170693-840,cagactttcttgagattaa (SEQ ID NO: 490) NM_170693-842,gactttcttgagattaaga (SEQ ID NO: 491) NM_170693-582,gtggtacccctgagtactt (SEQ ID NO: 492) NM_170693-183,cagtgaaggtactacagaa (SEQ ID NO: 493) (Target sequence effective for mouse homolog) NM_170693-287,gtgggcctgcgctactcct (SEQ ID NO: 494) (Target gene of RNAi) NM_013257/ Homo sapiens serum/glucocorticoid regulated<br><br>
kinase-like (SGKL). (Target sequences)<br>
NM__013257-273,caggactaaacgaattcat (SEQ ID NO: 495) NM__013257-944,gacaccactaccacatttt (SEQ ID NO: 496) NM_013257-1388,gtatcttctgactattcta (SEQ ID NO: 497) NM_013257-946,caccactaccacattttgt (SEQ ID NO: 498) NM_013257-790,gttttacgctgctgaaatt (SEQ ID NO: 499) (Target sequences effective for mouse homolog) NM_013257-693,caactgaaaagctttattt (SEQ ID NO: 500) NM_013257-225,gaatatttggtgataattt (SEQ ID NO: 501) NM„013257-38,ccaagtgtaagcattccca (SEQ ID NO: 502) (Target gene of RNAi)<br>
NM_002744, Homo sapiens protein kinase C, zeta (PRKCZ). (Target sequences)<br>
NM__002744-1233,gcggaaccccgaattacat (SEQ ID .NO:.. 503) NM__002744-398,caagccaagcgctttaaca (SEQ ID NO: 504) NM_002744-1447,caaagcctcccatgtttta (SEQ ID NO: 505) NM_O02744-823,ccaaatttacgccatgaaa (SEQ ID NO: 506) NM_002744-1100,cacgagagggggatcatct (SEQ ID NO: 507) (Target gene of RNAi)<br>
NM_006254, Homo sapiens protein kinase C, delta (PRKCD). (Target sequences)<br>
NM_006254-1524,gcggcacccctgactatat (SEQ ID NO: 508) NM_006254-1339,ctaccgtgccacgttttat (SEQ ID NO: 509) NM_006254-992/gggacctacggcaagatct (SEQ ID NO: 510) (Target sequences effective for mouse homolog) NM_006254-172/gttcgacgcccacatctat (SEQ ID NO: 511) NM_006254-659/cagaaagaacgcttcaaca (SEQ ID NO: 512)<br><br>
NM_006254-761,gtgaagcagggattaaagt (SEQ ID NO: 513) (Target gene of RNAi)<br>
NM_002737, Homo sapiens protein kinase C, alpha (PRKCA). (Target sequences)<br>
NM_002737-1571,ggcgtcctgttgtatgaaa (SEQ ID NO: 514) NM_002737-393,gtgacacctgcgatatgaa (SEQ ID NO: 515) NM_002737-711,gacgactgtctgtagaaat (SEQ ID NO: 516) NM_002737-1085/gaactgtatgcaatcaaaa (SEQ ID NO: 517) NM_002737-1924,gctggttattgctaacata (SEQ ID NO: 518) (Target sequences effective for mouse homolog) NM_002737-1958,gaagggttctcgtatgtca (SEQ ID NO: 519) NM_002737-1835,ccattcaagcccaaagtgt (SEQ ID NO: 520) NM_002737-1234,gctgtacttcgtcc.tggaa (SEQ ID NO: 521) (Target gene of RNAi)<br>
NM_002738, Homo sapiens protein kinase _C_, beta 1 (PRKCB1) . (Target sequences)<br>
NM_002738-573,cagatccctacgtaaaact (SEQ ID NO: 522) NM_002738-1791,catttttccggtatattga (SEQ ID NO: 523) NM_002738-1384,catttaccgtgacctaaaa (SEQ ID NO: 524) NM_002738-575,gatccctacgtaaaactga (SEQ ID NO: 525) NM_002738-1315,ggagccccatgctgtattt (SEQ ID NO: 526) (Target sequences effective for mouse homolog) NM_002738-1006,gatgaaactgaccgatttt (SEQ ID NO: 527) NM_002738-1961/gaattcgaaggattttcct (SEQ ID NO: 528) NM_002738-1233,ccatggaccgcctgtactt (SEQ ID NO: 529) (Target gene of RNAi)<br>
NM_015282, Homo sapiens cytoplasmic linker associated protein 1 (CLASP1).<br><br>
(Target sequences)<br>
NM_015282-2447,gagccgtatgggatgtatt (SEQ ID NO: 530) NM__015282-4151, gccgagctgacgattatga (SEQ ID NO: 531) NM_015282-4152,ccgagctgacgattatgaa (SEQ ID NO: 532) NM_015282-1786,gcgatctcgaagtgatatt (SEQ ID NO: 533) NM_01528"2-635,cagtcccggttgaatgtaa (SEQ ID NO: 534) (Target gene of RNAi)<br>
NM__006287, Homo sapiens tissue factor pathway inhibitor (lipoprotein-associated coagulation inhibitor) (TFPI). (Target sequences)<br>
NM-006287-225,ctcgacagtgcgaagaatt (SEQ ID NO: 535) NM_006287-227,cgacagtgcgaagaattta (SEQ ID NO: 536) NM_006287-228,gacagtgcgaagaatttat (SEQ ID NO: 537) NM_006287-230,cagtgcgaagaatttatat (SEQ ID NO: 538) NM_006287~393,gaatatgtcgaggttatat (SEQ ID NO: 539) (Target gene of RNAi)<br>
NM_004073, Homo sapiens cytokine-inducible kinase (CNK). (Target sequences)<br>
NM_J)04073-1283,gttgactactccaataagt (SEQ ID NO: 540) NM_J304073-138,gcgcctacgctgtcaaagt (SEQ ID NO: 541) NM__004073-239,cgccacatcgtgcgttttt (SEQ ID NO: 542) NM_004073-1281,gggttgactactccaataa (SEQ ID NO: 543) (Target sequences effective for mouse homolog) NM_004073-192,gcgagaagatcctaaatga (SEQ ID NO: 544) NM_004073-183,cgcatcagcgcgagaag_at (SEQ ID NO: 545)<br>
NM_004073-190,gcgcgagaagatcctaaat (SEQ ID NO: 546) (Target gene of RNAi)<br><br>
NM_003384, Homo sapiens vaccinia related kinase 1 (VRK1). (Target sequences)<br>
NM_003384-776,ccttgggaggataatttga (SEQ ID NO: 547) NM_003384-773#cttccttgggaggataatt (SEQ ID NO: 548) NM_003384-195,caccttgtgttgtaaaagt (SEQ ID NO: 549)<br>
NM_003384-777,cttgggaggataatttgaa (SEQ ID NO: 550) (Target sequences effective for mouse homolog) NM_003384-372,gttacaggtttatgataat (SEQ ID NO: 551) NM_003384-463,gcagctaagcttaagaatt (SEQ ID NO: 552) NM_003384-977,ggactaaaagctataggaa (SEQ ID NO: 553) (Target gene of RNAi)<br>
NM_006296, Homo sapiens vaccinia related kinase 2 (VRK2). (Target sequences)<br>
NM_006296-366,gactaggaatagatttaca (SEQ ID NO: 554) NM_006296-165,caagacatgtagtaaaagt (SEQ ID NO: 555) NM_006296-874,ggtatgtgctcatagttta (SEQ ID NO: 556) NM_006296-541.ggtttatcttgcagattat (SEQ ID NO: 557) NM_006296-113,ggatttggattgatat:att (SEQ ID NO: 558) (Target sequences effective for mouse homolog) NM_006296-560,ggactttcctacagatatt (SEQ ID NO: 559) NM_006296-626,cataatgggacaatagagt (SEQ ID NO: 560) NM_006296-568,ctacagatattgtcccaat (SEQ ID NO: 561) (Target gene of RNAi)<br>
NM_004672, Homo sapiens mitogen-activated protein kinase kinase kinase 6 (MAP3K6). (Target sequences) NM_004672-2221,ctttctcctccgaactttt (SEQ ID NO: 562)<br><br>
NM_004672-1489,gatgttggagtttgattat (SEQ ID NO: 563) NM_004672-814,caaagagctccggctaata (SEQ ID NO: 564) NM_004672-51,ccctgcgggaggatgtttt (SEQ ID NO: 565) NM__004672-503,gccgagcagcataatgtct (SEQ ID NO: 566) (Target sequences effective for mouse homolog) NM„004672-442,ggactactcggccatcatt (SEQ ID NO: 567) NM_004672-277fctatttccgggagaccatt (SEQ ID NO: 568) NM_004672-1929,ggctgctcaagatttctga (SEQ ID NO: 569) (Target gene of RNAi)<br>
NM_005923, Homo sapiens mitogen-activated protein kinase kinase kinase 5 (MAP3K5). (Target sequences)<br>
NM_005923-3294,gatccactgaccgaaaaat (SEQ ID NO: 570) NM_005923-838,caggaaagctcgtaattta (SEQ ID NO; 571) NM_005923-840/ggaaagctcgtaatttata (SEQ ID.NO: 57_2) NM_005923-1525,gtacctcaagtctattgta (SEQ ID NO: 573) NM_005923-2517,ctggtaccctccagtatat (SEQ ID NO: 574) (Target gene of RNAi)<br>
NM_020998, Homo sapiens macrophage stimulating 1 (hepatocyte growth factor-like) (MST1). (Target sequences)<br>
NM_020998-943,ccgatttacgccagaaaaa (SEQ ID NO: 575) NM_020998-944,cgatttacgccagaaaaat (SEQ ID NO: 576) NM_020998-945,gatttacgccagaaaaata (SEQ ID NO: 577) NM_020998-698fggtctggacgacaactatt (SEQ ID NO: 578) NM_020998-1827/ccaaaggtacgggtaatga (SEQ ID NO: 579) (Target gene of RNAi) NM_003576, Homo sapiens serine/threonine kinase 24 (STE20<br><br>
homolog, yeast) (STK24).<br>
(Target sequences)<br>
NM_003576-348,gctccgcactagatctatt (SEQ ID NO: 580)<br>
NM_003576-349,ctccgcactagatctatta (SEQ ID NO: 581)<br>
NM_003576-351,ccgcactagatctattaga (SEQ ID NO: 582)<br>
NM_003576-352#cgcactagatctattagaa (SEQ ID NO: 583)<br>
NM__003576-437,ctccattcggagaagaaaa (SEQ ID NO: 584)<br>
(Target sequence effective for mouse homolog)<br>
NM_003576-148,gttcaaaggcattgacaat (SEQ ID NO:   585)<br>
(Target gene of RNAi)<br>
NM_016542, Homo sapiens Mst3 and SOKl-related kinase (MST4).<br>
(Target sequences)<br>
NM__016542-857,ctgatagatcgttttaaga' (SEQ ID NO: 586)<br>
NM_016542-139,gcaagtcgttgctattaaa (SEQ ID NO: 587)<br>
NM__016542-1133,gaagaactcgagaaaagta (SEQ ID NO: 588)<br>
NM_016542-556,ggctcctgaagttattcaa (SEQ ID NO: 589)<br>
(Target sequences effective for mouse homolog)<br>
NM_016542-613,gggaattactgctattgaa (SEQ ID NO: 590)<br>
NM_016542-669,caatgagagttctgtttct (SEQ ID NO: 591)<br>
NM__016542-1063,gataatcacacctgcattt (SEQ ID NO: 592)<br>
(Target gene of RNAi)<br>
NM_002576, Homo sapiens p21/Cdc42/Racl-activated kinase 1<br>
(STE20 homolog, yeast) (PAK1).<br>
(Target sequences)<br>
NM_002576-38#gcccctccgatgagaaata (SEQ ID NO: 593)<br>
NM_002576-788,ggcgatcctaagaagaaat (SEQ ID NO: 594)<br>
NM_002576-3,caaataacggcctagacat (SEQ ID NO: 595)<br>
NM_002576-154,ccgattttaccgatccatt (SEQ ID NO: 596)<br><br>
(Target sequences effective for mouse homolog)<br>
NM_002576-1020,gggttgttatggaatactt (SEQ ID NO: 597)<br>
NM_002576-1165,catcaagagtgacaatatt (SEQ ID NO: 598)<br>
NM_002576-1015,gctgtgggttgttatggaa (SEQ ID NO: 599)<br>
(Target gene of RNAi)<br>
NM_002577, Homo sapiens p21 (CDKN1A)-activated kinase 2<br>
(PAK2).<br>
(Target sequences)<br>
NM_002577-721,cataggtgaccctaagaaa (SEQ ID NO: 600)<br>
NM_002577-908,cccaacatcgttaactttt (SEQ ID NO: 601)<br>
NM__002577-909,ccaacatcgttaacttttt (SEQ ID NO: 602)<br>
NM_002577-557,ccggatcatacgaaatcaa (SEQ ID NO: 603)<br>
NM__002577-558,cggatcatacgaaatcaat (SEQ ID NO: 604)<br>
(Target gene of RNAi)<br>
NM_002578/ Homo sapiens p21 (CDKN1A)-activated kinase 3<br>
(PAK3).<br>
(Target sequences)<br>
NM__002578-458,catccttcgagtacaaaaa (SEQ ID NO: 605)<br>
NM_002578-1467,ctgtattccgtgacttttt (SEQ ID NO: 606)<br>
NM_002578-1469,gtattccgtgacttt:ttaa (SEQ ID NO: 607)<br>
NM_002578-706,cacagatcggcaaagaaaa (SEQ ID NO: 608)<br>
NM_002578-3,ctgacggtctggataatga (SEQ ID NO: 609)<br>
(Target sequences effective for mouse homolog)<br>
NM_002578-1376,cccccttaccttaatgaaa (SEQ ID NO: 610)<br>
NM^002578-219/cagactttgagcatacgat (SEQ ID NO: 611)<br>
NM_002578-254/gcagtcaccggggaattca (SEQ ID NO: 612)<br>
(Target gene of RNAi)<br>
NM__005884, Homo sapiens p21(CDKN!A)-activated kinase 4<br><br>
(PAK4) .<br>
(Target sequences)<br>
NM_005884-1502fgggataatggtgattgaga (SEQ ID NO: 613) NM_005884-I503,ggataatggtgat1:gagat (SEQ ID NO: 614) NM_005884-883,gccacagcgagtatcccat (SEQ ID NO: 615) NM_005884-77,cagcacgagcagaagttca (SEQ ID NO: 616) NM_005884-1494,ggtcgctggggataatggt (SEQ ID NO: 617) (Target gene of RNAi)<br>
NM_002755, Homo sapiens mitogen-activated protein kinase kinase 1 (MAP2K1). (Target sequences)<br>
NM__002755-280,ggccagaaagctaattcat (SEQ ID NO: 618) NM_002755-402,gcgatggcgagatcagtat (SEQ ID NO: 619) NM_002755-404,gatggcgagatcagtatct (SEQ ID NO: 620) NM_002755-682,ctacatgtcgccagaaaga (SEQ ID NO: 621) NM__002755-1128,ccaccatcggccttaacca (SEQ ID NO: 622) (Target sequences effective for mouse homolog) NM_002755-912,gacctcccatggcaatttt (SEQ ID NO: 623) NM_002755-915,ctcccatggcaatttttga (SEQ ID NO: 624) NM_002755-911,cgacctcccatggcaattt (SEQ ID NO: 625) (Target gene of RNAi)<br>
NM_030662, Homo sapiens mitogen-activated protein kinase kinase 2 (MAP2K2). (Target sequences)<br>
NM_030662-1136,gccggctggttgtgtaaaa (SEQ ID NO: 62.6) NMJ)30662-I84,caaggtcggcgaactcaaa (SEQ ID NO: 627) NM_030662-959,ctcctggactatattgtga (SEQ ID NO: 628) NM_030662-183/ccaaggtcggcgaactcaa (SEQ ID NO: 629)<br><br>
NM_030662-711,ggttgcagggcacacatta (SEQ ID NO: 630) (Target gene of RNAi)<br>
NM_002756, Homo sapiens mitcgen-activated protein kinase kinase 3 (MAP2K3). (Target sequences)<br>
NM_002756-257,cgcacggtcgactgtttct (SEQ ID NO: 631) NM_002756-258,gcacggtcgactgtttcta (SEQ ID NO: 632) NM_002756-289.ctacggggcactattcaga (SEQ ID NO: 633) NM_002756-285,ccttctacggggcactatt (SEQ ID NO: 634) NM_002756-44,gactcccggaccttcatca (SEQ ID NO: 635) (Target sequences effective for mouse homolog) NM_002756-129»gagcctatggggtggtaga (SEQ ID NO: 636) NM_002756-41,ctggactcccggaccttca (SEQ ID NO: 637) NM_002756-89,gaggctgatgacttggtga (SEQ ID NO: 638) (Target gene of RNAi)<br>
NM_002758# Homo sapiens mitogen-activated protein kinase kinase 6 (MAP2K6). (Target sequences)<br>
NM_002758-394,ggatacatcactagataaa (SEQ ID NO: 639) NM_002758-395,gatacatcactagataaat (SEQ ID NO: 640) NM_002758-755,cttcgatttccctatgatt (SEQ ID NO: 641) NM_002758-340,cttttatggcgcactgttt (SEQ ID NO: 642) NM_002758-399,catcactagataaattcta (SEQ ID NO: 643) (Target sequences effective for mouse homolog) NM_002758-312,ggacggtggactgtccratt (SEQ ID NO: 644) NM_002758-418,caaacaagttattgataaa (SEQ ID NO: 645) NM_002758-415.ctacaaacaagttattgat (SEQ ID NO: 646) (Target gene of RNAi)<br><br>
NM_003010, Homo sapiens mitogen-activated protein kinase kinase 4 (MAP2K4). (Target sequences)<br>
NM__003010-543,ctacctcgtttgataagtt (SEQ ID NO: 647) NM_003010-1130,gcatgctatgtttgtaaaa (SEQ ID NO: 648) NM_003010-1056,ccaaaaggccaaagtataa (SEQ ID NO: 649) (Target sequences effective for mouse homolog) NM_003010-1129,cgcatgctatgtttgtaaa (SEQ ID NO: 650) NM_003010-1057,caaaaggccaaagtataaa (SEQ ID NO: 651) NM_003010-452,gtaatgcggagtagtgatt (SEQ ID NO: 652) (Target gene of RNAi)<br>
NM_016123, Homo sapiens interleukin-1 receptor-associated kinase 4 (IRAK4). (Target sequences) NM__016123-1299,gccaatgtcggcatgaaaa (SEQ ID NO: 653)<br>
NM_016123-1073,gctttgcgtggagaaataa (SEQ ID NO: 654) NM_016123-38,ctcaatgttggactaatta (SEQ ID NO: 655) NM_016123-769,cctctgcttagtatatgtt (SEQ ID NO: 656) NM_016123-1180#gttattgctagatat:taaa (SEQ ID NO: 657) (Target gene of RNAi)<br>
NM_002880, Homo sapiens v-raf-1 murine leukemia viral oncogene homolog 1 (RAF1). (Target sequences)<br>
NM_002880-1703,gatcttagtaagctatata (SEQ ID NO: 658) NM_0Q2880-232,gcatgactgccttatgaaa (SEQ ID NO: 659) NM_002880-1597,ctatggcatcgtattgtat (SEQ ID NO: 660) NM__002880-1706,cttagtaagctatataaga (SEQ ID NO: 661)<br><br>
NM_002880-568,cagacaactcttattgttt (SEQ ID NO: 662)<br>
(Target gene of RNAi)<br>
NM_000020, Homo sapiens activin A receptor type II-like 1<br>
(ACVRL1).<br>
(Target sequences)<br>
NM_000020-1453,caagaagacactacaaaaa (SEQ ID NO: 663)<br>
NM_000020-722,gagactgagatctataaca (SEQ ID NO: 664)<br>
NM_000020-1456,gaagacactacaaaaaatt (SEQ ID NO: 665)<br>
NM„000020-728,gagatctataacacagtat (SEQ ID NO: 666)<br>
NM_000020-846,gctccctctacgactttct (SEQ ID NO: 667)<br>
(Target gene of RNAi)<br>
NM__001105, Homo sapiens activin A receptor, type I (ACVR1).<br>
(Target sequences)<br>
NM_001105-1456,cacagcactgcgtatcaaa (SEQ ID NO: 668)<br>
NM_001105-428,gttgctctccgaaaattta (SEQ ID NO: 669)<br>
NM_001105-431,gctctccgaaaatttaaaa (SEQ ID NO: 670)<br>
NM_001105-1460,gcactgcgtatcaaaaaga (SEQ ID NO: 671)<br>
NM__001105-1458,cagcactgcgtatcaaaaa (SEQ ID NO: 672)<br>
(Target sequences effective for mouse homolog)<br>
NM_001105-1306,caatgacccaagttttgaa (SEQ ID NO: 673)<br>
NM_001105-1381,gttctcagacccgacatta (SEQ ID NO: 674)<br>
NM_001105-281,caaggggactggtgtaaca (SEQ ID NO: 675)<br>
(Target gene of RNAi)<br>
NM_004302, Homo sapiens activin A receptor, type IB<br>
(ACVR1B).<br>
(Target sequences)<br>
NM_004302-609,cccgaaccatcgttttaca (SEQ ID NO: 676)<br>
NM_004302-610,ccgaaccatcgttttacaa (SEQ ID NO: 677)<br><br>
NM_004302-897,caattgaggggatgeattaa (SEQ ID NO: 678) NM_004302-857,cacgggtccctgttt:gatt (SEQ ID NO: 679) NM_004302-859,cgggtccctgtttgattat (SEQ ID NO: 680) (Target sequences effective for mouse homolog) NM_004302-1119,gggtggggaccaaacgata (SEQ ID NO: 681) NM_004302-1063,cctggctgtccgtcatgat (SEQ ID NO: 682) NM_004302-1121,gtggggaccaaacgataca (SEQ ID NO: 683) (Target gene of RNAi)<br>
NM_145259, Homo sapiens activin A receptor, type IC (ACVR1C).<br>
(Target sequences)<br>
NM_145259-1419,ctgctCttcgtattaagaa (SEQ ID NO: 684) NM_145259-956,gctcatcgagacataaaat (SEQ ID NO: 685) NM_145259-825,gctccttatatgactattt (SEQ ID NO: 686) NM_145259-959,catcgagacataaaatcaa (SEQ ID NO: 687)_ NM_145259-1237,gtaccaattgccttattat (SEQ ID NO: 688) (Target gene of RNAi)<br>
NM_004612, Homo sapiens transforming growth factor, beta receptor I (activin A receptor type II-like kinase, 53kDa) (TGFBR1).<br>
(Target sequences)<br>
NM_004612-236,cgagataggccgtttgtat (SEQ ID NO: 689) NM_004612-1451,gcattgcggattaagaaaa (SEQ ID NO: 690) NM_004612-463,ccatcgagtgccaaatgaa (SEQ ID NO: 691) NM_004612-492,cattagatcgcccttt:tat (SEQ ID NO: 692) NM_004612-1449,cagcattgcggattaagaa (SEQ ID NO: 693) (Target sequences effective for mouse homolog) NM_004612-829,gttggtgtcagattatcat (SEQ ID NO: 694)<br><br>
NM_004612-288,caacatattgctgcaatca (SEQ ID NO: 695) NM_004612-839,gattatcatgagcatggat (SEQ ID NO: 696) (Target gene of RNAi)<br>
NM_004836# Homo sapiens eukaryotic translation initiation factor 2-alpha kinase 3 (EIF2AK3). (Target sequences)<br>
NM_004836-1594,catagcaacaacgtttatt (SEQ ID NO: 697) NM_004836-1419,catatgataatggttatta (SEQ ID NO: 698) NM_004836-1900,ggtaatgcgagaagttaaa (SEQ ID NO: 699) NM_004836-1248,ctaatgaaaacgcaattat (SEQ ID NO: 700) (Target sequences effective for mouse homolog) NM_004836-784,ctttgaacttcggtatatt (SEQ ID NO: 701) NM_004836-782,cactttgaacttcggtata (SEQ ID NO: 702) NM_004836-983,gaatgggagtaccagtttt (SEQ ID NO: 703) (Target gene of RNAi)<br>
NM_001433, Homo sapiens ER to nucleus signalling 1 (ERN1) (Target sequences)<br>
NM_001433-2407,cattgcacgagaattgata (SEQ ID NO: 704) NM_001433-2277,caggctgcgtcttttacta (SEQ ID NO: 705) NM_001433-2530,cgtgagcgacagaatagaa (SEQ ID NO: 706) NM_001433-1149,ccaaacatcgggaaaatgt (SEQ ID NO: 707) NM_001433-364,ggacatctggtatgttatt (SEQ ID NO: 708) (Target sequences effective for mouse homolog) NM_001433-319,cccatgccgaagttcagat (SEQ ID NO: 709) NM_001433-2254,ctacacggtggacatcttt (SEQ ID NO: 710) NM_001433-324,gccgaagttcagatggaat (SEQ ID NO: 711) (Target gene of RNAi) NM_001278, Homo sapiens conserved helix-loop-helix<br><br>
ubiquitous kinase (CHUK.) . (Target sequences)<br>
NM_001278-746,ggagaagttcggtttagta (SEQ ID NO: 712) NM_001278-1879,ggccctcagtaatatcaaa (SEQ ID NO: 713) 5  NM_001278-864,gacctgttgaccttacttt (SEQ ID NO: 714) NM_001278-2150,ggccatttaagcactatta (SEQ ID NO: 715) NM_001278-2151,gccatttaagcactattat (SEQ ID NO: 716) (Target sequences effective for mouse homolog) NM_001278-645,ctggatataggcctttttt (SEQ ID NO: 717)<br>
10 NM_001278-1354,gttaagtcttcttagatat (SEQ ID NO: 718) NM_001278-1203,gtttatctgattgtgtaaa (SEQ ID NO: 719) (Target gene of RNAi)<br>
NM_014002, Homo sapiens inhibitor of kappa light polypeptide gene enhancer in B-cells, kinase epsilon<br>
15  (IKBKE).<br>
(Target sequences)<br>
NM_014002-2107,catcgaacggctaaataga (SEQ ID NO: 720) NM_014002-1724,ctggataaggtgaatttca (SEQ ID NO: 721) NM_014002-535,cctgcatcccgacatgtat (SEQ ID NO: 722)<br>
20  NM_014002-1220,ctgcaggcggattacaaca (SEQ ID NO: 723)<br>
w	NM_014002-1726,ggataaggtgaatttcagt (SEQ ID NO: 724)<br>
(Target sequence effective for mouse homolog) NM_014002-54,ccactgccagtgtgtacaa (SEQ ID NO: 725) (Target gene of RNAi)<br>
25  NML003177, Homo sapiens spleeji tyrosine kinase (SYK) . (Target sequences) NM_003177-1222,caatgaccccgctcttaaa (SEQ ID NO: 726)<br>
NM_003177-713,cagctagtcgagcattatt (SEQ ID NO: 727)<br>
- 141 -<br><br>
NM_003177-849,ggtcagcgggtggaataat (SEQ ID NO: 728) NM_003177-715,gctagtcgagcattattct (SEQ ID NO: 729) NM_003177-1389,gacatgtcaaggataagaa (SEQ ID NO: 730) (Target sequences effective for mouse homolog) NM_003177-1559,gctgatgaaaactactaca (SEQ ID NO: 731) NM_003177-1028,gacacagaggtgtacgaga (SEQ ID NO: 732) NM_003177-1560,ctgatgaaaactactacaa (SEQ ID NO: 733) (Target gene of RNAi)<br>
NM_153831, Homo sapiens PTK2 protein tyrosine kinase 2 (PTK2).<br>
(Target sequences)<br>
NM_153831-451,gaagagcgattatatgtta (SEQ ID NO: 734) NM_153831-1889,gtaatcggtcgaattgaaa (SEQ ID NO: 735) NM_153831-93,caatggagcgagtattaaa (SEQ ID NO: 736) NM_153831-2747,ctggaccggtcgaatgata (SEQJED NO: 737) NM_153831-92,gcaatggagcgagtattaa (SEQ ID NO: 738) (Target sequences effective for mouse homolog) NM_153831-1767,ctccagagtcaatcs.attt (SEQ ID NO: 739) NM_153831-1766,gctccagagtcaatcaatt (SEQ ID NO: 740) NM_153831-599,gttggtttaaagcgatttt (SEQ ID NO: 741) (Target gene of RNAi)<br>
NM_173174, Homo sapiens PTK2B protein tyrosine kinase 2 beta (PTK2B). (Target sequences)<br>
NM_173174-1273,ggtcctgaatcgtattctt (SEQ ID NO: 742) NM_173174-1776,ccccagagtccattaactt (SEQ ID NO: 743) NM_173174-1723,ggacgaggactattacaaa (SEQ ID NO: 744) NM_173174-2486,gaccccatggtttatatga (SEQ ID NO: 745)<br><br>
(Target sequences effective for mouse homolog) NM_173174-378,ggaggtatgaccttcaaat (SEQ ID NO: 746) NM_173174-1182,gcagcatagagtcagacat (SEQ ID NO: 747) NM_173174-376,gtggaggtatgaccttcaa (SEQ ID NO: 748) (Target gene of RNAi)<br>
NM_002944, Homo sapiens v-ros UR2 sarcoma virus oncogene homolog 1 (avian) (ROS1). (Target sequences) NM_002944-417,gaagctggacttatactaa (SEQ ID NO: 749)<br>
NM_002944-2123,gacatggattggtataaca (SEQ ID NO:	750)<br>
NM_002944-2163,cgaaaggcgacgtttttgt (SEQ ID NO:	751)<br>
NM_002944-1385,caagccaagcgaatcattt (SEQ ID NO:	752)<br>
NM_002944-416,ggaagctggacttatacta (SEQ ID NO: 753)<br>
(Target sequences effective for mouse homolog)<br>
NM_002944-3048,ctgtcactccttataccta (SEQ ID NO:	754)<br>
NM_002944-3044,ctttctgtcactccttata (SEQ ID NO:	755)<br>
NM_002944-1051,caacatgtctgatgtatct (SEQ ID NO:	756)<br>
(Target gene of RNAi)<br>
NM_004304, Homo sapiens anaplastic lymphoma kinase (Ki-1) (ALK).<br>
(Target sequences)<br>
NM_004304-2469,ccacctacgtatttaagat (SEQ ID NO:	757)<br>
NM_004304-4067,cctgtataccggataatga (SEQ ID NO:	758)<br>
NM_004304-2468,gccacctacgtatttaaga (SEQ ID NO:	759)<br>
NM_004304-4183,cgctttgccgatagaatat (SEQ ID NO:	760)<br>
NM_004304-2922,gccacggggaagtgaatat (SEQ ID NO:	761)<br>
(Target sequences effective for mouse homolog)<br><br>
NM_004304-3258,ccatcatgaccgactacaa (SEQ ID NO: 762) NM_004304-2833,caatgaccccgaaatggat (SEQ ID NO: 763) NM_004304-3156,ccggcatcatgattgtgta (SEQ ID NO: 764) (Target gene of RNAi)<br>
NM_000245, Homo sapiens met proto-oncogene (hepatocyte growth factor receptor) (MET). (Target sequences)<br>
NM_000245-2761,gaacagcgagctaaatata (SEQ ID NO: 765) NM_000245-1271,cagcgcgttgacttattca (SEQ ID NO: 766) NM_000245-1086,gtgcattccctatcaaata (SEQ ID NO: 767) NM_000245-725,gattcttaccccattaagt (SEQ ID NO: 768) NM_000245-3619,caaagcgatgaaatatctt (SEQ ID NO: 769) (Target sequences effective for mouse homolog) NM_000245-2987,catttggataggcttgtaa (SEQ ID NO: 7.70) NM_000245-801,ctctagatgctcagacttt (SEQ ID NO: 771) NM_000245-2660,gttaaaggtgaagtgttaa (SEQ ID NO: 772) (Target gene of RNAi)<br>
NM_002529, Homo sapiens neurotrophic tyrosine kinase, receptor, type 1 (NTRK1). (Target sequences)<br>
NM_002529-2091,gcatcctgtaccgtaagtt (SEQ ID NO: 773) NM_002529-345,ggctcagtcgcctgaatct (SEQ ID NO: 774) NM_002529-347,ctcagtcgcctgaatctct (SEQ ID NO: 775) NM_002529-953,ggctccgtgctcaatgaga (SEQ ID NO: 776) NM_002529-1987,ggtcaagattggtgatttt (SEQ ID NO: 777) (Target gene of RNAi)<br>
NM_006180, Homo sapiens neurotrophic tyrosine kinase, receptor, type 2 (NTRK2).<br><br>
(Target sequences)<br>
NM_006180-358,caattttacccgaaacaaa (SEQ ID NO: 778) NM_J)06180-1642,catcaagcgacataacatt (SEQ ID NO: 779) NM_006180-663,gtgatccggttcctaatat (SEQ ID NO: 780) NM„006180-665,gatccggttcctaatatgt (SEQ ID NO: 781) NM_006180-792,cttgtgtggcggaaaatct (SEQ ID NO: 782) (Target sequences effective for mouse homolog) NM„006180-562,cctgcagatacccaattgt (SEQ ID NO: 783) NM„006180-898,ctggtgcattccattcact (SEQ ID NO: 784) NM„006180-735,cacagggctccttaaggat (SEQ ID NO: 785) (Target gene of RNAi)<br>
NM_O00208, Homo sapiens insulin receptor (INSR). (Target sequences)<br>
NM_000208-2562,gccctgtgacgcatgaaat (SEQ ID NO: 786) NM_000208-2565,ctgtgacgcatgaaatctt (SEQ ID NO: 787) NM_000208-3492,gcatggtcgcccatgattt (SEQ ID NO; 788) NM_000208-3493,catggtcgcccatgatttt (SEQ ID NO: 789) NM_O00208-329,ggatcacgactgttcttta (SEQ ID NO: 790) (Target sequences effective for mouse homolog) NM_000208-2911,gattggaagtatttatcta (SEQ ID NO: 791) NM_000208-902,caccaatacgtcattcaca (SEQ ID NO: 792) NM_000208-1514,cggacatcttttgacaaga (SEQ ID NO: 793) (Target gene of RNAi)<br>
NM_000323, Homo sapiens ret proto-oncogene (multiple endocrine neoplasia and medullary thyroid carcinoma 1, Hirschsprung disease) (RET). (Target sequences) NM_000323-2679,gcttgtcccgagatgttta (SEQ ID NO: 794)<br><br>
NM_000323-3066,catctgactccctgattta (SEQ ID NO: 795) NM_000323-3069,ctgactccctgatttatga (SEQ ID NO: 796) NM_000323-2680,cttgtcccgagatgtttat (SEQ ID NO: 797) NM_000323-2728,gggtcggattccagttaaa (SEQ ID NO: 798) (Target sequences effective for mouse homolog) NM_000323-3159,ccacatggattgaaaacaa (SEQ ID NO: 799) NM_000323-3156,cttccacatggattgaaaa (SEQ ID NO: 800) NM_000323-3155,ccttccacatggattgaaa (SEQ ID NO: 801) (Target gene of RNAi)<br>
NM_006293, Homo sapiens TYR03 protein tyrosine kinase (TYR03).<br>
(Target sequences)<br>
NM_006293-1494,gcatcagcgatgaactaaa (SEQ ID NO: 802) NM_006293-2207,gaaaacgctgagatttaca (SEQ ID NO: 803) NM_006293-2394,gccaggaccccttatacat (SEQ ID NO: 804) NM_006293-2399,gaccccttatacatcaaca (SEQ ID NO: 805) NM_006293-1493,ggcatcagcgatgaactaa (SEQ ID NO: 806) (Target gene of RNAi)<br>
NM__182925# Homo sapiens fms-related tyrosine kinase 4 (FLT4).<br>
(Target sequences)<br>
NM__182925-758,gtgtgggctgagtttaact (SEQ ID NO: 807) NM_182925-756,ccgtgtgggctgagtttaa (SEQ ID NO: 808) NM_182925-1217,ggcctgaggcgcaacatca (SEQ ID NO: 809) NM_l82925-l827/gcaagaacgtgcatc1:gtt (SEQ ID NO:. 810) NM_182925-908,gacctgggctcgtatgtgt (SEQ ID NO: 811) (Target sequences effective for mouse homolog) NML_182925-2033/cggctcacgcagaacttga (SEQ ID NO: 812)<br><br>
NM_182925-330,gctactacaagtacatcaa (SEQ ID NO: 813) (Target gene of RNAi)<br>
NM_004119, Homo sapiens fras-related tyrosine kinase 3 (FLT3).<br>
(Target sequences)<br>
NM_004119-1569,gtgagacgatccttttaaa (SEQ ID NO: 814) NM_004119-2490.gattggctcgagatatcat (SEQ ID NO: 815) NM_004119-I57l,gagacgatccttt1:aaact (SEQ ID NO: 816) NM_004119-32.ccgctgctcgttgttt.ttt (SEQ ID NO: 817) NM_004119-730,gttcacaatagatctaaat (SEQ ID NO: 818) (Target sequences effective for mouse homolog) NM_004119-92,gtgatcaagtgtgttttaa (SEQ ID NO: 819) NM_004119-1483,ggtgtcgagcagtactcta (SEQ ID NO: 820) NM_004119-1456,ggctaacagaaaagtgttt (SEQ ID NO: 821) (Target gene of RNAi)<br>
NM_002253, Homo sapiens kinase insert domain receptor (a type III receptor tyrosine kinase) (KDR). (Target sequences)<br>
NM_002253-617,gaaagttaccagtctatta (SEQ ID NO: 822) NM_002253-865,gagcaccttaactatagat (SEQ ID NO: 823) NM_002253-2020,gaatcagacgacaagtatt (SEQ ID NO: 824) NM_002253-815,gtaaaccgagacctaaaaa (SEQ ID NO: 825) NM_002253-2586,ggacagtagcagtcaaaat (SEQ ID NO: 826) (Target sequences effective for mouse homolog) NM_002253-3032.gtggctaagggcatggagt (SEQ ID NO: 827) NM_002253-3627,ccaaattccattatgacaa (SEQ ID NO: 828) NM_002253-3626,cccaaattccattatgaca (SEQ ID NO: 829)<br><br>
(Target gene of RNAi)<br>
NM„002609, Homo sapiens platelet-derived growth factor receptor, beta polypeptide (PDGFRB). (Target sequences)<br>
NM_002609-961,ggtgggcacactacaattt (SEQ ID NO: 830) NM_002609-2881,gttgggcgaaggttacaaa (SEQ ID NO: 831) NM_002609-409,ctttctcacggaaataact (SEQ ID NO: 832) NM_J)02609-278,gacacgggagaatactttt (SEQ ID NO: 833) NM_002609-3048,gtgacaacgactatatcat (SEQ ID NO: 834) (Target sequences effective for mouse homolog) NM_002609-633,catccatcaacgtctctgt (SEQ ID NO: 835) NM_0Q2609-2784,cctccgacgagatctatga (SEQ ID NO: 836) (Target gene of RNAi)<br>
NM„005433, Homo sapiens v-yes-1 Yamaguchi sarcoma viral oncogene homolog 1 (YES1), (Target sequences)<br>
NM_005433-525,gaaatcaacgaggtat:ttt (SEQ ID NO: 837) NM_005433-670/cacaaccagagcacaattt (SEQ ID NO: 838) NM_005433-1333,gtatggtcggtttacaata (SEQ ID NO: 839) NM_005433-1331,ctgtatggtcggtttacaa (SEQ ID NO: 840) NM_005433-416,ggttatatcccgagcaatt (SEQ ID NO: 841) (Target sequences effective for mouse homolog) NM_005433-953,caagaagctcagataatga (SEQ ID NO: 842) NM_J)05433-l,gggctgcattaaaagtaaa (SEQ ID NO: 843) NM_005433-4,ctgcattaaaagtaaagaa (SEQ ID NO: 844) (Target gene of RNAi)<br>
NM_002005, Homo sapiens feline sarcoma oncogene (FES). (Target sequences)<br><br>
NM_002005-1696,gattggacgggggaacttt (SEQ ID NO: 845)<br>
NM_002005-2181,cacctgaggcccttaacta (SEQ ID NO: 846)<br>
NM_002005-1553,ggctttcctagcattcctt (SEQ ID NO: 847)<br>
NM_002005-683,gaatacctggagattagca (SEQ ID NO: 848)<br>
NM_002005-74,ctactggagggcatgagaa (SEQ ID NO: 849)<br>
(Target gene of RNAi)<br>
NM_000633, Homo sapiens B-cell CLL/lymphoma 2 (BCL2).<br>
(Target sequences)<br>
NM_000633-43,gatgaagtacatcca1:tat (SEQ ID NO: 850)<br>
NM_000633-41,gtgatgaagtacatccatt (SEQ ID NO: 851)<br>
(Target sequences effective for mouse homolog)<br>
NM_000633-452.gagttcggtggggtc.atgt (SEQ ID NO: 852)<br>
NM_000633-454,gttcggtggggtcatgtgt (SEQ ID NO: 853)<br>
NM_000633-525,ggatgactgagtacctgaa (SEQ ID NO: 854)<br>
(Target gene of RNAi)<br>
NM_O01167, Homo sapiens baculoviral IAP repeat-containing 4<br>
(BIRC4).<br>
(Target sequences)<br>
NM_001167-302,gccacgcagtctacaaatt (SEQ ID NO: 855)<br>
NM_001167-794,gaagcacggatcttt<ictt id no:></ictt>
NM_001167-485,gaagaagctagattaciagt (SEQ ID NO: 857)<br>
NM_001167-402,cacatgcagactatctttt (SEQ ID NO: 858)<br>
(Target sequences effective for mouse homolog)<br>
NM_001167-71,gaagagtttaatagattaa (SEQ ID NO: 859)<br>
NM_001167-68/gtagaagagtttaatagat (SEQ ID NO: 860)<br>
NM_001167-1354,ctgtatggatagaaatatt (SEQ ID NO: 861)<br>
(Target gene of RNAi)<br>
NM_139317, Homo sapiens baculoviral IAP repeat-containing 7<br><br>
(livin) (BIRC7). {Target sequences)<br>
NM_139317-458,ctgctccggtcaaaaggaa	(SEQ ID NO:	862)<br>
NM__139317-457,cctgctccggtcaaaagga	(SEQ ID NO:	863)<br>
NM_139317-743,gagaggacgtgcaaggtgt	(SEQ ID NO:	864)<br>
NM_139317-774/ccgtgtccatcgtctttgt	(SEQ ID NO:	865)<br>
NM_139317-417,cctggacggagcatgccaa	(SEQ ID NO:	866)<br>
(Target gene of RNAi)<br>
NM_005036, Homo sapiens peroxisome proliferative activated receptor, alpha (PPARA). (Target sequences)<br>
NM_005036-922,gctaaaatacggagtttat	(SEQ ID NO:	867)<br>
NM_005036-1243.ccacccggacgatatcttt (SEQ ID NO: 868)<br>
NM_005036~711,cttttgtcatacatgatat	(SEQ ID NO:	869)<br>
NM_005036-498,cacacaacgcgattcgttt	(SEQ ID NO:	870)<br>
NM_005036-988,gctggtagcgtatggaaat	(SEQ ID NO:	871)<br>
(Target gene of RNAi)<br>
NM_138712, Homo sapiens peroxisome proliferative activated receptor, gamma (PPARG). (Target sequences)<br>
NM_138712-953,ggagtccacgagatcattt	(SEQ ID NO:	872)<br>
NM_138712-304,ctccctcatggcaattgaa	(SEQ ID NO:	873)<br>
NM_138712-954,gagtccacgagatcattta	(SEQ ID NO:	874)<br>
NM_138712-445,ctgtcggatccacaaaeiaa	(SEQ ID NO:	875)<br>
NM_138712-409,cagatt.gaagcttatctat	(SEQ ID NO:	876)<br>
(Target sequences effective for mouse homolog)<br>
NM_138712-239,gcatctccaccttattatt	(SEQ ID NO:	877)<br>
NM_138712-688,ggcgagggcgatcttgaca	(SEQ ID NO:	878)<br><br>
NM_138712-664,gtccttcccgctgaccaaa (SEQ ID NO: 879)<br>
(Target gene of RNAi)<br>
NM_004421, Homo sapiens dishevelled, dsh homolog 1<br>
(Drosophila) (DVL1).<br>
(Target sequences)<br>
NM__004421-1173,ccgtcgtccgggtcatgca (SEQ ID NO: 880)<br>
(Target gene of RNAi)<br>
NM_004422, Homo sapiens dishevelled, dsh homolog 2<br>
(Drosophila) (DVL2).<br>
(Target sequences)<br>
NM_004422-1253,gtccatacggacatggcat (SEQ ID NO: 881)<br>
(Target gene of RNAi)<br>
NM_004423, Homo sapiens dishevelled, dsh homolog 3<br>
(Drosophila) (DVL3).<br>
(Target sequences)<br>
NM„004423-1197,gcctagacgacttccactt (SEQ ID NO: 882)<br>
(Target gene of RNAi)<br>
NC_001802, Human immunodeficiency virus 1, complete genome.<br>
(Target sequences)<br>
NC_001802-8242,ggacagatagggttataga (SEQ ID NO: 883)<br>
NC__001802-340,gcgagagcgtcagtattaa (SEQ ID NO: 884)<br>
NC_001802-1222,gtagaccggttctataaaa (SEQ ID NO: 885)<br>
NC_001802-1818,cgacccctcgtcacaataa (SEQ ID NO: 886)<br>
NC_001802-4973,gccctaggtgtgaatatca (SEQ ID NO: 887)<br>
NC_001802-5224,gcttagggcaacatatcta (SEQ ID NO: 888)<br>
NC„001802-550,gaagaacttagatcattat (SEQ ID NO: 889)<br>
NC_001802-1777,gaactgtatcctttaactt (SEQ ID NO: 890)<br>
NCL001802-3244,gaaagactcctaaatttaa (SEQ ID NO: 891)<br><br>
NC„001802-5225,cttagggcaacatatctat (SEQ ID NO: 892)<br>
[Advantages of the Invention]<br>
According to the present invention, siRNA actually 5  having an RNAi effect can be obtained with high probability. Thus, when preparing novel siRNA, it is possible to greatly reduce the effort required to carry out repeated tests of trial and error, based on the experiences of the researcher, in actually synthesizing siRNA and in confirming whether the synthesized product has an RNAi effect.  Namely, the present invention is extremely preferred for carrying out a search or for creation of siRNA having a novel sequence. Furthermore, by using the present invention, a wide variety of desired siRNA can be obtained in a short time.  Since necessity for actual preparation of siRNA in a trial-and-error manner has been reduced, it becomes possible to greatly reduce the cost required for testing and manufacturing techniques, in which RNA interference is used. Additionally, the present invention not only greatly simplifies all testing and manufacturing techniques, in which the RNAi effect is used, but also significantly improves their reliability as techniques.  The present invention is particularly effective in performing RNA interference in higher animals such as mammals.<br>
INDUSTRIAL APPLICABILITY<br>
As described above, the present invention relates to RNA interference and more particularly, for example, to a<br><br>
method for designing sequences of polynucleotides for causing RNA interference, the method improving efficiency in testing, manufacturing, etc., in which RNA interference is used.<br><br><br>
CLAIMS<br>
1.	A method for searching a target base sequence of RNA<br>
interference comprising: searching a sequence segment,<br>
conforming to the following rules (a) to (d), from the base<br>
sequences of a target gene of RNA interference:<br>
(a)	The 3' end base is adenine, thymine, or uracil,<br>
(b)	The 5' end base is guanine or cytosine,<br>
(c)	A 7-base sequence from the 3' end is rich in one or more types of bases selected from the group consisting of adenine, thymine, and uracil, and<br>
(d)	The number of bases is within a range that allows RNA interference to occur without causing cytotoxicity.<br><br>
2.	The method for searching the target base sequence according to claim 1, wherein, in the rule (c)# at least three bases among the seven bases are one or more types of bases selected from the group consisting of adenine, thymine, and uracil.<br>
3.	The method for searching the target base sequence according to claim 1 or 2, wherein, in the rule (d) , the number of bases is 13 to 28.<br>
4.	A method for designing a base sequence of a polynucleotide for causing RNA interference comprising: searching a base sequence, conforming to the rules (a) to (d) below, from the base sequences of a target gene ,and designing a base sequence homologous to the searched base sequence:<br>
(a) The 3' end base is adenine, thymine, or uracil.<br><br>
(b)	The 5' end base is guanine or cytosine,<br>
(c)	A 7-base sequence from the 3' end is rich in one or more types of bases selected from the group consisting of adenine, thymine, and uracil, and<br>
(d)	The number of bases is within a range that allows RNA interference to occur without causing cytotoxicity.<br><br>
5.	The method for designing the base sequence according to claim 4, wherein, in the rule (c), at least three bases among the seven bases are one or more types of bases selected from the group consisting of adenine, thymine, and uracil.<br>
6.	The method for designing the base sequence according to claim 4 or 5, wherein the number of bases in the homologous base sequence designed is 13 to 28,<br>
7.	The method for designing the base sequence according<br>
to any one of claims 4 to 6, wherein designing is performed<br>
so that at least 80% of bases in the homologous base<br>
sequence designed corresponds to the base sequence searched.<br>
8.	The method for designing the base sequence according to any one of claims 4 to 7, wherein the 3' end base of the base sequence searched is the same as the 3' end base of the base sequence designed, and the 5' end base of the base sequence searched is the same as the 5' end base of the base sequence designed.<br>
9.	The method for designing the-base sequence according to any one of claims 4 to 8, wherein an overhanging portion is added to the 3' end of the polynucleotide.<br>
10.	A method for producing a double-stranded<br><br>
polynucleotide comprising:<br>
forming one strand by providing an overhanging portion to the 3' end of a base sequence homologous to a prescribed sequence which is contained in the base sequences of a target gene and which conforms to the following rules (a) to (d); and<br>
forming the other strand by providing an overhanging portion to the 3' end of a base sequence complementary to the base sequence homologous to the prescribed sequence,<br>
wherein the number of bases in each strand is 15 tp 30:<br>
(a)	The 3' end base is adenine, thymine, or uracil,<br>
(b)	The 5' end base is guanine or cytosine,<br>
(c)	A 7-base sequence from the 3' end is rich in one or more types of bases selected from the group consisting of adenine, thymine, and uracil, and<br>
(d)	The number of bases is within a range that allows RNA interference to occur without causing cytotoxicity.<br>
11.  A double-stranded polynucleotide synthesized by searching a sequence segment having 13 to 28 bases, conforming to the following rules (a) to (d), from the base sequences of a target gene for RNA interference, forming one strand by providing an overhanging portion to the 3' end of a base sequence homologous to a prescribed sequence which is contained in the base sequences of the target gene and which conforms to the following rules (a) to (d), and forming the other strand by providing an overhanging portion to the 3' end of a base sequence complementary to<br><br>
the base sequence homologous to the prescribed sequence, wherein the number of bases in each strand is 15 to 30:<br>
(a)	The 3' end base is adenine, thymine, or uracil,<br>
(b)	The 5' end base is guanine or cytosine,<br>
(c)	A 7-base sequence from the 3' end is rich in one or more types of bases selected from the group consisting of adenine, thymine, and uracil, and<br>
(d)	The number of bases is within a range that allows RNA interference to occur without causing cytotoxicity.<br>
12.  A method for inhibiting gene expression comprising the steps of:<br>
searching a sequence segment having 13 to 28 bases, conforming to the following rules (a) to (d), from the base sequences of a target gene for RNA interference;<br>
synthesizing a double-stranded polynucleotide such that one strand is formed by providing an overhanging portion to the 3' end of a base sequence homologous to a prescribed sequence which is contained in the base sequences of the target gene and which conforms to the following rules (a) to (d), the other strand is formed by providing an overhanging portion to the 3' end of a base sequence complementary to the base sequence homologous to the prescribed sequence, and the number of bases in each strand is 15 to 30; and<br>
introducing the synthesized double-stranded polynucleotide into an expression system of the target gene of which expression is to be inhibited to inhibit the expression of the target gene:<br><br>
(a)	The 3' end base is adenine, thymine, or uracil,<br>
(b)	The 5' end base is guanine or cytosine,<br>
(c)	A 7-base sequence from the 3' end is rich in one or more types of bases selected from the group consisting of adenine, thymine, and uracil, and<br>
fd) The number of bases is within a range that allows RNA interference to occur without: causing cytotoxicity. 13.  A base sequence processing apparatus characterized in that it comprises:<br>
partial base sequence creation means for acquiring base sequence information of a target gene for RNA interference and creating partial base sequence information corresponding to a sequence segment having a predetermined number of bases in the base sequence information;<br>
3' end base determination means for determining whether the 3' end base in the partial base sequence information created by the partial base sequence creation means is adenine, thymine, or uracil;<br>
5' end base determination means for determining whether the 5' end base in the partial base sequence information created by the partial base sequence creation means is guanine or cytosine;<br>
predetermined base inclusion determination means for determining whether base sequence information comprising 7 bases at the 3' end in the partial base sequence information created by the partial base sequence creation means is rich in one or more types of bases selected from the group consisting of adenine, thymine, and uracil; and<br><br>
prescribed sequence selection means for selecting prescribed sequence information which specifically causes RNA interference in the target gene from the partial base sequence information created by the partial base sequence creation means, based on the results determined by the 3f base determination means, the 5' end base determination means, and the predetermined base inclusion determination means.<br>
14.	The base sequence processing apparatus according to claim 13, characterized in that the partial base sequence creation means further comprises region-specific base sequence creation means for creating the partial base sequence information having the predetermined number of bases from a segment corresponding to a coding region or transcription region of the target gene in the base sequence information.<br>
15.	The base sequence processing apparatus according to claim 13 or 14, characterized in that the partial base sequence creation means further comprises common base sequence creation means for creating the partial base sequence information having the predetermined number of bases which is common in a plurality of base sequence information derived from different organisms.<br>
16.	The base sequence processing apparatus according to any one of claims 13 to 15, characterized in that the base sequence information that is rich corresponds to base sequence information comprising the 7 bases containing at least 3 bases which are one or more types of bases selected<br><br>
from the group consisting of adenine, thymine, and uracil.<br>
17.	The base sequence processing apparatus according to any one of claims 13 to 16, wherein the predetermined number of bases is 13 to 28.<br>
18.	The base sequence processing apparatus according to any one of claims 13 to 17, characterized in that the partial base sequence creation means further comprises overhanging portion-containing base sequence creation means for creating the partial base sequence information containing an overhanging portion.<br>
19.	The base sequence processing apparatus according to any one of claims 13 to 17, characterized in that it comprises:<br>
overhanging-portion addition means for adding an overhanging portion to at least one end of the prescribed sequence information.<br>
20.	The base sequence processing apparatus according to claim 18 or 19, wherein the number of bases in the overhanging portion is 2.<br>
21.	The base sequence processing apparatus according to any one of claims 13 to 20, characterized in that it comprises:<br>
identical/similar base sequence search means for searching base sequence information, identical or similar to the prescribed sequence information, from other base sequence information; and<br>
unrelated gene target evaluation means for evaluating whether the prescribed sequence information targets genes<br><br>
unrelated to the target gene based on the identical or similar base sequence information searched by the identical/similar base sequence search means.<br>
22.	The base sequence processing apparatus according to<br>
claim 21, characterized in that the unrelated gene target<br>
evaluation means further comprises:<br>
total sum calculation means for calculating the total sum of reciprocals of the values showing the degree of identity or similarity based on the total amount of base sequence information on the genes unrelated to the target gene in the identical or similar base sequence information searched by the identical/similar base sequence search means and the values showing the degree of identity or similarity attached to the base sequence information on the genes unrelated to the target gene; and<br>
total sum-based target evaluation means for evaluating whether the prescribed sequence information targets the genes unrelated to the target gene based on the total sum calculated by the total sum calculation means*<br>
23.	A program for running base sequence processing method<br>
on computer, characterized in that it comprises:<br>
a partial base sequence creation step of acquiring base sequence information of a target gene for RNA interference and creating partial base sequence information corresponding to .a sequence segment having a predetermined number of bases in the base sequence information;<br>
a 3' end base determination step of determining whether the 3' end base in the partial base sequence<br><br>
information created in the partial base sequence creation step is adenine, thymine, or uracil;<br>
a 5' end base determination step of determining whether the 5' end base in the partial base sequence information created in the partial base sequence creation step is guanine or cytosine;<br>
a predetermined base inclusion determination step of determining whether base sequence information comprising 7 bases at the 31 end in the partial base sequence information created in the partial base sequence creation step is rich in one or more types of bases selected from the group consisting of adenine, thymine, and uracil; and<br>
a prescribed sequence selection step of selecting, based on the results determined in the 3' base determination step, the 5' end base determination step, and the predetermined base inclusion determination step, prescribed sequence information which specifically causes RNA interference in the target gene from the partial base sequence information created in the partial base sequence creation step.<br>
24-   A computer-readable recording medium characterized in that the program according to claim 23 is recorded in the medium.<br>
25.   A base sequence processing system which comprises a base sequence processing apparatus processing base sequence information of a target gene for RNA interference and a client apparatus, the base sequence processing apparatus and the client apparatus being connected to each other via<br><br>
a network in a communicable manner, characterized in that the client apparatus comprises:<br>
base sequence transmission means for transmitting a name of the target gene or the base sequence information to the base sequence processing apparatus; and<br>
prescribed sequence acquisition means for acquiring prescribed sequence information which is transmitted from the base sequence processing apparatus and which specifically causes RNA interference in the target gene, and<br>
the base sequence processing apparatus comprises:<br>
partial base sequence creation means for acquiring base sequence information corresponding to the name of the target gene or the base sequence information transmitted from the client apparatus and creating partial base sequence information corresponding to a sequence segment having a predetermined number of bases in the base sequence information;<br>
3' end base determination means for determining whether the 3' end base in the partial base sequence information created by the partial base sequence creation means is adenine, thymine, or uracil;<br>
5' end base determination means for determining whether the 5' end base in the partial base sequence information created by the partial base sequence creation means is guanine or cytosine;<br>
predetermined base inclusion determination means for determining whether base sequence information comprising 7<br><br>
bases at the 3' end in the partial base sequence information created by the partial base sequence creation means is rich in one or more types of bases selected from the group consisting of adenine, thymine, and uracil;<br>
prescribed sequence selection means for selecting the prescribed sequence information from the partial base sequence information created by the partial base sequence creation means, based on the results determined by the 3' base determination means, the 5' end base determination means, and the predetermined base inclusion determination means; and<br>
prescribed sequence transmission means for transmitting the prescribed sequence information selected by the prescribed sequence selection means to the client apparatus.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3OC1jaGVucC0yMDA1LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1378-chenp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3OC1jaGVucC0yMDA1LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1378-chenp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3OC1jaGVucC0yMDA1LWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1378-chenp-2005-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3OC1jaGVucC0yMDA1LWNvcnJlc3BvbmRuZWNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">1378-chenp-2005-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3OC1jaGVucC0yMDA1LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">1378-chenp-2005-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3OC1jaGVucC0yMDA1LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1378-chenp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3OC1jaGVucC0yMDA1LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">1378-chenp-2005-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3OC1jaGVucC0yMDA1LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">1378-chenp-2005-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3OC1jaGVucC0yMDA1LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">1378-chenp-2005-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3OC1jaGVucC0yMDA1LWZvcm0xOC5wZGY=" target="_blank" style="word-wrap:break-word;">1378-chenp-2005-form18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM3OC1jaGVucC0yMDA1LXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1378-chenp-2005-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="219121-a-heat-curable-polyurethane-reactive-composition.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="219123-a-process-for-the-preparation-of-detergents.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>219122</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1378/CHENP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>23/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>06-Jun-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>25-Apr-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>22-Jun-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>BIO-THINK TANK CO. LTD</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SAIGO, Kaoru</td>
											<td></td>
										</tr>
										<tr>
											<td>2</td>
											<td>NAITO, Yuki</td>
											<td></td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/11</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/JP2003/014893</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-11-21</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2002-340053</td>
									<td>2002-11-22</td>
								    <td>Japan</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/219122-a-method-for-identifying-a-target-base-sequence-of-rna-interference-and-an-apparatus-used-therein by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:11:53 GMT -->
</html>
