<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/219444-method-to-modify-plant-growth-characteristics-by-recombinant-means-using-a-nucleic-acid-encoding-a-2xc2h2-protein by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:50:44 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 219444:&quot;METHOD TO MODIFY PLANT GROWTH CHARACTERISTICS BY RECOMBINANT MEANS USING A NUCLEIC ACID ENCODING A 2XC2H2 PROTEIN&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;METHOD TO MODIFY PLANT GROWTH CHARACTERISTICS BY RECOMBINANT MEANS USING A NUCLEIC ACID ENCODING A 2XC2H2 PROTEIN&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention concerns a method formodifying the growth characteristics of plants by modifying expression in a plant of a nucleic acid sequence encoding a 2xC2H2 zinc finger protein and/or modifying level and/or activity in a plant of a 2xC2H2 zinc linger pivilein. The invention also relates lo transgenic plants having modified growth characteristics, which plants have modified expression ot a nucleic acid encoding a 2xC2H2 y.inc finger protein. For example yield of crop plants are improved by ihe methods of the present invention.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Plants having modified growth characteristics and a method for<br>
making the same<br>
The present invention concerns a method for modifying plant growth characteristics. More specifically, the present invention concerns a method for modifying the growth characteristics of a plant by modifying expression of a nucleic acid encoding a zinc finger protein and/or by modifying the level and/or activity of a zinc finger protein in a plant, which zinc finger protein has two zinc finger domains of the type C2H2 (2xC2H2). The present invention also concerns plants having modified expression of a nucleic acid encoding a 2xC2H2 zinc finger protein and/or modified levels and/or activity of a 2xC2H2 zinc finger protein, which plants have modified growth characteristics relative to corresponding wild type plants.<br>
Given the ever-increasing world population, it remains a major goal of agricultural research to improve the efficiency of agriculture. Conventional means for crop and horticultural improvements utilise selective breeding techniques to identify plants having desirable characteristics. However, such selective breeding techniques have several drawbacks, namely that these techniques are typically labour intensive and result in plants that often contain heterogeneous genetic components that may not always result in the desirable trait being passed on from parent plants. Advances in molecular biology have allowed mankind to modify the germplasm of animals and plants in a specific and controlled way. Genetic engineering of plants entails the isolation and manipulation of genetic material (typically in the form of DNA or RNA) and the subsequent introduction of that genetic material into a plant. Such technology has led to the development of plants having various improved economic, agronomic or horticultural traits. A trait or growth characteristic of particular economic interest is high yield. Yield is normally defined as the measurable produce of economic value from a crop. This may be defined in terms of quantity and/or quality. Other important growth characteristics include modified architecture, modified growth rate, among others.<br>
The ability to influence one or more of the abovementioned growth characteristics, would have many applications in areas such as crop enhancement, plant breeding, production of ornamental plants, arboriculture, horticulture, forestry, production, of algae or plants (for example for use as bioreactors, for the production of substances such as Pharmaceuticals, antibodies, or vaccines, or for the bioconversion of organic waste or for use as fuel in the case of high-yielding algae and plants).<br><br>
The term "zinc finger" describes a nucleic acid-binding domain in a protein that is folded around a tetrahedrally coordinated Zinc ion (Miller et al. 1985. EMBO, 4, 1609-1614). The amino acids that coordinate the zinc ion, are always cystein or histidine residues, however, diversity occurs in the sequence and length of the zinc finger domain. Zinc finger proteins may contain several zinc finger domains of the same or different type. Further variability is encountered in nature by association of zinc finger domains with other domains. For example, some zinc finger proteins are found in association with ring finger or coil-coil domains, to form a so-called tripartite domain. There are several types of zinc fingers, such as C2H2, C2HC, C2C2. C2H2 is known as the classical zinc finger domain. There are typically two criteria used to classify zinc finger proteins, the first being the type of zinc finger and the second being the number of zinc fingers present in the protein. Zinc finger proteins having a single C2H2 domain have been characterised, for example Superman from Arabidopsis and Ramosa I from maize. A well-characterised zinc finger protein having three C2H2 domains is the Indeterminate 1 protein from Maize. Although the first report of this gene (Colasanti et al., Cell. 1998 May 15;93(4):593-603)  only  mentions the  presence  of two zinc finger  domains,   a  more sophisticated analysis, using pFAM domain search, revealed the presence of three C2H2 zinc finger domains. Also known are zinc-finger proteins having only two C2H2 domains, for example 2AT10 (STZ )and SCOF-1. This subset of plant zinc finger proteins having two C2H2 domains have been implicated in plant responses to various stresses (Sakamoto et al., Gene 248 (1-2) 23-32 (2000)). Both STZ and SCOF-1 have been used to enhance abiotic stress tolerance. When over-expressed, STZ has been reported to increase salt tolerance in yeast (Lippuner et al,, J Biol Chem. 271 (22) 12859-12866 (1996)) and over-expression of the SCOF-1 gene under control of the CaMV 35 S promoter has been reported to enhance cold tolerance in Arabidopsis ihaliana (Kim et al., Plant J. 25 (3) 247-259 (2001)). Reports of plants having modified expression of a zinc finger encoding gene (whether the zinc finger gene is mutated,    over-expressed   or   otherwise)   describe   plants   having   abnormal   growth characteristics, none of which (with the exception of cold stress tolerance in transgenic plants expressing SCOF-1) are desirable for crops or describe effects that are only detectable under particular stress conditions.<br>
It has now been found that modifying expression in a plant of a 2xC2H2 zinc finger gene and/or modifying the level and/or activity in a plant of a 2xC2H2 zinc finger protein gives plants having modified growth characteristics. In particular it has been found that introduction into a plant of a 2xC2H2 zinc finger nucleic acid gives plants modified growth characteristics, such as increased yield, modified leaf architecture and altered cycle time, each relative to wild type plants.<br><br>
Therefore according to one embodiment of the present invention there is provided a method for modifying the growth characteristics of a plant, comprising modifying expression in a plant of a nucleic acid encoding a 2xC2H2 zinc finger protein and/or modifying level and/or activity in a plant of a 2xC2H2 zinc finger protein.<br>
The term "modifying" as used herein is taken to mean enhancing, decreasing and/or changing in place and/or time. Modifying expression of a nucleic acid encoding a 2xC2H2 zinc finger protein or modifying the level and/or activity of the 2xC2H2 zinc finger protein itse/f encompasses altered expression of a gene and/or altered level and/or activity of a gene product, namely a polypeptide, in specific cells or tissues, when compared to expression, level • and/or activity of a 2xC2H2 zinc finger gene or protein in corresponding wild-type plants. The modified gene expression may result from modified expression of an endogenous 2xC2H2 zinc finger gene and/or may result from modified expression of a 2xC2H2 zinc finger gene previously introduced into a plant. Similarly, modified levels and/or activity of a 2xC2H2 zinc finger protein may be due to modified expression of an endogenous 2xC2H2 zinc finger nucleic acid/gene and/or due to modified expression of a 2xC2H2 zinc finger nucleic acid/gene previously introduced into a plant. Modified expression of a gene/nucleic acid and/or modified level and/or activity of a gene product/protein may be effected, for example, by chemical means and/or recombinant means.<br>
Therefore there is provided by the present invention, a method for modifying the growth characteristics of a plant, comprising modifying expression, level and/or activity of a 2xC2H2 zinc finger gene or protein by recombinant means and/or by chemical means.<br>
Advantageously, modifying expression of a nucleic acid encoding a 2xC2H2 zinc finger protein and/or modifying level and/or activity of the 2xC2H2 zinc finger protein itself may be effected by chemical means, i.e. by exogenous application of one or more compounds or elements capable of modifying activity of the 2xC2H2 zinc finger protein and/or capable of modifying expression of a 2xC2H2 zinc finger gene (which may be either an endogenous gene or a transgene introduced into a plant). The term "exogenous application" as defined herein is taken to mean the contacting or administering of a suitable compound or element to a plant. The compound or element may be exogenously applied to a plant in a form suitable for plant uptake (such as through application to the soil for uptake via the roots, or in the case of some plants by applying directly to the leaves, for example by spraying). The exogenous application may take place on wild-type plants or on transgenic plants that have previously been transformed with a 2xC2H2 zinc finger nucleic acid/gene or other transgene.<br><br>
Suitable compounds or elements for exogenous application include 2xC2H2 zinc finger proteins or 2xC2H2 zinc finger nucleic acids. Alternatively, exogenous application of compounds or elements capable of modifying levels of factors that directly or indirectly activate or inactivate a 2xC2H2 zinc finger, protein will also be suitable in practising the invention. Also included are antibodies that can recognise or mimic the function of 2xC2H2 zinc finger proteins. Such antibodies may comprise "plantibodies", single chain antibodies, IgG antibodies and heavy chain camel antibodies, as well as fragments thereof.<br>
Additionally or alternatively, the resultant effect may also be achieved by the exogenous application of an interacting protein or activator or an inhibitor of a 2xC2H2 zinc finger gene/gene product. Additionally or alternatively, the compound or element may be a mutagenic substance, such as a chemical selected from any one or more of: N-nitroso-N-ethylurea, ethylene imine, ethyl methanesulphonate and diethyl sulphate. Mutagenesis may also be achieved by exposure to ionising radiation, such as X-rays or gamma-rays or ultraviolet light. Methods for introducing mutations and for testing the effect of mutations (such as by monitoring gene expression and/or protein activity) are well known in the art.<br>
Additionally or alternatively, and according to a preferred embodiment of the present invention, modifying expression of a nucleic acid encoding a 2xC2H2 zinc finger protein and/or modifying level and/or activity of the 2xC2H2 zinc finger protein may be effected by recombinant means. Such recombinant means may comprise a direct and/or indirect approach for modifying expression of a nucleic acid and/or level and/or activity of a protein.<br>
For example, an indirect approach may comprise introducing, into a plant, a nucleic acid capable of modifying expression of the gene in question (a gene encoding a 2xC2H2 zinc finger protein) and or capable of modifying the level and/or activity of the protein in question (a 2xC2H2 zinc finger protein). Examples of such nucleic acids to be introduced into a plant include nucleic acids encoding transcription factors or activators or inhibitors that bind to the promoter of a 2xC2H2 zinc finger gene or that interact with a 2xC2H2 zinc finger protein. Methods to test these types of interactions and methods for isolating nucleic acids encoding such interactors include yeast one-hybrid or yeast two-hybrid screens in which the 2xC2H2 zinc finger gene/protein is used as bait. One example of such a transcription regulator is LOS2, described as a transcription regulator for the STZ gene. Therefore, the method of the invention may also be performed using LOS2, wherein expression of a 2xC2H2 zinc finger gene may be increased or further increased by decreasing expression of LOS2 in plants.<br><br>
Also encompassed by an indirect approach for modifying expression of a 2xC2H2 zinc finger gene and/or for modifying level and/or activity of a 2xC2H2 zinc finger protein is the provision of, or the inhibition or stimulation of regulatory sequences that drive expression of a native 2xC2H2 zinc finger gene or transgene. Such regulatory sequences may-be introduced into a plant. For example, the regulatory sequence to be introduced into a plant may be a promoter capable of driving expression of an endogenous 2xC2H2 zinc finger gene.<br>
A further indirect approach for modifying expression of a 2xC2H2 zinc finger gene and/or for modifying level and/or activity of a 2xC2H2 zinc finger protein in a plant encompasses modifying levels in a plant of a factor capable of interacting with a zinc finger protein. Such factors may include ligands of a 2xC2H2 zinc finger protein. Therefore, the present invention also provides a method for modifying growth characteristics of a plant, comprising modifying expression of a gene coding for a protein which is a natural ligand of a 2xC2H2 zinc finger protein. Furthermore, the present invention also provides a method for modifying growth characteristics of a plant, comprising modifying expression of a gene coding for a protein which is a natural target/substrate of a 2xC2H2 zinc finger protein. Examples of such targets/substrates include stretches of DMA that are bound by the zinc-finger domains.<br>
A direct and preferred approach on the other hand comprises introducing into a plant a nucleic acid encoding a 2xC2H2 zinc finger protein or a portion thereof or sequences capable of hybridising therewith, which nucleic acid preferably encodes a 2xC2H2 zinc finger protein or a homologue, derivative or active fragment thereof. The nucleic acid may be introduced into a plant by, for example, transformation.<br>
Therefore, there is provided a method for modifying growth characteristics of a plant, comprising introducing into a plant a 2xC2H2 zinc finger nucleic acid or a portion thereof.<br>
The 2xC2H2 zinc finger nucleic acid may be derived (either directly or indirectly (if subsequently modified)) from any source provided that the sequence, when expressed in a plant, leads to modified expression of a 2xC2H2 zinc finger-encoding nucleic acid/gene and/or modified level and/or activity of a 2xC2H2 zinc finger protein. The 2xC2H2 zinc finger gene or protein may be wild type, i.e. the native or endogenous nucleic acid or polypeptide. Alternatively, it may be a protein or nucleic acid derived from the same or another species. The nucleic acid/gene may then be introduced into a plant as a transgene, for example by transformation.<br><br>
The nucleic acid may be isolated from a bacteria, yeast or fungi, or from a plant, algae, insect or animal (including human) source. This nucleic acid may be substantially modified from its native form in composition and/or genomic environment through deliberate human manipulation. The nucleic acid is preferably obtained.from a plant, whether from the same, plant species in which it is to be introduced or whether from a different plant species. Further preferably, the nucleic acid is from a dicot, preferably from the family Brassicaceae, further preferably from Arabidopsis thaliana. More preferably, the nucleic acid is essentially similar to a nucleic acid as represented by SEQ ID NO 1, or a portion of SEQ ID NO 1, or a nucleic acid capable of hybridising therewith or is a nucleic acid encoding an amino acid sequence essentially similar to an amino acid as represented by SEQ ID NO 2, or a homofogue, derivative or active fragment thereof.<br>
Advantageously, the methods according to the invention may also be practised using variant<br>
2xC2H2 zinc finger nucleic acids and variant 2xC2H2 zinc finger amino acids, preferably<br>
wherein the variant nucleic acids are variants of SEQ ID NO 1 and wherein the variant amino<br>
acids are van'ants of SEQ ID NO 2. Examples of variant sequences suitable in performing the<br>
methods of the invention include:<br>
(i)       Functional portions of a 2xC2H2 zinc finger nucleic acid/gene;<br>
(ii)       Sequences capable of hybridising with a 2xC2H2 zinc finger nucleic acid/gene;<br>
(Hi)      Alternative splice variants of a 2xC2H2 zinc finger nucleic acid/gene;<br>
(iv)      Allelic variants of a 2xC2H2 zinc finger nucleic acid/gene;<br>
(v)      Homologues, derivatives and active fragments of a 2xC2H2 zinc finger protein.<br>
The abovementioned variants may also be described as being "essentially similar" to a 2xC2H2 zinc finger nucleic acid/gene, particularly to the 2xC2H2 zinc finger encoding nucleic acid of SED ID NO 1, or essentially similar to a 2xC2H2 zinc finger amino acid/protein, particularly that of SED ID NO 2. The term "essentially similar to" also includes variants of SEQ ID NO 1 in the form of a complement, DNA, RNA, cDNA or genomic DNA. The variant nucleic acid encoding a 2xC2H2 zinc finger protein or the variant of a 2xC2H2 zinc finger protein may be synthesized in whole or in part, it may be a double-stranded nucleic acid or a single-stranded nucleic acid. Also, the term encompasses a variant due to the degeneracy of the genetic code; a family member of the gene or protein; and variants that are interrupted by one or more intervening sequences.<br>
An example of a variant 2xC2H2 zinc finger nucleic acid is a functional portion of a 2xC2H2 zinc-finger gene. Advantageously, the method according to the present invention may also be practised using portions of a DNA or nucleic acid encoding a 2xC2H2 zinc finger protein. A<br><br>
functional portion refers to a piece of DNA derived or prepared from an original (larger) DNA molecule, which DNA portion, when expressed in a plant, gives plants having modified growth characteristics. The portion may comprise many genes, with or without additional control elements or may contain spacer sequences. The portion may be made by making one or more deletions and/or truncations to the nucleic acid. Techniques for introducing truncations and deletions into a nucleic acid are well known in the art. Portions suitable for use in the methods according to the invention may readily be determined by following the methods described in the Examples section by simply substituting the sequence used in the actual Example with the portion to be tested for functionality.<br>
An example of a further variant 2xC2H2 zinc finger nucleic acid is a sequence that is capable<br>
*<br>
of hybridising to a 2xC2H2 zinc finger nucleic acid, for example to any of SEQ ID NO 1,10,12, 14,16,18, 20, 22,24, 26, 28,30, 32, 34, 36, 38,40,41,43,45, 47 or 49. Advantageously, the methods according to the present invention may also be practised using these variants. Hybridising sequences suitable for use in the methods according to the invention may readily be determined for example by following the methods described in the Examples section by simply substituting the sequence used in the actual Example with the hybridising sequence.<br>
The term "hybridisation" as defined herein is a process wherein substantially homologous complementary nucleotide sequences anneal to each other. The hybridisation process can occur entirely in solution, i.e. both complementary nucleic acids are in solution. Tools in molecular biology relying on such a process include the polymerase chain reaction (PCR; and all methods based thereon), subtractive hybridisation, random primer extension, nuclease S1 mapping, primer extension, reverse transcription, cDNA synthesis, differential display of RNAs, and DNA sequence determination. The hybridisation process can also occur with one of the complementary nucleic acids immobilised to a matrix such as magnetic beads, Sepharose beads or any other resin. Tools in molecular biology relying on such a process include the isolation of poly (A+) mRNA. The hybridisation process can furthermore occur with one of the complementary nucleic acids immobilised to a solid support such as a nitro-cellulose or nylon membrane or immobilised by e.g. photolithography to, for example, a siliceous glass support (the latter known as nucleic acid arrays or microarrays or as nucleic acid chips). Tools in molecular biology relying on such a process include RNA and DNA gel blot analysis, colony hybridisation, plaque hybridisation, in situ hybridisation and microarray hybridisation. In order to allow hybridisation to occur, the nucleic acid molecules are generally thermally or chemically denatured to melt a double strand into two single strands and/or to remove hairpins or other secondary structures from single stranded nucleic acids. The stringency of hybridisation is influenced by conditions such as temperature, salt concentration and hybridisation buffer<br><br>
composition. High stringency conditions for hybridisation include high temperature and/or low salt concentration (salts include NaCI and Na3-citrate) and/or the inclusion of formamide in the hybridisation buffer and/or lowering the concentration of compounds such as SDS (detergent) in the hybridisation buffer and/or exclusion of compounds such as dextran sulphate or polyethylene glycol (promoting molecular crowding) from the hybridisation buffer. Conventional hybridisation conditions are described in, for example, Sambrook (2001) Molecular Cloning: a laboratory manual, 3rd Edition Cold Spring Harbor Laboratory Press, CSH, New York, but the skilled craftsman will appreciate that numerous different hybridisation conditions may be designed in function of the known or the expected homology and/or length of the nucleic acid sequence. Sufficiently low stringency hybridisation conditions are particularly preferred (at least in the first instance) to isolate nucleic acids heterologous to the DMA sequences of the invention defined supra. An example of low stringency conditions is 4-6x SSC / 0.1-0.5% w/v SDS at 37-45°C for 2-3 hours. Depending on the source and concentration of the nucleic acid involved in the hybridisation, alternative conditions of stringency may be employed, such as medium stringency conditions. Examples of medium stringency conditions include 1-4x SSC / 0.25% w/v SDS at a 45°C for 2-3 hours. An example of high stringency conditions includes 0.1 to 2x SSC / 0.1% w/v SDS at 60°C for 1-3 hours.  The skilled man will be aware of various parameters which may be altered during hybridisation and washing and which will either maintain or change the stringency conditions. The stringency conditions may start low and be progressively increased until there is provided a hybridising nucleic acid, as defined hereinabove. Elements contributing to heterology include allelism, degeneration of the genetic code and differences in preferred codon usage.<br>
Another variant 2xC2H2 zinc finger nucleic acid useful in practising the methods according to the present invention is an alternative splice variant of a nucleic acid sequence encoding a 2xC2H2 zinc finger protein. The term "alternative splice variant" as used herein encompasses variants of a nucleic acid sequence in which selected introns and/or exons have been excised, replaced or added. Such splice variants may be found in nature or may be manmade. Methods for making such splice variants are well known in the art. Splice variants suitable for use in the methods according to the invention may readily be determined for example by following the methods described in the Examples section by simply substituting the sequence used in the actual Example with the splice variant.<br>
Another variant 2xC2H2 zinc finger nucleic acid useful in practising the methods according to the present invention is an allelic variant of a nucleic acid encoding a 2xC2H2 zinc finger protein. Allelic variants exist in nature and encompassed within the methods of the present invention is the use of these natural alleles. Allelic variants also encompass Single Nucleotide<br><br>
Polymorphisms (SNPs), as well as Small Insertion/Deletion Polymorphisms (INDELs). The size of INDELs is usually less than 100 bp. SNPs and INDELs form the largest set of sequence variants in naturally occurring polymorphic strains of most organisms. Allelic variants suitable for use in the methods according to the invention may readily be determined for example by following the methods described in the Examples section by simply substituting the sequence used in the actual Example with the allelic variant.<br>
The present invention provides a method for modifying plant growth characteristics, comprising modifying expression in a plant of an alternative splice variant or expression in a plant of an allelic variant of a nucleic add encoding a 2xC2H2 zinc finger protein and/or by modifying level and/or activity in a plant of a 2xC2H2 zinc finger protein encoded by the alternative splice variant or allelic variant.<br>
Examples of variant 2xC2H2 zinc finger proteins useful in practicing the methods of the present invention are homologues, derivatives or functional fragments of a 2xC2H2 zinc finger protein.<br>
"Homologues" of a 2xC2H2 zinc finger protein encompass peptides, oligopeptides, polypeptides, proteins and enzymes having amino acid substitutions, deletions and/or insertions relative to the unmodified protein in question and having similar biological and functional activity as the unmodified protein from which they are derived. To produce such homologues, amino acids of the protein may be replaced by other amino acids having similar properties (such as similar hydrophobicity, hydrophilicity, antigenicity, propensity to form or break cc-helical structures or p-sheet structures). Conservative substitution tables are well known in the art (see for example Creighton (1984) Proteins. W.H. Freeman and Company). The homologues useful in the method according to the invention have at least in increasing order of preference 30%, 31%, 32%, 33%, 34%, 35%, 36%, 37%, 38%, 39%, 40%, 41%, 42%, 43%, 44%, 45%, 46%, 47%, 48%, 49%, 50%, 52%, 54%, 56%, 58%, 60%, 62%, 64%, 66%, 68%, 70%, 72%, 74%, 76%, 78%, 80%, 82%, 84%, 86%, 88%, 90%, 92%, 94%, 96%, 98% sequence identity or similarity to an unmodified protein.<br>
The percentage of identity may be calculated by using an alignment program well known in the art. For example, the percentage of identity may be calculated using the program GAP, or needle (EMBOSS package) or stretcher (EMBOSS package) or the program align X, as a module of the vector NTI suite 5.5 software package, using the standard parameters (for example GAP penalty 5, GAP opening penalty 15, GAP extension penalty 6.6).<br><br>
According to another embodiment of the present invention, the nucleic acid sequence useful in the methods of the present invention is a nucleic acid encoding a protein homologous to SEQ ID NO 2.<br>
Methods for the search and identification of 2xC2H2 zinc finger protein homologges, for example STZ zinc finger homologues, would be well within the realm of a person skilled in the art. Such methods, involve screening sequence databases with the sequences provided by the present invention, for example SEQ ID NO 2 (or SEQ ID NO 1), preferably in a computer readable format. This sequence information may be available in public databases, that include but are not limited to Genbank (http://www,ncbi.nlm.nih.gov/web/Genbank). the European Molecular Biology Laboratory Nucleic acid Database (EMBL) (http:/w.ebi.ac.uk/ebi-docs/embl-db.html) or versions thereof or the MIPS database (http://mips.gsf.de/). Different search algorithms and software for the alignment and comparison of sequences are well known in the art. Such methods include GAP, BESTFIT, BLAST, FASTA and TFASTA. GAP uses the algorithm of Needleman and Wunsch (J. Mol. Biol. 48:443-453,1970) to find the alignment of two complete sequences that maximises the number of matches and minimises the number of gaps. The BLAST algorithm calculates percent sequence identity and performs a statistical analysis of the similarity between the two sequences. The suite of programs referred to as BLAST programs has 5 different implementations: three designed for nucleotide sequence queries (BLASTN, BLASTX, and TBLASTX) and two designed for protein sequence queries (BLASTP and TBLASTN) (Coulson, Trends in Biotechnology: 76-80, 1994; Birren et al., GenomeAnalysis, 1: 543, 1997). The software for performing BLAST analysis is publicly available through the National Centre for Biotechnology Information.<br>
Default blast parameters to find useful homologues of any of SEQ ID NO 1, SEQ ID NO 2 or any of SEQ ID NO 10 to SEQ ID NO 50, are, when comparing nucleotide sequence G (Cost to open a gap) 5, E (Cost to extend a gap default) 2, q (Penalty for a mismatch) -3, r (Reward for a match) 1, e (Expectation value (E)) 10.0, W (Word size) 11, V (Number of one-line descriptions) 100 and B (Number of alignments to show) 100. When comparing protein sequences, the default parameters are preferably G 11, E 1, e value 10.0, W 3, V100 and B 100.<br>
The above-mentioned analyses for comparing sequences, for the calculation of sequence identity and for the search for homologues, is preferentially done with full-length sequences or within a conserved region of the sequence. Therefore, these analyses may be based on a comparison of certain regions such as conserved domains, motifs or boxes.<br><br>
The identification of such domains or motifs for examples the motif and boxes as represented by SEQ ID NO 5, 6,7, 8 and 9, would also be well within the realm of a person skilled in the art and involves for example, a computer readable format of proteins of the present invention, the use of alignment software programs and the use of publidy available information on protein domains, conserved motifs and boxes. This protein domain information is available in the PRODOM (http://www.biochem.ucl.ac.uk/bsrn/dbbrowser/ij/prodomsrchjj.html), PIR (http://pir.georgetown.edu/) or pFAM (http://pfam.wustl.edu/) database. For the identification of Zinc finger domains, such as the 2xC2H2 zinc finger domain, pFAM is preferred. Sequence analysis programs designed for motif searching may be used for identification of fragments, regions and conserved domains as mentioned above. Preferred computer programs would include but are not limited to MEME, SIGNALSCAN, and GENESCAN. A MEME algorithm (Version 3.0) may be found in the GCG package; or on the Internet site http://www.sdsc.edu/MEME/meme. SIGNALSCAN version 4.0 information is available on the Internet site http://biosci.cbs.umn.edu/software/sigscan.html. GENESCAN may be found on the Internet site http://gnomic.stanford.edu/GENESCANW.html.<br>
At present, zinc finger motifs are subdivided in more than 40 different classes as can be found in the Pfam database of protein families present at the Sanger institute (http://www.sanger.ac.Uk/Software/Pfam/browse/Z.shtml).<br>
The C2H2 zinc finger (Zf-C2H2) motif is the classical zinc finger domain. It was first recognized in the transcription factor IIIA (TFIIIA) of Xenopus (Miller at al. 1985). The domain is typically 25 to 30 amino-acid residues in length. The following pattern describes the zinc finger *X-C-X(1-5)-C-X3-*X5-*X2-H-X(3-6)-[H/C] where X can be any amino acid, and numbers in brackets indicate the number of residues. The positions marked * are those that are important for the stable folding of the zinc finger. The final position can be either his or cys, while still being a C2H2 zinc finger domain. In view of recent publications on the design of zinc finger domains it becomes feasible also to replace one or more of the Cys or His amino acids, whilst still retaining the original functionality of the C2H2 domain. The residues separating the second Cys and the first His are mainly polar and basic. The canonical C2H2 zinc finger is composed of two short beta strands followed by an alpha helix. DNA binding of the zinc finger motif is mediated by amino terminal part of the alpha helix which binds the major groove in DNA binding zinc fingers. C2H2 domains have been shown to interact with RNA, DNA and proteins. The tetracoordination of a Zinc ion by the conserved cystein and histidine residues determines the conserved tertiary structure of the motif. Conserved hydrophobic residues are commonly found at positions -2 and also at 4 amino acids after the second cystein (that participates in zinc binding) and at position three before the first histidine (that participates in zinc binding). In<br><br>
plant multi zinc finger proteins, spacing between the C2H2 domains is generally about 15 to about 65 amino acids.<br>
Thus, plant zinc finger proteips are characterized by long spacers of diverse lengths between adjacent fingers. Moreover, they are characterised by a highly conserved sequence of six amino acids, located within a putative DNA-contacting surface of each finger. Two forms of such conserved sequence are most commonly found in plant C2H2 zinc fingers, the QALGGH (SEQ ID NO 5) and the NNM/WQMH (SEQ ID NO 6). Despite the high sequence conservation of the QALGGH, some variants or the so-called 'modified type' occur in nature where one or two amino acids can have a different form, most typically the +1 "Q" can be a "G"," K" or "R" (these amino acids share the same turn-like characteristic), the +2 "A" can be "S" (both of which share the characteristic of being small amino acids) or the +3" "L can be "F" (these two amino acids are both hydrophobia). The QALGGH-motif as used herein comprises all these variants. In the NNM/WQMH motif at position 3 there is mostly an "M" or a "W. Therefore, the present invention provides a method as described hereinabove, wherein said 2xC2H2 zinc finger protein comprises a QALGGH motif. Further, The present invention provides a s described hereinabove, wherein said 2xC2H2 zinc finger protein comprises a NNM/WQMH motif.<br>
According to one embodiment of the invention, both C2H2 domains are of the same type. More preferably, both C2H2 zinc finger domains have the same conserved GALGGH or NNM/WQMH motif. According to another embodiment, each C2H2 zinc finger domain has a different conserved motif.<br>
According to one embodiment, the 2xC2H2 protein useful in the methods of the present invention is characterized by an EAR motif, which is an ERF-Associated amphiphilic repression motif. This motif has been recognized in two unrelated types of transcription factors, namely the ERF transcription factors of the AP2 type and in the zinc finger transcription factors. In the latter class, the EAR motif is generally located at the C-terminus of the protein. The pattern for the EAR motif has the conserved sequence hDLNh(X)P (SEQ ID NO 7), where "h" is a hydrophobic residue (any one of A,C,F,G,H,I,K,L,M,R,T,V,W,Y) most typically L/F/I and where "X" can be one (any amino acid) or no amino acid. A characteristic feature of the EAR motif is the alternation of hydrophilic and hydrophobic residues with the aspartic acid (D) residue being amphiphilic. Ohta et al. (The plant cell, 2001, 13, p1959-1968), which reference is cited herein by reference, previously characterized EAR motifs present in 2xC2H2 zinc finger proteins.<br><br>
Therefore, the present invention provides a method as described hereinabove, wherein the 2xC2H2 zinc finger protein comprises an EAR motif. According to one embodiment, the EAR motif is located in the C-terminal region of the protein, preferably between the second zinc finger domain and the C-terminus.<br>
According to a further embodiment, the zinc finger proteins used in the methods of the present invention have two zinc finger domains and a nuclear localization signal (B-box). A cluster of basic amino acids that resembles the B-box (Basic box) were described by Chua et af. (EMBO 1992- 11, 241-9) and were hypothesized to be a nuclear localization signal for the protein. These have been recognized in 2xC2H2 proteins (Sakamoto et al., Gene 248 (2000) 23-32). The cluster is rich in Lysine (K) and Arginine (R) residues. A consensus sequence defining the most frequent form of the B-box in 2xC2H2 genes is KR(S)KRXR (SEQ ID NO 8) where "S" at the 3rd position may be absent or present. However other variants may occur in nature that still retain the characteristic of being a charged region rich in basic amino acids. The location of the basic box is most frequently at the N-terminus of the protein, but can also occur in other locations. It has been speculated that due to its basic nature the B-box could also participate in DNA binding.<br>
Accordingly, the present invention provides a method as described hereinabove, wherein the 2xC2H2 zinc finger protein further comprises a B-box. According to one embodiment the B-box is located in the N-terminal region of the zinc finger protein. Preferably the proteins useful in the methods of the present invention have a B-box located between the N-terminus and the first zinc finger domain.<br>
According to a further embodiment, the zinc finger proteins useful in the methods of the present invention have two C2H2 zinc finger domains and an L-box. A conserved motif, named L-box, of yet unknown function has been identified in 2xC2H2 proteins and has been described previously by Sakamoto et al. (Gene 248 (2000) 23-32). The L-box is typically located at the N-terminus, between the B-box and the first C2H2 zinc finger. The L-box is represented by the sequence EXEXXAXCLXXL (SEQ ID NO 9). This region may be involved in protein-protein interactions. Zinc finger proteins lacking the L-box, may for example have serine rich regions at a similar position, which regions are putative sites for protein-protein interactions.<br>
Therefore, the present invention provides a method as described hereinabove, wherein the 2xC2H2 protein comprises an L-box.<br><br>
Particular zinc finger homologues useful in the methods of the present invention have one or more of the conserved motifs as depicted in SEQ ID NO 5, 6, 7, 8 and 9, or motifs that are 80% identical to these motifs or motifs that have conserved substitutions of amino acids. The 2xC2H2 protein as set forth in SEQ ID NO 2 comprises all the boxes as set forth in SEQ ID NO 5, 7, 8 and 9. All its paralogues and orthologues also comprise all of these boxes.<br>
Homologues of a 2xC2H2 protein as presented in SEQ ID NO 2 and isolated from Arabidopsis thaliana, that are useful in the constructs and the methods of the present invention are also identified in other plant species.<br>
Two special forms of homologue, orthologues and paralogues, are evolutionary concepts used to describe ancestral relationships of genes. The term "paralogue" relates to a gene-duplication within the genome of a species leading to paralogous genes. The term "orthotogue" relates to a homologous gene in different organisms due to ancestral relationship. The term "homologue" as used herein also encompasses paralogues and orthologues of the proteins useful in the methods according to the invention.<br>
Othologues in other plant species may easily be found by performing a so-called reciprocal blast search. Orthologous genes can be identified by querying one or more gene databases with a query gene or protein of interest (SEQ ID NO 1 or 2), using for example BLAST program. The highest-ranking subject genes that result from the search are then again subjected to a BLAST analysis, and only those subject genes that match again with the query sequence (SEQ ID NO 1 or 2) are retained as true orthologous genes. For example, to find a rice orthologue of an Arabidopsis thaliana gene, one may perform a BLASTN or TBLASTX analysis on a rice database such as (but not limited to) the Oryza sativa Nipponbare database available at the NCBI website (http://www.ncbi.nlm.nih.gov) or the genomic sequences of rice (cultivars indica orjapon/ca). In a next step, the obtained rice sequences are used in a reverse BLAST analysis using an Arabidopsis database. The results may be further refined when the resulting sequences are analysed with ClustalW and visualised in a neighbour joining tree. The method can be used to identify orthologues from many different species.<br>
The closest homologues in other species (orthologues of the protein of SEQ ID NO 2), include those from a variety of dicot and monocot plants, for example from Datisca glomerata (AF119050_1, MD26942, SEQ ID NO 10 and 11), from soybean (T09602, SCOF-1, SEQ ID NO 12 and 13), Medicago sativa (CAB77055.1, SEQ ID NO 14 and 15), from tobacco (T01985, SEQ ID NO 16 and 17) from rice, (AF332876J, AAK01713.1, SEQ ID NO 18 and 19), from petunia (BAA05079.1, SEQ ID NO 20 and 21), from wheat (S39045 and BAA03901,<br><br>
WZF1, SEQ ID NO 22 and 23), from Capsicum annum (SEQ ID NO 24 and 25), from turnip (T14408, T14409) and from sugarcane (CA279020).<br>
Close nomologues of the same species (paralogues of the protein of SEQ ID NO 2 from Arabidopsis thaliana) are described below.<br>
The MIPS database contains the sequence of the Arabidopsis thaliana genome with prediction and functional annotation of the proteins encoded. Searching this database with the STZ gene of SEQ ID NO 1 (MIPS accession number At1g27730), showed that in the Arabidopsis genome there are 2 genes encoding very close homologues of SEQ ID NO 2, At5g43170 (NMJ23683, SEQ ID NO 32 and 33) and At5g04340 (NMJ20516 SEQ ID NO 28 and 29), and Sothers with high similarity: At3g19580 (NMJ12848, SEQ ID NO 26 and 27), At5g67450 (NMJ26145, SEQ ID NO 34 and 35) and At3g49930 (NM_114853, SEQ ID NO 30-31). These genes are spread over 3 chromosomes, 1, 3 and 5. Similarly, a number of paralogues of the orthologue in Petunia have been isolated and sequenced. Advantageously, paralogues from the same species may be used in the methods of the present invention.<br>
Furthermore, a number of family members of the STZ protein of SEQ ID NO 2 have been found in Arablsopsis. The STZ gene and protein of SEQ ID NO 1 and 2 have been previously published in the database under the MIPS accession number At1g27730 or in Genbank under the accession numbers NP_174094.1, X95573 or CAA64820. Additionally, several other cDNA's, isolated from other tissues or at different developmental stages of Arabisopsis have been reported and encode the same protein as that of SEQ ID NO 2. S uch sequences sequences deposited under the Genbank accession number AY034998, NM_102538, AC12375, X95573, AY063006, X98671, X98670, or AF250336. These isolates illustrate the differential expression of the STZ gene in different plant tissues at different developmental stages. The differential regulation of these different cDNA's is reflected by the differences at their 5'UTR and the 3'UTR regions, while the encoded protein remains the same. Advantageously, the members of the same gene family as SEQ ID NO 1 or the members of the same family of any of the orthologues of SEQ ID NO 1, may be used in the methods of the present invention.<br>
Other close homologues useful in the methods of the present invention are the sequences as deposited in the public database under the following accession numbers, which sequences are herein incorporated by reference: homologues isolated from Petunia: BAA21923.1, BAA21922.1, BAA21926.1, BAA21925.1, BAA19110.1, BAA19926.1, BAA21924.1, BAA19111.1, BAA21921.1, BAA19114.1, BAA05076.1, BAA05079.1, CAA43111.1, BAA21920.1, BAA21919.1, BAA05077.1, BAA05078.1, BAA20137.1; homologues isolated<br><br>
from Arabidopsis: CAA67229.1, BAC43454.1, NPJ96054.1, AAM67193.1, NPJ99131.1, NP_188592.1, NP_201546.1, NPJ90562.1, NP_182037.1, BAC43008.1, Q8VWG3, CAC86393.1, CAC86168.1, CAC86167.1, CAC86166.1, CAB67667.1, CAC01747.1, CAB90936.1, CAB90935.1, CAB80245.1, CAB41188.1, CM18741.1, CAA67234.1, CAA67236.1. CAA67231.1, CAA67230.1, CAA67228.1, CAA67235.1, CAA67233.1, CAA67232.1, CM67229.1, CM64820.1 and homologues isolated from rice: BAB16855.1, AAO06972.1, CAC09475.1, BAB63718.1, P0683F02.21, BAB67885.1, P0031D11.19, BAB64114.1, AAK01713.1, AF332876_1, ML76091.1, BAB67879.1, P0031D11.12 and BAC15513.1.<br>
A phylogenetic tree may be constructed with all the homologues, paralogues and orthologues are defined herein above. Multiple alignment q may be made using clustal W present in the VNTi (version 5.0) program with for example Gap opening penalty 10 and Gap extention 5. For making a phylogenetic tree the Phylic software package available at http://evolution.Qenetjcs.washington.edu/Dhvlip.html may be used. Sequences clustering around SEQ ID NO 1 or SEQ ID NO 2, identify genes or proteins suitable for use in the methods of the present invention.<br>
The sequence of SEQ ID NO 2 and its rice orthologue AF332876 (SEQ ID NO 19) have 36% sequence identity when using the program Needle with the parameters Gap penalty 5 and Gap extension penalty 6. Therefore, homologues particularly useful in the methods of the present invention are homologues having 36% or more sequence identity with the 2xC2H2 zinc finger protein as presented in SEQ ID NO 2 or having 36% or more sequence identity to the closest orthologue of SEQ ID NO 2 from another species.<br>
Preferred homologues useful in practicing the methods of the present invention are plant homologues, i.e. proteins obtained from a plant nucleic acid. More preferably, the nucleic acid sequence is from a dicot, more preferably from the family Brassicaceae, further preferably from Arabidopsis thaliana.<br>
Preferably the 2xC2H2 zinc finger protein useful in the methods of the present invention belongs to the same gene family as the salt tolerant zinc finger protein (STZ) of Arabidopsis thaliana, or is a homologues thereof. The name ZAT10 can also be used to identify the STZ zinc finger protein of Arabidopsis thaliana.<br>
Another variant of a zinc finger protein useful in the methods of the present invention is a substitutional variant. The term "Substitution^ variants" of a protein refers to those variants in<br><br>
which at least one residue in an amino acid sequence has been removed and a different residue inserted in its place. Amino acid substitutions are typically of single residues, but may be clustered depending upon functional constraints placed upon the polypeptide; insertions will usually be of the order of about 1-10 amino acid residues, and deletions will range from about 1-20 residues. Preferably, amino acid substitutions comprise conservative amino acid substitutions. Particular substitutional variants of the C2H2 zinc finger protein are substitutional variants in which one or more of the conserved Cys and/or His residues is replaced, whilst retaining the same zinc finger functionality. To retain the same functionality, the residues around these conserved Cys of His residues may also be substituted.<br>
"Insertional variants" of a protein are those in which one or more amino acid residues are introduced into a predetermined site in said protein. Insertions can comprise amino-terminal and/or carboxy-terminal fusions as well as intra-sequence insertions of single or multiple amino acids. Generally, insertions within the amino acid sequence will be smaller than amino- or carboxy-terminal fusions, of the order of about 1 to 10 residues. Examples of amino-or carboxy-terminal fusion proteins or peptides include the binding domain or activation domain of a transcriptional activator as used in the yeast two-hybrid system, phage coat proteins, (histidine)6-tag, glutathione S-transferase-tag, protein A, maltose-binding protein, dihydrofolate reductase, Tag»100 epitope, c-myc epitope, FLAG®-epitope, lacZ, CMP (calmodulin-binding peptide), HA epitope, protein C epitope and VSV epitope.<br>
"Deletion variants" of a protein are characterised by the removal of one or more amino acids from the protein. Amino acid variants of a protein may readily be made using peptide synthetic techniques well known in the art, such as solid phase peptide synthesis and the like, or by recombinant DNA manipulations. Methods for the manipulation of DNA sequences to produce substitution, insertion or deletion variants of a protein are well known in the art. For example, techniques for making substitution mutations at predetermined sites in DNA are well known to those skilled in the art and include M13 mutagenesis, T7-Gen in vitro mutagenesis (USB, Cleveland, OH), QuickChange Site Directed mutagenesis (Stratagene, San Diego, CA), PCR-mediated site-directed mutagenesis or other site-directed mutagenesis protocols.<br>
The term "derivatives" refers to peptides, oligopeptides, polypeptides, proteins and enzymes which may comprise substitutions, deletions or additions of naturally and non-naturally occurring amino acid residues compared to the amino acid sequence of a naturally-occurring form of the 2xC2H2 protein such as for example the 2xC2H2 zinc finger protein as presented in SEQ ID NO 2. "Derivatives" of a 2xC2H2 zinc finger protein encompass peptides, oligopeptides, polypeptides, proteins and enzymes which may comprise naturally occurring<br><br>
altered, glycosylated, acylated or non-naturally occurring amino acid residues compared to the amino acid sequence of a naturally-occurring form of the polypeptide. A derivative may also comprise one or more non-amino acid substituents compared to the amino acid sequence from which it is derived, for example a reporter molecule or other ligand, covalently pr non-covalently bound to the amino acid sequence such as, for example, a reporter molecule which is bound to facilitate its detection, and non-naturally occurring amino acid residues relative to the amino acid sequence of a naturally-occurring protein.<br>
Another variant of a 2xC2H2 zinc finger protein useful in the methods of the present invention is an active fragment of a zinc finger protein. "Active fragments" of a 2xC2H2 zinc finger protein encompasses at least five contiguous amino acid residues of a protein, which residues retain similar biological and/or functional activity to the naturally occurring protein. For example, useful fragments comprise at least 10 contiguous amino acid residues of a 2xC2H2 zinc finger protein. Other preferred fragments are fragments of a 2xC2H2 zinc finger protein starting at the second or third or further internal methionin residues. These fragments originate from protein translation, starting at internal ATG codons. Functional fragments of a 2xC2H2 zinc finger protein useful in practising the methods of the present invention may have one, two or no C2H2 domains, without affecting its functionality in the methods of the present invention.<br>
According to a preferred feature of the present invention, enhanced or increased expression of a nucleic acid encoding a 2xC2H2 zinc finger protein is envisaged. Methods for obtaining enhanced or increased expression of genes or gene products are well documented in the art and include, for example, over-expression driven by a strong promoter, the use of transcription enhancers or translation enhancers. The term over-expression as used herein means any form of expression that is additional to the original wild-type expression level. Preferably the nucleic acid to be introduced into the plant and/or the nucleic acid that is to be overexpressed in the plant is in the sense direction with respect to the promoter to which it is operably linked. Preferably, the nucleic acid sequence represented by SED ID NO 1 is over-expressed in a plant. However, it should be clear that the applicability of the invention is not limited to use of the nucleic acid represented by SEQ ID NO 1 nor to the nucleic acid sequence encoding the amino acid sequence of SEQ ID NO 2, but that other nucleic acid sequences encoding homologues, derivatives or active fragments of SED ID NO 1 or SED ID NO 2 may be useful in the methods of the present invention. Examples of nucleic acids or proteins are provided in SEQ ID NO 10 to SEQ ID NO 50.<br>
Alternatively and/or additionally,  increased expression of a 2xC2H2 encoding gene or increased level and/or activity of a 2xC2H2 protein in a plant cell, is achieved by mutagenesis.<br><br>
For example these mutations may be responsible for altered control of the 2xC2H2 gene, resulting in more expression of the gene, relative to the wild-type gene. Mutations can also cause conformational changes in a protein, resulting in more activity and/or higher levels of the 2xC2H2 protein.<br>
Modifying gene expression (whether by a direct or indirect approach) encompasses altered transcript levels of a gene. Altered transcript levels may be sufficient to induce certain phenotypic effects, for example via the mechanism of cosuppression. Here the overall effect of introduction of a transgene is that there is less activity in the cell of the protein encoded by a native gene having homology to the introduced transgene. Therefore, according to another embodiment of the present invention, there is provided a method for modifying growth characteristics in a plant, comprising decreasing expression of a gene encoding a 2xC2H2 zinc finger protein or decreasing level and/or activity of a 2xC2H2 zinc finger protein. Examples of decreasing expression, level and/or activity of a protein in a cell are well documented in the art and include, for example, downregulation of expression by anti-sense techniques, RNAi techniques, small interference RNAs (siRNAs) and microRNA (miRNA).<br>
Another method for downregulation of gene expression or gene silencing comprises use of ribozymes, for example as described in Atkins et al. 1994 (WO 94/00012), Lenee et al. 1995 (WO 95/03404), Lutziger et al. 2000 (WO 00/00619), Prinsen et al. 1997 (WO 97/3865) and Scott et al. 1997 (WO 97/38116).<br>
Gene silencing may also be achieved by insertion mutagenesis (for example, T-DNA insertion or transposon insertion) or by gene silencing strategies as described by, among others, Angell and Baulcombe 1998 (WO 98/36083), Lowe et al. 1989 (WO 98/53083), Lederer et al. 1999 (WO 99/15682) or Wang et al. 1999 (WO 99/53050).<br>
Expression of an endogenous gene may also be reduced if it contains a mutation. Such a mutation or such a mutant gene may be isolated and introduced into the same or different plant species in order to obtain plants having modified growth characteristics. Examples of such mutants are dominant negative mutants of a 2xC2H2 zinc finger gene.<br>
Genetic constructs aimed at silencing gene expression may comprise the 2xC2H2 zinc finger nucleic acid, for example as represented by SEQ ID NO 1 (or one or more portions thereof or a sequence capable of hybridising therewith), in a sense and/or antisense orientation relative to the promoter sequence. The sense or antisense copies of at least part of the endogenous gene in the form of direct or inverted repeats may also be utilised in the methods according to<br><br>
the invention. The growth characteristics of plants may also be modified by introducing into a plant at least part of an antisense version of the nucleotide sequence represented by SEQ ID NO1.<br>
According to a further embodiment of the present invention, genetic constructs and vectors to<br>
.facilitate introduction and/or to facilitate expression of the 2xC2H2 zinc finger nucleotide<br>
sequences useful in the methods according to the invention are provided. Therefore, according<br>
to the present invention, there is provided a construct comprising:<br>
(i)	a nucleic acid capable of modifying expression of a nucleic acid encoding a<br>
2xC2H2 zinc finger protein and/or modifying  level and/or activity of a 2xC2H2 zinc finger<br>
protein;<br>
(ii)	one or more control sequence capable of driving expression of the nucleic acid<br>
sequence of (i); and optionally<br>
(iii)	a transcription termination sequence.<br>
Constructs useful in the methods according to the present invention may be constructed using recombinant DNA technology well known to persons skilled in the art. The gene constructs may be inserted into vectors, which may be commercially available, suitable for transforming into plants and suitable for expression of the gene of interest in the transformed cells. Preferably the genetic construct is a plant expression vector.<br>
The nucleic acid according to (i) is advantageously any of the nucleic acids described hereinbefore. A preferred nucleic acid is the nucleic acid represented by SEQ ID NO 1 or a variant thereof as hereinbefore defined, or is a nucleic acid sequence encoding a sequence represented by SEQ ID NO 2 or a variant as hereinbefore defined. For example such variants encode a protein as presented in any of SEQ ID NO 11,13,15, 17,19, 21, 23, 25, 27, 29, 31, 33,35, 37, 39,42,44,46,48 and 50.<br>
The terms "regulatory element" and "control sequence" are used herein interchangeably and are to be taken in a broad context to refer to regulatory nucleic acids capable of effecting expression of the sequences to which they are operably linked. Encompassed by the aforementioned terms are promoters. A "promoter" encompasses transcriptional regulatory sequences derived from a classical eukaryotic genomic gene (including the TATA box which is required for accurate transcription initiation, with or without a CCAAT box sequence) and additional regulatory elements (i.e. upstream activating sequences, enhancers and silencers) which alter gene expression in response to developmental and/or external stimuli, or in a tissue-specific manner. Also included within the term is a transcriptional regulatory sequence<br><br>
of a classical prokaryotic gene, in which case it may include a -35 box sequence and/or -10 box transcriptional regulatory sequences. The term "regulatory element" also encompasses a synthetic fusion molecule or derivative which confers, activates or enhances expression of a nucleic acid molecule in a cell, tissue or organ. The term "operably linked" as used herein refers to a functional linkage between the promoter sequence and the gene of interest, such that the promoter sequence is able to initiate transcription of the gene of interest. Preferably, the gene of interest is operably linked to a promoter in a sense direction.<br>
Advantageously, any type of promoter may be used to drive expression of the nucleic acid sequence depending on the desired outcome.<br>
Promoters useful for the present invention are described in EP 03075331.3, which promoters and sequences are incorporated herein by reference.<br>
Other examples of preferred promoters are presented in Table I (a) to (c), which promoters or derivatives thereof are useful in the methods and/or in making the constructs of the present invention. Accordingly, genetic constructs comprising of the nucleic acids of (i), for example a 2xC2H2 nucleic acid, and at least part of a promoter from Table I (a) to (c) or from EP 03075331.3, preferably, wherein said parts are operably linked, are also provided by the present invention.<br>
According to onother embodiment, the nucleic acid of (i) is operably linked to a constitutive promoter. The term "constitutive" as defined herein refers to a promoter that is expressed substantially continuously. Furthermore, preferably the constitutive promoter is a ubiquitous promoter, which is expressed in more than one, preferably in most or substantially all tissues of the plant. Preferably, the constitutive promoter to be used in the methods of the present invention, or cloned in the genetic constructs of the present invention, is a plant promoter, preferably a constitutive promoter, such as a GOS2 promoter or a promoter with similar strength and/or similar expression pattern. Preferably plant promoters derived from a plant nucleic acid are used. Alternatively, promoters operable in plant, such as promoters derived from plant pathogens are used.<br>
According to another embodiment of the invention, the nucleic acid of (i) is operably linked to a plant promoter, preferabiya tissue-preferred promoter. The term "tissue-preferred" as used herein refers to a promoter that is expressed predominantly in at least one tissue or organ. For example, the tissue-preferred promoter is a seed-preferred promoter, such as a pWS18 (Joshee et al. Plant Cell Physiol. 1998 Jan;39(1):64-72.) or a promoter of similar strength and/or similar expression pattern.<br><br>
Promoters with similar strength and/or similar expression pattern may be found by coupling the promoter to a reporter gene and checking the function of the reporter gene in different tissues of a plant. One suitable reporter gene is beta-glucuronidase and the colorimetric GUS staining to visualize the beta-glucuronidase activity in a plant tissue is well known to a person skilled in the art.<br>
Table I (a): flower preferred promoters useful in the present invention. Sequences of these promoters are described in the cited reference, which sequences are herein incorporated by reference.  (Table Remove)<br>
Table I (b): seed-preferred promoters useful in the present invention. Sequences of these promoters are described in the cited reference, which sequences are herein incorporated by reference. (Table Remove)<br>
 Table I (c): constitutive promoters useful in the present invention. Sequences of these promoters are described in the cited reference, which sequences are herein incorporated by reference. (Table Remove)<br><br>
 Optionally, one or more terminator sequences may also be used in the construct introduced into a plant. The term "terminator" encompasses a control sequence which is a DNA sequence at the end of a transcriptional unit which signals 3' processing and polyadenylation of a primary transcript and termination of transcription. Additional regulatory elements may include transcriptional as well as translational enhancers. Those skilled in the art will be aware of terminator and enhancer sequences which may be suitable for use in the invention. Such sequences would be known or may readily be obtained by a person skilled in the art.<br>
The genetic constructs of the invention may further include an origin of replication sequence which is required for maintenance and/or replication in a specific cell type. One example is when a genetic construct is required to be maintained in a bacterial cell as an episomal genetic element (e.g. plasmid or cosmid molecule). Preferred origins of replication include, but are not limited to, the f1-ori and colE1.<br>
The genetic construct may optionally comprise a selectable marker gene. As used herein, the term "selectable marker gene" includes any gene which confers a phenotype on a cell in which it is expressed to facilitate the identification and/or selection of cells which are transfected or transformed with a genetic construct of the invention. Suitable markers may be selected from markers that confer antibiotic or herbicide resistance. Cells containing the recombinant DNA will thus be able to survive in the presence of antibiotic or herbicide concentrations that kill untransformed cells. Examples of selectable marker genes include genes conferring resistance to antibiotics (such as nptll encoding neomycin phosphotransferase capable of phosphorylating neomycin and kanamycin, or hpt encoding hygromycin phosphotransferase<br>
capable of phosphorylating hygromycin), to herbicides (for example bar which provides resistance to Basta; aroA or gox providing resistance against glyphosate), or genes that provide a metabolic trait (such as manA that allows plants to use mannose as sole carbon source). Visual marker genes result in the formation of colour (for example beta-glucuronidase, GUS), luminescence (such as luciferase) or fluorescence (Green Fluorescent Protein, GFP, and derivatives thereof). Further examples of suitable selectable marker genes include the ampicillin resistance (Ampr), tetracycline resistance gene (Tor), bacterial kanamycin resistance gene (Kanr), phosphinothricin resistance gene, and the chloramphenicol acetyltransferase (CAT) gene, amongst others<br>
The present invention also encompasses plants obtainable by the methods according to the present invention. The present invention therefore provides plants obtainable by the method according to the present invention, which plants have modified growth characteristics, which plants have altered 2xC2H2 zinc finger protein level and/or activity and/or altered expression of a nucleic acid sequence encoding a 2xC2H2 zinc finger protein.<br>
Therefore, according to one aspect of the present invention, there is provided a method for the production of plants, having modified growth characteristics, comprising introducing, into a plant, a nucleic acid capable of modifying activity of a 2xC2H2 zinc finger protein and/or capable of modifying expression of a 2xC2H2 zinc-finger gene. According to a further embodiment of the present invention, there is provided a method for the production of transgenic plants having modified growth characteristics, comprising introduction and expression in a plant of a 2xC2H2 nucleic acid.<br>
More specifically, the present invention provides a method for the production of transgenic<br>
plants having modified growth characteristics, which method comprises:<br>
(i)	introducing into a plant or plant cell a 2xC2H2 zinc finger nucleic acid;<br>
(iii)	cultivating the plant cell under conditions promoting plant growth.<br>
The growth characteristic may be any of the characteristics defined hereinunder.<br>
The 2xC2H2 zinc finger nucleic acid includes all variant nucleic acids as described herein before and includes all nucleic acids encoding all variant proteins as described herein before. Cultivating the plant cell under conditions promoting plant growth, may or may not include regeneration and or growth to maturity.<br><br>
The protein itself and/or the nucleic acid itself may be introduced directly into a plant cell or into the plant itself (including introduction into a tissue, organ or any other part of the plant). According to a preferred feature of the present invention, the nucleic acid is preferably introduced into a plant by transformation. .<br>
The term "transformation" as referred to herein encompasses the transfer of an exogenous polynucleotide into a host cell, irrespective of the method used for transfer. Plant tissue capable of subsequent donal propagation, whether by organogenesis or embryogenesis, may be transformed with a genetic construct of the present invention and a whole plant regenerated therefrom. The particular tissue chosen will vary depending on the clonal propagation systems available for, and best suited to, the particular species being transformed. Exemplary tissue targets include leaf disks, pollen, embryos, cotyledons, hypocotyls, megagametophytes, callus tissue, existing meristematic tissue (e.g., apical meristem, axillary buds, and root meristems), and induced meristem tissue (e.g., cotyledon meristem and hypocotyl meristem). The polynucleotide may be transiently or stably introduced into a host cell and may be maintained non-integrated, for example, as a plasmid. Alternatively, it may be integrated into the host genome. The resulting transformed plant cell can then be used to regenerate a transformed plant in a manner known to persons skilled in the art.<br>
Transformation of a plant species is now a fairly routine technique. Advantageously, any of several transformation methods may be used to introduce the nucleic acid of interest (e.g. the 2xC2H2 nucleic acid) into a suitable ancestor cell. Transformation methods include the use of liposomes, electroporation, chemicals that increase free DNA uptake, injection of the DNA directly into the plant, particle gun bombardment, transformation using viruses or pollen and microprojection. Methods may be selected from the calcium/polyethylene glycol method for protoplasts (Krens, F.A. et al., 1882, Nature 296, 72-74; Negrutiu I. et al., June 1987, Plant Mol. Biol. 8, 363-373); electroporation of protoplasts (Shillito R.D. et al., 1985 Bio/Technol 3, 1099-1102); microinjection into plant material (Crossway A. et al., 1986, Mol. Gen Genet 202, 179-185); DNA or RNA-coated particle bombardment (Klein T.M. et al., 1987, Nature 327, 70) infection with (non-integrative) viruses and the like. A preferred transformation method is an Argobacterium mediated transformation method.<br>
Transgenic rice plants expressing a 2xC2H2 gene are preferably produced via Agrobacterium-mediated transformation using any of the well-known methods for rice transformation, such as the ones described in any of the following: published European patent application EP 1198985 A1, Aldemita and Hodges (Planta, 199, 612-617, 1996); Chan et al. (Plant Mol. Biol. 22 (3) 491-506, 1993); Hiei et al. (Plant J. 6 (2) 271-282, 1994); which disclosures are incorporated<br><br>
by reference herein as if fully set forth. In the case of com transformation, the preferred method is as described in either Ishida et al. (Nat. Biotechnol. 1996 Jun; 14(6): 745-50) or Frame et al. (Plant Physiol. 2002 May; 129(1): 13-22), which disclosures are incorporated by reference herein as if fully set forth.<br>
Generally after transformation, plant cells or cell groupings are selected for the presence of one or more markers which are encoded by plant-expressible genes co-transferred with the gene of interest, following which the transformed material is regenerated into a whole plant.<br>
Following DNA transfer and regeneration, putatively transformed plants may be evaluated, for instance using Southern analysis, for the presence of the gene of interest, copy number and/or genomic organisation. Alternatively or additionally, expression levels of the newly introduced DNA may be undertaken using Northern and/or Western analysis, both techniques being well known to persons having ordinary skill in the art.<br>
The generated transformed plants may be propagated by a variety of means, such as by clonal propagation or classical breeding techniques. For example, a first generation (or T1) transformed plant may be selfed to give homozygous second generation (or T2) transformants, and the T2 plants further propagated through classical breeding techniques.<br>
The generated transformed organisms may take a variety of forms. For example, they may be chimeras of transformed cells and non-transformed cells; clonal transformants (e.g., all cells transformed to contain the expression cassette); grafts of transformed and untransformed tissues (e.g., in plants, a transformed rootstock grafted to an untransformed scion).<br>
The present invention clearly extends to any plant cell or plant produced by any of the methods described herein, and to all plant parts and propagules thereof. The present invention extends further to encompass the progeny of a primary transformed or transfected cell, tissue, organ or whole plant that has been produced by any of the aforementioned methods, the only requirement being that progeny exhibit the same genotypic and/or phenotypic characteristic(s) as those produced in the parent by the methods according to the invention. The invention also includes host cells having modified expression and/or level and/or activity of a 2xC2H2 zinc finger protein. Such host cells for example comprise genetic constructs as mentioned above. Preferred host cells according to the invention are derived from a plant, algae, bacterium, fungus, yeast, insect or animal. The invention also extends to harvestable parts of a plant such as but not limited to seeds, leaves, fruits, flowers, petals, stamen, stem cultures, stem, rhizomes, roots, tubers, bulbs or cotton fibers.<br><br>
The term "plant" as used herein encompasses whole plants, ancestors and progeny of the plants and plant parts, including seeds, shoots, stems, roots (including tubers), and plant cells, tissues and organs. The term "plant" also encompasses suspension cultures, embryos, meristematic  regions,   callus  tissue,   leaves,   gametophytes,   sporophytes,   pollen,   and microspores. Plants that are particularly useful in the methods of the invention include all plants which belong to the superfamily Viridiplantae, in particular monocotyledonous and dicotyledonous plants including, fodder or forage legumes, ornamental plants, food crop, tree, or shrub selected from the list comprising Acacia spp., Acer spp., Actinidia spp.,Aesculus spp., Agathis australis, Albizia amara, Alsophila tricolor, Andropogon spp., Arachis spp, Areca catechu, Astelia fragrans, Astragalus deer, Baikiaea plurijuga, Betula spp., Brassica spp., Bruguiera gymnorrhiza, Burkea africana, Butea frondosa, Cadaba farinosa, Calliandra spp, Camellia sinensis,  Canna indica,  Capsicum spp.,  Cassia spp.,   Centroema pubescens, Chaenomeles   spp.,Cinnamomum   cassia,   Coffea   arabica,   Colophospermum   mopane, Coronillia varia, Cotoneaster serotina,  Crataegus spp.,  Cucumis spp.,  Cupressus spp., Cyathea dealbata,  Cydonia oblonga,  Cryptomeria japonica, Cymbopogon spp., Cynthea dealbata,  Cydonia oblonga, Dalbergia monetaria, Davallia divaricata,  Desmodium spp., Dicksonia squarosa, Diheteropogon amplectens, Dioclea spp, Dolichos spp., Dorycnium rectum, Echinochloa pyramidalis, Ehrartia spp., Eleusine coracana, Eragrestis spp., Erythrina spp., Eucalyptus spp., Euclea schimperi, Eulalia villosa, Fagopyrum spp., Feijoa sellowiana, Fragaria spp., Flemingia spp, Freycinetia banksii, Geranium thunbergii, Ginkgo biloba, Glycine javanica,   Gliriddia  spp,   Gossypium  hirsutum,   Grevillea  spp.,   Guibourtia  coleosperma, Hedysarum spp., Hemarthia altissima, Heteropogon contortus, Hordeum vulgare, Hyparrhenia rufa, Hypericum erectum, Hyperthelia dissoluta, Indigo incarnata, Iris spp., Leptarrhena pyrolifolte, Lespediza spp., Lettuca spp., Leucaena leucocephala, Loudetia simplex, Lotonus bainesii, Lotus spp., Macrotyloma axillare, Malus spp., Manihot esculenta, Medicago sativa, Metasequoia   glyptostroboides,   Musa   sapientum,   Nicotianum   spp.,   Onobrychis   spp., Omithopus spp., Oryza spp., Peltophorum africanum, Pennisetum spp., Persea gratissima, Petunia spp., Phaseolus spp., Phoenix canariensis, Phormium cookianum, Photinia spp., Picea  glauca,   Pinus   spp.,   Pisum   sativum,   Podocarpus  totara,   Pogonarthria   fleckii, Pogonarthria   squarrosa,   Populus   spp.,   Prosopis   cineraria,   Pseudotsuga   menziesii, Pterolobium   stellatum,    Pyrus   communis,    Quercus   spp.,   Rhaphiolepsis   umbellata, Rhopalostylis sapida, Rhus natalensis, Ribes grossularia, Ribes spp., Robinia pseudoacacia, Rosa spp., Rubus spp., Salix spp., Schyzachyrium sanguineum, Sciadopitys verticillata, Sequoia   sempervirens,   Sequoiadendron  giganteum,   Sorghum   bicolor,   Spinacia   spp., Sporobolus fimbriatus, Stiburus alopecuroides, Stylosanthos humilis, Tadehagi spp, Taxodium distichum, Themeda triandra, Trifolium spp., Triticum spp., Tsuga heterophylla, Vaccinium<br><br>
spp., Vicia spp.Vitis vinifera, Watsonia pyramidata, Zantedeschia aethiopica, Zea mays, amaranth, artichoke, asparagus, broccoli, brussel sprout, cabbage, canola, carrot, cauliflower, celery, collard greens, flax, kale, lentil, oilseed rape, okra, onion, potato, rice, soybean, straw, sugarbeet, sugar cane, sunflower, tomato, squash, and tea, trees and algae amongst others. According to a preferred embodiment of the present invention, the plant is a crop plant such soybean, sunflower, canola, alfalfa, rapeseed, cotton, tomato, potato or tobacco. According to another preferred embodiment of the present invention, the plant is a monocotyledonous plant, such as sugar cane, further preferably a cereal, most preferably the plant is selected from the group consisting of rice, maize, wheat, barley, millet, rye or oats.<br>
In a particular embodiment of the present invention, proteins of one plant species (for example Arabidopsis) are introduced in another plant species (for example rice). It has been shown in the present invention that plant growth characteristics are improved by introduction of a 2xC2H2 zinc finger gene or protein from a dicot into a monocot.<br>
According to a particular embodiment of the invention, there are provided methods as described above, wherein the plant is a monocot. More preferably the plant is rice or corn.<br>
Advantageously, performance of the methods according to the present invention leads to plants having modified growth characteristics.<br>
The term "growth characteristic" as used herein, preferably refers to anyone or more of, but is not limited to, yield, architecture and cycle time.<br>
The term "yield" means the amount of harvested material. For crop plants yield also means the amount of harvested material per acre of production. Depending on the crop the harvested part of the plant may be a different part or tissue of the plant, such as seed (e.g. rice, sorghum or com when grown for seed); total above-ground biomass (e.g. for com, when used as silage), root (e.g. sugarbeet), fruit (e.g. tomato), cotton fibers, or any other part of the plant which is of economic value. "Yield" also encompasses yield stability of the plants, meaning that year after year, one can obtain the same yield from the progeny of the plants, without too much nterference of external factors, such as weather conditions. "Yield" also encompasses yield aotential, which as the maximum obtainable yield.<br>
field maybe dependent on a number of yield components. The parameters for these components are well known by a person skilled in the art. For example breeders are well ware of the specific yield components and the corresponding parameters for the crop they are jiming to improve.<br><br>
For example key yield components for com include number of plants per hectare or acre, number of ears per plant, number of rows (of seeds) per ear, number of kernels per row, and thousand kernel weight. For silage corn typical parameters are the above ground biomass and energy content.<br>
Key yield components for rice include number of plants per hectare or acre, number of panicles per plant, number of spikelets per panicle, seed filling rate (number of filled seeds) and thousand kernel weight. Preferentially methods for increasing yield of rice encompass increased number of flowers per panicle and an increased number of filled seeds. The parameter of increased total number of seeds may be linked to increased number of flowers. "Yield" further encompasses typical biomass components, such as above ground parts of a plant and the root system. General biomass parameters are area and dry weight. Specific parameters for above ground biomass further encompass above ground area and plant height. Specific parameters for the root system encompass root ratio, root length and penetration depth, root branching, root hair density, root pulling resistance and aerenchyma formation.<br>
The plants of the present invention are characterized by increased number of filled seeds, increased total seed weight, increased total number of seeds and increased harvest index. Therefore the methods of the present invention are particularly favorable to be applied in cereals such as rice and corn (maize). Accordingly, a particular embodiment of the present invention relates to a method to increase yield of com, comprising modifying expression of a nucleic acid encoding a 2xC2H2 zinc finger protein.<br>
The plants of the present invention are characterized by an increase in thousand kernel weight and therefore the seed size or seed volume and/or the seed content and/or seed composition are altered by the methods of the present invention. The seeds provided by the methods of the present invention may have more nutritional value, more starch and/or more oil, possibly due to their bigger size.<br>
The plants of the present invention are characterized by more above ground area. Therefore, the methods of the present invention are particularly favorable for crops grown for their green tissue and/or grown for their above ground biomass. The methods of the present invention are particularly useful for grasses, forage crops (such as forage com (maize), clover, medicago etc.), trees, sugar cane etc.<br>
The improvement in yield as obtained by the methods of the invention, may be obtained as a result of improvement of one or more of the above mentioned yield components and/or parameters.<br><br>
The term "architecture" as used herein encompasses the appearance or morphology of a plant, including any one or more structural features or combination of structural features thereof. Such structural features include the shape, size, number, position, texture, arrangement, and pattern of any cell, tissue or organ or groups of cells, tissues or organs of a plant, including the root, leaf, shoot, stem, petiole, trichome, flower, petal, stigma, style, stamen, pollen, ovule, seed, embryo, endosperm, seed coat, aleurone, fibre, cambium, wood, heartwood, parenchyma, aerenchyma, sieve element, phloem or vascular tissue, amongst others. Particular architectural characteristics that may be modified by the methods of the present invention are increased plant height, increased number or size of stems or stalks or tillers or panicles or pedicles, increased number or size of inflorescences, increased branching of for example of tassels and ears or altered flowering characteristics. A preferred architectural characteristic that may be modified by the methods of the present invention is leaf architecture. The term "leaf architecture" as used herein comprises typical leaf characteristics such as length, width, thickness, cell number, cell size and greenness.<br>
Typically, the plants of the present invention display increased leaf surface area and /or increased leaf blade width. This trait is particularly important as it allows the plant to optimize the shape of its leaf to maximize the area used for photosynthesis. For that purpose, preferably the leaf blade is widened, but alternatively, the leaves are longer or smaller or rounder. These effects may lead to more healthy plants. Alternatively, this trait attributes aesthetic properties to the plant such as greenness and stronger leafs.<br>
"cycle time" of the plant as used herein means the time wherein a plant reaches 90% of its maximum total area. This parameter is an indication of the duration of the vegetative growth. Prolonged vegetative growth was only displayed in some of the plants according to the present invention and may be controlled by choice of the transformation event and/or by choice of the promoter driving the 2xC2H2 nucleic acid. For example this characteristic was not displayed when a seed-preferred promoter was used.<br>
Other "growth characteristics" that may be improved by the methods of the present invention are growth rate, early vigour, modified Tmid, T90 or A42 or altered growth curve.<br>
It is clear from the data as presented in the examples that one or more of the growth characteristics as defined herein above, may be combined in one plant. Alternatively, depending on the chosen transformation event and/or depending on the promoter used, one<br><br>
or more of these growth characteristics may be present or absent or more or less pronounced in the plant.<br>
The methods of the present invention may also be used to confer, stress tolerance to plants. In particular, a 2xC2H2 of the STZ type may be used to confer to a plant salt stress tolerance and/or drought stress tolerance. According to a specirfic embodiment, a tissue preferred promoter, such as a seed-preferred promoter" is used in these methods.<br>
The present invention also relates to use of a nucleic acid sequence encoding a zinc finger protein and homologues, derivatives and active fragments thereof in modifying the growth characteristics of plants, preferably in increasing yield, further preferably increasing seed yield. The present invention also relates to use of a nucleic acid sequence encoding a 2xC2H2 zinc finger protein and homologues, derivatives and active fragments thereof and to the 2xC2H2 zinc finger protein itself and to homologues, derivatives and active fragments thereof as a growth regulator. The sequences represented by SEQ ID NO 1, and portions thereof and SEQ ID NO 2, and homologues, derivatives and active fragments thereof are useful in modifying the growth characteristics of plants, as hereinbefore described. The sequences would therefore find use as growth regulators, such as herbicides or growth stimulators. The present invention also provides a composition comprising a protein represented by SEQ ID NO 2, or a homologue, derivative or active fragment thereof for the use as a growth regulator. A growth regulator is used herein as meaning a regulator that increased yield and is therefore also referred to as yield regulator.<br>
In particular, the present invention provides a yield regulating composition comprising a nucleic acid encoding a 2xC2H2 protein, and/ or comprising a 2xC2H2 protein, and/or comprising a construct as defined herein above. Such a yield regulating composition further comprises additives normally use in yield regulating compositions, such as a solvent or carrier.<br>
Conversely, the sequences according to the present invention may also be interesting targets for agrochemical compounds, such as herbicides or growth stimulators. Accordingly, the present invention encompasses use of a nucleic acid encoding a 2xC2H2 protein, of a 2xC2H2 protein and/or of a construct as defined in any of claims 20 to 22 as target for an agrochemical, such as a herbicide or a growth stimulator.<br>
The methods according to the present invention may also be practised by co-expression of a gene encoding a 2xC2H2 zinc finger protein in a plant with at least one other gene that cooperates with the gene encoding a 2xC2H2 zinc finger protein. Such a gene may be a gene encoding a target protein of the 2xC2H2 zinc finger protein. Co-expression may be effected by cloning the genes under the control of a plant expressible promoter in a plant expressible<br><br>
vector and introducing the expression vector(s) into a plant cell using ^orooactera/m-mediated plant transformation. Therefore, the methods according to the present invention may result in plants having modified growth characteristics, particularly increased yield, as described hereinbefore in combination with other economically advantageous traits, such as further yield-enhancing traits, tolerance to various stresses, traits modifying various architectural features and/or biochemical and/or physiological features.<br>
Since the plants of the present invention have excellent growth characteristics and have high<br>
yield, they are suitable for the production of enzymes, Pharmaceuticals or agrochemicals. Also,<br>
there are suitable to produce food or feed products.<br>
The invention clearly extends to enzymes, Pharmaceuticals or agrochemicals as well as food<br>
or feed products isolated from these plants.<br>
Further a nucleic acid encoding a 2xC2H2 protein, a 2xC2H2 protein and/or the constructs of<br>
the present invention may be used breeding programs aiming at the development of plants<br>
with increased yield.<br>
Particularly, the use of allelic variants as defined above in particular conventional breeding programmes, such as in marker-assisted breeding is also encompassed by the present invention; this may be in addition to their use in the methods according to the present invention. Such breeding programmes sometimes require the introduction of allelic variations in the plants by mutagenic treatment of a plant. One suitable mutagenic method is EMS mutagenesis. Identification of allelic variants then takes place by, for example, PCR. This is followed by a selection step for selection of superior allelic variants of the sequence in question and which give rise to altered growth characteristics in a plant. Selection is typically carried out by monitoring growth performance of plants containing different allelic variants of the sequence in question, for example, SEQ ID NO 1. Monitoring growth performance may be done in a greenhouse or in the field. Further optional steps include crossing plants in which the superior allelic variant was identified with another plant. This could be used, for example, to make a combination of interesting phenotypic features<br>
According to another type of breeding programme a DNA marker is identified which may be genetically linked to a gene capable of modifying expression of a nucleic acid encoding a 2xC2H2 zinc finger protein in a plant, which gene may be a gene encoding the 2xC2H2 zinc finger protein itself or any other gene which may directly or indirectly influence expression of the gene encoding a 2xC2H2 zinc finger protein and/or activity of the 2xC2H2 zinc finger protein itself. This DNA marker may then be used in breeding programs to select plants having altered growth characteristics.<br><br>
The methods according to the present invention may also be practised by introducing into a plant at least a part of a (natural or artificial) chromosome (such as a Bacterial Artificial Chromosome (BAG)), which chromosome contains at least a gene encoding a 2xC2H2 zinc finger protein, optionally together with one or more related gene family members. Therefore, according to a further aspect of the present invention, there is provided a method for modifying growth characteristics of plants by expressing in a plant at least a part of a chromosome comprising at least a gene encoding a 2xC2H2 zinc finger protein.<br>
The present invention will now be described with reference to the following figures in which:<br>
Fig. 1 is a map of an expression vector for the expression in plants of a 2xC2H2zinc finger protein under the control of a GOS2 promoter. CDS1536 is the internal code for the Arabidopsis thaliana salt tolerant zinc finger (STZ) protein cDNA. The zinc finger protein expression cassette has a GOS2 promoter and a double terminator sequence (T-zein and T-rbcS-deltaGA) located within the left border (LB repeat) and the right border (RB repeat) of the Ti plasmid. Cloned within these T-borders are also a screenable marker and a selectable marker, each under the control of a constitutive promoter (Prom), followed by a terminator sequence (poly a and t-NOS). Furthermore, this vector also contains an origin of replication (pBR322 (ori + bom)) for bacterial replication and a selectable marker (Sm/SpR) for bacterial selection.<br>
Fig. 2A shows digital images from a T1 positive line transformed with an STZ zinc finger transgene under control of a GOS2 promoter and Fig. 2B shows digital images of corresponding nullizygotes plants.<br>
Fig. 3 lists sequences useful in the methods of the present invention. SEQ ID NO 1 is an STZ encoding nucleic acid isolated from Arabidopsis thaliana; the start and the stop codon are highlighted in bold. SEQ ID NO 2 is the STZ protein sequence encoded by SEQ ID NO 1. In the STZ protein the nuclear localization signal also called the KRS motif or B-box is annotated (bold, italics, underlined), as well as the L-box (bold, underlined), the EAR motif (bold, italics), and the two C2H2 zinc finger domains with QALGGH motif (bold and boxed). SEQ ID NO 10 to SEQ ID NO 25 provides the sequences of various orthologs of the Arabidopsis thaliana STZ protein from other plant species. SEQ ID NO 26 to SEQ ID NO 35 provides the sequences of various paralogs (from Arabidopsis) of the STZ protein. SEQ ID NO 36 to SEQ ID NO 50 provides the sequences of related 2xC2H2 genes and proteins useful in the methods of the present invention.<br><br>
Fig. 4 is a photograph of T3 plants grown in a greenhouse (A) or in a field (B). The photograph<br>
shows yield increase (especially in aboveground biomass and plant height) in subsequent<br>
generations of STZ transformed plants.	-<br>
Fig. 5 shows the binary vector for expression in Oryza sativa of the Arabidopsis thaliana STZ gene (CDS1536) under the control of a seed preferred WSI18 promoter (PRO0151). This vector contains a T-DNA derived from the Ti Plasmid, limited by a left border (LB repeat, LB Ti C58) and a right border (RB repeat, RB Ti C58)).<br>
The zinc finger protein expression cassette has a WSI18 (PRO0151) promoter and a double terminator sequence (T-zein and T-rbcS-deltaGA) located within the left border (LB repeat) and the right border (RB repeat) of the Ti plasmid. Cloned within these T-borders are also a screenable marker and a selectable marker, each under the control of a constitutive promoter (Prom), followed by a terminator sequence (poly a and t-NIOS). Furthermore, this vector also contains an origin of replication (pBR322 (ori + bom)) for bacterial replication and a selectable marker (Sm/SpR) for bacterial selection.<br>
Examples<br>
The present invention will now be described with reference to the following examples, which are by way of illustration alone.<br>
DMA Manipulation<br>
Unless otherwise stated, recombinant DNA techniques are performed according to standard protocols described in Sambrook (2001) Molecular Cloning: a laboratory manual, 3rd Edition Cold Spring Harbor Laboratory Press, CSH, New York or in Volumes 1 and 2 of Ausubel et a/. (1988), Current Protocols in Molecular Biology, Current Protocols. Standard materials and methods for plant molecular work are described in Plant Molecular Biology Labfase (1993) by R.D.D. Croy, published by BIOS Scientific Publications Ltd (UK) and Blackwell Scientific Publications (UK).<br>
Example 1: Gene Cloning<br>
A gene encoding an STZ protein was amplified by PCR from an Arabidopsis thaliana seedling cDNA library (Invitrogen, Paisley, UK). After reverse transcription of RNA extracted from seedlings, the cDNAs were cloned into pCMV Sport 6.0. Average insert size of the bank was 1.5 kb, and original number of clones was of 1.59x107 cfu. Original titer was determined to be 9.6x105 cfu/ml, after first amplification of 6x1011 cfu/ml. After plasmid extraction, 200 ng of template was used in a 50ul PCR mix. Sequences of the primers used for PCR amplification<br><br>
were, including the attB sites for Gateway recombination (in bold) were PRM3204 (sense, start codon in italics) 5' GGGGACAAGTTTGTACAAAAAAGCAGGCTTCACA4TGGCG CTCGAGGCTC 3' (SEQ ID NO 3) and PRM3205 (reverse, complementary stop codon in italics) 5' GGGGACCACTTTGTACAAGAAAGCTGGGTAAT7TCC7T4MGTTGAAGTTTGA 3' (SEQ ID NO 4).<br>
PCR was performed using Hifi Taq DMA polymerase in standard conditions. The PCR fragment (CDS1536) was amplified and purified using standard methods. The first step of the Gateway procedure, the BP reaction, was then performed, during which the PCR fragment was recombined In vivo with the pDONR pfasmid to produce, according to Gateway terminology, an "entry clone", p3359. PDONR was purchased from Invitrogen, as part of the Gateway technology.<br>
Example 2: Vector construction for rice transformation with pGOS2::/lfSTZ<br>
The entry done p3359 was subsequently used in an LR reaction with p0640, a destination vector used for rice transformation. This vector contains as functional elements within the T-DNA borders a plant selectable marker and a Gateway cassette intended for LR in vivo recombination with the sequence of interest already cloned in the donor vector. Upstream of this Gateway cassette lies the rice GOS2 promoter for constitutive expression of the zinc finger gene (De Pater et a/., Plant J. 2 (6) 837-844,1992). After the recombination step, the resulting expression vector with the expression cassette CD4398 (Figure 1) was transformed into Agrobacterium strain LBA4404 and subsequently into plants. Transformed rice plants were allowed to grow and then examined for various parameters as described in Example 3.<br>
Example 3: Evaluation of TO, T1 and T2 transgenic rice plants transformed with pGOS2::AfSTZ (CD4398)<br>
Approximately 15 to 20 independent TO transformants were generated. The primary TO transformants were transferred from tissue culture chambers to a greenhouse for growing and harvest of T1 seed. Six events of which the T1 progeny segregated 3:1 for presence/absence of the transgene were retained. For each of these events, approximately 10 T1 seedlings containing the transgene (hetero- and homo-zygotes), and approximately 10 T1 seedlings lacking the transgene (nullizygotes), were selected by PCR. Based on the results of the T1 evaluation three events were chosen, for further characterisation in the T2 generation, one event being very positive for a number of parameters, a second event being positive for a number of parameters, but less pronounced, and a third event being neutral. Seed batches from the positive plants (both hetero- and homozygotes) in T1, were screened by monitoring marker expression. For each chosen event, the heterozygote seed batches were then selected<br><br>
for T2 evaluation. An equal number of positives and negatives within each seed batch were transplanted for evaluation in the greenhouse (i.e., for each event 40 plants were grown of which there were about 20 positives for the transgene and about 20 negative). Therefore, the total number for the three events amounted to 120 plants for evaluation in the T2 generation.<br>
T1 and T2 plants were transferred to the greenhouse and evaluated for vegetative growth parameters and seed parameters, as described hereunder.<br>
(I) Statistical analysts of phenotvplc characteristics<br>
A two factor ANOVA (analyses of variance) was used as statistical model for the overall evaluation of plant phenotypic characteristics. An F-test was carried out on all the parameters measured, for all the plants of all the events transformed with the gene of interest. The F-test was carried out to check for an effect of the gene over all the transformation events and to verify an overall effect of the gene or "global gene effect". Significant data, as determined by the value of the f-test, indicates a "gene" effect, meaning that the phenotype observed is caused by more than the presence or position of the gene. In case of the F-test, the threshold for significance for a global gene effect is set at 5% probability level.<br>
To check for an effect of the genes within an event, i.e., for a line-specific effect, a t-test was performed within each event using data sets from the transgenic plants and the corresponding null plants. "Null plants" or "Null segregants"are the plants treated in the same way as the transgenic plant, but from which the transgene has segregated. Null plants can also be described as homozygous negative transformant plants. The threshold for significance for the t-test is set at 10% probability level. Within one population of transformation events, some events may be under or above this t-test threshold. This is based on the hypothesis that a gene might only have an effect in certain positions in the genome, and that the occurrence of this positron-dependent effect is not uncommon. This kind of gene effect may also be referred to as a "line effect of a gene". The p- value is obtained by comparing the t-value to the t-distribution or alternatively, by comparing the F-value to the f-distribution. The p- value stand for the probability that the null hypothesis (null hypothesis being "there is no effect of the transgene") is correct.<br>
flh Vegetative growth measurements<br>
The selected plants were grown in a greenhouse. Each plant received a unique barcode label to link unambiguously the phenotyping data to the corresponding plant. The selected plants were grown on soil in 10 cm diameter pots under the following environmental settings: photoperiod= 11.5 h, daylight intensity= 30,000 lux or more, daytime temperature= 28°C or<br><br>
higher, night time temperature= 22°C, relative humidity= 60-70%. Transgenic plants and the corresponding nullizygotes were grown side-by-side at random positions. From the stage of sowing until the stage of maturity (which is the stage were there is no more increase in biomass) the plants were, passed weekly through a digital imaging cabinet (examples of pictures are shown in Figures 2A and 2B). At each time point digital images (2048x1536 pixels, 16 million colours) were taken of each plant from at least 6 different angles. The parameters described below were derived in an automated way from the digital images using image analysis software.<br>
(a) Aboveground area<br>
Plant above ground area was determined by counting the total number of pixels from aboveground plant parts discriminated from the background. This value was averaged for the pictures taken on the same time point from the different angles and was converted to a physical surface value expressed in square mm by calibration. Experiments show that the aboveground plant area measured this way correlates with the biomass of plant parts above ground.<br>
Results of the maximum above ground area values of the lines selected for T2 evaluation are summarized in Table 1. The plants of the best performing line showed an increase in biomass of 34 % , compared to the nullizygotes.<br>
When an F-test was carried out on all the plants of all the T2 events it became clear that the transgenic plants show a significant increase in above ground area, in average an increase of approximately 18%. A significant increase in above ground biomass is also displayed by STZ transformed plants grown under field conditions (see figure 4).<br>
Table 1: Aboveground area of STZ transgenic T2 plants. Each row corresponds to one event, for which the average maximum aboveground area (expressed in mm2) was determined for the transgenics (TR) and the null plants (null). The difference in absolute values between the transgenic population and the nullizygotes of each event are presented (dif.) as well as the percentage of difference between the two populations (% dif). P stands for the probability produced by the t-test for each event. The last row presents the average numbers calculated from all the events. Here the p-value is produced by the F-test.<br>
(Table Remove)<br><br>
 (b) Plant height measurements<br>
Plant height was determined by the distance between the horizontal lines going through the upper pot edge and the uppermost pixel corresponding to a plant part above ground. This value was averaged for the pictures taken on the same time point from the different angles and was converted, by calibration, to a physical distance expressed in mm. Experiments showed that plant height measured (his way correlate with plant height measured manually with a ruler.<br>
The increase in plant height was displayed very clearly in STZ transformed plants when measured at the end of the vegetative growth (see figure 4A). Also, this parameter, was displayed by STZ transformed plants when grown in the field (see figure 4B) at the time of harvest<br>
(c) Total area cycle time measurements<br>
Plants were imaged weekly along the complete cell cycle and the maximum total area of the plants was determined as mentioned above. Total Area Cycle Time is the time when a plant reaches 90% of its maximum total area. This parameter is an indication of the duration of the vegetative growth.<br>
Only in some transgenic lines there was an effect on cycle time. These few lines showed a prolonged vegetative growth.<br>
(IN) Measurement of seed-related parameters<br>
The mature primary panicles were harvested, bagged, barcode-labelled and then dried for three days in the oven at 37°C. The panicles were then threshed and all the seeds collected. The filled husks were separated from the empty ones using an air-blowing device. After separation, both seed lots were then counted using a commercially available counting machine. The empty husks were discarded. The filled husks were weighed on an analytical balance and the cross-sectional area of the seeds was measured using digital imaging. This procedure resulted in the set of seed-related parameters described below.<br>
(a) Total number of filled seeds per plant<br>
The number of filled seeds was determined by counting the number of filled husks that remained after the separation step.<br>
Total numbers of filled seeds per plant are summarized in Table 2. The t-test shows that for two events, transgenlc plants produce 106% and 130% more filled seeds than the nullizygotes.<br>
Table 2: Number of filled seeds of STZ transgenic T2 plants. Each row corresponds to one event, for which the average number of filled seeds was determined for the transgenics (TR) and the null plants (null). The difference in absolute values between the transgenic population and the nullizygotes of each event are presented (dif.) as well as the percentage of difference between the two populations (% dif). P stands for the probability produced by the t-test for each event. The last row presents the average numbers calculated from all the events. Here the p-value is produced by the F-test. (Table Remove)<br><br>
 (b) Total seed weight per plant<br>
The total seed weight was measured by weighing all filled husks harvested from a plant.<br>
The total seed weight values of STZ transformed plants are summarized in Table 3. STZ transgenic plants produce significantly more seed weight than the corresponding nullizygotes. The difference in seed weight of the transgenics may be as high as 138% or higher.<br>
Table 3: Total seed weight per plant of STZ transgenic T2 plants. Each row corresponds to one event, for which the average total seed weigh (in gram) was determined for the transgenics (TR) and the null plants (null). The difference in absolute values between the transgenic population and the nullizygotes of each event are presented (dif.) as well as the percentage of difference between the two populations (% dif). P stands for the probability produced by the t-test for each event. The last row presents the average numbers calculated from all the events. Here the p-value is produced by the F-test. (Table Remove)<br><br>
 (c) Harvest Index<br>
The harvest index in the present invention is defined as the ratio between the total seed yield and the above ground area (mmz), multiplied by a factor 106.<br>
The harvest index values of the STZ-transgenic plants are summarized in Table 4. STZ transgenic plants have a significant increase in harvest index. The increase in harvest index of the transgenic plants may be as high as 66%, when compared to the corresponding nullizygotes.<br>
Table 4: Harvest index of STZ transgenic T2 plants. Each row corresponds to one event, for which the average harvest index was determined for the transgenics (TR) and the null plants (null). The difference in absolute values between the transgenic population and the nullizygotes of each event are presented (dif.) as well as the percentage of difference between the two populations (% dif). P stands for the probability produced by the t-test for each event. The last row presents the average numbers calculated from all the events. Here the p-value is produced by the F-test. (Table Remove)<br><br>
 (d) Thousand kernel welgth (TKW) of plants<br>
Thousand Kernel Weight (TKW) is a parameter extrapolated from the number of filled seeds counted, and their total weight.<br>
The weight values of thousand kernels of STZ transgenic plants are presented in Table 5. STZ transgenic plants have increased thousand kernel weight. The increase of TKW of transgenic plants may be as high as 6% when compared to the corresponding nullizygotes.<br><br>
Table 5: Thousand kernel weight of STZ transgenic 72 plants. Each row corresponds to one event, for which the average TKW was determined for the transgenics (TR) and the null plants (null). The difference in absolute values between the transgenic population and the nullizygotes of each event are presented (dif.) as well as the percentage of difference between the two populations (% dif). P stands for the probability produced by the t-test for each event. The last row presents the average numbers calculated from all the events. Here the p-value is produced by the F-test. (Table Remove)<br><br><br>
 (e) Total number of seeds<br>
The total number of seeds per plant was measured by counting the number of husks harvested from a plant.<br>
The total numbers of seeds per plant are summarized in Table 6. STZ transformed plants have an increase in total number of seeds. The increase of total number of seeds may be as high as 68%, when compared to the corresponding nullizygotes.<br>
Table 6: Total number of seeds of STZ transgenic 72 plants. Each row corresponds to one event, for which the average total number of seeds was determined for the transgenics (TR) and the null plants (null). The difference in absolute values between the transgenic population and the nullizygotes of each event are presented (dif.) as well as the percentage of difference between the two populations (% dif). P stands for the probability produced by the t-test for each event. The last row presents the average numbers calculated from all the events. Here the p-value is produced by the F-test. (Table Remove)<br>
Conclusion<br>
It may be concluded that vegetative growth is increased in the STZ transgenic plants when<br>
compared to the control non-transgenic plants, as reflected by parameters such as above<br>
ground area, where the increase is above 20 %.   This effect may be attributed to the<br>
expression of the STZ gene in the transgenic plants.   Additionally, in some transformation<br>
events, the length of the vegetative growth is altered in the STZ transgenic plants. For those<br>
transformation events in which this effect occurs, in average the vegetative growth was<br>
prolonged with about 4 to 6 days, under the conditions tested.<br>
Furthermore, yield was increased in STZ transgenic plants. Several seed parameters reflect<br>
this yield increase. The total number of seeds harvested was at least 100% higher in the<br>
transgenics than in the control plants, for those events showing a differential.   For these<br>
events, there was also an increase in the total number of seeds of the transgenics, which<br>
increase was higher than 30 %. Seed filling in those transgenics was greatly improved,<br>
reaching differences above 100% in the number of filled seeds.<br>
Seed of the transgenic plants were also heavier, and probably bigger, as suggested by the<br>
higher values obtained for the thousand kernel weigh. The TKW parameter is a very stable<br>
parameter in rice cultivars, such as nipponbare, and in the growth conditions here used. This<br>
means that this parameter is not easily influenced and makes it an important yield parameter.<br>
Therefore a TKW increase of 6 % represents a significantly increase in yield.<br>
Harvest index, another important yield parameter, was increased in the transgenic plants with<br>
more than 50 %.<br>
In summary, based on the evaluation of STZ transgenic plants in the T1, T2 and further<br>
generations, it may be concluded that the presence of an STZ transgene, has a positive effect<br>
on the size of the plant and/or its organs, as well as a positive effect on the final yield<br>
harvested.<br>
nil) Root growth measurements<br>
Transgenic plants are grown next to their corresponding non-transgenic null segregant in<br>
transparent pots. In average, for each construct comprising a particular promoter-2xC2H2<br>
combination, a minimum of 5 independent transformation events are evaluated for root growth,<br>
root development and root architecture. Typically, per transformation event, 10 transgenics are<br>
compared to 10 nullizygotes. Root pictures are taken weekly during plant growth. The pictures<br>
are processed and analyzed to extract the values for the root parametes as detailed below.<br>
Statistical analysis as described above are applied to these data.<br>
a)	Root Area<br>
Total root area is calculated from the summed number of pixels of each root images. A positive linear correlation between root area and dry weight and root biomass of the root has been previously established by similar experiments. Therefore, root area is a good approximation for root biomass.<br>
b)	Root Length<br>
The total perimeter of the roots of a plant is calculated as the sum of the perimeter of all roots in the images. A linear correlation between this measurement and root length has been previously established. Thus, root length is extrapolated from the total root perimeter.<br>
c)	Root Width<br>
Average root width of a plant is expressed as the ratio between the Root Area and the Root Length.<br>
STZ transgenic plants of the invention show a superior performance when compared to control pants. Transgenic plants are altered in one or more the root parameters detailed above. In particular the transgenic have increased root biomass, for example due to increased root dry weight or area, and/or increased root length and/or increased root width.<br>
Example 4: Leaf Blade Width Measurement<br>
Leaves of STZ transgenic plants appeared bigger and wider when compared to the corresponding control non-transgenic plants. To quantify the increase in leaf width, leaf blade width (length of transversal axe) of the flag leaf was measured with a ruler at the widest point of the leaf, which is approximately at half of the length, in plants that have reached the end of the vegetative growth phase. The results shown in the Table 7, indicate that the increase in the leaf blade width in at least the event here measured was around 15 % when compared to the corresponding nullizygote.<br>
Table 7: Leaf blade width of STZ transgenic T2 plants. The average leaf blade width was determined for the transgenics (TR) and the null plants (null) of the selected event. The difference in absolute values between the transgenic population and the nullizygotes of the event is presented (dif.) as well as the percentage of difference between the two populations (% dif). P stands for the probability produced by the t-test. (Table Remove)<br><br>
 Example 5: Vector construction for rice transformation with pWSI18::AtSTZ<br>
Vector construction for transformation with the pWSI18 (PRO0151) - AtSTZ (CDS1536) cassette was carried out essentially as in example 2. The entry clone p3359, described earlier, was subsequently used in an LR reaction with p05653, a destination vector used for rice transformation. This destination vector contains as functional elements within the T-DNA borders a plant selectable marker and a Gateway cassette intended for LR in vivo recombination with the sequence of interest already cloned in the donor vector. A WSI18 promoter for seed preferred expression (PRO0151) is located upstream of this Gateway cassette. After the recombination step, the resulting expression vector with the expression cassette CD4398 (Figure 5) was transformed into Agrobacterium strain LBA4404 and subsequently this vector was transformed to Oryza sativa plants. Transformed rice plants were allowed to grow and then examined for various parameters as described in example 3.<br>
Example 6: Evaluation of TO and T1 transgenlc rice plants transformed with the seed preferred expression cassette pWS118::AtSTZ (CD4398)<br>
Preparations of calli and of the Agrobacterium tumefaciens strain containing the expression vector with the CD4398 expression cassette, were carried out as described in example 3, as were the calli transformation and plant regeneration.<br>
Approximately 15 to 20 independent TO rice transformants were generated. The primary transformants were transferred from tissue culture chambers to a greenhouse for growing and harvest of T1 seed. Events, of which the T1 progeny segregated 3:1 for presence/absence of the transgene, were retained. For each of these events, approximately 10 T1 seedlings containing the transgene (hetero- and homo-zygotes), and approximately 10 T1 seedlings lacking the transgene (nullizygotes), were selected by monitoring marker expression. Transgenic plantlets were grown next to control nullyzygotes, seeds were harvested and thousand kernel weight determined as previously described.<br>
Transformed plants comprising the expression cassette CD8490 (seed preferred pWSI18::STZ), had a normal and healthy appearance and were harvested at the same time as the control plants. The seeds harvested from the transgenic plants had an increase in<br><br>
thousand kernel weight when compared to the control plants. As shown in Table 8 increase in thousand kernel weight was above 10%.<br>
Table 8: Thousand kernel weight ofSTZ transgenic T1 plants. The average I thousand kernel weight was determined for the transgenics (TR) and the null plants (null) of the selected event. The difference in absolute values between the transgenic population and the nullizygotes of the event is presented (dif.) as well as the percentage of difference between the two populations (% dif). P stands for the probability produced by the t-test.<br>
(Table Remove)<br><br>
Example 7: Cloning, transformation and evaluation of other 2xC2H2 encoding genes.<br>
In Table 9 an overview is given of constructs with STZ or other 2xC2H2 zinc finger proteins, under control of various promoters, which constructs are made for use in the methods of the present invention. The coding regions of the 2xC2H2 genes to be cloned (GOI, Gene of Interest) are amplified by PCR from cDNA , following the protocol as in Example 1. Specific primers for each 2xC2H2 gene were designed at the start and stop codons of the gene sequence as present in the public database under the accession number as indicated in Table 3. These cloned sequences are also herein incorporated under the SEQ ID NO number as Ttentioned in the table. Moreover, the isolated PCR fragments were also given a unique CDS lumber.<br>
The PCR fragment with a 2xC2H2 gene is then cloned under the control of a particular iromoter. Different combinations for different genes are made (see Table 9). Chimeric constructs are made and CD numbers represent bacterial strains carrying the chimeric construct Corresponding transgenic plants are obtained by transforming the plants with the ;himeric constructs, following the protocols as mentioned herein before. Evaluation of the ransgenic events reveals an increase in yield, and increase in leaf surface area and/or an ncrease in duration of vegetative growth in the transgenic plants when compared to the control ion~transgenic plants.<br>
CD-070-PCT<br>
Table 9: examples of 2xC2H2 chimeric constructs useful for (Table Remove)<br><br>
Example 8: use of the invention in com.<br>
The methods of the invention described herein are also used in maize. To this aim, an STZ encoding gene, for example a maize or other STZ ortholog, is cloned under control of a promoter operable in maize, in a plant transformation vector suitable for Agrobacterium-mediated corn transformation. Methods to use for com transformation have been described in literature (Ishida et al., Nat Biotechnol. 1996 Jun;14(6):745-50; Frame et al., Plant Physiol. 2002 May; 129(1): 13-22).<br>
Transgenic plants made by these methods are grown in the greenhouse for T1 seed production. Inheritability and copy number of the transgene are checked by quantitative realtime PCR and Southern blot analysis and expression levels of the transgene are determined by reverse PCR and Northern analysis. Transgenic lines with single copy insertions of the transgene and with varying levels of transgene expression are selected for T2 seed production.<br>
Progeny seeds are germinated and grown in the greenhouse in conditions well adapted for maize (16:8 photoperiod, 26-28°C daytime temperature and 22-24°C nighttime temperature)<br>
as well under water-deficient, nitrogen-deficient, and excess NaCI conditions. Null segregants from the same parental line, as well as wild type plants of the same cultivar are used as controls. The progeny plants resulting from the selfing or the crosses are evaluated on different biomass and developmental parameters, including, plant height, stalk/stem thickness, stem size, number of leaves, total above ground area, leaf greenness, time to maturity, time to silking, flowering time, time to flower, ear number, ear length, row number, kernel number, kernel size , kernel oil content, grain maturity, harvesting time. The seeds of these lines are also checked on various parameters, such as grain size, total grain yield per plant, and grain quality (starch content, protein content and oil content).<br>
Lines that are most significantly improved compared to corresponding control lines are selected for further field-testing and marker-assisted breeding, with the objective of transferring the field-validated transgenic traits into commercial germplasm. The testing of maize for growth and yield-related parameters in the field is conducted using well-established protocols. The com plants are particularly evaluated on yield parameters, such as for example, amount of plants per acre, amount of ears per plant, amount of rows per ear, amount of seeds per row and TKW. Subsequent improvements for introgressing specific loci (such as transgene containing loci) from one germplasm into another is also conducted using well-established protocols.<br><br>
Claims<br>
1.	Method for increasing plant yield relative to corresponding wild type plants, comprising<br>
modifying expression in a plant of a nucleic acid sequence encoding a 2xC2H2 7inc finger<br>
protein and/or modifying in a plant level and/or activity of a 2xC2H2 zinc finger protein.<br>
2.	Method for increasing leaf surface area relative to corresponding wild type plants,<br>
comprising modifying expression in a plant of a nucleic acid sequence encoding a 2xC2H2<br>
zinc finger protein and/or modifying in a plant level and/or activity of a 2xC2H2 zinc finger<br>
protein.<br>
3.	Method for prolonging vegetative growth phase of a plant relative to corresponding wild<br>
type plants, comprising modifying expression in a plant of a nucleic acid sequence<br>
encoding a 2xC2H2 zinc finger protein and/or modifying in a plant level and/or activity of a<br>
2xC2H2 zinc finger protein.<br>
4.	Method according to any of claims 1 to 3, wherein said modifying expression, level and/or<br>
activity is effected by recombinant means and/or chemical means.<br>
5.	Method according to any of claims 1 to 4, wherein said 2xC2H2 zinc finger protein<br>
comprises a QALGGH motif.<br>
6.	Method according to any of claims 1 to 4, wherein said 2xC2H2 zinc finger protein<br>
comprises a NNM(W)QMH motif.<br>
7.	Method according to any of claims 1 to 6, wherein said 2xC2H2 zinc finger protein<br>
comprises an EAR motif.<br>
8.	Method according to any of claims 1 to 7, wherein said 2xC2H2 zinc finger protein further<br>
comprises a B-box.<br>
9.	Method according to any of claims 1 to 8, wherein said 2xC2H2 zinc finger protein further<br>
comprises an L-box.<br>
10.	Method according to any of claims 1 to 9, wherein said 2xC2H2 zinc finger protein is<br>
derived from a dicotyledonous plant, preferably from the family Brassicaceae, further<br>
preferably from Arabidopsis thaliana, more preferably the nucleic acid is as represented by<br><br>
SEQ ID NO 2 or a homologue, derivative or active fragment thereof and/or wherein said nucleic acid is as represented by SED ID NO 1 or a portion thereof or sequences capable of hybridising therewith.<br>
11.	Method according to claim 10, wherein said homologue, derivative or active fragment has,<br>
in increasing order of preference, at least 30%, 31%, 32%, 33%, 34%, 35%, 36%, 37%,<br>
38%, 39%, 40%, 41%, 42%, 43%, 44%, 45%, 46%, 47%, 48%, 49%, 50%, 52%, 54%,<br>
56%, 58%, 60%, 62%, 64%, 66%, 68%, 70%, 72%, 74%, 76%, 78%, 80%, 82%, 84%,<br>
86%, 88%, 90%, 92%, 94%, 96%, 98% sequence identity with the sequence of SEQ ID NO<br>
2.<br>
12.	Method according to any of claims 1 to 11, wherein said plant is a monocot<br>
13.	Method according to any of claims 1 to 12, wherein said modifying expression is effected<br>
by introducing into a plant a nucleic acid capable of modifying expression of a gene<br>
encoding a 2xC2H2 zinc finger protein and/or capable of modifying level and/or activity of a<br>
2xC2H2 zinc finger protein.<br>
14.	Method according to claim 13, wherein said nucleic acid capable of modifying expression is<br>
a nucleic acid encoding a 2xC2H2 protein, such as a 2xC2H2 protein as defined in any of<br>
claims 5 to 11.<br>
15.	Method according to claims 13 or 14, wherein said nucleic acid introduced into a plant is an<br>
alternative splice variant of a nucleic acid as defined in claim 14.<br>
16.	Method according to claims 13 or 15, wherein said nucleic acid introduced into a plant is an<br>
allelic variant of a nucleic acid as defined in claim 14.<br>
17.	Method according to claims 13 or 16, wherein said nucleic acid introduced into a plant is<br>
comprised on at least part of a chromosome.<br>
18.	Method according to any of claims 1 to 17, wherein said modifying expression comprises<br>
increased expression.<br>
19.	Method according to any of claims 1 to 18, wherein expression of said nucleic acid is<br>
driven by a plant promoter, preferably a constitutive promoter, such as a GOS2 promoter.<br><br>
20.	Method according to any of claims 1 to 18, wherein expression of said nucleic acid is<br>
driven by a plant promoter, preferably a tissue preferred promoter, such as seed-preferred<br>
promoter.<br>
21.	Method according to any of claims 1 to 20, wherein said increased yield comprises<br>
increased above ground biomass.<br>
22.	Method according to any of claim 1 to 20, wherein said increased yield comprises<br>
increased seed yield.<br>
23.	Method according to any of claim 1 to 20, wherein said increased yield comprises<br>
increased root yield.<br>
24.	Construct comprising:<br>
(i) A nucleic acid capable of modifying expression of a nucleic acid encoding a 2xC2H2 zinc finger protein and/or capable of modifying level and/or activity of a 2xC2H2 zinc finger protein;<br>
(ii) One or more plant control sequence capable of driving expression of the nucleic acid sequence of (i); and optionally<br>
(iii)      A transcription termination sequence.<br>
25.	Construct according to claim 24, wherein said nucleic acid of (i) is a nucleic acid as defined<br>
in any of claims 14 to 17.<br>
26.	Construct according to claim 24 or 25, wherein said control sequences of (ii) is at least a<br>
constitutive promoter, such as a GOS2 promoter.<br>
27.	Construct according to claim 24 or 25, wherein said control sequences of (ii) is at least a<br>
tissue preferred promoter, such as seed-preferred promoter.<br>
28.	Host cell comprising a construct according to any of claims 24 to 27.<br>
29.	Method for the production of a transgenic plant having increased yield, increased leaf<br>
surface area and/or prolonged vegetative growth, which method comprises<br>
(i)        introducing into a plant or plant cell a 2xC2H2 zinc finger nucleic acid; (ii)       Cultivating the plant or plant cell under conditions promoting plant growth.<br><br>
30.	Plant obtainable by a method according to any of claims 1 to 23 and 29, which plant has<br>
increased yield, modified leaf surface area and/or prolonged vegetative growth, relative to<br>
corresponding wild type plants.<br>
31.	Transgenic plant having increased yield, increased leaf surface area and/or prolonged<br>
vegetative growth, which transgenic plant has modified expression of a nucleic acid<br>
encoding a 2xC2H2 zinc finger protein and/or modified level and/or activity of a 2xC2H2<br>
zinc finger protein, relative to corresponding wild type plants.<br>
32.	Plant part, preferably a harvestable part, a propagule or progeny of a plant as defined in<br>
claim 30 or 31, which progeny has modified expression of a nucleic acid encoding 2xC2H2<br>
zinc finger protein and/or modified level and/or activity of a 2xC2H2 zinc finger protein,<br>
relative to corresponding wild type plants.<br>
33.	Plant or plant part according to any of claims 30 to 32, which plant is a monocotyledonous<br>
plant, preferably a cereal.<br>
34.	Plant or plant part according to any of claims 30 to 33 selected from rice, maize, wheat,<br>
barley,  millet,  oats,  rye,  sorghum,  soybean,  sunflower,   canola,  sugarcane,  alfalfa,<br>
leguminosae (bean, pea), flax , lupinus, rapeseed, tobacco, tomato, potato, squash,<br>
papaya, poplar and cotton.<br>
35.	Use of a nucleic acid encoding a 2xC2H2 protein, of a 2xC2H2 protein and/or of a<br>
construct as defined in any of claims 24 to 27 to increase plant yield.<br>
36.	A yield regulating composition comprising a nucleic acid encoding a 2xC2H2 protein, and/<br>
or comprising a 2xC2H2 protein, and/or comprising a construct as defined in any one of<br>
claims 24 to 27.<br>
37.	Use of a nucleic acid encoding a 2xC2H2 protein, of a 2xC2H2 protein and/or of a<br>
construct as defined in any of claims 24 to 27 to increase leaf surface area.<br>
38.	Use of a nucleic acid encoding a 2xC2H2 protein, of a 2xC2H2 protein and/or of a<br>
construct as defined in any of claims 24 to 27 to prolong vegetative growth.<br><br>
39.	Use of a nucleic acid encoding a 2xC2H2 protein, of a 2xC2H2 protein and/or of a<br>
construct as defined in any of claims 24 to 27 as target for an agrochemical.<br>
40.	Use of a nucleic acid encoding a 2xC2H2 protein, of a 2xC2H2 protein and/or of a<br>
construct as defined in any of claims 24 to 27 in a breeding program.<br>
41 . Use of a plant as defined in any of claims 30 to 34 to produce enzymes, Pharmaceuticals or agrochemicals.<br>
42. Use of a plant as defined in any one of claims 30 to 34 to produce food or feed products.  <br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNy1kZWxucC0yMDA1LUFic3RyYWN0LTEwLTA0LTIwMDgucGRm" target="_blank" style="word-wrap:break-word;">2407-delnp-2005-Abstract-10-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNy1kZWxucC0yMDA1LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2407-delnp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNy1kZWxucC0yMDA1LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2407-delnp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNy1kZWxucC0yMDA1LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjYtMDQtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2407-delnp-2005-Correspondence Others-(26-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNy1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlLU90aGVycy0xMC0wNC0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">2407-DELNP-2005-Correspondence-Others-10-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNy1kZWxucC0yMDA1LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2407-delnp-2005-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNy1kZWxucC0yMDA1LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">2407-delnp-2005-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNy1ERUxOUC0yMDA1LURlc2NyaXB0aW9uIChDb21wbGV0ZSkxMC0wNC0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">2407-DELNP-2005-Description (Complete)10-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNy1kZWxucC0yMDA1LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">2407-delnp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNy1ERUxOUC0yMDA1LUZvcm0tLTItMTAtMDQtMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">2407-DELNP-2005-Form--2-10-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNy1ERUxOUC0yMDA1LUZvcm0tMS0xMC0wNC0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">2407-DELNP-2005-Form-1-10-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNy1ERUxOUC0yMDA1LUZvcm0tMS0yMy0wNi0yMDA1LnBkZg==" target="_blank" style="word-wrap:break-word;">2407-DELNP-2005-Form-1-23-06-2005.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNy1kZWxucC0yMDA1LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">2407-delnp-2005-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNy1kZWxucC0yMDA1LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">2407-delnp-2005-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNy1kZWxucC0yMDA1LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">2407-delnp-2005-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNy1ERUxOUC0yMDA1LUZvcm0tMy0oMTctMDEtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">2407-DELNP-2005-Form-3-(17-01-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNy1kZWxucC0yMDA1LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">2407-delnp-2005-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNy1kZWxucC0yMDA1LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">2407-delnp-2005-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNy1ERUxOUC0yMDA1LUdQQS0oMTctMDEtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">2407-DELNP-2005-GPA-(17-01-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNy1kZWxucC0yMDA1LUdQQS0oMjYtMDQtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2407-delnp-2005-GPA-(26-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNy1kZWxucC0yMDA1LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">2407-delnp-2005-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNy1kZWxucC0yMDA1LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">2407-delnp-2005-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNy1kZWxucC0yMDA1LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">2407-delnp-2005-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNy1kZWxucC0yMDA1LXBjdC0zMDgucGRm" target="_blank" style="word-wrap:break-word;">2407-delnp-2005-pct-308.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNy1kZWxucC0yMDA1LXBjdC1zZWFyY2ggcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">2407-delnp-2005-pct-search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNy1kZWxucC0yMDA1LVBldGl0aW9uIE90aGVycy0oMjYtMDQtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2407-delnp-2005-Petition Others-(26-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQwNy1ERUxOUC0yMDA1LVBldGl0aW9uLTEzNy0oMTctMDEtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">2407-DELNP-2005-Petition-137-(17-01-2008).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="219443-a-novel-process-for-the-preparation-of-thermolabile-caffeine-fraction-from-the-tea-plant-leaves.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="219445-apparatus-for-evaluating-the-condition-of-a-patient-after-stroke.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>219444</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2407/DELNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>25/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>20-Jun-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>06-May-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>06-Jun-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CROPDESIGN N.V.,</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ANA ISABEL SANZ MOLINERO</td>
											<td></td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/82</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2003/051104</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-12-24</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>02080654.3</td>
									<td>2002-12-24</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/219444-method-to-modify-plant-growth-characteristics-by-recombinant-means-using-a-nucleic-acid-encoding-a-2xc2h2-protein by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:50:45 GMT -->
</html>
