<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/219483-a-dual-specific-ligand by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:48:06 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 219483:&quot;A DUAL SPECIFIC LIGAND&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A DUAL SPECIFIC LIGAND&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention provides a dual-specific ligand comprising a first immunoglobulin variable domain having a first binding specificitiy for a target ligand and a complementary or non-complementary immunoglobulin variable domain having a second binding specificity for a receptor of the target ligand.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Ligand<br>
The present invention relates to dual specific ligands. In particular, the invention provides a method for the preparation of dual-specific ligands comprising a first immunoglobulin single variable domain binding to a first antigen or epitope, and a second immunoglobulin single variable domain binding to a second antigen or epitope. More particularly, the invention relates to dual-specific ligariids wherein binding to at least one of the first and second antigens or epitopes acts to increase the half-life of the ligand in vivo. Open and closed conformation ligands comprising more than one binding specificity are described.<br>
Introduction<br>
The antigen binding domain of an antibody comprises two separate regions: a heavy chain variable domain (yH) and a light chain variable domain (VL: which can be either VK °r VjJ. The antigen binding site itself is formed by six polypeptide loops: three from VH domain (HI, H2 and H3) and three from VL domain (LI, L2 and L3). A diverse primary repertoire of V genes that encode the VH an(^ VL domains is produced by the combinatorial rearrangement of gene segments. The VH §ene *s produced by the recombination of three gene segments, VH&gt; D and JH- hi humans, there are approximately 51 functional VH segments (Cook and Tomlinson (1995) Immunol Today, 16: 237), 25 functional D segments (Corbett et al. (1997) J. Mol. Biol, 268: 69) and 6 functional JH segments (Ravetch et al. (1981) Cell, 27: 583), depending on the haplotype. The VH segment encodes the region of the polypeptide chain which forms the first and second antigen binding loops of the VH domain (HI and H2), whilst the VH&gt; D and JH segments combine to form the third antigen binding loop of the VH domain (H3). The VL Sene is produced by the recombination of only two gene segments, VL aQd JL- In humans, there are approximately 40 functional VK segments (Schable and Zachau (1993) Biol. Chern. Hoppe-Seyler, 374: 1001), 31 functional VJL segments (Williams et al. (1996) J. Mol Biol., 264: 220; Kawasaki et al. (1997) Genome Res., 7: 250), 5 functional JK segments (Hieter et al. (1982) J. Biol. Chem., 257: 1516) and 4 functional JA, segments (Vasicek<br>
and Leder (1990) J. Exp. Med., 172: 609), depending on the haplotype. The yL segment encodes the region of the polypeptide chain which forms the first and second antigen binding loops of the VL domain (LI and L2), whilst the yL and JL segments combine to form the third antigen binding loop of the VL domain (L3). Antibodies selected from this primary repertoire are believed to be sufficiently diverse to bind almost all antigens with at least moderate affinity. High affinity antibodies are produced by "affinity maturation" of the rearranged genes, in which point mutations are generated and selected by the immune system on the basis of improved binding.<br>
Analysis of the structures and sequences of antibodies has shown that five of the six antigen binding loops (HI, H2, LI, L2, L3) possess a limited number of main-chain conformations or canonical structures (Chothia and Lesk (1987) J. Mol. Biol., 196: 901; Chothia et al. (1989) Nature, 342: 877). The main-chain conformations are determined by (i) the length of the antigen binding loop, and (ii) particular residues, or types of residue, at certain key position in the antigen binding loop and the antibody framework. Analysis of the loop lengths and key residues has enabled us to the predict the main-chain conformations of HI, H2, LI, L2 and L3 encoded by the majority of human antibody sequences (Chothia et al. (1992) J. Mol. Biol, 227: 799; Tomlinson et al. (1995) EMBO J., 14: 4628; Williams et al (1996) J. Mol Biol, 264: 220). Although the H3 region is much more diverse in terms of sequence, length and structure (due to the use of D segments), it also forms a limited number of main-chain conformations for short loop lengths which depend on the length and the presence of particular residues, or types of residue, at key positions in the loop and the antibody framework (Martin et al. (1996) J. Mol Biol, 263: 800; Shirai etal (1996) FEBS Letters, 399: 1.<br>
Bispecific antibodies comprising complementary pairs of yH an^ VL regions are known in the art. These bispecific antibodies must comprise two pairs of VH and VLS&gt; eac^ VH/VL pair binding to a single antigen or epitope. Methods described involve hybrid hybridomas (Milstein &amp; Cuello AC, Nature 305:537-40), minibodies (Hu et al, (1996) Cancer Res 56:3055-3061;), diabodies (Holliger et al, (1993) Proc. Natl. Acad. Sci. USA 90, 6444-6448;  WO 94/13804), chelating recombinant antibodies (CRAbs; (Neri et al, (1995) J. Mol. Biol. 246, 367-373), biscFv (e.g. Atwell et al, (1996) Mol. Immunol. 33, 1301-1312), "knobs in holes" stabilised antibodies (Carter et al, (1997) Protein Sci. 6, 781-<br>
788). In each case each antibody species comprises two antigen-binding sites, each fashioned by a complementary pair of VH an^ VL domains. Each antibody is thereby able to bind to two different antigens or epitopes at the same time, with the binding to EACH antigen or epitope mediated by a VH an
There is some evidence that two different antibody binding specificities might be incorporated into the same binding site, but these generally represent two or more specificities that correspond to structurally related antigens or epitopes or to antibodies that are broadly cross-reactive.. For example, cross-reactive antibodies have been described, usually where the two antigens are related in sequence and structure, such as hen egg white lysozyme and turkey lysozyme (McCafferty et al., WO 92/01047) or to free hapten and to hapten conjugated to carrier (Griffiths AD et al. EMBO J 1994 13:14 3245-60). In a further example, WO 02/02773 (Abbott Laboratories) describes antibody f. molecules with "dual specificity".   The antibody molecules referred to are antibodies raised or selected against multiple antigens, such that their specificity spans more than a single antigen.   Each complementary VH/VL P3^ ^ me antibodies of WO 02/02773 specifies a single binding specificity for two or more structurally related antigens; the VH and VL domains in such complementary pairs do not each possess a separate specificity. The antibodies thus have a broad single specificity which encompasses two antigens, which are structurally related.   Furthermore natural autoantibodies have been described that are polyreactive (Casali &amp; Notkins, Ann. Rev. Immunol. 7, 515-531), reacting with at least two (usually more) different antigens or epitopes that are not structurally related. It<br>
has also been shown that selections of random peptide repertoires using phage display technology on a monoclonal antibody will identify a range of peptide sequences that fit the antigen binding site. Some of the sequences are highly related, fitting a consensus sequence, whereas others are very different and have been termed mimotopes (Lane &amp; Stephen, Current Opinion in Immunology, 1993, 5, 268-271). It is therefore clear that a natural four-chain antibody, comprising associated and complementary VH an
Protein engineering methods have been suggested that may have a bearing on this. For example it has also been proposed that a catalytic antibody could be created with a binding activity to a metal ion through one variable domain, and to a hapten (substrate) through contacts with the metal ion and a complementary variable domain (Barbas et al., 1993 Proc. Natl. Acad. Sci USA 90, 6385-6389). However in this case, the binding and catalysis of the substrate (first antigen) is proposed to require the binding of the metal ion (second antigen). Thus the binding to the VJ/VL pairing relates to a single but multi-component antigen.<br>
Methods have been described for the creation of bispecific antibodies from camel antibody heavy chain single domains in which binding contacts for one antigen are created in one variable domain, and for a second antigen in a second variable domain. However the variable domains were not complementary. Thus a first heavy chain variable domain is selected against a first antigen, and a second heavy chain variable domain against a second antigen, and then both domains are linked together on the same chain to give a bispecific antibody fragment (Conrath et al., J. Biol. Chem. 270, 27589-27594). However the camel heavy chain single domains are unusual in that they are derived from natural camel antibodies which have no light chains, and indeed the heavy chain single domains are unable to associate with camel light chains to form complementary VH an^ VL pairs.<br>
Single heavy chain variable domains have also been described, derived from natural antibodies which are normally associated with light chains (from monoclonal antibodies<br>
or from repertoires of domains; see EP-A-0368684). These heavy chain variable domains have been shown to interact specifically with one or more related antigens but have not been combined with other heavy or light chain variable domains to create a ligand with a specificity for two or more different antigens . Furthermore, these single domains have been shown to have a very short in vivo half-life. Therefore such domains are of limited therapeutic value.<br>
It has been suggested to make bispecific antibody fragments by linking heavy chain variable domains of different specificity together (as described above). The disadvantage with this approach is that isolated antibody variable domains may have a hydrophobic interface that normally makes interactions with the light chain and is exposed to solvent and may be "sticky" allowing the single domain to bind to hydrophobic surfaces. Furthermore, hi the absence of a partner light chain the combination of two or more different heavy chain variable domains and their association, possibly via their hydrophobic interfaces, may prevent them from binding to one in not both of the ligands they are able to bind in isolation. Moreover, in this case the heavy chain variable domains would not be associated with complementary light chain variable domains and thus may be less stable and readily unfold (Worn &amp; Pluckthun, 1998 Biochemistry 37, 13120-7).<br>
Summary of the invention<br>
The inventors have described, in their copending international patent application WO 03/002609 as well as copending unpublished UK patent application 0230203.2, dual specific immunoglobulin ligands which comprise irmnunoglobulin single variable domains which each have different specificities. The domains may act in competition with each other or independently to bind antigens or epitopes on target molecules.<br>
In a first configuration, the present invention provides a further improvement in dual specific ligands as developed by the present inventors, in which one specificity of the ligand is directed towards a protein or polypeptide target, and another specificity is directed to a receptor for the target.<br>
Therefore, in a first aspect, the invention provides a dual specific ligand comprising a first dAb specific for a target ligand, and a second dAb specific for a receptor for the target ligand.<br>
Preferably, the dual specific ligand is an open conformation ligand and can bind both the target ligand and the target ligand receptor simultaneously.<br>
Preferred dual specific ligands comprise at least on specificity for TNF alpha and at least one specificity for TNF Receptor 1 (p55). Advantageously, the specificities are provided by one or more dAbs arranged in Fab, F(ab')2 or IgG formats. Preferred dAbs are TAR1-5-19 VK and TAR2h-10-27 VH as set forth below.<br>
The invention may also comprise further modifications and configurations of the dual specific ligands as set forth in the accompanying claims and detailed herein.<br>
Accordingly, in a further aspect, there is provided a dual-specific ligand comprising a first immunoglobulin single variable domain having a binding specificity to a first antigen or epitope and a second complementary immunoglobulin single variable domain having a binding activity to a, second antigen or epitope, wherein one or both of said antigens or epitopes acts to increase the half-life of the ligand in vivo and wherein said first and second domains lack mutually complementary domains which share the same specificity, provided that said dual specific ligand does not consist of an anti-HSA VH domain and an anti-p galactosidase VK domain. Preferably, neither of the first or second variable domains binds to human serum albumin (HSA).<br>
Antigens or epitopes which increase the half-life of a ligand as described herein are advantageously present on proteins or polypeptides found in an organism in vivo. Examples include extracellular matrix proteins, blood proteins, and proteins present in various tissues in the organism. The proteins act to reduce the rate of ligand clearance from the blood, for example by acting as bulking agents, or by anchoring the ligand to a desired site of action. Examples of antigens/epitopes which increase half-life in vivo are given in Annex 1 below.<br>
Increased half-life is useful in in vivo applications of irnmunoglobulins, especially antibodies and most especially antibody fragments of small size. Such fragments (Fvs, disulphide bonded Fvs, Fabs, scFvs, dAbs) suffer from rapid clearance from the body; thus, whilst they are able to reach most parts of the body rapidly, and are quick to produce and easier to handle, their in vivo applications have been limited by their only brief persistence in vivo. The invention solves this problem by providing increased half-life of the ligands in vivo and consequently longer persistence times in the body of the functional activity of the ligand.<br>
Methods for pharmacokinetic analysis and determination of ligand half-life will be familiar to those skilled in the art. Details may be found in Kenneth, A et al: Chemical Stability of Pharmaceuticals: A Handbook for Pharmacists -and in Peters et al, Pharmacokinetc analysis: A Practical Approach (1996). Reference is also made to "Pharmacokinetics", M Gibaldi &amp; D Perron, published by Marcel Dekker, 2nd Rev. ex edition (1982), which describes pharmacokinetic parameters such as t alpha and t beta half lives and area under the curve (AUC).<br>
Half lives (t'/a alpha and ilA beta) and AUC can be determined from a curve of serum concentration of ligand against time.   The WinNonlin analysis package (available from Pharsight Corp., Mountain View, CA94040, USA) can be used, for example, to model the curve, hi a first phase (the alpha phase) the ligand is undergoing mainly distribution hi the patient, with some elimination. A second phase (beta phase) is the terminal phase when the ligand has been distributed and the serum concentration is decreasing as the ligand is cleared from the patient. The t alpha half life is the half life of the first phase and the t beta half life is the half life of the second phase. Thus, advantageously, the present invention provides a ligand or a composition comprising a ligand according to the invention having a ta half-life in the range of 15 minutes or more. In one embodiment, the lower end of the range is 30 minutes, 45 minutes, 1 hour, 2 hours, 3 hours, 4 hours, 5 hours, 6 hours, 7 hours, 10 hours, 11 hours  or 12 hours. In addition, or alternatively,a ligand or composition according to the invention will have a ta half life in the range of up to and including 12 hours. In one embodiment, the upper end of the range is 11,10, 9, 8, 7, 6 or 5 hours.  An example of a suitable range is 1 to 6 hours, 2 to 5 hours or 3 to 4 hours.<br>
Advantageously, the present invention provides a ligand or a composition comprising a ligand according to the invention having a tp half-life in the range of 2.5 hours or more. In one embodiment, the lower end of the range is 3 hours, 4 hours, 5 hours, 6 hours, 7 hours, 10 hours , 11 hours, or 12 hours, hi addition, or alternatively, a ligand or composition according to the invention has a tp half-life hi the range of up to and including 21 days, hi one embodiment, the upper end of the range is 12 hours, 24 hours, 2 days, 3 days, 5 days, 10 days, 15 days or 20 days. Advantageously a ligand or composition according to the invention will have a tp half life in the range 12 to 60 hours. In a further embodiment, it will be in the range 12 to 48 hours. In a further embodiment still, it will be in the range 12 to 26 hours.<br>
In addition, or alternatively to the above criteria, the present invention provides a ligand or a composition comprising a ligand according to the invention having an AUC value (area under the curve) in the range of 1 mg.min/ml or more, hi one embodiment, the lower end of the range is 5, 10, 15, 20, 30, 100, 200 or 300mg.min/ml. hi addition, or alternatively, a ligand or composition according to the invention has an AUC hi the range of up to 600 mg.min/ml. In one embodiment, the upper end of the range is 500, 400, 300, 200, 150, 100, 75 or 50 mg.min/ml. Advantageously a ligand according to the invention will have a AUC hi the range selected from the group consisting of the following: 15 to 150mg.min/ml, 15 to 100 mg.min/ml, 15 to 75 mg.min/ml, and 15 to 50mg.min/ml.<br>
hi a first embodiment, the dual specific ligand comprises two complementary variable domains, i.e. two variable domains that, in their natural environment, are capable of operating together as a cognate pair or group even if in the context of the present invention they bind separately to their cognate epitopes. For example, the complementary variable domains may be immunoglobulin heavy chain and light chain variable domains (Vn and VL). VH and VL domains are advantageously provided by scFv or Fab antibody fragments. Variable domains may be linked together to form multivalent ligands by, for example: provision of a hinge region at the C-terminus of each V domain and disulphide bonding between cysteines in the hinge regions; or provision of dAbs each with a cysteine at the C-terminus of the domain, the cysteines being disulphide bonded together; or production of V-CH &amp; V-CL to produce a Fab format; or use of peptide linkers (for<br>
example Gly^Ser linkers discussed hereinbelow) to produce dimers, rrimers and further multimers.<br>
The inventors have found that the use of complementary variable domains allows the two domain surfaces to pack together and be sequestered from the solvent. Furthermore the complementary domains are able to stabilise each other. In addition, it allows the creation of dual-specific IgG antibodies without the disadvantages of hybrid hybridomas as used in the prior art, or the need to engineer heavy or light chains at the sub-unit interfaces. The dual-specific ligands of the first aspect of the present invention have at least one VH/VL pair.   A bispecific IgG according to this invention will therefore comprise two such pairs, one pair on each arm of the Y-shaped molecule. Unlike conventional bispecific  antibodies  or diabodies,  therefore,  where  the ratio  of chains  used  is determinative in the success of the preparation thereof and leads to practical difficulties, the dual specific ligands of the invention are free from issues of chain balance. Chain imbalance in conventional bi-specific antibodies results from the association of two different VL chains with two different VH chains, where VL chain 1 together with VH chain 1 is able to bind to antigen or epitope 1 and VL chain 2 together with VH chain 2 is able to bind to antigen or epitope 2 and the two correct pairings are in some way linked to one another. Thus, only when VL chain 1 is paired with VH chain 1 and VL chain 2 is paired with VH chain 2 in a single molecule is bi-specificity created. Such bi-specific molecules can be created in two different ways. Firstly, they can be created by association of two existing VH/VL pairings that each bind to a different antigen or epitope (for example, in a bi-specific IgG). In this case the VH/VL pairings must come all together in a 1:1 ratio in order to create a population of molecules all of which are bi-specific. This never occurs (even when complementary CH domain is enhanced by "knobs into holes" engineering) leading to a mixture of bi-specific molecules and molecules that are only able to bind to one antigen or epitope but not the other. The second way of creating a bispecific antibody is by the simultaneous association of two different VH chain with two different VL chains (for example in a bi-specific diabody). In this case, although there tends to be a preference for VL chain 1 to pair with VH chain 1 and VL chain 2 to pair with VH chain 2 (which can be enhanced by "knobs into holes" engineering of the VL and VH domains), this paring is never achieved hi all molecules, leading to a mixed formulation whereby incorrect pairings occur that are unable to bind to either antigen or epitope.<br>
Hi-specific antibodies constructed according to the dual-specific ligand approach according to the first aspect of the present invention overcome all of these problems because the binding to antigen or epitope 1 resides within the VH or VL domain and the binding to antigen or epitope 2 resides with the complementary VL or VH domain, respectively. Since VH and VL domains pair on a 1:1 basis all VH/VL pairings will be bi-specific and thus all formats constructed using these VH/VL pairings (Fv, scFvs, Fabs, minibodies, IgGs etc) will have 100% bi-specific activity.<br>
hi the context of the present invention, first and second "epitopes" are understood to be epitopes which are not the same and are not bound by a single monospecific ligand. In the first configuration of the invention, they are advantageously on different antigens, one of which acts to increase the half-life of the ligand in vivo. Likewise, the first and second antigens are advantageously not the same.<br>
The dual specific ligands of the invention do not include ligands as described in WO 02/02773. Thus, the ligands of the present invention do not comprise complementary VH/VL pairs which bind any one or more antigens or epitopes co-operatively. Instead, the ligands according to the first aspect of the invention comprise a VH/VL complementary pair, wherein the V domains have different specificities.<br>
Moreover, the ligands according to the first aspect of the invention comprise complementary pairs having different specificities for non-structurally related epitopes or antigens. Structurally related epitopes or antigens are epitopes or antigens which possess sufficient structural similarity to be bound by a conventional VH/VL complementary pair which acts in a co-operative manner to bind an antigen or epitope; in the case of structurally related epitopes, the epitopes are sufficiently similar in structure that they "fit" into the same binding pocket formed at the antigen binding site of the VH/VL dimer.<br>
hi a further aspect, the present invention provides a ligand comprising a first immunoglobulin variable domain having a first antigen or epitope binding specificity and a second immunoglobulin variable domain having a second antigen or epitope binding specificity wherein one or both of said first and second variable domains bind to an<br>
antigen which increases the half-life of the ligand in vivo, and the variable domains are not complementary to one another.<br>
In one embodiment, binding to one variable domain modulates the binding of the ligand to the second variable domain.<br>
In this embodiment, the variable domains may be, for example, pairs of VH domains or pairs of VL domains. Binding of antigen at the first site may modulate, such as enhance or inhibit, binding of an antigen at the second site. For example, binding at the first site at least partially inhibits binding of an antigen at a second site. In such an embodiment, the ligand may for example be maintained in the body of a subject organism in vivo through binding to a protein which increases the half-life of the ligand until such a time as it becomes bound to the second target antigen and dissociates from the half-life increasing protein.<br>
Modulation of binding in the above context is achieved as a consequence of the structural proximity of the antigen binding sites relative to one another. Such structural proximity can be achieved by the nature of the structural components linking the two or more antigen binding sites, eg by the provision of a ligand with a relatively rigid structure that holds the antigen binding sites in close proximity. Advantageously, the two or more antigen binding sites are in physically close proximity to one another such that one site modulates the binding of antigen at another site by a process which involves steric hindrance and/or conformational changes within the immunoglobulin molecule.<br>
The first and the second antigen binding domains may be associated either covalently or non-covalently. In the case that the domains are covalently associated, then the association may be mediated for example by disulphide bonds or by a polypeptide linker such as (Gly4Ser)n, where n = from 1 to 8, eg, 2, 3,4, 5 or 7.<br>
Ligands according to the invention may be combined into non-immunoglobulin multi-ligand structures to form multivalent complexes, which bind target molecules with the same antigen, thereby providing superior avidity, while at least one variable domain binds an antigen to increase the half life of the multimer. For example natural bacterial<br>
receptors such as SpA have been used as scaffolds for the grafting of CDRs to generate ligands which bind specifically to one or more epitopes. Details of this procedure are described in US 5,831,012. Other suitable scaffolds include those based on fibronectin and affibodies. Details of suitable procedures are described in WO 98/58965. Other suitable scaffolds include lipocallin and CTLA4, as described in van den Beuken et al., J. Mol. Biol. (2001) 310, 591-601, and scaffolds such as those described in WO0069907 (Medical Research Council), which are based for example on the ring structure of bacterial GroEL or other chaperone polypeptides.<br>
Protein scaffolds may be combined; for example, CDRs may be grafted on to a CTLA4 scaffold and used together with immunoglobulin VH or VL domains to form a ligand. Likewise, fibronectin, lipocallin and other scaffolds may be combined.<br>
In the case that the variable domains are selected from V-gene repertoires selected for instance using phage display technology as herein described, then these variable domains can comprise a universal framework region, such that is they may be recognised by a specific generic ligand as herein defined. The use of universal frameworks, generic ligands and the like is described in W099/20749. • In the present invention, reference to phage display includes the use of both phage and/or phagerm'ds.<br>
Where V-gene repertoires are used variation in polypeptide sequence is preferably located within the structural loops of the variable domains. The polypeptide sequences of either variable domain may be altered by DNA shuffling or by mutation in order to enhance the interaction of each variable domain with its complementary pair.<br>
In a preferred embodiment of the invention the 'dual-specific ligand' is a single chain Fv fragment. In an alternative embodiment of the invention, the 'dual-specific ligand' consists of a Fab region of an antibody. The term "Fab region" includes a Fab-like region where two VH or two VL domains are used.<br>
The variable regions may be derived from antibodies directed against target antigens or epitopes. Alternatively they may be derived from a repertoire of single antibody domains<br>
such as those expressed on the surface of filamentous bacteriophage.  Selection may be performed as described below.<br>
In a further aspect, the invention provides a method for producing a ligand comprising a first immunoglobulin single variable domain having a first binding specificity and a second single immunoglobulin single variable domain having a second (different) binding specificity, one or both of the binding specificities being specific for an antigen which increases the half-life of the ligand in vivo, the method comprising the steps of:<br>
(a)	selecting a first variable domain by its ability to bind to a first epitope,<br>
(b)	selecting a second variable region by its ability to bind to a second epitope,<br>
(c)	combining the variable domains; and<br>
(d)	selecting the ligand by its ability to bind to said first epitope and to said second<br>
epitope.<br>
The ligand can bind to the first and second epitopes either simultaneously or, where there is competition between the binding domains for epitope binding, the binding of one domain may preclude the binding of another domain to its cognate epitope. In one embodiment, therefore, step (d) above requires simultaneous binding to both first and second (and possibly further) epitopes; in another embodiment, the binding to the first and second epitoes is not simultaneous.<br>
The epitopes are preferably on separate antigens.<br>
Ligands advantageously comprise VH/VL combinations, or VH/VH or VL/VL combinations of immunoglobulin variable domains, as described above. The ligands may moreover comprise camelid VHH domains, provided that the VHH domain which is specific for an antigen which increases the half-life of the ligand in vivo does not bind Hen egg white lysozyme (HEL), porcine pancreatic alpha-amylase or NmC-A; hcg, BS A-linked RR6 azo dye or S. mutans HG982 cells, as described in Conrath et al., (2001) JBC 276:7346-7350 and WO99/23221, neither of which describe the use of a specificity for an antigen which increases half-life to increase the half life of the ligand in vivo.<br>
In one embodiment, said first variable domain is selected for binding to said first epitope in absence of a complementary variable domain. In a further embodiment, said first variable domain is selected for binding to said first epitope/antigen in the presence of a third variable domain in which said third variable domain is different from said second variable domain and is complementary to the first domain. Similarly, the second domain may be selected in the absence or presence of a complementary variable domain.<br>
The antigens or epitopes targeted by the ligands of the invention, in addition to the half-life enhancing protein, may be any antigen or epitope but advantageously is an antigen or epitope that is targeted with therapeutic benefit.    The invention provides ligands, including open conformation, closed conformation and isolated dAb monomer ligands, specific for any such target, particularly those targets further identified herein.   Such targets may be, or be part of, polypeptides, proteins or nucleic acids, which may be naturally occurring or synthetic. In this respect, the ligand of the invention may bind the epiotpe or antigen and act as an antagonist or agonist (eg, EPO receptor agonist). One skilled in the art will appreciate that the choice is large and varied. They may be for instance human or animal proteins, cytokines, cytokine receptors, enzymes co-factors for enzymes or DNA binding proteins. Suitable cytokines and growth factors include but are not limited to: ApoE, Apo-SAA, BDNF, Cardiotrophin-1, EGF, EOF receptor, ENA-78, Eotaxin, Eotaxin-2, Exodus-2, EpoR, FGF-acidic, FGF-basic, fibroblast growth factor-10, FLT3 ligand,   Fractalkine (CX3C), GDNF, G-CSF, GM-CSF, GF-pl, insulin, IFN-y, IGF-I, IGF-H, IL-lcc, IL-lp, DL-2, IL-3, DL-4, IL-5, IL-6, DL-7, IL-8 (72 a.a.), IL-8 (77 a.a.), IL-9, IL-10, IL-11, IL-12, IL-13, IL-15, EL-16, EL-17, IL-18 (IGIF), Inhibin a, Inhibin p, IP-10, keratinocyte growth factor-2 (KGF-2), KGF, Leptin, LIF, Lymphotactin, Mullerian inhibitory substance, monocyte colony inhibitory factor, monocyte attractant protein, M-CSF, MDC (67 a.a.), MDC (69 a.a.), MCP-1 (MCAF), MCP-2, MCP-3, MCP-4, MDC (67 a.a.), MDC (69 a.a.), MIG, MlP-la, Mff-lp, MIP-3a, MIP-3p, Mff-4, myeloid progenitor inhibitor factor-1 (MPIF-1), NAP-2, Neurturin, Nerve growth factor, p-NGF, NT-3, NT-4, Oncostatin M, PDGF-AA, PDGF-AB, PDGF-BB, PF-4, RANTES, SDFla, SDFlp, SCF, SCGF, stem cell factor (SCF), TARC, TGF-cc, TGF-p, TGF-P2, TGF-p3, tumour necrosis factor (TNF), TNF-cc, TNF-p, TNF receptor I, TNF receptor E, TNIL-1, TPO, VEGF, VEGF receptor 1, VEGF receptor 2, VEGF receptor 3, GCP-2, GRO/MGSA, GRO-p, GRO-y, HCC1, 1-309, HER 1, HER 2, HER 3 and HER 4.<br>
Cytokine receptors include receptors for the foregoing cytokines. It will be appreciated that this list is by no means exhaustive.<br>
In one embodiment of the invention, the variable domains are derived from a respective antibody directed against the antigen or epitope. In a preferred embodiment the variable domains are derived from a repertoire of single variable antibody domains.<br>
In a further aspect, the present invention provides one or more nucleic acid molecules encoding at least a dual-specific ligand as herein defined. The dual specific ligand may be encoded on a single nucleic acid molecule; alternatively, each domain may be encoded by a separate nucleic acid molecule. Where the ligand is encoded by a single nucleic acid molecule, the domains may be expressed as a fusion polypeptide, in the manner of a scFv molecule, or may be separately expressed and subsequently linked together, for example using chemical linking agents. Ligands expressed from separate nucleic acids will be linked together by appropriate means.<br>
The nucleic acid may further encode a signal sequence for export of the polypeptides from a host cell upon expression and may be fused with a surface component of a filamentous bacteriophage particle (or other component of a selection display system) upon expression.<br>
In a further aspect the present invention provides a vector comprising nucleic acid encoding a dual specific ligand according to the present invention.<br>
In a yet further aspect, the present invention provides a host cell transfected with a vector encoding a dual specific ligand according to the present invention.<br>
Expression from such a vector may be configured to produce, for example on the surface of a bacteriophage particle, variable domains for selection. This allows selection of displayed variable regions and thus selection of 'dual-specific ligands' using the method of the present invention.<br>
The present invention further provides a kit comprising at least a dual-specific ligand according to the present invention.<br>
Dual-Specific ligands according to the present invention preferably comprise combinations of heavy and light chain domains. For example, the dual specific ligand may comprise a VH domain and a VL domain, which may be linked together in the form of an scFv. In addition, the ligands may comprise one or more CH or CL domains. For example, the ligands may comprise a CH! domain, Cy2 or Cn3 domain, and/or a CL domain, Qj.1, Cu2, Cu3 or Cjj.4 domains, or any combination thereof. A hinge region domain may also be included. Such combinations of domains may, for example, mimic natural antibodies, such as IgG or IgM, or fragments thereof, such as Fv, scFv, Fab or F(ab')2 molecules. Other structures, such as a single arm of an IgG molecule comprising , VL, CH! and CL domains, are envisaged.<br>
In a preferred embodiment of the invention, the variable regions are selected from single domain V gene repertoires. Generally the repertoire of single antibody domains is displayed on the surface of filamentous bacteriophage. In a preferred embodiment each single antibody domain is selected by binding of a phage repertoire to antigen.<br>
In a preferred embodiment of the invention each single variable domain may be selected for binding to its target antigen or epitope in the absence of a complementary variable region. In an alternative embodiment, the single variable domains may be selected for binding to its target antigen or epitope in the presence of a complementary variable region. Thus the first single variable domain may be selected in the presence of a third complementary variable domain, and the second variable domain may be selected in the presence of a fourth complementary variable domain. The complementary third or fourth variable domain may be the natural cognate variable domain having the same specificity as the single domain being tested, or a non-cognate complementary domain - such as a "dummy" variable domain.<br>
Preferably, the dual specific ligand of the invention comprises only two variable domains although several such ligands may be incorporated together into the same protein, for example two such ligands can be incorporated into an IgG or a multimeric<br>
immunoglobulm, such as igM. Alternatively, in another embodiment a plurality of dual specific ligands are combined to form a multimer. For example, two different dual specific ligands are combined to create a tetra-specific molecule.<br>
It will be appreciated by one skilled in the art that the light and heavy variable regions of a dual-specific ligand produced according to the method of the present invention may be on the same polypeptide chain, or alternatively, on different polypeptide chains. Li the case that the variable regions are on different polypeptide chains, then they may be linked via a linker, generally a flexible linker (such as a polypeptide chain), a chemical linking group, or any other method known in the art.<br>
In a further aspect, the present invention provides a composition comprising a dual-specific ligand, obtainable by a method of the present invention, and a pharmaceutically acceptable carrier, diluent or excipient.<br>
Moreover, the present invention provides a method for the treatment and/or prevention of disease using a 'dual-specific ligand' or a composition according to the present invention.<br>
hi a second configuration, the present invention provides multispecific ligands which comprise at least two non-complementary variable domains. For example, the ligands may comprise a pair of VH domains or a pair of VL domains. Advantageously, the domains are of non-camelid origin; preferably they are human domains or comprise human framework regions (FWs) and one or more heterologous CDRs. CDRs and framework regions are those regions of an immunoglobulin variable domain as defined in the Kabat database of Sequences of Proteins of Immunological Interest.<br>
Preferred human framework regions are those encoded by germline gene segments DP47 and DPK9. Advantageously, FW1, FW2 and FWS of a VH or VL domain have the sequence of FW1, FW2 or FWS from DP47 or DPK9. The human frameworks may optionally contain mutations, for example up to about 5 amino acid changes or up to about 10 amino acid changes collectively in the human frameworks used in the ligands of the invention.<br>
The variable domains in the multispecific ligands according to the second configuration of the invention may be arranged in an open or a closed conformation; that is, they may be arranged such that the variable domains can bind their cognate ligands independently and simultaneously, or such that only one of the variable domains may bind its cognate ligand at any one time.<br>
The inventors have realised that under certain structural conditions, non-complementary variable domains (for example two light chain variable domains or two heavy chain variable domains) may be present in a ligand such that binding of a first epitope to a first variable domain inhibits the binding of a second epitope to a second variable domain, even though such non-complementary domains do not operate together as a cognate pair.<br>
Advantageously, the ligand comprises two or more pairs of variable domains; that is, it comprises at least four variable domains. Advantageously, the four variable domains comprise frameworks of human origin.<br>
In a preferred embodiment, the human frameworks are identical to those of human germline sequences.<br>
The present inventors consider that such antibodies will be of particular use in ligand binding assays for therapeutic and other uses.<br>
Thus, in a first aspect of the second configuration, the present invention provides a method for producing a multispecific ligand comprising the steps of:<br>
a)	selecting a first epitope binding domain by its ability to bind to a first epitope,<br>
b)	selecting a second epitope binding domain by its ability to bind to a second<br>
epitope,<br>
c)	combining the epitope binding domains; and<br>
d)	selecting the closed conformation multispecific ligand by its ability to bind to said<br>
first second epitope and said second epitope.<br>
In a further aspect of the second configuration, the invention provides method for preparing a closed conformation multi-specific ligand comprising a first epitope binding<br>
domain having a first epitope binding specificity and a non-complementary second epitope binding domain having a second epitope binding specificity, wherein the first and second binding specificities compete for epitope binding such that the closed conformation multi-specific ligand may not bind both epitopes simultaneously, said method comprising the steps of:<br>
a)	selecting a first epitope binding domain by its ability to bind to a first epitope,<br>
b)	selecting a second epitope binding domain by its ability to bind to a second<br>
epitope,<br>
c)	combining the epitope binding domains such that the domains are in a closed<br>
conformation; and<br>
d)	selecting the closed conformation multispecific ligand by its ability to bind to said<br>
first second epitope and said second epitope, but not to both said first and second<br>
epitopes simultaneously.<br>
Moreover, the invention provides a closed conformation multi-specific ligand comprising a first epitope binding domain having a first epitope binding specificity and a non-complementary second epitope binding domain having a second epitope binding specificity, wherein the first and second binding specificities compete for epitope binding such that the closed conformation multi-specific ligand may not bind both epitopes simultaneously.<br>
An alternative embodiment of the above aspect of the of the second configuration of the invention optionally comprises a further step (bl) comprising selecting a third or further epitope binding domain, hi this way the multi-specific ligand produced, whether of open or closed conformation, comprises more than two epitope binding specificities. In a preferred aspect of the second configuration of the invention, where the multi-specific ligand comprises more than two epitope binding domains, at least two of said domains are in a closed conformation and compete for binding; other domains may compete for binding or may be free to associate independently with their cognate epitope(s).<br>
According to the present invention the term 'multi-specific ligand' refers to a ligand which possesses more than one epitope binding specificity as herein defined.<br>
As herein defined the term 'closed conformation' (multi-specific ligand) means that the epitope binding domains of the ligand are attached to or associated with each other, optionally by means of a protein skeleton, such that epitope binding by one epitope binding domain competes with epitope binding by another epitope binding domain. That is, cognate epitopes may be bound by each epitope binding domain individually but not simultaneosuly. The closed conformation of the ligand can be achieved using methods herein described.<br>
"Open conformation" means that the epitope binding domains of the ligand are attached to or associated with each other, optionally by means of a protein skeleton, such that epitope binding by one epitope binding domain does not compete with epitope binding by another epitope binding domain.<br>
As referred to herein, the term 'competes' means that the binding of a first epitope to its cognate epitope binding domain is inhibited when a second epitope is bound to its cognate epitope binding domain. For example, binding may be inhibited sterically, for example by physical blocking of a binding domain or by alteration of the structure or environment of a binding domain such that its affinity or avidity for an epitope is reduced.<br>
In a further embodiment of the second configuration of the invention, the epitopes may displace each other on binding. For example, a first epitope may be present on an antigen which, on binding to its cognate first binding domain, causes steric hindrance of a second binding domain, or a coformational change therein, which displaces the epitope bound to the second binding domain.<br>
Advantageously, binding is reduced by 25% or more, advantageously 40%, 50%, 60%, 70%, 80%, 90% or more, and preferably up to 100% or nearly so, such that binding is completely inhibited. Binding of epitopes can be measured by conventional antigen binding assays, such as ELISA, by fluorescence based techniques, including FRET, or by techniques such as suface plasmon resonance which measure the mass of molecules.<br>
According to the method of the present invention, advantageously, each epitope binding domain is of a different epitope binding specificity.<br>
In the context of the present invention, first and second "epitopes" are understood to be epitopes which are not the same and are not bound by a single monospecific ligand. They may be on different antigens or on the same antigen, but separated by a sufficient distance that they do not form a single entity that could be bound by a single .mono-specific VH/VL binding pair of a conventional antibody. Experimentally, if both of the individual variable domains in single chain antibody form (domain antibodies or dAbs) are separately competed by a monospecific VH/VL ligand against two epitopes then those two epitopes are not sufficiently far apart to be considered separate epitopes according to the present invention.<br>
The closed conformation multispecific ligands of the invention do not include ligands as described in WO 02/02773. Thus, the ligands of the present invention do not comprise complementary VH/VL Pau"s which bind any one or more antigens or epitopes cooperatively. Instead, the ligands according to the invention preferably comprise non-complementary VH~VH or VL~VL Pairs- Advantageously, each VH or VL domain in each VH~VH or VL~VL Pau*nas a different epitope binding specificity, and the epitope binding sites are so arranged that the binding of an epitope at one site competes with the binding of an epitope at another site.<br>
According to the present invention, advantageously, each epitope binding domain comprises an immunoglobulin variable domain. More advantageously, each immunoglobulin variable domain will be either a variable light chain domain (VL) or a variable heavy chain domain VH- &amp;1 the second configuration of the present invention, the immunoglobulin domains when present on a ligand according to the present invention are non-complementary, that is they do not associate to form a VH/VL antigen binding site. Thus, multi-specific ligands as defined hi the second configuration of the invention comprise immunoglobulin domains of the same sub-type, that is either variable light chain domains (VL) or variable heavy chain domains (VH)- Moreover, where the ligand according to the invention is in the closed conformation, the immunoglobulin domains may be of the camelid VHH type.<br><br>
Li an alternative embodiment, the ligand(s) according to the invention do not comprise a camelid VHH domain. More particularly, the ligand(s) of the invention do not comprise one or more amino acid residues that are specific to camelid VHH domains as compared to human VH domains.<br>
Advantageously, the single variable domains are derived from antibodies selected for binding activity against different antigens or epitopes. For example, the variable domains may be isolated at least in part by human immunisation. Alternative methods are known in the art, including isolation from human antibody libraries and synthesis of artificial antibody genes.<br>
The variable domains advantageously bind superantigens, such as protein A or protein L. Binding to superantigens is a property of correctly folded antibody variable domains, and allows such domains to be isolated from, for example, libraries of recombinant or mutant domains.<br>
Epitope binding domains according to the present invention comprise a protein scaffold and epitope interaction sites (which are advantageously on the surface of the protein scaffold).<br>
Epitope binding domains may also be based on protein scaffolds or skeletons other than immunoglobulin domains. For example natural bacterial receptors such as SpA have been used as scaffolds for the grafting of CDRs to generate ligands which bind specifically to one or more epitopes. Details of this procedure are described in US 5,831,012. Other suitable scaffolds include those based on fibronectin and affibodies. Details of suitable procedures are described in WO 98/58965. Other suitable scaffolds include lipocallin and CTLA4, as described in van den Beuken et al., J. Mol. Biol. (2001) 310, 591-601, and scaffolds such as those described in W00069907 (Medical Research Council), which are based for example on the ring structure of bacterial GroEL or other chaperone polypeptides.<br>
Protein scaffolds may be combined; for example, CDRs may be grafted on to a CTLA4 scaffold and used together with immunoglobulin VH or VL domains to form a multivalent ligand. Likewise, fibronectin, lipocallin and other scaffolds may be combined.<br>
It will be appreciated by one skilled in the art that the epitope binding domains of a closed conformation multispecific ligand produced according to the method of the present invention may be on the same polypeptide chain, or alternatively, on different polypeptide chains. In the case that the variable regions are on different polypeptide chains, then they may be linked via a linker, advantageously a flexible linker (such as a polypeptide chain), a chemical linking group, or any other method known in the art.<br>
The first and the second epitope binding domains may be associated either covalently or non-covalently. In the case that the domains are covalently associated, then the association may be mediated for example by disulphide bonds.<br>
In the second configuation of the invention, the first and the second epitopes are preferably different. They may be, or be part of, polypeptides, proteins or nucleic acids, which may be naturally occurring or synthetic. In this respect, the ligand of the invention may bind an epiotpe or antigen and act as an antagonist or agonist (eg, EPO receptor agonist). The epitope binding domains of the ligand in one embodiment have the same epitope specificity, and may for example simultaneously bind their epitope when multiple copies of the epitope are present on the same antigen. In another embodiment, these epitopes are provided on different antigens such that the ligand can bind the epitopes and bridge the antigens. One skilled in the art will appreciate that the choice of epitopes and antigens is large and varied. They may be for instance human or animal proteins, cytokines, cytokine receptors, enzymes co-factors for enzymes or DNA binding proteins. Suitable cytokines and growth factors include but are not limited to: ApoE, Apo-SAA, BDNF, Cardiotrophin-1, EGF, EOF receptor, ENA-78, Eotaxin, Eotaxin-2, Exodus-2, EpoR, FGF-acidic, FGF-basic, fibroblast growth factor-10,    FLT3 ligand,   Fractallcine (CX3C), GDNF, G-CSF, GM-CSF, GF-01, insulin, IFN-y, IGF-I, IGF-H, IL-la, IL-lp, IL-2, IL-3, IL-4, IL-5, IL-6, IL-7, IL-8 (72 a.a.), BL-8 (77 a.a.), EL-9, IL-10, IL-11, IL-12, IL-13, IL-15, DL-16, IL-17, IL-18 (IGIF), Inhibin a, Inhibin p, IP-10, keratinocyte growth factor-2 (KGF-2), KGF, Leptin, LIF, Lymphotactin, Mullerian inhibitory substance,<br>
monocyte colony inhibitory factor, monocyte attractant protein, M-CSF, MDC (67 a.a.), MDC (69 a.a.), MCP-1 (MCAF), MCP-2, MCP-3, MCP-4, MDC (67 a.a.), MDC (69 a.a.), MIG, Mff-la, MTP-lp, MIP-3a, M1P-3P, M1P-4, myeloid progenitor inhibitor factor-1 (MPIF-1), NAP-2, Neurturin, Nerve growth factor, P-NGF, NT-3, NT-4, OncostatinM, PDGF-AA, PDGF-AB, PDGF-BB, PF-4, RANTES, SDFla, SDFlp, SCF, SCGF, stem cell factor (SCF), TARC, TGF-a, TGF-p, TGF-P2, TGF-p3, tumour necrosis factor (TNF), TNF-a, TNF-p, TNF receptor I, TNF receptor 11, TNTL-1, TPO, VEGF, VEGF receptor 1, VEGF receptor 2, VEGF receptor 3, GCP-2, GRO/MGSA, GRO-p, GRO-y, HCC1, 1-309, HER 1, HER 2, HER 3, HER 4, TACE recognition site, TNF BP-I and TNF BP-II, as well as any target disclosed in Annex 2 or Annex 3 hereto, whether in combination as set forth in the Annexes, in a different combination or individually. Cytokine receptors include receptors for the foregoing cytokines, e.g. IL-1 Rl; IL-6R; IL-10R; IL-18R, as well as receptors for cytokines set forth in Annex 2 or Annex 3 and also receptors disclosed in Annex 2 and 3. It will be appreciated that this list is by no means exhaustive. Where the multispecific ligand binds to two epitopes (on the same or different antigens), the antigen(s) may be selected from this list.<br>
Advantageously, dual specific ligands may be used to target cytokines and other molecules which cooperate synergistically in therapeutic situations in the body of an organism. The invention therefore provides a method for synergising the activity of two or more cytokines, comprising administering a dual specific ligand capable of binding to said two or more cytokines. In this aspect of the invention, the dual specific ligand may be any dual specific ligand, including a ligand composed of complementary and/or non-complementary domains, a ligand in an open conformation, and a ligand in a closed conformation. For example, this aspect of the invention relates to combinations of VH domains and VL domains, VH domains only and VL domains only.<br>
Synergy in a therapeutic context may be achieved in a number of ways. For example, target combinations may be therapeutically active only if both targets are targeted by the ligand, whereas targeting one target alone is not therapeutically effective. In another embodiment, one target alone may provide some low or minimal therapeutic effect, but together with a second target the combination provides a synergistic increase in therapeutic effect.<br>
Preferably, the cytokines bound by the dual specific ligands of this aspect of the invention are sleeted from the list shown in Annex 2.<br>
Moreover, dual specific ligands may be used in oncology applications, where one specificity targets CD89, which is expressed by cytotoxic cells, and the other is turnour specific. Examples of tumour antigens which may be targeted are given in Annex 3.<br>
In one embodiment of the second configuration of the invention, the variable domains are derived from an antibody dkected against the first and/or second antigen or epitope. In a preferred embodiment the variable domains are derived from a repertoire of single variable antibody domains. In one example, the repertoire is a repertoire that is not created in an animal or a synthetic repertoire. In another example, the single variable domains are not isolated (at least in part) by animal immunisation. Thus, the single domains can be isolated from a naive library.<br>
The second configuration of the invention, in another aspect, provides a multi-specific ligand comprising a first epitope binding domain having a first epitope binding specificity and a non-complementary second epitope binding domain having a second epitope binding specificity. The first and second binding specificities may be the same or different.<br>
In a further aspect, the present invention provides a closed conformation multi-specific ligand comprising a first epitope binding domain having a first epitope binding specificity and a non-complementary second epitope binding domain having a second epitope binding specificity wherein the first and second binding specificities are capable of competing for epitope binding such that the closed conformation multi-specific ligand cannot bind both epitopes simultaneously.<br>
In a still further aspect, the invention provides open conformation ligands comprising non-complementary binding domains, wherein the deomains are specific for a different epitope on the same target. Such ligands bind to targets with increased avidity. Similarly, the invention provides multivalent ligands comprising non-complementary<br>
binding domains specific for the same epitope and directed to targets which comprise multiple copies of said epitope, such as IL-5, PDGF-AA, PDGF-BB, TGF beta, TGF beta2, TGF betaS and TNFoc, for eample human TNF Receptor 1 and human TNFa.<br>
In a similar aspect, ligands according to the invention can be configured to bind individual epitopes with low affinity, such that binding to individual epitopes is not therapeutically significant; but the increased avidity resulting from binding to two epitopes provides a theapeutic benefit. In a perticular example, epitopes may be targetted which are present individually on normal cell types, but present together only on abnormal or diseased cells, such as tumour cells. In such a situaton, only the abnormal or diseased cells are effectively targetted by the bispecific ligands according to the invention.<br>
Ligand specific for multiple copies of the same epitope, or adjacent epitopes, on the same target (known as chelating dAbs) may also be trimeric or polymeric (tertrameric or more) ligands comprising three, four or more non-complementary binding domains. For example, ligands may be constructed comprising three or four VH domains or VL domains.<br>
Moreover, ligands are provided which bind to multisubunit targets, wherein each, binding domain is specific for a subunit of said target. The ligand may be dimeric, trimeric or polymeric.<br>
Preferably, the multi-specific ligands according to the above aspects of the invention are obtainable by the method of the first aspect of the invention.<br>
According to the above aspect of the second configuration of the invention, advantageously the first epitope binding domain and the second epitope binding domains are non-complementary immunoglobulin variable domains, as herein defined. That is either VH~VH or VL-VL variable domains.<br>
Chelating dAbs in particular may be prepared according to a preferred aspect of the invention, namely the use of anchor dAbs, in which a library of dimeric, trimeric or<br>
multimeric dAbs is constructed using a vector which comprises a constant dAb upstream or downstream of a linker sequence, with a repertoire of second, third and further dAbs being inserted on the other side of the linker. For example, the anchor or guiding dAb maybe TAR1-5 (Vie), TAR1-27(VK), TAR2h-5(VH) or TAR2h-6(Vic).<br>
hi alternative methodologies, the use of linkers may be avoided, for example by the use of non-covalent bonding or naturall affinity between binding domains such as VH and VK. The invention accordingly provides a method for preparing a chelating multimeric ligand comprising the steps of:<br>
(a)	providing a vector comprising a nucleic acid sequence encoding a single<br>
binding domain specific for a first epitope on a target;<br>
(b)	providing a vector encoding a repertoire comprising second binding domains<br>
specific for a second epitope on said target, which epitope can be the same or different to<br>
the first epitope, said second epitope being adjacent to said first epitope; and<br>
(c)	expressing said first and second binding domains; and<br>
(d)	isolating those combinations of first and second binding domains which<br>
combine together to produce a target-binding dirtier.<br>
The first and second epitopes are adjacent such that a multimeric ligand is capable of binding to both epitopes simultaneously. This provides the ligand with the advantages of increased avidity if binding. Where the epitopes are the same, the increased avidity is obtained by the presence of multiple copies of the epitope on the target, allowing at least two copies to be simultaneously bound in order to obtain the increased avidity effect.<br>
The binding domains may be associated by several methods, as well as the use of linkers. For example, the binding domains may comprise cys residues, avidin and streptavidin groups or other means for non-covalent attachment post-synthesis; those combinations which bind to the target efficiently will be isolated. Alternatively, a linker may be present between the first and second binding domains, which are expressed as a single polypeptide from a single vector, which comprises the first binding domain, the linker and a repertoire of second binding domains, for instance as described above.<br>
In a preferred aspect, the first and second binding domains associate naturally when bound to antigen; for example, VH and VK domains, when bound to adjacent epitopes, will naturally associate in a three-way interaction to form a stable dirtier. Such associated proteins can be isolated in a target binding assay. An advantage of this procedure is that only binding domains which bind to closely adjacent epitopes, in the correct conformation, will associate and thus be isolated as a result of their increased avidity for the target.<br>
In an alternative embodiment of the above aspect of the second configuration of the invention, at least one epitope binding domain comprises a non-immunoglobulin 'protein scaffold' or 'protein skeleton' as herein defined. Suitable non-immunoglobulin protein scaffolds include but are not limited to any of those selected from the group consisting of: SpA, fibronectin, GroEL and other chaperones, lipocallin, CCTLA4 and affibodies, as set forth above.<br>
According to the above aspect of the second configuration of the invention, advantageously, the epitope binding domains are attached to a 'protein skeleton'. Advantageously, a protein skeleton according to the invention is an immunoglobulin skeleton.<br>
According to the present invention, the term 'immunoglobulin skeleton' refers to a protein which comprises at least one immunoglobulin fold and which acts as a nucleus for one or more epitope binding domains, as defined herein.<br>
Preferred immunoglobunn skeletons as herein defined includes any one or more of those selected from the following: an immunoglobulin molecule comprising at least (i) the CL (kappa or lambda subclass) domain of an antibody; or (ii) the CHI domain of an antibody heavy chain; an immunoglobulin molecule comprising the CHI and CH2 domains of an antibody heavy chain; an immunoglobulin molecule comprising the CHI, CH2 and CHS domains of an antibody heavy chain; or any of the subset (ii) in conjunction with the CL (kappa or lambda subclass) domain of an antibody. A hinge region domain may also be included. Such combinations of domains may, for example, mimic natural antibodies,<br>
such as IgG or IgM, or fragments thereof, such as Fv, scFv, Fab or F(ab')2 molecules. Those skilled in the art will be aware that this list is not intended to be exhaustive.<br>
Linking of the skeleton to the epitope binding domains, as herein defined may be achieved at the polypeptide level, that is after expression of the nucleic acid encoding the skeleton and/or the epitope binding domains. Alternatively, the linking step may be performed at the nucleic acid level. Methods of linking a protein skeleton according to the present invention, to the one or more epitope binding domains include the use of protein chemistry and/or molecular biology techniques which will be familiar to those skilled in the art and are described herein.<br>
Advantageously, the closed conformation multispecific ligand may comprise a first domain capable of binding a target molecule, and a second domain capable of binding a molecule or group which extends the half-life of the ligand. For example, the molecule or group may be a bulky agent, such as HSA or a cell matrix protein.  As used herein, the phrase "molecule or group which extends the half-life of a ligand" refers to a molecule or chemical group which, when bound by a dual-specific ligand as described herein increases the in vivo half-life of such dual specific ligand when administered to an animal, relative to a ligand that does not bind that molecule or group.   Examples of molecules or groups that extend the half-life of a ligand are described hereinbelow. In a preferred embodiment, the closed conformation multispecific ligand may be capable of binding the target molecule only on displacement of the half-life enhancing molecule or group.  Thus, for example, a closed conformation multispecific ligand is maintained in circulation in the bloodstream of a subject by a bulky molecule such as HSA.   When a target molecule is encountered, competition between the binding domains of the closed conformation multispecific ligand results in displacement of the HSA and binding of the target.<br>
Ligands according to any aspect of the present invention, as well as dAb monomers useful in constructing such ligands, may advantageously dissociate from their cognate target(s) with a Kd of SOOnM to 5pM (ie, 3 x 10~7 to 5 x 10"12M), preferably 50nM to20pM, or 5nM to 200pM or InM to lOOpM, 1 x 10~7 M or less, 1 x 10'8 M.or less, 1 x 10"9 M or less, 1 x 10~10 M or less, 1 x 10~n M or less; and/or a K<jff rate constant of x></jff>
10"1 to 1 x 10"7 S'1, preferably 1 x 10"2 to 1 x 10"6 S"1, or 5 x 10"3 to 1 x 10"5 S"1, or 5 x 10" 1 S"1 or less, or 1 x 10"2 S"1 or less, or 1 x 10"3 S"1 or less, or 1 x 10"4 S"1 or less, or 1 x 10"5 S"1 or less, or 1 x 10~6 S"1 or less as determined by surface plasmon resonance. The K<i rate constand is defined as koff></i>
In particular the invention provides an anti-TNFa dAb monomer (or dual specific ligand comprising such a dAb), homodimer, heterodimer or homotrirner ligand, wherein each dAb binds TNFa. The ligand binds to TNFa with a Kd of SOOnM to 5pM (ie, 3 x 10'7 to 5 x 10~1ZM), preferably 50nM to 20pM, more preferably 5nM to 200pM and most preferably InM to lOOpM; expressed in an alternative manner, the K<j is x m or></j>
ft	Q<br>
less, preferably 1x10" M or less, more preferably 1 x 10 M or less, advantageously 1 x 10"10 M or less and most preferably 1 x 10"11 M or less; and/or a Koff rate constant of 5 x 10"1 to 1 x lO^S"1, preferably 1 x 10"2 to 1 x 10"6 S'1, more preferably 5 x 10"3 to 1 x 10"5 S"1, for example 5 x 10"1 S"1 or less, preferably 1 x 10~2 S"1 or less, more preferably 1 x 10" 3 S"1 or less, advantageously 1 x 10"4 S"1 or less, further advantageously 1 x 10"5 S"1 or less, and most preferably 1 x 10"6 S"1 or less, as determined by surface plasmon resonance.<br>
Preferably, the ligand neutralises TNFa in a standard L929 assay with an ND50 of SOOnM to SOpM, preferably or lOOnM to 50pM, advantageously lOnM to lOOpM, more preferably InM to lOOpM; for example 50nM or less, preferably 5nM or less, advantageously SOOpM or less, more preferably 200pM or less and most preferably lOOpMorless.<br>
Preferably, the ligand inhibits binding of TNF alpha to TNF alpha Receptor I (p55 receptor) with an IC50 of SOOnM to SOpM, preferably lOOnM to SOpM, more preferably lOnM to lOOpM, advantageously InM to lOOpM; for example 50nM or less, preferably 5nM or less, more preferably SOOpM or less, advantageously 200pM or less, and most preferably lOOpM or less. Preferably, the TNFa is Human TNFa.<br>
Furthermore, the invention provides a an anti-TNF Receptor I dAb monomer, or dual specific ligand comprising such a dAb, that binds to TNF Receptor I with a Kd of SOOnM to 5pM (ie, 3 x 10"7 to 5 x 10"12M), preferably 50nM to20pM, more preferably 5nM to 200pM and most preferably InM to lOOpM, for example 1 x 10"7 M or less, preferably 1<br>
x 10"8 M or less, more preferably 1 x 10"9 M or less, advantageously 1 x 10~l° M or less and most preferably 1 x 10~n M or less; and/or a K^ff rate constant of 5 x 10"1 to 1 x 10"7 S"1, preferably 1 x 10'2 to 1 x 10'6 S"1, more preferably 5 x KT3 to 1 x 10'5 S"1, for example 5 x 10"1 S"1 or less, preferably 1 x 10"2 S"1 or less, advantageously 1 x 10~3 S"1 or less, more preferably 1 x 10"4 S"1 or less, still more preferably 1 x 10~5 S"1 or less, and most preferably 1 x 10~6 S"1 or less as determined by surface plasmon resonance.<br>
Preferably, the dAb monomeror ligand neutralises TNFoc in a standard assay (eg, the L929 or HeLa assays described herein) with an ND50 of SOOnM to 50pM, preferably lOOnM to 50pM, more preferably lOnM to lOOpM, advantageously InM to lOOpM; for example 50nM or less, preferably 5nM or less, more preferably 500pM or less, advantageously 200pM or less, and most preferably lOOpM or less.<br>
Preferably, the dAb monomer or ligand inhibits binding of TNF alpha to TNF alpha Receptor I (p55 receptor) with an IC50 of SOOnM to 50pM, preferably lOOnM to 50pM, more preferably lOnM to lOOpM, advantageously InM to lOOpM; for example 50nM or less, preferably 5nM or less, more preferably 500pM or less, advantageously 200pM or less, and most preferably lOOpM or less. Preferably, the TNF Receptor I target is Human TNFa.<br>
Furthermore, the invention provides a dAb monomer(or dual specific ligand comprising such a dAb) that binds to serum albumin (SA) with a Kd of InM to SOOjttM (ie, x 10"9 to 5 x 10"4), preferably lOOnM to lOjuM. Preferably, for a dual specific ligand comprising a first anti-SA dAb and a second dAb to another target, the affinity (eg Kd and/or Kog- .as measured by surface plasmon resonance, eg using BiaCore) of the second dAb for its target is from 1 to 100000 times (preferably 100 to 100000, more preferably 1000 to 100000, or 10000 to 100000 times) the affinity of the first dAb for SA. For example, the first dAb binds SA with an affinity of approximately 10/uM, while the second dAb binds its target with an affinity of lOOpM. Preferably, the serum albumin is human serum albumin (HSA).<br>
hi one embodiment, the first dAb (or a dAb monomer) binds SA (egj HSA) with a K<i of approximately preferably and more or nm.></i>
The invention moreover provides dimers, trimers and polymers of the aforementioned dAb monomers, in accordance with the foregoing aspect of the present invention.<br>
Ligands according to the invention, including dAb monomers, dimers and trimers, can be linked to an antibody Fc region, comprising one or both of Cn2 and Cn3 domains, and optionally a hinge region. For example, vectors encoding ligands linked as a single nucleotide sequence to an Fc region may be used to prepare such polypeptides.<br>
In a further aspect of the second configuration of the invention, the present invention provides one or more nucleic acid molecules encoding at least a multispecific ligand as herein defined. In one embodiment, the ligand is a closed conformation ligand. In another embodiment, it is an open conformation ligand. The multispecific ligand may be encoded on a single nucleic acid molecule; alternatively, each epitope binding domain may be encoded by a separate nucleic acid molecule. Where the ligand is encoded by a single nucleic acid molecule, the domains may be expressed as a fusion polypeptide, or may be separately expressed and subsequently linked together, for example using chemical linking agents. Ligands expressed from separate nucleic acids will be linked together by appropriate means.<br>
The nucleic acid may further encode a signal sequence for export of the polypeptides from a host cell upon expression and may be fused with a surface component of a filamentous bacteriophage particle (or other component of a selection display system) upon expression. Leader sequences, which may be used in bacterial expresion and/or phage or phagemid display, include pelB, stll, ompA, phoA, bla and pelA.<br>
In a further aspect of the second configuration of the invention the present invention provides a vector comprising nucleic acid according to the present invention.<br>
hi a yet further aspect, the present invention provides a host cell transfected with a vector according to the present invention.<br>
Expression from sucli a vector may be configured to proauce, ror example on me suriace of a bacteriophage particle, epitope binding domains for selection. This allows selection of displayed domains and thus selection of 'multispecific ligands' using the method of the present invention.<br>
In a preferred embodiment of the second configuration of the invention, the epitope binding domains are immunoglobulin variable regions and are selected from single domain V gene repertoires. Generally the repertoire of single antibody domains is displayed on the surface of filamentous bacteriophage. In a preferred embodiment each single antibody domain is selected by binding of a phage repertoire to antigen.<br>
The present invention further provides a kit comprising at least a multispecific ligand according to the present invention, which may be an open conformation or closed conformation ligand. Kits according to the invention may be, for example, diagnostic kits, therapeutic kits, kits for the detection of chemical or biological species, and the like.<br>
In a further aspect still of the second configuration of the invention, the present invention provides a homogenous immunoassay using a ligand according to the present invention.<br>
In a further aspect still of the second configuration of the invention, the present invention provides a composition comprising a closed conformation multispecific ligand, obtainable by a method of the present invention, and a pharmaceutically acceptable carrier, diluent or excipient.<br>
Moreover, the present invention provides a method for the treatment of disease using a 'closed conformation multispecific ligand' or a composition according to the present invention.<br>
In a preferred embodiment of the invention the disease is cancer or an inflammatory disease, eg rheumatoid arthritis, asthma or Crohn's disease.<br>
In a further aspect of the second configuration of the invention, the present invention provides a method for the diagnosis, including diagnosis of disease using a closed<br>
conformation multispecific ligand, or a composition according to the present invention. Thus in general the binding of an analyte to a closed conformation multispecific ligand may be exploited to displace an agent, which leads to the generation of a signal on displacement. For example, binding of analyte (second antigen) could displace an enzyme (first antigen) bound to the antibody providing the basis for an immunoassay, especially if the enzyme were held to the antibody through its active site.<br>
Thus hi a final aspect of the second configuration, the present invention provides a method for detecting the presence of a target molecule, comprising:<br>
(a)	providing a closed conformation multispecific ligand bound to an agent, said ligand<br>
being specific for the target molecule and the agent, wherein the agent which is bound by<br>
the ligand leads to the generation of a detectable signal on displacement from the ligand;<br>
(b)	exposing the closed conformation multispecific ligand to the target molecule; and<br>
(c)	detecting the signal generated as a result of the displacement of the agent.<br>
According to the above aspect of the second configuration of the invention, advantageously, the agent is an enzyme, which is inactive when bound by the closed conformation multi-specific ligand. Alternatively, the agent may be any one or more selected from the group consisting of the following: the substrate for an enzyme, and a fluorescent, luminescent or chromogenic molecule which is inactive or quenched when bound by the ligand.<br>
Sequences similar or homologous (e.g., at least about 70% sequence identity) to the sequences disclosed herein are also part of the invention. In some embodiments, the sequence identity at the amino acid level can be about 80%, 85%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99% or higher. At the nucleic acid level, the sequence identity can be about 70%, 75%, 80%, 85%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99% or higher. Alternatively, substantial identity exists when the nucleic acid segments will hybridize under selective hybridization conditions (e.g., very high stringency hybridization conditions), to the complement of the strand. The nucleic acids may be present in whole cells, in a cell lysate, or hi a partially purified or substantially pure form.<br>
Calculations of "homology" or "sequence identity" or "similarity" between two sequences (the terms are used interchangeably herein) are performed as follows.   The sequences are aligned for optimal comparison purposes (e.g., gaps can be introduced in one or both of a first and a second amino acid or nucleic acid sequence for optimal alignment and non-homologous sequences can be disregarded for comparison purposes). In a preferred embodiment, the length of a reference sequence aligned for comparison purposes is at least 30%, preferably at least 40%, more preferably at least 50%, even more preferably at least 60%, and even more preferably at least 70%, 80%, 90%, 100% of the length of the reference sequence.     The  amino   acid residues  or nucleotides   at corresponding amino acid positions or nucleotide positions are then compared.  When a position in the first sequence is occupied by the same amino acid residue or nucleotide as the corresponding position in the second sequence, then the molecules are identical at that position (as used herein amino acid or nucleic acid "homology" is equivalent to amino acid or nucleic acid  "identity").  The percent identity between the two sequences is a function of the number of identical positions shared by the sequences, taking into account the number of gaps, and the length of each gap, which need to be introduced for optimal alignment of the two sequences.<br>
Advantageously, the BLAST algorithm (version 2.0) is employed for sequence alignment, with parameters set to default values. The BLAST algorithm is described in detail at the world wide web site ("www") of the National Center for Biotechnology Information (".ncbi") of the National Institutes of Health ("nih") of the U.S. government (".gov"), in the "/Blast/" directory, in the "blast_help.html" file. The search parameters are defined as follows, and are advantageously set to the defined default parameters.<br>
BLAST (Basic Local Alignment Search Tool) is the heuristic search algorithm employed by the programs blastp, blasm, blastx, tblastn, and tblastx; these programs ascribe significance to their findings using the statistical methods of Karlin and Altschul, 1990, Proc. Natl. Acad. Sci. USA 87(6):2264-8 (see the "blast_help.html" file, as described above) with a few enhancements. The BLAST programs were tailored for sequence similarity searching, for example to identify homologues to a query sequence. The programs are not generally useful for motif-style searching. For a discussion of basic issues in similarity searching of sequence databases, see Altschul et al. (1994).<br>
The  five  BLAST  programs   available  at  the  National  Center  for  Biotechnology<br>
Information web site perform the following tasks:<br>
"blastp" compares an amino acid query sequence against a protein sequence database;<br>
"blastn" compares a nucleotide query sequence against a nucleotide sequence database;<br>
"blastx" compares the six-frame conceptual translation products of a nucleotide query<br>
sequence (both strands) against a protein sequence database;<br>
"tblastn" compares a protein query sequence against a nucleotide sequence database<br>
dynamically translated in all six reading frames (both strands).<br>
"tblastx" compares the six-frame translations of a nucleotide query sequence against the<br>
six-frame translations of a nucleotide sequence database. BLAST uses the following search parameters:<br>
HISTOGRAM Display a histogram of scores for each search; default is yes. (See parameter H in the BLAST Manual).<br>
DESCRIPTIONS Restricts the number of short descriptions of matching sequences reported to the number specified; default limit is 100 descriptions. (See parameter V in the manual page). See also EXPECT and CUTOFF.<br>
ALIGNMENTS Restricts database sequences to the number specified for which high-scoring segment parrs (HSPs) are reported; the default limit is 50. If more database sequences than this happen to satisfy the statistical significance threshold for reporting (see EXPECT and CUTOFF below), only the matches ascribed the greatest statistical significance are reported. (See parameter B hi the BLAST Manual). EXPECT The statistical significance threshold for reporting matches against database sequences; the default value is 10, such that 10 matches are expected to be found merely by chance, according to the stochastic model of Karlin and Altschul (1990). If the statistical significance ascribed to a match is greater than the EXPECT threshold, the match will not be reported. Lower EXPECT thresholds are more stringent, leading to fewer chance matches being reported. Fractional values are acceptable. (See parameter E in the BLAST Manual).<br>
CUTOFF Cutoff score for reporting high-scoring segment pairs. The default value is calculated from the EXPECT value (see above). HSPs are reported for a database sequence only if the statistical significance ascribed to them is at least as high as would be ascribed to a lone HSP having a score equal to the CUTOFF value. Higher CUTOFF<br>
values are more stringent, leading to fewer chance matches being reported. (See parameter S in the BLAST Manual). Typically, significance thresholds can be more intuitively managed using EXPECT.<br>
MATRIX Specify an alternate scoring matrix for BLASTP, BLASTX, TBLASTN and TBLASTX. The default matrix is BLOSUM62 (Henikoff &amp; Henikoff, 1992, Proc. Natl. Aacad. Sci. USA 89(22): 10915-9). The valid alternative choices include: PAM40, PAM120, PAM250 and IDENTITY. No alternate scoring matrices are available for BLASTN; specifying the MATRIX directive in BLASTN requests returns an error response.<br>
STRAND Restrict a TBLASTN search to just the top or bottom strand of the database sequences; or restrict a BLASTN, BLASTX or TBLASTX search to just reading frames on the top or bottom strand of the query sequence.<br>
FILTER Mask off segments of the query sequence that have low compositional complexity, as determined by the SEG program of Wootton &amp; Federhen (1993) Computers and Chemistry 17:149-163, or segments consisting of short-periodicity internal repeats, as determined by the XNU program of Claverie &amp; States, 1993, Computers and Chemistry 17:191-201, or, for BLASTN, by the DUST program of Tatusov and Lipman (see the world wide web site of the NCBI). Filtering can eliminate statistically significant but biologically uninteresting reports from the blast output (e.g., hits against common acidic-, basic- or proline-rich regions), leaving the more biologically interesting regions of the query sequence available for specific matching against database sequences.<br>
Low complexity sequence found by a filter program is substituted using the letter "N" in nucleotide sequence (e.g., "N" repeated 13 tunes) and the letter "X" in protein sequences (e.g., "X" repeated 9 times).<br>
Filtering is only applied to the query sequence (or its translation products), not to database sequences. Default filtering is DUST for BLASTN, SEG for other programs. It is not unusual for nothing at all to be masked by SEG, XNU, or both, when applied to sequences in SWISS-PROT, so filtering should not be expected to always yield an effect. Furthermore, in some cases, sequences are masked in their entirety, indicating that the<br>
statistical significance of any matches reported against the unfiltered query sequence should be suspect.<br>
NCBI-gi Causes NCBI gi identifiers to be shown in the output, in addition to the accession and/or locus name.<br>
Most preferably, sequence comparisons are conducted using the simple BLAST search algorithm provided at the NCBI world wide web site described above, in the "/BLAST" directory.<br>
According to a further aspect the present invention provides a dual specific ligand comprising a first single immunoglobulin variable domain having a binding specificity to a first antigen or epitope and a second immunoglobulin single variable doamin having a binding activity to a second antigen or epitope wherein said first and second domains lack mutually complementary domains which share the same specificity.<br>
According to the above aspect of the invention, preferably a dual-specific ligand has an IgG format which comprises two complementary pairs of mammalian dAbs wherein each Dab comprising each complementary pair has a different target binding specificity. Advantageously a dual specific molecule according to this embodiment of the invention comprises one or more Dabs which exhibits an epitope binding specificity of of 50nM or more.<br>
According to the above aspect of the invention the two different dAbs may be both VH domains, both VL domains or at least one VH and a VL domain.<br>
According to the above aspect of the invention, preferably the dual-specific ligand comprises at least one pair of Dabs which are complementary to one another.<br>
Preferably a dual-specific ligand having an IgG format as described binds to its respective targets in a non-competitive manner.<br>
In an alternative embodiment of the above aspect of the invention, a dual-specific liagnd having an IgG format as described above binds to its respective targets in a competitive manner.<br>
Advantageously a dual-specific ligand according to the above aspect of the invention has an IgG format and comprises one pair of identical dAbs which can bind simultaneously to two copies of the corresponding target.<br>
More advantageously, a dual-specific ligand according to the above aspect of the invention comprise two pairs of identical dAbs wherein both pairs of identical dAbs can bind simultaneously to two copies of the corresponding targets<br>
Advantageously, a dual-specific ligand according to the above aspect of the invention comprises 4 identical dAbs, preferably mammalian Dabs.<br>
hi an alternative embodiment of the above aspect of the invention, a dual-specific molecule according to the invention has a Fab format.<br>
Advantageously, a dual-specific ligand having a Fab format as herein described binds to its respective targets hi a non-competitive manner.<br>
In an alternative embodiment of the above aspect of the invention, a dual-specific liagnd having a Fab format as described above binds to its respective targets in a competitive manner.<br>
Suitable targets for the dual-specific ligands according to the aspect of the invention described above inlcude any one or more of those in the list consisting of the following:. One skilled in the art will appreciate that the choice of epitopes and antigens is large and varied. They may be for instance human or animal proteins, cytokines, cytokine receptors, enzymes co-factors for enzymes or DNA binding proteins. Suitable cytokines and growth factors include but are not limited to: ApoE, Apo-SAA, BDNF, Cardiotrophin-1, EGF, EGF receptor, ENA-78, Eotaxin, Eotaxin-2, Exodus-2, EpoR, FGF-acidic, FGF-basic, fibroblast growth factor-10, FLT3 ligand, Fractalkine (CX3C), GDNF, G-CSF,<br><br>
GM-CSF, GF-pl, insulin, IFN-y, IGF-I, IGF-H, IL-la, IL-1B, IL-2, DL-3, IL-4, IL-5, IL-<br>
6, IL-7, IL-8 (72 a.a.), IL-8 (77 a.a.)3 IL-9, IL-10, IL-11, IL-12, IL-13, IL-15, IL-16, EL-<br>
17, IL-18 (IGIF), Inhibin a, Inhibin p, IP-10, keratinocyte growth factor-2 (KGF-2),<br>
KGF, Leptin, LIF, Lymphotactin, Mullerian inhibitory substance, monocyte colony<br>
inhibitory factor, monocyte attractant protein, M-CSF, MDC (67 a.a.), MDC (69 a.a.),<br>
MCP-1 (MCAF), MCP-2, MCP-3, MCP-4, MDC (67 a.a.), MDC (69 a.a.), MIG, MIP-<br>
loc, MIP-1B, MIP-3a, MTP-3B, MIP-4, myeloid progenitor inhibitor factor-1 (MPIF-1),<br>
NAP-2, Neurturin, Nerve growth factor, p-NGF, NT-3, NT-4, Oncostatin M, PDGF-AA,<br>
PDGF-AB, PDGF-BB, PF-4, RANTES, SDFla, SDFIB, SCF, SCGF, stem cell factor<br>
(SCF), TARC, TGF-a, TGF-P, TGF-p2, TGF-p3, tumour necrosis factor (TNF), TNF-a,<br>
TNF-p, TNF receptor I, TNF receptor E, TNIL-1, TPO, VEGF, VEGF receptor 1, VEGF<br>
receptor 2, VEGF receptor 3, GCP-2, GRO/MGSA, GRO-p, GRO-y, HCC1, 1-309, HER<br>
1, HER 2, HER 3, HER 4, TACE recognition site, TNF BP-I and TNF BP-H, as well as<br>
any target disclosed in Annex 2 or Annex 3 hereto, whether in combination as set forth in<br>
the Annexes, in a different combination or individually.<br>
Advantageously, according to the final aspect of the invention, preferably the dual-specifc ligands exhibit the ability to neutralise in vitro or in cell based assays<br>
Ligands according to any aspect of the present invention, as well as dAb monomers useful in constructing such ligands, may advantageously dissociate from their cognate target(s) with a Kj of 300nM to 5pM (ie, 3 x 10"7 to 5 x 10"12M), preferably 50nM to20pM, or 5nM to 200pM or InM to lOOpM, 1 x 10'7 M or less, 1 x 10'8 M or less, 1 x 10"9 M or less, 1 x 10"10 M or less, 1 x 10"11 M or less; and/or a Koff rate constant of 5 x 10'1 to 1 x 10'7 S"1, preferably 1 x 10"2 to 1 x 10"6 S"1, or 5 x 10"3 to 1 x 10"5 S"1, or 5 x 10'1 S"1 or less, or 1 x 10"2 S"1 or less, or 1 x 10"3 S"1 or less, or 1 x 10"4 S"1 or less, or 1 x 10~5 S"1 or less, or 1 x 10"6 S"1 or less as determined by surface plasmon resonance. The Kd rate constand is defined as Koff/Kon.<br>
In particular the invention provides a dual-specific ligand wherein the affinity of binding to target with a Kd of 300nM to 5pM (ie, 3 x 10"7 to 5 x 10'12M), preferably 50nM to 20pM, more preferably 5nM to 200pM and most preferably InM to lOOpM; expressed in<br>
7	Q<br>
an alternative manner, the K<j is x m or less preferably more></j><br>
preferably 1 x 10"9 M or less, advantageously 1 x 1CT10 M or less and most preferably 1 x 10"11 M or less; and/or a Koff rate constant of 5 x 10"1 to 1 x 10"7 S"1, preferably 1 x 10~2 to 1 x 10"6 S"1, more preferably 5 x 10"3 to 1 x 10'5 S'1, for example 5 x 10"1 S"1 or less, preferably 1 x 10"2 S"1 or less, more preferably 1 x 10"3 S"1 or less, advantageously 1 x 10"4 S"1 or less, further advantageously 1 x 10~5 S"1 or less, and most preferably 1 x 10"6 S"1 or less, as determined by surface plasmon resonance.<br>
Preferably, the ligand neutralises TNFoc in a standard L929 assay with an ND50 of 500nM to 50pM, preferably or lOOnM to 50pM, advantageously lOnM to lOOpM, more preferably InM to lOOpM; for example 50nM or less, preferably 5nM or less, advantageously SOOpM or less, more preferably 200pM or less and most preferably lOOpMorless.<br>
Preferably, the ligand inhibits binding of TNF alpha to TNF alpha Receptor I (p55 receptor) with an IC50 of SOOnM to 50pM, preferably lOOnM to 50pM, more preferably lOnM to lOOpM, advantageously InM to lOOpM; for example 50nM or less, preferably 5nM or less, more preferably SOOpM or less, advantageously 200pM or less, and most preferably lOOpM or less. Preferably, the TNFa is Human TNFa.<br>
According to the above apsect of the invention dual-specific ligands preferably exhibit a binding affinity of at least 50nM.<br>
hi a preferred aspect, the invention relates to a dual specific ligand which binds to a target ligand and a receptor for the target ligand. For example, the ligand may be TNFa and the target ligand recpetor may be TNF Receptor 1. Advantageously, the dual specific ligand according to the invention is able to bind both the target ligand and the target ligand receptor simultaneously, i.e. is in an open configuration.<br>
According to the present invention, advantageously a dual-specific ligand as described herein is a TAR1/TAR2 dual specific Fab, F(ab')2 or IgG as herein described and is specific for human TNF alpha and the human TNFR1 (p55 receptor). Preferably, each, arm comprises a complementary VH/VL pair. More preferably, the VL of each pair is Vk. More preferably still the VK has TNF as target and the VH of each pair has the p55<br>
receptor as a target. According to these Fab or IgG formats the dAbs advantageously bind their targets simunltaneosuly, that is with no significant competition.<br>
Most advantageously a TAR1/TAR2 IgG or Fab format dual-specific ligand is as described herein in the Examples.<br>
*<br>
Those skilled in the art will appreciate that the vectors/constructs provided in the Examples and used for the eneration of TAR1/TAR2 dual-specific ligands and the dAbs comprising them represent a mere sample of suitable vectors/constructs for use according to the above aspect of the invention. Vectors/contructs suitable for use include the following:<br>
(a) Eukaryotic leader-VH or VL-CH1-hinge-CH2-CH3. In this embodiment the leader may be mammalian, for example a CD33 or IgG K leader or functional variant/fragment of these, or at least 80% homologous with any of these leaders.<br>
According to the present invention there is also provided an expression vector, preferably yeast or mammalian in nature comprising a construct as described above in (a).<br>
According to a further aspect still, there is provided a host, prefererably mammalian cells such as Cos cells comprising a vector as described above.<br>
In a final aspect of the invention there is provided a VH dAb monomer designated TAR2h-10-27 dAb having the amino acid sequence given below (a) and which binds to the human TNF receptor 1 (p55 receptor):<br>
EVQLLESGGGLVQPGGSLRLSCAASGFTFEWYWMGWVRQAPGKGLEWVSAISG<br>
SGGSTYYADSVKGRFTISRDNSKNTLYLQMNSLRAEDAAVYYCAKVKLGGGPN<br>
FGYRGQGTLVTVSSAA<br>
TAR2h-10-27 nucleic acid coding sequence<br>
GAGGTGCAGCTGTTGGAGTCTGGGGGAGGCTTGGTACAGCCiGUGLrUUiL;^ CTGCGTCTCTCCTGTGCAGCCTCCGGATTCACCTTTGAGTGGTATTGGATGGG TTGGGTCCGCCAGGCTCCAGGGAAGGGTCTAGAGTGGGTCTCAGCTATCAGT GGTAGTGGTGGTAGCACATACTACGCAGACTCCGTGAAGGGCCGGTTCACCA TCTCCCGCGACAATTCCAAGAACACGCTGTATCTGCAAATGAACAGCCTGCGT GCCGAGGACGCCGCGGTATATTACTGTGCGAAAGTTAAGTTGGGGGGGGGGC CTAATTTTGGCTACCGGGGCCAGGGAACCCTGGTCACCGTCTCGAGCGCGGC CGC<br>
Advantageously, this dAb is comprised within a dual-specific ligand. Dual specific ligands include scFv, Fab and Ig molecules, and may be in open or closed conformations. Particularly preferred are dual specific Fab and IgG formats, comprising complementary TAR1-5-19 VK and TAR2h-10-27 VH domains. Advantageously, the polypeptide is in an open conformation.<br>
Brief Description of the Figures<br>
Figure 1	shows the diversification of Vn/HSA at positions H50, H52, H52a, H53,<br>
H55, H56, H58, H95, H96, H97, H98 (DVT or NNK encoded respectively) which are in the antigen binding site of VH HSA. The sequence of VK is diversified at positions L5Q, L53.<br>
Figure 2	shows Library 1: Germline Vic/DVT VH,<br>
Library 2: Germline VK /NNK VH,<br>
Library 3: Germline VH/D VT VK<br>
Library 4: Germline VH/NNK VK<br>
In phage display/ScFv format. These libraries were pre-selected for binding to generic ligands protein A and protein L so that the majority of the clones and selected libraries are functional. Libraries were selected on HSA (first round) and /3-gal (second round) or HSA /3-gal selection or on /3-gal (first round) and HSA (second round) /3-gal HSA selection. Soluble<br>
scFv from these clones of PCR are amplified in the sequence. One clone encoding a dual specific antibody K8 was chosen for further work.<br><br>
Figure 3<br><br>
shows an alignment of VH chains and VK chains.<br><br>
Figure 4 shows the characterisation of the binding properties of the K8 antibody, the binding properties of the K8 antibody characterised by monoclonal phage ELISA, the dual specific K8 antibody was found to bind HSA and /8-gal and displayed on the surface of the phage with absorbant signals greater than 1.0. No cross reactivity with other proteins was detected.<br>
FigureS	shows soluble scFv ELISA performed using known concentrations of the<br>
K8 antibody fragment. A 96-well plate was coated with lOCtyig of HSA, BSA and /3-gal at 10/itg/ml and 10(tyig/ml of Protein A at 1/ig/ml concentration. 50/zg of the serial dilutions of the K8 scFv was applied and the bound antibody fragments were detected with Protein L-HRP. ELISA results confirm the dual specific nature of the K8 antibody.<br>
Figure 6 shows the binding characteristics of the clone K8 VK/dummy VH analysed using soluble scFv ELISA. Production of the soluble scFv fragments was induced by IPTG as described by Harrison et al, Methods Enzymol. 1996;267:83-109 and the supernatant containing scFv assayed directly. Soluble scFv ELISA is performed as described in example 1 and the bound scFvs were detected with Protein L-HRP. The ELISA results revealed that this clone was still able to bind /8-gal, whereas binding BSA was abolished.<br>
Figure_7         shows the sequence of variable domain vectors 1 and 2.<br>
FigureS         is a map of the CH vector used to construct a VH1/VH2 multipsecific ligand.<br>
Figure9         is a map of the VK vector used to construct a VK1/VK2 multispecific ligand.<br>
Figure 10        TNF receptor assay comparing TAR1-5 dimer 4, TAR1-5-19 dimer 4 and TAR1-5-19 monomer.<br>
Figure 11 TNF receptor assay comparing TAR1-5 dimers 1-6.A11 dimers have been FPLC purified and the results for the optimal dimeric species are shown.<br>
Figure 12 TNF receptor assay of TAR1-5 19 homodimers in different formats: dAb-linker-dAb format with 3U, 5U or 7U linker, Fab format and cysteine hinge linker format.<br>
Figure 13 Dummy VH sequence for library 1. The sequence of the VH framework based on germline sequence DP47 - JH4b. Positions where NNK randomisation (N=A or T or C or G nucleotides; K = G or T nucleotides) has been incorporated into library 1 are indicated in bold underlined text.<br>
Figure 14 Dummy VH sequence for library 2. The sequence of the VH framework based on germline sequence DP47 - JH4b. Positions where NNK randomisation (N=A or T or C or G nucleotides; K = G or T nucleotides) has been incorporated into library 2 are indicated in bold underlined text.<br>
Figure 15 Dummy V/c sequence for library 3. The sequence of the VK framework based on germline sequence DPs9 - J K!- Positions where NNK randomisation (N=A or T or C or G nucleotides; K = G or T nucleotides) has been incorporated into library 3 are indicated in bold underlined text.<br>
Figure 16 Nucleotide and arnino acid sequence of anti MSA dAbs MSA 16 and MSA 26.<br>
Figure 17 Inhibition biacore of MSA 16 and 26. Purified dAbs MSAl6 and MSA26 were analysed by inhibition biacore to determine Ka. Briefly, the dAbs were tested to determine the concentration of dAb required to achieve 200RUs of response on a biacore CMS chip coated with a high density of MSA. Once the required concentrations of dAb had been determined,<br>
MSA antigen at a range of concentrations around the expected Kd was premixed with the dAb and incubated overnight. Binding to the MSA coated biacore chip of dAb in each of the premixes was then measured at a high flow-rate of 30 /il/minute.<br>
Figure 18 Serum levels of MSA16 following injection. Serum half life of the dAb MSA16 was determined in mouse. MSA16 was dosed as single i.v. injections at approx 1.5rng/kg into GDI mice. Modelling with a 2 compartment model showed MSA16 had a tl/2a of 0.98hr, a tl/2/3 of 36.5hr and an AUC of 913hr.mg/ml. MSA16 had a considerably lengthened half life compared with HEL4 (an anti-hen egg white lysozyme dAb) which had a tl/2a of 0.06hr and a tl/2/3 of 0.34hr.<br>
Figure 19       ELISA (a) and TNF receptor assay (c) showing inhibition of TNF binding with a Fab-like fragment comprising MSA26Ck and TAR1-5-19CH. Addition of MSA with the Fab-like fragment reduces the level of inhibition. An  ELISA plate coated with 1/jg/ml TNFa was probed with dual specific VK CH and VK CK Fab like fragment and also with a control TNFa binding dAb at a concentration calculated to give a similar signal on the ELISA. Both the dual specific and control dAb were used to probe the ELISA plate in the presence and in the absence of 2mg/ml MSA.   The signal in the dual specific well was reduced by more than 50% but the signal in the dAb well was not reduced at all (see figure 19a). The same dual specific protein was also put into the receptor assay with and without MSA and competition by MSA was also shown (see figure 19c).   This demonstrates that binding of MSA to the dual specific is competitive with binding to TNFa.<br>
Figure 20        TNF receptor assay showing inhibiton of TNF binding with a disulphide bonded heterodimer of TAR1-5-19 dAb and MSA16 dAb.   Addition of MSA with the dimer reduces the level of inhibiton in a dose dependant manner. The TNF receptor assay (figure 19 (b)) was conducted in the presence of a constant concentration of heterodimer (18nM) and a dilution<br>
series of MSA and HSA. The presence of HSA at a range of concentrations (up to 2 mg/ml) did not cause a reduction in the ability of the dimer to inhibit TNFa. However, the addition of MSA caused a dose dependant reduction in the ability of the dimer to inhibit TNFa (figure 19a).This demonstrates that MSA and TNFa compete for binding to the cys bonded TAR1-5-19, MSA16 dimer. MSA and HSA alone did not have an effect on the TNF binding level in the assay.<br>
Figure 21:       Shows the vectors used for Fab construction according to the invention.<br>
Figure 22:	Shows the binding of Fab comprising TAR1/TAR2 Dabs to TNF<br>
and TNFR1 via an ELISA assay.<br>
Figure 23: Shows the results of sandwich ELISA to test the ability of TAR1/TAR2 Fab to bind to both TNF and TNFR antigens simultaneously, that is to test whether the Fab is of open or closed conformation.<br>
Figure 24: Shows the results of competition ELISA to test the ability of TAR1/TAR2 Fab • to bind to both antigens simultaneously, that is to test whether the Fab is of open or closed conformation.<br>
Figure 25: Shows the rresults of cell based assays using Fab dual specific ligands<br>
according to the invention:<br>
(a)	to test human TNF cytotoxicity on murine cells<br>
(b)	shows a murine TNF cytotoxicity assay on murine cells with human soluble TAR2.<br>
(c)	Shows murine TNF induced IL-8 secretion on human cells.<br>
(d)	Shows human TNF induced IL-8 secretion on human cells.<br>
Figure 26: Shows murine TNF cytoxicity on murine cells with soluble human TNFR1 and increasing concentrations of mutant TNF (competition on cells).<br>
Figure 27: shows the construction of IgG vectors which express IgGl heavy chain constant region and light chain kappa constant region respectively.<br>
Figure 28 shows the binding of TAR1/TAR2 IgG to TNF and TNFR1 in ELISA assay.<br>
Figure 29: Shows the analysis of TAR1/TAR2 IgG properties in cell assays.<br>
(a)	Human TNF cytotoxicity on murine cells.<br>
(b)	Murine TNF cytotoxicity assay on murine cells with human soluble TNF receptor.<br>
(c)	Human TNF induced IL-8 release from human cells.<br>
(d)	Murine TNF induced IL-8 secretion from human cells.<br>
Figure 30: Shows Human TNF induced IL-8 secretion on human cells<br>
Figure 31: Shows the amino acid sequence of the Dab designated TAR2 which binds to human TNFR1 (p55 receptor).<br>
Detailed Description of the Invention Definitions<br>
Complementary Two imniunoglobulin domains are "complementary" where they belong to families of structures which form cognate pairs or groups or are derived from such families and retain this feature. For example, a VH domain arid a VL domain of an antibody are complementary; two VH domains are not complementary, and two VL domains are not complementary. Complementary domains may be found in other members of the immunoglobulin superfamily, such as the Va and Vp (or y and 6) domains of the T-cell receptor. In the context of the second configuration of the present invention, non-complementary domains do not bind a target molecule cooperatively, but act independently on different target epitopes which may be on the same or different molecules. Domains which are artificial, such as domains based on protein scaffolds which do not bind epitopes unless engineered to do so, are non-complementary. Likewise, two domains based on (for example) an immunoglobulin domain and a fibronectin domain are not complementary.<br>
Immunoglobulin This refers to a family of polypeptides which retain the immunoglobulin fold characteristic of antibody molecules, which contains two P sheets and, usually, a conserved disulphide bond. Members of the immunoglobulin superfamily are involved in many aspects of cellular and non-cellular interactions in vivo, including widespread roles in the immune system (for example, antibodies, T-cell receptor molecules and the like), involvement in cell adhesion (for example the ICAM molecules) and intracellular signalling (for example, receptor molecules, such as the PDGF receptor). The present invention is applicable to all irnmunoglobulin superfamily molecules which possess binding domains. Preferably, the present invention relates to antibodies.<br>
Combining Variable domains according to the invention are combined to form a group of domains; for example, complementary domains may be combined, such as VL domains being combined with VH domains. Non-complementary domains may also be combined. Domains may be combined in a number of ways, involving linkage of the domains by covalent or non-covalent means.<br><br>
Domain	A domain is a folded protein structure which retains its tertiary structure<br>
independently of the rest of the protein. Generally, domains are responsible for discrete functional properties of proteins, and in many cases may be added, removed or transferred to other proteins without loss of function of the remainder of the protein and/or of the domain. By single antibody variable domain is meant a folded polypeptide domain comprising sequences characteristic of antibody variable domains. It therefore includes complete antibody variable domains and modified variable domains, for example in which one or more loops have been replaced by sequences which are not characteristic of antibody variable domains, or antibody variable domains which have been truncated or comprise N- or C-terminal extensions, as well as folded fragments of variable domains which retain at least in part the binding activity and specificity of the full-length domain.<br>
Repertoire A collection of diverse variants, for example polypeptide variants which differ hi their primary sequence. A library used in the present invention will encompass a repertoire of polypeptides comprising at least 1000 members.<br>
Library         The term library refers to a mixture of heterogeneous polypeptides or nucleic acids. The library is composed of members, each of which have a single polypeptide or nucleic acid sequence. To this extent, library is synonymous with repertoire. Sequence differences between library members are responsible for the diversity present in the library. The library may take the form of a simple mixture of polypeptides or nucleic acids, or may be in the form of organisms or cells, for example bacteria, viruses, animal or plant cells and the like, transformed with a library of nucleic acids. Preferably, each individual organism or cell contains only one or a limited number of library members. Advantageously, the nucleic acids are incorporated into expression vectors, in order to allow expression of the polypeptides encoded by the nucleic acids. In a preferred aspect, therefore, a library may take the form of a population of host organisms, each organism containing one or more copies of an expression vector containing a single member of the library in nucleic acid form which can be expressed to produce its corresponding polypeptide member. Thus, the population of host organisms has the potential to encode a large repertoire of genetically diverse polypeptide variants.<br>
A 'closed conformation multi-specific ligand' describes a multi-specific ligand as herein defined comprising at least two epitope binding domains as herein defined. The term 'closed conformation' (multi-specific ligand) means that the epitope binding domains of the ligand are arranged such that epitope binding by one epitope binding domain competes with epitope binding by another epitope binding domain. That is, cognate epitopes may be bound by each epitope binding domain individually but not simultaneosuly. The closed conformation of the ligand can be achieved using methods herein described.<br>
Antibody An antibody (for example IgG, IgM, IgA, IgD or IgE) or fragment (such as a Fab , F(ab')a, Fv, disulphide linked Fv, scFv, closed conformation multispecific antibody, disulphide-linked scFv, diabody) whether derived from any species naturally producing an antibody, or created by recombinant DNA technology; whether isolated from serum, B-cells, hybridomas, transfectomas, yeast or bacteria).<br>
Dual-specific ligand A ligand comprising a first immunoglobulin single variable domain and a second immunoglobulin single variable domain as herein defined, wherein the variable regions are capable of binding to two different antigens or two epitopes on the same antigen which are not normally bound by a monospecific immunoglobulin. For example, the two epitopes may be on the same hapten, but are not the same epitope or sufficiently adjacent to be bound by a monospecific ligand. The dual specific ligands according to the invention are composed of variable domains which have different specificities, and do not contain mutually complementary variable domain pairs which have the same specificity.<br>
Antigen A molecule that is bound by a ligand according to the present invention. Typically, antigens are bound by antibody ligands and are capable of raising an antibody response in vivo. It may be a polypeptide, protein, nucleic acid or other molecule. Generally, the dual specific ligands according to the invention are selected for target specificity against a particular antigen. In the case of conventional antibodies and fragments thereof, the antibody binding site defined by the variable loops (LI, L2, L3 and HI, H2, H3) is capable of binding to the antigen.<br>
Epitope A unit of structure conventionally bound by an immunoglobuh'n VH/VL pair. Epitopes define the minimum binding site for an antibody, and thus represent the target of specificity of an antibody. In the case of a single domain antibody, an epitope represents the unit of structure bound by a variable domain in isolation.<br>
Generic ligand A ligand that binds to all members of a repertoire. Generally, not bound through the antigen binding site as defined above. Non-limiting examples include protein A, protein L and protein G.<br>
Selecting Derived by screening, or derived by a Darwinian selection process, in which binding interactions are made between a domain and the antigen or epitope or between an antibody and an antigen or epitope. Thus a first variable domain may be selected for binding to an antigen or epitope in the presence or in the absence of a complementary variable domain.<br>
Universal framework	A single antibody framework sequence corresponding to<br>
the regions of an antibody conserved hi sequence as defined by Kabat ("Sequences of Proteins of Immunological Interest", US Department of Health and Human Services) or corresponding to the human germline immunoglobulin repertoire or structure as defined by Chothia and Lesk, (1987) J. Mol. Biol. 196:910-917. The invention provides for the use of a single framework, or a set of such frameworks, which has been found to permit the derivation of virtually any binding specificity though variation in the hypervariable regions alone.<br>
Half-life The time taken for the serum concentration of the ligand to reduce by 50%, in vivo, for example due to degradation of the ligand and/or clearance or sequestration of the ligand by natural mechanisms. The ligands of the invention are stabilised in vivo and their half-life increased by binding to molecules which resist degradation and/or clearance or sequestration. Typically, such molecules are naturally occurring proteins which themselves have a long half-life in vivo. The half-life of a ligand is increased if its functional activity persists, in vivo, for a longer period than a similar ligand which is not specific for the half-life increasing molecule. Thus, a ligand specific for HSA and a target molecule is compared with the same ligand wherein the specificity for HSA is not<br>
present, that it does not bind HSA but binds another molecule. For example, it may bind a second epitope on the target molecule. Typically, the half life is increased by 10%, 20%, 30%, 40%, 50% or more. Increases in the range of 2x, 3x, 4x, 5x, lOx, 20x, 30x, 40x, 50x or more of the half life are possible. Alternatively, or in addition, increases in the range of up to 30x, 40x, 50x, 60x, 70x, 80x, 90x, lOOx, 150x of the half life are possible.<br>
Homogeneous immunoassay	An immunoassay in  which  analyte  is  detected<br>
without need for a step of separating bound and un-bound reagents.<br>
Substantially identical (or "substantially homologous")	A first amino acid or<br>
nucleotide sequence that contains a sufficient number of identical or equivalent (e.g., with a similar side chain, e.g., conserved amino acid substitutions) amino acid residues or nucleotides to a second ammo acid or nucleotide sequence such that the first and second amino acid or nucleotide sequences have similar activities. In the case of antibodies, the second antibody has the same binding specificity and has at least 50% of the affinity of the same.<br>
As used herein, the terms "low stringency," "medium stringency," "high stringency," or "very high stringency conditions" describe conditions for nucleic acid hybridization and washing.  Guidance for performing hybridization reactions can be found in Current Protocols in Molecular Biology, John Wiley &amp; Sons, N.Y. (1989), 6.3.1-6.3.6, which is incorporated herein by reference in its entirety.  Aqueous and nonaqueous methods are described in that reference and either can be used.   Specific hybridization conditions referred to herein are as follows:   (1) low stringency hybridization conditions in 6X sodium chloride/sodium citrate (SSC) at about 45°C, followed by two washes in 0.2X SSC, 0.1% SDS at least at 50°C (the temperature of the washes can be increased to 55°C for low stringency conditions); (2) medium stringency hybridization conditions in 6X SSC at about 45°C, followed by one or more washes in 0.2X SSC, 0.1% SDS at 60°C; (3) high stringency hybridization conditions in 6X SSC at about 45°C, followed by one or more washes in 0.2X SSC, 0.1% SDS at 65°C; and preferably (4) very high stringency hybridization conditions are 0.5M sodium phosphate, 7% SDS at 65°C, followed by one<br>
or more washes at 0.2X SSC, 1% SDS at 65°C. Very high stringency conditions (4) are the preferred conditions and the ones that should be used unless otherwise specified.<br>
TAR1-5-19 Dab-is a single domain antibody (Dab) specific for human TNFalpha.<br>
TAR2h-10-27 Dab- is a single domain antibody (Dab) specific for human TNF receptor 1 (p55 receptor).<br>
TAR1/TAR2 Fab, F(ab')2 or   IgG are Fab, F(ab')2 or IgG formatted dual specific antibodies comprising TAR1-5-19 and TAR2h-10-27 Dabs as herein described.<br>
Detailed Description of the Invention<br>
Unless defined otherwise, all technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill hi the art (e.g., in cell culture, molecular genetics, nucleic acid chemistry, hybridisation techniques and biochemistry). Standard techniques are used for molecular, genetic and biochemical methods (see generally, Sambrook et al, Molecular Cloning: A Laboratory Manual, 2d ed. (1989) Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y, and Ausubel et al, Short Protocols in Molecular Biology (1999) 4th Ed, John Wiley &amp; Sons, Lie. which are incorporated herein by reference) and chemical methods.<br>
Preparation of immunoglobulin based multi-specific ligands<br>
Dual specific ligands according to the invention, whether open or closed in conformation according to the desired configuration of the invention, may be prepared according to previously established techniques, used in the field of antibody engineering, for the preparation of scFv, "phage" antibodies and other engineered antibody molecules. Techniques for the preparation of antibodies, and in particular bispecific antibodies, are for example described in the following reviews and the references cited therein: Winter &amp; Milstein, (1991) Nature 349:293-299; Plueckthun (1992) Immunological Reviews 130:151-188; Wright et al., (1992) Crti. Rev. hnmunol. 12:125-168; Holliger, P. &amp;<br>
Winter, G. (1993) Curr. Op. Biotechn. 4, 446-449; Carter, etal. (1995) J. Hematother. 4, 463-470; Chester, K.A. &amp; Hawkins, R.E. (1995) Trends Biotechn. 13, 294-300; Hoogenboom, H.R. (1997) Nature Biotechnol. 15, 125-126; Fearon, D. (1997) Nature Biotechnol. 15, 618-619; Phlckthun, A. &amp; Pack, P. (1997) Immunotechnology 3, 83-105; Carter, P. &amp; Merchant, A.M. (1997) Curr. Opin. Biotechnol. 8, 449-454; Holliger, P. &amp; Winter, G. (1997) Cancer hnmunol. Immunother. 45,128-130.<br>
The invention provides for the selection of variable domains against two different antigens or epitopes, and subsequent combination of the variable domains.<br>
The techniques employed for selection of the variable domains employ libraries and selection procedures which are known in the art. Natural libraries (Marks et al. (1991) J. Mol. Biol, 222: 581; Vaughan et al. (1996) Nature Biotech., 14: 309) which use rearranged V genes harvested from human B cells are well known to those skilled in the art. Synthetic libraries (Hoogenboom &amp; Winter (1992) J. Mol. Biol., 227: 381; Barbas et al. (1992) Proc. Natl. Acad. Sci. USA, 89: 4457; Nissim et al. (1994) EMBO J., 13: 692; Griffiths et al. (1994) EMBO J., 13: 3245; De Kruif et al. (1995) J. Mol. Biol, 248: 97) are prepared by cloning immunoglobulin V genes, usually using PCR. Errors in the PCR process can lead to a high degree of randomisation. VH and/or VL libraries may be selected against target antigens or epitopes separately, in which case single domain binding is directly selected for, or together.<br>
A preferred method for making a dual specific ligand according to the present invention comprises using a selection system in which a repertoire of variable domains is selected for binding to a first antigen or epitope and a repertoire of variable domains is selected for binding to a second antigen or epitope. The selected variable first and second variable domains are then combined and the dual-specific ligand selected for binding to both first and second antigen or epitope. Closed conformation ligands are selected for binding both first and second antigen or epitope in isolation but not simultaneously.<br>
A.       Library vector systems<br>
A variety of selection systems are known in the art which are suitable for use in the present invention. Examples of such systems are described below.<br>
Bacteriophage lambda expression systems may be screened directly as bacteriophage plaques or as colonies of lysogens, both as previously described (Huse et al. (1989) Science, 246: 1275; Caton and Koprowski (1990) Proc. Nail. Acad. Set U.S.A., 87; Mullinax et al. (1990) Proc. Natl. Acad. Sci. U.S.A., 87: 8095; Persson et al. (1991) Proc. Natl. Acad. Sci. U.S.A., 88: 2432) and are of use in the invention. Whilst such expression systems can be used to screen up to 106 different members of a library, they are not really suited to screening of larger numbers (greater than 106 members). •<br>
Of particular use in the construction of libraries are selection display systems, which enable a nucleic acid to be linked to the polypeptide it expresses. As used herein, a selection display system is a system that permits the selection, by suitable display means, of the individual members of the library by binding the generic and/or target ligands.<br>
Selection protocols for isolating desired members of large libraries are known in the art, as typified by phage display techniques. Such systems, in which diverse peptide sequences are displayed on the surface of filamentous bacteriophage (Scott and Smith (1990) Science, 249: 386), have proven useful for creating libraries of antibody fragments (and the nucleotide sequences that encoding them) for the in vitro selection and amplification of specific antibody fragments that bind a target antigen (McCafferty et al., WO 92/01047). The nucleotide sequences encoding the VH and VL regions are linked to gene fragments which encode leader signals that direct them to the periplasmic space of E. coli and as a result the resultant antibody fragments are displayed on the surface of the bacteriophage, typically as fusions to bacteriophage coat proteins (e.g., pHI or pVID). Alternatively, antibody fragments are displayed externally on lambda phage capsids (phagebodies). An advantage of phage-based display systems is that, because they are biological systems, selected library members can be amplified simply by growing the phage containing the selected library member in bacterial cells. Furthermore, since the nucleotide sequence that encode the polypeptide library member is contained on a phage<br>
or phagemid vector, sequencing, expression and subsequent genetic manipulation is relatively straightforward.<br>
Methods for the construction of bacteriophage antibody display libraries and lambda phage expression libraries are well known in the art (McCafferty et al. (1990) Nature, 348: 552; Kang et al. (1991) Proc. Natl. Acad. Sci. U.S.A., 88: 4363; Clackson et al. (1991) Nature, 352: 624; Lowman et al (1991) Biochemistry, 30: 10832; Burton et al.<br>
(1991)	Proc. Natl. Acad. Sci U.S.A., 88: 10134; Hoogenboom et al. (1991) Nucleic Acids<br>
Res., 19: 4133; Chang et al. (1991) /. Immunol, 147: 3610; Breitling et al. (1991) Gene,<br>
104: 147; Marks et al. (1991) supra; Barbas et al. (1992) supra; Hawkins and Winter<br>
(1992)	J. Immunol., 22: 867; Marks et al, 1992, J. Biol. Chem., 267: 16007; Lemer et al.<br>
(1992) Science, 258: 1313, incorporated herein by reference).<br>
One particularly advantageous approach has been the use of scFv phage-libraries (Huston et.al., 1988, Proc. Natl. Acad. Sci U.S.A., 85: 5879-5883; Chaudhary et al (1990) Proc. Natl. Acad. Sci U.S.A., 87: 1066-1070; McCafferty et al. (1990) supra; Clackson et al<br>
(1991)	Nature, 352: 624; Marks et al. (1991) J. Mol. Biol, 222: 581; Chiswell et al<br>
(1992)	Trends Biotech., 10: 80; Marks et al (1992) J. Biol. Chem., 267). Various<br>
embodiments of scFv libraries displayed on bacteriophage coat proteins have been<br>
described. Refinements of phage display approaches are also known, for example as<br>
described in WO96/06213 and W092/01047 (Medical Research Council et al} and<br>
WO97/08320 (Morphosys), which are incorporated herein by reference.<br>
Other systems for generating libraries of polypeptides involve the use of cell-free enzymatic machinery for the in vitro synthesis of the library members. Li one method, RNA molecules are selected by alternate rounds of selection against a target ligand and PCR amplification (Tuerk and Gold (1990) Science, 249: 505; Ellington and Szostak (1990) Nature, 346: 818). A similar technique may be used to identify DNA sequences which bind a predetermined human transcription factor (Thiesen and Bach (1990) Nucleic Acids Res., 18: 3203; Beaudry and Joyce (1992) Science, 257: 635; WO92/05258 and WO92/14843). In a similar way, in vitro translation can be used to synthesise polypeptides as a method for generating large libraries. These methods which generally comprise stabilised polysome complexes, are described further in WO88/08453,<br>
W090/05785, WO90/07003, WO91/02076, WO91/05058, and WO92/02536. Alternative display systems which are not phage-based, such as those disclosed in W095/22625 and WO95/11922 (Affymax) use the polysomes to display polypeptides for selection.<br>
A still further category of techniques involves the selection of repertoires in artificial compartments, which allow the linkage of a gene with its gene product. For example, a selection system in which nucleic acids encoding desirable gene products may be selected in microcapsules formed by water-in-oil emulsions is described in WO99/02671, WOOO/40712 and Tawfik &amp; Griffiths (1998) Nature Biotechnol 16(7), 652-6. Genetic elements encoding a gene product having a desired activity are compartmentalised into microcapsules and then transcribed and/or translated to produce their respective gene products (RNA or protein) within the microcapsules. Genetic elements which produce gene product having desired activity are subsequently sorted. This approach selects gene products of interest by detecting the desired activity by a variety of means.<br>
B.       Library Construction.<br>
Libraries intended for selection, may be constructed using techniques known in the art, for example as set forth above, or may be purchased from commercial sources. Libraries which are useful in the present invention are described, for example, in W099/20749. Once a vector system is chosen and one or more nucleic acid sequences encoding polypeptides of interest are cloned into the library vector, one may generate diversity within   the   cloned   molecules   by   undertaking   mutagenesis   prior  to   expression; alternatively, the encoded proteins may be expressed and selected, as described above, before mutagenesis and additional rounds of selection are performed.   Mutagenesis of nucleic acid sequences encoding structurally optimised polypeptides is carried out by standard molecular methods. Of particular use is the polymerase chain reaction, or PCR, (Mullis and Faloona (1987) Methods EnzymoL,  155: 335, herein incorporated by reference). PCR, which uses multiple cycles of DNA replication catalysed by a thermostable, DNA-dependent DNA polymerase to amplify the target sequence of interest, is well known in the art. The construction of various antibody libraries has been<br>
discussed in Winter et al. (1994) Ann. Rev. Immunology 12, 433-55, and references<br>
cited therein.	:<br>
PCR is performed using template DNA (at least Ifg; more usefully, 1-1000 ng) and at least 25 pmol of oligonucleotide primers; it may be advantageous to use a larger amount of primer when the primer pool is heavily heterogeneous, as each sequence is represented by only a small fraction of the molecules of the pool, and amounts become limiting in the later amplification cycles. A typical reaction mixture includes: 2ul of DNA, 25 pmol of oligonucleotide primer, 2.5 ul of 10X PCR buffer 1 (Perkin-Ehner, Foster City, CA), 0.4 u.1 of 1.25 uM dNTP, 0.15 ul (or 2.5 units) of Taq DNA polymerase (Perkin Elmer, Foster City, CA) and deionized water to a total volume of 25 ul. Mineral oil is overlaid and the PCR is performed using a programmable thermal cycler.    The length and temperature of each step of a PCR cycle, as well as the number of cycles, is adjusted in accordance to the stringency requirements in effect. Annealing temperature and timing are determined both by the efficiency with which a primer is expected to anneal to a template and the degree of mismatch that is to be tolerated; obviously, when nucleic acid molecules are simultaneously amplified and mutagenised, mismatch is required, at least in the first round of synthesis. The ability to optimise the stringency of primer annealing conditions is well within the knowledge of one of moderate skill in the art. An annealing temperature of between 30 °C and 72 °C is used. Initial denaturation of the template molecules normally occurs at between 92°C and 99°C for 4 minutes, followed by 20-40 cycles consisting of denaturation (94-99°C for 15 seconds to 1 minute), annealing (temperature determined as discussed above; 1-2 minutes), and extension (72°C for 1-5 minutes, depending on the length of the amplified product). Final extension is generally for 4 minutes at 72°C, and may be followed by an indefinite (0-24 hour) step at 4°C.<br>
C.       Combining single variable domains<br>
Domains useful in the invention, once selected, may be combined by a variety of methods known in the art, including covalent and non-covalent methods.<br>
Preferred methods include the use of polypeptide linkers, as described, for example, in connection with scFv molecules (Bird et al, (1988) Science 242:423-426). Discussion of<br>
suitable linkers is provided in Bird et al. Science 242, 423-426; Hudson et al, Journal Immunol Methods 231 (1999) 177-189; Hudson et al, Proc Nat Acad Sci USA 85, 5879-5883. Linkers are preferably flexible, allowing the two single domains to interact. One linker example is a (Gly4 Ser)n linker, where n=l to 8, eg, 2, 3, 4, 5 or 7. The linkers used in diabodies, which are less flexible, may also be employed (Holliger et al., (1993) PNAS (USA) 90:6444-6448).<br>
In one embodiment, the linker employed is not an immunoglobulin hinge region.<br>
Variable domains may be combined using methods other than linkers. For example, the use of disulphide bridges, provided through naturally-occurring or engineered cysteine residues, may be exploited to stabilise VH~VH&gt;VL~VL or VH-VL dimers (Reiter et al., (1994) Protein Eng. 7:697-704) or by remodelling the interface between the variable domains to improve the "fit" and thus the stability of interaction (Ridgeway et al, (1996) Protein Eng. 7:617-621; Zhu et al, (1997) Protein Science 6:781-788).<br>
Other techniques for joining or stabilising variable domains of immunoglobulins, and in particular antibody VH domains, may be employed as appropriate.<br>
In accordance with the present invention, dual specific ligands can be in "closed" conformations in solution. A "closed" configuration is that in which the two domains (for example VH and VL) are present in associated form, such as that of an associated VH-VL pair which forms an antibody binding site. For example, scFv may be in a closed conformation, depending on the arrangement of the linker used to link the VH and VL domains. If this is sufficiently flexible to allow the domains to associate, or rigidly holds them in the associated position, it is likely that the domains will adopt a closed conformation.<br>
Similarly, VH domain pairs and VL domain pairs may exist in a closed conformation. Generally, this will be a function of close association of the domains, such as by a rigid linker, in the ligand molecule. Ligands in a closed conformation will be unable to bind both the molecule which increases the half-life of the ligand and a second target molecule.<br>
Thus, the ligand will typically only bind the second target molecule on dissociation from the molecule which increases the half-life of the ligand.<br>
Moreover, the construction of VH/VH, VL/VL or VH/VL dimers without linkers provides for competition between the domains.<br>
Ligands according to the invention may moreover be in an open conformation. In such a conformation, the ligands will be able to simultaneously bind both the molecule which increases the half-life of the ligand and the second target molecule. Typically, variable domains in an open configuration are (in the case of VH-VL pairs) held far enough apart for the domains not to interact and form an antibody binding site and not to compete for binding to their respective epitopes. In the case of VH/VH or VJ/VL dimers, the domains are not forced together by rigid linkers. Naturally, such domain pairings will not compete for antigen binding or form an antibody binding site.<br>
Fab fragments and whole antibodies will exist primarily in the closed conformation, although it will be appreciated that open and closed dual specific ligands are likely to exist in a variety of equilibria under different circumstances. Binding of the ligand to a target is likely to shift the balance of the equilibrium towards the open configuration. Thus, certain ligands according to the invention can exist in two conformations in solution, one of which (the open form) can bind two antigens or epitopes independently, whilst the alternative conformation (the closed form) can only bind one antigen or epitope; antigens or epitopes thus compete for binding to the ligand in this conformation.<br>
Although the open form of the dual specific ligand may thus exist in equilibrium with the closed form in solution, it is envisaged that the equilibrium will favour the closed form; moreover, the open form can be sequestered by target binding into a closed conformation. Preferably, therefore, certain dual specific ligands of the invention are present in an equilibrium between two (open and closed) conformations.<br>
Dual specific ligands according to the invention may be modified in order to favour an open or closed conformation.   For example, stabilisation of VH-VL interactions with . disulphide bonds stabilises the closed conformation.   Moreover, linkers used to join the<br>
domains, including VH domain and VL domain pairs, may be constructed such that the open from is favoured; for example, the linkers may sterically hinder the association of the domains, such as by incorporation of large amino acid residues in opportune locations, or the designing of a suitable rigid structure which will keep the domains physically spaced apart.<br>
D.       Characterisation of the dual-specific ligand.	i<br>
The binding of the dual-specific ligand to its specific antigens or epitopes can be tested by methods which will be familiar to those skilled in the art and include ELISA. In a preferred embodiment of the invention binding is tested using monoclonal phage ELISA.<br>
Phage ELISA may be performed according to any suitable procedure: an exemplary protocol is set forth below.<br>
Populations of phage produced at each round of selection can be screened for binding by ELISA to the selected antigen or epitope, to identify "polyclonal" phage antibodies. Phage from single infected bacterial colonies from these populations can then be screened by ELISA to identify "monoclonal" phage antibodies. It is also desirable to screen soluble antibody fragments for binding to antigen or epitope, and this can also be undertaken by ELISA using reagents, for example, against a C- or N-terminal tag (see for example Winter et al. (1994) Ann. Rev. Immunology 12,433-55 and references cited therein.<br>
The diversity of the selected phage monoclonal antibodies may also be assessed by gel electrophoresis of PCR products (Marks et al. 1991, supra; Nissim et al. 1994 supra), probing (Tomlinson et al, 1992) J. Mol. Biol. 227, 776) or by sequencing of the vector DNA.<br>
E.        Structure of 'Dual-specific ligands'.<br>
As described above, an antibody is herein defined as an antibody (for example IgG, IgM, IgA, IgA, IgE) or fragment (Fab, Fv, disulphide linked Fv, scFv, diabody) which comprises at least one heavy and a light chain variable domain, at least two heavy chain<br>
variable domains or at least two light chain variable domains. It may be at least partly derived from any species naturally producing an antibody, or created by recombinant DNA technology; whether isolated from serum, B-cells, hybridomas, transfectomas, yeast or bacteria).<br>
In a preferred embodiment of the invention the dual-specific ligand comprises at least one single heavy chain variable domain of an antibody and one single light chain variable domain of an.antibody, or two single heavy or light chain variable domains. For example, the ligand may comprise a VH/VL pair, a pair of VH domains or a pair of VL domains.<br>
The first and the second variable domains of such a ligand may be on the same polypeptide chain. Alternatively they may be on separate polypeptide chains. In the case that they are on the same polypeptide chain they may be linked by a linker, which is preferentially a peptide sequence, as described above.<br>
The first and second variable domains may be covalently or non-covalently associated. In the case that they are covalently associated, the covalent bonds may be disulphide bonds.<br>
hi the case that the variable domains are selected from V-gene repertoires selected for instance using phage display technology as herein described, then these variable domains comprise a universal framework region, such that is they may be recognised by a specific generic ligand as herein defined. The use of universal frameworks, generic ligands and the like is described in WO99/20749.<br>
Where V-gene repertoires are used variation in polypeptide sequence is preferably located within the structural loops of the variable domains. The polypeptide sequences of either variable domain may be altered by DNA shuffling or by mutation in order to enhance the interaction of each variable domain with its complementary pair. DNA shuffling is known in the art and taught, for example, by Stemmer, 1994, Nature 370: 389-391 and U.S. Patent No. 6,297,053, both of which are incorporated herein by reference. Other methods of mutagenesis are well known to those of skill in the art.<br>
In a preferred embodiment of the invention the 'dual-specific ligand' is a single chain Fv fragment. In an alternative embodiment of the invention, the 'dual-specific ligand' consists of a Fab format.<br>
In a further aspect, the present invention provides nucleic acid encoding at least a 'dual-specific ligand' as herein defined.<br>
One skilled in the art will appreciate that, depending on the aspect of the invention, both antigens or epitopes may bind simultaneously to the same antibody molecule. Alternatively, they may compete for binding to the same antibody molecule. For example, where both epitopes are bound simultaneously, both variable domains of a dual specific ligand are able to independently bind their target epitopes. Where the domains compete, the one variable domain is capable of binding its target, but not at the same time as the other variable domain binds its cognate target; or the first variable domain is capable of binding its target, but not at the same tune as the second variable domain binds its cognate target.<br>
The variable regions may be derived from antibodies directed against target antigens or epitopes. Alternatively they may be derived from a repertoire of single antibody domains" such as those expressed on the surface of filamentous bacteriophage.  Selection may be performed as described below.<br>
In general, the nucleic acid molecules and vector constructs required for the performance of the present invention may be constructed and manipulated as set forth in standard laboratory manuals, such as Sambrook et al. (1989) Molecular Cloning: A Laboratory Manual, Cold Spring Harbor, USA.<br>
The manipulation of nucleic acids useful in the present invention is typically carried out in recombinant vectors.<br>
Thus in a further aspect, the present invention provides a vector comprising nucleic acid encoding at least a 'dual-specific ligand' as herein defined.<br>
As used herein, vector refers to a discrete element that is used to introduce heterologous DNA into cells for the expression and/or replication thereof. Methods by which to select or construct and, subsequently, use such vectors are well known to one of ordinary skill in the art. Numerous vectors are publicly available, including bacterial plasmids, bacteriophage, artificial chromosomes and episomal vectors. Such vectors may be used for simple cloning and mutagenesis; alternatively gene expression vector is employed. A vector of use according to the invention may be selected to accommodate a polypeptide coding sequence of a desired size, typically from 0.25 kilobase (kb) to 40 kb or more in length -A suitable host cell is transformed with the vector after in vitro cloning manipulations. Each vector contains various functional components, which generally include a cloning (or "polylinker") site, an origin of replication and at least one selectable marker gene. If given vector is an expression vector, it additionally possesses one or more of the following: enhancer element, promoter, transcription termination and signal sequences, each positioned in the vicinity of the cloning site, such that they are operatively linked to the gene encoding a ligand according to the invention.<br>
Both cloning and expression vectors generally contain nucleic acid sequences that enable the vector to replicate in one or more selected host cells. Typically in cloning vectors, this sequence is one that enables the vector to replicate independently of the host chromosomal DNA and includes origins of replication or autonomously replicating sequences. Such sequences are well known for a variety of bacteria, yeast and viruses. The origin of replication from the plasmid pBR322 is suitable for most Gram-negative bacteria, the 2 micron plasmid origin is suitable for yeast, and various viral origins (e.g. SV 40, adenovirus) are useful for cloning vectors in mammalian cells. Generally, the origin of replication is not needed for mammalian expression vectors unless these are used in mammalian cells able to replicate high levels of DNA, such as COS cells.<br>
Advantageously, a cloning or expression vector may contain a selection gene also referred to as selectable marker. This gene encodes a protein necessary for the survival or growth of transformed host cells grown in a selective culture medium. Host cells not transformed with the vector containing the selection gene will therefore not survive in the culture medium. Typical selection genes encode proteins that confer resistance to antibiotics and other toxins, e.g. ampicillin, neomycin, methotrexate or tetracycline,<br>
complement auxotrophic deficiencies, or supply critical nutrients not available in the growth media.<br>
Since the replication of vectors encoding a ligand according to the present invention is most conveniently performed in E. coli, an E. co/z-selectable marker, for example, the (3-lactamase gene that confers resistance to the antibiotic ampicillin, is of use. These can be obtained from E. coli plasmids, such as pBR322 or a pUC plasmid such as pUCIS or pUC19.<br>
Expression vectors usually contain a promoter that is recognised by the host organism and is operably linked to the coding sequence of interest. Such a promoter may be inducible or constitutive. The term "operably linked" refers to a juxtaposition wherein the components described are in a relationship permitting them to function in their intended manner. A control sequence "operably linked" to a coding sequence is ligated in such a way that expression of the coding sequence is achieved under conditions compatible with the control sequences.<br>
Promoters suitable for use with prokaryotic hosts include, for example, the p-lactamase and lactose promoter systems, alkaline phosphatase, the tryptophan (tip) promoter system and hybrid promoters such as the tac promoter. Promoters for use in bacterial systems will also generally contain a Shine-Delgarno sequence operably linked to the coding sequence.<br>
The preferred vectors are expression vectors that enables the expression of a nucleotide sequence corresponding to a poiypeptide library member. Thus, selection with the first and/or second antigen or epitope can be performed by separate propagation and expression of a single clone expressing the poiypeptide library member or by use of any selection display system. As described above, the preferred selection display system is bacteriophage display. Thus, phage or phagemid vectors may be used, eg pITl or pIT2. Leader sequences useful in the invention include pelB, sifl, ompA, phoA, bla and pelA. One example are phagemid vectors which have an E. coli. origin of replication (for double stranded replication) and also a phage origin of replication (for production of single-stranded DNA). The manipulation and expression of such vectors is well known in<br><br>
the art (Hoogenboom and Winter (1992) supra; Nissim et al. (1994) supra). Briefly, the vector contains a p-lactamase gene to confer selectivity on the phagemid and a lac promoter upstream of a expression cassette that consists (N to C terminal) of a pelB leader sequence (which directs the expressed polypeptide to the periplasmic space), a multiple cloning site (for cloning the nucleotide version of the library member), optionally, one or more peptide tag (for detection), optionally, one or more TAG stop codon and the phage protein pin. Thus, using various suppressor and non-suppressor strains of E. coli and with the addition of glucose, iso-propyl thio-p-D-galactoside (IPTG) or a helper phage, such as VCS M13, the vector is able to replicate as a plasmid with no expression, produce large quantities of the polypeptide library member only or produce phage, some of which contain at least one copy of the polypeptide-pin fusion on their surface.<br>
Construction of vectors encoding ligands according to the invention employs conventional ligation techniques. Isolated vectors or DNA fragments are cleaved, tailored, and religated in the form desired to generate the required vector. If desired, analysis to confirm that the correct sequences are present in the constructed vector can be performed in a known fashion. Suitable methods for constructing expression vectors, preparing in vitro transcripts, introducing DNA into host cells, and performing analyses for assessing expression and function are known to those skilled in the art. The presence of a gene sequence in a sample is detected, or its amplification and/or expression quantified by conventional methods, such as Southern or Northern analysis, Western blotting, dot blotting of DNA, RNA or protein, in situ hybridisation, immunocytochemistry or sequence analysis of nucleic acid or protein molecules. Those skilled in the art will readily envisage how these methods may be modified, if desired.<br>
Structure of closed conformation multispecific ligands<br>
According to one aspect of the second configuration of the invention present invention, the two or more non-complementary epitope binding domains are linked so that they are in a closed conformation as herein defined. Advantageously, they may be further attached to a skeleton which may, as a alternative, or on addition to a linker described -<br>
herein, facilitate the formation and/or maintenance of the closed conformation of the epitope binding sites with respect to one another.<br>
(I) Skeletons<br>
Skeletons may be based on immunoglobulin molecules or may be non-immunoglobulin in origin as set forth above. Preferred immunoglobulin skeletons as herein defined includes any one or more of those selected from the following: an immunoglobulin molecule comprising at least (i) the CL (kappa or lambda subclass) domain of an antibody; or (ii) the CHI domain of an antibody heavy chain; an immunoglobulin molecule comprising the CHI and CH2 domains of an antibody heavy chain; an immunoglobulin molecule comprising the CHI, CH2 and CH3 domains of an antibody heavy chain; or any of the subset (ii) in conjunction with the CL (kappa or lambda subclass) domain of an antibody. A hinge region domain may also be included.. Such combinations of domains may, for example, mimic natural antibodies, such as IgG or IgM, or fragments thereof, such as Fv, scFv, Fab or F(ab')2 molecules. Those skilled in the art will be aware that this list is not intended to be exhaustive.<br>
(II) Protein scaffolds<br>
Each epitope binding domain comprises a protein scaffold and one or more CDRs which are involved in the specific interaction of the domain with one or more epitopes. Advantageously, an epitope binding domain according to the present invention comprises three CDRs. Suitable protein scaffolds include any of those selected from the group consisting of the following: those based on immunoglobulin domains, those based on fibronectin, those based on affibodies, those based on CTLA4, those based on chaperones such as GroEL, those based on lipocallin and those based on the bacterial Fc receptors SpA and SpD. Those skilled in the art will appreciate that this list is not intended to be exhaustive.<br>
F:        Scaffolds for use in Constructing Dual Specific Ligands<br>
i. Selection of the main-chain conformation<br>
The members of the immunoglobulin superfamily all share a similar fold forJheir polypeptide chain. For example, although antibodies are highly diverse in terms of their<br>
primary sequence, comparison of sequences and crystallographic structures has revealed that, contrary to expectation, five of the six antigen binding loops of antibodies (HI, H2, LI, L2, L3) adopt a limited number of main-chain conformations, or canonical structures (Chothia and Lesk (1987) J. Mol. Biol, 196: 901; Chothia et al (1989) Nature, 342: 877). Analysis of loop lengths and key residues has therefore enabled prediction of the main-chain conformations of HI, H2, LI, L2 and L3 found in the majority of human antibodies (Chothia et al. (1992) J. Mol. Biol, 227: 799; Tomlinson et al. (1995) EMBO J., 14: 4628; Williams et al. (1996) J. Mol. Biol, 264: 220). Although the H3 region is much more diverse in terms of sequence, length and structure (due to the use of D segments), it also forms a limited number of main-chain conformations for short loop lengths which depend on the length and the presence of particular residues, or types of residue, at key positions in the loop and the antibody framework (Martin et al. (1996) J. Mol. Biol, 263: 800; Shirai etal (1996) FEBSLetters, 399: 1).<br>
The dual specific ligands of the present invention are advantageously assembled from libraries of domains, such as libraries of VH domains and/or libraries of VL domains. Moreover, the dual specific ligands of the invention may themselves be provided hi the form of libraries. In one aspect of the present invention, libraries of dual specific ligands and/or domains are designed in which certain loop lengths and key residues have been chosen to ensure that the main-chain conformation of the members is known. Advantageously, these are real conformations of immunoglobulin superfamily molecules found in nature, to minimise the chances that they are non-functional, as discussed above. Germline V gene segments serve as one suitable basic framework for constructing antibody or T-cell receptor libraries; other sequences are also of use. Variations may occur at a low frequency, such that a small number of functional members may possess an altered main-chain conformation, which does not affect its function.<br>
Canonical structure theory is also of use to assess the number of different main-chain conformations encoded by ligands, to predict the main-chain conformation based on ligand sequences and to chose residues for diversification which do not affect the canonical structure. It is known that, in the human VK domain, the LI loop can adopt one of four canonical-structures, the L2 loop has a single canonical structure and that 90% of human VK domains adopt one of four or five canonical structures for the L3 loop<br>
(Tomlinson et al. (1995) supra); thus, in the VK domain alone, different canonical structures can combine to create a range of different main-chain conformations. Given that the V\ domain encodes a different range of canonical structures for the LI, L2 and L3 loops and that VK and V\ domains can pair with any VH domain which can encode several<br>
canonical structures for the HI and H2 loops, the number of canonical structure<br>
*<br>
combinations observed for these five loops is very large. This imph'es that the generation of diversity in the main-chain conformation may be essential for the production of a wide range of binding specificities. However, by constructing an antibody library based on a single known main-chain conformation it has been found, contrary to expectation, that diversity in the main-chain conformation is not required to generate sufficient diversity to target substantially all antigens. Even more surprisingly, the single main-chain conformation need not be a consensus structure - a single naturally occurring conformation can be used as the basis for an entire library. Thus, in a preferred aspect, the dual-specific ligands of the invention possess a single known main-chain conformation.<br>
The single main-chain conformation that is chosen is preferably commonplace among molecules of the immunoglobulin superfamily type in question. A conformation is commonplace when a significant number of naturally occurring molecules are observed to adopt it. Accordingly, in a preferred aspect of the invention, the natural occurrence of the different main-chain conformations for each binding loop of an immunoglobulin domain are considered separately and then a naturally occurring variable domain is chosen which possesses the desired combination of main-chain conformations for the different loops. If none is available, the nearest equivalent may be chosen. It is preferable that the desired combination of main-chain conformations for the different loops is created by selecting germline gene segments which encode the desired main-chain conformations. It is more preferable, that the selected germline gene segments are frequently expressed in nature, and most preferable that they are the most frequently expressed of all natural germline gene segments.<br>
In designing dual specific ligands or libraries thereof the incidence of the different main-chain conformations for each of the six antigen binding loops may be considered separately. For HI, H2, LI, L2 and L3, a given conformation that is adopted by between 20% and 100% of the antigen binding loops of naturally occurring molecules is chosen.<br>
•Typically, its observed incidence is above 35% (i.e. between 35% and 100%) and, ideally, above 50% or even above 65%. Since the vast majority of H3 loops do not have canonical structures, it is preferable to select a main-chain conformation which is commonplace among those loops which do display canonical structures. For each of the loops, the conformation which is observed most often in the natural repertoire is therefore selected. In human antibodies, the most popular canonical structures (CS) for each loop are as follows: HI - CS 1 (79% of the expressed repertoire), H2 - CS 3 (46%), LI - CS 2 of VK (39%), L2 - CS 1 (100%), L3 - CS 1 of VK (36%) (calculation assumes a K:A, ratio of 70:30, Hood et al. (1967) Cold Spring Harbor Symp. Quant. Biol, 48: 133). For H3 loops that have canonical structures, a CDR3 length (Kabat et al. (1991) Sequences of proteins of immunological interest, U.S. Department of Health and Human Services) of seven residues with a salt-bridge from residue 94 to residue 101 appears to be the most common. There are at least 16 human antibody sequences in the EMBL data library with the required H3 length and key residues to form this conformation and at least two crystallographic structures in the protein data bank which can be used as a basis for antibody modelling (2cgr and Itet). The most frequently expressed germline gene segments that this combination of canonical structures are the VH segment 3-23 (DP-47), the JH segment JH4b, the VK segment O2/O12 (DPK9) and the JK segment JK1. VH segments DP45 and DP38 are also suitable.   These segments can therefore be used in combination as a basis to construct a library with the desired single main-chain conformation.<br>
Alternatively, instead of choosing the single main-chain conformation based on the natural occurrence of the different main-chain conformations for each of the binding loops in isolation, the natural occurrence of combinations of main-chain conformations is used as the basis for choosing the single main-chain conformation. In the case of antibodies, for example, the natural occurrence of canonical structure combinations for any two, three, four, five or for all six of the antigen binding loops can be determined. Here, it is preferable that the chosen conformation is commonplace in naturally occurring antibodies and most preferable that it observed most frequently in the natural repertoire. Thus, in human antibodies, for example, when natural combinations of the five antigen binding loops, HI, H2, LI, L2 and L3, are considered, the most frequent combination of<br>
canonical   structures  is  determined  and  then  combined  with  the  most popular conformation for the H3 loop, as a basis for choosing the single main-chain conformation.<br>
ii. Diversification of the canonical sequence<br>
Having selected several known main-chain conformations .or, preferably a single known main-chain conformation, dual specific ligands according to the invention or libraries for use in the invention can be constructed by varying the binding site of the molecule in order to generate a repertoire with structural and/or functional diversity. This means that variants are generated such that they possess sufficient diversity hi their structure and/or in their function so that they are capable of providing a range of activities.<br>
The desired diversity is typically generated by varying the selected molecule at one or more positions. The positions to be changed can be chosen at random or are preferably selected. The variation can then be achieved either by randomisation, during which the resident amino acid is replaced by any amino acid or analogue thereof, natural or synthetic, producing a very large number of variants or by replacing the resident amino acid with one or more of a defined subset of amino acids, producing a more limited number of variants.<br>
Various methods have been reported for introducing such diversity. Error-prone PCR (Hawkins et al. (1992) J. Mol. Bid., 226: 889), chemical mutagenesis (Deng et al. (1994) J. Biol Chem., 269: 9533) or bacterial mutator strains (Low et al. (1996) J. Mol. Biol, 260: 359) can be used to introduce random mutations into the genes that encode the molecule. Methods for mutating selected positions are also well known in the art and include the use of mismatched oligonucleotides or degenerate oligonucleotides, with or without the use of PCR. For example, several synthetic antibody libraries have been created by targeting mutations to the antigen binding loops. The H3 region of a human tetanus toxoid-binding Fab has been randomised to create a range of new binding specificities (Barbas et al. (1992) Proc. Natl. Acad. Set. USA, 89: 4457). Random or semi-random H3 and L3 regions have been appended to gennline V gene segments to produce large libraries with unmutated framework regions (Hoogenboom-&amp; Winter (1992) J. Mol. Biol, 221: 381; Barbas et al (1992) Proc. Natl Acad. Sci. USA, 89: 4457;<br>
Nissim el al. (1994) EMBO J., 13: 692; Griffiths et al. (1994) EMBO J., 13: 3245; De Kruif et al. (1995) J. Mol Biol, 248: 97). Such diversification has been extended to include some or all of the other antigen binding loops (Crameri et al. (1996) Nature Med., 2: 100; Riechmann et al. (1995) Bio/Technology, 13: 475; Morphosys, WO97/08320, supra).<br>
Since loop randomisation has the potential to create approximately more than 1015 structures for H3 alone and a similarly large number of variants for the other five loops, it is not feasible using current transformation technology or even by using cell free systems to produce a library representing all possible combinations. For example, in one of the largest libraries constructed to date, 6 x 1010 different antibodies, which is only a fraction of the potential diversity for a library of this design, were generated (Griffiths et al. (1994) supra).<br>
In a preferred embodiment, only those residues which are directly involved in creating or modifying the desired function of the molecule are diversified. For many molecules, the function will be to bind a target and therefore diversity should be concentrated in the target binding site, while avoiding changing residues which are crucial to the overall packing of the molecule or to maintaining the chosen main-chain conformation.<br>
Diversification of the canonical sequence as it applies to antibody domains<br>
In the case of antibody dual-specific ligands, the binding site for the target is most often the antigen binding site. Thus, in a highly preferred aspect, the invention provides libraries of or for the assembly of antibody dual-specific ligands in which only those residues in the antigen binding site are varied. These residues are extremely diverse in the human  antibody  repertoire  and  are  known  to  make  contacts  in high-resolution antibody/antigen complexes. For example, in L2 it is known that positions 50 and 53 are diverse in naturally occurring antibodies and are observed to make contact with the antigen, ha contrast, the conventional approach would have been to diversify all the residues in the corresponding Complementarity Determining Region (CDR1) as defined by Kabat et al. (1991, supra), some seven residues compared to the two diversified in the library for use according to the invention. This represents a significant improvement in terms of the functional diversity required to create a range of antigen binding specificities.<br>
In nature, antibody diversity is the result of two processes: somatic recombination of germline V, D and J gene segments to create a naive primary repertoire (so called germline and junctional diversity) and somatic hypermutation of the resulting rearranged V genes. Analysis of human antibody sequences has shown that diversity in the primary repertoire is focused at the centre of the antigen binding site whereas somatic hypermutation spreads diversity to regions at the periphery of the antigen binding site that are highly conserved in the primary repertoire (see Tomlinson et al. (1996) J. Mol. Biol, 256: 813). This complementarity has probably evolved as an efficient strategy for searching sequence space and, although apparently unique to antibodies, it can easily be applied to other polypeptide repertoires. The residues which are varied are a subset of those that form the binding site for the target. Different (including overlapping) subsets of residues in the target binding site are diversified at different stages during selection, if desired.<br>
hi the case of an antibody repertoire, an initial 'naive' repertoire is created where some, but not all, of the residues in the antigen binding site are diversified. As used herein in this context, the term "naive" refers to antibody molecules that have no pre-determined target. These molecules resemble those which are encoded by the immunoglobulin genes of an individual who has not undergone immune diversification, as is the case with fetal and newborn individuals, whose immune systems have not yet been challenged by a wide variety of antigenic stimuli. This repertoire is then selected against a range of antigens or epitopes. If required, further diversity can then be introduced outside the region diversified in the initial repertoire. This matured repertoire can be selected for modified function, specificity or affinity.<br>
The invention provides two different naive repertoires of binding domains for the construction of dual specific ligands, or a naive library of dual specific ligands, in which some or ah1 of the residues in the antigen binding site are varied. The "primary" library mimics the natural primary repertoire, w^th diversity restricted to residues at the centre of the antigen binding site that are diverse in the germline V gene segments (germline diversity) or diversified during the recombination process (junctional diversity). Those residues which are diversified include, but are not limited to, H50, H52, H52a, H53, H55,<br>
H56, H58, H95, H96, H97, H98, L50, L53, L91, L92, L93, L94 and L96. In the "somatic" library, diversity is restricted to residues that are diversified during the recombination process (junctional diversity) or are highly somatically mutated). Those residues which are diversified include, but are not limited to: H31, H33, H35, H95, H96, H97, H98, L30, L31, L32, L34 and L96. All the residues listed above as suitable for diversification in these libraries are known to make contacts in one or more antibody-antigen complexes. Since in both libraries, not all of the residues in the antigen binding site are varied, additional diversity is incorporated during selection by varying the remaining residues, if it is desired to do so. It shall be apparent to one skilled in the art that any subset of any of these residues (or additional residues which comprise the antigen binding site) can be used for the initial and/or subsequent diversification of the antigen binding site.<br>
In the construction of libraries for use in the invention, diversification of chosen positions<br>
is typically achieved at the nucleic acid level, by altering the coding sequence which<br>
specifies the sequence of the polypeptide such that a number of possible arnino acids (all<br>
20 or a subset thereof) can be incorporated at that position. Using the IUPAC<br>
nomenclature, the most versatile codon is NNK, which encodes all amino acids as well as<br>
the TAG stop codon. The NNK codon is preferably used in order to introduce the<br>
required diversity. Other codons which achieve the same ends are also of use, including<br>
the NNN codon, which leads to the production of the additional stop codons TGA and<br>
TAA.	'.<br>
A feature of side-chain diversity in the antigen binding site of human antibodies is a pronounced bias which favours certain amino acid residues. If the amino acid composition of the ten most diverse positions in each of the VH, VK and Vx regions are summed, more than 76% of the side-chain diversity comes from only seven different residues, these being, serine (24%), tyrosine (14%), asparagine (11%), glycine (9%), alanine (7%), aspartate (6%) and threonine (6%). This bias towards hydrophilic residues and small residues which can provide main-chain flexibility probably reflects the evolution of surfaces which are predisposed to binding a wide range of antigens or epitopes and may help to explain the required promiscuity of antibodies in the primary repertoire.<br>
Since it is preferable to mimic this distribution of amino acids, the distribution of amino acids at the positions to be varied preferably mimics that seen in the antigen binding site of antibodies. Such bias in the substitution of amino acids that permits selection of certain polypeptides (not just antibody polypeptides) against a range of target antigens is easily applied to any polypeptide repertoire. There are various methods for biasing the amino acid distribution at the position to be varied (including the use of tri-nucleotide mutagenesis, see WO97/08320), of which the preferred method, due to ease of synthesis, is the use of conventional degenerate codons. By comparing the amino acid profile encoded by all combinations of degenerate codons (with single, double, triple and quadruple degeneracy in equal ratios at each position) with the natural amino acid use it is possible to  calculate the most representative  codon.  The  codons  (AGT)(AGC)T, (AGT)(AGC)C and (AGT)(AGC)(CT) - that is, DVT, DVC and DVY, respectively using IUPAC nomenclature - are those closest to the desired amino acid profile: they encode 22% serine and 11% tyrosine, asparagine, glycine, alanine, aspartate, threonine and cysteine. Preferably, therefore, libraries are constructed using either the DVT, DVC or DVY codon at each of the diversified positions.<br>
G:       Antigens capable of increasing ligand half-life<br>
The dual specific ligands according to the invention, in one configuration thereof, are capable of binding to one or more molecules which can increase the half-life of the ligand in vivo. Typically, such molecules are polypeptides which occur naturally in vivo and which resist degradation or removal by endogenous mechanisms which remove unwanted material from the organism. For example, the molecule which increases the half-life of the organism may be selected from the following:<br><br>
Proteins from the extracellular matrix; for example collagen, laminins, integrins and fibronectin. Collagens are the major proteins of the extracellular matrix. About 15 types of collagen molecules are currently known, found in different parts of the body, eg type I collagen (accounting for 90% of body collagen) found in bone, skin, tendon, ligaments, cornea, internal organs or type II collagen found in cartilage, invertebral disc, notochord, vitreous humour of the eye.<br>
Proteins found in blood, including:<br>
Plasma proteins such as fibrin, a-2 macroglobulin, serum albumin, fibrinogen A, fibrinogen B, serum amyloid protein A, heptaglobin, profilin, ubiquitin, uteroglobulin and /3-2-microglobulm;<br>
Enzymes and inhibitors such as plasminogen, lysozyme, cystatin C, alpha- 1-antirrypsin and pancreatic trypsin inhibitor. Plasminogen is the inactive precursor of the trypsin-like serine protease plasmin. It is normally found circulating through the blood stream. When plasminogen becomes activated and is converted to plasmin, it unfolds a potent enzymatic domain that dissolves the fibrinogen fibers that entgangle the blood cells in a blood clot. This is called fibrinolysis.<br>
Immune system proteins, such as IgE, IgG, IgM.<br>
Transport proteins such as retinol binding protein, cn-1 micro globulin.<br>
Defensins such as beta-defensin 1, Neutrophil defensins 1,2 and 3.<br>
Proteins found at the blood brain barrier or in neural tissues, such as melanocortin receptor, myelin, ascorbate transporter.<br>
Transferrin receptor specific ligand-neuropharmaceutical agent fusion proteins (see US5977307);<br>
brain capillary endothelial cell receptor, transferrin, transferrin receptor, insulin, insulin-like growth factor 1 (IGF 1) receptor, insulin-like growth factor 2 (IGF 2) receptor, insulin receptor.<br>
Proteins localised to the kidney, such as polycystin, type IV collagen, organic anion transporter Kl, Heymann's antigen.<br>
Proteins localised to the liver, for example alcohol dehydrogenase, G250.<br>
Blood coagulation factor X<br>
cd antitrypsin<br>
HNFla<br>
Proteins localised to the lung, such as secretory component (binds IgA).<br>
Proteins localised to the Heart, for example HSP 27.   This is associated with dilated cardiomyopathy.<br>
Proteins localised to the skin, for example keratin.<br>
Bone specific proteins, such as bone morphogenic proteins (BMPs), which are a subset of the transforming growth factor /? superfamily that demonstrate osteogenic activity. Examples include BMP-2, -4, -5, -6, -7 (also referred to as osteogenic protein (OP-1) and -8 (OP-2).<br>
Tumour specific proteins, including human trophoblast antigen, herceptin receptor, oestrogen receptor, cathepsins eg cathepsin B (found in liver and spleen).<br>
Disease-specific proteins, such as antigens expressed only on activated T-cells: including LAG-3 (lymphocyte activation gene), osteoprotegerin ligand (OPGL)  see Nature 402, 304-309; 1999, OX40 (a member of the TNF receptor family, expressed on activated T cells and the only costimulatory T cell molecule known to be specifically up-regulated in human T cell leukaemia virus type-I (HTLV-I)-producing cells.)  See / Immunol 2000<br>
Jul 1 ;165(1):263-70; Metalloproteases (associated with arthritis/cancers), including CG6512 Drosophila, human paraplegin, human FtsH, human AFG3L2, murine ftsH; angiogenic growth factors, including acidic fibroblast growth factor (FGF-1), basic fibroblast growth factor (FGF-2), Vascular endothelial growth factor / vascular permeability factor (VEGF/VPF), transforming growth factor-a (TGF a), tumor necrosis factor-alpha (TNF-a), angiogenin, interleukin-3 (IL-3), interleukin-8 (IL-8), platelet-derived endothelial growth factor (PD-ECGF), placental growth factor (P1GF), midkine platelet-derived growth factor-BB (PDGF), fractalkine.<br>
Stress proteins (heat shock proteins)<br>
HSPs are normally found intracellularly. When they are found extracellularly, it is an indicator that a cell has died and spilled out its contents. This unprogrammed cell death (necrosis) only occurs when as a result of trauma, disease or injury and therefore in vivo, extracellular HSPs trigger a response from the immune system that will fight infection and disease, A dual specific which binds to extracellular HSP can be localised to a disease site.<br>
Proteins involved in Fc transport<br>
Brambell receptor (also known as FcRB)<br>
This Fc receptor has two functions, both of which are potentially useful for delivery<br>
The functions are<br>
(1)	The transport of IgG from mother to child across the placenta<br>
(2)	the protection of IgG from degradation thereby prolonging its serum half life of<br>
IgG. It is thought that the receptor recycles IgG from endosome.<br>
See Holliger et al, Nat Biotechnol 1997 Jul;15(7):632-6.<br>
Ligands according to the invention may designed to be specific for the above targets without requiring any increase in or increasing half life in vivo. For example, ligands according to the invention can be specific for targets selected from the foregoing which are tissue-specific, thereby enabling tissue-specific targeting of the dual specific ligand, or a dAb monomer that binds a tissue-specific therapeutically relevant target, irrespective of any increase in half-life, although this may result. Moreover, where the ligand or dAb<br>
monomer targets kidney or liver, this may redirect the ligand or dAb monomer to an alternative clearance pathway in vivo (for example, the ligand may be directed away from liver clearance to kidney clearance).<br>
H: Use of multispecific ligands according to the second configuration of the invention<br>
Multispecific ligands according to the method of the second configuration of the present invention may be employed in in vivo therapeutic and prophylactic applications, in vitro and in vivo diagnostic applications, in vitro assay and reagent applications, and the like. For example antibody molecules may be used in antibody based assay techniques, such as ELISA techniques, according to methods known to those skilled in the art.<br>
As alluded to above, the multispecific ligands according to the invention are of use in diagnostic, prophylactic and therapeutic procedures. Multispecific antibodies according to the invention are of use diagnostically in Western analysis and in situ protein detection by standard immunohistochemical procedures; for use in these applications, the ligands may be labelled in accordance with techniques known to the art. In addition, such antibody polypeptides may be used preparatively in affinity chromatography procedures, when complexed to a chromatographic support, such as a resin. All such techniques are well known to one of skill in the art.<br>
Diagnostic uses of the closed conformation multispecific ligands according to the invention include homogenous assays for analytes which exploit the ability of closed conformation multispecific ligands to bind two targets in competition, such that two targets cannot bind simultaneously (a closed conformation), or alternatively their ability to bind two targets simultaneously (an open conformation).<br>
A true homogenous immunoassay format has been avidly sought by manufacturers of diagnostics and research assay systems used in drug discovery and development.   The main diagnostics markets include human testing in hospitals, doctor's offices and clinics, commercial reference laboratories, blood banks, and the home, non-human diagnostics (for example food testing, water testing,  environmental testing,  bio-defence,  and<br>
veterinary testing), and finally research (including drug development; basic research and academic research).<br>
At present  all these markets utilise immune-assay systems that are built around chemiluminescent,    ELISA,    fluorescence   or   in   rare   cases   radio-immunoassay technologies.  Each of these assay formats requires a separation step (separating bound from un-bound reagents).  In some cases, several separation steps are required. Adding these additional steps adds reagents and automation, takes time, and affects the ultimate outcome of the assays.   In human diagnostics, the separation step may be automated, which masks the problem, but does not remove it.   The robotics, additional reagents, additional incubation times, and the like add considerable cost and complexity. In drug development, such as high throughput screening, where literally millions of samples are tested at once, with very low levels of test molecule, adding additional separation steps can eliminate the ability to perform a screen.  However, avoiding the separation creates too much noise in the read out. Thus, there is a need for a true homogenous format that provides sensitivities at the range obtainable from present assay formats. Advantageously, an assay possesses fully quantitative read-outs with high sensitivity and a large dynamic range.   Sensitivity is an important requirement, as is reducing the amount of sample required.  Both of these features are features that a homogenous system offers.  This is very important in point of care testing, and in drug development where samples are precious. Heterogenous systems, as currently available in the art, require large quantities of sample and expensive reagents<br>
Applications for homogenous assays include cancer testing, where the biggest assay is , that for Prostate Specific Antigen, used in screening men for prostate cancer. Other applications include fertility testing, which provides a    series of tests for women attempting to conceive including beta-hcg for pregnancy.  Tests for infectious diseases, including hepatitis, HTV, rubella, and other viruses and microorganisms and sexually transmitted diseases. Tests are used by blood banks, especially tests for HIV, hepatitis A, B, C, non A non B. Therapeutic drug monitoring tests include monitoring levels of prescribed drugs in patients for efficacy and to avoid toxicity, for example digoxin for arrhythmia, and phenobarbital levels in psychotic cases; theophylline for asthma. Diagnostic tests are moreover useful in abused drug testing, such as testing for cocaine,<br>
marijuana and the like. Metabolic tests are used for measuring thyroid function, anaemia and other physiological disorders and functions.<br>
The homogenous immunoassay format is moreover useful in the manufacture of standard clinical chemistry assays. The inclusion of immunoassays and chemistry assays on the same instrument is highly advantageous in diagnostic testing. Suitable chemical assays include tests for glucose, cholesterol, potassium, and the like.<br>
A further major application for homogenous immunoassays is drug discovery and development: high throughput screening includes testing combinatorial chemistry libraries versus targets in ultra high volume. Signal is detected, and positive groups then split into smaller groups, and eventually tested in cells and then animals. Homogenous assays may be used in all these types of test. In drug development, especially animal studies and clinical trials heavy use of immunoassays is made. Homogenous assays greatly accelerate and simplify these procedures. Other Applications include food and beverage testing: testing meat and other foods for E. coli, salmonella, etc; water testing, including testing at water plants for all types of contaminants including E. coli; and veterinary testing.<br>
In a broad embodiment, the invention provides a binding assay comprising a detectable agent which is bound to a closed conformation multispecific ligand according to the invention, and whose detectable properties are altered by the binding of an analyte to said closed conformation multispecific ligand. Such an assay may be configured in several different ways, each exploiting the above properties of closed conformation multispecific ligands.<br>
The assay relies on the direct or indirect displacement of an agent by the analyte, resulting in a change in the detectable properties of the agent. For example, where the agent is an enzyme which is capable of catalysing a reaction which has a detectable end-point, said enzyme can be bound by the ligand such as to obstruct its active site, thereby inactivating the enzyme. The analyte, which is also bound by the closed conformation multispecific ligand, displaces the enzyme, rendering it active through freeing of the active site. The enzyme is then able to react with a substrate, to give rise to a detectable event. In an<br>
alternative embodiment, the ligand may bind the enzyme outside of the active site, influencing the conformation of the enzyme and thus altering its activity. For example, the structure of the active site may be constrained by the binding of the ligand, or the binding of cofactors necessary for activity may be prevented.<br>
The physical implementation of the assay may take any form known in the art. For example, the closed conformation rnultispecific ligand/enzyme complex may be provided on a test strip; the substrate may be provided in a different region of the test strip, and a solvent containing the analyte allowed to migrate through the ligand/enzyme complex, displacing the enzyme, and carrying it to the substrate region to produce a signal. Alternatively, the ligand/enzyme complex may be provided on a test stick or other solid phase, and dipped into an analyte/substrate solution, releasing enzyme into the solution in response to the presence of analyte.<br>
Since each molecule of analyte potentially releases one enzyme molecule, the assay is quantitative, with the strength of the signal generated in a given time being dependent on the concentration of analyte in the solution.<br>
Further configurations using the analyte in a closed conformation are possible. For example, the closed conformation multispecific ligand may be configured to bind an enzyme in an allosteric site, thereby activating the enzyme. In such an embodiment, the enzyme is active in the absence of analyte. Addition of the analyte displaces the enzyme and removes allosteric activation, thus inactivating the enzyme.<br>
In the context of the above embodiments which employ enzyme activity as a measure of the analyte concentration, activation or inactivation of the enzyme refers to an increase or decrease in the activity of the enzyme, measured as the ability of the enzyme to catalyse a signal-generating reaction. For example, the enzyme may catalyse the conversion of an undetectable substrate to a detectable form thereof. For example, horseradish peroxidase is widely used in the art together with chromogenic or chemiluminescent substrates, which are available commercially. The level of increase or decrease of the activity of the enzyme may between 10% and 100%, such as 20%, 30%, 40%, 50%, 60%, 70%, 80% or 90%; in the case of an increase in activity, the increase may be more than 100%, i.e.<br>
200%, 300%, 500% or more, or may not be measurable as a percentage if the baseline activity of the inhibited enzyme is undetectable.<br>
In a further configuration, the closed conformation multispecific ligand may bind the substrate of an enzyme/substrate pair, rather than the enzyme. The substrate is therefore unavailable to the enzyme until released from the closed conformation multispecific ligand through binding of the analyte. The implementations for this configuration are as for the configurations which bind enzyme.<br>
Moreover, the assay may be configured to bind a fluorescent molecule, such as a fluorescein or another fluorophore, in a conformation such that the fluorescence is quenched on binding to the ligand. In this case, binding of the analyte to the ligand will displace the fluorescent molecule, thus producing a signal. Alternatives to fluorescent molecules which are useful in the present invention include luminescent agents, such as luciferin/luciferase, and chromogenic agents, including agents commonly used in immunoassays such as HRP.<br>
Therapeutic and prophylactic uses of multispecific ligands prepared according to the invention involve the administration of ligands according to the invention to a recipient mammal, such as a human. Multi-specificity can allow antibodies to bind to multimeric antigen with great avidity. Multispecific ligands can allow thecross- linking of two antigens, for example in recruiting cytotoxic T-cells to mediate the killing of tumour cell lines.<br>
Substantially pure ligands or binding proteins thereof, for example dAb monomers, of at least 90 to 95% homogeneity are preferred for administration to a mammal, and 98 to 99% or more homogeneity is most preferred for pharmaceutical uses, especially when the mammal is a human. Once purified, partially or to homogeneity as desired, the ligands may be used diagnostically or therapeutically (including extracorporeally) or in developing and performing assay procedures, immunofluorescent stainings and the like (Lefkovite and Perm's, (1979 and 1981) Immunological Methods, Volumes I and n, Academic Press, NY).<br>
The ligands or binding proteins thereof, for example dAb monomers, of the present invention will typically find use in preventing, suppressing or treating inflammatory states, allergic hypersensitivity, cancer, bacterial or viral infection, and autoimmune disorders (which include, but are not limited to, Type I diabetes, asthma, multiple sclerosis, rheumatoid arthritis, systemic lupus erythematosus, Crohn's disease and myasthenia gravis).<br>
In the instant application, the term "prevention" involves administration of the protective composition prior to the induction of the disease. "Suppression" refers to administration of the composition after an inductive event, but prior to the clinical appearance of the disease. "Treatment" involves administration of the protective composition after disease symptoms become manifest.<br>
Animal model systems which can be used to screen the effectiveness of the antibodies or binding proteins thereof in protecting against or treating the disease are available. Methods for the testing of systemic lupus erythematosus (SLE) in susceptible mice are known in the art (Knight et al. (1978) J. Exp. Med., 147: 1653; Reinersten et al. (1978) NewEng. J. Med., 299: 515). Myasthenia Gravis (MG) is tested in SJL/J female mice by inducing the disease with soluble AchR protein from another species (Lindstrom et al. (1988) Adv. ImmunoL, 42: 233). Arthritis is induced in a susceptible strain of mice by injection of Type n collagen (Stuart et al. (1984) Ann. Rev. ImmunoL, 42: 233). A model by which adjuvant arthritis is induced in susceptible rats by injection of mycobacterial heat shock protein has been described (Van Eden et al. (1988) Nature, 331: 171). Thyroiditis is induced in mice by administration of thyroglobulin as described (Maron et al. (1980) J. Exp. Med., 152: 1115). Insulin dependent diabetes mellitus (IDDM) occurs naturally or can be induced in certain strains of mice such as those described by Kanasawa et al. (1984) Diabetologia, 27: 113. EAE in mouse and rat serves as a model for MS in human. In this model, the demyelinating disease is induced by administration of myelin basic protein (see Paterson (1986) Textbook of Immunopathology, Mischer et al., eds., Grime and Stratton, New York, pp. 179-213; McFarlin et al. (1973) Science, 179: 478: and Satoh etal. (1987) J. ImmunoL, 138:179).<br>
Generally, the present ligands will be utilised in purified form together with pharmacologically appropriate carriers. Typically, these carriers include aqueous or alcoholic/aqueous solutions, emulsions or suspensions, any including saline and/or buffered media. Parenteral vehicles include sodium chloride solution, Ringer's dextrose, dextrose and sodium chloride and lactated Ringer's. Suitable physiologically-acceptable adjuvants, if necessary to keep a polypeptide complex in suspension, may be chosen from thickeners such as carboxymethylcellulose, polyvinylpyrrolidone, gelatin and alginates.<br>
Intravenous vehicles include fluid and nutrient replenishers and electrolyte replenishers, such as those based on Ringer's dextrose. Preservatives and other additives, such as antimicrobials, antioxidants, chelating agents and inert gases, may also be present (Mack (1982) Remington's Pharmaceutical Sciences, 16th Edition).<br>
The ligands of the present invention may be used as separately administered compositions or in conjunction with other agents. These can include various immunotherapeutic drugs, such as cylcosporine, methotrexate, adriamycin or cisplatinum, and immunotoxins. Pharmaceutical compositions can include "cocktails" of various cytotoxic or other agents in conjunction with the ligands of the present invention, or even combinations of lignds according to the present invention having different specificities, such as ligands selected using different target antigens or epitopes, whether or not they are pooled prior to administration.<br>
The route of administration of pharmaceutical compositions according to the invention may be any of those commonly known to those of ordinary skill in the art. For therapy, including without limitation immunotherapy, the selected ligands thereof of the invention can be administered to any patient in accordance with standard techniques. The administration can be by any appropriate mode, including parenterally, intravenously, intramuscularly, intraperitoneally, transdermally, via the pulmonary route, or also, appropriately, by direct infusion with a catheter.  The dosage  and frequency of administration will depend on the age, sex and condition of the patient, concurrent administration of other drugs, counterindications and other parameters to be taken into account by the clinician.<br>
The ligands of this invention can be lyophilised for storage and reconstituted in a suitable carrier prior to use. This technique has been shown to be effective with conventional immunoglobulins and art-known lyophilisation and reconstitution techniques can be employed. It will be appreciated by those skilled in the art that lyophilisation and reconstitution can lead to varying degrees of antibody activity loss (e.g. with conventional immunoglobulins, IgM antibodies tend to have greater activity loss than IgG antibodies) and that use levels may have to be adjusted upward to compensate.<br>
The compositions containing the present ligands or a cocktail thereof can be administered for prophylactic and/or therapeutic treatments, hi certain therapeutic applications, an adequate amount to accomplish at least partial inhibition, suppression, modulation, killing, or some other measurable parameter, of a population of selected cells is defined as a "therapeutically-effective dose". Amounts needed to achieve this dosage will depend upon the severity of the disease and the general state of the patient's own immune system, but generally range from 0.005 to 5.0 mg of ligand, e.g. antibody, receptor (e.g. a T-cell receptor) or binding protein thereof per kilogram of body weight, with doses of 0.05 to 2.0 mg/kg/dose being more commonly used. For prophylactic applications, compositions containing the present ligands or cocktails thereof may also be administered in similar or slightly lower dosages.<br>
Treatment performed using the compositions described herein is considered "effective" if one or more symptoms is reduced (e.g., by at least 10% or at least one point on a clinical assessment scale), relative to such symptoms present before treatment, or relative to such symptoms in an individual (human or model animal) not treated with such composition. Symptoms will obviously vary depending upon the disease or disorder targeted, but can be measured by an ordinarily skilled clinician or technician.   Such symptoms can be measured, for example, by monitoring the level of one or more biochemical indicators of the disease or disorder (e.g., levels of an enzyme or metabolite correlated with the disease, affected cell numbers, etc.), by monitoring physical manifestations (e.g., inflammation, tumor size, etc.), or by an accepted clinical assessment scale, for example, the Expanded Disability Status Scale (for multiple sclerosis), the Irvine Inflammatory Bowel Disease Questionnaire (32 point assessment evaluates quality of life with respect to bowel function, systemic symptoms, social function and emotional status - score ranges<br>
from 32 to 224, with higher scores indicating a better quality oflife), the Quality of Life Rheumatoid Arthritis Scale, or other accepted clinical assessment scale as known in the field. A sustained (e.g., one day or more, preferably longer) reduction in disease or disorder symptoms by at least 10% or by one or more points on a given clinical scale is indicative of "effective" treatment. Similarly, prophylaxis performed using a composition as described herein is "effective" if the onset or severity of one or more symptoms is delayed, reduced or abolished relative to such symptoms in a similar individual (human or animal model) not treated with the composition.<br>
A composition containing a ligand or cocktail thereof according to the present invention may be utilised in prophylactic and therapeutic settings to aid in the alteration, inactivation, killing or removal of a select target cell population in a mammal. In addition, the selected repertoires of polypeptides described herein may be used extracorporeally or in vitro selectively to kill, deplete or otherwise effectively remove a target cell population from a heterogeneous collection of cells. Blood from a mammal may be combined extracorporeally with the ligands, e.g. antibodies, cell-surface receptors or binding proteins thereof whereby the undesired cells are killed or otherwise removed from the blood for return to the mammal in accordance with standard techniques.<br>
I:        Use of half-life enhanced dual-specific ligands according to the invention<br>
Dual-specific ligands according to the method of the present invention may be employed in in vivo therapeutic and prophylactic applications, in vivo diagnostic applications and the like.<br>
Therapeutic and prophylactic uses of dual-specific ligands prepared according to the invention involve the administration of ligands according to the invention to a recipient mammal, such as a human. Dual specific antibodies according to the invention comprise at least one specificity for a half-life enhancing molecule; one or more further specificities may be directed against target molecules. For example, a dual-specific IgG may be specific for four epitopes, one of which is on a half-life enhancing molecule. Dual-specificity can allow antibodies to bind to multimeric antigen with great avidity. Dual-<br>
specific antibodies can allow the cross-linking of two antigens, for example in recruiting cytotoxic T-cells to mediate the killing of tumour cell lines.<br>
Substantially pure ligands or binding proteins thereof, such as dAb monomers, of at least 90 to 95% homogeneity are preferred for administration to a mammal, and 98 to 99% or more homogeneity is most preferred for pharmaceutical uses, especially when the mammal is a human. Once purified, partially or to homogeneity as desired, the ligands may be used diagnostically or therapeutically (including extracorporeally) or in developing and performing assay procedures, immunofluorescent stainings and the like (Lefkovite and Perm's, (1979 and 1981) Immunological Methods, Volumes I and n, Academic Press, NY).<br>
The ligands of the present invention will typically find use in preventing, suppressing or treating inflammatory states, allergic hypersensitivity, cancer, bacterial or viral infection, and autoimmune disorders (which include, but are not limited to, Type I diabetes, multiple sclerosis, rheumatoid arthritis, systemic lupus erythematosus, Crohn's disease and myasthenia gravis).<br>
In the instant application, the term "prevention" involves administration of the protective composition prior to the induction of the disease. "Suppression" refers to administration of the composition after an inductive event, but prior to the clinical appearance of the disease. "Treatment" involves administration of the protective composition after disease symptoms become manifest.<br>
Animal model systems which can be used to screen the effectiveness of the dual specific ligands in protecting against or treating the disease are available. Methods for the testing of systemic lupus erythematosus (SLE) in susceptible mice are known in the art (Knight et al. (1978) /. Exp. Med., 147: 1653; Reinersten et al (1978) New Eng. J. Med., 299: 515). Myasthenia Gravis (MG) is tested in SJL/J female mice by inducing the disease with soluble AchR protein from another species (Lindstrom et al. (1988) Adv. Immunol., 42: 233). Arthritis is induced in a susceptible strain of mice by injection of Type n collagen (Stuart et al. (1984) Ann. Rev. Immunol., 42: 233). A model .by which adjuvant arthritis is induced in susceptible rats by injection of mycobacterial heat shock protein has<br>
been described (Van Eden et al. (1988) Nature, 331:171). Thyroiditis is induced in mice by administration of thyroglobulin as described (Maron et al. (1980) J. Exp. Med., 152: 1115). Insulin dependent diabetes mellitus (IDDM) occurs naturally or can be induced in certain strains of mice such as those described by Kanasawa et al. (1984) Diabetologia, 27: 113. EAE in mouse and rat serves as a model for MS in human. In this model, the demyelinating disease is induced by administration of myelin basic protein (see Paterson (1986) Textbook of Immunopathology, Mischer et al., eds., Grune and Stratton, New York, pp. 179-213; McFarlin et al. (1973) Science, 179: 478: and Satoh et al. (1987) J. Immunol., 138: 179).<br>
Dual specific ligands according to the invention and dAb monomers able to bind to extracellular targets involved in endocytosis (e.g. Clathrin) enable dual specific ligands to be endocytosed, enabling another specificity able to bind to an intracellular target to be delivered to an intracellular environment. This strategy requires a dual specific ligand with physical properties that enable it to remain functional inside the cell. Alternatively, if the final destination intracellular compartment is oxidising, a well folding ligand may not need to be disulphide free.<br>
Generally, the present dual specific ligands will be utilised in purified form together with pharmacologically appropriate carriers. Typically, these carriers include aqueous or alcoholic/aqueous solutions, emulsions or suspensions, any including saline and/or buffered media. Parenteral vehicles include sodium chloride solution, Ringer's dextrose, dextrose and sodium chloride and lactated Ringer's. Suitable physiologically-acceptable adjuvants, if necessary to keep a polypeptide complex in suspension, may be chosen from thickeners such as carboxymethylcellulose, potyvmylpyrrolidone, gelatin and alginates.<br>
Intravenous vehicles include fluid and nutrient replenishers and electrolyte replenishers, such as those based on Ringer's dextrose. Preservatives and other additives, such as antimicrobials, antioxidants, chelating agents and inert gases, may also be present (Mack (1982) Remington's Pharmaceutical Sciences, 16th Edition).<br>
-The ligands of the.present invention maybe used as separately administered compositions	<br>
or in conjunction with other agents. These can include various immunotherapeutic drugs,<br>
such as cylcosporine, methotrexate, adriamycin or cisplatinum, and immunotoxins. Pharmaceutical compositions can include "cocktails" of various cytotoxic or other agents in conjunction with the ligands of the present invention.<br>
The route of administration of pharmaceutical compositions according to the invention may be any of those commonly known to those of ordinary skill in the art. For therapy, including without limitation immunotherapy, the ligands of the invention can be administered to any patient in accordance with standard techniques. The administration can be by any appropriate mode, including parenterally, intravenously, intramuscularly, intraperitoneally, transdermally, via the pulmonary route, or also, appropriately, by direct infusion with a catheter. The dosage and frequency of administration will depend on the age, sex and condition of the patient, concurrent administration of other drugs, counterindications and other parameters to be taken into account by the clinician.<br>
The ligands of the invention can be lyophilised for storage and reconstituted in a suitable carrier prior to use. This technique has been shown to be effective with conventional immunoglobulins and art-known lyophilisation and reconstitution techniques can be employed. It will be appreciated by those skilled in the art that lyophilisation and reconstitution can lead to varying degrees of antibody activity loss (e.g. with conventional immunoglobulins, IgM antibodies tend to have greater activity loss than IgG antibodies) and that use levels may have to be adjusted upward to compensate.<br>
The compositions containing the present ligands or a cocktail thereof can be administered for prophylactic and/or therapeutic treatments. In certain therapeutic applications, an adequate amount to accomplish at least partial inhibition, suppression, modulation, killing, or some other measurable parameter, of a population of selected cells is defined as a "therapeutically-effective dose". Amounts needed to achieve this dosage will depend upon the severity of the disease and the general state of the patient's own immune system, but generally range from 0.005 to 5.0 mg of ligand per kilogram of body weight, with doses of 0.05  to 2.0 mg/kg/dose being more commonly used. For prophylactic applications, compositions containing the present ligands or cocktails thereof may also be • administered in similar or slightly lower dosages.<br>
A composition containing a ligand according to the present invention may be utilised in prophylactic and therapeutic settings to aid in the alteration, inactivation, killing or removal of a select target cell population in a mammal.<br>
In addition, the selected repertoires of polypeptides described herein may be used extracorporeally or in vitro selectively to kill, deplete or otherwise effectively remove a target cell population from a heterogeneous collection of cells. Blood from a mammal may be combined extracorporeally with the ligands, e.g. antibodies, cell- surface receptors or binding proteins thereof whereby the undesired cells are killed or otherwise removed from the blood for return to the mammal in accordance with standard techniques.<br>
The invention is further described, for the purposes of illustration only, hi the following examples. As used herein, for the purposes of dAb nomenclature, human TNFa is referred to as TAR1 and human TNFa receptor 1 (p55 receptor) is referred to as TAR2.<br>
Example 1. Selection of a dual specific scFv antibody (K8) directed against human serum albumin (HSA) and (3-galactosidase (P -gal)<br>
This example explains a method for making a dual specific antibody directed against P-gal and HSA in which a repertoire of VK variable domains linked to a germline (dummy) VH domain is selected for binding to P-gal and a repertoire of VH variable domains linked to a germline (dummy) VK domain is selected for binding to HSA. The selected variable VH HSA and VK p-gal domains are then combined and the antibodies selected for binding to P-gal and HSA. HSA is a half-life increasing protein found in human blood.<br>
Four human phage antibody libraries were used in this experiment.<br>
Library 1        Germline VK/DVT VH	8.46 x 1O7<br>
Library 2        Germline VK/NNK VH	9.64 x 107<br>
Library 3        Germline VH/DVT VK	1.47 x 1()8<br>
Library 4        Germline VH/NNK VK	1.45 x 108<br>
All libraries are based on a single human framework for VH (V3-23/DP47 and Jn4b) and VK (O12/02/DPK9 and JK1) with side chain diversity incorporated in complementarity<br>
determining regions (CDR2 and CDR3).<br>
Library 1 and Library 2 contain a dummy VK sequence, whereas the sequence of VH is<br>
diversified at positions H50,'H52, H52a, H53, H55, H56, H58, H95, H96, H97 and H98 (DVT or NNK encoded, respectively) (Figure 1). Library 3 and Library 4 contain a dummy VH sequence, whereas the sequence of VK is diversified at positions L50, L53,<br>
L91, L92, L93, L94 and L96 (DVT or NNK encoded, respectively) (Figure 1). The libraries are in phagemid pIT2/ScFv format (Figure 2) and have been preselected for binding to generic ligands, Protein A and Protein L, so that the majority of clones hi the unselected libraries are functional. The sizes of the libraries shown above correspond to the sizes after preselection. Library 1 and Library 2 were mixed prior to selections on antigen to yield a single VH/dummy VK library and Library 3 and Library 4 were mixed<br>
to form a single VK/dummy VH library.<br>
Three rounds of selections were performed on (3-gal using VK/dummy VH library and three rounds of selections were performed on HSA using VH/dummy VK library. In the case of P-gal the phage litres went up from 1.1 x 10^ in the first round to 2.0 x 10^ in the third round. In the case of HSA the phage titres went up from 2 x 1()4 in the first round to<br>
1.4 x 10^ in the third round. The selections were performed as described by Griffith et al., (1993), except that KM13 helper phage (which contains a pni protein with a protease cleavage site between the D2 and D3 domains) was used and phage were eluted with 1 mg/ml trypsin in PBS. The addition of trypsin cleaves the pDI proteins derived from the helper phage (but not those from the phagemid) and elutes bound scFv-phage fusions by cleavage in the c-myc tag (Figure 2), thereby providing a further enrichment for phages expressing functional scFvs and a corresponding reduction in backgrqund (Kristensen &amp; Winter, Folding &amp; Design 3: 321-328, Jul 9, 1998). Selections were performed using immunotubes coated with either HSA or P-gal at 100jj.g/ml concentration.<br>
To check for binding, 24 colonies from the third round of each selection were screened by monoclonal phage ELISA. Phage particles were produced as described by Harrison et al., Methods Enzymol. 1996;267:83-109. 96-well ELISA plates were coated with lOOul of HSA or P-gal, at lOug/ml concentration in PBS overnight at 4°C. A standard ELISA protocol was followed (Hoogenboom et al., 1991) using detection of bound phage with anti-Mi 3-HRP conjugate. A selection of clones gave ELISA signals of greater than 1.0 with 50ul supernatant.<br>
Next, DNA preps were made from VH/dummy VK library selected on HSA and from VK/dummy VH library selected on p-gal using the QIAprep Spin Miniprep kit (Qiagen).<br>
To access most of the diversity, DNA preps were made from each of the three rounds of selections and then pulled together for each of the antigens. DNA preps were then digested with Sal//Not7 overnight at 37°C. Following gel purification of the fragments, VK chains from the VK/dummy VH library selected on P-gal were ligated in place of a dummy VK chain of the Vn/dummy VK library selected on HSA creating a library of 3.3<br>
clones.<br>
This library was then either selected on HSA (first round) and P-gal (second round), HSA/p-gal selection, or on p-gal (first round) and HSA (second round), p-gal/HSA selection. Selections were performed as described above. In each case after the second round 48 clones were tested for binding to HSA and P-gal by the monoclonal phage ELISA (as described above) and by ELISA of the soluble scFv fragments. Soluble antibody fragments were produced as described by Harrison et al., (1996), and standard ELISA protocol was followed Hoogenboom et al. (1991) Nucleic Acids Res., 19: 4133, except that 2% Tween/PBS was used as a blocking buffer and bound scFvs were detected with Protein L-HRP. Three clones (E4, E5 and E8) from the HSA/p-gal selection and two clones (K8 and K10) from the p-gal/HSA selection were able to bind both antigens. scFvs from these clones were PCR amplified and sequenced as described by Ignatovich et al., (1999) J Mol Biol 1999 Nov 26;294(2):457-65, using the primers LMB3 and pHENseq. Sequence analysis revealed that all clones were identical. Therefore, only one clone encoding a dual specific antibody (K8) was chosen for further work (Figure 3).<br>
Example 2. Characterisation of the binding properties of the K8 antibody.<br>
Firstly, the binding properties of the K8 antibody were characterised by the monoclonal phage ELISA. A 96-well plate was coated with lOOul of HSA and 0-gal alongside with alkaline phosphatase (APS), bovine serum albumin (BSA), peanut agglutinin, lysozyme and cytochrome c (to check for cross-reactivity) at 10jo,g/ml concentration in PBS overnight at 4°C. The phagemid from K8 clone was rescued with KM13 as described by Harrison et al., (1996) and the supernatant (50ul) containing phage assayed directly. A standard ELISA protocol was followed (Hoogenboom et al., 1991) using detection of bound phage with anti-Mi3-HRP conjugate. The dual specific K8 antibody was found to bind to HSA and p-gal when displayed on the surface of the phage with absorbance signals greater than 1.0 (Figure 4). Strong binding to BSA was also observed (Figure 4). Since HSA and BSA are 76% homologous on the ammo acid level, it is not surprising that K8 antibody recognised both of these structurally related proteins. No cross-reactivity with other proteins was detected (Figure 4).<br>
Secondly, the binding properties of the K8 antibody were tested in a soluble scFv ELISA. Production of the soluble scFv fragment was induced by IPTG as described by Harrison et al, (1996). To determine the expression levels of K8 scFv, the soluble antibody fragments were purified from the supernatant of 50ml inductions using Protein A-Sepharose columns as described by Harlow and Lane, Antibodies: a Laboratory Manual, (1988) Cold Spring Harbor. OD28Q was raen measured and the protein concentration calculated as described by Sambrook et al., (1989). K8 scFv was produced in supernatant at 19mg/l.<br>
A soluble scFv ELISA was then performed using known concentrations of the K8 antibody fragment. A 96-well plate was coated with lOOjal of HSA, BSA and P-gal at 10|j,g/ml and lOOjal of Protein A at lug/ml concentration. SOjal of the serial dilutions of the K8 scFv was applied and the bound antibody fragments were detected with Protein L-HRP. ELISA results confirmed the dual specific nature of the K8 antibody (Figure 5).<br>
To confirm that binding to (3-gal is determined by the VK domain and binding to HSA/BSA by the VH domain of the K8 scFv antibody, the VK domain was cut out from<br>
K8 scFv DNA by Sal//Not/ digestion and hgated into a Sal//Not/ digested pIT2 vector containing dummy VH chain (Figures 1 and 2). Binding characteristics of the resulting clone K8VK/dummy VH were analysed by soluble scFv ELISA. Production of the soluble<br>
scFv fragments was induced by IPTG as described by Harrison et al., (1996) and the supernatant (50p,) containing scFvs assayed directly. Soluble scFv ELISA was performed as described in Example 1 and the bound scFvs were detected with Protein L-HRP. The ELISA results revealed that this clone was still able to bind p-gal, whereas binding to BSA was abolished (Figure 6).<br>
Example 3. Selection of single Vjj domain antibodies antigens A and B and single VK domain antibodies directed against antigens C and D.<br>
This example describes a method for making single Vjj domain antibodies directed against antigens A and B and single VK domain antibodies directed against antigens C<br>
and D by selecting repertoires of virgin single antibody variable domains for binding to these antigens in the absence of the complementary variable domains.<br>
Selections and characterisation of the binding clones is performed as described previously (see Example 5, PCT/GB 02/003014). Four clones are chosen for further work:<br>
VHl-AntiAVH VH2-AntiBVH VKl-AntiCVK VK2-AntiDVK<br>
The procedures described above in Examples 1-3 may be used, in a similar manner as that described, to produce dimer molecules comprising combinations of VH domains (i.e., VH-VH ligands) and cominations of VL domains (VL-VL ligands).<br>
Example 4. Creation and characterisation of the dual specific ScFv antibodies (VH1/VH2 directed against antigens A and B and VK1/VK2 directed against antigens C and D).<br>
This example demonstrates that dual specific ScFv antibodies (VH1/VH2 directed against antigens A and B and VK1 /VK2 directed against antigens C and D) could be created by combining VK and VH single domains selected against respective antigens in a ScFv vector.<br>
To create dual specific antibody VH1/VH2, VH1 single domain is excised from variable domain vector 1 (Figure 7) by NcoUXhol digestion and ligated into NcdUXhol digested variable domain vector 2 (Figure 7) to create VH1/ variable domain vector 2. VH2 single domain is PCR amplified from variable domain vector 1 using primers to introduce SaH restriction site to the 5' end and Notl restriction site to the 3' end. The PCR product is then digested with SaWNotl and ligated into SaWNotl digested VH1/ variable domain vector 2 to create VH1/VH2/ variable domain vector 2.<br>
VK1/VK2/ variable domain vector 2 is created in a similar way. The dual specific nature of the produced VH1/VH2 ScFv and VK1/VK2 ScFv is tested in a soluble ScFv ELISA as described previously (see Example 6, PCT/GB 02/003014). Competition ELISA is performed as described previously (see Example 8, PCT/GB 02/003 014).<br>
Possible outcomes:<br>
-VH1/VH2 ScFv is able to bind antigens A and B simultaneously<br>
-VK1/VK2 ScFv is able to bind antigens C and D simultaneously<br>
-VH1/VH2 ScFv binding is competitive (when bound to antigen A, VH1/VH2 ScFv<br>
cannot bind to antigen B)<br>
-VK1/VK2 ScFv binding is competitive (when bound to antigen C, VK1/VK2 ScFv<br>
cannot bind to antigen D)<br>
Example 5. Construction of dual specific VH1/VH2 Fab and VK1/VK2 Fab and analysis of their binding properties.<br><br>
To create VH1/VH2 Fab, VH1 single domain is ligated into NcdVXhol digested CH vector (Figure 8) to create VH1/CH and VH2 single' domain is ligated into SaWNotl digested CK vector (Figure 9) to create VH2/CK. Plasmid DNA from VH1/CH and VH2/CK is used to co-transform competent E. coli cells as described previously (see Example 8, PCT/GB02/003014).<br>
The clone containing VH1/CH and VH2/CK plasmids is then induced by IPTG to produce soluble VH1/VH2 Fab as described previously (see Example 8, PCT/GB 02/003014).<br>
VK1/VK2 Fab is produced in a similar way.<br>
Binding properties of the produced Fabs are tested by competition ELISA as described previously (see Example 8, PCT/GB 02/003014).<br>
Possible outcomes:<br>
-VH1/VH2 Fab is able to bind antigens A and B simultaneously<br>
-VK1/VK2 Fab is able to bind antigens C and D simultaneously<br>
-VH1/VH2 Fab binding is competitive (when bound to antigen A, VH1/VH2 Fab cannot<br>
bind to antigen B)<br>
-VK1/VK2 Fab binding is competitive (when bound to antigen C, VK1/VK2 Fab cannot<br>
bind to antigen D)<br>
Example 6<br>
Chelatiug dAb Dirners<br>
Summary<br>
VH and VK homo-dimers are created in a dAb-linker-dAb format using flexible polypeptide linkers. Vectors were created in the dAb linker-dAb format containing glycine-serine linkers of different lengths 3U:(Gly4Ser)3, 5U:(Gly4Ser)5, 7U:(Gly4Ser)7. Dimer libraries were created using guiding dAbs upstream of the Linker: TAR1-5 (VK), TAR1-27(VK), TAR2-5(VH) or TAR2-6(VK) and a library of corresponding second<br><br>
dAbs after the linker. Using this method, novel dimeric dAbs were selected. The effect of dimerisation on antigen binding was determined by ELIS A and BIAcore studies and in cell neutralisation and receptor binding assays. Dimerisation of both TAR1-5 and TAR1-27 resulted in significant improvement in binding affinity and neutralisation levels.<br>
1.0	Methods<br>
1.1	Library generation<br>
1.1.1 Vectors<br>
pEDA3U, pEDASU and pEDA7U vectors were designed to introduce different linker lengths compatible with the dAb-linker-dAb format. For pEDASU, sense and anti-sense 73-base pair oligo linkers were annealed using a slow annealing program (95°C-5mins, 80°C-10mins,   70°C-15mins,   56°C-15mins,   42°C   until   use)   in   buffer   containing O.lMNaCl, lOmM Tris-HCl pH7.4 and cloned using the Xliol and Notl restriction sites. The linkers encompassed 3 (Gly4Ser) units and a stuffer region housed between Sail and Notl cloning sites (scheme 1). In order to reduce the possibility of monomeric dAbs being selected for by phage display, the stuffer region was designed to include 3 stop codons, a Sad restriction site and a frame shift mutation to put the region out of frame when no second dAb was present. For pEDASU and 7U due to the length of the linkers required, overlapping oligo-linkers were designed for each vector, annealed and elongated using Klenow. The fragment was then purified and digested using the appropriate enzymes before cloning using the Xhol and Notl restriction sites.<br>
Scheme 1<br>
1.1.2 Library preparation<br>
The N-terminal V gene corresponding to the guiding dAb was cloned upstream of the linker using Ncol and Xhol restriction sites. VH genes have existing compatible sites, however cloning VK genes required the introduction of suitable restriction sites. This was<br>
achieved by using modifying PCR primers (VK-DLJDBF: 5' cggccatggcgtcaacggacat; VKXholR: 5' atgtgcgctcgagcgtttgattt 3') in 30 cycles of PCR amplification using a 2:1 mixture of SuperTaq (HTBiotechnology Ltd)aadpfu turbo (Stratagene). This maintained the Ncol site at the 5' end while destroying the adjacent Sail site and introduced the Xliol site at the 3' end. 5 guiding dAbs were cloned into each of the 3 dimer vectors: TAR1-5 (VK), TAR1-27(VK), TAR2-5(VH), TAR2-6(VK) and TAR2-7(VK). All constructs were verified by sequence analysis.<br>
Having cloned the guiding dAbs upstream of the linker in each of the vectors (pEDASU, 5U and 7U): TAR1-5 (VK), TAR1-27(VK), TAR2-5(VH) or TAR2-6(VK) a library of corresponding  second dAbs were  cloned after the linker.  To  achieve this, the complimentary dAb libraries were PCR amplified from phage recovered from round 1 selections of either a VK library against Human TNFa (at approximately 1 x 106 diversity after round 1) when TAR1-5 or TARl-27 are the guiding dAbs, or a VH or VK library against human p55 TNF receptor (both at approximately 1 x 105 diversity after round 1) when TAR2-5 or TAR2-6 respectively are the guiding dAbs. For VK libraries PCR amplification was conducted using primers in 30 cycles of PCR amplification using a 2:1 mixture of SuperTaq and pfu turbo. VH libraries were PCR amplified using primers in order to introduce a Sail restriction site at the 5' end of the gene. The dAb library PCRs were digested with the appropriate restriction enzymes, ligated into the corresponding vectors down stream of the linker, using Sall/Notl restriction sites and electroporated into freshly prepared competent TGI cells.<br>
The titres achieved for each library are as follows: TAR1-5: pEDA3U = 4xl08, pEDASU = SxlO7, pEDA7U = IxlO8 TARl-27: pEDA3U = 6.2xl08, pEDASU = IxlO8, pEDA7U = IxlO9 TAR2h-5: pEDASU = 4xl07, pEDASU = 2 x 108, pEDATU = SxlO7 TAR2h-6: pEDASU = 7.4xl08, pEDASU = 1.2 x 108, pEDA7U = 2.2xl08<br>
1.2 Selections 1.2.1 TNFa<br>
Selections were conducted using human TNFa passively coated on immunotubes. Briefly, Immunotubes are coated overnight with l-4mls of the required antigen. The<br>
immunotubes were then washed 3 times with PBS and blocked with 2%milk powder in PBS for l-2hrs and washed a further 3 times with PBS. The phage solution is diluted in 2%milk powder in PBS and incubated at room temperature for 2hrs. The tubes are then washed with PBS and the phage eluted with Img/ml trypsin-PBS. Three selection strategies were investigated for the TAR1-5 dimer libraries. The first round selections were carried out in immunotubes using human TNFcc coated at l|ag/ml or 20jj,g/ml with 20 washes in PBS 0.1%Tween. TGI cells are infected with the eluted phage and the titres are determined (eg, Marks et al J Mol Biol. 1991 Dec 5;222(3):581-97, Richmann et al Biochemistry. 1993 Aug 31;32(34):8848-55).<br>
The titres recovered were:<br>
PEDA3U = 2.8xl07(Ijig/ml TNF) l.SxlO8 (20^ig/mlTNF), pEDASU = 1.8x 107 (lug/ml TNF), 1.6xl08 (20ug/ml TNF) pEDAVU = 8xl06 (lug/ml TNF), 7xl07 (20ug/ml TNF).<br>
The second round selections were carried out using 3 different methods.<br>
1.	In immunotubes, 20 washes with overnight incubation followed by a further 10<br>
washes.<br>
2.	hi immunotubes, 20 washes followed by Ihr incubation at RT in wash buffer with<br>
(1 ug/ml TNFa) and 10 further washes.<br>
3.	Selection on streptavidin beads using 33 pmoles biotinylated human TNFa<br>
(Henderikx et al., 2002, Selection of antibodies against biotinylated antigens.<br>
Antibody Phage Display: Methods and protocols, Ed. O'Brien and Atkin, Humana<br>
Press). Single clones from round 2 selections were picked into 96 well plates and<br>
crude supernatant preps were made in 2ml 96 well plate format.<br><br>
For TAR1-27, selections were carried out as described previously with the following modifications. The first round selections were carried out in immunotubes using human TNFcc coated at lju,g/ml or 20ug/ml with 20 washes in PBS 0.1%Tween. The second round selections were carried out in immunotubes using 20 washes with overnight incubation followed by a further 20 washes. Single clones from round 2 selections were picked into 96 well plates and crude supernatant preps were made in 2ml 96 well plate format.<br>
TAR1-27 titres are as follows:<br><br>
1.2.2 TNF RECEPTOR 1 (p55 RECEPTOR; TAR2)<br>
Selections were conducted as described previously for the TAR2h-5 libraries only. 3 rounds of selections were carried out in immunotubes using either 1 pig/ml human p55 TNF receptor or 10jig/ml human p55 TNF receptor with 20 washes in PBS 0.1%Tween with overnight incubation followed by a further 20 washes. Single clones from round 2 and 3 selections were picked into 96 well plates and crude supernatant preps were made in 2ml 96 well plate format.<br>
TAR2h-5 titres are as follows:<br>
1.3 Screening<br>
Single clones from round 2 or 3 selections were picked from each of the 3U, 5U and 7U libraries from the different selections methods, where appropriate. Clones were grown in 2xTY with 100jj,g/ml ampicillin and 1% glucose overnight at 37°C. A 1/100 dilution of this culture was inoculated into 2mls of 2xTY with lOOp-g/ml ampicillin and 0.1% glucose in 2ml, 96 well plate format and grown at 37°C shaking until OD600 was approximately 0.9. The culture was then induced with ImM IPTG overnight at 30°C. The supernatants were clarified by centrifugation at 4000rpm for 15 mins in a sorval plate centrifuge. The supernatant preps the used for initial screening.<br>
1.3.1 ELISA<br>
Binding activity of dimeric recombinant proteins was compared to monomer by Protein A/L ELISA or by antigen ELISA. Briefly, a 96 well plate is coated with antigen or Protein A/L overnight at 4°C. The plate washed with 0.05% Tween-PBS, blocked for 2hrs with 2% Tween-PBS.   The sample is added to the plate incubated for 1 hr at room temperature. The plate is washed and incubated with the secondary reagent for Ihr at room temperature. The plate is washed and developed with TMB substrate. Protein A/L-HRP or India-HRP was used as a secondary reagent. For antigen ELISAs, the antigen concentrations used were 1 jig/ml in PBS for Human TNFa and human THF receptor 1. Due to the presence of the guiding dAb in most cases dimers gave a positive ELISA signal therefore off rate determination was examined by BIAcore.<br>
1.3.2 BIAcore<br>
BlAcore analysys was conducted for TAR1-5 and TAR2h-5 clones. For screening,<br>
Human TNFawas coupled to a CMS chip at high density (approximately 10000 RUs).<br>
50 |al of Human TNFa(50 fig/ml) was coupled to the chip at 5 ul/min in acetate buffer -<br>
pH5.5. Regeneration of the chip following analysis using the standard methods is not<br>
possible due to the instability of Human TNFoc, therefore after each sample was analysed,<br>
the chip was washed for lOmins with buffer.<br>
For TAR1-5, clones supernatants from the round 2 selection were screened by BIAcore.<br>
48 clones were screened from each of the 3U, 5U and 7U libraries obtained using the<br>
following selection methods:<br>
Rl: 1/ig/ml human TNFa immunotube, R2 l^g/ml human TNFoc immunotube, overnight<br>
wash.<br>
Rl: 20jig/ml human TNFa immunotube, R2 20jig/ml human TNFa immunotube,<br>
overnight wash.<br>
Rl: 1/ig/ml human TNFa immunotube, R2 33 pmoles biotinylated human TNFa on<br>
beads.<br>
Rl: 20/ig/ml human TNFa immunotube, R2 33 pmoles biotinylated human TNFa beads.<br>
For screening, human p55 TNF receptor was coupled to a CMS chip at high density<br>
(approximately 4000 RUs).  100 jil of human p55 TNF receptor (10 jiig/ml) was coupled<br>
to the chip at Sjil/min in acetate buffer - pHS.5. Standard regeneration conditions were<br>
examined ( glycine pH2 or pH3) but in each case antigen was removed from the surface<br>
of the chip therefore as with TNFa, therefore after each sample was analysed, the chip<br>
was washed for lOmins with buffer.<br>
For TAR2-5, clones supernatants from the round 2 selection were screened.<br>
48 clones were screened from each of the 3U, 5U and 7U libraries, using the following<br>
selection methods:<br>
Rl: 1/ig/ml human p55 TNF receptor immunotube, R2 l^g/ml human p55 TNF receptor<br>
immunotube, overnight wash.<br>
Rl: 10/ig/ml human p55 TNF receptor immunotube, R2 lO/ig/ml human p55 TNF<br>
receptor immunotube, overnight wash.<br>
1.3.3 Receptor and Cell Assays The ability of the dimers to neutralise in the receptor assay was conducted as follows:<br>
Receptor binding<br>
Anti-TNF dAbs were tested for the ability to inhibit the binding of TNF to recombinant TNF receptor 1 (p55). Briefly, Maxisorp plates were incubated overnight with 30mg/ml anti-human Fc mouse monoclonal antibody (Zymed, San Francisco, USA). The wells were washed with phosphate buffered saline (PBS) containing 0.05% Tween-20 and then blocked with 1% BSA in PBS before being incubated with lOOng/ml TNF receptor 1 Fc fusion protein (R&amp;D Systems, Minneapolis, USA). Anti-TNF dAb was mixed with TNF which was added to the washed wells at a final concentration of lOng/ml. TNF binding was detected with 0.2mg/ml biotinylated anti-TNF antibody (HyCult biotechnology, Uben, Netherlands) followed by 1 in 500 dilution of horse radish peroxidase labelled streptavidin (Amersham Biosciences, UK) and then incubation with TMB substrate (KPL, Gaithersburg, USA). The reaction was stopped by the addition of HC1 and the absorbance was read at 450nm. Anti-TNF dAb activity lead to a decrease in TNF binding and therefore a decrease in absorbance compared with the TNF only control.<br>
L929 Cytotoxicity Assay<br>
Anti-TNF dAbs were also tested for the ability to neutralise the cytotoxic activity of TNF on mouse L929 fibroblasts (Evans, T. (2000) Molecular Biotechnology 15, 243-248). Briefly, L929 cells plated in microtitre plates were incubated overnight with anti-TNF dAb, lOOpg/ml TNF and Img/ml actinomycin D (Sigma, Poole, UK). Cell viability was measured by reading absorbance at 490nm following an incubation with [3-(4,5-dimethyltruazol-2-yl)-5-(3-carbboxymethoxyphenyl)-2-(4-sulfophenyl)-2H-tetrazoh'um (Promega, Madison, USA). Anti-TNF dAb activity lead to a decrease in TNF cytotoxicity and therefore an increase in absorbance compared with the TNF only control.<br>
In the initial screen, supernatants prepared for BIAcore analysis, described above, were also used in the receptor assay. Further analysis of selected dimers was also conducted in the receptor and cell assays using purified proteins.<br>
HeLa IL-8 assay<br>
Anti-TNFRl or anti-TNF alpha dAbs were tested for the ability to neutralise the induction of IL-S secretion by TNF in HeLa cells (method adapted from that of Akeson, L. et al (1996) Journal of Biological Chemistry 271, 30517-30523, describing the induction of IL-8 by EL-1 in HUVEC; here we look at induction by human TNF alpha and we use HeLa cells instead of the HUVEC cell line). Briefly, HeLa cells plated in microtitre plates were incubated overnight with dAb and 300pg/ml TNF. Post incubation the supernatant was aspirated off the cells and IL-8 concentration measured via a sandwich ELISA (R&amp;D Systems). Anti-TNFRl dAb activity lead to a decrease in IL-8 secretion into the supernatant compared with the TNF only control.<br>
The L929 assay is used throughout the following experiments; however, the use of the HeLa IL-8 assay is preferred to measure anti-TNF receptor 1 (p55) ligands; the presence of mouse p55 in the L929 assay poses certain limitations in its use.<br>
1.4	Sequence analysis<br>
Dimers that proved to have interesting properties in the BIAcore and the receptor assay screens were sequenced. Sequences are detailed in the sequence listing.<br>
1.5	Formatting<br>
1.5.1	TAR1-5-19 dimers<br>
The TAR1-5 dimers that were shown to have good neutralisation properties were reformatted and analysed in the cell and receptor assays. The TAR1-5 guiding dab was substituted with the affinity matured clone TAR1-5-19. To achieve this TAR1-5 was cloned out of the individual dimer pair and substituted with TAR1-5-19 that had been amplified by PCR. In addition, TAR1-5-19 homodimers were also constructed in the 3U, 5U and 7U vectors. The N terminal copy of the gene was amplified by PCR and cloned as described above and the C-terminal gene fragment was cloned using existing Sail and Notl restriction sites.<br>
1.5.2	Mutagenesis<br>
The amber stop codon present in dAb2, one of the C-terminal dAbs in the TAR1-5 dimer pairs was mutated to a glutamine by site-directed mutagenesis.<br>
1.5.3 Fabs<br>
The dimers containing TAR1-5 or TAR1-5-19 were re-formatted into Fab expression vectors. dAbs were cloned into expression vectors containing either the CK or CH genes using Sfi 1 and Notl restriction sites and verified by sequence analysis. The CK vector is derived from a pUC based ampicillin resistant vector and the CH vector is derived from a pACYC chloramphenicol resistant vector. For Fab expression the dAb-CH and dAb-CK constructs were co-transformed into HB2151 cells and grown in 2xTY containing 0.1% glucose, lOOfig/ml ampicillin and 10/ig/ml chloramphenicol.<br>
1.5.3 Hinge dimerisation<br>
Dimerisation of dAbs via cystine bond formation was examined. A short sequence of amino acids EPKSGDKTHTCPPCP a modified form of the human IgGCl hinge was engineered at the C terminal region on the dAb. An oligo linker encoding for this sequence was synthesised and annealed, as described previously. The linker was cloned into the pEDA vector containing TAR1-5-19 using Xliol and Notl restriction sites. Dimerisation occurs in situ in the periplasm.<br>
1.6 Expression and purification 1.6.1 Expression<br>
Supernatants were prepared in the 2ml, 96-well plate format for the initial screening as described previously. Following the initial screening process selected dimers were analysed further. Dimer constructs were expressed in TOPI OF' or HB2151 cells as supernatants. Briefly, an individual colony from a freshly streaked plate was grown overnight at 37°C in 2xTY with lOOug/ml ampicillin and 1% glucose. A 1/100 dilution of this culture was inoculated into 2xTY with 100jj,g/ml ampicillin and 0.1% glucose and grown at 37°C shaking until OD600 was approximately 0.9. The culture was then induced with ImM IPTG overnight at 30°C. The cells were removed by centrifugation and the supernatant purified with protein A or L agarose.<br>
Fab and cysteine hinge dimers were expressed as periplasmic proteins in HB2152 cells. A 1/100 dilution of an overnight culture was inoculated into 2xTY with 0.1% glucose and the appropriate antibiotics and grown at 30°C shaking until OD600 was approximately 0.9. The culture was then induced with ImM IPTG for 3-4 hours at 25°C. The cells were harvested by centrifugation and the pellet resuspended in periplasmic<br>
preparation buffer (30mM Tris-HCl pHS.O, ImM EDTA, 20% sucrose). Following centrifugation the supernatant was retained and the pellet resuspended in 5mM MgS04. The supernatant was harvested again by centrifugation, pooled and purified.<br>
1.6.2	Protein A/L purification<br>
Optimisation of the purification of dimer proteins from Protein L agarose (Affitech, Norway) or Protein A agarose (Sigma, UK) was examined. Protein was eluted by batch or by column elution using a peristaltic pump. Three buffers were examined 0.1M Phosphate-citrate buffer pH2.6, 0.2M Glycine pH2.5 and 0.1M Glycine pH2.5. The optimal condition was determined to be under peristaltic pump conditions using 0.1M Glycine pH2.5 over 10 column volumes. Purification from protein A was conducted peristaltic pump conditions using 0.1M Glycine pH2.5.<br>
1.6.3	FPLC purification<br>
Further purification was carried out by FPLC analysis on the AKTA Explorer 100 system (Amersham Biosciences Ltd). TAR1-5 and TAR1-5-19 dimers were fractionated by cation exchange chromatography (1ml Resource S - Amersham Biosciences Ltd) eluted with a 0-1M NaCl gradient in 50mM acetate buffer pH4. Hinge dimers were purified by ion exchange (1ml Resource Q Amersham Biosciences Ltd) eluted with a 0-1M NaCl gradient in 25mMTris HC1 pH 8.0. Fabs were purified by size exclusion chromatography using a superose 12 (Amersham Biosciences Ltd) column run at a flow rate of 0.5ml/min in PBS with 0.05% tween. Following purification samples were concentrated using vivaspin 5K cut off concentrators (Vivascience Ltd).<br>
2.0	Results<br>
2.1	TAR1-5 dimers<br>
6 x 96 clones were picked from the round 2 selection encompassing all the libraries and selection conditions. Supernatant preps were made and assayed by antigen and Protein L ELISA, BIAcore and in the receptor assays. In ELISAs, positive binding clones were identified from each selection method and were distributed between 3U, 5U and 7U libraries. However, as the guiding dAb is always present it was not possible to discriminate between high and low affinity binders by this method therefore BIAcore analysis was conducted.<br>
BIAcore analysis was conducted using the 2ml supematants. BIAcore analysis revealed that the dimer Koff rates were vastly improved compared to monomeric TAR1-5. Monomer Koff rate was in the range of 10"'M compared with dimer Koff rates which were in the range of 10~3 - lO^M. 16 clones that appeared to have very slow off rates were selected, these came from the 3U, 5U and 7U libraries and were sequenced. In addition the supematants were analysed for the ability to neutralise human TNFa in the receptor assay.<br>
6 lead clones (dl-d6 below) that neutralised in these assays and have been sequenced. The results shows that out of the 6 clones obtained there are only 3 different second dAbs (dAbl, dAb2 and dAb3) however where the second dAb is found more than once they are linked with different length linkers.<br>
TARl-5dl: 3U linker 2nd dAb=dAbl - lug/ml Ag immunotube overnight wash TARl-5d2: 3U linker 2nd dAb=dAb2 - lug/ml Ag immunotube overnight wash TARl-5d3: 5U linker 2nd dAb=dAb2 - lug/ml Ag immunotube overnight wash TARl-5d4: 5U linker 2nd dAb=dAb3 - 20ug/ml Ag immunotube overnight wash TARl-5d5: 5U linker 2nd dAb=dAbl - 20ug/ml Ag immunotube overnight wash TARl-5d6: 7U h'nker 2nd dAb=dAbl- Rl:lug/ml Ag immunotube overnight wash, R2:beads<br>
The 6 lead clones were examined further. Protein was produced from the periplasm and supernatant, purified with protein L agarose and examined in the cell and receptor assays. The levels of neutralisation were variable (Table 1). The optimal conditions for protein preparation were determined. Protein produced from HB2151 cells as supematants gave the highest yield (approximately lOmgs/L of culture). The supematants were incubated with protein L agarose for 2hrs at room temperature or overnight at 4°C. The beads were washed with PBS/NaCl and packed onto an FPLC column using a peristaltic pump. The beads were washed with 10 column volumes of PBS/NaCl and eluted with 0.1M glycine pH2.5. In general, dimeric protein is eluted after the monomer.<br>
TARl-5dl-6 dimers were purified by FPLC. Three species were obtained, by FPLC purification and were identified by SDS PAGE. One species corresponds to monomer and the other two species corresponds to dimers of different sizes. The larger of the two species is possibly due to the presence of C terminal tags. These proteins were examined in the receptor assay. The data presented in table 1 represents the optimum results obtained from the two dinieric species (Figure 11)<br>
The three second dAbs from the dimer pairs (ie, dAbl, dAb2 and dAb3) were cloned as monomers and examined by ELISA and in the cell and receptor assay. All three dAbs bind specifically to TNF by antigen ELISA and do not cross react with plastic or BSA. As monomers, none of the dAbs neutralise in the cell or receptor assays.<br>
2.1.2 TARl-5-19 dimers<br>
TARl-5-19 was substituted for TAR1-5 in the 6 lead clones. Analysis of all TARl-5-19 dimers in the cell and receptor assays was conducted using total protein (protein L purified only) unless otherwise stated (Table 2). TARl-5-19d4 and TARl-5-19d3 have the best NDso (~5nM) in the cell assay, this is consistent with the receptor assay results and is an improvement over TARl-5-19 monomer (ND5o~30nM). Although purified TAR1-5 dimers give variable results in the receptor and cell assays TARl-5-19 dimers were more consistent. Variability was shown when using different elution buffers during the protein purification. Elution using 0.1M Phosphate-citrate buffer pH2.6 or 0.2M Glycine pH2.5 although removing all protein from the protein L agarose in most cases rendered it less functional.<br>
TARl-5-19d4 was expressed in the fermenter and purified on cation exchange FPLC to yield a completely pure dimer. As with TARl-5d4 three species were obtained, by FPLC purification corresponding to monomer and two dimer species. This dimer was amino acid sequenced. TARl-5-19 monomer and TARl-5-19d4 were then examined in the receptor assay and the resulting IC50 for monomer was 30nM and for dimer was 8nM. The results of the receptor assay comparing TARl-5-19 monomer, TARl-5-19d4 and TARl-5d4 is shown in figure 10.<br>
TARl-5-19 homodimers were made in the 3U, 5U and 7U vectors, expressed and purified on Protein L. The proteins were examined in the cell and receptor assays and the resulting IC5os (for receptor assay) and ND5os (for cell assay) were determined (table 3, figure 12).<br>
2.2 Fabs<br>
TARl-5 and TARl-5-19 dimers were also cloned into Fab format, expressed and purified on protein L agarose. Fabs were assessed in the receptor assays (Table 4). The results showed that for both TARl-5-19 and TARl-5 dimers the neutralisation levels were similar to the original Gly^Ser linker dimers from which they were derived. A TARl-5-19 Fab where TARl-5-19 was displayed on both CH and CK was expressed, protein L purified and assessed in the receptor assay. The resulting IC50 was approximately InM.<br>
2.3 TAR1-27 dimers<br>
3 x 96 clones were picked from the round 2 selection encompassing all the libraries and selection conditions. 2ml supernatant preps were made for analysis in ELISA and bioassays. Antigen ELISA gave 71 positive clones. The receptor assay of crude supernatants yielded 42 clones with inhibitory properties (TNF binding 0-60%). In the majority of cases inhibitory properties correlated with a strong ELISA signal. 42 clones were sequenced, 39 of these have unique second dAb sequences. The 12 dimers that gave the best inhibitory properties were analysed further.<br>
The 12 neutralising clones were expressed as 200ml supernatant preps and purified on protein L. These were assessed by protein L and antigen ELISA, BIAcore and in the receptor assay. Strong positive ELISA signals were obtained in all cases. BIAcore analysis revealed all clones to have fast on and off rates. The off rates were improved compared to monomeric TAR1-27, however the off rate of TAR1-27 dimers was faster (Koff is approximately in the range of 10"1 and 10"2M) than the TARl-5 dimers examined previously (Koff is approximately in the range of 10"3 - lO^M). The stability of the purified dimers was questioned and therefore in order to improve stability, the addition on 5%glycerol, 0.5% Triton X100 or 0.5% NP40 (Sigma) was included in the. purification of 2 TAR1-27 dimers (d2 and d!6). Addition of NP40 or Triton XI00™ improved the yield<br>
of purified product approximately 2 fold. Both dimers were assessed in the receptor assay. TARl-27d2 gave IC50 of ~30nM under all purification conditions. TARl-27dl6 showed no neutralisation effect when purified without the use of stabilising agents but gave an IC50 of ~50nM when purified under stabilising conditions. No further analysis was conducted.<br>
2.4 TAR2-5 dimers<br>
3 x 96 clones were picked from the second round selections encompassing all the libraries and selection conditions. 2ml supernatant preps were made for analysis. Protein A and antigen ELISAs were conducted for each plate. 30 interesting clones were identified as having good off-rates by BIAcore (Koff ranges between 10"2 - 10"3M). The clones were sequenced and 13 unique dimers were identified by sequence analysis.<br>
Table 1: TAR1-5 dimers<br><br>
*note dimer 2 and dimer 3 have the same second dAb (called dAb2), however have different linker lengths (d2 = (Gly4Ser)3, d3 = (Gly4Ser)3). dAbl is the partner dAb to dimers 1, 5 and 6. dAb3 is the partner dAb to dimer4. None of the partner dAbs neutralise alone. FPLC purification is by cation exchange unless otherwise stated. The optimal dimeric species for each dimer obtained by FPLC was determined in these assays.<br><br>
Example 7<br>
dAb dimerisation by terminal cysteine linkage<br>
Summary<br>
For dAb dimerisation, a free cysteine has been engineered at the C-terminus of the protein. When expressed the protein forms a dirtier which can be purified by a two step purification method.<br>
PCR construction of TAR1-5-19CYS dimer<br>
See example 8 describing the dAb trimer. The trimer protocol gives rise to a mixture of monomer, dimer and trimer.<br>
Expression and purification of TAR1-5-19CYS dimer<br>
The dimer was purified from the supernatant of the culture by capture on Protein L agarose as outlined in the example 8.<br>
Separation of TAR1-5-19CYS monomer from the TAR1-5-19CYS dimer<br>
Prior to cation exchange separation, the mixed monomer/dimer sample was buffer exchanged into 50 mM sodium acetate buffer pH 4.0 using a PD-10 column (Amersham<br>
Pharmacia), following the manufacturer's guidelines. The sample was then applied to a ImL Resource S cation exchange column (Amersham Pharmacia), which had been pre-equilibrated with 50 mM sodium acetate pH 4.0. The monomer and dimer were separated using the following salt gradient in 50 mM sodium acetate pH 4.0:<br>
150 to 200 mM sodium chloride over 15 column volumes 200 to 450 mM sodium chloride over 10 column volumes 450 to 1000 mM sodium chloride over 15 column volumes<br>
Fractions containing dimer only were identified using SDS-PAGE and then pooled and the pH increased to 8 by the addition of 1/5 volume of 1M Tris pH 8.0.<br>
In vitro functional binding assay: TNF receptor assay and cell assay The affinity of the dimer for human TNFo; was determined using the TNF receptor and cell assay. IC50 in the receptor assay was approximately 0.3-0.8 nM; ND50 in the cell assay was approximately 3-8 nM.<br>
Other possible TAR1-5-19CYS dimer formats<br>
PEG dimers and custom synthetic maleimide dimers<br>
Nektar (Shearwater) offer a range of bi-fnaleimide PEGs [mPEG2-(MAL)2 or mPEG-(MAL)2j which would allow the monomer to be formatted as a dimer, with a small linker separating the dAbs and both being linked to a PEG ranging in size from 5 to 40 kDa. It has been shown that the 5kDa mPEG-(MAL)2 (ie, [TARl-5-19]-Cys-maleimide-PEG x 2, wherein the maleimides are linked together in the dimer) has an affinity in the TNF receptor assay of ~ 1-3 nM. Also the dimer can also be produced using TMEA (Tris[2-maleimidoethyljamine) (Pierce Biotechnology) or other bi-functional linkers.<br>
It is also possible to produce the disulphide dimer using a chemical coupling procedure using 2,2'-dithiodipyridine (Sigma Aldrich) and the reduced monomer.<br>
Addition of a polypeptide linker or hinge to the C-terminns of the dAb.<br>
A small linker, either (Gly4Ser)n where n=l to 10, eg, 1, 2, 3, 4, 5, 6 or 7, an immunoglobulin (eg, IgG hinge region or random peptide sequence (eg, selected from a library of random peptide sequences) can be engineered between the dAb and the terminal cysteine residue. This can then be used to make dimers as outlined above.<br>
Example 8<br>
dAb trimerisation<br>
Summary<br>
For dAb trimerisation, a free cysteine is required at the C-terminus of the protein. The cysteine residue, once reduced to give the free thiol, can then be used to specifically couple the protein to a trimeric maleimide molecule, for example TMEA (Tris[2-maleimidoethyljamine).<br>
PCR construction of TAR1-5-19CYS<br>
The following oligonucleotides were used to specifically PCR TAR1-5-19 with a SalL and BamEI sites for cloning and also to introduce a C-terminal cysteine residue:<br>
Sail<br>
Trp Ser Ala Ser Thr Asp* He Gin Met Thr Gin Ser Pro Ser Ser Leu Ser Ala Ser Val 1  TGG AGC GCG TCG ACG GAG ATC CAG ATG ACC CAG TCT CCA TCC TCT CTG TCT GCA TCT GTA ACC TCG CGC AGC TGC CXG TAG GTC TAG TGG GTC AGA GGT AGG AGA GAC AGA CGT AGA CAT<br>
Gly Asp Arg Val Tar lie Thr Cys Arg Ala Ser Gin Ser lie Asp Ser Tyr Leu His Trp<br>
61  GGA GAC CGT GTC ACC ATC ACT TGC CGG GCA ACT CAG AGC ATT GAT AGT TAT TTA CAT TGG<br>
CCT CTG GCA CAG TGG TAG TGA ACG GCC CGT TCA GTC TCG TAA CTA TCA ATA AAT GTA ACC<br>
Tyr Gin Gin Lys Pro Gly Lys Ala Pro Lys Leu Leu He Tyr Ser Ala Ser Glu Leu Gin<br>
121  TAG CAG CAG AAA CCA GGG AAA GCC CCT AAG CTC CTG ATC TAT AGT GCA TCC GAG TTG CAA<br>
ATG GTC GTC TTT GGT CCC TTT CGG GGA TTC GAG GAC TAG ATA TCA CGT AGG CTC AAC GTT<br>
Ser Gly Val Pro Ser Arg Phe Ser Gly Ser Gly Ser Gly Thr Asp Phe Thr Leu Thr He<br>
181  AGT GGG GTC CCA TCA CGT TTC AGT GGC AGT GGA TCT GGG ACA GAT TTC ACT CTC ACC ATC<br>
TCA CCC CAG GGT AGT GCA AAG TCA CCG TCA CCT AGA CCC TGT CTA AAG TGA GAG TGG TAG<br>
Ser Ser Leu Gin Pro Glu Asp Phe Ala Thr Tyr Tyr Cys Gin Gin Val Val Trp Arg Pro<br>
241  AGC AGT CTG CAA CCT GAA GAT TTT GCT ACG TAG TAG TGT CAA CAG GTT GTG TGG CGT CCT<br>
TCG TCA GAC GTT GGA CTT CTA AAA CGA TGC ATG ATG ACA GTT GTC CAA CAC ACC GCA GGA<br>
BamHI<br>
Phe Thr Phe Gly Gin Gly Thr Lys Val Glu lie Lya Arg Cya *** *** Ely ser GJ.y 301  TTT ACG TTC GGC CAA GGG ACC AAG GTG GAA ATC AAA CGG TGC TAA TAA GGA TCC GGC AAA TGC AAG CCG GTT CCC TGG TTC CAC CTT TAG TTT GCC ACG ATT ATT CCT AGG CCG<br>
(* start of TAR1-5-19CYS sequence)<br>
Forward primer<br>
5 '-TGGAGCGCGTCGACGGACATCCAGATGACCCAGTCTCCA-3'<br>
Reverse primer<br>
5 '-TTAGCAGCCGGATCCTTATTAGCACCGTTTGATTTCCAC-3'<br>
The PCR reaction (50uL volume) was set up as follows: 200uM dNTPs, OA\M of each primer, 5 uL of lOx P/wTurbo buffer (Stratagene), 100 ng of template plasmid (encoding TAR1-5-19), ljiL of P/wTurbo enzyme (Stratagene) and the volume adjusted to 50pL using sterile water. The following PCR conditions were used: initial denaturing step 94 °C for 2 mins, then 25 cycles of 94 °C for 30 sees, 64 °C for 30 sec and 72 °C for 30 sec. A final extension step was also included of 72 °C for 5 mins. The PCR product was purified and digested with Sail and BamEl and ligated into the vector which had also been cut with the same restriction enzymes. Correct clones were verified by DNA sequencing.<br>
Expression and purification of TAR1-5-19CYS<br>
TAR1-5-19CYS vector was transformed into BL21 (DE3) pLysS chemically competent cells (Novagen) following the manufacturer's protocol. Cells carrying the dAb plasmid were selected for using 100(j,g/mL carbenicillin and 37 ug/mL chloramphenicol. Cultures were set up in 2L baffled flasks containing 500 mL of terrific broth (Sigma-Aldrich), 100jj,g/mL carbenicillin and 37 |j,g/mL chloramphenicol. The cultures were grown at 30 °C at 200rpm to an O.D.600 of 1-1.5 and then induced with ImM IPTG (isopropyl-beta-D-thiogalactopyranoside, from Melford Laboratories). The expression of the dAb was allowed to continue for 12-16 hrs at 30 °C. It was found that most of the dAb was present in the culture media. Therefore, the cells were separated from the media by centrifugarion<br>
(8,000xg for 30 mins), and the supernatant used to purify the dAb. Per litre of supernatant, 30 mL of Protein L agarose (Affitech) was added and the dAb allowed to batch bind with stirring for 2 hours. The resin was then allowed to settle under gravity for a further hour before the supernatant was siphoned off. The agarose was then packed into a XK 50 column (Amersham Phamacia) and was washed with 10 column volumes of PBS. The bound dAb was eluted with 100 mM glycine pH 2.0 and protein containing fractions were then neutralized by the addition of 1/5 volume of 1 M Tris pH 8.0. Per litre of culture supernatant 20 mg of pure protein was isolated, which contained a 50:50 ratio of monomer to dirtier.<br>
Trimerisation of TAR1-5-19CYS<br>
2.5 ml of 100 uM TAR1-5-19CYS was reduce with 5 mM dithiothreitol and left at room temperature for 20 minutes. The sample was then buffer exchanged using a PD-10 column (Amersham Pharmacia). The column had been pre-equilibrated with 5 mM EDTA, 50 mM sodium phosphate pH 6.5, and the sample applied and eluted following the manufactures guidelines. The sample was placed on ice until required. TMEA (Tris[2-maleunidoethyljamine) was purchased from Pierce Biotechnology. A 20 mM stock solution of TMEA was made in 100% DMSO (dimethyl sulphoxide). It was found that a concentration of TMEA greater than 3:1 (molar ratio of dAb:TMEA) caused the rapid precipitation and cross-linking of the protein. Also the rate of precipitation and cross-linking was greater as the pH increased. Therefore using 100 jjM reduced TAR1-5-19CYS, 25 joM TMEA was added to trimerise the protein and the reaction allowed to proceed at room temperature for two hours. It was found that the addition of additives such as glycerol or ethylene glycol to 20% (v/v), significantly reduced the precipitation of the trimer as the coupling reaction proceeded. After coupling, SDS-PAGE analysis showed the presence of monomer, dimer and trimer in solution.<br>
Purification of the trimeric TAR1-5-19CYS<br>
40 juL of 40% glacial acetic acid was added per mL of the TMEA-TARl-5-19cys reaction to reduce the pH to ~4. The sample was then applied to a ImL Resource S cation exchange column (Amersham Pharmacia), which had been pre-equilibrated with 50 mM sodium acetate pH 4.0. The dimer and trimer were partially separated using a salt gradient of 340 to 450 mM Sodium chloride, 50 mM sodium acetate pH 4.0 over 30 column<br>
volumes. Fractions containing trimer only were identified using SDS-PAGE and then pooled and the pH increased to 8 by the addition of 1/5 volume of 1M Tris pH 8.0. To prevent precipitation of the trimer during concentration steps (using 5K cut off Viva spin concentrators; Vivascience), 10% glycerol was added to the sample.<br>
In vitro functional binding assay: TNT receptor assay and cell assay<br>
The affinity of the trimer for human TNFa was determined using the TNF receptor and cell assay. IC50 in the receptor assay was O.SnM; ND50 in the cell assay was in the range of 3 to lOnM (eg, 3nM).<br>
Other possible TAR1-5-19CYS trimer formats<br>
TAR1-5-19CYS may also be formatted into a trimer using the following reagents:<br>
PEG trimers and custom synthetic maleimide trimers<br>
Nektar (Shearwater) offer a range of multi arm PEGs, which can be chemically modified at the terminal end of the PEG. Therefore using a PEG trimer with a maleimide functional group at the end of each arm would allow the trimerisation of the dAb in a manner similar to that outlined above using TMEA. The PEG may also have the advantage in increasing the solubility of the trimer thus preventing the problem of aggregation. Thus, one could produce a dAb trimer in which each dAb has a C-terminal cysteine that is linked to a maleimide functional group, the maleimide functional groups being linked to a PEG trimer.<br>
Addition of a polypeptide linker or hinge to the C-terminus of the dAb A small linker, either (Gly4Ser)n where n= 1 to 10, eg, 1, 2, 3, 4, 5, 6 or 7 , an immunoglobulin (eg, IgG hinge region or random peptide sequence (eg, selected from a library of random peptide sequences) could be engineered between the dAb and the terminal cysteine residue. When used to make multimers (eg, dimers or trimers), this again would introduce a greater degree of flexibility and distance between the individual monomers, which may improve the binding characteristics to the target, eg a multisubunit target such as human TNFo.<br>
Example 9.<br>
Selection of a collection of single domain antibodies (dAbs) directed against human serum albumin (HSA) and mouse serum albumin (MSA).<br>
This example explains a method for making a single domain antibody (dAb) directed against serum albumin. Selection of dAbs against both mouse serum albumin (MSA) and human serum albumin (HSA) is described. Three human phage display antibody libraries were used in this experiment, each based on a single human framework for VH (see Figure 13: sequence of dummy VH based on V3-23/DP47 and JH4b) or VK (see Figure 15: sequence of dummy VK based on o!2/o2/DPK9 and Jkl) with side chain diversity encoded by NNK codons incorporated in complementarity determining regions (CDR1, CDR2 and CDR3).<br>
Library 1 (VH):<br>
Diversity at positions: H30, H31, H33, H35, H50, H52, H52a, H53, H55, H56, H58, H95,<br>
H97, H98.<br>
Library size:   6.2 xlO9<br>
Library 2 (VH):<br>
Diversity at positions: H30, H31, H33, H35, H50, H52, H52a, H53, H55, H56, H58, H95,<br>
H97, H98, H99, H100, HlOOa, HlOOb.<br>
Library size:	4.3 x 109<br>
Library 3 (VK):<br>
Diversity at positions: L30, L31, L32, L34, L50, L53, L91, L92, L93, L94, L96<br>
Library size:   2 x 109<br>
The VH and VK libraries have been preselected for binding to generic ligands protein A and protein L respectively so that the majority of clones in the unselected libraries are functional. The sizes of the libraries shown above correspond to the sizes after preselection.<br>
Two rounds of selection were performed on serum albumin using each of the libraries separately. For each selection, antigen was coated on immunotube (mine) in 4ml of PBS at a concentration of 100/ig/ml. In the first round of selection, each of the three libraries was panned separately against HSA (Sigma) and MSA (Sigma). In the second round of selection, phage from each of the six first round selections was panned against (i) the same antigen again (eg 1st round MSA, 2nd round MSA) and (ii) against the reciprocal antigen (eg 1st round MSA, 2nd round HSA) resulting in a total of twelve 2nd round selections. In each case, after the second round of selection 48 clones were tested for binding to HSA and MSA. Soluble dAb fragments were produced as described for scFv fragments by Harrison et al, Methods Enzymol. 1996;267:83-109 and standard ELISA protocol was followed (Hoogenboom et al. (1991) Nucleic Acids Res., 19: 4133) except that 2% tween PBS was used as a blocking buffer and bound dAbs were detected with either protein L-HRP (Sigma) (for the V/cs) and protein A -HRP (Amersham Pharmacia Biotech) (for the VHs).<br>
dAbs that gave a signal above background indicating binding to MSA, HSA or both were tested in ELISA insoluble form for binding to plastic alone but all were specific for serum albumin. Clones were then sequenced (see table below) revealing that 21 unique dAb sequences had been identified. The minimum similarity (at the amino acid level) between the VK dAb clones selected was 86.25% ((69/80)xlOO; the result when all the diversified residues are different, eg clones 24 and 34). The minimum similarity between the VH dAb clones selected was 94 % ((127/136)xlOO).<br>
Next, the serum albumin binding dAbs were tested for their ability to capture biotinylated antigen from solution. ELISA protocol (as above) was followed except that ELISA plate was coated with lj«g/ml protein L (for the VK clones) and IjKg/ml protein A (for the VH clones). Soluble dAb was captured from solution as in the protocol and detection was with biotinylated MSA or HSA and streptavidin HRP. The biotinylated MSA and HSA had been prepared according to the manufacturer's instructions, with the aim of achieving an average of 2 biotins per serum albumin molecule. Twenty four clones were identified that captured biotinylated MSA from solution in the ELISA. Two of these (clones 2 and 38 below) also captured biotinylated HSA. Next, the dAbs were tested for their ability to<br>
bind MSA coated on a CMS biacore chip. Eight clones were found that bound MSA on the biacore.<br>
In all cases the frameworks were identical to the frameworks in the corresponding dummy sequence, with diversity in the CDRs as indicated in the table above.<br>
Of the eight clones that bound MSA on the biacore, two clones that are highly expressed in E. coli (clones MSA16 and MSA26) were chosen for further study (see example 10). Full nucleotide and amino acid sequences for MSA16 and 26 are given hi figure 16.<br>
Example 10.<br>
Determination of affinity and serum half-life in mouse of MSA binding dAbs MSA16 and MSA26.<br>
dAbs MSA16 and MSA26 were expressed in the periplasm of E. coli and purified using batch absorbtion to protein L-agarose affinity resin (AfQtech, Norway) followed by elution with glycine at pH 2.2. The purified dAbs were then analysed by inhibition biacore to determine K<j. briefly purified msa16 and msa26 were tested to determine the concentration of dab required achieve response on a biacore cms chip coated with high density msa. once concentrations had been determined msa antigen at range around expected kd was premixed incubated overnight. binding in each premixes then measured flow-rate resulting curves used create klotz plots which gave an estimated k for b></j.>
Next, clones MSA16 and MSA26 were cloned into an expression vector with the HA tag (nucleic acid sequence: TATCCTTATGATGTTCCTGATTATGCA and amino acid sequence: YPYDVPDYA) and 2-10 mg quantities were expressed in E. coli and purified from the supernatant with protein L-agarose affinity resin (Affitech, Norway) and eluted with glycine at pH2.2.  Serum half life of the dAbs was determined in mouse.  MSA26 and MSA16 were dosed as single i.v. injections at approx 1.5mg/kg into GDI mice. Analysis of serum levels was by goat anti-HA (Abeam, UK) capture and protein L-HRP (invitrogen) detection ELISA which was blocked with 4% Marvel. Washing was with 0.05% tween PBS.  Standard curves of known concentrations of dAb were set up in the presence of Ixmouse serum to ensure comparability with the test samples. Modelling with a 2 compartment model showed MSA-26 had a tl/2d! of 0.16hr, a tl/2/3 of 14.5hr and an area under the curve (AUC) of 465hr.mg/ml (data not shown) and MS A-16 had a tl/2a of 0.98hr, a tl/2/3 of 36.5hr and an AUC of 913hr.mg/ml (figure 18). Both anti-MSA clones had considerably lengthened half life compared with HEL4 (an anti-hen egg white lysozyme dAb) which had a tl/2a of 0.06hr, and a tl/2/3 of 0.34hr.<br>
Example 11.<br><br>
Creation<br><br>
and VK- VK dual specific Fab like fragments<br><br>
This example describes a method for making VH- VH and Vtf-V/c dual specifics as Fab like fragments. Before constructing each of the Fab like fragments described, dAbs that bind to targets of choice were first selected from dAb libraries similar to those described in example 9. A VH dAb, HEL4, that binds to hen egg lysozyme (Sigma) was isolated and a second VH dAb (TAR2h-5) that binds to TNFa receptor (R and D systems) was also isolated. The sequences of these are given in the sequence listing. A VK dAb that binds TNFa (TAR1-5-19) was isolated by selection and affinity maturation and the sequence is also set forth in the sequence listing. A second VK dAb (MSA 26) described in example 9 whose sequence is in figure 17B was also used hi these experiments.<br>
DNA from expression vectors containing the four dAbs. described above was digested with enzymes Sail and NotI to excise the DNA coding for the dAb. A band of the expected size (300-400bp) was purified by running the digest on an agarose gel and excising the band, followed by gel purification using the Qiagen gel purification kit (Qiagen, UK). The DNA coding for the dAbs was then inserted into either the CH or CK vectors (Figs 8 and 9) as indicated in the table below.<br><br>
The VH CH and VH CK constructs were cotransformed into HB2151 cells. Separately, the VK CH and VK CK constructs were cotransformed into HB2151 cells. Cultures of each of the cotransformed cell lines were grown overnight (in 2xTy containing 5% glucose, 10/ig/ml Chloramphenicol and 100/ig/ml ampicillin to maintain antibiotic selection for both CH and CK plasmids). The overnight cultures were used to inoculate fresh media (2xTy, 10/xg/ml Chloramphenicol and IQQ/tg/ml ampicillin) and grown to OD 0.7-0.9 before induction by the addition of IPTG to express their CH and CK constructs.<br>
Expressed Fab like fragment was then purified from the periplasm by protein A purification (for the contransformed VH CH and VH C/c) and MSA affinity resin purification (for the contransformed V/c CH and V/c C/c).<br>
dual specific<br>
Expression of the VH CH and VH C/c dual specific was tested by running the protein on a gel. The gel was blotted and a band the expected size for the Fab fragment could be detected on the Western blot via both the myc tag and the flag tag, indicating that both the VH CH and VH C/c parts of the Fab like fragment were present. Next, in order to determine whether the two halves of the dual specific were present in the same Fab-like fragment, an ELISA plate was coated overnight at 4°C with 100 fil per well of hen egg lysozyme (HEL) at 3 mg/ml in sodium bicarbonate buffer. The plate was then blocked (as described in example 1) with 2% tween PBS followed by incubation with the VH CH /VK C/c dual specific Fab like fragment. Detection of binding of the dual specific to the HEL was via the non cognate chain using 9elO (a monoclonal antibody that binds the myc tag, Roche) and anti mouse IgG-HRP (Arnersham Pharmacia Biotech). The signal for the VH CH /VH C/c dual specific Fab like fragment was 0.154 compared to a background signal of 0.069 for the VH C/c chain expressed alone. This demonstrates that the Fab like fragment has binding specificity for target antigen.<br>
VK-VK dual specific<br>
After purifying the contransformed V/c CH and V/c CK dual specific Fab like fragment on an MSA affinity resin, the resulting protein was used to probe an ELISA plate coated with Ijitg/ml TNFa and an ELISA plate coated with 10/ig/ml MSA.  As predicted, there was signal above background when detected with protein L-HRP on bot ELISA plates (data not shown). This indicated that the fraction of protein able to bind to MSA (and therefore purified on the MSA affinity column) was also able to bind TNFa in a subsequent ELISA, confirming the dual specificity of the antibody fragment. This fraction of protein was then used for two subsequent experiments.   Firstly, an ELISA plate coated with 1/ig/ml TNFa was probed with dual specific V/c CH and V/c C/c Fab like fragment and also with a control TNFa binding dAb at a concentration calculated to give a similar signal on the ELISA.  Both the dual specific and control dAb were used to probe the ELISA plate in the presence and in the absence of 2mg/ml MSA. The signal in the dual<br>
specific well was reduced by more than 50% but the signal in the dAb well was not reduced at all (see figure 19a). The same protein was also put into the receptor assay with and without MSA and competition by MSA was also shown (see figure 19c). This demonstrates that binding of MSA to the dual specific is competitive with binding to TNFo.<br>
Example 12.<br>
Creation of a VK- VK dual specific cys bonded dual specific with specificity for mouse serum albumin and TNFa<br>
This example describes a method for making a dual specific antibody fragment specific for both mouse serum albumin and TNFo! by chemical coupling via a disulphide bond. Both MSA16 (from example 1) and TARl-5-19 dAbs were recloned into a pET based vector with a C terminal cysteine and no tags. The two dAbs were expressed at 4-10 mg levels and purified from the supernatant using protein L-agarose affinity resin (Affitiech, Norway). The cysteine tagged dAbs were then reduced with dithiothreitol. The TARl-5-19 dAb was then coupled with dithiodipyridine to block reformation of disulphide bonds resulting in the formation of PEP 1-5-19 homodimers. The two different dAbs were then mixed at pH 6.5 to promote disulphide bond formation and the generation of TARl-5-19, MSA16 cys bonded heterodimers. This method for producing conjugates of two unlike proteins was originally described by King et al. (King TP, Li Y    Kochoumian L Biochemistry. 1978 vol!7:1499-506 Preparation of protein conjugates via intermolecular disulfide bond formation.)  Heterodimers were separated from monomeric species by cation exchange-  Separation was confirmed by the presence of a band of the expected size on a SDS gel.  The resulting heterodimeric species was tested in the TNF receptor assay and found to have an IC50 for neutralising TNF of approximately 18 nM. Next, the receptor assay was repeated with a constant concentration of heterodimer (18nM) and a dilution series of MSA and HSA. The presence of HSA at a range of concentrations (up to 2 mg/ml) did not cause a reduction in the ability of the dimer to inhibit TNFa . However, the addition of MSA caused a dose dependant reduction in the ability of the<br>
dimer to inhibit TNFa (figure 20).This demonstrates that MSA and TNFa compete for binding to the cys bonded TAR1-5-19, MSA16 dimer.<br>
Example 13. Cloning and expression of the TAR1/TAR2 dual specific Fab<br>
TAR1-5-19 VK dAb (specific to human TNF alpha) was cloned into pDOMS CK Amp<br>
vector (Figure 21) as a Sall/NotI firagment. TAR2h-10-27 VH dAb (specific to human<br>
i<br>
TNFRI) was cloned into pDOMS CH Clilor vector (Figure 21) as a Sall/NotI fragment.<br>
The two vectors with cloned in dAbs were used to co-transform competent HB2151 cells. Amp/Chlor resistant clones (containing both plasmids) were used to make a large scale (101) fennentor prep of the Fab.<br>
The produced Fab was isolated from the culture supernatant (after 3 hours induction at 25C) using sequential Protein A/Protein L purification. The yield of the Fab was l.Smg.<br>
Example 14. Analysis of Fab properties in ELISA<br>
a)	Binding of the Fab to TAR1 and TAR2<br>
Binding of the TAR1/TAR2 Fab to TNF and TNFRI was tested in ELISA. A 96 well plate was coated with lOOul of TNF and TNFRI at lug/ml concentration in PBS overnight at 4C. 50ul (3uM) of Fab was then added to the wells and bound Fab was detected via non-cognate chain, ie using Protein A-HRP on TNF coated wells and Protein L-HRP on TNFRI coated wells. ELISA demonstrated the ability of the Fab to bind both antigens (Figure 22).<br>
b)	Sandwich ELISA<br>
To test the ability of the TAR1/TAR2 Fab to bind both antigens simultaneously (open/closed conformation?) a sandwich ELISA was performed. Here a 96 well plate was coated with mutant TNF (that does not bind to TNFRI, but does bind to PEP 1-5-19, data<br>
no shown; mutant TNF contains a single point mutation (N141Y) which renders it incapable of binding to TNFRI (Yamadishi et al., 1990, Protein Eng., 3, 713-9)) at lug/ml concentration in PBS overnight at 4C. 50ul of Fab (O.SuM) was then added. This was followed by addition of TNFRI-Fc fusion protein (R&amp;D Systems) and detection with Anti-Fc-HRP. The same sandwich ELISA was performed using a control Fab containing TARl/Ck chain and an irrelevant VH fused to the CH chain. ELISA results demonstrated the ability of the Fab to engage both antigens (TNF and TNFRI) simultaneously, suggesting an open conformation of the molecule (Figure 23).<br>
c) Competition ELISA<br>
To test the ability of the TAR1/TAR2 Fab to bind both antigens simultaneously two competition ELISAs were performed.<br>
A 96 well plate was coated with lOOul of TNFRI at lug/ml concentration in PBS overnight at 4C. A dilution of Fab was chosen such that OD450 of 0.3 was achieved upon detection with Protein L-HRP. This concentration was 6nM. The Fab was pre-incubated for an hour at room temperature with increasing concentrations of mutant TNF (up to 160x molar excess). As a negative control Fab was subjected to the same incubation with BSA. Following these incubations the mixtures were then put onto TNFRI coated ELISA plate and incubated for another hour. Bound TAR1/TAR2 Fab was detected using ProteinL-HRP. ELISA demonstrated that TAR1/TAR2 Fab binding to TNFRI was not affected by competing antigen (Figure 24).<br>
A 96 well plate was coated with lOOul of mutant TNF at lug/ml concentration in PBS overnight at 4C. A dilution of Fab was chosen such that OD450 of 0.3 was achieved upon detection with 9E10 (Sigma) followed by anti mo-HRP (Sigma). This concentration was 25nM. The Fab was pre-incubated for an hour at room temperature with increasing concentrations of soluble TNFRI (up to lOx molar excess). As a negative control Fab was subjected to the same incubation with BSA. Following these incubations the mixtures were then put onto mutant TNF coated ELISA plate and incubated for another hour. Bound TAR1/TAR2 Fab was detected using 9E10 followed by anti mo-HRP. ELISA<br>
demonstrated that TAR1/TAR2 Fab binding to mutant TNF was not affected by competing antigen (Figure 24).<br>
Example 15. Analysis of Fab properties in cell assays.<br>
To check the degree of functionality of each dAb in a TAR1/TAR2 Fab, the performance of the dual specific molecule was tested in the following cell assays:<br>
Human TNF cytotoxicity on murine cells.<br>
This assay tests the activity of TAR1 Dab, as TAR2 Dab cannot bind to murine TNF receptor expressed on the surface of the cells. TAR1-5-19 Dab and TAR2h-10-27 dAbs as well as TAR1-5-19 + TAR2h-10-27 dAb mixture were used as controls in this assay. The results demonstrate that TAR1-5-19 Dab in a Fab behaves as well as a monomeric TAR1-5-19 dAb (Figure 25).<br>
Murine TNF cytotoxicity assay on murine cells with human soluble TNF receptor.<br>
This assay tests the activity of TAR2h-10-27 (in this assay binding to soluble human TNFRI). TAR1-5-19 and TAR2h-10-27 dAbs as well as TAR1-5-19 + TAR2h-10-27 dAb mixture were used as controls in this assay. The results demonstrate that TAR2h-10-27 in a Fab behaves as well as a monomeric TAR2h-10-27 dAb (Figure 25).<br>
Murine TNF induced EL-8 secretion on human cells.<br>
This assay tests the activity of TAR2h-10-27 Dab (in this assay binding to membrane bound human TNFRI). TAR1-5-19 and TAR2h-10-27 dAbs as well as TAR1 + TAR2 dAb mixture were used as controls in this assay. The results demonstrate that TAR2h-10-27 in a Fab behaves as well as a monomeric TAR2h-10-27 dAb (Figure 25).<br>
Human TNF induced IL-8 secretion on human cells.<br>
This assay tests the activity of both TARl-5-19 and TAR2h-10-27 Dabs. TARl-5-19 Dab and TAR2h-10-27 dAbs as well as TARl-5-19 + TAR2h-10-27 dAb mixture were used as controls in this assay. The results demonstrate that Fab has a similar effect to the TAR2h-10-27 dAb and TARl-5-19 + TAR2h-10-27 dAb mixture (Figure 25).<br>
Murine TNF cytotoxicity on murine cells with soluble human TNFRI and increasing concentrations of mutant TNF (competition on cells).<br>
This assay was performed to test whether increasing concentration of mutant TNF (binding to TAR 1-5-19 Dab) will compromise binding of TAR2h-10-27 Dab to TNFRI in solution. The results of the assay indicate that that is not the case, thus the Fab is able to engage two antigens simultaneously (Figure 26).<br>
The assays described above demonstrate that each dAb in a Fab molecule functions as well as a monomeric dAb.<br>
Example 16. Construction of IgG vectors.<br>
pcDNA3.1(+) and pcDNA3.1/Zeo(+) backbones (Invitrogen) were used for cloning IgGl heavy chain constant region and light chain kappa constant region, respectively. The overview of the vectors is shown in Figure 27.<br>
Leaders:<br>
Two alternative types of leaders were used to facilitate secretion of the expressed protein:<br>
CD33 leader<br>
IgG K- chain leader<br>
The leaders were assembled by the annealing of the two complementary oligos (Table 5) and were cloned into pcDNA3.1(+) and pcDNA3.1/Zeo(+) as Nhel/HindHI fragments (Figure 27).<br>
IgGl heavy chain cloning:<br>
CHI domain was PCR amplified from the CH vector (as described in WO 03/002609) using primers shown in Table 5.<br>
Hinge region, CH2 and CH3 domains were PCR amplified from plgplus vector (Novagen) using primers shown in Table 5.<br>
The two products were then PCR assembled to create an IgGl heavy chain constant region which was cloned into pcDNA3.1(+) as a Notl/XhoI fragment (Figure 27).<br>
Kappa light chain cloning:<br>
CK domain was PCR amplified from the CK vector (see see WO 03/002609) using primers shown in Table 5.<br>
It was then cloned into pcDNA3.1/Zeo (+) as a Notl/XhoI fragment (Figure 27).<br>
Example 17. Cloning of TAR1-5-19 and TAR2h-10-27 dAbs into IgG vectors and production of IgG:<br>
TAR1-5-19 VK dAb (specific to human TNF alpha) was cloned into IgG kappa vectors (with CD33 and IgK leaders) as a Hindm/Notl fragment (Figure 27).<br>
TAR2h-10-27 VH dAb (specific to human TNFRI) was cloned into IgG heavy chain vectors (with CD33 and IgK leaders) as a HindlO/Notl fragment (Figure 27).<br>
Heavy and light chain plasraids were then co-transfected into COS? cells and IgG was expressed transiently for five days. The produced IgG was purified using streamline Protein A. Expression level - 250ng/ml. CD33 and IgG K leaders gave the same level of expression.<br>
Purified IgG was checked on a reducing and non-reducing SDS gel (produced bands of expected size) (data not shown).<br>
Example 18. Analysis of IgG properties in ELISA a) Binding of the IgG to TNF and TNFRI<br>
Binding of the TAR1/TAR2 IgG to TNF and TNFRI was tested in ELISA. A 96 well plate was coated with lOOul of TNF and TNFRI at lug/ml concentration in PBS overnight at 4C. 50ul (200nM) of IgG was then added to the wells and bound IgG was detected via anti-Fc-HRP. ELISA demonstrated the ability of the IgG to bind both antigens (Figure 28).<br>
Example 19. Analysis of IgG properties in cell assays.<br>
To check the degree of functionality of each dAb in a TAR1/TAR2 IgG, the performance of the dual specific molecule was tested in the following cell assays:<br>
Human TNF cytotoxicity on murine cells.<br>
This assay tests the activity of TARl-5-19 Dab, as TAR2h-10-27 Dab cannot bind to murine TNF receptor expressed on the surface of the cells. TARl-5-19 and TAR2h-10-27 dAbs as well as TARl-5-19 + TAR2h-10-27 dAb mixture were used as controls in this assay. The results demonstrate that TARl-5-19 hi the IgG behaves better than<br>
monomeric TARl-5-19 dAb, which indicates that igCi is able to simultaneously engage two molecules of TNF (ND50 of the dimeric molecule) (Figure 29).<br>
Murine TNF cytotoxicity assay on murine cells with human soluble TNF receptor.<br>
This assay tests the activity of TAR2h-10-27 (in this assay binding to soluble human TNFRI). TARl-5-19 and TAR2h-10-27 dAbs as well as TARl-5-19 + TAR2h-10-27 dAb mixture were used as controls in this assay. The results demonstrate that TAR2h-10-27 in IgG behaves as well as a monomeric TAR2h-10-27 dAb (Figure 29).<br>
Murine TNF induced IL-8 secretion on human cells.<br>
This assay tests the activity of TAR2h-10-27 (in this assay binding to membrane bound human TNFRI). TARl-5-19 and TAR2h-10-27 dAbs as well as TARl-5-19 + TAR2h-10-27 dAb mixture were used as controls in this assay. The results demonstrate that IgG is able to engage two molecules of TNFRI on the surface of the cell (agonistic activity) (Figure 29). This assay was also repeated with no human TNF present. The results demonstrate that the IgG induces IL-8 release on human cells up to a concentration of 30nM after which the agonistic activity goes down (Figure 30).<br>
Human TNF induced IL-8 secretion on human cells.<br>
This assay tests the activity of both TARl-5-19 and TAR2h-10-27. TARl-5-19 and TAR2h-10-27 dAbs as well as TARl-5-19 + TAR2h-10-27 dAb mixture were used as controls in this assay. The results demonstrate that IgG has a similar effect to the TARh-10-27 dAb and TARl-5-19 + TAR2h-10-27 dAb mixture (Figure 29). This assay was also repeated with no murine TNF present. The results demonstrate that the IgG induces IL-8 release on human cells up to a concentration of 30nM after which the agonistic activity goes down (Figure 30).<br>
Table 5. Primers/Oligos<br>
CkbckNot<br>
5' AAGGAAAAAAGCGGCCGCAACTGTGGCTGCACCATC 3'<br>
CkforXho<br>
5' CCGCTCGAGTCAACACTCTCCCCTGTTGAAGCTCTTTGTG 3'<br>
ChlbckNot<br>
5' AAGGAAAAMGCGGCCGCCTCCACCAAGGGCCCATCGGTC 3'<br>
Chlfor<br>
5' GTGAGGTTTGTCACAAGATTTGGGCTCAACTTTCTTGTCCACC 3'<br>
Fcbck<br>
5' CCCAAATCTTGTGACAAACCTCAC 3'<br>
FcforXho<br>
5' CCGCTCGAGTCATTTACCCGGAGACAGGGAG 3'<br>
LEADER CD33:<br>
Leacdl<br>
5'P CTAGCCACCATGCCGCTGCTGCTACrGCTGCCACTGCTGTGGGCAGGAGCACTGGCTATGGATA 3'<br>
LeacdZ<br>
5'PAGCTTATCCATAGCCAGTGCTCCTGCCCACAGCAGTGGCAGCAGTAGCAGCAGCGGCATGGTGG 3'<br>
LEADER IGGK:<br>
Leakl<br>
5'P<br>
CTAGCCACCATGGAGACAGACACACTCCTGCTATGGGTACTGCTGCTCTGGGTTCCAGGTTCCACTGGT GACA 3 '<br>
Leak2<br>
5'P<br>
AGCTTGTCACCAGTGGAACCTGGAACCCAGAGCAGCAGTACCCATAGCAGGAGTGTGTCTGTCTCCAT GGTGG 3'<br>
SEQBACK<br>
5'TAATACGACTCACTATAGGG 3'<br>
SEQFOR<br>
5' TAGMGGCACAGTCGAGG 3'<br>
Data Summary<br>
A summary of data obtained in the experiments set forth in the foregoing examples is set forth in Annex 4.<br>
All publications mentioned in the present specification, and references cited in said publications, are herein incorporated by reference. Various modifications and variations of the described methods and system of the invention will be apparent to those skilled in the art without departing from the scope and spirit of the invention. Although the invention has been described hi connection with specific preferred embodiments, it should be understood that the invention as claimed should not be unduly limited to such specific embodiments. Indeed, various modifications of the described modes for carrying out the invention which are obvious to those skilled in molecular biology or related fields are intended to be within the scope of the following claims.<br><br>
Annex 1; polypeptides which enhance half-life in vivo.<br>
Alpha-1 Glycoprotein (Orosomucoid) (AAG)<br>
Alpha-1 Antichyromotrypsin (ACT)<br>
Alpha-1 Antitrypsin (AAT)<br>
Alpha-1 Microglobulin (Protein HC) (AM)<br>
Alpha-2 Macroglobulin (A2M)<br>
Antithrombin m (AT HI)<br>
Apolipoprotein A-l (Apo A-l)<br>
Apoliprotein B (Apo B)<br>
Beta-2-microglobulin (B2M)<br>
Ceruloplasmin (Cp)<br>
Complement Component (C3)<br>
Complement Component (C4)<br>
Cl Esterase Inhibitor (Cl DSfH)<br>
C-Reactive Protein (CRP)<br>
Cystatin C (Cys C)<br>
Ferritin (PER)<br>
Fibrinogen (FIB)<br>
Fibronectin (FN)<br>
Haptoglobin (Hp)<br>
Hemopexin (HPX)<br>
Immunoglobulin A (IgA)<br>
hnmunoglobulin D (IgD)<br>
hnmunoglobulin E (IgE)<br>
Immunoglobulin G (IgG)<br>
hmnunoglobulin M (IgM)<br>
Immunoglobulin Light Chains (kapa/lambda)<br>
Lipoprotein(a) [Lp(a)]<br>
Mannose-bindign protein (MBP)<br>
Myoglobhi (Myo)<br>
Plasminogen (PSM)<br>
Prealbumin (Transthyretin) (PAL)<br>
Retinol-binding protein (RBP) Rheomatoid Factor (RF) Seruni Amyloid A (SAA) Soluble Tranferrin Receptor (sTfR) Transferrin (Tf)<br>
TNF  ALPHA/IL-13<br>
TNF  ALPHA/IL-14<br>
TNF ALPHML-15<br>
TNF  ALPHA/IL-16<br>
TNF  ALPHA/IL-17<br>
TNF  ALPHA/IL-18<br>
TNF  ALPHA/IL-19<br><br>
Sustained elevated levels of VCAM-1 in cultured fibroblast-like synoviocytes can be achieved by TNF-alpha in combination with either IL-4 or IL-13 through increased mRNA stability. Am J Pathol. 1999 Apr; 154(4): 1149-58.<br>
Interleukin-13 and tumour necrosis factor-alpha synergistically induce eotaxin production in human nasal fibroblasts. Clin Exp Allergy. 2000 Mar;30(3):348-55.<br>
Interleukin-13 and tumour necrosis factor-alpha synergistically induce eotaxin production in human nasal fibroblasts. Clin Exp Allergy. 2000 Mar;30(3):348-55 (allergic inflammation)<br>
Implications of serum IMF-beta and IL-13 in the treatment response of<br>
childhood nephrotic syndrome. Cytokine. 2003 Feb 7;21(3): 155-9.	<br>
Effects of inhaled tumour necrosis factor alpha in subjects with mild<br>
asthma. Thorax. 2002 Sep;57(9):774-8.	<br>
Effects of inhaled tumour necrosis factor alpha hi subjects with mild<br>
asthma. Ttiorax. 2002 Sep;57(9):774-8.	____^__<br>
Tumor necrosis factor-alpha-induced synthesis of interleukin-16 in airway<br>
epithelial cells: priming for serotonin stimulation. Am J Respir Cell Mol<br>
Biol. 2003 Mar;28(3):354-62. (airway inflammation)<br>
Correlation of circulating interleukin 16 with proinflammatory cytokines in<br>
patients   with   rheumatoid   arthritis.   Rheumatology   (Oxford).   2001<br>
Apr;40(4):474-5. No abstract available.<br>
Interleukin 16 is up-regulated in Crohn's disease and participates in TNBS<br>
colitis in mice. Gastroenterology. 2000 Oct;119(4):972-82.	<br>
Inhibition of interleukin-17 prevents the development of arthritis in<br>
vaccinated mice challenged with Borrelia burgdorferi. Infect Immiin. 2003<br>
Jun;71(6):3437-42.<br>
Interleukin 17 synergises with tumour necrosis factor alpha to induce<br>
cartilage destruction in vitro. Ann Rheum Dis. 2002 Oct;61(10):870-6.<br>
A role of GM-CSF hi the accumulation of neutrophils hi the airways caused<br>
by IL-17 and TNF-alpha. Eur Respir J. 2003 Mar;21(3):387-93. (Airway<br>
inflammation)<br>
Abstract Interleukin-1, tumor necrosis factor alpha, and interleukin-17<br>
synergistically up-regulate nitric oxide and prostaglandin E2 production hi<br>
explants of human osteoarthritic knee menisci. Arthritis Rheum. 2001<br>
Sep;44(9):2078-83.	<br>
Association of interleukin-18 expression with enhanced levels of both<br>
interleukin-lbeta and tumor necrosis factor alpha in knee synovial tissue of<br>
patients with rheumatoid arthritis.  Arthritis Rheum. 2003 Feb;48(2):339-<br>
47.<br>
Abstract Elevated levels of interleukin-18 and tumor necrosis factor-alpha<br>
in serum of patients with type 2 diabetes mellitus: relationship with diabetic<br>
nephropathy. Metabolism. 2003 May;52(5):605-8.	<br>
Abstract IL-19 induces production of IL-6 and TNF-alpha and results in cell apoptosis through TNF-alpha. J Immunol. 2002 Oct 15;169(8):4288-97.<br><br><br><br>
TNF  ALPHA/IL-20<br>
TNF<br>
ALPHA/Complem<br>
ent<br>
TNF ALPHA/IFN-y<br><br>
Abstract Cytokines: IL-20 - a new effector in skin inflammation. Curr Biol<br>
2001 Jul 10;ll(13):R531-4			   	<br>
Inflammation and coagulation: implications for the septic patient.   Clin Infect Dis. 2003 May 15;36(10):1259-65. Epub 2003 May OS. Review.<br>
•	MHC induction hi the brain.<br>
•	Synergize in anti-viral response/IFN-p induction.<br>
•	Neutrophil activation/ respiratory burst.<br>
•	Endotheh'al cell activation<br>
•	Toxicities noted when patients treated with TNF/IFN-Y as anti-viral therapy<br><br>
•	Fractalkine expression by human astrocytes.<br>
•	Many papers on inflammatory responses - i.e. LPS, also macrophage<br>
activation.<br>
•	Anti-TNF   and   anti-IFN-y   synergize   to   protect   mice   from   lethal<br>
endotoxemia.<br><br>
Annex 3: Oncology combinations<br><br>
Annex 4<br>
Data Summary<br><br><br>
TAR1-5 heterodimers<br><br>
With (Qly4Ser)n linker TARl-5dl=30nM TARl-5d2 = 50nM TARl-5d3 = 300nM TARl-5d4 = 3nM TARl-5d5 = 200nM TARl-5d6 = lOOnM<br>
In Fab format TAR1-5CH d2CK = 30nM TAR1-5CK d3CH = lOOnM<br><br>
= 60nM<br><br><br><br>
TAR1-5-19 homorrimer<br><br>
0.3nM<br><br>
3-10nM (eg, 3nM)<br><br><br><br>
TAR2<br><br>
TAR2 monomers<br><br>
As TAR1 monomer<br><br>
As TAR1 monomer<br><br>
SOOnM to lOOpM<br><br>
SOOnM to 50pM<br><br><br><br>
Serum Albumin<br><br>
TAR2-10<br>
TAR2-5<br>
Anti-SA monomers<br><br>
InMtoSOO/iM, preferably   lOOnM  to   10<br><br>
InM to 500/iM, preferably lOOnM to 10<br><br><br><br>
In Dual Specific format, target affinity is 1 to 100,000 x affinity of SA dAb affinity, eg lOOpM (target) and 10 /*M SA affinity.<br><br>
In Dual Specific format, target affinity is 1 to 100,000 x affinity of SA dAb affinity, eg lOOpM (target) and 10 /JM SA affinity.<br><br><br><br>
MSA-16<br>
MSA-26<br><br>
200nM<br>
70nM<br><br><br>
Claims<br>
1.	A dual specific ligand comprising a first dAb specific for a target ligand, and a<br>
second dAb specific for a receptor for the target ligand.<br>
2.	A dual specific ligand according to claim 1, which is an open conformation ligand<br>
and can bind both the target ligand and the target ligand receptor simultaneously.<br>
3.	A dual specific ligand according to claim 1 or claim 2, wherein the target ligand is<br>
TNFa.<br>
4.	A dual specific ligand according to claim 3, wherein the dAb specific for TNFa<br>
dissociates from human TNFa with a dissociation constant (Kd) of 50nM to 20pM, and a<br>
Koff rate constant of SxlO'1 to IxlO"7 s"1, as determined by surface plasmon resonance.<br>
5.	A dual specific ligand according to claim 3 or claim 4, wherein the dAb specific<br>
for TNFa is a VK.<br>
6.	A dual specific ligand according to claim 5, wherein the dAb specific for<br>
TNFa comprises the amino acid sequence of TAR1-5-19 or a sequence that is at least<br>
80% homologous thereto.<br>
7.	A dual specific ligand according to claim 5, wherein the dAb specific for<br>
TNFa comprises the amino acid sequence of TAR1-5 or a sequence that is at least 80%<br>
homologous thereto.<br>
8.	A dual specific ligand according to claim 5, wherein the dAb specific for<br>
TNFa comprises the amino acid sequence of TARl-27 or a sequence that is at least 80%<br>
homologous thereto.<br>
9.	A dual specific ligand according to any one of claims 1 to 3, wherein the target<br>
ligand receptor is TNF receptor 1 (p55).<br><br>
10. A dual specific ligand according to claim 9, wherein the dAb specific for TNF receptor 1 (p55) dissociates from human TNF receptor 1 with a dissociation constant (Kd) of 50nM to 20pM, and a Koff rate constant of SxlO"1 to IxlO'7 s"1, as determined by surface plasmon resonance.<br>
11.	A dual specific ligand according to claim 10, wherein the dAb specific for<br>
TNF receptor 1 (p55) neutralises human TNFoc or TNF receptor 1 in a standard cell assay<br>
with an ND50 of SOOnM to 50pM.<br>
12.	A dual specific ligand according to claim 10 or claim 11, wherein the dAb specific<br>
for TNF receptor 1 (p55) antagonises the activity of the TNF receptor 1 in a standard cell<br>
assay with an NDso of ^OOnM, and at a concentration of ^0/iM the dAb agonises the<br>
activity of the TNF receptor 1 by ^% in the assay.<br>
13.	A dual specific ligand according to any one of claims 10 to 12, wherein the dAb<br>
specific for TNF receptor 1 (p55) comprises the amino acid sequence of TAR2h-10 or a<br>
sequence that is at least 80% homologous thereto.<br>
14.	A dual specific ligand according to any one of claims 10 to 12, wherein the dAb<br>
specific for TNF receptor 1 (p55) comprises the amino acid sequence of TAR2h-10 or a<br>
sequence that is at least 90% homologous thereto.<br>
15.	A dual specific ligand according to any one of claims 10 to 12, wherein the dAb<br>
specific for TNF receptor 1 (p55) comprises the amino acid sequence of TAR2h-5 or a<br>
sequence that is at least 80% homologous thereto.<br>
16.	A dual specific ligand according to any one of claims 10 to 12, wherein the dAb<br>
specific for TNF receptor 1 (p55) comprises the amino acid sequence of TAR2h-5 or a<br>
sequence that is at least 90% homologous thereto.<br>
17.	A dual specific ligand according to any one of claims 10 to 12, wherein the dAb<br>
specific for TNF receptor 1 (p55) comprises the amino acid sequence of TAR2h-10-27 or<br>
a sequence that is at least 80% homologous thereto.<br><br>
18.	A dual specific ligand according to any one of claims 10 to 12, wherein the dAb<br>
specific for TNF receptor 1 (p55) comprises the amino acid sequence of TAR2h-10-27 or<br>
a sequence that is at least 90% homologous thereto.<br>
19.	A dual specific ligand according to any one of claims 3 to 18, wherein the TNFcc<br>
and/or TNF receptor 1 (p55) is in human form.<br>
20.	A dual specific ligand according to any preceding claim, wherein one or more<br>
dAbs further comprises a terminal Cys residue.<br>
21.	A dual specific ligand according to claim 1 or claim 2, wherein the target ligand is<br>
TNFoc and the target ligand receptor is TNF Receptor 1 (p55).<br>
22.	A dual specific ligand according to claim 21, comprising a TAR1-5-19 VK domain<br>
or a sequence that is at least 80% homologous thereto and a TAR2h-10-27 VH domain or<br>
a sequence that is at least 80% homologous thereto.<br>
23.	A dual specific ligand according to claim 22, wherein the TAR1-5-19 VK domain<br>
and  a  TAR2h-10-27  VH  domain,   or  the   homologues  thereof,   are  arranged  as<br>
complementary domains in a binding site.<br>
24.	A dual specific ligand according to any preceding claim, which is a dimer.<br>
25.	A dual specific ligand according to any one of claims 1 to 23, which is a trimer.<br>
26.	A dual specific ligand according to claim 23, comprising two ligands specific for<br>
TNFa and one ligand specific for TNF Receptor 1 (p55).<br>
27.	A dual specific ligand according to any preceding claim, said dual specific tigand<br>
being selected from the group consisting of a Fab, a F(ab')2 and an IgG immunoglobulin.<br><br>
28. A dual specific ligand selected from the group consisting of a Fab, a F(ab')2 and an IgG immunoglobulin, wherein said dual specific ligand is in an open conformation and can bind two or more targets simultaneously, and said dual specific ligand comprises at least one dAb specific for TNFa which is TAR1-5-19 or a sequence at least 80% homologous thereto.<br>
t 29.      A dual specific ligand according to claim 28, which comprises at least one further<br>
dAb, which further dAb is a VH domain.<br>
30.	A dual specific ligand selected from the group consisting of a Fab, a<br>
an IgG irnmunoglobulin, wherein said dual specific ligand is in an open conformation and can bind two or more targets simultaneously, and said dual specific ligand comprises at least one dAb specific for TNF Receptor 1 (p55) which is TAR2h-10-27 or a sequence at least 80% homologous thereto.<br>
31.	A dual specific ligand according to claim 28, which comprises at least one further<br>
dAb, which further dAb is a VK domain.<br>
32.	A dual specific ligand according to claim 29 or claim 31, wherein the further dAb<br>
binds serum albumin (SA).<br>
33.	A dual specific ligand according to claim 32, wherein the dAb specific for serum<br>
albumin (SA) dissociates from SA with a dissociation constant (Kd) of InM to 500uM, as<br>
determined by surface plasmon resonance.<br>
34.	A dual specific ligand according to claim 32 or claim 33, wherein the dAb specific<br>
for serum albumin (SA) binds SA in a standard ligand binding assay with an IC50 of InM<br>
to SOOuM.<br>
35.	A dual specific ligand according to any one of claims 32 to 34, wherein the dAb<br>
specific for serum albumin (SA) comprises the amino acid sequence of MS A- 16 or a<br>
sequence that is at least 80% homologous thereto.<br><br>
36. A dual specific ligand according to any one of claims 32 to 34, wherein the dAb specific for serum albumin (SA) comprises the amino acid sequence of MSA-26 or a sequence that is at least 80% homologous thereto.<br>
37.	A dual specific selected from the group consisting of a Fab, a F(ab^2 and an IgG<br>
immunoglobulin, wherein said dual specific ligand is in an open conformation and can<br>
bind two or more targets simultaneously, and said dual specific ligand comprises at least<br>
one dAb specific for TNF Receptor 1 (p55) which is TAR2h-10-27 or a sequence at least<br>
80% homologous thereto, and at least one dAb specific for TNFa which is TAR1-5-19 or<br>
a sequence at least 80% homologous thereto.<br>
38.	A dual specific ligand according to any one of claims 27 to 37, which comprises a<br>
universal framework.<br>
39.	A dual specific ligand according to claim 38, wherein the universal framework<br>
comprises a VH framework selected from the group consisting of DP47, DP45 and DP38;<br>
and/or the VL framework is DPK9.<br>
40.	A dual specific ligand according to any one of claims 27 to 39, which comprises a<br>
binding site for a generic ligand.<br>
41.	A dual specific ligand according to claim 40, wherein the generic ligand binding<br>
site is selected from the group consisting of protein A, protein L and protein G.<br>
42.	A dual specific ligand according to any one of claims 27 to 41, wherein the ligand<br>
comprises a variable domain having one or more framework regions comprising an amino<br>
acid sequence that is the same as the amino acid sequence of a corresponding framework<br>
region encoded by a human germline antibody gene segment, or the amino acid sequences<br>
of one or more of said framework regions collectively comprises up to 5 amino acid<br>
differences relative to the amino acid sequence of said corresponding framework region<br>
encoded by a human germline antibody gene segment.<br><br>
43. A dual specific ligand according to any one of claims 27 to 41, wherein the ligand comprises a variable domain, wherein the amino acid sequences of FW1, FW2, FW3 and FW4 are the same as the amino acid sequences of corresponding framework regions encoded by a human germline antibody gene segment, or the amino acid sequences of FW1, FW2, FW3 and FW4 collectively contain up to 10 amino acid differences relative to the amino acid sequences of corresponding framework regions encoded by said human germline antibody gene segment.<br>
44.	A dual specific ligand according to claim 42 or claim 43, which comprises an<br>
antibody variable domain comprising FW1, FW2 and FW3 regions, and the aminp acid<br>
sequence of said FW1, FW2 and FW3 are the same as the amino acid sequences of<br>
corresponding framework regions encoded by human germline antibody gene segments.<br>
45.	A dual specific ligand according to any one of claims 42 to 44, wherein said<br>
human germline antibody gene segment is selected from the group consisting of DP47,<br>
DP45,DP48andDPK9.<br>
46.	A dual specific ligand according to any preceding claim, comprising a VH domain<br>
that is not a Camelid immunoglobulin variable domain.<br>
47.	The ligand of claim 46, comprising a VH domain that does not contain one or<br>
more amino acids that are specific to Camelid immunoglobulin variable domains as<br>
compared to human VH domains.<br>
48.	A dAb monomer dAb specific for TNF Receptor 1 (p55) which is TAR2h-10-27<br>
or a sequence at least 80% homologous thereto.<br>
49.	A nucleic acid encoding a dAb monomer according to claim 48.<br>
50.	A nucleic acid vector comprising a nucleic acid according to claim 49.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1ERUxOUC0yMDA1LUFic3RyYWN0LSgxMS0wMy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">2381-DELNP-2005-Abstract-(11-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1kZWxucC0yMDA1LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2381-delnp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1ERUxOUC0yMDA1LUNsYWltcy0oMTEtMDMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">2381-DELNP-2005-Claims-(11-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1kZWxucC0yMDA1LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2381-delnp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTEtMDMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">2381-DELNP-2005-Correspondence-Others-(11-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjAtMDktMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">2381-DELNP-2005-Correspondence-Others-(20-09-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1kZWxucC0yMDA1LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2381-delnp-2005-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1ERUxOUC0yMDA1LURlc2NyaXB0aW9uIChDb21wbGV0ZSktKDExLTAzLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">2381-DELNP-2005-Description (Complete)-(11-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1kZWxucC0yMDA1LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">2381-delnp-2005-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1ERUxOUC0yMDA1LURyYXdpbmdzLSgxMS0wMy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">2381-DELNP-2005-Drawings-(11-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1kZWxucC0yMDA1LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">2381-delnp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1ERUxOUC0yMDA1LUZvcm0tMS0oMTEtMDMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">2381-DELNP-2005-Form-1-(11-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1kZWxucC0yMDA1LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">2381-delnp-2005-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1kZWxucC0yMDA1LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">2381-delnp-2005-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1ERUxOUC0yMDA1LUZvcm0tMi0oMTEtMDMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">2381-DELNP-2005-Form-2-(11-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1kZWxucC0yMDA1LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">2381-delnp-2005-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1kZWxucC0yMDA1LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">2381-delnp-2005-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1kZWxucC0yMDA1LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">2381-delnp-2005-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1ERUxOUC0yMDA1LUdQQS0oMTEtMDMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">2381-DELNP-2005-GPA-(11-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1kZWxucC0yMDA1LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">2381-delnp-2005-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1ERUxOUC0yMDA1LU90aGVyLSgxMS0wMy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">2381-DELNP-2005-Other-(11-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1kZWxucC0yMDA1LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">2381-delnp-2005-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1kZWxucC0yMDA1LXBjdC0zMDYucGRm" target="_blank" style="word-wrap:break-word;">2381-delnp-2005-pct-306.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1kZWxucC0yMDA1LXBjdC00MDgucGRm" target="_blank" style="word-wrap:break-word;">2381-delnp-2005-pct-408.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1kZWxucC0yMDA1LXBjdC00MDkucGRm" target="_blank" style="word-wrap:break-word;">2381-delnp-2005-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1kZWxucC0yMDA1LXBjdC00MTYucGRm" target="_blank" style="word-wrap:break-word;">2381-delnp-2005-pct-416.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1kZWxucC0yMDA1LXBjdC1kZW1hbmQgZm9ybS5wZGY=" target="_blank" style="word-wrap:break-word;">2381-delnp-2005-pct-demand form.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1kZWxucC0yMDA1LXBjdC1ub3RpZmljYXRpb24ucGRm" target="_blank" style="word-wrap:break-word;">2381-delnp-2005-pct-notification.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1kZWxucC0yMDA1LXBjdC1yZXF1ZXN0IGZvcm0ucGRm" target="_blank" style="word-wrap:break-word;">2381-delnp-2005-pct-request form.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM4MS1kZWxucC0yMDA1LXBjdC1zZWFyY2ggcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">2381-delnp-2005-pct-search report.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="219482-a-piperidine-hydroxyl-alkyl-amide-compound.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="219484-pesticide-composition-for-preventing-or-reducing-resistance-to-pesticides-of-substrate-pest.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>219483</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2381/DELNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>26/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Jun-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>07-May-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>03-Jun-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>DOMANTIS LIMITED</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>IAN TOMLINSON</td>
											<td></td>
										</tr>
										<tr>
											<td>2</td>
											<td>GREG WINTER</td>
											<td></td>
										</tr>
										<tr>
											<td>3</td>
											<td>PHILIP JONES</td>
											<td></td>
										</tr>
										<tr>
											<td>4</td>
											<td>OLGA IGNATOVICH</td>
											<td></td>
										</tr>
										<tr>
											<td>5</td>
											<td>BEN WOOLVEN</td>
											<td></td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 16/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/GB2003/005646</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-12-24</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0327706.8</td>
									<td>2003-11-28</td>
								    <td>U.K.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>PCT/GB03/002804</td>
									<td>2003-06-30</td>
								    <td>U.K.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>0230202.4</td>
									<td>2002-12-27</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/219483-a-dual-specific-ligand by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:48:07 GMT -->
</html>
