<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/219765-a-set-of-fissr-pcr-primers-and-methods-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:29:28 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 219765:A SET OF FISSR-PCR PRIMERS AND METHODS THEREOF</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A SET OF FISSR-PCR PRIMERS AND METHODS THEREOF</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to set of inter-simple sequence repeats (ISSR)-PCR primers of SEQ ID Nos. 1 to 37 for genotyping eukaryotes and a method of genotyping diverse genomes of plant and animal systems using FISSR-PCR primers and SSR markers; more particularly, a FISSR and SSR method of distinguishing Basmati rice varieties from NonBasmati (NB) rice varieties, and Traditional Basmati (TB) rice varieties from Evolved Basmati (NB) rice varieties, and also, a method for determining adulteration of Basmati rice with other rice varieties and a kit thereof. Abstract The present invention relates to set of inter-simple sequence repeats (ISSR)-PCR primers of SEQ ID Nos. 1 to 37 for genotyping eukaryotes and a method of genotyping diverse genomes of plant and animal systems using FISSR-PCR primers and SSR markers more particularly, a FISSR and SSR method of distinguishing Basmati rice varieties from Non-Basmati (NB) rice varieties and Traditional Basmati (TB) rice varieties from Evolved Basmati (NB) rice varieties, and also a method for determining adulteration of Basmati rice with other rice varieties and a kit thereof</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Field of the present invention<br>
The present invention relates to a set of inter-simple sequence repeats (ISSR)-PCR primers of Slid ID Nos. 1 to 37 for genotyping cukaryoles and a method of genotyping diverse genomes of plant and animal systems using FISSR-PCR primers and SSR markers; more particularly, a it’s and SSR method of distinguishing Basmati rice varieties from Non-Basmati (NB) rice varieties, and Traditional Basmati ('fB) rice varieties from Evolved Basmati (NB) rice varieties, and also  a method for determining adulteration of Basmati rice with other rice varieties and a kit thereof<br>
Background and prior art reference of the present invention<br>
Rice is the staple ibid for more than half of the world's population. Its rich genetic diversity in the form of thousands of land races and progenitor species besides its economic significance have aroused unending interest among scientists for several decades. In the evolution of rice and its genetic differentiation into distinct varietal groups, consumer quality preferences have played a significant role besides agro-ecological factors. One such varietal group comprising of aromatic pulao/biryani rice of Indian subcontinent known as 'Basmati' is the highly priced rice in the domestic as well as international markets. Originated in the foothills of the Himalayas Basmati rice is characterized by extra long slender grain, pleasant and distinct aroma and soft and Huffy texture of cooked rice. These unique features of Basmati said to be the culmination of centuries of selection and colligation by fanners, arc well preserved and maintained in their purest form in the traditional Basmati ('f B) varieties.<br>
■["he historical and areheological findings infer that the varieties with such unique morphological and quality attributes are not present in traditional rice growing areas anywhere in the world'. A number of undesirable traits of Basmati such as tall stature, low yield, sensitivity to photopcriod and poor response to fertilizer application prompted breeders to develop 'elite' Basmati varieties by making use of the high yielding semi-dwarf<br><br>
non-Basmati (NB) rice varieties. Such 'elite' evolved lines of Basmati (EB), however, fall short of the quality features of traditional varieties. Difficulty in recovering desirable recombinants from crosses involving NB and TB varieties, and reversion often to parental types in the backcross generations, suggest that probably indica and Basmati types are phylogenetically divergent^. A study on Asian rice varieties using isozyme markers clustered Basmati varieties in the group V gene pool, which is well separated from groups I and VI comprising of indica and japonica types, respectively^. Further evidence of high degree of divergence of Basmati from other indica varieties comes from high percentage of hybrid sterility'. The difficulties experienced in evolving 'elite' Basmati varieties combining all the desirable traits of TB and NB varieries have retained the preeminent status of TB varieties in the rice industry.<br>
Consequently, TB varieties command considerable price advantage in the market over the EB varieties. The adulteration of TB grains with EB and NB grains is reported to be common and thus hampering the Basmati rice export market. Hence, identifying the genuine Basmati variety from the other Basmati-like non-Basmati varieties is considered to be important from the viewpoint of trade.<br>
Traditionally employed morphological and chemical parameters have not been found to be discriminative enough warranting more precise techniques. Several molecular techniques are available for detecting genetic differences within and among cultivars^'^. Among these, Simple Sequence Repeat (SSR) markers are efficient and cost effective and detect a significantly higher degree of polymorphism in rice^''. They are ideal for genetic diversity studies and intensive genetic mapping'^"''. An altemafive method to SSRs, called Inter-SSR-PCR'^ has also been used to fingerprint the rice varieties'^.<br>
The well-characterized Basmati rice specific molecular markers could serve as marker tags for Basmati varieties. If the markers are shown to be tightly linked to any of the distinct traits of Basmafi they could be used in marker assisted selection (MAS) programs. Such markers could be fiirther verified on the fully sequenced rice genome with regard to their location and linkage to the gene(s) of interest.<br><br>
Recent progress in DNA marker technology, particularly PCR based markers, such as randomly amplified polymorphic DNA markers (RAPD),^- ''* amplified fragment length polymorphisms (AFLP),^"'^ and microsatellite markers^""^^ have augmented the marker resources for genetic analyses of a wide variety of genomes. As PCR technology finds increased use in various genetic analyses, additional novel variations of this technique are emerging in order to augment the high-resolution genotyping and genetic mapping of various complex animal, plant and microbial genomes. The PCR analysis using anchored simple sequence repeat primers, referred to as ISSR-PCR or anchored SSR-PCR, has gained attention recently as an attractive means of characterizing complex genomes.'^"'^'^ The ISSR-PCR approach employs oligonucleotides based on simple sequence repeats (SSR) anchored either at the 5' or 3' end with two or four purine or pyrimidine residues, to initiate PCR amplification of genomic segments flanked by inversely oriented, closely spaced microsatellite repeats. ^<br>
The ISSR-PCR strategy is especially attractive because it avoids the need to carry out costly cloning and sequencing inherent in the original microsatellite-based approach. As a result, ISSR-PCR has been profitably used for genetic linkage analysis of various plant species^'"^' and the silkworm, Bombyx mori?^'^^<br>
The most commonly used approach for generating ISSR-PCR markers is either 5'-end labeling of ISSR primers with y[^^P] ATP or one of the a[^^P] labeled dNTPs is added to the PCR reaction along with cold dNTPs in appropriate ratio, followed by resolution of PCR products on PAGE and autoradiographic detection of iSSR markers. Alternatively, some investigators have also resolved ISSR-PCR products on Nusieve agarose gels, of course with a marked reduction in number of markers compared to PAGE. While the former involves stringent standardization and extensive use of radioactive isotopes, the latter compromises with the number of markers generated per PCR reaction. Besides, both the methods require higher quantity {&gt; 10 ng) of template DNA per PCR reaction. These feamres prove to be disadvantageous in high resolution genetic mapping experiments where a large number of markers are analyzed using a single mapping population and genetic analysis of infectious organisms and parasites where DNA yield per sample may be too low for convenfional PCR assays.<br><br>
In the present study, the inventors have automated the ISSR-PCR marker assay to enhance<br>
genetic  informativeness  and used it  along with  rice  SSRs  to  analyze  the  genetic<br>
relationships of traditional and evolved Basmati and non-Basmati varieties.<br>
Further, the inventors have designed and disclosed novel ISSR-PCR primers which have<br>
shown more resolving power than the known ISSR-PCR primers.<br>
Objects of the present invention<br>
The main object of the present invention is to develop a set of inter-simple sequence<br>
repeats (ISSR)-PCR primers for genotyping eukaryotes.<br>
Another main object of the present invention is to develop a method of genotyping diverse<br>
genomes of plant and animal systems using FISSR-PCR primers.<br>
Yet another object of the present invention is to develop a FISSR method of distinguishing<br>
Basmati rice varieties from Non-Basmati (NB) rice varieties.<br>
Still another object of the present invention is to develop a FISSR method of distinguishing<br>
Traditional Basmati (TB) rice varieties from Evolved Basmati (NB) rice varieties.<br>
Still another object of the present invention is to develop a SSR method of distinguishing<br>
Basmati rice varieties from Non-Basmati (NB) rice varieties.<br>
Still another object of the present invention is to develop a SSR method of distinguishing<br>
Traditional Basmati (TB) rice varieties from Evolved Basmati (NB) rice varieties.<br>
Still another object of the present invention relates to develop a method of using the novel<br>
primers and markers in rice breeding to develop rice varieties of desired characteristics.<br>
Still another main object of the present invention is to develop a method of genotyping<br>
animal systems including silkworm.<br>
Still another object of the present invention is to develop a method of using the novel<br>
primers and markers to identify lineage of rice varieties.<br>
Still another object of the present invention is to develop newer rice varieties of both<br>
Basmati and non-Basmati type.<br>
Still another object of the present invention is to identify monomorphic, polymorphic, and<br>
diverse nature of the rice varieties using the said markeis.<br><br>
Still another object of the present invention is to develop a method of using simple<br>
sequence repeat (SSR) loci and corresponding 70 SSR alleles along with the said primers<br>
and markers to develop rice varieties of desired characteristics.<br>
Another main object of the present invention is to determine adulteration of Basmati rice<br>
varieties with other kinds of rice varieties.<br>
Yet another main object of the present invention is to develop a kit using FISSR-PCR<br>
primers to determine adulteration of Basmati rice varieties with other kinds of rice<br>
varieties.<br>
Yet another main object of the present invention is to develop a kit using SSR markers to<br>
determine adulteration of Basmati rice varieties with other kinds of rice varieties.<br>
Statement of the present invention<br>
The present invention relates to set of inter-simple sequence repeats (ISSR)-PCR primers<br>
of SEQ ID Nos. I to 37 for genotyping eukaryotes and a method of genotyping diverse<br>
genomes of plant and animal systems using FISSR-PCR primers and SSR markers; more<br>
particularly, a FISSR and SSR method of distinguishing Basmati rice varieties from Non-<br>
Basmati (NB) rice varieties, and Traditional Basmati (TB) rice varieties from Evolved<br>
Basmati (NB) rice varieties, and also, a method for determining adulteration of Basmati<br>
rice with other rice varieties and a kit thereof<br>
Detailed description of the present invention<br>
Accordingly, the present invention relates to set of inter-simple sequence repeats (ISSR)-<br>
PCR primers of SEQ ID Nos.  1 to 37 for genotyping eukaryotes and a method of<br>
genotyping diverse genomes of plant and animal systems using FISSR-PCR primers; more<br>
particularly, a FISSR method of distinguishing Basmati rice varieties trom Non-Basmati<br>
(NB) rice varieties, and Traditional Basmati (TB) rice varieties from Evolved Basmati<br>
(NB) rice varieties.<br>
SEQ ID NO. 1. GATGCTGATACACACACACACACA SEQ ID NO. 2 GCACATGCAGTGTGTGTGTGTGTG SEQ ID NO. 3 CATGCACATTGTGTGTGTGTGTGT SEQ ID NO. 4 GCTAGTGCTCACACACACACACAC<br><br>
SEQ ID NO. 5 CGTATGTGTGTGTGTGTGTGTGT<br>
SEQ ID NO. 6 TGTAATGAGAGAGAGAGAGAGA<br>
SEQ ID NO. 7 GACGATACGAGAGAGAGAGAGA<br>
SEQ ID NO. 8 CCCGGGATTATTATTATT<br>
SEQ ID NO. 9 AAATACAGCAGCAGCAG<br>
SEQ ID NO. 10 GTGCTAATAATAATAAT<br>
SEQ ID NO. 11 AATTTATTATTATTATT<br>
SEQ ID NO. 12 GAGTCATTATTATTATT<br>
SEQ ID NO. 13 AGCGAATTATTATTATT<br>
SEQ ID NO. 14 TAAAAAATAATAATAAT<br>
SEQ ID NO. 15 ACAAAAATAATAATAAT<br>
SEQ ID NO. 16 AGTGAATTATTATTATT<br>
SEQ ID NO. 17 GTGATATTATTATTATT<br>
SEQ ID NO. 18 TGAGCGCCGCCGCCGCC<br>
SEQ ID NO. 19 TCGATACCACCACCACCACCACCACCA<br>
SEQ ID NO. 20 ATAGAGCTGCTGCTGCTGCTGCT<br>
SEQ ID NO. 21 AATCGAAAGAAGAAGAAG<br>
SEQ ID NO. 22 CATAATAAGAAGAAGAAG<br>
SEQ ID NO. 23 ATCGAATAATAATAATAATAAT<br>
SEQ ID NO. 24 GCATATGATGATGATG<br>
SEQ ID NO. 25 (A/T)T(G/C)GACAGACAGACAGACA<br>
SEQ ID NO. 26 GTGTGTGTGTGTGTGTATCC<br>
SEQ ID NO. 27 GAGAGAGAGAGAGAGACGG<br>
SEQ ID NO. 28 AAGAAGAAGAAGAACTA<br>
SEQ ID NO. 29 AAGAAGAAGAAGTACGA<br>
SEQ ID NO. 30 CTTCTTCTTCTTATGCT<br>
SEQ ID NO. 31 GGCGGCGGCGGCGCTAA<br>
SEQ ID NO. 32 ATGATGATGATGGACT<br>
SEQ ID NO. 33 AAACAAACAAACATC<br>
SEQ ID NO. 34 CACACACACACACAATGCACAGC<br><br>
SEQ ID NO. 35 GAGAGAGAGAGAGAACTAT SEQ ID NO. 36 GCCGCCGCCGCCGCACTC SEQ ID NO. 37 AACAACAACAACGT<br>
In an embodiment of the present invention, wherein a set of inter-simple sequence repeats<br>
(ISSR)-PCR primers of SEQ ID Nos. 1 to 37 for genotyping eukaryotes.<br>
In another embodiment of the present invention, wherein primers of SEQ ID No. 1 to 25<br>
are 5' anchored primers.<br>
In yet another embodiment of the present invention, wherein primers of SEQ ID Nos, 26 to<br>
and 37 are 3' anchored primers.<br>
In another embodiment of the present invention, wherein a method of genotyping diverse<br>
genomes of plant and animal systems using the said FISSR-PCR primers, said method<br>
comprising steps of:<br>
(a), extracting DNA from said systems,<br>
(b). conducting a polymerase chain reaction (PCR) using extracted DNA, the said primers,<br>
and a flourescent label,<br>
(c). obtaining a plurality of flourescent amplified products,<br>
(d). separating the amplified products to produce fingerprint pattern using conventional<br>
techniques,<br>
(e). identifying Monomorphism (M), and Polymorphism (?) amplified products, and<br>
(f). genotyping the genomes of the said systems based on the polymorphic amplified<br>
fragments.<br>
In an embodiment of the present invention, wherein the flourescent label can be selected<br>
from a group comprising Tamara dye, R6G, and Rl 10.<br>
In yet another embodiment of the present invention, wherein a FISSR method of<br>
distinguishing Basmati rice varieties from Non-Basmati (NB) rice varieties, using primers<br>
ofSEQIDNos. 1-5,7, U, 19, 20, 25,26, and 27, said method comprising steps of:<br>
(a), extracting DNA from said rice varieties,<br>
(b). conducting a polymerase chain reaction (PCR) using extracted DNA, the said primers,<br>
and a flourescent label,<br><br>
(c). obtaining a plurality of flourescent amplified products,<br>
(d). separating the amplified products to produce fingerprint pattern using conventional<br>
techniques,<br>
(e). identifying Monomorphism (M), and Polymorphism (P) amplified products, and<br>
(f). distinguishing Basmati rice varieties from Non-Basmati (NB) rice varieties based on<br>
the polymorphic amplified Augments<br>
In still another embodiment of the present invention, wherein the flourescent label can be<br>
selected from a group comprising Tamara dye, R6G, and RI10.<br>
In   another   embodiment   of the   present   invention,   wherein   a   FISSR   method   of<br>
distinguishing Traditional Basmati (TB) rice varieties from Evolved Basmati (NB) rice<br>
variefies, using primers of SEQ ID Nos. 1-5, 7, 11, 19, 20, 25, 26, and 27, said method<br>
comprising steps of<br>
(a), extracting DNA from said rice varieties,<br>
(b), conducting a polymerase chain reaction (PCR) using extracted DNA, the said primers,<br>
and a flourescent label,<br>
(c). obtaining a plurality of flourescent amplified products,<br>
(d). separating the amplified products to produce fingerprint pattern using conventional<br>
techniques,<br>
(e). identifying Monomorphism (M), and Polymorphism (P) amplified products, and<br>
(f). distinguishing Traditional Basmati (TB) rice varieties from Evolved Basmati (EB) rice<br>
varieties based on the polymorphic amplified fragments<br>
In yet another embodiment of the present invention, wherein the flourescent label can be<br>
selected from a group comprising Tamara dye, R6G, and R110.<br>
In still an embodiment of the present invention, wherein the average number of bands<br>
produced by the primers with different repeat motifs negatively correlated with the number<br>
of nucleotides in the repeat unit of the motif.<br>
In still another embodiment of the present invention, wherein the number of products<br>
amplified in different repeat length classes reflect the frequency of different repeat motifs<br>
distributed in the rice genome.<br><br>
In another main embodiment of the present invention, wherein a method of genotyping<br>
diverse genomes of plant and animal systems using SSR-PCR markers of table 3, said<br>
method comprising steps of:<br>
(a), extracting DNA from said systems,<br>
(b). conducting a polymerase chain reaction (PCR) using extracted DNA, the said primers,<br>
and a flourescent label,<br>
(c). obtaining a plurality of flourescent amplified products,<br>
(d). separating the amplified products to produce fingerprint pattern using conventional<br>
techniques,<br>
(e). identifying Monomorphism (M), and Polymorphism (?) amplified products, and<br>
(f). genotyping the genomes of the said systems based on the polymorphic amplified<br>
fragments.<br>
In yet another embodiment of the present invention, wherein the flourescent label can be<br>
selected from a group comprising Tamara dye, R6G, and Rl 10.<br>
In still  another embodiment  of the  present  invention,  wherein  a  SSR  method of<br>
distinguishing Basmati rice varieties from Non-Basmati (NB) rice varieties, using markers<br>
of table 4, said method comprising steps of:<br>
(a), extracting DNA from said rice varieties,<br>
(b). conducting a polymerase chain reaction (PCR) using extracted DNA, the said primers,<br>
and a flourescent label,<br>
(c). obtaining a plurality of ampUfied products,<br>
(d). separating the amplified products to produce fingerprint pattern using conventional<br>
techniques,<br>
(e). identifying Monomorphism (M), and Polymorphism (?) amplified products, and<br>
(f). distinguishing Basmati rice varieties from Non-Basmati (NB) rice varieties based on<br>
the polymorphic amplified fragments<br>
In still another embodiment of the present invention, wherein the flourescent label can be<br>
selected from a group comprising Tamara dye, R6G, and Rl 10.<br><br>
-f<br>
111  still  another  embodiment of the present  invention,  wherein  a  SSR  method  of<br>
distinguishing Traditional Basmati (TB) rice varieties from Evolved Basmati (NB) rice<br>
varieties, using markers of Table 5, said method comprising steps of:<br>
(a), extracting DNA from said rice varieties,<br>
(b). conducting a polymerase chain reaction (PCR) using extracted DNA, the said primers,<br>
and a flourescent label,<br>
(c). obtainii\g a plurality of flourescent ampUfled products,<br>
(d). separating the amplified products to produce fingerprint pattern using conventional<br>
techniques,<br>
(e). identifying Monomoiphism (M), and Polymorphism (P) amplified products, and<br>
(f). distinguishing Traditional Basmati (TB) rice varieties from Evolved Basmati (EB) rice<br>
varieties based on the polymorphic amplified fragments<br>
In still another embodiment of the present invention, wherein the flourescent label can be<br>
selected from a group comprising Tamara dye, R6G, and Rl 10.<br>
In another main embodiment of the present invention, wherein a kit for determining<br>
adulteration of Basmati rice with other rice varieties, said kit comprising<br>
(a), at least one ISSR-PCR primers from a set of primers of SEQ ID Nos. 1 to 37, and/or<br>
(b). at least one SSR markers from a set of markers of Table-4.<br>
In yet  another main embodiment of the present  invention,  wherein  a method for<br>
determining adulteration of Basmati rice with other rice varieties (Fig 4) using at least one<br>
ISSR-PCR primers from a set of primers of SEQ ID Nos. 1 to 37, and/or at least one SSR<br>
markers from a set of markers of Table-4, said method comprising steps of<br>
(a), extracting DNA from various rice varieties,<br>
(b).  conducting a polymerase chain reaction (PCR) using extracted DNA, the said<br>
primer(s) or marker(s), and a flourescent label,<br>
(c). obtaining a plurality of flourescent amplified products,<br>
(d). separating the amplified products to produce fingerprint pattern using conventional<br>
techniques,<br>
(e). identifying Monomorphism (M), and Polymorphism (P) amplified products, and<br><br>
1<br>
(i). determining adulteration in Basraati rice varieties with other rice varieties based on the<br>
polymorphic amphfied fragments.<br>
In still another embodiment of the present invention, wherein accordingly, the present invention relates to 37 inter-SSR-PCR primers and corresponding 481 inter-SSR-PCR markers useful in revealing genetic relationship in Basmati and non-Basmati rice varieties and a method of using the said primers, markers, and selected simple sequence repeat (SSR) loci and corresponding SSR alleles in rice breeding, for developing rice varieties of desired characteristics and also, a method of using the said primers, markers, SSR loci and SSR alleles in identifying genetic lineage of rice varieties.<br>
In still another embodiment of the present invention, wherein the recently developed Inter-Simple Sequence Repeat PCR (ISSR-PCR) or microsatellite primed FCR or Simple Sequence Repeat (SSR)-Anchored PCR technique detects polymorphic markers in a wide variety of genomes. Usually the ISSR primers are either 5' end-labeled with y[''^P]ATP or one of the a[^^P] labeled dNTPs is added to the PCR reaction and the PCR products are resolved on PAGE and autoradiographed. Alternatively, cold PCR products are resolved on agarose gel electrophoresis. In the present study, we show that informativity, sensitivity and speed of the ISSR-PCR can be substantially enhanced by adding fluorescent nucleotide in the PCR reaction followed by resolution of PCR products on an ABI 377 automated sequencer. The informativeness, measured as a number of detectable amplified fragments, was two-fold higher and the quantity of required template DNA is two-fold lower than the regular ISSR-PCR. We have termed this method as FISSR-PCR and show its usefulness in generating large number of species and varietal specific markers in plants, insects, parasites of insects and human and various infectious organisms. Further, we show that the FISSR markers are inherited and segregated in Mendelian fashion as demonstrated on a panel of 99 F2 offspring derived from a cross of two divergent silkworm strains. The FISSR-PCR marker assay could be a method of choice for large scale screening of varieties/cultivers and highthroughput genotyping in mapping of genomes where microsatellite information is scanty or absent.<br><br>
In still another embodiment of the present invention, wherein the objective of the present study was to make use of the efficient molecular marker systems to reveal genetic relationships in traditional Basmati (TB) and evolved Basmati (EB) and semi-dwarf non-Basmati (NB) rice varieties. A subset of three rice groups was analyzed using 19 SSR loci and 37 (hiter simple Sequence Repeat) ISSR-PCR primers. A total of 70 SSR (Simple Sequence Repeat) alleles and 481 ISSR-PCR markers were revealed in 24 varieties from the three groups. The lowest genetic diversity was observed among the traditional Basmati varieties whereas the evolved Basmati varieties showed the highest genetic diversity by both the marker assays. The results indicated that the subset of aromatic rice varieties analyzed in the present study is probably derived from a single land race. The traditional Basmati and semi-dwarf non-Basmati rice varieties used in the present study were clearly delineated by both the marker assays. A number of markers, which could unambiguously distinguish the traditional Basmati varieties used in the present study from the evolved ones and non-Basmati rice varieties, were identified. The potential use of these markers in Basmati rice breeding program and authentication of traditional Basmati varieties used in the present study is envisaged.<br>
In still another embodiment of the present invention, wherein the certified Basmati rice materials used in this study were provided by the Ministry of Commerce, Govt, of India and non-Basmati varieties by the Directorate of Rice Research, Hyderabad, India. The details of the rice varieties are given in Table 1 as shown here below.<br>
Table 1<br>
List of rice varieties (Oryza sativa) included in the study<br><br>
Name	Origin<br>
Traditional Basmati (TB)	<br>
Basmati 217	Punjab (Indian sub- continent)<br>
Basmati 370	Punjab (Indian sub- continent)<br>
Dehraduni (Type~3)	Uttar Pradesh<br>
Ranbir Basmati	Jammu &amp; Kashmir<br><br>
Tarori(HBC-19)	Haryana<br>
Basraati 386	India<br>
Evolved Basmati (EB)	<br>
Basmati 385 (385)	TN-1               X Basmati 370<br>
Super Basmati (SB)	Basraati 320    xIR661<br>
Pusa Basmati (PB)	Pusa 150         X Karaal local<br>
Kasturi (Kas)	CK 88-17-1-5 X Basmati 370<br>
Haryana Basmati (HB)	Sona                X Basmati 370<br>
Mahi Sugantha (MS)	BK 79             X Basmati 370<br>
Haryana Gaiirav (HG)	Mutant of Basmati 370<br>
Super (SU)	Equivalent to SB<br>
Terricot (TER)	NA<br>
Sharbati (SHA)	NA<br>
CSR 30B (CSR)	Buraratha 4-10 x Pak Basmati<br>
9<br>
Semi-dwarf non Basmati	<br>
(NB)	<br>
IRS	Pesa                 X Dee-geoe-<br>
Jaya	woo-gen<br>
Taichung (N) 1 (TC)	TNI                 xT141<br>
IR22	NA<br>
IR20	IR 8                 X Tadukan<br>
IRBB5	IR 262             X TKM 6<br>
PR 106 (PR)	IR24               X DZ192<br>
	IR8 xPeta5   x Bella patna<br>
Varietal abbreviations are indicated in the parentheses<br>
In still another embodiment of the present invention, wherein the DNA was extracted from 5 g of grains from each of the varieties using Phytopure plant DNA extraction kit (Pharmacia Amersham Biotech).<br><br>
hi still another embodiment of the present invention, wherein DNA markers and laboratory assay. Two classes of markers were employed in the present study: fluorescence based ISSR-PCR and SSRs.<br>
ISSR-PCR, The ISSR-PCR method (15) was modified with a view to enhance the speed and sensitivity of detection of markers. We designed and synthesized 12 5' and 3' anchored primers (Please refer Table 2 here below).<br>
Table 2<br>
List of ISSR primers, marker information and diversity in three rice groups<br><br>
Primers	Sequences	Mol. Wl Range [bp)	Total No<br>
of markers				No. of markers			<br><br><br><br><br>
		TB			EB			NB<br><br><br><br><br>
	M	P	D	M	P	D	M	P	D<br>
5' Bnchorrd												<br>
C (GA),	SEQlDNO.l	150-1200	50	31	H	0.18	10	40	0.S6	15	22	U.4i<br>
R (CA)7	SEQ ID NO, 1	150- 1500	45	20	9	0.27	14	27	0.51	24	18	0 23<br>
T(CAh	SEQ ID NO. 4	180-1200	38	17	6	0.16	6	30	0.68	16	9	0,21<br>
T (GT),	SEQ ID NO. 5	200 - 1500	61	29	11	0.18	6	53	').67	12	21	0 42<br>
R (TG),	SEQ ID NO. 2	160-1400	42	23	6	0.15	14	27	0.49	14	16	029<br>
Y(TG)7	SEQ ID NO. 3	200 - 1050	33	14	5	0.18	7	26	0,63	11	!3	0.34<br>
T3(ATT).	SEQ ID NO. 11	250-1500	50	3	32	0.60	I	47	0 72	2	22	0.65<br>
EtA(GCT)6	SEQ ID NO, 20	180-760	30	20	4	0,07	11	18	0.37	13	7	0.24<br>
Y (ACC)7	SEQ ID NO. 19	200 - 960	33	19	6	0.17	9	22	0.51	13	7	0.19<br>
tGACA)4	SEQ ID NO. 25	315-1400	IS	7	1	0.10	4	14	0.S9	S	1(1	060<br>
3' anchored												<br>
(GA),C	SEQ ID NO. 27	150-1150	49	28	10	0.17	18	30	0.45	20	13	0.23<br>
(O'DiR	SEQ ID NO. 26	140-   960	32	14	g	0.27	4	26	0.63	10	8	0.28<br>
Total			481			0,23± 0.13			0,57± 0.10			0.3 ±0.15<br>
M: Monomorphic ; P; Polymorphic; D: Diversity<br>
In still another embodiment of the present invention, wherein amplification was performed in 10 mM Tris-HCl, pH 8.3 (50 mM KCl; 1.5 mM MgCh; 0.01% gelatin; 0.01% Triton X-100), 1 mM dNTPs, 0.2 |iM Fluorescent dUTP (TAMARA, Perkin Elmer), 0.3 unit of AmpUTaq Gold (Perkin Elmer), 4 ^M primer with 5 ng of genomic DNA per 5 |il reaction.<br><br>
T<br>
The thermal cycling conditions were as follows: initial denaturation of 10 min at 94°C, 35 cycles of 30 s at 94°C, 30 s at 5D°C and 1 min at 72^C, final extension of 10 min at 72°C. The PCR was performed on a Perkin Elmer thermal cycler (9600). One ^1 of PCR product was mixed with 1.5 fjl of 6 x loading buffer (1:4 mixture of loading buffer and formamide, Sigma Chemicals Co.) and 0.4 ^1 of GENESCAN-1000 ROX labelled molecular weight standard (red fluorescence) was included in the loading samples. The samples were denamred at 92°C for 1 min prior to loading onto an ABI 377 automated sequencer (Applied Biosystems) and electrophoresed on 5% polyacrylamide gel (Long ranger, FMC) under denaturing conditions containing 7 M urea, in 1 x TBE buffer (90 mM Tris borate, pH 8.3 and 2 mM EDTA). Three replicate experiments were carried out to verify the reproducibility of markers.<br>
In still another embodiment of the present invention, wherein Selection of primers and SSR survey. We selected 19 primer pairs (Please refer Table 3 here below) from the list of 351 rice microsatellite loci displayed on the Cornell imiversity RiceGenes website ('http://ars-genQme.comell.edu/rice/) for analysis.<br><br>
Table 3<br>
Microsalellite marker information, allele distribution and diversity in three rice groups<br><br>
Locus	Repeal motif	Total No. of Alleles	Allele Size range (bp)				No	, of alleles			<br><br><br><br><br>
		TB			EB			NB	<br><br><br><br><br>
	M	P	D	M	P	D	M	P	D<br>
RM224	tGA),j	4	130-155	1	1	0,49	0	3	0,74	0	4	0.91<br>
RM 16	(GAli,	J	16S - 225	1	0	0,00	0	2	0,95	0	2	0,83<br>
RM 13	(GASi,	6	13S-158	0	2	0.62	0	4	0,91	0	4	0.86<br>
RM252	(GAJi,	8	195-255	0	4	0.86	0	4	0,92	0	3	0.82<br>
RM23S	IGA),,	3	102-138	0	2	0.62	0	2	0 74	0	3	0.82<br>
RM234	(GA.J,,	2	148-150	I	0	0.00	0	2	0.75	1	0	0,00<br>
RM225	(GAh^	4	no- 180	1	0	0.00	0	4	0.88	1	0	0.00<br>
RM !	(,
RM3I0	tGT),.^	5	85 - no	0	2	0.37	0	4	0,92	0	3	0,88<br>
RM302	CGT)^AT)i	4	131,-205	1	0	0.00	0	4	0,93	0	2	0,62<br>
RM 160	(GAAhi	2	100-105	I	0	0.00	0	2	0,71	I	0	0,00<br>
RM330	(CAT),	5	160-225	3	0	0.00	1	3	0,42	i	3	0,26<br>
R.M72	(TAT),C(ATT]|.	3	150-175	0	2	0.62	0	3	0.89	0	2	0.82<br>
RMI02	(G0C),(CG)6	3	43 S-445	2	0	0.00	0	3	0,58	1	2	0.42<br>
RM 171	(GATG),	3	325 - 346	0	2	0,49	0	2	0.77	0	2	0,55<br>
RM 163	(GGAGA]4 (GA)MC(GA),M	4	130-175	1	0	0,00	0	3	0,86	0	4	0.89<br>
RM 161	(AG),„	3	163-180	1	1	0.49	0	2	0,74	1	0	0.00<br>
RMI36	(AOG),	3	100-124	1	0	0.00	0	2	0,71	1	1	0.49<br>
RM238	NA	2	130-150	1	0	0.00	0	2	0.85	1	0	0,00<br>
Total												<br>
No, of		70		16	16	0,23	1	54	0.79	8	37	0,30<br>
alleles						± 0.30			± 0.13			+ 0,36<br>
M: Manomarphic ; P: Polymorphic; D; Diversity<br>
hi still another embodiment of the present invention, wherein the SSR primers are obtained<br>
from the databank. The details of the Accession numbers are as given below.<br>
1.	RM 171 -D84275<br>
2.	RM161-D41873<br>
3.	RM72   -AF344086<br><br>
4.	RM 13 -AF344014<br>
5.	RM238A-AF 344058<br>
6.	RM 16--AF 344016<br>
7.	RM 330-AF 344154<br>
8.	RM 302-AF 344127<br>
9.	RM 224 - AF 344045<br>
10.	RM 252-AF 344072 n.RM 234-AF 344054<br><br>
12.	RM 223-AF 344044<br>
13.	RM 102-D 17586<br>
14.	RM136-RM136<br>
In still another embodiment of the present invention, wherein the primers for the selected loci were synthesized by Research Genetics (HuntsviUe, Ala). Wherever possible at least two loci with non-overlapping alleles were multiplexed in the PCR reaction to increase the efficiency of gene typing.<br>
In still another embodiment of the present invention, wherein PCR amplification was performed in a 5 |il volume containing 10 mM Tris-HCl (pH 8.3), 50 mM KCl, 1.5 mM MgCl2, 0.25 unit of AmpliTaq Gold (Perkin Elmer), 50 pM of dNTPs, 0.2 pM fluorescent dUTP (TAMARA or Rl 10 or R6G, Perkin Elmer), 0.35 nM of each primer with 5 ng of genomic DNA on a Themal Cycler PTC 100 (MJ Research hic). The basic PCR program used to amplify the SSR DNA was as described in the RiceGenes website. The sample preparation, loading and electrophoretic conditions were as described under ISSR-PCR. When the allelic polymorphisms of any of the microsatellite loci revealed &gt; 20 bp difference between the varieries, such loci were resolved on 3.5% MetaPhor agarose gels (FMC).<br>
In still another embodiment of the present invention, wherein evaluation of polymorphisms and data analysis. Polymorphic products from the SSR and ISSR analyses were scored qualitatively for presence (+) or absence (-). The proportion of bands that were shared between any of the two varieties screened averaged over loci (SSRs) and primers (ISSR-<br><br>
PCR) were used as the measure of similarity. The genetic diversity^ was calculated as follows:<br>
n<br>
PlCi=l-SP^jj<br>
ri<br>
where Pij is the frequency of the ^'^ allele for marker i and the summation extends over n alleles. The calculation was based on the number of alleles / locus in SSR, and number of bands / primer in case of ISSR.<br>
In still another embodiment of the present invention, wherein Cluster analysis was based on distance matrices using the unweighted pair group method analysis (UPGMA) program in WINBOOT software . The relationships between varieties were represented graphically in the form of dendrograms.<br>
In still another embodiment of the present invention, wherein it has been shown that by using DNA markers identified in the present study, the detection of adulteration of hybrid Basmati with Pure Basmati could be detected upto the level of 1% adulterant in the mixture (Fig 4),<br>
Brief description of the accompanying drawings<br>
Fig 1 shows Fluorescence based ISSR profiles of TB, EB and NB rice varieties for three representative primers: a) SEQ ID NO. 5; b) SEQ ID NO. 3; c) SEQ ID NO. 26. Arrows and arrowheads indicate the markers, which differentiate Basmati and non-Basmati rice varieties respectively.<br>
Fig 2 shows Fluorescence based SSR polymorphisms detected by 8 representative SSR loci in TB, EB and NB rice varieties. Arrows and arrowheads indicate the markers, which differentiate Basmati and non-Basmati rice varieties respectively. The asterisk indicates the duplication of locus 330 only in NB and some of the EB varieties. The first 7 loci are<br><br>
GENESCAN images and the last locus (RM 234) is 3.5% Metaphor agarose run ethidium<br>
bromide stained image.<br>
Fig. 3 shows Dendrogram derived from a XJPGMA cluster analysis using Nei and Li<br>
Coefficients based on (a) ISSR markers; (b) SSR markers. Numbers on the nodes indicate<br>
the number of times a pariicular branch was recorded per 100 bootstrap replications<br>
following 1000 replications.<br>
Fig. 4 shows the method for determining adulteration of Basmati rice with other rice<br>
varieties.<br>
Fig. 5 (a-c). Comparison of regular ISSR-PCR and FISSR-PCR assays using the primer 5'<br>
CRT RT(GT)9 3' (i.e. SEQ. ID. NO.5) in eight crop species on three electrophoretic<br>
systems   showing   reduced   sensitivity   on   Nusieve   agarose   (a);   intermediate   on<br>
polyacrylamide sequencing gel with radiolabehng (b), and high sensitivity in FISSR-PCR<br>
products resolved on an ABI377 sequencer (c).<br>
Fig. 6 shows The Mendelian segregation of FISSR markers in silkworm. The FISSR<br>
markers were generated using a primer 5' RAY RAT RC(GA)7 3' on two parental strains,<br>
P^" and C log, and their F| and Fi offspring. The arrows and arrowheads indicate markers<br>
specific to P^" and Cios, respectively.<br>
Fig. 7 shows that the ISSR-PCR markers can be used to identily the varieties. Here, Lane 1 represents an<br>
elite breed of tomato. Lanes 2 and 3 represents tomato breeds sold in the name of the elite breed profiled<br>
in Lane 1.<br>
Fig. 8 shows FISSR-PCR profiles of different clones of Casuarina. Here, A stands for Allocasurina.<br>
Fig. 9 shows the DNA profiles of various micro-organisms using FISSR primer.<br>
Fig.Ifl shows the DNA profiles of different species of silkmoths using FISSR primer<br><br>
Table - 4<br>
A sample of FlSSR markers that distinguish Basmati and Non-Basraati rice varieties<br>
{see Fig. 1)<br><br>
Primers	Size(b	Basma	Non-<br>
	P)	ti	Basmati<br>
CGTAT(GT)9	377	4-	-<br>
	55Q	+	-<br>
	558	+	-<br>
	571	-	+<br>
	743	+	-<br>
CATGCACAT(T	208	-	+<br>
G)7T	262	+	-<br>
	326	-	+<br>
	345	+	-<br>
	378	4-	-<br>
	448	-	+<br>
	480	+	-<br>
	611	+	-<br>
	908	+	-<br>
(GDsATCC	436	+	-<br>
	497	+	-<br>
	790	-	+<br><br>
Table 5<br>
A sample of FISSR markers that distinguish traditional and evolved Basmati rice varieties<br><br>
Primers	Size(bp)	Pure Basmati	Evolved Varieties<br>
CGTAT(GT)9	1200	+	- (except CSR, SU)<br>
	730	+	- (except SU)<br>
	750	+	- (except SU, SB)<br>
	550	+	- (except CSR, SU, PB, 385)<br>
CATGCACAT(TG)7T	908	+	- (except CSR, SU, PB, 385)<br>
	378	+	- (except CSR, SB)<br>
	326	+	- (except CSR, SU, SB, PB, 385)<br>
	262	+	- (except CSR, SB, SH, HB)<br>
	208	-	+(Except CSR, SU, 385, MS, KAS)<br>
{GT)sATCC	436	+	- (except CSR, SU, SB, PB, 385)<br>
	443	-	+(except CSR, SU, SB, PB, 385)<br>
	497	+	-(exceptCSR, SU, SB,385)<br>
	790	-	+(exceptCSR, SU, SB,385)<br>
	830	+	- (except CSR, SU, SB, PB, 385)<br>
In still another embodiment of the present invention, wherein evaluation of ISSR-PCR markers. The fluorescence based ISSR-PCR markers could be clearly resolved on an ABI automated sequencing gels. All the 12 anchored SSR motifs employed in the present smdy produced varying number of DNA fragments in different size range (Table 2, Fig.l). The average number of bands produced by ISSR primers with different repeat motifs was negatively correlated with the number of nucleotides in the repeat unit of the motif For example, among the 5' anchored primers, the dinucleotide-based primers produced more number of bands (43.8 ± 9.82) than tri- (37.6 ± 7.89) and tetranucleotide (32) primers. Although we have not used similar number of primers in each repeat class of primers for<br><br>
such a comparison, the number of products amplified in different repeat length class reflected the frequency of different repeat motifs distributed in the rice genome. The 12 primers produced a total of 481 PCR products, of which 389 (80.9%) were polymorphic in all the 24 rice varieties. There appeared to be no conelation between the number of bands amplified and the degree of polymorphisms. For example, the primers R(TG)7 and YfTG)? generated 42 and 33 bands respectively, out of which 73.8 % and 78.8% were polymorphic, on the other hand 98% of the 50 bands amplified by T3(ATT)4 primer were polymorphic. Among the 12 primers, two 5' anchored, T3(ATT)4 and T(GT)9 and one 3' anchored primer, (GT)sR generated more than 90% scorable polymorphisms.<br>
In still another embodiment of the present invention, wherein since our efforts were directed towards comparative genetic analysis of TB, EB and NB rice varieties, we evaluated the polymorphisms in these three groups. The degree of polymorphisms differed substantially among the three groups. The TB varieties showed very low level of polymorphism as compared to EB and NB varieties. Out of 340 bands scored in the TB varieties, only 115 (33.8%) were polymorphic (Table 2). On the other hand, 360 {77.4%) out of 465 bands were polymorphic in the EB and 166 (51.7%)) out of 321 bands were polymorphic in NB varieties. The primer T3(ATT)d was found to produce the most polymorphic bands in all the three rice groups: TB (91.4%), EB (92%) and NB (91.66) whereas RA(GCT)6 generated the least degree of polymorphisms (Table 2).<br>
In still another embodiment of the present invention, wherein the genetic diversity was calculated for each of the ISSR primers in all the three rice groups (Table 2). The difference in diversity values was as striking as the degree of polymorphism. The two primers, T3(ATT)4 and RA(GCT)6 which recorded the highest and the lowest polymorphisms respectively, in the three groups also recorded the highest and the lowest diversity values. The 12 primers had an average diversity value of 0.23 ± 0.132 with a range of 0.07 to 0.6 for TB. The diversity values ranged from 0.37 to 0.72 averaging 0.57 ± 0.10 for the EB, and 0.19 to 0.65 with a mean of 0.34 ± 0.15 for the NB varieties. The difference was significant ( p 
 <br>
&gt;<br>
groups). Out of the 12 ISSR-PCR primers, 5 revealed 12 PCR products, a combination of which could distinguish Basmati from NB varieties (data not shown). On the other hand 21 specific PCR products generated by 9 ISSR-PCR primers could distinguish EB from NB varieties (Fig. la to Ic). By making use of 2 to 3 informative primers the TB from the EB and NB varieties could be unambiguously distinguished. For example, by using two primers SEQ ID NO. 5, and SEQ ID NO. 26 (Fig la-and Ic) one can make a decision whether a given variety is a TB or an EB rice variety.<br>
In still another embodiment of the present invention, wherein evaluation of SSR polymorphisms. We also used 19 microsatellite loci for the genetic analysis of the three rice groups. Table 3 summarizes the total number of alleles detected and their size range across 24 rice varieties for each of the 19 microsatellite loci used. The number of alleles ranged from 2 to 8 with an average of 3.8 alleles. Only RM 252 detected a maximum of 8 alleles. As could be seen from Table 3 there appears to be no correlation between the number of alleles detected and the number of SSR repeats in the SSR loci. For example, the microsatellite loci containing the (GA) repeat motifs varying from (GA)i5 to (GA)25 did not show any correlation with the number of alleles they revealed. The allele number varied between the three rice groups (Table 3). More number of alleles were resolved in the EB (56) and NB (42) as compared to TB (28) varieties. Out of 19 loci, only 8 revealed polymorphisms in the TB varieties whereas 14 were polymorphic in the NB varieties and all the loci were polymorphic in the EB varieties (Table 3). The diversity values also varied from one locus to another and between the three rice groups (Table 3). An average diversity of 0.23 ± 0.3 was observed for the TB group whereas EB and NB rice groups recorded much higher diversity values of 0.79 ± 0.13 and 0.50 ± 0.36, respectively. However, the extent of variation in average diversity is not as large as the difference in the number of alleles per locus and it does not appear to correlate with the number of alleles. Across the three rice groups, the highest diversity of 0.87 was observed for RM 252 with 8 alleles whereas RM 72 and RM 16 with 3 alleles each displayed diversity of 0.70 and 0.76, respectively. The difference in diversity between the 3 groups was statistically significant (p 
 <br>
In still another embodiment of the present invention, wherein we scored the SSR alleles which showed preponderance in Basmati varieties. Out of 70 alleles, 9 were found only in TB and in some of the EB varieties. These alleles were absent in all the 7 NB varieties analyzed in the present study (Please refer Table 6 here below)<br>
Table 6<br>
SSR loci that distinguish Basmati and semi dwarf rice varieties<br><br>
LOCUS					<br>
	Fonvard primer	Reverse primer	Allele<br>
Size<br>
(bp)	Basmati varieties	Seitii dwarf varieties<br>
RM163	ATCCATGTGCGCCTTTATGAGGA	CGCTACCTCCTTCACTTACTAGT	162	._	+ (only PR, TC(N)1<br>
RMi71	AACGCGAGGACACGTACTTAC	ACGAGATACGTACGCCTTTG	438	+	-<br>
RM16I	TGCAGATGAGAAGCGGCGCCTC	TGTGTCATCAGACOGCGCTCCG	180	-t	-<br>
RM72	CCGGCGATAAAACAATGAG	GCATCGGTCCTAACTAAGGG	175	+	-<br>
RMI	GCGAAAACACAATGCAAAAA	GCGTTGGTTGGACCTGAC	85	+	--<br>
R.M15	TCCAACATGGCAAGAGAGAG	GGTGGCATTCGATTCCAG	158	--	■t- (only PRl<br>
RM 238 A	GATGGAAAGCACGTGCACTA	ACAOGCAATCCGTAGACTCQ	445	+	-<br>
RM 16	CGCTAOGGCAGCATCTAAAA	AACACAGCAGGTACGCGC	225 165 220	4-	+ (oni&gt; JA)<br>
RM330	CAATGAAGTOGATCTCGOAG	CATCAATCAGCGAAGGTCC	<br>
	+	-<br>
"	"	"	160	--	+- (except 1R8. PR)<br>
RM302	TCATGTCATCTACCATCACAC	ATGGAGAAGATGGAATACTTGC	140	+	--<br>
RM224	ATCGATCGATCTTCACGAGG	TOCTATAAAAGGCATTCGGO	148	--	+ (only PR)<br>
RM252	TTCGCTGACGTGATAGGTTG	ATGACTTGATCCCGAGAACG	255	-	+ (only 1R20)<br>
■'	■■	"	228	-	■&gt;- (only TC(N))<br>
RM234	ACAGTATCCAAGGCCCTGG	CACGTGAGACAAAGACGGAG	148	+	-<br>
RM223	GAGTGAGCTTGGGCTGAAAC	GAAGGCAAOTCTTGQCACyO	200	+	-<br>
In still another embodiment of the present invention, wherein further analysis of seven more NB varieties confirmed that these 9 alleles are confined only to the TB varieties analyzed in the present study (data not shown). Out of 19 loci, one locus RM 330 was found to be duplicated only in NB varieties (except PR 106) and none of the TB varieties showed any such duphcation for this locus. None of the 19 SSR loci could distinguish the TB from the EB varieties independently. However, a combination of the polymorphic loci<br><br>
with different Basmati specific alleles enabled discrimination of the traditional fi-om the evolved ones except CSR 30B. For example, RM 171 locus in combination with RM 238 or RM 16 or RM 302 could discriminate all the traditional ones from the evolved ones (except CSR 30B) (Please refer Table 7 here below).<br>
Table 7<br>
SSR loci that distinguish pure Basmati and evolved Basmati rice varieties<br><br>
LOCUS	Forward primK	Reverse pnmer	Alklt Sizs |bp)	Pure Baamali	E^'olved flasmalL vancries<br>
RM163	ATCC ATGTGCGCCTTT ATG AOO \	CGCTACCTCCTTCACTTACTAGT	140		-■ (esccpt !8i, SB. KARJ<br>
RMlTl	\ACGCGfcGG AC ftCGT ACTTAC	ACGAGAT ACGT ACGCCTTTG	M5	.-	■t tatepi i&amp;'.,TtB.)<br>
"			438	+-	■■   (Bitcpt   3BS CSR )<br>
BM   71	CCGOCG AT A AAftC A ATG AG	GCATCGGTCCT AACTAAGGO	:7s	-f	- (cKccpi 3ai,*a(<br>
RM102	AACTTTCCCACCACCACCGCGO	AOC AGC AOC AAGCC AGC A AGCG	140	*■	- (eicer&gt;[ KAH, PB, SB)<br>
RM 23S A	GATGGAAAOCACGTGCACTA	AC AGGC AATCCGT AO ACTCG	■MJ	-I-	- (cicepl PB, SB)<br>
RM   16	CGCTAGGGCAGCATCTAAAA	A AC AC AGC AGGT ACGCGC	I6i	1-	■- (citspl  SB, KB|<br>
RMi02	TC ATGTC ATCTACC ATC AC AC	ATGGAOAAGATGOAATACTTGC	140	-t	- (sicept KAR, PB. SB)<br>
RMBJQA	CAATGAAOTGGATCTCGGAO	CATC A ATC AGCGAAGGTCC	ISO	-	+ (evcepl 38!,PB, SB, KAR)<br>
RM 136	GAOAGCTC AGCTGCTOC CTCT AGC	GAOGAGCGCCACGGTGTACGCC	124	-	■ (cicepl :ia^ KAR, SB)<br>
In still another embodiment of the present invention, wherein the ISSR-PCR and SSR profiles were used to determine the genetic similarity matrices, which were then used to construct dendrograms. Both the methods separated the TB from the NB varieties (Fig. 3). The EB varieties got clustered in between depending upon the degree of genetic similarity to the two groups. In ISSR-PCR analysis, all the TB varieties except Basmati 217, displayed 95% similarity among themselves. Genetic similarity estimates obtained by SSR analysis also revealed similar results. Both the marker assays included the land race Basmati 217 along with the EB varieties. As revealed by ISSR analysis, among the EB varieties, CSR, SB, 385, PB and SU showed higher genetic similarity to the TB varieties as compared to the other EB varieties which showed higher similarity to NB varieties (Fig. 2).<br>
In still another embodiment of the present invention, wherein with SSR markers (Fig. 2) the evolved Basmati have clustered aloi^ with Non-Basmati cultivars. Thus, ISSR primers are better than SSR markers in separating Evolved Basmati fi-om Non-Basmati.<br><br>
In still anodier embodiment of the present invention, wherein it is demonstrate that the sensitivity, speed and informativeness of the existing ISSR-PCR method could be enhanced substantially by using fluorescent dye labeled nucleotide in the ISSR-PCR reaction, which we call FISSR-PCR, followed by separation of PCR products on an ABI automated sequencer 377, using diverse species of plants, and parasitic organisms (Fig 5 and Fig 9).<br>
In still another embodiment of the present invention, wherein Plant genomic DNA was<br>
la<br>
extracted as described by Dellaporta et al. Briefly, the seeds were ground in 200 \x\ of grinding / lysis buffer (50 mM Tris-HCI pH 7.2, 50 mM EDTA pH 8.0, 3% SDS, 1% p-mercaptoethanol). The DNA was precipitated with 0.1 final volume of 3.0 M sodium acetate pH 5,2 and 2.5 volume of absolute ethanol, incubated at -VO^C. DNA was quantified on 0.8% agarose gel and diluted to a uniform concentration (10 ng/|il). Genomic DNA from silk moths was isolated either from the posterior silk glands collected from day 3 fifth instai larvae or from the whole pupae.'^ Briefly, silk glands or whole pupae were ground in liquid nitrogen using a pestle and mortar. Extraction buffer (100 mM Tris-HCI, pH 8.0, 50 mM NaCI, 50 mM EDTA and 1 % SDS) was added to the ground tissue and incubated at 37°C for 2hrs with occasional swirling. The DNA was extracted twice with phenol-choloroform-isoamyl alcohol (25:24:1) and once with chloroform. The supernatant DNA was ethanol precipitated, resuspended in TE (10 mM Tris-HCI, 1 mM EDTA, pH 8.0) buffer and incubated at 37°C for Ih after addition of RNase A (100 yig/ml).<br>
In still another embodiment of the present invention, wherein we designed and synthesized in-house six 5'-anchored primers SEQ ID NO. 1, SEQ ID N0.3, SEQ ID NO. 4, SEQ ID NO. 5, SEQ ID NO. 6, and SEQ ID NO. 7 and two 3'-anchored primers of SEQ ID NO. 26 and SEQ ID NO. 27 on an ABI DNA synthesizer, purified by PAGE and then desalted through an oligonucleotide purification cartridge (OPC).<br><br>
In still another embodiment of the present invention, wherein Eight diiTerent plant species of Wheat, Rice, Maize, Lentil, Chickpea, CowPea, Pigeon Pea, and Pea and two to three varieties from each of the plant species were used for FISSR analysis (Fig. 5).<br>
In still another embodiment of the present invention, wherein for FISSR-PCR assay, amplification was performed in 10 mM Tris-HCl, pH 8.3 / 50 mM KCl / 2.5 mM MgCb / 0.01% gelatin / 0.01% Triton X-100 / 1 mM dNTPs / 0.4 (iM Fluorescent dUTP (TAMARA Perkin Elmer) / 0.25 unit of Taq DNA polymerase (Ampli Taq Gold, Perkin Elmer), 4 ^M primer with 5 ng of genomic DNA per 5 |al reaction. The thermal cycling conditions were as follows: initial denaturation of 10 min at 94*'C, 35 cycles of 30s at 94°C, 30s at 50°C and 1 min at 72°C, fmal extension 10 min at ll^C. The PCR was performed on a DNA thermal cycler (Perkin Elmer 9600). One |^I of PCR products was mixed with 1.5 fil of 6 X loading buffer (1:4 mixture of loading buffer and formamide (Sigma) and 0.4 (il of Gene scan-1000 (ROX) size was included in the loading samples.<br>
In still another embodiment of the present invention, wherein the samples were denatured at 92°C for 1 min prior to loading on to an ABI 377 automated sequencer and electrophoresed on 5% polyacrylamide gel (Long ranger, FMC) under denaturing conditions containing 7 M urea, in IX TBE buffer (90 niM Tris borate, pH 8.3 and 2 mM EDTA). We also analyzed ISSR-PCR products on agarose gel electrophoresis. (Fig -5)<br>
In still another embodiment of the present invention, wherein to compare the efficacy of the FISSR-PCR assay with the regular ISSR-PCR assay, the ISSR-PCR reactions were carried out. The PCR and thermal cycling conditions used were as in FISSR-PCR assay except 20 ng template DNA and 8 (iM primer without fluorescent dUTP were used in 10 |il reaction. The PCR products were electrophoresed on 2% agarose (50% Sigma + 50% Nusieve) for 3 hrs at lOOV (Fig - 5). The ISSR-PCR reactions were also carried out as previously reported^^ in the presence of a[^^P] labeled dCTP and cold dCTP (added in the ration of 1:4) in the PCR reaction. The PCR products were denatured at 75"C for 2 min,<br><br>
chilled on ice and electrophoresed on a standard sequencing gel (5% acrylamide, 7M urea, Ix TBE) and run at 900 V under constant power supply for 15 hrs (Fig- 5).<br>
In still another embodiment of the present invention, wherein each of the 8 SSR-anchored primers revealed distinct PCR profiles in agarose, PAGE (PCR products incorporated with a[^'P] labeled nucleotide) and ABI 377 automated sequencer (PCR products incorporated with fluorescent nucleotide) in all the 8 plant species with characteristic species and varietal specific profiles as shown in Fig. 5. However, the number of markers revealed in each of the methods differed substantially. The FISSR assay was found to be the most informative in all the species studied. It resolved almost two-fold more number of markers as compared to the existing ISSR-PCR methods based on agarose and PAGE analyses. As a result, the number of species and varietal specific markers available for scoring increased substantially (Fig. 3, and Table 8). For example, the primer SEQ ID NO. 5 in FISSR assay resolved 45 scorable bands in rice out of which as many as 14 markers were specific to rice as compared with the other seven plant species. Ten markers could distinguish all the 3 rice varieties.<br>
In still another embodiment of the present invention, wherein since FISSR-PCR technique is very sensitive it is a method of choice for detecting polymorphic markers in closely related varieties/populations which are otherwise difficult to discriminate by using other marker systems. The inter-varietal polymorphisms among the self pollinated and most conserved legume varieties suggest the efficiency and usefulness of this assay for varietal and cultivar identification. The FISSR-PCR assay could be successfiilly used as a rapid, sensitive and informative technique to quickly fingerprint a large number of genotypes of a given species in a cost-effective manner. Since FISSR-PCR assay is automated, a single assay including analysis could be completed within a day.<br>
In still another embodiment of the present invention, wherein the method is also very useful to quickly evaluate the abundance of microsatellite repeats in different genomes using anchored primers with different microsatellite motifs. For example, (GTV(CA)n is<br><br>
more abundant in wheat and rice genomes than other genomes studied.  Such information is crucial in the projects aimed at microsatellite marker development^"<br>
Table 8<br>
Comparison of FISSR-PCR assay using SEQ ID NO. 5 with regular ISSR-PCR based on agarose gel and PAGE analysis<br><br>
		Total	Molecula<br>
Crop	Assay	number of	r size<br>
		bands	(bp)<br>
	Agarose ISSR-PCR	18(06,02)	300-1900<br>
Wheat	PAGE ISSR-PCR	22 (05,03)	290-1200<br>
	FISSR-PCR	47 (14,03)	250-1200<br>
	Agarose ISSR-PCR	21 (06,04)	250-1700<br>
Rice	PAGE ISSR-PCR	24 (05,03)	260-1100<br>
	FISSR-PCR	45(14,10)	250-1200<br>
	Agarose ISSR-PCR	14 (08,03)	300-1000<br>
Maize	PAGE ISSR-PCR	21 (09,02)	270-1200<br>
	FISSR-PCR	32(12,05)	260-1100<br>
	Agarose ISSR-PCR	13(04,01)	300-1000<br>
Lentil	PAGE ISSR-PCR	23 (07,02)	280-1200<br>
	FISSR-PCR	32 (14,02)	250-1200<br>
	Agarose ISSR-PCR	14 (08,02)	350-1200<br>
Chick Pea	PAGE ISSR-PCR	16 (08,03)	250-1200<br>
	FISSR-PCR	22 (10,00)	250-1200<br><br>
	Agarose ISSR-PCR	08 (04,02)	350-0900<br>
Cow Pea	PAGE ISSR-PCR	10 (05,03)	250-1200<br>
	FISSR-PCR	19(08,03)	250-1000<br>
	Agarose ISSR-PCR	12 (04,02)	400-1700<br>
Pigeon Pea	PAGE ISSR-PCR	17 (08,02)	260-1200<br>
	FISSR-PCR	33 (16,02)	250-1200<br>
	Agarose ISSR-PCR	09 (05,-)	270-0900<br>
Pea	PAGE ISSR-PCR	20(12,-)	250-1000<br>
	FISSR-PCR	26 (08,-)	290-1000<br>
Figures in the parentheses indicate species and varietal specific markers, respectively.<br>
In still another embodiment of the present invention, wherein we also used this technique successfully to amplify the diverse genomes of insects, parasites of insects and human such as Plasmodium, Leishmania and Brugia inalayi and many infectious organisms such as Vibrio cholerae Mycobacterium tuberculosis, Helicobacterium pylori, Pseudomonas aeruginosa and other organisms (Fig 9) where DNA yield per sample may be too low to be useful for conventional PCR-based marker assays. In all the cases, very clear, easily<br><br>
scorable and highly reproducible FISSR markers could be resolved depending on the type of core microsateiiite repeat included in the anchored ISSR primers.<br>
hi still another embodiment of the present invention, wherein An F2 mapping population of 99 offspring developed by sib-mating of Fi hybrid silk moths derived from a cross of two divergent strains of B. mori, ?^^ x C,og was used to demonstrate the inheritance and segregation of FISSR markers and thus showing FISSR-PCR is a robust method for high resolution mapping of complex genome (Fig. 6 and Fig 10)<br>
In still another embodiment of the present invention, wherein to demonstrate the utility of FISSR-PCR markers in genetic mapping experiments, we analyzed 99 F3 offspring of silkworm, B. mori derived from a cross of two divergent silkworm strains, P and Cios using the markers amplified by the primer, SEQ ID NO. 7. As could be seen from Fig. 6 and Table 9, the 12 markers which were polymorphic between the two parental strains inherited and segregated according to Mendelian principle. These results show that the FISSR-PCR markers are very useful for quick and high throughput genotyping of mapping population using very small quantity of template DNA. For constructing high density linkage maps, a large number of F2 individuals need to be analyzed using a number of molecular markers and the template DNA requirement in the case of other conventional PCR methods is relatively high. The FISSR assay provides a large number of DNA markers per primer and allows detection of markers with as little as 2-5 ng of template DNA, on an automated sequencer obviating the necessity for using radioactive isotopes.<br>
Table 9<br>
Mendelian segregation of FISSR-PCR markers amplified by RAY RAT RC(GA)7 in 99 F2 offspring derived fi-om P^" and Ciog cross in silkworm.<br><br><br>
Polymorphic					<br>
markers (bp)	No. of F2 offspring scored	Expected Ratio	Observed Ratio	X2	P&gt;<br>
p3U	Ciog	<br><br><br><br><br><br>
	940	99	74.25 :24.75	70:29	0.973	0.300<br>
	850	99	74.25 ; 24.75	70:29	0.973	0.30O<br>
835		99	74.25 : 24.75	67:32	2.876	0.100<br>
	575	99	74.25 :24.75	69:30	1.485	0.200<br>
560		99	74.25 :24.75	75:24	7.600	0.006<br>
	500	99	74.25 :24.75	70:29	0.973	0.300<br>
420		99	74,25 ; 24.75	74:25	10,943	0.001<br>
	390	99	74.25 : 24.75	78 -. 21	0.379	0.500<br>
340		99	74.25 : 24.75	77 -. 22	0.406	0.500<br>
	335	99	74.25 : 24.75	72 :27	0.372	0.500<br>
330		99	74.25 : 24.75	75 -. 24	7.600	0.005<br>
260		99	74.25 : 24.75	73:26	0.084	0.800<br>
hi still another embodiment of the present invention, wherein diverse Casuarina species were analyzed using FISSR PCR method. (Fig. 8)<br>
In still another embodiment of the present invention, wherein in conclusion, we show that this rapid, less hazardous, simple and informative assay could be used in large scale screening of varieties/inbred lines, high resolution genetic mapping of complex genomes and quick genetic analysis of large sample sizes of various infectious organisms which yield very little quantity of DNA, with high degree of accuracy.<br>
In still another embodiment of the present invention, wherein the traditional and evolved Basmati varieties included in the present study probably represent a major component of the Basmati gene pool of the Indian sub-continent. In addition to the Basmati varieties, we<br><br>
^<br><br>
also included in our study many semi-dwarf non-Basmati varieties, some of which have been utilized for development of EB varieties. We used two molecular marker assays, fluorescent based ISSR-PCR and SSR of which the former is an improvised version of ISSR-PCR method developed earlier '^<br>
hi still another embodiment of the present invention, wherein the number of bands produced across 24 rice varieties by different anchored SSR motifs is consistent with the published reports on microsatellite frequency in the rice genome. Among dinucleotide repeats, (GA)n and (CA),! are the most abundant in the rice genome^'"''. Both the repeat classes were amenable to fluorescent-based ISSR-PCR analysis of the rice genome as both were equally polymorphic. Out of the two 3' anchored primers, one with (GA)n and another with (GT)n motif, the (GA)n produced more number of bands probably due to its greater abundance in the rice genome as reported earher^''. The 2 (GT)n based primers, one anchored at the 5' and the other at the 3' end amplified 61 and 31 bands respectively. Although comparison of only two primers may not allow us to make definitive conclusions, taken together the earlier inferences'^ such a difference may be due to the lack of selective nucleotides at the 3' end of the 5' anchored primers. On the other hand 5' and 3' anchored (GA),! repeats did not reveal any such difference probably because 5' anchored primers impose selection for long stretch of SSRs, while amplification with 3' anchored primer would not impose selection for repeat length. Since (GA)n motifs are reported to be longer in the rice genome as compared to (GT)n motifs, both 5' and 3' anchored (GA)n primers produce similar number of bands. On the other hand the lack of length advantage in (GT)n motif probably results in difference between 5' and 3' end anchored (GT)n primers.<br>
In still another embodiment of the present invention, wherein all the (ATT)n and (GACA)4 anchored primers amplified a large number of bands in the present study in contrast to earlier smdies'^ Although (GACAV repeats are fewer than the di- and trinucleotide repeats they appear to be in good number in the rice genome as reported earlier^^. Since the lengths of tri- and tetranucleotide repeats in the rice genome are mostly 5 to 8 and 5 to 6<br><br>
respectively, the longer repeat motifs used by earlier studies'^ would have precluded the amplification. Our results indicate that tri- and tetranucleotide bsed ISSR-PCR markers could provide potential markers in the rice genome.<br>
hi still another embodiment of the present invention, wherein there has been wide range of interests in the genetic differences between TB, EB and NB rice varieties. The information available on genetic diversity and differences of the three groups is scanty except for the Asian rice varieties using isozyme markers^. The results of the present study using fluorescent-based ISSR and SSR markers indicate that TB varieties have the least diversity as compared to the EB and NB varieties. Besides, both the marker assays showed that there are no significant differences between the TB varieties. We believe that the high degree of genetic similarity among the TB varieties indicate that they are possibly the descendents of a single land race and the minor genetic variation is maintained as a result of selection and preference imposed by farmers for several years. This observation draws support from the historical relationship of Basmati varieties used in the present smdy. Most of the TB varieties classified under different names are likely to have been selected from the local variety such as Basmati 370 released for commercial cultivation in 1933 at the Rice Research Station, Kalashah Kaku (now in Pakistan). For example, the isozyme patterns of 60 out of 65 Pakistani accessions described as Basmati matched the isozyme pattern of Basmati 370 and Type 3. Similarly, of the nine varieties from India all except Kamal local were identical to the isozyme pattern of Basmati 370 and Type 3 (36). Among the 19 (33) and 5 (34) EB varieties released since 1965 for cultivation in hidia and Pakistan respectively, 12 and 4 had Basmati 370 as one of the donor parents. Our studies along with these reports suggest that the TB varieties used in the present study could be considered as bulk of the narrow TB gene pool of the hidian sub-continent. Recent report on RAPD profiling of aromatic rices also shows low level of genetic diversity (35). The variety, Basmati 217 which we received as land race showed only 75% and 66% similarities whereas, CSR 30B which was received as EB showed 82% and 96% similarities based on ISSR and SSR marker assays respectively, to the five TB varieties. Considering the genetic diversity in the other EB varieties it is unlikely that CSR 30B evolved from a direct cross<br><br>
;tween a Pakistani Basmati variety and the salt tolerant non-Basmati variety, Buraratha, has attained such high level of genetic similarity to the TB varieties. However, further insights into its grain quality vis-a vis TB and EB varieties and the pedigree details may be required for ascertaining its status. A reasonable explanation for the higher genetic distance of Basmati 270 from the other TB varieties is that it has differentiated from the rest of the TB varieties and probably represents a separate lineage. This is supported by the observation that out of the 70 SSR alleles among 24 varieties, 3 were unique to Basmati 217. The high level of genetic diversity and preponderance of NB alleles in most of the EB varieties indicate that most of them still retain a large genomic fraction of the NB varieties used in the breeding program.<br>
In still another embodiment of the present invention, wherein both the marker assays clearly differentiate the Basmati and non-Basmati varieties as highlighted by isozyme and RFLP studies . The high level of genetic differentiation of Basmati and NB rice varieties suggests that the fonner might have possibly diverged a long time ago from NB varieties through conscious selection and patronage. The duplication event at locus RM 330 only in semi-dwarf non-Basmati varieties including the old japonica varieties, Taipai and Wu lOB supports the divergence of aromatic varieties from the common ancestor, It would be interesting to study the other varieties in Group V, which embraces most of the long grain varietes (3), for duplication event in this locus. The high level genetic differentiation of the two groups may be the possible reason for lack of finding desirable recombinants and introgression of useftil genes in Basmati breeding programmes (2). This is reflected by the preponderance of NB alleles in most of the EB varieties. Some of the bands/alleles unique to the TB varieties used in the present study are not at all found in any of the EB varieties either because of incompatible chromosome regions (coadapted gene complex) or they are possibly linked to the negative traits of Basmati and are thus selected against. On the other hand, alleles from the NB varieties have survived in greater number in the EB varieties possibly because they are in close proximity to the genes, which confer usefLiI traits of NB varieties.<br><br>
hi still another embodiment of the present invention, wherein the high resolution of fluorescence based ISSR-PCR assay described in the present study provides a large number of highly reproducible markers using as little as 5 ng template DNA per PCR reaction without using radioactive isotopes or chemicals involved in silver staining. By varying SSR motifs and their anchors a large number of markers could be generated which can be used for further saturation of the rice genome. We believe that since bulk of the global Basmati rice trade and breeding programs center around the TB varieties analyzed in the present study, the combination of SSR and ISSR markers can be used to identify these varieties from both NB as well as EB varieties. This will also enable the determination of adulteration of this set of traditional Basmati varieties. It is hazardous to venture to the conclusion that the markers observed in the present study could be universally applied to all aromatic rice varieties. Further studies on geographically random samples of aromatic rice varieties would probably give a more complete and less discontinuous picture with regard to the allelic/marker association. Nevertheless, the markers specific to the TB varieties used in the present study should be fiirther pursued to look for allelic association, if any, with the Basmati phenotype. Such a study would provide markers which would help to eliminate unnecessary chromosome regions in the early stages of backcrossing, thus helping the breeders to shorien the breeding cycles by rapid incorporation of Basmati traits into breeding lines.<br>
In still another embodiment of the present invention, wherein recent progress in DNA marker technology, particularly PCR based markers have augmented the molecular marker resources for the genetic analysis of a wide variety of genomes. As PCR technology finds increased use in various genetic analyses, additional novel variations of this technique are emerging. PCR analysis using anchored simple sequence repeat primers has gained attention recently as an attractive means of characterizing genomes. In the present invention we have developed a technique which we have termed as FISSR-PCR (for fluorescent-inter-simple-sequence-PCR) which makes use of the fluorescent dye labeled nucleotide in the PCR reaction, followed by separation of PCR products on an ABI automated sequencer 377, using template DMA from diverse species of plants and their varieties and animals. We show that the FISSR-PCR polymorphic markers could be unambiguously scored to provide varietal and species specific<br><br>
molecular profiles. Besides, the FISSR-PCR markers could be followed in the segregating population such as second filial generation (F2) thus showing their applicability in high resolution mapping of complex genomes.<br>
In still another embodiment of the present invention, wherein the studies using anchored primer based PCR have shown that the polymorphisms could be detected in a variety of complex genomes. The applicant has extended the study to address the issue of automating the method by using fluorescent oligonucleotides in the PCR reaction and a variety of combinations of 5' and 3' anchored microsatellite primers and resolution of the PCR products on an automated sequencer. This would increase the sensitivity and speed of the assay by several manifolds thus providing an easy handle to the geneticists for high-resolution maping of complex genomes and identification and authentication of varieties and species using as little as 2 to 5 ng DNA per assay.<br>
In still another embodiment of the present invention, wherein we assayed eight plant species (2 varieties from each plant species except rice and pea which involved three and one sample respectively eg. Casuarina), using as many as 36 anchored primers designed by us. We genotyped 100 offspring of second filial hybrid offspring along with their first filial hybrid offspring and their parental strains of silkworm. In all the cases DNA was extracted from liquid nitrogen frozen tissue samples stored at -WC. High molecular weight genomic DNA was extracted from all the plant and animal samples, quantified by spectrophotometer as well as by agarose gel electrophoresis and used as PCR templates. Each of the different 5' and 3' anchored microsatellite primers (Table 1) was tested on each of the template DNA samples.<br>
In still another embodiment of the present invention, wherein the PCR reactions were earned out in a 5 microliter reaction volume carrying 5 ng of genomic DNA, 25 ^M each of dCTP, dGTP, dTT and dATP, 0.8 iM anchored primer, and 0.4 |iM fluorescent dUTP {Tamara dye, Perkin Elmer) using Taq gold DNA polymerase (Perkin Elmer). Amplification was performed on a termal cycler (Model 2400, from Perkin Elmer) with a programme of initial denaturation at 94°C for 10 minutes followed by 35 cycles of 94°C for I minute, 50°C for 1 minute and 72'C for 2 minutes followed by final extension at 72°C for 10 minutes, and finally stored at 4'C.<br><br>
In still another embodiment of the present invention, wherein the products were run on 5% polyacrylamide gel with 7M urea, on an ABI automated sequencer 377 at a constant voltage of 3000 volts for 7 hours. The data were analysed using Genescan analysis software. Under optimized reaction conditions, the replicate experiments were carried out and only consistently reproducible bands were scored for genotyplng varieties, species and clones.<br>
In another main embodiment of the present invention, wherein SSR markers are used to detect the adulteration of BasmatJ rice varieties (Fig. 4), For example, basraari 370, a traditional popular Basmali (TB) variety is adultered with an Evolved Basmati (EB) variety, Haryana Basmati at different levels i.e., 70:30 (TB:EB), 75:25 {TB:EB), 80:20 (TB:EB), 85:15 (TB:EB), 90:10 (TB:EB), 95:5 (TB:EB), 99:1 (TB:EB). The DNA extracted for the adultered sample was analyzed using RM 234 and RM 330 primers in independent experiments. The peaks generated scanning the intensity of the alleles, reveal that aduherationofTraditional Basmati by even 1% of Evolved Basmati could be detected in a sample.<br>
In another main embodiment of the present invention, wherein DNA profiles of different species of silk moths is shown (Fig. 10) using FISSR primers. The silk moths nssd for profiling comprises Bombyx mori, Bombyx mandarina, Antheraea roylei, Antheraea proylei, Antheraea pemyl, Antheraea mylitta, Antheraea yamamai, Philosamia cynthia ricini, and Antheraea assama.<br>
In still another embodiment of the present invention, wherein said experiment (Fig 10) further establishes that the FISSR-PCR primers can be used for genotyping eukaryotes including silk moths.<br>
In yet another embodiment of the present invention, wherein the spurious varieties are sorted out from the elite verities (Fig. 7) using ISSR markers.<br><br><br>
We Claim:<br>
1.	A sel of inter-simple sequence repeal (ISSR)-PCR primers as set torch in SHQ ID Nos. 1 lo 37 for genotyping.<br>
2.	The set of primers as claimed in claim 1. wherein Ice primers as set forth in SI-'Q ID Nos. 1 lo 25 arc 5' anchored primers.<br>
3.	The set of primers as claimed in claim 1, wherein the primers as sel forth in Sl ID Nos. 26 to 37 are 3' anchored primers <br>
4.	A  method  of genotyping diverse  genomes  using   IKSR-PCR   primers,   said  method comprising steps of:<br><br>
(a)	amplifying DNA from the genomes using an ISSR PCR primer selected from a group comprising ST.Q ID Nos. 1 lo 37 and a lluoreseently labeled nucleotide:<br>
(b)	separating fluorescent amplified products;<br>
(c)	identifying monomorphic and polymorphic amplified products; and<br>
(d)	genotyping the genomes based on the polymorphic amplified produces,<br><br>
5.	ice method as claimed in claim 4, wherein the lluoresccnt label is selected from a group comprising Tamara dye. R6G and Rl 10.<br>
6.	Ice method as claimed in claim 4. wherein the genomes are selected from Plant and animal systems.<br>
7.	A method of distinguishing Basmati rice variclies from Non- (NB) rice varieties, said method comprising steps of:<br>
(a)	amplifying DNA from Basmati and Non-Basmati rice varieties using DNA<br>
extracted from each variety, an ISSR primer and a fluorcseently labeled nucleotide,<br>
wherein said ISSR primer of Slid ID Nos. 1-5. 7. 1   19. 20, 25, 26, and 27:<br>
(b)	separating fluorescent amplified products;<br>
(e)	identifying polymorphic amplified products, wherein said polymorphic amplillcd<br>
produces arc poiymoi-phic between Basmati rice varieties and Non-Basmati (NB) rice<br>
varieties; and<br>
(d) distinguishing Basmati rice varieties from Non-Basmati (NB) rice varieties based on (he presence and/or absence of said polymorphic amplillcd products.<br><br>
8.	A method as claimed in claim 7. wherein the fluorescent label is selected from a group compris'ing Tamara dye. R6G and R) 10.<br>
9.	A   meihod of distinguishing  Traditional   Basmati  (TB)  rice  varieties   from   livolvcd Basmati (Hli) rice varieties, said method comprising steps of:<br><br>
(a)	amplifying ONA from Traditional Basmati {'fB) and Involved Basmati rice varieties using DNA extracted from each variety, an [SSR primer, and a fluorescently labeled nucleotide wherein said ISSR primer of Sl-Q ID Nos. 1-5, 7, 11. 19. 20, 25. 26. and 27;<br>
(b)	separating fluorescent amplified products;<br>
(c)	identifying polymorphic amplified products, wherein said polymorphic amplified products are polymorphic between IVaditional Basmati (TB) rice variefies and F.voived Basmati (NB) rice varieties; and<br>
(d)dislinguishing 'fradifional Basmati (TB ) rice varieties from livolved Basmati (l-^B) rice varieties based on the presence and/or absence of said polymorphic amplified products.<br>
10.	The method as claimed in claim 9. wherein the fiuorcsccnl label is selected from a group comprising 'lamara dye. R6G and Rl 10.<br>
11.	The meihod as claimed in claim 9, wherein the genomes arc selected from Plant and animal systems.<br>
12.	A   method  of genotyping   diverse   genomes  using  SSR-PCR   primers,   said   method comprising steps of:<br><br>
(a)	amplifying DNA from the genomes using an SSR-PCR primer selected from Table 3, and a fluorescently labeled nucleotide;<br>
(b)	separating fluorescent amplified products;<br>
(c)	identifying Monomorphism (M) and Polymorphism (P) amplified products: and<br>
(d)	genotyping the genomes based on the polymorphic amplified fragments.<br>
13- fhe method as claimed in elaim 12, wherein the flourescent label is selected from a group<br>
comprising 'famara dye. R6Ci and Rl 10. 14. 'fhe method as claimed in claim 12, wherein the genomes are plant and animal systems.<br><br>
15.	A method of di,sling(jish!ny Basmati rice varieties from Nnn-Iiasma!i (NB) rice varieties,<br>
said method comprising steps of:<br>
{a) amplifying DNA from Basmati and Non-Basmati rice varieties using DNA extracted from each variety, an SSR primer directed to a locus selected from table 6 and a lluorescently labeled nucleotide;<br>
(b)	separating lluoresccnt amplified products;<br>
(c)	identifying polymorphic amplified products, wherein said polymorphic amphlied products are polymorphic between YiasmaU rice varieties and Non-Basmali (NB) rice varieties: and<br>
(d) distinguishing liasmati rice varieties from Non-Basmati (NB) rice varieties based on the presence and/or absence of said polymorphic amplified products.<br>
16.	The method as claimed in claim 15, wherein the Ilourescent label is selected from a group comprising Tamara dye. R6G and R! 10.<br>
17.	A method of distinguishing 'lYaditional  Basmati  (TB)  rice varieties  from  Hvolved Basmati (EB) rice varieties, said method comprising steps of:<br><br>
(a)	amplifying DNA from "Iraditional Basmati (TB) and Evolved Basmati (KB) rice varieties using 13NA extracted from each variety, SSR markers of Table 7. and a Huorcscently labeled nucleotide;<br>
(b)	separating fluorescent amplified products;<br>
(c)	identifying polymorphic amplified products, wherein said polymorphic amplified products are polymorphic between 'fraditional Basmati (TB) rice varieties and Involved Basmati (NB) rice varieties; and<br>
(d)disl!ngu)shing "Jradilional Basmati (TB) rice varieties from Hvolved Basmati (HB) rice varieties based on the presence and/or absence of said polymorphic amplified products. IX. The method as claimed in claim 17, wherein the fiourescent label is selected from a group<br>
comprising 'famara dye, R6G and Rl 10. 19, A kit for determining adulteration of Basmati rice with other rice varieties, said kit com prising:<br>
(a) at least one ISSR-PCR primers from a set of primers as set forth in SHQ ID Nos. I io 37, and/or<br><br>
(b) al least one SSR marker from a set oi'markcrs ol'Table-6.<br>
20.	A method of determining adulteration of Basmali rice varieties with other rice varieties,<br>
said method comprising steps of:<br>
(a)	amplifying DNA from Basmati and other rice varieties using DNA extracted (rnni<br>
each variety, an SSR primer directed to a locus selected from table 6 or an ISSR-<br>
PCR primer selected from sequences as set forth in SEQ ID Nos. 1 to 37 and a<br>
fluorescently labeled nucleotide;<br>
(b)	separating tluorescent amplified products;<br>
(c)	identifying polymorphic amplified products, wherein said polymorphic ampliticd products are polymorphic between Basmali rice variefies and other rice varieties: and<br>
(d)	determining adulteration of Basmati rice varieties with other rice variefies based on<br>
the presence and/or absence of said polymorphic amplified products.<br>
21.	A set of intcr-simplc sequence repeat (ISSR)-PCR primers are substantially as herein described and illustrated therein.<br>
22.	A method of genotyping diverse genomes of plant and animal systems using I'ISSR-FCR primers are substantially as herein described and illustrated therein.<br>
23.	A I'lSHR method of distinguishing Basmati rice varieties from Non-Basmati (NB) rice<br>
varieties arc subsianlially as herein described and illustrated therein.<br>
24.	A FiSSR method ol" distinguishing Traditional Basmati (Tli) rice varieties from Hvolvcd Basmali (NB) rice varieties arc substanfially as herein described and illustrated therein.<br>
25.	A method for determining adulteration of Basmati rice with other rice varieties using at least one ISSR-PCR primers are substantially as herein described and illustrated therein.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwLW1hcy0yMDAyIGFic3RyYWN0LWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">260-mas-2002 abstract-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwLW1hcy0yMDAyIGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">260-mas-2002 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwLW1hcy0yMDAyIGNsYWltcy1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">260-mas-2002 claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwLW1hcy0yMDAyIGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">260-mas-2002 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwLW1hcy0yMDAyIGNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">260-mas-2002 correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwLW1hcy0yMDAyIGNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">260-mas-2002 correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwLW1hcy0yMDAyIGRlc2NyaXB0aW9uIChjb21wbGV0ZSktZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">260-mas-2002 description (complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwLW1hcy0yMDAyIGRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">260-mas-2002 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwLW1hcy0yMDAyIGRlc2NyaXB0aW9uIChwcm92aXNpb25hbCktZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">260-mas-2002 description (provisional)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwLW1hcy0yMDAyIGRlc2NyaXB0aW9uIChwcm92aXNpb25hbCkucGRm" target="_blank" style="word-wrap:break-word;">260-mas-2002 description (provisional).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwLW1hcy0yMDAyIGRyYXdpbmdzLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">260-mas-2002 drawings-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwLW1hcy0yMDAyIGRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">260-mas-2002 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwLW1hcy0yMDAyIGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">260-mas-2002 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwLW1hcy0yMDAyIGZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">260-mas-2002 form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwLW1hcy0yMDAyIGZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">260-mas-2002 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwLW1hcy0yMDAyIGZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">260-mas-2002 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwLW1hcy0yMDAyIGZvcm0tNC5wZGY=" target="_blank" style="word-wrap:break-word;">260-mas-2002 form-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjYwLW1hcy0yMDAyIGZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">260-mas-2002 form-5.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="219764-a-novel-conjugate.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="219766-a-method-for-locating-and-tracking-a-wireless-terminal-and-a-wireless-terminal-thereof.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>219765</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>260/MAS/2002</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>27/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>04-Jul-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>13-May-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>08-Apr-2002</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CENTRE FOR DNA FINGERPRINTING AND DIAGNOSTICS (CDFD)</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>JAVARE GOWDA NAGARAJU</td>
											<td></td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A01H 1/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/219765-a-set-of-fissr-pcr-primers-and-methods-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:29:29 GMT -->
</html>
