<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/219745-a-novel-enzyme-preparation by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:30:48 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 219745:A NOVEL ENZYME PREPARATION</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A NOVEL ENZYME PREPARATION</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>&quot;A NOVEL ENZYME PREPARATION&quot; An enzyme preparation exhibiting endoglucanase activity, which enzyme is a) encoded by a DNA construct comprising the DNA sequence shown in SEQ ID No. 8 or a DNA construct comprising the DNA sequence obtainable from the plasmid in Saccharomyces cerevisiae DSM 10081, or b) encoded by a DNA construct comprising an analogue of the DNA sequence shown in SEQ ID No. 8 or a DNA construct comprising the DNA sequence obtainable from the plasmid in Saccharomyces cerevisiae DSM 10081, which DNA sequence has at least 75% identity with the DNA sequence shown in SEQ ID No. 8 or the DNA sequence obtainable from the plasmid in Saccharomyces cerevisiae DSN 10081, or c) a polypeptide which has at least 70% identity with the amino acid sequence shown in SEQ ID No. 9.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
The present invention relates to novel enzyme preparations cotnprising an enzyme exhibiting endoglucanase activity which performs very good in indus¬trial applications such as laundry compositions, for biopolishing of newly manufactured textiles, for provid¬ing an abraded look of cellulosic fabric or garment, and for treatment of paper pulp. Further, the invention relates to DNA constructs encoding such enzymes, a method for providing a gene encoding for such enzymes, a method of producing the   enzyme preparations containing such enzymes, and the use of these enzymes for a number of industrial applications.<br>
BACKGROUND OF THE INVENTION<br>
Cellulases or cellulytic enzymes are enzymes involved in hydrolyses of cellulose. In the hydrolysis of native cel¬lulose, it is known that there are three major types of cellulase enzymes involved, namely cellobiohydrol'ase (l,4-j3-D-glucan cellobiohydrolase, EC 3.2.1.91), endo-jS-1,4-glucanase (endo-1,4-/3-D-glucan 4-glucanohydrolase, EC 3.2.1.4) and i3-glucosidase (EC 3.2.1.21).<br>
Cellulases are synthesized by a large number of microor¬ganisms which include fungi, actinomycetes, myxobacteria and true bacteria but also by plants. Especially endoglucanases of a wide variety of specificities have been identified.<br>
'A very important industrial use of cellulytic enzymes is the use for treatment of cellulosic textile or fabric, e.g. as ingredients in detergent compositions or fabric<br><br>
softener compositions, for bio-polishing of new fabric (garment finishing), and for obtaining a "stone-washed" look of cellulose-containing fabric, especially denim, and several methods for such treatment have been sug¬gested, e.g. in GB-A-1 368 599, EP-A-0 307 564 and EP-A-0 435 876, WO 91/17243, WO 91/10732, WO 91/17244, PCT/DK95/000108 and PCT/DK95/00132.<br>
Another important industrial use of cellulytic enzymes is the use for treatment of paper pulp, e.g. for improving the drainage or for deinking of recycled paper.<br>
Especially the endoglucanases _(EC No. 3.2.1.4) constitute an interesting group of hydrolases for the mentioned industrial uses. Endoglucanases catalyses endo hydrolysis of l,4-i3-D-glycosidic linkages in cellulose, cellulose derivatives (such as carboxy methyl cellulose and hydroxy' ethyl cellulose), lichenin, /3-l,4 bonds in mixed (3-1,3 glucans such as cereal /3-D—glucans or xyloglucans and other plant material containing cellulosic parts. The authorised name is endo-l,4-^-D-glucan 4-glucano hydrolase, but the abbreviated term endoglucanase is used in the present specification. Reference can be made to T.-M. Enveri, "Microbial Cellulases" in W.M. Fogarty, Microbial Enzymes and Biotechnology, Applied Science Pub¬lishers, p. 183-224 (1983) ; Methods in Enzymology, (1988) Vol. 160, p. 200-391 (edited by Wood, W.A. and Kellogg, S.T.); Beguin, P., "Molecular Biology of Cellulose Degra¬dation", Annu. Rev. Microbiol. (1S90), Vol. 44, pp. 219-24S; Beguin, P. and Aubert, J-P,, "The biological degra¬dation of cellulose", FEMS Microbiology Reviews H (1994) p.25-58; Henrissat, B., "Cellulases and their interaction with cellulose". Cellulose (1994), Vol. 1, pp. 169-196.<br>
Fungal endoglucanases have been described in numerous publications, especially those derived from species as e.g. Fusarium oxyspoxrum,   Trlchoderma reesei,   Trichoderma<br><br>
longibrachiatum,   Aspergillus aculeatus,   Neocallimastix patriciarum,   and e.g. from species of the genera Piromyces,   Humicola,   Myceliophthora,   Geotricum, Penicillium,   Irpex,   Coprinus.<br>
For example, fungal endqgluc_anase_s; have been described by Sheppard, P.O., et al., "The use of conserved cellulase family-specific sequences to clone Cellulase homologue cDNAs from Fusarium  oxysporum,   Gene, (1994), Vol. 15, pp. 163-167, Saloheimo, A., et al., "A novel, small endoglucanase gene, eglSj   from Trichoderma  reesei   iso¬lated by expression in yeast". Molecular Microbiology (1994), Vol. 13(2), pp. 219-228; van Arsdell, J.N. et al, (1937), Cloning, characterization, and expression in Saccharomyces  cerevisiae  of endoglucanase I from Trichoderma reesei,   Bio/Technology 5: 60-64; Penttila, M. et al., (1986), "Homology between cellulase genes of Trichoderma reesei:   complete nucleotide sequence of the endoglucanase I gene". Gene   45:253-263; Saloheimo, M. et al, (1988) , •VEGIII, a new endoglucanase from Trichoderma reeseiz  the chaxacterization of both gene and enzyme". Gene 63:11-21; Gonzales, R., et al., "Cloning, sequence analysis and yeast expression of the egll  gene fr-ccm Tricho'derma longibrachiatum",   Appl. Microbiol. Biotechnol., (1992), Vol. 38, pp. 370-375; Ooi, T. et al. "Cloning and sequence analysis of a cDNA for cellulase (FI-CMCase) from Aspergillus  aculeatus",   Curr. Genet., (1990), Vol. 18, pp. 217-222; Ooi, T. et al, "Complete nucleotide sequence of a gene coding for Aspergillus aculeatus  cellulase (FI-CMCase)", Nucleic Acids Research, (1990), Vol. 18, No. 19, p. 5884; Xue, G. et al., "Clon¬ing and expression of multiple cellulase cDNAs from the anaerobic rumen fungus Neocallimastix patriciairum  in E. coli",   J. Gen. Microbiol., (1992), Vol. 138, pp. 1413-<br>
1420; -Xue, C. et al., "A novel polysaccharide hydrolase cDNA (celD)   from Neocallimastix patriciarum  encoding three multi-functional catalytical domains with high<br><br>
endoglucanase, cellobiohydrolase and xylanase activ¬ities", J. Gen. Microbiol., (1992), Vol. 138, pp. 2397-2403; Zhou, L. et al., "Intronless celB   from the anaerobic fungus Neocallimastix patriciarum  encodes a modular family A endoglucanase", Biochem. J., (1994), Vol. 297, pp. 359-364; Dalboge, H. and Heldt-Hansen, H.P., "A novel method for efficient expression cloning of fungal enzyme genes", Mol. Gen. Genet., (1994), Vol. 243, pp. 253-260; Ali, B.R.S. et al., "Cellulases and hemicellulases of the anaerobic fungus Piromyces  consti¬tute a multiprotein cellulose-binding complex and are encoded by multigene families", FEMS Microbiol. Lett., (1995), Vol. 125, No. 1, pp. 15-21. Further, the DNA Data Bank of Japan (DDBJ database publicly available at Internet) comprises two DNA sequences cloned from Penicillium janthinellum  encoding endoglucanases (cloned by A. Koch and G. Mernitz, respectively) and a DNA sequence cloned from Humicola  qrisea  var.   thermoidea encoding an endoglucanase (cloned by T. Uozumi). Two endoglucanases from Macrophomina phaseolina  have been cloned and sequenced, see Wang, H.Y- and Jones, R.W.: "Cloning, characterization and functional expression of an endoglucanase-encoding gene from the phytopathogenic fungus Wacropho/nina phaseolina"   in Gene, 158:125-128, 1995, and Wang, H.Y. and Jones, R.W.: "A unique endoglucanase-encoding gene cloned from the phytopathogenic fungus Macrophomina phaseolina"   in Applied And Environmental Microbiology, 61:2004-2006, 199 5. One of these endoglucanases shows high homology to the egl3 endoglucanase from the fungus Trichoderma reesei,   the other shows homology to the egll from the microbial phytopathogen Pseudomonas solanacearum  indicat¬ing that both endoglucanases belong to family 5 of glycosyl hydrolases (B. Henrissat, Biochem J 280:309-316 (ig^l))"^ Filament-specific expression of a cellulase gene-in the dimorphic fungus Ustilago maydis   is disclosed in Schauwecker, F. et al. (1995).<br><br>
wo 91/17243 (Novo Nordisk A/S) discloses a cellulase pre¬paration consisting of a homogenous endoglucanase compo¬nent immunoreactive with an antibody raised against a highly purified 43 kDa endoglucanase derived from Humicola  insolens,   DSM 1800; WO 91/17244 (Novo Nordisk A/S) discloses a new (hemi)cellulose degrading enzyme, such as an endoglucanase, a cellobiohydrolase or a /3-glucosidase, which may be derived from fungi other than Trichoderma  and Phanerochaete;   WO 93/20193 discloses an endoglucanase derivable from Aspergillus  aculeatus;   WO 94/21801 (Genencor Inc.) concerns a cellulase system iso¬lated from Trichoderma  longibrachiatum  exhibiting endoglucanase activity; WO 94/26880 (Gist Brocades N.V.) discloses an isolated mix±ure of cellulose degrading enzymes, which preferable are obtained from Trichoderma, Aspergillus  or Disporotrichum,   comprising endoglucanase, cellobiohydrolase, and xyloglucanase activity; and WO 95/02043 (Novo Nordisk A/S) describes an enzyme wirh endoglucanase activity derived from Trichoderma harzianum,   which can be used for a number of purposes including e.g.   degradation or modification of plant cell walls.<br>
It is also known that cellulases may or may not have a cellulose binding domain (a C^BD) . The CBD enhances the binding of the enzyme to a cellulose-containing fiber and increases the efficacy of the catalytic active part of the enzyme.<br>
There is an ever existing need for providing novel cellulase enzyme preparations which may be used for applications where cellulase, preferably an endoglucanase, activity is desirable.<br>
The object of the present invention is to provide novel enzyme preparations Jiaving substantial cellulytic activ¬ity at acid, neutral or alkaline conditions and improved<br><br>
performance in paper pulp processing, textile treatment, laundry processes or in animal feed; preferably novel cellulases, more preferably well-performing endoglucana-ses, which are contemplated to be producible or produced<br>
by recombinant techniques.<br>
SUMMARY OF THE INVENTION<br>
Surprisingly, it has been found that a group of endoglucanases having certain unique characteristics per¬form very good in those industrial applications for which endoglucanases are conventionally used. These unique characteristics can be described in terms of conserved regions of the amino acid sequence of the enzyme protein and the inventors have found that cellulytic enzymes, i.e. enzymes exhibiting cellulytic activity, having cer¬tain conserved regions arTe very effective e.g. in the treatment of laundry, in the treatment of newly manufac¬tured textile, in the treatment of papermaking pulp.<br>
Accordingly, in its first aspect the preseait invention relates to an enzyme preparation consisting essentially of an enzyme having cellulytic activity and comprising a first amino acid sequence consisting of 14 amino acid residues having the following sequence<br>
Thr Arg Xaa Xaa Asp Cys Cys Xaa Xaa Xaa Cys Xaa Trp Xaa 1   2   3   4   5   6   7   8   9   10  11  12  13  14<br>
and a second amino acid sequence consisting of 5 amino acid residues having the following sequence<br>
Trp Cys Cys Xaa Cys 1   2 ■  3   4  "3<br>
wherein.<br><br>
in position 3 of the first sequence, the amino acid is<br>
Trp, Tyr or Phe;<br>
in position 4 of the first sequence, the amino acid is<br>
Trp, Tyr or Phe;<br>
in position 8 of the first sequence, the amino acid is<br>
Arg, Lys or His;<br>
in position 9, 10, 12 and 14, respectively, of the first<br>
sequence, and in position 4 of the second sequence, the<br>
amino acid is any of the 20 naturally occurring amino<br>
acid residues with the provisos that, in the first amino<br>
acid sequence, (i) when the amino residue in position 12<br>
is Ser, then the amino acid residue in position 14 is not<br>
Ser, and (ii) when the amino residue in position 12 is<br>
Gly, then the amino acid residue in position 14 is not<br>
Ala-.<br>
This surprising finding of clearly recognisable conserved regions, in spite of rather prominent variations found within well-performing endoglucanase enzymes, is a result of studies of a number of fungal DNA sequences encoding for specific a^nino acid sequences of enzymes having sig¬nificant cellulytlc, especially endoglucauiase, activ¬ities.<br>
Based on this finding, a novel molecular method taylored to screen specifically for genomic DNA or cDNA characterised by encoding the enzymes of the invention has been developed. As tools for this three sets of degenerated primers were constructed. Accordingly, in its second aspect, the invention relates to a method for providing a gene encoding for cellulytic enzymes having the above conserved regions.<br>
By using this method, i.e. the set of primers for a PCR screening on genomic DNA, it was surprisingly found that DNA encoding for said enzymes can be found from a broad range of fungi, belonging to taxonomically very different<br><br>
organisms and inhabiting ecologically very different niches.<br>
Further, by using this method it has been possible to find DNA sequences encoding for the core regions (cata-lytically active regions or domains) of said enzymes without any attached cellulose binding domain (CBD) which core regions of enzymes would not have been selected by using conventional performance based screening approaches. The inventors have verified experimentally that the linking of a CBD region to a core region enzyme (comprising the catalytically active region or domain of the enzyme) of the present invention results in a sig¬nificantly improved performance, e.g. a fifty times higher performance, of the multiple domain enzyme.<br>
Accordingly, the present invention provides novel cellulases, especially endoglucanases, having improved performance in industial applications, either in their native form, or homo- or heterologously produced.<br>
Ln further aspects, the present invention relates to novel cellulytic enzyme preparations which are derivable from tax'onomically specific phyli, classes, orders, families, genera, and species; e.g. from Basidiomycotous Hymenomycetes,   Zygomycota,   Chytridiomycota;   or from the classes Discomycstes,     Loculoascomycetes,   Plectomycetes; Archaeascomycetes,   Hernias cornycetes  or from the orders Di-aportales,   Xylariales,   Trichosphaeriales,   Phyllachorales; or from the families Nectriaeae,   Sordariaceae, Chaetomiaceae,   Ceratostomaceae,   Lasiosphaeriaceae;   or from the genera Cylindrocarpon,   Gliocladium,   Volutella, Scytalidium,   Acremonium,   ot from the species Fusarium lycopersici,   Fusarium passiflora,   Fusarium solani, Fusarium anquioides,   Fusarium poae,   Humicola  nigrescens', Humicola grisea,   especially such consisting essentially of an enzyme comprising an amino acid sequence selected<br><br>
from the group consisting of the sequences Xaa Thr Arg Xaa Phe Asp Xaa 12   3   4   5   6   7; Xaa Thr Arg Xaa Tyr Asp Xaa 12   3   4   5   6   7; and<br>
Xaa Thr Arg Xaa Trp Asp Xaa<br>
12   3   4   5   6   7<br>
wherein, in position 4, Xaa is Trp, Tyr or Phe; and<br>
in position 1 and 7, Xaa is any of the 20 naturally<br>
occurring ainino acid residues.<br>
More specifically, the enzyme preparation of the<br>
invention is preferably obtainable from the taxonoraically<br>
specific phyli, classes, orders, families, genera, and"<br>
species mentioned above which all produce endoglucanases<br>
comprising a first peptide consisting of 13 amino acid<br>
residues having trhe following sequence<br>
Thr Arg Xaa Xaa Asp Cys Cys Xaa Xaa Xaa Cys Xaa Trp<br>
1   2   3   4   5   6   7   8   9   10  11  12  13<br>
and a second peptide consisting of 5 amino acid residues<br>
having the fo11 awing sequence<br>
Trp Cys Cys Xaa Cys<br>
12   3   4   5<br>
wherein, in position 3 of the first sequence, the amino<br>
acid is Trp, Tyr or Phe; in position 4 of the first<br>
sequence, the amino acid is Trp, Tyr or Phe; in position<br>
S of the first sequence, the amino acid is Arg, Lys or<br>
His; in position 9, 10, and 12, respectively, of the<br>
first sequence, and in position 4 of the second sequence,<br>
the amino acid is any of the 20 naturally occurring amino<br>
acid residues.<br>
In yet further aspects, the present invention provides DNA constructs comprising a DNA sequence encoding an en¬zyme exhibiting endoglucanase activity, which DNA sequence comprises the DNA sequence shown in SEQ ID Nos.<br><br>
1, 4, 6, 8, 10, 12, 16, and 19, respectively, or analogues thereof.<br>
The present invention also relates to a recombinant expression vector comprising a DNA construct of the invention; to a cell comprising a DNA construct or a recombinant expression vector of the invention; to a method of producing an enzyme, e.g a recombinant enzyme, of the invention; to a method of providing colour clari¬fication of laundry by using the enzyme of the invention; to a laundry composition comprising the enzyme of the invention; to uses of the enzyme of the invention for degradation or modification of plant material, e.g. cell walls, for treatment of fabric, textile or garment, for treatment of paper pulp; and to an enzyme preparation which is enriched in an enz^Tiie of the present invention.<br>
THE DRAVTENGS<br>
Figure l is an alignment, of the deduced encoded amino acid sequences of Arrrp-anniixa sp.   (I),  Volutella colletotrichoides,   Cx'inipellis scabella,   Acremonium sp. (II),   Myceliophthora  thermophila,   Thielavia  terrestris, Macrophomina phaseolina.   The Pileup program (Feng and Doolittle, 1987) {GCG  package, version 8.0) was used to create the best alignment. Identical residues in at least four sequences (boxed) are indicated around the corre¬sponding amino acids.<br>
Figure 2<br>
Figure 2a,b,c illustrates the taxonomic classification within the Fungal Kingdom of all the microorganisms dis¬closed herein as being capable of producing said enzyme preparations and- enaymes of the invention.<br>
The taxonomic classification used herein builds primarily<br><br>
on the system used in the :NIH Data Base (Entrez, version<br>
spring 1996) available on World Wide Web:<br>
(http: //www3 .ncbi.nliti.nih.gov/htbin/ef/entrezTAX) .<br>
Regarding classification of organisms which are not<br>
included in the Entrez data base the following generally<br>
available and world wide accepted reference books have<br>
been used:<br>
For Ascomycetes:   Eriksson, O.E. &amp; Hawksworth, D,L.:<br>
Systema Ascomycetum vol 12 (1993).<br>
For Basidiomycetes:   Jiilich, W. : Higher Taxa of<br>
Basidiomycetes, Bibliotheca Mycologia 85, 485pp (1981).<br>
For Zygomycetes:   O'Donnell, K.: Zygomycetes in culture,<br>
University of Georgia, US,  257pp (1979) .<br>
General  mycological reference books:<br>
Hawksworth, D.L., Kirk, P.M., Sutton, B.C. and Pegler,<br>
D.N.: Dictionary of the fungi. International Mycological<br>
Institute, 616pp (1995);<br>
Von Arx, J.A. : The genera of fungi spomlccting in<br>
culture, 424pp (1981).<br>
The taxonomic implacement of the genus Humicola has untill recently remained unclear- However, studies of 18SRNA of a wide selection of Sordariales has given strong indications of referring Humicola to the order Sordariales (Taylor, Clausen &amp; Oxenboll, unpublished). Further these data suggests Humicola along with Scytalid-ium to be only rather distantly related to the families Sordariaceae, Chaetomiaceae, Ceratostomataceae, and Lasiosphaeriaceae. In accordance with the above Humicola and Scytalidium are here placed within the order Sordariales, with unclassified Family.<br>
Figure 3 is an alignment of the deduced partial amino acid sequences derived from a selection of 26 of the 46 microorganisms described in Example—5T—'<br><br>
DETAILED DESCRIPTION OF THE INVENTION<br>
In the present context, the term "the 20 naturally occuring amino acid residues" denotes the 20 amino acid residues usually found in proteins and conventionally known as alanine (Ala or A), valine (Val or V), leucine (Leu or L), isoleucine (lie or I), proline (Pro or P), phenylalanine (Phe or F), tryptophan (Trp or W), methionine (Met or M), glycine (Gly or G), serine (Ser or S), threonine (Thr or T), cysteine (Cys or C), tyrosine (Tyr or Y), asparagine (Asn or N), glutamine (Gin or Q), aspartic acid (Asp or D), glutamic acid (Glu or E), lysine (Lys or K), arginine (Arg or R), and histidine (His or H).<br>
According to the present invention there is provided novel well-performing endoglucanases comprising consei-ved amino acid sequence regions, especially a first amino acid sequence consisting of 14 amino acid residues having the following sequence<br>
Thr Arg Xaa Xaa Asp Cys Cys Xaa Xara Xaa Cys Xaa Trp Xaa 1   2   3   4   5   6   7   8   9   10  11  12  13  14 and a second amino acid sequence consisting of 5 amino acid residues having the following sequence<br>
Trp Cys Cys Xaa Cys<br>
12   3   4   5<br>
wherein,<br>
in position 3 of the first sequence, the amino acid is<br>
Trp, Tyr or Phe;<br>
in position 4 of the first sequence, the amino acid is<br>
Trp, Tyr or Phe;<br>
m position 8 of the first sequence, the ammo acid is<br>
Arg, Lys or His;<br>
in position 9, 10, 12 and 14, respectively, of the first<br><br>
sequence, and in position 4 of the second sequence, the amino acid is any of the 20 naturally occurring amino acid residues with the provisos that, in the first amino acid sequence, (i) when the amino residue in position 12 is Ser, then the amino acid residue in position 14 is not Ser, and (ii) when the amino residue in position 12 is Gly, then the amino acid residue in position 14 is not Ala.<br>
Preferably, the enzyme of the invention is of microbial origin, i.e. obtainable from a microorganism such as a fungus.<br>
In a preferred embodiment, the amino acid residue in position 9 of the first sequence is selected from the group consisting of proline, threonine, valine, alanine, leucine, isoleucine, phenylalanine, glycine, cysteine, asparaglne, glutamine, tyrosine, serine, methionine and tryptophan, preferably from the group consisting of proline and threonine.<br>
In another preferred embodiment, the amino acid r^idne in position 10 of the first sequence is selected from the group consisting of proline, threonine, valine, alanine, leucine, isoleucine, phenylalanine, glycine, cysteine, asparagine, glutamine, tyrosine, serine, methionine and tryptophan, preferably serine.<br>
In yet another preferred embodiment, the amino acid resi¬<br>
due in position 12 of the first sequence is selected from<br>
the group consisting of proline, threonine, valine,<br>
alanine, leucine, isoleucine, phenylalanine, glycine, cy¬<br>
steine, asparagine, glutamine, tyrosine, serine,<br>
methionine and tryptophan, preferably from the group con¬<br>
sisting of alanine and glycine.		'<br>
In yet another preferred embodiment, the amino acid resi-<br><br>
due in position 14 of the first sequence is selected from the group consisting of proline, threonine, valine, alanine, leucine, isoleucine, phenylalanine, glycine, cy¬steine, asparagine, glutamine, tyrosine, serine, methionine, tryptophan, glutamic acid and aspartic acid, preferably from the group consisting of proline, threonine, serine, alanine, glutamic acid and aspartic acid.<br>
Preferably, the amino acid residue in position 4 of the second sequence is selected from the group consisting of proline, threonine, valine, alanine, leucine, isoleucine, phenylalanine, glycine, cysteine, asparagine, glutamine, tyrosine, serine, methionine, tryptophan, glutamic acid and aspartic acid, more preferably from the group con¬sisting of alanine, glycine, and glutcunine.<br>
Examples of more preferred embodiments are such wherein, in the first sequence, the amino acid residue in position 3 is tyrosine; or the amino acid residue in position 4 is tryptophan; or the amino acid residue in position 8 is lysine.<br>
In an especially preferred embodiment,' the enzyme of the<br>
invention has a first sequence comprising the amino acid<br>
sequence<br>
Thr Arg Tyr Trp Asp Cys Cys Lys Pro Ser Cys Ala Trp<br>
1   2   3   4   5   6   7   8   9   10  11  12  13  ,<br>
or the amino acid sequence<br>
Thr Arg Tyr Trp Asp Cys Cys Lys Thr Ser Cys Ala Trp<br>
1   2   3   4   5   6   7   8   9   10  11  12  13  ,<br>
or the amino acid sequence<br>
Thr Arg Tyr Trp Asp Cys Cys Lys Pro Ser Cys Gly Trp<br>
1   2   3   4   5   6   7   8   9   10  11  12  13  .<br>
In a second aspect, the present invention provides a method for providing a microbial strain comprising a gene<br><br>
encoding such an enzyme which method comprises hybridization, e.g. PCR amplification, under standard conditions with an oligonucleotide derived from any of the conserved regions, illustrated in Fig.l.<br>
A useful oligonucleotide comprises a nucleotide sequence encoding at least a pentapeptide comprised in a peptide selected from the group consisting of a.<br>
Thr Arg Xaa Xaa Asp Cys Cys Xaa Xaa Xaa Cys Xaa Trp Xaa 1   2   3   4   5   6   7   8   9   10  11  12  13  14 the amino acid in position 3 or 4 being Trp, Tyr or Phe; the amino acid in postion 8 being Arg, Lys or His; he amino acid in position 9, 10, 12 and 14, respectively, being any of the 20 naturally occurring amino acid resi¬dues ; and b.<br>
Trp Cys Cys Xaa Cys Tyr 12   3   4   5   6<br>
the amino acid in position 4 being any of the 2 0 natural¬ly occurring amino acid residues ; and c.<br>
Xaa Pro Gly Gly Gly Xaa Gly Xaa Phe 123456789 the amino acid in position 1 being Met or lie; the amino acid in position 6 and 8, respectively, being Leu, lie or Val; and d.<br>
Gly Cys Xaa Xaa Arg Xaa Asp Trp Xaa 123456789<br>
the amino acid in position 3 being any of the 20 natural¬ly occurring amino acid residues;<br>
the amino acid in position 4 and 6, respectively, being Trp, Tyr or Phe; and the amino acid in position 9 being Phe or Met;<br>
The useful oligonucleotides also comprises nucleotide<br><br>
sequences complementary to the sequences mentioned.<br>
In a preferred embodiment of the method of the invention,<br>
the oligonucleotide corresponds to a PCR primer selected<br>
from the PCR primers<br>
sense:<br>
5 ' -CCCCAAGCTTACIVCGITA^/TTGGGA^VTTG^/TTG^/TAA^/G^/CC-S '<br>
antisense   1:<br>
5'- CTAGTCTAGATA'^/GCAIGC'^/GCA^/GCACC -3 ' ;<br>
antisense 2:<br>
CTAGTCTAGAAAIA'^/G/'^ICCIA'^/^/°ICCICCICCIGG -3 ' ; and<br>
antisense 3:<br>
5 ' - CTAGTCTAGAIAACCA^/GTCA'^/G^/TAIC°/TCC-3 .<br>
In a third aspect, the present invention provides an enzyme preparation which essentially consists of an enzyme having cellulytic activity and having the conserved regions found by the inventors, i.e. which com¬prises a peptide consisting of 7 amino acid residues ha¬ving the following sequence Xaa Thr Arg Xaa Phe Asp Xaa 12   3   4   5   6   7; Xaa Thr Arg Xaa Tyr Asp Xaa 1   2   3   4 ' 5   6   7   ; and Xaa Thr Arg Xaa Trp Asp Xaa 12   3   4   5   6   7<br>
wherein, in position 4, Xaa is Trp, Tyr or Phe; and in position 1 and 7, Xaa is any of the 20 naturally occurring amino acid residues.<br>
This enzyme is obtainable from a strain belonging to Basidiomycotous Hymenomycetes      (see Fig.2 ), more preferably to the group consisting of the orders Agarica-les,   Auriculariales,   and Aphyllophorales,   even more preferably to the group consisting of the families Exidiaceae,   Tricholomataceae,   Coprinaceae, Schizophyllaceae,   Bjerkanderaceae  and Polyporaceae,<br><br>
especially to the group consisting of the genera Exidia, Crinipellis,   Fames,   Panaeolus,   Trametes,   Schizophyllum, and Spongipellis.<br>
Specific examples are endoglucanases obtainable from a strain belonging to the group consisting of the species Exidia  glandulosa,   Crinipellis  scabella,   Fomes fomentarius,   and Spongipellis  sp.,   more specific examples being Exidia glandulosa,   CBS 277.96, Crinipellis scabella,   CBS 280.96, Fames  fomentarius,   CBS 276.96, and Spongipellis sp.,   CBS 2 83.96.<br>
Exidia  glandulosa  was deposited at Centraalbureau voor Schimmelcultures, Oosterstraat 1, Postbus 273, NL-3740 AG Baarn, the Netherlands, on 12 March, 1996, under thfe deposition number CBS 277.96; Crinipellis  scabella  was deposited at Centraalbureau voor Schimmelcultures on 12 March, 19S-6, under the deposition number CBS 280.96, Fames fomentarius  was deposited at Centraalbureau voor Schimmelcultures on 12 March, 1996, under the deposition number CBS 276.96, and Spongipellis  sp.   was deposited at CentraaJLbareau voor Schimmelcultures on 12 Maxcfa, 1996, under the deposition number CBS 283.96; all deposited tinder the Budapest Treaty.<br>
The enzyme preparation of the invention is also obtain¬able from a strain belonging to Chytridiomycota, preferably from a strain belonging to the class of Chytridiomycetes,   more preferably belonging to the group consisting of the order Spizellomycetales   , even more preferably to the family Spizellomycetaceae,   especially belonging to the genus Rhizophlyctis.   A specific example is a strain belonging to the species Rhizophlyctis  rosea, more specifically to Rhizophlyctis  rosea,     CBS 282.96.<br>
Rhizophlyctis  rosea  was deposited at Centraalbureau voor Schimmelcultures on 12 March, 1996, under the deposition<br><br>
number CBS 28 2.96; under the Budapest Treaty.<br>
The enzyme preparation of the invention is also obtain¬able from a strain belonging to Zygomycota,   preferably belonging to the class Zygomycetes,   more preferably to the order Mucorales,   even more preferably to the group of families consisting of Mucoraceae  and Thamnidiaceae, especially belonging to the group consisting of the ge¬nera Rhizomucor,   Phycomyces  and Chaetostylum.   Specific examples are strains belonging to the genera Rhizomucor pusillus,     Phycomyces nitens,   and Chaetostylum  fresenii more specifically to Rhizomucor pusillus,   IFO 4578, and Phycomyces nitens,   IFO 4814 and Chaetostylum  fresenii, NRRL 2305.<br>
Further, the enzyme preparation of the invention is also obtainable from a strain belonging to the group consist¬ing of Archaeas^cumycetes,   Discomycetes,   Hemiascomycetes, Loculoascomycetes,   and Plectomycetes,   preferably belonging to the group consisting of the orders Pezizales,   Rhytismatales,   Dothideales,   and Eurotiales. Especially, the enzyme is obtainaJale from a strain belonging the the group consisting of the families Cucurbitariaceae,   Ascobolaceae,   Rhytismataceae,     and Trichocomaceae,   preferably belonging the the group con¬sisting of the genera Diplodia,   Microsphaeropsis, Ulospora,     Macrophomina,   Ascoholus,   Saccobolus,     Penicil-lium,   and Thermomyces.   Specific examples are enzymes obtainable from a strain belonging the the group consist¬ing of the species Diplodia  gossypina,   Microsphaeropsis sp.,   Ulospora  bilgramii,   Aureobasidium sp.,   Macrophomlna phaseolina,   Ascobolus stictoides,   Saccobolus  dilutellus, Peziza,   Penicillium  verruculosum,   Penicillium chrysogenum,   and Thermomyces  verrucosus;  more specifically Diprlodia  gossypina,   CBS 274.96,   Ulospora bilgramii,   NKBC 1444,   Macrophomina phaseolina,   CBS 281.96,   Saccobolus  dilutellus,   CBS 275.96,   Penicillium<br><br>
verruculosum,   ATCC  62396,   Penicillium  chrysogenum,   ATCC 9480, and Therwomyces  verrucosus,   CBS 285.96.<br>
Diplodia  gossypina  was deposited at Centraalbureau voor Schimmelcultures on 12 March, 1996, under the deposition number CBS 274.96, Macrophomina  phaseolina  was deposited at Centraalbureau voor Schimmelcultures on 12 March, 1996, under the deposition number CBS 281.96, Saccoholus dilutellus  was deposited at Centraalbureau voor Schimmelcultures on 12 March, 1996, under the deposition number  CBS 275.96; Thermomyces  verrucosus  was deposited at Centraalbureau voor Schimmelcultures on 12 March, 1996, under the deposition number CBS 285.96; all under the Budapest Treaty.<br>
Yet fujrther, the enzyme is obtainable from a strain belonging to the group consisting of the orders Dia-portales,   Xylariales,   Tirlchosphaeriales  and Phyllachorales,   preferably from a strain belonging to the group consistixjg of the families Xylariaceae,   Valsaceae, and PhyllacborBLCBae,   more preferably belonging to the ge¬nera Diaporthe,   Colletotrichum,   Nigrospora,   Xylaria, NodulisporuM  and Poronia.   Specific examples are the specie's Diaporthe syngenesia,   Colletotrichum lagenarium, Xylaria hypoxylon,   Nigrospora  sp., Nodulisporum sp.,   and Poronia punctata,   more specifically Diaporthe  syngenesia, CBS  278.96,   Colletotrichum lagenarium,   ATCC 52 609, Nigrospora  sp.,   CBS 272.96,   Xylaria hypoxylon,   CBS 284.96.<br>
Diaporthe  syngenesia  was deposited at Centraalbureau voor Schimmelcultures on 12 March, 1996, under the deposition number CBS 278.96, Nigrospora  sp.   was deposited at Centraalbureau voor Schimmelcultures on 12 March, 199 6, under^he—deposition number CBS 272.96, Xylaria hypoxylon was deposited at Centraalbureau voor Schimmelcultures on 12 March, 1996, under the deposition number CBS 284.96;<br><br>
all under the Budapest Treaty.<br>
The enzyme is also obtainable from the unidentified fungal, mitosporic, coleomycetous deposited at Centraalbureau voor Schimmelcultures on 12 March, 1996, under the deposition numbers CBS 270.96, CBS 271.96 and CBS 273.96, respectively, under the Budapest Treaty.<br>
The enzyme is also obtainable from a strain belonging to the group consisting of the genera Cylindrocarpon, Gliocladium,   Nectria,   Volutella,   Sordaria,   Scytalidium, Thielavia,   Syspastospora,   Cladorrhinum,   Chaetomium,   Myce-liphthora  and Acremonium,   especially from a strain belon¬ging to the group consisting of the species Cylin¬drocarpon sp.,   Nectria pinea,   Volutella colletotrichoides,   Sordaria  fimicola,   Sordaria macrospora,   Thielavia  terrestris,   Thielavia   thermophila, Syspastospora boninensis,   Cla^dorrhinxrm foecundissimum, Chaetomium muronrm,   Chaetomium  virescens,   Chaetomium brasiliensis,   Chaetomium cunicolojrum,   Myceliophthora thermophila,   Gliocladium catenulatum,   Scytalidium thermophila,   and Ai:ri^nir&gt;nJmn sp.,   more specifically from Nectria pinea,   CBS 273.96,   Volutella  colletotrichoides, CBS 400.58,   Sordaria  fimicola,   ATCC 52644,   Sordaria macrospora,   ATCC  60255,   Thielavia  terrestris,   NRRL  8126, Thielavia  thermophila,   CCBS 174.70,   Chaetomium murorum, CBS  163.52,   Chaetomium  virescens,   CBS  547.75,   Chaetomium brasiliensis,   CBS 122.65,   Chaetomium  cunicolorum,   CBS 799.83,   Syspastospora boninensis,   NKBC 1515,   Cladorrhinum foecundissimum,   ATCC 62373, Myceliophthora   thermophila, CBS 117.65, Scytalidiim  thermophila,   ATCC 28085, Gliocladium catenulatum, ATCC 10523, and Acremonium sp., CBS 478.94.<br>
Nectria pinea  was deposited at Centraalbureau voor Schimmelcultures on 12 March, 1996, under the deposition number CBS 279.96, and Acremonium  sp.   was deposited on<br><br>
28 September 1994 under the deposition number CBS 478.94, both according to the Budapest Treaty.<br>
The enzyme is also obtainable from a strain belonging to the group consisting of the species Fusarium solani, Fusarium anguioides,   Fusarium poae,   Fusarium oxysporum ssp.   lycopersici,   Fusarium  oxysporum  ssp.   passiflora, Humicola nigrescens  and Humicola  grisea,   especially Fusarium oxysporum ssp lycopersici,   CBS  645.78,   Fusarium oxysporum ssp passiflora,   CBS  744.79,   Fusarium  solani, IMI  107.511,   Fusarium anguioides,   IFO 44 67,   Fusarium poae,   ATCC  60883,   Humicola  nigrescens,   CBS 819.73 and Humicola  grisea,   ATCC 22726. It is to be noted that Humi¬cola grisea   is different from Humicola grisea  var. thermoidea.<br>
In a preferred embodiment, the enzviae preparation of the invention is derived from the disclosed classes, orders, families, genera and species and essentially consists of an enzyme comprising a first peptide consisting of 13 amino acid residues having the following sequence<br>
Thr Arg Xaa Xaa Asp Cys Cys Xaa Xaa Xaa Cys Xaa Trp 1   2   3   4   5   6*7   8   9   10  11  12  13<br>
and a second peptide consisting of 5 amino acid residues having the following sequence<br>
Trp Cys Cys Xaa Cys 12   3   4   5<br>
wherein, in position 3 of the first sequence, the amino acid is Trp, Tyr or Phe; in position 4 of the first sequence, the amino acid is Trp, Tyr or Phe; in position 8 of the first sequence-;—Lhi=i 'amino acid is Arg, Lys or His; in position 9, 10, and 12, respectively, of the first sequence, and in position 4 of the second sequence.<br><br>
coli,   DSM 10512, DSM 10511, DSM 10571, DSM 10576, respectively; or<br>
b) an analogue of the DNA sequence shown in SEQ ID No. 1, 4, 6, 8, 10, 12, 16, or 19, respectively, or the DNA sequence obtainable from the plasmid in Saccharomyces cerevisiae  DSM 9770, DSM 10082, DSM 10080, DSM 10081, Escherichia  coli,   DSM 10512, DSM 10511, DSM 10571, DSM 10576, respectively, which<br>
i)    is homologous with the DNA sequence shown in SEQ ID No. 1, 4, 6, 8, 10, 12, 16, or 19, respectively, or the DNA sequence obtainable from the plasmid in Saccharomyces  cerevisiae  DSM 9770, DSM 10082, DSM 10080, DSM 10081, Escherichia  coli, DSM 10512, DSM 10511, DSM 10571, DSM 10576, respectively, ii)   hybridizes with the same oligonucleotide probe as the DNA sequence shown in SEQ ID No. 1, 4, 6, 8, 10, 12, 16, or 19, respectively, or the DNA sequ¬ence obtainable from the plasmid in Saccharomyces cerevisiae   DSM 9770, DSM 10082, DSM 10080, DSM 10O31, Escherichia  coli,   DSM 10512, DSM 10511, DSM 10571, DSM 10576, respectively, iii)  encodes a polypeptide which is homologous with the polypeptide encoded by a DNA sequence comprising the DNA sequence shown in SEQ ID No. 1, 4, 6, 8, 10, 12, 16, or 19, respectively, or the DNA sequ¬ence obtainable from the plasmid in Saccharomyces cerevisiae   DSM 9770, DSM 10082, DSM 10080, DSM 10081, Escherichia   coli,   DSM 10512, DSM 10511, DSM 10571, DSM 10576, respectively, iv)   encodes a polypeptide which is immxmologically re¬active with an antibody raised against the purified endoglucanase encoded by the DNA sequence shown in SEQ ID No 1, 4, 6, 8, 10, 12, 16, or 19, respectively, or the DNA sequence obtainable, from the plasmid in Saccharomyces  cerevisiae  DSM 977 0, DSM 10082, DSM 10080, DSM 10081, Escherichia  coli,<br><br>
DSM 10512, DSM 10511, DSM 10571, DSM 10576, respectively.<br>
Escherichia  coli   DSM 10512 was deposited under the Buda¬pest Treaty on 2 February, 1996, at DSM (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany).<br>
Escherichia  coli   DSM 10511 was deposited under the Buda¬pest Treaty on 2 February, 1996, at DSM (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany).<br>
Escherichia  coli  DSM 10571 was deposited under the Buda¬pest Treaty on 6 March, 1996, at DSM (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany).<br>
Escherichia  coli   DSM 10576 was deposited under the Buda¬pest Treaty on 12 March, 1996, at DSM (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braxonschweig, Germany) .<br>
Escherichia  coli  DSM 10583 was deposited under the Buda¬pest Treaty on 13 March, 1996, at DSM (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany).<br>
Escherichia  coli   DSM 10584 was deposited under the Buda¬pest Treaty on 13 March, 1996, at DSM (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany).<br>
Escherichia  coli  DSM 10585 was deposited under the Buda-pest Treaty on 13 March, 1996, at DSM (Deutsche Sammlufvj-von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany).<br><br>
Escherichia   coli   DSM 10586 was deposited under the Buda¬pest Treaty on 13 March, 1996, at DSM (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany).<br>
Escherichia  coli   DSM 10587 was deposited under the Buda¬pest Treaty on 13 March, 1996, at DSM (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany).<br>
Escherichia  coli   DSM 10588 was deposited under the Buda¬pest Treaty on 13 March, 1996, at DSM (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany).<br>
Saccrharomyces  cerevisiae  DSM 9770 was deposited under the Budapest Treaty on 24 February, 1995, at DSM (Deutsche ScTncralung von Mikrrrorg^nismeTi und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany).<br>
SacL.hc±iomyces  cerevisiae  DSM 100B2 was deposited under the.  Budapest Treaty on 3 0 June, 1995, at DSM (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany).<br>
Saccharomyces  cerevisiae  DSM 10080 was deposited under the Budapest Treaty on 30 June, 1995, at DSM (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany).<br>
Saccharomyces  cerevisiae  DSM 10081 was deposited under the Budapest Treaty on 30 June, 1995, at DSM (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany).<br>
The DNA construct of■the invention relating to SEQ ID No. 1 can be isolated from or produced on the basis of a DNA<br><br>
library of a strain of Myceliophthora,   in particular a strain of M.   thermophila,   especially M.   thermophila,   CBS 117.65.<br>
The DNA constructs of the invention relating to SEQ ID Nos. 4 and 6 can be isolated from or produced on the basis of a DNA library of a strain of Acremonium, especially Acremonium sp.,   CBS 478,94.<br>
The DNA construct of the invention relating to SEQ ID No. 8 can be isolated from or produced on the basis of a DNA library of a strain of Thielavia   in particular a strain of Thielavia  terrestris,   especially Thielavia  terrestris, NRRL 8126.<br>
The DNA construct of the invention relating to SEQ ID No. 10 can be isolated from or produced on the basis of a DNA library of a strain of Macrcp2io2±na, in particular a strain of M. phaseolina,   especially M.phaseolina,   CBS 281.96.<br>
The DNA construct of the invention relating to SEQ ID No. 12 can be isolated from or produced on the basis of a DNA library of a strain of .Crinipellis,   in particular a strain of C. scaJbeila, especially C.scabella,   CBS 280.96.<br>
The DNA construct of the invention relating to SEQ ID No. 19 can be isolated from or produced on the basis of a DNA library of a strain of Sordaria,   in particular a strain of Sordaria fimicola.<br>
In the present context, the "analogue" of the DNA sequence shown in SEQ ID No. 1, 4, 6, 8, 10, 12, 16 or 19, respectively, is intended to indicate any DNA sequence encoding an enzyme exhibiting endoglucanase activity, which has any or all of the properties i)-iv).<br><br>
The analogous DNA sequence<br>
a)	may be isolated from another or related (e.g. the same) organism producing the enzyme with endoglucanase activity on the basis of the DNA sequence shown in SEQ ID No. 1, 4, 6, 8, 10, 12, 16 or 19, respectively, e.g. using the procedures described herein; the homologue may be an allelic variant of the DNA sequence comprising the DNA sequences shown herein, i.e. an alternative form of a gene that arises through mutation; mutations can be silent (no change in the encoded enzyme) or may encode enzymes having altered amino acid sequence; the homologue of the present DNA sequence may also be a genus or spe¬cies homologue, i.e. encoding an enzyme with a similar activity derived from another species,<br>
b)	may be constructed on the basis of the DNA sequences shown in SEQ ID No. 1, 4, 6, 8, 10, 12, 16 or 19, respectively, e.g. by introduction of nucleotide substitutions which do not give rise to another amino acid sequence of the endoglucanase encoded by the DNA sequence, but which correspond to the codon usage of the host organism intfrnded for production of the enzyme, or by introduction of nucleotide substitutions which may give rise to a different amino acid sequence. However, in the latter case amino acid changes are preferably of a minor nature, that is conservative amino acid substitu¬tions that do not significantly affect the folding or activity of the protein, small deletions, typically of one to about 30 amino acids; small amino- or carboxyl-terminal extensions, such as an amino-terminal methionine residue, a small linker peptide of up to about 20-25 residues, or a small extension that facilitates purifica¬tion, such as a poly-histidine tract, an antigenic epitope or a bind-ing domain. See in general Ford et al., Protein Expression and Purification. 2: 95-107, 1991. Examples of conservative substitutions are within the<br><br>
group of basic amino acids (such as arginine, lysine, histidine), acidic amino acids (such as glutamic acid and aspartic acid), polar amino acids (such as glutamine and asparagine), hydrophobic amino acids (such as leucine, isoleucine, valine), aromatic amino acids (such as phenylalanine, tryptophan, tyrosine) and small amino acids (such as glycine, alanine, serine, threonine, methionine).<br>
It will be apparent to persons skilled in the art that such substitutions can be made outside the regions criti¬cal to the function of the molecule and still result in an active polypeptide. Amino acids essential to the activity of the polypeptide encoded by the DNA construct of the invention, and therefore preferably not subject to substitution, may be identified according to procedures known in the art, such as site-directed mutagenesis or alanine-scanning mutage-nersls (Cunninghara and Wells, Science 244. 1081-1085, 1989). In the  latter technique mutations are introduced at every residue in the mol¬ecule, and the resultamrt mutant molecules are tested for biological (i.e. endoglucanasa) activity to identify amino acid residues that are critical to the activity of the molecule. Sites of substrate-enzyme interaction can also be determined by analysis of crystal structure as determined by such techniques as nuclear magnetic reson¬ance, crystallography or photoaffinity labeling. See, for example, de Vos et al.. Science 255: 306-312, 1992; Smith et al., J. Mol. Biol. 224: 899-904, 1992; Wlodaver et al., FEBS Lett. 3 09: 59-64, 1992.<br>
The endoglucanase encoded by the DNA sequence of the DNA construct of the invention may comprise a cellulose bind¬ing domain (CBD) existing as an integral part of the encoded enzyme,"or a CBD from another origin may be introduced into the endoglucanase enzyme thus creating an enzyme hybride. In this context, the term "cellulose-<br><br>
binding domain" is intended to be understood as defined by Peter Tomme et al. "Cellulose-Binding Domains: Classi¬fication and Properties" in "Enzymatic Degradation of Insoluble Carbohydrates", John N. Saddler and Michael H. Penner (Eds.), ACS Symposium Series, No. 618, 1996. This definition classifies more than 120 cellulose-binding domains (CBDs) into 10 families (I-X), and it demon¬strates that CBDs are found in various enzymes such as cellulases, xylanases, mannanases, arabinofuranosidases, acetyl esterases and chitinases. CBDs have also been found in algae, e.g., the red alga Porphyra purpurea as a non-hydrolytic polysaccharide-binding protein, for refer¬ence see Peter Tomme et al., supra. However, most of the CBDs are from cellulases and xylanases- CBDs are found at the N or C termini of proteins or are internal. Enzyme hybrids are known in the art, see e.g. WO SO/00609 and WO 95/167 82, and may be prepared by transfoi"iaing into a host cell a DNA construct comprising at. least: a fragment of DNA encoding the cellulose-binding domain ligated, with or without a linker, to a DNA sequence encoding the enzyme of interest and growing tlie hosrt cell to express the fused gene. Enzyme hybrids may be described by the following formula:<br>
CBD - MR - X,<br>
wherein CBD is the N-terminal or the C-terminal region of an amino acid sequence corresponding to at least the cellulose-binding domain; MR is the middle region (the linker), and may be a bond, or a short linking group preferably of from about 2 to about 100 carbon atoms, more preferably of from 2 to 4 0 carbon atoms; or is preferably from about 2 to about 100 amino acids, more preferably of from 2 to 40 amino acids; and X is an N-terminal or C-terminal region of a po^lypeptide encoded by the DNA sequence of the invention.<br><br>
The homology referred to in i) above is determined as the degree of identity between the two sequences indicating a derivation of the first sequence from the second. The homology may suitably be determined by means of computer programs known in the art such as GAP provided in the GCG program package (Needleman, S.B. and Wunsch, CD., Jour¬nal of Molecular Biology,   48_:   443-453, 1970). Using GAP with the following settings for DNA sequence comparison: GAP creation penalty of 5.0 and GAP extension penalty of 0.3, the coding region of the DNA sequence exhibits a degree of identity preferably of at least 60%, more preferably at least 65%, more preferably at least 70%, even more preferably at least 80%, especially at least 90%, with the coding region of the DNA sequence shown in SEQ ID No.l, 4, 6, 8, 10, 12-, or 16, respectively, or the DNA sequence obtainable from the plasmid in Saccharomyces cerevisiae,   DSM 9770, DSM 10082, DSM 10080, DSM 10081, Escherdchia  coli, DSH 10512, DSM 10511, DSM 10571, or DSM 10576, respectively.<br>
The hybridization referred to in ii) above is intended to i nd i crat.e that the analogous UNA sequence hybridizes to the same probe as the DNA sequence encoding the endoglucanase enzyme under certain specified conditions which are described in detail in the Materials and Methods section hereinafter. The oligonucleotide probe to be used is the DNA sequence corresponding to the endoglucanase encoding part of the DNA sequence shown in SEQ ID NO 1, 4, 6, 8, 10, 12, or 16  respectively, or the DNA sequence obtainable from the plasmid in Saccharomyces cerevisiae,   DSM 9770, DSM 10082, DSM 10080, DSM 10081, Escherichia  coli,   DSM 10512, DSM 10511, DSM 10571 or DSM 10576, respectively.<br>
—5^e homology referred to in iii) above is determined as the degree of identity between the two sequences indicat¬ing a derivation of the first sequence from the second.<br><br>
:he homology may suitably be determined by means of com-5uter programs known in the art such as GAP provided in :he GCG program package (Needleman, S.B. and Wunsch, :.D., Journal   of Molecular Biology,   AB_:   443-453, 1970). Jsing GAP with the following settings for polypeptide sequence comparison: GAP creation penalty of 3.0 and GAP extension penalty of 0.1, the polypeptide encoded by an analogous DNA sequence exhibits a degree of identity pre¬ferably of at least 55%, more preferably at least 60%, nore preferably at least  65%, even more preferably at least 70%, more preferably at least 80%, especially at least 90%, with the enzyme encoded by a DNA construct :::omprising the DNA sequence shown in SEQ ID No.l, 4, 6, 3, 10, 12, 16 or 19, respectively, or the DNA sequence obtainable from the plasmid in Saccharomyces  cerevisiae, DSM 9770, DSM 10082, DSM 10080, DSM 10081, Escherichia coli,   DSM 10512, DSM 10511, DSM 10571 or DSM 10576, respectively.<br>
In connection with property iv) above it is intended to indicate an endoglucanase encoded by a DNA sequence iso¬lated txom  strain Saccharomyces cerevisiae,   DSM 9T70, DSM 100B2, DSM 10080, DSM 10081, Escherichia  coli,   DSM 10512, DSM 10511, DSM 10571 or DSM 10576, respectively, arid produced in a host organism transformed with said DNA sequence or the corresponding endoglucanase naturally produced by Myceliophthora  thermophila,   Acremonium sp., Thielavia  terrestris,   Macrophomina phaseolina, Crinipellis scabella,   Volutella  colletotrichoides,   or Sordaria fimicola,   respectively. The immunological reac¬tivity may be determined by the method described in the Materials and Methods section below.<br>
In further aspects the invention relates to an expression<br>
vector harbouring a DNA construct of the invention,—a	<br>
cell comprising the DNA construct or expression vector and a method of producing an enzyme exhibiting<br><br>
vative amino acid substitutions that do not significantly affect the folding or activity of the protein, small deletions, typically of one to about 30 amino acids; small amino- or carboxyl-terminal extensions, such as an amino-terminal methionine residue, a small linker peptide of up to about 20-25 residues, or a small extension that facilitates purification, such as a poly-histidine tract, an antigenic epitope or a binding domain. See in general Ford et al. , Protein Expression and Purification 2: 95-107, 1991. Examples of conservative substitutions are within the group of basic amino acids (such as arginine, lysine, histidine), acidic amino acids (such as glutamic acid and aspartic acid), polar amino acids (such as glutamine and asparagine), hydrophobic amino acids (such as leucine, isoleucine, valine), aromatic amino acids (such as phenylalanine, tryptophan, tyrosine) and small amino acids (such as glycine, alanine, serine, threonine, methionine).<br>
It will be apparent to persons skilled in the art that such substitutions can be made outside the regions criti¬cal to the function of tiie laolecule and still result in an active enzyme. Amino acids essential to the activity of the enzyme of the invention, and therefore preferably not subject to substitution, may be identified according to procedures known in the art, such as site-directed mutagenesis or alanine-scanning mutagenesis (Cunningham, 1989). In the  latter technique mutations are introduced at every residue in the molecule, and the resultant mutant molecules are tested for cellulytic activity to identify amino acid residues that are critical to the activity of the molecule. Sites of ligand-receptor interaction can also be determined by analysis of crystal structure as determined by such techniques as nuclear magnetic resonancBr;—Crystallography or photoaffinity labelling. See, for example, de Vos et al., 1992; Smith et al., 1992. Wlodaver et al., 1992.<br>
The homologue may be an allelic variant, i.e. an alterna¬tive form of a gene that arises through mutation, or an altered enzyme encoded by the mutated gene, but having substantially the same activity as the enzyme of the invention  Hence mutations can be silent (no change in the encoded enzyme) or may encode enzymes having altered amino acid sequence.<br>
The homologue of the present enzyme may also be a genus or species homologue, i.e. an enzyme with a similar activity derived from another species.<br>
A homologue of the enzyme may be isolated by  using the procedures described herein.<br>
Molecular screening and cloning bv polymerase chain reaction fact)<br>
Molecular screening for DNA sequences of the invention may be carried out by polymerase chain reaction (PCR) using genomic DNA or double—strained cDNA isolated from a sraitable source, such as any of the herein mentioned organisms, and synthetic oligonucleotide primers prepared on the basis of the DNA sequences or the amino acid sequences disclosed herein. For instance, suitable oligonucleotide primers may be the primers described in the Materials and Methods section.<br>
In accordance with well-known procedures, the PCR fragment generated in the molecular screening may be isolated and suctioned into a suitable vector. The PCR fragment may be used for screening DNA libraries by e.g. colony or plaque hybridization.<br><br>
Expression cloning in yeast<br>
The DNA sequence of the invention encoding an enzyme exhibiting endoglucanase activity may be isolated by a general method involving<br>
cloning, in suitable vectors, a DNA library from a<br>
suitable source, such as any of the herein<br>
mentioned organisms<br>
transforming suitable yeast host cells with said<br>
vectors,<br>
culturing the host cells under suitable conditions<br>
to express any enzyme of interest encoded by a<br>
clone in the DNA library,<br>
screening for positive clones by determining any<br>
endoglucanase activity of the enzyme produced by<br>
such clones, and<br>
isolating the enzyme encoding DNA from such<br>
clones.<br>
The general method is further disclosed in WO	94/14953<br>
the contents of which are hereby incorporated	by<br>
rejference. A acre detailed description of the	screening<br>
method is given in Example 1 belov.<br>
The DNA sequence coding for the enzyme may for instance be isolated by screening a cDNA library of Macrophomina phaseolina,   Crinipellis  scabella,   Sordaria  flmicola   or Volutella  colletotricboides,   and selecting for clones ex¬pressing the appropriate enzyme activity (i.e. endoglucanase activity) or from Escherichia  coli  DSM 10512 deposited under the Budapest Treaty on 2 February, 1996, at DSM (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany), or- from Escherichia  coli   DSM<br>
lOSl-l—dapcsited under the Budapest Treaty on 2 February,	<br>
1996, at DSK, or  from Escherichia coli DSM 10576, deposited under the Budapest Treaty on 12 March, 1996, at<br><br>
DSM; or  from Escherichia -coli   DSM 10571 deposited under the Budapest Treaty on 6 March, 19S6, at DSM; or by screening a cDNA library of Myceliphthora   thermophila, CBS 117.55, Acremonium  sp.,   CBS 473.94, or Thielavia terrestris,   NRRL 8126, and selecting for clones expres¬sing the appropriate enzyme activity (i.e. endoglucanase activity) or from Saccharomyces  cerevisiae   DSM 9770 deposited under the Budapest Treaty on 24 February, 1995, at DSM (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany), or from Saccharomyces  cerevisiae DSM 10032 deposited under the Budapest Treaty on 30 June, 1995, at DSM, from Saccharomyces  cerevisiae  DSM lOOSO deposited under the Budapest Treaty on 30 June, 1995, or from Saccharomyces  cerevisiae  DSM 10081 deposited under the Budapest Treaty on 30 June, 19S5, at DSM. The appropriate DNA sequence may then be isolated from the clone by s~ar.dari proced"ai"es, e-g- as aescrj-sed m Example 1.<br>
WTJcIeic acid construct<br>
As used herein the term '"nucleic acid construct" is intended to indicate any nucleic acid molecule of cDNA, genomic DNA, syn-hetic DNA or RNA origin. The term "construct" is intended to indicate a nucleic acid segment which may be single- or double-stranded, and which may be based on a complete or partial naturally occurring nucleotide sequence encoding an enzyme of interest. The construct may optionally contain other nucleic acid segments.<br>
The nucleic acid construct encoding the enzyme of the invention may suitably be_Qi_genomic orcDNA origin, for instance obtained by preparing a genomic or cDNA library and screening for DNA sequences coding for all or part of<br><br>
the enzyme by hybridization using synthetic oligonucleotide probes in accordance with standard techniques (cf. Sambrook et al., 1989).<br>
The nucleic acid construct encoding the enzyme may also be prepared synthetically by established standard methods, e.g. the phosphoamidite method described by Beaucage and Caruthers, (1981), or the method described by Matthes et al., (1984). According to the phospho¬amidite method, oligonucleotides are synthesized, e.g. in an automatic DNA synthesizer, purified, annealed, ligated and cloned in suitable vectors.<br>
Furthermore, the nucleic acid construct may be of mixed synthetic and genomic, mixed synthetic and cDNA or mixed genomic and cDNA origin prepared by ligating fragments of s%Tithetic, genomic or cDNA origin (as appropriate), the fragmentLS corresponding to various parts cf rhe entire nucleic acid construct, in accordance with standard tech¬niques.<br>
The nncleic acid construct nay also be prepared by polymerase chain reaction using specific priaers, for instance as described in US 4,683,202 or Saiki et al., (1988) .<br>
The nucleic acid construct is preferably a DNA construct which term will be used exclusively in this specification and claims.<br>
Recombinant vector<br>
A recombinant vector comprising a DNA construct encoding the enzyme of the invention may be any -vector which s.^y conveniently be subjected to recombinant DNA procedures, and the choice of vector will often depend on the host<br><br>
cell into which it is to be introduced. Thus, the vector may be an autonomously replicating vector, i.e. a vector which exists as an extrachromosomal entity, the replication of which is independent of chroinosomal replication, e.g. a plasmid. Alternatively, the vector may be one which, when introduced into a host cell, is integrated into the host cell genome and replicated together with the chromosome(s) into which it has been integrated.<br>
The vector is preferably an expression vector in which the DNA seguence encoding the enzyme of the invention is operably linked to additional segments required for transcription of the DNA. In general, the expression vector is derived from plasmid or viral DNA, or may contain elements of both. The term, "operably linked" indicates that the segments are arranged so that they function in concert for their intended purposes, e.g. transcription initiates in a promoter and proceeds through the DNA sequence coding for the enzyme.<br>
The prnmnter may be any DNA sequence which shows transcriptional activity in the host cell of choice and may be derived from genes encoding proteins either homologous or heterologous to the host cell.<br>
Examples of suitable promoters for use in yeast host cells include promoters from yeast glycolytic genes (Hitzeman et al., J. Biol. Chem. 255 (1980), 12073 -12080; Alber and Kawasaki, J. Mol. APPI. Gen. 1 (1982), 419 - 434) or alcohol dehydrogenase genes (Young et al., in Genetic Engineering of Microorganisms for Chemicals (Hollaender et al, eds.), Plenum Press, New York, 1982), or the TPIl (US 4,599,311) or ADH2-4C (Russell et al.. Nature 3 0^ rT98 3). 652 - 654) promoters.<br>
Examples of suitable promoters for use in filamentous<br><br>
fungus host cells are, for instance, the ADH3 promoter (McKnight et al.. The EMBO J. i (1985), 2093 - 2099) or the tpJA promoter. Examples of other useful promoters are those derived from the gene encoding A.   oryzae  TAKA amylase, Rhizomucor miehei  aspartic proteinase, A.   niger neutral a-amylase, A.   niger  acid stable a-amylase, A. niger  or A.   avamori   glucoamylase (gluA), Rhizomucor miehei   lipase, A.   oryzae  alkaline protease, A.   oryzae triose phosphate isomerase or A.   nidulans  acetamidase. Preferred are the TAKA-amylase and gluA promoters.<br>
Examples of suitable promoters for use in bacterial host<br>
cells include the promoter of the Bacillus<br>
stearothermophilus  maltogenic amylase gene, the Bacillus<br>
licheniformis  alpha-amylase.  gene, the Bacillus	. : . ,<br>
ajuyloliquefaciens  BAN amylase gene, the Bacillus  subtilis alkaline protease gen, or the Bacillus  pumilus   xylcsidase gene, cr by the phage Lambda PR or P^ promoters or rhe E. coli lac, trp or tac promoters.<br>
Th-e DNA sequence encoding th-e enzyme of the invention may also. i:f necessary, be operably connected to a suitable<br>
terminator.<br>
The recombinant vector of the invention may further comprise a DNA sequence enabling the vector to replicate in the host cell in question.<br>
The vector may also comprise a selectable marker, e.g. a gene the product of which complements a defect in the host cell, such as the gene coding for dihydrofolate reductase (DHFR) or the Schizosaccharomyces  pombe  TPI gene (described by P.R. Russell, Gene 40, 19S5, pp. 125-130). For filamentous fungi, selectable markers include amdS7~PVrG. argB, niaP, sC.<br>
To direct an enzyme of the present invention into the<br><br>
secretory pathway of the host cells, a secretory signal sequence (also known as a leader sequence, prepro sequence or pre sequence) rr.ay be provided in the recombinant vector. The secretory signal sequence is joined to the DNA sequence encoding the enzyme in the correct reading frame. Secretory signal sequences are commonly positioned 5' to rhe DliA sequence encoding the enzyme. The secretory signal sequence may be that normally associated with the enzyme or may be from a gene encoding another secreted protein.<br>
For secretion from yeast cells, the secretory signal sequence may encode any signal peptide which ensures efficient direction of the expressed enzyme into the secretory pathway of the cell. The signal peptide may be naturally occurring signal peptide, or a functional part thereof, or it may be a syntheric peptide. Suitable signal peptides have been fO'jnd to be che a-facror signal peptide (cf. US 4,370,COS), the signal peptide of mouse salivary amylase (of. O. Kagencuchle et al., Nature 289, 1981, pp. 643-646), a modified carbcxypeptidase signal peptide (cf. L.A. Vails er al.. Cell 48, 19S7, pp. 887-897), the yeast BARl signal peptide (cf. WO 87/02670), or the yeast aspartic protease 3 (YAP3) signal peptide (cf. M. Egel-Mitani et al., Yeast 6,    1990, pp. 127-137).<br>
For efficient secretion in yeasr, a sequence encoding a leader peptide may also be inserted downstream of the signal sequence and upstream of the DNA sequence encoding the enzyme. The function of the leader peptide is to allow the expressed enzyme to be directed from the endoplasmic reticulum to the Golgi apparatus and further to a secretory vesicle for secrerion into the culture medium, (i.e. exportation of the enzyme across the cell wail or at least through the cellxrXar m.embrane into the periplasmic space of the yeast cell). The leader peptide may be the yeast a-factor leader (the use of which is<br><br>
described in e.g. US 4,546,082, EP 16 201, EP 123 294, EP 123 544 and EP 163 529). Alternatively, the leader peptide may be a synthetic leader peptide, which is to say a leader peptide not found in nature. Synthetic leader peptides may, for instance, be constructed as described in WO S9/02463 or WO 92/11378.<br>
For use in filamentous fungi, the signal peptide may conveniently be derived from a gene encoding an Aspergillus  sp. amylase or glucoaraylase, a gene encoding a Rhizomucor miehei   lipase or protease, a Humicola lanuginosa   lipase. The signal peptide is preferably derived from, a gene encoding .4. oryzae   TAKA amylase, A. niger  neutral a-amylase, .-.. niger  acid-stable amylase, or A.   niger  glucoam.ylase.   ^ -<br>
The procedures used to ligate the DN'A sequences coding for the present enzyme, the promoter and optionally the terminator and/or secretory signal sequence, respectively, and to insert them into suitable vectors containing the information necessary for replication, are well known to persons skilled in the art (cf., for instance, Sambrook et al., op.cit.).<br>
Host cells<br>
The DNA sequence encoding the present enzyme introduced into the host cell may be either homologous or heterologous to the host in question. If homologous to the host cell, i.e. produced by the host cell in nature, it will typically be operably connected to another promoter sequence or, if applicable, anoTiher secretory signal sequence and/or terminator sequence than in its natural environment. The term "homologous" is intended to include a cDNA sequence encoding an enzyme native to the host organism in question. The term "heterologous" is<br><br>
intended to include a DNA sequence not expressed by the host cell in nature. Thus, the DNA sequence may he from another organism, or it may be a synthetic sequence.<br>
The host cell into which the DNA construct or the recombinant vector of the invention is introduced may be any cell which is capable of producing the present enzyme and includes bacteria, yeast, fungi and higher eukaryotic cells.<br>
Examples of bacterial host cells which, on cultivation, are capable of producing the enzyme of the invention are gram-positive bacteria such as strains of Bacillus,   such as strains of B.   subtilis,   B.   licheniformis,   5.   ler.tus, B.   brevis,   B.   stearothermophilus,   B.   alkalophilus,   3. amyloliquefacier.s,   B.   coagulans,   B.   circvlans,   B.   lautus, B.   megatherium   or 3.   tburingier.sis,   or   strair.s cf Szrepzoi?.yces,   such as S.   livida::s   or S. jnurinus,   or gram-negative bacteria such as Echerichia  coli.   The transformation of the bacteria may be effected by protoplast transformation or by using comperent cells in a manner known per se (cf. Sambrook et al., suora).<br>
When expressing the enzyme in bacteria' such as E.   coli, the enzyme may be retained in the cytoplasm, typically as insoluble granules (known as inclusion bodies), or may be directed to the periplasmic space by a bacterial secretion sequence. In the for»-er case, the cells are lysed and the granules are recovered and denatured after which the enzyme is refolded by diluting the denaturing agent. In the latter case, the enzyme may be recovered from the periplasmic space by disrupting the cells, e.g. by sonication or osmotic shock, to release the contents of the periplasmic space and recovering the enzyme.<br>
Examples of suitable yeasts cells include cells of Saccharomyces  spp. or Schizosaccharomyces  spp., in<br><br>
pa:.-ticular strains of Saccharomyces  cerevisiae   or Saccharomyces  kluyveri.   Methods for transforming yeast cells with heterologous DNA and producing heterologous enzymes therefrom are described, e.g. in US 4,599,311, US 4,931,373, US 4,870,008, 5,037,743, and US 4,845,075, all of which are hereby incorporated by reference. Transformed cells are selected by a phenotype determined by a selectable marker, commonly drug resistance or the ability to grow in the absence of a particular nutrient, e.g. leucine. A preferred vector for use in yeast is the POTl vector disclosed in US 4,931,373. The DNA sequence encoding the enzyme of the invention may be preceded by a signal sequence and optionally a leader sequence , e.g. as described above. Further examples of suitable yeasr cells are strains of Kluyveromyces,   such as .^'. lactis, Hansenula,   e.g. H.   polymorpha,   or Pichia,   e.g. P. pastoris   (cf. Gleescn er al., J. Gen. Microbiol. 132, 1986, pp. 3455-3465; US 4,882,275).<br>
Examples cf other fungal cells are cells of filamentcus fungi, e.g. Aspergillus   spp., Neurospora   spp., Fusarium spp. cr Trichoderma  spp., in particular strains of A. oryzae,   A.   nldulans,   A.   niger,   or Fusarium  graminearum. The use of Aspergillus  spp. for the expression of pro¬teins is described in, e.g., EP 272 277, EP 230 023. The transformation of F.   oxysporuw  may, for instance, be carried out as described by Malardier et al., 1989, Gene 73: 147-156.<br>
When a filamentous fungus is used as the host cell, it may be transformed with the DNA construct of the invention, conveniently by integrating the DNA construct in the host chromosome to obtain a recombinant host cell. This integration is- generally considered to be an advantage as the DNA sequence is more likely to be stably maintained in the cell.  Integration of the DNA constructs into the host chromosome may be performed<br><br>
according to conventional methods, e.g. by homologous or heterologous recombination.<br>
The transformed or transfected host cell described above is then cultured in a suitable nutrient medium under conditions permitting the expression of the present enzyme, after which the resulting enzyme is recovered from the culture.<br>
The medium used to culture the cells may be any conventional medium suitable for growing the host cells, such as minimal or complex media containing appropriate supplements. Suitable media are available from commercial suppliers or may be prepared according to published recipes (e.g. in catalogues of the American Type Culture Collection) . The enz^ine produced by the cells may then be recovered from the culture medium by conventional procedures including separating the host cells from z'r.e. medium by csnrrifugarion or fiitra-icn. precipiraning zr.e proteinaceous components of the supernatant or filtrate by means of a salt, e.g. ammonium sulphate, purification by a variety of chromarographic procedures, e.g. ion exchange chromatography, gel filtration chromiatography, affinity chromatography, or the like, dependent on the type of enzyme in question.<br>
In a still further aspect, the present invention relares to a method of producing an enzyme according to the invention, wherein a suitable host cell transformed with a DNA sequence encoding the enzyme is cultured under conditions permitting the production of the enzyme, and the resulting enzyme is recovered from the culture.<br>
Enzyme Screening driven by taxonomy as well as ecology:<br>
A powerful tool like the molecular screening disclosed herein, designed to detect and select said type of<br><br>
interesting enzymes, can still not stand on its own. In order to maximize the chances of making interesting discoveries the molecular screening approach was in the present investigation combined with careful selection of which fungi to screen. The selection was done through a thorough insight in the identification of fungi, in taxonom.ical classification and in phylogenetic relationships.<br>
A taxonomic hot spot for production of cellulytic enzymes can further only be fully explored if also the ecological approach is included. Thorough knowledge about the adaptation to various substrates (especially saprotrophic, necrotrophic or biotrophic degradation of •plant m-aterials) are prerequisites for designing an intelligent screening and for managing a successful selection of strains and ecological niches to be searc¬hed .<br>
Both the taxonomy and the ecological approach disclosed herein aim at maximizing discovery" of said enzymes in the molecular screening program. However, still several hundreds (or if all preliminary work is included) several thousand fungi have been brought in culture in order to detect the 53 hits of said type of cellulytic enzy^m.e here reported.<br>
The screening and cloning may be carried out using the following:<br>
MATERIALS AND METHODS<br>
List of organisms:<br>
Saccharomyces cerevisiaB, -USiL. 917 0, DSM 10082, DSM 10080, DSM 10081, or Escherichia coli, DSM 10512, DSM 10511, DSM 10571, DSM 10576, respectively, containing the plasmid<br><br>
comprising the full length DNA sequence, coding for the endoglucanase of the invention, in the shuttle, vector pYES 2.0.<br>
Escherichia   coli   DSM 10583, 10584, 10585, 10586, 10587, and 10588.<br>
Diplodia   gossypir.a   Cooke Deposit of Strain, Ace No: CBS 274.96 Classification: Ascomycota , Loculoascorr.ycetes , Dothideales, Cucurbitariaceae<br>
Ulospora   bilgramii    {Hawksw.   et   al.)   Hawksw.   et   al . Ace No of strain: NK3C 1444, Nippon University, (Prof. 'Tubaki collection) Classification: Ascorriycota , Loculoascomycetes, Dothideales, (family unclassified)<br>
Microsphaeropsis   sp.<br>
Isolated from: Leaf of Camellia japonica (Theaceae,<br>
Guttifsrales), grown in Kunming Botanical garden, Yunnan<br>
Province, China<br>
Classification: Ascomycota, Loculoascomycetes,<br>
Dothideales, (family unclassified)<br>
Macrophomina  phaseolina     (Tassi)   Goidannich<br>
Syn: Rhizoctonia baraticola<br>
Deposit of Strain, Ace No.:CBS 281.96<br>
Isolated from seed of Glycine max (Leguminosa), cv CMM<br>
60, grown in Thailand, 1990<br>
Classification: Ascomycota, Discomycetes, Rhytism.atales,<br>
Rhytismataceae<br>
Ascojbolus stictoideus  Speg.<br>
Isolated from goose dung, Svalbard, "TTorway<br>
Classification: Ascomycota, Discomycetes, Pezizales,<br>
Ascobolaceae<br><br>
Saccobolus  dilutellus   (Fuck.)   Sacc. Deposit of strain: Ace No CBS 275.96<br>
Classification: Ascomycota, Discomycetes, Pezizales, Ascobolaceae<br>
Penicillium  verruculosum     Peyronel<br>
Ex on Ace No of species: ATCC 62396<br>
Classification: Ascomycota, Plectomycetes, Eurotiales,<br>
Trichocomaceae<br>
Penicillium  ckrysogenum  Thorn<br>
Ace No of Strain: ATCC 9480<br>
Classification: Ascomycota, Plectomycetes, Eurotiales,<br>
Trichocomaceae<br>
Thermomyces  verrucosus  Pugh  et   ai Deposit of Strain, Ace No.: CES 235.96<br>
Classification: Aseor.yecra, PlectcmyceTies, Eurcriales, (family unclassified; affiliation based on ISS RNA, sequencing and  homologies)<br>
Xylaria hypcxylcn  L.   ex  Greville<br>
Deposit of srrain, Ace No: CBS 2S4.96<br>
Classification: Ascomycota, Pyrenomycetes, Xylariales,<br>
Xylariaceae<br>
Poronia  punctata   (Fr.ex  L.}   Fr.<br>
Classification: Ascomycota, Pyrenomycetes, Xylariales,<br>
Xylariaceae<br>
Nodulisporum  sp<br>
Isolated from leaf of Camellia reticulata (Theaeeae,<br>
Guttiferales), grown in Kunming Botanical Garden, Yunnan<br>
Province, China<br>
Classification: Ascomycota-,—Pyrenomycetes, Xylariales,<br>
Xylariaceae<br><br>
Cylindrocarpon  sp<br>
Isolated from marine sample, the Bahamas<br>
Classification: Ascomycota, Pyrenomycetes, Hypocreales<br>
(unclassified)<br>
Acremonium  so<br>
Deposit of Strain, Ace. No.: CBS 478.94<br>
Classification: Ascomycota, Pyrenomycetes, Hypocreales,<br>
Hypocreaceae<br>
Fusarium  anguioides  Sherbakoff<br>
Ace No of strain: IFO 4467<br>
Classification: Ascomycota, Pyrenomycetes, Hypocreales,<br>
H&gt;'pocr eaceae<br>
Fusarium poae   (Peck)   Wr.<br>
Ex on Ace No of species: ATCC 6OS S3<br>
i_iassir ica~ ion: Ascomyccta , Pyrenomyce'ces , nypocreaJ.es,<br>
Hypocreaceae<br>
Fusariun     solani   (Mart.)Sacc.emnd,Snyd  5 Hans.<br>
Ace No of strain: IMI 107.511<br>
Classification: Ascomycota, Pyrenomycetes, Hypocreales,<br>
Hypocreaceae<br>
Fusariuui oxysporum  ssp  lycopersici    (Sacc .) Sr.yd .   &amp;   Har.s .<br>
Ace No of strain: CBS 64 5.7 3<br>
Classification: Ascomycota, Pyrenomyeeres, Hypocreales,<br>
Hypocreaceae<br>
Fusarium oxysporum ssp passiflora<br>
hoc  No of strain: CBS 7 4 4.79<br>
::lassif ication: Ascomycota, Pyrenomycetes, Hypocreales,<br>
-lypocreaceae<br>
Gliocladium  catenulatum  Gillman   &amp;  Abbott Ace. No. of strain: CBS 227.48<br><br>
classification: Ascoinycota, Pyrenomycetes, Hypocreales, Hypocreaceae<br>
Nectria  pinea     Dingley<br>
Deposit of Strain, Ace. No. CBS 279.96<br>
Classification: Ascomycota, Pyrenomycetes, Hypocreales,<br>
Nectriaceae<br>
Volutella   colletotrichoides<br>
Ace No of Strain: CBS 400.53<br>
Classification: Ascoinycota, Pyrenomycetes, Hypocreales<br>
(unclassified)<br>
Sordaria  macrospcra   Auerswala<br>
Ex on Ace No of species: ATCC 60255<br>
Classification: Ascomycota, Pyrenomycetes, Sordariales,<br>
Sordariaceae<br>
Sordaria     fimicola   (Roberge)   Cesati   ez  De   '^otaris Ex on Ace. No. for the species: ATCC 52644 Isolat.ed from dung by H.Dissi.ng, ISP, KU, Denmark Classification: Ascomycota, Py^enonycetes, Sordariales, Sordariaceae<br>
Humicola  grisea  Traeen<br>
ex on Ace No for the species: ATCC 2 2726<br>
Source: Hatfield Polytechnic<br>
Classification: Ascomycota, Pyrenomycetes, Sordariales,<br>
(fam. unclassified)<br>
Humicola  nigrescens  Omvlk<br>
Ace No of strain: CBS S19.73<br>
Classification: Ascomycota, Pyrenomycetes, Sordariales,<br>
(fam. unclassified)<br>
Scytalidium  thermophilum   (Cooney  et  Emerson)   Austwick Ace No of strain: ATCC 28085<br><br>
Classification: Ascomycota, PyrenomycGtes, Sordariales, (fam. unclassified)<br>
Thielavia   thermophila   Fergus   et  Sinden I  (syn Corynascus thermophilus)<br>
Ace No of strain: CBS 174.70, IMI 145.136<br>
Classification: Ascomycota, Pyrenomycetes, Sordariales,<br>
Chaetomiaceae<br>
Isolated from  Mushroom compost<br>
Thielavia   terrestris   (Appinis)   Malloch  et   Cain<br>
Ace No of strain: NRRL3126<br>
Classification: Ascomycota, Pyrenomycetes, Sordariales,<br>
Chaetomiaceae<br>
Cladorrhinum  foecundissimum Saccardo  et  Marchal<br>
Ex on Ace No of species: ATCC 62 27 3<br>
Ciassif icatio.n: A.sccr.ycora , ?'j"rencmyce"ces , Sordariales,<br>
Lasiosphaeriaceae<br>
Isolared from leaf of Selandin sp. (Ccmpositaceae,<br>
A.sterales) , Dallas .Mountain, Jamaica<br>
Syspastospora  boninensis<br>
Ace No of strain: NKBC 1515 (Nippon University, profe<br>
Tubaki Collection)<br>
Classification: Ascomycota, Pyrenomycetes, Sordariales,<br>
Cerastomataceae<br>
Chaetomium  cuniculorum  Fuckel<br>
Ace. No. of strain: CBS 799.83<br>
Classification: Ascomycota, Pyrenomycetes, Sordariales,<br>
Chaetomiaceae<br>
Chaetomium  brasiliense  Batista   et  Potual<br>
Ace No of strain; CBS -L22^ 65<br>
Classification: Ascomycota, Pyrenomycetes, Sordariales,<br>
Chaetomiaceae<br><br>
Chaetojp.ium  murorum  Corda<br>
Ace No of strain: CES 163.52<br>
Classification: Ascomycota, Pyrenomycetes, Sordariales,<br>
Chaetomiaceae<br>
Chaetowium  virescens   (von  Arx)   Udagawa<br>
Ace.No. of strain: CBS 5 4 7.75<br>
Classification: Ascomycota, Pyrenomycetes, Sordariales,<br>
Chaetomiaceae<br>
Myceliophthora   tbermophlla   (Apinis)   Oorschot<br>
Deposit of Strain, Ace No:CBS 117.65<br>
Classification: Ascomycota, Pyrenomycetes, Sordariales,<br>
Chaetomiaceae<br>
Nigrospora  sp<br>
Deposit of strain. Ace No: CBS 272.96<br>
Isolated from leaf of Artocarp'uS aitiiis, Moraceae,<br>
Urticales qrowv,   in Christiana, Jamaica<br>
Classification: Ascomycota, pyrenomycetes,<br>
Trichcsphaeriales, (family unclassified)<br>
Nigrospora  sp<br>
Isolated from leaf of Pinus yuannanensis, Botanical<br>
Garden, Kuning, Yunnan.<br>
Classification: A.scomycota, Pyrenomycetes,<br>
Triehosphaeriales, Abietaceae, Pinales.<br>
Diaporthe  syngenesia<br>
Deposit of strain, .Ace No: CBS 278.96<br>
Classification: .Ascomycota, Pyrenomycetes, Diaporthales,<br>
Valsaceae<br>
Colletotrichum  lagenarium   (Passerini)   Ellis  at  Halsted<br>
syn Glomerella cinguiata var orbiculare Jenkins et<br>
Winstead<br>
Ex on ace No of species: ATCC 52609<br><br>
Classification: Ascomycota, Pyrenomycetes, Phyllachorales<br>
Exidia   glandulosa  Fr.<br>
Deposit of Strain, Ace No: CBS 277.96 Classification: Basidiomycota, Hymenoiriycetes, Auriculariales, Exidiaceae<br>
Crinipellis  scabella   (Alb.&amp;Schw.:Fr.)Murr<br>
Deposit of strain: Ace No CBS 280.96<br>
Classification: Basidiomycota, Hymenomycetes, Agaricales,<br>
Panaeolus  retivugis   (Fr.)   Gill.<br>
Ace.No. of strain: CBS 275.47<br>
Classification: Basidiomycota, Kynenc-ycetes, Agaricales,<br>
Coprinaceae		<br>
Fojnes   fomentarius   (L.)   Fr.<br>
Deposit of strain: Ace No. CBS 2 7 6.96<br>
Classification: Basidiorr.ycora, Hymenomycetes,<br>
Aphyllophorales,<br>
Fomitaceae<br>
Spongipellis  sp.<br>
Deposit of Strain: Ace No CBS 283.96<br>
Classification: Basidiomycota, Hymenomycetes,<br>
Aphyllophorales,<br>
Bjerkanderaceae (identified and affiliated taxonomieally<br>
by ISS sequence and homology)<br>
Trametes  sanguinea   (Fr.)   Lloyd<br>
syn: Polyporus sanguineus; Pycnoporus sanguineus (L.:Fr.)<br>
Murrill<br>
Ace No of strain: AKU 5062 (Kyoto University Culture<br>
Collection)<br>
Classification: Basidiomycota, Aphy-Llophorales,<br>
Polyporaceae<br><br>
Schizophyllum  commune  Fr<br>
Ace. No. of species: ATCC 38548<br>
Classification: Basidiomycota, Aphyllophorales,<br>
Schizophyllaceae<br>
Rhizophlyctis  rosea   (de  Bary  &amp;  Wor)   Fischer Deposit of Strain: Ace No.: CBS 282.96 Classification: Chytridiomycota, Chytridiomycetes, Spizellomycetales, Spizellomycetaceae<br>
Rhizomucor pusillus   (Lindt)   Schipper<br>
syn: Mucor pusillus<br>
Ace No of strain: IFO 4 578<br>
Ex en Ace No of species: ATCC 468S3<br>
Classification: Zygomycota, Zygomycetes, Mucorales,<br>
Mucoraceae<br>
Phycomyces   nitens    (Kur.ze)   van   Tieghem   &amp;   Le   Monnier<br>
Ace No of strain: IFO.4314<br>
Ex en Kco  No of species: ATCC 16327<br>
Classification: Zygomycota, Zygomycetes, Mucorales,<br>
Mucoraceae<br>
Chaetostylum  fresenii   van Tieghem &amp; Le Monnier<br>
syn. Helicostylum fresenii<br>
Ace No of strain NRRL 2 3 05<br>
Classification: Zygomycota, Zygomyceres, Mucorales,<br>
Thamnidiaceae<br>
Unclassified:<br>
Trichothecium roseum<br>
Ace No of strain: IFO 5372<br>
Coniothecium sp<br>
Endophyte, isolated from leaf of unidentifed higher<br><br>
plant, growing in Kunming, Yunnan, China<br>
Unclassified and Un-identified:<br>
Deposit of strain, Ace No.: CBS 271.96<br>
Isolated from leaf of Artocarpus altilis (Moraceae,<br>
Urticales), grown in Christiana, Jamaica<br>
Deposit of strain. Ace No.: CBS 273.96<br>
Isolated from leaf of Pimenta dioica (Myrtaceae,<br>
Myrtales) grown in Dallas Mountain, Jamaica<br>
Deposit of strain: CBS 270.96<br>
Isolated from leaf of PseudocalyroiBa alliaceum  -  -. ..- 	<br>
(Bignoniaceae, Solanales) growing in Dallas Mountain,<br>
u s m a 1 c a<br>
Other strains:<br>
Escherichia  coli   MC1061 and DKIOB.<br>
Yeast strain: The Saccharomyces  csrsvisiae  strain used was W3124 (MATa; ura 3-52; leu 2-3, 112; his 3-D200; pep 4-1137; prcl::HIS3; prbl:: LEU2; cir+).<br>
Plasmids:<br>
The Aspergillus  expression vector pHD414 is a derivative of the plasmid p775 (described in EP 238 023). The construction of pHD414 is further described in WO 93/11249.<br>
pYES 2.C (Invitrogen)<br>
pA2C477, PA2C193, pA2C357, pA2C371, pA2C385, pA2C475, pA2C438, p/v2C502 (See example 1, 2, 2 and 4).<br><br>
Isolation of the DNA sequence shown in SEQ ID No. l, 4, 6, 8, 10, 12, 16, or 19 respectively:<br>
The full length DNA sequence, comprising the cDNA sequence shown in SEQ ID No. 1, 4, 6, 8, 10, 12, 16 or 19, respectively, coding for the endoglucanase of the invention, can be obtained from the deposited organism S. cerevisiae,   DSM 9770, DSM 100S2, DSM 10080, DSM 10081, E. coli,   DSM 10512, DSM 10511, DSM 10571 or DSM 10576, respectively, by extraction of plasmid DNA by methods known in the art (Sambrook et al. (1989) Molecular cloning: A laboratory manual. Cold Spring Harbor lab.. Cold Spring Harbor, NY).<br>
PCR primers for molecular screening of cellulases of the present invention:<br>
The four degenerate, deoxyinosine-conraining cligcnucieotide primers (sense; s and antisense; asl, as2 and as3) corresponding to four highly conserved amino acid regions fou.nd in the deduced amino acid sequences of Thielavia   terrestris  cellulase, Myceliophthora thermophllum  cellulase, and two cellulases frcs Acremonlum sp.   The residues are numbered according to the Myceliophthora   thermophilum  sequence. The deoxyinosines are depicted by an I in the primer sequences, and the restriction sites are underlined.<br>
(klippe-klistre fra SKa<br><br>
Molecular screening by polymerase chain reaction (PCR):<br>
In vitro amplification of genomic DNA and double-stranded cDNA.<br>
j  Directional, double-stranded cDNA was synthesized from 5 /ig of poly(A)*RNA as described below. Genomic DNA was isolated according to Yelton et al.<br>
Approximately 10 to 20 ng of double-stranded, cellulase-induced cDNA or 100 to 200 ng of genomic DNA from a selection of fungal strains was PCR amplified in PCR buffer (10 mM Tris-HCl, pH 8.3, 50 mM KCl, 1.5 mM MgClj, 0.01 % (w/v) gelatin) containing 200 ^M of each dNTF and 100 pmoi of each degenerate primer in three combinations:<br>
1)<br>
sense,<br>
D   — V-'v_'^ _ .-..^. vj c i lA'-—!   /(-oiiA' / JTGLJGA  /-[TG   /[-i'lj  /-[/"..-i   /,;  /(-C—-i<br>
antisense 1,<br>
5'- CTAGTCTAGATA\',;CAIGC\/J-;C.VV,;CACC -3'; or<br>
2) sense,<br>
5 ' - CCCCAAGCTTAClVrGITA'-7TTGGGA'7TTG^/TTG'7T^-A'^/n^/cC-3 ' antisense 2, CTAGTCTAGA.-J'.IAV.;/'^ICCIAV'7"ICCICCICCIGG -3 ■ ; or<br>
3) sense,<br>
5' - CCCCAAGCTTACIVCC-ITA^7TTGGGA'7TTG'7TTG'7T^"V\/,/'/,.C-3 ■<br>
antisense 3,<br>
5 ■ - CTAGTCTAGAIA;^.CCA\/nTCA'\/n-7'TAIC
a DNA thermal cycler (Landgraf, Germany) and 2.5 units of Taq polymerase (Perkln-Elmer, Cetus, USA). Thirty cycles of PCR were performed using a cycle profile of denaturation at 94 °C for 1 min, annealing at 64 °C for 2<br><br>
min, and extension at 72 °C for 3 min. Ten-/xl aliquots of the amplification products were analyzed by electrophoresis in 3 % agarose gels (NuSieve, FMC) with Haelll-digested ^XIV^ RF DNA as a size marker.<br>
Direct sequencing of the PCR products. Eighty-/il aliquots of the PCR products were purified using the QIAquick PCR purification kit (Qiagen, USA) according to the manufacturer's instructions. The nucleotide sequences of<br>
I  the amplified PCR fragments were determined directly on the purified PCR products by the dideoxy<br>
chain-termination method, using 50-150 ng template, the Taq deoxy-terminal cycle sequencing kit (Perkin-Elmer, US.A) , fluorescent labeled terminators and 5 pmol of the<br>
i  sense primer: 5'- CCCCAAGCTTACI"/(-GITA'\'TTGGGA'~/-T-<br>
G'^/T-TG'/-AA'^/G'/(-C-3 ' . Analysis of the sequence data were perforned according to Devereux et al.<br>
Cloning by polymerase chain reaction (PCR): Su±)cloning of PCR fragments.<br>
Twentyfive-^i aliquots of the PCR products generared as described above were electrophoresed in 0.8 % low gelling temperature agarose (SeaPlaque GTG, FMC) gels, the relevant fragments were excised from the gels, and recovered by agarase treatment by adding 0.1 vol of 10 x agarase buffer (New England Biolabs) and 2 units per 100 jj.1  molten agarose to the sample, followed by incubation at 45 °C  for 1.5 h. The sample was phenol and chloroform extracted, and precipitated by addition of  2 vols of 96 % EtOH and 0.1 of 3 M NaAc, pH 5.2. The PCR fragments were recovered by centrifugarion, washed in 70 % EtOH, dried and resuspended in 20 fxl   of restriction enzyme buffer (10 mM Tris-HCl, 10 mM MgCl2, 50 mM NaCl, 1 mM DTT) . The fragments were digested witrh—^indlll and Xbal, phenol and chloroform extracted, recovered by precipitation with  2 vols of 96 % EtOH and 0.1 of 3 M<br><br>
NaAc, pH 5.2, and subcloned into HindIII/Xi&gt;aI-cleaved pYES 2.0 vector.<br>
Screening of cDNA libraries and characterization of the positive clones. cDNA libraries in S. cerevisiae or E. coli, constructed as described below, were screened by colony hybridization (Sambrook, 1989) using the corresponding randoiTi-primed (Feinberg and Vogelstein) "p-labeled (&gt;1 x lo"cpm/^q)   PCR products as probes. The hybridizations were carried out in 2 x SSC (Sambrook, 1989), 5 X Denhardfs solution (Sambrook, 1989), 0.5 % (w/v) SDS, 100 fig/ml denatured salmon sperm DNA for 20 h at 65^C followed by washes in 5 x SSC at 25°C (2 x 15 min), 2 X SSC, 0.5 % SDS   at 65°C (30 min), 0.2 x SSC,    0.5 % SDS at- 65°C (30 min) and finally in 5 x SSC ( 2 x 15   '       ' min) at 25=C. The positive cDNA clones were characterized by sequencing rhe ends of the cD.N'.A inserts with pYES 2.0 polylinker primers (Invitrogen, USA.), and by derermining the nucleotide seuence of the longest cDNA from both strands by the dideoxy chain termination method (Sanger et al.) using fluorescent labeled terminators. Qiagen purified plasmid D.^^'.-. (Qiagen, USA) was sequenced with the Taq deoxy terminal cycle sequencing kit (Perkin Elmer, USA) and either pYES 2.0 polylinker primers (Invitrogen, US.A) or synthetic oligonucleotide primers using an Applied Biosystems 373A automated sequencer according to the ma.nufacturers instructions.  Analysis of the sequence data was performed according to Devereux et al.<br>
Extraction of total RNA was performed with guanidinium thiocya.nate followed by ultracentrifugation through a 5.7 M CsCl cushion, and isolation of poly(A)"RNA was carried out by oligo(dT)-cellulose affinity chromatography using the procedures described in WO 94/14953.<br>
cDNA synthesis: Double-stranded cDNA was synthesized from 5 /ig poly (A)* RNA by the RNase H method (Gubler and<br><br>
Hoffman (1983) Gene 25:263-269, Sambrook et al. (1989) Molecular cloning: A laboratory manual, Cold Spring Harbor lab.. Cold Spring Harbor, NY) using the hair-pin modification developed by F. S. Hagen (pers. comm.). The poly(A)' RNA (5 Mg in 5 ^1 of DEPC-treated water) was heated at 70°C for 8 min. in a pre-siliconized, RNase-free Eppendorph tube, quenched on ice and combined in a final volume of 50 IJ.1   with reverse transcriptase buffer (50 mM Tris-Cl, pH 8.3, 75 mM KCl, 3 mM MgCl,, 10 mM DTT, Bethesda Research Laboratories) containing 1 mM of dATP, dGTP and dTTP and 0.5 mM  5-methyl-dCTP (Pharmacia), 40 units human placental ribonuclease inhibitor (RNasin, Promega) , 1.45 ug cf cligc (dT) ,,-Not I prim.er (Pharmacia) and 1000 units Superscript II RNase H reverse - trans-criptase (Bethesda Research Laboratories) --First- '•     . , strand cDNA was synthesized by incubating the reacrion mixture at 4 5"C fcr 1 hour. After synthesis, rhe mi<cdna nybrid mi>rture was gelfij.'craTec ~hrougn a MicroSpin S-4 00 HR (Pharmacia) spin column according ro the manufacturer's instructions.<br>
After rhe gelfiltration, the hybrids were diluted in 2 50 /il second strand buffer (20 mM Tris-Cl, pH 7.4, 90 miM KCl, 4.6 mM'MgCl,, 10 mM (NH4)2S04, 0.16 mM iSViAD^) containing 200 ^M of each dNTP, 60 units I.   coli   DNA polymerase I (Pharmacia), 5.25 units RNase K (Promega) and 15 units E.   coli   DNA ligase (Boehringer Mannheim). Second strand cDNA synTihesis was performied by incubating the reaction tube at 16°C  for 2 hours and additional 15 min. at 25°C. The reaction was stopped by addition of EDTA to a final concentration of 2 0 inM followed by phenol and chloroform extractions.<br>
Mung bean nuclease treatment: The double-stranded cDNA was precipitateti—at -20°C for 12 hours by addition of 2 vols 96% EtOH, 0.2 vol 10 M NH4AC, recovered by centrifugation, washed in 70% EtOH, dried and resuspended<br><br>
in 30 Ml Mung bean nuclease buffer (30 mM NaAc, pH 4.6, 300 mM NaCl, 1 rc.M ZnSO,, 0.35 mM DTT, 2% glycerol) containing 25 units Mung bean nuclease (Pharmacia). The single-stranded hair-pin DNA was clipped by incubating the reaction at 30^C for 30 min., followed by addition of 70 /il 10 mM Tris-Cl, pH 7.5, 1 mM EDTA, phenol extraction and precipitation with 2 vols of 96% EtOH and 0.1 vol 3 M NaAc, pH 5.2 on ice for 30 min.<br>
Blunt-ending with T4 DNA polymerase: The double-stranded cDNAs were recovered by centrifugation and blunt-ended in 30 fil   T4 DNA polymerase buffer (20 mM Tr is-acetate, pH 7.9, 10 mM MgAc. 50 mM KAc, 1 mM DTT) containing 0.5 ir.M of each dNTP and 5 units T4 DNA polymerase (N'ew England Biolabs) by incubating, the reaction mixture at 16°C for 1 hour. The reaction was stopped by addition of EDTA to a final coficentrarion of 20 mM, followed by phenol and chlcrcfcr:?. e&gt;rcrac-icns, and precipitarion for 12 hours at -20-C by adding 2 vols 96% EtOH and 0.1 vol 3 M NaAc pH<br>
Adapter ligation/ Not I digestion and size selection:<br>
After the fill-in reaction the cDNAs were recovered by centrifugation, washed in 70% EtOH and dried. The cDNA pellet was resuspended in 25 /il ligation buffer (30 mM Tris-Cl, pH 7.S, 10 -uM MgCl,, 10 mM DTT, 0. 5 mM ATP) containing 2.5 ^g non-palindromic BstXI adaptors (Invitrogen) and 30 units T4 ligase (Promega) and incubated at 16=C for 12 hours. The reaction was stopped by heating at 65°C for 20 min. and then cooling on ice for 5 min. The adapted cDNA was digested with Not I restriction enzvTue by addition of 2 0 ul   water, 5 ul ICx Not I restriction enzyme buffer (New England Biolabs) and 50 units Not I (New England Biolabs), followed by incubation for 2.5 hours' at 37='C. The reaction was stopped by heating at 65°C for 10 min. The cDNAs were size-fractionated by gel electrophoresis on a 0.8%<br><br>
SeaPlaque GTG low melting temperature agarose gel (FMC) in Ix TBE to separate unligated adaptors and small cDNAs. The cDNA was size-selected with a cut-off at 0.7 kb and rescued from the gel by use of /J-Agarase (New England Biolabs) according to the manufacturer's instructions and precipitated for 12 hours at -20°C by adding 2 vols 96% EtOH and 0.1 vol 3 M NaAc pH 5.2.<br>
Construction of libraries: The directional, size-selected cDNA was recovered by centrifugation, washed in 70% EtOH, dried and resuspended in 30 ^1 10 mM Tris-Cl, pH 7.5, 1 mM EDTA. The cDNAs were desalted by gelfiltration through a MicroSpin S-300 HR (Pharmacia) spin coluinn according to the manufacturer's instructions. Three test ligations were carried out in 10 /il.ligation buffer (30 mM Tris-Cl, pH 7.8, 10 mM MgCl., 10 mM DTT, 0.5 nsM ATP) containing 5 til double-stranded cDNA (reacricn rubes -1 and #'2), 15 units T4 iigase (Fromega) and 50 ng (zube fl), 40 ng (tube #2) and 40 ng (tube #3, the vector background control) of BstXI-NctI cleaved pYES 2.0 vector. The ligation reactions were perfcruied by incubation at 16'C for 12 hours, heating at 70'C for 20 ir.in. and addition of 10 fil water to each tube. 1 ^1 of each ligation mixture was electroporated into 40'^1   electrocompetent E.   coli DHIOB cells (Bethesda research Laboratories) as described (Sambrook et al. (1939) Molecular cloning: A laboratory manual, Cold Spring Harbor lab., Cold Spring Harbor, NY). Using the optimal conditions a library was established in E.   coli  consisting of pools. Each pool was made by spreading transformed E.   coli   on LB+ampicillin agar plates giving 15.000-30.000 colonies/plate after incubation at 37°C for 24 hours. 20 ml LB+ampicillin was added to the plate and the cells were suspended herein. The cell suspension was shaked in a 50 mi tube for 1 hour at 37°C. Plasmid DNA was isolated from the cells according to the manufacturer's instructions using QIAGEN plasmid kit and stored at -20°C.<br><br>
1 ^1 aliquots of purified plasmid DNA (100 ng//il) from individual pools were transformed into S. cerevisiae W3124 by electroporation (Becker and Guarante (1991) Methods Enzymol. 194:182-137) and the transformants were plated on SC agar containing 2% glucose and incubated at 30°C.<br>
Identification of positive colonies:  After 3-5 days of growth, the agar plates were replica plated onto a set of SC + galactose-uracil agar plates containing 0.1% AZCL HE cellulose. These plates were incubated for 3-7 days at 30°C. Endoglucanase positive colonies were identified as colonies surrounded by a blue halo.<br>
Cells from enz-v'-rrie-positive colonies were spread for single colony isolation on agar, and an enzy-e-producing single colony was selected for each of the endoglucanase-<br>
Characterization of positive clones: The positive clones were obtained as single colonies, the cDNA inserts were aaplified direcrly from the yeast colony using biotinylated polylinker primers, purified by magnetic beads (Dynabead M-280, Dynal) system and characterized individually by sequencing the 5"-end of each cDNA clone using the chain-termination method (Sanger et al. (1977) Proc. Natl. Acad. Sci. U.S.A. 74:54 63-5467) and the Soquenase system (United States Biochemical).<br>
The nucleotide sequence was determined of the longest cDNA from, both strands by the dideoxy chain termination -cethod (Sanger et al.) using fluorescent labeled terminators. Plasmid DNA was rescued by transformation into E.   coll   as described below. Qiagen purified plasmid DNA (Qiagen, USA) was sequenced  with the Tag deoxy terminal cycle sequencing kit (Perkin Elmer, USA) and either pYES 2.0 polylinker primers (Invitrogen, USA) or<br><br>
synthetic oligonucleotide primers using an Applied Biosystems 373A automated sequencer according to the manufacturers instructions.  Analysis or the sequence data was performed according to Devereux et al .<br>
'■)<br>
Isolation of a cDNA gene for expression in Aspergillus: An endoglucanase-producing yeast colony was inoculated into 20 ml YPD broth in a 50 ml glass test tube. The tube was shaken for 2 days at 30°C. The cells were harvested<br>
I  by centrifugation for 10 min. at 3000 rpm.<br>
DNA was isolated according to WO 94/14953 and dissolved in 50 /il water. The DNA was transformed into E.   coli   by standard procedures. Flasmid DNA was isolated from E. coli   using standard procedures, and analyzed by restriction enzyme analysis. The cDNA insert was excised using appropriate restriction enzymes and ligated into an Aspergillus expression vector.<br>
Transformation of Aspergillus  oryzae  or Aspergillus  niger<br>
Protoplasts may be prepared as described in WO 95/0204 2.. p. 16, line 21 - page 17, line 12, which is hereby incorporated by reference.<br>
100 ^1 of protoplast suspension is mixed with 5-25 ug of the appropriate DNA in 10 ,ul of STC (1.2 M sorbitol, 10 my. Tris-HCl, pK = 7.5, 10 mM CaCl-,) . Protoplasts are mixed with p3SR2 (an A.   nidulans  amdS gene carrying plasmid). The mixture is left at room temperature for 25 minutes. 0.2 ml of 60% PEG 4000 (BDH 29576), 10 ir.M CaCl, and 10 muM Tris-HCl, pH 7.5 is added and carefully mixed (twice) and. finally 0.85 ml of the same solution is added and carefully mixed. The mixture is left at room temperature . f-cy^ 2 5 minutes, spun at 2 500 g for 15 minutes and the pellet is resuspended in 2 ml of 1.2 M sorbitol. After one more sedimentation the protoplasts are spread<br><br>
on mininial plates (Cove, Biochem. Biophys. Acta 113 (1966) 51-56) containing 1.0 M sucrose, pH 7.0, 10 mM acetamide as nitrogen source and 20 mM CsCl to inhibit background growth. After incubation for 4-7 days at 37=C spores are picked and spread for single colonies. This procedure is repeated and spores of a single colony after the second reisolation is stored as a defined transformant.<br>
Test of A.   oryzae  transformants<br>
Each of the transformants were inoculated in 10 ml YPM and propagated. After 2-5 days of incubation at 37°C, 10 ml supernatant was removed. The endoglucanase activity was identified by AZCL  HE cellulose as described above.<br>
Hybridization conditions (to be used in evaluating<br>
property ii) of the DN'A construct of the invention): Suirable ccnditicns for determining hybridizarion between a nucleotide probe and a homologous DNA or RNA sequence involves presoaki.ng of the filter containing the DN.-. fragments or RN.A. to hybridize in 5 x SSC (standard saline citrate) for 10 min, and prehybridization of the filter in a solution of 5 x SSC (Sambrook et al. 1939), 5 x Denhardt's solution (Sambrook et al. 1989), 0:5 % SDS and 100 ^g/ml of denatured sonicated salmon sperm DNA (Sambrook et al. 19S9), followed by hybridization in the same solution containing a random-primed (Feinberg, A. ?. and Vogelstein, B. (19o3) Anal.   Biochem.    132:6-13), 'P-dCTP-labeled (specific activity &gt; 1 x lO' cpm/^g ) probe for 12 hours at ca. 45»C. The filter is then washed two times for 30 minutes in 2 x SSC, 0.5 % SDS at preferably not higher than 50°C, more preferably not higher than 55°C, more preferably not higher than 60°C, more preferably not higher than 65°C,   even more preferably not "higher than 70°C, especially not higher than 75s€-,— The nucleotide probe to be used in the hybridization is the DNA sequence corresponding to the endoglucanase<br><br>
encoding part of the DNA sequence shown in SEQ ID No. l, 4, 6, 8, 10, 12, or 16, resepctively, and/or the DNA sequence obtainable from the plasmid in S. cerevisiae, DSM 9770, DSM 10082, DSM 10030, DSM 10081, E.    coli,    DSM 10512, DSM 10511, DSM 10571 or DSM 10576, respectively.<br>
Immunological cross-reactivity: Antibodies tc be used in determining imrriunological cross-reactivity may be prepared by use of a purified cellulase. More specifically, antiserum against the cellulase of the invention may be raised by immunizing rabbits (or other rodents) according to the procedure described by N. Axelsen ejt a 1. in: A Manual of Quantitative Immunoelec¬trophoresis , Blackvell Scientific Publications, 1973, Chapter 2 3, or A. Johnstone and R. Thorpe, Immunochemistrv in Practice. Blackwell Scientific Publications, 1932 fr.cre specifically pp. 27-21). Purified im-unociooulins may be obtained from the anti-sera, for example by salt precipitation ( (NH/) , SOj) , followed by dialysis and ion exchange c.hromiatography, e.g. on DEAE-Sephadex. IiruEunochemicai characterization cf proteins may be done either by Gutcherlony double-diffusion analysis (O. Cuchterlony in: Handbook of Experimental I~munologv (D.M. Weir, Ed.), Blackwell Scientific Publications, 1967, pp. 655-706), by crossed immunoelectrophoresis (N. .Axelsen et al. , supra, Chapters 3 and 4), or by rocket immiunoelectrophoresis (N. Axelsen et al., Chapter 2).<br>
Media<br>
YPD: 10 g yeast extract, 20 g peptone, H^O to 900 m^l. Autoclaved, 100 ml 20% glucose (sterile filtered) added.<br>
YPM: 10 g yeast extract, 20 g peptone, H^O to 900 ml. Autoclaved-,—;-ao ml 20% maltodextrin (sterile filtered) added.<br><br>
10 X Basal salt: /5 g yeast nitrogen base, 113 g succinic acid, 63 g NaOH, H,0 ad 1000 ml, sterile filtered.<br>
SC-\JPJ\:    100 ml 10 X Basal salt, 28 ml  20% casamino acids 'j     without vitamins, 10 ml 1% tryptophan, H^O ad 900 ml,<br>
autoclaved, 3.6 mi 5% threonine and 100 ml 20% glucose or 20% galactose added.<br>
SC-URA agar: SC-URA, 20 g/1 agar added.<br>
PD agar: 39g potato dextrose agar, DIFCO 0013; add deionized water up to 1000ml; autoclave (12l"C for 15-20 min) .<br>
PC agar: Potatoes and carrots (grinded, 20 g of each) and water, added up to IGOGm.l, are boiled for 1 hr; agar (20g/l of Merck 16:4);  autoclave (12l"C for 20 m.in)<br>
PC liquid broth: as PC agar but without the Agar<br>
?D liquid bro~h: 24g po~ax.o  dextrose broth, Difco 0545, deionized water up to lOOOml; autoclave (121'~C for 15-20 min)<br>
PC and PD liquid broth with cellulose: add 30 g Solcafloc<br>
(Dicacel available from, Dicalite-Eurcpe-Nord, 9000 Gent, Belgium) per lOOOmI<br>
PB-9 liquid broth: 12 g Rofec (Roquette 101-0441)  and 24 g glucose are added to lOOOmI water; pH is adjusted to 5.5; 5ml mineral oil and 5 g CaCo, are added per lOOOr.l. autoclave (121'C for 40 min)<br>
YPG liquid broth: 4g yeast extract (Difco 0127), Ig KK^PO, (Merck4873), 0.5g MgS04.7H20 Merck 5886, 15g Dextrose, Roquette 101-0441, 0.1ml Pluronic (101-3088); deionized water up to lOOOml; autoclave (20min at 121°C)<br><br>
Dilute salt solution (DS): Make up  two stock solutions:<br>
P-stock: 13.61g  KH.POj,- 13.21g (NHJ2P0j, 17.4 2g KH,POj;<br>
deionized water up to 100ml<br>
Ca/Mg stock: 7.35g  CaCl,, 2H2O, 10.17g MgCl,, 6H_,0,<br>
deionized water up to lOOml; pH adjusted to 7.0;<br>
autoclaving (121'C; 2 0min)<br>
Mix O..51T1I P-stock with 0.1ml Ca/Mg stock<br>
add deionized water up to 1000ml<br>
AZCL HE cellulose (Megazyme, Australia).<br>
Uses<br>
During washing and wearing, dyestuff from dyed fabrics cr garment will conventionally bleed from the fabric which then looks faded and worn. Removal of surface fibers from the fabric will partly restore the original colours and looks cf the fabric. By the term "colour clarification", as used herein, is meant the partly restoration cf the initial colours of fabric cr garment throughout m.ultiple washing cycles.<br>
The term "de-pilling" denotes removing of pills fro- the fabric surface.<br>
The term "soaking liquor" denotes an aqueous liquor in which laundry may be immersed prior to being subjected to a conventional washing proccoc. The soaking liquor may contain one or m.ore ingredients conventionally used in a washing or laundering process.<br>
The term, "washing liquor" denotes an aqueous liquor in which laundry is subjected to a washi.ng process, i.e. usually a combined chemical and mechanical action either manually or in a washing machine. Conventionally, the<br><br>
washing liquor is an aqueous solution of a powder or liquid detergent composition.<br>
The term "rinsing liquor" denotes an aqueous liquor in which laundry is immersed and treated, conventionally immediately after being subjected to a washing process, in order to rinse the laundry, i.e. essentially remove the detergent solution from the laundry. The rinsing liquor may contain a fabric conditioning or softening composition.<br>
The laundry subjected to the method of the present inven¬tion may be conventional washable laundry. Preferably, the major part of the laundry is sewn or unsewn fabrics, including knits, wover.s,_ denims, yarns, and toweling,' made from cotton, cotton blends or natural or manmade cellulosics (e.g. originating frr- xylan-containing cel¬lulose fibers s_ch as from wood pulp) or blends rhereoi. Examples of blends are blends of cotton or rayon/viscose with one or -ore companion material such as wool, syn¬thetic fibers (e.g. pclyamide fibers, acrylic fibers, polyester fibers, polyvinyl alcohol fibers, polyvinyl chloride fibers, pclyvinylidene chloride fibers, polyurethane fibers, polyurea fibers, aramid fibers), and cellulose-containing fibers (e.g. rayon/viscose, ramie, flax/linen, jute, cellulose acetate fibers, lyocell).<br>
Detergent Compositions<br>
According to one aspect of the present invention, the present enaoglucanases m.ay typically be componenrs of a detergent composition. As such, rhey may be included in the detergent comiposition in the form of a non-dusting granulate, a stabilized JJ-quid, or protected enzymes. Non-dusting granulates may be produced, e.g., as dis¬closed in US 4,106,991 and 4,661,452 (both to Novo<br><br>
Industri A/S) and may optionally be coated by methods known in the art. Examples of waxy coating materials are poly(ethylene oxide) products (polyethyleneglycol, PEG) with mean molecular weights of 1000 to 20000; ethoxylated nonylphenols having from 16 to 50 ethylene oxide units; ethoxylated fatty alcohols in which the alcohol contains from 12 to 20 carbon aroms and in which there are 15 to 80 ethylene oxide units; fatty alcohols; fatty acids; and mono- and di- and triglycerides of fatty acids. Examples of film-forming coating materials suitable for application by fluid bed techniques are given in patent GB 1483591. Liquid enzyme preparations may, for instance, be stabilized by adding a polyol such as propylene glycol, a sugar or sugar alcohol, lactic acid or boric acid according to established methods. Other enzyme stabilizers are well known in the art. Protected enzymes may be prepared according to the method disclosed in EP<br>
The detergent composition of the invention may be in any convenient form, e.g. as powder, granules, paste or liquid. .-. liquid detergent may be aqueous, typically con¬taining up to 70% water and 0-30% organic solvent, or nonaqueous.<br>
The detergent composition comprises one or m.ore surf¬actants, each of which may be anionic, nonionic, cationic, or zwittericnic. The detergent will usually contain 0-50% of anionic surfactant such as linear alkylbenzenesulfonate (LAS), alpha-olefinsulfonate (AOS), alkyl sulfate (fatry alcohol sulfate) (AS), alcohol ethoxysulfate (.AEOS or AES) , secondary alkanesulfonates (SAS), alpha-sulfo fatty acid methyl esters, alkyl- or alkenylsuccinic acid, or soap. It may also contain 0-40% of nonionic surfaxrtant such as alcohol ethoxylate (AEO or AE), carboxylated alcohol ethoxylates, nonyiphenol ethoxylate, alkylpolyglycoside, alkyldimethylamine oxide.<br><br>
ethoxylated fatty acid monoethanolamide, fatty acid monoethanolamide, or polyhydroxy alkyl fatty acid amide (e.g. as described in WO 92/06154).<br>
The detergent composition may additionally comprise one or more other enzymes such as amylase, lipase, cutinase, protease, peroxidase, and oxidase, e.g. laccase.<br>
The detergent may contain 1-65% of a detergent builder or complexing agent such as zeolite, diphosphate, triphosphate, phosphonate, citrate, nitrilotriacetic acid (NTA), ethylenediaminetetraacetic acid (EDTA), diethylenetriaminepentaacetic acid (DTMPA), alkyl- or =Ikenylsuccinic acid, soluble silicates or layered silicates (e.g. SKS-6 from.. Hoechst) . The detergent may also be unbuilt, i.e. essentially free of detergent JDUiider.<br>
The detergent may comprise one or more polymers. Examiples are carboxymethylcellulose (CMC), poly(vinylpyrrolidone) (?'/?) , polyethyleneglycol (PEG; , pcly(vir.yl alcohol) (PVA), polycarboxylates such as pclyacrylates, maleic/acrylic acid copolymers and lauryl methacrylate/acrylic acid 'copolymers.<br>
The detergent m.ay contain a bleaching system which may comprise a H;0; source such as perborate or percarbonare which -ay be combined with a peracid-forming bleach activator such as tetraacetylethylenediamine (TAED) or nonajnoyloxybenzenesulfonate (NOBS) . Alternatively, the bleaching system may comprise peroxyacids of, e.g., the amide, imide, or sulfone type.<br>
The enzymes of the detergent composition of the invention may be stabilized using convfiai_i.onal stabilizing agents, e.g. a polyol such as propylene glycol or glycerol, a sugar or sugar alcohol, lactic acid, boric acid, or a<br><br>
boric acid derivative such as, e.g., an aromatic borate ester, and the composition may be formulated as described in, e.g., WO 92/19709 and WO 92/19708.<br>
The detergent may also contain other conventional deter¬gent ingredients such as, e.g., fabric conditioners in¬cluding clays, foam boosters, suds suppressors, anti-corrosion agents, soil-suspending agents, anti-soil-redeposit ion agents, dyes, bactericides, optical brighteners, or perfume.<br>
The pH (measured in aqueous solution at use concentra¬tion) will usually be neutral or alkaline, e.g. in the ranee of 7—li.<br>
Particular forms of detergent compositions within the scope of -he invention include:<br>
1}  .A deterge.nt composition formulated as a granulate having a bulk density of at least 600 g/l comprising<br><br>
00 01 - 0.1"<br><br>
2)  A detergent composition formulated as a granulate having a bulk density of at least 600 g/1 comprising<br>
3)  A detergent composition formulated as a granulate having a bulk density of at least 600 g/1 comprising<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
detergent liquid comprising a liquid nonionic surfactant such as, e.g., linear alkoxylated primary alcohol, a builder system (e.g. phosphate), enzyme and alkali  The detergent may also corr.prise anionic surfactant and/or a bleach system.<br>
The endoglucanase may be incorporated in concentrations conventionally employed in detergents. It is at present contemplated that, in the laundry composition of the in¬vention, the cellulase may be added in an amount corre¬sponding to 0.0001-10 mg (calculated as pure enzyme protein) of cellulase per liter of wash liquor.<br>
According to yet another aspect of the present invention, e.ndoglucanase may typically be a component of a  conditioning or softener composition. Examples of   softener co-positions are disclosed in e.g. EP 0<br>
textile applications<br>
In another embodiment, the present invention relates to use of the endoglucanase of the invention in the bio-polishing process. Bio-Polishing is a specific treatment of the yarn surface which improves fabric quality with respect to handle and appearance without loss of fabric wet ability.  The most important effects of Bio-Polishing can be characterized by less fuzz and pilling, increased gloss/luster, improved fabric handle, increased durable softness and altered water absorbency. Bio-Polishing usually takes place in the wet processing of the manufacture of knitted and woven fabrics.  Wet processing comprises such steps as e.g. desisting, scouring, bleaching, washing, dying/printing and finishing.  During each of these steps, the fabric is more or less subjected to mechanical action. In general, after the textiles have<br><br>
been knitted or woven, the fabric proceeds to a desizing stage, followed by a scouring stage, etc.  Desizing is the act of removing size from textiles.  Prior to weaving on mechanical looms, warp yarns are often coated with size starch or starch derivatives in order to increase their tensile strength.  After weaving, the size coating must be removed before further processing the fabric in order to ensure a homogeneous and wash-proof result. It is known that in order to achieve the effects of Bio-Polishing, a combination of cellulytic and mechanical action is required.  It is also known that "super-softness" is achievable when the treatment with a cellulase is combined with a conventional treatment with softening agents. It is contemplated that use of the endoglucanase of . the invention for bio-polishing of -cellulosic fabrics is advantageous, e.g. a more thorough polishing can be achieved. Bio-polishing may be obtained cy applying zhe -ernod described e.g. in WO S3/'2027£.<br>
Stone-washing<br>
It is known to provide a "stone-washed" look (localized abrasion of zhe  colour) in dyed fabric, especially in denim fabric or jeans, either by washing the denim or jeans made from such fabric in the presence of pumice stones to provide the desired localized lightening of the colour of the fabric or by treating the fabric enzymatic ally, in particular with cellulytic enzymes. The treatment with an endoglucanase of the present invention may be carried out either alone such as disclosed in US 4,832,864, together with a smaller amount of pumice than required in the traditional process, or together with prelate such as disclosed in WO 95/09225.<br><br>
Pulp and paper applications<br>
In the papermaking pulp industry, the endoglucanase of the present invention may be applied advantageously e.g. as follows:<br>
-	For debarking: pretreatraent with the endoglucanase may degrade the cambium layer prior to debarking in mechanical drums resulting in advantageous energy savings.<br>
-	For defibration: treatment of a material containing cellulosic fibers with the endoglucanase prior to refin¬ing or beating may result in reduction of the energy consumption due to the hydrolysing effect of the cellulase on the interfere surfaces. Use of the endoglucanase may result in improved energy savings as compared to the use of known enzymes, since i~ is believed that the enzyme composition of the invention may possess a higher ability to penetrate fibre walls.<br>
-	For fibre modification, i.e. improvemenr of fibre prop¬erties where partial hydrolysis across the fibre wall is needed which  requires deeper penetrating enzymes (e.g. in order to make coarse fibers more flexible). Deep treatment of fibers has so far not been possible for high yield pulps e.g. mechanical pulps or mixtures of recycled pulps. This has been ascribed to the nature of the fibre wall structure that prevents the passage of enzyme molecules due to physical restriction of the pore matrix of the fibre wall. It is contemplated that the present endoglucanase is capable of penetrating into the fibre wall.<br>
-	For drainage improvement. The  — papermaking pulps may be improved by treatment of the pulp with hydrolysing enzymes, e.g. cellulose. Use of<br><br>
the present endoglucanase may be more effective, e.g. result in a higher degree of loosening bundles of strongly hydrated micro-fibrils in the fines fraction (consisting of fibre debris) that limits the rate of drainage by blocking hollow spaces between fibers and in the wire mesh of the paper machine. The Canadian standard freeness (CSF) increases and the Schopper-Riegler drainage index decreases when pulp in subjected to cellulase treatment, see e.g. US patent 4,923,565; TAPPI T227, SCAN 019:65.ence.<br>
-	For inter fibre bonding. Hydrolytic enzymes are applied<br>
in the manufacture of papermaking pulps for improving the<br>
inter fibre bending. The enzymes rinse the fibre surfaces<br>
for impurities e.g. cellulosic debris, thus enhancing the<br>
area of exposed cellulose with attachment to the fibre<br>
wall, thus improving the fibre-to-fibre hydrogen binding<br>
capaci'cv. This -recess is also referred tr as<br>
dehcrnification. Paper and board produced wirh a cellulase containing enzyme preparation r.ay have an improved strength or a reduced grammage,, a smoother surface and an improved printability.<br>
-	For enzymatic deinking. Partial hydrolysis of recycled<br>
paper during or upon pulping by use of hydrolysing<br>
enzymes such as cellulases are known to facilitate the<br>
removal and agglomeration of ink particles. Use of the<br>
present endoglucanse may give a more effective loosening<br>
of ink from the surface structure due to a better pen¬<br>
etration of the enzyme molecules into the fibrillar<br>
matrix of the fibre wall, thus softening the surface<br>
whereby ink particles are effectively loosened. The<br>
agglomeration of loosened ink particles are also<br>
improved, due to a more efficient hydrolysis of cellulo-<br>
sic fragments found attached to ink particles or-i-ginating<br>
from the fibres.<br><br>
The treatment of lignocellulosic pulp may, e.g., be performed as described in WO 91/14819, WO 91/14822, WO 92/17573 and WO 92/18688.<br>
Degradation of plant material<br>
In yet another embodiment, the present invention relates to use of the endoglucanase and/or enzyme preparation according to the invention for degradation of plant material e.g. cell walls.<br>
It is contemplated that the novel endoglucanase and/or enzyme preparation of the invention is useful in the pre¬paration of wine, fruit or vegetable juice in order to increase yield. Endoglucanases according to the invention may also be applied for enzymatic hydrolysis of various plant cell-wall derived materials cr waste materials, e.g. agricultural residues such as wheat-straw, corn cobs, whole corn plants, nut shells, grass, vegetable hulls, bean .hulls, spent grains, sugar beet pulp, and the like. The plant marerial may be degraded in order to improve different kinds of processing, facilitate purification or extraction of other components like purification of beta-glucan or beta-glucan oligomers from cereals, improve the feed value, decrease the water binding capacity, improve the degradability in waste v.'ater plants, improve the conversion of e.g. grass and corn to ensilage, etc.<br>
The following examples illustrate the invention<br><br>
EXAMPLE 1<br>
Cellulytic enzymes from 4 fungi, belonging to 3 families under two orders within the Ascomycetes were detected by expression cloning; corresponding DNA sequences were determined; the enzymes heterologously expressed, and produced by liquid fermentation, characterized and demonstrated to give good performance in colour clarifi¬cation assays.<br>
Isolate CBS 117.65, CBS 478.94, NRRL 8126, and ATCC 10523 were grown in shake flask cultures on cellulose enriched potato dextrose broth, incubated for 5 days at 2 6"c (shaking conditions, 150 rpm).<br>
A. Cloning and expression of an endoglucanase from Myceliophthora   tbermophila,   Acremonlum  sp..   and Thielavia terrssTiris  and Volutella  collstotrichcldes<br>
mRNA was isolated from Myceliophthora   thermophila, Acremcnium  sp.,   Thielavia  terrestris   and"'/olu tell a colletotrichoides,   respectively, crcwTi in a cellulose-containing fermentation medium with agitation to ensure sufficient aeration. Mycelia were harvested after 3-5 days' growth, immediately frozen in liquid nitrogen and stored at -80°C. Libraries from y.yceliophthora thermophila,   Acreiuohium sp.,   Thielavia   terrestris   and Volutella   colletotrichoides,   respectively, each consisting of approx. 10* individual clones were constructed i.n E.   coli   as described with a vector background of 1%.<br>
Plasmid DNA from some of the pools from each library was transformed into yeast, and 50-100 plates containing 250-400 yeast colonies were obtained—^om each pool.<br>
Endoglucanase-positive colonies were identified and<br><br>
isolated on SC-agar plates with the AZCL HE cellulose assay. cDNA inserts were amplified directly from the yeast colonies and characterized as described in the Materials and Methods section above.<br>
The DNA sequence of the cDNA encoding the endoglucanase from Myceliophthora   thermophila   is shown in SEQ ID No. l and the corresponding amino acid sequence is also shown in SEQ ID No. 1. The cDNA is obtainable from the plasmid in DSM 9770.<br>
The DNA sequence of the cDNA encoding the endoglucanase from Acremonium sp.   is shown in SEQ ID No. 4 and the corresponding amino acid sequence is shown in SEQ ID No. 5. The cDN.A is obtainable from^ the pl3sm,id in DSM 10082.<br>
The DNA sequence of the cDKA encoding the endoglucanase rrorT; __\nej.5V'i3 ^err^s^rxs is snovn m ^r-v XLJ   HO. O anc the corresponding amino acid sequence is shown in SEQ ID No. 9. The cDNA is obtainable from the plasmid in DSM 10031.'<br>
The DNA sequence of the cDNA encoding the endoglucanase from Volutella  colletotrichoides   is shown in SEQ ID No. 16 and the corresponding amino acid sequence is shown in SEQ ID No. 17. The cDNA is obtainable from the plasmid in DSM 10571.<br>
Total DNA was isolated from a yeast colony and plasmid DNA was rescued by transformation of E.   coli   as described above. In order to express the endoglucanases in Aspergillus,   the D.N'A was digested with appropriate restriction enzymes, size fractionated on gel, and a fragment corresponding to the endoglucanase gene from<br>
	Hyceliophthora   thermophila,   Acrewdnium  sp.,   ThTWlavia<br>
terrestris  and Volutella  colletotrichoides,   respectively, was purified. The genes were subsequently ligated to<br><br>
pHD414, digested with appropriate restriction enzymes, resulting in the plasmids pA2Cl93, pA2C357, pA2C385 and pA2C488, respectively.<br>
After amplification of the DNA in E.   coli   the plasmids were transformed into Aspergillus  oryzae   as described above.<br>
Test of A. oryzae  transformants<br>
Each of the transformants were tested for endoglucanase activity as described above. Some of the transformants had endoglucanase activity which was significantly larger than the Aspergillus  oryzae  background. This demonstrates efficient expression of the eridoglucanases in Aspergillus oryzae.   The transformants with the highest endoglucanase activity were selected and inoculated in a 500 ml shake flask vith YPM r.edia. .After 3-5 days of fer-entation with sufficient agira~io- to ensure good aerarion, "he culture broth was centrifuged for 10 minutes at 20C0 g and the suoernatant recovered.<br>
B. Determination of endoglucanase activity<br>
The cellulytic activity of the endoglucanase may be determined relative to an analytical standard and expressed in the unit S-CEVU.<br>
Cellulytic enzymes hydrolyse CMC, thereby decreasing the viscosity of the incubation mixture. The resulting reduction in viscosity may be determined by a vibration viscosimeter (e.g. MIVI 3000 from Sofraser, France).<br>
Determination of the cellulytic activity, measured in terms of S-CEVU, may be determined according to the anal^Lsis method AF 301.1 which is available fro- the-Applicant upon request.<br><br>
The S-CEV^U assay quantifies the amount of catalytic activity present in the sample by measuring the ability of the sample to reduce the viscosity of a solution of carboxy-methylcellulose (CMC). The assay is carried out at 40°C, pH 7.5 using a relative enzyme standard for reducing the viscosity of the CMC substrate.<br>
Assay for determination of endoglucanase activity in terms of SAVI units using phosphoric-acid swollen cellu¬lose (PASC):<br>
Definition:<br>
1	SAVI-U is the amount of enzyme which form.s an amount of<br>
reducing carbohydrates equivalent to 1 umol of glucose<br>
per minute.<br>
Assay condition:<br>
Enzyme solution: 0,5 ml<br>
4 g/1 PASC in 0,1 M Buffer: 2.0 ml<br>
2	0 m,i.n, 40 ■ °C<br>
Sensitivity:<br>
Max   0.1   5AVIU/ml   =     approx.    1   S-CE\nj/mi    (CMC  viscosity)<br>
Min   0.01   SAVIU/ml  =  approx.   0.1   S-CE'vTO/ml<br>
Determination of for.~ation of reducing sugars:<br>
The reducing groups assay was performed according to Lever, M. A new reaction for colormetric determination of carbohydrates. Anal. Biochem, 1972. Vol 47 (273-279). Reagent mixture was  prepared by mixing 1,5 gram, p-hydroxybenzoic-acide hydracide (PHBAH) with 5 gram sodium tartrate in IGO ml 2 % sodium hydroxide.<br>
Substrate:<br>
PASC stock solution was prepared the following way using<br><br>
ice cold acetone and phosphoric acid. 5 gram of cellulose (Avicel*) was moistered with water, and 150 ml ice cold 85% ortho-phosphoric acid was added. The mixture was placed in ice bath under slow stirring for 1 hr. Then 100 ml ice cold acetone was added with stirring. The slurry was transferred to a Buchner filter with pyrex sintered disc number 3 and then washed three times with 100 ml ice cold acetone, and sucked as dry as possible after each wash. Finally, the filter cake was washed twice with 500 ml water, sucked as dry as possible after each wash. The PASC was mixed with deionized water to a total volume of 300 ml, blended to homogeneity (using the Ultra Turrax Homogenizer) and stored in refrigerator (up to one month).<br>
Substrate equilibration with buffer: 20 gram phosphoric acid swollen cellulose PASC stock solution was centrifu-ged for 20 r.in ar 5CC0 rpm. , rhe superna-an- was poured of; the sediment was resuspended in 30 ml of buffer and centrifuged for 20 -in. at 5000 rpm., the supernatant was poured of, and the sediment was resu^spended i.n buffer io a total of 60 g corresponding to a subsrrate concentra¬tion of 5 g cellulose/litre.<br>
Buffer for pH S,5 determination: 0.1 M Barbital. Buffer for pH 10 determination: 0.1 M Glycine.<br>
1.	Dilution of enzyme samples<br>
The enzyme solution is diluted in the same buffer as the substrate.<br>
2.	Enzyme reaction<br>
The substrate in buffer solution is preheated for 5 min.<br>
at 40*C  (2 ml).		<br>
Then the enzyme solution (diluted to between 0.2 and 1 S-CEVU/ml) 0,5 ml is added and mixed for 5 sec. Enzymes<br><br>
blanks are obtained by adding the stop reagent before enzyme solution. Incubate for 20 min. at 40 'c. The reaction is stopped by adding 0.5 ml 2% NaOH solution and mixing for 5 sec.<br>
The samples are centrifuged for 20 min. at 5000 rpm. 1 ml supernatant is mixed with 0.5 ml PHBAH reagent and boiled for 10 min. The test tubes are cooled in a ice water bath.<br>
3.	Determination of reducing end groups:<br>
The absorbancy at 410 nm is measured usi.ng a spectrophotom.eter. Blanks are prepared by adding sodium hydroxide before adding enzyme solution. k  standard glucose curve was obtained by using glucose ccncentraticns of 5, 10, 15 and 25 mg/1 in the sane buffer and addin^ ?HH.-vH reacent before boilinc. The release of reducing glucose equivalent is calculated using this standard curve.<br>
4.	Calculation cf catalytic activity:<br>
Measure absorbance at 410 nm<br>
1)	Standard curve<br>
(Glucose)  -  (HpO)   vs    concentration of glucose<br>
2)	Enzyme sample<br>
(Sample) - (Blank)<br>
Calculate glucose concentration according to a standard<br><br>
Activity (SAVIU/ml):<br>
X (mg glucose/1) * Dilution<br>
ISO.16 (MW of glucose) * 20 (min)<br>
C. Purification and characterisation of the endoglucanase from M.   thermophila<br>
Aspergillus  oryzae  transformed with pA2C193 was grown on YPM medium for 4 days. The liquid was then centrifuged and sterile filtered.<br>
-The.sample was.concentrated by ultrafilrration -on AMICON cells using a DOW membrane GR61P? with cut-off 20 kD. The Uf-concentrate was analyzed for S-CE'v'U/ml and SaviL'/mi<br>
'—■ ^ ^ '^- V* X ;<br><br>
UF-concentrate    \  S-<br>
i<br>
; CE'.'TF/ml	Saviu/ml<br>
9.25 ml           1  570<br>
1	41<br>
Purification:<br>
2 ml of the UF-concentrate was diluted 5 times to lower the ionic strength and filtered through 0.2 2 um disk filter. This sample was applied to a Mono Q* HR5/5 Phar¬macia column, equilibrated with 50 mM Tris/HCl buffer. pH 7.5, (buffer A) and a flow of 1 ml/min. After wash to baseline, with buffer h,   the colum.n was eluted with a Tris/HCl buffer, pK 7.5, containing 1 M IJaCl (buffer B) , the elation gradient was 0-50% buffer B in 1 hour.<br>
After 36 min. a peak complex showed up, 1 ml fractions were picked up and the first 10 fractions showed cellulase activity on CMC/Agarose/congo-red plates.<br><br>
These fractions were pooled and concentrated, by ultrafiltration on AMICON cells using a DOW membrane GR61PP with cut-off 20 kD, to 3 ml.<br>
This sample was applied to a HiLoad 26/60 Superdex 75"" prep grade Pharmacia column, equilibrated with 100 mM Na-Acetate buffer, pH 6.35, and a 1 ml/min flow.<br>
After 82 min. a peak showed up, 1 ml fractions were picked up and.the first 10 fractions showed cellulase activity on CMC/Agarose/congo-red plates.<br>
These fractions were pooled and the following results were obtained: A;so='^- 15, A2so/A26o=l-62 Mw(5DS:=2 2 kD pl=3.5-5<br>
Purity on SDS-PAGE =100% S-CEVU/ml=28.5 S-CEVU/A:so=lS3 S-CEV~u/mg=4 3 6<br>
Extinction ccefficient=54880 (calculated) •Mw(calculated)=22 kD<br>
The Extinction coefficient is based on the content of tyrosine, tryptophane and cystein calculated fro- the sequence cf the enclcsad SEQ ID No. 1 (the amino acid sequence). SDS-Page was performed on NOVEX Pre-Cast Gels 4-20% Tris-Glycine Gel 1.0 mm x 10 Well<br>
lEF was performed on Pharmacia FAGplate pH 3.5 - 9.5, the activity was visualized by CMC-Congored overlaying.<br>
DPtPrmination of K^, &amp; k^..,:<br>
k„ and k„,  was determined in the same manner as the<br>
determination of SAVI Units at pH 8.5 with a substrate<br><br>
concentration   up  to  8   g/1.<br>
The   following   results   were  obtained:<br>
k^.i  3 8   per   sec.<br>
&gt;v„,   5   g/1,<br>
phosporic acid swollen cellulose, pH S<br>
Specific activity on CMC at pH 7.5: 435 S-CEVU per mg protein.<br>
D. Determination of pH and temperature profile of the endoglucanase from M. thermophila<br>
The pK profile was determined at the following conditions:<br>
mixing G. IM Tri-sodium phosphate with O.IM cimc acid. Purified -endoglucanase was diluted to ensure rhe assay response to be within the linear range of the assay. The substrate was a 0.4% suspension of AZCL-HZ-cellulcse (MegaZyme) mixed 1:1 with the citrate/phosphate buffer to a final substrate concentration of 0.2% AZCL-HE-cellulose. 1 ml substrate in Eppendorf® 1.5-i polypropylene tubes were added 10 p.1   of enzyme solution and incubated for 15 minutes in Eppendorf^' temperature controlled Thermomixers before heat-inactivation of enzymes for 2U minutes at 95°C in a separate Thermomixer. The tubes were centrirugea and 200 /il of eacn supernatant was transferred to a well in a 96 well micrctiter plate and OD was measured at 620nm in an ELISA reader (Labsystems Multiskan® MCC/340).<br>
For the pH optiraumi incubations took place an 30°C. For each pH value, three tubes were added enzyme and<br><br>
incubated before heat-inactivation, whereas one tube (the blank) was added enzyme and heat-inactivated immediately. The mean value of the three incubated samples was calculated and the blank value was substracted.<br>
The following pH profile was determined:<br><br>
pH<br><br>
Relative Activity<br><br><br><br>
2.5<br><br>
:10%<br><br>
■'10%<br>
3 . 5<br>
4 . 5<br><br>
5	100%             1<br>
S	<br>
6.5	S6%<br>
7	7S%<br>
7.5	<br>
8	68%<br>
S. 5	= ■ 3-_/ -t   o<br>
9	31%<br>
10	lO-S<br>
It is seen that the endoglucanase has more than 60% activity between pH 4.0 and 8.0 and optimal activity at pH 5.0-6.0.   ~<br>
Temperature profile:<br><br>
The temperature optimum was determined in the same manner at pH 5.5. The temperatures ranged from 30^0  to 80°C. For each temperature three incubations were carried out and the mean calculated. Three blanks were produced by immediate heat-inactivation of enzyme and the mean was subtracted from the incubated sample values.<br>
It is seen that the endoglucanase has optimal activity at 50-70°C.<br><br>
Temp. {°C)	30	40	50	60	70	80<br>
Relative<br>
Activity	7 4^0	77%	99%	100%	93%	62%<br>
The temperature stability was determined in the same manner at pH 5.5 and 3 0°C, and, further, the enzyme solu¬tions were preneate'.i _or 1 hour at une accuaj. cemperatur^ and cooled on ice. The residual activity is shown below in % of the activitv of a non-oreheated enzvme samnle:<br><br>
	40	50	60	70	80<br>
Relative Activity	95%	84%	92%	86%	24%<br><br>
E. Color clarification of Myceliophthora cellulase (SEQ ID No. 1) measured as removal of surface fibrils and fibers protruding from the yarn of a textile containing cellulosic fibers<br><br>
Apparatus<br>
Liquid volume<br>
Agitation<br>
stirrer<br>
Rinse time<br>
Washing temp<br>
Washing liqour<br>
pH<br>
Washing time<br>
Repetitions<br>
Enzymes<br>
Dosage<br>
Texrile<br>
Drying Evaluation<br><br>
Terg-o-tometer<br>
100 rol<br>
150 movements/min with vertical<br>
5 min in tapwater 40°<br>
0.05 M phosphate buffer 7.0<br>
3 0 nin 2<br>
Myceliophthora SEQ ID N'o. IB 500 and 2500 S-CEVU/1<br>
2 swatches cf aged clack 100% ccrrcn 5x6 cm (0.9 gram) Tumble dry<br>
The light re-issicn is -easured by a Datacolcr Elrepho Remission spectrophotometer. Rem.issicn is calculated as delta L (Hunter Lab-values) . When the surface fibrils and fibers protruding from the yarn are removed by the cellulase, the surface of the black fabric appears darker, and lower L values are obtained.<br><br>
The sample is compared with a blind sample, i.e. washed without enzyme:<br>
No cellulase   500 ECU/1 2500 ECU/i<br>
0.00	-1.41     -1.3-3.—<br>
Delta L-values compared to blind sample.<br><br>
The data shows that Mvceliophthora  cellulase without CBD gives good color clarification under the conditions tested.<br>
F. Construction of the gene fusions between the endoglucanase from Myceliophthora thermophila and the 43k.D endoglucanase from Humicola insolens<br>
The purpose of the two constructions was to make derivatives of the endoglucanase from M. thermophila with the linker and CBD from the 43kD endoglucanase from H. insolens   (disclosed in WO 91/17243). The native endoglucanase from M.   thermophila  do not have a linker and/or a cellulose binding domain, CBD.<br>
CMl: Construction 1 consist of the endoglucanase from .M. thermophila (225 amino acids) and the 72 C-ter~inai amiinc acids from the H. insolens 43kD endoglucanase.<br>
CM2: Construction 2 consist of the endoglucanase from .M. thermophila (225 amino acids) and the S3 C-terminal amino acids from the H. insolens 43kD endoglucanase.<br>
The 43kD endoglucanase cDNA from H. insolens was cloned into prtD414 in such a way that the endoglucanase gene was transcribed from the Taka-promoter. The resulting plasm.id v;as nam.ed pCaHj4 1S.<br>
In a similar way the cDNA encoding the endoglucanase from M.   thermophila  was cloned into pHD414 and the resulting plasm.id was named pA2C193.<br><br>
Primers:<br>
primer 1: 5'-<br>
CGGAGCTCACGTCCAAGAGCGGCTGCTCCCGTCCCTCCAGCAGCACCAGCTCTCCGG<br>
-3'<br>
primer 2: 5'<br>
CCGGAGAGCTGGTGCTGCTGGAGGGACGGGAGCAGCCGCTCTTGGACGTGAGCTCCG<br>
-3 '<br>
primer 3: 5'-<br>
CGGAGCTCACGTCCAAGAGCGGCTGCTCCCGTAACGACGACGGCAACTTCCCTGCCG<br>
-3'<br>
primer 4: 5'-<br>
CGGCAGGGiiJvGTTGCCGTCGTCGTTACGGGAGCAGCCGCTCTTGC-ACGTGAGCTCCG<br>
-3 '<br>
Taka-pro. primer:  5' CAACATCACATCAAGCTCTCC -2'<br>
AMG-term. primer:  5' CCCCATCCTTTAACTATAGCG -3'<br>
The endoglucanase fusions were constructed by the PCR overlap-extension method as described by Higuchi et al. 1988.<br>
Construction 1:<br>
Reaction A: The Polymerase Chain Reaction (PCR) was used<br>
to amplify the fragment of pCaKj418 between primer 1 and<br>
;i&gt;IG-term. primer (the linker and CBD from the 4 3kD<br>
endoglucanase from H.insolens).<br>
Reaction B: PCR amplification of the fragment between<br>
Taka-pro. primer and primer 2 in pA2C193, the<br>
endoglucanase gene from M.thermophila.<br>
Reaction C: The two purified fragments were used in a<br>
third PCR in the presence of the primers flanking the<br>
total region, i.e. Taka-pro. primer and AMG-term. primer.<br>
Cons^ifttction   2 :<br>
The same procedure was used where primer 3 and primer 4<br><br>
had replaced respectively primer 1 and primer 2.<br>
The fragment amplified in reaction C was purified, digested with restriction enzymes Xba I and BsstE II. The purified digested fragment was ligated into pA2C193 digested with restriction enzymes Xba I and BsstE II.<br>
Competent cells from E. coli strain DH5aF' (New England Biolabs.) were transformed with the ligated plasmid and colonies containing the gene fusion were isolated. The sequence of the cloned part was verified by DNA sequencing.<br>
The sequence of the gene in the two constructs are shown in SEQ ID No. 2A and SEQ ID No. 3A.<br>
Polymerase Chain Reactions were carried out under standard conditiicns, as reccmmended by Perkin-Elmer.<br>
Reaction A and B started with 2 min. at 94°C followed by 20 cycles of {30 sec. at 94-C, 30 sec. at 50°C and 1 min. at 72°C) and end with 4 min. at 72 -C.<br>
Reaction C started with (2 min. at 94=C, 1 min. at 52°C and 2 min. at 72°C), followed by 15 cycles of (30 sec. at 94°C, 30 sec. at 52°C and 90 sec. at 72°C) and end with 4 min. at 72°C.<br>
The two constructs were transformed into Aspergillus ory-zae as described above.<br>
G. Purification and characterisation of cloned cellulases with cellulose binding domains:<br>
The cloned product is recovered after fermentation by separation of the extracellular fluid from the production<br><br>
organism.<br>
About one gram of cellulase is then highly purified by affinity chromatography using 150 gram of Avicel in a slurry with 20 mm Sodium- phosphate pH 7.5.<br>
The Avicel is mixed with the crude fermentation broth which contain total about 1 gram of cellulase. .After mixing at 4 C for 20 rain the Avicel enzyme is packed into a column with a dimension of 50 times 200 mm about 400 ml total.<br>
The column is washed with the 200 ml buffer, then washed with 0.5 M NaCl in the same buffer until no more protein elutes. Then washed with 500 ml 20 mir. Tris pH 8.5. Finally the pure full length enzyme is eluted with 1% triethylainine pH 11.8.<br>
The eiured enzy~e  solution is adjusted ro pH S and concentrated using a Amicon cell unit with a membrane DOW GR61?? (polypropylene with a cut off of 20 KD) to above 5<br>
The purified cellulases were characterised as follow:<br>
Mw     pi     Molar E.280    S-CEVU per<br>
SDS-PAGE	A.280<br>
Myceliophthora<br><br><br><br>
/4.950	13 5<br><br>
Acremonium<br>
(SEQ ID No.5)  40 kD     5    68.020	185<br>
Thielavia<br>
(SEC ID No.S)  3 5 kD     4.3  52.4 70	~5<br><br>
instructions.<br>
Two of the cellulases were blocked, this is due to the N-terminal glutamine which form a pyroglutamate which can not be detected and which block for further sequencing. DSC Differential scanning calometry was done at neutral pH (7.0) using a MicroCalc Inc. MC calorimeter with a constant scan rate and raising the temperature from 20 to 90° at a rate of 90° per hour.<br>
Raising antibody. The cellulases from Myceliophthora, Acremonium and Thielavia were used for raising antibody in rabits. 0.1 mg of the purified cellulase in 0.9 % NaCl solution mixed with Freunds adjuvant immediately prior to injection. The rabits were immunized 10 times with one week interval. The immunoglobulin G fraction (IgG) was purified by a-r.cnium sulfate precipitation (25% saturation), the precipitate was solubiiized in water and then dialyzed extensively against sodium acetate buffer (pH 5.0, 50 m-M) altering with deionized water. .A.fter filtration, the IgG fraction was stabilized with sodium" azide (0.01%) .<br>
Using immunodiffusion in agar plates all three cellulases form a single immunoprecipitate with its homologous antiserum and no precipitate was seen between the 3 cloned cellulases and the sera raised against the other two cellulases.<br><br>
H-I. Performance of endoglucanase of construction 1 (SEQ ID No. 2) measured in buffer as removal of surface fibrils and fibers protruding from the yarn of a textile containing cellulosic fibers<br><br>
Apparatus<br>
Liquid volume<br>
Agitation<br>
Rinse time<br>
Washing temp<br>
Water Hardness<br>
Washing liquor<br>
PH<br>
Washing rime<br>
Repetitions<br>
Textile<br>
Drying<br><br>
Terg-o-tometer<br>
100 ml<br>
150 movements/min (rpm)<br>
5 min in tap water<br>
40°C<br>
1	mM CaCl,<br>
0.05 M phosphate buffer 7 .0<br>
30 min<br>
2	<br>
2	swatches of aged black,<br>
100% cotton 5x6 c-<br>
Tumfcle dry<br><br>
Evaluation:<br>
The light "remission was measured by a Macberh Color Eye 7000 Remission spectrophotometer. Remission is calculated as delta L (Hunter Lab-values). When the surface fibrils and fibers protruding from the yarn were removed by the cellulase, the surface appeared more bright, and lower L values were obtained.<br>
Results:<br><br>
		2 50       1       1000<br>
Inventive enzyme	0	-1.4	-1. 6<br>
	,<br>
The data show that the enzyme of the invention gives very<br>
good'color clarification under the conditions tested<br><br>
a-II. Performance of cloned endoglucanase from Thielavia terrestris   (SEQ ID No.9) in buffer measured as removal of surface fibrils and fibers protruding from the yarn of a textile containing cellulosic fibers<br><br>
Apparatus Liquid voluine Agitation<br>
Rinse time<br>
Washing temp<br>
Washing liqour<br>
pH<br>
Washing  time<br>
Repetitions<br>
Textile<br>
Terg-o-tometer<br>
100  ml<br>
150   movements/min   with  vertical<br>
stirrer<br>
10 min in tapwater<br>
40°<br>
0.05 M phosphate buffer.<br>
7.0<br>
30 min<br>
2<br>
2 swatches of aged black cotton 5x6<br>
cm (app. 150 g/m2)<br>
:Lva j.uation<br>
Tumble dry<br>
~oior Elrepho<br>
Remission- spectrophotometer. Remission is calculated as delta L (Hunter Lab-values). When the surface fibrils and fibers protruding from the yarn are removed by the cellulase, the surface of the black fabric appears darker and nicer, and lower L values are obtained.<br>
Results<br><br>
S-CEVU/1	0	50	2 00<br>
Inventive enzyme	0	-0.56±0.10	-1.3210.06<br>
The data show that the cellulase gives good color cLaxification under the conditions tested.<br><br>
H-III. Performance of endoglucanase of Volutella colletrichoides   (SEQ ID No. 17) measured in buffer as removal of surface fibrils and fibers protruding from the yarn of a textile containing cellulosic fibers<br><br>
Apparatus Liquid volume Agitation<br>
Rinse time<br>
Washing temp<br>
Washing liqour<br>
pH<br>
Washing time<br>
Repetitions<br>
Dosage<br>
Textile<br>
Terg-o-tometer<br>
100 ml<br>
150 movements/min with vertical<br>
stirrer<br>
5 min in tapwater<br>
40°<br>
0.05 M phosphate buffer<br>
7.0<br>
30 min<br>
2<br>
2.5 S-CEVU/ml<br>
2 swatches of aged black 100%<br>
5x5 cm (0.5 gra.~)<br>
Drying	:    Tumble dry<br>
Evaluation:<br>
The light remission is measured by a Datacoior Eirephc Remission spectrophotometer. Remission is calculated as delta L (Hunter Lab-values). When the surface fibrils and fibers protruding from the yarn are removed by the cellulase, the surface of the black fabric appears darker, and lower L values are obtained.<br>
The sample is compared with a blind sample, i.e. washed without enzyme:<br>
No cellulase       With cellulase<br>
O.CO	-0.57<br>
Delta L remission values compared to blind sample.<br>
The data~"STTows that the Volutella  colletrichoides cellulase gives good color clarification under the conditions tested.<br><br>
H-IV. Performance of cloned cellulases from Thielavia terrestris  and Acremonium sp.   CBS 4 7 8.94 in high pH heavy duty detergent measured as removal of surface fibrils and fibers protruding from the yarn of a textile containing cellulosic fibers<br><br>
Apparatus Liquid volume Agitation<br>
Rinse time Washing temp Washing liqour<br>
Terg-o-toineter<br>
150   ml<br>
150  movements/min with  vertical<br>
stirrer<br>
10 min in tapwater<br>
35'C<br>
l.D g/1 US type KDG<br>
(zeolite/soda built, anionic/ncnionic<br>
weight ratio &gt; 2.5)<br>
PH<br>
Hardness<br>
Washing time<br>
Repetitions<br>
Textile<br>
10. 0<br>
1.0 sM CaCl2<br>
■-'.—' T   iil^  -<br>
jn -. &gt;:6<br>
2   swatches  of  aged   black   co:<br>
Cm   (app.   150  g/rri2)<br>
2 swatches of heavy knitted co~zor.<br>
5x6 cm (app. 600 g/in2)<br>
Drying	:    Tumble dry<br>
Evaluation :<br>
The light remission is measured by a Datacolor Elrepho Remission spectrophotometer. Remission is caiculared as delta L (Hunter Lab-values). When the surface fibrils and fibcLti pruLruding from the yarn are removed by the cellulase, the surface of the black fabric appears darker and nicer, and lower L values are obtained. Different dosages of cloned cellulases from Thielavia terrestris (SEQ ID No. 9) and Acremonium  sp.   CBS 478.94 (SEQ ID No.5), respectively, (denoted A and B, respectively) were tested.<br><br>
Jesuits:<br><br>
S-CEVU/1	0	500	2000<br>
A	0	-2.09 ±0.22	-2 .36 ±0 . 19<br>
B	0	-0. 60 ±0.36	-1.96 ±0. 23<br>
The data show that both cellulases gives good color clarification under the conditions tested.<br>
K-V. Performance of cellulases cloned from Tbielavia terrestris  and Acremonium sp.   CBS 47S.94, and construction 1 (SEQ ID No.2) measured as removal of surface fibrils and fibers protruding from the yarn of a<br>
textile containing cellulosic fibers<br><br>
.ppc<br>
Liauid volume<br>
Rinse time Washing temp Hardness<br>
Washing liqour<br>
Washing time<br>
Repetitions<br>
Textile<br><br>
Terg-o-tometer 150 ml<br>
150 mcvemencs/ir.in virh vertical stirrer<br>
10 min in tapwater 35°C<br>
1.0 mM CaCl2 0.3 4 mM MgC12<br>
2.0 g/1 HDL {neutral, citrate built HDL, with nonionic/anionic weight ra¬tion &gt;   0.5) 7 . 5<br>
3 0 min 2 2 swatches of aged black cotton 5x6<br>
cm (app. I5a g/m2}    , 	<br>
2 swatches of heavy knitted cotton 4x7 cm (app. 600 g/m2)<br><br>
Drying	:    Tumble dry-<br>
Evaluation :<br>
The light remission is measured by a Datacolor Elrepho Remission spectrophotometer. Remission is calculated as delta L (CIE Lab-values). When the surface fibrils and fibers protruding from the yarn are removed by the cellulase, the surface of the black fabric appears darker and nicer, and lower L values are obtained. Three different dosages Different dosages of cloned cellulases from Thielavia   terrestris   (SEQ ID No. 9) and Acremonium sp.   CBS 478.94 (SEQ ID No.5) and the construction 1 (SEQ ID No.2), respectively, (denoted A and B and C, respectively) were tested. Results:<br><br>
s-<br>
CEVU/1	0	100	200	400<br>
	^<br>
'^	-3 . 06 ±0. 24	-3. 15<br>
±0. 27	-3.92 ±0.26<br>
B	c	-1.75 ±0.27	-3 . OS ±0.32	-3.51 ±0.44<br>
C	0	-1.84 ±0.39	-1.70 ±0.47	-2.30 ±0.61<br>
The data show that all cellulases gives very good color clarification under the conditions tested.<br>
I. Application of endoglucanases from Thielavia terre¬stris, Acremonium sp. and construction 1 (SEQ ID No. 2) in denim finishing<br><br>
Experimental<br>
Apparatus: Liquid volume: Fabric:<br><br>
Washing machine Wascator FL 120<br>
20 L<br>
1.1 kg denim fabric, 14% oz 100<br>
cotton<br>
Desizing:	10 min, 55°C, pH 7<br>
50 ml Aquazyme 120L 2.5 g/1 Phosphate buffer<br>
Abrasion:	2 hours <br>
pH and temperature varied according to the following table:<br>
Enzyme   Activity      pH/temp   Buffer system<br>
SEQ ID<br>
No. 2    1400 S-CEVU/g  6/55°C    2.5 g/1 phosphate<br>
buffer No. 9    292 S-CEVU/g   5/65"C    1 g/1 citrate buffer No. 5    782 S-CEVU/g   7/45'C    2.5 g/1 phosphate<br>
buffer<br>
Inactivaricn: 15 -in, SO'C<br>
1 g/ sodium carbonate Rinses:  Three rinse cycles of 5 min in cold tap water<br>
Evaluation:<br>
Abrasion: The remission' from the fabric was determined at 420 nm using a Ticklish 2000 as a measure of the abrasion level.<br>
The results from the treatment of the denim fabric with different endoglucanases of the invention is shown in the following table:<br><br>
Enzyme	Dosage	Trial	conditions	Abrasion 42 0 mn<br>
Blank	0 S-CEVU/g textile	pH 6,	55'C	9 . 96<br>
SEQ ID No. 2	10 S-CEVU/g textile	pH 6,	55°C	14 . 37<br>
Blank	0 S-CEVU/g textile	pH 5,	65*C	9. 26<br>
SEQ ID No. 9	10 S-CEVU/g textile	pH 5,	65°C	16.86<br>
Blank	0 S-CZ\Ti/g textile	pH 7,	45*C	9 . 47<br>
SEQ ID<br>
No. 5<br>
1	10 S-CEVU/g	pH 7,	45*C	14 . 03<br>
1	i<br>
All rested cellulases show excellent performance in denim finishing, although each enz&gt;Tne is unique in its own way. When applying the enzyme corresponding ro SEQ ID No. 2 for denim finishing it is possible to reach a high abrasion level with a minimum of strength loss. When treating denim with the enzyme corresponding to SEQ ilD No. 9, a very high wash down can be reached which leaves the fabric with an almost bleached appearance. Denim finishing with the enzyme corresponding to SEQ ID No. 5 gives a high abrasion level at a low temperature optimum which makes it possible to reduce the processing tempera¬ture and save energy.<br><br>
Lyocell fibers which are sold under the trade name Te'<br><br>
are spun from wood pulp cellulose in a more environmentally friendly waterbased solvent than is the case for normal viscose production). However, the fibers have a tendency to fibrillate when they are processed into textiles which is seen on the surface and denoted "fuzz". By using cellulases it is possible to permanently remove the exposed and fuzzy fibers and significantly improve the look of the finished fabric, the treatment generally known as Biopolishing. The endoglucanases of the present invention are especially suited for the removal of Lyocell surface fibers.<br>
MATERIALS AND METHODS<br>
The textile substrate was either 100 % woven or different kinds of jersey knitted dark blue Tencel. The dark -colour and jersey knit was preferred in order to enhance the visual effects which simplifed the evaluation. A woven 70/3C Tencel/Kayon blend was also used to a lesser extent.<br>
The assays were either performed in 200 ml scale using a Launder-c-meter or in the 20 1 scale using a Wascator. The treatment time was 6Cmin at 55' C in Wascator and SO-SO min in LOM. The buffer was 2 g/i sodium acetate adjusted to pH 5 with acetic acid. The fabric to liquid ratio was 1:10 but in the Launder-o-meter 20 steel balls with a diameter of 14 mm (11 g each) was used to obtain sufficient mechanical abrasion. The biopolishing was immediately followed by inactivation using 2 g/lit sodium carbonate at 80° C for 15 min followed by rinsing in cold water.<br>
The results were evaluated using a fuzz note scale from 1 - 5 were 1 is the fibrillated look of the starting material and 5 is a high quality look with no visible fibers on the surface. Since the performance of endocellulases is specific towards a'^snxface treatment the weightloss is below 2 % and is therefore not included in the evaluation. Two cellulases were evaluated: the<br><br>
cellulases cloned from Acremonium  sp.    (SEQ ID No. 5) and from Thielavia   terrestris   (SEQ ID No. 9).<br>
The two cellulases are able to defibrillate both Tencel and Tencel blended fabrics. By using an endoglucanase of the invention, only small fibrils are removed rather than whole fibers such as is the case when using acid cellulase mixtures from Trichoderma.   The strength loss of the treated fabric is threrefore kept at a minimum when using endoglucanases of the present invention.<br>
The following dosages gave a superior defibrillation, i.e. fuzz note 4 or above:<br>
15 S-CEVU/g fabric of cellulase from Acrerao.iium sp   (SEQ ID No. 5); and<br>
10 S-CEVU/g fabric of cellulase from Thelavia   terrestris (SEQ ID No.S).<br>
EXAMPLE 2<br>
A new cellulytic enzyme was by expression cloning as well as by PCR cloning detected to be produced by a plant pat¬hogen, isolated from soy bean seeds and identified as Ma-crophomina phaseolina.<br>
Production of biomass for PCR and expression cloning procedures:<br>
Isolate CBS 281.96 was grown in shake flask cultures on I  cellulose enriched potato dextrose broth, incubated for 5 days at 260C (shaking conditions: 150 rpm).<br>
A. Cloning and expression of an endoglucanase from Macrophomina  phaseolina<br>
mRNA was isolated from Macrophomina   phaseolina,   grown in a cellulose-containing fermentation medium with agitation<br><br>
to ensure sufficient aeration. Mycelia were harvested after 3-5 days' growth, immediately frozen in liquid nitrogen and stored at -30°C. A library from Macrophomina phaseolina,   consisting of approx. 10" individual clones was constructed in E.   coli   as described with a vector background of 1%.<br>
Plasmid DNA from some of the pools was transformed into yeast, and 50-100 plates containing 250-400 yeast colonies were obtained from each pool.<br>
Endoglucanase-positive colonies were identified and isolated on SC-agar plates with the AZCL HE cellulose assay. cDNA inserts were amplified directly from the yeast colonies and characterized as described in the Materials and Methods section above. The D^
The cDNA' is obtainable from rhe plasmid in DSM 10512.<br>
Total DNA was isolated from a yeast colony and plasmid DNA was rescued by transformation of E.   coli   as described above. In order to express the endoglucanse in Aspergillus,   the DNA was digested with appropriate restriction enzymes, size fractionated on gel, and a fragment corresponding to the endoglucanase gene was purified. The gene was subsequently ligated to pHD414, digested with appropriate restriction enzymes, resulting in the plasmid pA2C477.<br>
After amplification of the DNA in E.   coli   the plas-.id was transformed into Aspergillus  oryzaa   as described above.<br>
Screening of the cDNA library by hybridization and characterization of the positive clones. Approximately<br><br>
6000 colony forming units (c.f.u.) from the Macrophomina phaseolina   cDNA library in E. coli was screened by colony hybridization using a random-primed "P-labeled PCR product from M.   phaseolina  as probe. The PCR product was generated as described in the Materials and methods section. The positive cDNA clones were characterized by sequencing the ends of the cDNA inserts, and by determining the nucleotide seuence of the longest cDNA from both strands. The DNA sequence of the cDNA encoding the endoglucanase is shown in SEQ ID No. 10 and the corresponding amino acid sequence is shown in SEQ ID No. 11.<br>
B. Construction of gene fusion between the endoglucanase from Macrophomina phaseolina  and the 4 3 k.D endoglucanase<br>
from Humicola insolens<br>
One construction was prepared in order to make a derivative of the endoglucanase from M. phaseolina  with the linker and C5D from the 4 3 kD endoglucanase from H. insolens   (disclosed in WO 91/17243). The native endoglucanase from M.   phaseolina  does not have a linker and/or a cellulose binding domain, CBD.<br>
The construction consists of the endoglucanase from, M. phaseolina   (223 anino acids) and the 72 C-terminal amino acids from the H.   insolens   4 3 kD endoglucanase.<br>
The 43 kD endoglucanase cDNA from H.   insolens   is cloned into pHD414 in such a way that the endoglucanase gene is transcribed from the Taka-promoter. The resulting plasmid is named pCaHj4lS.<br>
The'cDNA encodirrg"the endoglucanase from M. phaseolina   is cloned into pYES2.0 as a BstX I/Not I fragment and the resulting plasmid is named pClC477.<br><br>
Primers:<br>
primer 1: 5'-<br>
GGTCGCCCGGACTGGCTGTTCCCGTACCCCCTCCAGCAGCACCAGCTCTCCGG -3' primer 2: 5'<br>
CCGGAGAGCTGGTGCTGCTGGAGGGGGTACGGGAACAGCCAGTCCGGGCGACC -3' pYES2.0 F.HT primer:  5' CGGACTACTAGCAGCTGTAATACG -3' AMG-term. primer:  5' CCCCATCCTTTAACTATAGCG -3'<br>
The endoglucanase fusion is constructed by the PCR overlap-extension method as described by Higuchi et al. 1988.<br>
Reaction A: The Pclymerase Chain Reaction (PCR) is used<br>
to'amplify the fragment of pCaHj418 between printer 1 and<br>
AMG-term. primer (the linker and CBD from the 4 3 kD<br>
endoglucanase from H.insolens).<br>
Reaction B: PC?, amplification of the fragmen" berwier.<br>
pYES2.0 F.HT primer and primer 2 in pClC477, the<br>
endoglucanase gene from M. p'naseolina.<br>
Reaction C: The two purified fragments are used in a<br>
third PCR in the presence of the primers flanking rhe<br>
total region, i.e. pYES2.0 F.HT primer and AMG-term.<br>
primer.<br>
The fragment amplified in reaction C is purified, digested with restriction enzymes, e.g. Xba I and BamH I. The purified digested fragment is ligated into pHD4 14 digested with restriction enzymes, e.g. Xba I and BamH I.<br>
Competent cells from E.   coli   strain DHSaF' (New England<br>
Biolabs) are transformed with the ligated plasmid and<br>
colonies containing the gene fusion are isolated. The<br>
sequence of the cloned part was verified by DNA sequen¬<br>
cing.		<br>
Polymerase Chain Reactions are carried out under standard<br><br>
conditions, as recommended by Perkin-Elmer.<br>
Reaction A and B start with 2 min. at 94°C followed by 20 cycles of (30 sec. at 94°C, 30 sec. at 52°C and 1 min. at 72°C) and ends with 4 min. at 72 =C.<br>
Reaction C starts with (2 min. at 94°C, 1 min. at 52°C and 2 min. at 72°C), followed by 20 cycles of (30 sec. at 94°C, 30 sec. at 52°C and 90 sec. at 72°C) and ends with 4 min. at 72°C.<br>
The construct may be transformed into Aspergillus   oryzae as described above.<br>
EXAMPLE 3<br>
Cloning and expression of an endoglucanase froE Acremonlum sp.   and Sordaria  fimicola<br>
Production of biomass for expression cloning procedures: Isolates CBS 473.94 and ATCC 52644, respec~iveiy, were grown in shake flask cultures on cellulose enriched pota¬to dextrose broth, incubated for 5 days at 260C (shaking conditions: 150 rpm).<br>
mRNA was isolated from Acremonium  sp.,    C5S 478.94, and Scrdaria  flrrdcola,   ATCC 52644, respectively, grown in a cellulose-containing fermentation medium with agitation to ensure sufficient aeration. Mycelia were harvested after 3-5 days' growth, immediately frozen in liquid nitrogen and stored at -80°C.   Libraries from Acrer-:onlum sp.,   and Sordaria  fimicola,   respectively, each consisting of approx. lo' individual clones were constructed in E. jzpli   as described with a vector background ofTT.<br>
Plasmid DNA from some of the pools from each library was<br><br>
transformed into yeast, and 50-100 plates containing 250-400 yeast colonies were obtained from each pool.<br>
Endoglucanase-positive colonies were identified and isolated on SC-agar plates with the AZCL HE cellulose assay. cDNA inserts were amplified directly from the yeast colonies and characterized as described in the Materials and Methods section above.<br>
The DNA sequence of the cDNA encoding the endoglucanase from Acremonium  sp.   is shown in SEQ ID No. 6 and the corresponding amino acid sequence is shown in SEQ ID No.7. The cDNA is obtainable from the plasmid in DSM<br>
1003 0.<br>
The partial DNA sequence of the cDNA encoding the endoglucanase from Sordaria   fi~dccla   is shown in SEQ ID No. 19 (Nucleotide sequence of rhe 5'-e.nd of the cDNA) and the corresponding amino acid sequence is shown in SEQ ID No. 20. The cDNA is obtainable from the plasmid in DSM<br>
Total DNA was isolated from a yeast colony and plasmid DNA was rescued by transformation of E.   coli   as described above. In order to express the endoglucanase in Aspergrillus, the Dl^k  was digested with appropriate restriction enzymes, size fractionated on gel, and a fragm.ent corresponding to the endoglucanase gene from Acremonium sp.   and Sordaria  fimicola,   respectively, was purified. The genes were subsequently ligated to pHD414, digested with appropriate restriction enzymes, resulting in the plasmids pA2C371 and pA2C502, respectively.<br>
After amplification of the DNA in E.   coli   the plasmids were trarrsformed into Aspergillus  oryzae  as described above.<br><br>
EXAMPLE 4<br>
A. Cloning by PCR an endoglucanase from Crinipellis  sca-bella,   CBS 280.96<br>
Isolate CBS 280.96 was grown in static flask cultures, holding wheat bran medium (per flask: 300g wheat bran added 450 ml salt solution), incubated for 6 days at 26C. After incubation the wheat bran was extracted with destilled water (300ml per flask) and the extract tested for endoglucanase activity (0.1% AZCL-HE-Cellu]ose (megazyme) in 1% agarose (Litex agarose, Medinova). Activity was observed on the plates holding pH of 3.0, 7.0 and S.5.<br>
mRN-ii was isolated from Crinipellis scabella grown as describe above. Mycelia were harvested after 3-5 days' growxh, luiTTiediatiely froze.n m liquid nitrogen and stored at -60'C. A library from Crinipellis  scabella,   consisting of apprcx. iO' individual clones was constructed in I. coli   as described with a vector background of 1%.<br>
Approximately 10 000 colony forming units (c.f.u.) from the Crinipellis  scabella  cDNA library in E. coli was screened by colony hybridization using a random-primed "P-labeled PCR product from C. scabella   as probe. The PCR product was generated as described in the Materials and methods section. The positive cDNA clones were characterized by sequencing the ends of the cDNA inserts, and by determining the nucleotide seuence of the longest cDNA from both strands.<br>
The DNA sequence of the cDNA encoding the endoglucanase is shown in SEQ ID No. 12 and the corresponding amino acid-ssqiience is shown in SEQ ID No. 13.<br>
The cDNA is obtainable from the plasmid in DSM 10511.<br><br>
Total DNA was isolated from a yeast colony and plasmid DNA was rescued by transformation of E.   coli   as described above. In order to express the endoglucanse in Aspergillus,   the DNA was digested with appropriate restriction enzymes, size fractionated on gel, and a fragment corresponding to the endoglucanase gene was purified. The gene was subsequently ligated to pHD414, digested with appropriate restriction enzymes, resulting in the plasmid pA2C475.<br>
After amplification of the DNA in E.   coli  the plasmid was transformed into Aspergillus  oryzae  as described above.<br>
Construction of two gene fusions between the endoglucanase from Crinipellis  scahella   and the linker/CBD region of the 43 kDa endoglucanase from<br>
cusj.ccxa x~i.sox^ns.<br>
The native endoglucanase from Crinipellis  scabella  neit¬her has a linker nor a cellulose binding domain (CBD). In addition, the full-length cDNA contains an ATG start codon upstream from the endoglucanase encoding open reading frame (ORF), presumably resulting in scrambled translation initiation upon heterologous expression of the cDNA, such as in the yeast Saccharomyces  cerevisiae and the filamentous fungus Aspergillus   oryzae.   Thus, two gene fusions between the endoglucanase from Crinipellis scabella   and the linker/CBD region of the 43kD endoglucanase from Humicola  insolens   (disclosed in WO 91/17243) has been constructed using splicing by overlap extension (SOE) (Horton et al, 1S39).<br>
Construction 1 consists of the cDNA encoding the 226-re-sidue fendoglucanase from C. scabella   fused by PCR with the 3'-end cDNA of H.   insolens  coding  for  the linker and CBD region (72 amino acids) at the COOH-terminus of the<br><br>
H.   insolens  43kD endoglucanase. The second hybrid construct is identical to the abovementioned gene fusion, except that the first five residues from the putative signal peptide have been deleted by PCR resulting in a shorter signal, which starts with the second in-frame ATG start codon.<br>
Plasmid constructs<br>
The plasmid pClC475 contains the full-length cDNA from C. scabella,   cloned into BstXI/Notl-cut yeast expression vector pYES 2.0, the plasmid pClC144 contains the full-length cDNA from H.   insoiens,   cloned into the BstXI site of pYES 2.0.   ■■<br>
Splicing by overlap extension<br>
Two PCR fragments encoding the core region of the endog¬lucanase from C. scahella  were generated in PCR buffer (10 mM Tris-HCl, pH 8.3, 50 inM"KCl, 1.5 r^M MgCl-,, 0.01 % gelatin; containing 200 .LLM each dNTP), usi.ng 50-100 ng of pClC475 as template, and 250 pmol of the reverse primer (5'-GACCGGAGAGCTGGTGCTGCTGGAGGGTTTACGAACACAGCCCGAGATATTAG TG- 3*) in two combinations with 300-350 pmol of each forward primer (forward no. 1 5'-<br>
CCCCA„AGCTTGACTTGGAACCAJ^TGGTCCATCC-3 " , forward no. 2 5"-CCCCAAGCTTCCATCCAAACATGCTTAAAACGCTCG- 3"), a DNA thermal cycler (Landgraf, Germany) and 2.5 units of Tag polymerase (Perkin-Elmer, Cetus, USA). Thirty cycles of PCR were performed using a cycle profile of denaturation at 94 °C for 1 min, annealing at 55 °C for 2 m.in, and extension at 72 °C for 3 m.in. The PCR' fragffient coding for the linker and CBD of the endoglucanase of H.   insolens was generated in PCR- buffer (10 mM Tris-HCl, pH 8.3, 50 mM KCl, 1.5 mM MgCl,, 0.01 % gelatin; containing 200 ^M each dNTP) using 100 ng of the pClC144 template, 250 pmol<br><br>
forward primer (5'-CACTAATATCTCGGGC-<br>
TGTGTTCGTAAACCCTCCAGCAGCACCA-GCTCTCCGGTC-3') , 250 pmol of the pYES 2.0 reverse primer<br>
(5'-GGGCGTGAATGTAAGCGTGACATA-3■) a DNA thermal cycler (Landgraf, Germany) and 2.5 units of Tag polymerase (Perkin-Elmer, USA). Thirty cycles of PCR were performed as above. The PCR products were electrophoresed in 0.7 % low gelling temperature agarose gels (SeaPlaque, FMC), the fragments of interest were excised from the gel and recovered by treatment with agarase (New England Biolabs, USA) according to the manufacturer's instructions, followed by phenol extraction and ethanol precipitation at - 20 °C for 12 h by adding  2 vols of 96 % EtOH and 0. 1 vol of 3.M NaAc.<br>
The recombinant hybrid genes between the endoglucanase from Crinipallis   scabella   and the linker/CBD region of the 4 3 kD endoglucanase from Humicola   insolsr.s   were generated by combining the overlapping PCR fragments from above (ca. 50 ng of each template) in two combinations in PCR buffer (10 mM Tris-HCl, pH S.3, 50 ir^M KCl, 1.5 mM MgCl-,, C.Ol % gelatin; containing 200 uM each dNT?) . The SOE reaction was carried out using the DNA thermal cycler (Landgraf, Germany) and 2.5 units of Taq polymerase (Perkin-Elmer, Cetus, USA). Two cycles of PCR were performed using a cycle profile of denaturation at S4 °C for 1 min, annealing at 55 °C for 2 min, and extension at 72 °C for 3 min, the reaction was stopped, 2 50 pmol of each end-primer (forward no. 1<br>
5'-CCCCAAGCTTGACTTGGAACCAATGGTCCATCC-3', forward no. 2 5 ' -CCCCAAGCTTCCATCC.ILAJ\CATGCTTAAAACGCTCG-3 ' , reverse primer 5 '-GGGCGTGAATGT.AAGCGTGACATA-3 ' ) was added to rhe reaction mixture, and an additional 30 cycles of PCR were performed using a cycle profile of denaturation at 94 °C for 1 min, annealing at 55 'tnor 2 min, and extension at 72 °C for 3 min.<br><br>
Construction of the expression cassettes for heterologous expression in Aspergillus oryzae<br>
The PCR-generated, recombinant fragments were electrophoresed in a 0.7 % low gelling temperature agarose gel (SeaPlaque, FMC), the fragments of interest were excised from the gel and recovered by treatment with agarase (New England Biolabs, USA) according to the manufacturer's instructions, followed by phenol extraction and ethanol precipitation at - 20 °C  for 12 h. The DNA fragments were digested to completion with Hindlll and Xbal, and ligated into Hi/idlll/A'bal-cleaved pHD414 vector followed by electroporation of the constructs into £.   coli   DHIOB cells according to the manufacturer's instructions (Life Technologies, -USA). -<br>
The nucleotide sequence of the resulting gene fusions were cererTrined froir, born strands as described in rne Materials and methods section, SEQ ID no. 14A and 15A. The constructs may be transformed into A.   oryzae   as described.<br>
EXAMPLE 5<br>
PCR facilitated detection of said type of cellulytic en¬zyme from 46 filamentous and monocentric fungi, representing 32 genera, from 23 families, belonging to 15 orders of 7 classes, covering all in all all four groups of the true Fungi: Ascomycetous, Basidiomycetous, Chytri-diomycetous and Zygomycetous fungi<br>
5.1 Materials<br>
l.Diplodia gossypina Cooke<br>
Deposit of Strain, Ace No: CBS 274.S6<br>
2.Ulospora bilgramii (Hawksw. et al.) Hawksw. et al. Ace No of strain: NKBC 1444,<br><br>
3.	Microsphaeropsis sp<br>
4.	Ascobolus stictoideus Speg.<br>
Ace No of strain: Q026 (Novo Nordisk collection) Isolated from goose dung, Svalbard, Norway<br>
5.	Saccobolus dilutellus (Fuck) Sacc. Deposit of strain: Ace No CBS 275.96<br>
6.	Penicillium verruculosum  Peyronel Ex on Ace No of species: ATCC 62396<br>
7.	Penicillium chrysogenum Thorn Ace No of Strain: ATCC 9480<br>
8.	Thermomyces verrucosus Pugh et al Deposit of Strain, Ace No.: CES 2 3 5.56<br>
9.	Xylaria hypoxylon L. ex Greville Deposit of Strain, Ace No: CBS 2 3 4.96<br><br>
10.	Poronia punctata (Fr.ex L.) Fr. Ref:A.Munk: Danish Pyrenomyeetes, Dansk Botanisk Arkiv, Voll7,1 1957<br>
11.	Nodulisporum sp<br>
Isolated from leaf of Camellia reticulata (Theaceae,<br>
Guttiferales),<br>
Kunming Botanical Garden, Yunnan Province, China<br>
12.	Cylindrocarpon sp<br>
Isolated from marine sample, the Bahamas<br>
13.	Fusarium anguioides Sherbakoff<br>
Ace No of strain: IFO 4467<br><br>
14.	Fusarium poae (Peck) Wr.<br>
Ex on Ace No of species: ATCC 50883<br>
15.	Fusarium  solani (Mart.)Sacc.emnd.Snyd &amp; Hans. Ace No of strain: IMI 107.511<br>
16.	Fusarium oxysporum ssp lycopersici (Sacc.)Snyd. &amp; Hans.<br>
Ace No of strain: CBS 645.78<br>
17.	Fusarium oxysporum ssp passiflora<br>
Ace No of strain: CBS 744.79<br>
IS. Glioeladium catenulatum Gillman &amp; Abbott Ace.   No. of strain: ATCC 10523<br>
19.	Nectria pinea Dingley<br>
Deposit of Surain, Ace. No. CBS 279.95<br>
20.	Sordaria macrospora .a.uersvald Ex on Ace No of species: .a.TCC 602 55<br>
21.	Humicola grisea Traeen<br>
ex on Ace No for the species: ATCC 22726<br>
22.	Humicola nigrescens Omvik Ace No of strain: CBS 819.73<br>
23.	Scytalidium thermophilum (Cooney et Emerson) Austwick Ace No of strain: ATCC 28085<br>
24.	T.hielavia thermophila Fergus et Sinden (syn Corynascus thermophiius)<br>
Ace No of strain: CBS 174.70, IMI 145.136<br>
25. Cladorrhinum foecundissimum Saccardo et Marchal Ex on Ace No of species: ATCC 62373<br><br>
26.	Syspastospora boninensis<br>
Ace No of strain: NKBC 1515 (Nippon University, profe Tubaki Collection)<br>
27.	Chaetoir.ium cuniculorum Fuckel Ace. No. of strain: CBS 799.83<br>
28.	Chaetomium brasiliense Batista et Potual Ace No of strain: CBS 122.65<br>
29.	Chaetomium murorum Corda Aec No of strain: CBS 163.52<br>
30.	Chaetomium virescens (von Arx) Udagawa Ace.No. of strain: C5S 547.75<br>
31.	Nigrcspora sp<br>
Deposit of strain, Ace No: CBS 2"2.96<br>
32.	Nigrospora sp Isolated from:<br>
33.	Diaport-he syngenesia<br>
Deposit of strain, Aec No: CBS 278.96<br>
34.	Colletotrichum lagenarium (Passerini) Ellis et<br>
Kalsted<br>
syn Glomerella cingulata var orbiculare Jenkins et<br>
Winstead<br>
Ex on a^^c No u£ species: ATCC 52609<br>
35.	Exidia glandulosa Fr.<br>
Deposit of Strain, Aec No: CBS 277.96<br>
36.	Pomes fomentarius (L.) Fr.<br>
Deposit of strain: Aec No. CBS 276.96<br><br>
37.	Spongipellis (?)<br>
Deposit of Strain: Ace No CBS 233.96<br>
38.	Rhizophlyctis rosea (de Bary &amp; Wor) Fischer Deposit of Strain: Ace No.: CBS 282.96<br>
39.	Rhizomucor pusillus (Lindt) Schipper syn: Mucor pusillus<br>
Ace No of strain: IFO 4578<br>
40.	Phycomyces nitens (Kunze) van Tieghem &amp; Le Monnier<br>
Ace No of strain: IFO 4814<br>
41 Chaetostylum fresenii van  Tieghem &amp; Le Monnier syn. Helicostylum fresenii Ace No of strain NRRL 2 3 05<br>
D'.neciuiTi rcseum, Ace No or s-crain: xru o j, /:<br>
43.	Coniothecium sp<br>
Endophyte, isolated from leaf of flevering plant, Kunming , Yunnan, China<br>
44.	Deposit of strain, Ace No.: CBS 271.96 Coelomycete, Isolated from leaf of Artocarpus altilis (Moraceae, Urticales), Christiana, Jamaica<br>
45.	Deposit of strain. Ace No.: CBS 273.96 Coelomycete, isolated from leaf of Pimenta dioica (Myrtaceae, Myrtales), Dallas Mountain, Jamaica<br>
45. Deposit of strain: CBS 270.96<br>
Coelomycete, isolated from leaf of Pseudocalymma<br>
alliaceum (Bignoniaceae, Solanales) growing in Dallas<br>
Mountain, Jamaica	"<br><br>
JO<br><br>
5.2 Procedure<br>
Maintenance of strains and production of biomass:<br>
The strains were maintained on agar in petrie dishes (9cm) or on  slants (see list of Media: PCA and PDA). 44 of the strains were grown in shake flasks under the following  growth conditions: general fungal media as PC, PD and PB 9 or YPG (see list of media);  incubation time from 3 to 9 days; temperature 26"C; rpm between 150 and 175 . Strain No 14 (F.poae) was grown on wheat bran for 15 days (26"C; static). Strain No 38 was grown in dilute salt solution (DS/2), added 1 cm- pieces of autoclaved filter paper.<br>
Activity test:<br>
Activity was tested on 0.1% AZCL-HE-Cellulose (Megazyme) plates (14 era Petrie dishes), made up in 1% agarose (HSE, Lirex Agarose, Medmova). .Ail tests were done in triplicate, viz. A2CL-HE-Cellulose dissolved in three buffers, adjusted ro pH 3, 7 or S.5 (using various proportions of t:he following two ingredients Citric acid sonohydrat, Merck art. No 100244 (21.0 g) dissolved in water, making a total of 1000 ml; 0.IM tri-Sodium dodeca-brohydrate, Merck art.no. 6578 (38 g), dissolved in wa¬ter, making a total of 1000 ml. The mixing is done imrnidiately before use.<br>
Harvesting of Biomass:<br>
The biomass was harvested by filtering (mesh adjusted to the growth of the fungus, the finest used for fungi which have highly sporulating mycelium as e.g. Fusarium spp.). The biomass on the filter was scraped into a sterile plastic bag and imrnidiately frozen (by submerging into liquid nitrogen).<br><br>
5.3 Results<br>
I. Using the PCR screening and amplification techniques described in Materials and Methods the following partial cDNA sequences were obtained:<br>
Saccobolus dilutellus (Fuck) Sacc, CBS 275.96: SEQ ID<br>
No. 21 (and the deduced amino acid sequence in SEQ ID No.<br>
22) ;<br>
Thermomyces verrucosus, CBS 285.96: SEQ ID No. 23 (and<br>
the deduced amino acid sequence in SEQ ID No. 24);<br>
Xylaria hypoxylon, CBS 284.96: SEQ ID No. 25 (and the<br>
deduced amino acid sequence in SEQ ID No. 26);<br>
Fusarium oxysporum ssp lycopersici, CBS 645.78: SEQ ID<br>
No. 27 (and the deduced amino acid sequence in SEQ ID No.<br>
28);<br>
Nectria pinea, CBS 279.96: SEQ ID No. 29 (and the deduced<br>
amino acid sequence in SEQ ID No. 30);<br>
Humicola grisea, ATCC 2272 6: SEQ ID No. 31 (and rhe<br>
deduced amino acid sequence in SEQ ID No. 32);<br>
Humicola nigrescens, C3S 819.73: SEQ ID No. 33 (and the<br>
deduced amino acid sequence in SEQ" ID No. 34);<br>
Cladorrhinum foecundissimum, ATCC 62373: SEQ ID No. 35<br>
(and the deduced amino acid sequence in SEQ ID No. 36);<br>
Syspastospora boninensis, NKBC 1515: SEQ ID No. 37 (and<br>
the deduced amino acid sequence in SEQ ID No. 38);<br>
Nigrospora sp., CBS 272.96: SEQ ID No. 39 (and the<br>
deduced amino acid sequence in SEQ ID No. 40);<br>
Chaetostylum fresenii: SEQ ID No. 41 (and the deduced<br>
amino acid sequence in SEQ ID No. 42);<br>
Exidia glandulosa, CBS 277.96: SEQ ID No. 43 (and the<br>
deduced amino acid sequence in SEQ ID No. 44);<br>
Coniothecium sp.: SEQ ID No. 45 (and the deduced amino<br>
acid sequence in SEQ ID No. 46);<br>
Deposition No. CBS 271.96: SEQ ID No. 47 (and the deduced<br>
amino acid sequence in S£i2—LD No. 48);<br>
Deposition No. CBS 270.96: SEQ ID No. 49 (and the deduced<br>
amino acid sequence in SEQ ID No. 50);<br><br>
Diplodia gossypina, CBS 274.96: SEQ ID No. 51 (and the<br>
deduced amino acid sequence in SEQ ID No. 52);<br>
Ulospora bilgramii, NKBC 1444: SEQ ID No. 53 (and the<br>
deduced amino acid sequence in SEQ ID No. 54);<br>
Penicillium verruculosum, ATCC 62396: SEQ ID No. 55 (and<br>
the deduced amino acid sequence in SEQ ID No. 56);<br>
Poronia punctata: SEQ ID No. 57 (and the deduced amino<br>
acid sequence in SEQ ID No. 58);<br>
Fusarium anguioides, IFO 4467: SEQ ID No. 59 (and the<br>
deduced amino acid sequence in SEQ ID No. 60);<br>
Thielavia thermophila, CBS 174.70: SEQ ID No. 61 (and the<br>
deduced amino acid sequence in SEQ ID No. 62);<br>
Chaetomium cuniculorum, CBS 799.83: SEQ ID No. 63 (and<br>
the deduced amino acid sequence in SEQ ID No. 64);<br>
Chaetomium virescens: SEQ ID No. 65 (and the deduced<br>
amino acid sequence in SEQ ID No. 66);<br>
Colletctrichum lagenarium: SEQ ID No. 67 (and the deduced<br>
amino acid sequence in SEQ ID No. 65);<br>
Phycomyces nitens: SEQ ID No. 69 (and the deduced amino<br>
acid sequence in SEQ ID No. 70); and<br>
Trichothecium roseum: SEQ ID No. 71 (and the deduced<br>
amino acid sequence in SEQ ID No. 72);<br>
II. Using the PCR screening and amplification techniques<br>
described in Materials and Methods partial cDNA encoding<br>
partially for the enzyme of the invention was obtained<br>
and the plasmid was deposited according to the Budapest<br>
Treaty:<br>
Escherichia  coli,   DSM 10583, deposition date 13 March,<br>
1996;<br>
cDNA from Trichothecium roseum;<br>
Escherichia  coli,   DSM 10584, deposition date 13 March,<br>
1996;<br>
cDNA from Syspastospora boninensis;<br>
Escherichia  coli,   DSM 10585, depbsitrTon date 13 March,<br>
1996;<br>
cDNA from Cheatomium murorum<br><br>
Escherichia  coli,   DSM 10587, deposition date 13 March,<br>
1996;<br>
cDNA from Sordaria fimicola;<br>
Escherichia  coli,   DSM 10588, deposition date 13 March,<br>
1996;<br>
cDNA from the unidentified strain CBS 273.96;<br>
Escherichia  coli,   DSM 10586, deposition date 13 March,<br>
1996;<br>
cDNA from Spongipellis sp.<br>
Color clarification of crude supernatants<br>
During normal wash the fabric will often fade. However, the fabric appearance is improved and the original colours are much better preserved or maintained if the fabric is washed with a cellulase giving color clarificarion. Color clarification is measured as removal of surface fibrils and fibers protruding from the yarn of a textile containing cellulosic fibers.<br><br>
Apparatus Liquid volume Agitation<br>
Rinse time<br>
Washing temp<br>
Washing liqour<br>
PH<br>
Washing time<br>
Repetitions<br>
Enzymes<br>
Dosage<br>
Textile cotton<br><br>
Terg-o-tometer<br>
100 ml<br>
150 movements/min with vertical<br>
stirrer<br>
5 min in tapwater<br>
40°<br>
0.05 M phosphate buffer<br>
7.0<br>
30 min<br>
2<br>
Crude supernatants from the<br>
strains shown below.<br>
Two dosages from : 200, 500,<br>
1000 or 2500—S=CEVU/1<br>
2 swatches of aged black 100%<br>
5x6 cm (0.9 gram)<br><br>
Drying	:    Tumble dry-<br>
Evaluation:<br>
The light remission is measured by a Datacolor Elrepho Remission spectrophotometer. Remission is calculated as delta L (Hunter Lab-values). When the surface fibrils and fibers protruding from the yarn are removed by the cellulase, the surface of the black fabric appears darker, and lower L values are obtained.<br>
The samples are compared with a blind sample, i.e. washed without enzyme. Below is shown the delta L remission values compared to a blind sample.<br>
SEQUENCE   LISTING<br>
(1)	GENERAL INFORMATION:<br>
(i) APPLICANT <br>
(A)	NAME: Novo Nordisk A/S<br>
(B)	STREET: Novo Alle<br>
(C)	CITY: DK-2880 Bagsvaerd<br>
(E)	COUNTRY: Denmark<br>
(F)	POSTAL CODE (ZIP): DK-2880<br>
(G)	TELEPHONE: +45 44 88 88<br>
(H)	TELEFAX: +45 44 49 32 56<br>
(ii) TITLE OF INVENTION: NOVEL ENDOGLUCANASES (iii) NU'MBER OF SEQUENCES: 72<br>
(iv) COMPUTER READABLE FORM:<br>
(A)	MEDIUM TYPE: Floppy disk<br>
(B)	COMPUTER: IBM PC compatible<br>
(C)	OPERATING SYSTEM: PC-DOS/MS-DOS<br>
(D)	SOFTWARE: Patentin Release #1.0, Version<br>
#1.30 (EPO)<br>
(2)	INFORMATION FOR SEQ ID NO: lA:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 891 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br><br>
(vi) ORIGINAL SOURCE:<br>
(A)	ORGANISM: Saccharomyces cerevisiae<br>
(B)	STRAIN: DSM 9770<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: lA:<br>
AAAGAAAGGC TCTCTGCTGT CGTCGCTCTC GTCGCTCTCG TCGGCATCCT CCATCCGTCC      60<br>
GCCTTTGATA ACCCGCTCCC CGACTCAGTC AAGACGACGC ATACTTGGCA CCATGCATCT     12 0<br>
CTCCGCCACC ACCGGGTTCC TCGCCCTCCC GGTCCTGGCC CTGGACCAGC TCTCGGGCAT     180<br>
CGGCCAGACG ACCCGGTACT GGGACTGCTG CAAGCCGAGC TGCGCCTGGC CCGGCAAGGG     2 4 0<br>
CCCCTCGTCT CCGGTGCAGG CCTGCGACAA GAACGACAAC CCGCTCAACG ACGGCGGCTC     3 00<br>
CACCCGGTCC GGCTGCGACG CGGGCGGCAG CGCCTACATG TGCTCCTCCC AGAGCCCCTG     3 60<br>
GGCCGTCAGC GACGAGCTGT CGTACGGCTG GGCGGCCGTC AAGCTCGCCG GCAGCTCCGA     4 2 0<br>
GTCGCAGTGG TGCTGCGCCT GCTACGAGCT GACCTTCACC AGCGGGCCGG TCGCGGGCAA     480<br>
GAAGATGATT GTGCAGGCGA CCAACACCGG TGGCGACCTG GGCGACAACC ACTTTGACCT     540<br>
GGCCATCCCC GGTGGCGGTG TCGGTATTTT CAACGCCTGC ACCGACCAGT ACGGCGCTCC     600<br>
CCCGAACGGC TGGGGCGACC GCTACGGCGG CATCCATTCC AAGGAAGAGT GCGAATCCTT     660<br><br>
CCCGGAGGCC CTCAAGCCCG GCTGCAACTG GCGCTTCGAC TGGTTCCAAA ACGCCGACAA     72 0<br>
CCCGTCGGTC ACCTTCCAGG AGGTGGCCTG CCCGTCGGAG CTCACGTCCA AGAGCGGCTG     780<br>
CTCCCGTTAA GAGGGAAGAG AGGGGGCTGG 7AGGACCGAA AGATTCAACC TCTGCTCCTG     84 0<br>
CTGGGGAAGC TCGGGCGCGA GTGTGAAACT GGTGTAAATA TTGTGGCACA CACAAGCTAC     900<br>
TACAGTCCGT CTCGCCGTCC GGCTAACTAG CCTTGCTGCG GATCTGTCCA AAAAAAAAAA     960<br>
(2) INFORMATION FOR SEQ ID NO: IB:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 225 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii)   MOLECULE  TYPE:   protein<br>
(xi)    SEQUENCE   DESCRIPTION:   SEQ   ID  NO:    IB:<br>
Met  His  Leu   Ser Ala  Thr  Thr  Gly  Phe  Leu  Ala  Leu   Pro  Val  Leu  Ala<br>
15	10	15<br>
Leu Asp Gin Leu Ser Gly lie Gly Gin Thr Thr Arg Tyr Trp Asp Cys<br>
20	25	30<br>
Cys Lys Pro Ser Cys Ala Trp Pro Gly Lys Gly Pro Ser Ser Pro Val<br><br>
35	40	45<br>
Gin Ala Cys Asp Lys Asn Asp Asn Pro Leu Asn Asp Gly Gly Ser Thr<br>
50	55	60<br>
Arg Ser Gly Cys Asp Ala Gly Gly Ser Ala Tyr Met Cys Ser Ser Gin<br>
65	70	75	80<br>
Ser Pro Trp Ala Val Ser Asp Glu Leu Ser Tyr Gly Trp Ala Ala Val<br>
85	90	95<br>
Lys Leu Ala Gly Ser Ser Glu Ser Gin Trp Cys Cys Ala Cys Tyr Glu<br>
100	105	110<br>
Leu Thr Phe Thr Ser Gly Pro Val Ala Gly Lys Lys Met lie Val Gin<br>
115	120	125<br>
Ala Thr Asn Thr Gly Gly Asp Leu Gly Asp Asn His Phe Asp Leu Ala<br>
130	135	140<br>
lie Pro Gly Gly Gly Val Gly lie Phe Asn Ala Cys Thr Asp Gin Tyr<br>
145	150	155	160<br>
Gly Ala Pro Pro Asn Gly Trp Gly Asp Arg Tyr Gly Gly lie His Ser<br>
165	170	175<br><br>
Lys Glu Glu Cys Glu Ser Phe Pro Glu	Ala Leu Lys Pro Gly Cys Asn<br>
180                185	190<br>
Trp Arg Phe Asp Trp Phe Gin Asn Ala	Asp Asn Pro Ser Val Thr Phe<br>
195                200	205<br>
Gin Glu Val Ala Cys Pro Ser Glu Leu	Thr Ser Lys Ser Gly Cys Ser<br>
210                215	220<br>
Arg 225<br>
2) INFORMATION FOR SEQ ID NO: 2A:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 894 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(vi) ORIGINAL SOURCE:<br>
(A) ORGANISM: "Construction 1"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 2A:<br>
\TGCATCTCT  CCGCCACCAC CGGGTTCCTC GCCCTCCCGG   TCCTGGCCCT GGACCAGCTC 60<br>
rCGGGCATCG GCCAGACGAC CCGGTACTGG GACTGCTGCA AGCCGAGCTG CGCCTGGCCC 120<br>
GGCAAGGGCC CCTCGTCTCC GGTGCAGGCC TGCGACAAGA ACGACAACCC GCTCAACGAC<br><br>
180<br>
GGCGGCTCCA CCCGGTCCGG CTGCGACGCG GGCGGCAGCG CCTACATGTG CTCCTCCCAG 240<br>
AGCCCCTGGG CCGTCAGCGA CGAGCTGTCG TACGGCTGGG CGGCCGTCAA GCTCGCCGGC 300<br>
AGCTCCGAGT CGCAGTGGTG CTGCGCCTGC TACGAGCTGA CCTTCACCAG CGGGCCGGTC 360<br>
GCGGGCAAGA AGATGATTGT GCAGGCGACC AACACCGGTG GCGACCTGGG CGACAACCAC 420<br>
TTTGACCTGG CCATCCCCGG TGGCGGTGTC GGTATTTTCA ACGCCTGCAC CGACCAGTAC 480<br>
GGCGCTCCCC CGAACGGCTG GGGCGACCGC TACGGCGGCA TCCATTCCAA GGAAGAGTGC 540<br>
GAATCCTTCC CGGAGGCCCT CAAGCCCGGC TGCAACTGGC GCTTCGACTG GTTCCAAAAC 600<br>
GCCGACAACC CGTCGGTCAC CTTCCAGGAG GTGGCCTGCC CGTCGGAGCT CACGTCCAAG 660<br>
AGCGGCTGCT CCCGTCCCTC CAGCAGCACC AGCTCTCCGG TCAACCAGCC TACCAGCACC 720<br>
AGCACCACGT CCACCTCCAC CACCTCGAGC CCGCCAGTCC AGCCTACGAC TCCCAGCGGC 780<br>
TGCACTGCTG AGAGGTGGGC TCAGTGCGGC GGCAATGGCT GGAGCGGCTG CACCACCTGC 840<br>
GTCGCTGGCA GCACTTGCAC GAAGATTAAT GACTGGTACC ATCAGTGCCT GTAG 894<br><br>
INFORMATION FOR SEQ ID NO: 2B:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 297 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii)   MOLECULE TYPE:   protein<br>
(xi)   SEQUENCE  DESCRIPTION:   SEQ   ID  NO:   2B:<br>
Met  His Leu  Ser Ala Thr Thr Gly Phe Leu Ala Leu  Pro Val  Leu Ala<br>
15	10	15<br>
Leu  Asp  Gin Leu  Ser  Gly   lie  Gly Gin  Thr  Thr  Arg  Tyr  Trp Asp  Cys<br>
20	25	30<br>
Cys  Lys  Pro Ser Cys Ala  Trp Pro Gly Lys Gly Pro  Ser  Ser Pro Val<br>
35	40	45<br>
Gin Ala Cys Asp Lys Asn Asp Asn Pro Leu Asn Asp Gly Gly  Ser Thr<br>
50	55	60<br>
Arg Ser Gly Cys Asp Ala Gly Gly Ser Ala Tyr Met  Cys  Ser  Ser Gin<br>
65	70	75	80<br>
Ser  Pro Trp Ala Val  Ser Asp Glu  Leu  Ser Tyr Gly Trp Ala Ala Val<br>
85	90	95<br><br>
Lys Leu Ala Gly Ser Ser Glu Ser Gin Trp Cys Cys Ala Cys Tyr Glu<br>
100	105	110<br>
Leu Thr Phe Thr Ser Gly Pro Val Ala Gly Lys Lys Met lie Val Gin<br>
115	120	125<br>
Ala Thr Asn Thr Gly Gly Asp Leu Gly Asp Asn His Phe Asp Leu Ala<br>
130	135	140<br>
lie Pro Gly Gly Gly Val Gly lie Phe Asn Ala Cys Thr Asp Gin Tyr<br>
145	150	155	160<br>
Gly Ala Pro Pro Asn Gly Trp Gly Asp Arg Tyr Gly Gly He His Ser<br>
165	170	175<br>
Lys Glu Glu Cys Glu Ser Phe Pro Glu Ala Leu Lys Pro Gly Cys Asn<br>
180	185	190<br>
Trp Arg Phe Asp Trp Phe Gin Asn Ala Asp Asn Pro Ser Val Thr Phe<br>
195	200	205<br>
Gin Glu Val Ala Cys Pro Ser Glu Leu Thr Ser Lys Ser Gly Cys Ser<br>
210	215	220<br>
Arg Pro Ser Ser Ser Thr Ser Ser Pro Val Asn Gin Pro Thr Ser Thr<br>
225	230	235	240<br><br>
Ser  Thr  Thr  Ser  Thr  Ser  Thr  Thr  Ser  Ser  Pro  Pro Val  Gin  Pro  Thr<br>
245	250	255<br>
Thr  Pro  Ser Gly Cys  Thr Ala  Glu  Arg Trp Ala  Gin  Cys  Gly Gly Asn<br>
260	265	270<br>
Gly  Trp Ser Gly  Cys  Thr  Thr  Cys Val  Ala  Gly  Ser  Thr  Cys  Thr Lys<br>
275	280	285<br>
lie Asn Asp Trp Tyr His  Gin  Cys Leu<br>
290	295<br>
(2)   INFORMATION  FOR  SEQ   ID  NO:   3A:<br>
(i)   SEQUENCE  CHARACTERISTICS:<br>
(A)	LENGTH:   927   base  pairs<br>
(B)	TYPE:   nucleic  acid<br>
(C)	STRANDEDNESS:   single<br>
(D)	TOPOLOGY:   linear<br>
(ii)   MOLECULE TYPE:   cDNA<br>
(vi)   ORIGINAL  SOURCE:<br>
(A)   ORGANISM:   "Construction  2"<br>
(xi)   SEQUENCE  DESCRIPTION:   SEQ   ID  NO:   3A:<br>
ATGCATCTCT  CCGCCACCAC  CGGGTTCCTC  GCCCTCCCGG   TCCTGGCCCT  GGACCAGCTC 60<br>
TCGGGCATCG  GCCAGACGAC  CCGGTACTGG   GACTGCTGCA  AGCCGAGCTG   CGCCTGGCCC 120<br>
GGCAAGGGCC   CCTCGTCTCC  GGTGCAGGCC   TGCGACAAGA  ACGACAACCC  GCTCAACGAC<br><br>
180<br>
GGCGGCTCCA  CCCGGTCCGG  CTGCGACGCG GGCGGCAGCG CCTACATGTG CTCCTCCCAG 240<br>
hGCCCCTGGG   CCGTCAGCGA  CGAGCTGTCG TACGGCTGGG CGGCCGTCAA GCTCGCCGGC 300<br>
AGCTCCGAGT CGCAGTGGTG CTGCGCCTGC TACGAGCTGA CCTTCACCAG CGGGCCGGTC 360<br>
GCGGGCAAGA AGATGATTGT GCAGGCGACC AACACCGGTG GCGACCTGGG CGACAACCAC 420<br>
TTTGACCTGG CCATCCCCGG TGGCGGTGTC GGTATTTTCA ACGCCTGCAC CGACCAGTAC 480<br>
GGCGCTCCCC CGAACGGCTG GGGCGACCGC TACGGCGGCA TCCATTCCAA GGAAGAGTGC 540<br>
GAATCCTTCC CGGAGGCCCT CAAGCCCGGC TGCAACTGGC GCTTCGACTG GTTCCAAAAC 600<br>
GCCGACAACC CGTCGGTCAC CTTCCAGGAG GTGGCCTGCC CGTCGGAGCT CACGTCCAAG 660<br>
AGCGGCTGCT CCCGTAACGA CGACGGCAAC TTCCCTGCCG TCCAGATCCC CTCCAGCAGC 720<br>
ACCAGCTCTC CGGTCAACCA GCCTACCAGC ACCAGCACCA CGTCCACCTC CACCACCTCG 780<br>
AGCCCGCCAG TCCAGCCTAC GACTCCCAGC GGCTGCACTG CTGAGAGGTG GGCTCAGTGC 840<br>
GGCGGCAATG GCTGGAGCGG CTGCACCACC TGCGTCGCTG GCAGCACTTG CACGAAGATT 900<br>
AATGACTGGT ACCATCAGTG CCTGTAG 927<br><br>
)    INFORMATION  FOR  SEQ   ID  NO:    3B:<br>
(i)    SEQUENCE  CHARACTERISTICS:<br>
(A)	LENGTH:   3 08   amino  acids<br>
(B)	TYPE:   amino acid<br>
(C)	STRANDEDNESS:   single<br>
(D)	TOPOLOGY:   linear<br>
(ii)   MOLECULE  TYPE:   protein<br>
(xi)   SEQUENCE  DESCRIPTION:   SEQ   ID	NO:   3B:<br>
Mer  His Leu  Ser .Ala Thr,Thr,Gly P.he  Leu	Ala Leu  Pro Val  Leu Ala<br>
15                                               10	15<br>
Leu  Asp  Gin  Leu   Ser  Gly   lie  Gly Gin   Thr	Thr  Arg  Tyr  Trp Asp  Cys<br>
20                                            25	30<br>
Cys  Lys   Pro  Ser  Cys  Ala  Trp  Pro  Gly  Lys	Gly  Pro   Ser  Ser  Pro  Val<br>
35                                               40	45<br>
Gin Ala Cys Asp Lys Asn Asp Asn  Pro Leu	Asn Asp Gly Gly Ser  Thr<br>
50                                            55	50<br>
Arg  Ser  Gly  Cys  Asp Ala  Gly  Gly   Ser  Ala	Tyr  Met   Cys   Ser  Ser  Gin<br>
65	70	75	80<br>
Ser  Pro Trp Ala Val  Ser Asp Glu  Leu  Ser	Tyr Gly Trp Ala Ala Val<br><br>
85<br><br>
90<br><br>
95<br><br>
Lys Leu Ala Gly Ser Ser Glu Ser Gin Trp Cys Cys Ala Cys Tyr Glu<br><br>
100<br><br>
105<br><br>
110<br><br>
Leu Thr Phe Thr Ser Gly Pro Val Ala Gly Lys Lys Met lie Val Gin<br><br>
115<br><br>
120<br><br>
12 5<br><br>
Ala Thr Asn Thr Gly Gly Asp Leu Gly Asp Asn His Phe Asp Leu Ala<br><br>
130<br><br>
135<br><br>
140<br><br>
lie Pro Gly Gly Gly Val Gly lie Phe Asn Ala Cys Thr Asp Gin Tyr<br><br>
145<br><br>
150<br><br>
155<br><br>
160<br><br>
Gly Ala Pro Pro Asn Gly Trp Gly Asp Arg Tyr Gly Gly lie His Ser<br><br>
165<br><br>
170<br><br>
175<br><br>
Lys Glu Glu Cys Glu Ser Phe Pro Glu Ala Leu Lys Pro Gly Cys Asn<br><br>
180<br><br>
185<br><br>
190<br><br>
Trp Arg Phe Asp Trp Phe Gin Asn Ala Asp Asn Pro Ser Val Thr Phe<br><br>
195<br><br>
200<br><br>
205<br><br>
Gin Glu Val Ala Cys Pro Ser Glu Leu Thr Ser Lys Ser Gly Cys Ser<br><br>
210<br><br>
215<br><br>
220<br><br>
Arg Asn Asp Asp Gly Asn Phe Pro Ala Val Gin lie Pro Ser Ser Ser<br><br>
225	230	235	240<br>
Thr Ser Ser Pro Val Asn Gin Pro Thr Ser Thr Ser Thr Thr Ser Thr<br>
245	250	255<br>
Ser Thr Thr Ser Ser Pro Pro Val Gin Pro Thr Thr Pro Ser Gly Cys<br>
260	265	270<br>
Thr Ala Glu Arg Trp Ala Gin Cys Gly Gly Asn Gly Trp Ser Gly Cys<br>
275	2S0	285<br>
Thr  Thr  Cys  Val  Ala  Gly  Ser  Thr  Cys  Thr  Lys   lie  Asn  Asp  Trp  Tyr<br>
290	255	300<br>
His  Gin  Cys  Leu 305<br>
(2)    INFORMATION   FOR   SEQ   ID  NO:   4:<br>
(i)   SEQUENCE  CHARACTERISTICS:<br>
(A)	LENGTH:   888   base  pairs<br>
(B)	TYPE:   nucleic  acid<br>
(C)	STRANDEDNESS:   single<br>
(D)	TOPOLOGY:   linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(vi) ORIGINAL SOURCE:<br>
(A) ORGANISM: Saccharoinyces cerevisiae, DSM 10082<br><br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 4:<br>
CCAGTGTGCT GGAAAGCCTT CGTGCTGTCC CCGACGTATC CCTGACCGCC ATGCGTTCCA 60<br>
CCAGCATCTT GATCGGCCTT GTTGCCGGCG TCGCTGCTCA GAGCTCTGGC TCTGGCCATA 120<br>
CAACCAGGTA CTGGGACTGC TGCAAGCCCT CATGCGCCTG GGATGAGAAG GCGGCTGTCA 180<br>
GCCGGCCGGT CACAACATGC GACAGGAACA ACAGCCCCCT TTCGCCCGGC GCTGTGAGCG 240<br>
GCTGCGACCC CAACGGCGTT GCATTCACCT GCAACGACAA CCAGCCTTGG GCCGTAAACA 300<br>
ACAATGTCGC CTACGGTTTT GCGGCTACCG CCTTCCCTGG TGGCAATGAG GCGTCGTGGT 360<br>
GCTGTGCCTG CTATGCTCTT CAATTCACAT CCGGCCCCGT TGCTGGCAAG ACGATGGTTG 420<br>
TGCAATCCAC CAACACTGGC GGAGATCTCA GCGGCACTCA CTTCGATATC CAGATGCCCG 480<br>
GTGGAGGTCT CGGCATCTTC GACGGCTGCA CCCCGCAGTT CGGCTTCACG TTCCCCGGCA 540<br>
ACCGCTACGG CGGTACCACG AGCCGCAGCC AGTGCGCCGA GCTGCCCTCC GTCCTCCGTG 600<br>
ACGGCTGCCA CTGGCGTTAC GACTGGTTCA ACGATGCCGA CAACCCCAAC GTCAACTGGC 660<br>
GCCGCGTCCG ATGCCCGGCG GCCCTCACGA ACCGCTCCGG CTGCGTCCGC AACGACGACA 720<br>
ACAGCTACCC CGTCTTCGAG CCCGGCACGG GCACCCCGCC GACCCCCACG ACCACGACTA 780<br>
CCAGCTCCCC TCCTCAGCCC ACCAACGGCG GAGGCGGCGG CACTTCTCCT CACTGGGGCC 840<br><br>
AGTGCGGCGG CCAGGGCTGG TCTGGCCCGA CGGCCTGTGC CGGTGGGTCG ACCTGCAACC 900<br>
TGATCAACCC GTGGTACTCC CAGTGCATTC CCAACTAAGT GATCCGGGCA TTGCGGTCGA 960<br>
AAGGGGACCG TTAGTCGACA AGGCCCAGCC AGACCTCAGG CAGGTGGCTG CCATGGCAGA<br>
1020<br>
TTGTATATAG TCTTCCGAGT ACATACTATT GAATGAAAAT AAGAGCGGCT CGGACCATGA<br>
1080<br>
GCAGATGCCA TTTGATAAAA AAAAAAAAAA AAAAAAAAAA AAAAAAAAAA AAAAAAAAAA<br>
1140<br>
AAAAAAAAAA AAAA<br>
1154<br>
(2) INFORMATION FOR SEQ ID NO: 5:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 295 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS; single<br>
(D)	TOPOLOGY: linear<br>
(ii)   MOLECULE  TYPE:   peptide<br>
(xi)    SEQUENCE  DESCRIPTION:   SEQ   ID  NO:   5:<br>
Met Arg  Ser Thr  Ser  lie Leu  lie Gly Leu Val  Ala Gly Val  Ala Ala<br>
15	10	15<br>
Gin Ser Ser Gly Ser Gly His Thr Thr Arg Tyr Trp Asp Cys Cys Lys<br>
20	25	30<br><br>
Pro Ser Cys Ala Trp Asp Glu Lys Ala Ala Val Ser Arg Pro Val Thr<br><br>
35<br><br>
40<br><br>
45<br><br>
Thr Cys Asp Arg Asn Asn Ser Pro Leu Ser Pro Gly Ala Val Ser Gly<br><br>
50<br><br>
55<br><br>
60<br><br>
Cys Asp Pro Asn Gly Val Ala Phe Thr Cys Asn Asp Asn Gin Pro Trp<br><br>
65<br><br>
70<br><br>
75<br><br>
80<br><br>
Ala Val Asn Asn Asn Val Ala Tyr Gly Phe Ala Ala Thr Ala Phe Pro<br><br>
85<br><br>
90<br><br>
95<br><br>
Gly Gly Asn Glu Ala Ser Trp Cys Cys Ala Cys Tyr Ala Leu Gin Phe<br><br>
100<br><br>
105<br><br>
110<br><br>
Thr Ser Gly Pro Val Ala Gly Lys Thr Met Val Val Gin Ser Thr Asn<br><br>
115<br><br>
120<br><br>
125<br><br>
Thr Gly Gly Asp Leu Ser Gly Thr His Phe Asp lie Gin Met Pro Gly<br><br>
130<br><br>
135<br><br>
140<br><br>
Gly Gly Leu Gly lie Phe Asp Gly Cys Thr Pro Gin Phe Gly Phe Thr<br><br>
145<br><br>
150<br><br>
155<br><br>
160<br><br>
Phe Pro Gly Asn Arg Tyr Gly Gly Thr Thr Ser Arg Ser Gin Cys Ala<br><br>
165<br><br>
170<br><br>
175<br><br>
Glu Leu Pro Ser Val Leu Arg Asp Gly Cys His Tjrp Arg Tyr Asp Trp<br>
180	185	190<br>
Phe Asn Asp Ala Asp Asn Pro Asn Val Asn Trp Arg Arg Val Arg Cys<br>
195	200	205<br>
Pro Ala Ala Leu Thr Asn Arg Ser Gly Cys Val Arg Asn Asp Asp Asn<br>
210	215	220<br>
Ser Tyr Pro Val Phe Glu Pro Gly Thr Gly Thr Pro Pro Thr Pro Thr<br>
225	230	235	240<br>
Thr Thr Thr Thr Ser Ser Pro Pro Gin Pro Thr Asn Gly Gly Gly Gly<br>
245	250	255<br>
Gly Thr Ser Pro His Trp Gly Gin Cys Gly Gly Gin Gly Trp Ser Gly<br>
260	265	270<br>
Pro Thr Ala Cys Ala Gly Gly Ser Thr Cys Asn Leu lie Asn Pro Trp<br>
275	280	285<br>
Tyr Ser Gin Cys lie Pro Asn<br>
290	295<br>
(2) INFORMATION FOR SEQ ID NO: 6:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A) LENGTH: 1423 base pairs<br><br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(Vi) ORIGINAL SOURCE:<br>
(A) ORGANISM: Saccharomyces cerevisiae, DSM 10080<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 6:<br>
AAAGTTCTGG CCGGAACAGA TCTCCGTTGT CGATCTTCGA TTTTCCAGAC TCAGTCTGTG 60<br>
ACACTCCTTC AATCCACATT CCTTTACTTC TTCGTCACTC ATTCACATCA TGATTTCAGC 120<br>
TTGGATTCTC CTGGGGCTGG TAGGCGCCGT GCCCTCCTCC GTCATGGCCG CCTCGGGCAA 180<br>
AGGCCACACC ACCCGCTACT GGGATTGCTG CAAGACTTCT TGCGCATGGG AGGGCAAGGC 240<br>
ATCCGTCTCC GAGCCTGTCC TGACCTGTAA CAAGCAGGAC AACCCCATCG TCGATGCCAA 300<br>
CGCCAGAAGC GGCTGCGACG GCGGCGGGGC ATTTGCCTGT ACCAACAATT CCCCTTGGGC 360<br>
CGTGAGCGAG GACCTGGCCT ACGGATTTGC TGCCACAGCC CTCAGCGGCG GCACTGAGGG 420<br>
CAGCTGGTGC TGCGCGTGTT ACGCCATCAC ATTCACGAGT GGCCCTGTGG CTGGCAAGAA 480<br>
GATGGTCGTC CAGTCCACGA ACACGGGAGG CGACCTGTCC AACAACCACT TTGACCTGAT 540<br>
GATTCCCGGT GGAGGCCTCG GCATCTTTGA CGGTTGCTCG GCTCAGTTCG GACAACTTCT 600<br><br>
TCCCGGCGAG CGTTACGGAG GTGTTTCGTC CCGCTCTCAA TGCGATGGCA TGCCCGAGCT 660<br>
CTTGAAAGAC GGTTGCCAGT GGCGCTTCGA CTGGTTCAAG AACTCAGACA ACCCTGACAT 720<br>
CGAGTTCGAG CAGGTCCAGT GTCCCAAAGA GCTCATTGCG GTCTCTGGGT GCGTCCGTGA 780<br>
CGACGATAGC AGCTTTCCCG TCTTCCAAGG TTCGGGCTCA GGAGATGTCA ACCCACCTCC 840<br>
CAAGCCGACT ACGACTACGA CCTCGTCAAA GCCGAAAACA ACCTCTGCAC CATCCACTCT 900<br>
CTCGAACCCA TCCGCCCCTC AACAGCCAGG GAACACTGAT AGACCTGCCG AGACAACCAC 960<br>
TACCAAGCTG CCTGCCCTGC CGGCCACGAC GAGCAGCCCT GCTGTCTCAG TTCCTTCGTC 1020<br>
CAGCGCTCGC GTGCCTTTGT GGGGGCAATG CGACTCGGAA GCTTCATGGG ACGCACCTAA 1080<br>
GAAGTGTGCA AAGGGCACCA AGTGTGTCTA CGTCAACGAC TGGTACTCTC AATGCCAGCC 1140<br>
GAAGAACTCT TGTGCTTGAG AAGCAATGCT CACAGCATGT CCTCTTGTCA CCCCTTCTTT 1200<br>
TCATTCCCAA ACATACTTAC TGTATTATTA TTTCCGATGC TTCATTTCTT GCTTGTTTCT 1260<br>
GTCTTTCCTG CACGCAGCTT TCAACGATAC CCTTCATGCG ATTGCCCTAC GATCAGATGA 1320<br>
TGGGCACGAC ATGGAGGATG GTTTGGGCAC TCACGCGTTC AGGACGGGAA AATTTATTAG 1380<br>
GGCTGAGATC CGTGAATTGA CTTCATTTCG GCGGAATGTC TGC 1423<br><br>
INFORMATION FOR SEQ ID NO: 7:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 349 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii)   MOLECULE TYPE:   protein<br>
(xi)    SEQUENCE  DESCRIPTION:   SEQ  ID  NO:   7:<br>
Met  lie  Ser Ala Trp lie Leu Leu Gly Leu Val Gly Ala Val  Pro Ser<br>
15	10	15<br>
Ser Val Met Ala Ala Ser Gly Lys Gly'His  Thr Thr Arg Tyr Trp Asp<br>
20	25	30<br>
Cys  Cys Lye Thr  Ser  Cys Ala  Trp Glu Gly Lys  Ala  Ser Val  Ser Glu<br>
35	40	45<br>
Pro Val  Leu  Thr Cys  Asn Lys  Gin Asp Asn  Pro  lie Val  Asp Ala Asn<br>
50	55	60<br>
Ala Arg  Ser Gly  Cys Asp Gly Gly Gly Ala  Phe Ala  Cys  Thr Asn Asn<br>
65	70	75	80<br>
Ser  Pro Trp Ala Val  Ser Glu  Asp Leu  Ala Tyr Gly Phe Ala Ala  Thr<br>
85	90	95<br><br>
Ala Leu Ser Gly Gly Thr Glu Gly Ser Trp Cys Cys Ala Cys Tyr Ala<br>
100	105	110<br>
He Thr Phe Thr Ser Gly Pro Val Ala Gly Lys Lys Met Val Val Gin<br>
115	120	125<br>
Ser Thr Asn Thr Gly Gly Asp Leu Ser Asn Asn His Phe Asp Leu Met<br>
130	135	140<br>
He Pro Gly Gly Gly Leu Gly He Phe Asp Gly Cys Ser Ala Gin Phe<br>
145	150	155	160<br>
Gly Gin Leu Leu Pro Gly Glu Arg Tyr Gly Gly Val Ser Ser Arg Ser<br>
165	170	175<br>
Gin Cys Asp Gly Met Pro Glu Leu Leu Lys Asp Gly Cys Gin Trp Arg<br>
180	185	190<br>
Phe Asp Trp Phe Lys Asn Ser Asp Asn Pro Asp He Glu Phe Glu Gin<br>
195	200	205<br>
Val Gin Cys Pro Lys Glu Leu He Ala Val Ser Gly Cys Val Arg Asp<br>
210	215	220<br>
Asp Asp Ser Ser Phe Pro Val Phe Gin Gly Ser Gly Ser Gly Asp Val<br>
225	230	235	240<br><br>
Asn Pro Pro Pro Lys Pro Thr Thr Thr	Thr Thr	Ser Ser Lys Pro Lys<br>
245	250	255<br>
Thr Thr Ser Ala Pro Ser Thr Leu Ser	Asn Pro	Ser Ala Pro Gin Gin<br>
250                265	270<br>
Pro Gly Asn Thr Asp Arg Pro Ala Glu	Thr Thr	Thr Thr Lys Leu Pro<br>
275                280	285<br>
Ala Leu Pro Ala Thr Thr Ser Ser Pro	Ala Val	Ser Val Pro Ser Ser<br>
290	295	300<br>
Ser Ala Arg Val Pro Leu Trp Gly Gin	Cys Asp	Ser Glu Ala Ser Trp<br>
305                310                315	320<br>
Asp Ala Pro Lys Lys Cys Ala Lys Gly	Thr Lys	Cys Val Tyr Val Asn<br>
325	330	335<br>
Asp Trp Tyr Ser Gin Cys Gin Pro Lys Asn Ser Cys Ala<br>
340	345<br>
(2) INFORMATION FOR SEQ ID NO: 8:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 1174 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br><br>
(ii) MOLECULE TYPE: cDNA<br>
(vi) ORIGINAL SOURCE:<br>
(A) ORGANISM: Saccharomyces cerevisiae, DSM<br>
; 10081<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 8 I<br>
GAGCAGCACC CCTCAAGCTG TACAGTTTCC ACCCCGCTCT CTTTTCTTCG GCCCCCAGGA 60<br>
TGCGCTCTAC TCCCGTTCTT CGCACAACCC TGGCOGCTGC ACTTCCTCTG GTCGCCTCCG 120<br>
CGGCCAGTGG CAGTGGCCAG TCCACXJAGAT ACTGGGACTG CTGC3AGCCG  TCGTGCGCTT 180<br>
GGCCCGGGAA GGCCGCCGTC AGCCAACCGG TCTACGCGTG CGATGCCAAC TTCCAGCGCC 240<br>
TGTCCGACTT CAATGTCCAG TCGGGCTGCA ACGGCGGCTC GGCCTACTCC TGCGCCGACC 300<br>
AGACTCCCTG GGCGGTGAAC GACAATCTCG CCTACGGCTT CGCCGCGACG AGCATCGCCG 360<br>
GCGGGTCCGA ATCCTCGTGG TGCTGCGCCT GCTACGCGCT CACCTTCACT TCCGGTCCCG 420<br>
TCGCCGGCAA GACAATGGTG GTGCAGTCAA CGAGCACTGG CGGCGACCTG GGAAGTAACC 480<br>
AGTTCGATAT CGCCATGCCC GGCGGCGGCG TGGGCATCTT CAACGGCTGC AGCTCGCAGT 540<br>
TCGGCGGCCT CCCCGGCGCT CAATACGGCG GCATTTCGTC GCGCGACCAG TGCGATTCCT 600<br>
TCCCCGCGCC GCTCAAGCCC GGCTGCCAGT GGCGGTTTGA CTGGTTCCAG AACGCCGACA 660<br><br>
ACCCGACGTT  CACGTTCCAG  CAGGTGCAGT   GCCCCGCCGA GATCGTTGCC  CGCTCCGGCT 720<br>
GCAAGCGCAA  CGACGACTCC AGCTTCCCCG  TCTTCACCCC  CCCAAGCGGT  GGCAACGGTG 780<br>
GCACCGGGAC  GCCCACGTCG  ACTGCGCCTG  GGTCGGGCCA GACGTCTCCC  GGCGGCGGCA 840<br>
GTGGCTGCAC  GTCTCAGAAG  TGGGCTCAGT  GCGGTGGCAT  CGGCTTCAGC  GGATGCACCA 900<br>
CCTGTGTCTC  TGGCACCACC  TGCCAGAAGT  TGAACGACTA  CTACTCGCAG   TGCCTCTAAA 960<br>
CAGCTTTTCG  CACGAGGTGG CGGGAC»GAG  CAAGGAGACC  GTCAACTTCG  TCATGCATAT 1020<br>
TTTTTGAGCG  CTCAATACAT ACATAACCTT  CGATTCTTGT  ACATAGCACG  CCGGTACACA 1080<br>
TCTCACACCG  ACTTTGGGGG  CGGAATCAGG   CCCGTTTTAA  AAAAAAAAAA  AAAAAAAAAA 1140<br>
AAAAAAAAAA  AAAAAAAAAA AAAAAAAAAA  AAAA 1174<br>
(2)   INFORMATION  FOR  SEQ   ID  NO:   9:<br>
(i)   SEQUENCE  CHARACTERISTICS:<br>
(A)	LENGTH:   299   amino  acids<br>
(B)	TYPE:   amino acid<br>
(C)	STRANDEDNESS:   single<br>
(D)	TOPOLOGY:   linear<br>
(ii)   MOLECULE TYPE:   peptide<br>
(xi)    SEQUENCE  DESCRIPTION:   SEQ   ID  NO:   9:<br>
Met Arg  Ser Thr  Pro Val  Leu Arg Thr  Thr Leu Ala Ala Ala Leu  Pro<br>
15	10	15<br><br>
Leu Val Ala Ser Ala Ala Ser Gly Ser Gly Gin Ser Thr Arg Tyr Trp<br>
20	25	30<br>
Asp Cys Cys Lys Pro Ser Cys Ala Trp Pro Gly Lys Ala Ala Val Ser<br>
35	40	45<br>
Gin Pro Val Tyr Ala Cys Asp Ala Asn Phe Gin Arg Leu Ser Asp Phe<br>
50	55	60<br>
Asn Val Gin Ser Gly Cys Asn Gly Gly Ser Ala Tyr Ser Cys Ala Asp<br>
65	70	75	80<br>
Gin Thr Pro Trp Ala Val Asn Asp Asn Leu Ala Tyr Gly Phe Ala Ala<br>
85	90	95<br>
Thr Ser lie Ala Gly Gly Ser Glu Ser Ser Trp Cys Cys Ala Cys Tyr<br>
100	105	110<br>
Ala Leu Thr Phe Thr Ser Gly Pro Val Ala Gly Lys Thr Met Val Val<br>
115	120	125<br>
Gin Ser Thr Ser Thr Gly Gly Asp Leu Gly Ser Asn Gin Phe Asp lie<br>
130	135	140<br>
Ala Met Pro Gly Gly Gly Val Gly lie Phe Asn Gly Cys Ser Ser Gin<br><br>
145<br><br>
150<br><br>
155<br><br>
160<br><br>
Phe Gly Gly Leu Pro Gly Ala Gin Tyr Gly Gly lie Ser Ser Arg Asp<br><br>
165<br><br>
170<br><br>
175<br><br>
Gin Cys Asp Ser Phe Pro Ala Pro Leu Lys Pro Gly Cys Gin Trp Arg<br><br>
180<br><br>
185<br><br>
190<br><br>
Phe Asp Trp Phe Gin Asn Ala Asp Asn Pro Thr Phe Thr Phe Gin Gin<br><br>
195<br><br>
200<br><br>
205<br><br>
Val Gin Cys Pro Ala Glu lie Val Ala Arg Ser Gly Cya Lys Arg Asn<br><br>
210<br><br>
215<br><br>
220<br><br>
Asp Asp Ser Ser Phe Pro Val Phe Thr Pro Pro Ser Gly Gly Asn Gly<br><br>
225<br><br>
230<br><br>
235<br><br>
240<br><br>
Gly Thr Gly Thr Pro Thr Ser Thr Ala Pro Gly Ser Gly Gin Thr Ser<br><br>
245<br><br>
250<br><br>
255<br><br>
Pro Gly Gly Gly Ser Gly Cys Thr Ser Gin Lys Trp Ala Gin Cys Gly<br><br>
260<br><br>
265<br><br>
270<br><br>
Gly lie Gly Phe Ser Gly Cys Thr Thr Cys Val Ser Gly Thr Thr Cys<br><br>
275<br><br>
280<br><br>
285<br><br>
Gin Lys Leu Asn Asp Tyr Tyr Ser Gin Cys Leu<br>
290	295<br><br>
(2) INFORMATION FOR SEQ ID NO: 10:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 913 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(vi) ORIGINAL SOURCE:<br>
(A) ORGANISM: Escherichia coli, DSM 10512<br>
(Xi) SEQUENCE DESCRIPTION: SEQ ID NO: 10:<br>
GCACTATTCT CAGCTCCATT CTCCCTTGAA GTAATTCACC ATGTTCTCTC CGCTCTGGGC 60<br>
CCTGTCGGCT CTGCTCCTAT TTCCTGCCAC TGAAGCCACT AGCGGCGTGA CAACCAGGTA 120<br>
CTGGGACTGC TGCAAGCCGT CTTGTGCTTG GACGGGCAAA GCATCCGTCT CCAAGCCCGT 180<br>
CGGAACCTGC GACATCAACG ACAACGCCCA GACGCCGAGC GATCTGCTCA AGTCGTCCTG 240<br>
TGATGGCGGC AGCGCCTACT ACTGCAGCAA CCAGGGCCCA TGGGCCGTGA ACGACAGCCT 300<br>
TTCCTACGGC TTCGCTGCCG CCAAGCTGTC CGGAAAGCAG GAGACTGATT GGTGCTGTGG 360<br>
CTGCTACAAG CTCACATTCA CCTCCACCGC CGTTTCCGGC AAGCAAATGA TCGTGCAAAT 420<br>
CACGAACACG GGCGGCGhCC  TCGGCAACAA CCACTTCGAC ATCGCCATGC CGGGCGGCGG 480<br>
CGTCGGCATC TTCAACGGGT GCTCCAAGCA ATGGAACGGC ATCAATCTGG GCAACCAGTA 540<br><br>
TGGCGGCTTC ACTGACCGCT CGCAATGTGC GACGCTCCCG TCCAAGTGGC AGGCCAGCTG 600<br>
CAACTGGCGC TTCGACTGGT TCGAGAATGC CGACAACCCC ACCGTCGATT GGGAGCCTGT 660<br>
CACTTGCCCA CAGGAATTGG TCGCCCGGAC TGGCTGTTCC CGTACCTAAG TGGGGGTGGA 720<br>
ACCTCCATGT GAATTGGTGT ATATAGCTCC TGCCTGAGCA TCCACCAGTT CGCATGTGTT 780<br>
6ATCAGGAGT TGTGTTGCCT TGCTAGGAAA GACTTTGTTG GAAACTTGCG TGTTTATTCC 840<br>
AATTGAATAA CCCTGTATAG ACCGGTCACA TTTTTCTCTG AAAAAAAAAA AAAAAAAAAA 900<br>
AAAAAAAAAA AAA<br>
913<br>
(2) INFORMATION FOR SEQ ID NO: 11:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 222 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii)   MOLECULE TYPE:   protein<br>
(xi)   SEQUENCE  DESCRIPTION:   SEQ   ID NO:    11:<br>
Met Phe  Ser Pro Leu Trp Ala Leu  Ser Ala Leu Leu Leu  Phe Pro Ala<br>
1	5	10	15<br>
Thr Glu Ala Thr Ser Gly Val  Thr Thr Arg Tyr Trp Asp Cys Cys  Lys<br>
20	25	30<br><br>
Pro Ser Cys Ala Trp Thr Gly Lys Ala Ser Val Ser Lys Pro Val Gly<br>
35	40	45<br>
Thr Cys Asp lie Asn Asp Asn Ala Gin Thr Pro Ser Asp Leu Leu Lys<br>
50	55	60<br>
Ser Ser Cys Asp Gly Gly Ser Ala Tyr Tyr Cya Ser Asn Gin Gly Pro<br>
65	70	75	80<br>
Trp Ala Val Asn Asp Ser Leu Ser Tyr Gly Phe Ala Ala Ala Lys Leu<br>
85	90	95<br>
Ser Gly Lys Gin Glu Thr Asp Trp Cys Cys Gly Cys Tyr Lys Leu Thr<br>
100	105	110<br>
Phe Thr Ser Thr Ala Val Ser Gly Lys Gin Met He Val Gin He Thr<br>
115	120	125<br>
Asn Thr Gly Gly Asp Leu Gly Asn Asn His Phe Asp He Ala Met Pro<br>
130	135	140<br>
Gly Gly Gly Val Gly He Phe Asn Gly Cys Ser Lys Gin Trp Asn Gly<br>
145	150	155	160<br>
He Asn Leu Gly Asn Gin Tyr Gly Gly Phe Thr Asp Arg Ser Gin Cys<br>
165	170	175<br><br>
Ala Thr Leu  Pro Ser Lys  Trp Gin Ala Ser Cys Asn Trp Arg Phe Asp<br>
180	185	190<br>
Trp Phe Glu Asn Ala Asp Asn Pro Thr Val Asp Trp Glu  Pro Val  Thr<br>
195	200	205<br>
Cys  Pro Gin Glu Leu Val Ala Arg Thr Gly Cys  Ser Arg Thr<br>
210	215	220<br>
2)   INFORMATION  FOR  SEQ   ID NO:   12:<br>
(i)   SEQUENCE  CHARACTERISTICS:<br>
(A)	LENGTH:   808   base pairs<br>
(B)	TYPE:   nucleic acid<br>
(C)	STRANDEDNESS:   single<br>
(D)	TOPOLOGY:   linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(vi) ORIGINAL SOURCE:<br>
(A) ORGANISM: Escherichia coli, DSM 10511<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 12:<br>
CCGCTGCTGG GTATATAATG CTCAGACTTG GAACCAATGG TCCATCCAAA CATGCTTAAA 60<br>
ACGCTCGCTC CATTGATCAT CTTGGCCGCC TCGGTCACAG CGCAAACAGC AGGAGTTACG 120<br>
ACCCGCTACT GGGACTGCTG CAAGCCAAGC TGTGGATGGA GTGGAAAGGC TTCTGTTTCT 180<br>
GCTCCAGTCA GAACTTGCGA TCGTAATGGA AATACACTTG GCCCAGACGT GAAAAGCGGA 240<br><br>
TGTGATAGCG  GTGGAACGTC  ATTCACTTGC  GCGAACAATG  GTCCATTTGC  GATTGACAAT 300<br>
AACACTGCAT  ATGGTTTTGC  TGCAGCCCAC  TTAGCGGGCT   CTAGCGAAGC  A6CCTGGTGT 360<br>
TGCCAGT6CT  ACGAATTGAC  GTTTACGAGT  GGACCCGTAG  TTGGGAAGAA ACTGACCGTT 420<br>
CAAGTCACAA ACACGGGAGG  TGACCTCGGA AATAATCACT   TTGACCTGAT  GATCCCCGGT 480<br>
GGAGGTGTTG  GCCTCTTCAC ACAAGGATGT  CCTGCTCAGT  TTGGGAGCTG  GAACGGGGGT 540<br>
GCTCAATACG GGGGTGTGTC  CAGCCGTGAC  CAATGCTCCC AACTTCCAGC AGCTGTGCAA 600<br>
GCTGGATGTC AATTCCGTTT CGACTGGATG  GGTGGCGCGG  ATAACCCCAA  CGTCACCTTC 660<br>
CGACCTGTGA  CCTGCCCAGC  GCAGCTCACT  AATATCTCGG  GCTGTGTTCG  TAAATGATTC 720<br>
f      ACGAATATGT  AGTGTCGAAT  ATGTACATGT  GTATGTACTA  TAGCTTCAAA  GATGGAGGGT 780<br>
CTGTTTAAAA  AAAAAAAAAA  AAAAAAAA 808 )<br>
(2)   INFORMATION   FOR  SEQ  ID  NO:   13:<br>
(i)   SEQUENCE  CHARACTERISTICS:<br>
(A)	LENGTH:   226  amino acids<br>
(B)	TYPE:   amino  acid<br>
(C)	STRANDEDNESS:   single<br>
(D)	TOPOLOGY:   linear<br>
(ii) MOLECULE TYPE: protein<br><br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 13:<br>
Met Val His Pro Asn Met Leu Lys Thr Leu Ala Pro Leu lie lie Leu<br>
15	10	15<br>
Ala Ala Ser Val Thr Ala Gin Thr Ala Gly Val Thr Thr Arg Tyr Trp<br>
20	25	30<br>
Asp Cys Cys Lys Pro Ser Cys Gly Trp Ser Gly Lys Ala Ser Val Ser<br>
35	40	45<br>
Ala Pro Val Arg Thr Cys Asp Arg Asn Gly Asn Thr Leu Gly Pro Asp<br>
50	55	60<br>
Val Lys Ser Gly Cys Asp Ser Gly Gly Thr Ser Phe Thr Cys Ala Asn<br>
65	70	75	80<br>
Asn Gly Pro Phe Ala lie Asp Asn Asn Thr Ala Tyr Gly Phe Ala Ala<br>
85	90	95<br>
Ala His Leu Ala Gly Ser Ser Glu Ala Ala Trp Cys Cys Gin Cys Tyr<br>
100	105	110<br>
Glu Leu Thr Phe Thr Ser Gly Pro Val Val Gly Lys Lys Leu Thr Val<br>
115	120	125<br>
Gin Val Thr Asn Thr Gly Gly Asp Leu Gly Asn Aen His Phe Asp Leu<br><br>
130	135	140<br>
Met lie Pro Gly Gly Gly Val Gly Leu Phe Thr Gin Gly Cys Pro Ala<br>
145	150	155	160<br>
Gin Phe Gly Ser Trp Asn Gly Gly Ala Gin Tyr Gly Gly Val Ser Ser<br>
165	170	175<br>
Arg Asp Gin Cys Ser Gin Leu Pro Ala Ala Val Gin Ala Gly Cys Gin<br>
180	185	190<br>
Phe Arg Phe Asp Trp Met Gly Gly Ala Asp Asn Pro Asn Val Thr Phe<br>
195	200	205<br>
Arg Pro Val Thr Cys Pro Ala Gin Leu Thr Asn lie Ser Gly Cys Val<br>
210	215	220<br>
Arg Lys 225<br>
(2) INFORMATION FOR SEQ ID NO: 14-A:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 1048 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: CDNA<br><br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 14-A:<br>
GACTTGGAAC CAATGGTCCA TCCAAACATG CTTAAAACGC TCGCTCCATT GATCATCTTG 60<br>
GCCGCCTCGG TCACAGCGCA AACAGCAGGA GTTACGACCC GCTACTGGGA CTGCTGCAAG 120<br>
CCAAGCTGTG GATGGAGTGG AAAGGCTTCT GTTTCTGCTC CAGTCAGAAC TTGCGATCGT 180<br>
AATGGAAATA CACTTGGCCC AGACGTGAAA AGCGGATGTG ATAGCGGTGG AACGTCATTC 240<br>
ACTTGCGCGA ACAATGGTCC ATTTGCGATT GACAATAACA CTGCATATGG TTTTGCTGCA 300<br>
GCCCACTTAG CGGGCTCTAG CGAAGCAGCC TGGTGTTGCC AGTGCTACGA ATTGACGTTT 360<br>
ACGAGTGGAC CCGTAGTTGG GAAGAAACTG ACCGTTCAAG TCACAAACAC GGGAGGTGAC 420<br>
CTCGGAAATA ATCACTTTGA CCTGATGATC CCCGGTGGAG GTGTTGGCCT CTTCACACAA 480<br>
GGATGTCCTG CTCAGTTTGG GAGCTGGAAC GGGGGTGCTC AATACGGGGG TGTGTCCAGC 540<br>
CGTGACCAAT GCTCCCAACT TCCAGCAGCT GTGCAAGCTG GATGTCAATT CCGTTTCGAC 600<br>
TGGATGGGTG GCGCGGATAA CCCCAACGTC ACCTTCCGAC CTGTGACCTG CCCAGCGCAG 660<br>
CTCACTAATA TCTCGGGCTG TGTTCGTAAA CCCTCCAGCA GCACCAGCTC TCCGGTCAAC 720<br>
CAGCCTACCA GCACCAGCAC CACGTCCACC TCCACCACCT CGAGCCCGCC AGTCCAGCCT 780<br>
ACGACTCCCA GCGGCTGCAC TGCTGAGAGG TGGGCTCAGT GCGGCGGCAA TGGCTGGAGC 840<br><br>
G6CTGCACCA  CCTGCGTCGC  TGGCAGCACT  TGCACGAAGA   TTAATGACTG  GTACCATCAG 900<br>
TGCCTGTAGA  CGCAGGGCAG  CTTGAGGGCC  TTACTGGTGG   CGCAACGAAA  TGACACTCCC 960<br>
AATCACTGTA  TTAGTTCTTG  TACATAATTT  CGTCATCCCT  CCAGGGATTG   TCACATAAAT 1020<br>
GCAATGAGGA ACAATGAGTA  CAGAATTC 1048<br>
(2)   INFORMATION  FOR SEQ  ID NO:   14-B:<br>
(i)   SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH:   298   amino acids<br>
(B)	TYPE:   amino acid<br>
(C)	STRANDEDNESS:   single<br>
(D)	TOPOLOGY-   linear<br>
(ii)   MOLECULE TYPE:   peptide<br>
(xi)    SEQUENCE DESCRIPTION:   SEQ  ID NO:   14-B:<br>
Met Val  His  Pro Asn Met Leu Lys Thr Leu Ala  Pro Leu   lie  lie Leu<br>
15	10	15<br>
Ala Ala  Ser Val  Thr Ala Gin Thr Ala Gly Val  Thr  Thr Arg Tyr  Trp<br>
20	25	30<br>
Asp Cys  Cys Lys  Pro Ser  Cys Gly Trp  Ser Gly Lys Ala  Ser Val  Ser<br>
35	40	45<br>
Ala Pro Val Arg Thr  Cys  Asp Arg Asn Gly Asn Thr Leu Gly Pro Asp<br>
50	55	60<br><br>
Val Lys Ser Gly Cys Asp Ser Gly Gly Thr Ser Phe Thr Cys Ala Asn<br>
65	70	75	80<br>
Asn Gly Pro Phe Ala lie Asp Asn Asn Thr Ala Tyr Gly Phe Ala Ala<br>
85	90	95<br>
Ala His Leu Ala Gly Ser Ser Glu Ala Ala Trp Cys Cys Gin Cys Tyr<br>
100	105	110<br>
Glu Leu Thr Phe Thr Ser Gly Pro Val Val Gly Lys Lys Leu Thr Val<br>
115	120	125<br>
Gin Val Thr Asn Thr Gly Gly Asp Leu Gly Asn Asn His Phe Asp Leu<br>
130	135	140<br>
Met lie Pro Gly Gly Gly Val Gly Leu Phe Thr Gin Gly Cys Pro Ala<br>
145	150	155	160<br>
Gin Phe Gly Ser Trp Asn Gly Gly Ala Gin Tyr Gly Gly Val Ser Ser<br>
165	170	175<br>
Arg Asp Gin Cys Ser Gin Leu Pro Ala Ala Val Gin Ala Gly Cys Gin<br>
180	185	190<br>
Phe Arg Phe Asp Trp Met Gly Gly Ala Asp Asn Pro Asn Val Thr Phe<br><br>
195	200	205<br>
Arg Pro Val Thr Cys Pro Ala Gin Leu Thr Asn lie Ser Gly Cys Val<br>
210	215	220<br>
Arg Lys Pro Ser Ser Ser Thr Ser Ser Pro Val Asn Gin Pro Thr Ser<br>
225	230	235	240<br>
Thr Ser Thr Thr Ser Thr Ser Thr Thr Ser Ser Pro Pro Val Gin Pro<br>
245	250	255<br>
Thr Thr Pro Ser Gly Cys Thr Ala Glu Arg Trp Ala Gin Cys Gly Gly<br>
260	265	270<br>
Asn Gly Trp Ser Gly Cys Thr Thr Cys Val Ala Gly Ser Thr Cys Thr<br>
275	280	285<br>
Lys lie Asn Asp Trp Tyr His Gin Cys Leu<br>
290	295<br>
(2) INFORMATION FOR SEQ ID NO: 15-A:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 1031 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br><br>
(Xi) SEQUENCE DESCRIPTION: SEQ ID NO: 15-A:<br>
CCATCCAAAC ATGCTTAAAA CGCTCGCTCC ATTGATCATC TTGGCCGCCT C6GTCACAGC 60<br>
GCAAACAGCA GGAGTTACGA CCCGCTACTG GGACTGCTGC AAGCCAAGCT GTGGATGGAG 120<br>
TGGAAAGGCT TCTGTTTCTG CTCCAGTCAG AACTTGCGAT CGTAATGGAA ATACACTTGG 180<br>
CCCAGACGTG AAAAGCGGAT GTGATAGCGG TGGAACGTCA TTCACTTGCG CGAACAATGG 240<br>
TCCATTTGCG ATTGACAATA ACACTGCATA TGGTTTTGCT GCAGCCCACT TAGCGGGCTC 300<br>
TAGCGAAGCA GCCTGGTGTT GCCAGTGCTA CGAATTGACG TTTACGAGTG GACCCGTAGT 360<br>
TGGGAAGAAA CTGACCGTTC AAGTCACAAA CACGGGAGGT GACCTCGGAA ATAATCACTT 420<br>
TGACCTGATG ATCCCCGGTG GAGGTGTTGG CCTCTTCACA CAAGGATGTC CTGCTCAGTT 480<br>
TGGGAGCTGG AACGGGGGTG CTCAATACGG GGGTGTGTCC AGCCGTGACC AATGCTCCCA 540<br>
ACTTCCAGCA GCTGTGCAAG CTGGATGTCA ATTCCGTTTC GACTGGATGG GTGGCGCGGA 600<br>
TAACCCCAAC GTCACCTTCC GACCTGTGAC CTGCCCAGCG CAGCTCACTA ATATCTCGGG 660<br>
CTGTGTTCGT AAACCCTCCA GCAGCACCAG CTCTCCGGTC AACCAGCCTA CCAGCACCAG 720<br>
CACCACGTCC ACCTCCACCA CCTCGAGCCC GCCAGTCCAG CCTACGACTC CCAGCGGCTG 780<br>
CACTGCTGAG AGGTGGGCTC AGTGCGGCGG CAATGGCTGG AGCGGCTGCA CCACCTGCGT 840<br><br>
CGCTGGCAGC  ACTTGCACGA AGATTAATGA  CTGGTACCAT  CAGTGCCTGT  AGACGCAGGG 900<br>
CAGCTTGAGG  GCCTTACTGG  TGGCGCAACG  AAATGACACT  CCCAATCACT  GTATTAGTTC 960<br>
TTGTACATAA  TTTCGTCATC  CCTCCAGGGA TTGTCACATA  AATGCAATGA GGAACAATGA 1020<br>
GTACAGAATT C 1031<br>
(2)   INFORMATION  FOR SEQ  ID NO:   15-B:<br>
(i)   SEQUENCE  CHARACTERISTICS:<br>
(A)	LENGTH:   293   amino acids<br>
(B)	TYPE:   amino acid<br>
(C)	STRANDEDNESS:   single<br>
(D)	TOPOLOGY:   linear<br>
(ii)   MOLECULE TYPE:   peptide<br>
(Xi)   SEQUENCE  DESCRIPTION:   SEQ   ID  NO:   15-B:<br>
Met Leu  Lya  Thr Leu Ala  Pro Leu  lie  lie Leu Ala Ala  Ser Val  Thr<br>
15	10	15<br>
Ala Gin  Thr Ala Gly Val  Thr  Thr Arg Tyr Trp Asp Cys  Cys  Lys  Pro<br>
20	25	30<br>
Ser Cys  Gly Trp  Ser Gly Lys Ala Ser Val  Ser Ala Pro Val Arg  Thr<br>
35	40	45<br>
Cys Asp Arg Asn Gly Asn Thr Leu Gly Pro Asp Val Lys  Ser Gly Cys<br>
50	55	60<br><br>
Asp Ser Gly Gly Thr Ser Phe Thr Cys Ala Asn Asn Gly Pro Phe Ala<br>
65	70	75	80<br>
lie Asp Asn Asn Thr Ala Tyr Gly Phe Ala Ala Ala His Leu Ala Gly<br>
85	90	95<br>
Ser Ser Glu Ala Ala Trp Cys Cys Gin Cys Tyr Glu Leu Thr Phe Thr<br>
100	105	110<br>
Ser Gly Pro Val Val Gly Lys Lys Leu Thr Val Gin Val Thr Asn Thr<br>
115	120	125<br>
Gly Gly Asp Leu Gly Asn Asn His Phe Asp Leu Met lie Pro Gly Gly<br>
130	135	140<br>
Gly Val Gly Leu Phe Thr Gin Gly Cys Pro Ala Gin Phe Gly Ser Trp<br>
145	150	155	160<br>
Asn Gly Gly Ala Gin Tyr Gly Gly Val Ser Ser Arg Asp Gin Cys Ser<br>
165	170	175<br>
Gin Leu Pro Ala Ala Val Gin Ala Gly Cys Gin Phe Arg Phe Asp Trp<br>
180	185	190<br>
Met Gly Gly Ala Asp Asn Pro Asn Val Thr Phe Arg Pro Val Thr Cys<br><br>
195	200	205<br>
Pro Ala Gin Leu Thr Asn He Ser Gly Cys Val Arg Lys Pro Ser Ser<br>
210	215	220<br>
Ser Thr Ser Ser Pro Val Asn Gin Pro Thr. Ser Thr Ser Thr Thr Ser<br>
225	230	235	240<br>
Thr Ser Thr Thr Ser Ser Pro Pro Val Gin Pro Thr Thr Pro Ser Gly<br>
245	250	255<br>
Cys Thr Ala Glu Arg Trp Ala Gin Cys Gly Gly Asn Gly Trp Ser Gly<br>
260	265	270<br>
Cys Thr Thr Cys Val Ala Gly Ser Thr Cys Thr Lys He Asn Asp Trp<br>
275	280	285<br>
Tyr His Gin Cys Leu 290<br>
(2) INFORMATION FOR SEQ ID NO: 16:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 1132 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECtTLE TYPE: cDNA<br><br>
(vi) ORIGINAL SOURCE:<br>
(A) ORGANISM: Escherichia coli, DSM 10571<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 16:<br>
CAACAGTTCA AACACCTACA AGGTCCCGTG CCCTGTAGAC CATGCGTTCC TCTGCAGTCC 60<br>
TCATCGGCCT CGTGGCCGGT GTGGCCGCCC AGTCCTCTGG CACCGGCCGC ACCACCAGAT 120<br>
ACTGGGACTG CTGCAAGCCG TCCTGCGGGT GGGACGAAAA GGCCTCCGTC AGCCAGCCCG 180<br>
TCAAGACGTG CGATAGGAAC AACAACCCTC TCGCGTCCAC GGCCAGGAGC GGCTGCGATT 240<br>
CCAACGGCGT CGCCTACACG TGCAACGATA ACCAGCCGTG GGCTGTCAAC GATAACCTGG 300<br>
CCTATGGTTT TGCTGCCACG GCTTTCAGTG GTGGATCGGA GGCCAGCTGG TGCTGTGCCT 360<br>
GCTATGCCCT TCAGTTCACC TCCGGCCCTG TTGCGGGAAA GACCATGGTC GTCCAGTCGA 420<br>
CAAACACCGG CGGCGACCTC AGCGGCAACC ACTTTGACAT CCTCATGCCC GGCGGCGGCC 480<br>
TGGGCATCTT CGACGGCTGC ACCCCGCAAT GGGGCGTCAG CTTCCCCGGA AACCGCTACG 540<br>
GCGGCACCAC CAGCCGCAGC CAGTGCTCCC AAATCCCCTC GGCCCTGCAG CCCGGCTGCA 600<br>
ACTGGCGGTA CGACTGGTTC AACGACGCCG ACAACCCCGA CGTCTCGTGG CGCCGCGTCC 660<br>
AGTGCCCCGC CGCACTCACC GACCGCACCG GCTGCCGCCG CTCCGATGAC GGGAACTATC 720<br>
CCGTCTTCCA GCCCGGTCCG CCCCCGGCCA CGACGATCAG GACATCGACT ACCATCACAG 780<br><br>
CCTCATCGTC   GTCTTCGTCT  TCGTCGTCGT   CGACTACGGC  TGGTAGCCCG   CCTGTGCCGA 840<br>
CTGGTGGTGG  TAGTGGGCCA  ACGTCGCCTG  TCTGGGGACA  GTGCGGCGGT   CAGGGATGGA 900<br>
GTGGTCCTAC  GCGTTGTGTT  GCTGGGTCGA  CATGCAGTGT  GGTCAACCCG   TGGTACTCGC 960<br>
AGTGTTTTCC  TTAAGGAGCC  TCTGGCTGAG   CAGATCCTTT  CGAAGAGGAG  GGTCTCTCTG 1020<br>
CTCTTTCAGT  CTGTTCAGGG  AACGGCCGTC  TCGGCTACAT  TGTACATATC  CCACCTCGTA 1080<br>
TATAGCTAGC  TCATCTACAC  TTGTGATCTC  CAAAAAAAAA AAAAAAAAAA  AA<br>
1132<br>
(2) INFORMATION FOR SEQ ID NO: 17:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 310 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii)   MOLECULE TYPE:   peptide<br>
(xi)   SEQUENCE  DESCRIPTION:   SEQ ID  NO:   17:<br>
Met Arg  Ser  Ser Ala Val  Leu  lie Gly Leu Val  Ala Gly Val  Ala Ala<br>
15	10	15<br>
Gin Ser  Ser Gly Thr Gly Arg Thr  Thr Arg Tyr  Trp Asp Cys  Cys  Lys<br>
20	25	30<br>
Pro Ser  Cys  Gly  Trp Asp Glu Lys Ala  Ser Val  Ser Gin  Pro Val  Lys<br><br>
35<br><br>
40<br><br>
45<br><br>
Thr Cys Asp Arg Asn Asn Asn Pro Leu Ala Ser Thr Ala Arg Ser Gly<br><br>
50<br><br>
55<br><br>
60<br><br>
Cys Asp Ser Asn Gly Val Ala Tyr Thr Cys Asn Asp Asn Gin Pro Trp<br><br>
65<br><br>
70<br><br>
75<br><br>
80<br><br>
Ala Val Asn Asp Asn Leu Ala Tyr Gly Phe Ala Ala Thr Ala Phe Ser<br><br>
85<br><br>
90<br><br>
95<br><br>
Gly Gly Ser Glu Ala Ser Trp Cys Cys Ala Cys Tyr Ala Leu Gin Phe<br><br>
100<br><br>
105<br><br>
110<br><br>
Thr Ser Gly Pro Val Ala Gly Lys Thr Met Val Val Gin Ser Thr Asn<br><br>
115<br><br>
120<br><br>
125<br><br>
Thr Gly Gly Asp Leu Ser Gly Asn His Phe Asp He Leu Met Pro Gly<br><br>
130<br><br>
135<br><br>
140<br><br>
Gly Gly Leu Gly He Phe Asp Gly Cys Thr Pro Gin Trp Gly Val Ser<br><br>
145<br><br>
150<br><br>
155<br><br>
160<br><br>
Phe Pro Gly Asn Arg Tyr Gly Gly Thr Thr Ser Arg Ser Gin Cys Ser<br><br>
165<br><br>
170<br><br>
175<br><br>
Gin He Pro Ser Ala Leu Gin Pro Gly Cys Asn Trp Arg Tyr Asp Trp<br><br>
180<br><br>
185<br><br>
190<br><br>
Phe Asn Asp Ala Asp Asn Pro Asp Val Ser Trp Arg Arg Val Gin Cyj<br><br>
195<br><br>
200<br><br>
205<br><br>
Pro Ala Ala Leu Thr Asp Arg Thr Gly Cys Arg Arg Ser Asp Asp Gly<br><br>
210<br><br>
215<br><br>
220<br><br>
Asn Tyr Pro Val Phe Gin Pro Gly Pro Pro Pro Ala Thr Thr lie Arg<br><br>
225<br><br>
230<br><br>
235<br><br>
240<br><br>
Thr Ser Thr Thr lie Thr Ala Ser Ser Ser Ser Ser Ser Ser Ser Ser<br><br>
245<br><br>
250<br><br>
255<br><br>
Ser Thr Thr Ala Gly Ser Pro Pro Val Pro Thr Gly Gly Gly Ser Gly<br><br>
260<br><br>
265<br><br>
270<br><br>
Pro Thr Ser Pro Val Trp Gly Gin Cys Gly Gly Gin Gly Trp Ser Gly<br><br>
275<br><br>
280<br><br>
285<br><br>
Pro Thr Arg Cys Val Ala Gly Ser Thr Cys Ser Val Val Asn Pro Trp<br><br>
290<br><br>
295<br><br>
300<br><br>
Tyr Ser Gin Cys Phe Pro<br>
305	310<br><br>
(2) INFORMATION FOR SEQ ID NO: 18-A:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 885 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: CDNA<br>
(vi) ORIGINAL SOURCE:<br>
"Construction from Macrophomina"<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: CDS<br>
(B)	LOCATION:1..885<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 18-A:<br>
ATG TTC TCT CCG CTC TGG GCC CTG TCG GCT CTG CTC	CTA TTT CCT GCC<br>
48<br>
Met Phe Ser Pro Leu Trp Ala Leu Ser Ala Leu Leu	Leu Phe Pro Ala<br>
145                150	155<br>
ACT GAA GCC ACT AGC GGC GTG ACA ACC AGG TAC TGG GAC TGC TGC AAG<br>
96<br>
Thr Glu Ala Thr Ser Gly Val Thr Thr Arg Tyr Trp Asp Cys Cys Lys<br>
160	165	170<br>
CCG TCT TGT GCT TGG ACG GGC AAA GCA TCC GTC TCC AAG CCC GTC GGA 144<br>
Pro Ser Cys Ala Trp Thr Gly Lys Ala Ser Val Ser Lys Pro Val Gly<br>
175	180	185<br>
ACC TGC GAC ATC AAC GAC AAC GCC CAG ACG CCG AGC GAT CTG CTC AAG<br>
192<br>
Thr Cyg Asp He Asn Asp Asn Ala Gin Thr Pro Ser Asp Leu Leu Lys<br>
190	195	200	205<br>
TCG TCC TGT GAT GGC GGC AGC GCC TAC TAC TGC AGC AAC CAG GGC CCA<br><br>
240<br>
Ser Ser Cys Asp Gly Gly Ser Ala Tyr Tyr Cys Ser Asn Gin Gly Pro<br>
210	215	220<br>
TGG GCC GTG AAC GAC AGC CTT TCC TAC GGC TTC GCT GCC GCC AAG CTG 288<br>
Trp Ala Val Asn Asp Ser Leu Ser Tyr Gly Phe Ala Ala Ala Lys Leu<br>
225	230	235<br>
TCC GGA AAG CAG GAG ACT GAT TGG TGC TGT GGC TGC TAC AAG CTC ACA 336<br>
Ser Gly Lys Gin Glu Thr Asp Trp Cys Cys Gly Cys Tyr Lys Leu Thr<br>
240	245	250<br>
TTC ACC TCC ACC GCC GTT TCC GGC AAG CAA ATG ATC GTG CAA ATC ACG 384<br>
Phe Thr Ser Thr Ala Val Ser Gly Lys Gin Met lie Val Gin lie Thr<br>
255	260	265<br>
i  AAC ACG GGC GGC GAC CTC GGC AAC AAC CAC TTC GAC ATC GCC ATG CCG 432<br>
Asn Thr Gly Gly Asp Leu Gly Asn Asn His Phe Asp He Ala Met Pro<br>
270	275	280	285<br>
GGC GGC GGC GTC GGC ATC TTC AAC GGG TGC TCC AAG CAA TGG AAC GGC 480<br>
Gly Gly Gly Val Gly He Phe Asn Gly Cys Ser Lys Gin Trp Asn Gly<br>
290	295	300<br>
I  ATC AAT CTG GGC AAC CAG TAT GGC GGC TTC ACT GAC CGC TCG CAA TGT 528<br>
He Asn Leu Gly Asn Gin Tyr Gly Gly Phe Thr Asp Arg Ser Gin Cys<br>
305	310	315<br>
GCG ACG CTC CCG TCC AAG TGG CAG GCC AGC	TGC AAC TGG CGC TTC GAC<br>
576<br>
Ala Thr Leu Pro Ser Lys Trp Gin Ala Ser	Cys Asn Trp Arg Phe Asp<br>
320                325	330<br>
TGG TTC GAG AAT	GCC GAC AAC CCC ACC GTC GAT TGG GAG COT GTC ACT<br>
624<br>
Trp Phe Glu Asn	Ala Asp Asn Pro Thr Val Asp Trp Glu Pro Val Thr<br>
335	340	345<br><br>
TGC CCA CAG GAA TTG GTC GCC CGG	ACT GGC TGT TCC CGT ACC CCC TCC<br>
672<br>
Cys Pro Gin Glu Leu Val Ala Arg	Thr Gly Cys Ser Arg Thr Pro Ser<br>
350                355	360                365<br>
AGC AGC ACC AGC TCT CCG GTC AAC CAG CCT ACC AGC ACC AGC ACC ACG 720<br>
Ser Ser Thr Ser Ser Pro Val Asn Gin Pro Thr Ser Thr Ser Thr Thr<br>
370	375	380<br>
TCC ACC TCC ACC ACC TCG AGC	CCG CCA GTC CAG	CCT ACG ACT CCC AGC<br>
768<br>
Ser Thr Ser Thr Thr Ser Ser	Pro Pro Val Gin	Pro Thr Thr Pro Ser<br>
385                 390	395<br>
GGC TGC ACT GCT GAG AGG TGG GCT CAG TGC	GGC GGC AAT GGC TGG AGC<br>
816<br>
Gly Cys Thr Ala Glu Arg Trp Ala Gin Cys	Gly Gly Asn Gly Trp Ser<br>
400                405	410<br>
GGC TGC ACC ACC TGC GTC GCT GGC AGC ACT TGC ACG AAG ATT AAT GAC 864<br>
Gly Cys Thr Thr Cys Val Ala Gly Ser Thr Cys Thr Lys lie Asn Asp<br>
415	420	425<br>
TGG TAC CAT CAG	TGC CTG TAG<br>
885<br>
Trp Tyr His Gin	Cys Leu  *<br>
430	435<br>
(2) INFORMATION FOR SEQ ID NO: 18-B:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 295 amino acids<br>
(B)	TYPE: amino acid (D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 18-B:<br>
Met Phe Ser Pro Leu Trp Ala Leu Ser Ala Leu Leu Leu Phe Pro Ala<br><br>
15	10	15<br>
Thr Glu Ala Thr Ser Gly Val Thr Thr Arg Tyr Trp Asp Cys Cys Lys<br>
20	25	30<br>
Pro Ser Cys Ala Trp Thr Gly Lys Ala Ser Val Ser Lys Pro Val Gly<br>
35	40	45<br>
Thr Cys Asp lie Asn Asp Asn Ala Gin Thr Pro Ser Asp Leu Leu Lys<br>
50	55	60<br>
Ser Ser Cys Asp Gly Gly Ser Ala Tyr Tyr Cys Ser Asn Gin Gly Pro<br>
65	70	75	80<br>
Trp Ala Val Asn Asp Ser Leu Ser Tyr Gly Phe Ala Ala Ala Lys Leu<br>
85	90	95<br>
Ser Gly Lys Gin Glu Thr Asp. .Trp Cys Cys Gly Cys Tyr Lys Leu Thr<br>
100	105	110<br>
Phe Thr Ser Thr Ala Val Ser Gly Lys Gin Met lie Val Gin lie Thr<br>
115	120	125<br>
Asn Thr Gly Gly Asp Leu Gly Asn Asn His Phe Asp lie Ala Met Pro<br>
130	135	140<br>
Gly Gly Gly Val Gly lie Phe Asn Gly Cys Ser Lys Gin Trp Asn Gly<br>
145	150	155	160<br>
lie Asn Leu Gly Asn Gin Tyr Gly Gly Phe Thr Asp Arg Ser Gin Cys<br>
165	170	175<br>
Ala Thr Leu Pro Ser Lys Trp Gin Ala Ser Cys Asn Trp Arg Phe Asp<br>
180	185	190<br>
Trp Phe Glu Asn Ala Asp Asn Pro Thr Val Asp Trp Glu Pro Val Thr<br>
195	200	205<br>
Cys Pro Gin Glu Leu Val Ala Arg Thr Gly Cys Ser Arg Thr Pro Ser<br>
210	215	220<br>
Ser Ser Thr Ser Ser Pro Val Asn Gin Pro Thr Ser Thr Ser Thr Thr<br>
225	230	235	240<br><br>
Ser Thr  Ser Thr Thr Ser  Ser  Pro  Pro Val  Gin  Pro Thr  Thr  Pro  Ser<br>
245	250	255<br>
Gly Cys  Thr Ala Glu Arg Trp Ala Gin  Cys Gly Gly Asn Gly Trp Ser<br>
260	265	270<br>
Gly  Cys  Thr Thr  Cys Val  Ala  Gly  Ser  Thr  Cys  Thr  Lys   He  Asn  Asp<br>
275	280	285<br>
Trp Tyr His Gin Cys Leu     *<br>
290	295<br>
(2)   INFORMATION  FOR  SEQ   ID  NO:    19:<br>
(i)    SEQUENCE  CHARACTERISTICS:<br>
(A)	LENGTH:   425  base pairs<br>
(B)	TYPE:   nucleic acid<br>
(C)	STRANDEDNESS:   single<br>
(D)	TOPOLOGY:   linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(vi) ORIGINAL SOURCE:<br>
(A) ORGANISM: Escherichia coli, DSM 10576<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: CDS<br>
(B)	LOCATION:l..4 25<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 19:<br>
CAAGATACAA TATGCGTTCC TCCACTATTT TGCAAACCGG CCTGGTGGCC GTTCTCCCCT 60<br>
TCGCCGTCCA GGCCGCCTCA GGATCCGGCA AGTCCACCAG ATATTGGGAC TGCTGCAAAC 120<br>
CATCTTGTGC CTGGTCCGGC AAGGCTTCTG TCAACCGCCC TGTTCTCGCC TGCAACGCAA 180<br><br>
ACAACAACCC GCTGAACGAC GCCAACGTCA AGTCAGGATG TGATGGCGGT TCTGCATACA 240<br>
CCTGTGCCAA CAACTCTCCC TGGGCAGTGA ATGACAATCT GGCCTACGGC TTCGCGGCCA 300<br>
CAAAACTCAG CGGGGGGACC GAGTCATCTT GGTGCTGCGC CTGTTATGCC CTCACATTCA 360<br>
CATCGGGTCC TGTTTCTGGC AAAACCTTGG TTGTCCAGTC TACCAGTACC GGTGGTGATC 420<br>
TTGGC 425<br>
(2)    INFORMATION  FOR  SEQ   ID  NO:   20:<br>
(i)    SEQUENCE  CHARACTERISTICS:<br>
(A)	LENGTH:   141   amino  acids<br>
(B)	TYPE:   amino  acid (D)   TOPOLOGY:   linear<br>
(ii)   MOLECULE TYPE:   protein<br>
(xi)    SEQUENCE  DESCRIPTION:   SEQ   ID  NO:   20:<br>
Met Arg Ser Ser Thr He Leu Gin Thr Gly Leu Val Ala Val  Leu  Pro<br>
15	10	15<br>
Phe Ala Val Gin Ala Ala Ser Gly Ser Gly Lys Ser Thr Arg Tyr Trp<br>
20	25	30<br>
Asp Cys Cys Lys Pro Ser Cys Ala Trp Ser Gly Lys Ala Ser Val Asn<br>
35	40	45<br>
Arg Pro Val Leu Ala Cys Asn Ala Asn Asn Asn Pro Leu Asn Asp Ala<br><br>
Asn Val Lys Ser Gly Cys Asp Gly Gly  Ser Ala Tyr Thr Cys Ala Asn<br>
65	70	75	80<br>
Asn Ser Pro Trp Ala Val  Asn Asp Asn Leu Ala Tyr Gly  Phe Ala Ala<br>
85	90	95<br>
Thr Lys Leu Ser Gly Gly Thr Glu Ser Ser Trp Cys Cys Ala Cys Tyr<br>
100	105	110<br>
Ala Leu  Thr  Phe Thr  Ser  Gly  Pro Val  Ser  Gly Lys Thr  Leu  Val  Val<br>
115	120	125<br>
Gin Ser Thr Ser Thr Gly Gly Asp Leu Gly<br>
130	135<br>
2)   INFORMATION  FOR  SEQ   ID NO:   21:<br>
(i)    SEQUENCE  CHARACTERISTICS:<br>
(A)	LENGTH:   108   base  pairs<br>
(B)	TYPE:   nucleic  acid<br>
(C)	STRANDEDNESS:   single<br>
(D)	TOPOLOGY:   linear<br>
(ii)   MOLECULE  TYPE:   cDNA<br>
(vi)   ORIGINAL  SOURCE:<br>
(A)	ORGANISM:   Saccobolus  dilutellus<br>
(B)	STRAIN:   CBS   275.96<br><br>
(ix)   FEATURE:<br>
(A)	NAME/KEY: CDS<br>
(B)	LOCATION:!..108<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 21:<br>
TCG GOT TGC GAT AAC GGT GGT GGC ACT GCA TAC ATG TGT GCC AGC CAG<br>
Ser Ala Cys Asp Asn Gly Gly Gly Thr Ala Tyr Met Cys Ala Ser Gin<br>
is	10	15<br>
GAG CCG TGG GCA GTG AGC TCC AAC GTC GCG TAC GGC TTT GCT GCA GTT<br>
Glu Pro Trp Ala Val Ser Ser Asn Val Ala Tyr Gly Phe Ala Ala Val<br>
20	25	30<br>
AGA ATC AGC GGA<br>
Arg   lie  Ser Gly 35<br>
(2)    INFORMATION  FOR  SEQ   ID  NO:   22:<br>
(i)    SEQUENCE  CHARACTERISTICS:<br>
(A)	LENGTH:   36  amino  acids<br>
(B)	TYPE:   amino  acid (D)   TOPOLOGY:   linear<br>
(ii)   MOLECULE TYPE:   protein<br>
(xi)    SEQUENCE  DESCRIPTION:   SEQ   ID  NO:   22:<br>
Ser Ala Cys Asp Asn Gly Gly Gly Thr Ala Tyr Met  Cys Ala Ser Gin<br>
15	10	15<br>
Glu  Pro Trp Ala Val  Ser  Ser Asn Val Ala Tyr Gly Phe Ala Ala Val<br>
20	25	30<br>
Arg  He  Ser Gly 35<br><br>
(2) INFORMATION FOR SEQ ID NO: 23:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 99 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii)   MOLECULE  TYPE:   cDNA<br>
(vi)   ORIGINAL  SOURCE:<br>
(A)	ORGANISM:   Thermomyces  verrucosus<br>
(B)	STRAIN:   CBS   285.96<br>
(ix)   FEATURE:<br>
(A)	NAME/KEY:   CDS<br>
(B)	LOCATION:!..99<br>
(xi)    SEQUENCE  DESCRIPTION:   SEQ   ID  NO:   23:<br>
GCC  TGC   AAC  GCA  AAC  TTC  CAG   CGC  ATC  AGT  GAC  CCC  AAC   GCC   AAG   TCG Ala Cys  Asn Ala Asn  Phe Gin  Arg   lie  Ser Asp  Pro Asn  Ala Lys   Ser<br>
GGC  TGC   GAT   GGT   GGC  TCG   GCC   TTC   TCT   TGC  GCC  AAA   CAA   ACC   CCT   TGG Gly Cys  Asp Gly Gly  Ser Ala  Phe  Ser  Cys Ala  Lys  Gin  Thr  Pro  Trp<br>
GCC<br>
Ala<br>
(2) INFORMATION FOR SEQ ID NO: 24:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 33 amino acids<br>
(B)	TYPE: amino acid (D) TOPOLOGY: linear<br><br>
(ii)   MOLECULE  TYPE:   protein<br>
(xi)   SEQUENCE  DESCRIPTION:   SEQ  ID  NO:   24:<br>
Ala Cys Asn Ala Asn Phe Gin Arg  lie Ser Asp Pro Asn Ala Lys  Ser<br>
1	5	10	15<br>
Gly Cys Asp Gly Gly Ser Ala Phe Ser Cys Ala Lys Gin Thr Pro Trp<br>
20	25	30<br>
Ala<br>
(2) INFORMATION FOR SEQ ID NO: 25:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 225 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(vi) ORIGINAL SOURCE:<br>
(A)	ORGANISM: Xylaria hypoxylon<br>
(B)	STRAIN: CBS 284.96<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: CDS<br>
(B)	LOCATION:1..225<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 25:<br>
GAC CAG CCG CTC GGC GGA	CAA	CGG ACG CGA CCA AGG AGC GCG TGC GAC<br>
48<br>
Asp Gin Pro Leu Gly Gly	Gin	Arg Thr Arg Pro Arg Ser Ala Cys Asp<br>
35	40                 45<br>
AAT GGC GGC TCT GCA TAG ATG TGC AGC AAC CAG AGC CCG TGG GCC GTC<br><br>
96<br>
Asn Gly Gly  Ser Ala Tyr Met  Cys  Ser Asn Gin  Ser  Pro  Trp Ala Val<br>
50	55	60	65<br>
GAC  GAT   TCT   CTC  AGT  TAG  GGA  TGG  GCT  GCC	GTT  AGG   ATC   TAT  GGA  CAT<br>
144<br>
Asp Asp Ser  Leu Ser Tyr Gly Trp Ala Ala	Val  Arg  lie Tyr  Gly His<br>
70                                             75	80<br>
ACC  GAA  ACT  ACT  TGG   TGC   TGC  GCT  TGC	TAC  GAG   TTG  ACT  TTT  ACC  AGC<br>
192<br>
Thr Glu  Thr Thr Trp Cys  Cys  Ala  Cys	Tyr Glu  Leu  Thr  Phe  Thr  Ser<br>
85                                             90	95<br>
GGT  CCG   GTT   AGC  GGC  AAG   AAG	ATG  ATT   GTT   CAG<br>
225<br>
Gly Pro Val   Ser Gly Lys  Lys	Met   lie Val  Gin<br>
100	105<br>
(2)    INFORMATION  FOR   SEQ   ID  NO:   26:<br>
(i)    SEQUENCE   CHARACTERISTICS:<br>
(A)	LENGTH:   75  amino  acids<br>
(B)	TYPE:   amino  acid (D)   TOPOLOGY:   linear<br>
(ii)   MOLECULE TYPE:   protein<br>
(xi)   SEQUENCE  DESCRIPTION:   SEQ   ID  NO:   26:<br>
Asp Gin  Pro  Leu  Gly Gly Gin  Arg  Thr  Arg  Pro Arg   Ser  Ala  Cys  Asp<br>
15	10	15<br>
Asn Gly  Gly  Ser Ala  Tyr  Met   Cys   Ser Asn  Gin  Ser  Pro  Trp Ala Val<br>
20	25	30<br>
Asp Asp  Ser  Leu  Ser  Tyr  Gly  Trp Ala Ala  Val  Arg   lie  Tyr   Gly  His<br>
35	40	45<br>
Thr Glu  Thr  Thr  Trp  Cys  Cys  Ala  Cys  Tyr  Glu  Leu  Thr  Phe  Thr  Ser<br>
50	55	60<br><br>
Gly Pro Val Ser Gly Lys Lys Met lie Val Gin<br>
65	70	75<br>
(2) INFORMATION FOR SEQ ID NO: 27:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 177 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(vi) ORIGINAL SOURCE:<br>
(A)	ORGANISM: Fusarium oxysporum ssp lycopersici<br>
(B)	STRAIN: CBS 645.78<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: CDS<br>
(B)	LOCATION:!..177<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 27:<br>
AGA AAC GAC AAC CCC ATC TCC AAC ACC AAC GCT GTC AAC	GGT TGT GAG<br>
48<br>
Arg Asn Asp Asn Pro lie Ser Asn Thr Asn Ala Val Asn	Gly Cys Glu<br>
30                 35                 40	45<br>
GGT GGT GGT TCT GCT TAT GCT	TGC ACC AAC	TAC TCT CCC TGG GCT GTC<br>
96<br>
Gly Gly Gly Ser Ala Tyr Ala	Cys Thr Asn	Tyr Ser Pro Trp Ala Val<br>
50	55                 60<br>
AAC GAT GAG CTT GCC TAC GGT TTC GCT GCT ACC AAG ATC TCC GGT GGC<br>
144<br>
Asn Asp Glu Leu Ala Tyr Gly Phe Ala Ala Thr Lys lie Ser Gly Gly<br><br>
65	70	75<br>
TCC  GAG  GCC  AGC  TGG   TGC   TGT  GCC   TGC  TAT   CTA 177<br>
Ser Glu Ala Ser Trp Cys  Cys  Ala Cys  Tyr Leu<br>
80	85<br>
(2)   INFORMATION  FOR  SEQ  ID  NO:   28:<br>
(i)    SEQUENCE   CHARACTERISTICS:<br>
(A)	LENGTH:   59  amino  acids<br>
(B)	TYPE:   amino acid (D)   TOPOLOGY:   linear<br>
(ii)   MOLECULE TYPE:   protein<br>
(xi)   SEQUENCE  DESCRIPTION:   SEQ   ID  NO:   28:<br>
Arg Asn Asp Asn Pro  lie  Ser Asn Thr Asn  Ala Val  Asn Gly Cys Glu<br>
15	10<br>
15<br>
Gly Gly Gly Ser Ala Tyr Ala Cys Thr Asn Tyr Ser Pro Trp Ala Val<br>
20	25	30<br>
Asn Asp Glu Leu Ala Tyr Gly Phe Ala Ala Thr Lys lie Ser Gly Gly<br>
35	40	45<br>
Ser Glu Ala Ser Trp Cys Cys Ala Cys Tyr Leu<br>
50	55<br>
(2) INFORMATION FOR SEQ ID NO: 29:<br><br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 63 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: LINEAR<br>
(ii) MOLECULE TYPE: cDNA<br>
(vi) ORIGINAL SOURCE:<br>
(A)	ORGANISM: Nectria pinea<br>
(B)	STRAIN: CBS 279.96<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: CDS<br>
(B)	LOCATION:1..63<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 29:<br>
AGC GGC  TGT GAC GGT GGT TCT GCC TAC GCC TGT GCA AAC AAC TCC CCT        4 8<br>
Ser Gly Cys Asp Gly Gly Ser Ala Tyr Ala Cys Ala Asn Asn<br>
Ser Pro<br>
60	65	70<br>
75<br>
TGG GCT GTC AAC GAT<br>
63 Trp Ala Val Asn Asp<br>
80<br>
(2) INFORMATION FOR SEQ ID NO: 30:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 21 amino acids<br>
(B)	TYPE: amino acid<br><br>
(D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 30:<br>
Ser Gly Cys Asp Gly Gly Ser Ala Tyr Ala Cys Ala Asn Asn Ser Pro<br>
15	10<br>
15<br>
Trp Ala Val Asn Asp 20<br>
(2) INFORMATION FOR SEQ ID NO: 31:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 177 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(vi) ORIGINAL SOURCE:<br>
(A)	ORGANISM: Humicola grisea Traeen<br>
(B)	STRAIN: ATCC 22726<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: CDS<br>
(B)	LOCATION:1..177<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 31:<br>
AAC CAG CCT GTC TTC ACT TGC GAC GCC AAA TTC CAG CGC  ATC<br>
ACC GAC        48<br>
Asn Gin Pro Val Phe Thr Cys Asp Ala Lys Phe Gin Arg lie<br><br>
Thr Asp<br>
25	30	35<br>
CCC  AAT ACC AAG TCG GGC TGC GAT GGC GGC TCG GCC TTT TCG TGT GCT        96<br>
Pro Asn Thr Lys Ser Gly Cys Asp Gly Gly Ser Ala Phe Ser Cys Ala<br>
40	45	50<br>
GAC CAA ACC CCC TGG GCT CTG AAC GAC GAT TTC GCC TAT GGC TTC GCT       144<br>
Asp Gin Thr Pro Trp Ala Leu Asn Asp Asp Phe Ala Tyr Gly Phe Ala<br>
55	60	65<br>
GCC ACG GCT ATT TCG GGT GGA TCG GAA GCC TCG<br>
177<br>
Ala Thr Ala lie Ser Gly Gly Ser Glu Ala Ser<br>
70	75	80<br>
(2) INFORMATION FOR SEQ ID NO: 32:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 59 amino acids<br>
(B)	TYPE: amino acid (D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 32:<br>
Asn Gin Pro Val Phe Thr Cys Asp Ala Lys Phe Gin Arg lie Thr Asp<br>
15	10<br><br>
15<br>
Pro Asn Thr Lys Ser Gly Cys Asp Gly Gly Ser Ala Phe Ser Cys Ala<br>
20	25	30<br>
Asp Gin Thr Pro Trp Ala Leu Asn Asp Asp Phe Ala Tyr Gly Phe Ala<br>
35	40	45<br>
Ala Thr Ala lie Ser Gly Gly Ser Glu Ala Ser<br>
50	55<br>
(2) INFORMATION FOR SEQ ID NO: 33:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 153 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(vi) ORIGINAL SOURCE:<br>
(A)	ORGANISM: Humicola nigrescens Oinvik<br>
(B)	STRAIN: CBS 819.73<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: CDS<br>
(B)	LOCATION:!..15 3<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 33: GTC TAC GCC TGC AAC GCA AAC TTC CAG CGC ATC ACC GAC GCC<br><br>
AAC GCC        48<br>
Val Tyr Ala Cys Asn Ala Asn Phe Gin Arg lie Thr Asp Ala<br>
Asn Ala<br>
60	65	70<br>
75<br>
AAG TCC GGC TGC GAT GGC GGC TCC GCC TTC TCG TGC GCC AAC CAG ACC        96<br>
Lys Ser Gly Cys Asp Gly Gly Ser Ala Phe Ser Cys Ala Asn Gin Thr<br>
80	85<br>
90<br>
CCG TGG GCC GTG AGC GAC GAC TTT GCC TAC GGT TTC GCG GCT ACG GCG       144<br>
Pro Trp Ala Val Ser Asp Asp Phe Ala Tyr Gly Phe Ala Ala Thr Ala<br>
95	100	105<br>
CTC GCC GGC<br>
153 Leu Ala Gly 110<br>
(2) INFORMATION FOR SEQ ID NO: 34:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 51 amino acids<br>
(B)	TYPE: amino acid (D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein .<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 34:<br>
Val Tyr Ala Cys Asn Ala Asn Phe Gin Arg lie Thr Asp Ala<br><br>
Asn Ala<br>
15	10<br>
15<br>
Lys Ser Gly Cys Asp Gly Gly Ser Ala Phe Ser Cys Ala Asn Gin Thr<br>
20	25	30<br>
Pro Trp Ala Val Ser Asp Asp Phe Ala Tyr Gly Phe Ala Ala Thr Ala<br>
35	40	45<br>
Leu Ala Gly 50<br>
(2) INFORMATION FOR SEQ ID NO: 35:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 181 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(vi) ORIGINAL SOURCE:<br>
(A)	ORGANISM: Cladorrhinum foecundissimum<br>
(B)	STRAIN: ATCC 62373<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: CDS<br>
(B)	LOCATION:1..181<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 35;<br><br>
GTC AAC CGC CCT GTC CTC GCC TGC GAC GCA AAC AAC AAC CCT CTG ACC        48<br>
Val Asn Arg Pro Val Leu Ala Cys Asp Ala Asn Asn Asn Pro Leu Thr<br>
15	10<br>
15<br>
GAC GCC GGC  GTC AAG TCC GGA TGT GAT GGC GGT TCT GCA TAC ACC TGT        96<br>
Asp Ala Gly Val Lys Ser Gly Cys Asp Gly Gly Ser Ala Tyr Thr Cys<br>
20	25	30<br>
GCC  AAC AAC TCC CCA TGG GCA GTG AAC GAC CAG CTC GCC  TAC GGC  TTT       144<br>
Ala Asn Asn Ser Pro Trp Ala Val Asn Asp Gin Leu Ala Tyr Gly Phe<br>
35	40	45<br>
GCC GCC ACC AAA CTG AGC GGC GGA ACT GAG TCG TCA<br>
180<br>
Ala Ala Thr Lys Leu Ser Gly Gly Thr Glu Ser Ser<br>
50	55	60<br>
(2) INFORMATION FOR SEQ ID NO: 36:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 60 amino acids<br>
(B)	TYPE: amino acid (D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 36;<br><br>
Val Asn Arg Pro Val Leu Ala Cys Asp Ala Asn Asn Asn Pro Leu Thr<br>
15	10<br>
15<br>
Asp Ala Gly Val Lys Ser Gly Cys Asp Gly Gly Ser Ala Tyr Thr Cys<br>
20	25	30<br>
Ala Asn Asn Ser Pro Trp Ala Val Asn Asp Gin Leu Ala Tyr Gly Phe<br>
35	40	45<br>
Ala Ala Thr Lys Leu Ser Gly Gly Thr Glu Ser Ser<br>
50	55	60<br>
(2) INFORMATION FOR SEQ ID NO: 37:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 64 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(vi) ORIGINAL SOURCE:<br>
(A)	ORGANISM: Syspastospora boninensis<br>
(B)	STRAIN: NKBC 1515<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: CDS<br>
(B)	LOCATION:!..64<br><br>
(xi) SEQUENCE DESCRIPTION; SEQ ID NO: 37:<br>
GGC TGC GAC GGC  GGC  AGC GCC TTC ACC TGC TCC AAC AAC TCT CCA TGG        48<br>
Gly Cys Asp Gly Gly Ser Ala Phe Thr Cys Ser Asn Asn Ser Pro Trp<br>
GCT GTG AAC GAA GAT<br>
63 Ala Val Asn Glu Asp<br>
(2) INFORMATION FOR SEQ ID NO: 38:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 21 amino acids<br>
(B)	TYPE: amino acid (D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 38:<br>
Gly Cys Asp Gly Gly Ser Ala Phe Thr Cys Ser Asn Asn Ser Pro Trp<br>
Ala Val Asn Glu Asp<br>
(2) INFORMATION FOR SEQ ID NO: 39:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A) LENGTH: 153 base pairs<br><br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(vi) ORIGINAL SOURCE:<br>
(A)	ORGANISM: Nigrospora sp<br>
(B)	STRAIN: CBS 272.96<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: CDS<br>
(B)	LOCATION:1.-153<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 39:<br>
ACA AGA AAC GAC GGG CCC CTG TCC AGC CCC GAT GCC GCC TCC GGC TGT        48<br>
Thr Arg Asn Asp Gly Pro Leu Ser Ser Pro Asp Ala Ala Ser Gly Cys<br>
25	30	35<br>
GAT GGC  GGC  GAA GCC TTT GCC TGT TCT AAT ACC TCG CCT TGG GCC  GTC        95<br>
Asp Gly Gly Glu Ala Phe Ala Cys Ser Asn Thr Ser Pro Trp Ala Val<br>
40	45	50<br>
AGC GAC GAG CTC GCG TAG GGA TAG GTC GCC ACG TCC ATC TCC GGC GGC       144<br>
Ser Asp Gin Leu Ala Tyr Gly Tyr Val Ala Thr Ser lie Ser Gly Gly<br>
55	60	65<br><br>
ACC GAG TCA<br>
153 Thr Glu Ser 70<br>
(2) INFORMATION FOR SEQ ID NO: 40:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 51 amino acids<br>
(B)	TYPE: amino acid (D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 40:<br>
Thr Arg Asn Asp Gly Pro Leu Ser Ser Pro Asp Ala Ala Ser Gly Cys<br>
15	10<br>
15<br>
Asp Gly Gly Glu Ala Phe Ala Cys Ser Asn Thr Ser Pro Trp Ala Val<br>
20	25	30<br>
Ser Asp Gin Leu Ala Tyr Gly Tyr Val Ala Thr Ser lie Ser Gly Gly<br>
35	40	45<br>
Thr Glu Ser 50<br>
(2) INFORMATION FOR SEQ ID NO: 41:<br>
(i) SEQUENCE CHARACTERISTICS:<br><br>
(A)	LENGTH: 159 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(Vi) ORIGINAL SOURCE:<br>
(A) ORGANISM: Chaetostylum fresenii<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: CDS<br>
(B)	LOCATION:1..159<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 41:<br>
GTC CGA ACG TGT AGT GCC AAC GAC TCG CCC TTG TCC GAC CCA AAT GCC        48<br>
Val Arg Thr Cys Ser Ala Asn Asp Ser Pro Leu Ser Asp Pro Asn Ala<br>
55	60	65<br>
CCA AGT GGG TGT GAC GGT GGT AGC GCC TTC ACT TGT TCC AAC AAC TCC        9 6<br>
Pro Ser Gly Cys Asp Gly Gly Ser Ala Phe Thr Cys Ser Asn Asn Ser<br>
70	75	80<br>
CCG TGG GCA GTC GAT GAC CAG ACA GCT TAT GGC TTT GCG GCA ACA GCC       14 4<br>
Pro Trp Ala Val Asp Asp Gin Thr Ala Tyr Gly Phe Ala Ala Thr Ala<br>
85	90	95<br><br>
ATC AGT GGC CAG TCC<br>
159 lie Ser Gly Gin Ser 100<br>
(2) INFORMATION FOR SEQ ID NO: 42:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 53 amino acids<br>
(B)	TYPE: amino acid (D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 42:<br>
Val Arg Thr Cys Ser Ala Asn Asp Ser Pro Leu Ser Asp Pro Asn Ala<br>
15	10<br>
15<br>
Pro Ser Gly Cys Asp Gly Gly Ser Ala Phe Thr Cys Ser Asn Asn Ser<br>
20	25	30<br>
Pro Trp Ala Val Asp Asp Gin Thr Ala Tyr Gly Phe Ala Ala Thr Ala<br>
35	40	45<br>
lie Ser Gly Gin Ser 50<br>
(2) INFORMATION FOR SEQ ID NO: 43;<br>
(i) SEQUENCE CHARACTERISTICS;<br><br>
(A)	LENGTH: 153 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(vi) ORIGINAL SOURCE:<br>
(A)	ORGANISM: Exidia glandulosa<br>
(B)	STRAIN: CBS 277.96<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: CDS<br>
(B)	LOCATION:1..153<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 43:<br>
TGT GAG AAG AAC GAC AAC CCC  TTA GCT GAC TTC AGC ACG AAA TCC GGG        48<br>
Cys Glu Lys Asn Asp Asn Pro Leu Ala Asp Phe Ser Thr Lys Ser Gly<br>
55	60	65<br>
TGT GAA AGC GGA GGT TCG GCT TAT ACG TGT AAC AAC CAA TCA CCA TGG        96<br>
Cys Glu Ser Gly Gly Ser Ala Tyr Thr Cys Asn Asn Gin Ser<br>
Pro Trp<br>
70	75	80<br>
85<br>
GCC GTC AAT GAC TTG GTG TCG TAT GGC TTC GCC GCC ACA GCG ATC AAT       144<br>
Ala Val Asn Asp Leu Val Ser Tyr Gly Phe Ala Ala Thr Ala lie Asn<br>
90	95<br><br>
100<br>
GGT GGC AAT<br>
153 Gly Gly Asn<br>
(2) INFORMATION FOR SEQ ID NO: 44:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 51 amino acids<br>
(B)	TYPE: amino acid (D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 44:<br>
Cys Glu Lys Asn Asp Asn Pro Leu Ala Asp Phe Ser Thr Lys Ser Gly<br>
15	10<br>
15<br>
Cys Glu Ser Gly Gly Ser Ala Tyr Thr Cys Asn Asn Gin Ser Pro Trp<br>
20	25	30<br>
Ala Val Asn Asp Leu Val Ser Tyr Gly Phe Ala Ala Thr Ala lie Asn<br>
35	40	45<br>
Gly Gly Asn 50<br>
(2) INFORMATION FOR SEQ ID NO: 45:<br><br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 171 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(vi) ORIGINAL SOURCE:<br>
(A) ORGANISM: Coniothecium sp<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: CDS<br>
(B)	LOCATION:l..171<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 45:<br>
AGG GGC CCC GTC GGA ACC TGC AAG AGG AAC GAG AAC CCC CTC TCC GAC        48<br>
Ser Arg Pro Val Gly Thr Cys Lys Arg Asn Asp Asn Pro Leu Ser Asp<br>
55	60	65<br>
CCC GAT GCC AAG TCC GGC TGC GAC GGC GGC GGC GCC  TTC ATG TGC TCC        96<br>
Pro Asp Ala Lys Ser Gly Cys Asp Gly Gly Gly Ala Phe Met Cys Ser<br>
70	75	80<br>
ACC CAG CAG CCG TGG GCC GTC AAC GAC AAT CTG GCA TAT GGC TTC GCC       144<br>
Thr Gin Gin Pro Trp Ala Val Asn Asp Asn Leu Ala Tyr Gly Phe Ala<br>
85	90	95<br><br>
GCC ACG GCC ATC AGC GGC GGC AAC GAG<br>
171<br>
5 Ala Thr Ala lie Ser Gly Gly Asn Glu<br>
100	105<br>
(2) INFORMATION FOR SEQ ID NO; 46: )<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 57 amino acids<br>
(B)	TYPE: amino acid (D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 46:<br>
Ser Arg Pro Val Gly Thr Cys Lys Arg Asn Asp Asn Pro Leu Ser Asp<br>
15	10<br>
15<br>
Pro Asp Ala Lys Ser Gly Cys Asp Gly Gly Gly Ala Phe Met Cys Ser<br>
20	25	30<br>
Thr Gin Gin Pro Trp Ala Val Asn Asp Asn Leu Ala Tyr Gly Phe Ala<br>
35	40	45<br>
Ala Thr Ala lie Ser Gly Gly Asn Glu<br>
50	55<br>
(2) INFORMATION FOR SEQ ID NO: 47:<br><br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 159 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(vi) ORIGINAL SOURCE:<br>
(B) STRAIN: CBS 271.96<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: CDS<br>
(B)	LOCATION:!..159<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 47:<br>
ACT TGC AAC AAG AAC GAC GGG  CCC  CTG TCG AGC CCC GAT GCC I  GCC TCC        4 8<br>
Thr Cys Asn Lys Asn Asp Gly Pro Leu Ser Ser Pro Asp Ala Ala Ser<br>
60	65	70<br>
GGC TGT GAT GGC GGC GAA GCC TTT GCC TGT TCT AAT AGC TCG CCT TGG        96<br>
Gly Cys Asp Gly Gly Glu Ala Phe Ala Cys Ser Asn Thr Ser Pro Trp<br>
75	80	85<br>
GCC GTC AGC GAC GAG CTC GCG TAG GGA TAG CTC GCC ACG TCC ATC TCC       144<br>
Ala Val Ser Asp Gin Leu Ala Tyr Gly Tyr Leu Ala Thr Ser<br>
lie Ser<br>
90	95	100<br><br>
105<br>
GGC GGC ACC GAG TCG<br>
159 Gly Gly Thr Glu Ser<br>
110<br>
(2) INFORMATION FOR SEQ ID NO: 48:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 53 amino acids<br>
(B)	TYPE: amino acid (D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 48:<br>
Thr Cys Asn Lys Asn Asp Gly Pro Leu Ser Ser Pro Asp Ala Ala Ser<br>
15	10<br>
15<br>
Gly Cys Asp Gly Gly Glu Ala Phe Ala Cys Ser Asn Thr Ser Pro Trp<br>
20	25	30<br>
Ala Val Ser Asp Gin Leu Ala Tyr Gly Tyr Leu Ala Thr Ser He Ser<br>
35	40	45<br>
Gly Gly Thr Glu Ser 50<br>
(2) INFORMATION FOR SEQ ID NO: 49<br><br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 84 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(vi) ORIGINAL SOURCE:<br>
(B) STRAIN: CBS 270.96<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: CDS<br>
(B)	LOCATION:1..84<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 49:<br>
CCA GTT TTC TCC TGT GAC AAG TAC GAC AAC CCT CTA CCT GAC GCC AAT        4 8<br>
Pro Val Phe Ser Cys Asp Lys Tyr Asp Asn Pro Leu Pro Asp Ala Asn<br>
55	60	65<br>
GCT GTG TCC GGG TGT GAC CCC GGA	GGT ACT GCC TTC<br>
84<br>
Ala Val Ser Gly Cys Asp Pro Gly	Gly Thr Ala Phe<br>
70                  75	80<br>
(2) INFORMATION FOR SEQ ID NO: 50:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 28 amino acids<br>
(B)	TYPE: amino acid (D) TOPOLOGY: linear<br><br>
(ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 50:<br>
Pro Val Phe Ser Cys Asp Lys Tyr Asp Asn Pro Leu Pro Asp 5 Ala Asn<br>
15	10<br>
15<br>
Ala Val Ser Gly Cys Asp Pro Gly Gly Thr Ala Phe<br>
20	25<br>
(2) INFORMATION FOR SEQ ID NO: 51:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 147 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(vi) ORIGINAL SOURCE:<br>
(B) STRAIN: Diplodia gossypina, CBS 274.96<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 51:<br>
ACCTGCGACG CCTGCGACAG CCCCCTCAGC GACTACGACG CCAAGTCCGG CTGCGACGGC      60<br>
GGTAGCGCAT ACACCTGCAC CTACTCTACC CCCTGGGCCG TCGACGACAA CCTCTCCTAC     120<br>
GGTTTCGCCG CCGCCAAGCT GAGCGGA 147<br><br>
(2) INFORMATION FOR SEQ ID NO: 52:<br>
(i) SEQUENCE CPiARACTERISTICS:<br>
(A)	LENGTH: 49 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: peptide<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 52:<br>
Thr Cys Asp Ala Cys Asp Ser Pro Leu Ser Asp Tyr Asp Ala Lys Ser<br>
15	10<br>
15<br>
Gly Cys Asp Gly Gly Ser Ala Tyr Thr Cys Thr Tyr Ser Thr Pro Trp<br>
20	25<br>
30<br>
Ala Val Asp Asp Asn Leu Ser Tyr Gly Phe Ala Ala Ala Lys Leu Ser<br>
35	40	45<br>
Gly<br>
(2) INFORMATION FOR SEQ ID NO: 53:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 13 5 base pairs<br>
(B)	TYPE: nucleic acid<br><br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(vi) ORIGINAL SOURCE:<br>
(B) STRAIN: Ulospora bilgramii, NKBC 1444<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 53:<br>
CCACTAGCAG ATTTCACCGG TGGAACCGGC TGTAATGGCG GTTCGACATT CTCATGCTCA      60<br>
AACCAACAAC CATGGGCGGT CAACGACACA TTCTCGTACG GCTTTGCGGG CATCTTTATC     120<br>
ACAGGCCATG TCGAG 135<br>
(2) INFORMATION FOR SEQ ID NO: 54:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 4 5 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: peptide<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 54:<br>
Pro Leu Ala Asp Phe Thr Gly Gly Thr Gly Cys Asn Gly Gly Ser Thr<br>
15	10<br><br>
15<br>
Phe Ser Cys Ser Asn Gin Gin Pro Trp Ala Val Asn Asp Thr Phe Ser<br>
20	25<br>
30<br>
Tyr Gly Phe Ala Gly He Phe He Thr Gly His Val Glu<br>
35	40	45<br>
(2) INFORMATION FOR SEQ ID NO: 55:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 114 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(vi) ORIGINAL SOURCE:<br>
(B) STRAIN: Penicillium verruculosum, ATCC 62396<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 55:<br>
GCCAAATCTG GATGTGATGC TGGTGGAGGT CAAGCCTACA TGTGCTCCAA CCAACAACCT      60<br>
TGGGTAGTCA ACGACAACCT CGCCTACGGT TTCGCCGCAG TCAACATTGC<br>
CGGC	114<br>
(2) INFORMATION FOR SEQ ID NO: 56:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A) LENGTH: 38 amino acids<br><br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: peptide<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 56:<br>
Ala Lys Ser Gly Cys Asp Ala Gly Gly Gly Gin Ala Tyr Met Cys Ser<br>
15	10<br>
15<br>
Asn Gin Gin Pro Trp Val Val Asn Asp Asn Leu Ala Tyr Gly Phe Ala<br>
20	25<br>
30<br>
Ala Val Asn lie Ala Gly 35<br>
(2) INFORMATION FOR SEQ ID NO: 57:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 113 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(vi) ORIGINAL SOURCE:<br>
(B) STRAIN: Poronia punctata<br><br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 57:<br>
TTCGACGTCC GGGTGCGACA ATGGCGGCAG CGCCTTCATG TGCTCTAACC AAAGCCCCTG      60<br>
GGCCGTCAAC GACGATCTGG CCTACGGCTG GGCCGCCGTC TCAATCGCGG<br>
GCC	113<br>
(2) INFORMATION FOR SEQ ID NO: 58:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 37 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: peptide<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 58:<br>
Ser Thr Ser Gly Cys Asp Asn Gly Gly Ser Ala Phe Met Cys Ser Asn<br>
15	10<br>
15<br>
Gin Ser Pro Trp Ala Val Asn Asp Asp Leu Ala Tyr Gly Trp Ala Ala<br>
20	25<br>
30<br>
Val Ser lie Ala Gly 35<br>
(2) INFORMATION FOR SEQ ID NO: 59:<br><br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 177 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(vi) ORIGINAL SOURCE:<br>
(B) STRAIN: Fusariura anguioides, IFO 4467<br>
(XI) SEQUENCE DESCRIPTION: SEQ ID NO: 59:<br>
[■CAACACCGG TGCAGACGTG CGACCGCAAC GACAACCCGC TCTACGACGG :GGGTCGACG    6O<br>
'.GGTCCGGCT GCGACGCCGG CGGCGGCGCC TACATGTGCT CGTCGCACAG ICCGTGGGCC   120<br>
TCAGCGACA GCCTCTCGTA CGGCTGGGCG GCCGTCCGCA TCGCCGGCCA<br>
TCCGAG        177<br>
2) INFORMATION FOR SEQ ID NO: 60:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 59 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: peptide<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 60:<br><br>
Ser Thr Pro Val Gin Thr Cys Asp Arg Asn Asp Asn Pro Leu Tyr Asp'<br>
15	10<br>
15<br>
Gly Gly Ser Thr Arg Ser Gly Cys Asp Ala Gly Gly Gly Ala Tyr Met<br>
20	25<br>
30<br>
Cys Ser Ser His Ser Pro Trp Ala Val Ser Asp Ser Leu Ser Tyr Gly<br>
35	40	45<br>
Trp Ala Ala Val Arg lie Ala Gly Gin Ser Glu<br>
50	55<br>
(2) INFORMATION FOR SEQ ID NO: 61:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 150 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(vi) ORIGINAL SOURCE:<br>
(B) STRAIN: Thielavia thermophila, CBS 174.70<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 61: AACGACAACC CCATCTCCAA CACCAACGCT GTCAACGGTT GTGAGGGTGG<br><br>
TGGTTCTGCT      60<br>
TACGCTTGCT CCAACTACTC TCCCTGGGCT GTCAACGATG ACCTTGCCTA CGGTTTCGCT     12 0<br>
GTTACCAAGA TCTCCGGTGG CTCCGAGGCC 150<br>
(2) INFORMATION FOR SEQ ID NO: 62:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 50 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: peptide<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 62:<br>
Asn Asp Asn Pro lie Ser Asn Thr Asn Ala Val Asn Gly Cys Glu Gly<br>
15	10<br>
15<br>
Gly Gly Ser Ala Tyr Ala Cys Ser Asn Tyr Ser Pro Trp Ala Val Asn<br>
20	25<br>
30<br>
Asp Asp Leu Ala Tyr Gly Phe Ala Val Thr Lys lie Ser Gly Gly Ser<br>
35	40	45<br><br>
Glu Ala 50<br>
(2) INFORMATION FOR SEQ ID NO: 63:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 180 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(vi) ORIGINAL SOURCE:<br>
(A) ORGANISM: Chaetomium cuniculoruin, CBS 799-83<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 63:<br>
GTCAATCAGC CCATCCGAAC GTGTAGTGCC AACGACTCGC CCTTGTCCGA CCCAAATACC      60<br>
CCAAGTGGCT GTGACGGTGG TAGCGCCTTC ACTTGTTCCA ACAACTCCCC GTGGGCAGTC     120<br>
GATGACCAGA CAGCTTATGG CTTTGCGGCA ACAGCCATCA GTGGCCAGTC CGAGAGCAGC     180<br>
(2) INFORMATION FOR SEQ ID NO: 64:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 60 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br><br>
(ii) MOLECULE TYPE: peptide<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 64:<br>
Val Asn Gin Pro lie Arg Thr Cys Ser Ala Asn Asp Ser Pro Leu Ser<br>
15	10<br>
15<br>
Asp Pro Asn Thr Pro Ser Gly Cys Asp Gly Gly Ser Ala Phe Thr Cys<br>
20	25<br>
30<br>
Ser Asn Asn Ser Pro Trp Ala Val Asp Asp Gin Thr Ala Tyr Gly Phe<br>
35	40	45<br>
Ala Ala Thr Ala lie Ser Gly Gin Ser Glu Ser Ser<br>
50	55	60<br>
(2) INFORMATION FOR SEQ ID NO: 65:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 159 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: CDNA<br>
(iv) ORIGINAL SOURCE: Chaetomium virescens, CBS 547.75<br><br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 65:<br>
ACCTGCGACA AGAAGGACAA CCCCATCTCT GATGCCAACG CCAAGAGCGG CTGTGATGGC      60 5<br>
GGTTCTGCTT TCGCCTGCAC CAACTACTCT CCCTTCGCCG TCAACGACAA CCTCGCCTAC     120<br>
GGTTTCGCTG CCACCAAGCT TGCTGGAGGC TCCGAGGCT<br>
I	159<br>
(2) INFORMATION FOR SEQ ID NO: 66:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 53 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: peptide<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 66:<br>
Thr Cys Asp Lys Lys Asp Asn Pro lie Ser Asp Ala Asn Ala Lys Ser<br>
15	10<br>
15<br>
Gly Cys Asp Gly Gly Ser Ala Phe Ala Cys Thr Asn Tyr Ser Pro Phe<br>
20	25<br>
30<br>
Ala Val Asn Asp Asn Leu Ala Tyr Gly Phe Ala Ala Thr Lys Leu Ala<br><br>
35	40	45<br>
Gly Gly Ser Glu Ala 50<br>
(2) INFORMATION FOR SEQ ID NO: 67:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 81 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(vi) ORIGINAL SOURCE:<br>
(B) STRAIN: Colletotrichum lagenarium<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 67:<br>
ACCTGCTACG CCAATGACCA GCGCATCGCC  GACCGCAGCA CCAAGTCCGG CTGCGACGGC      60<br>
GGCTCGGCCT ACTCCTGTTC T 81<br>
(2) INFORMATION FOR SEQ ID NO: 68:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 27 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: peptide<br><br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 68:<br>
Thr Cys Tyr Ala Asn Asp Gin Arg lie Ala Asp Arg Ser Thr Lys Ser<br>
15	10<br>
15<br>
Gly Cys Asp Gly Gly Ser Ala Tyr Ser Cys Ser<br>
20	25<br>
(2) INFORMATION FOR SEQ ID NO: 69:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 160 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(vi) ORIGINAL SOURCE:<br>
(B) STRAIN: Phycomyces nitens<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 69:<br>
ACCTGTGACA AGAAGGACAA CCCCATCTCA AACTTGAACG CTGTCAACGG TTGTGAGGGT      60<br>
GGTGGTTCTG CCTTCGCCTG CACCAACTAC TCTCCTTGGG CGGTCAATGA CAACCTTGCC     12 0<br>
TACGGCTTCG CTGCAACCAA GCTTGCCGGT GGCTCCGAGG 160<br><br>
(2) INFORMATION FOR SEQ ID NO: 70:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 53 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: peptide<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 70:<br>
Thr Cys Asp Lys Lys Asp Asn Pro lie Ser Asn Leu Asn Ala Val Asn<br>
15	10<br>
15<br>
Gly Cys Glu Gly Gly Gly Ser Ala Phe Ala Cys Thr Asn Tyr Ser Pro<br>
20	25<br>
30<br>
Trp Ala Val Asn Asp Asn Leu Ala Tyr Gly Phe Ala Ala Thr Lys Leu<br>
35	40	45<br>
Ala Gly Gly Ser Glu 50<br>
(2) INFORMATION FOR SEQ ID NO: 71:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 165 base pairs<br>
(B)	TYPE: nucleic acid<br><br>
5372<br><br><br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(iv) ORIGINAL SOURCE: Trichothecium roseum, IFO<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 71:<br><br>
CCAGTAGGCA CCTGCGACGC CGGCAACAGC CCCCTCGGCG ACCCCCTGGC CAAGTCTGGC      60<br>
TGCGAGGGCG GCCCGTCGTA CACGTGCGCC AACTACCAGC CGTGGGCGGT CAACGACCAG     120<br>
CTGGCCTACG GCTTCGCGGC CACGGCCATC AACGGCGGCA CCGAG 165<br>
(2) INFORMATION FOR SEQ ID NO: 72:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 55 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: peptide<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 72:<br>
Pro Val Gly Thr Cys Asp Ala Gly Asn Ser Pro Leu Gly Asp Pro Leu<br>
1	5	10<br>
15<br><br>
Ala Lys Ser Gly Cys Glu Gly Gly Pro Ser Tyr Thr Cys Ala Asn Tyr<br>
20	25<br>
30<br>
Gin Pro Trp Ala Val Asn Asp Gin Leu Ala Tyr Gly Phe Ala Ala Thr<br>
35	40	45<br>
Ala lie Asn Gly Gly Thr Glu<br>
50	55<br><br>
REFERENCES<br>
Background of the invention:<br>
GB-A-1368599<br>
EP-A-0 307 564<br>
EP-A-0 435 876<br>
WO 91/17243<br>
WO 91/10732<br>
WO 91/17244<br>
WO 95/24471<br>
WO 95/26398<br>
Methods in Enzymology, 1988, Vol. 160, p. 200-391 (edited by Wood, W.A. and Kellogg, S.T.).<br>
Beguin, P., "Molecular Biology of Cellulose Degradation", Annu. Rev. Microbiol. (1990), Vol. 44, pp. 219-248.<br>
Henrissat, B., "Cellulases and their interaction with cellulose". Cellulose (1994), Vol. 1, pp. 169-196.<br>
T.-M. Enveri, "Microbial Cellulases" in W.M. Fogarty, Microbial Enzymes and Biotechnology, Applied Science Publishers, 183-224 (1983).<br>
Beguin, P. and Aubert, J-P., "The biological degradation of cellulose", FEMS Microbiology Reviews IJ. (1994) 25-58.<br>
Sheppard, P.O., et al., "The use of conserved cellulase<br><br>
family-specific sequences to clone Cellulase horoologue cDNAs from Fusarium oxysporum,   Gene, (1994), Vol. 15, pp. 163-167.<br>
Saloheimo, A., et al., "A novel, small endoglucnaase gene, eglS,   from Trichoderma reesei   isolated by expression in yeast", Molecular Microbiology (1994), Vol. 13(2), pp. 219-228.<br>
van Arsdell, J.N. et al, (1987) Cloning, characterization, and expression in Saccharomyces cerevisiae  of endoglucanase I from Trichoderma reesei, Bio/Technology 5: 60-64.<br>
Penttila, M. et al, (1986) Homology between cellulase gnees of Trichoderma reesei:   complete nucleotide sequence of the endoglucanase I gene. Gene  45:253-263.<br>
Saloheimo, M. et al, (1988) EGIII, a new endoglucanase from Trichoderma reesei:   the characterization of both gene and enzyme. Gene  63:11-21.<br>
Gonzales, R., et al., "Cloning, sequence analysis and yeast expression of the egll  gene from Trichoderma longibrachiatum",  Appl. Microbiol. Biotechnol. (1992), Vol. 38, pp. 370-375.<br>
Ooi, T. et al. "Cloning and sequence analysis of a cDNA for cellulase (FI-CMCase) from Aspergillus  aculeatus" Curr. Genet. (1990), Vol. 18, pp. 217-222.<br>
Ooi, T. et al, "Complete nucleotide sequence of a gene coding for Aspergillus  aculeatus  cellulase (FI-CMCase)" Nucleic Acids Research (1990), Vol. 18, No. 19, p. 5884.<br>
Xue, G. et al., "Cloning and expression of multiple cellulase cDNAs fromthe anaerobic rumen fungus<br><br>
Neocallimastix patriciarum  in E.   coli,   J. Gen. Microbiol. (1992), Vol. 138, pp. 1413-1420.<br>
Xue, G. et al., "A novel polysaccharide hydrolase cDNA {celD)   from Neocallimastix patriciarum  encoding three multi-functional catalytical domains with high endoglucanase, cellobiohydrolase and xylanase activities", J. Gen. Microbiol. (1992), Vol. 138, pp. 2397-2403.<br>
Zhou, L. et al., "Intronless celB  from the anaerobic fungus Neocallimastix patriciarum  encodes a modular family A endoglucanase", Biochem. J. (1994), Vol. 297, pp. 359-364.<br>
Dalboge, H. and Heldt-Hansen, H.P., "A novel method for efficient expression cloning of fungal enzyme genes", Mol. Gen. Genet. (1994), Vol. 243, pp. 253-250.<br>
Ali, B.R.S. et al., "Cellulases and hemicellulases of the anaerobic fungus Pirojnyces constitute a multiprotein cellulose-binding complex and are encoded by multigene families", FEMS Microbiol. Lett. (1995), Vol. 125, No. 1, pp. 15-21.<br>
DNA Data Bank of Japan (DDBJ).<br>
Wang, H.Y. and Jones, R.W.: "Cloning, characterization and functional expression of an endoglucanase-encoding gene from the phytopathogenic fungus Macrophomina phaseplina",   Gene, 158:125-128, 1995.<br>
Wang, H.Y. and Jones, R.W.: "A unique endoglucanase-encoding gene cloned from the phytopathogenic fungus Macrophomina phasedina",   Appl. and Environm. Microbiology, 61:2004-2006, 1995.<br><br>
B. Henrissat: Biochem. J., 280:309-316, 1991.<br>
Schauwecker, F., Wanner, G., Kahmann, R.: "Filament-specific expression of a cellulase gene in the dimorphic fungus Ustilago maydis", 1995, Biological Chemistry Hoppe-Seyler, 376:617-625.<br>
WO 93/20193<br>
WO 94/21801<br>
WO 94/26880<br>
WO 95/02043<br>
The Drawings:<br>
Feng and Doolittle, 1987, J. Mol. Evol. 25: 351-3 60.<br>
NIH Data Base (Entrez, version spring 1996) available on<br>
World Wide Web: (htt-<br>
p://www3.ncbi.nlm.nih.gov/htbin/ef/entrezTAX).<br>
Eriksson, O.E. &amp; Hawksworth, D.L.: Systema Ascomycetum vol 12 (1993).<br>
Jiilich, W. : Higher Taxa of Basidiomycetes, Bibliotheca Mycologia 85, 485pp (1981).<br>
O'Donnell, K.: Zygomycetes in culture, University of Georgia, US,  257pp (1979).<br>
Hawksworth, D.L., Kirk, P.M., Sutton, B.C. and Pegler, D.N.: Dictionary of the fungi, International Mycological Institute, 616pp (1995);<br>
Von Arx, J.A.: The genera of fungi sporulating in<br><br>
culture, 424pp (1981).<br>
Detailed Description:<br>
Ford, et al., Protein Expression and Purification 2:   95-107, 1991.<br>
Cunningham and Wells, Science 244. 1081-1085, 1989.<br>
de Vos et al.. Science 255: 306-312, 1992.<br>
Smith et al., J. Mol. Biol. 224: 899-904, 1992.<br>
Wlodaver et al., FEES Lett. 309: 59-64, 1992.<br>
Tomme, P. et al. "Cellulose-Binding Domains: Classi¬fication and Properties" in "Enzymatic Degradation of Insoluble Carbohydrates", John N. Saddler and Michael H. Penner (Eds.), ACS Symposium Series, No. 618, 1996.<br>
WO 90/00609<br>
WO 95/16782<br>
Needleman, S.B. and Wunsch, CD., Journal  of Molecular Biology,   41- 443-453, 1970.<br>
WO 94/14953<br>
Sambrook, J., Fritsch, E. F. &amp; Maniatis, T. 1989. Molecular Cloning: A Laboratory Manual. Cold Spring Harbor Lab., Cold Spring Harbor, NY.<br>
&gt;     Beaucage and Caruthers, Tetrahedron Letters 22 (1981), 1859 - 1869.<br><br>
Matthes et al., EMBO Journal 3. (1984), 801 - 805.<br>
US 4,683,202<br>
Saiki et al.. Science 239 (1988), 487 - 491.<br>
Hitzeman et al., J. Biol. Chero. 255 (1980), 12073 -12080.<br>
Alber and Kawasaki, J. Mol. Appl. Gen. 2, (1982), 419 -434) .<br>
Young et al., in Genetic Engineering of Microorganisms for Chemicals (Hollaender et al, eds.). Plenum Press, New York, 1982).<br>
US 4,599,311<br>
Russell et al., -Nature 304 (1983), 652 - 654.<br>
McKnight et al., The EMBO J. 4 (1985), 2093 - 2099<br>
P.R. Russell, Gene 40, 1985, pp. 125-130.<br>
US 4,870,008<br>
O. Hagenbuchle et al., Nature 289. 1981, pp. 643-646.<br>
L.A. Vails et al., Cell 48., 1987, pp. 887-897.<br>
WO 87/02670<br>
M. Egel-Mitani et al., Yeast 6. 1990, pp. 127-137.<br>
US 4,546,082<br>
EP 16 201<br><br>
EP 123 294<br>
EP 123 544<br>
EP 163 529<br>
WO 89/02463<br>
WO 92/11378<br>
US 4,599,311<br>
US 4,931,373<br>
US 4,870,008<br>
US 5,037,743<br>
US 4,845,075<br>
US 4,931,373<br>
Gleeson et al., J. Gen. Microbiol. 132. 1986, pp. 3459-3465.<br>
US 4,882,279<br>
EP 272 277<br>
EP 230 023<br>
Malardier et al., 1989, Gene 78: 147-156.<br>
WO 93/11249.<br>
WO 94/14953.<br><br>
wo 95/02043.<br>
Horton, R. M., Hunt, H. D., Ho, S. N., Pullen, J. K., and Pease, L. R. (1989) Gene  77, 61-68 i<br>
Dalbcsge, H., and Heldt-Hansen, H. (1994) Mol.   Gen.   Genet. 243, 253-260<br>
Christensen, T., Woldike, H., Boel, E., Mortensen, S. B., ) Hjortshcj, K., Thim, L., and Hansen, M. T. (1988) Bio/Technology  6, 1419-1422<br>
Sanger, F., Nicklen, S., and Coulson, A. R. (1977) Proc. Natl.   Acad.   Sci.   D.   S.   A.   74, 5463-5467<br>
Devereux, J., Haeberli, P., and Smithies, O. (1984) Nucleic Acids Res.   12, 387-395<br>
Becker, D. M. &amp; Guarante, L. 1991. Methods Enzymol. 194: )  182-187.<br>
Gubler, U. &amp; Hoffman, B. J. 1983. Gene 25: 263-269.<br>
R. Higuchi, B. Krummel, and R.K. Saiki (1988). A general &gt; method of in vitro preparation and specific mutagenesis of DNA fragments: study of protein and DNA interactions. Nucl. Acids Res. 16: 7351-7367.<br>
Sanger, F., Nicklen, S. &amp; Coulson, A. R. 1977. Proc. )  Natl. Acad. Sci. U. S. A. 74: 5463-5467.<br>
N. Axelsen et al., A Manual of Quantitative Immunoelectrophoresis. Blackwell Scientific Publications, 1973, Chapters 2,3,4 and 23.<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
WE CLAIM;<br>
1.	An enzyme preparation exhibiting endoglucanase activity, wherein the enzyme is<br>
a)	encoded by a DNA construct comprising the DNA sequence shown in SEQ ID No. 8  or a DNA construct comprising the DNA sequence obtainable from the plasmid in Saccharomyces cerevisiae DSM 10081, or<br>
b)	encoded by a DNA construct comprising an analogue of the DNA sequence shown in SEQ ID No. 8 or a DNA construct comprising the DNA sequence obtainable from the plasmid in Saccharomyces cerevisiae DSM 10081, which DNA sequence has at least 75% identity with the DNA sequence shown in SEQ ID No. 8 or the DNA sequence obtainable from the plasmid in Saccharomyces cerevisiae DSN 10081, or<br>
c)	a polypeptide which has at least 70% identity with the amino acid sequence shown<br>
_      in SEQ ID No. 9.<br>
2.	The enzyme preparation according to claim 1, wherein it   said enzyme.<br>
3.	The enzyme preparation according to claim 1 or 2, wherein it additionally comprises one or more enzymes selected from the group consisting of galactanases, xylanases, arabinanases, pectin acetyl esterases, polygalacturonases, rhamnogalacturonases, pectin lyases, pectate lyases, endogucanases, pectin methylesterases, proteases, lipases, amylases, cutinases, peroxidases, laccases, cellobiohydrolases and transglutaminases.<br>
4.	A detergent composition comprising the enzyme preparation according to any of<br>
the claims 1 to 3, and a compound selected from the group consisting of a surfactant, a<br>
builder compound, and a fabric softening agent.<br>
5.	The composition according to claim 4, wherein the surfactant is a nonionic,<br>
anionic, cationic, zwitterionic, ampholytic or amphoteric surfactant.<br>
-270-<br><br>
6.	The composition according to claim 4, wherein the fabric softening agent is a cationic or nonionic softening agent, preferably a quaternary ammonium compound, and which optionally further comprises one or more compounds selected from a surfactant, an electrolyte, a buffer, an antioxidant and a liquid carrier.<br>
7.	A method of providing colour clarification of laundry, wherein said method comprises<br>
treating the laundry with a soaking, washing or rinsing liquor comprising an enzyme preparation according to any of claims 1 to 3 or<br>
with a detergent composition according to any one of the claims 4 to 6.<br>
8.	The method according to claim 7, wherein the laundry is treated in a washing machine.<br>
9.	The method according to claim 7 or 8, wherein the endoglucanase is present in the soaking, washing, or rinsing liquor in an effective amount of between 1 and 1000 S— CEVU, preferably between 5 and 200 S-CEVU, per liter of liquor during machine cycle use conditions.<br><br>
10.	The method according to any of the claims 7 to 9, wherein the pH of the soaking, washing, or rinsing liquor is between 4 and 11, preferably between 6 and 10.5.<br>
11.	The method according to any of the claims 7 to 10, wherein the temperature is between 15°C and 60°C.<br>
12.	A method for biopolishing of a cellulosic fabric which method comprises treating the yam surface of said fabric with the enzyme preparation according to any of claims lto3.<br>
-271-<br><br>
13.	The method according to claim 12, wherein it takes place in the wet processing of the manufacture of knitted and woven fabrics.<br>
14.	A method for "stone-washing" which method comprises treating a dyed cellulosic fabric with the enzyme preparation according to any of claims 1 to 3.<br>
15.	An enzyme preparation exhibiting endoglucanase activity substantially as herein described and exemplified.<br><br>
16.	A detergent composition comprising the enzyme preparation substantially as herein described and exemplified.<br>
17.	A method of providing colour clarification substantially as herein described and exemplified.<br><br>
18.	A method for biopolishing of a cellulosic fabric substantially as herein described and exemplified.<br>
19.	A method for "stone-washing" substantially as herein described and exemplified.<br><br></cdna></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDkxOS1jaGUtMjAwMyBhYnN0cmFjdCBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">0919-che-2003 abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDkxOS1jaGUtMjAwMyBkZXNjcmlwdGlvbiAoY29tcGxldGUpIGR1cGxpY2F0ZS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">0919-che-2003 description (complete) duplicate-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDkxOS1jaGUtMjAwMyBkZXNjcmlwdGlvbiAoY29tcGxldGUpIGR1cGxpY2F0ZS0yLnBkZg==" target="_blank" style="word-wrap:break-word;">0919-che-2003 description (complete) duplicate-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDkxOS1jaGUtMjAwMyBkZXNjcmlwdGlvbiAoY29tcGxldGUpIGR1cGxpY2F0ZS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">0919-che-2003 description (complete) duplicate-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDkxOS1jaGUtMjAwMyBkZXNjcmlwdGlvbiAoY29tcGxldGUpIGR1cGxpY2F0ZS00LnBkZg==" target="_blank" style="word-wrap:break-word;">0919-che-2003 description (complete) duplicate-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDkxOS1jaGUtMjAwMyBkZXNjcmlwdGlvbiAoY29tcGxldGUpIGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">0919-che-2003 description (complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDkxOS1jaGUtMjAwMyBkZXNjcmlwdGlvbiAoY29tcGxldGUpLTEucGRm" target="_blank" style="word-wrap:break-word;">0919-che-2003 description (complete)-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDkxOS1jaGUtMjAwMyBkZXNjcmlwdGlvbiAoY29tcGxldGUpLTIucGRm" target="_blank" style="word-wrap:break-word;">0919-che-2003 description (complete)-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDkxOS1jaGUtMjAwMyBkZXNjcmlwdGlvbiAoY29tcGxldGUpLTMucGRm" target="_blank" style="word-wrap:break-word;">0919-che-2003 description (complete)-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDkxOS1jaGUtMjAwMyBkZXNjcmlwdGlvbiAoY29tcGxldGUpLTQucGRm" target="_blank" style="word-wrap:break-word;">0919-che-2003 description (complete)-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDkxOS1jaGUtMjAwMyBmb3JtLTI2LnBkZg==" target="_blank" style="word-wrap:break-word;">0919-che-2003 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTE5LWNoZS0yMDAzLWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">919-che-2003-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTE5LWNoZS0yMDAzLWFzc2lnbmVtZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">919-che-2003-assignement.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTE5LWNoZS0yMDAzLWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">919-che-2003-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTE5LWNoZS0yMDAzLWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">919-che-2003-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTE5LWNoZS0yMDAzLWNvcnJlc3BvbmRuZWNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">919-che-2003-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTE5LWNoZS0yMDAzLWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">919-che-2003-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTE5LWNoZS0yMDAzLWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">919-che-2003-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTE5LWNoZS0yMDAzLWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">919-che-2003-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTE5LWNoZS0yMDAzLWZvcm0gMjYucGRm" target="_blank" style="word-wrap:break-word;">919-che-2003-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTE5LWNoZS0yMDAzLWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">919-che-2003-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTE5LWNoZS0yMDAzLW90aGVyZG9jdW1lbnQucGRm" target="_blank" style="word-wrap:break-word;">919-che-2003-otherdocument.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTE5LWNoZS0yMDAzLXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">919-che-2003-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="219744-an-adjustable-bluetooth-wireless-earphone.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="219746-method-of-manufacturing-visco-elastic-foam-pillow.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>219745</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>919/CHE/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>27/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>04-Jul-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>13-May-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>10-Nov-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVOZYMES A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>LASSEN, SOREN FLENSTED</td>
											<td></td>
										</tr>
										<tr>
											<td>2</td>
											<td>SCHULEIN MARTIN</td>
											<td></td>
										</tr>
										<tr>
											<td>3</td>
											<td>NIELSEN, RUBY ILIM</td>
											<td></td>
										</tr>
										<tr>
											<td>4</td>
											<td>TAKAGI SHINOBU</td>
											<td></td>
										</tr>
										<tr>
											<td>5</td>
											<td>IHARA MICHIKO</td>
											<td></td>
										</tr>
										<tr>
											<td>6</td>
											<td>ANDERSEN, LENE NONBOE</td>
											<td></td>
										</tr>
										<tr>
											<td>7</td>
											<td>LANGE, LENE</td>
											<td></td>
										</tr>
										<tr>
											<td>8</td>
											<td>KAUPPINEN, MARKUS SAKARI</td>
											<td></td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/56</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/219745-a-novel-enzyme-preparation by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:30:49 GMT -->
</html>
