<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/220139-a-process-for-construction-of-a-chimeric-promoter-of-expression by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:04:46 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 220139:&quot;A PROCESS FOR CONSTRUCTION OF A CHIMERIC PROMOTER OF EXPRESSION&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A PROCESS FOR CONSTRUCTION OF A CHIMERIC PROMOTER OF EXPRESSION&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A process for construction of a chimeric promoter of expression, said process comprising a series of insertions of at least one nucleic acid sequence into a nucleic acid derived from a gene encoding a high molecular weight wheat glutenin according to SEQ.ID01, and wherein the inserted nucleic acid sequences comprise regulatory or functional boxes of the kind such as herein described; and performing the conventional steps of amplification and ligation to prepare the chimeric promoter.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present invention relates to a process for construction of a chimeric promoter of expression.<br>
[DESCRIPTION]<br>
The present invention relates to chimeric promoters of<br>
expression, intended in particular for use in the field of plant<br>
biotechnology.<br>
In general, the promoters of expression are known in the field<br>
of biotechnology and genetic.manipulation. With regard more<br>
particularly to plant biotechnology, the level of expression of<br>
a gene encoding a polypeptide to be produced in a host cell is<br>
t<br>
often dependent on the promoter used. The various promoters commonly used are often limited to specific applications or tissues, because of their tissue specificity or strength of expression. Mention may be made/ for example, of the 35S promoter of the cauliflower mosaic virus, as a promoter which is relatively strong compared, for example, with the promoter originating from- the nos gene, these two promoters being more particularly used in the field of plant biotechnology. There is thus a need for novel promoters which make it possible to overcome the drawbacks o-f using the current promoters. An attempt at satisfying this need has been reported in the PCT patent application published under the number WO 97/20056, which describes increasing the level of gene expression by using enhancers, (i.e. having a positive effect on the activity of a promoter), in known promoters. The nucleotide sequences of enhancers are rich in A and T bases, the total amount of these bases constituting more than 50% of the nucleotide sequence of<br>
the enhancer. In particular, the Applicants of this application<br>
i recommend the use of an enhancer region originating from the<br>
plastocyanine promoter of,pea.<br>
The expressions used in the description and claims have the<br>
following meaning:<br><br>
-	the term "nucleic acid" means DNA or RNA;<br>
-	the term "nucleic acid sequence" means a single- or double-<br>
stranded oligomer or polymer of nucleotide bases read from the<br>
5' end towards the 3' end, which comprises self-replicating<br>
plasmids, genes and DNA or RNA polymers which are or are not<br>
infectious, and functional or nonfunctional DNA or RNA. In the<br>
nucleotide notation used in the present application, except<br>
where specifically mentioned, the left-hand end of a single-<br>
stranded nucleotide sequence is the 5' end;<br>
-	the phrase "nucleic acid sequence derived" means that the<br>
sequence is derived directly or indirectly from the sequence to<br>
which reference is made, for example by substitution, deletion,<br>
addition, mutation, fragmentation and/or synthesis of one or<br>
more nucleotides;<br>
-	the term "promoter" or the phrase "promoter nucleic acid<br>
sequence" means a nucleic acid region which is upstream of the<br>
translation start codon, and which is involved in the<br>
recognition and binding of RNA polymerase and of other proteins<br>
for transcription;<br>
-	"plant promoter" is a promoter which is capable of initiating<br>
transcription in plant cells;<br>
-	"constitutive promoter" is a promoter which is capable of<br>
expressing nucleic acid sequences which are linked in a<br>
functional manner to said promoter, in all or practically all<br>
the tissues of the host organism throughout the development of<br>
said organism;<br>
-	"specific tissue promoter" is a promoter which is capable of<br>
selectively expressing nucleic acid sequences which are linked<br>
in functional manner to said promoter, in certain specific<br>
tissues of the host organism;<br>
-	the phrase "linked in a functional manner" or "functionally<br>
linked" means the linking of a nucleic acid sequence, for<br>
example a promoter or a regulatory or functional box, to another<br>
nucleic acid sequence, for example another regulatory or<br><br>
functional box or a gene to be expressed which encodes a protein co be produced, such that the sequences exert their primary functions in a cell, a genome, a vector or an expression cassette into which they have been introduced, i.e. that they are functional in such environments. It should also be understood that, in the case of a promoter, the sequence of the promoter also includes sequences transcribed between the transcription start site and the translation start site;<br>
-	the term "expression cassejtte" means nucleotide sequences<br>
which are capable of directing the expression of a nucleic acid<br>
sequence, or of a gene, encoding a polypeptide to be produced in<br>
a host organism which is compatible with such sequences. Such<br>
cassettes include at least one promoter and one transcription<br>
termination signal, and optionally other factors which are<br>
required or useful for expression;<br>
-	the term "vector" means expression systems, for example DNA-<br>
coated projectiles, nucleic acid-based transit vehicles and<br>
nucleic acid molecules which have been adapted to deliver the<br>
nucleic acid, and autonomous self-replicating circular DNA, for<br>
example plasmids, cosmids, phagemids, etc. If a recombinant cell<br>
culture or microorganism is described as being the host of an<br>
"expression vector", this includes extrachromosomal circular DNA<br>
(such as, for example, mitochondrial or chloroplastic DNA) and<br>
DNA which has been integrated into the host chromosome(s), the<br>
vector being able to be either replicated stabily by the cells<br>
during mitosis, as an autonomous structure which is integrated<br>
into the genome of the host, or maintained in the nucleus or<br>
cytoplasm of the host;<br>
-	the term "plasmid" means an autonomous circular DNA molecular<br>
capable of replicating in a cell, and comprises both the so-<br>
called "expression" plasmids and the so-called "nonexpression"<br>
plasmids. If a recombinant cell culture or microorganism is<br>
described as being the host of an "expression" plasmid, this<br>
comprises both extrachromosomal circular DNA molecules and DNA<br>
which has been integrated into the host chromosome(s). If the plasmid is maintained by a host cell, the plasmid is either stabily replicated by the cells during mitosis, as an autonomous structure, or integrated into the genome of the host;<br>
-	the term "heterologous sequence" or "heterologous nucleic acid<br>
sequence" means a sequence originating from a source, or from a<br>
species, which is foreign to its environment or, if it<br>
originates from the same environment, has been modified with<br>
respect to its original form. The modification of the nucleic<br>
acid sequence can take place, for example, by treating the<br>
nucleic acid with a restriction enzyme so as to generate a<br>
nucleic acid fragment which is capable of being linked in a<br>
functional manner to a promoter. The modification can also take<br>
place via techniques such as site-direct mutagenesis;<br>
-	the term "box" means a nucleic acid sequence to which a<br>
regulatory function is attributed;<br>
-	the term "like" means that the box and/or the nucleic acid<br>
sequence with which this term is associated comprises a certain<br>
sequence identity or a consensus with a box and/or a known so-<br>
called reference nucleic acid sequence, preferably a sequence<br>
identity of at least 50%, more preferably a sequence identity of<br>
at least 75%, and more particularly a sequence identity of at<br>
least 90% with the reference sequence. The percentage sequence<br>
identity is calculated on the basis of a window of comparison of<br>
at least 6 nucleotide bases. The determination of a window of<br>
comparison can be carried out using sequence alignment<br>
algorithms in order to determine homology with a reference<br>
sequence, for example the local homology algorithm, the homology<br>
alignment algorithm, and the similarity search algorithm, these<br>
algorithms also existing in computer form, known under the names<br>
GAP, BESTFIT, FASTA and TFASTA. The percentage sequence identity<br>
is obtained by comparing the reference sequence with the box<br>
and/or the nucleic acid sequence;<br>
-	the term "located" means the position on a nucleic acid<br><br>
sequence of an identified element, such as a "box", a restriction site or a codon having a specific function. The position, which is given by a number, refers to the position of the start of the element in the nucleic acid sequence, except where specifically mentioned, in the direction of reading of the latter, i.e. in the 5'-&gt;3' direction;<br>
-	the term "Element 300", "EM", "endosperm motif", "P-box" or<br>
"Prolamine-like" box means a regulatory or functional motif or<br>
element which encodes storage proteins of many cereals, and<br>
which is involved in the common regulatory mechanism mediating<br>
the coordinated expression of zein genes during the development<br>
of the maize albumen;<br>
-	the term "G-like" box means an ACGT core motif, the functional<br>
contribution to transcriptional regulation of which has been<br>
defined in few cases, but which appears to be necessary for<br>
maximum expression of a promoter;<br>
-	the term "enhancer" box means regulatory DNA sequences which<br>
can act in cis at a distance, independently of their orientation<br>
and upstream or downstream of their target promoter, and which<br>
can generally consist of multiple short motifs which bind a<br>
combination of trans factors so as to possibly confer<br>
inductibility, tissue specificity or a general improvement in<br>
the activity of a promoter;<br>
-	the term "GATA" box means a regulatory or functional motif or<br>
element which may be required for the activity of many light-<br>
regulated promoters;<br>
-	the term "asl" or "activating sequence 1" box means a<br>
regulatory or functional motif or element preferably originating<br>
from the 35S promoter of the CaMV (cauliflower mosaic virus),<br>
which can confer expression in roots and which can play a more<br>
complex role in promoter regulation through synergistic<br>
interactions with other cis-activating elements, and which can<br>
optionally be salicylic acid-inducible;<br>
-	the term "as2" or "activating sequence 2" box means a<br><br>
regulatory or functional motif or element preferably originating from the 35S promoter of the CaMV (cauliflower mosaic virus), which can confer expression in the photosynthetic tissues, and which can have transcriptional activator activity;<br>
-	the term "cereal" box means means a regulatory or functional<br>
motif or element which may be involved in specific expression in<br>
wheat seeds;<br>
-	the term "GC-rich" box means a regulatory or functional motif<br>
or element which is rich in G or C nucleotides, for example<br>
originating from a geminivirus;<br>
-	the term "transgenic plant" means a plant which has been<br>
obtained by genetic manipulation techniques, and covers whole<br>
plants obtained by such manipulations, regenerated plants which<br>
integrate into their genome, or which express, such<br>
manipulations, their progeny, and the plant organs, for example<br>
roots, stalks and leaves, obtained by these techniques. The<br>
transgeneic plants according to the present invention can have<br>
various levels of ploidy, and can in particular be polyploid,<br>
diploid or haploid;<br>
-	the term "propagule" means a mass or group of plant cells<br>
which is structured or unstructured, and which enables the<br>
regeneration of a whole plant, for example explants, calluses,<br>
stalks, leaves, roots, cuttings and even seeds.<br>
The Applicant of the present invention has taken an approach which is different from that of the Applicant of the PCT application previously discussed. Specifically, the Applicant of the present invention has succeeded, surprisingly, in producing chimeric promoters which make it possible to satisfy the need described above, and in particular which make it possible to increase the level of expression of a gene or of a nucleic acid sequence encoding a polypeptide to be produced, in a host cell, and in particular a plant cell, with respect to the existing promoters most commonly used. Moreover, the Applicant has succeeded, at the same time, in producing a complete range of<br><br>
promoters so as to be able to choose the one which is suitable<br>
i<br>
for use according to the use envisaged and the environment of its implementation, and thus to be able to control in some way the level of expression of a gene to be expressed which encodes a polypeptide to be produced. One example of use of this principle would be to use one of the weaker promoters of the invention to direct and/or control the expression of a protein or enzyme, for example an agent for selection in a plant, for example, resistance to antibiotics or to herbicides, or a coenzyme or cofactor required for assembling a more important protein, and to use a stronger promoter in accordance with the invention to, for example, control the expression of a polypeptide having a therapeutic effect.<br>
Yet another advantage of the present invention is that the promoters prepared in accordance with the invention allow both a specific expression in the seeds, but also a deregulation in order to favour expression in other organs, for example, leaves, stalks and the plant vascular system.<br>
Consequently, a main subject of the present invention is a chimeric promoter comprising at least one nucleic acid sequence derived from a gene encoding a high molecular weight wheat glutenin, and preferably a nucleic acid sequence derived from the wheat Dx5 or Bx7 gene encoding a high molecular weight wheat glutenin.<br>
Preferably, the chimeric promoter comprises at least one nucleic acid sequence derived from a gene encoding a high molecular weight wheat glutenin, the sequence of which is identified under the number SEQ.ID01.<br>
More preferably, the nucleic acid sequence derived from a gene encoding a high molecular weight wheat glutenin corresponds to a sequence chosen from the group consisting of the sequences identified under the numbers SEQ.ID02, SEQ.ID03, SEQ.ID04, SEQ.ID05, SEQ.ID06, SEQ.ID07, SEQ.ID08, SEQ.ID09, SEQ.ID10, SEQ.ID11, SEQ.ID12, SEQ.ID13, SEQ.ID16, SEQ.ID17, SEQ.ID18,<br><br>
SEQ.ID19, SEQ.ID20, SEQ.ID21 and SEQ.ID22.<br>
t<br>
In addition, the Applicant has noted that it is possible to<br>
construct promoters, and in particular plant promoters, which<br>
have an advantageous promoter activity both in dicotyledonous<br>
plants and in monocotyledonous plants, by combining a certain<br>
number of regulatory or functional boxes, using a nucleic acid<br>
sequence derived from a gene encoding a high molecular weight<br>
wheat glutenin as defined above.<br>
Thus, another subject of the present invention is a chimeric<br>
promoter of expression which comprises a nucleic acid sequence<br>
derived from a gene encoding a high molecular weight wheat<br>
glutenin, and which comprises, in the 3' position, a "TATA" box<br>
and a transcription start site (+1).<br>
Preferably, the promoter also comprises at least one "enhancer"<br>
box functionally linked in the 5' position upstream of the<br>
"TATA" box and the transcription start site (+1).<br>
More preferably, the promoter also comprises at least one "G-<br>
like" box functionally linked in the 5' position upstream of the<br>
"enhancer" box.<br>
Even more preferably, the promoter also comprises at least one<br>
"P-like" box functionally linked in the 5' position upstream of<br>
the "enhancer" box.<br>
Avantageously, the promoter also comprises at least one "GATA"<br>
box functionally linked in the 5' position upstream of at least<br>
the "enhancer" box.<br>
Preferably, the promoter also comprises at least one cereal box<br>
functionally linked in the 5' position upstream of the<br>
"enhancer" box.<br>
Even more preferably, the promoter comprises two contiguous<br>
"cereal" boxes functionally linked in the 5' position upstream<br>
of the "enhancer" box.<br>
According to one preferred embodiment of the promoters according<br>
to the present invention, they also comprise at least one "asl"<br>
box or at least one "as2" box, or an "asl/as2" or "as2/asl"<br><br>
combination of boxes,   or repeat permutations of  these,<br>
functionally linked in the  5'   position upstream of the<br>
transcription  start  site.<br>
According to  another preferred embodiment of the  invention,   the<br>
"asl",   "as2",   "asl/as2"  or "as2/asl" box(es),   or  its(their)<br>
repeat permutations, is(are) functionally linked downstream, in<br>
the 3' position, of the enhancer box.<br>
According to yet another preferred embodiment of the invention,<br>
the promoters also comprise a "GC" box functionally linked<br>
upstream of the "enhancer" box.<br>
Particularly preferably, the promoter comprises two "cereal"<br>
boxes functionally linked in the 5' position upstream of the<br>
"enhancer [lacuna] box, which is itself functionally linked in<br>
the 5' position upstream of an "as2/asl" box.<br>
According to yet another preferred embodiment of the promoters<br>
of the invention, they comprise at least one element chosen from<br>
the group consisting of an "enhancer" box, a "G-like" box, a "P-<br>
like" box, a "GATA" box, a "cereal" box, an "asl" box, an "as2"<br>
box and/or an asl/as2 or "as2/asl" combination of boxes,<br>
optionally repeated, and a "GC-rich" box, functionally linked in<br>
reverse orientation and/or downstream, in the 3' position, of<br>
the transcription start site.<br>
Finally, the chimeric promoters according to the present<br>
invention advantageously comprise at least one nucleic acid<br>
sequence chosen from the group consisting of SEQ.ID02, SEQ.ID03,<br>
SEQ.ID04, SEQ.ID05, SEQ.ID06, SEQ.ID07, SEQ.ID08, SEQ.ID09,<br>
SEQ.ID10, SEQ.ID11, SEQ.ID12, SEQ.ID13, SEQ.ID16, SEQ.ID17,<br>
SEQ.ID18, SEQ.ID19, SEQ.ID20, SEQ.ID21 and SEQ.ID22.<br>
Yet another subject of the present invention is an expression<br>
cassette comprising at least one promoter nucleic acid sequence<br>
which is derived from a gene encoding a high molecular weight<br>
wheat glutenin, and which is linked in a functional manner to a<br>
nucleic acid sequence to be expressed encoding a polypeptide to<br>
be produced, itself linked to a transcription termination<br>
nucleic acid sequence.<br>
Preferably, this expression cassette comprises at least one<br>
promoter nucleic acid sequence, derived from a gene encoding a<br>
high molecular weight wheat glutenin, the sequence of which is<br>
identified under the number SEQ.ID01.<br>
Even more preferably, the expression cassette comprises at least<br>
one promoter nucleic acid sequence which is derived from a gene<br>
encoding a high molecular weight wheat glutenin, and which is<br>
chosen from the group consisting of the sequences identified<br>
under the numbers SEQ.ID02, SEQ.ID03, SEQ.ID04, SEQ.ID05,<br>
SEQ.ID06, SEQ.ID07, SEQ.ID08, SEQ.ID09, SEQ.ID10, SEQ.ID11,<br>
SEQ.ID12, SEQ.ID13, SEQ.ID16, SEQ.ID17, SEQ.ID18, SEQ.ID19,<br>
SEQ.ID20, SEQ.ID21 and SEQ.ID22.<br>
Another subject of the present invention is an isolated promoter<br>
nucleic acid sequence, characterized in that it corresponds to a<br>
sequence derived from the sequence identified under the number<br>
SEQ.ID01.<br>
Preferably, the isolated promoter nucleic acid sequence<br>
corresponds to a sequence chosen from the group consisting of<br>
the sequences identified under the numbers SEQ.ID02,SEQ.ID03,<br>
SEQ.ID04, SEQ.ID05, SEQ.ID06, SEQ.ID07, SEQ.ID08, SEQ.ID09,<br>
SEQ.ID10, SEQ.ID11, SEQ.ID12, SEQ.ID13, SEQ.ID16, SEQ.ID17,<br>
SEQ.ID18, SEQ.ID19, SEQ.ID20, SEQ.ID21 and SEQ.ID22.<br>
Another subject of the present invention is a vector comprising<br>
a promoter, or a promoter nucleic acid sequence, which is<br>
capable of initiating the transcription of a nucleic acid<br>
sequence encoding a polypeptide to be produced, characterized in<br>
that the promoter or the promoter nucleic acid sequence<br>
corresponds to a chimeric promoter or to a promoter nucleic acid<br>
sequence as defined above.<br>
Preferably, the vector is chosen from the group consisting of<br>
the binary vectors identified under the numbers pMRT1207,<br>
pMRT1177, pMRT1178, pMRT1179, pMRTUSO and pMRTHSl.<br>
Yet another subject of the present invention is a transgenic<br>
plant which has stabily integrated into its genome at least one<br>
promoter or at least one promoter nucleic acid sequence as<br>
defined above.<br>
Preferably, the transgenic plant is chosen from dicotyledonous<br>
species, such as potato, tobacco, cotton, lettuce, tomato,<br>
melon, cucumber, pea, rapeseed, beetroot or sunflower, or<br>
monocotyledonous species, such as wheat, barley, oat, rice or<br>
maize.<br>
Another subject of the present invention is a propagule of a<br>
transgenic plant as defined above, and preferably, this<br>
propagule is a seed.<br>
Yet another subject of the present invention is a cell<br>
containing a promoter or a promoter nucleic acid sequence as<br>
defined above, and preferably, this cell is a plant cell.<br>
Among the preferred subjects of the invention, there is also a<br>
method for expressing a nucleic acid sequence, or gene, encoding<br>
a polypeptide to be produced, in a cell, characterized in that<br>
it comprises the steps consisting in:<br>
-	transforming the cell with a vector comprising at least one<br>
promoter or at least one promoter nucleic acid sequence as<br>
defined above;<br>
-	preparing a culture of the cell under conditions which allow<br>
the expression of the nucleic acid sequence, or gene, encoding<br>
the polypeptide.<br>
Preferably, the polypeptide to be produced is an enzyme or protein, or derivative of the latter, which has activity in vitro and/or in humans and/or in animals, said activity comprising digestive, pancreatic, biliary, antiviral, anti-inflammatory, pulmonary, antimicrobial, nutritive, cosmetic and structural activity, activity in the blood, cardiovascular, ophthalmic, antigenic and immunostimulating activity, and activity in the brain. Examples of such proteins are, for example, insulins, interferons, gastric, pancreatic or biliary lipases, elastases, antiproteases such as alpha-1 antitrypsin,<br><br>
structure-forming proteins such as collagen, transferrins such<br>
as lactoferrin, blood-derived proteins, such as human<br>
haemoglobin or albumin, and blood cofactors, and antioxidants<br>
such as superoxide dismutase.<br>
Preferably, the cell used in this method is a procaryotic or<br>
eucaryotic cell.<br>
Evea more preferably, the cell is a cell chosen from the group<br>
consisting of microbial cells, fungal cells, insect cells,<br>
animal cells and plant cells, and even more preferably it is a<br>
plant cell.<br>
Finally, another subject of the present invention is a method<br>
for obtaining a transgenic plant or a propagule as defined<br>
above, characterized in that it comprises the steps consisting<br>
in:<br>
-	transforming a plant cell with a vector comprising at least<br>
one promoter or at least one promoter nucleic acid sequence as<br>
defined above;<br>
-	selecting the plant cell which has integrated the promoter or<br>
promoter nucleic acid sequence;<br>
-	propagating the transformed and selected plant cell either in<br>
culture or by regenerating chimeric or transgenic whole plants.<br>
According to the present invention there is provided a process for construction of a chimeric promoter of expression, said process comprising a series of deletions or insertions or duplications of at least one nucleic acid sequence derived from a gene encoding a high molecular weight wheat glutenin, wherein said nucleic acid sequences are derived from a gene encoding a high molecular weight wheat glutenin and preferably inserting regulatory or functional boxes of the kind such as herein described; and performing the conventional steps of amplification and ligation to prepare the chimeric promoter.<br>
DESCRIBED OF THE FIGURES;<br>
The invention will be more clearly understood through the detailed description of the various embodiments given hereafter by way of nonlimiting examples, and with reference to the attached drawings, in which:<br>
Figures 1, 2, 3 and 4 represent, schematically, synthetic and chimeric promoter constructs in accordance with the invention.<br>
In figure 1, they are constructs comprising a series of deletions starting from the whole promoter originating from the Dx5 gene which is from wheat and which encodes a high molecular weight wheat glutenin, and a series of constructs comprising repeats of elements which comprise an "enhancer" box combined<br>
with a "G-like" box;<br>
-	Figure 2 represents, schematically, constructs of other<br>
synthetic and chimeric promoters in accordance with the<br>
invention, containing insertions of regulatory and/or functional<br>
elements which comprise combined "as2/asl" boxes;<br>
-	Figure 3 represents, schematically, constructs of other<br>
synthetic and chimeric promoters in accordance with the<br>
invention, containing insertions of regulatory and/or functional<br>
elements which comprise combined "as2/as2/asl" boxes;<br>
-	Figure 4 represents, schematically, constructs of other<br>
synthetic and chimeric promoters in accordance with the<br>
invention, containing insertions of regulatory and/or functional<br>
elements which comprise combined "cereal" boxes, originating<br>
from the Bx7 gene encoding a high molecular weight wheat<br>
glutenin, alone or in combination with "as2/asl" boxes, and a<br>
construction comprising a "GC-rich" box;<br>
-	Figure 5 represents photographs of tobacco leaves which have<br>
been transformed with vectors containing the promoters or<br>
[lacuna] nucleic acid sequences described above functionally linked to the gene encoding GUS (beta-glucuronidase), and after histochemical revelation, the blue dots indicating the presence of cells transformed with said constructs, and thus the activity of the promoter;<br>
-	Figures 6 and 7 represent graphs comparing the promoter<br>
activity relating to the various constructs after transient<br>
expression in maize albumens. Three days after bombarding, the<br>
leaves are ground, and then the crude extract is clarified by<br>
centrifugation. The b-glucuronidase activity and the luciferase<br>
activity are measured by fluorimetry on an aliquot of the crude<br>
extract, and then the GUS activity/LUC activity ratio is<br>
determined. The histograms correspond to the mean of ratios for<br>
the same construct +/- Standard Error of the Mean (SEM).<br>
-	Figure 8 represents a graph comparing the promoter activity of<br>
MPrll39, MPrl200 and MPrll31 to that of the 512 gamma-zein<br>
promoter in stable expression in maize albumen, 30 days after<br>
»<br>
pollinisation (30 DAP) . The p-glucuronidase activity and the total quantity of proteins were determined respectively by luminometry and spectrometry . The histograms correspond to the average ratios of GUS activity / total proteins measured seed by seed for each plant, + /- standard mean error. The name of each transformant is indicated.<br>
Figure 9 represents the temporal (3-glucuronidase activity controlled by the promoter MPrll39 in stable expression in maize albumen. The p-glucuronidase activity and the quantity of total proteins were determined by luminometry and spectrometry. The histograms correspond to the average ratiosn of GUS activity / total proteins measured seed per seed for the plant 151. Cl at different stages of development, + /- standard mean error.<br>
-	Figure lOa represents a longitudinal section of a maize seed<br>
at 13 DAP, figure lOb represents a longitudinal section of a<br>
maize seed at 20 DAP, and figure lOc is a top plan view of a<br>
dissected maize seed. All reveal the {5-glucuronidase activity<br>
under the control of the promoter MPrll39 in stable expression<br>
in maize  seeds,  visualized by histochemical  staining  (blue<br>
color) , where the letters indicate the following : E (embryo) ;<br>
Em (endosperm) ; AC (aleurone cells) ; P (pericarp)<br>
-	Figure 11 represents a graph comparing the promoter activity<br>
of MPrll39 in first generation maize seeds  (Tl)  to that of<br>
second generation transgenic maize seeds  (T2),  18 days after<br>
pollinisation  (18 DAP) . The p-glucuronidase activity and the<br>
quantity of total proteins were measured by luminometry and<br>
spectrometry respectively.  The  histograms  correspond  to  the<br>
average ratios of GUS activity / total proteins measured seed<br>
per seed for each plant, +/- standard mean error. The name of<br>
each transformant is indicated.<br>
-	Figure 12 represents a graph comparing the promoter activity<br>
of MPrll39, MPrl200 and MPrll31 to that of the 512 gamma-zein<br>
promoter in stable expression of maize leaves, 3 weeks after<br><br>
acclimatization in a greenhouse. The p-glucuronidase activity and the quantity of total proteins were measured by luminometry and spectrometry respectively. The histograms correspond to the ratios of GUS activity / total proteins measured in the leaves of different maize leaves. The name of each transformant is indicated.<br>
-	Figure 13 represents a graph comparing the promoter activity<br>
of MPrllSO, MPrll31, MPrll35, MPrll38 and MPrll39 to that of the<br>
reference promoters CaMV D35S and HMWG-Dx5 in stable expression<br>
in tobacco leaves, at the 11 week stage of development after<br>
acclimatization in a greenhouse. The (3-glucuronidase activity and<br>
the quantity of total proteins were measured by luminometry and<br>
spectrometry respectively.  The histograms correspond to the<br>
ratios of GUS activity / total proteins measured in the leaves<br>
of different tobacco plants.<br>
-	Figure 14 represents a graph comparing the promoter activity<br>
promotrice of MPrll30, MPrll31, MPrll35, MPrll38 and MPrll39 to<br>
that of the reference promoter CaMV D35S, in stable expression<br>
in mature first generation tobacco seeds. The |3-glucuronidase<br>
activity and the quantity of total proteins were measured by<br>
luminometry  and  spectrometry  respectively.  The  histograms<br>
correspond to the ratios of GUS activity / total proteins<br>
measured in the seeds of different tobacco plants.<br>
In the detailed description which follows, the enzymatic treatments performed with the restriction enzymes and the DNA modification enzymes were carried out according to the recommendations of the supplier New England Biolabs. Following each enzymatic treatment, the DNA was systematically purified with the aid of the "QIAquick PCR Purification" (QIAGEN) or "Concert Rapid PCR Purification System" (GIBCO BRL Life Technologies), or, if specified, with the aid of the "QIAquick Gel Extraction" (QIAGEN) or "Concert Rapid Gel Extraction System" (GIBCO BRL Life Technologies) kits according to the manufacturer's instructions. The "GeneAmp PCR System 9700"<br><br>
thermocycler used is sold by Perkin Elmer Applied Biosystems. EXAMPLES Example 1 Constructs for comparative purpose (controls).<br>
In order to enable the comparison of the chimeric promoters described in this patent, the uidA gene encoding b-glucuronidase (Jefferson et al./ 1986) containing the sequence of intron IV2 of the potato patatin gene ST-LS1 (Vancanneyt et al., 1990) (uidA-IV2) was placed under the control of one of the reference promoters and of the nopaline synthase gene terminator (term-nos) of Agrobacterium tumefaciens, in the plasmid pGEM3Z sold by Promega Corp. (Madison, USA).<br>
1.1. Construction of the negative control pMRT1144.<br>
The plasmid pMRT1144, devoid of any promoter sequence, is used as a negative control. It is derived from the plasmid pGEM3Z into which the sequence "uidA-IV2/term-nos" has been introduced.<br>
Firstly, 5 ug of the plasmid pBI221 (Clonetech, CA, USA) were digested for 1 h at 37°C with the restriction enzymes EcoRI and BamHI. The uidA sequence under the control of the nopaline synthase terminator was then isolated on a 0.8% agarose gel with the aid of the "QIAquick Gel Extraction" kit.<br>
In parallel, 5 ug of plasmid pGEMSZ were digested with the restriction enzyme pair EcoRI and BamHI for 1 h at 37°C. The vector fragment was then isolated on a 0.8% agarose gel with the aid of the "QIAquick Gel Extraction" kit, and dephosphorylated with 40 U of calf intestine alkaline phosphatase (New England Biolabs) in the presence of buffer 3 (IX) at 37°C for 1 h.<br>
The ligation reaction was carried out with 50 ng of the "uidA-IV2/term-nos" fragment and 100 ng of plasmid pGem3Z, thus treated, in a 10 ul reaction mixture, in the presence of T4 DNA ligase buffer (IX) and of 400 units of T4 DNA ligase (New England Biolabs) in the "GeneAmp PCR System 9700" thermocycler. It consists of one cycle at 10°C for 30 sec. and of 200<br><br>
identical cycles each consisting of the following steps: 30 sec. at 30°C, 30 sec. at 10°C and 30 sec. at 30°C. Escherichia coli DH5a bacteria, which had been made competent beforehand, were transformed with all of the ligation reaction mixture. The plasmid DNA of the clones obtained, which were selected on Luria-Bertani medium (LB, 10 g/1 bactotryptone, 5 g/1 yeast extract, 10 g/1 NaCl, pH 7.2 and 15 g/1 Agar-Agar) supplemented with ampicillin (50 mg/1), was extracted according to the alkaline lysis method (Birnboim and Doly, 1983) and analysed with enzymatic digestions. The resulting plasmid was called pGEMSZ/uidA/term-nos.<br>
Secondly, in order to insert the 192-bp intron IV2 of the potato patatin gene into the uidA coding sequence of pGEM3Z/uidA/term-nos, an internal portion of this gene (710-bp SnaBI/BstBI fragment) was excised and then replaced with the equivalent sequence containing intron IV2 (902-bp SnaBI/BstBI fragment).<br>
In order to do this, 10 ug of the plasmid<br>
pGEM3Z/uidA/term-nos were digested for 1 h at 37°C with SnaBI (restriction site located at position +383 bp downstream of the ATG start codon of the uidA gene), and then for 1 h at 65°C with BstBI (site located at position +1093 bp). The plasmid thus deleted of the 710-bp fragment was isolated on a 0.8% agarose gel with the aid of the "QIAquick Gel Extraction" kit, and dephosphorylated with 40 U of calf intestine alkaline phosphatase (New England Biolabs) in the presence of buffer 3 (IX) at 37°C for 1 h.<br>
The 902-bp BstBI/SnaBI fragment corresponding to the sequence of intron IV2 followed by the uidA sequence was obtained by digesting 10 ug of the plasmid p35S GUS INT (Vancanneyt et al., 1990) with the restriction enzyme SnaBI (restriction site located at position +383 bp downstream of the ATG start codon of the uidA gene) for 1 h at 37°C, and restriction enzyme BstBI (site located at position +1285 bp) for<br><br>
1 h at 37°C. The 902-bp fragment was then isolated on a 1% agarose gel with the aid of the "Concert Rapid Gel Extraction System" kit.<br>
The ligation reaction was carried out with 100 ng of vector pGEM3Z/uidA/term-nos and 50 ng of the 902-bp BstBI/SnaBI fragment thus treated, in a 10 ul reaction mixture, in the presence of the T4 DNA ligase buffer (IX) and of 400 units of T4 DNA ligase (New England Biolabs) in the "GeneAmp PCR System 9700" thermocycler as described above. Escherichia coli DH5a bacteria, which had been made competent beforehand, were transformed with all of the ligation reaction mixture. The plasmid DNA of the clones obtained, which were selected on LB medium supplemented with ampicillin (50 mg/1), was extracted according to the alkaline lysis method and analysed with enzymatic digestions. The plasmid obtained was called pMRT1144.<br>
1.2. Construction of the positive control pMRT1218.<br>
In order to have a reference sequence which is a promoter in the maize albumen SN 87 165 (L2), the 1.7-kb whole g-zein promoter (Prg-zein) contained in the plasmid p63 described by Reina et al. (1990) was placed upstream of the sequence uidA-IV2/term-nos.<br>
The 1.7-kb g-zein promoter was obtained by digesting 15 ug of plasmid p63 with the restriction enzymes HindiII and BamHI for 1 h at 37°C. The 1.7-kb Prg-zein fragment thus released was isolated on a 0.8% agarose gel with the aid of the "Concert Rapid Gel Extraction System" kit.<br>
In parallel, 10 ug of plasmid pMRT1126 (described in section 3.4 of Example 3) were also digested with the restriction enzymes Hindlll and BamHI for 1 h at 37°C. The vector fragment was then isolated on a 0.8% agarose gel with the aid of the "Concert Rapid Gel Extraction System" kit, and dephosphorylated with 40 U of calf intestine alkaline phosphatase (New England Biolabs) in the presence of buffer 3 (IX) at 37°C for 1 h.<br><br>
The ligation reaction was carried out with 50 ng of the g-zein promoter fragment and 100 ng of plasmid pMRT1126, thus treated, in a 10 ul reaction mixture, in the presence of the T4 DNA ligase buffer (IX) and of 400 units of T4 DNA ligase (New England Biolabs), in the "GeneAmp PCR System 9700" thermocycler as described above. Escherichia coli DH5a bacteria, which had been made competent beforehand, were transformed with all of the ligation reaction mixture. The plasmid DNA of the clones obtained, which were selected on LB medium supplemented with ampicillin (50 mg/1), was extracted according to the alkaline lysis method and analysed with enzymatic digestions. The resulting plasmid was called pMRT1218.<br>
1.3. Construction of the positive control pMRT1092.<br>
In order to have a reference sequence which is a promoter in the photosynthetic tissues of tobacco (Nicotiana tabacum L., cultivar PBD6), the double 35S promoter of the cauliflower mosaic virus (CaMV PrD35S) was placed upstream of the sequence uidA-IV2/term-nos.<br>
Firstly, the 192-bp intron IV2 of the potato patatin gene was inserted into the uidA coding sequence at position +383 bp as described in section 1.1. In order to do this, 1 ug of plasmid pBI221 (Clontech, CA, USA) was digested for 1 h 30 min. at 37°C with SnaBI, and then for 1 h 30 min. at 65°C with BstBI. The plasmid deleted of a 710-bp fragment was isolated on a 0.8% agarose gel, and then purified on a Qiaquick affinity column. A 20 ng amount of BstBI/SnaBI pBI221 vector and 80 ng of the 902-bp BstBI/SnaBI fragment originating from p35S GUS INT as described above were ligated overnight at 18°C in a 10 ul reaction mixture, in the presence of the T4 DNA ligase buffer (IX) and of 400 units of T4 DNA ligase enzyme (New England Biolabs). Escherichia coli DH5a bacteria, which had been made competent beforehand, were transformed with half of the ligation reaction mixture. The DNA of the clones obtained, which were selected on LB medium supplemented with ampicillin (50 mg/1),<br><br>
was extracted according to the alkaline lysis method and c<br>
analysed with enzymatic digestions. The plasmid obtained was called pBI221/uidA-IV2.<br>
Secondly, the sequence of the CaMV 35S promoter present in the plasmid pBI221/uidA-IV2 was replaced with the sequence of "CaMV PrD35S". The plasmid pBI221/uidA-IV2 was digested for 10 h 30 min. at 37°C with Hindlll, and then the sticky ends were made blunt-ended using the Klenow fragment (New England Biolabs) for 30 min at 37°C. The product of this reaction was then digested overnight at 37°C with BamHI. The plasmid DNA fragment, corresponding to the vector deleted of the 828-bp CaMV 35S promoter fragment, was isolated on a 0.8% agarose gel, and then purified on a Qiaquick affinity column.<br>
In parallel, the CaMV D35S promoter was obtained from the plasmid pJIT163D. This plasmid is derived from the plasmid pJIT163, which is itself derived from the plasmid pJIT160 (Guerineau and Mullineaux, 1993) . The plasmid pJIT163 has an ATG codon between the HindiII and Sail sites of the polylinker. In order to delete this ATG and to obtain the plasmid pJIT163D, the pJIT163 plasmid DNA was digested with Hindlll and Sail, purified on a 0.8% agarose gel, electroeluted, precipitated in the presence of a 1/10 volume of 3M sodium acetate, pH 4.8, and of 2.5 volumes of absolute ethanol at -80°C for 30 min, centrifuged at 12,000 g for 30 min, washed with 70% ethanol, dried, subjected to the action of the Klenow fragment (New England Biolabs) for 30 min at 37°C, deproteinized by extraction with one volume of phenol, then one volume of<br>
phenol/chloroform/isoamyl alcohol (25/24/1 v/v/v) and finally one volume of chloroform/isoamyl alcohol (24/1 v/v) , precipitated in the presence of a 1/10 volume of 3M sodium acetate, pH 4.8, and of 2.5 volumes of absolute ethanol at -80°C for 30 min, then centrifuged at 12,000 g for 30 min, washed with 70% ethanol, dried and finally ligated in the presence of the T4 DNA ligase buffer (IX) and 2.5 units of T4 DNA ligase (Amersham)<br><br>
at 14°C for 16 h. Escherichia coli DH5a bacteria, which had been<br>
V<br>
made competent beforehand, were transformed. The plasmid DNA of the clones obtained, which were selected on LB medium supplemented with ampicillin (50 mg/1), was extracted according to the alkaline lysis method and analysed with enzymatic digestions. Next, ten ug of plasmid pJIT163D were digested for 10h30 at 37°C with Kpnl (site located 5' of the promoter), and then the sticky ends were made blunt-ended using 6 units of T4 DNA polymerase (New England Biolabs) for 30 min at 37°C. The product of this reaction was then digested overnight at 37°C with BamHI. The resulting 761-bp DNA fragment, corresponding to the CaMV D35S promoter, was isolated on a 1% agarose gel, and then purified on a Qiaquick affinity column. The ligation was carried out with 10 ng of plasmid vector and 100 ng of the 761-bp fragment, in a 10 ul reaction mixture, in the presence of the T4 DNA ligase buffer (IX) and 400 units of T4 DNA ligase (New England Biolabs) overnight at 18°C. Escherichia coli DH5a bacteria, which had been made competent beforehand, were transformed with half of the ligation reaction mixture. The plasmid DNA of the clones obtained, which were selected on LB medium supplemented with ampicillin (50 mg/1), was extracted according to the alkaline lysis method and analysed with enzymatic digestions. The plasmid obtained was called pMRT1092.<br>
1.3. Description of the reference plasmid pCaMV35Sluc.<br>
The plasmid used as an internal reference in the transient expression is pCaMV35Sluc (Torrent et al., 1997), which contains the cassette for expression of the luciferase (luc) reporter gene under the control of the CaMV 35S promoter and RNA terminator.<br>
Example 2.<br>
Construction of plasmids containing the whole promoter sequence and deleted or duplicated promoter sequences of a high molecular weight wheat glutenin gene.<br>
The whole promoter (PrHMWG-Dx5 (SEQ.ID01)) of the high<br><br>
nolecular weight glutenin gene encoding the Dx5 subunit, also called GluDl-lb, of the hexaploid wheat Triticum aestivum L. cv Cheyenne (Anderson et al., 1989) corresponds to a 417-bp sequence (accession No. X12928) ranging from position -378 bp to position +39 bp, on which diverse potentially regulatory sequences are identified and listed on the 5' side towards the 3' side, with respect to the +1 transcription start point (Fig. I) :<br>
-	a prolamine-"like" box, stretching from position -357 to<br>
position -350 bp,<br>
-	two GATA boxes, stretching from position -309 to<br>
position -306 bp, and position -292 to position -289 bp,<br>
-	a prolamine-"like" box, stretching from position -252 to<br>
position -246 bp,<br>
-	an 8-bp "G"-like box, stretching from position -218 to<br>
position -211 bp,<br>
-	a 38-bp activating element, stretching from position -<br>
186 to position -149 bp, composed of a mosaic of putative de<br>
activating motifs:<br>
. a prolamine box, stretching from position -182 to position -176 bp,<br>
. a sequence with imperfect symmetry, stretching from position -178 to position -161 bp,<br>
. a direct repeat of the pentanucleotide GCTCC between positions -176 and -163 bp,<br>
. an "E" box, stretching from position -172 to position -167 [lacuna],<br>
. a direct repeat of the pentanucleotide TTGCT between positions -169 and -158 bp,<br>
-	a "TATA" box, having the consensus TATAAAA from position<br>
-30 to -24 [lacuna],<br>
-	the +1 transcription start point (position 1),<br>
-	an untranslated 5' region ranging from position +1 to<br>
position +39 bp.<br><br>
In order to study the effect of the various putative cis-activating elements described above, a detailed functional analysis of the HMWG-Dx5 promoter (SEQ.IDOl) was carried out. The uidA-IV2 reporter gene was placed under the control of the whole HMWG-Dx5 promoter (SEQ.IDOl) and under the control of the synthetic HMWG-Dx5 (SEQ.IDOl) promoters having either increasing deletions of the 5' regions, or an internal deletion, or duplications of an internal portion. 2.1. Construction of the plasmid pMRT1125.<br>
The plasmid pMRT1125 is the result of cloning the whole promoter of the high molecular weight glutenin gene (PrHMWG-Dx5 (SEQ.IDOl), Fig. I) upstream of the uidA-IV2 reporter gene, and constitutes the reference construct for all of the synthetic HMWG-Dx5 (SEQ.IDOl) promoters described in this patent. The HMWG-Dx5 promoter (SEQ.IDOl) described by Anderson et al. (1989) was obtained from the expression cassette "PrHMWG-Dx5 (SEQ.IDOl)/uidA/term-nos" introduced into a pUC19 plasmid (Stratagene) according to the usual cloning techniques. Ten ug of the resulting plasmid (pPUC19-HMWG) were hydrolysed with EcoRI for 1 h at 37°C, and subjected to the action of 20 U of the Klenow fragment (New England Biolabs) for 30 min at 37°C in the presence of 60 nmol of each of the dNTPs, of 12 ul of 500 mM Tris-HCL, pH 7.5/500 mM MgC12 buffer and 6 ul of 1 M dithiothreitol (DTT). The DMA was then digested with BamHI for 1 h at 37 °C, and the HMWG-Dx5 promoter fragment (SEQ.IDOl) thus released and treated was isolated on a 1.5% agarose gel with the aid of the "QIAquick Gel Extraction" kit.<br>
In parallel, the vector fragment was prepared from the plasmid pMRT1097 (unpublished French patent application FR 9903635). Twenty ug of plasmid pMRT1097 were digested for 1 h at 37°C with SphI, and the sticky ends of the vector pMRT1097 thus linearized were made blunt-ended using 6 U of the T4 DNA polymerase enzyme (New England Biolabs) for 30 min. at 37°C. The product of this reaction was then hydrolysed with BamHI, and the vector fragment<br><br>
was isolated on a 0.8% agarose gel with the aid of the "QIAquick Gel Extraction" kit, before being dephosphorylated with 40 U of calf intestine alkaline phosphatase (New England Biolabs) in the presence of buffer 3 (IX) at 37°C for 1 h. The resulting cloning vector was called pGEM3Z-l.<br>
The ligation was carried out with 100 ng of the HMWG-Dx5 promoter fragment (SEQ.ID01) thus treated and 50 ng of plasmid pGem3Z-l overnight at 16°C in a 10 ul reaction mixture, in the presence of the T4 DNA ligase buffer (IX) and of 400 units of T4 DNA ligase (New England Biolabs). Escherichia coli DH5a bacteria, which had been made competent beforehand, were transformed with all of the ligation reaction mixture. The plasmid DNA of the clones obtained, which were selected on LB medium supplemented with ampicillin (50 mg/1), was extracted according to the alkaline lysis method and analysed with enzymatic digestions.<br>
The plasmid obtained was called pMRT1125, and the HMWG-Dx5 promoter (SEQ.ID01) (shown diagrammatically in Fig. 1 was verified by sequencing.<br>
2.2. Construction of the MPrll28 promoter.<br>
The MPR1128 promoter (SEQ.ID04) is derived from PrHMWG-Dx5 (SEQ.ID01) by deleting the sequence located upstream of nucleotide -238, this sequence comprising the two prolamine-"like" boxes and the two GATA boxes. The promoter fragment was amplified by PCR from 5 ng of pMRT1125 matrix DNA (described in section 2.1 of Example 2) with the aid of 100 pmol of each of the 2 oligodeoxynucleotides 5' ATCGGAATTCCAGAACTAGGATTACGCCG 3', containing the EcoRI restriction site, and<br>
5' TACggATCCCCggggATCTCTAgTTTgTggTgC 3', possessing the BamHI restriction site, in the presence of 50 nmol of each of the dNTPs, of the Vent DNA polymerase buffer (IX) and of 2 U of Vent DNA polymerase (New England Biolabs). The PCR amplification reaction was carried out in the "GeneAmp PCR System 9700" thermocycler. After a denaturation at 94°C for 5 min., the DNA<br><br>
was subjected to 30 cycles each consisting of the steps of denaturation at 94°C for 1 min., of hybridization at 50°C for 1 min. and elongation at 12°C for 1 min. 30 sec. During the final cycle, the elongation was continued at 72°C for 5 min. The DNA fragment derived from the amplification was isolated on a 1.5% agarose gel with the aid of the "QIAquick Gel Extraction" kit, hydrolysed with EcoRI for 1 h at 37°C and subjected to the action of 20 U of the Klenow fragment (New England Biolabs) for 30 min at 37°C in the presence of 60 nmol of each of the dNTPs, of 12 ul of 500 mM Tris-HCL, pH 7.5/500 mM MgC12 buffer and 6 ul of 1 M DTT. The DNA thus treated was then digested with BamHI for 1 h at 37°C.<br>
The ligation was carried out with 100 ng of the MPR1128 promoter fragment (SEQ.ID04) thus treated and 50 ng of plasmid pGEM3Z-l (described in section 2.1 of Example 2) overnight at 16°C in a 10 ul reaction mixture, in the presence of the T4 DNA ligase buffer (IX) and of 400 units of T4 DNA ligase (New England Biolabs). Escherichia coli DH5a bacteria, which had been made competent beforehand, were transformed with all of the ligation reaction mixture. The plasmid DNA of the clones obtained, which were selected on LB medium supplemented with ampicillin (50 mg/1), was extracted according to the alkaline lysis method and analysed with enzymatic digestions.<br>
The plasmid obtained was called pMRT1128, and the MPR1128 promoter sequence (SEQ.ID04) represented diagrammatically in Fig. I was verified by sequencing.<br>
2.3. Construction of the MPrll27 promoter (SEQ.ID03). The MPrll27 promoter (SEQ.ID03) is derived from the HMWG-Dx5 promoter (SEQ.ID01) by deleting the sequence located upstream of nucleotide -205, this sequence comprising the two prolamine-"like" boxes, the two GATA boxes and the "G" box. The promoter fragment was amplified by PCR and treated in the same way as the MPR1128 promoter (SEQ.ID04)(described in section 2.2 of Example 2), except that the 2 oligodeoxynucleotides used are<br><br>
'-)' ATCGGGAATTCGCAGACTGTCCAAAAATC 3', containing the EcoRI restriction site, and 5' TACggATCCCCggggATCTCTAgTTTgTggTgC 3', possessing the BamHI restriction site.<br>
The plasmid obtained was called pMRTl!27, and the MPrll27 promoter sequence (SEQ.ID03) represented diagrammatically in Fig. I was verified by sequencing.<br>
2.4.	Construction of the MPrll26 promoter (SEQ.ID02).<br>
The MPrll26 promoter (SEQ.ID02) is derived from the HMWG-Dx5 promoter (SEQ.ID01) by deleting the sequence located upstream of nucleotide -142, this sequence comprising the two prolamine-"like" boxes, the two GATA boxes, the "G" box and the activating element. The promoter fragment was amplified by PCR and treated in the same way as the MPR1128 promoter (SEQ.ID04)(described in section 2.2 of Example 2), except that the 2<br>
oligodeoxynucleotides used are 5' ATCGGAATTCGTGTTGGCAAACTGC 3', containing the EcoRI restriction site, and 5' TACggATCCCCgggg-ATCTCTAgTTTgTggTgC 3', possessing the BamHI restriction site. The plasmid obtained was called pMRT1126, and the MPrll26 promoter sequence (SEQ.ID02) represented diagrammatically in Fig. I was verified by sequencing.<br>
2.5.	Construction of the MPrll83 intermediate promoter.<br>
The MPrll83 promoter results ffom the insertion of an Xbal<br>
restriction site upstream of the MPR1128 promoter (SEQ.ID04)<br>
(described in section 2.2 of Example 2). The promoter fragment<br>
was amplified by PCR from 5 ng of pMRT1128 matrix DNA with the aid of 100 pmol of each of the 2 oligodeoxynucleotides 5' ATCggAATTCTAgACgCCgATTACgTggCTTTAgC 3', containing the EcoRI and Xbal restriction sites, and 5' TACggATCCCCggggATCTCTA-gTTTgTggTgC 3', possessing the BamHI restriction site, in the presence of 50 nmol of each of the dNTPs, of Vent DNA polymerase buffer (IX) and of 2 U of Vent DNA polymerase (New England Biolabs). The PCR amplification reaction was carried out in the "GeneAmp PCR System 9700" thermocycler. After a denaturation at 94°C for 5 min., the DNA was subjected to 30 cycles each<br>
consisting of the steps of denaturation at 94°C for 1 min., of * hybridization at 50°C for 1 min. and of elongation at 72°C for 1<br>
min. 30 sec. During the final cycle, the elongation was continued at 72°C for 5 min.<br>
The DNA fragment derived from the amplification was isolated on a 1.5% agarose gel with the aid of the "Concert Rapid Gel Extraction System" kit, hydrolysed with EcoRI for 1 h at 37°C and then subjected to the action of 20 U of the Klenow fragment (New England Biolabs) for 30 min at 37°C in the presence of 60 nmol of each of the dNTPs, of 12 ul of 500 mM Tris-HCL, pH 7.5/500 mM MgCl2 buffer and 6 ul of 1 M DTT, and digested with BamHI for 1 h at 37°C.<br>
The ligation was carried out with 100 ng of the MPR1128 promoter fragment thus treated and 50 ng of plasmid pGem3Z-l (described in section 2.1 of Example 2) overnight at 16°C in a 10 pi reaction mixture, in the presence of the T4 DNA ligase buffer (IX) and of 400 units of T4 DNA ligase (New England Biolabs). Escherichia coli DH5a bacteria, which had been made competent beforehand, were transformed with all of the ligation reaction mixture. The plasmid DNA of the clones obtained, which were selected on LB medium supplemented with ampicillin (50 mg/1), was analysed by PCR with the aid of 25 pmol of each of the 2 oligodeoxynucleotides 5' ATCggAATTCgCAgCCATggTCCTgAACC 3' and 5' TACggATCCCCggggATCTCTAgTTTgTggTgC 3', in the presence of 15 nmol of each of the dNTPs, of the Taq DNA polymerase buffer (IX), of 75 nmol of MgC12 and of 1.25 U of Taq DNA polymerase (Promega Corp.) in a 50 ul reaction volume. The amplification reaction was carried out in the "GeneAmp PCR System 9700" thermocycler. After a denaturation at 94°C for 3 min., the DNA was subjected to 25 cycles each consisting of the steps of denaturation at 94°C for 30 sec., of hybridization at 55°C for 1 min. and of elongation at 72°C for 1 min. During the final cycle, the elongation was continued at 72°C for 5 min. The plasmid obtained was called pMRT1183, and the MPrll83<br><br>
promoter sequence was verified by sequencing.<br>
2.6.	Construction of the MPrll97 promoter (SEQ.ID16).<br>
The MPrll97 promoter (SEQ.ID16) is derived from MPrll83 (described in section 2.5 of Example 2) by a deletion of the promoter sequence located upstream of nucleotide -57 bp, and constitutes the minimum HMWG-Dx5 (SEQ.ID01) promoter studied in this patent.<br>
In order to do this, 5 ug of plasmid pMRT1183 were digested successively for 1 h at 37°C with Xbal and Ncol. The vector pMRT1183 thus deleted of the Xbal/Ncol fragment of the MPrllSS promoter was isolated on a 0.8% agarose gel with the aid of the "Concert Rapid Gel Extraction System" kit, and subjected to the action of 20 U of the Klenow fragment (New England Biolabs) for 30 min at 37°C in the presence of 60 rimol of each of the dNTPs, of 12 ul of 550 mM Tris-HCL, pH 7.5/500 mM MgCl2 buffer and 6 ul of 1 M DTT.<br>
The ligation reaction was carried out with 150 ng of plasmid thus modified, in a 10 ul reaction mixture, in the presence of the T4 DNA ligase buffer (IX) and of 400 units of T4 DNA ligase (New England Biolabs) in the "GeneAmp PCR System 9700" thermocycler, as described above. Escherichia coli DH5a bacteria, which had been made competent beforehand, were transformed with all of the ligation reaction mixture. The plasmid DNA of the clones obtained, which were selected on LB medium supplemented with ampicillin (50 mg/1), was extracted according to the alkaline lysis method and verified with enzymatic digestions.<br>
The plasmid obtained was called pMRT1197, and the MPrll97 promoter sequence (SEQ.ID16) is represented diagrammatically in Fig. I.<br>
2.7.	Construction of the MPrll98 promoter (SEQ.ID17).<br>
The MPrll98 promoter (SEQ.ID17) is derived from the MPR1128 promoter (SEQ.ID04) (described in section 2.2 of Example 2) by deleting the internal promoter sequence stretching from position<br><br>
-59 to position -135 bp, this sequence lacking the cis-<br>
f<br>
activating elements identified above.<br>
It was constructed by fusing, at the Ncol restriction site of pMRT1183 (described in section 2.5 of Example 2), a fragment amplified by PCR from 5 ng of pMRT1128 matrix DNA with the aid of 100 pmol each of the 2 oligodeoxynucleotides<br>
5' ATCggAATTCTAgACgCCgATTACgTggCTTTAgC 3', containing the EcoRI and Xbal restriction sites, and 5' CATgCCATggCCAACACAAAAg-AAgCTgg 3' , possessing the Ncol restriction site, in the presence of 50 nmol of each of the dNTPs, of Vent DNA polymerase buffer (IX) and of 2 U of Vent DNA polymerase (New England Biolabs). The PCR amplification reaction was carried out in the "GeneAmp PCR System 9700" thermocycler. After a denaturation at 94°C for 10 min., the DNA was subjected to 25 cycles each consisting of the steps of denaturation at 94°C for 1 min., of hybridization at 55°C for 1 min. and of elongation at 72°C for 1 min. 30 sec. During the final cycle, the elongation was continued at 72°C for 5 min. The DNA fragment derived from the amplification was isolated on a 1.5% agarose gel with the aid of the "Concert Rapid Gel Extraction System" kit, and hydrolysed successively with Ncol and Xbal, for 1 h at 37°C. In parallel, the vector fragment was prepared from the plasmid pMRT1183 by deleting the MPrll83 promoter region located 5' of the Ncol restriction site. In order to do this, 5 pg of plasmid pMRT1183 were digested successively for 1 h at 37°C with Xbal and Ncol, and the vector fragment of pMRT1183 was isolated on a 0.8% agarose gel with the aid of the "Concert Rapid Gel Extraction System" kit, before being dephosphorylated with 40 U of calf intestine alkaline phosphatase (New England Biolabs) in the presence of buffer 3 (IX) at 37°C for 1 h.<br>
The ligation reaction was carried out with 50 ng of the promoter fragment and 100 ng of plasmid thus treated, in a 10 pi reaction mixture, in the presence of the T4 DNA ligase buffer (IX) and of 400 units of T4 DNA ligase (New England Biolabs), in the<br><br>
"GeneAmp PCR System 9700" thermocycler, as described above. Escherichia coli DH5a bacteria, which had been made competent beforehand, were transformed with all of the ligation reaction mixture. The plasmid DNA of the clones obtained, which were selected on LB medium supplemented with ampicillin (50 mg/1), was extracted according to the alkaline lysis method and analysed with enzymatic digestions.<br>
The plasmid obtained was called pMRT1198, and the MPrll98 promoter sequence (SEQ.ID17) was represented diagrammatically in Fig. I was verified by sequencing.<br>
2.8. Construction of the MPrl216 promoter (SEQ.ID21). The MPrl216 promoter (SEQ.ID21) is derived from MPR1128 (SEQ.ID04) (described in section 2.2 of Example 2) by duplicating the sequence stretching from nucleotides -225 to -136 bp, this sequence comprising the XXG" box and the activating element.<br>
It was constructed by cloning into the vector pGEM3Z-l (described in section 2.1 of Example 2) the following two promoter fragments:<br>
- The "MPrl216 (SEQ.ID21) 5' fragment", synthesized by PCR, was amplified from 5 ng of pMRT1128 matrix DNA with the aid of 100 pmol of each of the 2 oligodeoxynucleotides 5' ATCggAATTCgCCgATTACgTggCTTTAgC 3', containing the EcoRI restriction site, and 5' gCTCTAgACCAACACAAAAgAAgCTgg 3' possessing the Xbal restriction site, in the presence of 50 nmol of each of the dNTPs, of Vent DNA polymerase buffer (IX) and of 2 U of Vent DNA polymerase (New England Biolabs). The PCR amplification reaction was carried out in the "GeneAmp PCR System 9700" thermocycler. After a denaturation at 94°C for 10 min., the DNA was subjected to 25 cycles each consisting of the steps of denaturation at 94°C for 1 min., of hybridization at 55°C for 1 min. and of elongation at 72°C for 1 min. 30 sec. During the final cycle, the elongation was continued at 72°C for 5 min. The DNA fragment derived from the PCR amplification was<br><br>
isolated on a 1.5% agarose gel with the aid of the "Concert Rapid Gel Extraction System" kit, and hydrolysed with EcoRI for 1 h at 37°c. The DNA fragment was then subjected to the action of 20 U of the Klenow fragment (New England Biolabs) for 30 min at 37°C in the presence of 60 nmol of each of the dNTPs, of 12 ul of 500 mM Tris-HCL, pH 7.5/500 mM MgC12 buffer and 6 ul of 1 M DTT, and digested with Xbal for 1 h at 37°C. - The "MPrl216 (SEQ.ID21) 3' fragment", obtained by the hydrolysis of 5 ug of the plasmid pMRTH83 with the restriction enzymes Xbal and BamHI, was isolated on a 1.5% agarose gel with the aid of the "Concert Rapid Gel Extraction System" kit.<br>
The ligation reaction was carried out with 50 ng of the "MPrl216 (SEQ.ID21) 5' fragment" and 50ng of the "MPrl216 (SEQ.ID21) 3' fragment" thus treated, and 50 ng of plasmid pGem3Z-l, in a 10 ul reaction mixture, in the presence of the T4 DNA ligase buffer (IX) and of 400 units of T4 DNA ligase (New England Biolabs), in the "GeneAmp PCR System 9700" thermocycler, as described above. Escherichia coli DH5a bacteria, which had been made competent beforehand, were transformed with all of the ligation reaction mixture. The plasmid DNA of the clones obtained, which were selected on LB medium supplemented with ampicillin (50 mg/1), was analysed by PCR with the aid of 10 pmol of each of the 2 oligodeoxynucleotides 5' ATCggAATTCgCCgATTACgTggCTTTAgC 3' and<br>
5' TACggATCCCCggggATCTCTAgTTTgTggTgC 3' in the "GeneAmp PCR System 9700" thermocycler, as described above.<br>
The plasmid obtained was called pMRT1216, and the MPrl216 promoter sequence (SEQ.ID21) represented diagrammatically in Fig. I was verified by sequencing.<br>
2.9. Construction of the MPrl217 promoter (SEQ.ID22).<br>
The MPrl217 promoter (SEQ.ID22) is derived from MPR1128 (SEQ.ID04) (described in section 2.2 of Example 2) by direct repeat triplication of the sequence stretching from nucleotides -225 to -136 bp, this sequence comprising the "G" box and the<br><br>
activating element.<br>
The MPrl217 promoter (SEQ.ID22) was constructed by inserting, into the Xbal restriction site of pMRT1183 (described in section 2.5 of Example 2), two identical promoter fragments synthesized by PCR from 5 ng of matrix DNA with the aid of 100 pmol of each of the 2 oligodeoxynucleotides<br>
5' ATCggAATTCTAgACgCCgATTACgTggCTTTAgC 3', containing the EcoRI and Xbal restriction sites, and 5' gCTCTAgACCAACACAAAAgAag-CTgg 3', possessing the Xbal restriction site, in the presence of 50 nmol of each of the dNTPs, of Vent DNA polymerase buffer (IX) and of 2 U of Vent DNA polymerase (New England Biolabs). The PCR amplification reaction was carried out in the "GeneAmp PCR System 9700" thermocycler. After a denaturation at 94 °C for 10 min., the DNA was subjected to 25 cycles each consisting of the steps of denaturation at 94°C for 1 min., of hybridization at 55°C for 1 min. and of elongation at 72°C for 1 min. 30 sec. During the final cycle, the elongation was continued at 72°C for 5 min. The DNA fragment derived from the PCR amplification was isolated on a 1.5% agarose gel with the aid of the "Concert Rapid Gel Extraction System" kit, and hydrolysed with Xbal for 1 h at 37°C.<br>
In parallel, the vector fragment was prepared from 10 ug of plasmid pMRT1183 by enzymatic digestion of the Xbal restriction site, located 5' of MPrll83, for 1 h at 37°C. The vector fragment thus linearized was dephosphorylated with 40 U of calf intestine alkaline phosphatase (New England Biolabs) in the presence of buffer 3 (IX) at 37°C for 1 h.<br>
The ligation reaction was carried out with 50 ng of promoter fragment and 100 ng of vector fragment thus prepared, in a 10 ial reaction mixture, in the presence of the IX T4 DNA ligase buffer (New England Biolabs) and of 400 units of T4 DNA ligase (New England Biolabs) in the "GeneAmp PCR System 9700" thermocycler, as described above. Escherichia coli DH5a bacteria, which had been made competent beforehand, were<br><br>
transformed with all of the ligation reaction mixture. The plasmid DNA of the clones obtained, which were selected on LB medium supplemented with ampicillin (50 mg/1), was analysed by PCR with the aid of 10 pmol of each of the 2 oligodeoxy-nucleotides 5' ATCggAATTCgCCgATTACgTggCTTTAgC 3' and 5' TACggATCCCCggggATCTCTAgTTTgTggTgC 3' in the "GeneAmp PCR System 9700" thermocycler, as described above.<br>
The plasmid obtained was called pMRT1217, and the MPrl217 promoter sequence (SEQ.ID22) represented diagrammatically in Fig. I, which was verified by sequencing, has a C deleted at position -317 [lacuna], 4 bp upstream of a "G"-like box.<br>
Example 3.<br>
Construction of plasmids containing chimeric promoter sequences.<br>
3.1. Construction of the MPrllSO promoter (SEQ.ID05).<br>
The MPrll30 promoter (SEQ.ID05) results from inserting, at position -65 bp of PrHMWG-Dx5 (SEQ.ID01), a 55-bp sequence corresponding to a duplication of the as-2 motif (Lam and Chua, 1989) and to the as-1 motif (Lam et al., 1989) of CaMV 35S. It was constructed by splicing by overlap extension the "MPrllSO (SEQ.ID05) 5' fragment" and the "MPrll30 (SEQ.ID05) 3' fragment", which had been synthesized by PCR.<br>
The "MPrllSO (SEQ.ID05) 5' fragment" was amplified by PCR from 5 ng of pUC19-HMWG matrix DNA (described in section 2.1 of Example 2) with the aid of 20 pmol of each of the 2 oligodeoxynucleotides 5' TACgAATTCCCAgCTTTgAgTggCCgTAg 3', containing the EcoRI restriction site, and 5' TgCgTCATCCCTTAC-gTCA-<br>
gTggAgATATCACATCAATCTTgATATCACATCAATCACggTgAggTTTgTTTAgCCTAAg 3' , possessing the 55-bp sequence corresponding to a duplication of the as-2 motif (Lam and Chua, 1989) and to the as-1 motif (Lam et al., 1989) of CaMV 35S, in the presence of 10 nmol of each of the dNTPs, of Vent DNA polymerase buffer (IX) and of 2 U of Vent DNA polymerase (New England Biolabs), in a 50 pi<br><br>
reaction volume. The PCR amplification reaction was carried out in the "GeneAmp PCR System 9700" thermocycler. After a denaturation at 94°C for 5 min., the DNA was subjected to 15 cycles each consisting of the steps of denaturation at 94°C for 1 min., of hybridization at 55°C for 1 min. and of elongation at 72°C for 1 min. 30 sec. During the final cycle, the elongation was continued at 72°C for 5 min. Forty ul of the PCR reaction medium were then subjected to the action of 12.5 U of the Klenow fragment (New England Biolabs) in the presence of 20 nmol of each of the dNTPs for 10 min. at 37°C. The PCR product thus treated was then isolated on a 1.5% agarose gel with the aid of the "QIAquick Gel Extraction" kit.<br>
The "MPrllSO (SEQ.ID05) 3' fragment" was synthesized and treated in the same way as the "MPrll30 (SEQ.ID05) 5' fragment", except that the 2 oligodeoxynucleotides used are 5' ATTgATgTgATATCAAg-<br>
ATTgATgTgATATCTCCACTgACgTAAgggATgACgCACACgCAgCCATggTCCTgAACCTTC 3', possessing the 55-bp sequence corresponding to a duplication of the as-2 motif (Lam and Chua, 1989) and to the as-1 motif (Lam et al., 1989) of CaMV 35S, and 5' TACggATCCCCggggATCTCTA-gTTTgTggTgC 3', containing the BamHI restriction site.<br>
The "MPrllSO (SEQ.ID05) 5' fragment" and the "MPrllSO (SEQ.ID05) 3' fragment" were then assembled by overlap extension so as to generate the "MPrllSO fragment (SEQ.ID05)". In order to do this, a PCR amplification was carried out using 7.5 ul of each of the two PCR products thus treated, with the aid of 20 pmol of each of the oligodeoxynucleotides 5' TACgAATTCCCAgCTTTgAgTggCCgTAg 3', containing the EcoRI restriction site, and 5' TACggATCCCCggggATCTCTAgTTTgTggTgC 3', possessing the BamHI restriction site, in the presence of 10 nmol of each of the dNTPs, of Vent DNA polymerase buffer (IX) and of 2 U of Vent DNA polymerase (New England Biolabs), in a 50 ul reaction volume. The PCR amplification reaction was carried out in the "GeneAmp PCR System 9700" thermocycler. After a<br><br>
denaturation at 94°C for 5 min., the DMA was subjected to 15 cycles each consisting of the steps of denaturation at 94°C for 1 min., of hybridization at 55°C for 1 min. and of elongation at 72°C for 1 min. 30 sec. During the final cycle, the elongation was continued at 72°C for 5 min. The "MPrllSO fragment (SEQ.ID05)" thus synthesized was isolated on a 1.5% agarose gel with the aid of the "QIAquick Gel Extraction" kit. This fragment was then hydrolysed with EcoRI for 1 h at 37°C, and subjected to the action of 20 U of the Klenow fragment (New England Biolabs) for 30 min at 37°C in the presence of 60 nmol of each of the dNTPs, of 12 ill of 500 mM Tris-HCL, pH 7.5/500 mM MgC12 buffer and 6 of ill of 1 M DTT. Finally, the MPrll30 fragment (SEQ.ID05) was digested with BamHI for 1 h at 37°C.<br>
The ligation was carried out with 100 ng of the "MPrllSO fragment (SEQ.ID05)" thus treated and 50 ng of plasmid pGem3Z-l (described in section 2.1 of Example 2) overnight at 16°C in a 10 ul reaction mixture, in the presence of 1 ul of the T4 DNA ligase buffer (IX) and of 400 units of T4 DNA ligase (New England Biolabs). Escherichia coli DH5a bacteria, which had been made competent beforehand, were transformed with all of the ligation reaction mixture. The plasmid DNA of the clones obtained, which were selected on LB medium supplemented with ampicillin (50 mg/1), was analysed by PCR with the aid of 25 pmol of each of the 2 oligodeoxynucleotides 5' TACgAATTCCCAgCTTTgAgTggCCgTAg 3r and<br>
5' TACggATCCCCggggATCTCTAgTTTgTggTgC 3' in the "GeneAmp PCR System 9700" thermocycler, as described above.<br>
The plasmid obtained was called pMRT1130, and the MPrll30 promoter sequence (SEQ.ID05) represented diagrammatically in Fig. Ill was verified by sequencing.<br>
3.2. Construction of the MPrll31 promoter (SEQ.ID06).<br>
The MPrllSl promoter (SEQ.ID06) results from inserting, at position -65 bp of PrHMWG-Dx5 (SEQ.ID01) (described in section 2.1 of Example 2), a 38-bp sequence corresponding to the as-2<br><br>
motif (Lam and Chua, 1989) and to the as-1 motif (Lam et al., 1989) of CaMV 35S. It was constructed by splicing by overlap extension the "MPrll31 (SEQ.ID06) 5' fragment" and the "MPrll31 (SEQ.ID06) 3' fragment", which had been synthesized by PCR.<br>
The "MPrllSl (SEQ.ID06) 5' fragment" was synthesized and treated in the same way as the "MPrllSO (SEQ.ID05) 5' fragment" (described in section 3.1 of Example 3), except that the 2 oligodeoxynucleotides used are<br>
5' TACgAATTCCCAgCTTTgAgTggCCgTAg 3', containing the EcoRI restriction site, and<br>
5' TgCgTCATCCCTTACgTCAgTggAgATATCACATCAATCACggTgAggTTTgTTTAgCCTA Ag 3', possessing the 38-bp sequence corresponding to the as-2 motif (Lam and Chua, 1989) and to the as-1 motif (Lam et al., 1989) of CaMV 35S.<br>
The "MPrllSl (SEQ.ID06) 3' fragment" was synthesized and treated in the same way as the "MPrllSO (SEQ.ID05) 5' fragment" (described in section 3.1 of Example 3), except that the 2 oligodeoxynucleotides used are 5' ATTgATgTgATATCTCCACTgACgTAAgggATgAC-<br>
gCACACgCAgCCATggTCCTgAACCTTC 3' possessing the 38-bp sequence corresponding to the as-2 motif (Lam and Chua, 1989) and to the as-1 motif (Lam et al., 1989) of CaMV 35S, and 5' TACggATCCCCggggATCTCTAgTTTgTggTgC 3' containing the BamHI restriction site.<br>
The "MPrllSl (SEQ.ID06) 5' fragment" and the "MPrll31 (SEQ.ID06) 3' fragment" were then assembled by overlap extension so as to generate the "MPrll31 fragment (SEQ.ID06)". In order to do this, a PCR amplification was carried out using 7.5 ul of each of the two PCR products thus treated, with the aid of 20 pmol of each of the oligodeoxynucleotides 5' TACgAATTCCCAgCTTTgAgTggCCgTAg 3', containing the EcoRI restriction site, and 5' TACggATCCCCggggATCTCTAgTTTgTggTgC 3', possessing the BamHI restriction site, in the presence of 10 nmol of each of the dNTPs, of Vent DMA polymerase buffer (IX)<br><br>
and of 2 U of Vent DNA polymerase (New England Biolabs), in a 50 pi reaction volume. The PCR amplification reaction was carried out in the "GeneAmp PCR System 9700" thermocycler. After a denaturation at 94°C for 5 min., the DNA was subjected to 15 cycles each consisting of the steps of denaturation at 94°C for 1 min., of hybridization at 55°C for 1 min. and of elongation at 72°C for 1 min. 30 sec. During the final cycle, the elongation was continued at 72°C for 5 min. The "MPrll31 fragment (SEQ.ID06)" thus synthesized was isolated on a 1.5% agarose gel with the aid of the "QIAquick Gel Extraction" kit. This fragment was then hydrolysed with EcoRI for 1 h at 31°C, and subjected to the action of 20 U of the Klenow fragment (New England Biolabs) for 30 min at 37°C in the presence of 60 nmol of each of the dNTPs, of 12 pi of 500 mM Tris-HCL, pH 7.5/500 mM MgC12 buffer and 6 of pi of 1 M DTT. Finally, the MPrll31 fragment (SEQ.ID06) was digested with BamHI for 1 h at 37°C.<br>
The ligation was carried out with 100 ng of the "MPrll30 fragment (SEQ.ID05)" thus treated and 50 ng of plasmid pGem3Z-l (described in section 2.1 of Example 2) overnight at 16°C in a 10 pi reaction mixture, in the presence of 1 pi of the T4 DNA ligase buffer (IX) and of 400 units of T4 DNA ligase (New England Biolabs). Escherichia coli DH5a bacteria, which had been made competent beforehand, were transformed with all of the ligation reaction mixture. The plasmid DNA of the clones obtained, which were selected on LB medium supplemented with ampicillin (50 mg/1), was analysed by PCR with the aid of 25 pmol of each of the 2 oligodeoxynucleotides 5' TACgAATTCCCAgCTTTgAgTggCCgTAg 3' and<br>
5' TACggATCCCCggggATCTCTAgTTTgTggTgC 3' in the "GeneAmp PCR System 9700" thermocycler, as described above.<br>
The plasmid obtained was called pMRT1131, and the MPrll31 promoter sequence (SEQ.ID06) represented diagrammatically in Fig. II was verified by sequencing.<br>
3.3. Construction of the MPrll35 promoter (SEQ.ID09).<br><br>
The MPrll35 promoter (SEQ.ID09) is derived from the MPrllSO promoter (SEQ.ID05) (described in section 3.1 of Example 3) by deleting the sequence located upstream of nucleotide -293, this sequence comprising the two prolamine-^like boxes and the two GATA boxes.<br>
The promoter fragment was amplified by PCR from 5 ng of pMRT1130 matrix DNA with the aid of 100 pmol of each of the 2 oligodeoxynucleotides 5' ATCGGAATTCCAGAACTAGGATTACGCCG 3', containing the EcoRI restriction site, and<br>
5' TACggATCCCCggggATCTCTAgTTTgTggTgC 3', possessing the BamHI restriction site, in the presence of 50 nmol of each of the dNTPs, of the Vent DNA polymerase buffer (IX) and of 2 U of Vent DNA polymerase (New England Biolabs). The PCR amplification reaction was carried out in the "GeneAmp PCR System 9700" thermocycler. After a denaturation at 94°C for 5 min., the DNA was subjected to 25 cycles each consisting of the steps of denaturation at 94°C for 1 min., of hybridization at 55°C for 1 min. and elongation at 72°C for 1 min. 30 sec. During the final cycle, the elongation was continued at 72°C for 1 min.<br>
The DNA fragment derived from the amplification was isolated on a 2% agarose gel with the aid of the "QIAquick Gel Extraction" kit, hydrolysed with EcoRI for 1 h at 37°C and then subjected to the action of 20 U of the Klenow fragment (New England Biolabs) for 30 min at 37°C in the presence of 60 nmol of each of the dNTPs, of 12 ul of 500 mM Tris-HCL, pH 7.5/500 mM MgC12 buffer and 6 ul of 1 M DTT. The DNA fragment thus treated was then digested with BamHI for 1 h at 37°C.<br>
The ligation was carried out with 100 ng of the MPR1135 promoter fragment (SEQ.ID09) thus treated and 50 ng of plasmid pGEM3Z-l (described in section 2.1 of Example 2) overnight at 16°C in a 10 ul reaction mixture, in the presence of the T4 DNA ligase buffer (IX) and of 400 units of T4 DNA ligase (New England Biolabs). Escherichia coli DH5a bacteria, which had been made competent beforehand, were transformed with all of the<br><br>
ligation reaction mixture. The plasmid DNA of the clones<br>
w<br>
obtained, which were selected on LB medium supplemented with<br>
ampicillin (50 mg/1), was analysed by PCR with the aid of 25<br>
pmol of each of the 2 oligodeoxynucleotides<br>
5' ATCGGAATTCGTGTTGGCAAACTGC 3' and<br>
5' TACggATCCCCggggATCTCTAgTTTgTggTgC 3', in the "GeneAmp PCR<br>
System 9700" thermocycler, as described above.<br>
The plasmid obtained was called pMRT1135, and the MPrll35 promoter sequence (SEQ.ID09) represented diagrammatically in Fig. Ill was verified by sequencing.<br>
3.4.	Construction of the MPrll38 promoter (SEQ.ID12).<br>
The MPrll38 promoter (SEQ.ID12) is derived from the<br>
MPrllSl promoter (SEQ.ID06) (described in section 3.2 of Example 3) by deleting the sequence located upstream of nucleotide -276, this sequence comprising the two prolamine-"like" boxes and the two GATA boxes.<br>
The promoter fragment was amplified by PCR from 5 ng of pMRT1131 matrix DNA, treated and obtained in the same way as the MPrll35 promoter (SEQ.ID09) (described in section 3.3 of Example 3) .<br>
The plasmid obtained was called pMRT1138, and the MPrll38 promoter sequence (SEQ.ID12) represented diagrammatically in Fig. II was verified by sequencing.<br>
3.5.	Construction of the MPrll37 promoter (SEQ.ID11).<br>
The MPrll37 promoter (SEQ.ID11) is derived from the<br>
MPrll31 promoter (SEQ.ID06) (described in section 3.2 of Example 3) by deleting the sequence located upstream of nucleotide -243, this sequence comprising the two prolamine-"like" boxes, the two GATA boxes and the "G"-like box.<br>
The promoter fragment was amplified by PCR from 5 ng of pMRT1131 matrix DNA with the aid of 100 pmol of each of the 2 oligodeoxynucleotides 5' ATCGGGAATTCGCAGACTGTCCAAAAATC 3', containing the EcoRI restriction site, and 5' TACggATCCCCggggATCTCTAgTTTgTggTgC 3', possessing the BamHI<br>
restriction site, treated and obtained in the same way as the MPrll35 promoter (SEQ.ID09) (described in section 3.3 of Example 3) .<br>
The plasmid obtained was called pMRT1137, and the MPrllSV promoter sequence (SEQ.ID11) represented diagrammatically in Fig. II was verified by sequencing.<br>
3.6.	Construction of the MPrll34 promoter (SEQ.ID08).<br>
The MPrll34 promoter (SEQ.ID08) is derived from the<br>
MPrll30 promoter (SEQ.ID05) (described in section 3.1 of Example 3) by deleting the sequence located upstream of nucleotide -260, this sequence comprising the two prolamine-"like" boxes, the two GATA boxes and the "G"-like box.<br>
The promoter fragment was amplified by PCR from 5 ng of pMRTUSO matrix DNA with the aid of 100 pmol of each of the 2 oligodeoxynucleotides 5f ATCGGGAATTCGCAGACTGTCCAAAAATC 3', containing the EcoRI restriction site, and<br>
5' TACggATCCCCggggATCTCTAgTTTgTggTgC 3', possessing the BamHI restriction site, treated and obtained in the same way as the MPrll35 promoter (SEQ.ID09) (described in section 3.3 of Example 3) .<br>
The plasmid obtained was called pMRT1134, and the MPrll34 promoter sequence (SEQ.ID08) represented diagrammatically in Fig. Ill was verified by sequencing.<br>
3.7.	Construction of the MPrll36 promoter (SEQ.ID10).<br>
The MPrll36 promoter (SEQ.ID10) is derived from the<br>
MPrll31 promoter (SEQ.ID06) (described in section 3.2 of Example 3) by deleting the sequence located upstream of nucleotide -180, this sequence comprising the two prolamine-"like" boxes, the two GATA boxes, the "G"-like box and the activating element.<br>
The promoter fragment was amplified by PCR from 5 ng of pMRTHSl matrix DNA with the aid of 100 pmol of each of the 2 oligodeoxynucleotides 5' ATCGGAATTCGTGTTGGCAAACTGC 3', containing the EcoRI restriction site, and 5' TACggATCCCCggggATCTCTAgTTTgTggTgC 3', possessing the BamHI<br><br>
restriction site, treated and obtained in the same way as the jM'PrllSS promoter (SEQ.ID09) (described in section 3.3 of Example 3) .<br>
The plasmid obtained was called pMRT1136, and the MPrll36 promoter sequence (SEQ.ID10) represented diagrammatically in Fig. II was verified by sequencing.<br>
3.8.	Construction of the MPrll33 promoter (SEQ.ID07).<br>
The MPrll33 promoter (SEQ.ID07) is derived from the<br>
MPrll30 promoter (SEQ.ID05) (described in section 3.2 of Example 3) by deleting the sequence located upstream of nucleotide -197, this sequence comprising the two prolamine-"like" boxes, the two GATA boxes, the "G"-like box and the activating element.<br>
The promoter fragment was amplified by PCR from 5 ng of pMRT1130 matrix DMA with the aid of 100 pmol of each of the 2 oligodeoxynucleotides 5' ATCGGAATTCGTGTTGGCAAACTGC 3', containing the EcoRI restriction site, and<br>
5' TACggATCCCCggggATCTCTAgTTTgTggTgC 3', possessing the BamHI restriction site, treated and obtained in the same way as the MPrll35 promoter (SEQ.ID09) (described in section 3.3 of Example 3) .<br>
The plasmid obtained was called pMRT1133, and the MPrll33 promoter sequence (SEQ.ID07) represented diagrammatically in Fig. Ill was verified by sequencing.<br>
3.9.	Construction of the MPrll39 promoter (SEQ.ID13).<br>
The MPrll39 promoter (SEQ.ID13) results from inserting, at position -405 bp of MPrll31 (SEQ.ID06) (described in section 3.2 of Example 3), a 61-bp sequence which includes the duplication of the "cereal" box of the promoter of the high molecular weight glutenin gene encoding the Bx7 subunit (PrHMWG-Bx7) of the hexaploid wheat Triticum aestivum L. cv Cheyenne (Anderson et al., 1998).<br>
The MPrll39 promoter (SEQ.ID13) was amplified by PCR from 5 ng of pMRT1131 matrix DNA with the aid of 100 pmol of each of the 2 oligodeoxynucleotides<br><br>
5'   TACgAATTCCTCgACATggTTAgAAgTTTTgAgTgCCgCCACTACTCgACAT-<br>
t<br>
ggTTAgAAgTTTTgAgTggCCgTAgATTTgC 3', containing the EcoRI restriction site and the two "cereal" boxes described above, and 5' TACggATCCCCggggATCTCTAgTTTgTggTgC 3', possessing the BamHI restriction site, in the presence of 50 nmol of each of the dNTPs, of the Vent DNA polymerase buffer (IX) and of 2 U of Vent DNA polymerase (New England Biolabs). The PCR amplification reaction was carried out in the "GeneAmp PCR System 9700" thermocycler. After a denaturation at 94°C for 5 min., the DNA was subjected to 25 cycles each consisting of the steps of denaturation at 94°C for 1 min., of hybridization at 55°C for 1 min. and of elongation at 72°C for 1 min. 30 sec. During the final cycle, the elongation was continued at 72°C for 1 min.<br>
The DNA fragment derived from the amplification was isolated on a 2% agarose gel with the aid of the "QIAquick Gel Extraction" kit and hydrolysed with EcoRI for 1 h at 37°C. The DNA fragment was then subjected to the action of 20 U of the Klenow fragment (New England Biolabs) for 30 min at 37°C in the presence of 60 nmol of each of the dNTPs, of 12 ul of 500 mM Tris-HCL, pH 7.5/500 mM MgC12 buffer and 6 ul of 1 M DTT, and digested with BamHI for 1 h at 37°C.<br>
The ligation was carried out with 100 ng of the MPrll39 promoter fragment (SEQ.ID13) thus treated and 50 ng of plasmid pGem3Z-l (described in section 2.1 of Example 2) overnight at 16°C, in a 10 ul reaction mixture, in the presence of the T4 DNA ligase buffer (IX) and of 400 units of T4 DNA ligase (New England Biolabs). Escherichia coli DH5a bacteria, which had been made competent beforehand, were transformed with all of the ligation reaction mixture. The plasmid DNA of the clones obtained, which were selected on LB medium supplemented with ampicillin (50 mg/1), was analysed by PCR with the aid of 25 pmol of each of the 2 oligodeoxynucleotides 5' ATCGGAATTCCAGAACTAGGATTACGCCG 3' and 5' TACggATCCCCggggATCTCTAgTTTgTggTgC 3', in the "GeneAmp PCR<br><br>
System 9700" thermocycler, as described above.<br>
The plasmid obtained was called pMRT1139, and the MPrll39 promoter sequence (SEQ.ID13) represented diagrammatically in Fig. IV was verified by sequencing.<br>
3.10. Construction of the MPrl200 promoter (SEQ.ID19).<br>
The MPrl200 promoter (SEQ.ID19) results from inserting, at position -263 bp of MPrll38 (SEQ.ID12) (described in section 3.4 of Example 3), a 79-bp sequence which includes the duplication of the "cereal" box of the promoter of the high molecular weight glutenin gene encoding the Bx7 subunit (PrHMWG-Bx7) of the hexaploid wheat Triticum aestivum L. cv Cheyenne (Anderson et al., 1998).<br>
The MPrl200 promoter (SEQ.ID19) was constructed by cloning into the vector pGEM3Z-l (described in section 2.1 of Example 2) the following two promoter fragments:<br>
- The "MPrl200 (SEQ.ID19) 5' fragment", synthesized by PCR, was amplified from 5 ng of pMRT1139 matrix DNA (described in section 3.9 of Example 3) with the aid of 100 pmol of each of the 2 oligodeoxynucleotides 5' TACgAATTCCTCgACATgg 3', containing the EcoRI restriction site, and 5' gCTCTAgAgCAAATCTACggCCACTC 3', possessing the Xbal restriction site, in the presence of 50 nmol of each of the dNTPs, of Vent DNA polymerase buffer (IX) and of 2 U of Vent DNA polymerase (New England Biolabs). The PCR amplification reaction was carried out in the "GeneAmp PCR System 9700" thermocycler. After a denaturation at 94°C for 10 min., the DNA was subjected to 25 cycles each consisting of the steps of denaturation at 94°C for 1 min., of hybridization at 55°C for 1 min. and of elongation at 72°C for 1 min. 30 sec. During the final cycle, the elongation was continued at 72°C for 5 min. The DNA fragment derived from the PCR amplification was isolated on a 1.5% agarose gel with the aid of the "Concert Rapid Gel Extraction System" kit, and hydrolysed with EcoRI for 1 h at 37°C. The DNA fragment was then subjected to the action of 20 U of the Klenow fragment (New England Biolabs) for 30 min<br>
at 37°C in the presence of 60 nmol of each of the dNTPs, of 12 il of 500 mM Tris-HCL, pH 7.5/500 mM MgC12 buffer and 6 ul of 1 M DTT, and then digested with Xbal for 1 h at 37°C. - The "MPrl200 (SEQ.ID19) 3' fragment", synthesized by PCR, as amplified from 5 ng of pMRT1138 matrix DMA with the aid of 100 pmol of each of the 2 oligodeoxynucleotides<br>
5' ATCggAATTCTAgACgCCgATTACgTggCTTTAgC 3', containing the EcoRI restriction site and Xbal restriction site, and 5' TACggATCCCCggggATCTCTAgTTTgTggTgC 3', possessing the BamHI restriction site, in the presence of 50 nmol of each of the dNTPs, of Vent DNA polymerase buffer (1X) and of 2 U of Vent DNA polymerase (New England Biolabs) in the "GeneAmp PCR System 9700" thermocycler, under the same conditions as the "MPrl200 (SEQ.1D19) 5' fragment". The DNA fragment derived from the PCR amplification was isolated on a 1.5% agarose gel with the aid of the "Concert Rapid Gel Extraction System" kit and hydrolysed successively with Xbal and BamHI for 1 h at 37°C.<br>
The ligation reaction was carried out with 50 ng of the "MPrl200 (SEQ.ID19) 5' fragment", 50 ng of the "MPrl200 (SEQ.ID19) 3' fragment" thus treated, and 50 ng of plasmid pGem3Z-l, in a 10 ul reaction mixture, in the presence of the T4 DNA ligase buffer (IX) and of 400 units of T4 DNA ligase (New England Biolabs) in the "GeneAmp PCR System 9700" thermocycler, as described above. Escherichia coli DH5a bacteria, which had been made competent beforehand, were transformed with all of the ligation reaction mixture. The plasmid DNA of the clones obtained, which were selected on LB medium supplemented with ampicillin (50 mg/1), was analysed by PCR with the aid of 10 pmol of each of the 2 oligodeoxynucleotides 5' ATCggAATTCgCAgCCATggTCCTgAACC 3' and<br>
5' TACggATCCCCggggATCTCTAgTTTgTggTgC 3', in the "GeneAmp PCR System 9700" thermocycler, as described above.<br>
The plasmid obtained was called pMRT1200, and the MPrl200 promoter sequence (SEQ.ID19) represented diagrammatically in<br><br>
Fig. IV was verified by sequencing.<br>
3.11. Construction of the MPrl213 promoter (SEQ.ID20).<br>
The MPrl213 promoter (SEQ.ID20) results from inserting, upstream of position -225 bp of MPR1128 (SEQ.ID04) (described in section 2.2 of Example 2), a 79-bp sequence which includes the duplication of the "cereal" box of the promoter of the high molecular weight glutenin gene encoding the Bx7 subunit (PrHMWG-Bx7) of the hexaploid wheat Triticum aestivum L. cv Cheyenne (Anderson et al., 1998).<br>
The MPrl213 promoter (SEQ.ID20) was constructed by cloning into the vector pGEM3Z-l (described in section 2.1 of Example 2), the following two promoter fragments:<br>
- The "MPrl213 (SEQ.ID20) 5' fragment", synthesized by PCR, was amplified from 5 ng of pMRT1139 matrix DNA (described in section 3.9 of Example 3) with the aid of 100 pmol of each of the 2 oligodeoxynucleotides 5' TACgAATTCCTCgACATgg 3', containing the EcoRI restriction site, and 5' gCTCTAgAgCAAATCTACggCCACTC 3', possessing the Xbal restriction site, in the presence of 50 nmol of each of the dNTPs, of Vent DNA polymerase buffer (IX) and of 2 U Vent DNA polymerase (New England Biolabs). The PCR amplification reaction was carried out in the "GeneAmp PCR System 9700" thermocycler. After a denaturation at 94°C for 10 min., the DNA was subjected to 25 cycles each consisting of the steps of denaturation at 94°C for 1 min., of hybridization at 55°C for I min. and of elongation at 72°C for 1 min. 30 sec. During the final cycle, the elongation was continued at 72°C for 5 min. The DNA fragment derived from the PCR amplification was isolated on a 1.5% agarose gel with the aid of the "Concert Rapid Gel Extraction System" kit, and hydrolysed with EcoRI for 1 h at 37°C. The DNA fragment was then subjected to the action of 20 U of the Klenow fragment (New England Biolabs) for 30 min at 37°C in the presence of 60 nmol of each of the dNTPs, of 12 111 of 500 mM Tris-HCL, pH 7.5/500 mM MgC12 buffer and 6 ill of 1 M DTT, and digested with Xbal for 1 h at 37°C.<br><br>
- The "MPrl213 (SEQ.ID20) 3' fragment", obtained by thef-<br>
hydrolysis of 5 ug of the plasmid pMRT1183 (described in section 2.5 of Example 2) with the Xbal and BamHI restriction enzymes, was isolated on a 1.5% agarose gel with the aid of the "Concert Rapid Gel Extraction System" kit.<br>
The ligation reaction was carried out with 50 ng of the "MPrl213 (SEQ.ID20) 5' fragment" and 50 ng of the "MPrl213 (SEQ.ID20) 3' fragment" thus treated, and 50 ng of plasmid pGem3Z-l, in a 10 ul reaction mixture, in the presence of the T4 DNA ligase buffer (IX) and of 400 units of T4 DNA ligase (New England Biolabs) in the "GeneAmp PCR System 9700" thermocycler, as described above. Escherichia coli DH5a bacteria, which had been made competent beforehand, were transformed with all of the ligation reaction mixture. The plasmid DNA of the clones obtained, which were selected on LB medium supplemented with ampicillin (50 mg/1), was analysed by PCR with the aid of 10 pmol of each of the 2 oligodeoxynucleotides<br>
5' TACgAATTCCTCgACATgg 3' and 5' gCTCTAgAgCAAATCTACggCCACTC 3', in the "GeneAmp PCR System 9700" thermocycler, as described above.<br>
The plasmid obtained was called pMRT1213, and the MPrl213 promoter sequence (SEQ.ID20) represented diagrammatically in Fig. IV was verified by sequencing.<br>
3.12. Construction of the MPrll99 promoter (SEQ.ID18).<br>
The MPrll99 promoter (SEQ.ID18) results from inserting, at position -224 bp of MPR1128 (SEQ.ID04) (described in section 2.2 of Example 2), a 27-bp sequence which includes the "GC-rich" element of the intergenic region of the maize streak virus (MSV) (Fenoll et al., 1990).<br>
The MPrll99 promoter (SEQ.ID18) was amplified by PCR from 5 ng of pMRT1128 matrix DNA with the aid of 100 pmol of each of the 2 oligodeoxynucleotides<br>
5' ATCGGAATTCAAATGGGCCGGACCGGGCCGGCCCAGCGCCGATTACGTGGCTTTAGC 3', containing the "GC-rich" element described above and the EcoRI<br><br>
and Xbal restriction sites, and<br>
5' TACggATCCCCggggATCTCTAgTTTgTggTgC 3', possessing the BamHI restriction site, in the presence of 50 nmol of each of the dNTPs, of the Vent DNA polymerase buffer (IX) and of 2 U of Vent DNA polymerase (New England Biolabs). The PCR amplification reaction was carried out in the "GeneAmp PCR System 9700" thermocycler. After a denaturation at 94°C for 5 min., the DNA was subjected to 25 cycles each consisting of the steps of denaturation at 94°C for 1 min., of hybridization at 55°C for 1 min. and of elongation at 72°C for 1 min. 30 sec. During the final cycle, the elongation was continued at 72°C for 1 min.<br>
The DNA fragment derived from the amplification was isolated on a 1.5% agarose gel with the aid of the "Concert Rapid Gel Extraction System" kit and hydrolysed with EcoRI for 1 h at 37°C. The fragment was then subjected to the action of 20 U of the Klenow fragment (New England Biolabs) for 30 min at 37°C in the presence of 60 nmol of each of the dNTPs, of 12 ul of 500 mM Tris-HCL, pH 7.5/500 mM MgCl2 buffer and 6 ul of 1 M DTT, and digested with BamHI for 1 h at 37°C.<br>
The ligation was carried out with 100 ng of the MPrll99 promoter fragment (SEQ.ID18) thus treated and 50 ng of plasmid pGem3Z-l (described in section 2.1 of Example 2), with PCR cycles in the "GeneAmp PCR System 9700" thermocycler under the conditions described above. Escherichia coli DH5a bacteria, which had been made competent beforehand, were transformed with all of the ligation reaction mixture. The plasmid DNA of the clones obtained, which were selected on LB medium supplemented with ampicillin (50 mg/1), was analysed by PCR with the aid of 25 pmol of each of the 2 oligodeoxynucleotides 5' ATCGGAATTCCAGAACTAGGATTACGCCG 3' and<br>
5' TACggATCCCCggggATCTCTAgTTTgTggTgC 3' in the "GeneAmp PCR System 9700" thermocycler, as described above.<br>
The plasmid obtained was called pMRT1199mut, and the corresponding MPrll99 (SEQ.ID18) mut promoter sequence, which<br><br>
was verified by sequencing, has a mutation in the untranslated leader sequence at position +27 [lacuna]. To reestablish the unmutated MPrll99 sequence (SEQ.ID18), the "MPrll99 (SEQ.ID18) 5' fragment" stretching from position -251 to -58 bp was cloned in the place of the "MPrll83 5' fragment" stretching from position -225 to -58 bp.<br>
In order to do this, 10 ug of plasmid pMRT1199mut were digested with EcoRI for 1 h at 37°C, and subjected to the action of 20 U of the Klenow fragment (New England Biolabs) for 30 min at 37°C in the presence of 60 nmol of each of the dNTPs, of 12 pi of 500 mM Tris-HCL, pH 7.5/500 mM MgC12 buffer and 6 ul of 1 M DTT. After a digestion with Ncol for 1 h at 37°C, the "MPrll99 (SEQ.ID18) 5' fragment" was isolated on a 1.5% agarose gel with the aid of the "Concert Rapid Gel Extraction System" kit.<br>
In parallel, 5 ug of plasmid pMRT1183 (described in section 2.5 of Example 2) were digested for 1 h at 37°C with Xbal, and subjected to the action of 20 U of the Klenow fragment (New England Biolabs) for 30 min at 37°C in the presence of 60 nmol of each of the dNTPs, of 12 ul of 500 mM Tris-HCL, pH 7.5/500 mM MgC12 buffer and 6 ul of 1 M DTT. After a digestion with Ncol, the vector pMRT1183 thus deleted of the "MPR1183 5' fragment" was isolated on a 0.8% agarose gel with the aid of the "Concert Rapid Gel Extraction System" kit, and dephosphorylated with 40 U of calf intestine alkaline phosphatase (New England Biolabs) in the presence of buffer 3 (IX) at 37°C for 1 h.<br>
The ligation reaction was carried out with 100 ng of the "MPrll99 (SEQ.ID18) mut 5' fragment" and 50 ng of the plasmid pMRT1183 thus treated, in a 10 ul reaction mixture, in the presence of the T4 DNA ligase buffer (IX) and of 400 units of T4 DNA ligase (New England Biolabs), in the "GeneAmp PCR System 9700" thermocycler, as described above. Escherichia coli DH5a bacteria, which had been made competent beforehand, were transformed with all of the ligation reaction mixture. The<br>
plasmid DMA of the clones obtained, which were selected on LB medium supplemented with ampicillin (50 mg/1), was extracted according to the alkaline lysis method and analysed by enzymatic digestion.<br>
The plasmid obtained was called pMRT1199, and the MPrll99 promoter sequence (SEQ.ID18) is represented diagrammatically in Fig. IV.<br>
Example 4.<br>
Construction of the binary plasmids containing the MPrllSO (SEQ.ID05), MPrllSl (SEQ.ID06), MPrll35 (SEQ.ID09), MPrll38 (SEQ.ID12), MPrll39 (SEQ.ID13) and MPrl092 promoters used in the stable transformation of tobacco.<br>
4.1. Construction of the binary plasmid pMRT1177.<br>
The binary plasmid pMRT1177 was obtained by inserting the expression cassette "MPrllSO (SEQ.ID05)/uidA-IV2/term-nos" of pMRT 1130 (described in section 3.1 of Example 3) into the EcoRI site of the binary plasmid pMRTlllS (unpublished patent application FR 9911112).<br>
In order to do this, 10 \ig of plasmid pMRTUSO were digested successively with EcoRI and XmnI for 1 h at 37°C. The expression cassette was then isolated on a 0.8% agarose gel with the aid of the "Concert Rapid Gel Extraction System" kit.<br>
In parallel, 10 pg of binary plasmid pMRTlllS were digested with EcoRI for 1 h at 37°C. The linearized vector fragment was then dephosphorylated with 40 U of calf intestine alkaline phosphatase (New England Biolabs) in the presence of buffer 3 (IX) at 37°C for 1 h.<br>
The ligation was carried out with 50 ng of the expression cassette and 100 ng of plasmid pMRTlllS thus treated, overnight at 16°C in a 10 ul reaction mixture, in the presence of the T4 DNA ligase buffer (IX) and of 400 units of T4 DNA ligase (New England Biolabs). Escherichia coli DH5a bacteria, which had been made competent beforehand, were transformed with all of the ligation reaction mixture. The plasmid DNA of the clones<br>
obtained, which were selected on LB medium supplemented with *<br>
kanamycin (50 mg/1), was extracted according to the alkaline lysis method and analysed with enzymatic digestions.<br>
The plasmid obtained, called pMRT1177, was transferred into the LBA4404 Agrobacterium tumefaciens strain according to the technique described by Holsters et al. (1978). The plasmid DNA of the clones obtained, which were selected on 2YT medium (10 g/1 bactotryptone, 10 g/1 yeast extract, 5 g/1 NaCl, pH 7.2 and 15 g/1 Agar-Agar) supplemented with rifampicin (50 mg/1) and with kanamycin (50 mg/1), was extracted according to the alkaline lysis method, which was modified by adding lysozyme (25 mg/ml) to the cell resuspension buffer. The plasmid DNA was analysed with enzymatic digestions and the agrobacterium clone obtained was called A1177.<br>
4.2.	Construction of the binary plasmid pMRT1178.<br>
The binary plasmid pMRT1178 was obtained by inserting the expression cassette "MPrllSl (SEQ.ID06)/uidA-IV2/term-nos" into the EcoRI site of the binary plasmid pMRTlllS, which was described in section 4.1 of Example 4, except that the expression cassette was isolated from the plasmid pMRTHSl (described in section 3.2 of Example 3).<br>
The resulting plasmid was called pMRT1178, and was transferred into the LBA4404 Agrobacterium tumefaciens strain according to the protocol described above in section 4.1 of Example 4. The agrobacterium clone obtained was called A1178.<br>
4.3.	Construction of the binary plasmid pMRT1179.<br>
The binary plasmid pMRT1179 was obtained by inserting the expression cassette "MPrll35 (SEQ.ID09)/uidA-IV2/term-nos" into the EcoRI site of the binary plasmid pMRTlllS, which was described in section 4.1 of Example 4, except that the expression cassette was isolated from the plasmid pMRT1135 (described in section 3.3 of Example 3).<br>
The resulting plasmid was called pMRT1179, and was transferred into the LBA4404 Agrobacterium tumefaciens strain<br>
according to the protocol described above in section 4.1 of Example 4. The agrobacterium clone obtained was called A1179.<br>
4.4.	Construction of the binary plasmid pMRTHSO.<br>
The binary plasmid pMRTUSO was obtained by inserting the expression cassette "MPrll38 (SEQ.ID12)/uidA-IV2/term-nos" into the EcoRI site of the binary plasmid pMRT1138, which was described in section 4.1 of Example 4, except that the expression cassette was isolated from the plasmid pMRT1138 (described in section 3.4 of Example 3).<br>
The resulting plasmid was called pMRTUSO, and was transferred into the LBA4404 Agrobacterium tumefaciens strain according to the protocol described above in section 4.1 of Example 4. The agrobacterium clone obtained was called A1180.<br>
4.5.	Construction of the binary plasmid pMRTHSl.<br>
The binary plasmid pMRTHSl was obtained by inserting the expression cassette "MPrll39 (SEQ. ID13) /iudA-IV2/term-nos" into the EcoRI site of the binary plasmid pMRTlllS, which was described in section 4.1 of Example 4, except that the expression cassette was isolated from the plasmid pMRT1139 (described in section 3.9 of Example 3).<br>
The resulting plasmid was called pMRT1181, and was transferred into the LBA4404 Agrobacterium tumefaciens strain according to the protocol described above in section 4.1 of Example 4. The agrobacterium clone obtained was called A1181.<br>
4.6.	Construction of the binary plasmid pMRT1182.<br>
The binary plasmid pMRT1182 was obtained by inserting the CaMV PrD35S promoter fragment and the sequence uidfl-IV2/term-nos into the binary plasmid pMRTlllS.<br>
CaMV PrD35S was isolated by digesting 10 ug of the plasmid pJIT163D successively with Kpnl and with Hindlll for 1 h at 37°C. The 743-bp fragment corresponding to CaMV PrD35S was separated on a 0.8% agarose gel, and then purified with the aid of the "QIAquick Gel Extraction" kit.<br>
The sequence uidA-IV2/term-.nos was obtained by digesting 4<br>
lag of plasmid pMRT1092 with HindiII and EcoRI for 1 h at 37°C. The 2.2-kb fragment corresponding to the sequence uidA-IV2/term-nos was separated on a 0.8% agarose gel, and then purified with the aid of the "QIAquick Gel Extraction" kit.<br>
In parallel, 10 ug of binary plasmid pMRTlllS were digested successively with Kpnl and EcoRI for 1 h at 37°C. The linearized vector fragment was then dephosphorylated with 40 U of calf intestine alkaline phosphatase (New England Biolabs) in the presence of buffer 3 (IX), for 1 h at 37°C.<br>
The ligation was carried out in the presence of 100 ng of binary plasmid, 50 ng of the CaMV PrD35S fragment and 50 ng of the fragment corresponding to the sequence uidA-IV2/term-nos in a 20 ul reaction volume, in the presence of the T4 DNA ligase buffer (IX) and 400 units of T4 DNA ligase enzyme (New England Biolabs). The incubation was carried out with PCR cycles in the "GeneAmp PCR System 9700" thermocycler as described above. Escherichia coli DH5a bacteria, which had been made competent beforehand, were transformed with half of the ligation reaction medium. The plasmid DNA of the clones obtained, which were selected on LB medium supplemented with kanamycin (50 mg/1), was extracted according to the alkaline lysis method and analysed with enzymatic digestions.<br>
The resulting plasmid was called pMRT1182, and was transferred into the LBA4404 Agrobacterium tumefaciens strain according to the protocol described above in section 4.1 of Example 4. The agrobacterium clone obtained was called A1182.<br>
Example 5.<br>
Construction of the binary plasmid pMRT1207 containing the MPrll39 promoter (SEQ.ID13), used in the stable transformation of maize.<br>
The binary plasmid pMRT1207 was obtained by inserting the expression cassette "MPrll39 (SEQ.ID13)/uidA-IV2/term-nos" of pMRT 1139 into the Hpal site of the binary plasmid pMRT1195 (unpublished Patent application FR 9911112).<br>
In order to do this, 7 jag of plasmid pMRT1139 were digested successively with EcoRI and XmnI for 1 h at 37°C. The expression cassette was then isolated on a 0.7% agarose gel with the aid of the "Concert Rapid Gel Extraction System" kit, and subjected to the action of 20 U of the Klenow fragment (New England Biolabs) for 30 min. at 37°C in the presence of 60 nmol of each of the dNTPs, of 12 pi of MgC12 buffer (500 mM) and of 6 ul of DTT (1 M).<br>
In parallel, 5 ]iq of binary plasmid pMRT1195 were digested with Hpal for 1 h at 37°C. The linearized vector fragment was then dephosphorylated with 40 U of calf intestine alkaline phosphatase (New England Biolabs) in the presence of buffer 3 (IX) at 37°C for 1 h.<br>
The ligation was carried out with 100 ng of the expression cassette and 10 ng of plasmid pMRT1195 thus treated, with PCR cycles in the "GeneAmp PCR System 9700" thermocycler, as described above. Escherichia coli DH5a bacteria, which had been made competent beforehand, were transformed with all of the ligation reaction mixture. The plasmid DNA of the clones obtained, which were selected on LB medium supplemented with kanamycin (50 mg/1), was extracted according to the alkaline lysis method and analysed with enzymatic digestions.<br>
The plasmid obtained, called pMRT1207, was transferred, as described in section 4.1 of Example 4, into the LBA4404-pSBl Agrobacterium tumefaciens strain, this strain being derived from the LBA4404 Agrobacterium tumefaciens strain subsequent to the integration of the plasmid pSBl (unpublished Patent application FR 9911112), according to the protocol described above for the production of A1177. The plasmid DNA of the clones obtained, which were selected on 2YT medium supplemented with rifampicin (50 mg/1), with kanamycin (50 mg/1) and with tetracycline (5 mg/1), was extracted according to the alkaline lysis method, which was modified by adding lysozyme (25 mg/ml) to the cell resuspension buffer. The plasmid DNA was analysed with enzymatic<br>
digestions, and the agrobacterium clone obtained was called A1207.<br>
Example 6.<br>
Measurement and comparison of the activity of the various promoters in transient expression in maize and tobacco.<br>
6.1 Preparation of plant extracts.<br>
6.1.1.	Production and preparation of maize seeds.<br>
The transient expression experiments were carried out on<br>
the maize albumen SN 87 165 (L2), removed from maize plants cultivated in a phytotron at 24°C, under 60% humidity and a photoperiod of 16 h light/8 h darkness.<br>
Twelve days after pollination (DAP), the maizes were removed and sterilized in a bath of 20% domestos with stirring for 5 min. Following the removal of the domestos with successive rinses in sterile deionized water, the pericarp and the layer of aleurone cells are carefully removed under sterile conditions. Tangential sections of the albumen thus extracted were prepared and placed on filter paper soaked in minimum murashige and Skoog medium (MS 5524, Sigma).<br>
6.1.2.	In vitro culture of tobacco, preparation of the<br>
leaves.<br>
The transient expression experiments were carried out on 6-week-old leaves of tobacco (Nicotiana tabacum L.) of the cultivar PBD6. Mature seeds of tobacco cv. PBD6 were sterilized for 10 min in a saturated solution of calcium hypochlorite (70 g/1), and then rinsed three times for 5 min in sterile deionized water. The sterile seeds were placed on MS20 medium (Murashige and Skoog, 1962) and incubated for 6 weeks in a culture chamber (constant temperature of 24°C, 16 h light/8h darkness photoperiod).<br>
In order to minimize the destruction of the cells of the foliar mesophyll during the transformation by biolistics, the 2 main leaves of the 6-week-old PBD6 tobacco plants were removed 24 h before transformation, placed, with the upper side of the<br>
leaf facing upwards, on the BY3 gentle plasmolysis medium (4.4 g/1 MS-5519 salts, 100 mg/1 myoinositol, 1 mg/1 thiamine, 200 mg/1 KH2P04, 30 g/1 sucrose, 45.5 g/1 sorbitol, 1 mg/1 2.4 D, 8 g/1 agar, pH 5.8), and placed in a culture chamber (constant temperature of 25°C, 16 h light/ 8 h darkness photoperiod).<br>
6.2. Adsorption of the plasmid DNA onto tungsten or gold mi cropar tides.<br>
The transformation by biolistics required the DNA to be deposited beforehand on spherical microparticles made of tungsten or of gold, sterilized for 10 min in absolute ethanol (99.98%, containing less than 0.02% of water), washed four times in sterile deionized water, and conserved at -20°C in a solution of 50% glycerol for a maximum of 4 weeks.<br>
The concentration of all of the control and test plasmids used during the transformation was adjusted to 1 mg/ml. In each of the transformation experiments in which the activity of the promoter studied was evaluated using a luminometric assay, an internal reference control (pCaMV35Sluc) was cotransformed in order to normalize the variations in the GUS activity between the various experiments (Leckie et al., 1994). However, when the activity of the promoter studied was determined using a histochemical assay, the reference plasmid was not cotrans formed.<br>
The coating of the DNA onto the particles thus prepared was carried out in a sterile laminar flow chamber. A 1.8 mg aliquot fraction of sterile particle suspension in 30 ul of 50% glycerol was vigorously mixed by vortexing for 1 min., and then for 10 sec. with 20 ul of the DNA suspension containing 5 ug of one of the plasmids to be tested and 2 ug of the reference plasmid pCaMV35Sluc. Then 20 ul of 2.5 M CaCl2 were added and vigorously mixed for 10 sec. Next, 20 ul of 0.1 M spermidine were added to the mixture, and all of this was stirred by vortexing for a further 30 sec. The coating of the DNA onto the beads was continued by incubating the mixture in ice for 15 min,<br>
and then the coated beads were centrifuged at low speed for 5 sec and washed twice in absolute ethanol. The particles thus coated were resuspended in 32 pi of absolute ethanol, mixed vigorously by vortexing for 15 sec., and then immediately distributed as 4 identical aliquot fractions onto the sterile "microcarrier" discs of the PDS-1000/He Biolistic system which had been prepared according to the manufacturer's recommendations (Bio-Rad, Hercule, USA). The "microcarrier support/microcarrier bearing the particle deposit" set was dried for 5 min.<br>
6.3. Transient transformation of plant extracts by biolistics.<br>
6.3.1. Bombarding the maize albumens and transient expression.<br>
The bombarding of the maize albumens was carried out using the PDS-1000/He Biolistic system according to the general recommendations of the supplier (Bio-Rad, Hercule, USA) concerning the handling and assembly of the various elements of the equipment. Each albumen was bombarded twice successively with tungsten particles 0.6 urn in diameter, according to the following firing characteristics:<br>
-	the helium pressure chosen to accelerate the particles<br>
is 6200 kPa (900 psi),<br>
-	the plant sample is placed 6 cm from the bead<br>
acceleration zone,<br>
-	the firing is carried out under a reduced atmosphere of<br>
27 mm of mercury.<br>
Following the bombarding, the albumens were left in the same conditions and were incubated for 24 h in the dark in a culture chamber at 26°C, in order to allow the transient expression of the transgenes introduced into the cells.<br>
6.3.2. Bombarding the tobacco foliar tissues and transient expression.<br>
The bombarding of the tobacco leaves was carried out in<br>
the same way as the bombarding of the maize albumens, with two exceptions:<br>
-	the samples were bombarded with gold particles 1 um in<br>
diameter,<br>
-	the samples were placed 9 cm from the bead acceleration<br>
zone.<br>
After bombarding, the leaves were left in the same conditions and were incubated for 48 h in a culture chamber (constant temperature of 25°C, 16 h light/8 h darkness photoperiod), in order to allow the transient expression of the transgenes introduced into the cells.<br>
6.4. Revelation and evaluation of the activity of the various promoters by histochemical staining.<br>
6.4.1.	Revelation of fi-glucuronidase expression.<br>
A revelation of b-glucuronidase expression was carried out by histochemical staining as described by Jeffersson et al. (1987). Following the incubation period in a culture chamber, the plant extracts were incubated in the presence of the 15-glucuronidase substrate X-Gluc (500 mg/1 5-bromo-4-chloro-3-indolyl glucuronide), in 0.1 M phosphate buffer, 0.05% Triton X100, pH 7.0, for 24 h at 37°C.<br>
After staining, the maize albumens were directly analysed or conserved sterilely at 4°C for several weeks, whereas the tobacco leaves were depigmented by two successive passages through 95% ethanol baths for 3 and 12 h, respectively, and then rinsed in distilled water and dried flat between two cellophane sheets.<br>
The promoter activity of the various constructs was evaluated by estimating the number and intensity of the blue dots revealed on each plant extract.<br>
6.4.2.	Qualitative evaluation of the activity of the<br>
promoters in the maize albumen.<br>
The histochemical revelation of the b-glucuronidase expression made it possible to identify three categories of<br>
-	The albumens bombarded with the pMRT1197, pMRT1126,<br>
pMRT1127 and pMRT1199 construct systematically exhibit a number<br>
of blue dots which is lower than 10. The presence of blue spots<br>
on the albumens transformed with the pMRTl197 construct<br>
indicates that the 96-bp sequence of MPrll97 (SEQ.ID16)<br>
constitutes the minimum promoter sequence of the HMWG-Dx5<br>
promoter (SEQ.ID01), which is capable of directing basic<br>
transcriptional activity in the maize albumen. The absence of<br>
blue spots on the albumens bombarded with the pMPRT1144<br>
construct, which lacks promoter sequence (negative control),<br>
confirms the functionality of the promoters grouped together in<br>
this category.<br>
-	The albumens bombarded with the pMRT1128, pMRT1213,<br>
pMRT1216, pMRT1217, pMRT1136, pMRT1137, pMRT1135 and pMRT1138<br>
constructs exhibits on average a number of blue dots which is<br>
equivalent to the albumens transformed with the pMRT1125<br>
(PrHMWG-Dx5 (SEQ.ID01)) and pMRT1218 (Prg-zein, positive<br>
control) constructs.<br>
-	Finally, the albumens bombarded with the pMRT1130,<br>
pMRT1131, pMRT1139 and pMRT1200 constructs exhibit an intensive<br>
and diffuse blue staining which makes counting the number of<br>
blue dots difficult, but which leads to the suggestion that the<br>
MPrllSO (SEQ.ID05), MPrllSl (SEQ.ID06), MPrll39 (SEQ.ID13) and<br>
MPrl200 (SEQ.ID19) promoters are very highly active in the maize<br>
albumen 12 days after pollination.<br>
6.4.3. Quantitative evaluation of the activity of the promoters in the tobacco leaves.<br>
The results of the histochemical assays carried out on the tobacco leaves transformed with the pMRT1125 (PrHMWG-Dx5 (SEQ.ID01)), pMRT1130, pMRT1131, pMRT1133, pMRT1134, pMRT1135, pMRT1136, pMRT1137, pMRT1138 and pMRT1092 (CaMV PrD35S, positive control) constructs given in Figure 5 made it possible to classify the promoters studied in four categories. No blue spot<br>
observed on the leaves bombarded with the pMRT1125 (PrHMWG-Dx5 (SEQ.ID01)) construct. The leaves bombarded with the pMRT1133, pMRT1134, pMRT1136 and pMRT1137 constructs exhibit on average a number of blue dots which is between 50 and 100. The leaves transformed with the pMRT1135, pMRT1138 and pMRT1139 constructs (result not shown) exhibit a considerable number of blue dots, which is equivalent to that observed on the leaves bombarded with the reference construct pMRT1092 (CaMV PrD35S) . Finally, the leaves bombarded with the pMRT1130 and pMRT1131 constructs exhibit a much higher number of diffuse and intense blue spots than the leaves bombarded by the reference construct pMRT1092 (CaMV PrD35S) .<br>
In light of these results, several pieces of essential information can be derived:<br>
-	the CaMV 35S as-1 and as-2 activating sequences deregulate the<br>
activity of the HMWG-Dx5 promoter (SEQ.ID01) in the tobacco<br>
leaves,<br>
-	the CaMV 35S as-1 and as-2 activating sequences act<br>
synergistically with cis-regulating motifs present in the HMWG-<br>
Dx5 promoter (SEQ.ID01). The "G"-like box appears to be one of<br>
the key elements of this combinatorial control. The GATA boxes,<br>
in combination with the "G"-like box and the CaMV 35S as-1 and<br>
as-2 activating sequences, are perhaps responsible for the very<br>
high activity of MPrllSO (SEQ.ID05) and MPrllSl (SEQ.ID06),<br>
-	duplicating the CaMV 35S as-2 activating sequence does not<br>
confer a notable positive additional effect,<br>
-	the "cereal boxes", in combination with the CaMV 35S as-1 and<br>
as-2 activating sequences, appear to confer a negative<br>
transcriptional effect in the tobacco leaves.<br>
In conclusion, since the CaMV D35S promoter is commonly reported in the literature as being a chimeric promoter which provides an increase in the promoter activity of the GUS reporter gene which is about 8 to 12 times greater than the one provided by the CaMV 35S promoter (Kay et al., 1987), the<br>
MPrll35 (SEQ.ID09), MPrll38 (SEQ.ID12), MPrll39 (SEQ.ID13), MPrll30 (SEQ.ID05) and MPrll31 (SEQ.ID06) promoters are certainly the strongest chimeric promoters active in tobacco • leaves described to date.<br>
The MPrll33 (SEQ.ID07), MPrll34 (SEQ.ID08), MPrll36 (SEQ.ID10) and MPrll37 (SEQ.ID11) promoters, whose activity is weaker in tobacco leaves, also have an advantage, since they can direct the expression of resistance genes in order to allow the selection of transgenic plants, in the same way as the "nos"-type promoters for example.<br>
6.5. Quantification of the activity of the various promoters in the maize albumen by luminometric assay of B-glucuronidase expression.<br>
The albumens previously transformed by biolistics were frozen in liquid nitrogen and ground with the aid of a glass rod mounted on a drill. The powder was then thawed in extraction buffer (25 mM Tris Phosphate, pH 7.8, 2 mM dithiothreitol, 2 mM 1,2-diaminocyclohexane, N,N,N',N'-tetracetic acid, 10% glycerol, 1% Triton X100) in a proportion of 1 ml of buffer per 250 mg of tissue. The mixture was homogenized and then incubated for In at 4°C, before being clarified by centrifugation for 5 min at 16060<br>
g.<br>
The GUS activity was measured on 10 ul of clarified crude extract, with the aid of the "GUS-Light chemiluminescent reporter gene assay" detection kit (Tropix Inc., Bedford, USA) according to the manufacturer's recommendations. Measurement of light emission was carried out using a Lumat LB 9507 luminometer (EGG-Berthold, Bad Wildbad, Germany).<br>
In parallel, the luciferase activity was measured on 10 ul of clarified crude extract, with the aid of the "Luciferase assay system" detection kit (Promega Corp., Madison, USA) according to the manufacturer's recommendations. Measurement of light emission was carried out with the aid of the Lumat LB 9507 luminometer placed in a cold room at 4°C.<br>
The results are given in Figs. 6 and 7. For each assay (three half albumens = one crude extract), the ratio between the li-glucuronidase activity and the luciferase activity measured with the luminometer was calculated. The mean of at least 5 independent assays per construct and the standard error of the mean were determined.<br>
In order to analyse the effect of the various<br>
modifications brought to each of the promoters described in this patent, said promoters were divided up into two distinct groups. Group I (Fig. 6) consists of the promoters which make it possible to carry out a detailed functional dissection of the HMWG-Dx5 promoter (SEQ.ID01), and Group II (Fig. 7) contains the promoters which make it possible to determine the effect of the diverse cis-activating elements studied in this patent, in combination with the HMWG-Dx5 promoter (SEQ.ID01).<br>
Group I contains the MPrll28, MPrll27 (SEQ.ID03), MPrll26 (SEQ.ID02), MPrll97 (SEQ.ID16), MPrll98 (SEQ.ID17), MPrl216 (SEQ.ID21) and MPrl217 (SEQ.ID22) promoters, the HMWG-Dx5 reference promoter (SEQ.ID01) and the reference construct pMRT1144 (negative control), this construct lacking promoter sequence (Fig. 6). The results of the luminometric assays make it possible to derive several observations:<br>
-	the gradual deletions of the 5' region of the HMWG-Dx5<br>
promoter (SEQ.ID01) leads to a gradual decrease in the mean<br>
relative activity conferred by these promoters. The whole 417-bp<br>
PrHMWG-Dx5 promoter sequence (SEQ.ID01) thus appears to be<br>
required in order to allow maximum activity of the HMWG-Dx5<br>
promoter (SEQ.ID01).<br>
-	The slight difference in activity recorded between the<br>
MPR1128 (SEQ.ID04) and PrHMWG-Dx5 (SEQ.ID01) promoters shows<br>
that the sequence located upstream of nucleotide -238 does not<br>
contain the major cis-activating elements which are responsible<br>
for the activity of the HMWG-Dx5 promoter (SEQ.ID01).<br>
-	The significant decrease in activity between the MPR1128<br>
(SEQ.ID04) and MPrll27 (SEQ.ID03) promoters leads to the suggestion that the sequence located upstream of nucleotide -205, which contains a "G"-like box, plays a role which is capital for the activity of the HMWG-Dx5 promoter (SEQ.ID01).<br>
-	The slight difference in activity recorded between the<br>
MPrll27 (SEQ.ID03) and MPrll26 (SEQ.ID02) promoters does not<br>
allow a conclusion to be drawn regarding the real role of the<br>
enhancer element.<br>
-	The activity of the MPrll97 (SEQ.ID16) promoter, which<br>
is very weak but stronger than that obtained with the pMRT1144<br>
construct which does not contain a promoter, indicates that the<br>
96-bp minimum PrHMWG-Dx5 (SEQ.ID01) sequence confers a basic<br>
transcription level.<br>
-	The weak activity of MPrll98 (SEQ.ID17), compared with<br>
that of MPR1128 (SEQ.ID04), indicates that the PrHMWG-Dx5<br>
(SEQ.ID01) sequence stretching from nucleotide -59 to nucleotide<br>
-135, although lacking putative cis-activating elements, plays a<br>
predominant role in the activity of the HMWG-Dx5 promoter<br>
(SEQ.ID01). This result implies that the functionality and, consequently, the accessibility of the trans-activating factors on the "G" box and on the activating element, depend on the distance which separates them from the TATA box.<br>
-	Duplicating the MPR1128 (SEQ.ID04) sequence stretching<br>
from nucleotide -136 to nucleotide -225, which harbours the "G"<br>
box and the activating element, confers on the MPrl216 promoter<br>
(SEQ.ID21) b-glucuronidase activity which is at least as high as that directed by PrHMWG-Dx5 (SEQ.ID01). Conversely, triplicating this same sequence in the MPrl217 promoter (SEQ.ID22) has no additional additive effect.<br>
Group II contains the MPrll28, MPrl213 (SEQ.ID20), MPrll99 (SEQ.ID18), MPrll36 (SEQ.ID10), MPrll37 (SEQ.ID11), MPrll38 (SEQ.ID12), MPrll31 (SEQ.ID06), MPrll35 (SEQ.ID09), MPrll30 (SEQ.ID05), MPrll39 (SEQ.ID13) and MPrl200 (SEQ.ID19) promoters, the HMWG-Dx5 reference promoter (SEQ.ID01) and the g-zein<br>
.promoter which is used as a positive control (Fig. 7). In the same way as for the promoters of group I, the results of the luminometric assays make it possible to derive several observations:<br>
-	Fusing the CaMV 35S as-2 (Lam and Chua, 1989) and as-1<br>
(Lam et al., 1989) activating sequences at position -65 bp of<br>
the HMWG-Dx5 promoter (SEQ.ID01) and of the HMWG-Dx5 promoter<br>
(SEQ.ID01) derivatives very greatly potentiates the activity of<br>
the resulting MPrllSO (SEQ.ID05), MPrll31 (SEQ.ID06), MPrll36<br>
(SEQ.ID10), MPrll37 (SEQ.ID11), MPrll35 (SEQ.ID09) and MPrll38<br>
(SEQ.ID12) promoters. By way of indication, the MPrllSl<br>
(SEQ.ID06) and MPrll30 (SEQ.ID05) promoters are, respectively,<br>
3.2 and 3.8 times more active than the HMWG-Dx5 promoter<br>
(SEQ.ID01).<br>
-	The CaMV 35S as-2 and as-1 activating sequences act<br>
synergistically with the cis-activating elements of the HMWG-Dx5<br>
promoter (SEQ.ID01). The gradual decrease in the activity of the<br>
MPrllSl (SEQ.ID06), MPrll38 (SEQ.ID12), MPrll37 (SEQ.ID11) and<br>
MPrll36 (SEQ.ID10) promoters, which coincides, respectively,<br>
with the gradual 5' deletions of the HMWG-Dx5 promoter<br>
(SEQ.ID01), confirm this.<br>
-	The comparison of the MPrllSO (SEQ.ID05) and MPrll35<br>
(SEQ.ID09) promoters with the MPrllSl (SEQ.ID06) and MPrll38<br>
(SEQ.ID12) promoters, respectively, indicates that duplicating<br>
the CaMV 35S as-2 activating sequence does not engender a<br>
significant additive activating effect.<br>
-	Fusing the "cereal boxes" of the promoter of the high<br>
molecular weight glutenin gene encoding the Bx7 subunit of the<br>
hexaploid wheat Triticum aestivum L.cv Cheyenne (Anderson et<br>
al., 1998), upstream of the MPrll31 (SEQ.ID06) and MPrll38<br>
(SEQ.ID12) promoters very greatly improves the activity of the<br>
resulting MPrll39 (SEQ.ID13) and MPrl200 (SEQ.ID19) promoters.<br>
By way of indication, the activity of MPrll39 (SEQ.ID13) and of<br>
MPrl200 (SEQ.ID19) is approximately 5.5 times higher than thatof the HMWG-Dx5 promoter (SEQ.ID01).<br>
-	The weak activity of the MPrl213 promoter (SEQ.ID20),<br>
which corresponds to the fusion of the "cereal boxes" upstream<br>
1 of the MPrll28 promoter, implies that the "cereal boxes" act synergistically with the CaMV 35S as-2 and as-1 activating sequences so as to potentiate the activity of the MPrll39 (SEQ.ID13) and MPrl200 (SEQ.ID19) promoters.<br>
-	Finally, fusing the "GC-rich" element of the intergenic<br>
region of the maize streak virus (MSV) upstream of the MPR1128<br>
(SEQ.ID04) promoter does not contribute to increasing the<br>
activity of the resulting MPrll99 promoter (SEQ.ID18). On the<br>
contrary, the "GC-rich" element appears to confer a slightly<br>
inhibitory effect.<br>
In conclusion, the MPrll39 (SEQ.ID13) and MPrl200 (SEQ.ID19) promoters, since they are, respectively, 4 and 3.9 times more active than the Prg-zein promoter, which is commonly used in plant biotechnology to direct protein expression at high levels, are unquestionably powerful tools capable of improving the level of expression of heterologous proteins in the maize albumen. Moreover, it is to be noted that the MPrll30 (SEQ.ID05), MPrllSl (SEQ.ID06), MPrll35 (SEQ.ID09), MPrll38 (SEQ.ID12), MPrll37 (SEQ.ID11) and MPrll36 (SEQ.ID10) promoters confer b-glucuronidase activity in the maize albumen which is at least as great as that obtained with the Prg-zein promoter. Finally, the less effective promoters are also very valuable, since they can be used to provide the control of the expression of resistance genes or of genes encoding enzymatic proteins. Example 7.<br>
Expression and evaluations of the activity of the various promoters in stable expression in maize and tobacco.<br>
7.1. Stable gene transformation of maize with Agrobacterium tumefaciens.<br>
The technique used is described by Ishida et al. (1996). Immature embryos 1.0 to 1.2 mm in length (9 to 14 days after<br>
pollination) were washed in LS-inf medium, then immersed in the agrobacteria suspension, prepared as described by Ishida et al. (1996), vortexed for 30 sec., and incubated at room temperature for 5 min. The immature embryos thus treated were cultivated on LS-AS medium in the dark at 25°C for 3 days, then transferred onto LSD 1.5 medium supplemented with phosphinotricine at 5 mg/1 and cefotaxime at 250 mg/1, in the dark at 25°C for 2 weeks and, finally, placed on LSD 1.5 medium supplemented with phosphinotricine at 10 mg/1 and cefotaxime at 250 mg/1, in the dark at 25°C for 3 weeks. The type I calluses thus generated were isolated, fragmented and transferred onto LSD 1.5 medium supplemented with phosphinotricine at 10 mg/1 and cefotaxime at 250 mg/1, in the dark at 25°C for 3 weeks. Then, the type I calluses, which had proliferated, were isolated and placed on LSZ medium supplemented with phosphinotricine at 5 mg/1 and cefotaxime at 250 mg/1, under a 16 hours light/8 hours darkness photoperiod at 25°C for 2 to 3 weeks. The regenerated plantlets were then transferred onto LSF 1/2 medium under a 16 hours light/8 hours darkness photoperiod at 25°C for 1 to 2 weeks, and then to a phytotron and to a greenhouse.<br>
7.2. Stable gene transformation of tobacco with Agrobacteriurn tumefadens.<br>
The transformation of the tobacco {Nicotiana tabacum L., of the PBD6 cultivar) was carried out by infecting foliar discs isolated from 6-week-old tobacco plantlets in vitro, with recombinant agrobacteria according to the method described by Horsch et al. (1985).<br>
All the in vitro cultures are prepared in an air-conditioned area in which the light intensity is 200 iaE.m-2.s-l/ the photoperiod is 16 hours light/8 hours darkness, and the temperature is 25°C.<br>
Except for the initial coculturing step, the regeneration, development and rooting steps were carried out on diverse selective media supplemented with a bacteriostatic agent, namely<br>
augmentin at 400 mg/1, and with a selective agent, namely kanamycin at 200 or 100 mg/1.<br>
The various steps and media used are as follows:<br>
-	After preculturing the agrobacteria in 5 ml of 2YT medium (10<br>
g/1 bactotryptone, 5 g/1 yeast extract, 5 g/1 NaCl, pH 7.2)<br>
supplemented with 6 mM final CaCl2 and with suitable antibiotics,<br>
at 28°C for 48 hours, a culture in 10 ml of 2YT medium<br>
supplemented with CaCl2 and antibiotics is prepared at 28°C<br>
overnight. The culture is then centrifuged at 3000 g for 10 min<br>
and the bacteria are resuspended in 10 ml of liquid MS30 (4.4<br>
g/1 M0404 sold by SIGMA, supplemented with 30 g/1 sucrose, pH<br>
5.7) .<br>
-	The coculturing is carried out by placing the approximately 1<br>
cm2 foliar explants, which have been cut out from in vitro<br>
plantlet leaves, in contact with the agrobacteria suspension<br>
diluted 10-fold in liquid MS30, for 20 min. Then, the explants<br>
thus treated are rapidly dried on filter paper and placed on a<br>
solid coculture medium (CM) (MS30, benzylaminopurine (BAP) at 1<br>
mg/1, indole-3-acetic acid (IAA) at 0.1 mg/1, agar-agar at<br>
8 g/1) for 48 hours in the air-conditioned area.<br>
-	The treated explants are then placed on a solid regeneration<br>
medium (solid CM, augmentin at 400 mg/1, kanamycin at 200 mg/1).<br>
The explants are subcultured on the same medium after 2 weeks.<br>
-	After 2 weeks, the buds are subcultured on a solid development<br>
medium (4.4 g/1 M0404 sold by SIGMA, supplemented with 20 g/1<br>
sucrose, pH 5.7 (liquid MS20), augmentin at 400 mg/1, kanamycin<br>
at 100 mg/1, agar-agar at 8 g/1).<br>
-	After 2 weeks, the transformed plantlets are subcultured on<br>
solid rooting medium which is identical to the development<br>
medium. The rooting lasts 2 to 3 weeks, at the end of which the<br>
plantlets are removed to the phytotron in jiffy pots for 10 days<br>
(16 hours light/8 hours darkness photoperiod, 23°C and 70% humidity), and then transferred to a greenhouse.<br>
7.3. Measurement of B-glucuronidase activity in the maize<br>
and tobacco plants.<br>
To measure the 15-glucuronidase activity, the samples taken from the transgenic plants were frozen in liquid nitrogen and ground with the aid of a glass rod mounted on a drill. The powder was then resuspended in extraction buffer (25 mM Tris Phosphate, pH 7.8, 2 mM dithiothreitol, 2 mM 1,2-diaminocyclohexane, N,N,N',N'-tetracetic acid, 10% glycerol, !%• Triton X100) in a proportion of 1 ml of buffer per 250 mg of tissue. The mixture was homogenized and then incubated for Ih at 4°C, before being clarified by centrifugation for 5 min at 16060<br>
g.<br>
The GUS activity was measured on 10 ul of clarified crude extract, with the aid of the "GUS-Light chemiluminescent reporter gene assay" detection kit (Tropix Inc., Bedford, USA) according to the manufacturer's recommendations. Measurement of light emission was carried out using a Lumat LB 9507 luminometer (EGG-Berthold, Bad Wildbad, Germany).<br>
The amount of total protein present in the crude extract was measured according to the Bradford technique (1976), using the "Bio-Rad protein assay" reagent (Bio-Rad, Munich, Germany).<br>
7.4. Stable Expression and Chimeric Promoter Activity in Maize Endosperm and Leaves.<br>
7.4.1. Expression in Seeds.<br>
The p-glucuronidase activity controlled by the chimeric HMWG promoters in stable expression in maize endosperm was compared to controlled by the reference promoter 512 gamma-zein, which is known to be highly active in maize albumen (Marzabal et al., 1998) . Six seeds per cob were studied, taken starting from the apex of the cob and proceeding towards its base, at different stages of growth. As an indication, the 30 DAP stage corresponds to maize seeds taken 30 days after pollinisation. The luminometric amounts of p-glucuronidase activity were determined for each seed according to the method described in section 7.3<br>
of example 7.<br>
The results as reported in Figure 8 refelect the P-glucuronidase activity under the control of the chimeric HMWG-• Dx5 derived promoters (MPrll39, MPrl200 and MPrllSl) and the reference promoter 512 gamma-zein, during stable expression in mature corn seeds, harvested at 30 DAP. The comparison of the activities of each population of plants gives a good indication of the respective strength of the different promoters. The P-glucuronidase activity under the control of promoters MPrll39, MPrl200 andn MPrllSl is on average of the order of 1.5 to 2 times as great as that under the control of the reference promoter 512 gamma-zein. Nonetheless, the seeds of plants 302.A3 and 347.H1, which respectively express the GUS protein under the control of the promoters MPrll39 and MPrllSl, show a P-glucuronidase activity of 7 and 14 times respectively that of the activity controlled by the reference promoter 512 gamma-zein. No significant difference in p-glucuronidase activity was noted in plants expressing the GUS protein under the control of promoters MPrll39, MPrl200 et MPrll3l. However, p-glucuronidase activity varies considerably in each population of plants. This phenomena, which has already been observed in the majority of genes introduced into plants, can be explained by positioning effects of the transgene and copy number. The luminometric determinations carried out at the 13 and 18 DAP stages of development (results not shown) indicate that the p-glucuronidase activity varies over time, but that the promoters responsible for the highest P-glucuronidase activity at the 30 DAP stage are also the strongest promoters in the earlier stages of development, at 13 and 18 DAP respectively. Thus, the classification of the promoters is maintained during development. The histogram shown in Figure 9 shows temporal fluctuations in p-glucuronidase activity under the control of the promoter MPrll39. These results indicate the the GUS activity is<br>
detectable from 10 DAP, reaches a plateau between 16 and 28 DAP and declines thereafter up to 30 DAP. The GUS activity plateau, obtained from plants taken during the summer period, was also observed for the period of development between 12 and 20 DAP (results not shown). The histochemical tests carried out on longitudinal sections of corn seeds taken at 13 DAP (Figure lOa) , 18 DAP (Figure lOb) or on dissected maize seeds (Figure lOc) indicate that the promoter MPrll39, in maize seed, is specifically expressed in the albumen, no staining having been detected in the embryo, the aleurone or the pericarp. Furthermore, the histograms illustrated in Figure 11 indicate that the expression of MPrll39 is stable, or even greater in the second generation (T2).<br>
From the preceding data, the following information can be summarized :<br>
the activating sequences as-1 and as-2 of the CaMV 35S promoter, fused to the promoter sequence HMWG-Dx5, appear to be very strong cis-activating elements in maize albumen.<br>
the activating sequences as-1 and as-2 of the CaMV 35S promoter do not deregulate the activity of the HMWG-Dx5 promoter in maize seeds (Figure 10) .<br>
-	the cereal boxes do not have a significant cis-regulatory<br>
effect in stable expression in corn seeds, the promoter MPrll31<br>
being at least as active as the promoter MPrll39.<br>
-	the promoter sequence HMWG-Dx5 located upstream of the "G"<br>
box, stretching from nucleotides -238 to -378pb, does not play a<br>
a key regulatory role within the the chimeric promoters derived<br>
from HMWG, the promoter MPrl200 being at least as active as the<br>
promoter MPrll39 in stable expression of maize seeds.<br>
In conclusion, the chimeric promoters derived from HMWG (MPrll39, MPrllSl and MPrl200), being on average roughly 1.5 to 2 times as active or even greater for the best expressors 302.A3 and 347.HI, as the reference promoter 512 gamma-zein, are incontestably exceedingly useful tools capable of improving the<br>
expression  of  heterologous  proteins  in  maize  albumen.  The<br>
r<br>
chimeric promoters derived from HMWG according to the present invention can also be used to over-express endogenous proteins in monocotyledonous plant seeds, thereby representing an interest for agriculture for example, for the production of rice or wheat, in relation to starch production. 7.4.2 Stable Expression in Leaves.<br>
The p-glucuronidase activity under the control of promoters MPrll39, MPrllSl and MPrl200 was determined by stable expression in maize leaves. The luminometric determinations of 0-glucuronidase activity were carried out according to the method described in section 7.3 of example 7 from two leaf disks, each two centimeters in diameter, taken at 3 weeks after acclimatisation in a greenhouse from maize plants.<br>
The comparison of the activities for each population of plants indicates that the (3-glucuronidase activity controlled by the chimeric promoters MPrll39, MPrl200 and MPrllSl is at the most 30 times greater than the background noise measured in plants expressing the GUS protein under the control of the 512 gamma-zein promoter (Figure 12), with the result that the promoters MPrll39, MPrl200 and MPrllSl are slightly or not at all active in the leaves of maize plants at the three week development stage after acclimatization in a greenhouse. Nevertheless, the histochemical tests carried out on the leaves of the primary transformants (plantlets) expressing the GUS protein under the control of the chimeric promoters derived from HMWG, during rooting in in vitro cultivation, systematically reveals a blue staining (results not shown).<br>
These results are very interesting in that the activity of the promoters MPrll39, MPrl200 and MPrllSl is low but sufficient for carrying out early tests in the leaves of transgenic maize, without any major risks of toxicity.<br>
7.5. Stable Expression Activity of Chimeric Promoters in<br>
Tobacco Leaves and Seeds.<br>
7.5.1. Stable Expression in Leaves.<br>
The stable expression p-glucuronidase activity under control of the promoters MPrllSO, MPrll31, MPrll35, MPrll38 and MPrll39 was compared to that controlled by the CaMV D35S promoter, in tobacco leaves. The luminometric measurements of the P-glucuronidase activity were carried out according to the method described in section 7.3 of example 7 from two leaf disks each two centimeters in diameter, taken from different leaves located at the base of the upper third of the primary transformants, at the 2, 5, 8 and 11 week stages of development after acclimatization in a greenhouse. In order to limit the variations in the degree of expression of the reporter gene, introduced mainly by random integration and the number of copies of the expression cassette, 10 to 30 independent transformants were studied for each construction.<br>
The results illustrated in Figure 13 reflect the stable expression p-glucuronidase activity under the control of the chimeric promoters derived from HMWG and the reference promoters HMWG-Dx5 and CaMV D35S in tobacco leaves, 11 weeks after acclimatization in a greenhouse. The comparison of the activities of each plant population provides a good indication of the respective strength of the different promoters. The p-glucuronidase activity controlled by the chimeric promoters of the present invention derived from HMWG is significantly greater than that measured under the control of the HMWG-Dx5 promoter, but roughly 5 to 10 times lower than that under the control of the CaMV D35S promoter. Amongst the different HMWG chimeric promoters, no significant difference in activity was observed, except for the promoter Mprll39, which was slightly less active. The luminometric determinations made at the 2, 5 and 8 week development stage after acclimatization in a greenhouse (results not shown) indicate that the p-glucuronidase activity increases over time in any given plant, irrespective of the promoter used.<br>
However,  the strongest promoters at the 11 week development<br>
lr<br>
stage also confer the highest p-glucuronidase activity at earlier stages of development (2, 5 and 8 weeks after acclimatization in a greenhouse). Thus, the classification of the promoters at the 11 week stage also applies to all the other stages of development in tobacco.<br>
It is apparent from this data that the chimeric promoters derived from HMWG are functional but only weakly active in stable expression in tobacco leaves. The applicant concludes that the activating sequences as-1 and as-2 deregulate the activity of the HMWG-Dx5 promoters in tobacco leaves, but do not confer a strong activating effect in association with the cis-regulatory elements present in the HMWG-Dx5 promoter sequence.<br>
In order to explain this apparent contradiction in these results with those obtained in transient expression experiments, two hypotheses were raised :<br>
-	the chimeric promoters derived from HMWG are induced by injury<br>
and/or  stress  during  biolistic  transformation  of  tobacco<br>
leaves ;<br>
-	in stable tobacco expression, the chimeric promoters derived<br>
from HMWG are essentially expressed in the every early stages of<br>
development. The histochemical tests carried out on the leaves<br>
of the primary tobacco transformants in vitro (plantlets) during<br>
the regeneration step indicate high p-glucuronidase activity of<br>
the chimeric promoters derived from HMWG (results not shown).<br>
The chimeric promoters derived from HMWG, i.e. MPrll30, MPrllSl, MPrll35, MPrll38 and MPrll39, although weakly active in stable expression of tobacco leaves, can be used for example for controlling the expression of an enzyme implicated in a biosynthetic pathway of the metabolism of the plant. They can also be used to control the expression of genes conferring a resistance to the plant, for example, a resistance to an antibiotic or a herbicide, useful as a selection agent. 7.5.2. Stable Expression in Tobacco Seeds.<br>
The p-glucuronidase activity was determined by luminometry in mature Tl tobacco seeds (100 mg per transformant) taken from 10 to 30 separate primary transformants obtained independently for each construction. The activity varies from plant to plant within a given construct (cf. Figure 14), which can be explained by positioning effects and the transgene copy number in the genome.<br>
The results show that certain chimeric promoters derived from HMWG can promote p-glucuronidase activity in tobacco seeds at least as highly as the CaMV D35S promoter. Indeed, the promoters can be classified as follows :<br>
-	the promoters MPrll3O, MPrllSl and Mprll39, which promote p-<br>
glucuronidase activity to the same extent as the CaMV D35S<br>
promoter ;<br>
-	the promoters MPrll35 and MPrll38, which are twice as active<br>
in tobacco seeds as the CaMV D35S promoter.<br>
The results obtained indicate that :<br>
the activating sequences as-1 and as-2 of the CaMV 35S promoter confer an important cis-activating effect in a promoter sequence derived from HMWG containing only a "G -like" box and the "enhancer" element, as observed for promoters MPrll35 and MPrll38. Thus, the activating sequences as-1 and as-2 of the CaMV 35S promoter act in synergy with the cis-regulatory elements present in the HMWG-Dx5 promoter. The "G-like" box and the "enhancer" element seem to be the key elements in this combinatory control in tobacco seeds.<br>
-	the HMWG-Dx5 promoter sequence located upstream of the "G"<br>
box,  stretching  from nucleotides  -378  to  -238,  confers  a<br>
negative cis-regulatory effect to the promoters MPrllSO, MPrllSl<br>
and MPrll39 in tobacco seeds.<br>
-	the duplication of the activating sequence as-2 of the CaMV<br>
promoter does not confer any notable positive additive effect in<br>
tobacco seeds. Indeed, no difference in activity is obtained for<br>
MPrllSO with respect to MprllSl, nor for MPrll35 with respect ot<br>
MPrll38.<br><br>
- the "cereal" boxes do not confer any notable additive cis-activating effect in tobacco seeds, since the results obtained for the promoters MPrll3l and MPrll39 are similar.<br>
the promoters MPrll35 and MPrll38, which are highly active in stable expression in tobacco seeds, are powerful tools for controlling the expression of heterologous proteins in dicotyledonous plants, for example, such as in those plants of high agronomic interest.<br>
Example 8.<br>
Construction of the binary plasmid pMRT1231 including the HMV7G-Dx5 promoter (SEQ.ID01), used for stable transformation of tobacco.<br>
The binary plasmid pMRT1231 was obtained by insertion of the expression cassette "HMWG-Dx5 (SEQ.ID01) / uidA-IV2 I term-nos" from pMRT1125 into the resriction site Hpal from the binary plasmid pMRT1195. This is described in French patent application FR9911112, to be published, incorporated herein by reference with respect to the relevant passages.<br>
In order to do this, 1 ug of plasmid pMRT1125 were digested successively by EcoRI and XmnI for 1 h at 37°C. The expression cassette was then isolated on 0.7% agarose gel using a "Concert Rapid Gel Extraction System" kit and subjected to the action of 20 Units of Klenow fragment (New England Biolabs) for 30 min. at 37°C in the presence of 60 nanomoles of each of the dNTPs, 12 pi of MgC12 (500 mM) and 6 ul of DTT (1M).<br>
In parallel, 5 ug of binary plasmid pMRT1195 were digested by Hpal for 1 h at 37°C. The linearized vector frgament was then dephosphorylated by 40 Units of calf intestine alkaline phosphatase (New England Biolabs) in the presence of 3 buffer at 37°C for Ih.<br>
The ligation was carried out with 100 ng of the expression cassette and 10 ng of pMRT1195 plasmid as obtained above, by a succession of PCR cycles in a "GeneAmp PCR System 9700" thermocycler as described previously. Previously prepared competent Escherichia coli DH5a, were transformed with all of the ligation reaction mixture. The plasmid DNA of the obtained clones, selected on LB media supplemented with kanamycine (50 mg/1), was extracted according to the alkaline lysis method and analyzed by enzymatic digestion.<br>
The plamsid obtained, designated pMRT1231, was then transferred as described previously in section 4.1 of example 4, into the strain Agrobacterium tumefaciens LBA4404-pSBl, which strain derives from Agrobacterium tumefaciens LBA4404 after integration of the pSBl plasmid according to the protocol described recently for obtaining the strain A1177. This is described in French patent application FR9911112, to be published, incorporated herein by reference for the relevant passages. The plasmid DNA of the obtained clones, selected on 2YT media supplemented with rifampicine (50 mg/1), Kanamycine (50 mg/1) and tetracycline (5 mg/1), was extracted according to the alkaline lysis method, modified by adding lysozyme (25 mg/ml) to the cell resuspension buffer. The plasmid DNA obtained was analyzed by enzymatic digestion and the agrobacteria obtained designated A1231.<br>
The stable genetic transformation of tobacco was carried out as described in section 7.2 of example 7 except that the selection agent used in the regeneration and development media is glufosinate at 0,5 and 2mg/l respectively. Example 9.<br>
Construction of the binary plasrnids including the promoters MPrllSl, MPr l200 and 512 gamma-zein, used for stable genetic transformation of maize.<br>
9.1. Construction of binary plasmid pMRT1263.<br>
The binary plasmid pMRT1263 was obtained by insertion of the<br>
expression cassette "MPrll31 (SEQ. ID06) / uidA-IV2 / term-nos" into the restriction site Hpal of the binary plasmid pMRT1195. This plasmid is described in French patent application „ FR9911112, not yet published, and incorporated herein by<br>
reference for the relevant passages. The insertion was carried out in example 5, except that the expression cassette was isolated from plasmid pMRT1131, described in section 3.2 of example 3.<br>
The resulting plasmid was designated pMRT1263 and was transferred into the strain Agrobacterium tumefaciens LBA4404-pSBl according to the protocol described previously in section 5.1 of example 5. The agrobacteria clone obtained was designated A1263.<br>
9.2.	Construction of the binary plasmid pMRT1266.<br>
The binary plasmid pMRT1266 was obtained by insertion of the expression cassette "MPrl200 (SEQ. ID19) / uidA-IV2 / term-nos" into the Hpal restriction site of binary plasmid pMRT1195. This is described in French patent application FR9911112, not yet published, the text of the relevant passages of which is hereby incorporated by reference. The insertion was carried out as described in example 5, except that the expression cassette was isolated from plamsid pMRT1200, described in section 3.10 of example 3.<br>
The resulting plasmid was designated pMRT1266 and was transferred into the strain Agrobacterium tumefaciens LBA4404-pSBl according to the protocol described previously in section 5.1 of example 5. The agrobacteria clone obtained was designated A1266.<br>
9.3.	Construction of binary plasmid pMRT1209.<br>
In order to have a reference promoter sequence in stable expression in maize albumen SN 87 165 (L2), the uidA gene under the control of the promoter 512 gamma-zein and the nos terminator, contained in the 526 gamma-zein plasmids described by Marzabal et al. (1998), was inserted into binary plasmid<br><br>
jpMRTH95, as described previously in example 5, except that the expression cassette was isolated from 526 gamma-zein plasmid.<br>
The resulting plasmid pMRT1209 was transferred into the stain Agrobacterium tumefaciens LBA4404-pSBl according to the protocol in section 5.1 of example 5. The agrobacteria clone obtained was designated A1209.References cited;<br>
Anderson O.D., Green F.C., Yip R.E., Halford N.G., Shewry P.R.<br>
and Malpica-Romero J-M. (1989). Nucleotide sequences of the two<br>
high-molecular-weight glutenin genes from the D-genome of a<br>
hexaploid bread wheat, Triticum aestivum L. cv Cheyenne. Nucleic<br>
Acids Research 17, 461-462.<br>
Anderson O.D., Abraham-Pierce F.A. and Tarn A. (1998).<br>
Conservation in wheat high-molecular-weight glutenin gene<br>
promoter sequences : comparisons among loci and among alleles of<br>
the GLU-B1-1 locus. Theor Appl Genet 96, 568-576.<br>
Birnboim H. C. and Doly J. (1979). A rapid alkaline extraction<br>
procedure for screening recorabinant plasmid DNA. Nuc. Ac. Res.<br>
7, 1513.<br>
Bradford M. (1976). A rapid and sensitive method for the<br>
detection of microgram quantities of proteins utilizing the<br>
principle of protein-dye binding. Anal. Biochem. 72, 248-254.<br>
Fenoll C., Schwarz J.J., Black D.M., Schneider M. and Howell<br>
S.H. (1990). The intergenic region of maize streak virus<br>
contains a GC-rich element that activates rightward<br>
transcription and binds maize nuclear factors. PMB 15, 865-877.<br>
Guerineau and Mullineaux (1993). In Plant molecular biology<br>
labfax, Croy R.R.D. (Ed.), BioS Scientific Publishers, Blackwell<br>
Scientific Publications.<br>
Jefferson R.A., Burgess S.M. and Hirsh D (1986). fl-glucuronidase<br>
as a gene-fusion marker. Proc. Nat. Acad. Sci. USA, 83, 8447-<br>
8451.<br>
Jefferson R.A., Kavanagh T.A. and Bevan M.W. (1987). GUS<br>
fusions: ft-glucuronidase as a sensitive and versatile gene<br>
fusion marker in higher plants. EMBO J. 6, 3901-3907.<br>
Holsters M., Dewaele D., Depicker A., Messenf E., Van Montagu M.<br>
and Schell J. (1978). Transfection and transformation of<br>
Agrobacterium tumefaciens. Mol. Gen. Genet. 136, 181-187.<br>
Horsch R.B., Fry J.E., Hoffmann N.L., Eiholtz D., Rogers S.G.<br>
and Fraley R.T. (1985). A simple and general method for<br>
transferring genes into plants. Science 227, 129-1231.<br>
Kay R., Chan A., Daly M. and McPherson J. (1987). Duplication of<br>
CaMV 35S promoter sequences creates a strong enhancer for plant<br>
genes. Science 236, 1299-1302.<br>
Lam E., Benfey P. N., Gilmartin P. M., Fang R. X. and Chua N. H.<br>
(1989) . Site-specific mutations alter in vitro factor binding<br>
and change promoter expression pattern in transgenic plants.<br>
Proc. Natl. Acad. Sci. USA, 86, 7890-7894.<br>
Lam E. and Chua N. H. (1989) . ASF-2 : a factor that binds to the<br>
cauliflower mosaic virus 35S promoter and a conserved GATA motif<br>
in Cab promoters. Plant Cell, 1, 1147-1156.<br>
Leckie L., Devoto A. and De Lorenzo G. (1994). Normalisation of<br>
GUS by LUC activity from the same cell extract reduces<br>
transformation variability. Biotechniques 17, 52-56.<br>
Murashige T. and Skoog F. (1962). A revised.medium for rapid<br>
growth and bioassays with tobacco tissue cultures. Physiol.<br>
Plant. 15, 473-497.<br>
Reina M., Ponte I., Guillen P., Boronat A. and Palau J. Sequence<br>
analysis of a genomic clone encoding a Zc2 protein from Zea mays<br>
W64 A. Nucleic Acids Research 18, 6426.<br>
Torrent M., Alvarez I., Geli M.I., Dalcol I. and Ludevid D.<br>
(1997). Lysine-rich modified g-zein accumulate in protein bodies<br>
of transiently transformed maize endosperms. Plant Mol. Biol.<br>
34, 139-149.<br>
Vancanneyt G., Schmidt R., O'Connor-Sanchez A., Willmitzer L.<br>
and Rocha-Sosa M. (1990). Construction of an intron-containing<br>
marker gene: Splicing of the intron in transgenic plants and its<br>
use in monitoring early events in Agrobacterium-mediated plant<br>
transformation. Mol. Gen. Genet. 220, 245-250.<br>
Marzabal P., Busk P. K., Ludevid M. D. et Torrent M. (1998). The<br>
bifactorial endosperm box of g-zein gene : characterisation and<br>
function of the Pb3 and GZM cis-acting elements.  The Plant<br>
Journal 16 (1), 41-52.<br><br><br><br><br><br><br>
WE CLAIM<br>
1.	A process for construction of a chimeric promoter of expression, said<br>
process comprising a series of insertions of at least one nucleic acid sequence<br>
into a nucleic acid derived from a gene encoding a high molecular weight wheat<br>
glutenin   according  to   SEQ.ID01,   and   wherein   the   inserted   nucleic   acid<br>
sequences comprise regulatory or functional boxes of the kind such as herein<br>
described; and performing the conventional steps of amplification and ligation<br>
to prepare the chimeric promoter.<br>
2.	The  process  as  claimed  in  claim   1   wherein  the  resulting  chimeric<br>
promoter has a nucleic acid sequence consisting of the sequences chosen from<br>
sequences identified under the numbers SEQ. ID05, SEQ. ID06, SEQ. ID07,<br>
SEQ. ID08, SEQ. ID09, SEQ. ID10, SEQ. ID11, SEQ. ID14, SEQ. ID15, SEQ.<br>
ID16, SEQ. ID17,  SEQ.ID35, SEQ.ID36, and SEQ.ID37.<br>
3.	The process as claimed in claim 1, wherein said process comprises<br>
inserting at least one "enhancer" box functionally linked in the 5' position<br>
upstream of the 'TATA" box and the transcription start site (+1).<br>
4.	The process as claimed in claim  1, wherein said process comprises<br>
inserting at least on "G-like" box functionally linked in the 5' position upstream<br>
of the "enhancer" box.<br>
5.	The process as claimed in claim 1, wherein it also comprises inserting at<br>
least on "P-like" box functionally linked in the 5' position upstream of the<br>
"enhancer" box.<br>
6.	The process as claimed in claim 1, wherein it also comprises inserting at<br>
least one "GATA" box functionally linked in the 5' position the 5' position<br>
upstream of at least the "enhancer" box.<br>
7.	The process as claimed in claim 1, wherein it also comprises inserting at<br>
least one cereal box functionally linked in the 5' position upstream of the<br>
"enhancer" box.<br>
8.	The process as claimed in claim 1, wherein it also comprises inserting<br>
two contiguous "cereal" boxes functionally linked in the 5' position upstream of<br>
the "enhancer" box.<br>
9.	The process as claimed in claim 1, wherein it also comprises inserting at<br>
least one "asl" box or at least one "as2" box, or an "asl/as2" or "as2/asl"<br>
combination of boxes, or repeat permutations of these, functionally linked in<br>
the 5' position upstream of the transcription start site.<br>
10.	The process as claimed in claim 1, wherein it also comprises inserting<br>
the   "asl",   "as2"   ,   "asl/as2"   or   "as2/asl"   box   (es)   ,   or   its(their)   repeat<br>
permuations, is(are) functionally linked downstream, in the 3' position, of the<br>
enhancer box.<br>
11.	The process as claimed in claim 1, wherein it also comprises inserting a<br>
"GC—rich" box functionally linked upstream of the "enhancer" box.<br>
12.	The process as claimed in claim 1, wherein it also comprises inserting<br>
two  "cereal"  boxes functionally linked in the  5'  position  upstream of the<br>
"enhancer" box, which is itself functionally linked in the 5' position upstream of<br>
an "as2/asl" box.<br>
13.	The process as claimed in claim 1, wherein it also comprises inserting at<br>
least one element chosen from the group consisting of an "enhancer" box, a<br>
"C—like" box, a "P-like" box, a "GATA" box, a "cereal" box, an "asl" box, an "as2"<br>
box, and/or an asl/as2 or "asl/as2" combination of boxes, optionally repeated,<br>
and   a   "GC-rich"   box,   functionally   linked   in   reverse   orientation   and/or<br>
downstream, in the 3' position, of the transcription start site.<br>
14.	Method for expressing a nucleic acid sequence, or gene,  encoding a<br>
polypeptide to be produced, in a cell, characterized in that it comprises the<br>
steps consisting in:<br>
transforming the cell with a vector comprising at least one promoter nucleic acid sequence as claimed in any one of claims 1 to 3;<br>
preparing  a  culture  of the  cell  under  conditions   which   allow  the expression of the nucleic acid sequence, or gene, encoding the polypeptide.<br>
15.	Method as claimed in claim 14, wherein the cell is a procaryotic or<br>
eucaryotic cell.<br>
16.	Method as claimed in either claims 14 and 15, wherein the cell is a cell<br>
chosen from the group consisting of microbial cells, fungal cells, insect cells,<br>
animal cells and plant cells.<br>
17.	Method as claimed in any one of claims 14 to 16, wherein the cell is a<br>
plant cell.<br>
18.	A   process   for   construction   of  a   chimeric   promoter   of  expression<br>
substantially as hereinbefore described with reference to the accompanying<br>
drawings.<br>
19.	Method   for   expressing   a   nucleic   acid   sequence   substantially   as<br>
hereinbefore described with reference to the accompanying drawings.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQ1MS1kZWwtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-0451-del-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQ1MS1kZWwtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-0451-del-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQ1MS1kZWwtY29ycmVzcG9uZGVuY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-0451-del-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQ1MS1kZWwtY29ycmVzcG9uZGVuY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-0451-del-correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQ1MS1kZWwtZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-0451-del-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQ1MS1kZWwtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-0451-del-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQ1MS1kZWwtZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-0451-del-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQ1MS1kZWwtZm9ybS0xMy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-0451-del-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQ1MS1kZWwtZm9ybS0xOS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-0451-del-form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQ1MS1kZWwtZm9ybS0yLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-0451-del-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQ1MS1kZWwtZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-0451-del-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQ1MS1kZWwtZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-0451-del-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQ1MS1kZWwtZ3BhLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-0451-del-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQ1MS1kZWwtcGN0LTIxMC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-0451-del-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQ1MS1kZWwtcGN0LTMwNC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-0451-del-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDQ1MS1kZWwtcGV0aXRpb24tMTM4LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-0451-del-petition-138.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="220138-system-for-providing-printed-information-to-a-user.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="220140-a-method-for-producing-a-hot-co-and-h2-containing-gas-serving-for-the-reduction-of-fine-grained-metal-ore-and-apparatus-thereof.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>220139</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/00451/DEL</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>28/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>11-Jul-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>15-May-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>30-May-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MERISTEM THERAPEUTICS</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>FREDERIC NORRE</td>
											<td>18, RUE DE AL GRANDE CHARREYRE, F-63730 MIREFLEURS, FRANCE</td>
										</tr>
										<tr>
											<td>2</td>
											<td>MANFRED THEISEN</td>
											<td>1, RUE DES ROSERAIES, F-63400 CHAMALIERES, FRANCE.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>FREDERIC NORRE</td>
											<td>18, RUE DE AL GRANDE CHARREYRE, F-63730 MIREFLEURS, FRANCE</td>
										</tr>
										<tr>
											<td>4</td>
											<td>MANFRED THEISEN</td>
											<td>1, RUE DES ROSERAIES, F-63400 CHAMALIERES, FRANCE.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/82</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IB00/1383</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-09-28</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>99/12373</td>
									<td>1999-09-30</td>
								    <td>France</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/220139-a-process-for-construction-of-a-chimeric-promoter-of-expression by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:04:47 GMT -->
</html>
