<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/220186-method-useful-for-concurrently-performing-multiple-biological-or-chemically-assays-and-kit-thereto by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:01:38 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 220186:&quot;METHOD USEFUL FOR CONCURRENTLY PERFORMING MULTIPLE BIOLOGICAL OR CHEMICALLY ASSAYS AND KIT THERETO&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;METHOD USEFUL FOR CONCURRENTLY PERFORMING MULTIPLE BIOLOGICAL OR CHEMICALLY ASSAYS AND KIT THERETO&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method of detecting at least one target, comprising a) contacting a sample which may comprise said target(s) with a combination which comprises, before the addition of said sample, i) a surface, comprising multiple spatially discrete regions, at least two of which are substantially identical, each region comprising ii) at least eight different oligonucleotide anchors, each in association with iii) a bifunctional linker which has a first portion that is specific for the oligonucleotide anchor, and a second portion that comprises a probe which is specific for said target(s), wherein the target(s) is a protection fragment(s) and is amplified by PCR before said sample(s) is contacted with said combination, under conditions effective for said target(s) to bind to said combination, b) contacting said combination and any bound targets with a labeled detection probe, and c) detecting said detection probe.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>HIGH THROUGHPUT ASSAY SYSTEM<br>
This application is a C.I.P. of U.S. Application Serial Number 09/218,166, filed on December 22, 1998, which disclosure is incorporated by reference in its entirety herein. This application claims priority of provisional application 60/068,291, filed December 19,1997, and of U.S. application of serial number 09/109,076, filed on July 2, 1998, each of which disclosure is incorporated by reference in its entirety herein.<br>
Background Of Thg Invention<br>
This invention relates, e.g., to compositions, apparatus and methods useful for concurrently performing multiple biological or chemical assays, using repeated arrays of probes. A plurality of regions each contains an array of generic anchor molecules. The, anchors are associated with bifunctional linker molecules, each containing a portion wirier? is specific for at, least one of the anchors and a portion which is a probe specific for a target of interest. The resulting array of probes is used to analyze the presence of one or more target molecules which interact specifically with the probes. The invention relates to diverse fields distinguished by the nature of the molecular interaction, including but not limited to pharmaceutical drug discovery, molecular biology, biochemistry, pharmacology and medical diagnostic technology.<br>
Pluralities of molecular probes arranged on surfaces or "chips" have been used in a variety of biological and chemical assays. Assays are performed to determine if target molecules of interest interact with any of the probes. After exposing the probes to target molecules under selected test conditions, detection devices determine whether a target<br>
molecule has interacted with a given probe.<br>
These systems are useful in a variety of screening procedures for obtaining information about either the probes or the target molecules. For example, they have been used to screen for peptides or potential drugs "which bind to a receptor of interest, among others; to screen samples for the presence of, for example, genetic mutations, allelic variants in a population, or a particular pathogen or strain of pathogen, among many others; to study gene expression, for example to identify the mRNAs whose expression is correlated with a particular physiological condition, developmental stage, or disease state, etc.<br>
Summary of the Invention<br>
This invention provides compositions, apparatus and methods for concurrently performing multiple biological or chemical assays, and allows for high throughput analysis of multiple samples - for example, multiple patient samples to be screened in a diagnostic assay, or multiple potential drugs or therapeutic agents to be tested in a method of drug discovery. A combination is provided which is useful for the detection of one or more targets in a sample. This combination comprises a surface comprising a plurality of spatially discrete regions, which can be termed test regions and which can be wells, at least two of which are substantially identical. Each surface comprises at least two, preferably at least twenty or more, e.g., at least about 25, 50, 96, 864, or 1536, etc., of such substantially identical regions. Each test region defines a space for the introduction of a sample containing (or potentially containing) one or more targets and contains a biological or chemical array. (Phrases such as "sample containing a target" or "detecting a target in a sample" are not meant to exclude samples or determinations (detection attempts) where no target is contained or detected. In a general sense, this invention involves arrays to determine whether a target is contained in a sample irrespective of whether it is or is not detected.) This array comprises generic "anchors," each in association with a bifunctional linker molecule which has a first portion that is specific for the anchor and a second portion that comprises a probe which is specific for at least one of the target(s). The combination of this invention is placed in contact with a sample containing one or more targets, which<br>
optionally react with a detector molecule(s), and is then interrogated by a detection device which detects reactions between target molecules and probes in the test regions, thereby generating results of the assay.<br>
The invention provides methods and compositions particularly useful for high throughput biological assays. In especially preferred embodiments, the invention can be used for high throughput screening for drug discovery. For example, a high throughput assay can be run in many (100 for example).96-well microplates at one time. Each well of a plate can have, e.g., 36 different tests performed in it by using an array of about 36 anchor and linker pairs. That is, 100 plates, with 96 wells per plate, and each with 36 tests per well, can allow for a total of 345,000 tests; for example, each of 9,600 different drug candidates can be tested simultaneously for 36 different parameters or assays. High throughput assays provide much more information for each drug candidate than do assays which test only one parameter at a time. For example, it is possible in a single initial high throughput screening assay to determine whether a drug candidate is selective, specific and/or nontoxic. Non-high throughput methods necessitate extensive follow-up assays to test such parameters for each drug candidate of interest. Several types of high throughput screening assays are described, e.g., in Examples 15-17. The ability to perform simultaneously a wide variety of biological assays and to do very many assays at once (i.e., in very high throughput) are two important advantages of the invention.<br>
In one embodiment, for example, using 96-well DNA Bind plates (Coming Costar) made of polystyrene with a derivatized surface for the attachment of primary amines, such as amino acids or modified oligonucleotides, a collection of 36 different oligonucleotides can be spotted onto the surface of every well of every plate to serve as anchors. The anchors can be covalently attached to the derivatized polystyrene, and the same 36 anchors can be used for all screening assays. For any particular assay, a given set of linkers can be used to program the surface of each well to be specific for as many as 36 different targets or assay types of interest, and different test samples can be applied to each of the 96 wells in each plate. The same set of anchors can be used multiple times to re-program the surface of the wells for other targets and assays of interest, or it can be re-used multiple times with the<br>
same set of linkers.  This flexibility and reusability represent further advantages of the invention.<br>
One embodiment of the invention is zrcombination useful for the detection of one or more target(s) in a sample, which comprises, before the addition of said sample,<br>
a)	a surface, comprising multiple spatially discrete regions, at least two of which<br>
are substantially identical, each region comprising<br>
b)	at least eight different oligonucleotide anchors, each in association with<br>
c)	a bifunctional linker which has a first portion that is specific for the<br>
oligonucleotide anchor, and a second portion that comprises a probe which is specific for<br>
said target(s).<br>
Another embodiment of the invention is a combination useful for the detection of one or more target(s) in a sample, which comprises, before the addition of said sample,<br>
a)	a surface, comprising multiple spatially discrete regions, at least two of which<br>
are substantially identical, each region comprising<br>
b)	at least eight different anchors, each in association with<br>
c)	a bifunctional linker which has a first portion that is specific for the anchor,<br>
and a second portion that comprises a probe which is specific for said target(s).<br>
Another embodiment of the invention is a method for detecting at least one target, which comprises contacting a sample which may comprise the target(s) with a combination,, as described above, under conditions effective for said target(s) to bind to said combination. Another embodiment is a method for determining an RNA expression pattern, which comprises incubating a sample which comprises as target(s) at least two RNA molecules with a combination as described above, wherein at least one probe of the combination is a nucleic acid (e.g., oligonucleotide) which is specific (i.e. selective) for at least one of the RNA targets, under conditions which are effective for specific hybridization of the RNA target(s) to the probe(s). Another embodiment is a method for identifying an agent (or condition(s)) that modulates an RNA expression pattern, which is the method described above for determining an RNA expression pattern, further comprising comparing the RNA expression pattern produced in the presence of said agent (or condition(s)) to the RNA<br>
expression pattern produced under a different set of conditions.<br>
By way of example, Figures 1 and 2 illustrate a combination of the invention and a method of using it to detect an mRNA target: The surface of the invention, shown in Fig. 2, contains 15 identical test regions; in an especially preferred embodiment of the invention, each of these test regions is a well in a microtiter plate. Each of the test regions contains six different anchors, here indicated as numbers 1-6. Fig. 1 schematically illustrates one of those anchors, anchor 1, which, in a most preferred embodiment of the invention, is an oligonucleotide. To anchor 1 is attached a linker molecule, linker 1, which comprises two portions. The first portion, which is specific for the anchor, is in this illustration an oligonucleotide which-can hybridize specifically to the anchor. The second portion, which is a probe specific for the target of interest - here, target mRNA 1 - is in this illustration an oligonucleotide which can hybridize to that target. Although not illustrated in this figure, each of the remaining five anchors can hybridize to its own linker via the anchor-specific portion; each linker can contain a probe portion specific for, e.g., an mRNA different from (or the same as) mRNA 1. This illustrated combination can be used to assay as many as 15 different samples at the same time for the presence of mRNA 1 (or, simultaneously, for mRNA targets which are specified (programmed) by the other five probes in the array). To perform the assay, each sample, which in this example can be an RNA extract from, say, one. of 15 independent cell lines, is added in a small volume to one of the regions, or wells, and incubated under conditions effective for hybridization of the probe and the target. In order to determine if mRNA 1 is present in a sample, a detection device which can recognize patterns, and/or can interrogate specific locations within each region for the presence of a signal, is employed. If the cell lines are incubated under conditions in which their mRNAs are labeled in vivo with a tag, and if mRNA 1 is present in a sample, the detector will detect a signal emanating from the tagged mRNA at the location defined by anchor/probe complex 1. Alternatively, the mRNA can be directly labeled in vitro, before or after being added to the regions (wells). Alternatively, as is illustrated in Fig. 1, mRNA can be tagged indirectly, before or after it has hybridized to the probe, e.g., by incubating the RNA with a tagged "detector" oligonucleotide (target-specific reporter oligonucleotide) which is complementary<br>
to a sequence other than that recognized by the probe. In the illustrated example, 15 samples can be analyzed simultaneously. Because at least 20 or more, e.g., as many as 1536 or more, samples can be analyzed simultaneously with this invention, it is a very high throughput assay system.<br>
As used herein, "target" refers to a substance whose presence, activity and/or amount is desired to be determined and which has an affinity for a given probe. Targets can be man-made or naturally-occurring substances. Also, they can be employed in their unaltered state or as aggregates with other species. Targets can be attached, covalently or noncovalently, to a binding member, either directly or via a specific binding substance. Examples of targets which can be employed in this invention include, but are not limited to, receptors (on vesicles, lipids, cell membranes or a variety of other receptors); ligands, agonists or antagonists which bind to specific receptors; polyclonal antibodies, monoclonal antibodies and antisera reactive with specific antigenic determinants (such as on viruses, cells or other materials); drugs; nucleic acids or polynucleotides (including mRNA, tRNA, rRNA, oligonucleotides, DNA, viral RNA or DNA, ESTs, cDNA, PCR-amplified products derived from RNA or DNA, and mutations, variants or modifications thereof); proteins (including enzymes, such as those responsible for cleaving neurotransmitters, proteases, kinases and the like); substrates for enzymes; peptides; cofactors; lectins; sugars; polysaccharides; cells (which can include cell surface antigens); cellular membranes; organelles; etc., as well as other such molecules or other substances which can exist in complexed, covalently bonded crosslinked, etc. form. As used herein, the terms nucleic acid, polynucleotide, polynucleic acid and oligonucleotide are interchangeable. Targets can also be referred to as anti-probes.<br>
As used herein, a "probe" is a substance, e.g., a molecule, that can be specifically recognized by a particular target. The types of potential probe/target or target/probe binding partners include receptor/ligand; ligand/antiligand; nucleic acid (polynucleotide) interactions, including DNA/DNA, DNA/RNA, PNA (peptide nucleic acid)/nucleic acid; enzymes, other catalysts, or other substances, with substrates, small molecules or effector molecules; etc. Examples of probes that are contemplated by this invention include, but are not limited to, organic and inorganic materials or polymers, including metals, chelating<br>
agents or other compounds which interact specifically with metals, plastics, agonists and antagonists for cell membrane receptors, toxins and venoms, viral epitopes, hormones (e.g., opioid peptides, steroids, etc.), hormone Teceptors, lipids (including phospholipids), peptides, enzymes (such as proteases or kinases), enzyme substrates, cofactors, drugs, lectins, sugars, nucleic acids (including oligonucleotides, DNA, RNA, PNA or modified or substituted nucleic acids), oligosaccharides, proteins, enzymes, polyclonal and monoclonal antibodies, single chain antibodies, or fragments thereof. Probe polymers can be linear or cyclic. Probes can distinguish between phosphorylated and non-phosphorylated proteins, either by virtue of differential activity or differential binding. Probes such as lectins can distinguish among glycosylated proteins. As used herein, the terms nucleic acid, polynucleotide, polynucleic acid and oligonucleotide are interchangeable. Any of the substances described above as "probes" can also serve as "targets," and vice-versa.<br>
Any compatible surface can be used in conjunction with this invention. The surface (usually a solid) can be any of a variety of organic or inorganic materials or combinations thereof, including, merely by way of example, plastics such as polypropylene or polystyrene; ceramic; silicon; (fused) silica, quartz or glass, which can have the thickness of, for example, a glass microscope slide or a glass cover slip; paper, such as filter paper; diazotized cellulose; nitrocellulose filters; nylon membrane; or polyacrylamide or other type of gel pad, e.g., an aeropad or aerobead, made of an aerogel, which is, e.g., a highly porous solid, including a film, which is prepared by drying of a wet gel by any of a variety of routine, conventional methods. Substrates that are transparent to light are useful when the method of performing an assay involves optical detection. In a preferred embodiment, the surface is the plastic surface of a multiwell, e.g., tissue culture dish, for example a 24-, 96-, 256-, 384-, 864- or 1536-well plate (e.g., a modified plate such as a Corning Costar DNA Bind plate). Anchors can be associated, e.g., bound, directly with a surface, or can be associated with one type of surface, e.g., glass, which in turn is placed in contact with a second surface, e.g., within a plastic "well" in a microtiter dish. The shape of the surface is not critical. It can, for example, be a flat surface such as a square, rectangle, or circle; a curved surface;<br>
or a three dimensional surface such as a bead, particle, strand, precipitate, tube, sphere; etc. The surface comprises regions which are spatially discrete and addressable or identifiable. Each region comprises a set of anchors. How the regions are separated, their physical characteristics, and their relative orientation to one another are not critical. In one embodiment, the regions can be separated from one another by any physical barrier which is resistant to the passage of liquids. For example, in a preferred embodiment, the regions can be wells of a multiwell (e.g., tissue culture) dish, for example a 24-, 96-, 256-, 384-, 864- or 1536-well plate. Alternatively, a surface such as a glass surface can be etched out to have, for example, 864 or 1536 discrete, shallow wells. Alternatively, a surface can comprise regions with no separations or wells, for example a flat surface, e.g., piece of plastic, glass or paper, and individual regions can further be defined by overlaying a structure (e.g,. a piece of plastic or glass) which delineates the separate regions. Optionally, a surface can already comprise one or more arrays of anchors, or anchors associated with linkers, before the individual regions are delineated. In another embodiment, arrays of anchors within each region can be separated from one another by blank spaces on the surface in which there are no anchors, or by chemical boundaries, such as wax or silicones, to prevent spreading of droplets. In yet another embodiment, the regions can be defined as tubes or fluid control channels, e.g., designed for flow-through assays, as disclosed, for example, in Beattie et al (1995). Clin. Chem. 4, 700-706. Tubes can be of any size, e.g., capillaries or wider bore tubes; can allow the flow of liquids; or can be partially or completely filled with a gel, e.g., agarose or polyacrylamide, through which compounds can be transported (passed through, flowed through, pumped through), e.g., by electrophoresis. In a preferred embodiment, a tube is filled with a gel; the gel is activated for the binding of anchors, and different anchors are passed through sequentially, allowing the formation of a linear array of anchors within the gel; and linkers, targets, etc. are passed through in succession. Regions within or on, etc. a surface can also be defined by modification of the surface itself. For example, a plastic surface can comprise portions made of modified or derivatized plastic, which can serve, e.g., as sites for the addition of specific types of polymers (e.g., PEG can be attached to a polystyrene surface and then derivatized with<br>
carboxyl or amino groups, double bonds, aldehydes, and the like). Alternatively, a plastic surface can comprise molded structures such as protrusions or bumps, which can serve as platforms for the addition of anchors. In anotherlembodiment, regions can be gel pads, e.g., polyacrylamide gel pads or aeropads, which are arrayed in a desired pattern on a surface such as, e.g., glass, or are sandwiched between two surfaces, such as, e.g., glass and a quartz plate. Anchors, linkers, etc. can be immobilized on the surface of such pads, or can be imbedded within them. A variety of other arrangements of gel pads on surfaces will be evident to one of skill in the art, and can be produced by routine, conventional methods. The relative orientation of the test regions can take any of a variety of forms including, but not limited to, parallel or perpendicular arrays within a square or rectangular or other surface, radially extending arrays within a circular or other surface, or linear arrays, etc.<br>
The spatially discrete regions of the invention are present in multiple copies. That is, there are at least two, preferably at least twenty, or at least about 24, 50, 96, 256, 384, 864,1536,2025, or more, etc., substantially identical, spatially discrete (separated) regions. Increasing numbers of repeated regions can allow for assays of increasingly higher throughput. Substantially identical regions, as used herein, refers to regions which contain identical or substantially identical arrays of anchors and/or anchor/linker complexes. Substantially identical, as used herein, means that an array or region is intended to serve essentially the same function as another array or region in the context of analyzing a target in accordance with this invention. Differences not essentially affecting function, i.e., detectability of targets, are along the line of small nucleotide imperfections (omissions/inserts/substitutions) or oligo imperfections (poor surface binding), etc., which do not within assay accuracy significantly affect target determination results.<br>
Of course, one of skill in the art will recognize that not all of the regions on a surface need to be substantially identical to one another. For example, if two different sets of arrays are to be tested in parallel, it might be advantageous to include both sets of arrays on a single surface. For example, the two different sets of arrays can be arranged in alternating striped patterns, to facilitate comparison between them. In another embodiment, the practitioner may wish to include regions which can be detected in a distinguishable manner from the<br>
other regions on the surface and can thereby be used as a "registration region(s)." For example, a registration region can comprise oligonucleotides or peptides which display a distinctive pattern of fluorescent molecules/that can be recognized by a scanning detection device as a "starting point" for aligning the locations of the regions on a surface.<br>
The size and physical spacing of the test regions are not limiting. Typical regions are of an area of about 1 to about 700 mm2, preferably 1 to about 40 mm2, and are spaced about 0.5 to about 5 mm apart, and are routinely selected depending on the areas involved. In a preferred embodiment, the regions are spaced approximately 5 mm apart. For example, each region could comprise a rectangular grid, with, for example, 8 rows and 6 columns, of roughly circular spots of anchors which are about 100 micrometers in diameter and 500 micrometers apart; such a region would cover about a 20 millimeter square area. Larger and smaller region areas and spacings are included.<br>
The regions can also be further subdivided such that some or all anchors within a region are physically separated from neighboring anchors by means, e.g., of an indentation or dimple. For example, the number of subdivisions (subregions) in a region can range from about 10 to about 100 or more or less. In one embodiment, a region which is a well of a 1536-well dish can be further subdivided into smaller wells, e.g., about 4 to about 900, preferably about 16 to about 36 wells, thereby forming an array of wells-within-wells. See Fig. 4. Such a dimpled surface reduces the tolerance required for physically placing a single anchor (or group of anchors) into each designated space (locus), and the size of the areas containing anchors is more uniform, thereby facilitating the detection of targets which bind to the probe.<br>
The term "anchor" as used herein refers to any entity or substance, e.g., molecule (or "group" of substantially identical such substances (see, e.g., Fig. 7)) which is associated with (e.g., immobilized on, or attached either covalently or non-covalently to) the surface, or which is a portion of such surface (e.g., derivatized portion of a plastic surface), and which can undergo specific interaction or association with a linker or other substance as described herein. As used herein, an "anchor/linker complex" exists when an anchor and a linker have combined through molecular association in a specific manner. The interaction with the<br>
linker can be either irreversible, such as via certain covaient bonds, or reversible, such as via nucleic acid hybridization. In a preferred embodiment, the anchor is a nucleic acid, which can be of any length (e.g., an oligonucleotide) jir type (e.g., DNA, KNA, PNA, or a PCR product of an RNA or DNA molecule). The nucleic acid can be modified or substituted (e.g., comprising non naturally occurring nucleotides such as, e.g., inosine; joined via various known linkages such as sulfamate, sulfamide, phosphorothionate, methylphosphonate, carbamate, etc.; or a semisynthetic molecule such as a DNA-streptavidin conjugate, etc.). Single stranded nucleic acids are preferred. The anchor can also be a peptide or a protein. For example, it can be a polyclonal or monoclonal antibody molecule or fragment thereof, or single chain antibody or fragment thereof, which binds specifically to the portion of a linker that is an antigen or an anti-antibody molecule; in the obverse, the anchor can be a peptide, and the portion of the linker which binds to it can be an antibody or the like. In another embodiment, the anchor can be a lectin (such as concanavalin A or agglutinins from organisms such as Limulus, peanut, mung bean, Phaseolus, wheat germ, etc.) which is specific for a particular carbohydrate. In another embodiment, the anchor can be an organic molecule, such as a modified or derivatized plastic polymer which can serve, e.g., as the stage for specific solid phase chemical synthesis of an oligonucleotide. In this case, the derivatized plastic can be distributed as an array of discrete, derivatized, loci which are formed integrally into the plastic surface of a combination during the manufacturing process. In another embodiment, the anchor can take advantage of specific or preferential binding between metal ions, e.g., Ni, Zn, Ca, Mg, etc. and particular proteins or chelating agents. For example, the anchor can be polyhistidine, and the anchor-specific portion of the linker can be nickel, which is attached via a nickel chelating agent to a target-specific probe. Alternatively, the chelating agent can be the anchor and the polyhistidine the probe-related portion. Alternatively, the anchor can be an inorganic substance. For example, it can be a metal such as calcium or magnesium, and the anchor-specific portion of the linker can be a preferential chelating agent, such as EDTA or EGTA, respectively, which is attached to a target-specific probe. One of skill in the art will recognize that a wide range of other types of molecules can also serve as anchors, such as<br>
those general types also discussed in conjunction with probes and targets.<br>
The number of anchors in a test region can be at least two, preferably between about 8 and about 900 (more or less being included) more preferably between about 8 and about 300, and most preferably between about 30 and about 100 (e.g., about 64). In some preferred embodiments, there are about 16,36,45 or 100 anchors/test region for a surface with 96 test regions (e.g., wells), or about 9,16 or 25 anchors/test region for a surface with 384 test regions (e.g., wells). In a most preferred embodiment, each anchor in a test region has a different specificity from every other anchor in the array. However, two or more of the anchors can share the same specificity and all of the anchors can be identical. In one embodiment, in which a combination of the invention comprises a very large number of test regions (e.g., about 864, 1536, or more), so that a large number of test samples can be processed at one time, it might of interest to test those samples for only a limited number (e.g., about 2, 4, 6 or 9) of parameters. In other words, for combinations comprising a very large number of regions, it might be advantageous to have only about 2 to 9 anchors per region.<br>
The physical spacing and relative orientation of the anchors in or on a test region are not limiting. Typically, the distance between the anchors is about 0.003 to about 5 mm or less, preferably between about 0.03 and about 1. Larger and smaller anchor spacings (and areas) are included. The anchors can be arranged in any orientation relative to one another and to the boundaries of the region. For example, they can be arranged in a two-dimensional orientation, such as a square, rectangular, hexagonal or other array, or a circular array with anchors emanating from the center in radial lines or concentric rings. The anchors can also be arranged in a one-dimensional, linear array. For example, oligonucleotides can be hybridized to specific positions along a DNA or RNA sequence to form a supramolecular array, or in a linear arrangement in a flow through gel. Alternatively, the anchors can be laid down in a "bar-code"-like formation. (See Fig. 6). For example, anchors can be laid down as long lines parallel to one another. The spacing between or the width of each long line can be varied in a regular way to yield a simple, recognizable pattern much like a bar-code, e.g., the first and third lines can be twice as large as the rest, lines can be omitted, etc. An extra<br>
empty line can be placed after the last line to demarcate one test region, and the bar code pattern can be repeated in succeeding test regions.<br>
The pattern of anchors does not need to-be in strict registry with the positions of the separated assay wells (test regions) or separate assay droplets. The term "assay positions" will be used to refer to the positions of the assay surface where assay samples are applied. (These can be defined by the position of separate droplets of assay sample or by the position of walls or separators defining individual assay wells on a multi-well plate for example.) The anchor pattern itself (e.g., a "bar code"-like pattern of oligonucleotide anchors) is used to define where exactly each separate anchor is positioned by pattern recognition - just as each line of a barcode is recognized by its position relative to the remaining lines. Hence the first anchor need not be at one edge or one corner of each assay position. The first anchor will be found by pattern recognition, rather than position relative to the assay position. As long as the area used by each assay position (the area of the droplet or the area of the well for example) is large enough to be certain to contain at least one whole unit of the repeating pattern of anchors, then each assay point will test the sample for that assay position for all of the targets specified by the (bar-coded) pattern wherever the pattern lies within the area of the assay position.<br>
The anchors do not need to be arranged in a strict or even fixed pattern within each test region. For example, each anchor can be attached to a particle, bead, or the like, which assumes a random position within a test region. The location of each anchor can be determined by the use, e.g., of a detectable tag. For example, the linker molecule specific for each type of anchor can be labeled with a different fluorescent, luminescent etc. tag, and the position of a particle comprising a particular linker/anchor pair can be identified by the nature of the signal emanating from the linker, e.g., the excitation or emission spectrum. One skilled in the art can prepare a set of linkers with a variety of such attached tags, each with a distinguishable spectrum. Alternatively, the anchors can be labeled directly. For example, each type of anchor can be labeled with a tag which fluoresces with a different spectrum from the tags on other types of anchors. Alternatively, the particles, beads or the like can be different from one another in size or<br>
shape. Any of the labeling and detection methods described herein can be employed. For example, fluorescence can be measured by a CCD-based imaging system, by a scanning fluorescence microscope or Fluorescence Activated Cell Sorter (FACS).<br>
An anchor can interact or become associated specifically with one portion - the anchor-specific portion - of a linker molecule. By the terms "interact" or "associate", it is meant herein that two substances or compounds (e.g., anchor and anchor-specific portion of a linker, a probe and its target, or a target and a target-specific reporter) are bound (e.g., attached, bound, hybridized, joined, annealed, covalently linked, or otherwise associated) to one another sufficiently that the intended assay can be conducted. By the terms "specific" or "specifically", it is meant herein that two components (e.g., anchor and anchor-specific region of a linker, a probe and its target, or a target and a target-specific reporter) bind selectively to each other and, in the absence of any protection technique, not generally to other components unintended for binding to the subject components. The parameters required to achieve specific interactions can be determined routinely, e.g., using conventional methods in the art..<br>
For nucleic acids, for example, one of skill in the art can determine experimentally the features (such as length, base composition, and degree of complementarity) that will enable a nucleic acid (e.g., an oligonucleotide anchor) to hybridize to another nucleic acid (e.g., the anchor-specific portion of a linker) under conditions of selected stringency, while minimizing non-specific hybridization to other substances or molecules (e.g., other oligonucleotide linkers). Typically, the DNA or other nucleic acid sequence of an anchor, a portion of a linker, or a detector oligonucleotide will have sufficient complementarity to its binding partner to enable it to hybridize under selected stringent hybridization conditions, and the Tm will be about 10° to 20° C. above room temperature (e.g., about 37° C). In general, an oligonucleotide anchor can range from about 8 to about 50 nucleotides in length, preferably about 15,20,25 or 30 nucleotides. As used herein, "high stringent hybridization conditions" means any conditions in which hybridization will occur when there is at least 95%, preferably about 97 to 100%, nucleotide complementarity (identity) between the nucleic acids. However, depending on the desired purpose, hybridization conditions can be<br>
selected which require less complementarity, e.g., about 90%, 85%, 75%, 50%, etc. Among the hybridization reaction parameters which can be varied are salt concentration, buffer, pH, temperature, time of incubation, amount and type of denaturant such as formamide, etc. (see, e.g., Sambrook et al. (1989). Molecular Cloning: A Laboratory Manual (2d ed.) Vols. 1-3, Cold Spring Harbor Press, New York; Hames et al. (1985). Nucleic Acid Hybridization, IL Press; Davis et al. (1986), Basic Methods in Molecular Biology, Elsevir Sciences Publishing, Inc., New York). For example, nucleic acid (e.g., linker oligonucleotides) can be added to a test region (e.g., a well of a multiwell plate - in a preferred embodiment, a 96 or 384 or greater well plate), in a volume ranging from about 0.1 to about 100 or more ul (in a preferred embodiment, about 1 to about 50 ul, most preferably about 40 ul), at a concentration ranging from about 0.01 to about 5 uM (in a preferred embodiment, about 0.1 ^M), in a buffer such as, for example, 6X SSPE-T (0.9 M NaCl, 60mM NaH2P04,6 mM EDTA and 0.05% Triton X-100), and hybridized to a binding partner (e.g., an oligonucleotide anchor on the surface) for between about 10 minutes and about at least 3 hours (in a preferred embodiment, at least about 15 minutes) at a temperature ranging from about 4° C. to about 37° C. (in a preferred embodiment, at about room temperature). Conditions can be chosen to allow high throughput. In one embodiment of the invention, the reaction conditions can approximate physiological conditions.<br>
The design of other types of substances or molecules (e.g., polypeptides, lectins, etc.) which can, e.g., serve as anchors or as portions of linkers, and the reaction conditions required to achieve specific interactions with their binding partners, are routine and conventional in the art (e.g., as described in Niemeyer et al (1994). Nucl. Acids Res. 22, 5530-5539; Fodor et al (1996). U.S. Patent No. 5,510,270; Pirrung et al (1992), U.S. Patent No. 5,143,854). Among the incubation parameters are buffer, salt concentration, pH, temperature, time of incubation, presence of carrier and/or agents or conditions to reduce non-specific interactions, etc. For example, to a test region (e.g., a well of a multiwell plate - in a preferred embodiment, a 96 or 384 or greater well plate) which contains, as anchors, antibodies, can be added anti-antibodies (e.g., antigens or antibody-specific secondary antibodies) in a volume ranging from about 0.1 to about 100 or more p.1 (in a preferred<br>
embodiment, about 1 to about 50 µl, most preferably about 40 p.1), at a concentration ranging from about 10 pM to about 10 nM (in a preferred embodiment, about 1 nM), in a buffer such as, for example, 6X SSPE-T, PBS or physiological saline, and incubated with the anchors on the surface for between about 10 minutes and at least about 3 hours (in a preferred embodiment, at least about 15 minutes), at a temperature ranging from about 4° C. to about 45° C. (in a preferred embodiment, about 4° C.)- For peptide anchors, a length of about 5 to about 20 amino acids is preferred.<br>
In some embodiments of the invention, each anchor in an array can interact with the anchor-specific portion of its corresponding linker to substantially the same degree as do the other anchors in the array, under selected reaction conditions. This can insure that the anchors specify a substantially uniform array of linkers and, therefore, probes.<br>
The anchors within a test region can be a "generic" set, each anchor of which can interact with one or more of a variety of different linkers, each having a portion specific to such anchor but with differing "probe" portions; thus, a single array of generic anchors can be used to program or define a varied set of probes. The flexible nature of such a generic assay of anchors can be illustrated with reference to Figures 1 and 2. Fig. 2 illustrates a surface which comprises 15 test regions, each of which contains an array of 6 different anchors, which in this example can be oligonucleotides. Fig. 1 schematically illustrates one of these (oligonucleotide) anchors, anchor 1, which is in contact with linker 1, which comprises one portion that is specific for anchor 1 and a second portion that is specific for target mRNA 1. Alternatively, one could substitute, e.g., a linker 2, which, like linker 1, comprises a portion that is specific for anchor 1, but which comprises a second portion that is specific for target raRNA 2 instead of target mRNA 1. Thus, anchor 1 can be used to specify (or program, or define, or determine) probes for either of two or more different target mRNAs. The process of generating and attaching a high resolution pattern (array) of oligonucleotides or peptides can be expensive, time-consuming and/or physically difficult. .The ability to use a pre-formed array of anchors to program a wide variety of probe arrays is one advantage of this invention.<br>
Although the generic anchors illustrated in Fig. 2 define a pattern of oligonucleotide<br>
probes, the identical anchor array could also be used to program an array of other probes, for example receptor proteins (see, e.g., Fig. 3). Clearly, many permutations are possible, given the range of types of anchor/linker interactions, e.g., even more complex layers of "sandwiched" or "piggybacked" probes such as protein/antibody combinations. Thus, the surface of anchors per this invention, itself, offers novel advantages.<br>
In one embodiment of the invention, anchors can interact reversibly with linkers; thus, a generic set of anchors can be re-used to program a varied set of probes. For example, an oligonucleotide anchor can be separated from the oligonucleotide portion of a linker by, for example, a heating step that causes the two oligonucleotides to dissociate, and can then be rebound to a second linker. The ability to re-use anchor arrays, which can be expensive, time-consuming and/or physically difficult to make, is another advantage of the invention.<br>
An anchor does not necessarily have to interact with a linker. For example, an anchor can be coupled (directly or indirectly) to a detectable molecule, such as a fluorochrome, and can thereby serve to localize a spot within a grid, e.g., for purpose of registration between the test surface and the detector. Alternatively, an anchor can be labeled with a known amount of a detectable molecule so as to serve as internal quantitation marker, e.g., for purposes of calibration.<br>
The term "linker" as used herein refers to a bifunctional substance which comprises a first portion (or moiety or part) that is specific for a chosen (designated) anchor or subset of the anchors ("anchor-specific") and a second portion that is a probe which is specific for a target of interest ("target-specific"). The two portions of the linker can be attached via covalent or noncovalent linkages, and can be attached directly or through an intermediate.<br>
The chemical nature of the anchor-specific portion of the linker is, of course, a function of the anchor or anchors with which it interacts. For example, if the anchor is an oligonucleotide, the portion of the linker which interacts with it can be, for example, a peptide which binds specifically to the oligonucleotide,or a nucleic acid which can hybridize efficiently and specifically to it under selected stringent hybridization conditions. The nucleic acid can be, e.g., an oligonucleotide, DNA, RNA, PNA, PCR product, or substituted or modified nucleic acid (e.g., comprising non naturally-occurring nucleotides such as, e.g.,<br><br>
inosine; joined via various known linkages such as sulfamate, sulfamide, phosphorothionate, methylphosphonate, carbamate; or a semisynthetic molecule such as a DNA-streptavidin conjugate, etc.)- Single strand moieties are preferred. The portion of a linker which is specific for an oligonucleotide anchor can range from about 8 to about 50 nucleotides in length, preferably about 15, 20, 25 or 30 nucleotides. If the anchor is an antibody, the portion of the linker which interacts with it can be, e.g., an anti-antibody, an antigen, or a smaller fragment of one of those molecules, which can interact specifically with the anchor. Substances or molecules which interact specifically with the other types of anchors described above, and which can serve as the anchor-specific portion of a linker, are well-known in the art and can be designed using conventional procedures (e.g., see above).<br>
The chemical nature of the target-specific portion of the linker is, of course, a function of the target for which it is a probe and with which it interacts. For example, if the target is a particular mRNA, the target-specific portion of the linker can be, e.g., an oligonucleotide which binds specifically to the target but not to interfering RNAs or DNAs, under selected hybridization conditions. One of skill in the art can, using art-recognized methods, determine experimentally the features of an oligonucleotide that will hybridize optimally to the target, with minimal hybridization to non-specific, interfering DNA or RNA (e.g., see above). In general, the length of an oligonucleotide probe used to distinguish a target mRNA present in a background of a large excess of untargeted RNAs can range from about 8 to about 50 nucleotides in length, preferably about 18,20,22 or 25 nucleotides. An oligonucleotide probe for use in a biochemical assay in which there is not a large background of competing targets can be shorter. Using art-recognized procedures (e.g., the computer program BLAST), the sequences of oligonucleotide probes can be selected such that they are mutually unrelated and are dissimilar from potentially interfering sequences in known genetics databases. The selection of hybridization conditions that will allow specific hybridization of an oligonucleotide probe to an RNA can be determined routinely, using art-recognized procedures (e.g., see above). For example, target RNA [e.g., total RNA or mRNA extracted from tissues or cells grown (and optionally treated with an agent of interest) in any vessel, such as the well of a multiwell microtiter plate (e.g., 96 or 384 or<br>
more wells)] can be added to a test region containing a oligonucleotide probe array (see above) in a buffer such as 6X SSPE-T or others, optionally containing an agent to reduce non-specific binding (e.g., about 0.5 mg/mi degraded herring or salmon sperm DNA, or yeast RNA), and incubated at an empirically determined temperature for a period ranging from between about 10 minutes and at least 18 hours (in a preferred embodiment, about 3 hours). The stringency of the hybridization can be the same as, or less than, the stringency employed to associate the anchors with the anchor-specific portion of the linkers. The design and use of other types of probes are also routine in the art, e.g., as discussed above.<br>
The anchor-specific and the target-specific portions of a linker can be joined (attached, linked) by any of a variety of covalent or non-covalent linkages, the nature of which is not essential to the invention. The two portions can be joined directly or through an intermediate molecule. In one embodiment, in which both portions of the linker are oligonucleotides, they can be joined by covalent linkages such as phosphodiester bonds to form a single, colinear nucleic acid. In another embodiment, in which the anchor-specific portion is an oligonucleotide and the target-specific portion is a receptor, for example a receptor protein, the two portions can be joined via the interaction of biotin and streptavidin molecules, an example of which is illustrated in Fig. 3. Many variations of such linkages are known (e.g., see Niemeyer et al (1994).'NAR 22, 5530-5539). Alternatively, the two portions can be joined directly, e.g., an oligonucleotide can be amidated and then linked directly (e.g., crosslinked) to a peptide or protein via an amide bond, or joined to a membrane component via an amide bond or a lipid attachment. Methods to form such covalent or noncovalent bonds are conventional and are readily optimized by one of skill in the art.<br>
After two substances are associated (e.g., by incubation of two nucleic acids, two proteins, a protein plus a nucleic acid, or others) to form a complex (such as, e.g., an anchor/linker complex), the resulting complex can be optionally treated (e.g., washed) to remove unbound substances (e.g., linkers), using conditions which are determined empirically to leave specific interactions intact, but to remove non-specifically bound material. For example, reaction mixtures can be washed between about one and ten times<br>
or more under the same or somewhat more stringent conditions than those used to achieve the complex (e.g., anchor/linker complex).<br>
The combinations of this invention can be manufactured routinely, using conventional technology.<br>
Some of the surfaces which can be used in the invention are readily available from commercial suppliers. In a preferred embodiment, the surface is a 96-, 384- or 1536-well microtiter plate such as modified plates sold by Corning Costar. Alternatively, a surface comprising wells which, in turn, comprise indentations or "dimples" can be formed by micromachining a substance such as aluminum or steel to prepare a mold, then microinjecting plastic or a similar material into the mold to form a structure such as that illustrated in Fig. 4. Alternatively, a structure such as that shown in Fig. 4, comprised of glass , plastic, ceramic, or the like, can be assembled, e.g., from three pieces such as those illustrated in Fig. 5: a first section, called a well separator (Fig. 5a), which will form the separations between the sample wells; a second section, called a subdivider (Fig. 5b), which will form the subdivisions, or dimples, within each test well; and a third section, called a base (Fig. 5c), which will form the base of the plate and the lower surface of the test wells. The separator can be, for example, a piece of material, e.g., silicone, with holes spaced throughout, so that each hole will form the walls of a test well when the three pieces are joined. The subdivider can be, for example, a thin piece of material, e.g., silicone, shaped"* in the form of a screen or fine meshwork. The base can be a flat piece of material, e.g., glass, in, for example, the shape of the lower portion of a typical microplate used for a biochemical assay. The top surface of the base can be flat, as illustrated in Fig. 5c, or can be formed with indentations that will align with the subdivider shape to provide full subdivisions, or wells, within each sample well. The three pieces can be joined by standard procedures, for example the procedures used in the assembly of silicon wafers.<br>
Oligonucleotide anchors, linker moieties, or detectors can be synthesized by conventional technology, e.g., with a commercial oligonucleotide synthesizer and/or by ligating together subfragments that have been so synthesized. In one embodiment of the invention, preformed nucleic acid anchors, such as oligonucleotide anchors, can be situated<br>
on or within the surface of a test region by any of a variety of conventional techniques, including photolithographic or silkscreen chemical attachment, disposition by ink jet technology, capillary, screen or fluid channel chip, electrochemical patterning using electrode arrays, contacting with a pin or quill, or denaturation followed by baking or UV-irradiating onto filters (see, e.g., Rava et al (1996). U.S. Patent No. 5,545,531; Fodor et al (1996). U.S. Patent No. 5,510,270; Zanzucchi et al (1997). U.S. Patent No. 5,643,738; Brennan (1995). U.S. Patent No. 5,474,796; PCT WO 92/10092; PCT WO 90/15070). Anchors can be placed on top of the surface of a test region or can be, for example in the case of a polyacrylamide gel pad, imbedded within the surface in such a manner that some of the anchor protrudes from the surface and is available for interactions with the linker. In a preferred embodiment, preformed oligonucleotide anchors are derivatized at the 5' end with a free amino group; dissolved at a concentration routinely determined empirically (e.g., about 1 uM) in a buffer such as 50 mM phosphate buffer, pH 8.5 and 1 mM EDTA; and distributed with a Pixus nanojet dispenser (Cartesian Technologies) in droplets of about 10.4 nanoliters onto specific locations within a test well whose upper surface is that of a fresh, dry DNA Bind plate (Corning Costar). Depending on the relative rate of oligonucleotide attachment and evaporation, it may be required to control the humidity in the wells during preparation. In another embodiment, oligonucleotide anchors can be synthesized directly on the surface of a test region, using conventional methods such as, e.g., light-activated deprotection of growing oligonucleotide chains (e.g., in conjunction with the use of a site directing "mask") or by patterned dispensing of nanoliter droplets of deactivating compound using a nanojet dispenser. Deprotection of all growing sequences that are to receive a single nucleotide can be done, for example, and the nucleotide then added across the surface.<br>
Peptides, proteins, lectins, chelation embodiments, plastics and other types of anchors or linker moieties can also be routinely generated, and anchors can be situated on or within surfaces, using appropriate available technology (see, e.g., Fodor et al (1996). U.S. Patent No. 5,510,270; Pirrung et al (1992). U.S. Patent No. 5,143,854; Zanzucchi et al (1997). U.S. Patent No. 5,643,738; Lowe st al (1985). U.S. Patent No. 4,562,157; Niemeyer et al (1994). NAR 22, 5530-5539).<br>
In some embodiments of the invention, the disclosed combinations are used in a variety of screening procedures and/or to obtain information about the level, activity or structure of the probes or target molecules. Sucb^assays are termed Multi Array Elate Screen (MAPS) methods or assays, and the surfaces comprising arrays of anchors or anchors plus probes which are used for the assays are termed MAPS arrays or MAPS plates.<br>
The components of a reaction mixture, assay, or screening procedure can be assembled in any order. For example, the anchors, linkers and targets can be assembled sequentially; or targets and linkers, in the presence or absence of reporters, can be assembled in solution and then contacted with the anchors.<br>
One embodiment of the invention relates to a method of detecting at least one target, comprising<br>
a)	contacting a sample which may comprise said target(s) with a bifunctional<br>
linker which has a first portion that is specific for an oligonucleotide anchor and a second<br>
portion that comprises a probe which is specific for said target(s), under conditions effective<br>
to obtain a first hybridization product between said target(s) and said linker,<br>
b)	contacting said first hybridization product with a combination under<br>
conditions effective to obtain a second hybridization product between said first hybridization<br>
product and said combination, wherein said combination comprises, before the addition of<br>
said first hybridization product,<br><br>
1)	a surface comprising multiple spatially discrete regions, at least two<br>
of which are substantially identical, each region comprising<br>
2)	at least 8 different oligonucleotide anchors,<br><br>
c)	contacting said first hybridization product or said second hybridization<br>
product with a labeled detector probe, and<br>
d)	detecting said detection probe.<br>
Each of the assays or procedures described below can be performed in a high throughput manner, in which a large number of samples (e.g., as many as about 864, 1036, 1536, 2025 or more, depending on the number of regions in the combination) are assayed<br>
on each plate or surface rapidly and concurrently. Further, many plates or surfaces can be processed at one time. For example, in methods of drug discovery, a large number of samples, each comprising a drug candidate (e.g., a member of a combinatorial chemistry library, such as variants of small molecules, peptides, oligonucleotides, or other substances), can be added to separate regions of a combination as described or can be added to biological or biochemical samples that are then added to separate regions of a combination, and incubated with probe arrays located in the regions; and assays can be performed on each of the samples. With the recent advent and continuing development of high-density microplates, DNA spotting tools and of methods such as laser technology to generate and collect data from even denser microplates, robotics, improved dispensers, sophisticated detection systems and data-management software, the methods of this invention can be used to screen or analyze thousands or tens of thousands or more of compounds per day.<br>
For example, in embodiments in which the probes are oligonucleotides, the assay can be a diagnostic nucleic acid or polynucleotide screen (e.g., a binding or other assay) of a large number of samples for the presence of genetic variations or defects (e.g., polymorphisms or specific mutations associated with diseases such as cystic fibrosis. See, e.g., litia et al (1992). Molecular and Cellular Probes 6., 505-512)); pathogenic organisms (such as bacteria, viruses, and protozoa, whose hosts are animals, including humans, or plants), or mRNA transcription patterns which are diagnostic of particular physiological states or diseases. Nucleic acid probe arrays comprising portions of ESTs (including full-length copies) can be used to evaluate transcription patterns produced by cells from which the ESTs were derived (or others). Nucleic acid probes can also detect peptides, proteins, or protein domains which bind specifically to particular nucleic acid sequences (and vice-versa).<br>
In another embodiment, the combinations of the invention can be used to monitor biochemical reactions such as, e.g., interactions of proteins, nucleic acids, small molecules, or the like - for example the efficiency or specificity of interactions between antigens and antibodies; or of receptors (such as purified receptors or receptors bound to cell membranes) and their ligands, agonists or antagonists; or of enzymes (such as proteases or kinases) and<br>
their substrates, or increases or decreases in the amount of substrate converted to a product; as well as many others. Such biochemical assays can be used to characterize properties of the probe or target, or as the basis of a screening assay. For example, to screen samples for the presence of particular proteases (e.g., proteases involved in blood clotting such as proteases Xa and Vila), the samples can be assayed on combinations in which the probes are fluorogenic substrates specific for each protease of interest. If a target protease binds to and cleaves a substrate, the substrate will fluoresce, usually as a result, e.g., of cleavage and separation between two energy transfer pairs, and the signal can be detected. In another example, to screen samples for the presence of a particular kinase(s) (e.g., Src, tyrosine kinase, or ZAP70), samples containing one or more kinases of interest can be assayed on combinations in which the probes are peptides which can be selectively phosphorylated by one of the kinases of interest. Using art-recognized, routinely determinable conditions, samples can be incubated with the array of substrates, in an appropriate buffer and with the necessary cofactors, for an empirically determined period of time. (In some assays, e.g., for biochemical studies of factors that regulate the activity of kinases of interest, the concentration of each kinase can be adjusted so that each substrate is phosphorylated at a similar rate.) After treating (e.g., washing) each reaction under empirically determined conditions to remove kinases and undesired reaction components (optionally), the phosphorylated substrates can be detected by, for example, incubating them with detectable reagents such as, e.g., fiuorescein-labeled anti-phosphotyrosine or anti-phosphoserine antibodies (e.g., at a concentration of about 10 nM, or more or less), and the signal can be detected. In another example, binding assays can be performed. For example, SH2 domains such as GRB2 SH2 or ZAP70 SFI2 can be assayed on probe arrays of appropriate phosphorylated peptides; or blood sera can be screened on probe arrays of particular receptors for the presence of immune deficiencies. Also, enzyme-linked assays can be performed in such an array format. Combinations of the invention can also be used to detect mutant enzymes, which are either more or less active than their wild type counterparts, or to screen for a variety of agents including herbicides or pesticides.<br>
Of course, MAPS assays can be used to quantitate (measure, quantify) the amount<br>
of active target in a sample, provided that probe is not fully occupied, that is, not more than about 90% of available probe sites are bound (or reacted or hybridized) with target. Under these conditions, target can be quantitated because having more target will result in having more probe bound. On the other hand, under conditions where more than about 90% of available probe sites are bound, having more target present would not substantially increase the amount of target bound to probe. Any of the heretofore-mentioned types of targets can be quantitated in this manner. For example, Example 6 describes the quantitation of oligonucleotide targets. Furthermore, it demonstrates that even if a target is present in large excess (e.g., if it is present in such large amounts that it saturates the amount of available probe in a MAPS probe array), by adding known amounts of unlabeled target to the binding mixture, one can "shift the sensitivity" of the reaction in order to allow even such large amounts of target to be quantitated.<br>
In another embodiment, combinations of the invention can be used to screen for agents which modulate the interaction of a target and a given probe. An agent can modulate the target/probe interaction by interacting directly or indirectly with either the probe, the target, or a complex formed by the target plus the probe. The modulation can take a variety of forms, including, but not limited to, an increase or decrease in the binding affinity of the target for the probe, an increase or decrease in the rate at which the target and the probe 
agents which selectively inhibit specific kinases, or for selective inhibitors of the interaction between SH2 domains and phosphorylated peptides.<br>
In another embodiment, the combinations of the invention can be used to screen for agents which modulate a pattern of gene expression. Arrays of oligonucleotides can be used, for example, to identify mRNA species whose pattern of expression from a set of genes is correlated with a particular physiological state or developmental stage, or with a disease condition ("correlative" genes, RNAs, or expression patterns). By the terms "correlate" or "correlative," it is meant that the synthesis pattern of RNA is associated with the physiological condition of a cell, but not necessarily that the expression of a given RNA is responsible for or is causative of a particular physiological state. For example, a small subset of mRNAs can be identified which are expressed, upconverted and/or downconverted in cells which serve as a model for a particular disease state; this altered pattern of expression as compared to that in a normal cell, which does not exhibit a pathological phenotype, can serve as a indicator of the disease state ("indicator" genes, RNAs, or expression patterns). The terms "correlative" and "indicator" can be used interchangeably. For example, cells treated with a tumor promoter such as phorbol myristate might exhibit a pattern of gene expression which mimics that seen in the early stages of tumor growth. In another model for cancer, mouse insulinoma cells (e.g., cell line TGP61), when infected with adenovirus, exhibit an increase in the expression of, e.g., c-Jun and MIP-2, while the expression of' housekeeping genes such as GAPDH and L32 remains substantially unaffected.<br>
Agents which, after contacting a cell from a disease model, either directly or indirectly, and either in vivo or in vitro (e.g., in tissue culture), modulate the indicator expression pattern, might act as therapeutic agents or drugs for organisms (e.g., human or other animal patients, or plants) suffering from the disease. Agents can also modulate expression patterns by contacting the nucleic acid directly, e.g., in an in vitro (test tube) expression system. As used herein, "modulate" means to cause to increase or decrease the amount and/or activity of a molecule or the like which is involved in a measurable reaction. The combinations of the invention can be used to screen for such agents. For example, a series of cells (e.g., from a disease model) can be contacted with a series of agents (e.g., for<br>
a period of time ranging from about 10 minutes to about 48 hours or more) and, using routine, art-recognized methods (e.g., commercially available kits), total RNA or rnRNA extracts can be made. If it is desired to amplify^the amount of RNA, standard procedures such as RT-PCR amplification can be used (see, e.g., Innis et al eds., (1996) PCR Protocols: A Guide to Methods in Amplification, Academic Press, New York). The extracts (or amplified products from them) can be allowed to contact (e.g., incubate with) a plurality of substantially identical arrays which comprise probes for appropriate indicator RNAs, and those agents which are associated with a change in the indicator expression pattern can be identified. Example 15 describes a high throughput assay to screen for compounds which may alter the expression of genes that are correlative with a disease state.<br>
Similarly, agents can be identified which modulate expression patterns associated with particular physiological states or developmental stages. Such agents can be man-made or naturally-occurring substances, including environmental factors such as substances involved in embryonic development or in regulating physiological reactions, or substances important in agribusiness such as pesticides or herbicides. Also, such agents can be employed in their unaltered state or as aggregates with other species; and they can be attached, covalently or noncovalently, to a binding member, either directly or via a specific binding substance.<br>
Another embodiment of the invention is a kit useful for the detection of at least one target in a sample, which comprises:<br>
a)	a surface, comprising multiple spatially discrete regions, at least two of<br>
which are substantially identical, each region comprising at least eight different anchors<br>
(oligonucleotide, or one of the other types described herein), and<br>
b)	a container comprising at least one bifunctional linker molecule, which has<br>
a first portion specific for at least one of said anchors) and a second portion that comprises<br>
a probe which is specific for at least one of said target(s).<br>
In one embodiment, there is provided a surface as in a)above and a set of instructions for attaching to at least one of said anchors a bifunctional linker molecule, which has a first portion specific for at least one of said anchors) and a second portion that comprises a probe<br>
which is specific for at least one target. The instructions can include, for example (but are not limited to), a description of each of the anchors on the surface, an indication of how many anchors there are and where on the-surface they are located, and a protocol for specifically attaching (associating, binding, etc.) the linkers to the anchors. For example, if the anchors are oligonucleotides, the instructions can include the sequence of each anchor, from which a practitioner can design complementary anchor-specific moieties of linkers to interact specifically with (e.g., hybridize to) the anchors; if the anchors are peptides, the instructions can convey information about, e.g., antibodies which will interact specifically with the peptides. The instructions can also include a protocol for associating the anchors and linkers, e.g., conditions and reagents for hybridization (or other type of association) such as temperature and time of incubation, conditions and reagents for removing unassociated molecules (e.g., washes), and the like. Furthermore, the instructions can include information on the construction and use of any of the types of control linkers discussed herein, and of methods, e.g., to quantitate, normalize, "fine-tune" or calibrate assays to be performed with the combinations. The instructions can encompass any of the parameters, conditions or embodiments disclosed in this application, all of which can be performed routinely, with conventional procedures, by one of skill in the art.<br>
As discussed elsewhere in this application, a practitioner can attach to a surface of the invention comprising a given array (or arrays) of anchors, a wide variety of types of linkers, thereby programming any of a wide variety of probe arrays. Moreover, a practitioner can remove a given set of linkers from a surface of the invention and add to it another set of linkers (either the same or different from the first set), allowing a given surface to be reused many times. This flexibility and reusability constitute further advantages of the invention.<br>
In another embodiment, combinations of the invention can be used to map ESTs (Expressed Sequence Tags). That is, MAPS assays can be used to determine which, if any, of a group of ESTs were generated from different (or partially overlapping) portions of the same gene(s), and which, if any, are unique. Figures 18, 19, 20 and 21 illustrate such an assay, in this example an assay to determine which, if any, of 16 ESTs are "linked" to a<br><br>
common gene. A first step of the assay (see Figure 18) is to assemble arrays in which each of the ESTs to be mapped is represented by. at least one oligonucleotide probe that corresponds to it. A number of arrays equal to~(or greater than) the number of ESTs to be mapped are distributed in separate regions (e.g., wells) of a surface; in the illustrated example, the surface of the combination comprises 16 wells, each of which contains an array of 16 different EST-specific oligonucleotides, numbered 1-16. An oligonucleotide which "corresponds to" an EST (is "EST-specific") is one that is sufficiently complementary to an EST such that, under selected stringent hybridization conditions, the oligonucleotide will hybridize specifically to that EST, but not to other, unrelated ESTs. An EST-corresponding oligonucleotide of this type can bind specifically (under optimal conditions) to the coding or non-coding strand of a cDNA synthesized from the gene from which the EST was originally generated or to an mRNA synthesized from the gene from which the EST was originally generated. Factors to be considered in designing oligonucleotides, and hybridization parameters to be optimized in order to achieve specific hybridization, are discussed elsewhere in this application. In order to assemble the arrays, linker molecules are prepared, each of which comprises a moiety specific for one of the anchors of a generic array plus a moiety comprising an oligonucleotide probe that corresponds to one of the ESTs to be mapped; and the linkers are attached to anchors as described elsewhere in this application. In a subsequent step, an aliquot of a sample comprising a mixture of nucleic acids (e.g., mRNA or single stranded or denatured cDNA), which may contain sequences that are complementary to one or more of the oligonucleotide probes, is added to each of the regions (wells) which comprises a probe array; the mixture is then incubated under routinely determined optimal conditions, thereby permitting nucleic acid to bind to complementary probes. If several of the EST-specific probes are complementary to different portions of a single nucleic acid, that nucleic acid will bind to each of the loci in the array at which one of those probes is located.<br>
In a subsequent step, a different detector oligonucleotide (in the illustrated example, detectors #1 to 16) is added to each region (well) (see Fig. 19). A detector oligonucleotide is designed for each of the ESTs to be mapped. Each EST-specific detector corresponds to<br><br>
a different (at least partially non-overlapping) portion of the EST than does the probe oligonucleotide, so that the probe and the detector oligonucleotides do not interfere with one another. Consider, for example, the ESTs depicted in Figure 21, which correspond to ESTs 1, 2 and 6 of Figures 18-20. Figure 21 indicates that ESTs #1 and #2 were both obtained from gene X (they are "linked"), whereas EST #6 was obtained from a different, unrelated gene. If aliquots of a sample containing a mixture of mRNAs, including one generated from gene X, are incubated with the probe arrays shown in Figures 18-20, the gene X mRNA will, under optimal conditions, hybridize at the loci with probes 1 and 2, but not at those with probe 6. (Of course, each mRNA must be added in molar excess over the sum of the probes to which it can hybridize.) If detector oligonucleotide 1 is added to region (well) 1, it will hybridize to the gene X mRNA which is bound at loci 1 and 2 of the probe array, but not at locus 6. Similarly, if detector oligonucleotide 2 is added to another well — say, well #2 — it will also bind at loci 1 and 2, but not 6. In this fashion, one can determine in a high throughput manner which of the ESTs are linked, i.e. code for portions of the same gene, and which ESTs are unique. For the hypothetical example shown in Fig. 20, the first 3 ESTs encode portions of the same gene, the last 5 ESTs encode portions of another gene, and the remaining ESTs appear not to be linked. Conditions of hybridization, optional wash steps, methods of detection, and the like are discussed elsewhere in this application with regard to other MAPS assays. In order to confirm the linkage data obtained by the MAPS assay," one could perform PCR reactions using pairs of EST-specific oligonucleotide probes as sense and anti-sense primers. Every pair of linked ESTs should yield a PCR product. Note that this pairwise PCR test could be performed to determine linkage directly without using the Linkage MAPS assay; however, many reactions would be required, and each EST primer would have to be synthesized as both sense and anti-sense strands. For the illustrated example, 180 such reactions would be required.<br>
In one aspect, the invention relates to a method of determining which of a plurality of ESTs are complementary to a given nucleic acid, comprising,<br>
a)         incubating an immobilized array of oligonucleotide probes, at least one of which corresponds to each of said ESTs, with a test sample which may contain said given<br>
nucleic acid, to obtain a hybridization product between said oligonucleotide probes and said nucleic acid,<br>
b)	incubating said hybridization proauci wiui a detector oligonucleotide, which<br>
corresponds to an EST to which one of said oligonucleotide probes corresponds, but which<br>
is specific for a different portion of the EST than is said oligonucleotide probe, and<br>
c)	detecting which oligonucleotide probes of said array are labeled by said<br>
detector oligonucleotide,<br>
wherein said array of oligonucleotide probes is immobilized on a region of a combination, wherein said combination comprises<br>
1)	a surface comprising a number of spatially discrete, substantially identical,<br>
regions equal to the number of ESTs to be studied, each region comprising<br>
2)	a number of different anchors equal to the number of ESTs to be studied, each<br>
anchor in association with<br>
3)	a bifunctional linker which has a first portion that is specific for the anchor, and<br>
a second portion that comprises an oligonucleotide probe which corresponds to at least one<br>
of said ESTs.<br>
In another aspect, the invention relates to a method as above, wherein of said ESTs may be complementary to said nucleic acid, and wherein each of said ESTs comprises two different oligonucleotide sequences, the first of which defines an oligonucleotide probe corresponding to said EST, and the second of which defines a detector oligonucleotide corresponding to said EST, comprising,<br>
a)	contacting a sample which comprises molecules of said nucleic acid with at<br>
least one region of a combination, wherein said region comprises an array of oligonucleotide<br>
probes, at least one of which corresponds to each of said ESTs,<br>
b)	incubating said sample with said region, thereby permitting molecules of said<br>
nucleic acid to bind to said EST-corresponding oligonucleotide probes which are<br>
complementary to portions of said nucleic acid,<br>
c)	incubating said region comprising molecules of said nucleic acid bound to<br>
one  or more  of said EST-corresponding oligonucleotide  probes  with  a detector<br>
oligonucleotide which corresponds to an EST to which a given one of the oligonucleotide probes of said array corresponds, thereby binding detector oligonucleotides to nucleic acid molecules which have bound to said given oligonucleotide probe or to other oligonucleotide probes which are complementary to said nucleic acid,<br>
d) detecting the presence of said detector oligonucleotides, thereby identifying which EST-corresponding oligonucleotide probes of said array are complementary to portions of a nucleic acid which binds to said given oligonucleotide EST-corresponding probe, thereby identifying which ESTs are complementary to said given nucleic acid wherein said array of oligonucleotide probes is immobilized on a region of a combination, wherein said combination comprises<br>
1)	a surface comprising a number of spatially discrete, substantially<br>
identical regions equal to the number of ESTs to be studied, each region comprising<br>
2)	a number of different anchors equal to the number of ESTs to be<br>
studied, each anchor in association with<br>
3)	a bifunctional linker which has a first portion that is specific for the<br>
anchor, and a second portion that comprises an oligonucleotide probe which<br>
corresponds to at least one of said ESTs.<br>
The components of an EST mapping assay can be assembled in any order. Foi example, the anchors, linkers and ESTs can be assembled sequentially; or linkers and ESTs, in the presence or absence of reporters, can be assembled in solution and then added to the anchors.<br>
In another aspect, the invention relates to a method of determining which of a plurality of ESTs are complementary to a given nucleic acid, comprising,<br>
a)	incubating a collection of bifunctional oligonucleotide linker molecules, each<br>
of which comprises a first portion which is a probe that corresponds to at least one of said<br>
ESTs, and a second portion which is specific for an anchor oligonucleotide, with a test<br>
sample which may contain said given nucleic acid, to obtain a first hybridization product<br>
between said oligonucleotide probes and said nucleic acid,<br>
b)	incubating said first hybridization product with an immobilized array of<br>
anchor oligonucleotides, wherein each anchor oligonucleotide corresponds to the anchor-specific portion of at least one of said linker .molecules, to form a second hybridization product comprising said anchors, said oligonucleotide probes and said nucleic acid, and<br>
c)	incubating either said first or said second hybridization product with a<br>
detector oligonucleotide, which corresponds to an EST to which one of said oligonucleotide<br>
probes corresponds, but which is specific for a different portion of the EST than is said<br>
oligonucleotide probe, and<br>
d)	detecting which oligonucleotide probes of said array are labeled by said<br>
detector oligonucleotide,<br>
wherein said array of anchor oligonucleotides is immobilized on a region of a combination, wherein said combination comprises<br>
1)	a surface comprising a number of spatially discrete, substantially identical,<br>
regions equal to the number of ESTs to be studied, each region comprising<br>
2)	a number of different anchors equal to the number of ESTs to be studied.<br>
Of course, the above methods for mapping ESTs can be used to map test sequences (e.g., polynucleotides) onto any nucleic acid of interest. For example, one can determine if two or more cloned DNA fragments or cDNAs map to the same genomic DNA. Such a procedure could aid, for example, in the structural elucidation of long, complex genes. In a similar manner, one can determine if one or more spliced out sequences or coding sequences map to the same genomic DNA. Such a determination could be used, for example, in a diagnostic test to distinguish between a normal and a disease condition which are characterized by differential splicing patterns. Many other applications of the mapping method will be evident to one of skill in the art.<br>
In another aspect, the invention relates to a method of determining which of a plurality of polynucleotides are complementary to a given nucleic acid,<br>
wherein one or more of said polynucleotides may be complementary to said nucleic acid, and wherein each of said polynucleotides comprises two different oligonucleotide sequences, the first of which defines an oligonucleotide probe corresponding to said polynucleotide, and the second of which defines a detector oligonucleotide corresponding to said polynucleotide, comprising,<br>
a)	contacting a sample which comprises molecules of said nucleic acid with at<br>
least one region of a combination, wherein said region comprises an array of oligonucleotide<br>
probes, at least one of which corresponds to each of said polynucleotides,<br>
b)	incubating said sample with said region, thereby permitting molecules of said<br>
nucleic acid to bind to said polynucleotide-corresponding oligonucleotide probes which are<br>
complementary to portions of said nucleic acid,<br>
c)	incubating said region comprising molecules of said nucleic acid bound to<br>
one or more of said polynucleotide-corresponding oligonucleotide probes with a detector<br>
oligonucleotide which corresponds to a polynucleotide to which a given one of the<br>
oligonucleotide probes of said array corresponds, thereby binding detector oligonucleotides<br>
to nucleic acid molecules which have bound to said given oligonucleotide probe or to other<br>
oligonucleotide probes which are complementary to said nucleic acid,<br>
d)	detecting the presence of said detector oligonucleotides, thereby identifying<br>
which   polynucleotide-corresponding   oligonucleotide   probes   of   said   array   are<br>
complementary to portions of a nucleic acid which binds to said given oligonucleotide<br>
polynucleotide-corresponding probe,  thereby identifying which polynucleotides are<br>
complementary to said given nucleic acid,<br>
wherein said array of oligonucleotide probes is immobilized on a region of a combination, wherein said combination comprises<br>
1)	a surface comprising a number of spatially discrete, substantially identical,<br>
regions equal to the number of polynucleotides to be studied, each region comprising<br>
2)	a number of different anchors equal to the number of polynucleotides to be<br>
studied, each anchor in association with<br>
3)	a bifunctional linker which has a first portion that is specific for the anchor, and<br>
a second portion that comprises an oligonucleotide probe which corresponds to at least one<br>
of said polynucleotides.<br>
In another aspect of the invention, the above methods to map ESTs or other polynucleotides further comprise removing unbound portions of the sample between one or<br>
more of the steps.<br>
In another embodiment of the invention, one or more RNA targets of interest (e.g., mRNA, or other types of RNA) are converted into cDNAs by reverse transcriptase, and these cDNAs are then hybridized to a probe array. This type of assay is illustrated schematically in Figure 8. RNA extracts (or purified mRNA) are prepared from cells or tissues as described herein. Reverse transcriptase and oligonucleotide primers which are specific for the RNAs of interest are then added to the RNA sample, and, using art-recognized conditions and procedures, which can be routinely determined and optimized, the first strands of cDNAs are generated. The term "specific" primer refers to one that is sufficiently complementary to an mRNA of interest to bind to it under selected stringent hybridization conditions and be recognized by reverse transcriptase, but which does not bind to undesired nucleic acid (see above for a discussion of appropriate reaction conditions to achieve specific hybridization). Residual RNA — mRNAs which were not recognized by the specific primers, and/or other types of contaminating RNAs in an RNA extract, such as tRNA or rRNA -- can be removed by any of a variety of ribonucleases or by chemical procedures, such as treatment with alkali, leaving behind the single strand cDNA, which is subsequently placed in contact with a MAPS probe array. The use of reverse transcriptase in this method minimizes the need for extensive handling of RNA, which can be sensitive to degradation by nucleases and thus difficult to work with. Furthermore, the additional specificity engendered by the specific reverse transcriptase primers imparts an added layer of specificity to the assay.<br>
Optionally, the cDNAs described above can be amplified before hybridization to the probe array to increase the signal strength. The oligonucleotide reverse transcriptase primers described above can comprise, at their 5' ends, sequences (which can be about 22-27 nucleotides long) that specify initiation sites for an RNA polymerase (e.g., T7, T3 or SP2 polymerase, or the like). In the example shown in Figure 8, a T7 promoter sequence has been added to the reverse transcriptase primer. The polymerase recognition site becomes incorporated into the cDNA and can then serve as a recognition site for multiple rounds of transcription by the appropriate RNA polymerase (in vitro transcription, or rVT).<br>
Optionally, the mRNAs so generated can be amplified further, using PCR and appropriate primers, or the cDNA, itself, can be so amplified. Procedures for transcription and PCR are routine and well-known in the art.<br>
The flexibility of PCR allows for many variations in the methods of the invention. In one embodiment, one or both of the PCR primers which are used to amplify a target can comprise a chemical modification which allows the resulting PCR product to attach, specifically or non-specifically, to a solid support. Such chemical modifications include, for example, 5' amidation which allows binding to surfaces such as Costar's DNA Bind Plates, (e.g., which are modified with N-oxysuccinimide ester, or maleic anhydride coated plates such as Reacti-Bind plates from Pierce, Rockford, IL). Methods for generating oligonucleotides comprising such chemical modifications are routine and conventional in the art. A PCR product comprising such a modified primer can be attached to any desired support, including a solid support, e.g., the inner walls of a microtiter well, a bead (e.g., a non-magnetic or magnetic bead), or any of the types of surfaces described herein. Of course, a PCR primer can also be attached to a support before a PCR reaction is initiated. Several cycles of PCR can be repeated without washing but with an excess of bound primer, so that the resulting PCR product remains attached to the support. The attachment of an amplified target sequence to a support can facilitate the washing (or purification) of the target, either before it is contacted with (e.g., hybridized to) a surface comprising anchors • and/or linkers, or after it has been contacted with and then released from such a surface.<br>
In another embodiment, one or both of the PCR primers used to amplify a target can comprise one or more restriction enzyme sites, allowing the introduction of restriction sites adjacent to either end of, or flanking, a target sequence of interest. Restriction sites can be added to an amplified target by PCR either before or after it has contacted (e.g., hybridized to) a surface comprising anchors and/or linkers. Restriction site(s) introduced in this manner can, for example, facilitate the cloning of an amplified target by providing cloning sites which flank the target sequence. Restriction sites can also facilitate the purification of an amplified sequence. For example, one or more restriction sites can be placed in a PCR primer between a target specific sequence and a chemical modification which allows<br>
attachment to a support. After a target has been PCR amplified, using the modified PCR primer, and has bound to a support via the chemical modification, it can be washed and then cleaved at the restriction site(s) adjacent to the target sequence, thereby releasing the washed target. See, e.g., Figure 23.<br>
Of course, cleavable sites other than restriction enzyme sites can also be used in the methods described above, e.g., a peptide which can be cleaved by a specific protease, or another component which can be cleaved and/or released by physical, chemical or other means.<br>
In another embodiment, one or both of the PCR primers used to amplify a target can comprise a sequence (which is not necessarily present in the target) that is specific for, e.g., a target-specific reporter or a detection linker.<br>
Of course, the above-described primer modifications can be used together in any desired combination, and can be added to an amplified product at any stage of an assay. Examples 21 and 22 demonstrate protocols in which several of the primer modifications described above are incorporated into an amplified target.<br>
The above-described methods, in which mRNA targets are converted to cDNA with reverse transcriptase and/or are amplified by PCR before assaying on MAPS plates, can be used instead of the standard MAPS assay procedure for any of the RNA-based assays described above.<br>
In another embodiment of the invention, one or more nucleic acid targets of interest are hybridized to specific polynucleotide protection fragments and subjected to a nuclease protection procedure, and those protection fragments which have hybridized to the target(s) of interest are assayed on MAPS plates. If the target of interest is an RNA and the protection fragment is DNA, a Nuclease Protection/ MAPS Assay (NPA-MAPS) can reduce the need for extensive handling of RNA, which' can be sensitive to degradation by contaminating nucleases and thus difficult to work with. In such an NPA-MAPS assay, the probes in the probe array are oligonucleotides of the same strandedness as the nucleic acid targets of interest, rather than being complementary to them, as in a standard MAPS assay. One example of an NPA-MAPS assay is schematically represented in Figure 9.<br>
In an NPA-MAPS assay, the target of interest can be any nucleic acid, e.g., genomic DNA, cDNA, viral DNA or RNA, rRNA, tRNA, mRNA, oligonucleotides, nucleic acid fragments, modified nucleic acids, synthetic -nucleic acids, or the like. In a preferred embodiment of the invention, the procedure is used to assay for one or more mRNA targets which-are present in a tissue or cellular RNA extract. A sample which contains the target(s) of interest is first hybridized under selected stringent conditions (see above for a discussion of appropriate reaction conditions to achieve specific hybridization) to one or more specific protection fragment(s). A protection fragment is a polynucleotide, which can be, e.g., RNA, DNA (including a PCR product), PNA or modified or substituted nucleic acid, that is specific for a portion of a nucleic acid target of interest. By "specific" protection fragment, it is meant a polynucleotide which is sufficiently complementary to its intended binding partner to bind to it under selected stringent conditions, but which will not bind to other, unintended nucleic acids. A protection fragment can be at least 10 nucleotides in length, preferably 50 to about 100, or about as long as a full length cDNA. In a preferred embodiment, the protection fragments are single stranded DNA oligonucleotides. Protection fragments specific for as many as 100 targets or more can be included in a single hybridization reaction. After hybridization, the sample is treated with a cocktail of one or more nucleases so as to destroy substantially all nucleic acid except for the protection fragment(s) which have hybridized to the nucleic acid(s) of interest and (optionally) the portion(s) of nucleic acid target which have hybridized and been protected from nuclease digestion during the nuclease protection procedure (are in a duplexed hybrid). For example, if the sample comprises a cellular extract, unwanted nucleic acids, such as genomic DNA, tRNA, rRNA and mRNA's other than those of interest, can be substantially destroyed in this step. Any of a variety of nucleases can be used, including, e.g., pancreatic RNAse, mung bean nuclease, SI nuclease, RNAse A, Ribonuclease Tl, Exonuclease III, or the like, depending on the nature of the hybridized complexes and of the undesirable nucleic acids present in the sample. RNAse H can be particularly useful for digesting residual RNA bound to a DNA protection fragment. Reaction conditions for these enzymes are well-known in the art and can be optimized empirically. Also, chemical procedures can be used,<br>
e.g., alkali hydrolysis of RNA. As required, the samples can be treated further by well-known procedures in the art to remove unhybridized material and/or to inactivate or remove residual enzymes (e.g., phenol extraction, precipitation, column filtration, etc.). The process of hybridization, followed by nuclease digestion and (optionally) chemical degradation, is called a nuclease protection procedure; a variety of nuclease protection procedures have been described (see, e.g., Lee et al (1987). Meth. Enzymol. 152, 633-648. Zinn et al (1983). Cell 34, 865-879.). Samples treated by nuclease protection, followed by an (optional) procedure to inactivate nucleases, are placed in contact with a MAPS probe array and the usual steps of a MAPS assay are carried out. Bound protection fragments can be. detected by hybridization to labeled target-specific reporters, as described herein for standard MAPS assays, or the protection fragments, themselves, can be labeled, covalently or non-covalently, with a detectable molecule.<br>
In a preferred embodiment, the protection fragment is directly labeled, e.g., rather than being labeled by hybridization to a target-specific reporter. For example, the reporter is bound to the protection fragment through a ligand-antiligand interaction, e.g., a streptavidin enzyme complex is added to a biotinylated protection oligonucleotide. In another example, the protection fragment is modified chemically, (e.g., by direct coupling of horseradish peroxidase (HPsP) or of a fluorescent dye) and this chemical modification is detected, either with the nucleic acid portion of the protection fragment or without it, (e.g., after cleavage of the modification by, for example, ah enzymatic or chemical treatment). In any of the above methods, a protection fragment can be labeled before or after it has hybridized to a corresponding linker molecule.<br>
In order to control that the nuclease protection procedure has worked properly, i.e. that non-hybridized nucleic acids have been digested as desired, one can design one or more protection fragments to contain overhanging (non-hybridizing) segments that should be cleaved by the nucleases if the procedure works properly. The presence or absence of the overhanging fragments can be determined by hybridization with a complementary, labeled, detection probe, or the overhanging portion of the protection fragment, itself, can be labeled, covalently or non-covalently, with a detectable molecule. This control can be performed<br>
before the sample is placed in contact with the probe array, or as a part of the MAPS assay, itself. An example of such a control assay is described in Example 15. Of course, because different labels can be easily distinguished:(e..g., fluors with different absorption spectra), several differently labeled oligonucleotides can be included in a single assay. Further, the standard nuclease protection assay as analyzed by gel electrophoresis can be used during assay development to verify that the protection fragments are processed as expected.<br>
After detection of targets, the detection probe (e.g., HRP-labeled) signal can be eliminated (e.g. denatured, killed, quenched, suppressed, blocked), plates washed to remove any resulting reagents, agents, or buffers which might interfere in the next step (e.g., denaturing regent), and then the overhang can be detected with a different detection probe (e.g., also HRP-labeled). The use of signal denaturation followed by addition of a different detection probe with the same signaling moiety can be used at various stages of the assay. Utilization of two different flourescent probes and dual color detection can be used without denaturation or signal blocking.<br>
NPA-MAPS assays can be used to quantitate the amount of a target in a sample. If protection fragment is added at a large enough molar excess over the target to drive the hybridization reaction to completion, the amount of protection fragment remaining after the nuclease protection step will reflect how much target was present in the sample. One example of such a quantitation reaction is described in Examples 12 and 13.<br>
NPA-MAPS assays can be used to implement any of the methods described above that use standard MAPS assays.<br>
In a preferred embodiment, the polynucleotide protection fragments are measured by the mass spectrometer rather than on MAPS plates. In a most preferred embodiment, none of the polynucleotides are bound (attached) to a solid surface during the hybridization or nuclease digestion steps. After hybridization, the hybridized target can be degraded, e.g., by nucleases or by chemical treatments, leaving the protection fragment in direct proportion to how much fragment had been hybridized to target. Alternatively, the sample can be treated so as to leave the (single strand) hybridized portion of the target, or the duplex formed by the hybridized target and the protection fragment, to be further analyzed. The<br><br>
samples to be analyzed are separated from the rest of the hybridization and nuclease mixture (for example by ethanol precipitation or by adsorption or affinity chromatography, etc.), eluted or solubilized, and injected into the mass spectrometer by loop injection for high throughput. In a preferred embodiment, the samples to be analyzed (e.g., protection fragments) are adsorbed to a surface and analyzed by laser desorption, using well-known methods in the art. For highest sensitivity Fourier Transform Mass Spectrometry (FTMS) (or other similar advanced technique) may be used, so that femtomoles or less of each protection fragment can be detected.<br>
The protection fragments that are to be detected within one (or more) samples can be designed to give a unique signal for the mass spectrometer used. In one embodiment, the protection fragments each have a unique molecular weight after hybridization and nuclease treatment, and their molecular weights and characteristic ionization and fragmentation pattern will be sufficient to measure their concentration. To gain more sensitivity or to help in the analysis of complex mixtures, the protection fragments can be modified (e.g., derivatized) with chemical moieties designed to give clear unique signals. For example each protection fragment can be derivatized with a different natural or unnatural amino acid attached through an amide bond to the oligonucleotide strand at one or more positions along the hybridizing portion of the strand. With a mass spectrometer of appropriate energy, fragmentation occurs at the amide bonds, releasing a characteristic proportion of the amino acids. This kind of approach in which chemical moieties of moderate size (roughly 80 to 200 molecular weight) are used as mass spectrometric tags is desirable, because molecules of this size are generally easier to detect. In another example, the chemical modification is an organic molecule with a defined mass spectrometric signal, such as a tetraalkylammonium group which can, for example, derivatize another molecule such as, e.g., an amino acid. In another example, positive or negative ion signals are enhanced by reaction with any of a number of agents. For example, to enhance positive ion detection, one can react a pyrylium salt (such as, e.g., 2-4-dithenyl, 6-ethyl pyrylium tetrafiuoroborate, or many others) with an amine to form a pyridinium salt; any of a number of other enhancing agents can be used to form other positively charged functional groups (see, e.g.,<br>
Quirke et al (1994), Analytical Chemistry 66, 1302-1315). Similarly, one can react any of a number of art-recognized agents to form negative ion enhancing species. The chemical modification can be detected, of course, either after having been cleaved from the nucleic acid, or while in association with the nucleic acid. By allowing each protection fragment to be identified in a distinguishable manner, it is possible to assay (e.g., to screen) for a large number of different targets (e.g., for 2,6,10, 16 or more different targets) in a single assay. Many such assays can be performed rapidly and easily. Such an assay or set of assays can be conducted, therefore, with high throughput as defined herein.<br>
Regardless of whether oligonucleotides are detected directly by their mass or if unique molecular tags are used, the signals for each molecule to be detected can be fully characterized in pure preparations of known concentration. This will allow for the signal to be quantified (measured, quantitated) accurately. For any molecule to be detected by mass spectrometry, the intensity and profile cannot be predicted with accuracy. The tendency of the molecule to be ionized, the sensitivity of all chemical bonds within the molecule to fragmentation, the degree to which each fragment is multiply charged or singly charged, are all too complex to be predicted. However, for a given instrument with fixed energy and sample handling characteristics the intensity and profile of the signal is very reproducible. Hence for each probe the signal can be characterized with pure standards, and the experimental signals interpreted quantitatively .with accuracy.<br>
In one aspect, the invention relates to a method to detect one or more nucleic acids of interest, comprising subjecting a sample comprising the nucleic acid(s) of interest to nuclease protection with one or more protection fragments, and detecting the hybridized duplex molecules, or the protected nucleic acid, or the protection fragment, with mass spectrometry.<br>
Methods of analyzing nucleic acids by mass spectrometry are well-known in the art. See, e.g., Alper et al (1998). Science 279,2044-2045 and Koster, U.S. Pat. No. 5,605,798.<br>
In addition to the variety of high throughput assays described above, many others will be evident to one of skill in the art.<br>
An advantage of using multiprobe assays is the ability to include a number of<br>
"control" probes in each probe array which are subject to the same reaction conditions as the actual experimental probes. For example, each region in the array can comprise positive and/or negative controls. The term, a "positive- control probe," is used herein to mean a control probe that is known, e.g., to interact substantially with the target, or to interact with it in a quantitatively or qualitatively known manner, thereby acting as a(n internal) standard for the probe/target interaction. Such a probe can control for hybridization efficiency, for example. The term, a "negative control probe," is used herein to mean a control probe which is known not to interact substantially with the target. Such a probe can control for hybridization specificity, for example. As examples of the types of controls which can be employed, consider an assay in which an array of oligonucleotide probes is used to screen for agents which modulate the expression of a set of correlative genes for a disease. As an internal normalization control for variables such as the number of cells lysed for each sample, the recovery of mRNA, or the hybridization efficiency, a probe array can comprise probes which are specific for one or more basal level or constitutive house-keeping genes, such as structural genes (e.g., actin, tubulin, or others) or DNA binding proteins (e.g., transcription regulation factors, or others), whose expression is not expected to be modulated by the agents being tested. Furthermore, to determine whether the agents being tested result in undesired side effects, such as cell death or toxicity, a probe array can comprise probes which are specific for genes that are known to be induced as part of the apoptosis (programmed cell death) process, or which are induced under conditions of cell trauma (e.g., heat shock proteins) or cell toxicity (e.g., p450 genes).<br>
Other control probes can be included in an array to "fine tune" the sensitivity of an assay. For example, consider an assay for an agent which modulates the production of mRNAs associated with a particular disease state. If previous analyses have indicated that one of the correlative mRNAs (say, mRNA-A) in this set is produced in such high amounts compared to the others that its signal swamps out the other mRNAs, the linkers can be adjusted to "fine rune" the assay so as to equalize the strengths of the signals. "Blocked linkers," which comprise the anchor-specific oligonucleotide sequence designated for the mRNA-A target, but which lack the probe-specific sequence, can be added to dilute the pool<br>
of target-specific linkers and thus to reduce the sensitivity of the assay to that mRNA. The appropriate ratios of blocked and unblocked linkers can be determined with routine, conventional methods by one of skill in the-art.<br>
The "fine tuning" of an assay for a particular target by diluting an active element with an inactive element can also be done at other steps in the assay. For example; it can be done at the level of detection by diluting a labeled, target-specific reporter with an "inactive" target-specific reporter, e.g., one with the same target-specific moiety (e.g., an oligonucleotide sequence) but without a signaling entity, or with an inactivated or inactive form of the signaling entity. The term "signaling entity," as used herein, refers to a label, tag, molecule, or any substance which emits a detectable signal or is capable of generating such a signal, e.g., a fluorescent molecule, luminescence enzyme, or any of the variety of signaling entities which are disclosed herein). In an especially preferred embodiment, the "fine tuning" can be done at the step of contacting a target-containing complex with a detection linker (detection linkers are described below, e.g., in the section concerning complex sandwich-type detection methods, Example 23, and Figure 24). A set of detection linkers can be designed, e.g., to fine tune the sensitivity for each individual target in an assay. For example, if a particular target is known to be present in a sample at very high levels, the detection linker for that target can be diluted with an empirically-determinable amount of "blocked detection linker," comprising the target-specific moiety (e.g., oligonucleotide sequence) but no moiety specific for a reporter reagent, or comprising the target-specific moiety and a reporter reagent-specific moiety which is pre-bound to an inactive reporter reagent. That is, instead of comprising a moiety specific for a reporter reagent, that moiety can be absent, or prevented (e.g., blocked) from interacting with (e.g., hybridizing to) the reporter reagent. Such fine tuning is sometimes referred to herein as signal "attenuation."<br>
Samples to be tested in an assay of the invention can comprise any of the targets described above, or others. Liquid samples to be assayed can be of any volume appropriate to the size of the test region, ranging from about 100 nanoliters to about 100 microliters. In a preferred embodiment, liquid drops of about 1 micro liter are applied to each well of a 1536<br>
well microtiter dish. Samples can be placed in contact with the probe arrays by any of a variety of methods suitable for high throughput analysis, e.g., by pipetting, inkjet based dispensing or by use of a replicating pin tool. Samples are incubated under conditions (e.g., salt concentration, pH, temperature, time of incubation, etc.- see above) effective for achieving binding or other stable interaction of the probe and the target. These conditions are routinely determinable. After incubation, the samples can optionally be treated (e.g., washed) to remove unbound target, using conditions which are determined empirically to leave specific interactions intact, but to remove non-specifically bound material. For example, samples can be washed between about one and ten times or more under the same or somewhat more stringent conditions than those used to achieve the probe/target binding.<br>
Samples containing target RNA, e.g., mRNA, rRNA, tRNA, viral RNA or total RNA, can be prepared by any of a variety of procedures. For example, in vitro cell cultures from which mRNA is to be extracted can be plated on the regions of a surface, such as in individual wells of a microtiter plate. Optionally, these cells, after attaining a desired cell density, can be treated with an agent of interest, such as a stimulating agent or a potential therapeutic agent, which can be added to the cells by any of a variety of means, e.g., with a replicating pin tool (such as the 96 or 384 pin tools available from Beckman), by pipetting or by ink-jet dispensing, and incubated with the cells for any appropriate time period, e.g., between about 15 minutes and about 48 hours, depending upon the assay. Total RNA, mRNA, etc. extracts from tissues or cells from an in vitro or in vivo source can be prepared using routine, art-recognized methods (e.g., commercially available kits).<br>
Optionally, nucleic acid which might compete with an RNA of interest for hybridization to a specific probe (i.e. genomic DNA, rRNA, tRNA or mRNA which shares at least partial sequence homology with the RNA of interest) can be at least partially removed from an RNA sample by pretreating the sample with a nuclease protection (NP) procedure before subjecting it to hybridization. A nucleic acid (a "protection fragment," which can be, e.g., RNA, DNA or PNA), which is complementary to at least- a portion of the RNA of interest and whose sequence partially or completely overlaps that of the probe which is specific for the RNA of interest, is introduced in excess into the sample and<br>
incubated with it under selected stringent hybridization conditions in which the protection fragment hybridizes specifically to the RNA of interest (see above for a discussion of appropriate reaction conditions). At this step,-protection fragments specific for any or all of the RNAs of interest in the sample can be added (e.g., as many as 100, or more). After hybridization, the sample is treated with a cocktail of one or more nucleases so as to destroy substantially all nucleic acid except for the portions of each RNA of interest which are complementary to the protection fragments(s), or except for the duplexes formed between the protection fragment(s) and the protected RNA. In a subsequent step, the protection fragment(s) can be eliminated from such duplexes by denaturing the duplexes and digesting with an appropriate enzyme which will degrade protection fragment(s), leaving the protected RNA substantially intact. Any of a variety of nucleases can be used for the above-discussed digestion steps, including, e.g., pancreatic RNAse, mung bean nuclease, RNAse H, SI nuclease (under digestion conditions with either high or low salt), RNAse A, Ribonuclease Tl, Exonuclease III, Exonuclease VII, RNAse CL3, RNAse PhyM, Rnase U2, and the like, depending on the nature of the hybridized complexes and of the undesirable nucleic acids present in the sample. Reaction conditions for these enzymes are well-known in the art and can be optimized empirically. As required, the samples can be treated by well-known procedures in the art to remove unhybridized material and/or to inactivate or remove residual enzymes (e.g., phenol extraction, precipitation, column filtration, etc.). The treated samples are then placed in contact with the probe array. In order to control that specific hybridization and subsequent nuclease protection has occurred properly, one can include labeled protection fragments in the reaction mixture. In order to control that the nuclease protection procedure has worked properly, i.e. that non-hybridized nucleic acids has been digested as desired, one can design one or more protection fragments to contain overhanging (non-hybridizing) segments that should be cleaved by the nucleases if the assay works properly. The presence or absence of the overhanging fragments can be determined by hybridization with a complementary, labeled probe, or the overhanging portion of the protection fragment, itself, can be labeled with a detectable molecule.<br>
For any of the methods of this invention, targets can be labeled (tagged) by any of<br>
a variety of procedures which are well-known in the art and/or which are described elsewhere herein (e.g., for the detection of nuclease protection fragments). For example, the target molecules can be coupled directly or indirectly with chemical groups that provide a signal for detection, such as chemiluminescent molecules, or enzymes which catalyze the production of chemiluminsecent molecules, or fluorescent molecules like fluorescein or cy5, or a time resolved fluorescent molecule like one of the chelated lanthanide metals, or a radioactive compound. Alternatively, the targets can be labeled after they have reacted with the probe by one or more labeled target-specific reporters (e.g., antibodies, oligonucleotides as shown in Fig. 1, or any of the general types of molecules discussed above in conjunction with probes and targets).<br>
One type of fluorescent molecule can be an "upconverting phosphore," i.e., a fluor which absorbs and is excited at a long wavelength (e.g. IR), then emits at a shorter wavelength (e.g., visible light). Because upconverting phosphores absorb at a longer wavelength than do most potentially interfering materials present in a typical sample to be analyzed, upconverting phosphores allow a reduction in interference caused by material in the sample, compared to phosphores which absorb at a lower wavelength. The narrow emission spectrum of most upconverting phosphores also allows the simultaneous detection of a large number of different upconverting phosphores. Upconverting phosphores are well-known and conventional in the art, and include, e.g., rare earth metal ions such as, e.g., Ytterbium (Yb), Erbium (Er), Thulium (Tm) and Praseodymium (Pr), particularly in the form of an oxysulfide salt. As many as 80 or more independently detectable upconverting phosphores have been described. (See, e.g., Biological Agent Detection and Identification, April 27-30, 1999, DARPA, Biological Warfare Defense, Defense Sciences Office} The phosphores can optionally be attached to any surface, e.g., to a microsphere or a latex bead. Like other fluorescent labels, upconverting phosphores can be detected by energy transfer to (or modulation by) the label on a sufficiently close linker, target or reporter. Furthermore, as with other signaling entities disclosed herein, upconverting phosphores can be used to quantitate the amount of a target, and can be used in any of the variety of procedures described herein, e.g., to detect nuclease protection fragments.<br>
Of course, upconverting phosphores can also be used to detect targets which are distributed in any other fashion on a surface, e.g., targets (including nuclease protection fragments) which are bound directly to a surface, bound directly to an array of different oligonucleotides on a surface, or bound via bifunctional linkers to anchors (different or substantially identical) which are distributed substantially evenly, or in any desired organization or pattern, on a surface. Any surface can be used, e.g., a flow-through system, or a solid surface such as, e.g., a bead. Beads used in any of the assays of the invention can be of any type, e.g., made of any material, magnetic and/or non-magnetic; and the beads used in a single assay can be of substantially the same, or different, sizes and/or shapes.<br>
A variety of more complex sandwich-type detection procedures can also be employed. For example, a target can be hybridized to a bifunctional molecule containing a first moiety which is specific for the target and a second moiety which can be recognized by a common (i.e., the same) reporter reagent, e.g., a labeled polynucleotide, antibody or the like. The bifunctional molecules can be designed so that any desired number of common reporters can be used in each assay.<br>
For any of the methods of this invention, a variety of complex sandwich-type detection procedures can be employed to label (tag) targets. For example, a target can interact with, e.g., hybridize to, a bifunctional (or multifunctional) molecule (a "detection linker") containing a first moiety that is specific for the target and a second moiety that is. specific for a "reporter reagent." The term "specific for" has the meaning as used herein with respect to the interactions of, e.g., probes and targets. The term "reporter reagent," as used herein, refers to a labeled polynucleotide, antibody or any of the general types of molecules discussed herein in conjunction with probes and targets. These two moieties of a detection linker can recognize (interact or associate with) their respective binding partners in any of the manners discussed above in conjunction, e.g., with probes and targets. A detection linker can also comprise other sequences, e.g., sequences that are specific for a<br>
target but are different from (non-overlapping with) the target-specific moiety of the corresponding anchor-bound linker. Any sequence present in a detection linker can serve as a recognition sequence for a detection- probe or a reporter agent. In a preferred embodiment, a detection linker is a polynucleotide.<br>
Detection linkers can be designed so that any desired number of common reporter reagents can be used in an assay. For example, a set of detection linkers can be designed such that each detection linker is specific for a different target, but comprises a binding site for the same (common), or for one of a limited number of, reporter reagents. The ability to use a limited number of (e.g., one) reporter reagents to label a variety of targets in a single assay provides the advantage of reduced cost and lower backgrounds. Of course, detection linker/reporter reagent combinations can be used to detect targets which are distributed in any fashion on a surface, e.g., as described above for the types of target arrangements that can be detected by upconverting phosphores.<br>
In a most preferred embodiment, detection linkers can be designed to detect nuclease protection fragments in such a way that protection fragments which have been cleaved by a nuclease from control "overhang" sequences during a nuclease protection procedure (as described, e.g., in Example 15) are preferentially labeled. This type of detection procedure is illustrated schematically in Figure 24. In this embodiment, a detection linker comprises a first moiety that is specific for a target and a second moiety that is specific for the common control overhang sequence which, in a preferred embodiment, is present on substantially all of the nuclease protection fragments at the start of an assay. If, as desired, the control overhang sequence has been cleaved from a nuclease protection fragment during a nuclease protection reaction, the target-specific moiety of the detection linker will hybridize to the cleaved protection fragment, but the control overhang-specific moiety of the detection linker will be unbound and will remain available for further hybridization. If, on the other hand, the control overhang-specific sequence is not cleaved from a protection fragment, e.g., because of incomplete nuclease digestion during a nuclease protection procedure,, both the target-specific and the control overhang -specific moieties of the detection linker will hybridize to the protection fragment and will not be available for further hybridization. In<br>
a preferred embodiment, complexes comprising nuclease protection fragments and bound detection linkers are then hybridized in a further step to a reporter reagent which comprises a signaling entity (e.g., a fluorochrome, hapteS, enzyme, or any other molecule bearing a detectable signal or signal-generating entity, as described herein) and an moiety (e.g., an oligonucleotide) which is specific for the control overhang-specific moiety of a detection linker. The reporter reagent will preferentially bind to and label those complexes in which the control overhang sequence of the nuclease protection fragment has been cleaved off, (i.e., a complex in which the control overhang-specific moiety of the detection linker is available for further hybridization to the reporter reagent.)<br>
Numerous other variations of sandwich detection procedures will be evident to one of skill in the art.<br>
Methods by which targets can be incubated with a target-specific reporter(s) under conditions effective for achieving binding or other stable interaction are routinely determinable (see above). For example, fluorescent oligonucleotide reporters (at a concentration of about 10 nM to about 1 uM or more, preferably about 30 nM, in a buffer such as 6X SSPE-T or others) can be incubated with the bound targets for between about 15 minutes to 2 hours or more (preferably about 30 to 60 minutes), at a temperature between about 15° C. and about 45° C. (preferably about room temperature). After incubation, the samples, can optionally be treated (e.g., washed) to remove unbound target-specific reporters, using conditions which are determined empirically to leave specific interactions intact, but to remove non-specifically bound material. For example, samples can be washed between about one and ten times or more under the same or somewhat more stringent conditions than those used to achieve the target/reporter binding.<br>
Tagging with a target-specific reporter(s) can provide an additional layer of specificity to the initial hybridization reaction, e.g., in the case in which a target-specific oligonucleotide reporter is targeted to a different portion of the sequence of a target nucleic acid than is the probe oligonucleotide, or in which probe and reporter antibodies recognize different epitopes of a target antigen. Furthermore, tagging with target-specific reporters can allow for "tuning" the sensitivity of the reaction. For example, if a target mRNA which is<br><br>
part of a correlative expression pattern is expressed at very low levels, the level of signal can be enhanced (signal amplification) by hybridizing the bound target to several (e.g., about two to about five or more) target-specific' oiigonucleotide reporters, each of which hybridizes specifically to a different portion of the target mRNA.<br>
The ability to detect two types of labels independently allows for an additional type of control in MAPS assays. Some (e.g., about 10 to about 100%) of the linkers designated for a particular anchor locus (Fig. 7 shows 3 typical anchor loci, each comprising a plurality of substantially identical anchors (A, B or Q) can have a label (e.g., a fluor) attached to one end. For example, a rhodamine or Cy5 fluor can be attached at the 5' end of the linker. Such modified linkers are termed "control linkers." After a mixture of linkers and control linkers has been associated with anchors and a sample containing a target has been incubated with the resulting probe array, a target-specific reporter bearing a different fluor (e.g., fluorescein or another detection label such as a chemiluminescent one) can be used (or the target can be directly labeled with a fluor or other detection label); and the ratio of the two signals can be determined. The presence of control linkers permits calibration of the number of functional (e.g., able to interact with linkers) anchors within and between test regions (i.e. tests the capacity of each locus of the array to bind target, for purposes ,of normalizing signals), serves as a basis for quantitation of the amount of bound target, aids in localization of the anchor loci and/or provides a positive control, e.g., in cases in which there is no signal as a result of absence of target in a sample. In one embodiment of the invention, two different labels (e.g., fluorophores) can also be used to detect two different populations of target molecules; however, the ability to recognize the presence of targets by spatial resolution of signals allows the use of a single type of label for different target molecules.<br>
The ability to detect labels independently (e.g., fluorescent labels which emit at distinguishable wavelengths, such as, e.g., fluorescein and rhodamine, or different upconverting phosphores) allows additional flexibility in the methods of the invention. For example, each of two or more targets can be labeled, directly or indirectly, with its own, uniquely detectable, label. This allows for the detection of targets on the basis of features specific to the labels (e.g., color of the emission) in addition to (or instead of), e.g.,<br>
identifying the position of a localized target on a surface, or identifying a target by virtue of the size of a bead to which it is localized. In another embodiment of the invention, a multiplicity of targets can be detected independently at a single locus within a region. For example, two or more (e.g., 2, 3, 4, 5, 6 or more) targets can be detected at a locus which is defined by a single group of (substantially identical) anchors. That is, a set of linkers can be used, each of which has an anchor-specific portion specific for the same anchor plus a target-specific portion specific for a different target. If a set of, e.g., four such linkers is used, all four can bind to members of the group of anchors at a single locus, allowing four different targets to bind at that locus. If each of these targets is labeled (directly or indirectly) with a different, distinguishable, label, an investigator can determine the presence of each of the four targets at the locus independently. Therefore, an array of, e.g., five anchors (groups of anchors) in a region can be used in the scenario described above to detect as many as twenty different targets. Such an assay is illustrated in Example 24 and Figure 25. Similarly, a plurality of targets, e.g., as many as 80 or more, can be detected independently when a single type of anchor is distributed, not at a single locus, but evenly, or in any desired fashion, on a solid surface such as, e.g., a bead or a flow through apparatus; and other aspects such as bead size or scatter can be used to provide information about target identity or groups of targets.<br>
In another embodiment of the invention, "anchors" which are specific for a target(s) of interest are not associated with linkers, but rather are associated directly with the target(s); the target(s), in rum, can interact optionally with a target-specific reporter(s). Targets, whether labeled or unlabeled, can be detected by any of a variety of procedures, which are routine and conventional in the art (see, e.g., Fodor et al (1996). U.S. Pat. No. 5,510,270; Pirrung et al (1992). U.S. Pat. No. 5,143,854; Koster (1997). U.S.Pat. No. 5,605,798; Hollis et al (1997) U.S. Pat. No. 5,653,939; Heller (1996). U.S.Pat. No. 5,565,322; Eggers et al (1997). U.S.Pat. No. 5,670,322; Lipshutz et al (1995). BioTechniques 19,442-447; Southern (1996). Trends in Genetics 12, 110-115). Detection methods include enzyme-based detection, colorimetric methods, SPA, autoradibgraphy, mass spectrometry, electrical methods, detection of absorbance or luminescence (including<br>
chemiluminescence or electroluminescence), and detection of light scatter from, e.g., microscopic particles used as tags. Also, fluorescent labels can be detected, e.g., by imaging with a charge-coupled device (CCD) or fluorescence microscopy (e.g., scanning or confocal fluorescence microscopy), or by coupling a scanning system with a CCD array or photomultiplier tube, or by using array-based technology for detection (e.g., surface potential of each 10-micron part of a test region can be detected or surface plasmon resonance can be used if resolution can be made high enough.) Alternatively, an array can contain a label (e.g., one of a pair of energy transfer probes, such as fluorescein and rhodamine) which can be detected by energy transfer to (or modulation by) the label on a linker, target or reporter. Among the host of fluorescence-based detection systems are fluorescence intensity, fluorescence polarization (FP), time-resolved fluorescence, fluorescence resonance energy transfer and homogeneous time-released fluorescence (HTRF). Analysis of repeating bar-code-like patterns can be accomplished by pattern recognition (finding the appropriate spot or line for each specific labeled target by its position relative to the other spots or lines) followed by quantification of the intensity of the labels. Bar-code recognition devices and computer software for the analysis of one or two dimensional arrays are routinely generated and/or commercially available (e.g., see Rava et al (1996). U.S. Patent No. 5,545,531).<br>
Methods of making and using the arrays of this invention, including preparing surfaces or regions such as those described herein, synthesizing or purifying and attaching or assembling substances such as those of the anchors, linkers, probes and detector probes described herein, and detecting and analyzing labeled or tagged substances as described herein, are well known and conventional technology. In addition to methods disclosed in the references cited above, see, e.g., patents assigned to Affymax, Affymetrix, Nanogen, Protogene, Spectragen, Millipore and Beckman (from whom products useful for the invention are available); standard textbooks of molecular biology and protein science, including those cited above; and Cozette et al (1991). U.S. Pat. 5,063,081; Southern (1996), Current Opinion in Biotechnology I, 85-88; Chee et al (1996). Science 274T 610-614; and Fodor et al (1993). Nature 364, 555-556.<br>
Brief Description of the Drawings<br>
Fig. 1 illustrates a design scheme f6r-o?igonucleotides, in which a linker 1 contains a portion that is specific for anchor 1 and another portion (a probe) that is specific for target mRNA 1, and in which a labeled detection probe 1 is specific for a sequence of target mRNA 1 which is different from the sequence of the target-specific portion of the linker.<br>
Fig. 2 illustrates a surface which comprises 15 test regions, each of which comprises an array of six anchor oligonucleotides.<br>
Fig. 3 illustrates the design of a linker for a receptor binding assay, in which the anchor-specific portion of the linker is associated with the probe portion (the receptor protein) via biotin and streptavidin molecules, and in which a ligand specific for the receptor is labeled with a fluorescent labeling molecule. B: Biotin. SA: Streptavidin. Rec: Receptor protein. Ligand: a natural or synthetic ligand for the receptor. *: a fluorescent labeling molecule attached to the Ligand.<br>
Fig. 4 illustrates a surface which comprises 21 test regions, each of which is further subdivided into 16 subregions (indentations, dimples).<br>
Figs. 5a, 5b and 5c illustrate three pieces from which a surface such as that shown in Fig. 4 can be assembled. Fig. 5a represents a well separator; Fig. 5b represents a subdivider; and Fig. 5c represents a base.<br>
Fig. 6 represents two test regions, each of which comprises a linear array of probes (or anchors) which are in a "bar-code"- like formation.<br>
Fig. 7 schematically represents a test region comprising 3 anchors (A, B and C), each of which is present in multiple copies (a "group"). The location of each group of anchors is termed a "locus."<br>
Fig. 8 illustrates an assay in which cDNA(s) generated by specific reverse transcriptase are assayed on MAPS plates.<br>
Fig. 9 illustrates an assay which uses a nuclease protection procedure (NPA-MAPS assay). Sample RNA is prepared from cells or from tissue and is represented as thin wavy lines. To the RNA sample is added a group of polynucleotide protection fragments,<br>
portrayed as thick, dark and light lines. The dark sections of the protection fragments represent segments that are complementary to specific RNA targets and hybridize to those targets. The light sections represent overhanging portions: sequences contiguous with the complementary sequence but not complementary to target. The protection fragments are added in excess. Following hybridization of all available target to the protection fragments, the samples are treated with an appropriate cocktail of nucleases and with chemical treatments that destroy unwanted non-hybridized RNA and non-hybridized polynucleotide. For example, SI nuclease can destroy any single stranded DNA present. Hence', excess protection fragment is hydrolyzed as is the overhanging non-hybridized portion of bound protection fragment. RNA can be hydrolyzed by addition of ribonucleases including ribonuclease H and or by heating samples in base. Remaining is a collection of cleaved protection fragments that reflect how much of each target RNA had been present in the sample. The remaining protection fragments are measured by a MAPS hybridization assay.<br>
Fig. 10 illustrates hybridization specificity in a MAPS assay.<br>
Fig. 11 illustrates binding kinetics of an anchor to a linker.<br>
Fig. 12 illustrates a MAPS assay of two oligonucleotide targets.<br>
Fig. 13 illustrates the quantification of a sensitivity shift.<br>
Fig. 14 illustrates melting temperature determinations for four oligonucleotide linker/anchor combinations.<br>
Fig. 15 illustrates an mRJMA assay by NPA-MAPS.<br>
Fig. 16 illustrates a dilution curve with NPA-MAPS.<br>
Fig. 17 illustrates an assay to detect peptides containing phosphotyrosine residues.<br>
Fig. 18 illustrates the first step in an assay to map ESTs: assembling linkers corresponding to each of the ESTs to be mapped on arrays of generic anchors on a MAPS plate. To the surface of each of 16 wells of a microplate are attached linkers comprising 16 different oligonucleotide probes, arranged in a 4x4 matrix. The first locus has oligo 1, which is complementary to a portion of the first EST sequence, and so on for the 16 ESTs to be tested.<br>
cDNAs or mRNAs generated from the genes from which the ESTs were obtained are<br>
added to all 16 wells and allowed to hybridize under appropriate conditions. Hence, any cDNA or mRNA that contains one of the 16 EST sequences will be assembled at the locus where its complementary probe was placed. -<br>
Fig. 19 illustrates a subsequent step in an assay to map ESTs: adding detector oligonucleotides to the MAPS plate. Each well of the plate receives a detector oligonucleotide which corresponds to one of the ESTs to be mapped. Each detector oligonucleotide is an oligonucleotide coupled to a molecule used for detection, e.g., fluorescein if fluorescence imaging is to be the method of detection. Each detector oligonucleotide is complementary to one of the ESTs, but different from the EST-specific probe, so that a probe and a detector oligonucleotide which are complementary to a single EST can both bind at the same time.<br>
After washing, a single detector oligonucleotide is added to each well, as numbered in the figure. That is, the detector oligonucleotide with sequences complementary to the first EST is added to the first well, and so on.<br>
Fig. 20a and b illustrate the results of the assay to map ESTs shown in Figs.- 18 and 19. After hybridization of detector oligonucleotides and washing with appropriate .conditions of stringency, the 16 wells of the microplate are imaged with a CCD-based fluorescence imager, for example. Fig. 20a shows stylized results. It is expected that each EST-specific detector oligonucleotide should label the mRNA or cDNA held down by the corresponding EST-specific probe. For example, probe 5 assembles the cDNA or mRNA containing the fifth EST sequence at that locus, so the fifth detector oligonucleotide should also hybridize to the cDNA or mRNA at the same locus. This is the case for these stylized data, with each detection oligonucleotide labeling the matching probe. In addition, the first three detector oligonucleotides each label cDNA or mRNA held down by the first three probes, showing that these sequences lie along the same gene. Similarly, the last five ESTs appear to be linked. The linkage assigned from these data are presented graphically in Fig. 20b.<br>
Fig. 21 illustrates the relationships of the probes, detector oiigonucleotides and ESTs #1,2 and 6 shown in Figures 18-20.<br>
Fig. 22 illustrates a high throughput assay.<br>
Fig. 23 illustrates a method to prepare an-amplified target.<br>
Fig. 24 illustrates an assay with detection linkers and reporter agents.<br>
Fig. 25 illustrates a use of multiple flours.<br>
EXAMPLES Example 1   Hybridization Specificity (see Figure 10)<br>
A generic MAPS plate was produced by using an inkjet dispenser, the Pixus system (Cartesian Technologies, Inc., Irvine, CA) to form an identical grid of DNA within each well of a microtiter plate. All oligonucleotides were purchased from Biosource International (Camarillo, CA). For this plate, seven different oligonucleotide anchors were dispensed within each well in the pattern shown as the Key (left side of the figure). Each oligonucleotide was dispensed as a 10 nanoliter droplet to two spots, from a 2 uM solution containing 500 mM sodium phosphate pH 8.5 and 1 mM EDTA to the wells of a DNA Bind plate (Coming Costar), and allowed to dry. After attachment, wells were blocked with 50 mM Tris pH 8, and then oligonucleotide that had not covalently attached to the surface was washed away with 0.1% SDS in 5x SSP buffer.<br>
To the washed plate fluorescently labeled linker oligonucleotides were added and allowed to hybridize in 6x SSPE with 0.1% Triton X-100 at room temperature for thirty minutes. This is a preferred protocol for attachment of linkers. The linker oligonucleotides were cy5-derivatized during synthesis, and were complementary in 25 base-pair segments to specific anchoring oligonucleotides. The sequences of the seven anchors and linkers were as follows (all shown 5'to 3'):<br>
(Seqs ID Removed)<br>
'Anchors were synthesized with C12 spacer with amide at the 5' end "Linkers were synthesized with Cy5 attached at the 5' end ""Detector Oligonucleotides were synthesized with biotin attached at the 5' end<br>
To each well either one linker or a mixture of linkers (as indicated in the figure) was added in bulk. (To the well marked "all" was added a mixture of all seven linkers.) Following incubation and washing in 5x SSP 3 times, the fluorescence picture shown on the right portion of the figure was taken with a Tundra imager (IRI, St. Catherines, Ontario). As can be seen, the linkers self-assembled to the surface, by specifically associating with their complementary anchors.<br>
This process is repeated except that eight different anchors are dispersed in each well and linkers subsequently preferentially associated therewith. The entire process is repeated with 36, 64 etc. different anchors in each well of a 24, 96, 384, 864 or 1536 well plate.<br>
Example 2   Binding Kinetics (see Figure 11)<br>
The rate of hybridization of Cy5-derivatized linker number 1 to its complementary attached anchor is shown, for different concentrations of linker. The generic MAPS plate was prepared as for figure 1, except anchor 1 was attached at four spots per well. Incubations were done at room temperature in 5x SSP with 0.1% tween-20, wells were washed 3 times with 5x SSP, and bound fluorescence was measured. A fluorescence picture of the plate was taken with the Tundra, and background was subtracted and the integrated intensity of each spot within each well was calculated with Tundra software. Plotted is the average and standard deviation for the integrated intensity for the four spots within each of two duplicate wells.<br>
Example 3    Fluorescent Linker.<br>
A generic MAPS plate is produced with one anchoring oligonucleotide spotted to either 1 spot per well (top two rows), 4 spots per well (next four rows) or 16 spots per well (lower two rows), according to the methods discussed above. To each well complementary,, fluorescently labeled, linker is attached by the preferred protocol described in Example 1. Following washing the fluorescence picture of the plate is taken with the Tundra. The amount of fluorescence at each spot reports how much functional linker is available to hybridize to target. The amount of signal detected at repeated spots is highly reproducible.<br>
Example 4   Binding Carves.<br>
To the plate prepared as described in Example 3, is added different concentrations of a target oligonucleotide. The linker that has been associated contains a 25-mer sequence complementary to a portion of the target. The target is added in 5x SSC with 0.05% SDS in a total volume of either 30 or 100 microliters, and the plate is covered and incubated at 50° C overnight. Following hybridization of the target to the attached linker, the target is visualized by a preferred protocol using chemiluminescence. A biotinylated detector oligonucleotide, containing a 25-mer sequence complementary to a separate portion of the target (not to the same portion complementary to linker) is added at 30 nM. Biotinylated detector can be added for 30 minutes after washing away excess unattached target, or it can be added along with target for the length of the overnight hybridization. Following attachment of detector, the surface is washed twice with 5x SSC, once with Ix SSP containing 0.1% Tween-20 and 1% PEG (SSPTP), and a 1:50,000 dilution of 250 ug/ml Horse Radish Peroxidase conjugated to Streptavidin (HRP:SA, from Pierce, Rockford, 111.) is added for 5 hours in SSPTP at room temperature. Wells are washed four times with SSPTP, and washed once and then incubated with Super Signal Ultra reagent (Pierce). After a few minutes, pictures of luminescence are collected with the Tundra imager, e.g., the picture can accumulate within the CCD array for five minutes. Low levels of target can be visualized in some wells at a target concentration of as little as ~5xlO"13 M; the amount of signal generally becomes saturated at a target concentration of ~10"10 M. The amount of<br>
signal detected at repeated spots is highly reproducible.<br>
Example 5 Assay of Two Qligonucleotide^ "(see Figure 12)<br>
A binding curve demonstrating a MAPS hybridization assay using the preferred protocol discussed above for two different target oligonucleotides is shown. A generic MAPS plate was prepared with four different anchoring oligonucleotides each spotted four times within each well. For the second and fourth anchor, complementary linker oligonucleotides were self-assembled onto the surface as described. Two targets were added at the concentrations shown in 40 microliters to each well as described, and incubated at 50° C overnight. The amount of each target attached was visualized by attaching biotinylated detection oligonucleotide specific for each target followed by HRP:SA and chemiluminescence imaging as described. In the lower panel the intensity of the image is quantified. Software that is part of the Tundra Imager package was used to scan the intensity of the images along lines between the arrows shown in the upper panel. At the lowest concentration of target, 1.1 pM, the scanned images show well-defined gaussian peaks at each spot, while there are no discemable background peaks seen in the left-most sample, at 0 concentration of target.<br>
Example 6   Sensitivity Shifting (see Figure 13)<br>
A MAPS hybridization assay can be used for measuring the concentration of a set of oligonucleotides, by binding them to a surface and labeling them. This works well for those oligonucleotides which are at modest or low concentration. Two samples can be distinguished in such a case because if one sample contains more oligonucleotide, more will bind. On the other hand, if the concentration of targeted oligonucleotide is saturating for the surface (i.e. if it is high enough to occupy all binding sites), then if the concentration goes up no more can bind, so the amount cannot be measured. However, the binding curve of a target can be shifted by adding unlabeled competing ligand..<br>
Binding data are obtained for four different oligonucleotide targets, all of which saturate the surface (i.e. reach maximal binding) at roughly 3 nM. By adding unlabeled<br>
competitive targets to all wells, the binding of labeled oligonucleotide is shifted, so that less binds at the lower concentration, and the level at which saturation occurs is moved up. One can add competitive oligonucleotides for, stay targets 1 and 3 but not 2 and 4. This shifts the sensitivity of the assay only for targets 1 and 3. In this way oligonucleotide targets of widely different concentrations can be measured within one assay well, if the relative amount of oligonucleotide expected is known.<br>
The data can be quantified as explained above for the binding of one of the oligonucleotide targets. Figure 13 shows quantitatively that including competitive oligonucleotide in the assay shifts the binding curve used to assay for this target to higher concentrations.<br>
Example 7 Melting Temperature of Four Probes (see Figure 14)<br>
The amount of four different fluorescent labeled linker oligonucleotides specifically hybridized to anchor oligonucleotides by the MAPS assay is plotted as the temperature is raised. The four oligonucleotides were first allowed to hybridize at 50° C for 1 hour at 300 nM. Then the wells were washed with SSC without probes, and the amount bound was measured as above by fluorescence (50° C point). Then the surface was incubated at 55° C for 30 minutes and the fluorescence bound measured, and so on for all temperatures presented.<br>
Example 8   Detection Methods<br>
Two detection methods can be compared directly. To a MAPS plate with four oligonucleotide anchors attached, each at four spots per well, are added two oligonucleotides to each well, with both including a covalently attached cy5 moiety or both containing a biotin group. The epi-fluorescence measurement is performed as described for viewing and measurement of the fluorescent linker. The chemiluminescence measurements are performed as described for the MAPS assay using subsequent addition of HRP:SA and a chemiluminescence substrate. The signals generated are roughly of the same magnitude. However, for the geometry of the microplates, which contain walls separating each well, and<br>
occasional bubbles of liquid or a miniscus of fluid, reflections in the epi-fluorescence images can cause interference in data interpretation.<br>
Example 9   Chemiluminescence Products<br>
Two products available as Chemiluminescence substrates for horse radish peroxidase can be compared as detection procedures for the MAPS assay. A MAPS plate is prepared as for Example 8, and incubated with biotinylated linker oligonucleotides. Then either alkaline phosphatase coupled to streptavidin (AlkPhosrSA) or HRP:SA is added, followed by washing and addition of either CDP-Star (Tropix) to the wells with AlkPhos:SA or ECL-Plus to the wells with HRP:SA. Labeling with SA derivatized enzymes and substrates is as suggested by the manufacturers for use in labeling of western blots. These two (as well as other available substrates) can both be used to assess oligonucleotide hybridization to MAPS plates.<br>
Example 10   Resolution at 0.6 mm.<br>
The resolution of the current system for MAPS assay is tested by preparing a MAPS plate with four different oligonucleotide anchors per well each spotted four times per well, with a pitch (center-to-center spacing) of 0.6 mm. Then either cy5-derivatized linkers or biotinylated linkers are hybridized and detected and scanned as above. For the epi-fluorescence measurement the resolution is higher (and pitch could likely be reduced). For the Chemiluminescence detection procedure neighboring spots are not completely separated, yet at this spacing individual peaks may be resolved unambiguously by computer deconvolution.<br>
Example 11 Test Nuclease Protection Protocol.<br>
In an assay to test for the optimal conditions for hybridization and nuclease treatment for the nuclease protection protocol, the Nuclease Protection Assay kit from Ambion (Austin, Texas) is used to provide conditions, buffers and enzymes. Eight samples are prepared in one of three buffers. Hyb Buff 1 is 100% Hybridization Buffer (Ambion); Hyb<br>
Buff 2 is 75% Hybridization Buffer and 25% Hybridization Dilution Buffer (Ambion); and Hyb Buff 3 is 50% of each. A 70-mer oligonucleotide that contains 60 residues complementary to a test mRNA is synthesized^Biosource International, Camarillo, CA) and labeled with Psoralen-fluorescein (Schleicher and Schuell, Keene, NH) following the protocol as suggested for labeling of Psoralen-biotin by Ambion. Briefly, protection fragment is diluted to 50 ug/ml in 20 uls of TE buffer(10 mM Tris, 1 mM EDTA, pH 8) boiled for 10 minutes, and rapidly cooled in ice water. Four uls of 130 ug/ml Psoralen-fluorescein in DMF is added, and the sample is illuminated for 45 minutes at 40° C with a hand-held long wavelength UV source. Free Psoralen-fluorescein is removed by extraction with saturated butanol. The mRNA used is GAPDH anti-sense mRNA, prepared from antisense plasmid (pTPJ-GAPDH-Mouse antisense Control Template from Arnbion) using T7 promoter and the MaxiScript kit (Ambion). The short protection fragment is the 60-mer complementary portion synthesized separately and similarly labeled. The sequences of the protection fragments are as follows:<br>
Full length protection fragment:	<br>
(Seq ID Removed)<br>
Hybridizations are done by mixing protection fragments at 20 nM and GAPDH mPvNA at 60 nM in 10 µls final volume for two hours at 22 ° C or 37 ° C . Following hybridization, 200 uls of a mixture of nucleases is added according to instructions from the manufacturer (Ambion Nuclease Protection Kit, 1:200 dilution of nuclease mixture) and incubated again at the same temperatures for 30 minutes. Hydrolysis is stopped with Hybridization Inhibition Buffer (Ambion), and oligonucleotides are pelleted and washed with Ethanol. 10 uls of Ix Gel Loading Buffer (Ambion) is added and oligonucleotides are separated on a 15% TBE-urea gel. The gel is swirled in running buffer for 30 minutes, put on a plastic plate and imaged with the Tundra using fluorescein filters for selecting excitation and emission wavelengths. The image is accumulated on the CCD array for 2 minutes. Best conditions<br>
are those for samples incubated in Hyb Buff 2 at 37 ° C or in Hyb Buff 3 at 22 ° C. In these samples no detectable full-length protection.fragment remains, and significant amounts of a portion of the full-length protection fragment at a size apparently the same as the short protection fragment are seen.<br>
Example 12    mRNA Assay by NPA-MAPS. (see Figure 15)<br>
The full NPA-MAPS protocol was used, with conditions for hybridization and nuclease treatment similar to those described in Example 11. Ten samples were run for the assay. All contained the same amount of the 70-mer oligonucleotide protection fragment and different amounts of GAPDH mRNA. Hybridization samples in 10 µls in 50% Hybridization Buffer and 50% Dilution Buffer containing 0.08 mg/ml Yeast RNA (Ambion) were heated to 90 ° C for 6 minutes, briefly centrifuged, heated to 70 ° C for 5 minutes, and allowed to cool to 19° C and incubated for 19 hours. 200 µls of nuclease mixture was then added to each sample for 30 minutes at 19° C. 60 µls was aliquoted from each sample for the MAPS assay. 2 ¸l of 10 N NaOH and 2 µl of 0.5 M EDTA was added, and the sample heated to 90° C for 15 minutes, 37° C for 15 minutes, and allowed to sit at room temperature for 20 minutes. Then samples were neutralized with 2 µl of 10 M HC1, and 12 µls of 20x SSC containing 2 M HEPES pH 7.5 and 200 nM biotinylated detector oligonucleotide specific for the protection fragment was added along with 1 µl of 10% SDS. Samples were mixed, heated to 80° C for 5 minutes, and two 35 µl aliquots of each sample were pipetted to two wells of a MAPS plate (each sample was split in two and run in duplicate on the MAPS plate). The plate had been prepared as for standard MAPS protocol, with self-assembled CY5-derivatized linker specific for the protection fragment already attached. The MAPS plate was covered and incubated at 50° C overnight, and detection and luminescence performed as described. In the last sample, no nucleases were added during the assay as a control to visualize how tie protection fragment alone would be detected by MAPS. In the lower portion of the figure, the intensity scan (as analyzed by the imager) for the top row of wells is presented. The amount of GAPDH mPvNA. present in the sample (that is, the amount in each duplicate well after aliquoting to the MAPS plate) is listed in the figure.<br>
The oligonucleotides used for the MAPS plates were as follows:<br>
(Seq ID Removed)<br>
"Anchors were synthesized with C12 spacer with amide at the 5' end "Linkers were synthesized with Cy5 attached at the 5' end ""Detector Oligonucleotides were synthesized with biotin attached at the 5' end<br>
Example 13   Dilution Curve, NPA-MAPS (see Figure 16)<br>
The data discussed in Example 12 and shown in Figure 15 were quantified and plotted as a dilution curve. The average and standard deviations for all eight spots of the two duplicate wells are plotted for each concentration of mRNA. A binding curve is superimposed, of the form:<br>
Fraction Bound = Max Bound *!/(!+ ICSO/L)<br>
where Max Bound is the maximum bound at saturation, Fraction Bound is the amount bound at ligand concentration, L, and the IC50 is the concentration of ligand at which the Fraction Bound is half of Max Bound. The curve is shown as red dots on the figure, drawn with a best fit value of ICSO = 4.2 femtomoles as labeled in the figure.<br>
Example 14 NPA-MAPS assay of GAPDH mRNA in a total mouse liver RNA extract<br>
A total mouse RNA extract is assayed for GAPDH mRNA with an NPA-MAPS assay and a dilution curve is made. Total RNA from mouse liver is prepared using a Qiagen kit. RNA is precipitated in 70% EtOH with 0.5 M Mg-Acetate, and resuspended in 10 µls<br>
of 5x SSC with 0.05% SDS with 1.8 nM protection fragment. The protection fragment added is an oligonucleotide 70 bases long, 60 bases of which are complementary to mouse GAPDH. Either a fragment complementary-to-mouse GAPDH mRNA is used ("protection fragment"), or the complement of the sequence is used as a negative control ("antisense fragment").<br>
RNA samples with protection fragments are heated to 90° C for 5 minutes, and hybridizations are done by bringing samples to 70° C and allowing them to cool slowly to room temperature overnight. SI nuclease (Promega) at 1:10 dilution is added in 30 µls of Ix SI Nuclease Buffer (Promega) for 30 minutes at 19° C, and stopped by 1.6 µls of 10 N NaOH and 2.7 µls of 0.5 M EDTA. Samples are heated to 90° C for 15 minutes and then 37° C for 15 minutes to denature and destroy RNA, neutralized with 1.6 µls of 10 M HC1, and incubated on MAPS plates overnight in 5x SSC with 0.05% SDS supplemented with 200 mM HEPES pH 7.5 to which 30 nM biotinylated detection oligonucleotide is added. Washing and visualization with SA-HRP is done as described. The amount of signal decreases in parallel with decreasing amounts of mouse RNA (samples include 500,170, 50, 5, or 0.5 µg of total mouse RNA. Two control samples are included to which no SI nuclease is added. Signal is seen only for the complementary protection fragment.<br>
Oligonucleotides used:<br>
For Antisense Control (same oligonucleotides as for example 12):<br>
(Seq ID Removed)<br>
"Anchors were synthesized with C12 spacer with amide at the 5' end "Linkers were synthesized with Cy5 attached at the 5' end ""Probes were synthesized with biotin attached at the 5' end<br>
Example 15    A Nuclease Protection MAPS Assay with Controls.<br>
mRNA is extracted from mouse liver and nuclease protection is performed essentially as described in Example 14, except that the GADPH specific protection fragment comprises 60 nucleotides which are complementary to mouse GAPDH, followed by 15 "overhanging" nucleotides at the 3' end of the fragment which are not complementary to the target. After hybridization and nuclease digestion the remaining protection fragment is hybridized to a MAPS plate as indicated in Example 14, except that two different Oligonucleotide detection fragments are used to detect the immobilized protection fragment. One detection fragment is complementary to the GAPDH-specific portion of the protection fragment, and the other, a control, is complementary to the 15 base overhang portion of the protection fragment. Each detection fragment is used on different replicate samples (i.e., in different wells), so that both detection fragments can be labeled with the same detection molecule. In the present example both fragments are labeled with HRP. Without the addition of nuclease, signals from both of the detection fragments are seen; whereas, when nuclease digestion is performed only the signal corresponding to the GAPDH sequences can be detected . The amount of GAPDH-specific signal is reduced relative to that observed in the absence of nuclease digestion, because the protection fragment is added at excess relative to the amount of GAPDH mRNA present. This allows the amount of GAPDH mRNA to be limiting to the protective hybridization, so that the amount of double-stranded hybrid formed (and therefore the amount of protection fragment that is protected from the nuclease) reflects the amount of mRNA. When no mRNA is included in the reaction<br>
mixture, neither signal can be detected when nucleases are added.  The above findings demonstrate that the hybridization and digestion steps of the assay occurred as desired.<br>
When protection fragments corresponding to a variety of targets are included in a given assay, each of the protection fragments can comprise the same 15 base overhang portion. This allows for one detection fragment to be used to test for remaining overhang for all samples.<br>
Example 16   A transcription assay screening for compounds that may alter the expression of genes that are correlative with a disease state.<br>
A cell line derived from a human rumor is used. It is found to express 30 genes at higher levels than do normal cells. (That is, these 30 genes are being used more than in normal cells, to make mRNA and then to make the protein for which the genes are the instructions. A transcription assay measures how much the genes are being used by measuring how much mRNA for each gene is present.) Using a nuclease protection assay on MAPS plates (NPA-MAPS), 8800 chemical compounds are tested to see if growing the cells in the presence of the compounds can reduce the expression of some of the 30 correlative genes without affecting the expression of six normal (constitutive, "housekeeping") genes. Any compounds having that effect might be useful in the future development of drugs for treating this kind of tumor.<br>
About 10,000 to 100,000 cells are added to each well of 100 96-well polystyrene plates and the cells are grown for 2 days until they cover the surface of each well. For 8 wells of each plate, the cells are left to grow without additions. To the remaining 88 wells of each plate, a different chemical compound is added so that the effect of it alone can be tested. For the 100 plates used at one time, 8800 compounds can be tested or screened. The cells are grown for 24 hours in the presence of the compounds, and then the cells are harvested for assay. The cells in each plate are treated according to the instructions for preparing RNA in samples from 96-well plates (for example according to the Qiagen RNeasy 96 kit). After the RNA is prepared, the amount of each of 36 different mRNA species is quantified by the NPA-MAPS approach, including the 30 correlative genes and<br>
6 normal "housekeeping" genes. 36 DNA oligonucleotide protection fragments, each corresponding to one of the genes of interest, are added to each well and allowed to hybridize under selected stringent conditions to their target mRNA sequences. Then S1 nuclease is added to destroy excess unhybridized DNA, and the samples are treated chemically to destroy the RNA as well. Left is the oligonucleotide protection fragment for each of the 36 genes in proportion to how much mRNA had been present in the treated cells for each sample.<br>
One hundred 96-well plates, each of which comprises an array of a plurality of 36 different anchor oligonucleotides in each well, are prepared by adding to each well 36 different linker oligonucleotides. The linkers self-assemble on the surface of each well, converting the generic plates to MAPS plates comprising specific probes for each of the 36 oligonucleotide protection fragments. Each linker has a portion specific for one of the 36 anchors and a portion specific for a segment of one of the 36 protection oligonucleotides. The oligonucleotide sample from each well of the 100 sample plates is added to a corresponding well of the 100 MAPS plates. After hybridization under selected stringent conditions, a detection oligonucleotide for each target with a chemiluminescent enzyme attached is added, so that each specific spot of each well lights up in proportion to how much mRNA had been present in the sample. Any wells that show reduced amounts of correlative genes with no effect on the 6 house keeping genes are interesting. The compounds added to the cells for those samples are possible starting points to develop anti-tumor agents.<br>
Example 17    Induced and constitutive gene expression.<br>
RNA was prepared essentially as described in Example 14, from the livers of mice either not infected ("Control") or one hour after infection ("Infected") by adenovirus. 60 µgs of liver RNA was used for each sample, and samples were prepared in duplicate. Each assay well contained three sets of duplicate loci, corresponding to the three genes described above. Each locus contained an anchor, bound to a linker comprising a probe which was complementary to a protection fragment corresponding to one of the three genes. A nuclease protection MAPS assay was performed essentially as described in Figure 12, and<br>
the images were collected and scanned as described. Shown are the raw image data collected and the intensity scans for duplicate wells for each of the three mRNA targets. The numbers over the scan lines are the integrated intensity values and standard deviations for each condition (n = 4). The house-keeping gene, GAPDH, not expected to change, showed a modest increase of 1.3-fold in the infected sample that was not statistically significant. The transcription of MIP-2 and c-µm was increased 4 and 6-fold, respectively. These findings demonstrate that two genes, MIP-2 and c-µm, exhibit enhanced expression in response to adenovirus infection, compared to a control, constitutively expressed gene - GAPDH.<br>
Example 18 An enzyme assay screening for compounds that selectively inhibit tyrosine or serine kinases (see Figure 17).<br>
Kinases are enzymes that attach a phosphate to proteins. Many have been shown to stimulate normal and neoplastic cell growth. Hence, compounds that inhibit specific kinases (but not all kinases) can be used to test whether the kinases are involved in pathology and, if so, to serve as starting points for pharmaceutical development. For example, five tyrosine kinases that are involved in stimulating cell growth or in regulating the inflammatory response are src, Ick, fyn, Zap70, and yes.  Each kinase has substrates that are partially identified, as short peptides that contain a tyrosine. Some of the kinase specificities overlap f so that different kinases may phosphorylate some peptides equally but others preferentially. '* For the five kinases, 36 peptide substrates are selected that show a spectrum of specific and overlapping specificities.<br>
One hundred 96-well plates are used; each well comprises 36 generic oligonucleotide anchors. 36 linkers are prepared to convert the generic oligonucleotide array (with anchors only) to arrays comprising peptide substrates. The 36 peptide substrates are synthesized and each is attached covalently through an amide bond, for example, to an oligonucleotide containing a 5' amino group. The oligonucleotides contain sequences that hybridize specifically to the anchors. The peptide/oligo linkers are self assembled on the surface by adding them to all wells of the MAPS plates.<br>
For screening, the five kinases at appropriate concentrations (so that the rates of<br>
phosphorylation of the substrates are balanced as much as possible) are added to each well along with one of 8800 different compounds to be tested. The compounds are tested for their ability to directly inhibit the isolated enzymes. The amount of phosphorylation of each arrayed peptide is detected by adding labeled antibodies that bind only to peptides that are phosphorylated on tyrosine. Any wells that show a reduction in some of the phospho-tyrosine spots but not all of the spots are interesting. Compounds that had been added to those wells can be tested further as possible selective inhibitors of some of the kinases tested.<br>
The scheme of the assay is shown in the top panel of Figure 17. A chimeric linker molecule is prepared in which a 25 base pair oligonucleotide complementary to one of the anchors is crosslinked to a peptide substrate of a tyrosine phosphokinase enzyme. The chimeric oligo-peptide substrate self-assembles onto an array of oligonucleotide anchors, the kinase enzyme is used to phosphorylate the peptide portion of the chimera, and after the enzyme reaction is allowed to proceed, the amount of phosphorylation of the peptide is determined by anti-phoshotyrosine or anti-phosphoserine antibodies with an attached detection fluorophore or enzyme.<br>
The results of the assay are shown in the lower panel. The homobifunctional crosslinker, DSS (Pierce), was used to attach the 5' amino group of an oligonucleotide linker to the N terminus of a peptide synthesized with a phosphorylated tyrosine. The sequence of the peptide in single-letter code was: TSEPQpYQPGENL (SEQ ID: 32), where pY represents phosphotyrosine. The chimera was either used directly or first brought to pH 14 for 60 minutes in order to partially hydrolyze the phosphate group from the tyrosine. The phosphorylated or partially dephosphorylated chimeric molecules were self-assembled onto complementary anchor molecules within a MAPS plate at the concentrations shown for one hour. After washing and blocking the wells with 0.3% BSA in SSPTP antiphosphotyrosine antibody crosslinked to HRP (antibody 4G10 from Upstate Biotechnology, Lake Placid, NY) was added at a 1:3000 dilution in SSPTP for one hour, and the amount of antibody attached detected with chemiluminescence substrate, Super Signal Blaze. The image shown was accumulated on the CCD array for 1 minute. As expected a difference was seen in the<br>
amount of phosphate attached to the oligo-peptide. This difference is the basis for an assay measuring how active a series of kinases is when treated with different possible inhibitors.<br>
Example 19  A binding assay for the detection of selective inhibitors of the interaction between SH2 domains and phosphorylated peptides.<br>
SH2 domains serve as docking subunits of some growth regulatory proteins. The domains bind to phosphotyrosine containing proteins or peptides with imperfect specificity. That is, some phosphotyrosine peptides bind specifically to one or to few SH2 proteins while others bind widely to many SH2 proteins.<br>
For this assay, the linkers are phosphorylated peptides covalently attached to oligonucleotides. The peptide moieties are selected for their ability to bind to a group of selected SH2 proteins. The linkers convert generic MAPS plates to plates with ligands specific for the group of SH2 proteins. 100 96-well MAPS plates bearing the ligands are generated. The proteins are isolated and labeled with, for example, a cy5 fluorescent molecule.<br>
In order to screen for inhibitors of the SH2 domain/ phosphopeptide interaction, the group of labeled SH2 proteins is added to each well of the 100 96-well MAPS plates, and in each well a different test compound is added. Hence the effect of each compound individually on the interaction of the SH2 proteins with their phosphopeptide ligands is tested. The assay is to measure the fluorescence of bound SH2 protein associated with each surface-bound peptide linker. For any well showing reduced fluorescence at some spots but not all spots, the compound added can be further tested as a putative selective inhibitor of SH2 docking.<br>
Example 20    High Throughput Screening (see Figure 22)<br>
Shown is a high throughput MAPS plate demonstrating the detection of signal from 96 wells in a single experiment. Hybridization to the same oligonucleotide was measured with 16 replicates in 80 wells. As shown, the reproducibility of the 1280 hybridization assays was very high. The left-most and right-most columns served as controls to<br>
standardize the signal for different concentrations of the oligonucleotide.<br>
In a similar fashion, 16 different oligonucleotides can be tested in each well, and the test repeated in the 80 different wells of the plate. Of course, an even greater number of different oligonucleotides or other probes, (e.g., 100 nucleotide probes) can be assayed in each well, and many plates can be tested simultaneously (e.g., 100 plates, such as 96-well microtiter plates). The large number of assays which can be performed on each sample (e.g., in the latter case, about 100 different assays) and the large number of samples which can be assayed simultaneously (e.g., in the latter case, about 96 x 100, or 9600 different samples) provides for very high throughput.<br>
Example 21   Preparation of amplified target (see Figure 23)<br>
A PCR primer (Primer One) is attached to a solid support (e.g., a bead or a reaction vessel) via a chemical modification that has been introduced at the 5' terminus of the primer oligonucleotide. The primer comprises, 5' to 3', the chemical modification, a restriction enzyme site, and a sequence that is complementary to a target of interest (e.g., a cDNA copy of an mRNA of interest). The target is amplified by PCR, using as PCR primers the attached Primer One plus a Primer Two, which comprises, 5' to 3', a sequence that is specific for a detector oligonucleotide and a sequence that is complementary to a different portion of the target than that of Primer One. Following PCR amplification, the amplified target DNA is' washed to remove excess reaction material and is released from the solid support by cleavage with a restriction enzyme specific for the restriction site on Primer One. The amplified primer is thus released into the liquid phase. Thermal and/or chemical procedures can be used to deactivate the restriction enzyme and to denature the double stranded DNA product. The released, single stranded DNA target molecules can then be contacted with a surface comprising anchors and/or linkers, and the target can be detected using detector oligonucleotides complementary to the detector-specific sequences of Primer Two.<br>
Example 22   Preparation of amplified target<br>
A PCR primer (Primer One) is attached to a solid support (e.g., a bead or a reaction<br>
vessel) via a chemical modification that has been introduced into the 5' terminus of the primer oligonucleotide. The primer comprises, 5' to 3', the chemical modification, a peptide sequence which can be cleaved by a protease, -and a sequence which is complementary to a target of interest (e.g., a cDNA copy of an mRNA of interest). Instead of a peptide, any other element which can be cleaved specifically can also be used. Following PCR amplification as described, e.g., in Example 21, the PCR product, still attached to the solid support, is denatured and (optionally) washed, leaving behind a single stranded molecule attached to the support. The washed, attached, molecule can then be cleaved and released (e.g., by treatment with an appropriate protease), and contacted with a surface comprising anchors and/or linkers. Alternatively, the strand of the amplified target which is released following denaturation can be contacted with the surface comprising anchors and/or linkers. In either case, only one strand of the amplified target is contacted (e.g., hybridized) with a linker, so competition for hybridization from the opposite strand of the amplified target is eliminated and background is reduced. Linkers can be designed to be specific for either, or both, of the amplified target strands.<br>
Example 23 Assay with Detection Linkers and Reporter Agents (See Figure 24) A sample comprising an mRNA of interest is subjected to a nuclease protection procedure, using as a protection fragment an oligonucleotide which comprises a target specific moiety and a control overhang moiety, which is not complementary to the mRNA. Following nuclease digestion, the control overhang moiety can be cleaved off, as desired, as is illustrated in the left portion of the figure; or the overhang can fail to be digested, as is illustrated in the right portion of the figure. The resulting nuclease protection fragments are hybridized to a detection linker, which comprises a target-specific moiety and a control overhang-specific moiety. In the assay shown in the left part of the figure, the control overhang moiety of the detection linker remains unhybridized; by contrast, in the assay shown in the right part of the figure, the control overhang moiety of the detection linker hybridizes to the residual control overhang sequence of the protection fragment. In a subsequent step of the assay, a reporter reagent, which comprises a moiety that can interact<br>
with control overhang-specific moiety of the detection linker, is allowed to interact with the complexes. In the assay shown in the left part of the figure, the reporter reagent hybridizes to the control overhang-specific moiety of the-detection linker, which remains available for hybridization, and the complex can be detected by virtue of the signaling entity on the reporter reagent. By contrast, in the assay shown in the right part of the figure, the reporter reagent is unable to bind to the complex because the complementary sequences are not available for hybridization, so no signal is associated with the complex.<br>
In many of the assays of this invention, a reporter reagent can interact with any sequence present in a detection linker, not limited to a sequence specific for a control overhang.<br>
Example 24 Multiple Fluors (See Figure 25)<br>
A region comprising five loci, A - E, is shown in Figure 25. Each locus comprises a different group of substantially identical anchors, anchors A - E. To the anchors at locus A are hybridized four different types of linkers, each of which comprises a moiety specific for anchor A. However, each of the anchors comprises a different target-specific moiety: for targets 1, 2, 3 or 4. Therefore, after hybridization of targets to the anchor/linker complexes, targets 1, 2, 3, and 4 can all become localized at locus A. Similarly, four different types of linkers can hybridize to locus B. Each linker comprises a moiety specific for anchor B, but the target-specific moieties are specific for targets 5, 6, 7 or 8. In a similar fashion, targets 9-12 can become associated with locus C, targets 13-16 at locus D, and targets 17-20 at locus E. If each of these targets is labeled, either directly or indirectly, with a different, independently detectable fluor, such as, e.g., an upconvertable phosphore, one can independently detect all 20 targets at the five indicated loci.<br>
Example 25 An Assay in High Throughput Format<br>
In this example, a transcription assay of the invention is used to detect and quantify changes in a gene expression pattern, in a format ready for high throughput screening. All steps in the assay are performed robotically. Routine washing steps are not explicitly described. All reactions are carried out by conventional procedures, which are known in the art and/or described herein.<br>
THP-1 human monocytes are growrl in 96-well V-bottom microtiter plates, with 50,000 or 150,000 cells/well. The cells are either untreated or are differentiated with phorbol 12- myristate 13- acetate (PMA) for 48 hours, followed by activation with lipopolysaccharide (LPS) for four hours. Following treatment,,the cells are lysed in guanidium isothyocyanate and frozen until needed. roRNA is obtained using streptavidin-paramagnetic particles to which is bound biotin-poly dT. Alternately, total RNA is obtained by extraction with tri-reagent (Sigma Chemical Co., St. Louis, MO). Samples comprising either mRNA or total RNA are subjected to a nuclease protection procedure, using as DNA protection fragments a mixture of thirteen 60-mer single strand oligonucleotides, each of which comprises, 5' to 3', a 25-mer specific for one of the thirteen targets of interest (GAPDH, IL-1, TNF-α, cathepsin G, cox-2, cyclin-2, vimentin, LD78-ß, HMG-17, osteopontin,  ß-thromboglobin, angiotensin or actin); a 10-mer spacer; a 25-mer specific for a common oligonucleotide detector probe; and a 15-mer common control overhang sequence. mRNA is thereby converted into a stochiometric amount of "corresponding DNA protection fragment," which serves as target in the assay. Control experiments in which these corresponding DNA protection fragments are incubated with a probe specific for the control overhang sequence show that substantially only sequences specific for the mRNA targets of interest are present in the corresponding protection fragments, as expected if nuclease digestion has occurred as desired.<br>
Surfaces are prepared according to the methods of the invention. In each well of a 96-well DNA Bind Plate is placed an array of sixteen different 25-mer oligonucleotide anchors. Fourteen different anchor species are used. One anchor species is used at three of the four corners of the array, and 13 different anchor species are used, one each at the remaining locations in the array. The anchors are then hybridized, in a defined orthogonal pattern, to 60-mer oligonucleotide linkers, each of which comprises, 5' to 3', a 25-mer corresponding to one of the thirteen targets of interest, a 10-mer spacer, and a 25-mer specific for one of the anchors. Thus, in each of the multiply repeated 16-spot arrays, each<br>
of the thirteen target-specific linkers is localized at a defined position (locus) in the array. See Figure 18 for an illustration of such an orthogonal array. Linkers corresponding to GAPDH, a constitutiveiy expressed housekeeping gene which serves as an internal normalization control, are represented at three loci within each array. Control experiments indicate that the linkers, as well as the protection fragments and detector oligonucleotides used in the experiment, exhibit the desired specificity.<br>
Samples comprising the mixtures of corresponding protection fragments prepared as described above are hybridized to the anchor/linker arrays. Samples derived from either untreated or induced cultures are used. The presence and amount of hybridized protection fragments at each locus is then detected by hybridization to labeled detector oligonucleotides. In order to normalize the amount of signal at each locus, the detector oligonucleotides are diluted with appropriate amounts of blocked oligomers, as described herein. The amount of signal at each locus is processed and normalized to the control GAPDH signals. The data obtained are reproducible in eight replicate samples, as well as in samples prepared from three independent experiments, performed on different days. A summary of the relative abundance of the thirteen transcripts in one experiment is shown in the Table below.<br>
Relative Intensity (105 Cells/Well)<br><br>
Gene<br><br>
Control<br><br>
Induced<br><br>
Ratio<br><br>
Average    CV(n=16)     Average    CV(n=16)<br><br>
(Table Removed)<br>
Example 26 Computer Algorithm for Quantification of Multiple Array Plate Data<br>
A preferred algorithm finds the position of all spots for a MAPS plate and automatically calculates a best-fit estimate of the amplitude of the signal for each data point. Preferably, the algorithm is implemented by a computer program.<br>
1	- Select a small part of the image data, a 40x40 box, containing the intensity value of each<br>
pixel (picture element) of the image that includes the first well to be examined.<br>
2	- Define a function that calculates the intensity expected at each pixel position, using 16<br>
unknowns. The unknowns are:<br>
The amplitudes of each of 13 different microarray spots (that is, how bright are the<br>
real signals at each position of the DNA array). There are 13 of these for the<br>
4x4 (=16) spots within each well because some of the 16 spots are<br>
duplicates of the same target.<br>
The x offset and the y offset defining the exact position of the 4x4 array of spots<br>
within this particular well<br>
The background intensity of the picture within the well.<br>
The function for each pixel position calculates the distance between the pixel and each<br>
spot, and adds up the contribution that each spot makes to the intensity observed at that<br>
-pixel, by multiplying the spot amplitude by the impulse response function for the given<br>
distance. For the images used the impulse response function is defined by the sum of a<br>
Gaussian and a Lorentzian of appropriate (constant) radii.<br>
3	- Start the fitting for the current well by guessing the values of the parameters quickly. To<br>
do this, calculate the average image intensity for 16 regions of the picture where the spots<br>
are expected to be. Subtract an offset from these 16 averages, and scale the difference by a<br>
constant factor. The offset and scaling constant are defined empirically. Rearrange the<br>
results to match up the 16 spots with the 13 amplitudes. For the background and offets use<br>
any small numbers.<br>
4	- Optimize the fitted values (for the 16 unknowns) by curve fitting. In particular use a non-<br><br>
80<br>
linear least squares algorithm with Marquadt procedure for linearizing the fitting function, fitting 16 unknowns to 40x40 = 1600 equations (although of course not all equations are linearly independent).<br>
5	- Use the x,y offset as fitted for the current well to estimate with improved precision<br>
where the grid will be for the next well of the microplate. It is expected to be 9 millimeters<br>
offset relative to the next neighbor well (converted to distance in the number of pixels by<br>
the magnification factor of the imaging system). Since the distance between wells is small<br>
relative to the size of the plate, using local estimates of position is most accurate.<br>
6	- With the improved estimate of position, define a smaller box of image for the next well,<br>
moving to a 30x30 box of pixels. This makes the fitting proceed more quickly.<br>
Go back to step 2 and repeat for each well.<br>
From the foregoing description, one skilled in the art can easily ascertain the essential characteristics of this invention, and without departing from the spirit and scope thereof, can make changes and modifications of the invention to adapt it to various usage and conditions.<br>
Without further elaboration, it is believed that one skilled in the art can, using the preceding description, utilize the present invention to its fullest extent. The preceding preferred specific embodiments are, therefore, to be construed as merely illustrative, and not limitative of the remainder of the disclosure in any way whatsoever.<br>
The entire disclosure of all applications, patents and publications, cited above and in the figures are hereby incorporated by reference.<br><br><br><br><br>
WHAT IS CLAIMED IS:<br>
1.	A combination useful for the detection of one or more target(s) in a sample,<br>
which comprises, before the addition of said sample,<br>
a)	a surface, comprising multiple spatially discrete regions, at least two of which<br>
are substantially identical, each region comprising<br>
b)	at least eight different oligonucleotide anchors, each in association with<br>
c)	a bifunctional linker which has a first portion that is specific for the<br>
oligonucleotide anchor, and a second portion that comprises a probe which is specific for<br>
said target(s).<br><br>
2.	A method of detecting at least one target, comprising contacting a sample<br>
which may comprise said target(s) with the combination of claim 1, under conditions<br>
effective for said target to bind to said combination.<br>
3.	A method of detecting at least one target, comprising<br><br>
a)	contacting a sample which may comprise said target(s) with the combination<br>
of claim 1, under conditions effective for said target to bind to said combination,<br>
b)	contacting said combination and any bound targets with a labeled detection<br>
probe, and<br>
c)	detecting said detection probe.<br><br>
4.	The method of claim 2, wherein said target is a protection fragment.<br>
5.	The method of claim 2, wherein said target(s) are amplified by PCR before<br>
said sample is contacted with said combination.<br>
6.	The method of claim 5, wherein said target(s) are amplified by PCR using<br><br>
two PCR primers, either or both of which comprises a chemical modification which allows the primer to attach to a solid surface.<br>
7.	The method of claim 5, wherein said target(s) are amplified by PCR using<br>
two PCR primers, either or both of which comprises one or more restriction enzyme sites.<br>
8.	The method of claim 5, wherein said target(s) are amplified by PCR-using<br>
two PCR primers, either or both of which comprises one or more peptide sequences which<br>
can be cleaved by a protease.<br>
9.	The method of claim 5, wherein said target(s) are amplified by PCR using<br>
two PCR primers, either or both of which comprises a sequence which is specific for said<br>
detection probe.<br>
10.	The method of claim 3, wherein said labeled detection probe comprises an<br>
upconverting phosphore.<br>
11.	The method of claim 3, wherein said combination and any bound targets are<br>
contacted with at least two different labeled detection probes, each of which labeled probe<br>
comprises a different upconverting phosphore.<br>
12.	The method of claim 4, further comprising<br><br>
a)	contacting said combination and any bound targets with a labeled detection<br>
probe, and<br>
b)	detecting said labeled detection probe,<br>
wherein said labeled detection probe comprises an upconverting phosphore.<br>
13.	The method of claim 12, wherein said combination and any bound targets are<br>
contacted with at least two labeled detection probes, each of which labeled probe comprises<br>
a different upconverting phosphore.<br>
14.	The method of claim 2, further comprising<br>
a)	contacting said combination and any bound targets with a detection linker, which<br>
comprises a moiety specific for said target and a moiety specific for a reporter reagent,<br>
which interacts with said .detection linker and which comprises a signaling entity,<br>
b)	contacting said detection linker with said reporter reagent, and<br>
c)	detecting said signaling entity.<br><br>
15.	The method of claim 14, wherein said target is a nuclease protection<br>
fragment.<br>
16.	A method for detecting at least two targets, comprising<br><br>
a)	contacting a sample which may comprise said targets with the combination of<br>
claim 1, under conditions effective for said targets to bind to said combination,<br>
b)	contacting said combination and any bound targets with at least two detection<br>
linkers, each of which comprises a moiety specific for one of said targets and a moiety<br>
specific for a common reporter reagent,<br>
c)	contacting said detection linkers with said common reporter reagent, which<br>
interacts with said detection linkers and which comprises a signaling entity, and<br>
c) detecting said signaling entity.<br>
17.	The method of claim 16, wherein said targets are nuclease protection<br>
fragments.<br>
18.	The method of claim  16, wherein said signaling entity comprises an<br>
upconverting phosphore.<br>
19.	The method of claim 2, for detecting at least two targets, further comprising,<br>
a)	incubating an RNA extract with two or more protection fragments under<br>
conditions which are effective for hybridization of said protection fragments to RNAs of<br>
interest in said extract, wherein each of said protection fragments comprises a common 3'<br>
overhanging sequence which is not specific for said RNAs,<br>
b)	subjecting said incubated extract to treatment with one or more nucleases<br>
effective for digesting substantially all nucleic acid other than the portion of said protection<br>
fragments which have hybridized to the RNAs of interest and, optionally, the portions of<br>
said RNAs which have been hybridized,<br>
c)	removing substantially all nucleic acid material other than said protection<br>
fragments which have hybridized to said RNAs of interest, to provide a sample containing,<br>
as targets, the protection fragments,<br>
d)	contacting said combination and any bound protection fragments with at least<br>
two detection linkers, each of which comprises a moiety specific for one of said targets and<br>
a moiety specific for said common 3' overhanging sequence,<br><br>
e)	contacting said detection linkers with a reporter reagent which is specific for said<br>
common reporter reagent and which comprises a signaling entity, and<br>
f)	detecting said signaling entity.<br><br>
20.	The method of claim 19, wherein said signaling entity comprises an<br>
upconverting phosphore.<br>
21.	The method of claim 19, wherein one or more of said detection linkers is<br>
diluted with blocked detection linker.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTE3Ni1kZWwtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1176-del-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTE3Ni1kZWwtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1176-del-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTE3Ni1kZWwtY29ycmVzcG9uZGVuY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1176-del-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTE3Ni1kZWwtY29ycmVzcG9uZGVuY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1176-del-correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTE3Ni1kZWwtZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1176-del-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTE3Ni1kZWwtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1176-del-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTE3Ni1kZWwtZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1176-del-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTE3Ni1kZWwtZm9ybS0xOS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1176-del-form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTE3Ni1kZWwtZm9ybS0yLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1176-del-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTE3Ni1kZWwtZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1176-del-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTE3Ni1kZWwtZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1176-del-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTE3Ni1kZWwtZ3BhLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1176-del-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTE3Ni1kZWwtcGN0LTEwMS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1176-del-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTE3Ni1kZWwtcGN0LTMwMS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1176-del-pct-301.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTE3Ni1kZWwtcGN0LTMwNC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1176-del-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTE3Ni1kZWwtcGN0LTMwOC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1176-del-pct-308.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTE3Ni1kZWwtcGN0LTMzMi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1176-del-pct-332.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTE3Ni1kZWwtcGN0LTQwMi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1176-del-pct-402.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTE3Ni1kZWwtcGN0LTQwOC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1176-del-pct-408.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTE3Ni1kZWwtcGN0LTQwOS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1176-del-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTE3Ni1kZWwtcGN0LTQxNi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1176-del-pct-416.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="220185-process-for-the-preparation-of-monodisperse-polymer-particles.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="220187-a-process-for-the-preparation-of-porous-refractory-aggregate.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>220186</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/01176/DEL</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>28/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>11-Jul-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>16-May-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>19-Dec-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>STEPHEN FELDER</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>STEPHEN FELDER</td>
											<td></td>
										</tr>
										<tr>
											<td>2</td>
											<td>RICHARD M. KRIS</td>
											<td></td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US00/16952</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-06-21</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>09/337,325</td>
									<td>1999-06-21</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/220186-method-useful-for-concurrently-performing-multiple-biological-or-chemically-assays-and-kit-thereto by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:01:39 GMT -->
</html>
