<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/220821-a-method-of-preparing-a-dna by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:19:25 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 220821:A METHOD OF PREPARING A DNA</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD OF PREPARING A DNA</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method of preparing a DNA comprising sequences derived from the genomic RNA (gRNA) of a coronavirus said sequences having a homology of at least 60% to the natural sequence of the virus and coding for an RNA dependent RNA polymerase and at least one structural or non-structural protein, wherein a fragment of said DNA is capable of being transcribed into RNA and assembled to a virion, said method comprising the steps, wherein a coronavirus interfering defective genome is cloned under the expression of a promotor into a bacterial artificial chromosome (BAC) and the deleted sequences within the defective genome are re-inserted into said genome.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br>
FORM 2<br>
THE PATENTS ACT 1970<br>
[39 OF 1970]<br>
&amp;<br>
THE PATENTS RULES, 2003<br>
COMPLETE SPECIFICATION<br>
[See Section 10; rule 13]<br>
"A METHOD OF PREPARING A DNA "<br>
CONSEJO SUPERIOR DE INVESTIGACIONES CIENTIFICAS, of Serrano, 117, E-28006 Madrid, Spain,<br>
The following specification particularly describes the invention and the manner in which it is to be performed:<br><br><br>
This invention relates to methods of preparing a DNA or an RNA, nucleic acids obtainable by thisi method and their use as vacci-nes and for gene therapy.<br>
BACKGROUND OF THE INVENTION<br>
Advances in recombinant DNA technology have led to progress in the development of gene transfer between organisms. At this time, numerous efforts are being made to produce chemical, phar¬maceutical, and biological products of economic and commercial interest through the use.of gene transfer techniques.<br>
One of the key elements in genetic manipulation of both proka-ryotic and eukaryotic cells is the development of vectors and vector-host systems. In general, a vector is a nucleic acid molecule capable of replicating or expressing in a host cell. A vector-host system can be defined as a host cell that bears a vector and allows the genetic information it contains .to be replicated and expressed.<br>
Vectors have been developed from viruses with both DNA and RNA genomes. Viral vectors derived from DNA viruses that replicate in the nucleus of the host cell have the drawback of being able to integrate into the genome of said cell, so they are generally not very safe. In contrast, viral vectors derived from RNA viru¬ses, which replicate, in the cytoplasm of the host cell, are safer than those based.on DNA viruses, since the replication occurs through RNA outside, the nucleus. These vectors are thus very unlikely to integrate into the host cell's genome.<br>
cDNA clones have been obtained from single-chain RNA viruses with positive• polarity [ssRNA(+)], for example, picornavirus (Racanieilo &amp; Baltimore, 1981); bromovirus (Ahlguish et al. , 1984); alphavirus, a genus that includes the Sindbis virus; Semliki Forest virus (SFV) and the Venezuelan equine enceohali-<br><br>
tis virus (VEE) (Rice et al., 1987; Liljestrom and Garoff, 1991; Frolov et al., 1996; Smerdou and Liljestrom, 1999); flavivirus and pestivirus (Rice and Strauss, 1981; Lai et al., 1991; Rice et al,, 1989); and viruses of the Astroviridae family (Geigen-muller et al., 1997). Likewise, vectors for the expression of heterologous genes have been developed from clones of DNA com¬plementary to the genome of ssRNA(+) virus, for example alphavi-rus, including the Sindbis virus, Semliki Forest virus (SFV), and the Venezuelan equine encephalitis (VEE) virus (Frolov et al., 1996; Liljestrom, 1994; Pushko et al. , 1997). However, all methods of preparing recombinant viruses starting from RNA viru¬ses are still complicated by the fact that most of the viruses comprise sequences which are toxic for bacteria. Preparing a cDNA of the viral RNA and subcloning of the cDNA in bacteria therefore often leads to deletion or rearangement of the DNA sequences in the bacterial host. For this purpose most of the commonly used subcloning and expression vectors cannot be used for preparation of large DNA sections derived from recombinant RNA viruses. However, obtaining vectors, which can carry long foreign DNA sequences is required for a number of aspects in the development of pharmaceuticals, specifically vaccines.<br>
The coronaviruses are ssRNA(+) viruses that present the largest known genome for an RNA virus, with a length comprised between about 25 and 31 kilobases (kb) (Siddell, 1995; Lai &amp; Cavanagh, 1997; Enjuanes et al1; 1998). During infection by coronavirus, the genomic RNA (gRNA) replicates and a set of subgenomic RNAs (sgRNA) of positive and negative polarity is synthesized (Sethna e;t al., 1989; Sawicki and Sawicki, 1990; van der Most &amp; Spaan, 1995). The synthesis of the sgRNAs is an RNA-dependent process that occurs in the cytoplasm of the infected cell, although its precise mechanism is still not exactly known.<br>
The construction of cDNAs that code defective interfering (DI) genomes (deletion mutants that require the presence of a com¬plementing virus for their replication and transcription) of some coronaviruses, such as the murine hepatitis virus (MHV),<br><br><br>
infectious bronchitis virus (IBV), bovine coronavirus (BCV) (Chang et al. , 1994), and porcine gastroenteritis virus (TGEV) (Spanish Patent Application P9600620; Mendez et al. , 1996; Izeta et al. , 1999; Sanchez et al., 1999) has been described. However, the construction of a cDNA clone that codes a complete genome of a coronavirus has not been possible due to the large size of and the toxic sequences within the coronavirus genome.<br>
In summary, although a large number of viral vectors have been developed to replicate and express heterologous nucleic acids in host cells, the majority of the known vectors for expression of heterologous genes are not well suited for subcloning of RNA viruses. Further, the viral vectors so obtained present draw¬backs due to lack of species specificity and target organ or tissue limitation and to their limited capacity for cloning, which restricts the possibilities of use in both basic and ap¬plied research.<br>
Hence there is a need for methods to develop new vectors for expression of heterologous genes that can overcome the aforesaid problems. In particular, it would be advantagous to have large vectors for expression of heterologous genes with a high level of safety and cloning capacity, which can be designed so that their species specificity and tropism can be controlled.<br>
SUMMARY OF THE INVENTION<br>
According to the present invention the above problems are solved<br>
by a method of preparing a DNA comprising steps, wherein<br>
T<br>
[	;<br>
(a)	a DNA comprising a full length copy of the genomic RNA (gRNA) of an RNA virus; or<br>
(b)	a DNA comprising one or several fragments of a gRNA of an RNA virus, which fragments code for an RNA dependent RNA polymerase and at least one structural or non-structural protein; or<br>
(c)	a DNA having a homology of at least 60% to the sequences of (a) or (b);<br><br>
is cloned into a bacterial artificial chromosome (EAC).<br>
Surprisingly, the present inventors found that the problems encountered by the prior art methods to subclone and express large DNA sequences derived from viral gRNA can be overcome by using BACs as a cloning vector. The use of BACs has the particu¬lar advantage that these vectors are present in bacteria in a number of one or two copies per cell, which considerably limits the toxicity and reduces the possibilities of interplasmid re-combinantion.<br>
The invention further provides methods of preparing a viral RNA or a virion comprising steps, wherein a DNA is prepared accor¬ding to one of the above methods, the DNA is expressed and the viral RNA or the virion is isolated. Further, methods of prepa¬ring pharmaceuticals, specifically vaccines comprising the steps of the above methods to prepare a DNA are disclosed.<br>
According to another aspect of the present invention provides a DNA comprising sequences derived from the genomic RNA (gRNA) of a coronavirus which sequences have a homology of at least 6 0% to the natural sequence of the coronavirus and code for an RNA dependent RNA polymerase and at least one structural or non¬structural protein, wherein a fragment of said DNA is capable of being transcribed into RNA and which RNA can be assembled to a virion. The present invention also encompasses methods of prepa¬ring respective DNAs.<br>
The present invention further provides vectors, more specifical¬ly bacterial artificial chromosomes (BACs) comprising respective nucleic acids. According to a further embodiment the present in¬vention is directed to host cells and infectious, attenuated or inactivated viruses comprising the DNAs or RNAs of the present invention.<br>
The invention also provides pharmaceutical preparations, such as mono- cr multivalent vaccines comprising nucleic acids, vectors.<br><br>
host cells or virions of the present invention.<br>
Finally, the present invention provides methods for producing a virion or a viral RNA comprising steps, wherein a DNA according to the present invention is transcribed and the virions or viral RNAs are recovered, as well as viral RNAs obtainable by this method.<br>
BRIEF DESCRIPTION OF THE FIGURES<br>
Figure 1 shows the construction of a cDNA clone that codes an infective RNA of TGEV. Figure 1A shows the genetic structure of the TGEV, with the names of the genes indicated by letters and numbers (la, lb, S, 3a, 3b, E, M, N, and 7). Figure IB shows the cDNA-cloning strategy, which consisted in completing the DI-C genome. Deletions Al, A2, and A3 that have been completed to reestablish the full length of the cDNA are indicated. The num¬bers located beneath the structure of the DI-C genome indicate the nucleotides that flank each deletion in said DI-C genome. Figure 1C shows the four cDNA fragments constructed to complete deletion Al and the position of the principal restriction sites used during joining. The insertion of fragment Al produced an increase in the.toxicity of the cDNA.<br>
Figure 2 shows the structure of the pBeloBAC plasmid (Wang et al., 1997) used in cloning the infective cDNA of TGEV. The pBel-oBAC plasmid was provided by H. Shizuya and M. Simon (California Institute of Technology) and includes 7,507 base pairs (bp) that contain the replication origin of the F factor of E.coli (oriS), the genes necessary to keep one single copy of the plasmid per cell (parA, parB, parC, and repE), and the chloramphenicol-resi-stance gene (CM.'). The positions of the T7 and SP6 promoters and of the unique restriction sites are indicated. CosN: site cosN of lambda to facilitate the construction of the pBAC plasmid; lac Z: 0-galactosidase gene. Sequence loxP used during the gene¬ration of the plasmid is also indicated.<br>
Figure 3 shows the structure of the basic plasmids used in the<br><br>
construction of TGEV cDNA. The pBAC-TcDNAiCia~ plasmid contains all the information of the TGEV RNA except for one Clal-Clal fragment of 5,19 8 bp. The cDNA was cloned under the immediately early (IE) promoter of expression of cytomegalovirus (CMV) and is flanked at the 3'-end by a poly(A) tail with 2 4 residues of A, the ribozyme of the hepatitis delta virus (HDV), and the termination and polyadenylation sequences of bovine growth hor¬mone (BGH). The pBAC-B+C+D5' plasmid contains the Clal-Clal fragment required to complete the pBAC-TcDNAiCiai until the cDNA is full length. The pBAC-TcDNAF plasmid contains the full-length cDNA of TGEV. SAP: shrimp alkaline phosphatase.<br>
Figure 4 shows the differences in the nucleotide sequence of the S gene of the clones of TGEV PUR46-MAD (MAD) and Cll. The num¬bers indicate the positions of the substituted nucleotides, considering as nucleotide one of each gene the A of the initia¬ting codon. The letters within the bars indicate the correspon¬ding nucleotide in the position indicated. The letters located beneath the bars indicate the amino acid (aa) substitutions coded by the nucleotides that are around the indicated position. A6 nt indicates a 6-nucleotide deletion. The arrow indicates the position of the termination codon of the S gene.<br>
Figure 5 shows the strategy followed to rescue the infective<br>
TGEV from the full-length TGEV cDNA. The pBAC-TcDNA:L plasmid was<br>
transfected to ST ce^ls (pig testicle cells), and 48 h after<br>
transfection, the supernatant was used to infect new ST cells.<br>
The virus was passed at the times indicated. At each passage,<br>
aliquots of supernatant and of cellular monolayer were collected  <br>
 for virus titration and isolation of RNA for RT-PCR analysis,<br>
respectively. vgRNA: full-length viral RNA.<br>
Figure 6 shows the cytopathic effect (CPE) produced by the TGEV cDNA in the transfected ST cells. The absence of CPE in non— transfected (control) ST cells (Figure 6A) and the CPE observed 14 and 2 0- h after transfection with pBAC-TcDNA/L in ST cells are shown (Figures 6B and 6C, respectively).<br><br>
Figure 7 shows the evolution of the viral-titer with the passa¬ge. A graph representing the viral titer in the supernatant of two series of cellular monolayers (1 and 2) at different passa¬ges after transfection with pBAOTcDNA' is shown. Mock 1 and 2 refer to nontransfected ST cells. TcDNA 1 and 2 refer to ST cells transfected with pBAC-TcDNAPL.<br>
Figure 8 shows the results of the analysis of the sequence of the virus recovered after transfecting ST cells with pBAC— TcDNAFL. The,structure of the TGEV genome is indicated at the top of the figure. Likewise, the differences in the sequence of nucleotides (genetic markers) between the virus recovered from the pBAC-TcDNArL (TcDNA) plasmid, and TGEV clones C8 and Cll are indicated. The positions of the differences between the nucleo¬tides are indicated by the numbers located over the bar. The cDNA sequences of the TcDNA virus and of clone Cll were deter¬mined by sequencing the fragments obtained by RT-PCR (reverse— transcription and polymerase chain reaction). The sequence of clone C8 is being sent for publication (Penzes et al. , 1999) and is included at the end of this patent. The restriction patterns are shown with Clal and Drain of the fragments obtained by RT-PCR that include nucleotides 18,997 and 20,9 90 of the TcDNA and C8 viruses. The restriction patterns show the presence or absence of Clal and Drain sites in the cDNA of these viruses. The result of this analysis indicated that the TcDNA virus reco¬vered had the S-gene ' sequence expected for isolate Cll. MWM: molecular weight, markers.<br>
Figure 9 shows the results of the RT-PCR analysis of the virus recovered. The viral RNA was expressed under the control of the CMV promoter recognized by the cellular polymerase pol II. In principle, this RNA could undergo splicing during its transport to the cytoplasm. To study whether this was the case, the sites of the RNA with a high probability of splicing were determined using a program for predicting splicing sites in sequences of human DNA (Version 2.1-5.94, Department of Cell Biology, Baylor College of Medicine)  (Solovyev et al., 1994). The potential<br><br>
splicing site with maximum probability of cut had the donor site at nt 7,243 and the receiver at nt 7,570 (Figure 9A). To study whether this domain had undergone splicing, a RT-PCR fragment flanked by nt 7,078 and nt 7,802 (Figure 9B) was prepared from RNA of passages 0 and 2 of nontransfected cultures (control), or from ST cells transfected with TcDNA with the Clal fragment in reverse orientation (TcDNA ), or in the correct orienta¬tion (TcDNAr ), and the products resulting from the RT-PCR were analyzed in agarose gels. The results obtained are shown in Figures 9C (passage 0) and 9D (passage 2).<br>
Figure 10 shows the results of the immunofluorescence analysis of the virus produced in cultures of ST cells transfected with TcDNA. Staining for immunofluorescence was done with antibodies specific for the TGEV PUR46-MAD isolate, and for the virus reco-vered after transfection with the pBAC-TcDNA plasmid. For this, TGEV-specific monoclonal antibodies were used which bind to both isolates or only to PUR46-MAD (Sanchez et al., 1990). The result confirmed that the TcDNA virus had the expected antigenicity. The specific polyclonal antiserum for TGEV bound to both viru¬ses, but not to the uninfected cultures, and only the expected monoclonal antibodies specific for the S (ID.B12 and 6A.C3), M (3B.B3), and N (3B.D8) proteins bound to the TcDNA virus (San¬chez et al., 1999).<br>
Figure 11 shows the expression of GUS under different transcrip¬tion-regulatory sequences (TRSs) that vary flanking region 5' of the intergenic (IG) sequence. Minigenome M39 was cloned under the control of the CMV promoter. Inserted into this minigenome was a multiple cloning sequence (PL1, 5'-CCTAGGATTTAA-ATCCTAAGG-3'; SEQ ID NO:2) and the transcription unit formed by the selected transcription-regulating sequences (TRS), another mult i'p le cloning sequence (PL2, 5'-GCGGCCGCGCCGGCGAGGCCTGTCGAC-3'; SEQ ID NO:3; or PL3, 5'-GTCGAC-3'; SEQ ID N0:4), sequences with the structure of a Kozak (Kz) domain, the B-glucuronidase (GUS) gene, and another multiple cloning site (PL4, 5'-GCTAGCCCAGGCGCGCGGTACC-3'; SEQ ID<br><br>
NO:5). These sequences "were flanked at the 3'-end by the 3'-sequence of minigenome M39, the HDV ribozyme, and the termina¬tion and polyadenylation sequences of BGH. The TRSs had a diffe¬rent number (0, -3, -8, and -88) of nucleotides at the 5'-end of the IG sequence (CUAAAC)', and came from the N, S, or M genes, as indicated. ST cells were transfected with the different plas-mids, were infected with the complementing virus (PUR46-MAD), and the supernatants were passed 6 times. The GUS activity in the infected cells was determined by means of the protocol des¬cribed by Izeta (Izeta et al., 1999). The results obtained by relating the GUS activity to the passage number are collected in Figure 11B.<br>
Figure 12 shows the expression of GUS under different TRSs that vary in the 3'-flanking region of the IG sequence (see Figure 11A) . Using this transcription unit with the 5'-flanking region corresponding to the -88 nt of the N gene of TGEV plus the IG sequence (CUAAAC), the 3'-flanking sequences were modified. These sequences corresponded to those of the different TGEV genes (S, 3a, 3b, E, M, N, and 7), as is indicated in Figure 12A. In two cases, 3'-sequences were replaced by others that contained a restriction site (Sail) and an optimized Kozak se¬quence (Kz), or by a sequence identical to the one that follows the first IG sequence located following the leader of the viral genome. The activity of GUS in the infected cells was determined by means of the protocol described above (Izeta et al., 1999). cL12 indicates a sequence of 12 nucleotides identical to that of 3'-end of the "leader" sequence of the TGEV genome (see the virus sequence indicated at the end). The results obtained by relating the expression of GUS to the passage number are collec¬ted in Figure 12B.<br>
Figure 13 shows the effect of the s'ite of insertion of the modu-<br>
It should be noted that CTAAAC and CUAAC have the same mea¬ning for the purpose of this patent. The first represents the sequence of the DNA and the second that of the corresponding<br><br>
le of expression in the minigenome over the levels of GUS ex¬pression. The GUS transcription unit, containing -88 nt of the N gene flanking the 5'—end of the IG sequence (CUAAAC), and the Kz sequences flanking the 3'-end (see Figure 12A), was inserted into four single restriction sites in minigenome M39 (Figure 13A) to determine if all these sites were equally permissive for the expression of the heterologous gene. ST cells were trans-fected with these plasmids and infected with the complementing virus (PUR46-MAD) . The GUS activity in the infected cells was determined at passage 0 (PO) following the protocol described above (Izeta et al. , 1999). The results obtained are collected in Figure 13B.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
According to the present invention methods of preparing a DNA<br>
are provided, which comprise steps, wherein<br>
(a)	a DNA comprising a full length copy of the genomic RNA (gRNA) of an RNA virus; or<br>
(b)	a DNA comprising one or several fragments of a gRNA of an RNA virus, which fragments code for an RNA dependent RNA polymerase and at least one structural or non¬structural protein; or<br>
(c)	a DNA having a homology of at least 6 0% to the sequen¬ces of (a) or (b);<br>
i is cloned into a bacterial artificial chromosome (BAC).<br>
According to the present apolication a "bacterial artificial chromosome" is a DNA sequence which comprises the sequence of the F factor. Plasmids containing this sequences, so-called F plasmids, are capable of stably maintaining heterologous sequen¬ces longer than 300 Kb in low copy number (one or two copies per cell). Respective BACs are known in the art (Shizuya et al. , 1992) .<br>
According to the present invention the DNA cloned into the EAC<br><br>
has a homology of at least 60%, preferably 75% and more prefera¬bly 85 cr 95%, to a natural sequence of an RNA virus. Sequence homology is preferably determined using the Clustal computer program available from the European Bioinformatics Institute (EBI).<br>
According to the methods of the present invention the DNA cloned into the BAC may further comprise sequences coding for several or all except one of the structural or non-structural proteins of the virus.<br>
In a preferred embodiment of the present invention the DNA clo¬ned into the BAC further comprises sequences encoding one or several heterologous gene. According to the present application a gene is characterized as a "heterologous gene" if it is not derived from the virus which was used as a source for the genes encoding the RNA dependent RNA polymerase and the structural or non-structural protein. A "heterologous gene" thus also refers to genes derived from one type of virus and expressed in a vec¬tor comprising sequences derived from another type of virus. Any heterologous gene of interest can be inserted into the nucleic acids of the present invention. The insertion of genes encoding one or several peptides or proteins which are recognised as an antigen from an infectious agent by the immune system of a mam¬mal is especially preferred. Alternatively, the method of the present. invention may be performed using heterologous genes encoding at least one molecule interfering with the replication of an infectious agent or an antibody providing protection against an infectious agent. The heterologous sequences may contain sequences encoding an immune modulator, a cytokine, an immonenhancer and/or an anti-inflammatory compound.<br>
The method of the present invention may be performed using a DNA for cloning into a BAC that has any size. However, specific advantages over the known methods to prepare subcloned DNA from viral are obtained, if large sequences are used. The DNA cloned into the BAC may thus comprise a length of at least 5 Kb, whe-<br><br>
rein DNA with a size of at least 15, 25 or 30 Kb is specifically preferred.<br>
According to specifically preferred embodiments of the present invention methods are provided, wherein the BAC comprises a sequence controlling the transcription of the DNA cloned into the BAC. This will allow transcription of the viral RNA and thus enable expression of the virus. Any sequence controlling tran¬scription known in the art may be used for this purpose, inclu¬ding sequences driving the expression of genes derived from DNA or RNA genomes. The use of the immediately early (IE) promoter of cytomegalovirus (CMV) is preferred.<br>
The DNA cloned into the BAC may also be flanked at the 3'-end by a poly(A)tail. The nucleic acid may comprise termination and/or polyadenylation sequences of bovine growth hormone (BGH). Ad¬ditionally or alternatively, the nucleic acids may comprise sequences encoding a ribozyme, for example the ribozyme of the hepatitis S virus (HDV).<br>
Additional advantages may be achieved if at least one of the genes of the virus has been modified by substituting, deleting or adding nucleotides. For example the gene controlling tropism of the virus may be modified to obtain viruses with altered tropism. Alternativly, the gene controlling tropism of the virus has been substituted ,with the respective gene of another virus . The modification is preferably performed in the S, M and/or N genes of the virus. i<br>
In a preferred embodiment of the present invention a method is provided, wherein the DNA cloned into the BAC is capable of being transcribed into RNA which RNA can be assembled to an virion. The virion may be an infectious, attenuated, replication defective or inactivated virus.<br>
Any RNA virus may be used in the methods of the invention. The virus can for example be a picornavirus, flavivirus, togavirus,<br><br>
coronavirus, toroviruses, arterivurses, calcivirus, rhabdovirus, paramixovirus, filovirus, bornavirus, orthomyxovirus, bunyavi-rus, arenavirus or reovirus. The use of viruses naturally having a plus strand genome is preferred.<br>
Additionally, the present invention provides methods of prepa¬ring a viral RNA or a virion comprising steps, wherein a DNA is prepared according to one of above methods, the DNA is expressed in a suitable host cell and the viral RNA or the virion is iso¬lated from that host cell. Any of methods for isolating viruses from the cell culture known in the art may be used. Alternative¬ly, methods of preparing a viral RNA or a virion are disclosed, wherein the DNA of the present invention is transcribed or translated using chemicals, biological reagents and/or cell ex¬tracts and the viral RNA or the virion is subsequently isolated. For certain embodiments, the virus may subsequently be inacti¬vated or killed.<br>
The invention also provides methods for preparing a pharmaceuti¬cal composition comprising steps, wherein a DNA, a viral RNA or a virion is prepared according to one of the above methods and is subsequently mixed with a pharmaceutically acceptable adju-vans and/or carrier. A large number of adjuvans and carriers and diluents are known in the prior art and may be used in accordan¬ce with the present invention. The pharmaceutical is preferably a vaccine for protecting humans or animals against an infectious disease. The pharmaceutical can advantageously also be used for gene therapy. \ The present invention further provides for the first time a DNA comprising sequences derived from the genomic RNA (gRNA) of a coronavirus which sequences have a homology of at least 60% to the natural sequence of the coronavirus and code for an RNA dependent RNA polymerase and at least one structural or non¬structural protein, wherein a fragment of said DNA is capable of being transcribed into RNA which can be assembled to a virion.<br><br>
According to the present invention the term "sequence derived from a coronavirus" is used to refer to a nucleic acid sequence which has a homology of at least 60%, preferably 75% and more preferably 85 or 95%, to a natural sequence of a coronavirus. Sequence homology can be determined using the Clustal computer program available from the European Bioinformatics Institute (EBI).<br>
The term "coronavirus" is used according to the present inven¬tion to refer to a group of viruses having a single molecule of linear, positive sense, ssRNA of 25 to 33 Kb. These viruses usually contain 7 to 10 structural genes, i.e. genes encoding proteins that determine the viral structure. These genes are typically arranged in the viral genome in the order of 5' repli-case- (hemagglutinin-esterase) -spike-envelope-membrane-nucleopro-tein-3'. Additionally the viral genome may comprise a number of non-structural genes which encode a nested set of mRNAs with a common 3' end and are largely non-essential.<br>
The term "capable of being transcribed into RNA which can be assembled into a virion" is used to characterize a DNA sequence, which - once introduced into a' suitable host cell - will be transcribed into RNA and generate virions. The virions are pre¬ferably infectious viruses, but may also be inactivated, atte¬nuated or replication defective viruses comprising said RNA. Preferably all the information necessary for expression of the virion is encoded by the DNA sequence of the present invention.<br>
The nucleic acids of the present invention may further comprise a sequence encoding one or several heterologous genes of inte¬rest. According to the present invention a gene is characterized as a "heterologous gene" if it is not derived from the coronavi¬rus which was used as a source for the genes encoding the RNA dependent RNA polymerase and the structural or non-structural protein. A "heterologous gene" thus also refers to genes derived from one type of coronavirus and expressed in a vector compri¬sing seouences derived from another type of coronavirus. Any<br><br>
heterologous gene of interest can be inserted into the nucleic acids of the present invention. The insertion of genes encoding peptides or proteins which are recognised as an antigen from an infectious agent by the immune system of a mammal is especially preferred. The heterologous gene may thus encode at least one antigen suitable for inducing an immune response against an infectious ag^nt, at least one molecule interfering with the replication of an infectious agent or an antibody providing pro¬tection against an infectious agent. Alternatively or additio¬nally, the heterologous gene may encode an immune modulator, a cytokine, an immonenhancer or an anti-inflammatory compound.<br>
The fragment of the DNA according to the present invention which is transcribed into RNA preferably has a size of at least 25 Kb, Fragments with a size of at least 30  Kb are especially prefer¬red.<br>
According to a preferred embodiment of the present invention the DNA further comprises sequences derived from a coronavirus co¬ding for several or all except one of the structural or non¬structural proteins of a coronavirus. Alternatively, the DNA of the present invention further comprises sequences coding for all of the structural or non-structural proteins of a coronavirus.<br>
According to a further embodiment, the nucleic acids of the present invention comprise a sequence controlling the transcrip¬tion of a sequence derived from a coronavirus gRNA. Any sequence controlling transcription known in the art may be used for this purpose, including sequences driving the expression of genes derived from DNA or RNA genomes. The use of the immediately early (IE) promoter of cytomegalovirus (CMV) is preferred.<br>
The nucleic acid according to the present invention may also be flanked at the 3'-end by a poly (A) tail. The nucleic acid may comprise termination and/or polyadenylation sequences of bovine growth hormone (BGH). Additionally or alternatively, the nucleic acids may comprise sequences encoding a ribozyme, for.example<br><br>
  the ribczyme of the hepatitis 6 virus (HDV).<br>
The nucleic acids of the present invention may comprise sequen¬ces derived from any coronavirus, for example derived from an isolate of the porcine transmissible gastroenteritis virus (TGEV), murine hepatitits virus (MHV), infectious bronchitis virus (IBV), bovine coronavirus (BoCV), canine coronavirus (CCoV), feline coronavirus (FcoV) or human coronavirus. Accor¬ding to a preferred embodiment the nucleic acid is derived from a transmissable gastroenteritis virus.<br>
According to a further embodiment of the present invention, the DNAs of the present invention are part of a plasmid, preferably part of a bacterial artificial chromosome (BAC).<br>
The present invention further provides host cells comprising respective nucleic acids or vectors. The host cells may be euca-ryotes or procaryotes . Alternatively, the present invention provides virions comprising a nucleic acid according the present invention. Respective virions may for example be isolated from cell cultures transfected or infected with the nucleic acids of the present invention.<br>
According to a further embodiment, the present invention provi¬des methods for producing a virion or a viral RNA comprising steps, wherein a DNA of the present invention is introduced into a host cell, host cells containing the DNA are cultivated under conditions allowing the expression thereof and the virion or viral RNA is recovered. Additionally, methods for producing a virion or a viral RNA are provided, wherein a DNA of the present invention is mixed in vitro with chemicals, biological reagents and/or cell extracts under conditions allowing the expression of the DNA and the virion or viral RNA is recovered. The present invention also encompasses the virions and viral RNAs obtainable by either of the above methods. RNAs and virions carrying a heterologous gene are preferred. The viruses so obtained may have the form of an infectious, attenuated, replication defecti-<br><br>
ve or inactivated virus.<br>
The virus may comprise modified genes, for example a modified S, N or M gene. In a specific embodiment of the present invention the modification of the S, N or M gene gives raise to an atte¬nuated virus or a virus with altered tropism.<br>
According to a further embodiment the invention provides a phar¬maceutical preparation comprising nucleic acids, host cells or virions according to the present invention. According to a pre¬ferred embodiment the pharmaceutical, preparation is a vaccine capable of protecting an animal against deseases caused by an infectious agent. The vaccine may for example comprise sequences of at least one antigen suitable for inducing an immune response against the infectious agent or an antibody providing protection against said infectious agent. The vaccine may comprise a DNA expressing at least one molecule interfering with the replica¬tion of the infectious agent. Alternatively the vaccine may comprise a vector expressing at least one antigen capable of inducing a systemic immune response and/or an immune response in mucous membranes against different infectious agents that propa¬gate in respiratory, intestinal mucous membranes or in other tissues. The vaccine may also be a multivalent vaccine capable of protecting an animal against the infection caused by more<br>
than one infectious agent, that comprises more than one nucleic<br>
i acid of the present invention each of which expresses an antigen<br>
capable of inducing an immune response against each of said infectious agents, or antibodies that provide protection against each one of said infectious agents or other molecules that in¬terfere with the replication of any infectious agent.<br>
The vaccines of the present invention may further comprise any of the pharmaceutically acceptable carriers or diluents known in the state of the art.<br>
The present invention further provides methods for preparing a DNA of the present invention comprising steps, wherein an in-<br><br>
terfering defective genome derived from a coronavirus is cloned under the expression of a promotor into a BAG vector and the deletions within the defective genome are re-inserted. The me¬thod may further comprise steps, wherein toxic sequences within the viral genome are .identified before re-insertion into the remaining genomic DNA. Preferably, the toxic sequences within the viral genome are the last sequences to be re-inserted before completing the genome. According to the present invention this method is suitable to yield infectious clones of coronaviruses which are stable in bacteria for at least 80 generations and thus provides a very efficient cloning vector.<br>
The present invention provides the development of infective clones of cDNA derived from coronavirus (Almazan et al., 2000), as well as vectors constructed from said infective clones that also include heterologous nucleic acid sequences inserted into said clones. The infective clones and vectors provided by this invention have numerous applications in both basic :and applied research, as well as a high cloning capacity, and can be desi¬gned in such a way that their species specificity and tropism can be easily controlled.<br>
This patent describes the development of a method that makes it possible to obtain, for the first time in the history of corona¬virus, a full-length infective cDNA clone that codes the genome of a coronavirus (Almazan et al. , 2000).<br>
A new vector or system of expression of heterologous nucleic acids based on a coronavirus generated from an infective cDNA clone that codes the genomic RNA (gRNA) of a coronavirus has been developed. In one particular realization of this invention,• the coronavirus is the porcine transmissible gastroenteritis virus (TGEV).<br>
The new system of expression can be used in basic or apolied research, for example, to obtain products of interest (proteins, enzymes, antibodies, etc.), as a vaccinal vector, or in gene<br><br>
therapy in both humans and animals. The infective coronavirus obtained from the infective cDNA clone can be manipulated by conventional genetic engineering techniques so that new genes can be introduced into the genome of the coronavirus, and so that these genes can be expressed in a tissue- and species-spe¬cific manner to induce an immune response or for gene therapy. In addition, the expression has been optimized by the selection of new transcription-regulating sequences (TRS) that make it possible to increase the levels of expression more than a hun¬dredfold.<br>
The vectors derived from coronavirus, particularly TGEV, present several advantages for the induction of immunity in mucous mem¬branes with respect to other systems of expression that do not replicate in them: (i) TGEV infects intestinal and respiratory mucous membranes (Enjuanes and Van der Zeijst, 1995), that is, the best sites for induction of secretory immunity; (ii) its tropism can be controlled by modifying the £ (spike) gene (Bal-lesteros et al., 1997); (iii) there are nonpathogenic strains for the development of systems of expression that depend on complementing virus (Sanchez et al., 1992); and (iv) coronaviru-ses are cytoplasmic RNA viruses that replicate without passing through an intermediate DNA stage (Lai and Cavanagh, 1997), making its integration into the cellular chromosome practically impossible.<br>
The procedure that has made it possible to recover an infective<br>
coronavirus from a cDNA that codes the gRNA of a coronavirus<br>
includes the following strategies:<br>
(i) expression of the RNA of the coronavirus under the con¬trol of an appropriate promoter;<br>
(ii) cloning of the genome of the coronavirus in bacterial artificial chromosomes (BACs);<br>
(iii) identification of the sequences of cDNA of the corona¬virus that are directly or indirectly toxic to bacte¬ria;<br>
(iv)     identification of the precise order of joining of the<br><br><br>
comoonents c?f the cDNA that codes an infective RNA of coronavirus (promoters, transcription-termination se¬quences, poiyadenylation sequences, ribozymes, etc.);<br>
and (v) identification of a group of technologies and processes (conditions for the growth of BACs, modifications to the purification process of BAC DNA, transformation techniques, etc.) that, in combination, allow the effi¬cient rescue of an infective coronavirus of a cDNA.<br>
The promoter plays an important role in increasing the expres¬sion of viral RNA in the nucleus, where it is synthesized, to be transported to the cytoplasm later on.<br>
The use of BACs constitutes one of the key points of the proce¬dure of the invention- As is known, cloning of eukaryotic se¬quences in bacterial plasmids is often impossible due to the toxicity of the exogenous sequences for bacteria. In these ca¬ses, the bacteria usually eliminate toxicity by modifying the introduced sequences. Nevertheless, in the strategy followed in this case, to avoid the possible toxicity of these viral sequen¬ces, the necessary clonings were carried out to obtain a com¬plete cDNA of the coronavirus in BACs. These plasmids appear in only one copy or a maximum of two per cell, considerably limi¬ting their toxicity and reducing the possibilities of interplas-mid recombination.  •<br>
Through the identification of the bacteriotoxic cDNA sequences Of the coronavirus, the construction of the cDNA that codes the complete genome of a coronavirus can be completed, with the exception of the toxic sequences, which are added in the last step of construction of the complete genome, that is, just befo¬re transfection in eukaryotic cells, avoidinq their modification by the bacteria.<br>
One object of the present invention therefore consists in an infective double-chain cDNA clone that codes the gRNA of a coro-<br><br>
navirus, as well as the procedure for obtaining it.<br>
An additional object of this invention consists in a set of recombinant viral vectors that comprises said infective clone and sequences of heterologous nucleic acids inserted into said infective clone.<br>
An additional object of this invention consists in a method for producing a recombinant coronavirus that comprises the introduc¬tion of said infective clone into a host cell, the culture of the transformed cell in conditions that allow the replication of the infective clone and production of the recombinant coronavi¬rus, and recovering the recombinant coronavirus from the cultu¬re.<br>
Another object of this invention consists in a method for produ¬cing a modified recombinant coronavirus that comprises introdu¬cing the recombinant viral vector into a host cell, cultivating it in conditions that allow the viral vector to replicate and the modified recombinant coronavirus to be produced, and reco¬vering the modified recombinant coronavirus from the culture. Another object of this invention consists in a method for produ¬cing a product of interest that comprises cultivating a host cell that contains said recombinant viral vector in conditions that allow the expression of the sequence of heterologous DNA.<br>
Cells containing the aforementioned infective clones or recombi¬nant viral vector constitute another object of the present in¬vention.<br>
Another object of this invention consists in a set of vaccines that protect animals against infections caused by infectious agents. These vaccines comprise infective vectors that express at least one antigen adequate for inducing an immune response against each infective agent, or at least one antibody that provides protection against said infective agent, along with a pharmaceutically acceptable excipient. The vaccines can be mono-<br><br>
or multivalent, depending on whether the vectors express one or more antigens capable of inducing an immune response to one or mere infectious agents, or, alternatively, one or more antibo¬dies that provide protection against one or more infectious agents.<br>
Another object provided by this invention comprises a method of animal immunization that consists in the administration of said vaccine.<br>
The invention provides a cDNA clone that codes the infective RNA of a coronavirus, henceforth the infective clone of the inven¬tion, which comprises: (1) a copy of the complementary DNA (cDNA) to the infective genomic RNA (gRNA) of a coronavirus or the viral RNA itself; and (2) an expression module for an ad¬ditional gene, which includes optimized transcription-promoting sequences.<br>
In one particular realization of this invention, the coronavirus is a TGEV isolate, in particular, the PUR46-MAD isolate (Sanchez et al., 1990), modified by the replacement of the S gene of this virus by the S gene of the clone Cll TGEV isolate or the S gene of a canine or human coronavirus.<br>
The transcription-promoting sequence, or promoter, is an RNA sequence located at. the 5'-terminal end of each messenger RNA (mRNA) of coronavirus, to which the viral polymerase RNA binds to begin the transcription of the messenger RNA (mRNA) . In a particular and preferred embodiment the viral genome is expres¬sed from a cDNA using the IE promoter of CMV, due to the high level of expression obtained using this promoter (Dubensky et al., 1996), and to previous results obtained in our laboratory that indicated that large defective genomes (9.7 kb and 15 kb) derived from the TGEV coronavirus expressed RNAs that did not undergo splicing during their transport from the nucleus,- where they are synthesized, to the cytoplasm.<br><br><br>
The infective clone of the invention also contains a transcrip¬tion termination sequence and a polyadenylation signal such as that coming from the BGH gene. These termination sequences have to be placed at the 3'-end of the poly(A) tail. In one particu¬lar realization, the infective clone of the invention contains a poly(A) tail of 24 residues of A and the termination and po¬lyadenylation sequences of the BGH separated from the poly(A) tail by the sequence of the HDV ribozyme.<br>
The plasmid into which the infective cDNA of the virus has been inserted is a DNA molecule that possesses a replication origin, and is therefore potentially capable of replicating in a suit¬able cell. The plasmid used is a replicon adequate for maintai¬ning and amplifying the infective clone of the invention in an adequate host cell such as a bacterium, for example, Escherichia coll. The replicon generally carries a gene of resistance to antibiotics that allows the selection of the cells that carry it (for example, cat).<br>
In Example 1, the construction of an infective clone of TGEV under the control of the IE promoter of CMV is described. The 3'-end of the cDNA appears flanked by a 24 nt poly(A) sequence, the HDV ribozyme, and the transcription termination sequence of the BGH.<br>
The procedure for obtaining the infective clone of the invention comprises constructing the full-length cDNA from the gRNA of a coronavirus and joining the transcription-regulating elements.<br>
The cDNA that codes the infective gRNA of a coronavirus was obtained from a DI genome derived from a coronavirus cloned as a CDNA under the control of an appropriate promoter in a BAC, for the purpose of increasing the cDNA' s stability. Then the bacteriotoxic sequences were identified and, for the purpose of eliminating that toxicity, said toxic sequences were removed and inserted at the end of the construction of the complete genome, just before transfection in eukaryotic cells. The viral progeny<br><br>
can be reconstituted by means of transfection of the BAG plasmid that contains the coronavirus genome in eukaryotic cells that support viral replication.<br>
The transcription-regulating elements are joined by means of conventional techniques (Maniatis et al., 1989).<br>
The infective clone of the invention can be manipulated by con¬ventional genetic engineering techniques to insert at least one sequence of a heterologous nucleic acid that codes a determined activity, under the control of the promoter that is present in the infective clone and of the regulating sequences contained in the resulting expression vector.<br>
The infective clone of the invention presents numerous applica¬tions; for example, it can be used both in basic research, for example, to study the mechanism of replication and transcription of coronaviruses, and in applied research, for example, in the development of efficient systems of expression of products of interest (proteins, enzymes, antibodies, etc.).<br>
Appropriate cells can be transformed from the infective cDNA clone-of the invention, and the virions obtained containing the complete genome of the coronavirus can be recovered. Therefore, the invention moreover provides a method for producing a recom¬binant coronavirus that comprises the introduction of an infec¬tive cDNA of the invention into a host cell, the culture of said cell under conditions that allow the expression and replication of the infective clone and the recovery of the virions obtained from the recombinant coronavirus, which contain the infective genome of the coronavirus. The infective clone of the invention can be introduced into the host cell in various ways, for ex¬ample by transfection of the host cell with an RNA transcribed in vitro from an infective clone of the invention, or by infec¬ting the host cell with the infective cDNA clone of the, inven¬tion. Said host cells that contain the infective clone of the invention constitute an additional object of the present inven-<br><br><br>
tion.<br>
The invention also provides a set of recombinant viral vectors derived from an infective clone of the invention, henceforth viral vectors of the invention. The viral vectors of the inven¬tion comprise an infective cDNA clone of the invention modified to contain a heterologous nucleic acid inserted into said infec¬tive clone under conditions that allow said heterologous nucleic acid to be expressed.<br>
The term "nucleic acid," as it is used in this description, includes genes or gene fragments as well as, in general, any molecule of DNA or RNA.<br>
In the sense used in this description, the term "heterologous" applied to a nucleic acid refers to a nucleic acid sequence that is not normally present in the vector used to introduce the heterologous nucleic acid into a host cell.<br>
The heterologous nucleic acid that can contain the viral vector of the invention can be a gene or fragment that codes a protein, a peptide, an epitope, or any gene product of interest (such as antibodies, enzymes, etc.). The heterologous nucleic acid can be inserted into the infective clone of the invention by means of conventional genetic engineering techniques in any appropriate region of the cDNA, fbr example, after ORF lb or between genes N and 7, following the initiator codon (AUG), and in reading frame with that gene; or, alternatively, in the zones correspon¬ding to other ORFs. In the construction of the viral vector of the invention, it is essential that the insertion of the hetero¬logous nucleic acid not interfere with any of the basic viral functions.<br>
The viral vector of the invention can express one or more acti¬vities. In this latter case, the viral vector will include as many sequences of heterologous nucleic acid as activities to be expressed, preceded by one or several promoters, or by a promo-<br><br>
ter and various ribosome recognition sites (IRES, internal ribo-some entry sites), or by various promoters and one ribosome recognition site.<br>
Therefore, the invention provides a method for producing a pro¬duct of interest that comprises cultivating a host cell that contains a viral vector of the invention under conditions that allow the heterologous nucleic acid to be expressed and the product of interest to be recovered. Said host cells that con¬tain the viral vector of the invention constitute an additional object of the present invention.<br>
The viral vector of the invention can be designed so that its species specificity and tropism can be easily controlled. Due to these characteristics, a very interesting application of the viral vectors of the invention is their use in gene therapy as a vector of the gene of interest, or as a vaccinal vector to induce immune responses against different pathogens.<br>
The invention furthermore provides vaccines, capable of protec¬ting an animal against the infection caused by an infectious agent, that comprise (i) at least one viral vector of the inven¬tion that expresses at least one antigen suitable for inducing an immune response against said infectious agent, or an antibody that provides protection against said infectious agent, along with, optionally, (ii() a pharmaceutically acceptable excipient.<br>
In the sense used in this description, "inducing protection" should be understood as the immune response of the receiving organism (animal to be immunized) induced by the viral vector of the invention, through suitable mechanisms such as that induced by substances that potentiate cellular response (interleukins, interferons, etc.), cellular necrosis factors, and similar sub¬stances that protect the animal from infections caused by infec¬tious agents.<br>
Included under the term "animal" are all animals of any species,<br><br>
preferably mammals, including man.<br>
The term * infectious agent" in the sense used in this	descrip¬<br>
tion includes any viral, bacterial, fungal, parasitic,	or other<br>
infective agent that can infect an animal and cause it	a patho¬<br>
logy.<br>
In one particular realization, the vaccine provided by this invention comprises at least one viral vector of the invention that expresses at least one antigen capable of inducing a sy¬stemic immune response and/or an immune response in mucous mem¬branes against different infectious agents that propagate in respiratory or intestinal mucous membranes. The vectors of the invention are quite suitable to induce immunity in mucous mem¬branes as well as lactogenic immunity, which is of special inte¬rest in protecting newborns against intestinal tract infections.<br>
In another particular realization, the vaccine provided by this invention comprises at least one viral vector of the invention that expresses at least one gene that codes for the light and heavy chains of an. antibody of any isotype (for example, IgGlr IgA, etc.) that protects against an infectious agent.<br>
Species specificity can be controlled so that the viral vector may express the S protein of the envelope of a coronavirus that infects the desired species (man, dog, cat, pig, etc.), suitable to be recognized by the cellular receptors of the corresponding species.<br>
The vaccines provided by this invention can be monovalent or multivalent, depending on whether the viral vectors of the in¬vention express one or more antigens capable of inducing an immune response to one or more infectious agents, or one or more antibodies that provide protection against one or more infectio¬us agents,<br>
In a particular realization of this invention, monovalent vacci-<br><br>
nes capable of protecting man, pigs, dogs and cats against dif¬ferent infectious human, porcine, canine, and feline agents are provided, and tropism is controlled by expressing the S glyco¬protein of the coronavirus with the desired species specificity.<br>
The monovalent vaccines against porcine infectious agents can contain a vector that expresses an antigen selected from the group consisting essentially of antigens of the following porci¬ne pathogens: Actinobacillus pleuropneumonias, Actinobacillus suis, Haemophilus parasuis, porcine parvovirus, Leptospira, Escherichia coli, Erysipelotrix rhusiopathiaer Pasteurella mul-tocida, Bordetella bronchiseptica, Clostridium sp., Serpulina hydiosenteriae, Mycoplasma hyopneumoniae, porcine epidemic diar¬rhea virus (PEDV), porcine respiratory coronavirus, rotavirus, or against the pathogens that cause porcine respiratory and reproductive syndrome, Aujeszky's disease (pseudorabies) , swine influenza, or transmissible gastroenteritis', and the etiological agent of atrophic rhinitis and proliferative ileitis. The mono¬valent vaccines against canine infectious agents can contain an expression vector that expresses an antigen selected from the group essentially consisting of antigens of the following canine pathogens: canine herpes viruses, types 1 and 2 canine adenovi¬rus, types 1 and 2 canine parvovirus, canine reovirus, canine rotavirus, canine coronavirus, canine parainfluenza virus, cani¬ne influenza virus, distemper virus, rabies virus, retrovirus, and canine caliciviru(s.<br>
The monovalent vaccines against feline infectious agents can contain an expression vector that expresses an antigen selected from the group essentially consisting of antigens of the follo¬wing feline pathogens: cat calicivirus, feline immunodeficiency virus, feline herpes viruses, feline panleukopenia virus, feline reovirus, feline rotavirus, feline coronavirus, cat infectious peritonitis virus, rabies virus, feline Chlamydia psittaci, and feline leukemia virus<br>
The vectors can express an antibody that provides protection.<br><br>
against an infectious agent, for example, a porcine, canine or feline infectious agent such as those cited above. In one parti¬cular realization, the vector expresses the recombinant monoclo¬nal antibody identified as 6A.C3, which neutralizes TGEV, ex¬pressed with isotypes IgGt or IgA, in which the constant part of the immunoglobulin is of porcine origin, or neutralizing antibo¬dies for human and porcine rotaviruses.<br>
As the excipient, a diluent such as physiological saline or other similar saline solutions can be used. Likewise, these vaccines can also contain an adjuvant from those usually used in the formulation of both aqueous vaccines, such as aluminum hy¬droxide, QuilA, suspensions of alumina gels and the like, and oily vaccines based on mineral oils, glycerides, fatty acid derivatives, and their mixtures.<br>
The vaccines of the present invention can also contain cell-response-potentiating (CRP) substances, that is, substances that potentiate subpopulations of helper T-cells (Thx and Th2) such as interleukin-1 (IL-1), IL-2, IL-4, IL-5, IL-6, IL-12, gamma-IFN (gamma-interferon), cellular necrosis factor, and similar sub¬stances that could theoretically provoke cellular immunity in vaccinated animals. These CRP substances could be used in vacci¬ne formulations with aqueous or oily adjuvants. Another type of adjuvants that modulate and immunostimulate cellular response can also be used, such as MDP (muramyl dipeptide), ISCOM (Iiranu-nostimulant Complex), or liposomes.<br>
The invention provides multivalent vaccines capable of preven¬ting and protecting animals from infections caused by different infectious agents. These multivalent vaccines can be prepared from viral vectors of the invention into which the different sequences that code the corresponding antigens have been inser¬ted in the same recombinant vector, or by constructing indepen¬dent recombinant vectors that would later be mixed for joint inoculation. Therefore, these multivalent vaccines comprise a viral vector that contains more than one sequence of heterclo-<br><br>
gous nucleic acids that code for more than one antigen or, al¬ternatively, different viral vectors, each of which expresses at least one different antigen.<br>
Analogously, multivalent vaccines that	comprise multivalent<br>
vectors can  be prepared using sequences	that code antibodies<br>
that protect against infectious agents,	instead of sequences<br>
that code the antigens.<br>
In one particular realization of this invention, vaccines capa¬ble of immunizing humans, pigs, dogs, and cats against different porcine, canine and feline infectious agents, respectively, are provided. For this, the viral vectors contained in the vaccine must express different antigens of the human, porcine, canine or feline pathogens mentioned above or others.<br>
The vaccines of this invention can be presented in liquid or lyophilized form and can be prepared by suspending the recombi¬nant systems in the excipient. If said systems were in lyophili¬zed form, the.excipient itself could be the reconstituting sub¬stance.<br>
Alternatively, the vaccines provided by this invention can be used in combination with other conventional vaccines, either forming part of them or as a diluent or lyophilized fraction to be diluted with othericonventional or recombinant vaccines.<br>
The vaccines provided by this invention can be administered to the animal orally, nasally, subcutaneously, intradermally, in-traperitoneally, intramuscularly, or by aerosol.<br>
The invention also provides a method for the immunization of animals, in particular pigs, dogs and cats, against one or va¬rious infectious agents simultaneously, that comprises the oral, nasal, subcutaneous, intradermal,intraperitoneal, intramuscu¬lar, or aerosol administration (or combinations thereof) cf a vaccine that contains an immunologically efficacious quantity cf<br><br>
a recombinant system provided by this invention.<br>
In addition, the invention also provides a method for protecting newborn animals against infectious agents that infect said ani¬mals, consisting in the oral, nasal, subcutaneous, intradermal, intraperitoneal, intramuscular, or aerosol administration (or combinations thereof) of a vaccine of those provided by this invention to mothers before or during the gestation period, or to their offspring.<br>
The invention is illustrated by the following examples, which describe in detail the obtainment of infective clones and the construction of the viral vectors of the invention. These ex¬amples should not be considered as limiting the scope of the invention, but as illustrating it. In said example, the trans¬formation and growth of bacteria, DNA purification, sequence analysis, and the- assay to evaluate the stability of the plas¬mids were carried out according to the methodology described below.<br>
Transformation of bacteria<br>
All of the plasmids were electroporated in the E. call DHIOB strain (Gibco BRL), introducing slight modifications to previo¬usly described protocols (Shizuya et al. , 1992). For each trans¬formation, 2 uL of the ligation and 50 uL of competent bacteria were mixed in 0.2-cm dishes (BioRad) and electroporated at 200 Q, 2.5 kV, and 25 uF. Then, 1 mL of SOC medium (Maniatis et al. , 1989) was added at each transformation, the cells were incubated a'37°C for 45 min, and finally, the recombinant colonies were detected on plates of LB SOC media (Maniatis et al. , 1989) with 12.5 ug/mL of chloramphenicol.<br>
Growth conditions of the bacteria<br>
The bacteria containing the original plasmids, in which the<br>
incomplete genome of TGEV was cloned (Figure 3), were grown at<br>
37°C, showing normal'growth kinetics. On the other hand, the BAC<br>
that contained the complete cDNA was grown at 30°C for the pur-<br><br><br>
pose cf minimizing instability as much as possible. Even so, the size of the colonies was reduced and incubation periods of up to 24 h were necessary to achieve normal.colony sizes.<br>
Purification of DNA<br>
The protocol described by Woo (Woo et ai., 1994) was followed, with slight modifications. From a single colony, 4 L of LB were inoculated with chloramphenicol<br>
(12.5 ug/ml). After an incubation period of 18 h at 30°C, the bacteria were.collected by centrifugation at 6,000 G, and the plasmid was purified using the Qiagen Plasmid Maxipreparations kit according to the manufacturer' s recommendations. By means of this procedure, it was observed that the plasmid DNA obtained was contaminated with bacterial DNA. To eliminate the contamina¬ting bacterial DNA, the plasmidic DNA was purified by means of centrifugation at 55,000 rpm for 16 h on a CsCl gradient. The yield obtained was between 15 and 30 ug/L, depending on the size of the plasmid.<br>
Sequence Analysis<br>
The DNA was sequenced in an automatic sequencer (373 DNA Sequen¬cer, Applied Biosystems) using dideoxynucleotides marked with fluorochromes and a temperature-resistant polymerase (Perkin Elmer). The reagents were obtained by way of a kit (ABI PRISM Dye Terminator Cycle Sequencing Ready Reaction Kit) from the Applied Biosystems company. The thermocycler used to perform the sequencing reactions was a " GeneAmpPCR System 9 6 00" (Perkin Elmer).<br>
The joining of the sequences and their comparison with the con¬sensus sequence of the TGEV were carried out using the SeqMan II and Align (DNASTAR) programs, respectively. No differences in relation to the consent sequence were detected.<br>
Stability of the plasmids<br>
From the original glycerolates, the bacteria that contained recombinant pBeloEACll plasmids were grown in 20 mL of LB with chloramphenicol (12.5 ug/mL) for 16 h at 30°C and 37°C. This<br><br>
material was considered passage 0. The bacteria were diluted 106 times and grown at 30°C and 37°C for 16 h. Serial passages were realized during eight consecutive days (each passage represents approximately 20 generations). The plasmid DNA was purified by Miniprep at passages 0 and 8 (160 generations) and analyzed with restriction endonucleases. The two plasmids that contained part of the genome of TGEV were highly stable, whereas the plasmid that contained the complete genome of TGEV showed a certain instability after 40 generations (at this point approximately 80% of the DNA presented the correct restriction pattern).<br>
Example 1<br>
CONSTRUCTION OF A RECOMBINANT VECTOR<br>
BASED ON A CLONE OF INFECTIVE cDNA DERIVED FROM TGEV<br>
1.1 Generation of an infective cDNA of TGEV<br>
For the purpose of obtaining a cDNA that coded for the complete TGEV genome, we originally started with a cDNA that coded for the defective DI-C genome (Mendez et al., 1996). This cDNA, with an approximate length of one third of the TGEV genome, was clo¬ned in the low-copy pACNR1180 plasmid (Ruggli et al., 1996) and its sequence was determined. The cDNA that coded the defective genome was efficiently rescued (replicated and packaged) with the help of a complementing virus (Mendez et al., 1996; Izeta et al., 1999).<br>
The DI-C genome presents three deletions (Al, A2, and A3) of approximately 10, 1 and 8 kilobases (kb), at ORFs la, lb, and between genes S and 7, respectively (see Figure 1). The strategy followed to complete the sequence of a cDNA that would code for an infective TGEV genome was to incorporate, step by step, the sequences deleted in the DI-C genome, analyzing the bacteriotoxicity of the new generated constructions. This aspect is very important, since it is widely documented in the scienti¬fic literature that recombinant plasmids presenting cDNAs of RNA virus generally grew poorly and were unstable (Boyer and Haenni, 13S4; Rice et al. , 198 9; Mandl et al. , 19 97). The first deletion to be completed was deletion A2, of 1 kb, of<br><br>
ORF lb, yielding a stable recombinant plasmid. The sequence that lacked ORF la was introduced by cloning cDNA fragments A, E, C, and D (Figure 1) (Almazan et al. , 2000) in such a way that all the information required for the gene of the replicasa would be complete. The recombinant plasmid obtained was unstable in the bacteria, generating new plasmids that had incorporated addi¬tions and deletions in fragment B (Almazan et al. f 2000). Inter¬estingly, the elimination of a 5,198 bp Clal-Clal restriction fragment that encompassed the region of the genome comprised between nucleotides 4,417 and 9,615 (Penzes et al., 1999) yiel¬ded a relatively stable plasmid in the E.    coli   DH10B strain. Later, the sequence of deletion A3 was added by cloning all the genetic information for the structural and nonstructural pro¬teins of the 3'-end of the TGEV genome (Figure 1). For the purpose of incrementing the stability of the TGEV cDNA, it was decided that it would be subcloned in BAC using the pBel-oBACll plasmid (Kim et al., 1992) (see Figure 2). The pBeloBACll plasmid was a generous gift from H. Shizuya and M.. Simon (Cali¬fornia Institute of Technology). The plasmid, 7,507 bp in size, includes the replication origin of the F factor from parB, parC, E.   coli   (oriS) and the genes necessary to keep a single copy of the plasmid per cell (parA, and repE) . The plasmid also presents the gene of resistance to chloramphenicol (cat) as a selection marker. The cDNA was cloned under the control of the IE promoter of CMV, due to the high level of expression obtained using this promoter (Dubensky et al., 1996) and to previous results obtai¬ned in our laboratory, indicating that large (9.7 kb and 15 kb) defective genomes derived from TGEV expressed RRAs that did not undergo splicing during transport from the nucleus, where they are synthesized, to the cytoplasm (Izeta et al., 1999; Penzes et al.,  1999; Almazan et al. ,  2000). The generated TGEV cDNA (pBAC-TcDNA-AClal) contained the information for the genes of the replicase, with the exception of the deleted 5,198 bp Clal fragment, and all the information of the structural and non¬structural genes. The 3'-end of the cDNA appears flanked by a 24 nt polyA sequence, the KDV ribozyme, and the transcription ter¬mination seauence of BGH (Izeta et al. , 1999). On the other<br><br>
hand, the Clal fragment necessary to generate a complete genome of TGEV was cloned in BAC, generating the plasmid pEAC-B+C+DS', which contained the region of the TGEV genome between 4,310 and 9,758 (see Figure 3). Both plasmids were grown in the E. coll DH10B strain and sequenced in their entirety. The sequence ob¬tained was identical to the consent sequence of the PUR46-MAD isolate of TGEV provided at the end of this document (SEQ ID NO:l), with the exception of two replacements in the positions of nucleotides 6,752 (A =&gt; G, silent) and 18,997 (T =&gt; C, si¬lent) , and the changes in the S gene of the FUR46-MAD that has been replaced by the D gene of isolate Cll (these changes are indicated in Figure 4).<br>
Furthermore, for the purpose of generating a cDNA that would code a virulent TGEV, the S gene of the PUR46-MAD isolate, which replicates at highs levels in the respiratory tract (&gt;10 PFU/g of tissue) and at low levels in the intestinal tract (
A cDNA was constructed from the PUR46-MAD isolate of TGEV with the S gene of the intestinal isolate Cll, by means of cloning of the 5,19 8 bp Clal-Clal fragment, obtained from the pBAC-B+C+D5' plasmid, in the pBAC-TcDNA"4 ~aJ" plasmid, to generate the pBAC--TcDNA"'"' plasmid that contains the cDNA that codes for the com¬plete TGEV genome (Figure 3). . The stability in bacteria of the plasmids used in the construc-<br><br>
tion of the clone of infective cDNA (pBAC-TcDNA"40'*1 and pBAC — Clal"'), as well as the plasmid that contains the complete cDNA (pBAC-TcDNArL) , was analyzed after being grown in E. coll for 160 generations. The stability was analyzed by means of digestion with restriction enzymes of the purified DNAs. No deletions or insertions were detected, although the presence of minor changes not detected by the analysis technique used cannot be ruled out in the case of the pBAC-TcDNA"iC:La"1 plasmid and the pBAC-B+C+D5' plasmid. In the case of the pBAC-TcDNA plasmid, which contains the complete genome of TGEV, a certain instability was detected after 40 generations (at this point approximately 80% of the DNA presented the correct restriction pattern). This slight instabi¬lity, however, does not represent an obstacle to the rescue of the infective virus, since 20 generations (4 L of culture] of bacterial growth are sufficient to generate a quantity of plas¬mid DNA that allows the virus to be rescued.<br>
1.2 Rescue of an infective TGEV from a cDNA that codes for the complete genome<br>
ST cells were transfeeted with the pBAC-TcDNA5L plasmid. At 4 8 h posttransfection, the supernatant of the culture was collected      and passed into ST cells six times (see Figure 5). Starting at passage 2, at 14 h postinfection, the cytopathic effect became apparent, extending later, at 20 h postinfection, to practically all of these cells that formed the monolayer (see Figure 6). On the other hand, the titer of rescued virus increa-sed rapidly with the passages, reaching values on the order of 108 PFU/mL as of passage 3 (see Figure 7). The experiment was repeated five times, and in all cases, infective virus with 'similar titers were recovered, whereas, in the case of nontrans-fected ST cells or ST cells transfected with a similar plasmid, where the Clal-Clal fragment was found in the opposite orienta¬tion, virus was never recovered.<br>
For the purpose of eliminating the possibility that the virus obtained was the product of contamination, the sequence at posi¬tions 5,752 and 18,997 was determined by means of sequencing of cDNA fragments amplified by RT-PCR using the genomic RNA of the<br><br><br>
rescued virus as a template. The analysis of the sequence deter¬mined that the nucleotides in positions 6,752 and 18,997 were those present in the cDNA. Furthermore, the rescued virus pre¬sented, in the cDNA sequence of the S gene, a restriction site Drain at position 20,990, as was expected for the S gene of ell (Figure 8). The presence of these three genetic markers confir¬med that the isolated virus came from the cDNA. In a more in-depth characterization of the virus generated, a comparative analysis was made by immunofluorescence of infected cells with the virus recovered (TcDNA.) after transfection with the pBAC-TcDNA71 plasmid or cells infected with the PUR46-MAD isolate of the TGEV. For this, specific polyclonal and monoclo¬nal antibodies that recognized both the Cll isolate and the PUR46-MAD isolate, or only the latter, were used (see Figure 10). The results obtained confirmed the antigenicity expected for the new TcDNA virus. The polyclonal antibody specific for TGEV, the expected specific monoclonal of the S protein (ID.B12 and 6A.C3), as well as the specific monoclonal of the M (3B.B3) and N  (3B.D8)  proteins recognized both the TcDNA and the PUR46-MAD. The data obtained indicated that the virus generated presented the M and N proteins of the PUR46-MAD isolate and the S protein of the Cll isolate, as had been designed in the origi¬nal cDNA.<br>
1.3 In vivo infectivity and virulence<br>
?or the purpose of Analyzing the in vivo infectivity of the TcDNA virus, a group of five newborn pigs was inoculated withirus cloned from passage 6, and mortality was analyzed. The E\ive inoculated pigs died 3 to 4 days postinoculation, indica-:ing that the TcDNA virus was virulent. In contrast, two pigs Inoculated only with the diluent of the virus and maintained in :he same conditions did not suffer alterations.<br>
L.4 Optimization of the levels of expression by modification of :he transcription-regulating sequences<br>
rlNA synthesis in coronavirus takes place by means of an RNA--dependent process, in which the mRNAs are transcribed from tem-<br><br>
plates with negative polarity. In the TGEV,- a conserved consen¬sus sequence, CUAAAC, appears, which is located just in front of the majority of the genes. These sequences represent signals for the transcription of the subgenomic mRNAs. In coronavirus, there are between six and eight types of mRNAs with variable sizes, depending on the type of coronavirus and of the host. The lar¬gest corresponds to the genomic RNA, which in turn serves as mRNA for ORFs la and lb. The rest of the mRNAs correspond to subgenomic mRNAs. These RNAs are denominated mRNA 1 to 7, in decreasing size order. On the other hand, some mRNAs that have been discovered after the set of originally described mRNAs have been denominated with the name . of the corresponding mRNA, a dash, and a number, e.g., mRNA 2-1. The mRNAs present a cotermi-nal structure in relation to the structure of the genomic RNA. With the exception of the smallest mRNA, the rest are structu¬rally polycistronic, while, in general, only the ORF located closest to 5' is translated.<br>
The efficiency in the expression of a marker gene (GUS) has been studied using different sequences flanking the 5'-terminal of the minimal intergenic (IG) sequence CUAAAC (Figure 11), diffe¬rent sequences flanking the 3' -terminal of the IG sequence (Fi¬gure 12), and various insertion sites (Figure 13). The results obtained (Figures 11 to 13) indicated that optimal expression was achieved with a TRS consisting of: (i) the -88 nt flanking the consent sequence for the N gene of TGEV; (ii) the IG sequen¬ce; and (iii) the 3'-flanking sequence of the IG sequence of the S gene. Furthermore, in agreement with the results obtained in relationship to the point of insertion of the heterologous gene, the greatest levels of expression were achieved when the hetero¬logous gene was located at the 3'-end of the genome. A TRS such as that described allows the GUS to be expressed at levels bet¬ween 2 and 8 ug per 10 cells.<br>
1.5 Tissue specificity of the system of expression Many pathoqens enter the host through the mucous membranes. To prevent this type of infections, it is important to develop systems of expression that allow the induction of high levels cf<br><br><br>
secretory immunity. This can be achieved fundamentally through the administration of antigens in the lymph nodes associated with the respiratory and intestinal tract. To achieve this goal, and in general to direct the expression of a gene at the tissue of interest, the molecular bases of the tropism of TGEV have been studied. These studies have showed that the tissue specifi¬city of TGEV can be modified by the construction of recombinant viruses containing the S gene of coronavirus with the desired tropism (Ballesteros et al., 1997; Sanchez et al., 1999). This information makes it possible to construct systems of expression based on cDNA genomes of coronavirus with respiratory or inte¬stinal tropism.<br>
1.6	Expression of the viral antigen coded by the 0RF5 of PRRSV<br>
using infective cDNA<br>
For the purpose of optimizing the levels of expression of hete¬rologous genes, constructions were made from a vector of inter¬changeable modules flanked by cloning sequences that facilitate the exchange of TRSs and heterologous genes within the vector. The construction, which included ORF 5 of the PRRSV (Porcine   respiratory and reproductive syndrome virus) flanked at the 5'-end by the minimal IGS consensus sequence (CUAAAC) preceded by the -88 nts flanking the gene of the viral nucleocapsid (N), and at the- 3'-end by restriction site Sail (GTCGAC) and a sequence analogous to that of Kozak (AC)GACC, yielded an optimal expres¬sion (about 10 \ig/10° cells). In principle, these levels of expression of the heterologous gene are more than sufficient to induce an immune response. The heterologous gene was inserted into the position previously occupied by genes 3a and 3b of the virus, which are dispensable.<br>
1.7	Induction of an immune response in swine to an antigen ex¬<br>
pressed with the cDNA derived virus vector<br>
Using the same type of virus vector derived from the cDNA and the TRSs described above, the gene encoding the green fluores¬cent protein (GFP) was expressed at high levels (20 |ig of pro¬tein per million of cells in swine testis, ST, cells). The ax-<br><br>
pression levels were stable for more than 20 passages in cell culture. Furthermore, a set of swine were immunized with the live virus vector, that was administered by the oral, intra¬nasal and intragastric routes and a strong humoral immune re¬sponse was detected against both the virus vector and the GFP. Interestingly, no secondary effect was observed in the inocula¬ted animals after the administration of three doses of the virus vector.<br>
1.8 Construction of a safe virus vector that expresses the fo¬reign gene without leading to the generation of an infectious virus.<br>
To design vector for humans, biosafety is a priority. To achieve this goal, three types of safety guards are being engineered in the vector. Two of these are based on the deletion of two virus components, mapping at different positions of the virus genome, essential for the replication of the virus. These components are being provided in trans by a packaging cell line. This cell (Baby Hamster Kidney, BHK) expresses the missing TGEV genes encoding essential structural proteins of the virus (the envelo¬pe E and the membrane M proteins). The third safety guard is the relocation of the packaging signal of the virus genome, in such a way that the recovery of an infectious virus by recombination is prevented, leading to the generation of a suicide vector that efficiently expresses the heterologous genes but that is unable to propagate even to the closest neighbor cell.<br>
With the design of the new vector for use in humans, we are not producing a new virus that could be propagated within the human species, since this vector can not be transmitted from cell to cell in human beings, The vector is based on a replication de¬fective virus. It can only be grown in the vaccine factory by using packaging cells complementing the deletions of the virus. These safety guards represent novel procedures in the enginee¬ring of coronaviruses. The recombinant virus with a new tropism will be replication competent at least in feline cells, since these cells replicate human, porcine, canine and feline corona-viruses .<br><br>
DEPOSITION OF MICROORGANISMS:<br>
The bacterium derived from Escherichia coll, carrying the plas-mid with the infective clone of the invention, identified as Escherichia coli pBAC-TcDNA , has been deposited with the Spa¬nish Collection of Type Cultures (CECT), Burjassot (Valencia), on November 24"h 199 9, under registration number CECT 5 2 65.<br><br><br>
 BIBLIOGRAPHY Ahlguist, P., R.    French,  M.  Janda,  and L.S. Loesch-Fries.<br>
(1984). Multicomponent RNA plant'virus infection derived from cloned viral cDNA. Proc.   Natl.   Acad.   Sci.   USA.   81:7066-7070.<br>
Almazan, F., J.M. Gonzalez, Z. Penzes, A. Izeta, E. Calvo, J. Plana-Duran, and L. Enjuanes. (2000). Engineering the largest RNA virus genome as an infectious bacterial artificial chromoso¬me. Proc.   Natl.   Acad.   Sci.   USA.   97:5516-5521.<br>
Ballesteros, M.L., CM. Sanchez, and L. Enjuanes. (1997). Two amino acid changes at the N-terminus of transmissible gastroen¬teritis coronavirus spike protein result in the loss of enteric tropism. Virology.   227:37.8-388.<br>
Baron, M.D., and T. Barrett. (1997). Rescue of rinderpest virus from cloned cDNA. J.   Virol.   71:1265-1271.<br>
Boyer, J. C, A. L. and Haenni. (1994). Infectious transcripts and cDNA clones of RNA viruses. Virology.   198:415-426.<br>
Chang, R. Y., M. A. Hofmaun, P. B. Sethna, and D. A. Brian.<br>
(1994). A cis-acting function for the coronavirus leader in defective interfering RNA replication. J.   Virol.   68:8223-8231.<br>
Collins, P.L., M.G. Hill, E. Camargo, H. Grosfeld, R.M. Chanock, and B.R. Murphy. (1995). Production of infectious human respira¬tory syncytial virus from cloned dCNA confirms an essential role for the transcription elongation factor from the 5' proximal open reading frame of the M2 mRNA in gene expression and provi¬des a capability for vaccine development. Proc. Natl. Acad. Sci. USA.   92:11563-11567.<br>
Davis, N.L-, L.V. Willis, J.F. Smith, andR.E. Johnston. (1989). In vitro synthesis of infectious Venezuelan equine encephalitis virus RNA from a "cDNA clone: analysis of a viable deletion mu¬tant. Virology.   171:189-204.<br><br>
Dubensky, J., T. W. , D.A. Driver, J.M. Polo, B.A. Belli, E.M. Latham, C.E- Ibanez, S. Chada, D. Brumm, T.A. Banks, S.J. Mento, D.J. Jolly, and S.M.W. Chang. (1996). Sindbis virus DNA-based expression vectors: utility for in vitro and in vivo gene trans¬fer. J.   Virol.   70:508-519.<br>
Durbin, A.P., S.L. Hall, J.W. Slew, S.S. Whitehead, P.L. Col¬lins, and B.R. Murphy. (1997). Recovery of infectious human parainfluenza virus type 3 from cDNA. Virology.   235:32 3-332.<br>
Enjuanes, L. , S.G. Siddell, and W.J. Spaan. 1998. Coronaviruses and Arteriviruses.   Plenum Press, New York.<br>
Enjuanes, L., and B.A.M. Van der Zeijst. 1995. Molecular basis of transmissible gastroenteritis coronavirus epidemiology. In The Coronaviridae.   S.G. Siddell, editor. Plenum Press, New York. 337-376.<br>
Frolov, I., T.A. Hoffman, B.M. Pragai, S.A. Dryga, H.V. Huang, S. Schlesinger, and CM. Rice. (1996). Alphavirus-based expres¬sion vectors: Strategies and applications. Proc. Natl. Acad. Sci.   USA.   93:11371-11377.<br>
Garcin, D., T. Pelet, P. Calain, L. Roux, J. Curran, and D. Kolakofsky. (1995). A highly recombinogenic system for the reco¬very of infectious sendai paramyxovirus from cDNA: generation of a novel. EMBO J.   14:6087-6094.<br>
Geigenmuller, U., N.H. Ginzton, and S.M. Matsui. (1997). Con¬struction of a genome-length cDNA clone for human astrovirus serotype 1 and synthesis of infectious RNA transcripts. J. Vi¬rol.   71:1713-1717.<br>
Izeta, A., C. Smerdou, S. Alonso, 2. Penzes, A. Mendez, J. Pla-na-Duran, and L. Enjuanes. (1999). Replication and packaging of transmissible gastroenteritis coronavirus-derived svnthetic miniaenomes. J.   Virol.   73:1535-1545.<br><br><br>
Kim, U.-J-, H. Shizuya, P. de Jong, B.W. Birren, and M.I. Simon.<br>
(1992). Stable propagation of cosmid-sized human DNA inserts in an F-factor based vector. Nucleic Acids Res.   20:1083-1085.<br>
Lai, C.-J., B. Zhao, H. Hori, and M. Bray. (1991). Infectious RNA transcribed from stably cloned full-length cDNA of dengue type 4 virus. Proc,   Natl.  Acad.   Scl.   USA.   88:513 9-5143.<br>
Lai, M.M.C, and D. Cavanagh. (1997). The molecular biology of coronaviruses. Adv.   Virus Res.   48:1-100.<br>
Lai, M.M.C, C.-L. Liao, Y.-J. Lin, and X. Zhang. (1994). Coro-navirus: how a large RNA viral genome is replicated and tran-scribed. Infect.   Agents Dis.   3:98-105.<br>
Liljestrom, P. (1994)- Alphavirus expression systems. Curr. Opin.   Biotech.   5:495-500.<br>
Liljestrom, P., and H- Garoff. (1991). A new generation of ani¬mal cell expression vectors based on the Semliki Forest virus replicon. Bio/Technology.   9:1356-1361.<br>
Luytjes, W., M. Krystal, M. Enami, J.D. Parvin, and P. Palese.<br>
(1989). Amplification, expression, and packaging of a foreign gene by influenza virus- Ceii. 59:1107-1113.<br>
Mandl, C. W., M. EcKer, H. Holzmann, C- Kunz, F. X. Heinz.<br>
(1997).Infectious cDNA clones of tick-borne encephalitis virus European subtype prototypic strain Neudoerfl and high virulence strain Hypr. J.   Gen.   Virol.   78:1049-1057.<br>
Maniatis, T., E. F. Fritsh, and J. Sambrook, (1989;. Molecular cloning: a laboratory manual. Cold Spring Harbour Laboratory Press. New York<br>
Mendez, A., C. Sinerdotff A. Izeta, F. Gebauer, and L. Enjuanes.<br>
(19S6). Molecular characterization of transmissible gasrrcente-<br><br>
ritis coronavirus defective interfering genomes: packaging and heterogeneity. Virology.   217:495-507.<br>
Penzes, Z., A. Izeta, C. Smerdou, A. Mendez, M.L. Ballesteros, and L. Enjuanes. (1999). Complete nucleotide sequence of trans¬missible gastroenteritis coronavirus strain PUR4 6-MAD. Submitted for publication<br>
Pushko, P., M. Parker, G.V. Ludwing, N.L. Davis, R.E. Johnston, and J.F. Smith. (1997). Replication-helper systems from attenua¬ted Venezuelan equine encephalitis virus: expression of hetero¬logous genes in vitro and immunization against heterologous pathogens in vivo. Virology.   239:389-401.<br>
Racaniello, V.R., and D. Baltimore. (1981). Cloned poliovirus cDNA is infectious in mammalian cells. Science.   214:916-919.<br>
Radecke, F., P. Spielhofer, H. Schneider, K. Kaelin, M. Huber, C. Dotsch, G. Christiansen, and M.A. Billeter. (1995). Rescue of measles viruses form cloned DNA. EMBO J.   14:5773-5784.<br>
Rice, CM., A. Grakoui, R. Galler, and T.J. Chambers. (1989). Transcription of infectious yellow fever RNA from full-length cDNA templates produced by in vitro ligation. New Biologist. 1:285-296.<br>
Rice, CM., R. Levis, J.H. Strauss, and H.V. Huang. (1987). Production of infectious RNA transcripts from Sindbis virus cDNA dlones: Mapping of lethal mutations, rescue of a temperature-sensitive marker, and in vitro mutagenesis to generate defined mutants. J. Virol.   61:3809-3819.<br>
Rice, CM., and J.H. Strauss. (1981). Synthesis, cleavage, and sequence analysis of DNA complementary to the 2 6S messenger RNA of Sindbis virus. J.   Mol.   Biol.   150:315-340.<br>
Ruggli, N. , J.D. Tratschin, C Mittelholzer, M.A. Hofmann.<br><br>
(1996). Nucleotide sequence of classical swine fever virus strain Alfort/187 and transciption of infectious RNA from stably cloned full-length cDNA. J.   Virol.   70:3479-3487.<br>
Sanchez, C M., G. Jimenez, M. D. Laviada, I. Correa, C. Sune, M. J. Bullido, F. Gebauer, C. Smerdou, P. Callebaut, J. M. Es-cribano, and L. Enjuanes. (1990). Antigenic homology among coro-naviruses related to transmissible gastroenteritis virus. Viro¬logy.   174:410-417 '<br>
Sanchez, CM., F. Gebauer, C. Sune, A. Mendez, J. Dopazo, and L. Enjuanes. (1992). Genetic evolution and tropism of transmissible gastroenteritis coronaviruses. Virology.   190:92-105.<br>
Sanchez, CM., A. Izeta, J.M. Sanchez -Morgado, S. Alonso, I. Sola, M. Balasch, J. Plana-Durfin, and L. Enjuanes. (1999). Tar¬geted recombination demonstrates that the spike gene of trans¬missible gastroenteritis coronavirus is a determinant of its enteric tropism and virulence. J.   Virol.   73:7607-7618.<br>
Sawicki, S.G., and D.L. Sawicki. (1990). Coronavirus transcrip¬tion: subgenomic mouse hepatitis virus replicative intermediates function in RNA synthesis. J.   Virol.   64:1050-1056.<br>
Schnell, M.J., T. Mebatsion, and K.-K. Conzelmann. (1994). In¬fectious rabies viruses from cloned cDNA. EMBO J.   13:4195-4203.<br>
Sethna, P.B., S.-L. Hung, and D.A. Brian. (1989). Coronavirus subgenomic minus-strand RNAs and the potential for mRNA repli-cons. Proc.   Natl.   Acad.   Sci.   USA.   86:5626-5630.<br>
Shizuya, H. , B. Birren, U.-J. Kim, V. Mancino, T. Slepak, Y. Tachiiri, and M. Simon. (1992). Cloning and stable maintenance of 300-kilobase-pair fragmen-cs of human DNA in Escherichia coli using an F-factor-based vector. Proc. Natl. Acad. Sci. USA. 89:8794-8797.<br><br>
Siddell, S.G. 1995. The Coronaviridae. Plenum Press, New York. 418 pp.<br>
Smerdou, C, and P. Liljestrom. (1999). Non-viral amplification systems for gene transfer: vectors based on alphaviruses. Curr. Opin.   Hoi.   Therap.   1:244 — 251.<br>
Taniguchi, M., and F.A.P. Miller. (1978). Specific suppressive factors produced by hybridomas derived from the fusion of enri¬ched suppressor T cells and A T lymphoma cell line. J. Exp. Med. 148:373-382.<br>
van der Most, R.G., and W.J.M. Spaan. 1995. Coronavirus replica¬tion, transcription, and RNA recombination. In The Coronaviri¬dae. S.G. Siddell, editor. Plenum Press, New York. 11-31.<br>
Wang, K., C. Boysen, H. Shizuya, M.I. Simon, and L. Hood.<br>
(1997). Complete nucleotide sequence of two generations of a bacterial artificial chromosome cloning vector. BioTechniques. 23:992-994.<br>
Woo, S.-S-, J. Jiang, B.S. Gill, A.H. Paterson, and R.A. Wing.<br>
(1994). Construction and characterization of a bacterial artifi¬cial chromosome library of Sorghum bicolor. Nucleic Acids Res. 22:4922-4931.<br>
i Zhang, X., C.L. Liao, and M.M.C. Lai. (1994). Coronavirus leader RNA regulates and initiates subgenomic mRNA transcription both in trans  and in els.   J.   Virol.   68:4738-4746.<br><br><br>
We Claim:<br>
1.	A method of preparing a DNA comprising sequences derived from the genomic RNA (gRNA) of a coronavirus said sequences having a homology of at least 60% to the natural sequence of the virus and coding for an RNA dependent RNA polymerase and at least one structural or non-structural protein, wherein a fragment of said DNA is capable of being transcribed into RNA and assembled to a virion, said method comprising the steps, wherein a coronavirus interfering defective genome is cloned under the expression of a promotor into a bacterial artificial chromosome (BAC) and the deleted sequences within the defective genome are re-inserted into said genome.<br>
2.	A method of preparing a DNA as claimed in claim 1, wherein sequences within the viral genome which are toxic for the bacteria into which it is to be cloned are identified before re-insertion into said DNA.<br>
3.	A method of preparing a DNA as claimed in claim 1 or 2, wherein the toxic sequences within the viral genome are the last sequences to be re-inserted when completing the genome.<br><br>
4.	A method of preparing a DNA as claimed in one of claims 1 to 3, wherein an infective clone is obtained which comprises a full-length copy of complementary DNA (cDNA) to the genomic RNA (gRNA) of a coronavirus, cloned under a transcription-regulatory sequence, said method comprises constructing the full-length cDNA from the gRNA of a coronavirus and joining the transcription-regulatory elements to said full-length cDNA.<br>
5.	A method of preparing a DNAjas claimed in claim 4, in which the construction of the full-length cDNA of the gRNA of a coronavirus comprises: (i) cloning an interfering defective genome derived from said coronavirus under a promoter of expression in a BAC; (ii) completing the deletions of said interfering defective genome by regenerating the deleted sequences with respect to the infective gRNA; (iii) identifying the sequences in the coronavirus which are toxic for the bacteria in which it is going to be cloned, removing said toxic sequences, and inserting said toxic sequences just before effecting the transfection in eukaryotic cells to obtain the cDNA clone corresponding to the gRNA of the coronavirus.<br>
Dated this the 7th day of May, 2002.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA1NzktbXVtLWNhbmNlbGxlZCBwYWdlcygxNi0xMi0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00579-mum-cancelled pages(16-12-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDA1NzktTVVNLUNBTkNFTExFRCBQQUdFUyg2LTYtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-00579-MUM-CANCELLED PAGES(6-6-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDA1NzktTVVNLUNMQUlNUyg3LTUtMjAwMikucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-00579-MUM-CLAIMS(7-5-2002).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA1NzktbXVtLWNsYWltcyhncmFudGVkKS0oMTYtOC0yMDA3KS5kb2M=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00579-mum-claims(granted)-(16-8-2007).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA1NzktbXVtLWNsYWltcyhncmFudGVkKS0oMTYtOC0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00579-mum-claims(granted)-(16-8-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDA1NzktTVVNLUNMQUlNUyhHUkFOVEVEKS0oNi02LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-00579-MUM-CLAIMS(GRANTED)-(6-6-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDA1NzktTVVNLUNPUlJFU1BPTkRFTkNFKDIyLTMtMjAwNikucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-00579-MUM-CORRESPONDENCE(22-3-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA1NzktbXVtLWNvcnJlc3BvbmRlbmNlKDgtOC0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00579-mum-correspondence(8-8-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDA1NzktTVVNLUNPUlJFU1BPTkRFTkNFKElQTyktKDE2LTctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-00579-MUM-CORRESPONDENCE(IPO)-(16-7-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA1NzktbXVtLWNvcnJlc3BvbmRlbmNlKGlwbyktKDYtNi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00579-mum-correspondence(ipo)-(6-6-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDA1NzktTVVNLURFU0NSSVBUSU9OKENPTVBMRVRFKS0oNy01LTIwMDIpLnBkZg==" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-00579-MUM-DESCRIPTION(COMPLETE)-(7-5-2002).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDA1NzktTVVNLURFU0NSSVBUSU9OKEdSQU5URUQpLSg2LTYtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-00579-MUM-DESCRIPTION(GRANTED)-(6-6-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDA1NzktTVVNLURSQVdJTkcoNy01LTIwMDIpLnBkZg==" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-00579-MUM-DRAWING(7-5-2002).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDA1NzktTVVNLURSQVdJTkcoR1JBTlRFRCktKDYtNi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-00579-MUM-DRAWING(GRANTED)-(6-6-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDA1NzktTVVNLUZPUk0gMSgxNi0xMi0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-00579-MUM-FORM 1(16-12-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDA1NzktTVVNLUZPUk0gMSg3LTUtMjAwMikucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-00579-MUM-FORM 1(7-5-2002).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA1NzktbXVtLWZvcm0gMTMoMTYtOC0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00579-mum-form 13(16-8-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA1NzktbXVtLWZvcm0gMTkoMi0xMS0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00579-mum-form 19(2-11-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA1NzktbXVtLWZvcm0gMWEoMTYtOC0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00579-mum-form 1a(16-8-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDA1NzktTVVNLUZPUk0gMihDT01QTEVURSktKDctNS0yMDAyKS5wZGY=" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-00579-MUM-FORM 2(COMPLETE)-(7-5-2002).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA1NzktbXVtLWZvcm0gMihncmFudGVkKS0oMTYtOC0yMDA3KS5kb2M=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00579-mum-form 2(granted)-(16-8-2007).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA1NzktbXVtLWZvcm0gMihncmFudGVkKS0oMTYtOC0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00579-mum-form 2(granted)-(16-8-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDA1NzktTVVNLUZPUk0gMihUSVRMRSBQQUdFKS0oNy01LTIwMDIpLnBkZg==" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-00579-MUM-FORM 2(TITLE PAGE)-(7-5-2002).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDA1NzktTVVNLUZPUk0gMihUSVRMRSBQQUdFKS0oR1JBTlRFRCktKDYtNi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-00579-MUM-FORM 2(TITLE PAGE)-(GRANTED)-(6-6-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA1NzktbXVtLWZvcm0gMygxNi0xMi0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00579-mum-form 3(16-12-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA1NzktbXVtLWZvcm0gMyg3LTUtMjAwMikucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00579-mum-form 3(7-5-2002).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA1NzktbXVtLWZvcm0gNSg3LTUtMjAwMikucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00579-mum-form 5(7-5-2002).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA1NzktbXVtLWZvcm0tcGN0LWlwZWEtNDA5KDE2LTEyLTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-00579-mum-form-pct-ipea-409(16-12-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA1NzktbXVtLW90aGVyIGRvY3VtZW50KDIxLTUtMjAwMSkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00579-mum-other document(21-5-2001).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA1NzktbXVtLXBldGl0aW9uIG9mIHVuZGVyIHJ1bGUgMTM3KDE2LTEyLTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-00579-mum-petition of under rule 137(16-12-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDA1NzktTVVNLVBPV0VSIE9GIEFVVEhPUklUWSgxLTctMjAwMikucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-00579-MUM-POWER OF AUTHORITY(1-7-2002).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDA1NzktbXVtLXBvd2VyIG9mIGF1dGhvcml0eSg3LTUtMjAwMikucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00579-mum-power of authority(7-5-2002).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDA1NzktTVVNLVNFUVVFTkNFIExJU1RJTkcoNi02LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-00579-MUM-SEQUENCE LISTING(6-6-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDA1NzktTVVNLVNFUVVFTkNFIExJU1RJTkcoNy01LTIwMDIpLnBkZg==" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-00579-MUM-SEQUENCE LISTING(7-5-2002).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDA1NzktTVVNLVNQRUNJRklDQVRJT04oQU1FTkRFRCktKDE2LTEyLTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-00579-MUM-SPECIFICATION(AMENDED)-(16-12-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDA1NzktTVVNLVNQRUNJRklDQVRJT04oQU1FTkRFRCktKDYtNi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-00579-MUM-SPECIFICATION(AMENDED)-(6-6-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDA1NzktTVVNLVdPIElOVEVSTkFUSU9OQUwgUFVCTElDQVRJT04gUkVQT1JUKDctNS0yMDAyKS5wZGY=" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-00579-MUM-WO INTERNATIONAL PUBLICATION REPORT(7-5-2002).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="220820-clarifying-agent-formulations-for-thermoplastics-exhibiting-very-high-nucleation-efficacy.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="220822-a-pharmaceutical-composition-for-oral-use-comprising-acid-labile-substances.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>220821</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2002/00579/MUM</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>33/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>15-Aug-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>06-Jun-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>07-May-2002</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CONSEJO SUPERIOR DE INVESTIGACIONES CIENTIFICAS</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>SERRANO, 117, E-28006 MADRID, SPAIN.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>LUIS ENJUANES SANCHEZ</td>
											<td>CAMINO NUEVO 113, LA MORALEJA, E-28109 MADRID, SPAIN.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A61K 39/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP00/12063</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-11-30</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>9902673</td>
									<td>1999-12-03</td>
								    <td>Spain</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/220821-a-method-of-preparing-a-dna by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:19:26 GMT -->
</html>
