<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/221076-intact-minicells-as-vectors-for-dna-transfer-and-gene-therapy-in-vitro-and-in-vivo by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:02:29 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 221076:INTACT MINICELLS AS VECTORS FOR DNA TRANSFER AND GENE THERAPY IN VITRO AND IN VIVO</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">INTACT MINICELLS AS VECTORS FOR DNA TRANSFER AND GENE THERAPY IN VITRO AND IN VIVO</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A composition comprising recombinant, intact minicells that contain a therapeutic nucleic acid molecule is disclosed. Methods for purifying a preparation of such minicells also are disclosed. Additionally, a genetic transformation method is disclosed, which comprises (i) making recombinant, intact minicells available that contain a plasmid comprised of a first nucleic acid segment, and (ii) bringing the minicells into contact with mammalian cells that are engulfing-competent, such that the minicells are engulfed by the mammalian cells, which thereafter produce an expression product of the first nucleic acid segment.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>	FORM 2<br>
	THE PATENT ACT 1970<br>
	(39 of 1970)<br>
	&amp;<br>
	The Patents Rales, 2003<br>
	COMPLETE SPECIFICATION<br>
	(See section 10 and rule 13)<br>
1. INTACT MINICELLS AS VECTORS FOR DNA TRANSFER AND GENE<br>
THERAPY IN VITRO AND IN VIVO<br>
2. (A)	ENGENEIC MOLECULAR DELIVERY PTY LTD.<br>
(B)	AUSTRALIA<br>
(Q	105 Delhi Road, North Ryde,<br>
	Sydney, NSW 2113.<br>
	AUSTRALIA<br><br>
INTACT MINICELLS AS VECTORS<br>
FOR DNA TRANSFER AND GENE THERAPY<br>
IN VITRO AND IN VIVO<br>
BACKGROUND OF THE INVENTION<br>
The present invention relates to the delivery, by means of intact bacterial minicells, of oligonucleotides and polynucleotides to host cells, particularly but not exclusively in the context of gene therapy. The invention also relates to a pharmaceutically compatible method for purifying intact bacterial minicells.<br>
A U.S. patent to Salser et al., No. 4,497,7%, illuminates various early approaches, available circa 1980, for transforming mammalian cells. In particular, Salser et al. disclose transfer of a gene encoding dihydrofolate reductase, which confers methotrexate resistance, into mouse L1210 cells or bone marrow cells, by the technique of DNA co-precipitation with calcium phosphate.<br>
Salser et al. also mention a "number of [other] ways...for insertion of genetic materials into cells,' including, in addition to "viral vectors,'* certain cell-fusion techniques: "cell-cell fusion involving the fusion of cells of a limited number of chromosomes enveloped in nuclear membranes;...minicell fusion;...fusion with bacterial protoplasts containing plasmid DNA; and fusion with erythrocyte ghosts packaged with DNA" (column 5, lines 18 - 30; citations omitted). Common to these techniques is the use of a DNA-containing cellular structure, delimited by a single membrane, that is brought into contact with a target mammalian cell in the presence of a membrane-fusion promoting agent, such as polyethylene glycol (PEG). The mammalian cell and the cellular structure fuse, forming a hybrid cell, and the DNA contained in the latter is released into the cytoplasm of the former and transported to the nucleus of the hybrid, which then may express the DNA.<br>
2<br><br>
The Salser disclosure mentions "erythrocyte ghosts," erythrocyte remnants that are devoid of cytoplasmic contents but that retain original morphology, and "bacterial protoplasts," or bacterial cells from which the outer membrane (cell wall) has been stripped, typically by the action of a lytic enzyme or an antibiotic that inhibits peptkloglycan synthesis. Salser et al. also allude to the use of a third type of simplified cellular structure, a minicell protoplast.<br>
A minicell is an anucleate form of an E. coli or other bacterial cell, engendered by a disturbance in the coordination, during binary fission, of cell division with DNA segregation. Prokaryotic chromosomal replication is linked to normal binary fission, which involves mid-cell septum formation. In E. coli, for example, mutation of min genes, such as minCD, can remove the inhibition of septum formation at the cell poles during cell division, resulting in production of a normal daughter cell and an anulceate minicell (de Boer et al., 1992; Raskin &amp; de Boer, 1999; Hu &amp; Lutkenhaus, 1999; Harry, 2001).<br>
In addition to min operon mutations, anucleate minicells also are generated following a range of other genetic rearrangements or mutations that affect septum formation, for example in the divIVBl in B. subtilis (Reeve and Cornett, 1975; Levin et al., 1992). Minicells also can be formed following a perturbation in the levels of gene expression of proteins involved in cell division/chromosome segregation. For example, overexpression of rmriE leads to polar division and production of minicells. Similarly, chromosome-less minicells may result from defects in chromosome segregation for example the smc mutation in Bacillus subtilis (Britton et al., 1998), spoOJ deletion in B. subtilis (Ireton et al., 1994), muKB mutation in E. coli (Hiraga et al., 1989), and parC mutation in E. coli (Stewart and D'Ari, 1992). Gene products may be supplied in trans. When over-expressed from a high-copy number plasmid, for example, CafA may enhance the rate of cell division and/or inhibit chromosome partitioning after replication (Okada et al., 1994), resulting in formation of chained cells and anucleate minicells (Wachi et al., 1989; Okada etal., 1993).<br><br>
Minicells are distinct from other small vesicles that are generated and released spontaneously in certain situations and, in contrast to minicells, are not due to specific genetic rearrangements or episomal gene expression. Exemplary of such omer vesicles are bacterial blebs, which are small membrane vesicles (Dorward et al., 1989). Blebs have been observed in several bacterial species from Agrobacterium, Bacillus, Bordetella, Escherichia, Neisseria, Pseudomonas, Salmonella and Shigella, for example. Bacterial blebs can be produced, for instance, through manipulation of the growth environment (Katsui et al., 1982) and through the use of exogenous membrane-destabilizing agents (Matsuzaki et al., 1997).<br>
Other sub-bacterial components exist, such as bacterial ghosts (Lubhz et al., 1999), which are empty bacterial envelopes formed, in a variety of Gram-negative bacteria when the phiX174 lysis gene E is expressed. Bacterial ghosts are formed from a transmembrane tunnel structure, through a bacterial cell envelope. Due to high osmotic pressure inside the cell, cytoplasmic content is expelled into the surrounding media, leading to an empty bacterial cell envelope.<br>
Because plasmid replication within prokaryotic cells is independent of chromosomal replication, plasmids can segregate into both normal daughter cells and minicells during the aberrant cell division described above. Thus, minicells derived from recombinant min' E. coli carry significant numbers of plasmid copies, with all of the bacterial cellular components except for chromosomes, and have been used as such in studying plasmid-encoded gene expression in vitro. See Brahmbhatt (1987), Harlow et al. (1995), and Kihara et al. (1996). Brahmbhatt (1987) demonstrated, for example, that E. coli minicells can carry recombinant plasmids with DNA inserts as large as 20 kb, absent any chromosomal DNA, and can express nine or more recombinant proteins simultaneously.<br>
Such non-reproductive but metabolically active, intact minicells have been employed for analyzing proteins encoded by plasmid-bome genes. In the context of<br>
4<br><br>
the "minicell fusion" mentioned by Salser et al., however, minicells are stripped of their outer membranes to yield a structure that, like a bacterial protoplast, can undergo fusion with a target cell when both are incubated together with PEG or another agent which promotes cell fusion. Also like bacterial protoplasts, such minicell protoplasts must be maintained under isotonic conditions, in order to prevent osmotic lysis, and they are highly vulnerable to enzymatic attack. Thus, they are unsuitable for gene therapy. Further, with the advent of other, more convenient transformation methodology, the minicell technology referred to by Salser et al., fell into disuse.<br>
More recently, the advance of gene therapy has highlighted a variety of methods for introducing exogenous genetic material into the genome of a recipient mammal. See reviews by Romano et al. (1998, 1999), Balicki and Beutler (2002), and Wadhwa et al. (2002). The clinical application of these techniques, such as the utilization of adenovirus or recombinant retrovirus vectors, has been delayed because of serious safety concerns. Illustrative of the problems presented by transformation methodology now are recombination with wild-type viruses, insertional and oncogenic potential, virus-induced immunosuppression, limited capacity of the viral vectors to carry large segments of therapeutic DNA, reversion to virulence of attenuated viruses, difficulties in recombinant virus manufacture and distribution, low stability, and adverse reactions, such as an inflammatory response, caused by existing immunity. An approach that obviated these problems would offer significant benefit in making gene therapy safer and more effective.<br>
Live attenuated bacterial vectors also are being explored as gene delivery vectors for human gene therapy, including Salmonella (Darji et al., 1997; Paglia et al., 2000; Urashima et al., 2000), Shigella (Sizemore et al., 199S; Grillot-Courvalin et al., 2002), Listeria (Dietrich et al, 1998) and invasive E. coll (Grillot-Courvalin et al., 1998). However, bacterial vectors have significant limitations because live bacteria, though attenuated, must be engineered to carry phagolysosome membrane lysis mechanisms, to enable sufficient recombinant DNA<br>
5<br><br>
to escape to the mammalian cell cytosol and hence the nucleus. Such engineering is difficult and may be impossible for many intracellular bacterial pathogens. Moreover, mutations that attenuate bacterial pathogens are known for only a few bacterial species, for example mutations in the aromatic amino acid biosynthesis genes for Salmonella, E. ooti and Shigella.<br>
Because attenuating mutations are not known for many bacterial species, a bacterial gene delivery system cannot exploit the vast battery of bacterial intracellular pathogens. Bacterial vectors raise an additional concern regarding the presence of chromosomal DNA because parts of this DNA could be transferred to other microflora in the human or animal host receiving the gene therapy. Such promiscuous transfer of DNA between bacterial species is undesirable due to a potential for emergence of new virulent and/or drug resistant bacteria.<br>
SUMMARY OF THE INVENTION<br>
To address these and other needs, the present invention provides, in accordance with one aspect, a composition comprising (i) recombinant, intact minicells and (ii) a pharmaceutically acceptable carrier therefor, where the minicells contain a therapeutic nucleic acid molecule encoding, for example, interleukin-2. In a preferred embodiment, the composition contains fewer than one contaminating parent cell per 107, 108, or 109 minicells.<br>
According to another aspect, the present invention provides for a use of recombinant, intact minicells in the preparation of a medicament, die minicells containing a therapeutic nucleic acid molecule, for use in a method of treating a disease or modifying a trait by administration of said medicament to a cell, tissue, or organ. The disease treated in this context may be a cancer, for example, or an acquired disease, such as AIDS, pneumonia, emphysema, or a condition engendered by an inborn error of metabolism, such as cystic fibrosis. Alternatively, the treatment may affect a trait, such as fertility, or an immune response associated with an allergen or an infectious agent.<br>
6<br><br>
The invention also provides, pursuant to a further aspect, a genetic transformation method that comprises (i) providing recombinant, intact minicells that contain a plasmid comprised of a nucleic acid sequence, preferably coding for therapeutic expression product, and then (ii) bringing die minicells into contact with mammalian cells that are phagocytosis- or endocytosis-competent, such that the minicells are engulfed by the mammalian cells, whereby the latter cells produce the expression product of the nucleic acid sequence. The contact between minicells and mammalian cells may be in vitro or in vivo. The aforementioned plasmid also may contain a second nucleic acid segment that functions as a regulatory element, such as a promoter, a terminator, an enhancer or a signal sequence, and that is operably linked to the first nucleic acid segment. Further, the plasmid may contain a reporter element, such as a nucleic acid segment coding for green fluorescent protein.<br>
In accordance with yet another aspect of the present invention, a purification method is provided that comprises passing a sample containing minicells (i) over a series of cross-flow filters and then (ii) through a dead-end filter, whereby minicells are separated from contaminants in said sample to obtain a purified minicell preparation. The method optionally includes a treatment of the purified minicell preparation with an antibiotic. Also optional is a preliminary step of performing differential centrifugation on the mimcell-containing sample. In a preferred embodiment, the series of cross-flow filters comprises (A) at least one or two filters that employ a pore size greater than or equal to about 0.43 /un and (B) at least one filter employing a pore size less than or equal to about 0.2 µm.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figure 1 shows a schematic illustration of normal cell division in a bacterium and how mutations in the nun genes result in the formation of minicells.<br>
Figure 2 shows a schematic diagram of plasmid preparation useful for generating a minicell-producing bacterial strain. In particular, this figure shows a plasmid construction to generate a S. typhimurium-derwcd minicell producing strain.<br>
7<br><br>
Line drawings of cloned insert DNAs and parts of circular plasmid vectors are shown. Plasmid names are shown in bold to the left of insert DNAs and inside circular vector maps. Where PCR primers are used in generating a clone, the primer numbers are shown adjacent to the dashed arrow.<br>
Figure 3 shows a schematic diagram of plasmid preparation useful for generating a minicell-producing bacterial strain. In particular, this figure shows a plasmid construction to generate a Shigella flexneri 2a-derived, minicell producing strain.<br>
Figure 4 shows a schematic diagram of plasmid preparation useful for generating a minicell-producing bacterial strain. In particular, this figure shows a plasmid construction to generate a Listeria monocytogenes-derived, minicell producing strain.<br>
Figure 5 shows TEM images of mouse macrophage cells transfected with S. typhimurium-derived minicells. For each panel, the time post-infection and the magnification are provided. The images show minicell-like electron dense particles (arrows) within macrophage, vacuoles.<br>
Figure 6 shows minicell-mediated gene delivery to human breast cancer cells and heterologous gene expression. Panel A shows control breast cancer cell line SK-BR-3 96 hours post-transfection with non-recombinant minicells and labeled anti-HER-2 antibody, detected with Alexafluor-conjugated secondary antibody. Panel B shows GFP expression in SK-BR-3 cells, 96 hours post-transfection with recombinant minicells carrying plasmid pEGFP-Cl (eukaryotic expression of GFP). Panel C shows minicell/pEGFP-C1-transfected SK-BR-3 cells, labeled with anti-HER-2 antibody and detected with Alexafluor-conjugated secondary antibody. The images were visualized with confocal microscopy.<br>
Figure 7 shows mouse serum anti-GFP response 14 days after intraperitoneal administration of recombinant minicells and killed S. typhimurium<br>
8<br><br>
parent cells carrying plasmid pEGFP-Cl.   In the legend and in the legends of Figures 8 -10,10*8 and 10*9 represent 108 and 109, respectively.<br>
Figure 8 shows mouse serum anti-LPS response 14 days after intraperitoneal administration of recombinant minicells, and killed S. typhimurium parent cells that carry plasmid pEGFP-C1.<br>
Figure 9 shows mouse serum anti-GFP response, 14 days after intraperitoneal administration of recombinant minicells carrying plasmid pEGFP-C1.<br>
Figure 10 shows mouse serum anti-LPS response, 14 days after intraperitoneal administration of recombinant minicells carrying plasmid pEGFP-C1.<br>
DETAILED DESCRIPTION OF PREFERRED EMBODIMENTS<br>
The present inventors have determined that minicells with intact cell walls ("intact minicells") are effective vectors for delivering oligonucleotides and polynucleotides (collectively, "nucleic acid molecules") to host cells (preferably mammalian cells in humans or animals) in vitro and in vivo. Thus, the inventors have found mat an intact minicell is engulfed by mammalian cells mat likewise engulf the bacterial cell from which the minicell was obtained (the "parent bacterial cell"). Further, it has been discovered that, surprisingly, the contents of an intact, recombinant minicell, when engulfed by a host cell, are processed in such a way that at least some plasmid DNA from the minicell escapes degradation and is transported through the cytoplasm to the nucleus of the host cell, which then can express the plasmid DNA.<br>
In accordance with die present invention, therefore, an intact minicell that carries a DNA for which heterologous expression is desired, either in vitro or in vivo, is brought into contact with a mammalian cell that engulfs the parent bacterial cell, in the manner of the intracellular bacterial pathogens. By the same token, that<br>
9<br><br>
mammalian cell, denoted here as "engulfing-competent," engulfs the minicell and expresses the DNA in question.'<br>
A variety of mechanisms may be involved in the engulfing of intact minicells by a given type of host cell, and the present invention is not dependent on any particular mechanism in this regard. For example, phagocytosis is a well-documented process in which macrophages and other phagocyte cells, such as neutrophils, ingest particles by extending pseudopodia over the particle surface until the particle is totally enveloped. Although described as "non-specific" phagocytosis, the involvement of specific receptors in the process has been demonstrated. See Wright &amp; Jong (1986); Speert a al. (1988).<br>
Thus, one form of phagocytosis involves interaction between surface ligands and ligand-receptors located at the membranes of the pseudopodia (Shaw and Griffin, 1981). This attachment step, mediated by the specific receptors, is thought to be dependent on bacterial surface adhesins. With respect to less virulent bacteria, \, such as non-enterotoxigenic E. coli, phagocytosis also may occur in the absence of surface ligands for phagocyte receptors. See Pikaar et al. (1995), for instance. Thus, the present invention encompasses but is not limited to the use of intact minicells that either possess or lack surface adhesins, in keeping with the nature of their parent bacterial cells, and are engulfed by phagocytes (i.e., "phagocytosis-competent" host cells), of which neutrophils and macrophages are the primary types in mammals.<br>
Another engulfing process is endocytosis, by which intracellular pathogens exemplified by species of Salmonella, Escherichia, Shigella, Helicobacter, Pseudomonas and Lactobacilli gain entry to mammalian epithelial cells and replicate there. Two basic mechanisms in this regard are Gathrin-dependent receptor-mediated endocytosis, also known as "coated pit endocytosis" (Riezman, 1993), and Clathrin-independent endocytosis (Sandvig &amp; Deurs, 1994). Either or both may be involved when an engulfing-competent cell that acts by endocytosis (i.e., an<br>
10<br><br>
"endocytosis-competent" host cell) engulfs intact, recombinant minicells and expresses DNA carried by the minicells, in accordance with the invention. Representative endocytosis-competent cells are breast epithelial cells, enterocytes in the gastrointestinal tract, stomach epithelial cells, lung epithelial cells, and urinary tract and bladder epithelial cells.<br>
The present invention is particularly useful for introducing, into engulfing-competent cells, nucleic acid molecules mat, upon transcription and/or translation, function to ameliorate or otherwise treat a disease or modify a trait associated with a particular cell type, tissue, or organ of a subject. For purposes of the present description, these molecules are categorized as "therapeutic nucleic acid molecules."<br>
For instance, transcription or translation of a given therapeutic nucleic acid molecule may be useful in treating cancer or an acquired disease, such as AIDS, pneumonia, emphysema, or in correcting inborn errors of metabolism, such as cystic fibrosis. Transcription or translation of a therapeutic nucleic acid may also effect contraceptive sterilization, including contraceptive sterilization of feral animals. Allergen-mediated and infectious agent-mediated inflammatory disorders also can be countered by administering, via the present invention, a therapeutic nucleic acid molecule that, upon expression in a patient, affects immune response(s) associated with the allergen and infectious agent, respectively. A therapeutic nucleic acid molecule also may have an expression product, or there may be a downstream product of post-translational modification of the expression product, that reduces the immunologic sequalae related to transplantation or that helps facilitate tissue growth and regeneration.<br>
Alternatively, the expression product or a related, post-translational agent may be a protein, typified by erythropoietin, a growth factor such as TGF-β, a cytokine such as IL-2, IL-10, IL-12 or another of the interleukins, a serum leucoproteinase inhibitor, or an antibody such as CTLA4-Ig, that provides a desired<br>
21<br><br>
benefit for the host. Other such proteins are, without limitation, α-, β- and γ-globin, insulin, GM-CSF, M-CSF, G-CSF, EPO, TNF, MGF, adenosine deaminase, tumor-associated antigens such as viral, mutated or aberrantly-expressed antigens (CDK4, β -catenin, GnT-V, Casp8), cancer-specific antigens such as MAGE, BAGE, GAGE, PRAME, and NY-ESO-1, differentiation antigens such as tyrosinase, Melan-A/MART-1, gplOO, and TRP-l/gp75, and overexpressed antigens such as Her2/neu and CEA. Further exemplars of the class of therapeutic nucleic acid molecules are DNAs coding for the cystic fibrosis transmembrane regulator (CFTR), Factor VIII or another blood protein, low density lipoprotein receptor, β -glucocerebrosidase, α- and β -galactosidase, insulin, parathyroid hormone, α -l-antitrypsin.fasR, and fasL, respectively.<br>
In accordance with the present invention, expression of a therapeutic nucleic acid molecule by a host cell can supply a needed compound, mediate a targeted immune response, or interrupt a pathological process. For example, a therapeutic nucleic acid molecule can be implicated in an antisense or ribozyme therapy. In the present context, an antisense therapy involves introducing, pursuant to the invention, an antisense copy of a polynucleotide sequence, such mat the RNA transcript of the copy hybridizes in situ with a messenger RNA (mRNA) transcript which corresponds to the polynucleotide sequence. This hybridization typically prevents the transcript from being translated into a protein or initiates a degradation pathway which destroys the mRNA.<br>
The therapeutic nucleic acid also may encode short interfering RNA duplexes (siRNA's) within a mammalian cell. That is, siRNA's have been shown to suppress target genes in mammalian cells. This process of RNA interference (RNAi) is a method of sequence-specific, post-transcriptional gene silencing in animals, humans and plants and is initiated by double-stranded (ds) RNA that is homologous to the silenced gene. Viral-mediated delivery of siRNA to specifically reduce expression of targeted genes in various cell types has been successfully demonstrated (Haibin et al., 2002).<br>
11<br><br>
For all of these and other diverse uses of a therapeutic nucleic acid molecule, the present description employs die rubric of "gene therapy," which also is connoted by the phrases "gene transfer," "gene delivery," and "gene-based vaccines," in relation to methodology or systems for transferring a therapeutic nucleic acid molecule into host cells, both in vivo and ex vivo, as described, for instance, in U.S. patent No. 5,399,346. Thus, a minicell-containing composition of me present invention can be used to achieve an in situ therapeutic effect, and for transforming cells outside of the body and men (re)introducing diem to a subject. Pursuant to die present invention, die cells suitable for both in vivo and ex vivo approaches would be those that are engulfing-competent, such as phagocytes and epithelial cells.<br>
As noted above, gene therapy may be effected to treat or prevent a genetic or acquired disease or condition. The •terapeutic nucleic acid molecule encodes a product, such as functional RNA (e.g., antisense or siRNA) or a peptide, polypeptide or protein, the in vivo production of which is desired. For example, the genetic material of interest can encode a hormone, receptor, enzyme, or (poly) peptide of therapeutic value. Such methods can result in transient expression of non-integrated transferred DNA, extrachromosomal replication and expression of transferred replicons such as episomes, or integration of transferred genetic material into die genomic DNA of host cells.<br>
A therapeutic nucleic acid molecule may be the normal counterpart of a gene that expresses a protein that functions abnormally or that is present in abnormal levels in a disease state, as is die case, for example, with the cystic fibrosis transmembrane conductance regulator in cystic fibrosis (Kerem et al., 1989; Riordan et al., 1989; Rommens et al., 1989), with B-globin in sickle-cell anemia, and witti any of α -globin, β-globin and γ-globin in thalassemia. The merapeutic nucleic acid molecule can have an antisense RNA transcript or small interfering RNA, as mentioned above. Thus, an excess production of α-globin over β-globin which characterizes  β-thalassemia can be  ameliorated  by  gene therapy,   in<br>
13<br><br>
accordance with the present invention, using an intact minicell engineered to contain a plasmid incorporating a sequence that has an antisense RNA transcript vis-a-vis a target sequence of the α -globin mRNA.<br>
In the treatment of cancer, a therapeutic nucleic acid molecule suitable for use according to the present invention could have a sequence that corresponds to or is derived from a gene mat is associated with tumor suppression, such as the p53 gene, the retinoblastoma gene, and the gene encoding tumor necrosis factor. A wide variety of solid tumors - cancer, papillomas, and warts - should be treatable by this approach, pursuant to the invention. Representative cancers in this regard include colon carcinoma, prostate cancer, breast cancer, lung cancer, skin cancer, liver cancer, bone cancer, ovary cancer, pancreas cancer, brain cancer, head and neck cancer, and lymphoma. Illustrative papillomas are squamous cell papilloma, choroid plexus papilloma and laryngeal papilloma. Examples of wart condition:; are genital warts, plantar warts, epidermodysplasia verruciformis, and malignant warts.<br>
A therapeutic nucleic acid molecule for the present invention also can comprise a DNA segment coding for an enzyme that converts an inactive prodrug into one or more cytotoxic metabolites so that, upon in vivo introduction of the prodrug, the target cell in effect is compelled, perhaps with neighboring cells as well, to commit suicide. Preclinical and clinical applications of such a "suicide gene," which can be of non-human origin or human origin, are reviewed by Spencer (2000), Shangara et al. (2000) and Yazawa et al. 0002). Illustrative of suicide genes of non-human origin are those mat code for HSV-thymidine kinaseOfc), cytosine deaminase (CDA) + uracil phophoribosytransferase, xanthine-guanine phophoribosyl-transferase (GPT), nitroreductase (NTR), purine nucleoside phophrylase (PNP, DeoD), cytochrome P450 (CYP4B1), carboxypeptidase G2 (CPG2), and D-amino acid oxidase (DAAO), respectively. Human-origin suicide genes are exemplified by genes that encode carboxypeptidase Al (CPA), deoxycytidine kinase (dCK), cytochrome P450 (CYP2B1.6), LNGFR/FKBP/Fas, FKBP/Caspases, and ER/p53, respectively.<br>
14<br><br>
A suicide-gene therapy according to the present invention could be applied to the treatment of AIDS. This strategy has been tested with suicide vectors that express a toxic gene product as soon as treated mammalian cells become infected by HIV-1. These vectors use the HIV-1 regulatory elements, Tat and/or Rev, to induce the expression of a toxic gene such as a-diphtheria toxin, cytosine deaminase, or interferon-a2 after infection by HIV-1 (Curiel et al., 1993; Dinges et al., 1995; Harrison et al., 1992a; Harrison et al., 1992b; Ragheb et al., 1999). Cells could be transduced by these vectors, using the recombinant-minicell approach of this invention, and would be eliminated raster than untransduced cells after HIV infection, preventing viral replication at the expense of cell death.<br>
A nucleic acid molecule to be introduced via the approach of the present invention can include a reporter element. A reporter element confers on its recombinant host a readily detectable phenotype or characteristic, typically by encoding a polypeptide, not otherwise produced by the host, that can be detected, upon expression, by histological or in situ analysis, such as by in vivo imaging techniques. For example, a reporter element delivered by an intact minicell, according to the present invention, could code for a protein that produces, in the engulfing host cell, a colorimetrie or fluorometric change that is detectable by in situ analysis and that is a quantitative or semi-quantitative function of transcriptional activation. Illustrative of these proteins are esterases, phosphatases, proteases and other enzymes, the activity of which generates a detectable chromophore or fluorophore.<br>
Preferred examples are E. coli p-galactosidase, which effects a color change via cleavage of an indigogenic substrate, indolyl-P-D-galactoside, and a luciferase, which oxidizes a long-chain aldehyde (bacterial luciferase) or a heterocyclic carboxylic acid (luciferin), with the concomitant release of light. Also useful in this context is a reporter element mat encodes the green fluorescent protein (GFP) of the jellyfish, Aequorea victoria, as described by Prasher et al. (1995). The field of GFP-related technology is illustrated by two published PCT applications, WO<br>
15<br><br>
093/21191 (discloses a polynucleotide sequence encoding a 238 amino-acid GFP apoprotein, containing a chromophore formed from amino acids 65 through 67) and WO 095/21191 (discloses a modification of the cDNA for the apopeptide of A. victoria GFP, providing a peptide having altered fluorescent properties), and by a report of Heim et al. (1994) of a mutant GFP, characterized by a 4-to-6-fbld improvement in excitation amplitude.<br>
Another type of a reporter element is associated with an expression product that renders the recombinant minicell resistant to a toxin. For instance, the neo gene protects a host against toxic levels of the antibiotic G418, while a gene encoding dihydrofolate reductase confers resistance to methotrexate, and the chloramphenicol acetyltransferase (CAT) gene bestows resistance to chloramphenicol.<br>
Other genes for use as a reporter element include those that can transform a host minicell to express distinguishing cell-surface antigens, e.g., viral envelope proteins such as HIV gpl20 or herpes gD, which are readily detectable by immunoassays.<br>
A nucleic acid molecule to be introduced via the approach of the present invention also can have a desired encoding segment linked operatively to a regulatory element, such as a promoter, a terminator, an enhancer and/or a signal sequence. A suitable promoter can be tissue-specific or even tumor-specific, as the therapeutic context dictates.<br>
A promoter is "tissue-specific" when it is activated preferentially in a given tissue and, hence, is effective in driving expression, in the target tissue, of an operably linked structural, sequence. The category of tissue-specific promoters includes, for example: the hepatocyte-specific promoter for albumin and ai-antitrypsin, respectively; the elastase I gene control region, which is active in pancreatic acinar cells; the insulin gene control region, active in pancreatic beta cells; the mouse mammary tumor virus control region, which is active in testicular,<br>
16<br><br>
breast, lymphoid and mast cells; the myelin basic protein gene control region, active in oligodendrocyte cells in the brain; and the gonadotropic releasing hormone gene control region, which is active in cells of the hypothalamus. See Frain et al. (1990), Ciliberto et al. (1985), Pinkert et al., (1987), Kelsey et al. (1987), Swift et al. (1984), MacDonald (1987), Hanahan, (1985), Leder et al. (1986), Readhead et al. (1987), and Mason et al. (1986).<br>
There also are promoters that are expressed preferentially in certain tumor cells or in tumor cells per se, and that are useful for treating different cancers in accordance with the present invention. The class of promoters mat are specific for cancer cells is illustrated by: the tyrosinase promoter, to target melanomas; the MUCl/Df3 promoter, to target breast carcinoma; the hybrid myoD enhancer/SV40 promoter, which targets expression to rhabdomyosarcoma (RMS); the carcinoembryonic antigen (CEA) promoter, which is specific for CEA-expressing cells such as colon cancer cells, and the hexokinase type II gene promoter, to target non-small cell lung carcinomas. See Hart (1996), Morton &amp; Potter (1998), Kurane et al. (1998) and Katabi et al. (1999).<br>
A signal sequence can be used, according to the present invention, to effect secretion of an expression product or localization of an expression product to a particular cellular compartment. Thus, a therapeutic polynucleotide molecule mat is delivered via intact minicells may include a signal sequence, in proper reading frame, such that the expression product of interest is secreted by an engulfing cell or its progeny, thereby to influence surrounding cells, in keeping with the chosen treatment paradigm. Illustrative signal sequences include the haemolysin C-terminal secretion sequence, described in U.S. patent No. 5,143,830, the BAR1 secretion sequence, disclosed in U.S. patent No. 5,037,743, and the signal sequence portion of the zsig32 polypeptide, described in U.S. patent No. 6,025,197.<br>
The ability of intact, recombinant minicells of the invention to maintain integrity in vivo makes possible their use in gene therapy, as described above.<br>
17<br><br>
Intact minicells also carry none of the bacterial genomic DNA mat is present in recombinant bacterial cells, for example, of Shigella flexneri, listeria monocytogenes, Escherichia coli or Salmonella typhimuriwn, which others have used to transfer eukaryotic expression plasmids into host cells. See Sizemore et al. (1995); Gentschev et al. (2000); Catic et al. (1999); Dietrich et al. (1998); and Courvalin et al. (199S). Accordingly, gene therapy with intact minicells, pursuant to the present invention, does not entail the risk, associated with use of the recombinant bacteria, of the unintended transfer of a bacterial or an antibiotic resistance-marker gene to microbial flora which are indigenous to the patient. Furthermore, recombinant bacteria must be cleared from the patient by means of cell-mediated immunity and, hence, are unsuitable for gene therapy of an immunocompromised patient suffering, for example, from cancer or AIDS.<br>
Minicells can be prepared from any bacterial cell of Gram-positive or Gram-negative origin. Because the present invention employs intact minicells, rather than minicell protoplasts, the present invention does not require and preferably does not involve any antibiotic or chemical pretreatment of minicells.<br>
A minicell-producing bacterial strain can be generated by mutation of a min gene, for example, through a partial deletion of a minCDE gene sequence, as illustrated in Examples 1 and 2 below. To obtain recombinant, intact minicells, according to the invention, bacterial cells of the chosen minicell-producing strain are transformed via a standard technique, including but not limited to the use of electroporation (Shigekawa &amp; Dower, 1988), chemical methodology (Hanahan, 1983), shuttle vectors (Marcil &amp; Higgins, 1992), and conjugation (Firth et al., 1996) or transduction (Davis et al., 1980). For this transformation, a therapeutic nucleic acid molecule from any eukaryotic, prokaryotic or synthetic source is inserted into a suitable, commercially available or end user-proprietary plasmid vector. A selected DNA can be operably linked to the control elements required for gene delivery and/or expression of the DNA in the cells that engulf the recombinant minicells in vivo.<br>
17<br><br>
Recombinant minicells preferably are analyzed in vitro to ensure that they can deliver the recombinant plasmid or DNA sequences to the target cell nucleus, and mat recombinant gene expression occurs in the target cell. The assay for expression will depend upon the nature of the heterologous gene. Expression may be monitored by a variety of methods, including immunological, histochemical or activity assays. The expression of a fluorescent marker gene can be visualized microscopically, and this provides a particularly convenient assay. For example, the gene encoding green fluorescence protein under the control of a eukaryotic gene expression promoter, such as the CMV promoter, may be transferred on a plasmid by recombinant minicells to target eukaryotic cells (see below). Additionally, northern blot analysis or Reverse Transcriptase PCR (RT-PCR) may be used to assess transcription using appropriate DNA or RNA probes. If antibodies to the polypeptide encoded by the heterologous gene are available, Western blot analysis, immunohistochemistry or other immunological techniques can be used to assess die production of the polypeptide. Appropriate biochemical assays also can be used if the heterologous gene is an enzyme. For instance, if the heterologous gene encodes antibiotic resistance, then a determination of the resistance of infected cells to the antibiotic can be used to evaluate expression of the antibiotic resistance gene.<br>
The engulfing-competency of a putative host ("target") cell can be evaluated by testing the efficacy of DNA delivery in culture. Thus, a targeted tumor could be subjected to biopsy, and the resultant tissue sample would be used, in conventional manner, to obtain representative tumor cells in culture, to test for an ability to engulf recombinant minicells of the present invention that carry, for example, a suitable reporter element. Additionally or alternatively to testing such a primary cell culture, one may gauge me capacity to engulf minicells by testing a cell line mat is representative of the type of tissue to which the therapeutic protocol is keyed, pursuant to the present invention. Cell culture techniques are well documented, for instance, by Freshner (1992).<br>
18<br><br>
In accordance with the invention, recombinant minicells can be purified from parent cells by several means. One approach utilizes sucrose gradient metbodology described, for example, by Reeve (1979) and by Clark-Curtiss et al. (1983), followed by treatment with an antibiotic, such as gentamycin (200 iig/ml, 2 hours), to kill residual live bacteria.<br>
With the conventional methodology, the purity achieved is one contaminating parent cell per 106 to 107 minicells, at best. For in vivo applications, according to the present invention, doses greater than 106 may be required and may be as high as 1010 per dose, which, with the aforementioned contamination ratio, would translate into 10,000 live parent cells per dose. Such a contamination level could be fatal, particularly in immuno-compromised subjects.<br>
In addition, the conventional technology employs media mat contain a gradient-formation agent, such as sucrose, glycerol or Percoll, the presence of which is undesirable for in vivo uses, as presently contemplated. Thus, the toxicity of Percoll* restricts it to "research purposes only" contexts, while sucrose for a gradient imparts a high osmolarity mat can cause physiological changes in the minicells.<br>
For gene therapy applications according to the present invention, therefore, it is preferable to minimize parent cell contamination and to utilize media that are more biologically compatible. To achieve these goals, the present inventors have found it unexpectedly advantageous to combine cross-flow filtration (feed flow is parallel to a membrane surface) and dead-end filtration (feed flow is perpendicular to the membrane surface). Generally, see Forbes (1987). Optionally, this combination can be preceded by a differential centrifugation, at low centrifugal force, to remove some portion of the bacterial cells and thereby enrich the supernatant for minicells. Also optionally, the combination can be followed by an antibiotic treatment to kill residual parent bacterial cells.<br>
20<br><br>
Cross-flow filtration, depending on the filter pore size, can separate minicells from larger contaminants such as parent bacterial cells, and from smaller contaminants such as bacterial blebs, free endotoxin, nucleic acids cellular debris and excess liquid. To separate minicells from larger contaminants, the nominal pore size of cross-flow filters should allow minicells to permeate through the filters, but not large bacterial cells. A 0.45 µm pore size is preferred for this purpose because minicells are approximately 0.4 µm in diameter, whilst bacterial cells are larger. To separate minicells from smaller contaminants, the nominal pore size of cross-flow filters should allow smaller contaminants to permeate through the filters, but not minicells. A 0.2 µm pore size is preferred for this purpose because bacterial blebs range in diameter from 0.05 µm to 0.2 µm, and the other smaller contaminants are less than 0.2 µm<br>
Effective application of cross-flow filtration in this context typically entails at least one step involving a larger pore size, around 0.45 µm ,followed by at least one step with a smaller pore size, around 0.2 µm. Between or during serial cross-flow filtration steps, diafiltration may be performed to maximize recovery of minicells.<br>
The use of cross-flow filtration accommodates suspensions carrying heavy loads of particulate matter, such as bacterial cultures, which may carry loads of 1012 to 1013 bacterial and minicell populations per liter of culture. To rninimize filter fouling and the consequent loss of minicells, the bacterial/minicell culture may be diluted, preferably S-fold to 10-fold. Dilutions also permit use of appropriately low atmospheric pressure and flow rate.<br>
To remove residual parent bacterial cells remaining after cross-flow filtration, dead-end filtration is performed. For this purpose, the use of at least one dead-end filtration, employing a pore size of about 0.45 µm, is preferred.<br>
Generally, filtration provides a sterile preparation of minicells suitable for gene transfer studies. For in vivo gene transfer, an antibiotic treatment is preferably performed further to reduce the risks from bacterial cell contamination.    For<br>
21<br><br>
instance, minicells may be resuspended in growth medium that contains an antibiotic to which the parent bacterial strain is sensitive. The appropriate amount of a given antibiotic for this purpose can be determined in advance by conventional techniques.<br>
A serial cross-flow filtration/dead-end filtration arrangement, as described, not only dispenses with the use of gradient-formation agents but also provides for a purity that exceeds 10-7 (i.e., fewer than one parent cell per 107 minicells). Preferably, the purity exceeds 10-8 and, more preferably, is on the order of about 108. The serial cross-flow filtration/dead-end filtration arrangement also provides for better quality control. In addition, there is no need for an ampicillin-, cycloserine-, or other antibiotic-resistance gene, as required by the technique of Clarke-Curtiss and Curtiss (1983). Furthermore, the serial purification approach employs no DNA-damaging radiation, in contrast to a methodology described Sancar et al. (1979); hence, a therapeutic nucleic acid molecule delivered with minicells prepared via a serial cross-flow filtration/dead-end filtration arrangement can be free of radiation-induced, non-specific mutation.<br>
A composition consisting essentially of recombinant minicells of the present invention (that is, a composition that includes such minicells with other constituents that do not interfere unduly with the DNA-delivering quality of the composition) can be formulated in conventional manner, using one or more physiologically acceptable carriers or excipients. Formulations for injection may be presented in unit dosage form, e.g., in ampules or vials, or in multi-dose containers, with or without an added preservative. The formulation can be a solution, a suspension, or an emulsion in oily or aqueous vehicles, and may contain formulatory agents, such as suspending, stabilizing and/or dispersing agents. A suitable solution is isotonic with the blood of the recipient and is illustrated by saline. Ringer's solution, and dextrose solution. Alternatively, minicells may be in fyophitized powder form, for reconstitution with a suitable vehicle, e.g., sterile, pyrogen-free water or physiological saline. Minicells also may be formulated as a depot preparation. Such long-acting formulations may be<br>
22<br><br>
administered by implantation (for example, subcutaneously or intramuscularly) or by intramuscular injection.<br>
A minicell-containing composition of the present invention can be administered via various routes and to various sites in a mammalian body, to achieve the therapeutic effects) desired, either locally or systemically. Delivery may be accomplished, for example, by oral administration, by application of the formulation to a body cavity, by inhalation or insufflation, or by parenteral, intramuscular, intravenous, intraportal, intrahepatic, peritoneal, subcutaneous, intratumoral, or intradermal administration.<br>
The nature of the application contemplated likewise will influence (or be influenced by) the choice of bacterial source for the recombinant minicells employed to deliver a therapeutic nucleic acid molecule. For example, Salmonella, Escherichia and Shigella species carry adhesins that are recognized by endbcytosis-mediating receptors on enterocytes in the gastrointestinal tract may be suitable for oral administration, to deliver a therapeutic nucleic acid molecule that is effective for colon cancer cells. Similarly, minicells derived from Helicobacter pylori, carrying adhesins specific for stomach epithelial cells, could be suited for oral delivery aimed at stomach cancer cells. Inhalation or insufflation may be ideal for administering intact minicells derived from a Pseudomonas species that carry adhesins recognized by receptors on lung epithelial cells; this, for delivery, to the lungs of a cystic fibrosis patient, of a therapeutic nucleic acid molecule encoding CFTR protein, for example. Minicells derived from Lactobacilli bacteria, which carry adhesins specific for urinary tract and bladder epithelial cells, could be well-suited for intraurethral delivery of a therapeutic nucleic acid molecule to a urinary tract and or a bladder cancer.<br>
The present invention can be used to deliver a range of nucleic acid molecules, which can be cDNA as well as genomic DNA or RNA, and can be in die sense or die anti-sense orientation.  The nucleic acid molecule present in an intact<br>
25<br><br>
minicell, pursuant to the present invention, can take the form of a plasmid, expression vector, or other genetic construct, but is not genomic DNA originating from the bacterial cell that gave rise to the minicell. Suitable for use in die present invention is any desired DNA or RNA sequence from a eukaryotic, prokaryotic, or synthetic source which may be placed under the translational and transcriptional control of a eukaryotic gene expression promoter, or which may be expressed in the mammalian cell using trans-activating factors from the host cell.<br>
Example 1. Generation of bacterial minicells from Gram-negative bacteria, Salmonella typhimurium, Escherichia colt and Shigella flexneri<br>
Minicell-producing bacterial strains from Gram-negative bacteria were generated, as described here (A, B and C) and illustrated in Figures 2 and 3.<br>
General Materials and Methods<br>
The bacterial strains used in the instances below are listed and referenced in Table 1. All bacteria were grown from glycerol stocks maintained at -80°C. Salmonella, E. coli, Shigella and listeria strains were grown in Trypticase Soy Broth (TSB) (BBL brand purchased from Bacto Labs, Liverpool, NSW, Australia). It was prepared according to the manufacturer's instructions at 30 gm/1, and autoclaved at 121°C for IS minutes. Liquid culture was grown in a shaking incubator at 37°C. Shigella strains were differentiated from E. coli by plating on XLD agar (Xylose-Lysine-Desoxycholate Agar) plates to result in red and yellow colonies respectively. XLD was purchased from Oxoid (Melbourne, Australia). It was prepared according to the manufacturer's instructions at S3 gm/1, then boiled for 2 minutes. Antibiotics were used at the following concentrations in liquid and solid media: ampicillin, 100 µg/ml, chloramphenicol, 30 µg/ml, Kanamycin, 30 µg/ml.<br>
24<br><br>
E.coU strain	Genotype / relevant characteristics	Reference<br>
ENIhOOl	SMIODpir: F--supE44, thi-l, thr-1, leuB6, lacYl, tonA21, recA:RP4-2-Tc::Mu lambdapir, TnphoA, oriR6K, tra- mob+	Miller and Mekalanos (1988).<br>
ENE105	Strain ENIhOOl carrying plasmid pEN06O (Fig. 3)	The present disclosure<br>
ENIh003	JM109: P traD36 DroA+ pn&gt;B+ laclq lacZDMlS/recAl endkl gyrA96 (Nal) thi hsdR.17 supE44 re1A1 D(lac-proAB) mcrA	Yanisch-Perron et al. (1985)<br>
Salmonella strain		<br>
ENIhOO7	Salmonella enterica serovar Typhimurium. Clinical isolate from sheep.	Institute of Medical and Veterinary. Services, Adelaide, SA, Australia. Reference strain J98/00413<br>
ENSm083	Salmonella choleraesuis subsp. choleraesuis (Smith) Weldin serotype Typhimuiium deposited as Salmonella typhimurium	ATCC 14028<br>
SL3261	Salmonella typhimurium arok-	Hoiseth era/., (1981)<br>
SL5283	SalmoneUa typhimurium hsdRr, hs<m hoisethera></m>
Shigella strain		<br>
ENSfOOl	S. flexneri serotype 2b	ATCC 12022<br>
Listeria strain		<br>
ENLMOOl	Listeria monocytogenes Gibson ATCC 7644	J. Pathol. Bacterid. 45: 523, (1937)<br>
Where required, strains were grown in M9 minimal medium supplemented with 1% glucose. Additional supplements were added depending on the strain. For E. coli strain ENIh003, l mM Thiamine was added. For S. flexneri strain ENSfOOl, M9 was supplemented with 0.4 mM Serine, 0.2 mM Proline, 0.001% Phenylalanine, 0.001% Tryptophan, 0.001% Tyrosine, 0.001% Histidine, 0.002 mg/ml Valine, 0.012S mg/ml Nicotinic acid, 0.001% Thiamine, and 0.0225 mg/ml Methionine.<br>
Plasmid DNA was purified using the Qiaprep Spin Miniprep Kit (Qiagen Inc., Victoria, Australia).   Genomic DNA for Salmonella, Shigella and E. coli<br>
25<br><br>
strains was prepared using the basic protocol from Current Protocols in Molecular Biology (Chapter 2, Section I, Unit 2.4; John Wiley &amp; Sons, Inc.). All restriction and modification enzymes used were purchased from Promega (Madison, WI, USA) or Roche (Castle Hill, NSW, Australia), except for Deep Vent DNA Polymerase, which was purchased from New England Biolabs (Beverly, MA, USA).<br>
Genomic DNA of L. monocytogenes (ENLM001) was purified by conventional methods (Ausubel et. al., Current Protocols in Molecular Biology, John Wiley and Sons Inc.) with modifications as described. A l.5 ml sample of an overnight culture was centrifuged at 13,200 rpm for 3 minutes and the supernatant was discarded. The bacterial cell pellet was resuspended in 1 ml of TE buffer (lOmM Tris pH 8.0; 1 mM EDTA pH 8.0) with 2.5 mg/ml lysozyme, and incubated at 37°C for 1 hour. RNAse A was then added to a final concentration of IS mg/ml, and the solution was incubated at room temperature for 1 hour. Then, 100 Mg/ml Proteinase K and 0.5% SDS were added and the mixture was further incubated for an hour at 37°C. Subsequently, 200 µi of SM NaCl was mixed thoroughly into the solution, followed by 160 µi of CTAB/NaCl solution (10% CTAB in 0.7 M NaCl). This mixture was then incubated for 10 minutes at 65°C. The sample was extracted with an equal volume of chloroform/isoamyl alcohol followed by an extraction with an equal volume of phenol/chloroform/isoamyl alcohol. Genomic DNA was precipitated from solution with 0.6 volume of isopropanol and 1/10th  volume of SM sodium acetate. The DNA pellet was washed with ethanol and air dried before resuspension in TE buffer.<br>
PCR primers were synthesized and purchased from Sigma-Genosys (Castle Hill, NSW, Australia). The basic PCR protocol followed was as follows. Reaction components fbr all 50 µi PCR included IX buffer, 200 \M dNTPs, SO pmol each primer, 1 Unit of Deep Vent DNA polymerase, 50 pmol of genomic DNA template (25 pmol for plasmids), nuclease-free water to 50 µl, with PCR performed in 0.2 ml tubes in a Gradient PCR Express Thennalcycler from Thermo Hybaid (Ashford, Middlesex, UK).   PCR conditions to obtain the minCDE gene cluster were as<br>
26<br><br>
follows: 94°C for 4 minutes; followed by 30 cycles at 94°C for 35 seconds, 60°C for 30 seconds, 72°C for 2.5 minutes; followed by a final cycle at 94°C for 35 seconds, 60°C for 35 seconds, 72°C for 5 minutes. PCR conditions to obtain the ΔminCDE cassette were as follows: 94°C for 4 minutes; followed by 30 cycles at 94°C for 35 seconds, 60°C for 30 seconds, 72°C for 2 minutes; followed by a final cycle at 94°C for 35 seconds, 60°C for 35 seconds, 72°C for 4 minutes. PCR conditions to obtain the ΔminCDE::Cml cassette were as follows: 94°C for 4 minutes; followed by 30 cycles at 94°C for 35 seconds, 60°C for 30 seconds, 72°C for 3 minutes; followed by a final cycle at 94°C for 35 seconds, 60°C for 35 seconds, 72°C for 6 minutes.<br>
The standard molecular biology protocols followed were as described in Sambrook et al. (1989) and in CURRENT PROTOCOLS IN MOLECULAR BIOLOGY (John Wiley &amp; Sons, Inc., NJ, USA).<br>
The minicell yield was determined microscopically, using a Leica Model DMLB light microscope with image analysis by means of a Leica DC camera and Leica IM image management software. Samples were viewed using Darkfield microscopy at 40X or oil immersion at 100X magnification. Coverslips on glass slides were sealed with 1.5% agarose. Purity of each batch of minicells was determined by plating 10% of the volume on Trypticase Soy Agar plates and incubation overnight at 37°C. The method routinely provided a very high purity with one contaminating cell in 109 minicells. The minicell suspensions were resuspended in TSB with the appropriate antibiotics to which the parent bacteria were determined to be sensitive and the cultures were incubated with shaking at 37°C for 4 hours, to kill all residual parent bacteria. For example, the S. typhimurium minCDE- strain was determined to be sensitive to Ampicillin and hence the minicell suspension was incubated in TSB containing 50 /ig/ml of Ampicillin for 4 hours, to ensure that if there were any residual bacteria, then they would be<br>
25<br><br>
killed. The minicells then were collected by centrifugation at 10,000g for 30 minutes, washed four times in f x BSG buffer (to eliminate the growth medium), and resuspended in the desired volume for down-stream experiments.<br>
[A] Generation of minicell producing strains from two different strains 5     of Salmonella typhimurhun.<br>
A schematic diagram of plasmid construction is shown in Figure 2. Bacterial strains, plasmids and PCR primers used are shown in Tables 1, 2 and 3 respectively.<br>
The E coli minCDE gene sequences, see Mori (1996), were used to search the Genbank database for homologous DNA sequences using the method of FASTA analysis (Pearson and Lipman, 1988). The TYPN3 contig 101 DNA sequence from we S. typhi genome (CT18) was found to be homologous to the E. coli minCDE sequences. The oligonucleotide primers ENOL001 and ENOL002 were designed on the basis of these data and were used to prime the synthesis of the intact minCDE genes from S. typhimurium strain ENIh008 as an EcoRl-minCVEHindWl fragment.<br>
This fragment was cloned into the EcoBIIHindSH sites of pNEB193 to create a plasmid designated as pENOOl, which was propagated in E. coli strain JM109 to result in strain ENE001. Primers ENOL003 and ENOL004 were designed to delete a total of 1081 bp of sequence from the minCDE cassette in pENOOl, while ; simultaneously inserting 16 base pairs (bp) containing the unique Kpnl, Smal and Xbal restriction sites as locations for future insertion of one or more marker genes.<br>
The deleted sequence included 386 base pairs from the 3’end of the minC gene, the 23-base pair intervening sequence upstream of the miriD gene and 672 base pairs from the 5' end of the miriD gene. This resulted in the AminCDE deletion cassette (755 base pairs) in plasmid pEN002 harbored in strain ENE003. The chloramphenicol resistance gene from pHSG41S (Fig. 2, Table 2), 1330-base pair Haell fragment/blunt-ended) was cloned into the Smal site of pEN002 to obtain<br>
28<br><br>
plasmid pEN003, carrying the 6minCDE::Cml2 deletion cassette with the Cml2 gene cloned in the clockwise orientation. This was designated strain ENE006.<br>
Table 2.        Plasmids used in this study.<br><br>
Plasmid	Relevant characteristics	Reference<br>
pHSG415	Low-copy number, temperature sensitive, mobilisable	Hashimotoh-Gotohera/., (1981)<br>
DGP704	Derivative of pBR322 where die oriEl has been replaced	Miller and<br>
	with oriR6K. The R6K origin of replication requires for	Mekalanos<br>
	its function a protein called pi, encoded by die pir gene. In E. coli hie pi protein can be supplied in treats by a prophage (tjp/r) that carries a cloned copy of die pir gene. The plasmid also contains a 1.9-kb BamHI fragment encoding die mob region of RP4. Thus, pGP704 can be mobilized into recipient strains by transfer functions provided by a derivative of RP4 integrated in the chromosome of E. coli strain SM10. However, once transferred it is unable to replicate in recipients mat lack die pi protein. Amp*	(1988)<br>
pNEB193	It is a pUC19 derivative mat carries single sites for	Purchased from<br>
	unique 8-base cutters: Asc I, Pac I and Pme I. The	New England<br>
	polylinker carries me unique restriction sites JScoRI, Sad,	Biolabs, Inc.<br>
	Kpnl, Smal, Ascl, BssHU, BanOU, Pad, Xba\, Sail,	Beverly, MA,<br>
	Pmel, Sbfl, Psil, Sphl, HindKL. The polylinker does not interrupt die tocZ reading frame. AmpR	USA<br>
pMK4	E. coli. Staphylococcus aureus, Bacillus subtilis shuttle	Sullivan et al.,<br>
(S.6 kb)	vector, lacZ' (from pUC9), Cml2 (from pC194) in Bacillus, Amp" in E. coli. Cml2 in Bacillus	(1984)<br>
pVA838	E. coli and Streptococcus sanguis shutde vector,	Macrina et. al..<br>
(9.2 kb)	Erythromycin resistant (Em2) in 5. sanguis, Cml* in E. coli.	(1982)<br>
pRB373	E. coli and B. subtilis shutde vector, bla (Amp2) and E.	Bruckner (1992)<br>
(5.8 kb)	coli on were derived from pBR322. Kanamycin resistance (Km*), Belomycin resistance (Bm2) and Gram-positive ori were derived from pUBHO. to (transcription terminator) is from phage λ.	<br><br>
Ptasmid	Relevant characteristics	Reference<br>
pEGFP	Carries a red-shifted variant of wild-type green	Purchased from Clontech Laboratories, Palo Alto, CA, USA.<br>
	fluorescent protein (GFP) which has been optimized for	<br>
	brighter fluorescence and higher expression in	<br>
	mammalian cells. (Excitation maximum - 488 nm;	<br>
	emission maximum = 507 nm.) Upstream sequences	<br>
-	flanking EGFP have been converted to a Kozak consensus translation initiation site to further increase die translation efficiency in eukaryotic cells. The EGFP gene was cloned between the two MCS of die pUC19 derivative pPD16.43. The EGFP gene was inserted in frame with the lacZ initiation codon from pUC19 so mat a EGFP fusion protein is expressed from the lac promoter in E. coli. The pUC backbone of EGFP provides a high-copy-number origin of replication and an ampicillin resistance gene for propagation and selection in E. coli.	<br>
pEGFP-Cl	EGFP is as described for plasmid pEGFP. Sequences	Purchased fromClontechLaboratories,<br>
	flanking EGFP have been converted to a Kozak consensus	<br>
	translation initiation site to farther increase the translation	<br>
	efficiency in eukaryotic cells. The EGFP gene is	Palo Alto, CA, USA.<br>
	expressed from the human cytomegalovirus immediate	<br>
	early promoter and hence the plasmid only expresses EGFP in mammalian calls and not in bacterial cells. The MCS in pEGFP-Cl is between the EGFP coding sequences and the SV40 poly A (downstream of the EGFP gene) which directs proper processing of the 3' end of the EGFP mRNA. The vector backbone also contains an SV40 origin for replication in mammalian cells expressing die SV40 T-antigen. A neomycin resistance cassette (neo’), consisting of the SV40 early promoter, die neomycin/kanamycin resistance gene of TnS, and polyadenylation signals from die Herpes simplex thymidine kinase (HSV TK) gene, allows stably transfected eukaryotic cells to be selected using G418. A bacterial promoter upstream of this cassette expresses kanamycin resistance in E. coli. The pEGFP-Cl backbone also provides a pUC origin of replication for propagation in E. coli and an fl origin for single-stranded DNA production.	<br>
The ΔminCDE::Cml2 deletion cassette was amplified from plasmid pEN003 by polymerase chain reaction (PCR) using primers ENOL001 and ENOL002, blunt-ended and cloned into the SmaI site of suicide plasmid pGP704 (Fig. 2; Table 2). The plasmid, designated pENOOS, was transformed into strain ENIh00l (SMIOλpir;<br>
30<br><br>
Table 1) to give strain number ENE007. Strain ENE007 was used as the donor strain in a conjugation (filter mating) experiment, with S. typhimurium strains ENIh007 and ENSm083 (Table 1) as the recipient. Overnight, static cultures of die donor and recipient were grown in TSB at 37°C. Cultures were mixed in a filter mating conjugation on Hybond N+ membranes on TSA plates at a dononrecipient ratio of 1:3 and incubated at 37°C for 8 hours. The cells were recovered and washed twice in a sterile saline solution. The cell pellet was resuspended in saline and plated on ISO mm Petri plates. The plates were incubated for up to 72 hours at 37°C.<br>
The ex-conjugants were selected on M9 minimal medium wim 1.5 % glucose and 30 µg/ml chloramphenicol. The donor strain is counter-selected under mese conditions, due to extra auxotrophic rerniirements, whilst the recipient strain cannot grow, due to its chloramphenicol sensitivity. Therefore this experiment selected for S. typhimurium ex-conjugants carrying the plasmid-encoded chloramphenicol resistance. The colonies were screened for the desired phenotype of CmlR and Amps by patching isolates on to M9 minimal medium containing ampicillin (Amp) or chloramphenicol (Cml). From 79 isolates of the ENE007 x ENIhO07 conjugation that exhibited CmlR, a total of 18 isolates were found to be Amps. Similarly, from 56 isolates of the ENE007 x ENSm083 conjugation, 19 were found to be Amps.<br>
To determine whether die isolates were chromosomally deleted for the minCD genes, overnight cultures were visualized via darkfield microscopy at 40X magnification. Minicells were visualized in the mixed culture for all 27 isolates. All isolates showed the presence of minicells while the parent control strains were absent for minicells. Purified minicells were tested for agglutination wim 4-0 Salmonella Somatic Agglutinating Serum (rabbit) ZC13 (Murex Diagnostics, Norcross, Georgia USA).<br>
The recombinant bacterial strain was grown under laboratory conditions that are optimal for production of recombinant minicells.     Bacterial growth is<br>
31<br><br>
accomplished using standard bacteriological media, such as those described in Sambrook et al. (1989), and using optimal growth conditions, which can be readily determined by conventional technique.<br>
[B] Generation of minicell producing strain from Shigella Jlexneri.<br>
Figure 3 depicts the relevant steps in the genetic construction of a minicell-producing strain from S. Jlexneri serotype 2b. The cloning protocol was similar to mat followed for construction of nunicell-producing S. typhimurium strains, detailed above.<br>
To clone the minCDE gene cluster from S. jlexneri serotype 2b (Table 1), PCR cloning primers were designed based on a database search of the in-progress sequencing project for the complete Shigella flexneri, serotype 2a genome sequence. PCR primers ENOL059 and ENOL060 carrying EcoRl and HindBii tails respectively (Table 3) were used to amplify the 1808 bp minCDE gene cluster from genomic DNA purified from Shigella flexneri, serotype 2b. The amplified fragment was cloned into the EcoRl and HindSL sites of plasmid pNEB193 (Table 2), resulting in plasmid pENOSS. The insert DNA was sequenced and confirmed to be the minCDE DNA from the parent Shigella bacterium.<br>
Table 3.        Oligonucleotides used in this study. Restriction enzyme sites and DNA sequence characteristics are shown in brackets preceding the respective DNA segment. (F) and (R) represent forward and reverse PCR primers respectively.<br><br>
Oligonucleotide	Sequence	Product<br>
ENOL001	5' (roil) CTC TCA CTG (EcoJO) GAA TTC (minC ATG TCA AAC ACG CCA ATC GAG C 3'	(F)S.typhimuriummmCDE<br>
ENOL002	5' (Toil) CTC CTG GCA (HindiIII) AAG CTT (ninE) TTA TTT TGA CTC TTC GGC TTC CG 3'	(R)S.typhimuriumminCDE<br>
ENOL003	5’ (Tafl) CTC TGC TAG TCA {Smal-Kpnl) CCC GGG TAC C (minQ GCC GAA CCG CTT TCT CTT TAC C 3'	(R)S.typhimurium deletion of minCDE to get ΔmtnCDE<br><br>
Oligonucleotide	Sequence	Product<br>
ENOL004	5' (Tail)CTCTGCTAG-TCA(JSHMI)CCC GGG (»al)TCT AGA (rninD) GAA CCG GTG ATT CTT GAC GCC A 3’	S.typhimurwm deletion of minCDE to get ΔmfaCDE<br>
ENOL059	5' (rofl) CTC TCA CT (EcoRl) GAA TTC (mi»C) ATG TCA AAC ACT CCA ATC GAG CT 3'	(F) S.flexneri 2b minCDE cloning<br>
ENOL060	5' [Tail) CTC CTG GC (HindfH) AAG CTT (miuE, includes last 2bp from HindOY) ATT TCA GCT CTT CTG CTT CCG 3*	(R) S.flexneri 2b minCDE cloning<br>
ENOL062	5' (r«Q CTC TCA TAA (S«aD CCC GGG (Xk»D TCT AGA (miRD) GGC GTG ATC CCA GAG GAT CAA T 3'	(F) S.flexneri 2b deletion of minCDE to get Am/nCDE<br>
ENOL063	5* (Tail) CTC TCA TTC (Smal-Kpnl) CCC GGG TAC C (mfaC) TGT GGA GCA TAA ATA CGC TGA CC '3	(R) S.flexneri 2b deletion of minCDE to get AmfaCDE<br>
ENOL038	5* (TaU) CTC CAG TCT (HindW) AAG CTT (mmD) AGG AGC CGC GCT TAC TAT TAG C 3'	(R) ListeriamonocytogenesIIWICD<br>
ENOL048	5' (Toft CTC CAG TCT (Sod) GAG CTC (mi»Q GAA GAA GAA TGT TCA AAT TAA AGG C 3*	(3s) ListeriamonocytogenesminCD<br>
ENOL039	5’ {Tam CTC CAG TCT (BamBJ) GGA TCC (XbaJ) TCT AGA (minO ATC CCC TGG AAC CTG AAC AAC 3’	(R) Listeria monocytogenes deletion of minCD to get ΔmihCD<br>
ENOL040	5' (To® CTC CAG TCT {BamHI) GGA TCC {Kpnl) GGT ACC (minD) CCG GAA ATA TCA GCA GTT CG 3’	(F) Listeria monocytogenes deletion of minCDtoget Δmincd<br>
ENOL098	5' {Tail} CTC CAG TCT {Xball TCT AGA (Cm2) TTT TTG CGC TTA AAA CCA GTC AT 3'	(R) obtain Cm' frompMK4<br>
ENOL099	5' (r«fl) CTC CAG TCT (JjwD GGT ACC (Cm2) AAA ACC TTC TTC AAC TAA CGG GG 3’	(F) obtain Cm' frompMK4<br>
ENOL096	5' (Tail) CTC CAG TCT (Xbal) TCT AGA (Em’) GAG ATA AGA CGG TTC GTG TTC GT 3’	(R) obtain Emr frompVA838<br>
ENOL097	5' (Tail) CTC CAG TCT (Kbalwl) GGT ACC {Em’) AGA ATG CAG AAG ATG AAA GCT GG 3'	OF) obtain Emr from pVA838<br>
ENOL092	5* (TaO) CTC CAG TCT (EcoRI) GAA TTC (Kan1) TGA AGG ATG CTT AGG AAG ACG AG 3'	(F) obtain Kan' frompRB373<br>
ENOL093	5’  (Toty CTC CAG TCT (EcoTU) GAA TTC (Kan*) CGC CAT GAC AGC CAT GAT AA 3'	(R) obtain Kan' frompRB373<br>
ENOL094	5’ {Tail) CTC CAG CTC (EcoRI) GAA TTC (Kan'and G+ on) AAG GTG CGT TGA AGT GTT GGT AT 3’	(F) obtain Kan' and G+ve ori frompRB373<br><br>
A deletion in the minCDE gene cluster was obtained using PCR primers ENOL062 and ENOL063 (Table 3) and plasmid pENOSS as the template DNA in a reverse PCR reaction. This resulted in the deletion of 271 bp (3' terminal region of mj'nC), 23 bp (inter-geneic sequence between minC and minD), and 608 bp (5' terminal region of minD). Simultaneously, the PCR primer sequences inserted a multiple cloning site carrying Kpnl-Smal-Xbal restriction sites. The PCR product was digested with Smal and religated to create plasmid pEN056.<br>
The CmlR marker was gel-purified from plasmid pHSG415 (Table 2) as a 1330 bp HaeU fragment, blunt-ended with T4 DNA polymerase and cloned into the Smal site of plasmid pEN056. The new ΔminCDE::Cml/R plasmid was designated pEN057.<br>
The ΔminCDE::Cml/R cassette was obtained by PCR using primers ENOLO59 and ENOL060 (Table 3) and using plasmid pEN057 as the template. The 2,272 bp fragment was blunt-end cloned into the EcoRV site of suicide plasmid pGP704 (Table 2). The new suicide plasmid was designated pEN060 and the SMlOλpir strain carrying it was designated ENE105.<br>
A filter-mating conjugation was used to transfer plasmid pEN060 from ENE10S to Shigella flexneri strain ENSfOOl (Table 1) with a recipienf:donor ratio of 5:1, overnight cultures (static growth) of the recipient and donor were mixed on Hybond N+ membranes on TSA plates at the calculated ratio and incubated overnight at 37°C. The cells were recovered and washed twice prior to plating out on selective minimal media containing supplements and 30 µg/ml chloramphenicol. The donor and recipient cannot grow on this media due to either auxotrophic requirements (donor) or antibiotic sensitivity (recipient) so that any colonies found should be exconjugants. Thirty-two (32) isolates were picked and patched onto fresh minimal media plates containing supplements and 30 µg/ml chloramphenicol.<br>
34<br><br>
The 32 exconjugants were incubated at 37°C overnight before being patched onto fresh minimal media plates' containing supplements and 100 µg/ml ampicillin and then incubated at 37°C for 24-48 hours. All 32 isolates were able to grow on ampicillin indicating integration of the whole plasmid. Each isolate was examined under darkfield microscopy (40X) or under oil immersion (100X). Three of the isolates revealed the presence of minicells. The isolates were streaked onto XLD plates containing 30 µg/ml chloramphenicol to confirm that the isolate was Shigella and not the E. coli donor. The isolates also showed a strong positive agglutination reaction when tested with Shigella flexneri agglutinating sera (BIODESIGN International, Saco, Maine, USA). Plasmid purification and gel-electrophoretic analysis confirmed that the donor plasmid was not present in the exconjugants as an episome.<br>
[C] Generation of miniceil producing strain from Escherichia coli.<br>
Given that there is a 98% genetic homology between E. coli and Shigella genomes this study aimed to determine if heterologous ΔminCDE gene sequences can be used to generate miniceil producing strains especially where genome sequences are closely related. Therefore a filter-mating conjugation was used to transfer plasmid pEN060 (carries ΔminCDE::cml2 from S. flexneri 2a; Fig. 3) from ENE10S (Table 1) to E. coli strain ENIh003 (JM109; Table 1) with a recipient:donor ratio of 3:1. Overnight cultures (static growth) of the recipient and donor were mixed on Hybond N+ membranes on TSA plates at the calculated ratio and incubated overnight at 37°C. The cells were recovered and washed twice prior to plating out on selective minimal media containing supplements and 30 µg/ml chloramphenicol. The donor and recipient cannot grow on this media due to either auxotrophic requirements (donor) or antibiotic sensitivity (recipient). 106 isolates were picked and patched onto fresh minimal media plates containing supplements and 30 µg/ml chloramphenicol. After overnight incubation at 37°C the isolates were patched onto fresh minimal media plates containing supplements and 100 µg/ml <br>
ampicillin and incubated at 37°C for 24-48 hours. All 106 isolates were able to grow on ampicillin suggesting integration of the whole plasmid. Twenty-four (24) isolates were viewed under darkfield microscopy (40X) and 100X oil immersion. Six isolates showed the presence of large numbers of minicells. Although many different general mutagenesis and site-directed mutagenesis protocols have been employed in the past to generate minicell producing bacterial strains, tins invention is the first to demonstrate that the unique cloning/site-directed mutagenesis protocol employed in this study is versatile and can be reliably used to generate minicell producing strains from a range of Gram-negative and Gram-positive bacteria (example 2 shown below).<br>
Example 2. Generation of mincells from Listeria monocytogenes<br>
Minicell-producing bacterial strains from Gram-positive bacteria can be generated as described in this example. A schematic diagram of plasmid construction is shown in Figure 4. The bacterial strains, plasmids and PCR primers are respectively listed in Table 1, Table 2, and Table 3.<br>
To clone the minCD genes from the genome of L. monocytogenes, PCR was performed using primers ENOL038 and ENOL048 (Table 3) and purified L. monocytogenes genomic DNA as template. PCR reactions were carried out in 50 µd volumes using the Platinum• Pfx DNA Polymerase kit (Invitrogen Corporation, Hi Carlsbad, CA, USA). Reactions included IX Pfx buffer, 2 mM MgSO4, 0.2 mM dATP, dTTP, dGTP and dCTP, 50 pmol of each primer and 1U of Pfx polymerase. Cycling conditions included a 94°C denaturing step for two minutes; followed by 35 cycles of 94°C for 30 seconds, 55°C for 30 seconds and 68 °C for two minutes; followed by a 68°C final extension step for five minutes. A 2-5 µi sample of each PCR reaction mixture was visualised on a 1 % agarose gel stained with ethidium bromide. The amplified minCD fragment was purified by excision from the agarose gel, followed by electro-elution using the Model 422 Electro-Eluter (Bio-Rad Laboratories, Hercules, CA, USA) according to the manufacturer's specifications.<br>
36<br><br>
The minCD fragment (1,556 bp) was digested with Sacl and HindIII, and directionally cloned into the respective sites of plasmid pNEB193 (Table 2; Fig. 4). Recombinant clones transformed in strain ENIh003 (Table 1) were characterized by restriction digestion and gel-electrophoretic analysis. The correct clone (plasmid designated pEN045) was sequenced to confirm the identity of minCD genes.<br>
PCR deletion was performed on the cloned minCD fragment in plasmid pEN045 using primers ENOL039 and ENOL040 (Table 3). The primers carry restriction sites Xbal and Kpnl respectively, which serve as insertion sites for selection markers between bminC and AffnfnD. Reaction conditions for deletion of minCD, and fragment visualization, were as described above, but with a 4 minute 68°C extension step. The ΔminCD fragment was subsequently digested with Xbal and Kpnl, and purified by elcctro-elution with the Model 422 Electro-Eluter as preparation for ligation with selection markers.<br>
Two different antibiotic selection markers from Gram-positive bacteria were inserted between the ΔminC and ΔminD genes in pEN045 at the Kpnl and Xbal<br>
sites. The chloramphenicol resistance marker (CmlR) from the Bacillus subtilis plasmid pMK4 (Table 2), and the erythromycin resistance marker (EmR) from die Streptomyces sanguis plasmid pVA838 (Table 2) were used to construct plasmids pEN062 and pEN063 respectively (Fig. 4). Each marker was amplified by PCR using the oligonucleotides listed in Table 2 and shown in Fig. 4, which include Xbal and Kpnl sites. PCR of CmR and EmR was carried out using the same reaction volume and reagent concentrations as described above. Cycling conditions included a 94°C denaturation step for two minutes; followed by 35 cycles of 94°C for 30 seconds, 55°C for one minute and 68°C for two minutes; followed by a 68°C extension step for five minutes. PCR fragments were gel purified using the MinElute kit (Qiagen Inc, Victoria, Australia) according to the manufacturer's instructions. CmR and EmRwere men digested with Kpnl and Xbal, purified with a MinElute column and ligated to the respective sites between the ΔminC and ΔminD<br>
37<br><br>
genes in pEN045. JM109 recombinants were characterized by restriction digestion with the enzymes EcoBl and Ddel (CmR) and EcoRI and Aval (EmR) to give strain isolates ENE110 and ENE112 respectively.<br>
For the purpose of conjugation experiments between L. monocytogenes and E. cott SM10λ. pir (ENIhOOl), four new plasmids were constructed, using the conjugative plasmid pGP704. The plasmid pGP704 was modified to include either the kanamycin resistance marker (KmR), encoded by the neo gene, or, KmR as well as the ori for plasmid replication in Gram-positive bacteria. The former would act as a suicide plasmid in the absence of a Gram-positive origin of replication, with the neo gene (KmR) serving as a marker for plasmid presence. In contrast, the latter would act as a positive control for conjugation experiments.<br>
DNA segments containing the neo gene and the neo gene including Gra-o-positive ori were amplified by PCR from pRB373 using the set of oligonucleotides listed in Table 3. PCR reaction volumes and reagent concentrations were as described above. Cycling conditions included a denaturation step of 2 minutes at 94 °C; followed by 35 cycles of 94°C for 30 seconds, 56°C for one minute, and 68°C for 2 and a half minutes; followed by a long extension step of 68°C for S minutes. Products were purified from an agarose gel using the MinHute column. The fragments containing the neo gene and neo gene plus the Gram-positive ori were cut with Ecdtil and ligated into the EcoRI site of pGP704 (Fig. 4). This resulted in plasmids designated pEN064 and pEN066.<br>
The ΔminCDiiCmf- or ΔminCD::EmR were then excised as Sacl - HindUl fragments from pEN062 and pEN063 respectively, blunt ended and ligated into the EcoRV site of plasmids pEN064 and pEN066, to give the four different plasmids pEN069, pEN071, pEN073 and pEN075 as depicted in Fig. 4. The four plasmids were transformed into ENIhOOl (E. cott SM10λ pir) to create the strains ENE124, ENE126, ENE128 and ENE130 respectively. Thus ENE124 contained the plasmid with the AmmCD::CmR and the neo gene (KmR), while ENE126 has, in addition,<br>
38<br><br>
the Gram-positive ori. Similarly, ENE128 contained the plasmid with the ΔminCD::EmR and the neo gene (KmR), while ENE130 additionally contained the Gram-positive ori.<br>
The four strains ENE124, ENE126, ENE128 and ENE130 were used as donor strains for conjugation experiments with L. monocytogenes (ENLM001). Conjugations were carried out essentially as described in Trieu-Cuot et al. (1991). All strains were grown to mid-log phase and mixed in a 2:1 ratio (recipient-.donor) to a 1 ml volume. Conjugation mixes were washed twice in BHI before resuspending in 1 ml BHI. 200 pi volumes were plated on 0.45 pM nitrocellulose membrane filters (Millipore) on BHI plates and incubated for 18 hours at 37°C. Following incubation, the nitrocellulose membranes were sliced into strips and placed in 3 ml BHI. Vigorous vortexing was applied to dislodge bacterial cells from the filter membranes. Samples of 300 µI volume were plated out on large plates containing BHI, Nalidixic acid (Nal; 50 µg/ml), colistin (Col; Polymixin E) (10 µg/ml) and either chloramphenicol (10 /µg/ml) or erythromycin (10 /µg/ml). Plates were incubated for 48 hours at 37°C before picking of colonies.<br>
Colonies were patched onto BHI/Nal/Col plates that contained kanamycin (15 µg/ml), as well as onto plates with BHI/Nal/Col and either chloramphenicol (10 /µg/ml) or erythromycin (10 µg/ml), for antibiotc sensitivity testing. All ex-conjugants demonstrated an antibiotic profile that suggested the chromosomal integration of the minCD deletion without integration of the plasmid. That is, all ex-conjugants grew on antibiotic plates containing the internal marker erythromycin or chloramphenicol and no ex-conjugants grew on plates containing kanamycin (KmR encoded by the neo gene on donor plasmids).<br>
Over one hundred ex-conjugants were examined for the minicell phenotype, using dark field microscopy (40x) and oil immersion (l00x). All isolates demonstrated a varying number of minicell structures among the population of L. monocytogenes parent rods, when compared to parent cells under the microscope<br>
39<br><br>
(ENIhOOl and ENLmOOl).   This result also is consistent with integration of the minCD deletion and disruption of normal pericentral division.<br>
Thirty ex-conjugants were chosen and subcultured to test for maintenance of die minicell phenotype. Upon confirmation of maintenance, the isolates were stored as glycerol stocks for future experiments.<br>
Example 3. Purification of minicells from bacterial species<br>
Minicells were purified by the following inventive method. This example details purification of S. typhimurium minCDE- derived minicells. The same procedure was used to purify minicells from additional min mutant strains, including two mutants of S. typhimurium, and one mutant each of E. coli, S. flexneri and L. monocytogenes. The process was optimized and repeated more than 50 times to generate purified minicells. It was reliable, and routinely yielded 108 to 109 purified minicells from a 10 L bacterial culture.<br>
A 5. typhimurium mwCDE-/pEGFP-Cl culture was established from a glycerol stock in 50 ml TSB containing antibiotics Chloramphenicol and Kanamycin (50 ug/ml final concentration). The culture was incubated with shaking at 37°C overnight. A 2.5 ml aliquot of the overnight culture was used to inoculate IL (in a 2L baffled conical flask) of TSB containing the above-mentioned antibiotics, and five flasks were incubated with shaking at 37°C overnight.<br>
(A) Pre-Preparation (Stage I)<br>
A 100 L Bioprocess bag was filled with Type 1 water (MQ) by sterile hose via a 0.2 urn filter. Into two 20-liter carboys, previously autoclaved, that contained 2 L of 10X BSG, 18 L of sterile process water was transferred by peristaltic pump. One carboy was for diluting the minicell suspension and the other was for use in diafiltration.<br>
40<br><br>
(B)  Differential Centriftigatton and Pre-Preparation (Stage 2)<br>
The bacterial culture was centrifuged at 2000g for 10 minutes (Sorvall Legend T/RT; TTH 750 rotor). The supernatant was decanted into a sterile 5 L carboy mat was fitted with a 0.2 µm breather filter and a quick disconnect fitting. The decantation procedure was performed in a Class II biohazard cabinet. The carboy was sealed, sterile tubing was connected to the 5 L carboy, and the other end of the tubing was connected to the pre-filled 20 L carboy containing 20 L of 1 x BSG as described above. The minicell-carrying suspension was pumped from the 5 L carboy into the 20 L carboy, to give a dilution of 1:5.<br>
(Q Continuous Minicell Purification System<br>
Three cross-flow systems from Sartorius were connected in series. In duplicate, 0.45 µm Sartocon Slice Cassette filters were fitted in the first two slice holders, and a 0.2 µn Sartocon Slice filter cassette was fitted in the last holder. The torsion on each filter unit was tightened to 20 Nm (Newton meters), using a torsion wrench. Each unit was attached to a pump via a sanitary element. Feed, retentate and permeate lines were connected. Prior to attachment of carboys, the entire system was internally washed with 6 L of IN NaOH at 2 bar pressure for 15 minutes. This step internally sterilized the various hoses and filters. The system was drained of NaOH by reversing the pump direction of liquid flow and a water flux-rate test was performed to ensure proper filter cleaning. The test was performed according to die manufacturer's instructions (Sartorius manual). Acceptable flux-rates of 3,000 to 3,600 ml per minute for the retentate and 600 to 800 ml per minute for permeate were ensured prior to performing the minicell filtration. The system still carried a high pH and hence this was neutralized by flushing and recirculating through each system with sterile 1 x PBS (pH 7.4) until the measured pH of the PBS pool was in the range of 7.0 to 8.0. Carboys (20 L) were then connected. The minicell suspension (25 L) was carried in a first carboy, which was connected via a hose to the first 0.45 µm filter cassette. To prevent filter fouling, the minicell suspension was diluted as the filtration step proceeded (i.e.,<br><br>
diafiltration). The diluent (20L of 1 x BSG) was carried in a second carboy. Therefore, in the first cross-flow filtration step, the minicell suspension was filtered through in a volume of 45 L. The permeate valve was initially closed and the minicell suspension was pumped over the surface of the 0.45 µm filter at 2 bar pressure for 5 minutes. This conditioned the filter for the minicell suspension medium. The permeate valve was then opened and the minicell suspension was permeated (2 bar pressure, 600 ml per min) through the 0.45 µm filter and the permeate was collected in a third carboy. As the volume in the first carboy decreased, the amount of non-filtered solids increased, and hence diafiltration was switched on when the volume in the first carboy dropped to 15 L. This diluted the solids in the first carboy, preventing filter fouling and maximizing minicell recovery in the third carboy. Once the volume of permeate in the third carboy reached approximately 12.5 L, the second 0.45 µm cross-flow filter was conditioned for the minicell suspension found in the third carboy. When the volume in the third carboy reached the 15 L mark, the permeate valve was opened, allowing permeation of the minicell suspension into a fourth carboy.<br>
At this stage, the larger parent bacterial cell contamination in the minicell suspension was removed. The next stage was to eliminate smaller contaminants in the suspension such as bacterial blebs, free endotoxin, nucleic acids, cellular debris, and excess liquid. This was accomplished via filtration through a 0.2 µm cross-flow filter. Minicells are approximately 0.4 µm in diameter and, hence, do not permeate through a 0.2 µm pore size. Bacterial blebs, on the other hand, range in size (diameter) from 0.05 µm to 0.2 µm and hence are filtered out. Other contaminants also are less than 0.2 µm and hence me only constituents retained in this filtration step were the minicells and any residual parent bacterial cells.<br>
When the volume in the fourth carboy reached approximately 15 L, the 0.2 urn cross-flow filter was conditioned for the minicell suspension present in the fourth carboy. The permeate valve men was opened, allowing permeate to go to waste in a fifth carboy while the minicells were retained and concentrated in the<br>
42<br><br>
fourth carboy. Due to incorporation of the diafiltration system, the minicells were continually being diluted and filtered, which ensured complete removal of contaminants at die end of die process. The concentration step therefore reduced die minicell suspension volume to approximately 4 L from me starting volume of 45 <br>
5     L.<br>
(D)	Buffer Exchange for the Minkell Suspension<br>
The residual salts, media components and low molecular weight wastes in the minicell suspension were eliminated by diafiltration wim 1 x BSG. The apparatus was assembled and equilibrated as described before. The minicell suspension was placed in a first 4 L carboy, and 20 L of sterile 1 x BSG (diafiltration medium) were placed in a second carboy. The cross-flow unit was assembled with two 0.1 µm filter cassettes to ensure mat the minicells were unable to pass through but all contaminants less than 0.1 µm were eliminated. The pump was switched on and speed adjusted to provide 0.5bar pressure. The permeate valve was opened, and the minicell suspension flowed through the feed line, over the 0.1 µm filter. Minicells returned to die first carboy via me retentate line. The waste flowed dirough die permeate line and was collected in a third carboy. This reduced die volume of die minicell suspension and hence me diafiltration system was switched on to pump 1 x BSG into me first carboy. This step continuously replenished die volume of die minicell suspension to keep it at 4 L. The procedure was continued until die second carboy was emptied, resulting in five changes of me minicell suspension buffer.<br>
(E)	Sterilizing FUtraUon of the Minicell Suspension<br>
At tins stage, die minicell suspension still carried some parent bacterial contamination because tibe 0.45 µm cross-flow filters were not sterilizing filters. Therefore, it was important to eliminate any residual parent bacteria to obtain a minicell suspension that was optimal for in-vitro and in-vivo use. The 4 L minicell suspension from die previous step was initially diluted to 20 L in sterile 1 x BSG<br>
43<br><br>
and was held in a first carboy. A dead-end filter unit carrying a 0.45 um filter with a large surface area (500 cm2) was pre-wetted with 2 L of sterile 1 x BSG and integrity tested according to the manufacturer's instructions. The minicell suspension was pumped at a flow rate of 700 ml/min (i.e., slow flow rate to prevent forcing parent bacterial cells through the 0.45 µm filter) through the dead-end filter. Bacterial cells were retained by the filter, whilst the minicells flowed through into a second carboy via the filtrate line.<br>
(F)  Concentration of Purified Minicells<br>
The purified minicells, in a 20 L suspension, were concentrated to a smaller volume. This step was not readily accomplished by standard centrifugation and pellet resuspension technique, however, because the volumes were large and, in v»vactice, not conducive to centrifugation technique.<br>
The concentration step was performed in the following four stages.<br>
Stage 1: The minicell suspension was pumped at 0.5 bar pressure out of a first carboy and over a 100 kDa cross-flow filter via a sanitary element. The minicells were returned to the first carboy via a retentate line, and the liquid permeate was collected in a second carboy via a permeate waste line.<br>
Stage 2: Once the minicell suspension volume was reduced to 4 L, the process was stopped and the suspension was transferred into a third sterile 4 L carboy. The latter was fitted with 6.4 mm diameter hoses compared to the 12.5 mm hoses used for handling the larger volumes above. This reduced the void volume of the tubing. Because the feed and retentate lines were 12.5 mm diameter tubing, an adapter was designed to fit the hose in lid closure. Similarly, an adapter was designed to fit the larger bore tubing of feed and retentate lines. This second concentration stage was performed as in the previous stage until the minicell volume was further reduced to approximately 200 ml.<br>
44 <br>
Stage 3: The 200 ml minicell suspension was transferred into a modified Schott bottle that contained sterile internal glass tubing for feed and retentate. The glass tubing was bored through the cap of the Schott bottle, and was sealed with Marprene tubing and silicon. On the cap, the bottle also carried a breather filter (0.2 µm). To reduce void volume further, the previously used feed, sanitary element and retentate tubing was replaced with sterile 6.4 mm Marprene tubes, and the previously used pump was replaced with a smaller one. The process of concentration was performed as before (100 kDA cross:flow filter) until the minicell volume was reduced to 50 ml.<br>
Stage 4: The highly purified minicell suspension was transferred under sterile conditions into a 50 ml Falcon tube.<br>
Example  4. Scanning electron microscopy characterization of minicells from Gram-negative and Gram-positive minCD- strains<br>
Scanning (SEM) and Transmission (TEM) electron micrographs of minicells derived from S. typhimurium, E. coli, S. flexmeri, L. monocytogenes and the corresponding parent cells were taken to determine the morphology and dimensions of the various minicells. Briefly, bacterial cultures carrying minicells were centrifuged at 13,200 rpm for 20 minutes and resuspended in PBS containing 2.5% glutaraldehyde and fixed at room temperature for 40 minutes. Samples were centrifuged and washed three times in distilled water. For TEM, the following procedure was followed. To change solutions the cells were centrifuged at 10,000 rpm for 1 minute, the supernatant was pipetted off, then the cells were resuspended in the new reagent using a vortex mixer. The sequence of reagents was: (a) osmium tetroxide in 0.1 M cacodylate buffer at pH 7.2 -10 minutes, (b) sodium acetate 4% in distilled water - 1 minute, (c) uranyl acetate 4% in distilled water - S minutes, (d) 70% ethanol - S minutes, (e) 100% ethanol - S minutes, (f) 100% acetone - S minutes, (g) 1:1 acetone and Spurr's epoxy resin monomer - 30 minutes, (h) pure Spurr's epoxy resin - cure for 48 hours at 60°C.  Sections were cut from the cured<br>
45<br><br>
resin blocks with a diamond knife using a Reichert Ultracut E ultramicrotome. Sections were stained with uranyl acetate for 10 minutes followed by lead citrate for 2 minutes. The sections were examined using a Hitachi H-7000 transmission electron microscope operated with a beam energy of 75 kilovolts (University of New South Wales, NSW, Australia). Digital images were recorded using an Analysis MegaView II widefield CCD camera.<br>
For High Resolution Scanning Electron Microscopy the following method was followed. To change solutions the cells were centrifuged at 13,000 rpm for 20 minutes, the supernatant was pipetted off, and the cells were resuspended in the new reagent using a vortex mixer. The intention was to wash all ions and biomaterials off the cells and leave them suspended in a small volume of distilled water. The sequence of reagents was (a) 1 ml of distilled water - repellet, (b) 1 ml of distilled water - resuspend, (c) deposit 2S0 pi on a dean brass specimen plate, (d) dry overnight at 30°C, (e) coat just before microscopy with 2 nm of chromium.metal deposited in a Xenosput clean vacuum sputter coater. The coated specimens were examined using an Hitachi S-900 Field Emission Scanning Electron microscope using a beam energy of 3 kilovolts (University of New South Wales, NSW, Australia). Digital images at different magnifications were recorded using an ImageSlave digitiser.<br>
The results showed that the minicells derived from both the Gram-negative and Gram-positive bacteria were about 400 nm in diameter, and except for L. monocytogenes they appeared to have a ruffled surface as seen by SEM, presumably due to the lipopolysaccharide cell surface structures. There was no apparent difference in the surface ultrastructure of minicells and the parent bacteria for all species. TEM results showed that L. monocytogenes minicells had a rigid cell wall structure expected of a Gram-positive bacterial cell membrane. Salmonella, S. flexneri and E. coli membranes collapsed more readily under the microscope electron beam. The minicell formation event, i.e., asymmetric cell division, was observed in all samples.<br><br>
Example 5  Minicell uptake by mammalian cells such as macrophages<br>
To demonstrate the uptake of recombinant minicells by macrophages, it was first necessary to incorporate a tracer such as GFP, in order that the recombinant minicells could be seen, distinct from the mammalian cells. Therefore, plasmid pEGFP was transformed into the S. typhimurium, E. coli and S. flexneri minCDE minicell producing strains to determine if minicells would stably carry EGFP and fluoresce green.<br>
Plasmid pEGFP (Table 2) is a bacterial expression plasmid that expresses a red-shifted variant of wild-type green fluorescent protein (EGFP; excitation maximum: 488 nm; emission maximum: 507 nm) from the lac promoter. The plasmid backbone is the pUC19 derivative, pPD 16.43 (Fire et al., 1990), which provides a high-copy-number origin of replication and an ampicillin resistance gene for propagation and selection in bacterial cells. The plasmid was transformed into S. typhimurium, E. coli and S. flexneri minCDE strains and recombinant bacteria were grown in Brain Heart Infusion broth (BHI; Difco Laboratories, Detroit, Michigan USA) until an OD600 of 0.6 was reached. The minicells were isolated and purified from the mixed culture and were visualized by fluorescent microscopy (Fluorescence microscope DMLM, Leica Microsystems). The results revealed that all minicells fluoresced bright green, while minicells purified from non-recombinant S. typhimurium, E. coli and S. flexneri minCDE strains (controls) did not show any green fluorescence. The stored minicells (4°C and room temperature) were viewed by fluorescence microscopy at time intervals of 30 minutes, 1 hour, 6 hours, 12 hours, 18 hours, 24 hours, 2 days, 1 week, and 2 weeks. The results showed that the minicells were intact and continued to fluoresce bright green throughout the study. The plasmid pEGFP therefore provided a tracer (minicells fluorescing green) with which to follow the uptake of recombinant minicells by mammalian cells, such as macrophages, and other cells such as cancer cells. These results indicate that recombinant proteins, once expressed or segregated into minicells, are stable for a<br>
47<br><br>
significant period of time (probably until minicell cellular integrity is compromised) due to the absence of chromosomally encoded proteases.<br>
Cells of the mouse macrophage cell line RAW264.7 (Ralph &amp; Nakoinz, 1977) obtained from the American Type Culture Collection (ATCC), were cultured in vitro at a cell density of 105 per well, and were infected with purified recombinant minicells, carrying plasmid pEGFP, that were derived from S. typhimurium. E. coli, and S. flexneri, at a minicell:macropbage ratio of 50:1 and 100:1.<br>
Prior to macrophage infection, the purified minicells were visualized via fluorescence microscope to confirm that most minicells carried EGFP and hence fluoresced bright green. As a positive control, S. typhimurium aroA- strain SL3261 (Table 1) carrying the same plasmid was also used to transfect the macrophage cells at the same bacteria.macrophage ratio. Negative control (uninfected macrophages) were also processed in the same way as experimental infected cells. The culture plates were centrifuged at l000g for 10 minutes at 37°C. For the mmicell transfection study, the antibiotics gentamycin (100 µg/ml) and ampicillin (200 µg/ml) were added to kill any residual live parent bacterial cells. The positive control salmonellae were also killed with the same antibiotic treatment. The plates were incubated at 37°C for 30 minutes in 5% CO2, followed by three washes in PBS. For the S. typhimurium derived minicell/macrophage experiment, the slides were fixed with 4% formaldehyde for 40 minutes and then permeabilized with 0.2% Triton X-100. After blocking non-specific staining with 5% normal goat serum (NGS) in phosphate buffer containing 5% BSA, the coverslips were incubated with anti-upopolysaccharide (LPS) antibody (rabbit 4-0 Salmonella somatic agglutinating serum; 1:200 dilution; Murex Biotech, Dartfbrd, England) for four hours at room temperature. The cells were washed three times in PBS and incubated with secondary antibody (1:1000), Alexa Fluor 594 (Molecular Probes, Eugene, OR, USA; goat anti-mouse IgG conjugate, excitation: 590 nm, emission: 617 nm) in PBS-BSA.  Plates were incubated with second antibody for 1 hour in the dark and<br><br>
were washed three times with PBS for 5 minutes each.  Coverslips were mounted with glycerol and viewed by confocal microscopy.<br>
The results showed that S. typhimurium, E. coli and S. flexneri-derivcd minicells were all engulfed by approximately 20% to 30% of the macrophage cells in culture. The minicells fluoresced bright green and were associated with the macrophages. The control non-recombinant S. typhimurium, E. coli and S. flexneri-derived minicells did not reveal green fluorescent dots associated with macrophages . other than minor non-specific background fluorescence. Control recombinant S. typhimurium aroA- strain also gave a similar result to that seen with both the / recombinant minicells.<br>
To confirm that the green fluorescent dots were within the macrophage cells, i.e., were engulfed minicells, and not just adhered to the cell surface, three-dimensional images (using sagittal and coronal sections) were taken for the S.tryp/umunum-derived minicell-macrophage interaction. In both coronal and sagittal sections, the minicells were localized within the macrophages, indicating that the minicells had been engulfed by the macrophages. Additionally, anti-04 LPS labeling of the green fluorescent dots (yellow fluorescence following secondary antibody Alexa Fluor 594) showed that the green fluorescent dots in fact were EGFP-expressing minicells and not artifact background fluorescence. A similar result was observed with the positive control salmonellae, demonstrating that the bacterial surface structures required for receptor-mediated uptake of the cells by macrophages were conserved on the minicell surface.<br>
Example 6. Minicell uptake and breakdown in macrophage phagolysosomes<br>
To demonstrate the intracellular fate of minicells within macrophages, TEM studies were carried out on mouse macrophages infected with S. typhimurium-derived minicells.<br>
49<br><br>
Briefly, mouse macrophage cell line RAW 246.7 was grown to ~50% confluence in T25 flasks in standard culture media. Minicells on the order of 107, in 100 (d, were added directly to media in flasks, and the process of macrophage infection was carried out as described in Example 5. An approximate ratio of minicell: macrophage of 10:1 was used. Cells were collected for time points corresponding to 30 minutes, 60 minutes, and 2 hours post-infection. An additional flask was also included as a negative control, receiving no minicells. Cells were trypsinised &amp; pellets collected and fixed in 4% glutaraldehyde (500 µl). Samples were processed and analyzed by TEM (University of New South Wales, Sydney, Australia).<br>
The results showed mat as early as 30 minutes post-infection, electron-dense particles approximately the size of minicells (400 nm) were observed within macrophage vacuoles (Figure 5, panels A-F). With the progression of time (60 minutes and 2 hours), the electron-dense particles appeared less intact with surface irregularity and loss of electron-density.<br>
To confirm that the intra-vacuolar electron-dense particles were engulfed minicells, the above experiment was repeated with a difference being mat after the various time intervals post-infection, cells were fixed (4% paraformaldehyde, 0.1% glutaraldehyde) for 30 minutes at room temperature, washed with PBS and collected into 1.5 ml PBS by gentle cell scraping. Samples were processed for immunogold-TEM (EM Unit, ICPMR, Westmead Hospital, Sydney, Australia). Samples were gently pelleted and processed by freeze-substitution method. Briefly, the samples were labeled with primary antibody (anti-S. typhimurium lipopolysaccharide [Factor 4, Group B specificity]; Abbott Murex, USA) 1:200 dilution, followed by gold (10 nm) conjugated secondary antibody. The samples were viewed using a Philips CM-120 BioTWIN electron microscope at 80 kV. Images were captured onto type 4489 Kodak EM emulsion film.<br>
50<br><br>
The results showed that the miniceiis were clearly identified by the gold-labeled anti-04-LPS antibody and that the electron-dense particles observed in the macrophage vacuoles were the S. typhimurium-derived minicells. No gold-labeling was observed in control macrophages that had not been infected with minicells. This data also revealed that at later time points, gold particles not associated with minicells were observed in the vacuoles. There was a marked increase in the minicell-free gold particles at later time-points and this was also associated with increased numbers of minicells that had lost cell wall integrity and cellular electron-density. These data indicate that the minicells follow the classical pathway of antigen-uptake and processing exhibited by macrophages, which includes foreign particle ingestion into early endosomes followed by endosome-lysosome fusion and breakdown of the antigen in the acidic phagolysosome. The minicell-free gold particles in the late stages of infection may indicate LPS that is released from processed or digested minicells.<br>
These results show that recombinant minicells not only are engulfed by mammalian cells, such as macrophages, but also are degraded in intracellular vacuoles, presumably phagolysosomes.<br>
Example 7.     Expression of heterologous protein by minicell transfected macrophages<br>
To determine if recombinant minicells carrying a mammalian gene expression plasmid encoding EGFP (not expressed in bacterial cells or minicells) could deliver die plasmid to die mammalian cell nucleus and achieve expression of EGFP in the mammalian cell, die following experiment was performed.<br>
Cells of the mouse macrophage cell line RAW-264.7 were cultured in vitro and infected with purified, recombinant S. typhimurium, E. coli and S. flexneri-derived miniceiis carrying plasmid pEGFP-Cl (Table 2), as described in Example 5. Forty-eight hours post-transfection, the infected cells were visualized by three-dimensional confocal microscopy.<br>
51<br><br>
The results showed that approximately 20% of the macrophages fluoresced green, suggesting that the recombinant minicells were broken down within the macrophages, presumably in phagolysosomes, and mat at least some of the released plasmid DNA was taken up by the cell nucleus prior to expression of the green fluorescent protein. Control macrophages, i.e., macrophages transfected with non-recombinant minicells, did not reveal the green fluorescence. EGFP expression in the experimental cells took at least 48 hours. This result was similar to that observed with positive control macrophages transfected with plasmid pEGFP-Cl using electroporation with a BioRad Genepulser.<br>
To confirm further that EGFP expression within the minicell-transfected macrophages was not background fluorescence, this experiment was repeated for S. typhimirium-derived minicells. In this case, after fixation with formaldehyde, the cover slips were incubated with anti-GFP monoclonal antibody (Clontech Laboratories, Palo Alto, CA, USA; 1:300 dilution) and incubated overnight at 4°C.<br>
The cover slips were washed three times with PBS (S minutes per wash) and incubated with 2% normal goat serum in PBS/BSA for 20 minutes. The cover slips were washed twice with PBS and treated with secondary antibody Alexa Fluor 594-anti-mouse IgG conjugate in PBS (1:1000 dilution). The reaction was incubated in the dark for 1 hour and washed twice in PBS. The cover slips were visualized by confocal microscopy using red (S70 nm) and green (488 nm) fluorescence visualization filters. The results revealed that that the spots of green fluorescence (laser excitation at 488 nm) observed within the macrophage were identical to the spots of red fluorescence (laser excitation at 570 nm). When both lasers were used, the green and red fluorescence signals were co-localized, and appeared as a yellow fluorescence. Additionally, when die 3D image was constructed using Leica 3D image software, the fluorescence was found to be within the macrophage. These results show that the observed green fluorescent spots were due to macrophage expression of EGFP protein since the same spots were identified by anti-GFP monoclonal antibody (red fluorescence).<br>
52<br><br>
The results confirm that recombinant minicells do break down in host mammalian cells such as macrophages, releasing plasmid DNA and that this DNA is able to express foreign protein within the mammalian cell. This demonstrates the feasibility of in vitro gene therapy by means of recombinant, intact minicells.<br>
After recombinant minicells have been introduced into a patient, the presence of the heterologous gene product can be monitored or assessed by an appropriate assay for the gene product in the patient, for example in peripheral red blood cells of the patient when expression is erythroid cell-specific. As described above, the choice of assay is partly a function of the heterologous gene product and can be determined readily.<br>
F.framplg ft.  Minicell-mediated gene delivery to and gene expression in human breast cancer cells<br>
Minicells purified from recombinant S. typhimurium minCDE- strain carrying plasmid pEGFP-Cl (eukaryotic gene expression only; Table 2) were used to infect human breast cancer cells (SK-BR-3). Forty-eight hours and 96 hours post-transfection, the cells were visualized via confocal microscopy. As a negative control, non-recombinant minicells were used to transfect SK-BR-3 cells and were visualized similarly to experimental cells.<br>
SK-BR-3 breast cancer cells (source ATCC, reference No. HTB-30) were cultured on coverslips in 6-well plates and grown to approximately 50% confhiency. Minicells carrying eukaryotic GFP-expression plasmid pEGFP-Cl were added to cells and centrifuged for 10 minutes, l000g to allow minicell / SK-BR-3 cell contact. Cells were cultured for 48 hours after which G-418 (400 mg/ml) was added, with some wells receiving no G-418. After a further 48-hour incubation, all coverslips were fixed with 4% formaldehyde for 1 hour. The coverslips were incubated with PBS-BSA (2% BSA in PBS) for 20 minutes and washed once with PBS. The coverslips were incubated overnight at 4°C with anti-Her-2 antibody (Serotec, monoclonal mouse and- human IgG; 1:100 dilution).   The cells were<br>
53<br><br>
washed three times with PBS for 5 minutes each wash, and were incubated for 1 hour in the dark with Alexa Fluor 594-conjugated secondary antibody (Molecular Probe, goat anti-mouse IgG conjugate, excitation: 590 nm, emission: 617 nm; 1:1000 dilution in PBS-BSA). The cells were washed three times with.PBS for S minutes per wash, and coverslips were treated with antifade medium (Molecular Probe). All coverslips were visualized by three-dimensional confocal microscopy (excitation with wavelengths for red filter: 568 nm and green filter: 488 nm).<br>
The results revealed that approximately 10% of the breast cancer cells clearly expressed the Green Fluorescence Protein that was localized in the cytosol (see Figure 6, panels A-C). This was clearly visible over normal background autofluorescence exhibited by control cells. The cells were clearly identified with the anti-Her-2 antibody (red fluorescence).<br>
This result demonstrates that recombinant minicells are able to deliver mammalian gene expression plasmid DNA to non-phagocyte cells, exemplified by epithelial breast cancer cells, in a manner mat leads to heterologous expression within the cells.<br>
Example 9. Minicell-mediated gene delivery and gene expression in vivo in Balb/c mice<br>
To determine mat recombinant minicells could deliver a foreign gene to cells of the immune system in vivo, mice were vaccinated intraperitoneally with recombinant S. typhimurium minCDE- derived minicells carrying plasmid pEGFP-Cl (eukaryotic gene expression only; Table 2) and compared to mice vaccinated with S. typhimurium AaroA strain bearing the same plasmid.<br>
Recombinant minicells were purified as in Example 3. S. typhimurium (SL3261 AaroA strain, Table 1) was prepared as follows. Five (5) ml Trypticase Soy Broth (TSB, Becton Dickinson, Palo Alto, CA, USA) was inoculated with a 1:100 inoculum of an overnight culture of S. typhimurium in TSB, and grown at<br>
54<br><br>
37°C with shaking until the Optical density (O.D.) measured at 600 run reached 0.5. The bacteria were then incubated with gentamycin (Sigma-Aldrich, Castle Hill, NSW, Australia) at 150 /µg/ml and ampicillin (Roche) at 150 jig/ml for a further 2 hours at 37°C with shaking.<br>
The killed bacteria and minicells were pelleted by centrifugation at 8000 rpm, resuspended and washed a further three times in BSG (phosphate buffered saline [PBS; 1.44 gm disodium hydrogen phosphate, 0.24 gm potassium dihydrogen phosphate, 0.2 gm potassium chloride, 8.0g sodium chloride, pH 7.4 in 1 liter distilled water], containing 2% gelatin).<br>
Groups of eight 6-week old Balb/c mice were inoculated iotraperitoneally wim 100 µl of recombinant minicells or recombinant killed S. typhimurium according to the schedule in Table 4. One group of 8 mice remained unvaccinated as negative controls. Mice were bled by intraocular bleeding before inoculation and at day 14 post-primary vaccination. On day 23, all animals were anaesthetized with an intraperitoneal injection of 12 mg sodium phenobarbitone and 1 ml of blood was collected by cardiac puncture before the mice were sacrificed. Blood was centrifuged at 3000 rpm for 10 minutes in a microfuge (Eppendorf, Hamburg, Germany) and serum was collected for ELISA assays. Throughout the experiment, animals were weighed weekly and observed daily for signs of toxicity.<br>
Table 4.        Treatment allocation for in vivo gene delivery in Balb/c mice<br><br>
Group number	Animal number	Treatment	Dose (100 ul IP injection)	Dosing days	Bleeding days<br>
1	1-8	None- control	-	-	1,14,23<br>
2	9-16	Recombinant minicells	108 minicells per dose	1.5	14,23<br>
3	17-24	Recombinant killed S. typhimurium	108bacteria per dose	1,5	14,23<br>
4	25-32	Recombinant killed S. typhimurium	108 bacteria per dose	1,5,8	14,23<br>
5	33-40	Recombinant killed S. typhimurium	109 bacteria per dose	1,5	14,23<br>
6	41-48	Recombinant killed S. typhimurium	109 bacteria per dose	1,5,8	14,23<br><br>
ELISA assays were performed to determine whether antibody had been generated against GFP, that is, whether recombinant minicells were able to deliver the mammalian expression plasmid pEGFP-Cl in-vivo. S. typhimurium lipopolysaccharide (LPS) antibody levels were also determined for all groups. The indirect ELISA method was carried out as follows.<br>
Ninety six (96&gt;well microtitre plates (Greiner GMBH, NYrtingen, Germany) were coated with 50 pi per well of 0.5 /µg/ml rEGFP (Clontech, Palo Alto, CA, USA) or LPS antigen (Sigma). Plates were coated with 1 % BSA (Sigma) as negative control. Plates were sealed and incubated overnight at 4°C. Plates were inverted to remove antigen solution, and 200 pi blocking buffer (0.05% Tween-20 [Sigma], 1% BSA in PBS) was added to each well before incubating for 2 hours at room temperature. Blocking buffer was removed and the plates were washed twice for 5 minutes with wash buffer (0.05% Tween-20, PBS). Serum samples were diluted 1 in 80 and 1 in 300 for EGFP and LPS respectively in blocking buffer. Next, 100 µl of sample was added to each well, and incubated 1 hour at room temperature with shaking. Plates were men washed with wash buffer 3 times for 5 minutes. Secondary antibodies, namely, alkaline phosphatase conjugated anti-mouse immunoglobulin (γ- &amp; light chains; Chemicon, Temicula, CA, USA), or AP-conjugated anti-LPS monoclonal antibody (IgGl isotype, Biodesign International, Saco, Maine, USA) were diluted in blocking buffer, and 100 µl was added per well, followed by a 1 hour incubation at room temperature with shaking. Wells were washed three times with wash buffer, and 100 µl PNPP (p-nitrophenyl phosphate substrate; Zymed, San Francisco, CA, USA) was added. Absorbance at 405 nm was read after a 30 minute incubation at room temperature. The reaction was terminated by addition of 30 µl of 0.5 M NaOH. ELISA data significance was determined by Student t test (p).<br>
Results are shown in Figure 7. At 14 days post-vaccination, a strong and significant antibody response (p 
56<br><br>
response observed was greater than that obtained with the highest dose of killed S. typhimurium. Antibodies to EGFP protein would only be observed if the recombinant minicells bearing the mammalian expression vector pEGFP-Cl not only are engulfed by peritoneal macrophages but also are degraded in intracellular vacuoles (presumably, phagolysosomes), and that at least some plasmid DNA copies escaped die phagolysosomes and entered the mammalian cell nucleus. From the nucleus, EGFP mRNA would be produced and EGFP expressed in die cytoplasm. The EGFP would be a foreign protein in the macrophage and, hence, would be expected to be processed and peptides would be presented via MHC. This process would result in an antibody response to the EGFP peptides. Compared with die control killed S. typhimurium, the anti-EGFP antibody response was higher with die recombinant minicells.<br>
The anti-LPS response also was measured to determine the immune response to the gene therapy delivery vector, the recombinant minicells. The results showed that the anti-LPS antibody response was significant and similar for recombinant minicells (p = 0.0004) and killed S. typhimurium (p = 0.001). See Figure 8. This result indicated that the minicells had retained at least the LPS structure found on the parent bacterial cell surface. By day 23, the anti-EGFP antibody response was not different from the nonirnmunized controls (for both recombinant minicells and killed 5. typhimurium). This was not surprising because no booster immunizations had been administered to sustain the antibody response. The anti-LPS response at day 23 was similar to that seen at day 14. This is not unexpected because LPS is known to be a potent immunogen that induces high and sustained antibody titers.<br>
Example 10. Minicell-mediated gene delivery and gene expression in vivo in Balb/c mice with different dosing regimes<br>
Recombinant minicells were prepared as in Example 3. Groups of eight, 6-week old Balb/c mice were inoculated intraperitoneally with 100 µl of recombinant minicells (containing plasmid pEGFP-Cl; Table 2) according to die schedule shown<br>
57<br><br>
in Table 5. One group of eight mice remained unvaccinated, as negative controls. Mice were bled by intraocular bleeding before inoculation and at day 14 postprimary vaccination and serum collected as in Example 9.<br>
Table 5. Treatment allocation for in vivo gene delivery with different dose regimes of recombinant minicells in Balb/c mice<br><br>
Group number	Animal number	Treatment	Dose (100^1 IP injection)	Dosing days	Bleeding days<br>
1	1-8	None- control	-	-	1.14<br>
2	9-16	Recombinant minicells	108 minicells per dose	1.4	14<br>
3	17-24	Recombinant minicells	108 minicells per dose	1.4,8	14<br>
4	25-32	Recombinant minicells	109 minicells per dose	1.4	14<br>
5	33-40	Recombinant minicells	109 minicells tier dose	1.4,8	14<br>
ELISA assays were performed as previously described to determine whether antibody had been generated against GFP, and whether higher doses of recombinant minicells or three rather than two doses enabled the animal to mount a larger antibody response. S. typhimurium lipopolysaccharide (LPS) antibody levels were also determined for all groups.<br>
Results are shown in Figure 9. At 14 days post-vaccination, a very significant antibody response (p 
 <br>
The anti-LPS response also was measured, to determine me immune response to the recombinant minicells. The results showed that the anti-LPS antibody response was significant <p see figure></p>
59<br><br>
CITED PUBLICATIONS<br>
Balicki &amp; Beutler, "Gene therapy of human disease," Medicine (Baltimore) 81: 69 (2002).<br>
Brahmbhatt, "Cloning and molecular characterization of the rfb gene cluster of Salmonella typhimurium* Ph.D. Thesis, University of Adelaide, Australia (1987).<br>
Britton et al., "Characterization of a prokaryotic SMC protein involved in chromosome partitioning," Genes Dev. 12:1254 (1998).<br>
Bruckner, "A series of shuttle vectors for Bacillus subtilis and Escherichia coli," Gene 122: 187 (1992)<br>
Catic et al.. Introduction of protein or DNA delivered via recombinant Salmonella typhimurium into the major histocompatibility complex class I presentation pathway of macrophages," Microbes Infect. 1: 133 (1999).<br>
Ciliberto et al., "Cell-specific expression of a transfected human alpha 1-antitrypsin gene," Cell. 41; 531 (1985).<br>
Clark-Curtiss &amp; Curtiss, "Analysis of recombinant DNA using Escherichia coli minicells," Methods Enzymol. 101:347 (1983).<br>
Courvalin et al., "Gene transfer from bacteria to mammalian cells," C.R. Acad. Sd.IU 318: 1207(1995).<br>
Curiel et al., "Long-term inhibition of clinical and laboratory human immunodeficiency virus strains in human T-cell lines containing an HIV-regulated diphtheria toxin A chain gene," Hum. Gene Ther. 4: 741 (1993).<br>
Davis et al., ADVANCED BACTERIAL GENETICS: A MANUAL FOR GENETIC ENGINEERING (Cold Spring Harbor Laboratory Press, 1980;.<br>
de Boer et al., "Roles of MinC and MinD in the site-specific septation block mediated by the MinCDE system of Escherichia coti," J. Bacteriol. 174:63 (1992).<br>
Dietrich et al, "Delivery of antigen-encoding plasmid DNA into the cytosol of macrophages by attenuated suicide Listeria monocytogenes,” Nature Biotechnol. 16: 181 (1998).<br>
Dinges et al., "HIV-regulated dipthertia toxin A chain gene confers long-term protection against HIV type 1 infection in the human promonocytic cell line U937," Hum. Gene Ther. 6: 1437 (1995).<br>
Dorward et al., "Export and intercellular transfer of DNA via membrane blebs of Neisseria gonorrhoeae* J. Bacteriol. 171: 2499 (1989).<br>
60<br><br>
Fire et al., "A modular set of lacZ fusion vectors for studying gene expression in Caenorhabditis elegans," Gene-93:189 (1990).<br>
Firth et al., "Structure and function of die F factor and mechanism of conjugation,*' in ESCHERICHIA COU AND SALMONELLA: CELLULAR AND MOLECULAR BIOLOGY 2nd ed. (1996), at pages 2377-2401.<br>
Forbes, "Crossflow inicrofiltration," Australian J. Biotechnology 1: 30(1987).<br>
Frain et al., "Binding of a liver-specific factor to the human albumin gene promoter and enhancer," Mol. Cell Biol. 10: 991 (1990).<br>
Frazer &amp; Curtiss, "Production, properties and utility of bacterial minicells," Curr Top Microbiol. Immunol. 69: 1 (1975).<br>
Freshner, ANIMAL CELL CULTURE: A PRACTICAL APPROACH 2nd ed. (Oxford/New York, IRL Press, Oxford University Press, 1992).<br>
Gentschev et al., "Delivery of protein antigens and DNA by virulence-attenuated strains of Salmonella typhimurium and Listeria monocytogenes," J. Biotechnol. 83: 19 (2000).<br>
Grillot-Courvalin et al., "Functional gene transfer from intracellular bacteria to mammalian cells," Nat. Biotechnol. 16: 862 (1998).<br>
Grillot-Courvalin et al., "Wild-type intracellular bacteria deliver DNA into mammalian cells," Cell Microbiol. 4: 177 (2002).<br>
Hanahan, "Studies on transformation of Escherichia coli with plasmids," J. Mol. Biol. 166: 557 (1983).<br>
Haibin et al., "siRNA-mediated gene silencing in vitro and in vivo," Nat. Biotechnol. 20:1006 (2002).<br>
Hanahan, "Heritable formation of pancreatic beta-cell tumors in transgenic mice expressing recombinant insulin/simian virus 40 oncogenes," Nature 315: 115 (1985).<br>
Harlow et al., "Cloning and characterization of the gsk gene encoding guanosine kinase of Escherichia coli," J. Bacterid. 177:2236 (1995).<br>
Harrison et al., "Inhibition of human immunodeficiency virus-1 production resulting from transduction with a retrovirus containing an HIV-regulated diphtheria toxin A chain gene," Hum. Gene Ther. 3:461 (1992a).<br>
61<br><br>
Harrison et al., Inhibition of HIV production in cells containing an integrated, HIV- regulated diphtheria toxin A chain gene," AIDS Res. Hum. Retroviruses 8: 39 (1992b).<br>
Harry, "Bacterial cell division: Regulating Z-ring formation," Mol. Microbiol. 40: 795 (2001).<br>
Hart, "Tissue specific promoters in targeting systematically delivered gene therapy," Semin. Oncol. 23:154 (1996).<br>
Hashimotoh-Gotoh et al., "Specific-purpose plasmid cloning vectors. I. Low copy number, temperature-sensitive, mobilization-defective pSCl0l-derived containment vectors," Gene 16:227 (1981).<br>
Heim et al., "Wavelength mutations and posttranslational autoxidation of green fluorescent protein," Proc. Nat'l. Acad. Sci. USA 91: 12501 (1994).<br>
Hiraga et al., "Chromosome partitioning in Escherichia coli: novel mutants producing anucleate cells," J. Bacteriol. 171: 1496 (1989).<br>
Hoisem et al., "Aromatic-dependent Salmonella typhimurium are non-virulent and effective as live vaccines," Nature 291: 238 (1981).<br>
Hu &amp; Lutkenhaus, "Topological regulation of cell division in Escherichia coli involves rapid pole to pole oscillation of the division inhibitor MinC under the control of MinD and MinE," Mol. Microbiol. 34: 82 (1999).<br>
Ireton et al., "spoOJ is required for normal chromosome segregation as well as the initiation of sporulation in Bacillus subtilis," J. Bacteriol. 176: 5320 (1994).<br>
Katabi et al., "Hexokinase Type II: A Novel Tumor Specific Promoter for Gene-Targeted Therapy Differentially Expressed and Regulated in Human Cancer Cells," Human Gene Therapy 10: 155 (1999).<br>
Katsui et al., "Heat-induced blebbing and vesiculation of the outer membrane of Escherichia coli," J. Bacteriol. 151: 1523 (1982).<br>
Kelsey et al., "Species- and tissue-specific expression of human alpha 1-antitrypsin in transgenic mice," Genes andDevel. 1:161 (1987).<br>
Keren) et al., "Identification of the cystic fibrosis gene: genetic analysis," Science 245:1073 (1989).<br>
Kihara et al., "Analysis of a FliM-FliN flagellar switch fusion mutant of Salmonella typhimurium,' J. Bacteriol. 178:4582 (1996).<br>
62<br><br>
Kurane et al., "Targeted Gene Transfer for Adenocarcinoma Using a Combination of Tumor specific Antibody and Tissue-specific Promoter," Jpn. J. Cancer Res. 89: 1212 (1998).<br>
Leder et al., "Consequences of widespread deregulation of the c-myc gene in transgenic mice: multiple neoplasms and normal development," Cell 45: 485 (1986).<br>
Levin et al., "Identification of Bacillus subtilis genes for septum placement and shape determination," J. Bacteriol. 174: 6717 (1992).<br>
MacDonald et al., "Expression of the pancreatic elastase I gene in transgenic mice," Hepatology 7: 425 (1987).<br>
Macrina et al., "A cloning vector able to replicate in Escherichia coli and Streptococcus sanguis," Gene 19: 345 (1982).<br>
Marcil &amp; Higgins. "Direct transfer of plasmid DNA from yeast to E. coli by electroporation," Nucl. Acids Res. 20: 917 (1992).<br>
Mason et al., "The hypogonadal mouse: reproductive functions restored by gene thercpy," Science 234: 1372 (1986).<br>
Matsuzaki et al., "Interactions of an antimicrobial peptide, magainin 2, with outer and inner membranes of Gram-negative bacteria," Biochim Biophys. Acta. 1327: 119(1997).<br>
Miller &amp; Mekalanos, "A novel suicide vector and its use in construction of insertion mutations: osmoregulation of outer membrane proteins and virulence determinants in Vibrio cholerae requires toxR," J. Bacteriol. 170: 2575 (1988).<br>
Mori, "Complete and shotgun sequencing: yehV, minE, minD, minC, ypjA from Escherichia coli strain K12," direct submission to the DDBJ/EMBL/GenBank databases (1996).<br>
Morton &amp; Potter, "Rhabdomyosarcoma-specific expression of the herpes simplex virus thymidine kinase gene confers sensitivity to ganciclovir," J. Pharmacology &amp; Exper. Therapeutics 286:1066 (1998).<br>
Okada et al., "Possible function of the cytoplasmic axial filaments in chromosomal segregation and cellular division of Escherichia coli," Sci. Prog. 77: 253 (1993-94).<br>
Okada et al., "Cytoplasmic axial filaments in Escherichia coli cells: possible function in the mechanism of chromosome segregation and cell division," J. Bacteriol. 176: 917 (1994).<br>
Pearson &amp; Lipman, "Improved tools for biological sequence comparison," Proc. Nat'l. Acad. Sci. USA 85: 2444 (1988).<br>
63<br><br>
Pikaar et al., "Opsonic activities of surfactant proteins A and D in phagocytosis of gram-negative bacteria by alveolar macrophages," J. Infect. Dis. 172: 481 (1995).<br>
Pinkert et al., "An albumin enhancer located 10 kb upstream functions along with its promoter to direct efficient, liver-specific expression in transgenic mice,' Genes andDevel. 1:268(1987).<br>
Prasher et al., "Using GFP to see the light," Trends in Genetics 11: 320 (1995).<br>
Ragheb et al., "Inhibition of human immunodeficiency virus type 1 by Tat/Rev-regulated expression of cytosine deaminase, interferon alpha2, or diphtheria toxin compared with inhibition by transdominant Rev," Hum. Gene Tker. 10:103 (1999).<br>
Ralph &amp; Nakoinz, "Antibody-dependent killing of erythrocyte and tumor targets by macrophage-related cell lines: Enhancement by PPD and LPS," J. Immunol. 119: 950 (1977).<br>
Raskin &amp; de Boer, "MinDE-dependent pole-to-pole oscillation of division inhibitor MinC in Escherichia coti," J. Bacteriol. 181: 6419 (1999).<br>
Readhead et al., "Myelin deficient mice: expression of myelin basic protein and generation of mice with varying levels of myelin," Cell 48: 703 (1987).<br>
Reeve, "Use of minicells for bacteriophage-directed polypeptide synthesis," Methods Enzymol. 68: 493 (1979).<br>
Reeve &amp; Cornett, "Bacteriophage SPOl-induced macromolecular synthesis in minicells of Bacillus subtilis," J. Virol. 15: 1308 (1975).<br>
Riezman, "Three clathrin-dependent budding steps and cell polarity," Trends in Cell Biology. 3: 330 (1993).<br>
Riordan et al., "Identification of the cystic fibrosis gene: cloning and characterization of complementary DNA,* Science 245: 1066 (1989).<br>
Romano et al., "Gene transfer technology in therapy: current applications and future goals," Onocologist 3: 225 (1998).<br>
Romano et al., "Gene transfer technology in therapy: current applications and future goals," Stem Cells. 17: 191 (1999),<br>
Rommens et al., "Identification of the cystic fibrosis gene: Chromosome walking and jumping," Science 245: 1059 (1989).<br>
Sambrook et al., MOLECULAR CLONING: LABORATORY MANUAL 2nd ed. (Cold Spring Harbor Laboratory Press, 1989).<br>
64<br><br>
6'.<br>
Sancar et al., "Simple method for identification of plasmid-coded proteins," /. Bacteriol. 137: 692 (1979).<br>
Sandvig &amp; Deurs, "Endocytosis without clathrin," Trends in Cell Biology. 4: 27S (1994).<br>
Seeliger, LISTERIOSIS, 2nd ed„ at page 308 (Karger, 1961).<br>
Shangara et al., "Suicide genes: past, present and future perspectives," Immunology Today 21: 48 (2000).<br>
Shaw &amp; Griffon, "Phagocytosis requires repeated triggering of macrophage phagocytic receptors during particle ingestion," Nature 289: 409 (1981).<br>
Shigekawa &amp; Dower, "Electroporation of eukaryotes and prokaryotes: A general approach to the introduction of macroraolecules into cells," BioTechniques 6: 742 (1988).<br>
Simon et al., "A broad host range mobilization system for in vivo genetic engineering: transposon mutagenesis in gram negative bacteria," Biotechnology 1: 784 (1983).<br>
Sizemore et al., "Attenuated Shigella as a DNA delivery vehicle for DNA-mediated immunization," Science 270: 299 (1995).<br>
Speert et al., "Functional characterization of macrophage receptors for In-vitro phagocytosis of unopsonized pseudomonas-aeruginosa," /. Clin. Invest. 82: 872 (1988).<br>
Spencer, "Developments in suicide genes for preclinical and clinical applications," Molecular Therapeutics 2: 433 (2000).<br>
Stewart &amp; D'Ari, "Genetic and morphological characterization of an Escherichia coli chromosome segregation mutant," J. Bacteriol. 174: 4513 (1992).<br>
Sullivan et al., "New shuttle vectors for Bacillus subtulis and Escherichia coli which allow rapid detection of inserted fragments," Gene 29: 21 (1984).<br>
Swift et al., "Tissue-specific expression of the rat pancreatic elastase I gene in transgenic mice," Cell 38: 639 (1984).<br>
Trieu-Cuot et al., "Shuttle vectors containing a multiple cloning site and a lacZ alpha gene for conjugal transfer of DNA from Escherichia coli to gram-positive bacteria," Gene 102:99 (1991).<br>
Wachi et al., "New mre genes mreC and mreD, responsible for formation of the rod shape of Escherichia coli cells," /. Bacteriol. 171: 6511 (1989).<br>
65<br><br>
Wadhwa et al., "Cancer gene therapy:  Scientific basis," Ann. Rev. Med. 53: 437 (2002).<br>
Wright &amp; Jong, "Interferon-gama depresses binding of ligand by c3b and c3bi receptors on cultured human monocytes, an effect reversed by fibronectm, Experimental Medi. 163: 1245(1986).<br>
Yanisch-Perron et al., "Improved M13 phage cloning vectors and host strains: nucleotide sequences of the M13mpl8 and pUC19 vectors," Gene 33: 103 (1985).<br>
Yazawa et al., "Current progress in suicide gene therapy for cancer," World J. Surg. 26: 783 (2002).<br>
66<br><br>
We Claim:<br><br>
1.	A method for purification of minicells comprising, passing a sample containing minicells (i) over a series of cross-flow filters and then (ii) through a dead-end filter, whereby minicells are separated from contaminants in said sample to obtain a purified minicell preparation.<br>
2.	A method as claimed in claim 1, further comprising the step of treating said purified minicell preparation with an antibiotic.<br>
3.	A method as claimed in claim 1, further comprising a preliminary step of performing differential centrifugation on said sample containing minicells.<br>
4.	A method as claimed in claim 1, wherein said series of cross-flow filters comprises at least one filter employing a pore size greater than or equal to about 0.45 µm, and at least one filter employing a pore size less than or equal to about 0.2 µm.<br>
5.	A method as claimed in claim 4, wherein said dead-end filter employs a pore size of about 0.45 µm .<br>
6.	A method as claimed in claim 5, wherein said series of cross-flow filters comprises at least two filters employing a pore size of about 0.45 µm , and at least one filter employing a pore size of about 0.2 µm.<br><br>
7.	A method as claimed in claim 6, further comprising the step of treating said purified minicell preparation with an antibiotic.<br>
Dated this 30th Day of May 2006<br>
                                           					Dr.Rajeslikumar H. Acharya<br>
Advocate &amp; Patent Agent<br>
For and on Behalf of Applicant<br>
67<br><br>
Abstract<br>
A composition comprising recombinant, intact minicells that contain a therapeutic nucleic acid molecule is disclosed. Methods for purifying a preparation of such minicells also are disclosed. Additionally, a genetic transformation method is disclosed, which comprises (i) making recombinant, intact minicells available mat contain a plasmid comprised of a first nucleic acid segment, and (ii) bringing the minicells into contact with mammalian cells that are engumng-competent, such mat the minicells are engulfed by the mammalian cells, which thereafter produce an expression product of the first nucleic acid segment.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
		<br>
		<div class="pull-left">
			<a href="221075-substituted-indolepyridinium-as-anti-infective-compounds.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="221077-frame-structure-of-scooter-type-motorcycle.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>221076</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>629/MUMNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>35/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>29-Aug-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>16-Jun-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>31-May-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>ENGENEIC MOLECULAR DELIVERY PTY LTD</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>105 DELHI ROAD, NORTH RYDE, SYDNEY, NSW 2113. AUSTRALIA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>MACDIARMID,Jennifer</td>
											<td>8 Lombard Street Glebe, Sydney,NSW 2037 Australi</td>
										</tr>
										<tr>
											<td>2</td>
											<td>BRAHMBHATT, HIMANSHU</td>
											<td>96 PRAIRIE VALE ROAD, BOSSLEY PARK, SYDNEY, NSW 2176. AUSTRALIA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IB02/04632</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-10-15</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/328,801</td>
									<td>2001-10-15</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/221076-intact-minicells-as-vectors-for-dna-transfer-and-gene-therapy-in-vitro-and-in-vivo by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:02:30 GMT -->
</html>
