<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/221083-a-method-for-purifying-double-stranded-dna-from-a-solution by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:02:02 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 221083:&quot;A METHOD FOR PURIFYING DOUBLE-STRANDED DNA FROM A SOLUTION&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A METHOD FOR PURIFYING DOUBLE-STRANDED DNA FROM A SOLUTION&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method for purifying double-stranded DNA from a solution containing the double-stranded DNA mixed with other components, comprising: (i) passing the solution through a support at an acidic pH, the support comprising a covalently coupled oligonucleotide capable of forming a triple helix with the double-stranded DNA by hybridization at an acidic pH with a specific sequence present in the double-stranded DNA, with the covalently coupled oligonucleotide comprising the sequence TCTTTTTTTCCT (SEQ ID NO:28) or TTCTTTTTTTTCTT (SEQ ID NO:30) and the specific sequence present in the double-stranded DNA comprising the sequence AGAAAAAAAGGA (SEQ ID NO:27) or AAGAAAAAAAAGAA (SEQ ID NO:29); and (ii) raising the pH so that it is no longer acidic, to elute the bound DNA hence obtaining the double-stranded DNA.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>BACKGROUND OF THE INVENTION<br>
The present invention relates to a new method for DNA purification. The method according to the invention enables pharmacologically usable double-stranded DNA to be purified rapidly. More especially, the purification method according to the invention involves a specific hybridization between a sequence of the DNA and an oligonucleotide.<br>
Gene and cell therapy techniques are currently undergoing remarkable development. However, these techniques entail the possibility of producing large amounts of DNA of pharmaceutical purity. In effect, in these new therapies, the medicament often consists of DNA itself, and it is essential to be able to manufacture it in suitable amounts, to isolate it and to purify it in a manner suited to therapeutic use in man.<br>
In recent years, the feasibility of injection of plasmid DNA for gene therapy or vaccination has been demonstrated by numerous reports demonstrating that DNA expression vectors can be taken up by various cell types and genes encoded by these plasmids can be subsequently expressed (Ledley, 1995 Hum. Gene Ther. 6, 1129).<br>
The genes of interest for gene therapy or vaccination applications may include, for example, tumor suppressor gene, suicide genes, or anti-sense sequences. They can also encode proteins such as alpha-fetoprotein AFP (Morinaga, 1983, Proc. Natl. Acad. Sci. USA, 80,4604), enzymes, hormones, cytokines, growth factors such as FGF (Jouanneau et al, 1991, Proc. Natl. Acad. Sci. USA, 88,2893) or VEGFB (Olofsson B al., 1996, Proceedings 93, 576), clotting factors such as B-deleted Factor VIII (Truett et al., 1985, DNA 4, 333), apolipoproteins, neurotransmitters, neurotrophic factors, natural or chimeric immunoglobulin. Reporter genes such as lacZ encoding the Escherichia coli fj-galactosidase are also used.<br><br>
Major challenges for using plasmid DNA as a gene delivery vector in human are i) the manufacture and ii) the purity of this drug product. Technologies for the production of plasmids vectors with, high copy number in Escherichia coir hosts have been recently developed. The plasmids currently used are either ColEl-derived plasmids such as pBR322, pUC or pBluescript (Lahijani et al., 1996, Hum. Gene Ther., 1, 1971) or pCORplasmids (Soubrier et al., 1999, Gene Therapy, 6, 1482).<br>
The second concern raised by the use of plasmid DNA as a gene therapy vector is the purity of the plasmid vector itself. Current purification methods such as ultracentrifugation in CsCl gradients or chromatography can be inefficient in removing contaminants such as host genomic DNA and RNA or proteins. Particularly, host genomic DNA whose chemical structure is very close to that of plasmid DNA, is extremely difficult to remove using classical chromatography. Typical concentrations of up to 0.5 to 1 % host genomic DNA are found in plasmid preparations obtained by classical chromatography. Therefore, in order to develop plasmid DNA as a safe vector for human gene therapy, there is a need for purification technologies that will lower the content of host genomic DNA down to much lower levels, typically 0.1 % or even 0.01 % or lower.<br>
The present invention describes a simple and especially effective new method for DNA purification. It makes it possible, in particular, to obtain especially high purities with high yields.  The method according to the invention is based essentially on a specific interaction between a sequence inserted into the DNA to be purified and an oligonucleotide composed of natural or modified bases.<br>
It has recently been shown that some oligonucleotides are capable of interacting specifically hi the wide groove of the DNA double helix to form triple helices locally, leading to an inhibition of the transcription of target genes (H61ene et Toulme, Biochim. Biophys. Acta 1049 (1990) 99). These oligonucleotides selectively recognize the DNA double helix at oh'gopurine-oligopyrimidine sequences, that is to say at regions possessing an oligopurine sequence on one strand and an oligopyrimidine sequence on the complementary strand, and form a triple helix locally thereat. The bases of the third strand (the oligonucleotide) form hydrogen<br><br>
bonds (Hoogsteen or reverse Hoogsteen bonds) with the purines of the Watson-Crick base pairs.<br>
A use of this type of interaction to isolate a plasmid has been described in the prior art. Thus, Ito et al. (PNAS 89 (1992) 495) describe the use of biotinylated oligonucleotides capable of recognizing a particular sequence of a plasmid and of forming a triple helix therewith. The complexes thus formed are then brought into contact with streptavidin-coated magnetic beads. Interaction between the biotin and the streptavidin then enables the plasmid to be isolated by magnetic separation of the beads followed by elution. However, this method has some drawbacks. In particular, two successive specific interactions are needed, the first between the oligonucleotide and the plasmid and the second between the biotinylated complex and the streptavidin beads. Furthermore, the final solution may be contaminated with biotinylated oligonucleotide, which cannot be used in a pharmaceutical composition.<br>
SUMMARY OF THE INVENTION<br>
The present invention describes a new, improved method of DNA purification making use of this type of interaction. More especially, the method of the invention employs oligonucleotides coupled covalently to a support. This method is especially rapid, and it leads to especially high yields and degrees of purity. Moreover, it enables DNA to be purified from complex mixtures comprising, in particular, other nucleic acids, proteins, endotpxins (such as lipopolysaccharides), nucleases and the like. The supports used may, in addition, be readily recycled, and the DNAs obtained display improved properties of pharmaceutical safety. Lastly, this method entails only one step, contrary to the prior ait.<br>
Hence a first subject of the invention lies in a method for the purification of double-stranded DNA, according to which a solution containing the said DNA mixed with other components is passed through a support to which is coupled covalently an oligonucleotide capable of forming a triple helix by hybridization with a specific sequence present in said DNA. The specific sequence can be a sequence naturally present in the double-stranded DNA, or a synthetic sequence introduced artificially into the latter.<br><br>
The oligonucleotides used in the present invention are oligonucleotides which hybridize directly with the double-stranded DNA. These oligonucleotides can contain the following bases:<br>
thymidine (T), which is capable of forming triplets with A.T doublets of double-stranded DNA (Rajagopal et al, Biochem 28 (1989) 7859);<br>
adenine (A), which is capable of forming triplets with A.T doublets of double-stranded DNA;<br>
guanine (G), which is capable of forming triplets with G.C doublets of double-stranded DNA;<br>
protonated cytosine (C+), which is capable of forming triplets with G.C doublets of double-stranded DNA (Rajagopal et al., loc. cit.);<br>
uracil (U), which is capable of forming triplets with A.U or A.T base pairs.<br>
Preferably, the oligonucleotide used comprises a cytosine-rich homopyrimidine sequence and the specific sequence present in the DNA is a homopurine-hornopyrimidine sequence. The presence of cytosines makes it possible to have a triple helix which is stable at acid pH where the cytosines are protonated, and destablized at alkaline pH where the cytosines are neutralized.<br>
To permit the formation of a triple helix by hybridization, it is important for the oligonucleotide and the specific sequence present in the DNA to be complementary. In this connection, to obtain the best yields and the best selectivity, an oligonucleotide and a specific sequence which are fully complementary are used in the method of the invention. These can be, hi particular, an oligonucleotide poly(CTT) and a specific sequence poly(GAA). As an example, there may be mentioned the oligonucleotide of sequence<br>
(SEQ ID NO: 1), in which the bases GAGG do not form a triple helix but enable the oligonucleotide to be spaced apart from the coupling arm; the sequence (CTT)y (SEQ ID No: 26) may also be mentioned. These oligonucleotides are capable of forming a triple helix with a specific sequence containing complementary units (GAA). The sequence in question can, in particular, be a region containing 7,14 or 17 GAA units, as described in the examples.<br><br>
Another sequence of specific interest is the sequence: 5'- ' (SEQ. ID NO: 5). This sequence forms a triple helix with the oligonucleotides 5 (SEQ ID No: 6) or 5'- (SEQ ID NO: 7).<br>
In this case, the oligonucleotide binds in an antiparallel orientation to the polypurine strand. These triple helices are stable only in the presence of Mg2+ (Vasquez et al., Biochemistry, 1995, 34,7243-7251; Beal andDervan, Science, 1991, 251,1360-1363).<br>
As stated above, the specific sequence can be a sequence naturally present in the double-stranded DNA, or a synthetic sequence introduced artificially in the latter. It is especially advantageous to use an oligonucleotide capable of forming a triple helix with a sequence naturally present in the double-stranded DNA, for example in the origin of replication of a plasmid or in a marker gene.- In this connection, the Applicant has performed plasmid sequence analyses, and was able to show that some regions of these DNAs, in particular in the origin of replication, could possess homopurine-homopyrimidine regions. The synthesis of oligonucleotides capable of forming triple helices with these natural homopurine-homopyrimidine regions advantageously enables the method of the invention to be applied to unmodified plasmids, in particular commercial plasmids of the pUC, pBR322, pSV, and the like, type. Among the homopurine-homopyrimidine sequences naturally present in a double-stranded DNA, a sequence comprising all or part of the sequence S1 (SEQ ID NO: 2) present in the origin of replication of E. coli plasmid ColEl may be mentioned. In this case, the oligonucleotide forming the triple helix possesses the sequence: S'- ' (SEQ ID NO: 3), and binds alternately to the two strands of the double helix, as described by Beal and Dervan (J. Am. Chem. Soc. 1992, 114,4976-4982) and Jayasena and Johnston (Nucleic Acids Res. 1992, 20, 5279-5288). The sequence 5'- (SEQ ID NO: 4) of the plasmid pBR322 /Mactamase gene (Duval-Valentin et al., Proc. Natl. Acad. Sci. USA, 1992, 89, 504-508) may also be mentioned.<br>
Two additional target sequences which can form triplex structures with particular oligonucleotides have been identified in ColEl and in pCOR origins of replication. ColEl-derived plasmids contain a 12-mer homopurine sequence (5'- (SEQ ID NO: 27) mapped upstream of the RNA-H transcript involved inplasmid replication (Lacatena et al, 1981, Nature, 294, 623). This sequence forms a stable triplex structure with the 12-mer complementary 5'- (SEQ ID NO: 28) oligonucleotide. The pCOR backbone contains a homopurine stretch of 14 non repetitive bases<br>
 (SEQ ID NO: 29) located in the A+T-rich segment of the y origin replicon of pCOR (Levchenko et al., 1996, Nucleic Acids Res., 24,1936). This sequence forms a stable triplex structure with the 14-mer complementary oligonucleotide 5'- (SEQ ID NO: 30). The corresponding oligonucleotides (SEQ ID NO: 28) and 5'- ' (SEQ ID NO:30) efficiently and specifically target their respective complementary sequences located within the origin of replication of either ColEl ori or pCOR (oriy). In fact, a single non-canonical triad (T*GC or C*AT) may result in complete destabilization of the triplex structure.<br>
The use of an oligonucleotide capable of forming a triple helix with a sequence present in an origin of reph'cation or a marker gene is especially advantageous, since it makes it possible, with the same oligonucleotide, to purify any DNA containing the said origin of replication or said marker gene. Hence it is not necessary to modify the plasmid or the double-stranded DNA in order to incorporate an artificial specific sequence in it.<br>
Although fully complementary sequences are preferred, it is understood, however, that some mismatches may be tolerated between the sequence of the oligonucleotide and the sequence present in the DNA, provided they do not lead to too great a loss of affinity. The sequence (SEQ ID NO: 8) present in the |L coli p-lactamase gene may be mentioned, m this case, the thymine interrupting the polypurine sequence may be recognized by a guanine of the third strand, thereby forming a G*TA triplet which it is stable when flanked by two T*AT triplets (Kiessling et al., Biochemistry, 1992, 31_, 2829-2834).<br>
According to a particular embodiment, the oligonucleotides of the invention comprise the sequence (CCT)n, the sequence (CT)n or the sequence (CTT)m in which n is an integer between 1 and 15 inclusive. It is especially advantageous to use sequences of the type (CT)n or (CTT)n. The Applicant showed, in effect, that the purification yield was influenced by the amount of C in the oligonucleotide. In particular, as shown in Example 7, the purification yield increases when the oligonucleotide contains fewer cytosines. It is understood that the oligonucleotides of the invention can also combine (CCT), (CT) or (CTT) units.<br>
The oligonucleotide used may be natural (composed of unmodified natural bases) or chemically modified. In particular, the oligonucleotide may advantageously possess certain chemical modifications enabling its resistance to or its protection against nucleases, or its affinity for the specific sequence, to be increased.<br>
According to the present invention, oligonucleotide is also understood to mean any linked succession of nucleosides which has undergone a modification of the skeleton with the aim of making it more resistant to nucleases. Among possible modifications, oligonucleotide phosphorothioates, which are capable of forming triple helices with DNA (Xodo et al, Nucleic Acids Res., 1994, 22, 3322-3330), as well as oligonucleotides possessing formacetal or methylphosphonate skeletons (Matteucci et al., J. Am. Chem. Soc., 1991,113, 7767-7768), may be mentioned. It is also possible to use oligonucleotides synthesized with a anomers of nucleotides, which also form triple helices with DNA (Le Doan et al., Nucleic Acids Res., 1987, 15, 7749-7760). Another modification of the skeleton is the phosphoramidate link. For example, the N3 -P5 internucleotide phosphoramidate link described by Gryaznov and Chen, which gives oligonucleotides forming especially stable triple helices with DNA (J. Am. Chem. Soc., 1994, 116, 3143-3144), may be mentioned. Among other modifications of the skeleton, the use of ribonucleotides, of 2'-O-methylribose, phosphodiester, etc. (Sun and Helene, Curr. Opinion Struct. Biol., 116. 3143-3144) may also be mentioned. Lastly, the phosphorus-based skeleton may be replaced by a polyamide skeleton as in PNAs (peptide nucleic acids), which can also form triple helices (Nielsen et al., Science, 1991, 254,1497-1500; Kim et al., J. Am. Chem. Soc., 1993, 115, 6477-6481), or by a guanidine-based skeleton, as in DNGs (deoxyribonucleic<br>
guanidine, Proc. Natl. Acad. Sci. USA, 1995, 92, 6097-6101), or by polycationic<br>
analogues of DNA, which also formtriple helices.<br>
The thymine of the third strand may also be replaced by a 5-bromouracil,<br>
which increases the affinity of the oligonucleotide for DNA (Povsic and Dervan, J.<br>
Am. Chem. Soc., 1989,1.11,3059-3061). The third strand may also contain unnatural<br>
bases, among which there may be mentioned 7-deaza-2'-deoxyxanthosine (Milligan et<br>
al., Nucleic Acids Res., 1993, 21, 327-333),<br>
l-(2-deoxy-p-D-ribofuranosyl)-3-methyl-5-amino-<br>
l#-pyrazolo[4,3-^pyrimidin-7-one (Koh and Dervan, J. Am. Chem. Soc., 1992, 114,<br>
1470-1478), 8-oxoadenine, 2-aminopurine, 2'-0-methylpseudoisocytidine, or any<br>
other modification known to a person skilled in the art (for a review see Sun and<br>
Helene, Curr. Opinion Struct. Biol., 1993, 3, 345-356).<br>
Another type of modification of the oligonucleotide has the aim, more<br>
especially, of improving the interaction and/or affinity between the oligonucleotide<br>
and the specific sequence, hi particular, a most advantageous modification according to the invention consists in methylating the cytosines of the oligonucleotide (see Example 5). The oligonucleotide thus methylated displays the noteworthy property of forming a stable triple helix with the specific sequence in pH ranges closer to neutrality (&gt; 5). It hence makes it possible to work at higher pH values than the oligonucleotides of the prior art, that is to say at pH values where the risks of degradation of plasmid DNA are much smaller.<br>
The length of the oligonucleotide used in the method of the invention is at least 3 bases, and preferably between 5 and 30. An oligonucleotide of length greater than 10 bases is advantageously used. The length may be adapted by a person skilled in the art for each individual case to suit the desired selectivity and stability of the interaction.<br>
The oligonucleotides according to the invention may be synthesized by any known technique. In particular, they may be prepared by means of nucleic acid synthesizers, Any other method known to a person skilled in the art may quite obviously be used.<br>
To permit its covalent coupling to the support, the oligonucleotide is generally functionalized. Thus, it may be modified by a thiol, amine or carboxyl terminal group<br>
at the 5' or 3' position. In particular, the addition of a thiol, amine or carboxyl group makes it possible, for example, to couple the oligonucleotide to a support bearing disulphide, maleimide, amine, carboxyl, ester, epoxide, cyanogen bromide or aldehyde functions. These couplings form by establishment of disulphide, thioether, ester, amide or amine links between the oligonucleotide and the support. Any other method known to a person skilled in the art may be used, such as bifunctional coupling reagents, for example.<br>
Moreover, to improve the hybridization with the coupled oligonucleotide, it can be advantageous for the oligonucleotide to contain an "arm" and a "spacer" sequence of bases. The use of an arm makes it possible, in effect, to bind the oligonucleotide at a chosen distance from the support, enabling its conditions of interaction with the DNA to be improved. The arm advantageously consists of a linear carbon chain, comprising 1 to 18 and preferably 6 or 12 (CE^) groups, and an amine which permits binding to the column. The arm is linked to a phosphate of the oligonucleotide or of a "spacer" composed of bases which do not interfere with die hybridization. Thus, the "spacer" can comprise purine bases. As an example, the "spacer" can comprise the sequence GAGG. The arm is advantageously composed of a linear carbon chain comprising 6 or 12 carbon atoms.<br>
For implementation of the present invention, different types of support may be used. These can be functionalized chromatographic supports, hi bulk or prepacked hi a column,_functionalized plastic surfaces or functionalized latex beads, magnetic or otherwise. Chromatographic supports are preferably used. As an example, the chromatographic supports capable of being used are agarose, acrylamide or dextran as well as their derivatives (such as Sephadex, Sepharose, Superose, etc.), polymers such as poly(styrene/divinylbenzene), or grafted or ungrafted silica, for example. The chromatography columns can operate in the diffusion or perfusion mode.<br>
To obtain better purification yields, it is especially advantageous to use, on the plasmid, a sequence containing several positions of hybridization with the oligonucleotide. The presence of several hybridization positions promotes, in effect, the interactions between the said sequence and the oligonucleotide, which leads to an improvement in the purification yields. Thus, for an oligonucleotide containing n repeats of (CCT), (CT) or (CTT) motifs, it is preferable to use a DNA sequence<br>
containing at least n complementary motifs, and preferably n+1 complementary motifs. A sequence carrying n+1 complementary motifs thus affords two positions of hybridization with the oligonucleotide. Advantageously, the DNA sequence contains up to 11 hybridization positions, that is to say n+10 complementary motifs.<br>
The method according to the present invention can be used to purify any type of double-stranded DNA. An example of the latter is circular DNA, such as a plasmid, generally carrying one or more genes of therapeutic importance. This plasmid may also carry an origin of replication, a marker gene, and the like. The method of the invention may be applied directly to a cell lysate. In this embodiment, the plasmid, amplified by transformation followed by cell culture, is purified directly after lysis of the cells. The method of the invention may also be applied to a clear lysate, that is to say to the supernatant obtained after neutralization and centrifugation of the cell lysate. It may quite obviously be applied also to a solution prepurified by known methods. This method also enables linear or circular DNA carrying a sequence of importance to be purified from a mixture comprising DNAs of different sequences. The method according to the invention can also be used for the purification of double-stranded DNA.<br>
The cell lysate can be a lysate of prokaryotic or eukaryotic cells. As regards prokaryotic cells, the bacteria E. coli, B. subtilis. S. typMmurium or Strepomyces may be mentioned as examples. As regards eukaryotic cells, animal cells, yeasts, fungi, and the like, may be mentioned, and more especially Kluvveromyces or Saccharomyces yeasts or COS, CHO, C127, NTH3T3, and the like, cells.<br>
The method of the invention is especially advantageous, since it enables plasmid DNA of very high purity to be obtained rapidly and simply. In particular, as illustrated in the examples, this method enables the plasmid DNA in question to be separated effectively from contaminating components such as fragmented chromosomal DNA, endotoxins, proteins, nucleases, and the like. More especially, the method of the invention enables preparation of double-stranded DNA, in particular that of plasmid origin, having a chromosomal DNA content of less than or equal to 0.5 % to be obtained. Still more preferably, the DNA preparations obtained have a chromosomal DNA content of less than or equal to 0.2 %. The present<br>
invention hence describes compositions comprising plasmid DNA which can be used pharmaceutically, in particular in gene or cell therapy, hi this connection, the subject of the invention is also a pharmaceutical composition comprising double-stranded DNA, linear or of plasmid origin, prepared according to the method described above.<br>
The invention also relates to plasmid DNA preparations having a chromosomal DNA content of less than or equal to 0.5 %, preferably less than or equal to 0.2 % and still more preferably less than or equal to 0.1 %, and still more preferably less than or equal to 0.01%. As exemplified below, a triplex affinity interaction step has been incorporated in a purification process downstream of classical chromatographic steps. This affinity step significantly improves the purity of the plasmid preparation, whatever its initial purity. The formation of a triplex structure between an oligonucleotide (covalently bound to a chromatography support) and the plasmid of interest to be purified relies upon the presence on the plasmid of a sequence that can form a triplex structure with the oligonucleotide. This triplex structure is stable at acidic pH only, where the cytosines of the oligonucleotide are protonated. Then, plasmid DNA is eluted of the column simply by raising the pH to neutral.<br>
The compositions can contain plasmid DNA which is "naked" or combined with transport carriers such as liposomes, nanoparticles, cationic lipids, polymers, recombinant viruses or proteins, and the like.<br>
In one embodiment, the method according to the present invention can be used to purify one type of double-stranded DNA from a mixture comprising two or more double-stranded DNAs of different types and sequences. This method may be applied directly to a cell lysate, in which the double-stranded DNAs, amplified through cell culture, are purified after lysing the cultured cells. This method may also be applied to a clear lysate, i.e., to the supernatant obtained after neutralization and centrifugation of the cell lysate. The method may further be applied to a prepurified solution.<br>
More precisely, the method for purifying a first double-stranded DNA from a solution containing first and second double-stranded DNAs, comprises (i) passing the solution through a first support comprising a covalently coupled oligonucleotide capable of forming a triple helix with the second double-stranded DNA by<br>
hybridization with a specific sequence therein, (ii) recovering the solution that passes through the first support, which will be enriched with unbound, first double-stranded DNA, and (iii) passing the recovered solution through a second support comprising a covalently coupled oligonucleotide capable of forming a triple helix by hybridization with a specific sequence of said first double-stranded DNA. Following an optional washing step, the first double-stranded DNA can be eluted from the second support. Using this double purification method, the first double-stranded DNA can be recovered from the second support without any detectable levels of the second double-stranded DNA.<br>
In a specific embodiment of the present invention, the first double-stranded DNA molecule is a pCOR plasmid having a specific sequence 5'- (SEQ ID NO: 29), which forms a stable triplex structure with an oligonucleotide having a sequence S'- (SEQ ID NO: 30). The second double-stranded DNA molecule is a ColEl-derived plasmid having a specific sequence 5'-AGAAAAAAAGGA-3' (SEQ ID NO: 27), which forms a triplex with an oligonucleotide having a sequence 3' (SEQ ID NO: 28). Accordingly, the pCOR plasmid is advantageously purified from a solution containing other plasmids such as ColEl-derived plasmids by using the double purification method according to the present invention.<br>
The present application will be described in greater detail by means of the examples which follow, which are to be regarded as illustrative and non-lirniting.<br>
DETAILED DESCRIPTION General techniques of cloning and molecular biology<br>
The traditional methods of molecular biology, such as digestion with restriction enzymes, gel electrophoresis, transformation in K coli, precipitation of nucleic acids and the like, are described in the literature (Maniatis et al., T., E.F. Fritsch, and J. Sambrook, 1989. Molecular cloning: a laboratory manual, second edition. Cold Spring Harbor Laboratory, Cold Spring Harbor Laboratory Press, New York; Ausubel P.M., R. Brent, R.E. Kinston, D.D. Moore, J.A. Smith, J.G. Seidman andK. Struhl. 1987. Current protocols in molecular biology 1987-1988. John<br><br>
Willey and Sons, New York.). Nucleotide sequences were determined by the chain termination method according to the protocol already published (Ausubel et al, 1987).<br>
Restriction enzymes were supplied by New England Biolabs, Beverly, MA (Biolabs).<br>
To carry out ligations, DNA fragments are incubated in a buffer comprising 50 mM Tris-HCl pH 7.4,10 mM MgCl2,10 mM DTT, 2 mM ATP in the presence of phage T4 DNA ligase (Biolabs).<br>
Oligonucleotides are synthesized using phosphoramidite chemistry with the phosphoramidites protected at the ft position by a cyanoethyl group (Sinha, N.D., J. Biernat, J. McManus and H. KSster, 1984. Polymer support oligonucleotide synthesis, XVHI: Use of /?-cyanoethyl-N,N-dialkylammo-/N-morpholino phosphoramidite of deoxynucleosides for the synthesis of DNA fragments simplifying deprotection and isolation of the final product. Nucl. Acids Res., 12, 4539-4557: Giles, J.W. 1985. Advances in automated DNA synthesis. Am. Biotechnol., Nov./Dec.) with a Biosearch 8600 automatic DNA synthesizer, using the manufacturer's recommendations.<br>
Ligated DNAs or DNAs to be tested for their efficacy of transformation are used to transform the following strain rendered competent: E. coli DHSaHVondAl. hsdR17. supE44. tfai-1. recAl. gyrA96. relAl. AqacZYA-arqF)U169. deoR. 0)80dlac.(lacZAM15)] (for any Col El plasmid); or E. coli XAC-pir (for any pCor-derived plasmid).<br>
Minipreparations of plasmid DNA are made according to the protocol of Klein et al., 1980.<br>
LB culture medium is used for the growth of E. coli strains (Maniatis et al., 1982). Strains are incubated at 37°C. Bacteria are plated out on dishes of LB medium supplemented with suitable antibiotics. Example 1<br>
1.1. Preparation of the column Equipment<br>
The column used is a 1 ml HiTrap column activated with NHS (N-hydroxysuccinimide, Pharmacia) connected to a peristaltic pump (output 
ml/min. The specific oligonucleotide used possesses an NEb group at the 5' end, its sequence is as follows:<br>
«<br>
(SEQ ID NO: 1) The buffers used in this example are the following:<br>
Coupling buffer: 0.2 M NaHC03, 0.5 M NaCl, pH 8.3. Buffer A: 0.5 M ethanolamine, 0.5 M NaCl, pH 8.3. Buffer B: 0.1 M acetate, 0.5 M NaCl, pH 4.<br>
Method:<br>
The column is washed with 6 ml of 1 mM HC1, and the oligonucleotide diluted in the<br>
coupling buffer (50 nmol in 1 ml) is then applied to the column and left for 30<br>
minutes at room temperature. The column is washed three times in succession with 6<br>
ml of buffer A and then 6 ml of buffer B. The oligonucleotide is thus bound<br>
covalently to the column through a CONH link. The column is stored at 4°C in PBS,<br>
0.1% NaNa, and may be used at least four times.<br>
1.2. Construction of Plasmids<br>
The following two oligonucleotides were synthesized.<br>
oligonucleotide 4817:<br>
(SEQ ID NO: 9)<br>
oligonucleotide 4818:<br>
 (SEQ ID NO: 10)<br>
These oligonucleotides, when hybridized and cloned into a plasmid, introduce a homopurine-homopyrimidine sequence (GAA)n (SEQ ID NO: 33) into the corresponding plasmid, as described above.<br>
The sequence corresponding to these two hybridized oligonucleotides was cloned at the multiple cloning site of plasmid pBKS+ (Stratagene Cloning System, La Jolla CA), which carries an ampicillin-resistance gene. To this end, the oligonucleotides were hybridized in the following manner: one u,g of these two oligonucleotides were placed together in 40 ml of a final buffer comprising 50 mM Tris-HCl pH 7.4,10 mM MgCl2. This mixture was heated to 95°C and was then<br>
placed at room temperature so that the temperature would fall slowly. Ten ng of the mixture of hybridized oligonucleotides were ligated with 200 ng of plasmid pBKS+ (Stratagene Cloning System, La Jolla CA) digested with BamHI and EcoRI in 30 ul final. After ligation, an aliquot was transformed into DH5a. The transformation mixtures were plated out on L medium supplemented with ampicillin (50 mg/1) and X-gal (20 mg/1). The recombinant clones should display an absence of blue colouration on this medium, contrary to the parent plasmid (pBKS+) which permits a-complementation of fragment at of E. coli /?-galactosidase. After minipreparation of plasmid DNA from 6 clones, they all displayed the disappearance of the PstI site located between the EcoRI and BamHI sites of pBKS+, and an increase in molecular weight of the 448-bp PvuH band containing the multiple cloning site. One clone was selected and the corresponding plasmid was designated pXL2563. The cloned sequence was verified by sequencing using primer -20<br>
(SEQ ID NO: 11)) (Viera J. and J. Messing. 1982. The pUC plasmids, an M13mp7-derived system for insertion mutagenesis and sequencing with synthetic universal primers. Gene, 19, 259-268) for plasmid pBKS+ (Stratagene Cloning System, La Jolla CA). Plasmid pXL2563 was purified according to Wizard Megaprep kit (Prornega Corp. Madison, WE) according to the supplier's recommendations. This plasmid DNA preparation was used thereafter in examples described below. 1.3. Plasmid purification Equipment:<br>
Plasmid pXL2563 (described in 1.2) was purified on the HiTrap column coupled to the oligonucleotide, described in 1.1., from a solution also containing plasmid pBKS+. The buffers used in this purification are the following: Buffer F: 2 M NaCl, 0.2 M acetate, pH 4.5 to 5. Buffer E: 1 M Tris-HCl, pH 9, 0.5 mM EDTA. Method:<br>
The column is washed with 6 ml of buffer F, and the plasmids (20 ug of pXL2563 and 20 jig of pBKS+ in 400 ul of buffer F) are applied to the column and incubated for 2 hours at room temperature. The column is washed with 10 ml of buffer F and elution is then carried out with buffer E. The plasmids are detected after electrophoresis on 1<br>
% agarose gel and ethidium bromide staining. The proportion of the plasmids in the<br>
solution is estimated by measuring their transforming activity on E. coli.<br>
Result:<br>
Starting from a mixture containing 30 % of pXL2563 and 70 % of pBKS+, a solution<br>
containing 100 % of pXL2563 is recovered at the column outlet. The purity,<br>
estimated by the OD ratio at 260 and 280 nm, rises from 1.9 to 2.5, which indicates<br>
that contaminating proteins are removed by this method.<br>
Example 2<br>
2.1. - This example describes a plasmid DNA purification experiment. Coupling of the oligonucleotide (SEQ ID NO: 1)) to the column is performed as described in Example 1. For the coupling, the oligonucleotide is modified at the 5' end with an amine group linked to the phosphate of the spacer by an arm containing 6 carbon atoms (Modified oligonucleotide Eurogentec SA, Belgium). Plasmid pXL2563 was purified using the Wizard Megaprep kit (Promega Corp., Madison, WI) according to the supplier's recommendations. The buffers used in this example are the following: Buffer F: 0-2 MNaCl, 0.2 M acetate, pH 4.5 to 5. Buffer E: 1 M Tris-HCl pH 9, 0.5 mM EDTA.<br>
The column is washed with 6 ml of buffer F, and 100 jig of plasmid pXL2563 diluted in 400 (j.1 of buffer F are then applied to the column and incubated for 2 hours at room temperature. The column is washed with 10 ml of buffer F and elution is then carried out with buffer E. The plasmid is quantified by measuring optical density at 260 nm.<br>
In this example, binding is carried out in a buffer whose molarity with respect to NaCl varies from 0 to 2 M (buffer F). The purification yield decreases when the molarity of NaCl falls. The pH of the binding buffer can vary from 4.5 to 5, the purification yield being better at 4.5. It is also possible to use another elution buffer of basic pH: elution was thus carried out with a buffer comprising 50 mM borate, pH 9, 0.5 mM EDTA.<br>
2.2, - Coupling of the oligonucleotide<br>
(SEQ ID NO: 1) to the column is carried out as described in Example 1. Plasmid pXL2563 was purified using the<br>
Wizard Megaprep kit (Promega Corp., Madison, WT) according to the supplier's recommendations. The buffers used in this example are the following:<br>
Buffer F: 0.1 M NaCl, 0.2 M acetate, pH 5.<br>
Buffer E: 1 M Tris-HCl pH 9, 0.5 mM EDTA.<br>
The column is washed with 6 ml of buffer F, and 100 u£ of plasmid pXL2563 diluted in 400 ul of buffer F are then applied to the column and incubated for one hour at room temperature. The column is washed with 10 ml of buffer F and elution is then carried out with buffer E. The content of genomic or chromosomal E. coli DNA present in the plasmid samples before and after passage through the oligonucleotide column is measured. This genomic DNA is quantified by PCR using primers in the E. coli galK gene.- According to the following protocol: The sequence of these primers is described by Debouck et al. (Nucleic Acids Res. 1985,13,_1841-1853): 5'-CCG AAT TCT GGG GAC CAA AGC AGT TTC-31 (SEQ ID NO: 24) and 5'-CCA AGC TTC ACT GTT CAC GAC GGG TGT-3' (SEQ ID NO: 25). The reaction medium comprises, in. 25 ul of PCR buffer (Promega France, Charbonnieres): 1.5 mM MgC^; 0.2 mM dXTP (Pharmacia, Orsay); 0.5 uM primer; 20 U/ml Taq polymerase (Promega). The reaction is performed according to the sequence:<br>
-	5 min at 95°C<br>
-	30 cycles of 10 sec at 95°C<br>
30 sec at 60°C 1 min at 78°C<br>
-	10minat78°C.<br>
The amplified DNA fragment 124 base pairs in length is separated by electrophoresis on 3 % agarose gel in the presence of SybrGreen I (Molecular Probes, Eugene, USA), and then quantified by reference to an Ultrapur genomic DNA series from E. coli strainB (Sigma, ref D4889).<br>
There is 1 % of chromosomal DNA in the sample applied to the column, and 0.2 % in the sample purified on the oligonucleotide column.<br>
Example 3. Experiment on clear lysate<br>
This example describes plasmid DNA purification from a clear lysate of bacterial culture, on the so-called "miniprep" scale: 1.5 ml of an overnight culture of DH5a strains containing plasmid pXL2563 are centrifuged, and the pellet is resuspended in 100 \i\ of 50 mM glucose. 25 mM Tris-HCl, pH 8,10 mM EDTA. 200 ul of 0.2 M NaOH, 1 % SDS are added, the tubes are inverted to mix, 150 ul of 3 M potassium acetate, pH 5 are then added and the tubes are inverted to mix. After centrifugation, the supernatant is recovered and loaded onto the oligonucleotide column obtained as described in Example 1. Binding, washes and elution are identical to those described in Example 1. Approximately 1 ug of plasmid is recovered from 1.5 ml of culture. The plasmid obtained, analysed by agarose gel electrophoresis and ethidium bromide staining, takes the form of a single band of "supercoiled" circular DNA. No trace of high molecular weight (chromosomal) DNA or of RNA is detectable in the plasmid purified by this method. The ratio of the optical densities at 260 and 280 nm is greater than 2.<br>
Example 4<br>
4.1: This example describes a plasmid DNA purification experiment carried out under the same conditions as Example 3, starting from 20 ml of bacterial culture of DH5a strains containing plasmid pXL2563. The cell pellet is taken up in 1.5 ml of 50 mM glucose, 25 mM Tris-HCI, pH 8,10 mM EDTA. Lysis is carried out with 2 ml of 0.2 M NaOH, 1 % SDS, and neutralization with 1.5 ml of 3 M potassium acetate, pH 5. The DNA is then precipitated with 3 ml of 2-propanol, and the pellet is taken up in 0.5 ml of 0.2 M sodium acetate, pH 5, 0.1 M NaCl and loaded onto the oligonucleotide column obtained as described in Example 1. Binding, washing of the column and elution are carried out as described in Example 1, except for the washing buffer, the molariry of which with respect to NaCl is 0.1M. Approximately 16 p.g of plasmid DNA are obtained. The plasmid obtained, analysed by agarose gel electrophoresis and ethidium bromide staining, takes the form of a single band of "supercoiled" circular DNA. No trace of high molecular weight (chromosomal) DNA or of RNA is detectable in the purified plasmid. Digestion of the plasmid with a restriction enzyme gives a single band at the expected molecular weight of 3<br>
kilobases. The protein concentration in the samples falls from 125 ug/ml in the clear lysate to less than 1 jig/ml in the purified plasmid (Micro-BCA assay, Pierce). The endotoxin concentration, estimated by LAL assay (Biosepra) is divided by a factor of greater than 10 in the purified plasmid, relative to the starting clear lysate.<br>
4.2: The plasmid used contains a cassette containing the cytomegalovims promoter, the gene coding for luciferase and the homopurine-homopyrimidine sequence (GAA)n (SEQ ED NO: 33) originating from plasmid pXL2563. The strain DH1 (Maniatis et al., 1989) containing this plasmid is cultured in a 7-litre fermenter. A clear lysate is prepared from 200 grams of cells: the cell pellet is taken up in 2 litres of 25 mM Tris, pH 6:8, 50 mM glucose, 10 mM EDTA, to which 2 litres of 0.2 M NaOH, 1 % SDS, are added. The lysate is neutralized by adding one litre of 3M potassium acetate. After diafiltration, 4 ml of this lysate are applied to a 5 ml HiTrap-NHS column coupled to the oligonucleotide of sequence 5'- (SEQ ID NO: 1), according to the method described in Example 1.1. Washing and elution are carried out as described in Example 1. Approximately 400 micrograms of plasmid are recovered. The level of genomic DNA in this sample, measured by the technique described in Example 2.2, is 0.1%. Example 5: Use of a modified oligonucleotide<br>
This example describes the use of an oligonucleotide bearing methylated cytosines. The sequence of the oligonucleotide used is as follows: 5'  (SEQ ID NO: 12)<br>
This oligonucleotide possesses an NHa group at the 5' end. MeC = 5-methylcytosine. This oligonucleotide enables plasmid pXL2563 to be purified under the conditions of Example 1 with a binding buffer of pH 5 (the risk of degradation of the plasmid is thereby decreased).<br>
Example 6<br>
In the above examples, the oligonucleotide used is modified at the 5'-terminal end with an amine group linked to the phosphate through an arm containing 6 carbon atoms: NH2-(CH2)6. In this example, the amine group is linked to the phosphate of the 5'-terminal end through an arm containing 12 carbon atoms: NH2-(CH2)i2.<br>
Coupling of the oligonucleotide and passage through the column are carried out as described in Example 2 with a buffer F: 2 M NaCl, 0.2 M acetate, pH 4.5. This oligonucleotide makes it possible to have better purification yields: a 53 % yield is obtained, whereas, with the oligonucleotide containing 6 carbon atoms, this yield is of the order of 45 % under the same conditions.<br>
Example 7<br>
Following the cloning strategy described in Example 1.2, another two plasmids carrying homopurine-homopyrimidine sequences were constructed: the plasmid pXL2725 which contains the sequence (GGA)is, (SEQ ID NO: 34) and the plasmid pXL2726 which contains the sequence (GA)2s (SEQ ID NO: 35). Example 7.1: Construction of the plasmids<br>
Plasmids pXL2725 andpXL2726, analogous to plasmid pXL25 63, were constructed according to the cloning strategy described in Example 1.2, using the following oligonucleotide pairs: 5986: 5'-GATCC(GA)25GGG-3' (SEQ ID NO: 13) 5987: 5'-AATTCCC(TC)25G-3' (SEQ ID NO: 14) 5981: 5'-GATCC(GGA)i7GG-3' (SEQ ID NO: 15) 5982: 5'-AATT(CCT)i7CCG-3' (SEQ ID NO: 16)<br>
The oligonucleotide pair 5986 and 5987 was used to construct plasmid pXL2726 by cloning the oligonucleotides at the BamHI and EcoRI sites of pBKS+ (Stratagene Cloning System, La Jolla CA), while the oligonucleotides 5981 and 5982 were used for the construction of plasmid pXL2725. The same experimental conditions as for the construction of plasmid pXL2563 were used, and only the oligonucleotide pairs were changed. Similarly, the cloned sequences were verified by sequencing on the plasmids. This enabled it to be seen that plasmid pXL2725 possesses a modification relative to the expected sequence: instead of the sequence GGA repeated 17 times, there is GGAGA(GGA)i5 (SEQ ID NO: 17). Example 7.2: Preparation of the columns and Purification The oligonucleotides forming triple helices with these homopurine sequences were coupled to HiTrap columns according to the technique described in Example 1,1. The oligonucleotide of sequence 5'-(SEQ ED NO: 18) was used for the purification of plasmid pXL2725, and the oligonucleotide of sequence<br>
(SEQ ID NO: 19) was used for the purification of plasmid pXL2726.<br>
The two columns thereby obtained enabled the corresponding plasmids to be purified according to the technique described in Example 2, with the following buffers:<br>
Buffer F: 2 M NaCl, 0.2 M acetate, pH 4.5.<br>
Buffer E: 1 M Tris-HCl, pH 9, 0.5 mM EDTA. The yields obtained are 23 % and 31 % for pXL2725 and pXL2726, respectively.<br>
Example 8<br>
This example illustrates the influence of the length of the specific sequence present in the plasmid on the purification yields.<br>
Example 8.1: Construction of the Plasmids<br>
The reporter gene used in these experiments to demonstrate the activity of the compositions of the invention is the gene coding for luciferase (Luc).<br>
The plasmid pXL2621 contains a cassette containing the 661-bp cytomegalovirus (CMV) promoter, extracted frompcDNA3 (Invitrogen Corp., San Diego, CA) by cleavage with the restriction enzymes MM and Hindin, cloned upstream of the gene coding for luciferase, at the Mlul and Hindlll sites, into the vector pGL basic Vector (Promega Corp., Madison, WI). This plasmid was constructed using standard techniques of molecular biology.<br>
The plasmids pXL2727-l and pXL2727-2 were constructed in the following manner:<br>
Two micrograms of plasmid pXL2621 were linearized with BamHI; the enzyme was inactivated by treatment for 10 min at 65°C; at the same tune, the oligonucleotides 6006 and 6008 were hybridized as described for the construction of plasmid pXL25 63.<br>
This hybridization mixture was cloned at the BamHI ends of plasmid pXL2621 and, after transformation into DH5a, recombinant clones were identified by Pstl enzymatic restriction analysis, since the oligonucleotides introduce a Pstl site. Two clones were selected, and the nucleotide sequence of the cloned fragment was verified using the primer (6282, 5'-ACAGTCATAAGTGCGGCGACG-3' (SEQ ID NO: 22)) as a sequencing reaction primer (Viera J. and J. Messing, 1982. The pUC plasmids an M13mp7-derived system for insertion mutagenesis and sequencing with synthetic universal primers. Gene 19:259-268).<br>
The first clone (pXL2727-l) contains the sequence GAA repeated 10 times. The second (pXL2727-2) contains the sequence 5l-<br>
Example 8.2: Preparation of the columns and Purification<br>
A column such as the one described in Example 1, and which is coupled to the oligonucleoride (SEQ ED NO: 1), is used.<br>
The plasmid pXL2727-l carries 14 repeats of the sequence GAA. The oligonucleotide described above, which contains only 7 repeats of the corresponding hybridization sequence CTT, can hence hybridize with the plasmid at 8. different positions. Plasmid pXL2727-2, in contrast, possesses a hybridizing sequence (GAA)? (SEQ ID NO: 36) of the same length as that of the oligonucleotide bound to the column. This oligonucleotide can hence hybridize at only one position on pXL2727-2.<br>
The experiment is identical to the one described in Example 2, with the following buffers:<br>
Buffer F: 2 M NaCl, 0.2 M acetate, pH 4.5.<br>
Buffer E: IM Tris-HCl, pH 9, 0.5 mM EDTA.<br>
The purification yield is 29 % with plasmid pXL2727-l and 19 % with pXL2727-2.<br>
Example 8.3: in vitro transfection of mammalian cells<br>
The cells used are NTH 3T3 cells, inoculated on the day before the experiment into 24-well culture plates on the basis of 50,000 cells/well. The plasmid is diluted in<br>
150 mM NaCl and mixed with the lipofectant RPR115335. A lipofectant positive charges/DNA negative charges ratio equal to 6 is used. The mixture is vortexed, left for ten minutes at room temperature, diluted in medium without foetal calf serum and then added to the cells in the proportion of 1 u# of DNA per culture well. After two hours at 37°C, 10 % volume/volume of foetal calf serum is added and the cells are incubated for 48 hours at 37°C in the presence of 5 % of CO2. The cells are washed twice with PBS and the luciferase activity is measured according to the protocol described (Promega kit, Promega Corp. Madison, WT) on a Lumat LB9501 luminometer (EG and G Berthold, Evry). Plasmid pXL2727-l, purified as described in Example 8.2, gives transfection yields twice as large as those obtained with the same plasmid purified using the Wizard Megaprep kit (Promega Corp. Madison, WT). Example 9: Purification of pCOR-derived plasmids<br>
The following example demonstrates the purification of pCOR-derived plasmids using triple-helix affinity chromatography. This technology has been shown to remove nucleic acid contaminants (particularly host genomic DNA and RNA) down to levels that have not been achieved with conventional chromatography methods,<br>
A triplex affinity gel was synthesized with Sephacryl S-1000 SF (Amersham-Pharmacia Biotech) as the chromatography matrix. Sephacryl S-1000 was first activated with sodium m-periodate (3 mM, room temperature, 1 h) in 0.2 M sodium acetate (pH 4.7). Then the oligonucleotide was coupled through its S'-NBk terminal moiety to aldehyde groups of the activated matrix by reductive amination in the presence of ascorbic acid (5 mM) as described previously for the coupling of proteins (Hornsey et al., J. Immunol. Methods, 1986, 93, 83-88). The homopyrimidine oligonucleotide used for these experiments (from Eurogentec, HPLC-purified) had a sequence which was complementary to a short 14-mer homopurine sequence (5'- (SEQ ID NO: 29) present in the origin of replication (oriy) of the pCOR plasmid (Soubrier et al., Gene Therapy, 1999, 6, 1482-1488). As discussed above, the sequence of the homopyrimidine oligonucleotide is S'- (SEQ ID NO: 30).<br>
The following plasmids were chromatographed: pXL3296 (pCOR with no transgene, 2.0 kpb), pXL3179 (pCOR-FGF, 2.4 kpb), pXL3579 (pCOR-VEGFB, 2.5<br>
kbp), pXL3678 (pCOR-AFP, 3.7 kbp), pXL3227 (pCOR-lacZ 5.4 kbp) andpXL3397 (pCOR-Bdeleted FVTII, 6.6 kbp). All these plasmids were purified by two anion-exchange chromatography steps from clear lysates obtained as described in example 4. Plasmid pBKS+ (pBluescript n KS + from Stratagene), a ColEl-derived plasmid, purified by ultracentrifugation in CsQ was also studied. All plasmids used were in their supercoiled (&gt; 95 %) topological state.<br>
In each plasmid DNA purification experiment, 300 |ig of plasmid DNA in 6 ml of 2 M NaCl, 0.2 M potassium acetate (pH 5.0) was loaded at a flow rate of 30 cm/h on an affinity column containing the above-mentioned oligonucleotide S'- (SEQ ED NO: 30). After washing the column with 5 volumes of the'same buffer, bound plasmid was eluted with 1 M Tris/HCl, 0.5 mM EDTA (pH 9.0) and quantitated by UV (260 nm) and ion-exchange chromatography with a Millipore Gen-Pak column (Marquet et al., BioPharm, 1995, 8,26-37). Plasmid recoveries in the fraction collected were 207 p,g for pXL3296,196 fig for PXL3179, 192 ug for pXL3579,139 ^g forpXL3678, 97 \ig for pXL 3227, and 79 pig forpXL3397.<br>
No plasmid binding could be detected (
As a control, no plasmid binding (
By operating this affinity purification column in the conditions reported here, the level of contamination by host genomic DNA was reduced from 2.6 % down to 0.07 % for a preparation of pXL3296. Similarly the level of contamination by host DNA was reduced from 0.5 % down to 0.008 % for a preparation of pXL3179 when the sample was chromatographed through the same affinity column. In addition, the<br>
level of contamination by RNA was largely reduced from 43 % RNA down to 0.2 % RNA in a prep aration of pXL3179 by using this affinity purification column.<br>
In addition, plasmid PXL3579 recovery was less than 8% when oligonucleotide (SEQ ID NO: 30) was replaced by oligonucleotide 5'- (SEQ ID NO: 31) on the affinity column. While the oligonucleotide as set forth in SEQ ID NO: 31 is complementary to a portion of the VEGFB sequence within pXL3579 (i.e., nucleotides 379 to 389 relative to ATG), no significant triplex affinity occurs. This indicates that this affinity purification requires a non-random homopurine-hornopyrimidine DNA sequence.<br>
Example 10: Purification of a ColEl-derived plasmid<br>
The following example demonstrates the purification of ColEl-derived plasmids using triple-helix affinity chromatography. This technology has been shown to remove nucleic acid contaminants (particularly host genomic DNA and RNA) down to levels that have not been achieved with conventional chromatography methods.<br>
A triplex affinity gel was synthesized by coupling of an oligonucleotide having the sequence (SEQ ID NO: 28) onto periodate-oxidized Sephacryl S-1000 SF as described in Example 9.<br>
Plasmids pXL3296 (pCOR with no transgene) and pBKS, a ColEl-derived plasmid, were chromatographed on a 1-ml column containing oligonucleotide 5'- (SEQ ID NO: 28) in conditions described in Example 9. Plasmid recoveries in the fraction collected were 175 p,g for pBKS and 
Example 11: Double purification method<br>
The following example demonstrates the purification of a supercoiled double-stranded DNA molecule, such as pXL3296, in a mixture containing another supercoiled double-stranded molecule, such as pBSK, using triple helix affinity chromatography. Both double-stranded DNA molecules may have a similar size, but each DNA molecule contains a unique sequence that is capable of forming a triple helix with a different target sequence. As previously discussed, molecules such as pXL3296 contain a sequence (SEQ ID NO: 29), but do not contain the sequence 5'- (SEQ ID NO: 27). In contrast, molecules such as pBSK contain SEQ ID NO: 27, but do not contain SEQ ID NO: 29.<br>
In a first step, the mixture containing pXL3296 and pBSK was loaded on a first affinity column containing the oligonucleotide (SEQ ID NO: 28), such as the column described in Example 10. The solution was passed through the first column which contained unbound DNA molecules. In the second step, the unbound DNA molecules from the first step were loaded on a second affinity column containing the oligonucleotide  (SEQ ID NO: 30), such as the column described in Example 9. The second column was then washed and the bound molecules were eluted, as described in Example 9. Only pXL3296 molecules eluted from the second column. No pBSK molecules were detected in the eluate (i.e., the solution that elutes from the column) from the second column.<br><br>
WE CLAIM:<br>
1.	A method for purifying double-stranded DNA from a solution<br>
containing the double-stranded DNA mixed with other components,<br>
comprising:<br>
(i) passing the solution through a support at an acidic pH, the support comprising a covalently coupled oligonucleotide capable of forming a triple helix with the double-stranded DNA by hybridization at an acidic pH with a specific sequence present in the double-stranded DNA, with the covalently coupled oligonucleotide comprising the sequence TCTTTTTTTCCT (SEQ ID NO:28) or TTCTTTTTTTTCTT (SEQ ID NO:30) and the specific sequence present in the double-stranded DNA comprising the sequence AGAAAAAAAGGA (SEQ ID NO:27) or AAGAAAAAAAAGAA (SEQ ID NO:29); and<br>
(ii) raising the pH so that it is no longer acidic, to elute the bound DNA hence obtaining the double-stranded DNA.<br>
2.	A method as claimed in claim 1, wherein the desired first double-<br>
stranded DNA is purified from a solution containing the first double-stranded<br>
DNA and a second double-stranded DNA, where prior to steps (i) and (ii) the<br>
method comprises:<br>
(a)	passing the solution through a support comprising a covalently<br>
coupled oligonucleotide capable of forming a triple helix with said second<br>
double-stranded DNA by hybridization with a specific sequence therein; and<br>
(b)	recovering the solution that passes through the first support,<br>
with steps (i) and (ii) then being performed on the recovered solution.<br>
3.	The method as claimed in claim 1 or 2, wherein the solution is a cell lysate.<br>
4.	The method as claimed in any one of claims 1 to 3, wherein the specific sequence has been introduced artificially into the double-stranded DNA or is naturally present in the double-stranded DNA.<br>
5.	The method as claimed in any one of claims 1 to 4, wherein the oligonucleotide is coupled to the support through a disulphide, thioether, ester, amide or amine link.<br>
6.	The method as claimed in claim 5, wherein the oligonucleotide is bound to the column via an arm comprising a carbon chain (CH2)n wherein n is an integer between 1 and 18 inclusive, and wherein the arm is linked to the oligonucleotide through a phosphate and to the column through an amide link.<br><br>
7.	The method as claimed in any one of claims 1 to 6, wherein the oligonucleotide possesses at least one chemical modification making it resistant to or protected against nucleases, or increasing its affinity for the specific sequence.<br>
8.	The method as claimed in claim 7, wherein at least one of the cytosines of the oligonucleotide is methylated.<br>
9.	The method as claimed in any one of claims 1 to 8, wherein the double-stranded DNA is a circular DNA.<br>
10.	The method as claimed in claim 9, wherein the circular DNA is a plasmid.<br>
11.	The method as claimed in any one of claims 1 to 10, wherein the specific sequence present in the double-stranded DNA comprises several positions for hybridization with the oligonucleotide.<br>
12.	The method as claimed in any one of claims 1 to 11, wherein the support is a functionalized chromatographic support, a functionalized plastic surface or functionalized latex beads.<br>
13.	The method as claimed in claim 12, wherein the support is a functionalized chromatographic support.<br>
14.	The method as claimed in claim 13, wherein the purified double-stranded DNA has a chromosomal DNA content of less than or equal to 0.5%.<br>
15.	The method as claimed in claim 14, wherein the purified double-stranded DNA has a chromosomal DNA content of less than or equal to 0.01%.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDExNTQtREVMLUFic3RyYWN0LSgwOC0wNS0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-01154-DEL-Abstract-(08-05-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SW4tUGN0LTIwMDItMDExNTQtRGVsLUFic3RyYWN0LTIzLTA0LTIwMDgucGRm" target="_blank" style="word-wrap:break-word;">In-Pct-2002-01154-Del-Abstract-23-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDExNTQtZGVsLWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-01154-del-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDExNTQtREVMLUNsYWltcy0oMDgtMDUtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-01154-DEL-Claims-(08-05-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDExNTQtZGVsLWNsYWltcy0oMTEtMDUtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-01154-del-claims-(11-05-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SW4tUGN0LTIwMDItMDExNTQtRGVsLUNsYWltcy0yMy0wNC0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">In-Pct-2002-01154-Del-Claims-23-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDExNTQtZGVsLWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-01154-del-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDExNTQtREVMLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDgtMDUtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-01154-DEL-Correspondence-Others-(08-05-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDExNTQtREVMLUNvcnJlc3BvbmRlbmNlLU90aGVycy0xMi0wNS0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-01154-DEL-Correspondence-Others-12-05-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SW4tUGN0LTIwMDItMDExNTQtRGVsLUNvcnJlc3BvbmRlbmNlLU90aGVycy0yMy0wNC0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">In-Pct-2002-01154-Del-Correspondence-Others-23-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDExNTQtZGVsLWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-01154-del-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDExNTQtZGVsLWRlc2NyaXB0aW9uIChjb21wbGV0ZSktMDgtMDUtMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-01154-del-description (complete)-08-05-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDExNTQtZGVsLWRlc2NyaXB0aW9uIChjb21wbGV0ZSktMTEtMDUtMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-01154-del-description (complete)-11-05-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SW4tUGN0LTIwMDItMDExNTQtRGVsLURlc2NyaXB0aW9uIChDb21wbGV0ZSktMjMtMDQtMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">In-Pct-2002-01154-Del-Description (Complete)-23-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDExNTQtZGVsLWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-01154-del-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDExNTQtREVMLUZvcm0tMS0oMDgtMDUtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-01154-DEL-Form-1-(08-05-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SW4tUGN0LTIwMDItMDExNTQtRGVsLUZvcm0tMS0yMy0wNC0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">In-Pct-2002-01154-Del-Form-1-23-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDExNTQtZGVsLWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-01154-del-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDExNTQtZGVsLWZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-01154-del-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDExNTQtZGVsLWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-01154-del-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SW4tUGN0LTIwMDItMDExNTQtRGVsLUZvcm0tMi0yMy0wNC0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">In-Pct-2002-01154-Del-Form-2-23-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDExNTQtZGVsLWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-01154-del-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDItMDExNTQtREVMLUZvcm0tMy0oMDgtMDUtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2002-01154-DEL-Form-3-(08-05-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SW4tUGN0LTIwMDItMDExNTQtRGVsLUZvcm0tMy0yMy0wNC0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">In-Pct-2002-01154-Del-Form-3-23-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDExNTQtZGVsLWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-01154-del-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDExNTQtZGVsLWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-01154-del-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SW4tUGN0LTIwMDItMDExNTQtRGVsLUdQQS0yMy0wNC0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">In-Pct-2002-01154-Del-GPA-23-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDExNTQtZGVsLWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-01154-del-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDExNTQtZGVsLXBjdC0zMDYucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-01154-del-pct-306.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SW4tUGN0LTIwMDItMDExNTQtRGVsLVBldGl0aW9uLTEzNy0yMy0wNC0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">In-Pct-2002-01154-Del-Petition-137-23-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SW4tUGN0LTIwMDItMDExNTQtRGVsLVBldGl0aW9uLTEzOC0yMy0wNC0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">In-Pct-2002-01154-Del-Petition-138-23-04-2008.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="221082-a-composition-useful-for-hepatoprotection.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="221084-method-of-obtaining-anti-bacterial-compositions.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>221083</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2002/01154/DEL</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>31/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>01-Aug-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>16-Jun-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>25-Nov-2002</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CENTELION S.A.S.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>72-82 RUE LEON GEFFROY, 94400 VITRY SUR SEINE, FRANCE.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>JOEL CROUZET</td>
											<td>12, RUE MICHEL VOISIN,F-92330 SCEAUX, FRANCE</td>
										</tr>
										<tr>
											<td>2</td>
											<td>PIERRE WILS</td>
											<td>7 RUE NICOLAS HOUEL, 75005 PARIS, FRANCE.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>FRANCIS BLANCHE</td>
											<td>41, RUE DES SOLITARIES, F-75019 PARIS, FRANCE.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>BEATRICE CAMERON</td>
											<td>6, RUE TOURNEFORT, F-75005 PARIS, FRANCE.</td>
										</tr>
										<tr>
											<td>5</td>
											<td>DANIEL SCHERMAN</td>
											<td>10, RUE ERARD,F-75012 PARIS, FRANCE.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/10</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US01/17122</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-05-25</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>09/580,923</td>
									<td>2000-05-26</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/221083-a-method-for-purifying-double-stranded-dna-from-a-solution by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:02:03 GMT -->
</html>
