<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/221070-a-pharmaceutical-composition-for-altering-the-affinity-of-a-nucleic-acid-ligand by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:02:53 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 221070:&quot;A PHARMACEUTICAL COMPOSITION FOR ALTERING THE AFFINITY OF A NUCLEIC ACID LIGAND&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A PHARMACEUTICAL COMPOSITION FOR ALTERING THE AFFINITY OF A NUCLEIC ACID LIGAND&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The biological activity of nucleic acid ligand is regulated (i.d. enhanced or inhibited) in vivo to produce a desired biological effect. This is accomplished through the administration of a modulator, or regulator, that changes the binding of the nucleic acid ligand for its target or that degrades or otherwise ; ; i cleaves, metabolizes or breaks down the nucleic acid ligand while the ligand is i still exerting its effect. Modulators of the present invention can be administered in real time as needed based on various factors, including the progress of the patient, as well as the physician&#x27;s discretion in how to achieve optimal therapy. i Thus, this invention provides for the first time a regulatable therapeutic regime I in the course of nucleic acid ligand therapy.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>MODULATORS OF PHARMACOLOGICAL AGENTS<br>
This application claims priority from U.S. Provisional Application No. 60/293,231, filejl May 25,2001, and U.S. Provisional Application No. 60/33 l,037j filed November 7,2001^ the I entire contents of both applications being incorporated herein by reference.<br>
TECHNICAL FIELD	j<br>
:    !<br>
The present invention relates, in general, to an agent that modulates the pharmacological acdvityofanucleicacidligand(e.g., aptamer) and, m particular, to an agent that enhances or inhibits the activity of such a Ugand. The invention further relates to a composition comprising such an agent and to a method of using these agents and compositions in medical therapeutic and diagnostic procedures.<br>
BACKGROUND<br>
Nucleic acids have conventionally been thought of as primarily playing an informational role in biological processes. Through a method known as Systematic Evolution of Ligands by Exponential enrichment, termed SELEX, it has become clear that nucleic acids have three<br>
!<br>
dimensional structural diversity not unlike proteins. SELEX is a method for the in vitro synthesis and selection of nucleic acid molecules with highly specific binding to target molecules. The SELEX process was first described by Gold and Tuerk in U.S. patent application Ser. No. 07/536,428, filed Jim. 11,1990, now U.S. Pat. No. 5,475,096, and thereafter in U.S. patent application Ser. No. 07/931,473, filed Aug. 17,1992, entitled "Methods for Identifying Nucleic ] Acid Ligands", now U.S. P*. No. 5,270,163 (see also WO 91/19S13). See also Tuerk et al, Science 249:505-10 (1990).<br>
Nucleic acid ligands or aptamers are nonencoding single-stranded nucleic acid (DNA or j RNA) that have the property of binding specifically to a desired target compound or molecule, and that have sufficient capacity for forming a variety of two- and three-dimensional structures<br>
and sufficient chemical versatility available within their monomers to act as ligands (form specific binding pairs) with virtually any chemical compound, whether monomeric or polymeric. Molecules of any size or composition can serve as targets. The SELEX method involves       I<br>
selection from a mixture of candidate oligonucleotides and step-wise iterations of binding,<br>
*	*	i<br>
partitioning and amplification, using the same general selection scheme, to achieve virtually any<br>
desired criterion of binding affinity and selectivity. Starting from a mixture of nucleic acids,  j preferably comprising segments of randomized sequences, the SELEX method includes steps bf contacting the mixture with the target under conditions favorable for binding, partitioning unbound nucleic acids from those nucleic acids that have bound specifically to target molecules, dissociatillg the nucleic acid-target complexes, amplifying the nucleic acids dissociated from the nucleic acid-target complexes to yield a ligand-enriched mixture of nucleic acids, then reiterating the steps of binding, partitioning, dissociating and amplifying through as many cycles as desired to yield highly specific high affinity nucleic acid ligands to the target molecule.<br>
Nucleic acid ligands possess a number of features that can render them useful as<br>
therapeutic agents. They can be made as relatively small (e.g., 8 kDa to 15 kDa) synthetic<br>
compounds and can be selected to possess high affinity and specificity for target molecules<br>
(equilibrium dissociation constants ranging from, for example, 0,05-10 nM). Aptamers embody<br>
both the affinity properties of monoclonal antibodies and single chain antibodies (scFv's) and the<br>
manufacturing ease similar to that of a small peptide. Initial studies demonstrated the in vitro use<br>
of aptamers for studying proton function, and more recent studies have confirmed the utility of<br>
these compounds for studying m vivo protein function (Floege et al, Am J Pathol 154:169-179<br>
(1999), Ostendorf et al, J Clin Invest 104:913-923, (1999)). In addition, animal studies to date<br>
have shown that aptamers and compounds of similar composition are well tolerated, exhibit low<br>
or no immunogenicity, and are thus suitable for repeated administration as therapeutic	•<br>
compounds (Ftoege et al, Am J Pathol 154:169-179 (1999), Ostendorf et al, J Clin Invest 104:913-923 (1999), Griffin et al, Blood 81:3271-3276 (1993), HScke et al, J Clin Invest        j 106:923-928 (2000)).<br>
As synthetic compounds, site specific modifications (insertions or deletions) can be made to aptamers to rationally alter their bioavailability and mode of clearance. For example, it has \ been found that 2'-fluoro pyrimidine-modified aptamers in the 10 kDa to 12 kDa size range have<br>
f short circulating half-life (-10 minutes) following bolus intravenous administration but that   ;<br>
Simple chemical modification of the aptamer or conjugation of the aptamer to a high molecular j<br>
weight inert carrier molecule (e.g., PEG) increases circulating half-life substantially (6-12 hours})<br>
(Willis et al, Bioconjug Chem 9:573-582 (1998), Tucker et al, J Chromatogr Biomed Sci Appl '<br>
*732:203-212 (1999), Watspn et al, Antisense Nucleic Acid Drug Dev 10:63-75 (2000)).<br>
Bioactive and nuclease resistant single-stranded nucleic acid ligands comprising L-nucleotides<br>
have been described (Williams et ai, Proc. Natl. Acad. Sci. 94:11285 (1997); USP 5,780,221;   I<br>
Leva et al, Chem. Biot. 9:351 (2002)). These "L-aptamers" are reportedly stable under	;<br>
conditions in which aptamers comprising nucleotides of natural handedness (D-nucleotides) (that is, "D-aptaraers") are subject to degradation.<br>
A number of third parties have applied for and secured patents covering the identification, manufacture and use of aptamers. As stated above, Larry Gold and Craig Tuerk are generally credited with first developing the SET .EX method for isolating aptamers, and their method is described in a number of United States patents including those mentioned above and U.S. Patent Nos. 5,670,637, 5,696,249, 5,843,653,6,110,900, and 5,270,163, as well as other described in the Detailed Description of the Invention. Thomas Bruice et al reported a process for producing aptamers in U.S. Patent No. 5,686,242, which differs from the original SELEX process reported by Tuerk and Gold because it employs strictly random oligonucleotides during, the screening sequence. The oligonucleotides screened in the '242 patent process lack the oligonucleotide primers that are present in oligonucleotides screened in the SELEX process.<br>
Several patents to Gold et al. relate to aptamers themselves. For example, U.S. Patent No. 6,114,120 relates to an aptamer that binds to a cell macromolecule. U.S. Patent No. 5,670,637 relates to aptamers that bind to proteins. U.S. Patent No. 5,696,249 relates to an aptamer produced by tbe SELEX process.<br>
Other patents have issued that are directed to aptamers against specific biological targets, and to the methods for identifying these aptamers. U.S. Patent No. 5,756,291 and 5,582,981 to O'Toole, for example, disclose a method for detecting thrombin using a labeled aptamer that comprises a defined six nucleotide sequence. U.S. Patent Nos. 5,527,894 and 5,637,461 of Gol^ et al. relate to methods of identifying aptamers against the tat protein. Other patents that disclose I aptamers directed against specific biological targets include U.S. Patent Nos. 5,496,938 (HTV- i<br>
reverse transcriptase), 5,476,766 (thrombin), 5,459,015 (fibroblast growth factor), 5,472..841   i Tfneutrophil elastase), 5,849,479 (vascular endothelial growth factor), 5,726,017 (HTV GAG), ; 5,731,144 (TGFJ3), 5,827,456 (chorionic gonadotropin hormone), 5,780,228 (lectins), 5,766,853 (selectins), 5,674,685 (platelet derived growth factor), 5,763,173 (DNA polymerases), 6,140^400 (complement system proteins), and 5,869,641 \CD4).<br>
Sullenger, Rusconi, Kontos and White in WO 0226932 A2 describe RNA aptamers th£t bind to coagulation factors, E2F family transcription factors, Angl, Ang2: and fragments or    I peptides thereof, transcription factors, autoimmune antibodies and cell surface receptors useJuj in the modulation of hemostasis and other biologic events. (See also Rusconi et al, Thrombosis ahd Haemostasia 83:841-848 (2000), White et al, J. Clin Invest 106:929-34 (2000), Ishizaki et al» Nat Med 2:1386-1389 (1996), and Lee et al, Nat Biotechnol 15:41-45 (1997)).<br>
i<br>
A number of patents have also issued that relate to specific uses of aptamers. For<br>
example, Bruice et al in U.S. Patent No. 6,022,691 describe the use of aptamers identified by a<br>
SELEX-like process to detect drugs and other molecules in biological fluids. Gold et al. in U.S.<br>
Patent No. 5,843,653 provide a diagnostic method using aptamers. U.S. Patent No. 6,110,900 j<br>
discloses a diagnostic composition that contains an aptamer. U.S. Patent No. 5,789,163 discloses<br>
a sandwich assay that employs aptamers as the capture and/or detection ligand. U.S. Patent No.<br>
6,147,204 describes the use of aptamers/lipophile complexes to deliver therapeutic and	;<br>
diagnostic aptamers to intracellular locations in vivo. U.S. Patent Nos. 5,705,337,5,962,219, 5,763,595 and 5,998,142 disclose aptamers that are chemically modified to covalently bind to target proteins.<br>
Several methods have been developed that modify the base SELEX process to obtain j<br>
aptamers that satisfy objectives in addition to exhibiting high binding affinity toward a target<br>
molecule. For example, a number of patents disclose the use of modified nucleotides in the<br>
SELEX process to obtain aptamers that exhibit improved properties. U.S. Patent No. 5,660,985,<br>
for example, relates to SELEX using 2'-modified nucleotides that display enhanced in vivo    stability. U.S. Patent No. 6,083,696 discloses a "blended" SELEX process in which	i<br>
oligonucleotides covalently linked to non-nucleic acid functional units are screened for their j<br>
capacity to bind a target molecule. Other patents describe post-SELEX modifications to aptamers<br>
to decrease their size, increase their stability, or increase target binding affinity. See, e.g., UlSJ.<br>
Patent Nos. 5,817,785 and 5,648,214.         '•	i<br>
Still other patents describe unique SELEX processes. For example, U.S. Patent Nos. •^5,763:566 and 6,114,120 disclose processes for generating aptamers using the SELEX process with whole biological tissue as the target, to identify aptamers that have binding affinity toward biological tissues and components thereof. U.S. Patent No. 5,580,737 discloses a modification! to the SELEX process that yields aptarfters that can discriminate between two or more compourjdf U.S. Patent No. 5,567,588 discloses the "solution SELEX" method in which the nucleic acid  | candidate mixture is screened in solution in order to preferentially amplify the highest affinity aptamer.<br>
Kauffrnan has obtained patents disclosing the generation of large libraries of proteins  j<br>
from large pools of stochastically generated oligonucleotide vectors. See U.S. Patent Nos.<br>
5,814,476 and 5,723,323.	j<br>
Weis et al. disclose in U.S. Patent No. 5,245,022 an oligonucleotide of about 12-25 ba4es that is terminally substituted by a polyalkyleneglycol. These modified oligonucleotides are reported to be resistant to exonuclease activity.<br>
U.S. Patent Nos. 5,670,633 and 6,005,087 to Cook et aL describe thermally stable 2'- !<br>
fluoro oligonucleotides that are complementary to an RNA or DNA base sequence. U.S. Patent<br>
Nos. 6,222,025 and 5,760,202 to Cook et aL describe the synthesis of 2'-0 substituted	j<br>
pyrimidines and oligomers containing the modified pyrimidines. EP 0 593 901 Bl discloses ; oligonucleotide and ribozyme analogues with terminal 3',3'- and 5',5'-nucleoside bonds. U.S. Patent No. 6,011,020 to Gold et al. discloses an aptamer modified by polyethylene glycol.<br>
A number of U.S. patents have issued that describe methods of large scale manufacturing that can be used to produce aptamers. Caruthers et aL, for example, describe in U.S. Patent Nos. 4,973,679; 4,668,777; and 4,415,732 a class of phosphoramidite compounds that are useful in the manufacture of oligonucleotides. In another series of patents, Caruthers et al disclose a method of synthesizing oligonucleotides using an inorganic polymer support. See, e.g., U.S. Patent Nos. 4,500,707,4,458,066 and 5,153,319. In still another series of patents, Caruthers 4* al. discloses a class of nucleoside phosphorodithioates that can be used to manufacture oligonucleotides. See, e.g^, U.S. Patent Nos. 5,278,302, 5,453,496 and 5,602,244. Reports of aptamers designed to bind to other aptamers include: Aldaz-Carroll L, Tallet B, Dausse E, Yurchenko L, Toulme JJ.; Apical loop-internal loop interactions: a new RNA^KK^ recognition motif identified throujgh in vitro selection against KNA hairpins of the hepatitjis C<br>
f<br>
'rus mRNA; Biochemistry. 2002 May 7;41(18):5883-93; Darfeuille F, Cazenave C, Gryaznov^ , Duconge F, Di Primo C, Toutee JJ.; RNA and N3'--&gt;P5' kissing aptamers targeted to the     i trans-activation responsive (TAR) RKA of the human immunodeficiency' virus-1, Nucleosides ; Nucleotides Nucleic Acids. 2001 Apr-Jul;20(4-7):441-9; CoIIin D, van Heijenoort C, Boiziau 0,<br>
•	:<br>
Toulme JJ, Guittet E., NMR characterization of a kissing complex formed between the TAR<br>
RNA element of HIV-1 and a DNA aptamer. Nucleic Acids Res. 2000 Sep 1;28(17):3386-91;   Duconge F, Di Primo C, Toulme JJ., Is a closing "GA pair" a rule for stable loop-loop RNA    |<br>
complexes? J Biol Chem. 2000 Jul 14;275(28):21287-94.; Duconge F, Toulme JJ. In vitro<br>
selection identifies key determinants for loop-loop interactions: RNA aptamers selective for the<br>
TAR RNA element of HIV-1. RNA. 1999 Dec;5(12):1605-14.; Boiziau C, Dausse E, Yurchenko<br>
L, Toulme JJ., DNA aptamers selected against the HIV-1 frans^activation-responsive RNA<br>
element form RNA-DNA kissing complexes; J Biol Chem. 1999 Apr 30;274(18): 12730-7; and<br>
Le Tinevez R, Mishra RK, Toulme II., Selective inhibition of cell-free translation by<br>
oligonucleotides targeted to a mRNA hairpin structure; Nucleic Acids Res. 1998 May<br>
15;26(10):2273-8.	j<br>
Currently, many drugs elicit medical complications such as side effects and undesirable i or uncontrollable outcomes. Treating medical complications that result from side effects leads to additional healthcare costs. The recent identification of this range of nucleic acid ligands useful \ in medical therapy has opened new avenues of research and development. While progress has been made in this area, a strong need remains to provide methods and compositions to improve the manner in which these ligands are used and to increase their efficacy, to better control the process of therapy, and to provide therapies that have decreased side effects over traditional therapeutic methods. The present invention provides compounds, compositions and methods to improve the process of using nucleic acid ligands in medical therapy. The approach provided by the present invention allows for more control over the therapeutic effect, pharmacdkinetics and duration of activity of nucleic acid Ugands.<br>
SUMMARY OF THE INVENTION<br>
It has been discovered that the biological activity of nucleic acid ligands can be modulated (i.e., enhanced or inhibited) in vivo to produce a desired biological effect. This canibb<br>
wus mRNA; Biochemistry. 2002 May 7;41(18):5883-93; Darfeuille F, Cazenave C, Gryaznov I sfDuconge F, Di Primo C, Touhne JJ.; RNA and N3'-&gt;P5' kissing aptamers targeted to the trans-activation responsive (TAR) RNA of the human immunodeficiency virus-1, Nucleosides<br>
Nucleotides Nucleic Acids, 2001 Apr-JuI;20(4-7):441-9; Collin D^, van Heijenoort C, Boiziau C[<br>
•<br>
Toulme JJ, Guittet E,, NMR characterization of a kissing complex formed between the TAR RNA element of HTV-1 and a DNA aptamer. Nucleic Acids Res. 2000 Sep 1;28(17):3386-91; Duconge F, Di Prime C, Toulme JJ., Is a closing "GA pair" a rule for stable loop-loop RNA complexes? J Biol Chem. 2000 Jul 14;275(28):21287-94,; Duconge F, Toulme JJ. In vitro selection identifies key determinants for loop-loop interactions: KNA aptamers selective for the! TAR RNA element of fflV-1. RNA. 1999 Dec;5(12):1605-14.; Boiziau C, Dausse E, YurcheiOoi L, Toulme JJ., DNA aptamers selected against the HTV-1 trans-activation-responsive RNA dement form RNA-DNA kissing complexes; J Biol Chem. 1999 Apr 30;274(18): 12730-7; and Le Tinevez R, Mishra RK, Toulme JJ., Selective inhibition of cell-free translation by oligonucleotides targeted to a mRNA hairpin structure; Nucleic Acids Res. 1998 May 15;26(10):2273-8.<br>
Currently, many drugs elicit medical complications such as side effects and undesirable ! or uncontrollable outcomes. Treating medical complications that result from side effects leads to additional healthcare costs. The recent identification of this range of nucleic acid ligands useful j in medical therapy has opened new avenues of research and development. While progress has been made in this area, a strong need remains to provide methods and compositions to improve i the manner in which these ligands are used and to increase their efficacy, to better control the process of therapy, and to provide therapies that have decreased side effects over traditional therapeutic methods. The present invention provides compounds, compositions and methods to improve the process of using nucleic acid ligands in medical therapy. The approach provided by the present invention allows for more control over the therapeutic effect, pharmacokinetics and duration of activity of nucleic acid ligands.<br>
SUMMARY OF THE INVENTION<br>
It has been discovered that the biological activity of nucleic acid ligands can be modulated (i.e., enhanced or inhibited) in vivo to produce a desired biological effect. This can be<br>
^ccomplished through the administration of a modulator, or regulator, that changes the binding i<br>
of the nucleic acid ligand for its target or that degrades or otherwise cleaves, metabolizes or i<br>
breaks down the nucleic acid ligand while the ligand is still exerting its effect. Modulators of th£<br>
present invention can be administered in real time as needed based on various factors, including<br>
the progress of the patient, as well as the physician's discretion in how to achieve optimal * I<br>
therapy. Thus, this invention provides for the first time a regulatable therapeutic regime in the j<br>
course of nucleic acid ligand therapy.	j<br>
This regulatable therapeutic regime controls drug action by introducing a modulator that is easy to use, can be independent of the patient's health status, has a uniform mode of action,   '•   *
and does not require continuous drug infusion. In one example, an antidote is provided that is  i rationally designed to turn off aptamer activity when desired by the physician.<br>
The modulator can be a oligonucleotide, a small molecule, a peptide, oligosaccharide, for example an aminoglycoside, or other molecule that can bind to or otherwise modulate the       I activity of the therapeutic nucleic add ligand, including a small molecule, or a chimera or fusion or linked product of any of these. For example, the modulator can be an oligonucleotide that is complementary to at least a portion of the nucleic acid ligand. In another embodiment, the modulator can be a ribozyme or DNAzyme that targets the nucleic acid ligand. In a further      ; embodiment, the modulator can be a peptide nucleic acid (PNA), morpholino nucleic acid       | (MNA), locked nucleic acid (LNA) or pseudocyclic oligonucleobases (PCO) that includes a     \ sequence that is complementary to or hybridizes with at least a portion of the nucleic acid ligand. A typical nucleic acid ligand (e.g., aptamer) possesses some amount of secondary structure — its active tertiary structure is dependent on formation of the appropriate stable secondary structure. Therefore, while the mechanism of formation of a duplex between a complementary oligonucleotide modulator of the invention and a nucleic acid ligand is the same as between two short linear oligoribonucleotides, both the rules for designing such interactions and the kinetics of formation of such a product are impacted by the intramolecular aptamer structure. The rate of nucleation is important for formation of the final stable duplex, and the rate of this step is greatly enhanced by targeting the oligonucleotide modulator to single-stranded loops and/or single-stranded 3* or 5' tails present in the nucleic acid ligand. For the formation of the intermolecular duplex to occur, the free energy of formation of the intermolecular duplex has to be favorabl(e  j<br>
with respect to formation of the existing intramolecular duplexes within the targeted nucleic acid nd.<br>
In an alternative embodiment of the invention, the modulator itself is an aptamer. In this : embodiment, a nucleic acid ligand is first generated that binds to the desired therapeutic target. In a second step, a second nucleic acid ligand that binds to the first nucleic acid ligand is generated using the SELEX process described herein or other process, and modulates the interaction between the therapeutic nucleic acid ligand and the target. In one embodiment, the second nucleic acid ligand deactivates the effect of the first nucleic acid ligand.<br>
In other alternative embodiments, the aptamer which binds to the target can be a PNA, MNA, LNA or PCO and the modulator is a nucleic acid ligand. Alternatively, the aptamer whiclj binds to the target is a PNA, MNA, LNA or PCO, and the modulator is a PNA. Alternatively, the aptamer which binds to the target is a PNA, MNA, LNA or PCO, and the modulator is an MNA. Alternatively, the sptamer which binds to the target is a PNA, MNA, LNA or PCO, and the modulator is an LNA. Alternatively, the aptamer which binds to the target is a PNA, MNA, LNA or PCO, and the modulator is a PCO. Any of these can be used, as desired, in the naturally occurring stereochemistry or in non-naturally occurring stereochemistry or a mixture thereof.    ! For example, in a preferred embodiment, the nucleic acid ligand is in the D configuration, and in an alternative embodiment, the nucleic acid ligand is in the L configuration.<br>
The present invention also provides methods to identify the modulators of nucleic acid ligands. Modulators can be identified in general, through binding assays, molecular modeling, or in vivo or in vitro assays that measure the modification of biological function. In one embodiment, the binding of a modulator to a nucleic acid is determined by a gel shift assay. In another embodiment, the binding of a modulator to a nucleic acid ligand is determined by a Biacore assay. Other appropriate assays are described in the Detailed Description of the Invention.<br>
la another embodiment, the binding or interaction of the modulator with the nucleic acid I ligand is measured by evaluating the effect of the nucleic acid ligand with and without the modulator under appropriate biological conditions. As an example, modulators of the invention j can be identified which regulate antithrombotic and anticoagulant aptamers. Modulator efficacy can be assessed in vitro or in vivo through a coagulation test bioasfeay such as the activated<br>
coagulation time test, the activated partial thromboplastin test, the bleeding time test, the ipothrombin time test, or the thrombin clotting time test. Using an identified regime, a patient can be administered an anticoagulant nucleic acid ligand and then given the antidote when the time is appropriate to resume normal clotting action. This regime is useful, for example, during! cardiovascular and vascular surgery, percutaneous coroifiry interventions (angioplasty), orthopedic surgery, and treatment of acute myocardial infarction. In a non-limiting, illustrative example, modulators of the present invention can bind to nucleic acid ligands that target tissue factor (TF)/factor VHa (FWa), factor VHIa (FVma)/factor Ka (FDCa), factor Va (FVa/factor Xa (Fxa) enzyme complexes and platelet receptors such as gp Ubllla and gp IbDC and modulate the effects of the nucleic acid ligand. This invention also provides antidote controlled platelet inhibitors, antithrombotics and fibrinolytics.<br>
In a one embodiment, the modulator is an oligonucleotide that binds to a Factor EXa aptamer (for example, Aptamer 9.3 or Aptamer 9.3t) that targets Coagulation Factor DCa. The antidote oligonucleotide can be complementary to at least a portion of the Factor IXa aptamer. Specifically, the antidote 2'-0-methyl oligonucleotide can consist of the folio-wing sequence, 5'AUGGGGAGGCAGCAUUA3', 5'CAUGGGGAGGCAGCAUUA3', S'CAUGGGGAGGCAGCAS', 5'CAUGGGGAGGCA3',<br>
5'GCAUUACGCGGUAUAGUCCCCUA3', 5'CGCGGUAUAGUCCCCUA3', 5'CGC GGU AUA GUC CCC AU31. Modifications or derivatives thereof wherein a desired degree of hybridization is maintained.<br>
In another embodiment, the modulator is an oligonucleotide that binds to a Factor Xa aptamer (for example, Aptamer 1 lF7t) that targets Coagulation Factor Xa. The antidote oligonucleotide can be complementary to at least a portion of the Factor Xa aptamer. Specifically, the antidote oligonucleotide may consist of the following sequences, 5'CUCGCUGGGGCUCUC3', 5'UAUUAUCUCGCUGGG3', S'AAGAGCGGGGCCAAGS', 5'GGGCCAAGUAUUAU 3', 5' CAAGAGCGGGGCCAAG 3', 5'CGAGUAUUAUCUUG3' j or any modification or derivative thereof wherein a desired degree of hybridization is maintained.<br>
In another embodiment, the oligonucleotide modulators include nucleic acid sequences!<br>
That are substantially homologous to and that have substantially the same ability to bind nuclei^<br>
acid ligands, as the oligonucleotide modulators identified herein.	'<br>
In a further embodiment, modulators of Jhe invention can also be used to reverse the    j<br>
immunosuppressive effect of nucleic acid ligands that target interleukin, fdr example, in patiehts<br>
i<br>
subject to infection. The present modulators can be used to reverse the immunostimulatory    I effects of nucleic acid ligands that target CTLA4 in patients at risk of developing autoimmunjty.<br>
In a further embodiment, modulators of the invention can be used to reverse the effects of aptamers that target growth factors (e.g., PDGF or VEGF). Such nucleic acid ligands can be   \ used in the treatment of tumors and in the treatment of inflammatory proliferative diseases.     j Since growth factors play systemic roles in normal cell survival and proliferation, nucleic acid ! iigand treatment can result in a breakdown of healthy tissue if not tightly regulated (e.g., patients receiving nucleic acid ligands that target angiopoietin I can be subject to hemorrhaging). Modulators of the invention that target such nucleic acid ligands can be used to provide the     I necessary regulation.<br>
Modulators of the invention can be used to reverse the effects of nucleic acid ligands that. target receptors involved in the transmission of the nerve impulse at the neuromuscular junction of skeletal muscle and/or autonomic ganglia (e.g., nicotinic acetylcholine or nicotinic cholinergjc receptors). Such nucleic acid ligands can be made to produce muscular relaxation or paralysis during anesthesia. Agents that block the activity of acetylcholine receptors (agents that engender neuromuscular blockade) are commonly used during surgical procedures, and it is preferred that the patients regain muscular function as soon as possible after the surgical procedure is complete to reduce complications and improve patient turnover hi the operating arenas. Therefore, much j effort has been made to generate agents with predictable pharmacokinetics to match the duration of the drug activity to the anticipated duration of the surgical procedure. Alternatively, modulators of the invention that target such nucleic acid ligands can be used to provide the desired control of the activity of the neuromuscular blocker, and thus reduce the dependence on the patient's physiology to provide reversal of the neuromuscular blocking agent.<br>
In a still further embodiment, modulatop of the invention can be used to reverse the effect of nucleic acid ligands that target small molecules, such as glucose. Hypoglycemia can be<br>
avoided in patients receiving glucose-targeted nucleic acid ligands to regulate glucose uptake Wig the modulators of the invention.<br>
Further, modulators can also be used to regulate the activity of nucleic acid ligands directed against members of the E2F family, certain of which are pro-proliferative, certain of<br>
*<br>
which are repressive. The modulators of the invention can be used to "turn on" and "turn off'<br>
such nucleic acid ligands at desired points in the cell cycle.	<br>
In another embodiment, modulators of the invention can also be used to reverse the binding of nucleic acid ligands bearing radioactive or cytotoxic moieties to target tissue (e.g., neoplastic tissue) and thereby, for example, facilitate clearance of such moieties from a patient's i system. Similarly, the modulators of the invention can be used to reverse the binding of nucleic acid ligands labeled with detectable moieties (used, for example, in imaging or cell isolation or • sorting) to target cells or tissues (Tfficke et al. J. Clin. Invest. 106:923 (2000); Ringquist et ai, Cytometry 33:394 (1998)). This reversal can be used to jexpedite clearance of the detectable moiety from a patient's system.<br>
Modulators of the invention can also be used in in vitro settings to enhance or inhibit the ! effect of a nucleic acid ligand (e.g., aptamer) on a target molecule. For example, modulators of j the invention can be used in target validation studies. Using modulators of the invention, it is possible to confirm that a response observed after inhibiting a target molecule (with a nucleic acid ligand) is due to specifically inhibiting that molecule.<br>
The modulators of the invention can be formulated into pharmaceutical compositions thai can include, in addition to the modulator, a phannaceutically acceptable carrier, diluent or excipient. The precise nature of the composition will depend, at least in part, on the nature of the modulator and the route of administration. Optimum dosing regimens can be readily established by one skilled in the art and can vary with the modulator, the patient and the effect sought. Generally, the modulator can be administered IV, IM, IP, SC, or topically, as appropriate.<br>
Alternatively, and in view of the specificity of the modulators of the invention,	<br>
subsequent treatment can involve the administration of additional nucleic acid ligands that are either the same or different as the original nucleic acid ligand/antidote oligonucleotide pair first administered.<br>
Finally, one optional embodiment of the invention is the identification and selection of!<br>
and regulators that exhibit relevant species cross-reactivity to increase usefulness of<br>
preclinical animal studies.	;<br>
Objects and advantages of the present invention will be clear from the description that i<br>
•	•	:<br>
follows:<br>
Figure 1 shows the FDCa aptamer 9.3t (SEQ ID NO: 1). All purines are 2' -hydroxyl anc( all pyrimidines are 2- fluoro nucleotides.<br>
Figure 2 shows the anticoagulant activity of aptamer 9.3t Clot time increase is normalized to baseline clot time in the absence of aptamer.<br>
Figure 3 shows ACT change of pigs following aptamer injection. Data are normalized to pre-injection baseline.<br>
Figure 4 shows the clot time change of pigs following aptamer injection.<br>
Figures 5A-5C show the in vitro inhibitory activity of cholesterol-modified aptamer, 9.3t-C. Fig. 5 A. Cholesterol addition has a modest effect on the affinity of aptamer 9.3t-C for FDCa.1 A competition binding assay is used to measure affinity of 9.3t-C for FDCa. Fig. SB. In vitro anti-coagulant activity of aptamer 9.3t-C in human plasma. Fig. 5C. In vitro anticoagulant activity of aptamer 9.3t-C in pig plasma.<br>
Figures 6A-6C show the in vivo anticoagulant activity of aptamer 9.3t-C. Fig. 6A. In vivo anticoagulant activity of aptamer 9.3t-C in pigs following IV bolus injection, ACT assays (dotted line is 9.3t ACT data at 0.5 mg/ml from Fig. 3). Fig. 6B. In vivo anticoagulant activity of aptamer 9.3t-C in pigs following IV bolus injection, APTT and PT assays (0.5 rag/kg 9.3t<br>
i	i<br>
from Fig. 4). Fig. 6C. In vivo plasma concentration of 9.3t-C versus 9.3t over time following<br>
Jaohis IV injection. Concentrations were calculated by interpolation from in vitro dose response^<br>
curves of APTT assays for each aptamer.	<br>
Figure 7 shows alignment of minimal FDCa aptamers (SEQ ID N0:2 - SEQ ID NO: 17).!<br>
•	•       !<br>
Sequences in tower case letters are derived from the fixed region of the library used in the SELEX process and sequences in upper case letters are derived from the random region of the library used in the SELEX process. S = stem, L = loop.<br>
Figure 8 shows the native ge! analysis of binding of antidote oligonucleotide to aptamer 9.3t. The ability of the antidote oligonucleotide to bind to and denature aptamer 9.3t was evaluated by native gel electrophoresis. Briefly, radiolabeled 9.3t (125 nM) was incubated at   ! 37°C for 15 minutes with (from left to right) an 8 fold, 4 fold, 2 fold molar excess or equimolar amounts of the antidote (A.S.) or nonsense oligonucleotide (N.S.). Native 9.3t migrates faster than antidote bound 9.3t in this gel system (compare lanes 1 and 2).<br>
Figures 9 A and 9B show antidote oligonucleotide reversal of anticoagulant activity of aptamer 9.3t in human plasma. Fig. 9 A. Change in clot time versus antidote or nonsense oligonucleotide concentration. A value of 1.0 indicates no change hi clot time over the baseline value. 9.3tM is a mutant version of aptamer 9.3t that has no anticoagulant activity. Fig. 9B.    i Fraction of the anticoagulant activity of aptamer 9.3t reversed versus the molar excess of        | antidote oligonucleotide.<br>
Figure 10A-IOC show the specificity of oligonucleotide antidotes. Fig. 10A. Minimal \ secondary structure of aptamer 9.20t (SEQ ED NO: 18). Fig. 10B Anticoagulant activity of 9.2Qt Fig. IOC. Specificity of antidote oligonucleotide Anti D1.<br>
Figure 11 shows the secondary structure of tailed aptamer 9.3t-3NT (SEQ ID NO: 19); Also shown are antidote oligonucteotides (SEQ ID N0:20 - SEQ ID N0:22).<br>
Figure 12A and 12B show the activity of aptamer 9.3t-3NT. Fig. 12A. Competitive finding data. Fig. \28.Invitro anticoagulant data.<br>
Figures 13A and 13B show the in vitro reversal of the anticoagulant activity of aptamer 9.3t-3NT. •Fig. 13 A. Reversal of anticoagulant activity vs. concentration of antidote oligonucleotide. Fig. 13B, Reversal of anticoagulant activity vs. molar excess of antidote over! aptamer.<br>
Figures 14A and 14B show the impact of tail addition to FDCa aptamer 9.3t oh the ability to reverse anticoagulant activity.<br>
Figure 15. The antidote oligonucleotide 5-2C but not a scrambled version of-this antidote<br>
oligonucleotide, 5-2C scr, effectively reverses the activity of aptamers 9.3t and Peg-9.3t in<br>
human plasma.	;<br>
Figure 16. Kinetics of antidote activity in human plasma, as described in Example 6. Figure 17. Duration of antidote activity in vitro, as described in Example 6.<br>
Figures ISA and 18B. Fig. ISA. Predicted secondary structure of aptamer 1 IFTt (SEQI ID NO:23), which binds human coagulation factor Xa with a Kp of ~1.5 nM. Fig. 1 SB. Predicted secondary structure of a mutant version of aptamer 1 lF7t, termed I IFTtM (SEQ ID NO:24).<br>
Figures 19A and 19B. Aptamer 1 lF7t is a potent anticoagulant of human plasma. Varying concentrations of aptaraer 1 lF7t were added to human plasma m vitro, and the clot time was then measured in a PT (Fig. 19A) or APTT assay (Fig. 19B). All data are normalized to the baseline for that day, so that a value of 1 equals no change in clot time.<br>
i<br>
Figure 20. Sequences of 1 lF7t to which the antidote oligos! are complimentary.<br>
Figures 21A and 21B. Fig. 21 A. The antidote oligonucleotides effectively reverse the of aptamer 1 lF7t in human plasma. Fig. 21B. Characterization of antidote 5-2 activity over a larger concentration range of antidote 5-2, and comparison to the antidote activity of a<br>
scrambled sequence version of antidote 5-2, 5-2 scr.<br>
•<br>
•<br>
Figure 22. Kinetics of antidote activity in human plasma. Aptamer 1 lF7t was added to human plasma in vitro at a final concentration of 125 nM, and allowed to incubate for 5 minutes; at 37"C. Antidote oligonucleotide 5-2 at a 1:1 or 5:1 molar excess or no antidote were then added, and the clotting activity of the plasma determined by measuring the clotting time in a PT assay at the times indicated following antidote addition. The % residual anticoagulant activity remaining was calculated by comparing the difference over baseline between the clotting time in the presence of antidote to the difference over baseline in the absence of antidote at each time point. The data collected at a 5:1 molar excess of antidote 5-2 to aptamer 1 lF7t is not shown, as reversal of the anticoagulant activity was complete at the first time point (1 minute).<br>
Figure 23. Duration of antidote activity m vitro. The duration of the inactivation of the anticoagulant activity of aptamer llF7t by antidote oligonucleotide 5-2 was measured in      3 in human plasma. Briefly, 1 lF7t was added to human plasma to a final concentration of 125 nM and allowed to incubate for 5 minutes. Antidote oligonucleotide 5-2 was then added at a 4 fold \ molar excess, or hi a parallel experiment buffer alone was added in place of the antidote oligo, and the clotting time was measured in an a PT assay at various time points following antidote addition. All data is normalized to the baseline for that day, so that a value of 1s no change in clot time. It was found that after 5 hours of incubation at 37'C, the PT of the untreated plasma began to increase, indicating the loss of the clot forming activity of the plasma, and the experiment was thus stopped at 5 hours.<br>
Figure 24. Aptamers 9.3t and 1 lF7t and their respective antidotes function independent of each other in human plasma. Aptl«=9.3t (30 nM), Apt2=l lF7t (100 nM), AD1=AO 5-2c (300 nM), AD2»BA05-2 (500 nM)-final concentrations in plasma in 0- Aptamers were added to •human plasma at 37'C as indicated, and allowed to incubate for 5 minutes. Antidotes were then added as indicated, and the clotting activity was measured 10 minutes after antidote addition in<br>
APTT assays. In all assays, buffer alone was substituted for aptamer or antidote in cases in h only one aptamer or antidote was added to the plasma. All data is normalized to the baseline for that day, so that a value of 1 - no change in clot time.<br>
Figures 2^A - 25F. Antidote-controlled anticoagulation of plasma from patients with ! heparin-bduced thrombocytopenia. Figs. 25A-25C. The activity of aptamer Peg-9.3t and      ; antidote 5-2 were tested in plasma from hemodialysis-dependent patients diagnosed with HIT. j Fig. 25D-25F. The activity of aptamer Peg-9.3t and antidote 5-2 were tested in plasma from   i patients suffering from thromboembolic complications of BIT. Plasma samples were treated as indicated: aptamer, 125 nMPeg-9.3t; antidote, 1.25 uM AO 5-2; mutant aptamer, 125 nM 9.3t^l Experiments were performed as described in Example 2, Figure 9. Data is reported in seconds (s) and is the average ± range of duplicate measurements.<br>
Figures 26A and B. Antidote-controlled anticoagulation of plasma from patients with heparin-induced thrombocytopenia. The activity of aptamer 1 lF7t and antidote 5-2 were tested! in plasma from a hemodialysis-dependent patient diagnosed with HIT and from a patient        i suffering from thromboembolic complications of HIT. For patient 3, plasma samples were treated as indicated: aptamer, 250 nM 1 lF7t; antidote, 1.0 uM AO 5-2; mutant aptamer, 250 aKd 9.3tM. Experiments were performed as described hi Example 2, Figure 9. For Patient 6, plasma samples were treated as indicated: aptamer, 125 nMllFVt; antidote, 250 nMAO 5-2; mutant   \ aptamer, 125 nM 9.3tM. Data is reported in seconds (s) and is the average ± range of duplicate; measurements.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The present invention relates generally to modulators of pharmacological agents, including therapeutic and diagnostic agents. The invention further relates to methods of enhancing or inhibiting the efficacy of pharmacological agents by administering modulators of i the invention to a subject (e.g., a human) in need thereof. Additionally, the invention relates to methods of using modulators of the invention to assess the activity of nucleic acid ligands, in vivo and in vitro.<br>
The present invention relates to a method of modulating the activity of a nucleic acid    |<br>
, for example, by altering Hs conformation and thus its function. In accordance with the | invention, the modulator can be contacted with the targeted nucleic acid ligand under condition^ such that it binds to the nucleic acid ligand and modifies the interaction between the nucleic aoi<jl ligand and its target molecule. modification of that interaction can result from erf j the nucleic acid structure as a binding by modulator. modulator bind free bound to></jl>
Modulators of the invention can be designed so as to bind any particular nucleic acid ligand with a high degree of specificity and a desired degree of affinity. Modulators can be also be designed so that, upon binding, the structure of the nucleic acid ligand is modified to either a more or less active form. For example, the modulator can be designed so that upon binding to i the targeted nucleic acid ligand, the three-dimensional structure of that nucleic acid ligand is altered such that the nucleic acid ligand can no longer bind to its target molecule or binds to its ! target molecule with less affinity.<br>
Alternatively, the modulator can be designed so that, upon binding, the three dimensional structure of the nucleic acid ligand is altered so that the affinity of the nucleic acid ligand for its<br>
'•    target molecule is enhanced. That is, the modulator can be designed so that, upon binding, a \ structural motif is produced in the nucleic acid ligand so that the nucleic acid ligand can bind to its target molecule.<br>
In one embodiment, the modulator is an oligonucleotide. The oligonucleotide can be a sequence that is complementary to at least a portion of the nucleic acid ligand. In another embodiment, the modulator is a ribozyme or DNAzyme that targets the nucleic acid ligand. It a further embodiment, the modulator can be, for example, a peptide nucleic acid or morpholino nucleic acid that includes a sequence thai is complementary to or hybridizes with at least a      L portion of the nucleic acid ligand. A modulator is specifically hybridizable with the nucleic acid ligand when binding of the modulator to the nucleic acid ligand sufficiently interferes with the I normal function of the nucleic acid ligand to .cause a change in the biological activity of the nucleic acid ligand, under physiological conditions. In an alternative embodiment, there is a sufficient degree of non-Watson Crick binding of the modulator to the nucleic acid ligand to affect the activity of the nucleic acid ligand.<br>
In a still further embodiment, the modulator is nucleic acid binding peptide, polypeptiite<br>
or protein that binds to or otherwise interacts with the nucleic acid ligand. In a further "^mbodiment, the modulator is an oligosaccharide that binds the nucleic acid ligand. In a specific embodiment, the modulator is an aminoglycoside. In another embodiment, the modulator is a<br>
small organic molecule (i.e., a molecule that can be synthetic or naturally occurring that is npl<br>
•	•<br>
: otherwise found in vivo and typically has a molecular weight of less than 1000).<br>
The present invention also includes methods for identifying modulators of nucleic acid ligands. In one embodiment, the binding of a modulator to a nucleic acid is determined by any assay that measures binding affinity, such as a gel shift assay. In another embodiment, the binding of a modulator to a nucleic acid ligand is determined by a Biacore assay. Other exemplary assays are described below.<br>
In another embodiment, the binding or interaction of the modulator with the nucleic acid ligand is measured by evaluating the effect of the ligand with and without the regulate   nder appropriate biological conditions. For example, modulators can be identified that modify the antitnrombotic or anticoagulant activity of nucleic acid ligand in vitro or in vivo through a coagulation test bioassay such as an activated coagulation time test, the activated partial thromboplastin test, the bleeding time test, the prothrombin time test, or the thrombin clotting time test.<br>
The present invention further includes the use of such modulators in a variety of<br>
indications whereby control of nucleic acid ligand activity is desired. The modulators may act to<br>
inhibit nucleic acid ligand activity as an antidote to reverse the actions of the nucleic acid ligand.<br>
Additionally, the invention provides methods of using modulators of the invention to assess the;<br>
activity of nucleic acid ligands. The invention is also directed to methods of enhancing or<br>
inhibiting the efficacy of nucleic acid ligands by administering modulators of the invention to<br>
human or non-human mammals.	<br>
In a further embodiment, modulators of the invention can also be used to reverse the   immunosuppressive effect of nucleic acid ligands that target interleukin, for example, in patients subject to infection. The present modulators can be used to reverse the immunostimulatory    effects of nucleic acid ligands that target CTLA4 in patients at risk of developing autoimmuniry.<br>
In a further embodiment, modulators of the invention can be used to reverse the effects of<br>
nucleic acid ligands that target growth factors (e.g., PDGF or VEGF). Such nucieib acid ligands<br>
xan be used in the treatment of tumors and in the treatment of inflammatory proliferative<br>
diseases. Since growth factors play systemic roles in normal cell survival and proliferation,<br>
nucleic acid ligand treatment can result in a breakdown of healthy tissue if not tightly regulated <br>
(e.g., patients receiving nucleic acid ligands that target angiopoietin I can be subject to	<br>
hemorrhaging). Modulators of tike invention that target such nucleic acid ligands can be used tp i<br>
provide the necessary regulation.	<br>
Modulators of the invention can be used to reverse the effects of nucleic acid ligands thajt<br>
target receptors involved in the transmission of the nerve impulse at the neuromuscular junction<br>
of skeletal muscle and/or autonomic ganglia (e.g., nicotinic acetylcholine or nicotinic cholinergic<br>
receptors). Such nucleic acid ligands can be made to produce muscular relaxation or paralysis :<br>
during anesthesia. Agents that block the activity of acetylcholine receptors (agents that engender<br>
neuromuscular blockade) are commonly used during surgical procedures, and h is preferred that<br>
the patients regain muscular function as soon as possible alter the surgical procedure is complete<br>
to reduce complications and improve patient turnover in the operating arenas. Therefore, much<br>
effort has been made to generate agents with predictable pharmacokinetics to match the duration<br>
of the drug activity to the anticipated duration of the surgical procedure. Alternatively,	<br>
modulators of the invention that target such nucleic acid ligands can be used to provide the<br>
desired control of the activity of the neuromuscular blocker, and thus reduce the dependence on<br>
the patient's physiology to provide reversal of the neuromuscular blocking agent.	<br>
In a still further embodiment, modulators of the invention can be used to reverse the    <br>
effect of nucleic acid ligands that target small molecules, such as glucose. Hypoglycernia can be<br>
avoided in patients receiving glucose-targeted nucleic acid ligands to regulate glucose uptake<br>
using the modulators of the invention.	<br>
Further, modulators can also be used to regulate the activity of nucleic acid ligands directed against members of the E2F family, certain of which are pro-proiiferative, certain of  which are repressive. The modulators of the invention can be used to "turn on" and "turn off" ! such nucleic acid ligands at desired points in the cell cycle.<br>
In another embodiment, modulators of the invention can also be used to reverse the  binding of nucleic acid Uganda bearing radioactive or cytotoxic moities to target tissue (e,g., j neoplastic tissue) and thereby, for example, facilitate clearance of such moeity's from'a patient's<br>
system. Similarly, the modulators of the invention can be used to reverse the binding of nucleic ligands labeled with detectable moieties (used, for example, in imaging or cell isolation ojr sorting) to target cells or tissues (Hicke et al. J. Clin. Invest. 106:923 (2000); Ringquist et al, Cytometry 33:394 (1998)). This reversal can be used to expedite clearance of the detectable moiety from a patient's system.<br>
Modulators of the invention can also be used in in vitro settings to enhance or inhibit tljie<br>
effect of a nucleic acid ligand (e.g., aptamer) on a target molecule. For example, modulators of<br>
the invention can be used in target validation studies. Using modulators of the invention, it is<br>
possible to confirm that a response observed after inhibiting a targei     	ith a nucleic<br>
acid ligand) is due to specifically inhibiting that molecule.<br>
The modulators of the invention can be formulated into pharmaceutical compositions that<br>
can include, in addition to the modulator, a pharmaceutically	le carrier, diluent or<br>
excipient. The precise nature of the composition will depend, at lea^i in part, on the nature of the modulator and the route of administration. Optimum dosing regimens can be readily established by one skilled in the art and can vary with the modulator, the patient and the effect sought. Generally, the modulator is administered IV, IM, IP, SC, orally or topically, as appropriate.<br>
In another embodiment, the nucleic acid ligand or its regulator can be covalently attached to a lipophilic compound such as cholesterol, dialky 1 glycerol, diacyl glycerol, or a non-immunogenic, high molecular weight compound or polymer such as polyethylene glycol (PEG). In these cases, the pharmacofcinetic properties of the nucleic acid ligand or modulator can be enhanced. In still other embodiments, the nucleic acid ligand or the modulator can be comprised for example, of a nucleic acid or PNA or MNA encapsulated inside a liposome, and the enhanced intracellular uptake is seen over the un-complexed oligonucleotide or modulator. The lipophilic compound or non-immunogenic, high molecular weight compound can be covalently bonded or associated through non-covalent interactions with ligand or modulators). In embodiments where the lipophilic compound is cholesterol, dialkyl glycerol, diacyl glycerol, or the non-immunogenic, high molecular weight compound is PEG, a covalent association with the oligonucleotide modulator (s) is preferred. In embodiments where the lipophilic compound is a<br>
cationic liposome or where the oligonucleotide modulators are encapsulated within the liposome,<br><br>
a non-covalent association with the oligonucleotide modulator (s) is preferred. In embodiments<br>
where covalent attachment is employed, the lipophilic compound or non-immunogenic, high     i molecular weight compound may be covalently bound to a variety of positions on the oligonucleotide modulator, such as to an exocyclic amino group on the base, the 5-position of a pyrimidine nucleotide, the 8-position of a purine nucleotide, the hydroxyl group of the<br>
•	:<br>
phosphate, or a hydroxyl group or other group at the 5' or 3' terminus of the oligonucleotide modulator. Preferably, however, it is bonded to the 5' or 3' hydroxyl group thereof. Attachment of the oligonucleotide modulator to other components of the complex can be done directly or with the utilization of linkers or spacers. The lipophilic compound or non-immunogenic, high molecular weight compound can associate through non-covalent interactions with the oligonucleotide modulator(s). For example, in one embodiment of the present invention, the oligonucleotide modulator is encapsulated within the internal compartment of the lipophilic compound. In another embodiment of the present invention, the oligonucleotide modulator associates with the lipophilic compound through electrostatic interactions. For instance, a cationic liposome can associate with an anionic oligonucleotide modulator. Anc.   
Definitions<br>
The following terms are believed to have well-recognized meanings in the art. However, the following definitions are set forth to facilitate explanation of the invention.<br>
The terms "binding activity" and "binding affinity' are meant to refer to the tendency off a ligand molecule to bind or not to bind to a target. The energy of said interactions are significant in "binding activity' and "binding affinity" because they define the necessary concentrations of interacting partners, the rates at which these partners are capable of associating, and the relative<br>
concentrations of bound and free molecules-in a solution. The energetics are characterized through, among other ways, the determination of a dissociation constant, Ka. Preferably, the K^ is established using a double-filter nitrocellulose filter binding assay such as that disclosed by Wong and Lbhman, 1993, Proc. NatlAcad. Sci. 'USA 90,5428-54i2. "Specifically binding oligonucleotides", "nucleic acid Hgands" or "aptamers" in one embodiment of the invention arei<br>
Wgonucleotides having specific binding regions that are capable of forming complexes with an intended target molecule. The specificity of the binding is defined in terms of the comparative dissociation constants (K<i of a modulator nucleic acid ligand as compared to the></i>
dissociation constant with respect to other materials in the environment or unrelated molecules iti general. The Kd for a modulator of a nucleic acid ligand can be 2-fold, preferably 5-fold, more preferably 10-fold less than the Kd with respect to the modulator and the unrelated material or  accompanying material in the environment. Even more preferably the Kd will be 50-fold less, more preferably 100-fold less, and more preferably 200-fold less.<br>
Kd can be determined directly by well-known methods, and can be computed even for complex mixtures by methods such as those, for example, set forth in Caceci, M., et al., Byte (1984) 9:340-362. ft has been observed, however, that for some small oligonucleotides, direct determination of Kd is difficult, and can lead to misleadingly, high results. T     sr these circumstances, a competitive binding assay for the target molecule or other candidate substance can be conducted with respect to substances known to bind the target or candidate. The value of the concentration at which 50% inhibition occurs (Ki) is, under ideal conditions, equivalent to Kd. However, in no event will a K; be less than Kd. Thus, determination of Kj, in the alternative, sets a maximal value for the value of Kd. Under those circumstances where technical difficulties preclude accurate measurement of Kd, measurement of K; can conveniently be substituted to provide an upper limit for Kd. A K; value can also be used to confirm that a modulator binds a nucleic acid ligand.<br>
As specificity is defined in terms of Kd as set forth above, in certain embodiments of the present invention ft is preferred to exclude from the categories of unrelated materials and materials accompanying the target in the target's environment those materials which are sufficiently related to the target to be immunologically cross-reactive therewith. By "immunologically cross-reactive" is meant that antibodies raised with respect to the target cross-react under standard assay conditions with the candidate material. Generally, for antibodies to cross-react in standard assays, the binding affinities of the antibodies for cross-reactive materials as compared to targets should be in the range of 5-fold to 100-fold, generally about 10-fold.<br>
.	In the context of this invention, "hybridization" means hydrogen bonding, which may be<br>
^Watson-Crick, Hoogsteen or reversed Hoogsteen hydrogen bonding, between complementary<br>
nucleoside or nucleotide bases. For example, adenine and thymine are complementary<br>
nucleobases which pair through the formation of hydrogen bonds, "Complementary," as used   herein, refers to the capacity for precise pairing between two nucleotides. For example, if a      <br>
nucleotide at a certain position of an otigonucleotide is capable of hydrogen bonding with a     <br>
nucleotide at the same position of a DNA or KNA molecule, then the oligonucleotide and the   <br>
DNA or KNA are considered to be complementary to each other at that position. The	<br>
oligonucleotide and the DNA or KNA are complementary to each other when a sufficient         number of corresponding positions in each molecule are occupied by nucleotides which can hydrogen bond with each other. Thus, "specifically hybridizable" and "complementary" are terms which are used to indicate a sufficient degree of complementarity or precise pairing such: that stable and specific binding occurs between the oligonucleotide and the DNA or KNA target. It is understood in the art that the sequence of compound need not be 100% complementary to that of its target nucleic acid to be specifically hybridizable. A compound is specifically         hybridizable when binding of the compound to the target nucleic acid molecule interferes with the normal function of the target nucleic acid to cause a change in utility, and there is a sufficient degree of complementarity to avoid non-specific binding of the compound to non-target sequences under conditions in which specific binding is desired, i.e., under physiological conditions in the case of in vivo assays or therapeutic treatment, and in the case of in vitro assays, under conditions in which the assays are performed.<br>
"Qligomers" or .oligonucleotides" include SNA or DNA sequences or mixtures or analogs thereof, of more than one nucleotide in either single chain or duplex form and specifically includes short sequences such as dimers and trimers, in either single chain or duplex form, which can be intermediates in the production of the specifically binding oligonucleotide^. "Modified" forms used in candidate pools contain at least one non-native residue.<br>
The term "RNA analog" is meant to refer to a polymeric molecule which can contain one or more nucleotides that have a nonhydrogen substttuent other than a hydroxyl group in the 2*-position, and for example, can contains at least one of the following: 2'-deoxy, 2'-halo (including 2'-fluoro), 2'-amino (preferably not substituted or mono- or disubstituted), 2'~moin0-, di- ortti-halomethyl, 2*-O-aIkyl (including 2'-0-methyl or 0-ethyl), 2'-0-halo-substituted a%l,<br>
2'-alkyl, azido, phosphorothioate, sulfhydryl, methylphosphonate, fluorescein, rhodaminek      | ^"^rene, biotin, xanthine, hypoxanthine, 2,6-diamino purine, 2-hydroxy-6-mercaptopurine and pyrimidine bases substituted at the 6-position with sulfur or 5 position with halo or Ci-s alkyl groups, a basic linkers, 3'-deoxy-adenosine as well as other available "chain terminator" or    i "non-extendible" analogs (at the 3'-end"of the KNA), or labels such as 32P, 33P and the like. Aji of the foregoing can be incorporated into an KNA using the standard synthesis techniques disclosed herein.<br>
As used herein, a "target" or "target molecule" refers to a biomolecule that is the focus of a therapeutic drug strategy or diagnostic assay, including, -without limitation, enzymes, enzyme inhibitors, hormones, glycoproteins, lipids, phospholipids, nucleic acids, intracellulat, \ extracellular, and cell surface proteins, peptides, carbohydrates, including glycosaminoglycaas; lipids, including glycolipids and certain oligonucleotides, and generally, any biomolecule capable of turning a biochemical pathway on or off or modulating it, or which is involved in a \ predictable biological response. Targets can be free in solution, like thrombin, or associated with cells or viruses, as in receptors or envelope proteins. Any molecule that is of sufficient size to be specifically recognized by a nucleic acid ligand can be used as the target. Thus, membrane    : structures, receptors, organelles, and the like can be used as the complexation targets.<br>
An "KNA aptamer" is an aptamer comprising ribonucleoside units. "RNA aptamer" ls\ also meant to encompass RNA analogs as defined herein above.<br>
The term "coagulation factor aptamer" is meant to refer to a single- or double-stranded nucleic acid that binds a coagulation factor and modulates its function. The term "coagulation, factor" is meant to refer to a factor that acts in either or both of the intrinsic and the extrinsic coagulation cascade.<br>
As used herein, "consensus sequence" refers to a nucleotide sequence or region (whiclj might or might not be made up of contiguous nucleotides) that is found in one or more regions of at least two nucleic acid sequences. A consensus sequence can be as short as three nucleotides long. It also can be made up of one or more noncontiguous sequences, with nucleotide sequences or polymers of up to hundreds of bases long interspersed between the consensus sequences. Consensus sequences can be identified by sequence comparisons between individual nucleic acid species, which comparisons can be aided by computer programs and other, tooaj for<br>
modeling secondary and tertiary structure from sequence information. Generally, the consensus i<br>
w|uence will contain at least about 3 to 20 nucleotides, more commonly from 6 to 10<br>
nucleotides.	!<br>
The terms "cardiovascular disease" and "cardiovascular diseases" are meant to refer to ;<br><br>
	:<br>
any cardiovascular disease as would be understood by one of ordinary skill in the art.	<br>
Nonlimiting examples of particularly contemplated cardiovascular diseases include, but are not limited to, atherosclerosis, thrombophilia, embolisms, cardiac infarction (e.g., myocardial infarction), thromboses, angina, stroke, septic shock, hypertension, hyper-cholesteroiemia,        restenosis and diabetes.<br>
The term "about," as used herein when referring to a measurable value such as an amount of weight, time, dose etc. is meant ,to encompass variations of ±20% or ±10%, more preferably ! ±5%, even more preferably ±1%, and still more preferably ±0.1% from the specified amount, as such variations are appropriate to perform the disclosed method.<br>
Compounds of the present invention having a chiral center may exist in and be isolated in optically active and racemic forms. Some compounds may exhibit polymorphism. The present invention encompasses racemic, optically-active, polymorphic, or stereoisomeric form, or        mixtures thereof, of a compound of the invention, which possess the useful properties described herein. The optically active forms can be prepared by, for example, resolution of the racemic form by recrystallization techniques, by synthesis from optically-active starting materials, by chiral synthesis, or by chromatograpbic separation using a chiral stationary phase or by enzymatic resolution. In a preferred embodiment, the compounds are used in naturally occurring forms. However, alternatively, the compounds can be used in a non-naturally occurring form.<br>
n. Nucleic Acid Ligands<br>
A "Nucleic Acid Ligand" (sometimes also referred to as an "aptamer") as used herein is a non-naturally occurring nucleic acid having a desirable action on a target. A desirable action includes, but is not limited to, binding of the target, catalytically changing the target, reacting \ with the target in a way which modifies/alters the target or the functional activity of the target, j covalently attaching to the target as in a suicide inhibitor, or facilitating the reaction between the target and another molecule. In the preferred embodiment, the action is specific binding affinity!<br>
for a target molecule, such target molecule being a three dimensional chemical structure, wherein<br>
nucleic acid ligand is not a nucleic acid having the known physiological function of being I bound by the target molecule. In one embodiment of the invention, the nucleic acid Bgands are! identified using the SELEX methodology. Nucleic acid ligands includes nucleic acids that ar$ identified from a candidate mixture of nucleic acids, wherein the nucleic acfd ligand being a i ligand of a given target by the method comprising a) contacting the candidate mixture with t&amp;ei target, wherein nucleic acids having an increased affinity to the target relative to the candidate mixture may be partitioned from the remainder of the candidate mixture; b) partitioning the !<br>
i<br>
increased affinity nucleic acids from the remainder of the candidate mixture; and c) amplifying the increased affinity nucleic acids to yield a ligand-enriched mixture of nucleic acids. As used herein nucleic acid ligand or aptamer denotes both singular and plural sequences of nucleic acids which are capable of binding to a protein or other molecule, and thereby disturbing the proteuf s or other molecule's function.<br>
Nucleic acid ligands can be made with nucleotides bearing D or L stereochemistry, or a<br>
mixture thereof. Naturally occurring nucleosides are in the D configuration. Aptamers have been<br>
made from L nucleotides, and are called L-aptamers. Nucleic acid ligands used for therapeutic<br>
purposes are typically in the D configuration, but can exhibit any configuration that provides the<br>
desired effect. Typically, when nucleic acid ligands comprising L-nucleotides are the target of<br>
an oligonucleotide modulator., the modulator also comprises L-nucleotides (see, for example,<br>
USP 5,780,221).	|<br>
The nucleic acid ligands preferably comprise about 10 to about 100 nucleotides, preferably about 15 to about 40 nucleotides, more preferably about 20 to about 40 nucleotides, in that oligonucleotides of a length that falls within these ranges are readily prepared by conventional techniques. In one embodiment, aptamers or oligonucleotide modulators can comprise a minimum of approximately 6 nucleotides, preferably 10, and more preferably 14 or 15 nucleotides, that are necessary to effect specific binding. The only apparent limitations on the binding specificity of the target/oil gonucleotide couples of the invention concern sufficient sequence to be distinctive in the binding oligonucleotide and sufficient binding capacity of the target substance to obtain the necessary interaction. Aptamers of binding regions containing \ sequences shorter than 10, e.g., 6-mers, are feasible if the appropriate interaction can be obtained<br>
in the context of the environment in which the target is placed. Thus, if there is little interference ^Jtether materials, less specificity and less strength of binding can be required.<br>
Nucleic acid ligands and methods for their production and use, are described, for<br>
example, in the following U.S. patents. Any of the nucleic acid ligands described in the patents<br>
listed below or other patents, or any nucleic acid ligands described in publications as well as<br>
other desired nucleic acid ligands used in medical therapy can be modulated or regulated<br>
according to the present invention. U.S. Patent No. 6,387,635, entitled 2'-fluoropyrimidine anti-:<br>
calf intestinal phosphatase nucleic acid ligands; U.S. Patent No. 6,387,620, entitled	<br>
Transcription-free selex; U.S. Patent No. 6,379,900, entitled Compositions and methods of use of 8-nitroguanine; U.S. Patent No. 6,376,474, entitled Systematic evolution of ligands by exponential enrichment: tissue SELEX; U.S. Patent No. 6,376,190, entitled Modified SELEX processes without purified protein; U.S. Patent No. 6,355,787, entitled Purine nucleoside modifications by palladium catalyzed methods and compounds produced; U. S. Patent No. 6,355,431, entitled Detection of nucleic acid amplification reactions using bead arrays; U.S. Patent No. 6,346,611, entitled High affinity TGFJ3 nucleic acid ligands and inhibitors; U.S. Patent No. 6,344,321, entitled Nucleic acid ligands which bind to hepatocyte growth factor/scatter factor (HGF/SF) or its receptor c-met; U.S. Patent No. 6,344,318, entitled Methodjs of producing nucleic acid ligands; U.S. Patent No. 6,331,398, entitled Nucleic acid ligands; UJSi Patent No. 6,331,394, entitled Nucleic acid ligands to integrins; U.S. Patent No. 6,329,145, entitled Determining non-nucleic acid molecule binding to target by competition with nucleic acid ligand; U.S. Patent No. 6,306,598, entitled Nucleic acid-coupled colorimetric analyte detectors; U.S. Patent No. 6,303,316, entitled Organic semiconductor recognition complex and system; U.S. Patent No. 6,300,074, entitled Systematic evolution of ligands by exponential enrichment: Chemi-SELEX; U.S. Patent No. 6,291,184, entitled Systematic evolution of ligands by exponential enrichment: photoselection of nucleic acid ligands and solution selex; U.S. Patent No. 6,287,765, entitled Methods for detecting and identifying single molecules; U.S. Patent No. 6,280,943, entitled 2'-fluoropyrimidine anti-calf intestinal phosphatase nucleic acid ligands; U.$. Patent No. 6,280,932, entitled High affinity nucleic acid ligands to lectins; U.S. Patent No. 6,264,825, entitled Binding acceleration techniques for the detection of analytes; U.S. Patent Np. 6,261,783, entitled Homogeneous detection of a target through nucleic acid ligand-ligand beacon interaction; U.S. Patent No. 6,261,774, entitled Truncation selex method; U.S. Patent No.<br>
6,242,246, entitled Nucleic acid ligand diagnostic Biochip; U.S. Patent No. 6,232,071, entitled ^llnasein-C nucleic acid ligands; U.S. Patent No. 6,229,002, entitled Platelet derived growth factor (PDGF) nucleic acid ligand complexes; U.S. Patent No. 6,225,063, entitled RNA channel^ in biological membranes; U.S. Patent No. 6,207,816, entitled High affinity oligonucleotide ligands to growth factors; U.S. Patent No. 6,207,388, entitled Compositions, methods, kits and \ apparatus for determining the presence or absence of target molecules; U.S. Patent No. 6,184,364, entitled High affinity nucleic acid ligands containing modified nucleotides; U.S. Patent No. 6,183,967, entitled Nucleic acid ligand inhibitors to DNA polymerases; U.S. Patent i No. 6,180,348, entitled Method of isolating target specific oligonucleotide ligands; U.S. Patent No. 6,177,557, entitled High affinity ligands of basic fibroblast growth factor and thrombin; U.S. Patent No. 6,177,555, entitled Homogeneous detection of a target through nucleic acid ligand- i ligand beacon interaction; U.S. Patent No. 6,171,795, entitled Nucleic acid ligands to CD40 iigaad; U.S. Patent No. 6,168,778, entitled Vascular endothelial growth factor (VEGF) Nucleici<br>
I	'<br>
Acid Ligand Complexes; U.S. Patent No. 6,147,204, entitled Nucleic acid ligand complexes;    ; U.S. Patent No. 6,140,490, entitled High affinity nucleic acid ligands of complement system proteins; U.S. Patent No. 6,127,119, entitled Nucleic acid ligands of tissue target; U.S. Patent No. 6,124,449, entitled High affinity TGFJ5 nucleic acid ligands and inhibitors; U.S. Patent No. j 6,114,120, entitled Systematic evolution of ligands by exponential enrichment: tissue selex; U.S. Patent No. 6,110,900, entitled Nucleic acid ligands; U.S. Patent No. 6,083,696, entitled Systematic evolution of ligands exponential enrichment: blended selex; U.S. Patent No. 6,080,585, entitled Methods for discovering ligands; U.S. Patent No. 6,051,698, entitled Vascular endothelial growth factor (VEGF) nucleic acid ligand complexes; U.S. Patent No. 6,048,698, entitled Parallel SELEX; U.S. Patent No. 6,030,776, entitled Parallel SELEX; US. Patent No. 6,028,186, entitled High affinity nucleic acid iigands of cytokines; U.S. Patent No. 6,022,691, entitled Determination of oligonucleotides for therapeutics, diagnostics and research; reagents; U.S. Patent No. 6,020,483, entitled Nucleoside modifications by palladium catalyzed methods; U.S. Patent No. 6,020,130, entitled Nucleic acid ligands that bind to and inhibit DNA polymerases; U.S. Patent No. 6,013,443, entitled Systematic evolution of ligands by exponential enrichment: tissue SELEX; U.S. Patent No. 6,011,020, entitled Nucleic acid ligand complexes; U.S. Patent No. 6,001,988, entitled High affinity nucleic acid ligands to lectins; U.S. Patent nol 6,001,577, entitled Systematic evolution of ligands by exponential enrichment: photoselection cjf<br>
nucleic acid ligands and solution selex; U.S. Patent No. 6,001,570^ entitled Compositions,         ! lUthods, kits and apparatus for determining the presence or absence of target molecules; U.S. Patent No. 5,998,142, entitled Systematic evolution of ligands by exponential enrichment: ehemi-SELEX; U.S. Patent No. 5,989,823, entitled Homogeneous detection of a target through nucleic acid ligand-ligand beacon interaction; U.S. Patent No. 5,972,599, entitled High affinity i nucleic acid ligands of cytokines; U.S. Patent No. 5,962,219, entitled Systematic evolution of   \ ligands by exponential enrichment: chemi-selex; U.S. Patent No. 5,958,691, entitled High affinity nucleic acid ligands containing modified nucteotides; U.S. Patent No. 5,874,557,         \ flatted Nucleic acid ligand inhibitors to DNA polymerases; U.S. Patent No. 5,874,218, entitlecji Method for detecting a target compound in a substance using a nucleic acid ligand; U.S. Patent No. 5,871,924, entitled Method for the production of ligands capable of facilitating aminoacyl-RNA synthesis; U.S. Patent No. 5,869,641, entitled High affinity nucleic acid ligands of CD4j; U.S. Patent No.'5,864,026, entitled Systematic evolution of ligands by exponential enrichment: \ tissue selex; US. Patent No. 5,861,254, entitled Flow cell SEIEX; U.S. Patent No. 5,859,228, j entitled Vascular endothelial growth factor (VEGF) nucleic acid ligand complexes; U.S. Patent No. 5,858,660, entitled Parallel selex; U.S. Patent Na 5,853,984, entitled Use of nucleic acid ligands in flow cytometry; U.S. Patent No. 5,849,890, entitled High affinity oligonucleotide    j ligands to chorionic gonadotropin hormone and related glycoprotein hormones; U.S. Patent No. 5,849,479, entitled High-affinity oligonucleotide ligands to vascular endothelial growth factor (VEGF); U.S. Patent No. 5,846,713, entitled High affinity HKGF nucleic acid ligands and inhibitors; U.S. Patent No. 5,843,653, entitled Method for detecting a target molecule in a sample using a nucleic acid ligand; U.S. Patent No.5,837,834, entitled High affinity HKGF nucleic acid ligands and inhibitors; U.S. Patent No.5,837,456, entitled High affinity oligonucleotide ligands to chorionic gonadotropin hormone and related glycoprotein hormones;. U.S. Patent No. 5,834,199, entitled Methods of identifying transition metal complexes that selectively cleave regulatory elements of mRNA and uses thereof; U.S. Patent No.5,817,785, entitled Methods of producing nucleic acid ligands; U.S. Patent No.5,811,533, entitled High- ; affinity oligonucleotide ligands to vascular endothelial growth factor (VEGF); U.S. Patent No.5,795,72l, entitled High affinity nucleic acid ligands of ICP4; U.S. Patent No. 5,789,163, i entitled Enzyme linked oligonucleotide assays (ELONAS); U.S. Patent No. 5,789,160, entitiwj<br>
1	<br>
Parallel selex; U.S. Patent No.5,789,157,, entitled Systematic evolution of ligands by exponential<br>
enrichment: tissue selex; U.S. Patent No.5,786,462, entitled High affinity ssDNA ligands of     <br>
JCV-1 reverse transcriptase; U.S. Patent No. 5,780,228, entitled High affinity nucleic acid       <br>
ligands to lectins U.S. Patent No.5,773,598, entitled Systematic evolution of ligands by<br>
exponential enrichment: chimeric selex; U.S. Patent No.5,766,853, entitled Method for<br>
identification of high affinity nucleic acid ligands to selectins; U.S. Patent No.5,763,595,         <br>
entitled Systematic evolution of ligands by exponential enrichment: Chemi-SELEX; U.S. Patenjt<br>
No. 5,763,566, entitled Systematic evolution of ligands by exponential enrichment: tissue<br>
SELEX; U.S. Patent No.5,763,177, entitled Systematic evolution of ligands by exponential      <br>
eariehment: photoselection of nucleic acid ligands and solution selex; U.S. Patent No. 5,763,173!,<br>
entitled Nucleic acid Hgand inhibitors to DNA polymerases; U.S. Patent No.5,756,287, entitled |<br>
Ugh affinity HTV integrase inhibitors; U.S. Patent No. 5,750,342, entitled Nucleic acid ligands;<br>
of tissue target; U.S. Patent No.5,734,034, entitled Nucleic acid ligand inhibitors of human<br>
aeutrophil elastase; U.S. Patent No.5,731,424, entitled High affinity TGFp nucleic acid ligands<br>
and inhibitors; U.S. Patent No.5,731,144, entitled High affinity TGFJ3 nucleic acid ligands; U.Sj.<br>
Patent No. 5,726,017, entitled High affinity HTV-1 gag nucleic acid ligands; U.S. Patent<br>
No.5,723,594, entitled High affinity PDGF nucleic acid ligands; US. Patent No.5,723,592,<br>
entitled Parallel selex; U.S. Patent No. 5,723,289, entitled Parallel selex; U. S. Patent No.<br>
5,712,375, entitled Systematic evolution of ligands by exponential enrichment: tissue selex; U.S.!<br>
Patent No. 5,707,796, entitled Method for selecting nucleic acids on the basis of structure; U. S.<br>
Patent No.5,705,337, entitled Systematic evolution of ligands by exponential enrichment: chenti-j<br>
SELEX; U.S. Patent No.5,696,249, entitled Nucleic acid ligands; U.S. Patent No. 5,693,502,<br>
entitled Nucleic acid ligand inhibitors to DNA polymerases; U.S. Patent No. 5,688,935, entitled!<br>
Nucleic acid ligands of tissue target; U.S. Patent No.5,686,592, entitled High-affinity<br>
oligonucleotide ligands to immunoglobulm E (IgE); U.S. Patent No. 5,686,242, entitled<br>
Determination of oligonucleotides for therapeutics, diagnostics and research reagents; U.S.<br>
Patent No. 5,683,867, entitled Systematic evolution of ligands by exponential enrichment:<br>
blended SELEX; U.S. Patent No. 5,674,685, entitled High affinity PDGF nucleic acid ligands;  <br>
U.S. Patent No. 5,670,637, entitled Nucleic acid ligands; U.S. Patent No. 5,668,264, entitled<br>
High affinity PDGF nucleic acid ligands; U.S. Patent No. 5,663,064, entitled Ribozymes with    <br>
RNA protein binding site; U.S. Patent No.5,660,985, entitled High affinity nucleic acid ligands<br>
containing modified nucleotides; U.S. Patent No. 5,654,151, entitled High affinity HJV	<br>
Nucleocapsid nucleic acid Iigands; U.S. Patent No. 5,650,275, entitled Target detection method ^plg spectroscopically detectable nucleic acid Iigands; U.S. Patent No.5,648,214, entitled High-affinity oligoriucleotjde ligands to the tachykinin substance P; U.S. Patent No.5,641,629,: entitled Spectroscopically detectable nucleic acid ligands; U.S. Patent No.5,639,868, entitled High-affinity UNA ligands for basic fibroblast growth factor; U.S. Patent No. 5,637,6^2, entitled High-affinity oligonucleotide ligands to the tachykinin substance P; U.S. Patent No. 5,637,461,1 entitled Ligands of HTV-1 TAT protein; U.S. Patent No.5,637,459, entitled Systematic evolution of ligands by exponential enrictoaent: chimeric selex; U.S. Patent No.5,635,615, entitled High affinity HIV nucleocapsid nucleic acid ligands; U.S. Patent No.5,629,155, entitled High-affinity oligonueleotide ligands to imniunoglobuHn E (IgE); U.S. Patent No. 5,622,828, entitled High-affinity oligonucieotide ligands to secretory phospholipase A2 (sPLAsub.2); U.S. Patent N0. 5,595,877, entitled Methods of producing nucleic acid ligands; U.S. Patent No.5,587,468, entitled High affinity nucleic acid ligands to HTV integrase; U.S. Parent No.5,580,737, entitled High-affinity nucleic acid ligands that discriminate between theophylline and caffeine; U.S. Patent No.5,567,588, entitled Systematic evolution of ligands by exponential enrichment!: Solution SELEX; U.S. Patent No. 5,543,293, entitled DNA ligands of thrombin; U.S. Patent No. 5,527,894, entitled Ltfands of HIV-1 tat protein; U.S. Patent No.5,475,096, entitled Nubleijc acid ligands; U. S. Patent 5,866,334, entitled Determination and identification of active compounds in a compound library; U.S. Patent 5,864,026, entitled Systematic evolution of ligandsby exponential enrichment: tissue selex; U.S. Patent 5,861,254, entitled Flow cell SELEX; U.S. Patent 5,859,228, entitled Vascular endothelial growth fector (VEGF) nucleic acid ligand complexes; U.S. Patent 5,858,660, entitled Parallel selex; U.S. Patent 5,853,984, entitled Use of nucleic acid ligands in flow cytometry; U.S. Patent 5,849,890, entitled High affinity oligonucieotide ligands to chorionic gonadotropin hormone and related glycoprotein hormones; U.S. Patent 5,849,479, entitled High-affinity oligonucieotide ligands to vascular endothelial growth factor (VEGF); U.S. Patent 5,846,713, entitled High affinity HKGF nucleic acid Kgftndi and inhibitors; U.S. Patent 5,843,732, entitled Method and apparatus for determining eonseilsusi secondary structures for nucleic acid sequences; U.S. Patent 5,843,653, entitled Method ibr detecting a target molecule in a sample using a nucleic acid ligand; U.S. Patent 5,840^86?, entitled Aptamer analogs specific for biomolecules; U.S. Patent 5,840,580, entitled Phenotypic I characterization of the hematopoietic stem cell; U.S. Patent 5,837,838, enthledBax inhibitor<br>
proteins; U.S. Patent 5,837,834, entitled High affinity HKGF nucleic acid ligands and mhibitorjs;<br>
.S. Patent 5,837,456, entitled High affinity oligonucleotide ligands to chorionic gonadotropin hortnone and related glyeoprotein hormones; U.S. Patent 5,834,199, entitled Methods of identifying transition metal complexes that selectively cleave regulatory elements of mRISJA ar^d uses thereof; U.S. Patent 5,834,184, entitled In vivo selection of KNA-binding peptides; U.S. Patent 5,817,785, entitled Methods of producing nucleic acid ligands; U.S. Patent 5,811,533, entitled High-affinity oltgoaucJeotide ligands to vascular endothelial growth factor (VEGiF); U.S. Patent 5,804,390, entitled Use of nuclear magnetic resonance to identify ligands to target biomolecules; U.S. Patent 5,795,721, entitled High affinity nucleic acid ligands of ICP4; U.SJ Patent 5,789,163, entitled Enzyme linked oligonucleotide assays (ELONAS); U.S. Patent 5,789,160, entitled Parallel setex; U.S. Patent 5,789,157, entitled Systematic evolution of ligajnjis by exponential enrichment: tissue selex; U.S. Patent 5,786,462, entitled High affinity ssDHA ligands of HTV-1 reverMtranscriptase; U.S. Patent 5,786,203, entitled Isolated nucleic acid encoding o^cotropinHrelea^og fector.sub.2 receptors; U.S. Patent 5,786,145, entitled Oligonucleotide competitors for binding of HIV RRE to REV protein and assays for screening | inhibitors of this binding; U.S. Patent 5,783,566, entitled Method for increasing or decreasing j transfection efficiency; U.S. Patent 5,780,610, entitled Reduction of nonspecific hybridization by<br>
using novel base-pairing schemes; U.S. Patent 5,780,228, entitled High affinity nucleic acid     ligands to lectins; U.S. Patent 5,773,598, entitled Systematic evolution of ligands by exponential enrichment: chimeric selex; U.S. Patent 5,770,434, entitled Soluble peptides having constrained, secondary conformation in solution and method of making same; U.S. Patent 5,766,853, entitled Method for identification of high affinity nucleic acid ligands to selectins; U.S. Patent 5,763,595, entitled Systematic evolution of ligands by exponential enrichment: Chemi-SELEX; U.S. Patent 5,763,566, entitled Systematic evolution of ligands by exponential enrichment: tissue SELEX; U.S. Patent 5,763,177, entitled Systematic evolution of ligands by exponential enrichment: photoselection of nucleic acid ligands and solution selex; U.S. Patent 5,763,173, entitled Nucleic add ligand inhibitors to DNA polymerases; U.S. Patent 5,756,296, entitled Nucleotide-directed assembly of bimolecular and multimolecular drugs and devices; U.S. Patent 5,756,291, entitled Aptamers specific for biomolecules and methods of making; U.S. Patent 5,756,287, entitled High affinity HIV integrase inhibitors; U.S. Patent 5,750,342, entitle^ Nucleic acid'ligands of tissue target; U.S. Patent 5,739,305, entitled 'Nucleotide-directeid<br>
assembly of bimolecular and multimolecular drugs and devices; U.S. Patent 5,734,034, entitled | Jfccleic acid ligand inhibitors of human neutrophil elastase; U.S. Patent 5,733,732, entitled Methods for detecting primary adhalinopathy; U.S. Patent 5,731,424, entitled High affinity TGF.beta. nucleic acid ligands and inhibitors; U.S. Patent 5,731,144, entitled High affinity TGF.bfeta. nucleic acid ligands; U.S. Patent 5,726,017, entitled High affinity HIV-1 gag nucleic acid ligands; U.S. Patent 5,726,014, entitled Screening assay for the detection of DNA-binding molecules; U.S. Patent 5,723,594, entitled High affinity PDGF nucleic acid ligands; U.S. Patent No. 5,723,592, entitled Parallel selex;U.S. Patent No. 5,723,289, entitled Parallel selex;U.S. Patent No. 5,712,375, entitled Systematic evolution of ligands by exponential enrichment: tissue selex; U.S. Patent No. 5,707,796, entitled Method for selecting nucleic acids on the basis of structure; U.S. Patent No. 5,705,337, entitled Systematic evolution of ligands by exponential enrichment: chemi-SELEX; U.S. Patent No. 5,698,442, entitled DNA encoding an 18 Kd CDK6 inhibiting protein; U.S. Patent No. 5,698,426, entitled Surface expression libraries of heteromerifc receptors; U.S. Patent No. 5,698,401, entitled Use of nuclear magnetic resonance to identify ligands to target biomolecules; U.S. Patent No. 5,693,502, entitled Nucleic acid ligand inhibitors to DNA polymerases; U.S. Patent No. 5,688,935, entitled Nucleic acid ligands of tissue target; U.S. Patent No. 5,688,670, entitled Self-modifying RNA molecules and methods of making; U.S Patent No. 5,686,592, entitled High-affinity oligonucleotide ligands to immunoglobulin E (IgE); U.S. Patent No. 5,683,867, entitled Systematic evolution of ligands by exponential enrichment: Wended SELEX; U.S. Patent No. 5,681,702, entitled Reduction of nonspecific hybridization by using novel base-pairing schemes; U.S. Patent No. 5,674,685, entitled High affinity PDGF nucleic acid ligands; U.S. Patent No. 5,670,637, entitled Nucleic acid ligands; U.S. Patent No. 5,668,265, entitled Bi-directional oligonucleotides that bind thrombin; U.S. Patent No. 5,668,264, entitled High affinity PDGF nucleic acid ligands; U.S. Patent No. 5,660,985, entitled High affinity nucleic acid ligands containing modified nucleotides; U.S. Patent No. 5,660,855, entitled Lipid constructs for targeting to vascular smooth muscle tissue; U.S. Patent No.<br>
entitled Bi-directional oligonucleotides that bind thrombin; U.S. Patent No.<br>
entitled Nucleotide-dkected assembly of bimolecular and multimolecular drugs and<br>
devices; U.S. Patent No. 5,656,467, entitled Methods and materials for producing gene libraries;<br>
U.S. Patent No. 5,654,151, entitled High affinity HTV Nucleocapsid nucleic acid ligands; U.S.<br>
Patent; No. 5,650,275, entitled Target detection method using ispectroscopically detectable<br>
jiucleic acid ligands; U.S. Patent No. 5,648,214, entitled High-affinity oligonucleotide ligands t
fibroblast growth factor; U. S. Patent No. 5,639,428, entitled Method and apparatus for folly<br>
»&gt;<br>
automated nucleic acid amplification, nucleic acid assay and immunoassay; U.S. Patent No.<br>
5,637,682, entitled High-affinity oligonucleotide ligands to the tachykinin substance P; U.S.<br>
Patent No. 5,637,459, entitled Systematic evolution of ligands by exponential enrichment:<br>
cMmeric selex; U.S. Patent No. 5,635,615, entitled High affinity HTV nucleocapsid nucleic acid!<br>
ligands; U.S. Patent No. 5,631,156, entitled DNA encoding and 18 KD CDK6 inhibiting protehj;<br>
U.S. Patent No. 5,631,146, entitled DNA aptamers and catalysts that bind adenosine or	<br>
adenosine-S'-phosphates and methods for isolation thereof; U.S. Patent No. 5,629,407, entitled j DNA encoding an 18 KD CDK6 inhibiting protein and antibodies thereto; U.S. Patent No. !<br>
i<br>
5,629,155, entitled High*affinity oligonucleotide ligands to immunoglobulin E (IgE); U.S. Patent<br>
j<br>
No. 5,622,828, entitled High-affinity oligonucleotide ligands to secretory phospholipase A2 j<br>
(sPLA.sub.2); U.S. Patent No. 5,621,082, entitled DNA encoding an 18 Kd CDK6 inhibiting j<br>
protein; U.S. Patent No. 5,599,917, entitled Inhibition of interferon-.gamma, with	<br>
oligonucleotides; U.S. Patent No. 5,597,696, entitled Covalent cyanine dye oligonucleotide      conjugates; U.S. Patent No. 5,587,468, entitled High affinity nucleic acid ligands to HTV         Iintegrase; U.S. Patent No. 5,585,269, entitled Isolated DNA encoding c-mer protooncogene;    U.S. Patent No. 5,580,737, entitled High-affinity nucleic acid ligands that discriminate betweeji tfaeopbylline and caffeine; U.S. Patent No. 5,567,588, entitled Systematic evolution of ligands tjy es|&gt;onential enrichment: Solution SELEX; U.S. Patent No. 5,565,327, entitled Methods of       diagnosing parasitic infections and of testing drug susceptibility of parasites; U.S. Patent No. 5,527,894, entitled Ligands of HTV-1 tat protein; U.S. Patent No. 5,512,462, entitled         Methods and reagents for the polymerase chain reaction amplification of long DNA sequences;! U.S. Patent No. 5,503,978, entitled Method for identification of high affinity DNA ligands of j HTV-l reverse transcriptase; U.S. Patent No. 5,472,841, entitled Methods for identifying nucleib acid ligands of human neutrophil elastase; andU.S. Patent No. 5,459,015, entitled High-afBnityj KNA ligands of basic fibroblast growth factor.<br>
BQ. Types of Modulators<br>
Modulators (i.e., regulators) of the invention include any pharmaceutically acceptable agent that can bind a nucleic acid ligand and modify the interaction between that iigand and its target molecule (e.g., by modifying the structure of the aptamer) in a desired manner, or which degrades, metabolizes, cleaves or otherwise chemically alters the nucleic acid ligand to modify its biological effect. Examples of modulators include (A) oligonucleotides or analogues thereof that are complementary to at least a portion of the aptamer sequence (including ribozymes or      j DNAzymes or, for example, peptide nucleic acids (PNAs), mopholino nucleic acids (MNAs), or <br>
Locked Nucleic Acids (LNAs)), (B) nucleic acid binding peptides, polypeptides or proteins        (including nucleic acid binding tripeptides (see, generally, Hwang et al., Proc. Natl. Acad. Sci.   <br>
USA 96:12997 (1999)), (C) oligosaccharides (e.g. aminoglycosides (see, generally, Davis et al, | Chapter 8, p. 185, KNA World, Cold Spring Harbor Laboratory Press, eds Gestlaad and Atkins j (1993), (D) small molecules, and (E) chimeras, fusion products, or other combinations of any of] the above. Werstuck et al., Sciende 282:296 (1998), USPs 5,935,776 and 5,534,408). (See als0 j the following which disclose types of modulators that can be used in accordance with the present<br>
invention: Chase et al., Ann. Rev. Biochem. 56:103 (1986), Eichorn et al., J. Am. Chem. Soc. <br>
90:7323 (1968), Dale et al., Biochemistry 14:2447 (1975) and Lippard et al., Ace. Chem. Res. <br>
11:211(1978)).	<br>
In an alternative embodiment of the invention, the modulator itself is an aptamer. In this I<br>
embodiment, a nucleic acid ligand is first generated that binds to the desired therapeutic target,<br>
In a second step, a second aptamer that binds to the first aptamer is generated using the SELEX i<br>
process described herein or other process, and modulates the interaction between the therapeutic!<br>
aptamer and the target.	<br>
In other alternative embodiments, the nucleic acid ligand which binds to the target is a  jFNA, MNA, LNA or PCO and the modulator is an aptamer. Alternatively, the nucleic acid     <br>
i<br>
lipnd which binds to the target is a PNA, MNA, LNA or PCO, and the modulator is an MNA jAlternatively, the nucleic acid ligand which binds to the target is a PNA, MNA LNA or PCO,  and the modulator is an LNA. Alternatively, the nucleic acid iigand which binds to the target is ja PNA, MNA, LNA or PCO, and the modulator is a PCO.<br>
Oligonucleotides and Analogues Thereof 1. Polynueleic Acid<br>
In a preferred embodiment, the modulator of the invention is an oligonucleotide<br>
that comprises a sequence complementary to at least a portion of the targeted nucleic acid Jigan4<br>
sequence. For example, the modulator oligonucleotide can comprise a sequence complementary<br>
to 6-25 nucleotides of the targeted nucleic acid ligand, typically, 8-20 nucleotides, more<br>
typically, 10-15 nucleotides. Advantageously, the modulator oligonucleotide is complementary<br>
to 6-25 consecutive nucleotides of the nucleic acid ligand, or 8-20 or 10-15 consecutive<br>
nucleotides. The length of the modulator oligonucleotide can be readily optimized taking into !<br>
account the targeted nucleic acid ligand and the effect sought. Typically the modulator	j<br>
oligonucleotide is 5-80 nucleotides in length, more typically, 10-30 and most typically 15-20 I nucleotides (e.g., 15-17). The oligonucleotide can be made with nucleotides bearing D oirL i stereochemistry, or a mixture thereof. Naturally occurring nucleosides are in the D configuration.<br>
Formation of duplexes by binding of complementary pairs of short<br>
Oligonucleotides is a fairly rapid reaction with second order association rate constants generally between IxlO6 and 3xl05 M1 s1. The initial phase of the binding reaction involves formation of a two to three base pair duplex, termed "nucleation". After nucleation, pairing proceeds through i the entire length of the complementary sequence at a very fast rate - approximately 106 bases   I per second at 20°C. Thus, the moduiatory effect on a nucleic acid ligand by formation of a      j duplex with a complimentary oligonucleotide is rapid. Stability of short duplexes is highly      ! dependent on the length and base-composition of the duplex. The thermodynamic parameters £b!r formation of short nucleic acid duplexes have been rigorously measured, resulting in nearest-   J neighbor rules for all possible base pairs such that accurate predictions of the free energy, Tm thus half-life of a given oligoribonucleotide duplex can be calculated (e.g., Xia et al., Biochem. 37:14719 (1998) (see also Eguchi et al. Antigensis RNA, Annu. Rev. Biochem. 60:631 (1991)).<br>
Oligonucleotide modulators of the invention are advantageously targeted at single-stranded regions of the nucleic acid ligand. This facilitates nucleation and, therefore, the rate of nucleic acid ligand activity modulation, and also, generally leads to intermolecular duplexes contain more base pairs than the targeted nucleic acid ligand.<br>
Various strategies can be used to determine the optimal site for oligonucleotide finding to a targeted nucleic acid Hgand. An empirical strategy can be used in which complimentary oligonucleotides are "walked" around the aptamer A walking experiment can involve two experiments performed sequentially. A new candidate mixture can be produced in   1<br>
*	"	|<br>
which each of the members of the candidate mixture has a fixed nucleic acid- region that	I<br>
corresponds to a oligonucleotide modulator of interest. Each member of the candidate mixture |<br>
also contains a randomized region of sequences. According to this method it is possible to        identify what are referred to as "extended" nucleic acid ligands, which contain regions that can ]<br>
bind to more than one binding domain of an nucleic acid ligand. In accordance with this<br>
approach, 2'-0-methyl oligonucleotides (e.g., 2'-0-methyl oligonucleotides) about 15	!<br>
nucleotides in length can be used that are staggered by about 5 nucleotides on the aptamer ([e.g., i oligonucleotides complementary to nucleotides 1-15, 6-20,11-25, etc. of aptamer 9.3t. An ! empirical strategy can be particularly effective because the impact of the tertiary structure of the] aptamer on the efficiency of hybridization can be difficult to predict. Assays described in the ; Examples that follow can be used to assess the ability of the different oligonucleotides to ! hybridize to a specific nucleic acid ligand, with particular emphasis on the molar excess of the j oligonucleotide required to achieve complete binding of the nucleic acid ligand. The ability of j the different oligonucleotide modulators to increase the rate of dissociation of the nucleic acid j ligand from, or association of the nucleic acid ligand with, its target molecule can also be i determined by conducting standard kinetic studies using, for example, BIACORE assays. ! Oligonucleotide modulators can be selected such that a 5-50 fold molar excess of oligonucleotide, or less, is required to modify the interaction between the nucleic acid ligand and its target molecule in the desired manner.<br>
Alternatively, the targeted nucleic acid ligand can be modified so as to include a single- j stranded tail (3* or 5') in order to promote association with an oligonucleotide modulator. j Suitable tails can comprise 1 to 20 nucleotides, preferably, 1-10 nucleotides, more preferably, 1 j<br>
'	.    j<br>
5 nucleotides and, most preferably, 3-5 nucleotides (e.g., modified nucleotides such as 2'-0- i methyl sequences). Tailed nucleic acid ligands can be tested in binding and bioassays (e.&amp;, as ! described in the Examples that follow) to verify that addition of the single-stranded tail does np^ disrupt the active structure of the nucleic acid ligand. A series of oligonucleotides (for example,!<br>
i<br>
2'-0-methyl oligonucleotides) that can form, for example, 1, 3 or 5 base pairs with the tail<br>
sequence can be designed and tested for their ability to associate with the tailed aptamer alone,<br>
1 as their ability to increase the rate of dissociation of the nucleic acid ligand from, or association of the aptamer with, its target molecule. Scrambled sequence controls can be employed to verify that the effects are due to duplex formation and not non-specific effects. Ihje native gel assay'described in the Examples that follow can be used to measure the dissociation/association rate of an oligonucleotide modulator from/with a target aptamer.<br>
As an illustrative embodiment, the present invention provides modulators that specifically and rapidly reverse the anticoagulant and antithrombotic effects of nucleic acid ligands that target components of the coagulation pathway, particularly nucleic acid ligand      ; antagonists of the tissue factor (TF)/factor Vila (FYHa), factor Vma (FvTHa)/factor DCa (FDCaj), factor Va (FVa/factor Xa (Fxa) enzyme complexes and platelet receptors such as gpflbDIa, gpIblX, gpVI, factors involved in promoting platelet activation such as Gas6, factors involved ijn promoting or maintaining fibrin clot formation such as PAI-1 (plasminogen activator inhibitor j) or coagulation factor XTIIa (FXffla), and additional factors involved in promoting or preventing fibrin clot formation such as ATHt (anti-thrombin HI), thrombin or coagulation factor XIa (FXIa). In accordance with this embodiment, modulators (in this case, inhibitors, advantageously, oligonucleotide inhibitors) are administered that reverse the nucleic acid ligand activity.<br>
In specific embodiments, modulators of nucleic acid ligand activity according to the present invention are nucleic acids selected from the group consisting of, but not limited to the following secpiences; 5'AUGGGGAGGCAGCAUUA 3' (SEQ ID NO:25), 5'CAUGGGGAGGCAGCAUUA3' (SEQ ID N0.26), 5'CAUGGGGAGGCAGCA3' (SEQ Ip NO:27), 5'CAUGGGGAGGCA3' (SEQ ID NO:2S), 5'GCAUUACGCGGUAUAGUCCCCUA3' (SEQ ID N0:29),<br>
5'CGCGGUAUAGUCCCCUA3' (SEQ ID N0:30), 5'CUCGCUG
N0:31), 5'UAUUAUCUCGCUGC5G3' (SEQ ID N0:32), 5'AAGAGCGGGGCCAAG3' (SEQ<br>
ID NO:33), 5'GGGCCAAGUAUUAU 3' (SEQ ED N0:34), 5' CAAGAGCGGGGCCAAG 3'<br>
(SEQ ID NO:35), 5'CGAGUAUUAUCUUG3' (SEQ ID N0:36),	<br>
5'CGCGGUAUAGUCCCCAU3' (SEQ ID N0:41), or any modification or derivative thereof iij which hybridization is maintained or is optionally at least 95% homologous to the sequence.<br>
For example, the inhibitor of a nucleic acid ligand to Factor IXa of the present invention I be an antisense oligonucleotide. The antisense oligonucleotide hybridizes to the nucleic acidj ligand in vivo and blocks the binding of the nucleic acid ligand to factor IXa.<br>
"£he oligonucleotide modulators of the invention comprise a sequence complementary to I<br>
*	.	'    • i<br>
at least a portion of a nucleic acid ligand. However, absolute complementarity is not required. Ai<br>
sequence "complementary to at least a portion of a nucleic acid ligand," referred to herein, meanjs a sequence having sufficient complementarity to be able to hybridize with the nucleic acid        j ligand. The ability to hybridize can depend on both the degree of complementarity and the length of the antisense nucleic acid. Generally, the larger the hybridizing oligonucleotide, the moreb^sje mismatches with a target nucleic acid ligand it can contain and still form a stable duplex (or     | triplex as the case may, be). One skilled in the art can ascertain a tolerable degree of mismatch \ by use of standard procedures to determine the melting point of the hybridized complex. In      j specific aspects, the oligonucleotide can be at least S or at least 10 nucleotides, at least 15 or 17 nucleotides, at least 25 nucleotides or at least 50 nucleotides. The oligonucleotides of the invention can be BNA or KNA or Numeric mixtures or derivatives or modified versions thereof single-stranded.<br>
Antisense techniques are discussed for example, in Okano, J., Neurocheui. 56:560	I<br>
(1991);Oligodeoxynucleotides as Antisense Inhibitors of Gene Expression, CRC Press, Boca	j<br>
RatonJFL (1988). The methods are based on binding of a polynucleotide to a complementary	j<br>
DNAorRNA.	I<br>
Oligonucleotides of the invention can be modified at the base moiety, sugar moiety, or !<br>
phosphate backbone, for example, to improve stability of the molecule, hybridization, etc. The j<br>
oligonucleotide can include other appended groups. To this end, the oligonucleotide can be   I<br>
conjugated to another molecule, e.g., a peptide, hybridization triggered cross-linking agent,    i<br>
transport agent, hybridization-triggered cleavage agent, etc. The oligonucleotide can comprise! at<br>
least one modified base moiety which is selected from the group including, but not limited to, f&gt;-<br>
fluorouracil, 5-bromouracil, 5-chlorouracil, 5-iodouracil, hypoxanthine, xanthine, 4-	j<br>
acetylcytosine, 5-(carboxyhydroxymethyl) uracil, 5-carboxymethylaminomethyl thiouridine, 5J-carboxymethylaminomethyluracil, dihydrouracil, beta-D-galactosylqueosine, inosine,N6-      \ isopentenyladenine, 1 -methylguanine, 1-methylinosine, 2,2-diraethylguanme,2-methyladenin£,<br>
2-methylguanine, 3-inethylcytosine, 5-methylcytosine, N6-adenine,7-methylguanine, 5-	,<br>
j^thyiaminomethyluracil, 5-methoxyaminomethyl-2a-thiouracil, beta-D-mannosylqueosine, 5'-!<br>
methoxycarboxymethyluracil, 5-methoxyuracii, 2-inethyithio-N&amp;isopentenyladenine, uraci!     i<br>
oxyacetic acid (v), wybutoxosine, pseudouracil, queosine, 2-thiocytosine, 5-methyl thiouracil, 2 j<br>
i<br>
thiouracil, 4-thiouraeil, 5-methyluracil, - uracil-5-oxya*etic acid methylester, uracil oxyacetic acid (v), 5-methyl thiouracil, 3-(3-amino-3-N carboxypropyl) and 2,6-diaminopurine.<br>
An oligonucleotide modulator of the invention can also comprise at least one modified sugar moiety selected from the group including, but not limited to, arabinose, 2-fluoroaf abmose, xylose, and hexose.<br>
In yet another embodiment, an oligonucleotide modulator can comprise at least One modified phosphate backbone selected from the group including, but not limited to, a phosphorothioate, a phosphorodithioate, a phosphoramidothioate, a phosphoramidate, a phosphorodiamidate, a methylphosphonate, an alkyl phosphotriester, and a fonnacetal or analog thereof.<br>
Oligonucleotide modulators can be prepared that have desired characteristics, such as improved in vivo stability and/or improved delivery characteristics. Examples of such modifications include chemical substitutions at the sugar and/or backbone and/or base positions Oligonucleotide modulators can contain nucleotide derivatives modified at the 5- and 2* positions of pyrimidines, for example, nucleotides can be modified with 2* ammo, 2'-fluoro and/or 2-O-methyl. Modifications of the modulating oligonucleotides of the invention can include those that provide other chemical groups that incorporate additional charge, polarization, hydrophobicity,; hydrogen bonding and/or electrostatic interaction. Such modifications include but are not limited to 2* position sugar modifications, locked nucleic acids, 5 position pyrimidine modifications, 8 position purine modifications, modification at exocyclic amines, substitution of 4-thiouridine,   \ substitution of 5-bromo or 5-iodo-uracil, backbone modifications, phophorothioate or alkyl      j phosphate modifications, methylations, unusual base-pairing combinations such as isobases     i isocytidine and isoguanidine, etc. Modifications can also include 3* and 5' modifications, such as capping. (See also Manoharan, Biochem. Biophys. Acta 1489:117 (1999); Herdewija, Antisense Nucleic Acid Drug Development 10:297 (2000); Maier et al, Organic Letters 2:1819 (2000)).<br>
Oligonucleotides of the invention can be synthesized by standard methods known in the B,g. by use of an automated DNA synthesizer (such as are commercially available from Biosearch, Applied Biosystems, etc.).<br>
Using the instructions provided herein, one of ordinary skill can easily practice the<br>
»<br>
invention to regulate any therapeutic nucleic acid by the timely administration of a complementary nucleic acid that terminates or otherwise modifies the activity of the therapeutic ligand. This technique ctn be used, for example, in connection with any of the nucleic acid ligands described or referred to in the patent documents cited above.<br>
2. Ribozymes and DNAzymes<br>
Likewise, using the instructions provided herein, one of ordinary skill can easily practice! the invention to regulate any therapeutic nucleic acid ligand by the timely administration of a ribozyme or a DNAzyme.<br>
Enzymatic nucleic acids act by first binding to a target RNA (or DNA, see Cech U J». Pat. No. 5,180,818). Such binding occurs through the target binding portion of an enzymatic nucleic acid which is held in close proximity to an enzymatic portion of molecule that acts to cleave the target RNA. Thus, the enzymatic nucleic acid first recognizes and then binds a target RNA through complementary base-pairing, and once bound to the correct site, acts enzymatically to cut the target RNA. Cleavage of such a target RNA will destroy its ability to direct synthesis of an encoded protein. After an enzymatic nucleic acid has bound and cleaved its RNA target it is released from that RNA to search for another target and can repeatedly bind and cleave.new targets thereby allowing jot inactivation of RNA aptamers. There are at least five classes of ribozymes that each display a different type of specificity. For example, Group I Introns are ~300 to &gt;1000 nucleotides in size and require a U in the target sequence immediately 5' of the cleavage site and binds 4-6 nucleotides at the 5-side of the cleavage site. There are over 75 known members of this class, they are found in Tetrahymena thermophila rRNA, fungal mitochondria, chloroplasts, phage T4, blue-green algae, and others. Another class are RNaseP RNA (Ml RNA), which are -290 to 400 nucleotides in size. They qn|<br>
;   I<br>
the RNA portion of a ribonucleoprotein enzyme and cleave the tRNA precursors to form matutej tRNA. There are roughly 10 known members of this group and all are bacterial in origin. A<br>
example are Hammerhead Ribozyme, which are ~30 to 40 nucleotides in size. They require tiej<br>
et sequence UH immediately 5' of the cleavage site and bind a variable number nucleotides I<br>
i on both sides of the cleavage site. There are at least 14 known members of this'class that are    i<br>
found in a number of plant pathogens (virusoids) that use RNA as the infectious agent. A fburtji<br>
class are the Hairpin Ribozymes, which are -50 nucleotides in size. They requires the tarfet    I<br>
sequence GUC immediately 3' of the cleavage site and bind 4 nucleotides at the 5'-side of the   i<br>
cleavage site and a variable number to the 3-side of the cleavage site. There are found in one   !<br>
plant pathogen (satellite RNA of the tobacco ringspot virus) which uses RNA as<br>
the infectious agent. The fifth group are Hepatitis Delta Virus (HDV) Ribozymes, which are ^<br>
nucleotides in size.	'<br>
The enzymatic nature of a ribozyme can be advantageous over other technologies, such |<br>
I<br>
as antisense technology, in certain applications since the effective concentration of ribozyme necessary to effect a therapeutic treatment can be less than that of an antisense oligonucleotide. This advantage reflects the ability of the ribozyme to act enzymaticaily. Thus, a single ribozymje molecule is able to cleave many molecules of target RNA aptamers. In addition, the ribozyme is a highly specific inhibitor, with the specificity of inhibition depending not only on the base     |<br>
',<br>
pairing mechanism of binding, but also on the mechanism by which the molecule.inhibits the<br>
expression of the RNA to which it binds. That is, the inhibition is caused by cleavage of the    i<br>
RNA target and so specificity is defined as the ratio of the rate of cleavage of the targeted RNA<br>
over the rate of cleavage of non-targeted RNA. This cleavage mechanism is dependent upon<br>
factors additional to those involved in base pairing. Thus, the specificity of action of a ribozyme,<br>
under certain circumstances, can be greater than that of antisense oligonucleotide binding the<br>
same RNA site.	'<br>
Another class of catalytic molecules are called "DNAzymes". DNAzymes are single- j stranded, and cleave both RNA and DNA A general model for the DNAzyme has been proposed, and is known as the "10-23" model. DNAzymes following the "10-23" model, also ! referred to simply as "10-23 DNAzymes", have a catalytic domain of 15 deoxyribonucleotides,! flanked by two substrate-recognition domains of seven to nine deoxyribonucleotides each. In j vitro analyses show that this type of DNAzyme can effectively cleave its substrate RNA at j purine:pyrimidine junctions under physiological conditions. As used herein, "DNAzyme" mefn|s<br>
a DNA molecule that specifically recognizes and cleaves a distinct target nucleic acid sequence, ^•fch may be either DNA or RNA.<br>
Ribozymes and DNAzymes comprise both substrate binding domains and catalytic domains. Rules for the optimal design of the regulatory ribozymes can be found within Methods in Molecular Biology Volume 74 "Rifcozyme Protocols" edited by Philip C. Turner (Huifnana Press, Totowa, NJ, 1997). Strategies to identify reactive sites for ribozymes within a targeted    j nucleic acid are also described in this volume. Using these standard rules, hammerhead ribozymes, hairpin ribozymes, hepatitis delta virus ribozymes and ribozymes derived from group!<br>
I<br>
I introns can be specifically engineered to bind to and cleave a target nucleic acid ligand. The   I<br>
ability of such ribozymes or DNAzymes to cleave the target nucleic acid ligand can be determined in any number of assays. For example, following incubation of a nbozyme or DNAzyme with an unlabeled or a labeled (eg. 32P) target nucleic acid ligand under conditions favoring reaction, the target nucleic acid can be analyzed by denaturing gel electrophoresis to determine if the nbozyme or DNAzyme cleaved the backbone of the target nucleic acid ligand Alternatively, fluorescence resonance energy transfer, or change in fluorescence intensity can be used to measure cleavage of an appropriately labeled nucleic acid ligand.<br>
3. Polyniicleic Acid Analogues (Peptidc Nucleic Acids (PNAs), Morphelino I<br>
Nucleic Acids (MNAs), Locked Nucleic Acids (LNAs)), and Pseudo-Cyclic<br>
Oligonudeotides (PCOs)	!	I<br>
Nucleobases of the oligonucieotide modulators of the invention can be connected via    j internucleobase linkages, e.g., peptidyl linkages (as in the case of peptide nucleic acids (PNAs);<br>
i<br>
Nielsen et al. (1991) Science 254,1497 and U.S. Pat. No. 5,539,082) and morpholino linkages | (Qin et al., Antisense Nucleic Add Drug Dev. 10,11 (2000); Summerton, Antisense Nucleic j Acid Drug Dev. 7,187 (1997); Summerton et al., Antisense Nucleic Acid Drug Dev, 7,63 ' \ (1997); Taylor et al., J Btol Cbem. 271, 17445 (1996); Partridge et al., Antisense Nucleic Acid i Drug Dev. 6,169 (1996)), or by any other natural or modified linkage. The oligonucleobases cajn also be Locked Nucleic Acids (LNAs). Nielsen et al., J Biomol Struct Dyn 17,175 (1999); j Petersen et al., J Mol Recognit 13,44 (2000); Nielsen et al., Bioconjug Chem 11,228 (2000). j<br>
PNAs are compounds that are analogous to oligonucleotides, but differ in composition. In s, the deoxyribose backbone of oligonucleotide is replaced with a peptide backbone. Each subunit of the peptide backbone is attached to a naturally-occurring or non-naturally-occurring nucleobase. PNA often has an achiral polyamide backbone consisting of N-(2-aminoethyl)gr/cine units. The purine or pyrimidine bases are linked tb each unit via a methyjejn carbonyl linker (1-3) to target the complementary nucleic acid. PNA binds to complementary RNA or DNA in a parallel or aatiptrallel orientation following the Watson-Crick base-pairing rales. The uncharged nature of the PNA oligomers enhances the stability of the hybrid PNA/DNA(RNA) duplexes as compared to the natural homoduptexes. The non-natural character of the PNA makes PNA oligomers highly resistant to protease and nuclease attacks. These properties of PNA oligomers allow them to serve as efficient antisense or modulators of nucleic acid ligand activity. Indeed, peptide nucleic acids have been applied to block protein expression on the transcriptional and translational level, and microinjected PNA oligomers demonstrate a strong antisense effect in intact cells. See www.bjoscience.org/l 999A4/d/soomets/fijintjex^htni:<br>
and Frontiers in Bioscience, 4, d782-786 (Nov 1,1999) for details on recent achievements on PNA antisense application, especially these concerned with whole cell or tissue delivery of the PNA. See also Nielsen, Pi., Ifholm. M., Berg, RH. &amp; Buchardt, 0. (1993) Peptide nucleic acids (PNA). DNA analogues with a polyamide backbone. In "Antisense Research and Application" Crook, S. &amp; Lebleu, B. (eds.) CRC Press, Boca Raton, pp 363-373.<br>
PNAs bind to both DNA and RNA and form PNA/DNA or PNA/RNA duplexes. The resulting PNA/DNA or PNA/RNA duplexes are bound tighter than corresponding DNA/DNA or DNA/RNA duplexes as evidenced by their higher melting temperatures (Tm). This high thermal stability of PNA/DNA(RNA) duplexes has been attributed to the neutrality of the PNA backbone, which results elimination of charge repulsion that is present in DNA/DNA or RNA/KNA duplexes. Another advantage of PNA/DNA(RNA) duplexes is that Tm is practically independent of salt concentration. DNA/DNA duplexes, on the other hand, are highly dependent on the ionic strength.<br>
Since PNAs are an analogue of DNA in which the backbone is a pseudopeptide rather than a sugar, they mimic the behavior of DNA and bind complementary nucleic acid strands. Unnatural nucleobases, such as pseudo isocytosine, 5-methyl cytosine and 2,6-diaminopurine,I	i<br>
among many others, also can be incorporated in PNA synthons. PNAis are most commonly<br>
synthesized from monomers (PNA synthons) protected according to the t-Boc/benzyl protectldn<br>
Jr	'    i<br>
strategy, wherein the backbone amino group of the growing polymer is protected with the t- i butyloxycarbonyl (t-Boc) group and the exocyclic arahio groups of the nucleobases, if present, are protected with the benzyloxycarbonyl (benzyl) group. PNA synthons protected using the t» Boc/benzyl strategy are now commercially available.<br>
PNA is both biologically and chemically stable and readily available by automated<br>
synthesis ((Hyrup, B., Egholm, M., Holland, M, Nielsen, P. E,, Berg, R. H. &amp; Buchardt, 0.<br>
(1993) Modification of the binding affinity of peptide nucleic acids (PNA). PNA with extended<br>
backbones consisting of 2-Aminoethyl—Alanine or 3-Aminopropylglycine units. J. Chem. Sac.<br>
Cheat, Common. 518-519); Demidov, V., Frank-Kamenetskii, M.D., Egholm, M., Buchardt, O.<br>
&amp; Nielsen, P.E. (1993). Sequence selective double strand DNA cleavage by PNA targeting<br>
using nuclease SI. Nucleic Acids Res. 21,2103-2107). These properties have made PNA very<br>
good lead for antisense gene therapeutic drags, and in vitro studies have further substantiated the<br>
antisense potential (Nielsen, P£. "Peptide Nucleic Acid (PNA) A model structure for the<br>
primordial genetic materiar Origins of Life 1993,23,323-327; Eghohn, M., Behrens, C,<br>
Christensen, L., Berg, RJL, Nielsen, P.E. &amp; Buchardt, O. "Peptide nucleic acids containing<br>
adenine or guanine recognize thymine and cytosine in complementary DNA sequences" J.<br>
Chem. Soc. Chem. Common. 1993, 800-801; Kim, S.K., Nielsen, P.E., Egholm, M., Buchardt,<br>
0., Berg, RH. &amp; Norden, B. "Right-handed triplex formed between peptide nucleic acid PNA4<br>
It and poly(dA) shown by linear and circular dichroism spedroscopy" J. Amer. Chem. S&amp;c.<br>
1993,115,6477-6481; Egholm, M., Buchardt, 0., Christensen, L., Behrens, C., Freier, S.M.,<br>
Driver, D.A., Berg, R.H., Kim, S.K., Norden, B. &amp; Nielsen, P.E. "PNA hybridizes to<br>
complementary oligonucleotides obeying the Watson-Crick hydrogen bonding rules" Nature<br>
1993,365, 556-568; Buchardt, O., Egholm, M., Berg, R. &amp; Nielsen, P.E. "Peptide Nucleic Acijds<br>
(PNA) and their potential applications in medicine and biotechnology" Trends Biotechnology,<br>
1993, //, 384-386).	j<br>
PNAs appear to be the very useful for a number of special applications. When targeted! against rare all-purine sequences, PNAs can block translation anywhere in a mRNA by forming a doubte-claiap structure. With such rare RNA sequences one segment of the PNA binds to the j target sequence by Watson/Crick bonds and the other segment of the PNA binds to $ of tte resulting PNA/RNA duplex. Probably because of their very flexible backbone<br>
structure, PNAs also readily form triple helix structures with rare duplex DNA sequences jiteaprising mostly purines in one strand and mostly pyrimidines in the other strand. Lastly,<br>
^WIP^<br>
under low salt conditions in cell-free systems PNAs have been shown to achieve sequence-<br>
specific invasion of duplex DNA sequences, resulting in inhibftion of transcription of the<br>
invaded duplex.	»<br>
i;<br>
Recent studies by several groups have shown that coupling of PNA to different carriers can improve their uptake into cells. Among these, "cellular uptake peptides," fatty acids or DN<br>
'.	'•<br>
especially several peptide sequences have been shown to be able to carry PNA oligomers across the ceil membranes. Vector peptide-PNA conjugates have been shown to cross the neuron membrane and suppress targeted mRNA (Aldrian-Herrada, G. et al. (1998) Nucleic Acid Res. 26:4920). Biotinylated PNA linked to a conjugate of steptavidin and the OX26 murine monoclonal antibody to the rat transferrin receptor have been reported to cross the rat blood-brain barrier in vivo (Pardridge, W. et al. 1995 PNAS 92:5592-5596). Chinnery, P. F. et al attached the presequence peptide of the nuclear-encoded human cytochrome c oxidase (COX) subunh VIZI to biotinylated PNA which was successfully imported into isolated mitochondria in vitro (Chinnery, P. F. et al. 1999 Gene Ther. 6:1919-28). Delivery of the biotinylated peptide-PNA to mitochondria in intact cells was confinned by confocal microscopy. In addition, a short hydropbobic peptide with the sequence biotinyl-FUFL coupled to a PNA trimer has been shown to internalize into human erythrocytes and Namahva cells (Scarfi, S., A. Gasparini, G. Damonte &amp; U. Benatti: Synthesis, uptake, and intracellular metabolism of a hydrophobia tetrapeptide-peptide nucleic acid (PNA)-biotin molecule. BiochemBiophysRes Common 1997,236,323-    ! 326). Basu and Wickstrdm (Synthesis and characterization of a peptide nucleic acid conjugated to aD-peptide analog of insulin-like growth factor 1 for increased cellular uptake. Bioconjvgate Chem 1997,8,481-488) showed that PNA conjugated fc&gt; an all-D-amino acid insulin-like growth factor 1 (IGF1) mimicking peptide was specifically taken up by cells expressing the IGF1 receptor, although no antisense activity was described.<br>
For some examples of other works, see Nielsen, P. E., M. Egholm, R, H. Berg &amp; 0. Buchardt: Sequence-selective recognition of DNA by strand displacement with a thymine-substituted polyamide. Science 254, 1497-1500 (1991); Egholm, M., O. Buchart, P. E. Nielsen ^ It H. Berg: Peptide nucleic acids (PNA). Oligonucleotide analogues with an achiral peptide backbone. JAm Chan Soc 114,1895-1897 (1992)^ Nielsen, P. E., M. Egholm &amp; 0. Buchardt:<br>
?eptide nucleic acid (PNA). A DNA mimic with a peptide backbone. Biocoiyugate Chemistry 5 1 (1994); Egholm, M., P. E. Nielsen, 0. Buchardt &amp; R. H. Berg: Recognition of guanine and adenine in DNA by cytosine and thymine containing peptide nucleic acid. J Am Chem Soc 114t<br>
9677-9678 (1992); Egholm, M., 0. Buchardt, L. Christensen, C. Behrens, S. M. Freier, D. A<br>
*	»<br>
Driver, R. H. Berg, S. K. Kim, B. Norden &amp; P. E. Nielsen: PNA hybridizes to complementary<br>
oligonucleotides obeying the Watson-Crick hydrogen-bonding rules [see comments]. Nature<br>
365, 566-568 (1993); Wittung, P., P. E. Nielsen, O. Buchardt, M. Egholm &amp; B. Norden: DNA-<br>
like double helix formed by peptide nucleic acid. Nature 368, 561-563 (1994); Brown, S. C., S.<br>
A. Thomson, J. M. Veal &amp; D. G. Davis: NMR solution structure of a peptide nucleic acid<br>
complexed with RNA Science 265, 777-780 (1994); Demidov, V. V., V. N. Potaman, M. D.<br>
Frank-Kamenetskii, M. Egholm, 0. Buchard, S. H. Sdnnichsen &amp; P. E. Nielsen: Stability of<br>
peptide nucleic acids in human serum and cellular extracts. Biochemical Pharmacology 48,<br>
1310-1313 (1994); Peffer, N. I, J. C. Hanvey, J. E. Bisj, S. A. Thomson, C. F. Hassman, S. A<br>
Noble &amp; L. E. Babiss. Strand-invasion of duplex DNA by peptide nucleic acid oligomers. Proc<br>
NatAcadSciUSA 90,10648-10652(1993).	";<br>
US Patent No. 6,046,307 to Shay et al. discloses PNAs that inhibit telomerase activity in i<br>
mammalian cells. U.S. Patent No. 5,789,573 to Baker et al. disclose compositions and methods !<br>
for inhibiting the translation of capped targeted mRNA using PNAs. U.S. Patent No. 6,165,720 1<br>
disclose labeling of the PNA complex.	One can easily prepare a PNA or MNA that is complementary at least in part to a nucleic! acid ligand using the same rules of complementarity and Watson Crick base-pairing as is used hi the preparation of conventional oligonucleotide antisense molecules.<br>
Morpholino nucleic acids are so named because they are assembled from morpholino<br>
subunits, each of which contains one of the four genetic bases (adenine, cytosine, guanine, and<br>
thymine) linked to a 6-membered morpholine ring. Eighteen to twenty-five subunits of these four<br>
subunit types are joined in a specific order by non-ionic phosphorodiamidate intersubunh	;<br>
linkages to give a morpholino oligo. These morpholino oligos, with their 6-membered morpholine backbone moieties joined by non-ionic linkages, afford substantially better antisense properties than do RNA, DNA, and their analogs having 5-membered ribose or deoxyribose backbone moieties joined by ionic linkages (see wwwgene-tc
Morpholinos, devised by Summerton in 1985, constitute a radical redesign of natural leic acids, with the potential advantages of low cost starting materials and inexpensive assembly. Like PNAs, morpholiaos are completely resistant to nucleases and they appear to bie i<br>
|<br>
free of most or all of the non-aotisense effects that are seen with S-DNAs. In contrast to PNAs, i<br>
;    j<br>
most morpholinos exhibit excellent aqueous solubility. Morpholinos also have much higher RKA binding affinities than do S-DNAs, though not as high as PNAs. Probably as a result of their   | substantial RNA binding affinities, long morpholinos (25-mers) provide predictable targeting and very high efficacy. Most notable, morpholinos provide good sequence specificity. The same factors that underlie their exceptional sequence specificity also render them unsuitable for targeting point mutations.<br>
U.S. Patent No, 6,153,737 to Manoharan et al. is directed to derivatized oligonucleotides wherein the linked nucleosides are functionalized with peptides, proteins, water soluble<br>
: I or lipid soluble vitamins. This disclosure was directed towards antisense therapeutics by<br>
i modification of oligonucleotides with a peptide or protein sequence that aids in the selective<br>
entry of the complex into the nuclear envelope. Similarly, water-soluble and lipid-soluble vitamins can be used to assist in the transfer of the anti-sense therapeutic or diagnostic agent across cellular membranes.<br>
The efficient and sequence specific binding to RNA or DNA combined with very high<br>
biological stability makes PNAs and MNAs extremely attractive leads for the development of<br>
modulators of the present invention. Peptide nucleic acids which can be conjugated to the<br>
carriers are distinguished in U.S. Patent No. 5,864,010 entitled Peptide Nucleic Acid<br>
Combinatorial Libraries and Improved Methods of Synthesis, developed by ISIS<br>
Pharmaceuticals, which is hereby incxjrporated by reference. In addition, the peptide nucleic<br>
acids which can be conjugated to the carriers of the present invention are distinguished in U.S.<br>
Patent No. 5,986,053 entitled peptide nucleic acids complexes of two peptide nucleic acid<br>
strands and one nucleic add strand.	[<br>
LNA is a novel class of DNA analogues that possess some features that make it a prime candidate for modulators of the invention. The LNA monomers are bi-cyclic compounds        i structurally similar to RNA-monomers. LNA share most of the chemical properties of DNA and RNA, it is water-soluble, can be separated by gel electrophoreses, ethanol precipitated etc      I (Tetrahedron, 54,3607-3630'(1998)). However, introduction of LNA monomers into eithfer Off A<br>
or RNA oligos results in high thermal stability of duplexes with complementary DNA or RHA, njjtte, at the same time obeying the Watson-Crick base-pairing rules. This high thermal stability j of the duplexes formed with UNA oligomers together with the finding that primers containing 3f 1 located IJNA(s) are substrates for enzymatic extensions, e.g. the PCR reaction, is used in the I present invention to significantly increase the specificity of detection of variant nucleic acids in j the in vitro assays described in the application. The amplification processes of individual alleles j<br>
"    I<br>
occur highly discriminative (cross reactions are not visible) and several reactions may take place in the same vessel. See for example U.S. Patent No. 6,316,198.<br>
Pseudo-cyclic oligonucleobases (PCOs) can also be used as a modulator hi the present invention (see U.S. patent No. 6,383,752). PCOs contain two oligooucleotide segments attached through their 3'-3' or 5-5' ends. One of the segments (the "functional segment") of the PCO has some functionality (e.g., an antisense oligonucleotide complementary to a target mRNA). Another segment (the "protective segment") is complementary to the 3'- or S'-terminal end of the functional segment (depending on the end through which it is attached to the functional<br>
segment). As a result of complementarity between the functional and protective segment<br>
! segments, PCOs form intramolecular pseudo-cyclic structures in the absence of the target nucleic<br>
acids (e.g., RNA). PCOs are more stable than conventional antisense oiigonucleotides because of<br>
the presence of 3-3* or 5'-5'linkages and the formation of intramolecular pseudo-cyclic	j<br>
structures. Pharmacokinetic, tissue distribution, and stability studies hi mice suggest that PCOs have higher in vivo stability than and, pharmacokinetic and tissue distribution profiles similar to,<br>
i<br>
those of PS-oligonucleotides in general, but rapid elimination from selected tissues. When a      I<br>
fluorophore and quencher molecules are appropriately linked to the PCOs of the present	i<br>
invention, the molecule will fluoresce when it is in the linear configuration, but the fluorescence is quenched in the cyclic conformation.<br>
A.       Nucleic Acid Binding Peptides, Polypeptides or Proteins<br>
Protein-nucleic add interactions are involved in many cellular functions, including transcription, RNA splicing, and translation. Synthetic molecules that can bind with high affinity to specific sequences of single-stranded nucleic acids have the potential to interfere with these interactions in a controllable way.<br>
Many proteins have been identified that can bind directly to nucleic acid sequences. One! ain group of proteins that directly bind to DNA are transcription factors. Some examples of 1 scription factors fall into the following classes: Basic domain transcription factors (leucine- ! zipper factors, CKEB, helix-loop-helix factors, helix-loop-helix/ leucine zipper factors, cell-cy<?l controlling factors, NF-1, RF-X, bHSH), Zinc-coordinating DNA binding domain transcription | factors (cys4 zinc finger of nuclear receptor type, thyroid hormone receptor-like factors, diverse!<br/>
!<br>
Cys4 zinc fingers, Cys2His2 zinc finger domain, metabolic regulators in fungi, large factors with<br>
I<br>
NF-6B-iike binding properties, cy s6 cy steine-zinc cluster, zinc fingers of alternating	j<br>
composition, homeo domain), Helix-turn-helix (homeo domain only, POU domain factors,       I paired box, fork head/ winged helix, heat shock factors, tryptophan clusters, TEA domain), beta-j<br>
I<br>
Scaffold Factors with minor groove contacts (RHR, STAT, p53, MADS box, beta-Barrel alpha-1 helix transcription factors, TATA-binding proteins, HMG, heteromeric CCAAT factors, grainyhead, cold-shock domain factors, runt), Other transcription factors (copper fist proteins, HMGI(Y), Pocket domain, El A-like factors, AP2/EREBP-related factors).<br>
In addition to its primary structure, RNA has the ability to fold into complex tertiary<br>
structures consisting of such local motifs as loops, bulges, pseudoknots, and turns. It is not<br>
surprising that, when they occur in RNAs that interact with proteins, these local structures are<br>
found to play important roles in protein-RNA interactions. This diversity of local and tertiary<br>
structure, however, makes it impossible to design synthetic agents with general, simple-to-use   j<br>
recognition rules analogous to those for the formation of double- and triple-helical nucleic acids}<br>
Tripeptides have been identified that are able to specifically bind to tertiary structures of nucleic]<br>
acids (Hwang et al, Proe. Nail. Acad. Sci. USA 96:12997 (1999)). Thus, hi one embodiment,  I<br>
these tripeptides may be used as modulators of nucleic acid ligand activity.	!<br>
!<br>
Peptide-based modulators of nucleic acid ligands represent an alternative molecular class of modulators to oligonucleotides or their analogues. This class of modulators are particularly prove usefUl when sufficiently active oligonucleotide modulators of a target nucleic acid ligand j can not be isolated due to the lack of sufficient single-stranded regions to promote nucleation between the target and the oligonucleotide modulator. In addition, peptide modulators provide different bioavailabilities and pharmacokinetics than oligonucleotide modulators, and therefore,! for some target nucleic acid ligands may be preferable.<br>
Several strategies to isolate peptides capable of binding to and thereby modulating tie     j<br>
nty of a target nucleic acid ligand are available. For example, encoded peptide	|<br>
combinatorial libraries immobilized on beads have been described, and have been demonstrated j to contain peptides able to bind viral RNA sequences and disrupt the interaction between the      j<br>
viral RNA and a viral regulatory protein that specifically btnds said RNA (Hwang et al., Proc     !<br>
i'	' 1<br>
Natl Acad Sci USA 96:12997 (1999)). Using such libraries, modulators of nucleic acid ligands   |<br>
can be isolated by appending a label (eg. Dye) to the target nucleic acid ligand and incubating    i<br>
• i together the labeled-target and bead-immobilized peptide library under conditions in which        I<br>
binding between some members of the library and the nucleic acid are favored. The binding of ! the nucleic acid ligand to the specific peptide on a given bead causes the bead to be "colored" by the label on the nucleic acid ligand, and thus enables the identification of peptides able to bind<br>
the target but simple isolation of the bead. The direct interaction between peptides isolated by    , j such screening methods and the target nucleic acid ligand can be confirmed and quantified using j<br>
any number of the binding assays described to identify modulators of nucleic acid ligands. The ability of said peptides to modulate the activity of the target nucleic acid ligand can be confirmed! in appropriate bioassays.<br>
As another strategy, the target nucleic acid ligand can be immobilized on a solid support i<br>
(eg plastic, bead etc) by appending the appropriate affinity handle (eg biotin for streptavidin I<br>
coated surfaces) or reactive site (eg. Aliphatic amine) to either end of the target nucleic acid<br>
ligand during chemical synthesis. Alternatively, biotinylated RNA or RNA containing the j<br>
desired modification can be prepared according to standard transcription protocols, but including<br>
a 5-fold molar excess of a 5-biotin-modified GMP or otherwise modified GMP over GTP in the j<br>
reaction mixture. Methods for synthesizing 5'-biotin-modified guanosine nucleotides and	!<br>
additional guanosine nucleotide derivatives and incorporating said GMP derivatives into in vitro j transcripts are described in WO 98/30720 entitled "Bioconjugation of Oligonucleotides". Once the target RNA has been immobilized on a surface, peptides capable of binding said target RNA ( can be isolated by screening combinatorial peptide libraries, for example, random phage display libraries for peptides capable of binding the target nucleic acid ligand using standard methods for screening phage display libraries. Once isolated, the direct interaction between peptides and the j target nucleic acid ligand can be confirmed and quantified using any number of the binding assays described to identify potential modulators of nucldc acid ligands. The ability of said<br>
tides to modulate the activity of the target nucleic acid ligand can be confirmed in appropriate Sioassays.<br>
In addition, the mass spectral assays described can be used to screen amino acid and I small molecular weight peptide libraries to identify those capable of binding to the target nuclei]; acid Ugand (see U.S. Patent Nos. 6,342,393; 6,329,146; 6,253,168; and 6,221,587). Once j identified, the direct interaction between such agents and the target nucleic acid ligand can be I confirmed and quantified using any number of the binding assays described to identify potential modulators of nucleic acid ligands. The ability of said peptides to modulate the activity of the i<br>
I<br>
target nucleic acid ligand can be confirmed in appropriate bioassays.	j<br>
In addition, USPs 5,834,184, 5,786,145 and 5,770,434 disclose methods for generating j<br>
and screening for peptide modulators of the invention.	j<br>
|<br>
B. OHgosaccharides	j<br>
I<br>
Oligosaccharides can interact with nucleic acids. The antibiotic aminoglycosides are    J products of Streptomyces species and are represented by streptomycin, kanamycin, tobrarnycln| neomycin, netilmicin, amikacin and gentamicin. These antibiotics exert their activity by binding to bacterial ribosomes and preventing the initiation of protein synthesis. Aminoglycoside       j antibiotics interact specifically with diverse RNA molecules such as various ribozymes and the HTV-Vs TAR and RRE sequences. Aminoglycoside antibiotics that bind to the 30s ribosomal JA-site RNA cause misreading of the genetic code and inhibit translocation. The aminoglycoside antibiotic paromomycin binds specifically to the RNA oligonucleotide at the SOS A site. This j antibiotic binds the major groove on the A-site RNA within a pocket created by an A-A paired<br>
I<br>
a single bulged adenine. There are several interactions that occur between the aminoglycoside chemical groups and the conserved nucleotides in the RNA<br>
Thus, oligosaccharides, like aminoglycosides, can bind to nucleic acids and can be used to modulate the activity of nucleic acid ligands. In addition, the mass spectral assays described can be used to screen for oligosaccharides (e.g., aminoglycosides), alone or in a mixture, that {are capable of binding to the target nucleic acid ligand (see U.S. Patent Nos. 6,342,393; 6,329,14^; 6,253,168; and 6,221,587).<br>
D. Small Molecules<br>
A small molecule that intercalates between the nucleic acid ligand and the target or<br>
/ise disrupts or modifies the binding between the nucleic acid ligand and target can also be]<br>
used as the therapeutic regulator.	j<br>
Such small molecules can be identified by screening candidates in an assay that measured binding changes between the nucleic acid iigand and the target with and without the small     * j molecule, or by using an in vivo or in vitro assay that measures the difference in biological effecf; of tiie nucleic acid ligand for the target with and without the small molecule. Once a small molecule is identified that exhibits the desired effect, techniques such as combinatorial approaches can be used to optimize the chemical structure for the desired regulatory effect.<br>
Assays suitable for use in screening small molecules are described in USP 5,834,199 and in further detail below. In addition, the mass spectral assays described can be used to screen small molecule libraries to identify those capable of binding to the target nucleic acid ligand (see U.S. Patent Nos. 6,342,393; 6,329,146; 6,253,168; and 6,221,587).<br>
E. Chimeras, Fusion Products and Otherwise Linked Materials<br>
The oligonucleou'des or analogues thereof (including ribozymes or DNAzymes or peptidje nucleic acids (PNAs) or mopholino nucleic acids (MNAs)), nucleic acid binding peptides, polypeptides or proteins (including nucleic acid binding tripeptides, oligosaccharides, and small<br>
molecules can be combined covalently or noncovalently to provide a combination regulator as j desired. In one example, an oligonucleotide, PNA or MNA can be linked to a peptide or protein to provide desired properties or therapeutic effect. Alternatively, an oligonucleotide or analogue thereof or peptide or protein can be linked to a small molecule to enhance its properties. These j materials can be directly linked or linked via a spacer group as well known to those skilled in the art In one embodiment, the small molecule includes a radioligand or other detectable agent that can be monitored using standard techniques. In this way, the progress and location of the regulator can be monitored. Alternatively, the small molecule is a therapeutic that is brought to j the site of therapy by the regulator, and then optionally cleaved to provide its therapeutic effect! at the appropriate location. In a third embodiment, the small molecule is a chelator that joins two biological materials together to produce a combined effect.<br>
The chimeras, fission products or otherwise multicomponent regulator can be identified! evaluated for therapeutic efficacy as described in detail below.<br>
Identification and Selection of the Modulator<br>
Standard binding assays can be used to identify and select modulators of the invention.<br>
Nonlimiting examples are gel shift assays and BIACORE assays. That is, test modulators can be<br>
contacted with the nucleic acid ligands to be targeted under test conditions or typical	i<br>
physiological conditions and a determination made as to whether the test modulator in fact binis the nucleic acid ligand. Test modulators that are found to bind the nucleic acid ligand can then be analyzed in an appropriate bioassay (which will vary depending on the aptamer and its target<br>
i<br>
molecule, for example coagulation tests) to determine if the test modulator can affect the biological effect caused by the nucleic acid ligand on its target molecule.<br>
The Gel-Shift assay is a technique used to assess binding capability. For example, a DNA fragment containing the test sequence is first incubated with the test protein or a mixture j<br>
containing putative binding proteins, and then separated on a gel by electrophoresis. If the DNA<br>
I fragment is bound by protein, it will be larger hi size and its migration will therefore be retard«jd<br>
relative to that of the free fragment. For example, one method for a electrophoretic gel mobility<br>
shift assay can oe (a) contactmg m a mixture a nucleic acid binding protein with a non-	!<br>
radioactive or radioactive labeled nucleic acid molecule comprising a molecular probe under suitable conditions to promote specific binding interactions between the protein and the pfobe in<br>
!<br>
forming a complex, wherein said probe is selected from the group consisting of dsDNA, ssDNA, and RNA; (b) electrophoresing the mixture; (c) transferring, using positive pressure blot transfer or capillary transfer, the complex to a membrane, wherein the membrane is positively charged nylon; and (d) detecting the complex bound to the membrane by detecting the non-radioactive or radioactive label in the complex.<br>
The Biacore technology measures binding events on the sensor chip surface, so that the interactant attached to the surface determines the specificity of the analysis. Testing the specificity of an interaction involves simply analyzing whether different molecules can bind to the immobilized interactant. Binding gives an immediate change in the surface! plasmon resonance (SPR) signal, so that it is directly apparent whether an interaction takes place or not.<br>
SPR-based biosensors monitor interactions by measuring the mass concentration of biomolecules<br>
aRe to a surface. The surface is made specific by attaching one of the interacting partners.<br>
Sample containing the other partoer(s) flows over the surface: when molecules from the sample j<br>
bind to the interactant attached to the surface, the local concentration changes and an SPR j<br>
response is measured. The response is directly proportional to the mass of molecules that bind to j<br>
the surface.	j<br>
SPR arises when light is reflected under certain conditions from a conducting film at the interface between two media of different refractive index. In the Biacore technology, the media are the sample and the glass of the sensor chip, and the conducting film is a thin layer of gold on j the chip surface. SPR causes a reduction in the intensity of reflected light at a specific angle of reflection. This angle varies with the refractive index close to the surface on the side opposite from the reflected light. When molecules in the sample bind to the sensor surface, the concentration and therefore the refractive index at the surface changes and an SPR response is detected. Plotting the response against time during the course of an interaction provides a quantitative measure of the progress of the interaction. The Biacore technology measures the angle of minimum reflected light intensity. The light is not absorbed by the sample: instead; the light energy is dissipated through SPR hi the gold film. SPR response values are expressed in resonance units (RU). One RU represents a change of 0.0001° in the angle of the intensity minimum. For most proteins, this is roughly equivalent to a change in concentration of about 1 pg/mm2 on the sensor surface. The exact conversion factor between RU and surface concentration depends on properties of the sensor surface and the nature of the molecule responsible for the concentration change.<br>
There are a number of other assays that can determine whether an oligonucleotide or analogue thereof, peptide, potypeptide, oligosaccharide or small molecule can bind to the nucleic acid ligand in a manner such that the interaction with the target is modified. For example, electrophoretic mobility shift assays (EMSAs), titration calorimetry, scintillation proximity assays, sedimentation equilibrium assays using analytical uhracentrifugation (see for j eg. www.cores.utahedu/interaction), fluorescence polarization assays, fluorescence anisotropy I assays, fluorescence intensity assays, fluorescence resonance energy transfer (FRET) assays, nitrocellulose filter binding assays, ELISAs, ELONAs (see, for example, USP 5,789,163), RIA4, or equilibrium dialysis assays can be used to evaluate the ability of an agent to bind to a nucleic<br>
acid ligand. Direct assays in which the interaction between the agent and the nucleic acid ligaadj ^Jjprectly determined can be performed, or competition or displacement assays in which the ability of the agent to displace the nucleic acid Hgand from its target can be performed (for example, see Green, Bell and Janjic, Biotechniques 30(5), 2001, p 1094 and USP 6,306,598). Once a candidate modulating agent is identified, its ability to modulate the activity of a nucleic acid ligand for its target can be confirmed in a bioassay. Alternatively, if an agent is identified that can modulate the interaction of a nucleic acid ligand with its target, such binding assays can be used to verify that the agent is interacting directly with the nucleic acid Hgand and can measure the affinity of said interaction.<br>
In another embodiment, mass spectrometry can be used for the identification of an regulator that binds to a nucleic acid ligand, the srte(s) of interaction between the regulator and the nucleic acid ligand, and the relative binding affinity of agents for the nucleic acid ligand (see lor example U.S. Patent No. 6,329,146, Crooke et al). Such mass spectral methods can also be used for screening chemical mixtures or libraries, especially combinatorial libraries, for individual compounds that bind to a selected target nucleic acid ligand that can be used in as modulators of the nucleic acid ligand. Furthermore, mass spectral techniques can be used to screen multiple target nucleic acid ligands simultaneously against, e.g. a combinatorial library cjf compounds. Moreover, mass spectral techniques can be used to identify interaction between a plurality of molecular species, especially "small" molecules and a molecular interaction site on target nudeic acid ligand.<br>
In vivo or in vitro assays that evaluate the effectiveness of a regulator in modifying the interaction between a nudeic acid ligand and a target are specific for the disorder being treated. There are ample standard assays for biological properties that are well known and can be used. Examples of biological assays are provided in the patents cited in this application that describe j certain nucleic acid ligands for specific applications.<br>
As a nonlimhing example, coagulation tests can be performed as bioassays in clinical   i<br>
i<br>
laboratories. These tests are generally functional end-point assays, in which a patient sample   j (plasma or whole blood) is incubated with exogenous reagents that activate the coagulation      i cascade, and the time until clot formation is measured. The clotting time of the patient sample ii&gt; thjen compared to the clotting time of pooled normal plasma or whole blood to provide a standard measurement of the patient's hemostatic status. As described below, such dotting assays are    j<br>
commonly used as screening tests that evaluate the functioning of both the patient's intrinsic and<br>
insic coagulation systems.	<br>
The Activated Clotting Time Test (ACT) is a screening test that resembles the activated]<br>
partial thromboplastin time (APTT) test, but is performed using fresh whole blood samples. ACJT<br>
can be used to monitor a patient's coagulation status in connection with clinical procedures, such<br>
as those that involve the administration of high doses of heparin (e.g., CPB and PTCA),	<br>
The Activated Partial Thromboplastin Time Test (APTT) is a common central laboratory test, APTT is used to evaluate the intrinsic coagulation pathway, which includes factors I, H, VJ VH[, DC X, XI, and XH. The test is performed using a plasma sample, in which the intrinsic pathway is activated by the addition of phospholipid, an activator (ellagic acid, kaolin, or micronized silica), and Ca2+. Formation of the Xase and prothrombinase complexes on the surface of the phospholipid enables prothrombin to be converted into thrombin, with subsequent clot formation. The result of the APTT test is the time (in seconds) required for this reaction. APTT can be used to assess tine overall competence of a patient's coagulation system, as a preoperative screening test for bleeding tendencies, and as a routine test for monitoring heparin |<br><br>
therapy. Another example of die APTT is performed as follows. First, blood plasma is collected after the blood was subjected to centrifugal separation, and then acting is added thereto. In addition, calcium chloride is added. The period of time is measured until coagulation is formed More in detail, the blood plasma is stored in a refrigerator after it was obtained through centrifugal separation of blood. Activating agent of 0.1 ml, which was warmed in a water having a temperature of 37° C. for a minute, is poured into a test tube containing 0.1 ml of the plasma The mixture is allowed to stay in a water of 37° C. for two minutes. Then to this mixture, (X1 ml of CaCla of 0.02 M, which had been placed in a water having a temperature of 37° C., is added under pressure. At this moment a stop watch is switched on. The test tube is heated in the water of 37° C. for 25 seconds. The test tube is taken out, and if coagulation is observed, the stop watch is turned off. This is one way in which the blood coagulating time can be measured. The bleeding time test can be used for the diagnosis of hemostatic dysfunction, von Willebrand's disease, and vascular disorders. It also can be used to screen for platelet abnormalities prior to surgery. The test is performed by making a small incision on the forearm and wicking away the blood from the wound site. The time it takes for bleeding to stop is<br>
recorded and in control subjects is approximately 3.5 minutes. Prolongation of the bleeding tintej indicative of qualitative or quantitative platelet defects.<br>
"iflr<br>
The Prothrornbin Time Test (PT), which was first described by Quick in 1935, measures! the tissue factor-induced coagulation time of blood or plasma. It is used as a screening test to evaluate the integrity of the extrinsic coagulation pathway, and is sensitive to coagulation fa
The Thrombin Clotting Time Test (TCT) measures the rate of a patient's clot formation compared to that of a normal plasma control. The test can be performed by adding a standard amount of thrombin to a patient's plasma that has been depleted of platelets, and measuring the<br><br>
time required for a clot to form. This test has been used as an aid in the diagnosis of disseminated<br>
intravascular coagulation (DIC) and liver disease.	<br><br>
There are also a number of tests that can be used in the diagnosis of a patient's coagulau've status. These fall into two categories: complex tests, some of which are based on the screening tests outlined above, and immunoassays. Complex Tests include specific factor assays! based on laboratory tests, such as the APTT, PT, and TCT tests. One assay measures the level of the activation peptide factor IXa or the factor IXa-antithrombin m complex. These measurements are used to determine the levels of factor IXa or factor VH-tissue mediated complex. Assays for activated protein C resistance, antithrombin, protein C deficiency, and protein S deficiency are also part of this group. Asymptomatic individuals who have heterogeneous deficiencies of proteins C and S, and resistance to activated protein C, have significantly elevated levels of the prothrombin fragment F1.2 compared to controls.<br>
V.  Method of Regulating Nucleic Acid Ligand Therapy<br>
any take 2-4 hours, or platelet infusion (for Abciximab treatment). The third case is when ani<br>
jJKJcoagulant nucleic acid ligand is used during a cardiopuimonary bypass procedure. Bypass<br>
TIP^<br>
patients are predisposed to post operative bleeding. In each case, acute reversal of the anticoagulant effects of a compound via an antidote (e.g., an oligonucleotide modulator of the invention targeted to an anticoagulant or antithrombotic nucleic acid ligand) allows for improved, and likely safer, medical control of the anticoagulant or antithrombotic compound.<br>
A method of treating cardiovascular disease in patients is also provided in accordance  1 with the present inventioa. The method comprises administering an effective amount of an KNJA aptamer that selectively binds a coagulation pathway factor, the RNA aptamer having a dissociation constant for the coagulation pathway factor of about 20 nM or less, to a vertebrate' subject suffering from cardiovascular disease, whereby cardiovascular disease in the vertebrate<br>
subject is treated, then providing an antidote to reverse the effects of the aptamer by	<br>
administration of a modulator.<br>
A method of modulating E2F activity in patient, including a warm-blooded vertebrate, in which such modulation is desired is also provided. The method comprises: (a) administering to the warm-blooded vertebrate an effective amount of an RNA aptamer that selectively binds ani E2F family member, the RNA aptamer having a dissociation constant for the E2F family        j<br>
;     *<br>
member of about 20 nM or less; (b) modulating E2F in the warm-blooded vertebrate through tfte<br>
administering of the RNA aptamer of step (a); and (c) providing an antidote to reverse the effects of the aptamer by administration of a modulator.<br>
The patient treated in the present invention in its many embodiments is desirably a hunian patient, although it is to be understood that the principles of the invention indicate that the invention is effective with respect to all vertebrate species, including warm-blood vertebrates (e.g., birds and mammals), which are intended to be included in the term "patient". In this context, a mammal is understood to include any mammalian species in which treatment of cardiovascular disease is desirable, particularly agricultural and domestic mammalian species. -     !<br>
Contemplated is the treatment of mammals such as humans, as well as those mammals !of importance due to being endangered (such as Siberian tigers), of economical importance (animals raised on farms for consumption by humans and/or social importance (animals kept ajs pets or in zoos) to humans, for instance, carnivores other than humans such as cats and dogs), | swine (pigs, hogs, and wild board), ruminants (such as cattle, oxen, sheep, giraffes; deer goats,.<br>
bison, and camels), and horses. Also contemplated is the treatment of birds, including the          UBitment of those kinds of birds that are endangered, kept in zoos, as well as fowl, and more      particularly domesticated fowl, i.e., poultry, such as turkeys,'chickens, ducks, geese, guinea fowlj, and the like, as they are also of economical importance to humans.<br>
*	<br><br>
The present method for treating cardiovascular disease in a tissue contemplates	<br>
contacting a tissue in which cardiovascular disease is occurring, or is at risk for occurring, with a composition comprising a therapeutically effective amount of an KNA aptamer capable of        Ibinding a coagulation factor as well as providing an antidote to reverse the effects of the aptamer by administration of a modulator. Thus, the method comprises administering to a patient a therapeutically effective amount of a physiologically tolerable composition containing the KNA aptamer as well as a method to provide an antidote to reverse the effects of the aptamer by administration of a modulator.<br>
The dosage ranges for the administration of the modulator depend upon the form of the ; modulator, and its potency, as described further herein, and are amounts large enough to produce the desired effect. For situations in which coagulation is modulated, which can correspondingly ameliorate cardiovascular disease and the symptoms of cardiovascular disease, the dosage should not be so large as to cause adverse side effects, such as hyperviscosity syndromes, pulmonary  i edema, congestive heart failure, and the like. Generally, the dosage will vary with the age,       condition, sex and extent of the disease in the patient and can be determined by one of skill in the art. The individual physician in the event of any complication can also adjust the dosage.<br>
A therapeutically effective amount is an amount of a modulator sufficient to produce a measurable modulation of the effects of the nucleic acid ligand, including but not limited to a ; coagulation-modulating amount, an E2F activity-modulating amount coagulation, and/or angiogenesis factor activity (e.g., Angl or Ang2 activity)-modulating amount. Modulation of \ coagulation, E2F activity, and/or angiogenesis .factor activity (e.g., Angl or Ang2 activity) can be measured in situ by immunohistochemistry by methods disclosed in the Examples, or by other methods known to one skilled in the art.<br>
A preferred modulator has the ability to substantially bind to a nucleic acid ligand in   j<br>
solution at modulator concentrations of less than one (1.) micromolar (jiM), preferably less ttajn<br>
i	i	I<br>
0.1 |*M, and more preferably less than 0.01 \iM. By "substantially"1^ meant that at least a 501<br>
percent reduction in target biological activity is observed by modulation in the presence of the a!<br>
Jjjlf	"      ;  !<br>
w^et, and at 50% reduction is referred to herein as an ICso value.<br>
Preferred modes of administration of the materials of the present invention to a mammalian host are parenteral, intravenous, intradermal, intra-articular, intra-synovial, itttrathecal, intra-arterial, intractrdiae, intramuscular, subcutaneous, intraorbital, intracapsular, \ intraspinal, intrasternal, topical, transdermal patch, via rectal, vaginal or urethral suppository, peritoneal, percutaneous, nasal spray, surgical implant, internal surgical paint, infusion puinp or via catheter. .In one embodiment, the agent and carrier are administered in a slow release formulation such as an implant, bolus, microparticle, microsphere, nanoparticle or nanospftere. For standard information on pharmaceutical formulations, see Ansel, et a/., Pharmaceutical Dosage Farms and Drug Delivery Systems, Sixth Edition, Williams &amp; Wilkins (1995).<br>
The modulators of the present invention can be preferably administered parenterally by<br>
injection or by gradual infusion over time. Although the tissue to be treated can typically be<br>
accessed hi the body by systemic administration and therefore most often treated by intravenous<br>
administration of therapeutic compositions, other tissues and delivery techniques are provided<br>
where there is a likelihood that the tissue targeted contains the target molecule. Thus,<br>
modulators of the present invention are typically administered orally, topically to a vascular<br>
tissue, intravenously, intraperitoneally. intramuscularly, subcutaneously, uitra-cavity,<br>
transdermally, and can be delivered by peristaltic techniques. As noted above, the<br>
pharmaceutical compositions can be provided to the individual by a variety of routes such oraly,,<br>
topically to a vascular tissue, intravenously, intraperitoneally, intramuscularly, subcutaneously, |<br>
intra-cavity, transdermally, and can be delivered by peristaltic techniques. Representative, noflrj<br>
liming approaches for topical administration to a vascular tissue include (1) coating or	<br>
impregnating a blood vessel tissue with a gel comprising a nucleic acid ligand, for delivery in    vivo, e.g., by implanting the coated or impregnated vessel in place of a damaged or diseased vessel tissue segment that was removed or by-passed; (2) delivery via a catheter to a vessel in    which delivery is desired; (3) pumping a nucleic acid ligand composition into a vessel that is to « be implanted into a patient. Alternatively, the nucleic acid ligand can be introduced into cells byj microinjection, or by liposome encapsulation. Advantageously, nucleic acid ligands of the present invention can be administered in a single daily dose, or the total daily dosage can be<br>
administered in several divided doses. Thereafter, the modulator is provided by any suitable to alter the effect of the nucleic acid ligand by administration of the modulator.<br>
The therapeutic compositions comprising modulator polypeptides of the present<br>
invention are conventionally administered intravenously, as by injection of a unit dose, for<br>
•	:<br>
example. The term vunit dose" when used in reference to a therapeutic composition of the present invention refers to physically discrete units suitable as unitary dosage for the subject, etch unit containing a predetermined quantity of active material calculated to produce the desired therapeutic effect in association with the required diluent; i.e., carrier or vehicle.<br>
The compositions are administered in a manner compatible with the dosage formulation^ and in a therapeutically effective amount. The quantity to be administered depends on the subject to be treated, capacity of the subject's system to utilize the active ingredient, and degree of therapeutic effect desired. Precise amounts of active ingredient required to be administered depend on the judgment of the practitioner and are peculiar to each individual. However; suitable dosage ranges for systemic application are disclosed herein and depend on the route of administration. Suitable regimes for administration are also variable, but are typified by an initial administration followed by repeated doses at one or more hour intervals by a subsequent injection or other administration. Alternatively, continuous intravenous infusion sufficient to maintain concentrations in the blood in the ranges specified for in vz'w therapies are contemplated.<br>
As used herein, the terms "pharmaceutically acceptable," "physiologically tolerable/' and grammatical variations, thereof, as they refer to compositions, carriers, diluents and reagents, are used interchangeably and represent that the materials are capable of administration without substantial or debilitating toxic side effects.<br>
Pharmaceutically useful compositions comprising a modulator of the present invention can be formulated according to known methods such as by the admixture of a pharmaceutically acceptable carrier. Examples of such carriers and methods of formulation can be found in Remington 's Pharmaceutical Sciences. To form a pharmaceutically acceptable composition suitable for effective administration, such compositions will contain an effective amount of the aptamer. Such compositions can contain admixtures of more than one modulator.<br>
The effective amount can vary according to a variety of factors such as the individual's S'dition, weight, sex, age and amount of nucleic acid lidgand administered. Other factors include the mode of administration. Generally, the compositions will b'e administered in dosages adjusted for body weight, e.g., dosages ranging from about 1 ug/kg body weight to about 100 •. rag/kg body weight, preferably, 1 mg/kg body weight to 50 mg/kg body weight. *<br>
Modulators of nucleic acid ligands can be particularly useful for the treatment of diseases where it is beneficial to inhibit coagulation, E2F activity, and/or angiogenesis factor activity (e.g., Angl or Ang2 activity), or prevent such activity from occurring. The pharmaceutical compositions are administered in therapeutically effective amounts, that is, in amounts sufficient to generate a coagulation-, E2F activity-, and/or angiogenesis factor activity (e.g., Angl or Ang2 activity)-modulating response, or in prophylactically effective amounts, that is in amounts sufficient to prevent a coagulation factor from acting in a coagulation cascade, to prevent an E2F activity-mediated response, or to prevent an angiogenesis factor activity (e.g., Angl or Ang2 activity)-mediated response. The therapeuticaUy effective amount and prophylactically effective amount can vary according to the type of modulator. Hie pharmaceutical composition can be administered in single or multiple doses.<br>
Generally, oligonucleotide modulators of the invention can be administered using established protocols used in antisense therapies. The data presented in Example 6 indicate the activity of a therapeutic nucleic acid ligand can be modulated by the intravenous infusion of antidote oligonucleotides into a human or other animal. Furthermore, because the modulator's activity is durable, once the desired level of modulation of the nucleic acid ligand by the modulator is achieved, infusion of the modulator can be terminated, allowing residual modulator to clear the human or animal. This allows for subsequent re-treatment with the nucleic acid ligand as needed. Alternatively, and in view of the specificity of the modulators of the invention, subsequent treatment can involve the use of a second, different nucleic acid Hgand/modulator (e.g., oligonucleotide) pair.<br>
Modulators synthesized or identified according to the methods disclosed herein can be used alone at appropriate dosages defined by routine testing in order to obtain optimal modulation of nucleic acid ligand activity in coagulation, E2F, and/or angiogenesis factor cascades (e.g., Angl or Ang2 activity) while minimizing any potential toxicity. |n addition, cot<br>
For liquid forms the active drug component can be combined in suitably flavored	i<br>
^Spending or dispersing agents such as the synthetic and natural gums, for example, tragacanth, acacia, methyl-cellulose and the like. Other dispersing agents that can be employed include     <br>
glycerin and the like. For parenteral administration, sterile suspensions and solutions are desire^.<br>
*	:<br>
Isotonic preparations that generally contain suitable preservatives are employed when	<br>
intravenous administration is desired.<br>
Topical preparations containing the active drug component can be admixed with a variety<br>
of carrier materials well known in the art, such as, e.g., alcohols, aloe vera gel, allantoin,	<br>
glycerine, vitamin A and E oils, mineral oil, PPG2 mydstyl propionate, and the like, to form,     <br>
e.g., alcoholic solutions, topical cleansers, cleansing creams, skin gels, skin lotions, and	<br>
shampoos in cream or gel formulations.<br>
The compounds of the present invention can also be administered in the form of liposome delivery systems, such as small unilamellar vesicles, large unilamellar vesicles and multilamellar vesicles. Liposomes can be formed from a variety of phospholipids, such as cholesterol, stearylamine or phosphatidylcholines.<br>
The compounds of the present invention can also be coupled with soluble polymers as targetable drug carriers. Such polymers can include polyvinyl-pyrroUdone, pyran copolymer,<br>
polyhydroxypropylmethscryl-amidephenol, polyhydroxy-ethylaspartamidepbenot or polyethyH<br>
i eneoxidepolylysine substituted with palmitoyl residues. Furthermore, the compounds of the    |<br>
present invention can be coupled (preferably via a covalent linkage) to a class of biodegradable polymers useful in achieving controlled release of a drug, for example, polyethylene glycol (PEG), polyiactic acid, polyepsilon caprolactone, polyhydroxy butyric acid, polyorthoesters, j polyacetals, polydihydro-pyrans, polycyanoacrylates and cross-linked or amphipathic block ! copolymers of hydrogels. Cholesterol and similar molecules can be linked to the aptamers to  increase and prolong bioavailability.<br>
The oligonucleotide modulators can be administered directly (e.g., alone or in a	<br>
liposomal formulation or complexed to a carrier (e.g., PEG)) (see for example, USP 6,147,204,j<br>
USP 6,011,020).	<br>
The nucleic acid ligand or modulator can be comprised of a oligonucleotide modulator j attached to a non-immunogenic, high molecular weight compound such as polyethylene glycolj<br>
JG). In this embodiment, the pharmacokinetic properties of the complex are improved r41atiy^<br>
! nucleic acid ligand alone. As discussed supra, the association could be through covalent !<br>
bonds or non-covalent interactions. In one embodiment, the modulator is associated with the    i<br>
KEG molecule through covalent bonds. Also, as discussed supra, where covalent attachmeiit is j<br>
*	•      i<br>
employed, PEG may be covalently bound to a variety of positions on the oligonucleotide         i *<br>
modulator. In the preferred embodiment, an oligonucleotide modulator is bonded to the 5' thiol | through a maleimide or vinyl sulfone functionality. In one embodiment, a plurality of modulations<br>
i<br>
can be associated with a single PEG molecule. The modulator can be to the same or different   j<br>
1 i target. In embodiments where there are multiple modulators to the same nucleic acid ligand,    !<br>
there is an increase in avidity due to multiple binding interactions with the ligand. In yet a further<br>
embodiment, a plurality of PEG molecules can be attached to each other. In this embodiment, j<br>
one or more modulators to the same target or different targets can be associated with each PEG<br>
molecule. This also results in an increase in avidity of each modulator to its target. In<br>
embodiments where multiple modulators specific for the same target are attached to PEG, there<br>
is the possibility of bringing tile same targets in close proximity to each other in order to generate<br>
specific interactions between the same targets. Where multiple modulators specific for different<br>
targetsareattachedtoPEG, there is the possibiUty of bringing the distinct targets in close       j<br>
proximity to each other in order to generate specific interactions between the targets. In addition,<br>
in embodiments where there are modulators to the same target or different targets associated with<br>
PEG, a drug can also be associated with PEG. Thus the domplex would provide targeted delivery<br>
of the drug, with PEG serving as a Linker.	<br>
One problem encountered in the therapeutic and in vivo diagnostic use of nucleic acids is that oligonucleotides in their phosphodiester form may be quickly degraded in body fluids by j intracellular and extracellular enzymes such as endonucleases and exonucleases before the      desired effect is manifest. Certain chemical modifications of the nucleic acid can be made to increase the in vivo stability of the nucleic acid or to enhance or to mediate the delivery of thiej<br>
nucleic acid. Modifications of the nucleic acid ligands contemplated in this invention include, <br>
are not limited to, those which provide other chemical groups that incorporate additional j<br>
charge, polarizability, hydrophobicity, hydrogen bonding, electrostatic interaction, and fluxionality to the nucleic acid bases or to the nucleic acid as a whole. Such modificatioflis include, but are not limited to, 2'-position sugar modifications, 5-position pyrimidine<br>
modifications, 8-position purine modifications, modifications at exocyclic amines, substitution of<br>
^^jj^^<br>
^Pffuouridine, substitution of 5-bromo or 5-iodo-uracil; backbone modifications, phosphorothiQate or alkyl phosphate modifications, methylations, unusual base-pairing combinations such as the isobases isocytidine and isoguanidine and the like. Modifications can also include 3' and 51 modifications such as capping.<br>
Lipophilic compounds and non-immunogenic high molecular weight compounds with which the modulators of the invention can be formulated for use in the present invention and eaii be prepared by any of the various techniques presently known in the art or subsequently developed. Typically, they are prepared from a phospholipid, for example, distearoyl phosphatidylcholine, and may include other materials such as neutral lipids, for example, cholesterol, and also surface modifiers such as positively charged (e.g., sterylamine or aminomannose or aminomannitol derivatives of cholesterol) or negatively charged (e.g., diacetyl phosphate, phosphatidyl glycerol) compounds. Multilamellar Uposomes can be formed by the conventional technique, that is, by depositing a selected lipid on the inside wall of a suitable<br>
containerorvesselby dissolving the lipid in an appropriate solvent, and then evaporating the <br>
isolvent to leave a thin film on the inside of the vessel or by spray drying. An aqueous phase is i<br>
then added to the vessel with a swirling or vortexing motion which results in the formation of  ML Vs. UVs can then be tinned by homogenization, sonication or extrusion (through filters) of MLV's. In addition, UVs can be formed by detergent removal techniques. In certain embodiments of tills invention, the complex comprises a liposome with a targeting nucleic acid ligand(s) associated with the surface of the liposome and an encapsulated therapeutic or diagnostic agent. Preformed Uposomes can be modified to associate with the nucleic acid ligands. For example, a cationic liposome associates through electrostatic interactions with the nucleic acid. Alternatively, a nucleic acid attached to a tipophilic compound, such as cholesterol, can be added to preformed liposomes whereby the cholesterol becomes associated with the liposomal membrane. Alternatively, the nucleic acid can be associated with the liposome during the formulation of the liposome. Preferably, the nucleic acid is associated wkh the liposome by loading into preformed liposomes.<br>
Alternatively, oligonucleotide modulators of the invention can be produced in vrvo following administration of a construct comprising a sequence encoding the oligonucleotide.<br>
Techniques available for effecting intracellular delivery of RNA modulators of gene expressic^ *n be used (see generally Sullenger et al, Moi. Cell Biol. 10:6512 (1990)).<br>
Certain aspects of the invention can be described in greater detail in the non-limiting<br>
Examples that follow.	<br>
EXAMPLE 1 Aptamer to Factor IXa<br>
Nuclease-resistant 2'-fluoro pyrimidine-modified aptamers to human coagulation Factor IXa were generated as described in WO 0226932 A2. Eight iterative cycles of selection were performed, yielding a family of 16 aptamers with high affinity for FTXa; kd's ranging from ~ 0.6-15 nM in physiologic salt and pH at 37°C. Comparative sequence analysis made it possible to predict and synthesize the minimized version of the highest affinity aptamer, termed RNA 9.3t ("I" for truncate), shown in Fig. 1. This 34 nucleotide aptamer has a molecular weight of 11.5 kDa and binds FDCa whh essentially the same affinity as the full-length sequence (Ko 0.6 nM). As a control, a mutant version of RNA 9.3t termed 9.3tM was synthesized in which the absolutely conserved A's in the internal loop were mutated to G's (Fig. 1). This aptamer binds FTXa with a kd &gt; 5 uM as determined by competition binding assays. In all activity assays, j RNA 9.3tM is employed as a control to measure any non-specific effects caused by an aptamer | of this composition. Aptamer 9.3t blocks FX activation by FVTHa/FrXa/lipids, and also partially blocks synthetic substrate hydrolysis by FIXa.<br>
To examine the specificity of the aptamer for FDCa versus structurally similar coagplatie|n<br><br>
factors, the affinity of 9.3t for FTX, FVTla, FXa, FXIa and APC was measured in direct binding | assays as previously described (Rusconi et al, Thrombosis and Haemostasis 83:841-848 (2000)). The aptamer binds FIX only -5-50 fold less tightly than FIXa. It failed to exhibit significant binding at protein concentrations up to 5 uM (fraction RNA bound 5000fbld.<br>
To dettsrmine the anticoagulant potency of RNA 9.3t, the ability of 9.3t to proloag the clotting time Of human plasma has been evaluated in activated partial thromboplastin time<br>
(APTT) and protnrombin time (PT) clotting assays. Aptamer 9.3t, but not control aptanaer        'UProl, was able to prolong the clotting time of human plasma in a dose dependent manner (Fig.!<br><br>
2). Neither aptamer had an effect on the PT, demonstrating the functional specificity of aptameif<br>
9.3t for FIXa and demonstrating that this type of oiigonucleotide, at concentrations up to 1 uM,!<br>
"	'	«	: I<br>
does not non-specifically incfease the clotting time of human plasma. Thus, RNA 9,3 is a potenjt<br>
anticoagulant, with a maximal effect on the APTT similar to that observed in FIX deficient plasma. Similar experiments have been performed in porcine plasma, and similar potency and specificity have been observed. : To determine if this molecule is capable of hihibitineFIX/FIXa activity mvrvo,the | ability of this aptamer to systemically anticoagulate small (1.5-4 kg) pigs following bolus j intravenous injection was tested. For these experiments, an intravenous catheter was placed in the femoral vein for sample injection and an arterial catheter was placed in a femoral artery of j the animal for serial withdrawal of blood samples. A pre-injection blood sample was taken, and the ACT time measured on site to establish a baseline whole blood clot time for the animal. j Aptamer 9.3t at doses of 0.5 mg/kg 01=4) and 1.0 rag/kg (n«4), aptamer 9.3tM at 1.0 mg/fcg (n*3) or vehicle (n»3) was then delivered by intravenous bolus injection. Blood samples wer$ taken at various times post injection, and the ACT immediately determined. Additional blood was drawn for determination of APTT's at different times post-injection. As shown in Fig. 3,<br>
aptamer 9.3t, but not the control aptamer 9.3tM or vehicle, was able to inhibit FIXa activity m I<br>
vivo as evidenced by a significant dose-dependent increase in the animal's ACT post injection.<br>
As shown in Fig. 4, aptamer 9.3 specifically prolonged the APTT, but not PT of the animals in ft<br>
dose dependent manner. Using the w vz/ro dose response curves from the APTT experiments,<br>
the change in 9.3t plasma concentration over time following injection can be estimated.	<br>
As an attempt to increase the bioavailability of 9.3t, a version of this aptamer with a      5'cholesterol moiety was synthesized, termed 9.3t-C. The cholesterol modifed aptamer retaine^ Mgh affinity binding to FIXa (Fig- 5A) and potent anticoagulant activity (Figs. 5B and 5C). Thje m vivo effects of this modification on the circulating half-life of 9.3t-C versus 9.3t (n=2 animal for each aptamer) have been tested using the pig systemic anticoagulation model. Following injection of 0.5 mg/kg 9.3t-C, the ACT of the animal increased -1.4 fold and was sustained ati j tfiis level for 1 hour post injection (Fig. 6A). Fig. 6B shows analysis of the APTT and PT of th^<br>
animals from this experiment. While the anticoagulant potency of the two aptamers is similar, wrairation of the anticoagulant effect of 9.3t-C is significantly longer (Fig. 6C).<br>
EXAMPLE 2<br>
• Oligonucleotide that Reverses Interaction of the<br>
FIXa Aptamer with Coagulation FIXa<br>
The secondary structure model of 9.3t is shown in Fig. 1 and was developed from<br>
. comparative sequence analysis of the related FIXa aptamer sequences shown in Fig. 7. These<br>
data strongly support formation of the stem-looped structure depicted in Fig. 1. In addition,<br>
mutational analysis of aptamer 9.3t demonstrated that disruption of either stem 1 or stem 2<br>
resulted in a greater than 1000 fold loss of affinity for FIXa. Therefore, a 17 residue all<br>
2'Omethyl Oligonucleotide was designed (sequence 5' auggggaggcagcauua 3') (Anti-Dl)<br>
complementary to the 3'half of aptamer 9.3t beginning at the 5* end of loop 2 (L2 in Fig. 7); and i<br>
extending to the 3' end of the aptamer. This design allows for nucleation of an intermolecular   j<br>
duplex between the Oligonucleotide and loop 2 of the aptamer. Also, formation of the	j<br>
intermolecular duplex is thermodynamically favored due to both the length and base composition of the complimentary Oligonucleotide. A ribonucleotide duplex of this sequence has calculated ftee energy of-26.03 kcal/mol and a predicted Tm of 75.4 °C. The half-life of such a duplex at 37°C would greatly exceed 24 hours.<br>
To determine if this Oligonucleotide could denature aptamer 9.3t, radiolabeled aptamer 9.3t (125 nM) was incubated with increasing concentrations of the "antidote" Oligonucleotide (from equimolar to an 8 fold molar excess) at 37°C for 15 minutes (-heat Fig 8), and the amount! . of istermotecular duplex formed was visualized by native gel electrophoresis (12% acrylamide, 150 mM NaCl, 2 mM CaCk, run in IXtris-borate buffer +2 mM CaCI2) followed by phosphoriraaging (Fig. 8). To generate an oligonucleotide-aptamer complex as a gel-mobility control, the Oligonucleotide was annealed to aptamer 9.3t by heating the aptamer and an 8 Ibid molar access of the Oligonucleotide at 95°C for 5 minutes prior to the 37°C incubation (+heat Fig 8). The same set of experiments was performed with a nonsense Oligonucleotide of the same base composition as the antidote Oligonucleotide (N.S. in Fig. 8).' As can be seen in Fig. 8J the antidote Oligonucleotide readily denatures aptamer 9.3t as evidenced by near complete<br>
formation of the oligonucleotide-aptamer complex when the oligonucleotide was present at anJ8<br>
d molar excess to the aptamer. In addition, this interaction is very specific, as no complex ijs<br>
observed, with or without heating, between the aptamer and the nonsense control	!<br>
oligonucleotide.	I<br>
To determine if this antidote oligonucleotide could reverse the anticoagulant* activity of<br>
aptamer 9.3t, the APTT of pooled human plasma was measured in the presence of 50 nM 9.3t I<br>
and increasing concentrations of the antidote oligonucleotide (Fig. 9 A and 9B). In this	|<br>
experiment, aptamer 9.3t was pre-incubated 5 minutes in plasma prior to addition of the anti oligonucleotide to generate aptamer-FTX complexes, the antidote or nonsense oligonucleotide were then added, and the incubation continued for an additional 10 minutes prior to adding CaCh to initiate clot formation. As shown hi Figs. 9 A and 9B, the antidote oligonucleotide is able to effectively reverse about 80% of the anticoagulant activity of aptamer 9.3t. However; molar excess of antidote oligonucleotide required to achieve this effect is substantially larger than the amount required to effectively denature the aptamer in the absence of protein.<br>
Specificity of Oligonucleotide Antidotes<br>
In order to assess the specificity of an oligonucleotide antidote, aptamer 9.20twas      \ prepared (see Fig. 10A). Stem 1 of 9.20t is identical to stem 1 of aptamer 9.3t, as is the 3* hajlfjof stem 2. The main differences between 9.20t and 9.3t are found in loop 2 and loop 3 (compare Fig. 1 with Fig. 10A).<br>
Aptamer 9.20t binds FKa with a kd comparable to that of 9.3t. APTT assays were us4d to measure the clot time of human plasma as a function of the concentration of aptamer 9.20t Its in vitro anticoagulant potency in human plasma was comparable to that of 9.3t (Fig. 1<mb j aptamer was added to human plasma at a concentration of nm and allowed bind fix for minutes. varying concentrations antidote oligonucleotide anti dl were></mb>
i<br>
then added, and the APTT wts measured at 10 minutes after antidote addition to plasma. The | relative change in clot time caused by 9.20t addition to plasma was unaffected by the addition 6f this antidote oligonucleotide complimentary to aptamer 9.3t (Fig. IOC).<br>
EXAMPLE^ Tailed Aptamer 9.3t<br>
In order to determine if a single-stranded "tail" added to the end of an aptamer promotes j<br>
association of an "antidote" oligonucleotide with the aptamer, a 3' tail was added to aptamer 9.3t|<br>
the tailed-aptamer being designated 9.3t-3NT (Fig. 11). The 3' tail of 9.3t-3NT is a 3 nucleotidie |<br>
2'Omethyl modified KNA tail. This tail sequence was chosen to reduce the likelihood of	|<br>
impacting the activity of the aptamer, and to reduce potential secondary structures within the j complimentary antidote oligonucleotide.<br>
The affinity of aptamer 9.3t-3NT was compared to 9.3t in competition binding assays<br>
(Fig. 12A). The affinity of aptamer 9.3t-3NT for FTXa was comparable to that of 9.3t(Ko 1.5   j<br>
I<br>
nM or less). APTT assays were used to measure the clot time of human plasma as a function of &amp;e concentration of aptamers 9.3t-3NT and 9.3t (Fig. 12B). The anticoagulant activities of the j<br>
;        :   i<br>
aptamers were similar, and both aptamers were able to completely inhibit FIX activity in human<br>
plasma.	i<br>
Aptamer9.3t-3NT was added to human plasma at a concentration of 50 nM (-3 fold     j increase in APTT) and allowed to bind to plasma FIX for 5 minutes. Varying concentratioiis df| antidote oligonucleotides (see Fig. 11) were then added, and the APTT was measured at 10 minutes after antidote addition to plasma (Fig. 13A). The fraction of the anticoagulant activity \ reversed by the antidote oUgonucIeotide is the difference between the APTT in the presence of j aptamer alone and the APTT in the presence of aptamer+antidote divided by the change in      | APTT over baseline in the presence of the aptamer alone (0 = no effect, 1== complete reversal). Each of the complimentary antidote oligonucleotides tested was able to reverse &gt;90% of the    : anticoagulant activity of aptamer 9.3t-3NT within 10 minutes of addition in human plasma, with AS3NT-3 demonstratini the most potent reversal activity (Fig. 13B). This reversal activity is | comparable to the ability of protamine to reverse the APTT increase following heparin addition] to human plasma.<br>
The reversal of the anticoagulant activity of 9.3t-3"NT by antidote oligonucleotide AS3NT-3 was compared to the reversal of the anticoagulant activity of 9.3t by antidote oligonucleotide Anti Dl. The addition of the tail to the aptamer increases the efficiency of<br>
reversal by an antidote oligonucleotide with sequences complimentary to the 3' tail of the ^fliner (Figs. 14A and 14B).<br>
to addition to the foregoing, the following oligonucleotide modulators have been produced that target aptamer 9.3t, and are effective at reversing its anticoagulant activity in human plasma in vitro (all are 2'Omethyl oligonucleotides):*<br>
Anti D Tl: 5' cau ggg gag gca gca uua 3' AS 9.3t-2: 5' cau ggg gag gca gca 3' AS 9.3t-3: 5' cau ggg gag gca 3'.<br>
The following oligonucleotide modulators target aptamers 9.3t and 9.3t-3NT, and are efFectiveiajt<br>
!<br>
reversing its anticoagulant activity in human plasma in vitro (all are 2'0methyl oligonucleotides):<br>
AS 5-1:5' gca uua cgc ggu aua guc ccc ua 3' AS 5-2: 5' cgc ggu aua guc ccc ua 3'.<br>
The following oligonucleotide modulators target either aptamer 9.3t or aptamer 9.3t-3NT, and are mutant oligonucleotides designed to test specific aspects of the design of modulating oligonucleotides to these aptamers (all are 2'Omethyl oligonucleotides):<br>
AS 9.3t-M: 5' cau ggg gas gca 3' (SEQ ID N0:37) AS 9.3t-3NOH: 5' aug ggg agg ca 3' (SEQ ID N0:38).<br>
AS 3NT-3M: 5' gac aug ggg aag ca 3' (SEQ ID N0:39) AS 3NT-3 MT: 5' aca aug ggg agg ca 3' (SEQ ED NO:40)<br>
EXAMPLES Antidote Oligonucleotide to Aptamer 9.3t<br>
The antidote oligonucleotide 5-2C (5' CGC GGU AUA GUC CCC AU) but not a scrambled version of this antidote oligonucleotide, 5-2C scr, has been shown to effectively reverse the activity of aptamers 9.3t and Peg-9.3t in vitro in human plasma. Peg-9.3t is 9.3t wjtlf a 40 KDa polyethylene glycoJ appended to its 5'end via a linker.<br>
In these experiments, aptamer was added to plasma (50 nM 9.3t, 125 nM Peg-9.3t) and ; allowed to incubate for 5 minutes. Antidote oligonucleotides were then added, and APTT assay^ were initiated 10 minutes after antidote addition. The results are shown in Figure 15.<br>
EXAMPLE 6 Rapidity and Durability of Control of Aptamer 9.3t by Antidote Oligonueleotide<br>
Aptamers 9.3t or Peg-9.3t were added to human plasma in vitro at a final concentration of 50 nM for 9.3t or 125 nM for Peg-9.3t, and allowed to incubate for 5 minutes at 37°C. Antidote         i oligonucleotide 5-2C at the indicated molar excess to the aptamer was then added, and the       j residual aptamer activity was determined by measuring the clotting time in APTT assays at the \ times indicated following antidote addition. The % residual anticoagulant activity equals 1 - thfe ratio of (TAjttawr alone - tai***^ antidote) to (TApum,* alone -Tbweiine) X 100, where T=APTT clot time.<br>
The duration of the inactivation of the anticoagulant activity of Peg-9.3t by antidote oligonucleotide 5-2C was measured in vitro in human plasma. Briefly, Peg-9.3t was added to human plasma to a final concentration of 125 nM and allowed to incubate for 5 minutes. Antidote oligonucleotide 5-2C was then added at a 10 fold molar excess, or in a parallel experiment buffer alone was added in place of the antidote oligonucleotide, and the clotting tim? was measure in an APTT assays at various time points following antidote addition. The %      j residual anticoagulant activity was determined as above. The APTT of untreated human piasmaj was also measured in parallel to establish a baseline clotting time at each time point. It was found that after 5 hours of incubation at 37"C, the APTT of the untreated plasma began to       :<br>
increase, indicating the loss of the clot forming activity of the plasma, and the experiment was stopped at 5 hours.<br>
The data presented in Figures 16 and 17 demonstrate to the ability to rapidly and durably; control the anticoagulant activity of the FIXa antagonist aptamer 9.3t, and its derivatives, using j antidote oligonucleotides. Together these data demonstrate that the onset of action of the antidote is rapid, that the time needed for the antidote to act is at least in part dependent on the antidote concentration, and that once the antidote has inactivated the aptamer, this effect is durable.<br>
EXAMPLE?<br>
Antidote Oligonucleotide to Aptamer Against Coagulation Factor Xa<br>
Depicted in Figure ISA is an aptamer (designated 1 lF7t) against coagulation factor Xa • that is a potent anticoagulant in vitro in human plasma. Depicted in Figure 18B is a mutant      ! version of aptamer HF7t, designated HF7tM. Alterations of the identity of the positions showtn in Fig. 18B leads to a &gt;13QO fold loss of affinity of the mutant aptamer for coagulation Fxa.     j Varying concentrations of aptamers 1 IFTt and 1 IFTtM were added to human plasma in vitro,   \ and the clot time was then measured in a PT (Fig. 19A) or APTT assay (Fig. 19B). In Figure l§, dotted lines indicate relative change in clot time of plasmas containing 10% or less than 1% the! normal plasma level of FX, demonstrating the potent anticoagulant effects of aptamer 1 lF7t. Ajll<br>
1<br>
data are normalized to the baseline for that day, so that a value of 1= no change in clot time. \ Aptamer 1 lF7t is also a potent anticoagulant of human plasma when assays in PT clotting ! assays, as would be expected for a FXa inhibitor. The mutant aptamer, 1 !F7tM showed no aiitij coagulation activity in either the PT or APTT assay.<br>
The following antidote oligonucleotides were screened for the ability to reverse the anticoagulant activity of aptamer 1 lF7t in vitro in human plasma:<br>
AO 5-1: 5' CUC GCU GGG GCU CUC 3' AO 5-2: 5'UAU UAU CUC GCU GOG 3' AO 3-1: 5'AAG AGC GGG GCC AAG 3' AO 3-3: 5' GGG CCA AGU AUU AU 3'.<br>
Figure 20 shows the sequences of 1 lF7t to which these antidote oUgonucleotides are fcompJimentary. As shown in Figure 21 A, the antidote oiigonucleotides effectively reverse tike activity of aptamer 1 lF7t in human plasma. In these experiments, aptamer was added to plasma<br>
(final concentration 125 nM) and allowed to incubate for 5 minutes. Antidote oligonucleptidefc<br>
»	'	*<br>
were then added, and APTT assays were initiated 10 minutes after antidote addition. In addition<br>
to the antidote oligonucieotides described above, the following sequences were also found to have antidote activity against aptamer 1 IFTt:<br>
AO 3-2: 5y CAA GAG CGG CK5C CAA G 3'<br>
A05-3:5'CGAGUAUUAUCUUG3'.	;<br>
\	• |<br>
Figure 21B shows the characterization of antidote 5-2 activity over a larger concentration<br>
range of antidote 5-2, and comparison to the antidote activity of a scrambled sequence version of<br>
antidote 5-2,5-2scr. The data demonstrate potent reversal activity of antidote 5-2, and	j<br>
specificity of antidote oligonucleotide activity as demonstrated by lack of reversal activity of AO<br>
5-2scr.	|<br>
Figures 22 and 23 relate to the ability to rapidly and durably control the anticoagulant \ activity of the FXa antagonist aptamer 1 lF7t, and its derivatives, using antidote oiigonucleotides. Together these data demonstrate that the onset of action of the antidote is rapid (Fig. 22), that the tune needed for the antidote to act is at least in part dependent on die antidote concentration (Fig. 22), and that once the antidote has inactivated the aptamer, this effect is durable (Fig. 23). Together these data indicate that antidote oiigonucleotides can be intravenously infused into a human or other animal would be a potential method to use antidote oiigonucleotides to modulate the activity of a therapeutic aptamer. Furthermore, because the antidote activity is durable, oacje the desired level of modulation of the aptamer by the antidote is achieved, infusion of the        j antidote can be terminated, allowing residual antidote to clear the human or animal. This allows for subsequent re-treatment of the human or animal with the aptamer as needed.<br>
EXAMPLE 8 Independent Functioning of Aptamer Antidote Pairs<br>
To demonstrate that aptamer-antidote pairs (aptamer 9.3t and its antidote A05-2c and aptamer 1 lF7t and its antidote AQ5-2) function independently of each other, aptamers were added to human plasma at 37°C, as indicated in Figure 24, and allowed to incubate for 5 minutes-. Antidotes were then added and the clotting activity was measured 10 minutes after antidote addition in APTT assays. In all assays, buffer alone was substituted for aptamer or antidote in cases in which only one aptamer or antidote was added to the plasma. All data was normalized to the baseline for that day, so that a value of 1= no change in clot time.<br>
Comparing sample Apt 1&amp;2+AD1 to Apt 1 alone and sample Apt 1&amp;2+AD2 to Apt 2    I<br>
]<br>
atone (see Fig. 24), it is clear that antidote reversal activity is specific for the targeted aptamer    |<br>
(e,g., there was no loss of Apt 2 activity in the presence of AD 1 and vice versa), and the activity<br>
of the antidote was effectively unchanged by the presence of the second aptamer.	j<br>
These results have two important implications. First, they demonstrate the ability to dose<br>
;   |<br>
a patient with nucleic acid ligand 1 (e.g., 9.3t), reverse that nucleic acid Ugand with its matched | antidote, and then re-treat the patient with a second nucleic acid ligand. Second, the results<br>
demonstrate the utility of nucleic acid ligand-antidote pairs for target validation, and the study of<br>
]	: i<br>
biochemical pathways. The antidote enables one to determine that the response observed after inhibiting a target protein with an nucleic acid Ugand is due to specifically inhibiting that protein, i In addition, the antidote makes it possible to determine if binding of the nucleic acid ligand to the target protein leads to protein turnover. For example, if upon antidote addition complete protein activity is restored, it argues that there was no net change in protein concentration as a result of; nucleic acid ligand binding.<br>
EXAMPLE 9<br>
Aptamer PEG-9.3t and Antidote 5-2C Function<br>
in Plasma from Patients with Heparin-Induced Thrombocytopenia (HIT)<br>
*	'	»<br>
The ability to control the anticoagulant activity of heparin with protamine enables safer | treatment of patients undergoing procedures requiring a high level of anticoagulation, in whom | the post-procedural risk of hemorrhage is high. However. ~3-5% of patients receiving heparin j develop a drug-induced immunologic response termed heparin-induced thrombocytopeniaJ(HlTJ), which contraindicates further treatment of these patients with heparin (Warkentin et al, Thrombi Haemost 79:1-7. (1998)). This disorder is characterized by a decrease in the platelet count and | an increased risk for new or recurrent life and limb-threatening thromboembolism (Warkentin <k al thromb haemost several alternative anticoagulants are available but none>f these anticoagulants can be controlled by a reversing agent. This significantly limits the treatment options for patients with HIT, and hemorrhagic complications and recurrent thromboembolism while undergoing treatment are common Greinacher et al, Circulation 99:73-80. (1999), Lewis et al, Circulation 103:1838-1843. (2001)). Therefore, the ability of aptaimer Peg-9.3t and antidote 5-2C to serve as an anticoagulant-antidote pair in plasma samples from six patients with HIT was investigated, three with end-stage renal disease requiring hemodialysis and thus repeated anticoagulation, and three with thromboembolic complications requiring anticoagulant therapy. (Serologic criteria included a positive heparin-induced platelet aggregation assay (Ortel et al, Thromb Haemost 67:292-296. (1992)) and/or elevated heparin/platelet ftetor 4 antibody levels detected by ELIS A (GTI, Inc., Brookfield, WI). Five patients met both clinical and serologic criteria; one patient fulfilled clinical criteria but had negative serologic studies.) Aptamer PEG-9.3t prolonged the APTT clotting times of plasma from all six patients, and antidote 5-2 was able to effectively reverse this anticoagulant activity to the pre-treatment baseline of each patient (Fig. 25). Importantly, two patients were receiving   | anticoagulant therapy at the time samples were taken (patient 3 on danaproid sodium and patient 6 on warfarin), andPEG-9.3t addition to plasma from these patients increased the clot time! ovferj<br>
j<br>
the treatment baseline and antidote 5-2C reversed this response back to the treatment baseline,  |<br>
' i . i<br>
demonstrating .that in patient plasma this drug-antidote pair can function independently of to "on<br>
board" anticoagulant. In addition, treatment of these patient plasma samples with the control gitamer 9.3tM and antidote 5-2C yielded no increase in the clot time, further indicating that oligonucleotides of the composition of the aptamer or antidote do not inherently possess significant anticoagulant activity.<br>
* The ability of aptamer 1 lF7t and it's corresponding antidote 5-2 to serve as an	j<br>
anticoagulant-antidote pair in plasma samples from 2 patients with HIT, one with end-stage renal disease requiring hemodialysis and thus repeated anticoagulation, and one with thromboembolic complications requiring anticoagulant therapy. Aptamer 1 lF7t prolonged the APTT clotting times of plasma from both patients, and antidote 5-2 was able to effectively reversethis anticoagulant activity to the pre-treatment baseline of each patient (Fig. 26). Importantly, these two patients were receiving anticoagulant therapy at the time samples were taken (patient 3 on danaproid sodium and patient 6 on warfarin), and 1 lF7t addition to plasma from these patients increased the clot time over the treatment baseline and antidote 5-2 reversed this response back to the treatment baseline, demonstrating that in patient plasma this drug-antidote pair can function independently of an "on board" anticoagulant. In addition, treatment of these patient plasma samples with the control aptamer 9.3tM and antidote 5-2 yielded no increase in the clot time, further indicating that oligonucleotides of the composition of the aptamer or antidote do not inherently possess significant anticoagulant activity.<br>
All documents cited above are hereby incorporated hi their entirety by reference.<br><br>
If RAT IS CLAIMED IS:<br>
	LA method of altering the affinity of a njucleic acid lijgand for a target molecule ifl<br>
patient comprising administering to a pttient receiving said nucleic acid ligand a modulator thit I<br>
i<br>
binds lo said nucleic acid ligand, said administration being effected under conditions such that  : said modulator binds to said nucleic acid ligand and thereby alters the affinity of said nucleic : acid ligand for said target molecule.<br>
The method according to claim 1 wherein upon binding of said modulator to said j<br>
nucleic acid ligand, the affinity of said nucleic acid ligand for said target molecule is reduced.<br>
The method according to claim 1 wherein upon binding of said modulator to said<br>
nucleic acid ligand, the affinity of said nucleic acid ligand for said target molecule is enhanced]<br>
The method according to claim 1 wherein said modulator binds to free nucleic<br>
acid ligand present in said patient.<br>
The method according to claim 1 wherein said modulator binds to nucleic acid<br>
ligand present in said patient in association with said target molecule.<br>
The method according to claim 1 wherein said modulator is an oligonucleotide<br>
complementary to said nucleic acid ligand, a nucleic acid binding peptide, polypeptide or<br>
protein, or a nucleic acid binding oligosaccharide.<br>
The method according to claim 6 wherein said modulator is said oligonucleotide.<br>
The method according to claim 7 wherein said oligonucleotide comprises a<br>
sequence complementary to 6-25 nucleotides of said; nucleic acid ligand.<br>
9.        The method according to claim 8 wherein said oligonucleotide comprises a .ence complementary to 8-20 nucleotides of said nucleic acid ligand.<br>
10.	The method according to claim 9 wherein said oligonucleotide comprises a<br>
*<br>
sequence complementary to 10-15 nucleotides of said nucleic acid ligand.<br>
The method according to claim 7 wherein said oligonucleotide comprises 5-80<br>
nucleotides.<br>
The method according to claim 11 wherein said oligonucleotide comprises 10-30<br>
nucleotides.<br>
The method according to claim 12 wherein said oligonucleotide comprises 15-20<br>
nucleotides.<br><br>
The method according to daim 7 wherein said oligonucleotide bears a chemical<br>
substitution.<br>
The method according to daim 14 wherein said oligonucleotide comprises a<br>
pyrimidine substituted at the 5 position or a sugar substituted at the 2' position.<br>
The method according to claim 15 wherein said oligonucleotide comprises a 2-<br>
amino, 2'-fluoro or 2'-0-methyl substitution.<br><br>
The method according to claim 7 wherein said oligonucleotide comprises a locked<br>
nucleic acid.<br>
The method according to claim 7 wherein said oligonucleotide is complementary<br>
to a single-stranded region of said nucleic acid ligand.<br>
19.	The method according to claim 18 wherein said oligonucleotide is complementfrrjy<br>
:    1<br>
single-stranded region of said nucleic acid ligand and a double stranded region of said<br>
.	:<br>
nucleic acid ligand.<br>
The method accordmg to claim 7 wherein said nucleic acid ligand comprises a<br>
single-stranded tail.<br>
The method according to claim 20 wherein said oligonucleotide is complementer!'<br>
to said single-stranded tail of said nucleic acid ligand.<br>
The method according to claim 7 wherein said nucleic acid ligand and said<br>
oligonucleotide comprise D-nucleotides.<br>
The method according to claim 7 wherein said oligonucleotide is produced in saiid<br>
patient following administration to said patient of a construct comprising a sequence encoding<br>
said oligonucleotide.<br>
The method according to claim 1 wherein said nucleic acid ligand inhibits fibrin<br>
deposition or fibrin dissolution.<br>
The method according to claim 1 wherein said nucleic acid ligand is an<br>
anticoagulant or antithrombotic nucleic acid ligand.<br>
The method according to claim 25 wherein said modulator reverses the<br>
anticoagulant or antithrombotic effect of said nucleic acid ligand.<br>
The method according to claim 25 wherein said target molecule is a tissue factor,<br>
(TF)/faetor Vila (FVIIa), factor VMa {FVEttayfactor DCa (FKa), factor Va (FVa)/factor Xa<br>
(Fxa) enzyme complex, gpHblUa, gpIblX, gpVI, Gas6, PAI-1 (plasminogen activator inhibitor;<br>
1), coagulation factor Xffla (FXnia), Aim (anti-thrombin ID), thrombin or coagulation factor XIa(FXIa).<br>
The method according to claim 1 wherein said patient is subject to infection and<br>
receiving a nucleic acid ligand that binds interleukin, and wherein said modula^pr reduces the<br>
affinity of said nucleic acid ligand for interleukin, thereby reversing an immunosuppresive efTecjt<br>
of said nucleic acid ligand.<br>
The method according to claim 28 wherein said modulator is an oligonucleotide<br>
complementary to said nucleic acid ligand.<br>
The method according to daim 1 wherein said patient is at risk of developing<br>
autoimmunity and receiving a nucleic acid ligand that binds CTLA4, and wherein said modulator<br>
reduces the affinity of said nucleic acid ligand for CTLA4, thereby reversing an<br>
immunostimulatory effect of said nucleic acid ligand.<br>
The method according to claim 30 wherein said modulator is an oligonucleotide<br>
complementary to said nucleic acid ligand.<br>
The method according to claim 1 wherein said patient bears a tumor and is<br>
receiving a nucleic acid ligand that binds a growth factor, and wherein said modulator reduces<br>
the affinity of said nucleic acid ligand for said growth factor and thereby reverses an adverse<br>
effect of said nucleic acid ligand on healthy tissue of said patient.<br>
The method according to claim 32 wherein said modulator is an oligonucleotide<br>
complementary to said nucleic acid ligand.<br>
The method according to claim 1 wherein said patient is receiving a nucleic acid<br>
ligand that binds glucose, and wherein said modulator reduces the affinity of said nucleic acid<br>
ligand for glucose, thereby avoiding hypoglycemia in said patient.<br>
35.      The method according to claim 34 wherein said modulator is an oligonucleotide to said nucleic acid ligand.<br>
The method according to claim 1 wherein said patient is receiving a nucleic acid I<br>
ligandthat binds a receptor involved in the transmission of a iferve impulse to the neuromusculajr<br>
junction of skeletal muscle or autonomic ganglia thereby blocking neuromuscular action, and<br>
wherein said modulator reduces the affinity of said nucleic acid ligand for said receptor, thereby;<br>
reversing the neuromuscular blocking action of said nucleic acid ligand.<br>
The method according to claim 36 wherein said modulator is an oligonucleotide<br>
complementary to said nucleic acid ligand<br>
The method according to claim 1 wherein said nucleic acid ligand binds an E2F<br>
family member.<br>
A method of reversing the binding of a nucleic acid ligand bearing a label to a<br>
target tissue comprising administering to a patient receiving said nucleic acid ligand a modulator<br>
that inhibits the binding of said nucleic acid ligand to said target tissue.<br>
The method according to claim 39 wherein said modulator is an oligonucleotide<br>
complementary to said nucleic acid ligand.<br>
The method according to claim 39 wherein said label is a cytotoxic label.<br>
The method according to claim 39 wherein said label is a radioactive label.<br>
The method according to claim 39 wherein said label is a detectable label.<br>
The method according to claim 1 or 39 wherein said patient is a human.<br>
45.      The method according to claim 1 or 39 wherein said patient is a non-human<br>
A method of screening a test compound for its ability to alter the affinity of<br>
nucleic acid ligand for a target molecule comprising contacting said ribcleic acid ligand with said<br>
target molecule in the presence and absence of said test compound under conditions favoring<br>
binding of said nucleic acid ligand to said target molecule and determining whether said test<br>
compound enhances or inhibits binding of said nucleic acid ligand to said target molecule.<br>
A method of screening a test compound for potential as a modulator of nucleic<br>
acid ligand activity comprising contacting said test compound with said nucleic acid ligand and<br>
determining whether said test compound binds to said nucleic acid ligand, whereia a test<br>
compound that binds to said nucleic acid ligand is a potential modulator of said nucleic acid<br>
ligand activity.<br>
A method of altering the affinity of a nucleic acid ligand for a target molecule in<br>
vitro comprising contacting said nucleic acid ligand with a modulator that binds to said nucleic<br>
acid ligand, said contacting being effected under conditions such that said modulator binds to<br>
said nucleic acid ligand and thereby alters the affinity of said nucleic acid ligand for said target<br>
molecule, wherein said target molecule is not thrombin.<br>
A method according to claim 48 wherein said modulator is an oligonucleotide<br>
complementary to said nucleic acid ligand.<br>
An oligonucleotide having a sequence set forth in SEQ ID NO:20, 21, 22, 25-35,<br>
38, 39, 40 or 41, or a sequence at least 95% homologous therewith.<br>
A composition comprising an oligonucleotide according to claim 50 and a carrier,<br>
52.	A method of altering the affinity of a nucleic acid ligand fora target<br>
molecule in a patient comprising substantially as  herein  described (with<br>
&gt; i<br>
reference to the foregoing description and the accompanying examples and drawings.<br>
A method of reversing the binding of a nucleic acid ligand bearing a label<br>
to a target tissue comprising substantially     as   herein described   with<br>
reference to the   foregoing description and the accompanying exantples<br>
and drawings.<br>
A method of screening a test compound for its ability to alter the affinity of<br>
I nucleic acid ligand for a target molecule comprising substantially \   as<br>
herein described with reference to the foregoing description aiicf the accompanying examples and drawings.<br>
55.	A method of screening a test compound for potential as a modulator<br>
of nucleic acid ligand activity comprising substantially as herein des£ijibed<br>
• i<br>
!<br>
with reference to the foregoing description and the accompanying examples and drawings.<br>
56.	A method of altering the affinity of a nucleic acid ligand for a Nu-get<br>
I I<br>
molecule in vitro comprising substantially as herein described with reference to the foregoing description and the accompanying examples and drawings.<br>
57.	An oligonucleotide having a sequence set forth in SEQ ID No: 20, 21 j 22,<br>
25-36, 38, 39, 40 or 41 substantially   as herein described with refeirjence<br>
to    the      foregoing    description and the accompanying examples | and<br>
drawings.<br></k></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDIyMDktZGVsbnAtMjAwMy1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">02209-delnp-2003-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDIyMDktZGVsbnAtMjAwMy1hc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">02209-delnp-2003-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDIyMDktZGVsbnAtMjAwMy1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">02209-delnp-2003-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDIyMDktZGVsbnAtMjAwMy1jb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">02209-delnp-2003-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDIyMDktZGVsbnAtMjAwMy1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">02209-delnp-2003-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDIyMDktZGVsbnAtMjAwMy1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">02209-delnp-2003-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDIyMDktZGVsbnAtMjAwMy1mb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">02209-delnp-2003-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDIyMDktZGVsbnAtMjAwMy1mb3JtLTE4LnBkZg==" target="_blank" style="word-wrap:break-word;">02209-delnp-2003-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDIyMDktZGVsbnAtMjAwMy1mb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">02209-delnp-2003-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDIyMDktZGVsbnAtMjAwMy1mb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">02209-delnp-2003-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDIyMDktZGVsbnAtMjAwMy1mb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">02209-delnp-2003-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDIyMDktZGVsbnAtMjAwMy1wY3QtMjIwLnBkZg==" target="_blank" style="word-wrap:break-word;">02209-delnp-2003-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDIyMDktZGVsbnAtMjAwMy1wY3Qtc2VhcmNoIHJlcG9ydC5wZGY=" target="_blank" style="word-wrap:break-word;">02209-delnp-2003-pct-search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjIwOS1ERUxOUC0yMDAzLUNsYWltcy0xNS0wNC0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">2209-DELNP-2003-Claims-15-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjIwOS1ERUxOUC0yMDAzLUNvcnJlc3BvbmRlbmNlLU90aGVycy0tMTUtMDQtMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">2209-DELNP-2003-Correspondence-Others--15-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjIwOS1kZWxucC0yMDAzLWNvcnJlc3BvbmRlbmNlLW90aGVycy0xNi0wNC0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">2209-delnp-2003-correspondence-others-16-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjIwOS1ERUxOUC0yMDAzLUZvcm0tMS0xNS0wNC0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">2209-DELNP-2003-Form-1-15-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjIwOS1ERUxOUC0yMDAzLUZvcm0tMi0xNS0wNC0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">2209-DELNP-2003-Form-2-15-04-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjIwOS1ERUxOUC0yMDAzLUZvcm0tMy0xNS0wNC0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">2209-DELNP-2003-Form-3-15-04-2008.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="221069-temperature-compensated-voltage-controlled-oscillator.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="221071-a-n-6-substituted-7-deazapurine-compound.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>221070</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>02209/DELNP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>28/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>11-Jul-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>16-Jun-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>18-Dec-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>DUKE UNIVERSITY</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>OFFICE OF SCIENCE AND TECHNOLOGY P.O. BOX 90083 DURHAM, NC 27708-0083 U.S.A</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SULLENGER, BRUCE, A.</td>
											<td>11 ST. ANDREWS COURT, DURHAM, NC 27707, U.S.A</td>
										</tr>
										<tr>
											<td>2</td>
											<td>RUSCONI, CHRISTOPHER</td>
											<td>1429 PENNSYLVANIA AVENUE, DURHAM, NC 27705, U.S.A</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07H 21/04</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US02/16555</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-05-28</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/293,231</td>
									<td>2001-05-25</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/331,037</td>
									<td>2001-11-07</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/221070-a-pharmaceutical-composition-for-altering-the-affinity-of-a-nucleic-acid-ligand by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:02:54 GMT -->
</html>
