<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/221403-a-process-for-constructing-dna-based-molecular-marker-for-enabling-selection-of-draught-and-diseases-resistance-germplasm-screening by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:40:49 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 221403:A PROCESS FOR CONSTRUCTING DNA BASED MOLECULAR MARKER FOR ENABLING SELECTION OF DRAUGHT AND DISEASES RESISTANCE GERMPLASM SCREENING</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A PROCESS FOR CONSTRUCTING DNA BASED MOLECULAR MARKER FOR ENABLING SELECTION OF DRAUGHT AND DISEASES RESISTANCE GERMPLASM SCREENING</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>This invention relates to a process to constructing DNA-based molecular markers in plants to help identify genes and gene products that playa crucial role in detennining -how plants responds to a wide ngc of abiotic and biotic factors.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present invention relates to a process tor constructing DNA-based molecular markers in plants to delect molecular markers for various kinds stress tolerance traits in plants using a bioinformatic method.<br>
Background<br>
Plants are exposed to various adverse environmental conditions such as drought, high salt and high/low temperature"etc. and to different kinds of pathogens during their life cycle. These environmental stimuli are commonly known as abiotic stress, Biotic stress on the other hand is caused by various pathogens found in the environment.<br>
Plants respond to various kinds of stress by displaying complex, quantitative traits that involve the cumulative effect of several genes. The activation of response to any kind of stress recognition and inintiation of signal transduction processes finally result in a spatially and temporally regulated gene expression.<br>
Numerous stress inducible proteins have been identified and their corresponding genes have been isolated and sequenced. Regulatory Elements of stress-modulated genes have also been deciphered,   for example Abscisic Acid Responsive Element<br>
(ABRE),<br>
Recent developments in molecular biology and statistics along with application of information technology have opened the possibility of identifymg and using genomic variation and major genes for the improvement of commercially important crops. Application of marker based selection can be more effective in characteristics that are expressed late in plants or due to certain environmental conditions or affected by few genes.<br><br>
When  it  is  not possible  to distinguish  plant  materials  visually or by simple measurements, molecular markers can someiimes be used. The Molecular markers can used to easily discern phenotypic trails. These Molecular Markers are used as a probe a mark nucleus or chromosome. Molecular Markers may be applied for a number of purposes including determining :<br>
-	Genetic identity<br>
-	Parentage (maternity and paternity)<br>
-	Extended kinship<br>
-	Differentiation of geographic population<br>
-	Differentiation of close related relationship<br>
-	Phylogenetic relationship of species, family, genera, orders, phyla.<br>
-	Differentiation of Populations for various genetic traits like disease resistance, drought tolerance etc.<br>
There are two general types of molecular markers available for use depending on the plant and the type of assay required ;<br>
isoenzymes (isozymes) and<br>
DNA-based markers<br>
DNA -BASED MARKERS<br>
DNA is the fundamemal molecule of heredity consisting a double helix of linked nucleotides. DNA based Molecular markers are small sequences of DNA which are-associated with or "linked" to regions in a plants DNA that are responsible for a specific trait (eg. disease resistance, yield, etc.).<br><br>
There are various kinds of Conventional Markers used such as :<br>
1.	Restriction Fragment Length Polymorphism : Polymorphisms in the lengths of particular restriction fragments can be used as molecular markers. The DNA Molecule is fragmented using restriction endonuclease. Restriction endonucleases are prolein enzymes that recognize specific nucleotide sequences and cleave both strands of the DNA containing those sequences.<br>
2.	Random amplified polymorphic DNA The complexity of DNA is sufficiently high that by chance pairs of sites complementary to single octa-or decanucleotides may for amplification.<br><br>
3.	Microsatelliles : Polymorphisms in the lengths of tandemly repeated short sequences can be used as molecular markers<br>
4.	Single-Stranded Conformation Polymorphism (SSCP) : Polymorphisms in sequence, as well as in sequence length, can be used as molecular markers. The mobility in gel electrophoresis of double-stranded DNA of a given length is relatively independent of nucleotide sequence. In contrast, the mobility of single strands can vary considerably as a result of only small changes in nucleotide sequence. This fact led to the development of single-stranded conformation polymorphism (SSCP) techniques.<br>
5.	Single nucleotide Polymorphisms : Single nucleotide polymorphisms (SNP"s) can be used as Molecular markers<br><br>
However the conventional metho !s of developing markers in the laboratory is a very tedious process.<br>
Summary of the Invention<br>
The objective of the present invention is to correlatc the occurrence of Motifs (highly conserved amino acid sequences) in various stress related proteins for molecular marker development.<br>
Another objective is to identify a method for finding new markers from already existing sequences for the various kind of stress in plants.<br>
Further objective is to classify these markers for the different kinds of abiotic and biotic stress the plant face.<br>
To achieve the said objects, the present invention relates to a process for constructing DNA-based molecular markers in plants comprising:<br>
identifying and selecting the gene sequences relating to stress from<br>
available databases and literature<br>
submitting the selected gene sequence for similarity search to obtain<br>
other sequences from the database similar to the selected gene sequence<br>
subjecting the sequences obtained from similarity search to multiple<br>
alignment<br>
removing redundant sequences if any, to get a data set of proteins<br>
involved in biotic and abiotic stress response<br>
picking blocks or motifs from the data set of proteins on basis of<br>
statistical significance<br><br>
subjecting the data set of proteins to Blockmaker to pick the same set of<br>
blocks or motifs<br>
analysing the motifs for the functionality<br>
The invention can be used over a broad range of types of plants and organisms. Such plants inter alia includes cotton, maize, rice, soybeans, sugar beet, wheat, fruit, vegetables and vines. The major of use of the markers will be very useful to identify different varieties of plants that show stress tolerance.<br>
The protein sequences are of length 8 and 18.<br>
Detailed Description of the invention with the accompanying figures : Figure 1 displays the three motifs of the stress dataset along with the entropy plot, which is the n\easure of the information content at each position<br>
Figure 2 shows the motifs are mapped on to the Mannose binding letcin<br>
Table 1 shows the sequences details with their Swissprot codes.<br>
Table 2 shows the details of the evaluation of the first motif<br>
A Sequence analysis of stress related sequences, was done as follows:<br>
Stress related sequences were downloaded from Swissprot and the PIR databases and a literature study of the sequences were carried out to pick a protein, which was well characterized experimentally to be involved in stress.<br><br>
The salT gene of Oryza sativa was selected for further studies.<br>
Example 1:<br>
The salT protein was submitted for similarity search and around 65 proteins were obtained. 15 proteins were selected based on the threshold of 35% similarity and the set was reduced to 12 after removing the redundant sequences.   The data set of the<br>
twelve sequences consisted of proteins involved in various biotic and abiotic stress responses.<br>
An analysis was conducted to discover potential regions of sequence homology between twelve biotic and abiotic stress-related genes. The homology analysis resulted in 3 non-overlapping motifs that were common to both biotic and abiotic stress-related genes.<br>
A total of 113 new genes were identified. The annotation present for each of the genes supports the hypothesis that they are involved in stress-related response.<br>
Multiple alignment and statistical significance<br>
The length of sequences used for making the blocks or motifs are varied and the motifs do not occur in a specific position in all these sequences. Besides, since the proteins are made up of only 20 amino acids, a statisucai analysis is done to check whether the identified motif has occurred by chance, or whether its presence in the sequence is of any significance. The end result is of the probability of occurrence is as follows :<br>
a.	if the occurrence of this pattern is high then it is of no significance,<br>
b.	it the probability of occurrence is very low , then this probability has also a<br>
biological significance.<br><br>
The twelve sequences were then subjected to multiple alignment using clustalW. Three non-overlapping motifs were picked up manually by "eye". The statistical significance of blocks of similarity was evaluated using the MACAW (Multiple Alignment Construction and Analysis Workbench)<br>
The same data set was submitted to Blockmaker and analysed for the presence of Blocks. The same sets of blocTcs were picked up by the Program.<br>
Analysis of Motifs using MEME (Multiple Expectation Maximization for Motif Elicitation). The three strongest motifs in the set of 12 sequences of twelve divergent sequences were determined using MEME 2.0.<br>
These motifs were used to generate a Position Specific Scoring Matrix (PSSM) in order to identify further stress-related genes from the public sequence databases. The Position Specific Scoring Matrix of the MEME output was then used to search the Genbank and Swissprot 39.4 using the MAST (Motif Alignment and search tool)<br>
The three motifs map on to functionally important domains. The first motif relates to a common epitope and the third motif maps on lo an important N-glycosylation site.<br>
Motif listings:<br>
1	18   VrrSLTFKTNKKTYGPFG<br>
2	8   GPWGGNGG<br>
3	16   IVGFFGRSGWYLDAIG<br><br>
REFERENCES:<br>
1.	Thompson, J. D., Higgins, D.G. and Gibson, T.J. (1994) CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, positions-specific gap penalties and weight matrix choice. Nucleic Acids Research, 22:4673-4680.<br>
2.	Schuler, G.D., Altschul, S.F, Lipman, D.J. (1991) A workbench for multiple alignment construction and ai;alysis. Proteins: Struclure, Function and Genetics 9:180-190,<br>
3.	http://blocks.fhcrc.org/blocks/blockmkr/make_blocks.html<br>
4.	Henikoff, S., Heoikoff, J.G, Alford, W.J, and Pietrokovski, S. (1995), Automated construction and graphical presentation of protein blocks from unaligned sequences. Gene 163:GC17-26,<br>
5.	Timothy L. Bailey and Charles Elkan, "l-"itting a mixture model by expectation maximization to discover motifs in biopolymers", Proceedings of the Second International Conference on Intelligent Systems for Molecular Biology, pp. 28-36, AAAI Press, Menlo Park, California, 1994.<br>
6.	http://meme.sdsc.edu/meme/wcbsite/masl.htmi<br>
7.	Timothy L. Bailey and Michael Gribskov, "Combining evidence using p-values: application to sequence homology searches", Bioinformatics, 14:1, pp. 48-54.<br>
S.Tsuda.M    (1979) Purification    and characterisation    of a lectin  from rice.<br>
J.Biochem. 86 : 1451-1461 9. Ko Hirano Tohru Teraoka, Homare Yamanaka, Akane Harashima,   Akiko<br>
Kunisaki, Hideki Takashi and Daiiro Hosokawa Novel Mannose-Binding Rice<br>
Lectin Composed of some Isolectins and its relation to a Stress-lnducible salT<br>
Gene, Plant Cell Physiol. 41(3): 258-267 (2000)<br><br><br>
We claim:<br>
1.     A process for constructing DNA-based molecular markers in plants comprising:<br>
identifying and selecting the gene sequences relating to stress from<br>
available databases and literature<br>
submitting the seiccled gene sequence for similarity search to obtain<br>
other sequences from the database similar to the selected gene sequence<br>
subjecting the sequences obtained from similarity search to multiple<br>
alignment<br>
removing redundant sequences if any, to gel a data set of proteins<br>
involved in biotic and abiotic stress response<br>
picking blocks or motifs from the data set of proteins on basis of<br>
statistical significance<br>
subjecting the data set of proteins to Blockmaker lo pick the same set of<br>
blocks or motifs<br>
analysing the motifs "or the functionality<br>
2.	A process for constructing molecular markers as claimed in claim I wherein the gene selected is that of Oryza sativa<br>
3.	A process for constructing molecular markers as claimed in claim I wherein the database used is Swissprot and PIR<br>
4.	A process for constructing molecular markers as claimed in claim 1 wherein the software used to subject the sequences to multiple alignment is clustalW<br><br>
5.	A process for constructing molecular markers as claimed in claim 1 wherein the software used to conduct the similarity search is Multiple Alignment Construction and Analysis Workbench (MACAW)<br>
6.	A process for constructing molecular markers as claimed in claim 1 wherein the software used for marking blocks are the Blockmakers<br>
7.	A process for constructing molecular markers as claimed in claim 1 wherein the motifs are analyzed using Multiple Expectation Maximization for Motif Elicitation (MEME)<br>
8.	A process for constructing molecular markers as claimed in claim 1 wherein the amino acid sequence or the notif in the isolated protein sequences are 8 to 18<br>
9.	A process for constructing molecular markers as claimed in claim 1 wherein the motif 1 is VITSLTFKTNKKTYGPFG<br>
10.	A process for constructing molecular markers as claimed in claim 1 wherein the motif 2 is GPWGGNGG<br>
11.	A process for constructing molecular markers as claimed in claim 1 wherein themotif 3 is   IVGFFGRSGNYLDAIG<br>
12.	A process for constructing molecular markers as claimed in claim 9 wherein the motif I relates to a common epitope<br><br>
13.	A process for constructing molecular markers as claimed in claim 11 wherein<br>
the motif 3 maps an important n-glycosylation site<br>
14.	A process for constructing DNA-based markers molecular markers in plants<br>
using bioinformatics methods substantially as herein described with reference to<br>
the accompanying drawings.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDc0OS1jaGUtMjAwMyBhYnN0cmFjdCBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">0749-che-2003 abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDc0OS1jaGUtMjAwMyBhYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">0749-che-2003 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDc0OS1jaGUtMjAwMyBjbGFpbXMgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">0749-che-2003 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDc0OS1jaGUtMjAwMyBjbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">0749-che-2003 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDc0OS1jaGUtMjAwMyBjb3JyZXNwb25kYW5jZSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">0749-che-2003 correspondance others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDc0OS1jaGUtMjAwMyBjb3JyZXNwb25kYW5jZSBwby5wZGY=" target="_blank" style="word-wrap:break-word;">0749-che-2003 correspondance po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDc0OS1jaGUtMjAwMyBkZXNjcmlwdGlvbihjb21wbGV0ZSkgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">0749-che-2003 description(complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDc0OS1jaGUtMjAwMyBkZXNjcmlwdGlvbihjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">0749-che-2003 description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDc0OS1jaGUtMjAwMyBkcmF3aW5ncyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">0749-che-2003 drawings duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDc0OS1jaGUtMjAwMyBkcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">0749-che-2003 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDc0OS1jaGUtMjAwMyBmb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">0749-che-2003 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDc0OS1jaGUtMjAwMyBmb3JtLTE5LnBkZg==" target="_blank" style="word-wrap:break-word;">0749-che-2003 form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDc0OS1jaGUtMjAwMyBmb3JtLTI2LnBkZg==" target="_blank" style="word-wrap:break-word;">0749-che-2003 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDc0OS1jaGUtMjAwMyBmb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">0749-che-2003 form-3.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="221402-a-structure-for-mounting-on-a-container-and-a-method-for-securing-a-pump.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="221404-improvements-in-and-relating-to-liquid-dispensing-apparatus.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>221403</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>749/CHE/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>37/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>12-Sep-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>23-Jun-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>18-Sep-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>AVESTHA GENGRAINE TECHNOLOGIES PVT. LTD.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>SUNBEAM 106 PRENDERGHAST ROAD, SECUNDERABAD -500 003</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>DR. PATELL VILLOO MORAWALA</td>
											<td>SUNBEAM 106 PRENDERGHAST ROAD, SECUNDERABAD 500 003,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>JAGANNATHAN VIDYA</td>
											<td>21, 3RD STREET, TATABAD, COIMBATORE 641 012,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/221403-a-process-for-constructing-dna-based-molecular-marker-for-enabling-selection-of-draught-and-diseases-resistance-germplasm-screening by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:40:50 GMT -->
</html>
