<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/221836-a-method-of-selecting-a-protein-varient-having-reduced-immunogencity-and-to-the-protein-varient by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:12:11 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 221836:A METHOD OF SELECTING A PROTEIN VARIENT HAVING REDUCED IMMUNOGENCITY AND TO THE PROTEIN VARIENT</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD OF SELECTING A PROTEIN VARIENT HAVING REDUCED IMMUNOGENCITY AND TO THE PROTEIN VARIENT</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a method of selecting a protein variant having reduced immunogenicity as compared to the parent protein comprising the steps of screening a random peptide display package library with antibodies raised against any protein of interest, sequencing the amino acid sequence of the antibody binding peptides, or the DNA sequence encoding the antibody binding peptides, identifying epitope patterns of a protein by sequence alignment of the reactive peptide sequence, localisation of epitope patterns on the primary 3-dimensional structure of the parent protein, defining an epitope area including amino acids situated within 5 Å from the epitope amino acids, and affecting antibody binding to the epitope, changing the localised epitope patterns, or amino acids defining the epitope area of the parent protein by genetical engineering mutations of a DNA sequence encoding the parent protein without impairing functionality of the protein using the emerging epitope database for eliminating amino acid substitutions creating new or duplicating existing epitope patterns, introducing the mutated DNA sequence into a suitable host, culturing said host and expressing the protein variant, and evaluating the immunogenicity of the protein variant using the parent protein as reference. The invention further relates to the protein variant and use thereof, as well as to a method for producing said protein variant.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
The present invention relates to a method of selecting a protein variant having reduced immunogenicity. as compared to the parent protein, to the protein variant and use thereof, as well as to a method for producing said protein variant.<br>
Background of the invention<br>
An increasing number of proteins, including enzymes, are being produced industrially, for use in various industries, housekeeping and medicine. Being proteins they are likely to stimulate an immunological response in man and animals, including an allergic response.<br>
In the present context the terms allergic response, allergy, allergenic and allergenicity are used according to their usual definitions, i.e. to describe the reaction due to immune responses wherein the antibody most often is IgE, less often IgG4 and diseases due to this immune response. Allergic diseases include urticaria, hay-fever, asthma, and atopic dermatitis'. They may even 'evolve into an anaphylactic shock.<br>
Prevention of allergy in susceptible individuals is therefore a research area of great importance. Depending on the application, individual's get sensitised to the respective allergens by inhalation, direct contact with skin and eyes, or injection. The general mechanism behind an allergic response is divided in a sensitisation phase and a symptomatic phase. The sensitisation phase involves a first exposure of an individual to an allergen. This event activates specific T- and B-lymphocytes, and leads to the production of allergen specific immunoglobulin E (IgE) antibodies. These IgE antibodies even-<br><br>
tually facilitate allergen capturing and presentation to T-lymphocytes at the onset of the symptomatic phase. This phase is initiated by a second exposure to the same or a resembling antigen. The specific IgE antibodies bind to the specific IgE receptors on mast cells and basophils, among others, and capture at the same time the allergen. The polyclonal nature of this process results in bridging and clustering of the IgE receptors, and subsequently in the activation of mast cells and basophils. This activation triggers the release of various chemical mediators involved in the early as well as late phase reactions of the symptomatic phase of allergy.<br>
In the present context the antibodies are denoted as usual, i.e. immunoglobulin E is IgE etc.<br>
Various attempts to reduce the immunogenicity of .polypeptides and proteins have been conducted. It has been found that small changes in an epitope may affect the binding to an antibody. This may result in a reduced importance of such an epitope, maybe converting it from a high affinity to a low affinity epitope, or maybe even result in epitope loss, i.e. that the epitope cannot sufficiently bind an antibody to elicit an immunogenic response.<br>
In WO92/10755 a method for modifying proteins to obtain less immunogenicT'""'V'ariants is described. Randomly constructed pro-tein variants, revealing a reduced binding of antibodies to the parent enzyme as compared to the parent enzyme itself, are selected for the measurement in animal models in terms of al-lergenicity. Finally, it is assessed whether reduction in immunogenicity is due to true elimination of an epitope or a reduction in affinity for antibodies, A drawback of this approach is its xtrial and error' character, which makes it a long and expensive process.   Furthermore, no information is<br><br>
provided on the epitope as such, which implies that the information obtained for one protein can not be applied on another, since the limited number of animals involved do not allow the identification of epitope patterns.<br>
Summary of the invention<br>
The present invention relates to a method of selecting a protein variant having reduced immunogenicity as compared to a parent protein, comprising the steps of:<br>
screening a random peptide display package library with antibodies raised against any protein of interest,<br>
sequencing the amino acid sequence of the antibody .binding peptides, or the DNA sequence encoding the antibody binding peptides,<br>
identifying epitope patterns by sequence alignment of the reactive peptide sequence,<br>
localisation of epitope patterns on the primary and 3-dimensional structure of the parent protein,<br>
defining an epitope area including amino acids situated within 5 A from the epitope amino acids,<br>
changing the localised epitope patterns,, or amino acids defining the epitope area of the parent protein by genetical engineering mutations of a DNA sequence encoding the parent protein without impairing functionality of the protein using the emerging epitope database for eliminating amino acid<br><br>
substitutions creating new or duplicating existing epitope patterns,<br>
introducing the mutated DNA sequence into a suitable host, culturing said host and expressing the protein variant, and<br>
evaluating the immunogenicity of the protein variant using the parent protein as reference.<br>
A second aspect of the present invention is a protein variant having reduced immunogenicity as compared to parent protein. The amino acid sequence of the protein variant differs from the amino acid sequence of the parent protein with respect -to at least one epitope pattern of the parent protein, such that the immunogenicity .of the protein variant is reduced as compared with the immunogenicity of the parent protein. Preferably, the immunogenicity of the protein variant of a detergent enzyme is at least 10 times lower than the immunogenicity of the parent protein, preferably 25 times lower than the immunogenicity of the parent protein. In particular for a protien variant used in personal care products the .immunogenicity is preferably 100 times lower, more preferably 200 .times lower, even more preferably 500 times lower than the immunogenicity of the parent protein.  Furthermore,  for food products and pharmaceuticals the immunogenicity is preferably 1000 times lower than the immunogenicity of the parent protein. Normally, a reduction of at most 50.000 times of the immunogenicity is sufficient for all purposes of protein application. In this context the immunogenicity is assessed by the potential of the protein to evoke an antibody response with focus in IgE responses ♦<br><br>
Reduction of immunogenicity in tne context auuvt; xs ucuncu uy the differences observed among the kinetics obtained with various doses of the parent protein/ as compared to the variants under investigation. In the present study the effect of epitope guided protein engineering is assessed by kinetic studies using a single dose of the protein under investigation.<br>
The immunogenicity of the protein variant is measured in animal tests, wherein the animals are immunised with the protein variant and the immune -response is measured. By the present invention the immunogenicity is reduced at least 100 times as compared to the immunogenicity of the parent protein, preferably 200 times reduced, more preferably 500 times.<br>
However, the present inventors have demonstrated that the performance in a competitive ELISA correlates closely to the immunogenic responses measured in animal tests.<br>
To obtain a useful reduction of the immunogenicity or a pro-&gt; tein, the immunogenicity of the protein variant must be reduced to at least below 75 %, preferably below 50 % of the immunogenicity of the parent protein as measured by the perform-' ance in competitive ELISA, given the value for immunogenicity of the parent protein is set to 100 %. When the immunogenicity is below 50 % of the immunogenicity of the parent protein, the protein variant is practically non-allergenic, that is, no IgE response will be measurable in animal tests.<br>
A further aspect of the present invention is a composition comprising a protein variant as defined above, as well as the use of the composition for industrial, application, such as the production of a formulation for personal care products (for example shampoo; soap; skin, hand and face lotions; skin, hand and face cremes; hair dyes; toothpaste) , food (for example in<br><br>
the baking industry), detergents and for the production of pharmaceuticals.<br>
Yet another aspect is a DNA molecule encoding a protein variant as defined above.<br>
Further aspects are a vector comprising a DNA molecule as described above as well a host cell comprising said DNA molecule.<br>
Another aspect is a method of producing a protein variant having reduced immunogenicity as compared to the parent protein as defined above.<br>
Accordingly the present invention provides a method of selecting a protein variant having reduced immunogenicity as compared to a parent protein, comprising the steps of:<br>
screening a random peptide display package library with antibodies raised against any protein of interest, sequencing the amino acid sequence of the antibody binding peptides, or the DNA sequence encoding the antibody binding peptides, identifying epitope patterns by sequence alignment of the reactive peptide sequence, localisation of structural epitope patterns on the 3-dimensional structure of the parent protein, defining an epitope area including amino acids situated within 5 A from the epitope amino acids, changing the localised epitope patterns, or amino acids defining the epitope area of the parent protein by genetical engineering mutations of a DNA sequence encoding the parent protein without impairing functionality of the protein using the emerging epitope database for eliminating amino acid substitutions creating new or duplicating existing epitope patterns, introducing the mutated DNA sequence into a suitable host, culturing said host and expressing the protein variant, and evaluating the immunogenicity of the protein variant using the parent protein as reference.<br><br>
Accordingly the present invention also provides a protein variant having reduced immunogenicity as compared to its parent protein, wherein the amino acid sequence of the protein variant differs from the amino acid sequence of the parent protein with respect to at least one structural epitope area of the parent protein, and the epitope areas are defined on the parent protein structure by being localised less than 5 A from any of the following epitope patterns: R Y P R/K, S G P R A G, P R/K SDPG,DP&gt; RDTG,AR&gt;R&gt;A&gt;N,NN&gt;EL,R/KRFA/SN&gt; E/D, EY&gt;M,P&gt;PAP&gt; S, A K I D P R/K, ADS&gt;GYP, SRSA, L &gt; G R S S, with the exception of Subtilisin 309 with the mutation Y167I.<br>
Brief description of the drawings<br>
Figure 1 shows the 3D-struture of Savinase and Lipoprime with the location of 4 epitopes and their corresponding epitope areas.<br>
Figure 2 shows competitive ELISA data.<br>
Detailed description of the invention<br>
By the term a protein variant having reduced immunogenicity as compared to the parent protein is meant a protein variant which differs from the parent protein in one or more amino acids whereby the immunogenicity of the variant is reduced. The reduction of immunogenicity may be confirmed by testing the ability of the protein variant to elicit an IgE response.<br>
In the present context the term protein is intended to cover oligopeptides, polypeptides as well as proteins as such.<br><br>
The display package can be, for example, derived from bacterial viruses. A preferred display package may be a phage display system. In a phage display system, a sequence encoding a desired amino acid sequence is incorporated into a phage gene coding for a protein displayed on the surface of the phage. Thus, the phage will make and display the hybrid protein on its surface, where it can interact with specific target agents. Given that each phage was given one specific sequence of a determined length, an average phage display library can express 108 - 1012 different random sequences. The incorporated sequence being an epitope,# the target agent could for example be an epitope-specific antibody. Thus, it is possible to select specific phages from the bulk of a large number of phages, each expressing their one hybrid protein.<br>
• In a preferred embodiment, the length of the incorporated peptide is 9 amino acids.<br>
The antibodies used for reacting with the display package are preferably IgE antibodies to ensure that the epitopes identified are IgE epitopes, i.e. epitopes inducing and binding IgE. In a preferred embodiment the antibodies are polyclonal antibodies, optionally monospecific antibodies.<br>
By the term "monospecific antibodies" is meant polyclonal an-<br>
*<br>
tibodies isolated according to their specificity, for a certain epitope pattern. Such monospecific antibodies will only bind to one epitope pattern, but .they will most often be produced by a number of antibody producing cells, recognising similar epitope patterns, thereby being polyclonal.<br><br>
For the purpose of the present invention polyclonal antibodies are preferred in order to obtain a broader knowledge about the epitopes of a protein.<br>
Furthermore, the antibody IgE is normally found in very low concentration in serum. Normal serum IgE antibody concentration is below 10"3 g/1, as compared to normal IgG concentration of 12 g/1. Accordingly, in practice it is very difficult to obtain a monoclonal antibody library covering all possible epitope patterns on an intact protein. However, after affinity purification of IgE antibodies from serum raised against the protein it is possible to obtain polyclonal or monospecific IgE antibodies suitable for the present invention.<br>
In general, the parts of a protein recognized and bound by antibodies, i.e. the .epitopes, may be linear or structural. By linear epitope is meant that the amino acids of the epitope are placed in sequence in the primary protein structure. Structural epitopes differ from linear epitopes in that they do not appear until after the threedimensional structure of the protein is formed. A structural epitope is comprised of amino acid sequences from different parts .of the primary structure of the protein, only brought together, after the folding of the protein sequence forming the secondary and/or tertiary structures, thereby forming a contiguous epitope surface. By the term an epitope pattern or epitope motif is meant the epitope constituting surface of the protein independent of whether the epitope is linear or structural. Some of the epitopes are more important (high affinity epitopes) than other (low affinity) with respect to binding affinity.<br>
A protein may have more than one epitope. From theoretical models it is estimated that proteins with a molecular size of 30.000 Daltons may carry up to 6 epitope areas on its surface.<br><br>
The total number of actual epitopes however must be much higher. For example, a 9-mer amino acid sequence provides binding to at least 5 different monoclonal antibodies, considering a 5-mer as the minimum epitope size.<br>
It has been found that some of the IgGl antibody binding epitopes do not'bind IgE antibodies, and vice versa. Accordingly, the epitopes may be more or less specific for specific antibody classes.<br>
More than 80 % of the IgE epitopes are considered structural epitopes, whereby the epitopes can be destroyed during heating of the protein, the use of detergents or other procedures implying denaturation. This percentage may vary for different proteins- Consequently, the use of overlapping peptide fragments covering the entire protein for the identification of IgE epitopes is not applicable. However,'the present inventors find that by the use of small amino acid sequences in a display package it is possible to identify epitope patterns of linear as well as structural epitopes, due to the random character of the sequence of the expressed oligopeptides.<br>
The peptides presented on the display packages and identified by the antibodies are then sequenced, or their sequence is derived by translation of the corresponding DNA sequence. Both sequences are obtained by techniques known to the person skilled in the art of genetic engineering.<br>
The epitope patterns are identified by comparison of the sequences of the peptides bound by the antibody. Subsequently, the identified patterns are localised on the primary as well as 3-dimensional structure of the parent protein, and eventually on any 'protein of interest. Within the epitope patterns some amino acids may be highly conservative, called anchor amino acids. The anchor amino acids will be recurring in all<br><br>
the peptides bound by monospecific antibodies, and define the epitope pattern.<br>
It is of great importance that the amino acid sequence of the peptides presented by the display packages is long enough to present a significant part of the epitope to be identified. In a preferred embodiment of the invention the peptides of the peptide display package library are oligopeptides having from 5 to 25 amino acids, preferably at least 8 amino acids, such as 9 amino acids. The complexity of the package library, which is the number of display packages to get a suitable representation of all possible epitopes, may be calculated on basis of the estimated length of the epitope to be identified, such as by the formula below<br>
Number of packages = 20n, wherein n is the number of residues in the epitope.<br>
Typical actions required for the construction of a model structure are: alignment of homologous sequences for which 3-dimensional structures exist, definition of Structurally Conserved Regions (SCRs), assignment of coordinates to SCRs, search for structural fragments/loops in structure databases to replace Variable Regions, assignment of coordinates to these   regions,   and   structural   refinement   by   energy<br>
minimization. Regions containing large inserts (&gt;3 residues) relative to the known 3-dimensional structures are known to be quite difficult to model, and structural predictions must be considered with care.<br>
Having obtained the 3-dimensional structure of the polypeptide in question, or a model of the structure based on homology to known structures, this structure serves as an es-<br><br>
sential prerequisite for the fulfillment of the method described below.<br>
When the epitope(s) have been identified, a protein variant exhibiting a reduced immunogenicity may be produced by changing the identified epitope pattern of the parent protein by mutation and/or genetic engineering of a DNA sequence encoding the parent protein.<br>
The epitope identified may be changed by substituting at least one amino acid of the epitope area. In a preferred embodiment at least one anchor amino acid is changed. The change will often be substituting to an amino acid of different size, hydro-philicity, and/or polarity, such as a small amino acid versus a large amino acid, a hydrophilic amino acid versus a hydrophobic amino acid, a polar amino acid versus £ non-polar amino acid and a basic versus an acidic amino acid.<br>
Other changes may be the addition or deletion of at least one amino acid of the epitope, preferably deleting an anchor amino acid. Furthermore, an epitope pattern may be changed by substituting some amino acids, and deleting/adding other.<br>
Substitutions/Insertions for subsequent covalent conjugation to amino acids in the epitope area.<br>
Which amino acids to substitute and/or insert depends in principle on the coupling chemistry to be applied. The chemistry for preparation of covalent bioconjugates can be found in "Bioconjugate Techniques", Hermanson, G.T. (1996), Academic Press Inc. which is hereby incorporated as reference.<br>
It is preferred to conjugates activated polymers to amino acids in the epitope area.<br><br>
It is preferred to make . conservative substitutions in the polypeptide when the polypeptide has to be conjugated, as conservative substitutions secure that the impact of the substitution on the polypeptide structure is limited.<br>
In the case of providing additional amino groups this may be' done by substitution of Arginine to Lysine,  both residues being positively charged, but only the Lysine having a free amino group suitable as an attachment groups.<br>
In the case of providing additional carboxylic acid groups the conservative substitution may for instance be an Asparagine to Aspartic acid or Glutamine to Glutamic acid substitution. These residues resemble each other in size and shape, except from the carboxylic groups being present on the acidic residues.<br>
In the case of providing SH-groups the conservative substitution may be done by substitution of Threonine or Serine to Cysteine.<br>
Activation of polymers<br>
If the polymeric molecules to be conjugated with the polypeptide in question are not active it must be activated by the use of a suitable technique. It is also contemplated according to<br>
the invention to couple the polymeric molecules to the polypep-<br>
»<br>
tide through a linker. Suitable linkers are well-known to the skilled person.<br>
Methods and chemistry for activation of polymeric molecules as well as for conjugation of polypeptides are intensively described in the literature. Commonly used methods for activation of insoluble polymers include activation of functional groups with cyanogen  bromide,   periodate,   glutaraldehyde,   biepoxides,<br><br>
epichlorohydrin, divinylsulfone, carbodiimide, sulfonyl hali-des, trichlorotriazine etc. (see R.F. Taylor, (1991), "Protein immobilisation. Fundamental and applications", Marcel Dekker, N. Y. ; S. S. Wong, (1992), "Chemistry of Protein Conjugation and Crosslinking", CRC Press, Boca Raton; G. T. Hermanson et al., (1993), "Immobilized Affinity Ligand Techniques", Academic Press, N.Y.). Some of the methods concern activation of insoluble polymers but are also applicable to activation of soluble polymers e.g. periodate, trichlorotriazine, sulfonylhalides, divinylsulfone, carbodiimide etc. The functional groups being amino, hydroxyl, thiol, carboxyl, aldehyde or sulfydryl on the polymer and the chosen attachment group on the protein must be considered in choosing the activation and conjugation chemistry which normally consist of i) activation of polymer, ii) conjugation, and iii) blocking of residual active groups.<br>
In the following a number of suitable polymer activation methods will be described shortly. However, it is to be understood that also other methods may be used.<br>
Coupling polymeric molecules to the free acid groups of polypeptides may be performed with the aid of diimide and for example amino-PEG or hydrazino-PEG (Pollak et al., (1976), J. Amr. Chem. Soc., 98, 289-291) or diazoacetate/amide (Wong et al., (1992),  "Chemistry of Protein Conjugation and Crosslinking",<br>
CRC Press) .<br>
■<br>
Coupling polymeric molecules to hydroxy groups are generally very difficult as it must be performed in water. Usually hydrolysis predominates over reaction with hydroxyl groups.<br>
Coupling polymeric molecules to free sulfhydryl groups can be reached with special groups like maleimido or the ortho-pyridyl disulfide. Also vinylsulfone (US patent no. 5,414,135, (1995),<br><br>
Snow et al.) has a preference for sulfhydryl groups but is not as selective as the other mentioned.<br>
Accessible Arginine residues in the polypeptide chain may be targeted by groups comprising-two vicinal carbonyl groups.<br>
Techniques involving coupling electrophilically activated PEGs to the amino groups of Lysines may also be useful. Many of the usual leaving groups for alcohols give rise- to an amine linkage. For instance, alkyl sulfonates, such as tresylates (Nilsson et al., (1984), Methods in Enzymology vol, 104, Jacoby, W. B., Ed., Academic Press: Orlando, p. 56-66; Nilsson et al., (1987), Methods in Enzymology vol. 135; Mosbach, K. , Ed. ; Academic Press: Orlando, pp. 65-79; Scouten et al., (1987), Methods in Enzymology vol. 135, Mosbach, K., Ed., Academic Press: Orlando, 1987; pp 79-84; Crossland et al., (1971) , J. Amr. Chem. Soc. 1971, 93, pp. 4217-4219), mesylates (Harris, (1985), supra; Harris et al., (1984), J. Polym. Sci. Polym. Chem. Ed. 22, pp 341-352), aryl sulfonates like tosylates, and para-nitrobenzene sulfonates can be used.<br>
Organic sulfonyl chlorides, e.g. Tresyl chloride, effectively converts hydroxy groups in a number of polymers, e.g. PEG, into good leaving groups (sulfonates) that, when reacted with nu-cleophiles like amino groups in polypeptides allow stable linkages to be formed between polymer and polypeptide. In addition to high conjugation yields, the reaction conditions are in general mild (neutral or slightly alkaline pH, to avoid denatura-tion and little .or no disruption of activity), and satisfy the non-destructive requirements to the polypeptide.<br>
Tosylate is more reactive than the mesylate but also more unstable decomposing into PEG, dioxane, and sulfonic acid (Zalipsky,  (1995), Bioconjugate Chem.,  6,  150-165). Epoxides<br><br>
may also been used for creating amine bonds but are much less reactive than the above mentioned groups.<br>
Converting PEG into a chloroformate with phosgene gives rise to carbamate linkages to Lysines. This theme can be played in many variants substituting the chlorine with N-hydroxy succinimide<br>
(US patent no. 5,122,614, (1992); Zalipsky et al., (1992), Bio-technol. Appl. Biochem., 15, p. 100-114; Monfardini et al.,<br>
(1995), Bioconjugate Chem.*, 6, 62-69, with imidazole (Allen et al., (1991), Carbohydr. Res., 213, pp 309-319), with para-nitrophenol, DMAP (EP 632 082 Al, (1993), Looze, Y.) etc. The derivatives are usually made by reacting the chloroformate with the desired leaving group. All these groups give rise to carbamate linkages to the peptide.<br>
Furthermore, isocyanates and isothiocyanates may be employed yielding ureas and thioureas, respectively.<br>
Amides may be obtained from PEG acids using the same leaving groups as mentioned above and cyclic imid thrones (US patent no. 5,349,001, (1994), Greenwald et al.). The reactivity of these compounds are very high but may make the hydrolysis to fast.<br>
PEG succinate made from reaction with succinic anhydride can also be used. The hereby comprised ester group make the conju-gate much more susceptible to hydrolysis (US patent no. 5,122,614, (1992), Zalipsky). This group may be activated with N-hydroxy succinimide.<br>
Furthermore, a special linker can be introduced. The oldest being cyanuric chloride (Abuchowski et al., (1977), J. Biol. Chem., 252, 3578-3581; US patent no. 4,179,337,  (1979), Davis<br><br>
et al.; Shafer et al., (1986) , J. Polym. Sci. Polym. Chem. Ed., 24, 375-378.<br>
Coupling of PEG to an aromatic amine followed by diazotation yields a very reactive diazonium salt which in situ can be reacted with a peptide. An amide linkage may also be obtained by reacting an azlactone derivative of PEG (US patent no. 5,321,095, (1994) , Greenwald, R. B. ) thus introducing an additional amide linkage.<br>
As some peptides do not comprise many Lysines it may be advantageous to attach more than one PEG to the same Lysine. This can be done e.g. by the use of 1,3-diamino-2-propanol.<br>
PEGs may also be attached to the amino-groups of the enzyme with carbamate linkages (WO^ 95/11924, Greenwald et al.). Lysine residues may also be used as the backbone.<br>
The coupling technique used in the examples is the N-succinimidyl carbonate conjugation technique descried in WO 90/13590 (Enzon).<br>
Conjugation of polymeric molecules to substitution and/ or insertion variants for covalent conjugation.<br>
Only substitutions and/or insertions which provide polypeptide-conjugates with reduced immune response when evaluated in animal models are within the concept of the present invention. The mutation(s) performed may be performed by standard techniques well known in the art, such as site-directed mutagenesis (see, e.g., Sambrook et al. (1989) , Sambroolc—efe al., Molecular Cloning. A Laboratory Manual, Cold Spring Harbor, NY.<br><br>
A general description of nucleotide substitution can be found in e.g. Ford et al., 1991, Protein Expression and Purification 2, p* 95-107.<br>
The above-mentioned DNA sequence may be produced by any suitable technique known to the person skilled in the art of genetic engineering. The DNA sequence is introduced into a suitable host, which is cultured and the protein variant expressed.<br>
The risk linked to protein engineering in order to eliminate epitopes, is that new epitopes are made, or existing epitopes are duplicated. To reduce this risk, the planned mutations at a given position, were put on the 3-dimensional structure of the protein of interest, and emerging amino acid constellation was run through the epitope database. This invention evidences that the risk mutations can be identified by this procedure, thereby reducing the number of possible mutations per position.<br>
It is a well established fact that amino acids surrounding B-and T-cell epitopes can affect binding of the antibodies or T--ell receptors to the ligand. To anticipate to this possibility, an epitope area was defined on the 3-dimensional structure of the protein of interest.<br>
Et is of great importance that the changes of the epitope (s) io not interfere with or impair the functionality of the parent protein. Accordingly, the changes must not significantly :hange the secondary structure of the protein. By computer nodelling it is possible to assess whether changes made in one Dr more epitopes will interfere substantially with the func-:ionality of the protein. By the term "functionality" is meant<br><br>
the normal function of the protein, such as the enzymatic activity, thermal stability, chemical stability, and glycosyla-tion.<br>
In a preferred embodiment of the invention, more than one amino acid residue is substituted, added or deleted, these amino acids preferably being located in different epitope areas.  In that case, it may be difficult to assess a priori how well the functionality of the protein is maintained while antigenicity,  immunogenicity and/or allergenicity is reduced, especially since the possible number of combinations of mutations become very large, even for a small number of mutations. In that case, it will be an advantage, to establishi a library of diversified mutants each having one or more changed amino acids introduced and selecting those variants which show good retention of function and at the same time a good reduction in antigenicity. In the case of protease, this can be tested by assaying the secreted variants for enzyme activityand for antigen binding (e.g. by competitive ELISA) (as described below in the experimental section).  The scope of these embodiments of the invention is by no means limited to protease, which serves only to provide an example. A diversified library can be established by a range of techniques known to the person skilled in the art (Reetz MT; Jaeger KE, in Biocatalysis   -from Discovery to Application  edited by Fessner WD,  Vol. 200, pp. 31-57 (1999); Stemmer, Nature, vol. 370, p.389-391, 1994; Zhao and Arnold, Proc. Natl. Acad. Sci., USA, vol. 94, pp. 7997-8000, 1997; or Yano et al., Proc. Natl. Acad. Sci., USA, vol. 95, pp 5511-5515, 1998). In a more preferable embodiment, substitutions are found by a method comprising the following steps: 1) a range of substitutions, additions, and/or deletions are listed encompassing several epitope areas, 2) a library is designed which introduces a randomized subset of these changes in the amino acid sequence into the target gene,<br><br>
e.g. by random mutagenesis, 3) the library is expressed, and preferred variants are selected. In a most preferred embodiment, this method is supplemented "with additonal rounds of screening and/or family shuffling of hits from the first round of screening (J.E. Ness, et al, Nature Biotechnology, vol. 17, pp. 893-896, 1999) and/or combination with other methods of reducing allergenicity by genetic means (such as that disclosed in WO92/10755).<br>
Another object of the present invention is a protein variant having reduced immunogenicity as compared to the parent protein, wherein the amino acid sequence of the protein variant differs from the amino acid sequence of the parent protein with respect to one or more amino acids of the parent protein, such that the immunogenicity of the protein variant is reduced compared to the immunogenicity of the parent protein.<br>
The reduced immunogenicity may be confirmed by testing the ability of the protein variant to elicit an IgE response in animal testing. The test animal may be any suitable animal. Preferably, the test animal may be rats, such as Brown Norway rats, or mice, such as the strains BaBallb/c, BDF1 and CB6F1.<br>
The animal testing may be carried out as described in the following.<br>
» The following is a description of the processes involved in<br>
the determination of the immunogenicity in test animals. In<br>
the example the test animals are Brown Norway rats.<br>
Materials:<br>
Molecules of a protein variant.<br><br>
tJiiib'A reagents:<br>
Horse Radish Peroxidase labelled pig anti-rabbit-Ig (Dako, DK,<br>
P217, dilution 1:1000) . * Rat anti-mouse IgE (Serotec MCA419; dilution 1:100).<br>
Mouse anti-rat IgE (Serotec MCA193; dilution 1:200).<br>
Biotin-labelled mouse anti-rat IgGl monoclonal antibody (Zymed<br>
03-9140; dilution 1:1000)<br>
Biotin-labelled  rat  anti-mouse  IgGl  monoclonal  antibody i (Serotec MCA336B; dilution 1:2000)<br>
Streptavidin-horse radish peroxidase (Kirkegard &amp; Perry 14-30-<br>
00; dilution 1:1000).<br>
Buffers and Solutions:<br>
-	PBS (pH 7.2 (1 liter))<br>
NaCl	.8.00 g<br>
KC1	0.20 g<br>
K2HP04	1.04 g<br>
KH2P04	0.32 g<br>
-	Washing buffer	PBS, 0.05% (v/v) Tween 20<br>
-	Blocking buffer	PBS, 2% (wt/v) Skim Milk powder<br>
-	Dilution buffer PBS,  0.05%  (v/v)  Tween 20, ,0.5%  (wt/v)<br>
Skim Milk powder<br>
-	Citrate buffer (0.1M, pH 5.0-5.2 (1 liter))<br>
NaCitrate	20.60 g<br>
Citric acid	6.30 g<br>
-	Stop-solution (DMG-buffer)<br>
-	Sodium Borate, borax (Sigma)<br>
-	3,3-Dimethyl glutaric acid (Sigma)<br>
-	CaCl2 (Sigma)<br>
-	Tween 20: Poly oxyethylene sorbitan mono laurate (Merck cat no. 822184)<br>
-	N-Hydroxy succinimide (Fluka art. 56480))<br><br>
-	Phosgene (Fluka art. 79380)<br>
-	Lactose (Merck 7656)<br>
-	PMSF (phenyl methyl sulfonyl flouride) from Sigma<br>
-	Succinyl-Alanine-Alanine-Proline-Phenylalanine-paranitro-anilide (Suc-AAPF-pNP) Sigma no- S-7388, Mw 624.6 g/mol.<br>
-	mPEG (Fluka)<br>
Colouring substrate:<br>
OPD: o-phenylene-diamine, (Kementec cat"no. 4260)<br>
Test Animals:<br>
Female Balb/C mice (about 20 . grams) pruchased from Bomholdt-<br>
gaard Ry, Denmark.<br>
Equipment: XCEL II (Novex)<br>
ELISA reader (UVmax, Molecular Devices)<br>
HPLC (Waters)<br>
PFLC (Pharmacia)<br>
Superdex-75 column, Mono-Q, Mono S from Pharmacia, SW.<br>
SLT:  Fotometer from SLT Lablnstruments<br>
Size-exclusion chromatograph (Spherogel TSK-G2000 SW).<br>
Size-exclusion chromatograph (Superdex 200, Pharmacia, SW)<br>
Amicon Cell<br>
Methods<br>
Subcutanuous (SC) immunisation of Balb/C mice<br>
For SC administration of molecules  0.05 ml of a solution of the molecules is deposited.<br>
The test animals are Balb/C mice in groups of 10. Weight at time of s.tart is more than 20 grams.<br><br>
ELISA procedure to determine relative concentrations of IgE antibodies in Balb/C mice.<br>
A three layer sandwich ELISA is used to determine relative concentrations of specific IgE serum anti-bodies.<br>
1)	Coat the ELISA-plate with 10 mg rat anti-mouse IgE Buffer 1<br>
(50microL/well). Incubate over night at 4°C.<br>
2)	Empty the plates and block with Blocking_ buffer for at least H hour at room temperature (200 microL/well). Shake gently. Wash the plates 3 times with Washing Buffer.<br>
3)	Incubate with mouse sera (50 microL/well), starting from undiluted and continue with 2-fold dilutions. Keep some wells free for buffer 4 only (blanks) . Incubate for 30 minutes at room temperature. Shake gently- Wash the plates 3 times in washing Buffer.<br>
4)	Dilute the enzyme in Dilution buffer to the appropriate protein concentration. Incubate 50 microL/well for 30 minutes at room temperature. Shake gently. Wash the plates 3 times in Washing Buffer.<br>
5)	Dilute specific polyclonal anti-enzyme antiserum serum (pig) for detecting bound antibody in Dilution buffer. Incubate 50 microl/well for 30 minutes at room temperature. Shake gently. Wash the plates 3 times in Washing Buffer.<br>
6)	Dilute Horseradish Peroxidase-conjugated anti-pig-antibody<br>
in Dilution buffer. Incubate 50 microL/well at room tempera<br>
ture for 30 minutes. Shake gently. Wash the plates 3 times in<br>
Washing Buffer.<br>
7)	Mix 0.6 mg ODP/ml +0.4 microL H202/ml in substrate Buffer. Make the solution just before use. Incubate for 10 minutes. 50 microL/well.<br>
8)	To stop the reaction, add 50 microL Stop Solution/well.<br>
9)	Read the'plates at 492 nm with 620 nm as reference.<br><br>
Determination of the molecular weight<br>
Electrophoretic separation of proteins is performed by standard methods using 4-20% gradient SDS polyacrylamide gels(Novex). Proteins were detected by silver staining. The molecular weight was measured relatively to the mobility of Mark-12® wide range molecular weight standards from Novex.<br>
Protease activity<br>
Analysis with Suc-Ala-Ala-Pro-Phe-pNa:<br>
Proteases  cleave  the  bond  between  the  peptide  and  p-<br>
nitroaniline to give a visible yellow colour absorbing at 405<br>
nm.<br>
Buffer: e.g.   Britton and Robinson buffer pH 8,3<br>
Substrate: 100 mg suc-AAPF-pNa is dissolved into 1 ml dimethyl<br>
sulfoxide (DMSO) . 100 \xl   of this is diluted into 10 ml with Britton and Robinson buffer. Analysis:<br>
The substrate and protease solution is mixed and the absorbance is monitored at 405 nm as a function of time and ABS405 ^/min. The temperature should be controlled (20-50°C depending on protease) . This is a measure of the protease activity in the sample.<br>
The parent protein may be any protein, such ^ as any naturally occurring protein as well as any variant thereof, optionally a variant in order to obtain a better functionality of the protein in  question-Pharmaceutical polypeptides<br><br>
The term "pharmaceutical polypeptides" is defined as polypeptides , including peptides, such as peptide hormones, proteins and/or enzymes, being physiologically active when introduced into the circulatory system of the body of humans and/or animals.<br>
Pharmaceutical polypeptides are potentially immunogenic as they are introduced into the circulatory system.<br>
Examples of "pharmaceutical polypeptides' contemplated according to the invention include insulin, ACTH, glucagon, somatostatin, somatotropin, thymosin, parathyroid hormone, pigmentary hormones, somatomedin, erythropoietin, luteinizing hormone, chorionic gonadotropin, hypothalmic releasing factors, antidiuretic hormones, thyroid stimulating hormone, relaxin, interferon, thrombopoietin (TPO) and prolactin.<br>
However, the proteins are preferably to be used in industry, housekeeping and/or medicine, such as proteins used' in personal care products (for example shampoo; soap; skin, hand and face lotions; skin, hand and face cremes; hair dyes; toothpaste), food (for example in the baking industry), detergents and pharmaceuticals.<br>
In one embodiment of the invention the protein is an enzyme, such as glycosyl hydrolases, carbohydrases, peroxidases, proteases, lipases, phytases, polysaccharide lyases, oxidoreduc-tases, transglutaminases and glycoseisomerases, in particular the foll9wing.<br>
Parent Proteases<br>
Parent proteases (i.e. enzymes classified under the Enzyme Classification number E.C. 3.4 in accordance with the Recommendations (1992) of the International Union of Biochemistry and Molecular Biology (IUBMB)) include proteases within this group.<br><br>
Examples include proteases selected from those classified under the Enzyme Classification (E.C.) numbers:<br>
3 - 4.11 (i.e. so-called aminopeptidases), including 3.4.11.5 (Prolyl aminopeptidase), 3.4.11.9 (X-pro aminopeptidase), 3.4.11.10 (Bacterial leucyl aminopeptidase), 3.4.11.12 (Thermophilic aminopeptidase), 3.4.11.15 (Lysyl aminopeptidase) , 3.4.11.17 (Tryptophanyl aminopeptidase), 3.4.11.18 (Methionyl aminopeptidase).<br>
3.4.21	(i.e.  so-called  serine  endopeptidases),  including<br>
3.4.21.1	(Chymotrypsin), 3.4.21.4 (Trypsin), 3.4.21.25<br>
(Cucumisin), 3.4.21.32 (Brachyurin), 3.4.21.48 (Cerevisin) and<br>
3.4.21.62 (Subtilisin);<br>
3.4.22	(i.e.  so-called  cysteine  endopeptidases),  including<br>
3.4.22.2	(Papain), 3.4.22.3 (Ficain), 3.4.22.6 (Chymopapain),<br>
3.4.22.7 (Asclepain), 3.4 . 22.14 (Actinidain), 3.4.22.30<br>
(Caricain) and 3.4.22.31 (Ananain);<br>
3.4.23	(i.e. so-called aspartic endopeptidases), including<br>
3.4.23.1 (Pepsin A) , 3.4.23.18 (Aspergillopepsin I), 3.4.23.20<br>
(Penicillopepsin) and 3.4.23.25 (Saccharopepsin); and<br>
3.4.24	(i.e.   so-called  metalloendopeptidases) ,   including<br>
»<br>
3.4.24.28 (Bacillolysin).<br>
Examples of relevant subtilisins comprise subtilisin BPN1, subtilisin amylosacchariticus, subtilisin 168, subtilisin mesen-tericopeptidase, subtilisin Carlsberg, subtilisin DY, subtilisin 309, subtilisin 147, thermitase, aqualysin, Bacillus PB92 protease, proteinase K, Protease TW7, and Protease TW3.<br><br>
Specific examples of such readily available commercial proteases include Esperase®, Alcalase®, Neutrase®,  Dyrazym®, Sav-<br>
inase®, Pyrase®, Pancreatic Trypsin NOVO (PTN), Bio-Feed™ Pro, ■ Clear-Lens Pro (all enzymes* available from Novo Nordisk A/S).<br>
Examples of other commercial proteases include Maxtase®, Max-acal®, Maxapem® marketed by Gist-Brocades N.V., Opticlean® marketed by Solvay et Cie. and Purafect® marketed by Genencor International.<br>
It is to be understood that also protease variants are contemplated as the parent protease. Examples of such protease variants are disclosed in EP 130.756 (Genentech), EP 214.435 (Henkel), WO 87/04461 (Amgen), WO 87/05050 (Genex), EP 251.446 (Genencor), EP 260.105 (Genencor), Thomas et al., (1985), Nature. 318, p. 375-376, Thomas et al., (1987) , J. Mol. Biol., 193, pp. 803-813, Russel et al., (1987) , Nature, 328, p. 496-500, WO 88/08028 (Genex), WO 88/08033 (Amgen), WO 89/06279 (Novo Nordisk A/S), WO 91/00345 (Novo Nordisk A/S), EP 525 610 (Solvay) and WO 94/02618 (Gist-Brocades N.V.).<br>
The activity of proteases can be determined as described in "Methods of Enzymatic Analysis", third edition, 1984, Verlag Chemie, Weinheim, vol. 5.<br>
Parent Lipases<br>
Parent lipases (i.e. enzymes classified under the Enzyme Classification number E.C. 3.1.1 (Carboxylic Ester Hydrolases) in accordance with the Recommendations (1992) of the International Union of Biochemistry and Molecular -Biology (IUBMB)) include lipases within this group.<br><br>
Examples include lipases selected from those classified under the Enzyme Classification (E.G.) numbers:<br>
3.1.1 (i.e. so-called Carboxylic Ester Hydrolases), including (3.1.1.3) Triacylglycerol lipases, (3.1.1.4.) Phosphorlipase A2m<br>
Examples of lipases include lipases derived from the following microorganisms. The indicated patent publications are incorporated herein by reference:<br>
Humicola,    e.g. H.   brevispora,    H.   lanuginosa,    H.   brevis   var.<br>
thermoidea  and H.   insolens   (US 4,810,414).<br>
Pseudomonas,    e.g.    Ps.    fragi,    Ps.    stutzeri,    Ps.    cepacia   and<br>
Ps.     fluorescens     (WO 89/04361) ,  or Ps.    plantarii    or Ps.<br>
gladioli    (US patent no. 4,950,417 (Solvay enzymes)) or Ps.<br>
alcaligenes   and Ps.   pseudoalcaligenes    (EP 218 272) or Ps.<br>
*<br>
mendocina   (WO 88/09367; US 5,389,536).<br>
Fusarium,   e.g.   F.   oxysporum   (EP 130,064) or F. solani pisi (WO 90/09446).<br>
Mucor   (also called Rhizomucor), e.g. M.   miehei   (EP 238023). Chromobacterium   (especially C.   viscosum) . Aspergillus   (especially A.   niger).<br>
Candida, e.g. C. cylindracea (also called C. rugosa) or C. antarctica (WO 88/02775) or C. antarctica lipase A or B (WO 94/01541 and WO 89/02916).<br>
Geotricum,   e.g.   G.   candidum   (Schimada et al., (1989), J. Biochem., 106, 383-388).<br>
Penicillium,   e.g.   P.   camembertii   (Yamaguchi et al., (1991) , Gene 103, 61-67).<br>
Rhizopus,   e.g.   R.   delemar   (Hass et al., (1991), Gene 109, 107-113) or R.   niveus   (Kugimiya et al., (1992) Biosci. Biotech. Biochem 56, 716-719) or R.   oryzae. Bacillus,   e.g.   B.   subtilis   (Dartois et al., (1993)<br><br>
Biochemica et Biophysica.acta 1131, 253-260) or<br>
B. stearothermophilus   (JP 64/7744992) or B.   pumilus   (WO<br>
91/16422).<br>
Specific examples of readily available commercial lipases include Lipolase®, Lipolase™ Ultra, Lipozyme®, Palatase®, No-vozym® 435, Lecitase® (all available from Novo Nordisk A/S) .<br>
Examples of other lipases are Lumafast™, Ps.   mendocian   lipase<br>
from Genencor Int. Inc.; Lipomax™, Ps. pseudoalcaligenes lipase from Gist Brocades/Genencor Int. Inc.; Fusarium solani lipase (cutinase) from Unilever; Bacillus sp. lipase from Solvay enzymes. Other lipases are available from other companies.<br>
It is to be understood that also lipase variants are contem-<br>
* plated as the parent enzyme. Examples of such are described in<br>
e.g. WO 93/01285 and WO 95/22615.<br>
The activity of the lipase can be determined as described in "Methods of Enzymatic Analysis", Third Edition, 1984, Verlag Chemie, Weinhein, vol. 4, or as described in AF 95/5 GB (available on request from Novo Nordisk A/S).<br>
Parent Oxidoreductases<br>
Parent oxidoreductases (i.e. enzymes classified under the Enzyme Classification number E.C. 1 (Oxidoreductases) in accordance with the Recommendations (1992) of the International Union of Biochemistry and Molecular Biology (IUBMB)) include oxidoreductases within this group.<br>
Examples include oxidoreductases selected from those classified under the Enzyme Classification (E.C.) numbers:<br><br>
Glycerol-3-phosphate dehydrogenase _NAD+_ (1.1.1.8), Glycerol-3-phosphate  dehydrogenase  _NAD(P)+_  (1.1.1.94),  Glycerol-3-phosphate 1-dehydrogenase _NADP_ (1.1.1.94),  Glucose oxidase (1.1.3.4),   Hexose  oxidase   (1.1.3.5) ,   Catechol  oxidase (1.1.3.14), Bilirubin oxidase (1.3.3.5), Alanine dehydrogenase (1.4.1.1) , Glutamate dehydrogenase (1.4.1.2), Glutamate dehydrogenase _NAD(P)+_ (1.4.1,3), Glutamate dehydrogenase _NADP+_ (1.4.1.4), L-Amino acid dehydrogenase (1.4.1.5), Serine dehydrogenase  (1.4*1.7),  Valine dehydrogenase _NADP+_  (1.4.1.8), Leucine   dehydrogenase   (1.4.1.9) ,   Glycine   dehydrogenase (1.4.1.10), L-Amino-acid oxidase (1.4.3.2.), D-Amino-acid oxidase (1.4.3.3), L-Glutamate oxidase  (1.4.3.11),  Protein-lysine 6-oxidase (1.4.3.13), L-lysine oxidase (1.4.3.14), L-Aspartate oxidase (1.4.3.16), D-amino-acid dehydrogenase (1.4.99.1), Protein  disulfide  reductase  (1.6.4.4),  Thioredoxin  reductase (1.6.4.5), Protein disulfide reductase (glutathione) (1.8.4.2), Laccase (1.10.3.2), Catalase (1.11.1.6), Peroxidase (1.11.1.7), Lipoxygenase (1.13.11.12), Superoxide dismutase (1.15.1.1)<br>
Said Glucose oxidases may be derived from Aspergillus niger.<br>
Said Laccases may be derived from. Polyporus    pinsitus,    My-celiophtora   thermophila,   Coprinus  cinereus,   Rhizoctonia   solani, Rhizoctonia   praticola,    Scytalidium   thermophilum   and Rhus   ver-nicifera.<br>
Bilirubin oxidases may be derived from Myrothechecium   verru-caria.<br>
The Peroxidase may be derived from e.g. Soy bean, Horseradish or Coprinus cinereus.<br>
The Protein Disulfide reductase may be any of the mentioned in DK patent applications No.  7 68/93,  2 65/94 and 264/94  (Novo<br><br>
Nordisk A/S), which are hereby incorporated as references, including Protein Disulfide reductases of bovine origin, Protein Disulfide reductases derived from Aspergillus oryzae or Aspergillus niger,   and DsbA or DsbC derived from Escherichia  coli.<br>
Specific examples of readily available commercial oxidoreduc-tases include Gluzyme™ (enzyme available from Novo Nordisk A/S). However, other oxidoreductases are available from others. It is to be understood that also variants of oxidoreductases are contemplated as the parent enzyme.<br>
The activity of oxidoreductases can be determined as described in "Methods of Enzymatic Analysis11, third edition, 1984, Verlag Chemie, Weinheim, vol. 3.<br>
* Parent Carbohydrases<br>
Parent carbohydrases may be defined as all enzymes capable of breaking down carbohydrate chains (e.g. starches) of especially five and six member ring structures (i.e. enzymes classified under the Enzyme Classification number E.C. 3.2 (glycosidases) in accordance with the Recommendations (1992) of the International Union of Biochemistry and Molecular Biology (IUBMB)). Also included in the group of carbohydrases according to the invention are enzymes capable of isomerizi-ng carbohydrates e.g. six member ring structures, such as D-glucose to e.g. five member ring structures like D-fructose.<br>
Examples include carbohydrases selected from those classified under the Enzyme Classification (E.C.) numbers:<br><br>
a-amylase  (3.2.1.1)    p-amylase  (3.2.1.2),  glucan  1,4-a-glucosidase  (3.2.1.3),  cellulase . (3.2.1.4) ,  endo-1,3(4)-p-<br>
glucanase (3.2.1.6), endo-1,4-p-xylanase (3.2.1.8), dextranase (3.2.1.11), chitinase (3.2.1.14), polygalacturonase (3.2.1.15),<br>
lysozyme (3.2.1.17), p-glucosidase (3.2.1.21), a-galactosidase (3.2.1.22), p-galactosidase (3.2.1.23), amylo-1,6-glucosidase (3.2.1.33), xylan 1,4-p-xylosidase (3.2.1.37), glucan endo-1,3-<br>
p-D-glucosidase (3.2.1.39), a-dextrin endo-1,6-glucosidase (3.2.1.41), sucrose a-glucosidase (3.2.1.48) , glucan endo-1,3-<br>
a-glucosidase (3.2.1.59) , glucan 1,4-p-glucosidase (3.2.1.74),<br>
glucan endo-1,6-p-glucosidase (3.2.1.75), arabinan endo-1,5-a-arabinosidase (3.2.1.99), lactase (3.2.1.108), chitonanase (3.2.1.132) and xylose isomerase (5.3.1.5).<br>
'Examples of relevant carbohydrases include a-1,3-glucanases derived from Trichoderma harzianum; a-1,6-glucanases derived from a strain of Paecilomyces; P-glucanases derived from Bacillus subtills; p-glucanases derived from Humicola insolens; p-glucan-ases derived from Aspergillus niger; p-glucanases derived from a strain of Trichoderma; p-glucanases derived from a strain of Oerskovia xanthineolytica; exo-1,4-a-D-glucosidases (glucoamylases) derived from Aspergillus niger; a-amylases derived from Bacillus subtilis; a-amylases derived from Bacillus amyloliquefaciens', a-amylases derived from Bacillus stearother-~mophilus;     a-amylases  derived  from Aspergillus     oryzae;      a-<br>
amylases derived from non-pathogenic microorganisms; a-galactosidases derived from Aspergillus niger; Pentosanases, xylanases, cellobiases, cellulases, hemi-cellulases deriver from Humicola insolens; cellulases derived .from Trichoderma reesei; cellulases derived from non-pathogenic mold; pecti-nases, cellulases, arabinases, hemi-celluloses derived from As-<br><br>
pergillus niger; dextranases derived from Penicillium lilac-inum;   endo-glucanase derived from non-pathogenic mold; pullula-<br>
nases derived from Bacillus acidopullyticus; p-galactosidases derived from Kluyveromyces fragilis? xylanases derived from Trichoderma  reesei;<br>
Specific examples of readily available commercial carbohydrases include Alpha-Gal™, Bio-Feed™ Alpha, Bio-Feed™ Beta, Bio-Feed™ Plus, Bio-Feed™ Plus, Novozyme® 188, Carezyme®, Cellu-clast®, Cellusoft®, Ceremyl®, Citrozym™, Denimax™, Dezyme™, Dextrozyme™, Finizym®, Fungamyl™, Gamanase™, Glucanex®, Lac-, tozym®, Maltogenase™, Pentopan™, Pectinex™, Promozyme®, Pulp-zyme™, Novamyl™, Termamyl®, AMG (Amyloglucosidase Novo), Maltogenase®, Sweetzyme®, Aquazym®, Natalase® (all enzymes available from Novo Nordisk A/S). Other carbohydrases are available from other companies.<br>
It is to be understood that also carbohydrase variants are contemplated as the parent enzyme.<br>
The activity of carbohydrases can be determined as described in "Methods of Enzymatic Analysis", third edition, 1984, Verlag Chemie, Weinheim, vol. 4.<br>
Parent Transferases<br>
Parent transferases (i.e. enzymes classified under the Enzyme Classification number E.C. 2 in accordance with the Recommendations (1992) of the International Union of Biochemistry and Molecular Biology (IUBMB)) include transferases within this group.<br><br>
The parent transferases may be any transferase in the subgroups of transferases: transferases transferring one-carbon groups (E.C. 2.1); transferases transferring aldehyde or residues (E.C 2.2) ; acyltransferases (E.C. 2.3); glucosyltransferases (E.C. 2.4); transferases transferring alkyl or aryl groups, other that methyl groups (E.C. 2.5); transferases transferring nitro-geneous groups (2.6).<br>
In a preferred embodiment the parent transferase is a transglutaminase E.C 2.3.2.13 (Protein-glutamine |j.-glutamyltransferase) .<br>
Transglutaminases are enzymes capable of catalyzing an acyl transfer reaction in which a gamma-carboxyamide group of a pep-tide-bound glutamine residue is the acyl donor. Primary amino groups in a variety of compounds may function as acyl acceptors with the subsequent formation of monosubstituted gamma-amides of peptide-bound glutamic acid. When the epsilon-amino group of a lysine residue in a peptide-chain serves as the acyl acceptor, the transferases form intramolecular or intermolecular gamma-glutamyl-epsilon-lysyl crosslinks.<br>
Examples of transglutaminases are described in the pending DK patent application no. 990/94 (Novo Nordisk A/S).<br>
The parent transglutaminase may be of human, animal (e.g. bovine) or microbial origin.<br>
Examples of such parent transglutaminases are animal derived Transglutaminase, FXIIIa; microbial transglutaminases derived from Physarum polycephalum (Klein.et al., Journal of Bacteriology, Vol. 174, p. 2599-2605); transglutaminases derived from Streptomyces sp., including Streptomyces lavendulae, Streptomyces lydicus (former Streptomyces libani) and Strep-toverticillium   sp., including  Streptoverticillium   mobaraense,<br><br>
Streptoverticillium        cinnamoneum,	and   Streptoverticillium<br>
griseocarneum (Motoki et al., US 5,156,956; Andou et al., US 5,252,469; Kaempfer et al-, Journal of General Microbiology, Vol. 137, p. 1831-1892; Ochi et al., International Journal of Sytematic Bacteriology, Vol-. 44, p. 285-292; Andou et al., US 5,252,4 69; Williams et al., Journal of General Microbiology, Vol. 129, p. 1743-1813).<br>
It is to be understood that also transferase' variants are contemplated as the parent enzyme.<br>
The activity of transglutaminases .can be determined as described in "Methods of Enzymatic Analysis", third edition, 1984, Verlag Chemie, Weinheim, vol. 1-10.<br>
Parent Phytases<br>
Parent phytases are included in the group of enzymes classified under the Enzyme Classification number E.G. 3.1.3 (Phosphoric Monoester Hydrolases) in accordance with the Recommendations (1992) of the International Union of Biochemistry and Molecular Biology (IUBMB)).<br>
Phytases are enzymes produced by microorganisms which catalyse the conversion of phytate to inositol and inorganic phosphorus<br>
Phytase producing microorganisms comprise bacteria such as Bacillus subtilis, Bacillus natto and Pseudomonas; yeasts such as Saccharomyces cerevisiae; and fungi such as Aspergillus niger, Aspergillus ficuum, Aspergillus awamori, Aspergillus oryzae, Aspergillus terreus or Aspergillus nidulans, and various other Aspergillus  species).<br><br>
Examples of parent phytases include phytases selected from those classified under the Enzyme Classification (E.C.) numbers: 3-phytase (3.1.3.8) and 6-phytase (3.1,3.26).<br>
The activity of phytases can be determined as described in "Methods of Enzymatic Analysis", third edition, 1984, Verlag Chemie, Weinheim, vol. 1-10, or may be measured according to the method described in EP-A1-0 420 358, Example 2 A.<br>
Lyases<br>
Suitable lyases include Polysaccharide lyases: Pectate lyases (4.2,2.2) and pectin lyases (4.2.2.10), such as those from Bacillus licheniformis disclosed in WO 99/27083. Another aspect of the invention is a composition comprising at least one protein (polypeptide) or enzyme of the invention. The composition may comprise other polypeptides, proteins or enzymes and/or ingredients normally used in personal care products, such as shampoo, soap bars, skin lotion, skin creme, hair dye, toothpaste, household articles, agro chemicals, personal care products, such as cleaning preparations e.g. for contact lenses, cosmetics, toiletries, oral and dermal pharmaceuticals, compositions used for treating textiles, compositions used for manufacturing food, e.g. baking, and feed etc.<br>
Examples of said proteins(polypeptides)/enzymes include enzymes exhibiting protease, lipase, oxidoreductase, carbohydrase, transferase, such as transglutaminase, phytase and/or antimicrobial polypeptide activity. These enzymes may be present as conjugates with reduced activity.<br>
The protein of the invention may furthermore typically be used in detergent composition. It may be included in the detergent<br><br>
composition in the form of a non-dusting granulate, a stabilized liquid, or a protected enzyme. Non-dusting granulates may be produced, e.g., as disclosed in US 4,106,991 and 4,661,452 (both to Novo Industri A/S) and may optionally be coated by methods known in the art. .Examples of waxy coating materials are poly(ethylene oxide) products (polyethylene glycol, PEG) with mean molecular weights of 1000 to 20000; ethoxylated non-ylphenols having from 16 to 50 ethylene oxide units; ethoxylated fatty alcohols in which the alcohol contains from 12 to 20 carbon atoms and in which there are 15 to 80 ethylene oxide units; fatty alcohols; fatty acids; and mono- and di- and triglycerides of fatty acids. Examples of film-forming coating materials suitable for application by fluid bed techniques are given in patent GB 1483591. Liquid enzyme preparations may, for instance, be stabilized by adding a polyol such as propylene glycol, a sugar or ^sugar alcohol, lactic acid or boric acid according to established methods. Other enzyme stabilizers are well known in the art. Protected enzymes may be prepared according to the method disclosed in EP 238,216.<br>
The detergent composition may be in any convenient form, e.g. as powder, granules, paste or liquid. A liquid detergent may be aqueous, typically containing up to 70% water and 0-30% organic solvent, or non-aqueous.<br>
The detergent composition comprises one or more surfactants, each of which may be anionic, nonionic, cationic, or zwitteri-onic. The detergent will usually contain 0-50% of anionic surfactant such as linear alkylbenzenesulfonate (LAS), alpha-olefinsulfonate (AOS), alkyl sulfate (fatty alcohol sulfate) (AS), alcohol ethoxysulfate (AEOS or AES), secondary alkanesul-fonates (SAS), alpha-sulfo fatty acid methyl esters, alkyl- or . alkenylsuccinic acid, or soap. It may also contain 0-40% of nonionic surfactant such as alcohol ethoxylate  (AEO or AE),<br><br>
carboxylated alcohol ethoxylates, nonylphenol ethoxylate, al-kylpolyglycosicie, alkyldimethylamine oxide, ethoxylated fatty acid monoethanolamide, fatty acid monoethanolamide, or polyhy-droxy alkyl fatty acid amide (e.g. as described in WO 92/06154).<br>
The detergent composition may additionally comprise one or more other enzymes, such as e.g. proteases, amylases, lipases, cutinases, cellulases, peroxidases, oxidases, and further antimicrobial polypeptides.<br>
The detergent may contain 1-65% of a detergent builder or com-plexing agent such as zeolite, diphosphate, triphosphate, phos-phonate, citrate, nitrilotriacetic acid (NTA), ethylene-diaminetetraacetic acid (EDTA), diethylenetriaminepentaacetic acid (DTMPA) , alkyl- or. alkenylsuccinic acid, soluble silicates or layered silicates (e.g. SKS-6 from Hoechst). The detergent may also be unbuilt, i.e. essentially free of detergent builder.<br>
The detergent may comprise one or more polymers. Examples are carboxymethylcellulose (CMC), poly(vinylpyrrolidone) (PVP), polyethyleneglycol (PEG), poly(vinyl alcohol) (PVA), polycarbo-xylates such as polyacrylates, maleic/acrylic acid copolymers and lauryl methacrylate/acrylic acid copolymers.<br>
The detergent may contain a bleaching system which may comprise a H202 source such as perborate or percarbonate- which may be combined with a peracid-forming . bleach activator such as tetraacetylethylenediamine (TAED) or nonanoyloxybenzenesulfon-ate (NOBS). Alternatively, the bleaching system may comprise peroxyacids of, e.g., the amide, imide, or sulfone type.<br><br>
The detergent composition of the invention comprising the polypeptide of the invention may be stabilized using conventional stabilizing agents, e.g. a polyol such as propylene glycol or glycerol, a sugar or sugar alcohol, lactic acid, boric 1 acid, or a boric acid derivative such as, e.g., an aromatic borate ester, and the composition may be formulated as described in, e.g., WO 92/19709 and WO 92/19708.<br>
The detergent may also contain other conventional detergent ingredients such as, e.g., fabric conditioners including clays, foam boosters, suds suppressors, anti-corrosion agents, soil-suspending agents, anti-soil-redeposition -agents, dyes, bactericides, optical brighteners, or perfume.<br>
The pH (measured in aqueous solution at use concentration) will usually be neutral or alkaline, e.g. in the range of 7-11.<br>
Dishwashing composition<br>
Further, a modified enzyme according to the invention may also be used in dishwashing detergents.<br>
Dishwashing detergent compositions comprise a surfactant which may be anionic, non-ionic, cationic, amphoteric or a mixture of these types. The detergent will contain 0-90% of non—ionic surfactant such as low- to non-foaming ethoxylated propoxylated straight-chain alcohols.<br>
The detergent composition may contain detergent builder salts of inorganic and/or organic types. The detergent builders may be subdivided into phosphorus-containing and non-phosphorus-containing types. The detergent composition usually contains 1-90% of detergent builders.<br><br>
Examples of phosphorus-containing inorganic alkaline detergent builders, when present, include the water-soluble salts especially alkali metal pyrophosphates, orthophosphates, and polyphosphates . An example of•phosphorus-containing organic alkaline detergent builder, when present, includes the water-soluble salts of phosphonates. Examples of non-phosphorus-containing inorganic builders, when present, include water-soluble alkali metal carbonates, borates and silicates as well as the various types of water-insoluble crystalline or amorphous alu-mino silicates of which zeolites are the best-known representatives .<br>
Examples of suitable organic builders include the alkali metal, ammonium and substituted ammonium, citrates, succinates, ma-lonates, fatty acid sulphonates, carboxymetoxy succinates, ammonium polyacetates, carboxylates, polycarboxylates, amino-polycarboxylates, polyacetyl carboxylates and polyhydroxsul-phonates.<br>
Other suitable organic builders include the higher molecular weight polymers and co-polymers known to .have builder properties, for example appropriate polyacrylic acid, polymaleic and polyacrylic/polymaleic acid copolymers and their salts.<br>
The dishwashing detergent composition may contain bleaching agents of the chlorine/bromine-type or the oxygen-type. Examples of inorganic chlorine/bromine-type bleaches are lithium, sodium or calcium hypochlorite and hypobromite as well as chlorinated trisodium phosphate. Examples of organic chlorine/bromine-type bleaches are heterocyclic N-bromo and N-chloro imides such as trichloroisocyanuric, tribromoiso- • cyanuric, dibromoisocyanuric and dichloroisocyanuric acids, and<br><br>
salts thereof with water-solubilizing cations such as potassium and sodium. Hydantoin compounds are also suitable.<br>
The oxygen bleaches are preferred, for example in the form of an inorganic persalt, preferably with a bleach precursor or as a peroxy acid compound. Typical examples of suitable peroxy bleach compounds are alkali metal perborates, both tetra-hydrates and monohydrates, alkali metal percarbonates, per-silicates and perphosphates. Preferred activator materials are ' TAED and glycerol triacetate.<br>
The dishwashing detergent composition of the invention may be stabilized using conventional stabilizing agents for the enzyme (s) , e.g. a polyol such as e.g. propylene glycol, a sugar or a sugar alcohol, lactic acid, boric acid, or a boric acid derivative, e.g. an aromatic borate ester.<br>
The dishwashing detergent composition of the invention may also contain other conventional detergent ingredients, e.g. defloc-culant material, filler material, foam depressors, anti-corrosion agents, soil-suspending agents, sequestering agents, anti-soil redeposition agents, dehydrating agents, dyes, bactericides, fluorescers, thickeners and perfumes.<br>
Finally, the enzyme of the invention may be used in conventional dishwashing-detergents, e.g. in any of the detergents de-<br>
»<br>
scribed in any of the following patent publications:<br>
EP 518719, EP 518720, EP 518721, EP 516553, EP 516554,<br>
EP 516555, GB 2200132, DE 3741617, DE 3727911, DE 4212166,<br>
DE 4137470, DE 3833047, WO 93/17089, DE 4205071, WO 52/09680,<br>
WO 93/18129, WO 93/04153, WO 92/06157, WO 92/08777, EP 429124,<br>
WO 93/21299, US 5141664, EP 561452, EP 561446, GB 2234980,<br><br>
WO 93/03129, EP 481547, EP 530870, EP 533239, EP 554943, EP 346137, US 5112518, EP 318204, EP 318279, EP 271155, EP 271156, EP 346136, GB 2228945, CA"2006687, WO 93/25651,<br>
*<br>
EP 530635, EP 414197, US 5240632. Personal care applications .<br>
The protein of the invention is also of interest in connection with personal care applications.<br>
Proteases<br>
Proteases are well-known active ingredients for	cleaning of<br>
contact lenses. They hydrolyse the proteinaceous	soil on the<br>
lens and thereby makes it soluble. Removal of the	protein soil<br>
is essential for the wearing comfort.<br>
Proteases are also effective ingredients in skin cleaning products, where they remove the upper layer of dead keratinaseous skin cells and thereby make the skin look brighter and fresher.<br>
Proteases are also used in oral care products, especially for<br>
* cleaning of dentures, but also in dentifrices.<br>
Further, proteases are used in toiletries, bath and shower products, including shampoos, conditioners, lotions, creams, soap bars, toilet soaps, and liquid soaps.<br>
Lipases<br>
Lipases can be applied for cosmetic use as active ingredients in skin cleaning products and anti-acne products for removal of excessive skin lipids, and in bath and shower products such as creams and lotions as active ingredients for skin care.<br><br>
Lipases can also be used in hair cleaning products (e.g. shampoos) for effective removal of sebum and pther fatty material from the surface of hair.<br>
Lipases are also effective ingredients in products for cleaning of contact lenses, where they remove lipid deposits from the lens surface.<br>
Oxidoreductases<br>
The most common oxidoreductase for personal care purposes is an oxidase (usually glucose oxidase) with substrate (e.g. glucose) that ensures production of H202/ which then will initiate the oxidation of for instance SCN" or I" into antimicrobial reagents (SCNO" or I2) by a peroxidase (usually lactoperoxidase). This enzymatic complex is known in nature from e.g. milk and saliva.<br>
It is being utilised commercially as anti-microbial system in oral care products (mouth rinse, dentifrice,. chewing gum) where it also can be combined with an amyloglucosidase to produce the glucose. These systems are also known in cosmetic products for preservation.<br>
Anti-microbial systems comprising the combination of an oxidase and a peroxidase are know in the cleaning of contact lenses.<br>
Another application of oxidoreductases is oxidative hair dyeing using oxidases, peroxidases and laccases.<br>
Free radicals formed on the surface of the skin (and hair) known to be associated with the ageing process of the skin<br><br>
(spoilage of the hair). The free radicals activate chain reactions that lead to destruction of fatty membranes, collagen, and cells. The application of free radical scavengers such as Superoxide dismutase into cosmetics is well known (R. L. Goldemberg, DCI, Nov. 93, p. 48-52).<br>
Protein disulfide isomerase (PDI) is also an oxidoreductase. It can be utilised for waving of hair (reduction and reoxidation of disulfide bonds in hair) and' repair of spoiled hair (where the damage is mainly reduction of existing disulfide bonds).<br>
Carbohydrases<br>
Plaque formed on the surface of teeth is composed mainly of polysaccharides. They stick to the surface of the teeth and the<br>
microorganisms. The polysaccharides are mainly ct-1,6 bound glucose (dextran) and ct-1,3 bound glucose (mutan). The application of different types of glucanases such as mutanase and dex-tranase helps hydrolysing the sticky matrix of plaque, making it easier to remove by mechanical action.<br>
Also other kinds of biofilm for instance the biofilm formed in lens cases can be removed by the action of glucanases.<br>
Food and Feed<br>
Further conjugated enzymes or polypeptides with reduced immuno-genicity according to the invention may advantageously be used in the manufacturing of food and feed.<br>
Proteases<br><br>
The gluten in wheat flour is the essential ingredient responsible for the ability of flour to be used in baked foodstuffs. Proteolytic enzymes are sometimes needed to modify the gluten phase of the dough, e.g. a hard wheat flour can be softened with a protease.<br>
Neutrase® is a commercially available neutral metallo protease that can be used to ensure a uniform dough quality and bread texture, and to improve flavour. The gluten proteins are degraded either moderately or more extensively to peptides, whereby close control is necessary in order to avoid excessive softening of the dough.<br>
Proteases are also used for modifying milk protein.<br>
To coagulate casein in milk when producing cheese proteases such as rennet or chymosin may be used.<br>
In the brewery industry proteases are used for brewing with un-malted cereals and for controlling the nitrogen content.<br>
In animal feed products proteases are used so to speak to. expand the animals digestion system.<br>
Lipases<br>
The application of lipase in the baking industry is rather new. Addition of lipase results in improved dough properties and an improved breadmaking quality in terms of larger volume, improved crumb structure and whiter crumb colour. The observed effect can be explained by a mechanism where the lipase changes the interaction between gluten and some lipids fragment during dough mixing. This results in an improved gluten network.<br><br>
The flavour development of blue roan cheese (e.g. Danablue) , certain -Italian type cheese, and other dairy products containing butter-fat, are dependent on the degradation of milk fat into free fatty acids. Lipases may be used for developing flavour in such products.<br>
In the oil- and fat producing industry lipases are used e.g. to minimize the amount of undesirable side-products, to modify fats by interesterification, and to synthesis of esters.<br>
Oxidoreductases<br>
Further oxidoreductases with reduced immunogenic!ty according to the invention may advantageously be used in the manufacturing of food and feed.<br>
Several oxidoreductases are used for baking, glucose oxidase, lipoxygenase, peroxidase, catalase and combinations hereof. Traditionally, bakers strengthen gluten by adding ascorbic acid and potassium bromate. Some oxidoreductases can be used to re-place bromate in dough systems by oxidation of free sulfydryl units in gluten proteins. Hereby disulphide linkages are formed resulting in stronger, more elastic doughs with greater resistance.<br>
Gluzyme™ (Novo Nordisk A/S) is a glucose oxidase preparation with catalase activity that can be used to replace bromate. The dough strengthen is measured as greater resistance, to mechanical shock, better oven spring and larger loaf volume.<br>
Carbohydrases<br><br>
Flour has varying content of amylases leading to differences in the baking quality. Addition of amylases can be necessary in order to standardize the flour. Amylases and pentosanases generally provide sugar for the yeast fermentation, improve the bread volume, retard retrogradation, and decrease the staling rate and stickiness that results from pentosan gums. Examples of carbohydrases are given below.<br>
Certain maltogenic amylases can be used for prolonging the shelf life of bread for two or more days without causing gummi-ness in the product. Selectively modifies the gelatinized starch by cleaving from the non-reducing end of the starch molecules, low molecular wight sugars and dextrins. The starch is modified in such a way that retrogradation is less likely to occur. The produced low-molecular^weight sugars improve the baked goods water retention capacity without creating the intermediate-length dextrins that result in gumminess in the finished product. The enzyme is inactivated during bread baking, so it can be considered a processing aid that does not have to be declared on the label. Overdosing of Novamyl can almost be excluded.<br>
The bread volume can be improved by fungal a-amylases which further provide good and uniform structure of the bread crumb.<br>
Said a-amylases are endoenzymes that produce maltose, dextrins<br>
and glucose. Cereal and some bacterial* a-amylases are inactivated at temperatures above the gelatinization temperature of starch, therefore when added to wheat dough it results in a low bread volume and a sticky bread interior. Fungamyl has the ad-i vantage of being thermolabile and is inactivated just below the gelatinization temperature.<br><br>
Enzyme preparations containing a number of pentosanase and hemi-cellulase activities can improve the handling and stability of the dough, and improves the freshness, the crumb structure and the volume of the bread.<br>
By hydrolysing the pentosans fraction in flour, it will lose a great deal of its water-binding capacity, and the water will then be available for starch and gluten. The gluten becomes more pliable and extensible, and the starch gelatinizes more easily. Pentosanases can be used in combination with or as an alternative to emulsifiers.<br>
*<br>
Further  carbohydrases  are user  for  producing  syrups	from<br>
starch, which are widely used in soft drinks,  sweets,	meat<br>
products, dairy products, bread products, ice cream, baby	food,<br>
jam etc.<br>
The conversion of starch is normally carried out three steps.<br>
First the starch is liquefied, by the use of a-amylases. Mal-todextrins, primary consisting of oligosaccharides and dex-trins, are obtained.<br>
The mixture is then treated with an amyloglucosidase for hydrolysing the oligosaccharides and dextrins into glucose. This way a sweeter product is obtained. If high maltose syrups are desired p-amylases alone or in combination with a pullulanase (de-branching enzyme) may be used.<br>
The glucose mixture can be made even sweeter by isomerization to fructose. For this an immobilized glucose isomerase can be used.<br>
In the sugar industry, it is common practice to speed up * the break down of present starch in cane juices. Thereby the starch<br><br>
content in the raw sugar is reduced and filtration at the refinery facilitated.<br>
Furthermore dextranases are used to break down dextran in raw sugar juices and syrups.<br>
In the alcohol industry ct-amylases is advantageously being used for thinning of starch in distilling mashes.<br>
In the brewing industry a-amylases is used for adjunct liquefaction.<br>
In the dairy industry (3-galactosidases (lactase) is used when producing low lactose milk for persons suffering from lactose malabsorption.<br>
When flavoured milk drinks are produced from lactase-treated milk, the addition of sugar can be reduced without reducing the sweetness of the product.<br>
In the production of condensed milk, lactose crystallization can be avoided by lactase treatment, and the risk of thickening caused by casein coagulation in lactose crystals is thus reduced.<br>
When producing ice cream made from lactase-treated milk (or whey) no lactose crystals will be formed and the defect, sandi-ness, will not occur.<br>
Further, xylanases • are known to be used within a number of food/feed industrial applications as described in WO 94/21785 (Novo Nordisk A/S).<br><br>
.-amylases are used in the .animal feed industry to be added to ereal-containing feed to improve the digestibility of starch.<br>
Lnti-microbial polypeptides*<br>
lertain bacteriolytic enzymes may be used e.g. to wash car-:asses in the meat packing industry (see US patent no. &gt;, 354, 681 from Novo Industri A/S)<br>
transferases<br>
transglutaminases with reduced immunogenicity according to the .nvention may advantageously be used in the manufacturing of<br>
rood and feed.<br>
■<br>
Transglutaminases has the ability to crosslinking protein.<br>
This property can be used for gelling of aqueous phases containing proteins. This may be used for when producing of spreads (DK patent application no. 1071/84 from Novo Nordisk &amp;/S).<br>
Transglutaminases are being used for improvement of baking quality of flour e.g. by modifying wheat flour to be used in the preparation of cakes with improved properties, suph as improved taste, dent, mouth-feel and a higher volume (see JP 1-110147).<br>
Further producing paste type food material e.g. used as fat substitution in foods as ice cream, toppings, frozen desserts, mayonnaises and low fat spreads (see WO 93/22930 from Novo Nordisk A/S).<br><br>
Furthermore for preparation of gels for yoghurt, mousses, cheese, puddings, orange juice, from milk and milk-like products, and binding of chopped meat product, improvement of taste and texture of food proteins (see WO 94/21120 and WO 94/21129 from Novo Nordisk A/S) .<br>
Phytases<br>
Phytases of the invention may advantageously be used in the manufacturing of food, such as breakfast cereal, cake, sweets, drinks, bread or soup etc., and animal feed.<br>
Phytases may be used either for exploiting the phosphorus bound in the phytate/phytic acid present in vegetable protein sources or for exploiting the nutritionally important minerals bound in phytic acid complexes.<br>
Microbial phytase may be added to feedstuff of monogastric animals in order to avoid supplementing the feed with inorganic phosphorus (see US patent no- 3,297,548).<br>
Further phytases may be used in soy processing. Soyabean meal may contain high levels of the anti-nutritiona_l factor phytate , which renders this protein source unsuitable for application in baby food and feed for fish, calves and other non-ruminants, since the phytate chelates essential minerals present therein (see EP 0 420 358) .<br>
Also for baking purposes phytases may be used. Bread with better quality can be prepared by baking divided pieces of a dough containing wheat flour etc, and phytase (see JP-0-3076529-A).<br><br>
A high phytase activity as in koji mold are known to be used for producing refined sake (see JP-0-6070749-A).<br>
Textile applications<br>
Proteases<br>
Proteases-are used for degumming and sand washing of silk.<br>
Lipases<br>
Lipases are used for removing fatty matter containing hydro-<br>
■<br>
phobic esters (e.g. triglycerides) during the finishing of textiles (see e.g.   WO 93/13256 from Novo Nordisk A/S) .<br>
Oxidoreductases<br>
In bleach clean up of textiles catalases may serve to remove excess hydrogen peroxide.<br>
Carbohydrases<br>
Cellulolytic enzymes are widely used in the finishing of denim garments in order to provide a localized variation in the colour density of the fabric (Enzyme facilitated "stone wash").<br>
Also cellulolytic enzymes find use in the bio-polishing process. Bio-Polishing is a specific treatment of the yarn surface which improves fabric quality with respect to handle and appearance without loss of fabric wettability. Bio-polishing may<br><br>
be obtained by applying the method described e.g. in WO 93/20278.<br>
During the weaving of textiles, the threads are exposed to considerable mechanical strain. In order to prevent breaking, the threads are usually reinforced by the coating (sizing) with a gelatinous substance (size). The most common sizing agent is starch in native or modified form. A uniform and durable finish can thus be obtained only after removal of the siz*e from the fabric, the so-called desizing. Desizing of fabrics sized with a size containing starch or modified starch is preferably facilitated by use of amylolytic enzymes.<br>
Oral and dermal pharmaceuticals<br>
Proteases<br>
Different combinations of highly purified proteases (e.g. Trypsin and Chymotrypsin) are used in pharmaceuticals to be taken orally, and dermal pharmaceuticals for combating e. g inflammations, edemata and injuries.<br>
Leather production<br>
Transferase<br>
Transglutaminase is known to be used to casein-finishing leather by acting as a hardening agent (see WO 94/13839 from Movo Nordisk).<br><br>
Hard surface cleaning<br>
Cleaning of hard surfaces e.g. in the food industry is often &gt; difficult, as equipment used for producing dairies, meat, sea food products, beverages etc. often have a complicated shape. The use of surfactant compositions in the form gels and foams comprising enzymes have shown to facilitate and improve hard surface cleaning. Enzymes, which advantageously may be added in i such surfactant compositions, are in particular proteases, lipases, amylases and cellulases.<br>
Such hard surface cleaning compositions comprising enzymes may also advantageously be used in the transport sector, for in-■ stance for washing cars and for general vessel wash.<br>
■ Finally the invention relates to the use of the protein of the<br>
invention or a composition of the invention in products comprising polypeptides.<br>
First of all the conjugate or compositions of the invention can advantageously be used for personal care products, such as hair care and hair treatment products. This include products such as shampoo, balsam, hair conditioners, hair waving compositions, hair dyeing compositions, hair tonic, hair liquid, hair cream, shampoo, hair rinse, hair spray.<br>
Further contemplated are oral care products such as dentifrice, oral washes, chewing gum.<br>
Also contemplated are skin care products and cosmetics, such as skin cream, skin milk, cleansing cream, cleansing lotion, cleansing milk, cold cream, cream soap, nourishing essence, skin lotion, milky lotion, calamine lotion, hand cream, powder<br><br>
soap, transparent soap, sun oil, sun screen, shaving foam, shaving cream, baby oil lipstick, lip cream, creamy foundation, face powder, powder eye-shadow, powder, foundation, make-up base, essence powder, whitening powder.<br>
Also for contact lenses hygiene products the conjugate of the invention can be used advantageously. Such products include cleaning and disinfection products for contact lenses.<br>
The use of detergents such as washing powder, soap, soap bars and liquid soap are also contemplated.<br>
Furthermore this invention relates to the method by which the protein variants are being synthesised and expressed in host cells. This is achieved by culturing host cells capable of expressing a polypeptide in a suitable culture medium to obtain<br>
*<br>
expression and secretion of the polypeptide into the medium, followed by isolation of the polypeptide from the culture medium. The host cell may be any cell suitable for the large-scale production of proteins, capable of expressing a protein and being transformed by an expression vector.<br>
The host cell comprises a DNA construct as defined above, optionally the cells may be transformed with an expression vector comprising a DNA construct as defined above. The host cell<br>
is selected from any suitable cell, such as a bacterial cell,<br>
»<br>
a fungal cell, an animal cell, such as an insect cell or a mammalian cell, or a plant cell.<br>
Host Cells<br>
The present invention also relates to recombinant host cells, comprising a nucleic acid sequence of the invention, which are advantageously used in  the  recombinant  production  of  the<br><br>
polypeptides. The term "host cell" encompasses any progeny of a parent cell which is not identical to the parent cell due to mutations that occur during replication.<br>
The cell is preferably transformed with a vector comprising a nucleic acid sequence of the invention followed by integration of the vector into the host chromosome. "Transformation" means introducing a vector comprising a nucleic acid sequence of the present invention into a host cell so that the vector is maintained as a chromosomal integrant or as a self-replicating extra-chromosomal vector. Integration is generally considered to be an advantage as the nucleic acid sequence is more likely to be stably maintained in the cell. Integration of the vector into the host chromosome may occur by homologous or nonhomologous recombination as described above.<br>
• The choice of a host cell will to a large extent depend upon<br>
the gene encoding the polypeptide and its source.<br>
The host cell may be a unicellular microorganism, e.g., a prokaryote, or a non-unicellular microorganism, e.g., a eukaryote. Useful unicellular cells are bacterial cells such as gram positive bacteria including, but not limited to, a Bacillus cell, e.g., Bacillus alkalophilus, Bacillus amyloliquefaciens, Bacillus brevis, Bacillus circulans, Bacillus coagulans, Bacillus lautus, Bacillus lentus, Bacillus licheniformis, Bacillus megaterium, Bacillus stearothermophilus, Bacillus subtilis, and Bacillus thuringiensis; or a Streptomyces cell, e.g., Streptomyces lividans or Streptomyces murinus, or gram negative bacteria such as E. coli and Pseudomonas sp. In a preferred embodiment, the bacterial host cell is a Bacillus lentus, Bacillus licheniformis, Bacillus stearothermophilus or Bacillus subtilis cell.   The transformation of a bacterial host cell may, for<br><br>
instance, be effected by protoplast transformation (see, e.g., Chang and Cohen, 1979, Molecular General Genetics 168:111-115), by using competent cells (see, e.g., Young tand Spizizin, 1961, Journal of Bacteriology 81:823-829, or Dubnar and Davidoff-Abelson, 1971, Journal of. Molecular Biology 56:209-221), by electroporation (see, e.g., Shigekawa and Dower, 1988, Biotechniques 6:742-751), or by conjugation (see, e.g., Koehler and Thorne, 1987, Journal of Bacteriology 169:5771-5278).<br>
The host cell may be an eukaryote, such as a mammalian cell, an insect cell, a plant cell or a fungal cell. Useful mammalian cells include Chinese hamster ovary (CHO) cells, HeLa cells, baby hamster kidney (BHK) cells, COS cells, or any number of other immortalized cell lines available, e.g., from the American Type Culture Collection.<br>
In a preferred embodiment, the host cell is a fungal cell. "Fungi" as used herein includes the phyla Ascomycota, Basidiomycota, Chytridiomycota, and Zygomycota (as defined by Hawksworth et al., In, Ainsworth and Bisby's Dictionary of The Fungi, 8th edition, 1995, CAB International, University Press, Cambridge, UK) as 'well as the Oomycota (as cited in Hawksworth et al., 1995, supra, page 171) and all mitosporic fungi (Hawksworth et al., 1995, supra). Representative groups of Ascomycota include, e.g., Neurospora, Eupenicillium (=Penicillium) , Emericella (=Aspergillus), Eurotium (=Aspergillus), and the true yeasts listed above. Examples of Basidiomycota include mushrooms, rusts, and smuts. Representative groups of Chytridiomycota include, e.g., Allomyces, Blastocladiella, Coelomomyces, and aquatic fungi. Representative groups of Oomycota include, e.g., Saprolegniomycetous aquatic fungi (water molds) such as Achlya. Examples of mitosporic fungi include Aspergillus, Penicillium,<br><br>
Candida/ and Alternaria.  Representative groups of Zygomycota include, e.g., Rhizopus and Mucor.<br>
In a preferred embodiment, the fungal host cell is a yeast ' cell. "Yeast" as used herein includes ascosporogenous yeast (Endomycetales), basidiosporogenous yeast, and yeast belonging to the Fungi Imperfecti (Blastomycetes). The ascosporogenous yeasts are divided into the families Spermophthoraceae and Saccharomycetaceae. The latter is comprised of four subfamilies, Schizosaccharomycoideae (e.g., genus Schizosaccharomyces), Nadsonioideae, Lipomycoideae, and Saccharomycoideae (e.g., genera Pichia, Kluyverornyces and Saccharomyces). The basidiosporogenous yeasts include the genera Leucosporidim, Rhodosporidium, Sporidiobolus, Filobasidium, and Filobasidiella. Yeast belonging to the Fungi Imperfecti are divided into two families, Sporobolomycetaceae (e.g., genera Sorobolomyces and Bullera) and Cryptococcaceae (e.g., genus Candida). Since the classification of yeast may change in the future, for the purposes of this invention, yeast shall be defined as described in Biology and Activities of Yeast (Skinner, F.A., Passmore, S.M., and Davenport, R.R., eds, Soc. App. Bacterid. Symposium Series No. 9,"1980. The biology of yeast and manipulation of yeast genetics are well known in the art (see, e.g., Biochemistry and Genetics of Yeast, Bacil, M., Horecker, B.J., and Stopani, A.O.M., editors, 2nd edition, 1987; The Yeasts, Rose, A.H., and Harrison, J.S., editors, 2nd edition, 1987; and The Molecular Biology of the Yeast Saccharomyces, Strathern et al., editors, 1981). .<br>
In a more preferred embodiment, the yeast host cell is a cell of the species of Candida, Kluyverornyces, Saccharomyces, Schizosaccharomyces, Pichia, or Yarrowia.<br>
In another preferred embodiment, the yeast host cell is a Saccharomyces uvae,  Saccharomyces kluyveri,  Schizosaccharomyces<br><br>
pombe, Saccharomyces uvarum, Kluyveromyces lactis, Hansenula polymorpha, Pichia pastoris, Pichia methanolica, Pichia kluy-veri, Yarrowia lipolytica, Candida sp., Candida utilis, Candida cacaoi, Geotrichum sp., Geotrichum fermentans, preferably Saccharomyces cerevisiae.<br>
In another embodiment, protein variants may be produced in cells selected among other fungi, such as Aspergillus spp, or Humicola lanuginosa or Aspergillus oryzae.<br>
In a preferred embodiment, the fungal' host cell is a filamentous fungal cell. "Filamentous fungi" include all filamentous forms of the subdivision Eumycota and Oomycota (as defined by Hawksworth et al., 1995, supra). The filamentous fungi are characterized by a vegetative mycelium composed of chitin, cellulose, glucan, chitosan, mannan, and other complex<br>
*<br>
polysaccharides. Vegetative growth is by hyphal elongation and carbon catabolism is obligately aerobic. In contrast, vegetative growth by yeasts such as Saccharomyces cerevisiae is by budding of a unicellular thallus and carbon catabolism may be fermentative.<br>
In a more preferred embodiment, the filamentous fungal host cell is a cell of a species of, but not limited to, Acremonium, Aspergillus,   Fusarium,   Humicola,   Mucor,   Myceliophthora,<br>
Neurospora,   Penicillium,   Thielavia,   Tolypocladium,   and<br>
» Trichoderma or a teleomorph or synonym thereof.<br>
Further in a preferred embodiment, the filamentous fungal host cell is an Aspergillus cell.<br>
In yet another preferred embodiment, the filamentous fungal host cell is an Acremonium. cell, a Fusarium cell, a Humicola cell, a Mucor cell, a Myceliophthora cell, a Neurospora cell, a<br><br>
Penicillium cell, a Thielavia cell, a Tolypocladium cell or a Trichoderma cell.<br>
In a more preferred embodiment, the* filamentous fungal host cell is an Aspergillus awamori, Aspergillus foetidus, Aspergillus japonicus, Aspergillus niger or Aspergillus oryzae cell.<br>
In another preferred embodiment, the filamentous fungal host cell is a Fusarium cell of the section Discolor (also known as the s.ection Fusarium) . For example, the filamentous fungal parent cell may be a Fusarium bactridioides, Fusarium cerealis, Fusarium crookwellense, Fusarium culmorum, Fusarium graminearum, Fusarium graminum, Fusarium heterosporum, Fusarium negundi, Fusarium reticulatum, Fusarium roseum, Fusarium sambucinum, Fusarium sarcochroum, Fusarium sulphureum, or Fusarium trichothecioides cell.<br>
In yet another preferred embodiment, the filamentous fungal parent cell is a Fusarium strain of the section Elegans, e.g., Fusarium oxysporum.<br>
Furthermore in another preferred embodiment, the filamentous fungal host cell is a Humicola insolens, a Humicola lanuginosa cell, a Mucor miehei cell, a Myceliophthora thermophilum cell, a Neurospora crassa cell, a Penicillium purpurogenum cell, a Thielavia terrestris cell.<br>
In another preferred embodiment, the Trichoderma cell is a Trichoderma harzianum, Trichoderma koningii, Trichoderma longibrachiatum, Trichoderma reesei or Trichoderma viride cell.<br>
Fungal cells may be transformed by a process involving protoplast formation, transformation of the protoplasts, and regeneration of the cell .wall in a manner known per se. Suitable procedures  for transformation of Aspergillus host<br><br>
cells are described in EP 238 023 and Yelton et al., 1984, Proceedings of the National Academy of Sciences USA 81:1470-1474. A suitable method of transforming Fusarium species is described by Malardier et al., 1989, Gene 78:147-156 or in copending US Serial No. 08/269,449. Yeast may be transformed using the procedures described by Becker and Guarente, In Abelson, J.N. and Simon, M.I., editors, Guide to Yeast Genetics and Molecular Biology, Methods in Enzymology, Volume 194, pp 182-187, Academic Press, Inc., New York; Ito et al., 1983, • Journal of Bacteriology 153:163; and Hinnen et al., 1978, Proceedings of the National Academy of Sciences USA 75:1920. Mammalian cells may be transformed by direct uptake using the calcium phosphate precipitation method of Graham and Van der Eb (1978, Virology 52:546).<br>
In yet another embodiment the host is selected among plant cells, such as potato cells, or the host is selected among animal cells, such as insect cells, for example Spodoptera frugiperda Sf9 and Sf21 ovary cells and High Five Line from Trichoplusia ni eggs, or mammalian cells, such as monkey CV1 and COS cells, murine C127 fibroblasts and chines hamster ovary cells (CHO).<br>
The expression vector may be any vector to be subjected to the procedure of recombinant DNA, followed by succesfull expression in a host organism. When introduced into the cell the vector may function in an autonome fashion. With a replication origin a plasmid can replicate independently of the host cell. Alternatively it can be incorporated into the host cell genome, and be replicated together with the host cell genome.<br>
Another embodiment of the invention relates to the use of a random peptide display package library, such as a phage dis-<br><br>
play system for selecting a protein variant as described above.<br>
Examples<br>
Example 1<br>
Epitope patterns<br>
Screening a random phage display peptide library with IgG antibodies raised- against the parent protein revealed the following epitope patterns.<br>
Sequences marked with(*) may exist in the context of a Cys-Cys bridge.<br>
Bold:    anchor amino acid Italic:     conservative mutations<br>
Table 1:<br>
1) Parent protein laccase Epitope patterns<br><br><br><br><br><br><br><br>
Example 2<br>
Epitope Mapping using Phage Display Libraries and 3D Analysis,<br>
Identification of epitope sequences and epitope patterns. High diversity libraries (1012) of phages expressing random hexa-, nona- or dodecapetides as part of their membrane proteins, were screened for their capacity to bind purified specific rabbit IgG, and purified rat and mouse IgGl and IgE antibodies. The phage libraries were obtained according to prior art (se WO 9215679 hereby incorporated by reference).<br>
The antibodies were raised in the respective animals by subcu-tanuous, intradermal or intratracheal injection of relevant proteins (f.e. proteases, lipases, amylases, oxidoreductases) dissolved in phosphate buffered saline (PBS). The respective antibodies were purified from the serum of immunised animals by affinity chromatography using paramagnetic .immunobeads (Dynal AS) loaded with pig anti-rabbit IgG, mouse anti-rat IgGl or IgE, or rat anti-mouse IgGl or IgE antibodies.<br>
The respective phage libraries were incubated with the IgG, IgGl .and IgE antibody coated beads. Phages expressing-oligopeptides with affinity for rabbit IgG, or rat or mouse IgGl or IgE antibodies, were collected by exposing these paramagnetic beads to a magnetic field. The collected phages were eluted from the immobilised antibodies by mild acid treatment, or by elution with intact enzyme. The isolated phages were amplified as know to the 'specialist. Alternatively, immobolised phages were directly incubated with E.coli for infection. In<br><br>
short, F-factor positive E. coli (f.e XL-1 Blue, JM101, TGI) were infected with M13-derived vector in the presence of a helper-phage (f.e. M13K07), and incubated, typically in 2xYT containing glucose or IPTG, and appropriate antibiotics for selection. Finally, cells were removed by centrifugation. This cycle of events was repeated on the respective cell su-pernatants, minimal 2x and maximal 5x. After selection round 2, 3, 4 and 5, a fraction of the infected E.coli was incubated on selective 2xYT agar- plates, and the specificity of the emerging phages was assessed immunologically. Thus, phages were transferred to a nitrocellulase (NC) membrane. For each plate, 2 NC-replicas were made. One replica was incubated with the selection antibodies, the other replica was incubated with the selection antibodies and the immunogen used to obtain the antibodies as competitor. Those plaques that were absent in the presence of immunogen, were considered specific, and were amplified according to the procedure described above.<br>
The specific phage-clones were isolated from the cell supernatant by centrifugation in the presence of polyethylenglycol. DNA was isolated, the DNA sequence coding for the oligopeptide was amplified by ' PCR, and the DNA sequence was determined, all according to standard procedures. The amino acid sequence of the corresponding oligopeptide was deduced from the DNA sequence.<br>
Thus, a number of peptide sequences with specificity for the protein specific antibodies, described above, were obtained. These sequences were collected in a database, and analysed by sequence alignment to identify epitope patterns. Conservative exhanges (e.g. aspartate for glutamate, lysine for arginine, serine for threonine) were considered as one. This showed that<br>
*<br>
most sequences were specific for the protein the antibodies were raised against. However, several cross-reacting sequences<br><br>
were obtained from phages that went through 2 selection rounds only.  Yet, 22 epitope paterns were identified.<br>
The epitope paterns are shown in table 2 below<br>
Table 2: An extract from the epitope-database.<br><br><br><br>
In this example, the epitope S R S A was found on four different enzymes (laccase, a fungal lipase and two bacterial proteases) . Similarly, the epitope L &gt; G R S S was also found on different parent proteins. This demontrates that proteins of different structure, function and microbial origin may share one or more epitope patterns.<br>
Example 3<br>
Localisation of the respective epitopes and epitope areas on the 3D-structure  of proteins.<br>
Each epitope was assessed on the 3D-structure of the protein of interest, using an appropriate software (f.e. Swiss Pdb Viewer, WebLite Viewer).<br>
In a first step, the identified epitope patterns were fitted with the 3D-structure of the enzymes - A sequence of at least 3 amino acids, defining a specific epitope pattern, was localised on the 3D-structur of the protein. Conservative mutati-" ons (e.g. aspartate for glutamate, lysine for arginine, serine for threonine) were considered as one for those patterns for which phage display had evidenced such exchanges to occur. Among the possible sequences provided by the protein structure, only those were retained where the sequence matched a primary sequence, or where it matched a structural sequence of amino acids, where each amino acid was situated within a di-<br><br>
stance of 5A from the next one. Occasionally, the mobility of the amino acid side chains, as provided by the software programme, had to be taken in to consideration for this criterium to be fullfilled. This step gave typically between 0 and 7 possible epitopes per protein.<br>
Secondly, the remaining anchor amino acids as well as the variable amino acids, i.e. amino acids that were not defining a pattern .but were present in the individual sequences identified by phage library screening, were assessed in the area around the various amino acid sequences localised in step 1. Only amino acids situated within a distance of 5A from the next one were included. This step gave typically between 0 and 3 possible candidates.<br>
Finally, an accessibility criterium was introduced. The crite-rium was that at least half of the anchor amino acids had a surface that was &gt;30% accessible- Typically, 
Thus, a number of epitope sequences were identified and localised on the surface of various proteins. As suggested by sequence alignment, structural analysis confirmed most of the epitopes to be enzyme specific, with only few exceptions. Overall, most of the identified epitopes were at least partially structural. However, some proteins (e.g. amylase) expressed predominantly primary sequence epitopes. Typically, the epitopes were localised in very discrete areas of the enzymes, and different epitopes often shared amino acids (hot-spots) . 
It is common knowledge that amino acids that surround binding sequences can affect binding of a ligand without participating actively in the binding process. Based on this knowledge, areas covered by amino acids with potential steric effects on the<br><br>
epitope-antibody interaction, were defined around the identified epitopes. Practically, all amino acids situated within 5A from the amino acids defining the epitope were included. The accessibility criterium was not included as hidden amino acids can have an effect on-the surrounding structures.<br>
3D-strctures showing the respective epitopes and epitope areas of different enzymes are shown in figure 1.<br>
Example 4<br>
Production, selection, and evaluation of enzyme variants with reduced antigenicity or immunoqenicity.<br>
Hot-spots or epitopes were mutated using techniques know to bhe expert in the field (e.g. site-directed mutagenesis, error-prone PCR) .<br>
In the examples showed below, variants were made by site-directed mutagenesis. Amino acid exchanges giving new epitopes or duplicating existing epitopes according to the infor-nation collected in the epitope-database (See Example 1), were avoided in the mutagenesis process.<br>
Enzyme variants were screened for reduced binding of antibodies raised against the backbone enzyme. This antibody binding *as assessed by established assays (e.g. competitive ELISA, agglutination assay).<br>
Variants with reduced antibody binding capacity were further evaluated in animal studies.<br>
4ice were immunised subcutanuous weekly, for a period of 20 /eeks, with 50 nl 0.9% (wt/vol) NaCl (control group), or 50 |il<br><br>
0.9% (wt/vol) NaCl containing 10 p,g of protein. Blood samples (100 (xl) were collected from the eye one week after every second immunization. Serum was obtained by. blood clothing, and centrifugation.<br>
Specific IgGl and IgE levels were determined using the ELISA specific for mouse or rat IgGl or IgE. Differences between data sets were analysed by using appropriate statistical methods .<br>
A.Site-dLrected mutagenesis of amino acids defining epitopes, with an effect on IgGl and/or IgE responses in mice.<br>
■<br>
Epitope: A172/A169 R170 A194 G193 N261 Pattern: AR&gt;R&gt;A&gt;N Antibody: IgGl + IgE Backbone: Savinase<br>
The variant carried mutation R170F.<br>
In a competitive IgE ELISA, this variant was less effective in competing for anti-savinase antibodies, giving a 15% lower endpoint inhibition as compared to the savinase backbone.<br>
Mouse studies revealed an 80% reduction of the specific IgE levels, as compared to savinase backbone (p
Epitope: S216 E219 Y220<br><br>
Pattern: E Y &gt; M Antibody: IgGl Backbone: Lipoprime<br>
The variant  carried mutation S216W.<br>
In a competitive IgG ELISA, the variant was less effective in competing for Lipolase antibodies, giving a 38% decrease in endpoint inhibition as compared to the enzyme backbone.<br>
Mouse studies revealed a 69% decrease in specific IgGl levels, compared to the lipolase backbone (p
B,Site-directed mutagenesis of epitopes, with examples of epitope duplication, and new epitope formation, respectively,  predicted by the epitope-database.<br>
Epitope: T143 N173 N140 E136 L135 Pattern: S/T N N &gt; E L Antibody: IgGl Backbone: Savinase<br>
The variant  carried mutation E136R.<br>
In a competitive IgG ELISA, the variants was less effective in competing for savinase antibodies, giving a 38% decrease in endpoint inhibition as compared to the savinase backbone.<br><br>
Vlouse studies revealed a dramatic increase in specific IgGl levels, compared to savinase backbone (p
Mutation E136R establishes- an IgGl epitope of the R Y P R/K pattern, previously identified on PD498. Apparently, this new epitope was more antigenic in mice than the existing epitope. The introduction of a savinase unrelated epitope on the savi-lase backbone could explain the* observed discrepancy between :ompetitive ELISA and animal studies.<br>
[n this example, it was found that using information derived exclusively from screening phage libraries with anti-PD4 98 anybodies (to identify the R Y P R/K epitope pattern of Table I) one could predict the outcome of a genetic engineering experiment for Savinase in which the E136R mutation created the 5D498-epitope on the Savinase surface, leading to increased .mmunogenicity of this Savinase variant. This demonstrates :hat the epitope patterns identified may be used to predict ;he effect on immunogenicity of substitutions in proteins that ire different from the parent protein(s) used to identify the epitope pattern.<br><br>
C. Site-directed mutagenesis of amino acids defining epitope areas, with a differential effect on IgGl and IgE antibody levels in mice, and an inhibiting effect on IgG binding, respectively.<br>
Epitope: A172/A169 R170 A194 G193 N261<br>
Pattern:   AR&gt;R&gt;A&gt;N<br>
Antibody: IgGl + IgE<br>
Backbone: Savinase<br>
Epitope area: P131, S132, A133, L135, E136, V139, A151, A152,<br>
S153, G161, SI 62, 1165, S166, Y167, P168, Y171, N173, A174,<br>
A176, Q191, Y192, G195, L196, R247, S259, T260, L262, Y263,<br>
G264.<br>
The variant was different at position Y167 by the mutation<br>
Y167I.<br>
In a competitive IgE ELISA, the variant was less effective in competing for anti-savinase antibodies, giving a 8% lower endpoint inhibition as compared to the its backbone.<br>
yiouse studies revealed .an 75% reduction of the specific IgE levels, as compared to the backbone (p
Epitope: T143 N173 N140 E136 L135<br>
Pattern: S/T N N &gt; E L<br>
\ntibody: IgGl<br>
3ackbone: Savinase<br>
Epitope area: V10A, 1107, A108, Llll, E112, G115, S132, A133,<br>
T134, Q137, A138, V139, S141, A142, S14 4, R145, G14 6, V147,<br>
7149, Y167, P168, Y171, A172, A174, M175, N243, R247.<br><br>
While variant no.  1 was mutated at the epitope position (N140D) , variant no. 2 was mutated at N140 (N140D) , but also at the epitope area position  (A172D)'.<br>
In a competitive IgG ELISA, variant no. 1 was less effective in competing for anti-savinase antibodies, as compared to savinase. This variant revealed a 21% lower endpoint inhibition as compared to the its backbone.<br>
Variant no. 2 resulted in an endpoint inhibition that was 60% lower as compared to savinase, and 40% as compared to variant no, 1.<br>
Example 5<br>
Conjugation of Savinase variant E136K with activated bis-P,EG-1000<br>
4,9 mg of the Savinase variant was incubated in 50 inM Sodium Borate pH 9.5 with 12 mg of N-succinimidyl carbonate activated bis-PEG 1000 in a reaction volume of approximately 2 ml. The reaction was carried out at ambient temperature using magnetic stirring while keeping the pH within the interval 9.0-9.5 by addition of .0.5 M NaOH. The reaction time was 2 hours. The derivatives was purified and reagent excess removed by size exclusion chromatography on a Superdex-75 column (Pharmacia) equilibrated in 50 mM Sodium Borate, 5mM Succinic Acid, 150 mM NaCI, 1 mM CaCl2pH 6.0.<br>
The conjugate was stored at -20°C, in the above described buffer.<br>
Compared to the parent enzyme variant, the protease activity of the conjugate was retained (97% using Dimethyl-casein as substrate at pH 9).<br><br>
Example 6<br>
Competitive ELISA was performed according to established procedures. In short, a 96 well ELISA plate was coated with the parent protein. After proper blocking and washing, the coated antigen was incubated with rabbit anti-enzyme polyclonal antiserum in the presence of various amounts of modified protein (the competitior)-<br>
The amounts of residual rabbit antiserum was detected by pic anti-rabbit immunoglobulin, horshraddish peroxidase labelled.<br>
Epitope:	T143 N173 N140 E136 L135<br>
Pattern:	S/T N N &gt; E L<br>
Antibody:	IgGl<br>
Backbone:	Savinase<br>
Mutation:E136K<br>
Modification:     bis-NHS-PEGlOOO<br>
Competitive ELISA on the unmodified enzyme of the PEG-ylated derivative. The result is shown in figure 2.<br>
The data show that the derivative (60% endpoint inhibition) has reduced capacity to bind enzyme specific imraunoglobulines, as compared to the parent protein (100% endpoint inhibition).<br><br><br>
WE CLAIM<br>
1.	A method of selecting a protein variant having reduced immunogenicity as<br>
compared to a parent protein, comprising the steps of:<br>
screening a random peptide display package library with antibodies raised against any protein of interest, sequencing the amino acid sequence of the antibody binding peptides, or the DNA sequence encoding the antibody binding peptides, identifying epitope patterns by sequence alignment of the reactive peptide sequence, localisation of structural epitope patterns on the 3-dimensional structure of the parent protein, defining an epitope area including amino acids situated within 5 A from the epitope amino acids, changing the localised epitope patterns, or amino acids defining the epitope area of the parent protein by genetical engineering mutations of a DNA sequence encoding the parent protein without impairing functionality of the protein using the emerging epitope database for eliminating amino acid substitutions creating new or duplicating existing epitope patterns, introducing the mutated DNA sequence into a suitable host, culturing said host and expressing the protein variant, and evaluating the immunogenicity of the protein variant using the parent protein as reference.<br>
2.	The method according to claim 1, wherein the protein variant has reduced<br>
allergenicity.<br>
3.	The method according to any of the preceding claims, wherein the protein variants are selected by evaluation of antigenicity and/or functionality prior to the evaluation of allergenicity.<br>
4.	The method according to any of the preceding claims, wherein antibodies raised against the parent protein are used for screening the random peptide display package library.<br><br>
5.	The method according to any of the preceding claims, wherein the epitope patterns of a parent protein are identified by comparison of the sequences of the peptide with the sequence and 3-dimensional structure of the parent protein.<br>
6.	The method according to any of the preceding claims, wherein the epitope area of the parent protein is identified by comparing the sequences of the peptides with the sequence and 3-dimensional structure of the protein.<br>
7.	The method according to any of the preceding claims, wherein the parent protein is an enzyme.<br>
8.	The method according to claim 7, wherein the enzyme is selected from the group consisting of glycosyl hydrolases, carbohydrases, peroxidases, proteases, lipases, phytases, polysaccharide lyases, oxidoreductases, transglutaminases and glycoseisomerases.<br>
9.	The method according to any of the preceding claims, wherein the antibodies are IgG or IgE antibodies.<br><br>
10.	The method according to any of the preceding claims, wherein the peptide display package library is a phage display library.<br>
11.	The method according to any of the preceding claims, wherein the peptides of the peptide display package library are oligopeptides having from 5 to 25 amino acids.<br>
12.	The method according to any of the preceding claims, wherein the epitope area is changed by substituting at least one amino acid of the epitope area.<br>
13.	The method according to any of the claims 1-11, wherein the epitope area is changed by adding or deleting at least one amino acid of the epitope area.<br>
14.	The method according to any of claims 1-11, wherein the epitope pattern is changed by substituting at least one amino acid of the epitope pattern.<br><br>
15.	The method according to any of the claims 1-11, wherein the epitope pattern is changed by adding or deleting at least one amino acid of the epitope pattern.<br>
16.	The method according to any of the preceding claims, wherein amino acids in the epitope area is changed by substituting and/or inserting at least one amino acid by an amino acid which render the substituted and/or inserted amino acid a target for covalent conjugation to an activated molecule.<br>
17.	The method according to claim 16, wherein the amino acid for substitution and/or insertion is selected from the group consisting of K, R, C, D, E.<br>
18.	The method according to claim 16, wherein the molecule for covalent conjugation is selected from the group of activated synthetic or natural polymers.<br>
19.	The method according to claim 18, wherein the activated synthetic polymer is a polyethylene glycol.<br>
20.	The method according to any of the preceding claims, wherein the<br>
immunogenicity is measured by competitive ELISA.<br>
21.	The method according to claim 20, wherein the immunogenicity of the protein variant is below 75%, preferably below 50% of the immunogenicity of the parent protein.<br>
22.	A protein variant having reduced immunogenicity as compared to its parent protein, wherein the amino acid sequence of the protein variant differs from the amino acid sequence of the parent protein with respect to at least one structural epitope area of the parent protein, and the epitope areas are defined on the parent protein structure by being localised less than 5 A from any of the following epitope patterns: R Y P R/K, SGPRAG, PR/KSDPG, DP&gt;RDTG, AR&gt;R&gt;A&gt;N,NN&gt;EL, R/K R F A/S N &gt; E/D, EY&gt;M,P&gt;PAP&gt;S, AKIDP R/K, ADS&gt;GYP, SR S A, L &gt; G R S S, with the exception of Subtilisin 309 with the mutation Y167L<br><br>
*<br>
23.	A composition comprising a protein variant as defined in any of claims 22.<br>
24.	A DNA construct comprising a DNA sequence encoding a protein variant as defined in any of claims 22.<br>
25.	An expression vector comprising a DNA construct according to claim 24.<br>
26.	A host cell which is capable of expressing a polypeptide and comprising a DNA construct as defined in claim 24.<br>
27.	A host cell which is capable of expressing a polypeptide and which is transformed by an expression vector according to claim 25.<br>
28.	A host according to claims 26 or 27, which is a fungal cell, an insect cell, a mammalian cell, or a plant cell.<br>
29.	A method of producing a protein variant having reduced immunogenicity as compared to the parent protein, comprising:<br>
culturing a host according to any of claims 26 to 28 in a suitable culture medium to obtain expression and secretion of the protein into the medium, followed by -isolation of the protein from the culture medium.<br>
30.	A protein variant having reduced immunogenicity as compared to its wild-type<br>
protein, wherein the amino acid sequence of the protein variant differs from the amino<br>
acid sequence of the parent protein with respect to at least one epitope area of the<br>
parent protein, such that the immunogenicity of the protein variant is below 75% of<br>
the immunogenicity of the parent protein, the epitope areas are defined on the wild-<br>
type protein structure by being localised less than 5 A from any of the following<br>
epitope patterns: R Y P R/K, SGPRAG,PR/KSDPG,DP&gt;RDTG,AR&gt;R&gt;<br>
A &gt; N, N N &gt; E L, R/K R F A/S N &gt; E/D, EY&gt;M,P&gt;PAP&gt;S,AKIDP R/K, A<br>
DS&gt;GYP, SRSA,L&gt;GRSS.<br><br>
31.	A method of selecting a protein variant having reduced immunogenicity as<br>
compared to a parent protein, substantially as hereinbefore described with reference to<br>
the accompanying drawings.<br>
32.	A protein variant having reduced immunogenicity as compared to its parent<br>
protein, substantially as hereinbefore described with reference to the accompanying<br>
drawings.<br><br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDU4Mi1jaGUgY2xhaW1zLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-0582-che claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDU4Mi1jaGUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKS1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-0582-che description (complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDU4Mi1jaGUgZHJhd2luZ3MtZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-0582-che drawings-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNTgyLWNoZS1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-582-che-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNTgyLWNoZS1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-582-che-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNTgyLWNoZS1jb3JyZXNwb25kYW5jZSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-582-che-correspondance others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNTgyLWNoZS1jb3JyZXNwb25kYW5jZSBwby5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-582-che-correspondance po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNTgyLWNoZS1kZXNjcmlwdGlvbiBjb21wbGV0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-582-che-description complete.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNTgyLWNoZS1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-582-che-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNTgyLWNoZS1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-582-che-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNTgyLWNoZS1mb3JtIDE5LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-582-che-form 19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNTgyLWNoZS1mb3JtIDI2LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-582-che-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNTgyLWNoZS1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-582-che-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNTgyLWNoZS1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-582-che-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNTgyLWNoZS1wY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-582-che-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="221835-a-communication-system-a-receiver-a-transmitter-and-a-communication-method.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="221837-a-method-for-treating-cellulosic-fibers-to-remove-non-cellulosic-compounds.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>221836</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/582/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>37/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>12-Sep-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>08-Jul-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>26-Apr-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVOZYMES A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>Krogshoejvej 36, DK-2880 Bagsvaerd,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ERNST, Steffen</td>
											<td>Novo Nordisk A/S, Novo Alle, DK-2880 Bagsvaerd,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>ROGGEN, Erwin, Ludo</td>
											<td>Novo Nordisk A/S, Novo Alle, DK-2880 Bagsvaerd,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>OLSEN, Arne, Agerlin</td>
											<td>Novo Nordisk A/S, Novo Alle, DK-2880 Bagsvaerd,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K1/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/DK99/00541</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>1999-10-12</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>PA 1999 01417</td>
									<td>1999-10-04</td>
								    <td>Denmark</td>
								</tr>
								<tr>
									<td>2</td>
									<td>PA 1998 01645</td>
									<td>1998-11-25</td>
								    <td>Denmark</td>
								</tr>
								<tr>
									<td>3</td>
									<td>PA 1998 1402</td>
									<td>1998-10-30</td>
								    <td>Denmark</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/221836-a-method-of-selecting-a-protein-varient-having-reduced-immunogencity-and-to-the-protein-varient by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:12:12 GMT -->
</html>
