<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/222278-phytase-variants by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:42:54 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 222278:PHYTASE VARIANTS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">PHYTASE VARIANTS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Phytase variants, their preparation and uses, which phytase variants, when aligned according to Fig. 1, are amended as compared to a model phytase in at least one of a number of positions. Preferred model phytases are basidiomycete and ascomycete phytases, such as Peniophora phytase and Aspergillus phytases. Preferred phytase variants exhibits amended activity characteristics, such as improved specific activity and/or improved thermostability.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Phytase variants FIELD OF THE INVENTION<br>
This invention relates to variants of phytases, in particular variants of ascomycete phytases and variants of basidiomycete phytases, the corresponding cloned DNA sequences, a method of producing such phytase variants, and the use thereof for a number of industrial applications.<br>
BACKGROUND OF THE INVENTION<br>
Phytic acid or myo-inositol 1,2,3,4,5,6-hexakis dihvdrogen phosphate (or for short myo-inositol hexakisphosphate; is the primary source of inositol and the primary storage form of phosphate in plant seeds. Phytin is a mixed potassium, rr.agnesium and calcium salt of inositol.<br>
The phosphate moieties of phytic acid chelates divalent and trivalent cations such as metal ions, i.a. the nutritionally essential ions of calcium, iron, zinc and magnesium as well as the trace minerals manganese, copper and molybdenum.<br>
Phytic acid and its salts, phytates, are often not metabolized, i.e. neither the phosphorous thereof, nor the chelated metal ions are nutritionally available.<br>
Accordingly , food and feed preparations need to be supplemented with inorganic phosphate and often also the nutritionally essential ions such as iron and calcium, must be supplemented.<br>
Still further, the phytate phosphorus passes through the gastrointestinal tract of such animals and is excreted wrth the manure, resulting in an undesirable phosphate pollution of the-environment  resulting  e.g.  in  eutrophication  of  the  water environment and extensive growth of algae.<br><br>
Phytic acid or phytates, said terms being, unless otherwise indicated, in the present context used synonymously or at random, are degradable by phytases.<br>
The production of phytases by plants as well as by microorganisms has been reported. Amongst the microorganisms, phytase producing bacteria as well as phytase producing fungi are known.<br>
There  are  several  descriptions  of  phytase  producing<br>
filamentous fungi belonging to the fungal phylum of Ascornycota<br>
(ascomycetes). In particular, there are several references to<br>
phytase producing ascomycetes of the Aspergillus genus such as<br>
Aspergillus terreus (Yamada et al., 1986, Agric, Biol. Cnem.<br>
322:1275-1282). Also, the cloning and expression of the phytase<br>
gene from Aspergillus niger var.  awamori has been described<br>
(Piddington et al., 1993, Gene 133:55-62). EP 0420358 describes<br>
the cloning and expression of a phytase of Aspergillus ficuum<br>
(niger).  EP 0684313 describes the cloning and expression of<br>
phytases of the ascomycetes Aspergillus niger, Mycellophthora<br>
thermophila, Aspergillus terreus. Still further, some partial<br>
sequences  of phytases  of Aspergillus  nidulans,  Talaromyces<br>
thermophilus,  Aspergillus  fumigatus  and  another  strain  of<br>
Aspergillus terreus are given.<br>
The cloning and expression of a phytase of Thermomyces lanuginosus is described in WO 97/35017.<br>
There is a current need for phytases of amended properties or characteristics, e.g. phytases of increased thermostaoiliry, altered pH optimum (a high pH optimum being desirable tor in-vitro processing, a low for in-vivo processing in the gastrointestinal tract), and/or of a higher specific activity.<br><br>
SUMMARY OF THE INVENTION<br>
In a first aspect, the invention provides phytase variants, the characteristics of which are amended - as compared to a so-called model phytase.<br>
Any model phytase, which is of a certain similarity to thirteen herein specifically disclosed model phytases, can be made the model of such variants.<br>
In another aspect, the invention relates to a novel phytase derived from Cladorrhinum foecundissimum.<br>
In still another aspect, the invention provides DNA sequences encoding these phytase variants and this phytase, and methods of their production.<br>
Finally, the invention also relates generally to the use of the phytase and the phytase variants for liberating phosphorous from any phytase substrate, in particular inorganic phosphate from phytate or phytic acid.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
In the detailed description of the invention below, reference is made to the drawings, of which<br>
Fig. 1     is  an  alignment  of  thirteen  specific  phytase<br>
sequences (a multiple sequence alignment according to the program PileUp; GapWeight: 3.000; GapLengthWeight: 0.100);<br>
Fig. 2     this figure shows the amino acid and DNA sequence of<br>
a first phytase ("P_involtus-Al") derived from strain CBS 100231 of Paxillus involutus which was deposited on 28.11.97; the expression plasmid pYES 2.0 comprising the full length cDNA sequence was<br><br>
transformed into E. coli strain DSM 11842 which was deposited on 12.11.97 (see WO 98/28409);<br>
Fig. 3     this figure shows the amino acid and DNA sequence of<br>
a second phytase ('P_involtus-A2") derived from strain CBS 100231 of Paxillus involutus which was deposited on 28.11.97; the expression plasmid pYES 2.0 comprising the full length cDNA sequence was transformed into E. coli strain DSM 11843 which was deposited on 12.11.97 (see WO 98/28409);<br>
Fig. 4     this figure shows the amino acid and DNA sequence of<br>
a phytase ("T_pubescens") derived from strain CBS 100232 of Trametes pubescens, which was deposited on 28.11.97; the expression plasmid pYES 2.0 comprising the full length cDNA sequence was transformed into E. coli strain DSM 11844 which was deposited on 12.11.97 (see WO 98/28409);<br>
Fig. 5     this figure shows the amino acid and DNA sequence of<br>
a phytase ("A_pediades") derived from strain CBS 900.96 of Agrocybe pediades deposited on 04.12.96; the expression plasmid pYES 2.0 comprising the full length cDNA sequence was transformed into E. coli strain DSM 11313 which was deposited on 02.12.96 (see WO 98/28409);<br>
Fig, 6     this figure shows the amino acid and DNA sequence of<br>
a phytase ("P_lycii") derived from strain CBS 686.96 of Peniophora lycii which was deposited on 04.12.96; the expression plasmid pYES 2.0 comprising the full<br><br>
length cDNA sequence was transformed into E. coli strain DSM 11312 which was deposited on 02.12.96 (see WO 98/28409);<br>
Fig. 7     this figure equals figure 2 of EP 0684313 and shows<br>
the amino acid and DNA sequence of a phytase ('M_thermophila") derived from strain ATCC 48102 (=ATCC 74340) of Myceliophthora thermophila which was re-deposited on 14.03.97;<br>
Fig. 8     this figure shows the amino acid and DNA sequence of<br>
a phytase ("A-fumigatus") derived from strain ATCC 13073 of Aspergillus fumigatus (see EP 0897985);<br>
Fig. 9     this figure shows the amino acid ("Conphys") and DNA<br>
sequence of an ascomycete consensus phytase (in the present context called "consphyA") (see EP 0897985);<br>
Fig. 10    this figure shows the amino acid and DNA sequence of<br>
a phytase ("A__nidulans") derived from strain DSM 9743 of Aspergillus nidulans (see EP 0897985);<br>
Fig. 11    this figure equals figure 8 of EP 0420358 and shows<br>
the  amino  acid  and  DNA sequence  of  a  phytase ("A_ficuum") derived from Aspergillus ficuum strain NRRL-3135;<br>
Fig. 12    this figure equals figure 1 of EP 0684313 and shows<br>
the  amino acid  and  DNA sequence  of  a  phytase ("A_terreus")  derived from strain CBS 220.95 of Aspergillus terreus;<br><br>
Fig, 13    this figure shows the amino acid and DNA sequence of<br>
a phytase ("T-thermo") derived from strain ATCC 20186 (=ATCC 74338) of Talaromyces thermophilus which was redeposited on 14.03.97 (see EP 0897985);<br>
Fig. 14    this figure equals figure 2 of WO 97/35017 and shows<br>
the amino acid and DNA sequence of a phytase ("T_lanuginosa") derived from strain CBS 58 6.94 of Thermomyces lanuginosus; a plasmid comprising the full length cDNA sequence was transformed into E. coli DH5a (pMWR46) strain B-21527 which was deposited with NRRL on 23.02.96;<br>
Fig. 15    this figure shows the amino acid and DNA sequence of<br>
a phytase ("C_foecundissimum") derived from strain CBS 427.97 of Cladorrhinum foecundissimum which was deposited on 23 January 1997; the expression plasmid pYES 2.0 comprising the full length cDNA sequence was transformed into E. coli strain DSM 12742 which was deposited on 17 March 1999;<br>
Fig. 16    this  figure  shows  an alignment  of the phytase<br>
C__foecundissimum with the model phytase M_thermophila, using the program GAP gcg (Gap Weight 3.000; Length Weight 0.100); and<br>
Fig. 17    shows how the C_foecundissimum phytase can be pasted<br>
onto the alignment of Fig. 1.<br><br>
DETAILED DISCLOSURE OF THE INVENTION<br>
Phytase<br>
In the present context a phytase is an enzyme which catalyzes the hydrolysis of phytate (myo-inositol hexakisphosphate) to (1) myo-inositol and/or (2) mono-, di-, tri-, tetra- and/or penta-phosphates thereof and (3) inorganic phosphate. In the following, for short, the above compounds are sometimes referred to as IP6, I, IPl, IP2, IP3, IP4, IPS and P, respectively. This means that by action of a phytase, IP6 is degraded into P + one or more of the components IPS, IP4, IPS, IP2, IPl and I. Alternatively, myo-inositol carrying in total n phosphate groups attached to positions p, q, r, - . is denoted Ins(p,q,r,..)Pn. For convenience Ins(l,2,3,4,5,6)P6 (phytic acid) is abbreviated PA.<br>
According to the Enzyme nomenclature database ExPASy (a repository of information relative to the nomenclature of enzymes primarily based on the recommendations of the Nomenclature Committee of the International Union of Biochemistry and Molecular Biology (lUBMB) describing each type of characterized enzyme for which an EC (Enzyme Commission) number has been provided), two different types of phytases are known: A so-called 3-phytase (myo-inositol hexaphosphate 3-phosphohydrolase, EC 3.1,3.8) and a so-called 6-phytase (myoinositol hexaphosphate 6-phosphohydrolase, EC 3-1.3.26). The 3-phytase hydrolyses first the ester bond at the D-3-position, whereas the 6-phytase hydrolyzes first the ester bond at the D-6- or L-6-position.<br>
The  expression  "phytase"  or  "polypeptide  or  enzyme<br>
exhibiting phytase activity" is intended to cover any enzyme<br>
) capable of effecting the liberation of inorganic phosphate or<br>
phosphorous from various myo-inositol phosphates. Examples of<br><br>
such myo-inositol phosphates (phytase substrates) are phytic acid and any salt thereof, e.g. sodium phytate or potassium phytate or mixed salts. Also any stereoisomer of the mono-, di-, tri-, tetra- or penta-phosphates of myo-inositol might serve as a phytase substrate, A preferred phytase substrate is phytic acid and salts thereof.<br>
In accordance with the above definition, the phytase activity can be determined using any assay in which one of these substrates is used. In the present context (unless otherwise specified) the phytase activity is determined in the unit of FYT, one FYT being the amount of enzyme that liberates 1 |imol inorganic ortho-phosphate per min. under the following conditions: pH 5.5; temperature 31°C; substrate; sodium phytate (C6H6O24P6Na12) in a concentration of 0.0050 mol/1. A suitable phytase assay is described in the experimental part.<br>
The present invention provides a genetically engineered phytase as described in the appending claims,<br>
A genetically engineered phytase is a non-naturally occuring phytase which is different from a model phytase, e.g. a wild-type phytase. Genetically engineered phytases include, but are not limited to, phytases prepared by site-directed mutagenesis, gene shuffling, random mutagenesis etc.<br>
The invention also provides DNA constructs, vectors, host cells, and methods of producing these genetically engineered phytases and phytase variants, as well as uses thereof.<br>
A phytase variant is a polypeptide or enzyme or a fragment thereof which exhibits phytase activity and which is amended as compared to a model phytase.<br>
Amended means altered by way of one or more amino acid or peptide substitutions, deletions, insertions and/or additions -in each  case by,  or of,  one  or more amino acids.  Such<br><br>
substitutions, deletions, insertions, additions can be achieved by any method known in the art, e.g. gene shuffling, random mutagenesis, site-directed mutagenesis etc.<br>
The model or parent phytase, from which the phytase variant is derived, can be any phytase, e.g. a wild-type phytase or a derivative, mutant or variant thereof, including allelic and species variants, as well as genetically engineered variants thereof, which e.g. can be prepared by site-directed mutagenesis, random mutagenesis, shuffling etc.<br>
Included in the concept of model phytase is also any hybrid or chimeric phytase, i.e. a phytase which comprises a combination of partial amino acid sequences derived from at least two phytases.<br>
The hybrid phytase may comprise a combination of partial amino acid sequences deriving from at least two ascomycete phytases, at least two basidiomycete phytases or from at least one ascomycete and at least one basidiomycete phytase. These ascomycete and basidiomycete phytases from which a partial amino acid sequence derives may, e.g., be any of those specific phytases referred to herein.<br>
In the present context, a hybrid, shuffled, random mutagenised, site-directed mutagenised or otherwise genetically engineered phytase derived from ascomycete phytases only is also an ascomycete phytase; and a hybrid, shuffled, random mutagenised, site-directed mutagenised or otherwise genetically engineered phytase derived from model basidiomycete phytases only is also a basidiomycete phytase. Any hybrid derived from at least one ascomycete phytase  as well as at least one basidiomycete phytase is called a mixed ascomycete/basidiomycete I phytase and such phytase is also a model phytase in the present context.<br><br>
Analogously, a hybrid, shuffled, random mutagenised, site-directed mutagenised or otherwise genetically engineered phytase derived from one or more Aspergillus phytases is also an Aspergillus derived phytase; and a hybrid, shuffled, random mutagenised, site-directed mutagenised or otherwise genetically engineered phytase derived from any other taxonomic sub-grouping mentioned herein is also to be designated a phytase derived from this taxonomic sub-grouping.<br>
Still further, in the present context, "derived from" is intended to indicate a phytase produced or producible by a strain of the organism in question, but also a phytase encoded by a DNA sequence isolated from such strain and produced in a host organism transformed with said DNA sequence. Finally, the term is intended to indicate a phytase which is encoded by a DNA sequence of synthetic and/or cDNA origin and which has the identifying characteristics of the phytase in question.<br>
Preferably the model phytase is a phytase which can be aligned as described below to either of the thirteen phytases of Fig, 1 (which are particularly preferred model phytases) .<br>
Preferred wild-type model phytases (i.e. neither recombinant, or shuffled or otherwise genetically engineered phytases) have a degree of similarity or homology, preferably identity, to amino acid sequence no. 38-403 (Peniophora numbers) of either of these thirteen phytases of at least 40%, more preferably at least 50%, still more preferably at least 60%, in particular at least 70%, especially at least 80%, and in a most preferred embodiment a degree of similarity of at least 90%.<br>
Preferred recombinant or shuffled or otherwise genetically engineered model phytases have a degree of similarity or homology, preferably identity, to partial sequence no. 38-49, 63-77,  274-291,  281-300 and 389-403  (Peniophora numbers)  of<br><br>
either of these thirteen phytases of at least 60%, more preferably at least 70%, still more preferably at least 80%, in particular at least 90%.<br>
In a preferred embodiment the degree of similarity is based on a comparison with the complete amino acid sequence of either of the thirteen phytases.<br>
The degree of similarity or homology, alternatively identity, can be determined using any alignment programme known in the art. A preferred alignment programme is GAP provided in the GCG version 8 program package (Program Manual for the Wisconsin Package, Version 8, August 1994, Genetics Computer Group, 575 Science Drive, Madison, Wisconsin, USA 53711) (see also Needleman, S,B, and Wunsch, CD., (1970), Journal of Molecular Biology, 48, 443-453). Using GAP with the following settings for polypeptide sequence comparison: GAP weight of 3.000 and GAP lengthweight of 0.100.<br>
Also preferred is a wild-type model phytase which comprises an amino acid sequence encoded by a DNA sequence which hybridizes to a DNA sequence encoding amino acid sequence 38-403 (Peniophora numbers) of any of the DNA sequences encoding the thirteen specific phytase sequences of Fig. 1.<br>
A further preferred model phytase is a genetically engineered phytase, which comprises an amino acid sequence encoded by a DNA sequence which hybridizes to a DNA sequence encoding amino acid sequence 38-49, and to a DNA sequence encoding amino acid sequence 63-77, and to a DNA sequence encoding amino acid sequence 274-291, and to a DNA sequence encoding amino acid sequence 281-300, and to a DNA sequence encoding amino acid sequence 389-403 (Peniophora numbers) of any of the DNA sequences encoding the thirteen specific phytase sequences of Fig. 1.<br><br>
In a preferred embodiment the hybridization is to the complete phytase encoding part of any of the thirteen phytases.<br>
Suitable experimental conditions for determining whether a given DNA or RNA sequence "hybridizes" to a specified nucleotide or oligonucleotide probe involves presoaking of the filter containing the DNA fragments or RNA to examine for hybridization in 5 X SSC (Sodium chloride/Sodium citrate), (J. Sambrook, E.F. Fritsch, and T. Maniatis, 1989, Molecular Cloning, A Laboratory Manual, 2d edition. Cold Spring Harbor, New York) for 10 min, and prehybridization of the filter in a solution of 5 x SSC, 5 x Denhardt's solution (Sambrook et al. 1989) , 0.5 % SDS and 100 Ijg/ml of denatured sonicated salmon sperm DNA (Sambrook et al. 1989), followed by hybridization in the same solution containing a concentration of 10 ng/ml of a random-primed (Feinberg, A. P, and Vogelstein, B. (1983) Anal. Biochem. 132:6-13), 32P-dCTP-labeled (specific activity &gt; 1 x 109 cpm/pg) probe for 12 hours at approximately 45°C.<br>
The filter is then washed twice for 30 minutes in 2 x SSC, 0,5 % SDS at at least 55°C (low stringency), at at least 60°C (medium stringency), at at least 65°C (medium/high stringency), at at least VCC (high stringency) , or at at least 75°C (very high stringency).<br>
Molecules to which the oligonucleotide probe hybridizes under these conditions are detected using an x-ray film.<br>
It should be noted that a certain specific phytase variant need not actually have been prepared from a specific model phytase, for this model phytase to qualify as a "model phytase" in the present context. It is sufficient that the variant exhibits at least one of the herein indicated amendments when it is afterwards compared with the model phytase.<br><br>
The alignment of Fig 1 is made using the program PileUp (Program Manual for the Wisconsin Package, Version 8, August 1994, Genetics Computer Group, 575 Science Drive, Madison, Wisconsin, USA 53711), with a GapWeight of 3.000 and a GapLengthWeight of 0.100. When aligning a new model phytase or a new phytase variant all thirteen sequences can be included together with the new phytase (variant) in a multiple alignment, or, alternatively, at least one of the thirteen sequences of Fig. 1 is included together with the new phytase (variant) in an alignment.<br>
A preferred procedure for aligning according to Fig. 1 a new model phytase (or a phytase variant) is as follows: The new model phytase is aligned with that specific sequence of the thirteen sequences of Fig. 1 to which the new model phytase has the highest degree of homology. For calculating the degree of homology, and for making the "alignment according to Fig. 1" of the two sequences, the program GAP referred to below is preferably used. Having aligned the two sequences, the new model phytase (or phytase variant) is added (pasted) to the alignment at Fig. 1 using the result of the first alignment (placing identical and homologous amino acid residues above each other as prescribed by the alignment), following which corresponding positions are now easily identifiable.<br>
Example 7 shows an example of how to add a new model phytase to the alignment of Fig. 1 and deduce corresponding phytase variants thereof.<br>
Other model phytases can be aligned and variants deduced in analogy with Example 7. This is so in particular for the following model phytases: The phytase of Aspergillus niger var. awamori (US patent no. 5,830,733); the Bacillus phytase of WO 98/06858; the soy bean phytase of WO 98/20139; the maize<br><br>
phytase of WO 98/05785; the Aspergillus phytase of WO 97/38096; the phytases of Monascus anka of WO 98/13480; the phytase from Schwanniomyces occidentalis of EP 0699762 etc.<br>
When comparing a model phytase and a proposed phytase variant using the alignment as described herein, corresponding amino acid positions can be identified, viz. a model position of the model phytase and a variant position of the variant - the corresponding model position and variant position are simply placed one above the other in the alignment. An amendment is said to have occurred in a given position if the model amino acid of the model position and the variant amino acid of the variant position are different. Preferred amendments of these positions manifest themselves as amino acid substitutions, deletions or additions.<br>
Amended in at least one position means amended in one or more positions, i.e. in one, two, three, four, five, six, seven, eight, nine, ten, eleven, twelve etc. up to all N positions listed. This definition includes any possible sub-combinations thereof, e.g. any set of two substitutions, any set of three, any set of four, etc.  - to any set of (N-l) positions.<br>
In the present context all sequences, whatever the model phytase, and including the thirteen sequences of Fig. 1, are numbered using the numbering corresponding to the phytase P_lycii. These "Peniophora numbers" are indicated at Fig. 1, together with the "alignment numbers." The numbering of P_lycii starts at Ml and ends at E439.<br>
As explained above, the alignment reveals which positions in various phytase sequences other than P_lycii are equivalent or corresponding to the given P. lycii position.<br>
A substitution of amino acids is indicated herein as for instance "3S," which indicates, that at position 3 amino acid S<br><br>
should be substituted for the "original" or model position 3 amino acid, whichever it is. Thus, the substitution should result in an S in the corresponding variant position. Considering now the alignment at Fig. 1, a substitution like e-g. "3S" is to be interpreted as follows, for the respective phytases shown (the amino acid first indicated is the "original" or model amino acid in "Peniophora position" 3):<br>
P__involtus__Al:	F3S (number 3 F substituted by S)<br>
P_involtus__A2:	L3S<br>
T_pubescens:	MIS<br>
A_pediades:	MIS<br>
P_lycii:	redundant (already an S)<br>
A_fumigatus:	T5S<br>
consphyA:	V5S<br>
A_nidulans:	T5S<br>
A_ficuum_NRRL3135:     ASS<br>
A_terreus:	ASS<br>
T_thermo:	L5S<br>
T_lanuginosa:	VllS<br>
M_thermophila:	GSS<br>
However, in what follows the above specific substitutions<br>
will be designated	as follows (always using the Peniophora<br>
numbering):<br>
P_involtus_Al:	F3S<br>
P_involtus_A2:	L3S<br>
T_pubescens:	M3S<br>
A__pediades:	M3S<br>
P_lycii:	redundant (already an S)<br>
A_fumigatus:	T3S<br>
consphyA:	V3S<br><br>
A_nidulans:	T3S<br>
A_ficuum_NRRL3135:	ASS<br>
A_terreus:	ASS<br>
T_thermo:	LSS<br>
T_lanuginosa:	VSS<br>
M_thermophila:	GSS<br>
Still further, denotations like e.g. "3S,F,G" means that the amino acid in position S (Peniophora numbers) of the model phytase in question is substituted with either of S, F or G, i.e. e.g. the designation "3S,F,G" is considered fully equivalent to the designation 'SS, SF^ 3G".<br>
A denotation like {)3S means that amino acid S is added to the sequence in question (at a gap in the actual sequence), in a position corresponding to Peniophora number S - and vice versa for deletions (SS()).<br>
In case of regions in which the Peniophora phytase sequence has larger deletions than some of the other phytases in Fig. 1, for instance in the region between position 201 and 202 (Peniophora numbers), intermediate positions (amino acid residues in other sequences) are numbered by adding a,b,c,d, etc, in lower-case letters, to the last Peniophora position number, e.g. for the phytase M_thermophila: E201; G201a; P201b; Y201c; S201d; T201e; I201f; G202; D203 etc.<br>
In one of the priority applications of the present application there are two minor position numbering errors: According to the above definitions, the positions referred to in the first priority application as 204 and 205 (Peniophora numbers) are wrongly designated; they should have been numbered 203a and 204, respectively. Therefore, 204 has been substituted by 203a and 205 by 204 throughout the present application.<br><br>
A preferred phytase variant of the invention comprises an amino acid sequence which comprises, preferably contains, one or more of the following amino acid substitutions:<br>
24C; 27P; 31Y; 33C; 39H,S,Q; 40L,N; 42S,G; 43A,C,D,E,F,G,H,I,K,L,M,N,P,Q,R,S,T,V,W,Y; 44N; 45D,S; 47Y, F; 49P; 51E,A,R; 56P; 58D,K,A; 59G; 61R; 62V,I; 69Q; 75W,F; 78D,S; 79G; 80K,A; 81A,G,Q,E; 82T; 83A,I,K,R,Q; 84I,Y,Q,V; 881; 90R,A; 102Y; 115N; 116S; 118V,L; 119E; 120L; 122A; 123N,Q,T; 125M,S; 126H,S,V; 127Q,E,N; 128A,S,T; 132F,I,L/ 143N; 148V,I; 151A,S; 152G; 153D,Y; 154D,Q,S,G; 157V; 158D,A; 159T; 160A,S; 161T,N; 162N; 163W; 170fH; 170gA; 171N; 172P; 173Q,S; 184Q,S,P; 185S; 186A,E,P; 187A; 187aS; 190A,P; 193S; 194S,T; 195T,V,L; 198A,N,V; 200G,V; 201D,E; a deletion of at least one of 201a, 201b, 201c, 201d, 201e, 201f, preferably all; 201eT; 202S,A; 203R,K,S; 203aV,T; 204Q,E,S,A,V; 205E; 211L,V; 215A,P; 220L,N; 223H,D; 228N; 232T; 233E; 235Y,L,T; 236Y,N; 237F; 238L,M; 242P,S; 244D; 246V; 251eE,Q; 253P; 256D; 260A,H; 264R,I; 265A,Q; 267D; 270Y,A,L,G; 271D,N; 273D,K; 275F,Y; 278T,H; 280A,P; 283P; 287A,T; 288L,I,F; 292F,Y; 293A,V; 302R,H; 304P,A; 332F; 336S; 337T,G,Q,S; 3381; 339V,I; 340P,A; 343A,S,F, I, L; 348Y; 349P; 352K; 360R; 362P; 364W,F; 365V,L,A,S; 366D,S,V; 367A,K; 368K; 3691,L; 370V; 373A,S; 374S,A; 375H; 376M; 383k:Q,E; 387P; 393V; 396R; 404A,G; 409R; 411K,T; 412R; 417E,R; 421F,Y; 431E.<br>
In a preferred embodiment this is with the proviso that the model phytase does not already comprise the above suggested amino acid substitution or addition or deletion at the position indicated. Or, with the proviso that, for each position, the model amino acid is not already the variant amino acid hereby proposed. But these provisos can be said to be in fact already inherent in the above wording, because of the expression "amended."<br><br>
The various preferred phytase variants of claims 16-34 comprises, preferably contains or have, amino acid sequences which comprise or contain one or more of the amino acid substitutions, additions, or deletions listed in the respective claims.<br>
In a preferred embodiment the various phytase variants comprise 1, 2, 3, 4, 5, 6, 7, 8, 9 or even 10 of these substitutions; or a number of substitutions of 10-15, 15-20, 20-30 or even 30-50; eventually up to 60, 70, 80 or 90 substitutions.<br>
In another preferred embodiment, the amino acid sequence of the various phytase variants comprise one or more substitutions of the substitution sub-groupings listed hereinbelow; or combinations of substitutions classified in two or more sub-groupings.<br>
Generally, instead of "comprise," "contain" or "have," the amino acid sequences of preferred variants "consist essentially of" or "consist of" the specific model phytases of fig. 1, as modified by one or more of the substitutions described herein.<br>
In the present context a basidiomycete means a microorganism of the phylum Basidiomycota. This phylum of Basidiomycota is comprised in the fungal kingdom together with e.g. the phylum Ascomycota ("ascomycetes").<br>
Taxonomical questions can be clarified by consulting the references listed below or by consulting a fungal taxonomy database (NIH Data Base (Entrez)) which is available via the Internet on World Wide Web at the following address: http://www3.ncbi,nlm.nih•gov/Taxonomy/tax.html.<br>
For a definition of basidiomycetes, reference is made to either Jtilich, 1981, Higher Taxa of Basidiomycetes; Ainsworth &amp; Bisby's (eds.) Dictionary of the Fungi, 1995, Hawksworth, D-L.,<br><br>
P.M. Kirk, B.C. Sutton &amp; D.N. Pegler; or Hansen &amp; Knudsen (Eds.), Nordic Macromycetes, vol. 2 (1992) and 3 (1997). A preferred reference is Hansen &amp; Knudsen.<br>
For a definition of ascomycetes, reference is made to either of Ainsworth &amp; Brisby cited above or Systema Ascomycetum by Eriksson, O.E. &amp; D. L. Hawksworth, Vol. 16, 1998. A preferred reference is Eriksson et al.<br>
Generally, a microorganism which is classified as a basidiomycete/ascomycete in either of the references listed above, including the database, is a basidiomycete/ascomycete in the present context.<br>
Some Aspergillus strains are difficult to classify because they are anamorphous, and therefore they might be classified in Fungi Imperfecti. However, once the teleomorphous counterpart is found, it is re-classified taxonomically. For instance, the teleomorph of A. nidulans is Emericella nidulans (of the family Trichocomaceae, the order Eurotiales, the class Plectomycetes of the phylum Ascomycota). These subgroupings of Ascomycota are preferred, together with the family Lasiosphaeriaceae, the order Sordariales, the class Pyrenomycetes of the phylum Ascomycota.<br>
The wording "ascomycetes" and analogues as used herein includes any strains of Aspergillus, Thermomyces, Myceliophthora, Talaromyces which are anamorphous and thus would be classified in Fungi Imperfecti,<br>
Preferred basidiomycete phytases are those listed in WO 98/28409, in the very beginning of the section headed "Detailed description of the invention",<br>
DNA sequences encoding the thirteen specifically listed model phytases and other model phytases can be prepared according to the teachings of each of the documents listed under the brief description of the drawings.<br><br>
A DNA sequence encoding a model phytase may be isolated from any cell or microorganism producing the phytase in question, using various methods well known in the art. First, a genomic DNA and/or cDNA library should be constructed using chromosomal DNA or messenger RNA from the organism that produces the phytase. Then, if the amino acid sequence of the phytase is known, homologous, labelled oligonucleotide probes may be synthesized and used to identify phytase-encoding clones from a genomic library prepared from the organism in question. Alternatively, a labelled oligonucleotide probe containing sequences homologous to a known phytase gene could be used as a probe to identify phytase-encoding clones, using hybridization and washing conditions of lower stringency.<br>
Yet another method for identifying phytaseencoding clones would involve inserting fragments of genomic DNA into an expression vector, such as a plasmid, transforming phytase-negative bacteria with the resulting genomic DNA library, and then plating the transformed bacteria onto agar containing a substrate for phytase thereby allowing clones expressing the phytase to be identified.<br>
Alternatively, the DNA sequence encoding the enzyme may be prepared synthetically by established standard methods, e.g. the phosphoroamidite method described by S.L. Beaucage and M.H. Caruthers (1981) or the method described by Matthes et al. (1984) . In the phosphoroamidite method, oligonucleotides are synthesized, e.g. in an automatic DNA synthesizer, purified, annealed, ligated and cloned in appropriate vectors.<br>
Finally, the DNA sequence may be of mixed genomic and synthetic origin, mixed synthetic and cDNA origin or mixed genomic and cDNA origin, prepared by ligating fragments of synthetic, genomic  or  cDNA  origin  (as  appropriate,  the  fragments<br><br>
corresponding to various parts of the entire DNA sequence), in accordance with standard techniques. The DNA sequence may also be prepared by polymerase chain reaction (PCR) using specific primers, for instance as described in US 4,683,202 or R.K. Saiki et al. (1988).<br>
DNA encoding the phytase variants of the present invention can be prepared by methods known in the art, such as Site-directed Mutagenesis. Once a DNA sequence encoding a model phytase of interest has been isolated, and desirable sites for mutation identified, mutations may be introduced using synthetic oligonucleotides. These oligonucleotides contain nucleotide sequences flanking the desired mutation sites; mutant nucleotides are inserted during oligonucleotide synthesis. In a specific method, a single-stranded gap of DNA, bridging the phytase-encoding sequence, is created in a vector carrying the phytase-encoding gene. Then the synthetic nucleotide, bearing the desired mutation, is annealed to a homologous portion of the single-stranded DNA. The remaining gap is then filled in with DNA polymerase I (Klenow fragment) and the construct is ligated using T4 ligase. A specific example of this method is described in Morinaga et al. (1984). US 4,7 60,025 discloses the introduction of oligonucleotides encoding multiple mutations by performing minor alterations of the cassette. However, an even greater variety of mutations can be introduced at any one time by the Morinaga method because a multitude of oligonucleotides, of various lengths, can be introduced.<br>
Another method of introducing mutations into DNA sequences encoding a desired model phytase is described in Nelson and Long (198 9). It involves a 3-step generation of a PCR fragment containing the desired mutation introduced by using a chemically synthesized DNA strand as one of the primers in the PCR<br><br>
reactions. From the PCR-generated fragment, a DNA fragment carrying the mutation may be isolated by cleavage with restriction endonucleases and reinserted into an expression plasmid.<br>
Yet another method of mutating DNA sequences encoding a model phytase is Random Mutagenesis. Random mutagenesis is suitably performed either as localised or region-specific random mutagenesis in at least three parts of the gene translating to the amino acid sequence shown in question, or within the whole gene.<br>
The random mutagenesis of a DNA sequence encoding a model phytase may be conveniently performed by use of any method known in the art.<br>
In relation to the above, further aspects of the present invention relates to a method for generating a variant of a model phytase, wherein the variant preferably exhibits amended characteristics as described below, the method comprising:<br>
(a)	subjecting a DNA sequence encoding the model phytase<br>
to Site-directed Mutagenesis, or the Nelson and Long PCR<br>
mutagenesis method or to Random Mutagenesis,<br>
(b)	expressing the mutated DNA sequence obtained in step (a) in a host cell, and<br>
(c)	screening for host cells expressing a phytase variant which has an altered property relative to the model phytase.<br>
When using Random Mutagenesis, step (a) of the above method of the invention is preferably performed using doped primers.<br>
For instance, the random mutagenesis may be performed by use of a suitable physical or chemical mutagenizing agent, by use of a suitable oligonucleotide, or by subjecting the DNA<br><br>
sequence to PCR generated mutagenesis. Furthermore, the random mutagenesis may be performed by use of any combination of these mutagenizing agents. The mutagenizing agent may, e.g., be one which induces transitions, transversions, inversions, scrambling, deletions, and/or insertions.<br>
Examples of a physical or chemical mutagenizing agent suitable for the present purpose include ultraviolet (UV) irradiation, hydroxylamine, N-methyl-N*-nitro-N-nitrosoguanidine (MNNG), 0-methyl hydroxylamine, nitrous acid, ethyl methane sulphonate (EMS), sodium bisulphite, formic acid, and nucleotide analogues. When such agents are used, the mutagenesis is typically performed by incubating the DNA sequence encoding the parent enzyme to be mutagenized in the presence of the mutagenizing agent of choice under suitable conditions for the mutagenesis to take place, and selecting for mutated DNA having the desired properties.<br>
When the mutagenesis is performed by the use of an oligonucleotide, the oligonucleotide may be doped or spiked with the three non-parent nucleotides during the synthesis of the oligonucleotide at the positions which are to be changed. The doping or spiking may be done so that codons for unwanted amino acids are avoided. The doped or spiked oligonucleotide can be incorporated into the DNA encoding the phytase enzyme by any published technique, using e.g. PCR, LCR or any DNA polymerase and ligase as deemed appropriate.<br>
Preferably, the doping is carried out using "constant random doping", in which the percentage of wild-type and mutation in each position is predefined. Furthermore, the doping may be directed toward a preference for the introduction ) of certain nucleotides, and thereby a preference for the introduction of one or more specific amino acid residues.  The<br><br>
doping may be made, e.g., so as to allow for the introduction of 90% wild type and 10% mutations in each position. An additional consideration in the choice of a doping scheme is based on genetic as well as protein-structural constraints. The doping scheme may be made by using the DOPE program which, inter alia, ensures that introduction of stop codons is avoided.<br>
When PCR-generated mutagenesis is used, either a chemically treated or non-treated gene encoding a model phytase is subjected to PCR under conditions that increase the mis-incorporation of nucleotides (Deshler 1992; Leung et al., Technique, Vol.1, 1989, pp. 11-15).<br>
A mutator strain of E. coli (Fowler et al., Molec. Gen. Genet., 133, 1974, pp. 179-191), S. cereviseae or any other microbial organism may be used for the random mutagenesis of the DNA encoding the model phytase by, e.g., transforming a plasmid containing the parent glycosylase into the mutator strain, growing the mutator strain with the plasmid and isolating the mutated plasmid from the mutator strain. The mutated plasmid may be subsequently transformed into the expression organism.<br>
The DNA sequence to be mutagenized may be conveniently present in a genomic or cDNA library prepared from an organism expressing the model phytase. Alternatively, the DNA sequence may be present on a suitable vector such as a plasmid or a bacteriophage, which as such may be incubated with or otherwise exposed to the mutagenising agent. The DNA to be mutagenized may also be present in a host cell either by being integrated in the genome of said cell or by being present on a vector harboured in the cell. Finally, the DNA to be mutagenized may be in isolated form. It will be understood that the DNA ) sequence to be subjected to random mutagenesis is pre-ferably a cDNA or a genomic DNA sequence.<br><br>
In some cases it may be convenient to amplify the mutated DNA sequence prior to performing the expression step b) or the screening step c) . Such amplification may be performed in accordance with methods known in the art, the presently preferred method being PCR-generated amplification using oligonucleotide primers prepared on the basis of the DNA or amino acid sequence of the parent enzyme.<br>
Subsequent to the incubation with or exposure to the mutagenising agent, the mutated DNA is expressed by culturing a suitable host cell carrying the DNA sequence under conditions allowing expression to take place. The host cell used for this purpose may be one which has been transformed with the mutated DNA sequence, optionally present on a vector, or one which was carried the DNA sequence encoding the parent enzyme during the mutagenesis treatment. Examples of suitable host cells are the following: gram positive bacteria such as Bacillus subtilis. Bacillus licheniformis. Bacillus lentus, Bacillus brevis. Bacillus stearothermophilus, Bacillus alkalophilus. Bacillus amyloliquefaciens. Bacillus coagulans. Bacillus circulans. Bacillus lautus. Bacillus megaterium. Bacillus thuringiensis, Streptomyces lividans or Streptomyces murinus; and gram-negative bacteria such as E. coli.<br>
The mutated DNA sequence may further comprise a DNA sequence encoding functions permitting expression of the mutated DNA sequence.<br>
The random mutagenesis may be advantageously localised to a part of the model phytase in question using Localized random mutagenesis. This may, e.g., be advantageous when certain regions of the enzyme have been identified to be of particular importance for a given property of the enzyme, and when modified are expected to result in a variant having improved properties.<br><br>
Such regions may normally be identified when the tertiary structure of the parent enzyme has been elucidated and related to the function of the enzyme.<br>
The localized, or region-specific; random mutagenesis is conveniently performed by use of PCR generated mutagenesis techniques as described above or any other suitable technique known in the art. Alternatively, the DNA sequence encoding the part of the DNA sequence to be modified may be isolated, e.g., by insertion into a suitable vector, and said part may be subsequently subjected to mutagenesis by use of any of the mutagenesis methods discussed above.<br>
For region-specific random mutagenesis with a view to amending e.g. the specific activity of a model phytase, codon positions corresponding to the following amino acid residues from the amino acid sequences set forth in Fig. 1 may appropriately be targeted:<br>
Residues: 41-47, 68-80, 83-84, 115-118, 120-126, 128, 149-163, 184-185, 191-193, 198-201e, 202-203, 205, 235-236, 238-239, 242-243, 270-279, 285, 288, 332-343, 364-367, 369-375, 394.<br>
Regions: 41-47, 68-80, 120-128, 149-163, 270-279, 332-343, 364-375.<br>
The random mutagenesis may be carried out by the following steps:<br>
1.	Select regions of interest for modification in the parent enzyme<br>
2.	Decide on mutation sites and non-mutated sites in the selected region<br>
3.	Decide on which kind of mutations should be carried out, e.g. with respect to the desired stability and/or performance of the variant to be constructed<br>
4.	Select structurally reasonable mutations<br><br>
5.	Adjust the residues selected by step 3 with regard to step 4.<br>
6.	Analyse by use of a suitable dope algorithm the nucleotide distribution.<br>
7.	If necessary, adjust the wanted residues to genetic code realism, e.g. taking into account constraints resulting from the genetic code, e.g. in order to avoid introduction of stop codons; the skilled person will be aware that some codon combinations cannot be used in practice and will need to be adapted<br>
8.	Make primers<br>
9.	Perform random mutagenesis by use of the primers<br>
10.	Select resulting phytase variants by screening for the desired improved properties.<br>
Suitable dope algorithms for use in step 6 are well known in the art. One such algorithm is described by Tomandl, D. et al., 1997, Journal of Computer-Aided Molecular Design 11:29-38. Another algorithm is DOPE (Jensen, LJ, Andersen, KV, Svendsen, A, and Kretzschmar, T (1998) Nucleic Acids Research 26:697-702).<br>
A DNA sequence encoding a model phytase or a phytase variant of the invention can be expressed using an expression vector, a recombinant expression vector, which typically includes control sequences encoding a promoter, operator, ribosome binding site, translation initiation signal, and, optionally, a repressor gene or various activator genes.<br>
The recombinant expression vector may be any vector which may conveniently be subjected to recombinant DNA procedures, and the choice of vector will often depend on the host cell into which it is to be introduced. Thus, the vector may be an autonomously replicating vector, e.g. a plasmid, a bacteriophage or an extra-chromosomal element. Alternatively, the vector may<br><br>
be one which, when introduced into a host cell, is integrated into the host cell genome and replicated together with the chromosome(s) into which it has been integrated.<br>
In the vector, the DNA sequence should be operably connected to a suitable promoter sequence. The promoter may be any DNA sequence which shows transcriptional activity in the host cell of choice and may be derived from genes encoding proteins either homologous or heterologous to the host cell. An example of a suitable promoter for directing the transcription of the DNA sequence encoding a phytase variant of the invention, especially in a bacterial host, is the promoter of the lac operon of E.coli. For transcription in a fungal host, examples of useful promoters are those derived from the gene encoding A. oryzae TAKA amylase.<br>
The expression vector of the invention may also comprise a suitable transcription terminator and, in eukaryotes, polyadenylation sequences operably connected to the DNA sequence encoding the phytase variant of the invention. Termination and polyadenylation sequences may suitably be derived from the same sources as the promoter.<br>
The vector may further comprise a DNA sequence enabling the vector to replicate in the host cell in question. Examples of such sequences are the origins of replication of plasmids pUC19, PACYC177, pUBllO, pE194, pAMBl and pIJ702.<br>
The vector may also comprise a selectable marker, e.g. a gene the product of which complements a defect in the host cell, such as the dal genes from B. subtilis or B. licheniformis, or one which confers antibiotic resistance such as ampicillin resistance. Furthermore, the vector may comprise Aspergillus selection markers such as amdS, argB, niaD and sC, or the<br><br>
selection may be accomplished by co-transformation,  e.g.  as described in WO 91/17243.<br>
The procedures used to ligate the DNA construct of the invention encoding a phytase variant, the promoter, terminator and other elements, respectively, and to insert them into suitable vectors containing the information necessary for replication, are well known to persons skilled in the art (cf., for instance, Sambrook et al. (1989)).<br>
The cell of the invention, either comprising a DNA construct or an expression vector of the invention as defined above, is advantageously used as a host cell in the recombinant production of a phytase variant of the invention. The cell may be transformed with the DNA construct of the invention encoding the variant, conveniently by integrating the DNA construct (in one or more copies) in the host chromosome. This integration is generally considered to be an advantage as the DNA sequence is more likely to be stably maintained in the cell. Integration of the DNA constructs into the host chromosome may be performed according to conventional methods, e.g. by homologous or heterologous recombination. Alternatively, the cell may be transformed with an expression vector as described above in connection with the different types of host cells.<br>
An isolated DNA molecule or, alternatively, a "cloned DNA sequence" "a DNA construct," "a DNA segment" or "an isolated DNA sequence" refers to a DNA molecule or sequence which can be cloned in accordance with standard cloning procedures used in genetic engineering to relocate the DNA segment from its natural location to a different site where it will be replicated. The term refers generally to a nucleic acid sequence which is essentially free of other nucleic acid sequences, e.g., at least about 20% pure,  preferably at  least about  40% pure,  more<br><br>
preferably about 60% pure, even more preferably about 80% pure, most preferably about 90% pure, and even most preferably about 95% pure, as determined by agarose gel electrophoresis. The cloning procedures may involve excision and isolation of a desired nucleic acid fragment comprising the nucleic acid sequence encoding the polypeptide, insertion of the fragment into a vector molecule, and incorporation of the recombinant vector into a host cell where multiple copies or clones of the nucleic acid sequence will be replicated. The nucleic acid sequence may be of genomic, cDNA, RNA, semisynthetic, synthetic origin, or any combinations thereof.<br>
The term "vector" is intended to include such terms/objects as "nucleic acid constructs," "DNA constructs," expression vectors" or "recombinant vectors."<br>
The nucleic acid construct comprises a nucleic acid sequence of the present invention operably linked to one or more control sequences capable of directing the expression of the coding sequence in a suitable host cell under conditions compatible with the control sequences.<br>
"Nucleic acid construct" is defined herein as a nucleic acid molecule, either single or double-stranded, which is isolated from a naturally occurring gene or which has been modified to contain segments of nucleic acid which are combined and juxtaposed in a manner which would not otherwise exist in nature.<br>
The term nucleic acid construct may be synonymous with the term expression cassette when the nucleic acid construct contains all the control sequences required for expression of a coding sequence of the present invention, ' The term "coding sequence" as defined herein primarily comprises  a  sequence  which  is  transcribed  into  mRNA  and<br><br>
translated into a polypeptide of the present invention when placed under the control of the above mentioned control sequences. The boundaries of the coding sequence are generally determined by a translation start codon ATG at the 5' -terminus and a translation stop codon at the 3'-terminus. A coding sequence can include, but is not limited to, DNA, cDNA, and recombinant nucleic acid sequences.<br>
The term "control sequences" is defined herein to include all components which are necessary or advantageous for expression of the coding sequence of the nucleic acid sequence. Each control sequence may be native or foreign to the nucleic acid sequence encoding the polypeptide. Such control sequences include, but are not limited to, a leader, a polyadenylation sequence, a propeptide sequence, a promoter, a signal sequence, and a transcription terminator. At a minimum, the control sequences include a promoter, and transcriptional and translational stop signals. The control sequences may be provided with linkers for the purpose of introducing specific restriction sites facilitating ligation of the control sequences with the coding region of the nucleic acid sequence encoding a polypeptide.<br>
A "host cell" or "recombinant host cell" encompasses any progeny of a parent cell which is not identical to the parent cell due to mutations that occur during replication.<br>
The cell is preferably transformed with a vector comprising a nucleic acid sequence of the invention followed by integration of the vector into the host chromosome.<br>
"Transformation" means introducing a vector comprising a nucleic acid sequence of the present invention into a host cell so that the vector is maintained as a chromosomal integrant or as a self-replicating extra-chromosomal vector.  Integration is<br><br>
generally considered to be an advantage as the nucleic acid sequence is more likely to be stably maintained in the cell. Integration of the vector into the host chromosome may occur by homologous or non-homologous recombination as described above.<br>
The host cell may be a unicellular microorganism, e.g., a prokaryote, or a non-unicellular microorganism, e.g., a eukaryote. Examples of a eukaryote cell is a mammalian cell, an insect cell, a plant cell or a fungal cell. Useful mammalian cells include Chinese hamster ovary (CHO) cells, HeLa cells, baby hamster kidney (BHK) cells, COS cells, or any number of other immortalized cell lines available, e.g., from the American Type Culture Collection.<br>
In a preferred embodiment, the host cell is a fungal cell.<br>
Fungal cells may be transformed by a process involving protoplast formation, transformation of the protoplasts, and regeneration of the cell wall in a manner known per se.<br>
The present invention also relates to a transgenic plant, plant part, such as a plant seed, or plant cell, which has been transformed with a DNA sequence encoding the phytase of the invention so as to express or produce this enzyme. Also compositions and uses of such plant or plant part are within the scope of the invention, especially its use as feed and food or additives therefore, along the lines of the present use and food/feed claims.<br>
The transgenic plant can be dicotyledonous or monocotyledonous, for short a dicot or a monocot. Of primary interest are such plants which are potential food or feed components and which comprise phytic acid. A normal phytic acid level of feed components is 0.1-100 g/kg, or more usually 0.5-50 g/kg, most usually 0.5-20 g/kg. Examples of monocot plants are grasses, such as meadow grass (blue grass, Poa), forage grass<br><br>
such as festuca, lolium, temperate grass, such as Agrostis, and cereals, e.g. wheat, oats, rye, barley, rice, sorghuin and maize (corn) .<br>
Examples of dicot plants are legumes, such as lupins, pea, bean and soybean, and cruciferous (family Brassicaceae), such as cauliflower, oil seed rape and the closely related model organism Arabidopsis thaliana.<br>
Such transgenic plant etc. is capable of degrading its own phytic acid, and accordingly the need for adding such enzymes to food or feed comprising such plants is alleviated. Preferably, the plant or plant part, e.g. the seeds, are ground or milled, and possibly also soaked before being added to the food or feed or before the use, e.g. intake, thereof, with a view to adapting the speed of the enzymatic degradation to the actual use.<br>
If desired, the plant produced enzyme can also be recovered from the plant. In certain cases the recovery from the plant is to be preferred with a view to securing a heat stable formulation in a potential subsequent pelleting process.<br>
Examples of plant parts are stem, callus, leaves, root, fruits, seeds, tubers etc. But also any plant tissue is included in this definition.<br>
Any plant cell, whatever the tissue origin, is included in the definition of plant cells above.<br>
Also included within the scope of the invention are the progeny of such plants, plant parts and plant cells.<br>
The skilled man will know how to construct a DNA expression construct for insertion into the plant in question, paying regard i.a. to whether the enzyme should be excreted in a tissue specific way. Of relevance for this evaluation is the stability (pH-stability, degradability by endogenous proteases etc, )  of the phytase in the expression compartments of the<br><br>
plant. He will also be able to select appropriate regulatory sequences such as promoter and terminator sequences, and signal or transit sequences if required (Tague et al, Plant, Phys., 86, 506, 1988).<br>
The plant, plant part etc. can be transformed with this DNA construct using any known method. An example of such method is the transformation by a viral or bacterial vector such as bacterial species of the genus Agrobacterium genetically engineered to comprise the gene encoding the phytase of the invention. Also methods of directly introducing the phytase DNA into the plant cell or plant tissue are known in the art, e.g. micro injection and electroporation (Gasser et al, Science, 244, 1293; Potrykus, Bio/Techn. 8, 535, 1990; Shimamoto et al. Nature, 338, 274, 1989).<br>
Following the transformation, the transformants are screened using any method known to the skilled man, following which they are regenerated into whole plants.<br>
These plants etc. as well as their progeny then carry the phytase encoding DNA as a part of their genetic equipment.<br>
In general, reference is made to WO 9114782A and WO 9114772A.<br>
Agrobacterium tumefaciens mediated gene transfer is the method of choice for generating transgenic dicots (for review Hooykas &amp; Schilperoort, 1992. Plant Mol. Biol. 19: 15-38), however it can also be used for transforming monocots. Due to host range limitations it is generally not possible to transform monocots with the help of A. tumefaciens. Here, other methods have to be employed. The method of choice for generating transgenic monocots is particle bombardment (microscopic gold or tungsten particles coated with the transforming DNA) of embryonic calli or developing embryos (Christou, 1992. Plant J.<br><br>
2: 275-281; Shimamoto, 1994. Curr. Opin. Biotechnol. 5: 158-162; Vasil et al., 1992. Bio/Technology 10: 667-674).<br>
Also other systems for the delivery of free JDNA into these plants, including viral vectors (Joshi &amp; Joshi, 1991. FEES Lett. . 281: 1-8) , protoplast transformation via polyethylene glycol or electroperation (for review see Potyrkus, 1991. Annu. Rev. Plant Physiol. Plant Mol. Biol. 42: 205-225), microinjection of DNA into mesophyll protoplasts (Crossway et al., 1986. Mol. Gen. Genet. 202: 79-85) , and macroinjection of DNA into young floral tillers of cereal plants (de la Pena et al., 1987. Nature 325: 274-276) are preferred methods.<br>
In general, the cDNA or gene encoding the phytase variant of the invention is placed in an expression cassette  (e.g. Pietrzak et  al.,  1986.  Nucleic Acids  Res.  14:  5857-5868) consisting of a suitable promoter active in the target plant and a  suitable  terminator  (termination of  transcription).  This cassette (of course including a suitable selection marker, see below) will be transformed into the plant as such in case of monocots via particle bombardment.  In case of dicots  the expression cassette is placed first into a suitable vector providing the T-DNA borders and a suitable selection marker which in turn are transformed into Agrobacterium tumefaciens. Dicots will be transformed via the Agrobacterium harbouring the expression  cassette and  selection marker  flanked by  T-DNA following standard protocols  (e.g. Akama et al., 1992. Plant Cell Reports 12: 7-11). The transfer of T-DNA from Agrobacterium to the Plant cell has been recently reviewed (Zupan &amp; Zambryski, 1995.  Plant  Physiol.  107: ^ 1041-1047).  Vectors  for  plant transformation via Agrobacterium are commercially available or can be obtained from many labs that construct such vectors (e.g. Deblaere et al., 1985. Nucleic Acids Res. 13: 4777-4788; for<br><br>
review see Klee et al., 1987. Annu. Rev. Plant Physiol. 38: 467-486) .<br>
Available plant promoters: Depending on the process under manipulation, organ- and/or cell-specific expression as well as appropriate developmental and environmental control may be required. For instance, it is desirable to express a phytase cDNA in maize endosperm etc. The most commonly used promoter has been the constitutive 35S-CaMV promoter Franck et al., 1980. Cell 21: 285-294) . Expression will be more or less equal throughout the whole plant. This promoter has been used successfully to engineer herbicide- and pathogen-resistant plants (for review see Stitt &amp; Sonnewald, 1995. Annu. Rev. Plant Physiol. Plant Mol. Biol. 46: 341-368). Organ-specific promoters have been reported for storage sink tissues such as seeds, potato tubers, and fruits (Edwards &amp; Coruzzi, 1990. Annu. Rev. Genet, 24: 275-303), and for metabolic sink tissues such as meristems (Ito et al., 1994. Plant Mol. Biol. 24: 863-878).<br>
The medium used to culture the transformed host cells may be any conventional medium suitable for growing the host cells in question. The expressed phytase may conveniently be secreted into the culture medium and may be recovered therefrom by well-known procedures including separating the cells from the medium by centrifugation or filtration, precipitating proteinaceous com-ponents of the medium by means of a salt such as ammonium sulphate, followed by chromatographic procedures such as ion exchange chromatography, affinity chromatography, or the like.<br>
Preferred host cells are a strain of Fusarium, Hansenula, Trichoderma or Aspergillus, in particular a strain of Fusarium graminearum, Fusarium venenatum, Fusarium cerealis, Fusarium sp. having the identifying characteristic of Fusarium ATCC 20334, as further  described  in  PCT/US/95/07743,  Hansenula  polymorpha,<br><br>
Trichoderma harzianum or Trichoderma reesei, Aspergillus niger or Aspergillus oryzae.<br>
References for expression in Hansenula polymorpha: Gellissen, G., Piontek, M., Dahlems, U.,. Jenzelewski, V., Gavagan, J.E., DiCosiino, R. , Anton, D, I. &amp; Janowicz, Z.A. (1996) Recombinant Hansenula polymorpha as a biocatalyst: coexpression of the spinach glycolate oxidase (GO) and the S. cerevisiae catalase T (CTTl) gene- Appl. Microbiol. Biotechnol. 46, 46-54.<br>
Some more specific uses of the phytase variants according to the invention appear from PCT/DK97/00568, the last pages of the detailed description of the invention section.<br>
In a preferred embodiment, the phytase variant of the invention is essentially free of other non-phytase polypeptides, e.g., at least about 20% pure, preferably at least about 40% pure, more preferably about 60% pure, even more preferably about 80% pure, most preferably about 90% pure, and even most preferably about 95% pure, as determined by SDS-PAGE. Sometimes such polypeptide is alternatively referred to as a "purified" and/or '"isolated" phytase.<br>
A phytase polypeptide which comprises a phytase variant of the invention includes fused polypeptides or cleavable fusion polypeptides in which another polypeptide is fused at the N-terminus or the C-terminus of the polypeptide or fragment thereof. A fused polypeptide is produced by fusing a nucleic acid sequence (or a portion thereof) encoding another polypeptide to a nucleic acid sequence (or a portion thereof) encoding a phytase variant of the present invention. Techniques for producing fusion polypeptides are known in the art, and include, ligating the coding sequences encoding the polypeptides so that they are in frame and that expression of the fused<br><br>
polypeptide is under control of the  same promoter(s)  and terminator.<br>
A "feed" and a "food," respectively, means any natural or artificial diet, meal or the like or components of such meals ) intended or suitable for being eaten, taken in, digested, by an animal and a human being, respectively.<br>
The phytase variant of the invention may exert its effect in vitro or in vivo, i.e. before intake or in the stomach of the individual, respectively- Also a combined action is possible.<br>
A phytase composition according to the invention always comprises at least one phytase of the invention.<br>
Generally, phytase compositions are liquid or dry.<br>
Liquid compositions need not contain anything more than the phytase enzyme, preferably in a highly purified form. Usually, however, a stabilizer such as glycerol, sorbitol or mono propylen glycol is also added. The liquid composition may also comprise other additives, such as salts, sugars, preservatives, pH-adjusting agents, proteins, phytate (a phytase substrate). Typical liquid compositions are aqueous or oil-based slurries. The liquid compositions can be added to a food or feed after an optional pelleting thereof.<br>
Dry compositions may be spray-dried compositions, in which case the composition need not contain anything more than the enzyme in a dry form. Usually, however, dry compositions are so-called granulates which may readily be mixed with e.g. food or feed components, or more preferably, form a component of a pre-mix. The particle size of the enzyme granulates preferably is compatible with that of the other components of the mixture, This provides a safe and convenient means of incorporating enzymes into e.g. animal feed.<br><br>
Agglomeration granulates are prepared using agglomeration technique in a high shear mixer (e.g. Lodige) during which a filler material and the enzyme are co-agglomerated to form granules. Absorption granulates are prepared by having cores of a carrier material to absorb/be coated by the enzyme.<br>
Typical filler materials are salts such as disodium sulphate. Other fillers are kaolin, talc, magnesium aluminium silicate and cellulose fibres. Optionally, binders such as dextrins are also included in agglomeration granulates.<br>
Typical carrier materials are starch, e.g. in the form of cassava, corn, potato, rice and wheat. Salts may also be used.<br>
Optionally, the granulates are coated with a coating mixture. Such mixture comprises coating agents, preferably hydrophobic coating agents, such as hydrogenated palm oil and beef tallow, and if desired other additives, such as calcium carbonate or kaolin.<br>
Additionally, phytase compositions may contain other substituents such as colouring agents, aroma comjJounds, stabilizers, vitamins, minerals, other feed or food enhancing enzymes, i.e. enzymes that enhances the nutritional properties of feed/food, etc. This is so in particular for the so-called pre-mixes.<br>
A "food or feed additive" is an essentially pure compound or a multi component composition intended for or suitable for being added to food or feed. In particular it is a substance which by its intended use is becoming a component of a food or feed product or affects any characteristics of a food or feed product. It is composed as indicated for phytase compositions above. A typical additive usually comprises one or more compounds such as vitamins, minerals or feed enhancing enzymes and suitable carriers and/or excipients.<br><br>
In a preferred embodiment, the phytase compositions of the invention additionally comprises an effective amount of one or more feed enhancing enzymes, in particular feed enhancing enzymes selected from the group consisting of a-galactosidases, P-galactosidases, in particular lactases, other phytases, p^ glucanases, in particular endo-p-1,4-glucanases and endo-p-1,3(4)-glucanases, cellulases, xylosidases, galactanases, in particular arabinogalactan endo-1,4-p-galactosidases and arabinogalactan endo-1,3-p-galactosidases, endoglucanases, in particular endo-1,2-p-glucanase, endo-1,3-a-glucanase, and endo-<br>
1,3-p-glucanase, pectin degrading enzymes, in particular pectinases, pectinesterases, pectin lyases, polygalacturonases, arabinanases, rhamnogalacturonases, rhamnogalacturonan acetyl esterases, rhamnogalacturonan-a-rhamnosidase, pectate lyases, and a-galacturonisidases, mannanases, p-mannosidases, mannan acetyl esterases, xylan acetyl esterases, proteases, xylanases, arabinoxylanases and lipolytic enzymes such as lipases, phospholipases and cutinases.<br>
The animal feed additive of the invention is supplemented to the mono-gastric animal before or simultaneously with the diet. Preferably, the animal feed additive of the invention is supplemented to the mono-gastric animal simultaneously with the diet. In a more preferred embodiment, the animal feed additive is added to the diet in the form of a granulate or a stabilized liquid.<br>
An effective amount of phytase in food or feed is from about 10-20.000; preferably from about 10 to 15.000, more preferably from about 10 to 10.000, in particular from about 100 to 5.000, especially from about 100 to about 2.000 FYT/kg feed or food.<br><br>
Examples of other specific uses of the phytase of the invention is in soy processing and in the manufacture of inositol or derivatives thereof.<br>
The invention also relates to a method for reducing phytate levels in animal manure, wherein the animal is fed a feed comprising an effective amount of the phytase of the invention.<br>
Also comprised in this invention is the use of a phytase of the invention during the preparation of food or feed preparations or additives, i.e, the phytase exerts its phytase activity during the manufacture only and is not active in the final food or feed product. This aspect is relevant for instance in dough making and baking.<br>
The invention relates to a phytase variant which, when aligned according to Fig. 1, is amended as compared to a model phytase in at least one of the following positions, using the position numbering corresponding to P_lycii:<br>
24; 27; 31; 33; 39; 40; 41; 42; 43; 44; 45; 46; 47; 49; 51; 56; 58; 59; 61; 62; 68; 69; 70; 71; 72; 73; 74; 75; 76; 77; 78; 79; 80; 81; 82; 83; 84; 88; 90; 102; 115; 116; 117; 118; 119; 120; 121; 122; 123; 124; 125; 126; 127; 128; 132; 143; 148; 149; 150; 151; 152; 153; 154; 155; 156; 157; 158; 159; 160; 161; 162; 163; 170f; 170g; 171; 172; 173; 184; 185; 186; 187; 187a; 190; 191; 192; 193; 194; 195; 198; 199; 200; 201; 201a; 201b; 201c; 201d; 201e; 201f; 202; 203; 203a; 204; 205; 211; 215; 220; 223; 228; 232; 233; 234; 235; 236; 237; 238; 239; 242; 243; 244; 246; 251e; 253; 256; 260; 264; 265; 267; 270; 271; 272; 273; 274; 275; 276; 277; 278; 279; 280; 283; 285; 287; 288; 292; 293; 302; 304; 332; 333; 334; 335; 336; 337; 338; 339; 340; 341; 342; 343; 348; 349; 352; 360; 362; 364; 365; 366; 367; 368; 369; 370; 371;<br><br>
372; 373; 374; 375; 376; 383k; 387; 393; 394; 396; 404; 409; 411; 412; 413; 417; 421; 431.<br>
From these variants we expect amended characteristics, preferably amended activity characteristics. In fact, for several variants such amended characteristics have already been shown (see the experimental part). Like above, "amended" means as compared to the model phytase, '"Amended activity characteristics" means amended in at least one phytase activity related respect, such as (non-exclusive list): pH stability, temperature stability, pH profile, temperature profile, specific activity (in particular in relation to pH and temperature), substrate specificity, substrate cleavage pattern, substrate binding, position specificity, the velocity and level of release of phosphate from corn, reaction rate, phytate degradation rate), end level of released phosphate reached-<br>
Preferred amended activity characteristics are amended specific activity, preferably increased, and preferably increased at a pH of 3, 4, 5, or 6; amended pH or temperature profile; and/or amended, preferably increased, thermostability, e.g. of an increased melting temperature as measured using DSC-<br>
Preferred phytase variants are: Phytase variants which, when aligned according to Fig, 1, are amended as compared to a model phytase in at least one of the following positions, using the position numbering corresponding to P_lycii: 43; 44; 47; 51; 58; 62; 78; 80; 83; 88; 90; 102; 143; 148; 153; 154; 186; 187a; 195; 198; 201e; 204; 205; 211; 215; 220; 242; 244; 251e; 260; 264; 265; 267; 270; 273; 278; 302; 336; 337; 339; 352; 365; 373; 383k; 404; 417.<br>
The  following  variants  of  A_fumigatus  constitute  a subgroup: Q43L; Q270L; G273D,K; N336S; A205E; Y278H; Q43L+Q270L;<br><br>
Q43L+Q270L+G273D;      Q4 3L+Q270L+G273D+N335S;      G273K+A205E; G273K+A205E+Y278H (see EP 0897010).<br>
Generally, variants of the invention can be deduced or identified as follows: Looking at the alignment according to Fig. 1, comparing two sequences, one of which is a model phytase with improved properties, identifying amino acid differences in relevant positions/areas, and transferring (substituting with) from the model to the other phytase sequence the amino acid in a relevant position.<br>
The invention also relates to a process for preparing a phytase variant which includes the above method, and further includes the deducement and synthesis of the corresponding DNA sequence, the transformation of a host cell, the cultivation of the host cell and the recovery of the phytase variant.<br>
Relevant positions/areas include those mentioned below in relation to important phytase activity characteristics such as specific activity, thermostability, pH activity/stability.<br>
The present invention also relates to phytase variants (varied according to a model phytase as defined herein) which are obtainable, preferably obtained, by the process outlined above and which are expected to exhibit an amended characteristic/property, preferably does exhibit such amended characteristic, e.g. an improved specific activity.<br>
At least the basidiomycete model phytases P_lycii and T_pubescens exhibit a high specific activity (as determined using the method of Example 2 herein).<br>
This is an example of a desired property which can be transferred to other phytases, e.g. the other phytases listed in Fig. 1, in particular to the A_pediades and the ascomycete phytases such as A_fumigatus, A-ficuum, consphyA, by a deducement process such as the one mentioned above.<br><br>
Thus, amended specific activity, in particular an improved spe'cific activity, in particular at low pH and/or high temperature, is expected from variants, which have been amended in relevant areas, viz. (i) in the amino acid residues which point into the active site cleft; or (ii) in the amino acid residues in the close neighbourhood of these active site residues. Preferably, close neighbourhood means within lOA from the active site residues.<br>
From the pdb file IIHP (Brookhaven Database entry of 18.03.98 re IIHP, Structure of Phosphomonoesterase, D.Kostrewa; or as published in Nature Structural Biology, 4, 1997, p. 185-190) , active site regions can be identified, using the program INSIGHTII from Molecular Simulations MSI, San Diego, California, and using the subset command, an "active site shell" can be defined comprising those amino acid residues which lie close to the catalytic residues, defined as H59, D339 and R58 in A. ficuum phytase (corresponding to Peniophora numbers H71, D335 and R70, respectively). An "active site shell(lOA)" comprises those residues which lie within lOA from the above catalytic residues.<br>
The residues within lOA from H71 and D335 are the following (using Peniophora numbers): 41-47, 68-77, 115-118, 120-126, 128, 149-163, 185, 191-193, 199, 243, 270-271, 273-275, 277-279, 288, 332-343, 364-367, 369-375, 394 ("the active site shell(lOA)").<br>
Preferably, a "substrate binding shell" can also be defined which comprises those residues which are in close proximity to the substrate binding site and which can therefore be expected to be in contact with the substrate.<br>
This information can be deduced as described above, by docking a sugar analogue to phytin into the active site cleft<br><br>
(the residues making up the surface of the active site). If a sugar without any phosphate groups is docked into the active site cleft, e.g. alpha-D-glucose (chair conformation, structure provided by the INSIGHTII program), using a fixed distance as shown below, the residues pointing towards the active site cleft can be extracted using the subset command and using a distance of lOA from the substrate analogue. Alternatively, the compound inositol-1,4,5-triphosphate (Brookhaven database file Idjx, Inositol-1,4,5"triphosphate) can be docked into the active site cleft. This compound and glucose, however, are more or less superimposable.<br>
The distances in Angstrom (A) are: From oxygen atom in position 6 of the alpha-D-glucose to<br>
atom NDl of H59:        5.84<br>
atom NH2 of R58:       6.77<br>
atom NH2 of R142:       5.09<br>
atom ND2 of N340:       3.00<br>
atom NDl of H59:       7.76<br>
atom NH2 of R58:       8.58.<br>
(the Peniophora numbers of the above residues are: H71, R70, R155, N336, H71 and R70, respectively).<br>
In this way, the residues in contact with the substrate are identified as follows (Peniophora numbers): 43-44; 70-80; 83-84; 115; 153; 155-156; 184; 191-192; 198-202; 205; 235; 238; 242; 270; 272-273; 275-277; 332-336; 338; 369; 371 ("the substrate binding shell(loA)")-<br>
Variants being amended in one or more of (1) the active site shell or (2) the substrate binding shell, are strongly expected to have an amended specific activity. This leads to the following joint grouping of positions (still Peniophora numbers and lOA shells): 41-47, 68-80, 83-84,  115-118, 120-126,  128,<br><br>
149-163, 184-185, 191-193, 198-201e, 202-203, 205, 235-236, 238-239, 242-243, 270-279, 285, 288, 332-343, 364-367, 369-375, 394. Preferably, the active site shell and the substrate binding shell are defined as described above using the basidiomycete model phytases of Fig. 1, the Peniophora phytase being a preferred model. A deducement of corresponding variants of other model phytases is possible using the alignment of Fig. 1.<br>
In a preferred embodiment, a distance of 5A is used in the subset command, thus defining active site and substrate binding shells of a more limited size, e.g. an active site shell comprising the residues 4 3-4 4, 69-74, 117, 125, 155-156, 159, 274, 332-340, 370-374 (5A from H71 and D335) , "active site shell(SA)".<br>
Generally the active site shell and substrate binding shell regions form the basis for selecting random mutagenesis regions. Examples of preferred random mutagenesis regions are<br>
regions 69-74, 332-340, 370-374, doping to be added (a 5A approach); and<br>
regions 57-62, 142-146, 337-343, doping to be added (a lOA approach).<br>
It is presently contemplated that any amendment in either of these positions will lead to a phytase of amended characteristics, e.g. of an amended specific activity.<br>
The above expression ^'any amendment in either of the positions" is considered fully equivalent to listing each position and each substitution, e.g. as follows for the above sub-group 41-47:<br>
41A,C,D,E,F,G,H,I,K,L,M,N,P,Q,R,S,T,V,W,Y; 42A,C,D,E,F,G,H,I,K,L,M,N,P,Q,R,S,T,V,W,Y; 43A,C,D,E,F,G,H,I,K,L,M,N,P,Q,R,S,T,V,W,Y;<br><br>
44A,C,D,E,F,G,H,I,K,L,M,N,P,Q,R,S,T,V,W,Y; 45A,C,D,E,F,G,H,I,K,L,M,N,P,Q,R,S,T,V,W,Y; 4 6A,C,D,E,F,G,H,I,K,L,M,N,P,Q,R,S,T,V,W,Y; 47A,C,D,E,F,G,H,I,K,L,M,N,P,Q,R,S,T,V,W,Y.<br>
In a preferred embodiment, amended specific activity is expected from the following variants:<br>
42S,G; 43A,C,D.,E,F,G,H,I,K,L,M,N,P,Q,R,S,T,V,W,Y; 45D,S; 47Y,F/ 51E,A; 75W,F; 78S,D; 79G; 80K,A; 831,Q; 84Q,V; 116S; 118V,L; 119E; 120L; 122A; 123N,T; 125S; 126H,S; 127Q,E; 128A,T; 151A,S; 152G; 153D,Y; 154Q,D,G; 157V; 158D,A; 159T; 160A,S; 161T,N; 162N; 163W; 184Q,S; 186A,E; 198A,N; 200G,V; 201D; deletions of one or more of 201a, 201b, 201c, 201d, 201e, 201f -preferably all; 202S; 205Q,E; 235Y,L; 238L,M; 242P; 270Y,A,L; 271D; 273D,K; 275F,Y; 278T,H; 332F; 336S; 337T,Q; 339V; 340P,A; 343A,S; 364W,F; 365V,L; 366D,V; 367K; 368K; 3691,L; 370V; 373S; 374A; 375H; 376M; 393V.<br>
Particularly preferred variants are the following: 78S; 79G; 80A; 831,Q; 84Q,V; 198A,N; 200G,V; 201D; deletions in one or more of 201a, 201b, 201c, 201d, 201e, 201f - preferably all deletions; 202S; 205Q,E; 235Y,L; 238L,M; 242P, 273D; 275F,Y.<br>
Other particularly preferred variants are the following: 43A,C,D,E,F,G,H,I,K,L,M,N,P,Q,R,S,T,V,W,Y; in particular 43M,P; 75W,F; 80K; 153D; 184Q,S; 270Y,A; 332F; 3691,L.<br>
The following variants are especially preferred: 43L,G,N,V,A,I,T; 78D; 153Y; 154G; 270L; 273D,K. Double and triple variants (43L/270L); (43L/270L/273D) ; (43L/78D) and (43L/153Y/154G) are also especially preferred. Other preferred variants are 205E; 278H; 336S.<br>
These  especially preferred  single,  double  and triple variants are preferably variants of model phytases which can be<br><br>
aligned to Fig,1, in particular variants of the specific model phytases listed in Fig. 1.<br>
At least consphyA is known to have a high thermostability. Still further, the thermostability of P_lycii is rather high.<br>
This is an example of a desired property which can be transferred to other phytases, e.g. the other phytases listed in Fig. 1, in particular to the basidiomycete phytases such as P_lycii and A_pediades, by a deducement process such as the one mentioned above.<br>
Amended thermostability, in particular improved thermostability, is expected on this background from the following variants:<br>
39H,S; 40L,N; 43P; 47Y,F; 49P; 51E,A; 56P; 58D; 61R; 62V; 80K; 83A; 84Y; 172P; 184P; 195T; 198A; 204V; 211L; 223D; 236Y; 242P; 246V; 253P; 264R; 265Q; 280A,P; 283P; 287A; 292F,Y; 293A; 302R; 304P; 337S; 348Y; 387P; 396R; 409R; 411K; 412R; 417E; 421F,Y.<br>
The following variants of amended thermostability are particularly preferred: 39S; 40N; 47Y,F; 51A; 83A; 195T; 204V; 211L; 242P; 265A.<br>
Further variants of amended thermostability are the following: 42G; 43T,L,G; 44N; 58K,A; 59G; 621; 69Q; 75F; 78D; 79G; 80A; 81A,G; 82T; 83K,R; 841; 881; 90R,A; 102Y; 115N; 118V; 122A; 123Q,N; 125M,S; 126V,S; 127N,Q; 128S,A; 143N,K; 14 8V,I; 154S; 158D; 170fH; 170gA; 171T,N; 172N; 173W, 184S; 186A; 187A; 187aS;. 193S; 195V,L; 198V; 201E; 201eT; 202A, 203aT; 204A; 211V, 215P,A; 220L,N; 223H; 228N; 232T; 322E; 235T; 236N; 242S; 244D; 251eQ,E; 256D; 2641; 260A,H; 265A; 267D; 270G; 271D; 273K,D; 278T,H; 287T; 293V; 302H; 337T,G; 3381; 339V,I; 340A; 352K; 365A,S; 366S; 367A; 369L; 373S,A; 374S; 376M; 383kE,Q; 404G,A; 411T; 417R; 431E.<br><br>
other concepts of the invention, which can be expected to impart an improved thermostability to a phytase, are as follows - considering the IIHP structure previously referred to and transferring via an alignment according to Fig. 1 as outlined herein:<br>
(A)	Introduction of prolin residues in spatial positions where the prolin special dihedral angles are satisfied and the hydrogen bonding network are not hampered and no steric clashes are observed.<br>
(B)	Filling up holes: By substitution for bigger residues in internal cavities an improvement in stability can often be obtained.<br>
(C)	Cystin bridge: Cystin bridges will often make the proteins<br>
more rigid and increase the energy of unfolding.<br>
Further variants from which amended thermostability is expected according to these concepts of (A) to (C) are: 27P, 31Y, 132F, 1321, 132L, 184P, 186P, 190P, 280P, 343F, 3431, 343L, 349P, 362P and (33C and 24C) •<br>
Concept (A): 27P, 184P, 186P, 190P, 349P, 362P.<br>
Concept (B): 343F,I,L; 31Y; 132F,I,L; 273F.<br>
Concept (C): 33C/24C.<br>
Amended pH activity or stability, preferably stability, in particular at low pH, in particular improved, is another desired property which can be transferred by aligning according to Fig, 1 and transferring from models of improved pH profiles to other phytases - as outlined above.<br>
Other concepts of the invention, which can be expected to impart an improved stability at low pH to a phytase, are as follows - considering the IIHP structure previously referred to and transferring via an alignment according to Fig, 1 as outlined herein:<br><br>
(D)	Surface charges: Better distribution at low pH, to avoid<br>
cluster of negative or positive, and to avoid too close same<br>
charged residues.<br>
(E)	Prevent deamidation: Surface exposed Q or N in close<br>
contact to negative charged residues.<br>
Phytase variants having improved pH stability/activity at low pH are expected to be: 39H; 39Q; 80A; 203R; 271N; 51R; 154S; 185S; 194S; 194T; 288L; 2881; 288F; 360R; 17 3Q,S; 204Q,S; 303K,S; 81Q,E.<br>
Concept (D) : 203R, 271N, 51R, 185S, 360R; 173Q,S; 204Q,S; 303K,S; 81Q,E.<br>
Concept (E): 154S; 194S,T; 288L,I,F.<br>
A preferred model phytase for these concepts of (D) and (E) is P_lycii.<br>
Experimentally proven to have a lowered pH optimum is: Variant 80A of ascomycete phytases, in particular of A_fumigatus and consphyA.<br>
Especially preferred single, double and triple variants are 43L; (43L/270L) and (43L/270L/273D)• These variants have a changed pH profile. They are preferably variants of the specific model phytases listed in Fig. 1.<br>
For all preferred variants listed above:<br>
the stability is preferably amended at high temperature, viz. in the temperature range of 50-100°C, in particular 60-90°C, more preferably in the range of 70-90°C;<br>
the activity is preferably amended in a temperature range relevant for the use in the gastro-intestinal system of animals, e.g. 30-40°C, more preferably 32-38°C, most preferably in the range of 35-38 °C;<br>
the stability is preferably amended at low pH, viz. in the pH range of pH 1.5-7, preferably 2-6, more preferably 3-5;<br><br>
the activity is preferably amended in the pH range of pH l-5-5,5, more preferably at pH 2.5-4.5, still more preferably 3-5.<br>
Tests for amended phytase characteristics, such as those mentioned above, are well known in the art and any such test can be used to compare the performance of the phytase variants with the phytase models.<br>
A preferred test for specific activity is given in Example 2. Preferred tests for pH and temperature activity and stability are given in Example 3. An even more preferred test for thermal stability is the DSC method of Example 4.<br>
WO 98/28409 discloses tests for various other parameters, too, such as position specificity. All the tests of WO 98/28409 are preferred tests.<br>
Generally, of course all these tests can be conducted at desired pH values and temperatures.<br>
In the dependent claims, some preferred phytase variants based on five of the thirteen herein specifically disclosed model phytases are specified.<br>
In an analogous way other preferred variants based on the remaining eight specifically disclosed model phytases can easily be deduced by combining the suggested amendments with each of the corresponding sequences of Fig. 1. These preferred variants are specifically included in the present invention, and they are easily deducemed, viz. the following:<br>
Variants of a model phytase derived from Paxillus, preferably Paxillus involutus, preferably derived from strain CBS 100231, preferably variants of P_involtus-Al, the sequence of which is shown at Fig. 2, said variants comprising at least one of the following amendments: {)24C; T27P; F31Y; I33C; R39H,S,Q; N40L; S42G;<br><br>
P43A,C,D,E,F,G,H,I,K,L,M,N,Q,R,S,T,V,W,Y; Y44N; S45D; Y47F/ A51E,R; A58D;K; Q61R; I62V; F75W; S78D; A80K; T81Q,E,G,A; R83A,I,Q,K; I84Y;Q,V; L88I; K90R,A; F102Y; S115N; D116S; V118L/ P119E; F120L; A123N,T,Q; S125M; F126H,S,V; D127Q,E,N; A128T,S; A132F,I,L; I148V; D151A,S; S153D,Y; D154Q,S,G; D158A; S159T; A160S; T161N; {)170fH; ()170gA; S171N; H172P; N173Q,S; P184Q,S; Q185S; T186A,E,P; G187A; ()187aS; T190P,A; D193S; N194S,T; M195T,V,L; A198N,V; G200V; D201E; ()201eT; S202A; D203R,K,S; P203aV,T; Q204E,S,A,V; V205E; V211L; S215A,I; L220N; A223D,H; D233E; F235Y,L,T; N236Y; L237F; V238L,M; A242P,S; M244D; ()251eE,Q; D253P; T256D; P260A,H; E264R,I; A265Q; A267D; G270Y,A,L; D271N; D273K; F275Y; T278H; Y280A,P; E283P; V287A,T/ Q288L,I,F; Y292F; V293A; N302R,H; A304P; N336S; L337T,Q,S,G; M 3381; V339I; A340P; S343A,F,I,L/ F348Y; R349P; A352K; P360R; R362P; W364F; R365V,L,A,S; T366D,V,S; S367K,A; S368K; L369I; S373A; G374A,S; R375H; ()383kQ,E; T387P; Q396R; G404A; L409R; T411K; L412R; E417R; F421Y.<br>
Variants of a model phytase derived from a species of the genus Paxillus, preferably the   species Paxillus involutus, preferably derived from strain CBS 100231, preferably variants of P_involtus-A2, the sequence of which is shown at Fig. 3, said variants comprising at least one of the following amendments: P24C; I27P; F31Y; I33C; R39H,S,Q; N40L; S42G;<br>
P43A,C,D,E,F,G,H,I,K,L,M,N,Q,R,S,T,V,W,Y; Y44N; S45D/ Y47F; A51E,R; A58D,K; E61R; I62V/ F75W; S78D; A80K; A81Q,E,G; R83A,I,Q,R,K; I84Y,Q,V; L88I; K90R,A; F102Y; S115N; D116S; V118L; P119E; F120L; A123N,T,Q; S125M; F126H,S,V; D127Q,E,N; A128T,S; V132F,I,L; D143N; I148V; D151A,S; S153D,Y; D154Q,S,G; D158A; A160S; T161N; ()170fH; ()170gA; S171N; R172P; N173Q,S; P184Q,S; Q185S; T186A,E,P; G187A; ()187aS; T190P,A; D193S; N194S,T;  M195T,V,L/  A198N,V;  G200V;  E2G1D;  ()201eT;  S202A;<br><br>
D203R,K,S; P203aV,T; Q204E,S,A,V; V205E; S211L,V; S215A,P; L220N; A223D,H; A232T; F235Y,L,T; N236Y; L237F; V238L,M; P242S; M244D; ()251eE,Q; D253P; T256D; P260A,H; E264R,I; A265Q; A267D; G270y,A,L; D271N; D273K; F275Y; T278H; Y280A,P; A283P; V287A,T; Q288L,I,F; Y292F; I293A,V; N302R,H; A304P; N336S; L337T,Q,S,G; M338I; V339I; 340P,A; A343S,F,I,L; F348Y; R349P; A352K; P360R; R362P; W364F; L365V,A,S; T366D,V,S; S367K,A; S368K; V369I,L; S373A; R375H; ()383kQ,E; T387P; Q396R; G404A; L409R; A411K,T; L412R; E417R; Y421F.<br>
Variants of a model phytase derived from a species of the genus Trametes,  preferably the  species  Trametes pubescens, preferably derived from strain CBS 100232, preferably variants of T_pubescens, the sequence of which is shown at Fig. 4, said variants comprising at least one of the following amendments: R24C; T27P; L31Y; V33C; Q39H,S; S40L,N; S42G;<br>
M43A,C,D,E,F,G,H,I,K,L,N,P,Q,R,S,T,V,W,Y; Y44N; S45D; Y47F; A51E,R; A58D,K; S59G; Q61R; I62V; F75W; S78D; A80K; A81Q,E,G; R83A,I,Q,K; I84Y,Q,V; V88I; K90R,A; L102Y; D115N; V118L; T123N,Q; S125M; S126H,V; E127Q,N; A128T,S; A132F,I,L; D143N; V148I; S151A; S153D,Y; D154Q,S,G; A158D; A160S; N161T; ()170fH; ()170gA; S171N; S172P; N173Q,S; S184Q,P; E185S; A186E,P; G187A; ()187aS; T190P,A; N194S,T; M195T,V,L; A198N,V; G200V; ()201eT; S202A; D203R,K,S; P203aV,T; Q204E,S,A,V; V205E; Q211L,V; P215A; L220N; G223D,H; D233E; Y235L,T; N236Y; L237F; L238M; P242S; E244D; ()251eE,Q; E253P; Q260A,H; D264R,I; A265Q; A267D; A270Y,L,G; D271N; D273K; F275Y; T278H; y280A,P; V287A,T; Q288L,I,F; Y292F; I293A,V; A302R,H; N304P,A; N336S; Q337T,S,G; M338I; V339I; A340P; S343A,F,I,L;, F348Y; N349P; A352K; P360R; R362P; F364W; L355V,A,S; V366D,S; K367A; I369L; A373S; A374S; R375H; ()383kQ,E; Q387P; A396R; G404A; V409R; T411K; L412R; E417R; Y421F.<br><br>
Variants of a model phytase derived from a species of the genus Aspergillus, preferably the species Aspergillus nidulans, preferably derived from strain DSM 9743, preferably variants of A_nidulans, the sequence of which is shown at Fig. 10, said variants comprising at least one of the following amendments: V24C; A27P; H39S,Q; V40L,N; G42S;<br>
Q43A,C,D,E,F,G,H,I,K,L,M,N,P,R,S,T,V,W,Y; Y44N; S45D; Y47F; S49P; E51A,R; V56P; H58D,K,A; E61R; V62I; S69Q; y75W,F; E78D,S; S79G; K80A; S81Q,E,A,G; K82T; A83I,Q,K,R; Y84Q,V,I; A90R; D115N; D116S; T118V,L; I119E; F120L; E122A; N123T,Q; M125S; V126H,S; D127Q,E,N; S128A,T; F132I,L; K143N; I148V; S151A; S153D,Y; D154Q,S,G; A158D; S159T; A160S; E161T,N; K162N; F163W; G170fH; S170gA; ()171N; ()172P; K173Q,S; P184Q,S; E185S; I186A,E,P/ D187A; G187aS; T190P,A; H193S; S194T; S198A,N,V; E200G,V; N201D,E; D201e(); E201e(),T; R201f() (a deletion of at least one of 201d, 201e, 201f, preferably all); A202S; D203R,K,S; E203aV,T; I204Q,E,S,A,V; I211L,V/ P215A; L220N; D223H; K228N; E232T; N233E; I235Y,L,T; Y236N/ L237F; M238L; S242P; M246V; E251eQ; A256D; E260A,H/ L264R,I; Q270Y,A,L,G; S271D,N; S273D,K; Y275F; G278T,H; A280P; A287T; Q288L,I,F; F292y; T293A,V; Q302R,H; P304A; N336S; S337T,Q,G; M338I; I339V; S340P,A; F343A,S,I,L; N349P; Q352K; S360R; Q362P/ Y364W,F; A365V,L,S; A366D,V,S; S367K,A; W368K; T369I,L; G373S,A; A374S; R375H; A376M; E383kQ; A404G; T411K; L412R; E417R; F421Y; K431E.<br>
Variants of a model phytase derived from a species of Aspergillus, preferably Aspergillus terreus, preferably derived from strain CBS 220.95, preferably variants of A_terreus, the sequence of which is shown at Fig. 12, said variants comprising at least one of the following amendments: G24C; V27P; H39S,Q; K40L,N; G42S;<br><br>
L43A,C,D,E,F,G,H,I,K,M,N,P,Q,R,S,T,V,W,Y; Y44N; A45D,S; Y47F; S49P; Q51E,A,R; V56P; P58D,K,A; D59G; H61R; I62V; A69Q; S75W,F; H78D,S; S79G; K80A; T81Q,E,A,G; A83I,Q,K,R; Y84Q,V,I; A90R; E115N; E116S; T118V,L; P119E; F120L; R122A; N123T,Q; L125S,H; R126H,S,V; D127Q,E,N; L128A,T,S; F132I,L; H143N; V148I; T151A,S; D152G; A153D,Y; S154D,Q,G; H157V; E158D,A; S159T; A160S; E161T,N; K162N; F163W; H173Q,S; P184Q,S; E185S; G186A,E,P; S187A; A187aS; T190P,A; H193S; S194T; L195T,V; A198N,V; E200G,V; S201D,E; S201d(); T201e(); V201f(); G202S,A; D203R,K,S; D203aV,T; A204Q,E,S,V; V205E; V211L; A215P; L220N; D223H; Q228N; D232T; D233E; V235Y,L,T; N236Y; L237F; M238L; P242S; E244E; T251eE,Q; A260H; T264R,I; Q265A; N267D; L270Y,A,G; S271D,N; K273D; Y275F; H278T; G280A,P; V287A,T; Q288L,I,F; W292F,Y; A293V; Q302H; P304A; N337T,Q,S,G; L338I; V339I; S340P,A; W343A,S,F,I,L; N349P; A352K; S360R; S362P; Y364W,F; A365V,L,S; A366D,V,S; A367K; W368K; T369I,L; A373S; A374S; R375H; A376M/ R383kQ,E; P404A,G; K411T; A417E,R; F421Y; A431E.<br>
Variants of a model phytase derived from a species of Talaromyces, preferably the species Talaromyces thermophilus, preferably derived from strain ATCC 2018 6 or ATCC 74338, preferably variants of T_thermo, the sequence of which is shown at Fig. 13, said variants comprising at least one of the following amendments: H24C; V27P; H39S,Q; S40L,N; G42S;<br>
Q43A,C,D,E,F,G,H,I,K,L,M,N,P,R,S,T,V,W,Y; Y44N; S45D; F47Y; S49P; A51E,R; V56P; Q58D,K,A; N59G; K61R; I62V; Y75W,F; S78D; S79G; K80A; T81Q,E,A,G; E82T; L83A,I,Q,R,K; Y84Q,V,I; R90A; D116S; T118V,L; P119E; F120L; E122A; N123T,Q; M125S; I126H,S,V; Q127E,N; L128A,T,S; F132I,L; V148I; S151A; S153D,Y; D154Q,S,G; I157V; A158D; S159T; G160A,-S; R161T,N; L162N; F163W; S170gA;  D171N;  K172P;  H17,3Q,S;  E184Q,S,P;  E185S;  G186A,E,P;<br><br>
D187A; T190P,A; T193S; G194S,T; S195T,V,L; V198A,N; E200G,V; D201E; S201d(); S201e(),T; S201f(); G202S,A; H203R,K,S; D203aV,T; A204Q,E,S,V; Q205E; Q211L,V; A215P; I220N,L; H223D; D228N; S232T; D233E; P235Y,L,T; Y236N; M237F; D238L,M; P242S; E244D; L246V; ()251eE,Q; A256D; Q260A,H; Q264R,I; A265Q; Q270y,A,L,G; S271D,N; G273D,K; Y275F; N278T,H; G280A,P; A287T; Q288L,I,F; F292Y; V293A; H302R; P304A; N336S; T337Q,S,G/ M338I; T339V,I; S340P,A; A343S,F,I,L; N349P; A352K; S360R; E362P; Y364W,F; S365V,L,A; A366D,V,S; A367K; W368K; T369I,L; G373S,A; G374A,S; R375H; A376M; D383k:Q,E; E404A; K411T; R417E; F421Y.<br>
Variants of a model phytase derived from a species of Thermomyces, preferably the species Thermorayces lanuginosus, preferably derived from strain DBS 58 6.94, preferably variants of T_lanuginosa, the sequence of which is shown at Fig. 14, said variants comprising at least one of the following amendments: K24C; ()27P; ()31Y; ()33C; R39H,S,Q; H40L,N; G42S;<br>
Q43A,C,D,E,F,G,H,I,K,L,M,N,P,R,S,T,V,W,Y; Y44N; S45D; F47Y; S49P; A51E,R; V56P; K58D,A; V62I; S69Q; Y75W,F; A78D,S; H79G; K80A; S81Q,E,A,G; E82T; V83A,I,Q,K,R; Y84Q,V,I; L88I; R90A; F102Y; D115N; N116S; T118V,L; R119E; F120L; E122A; E123N,T,Q; M125S; M126H,S,V; E127Q,N; S128A,T; F132I,L; E143N; V148I; A151S; S153D,Y; A154D,Q,S,G; I157V; A158D; S159T; A160S; E161T,N; F162N; F163W; R170fH; S170gA; K172P; D173Q,S; S184Q,P; E185S; E186A,P; T187A; G187aS; T190P,A; G193S; L194S,T; T195V,L; A198N,V; E200G,V; E201D; A201d(); P201e(),T; D202S,A; P203R,K,S; T203aV; Q204E,S,A,V; P205E; V211L; R215A,P; I220L,N; H223D; E232T; D233E; P235Y,L,T; L236Y,N; M238L; P242S; Q251eE; H256D; Q260H; M264R,I; A265Q; Y270A,L,G; T271D,N; D273K; Y275F; H278T; G280A,P; A283P; S287A; R288L,I,F; F292Y; V293A; G302R,H; P304A; I N336S; T337Q,S,G; M338I; T339V,I; G340P,A; S343A,F,I,L; N349P; P360R;  T352P;  Y364W,F; A365V,L,S; A366D,V,S;  S367K,A; W368K;<br><br>
T369I,L; A373S; A374S; R375H; A376M; E383kQ/ R404A,G; R411K,T; K417E,R; F421Y; D431E.<br>
Variants of a model phytase derived from a species of Myceliophthora, preferably the species Myceliophthora thermophila, preferably derived from strain ATCC 48102 or ATCC 74340, preferably variants of M_thermophila, the sequence of which is shown at Fig. 7, said variants comprising at least one of the following amendments: S24C; F31Y; H39S,Q/ F40L,N; G42S;<br>
Q43A,C,D,E,F,G,H,I,K,L,M,N,P,R,S,T,V,W,Y; Y44N; S45D; Y47F; S49P; P51E,A,R; I56P; D58K,A; D59G; E61R; V62I; S69Q; A75W,F; L78D,S; K79G; R80K,A; A81Q,E,G; A82T; S83A,I, Q,K,R; Y84Q,V,I; R90A; D115N; E116S; T118V,L; R119E; T120L; Q122A; Q123N,T; M125S; V126H,S; N127Q,E; S128A,T; F132I,L; K143N; V148I; A151S; Q153D,Y; D154Q,S,G; H158D,A; S159T; A16GS; E161T,N; G170fH; S170gA; T171N; F163W; V172P; R173Q,S; P184Q,S; E185S; T186A,E,P; G187aS; T190P,A; N193S; D194S,T; L195T,V; A198N,V; E200G,V; E201D; G201a(); P201b(); Y201c{); S201d(); T201e(); I201f(); G202S,A; D203R,K,S; D203aV,T; A204Q,E,S,V; Q205E; T211L,V; P215A; V220N,L; N223D,H;. A232T; D233E; V235Y,L,T; A236Y,N; L237F; M238L; P242S; E244D; A251eE,Q; R256D; E260A,H; R264I; A265Q; Q270Y,A,L,G; S271D,N; K273D; Y275F; Y278T,H; P280A; T287A; Q288L,I,F; F292Y; V293A; ()302R,H; P304A; N336S; D337T,Q,S,G; M338I; M339V,I; G340P,A; G343A,S,F, I,L; D349P; P352K; D360R; E362P; Y364W,F; A365V,L,S; A366D,V,S; S367K,A; W368K; A369I,L; A373S; A374S; R375H; I376M; E383kQ; E387P; G404A; M409R; T411K; L412R; E417R; F421Y; D431E.<br>
This invention also provides a new phytase which has been derived from a strain of Cladorrhinum, viz. C. foecundissimum. Accordingly, the invention also relates to a polypeptide having phytase acitivity and which comprises SEQ ID NO: 2 or the mature<br><br>
part (amino acids nos 16-495) thereof; or a polypeptide being at least 70, more preferably 75, 80, 85, 90, 95% homologous thereto; homology meaning similarity, preferably identity, and being determined using the program GAP and the settings as defined hereinabove. And the invention relates to a DNA construct which encodes a polypeptide having phytase activity, said DNA construct comprising a DNA molecule which comprises SEQ ID N0:1 or nucleotides nos. 20-70 and 207-1560 thereof; or nucleotides nos. 20-70 and 207-1563 thereof; or nucleotides nos. 65-70 and 207-1560 thereof; or nucleotides nos. 65-70 and 207-1563 thereof; or a DNA construct or molecule which is at least 70, 75, 80, 85, 90, 95 % homologous to either of these nucleotide sequences; homology meaning similarity, preferably identity, and being determined using computer programs known in the art such as GAP provided in the GCG program package (Program Manual for the Wisconsin Package, Version 8, August 1996, Genetics Computer Group, 575 Science Drive, Madison, Wisconsin, USA 53711) (Needleman, S.B. and Wunsch, CD., (1970) , Journal of Molecular Biology, 48, 443-453). Using GAP with the following settings for DNA sequence comparison: GAP creation penalty of 5.0 and GAP extension penalty of 0.3. The invention also relates to a DNA construct which hybridizes with any of the above DNA sequences under the conditions mentioned hereinabove.<br>
EXAMPLES<br>
Example 1<br>
Phytase activity assay (FYT)<br>
Phytase activity can be measured using the following assay:<br>
10 pi diluted enzyme samples (diluted in 0.1 M sodium acetate, 0.01 % Tween20, pH 5.5) are added into 250 pi 5 mM sodium<br><br>
phytate (Sigma) in 0.1 M sodium acetate, 0.01 % Tween20, pH 5.5 (pH adjusted after dissolving the sodium phytate; the substrate is preheated) and incubated for 30 minutes at 37°C. The reaction is stopped by adding 250 \il 10 % TCA and free phosphate is measured by adding 500 ^l 7.3 g FeS04 in 100 ml molybdate reagent (2.5 g (NH^) 6MO7O24.4H2O in 8 ml H2SO4 diluted to 250 ml). The absorbance at 750 nm is measured on 200 ]il samples in 96 well microtiter plates. Substrate and enzyme blanks are included. A phosphate standard curve is also included (0-2 mM phosphate). 1 FYT equals the amount of enzyme that releases 1 ]imol  phosphate/min at the given conditions.<br>
Example 2<br>
Test for specific activity<br>
The specific activity can be determined as follows:<br>
A highly purified sample of the phytase is used (the purity is checked beforehand on an SDS poly acryl amide gel showing the presence of only one component).<br>
The protein concentration in the phytase sample is determined by amino acid analysis as follows: An aliquot of the phytase sample is hydrolyzed in 6N HCl, 0.1% phenol for 16 h at 110 C in an evacuated glass tube. The resulting amino acids are quantified using an Applied Biosystems 420A amino acid analysis system operated according to the manufacturers instructions. From the amounts of the amino acids the total mass - and thus also the concentration - of protein in the hydrolyzed aliquot can be calculated.<br>
The activity is determined in the units of FYT. One FYT equals the amount of enzyme that liberates 1 micromol inorganic phosphate from phytate (5 mM phytate) per minute at pH 5.5, 37°C; assay described e.g. in example 1.<br><br>
The specific activity is the value of  FYT/mg enzyme<br>
protein.<br>
Example 3<br>
Test for temperature and pH activity and stability<br>
Temperature and pH activity and stability can be determined as follows:<br>
Temperature profiles (i.e. temperature activity relationship) by running the FYT assay of Example 1 at various temperatures (preheating the substrate).<br>
Temperature stability by pre-incubating the phytase in 0.1 M sodium phosphate, pH 5.5 at various temperatures before measuring the residual activity.<br>
The pH"Stability by incubating the enzyme at pH 3 (25 mM glycine-HCl) , pH 4-5 (25 mM sodium acetate), pH 6 (25 mM MES), pH 7-9 (25 mM Tris-HCl) for 1 hour at 40°C, before measuring the residual activity.<br>
The pH-profiles (i.e. pH activity relationship) by running the assay at the various pH using the same buffer-systems (50 mM, pH re-adjusted when dissolving the substrate)•<br>
Example 4<br>
DSC as a preferred test for thermostability<br>
The thermostability or melting temperature, Tm, can be determined as follows:<br>
In DSC the heat consumed to keep a constant temperature increase in the sample-cell is measured relative to a reference cell. A constant heating rate is kept (e.g. 90°C/hour). An endo-thermal process {heat consuming process - e.g. the unfolding of an enzyme/protein)  is observed as an increase in the heat<br><br>
transferred  to  the  cell  in  order  to  keep  the  constant temperature increase.<br>
DSC can be performed using the MC2-apparatus from MicroCal. Cells are equilibrated 20 minutes at 20 °C before scanning to 90°C at a scan rate of 90°/h. Samples of e.g. around 2.5 mg/ml phytase in 0.1 M sodium acetate, pH 5.5 are loaded.<br>
Example 5<br>
Phytase variants of amended activity characteristics<br>
Variants of an Aspergillus fumigatus model phytase (a wild type phytase derived from strain ATCC 13073) were prepared as described in EP 98104858.0 (EP-A-0897010) , examples 2-3 and 5, and the phytase activity was determined as described in example 7 thereof. pH- and temperature optimum and melting point was determined as described in examples 9 and 10 of EP 98113176-6 (EP-A-0897985).<br>
In Table 1, variants of improved specific activity at pH 5.0 are listed. Table 2 lists variants of improved relative activity at pH 3.0, and Table 3 lists variants of improved thermostability (temperature optimum, e.g. determined by DSC).<br><br><br><br><br><br>
Example 6<br>
Further phytase variants of amended activity characteristics<br>
Variants of the ascomycete consensus sequence "conphys" of Fig. 9 were prepared as described in EP 98113176.6 (EP-A^ 0897985), examples 4-8. Phytase activity, including pH- and temperature optimum, and melting point was determined as described in examples 9 and 10, respectively, thereof.<br>
The tables below list variants of amended activity characteristics, viz.<br>
Table 4 variants of improved specific activity at pH 6.0;<br>
Table 5 variants of amended pH optimum (the pH-optimum indicated is an approximate value, determined as that pH-value (selected from the group consisting of pH 4.0; 4.5; 5.0; 5.5; 6.0; 6.5; and 7;0) at which the maximum phytase activity was obtained) ;<br>
Table 6 a variant of improved thermostability (expressed by way of the melting point as determined by differential scanning calorimetry (DSC)); and<br>
JT-able 7 variants of amended thermostability (temperature optimum); a "+'^ or "-" indicates a positive or a negative, respectively, effect on temperature optimum of up to l^C; and a "++" and "- -" means a positive or a negative, respectively, effect on temperature optimum of between 1 and 3°C.<br><br><br><br><br><br><br><br>
Example 7<br>
Cloning of a phytase of Cladorrhinum foecundissimum<br>
DNA encoding a phytase from Cladorrhinum foecundissimum CBS 427.97 has been cloned, and the enzyme isolated and purified, essentially as described in WO 98/28409.<br>
Fig. 15 shows the DNA sequence of the Hindlll/Xbal cloned PCR product in pA2phy8. The cloned PCR product is amplified from the genomic region encoding Cladorrhinum foecundissimum CBS 427,97 phyA gene. The putative intron is indicated by double underline of the excision-ligation points in accordance with the GT-AG rule (R. Breathnach et al. Proc. Natl. Acad. Sci. USA 75 (1978) pp4853-4857). The restrictions sites used for cloning are underlined.<br>
According to the SignalP VI. 1 prediction (Henrik Nielsen, Jacob Engelbrecht, Stren Brunak and Gunnar von Heijne: "Identification of prokaryotic and eukaryotic signal peptides and prediction of their cleavage sites," Protein Engineering 10, 1-6 (1997)), the signal peptide part of the enzyme corresponds<br><br>
to amino acids nos. 1-15, accordingly the mature enzyme is amino acids nos. 16-495.<br>
The enzyme exhibits a pH optimum around pH 6 with no activity at the low pH (pH 3), but significant activity up until pH 7.5; thus it is a more alkaline phytase as compared to the Aspergillus ficuum phytase.<br>
A temperature optimum around 60°C was found at pH 5.5. Thus, this phytase is more thermostable than the A. ficuum phytase.<br>
Example 8<br>
Alignment: of a new model phytase according to Fig. 1<br>
The phytase sequence of Cladorrhinum foecundissimum as disclosed in Example 7 is compared with the 13 model phytases of Fig, 1 using GAP version 8 referred to above with a GAP weight of 3.000 and a GAP lengthweight of 0.100. Complete amino acid sequences are compared. The M_thermophila phytase sequence turns up to be the most homologous sequence, showing a degree of similarity to the C. foecundissimum sequence of 70.86%.<br>
Still using the GAP program and the parameters mentioned above, the phytase sequence "C_foecundissimum" is now aligned to the "M-thermophila" phytase - see Fig. 16. The average match is 0.540;, the average mismatch -0.396; quality 445.2; length 505; ratio 0.914; gaps 9; percent similarity 70.860; percent identity 53.878.<br>
In a next step, see Fig, 17, the C_foecundissimum is pasted (or it could simply be written) onto the alignment of Fig. 1 as the bottom row, ensuring that those amino acid residues which according to the alignment at Fig. 16 are identical (indicated by a vertical line) or similar (indicated by one or two dots) are placed above each other. At 5 places along the sequence,  the C_foecundissimum sequence comprises<br><br>
"excess" amino acid residues, which the alignment of Fig. 1 does not make room for. At Fig. 17, these excess residues are transferred onto a next row (but they can be included in the multiple alignment and nun±&gt;ered as described previously in the position numbering related paragraphs (using the denotations a, b, c etc,).<br>
Corresponding variants of the phytase of C_foecundissimum are then easily deduced on the basis of Fig. 17, Some examples: The variants generally designated "80K,A" and "43T" in C_foecundissimum correspond to "K80A" and "Q43T," respectively.<br><br><br><br><br>
CLAIMS<br>
1.   A  genetically  engineered  phytase  which,  when  aligned<br>
according to Fig. 1, is amended as compared to<br>
(i)      either of the phytases of Fig. 17; or<br>
(ii)     either of the phytases mentioned at p. 13, line 29 to p.<br>
14, line 3; or (iii)     any phytase being at least 40% identical to amino acid<br>
sequence nos. 38-403 of either of the phytases of Fig.<br>
1; (iy)      any phytase comprising an amino acid sequence encoded by<br>
a DNA sequence which hybridizes under medium stringency<br>
conditions to a DNA sequence encoding either of the<br>
phytases of Fig. 1; in at least one of the following positions, using the position numbering corresponding to P_lycii:<br>
24; 27; 31; 33; 39; 40; 41; 42; 43; 44; 45; 46; 47; 49; 51; 56; 58; 59; 61; 62; 68; 69; 70; 71; 72; 73; 74; 75; 76; 77; 78; 79; 80; 81; 82; 83; 84; 88; 90; 102; 115; 116; 117; 118; 119; 120; 121; 122; 123; 124; 125; 126; 127; 128; 132; 143; 148; 149; 150; 151; 152; 153; 154; 155; 156; 157; 158; 159; 160; 161; 162; 163; 170f; 170g; 171; 172; 173; 184; 185; 186; 187; 187a; 190; 191; 192; 193; 194; 195; 198; 199; 200; 201; 201a; 201b; 201c; 201d; 201e; 201f; 202; 203; 203a; 204; 205; 211; 215; 220; 223; 228; 232; 233; 234; 235; 236; 237; 238; 239; 242; 243; 244; 246; 251e; 253; 256; 260; 264; 265; 267; 270; 271; 272; 273; 274; 275; 276; 277; 278; 279; 280; 283; 285; 287; 288; 292; 293; 302; 304; 332; 333; 334; 335; 336; 337; 338; 339; 340; 341; 342; 343; 348; 349; 352; 360; 362; 364; 365; 366; 367; 368; 369; 370; 371; 372; 373; 374; 375; 376; 383k; 387; 393; 394; 396; 404; 409; 411; 412; 413; 417; 421; 431.<br><br>
2.	The genetically engineered phytase of claim 1 which comprises<br>
at least one of the following amendments:<br>
24C; 27P; 31Y; 33C; 39H,S,Q; 40L,N; 42S,G;<br>
43A,C,D,E,F,G,H,I,K,L,M,N,P,Q,R,S,T,V,W,Y; 44N; 45D,S; 47Y,F; 49P; 51E,A,R; 56P; 58D,K,A; 59G; 61R; 62V,I; 69Q; 75W,F; 78D,S; 79G; 80K,A; 81A,G,Q,E; 82T; 83A,I,K,R,Q; 84I,Y,Q,V; 881; 90R,A; 102Y; 115N; 116S; 118V,L; 119E; 120L; 122A; 123N,Q,T; 125M,S; 126H,S,V; 127Q,E,N; 128A,S,T; 132F,I,L; 143N; 148V,I; 151A,S; 152G; 153D,Y; 154D,Q,S,G; 157V; 158D,A; 159T; 160A,S; 161T,N; 162N; 163W; 170fH; 170gA; 171N; 172P; 173Q,S; 184Q,S,P; 185S; 186A,E,P; 187A; 187aS; 190A,P; 193S; 194S,T; 195T,V,L; 198A,N,V; 200G,V; 201D,E; 201a(); 201b(); 201C(); 201d(); 201e {) ; 201f () ; 201eT; 202S,A; 203R,K,S; 203aV,T; 204Q,E,S,A,V; 205E; 211L,V; 215A,P; 220L,N; 223H,D; 228N; 232T; 233E; 235Y,L,T; 236Y,N; 237F; 238L,iyi; 242P,S; 244D; 246V; 251eE,Q; 253P; 256D; 260A,H; 264R,I; 265A,Q; 267D; 270Y,A,L,G; 271D,N; 273D,K; 275F,Y; 278T,H; 280A,P; 283P; 287A,T; 288L,I,F; 292F,Y; 293A,V; 302R,H; 304P,A; 332F; 336S; 337T,G,Q,S; 3381; 339V,I; 340P,A; 343A,S,F,I,L; 348Y; 349P; 352K; 360R; 362P; 364W,F; 365V,L,A,S; 366D,S,V; 367A,K; 368K; 3691,L; 370V; 373A,S; 374S,A; 375H; 376M; 383kQ,E; 387P; 393V; 396R; 404A,G; 409R; 411K,T; 412R; 417E,R; 421F,Y; 431E.<br>
3.	A genetically engineered phytase which, when	aligned<br>
according to Fig. 1, is amended as compared to the	phytase<br>
A_ficuum_NRRL3135, and which comprises at least one	of the<br>
following amendments:<br>
R24C; V27P; H39Q,S; L40N; G42S;<br>
Q43A,C,D,E,F,G,H,I,K,L,M,N,P,R,S,T,V,W,Y; Y44N; A45D,S; F47Y; S49P; A51E,R; V56P; A58D,K; V62I; S69Q; Y75W,F; D78S; S79G; K80A; G81A,Q,E; K82T; K83A,I,R,Q; Y84Q,I,V; E90R,A; D115N; D116S; T118V,L; P119E; F120L; E122A; Q123N,T; L125S,M ; V126H,S; N127Q,E; S128A,T; F132I,L; I148V; S151A; S153D,Y; S154Q,D,G; I157V; A158D; S159T:  G160A,S;  K161T,N;  K162N;  F163W;  R170fH;  Q171N;  G173Q,S;<br><br>
S184P,Q; E185S; A186E,P; S187A; T190P,A; P193S; G194S,T; T195V,L; V198A,N; E200G,V; D201E; S201d{); E201e(),T; L201f(); preferably all three deletions; A202S; D203R,K,S; D203aV,T/ V204Q,E,S,A; T211L,V; S215AP; L220N; D223H; T228N; T235Y,L; Y236N; L237F; M238L; S242P; I246V; K251eE,Q; H260A; I264R; N265Q,A; Q270Y,A,L,G; S271D,N; K273D; Y275F; H27BT; A280P; T2B7A; Q2B8L,I,F; Y292F; A293V; H302R; P304A; N336S; G337S,T,Q; I339V; S340P,A; F343A,S,F,I,L; N349P; N360R; T362P; F364W; S365V,L,A; S366D,V; A367K; W368K; T369I,L; A373S: S374A; R375H; L376M; Q383kE; P404A,G; T4ilK; R417E; F421Y; A431E.<br>
4.   A  genetically  engineered  phytase  which,  when  aligned 'according to Fig.  1,  is amended as compared to the phytase<br>
A_fumigatus, and which comprises at least one of the following<br>
amendment s:<br>
A24C; V27P; H39,S,Q; L40N; G42S; Q43C,D,E,F,H,K,M,P,R,S,W,Y; Y44N;<br>
S45D;  F47Y; S49P/ E51A,R; L56P; K58D,A; D59G; I62V; S69Q; Y75W,F;<br>
S78D;  S79G;  K80A;  S81A,G,Q,E; K82T;  K83A,I,Q,R; Y84Q,V,I; V88K;<br>
A90R; F102Y; D115N/ D116S; T118V,L; P119E; F120L; E122A; Q123N,T;<br>
L125S,M; V126H,S; N127Q,E; S128A,T; F132,I,L; S143N; I148V; S151A;<br>
S153D,Y; D154Q,S,G; I157V; A158D; S159T; G160A,S; E161T,N; K162N;<br>
F163W; G170fH;  {)171N; N173Q,S; T172P; P184Q,S; E185S; S186A,E,P;<br>
E187A;  T187aS/  T190P,A;  G194S,T;  V195L,T;  K198A,N,V;  E200G,V; 'A201D,E;  S201d();  Q201e(),T;  L201f();  preferably    all  three<br>
deletions;  G202S,A;  D203R,K,S;  E203aV,T;  V204Q,E,S,A;  A205E;<br>
L211V; A220L,N; H223D; T228N; E232T; D233E; V235Y,L,T; V236Y,N;<br>
L237F;  M238L;  C242P,S;  T246V;  Q251eE,Q;  Q256D;  H260A;  K264R,I;<br>
K265Q,A; N267D;   Q270Y,A,L,G;  S271D,N; G273D,K;  Y275F; Y278T,H;<br>
A280P;  A287T;  Q288L,I,F;  F292Y;  T293A,V;  R302H;  P304A;  F332F;<br>
N336S;  S337T,G,Q;  M338I;  V339I;  S340P,A;  F343A,S,I,L;  N349P;<br>
E352K;  S360R;  K362P;  Y364W,F;  S365V,L,A;  A366D,V,S;  S367A,K;<br>
W368K; V369I,L; G373S,A; R375H; A376M; K383kQ,E; D404A,G; K411T;<br>
I393V; L412R; K417E,R; W421F,Y; G431E.<br><br>
5.	A genetically engineered phytase which, when aligned<br>
according to Fig, 1, is amended as compared to the phytase<br>
consphyA, and which comprises at least one of the following<br>
amendments:<br>
G24C; V27P; H39S,Q; L40N; G42S;<br>
Q43A,C,D,E,F,G,H,I,K,L,M,N,P,Q,R,S,T,V,W,Y; Y44N; S45D; Y47F; S49P; E51A,R; V56P; D58K,A; D59G; V62I; S69Q; Y75W,F; S78D; S79G; K80A; S81A,G,Q,E; K82T; A83I,Q,K,R; Y84,Q,I,V; A90R; D115N; D116S; T118V,L; Fli9E; P120L; E122A; N123Q,T; M125S; V126H,S; N127Q,E; S128A,T; Y132F,I,L; K143N; I148V; S151A; S153D,Y; D154Q,S,G; I157V; A158D; S159T; A160S; E161T,N; K162N; F163W; G170fH; S170gA; Q171N; H173Q,S; P184Q,S; E185S; G186A,E,P; S187A; G187aS; T190P,A; H193S; G194S,T; T195V,L; A198N,V; E200G,V; D201E; S201d(); E201e(),T; L201f(); preferably all three; G202S,A; D203R,K,S; D203aV,T; V204Q,S,A,E; L211V; A215P; L220N; D223H, T228N; E232T; D233E; V235Y,L,T; Y236N; L237F; M238L; P242S; E244D; E251e,Q; A256D; H260A; R264I; Q265A; Q270Y,A,L,G; S271D,N; G273D,K; Y275F; Y278T,H; A280P; A2B7T; Q288L,I,F; F292Y; A293V; R302H; P304A; N336S; S337T,Q,G; M338I; I339V; S340P,A; F343A,S,I,L; N349P; A352K; S360R; E362P; Y364W,F; S365V,L,A; A366D,V,S; S367K,A; W368K; T369I,L; G373S,A; A374S; R375H; A376M; Q383kE; A404G; K411T; E417R; F421Y; A431E.<br>
6.	A genetically engineered phytase which, when aligned<br>
according to Fig. 1, is amended as compared to the phytase<br>
A_pediades, -and which comprises at least one of the following<br>
amendments:<br>
V24C; F27P; ()31Y; F33C; D39H,S,Q; S40L,N; A42S,G;<br>
A43C,D,E,F,G,H,I,K,L,M,N,P,Q,R,S,T,V,W,Y; Y44N; T45D,S; Y47F; Q51E,A,R; K58D,A; K61R; I62V; ' F75W; S78D; A80K; G81A,Q,E; R83A,I,Q,K; .I84Y,Q,V; V88I; K90R,A; L102Y; D115N; D116S; V118L; P119E;   F120L;  L123N,T,Q;  S125M;   S126H,V;   Q127E,N;  A128S,T;<br><br>
T132F,I,L; E143N; V148I; S151A; S152G; S153D,Y; N154D,Q,S,G; D158A; S159T; A160S; T161N; ()170fH; ()170gA; ()171N; H173Q,S; H172P; S184Q,P; E185S; S186A,E,P; L187A; ()187aS; T190P,A; D193S; A194S,T; M195T,V,L; N198A,V; G200V; S201D,E ()201eT; S202A; D203R,K,S; P203aV,T; Q204E,S,A,V; T205E; I211L,V; P215A; L220N; Q223D,H; A232T; D233E; S235Y,L,T; N236Y; L237F; I238L,M; A242P,S; E244D; I246V; ()251eE,Q; N256D; P260A,H; A264R,I; Q265A; E267D/ G270Y,A,L; L332F; D271N; D273K; F275Y; T278H; Y280A,P; Y283P; V287A,T; Q288L,I,F; Y292F; I293A,V; E302R,H; P304A; L332F; N336S; Q337T,S,G; M338I; I339V; A340P; S343A,F,I,L; F348Y; N349P; S352K; P360R; R362P; W364F; V365L,A,S; T366D,V,S; S367K,A; R368K; L369I; T370V; S373A; A374S; R375H; S383kQ,E; T387P; A396R; G404A; L409R; T411K; L412R; E417R; Y421F.<br>
7. A genetically engineered phytase which, when aligned according to Fig, 1, is amended as compared to the phytase P_lycii, and which comprises at least one of the following amendments:<br>
F24C; V27P; L31Y; I33C; S39H,Q; N40L; G42S; P43A,C,D,E,F,G,H,I,K,L,M,N,Q,R,S,T,V,W,Y; Y44N; D45S; F47Y; E51A,R; E58D,K,A; T61R; V62I; W75F; S78D; A80K; R81Q,E,G,A; S82T; R83A,I,Q,K; Q84Y,V,I; V88I; K90R,A; A115N; D116S; L118V; P119E; F120L; N123T,Q; S125M; H126S,V; Q127E,N; T128A,S; M132F,I,L; 'G143N; V148I; A151S; D153Y; Q154D,S,G; D158A; S159T; S160A; T161N; ()170fH; ()170gA; S171NG172P; E173Q,S; Q184S,P; E185S; E186A,P; G187A; ()187aS; T190P,A; N193S; N194S,T; M195T,V,L; N198A,V; V200G; D201E; ()201eT; G202S,A; D203R,K,S; ()203aV,T; E204Q,S,A,V; S205E; V211L; N215A,P; L220N; A223D,H; S232T; D233E; L235Y,T; T236Y,N; L237F; M238L; P242S; L246V; ()251eE,Q; A260H; V264R,I; S265Q,A; E267D; Y270A,L,G; D271N; D273K; F275Y; G278T,H; P280A; A283P; T287A; Q288L,I,F; Y292F; V293A; G302R,H; A304P; N336S; T337Q,S,G; M338I; V339I; P340A; A343S,F,I,L; F348Y; N349P; A352K; E360R;  R362P; W364F;  V365L,A,S; D366V,S;  S367K,A;  L369I;  S373A;<br><br>
G374A,S; ()383kQ,E; E387P; A396R; G404A; V409R; E411K,T; L412R; E417R; Y421F; A431E.<br>
8.	A genetically engineered phytase which, when aligned<br>
according to Fig. 1, is amended as compared to the phytase<br>
P_involtus_Al, and which comprises at least one of the following<br>
amendment s:<br>
()24C; T27P; F31Y; I33C; R39H,S,Q; N40L; S42G;<br>
P43A,C,D,E,F,G,H,I,K,L,M,N,Q,R,S,T,V,W,Y; Y44N; S45D; Y47F; A51E,R; A58D;K; Q61R; I62V; F75W; S78D; A80K; T81Q,E,G,A; R83A,I,Q,K; I84Y,Q,V; L88I; K90R,A; F102Y; S115N; D116S; V118L; P119E; F120L; A123N,T,Q; S125M; F126H,S,V/ D127Q,E,N; A128T,S; A132F,I,L; I148V; D151A,S; S153D,Y; D154Q,S,G; D158A; S159T; A160S; T161N; ()170fH; ()170gA; S171N; H172P; N173Q,S; P184Q,S; Q185S; T186A,E,P; G187A; ()187aS; T190P,A; D193S; N194S,T; M195T,V,L; A198N,V; G200V; D201E; ()201eT; S202A; D203R,K,S; P203aV,T; Q204E,S,A,V; V205E; V211L; S215A,I; L220N; A223D,H; D233E; F235Y,L,T; N236Y; L237F; V238L,M; A242P,S; M244D; ()251eE,Q; D253P/ T256D; P260A,H; E264R,I; A265Q; A267D; G270Y,A,L; D271N; D273K; F275Y; T278H/ Y280A,P; E283P; V287A,T; Q288L,I,F; Y292F; V293A; N302R,H; A304P; N336S; L337T,Q,S,G; M 3381; V339I; A340P; S343A,F,I,L; F348Y; R349P; A352K; P360R; R362P; W364F; R365V,L,A,S; T366D,V,S; S367K,A; S368K; L369I; S373A; G374A,S; R375H; ()383kQ,E; T387P; Q396R; G404A; L409R; T411K; L412R; E417R; F421Y.<br>
9.	A genetically engineered phytase which, when aligned<br>
according to Fig. 1, is amended as compared to the phytase<br>
P_involtus_A2, and which comprises at least one of the following<br>
amendments:<br>
P24C; I27P; F31Y; I33C; R39H,S,Q; N40L; S42G;<br>
P43A,C,D,E,F,G,H,I,K,L,M,N,Q,R,S,T,V,-W,Y;  Y44N;  S45D; Y47F; A51E,R;   A58D,K;   E61R;   I62V;   F75W; .  S78D;  A80K;  AB1Q,E,G;<br><br>
R83A,I,Q,R,K; I84Y,Q,V; L88I; K90R,A; F102y; S115N; D116S; V118L; P119E; F120L; A123N,T,Q; S125M; F126H,S,V; D127Q,E,N; A128T,S; V132F,I,L; D143N; I148V; D151A,S; S153D,Y; D154Q,S,G; D158A; A160S; T161N; ()170fH; ()170gA; S171N; R172P; N173Q,S; P184Q,S; Q185S; T186A,E,P; G187A; ()187aS; T190P,A; D193S; N194S,T; M195T,V,L; A198N,V; G200V; E201D; ()201eT; S202A; D203R,K,S; P203aV,T; Q204E,S,A,V; V205E; S211L,V; S215A,P; L220N; A223D,H; A232T; F235Y,L,T; N236Y; L237F; V238L,M; P242S; M244D; ()251eE,Q; D253P; T256D; P260A,H; E264R,I; A265Q; A267D; G270Y,A,L; D271N; D273K; F275Y; T278H; Y280A,P; A2B3P; V287A,T; Q288L,I,F; Y292F; I293A,V; N302R,H; A304P; N336S; L337T,Q,S,G; M338I; V339I; 340P,A; A343S,F,I,L; F348Y; R349P; A352K; P360R; R362P; W364F; L365V,A,S; T366D,V,S; S367K,A; S368K; V369I,L; S373A; R375H; ()383kQ,E; T387P; Q396R; G404A; L409R; A411K,T; L412R; E417R; Y421F.<br>
10.  A  genetically  engineered  phytase  which,  when  aligned according to Fig.  1,  is amended as compared to the phytase T__pubescens, and which comprises at least one of the following amendments: . R24C; T27P; L31Y; V33C; Q39H,S; S40L,N; S42G;<br>
M43A,C,D,E,F,G,H,I,K,L,N,P,Q,R,S,T,V,W,Y; Y44N; S45D; Y47F; A51E,R; A58D,K; S59G; Q61R; I62V; F75W; S78D; A80K; A81Q,E,G; R83A,I,Q,K; I84Y,Q,V; V88I; K90R,A; L102Y; D115N; V118L; T123N,Q; S125M; S126H,V; E127Q,N; A128T,S; A132F,I,L; D143N; V148I; S151A; S153D,Y; D154Q,S,G; A158D; A160S; N161T; ()170fH; ()170gA; S17,1N; S172P; N173Q,S; S184Q,P; E185S; A186E,P; Gia7A; ()187aS; T190P,A; N194S,T; M195T,V,L; A198N,V; G200V; ()201eT; S202A; D203R,K,S; P203aV,T; Q204E,S,A,V; V205E; Q211L,V; P215A; L220N; G223D,H; D233E; Y235L,T; N236Y; L237F; L238M; P242S; E244D; ()251eE,Q; E253P; Q260A,H; D264R,I; A265Q; A267D; A270Y,L,G; D271N; D273K; F275Y; T278H; Y280A,P; V287A,T; Q288L,I,F; Y292F; I293A,V; A302R,H; N304P,A; N336S; Q337T,S,G; M338I; V339I; A340P; S343A,F,I,L; F348Y; N349P; A352K; P360R.; R362P; F364W; L365V,A,S;<br><br>
V366D,S; K367A; I369L; A373S; A374S; R375H; ()383kQ,E; Q387P; A396R; G404A; V409R; T411K; L412R; E417R; Y421F.<br>
11.	A genetically engineered phytase which, when aligned<br>
according to Fig. 1, is amended as compared to the phytase<br>
A_nidulans, and which comprises at least one of the following<br>
amendments:<br>
V24C; A27P; H39S,Q; V40L,N; G42S;<br>
Q43A,C,D,E,F,G,H,I,K,L,M,N,P,R,S,T,V,W,Y;  Y44N;  S45D;  Y47F;<br>
S49P; E51A,R; V56P; ■H58D,K,A; E61R; V62I;  S69Q; Y75W,F; E78D,S;<br>
S79G; K80A;  S81Q,E,A,G; K82T; A83I,Q,K,R; Y84Q,V,I; A90R; D115N;<br>
D116S;  T118V,L;  I119E;  F120L;  E122A;  N123T,Q;  M125S;  V126H,S; "01270,E,N;  .S128A,T;   F132I,L;  K143N;   I148V;   S151A;   S153D,Y;<br>
D154Q,S,G;  A158D;  S159T;  A160S;  E161T,N;  K162N;  F163W;  G170fH;<br>
S170gA;  ()171N; ()172P; K173Q,S; P184Q,S; E185S; I186A,E,P; D187A;<br>
G187aS;  T190P,A;  H193S;  S194T;  S198A,N,V;  E200G,V;  N201D,E;<br>
D201e(); E201e(),T; R201f() (a deletion of at least one of 201d,<br>
201e,   201f,   preferably  all);   A202S;   D203R,K,S;   E203aV,T;<br>
I204Q,E,S,A,V; I211L,V; P215A; L220N; D223H; K228N; E232T; N233E;<br>
I235Y,L,T;  Y236N;  L237F;  M238L;  S242P;  M246V;  E251eQ;  A256D;<br>
E260A,H;  L264R,I; Q270Y,A,L,G; S271D,N;  S273D,K; Y275F; G278T,H;<br>
A280P; A287T;  Q288L,I,F;  F292Y;  T293A,V;  Q302R,H;  P304A; N336S;<br>
S337T,Q,G;  M338I;  I339V;  S340P,A;  F343A,S,I,L;  N349P;  Q352K; 'S360R;  Q362P;  Y364W,F;  A365V,L,S;  A366D,V,S;  S367K,A;  W368K;<br>
T369I,L;  G373S,A;  A374S;  R375H;  A376M;  E383kQ;  A404G;  T411K;<br>
L412R; E417R; F421Y; K431E.<br>
12.	A genetically engineered phytase which, when aligned<br>
according to Fig. 1, is amended as compared to the phytase<br>
A_terreus, and which comprises at least one of the following<br>
amendments:<br>
G24C; V27P; H39S,Q; K40L,N; G42S;<br><br>
L43A,C,D,E,F,G,H,I,K,M,N,P,Q,R,S,T,V,W,Y; Y44N; A45D,S; Y47F; S49P; Q51E,A,R; V56P; P58D,K,A; D59G; H61R; I62V; A69Q; S75W,F; H78D,S; S79G; K80A; T81Q,E,A,G; A83I,Q,K,R; Y84Q,V,I; A90R; E115N; E116S; T118V,L; P119E; F120L; R122A; N123T,Q; L125S,H; R126H,S,V; D127Q,E,N; L128A,T,S; F132I,L; H143N; V148I; T151A,S; D152G; A153D,Y; S154D,Q,G; H157V; E158D,A; S159T; A160S; E161T,N; K162N; ■ F163W; H173Q,S; P184Q,S; E1B5S; G186A,E,P; S187A; A187aS; T190P,A; H193S; S194T; L195T,V; A198N,V; E200G,V; S201D,E; S201d(); T201e(); V201f(); G202S,A; D203R,K,S; D203aV,T; A204Q,E,S,V; V205E; V211L; A215P; 'L220N; D223H; Q228N; D232T; D233E; V235Y,L,T; N236Y; L237F; M238L; P242S; E244E; T251eE,Q; A260H; T264R,I; Q265A; N267P; L270Y,A,G; S271D,N; K273D; Y275F; H278T; G280A,P; V287A,T; Q288L,I,F; W292F,Y; A293V; Q302H; P304A; N337T,Q,S,G; L338I; V339I; S340P,A; W343A,S,F,I,L; N349P; A352K; S360R; S362P; Y364W,F; A365V,L,S; A366D,V,S; A367K; W368K; T369I,L; A373S; A374S; R375H; A376M; R383kQ/E; P404A,G; K411T; A417E,R; F421Y; A431E.<br>
13.  A  genetically  engineered  phytase  which,  when  aligned according to Fig.  1,  is amended as compared to the phytase T_thermo,  and which comprises at least one of the following amendments: H24C; V27P; 'H39S,Q; S40L,N; G42S;<br>
Q43A,C,D,E,F,G,H,I,K,L,M,N,P,R,S,T,V,W,Y; Y44N; S45D; F47Y; S49P; A51E,R; V56P; Q58D,K,A; N59G; K61R; I62V; Y75W,F; S78D; S79G; K80A; T81Q,E,A,G; E82T; L83A,I,Q,R,K; Y84Q,V, I; R90A; D116S/ T118V,L; P119E; F120L/ E122A; N123T,Q; M125S; I126H,S,V; Q127E,N; L128A,T,S; •F132I,L; V148I; S151A; S153D,Y; D154Q,S,G; I157V; A158D; S159T; G160A,S; R161T,N; L162N; F163W; S170gA; D171N; K172P; H173Q,S; E184Q,S,P; E185S; G186A,E,P; D187A; T190P,A; T193S; G194S,T; S195T,V,L; V198A,N; E200G,V; D201E; S201d(); S201e(),T; S201f(); G202S,A; H203R,K,S; D203aV,T; A204Q,E,S,V; Q205E;  Q211L,V;  A215P;  I220N,L;  H223D;  D228N;  S232T;  D233E;<br><br>
P235Y,L,T; Y236N; M237F; D238L,M; P242S; E244D; L246V;  ()251eE,Q;<br>
A256D;	Q260A,H;  Q264R,I;  A265Q;  Q270Y,A,L,G;  S271D,N;  G273D,K;<br>
Y275F;	N278T,H;  G280A,P; A287T; Q288L,I,F;  F292Y;  V293A; H302R;<br>
P304A;	N336S;  T337Q,S,G;  M338I;  T339V,I;  S340P,A;  A343S,F,I,L;<br>
N349P;	A352K; S360R; E362P; Y364W,F; S365V,L,A; A366D,V,S; A367K;<br>
W368K;	T369I,L; G373S,A; G374A,S; R375H; A376M; D383kQ,E; E404A;<br>
K411T;	R417E; F421Y.<br>
14.	A genetically engineered phytase which, when aligned<br>
according to Fig. 1, is amended as compared to the phytase<br>
T_lanuginosa, and which comprises at least one of the following<br>
amendments:<br>
K24C; ()27P; ()31Y; ()33C; R39H,S,Q; H40L,N; G42S;<br>
Q43A,C,D,E,F,G,H,I,K,L,M,N,P,R,S,T,V,W,Y; Y44N; S45D; F47Y; S49P; A51E,R; V56P; K58D,A; V62I; S69Q; Y75W,F; A78D,S; H79G; K80A; S81Q,E,A,G; E82T; V83A,I,Q,K,R; Y84Q,V,I; L88I; R90A; F102Y; D115N; N116S; T118V,L; R119E; F120L; E122A; E123N,T,Q; M125S; iyil26H,S,V; E127Q,N; S128A,T; F132I,L; E143N; V148I; A151S; S153D,Y; A154D,Q,S,G; I157V; A158D; S159T; A160S; E161T,N; F162N; F163W; R170fH; S170gA; K172P; D173Q,S; S184Q,P; E185S; E186A,P; T187A; G187aS; T190P,A; G193S; L194S,T; T195V,L; A198N,V; E200G,V; E201D; A201d(); P201e(),T; D202S,A; P203R,K,S; T203aV; Q204E,S,A,V; P205E; V211L; R215A,P; I220L,N; H223D; E232T; D233E; P235Y,L,T; L236Y,N; M238L; P242S; Q251eE; H256D; Q260H; M264R,I; A265Q; Y270A,L,G; T271D,N; D273K; Y275F; H278T; G280A,P; A283P; S287A; R288L,I,F; F292Y; V293A; G302R,H; P304A; N336S; T337Q,S,G; M338I; T339V,I; G340P,A; S343A,F,I,L; N349P; P360R; T362P; Y364W,F; A365V,L,S; A366D,V,S; S367K,A; W368K; T369I,L; A373S; A374S; R375H; A376M; E383kQ; R404A,G; R411K,T; K417E,R; F421Y; D431E.<br>
15.	A genetically engineered phytase which, when aligned<br>
according to Fig. 1,     is amended as compared to the phytase<br><br>
M_thermophila, and which comprises at least one of the following<br>
amendments:<br>
S24C; F31Y; H39S,Q; F40L,N; G42S;<br>
Q43A,C,D,E,F,G,H,I,K,L,M,N,P,R,S,T,V,W,Y; Y44N; S45D; Y47F; S49P; P51E,A,R; I56P; D58K,A; D59G; E61R; V62I; S69Q; A75W,F; L78D,S; K79G; R80K,A; A81Q,E,G; A82T; S83A,I,Q,K,R; Y84Q,V,I; R90A; D115N; E116S; T118V,L; R119E; T120L; Q122A; Q123N,T; M125S; V126H,S; N127Q,E; S128A,T; F132I,L; K143N; V148I; A151S; Q153D,Y; D154Q,S,G; H158D,A; S159T; A160S; E161T,N; G170fH; S170gA; T171N; F163W; V172P; R173Q,S; P184Q,S; E185S; T186A,E,P; G187aS; T190P,A; N193S; D194S,T; L195T,V; A198N,V; E200G,V; E201D; G201a(); P201b(); Y201c(); S201d()-; T201e(); I201f(); G202S,A; D203R,K,S; 'D203aV,T; A204Q, E, S, V; Q205E; T211L,V; P215A; V220N,L; N223D,H; A232T; D233E; V235Y,L,T; A236Y,N; L237F; M238L; P242S; E244D; A251eE,Q; R256D; E260A,H; R264I; A265Q; Q270Y,A,L,G; S271D,N; K273D; Y275F; Y278T,H; P280A; T287A; Q288L,I,F; F292Y; V293A; ()302R,H; P304A; N336S; D337T,Q,S,G; M338I; M339V,I; G340P,A; G343A,S,F,I,L; D349P; P352K; D360R; E362P; Y364W,F; A365V,L,S; A366D,V,S; S367K,A; W368K; A369I,L; A373S; A374S; R375H; I376M; E383kQ; E387P; G404A; M409R; T411K; L412R; E417R; F421Y; D431E.<br>
16.	A polypeptide having phytase activity and which comprises<br>
(i)      SEQ ID NO: 2 or the mature part thereof; or<br>
Kii)     amino acids nos. 16-495 of SEQ ID NO: 2; or (iii)    an amino acid sequence of an identity of at least 70% to either of (i) or (ii) .<br>
17.	A  DNA  construct  encoding  a  polypeptide  having  phytase<br>
activity, said DNA construct comprising<br>
(i)	SEQ ID NO: 1; or<br>
(ii)	nucleotides nos.	20-70 and 207-1560	of	SEQ ID NO:	1;	or<br>
(iii)	nucleotides nos.	20-70 and 207-1563	of	SEQ ID NO:	1;	or<br>
(iv)	nucleotides nos.	65-70 and 207-1560	of	SEQ ID NO:	1;	or<br><br>
(v)       nucleotides nos, 65-70 and 207-1563 of SEQ ID NO: 1; or (vi)      a DNA molecule having at 70% identity to either of the nucleotide sequences of (i)-(v).<br>
18.	A DNA construct comprising a DNA sequence encoding a<br>
genetically engineered phytase according to any one of claims 1-<br>
15.<br>
19.	A recombinant expression vector which comprises a DNA<br>
construct according to claim 18,<br>
20.	A host cell which is  transformed with a DNA construct<br>
according to claim 18 or a vector according to claim 19.<br>
21.	A process for preparing a genetically engineered phytase, the process comprising culturing the host cell according to claim 2 0 under conditions permitting the production of the genetically engineered phytase, and recovering the phytase from the culture broth.<br>
22.	A feed or food comprising at least one genetically engineered phytase of any of claims 1-15.<br>
23.  A process for preparing a feed or food according to claim 22, wherein the . at least one genetically engineered phytase is added to the food or feed components.<br>
24.	A composition comprising at least one genetically engineered<br>
phytase of any of claims 1-15.<br>
25.	The composition according to claim 24 suitable for use in<br>
food or feed preparations.<br><br>
26.	The composition according to any of claims 24-2 5 which is an animal feed additive.<br>
27.	A process for reducing phytate levels in animal manure comprising feeding an animal with an effective amount of the feed according to claim 22 or obtainable according to claim 23.<br>
28.	Use of the genetically engineered phytase of any of claims 1-15; or the composition of  any of claims 24-26 for liberating phosphorous from a phytase substrate.<br>
29.	A transgenic plant or plant part which is capable of expressing a genetically engineered phytase according to any one of claims 1-15.<br><br>
30.	A   genetically   engineered   phytase   substantially   as   hereinbefore<br>
described with reference to the accompanying drawings.<br>
31.	A polypeptide having phytase activity substantially as hereinbefore<br>
described with reference to the accompanying drawings.<br>
32.	A DNA construct substantially as hereinbefore described with reference to the accompanying drawings.<br>
33.	A recombinant expression vector substantially as hereinbefore described with reference to the accompanying drawings.<br>
34.	A process or food substantially as hereinbefore described with reference to the accompanying drawings.<br><br>
35.	A process for preparing a genetically engineered phytase substantially as hereinbefore described with reference to the accompanying drawings.<br>
36.	A feed or food substantially as hereinbefore described with reference to the accompanying drawings.<br>
37.A process for preparing a feed or food substantially as hereinbefore described with reference to the accompanying drawings.<br>
38.A composition substantially as hereinbefore described with reference to the accompanying drawings.<br>
39.A process for reducing phytate levels in animal manure substantially as hereinbefore described with reference to the accompanying drawings.<br><br>
40. Use of the genetically engineered phytase substantially as hereinbefore described with reference to the accompanying drawings.<br>
4 LA transgenic plant or plant part substantially as hereinbefore described with reference to the accompanying drawings.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMDQwOC1jaGUgYWJzdHJhY3QgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2000-0408-che abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMDQwOC1jaGUgY2xhaW1zIGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-0408-che claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMDQwOC1jaGUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKSBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-0408-che description (complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMDQwOC1jaGUgZHJhd2luZ3MgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2000-0408-che drawings duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtNDA4LWNoZS1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-408-che-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtNDA4LWNoZS1hc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2000-408-che-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtNDA4LWNoZS1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-408-che-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtNDA4LWNoZS1jb3JyZXNwb25kZW5jZSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-408-che-correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtNDA4LWNoZS1jb3JyZXNwb25kZW5jZSBwby5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-408-che-correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtNDA4LWNoZS1kZXNjcmlwdGlvbiBjb21wbGV0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-408-che-description complete.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtNDA4LWNoZS1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-408-che-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtNDA4LWNoZS1mb3JtICAxOS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-408-che-form  19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtNDA4LWNoZS1mb3JtICA2LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2000-408-che-form  6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtNDA4LWNoZS1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-408-che-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtNDA4LWNoZS1mb3JtIDI2LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2000-408-che-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtNDA4LWNoZS1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-408-che-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtNDA4LWNoZS1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-408-che-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtNDA4LWNoZS1vdGhlciBkb2N1bWVudHMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-408-che-other documents.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtNDA4LWNoZS1wY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-408-che-pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtNDA4LWNoZS1wcmlvcml0eSBkb2N1bWVudHMgMS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-408-che-priority documents 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtNDA4LWNoZS1wcmlvcml0eSBkb2N1bWVudHMgMi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-408-che-priority documents 2.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="222277-embedding-supplemental-data-in-an-encoded-signal.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="222279-a-method-and-apparatus-for-transmitting-a-codeword-over-a-transmission-channel.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>222278</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2000/408/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>47/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Nov-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>05-Aug-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>20-Sep-2000</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVOZYMES A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>KROGSHOEJVEJ 36, DK-2880 BAGSVAERD,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SVENDSEN, Allan</td>
											<td>Novo Allé, DK-2880 Bagsvaerd,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>KOSTREWA, Dirk</td>
											<td>1 Peter-Thumb-Strasse, D-79100 Freiburg,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>LEHMANN, Martin</td>
											<td>258 Sayre Drive Princeton, NJ 08540,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>PASAMONTES, Luis</td>
											<td>8 Warfield Street Montclair, NJ 07043,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>TOMSCHY, Andrea</td>
											<td>45 Kraftwerkstrasse, D-79639 Grenzach-Wyhlen,</td>
										</tr>
										<tr>
											<td>6</td>
											<td>VAN LOON, Adolphus</td>
											<td>15 Waldhofstrasse CH-4310 Rheinfelden,</td>
										</tr>
										<tr>
											<td>7</td>
											<td>VOGEL, Kurt</td>
											<td>41 Eulerstrasse, CH-4051 Basel,</td>
										</tr>
										<tr>
											<td>8</td>
											<td>WYSS, Markus</td>
											<td>Rotackerstrasse, CH-4410 Liestal,</td>
										</tr>
										<tr>
											<td>9</td>
											<td>LASSEN, Soren, Flensted</td>
											<td>Lyngbyvej 19, DK-2100 Copenhagen,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A23K 1/165</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/DK1999/000153</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>1999-03-22</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0407/98</td>
									<td>1998-03-23</td>
								    <td>Denmark</td>
								</tr>
								<tr>
									<td>2</td>
									<td>PA 1998 00806</td>
									<td>1998-06-19</td>
								    <td>Denmark</td>
								</tr>
								<tr>
									<td>3</td>
									<td>PA 1998 01176</td>
									<td>1998-09-18</td>
								    <td>Denmark</td>
								</tr>
								<tr>
									<td>4</td>
									<td>PA 1999 00091</td>
									<td>1999-01-22</td>
								    <td>Denmark</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/222278-phytase-variants by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:42:55 GMT -->
</html>
