<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/222340-production-of-homotrimeric-fusion-proteins by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:38:49 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 222340:PRODUCTION OF HOMOTRIMERIC FUSION PROTEINS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">PRODUCTION OF HOMOTRIMERIC FUSION PROTEINS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to an isolated polypeptide, comprising (1) an extracellular domain of the transmembrane activator and CAML (calcium-signal modulating cyclophilin ligand) interactor (TACI), wherein the polypeptide sequence of TACI is SEQ ill NO:4, and (2) a trimerizing polypeptide wherein the presence of said trimerizing polypeptide allows the isolated polypeptide to form stable trimers.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>PRODUCTION OF HOMOTRIMERIC FUSION PROTEINS<br>
BACKGROUND OF THE INVENTION<br>
The tumor necrosis factor (TNF) receptor super family is a large family of molecules involved in host defense, inflammation, and autoimmunity, and have been implicated in human disease. Therapeutic agents aimed at inhibiting TNF are effective in controlling inflammatory diseases such as rheumatoid arthritis and inflammatory bowel disease. Additional members of the TNF/TNF receptor super family are currently being targeted for therapies against autoimmune disease, atherosclerosis, osteoporosis, allograft rejection and cancer.<br>
Although both TNF and TNF receptor family members are active as self-assembling trimers, only functionally dimeric molecules, such as antibodies or receptor-IgG fusions, have been used as therapeutic agents. The three-fold symmetry displayed by both ligands and receptors in the TNF super family indicate that trimeric receptor based antagonists should display an increased avidity and therefore, increased effectiveness compared to dimeric molecules.<br>
Accordingly, a need still exists for a simple method for expressing TNF ligands and TNF receptors as stable trimers.<br>
BRIEF SUMMARY OF THE INVENTION<br>
The present invention provides methods for producing trimeric TNF receptors that are more potent inhibitors of their cognate ligand's biological activities, when compared to dimeric receptor molecules.<br>
As described below, the present invention provides polypeptides that comprise: (1) an extracellular domain of the transmembrane activator and CAML (calcium-signal modulating cyclophilin ligand) interactor (TACI), and (2) a trimerizing polypeptide. Suitable TACI extracellular domains include: (1) amino acid residues 30 to 110 of SEQ ID NO:4, (2) amino acid residues 1 to 110 of SEQ ID NO:4, (3) amino acid residues 30 to 154 of SEQ ID NO:4, and (4) amino acid residues 1 to 154 of SEQ ID NO:4. Illustrative trimerizing polypeptides include the NC-1 fragment of human collagen X, and a trimerizing fragment of Heat Shock Binding Protein-1. The present invention further provides homotrimeric complexes of fusion proteins comprising a TACI extracellular domain and a trimerizing polypeptide.<br><br>
These and other aspects of the invention will become evident upon reference to the following detailed description and drawing. In addition, various references are identified below and are incorporated by reference in their entirety.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figure 1 shows the inhibition of zTNF4-induced luciferase activity by a TACI-Fc fusion protein ("TACI-IgG"), a TACI-HSBP-1 protein produced in mammalian cells ("TACI-HSBP (mamm)"), a TACI-HSBP-1 protein produced in E. coli ("TACI-HSBP (Ecoli)"), and by a control immunoglobulin fusion protein ("hwsxll-IgG").<br>
Figure 2 shows the inhibition of B-cell proliferation, through incorporation of 3H-thymidine, of the TACI-NC1 trimer and TACI-Fc fusion protein ("TACI-Fc5"). TACI-Fc5 is an alternative name for TACI-Fc or TACI-IgG. Also noted on this figure are the EC50 values (i.e., the concentration that inhibits the endpoint to 50% of the control) for the two molecules.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
1.        Definitions<br>
In the description that follows, a number of terms are used extensively. The following definitions are provided to facilitate understanding of the invention.<br>
As used herein, "nucleic acid" or "nucleic acid molecule" refers to polynucleotides, such as deoxyribonucleic acid (DNA) or ribonucleic acid (RNA), oligonucleotides, fragments generated by the polymerase chain reaction (PCR), and fragments generated by any of ligation, scission, endonuclease action, and exonuclease action. Nucleic acid molecules can be composed of monomers that are naturally-occurring nucleotides (such as DNA and RNA), or analogs of naturally-occurring nucleotides {e.g., Î±-enantiomeric forms of naturally-occurring nucleotides), or a combination of both. Modified nucleotides can have alterations in sugar moieties and/or in pyrimidine or purine base moieties. Sugar modifications include, for example, replacement of one or more hydroxyl groups with halogens, alkyl groups, amines, and azido groups, or sugars can be functionalized as ethers or esters. Moreover, the entire sugar moiety can be replaced with sterically and electronically similar structures, such as aza-sugars and carbocyclic sugar analogs. Examples of modifications in a base moiety include alkylated purines and pyrimidines, acylated purines or pyrimidines, or other well-<br><br>
known heterocyclic substitutes. Nucleic acid monomers can be linked by phosphodiester bonds or analogs of such linkages. Analogs of phosphodiester linkages include phosphorothioate, phosphorodithioate, phosphoroselenoate, phosphorodiselenoate, phosphoroanilothioate, phosphoranilidate, phosphoramidate, and the like. The term "nucleic acid molecule" also includes so-called "peptide nucleic acids," which comprise naturally-occurring or modified nucleic acid bases attached to a polyamide backbone. Nucleic acids can be either single stranded or double stranded.<br>
The term "complement of a nucleic acid molecule" refers to a nucleic acid molecule having a complementary nucleotide sequence and reverse orientation as compared to a reference nucleotide sequence. For example, the sequence 5' ATGCACGGG 3' is complementary to 5' CCCGTGCAT 3',<br>
The term "contig" denotes a nucleic acid molecule that has a contiguous stretch of identical or complementary sequence to another nucleic acid molecule. Contiguous sequences are said to "overlap" a given stretch of a nucleic acid molecule either in their entirety or along a partial stretch of the nucleic acid molecule. For example, representative contigs to the polynucleotide sequence 5' ATGGAGCTT 3' are 5' AGCTTgagt 3' and 3' tcgacTACC 5'.<br>
The term "structural gene" refers to a nucleic acid molecule that is transcribed into messenger RNA (mRNA), which is then translated into a sequence of amino acids characteristic of a specific polypeptide. A "gene of interest" can be a structural gene.<br>
"Complementary DNA (cDNA)" is a single-stranded DNA molecule that is formed from an mRNA template by the enzyme reverse transcriptase. Typically, a primer complementary to portions of mRNA is employed for the initiation of reverse transcription. Those skilled in the art also use the term "cDNA" to refer to a double-stranded DNA molecule consisting of such a single-stranded DNA molecule and its complementary DNA strand. The term "cDNA" also refers to a clone of a cDNA molecule synthesized from an RNA template.<br>
An "isolated nucleic acid molecule" is a nucleic acid molecule that is not integrated in the genomic DNA of an organism. For example, a DNA molecule that encodes a growth factor that has been separated from the genomic DNA of a cell is an isolated DNA molecule. Another example of an isolated nucleic acid molecule is a chemically-synthesized nucleic acid molecule that is not integrated in the genome of an organism. A nucleic acid molecule that has been isolated from a particular species is smaller than the complete DNA molecule of a chromosome from that species.<br>
A "nucleic acid molecule construct" is a nucleic acid molecule, either single- or double-stranded, that has been modified through human intervention to contain<br><br>
segments of nucleic acid combined and juxtaposed in an arrangement not existing in nature.<br>
"Linear DNA" denotes non-circular DNA molecules having free 5' and 3' ends. Linear DNA can be prepared from closed circular DNA molecules, such as plasmids, by enzymatic digestion or physical disruption.<br>
A "promoter" is a nucleotide sequence that directs the transcription of a structural gene. Typically, a promoter is located in the 5' non-coding region of a gene, proximal to the transcriptional start site of a structural gene. Sequence elements within promoters that function in the initiation of transcription are often characterized by consensus nucleotide sequences. These promoter elements include RNA polymerase binding sites, TATA sequences, CAAT sequences, differentiation-specific elements (McGehee et al, Mol. Endocrinol. 7:551 (1993)), cyclic AMP response elements, serum response elements (Treisman, Seminars in Cancer Biol. 1:4-1 (1990)), glucocorticoid response elements, and binding sites for other transcription factors, such as CRE/ATF (O'Reilly et al, J. Biol Chem. 267:19938 (1992)), AP2 (Ye et al., J. Biol. Chem. 269:25728 (1994)), SP1, cAMP response element binding protein (Loeken, Gene Expr. 3:253 (1993)) and octamer factors (see, in general, Watson et al, eds., Molecular Biology of the Gene, 4th ed. (The Benjamin/Cummings Publishing Company, Inc. 1987), and Lemaigre and Rousseau, Biochem. J. 303:1 (1994)). If a promoter is an inducible promoter, then the rate of transcription increases in response to an inducing agent. In contrast, the rate of transcription is not regulated by an inducing agent if the promoter is a constitutive promoter. Repressible promoters are also known.<br>
A "core promoter" contains essential nucleotide sequences for promoter function, including the TATA box and start of transcription. By this definition, a core promoter may or may not have detectable activity in the absence of specific sequences that may enhance the activity or confer tissue specific activity.<br>
A "regulatory element" is a nucleotide sequence that modulates the activity of a core promoter. For example, a regulatory element may contain a nucleotide sequence that binds with cellular factors enabling transcription exclusively or preferentially in particular cells, tissues, or organelles. These types of regulatory elements are normally associated with genes that are expressed in a "cell-specific," "tissue-specific," or "organelle-specific" manner.<br>
An "enhancer" is a type of regulatory element that can increase the efficiency of transcription, regardless of the distance or orientation of the enhancer relative to the start site of transcription.<br>
"Heterologous DNA" refers to a DNA molecule, or a population of DNA molecules, that does not exist naturally within a given host cell.    DNA molecules<br><br>
heterologous to a particular host cell may contain DNA derived from the host cell species (i.e., endogenous DNA) so long as that host DNA is combined with non-host DNA. For example, a DNA molecule containing a non-host DNA segment that encodes<br><br>
a polypeptide operably linked to a host DNA segment comprising a transcription promoter is considered to be a heterologous DNA molecule. Conversely, a heterologous DNA molecule can comprise an endogenous gene operably linked with a promoter derived from a non-host gene. As another illustration, a DNA molecule comprising a gene derived from a wild-type cell is considered to be heterologous DNA if that DNA molecule is introduced into a mutant cell that lacks the wild-type gene.<br>
A "polypeptide" is a polymer of amino acid residues joined by peptide bonds, whether produced naturally or synthetically. Polypeptides of less than about 10 amino acid residues are commonly referred to as "peptides."<br>
A "protein" is a macromolecule comprising one or more polypeptide chains. A protein may also comprise non-peptidic components, such as carbohydrate groups. Carbohydrates and other non-peptidic substituents may be added to a protein by the cell in which the protein is produced, and will vary with the type of cell. Proteins are defined herein in terms of their amino acid backbone structures; substituents such as carbohydrate groups are generally not specified, but may be present nonetheless.<br>
A peptide or polypeptide synthesized within a cell from a heterologous nucleic acid molecule is a "heterologous" peptide or polypeptide.<br>
An "integrated genetic element" is a segment of DNA that has been incorporated into a chromosome of a host cell after that element is introduced into the cell through human manipulation. Within the present invention, integrated genetic elements are most commonly derived from linearized plasmids that are introduced into the cells by electroporation or other techniques. Integrated genetic elements are passed from the original host cell to its progeny.<br>
A "cloning vector" is a nucleic acid molecule, such as a plasmid, cosmid, or bacteriophage, that has the capability of replicating autonomously in a host cell. Cloning vectors typically contain one or a small number of restriction endonuclease recognition sites that allow insertion of a nucleic acid molecule in a determinable fashion without loss of an essential biological function of the vector, as well as nucleotide sequences encoding a marker gene that is suitable for use in the identification and selection of cells transformed with the cloning vector. Marker genes typically include genes that provide tetracycline resistance or ampicillin resistance.<br>
An "expression vector" is a nucleic acid molecule encoding a gene that is expressed in a host cell. Typically, an expression vector comprises a transcription promoter, a gene, and a transcription terminator. Gene expression is usually placed under<br><br>
the control of a promoter, and such a gene is said to be "operably linked to" the promoter. Similarly, a regulatory element and a core promoter are operably linked if the regulatory element modulates the activity of the core promoter.<br>
A "recombinant host" is a cell that contains a heterologous nucleic acid molecule, such as a cloning vector or expression vector.<br>
"Integrative transformants" are recombinant host cells, in which heterologous DNA has become integrated into the genomic DNA of the cells.<br>
The term "expression" refers to the biosynthesis of a gene product. For example, in the case of a structural gene, expression involves transcription of the structural gene into mRNA and the translation of mRNA into one or more polypeptides.<br>
The term "secretory signal sequence" denotes a DNA sequence that encodes a peptide (a "secretory peptide") that, as a component of a larger polypeptide, directs the larger polypeptide through a secretory pathway of a cell in which it is synthesized. The larger polypeptide is commonly cleaved to remove the secretory peptide during transit through the secretory pathway.<br>
An "isolated polypeptide" is a polypeptide that is essentially free from contaminating cellular components, such as carbohydrate, lipid, or other proteinaceous impurities associated with the polypeptide in nature. Typically, a preparation of isolated polypeptide contains the polypeptide in a highly purified form, i.e., at least about 80% pure, at least about 90% pure, at least about 95% pure, greater than 95% pure, or greater than 99% pure. One way to show that a particular protein preparation contains an isolated polypeptide is by the appearance of a single band following sodium dodecyl sulfate (SDS)-polyacrylamide gel electrophoresis of the protein preparation and Coomassie Brilliant Blue staining of the gel. However, the term "isolated" does not exclude the presence of the same polypeptide in alternative physical forms, such as dimers or alternatively glycosylated or derivatized forms.<br>
The terms "amino-terminal" and "carboxyl-terminal" are used herein to denote positions within polypeptides. Where the context allows, these terms are used with reference to a particular sequence or portion of a polypeptide to denote proximity or relative position. For example, a certain sequence positioned carboxyl-terminal to a reference sequence within a polypeptide is located proximal to the carboxyl terminus of the reference sequence, but is not necessarily at the carboxyl terminus of the complete polypeptide.<br>
A "fusion protein" is a hybrid protein expressed by a nucleic acid molecule comprising nucleotide sequences of at least two genes. In this way, a fusion protein comprises as least two amino acid sequences that are not associated with each other in nature.<br><br>
When used to describe a component of an expression vector, the language "gene or gene fragment" refers to a nucleotide sequence that encodes a polypeptide or peptide. The gene or gene fragment can be obtained from genomic DNA, from cDNA, or by an in vitro synthesis technique (e.g., polymerase chain reaction, chemical synthesis, and the like).<br>
The term "affinity tag" is used herein to denote a polypeptide segment that can be attached to a second polypeptide to provide for purification or detection of the second polypeptide or provide sites for attachment of the second polypeptide to a substrate. In principal, any peptide or protein for which an antibody or other specific binding agent is available can be used as an affinity tag. Affinity tags include a poly-histidine tract, protein A (Nilsson et al, EMBO J. 4:1075 (1985); Nilsson et al, Methods Enzymol 198:3 (1991)), glutathione S transferase (Smith and Johnson, Gene 67:31 (1988)), Glu-Glu affinity tag (Grussenmeyer et al., Proc. Natl. Acad. Sci. USA 82:1952 (1985)), substance P, FLAG peptide (Hopp et al, Biotechnology 6:1204 (1988)), streptavidin binding peptide, or other antigenic epitope or binding domain. See, in general, Ford et al, Protein Expression and Purification 2:95 (1991). DNA molecules encoding affinity tags are available from commercial suppliers (e.g., Pharmacia Biotech, Piscataway, NJ).<br>
As used herein, the term "immunomodulator" includes cytokines, stem cell growth factors, lymphotoxins, co-stimulatory molecules, hematopoietic factors, and synthetic analogs of these molecules. Examples of immunomodulators include tumor necrosis factor, interleukins, colony stimulating factors, interferons, stem cell growth factors, erythropoietin, and thrombopoietin.<br>
The phrase an "immunoglobulin moiety" refers to a polypeptide that comprises a constant region of an immunoglobulin. For example, the immunoglobulin moiety can comprise a heavy chain constant region.<br>
The phrase "complement/anti-complement pair" denotes non-identical moieties that form a non-covalently associated, stable pair under appropriate conditions. For instance, biotin and avidin (or streptavidin) are prototypical members of a complement/anti-complement pair. Other exemplary complement/anti-complement pairs include receptor/ligand pairs, antibody/antigen (or hapten or epitope) pairs, sense/antisense polynucleotide pairs, and the like.<br>
An "antibody fragment" is a portion of an antibody such as F(ab')2, F(ab)2, Fab', Fab, and the like. Regardless of structure, an antibody fragment binds with the same antigen that is recognized by the intact antibody.<br>
The term "antibody fragment" also includes a synthetic or a genetically engineered polypeptide that binds to a specific antigen, such as polypeptides consisting of<br><br>
the light chain variable region, "Fv" fragments consisting of the variable regions of the heavy and light chains, recombinant single chain polypeptide molecules in which light and heavy variable regions are connected by a peptide linker ("scFv proteins"), and minimal recognition units consisting of the amino acid residues that mimic the hypervariable region.<br>
A "detectable label" is a molecule or atom which can be conjugated to a polypeptide to produce a molecule useful for identifying cells that express the binding partner of the polypeptide. Examples of detectable labels include chelators, photoactive agents, radioisotopes, fluorescent agents, paramagnetic ions, or other marker moieties.<br>
Due to the imprecision of standard analytical methods, molecular weights and lengths of polymers are understood to be approximate values. When such a value is expressed as "about" X or "approximately" X, the stated value of X will be understood to be accurate to Â±10%.<br>
2-        Expression Vectors For Producing Homotrimeric Polypeptides<br>
The present invention provides methods for producing homotrimeric proteins. Each protein of a homotrimer is a fusion protein that comprises a polypeptide of interest and a trimerizing amino acid sequence. Polypeptides of interest include the extracellular domains of receptors, which can be used to bind their cognate ligands. Suitable receptors include tumor necrosis factor receptors, such as TNFRSF1A (also designated "p55," "TNFR-60," and 'TNF-R"; see, for example, Genbank No. M75866), TNFRSF1B (also designated "p75," and "TNFR2"; see, for example, Genbank No. M32315), TNFRSF13B (also known as "TACI"), TNFRSF13C (also known as "BAFFR " and "Ztnfrl2"; see, for example, Genbank No. AF373846), TNFRSF17 (also known as "BCMA"; see, for example, Genbank No. Z29574), and the like. Other useful tumor necrosis factor receptors are known to those of skill in the art.<br>
The Examples illustrate the construction of fusion proteins comprising the extracellular domain of the transmembrane activator and CAML (calcium-signal modulating cyclophilin ligand) interactor (TACI). TACI nucleic acid and amino acid sequences are described by Bram and Gotz, U.S. Patent No. 5,969,102, and are included herein as SEQ ID NOs. 3 and 4. Illustrative TACI extracellular domains include polypeptides that have amino acid sequences comprising amino acid residues 30 to 110 of SEQ ID NO:4, amino acid residues 1 to 110 of SEQ ID NO:4, amino acid residues 30 to 154 of SEQ ID NO:4, and amino acid residues 1 to 154 of SEQ ID NO:4.<br>
The Examples also illustrate the use of two types of trimerizing amino acid sequences: the carboxy-terminal, 151 amino acid NC-1 region of human collagen X, and amino acids 1 to 65 of the human heat shock factor binding protein, HSBP-1.  The<br><br>
NC-1 domain is described by Frischholz et al.t J. Biol. Chem. 273:4547 (1998); nucleotide and amino acid sequences are provided herein has SEQ ID NOs. 19 and 20. HSBP-1 is described by Tai et al. J. Biol. Chem. 277:735 (2002). Nucleotide and amino acid sequences of a useful fragment of HSBP-1 are provided as SEQ ED NOs. 21 and 22.<br>
In addition to the trimerizing amino acid sequences, the fusion protein can further comprise an immunoglobulin moiety in order to make the protein soluble. The immunoglobulin moiety can comprise a heavy chain constant region, such as a human heavy chain constant region. An IgGl heavy chain constant region is one example of a suitable heavy chain constant region. An illustrative IgGl heavy chain constant region is an IgGl Fc fragment that comprises CH2, and CH3 domains. The IgGl Fc fragment can be a wild-type IgGl Fc fragment or a mutated IgGl Fc fragment.<br>
Expression vectors can be constructed that encode a fusion protein comprising a polypeptide of interest and a trimerizing amino acid sequence. Expression vectors that are suitable for production of a protein in eukaryotic cells typically contain (1) prokaryotic DNA elements coding for a bacterial replicationc origin and an antibiotic resistance marker to provide for the growth and selection of the expression vector in a bacterial host; (2) eukaryotic DNA elements that control initiation of transcription, such as a promoter; and (3) DNA elements that control the processing of transcripts, such as a transcription termination/polyadenylation signal sequence.<br>
To express a gene, a nucleic acid molecule encoding the protein must be operably linked to regulatory sequences that control transcriptional expression and then, introduced into a host cell. In addition to transcriptional regulatory sequences, such as promoters and enhancers, expression vectors can include transcriptional and translational regulatory sequences. As an illustration, the transcriptional and translational regulatory signals suitable for a mammalian host may be derived from viral sources, such as adenovirus, bovine papilloma virus, simian virus, or the like, in which the regulatory signals are associated with a particular gene that has a high level of expression. Suitable transcriptional and translational regulatory sequences also can be obtained from mammalian genes, such as actin, collagen, myosin, and metallothionein genes.<br>
Suitable transcriptional regulatory sequences include a promoter region sufficient to direct the initiation of RNA synthesis. Illustrative eukaryotic promoters include the promoter of the mouse metallothionein I gene (Hamer et al, J. Molec. Appl. Genet 7:273 (1982)), the TK promoter of Herpes virus (McKnight, Cell 31:355 (1982)), the SV40 early promoter (Benoist et ai, Nature 290:304 (1981)), the Rous sarcoma virus promoter (Gorman et al, Proc. Nat'l Acacl Sci, USA 79:6111 (1982)), the cytomegalovirus promoter (Foecking et al., Gene 45:101 (1980)), and the mouse mammary tumor virus promoter (see, generally, Etcheverry, "Expression of Engineered<br><br>
Proteins in Mammalian Cell Culture," in Protein Engineering: Principles and Practice, Cleland et al. (eds.), pages 163-181 (John Wiley &amp; Sons, Inc. 1996)).<br>
Alternatively, a prokaryotic promoter, such as the bacteriophage T3 RNA polymerase promoter, can be used to control expression of the gene of interest in mammalian cells if the prokaryotic promoter is regulated by a eukaryotic promoter (Zhou et aU MoL Cell Biol. 10:4529 (1990), and Kaufman et al, Nucl. Acids Res. 79:4485 (1991)).<br>
The inclusion of an affinity tag is useful for the identification or selection of cells displaying the fusion protein. Examples of affinity tags include polyHistidine tags (which have an affinity for nickel-chelating resin), c-myc tags (e.g., EQKLI SEEDL; SEQ ID NO:1) which are detected with anti-myc antibodies, calmodulin binding protein (isolated with calmodulin affinity chromatography), substance P, the RYIRS tag (which binds with anti-RYIRS antibodies), a hemagglutinin A epitope tag {e.g., YPYDV PDYA; SEQ ID NO:2) which is detected with an antibody, the Glu-Glu tag, and the FLAG tag (which binds with anti-FLAG antibodies). See, for example, Luo et al., Arch. Biochem. Biophys. 329:215 (1996), Morganti et al, Biotechnol Appl Biochem. 23:61 (1996), and Zheng et al, Gene 186:55 (1997). Nucleic acid molecules encoding such peptide tags are available, for example, from Sigma-Aldrich Corporation (St. Louis, MO).<br>
The cloning site can be a multicloning site. Any multicloning site can be used, and many are commercially available. Particularly useful multicloning sites allow the cloning of a gene or gene fragment in all three reading frames.<br>
The expression vector can include a nucleotide sequence that encodes a selectable marker. A wide variety of selectable marker genes are available (see, for example, Kaufman, Meth. Enzymol. 755:487 (1990); Kaufman, Meth. Enzymol 185:531 (1990)). For example, one suitable selectable marker is a gene that provides resistance to the antibiotic neomycin. In this case, selection is carried out in the presence of a neomycin-type drug, such as G-418 or the like. Bleomycin-resistance genes, such as the Sh ble gene, are also useful selectable marker genes for the presently described methods. These genes produce a protein that inhibits the activity of bleomycin/phleomycin-type drugs, such as ZEOCIN (Gatignol et al., MoL Gen. Genet. 207:342 (1987); Drocourt et al, Nucl. Acids Res. 75:4009 (1990)). ZEOCIN is toxic in a broad range of cell types, including bacteria, fungi, plant, avian, insect, and mammalian cells. Additional selectable markers include hygromycin B-phosphotransferase, the AUR1 gene product, adenosine deaminase, aminoglycoside phosphotransferase, dihydrofolate reductase, thymidine kinase, and xanthine-guanine phosphoribosyltransferase (see, for example, Srivastava and Schlessinger, Gene 103:53 (1991); Romanos et al, "Expression of Cloned Genes in Yeast," in DNA Cloning 2: Expression Systems, 2"  Edition, pages 123-<br><br>
167 (IRL Press 1995); Markie, Methods Mol Biol 54:359 (1996); Pfeifer et al., Gene 7SS:183 (1997); Tucker and Burke, Gene 199:25 (1997); Hashida-Okado et al, FEBS Letters 425:111 (1998)). Selectable marker genes can be cloned or synthesized using published nucleotide sequences, or marker genes can be obtained commercially.<br>
An expression vector can also include an SV40 origin. This element can be used for episomal replication and rescue in cell lines expressing SV40 large T antigen.<br>
A gene or gene fragment suitable for insertion into an expression vector can be obtained from cDNA, which is prepared by any method known in the art. For example, cDNA molecules can be synthesized by random priming. Moreover, such primers can be linked to restriction endonuclease sites found in the vector. Alternatively, cDNA molecules can be prepared by oligo d(T) priming. A gene or gene fragment can also be obtained from genomic DNA or by chemical synthesis. Standard methods for preparing suitable genes or gene fragments are known to those in the art (see, for example, Ausubel et al (eds.), Short Protocols in Molecular Biology, 3rd Edition (John Wiley &amp; Sons 1995) ["Ausubel 1995"]).<br>
After constructing the expression vector, the vector can be propagated in a host cell to synthesize nucleic acid molecules for the generation of a nucleic acid polymer. Vectors, often referred to as "shuttle vectors," are capable of replicating in at least two unrelated expression systems. To facilitate such replication, the vector should include at least two origins of replication, one effective in each replication system. Typically, shuttle vectors are capable of replicating in a eukaryotic system and a prokaryotic system. This enables detection of protein expression in eukaryotic hosts, the "expression cell type," and the amplification of the vector in the prokaryotic hosts, the "amplification cell type." As an illustration, one origin of replication can be derived from SV40, while another origin of replication can be derived from pBR322. Those of skill in the art know of numerous suitable origins of replication.<br>
Vector propagation is conveniently carried out in a prokaryotic host cell, such as E. coli or Bacillus subtilus. Suitable strains of E. coli include BL21(DE3), BL21(DE3)pLysS, BL21(DE3)pLysE, DH1, DH4I, DH5, DH5I, DH5IF, DH5IMCR, DH10B, DH10B/p3, DH11S, C600, HB101, JM101, JM105, JM109, JM110, K38, RR1, Y1088, Y1089, CSH18, ER1451, and ER1647 (see, for example, Brown (ed.), Molecular Biology Labfax (Academic Press 1991)). Suitable strains of Bacillus subtilus include BR151, YB886, MI119, MI120, and B170 (see, for example, Hardy, "Bacillus Cloning Methods," in DNA Cloning: A Practical Approach, Glover (ed.) (IRL Press 1985)). Standard techniques for propagating vectors in prokaryotic hosts are well-known<br><br>
to those of skill in the art (see, for example, Ausubel 1995; Wu et al, Methods in Gene Biotechnology (CRC Press, Inc. 1997)).<br>
3.        Production of Recombinant Protein by Host Cells<br>
The expression vector can be introduced into any eukaryotic cell, such as a mammalian cell, insect cell, avian cell, fungal cell, and the like. Examples of suitable mammalian host cells include African green monkey kidney cells (Vero; ATCC CRL 1587), human embryonic kidney cells (293-HEK; ATCC CRL 1573), baby hamster kidney cells (BHK-21, BHK-570; ATCC CRL 8544, ATCC CRL 10314), canine kidney cells (MDCK; ATCC CCL 34), Chinese hamster ovary cells (CHO-K1; ATCC CCL61; CHO DG44 (Chasin et aL, Sorn. Cell. Molec. Genet. 72:555, 1986)), rat pituitary cells (GH1; ATCC CCL82), HeLa S3 cells (ATCC CCL2.2), rat hepatoma cells (H-4-II-E; ATCC CRL 1548) SV40-transformed monkey kidney cells (COS-1; ATCC CRL 1650) and murine embryonic cells (NIH-3T3; ATCC CRL 1658).<br>
The baculovirus system provides an efficient means to introduce cloned genes of interest into insect cells. Suitable expression vectors are based upon the Autographa californica multiple nuclear polyhedrosis virus (AcMNPV), and contain well-known promoters such as Drosophila heat shock protein (hsp) 70 promoter, Autographa califomica nuclear polyhedrosis virus immediate-early gene promoter (ie-1) and the delayed early 39K promoter, baculovirus p10 promoter, and the Drosophila metallothionein promoter. A second method of making recombinant baculovirus utilizes a transposon-based system described by Luckow (Luckow, et al., J. Virol 67:4566 (1993)). This system, which utilizes transfer vectors, is sold in the BAC-to-BAC kit (Life Technologies, Rockville, MD). This system utilizes a transfer vector, PFASTBAC (Life Technologies) containing a Tn7 transposon to move the gene or gene fragment into a baculovirus genome maintained in E. coli as a large plasmid called a "bacmid." See, Hill-Perkins and Possee, /. Gen. Virol 77:971 (1990), Bonning, et al, J. Gen. Virol. 75:1551 (1994), and Chazenbalk, and Rapoport, J. Biol. Chem. 270:1543 (1995). These vectors can be modified following the above discussion<br>
The recombinant virus or bacmid is used to transfect host cells. Suitable insect host cells include cell lines derived from IPLB-5/-21, a Spodoptera frugiperda pupal ovarian cell line, such as 5/9 (ATCC CRL 1711), 5/21AE, and 5/21 (Invitrogen Corporation; San Diego, CA), as well as Drosophila Schneider-2 cells, and the HIGH FIVEO cell line (Invitrogen) derived from Trichoplusia ni (U.S. Patent No. 5,300,435). Commercially available serum-free media can be used to grow and to maintain the cells. Suitable media are Sf900 IIâ¢ (Life Technologies) or ESF 921â¢ (Expression Systems)<br><br>
for the Sf9 cells; and Ex-cellO405â¢ (JRH Biosciences, Lenexa, KS) or Express FiveOâ¢ (Life Technologies) for the T ni cells. When recombinant virus is used, the cells are typically grown up from an inoculation density of approximately 2-5 x 105 cells to a density of 1-2 x 106 cells at which time a recombinant viral stock is added at a multiplicity of infection (MOI) of O.I to 10, more typically near 3.<br>
Established techniques for producing recombinant proteins in baculovirus systems are provided by Bailey et al, "Manipulation of Baculovirus Vectors," in Methods in Molecular Biology, Volume 7: Gene Transfer and Expression Protocols, Murray (ed.), pages 147-168 (The Humana Press, Inc. 1991), by Patel et al, "The baculovirus expression system," in DNA Cloning 2: Expression Systems, 2nd Edition, Glover et al. (eds.), pages 205-244 (Oxford University Press 1995), by Ausubel (1995) at pages 16-37 to 16-57, by Richardson (ed.), Baculovirus Expression Protocols (The Humana Press, Inc. 1995), and by Lucknow, "Insect Cell Expression Technology," in Protein Engineering: Principles and Practice, Cleland et al. (eds.), pages 183-218 (John Wiley &amp; Sons, Inc. 1996).<br>
The expression vectors described herein can also be used to transfect fungal cells, including yeast cells. Yeast species of particular interest in this regard include Saccharomyces cerevisiae, Pichia pastoris, and Pichia methanolica. Suitable promoters for expression in yeast include promoters from GALl (galactose), PGK (phosphoglycerate kinase), ADH (alcohol dehydrogenase), AOX1 (alcohol oxidase), HIS4 (histidinol dehydrogenase), and the like. Many yeast cloning vectors readily available and can be modified following the above discussion. These vectors include Yip-based vectors, such as YIp5, YRp vectors, such as YRpl7, YEp vectors such as YEpl3 and YCp vectors, such as YCpl9. Methods for transforming S. cerevisiae cells with exogenous DNA and producing recombinant polypeptides therefrom are disclosed by, for example, Kawasaki, U.S. Patent No. 4,599,311, Kawasaki et al., U.S. Patent No. 4,931,373, Brake, U.S. Patent No. 4,870,008, Welch et al., U.S. Patent No. 5,037,743, and Murray et al, U.S. Patent No. 4,845,075. Transformed cells are selected by phenotype determined by the selectable marker, commonly drug resistance or the ability to grow in the absence of a particular nutrient (e.g., leucine). A preferred vector system for use in Saccharomyces cerevisiae is the POTl vector system disclosed by Kawasaki et al (U.S. Patent No. 4,931,373), which allows transformed cells to be selected by growth in glucose-containing media. Additional suitable promoters and terminators for use in yeast include those from glycolytic enzyme genes (see, e.g., Kawasaki, U.S. Patent No. 4,599,311, Kingsman et al, U.S. Patent No. 4,615,974, and Bitter, U.S. Patent No. 4,977,092) and alcohol dehydrogenase genes. See also U.S. Patents Nos. 4,990,446, 5,063,154, 5,139,936, and 4,661,454.<br><br>
Transformation systems for other yeasts, including Hansenula polymorpha, Schizosaccharomyces pombe, Kluyveromyces lactis, Kluyveromyces fragilis, Ustilago maydis, Pichia pastoris, Pichia methanolica, Pichia guillermondii and Candida maltosa are known in the art. See, for example, Gleeson et al, J. Gen. Microbiol 132:3459 (1986), and Cregg, U.S. Patent No. 4,882,279. Aspergillus cells may be utilized according to the methods of McKnight et al, U.S. Patent No. 4,935,349. Methods for transforming Acremonium chrysogenum are disclosed by Sumino et al, U.S. Patent No. 5,162,228. Methods for transforming Neurospora are disclosed by Lambowitz, U.S. Patent No. 4,486,533.<br>
For example, the use of Pichia methanolica as host for the production of recombinant proteins is disclosed by Raymond, U.S. Patent No. 5,716,808, Raymond, U.S. Patent No. 5,736,383, Raymond et al, Yeast 14:11-23 (1998), and in international publication Nos. WO 97/17450, WO 97/17451, WO 98/02536, and WO 98/02565. DNA molecules for use in transforming P. methanolica will commonly be prepared as double-stranded, circular plasmids, which are preferably linearized prior to transformation. For polypeptide production in P. methanolica, it is preferred that the promoter and terminator in the plasmid be that of a P. methanolica gene, such as a P. methanolica alcohol utilization gene (AUG1 or AUG2). Other useful promoters include those of the dihydroxyacetone synthase (DHAS), formate dehydrogenase (FMD), and catalase (CAT) genes. To facilitate integration of the DNA into the host chromosome, it is preferred to have the entire expression segment of the plasmid flanked at both ends by host DNA sequences. For large-scale, industrial processes where it is desirable to minimize the use of methanol, it is preferred to use host cells in which both methanol utilization genes (AUG1 and AUG2) are deleted. For production of secreted proteins, host cells deficient in vacuolar protease genes (PEP4 and PRB1) are preferred. Electroporation is used to facilitate the introduction of a plasmid containing DNA encoding a polypeptide of interest into P. methanolica cells. P. methanolica cells can be transformed by electroporation using an exponentially decaying, pulsed electric field having a field strength of from 2.5 to 4.5 kV/cm, preferably about 3.75 kV/cm, and a time constant (t) of from 1 to 40 milliseconds, most preferably about 20 milliseconds.<br>
An expression vector can be introduced into host cells using a variety of standard techniques including calcium phosphate transfection, liposome-mediated transfection, microprojectile-mediated delivery, electroporation, and the like.<br>
Standard methods for introducing expression vectors into mammalian, yeast, and insect cells are provided, for example, by Ausubel (1995). General methods for expressing and recovering foreign protein produced by a mammalian cell system are provided by, for example, Etcheverry, "Expression of Engineered Proteins in Mammalian<br><br>
Cell Culture," in Protein Engineering; Principles and Practice, Cleland et al. (eds.), page 163 (Wiley-Liss, Inc. 1996). Established methods for isolating recombinant proteins from a baculovirus system are described by Richardson (ed.), Baculovirus Expression Protocols (The Humana Press, Inc. 1995).<br>
Expression vectors can be isolated from cells that produce a polypeptide of interest. If desired, expression vectors can be subjected to another round of selection based on expression of the identifiable polypeptide or, transfected into the amplification cell type. The transfected amplification cell type is then selected by the selectable marker, the vectors are purified and the nucleotide sequence of the gene or gene fragment is sequenced by any method known in the art. If the nucleotide sequence encodes only a portion of a complete polypeptide, then the nucleotide sequence can be used as a probe by methods known in the art to retrieve the entire gene.<br>
The present invention, thus generally described, will be understood more readily by reference to the following examples, which are provided by way of illustration and are not intended to be limiting of the present invention.<br>
EXAMPLE 1<br>
Construction of  TACI-NCl<br>
An expression vector was constructed that encodes a fusion comprising an extracellular domain of the transmembrane activator and CAML (calcium-signal modulating cyclophilin ligand) interactor (TACI) protein and the NCI domain of human collagen X. TACI nucleic acid sequences are described by Bram and Gotz, U.S. Patent No. 5,969,102, and included herein as SEQ ID NOs. 3 and 4. The NC-1 domain is described by Frischholz et al., J. BioL Chem. 273A541 (1998); nucleotide and amino acid sequences are provided herein has SEQ ID NOs. 19 and 20. In this construct, the extracellular domain of TACI was fused to NCI with a Glu-Glu tag at the c-terminus and a Gly-Ser spacer of eight amino acids engineered between TACI and NC1.<br>
NCI was amplified by PCR from human genomic DNA (Clontech) using oligonucleotides zc40219 (5' GGGCCTCCAG GCCCACCAGG T 3'; SEQ ID NO:5) and zc40205 (5' TCACATTGGA GCCACTAGGA A 3'; SEQ ID NO:6). The extracellular portion of TACI was amplified by PCR from a clone that encoded a TACI-immunoglobulin fusion protein with oligonucleotides zc40915 (5' ACAGGTGTCC AGGGAATTCA TATAGGCCGG CCACCATGGA TGCAATGAAG AGAGGG 3'; SEQ ID NO:7) and zc40917 (5' ACCCTCAGGC ATCGAACCCG AACCCGAACC<br><br>
GGATCC 3': SEQ ID N0:8) with conditions of 30 cycles of 94Â°C for one minute, 55Â°C for one minute, and 72 Â°C for two minutes. The PCR products were precipitated and resuspended in 10 Î¼1 of water and then recombined in S xerevisiae into pZMP21 that had been digested with BglII. E. coli clones that resulted from the recombination were screened for proper incorporation by AscI digestion and three positive clones were submitted for sequencing. One clone was selected for further study. This clone contained a glycine to arginine mutation in NCI and lacked four amino acids from the C-terminal Glu-Glu tag.<br>
A vector encoding TACI/NC1-EE was linearized for electroporation by digesting 20 Î¼g of Qiagen-purified DNA with Pvul. This linearized DNA was electroporated in PF-CHO cells. The cells were allowed to recover for 24-hours before nutrient selection in HT- media for ten days. After recovery from nutrient selection, cells were transferred into 50 nm methotrexate selection for an additional ten days.<br>
Transfected cells were seeded into cell factories and two liters of factory conditioned media (CM) was isolated. The CM was combined with 1.5 mg Anti-TACI monoclonal antibody and incubated overnight at 4Â°C. The CM-antibody mixture was applied to a 1.6 mL bed volume POROS A50 column at a flow rate of 2 mL/min. Following addition, the column was washed with 100 column volumes of PBS, pH 7.2. The bound protein was then eluted directly into 2 M tris pH 8.0 with 200 mM glycine pH2.5. One milliliter fractions were collected. Based on western blot analysis, fractions containing TACI-NCl were pooled. The pooled fractions were concentrated to 300 Î¼l, and buffer exchanged three times with 14 mL PBS, pH 7.2, and then dialyzed against three changes of 4 L PBS, pH 7.2.<br>
EXAMPLE 2 Expression ofTACI-HSBl in Mammalian Cells<br>
A. Synthesis of the HSB1 gene<br>
Human heat shock binding protein (HSBP-1) is described by Tai et al. J. Biol. Chem. 277:135 (2002). HSBP-1 nucleotide and amino acid sequences are provided as SEQ ID NOs. 21 and 22. Four overlapping oligonucleotides, which encoded both sense and antisense strands of human heat shock binding protein (HSBP-1), were synthesized by solid phased synthesis, using the following primers: 5' GATCGGATCC ATGGCCGAAA CTGATCCTAA AACAGTTCAA GACCTTACCA GCGTAGTCCA GACGCTCCTG CAAGAGATCG AAGATAAGTT TCAGACTATG AGCGACCAAA<br><br>
TCATTGAG 3' (SEQ ID N0:9); 51 AGAATGCATG ACATGAGCTC CAGGATAGAT GACCTTGAGA AAAATATAGC AGATTTAATG ACGCAAGCTG GTGTGGAAGA GTTGGAAGGA AGTGGTTCTA 3' (SEQ ID NO: 10); 5' GATCTAGAAC CACTTCCTTC CAACTCTTCC ACACCAGCTT GCGTCATTAA ATCTGCTATA TTTTTCTCAA GGTCATCTAT CCTGGAGCTC ATGTCATCGA TTCTCTCAAT 3' (SEQ ID NO:11); and 5' GATTTGGTCG CTCATAGTCT GAAACTTATC TTGCATCTCT TGCAGGAGCG TCTGGACTAC GCTGGTAAGG TCTTGAACTG TTTTAGGATC AGTTTCGGCC ATGGATCC 3' (SEQ ID NO: 12). The 5' end of each oligonucleotide was phosphorylated by combining 120 pmoles of each oligonucleotide, 1.6 Î¼l 100 mM ATP, 34Î¼l 5x T4-Kinase buffer (Life Technologies, Bethesda, MD), 6.4Î¼l water, and 1Î¼l T4-polynucleotide kinase (Life Technologies), and incubating for 20 minutes at 37Â°C. The phosphorylation reaction was placed into a boiling water bath and then slowly cooled to 25Â°C to promote annealing. The fragments were ligated by adding 20 Î¼l of lOx T4-ligase buffer (Life Technologies), 0.5Î¼l of 100 mM ATP, and 2Î¼l of T4-DNA ligase (Life Technologies), and incubating overnight at 16Â°C. Following ligation, the DNA was collected by alcohol precipitation. The isolated DNA was resuspended in 3.2 (xl of B restriction buffer (Promega, Madison, WI), 26.8Î¼l of water, 1.5 Î¼l of BglII (Life Technologies), and 1.5Î¼l of Asp718 (Life Technologies), and incubated for two hours at 37Â°C. The ligase reaction product was fractionated on a 15% agarose gel and the 220 nucleotide fragment encoding human HSBP-1 was isolated using a Qiagen gel isolation kit according to manufacturer's protocol (Qiagen). The human HSBP-1 was inserted into an Asp718-BamiHI cleaved vector using T4-DNA ligase and manufacturer's guidelines (Life Technologies).<br>
A fragment encoding the extracellular domain of TACI was amplified from the pTACI-NCl vector using the oligonucleotides zc41712 (5' CACACGTACG AAGATGGATG CAATGAAGAG AGG 3'; SEQ ID NO: 13) and zc41638 (5' GGTTAGATCT CGAACCCGAA CCCGAACCGG 3'; SEQ ID NO: 14). The PCR product was cut with BsiW1 and BglII, and the amplified DNA fractionated on 1.5% agarose gel-and then isolated using a Qiagen gel isolation kit according to manufacturer's protocol (Qiagen). The isolated DNA was inserted into Asp7l8-BglII cleaved vector that included the HSBP-1-enconding sequence, using T4-DNA ligase, following the manufacturer's guidelines (Life Technologies). DNA sequencing confirmed the expected sequence of the vector, which was designated "pHZTACI-HSBP.9."<br><br>
B. Expression and Purification of TACI-HSBP-1<br>
The pHZTACI-HSBP.9 vector was transfected into BHK570 using TransTransfected and the cultures were selected for trasnfectants resistance to 10 /xM methotrexate. Resistant colonies were transferred to tissue culture dishes, expanded and analyzed for secretion of TACI-HSBP-His6 by western blot analysis with Anti-His (C-terminal) Antibody (Invitrogen, Carlsbad, CA). The resulting cell line, "BHK.TACI-HSBP.2," was expanded.<br>
BHK.TACI-HSBP.2 cells were seeded into cell factories and 12 liters of factory-conditioned media were isolated. The media were applied to a 25 milliliter bed volume Ni-NTA (Invitrogen) at a flow rate of two ml/min. Following addition, the column was washed with 50 column volumes of PBS, pH 7.2 and 20 column volumes of phosphate buffered saline, 5.0 mM imidizole, pH 6.0. The bound protein was then eluted with a linear gradient of increasing imidizole concencentration from 50 mM to 800 mM in PBS, pH 6.0, at 1 ml/min for 60 minutes. One milliliter fractions were collected. Based on western blot analysis, fractions containing TACI-HSBP-1 were pooled. The pooled fractions were concentrated to 300Î¼l, and buffer was exchanged three times with 14 ml of phosphate buffered saline (pH 7.2), and then dialyzed against three changes of four liters of phosphate buffered saline, pH 7.2. Western blot analysis and coomasie stained gels showed greater than 75% purity and a monomeric molecular weight of 20 kDa.<br>
EXAMPLE 3 Proliferation Assay for TACI-fusion Proteins<br>
Peripheral blood mononuclear cells from apheresis were isolated by density gradient centrifugation on Ficol-Hypaque and washed in phosphate buffered saline. Typically, about 1010 peripheral blood mononuclear cells can be isolated from one donor. About 108 cells were frozen per vial in 90% fetal calf serum and 10% dimethylsulfoxide.<br>
Multiple vials were thawed and cell viability was determined. B cells were isolated from peripheral blood mononuclear cells using CD 19 magnetic beads and the VarioMacs magnetic separation system (Miltenyi Biotec; Auburn, CA). Round bottom 96 well plates were pre-coated with goat anti-human IgM at 5 Î¼g/ml in phosphate buffered saline for 24 hours at 4Â°C. (Southern Biotechnology Assoc. Inc.; Birmingham AL). Purified B cells were plated, at 105 cells per well in the presence of 10 ng/mL human IL-4 (Pharmingen) and 20 ng/mL zTNF4, a ligand that binds with TACI. A three-fold dilution  series of TACI-Fc fusion protein, TACI-HSBP-1, or TACI-NC1<br><br>
starting at 1 Î¼g/ml, were included to compare their ability to inhibit zTNF4 stimulated B-cell proliferation. The cells were incubated for four days in the presence of zTNF4, human IL-4 and with or without inhibitors, and then pulsed overnight with 1 pCi of H3 thymidine (Amersham) per well. Plates were harvested using a Packard plate harvester and counted using the Packard reader. TACI-HSBP-1 was found to be three-fold more efficient, and TACI-NC1 ten-fold more efficient at inhibiting zTNF4 stimulated proliferation of human B-cells in this assay. As shown in Figure 2, TACI-NC1 reduced the TNF4 induced B-cell proliferation to a greater degree than the same concentration of TACI-Fc5. This difference is numerically expressed in the EC50 values indicated in the figure, with TACI-Fc5 having a value of 1.1 nM and TACI-NC1 having a value of 57 pM, indicating that TACI-NC1 is approximately 19 times more effective.<br>
EXAMPLE 4<br>
Expression ofTACI-HSBl in Bacterial Cells<br>
An expression plasmid containing a nucleotide sequence that encodes human TACI-HSBP-1, was inserted behind the G10 enhancer sequence via yeast homologous recombination. A DNA fragment of human TACI-HSBP-1 was isolated using PCR. Two primers were used in the production of human TACI-HSBP-1 in a PCR reaction. Primer zc42,728 (51 CTAGAAATAA TTTTGTTTAA CTTTAAGAAG GAGATATATA TATGGCTATG AGATCCTGCC CC 3'; SEQ ID NO: 15) containing 40 base pairs of vector flanking sequence, comprised of the g10 enhancer sequence and 24 base pairs corresponding to the amino terminus of human TACI-HSBP-1. Primer zc42,731 (5' TCTGTATCAG GCTGAAAATC TTATCTCATC CGCCAAAACA CTAGTGATGG TGATGGTGAT GGCC 3'; SEQ ID NO: 16) contained 40 base pairs of the vector flanking sequence and 24 base pairs corresponding to the carboxyl terminus of the TACI-HSBP-1 sequence. The template was pH2-TACI-HSBP9. The PCR reaction conditions were as follows: 25 cycles of 94Â°C for 30 seconds, 50Â°C for 30 seconds, and 72Â°C for 1 minute; followed by a 4Â°C soak. A 2 to 4 Î¼l volume of the PCR sample was run on a 1% agarose gel with lxTBE buffer for analysis, and the expected band of approximately 550 base pair fragment (5' ATGGCTATGA GATCCTGCCC CGAAGAGCAG TACTGGGATC CTCTGCTGGG TACCTGCATG TCCTGCAAAA CCATTTGCAA CCATCAGAGC CAGCGCACCT GTGCAGCCTT CTGCAGGTCA CTCAGCTGCC GCAAGGAGCA AGGCAAGTTC TATGACCATC TCCTGAGGGA CTGCATCAGC TGTGCCTCCA TCTGTGGACA GCACCCTAAG CAATGTGCAT ACTTCTGTGA GAACAAGCTC AGGAGCGGAT CCGGTTCGGG TTCGGGTTCG AGATCCATGG CCGAAACTGA TCCTAAAACA GTTCAAGACC TTACCAGCGT AGTCCAGACG CTCCTGCAAG AGATGCAAGA TAAGTTTCAG ACTATGAGCG<br><br>
ACCAAATCAT TGAGAGAATC GATGACATGA GCTCCAGGAT AGATGACCTT GAGAAAAATA TAGCAGATTT AATGACGCAA GCTGGTGTGG AAGAGTTGGA AGGAAGTGGT TCTAGATCCG GTGGCCATCA CCATCACCAT CACTGA 3'; SEQ ID NO: 17) (MAMRSCPEEQ YWDPLLGTCM SCKTICNHQS QRTCAAFCRS LSCRKEQGKF YDHLLRDCIS CASICGQHPK QCAYFCENKL RSGSGSGSGS RSMAETDPKT VQDLTSVVQT LLQEMQDKFQ TMSDQHERI DDMSSRIDDL EKNIADLMTQ AGVEELEGSG SRSGGHHHHH H; SEQ ID NO:18) was observed. The remaining volume of the 100 Î¼l reaction was precipitated with 200 Î¼l absolute ethanol. The pellet was resuspended in 10 Î¼l water to be used for recombining into Smal-cleaved recipient vector pTAP238 to produce the construct encoding TACI-HSBP-1.<br>
One hundred microliters of competent yeast cells (S. cerevisiae) were combined with 10Î¼l of a mixture containing approximately 1 Î¼g of each of the human TACI-HSBP-1 fragments (PCR products) and 100 Î¼g of SmaI-digested pTAP238 vector, and transferred to a 0.2-cm electroporation cuvette. The yeast/DNA mixture was electropulsed using instrument settings of 0.75 kV (5 kV/cm), infinite ohms, 25Î¼F, and then 600Î¼l of 1.2 M sorbitol were added to the cuvette. The yeast was then plated in two 300-Î¼l aliquots onto two -URA D (glucose-containing media lacking uracil) plates and incubated at 30Â°C. After about 48 hours, the Ura+ yeast transformants from a single plate were resuspended in 1 ml of water and spun briefly to pellet the cells. The cell pellet was resuspended in 1 ml of lysis buffer. DNA was recovered as disclosed above. The DNA pellet was resuspended in 100Î¼l of water.<br>
FortyÎ¼l of electrocompetent E. coli MCI061 cells were transformed with 1 Î¼l of the yeast DNA. The cells were electropulsed at 2.0 kV, 25 Î¼F and 400 ohms. Following electroporation, 0.6 ml SOC (2% Bactoâ¢ Tryptone (Difco, Detroit, MI), 0.5% yeast extract (Difco), 10 mM NaCl, 2.5 mM KC1, 10 mM MgCl2, 10 mM MgSO4, 20 mM glucose) was added to the cells. The cells were allowed to recover at 37Â°C for one hour, then were plated in one aliquot on LB + kanamycin plates (LB broth (Lennox), 1.8% Bactoâ¢ Agar (Difco), 30 mg/L kanamycin).<br>
Individual clones harboring the correct expression construct for human TACI-HSBP-1 were identified by diagnostic digest of the plasmid DNA. Cells were grown in Super Broth II (Becton Dickinson) with 30 Î¼g/ml of kanamycin overnight. The next day, the cells were harvested, and plasmid DNA was prepared using spin columns (QIAprepÂ® Spin Miniprep Kit; Qiagen Inc., Valencia, CA). The DNA was then cleaved with Notl and Xbal. The clones with the correct restriction pattern were designated pTAP415 and sequenced. The polynucleotide sequence of TACI-HSBP-1 in pTAP415 is shown in SEQ ID NO: 17.<br><br>
Ten microliters of pTAP415 were cleaved with two microliters of Noil in 3 \x\ of a commercially available buffer (buffer 3; New England Biolabs) and 15 Î¼l of water for one hour at 37Â°C. Seven microliters of the reaction mixture were combined with two microliters of 5x T4 DNA ligase buffer (Life Technologies; Gaithersburg, MD) and one microliter of T4 DNA ligase and incubated at room termperature for one hour. One microliter of the ligation mixture was used to transform E. coli strain W3110 (ATCC 27325). The cells were electropulsed at 2.0 kV, 25 Î¼F, and 400 ohms. Following electroporation, 0.6 ml of SOC was added to the cells. The cells were grown at 37Â°C for one hour, then plated in one aliquot on LB + kanamycin plates.<br>
Individual colonies were picked and grown. Plasmid DNA was prepared using spin columns. The DNA was cut diagnostically with PVMII and HindIIl to confirm the loss of yeast URA3 and CEN/ARS elements. An individual colony was picked. Cells were grown in Superbroth II (Becton Dickinson) containing 30 Î¼g/ml of kanamycin overnight. One hundred microliters of the overnight culture were used to inoculate two milliliters of fresh Superbroth II containing 30 Î¼g/rnl kanamycin. Cultures were grown at 37Â°C with shaking for about 2 hours in 15-ml conical tubes. One ml of the culture was induced with 1 mM EPTG. Two hours and 15 minutes later, an equal volume of culture was mixed with 250 Î¼l of Thorner buffer (8M urea, lOOmM Tris pH 7.0, 10% glycerol, 2mM EDTA, 5% SDS) with 5% Î²ME and dye. Samples were boiled for five minutes. Twenty-Î¼l samples were loaded on a 4%-12% PAGE gel (NOVEX). Gels were run in 1 x MES buffer. Expression was analyzed by Coomassie Blue staining.<br>
Bacterial cells were lysed using a French press, and inclusion bodies in the cell lysate were pelleted by low-speed centrifugation. The pellet fraction was washed with 2M urea to remove contaminants including membrane and cell wall material. TACI-HSBP-1E. coli inclusion bodies were then extracted overnight with stirring at 4Â°C in 7 M guanidine HC1 in 50 mM Tris pH 8 containing 0.1 M sodium sulfite and 0.05 M sodium tetrathionate. Extraction with the denaturant/sulfitolysis reagents simultaneously dissociates protein-protein interactions and unfolds the protein to monomer with sulfhydryl groups in the reduced state and sulphonated state. Before refolding, samples were centrifuged for 30 minutes at 35,000xg at 4Â°C and filtered with a 0.2 Î¼m filter. Concentrations were estimated by a RP HPLC assay.<br>
EXAMPLE 5 Biological Assay for TACI-fusion Proteins<br>
A Jurkat cell line transfected with human TACI and with KZ142 Luciferase was used to test the ability of various trimerizing TACI constructs to<br><br>
neutralize zTNF4 acitvity. The Jurkat TACI KZ142 Luciferase cells were grown and assayed in growth media (RPMI/10%FBS with L-glutamine, sodium pyruvate, 0.5 mg/ml G418, and 2 Î¼g/ml puromycin). These cells were plated at 40,000 cells/96-well in 100 Î¼l of growth media. One hundred microliters per well of trimerizing inhibitor weres added in the presence of 100 ng/ml of zTNF4.<br>
The assay cell line showed a maximal approximate 18-fold luciferase responsiveness to 1000 ng/ml of zTNF4. One hundred nanograms per milliliter of zTNF4 gave an approximate 10-fold luciferase responsiveness compared to the background control well. The combination of zTNF4 and inhibitor in a total of 200Î¼l of growth media was incubated for six hours at 37Â°C in a 5% CO2 incubator. The 96-well plate was then centrifuged at 2000xg in a Beckman GS-6KR centrifuge and media were discarded by a quick inversion of the plate. Twenty five microliters. of lysis buffer (Promega E153A) were added to each well and incubated for 15 minutes at room temperature. The lysed cells were then transferred to an opaque 96-well plate for purposes of luminometer readings. One bottle of luciferase assay buffer (Promega 152A) was added to one bottle of luciferase assay substrate (Promega E151A) and 40 Î¼l of this combination were added to each well. Each well was read on a luminometer (EG&amp;G Berthold Microlumat Plus) with five seconds of integration.<br>
As shown in Figure 1, a control immunoglobulin fusion protein (hwsxll-IgG) had little effect on the zTNF4-induced luciferase activity. Although a TACI-Fc immunoglobulin fusion protein (TACI-IgG) inhibited luciferase activity, greater inhibition was achieved with TACI-HSBP-1 proteins produced in mammalian cells or in E. coli.<br>
From the foregoing, it will be appreciated that, although specific embodiments of the invention have been described herein for purposes of illustration, various modifications may be made without deviating from the spirit and scope of the invention. Accordingly, the invention is not limited except as by the appended claims.<br><br>
CLAIMS <br>
We claim:<br>
1.	An isolated polypeptide, comprising (1) an extracellular domain of the<br>
transmembrane   activator   and   CAML   (calcium-signal   modulating   cyclophilin   ligand)<br>
interactor (TACI), and (2) a trimerizing polypeptide.<br>
2.	A homotrimeric protein complex, comprising the polypeptide of claim<br>
1.<br>
3.	The isolated polypeptide of claim 1, wherein the TACI extracellular<br>
domain is selected from the group consisting of: (1) amino acid residues 30 to 110 of SEQ ID<br>
NO:4, (2) amino acid residues 1 to 110 of SEQ ID NO:4, (3) amino acid residues 30 to 154 of<br>
SEQ ID NO:4, and (4) amino acid residues 1 to 154 of SEQ ID NO:4.<br>
4.	The   isolated   polypeptide   of   claim   1,   wherein   the   trimerizing<br>
polypeptide comprises the NC-1 fragment of human collagen X.<br>
5.	The   isolated   polypeptide   of  claim   4,   wherein   the   trimerizing<br>
polypeptide comprises the amino acid sequence of SEQ ID NO:20.<br>
6.	The isolated polypeptide of claim 5, wherein the TACI extracellular<br>
domain comprises the amino acid residues 30 to 110 of SEQ ID NO: 4.<br>
7.	A homotrimeric protein complex, comprising the polypeptide of claim<br>
6.<br>
8.	The   isolated   polypeptide   of   claim   1,   wherein   the   trimerizing<br>
polypeptide is a trimerizing fragment of Heat Shock Binding Protein-1.<br>
9.	The   isolated   polypeptide   of   claim   8,   wherein   the   trimerizing<br>
polypeptide has the amino acid sequence of SEQ ID NO:22.<br><br>
10.	The isolated polypeptide of claim 9, wherein the TACI extracellular<br>
domain comprises the amino acid residues 30 to 110 of SEQ ID NO: 4.<br>
11.	A homotrimeric protein complex, comprising the polypeptide of claim<br>
10.<br><br>
12.	An   expression   vector  comprising  the  following  operably   linked<br>
elements:<br>
a transcription promoter;<br>
the nucleic acid sequence encoding the polypeptide of claim 1 and<br>
a transcription terminator.<br>
13.	A cultured cell into which has been introduced the expression vector of<br>
claim 12, wherein said cell expresses said polypeptide.<br>
14.	A method of producing a homotrimeric protein complex comprising<br>
the steps of culturing the cell of claim 13 and recovering the homotrimeric protein complex<br>
comprising said polypeptide.<br>
15.	A method of inhibiting TNF4-induced B cell proliferation comprising<br>
exposing said B cells to a homotrimeric protein complex comprising a polypeptide, said<br>
polypeptide comprising (1) a TACI extracellular domain and (2) a trimerizing polypeptide.<br>
16.	The method of claim 15 wherein said homotrimeric complex is the<br>
complex of claim 7.<br>
17.	The method of claim 15 wherein said homotrimeric complex is the<br>
complex of claim 11.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTc0LUNIRU5QLTIwMDUgIEFCU1RSQUNULnBkZg==" target="_blank" style="word-wrap:break-word;">574-CHENP-2005  ABSTRACT.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTc0LUNIRU5QLTIwMDUgIENMQUlNUy5wZGY=" target="_blank" style="word-wrap:break-word;">574-CHENP-2005  CLAIMS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTc0LUNIRU5QLTIwMDUgIENPUlJFU1BPTkRFTkNFIE9USEVSUy5wZGY=" target="_blank" style="word-wrap:break-word;">574-CHENP-2005  CORRESPONDENCE OTHERS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTc0LUNIRU5QLTIwMDUgIENPUlJFU1BPTkRFTkNFIFBPLnBkZg==" target="_blank" style="word-wrap:break-word;">574-CHENP-2005  CORRESPONDENCE PO.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTc0LUNIRU5QLTIwMDUgIEZPUk0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">574-CHENP-2005  FORM-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTc0LUNIRU5QLTIwMDUgIFBFVElUSU9OUy5wZGY=" target="_blank" style="word-wrap:break-word;">574-CHENP-2005  PETITIONS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTc0LWNoZW5wLTIwMDUtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">574-chenp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTc0LWNoZW5wLTIwMDUtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">574-chenp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTc0LWNoZW5wLTIwMDUtY29ycmVzcG9uZG5lY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">574-chenp-2005-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTc0LWNoZW5wLTIwMDUtY29ycmVzcG9uZG5lY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">574-chenp-2005-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTc0LWNoZW5wLTIwMDUtZGVzY3JpcHRpb24oY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">574-chenp-2005-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTc0LWNoZW5wLTIwMDUtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">574-chenp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTc0LWNoZW5wLTIwMDUtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">574-chenp-2005-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTc0LWNoZW5wLTIwMDUtZm9ybSAxOC5wZGY=" target="_blank" style="word-wrap:break-word;">574-chenp-2005-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTc0LWNoZW5wLTIwMDUtZm9ybSAyNi5wZGY=" target="_blank" style="word-wrap:break-word;">574-chenp-2005-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTc0LWNoZW5wLTIwMDUtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">574-chenp-2005-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTc0LWNoZW5wLTIwMDUtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">574-chenp-2005-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTc0LWNoZW5wLTIwMDUtcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">574-chenp-2005-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="222339-an-invtro-method-of-detecting-the-presence-of-an-oncogenic-human-papilloma-virus.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="222341-a-process-for-reforming-an-alcohol.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>222340</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>574/CHENP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>47/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Nov-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>05-Aug-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>07-Apr-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>ZYMOGENETICS, INC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1201 EASTLAKE AVENUE EAST, SEATTLE, WASHINGTON 98102,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>WEST, JAMES, W</td>
											<td>7539 19TH NE, SEATTLE, WA 98115,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>BRANDT, CAMERON, S</td>
											<td>6828 54TH AVENUE NE, SEATTLE, WA 98115,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>JASPERS, STEPHEN, R</td>
											<td>5829 150TH PLACE SW, EDMONDS, WA 98026,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US03/32878</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-10-10</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/417,801</td>
									<td>2002-10-11</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/222340-production-of-homotrimeric-fusion-proteins by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:38:50 GMT -->
</html>
