<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/222499-methods-and-microorganisms-for-production-of-panto-compounds by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:24:49 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 222499:METHODS AND MICROORGANISMS FOR PRODUCTION OF PANTO-COMPOUNDS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHODS AND MICROORGANISMS FOR PRODUCTION OF PANTO-COMPOUNDS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention features methods of producing panto-compounds (e.g. pantothenate) using microorganisms in which the pantothenate biosynthetic pathway and/or the isoleucine-valine biosynthetic pathway and/or the coenzymeA biosynthetic pathway has been manipulated. Methods featuring ketopantoate reductase overexpressing microorganisms as well as aspartate alpha -decarboxylase overexpressing microorganisms are provided. Methods of producing panto-compounds in a precursor-independent manner and in high yield are described. Recombinant microorganisms, vectors, isolated nucleic acid molecules, genes and gene products useful in practicing the above methodologies are also provided. The present invention also features a previously unidentified microbial pantothenate kinase gene, coaX, as well as methods of producing panto-compounds utilizing microorganisms having modified pantothenate kinase activity. Recombinant microorganisms, vectors, isolated coaX nucleic acid molecules and purified CoaX proteins are featured. Also featured are methods for identifying pantothenate kinase modulators utilizing the recombinant microorganisms and/or purified CoaX proteins of the present invention.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
METHODS AND MICROORGANISMS FOR PRODUCTION OF PANTO-COMPOUNDS<br>
Background of the Invention<br>
Pantothenate, also known as pantothenic acid or vitamin B5, is a member of the B complex of vitamins and is a nutritional requirement for mammals, including livestock and humans (e.g., from food sources, as a water soluble vitamin supplement or as a feed additive). In cells, pantothenate is used primarily for the biosynthesis of coenzyme A (CoA) and acyl carrier protein (ACP). These coenzymes function in the metabolism of acyl moieties which form thioesters with the sulfhydryl group of the 4'-phosphopantetheine portion of these molecules. These coenzymes are essential in all cells, participating in over 100 different intermediary reactions in cellular metabolism.<br>
The conventional means of synthesizing pantothenate (in particular, the bioactive<br>
D isomer) is via chemical synthesis from bulk chemicals, a process which is hampered<br>
by excessive substrate cost as well as the requirement for optical resolution of racemic<br>
intermediates (e.g., resolution of DL-pantolactone to obtain D-pantolactone for chemical<br>
condensation with p-alanine). Accordingly, researchers have recently looked to<br>
bacterial or microbial systems that produce enzymes useful in pantothenate biosynthesis<br>
processes (as bacteria are themselves capable of synthesizing pantothenate). In<br>
particular, bioconversion processes have been evaluated as a means of favoring<br>
production of the D isomer of pantothenic acid, e.g., using microorganisms which<br>
selectively hydrolyze a DL-pantothenic acid ester to D-pantothenic acid;<br>
microorganisms which selectively decompose L-pantolactone resulting in D-<br>
pantolactone alone; and microorganisms which selectively hydrolyze DL-pantolactone<br>
to D-pantoic acid.	-<br>
There is still, however, significant need for improved pantothenate production processes, in particular, for processes requiring reduced quantities of substrates and/or less expensive substrates. To this end, methods of direct microbial synthesis have recently been examined as a means of improving D-pantothenate production. In microbes, pantothenate biosynthetis is a multistep pathway resulting in condensation of pantoate (derived from a-ketoisovalerate) and p-alanine to form D-pantothenate. The isoleucine-valine (ilv) pathway biosynthetic enzymes, acetohydroxyacid synthetase (the ilvBN or alsS gene product), acetohydroxyacid isomeroreductase (the ilvC gene product) and dihydroxyacid dehydratase (the ilvD gene product) catalyze the conversion of pyruvate to a-ketoisovalerate. The reactions are further catalyzed by the pantothenate (pan) pathway biosynthetic enzymes ketopantoate hydroxymethyltransferase (the panB gene product), ketopantoate reductase (IhtpanE gene product), aspartate-a-<br><br>
decarboxylase (the panD gene product) and pantothenate synthetase (ihtpanC gene product).<br>
The genes encoding the enzymes involved in the biosynthesis of pantothenic acid in Salmonella typhimuhum and Escherichia coli have recently been identified and characterized (Frodyma and Downs (1998) J. Biol Chem. 273:5572-5576 and Jackowski (1996) pp. 687-694, In Neidhardt et al (ed.) Escherichia coli and Salmonella: Cellular and Molecular Biology, 2nd ed. Am. Soc. Microbiol Wash, D.C). In E, coli, for example, the biosynthesis of pantothenic acid consists of four key steps. The first reaction is csftalyzed by the panB gene product,- ketopantoate hydroxymethyltransferase, and uses the L-valine intermediate a-ketoisovalerate to generate ketopantoate, which is subsequently reduced to pantoate by \hspanE gene product, ketopantoate reductase. The panD gene product, aspartate-a-decarboxylase, generates P-alanine from aspartate. The panC gene product, pantothenate synthetase, subsequently ligates P-alanine with pantoate to yield D-pantothenate.<br>
The authors Dusch et al. described the identification of the Corynebacterium glutamicum panD gene and reported that expression of the C. glutamicum panD gene in E. coli yielded a strain producing pantothenate with a specific productivity of 140 ng of pantothenate per mg (dry weight) per hour. (Dusch et al (1999) Appl Environ, Microbiol 65:1530-1539).<br>
The authors Sahm and Eggeling have further identified the Corynebacterium glutamicum panB and pan C genes and have described a genetically engineered strain of C. glutamicum which overexpresses the panBC genes (Sahm and Eggeling (1999) Appl Environ. Microbiol 65:1973-1979). The engineered strain produces pantothenate, however, it was necessary to overexpress the genes responsible for a-ketoisovalerate production in the host organism in order that pantothenic acid production could be detected. Moreover, without the addition of P-alanine, no substantial amounts of pantothenate accumulated with the strain constructed.<br>
Likewise, a method of producing D-pantothenic acid has been described that takes advantage of a sodium salicylate resistant mutant strain of E. coli which produces D-pantothenic acid when cultured in the presence of p-alanine (U.S. Patent No. 5,518,906). Generation of E. coli strains resistant to a-ketoisovaleric acid and/or a-ketobutyric acid, and/or a-aminobutyric acid, and/or p-hydroxyaspartic acid and/or O-methyl-threonine, in addition to salicylic acid, further increased pantothenic acid production. Moreover, transformation of a plasmid DNA carrying the panB, panC and panD genes into the salicylic acid resistant mutant strain resulted in increased pantothenate production, however, up to 20 g/L p-alanine or more was fed in the examples given. The panB-panC-panD genes are clustered on the E. coli chromosome.<br><br>
Finally, a method of producing D-pantothenic acid has been described which utilizes a salicylic acid-resistant, a-ketoisovalerate-resistant, a-ketobutyrate-resistant, 0-hydroxyaspartate-resistant, o-methylthreonine-resistent E. coli strain transformed with pantothenate biosynthesis gene-containing DNA fragments and/or branched amino acid biosynthesis gene-containing DNA fragments and cultured in the presence of p-alanine (U.S. Patent No. 5,932,457).<br>
Pantothenate production in bacteria results from the condensation of pantoate and P-alanine and involves the pantothenate biosynthetic enzymes ketopantoate hydroxymethyltransferase (thcpanB gene product), ketopantoate reductase (thepanE gene product), aspartate-a-decarboxylase (tiiepanD gene product) and pantothenate synthetase (thepanC gene product). Although pantothenate is biologically active as a vitamin, it is further metabolized in all cells to Coenzyme A (CoA) which participates as an acyl group carrier in the tricarboxylic acid (TCA) cycle, fatty acid metabolism and numerous other reactions of intermediary metabolism. The initial (and possibly rate-controlling) step in the conversion of pantothenate to Coenzyme A (CoA) is phosphorylation of pantothenate by pantothenate kinase. A pantothenate kinase activity was first identified in Salmonella typhimurium by screening for temperature-sensitive mutants which synthesized CoA at permissive temperatures but excreted pantothenate at non-permissive temperatures. The mutations were mapped in the Salmonella chromosome and the genetic locus was designated coaA. The gene encodes the enzyme that catalyzes the first step in the biosynthesis of coenzyme A from pantothenate (Dunn and Snell (1979) J. Bacteriol 140:805-808), Escherichia coli temperature sensitive mutants have also been isolated and characterized (Vallari and Rock (1987) J. Bacteriol 169:5795-5800). These mutants (named coaA15(Ts)) arexlefective in the conversion of pantothenate to CoA and further exhibit a temperature-sensitive growth phenotype, indicating that pantothenate kinase activity is essential for growth. Moreover, it was noted that CoA inhibited pantothenate kinase activity to the same degree in the mutant as compared to the wild-type enzyme.<br>
Feedback resistant E. coli mutants (named coaA16(Fr)) have also been isolated that posses a pantothenate kinase activity that is refractory to feedback inhibition by CoA (Vallari and Jackowski (1988) J. Bacteriol. 170:3961-3966). The mutation responsible for the reversion is, suprisingly, not genetically linked to the coaA gene by transduction. Additional data described therein support the view that the total cellular CoA content is controlled by both modulation of biosynthesis at the pantothenate kinase step and possibly by degradation of CoA to 4,-phosphopantetheine.<br><br>
The wild-type E. coli coaA gene was cloned by functional complementation of E, coli temperature-sensitive mutants. The sequence of the wild-type gene was determined (Song and Jackowski (1992) J. Bacteriol 174:6411-6417 and Flamm et al (1988) Gene (Amst.) 74:555-558). Strains containing multiple copies of the coaA gene possessed 76-fold higher specific activity of pantothenate kinase, however, there was only a 2.7-fold increase in the steady state level of CoA (Song and Jackowski, supra). It has farther been reported that the prokaryotic enzyme (encoded by coaA in E.coli and a variety of other microorganisms) is feedback inhibited by CoA both in vivo and in vitro with CoA being about five times more potent than acetyl-CoA in inhibiting the enzyme (Song and Jackowski, supra and Vallari et al, supra). Moreover, it has been reported that thepanB gene product in E. coli is inhibited by CoA (Powers and Snell (1976) J. Biol Chem. 251:3786-3793). These data further support the view that feedback inhibition of pantothenate kinase activity is a critical factor controlling intracellular CoA concentration.<br>
Using standard search and alignment tools, coaA homologues have been<br>
identified in Hemophilus influenzae, Mycobacterium tuberculosis, Vibrio cholerae,<br>
Streptococcus pyogenes and Bacillus subtilis. By contrast, proteins with significant<br>
similarity could not be identified in eukaryotic cells including Saccharomyces cerevisiae<br>
or in mammalian expressed sequence tag (EST) databases. Using a genetic selection<br>
strategy, a cDNA encoding pantothenate kinase activity has recently been identified<br>
from Aspergillus nidulans (Cdldcr et al. (\999) J. Biol. Chem. 274:2014-2020). The<br>
eukaryotic pantothenate kinase gene (panK) has distinct primary structure and unique<br>
regulatory properties that clearly distinguish it from its prokaryotic counterpart. A<br>
mammalian pantothenate kinase gene (mpanKla) has aha been isolated which encodes<br>
a protein having homology to the A. nidulans PanK protein and to the predicted gene<br>
product of GenBank™ Accession Number 927798 identified in the 5. cerevisiae        , . w<br>
genome (Rock et al. (2000) J. Biol Chem. 275:1377-1383).	vWl<br>
Summary of the Invention	^	~7^ ^X ^ °       -^—<br>
The present invention is based, at least in part, on the dislov^ry of key enzyme-encoding genes of the pantothenate biosynthetic pathway m Bactllufsubtilis. In particular, the present inventors have identified the panE gene of B. subtilis. Overexpression or deregulation of the panE gene in B. subtilis results in enhanced production of the panE gene product, ketopantoate reductase, further resulting in increased production of pantothenate. Likewise, mutations in this gene reduce pantothenate production in B. subtilis &gt;90%. The present inventors have further identified the presumptive panBCD operon in B. subtilis, overexpression or<br><br>
deregulation of which results in increased pantothenate production. The present inventors have further demonstrated that overexpression or deregulation of the panD gene in B. subtilis (resulting in enhanced production of the panD gene product, aspartate-a-decarboxylase) further results in increased production of pantothenate, in particular, in combination with deregulation of genes encoding key enzymes of the isoleucine-valine (i/v) biosynthetic pathway.<br>
Accordingly, the present invention features methods of producing pantothenate, as well as other compounds of the pantothenate biosynthetic pathway (e.g., ketopantoate, pantoate and p-alanine), termed "panto-compounds" herein, using microorganisms in which the pantothenate biosynthetic pathway and/or isoleucine-valine biosynthetic pathway has been manipulated such that pantothenate or other desired panto-compounds are produced. In one embodiment, the invention features a method of producing a panto-compound^ (e.g., pantothenate or pantoate) that invo]yes_ culturing a microorganism whichjoyerexpresses thepanE gene product, ketopantoate reductase, also referred to herein as a ketopantoate reductase-overexpressing or "KPAR-O" microorganism, under conditions such that the panto-compound (e.g., pantothenate or pantoate) is produced. In another embcdiiwnt,lHepresent invention features a method of producing panto-compounds (e.g., pantothenate or pantoate) which includes culturing a microorganism which overexpresses at least one pantothenate biosynthetic enzyme (e.g., at least one of the panB,panC oxpanD gene products), preferably in a KPAR-0 microorganism, under conditions such that the panto-compound (e.g., pantothenate or pantoate) is produced.<br>
Yet another aspect of the invention features methods of producing panto-compounds which are independent of the need to feed precursors (e.g., p-alanine or aspartate and/or a-ketoisovalerate or valine). In one embodiment, the invention features a method of producing pantothenate in a manner independent of precursor feed that includes culturing an aspartate-a-decarboxylase-overexpressing (AaD-O) microorganism having a deregulated isoleucine-valine (ilv) pathway under conditions such that pantothenate is produced. In another embodiment, the invention features a method of producing pantothenate in a manner independent of precursor feed that includes culturing an AaD-0 microorganism having a deregulated pantothenate (pari) pathway and a deregulated isoleucine-valine (ilv) pathway, under conditions such that pantothenate is produced. In another embodiment, the invention features a method of producing pantothenate in a manner independent of aspartate or p-alanine feed that includes culturing an AaD-0 microorganism under conditions such that pantothenate is produced. In another embodiment, the invention features a method of producing pantothenate in a manner independent of valine or a-ketoisovalerate feed that includes<br><br>
culturing a microorganism having a deregulated isoleucine-valine (i/v) biosynthetic pathway under conditions such that pantothenate is produced. In yet another embodiment, the invention features a high yield production method for producing pantothenate that includes culturing a manipulated microorganism under conditions such that pantothenate is produced at a significantly high yield (e.g., at a level greater than 10 g/L, 20 g/L, 30 g/L or 40g/L).<br>
The methods of the present invention further feature microorganisms that overexpresses acetohydroxyacid synthetase or acetohydroxyacid isomeroreductase (e.g., microorganisms transformed with a vector that includes milvBNC nucleic acid sequence), microorganisms that overexpresses dihydroxyacid dehydratase (e.g., microorganisms transformed with a vector that includes an ilvD nucleic acid sequence), microorganisms that overexpresses aspartate-ct-decarboxylase (e.g., microorganisms transformed with a vector that includes apanD nucleic acid sequence), microorganisms having a deregulated isoleucine-valine (ilv) biosynthetic pathway and microorganisms having a deregulated pantothenate biosynthetic pathway (e.g., microorganisms that overexpress any of ketopantoate hydroxymethyltransferase, ketopantoate reductase, pantothenate synthetase and aspartate-a-decarboxylase, for example, microorganisms transformed with a vector comprising apanBCD nucleic acid sequence or a vector comprising apanEl nucleic acid sequence). In one embodiment, the recombinant microorganism is Gram positive (e.g., microorganisms belonging to the genus Bacillus, Cornyebacterium, Lactobacillus, Lactococci or Streptomyces). In another embodiment, the recombinant microorganism is Gram negative. Particularly preferred is a Bacillus recombinant microorganism (e.g., Bacillus licheniformis, Bacillus amylaliquefaciens, Bacillus subtilis, Bacillus pumilus, Bacillus halodurans,and the like). Recombinant vectors that contain the genes encoding Bacillus pantothenate and/or isoleucine-valine biosynthetic enzymes (e.g., B. subtilis pantothenate and/or isoleucine-valine biosynthetic enzymes) are also described.<br>
Also featured are methods of producing p-alanine that include culturing an aspartate-a-decarboxylase-overexpressing (AaD-O) microorganism under conditions such that p-alanine is produced and methods of producing p-alanine that involve contacting a composition comprising aspartate with an isolated Bacillus aspartate-a-decarboxylase enzyme under conditions such that P-alanine is produced.<br>
The production rflethods of the present invention further can include recovering the panto-compound (e.g., pantothenate or pantoate).<br>
The present invention further features recombinant microorganisms (e.g., AaD-O microorganisms, microorganisms having a deregulated isoleucine-valine (ilv) pathway, microorganisms overexpressing at least one of ketopantoate<br><br>
hydroxymethyltransferase (thepanB gene product), pantothenate synthetase (the panC gene product), aspartate-a-decarboxylase (thcpanD gene product), ketopantoate reductase (the panEl gene product) and microorganisms having a deregulated panBCD operon. Also featured are panB, panC, panD, panE, ilvB, ilvN, alsS, ilvC, and/or ilvD nucleic acid molecules, as well as vectors including such nucleic acid molecules and gene products encoded by such nucleic acid molecules.<br>
The methodology of the present invention further includes, for example in addition to overexpressing at least one pantothenate biosynthetic enzyme, deleting or mutating a second pantothenate biosynthetic enzyme, said second pantothenate biosynthetic enzyme preferably being downstream of the desired product in the pantothenate biosynthetic pathway. For example, mutating panC, in addition to overexpressing the panE gene product, results in even further enhanced or increased production of pantoate. Accordingly, in one embodiment, the invention features a method of producing pantoate which includes culturing a microorganism which overexpresses the panE gene product and which has a deletion in the panC gene. In another embodiment, the invention features a method of producing pantoate which includes culturing a microorganism which overexpresses the panE gene product and/or panB gene product and which has a deletion in the panC gene. Other exemplary embodiments include a method of producing ketopantoate which includes culturing a microorganism which overexpresses the panB gene product and which has a deletion in the panE gene and a method of producing (^alanine which includes culturing a microorganism which overexpresses thepanD gene product and which has a deletion in the panC gene. Also included are methods of producing panto-compounds which include overexpressing at least one valine biosynthetic enzyme in a microorganism which has at least one pantothenate biosynthetic enzyme deletedr<br>
The present invention is also based at least in part, on the identification and characterization of a previously unidentified microbial pantothenate kinase gene, coaX. CoaX was first identified in Bacillus subtilis and corresponds to an open reading frame in a portion of the chromosomal DNA that includes the 5' end of the y?s//gene, and all of the yacB, yacQ yacD, cysK and pabB genes. The present inventors have demonstrated that the yacB open reading frame encodes a novel pantothenate kinase activity, the gene being unrelated by homology to any previously known pantothenate kinase gene. The gene has been renamed coaX, as it encodes the enzyme which catalyzes the first step in the pathway from pantothenate to CoaA.<br>
Accordingly, the present invention features new and improved methods of producing pantothenate and other key compounds of the pantothenate biosynthetic pathway {e.g., panto-compounds) utilizing microorganisms having modified<br><br>
pantothenate kinase activity. In particular, the present invention features recombinant microorganisms that contain the coaX gene or that contain a mutant coaXgene, having reduced pantothenate kinase activity. In one embodiment, the invention features such recombinant microorganisms further having a deregulated pantothenate biosynthetic pathway. In another embodiment, the invention features such recombinant microorganisms further having a deregulated isoleucine-valine (i/v) pathway. In a preferred embodiment, the microorganisms belong to the genus Bacillus (e.g., B. subtilis).<br>
The present invention also features recombinant microorganisms (e.g., microorganisms belonging to the genus Bacillus, for example, B. subtilis) that contain the coaA gene or that contain a mutant coaA gene, optionally including a cooV gene or mutant thereof, having reduced pantothenate kinase activity. In one embodiment, the invention features such recombinant microorganisms further having a deregulated pantothenate biosynthetic pathway or having a deregulated isoleucine-valine (ilv) pathway.<br>
Also featured are vectors that contain isolated coaX or coaA genes as well as mutant coaX and/or coaA genes. Isolated nucleic acid molecules that contain isolated coaX genes or mutant coaX genes are featured in addition to isolated CoaX proteins and mutant CoaX proteins.<br>
The nucleic acids, vectors and recombinant microorganisms described above are particularly useful in the methodologies of the present invention. In particular, the invention features methods of enhancing panto-compound production (e.g., ketopantoate, pantoate and or pantothenate production) that include culturing a recombinant microorganism having a mutant coaX gene under conditions such that panto-compound production is enhanced. In one embodiment, the recombinant microorganism further includes a mutant coaA gene. In another embodiment, the recombinant microorganism further includes a mutant avtA and/or mutant ilvE gene and/or mutant ansB gene and/or mutant alsD gene. Also featured are methods for identifying pantothenate modulators utilizing the recombinant microorganisms and purified CoaX proteins of the present invention.<br>
Other features and advantages of the invention will be apparent from the following detailed description and claims.<br><br>
Brief Description of the Drawings<br>
Figure 7 is a schematic representation of the pantothenate biosynthetic pathway.<br>
Figure 2 is a schematic representation of the plasmid pAN240, containing sequences ligated upstream of the P26panBCD cassette, equivalent to the integrated version in strain PA221.<br>
Figure 3A is a schematic representation of the plasmid pAN004, containing the panBCD operon expressed from P?6 and RBS1.<br>
Figure $B is a schematic representation of the plasmid pAN006, containing the panBCD operon expressed from P26 and RBS2.<br>
Figure 4 is a schematic representation of the plasmid pAN236, containing an integratable and amplifiable P26-KBS2-panEl expression cassette.<br>
Figure 5 is a schematic representation of the construction of plasmid pAN423.<br>
Figure 6 is a schematic representation of the construction of plasmids pAN426 and pAN427.<br>
Figure 7 is a schematic representation of the construction of plasmids pAN428 and pAN429.<br>
Figure 8 is a schematic representation of the construction of plasmid pAN431.<br>
Figure 9 is a schematic representation of the construction of plasmid pAN441.<br>
Figure 10 is a schematic representation of the construction of plasmid pAN440.<br>
Figure 77 is a schematic representation of the plasmid pAN251 designed to integrate a single copy of a P26-p
Figure 72 is a schematic representation of the plasmid pAN267 designed to integrate a single copy of a P26-UVBNC cassette at the amyE locus.<br>
Figure 13 is a schematic representation of the plasmid pAN257, a clone of<br>
Bacillus subtilis ilvD in a low copy vector.	-<br>
Figure 14 is a schematic representation of the plasmid pAN263, designed to integrate a single copy of a P^-iVvZ) cassette at the ilvD locus.<br>
Figure 75 is a schematic representation of the plasmid pAN261, designed to disrupt the Bacillus subtilis ilvD gene with the cat gene.<br>
Figure 76 is a schematic representation of the Coenzyme A biosynthetic pathway in E. coli.<br>
Figure 77 is a schematic representation of the structure of pAN296, a plasmid designed to delete most of the B. subtilis coaA gene and substitute a chloramphenicol resistance gene.<br>
Figure 18 is a schematic representation of the structure of the Bacillus subtilis genome in the region of the coaA gene. The scale is in base pairs and the significant open reading frames are shown by open arrows.<br><br>
Figure 19 is a schematic representation of the plasmid pAN281, a plasmid for expressing Bacillus subtilis coaA after integration at the bpr locus.<br>
Figure 20A-B depicts a multiple sequence alignment (MSA) of the amino acid sequences encoded by six known or predicted microbial coaA genes. SEQ ID NOs:4-6 and 1-3 correspond to the amino acid sequences of Mycobacterium leprae (SwissProt™ Accession No. Q9X795), Mycobacterium tuberculosis (SwissProt™ Accession No. 053440), Streptomyces coelicolor (SwissProt™ Accession No. 086799), Haemophilus influenzae (SwissProt™ Accession No. P44793), Escherichia coli SwissProt™ Accession No. PI5044) and Bacillus subtilis (SwissProt™ Accession No. P54556), respectively. The alignment was generated using ClustalW MSA software at the GenomeNet CLUSTALW Server at the Institute for Chemical Research, Kyoto University. The following parameters were used: Pairwise Alignment, K-tuple (word) size = 1, Window size = 5, Gap Penalty = 3, Number of Top Diagonals = 5, Scoring Method = Percent; Multiple Alignment, Gap Open Penalty =10, Gap Extension Penalty = 0.0, Weight Transition = No, Hydrophilic residues = Gly, Pro, Ser, Asn, Asp, Gin, Glu, Arg and Lys, Hydrophobic Gaps = Yes; and Scoring Matrix = BLOSUM.<br>
Figure 21 is a schematic representation of the structure of the Bacillus subtilis genome in the region of the coaX (yacB) gene. The scale is in base pairs, the significant open reading frames are shown by open arrows and certain predicted restriction fragments are indicated by thick bars.<br>
Figure 22 is a schematic representation of the structure of pAN341 and pAN342, two independent PCR-derived clones of B. subtilis yacB (remaned herein as coaX).<br>
Figure 23A-D depicts a multiple sequence alignment (MSA) of the amino acid sequences encoded by fourteen known or predicted microbial coaX genes. SEQ ID NOs:9, 74, 7-8, 75, 11, 10 and 12-18 correspond to the amino acid sequences of Bacillus subtilis (SwissProt™ Accession No. P37564), Clostridium acetobulyticum (WIT™ Accession No. RCA03301, Argonne National Laboratories), Streptomyces coelicolor (PIR™ Accession No. T36391), Mycobacterium tuberculosis (SwissProt™ Accession No. 006282), Rhodobacter capsulatus (WIT™ Accession No. RRC02473), Desulfovibrio vulgaris (DBJ™ Accession No. BAA21476.1), Deinococcus radiodurans (SwissProt™ Accession No. Q9RX54), Thermotoga maritima (GenBank™ Accession No. AAD35964.1), Treponema pallidum (SwissProt™ Accession No. 083446), Borrelia burgdorferi (SwissProt™ Accession No.051477), Aquifex aeolicus (SwissProt™ Accession No. 067753), Synechocystis sp. (SwissProt™ Accession No. P74045), Helicobacter pylori (SwissProt™ Accession No. 025533), and Bordetella pertussis (SwissProt™ Accession No. Q45338), respectively. The alignment was generated using ClustalW MSA software at the GenomeNet CLUSTALW Server at the<br><br>
Institute for Chemical Research, Kyoto University. The following parameters were used: Pairwise Alignment, K-tuple (word) size = 1, Window size = 5, Gap Penalty = 3, Number of Top Diagonals = 5, Scoring Method = Percent; Multiple Alignment, Gap Open Penalty = 10, Gap Extension Penalty = 0.0, Weight Transition = No, Hydrophilic residues = Gly, Pro, Ser, Asn, Asp, Gin, Glu, Arg and Lys, Hydrophobic Gaps = Yes; and Scoring Matrix = BLOSUM<br>
Figure 24 depicts a multiple sequence alignment of a portion of the protein sequences of the coaA gene products from the following microorganisms: Bacillus subtilis, Escherichia-coli, Haemophilus influenzae, Mycobacterium leprae, Mycobacterium tuberculosis, and Streptomyces coelicolor. The residues that are mutated in E. coli coaA15(Ts) and B. subtilis coaA282A are indicated below and above the alignment, respectively. The portions correspond to amino acid residues 168-187 of SEQIDNO:3, 167-186 of SEQ ID NO:2, 165-184 of SEQ ID NO: 1, 169-188 of SEQ IDNO:4, 169-188 of SEQ IDNO:5 and 179-198 of SEQ IDNO:6, respectively.<br>
Figure 25 is a schematic representation of the structure of pAN294, a plasmid for integrating mutagenized B. subtilis coaA at its native locus.<br>
Figure 26 is a schematic representation of the structure of pAN336, a plasmid designed to delete B. subtilis coaX from its chromosomal locus and replace it with a kanamycin resistence gene.<br>
Detailed Description of the Invention<br>
The present invention features new and improved methods of producing pantothenate and other key compounds of the pantothenate biosynthetic pathway (referred to herein as "panto-compounds", for example, pantothenate, ketopantoate, pantoate and p-alanine) using microorganisms in which the pantothenate biosynthetic pathway has been manipulated such that pantothenate or other desired panto-compounds are produced.<br>
The new and improved methodologies of the present invention include methods of producing panto-compounds (e.g., pantothenate) in microorganisms having at least one enzyme of the pantothenate biosynthetic pathway manipulated such that pantothenate or other desired panto-compounds are produced (e.g., produced at an increased level). For example, the invention features methods of producing panto-compounds (e.g., pantothenate) in microorganisms having at least one of ketopantoate hydroxymethyltransferase, ketopantoate reductase, pantothenate synthetase or aspartate-a-decarboxylase manipulated such that pantothenate or other desired panto-compounds are produced. The methodologies of the present invention also include methods of producing panto-compounds (e.g., pantothenate) in microorganisms having at least one<br><br>
valine-isoleucine biosynthetic enzyme, described herein, manipulated such that pantothenate or other desired panto-compounds are produced. For example, the invention features methods of producing panto-compounds (e.g., pantothenate) in microorganisms having at least one of acetohydroxyacid synthetase, acetohydroxyacid isomeroreductase or dihydroxyacid dehydratase manipulated such that pantothenate or other desired panto-compounds are produced.<br>
The invention also features methods of producing panto-compounds that involve culturing a ketopantoate reductase-overexpressing (KPAR-O) microorganism under conditions such that the panto-compound is produced. The invention also features methods of producing pantothenate in a manner independent of precursor feed that involve culturing an aspartate-a-decarboxylase-overexpressing (AaD-O) microorganism under conditions such that pantothenate is produced. Also featured are ^-alanine independent high yield pantothenate production methods as well as methods of producing fJ-alanine. The present invention also features methods for enhancing production of panto-compounds that involve culturing pantothenate kinase mutants. In particular, the present invention features new and improved methods of producing pantothenate and other key compounds of the pantothenate biosynthetic pathway {e.g., panto-compounds) utilizing microorganisms having modified pantothenate kinase activity, for example, microorganisms that include the coaXgenc or that include a mutant cooA'gene, having reduced pantothenate kinase activity.<br>
In order that the present invention may be more readily understood, certain terms are first defined herein.<br>
The term "pantothenate biosynthetic pathway" includes the biosynthetic pathway involving pantothenate biosynthetic enzymes (e.g., polypeptides encoded by biosynthetic enzyme-encoding genes), compounds (e.g., precursors, substrates, intermediates or products), cofactors and the like utilized in the formation or synthesis of pantothenate. The term "pantothenate biosynthetic pathway" includes the biosynthetic pathway leading to the synthesis of pantothenate in a microorganisms (e.g., in vivo) as well as the biosynthetic pathway leading to the synthesis of pantothenate in vitro. Figure 1 includes a schematic representation of the pantothenate biosynthetic pathway. Pantothenate biosynthetic enzymes are depicted in bold and their corresponding genes indicated in italics.<br>
The term "pantothenate biosynthetic enzyme" includes any enzyme utilized in the formation of a compound (e.g., intermediate or product) of the pantothenate »    biosynthetic pathway. According to Figure 1, synthesis of pantoate from a-<br>
ketoi so valerate (a-KIV) proceeds via the intermediate, ketopantoate. Formation of ketopantoate is catalyzed by the pantothenate biosynthetic enzyme ketopantoate<br><br>
hydroxymethyltransferase (thepanB gene product). Formation of pantoate is catalyzed by the pantothenate biosynthetic enzyme ketopantoate reductase (the panE gene product). Synthesis of p-alanine from aspartate is catalyzed by the pantothenate biosynthetic enzyme aspartate-ct-decarboxylase (the panD gene product). Formation of pantothenate from pantoate and p-alanine (e.g., condensation) is catalyzed by the pantothenate biosynthetic enzyme pantothenate synthetase (the panC gene product).<br>
The term "isoleucine-valine biosynthetic pathway" includes the biosynthetic pathway involving isoleucine-valine biosynthetic enzymes (e.g., polypeptides encoded by biosynthetic enzyme-encoding genes), compounds (e.g., precursors, substrates, intermediates or products), cofactors and the like utilized in the formation or synthesis of conversion of pyruvate to valine or isoleucine. The term "isoleucine-valine biosynthetic pathway" includes the biosynthetic pathway leading to the synthesis of valine or isoleucine in a microorganisms (e.g., in vivo) as well as the biosynthetic pathway leading to the synthesis of valine or isoleucine in vitro. Figure 1 includes a schematic representation of the isoleucine-valine biosynthetic pathway. Isoleucine-valine biosynthetic enzymes are depicted in bold italics and their corresponding genes indicated in italics<br>
The term "isoleucine-valine biosynthetic enzyme" includes any enzyme utilized in the formation of a compound (e.g., intermediate or product) of the isoleucine-valine biosynthetic pathway. According to Figure 1, synthesis of valine from pyruvate proceeds via the intermediates, acetolactate, a,p-dihydroxyisovalerate (a,p-DHIV) and ct-ketoisovalerate (a-KIV). Formation of acetolactate from pyruvate is catalyzed by the isoleucine-valine biosynthetic enzyme acetohydroxyacid synthetase (the ilvBN gene product, or alternatively, the alsS gene product). Formation of a,p-DHIV from acetolactate is catalyzed by the isoleucine-valine biosynthetic enzyme acetohydroxyacidisomero reductase (the ilvC gene product). Synthesis of a-KIV from a,p-DHIV is catalyzed by the isoleucine-valine biosynthetic enzyme dihydroxyacid dehydratase (the ilvD gene product). Moreover, valine and isoleucine can be interconverted by branched chain amino acid transaminases.<br>
As used herein, each of ketopantoate, pantoate, p-alanine and pantothenate are "panto-compounds". The term "panto-compound" includes a compound (e.g., a substrate, intermediate or product) in the pantothenate biosynthetic pathway which is downstream from a particular pantothenate biosynthetic enzyme. In one example, a panto-compound is downstream of the pantothenate biosynthetic enzyme ketopantoate hydroxymethyltransferase (tfiepanB gene product) and can include ketopantoate, pantoate and/or pantothenate. In another example, a panto-compound is downstream of the pantothenate biosynthetic enzyme ketopantoate reductase (the panE gene product)<br><br>
and can include pamoate and/or pantothenate. In yet another example, a panto-compound is downstream of the pantothenate biosynthetic enzyme pantothenate synthetase (the panC gene product) and can include pantothenate. In yet another example, a panto-compound is downstream of the pantothenate biosynthetic enzyme aspartate-a-decarboxylase (the panD gene product) and can include p-alanine and/or pantothenate.<br>
Preferred panto-compounds include pantothenate and pamoate. The term "pantothenate" includes the free acid form of pantothenate, also referred to as "pantothenic acid" as well as any salt thereof (e.g., derived by replacing the acidic hydrogen of pantothenate or pantothenic acid with a cation, for example, calcium, sodium, potassium, ammonium), also referred to as a "pantothenate salt". The term "panto-compound" also includes alcohol derivatives of pantothenate. Preferred pantothenate salts are calcium pantothenate or sodium pantothenate. A preferred alcohol derivative is pantothenol. Pantothenate salts and/or alcohols of the present invention include salts and/or alcohols prepared via conventional methods from the free acids described herein. In another embodiment, calcium pantothenate is synthesized directly by a microorganism of the present invention. A pantothenate salt of the present invention can likewise be converted to a free acid form of pantothenate or pantothenic acid by conventional methodology.<br>
The term "pantoate" includes the free acid form of pamoate, also referred to as "pantoic acid" as well as any salt thereof (eg., derived by replacing the acidic hydrogen of pantoate or pantoic acid with a cation, for example, calcium, sodium, potassium, ammonium), also referred to as a "pantoate salt". Preferred pantoate salts are calcium pantoate or sodium pantoate. Pantoate salts of the present invention include salts prepared via conventional methods from the free acids described-herein. A pantoate salt of the present invention can likewise be converted to a free acid form of pantoate or pantoic acid by conventional methodology. Moreover, a free acid form of pantoate or pantoic acid can be converted to pantolactone by conventional methodology.<br>
The term "CoA biosynthetic pathway" includes the biosynthetic pathway involving CoA biosynthetic enzymes (e.g., polypeptides encoded by biosynthetic enzyme-encoding genes), compounds (e.g., precursors, substrates, intermediates or products), cofactors and the like utilized in the formation or synthesis of CoA from pantothenate. A schematic representation of the CoA biosynthetic pathway in E. coli is set forth as Figure 16. (The pathway depicted is also presumed to be that utilized by other microorganisms.) The term "CoA biosynthetic pathway" includes the biosynthetic pathway leading to the synthesis of CoA in microorganisms (e.g., in vivo) as well as the biosynthetic pathway leading to the synthesis of CoA in vitro. The term "Coenzyme A<br><br>
or CoA biosynthetic enzyme" includes any enzyme utilized in the formation of a compound (e.g., intermediate or product) of the CoA biosynthetic pathway, for example, the coaA,panK or coaYgene product which catalyzes the phosphorylation of pantothenate to form 4'-phosphopantothenate, or the coaD gene product which catalyzes the conversion of 4'-phosphopantetheine to dephosphocoenzyme A.<br>
/	Recombinant Microorganisms and Methods for Cultvring<br>
Microorganisms Such That Panto-Compounds are Produced<br>
The methodologies of the present invention feature microorganisms, e.g., recombinant microorganisms, preferably including vectors or genes (e.g., wild-type and/or mutated genes) as described herein and/or cultured in a manner which results in the production of a desired product (e.g. a panto-compound or panto-compounds). The term "recombinant" microorganism includes a microorganism (e.g., bacteria, yeast cell, fungal cell, etc.) which has been genetically altered, modified or engineered (e.g., genetically engineered) such that it exhibits an altered, modified or different genotype and/or phenotype (e.g., when the genetic modification affects coding nucleic acid sequences of the microorganism) as compared to the naturally-occurring microorganism from which it was derived. Preferably, a "recombinant" microorganism of the present invention has been genetically engineered such that it overexpresses at least one bacterial gene or gene product (e.g., a pantothenate or isoleucine-valine biosynthetic enzyme encoding-gene) as described herein, preferably a biosynthetic enzyme encoding-gene included within a recombinant vector as described herein and/or a biosynthetic enzyme expressed from a recombinant vector. The ordinary skilled will appreciate that a microorganism expressing or overexpressing a gene product produces or overproduces the gene product as a result of expression or overexpression of nucleic acid sequences and/or genes encoding the gene product.<br>
The term "manipulated microorganism" includes a microorganism that has been engineered (e.g., genetically engineered) or modified such that the microorganism has at least one enzyme of the pantothenate biosynthetic pathway and/or at least one enzyme of the isoleucine-valine biosynthetic pathway modified such that pantothenate or other desired panto-compounds are produced. Modification or engineering of such microorganisms can be according to any methodology described herein including, but not limited to, deregulation of a biosynthetic pathway and/or overexpression of at least one biosynthetic enzyme. A "manipulated" enzyme (e.g., a "manipulated" biosynthetic enzyme) includes an enzyme, the expression or production of which has been altered or modified such that at least one upstream or downstream precursor, substrate or product<br><br>
of the enzyme is altered or modified, for example, as compared to a corresponding wild-type or naturally occurring enzyme.<br>
The term "overexpressed" or "overexpression" includes expression of a gene product (e.g., a pantothenate biosynthetic enzyme or isoleucine-valine biosynthetic enzyme) at a level greater than that expressed prior to manipulation of the microorganism or in a comparable microorganism which has not been manipulated. In one embodiment, the microorganism can be genetically manipulated (e.g., genetically engineered) to overexpress a level of gene product greater than that expressed prior to manipulation of the microorganism or in a comparable microorganism which has not been manipulated. Genetic manipulation can include, but is not limited to, altering or modifying regulatory sequences or sites associated with expression of a particular gene (e.g., by adding strong promoters, inducible promoters or multiple promoters or by removing regulatory sequences such that expression is constitutive), modifying the chromosomal location of a particular gene, altering nucleic acid sequences adjacent to a particular gene such as a ribosome binding site or transcription terminator, increasing the copy number of a particular gene, modifying proteins (e.g., regulatory proteins, suppressors, enhancers, transcriptional activators and the like) involved in transcription of a particular gene and/or translation of a particular gene product, or any other conventional means of deregulating expression of a particular gene routine in the art (including but not limited to use of antisense nucleic acid molecules, for example, to block expression of repressor proteins).<br>
In another embodiment, the microorganism can be physically or environmentally manipulated to overexpress a level of gene product greater than that expressed prior to manipulation of the microorganism or in a comparable mieroorganism which has not been manipulated. For example, a microorganism can be treated-with or cultured in the presence of an agent known or suspected to increase transcription of a particular gene and/or translation of a particular gene product such that transcription and/or translation are enhanced or increased. Alternatively, a microorganism can be cultured at a temperature selected to increase transcription of a particular gene and/or translation of a particular gene product such that transcription and/or translation are enhanced or increased.<br>
The term "deregulated" or "deregulation" includes the alteration or modification of at least one gene in a microorganism that encodes an enzyme in a biosynthetic pathway, such that the level or activity of the biosynthetic enzyme in the microorganism is altered or modified. Preferably, at least one gene that encodes an enzyme in a biosynthetic pathway is altered or modified such that the gene product is enhanced or increased. The phrase "deregulated pathway" can also include a biosynthetic pathway in<br><br>
which more than one gene that encodes an enzyme in a biosynthetic pathway is altered or modified such that the level or activity of more than one biosynthetic enzyme is altered or modified. The ability to "deregulate" a pathway (e.g., to simultaneously deregulate more than one gene in a given biosynthetic pathway) in a microorganism arises from the particular phenomenon of microorganisms in which more than one enzyme (e.g., two or three biosynthetic enzymes) are encoded by genes occurring adjacent to one another on a contiguous piece of genetic material termed an "operon".<br>
The term "operon" includes a coordinated unit of gene expression that contains a promoter and possibly a regulatory element associated with one or more, preferably at least two, structural genes (e.g., genes encoding enzymes, for example, biosynthetic enzymes). Expression of the structural genes can be coordinately regulated, for example, by regulatory proteins binding to the regulatory element or by anti-termination of transcription. The structural genes can be transcribed to give a single mRNA that encodes all of the structural proteins. Due to the coordinated regulation of genes included in an operon, alteration or modification of the single promoter and/or regulatory element can result in alteration or modification of each gene product encoded by the operon. Alteration or modification of the regulatory element can include, but is not limited to removing the endogenous promoter and/or regulatory element(s), adding strong promoters, inducible promoters or multiple promoters or removing regulatory sequences such that expression of the gene products is modified, modifying the chromosomal location of the operon, altering nucleic acid sequences adjacent to the operon or within the operon such as a ribosome binding site, increasing the copy number of the operon, modifying proteins (e.g., regulatory proteins, suppressors, enhancers, transcriptional activators and the like) involved in transcription of the operon and/or translation of the gene products of the operon, or any other conventional means of deregulating expression of genes routine in the art (including but not limited to use of antisense nucleic acid molecules, for example, to block expression of repressor proteins). Deregulation can also involve altering the coding region of one or more genes to yield, for example, an enzyme that is feedback resistant or has a higher or lower specific activity.<br>
A particularly preferred "recombinant" microorganism of the present invention has been genetically engineered to overexpress a bacterially-derived gene or gene product. The term "bacterially-derived" or "derived-from", for example bacteria, includes a gene which is naturally found in bacteria or a gene product (e.g., ketopantoate hydroxymethyltransferase, ketopantoate reductase, pantothenate synthetase, aspartate-a-decarboxylate, acetohydroxyacid synthetase, acetohydroxyacid isomeroreductase or<br><br>
dihydroxyacid dehydratase) which is encoded by a bacterial gene (e.g., encoded by panB, panE, panC, panD, ilvB, ilvN, alsS, ilvC, or ilvD).<br>
The methodologies of the present invention feature recombinant microorganisms which overexpress at least one of ketopantoate hydroxymethyltransferase, ketopantoate reductase, pantothenate synthetase or aspartate-a-decarboxylase. A particularly preferred recombinant microorganism of the present invention has been genetically engineered to overexpress a Bacillus (e.g., Bacillus licheniformis, Bacillus amyloliquefaciens, Bacillus halodurans, Bacillus subtilis, and Bacillus pumilus, etc.) biosynthetic enzyme (e.g., has been engineered to overexpress at least one of B. subtilis ketopantoate reductase (thepanE gene product) (e.g., ketopantoate reductase having the amino acid sequence of SEQ ID NO:30 or encoded by the nucleic acid sequence of SEQ ID N0.29), B. subtilis ketopantoate hydroxymethyltransferase (thepanB gene product) (e.g., ketopantoate hydroxymethyltransferase having the amino acid sequence of SEQ ID NO:24 or encoded by a nucleic acid molecule having the nucleotide sequence of SEQ ID NO:23), B. subtilis pantothenate synthetase (thepanC gene product) (e.g., pantothenate synthetase having the amino acid sequence of SEQ ID NO:26 or encoded by a nucleic acid molecule having the nucleotide sequence of SEQ ID NO:25) and/or B. subtilis aspartate-a-decarboxylase (thspanD gene product) (e.g., aspartate-a-decarboxylase having the amino acid sequence of SEQ ID NO:28 or encoded by a nucleic acid molecule having the nucleotide sequence of SEQ ID NO:27).<br>
In an exemplary embodiment, the invention features a microorganism (e.g., a KPAR-0 microorganism) that has been transformed with a vector comprising a panE nucleic acid sequence (e.g, a panE nucleic acid seqeunce as set forth in SEQ ID NO:29). In another embodiment, the invention features a-microorganism that has been transformed with a vector comprising apanB nucleic acid sequence (e.g., apanB nucleic acid sequence as set forth in SEQ ID NO:23), a vector comprising apanC nucleic acid sequence (e.g., apanC nucleic acid sequence as set forth in SEQ ID NO:25) or a vector comprising apanD nucleic acid sequence (e.g, apanD nucleic acid sequence as set forth in SEQ ID NO:27). In yet another embodiment, the invention features a microorganism having a deregulatedpanBCD operon (e.g., SEQ ID NO:59).<br>
Other preferred "recombinant" microorganisms of the present invention have a deregulated isoleucine-valine (ilv) pathway. The phrase "microorganism having a deregulated isoleucine-valine (i/v) pathway" includes a microorganism having an alteration or modification in at least one gene encoding an enzyme of the isoleucine-valine (ilv) pathway or having an alteration or modification in an operon including more than one gene encoding an enzyme of the isoleucine-valine (ilv) pathway. A preferred "microorganism having a deregulated isoleucine-valine (i/v) pathway" has been<br><br>
genetically engineered to overexpress a Bacillus (e.g., B. subtilis) ilv biosynthetic enzyme (e.g., has been engineered to overexpress at least one of acetohydroxyacid synthetase (the ilvBN gene products or the alsSgene product) (e.g., acetohydroxyacid synthetase having subunits having the amino acid sequences of SEQ ID NO:32 and SEQ ID NO:34 or encoded by nucleic acid molecules having the nucleotide sequence of SEQ ID NO:31 and SEQ ID NO:33 or the nucleotide sequence of SEQ ID NO:58 from nucleotides 1-2246 or acetohydroxyacid synthetase encoded by a nucleic acid molecule having the nucleotide sequence of SEQ ID NO:86), acetohydroxyacid isomeroreductase (the ilvC gene product) (e.g., acetohydroxyacid isomeroreductase having the amino acid sequence of SEQ ID NO:36 or encoded by a nucleic acid molecule having the nucleotide sequence of SEQ ID NO:35), dihydroxyacid dehydratase (the ilvD gene product) (e.g., dihydroxyacid dehydratase having the amino acid sequence of SEQ ID NO:38 or encoded by a nucleic acid molecule having the nucleotide sequence of SEQ ID NO:37), and/or has been transformed with a vector comprising an ilvBNC nucleic acid sequence (SEQ ID NO:58, coding regions from nucleotides 1-1725, 1722-2246 and 2263-3291) and/or an ilvD nucleic acid sequence (SEQ ID NO:37).<br>
In another preferred embodiment, a recombinant microorganism is designed or engineered such that a mutant CoaA and/or CoaX biosynthetic enzyme is expressed and at least one pantothenate biosynthetic enzyme and/or at least one isoleucine-valine biosynthetic enzyme is overexpressed or deregulated.<br>
In another preferred embodiment, a.microorganism of the present invention overexpresses or is mutated for a gene or biosynthetic enzyme (e.g., a Co A biosynthetic enzyme, pantothenate biosynthetic enzyme or isoleucine-valine biosynthetic enzyme) which is bacterially-derived. The term "bacterially-derived" or "derived-from", for example bacteria, includes a gene product (e.g., ketopantoate hydroxymethyltransferase, ketopantoate reductase, pantothenate synthetase, aspartate-a-decarboxylate, acetohydroxyacid synthetase, acetohydroxyacid isomeroreductase, dihydroxyacid dehydratase or pantothenate kinase) which is encoded by a bacterial gene (e.g.,panB, panE, panC, panD, ilvBN (or alsS), ilvC, ilvD, or encoded by coaA or coaX).<br>
Still other preferred recombinant microorganisms of the present invention are mutant microorganisms. As used herein, the term "mutant microorganism" includes a recombinant microorganism that has been genetically engineered to express a mutated gene or protein that is normally or naturally expressed by the microorganism. Preferably, a mutant microorganism expresses a mutated gene or protein such that the microorganism exhibits an altered, modified or different phenotype (e.g., has been engineered to express a mutated CoaA biosynthetic enzyme, for example, pantothenate kinase). In one embodiment, a mutant microorganism is designed or engineered such<br><br>
that it includes a mutant coaX gene, as defined herein. In another embodiment, a recombinant microorganism is designed or engineered such that it includes a mutant coaA gene, as defined herein. In another embodiment, a mutant microorganism is designed or engineered such that a coaXgem has been deleted (i.e., the protein encoded by the coaXgenc is not produced). In another embodiment, a mutant microorganism is designed or engineered such that a coaA gene has been deleted (i.e., the protein encoded by the coaA gene is not produced). Preferably, a mutant microorganism has a mutant coaX gene or a mutant coaA gene, or has been engineered to have a coaA'gene and/or coaA deleted, such that that the mutant microorganism encodes a "reduced pantothenate kinase activity". In the context of a whole microorganism, a "reduced pantothenate kinase activity" can be determined by measuring or assaying for a decrease in an intermediate or product of the CoA biosynthetic pathway, for example, measuring or assaying for 4,-phosphopantothenate, 4'-phosphopantothenylcysteine, 4'-phosphopantetheine, dephosphocoenzyme A, Coenzyme A, apo-acyl carrier protein (apo-ACP) or holo-acyl carrier protein (ACP) in the microorganism (e.g., in a lysate isolated or derived from the microorganism) or in the medium in which the microorganism is cultured (see e.g., Figure 16). Alternatively, a "reduced pantothenate kinase activity" can be determined by measuring or assaying for decreased growth of the microorganism. Alternatively, a "reduced pantothenate kinase activity" can be determined by measuring or assaying for an increase in a panto-compound (e.g., pantothenate) in the microorganism or surrounding media, as panto-compounds lie upstream of the CoA biosynthetic pathway, the first step of which is catalyzed by pantothenate kinase. The invention also features recombinant microorganisms that, in addition to having reduced pantothenate kinase activity (erg., expressing mutant coaA and/or mutant coaX genes) have a deregulated pantothenate biosynthesis pathway and/or a deregulated isoleucine-valine (i/v) biosynthetic pathway.<br>
In one embodiment, a recombinant microorganism of the present invention is a Gram positive organism (e.g., a microorganism which retains basic dye, for example, crystal violet, due to the presence of a Gram-positive wall surrounding the microorganism). In a preferred embodiment, the recombinant microorganism is a microorganism belonging to a genus selected from the group consisting of Bacillus, Cornyebacterium, Lactobacillus, Lactococci and Streptomyces. In a more preferred embodiment, the recombinant microorganism is of the genus Bacillus. In another preferred embodiment, the recombinant microorganism is selected from the group consisting of Bacillus subtilis, Bacillus lentimorbus, Bacillus lentus, Bacillus firmus, Bacillus pantothenticus, Bacillus amyloliquefaciens, Bacillus cereus, Bacillus circulans, Bacillus coagulans, Bacillus licheniformis, Bacillus megaterium, Bacillus pumilus,<br><br>
Bacillus thuringiensis, and other Group 1 Bacillus species, for example, as characterized by 16S rRNA type (Priest (1993) in Bacillus subtilis and Other Gram-Positive Bacteria eds. Sonenshein et al., ASM, Washington, D.C., p. 6). In another preferred embodiment, the recombinant microorganism is Bacillus brevis or Bacillus stearothermophilus. In another preferred embodiment, the recombinant microorganism is selected from the group consisting of Bacillus licheniformis, Bacillus amyloliquefaciens, Bacillus halodurans, Bacillus subtilis, and Bacillus pumilus.<br>
In another embodiment, the recombinant microorganism is a Gram negative (excludes basic dye) organism. In a preferred embodiment, the recombinant microorganism is a microorganism belonging to a genus selected from the group consisting of Salmonella, Escherichia, Klebsiella, Serratia, and Proteus. In a more preferred embodiment, the recombinant microorganism is of the genus Escherichia. In an even more preferred embodiment, the recombinant microorganism is Escherichia coli. In another embodiment, the recombinant microorganism is Saccharomyces {e.g., S. cerevisiae).<br>
An important aspect of the present invention involves culturing the recombinant microorganisms described herein, such that a desired compound (e.g., a desired panto-compound) is produced. The term "culturing" includes maintaining and/or growing a living microorganism of the present invention (e.g., maintaining and/or growing a culture or strain). In one embodiment, a microorganism of the invention is cultured in liquid media. In another embodiment, a microorganism of the invention is cultured in solid media or semi-solid media. In a preferred embodiment, a microorganism of the invention is cultured in media (e.g., a sterile, liquid media) comprising nutrients essential or beneficial to the maintenance and/or growth of the microorganism (e.g., carbon sources or carbon substrate, for example complex carbohydrates such as bean or grain meal, starches, sugars, sugar alcohols, hydrocarbons, oils, fats, fatty acids, organic acids and alcohols; nitrogen sources, for example, vegetable proteins, peptones, peptides and amino acids derived from grains, beans and tubers, proteins, peptides and amino acids derived form animal sources such as meat, milk and animal byproducts such as peptones, meat extracts and casein hydrolysates; inorganic nitrogen sources such as urea, ammonium sulfate, ammonium chloride, ammonium nitrate and ammonium phosphate; phosphorus sources, for example, phosphoric acid, sodium and potassium salts thereof; trace elements, for example, magnesium, iron, manganese, calcium, copper, zinc, boron, molybdenum, and/or cobalt salts; as well as growth factors such as amino acids, vitamins, growth promoters and the like).<br><br>
Preferably, microorganisms of the present invention are cultured under controlled pH. The term "controlled pH" includes any pH which results in production of the desired product (e.g., a panto-compound). In one embodiment, microorganisms are cultured at a pH of about 7. In another embodiment, microorganisms are cultured at a pH of between 6.0 and 8.5. The desired pH may be maintained by any number of methods known to those skilled in the art.<br>
Also preferably, microorganisms of the present invention are cultured under controlled aeration. The term "controlled aeration" includes sufficient aeration (e.g., oxygen) to result in production of the desired product {e.g., panto-compound). In one embodiment, aeration is controlled by regulating oxygen levels in the culture, for example, by regulating the amount of oxygen dissolved in culture media. Preferably, aeration of the culture is controlled by agitating the culture. Agitation may be provided by a propeller or similar mechanical agitation equipment, by revolving or shaking the growth vessel (e.g., fermentor) or by various pumping equipment. Aeration may be further controlled by the passage of sterile air or oxygen through the medium (e.g., through the fermentation mixture). Also preferably, microorganisms of the present invention are cultured without excess foaming (e.g., via addition of antifoaming agents).<br>
Moreover, microorganisms of the present invention can be cultured under controlled temperatures. The term "controlled temperature" includes any temperature which results in production of the desired product (e.g., a panto-compound). In one embodiment, controlled temperatures include temperatures between 15°C and 95°C. In another embodiment, controlled temperatures include temperatures between 15°C and 70°C. Preferred temperatures are between 20°C and 55°C, more preferably between 30°C and 45°C or between 30°C and 50°C.<br>
Microorganisms can be cultured (e.g., maintained and/or-grown) in liquid media and preferably are cultured, either continuously or intermittently, by conventional culturing methods such as standing culture, test tube culture, shaking culture (e.g., rotary shaking culture, shake flask culture, etc.), aeration spinner culture, or fermentation. In a preferred embodiment, the microorganisms are cultured in shake flasks. In a more preferred embodiment, the microorganisms are cultured in a fermentor (e.g., a fermentation process). Fermentation processes of the present invention include, but are not limited to, batch, fed-batch and continuous methods of fermentation. The phrase "batch process" or "batch fermentation" refers to a closed system in which the composition of media, nutrients, supplemental additives and the like is set at the beginning of the fermentation and not subject to alteration during the fermentation, however, attempts may be made to control such factors as pH and oxygen concentration to prevent excess media acidification and/or microorganism death. The phrase "fed-<br><br>
batch process" or "fed-batch" fermentation refers to a batch fermentation with the exception that one or more substrates or supplements are added (e.g., added in increments or continuously) as the fermentation progresses. The phrase "continuous process" or "continuous fermentation" refers to a system in which a defined fermentation media is added continuously to a fermentor and an equal amount of used or "conditioned" media is simultaneously removed, preferably for recovery of the desired product (e.g., panto-compound). A variety of such processes have been developed and are well-known in the art.<br>
The phrase "culturing under conditions such that a desired compound (e.g., a panto-compound, for example, pantothenate) is produce'd" includes maintaining and/or growing microorganisms under conditions (e.g., temperature, pressure, pH, duration, etc.) appropriate or sufficient to obtain production of the desired compound or to obtain desired yields of the particular compound being produced. For example, culturing is continued for a time sufficient to produce the desired amount of a panto-compound (e.g., pantothenate, pantoate or p-alanine). Preferably, culturing is continued for a time sufficient to substantially reach maximal production of the panto-compound. In one embodiment, culturing is continued for about 12 to 24 hours. In another embodiment, culturing is continued for about 24 to 36 hours, 36 to 48 hours, 48 to 72 hours, 72 to 96 hours, 96 to 120 hours, 120 to 144 hours, or greater than 144 hours. In another embodiment, culturing is continued for a time sufficient to reach production yields of panto-compound, for example, cells are cultured such that at least about 15 to 20 g/L of panto-compound are produced, at least about 20 to 25 g/L panto-compound are produced, at least about 25 to 30 g/L panto-compound are produced, at least about 30 to 35 g/L panto-compound are produced, at least about 35 to-40 g/L panto-compound are produced (e.g., at least about 37 g/L panto-compound) or at least-about 40 to 50 g/L panto compound are produced. In yet another embodiment, microorganisms are cultured under conditions such that a preferred yield of panto-compound, for example, a yield within a range set forth above, is produced in about 24 hours, in about 36 hours, in about 48 hours, in about 72 hours, or in about 96 hours.<br>
The methodology of the present invention can further include a step of recovering a desired compound (e.g., a panto-compound). The term "recovering" a desired compound (e.g., a panto-compound) includes extracting, harvesting, isolating or purifying the compound from culture media. Recovering the compound can be performed according to any conventional isolation or purification methodology known in the art including, but not limited to, treatment with a conventional resin (e.g., anion or cation exchange resin, non-ionic adsorption resin, etc.), treatment with a conventional adsorbent (e.g., activated charcoal, silicic acid, silica gel, cellulose, alumina, etc.),<br><br>
alteration of pH, solvent extraction (e.g., with a conventional solvent such as an alcohol, ethyl acetate, hexane and the like), dialysis, filtration, concentration, crystallization, recrystallization, pH adjustment, lyophilization and the like. For example, a compound (e.g., a panto-compound) can be recovered from culture media by first removing the microorganisms from the culture. Media is then passed through or over a cation exchange resin to remove unwanted cations and then through or over an anion exchange resin to remove unwanted inorganic anions and organic acids having stronger acidities than the panto-compound of interest (e.g., pantothenate). The resulting panto-compound (e.g., pantothenate) can subsequently be converted to a pantothenate salt (e.g., calcium pantothenate) as described herein.<br>
Preferably, a desired compound of the present invention is "extracted", "isolated" or "purified" such that the resulting preparation is substantially free of other components (e.g., free of media components and/or fermentation byproducts). The language "substantially free of other components" includes preparations of desired compound in which the compound is separated (e.g., purified or partially purified) from media components or fermentation byproducts of the culture from which it is produced. In one embodiment, the preparation has greater than about 80% (by dry weight) of the desired compound (e.g., less than about 20% of other media components or fermentation byproducts), more preferably greater than about 90% of the desired compound (e.g., less than about 10% of other media components or fermentation byproducts), still more preferably greater than about 95% of the desired compound (e.g., less than about 5% of other media components or fermentation byproducts), and most preferably greater than about 98-99% desired compound (e.g., less than about 1-2% other media components or fermentation byproducts). When the desired compound is a panto-compound that has been derivatized to a salt (e.g. a pantothenate salt or pamoate salt), the panto-compound is preferably further free (e.g., substantially free) of chemical contaminants associated with the formation of the salt. When the desired compound is a panto-compound that has been derivatized to an alcohol, the panto-compound is preferably further free (e.g., substantially free) of chemical contaminants associated with the formation of the alcohol.<br>
In an alternative embodiment, the desired panto-compound is not purified from the microorganism, for example, when the microorganism is biologically non-hazardous (e.g., safe). For example, the entire culture (or culture supernatant) can be used as a source of product (e.g., crude product). In one embodiment, the culture (or culture supernatant) supernatant is used without modification. In another embodiment, the culture (or culture supernatant) is concentrated. In yet another embodiment, the culture (or culture supernatant) is dried or lyophilized.<br><br>
II.        Panto-Compound Production Methodologies Featuring Ketopantoate Reductase-Overexpressing Microorganisms<br>
One aspect of the invention features methods of producing a panto-compounds<br>
that involve culturing a ketopantoate reductase-overexpressing (KPAR-O)<br>
microorganism under conditions such that the panto-compound is produced. The term "ketopantoate reductase-overexpressing (KPAR-O) microorganism" includes a microorganism which has been manipulated such that ketopantoate reductase is overexpressed (e.g.,ji B. subtilis ketopantoate reductase protein having the amino acid sequence of SEQ ID NO:30) and/or has been transformed with a vector comprising a panEI nucleic acid sequence (e.g., a B. subtilispanE1 nucleic acid sequence as set forth in SEQ ID NO:29). In one embodiment, the panto-compound is pantothenate. In another embodiment, the panto-compound is pantoate. In another embodiment, the ketopantoate reductase is bacterial-derived. In another embodiemnt, the ketopantoate reductase is derived from Bacillus (e.g., is derived from Bacillus subtilis). In yet another embodiment, the KPAR-O microorganism is Gram positive. In yet another embodiment, the KPAR-O microorganism is a microorganism belonging to a genus selected from the group consisting of Bacillus, Cornyebacterium9 Lactobacillus, Lactococci and Streptomyces. In a preferred embodiemnt, the KPAR-O microorganism is of the genus Bacillus. In a more preferred embodiment, the KPAR-O microorganism is selected from the group consisting of Bacillus licheniformis, Bacillus amyloliquefaciens, Bacillus haloduransf Bacillus subtilis and Bacillus pumi I us. In a particularly preferred embodiemnt, the KPAR-O microorganism is Bacillus subtilis.<br>
In still other embodiments, the KPAR-O microorganism further overexpresses at least one pantothenate biosynthetic enzyme in addition to-ketopantoate reductase. In an exemplary embodiment, the KPAR-O microorganism further overexpresses at least one of ketopantoate hydroxymethyltransferase, pantothenate synthetase and aspartate-a-decarboxylase. Also featured are methods of producing panto-compounds, for example, methods that involve culturing a KPAR-O microorganism, which further include the step of recovering the panto-compound.<br>
///       Methods of Producing Panto-Compounds Independent of Precursor Feed Requirements<br>
Depending on the biosynthetic enzyme or combination of biosynthetic enzymes manipulated, it may be desirable or necessary to provide (e.g., feed) microorganisms of the present invention at least one pantothenate biosynthetic precursor such that pantothenate or other desired panto-compounds are produced. The term "pantothenate biosynthetic precursor" or "precursor" includes an agent or compound which, when<br><br>
provided to, brought into contact with, or included in the culture medium of a microorganism, serves to enhance or increase pantothenate biosynthesis. In one embodiment, the pantothenate biosynthetic precursor or precursor is aspartate. In another embodiment, the pantothenate biosynthetic precursor or precursor is p-alanine. The amount of aspartate or P-alanine added is preferably an amount that results in a concentration in the culture medium sufficient to enhance productivity of the microorganism (e.g., a concentration sufficient to enhance production of a panto-compound, for example, P-alanine, ketopantoate, pamoate or pantothenate). Pantothenate biosynthetic precursors of the present invention can be added in the form of a concentrated solution or suspension (e.g., in a suitable solvent such as water or buffer) or in the form of a solid (e.g., in the form of a powder). Moreover, pantothenate biosynthetic precursors of the present invention can be added as a single aliquot, continuously or intermittently over a given period of time.<br>
In yet another embodiment, the pantothenate biosynthetic precursor is valine, see e.g., Example III. In yet another embodiment, the pantothenate biosynthetic precursor is ct-ketoisovalerate. Preferably, valine or a-ketoisovalerate is added in an amount that results in a concentration in the medium sufficient for production of the desired product (e.g., panto-compound) to occur. Pantothenate biosynthetic precursors are also referred to herein as "supplemental pantothenate biosynthetic substrates".<br>
Providing pantothenate biosynthetic precursors in the pantothenate biosynthetic methodologies of the present invention, can be associated with high costs, for example, when the methodologies are used to produce high yields of panto-compounds. Accordingly, preferred methodologies of the present invention feature microorganisms having at least one biosynthetic enzyme or combination of biosynthetic enzymes (e.g., at least one pantothenate biosynthetic enzyme and/or valine-isoleucine biosynthetic enzyme) manipulated such that pantothenate or other desired panto-compounds are produced in a manner independent of precursor feed. The phrase "a manner independent of precursor feed", for example, when referring to a method for producing a desired compound (e.g., a panto-compound), includes an approach to or a mode of producing the desired compound that does not depend or rely on precursors being provided (e.g., fed) to the microorganism being utilized to produce the desired compound. For example, microorganisms featured in the methodologies of the present invention can be used to produce panto-compounds in a manner requiring no feeding of the precursors aspartate, P-alanine, valine and/or a-KJV.<br>
Alternative preferred methodologies of the present invention feature microorganisms having at least one biosynthetic enzyme or combination of biosynthetic enzymes manipulated such that pantothenate or other desired panto-compounds are<br><br>
produced in a manner substantially independent of precursor feed. The phrase "a manner substantially independent of precursor feed" includes an approach to or a method of producing the desired compound that depends or relies to a lesser extent on precursors being provided (e.g., fed) to the microorganism being utilized. For example, microorganisms featured in the methodologies of the present invention can be used to produce panto-compounds in a manner requiring feeding of substantially reduced amounts of the precursors aspartate, j3-alanine, valine and/or a-KIV. In one embodiment, the invention features methods of producing panto-compounds (e.g., pantothenate) in a manner that requires feeding of less than 5%-10% of the amount of precursor required by a control microorganism (e.g., a microorganism that is dependent, for example is wholly dependent, on precursor feed to efficiently produce the desired compound). In another embodiment, the invention features methods of producing panto-compounds in a manner that requires feeding of less than 15-20% of the amount of precursor required by a control microorganism. In another embodiment, the invention features methods of producing panto-compounds in a manner that requires feeding of less than 25-30%, 35-40%, 45-50% or 55-60% of the amount of precursor required by a control microorganism. As described in Examples Mil herein, particular microorganisms featured in the methodologies of the present invention require, for example, 5 g/L of aspartate, p-alanine, valine or a-KIV (e.g., in test tube or in shake flask cultures). Accordingly, in a preferred embodiment, the present invention features methods of producing panto-compounds (e.g., pantothenate) in a manner requiring feeding of less than 0.25 g/L, 0.5 g/L, 0.75 g/L, 1 g/L, 1.25 g/L, 1.5 g/L, 1.75 g/L, 2 g/L, 2.25 g/L, 2.5 g/L, 2.75 g/L or 3 g/L.<br>
Preferred methods of producing desired compounds (e.g., panto-compounds) in a manner independent of precursor feed or alternatively, in a manner substantially independent of precursor feed, involve culturing microorganisms which have been manipulated (e.g., designed or engineered, for example, genetically engineered) such that expression of at least one pantothenate biosynthetic enzyme, and/or at least one isoleucine-valine biosynthetic enzyme, is modified. For example, in one embodiment, a microorganism is manipulated (e.g., designed or engineered) such that the production of at least one pantothenate biosynthetic enzyme, and/or at least one isoleucine/valine biosynthetic enzyme is deregulated. In a preferred embodiment, a microorganism is manipulated (eg, designed or engineered) such that it has a deregulated biosynthetic pathway, for example, a deregulated pantothenate biosynthesis pathway and/or a deregulated isoleucine-valine biosynthetic pathway, as defined herein. In another preferred embodiment, a microorganism is manipulated (e.g., designed or engineered)<br><br>
such that at least one pantothenate biosynthetic enzyme, and/or at least one isoleucine-valine biosynthetic enzyme is overexpressed.<br>
Preferred methods of producing desired compounds (e.g., panto-compounds) in a manner independent of precursor feed or alternatively, in a manner substantially independent of precursor feed, are as follows. In one embodiment, the invention features a method of producing pantothenate in a manner independent of precursor feed comprising culturing an aspartate-a-decarboxylase-overexpressing (AaD-O) microorganism having a deregulated isoleucine-valine (/7v) pathway under conditions such that pantothenate is produced. In another embodiment, the invention features a method of producing pantothenate in a mannef independent of precursor feed comprising culturing an aspartate-a-decarboxylase-overexpressing (AaD-O) microorganism having a deregulated pantothenate (pan) pathway and a deregulated isoleucine-valine (i/v) pathway, under conditions such that pantothenate is produced. In another embodiment, the invention features a method of producing pantothenate in a manner independent of aspartate or p-alanine feed comprising culturing an aspartate-a-decarboxylase-overexpressing (AaD-O) microorganism under conditions such that pantothenate is produced. In yet another embodiment, the invention features a method of producing pantothenate in a manner independent of valine or a-ketoisovalerate feed comprising culturing a microorganism having a deregulated isoleucine-valine (z/v) biosynthetic pathway under conditions such that pantothenate is produced.<br>
The term "aspartate-a-decarboxylase-overexpressing (AaD-O) microorganism" includes a microorganism which has been manipulated such that aspartate-a-decarboxylase is overexpressed. A preferred "aspartate-a-decarboxylase-overexpressing (AaD-O) microorganism" has been transformed with a vector comprising a B. subtilis panD nucleic acid sequence (e.g., zpanD nucleic acid sequence that encodes an aspartate-a-decarboxylase protein having the amino acid sequence of SEQ ID NO:28, for example, apanD nucleic acid sequence as set forth in SEQ ID NO:27).<br>
The phrase "microorganism having a deregulated isoleucine-valine (z/v) pathway" includes a microorganism having an alteration or modification in at least one gene encoding an enzyme of the isoleucine-valine (ilv) pathway or having an alteration or modification in an operon including more than one gene encoding an enzyme of the isoleucine-valine (i/v) pathway. A preferred "microorganism having a deregulated isoleucine-valine (ilv) pathway" overexpresses acetohydroxyacid synthetase (e.g., acetohydroxyacid synthetase having subunits having the amino acid sequences of SEQ ID NO:32 and SEQ ID NO:34 or acetohydroxyacid synthetase having the amino acid sequence of SEQ ID NO:87), acetohydroxyacid isomeroreductase (having the amino acid sequence of SEQ ID NO:36), or dihydroxyacid dehydratase (having the amino acid<br><br>
sequence of SEQ ID NO:38) and/or has been transformed with a vector comprising ilvB, ilvN, ilvC, ilvBN, ilvBNC or alsS nucleic acid sequences (SEQ ID NO:31, SEQ ID NO:33, SEQ ID NO:35, nucleotides 1-2246 of SEQ ID NO:58, SEQ ID NO:58 having coding regions from nucleotides 1-1725, 1722-2246 and 2263-3291, or SEQ ID NO:86, respectively) and/or an ilvD nucleic acid sequence (SEQ ID NO:37).<br>
IV.       High Yield Production Methodologies<br>
A particularly preferred embodiment of the present invention is a high yield production method for producing pantothenate tomprising culturing a manipulated microorganism under conditions such that pantothenate is produced at a significantly high yield. The phrase "high yield production method", for example, a high yield production method for producing a desired compound (e.g., for producing a panto-compound) includes a method that results in production of the desired compound at a level which is elevated or above what is usual for comparable production methods. Preferably, a high yield production method results in production of the desired compound at a significantly high yield. The phrase "significantly high yield" includes a level of production or yield which is sufficiently elevated or above what is usual for comparable production methods, for example, which is elevated to a level sufficient for commercial production of the desired product (e.g., production of the product at a commercially feasible cost). In one embodiment, the invention features a high yield production method of producing pantothenate that includes culturing a manipulated microorganism under conditions such that pantothenate is produced at a level greater than 2 g/L. In another embodiment, the invention features a high yield production method of producing pantothenate that includes culturing-a manipulated microorganism under conditions such that pantothenate is produced at a level greater than 10 g/L. In another embodiment, the invention features a high yield production method of producing pantothenate that includes culturing a manipulated microorganism under conditions such that pantothenate is produced at a level greater than 20 g/L. In yet another embodiment, the invention features a high yield production method of producing pantothenate that includes culturing a manipulated microorganism under conditions such that pantothenate is produced at a level greater than 30 g/L. In yet another embodiment, the invention features a high yield production method of producing pantothenate that includes culturing a manipulated microorganism under conditions such that pantothenate is produced at a level greater than 40 g/L.<br>
The invention further features a high yield production method for producing a desired compound (e.g., for producing a panto-compound) that involves culturing a manipulated microorganism under conditions such that a sufficiently elevated level of<br><br>
compound is produced within a commercially desireable period of time. In an exemplary embodiment, the invention features a high yield production method of producing pantothenate that includes culturing a manipulated microorganism under conditions such that pantothenate is produced at a level greater than 15-20 g/L in 36 hours. In another embodiment, the invention features a high yield production method of producing pantothenate that includes culturing a manipulated microorganism under conditions such that pantothenate is produced at a level greater than 25-30 g/L in 48 hours. In another embodiment, the invention features a high yield production method of producing pantothenate that includes culturing a manipulated microorganism under conditions such that pantothenate is produced at a level greater than 35-40 g/L in 72 hours, for example, greater that 37 g/L in 72 hours. In another embodiment, the invention features a high yield production method of producing pantothenate that includes culturing a manipulated microorganism under conditions such that pantothenate is produced at a level greater than 30-40 g/L in 60 hours, for example, greater that 30, 35 or 40 g/L in 60 hours. Values and ranges included and/or intermediate within the ranges set forth herein are also intended to be within the scope of the present invention. For example, pantothenate production at levels of at least 31, 32, 33, 34, 35, 36, 37, 38 and 39 g/L in 60 hours are intended to be included within the range of 30-40 g/L in 60 hours. In another example, ranges of 30-35 g/L or 35-40 g/L are intended to be included within the range of 30-40 g/L in 60 hours. Moreover, the skilled artisan will appreciate that culturing a manipulated microorganism to achieve a production level of, for example, "30-40 g/L in 60 hours" includes culturing the microorganism for additional time periods (e.g., time periods longer than 60 hours), optionally resulting in even higher yields of pantothenate being produced.<br>
V.        Panto-Compound Production Methodologies Featuring Pantothenate Kinase Mutant Microorganisms<br>
The present invention relates to methods of producing pantothenate using microorganisms engineered to produce high yields of pantothenate as well as other panto-compounds. Cells overproducing pantothenate result in high intracellular pantothenate levels that could overcome the feedback inhibition of pantothenate kinase by CoA, leading to overproduction of CoA. Besides consuming pantothenate, increased synthesis of CoA may cause increased feedback inhibition of the PanB, PanD, PanE or PanC reaction, thereby limiting pantothenate production. Accordingly, a reduction in pantothenate kinase activity may lead to a decrease in CoA levels with resulting increases in PanB, PanD, PanE or PanC activity and pantothenate production.<br><br>
Thus, certain methodologies of the present invention are based, at least in part, on the identification and characterization of the B. subtilis coaA gene and the demonstration that the gene is neither essential for B. subtilis growth (i.e., deletion of the coaA gene from the chromosome of B. subtilis is not a lethal event) nor for pantothenate kinase activity in B, subtilis. A second pantothenate kinase-encoding gene has been identified and characterized in B. subtilis, and is termed "coaX\ This gene complements an E. coli mutant that contains a temperature sensitive pantothenate kinase and is not related by homology to any previously known pantothenate kinase gene.<br>
In one aspect, the methodologies of the invention feature recombinant microorganisms that include the coaA'gene or that include a mutant cooA'gene, having reduced pantothenate kinase activity. In one embodiment, the methodologies feature such recombinant microorganisms further having a deregulated pantothenate biosynthetic pathway. In another embodiment, the methodologies feature such recombinant microorganisms further having a deregulated isoleucine-valine (i/v) pathway. In a preferred embodiment, the microorganisms belong to the genus Bacillus (e.g,B. subtilis).<br>
The methodologies of the invention also feature recombinant microorganisms (e.g., microorganisms belong to the genus Bacillus, for example, B. subtilis) that include the coaA gene or that include a mutant coaA gene, optionally including a coaX gene or mutant thereof, having reduced pantothenate kinase activity. In one embodiment, the methodologies feature such recombinant microorganisms further having a deregulated pantothenate biosynthetic pathway or having a deregulated isoleucine-valine (ilv) pathway. Also featured are vectors that include isolated coaX or coaA genes as well as mutant coaX and/or coaA genes. Isolated nucleic acid molecules that include isolated coaX genes or mutant coaX genes are features in addition to isolated CoaX proteins and mutant CoaX proteins.<br>
The above-described nucleic acid molecules (e.g., genes), proteins, vectors, and recombinant microorganisms (e.g., mutant microorganisms), are particularly suited for use in methods of producing panto-compounds and/or methods of enhancing panto-compound production. In one embodiment, the invention features a method for producing a panto-compound (e.g., pantothenate) that includes culturing a pantothenate kinase mutant (e.g., a recombinant microorganism that misexpresses, e.g., is mutated for, pantothenate kinase, as defined herein) under conditions such that panto-compound is produced. In another embodiment, the invention features a method for enhancing production of a panto-compound (e.g., pantothenate) that includes culturing a pantothenate kinase mutant (e.g., a recombinant microorganism that misexpresses, e.g., is mutated for, pantothenate kinase, as defined herein) under conditions such that<br><br>
production of the panto-compound is produced. As used herein, the term "enhancing" (for example, in the context of the phrase "enhancing production") includes increasing the level or rate of production of panto-compound (e.g., pantothenate) as compared to the level or rate of production in a non-mutant microorganism (e.g., a microorganism having a normal pantothenate kinase gene(s) and/or having normal pantothenate production rates and/or levels.<br>
Preferably, the level of panto-compound produced in methodologies featuring the pantothenate kinase mutants of the present invention is increased by at least 5% as compared to the level produced by a non-mutant (e.g., a recombinant microorganism expressing non-mutated pantothenate kinase). Even more preferably, the level of panto-compound is increased 10% as compared to methodologies featuring non-mutants. Even more preferably, panto-compound levels (e.g., pantothenate levels) are increased 20%, 30%, 40%, 50%, 60%, 70%, 80%, 90%, are increased 2-fold, 5-fold, 10-fold, 50-fold, 100-fold or more as compared to methodologies featuring non-mutants.<br>
VI       Additional Mutations Resulting in Enhanced Panto-Compound Production<br>
The methodologies of the present invention further can include, for example in addition to overexpressing or deregulating a pantothenate biosynthetic enzyme and/or an isoleucine-valine biosynthetic enzyme, or in addition to mutating a pantothenate-kinase encoding gene, deleting or mutating an enzyme that catalyzes the conversion of key pantothenate biosynthesis substrates or precursors to unwanted or undesirable products. For example, mutating the ilvE gene (Kuramitsu et al. (1985) J. Biochem. 97:993-999) or a homologue thereof (SEQ ID NO:62 or SEQ ID NO:G4), thereby limiting the conversion of a-ketoiso valerate to valine, in addition to mutating" a pantothenate kinase encoding enzyme, is predicted to result in even further enhanced or increased production of panto-compound. Alternatively, mutating the ansB gene (Sun and Seflow (1991) J. Bacteriol. 173:3831-3845) or a homologue thereof (SEQ ID NO:66), thereby limiting the degradation of aspartate, in addition to mutating a pantothenate kinase encoding enzyme, is predicted to result in even further enhanced or increased production of panto-compound. Alternatively, mutating the alsD gene (Renna et al. (1993) J. Bacteriol 175:3863-3875) or a homologue thereof (SEQ ID NO:68), thereby limiting the conversion of acetolactate to acetoin, in addition to mutating a pantothenate kinase encoding enzyme, is predicted to result in even further enhanced or increased production of panto-compound. Alternatively, mutating the avtA gene encoding alanine-valine transaminase or a homologue thereof, thereby limiting the conversion of a-ketoisovalerate to valine, in addition to mutating a pantothenate kinase encoding<br><br>
enzyme, is predicted to result in even further enhanced or increased production of panto-compound. Mutating the avtA gene can include mutating, for example, an avtA gene having the nucleotide sequence of SEQ ID NO:70 (e.g., the E. colt avtA gene), or a structural homolog thereof (e.g., a homologue encoding a protein having 30-40%, 40-50%, 50-60%, 60-70%, 70-80%, 80-90%, 90-95% or more identity with the amino acid sequence of SEQ ID NO:71) or a functional homologue (e.g., a gene encoding a structurally unrelated protein having alanine-valine transmainase activity. Such mutations can be accomplished using the methodologies as exemplified in the Examples (e.g., Examples XIII, XV, XVI and XVII).<br>
Accordingly, in one embodiment, the invention features a method of producing a panto-compound which includes culturing a microorganism having a mutant pantothenate kinase-encoding gene and which further has a deletion or mutation in an avtA, ilvE, ansB, and/or alsD gene, or homologue thereof. In another embodiment, the invention features a method of producing a panto-compound which includes culturing a microorganism having a mutant pantothenate-kinase encoding gene and a deregulated pantothenate biosynthetic pathway enzyme and which further has a deletion or mutation in an avtA. ilvE, ansB, and/or alsD gene, or homologue thereof. In another embodiment, the invention features a method of producing a panto-compound which includes culturing a microorganism having a mutant pantothenate-kinase encoding gene and a deregulated isoleucine-valine biosynthetic pathway enzyme and which further has a deletion or mutation in an avtA, ilvE, ansB,and/or alsD gene, or homologue thereof.<br>
Mutating the alsD gene can be particularly useful when accomplished in conjunction with overexpression or deregulation of the alsS gene, for example, to prevent carbon (e.g., acetolactate) from being drawn away from the precursor pool utilized for a-KIV production. Accordingly, to maximize the contribution of the als locus to panto-compound production, it is desirable to disrupt the alsD gene in addition to overexpressing the alsS gene. To disrupt the alsD gene, appropriate fragments of the als operon, flanking the alsD gene, are amplified by PCR and cloned to provide homology for creating the disruptions. A drug resistance gene, such as the cat gene, is cloned between the flanking DNA fragments in place of the alsD gene, and the linearized DNA is transformed into a pantothenate production strain such as PA824, selecting for drug-resistance. To overexpress alsS, the alsS coding sequence (e.g., an alsS coding sequence that has been engineered by PCR for expression) is cloned into an expression vector. Vectors which express alsS (or alternatively, vectors which express alsS plus ilvC) are the introduced into panto-compound production strains (e.g., the pantothenate producing strain PA824).<br><br>
The methodologies of the present invention further can include, for example in addition to overexpressing or deregulating a pantothenate biosynthetic enzyme and/or an isoleucine-valine biosynthetic enzyme, or in addition to mutating a pantothenate-kinase encoding gene, deleting or mutating an enzyme that catalyzes the conversion of desired panto-compounds to unwanted or undesireable downstream products.<br>
VIL      Isolated Nucleic Acid Molecules and Genes<br>
Another aspect of the present invention features isolated nucleic acid molecules that encode Bacillus proteins {e.g., B. subtilis proteins), for example, Bacillus pantothenate biosynthetic enzymes (e.g.* B. subtilis pantothenate biosynthetic enzymes) or Bacillus valine-isoleucine biosynthetic enzymes (e.g., B. subtilis valine-isoleucine biosynthetic enzymes). Also featured are isolated coaX and/or coaA nucleic acid molecules (e.g., isolated coaX and/or coaA genes) as well as isolated nucleic acid molecules that include such coaX and/or coaA nucleic acid molecules or genes.<br>
The term "nucleic acid molecule" includes DNA molecules (e.g., linear, circular, cDNA or chromosomal DNA) and RNA molecules (e.g., tRNA, rRNA, mRNA) and analogs of the DNA or RNA generated using nucleotide analogs. The nucleic acid molecule can be single-stranded or double-stranded, but preferably is double-stranded DNA. The term "isolated" nucleic acid molecule includes a nucleic acid molecule which is free of sequences which naturally flank the nucleic acid molecule (i.e., sequences located at the 5* and 3* ends of the nucleic acid molecule) in the chromosomal DNA of the organism from which the nucleic acid is derived. In various embodiments, an isolated nucleic acid molecule can contain less than about 10 kb, 5 kb, 4kb, 3kb, 2kb, 1 kb, 0.5 kb, 0.1 kb, 50 bp, 25 bp or 10 bp of nucleotide sequences which naturally flank the nucleic acid molecule in chromosomal DNA of the microorganism from which the nucleic acid molecule is derived. Moreover, an "isolated" nucleic acid molecule, such as a cDNA molecule, can be substantially free of other cellular materials when produced by recombinant techniques, or substantially free of chemical precursors or other chemicals when chemically synthesized.<br>
The term "gene", as used herein, includes a nucleic acid molecule (e.g., a DNA molecule or segment thereof), for example, a protein or RNA-encoding nucleic acid molecule, that in an organism, is separated from another gene or other genes, by intergenic DNA (i.e., intervening or spacer DNA which naturally flanks the gene and/or separates genes in the chromosomal DNA of the organism). A gene may direct synthesis of an enzyme or other protein molecule (e.g., may comprise coding sequences, for example, a contiguous open reading frame (ORF) which encodes a protein) or may itself be functional in the organism. A gene in an organism, may be clustered in an<br><br>
operon, as defined herein, said operon being separated from other genes and/or operons by the intergenic DNA. Individual genes contained within an operon may overlap without intergenic DNA between said individual genes. An "isolated gene", as used herein, includes a gene which is essentially free of sequences which naturally flank the gene in the chromosomal DNA of the organism from which the gene is derived (i.e., is free of adjacent coding sequences which encode a second or distinct protein or RNA molecule, adjacent structural sequences or the like) and optionally includes 5* and 3' regulatory sequences, for example promoter sequences and/or terminator sequences. In one embodiment, an isolated gene includes predominantly coding sequences for a - protein (e.g., sequences which encode Bacillus proteins). In another embodiment, an isolated gene includes coding sequences for a protein (e.g., for a Bacillus protein) and adjacent 5' and/or 3' regulatory sequences from the chromosomal DNA of the organism from which the gene is derived (e.g., adjacent 5' and/or 3' Bacillus regulatory sequences). Preferably, an isolated gene contains less than about 10 kb, 5 kb, 2 kb, 1 kb, 0.5 kb, 0.2 kb, 0.1 kb, 50 bp, 25 bp or 10 bp of nucleotide sequences which naturally flank the gene in the chromosomal DNA of the organism from which the gene is derived.<br>
In one aspect, the present invention features isolated panB nucleic acid sequences or genes, isolated panC nucleic acid sequences or genes, isolated panD nucleic acid sequences or genes, isolated panE nucleic acid sequences or genes, isolated ilvB, ilvN, ilvBN nucleic acid sequences or genes, isolated alsS nucleic acid sequences or genes, isolated ilvC nucleic acid sequences or genes and/or isolated ilvD nucleic acid sequences or genes.<br>
In a preferred embodiment, the nucleic acid or gene is derived from Bacillus (e.g., is Ztac/V/ws-derived). The term "derived from Bacillus" or*5ac///ws-derived" includes a nucleic acid or gene which is naturally found in microorganisms of the genus Bacillus. Preferably, the nucleic acid or gene is derived from a microorganism selected from the group consisting of Bacillus subtilist Bacillus lentimorbus, Bacillus lentus, Bacillus firmus, Bacillus pantothenticus, Bacillus amyloliquefaciens, Bacillus cereus, Bacillus circulans, Bacillus coagulans, Bacillus licheniformis, Bacillus megaterium, Bacillus pumilus, Bacillus thuringiensis, and other Group 1 Bacillus species, for example, as characterized by 16S rRNA type (Priest, supra). In another preferred embodiment, the nucleic acid or gene is derived from Bacillus brevis or Bacillus stearothermophilus. In another preferred embodiment, the nucleic acid molecules and/or genes of the present invention are derived from a microorganism selected from the group consisting of Bacillus licheniformis, Bacillus amyloliquefaciens, Bacillus halodurans, Bacillus subtilis, and Bacillus pumilus. In a particularly preferred<br><br>
embodiment, the nucleic acid or gene is derived from Bacillus subtilis (e.g., is Bacillus subtilis-derived). The term "derived from Bacillus subtilis" or "Bacillus subtilis-derived" includes a nucleic acid or gene which is naturally found in Bacillus subtilis. In yet another preferred embodiment, the nucleic acid or gene is a Bacillus gene homologue (e.g., is derived from a species distinct from Bacillus but having significant homology to a Bacillus gene of the present invention, for example, a Bacillus pan gene or Bacillus ilv gene).<br>
Included within the scope of the present invention are bacterial-derived nucleic acid molecules or genes and/or £ac/7/ws-derived nucleic acid molecules or genes (e.g., B. subtilis-derivtd fcucleic acid molecules or genes), for example, the genes identified by the present inventors, for example, Bacillus or B. subtilis coaX genes, coaA genes, pan genes and/or ilv genes. Further included within the scope of the present invention are bacterial-derived nucleic acid molecules or genes and/or #acz7/us-derived nucleic acid molecules or genes (e.g., B. subtilis-derived nucleic acid molecules or genes) (e.g., B. subtilis nucleic acid molecules or genes) which differ from naturally-occurring bacterial and/or Bacillus nucleic acid molecules or genes (e.g., B. subtilis nucleic acid molecules or genes), for example, nucleic acid molecules or genes which have nucleic acids that are substituted, inserted or deleted, but which encode proteins substantially similar to the naturally-occurring gene products of the present invention. In one embodiment, an isolated nucleic acid molecule comprises at least one of the nucleotide sequences set forth as SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID NO 31, SEQ ID NO:33, SEQ ID NO:86, SEQ ID NO:35 or SEQ ID NO:37. In another preferred embodiment, an isolated nucleic acid molecule comprises at least two, three or four of the nucleotide sequences set forth as SEQ ID NOr23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID NO 31, SEQ ID NO:33, SEQ ID NO:88, SEQ ID NO:35 or SEQ ID NO:37. For example, a preferred isolated nucleic acid molecule of the present invention can include the nucleotide sequences of SEQ ID NO:23, SEQ ID NO:25 and SEQ ID NO:27, preferably linked such that the proteins encoded by the nucleotide sequences of SEQ ID N0.23, SEQ ID NO:25 and SEQ ID NO:27 are each produced when the isolated nucleic acid molecule is expressed in a microorganism (e.g., SEQ ID NO:59). In another example, a preferred isolated nucleic acid molecule of the present invention can include the nucleotide sequences of SEQ ID NO:31 and SEQ ID NO:33, preferably linked such that the proteins encoded by the nucleotide sequences of SEQ ID NO:31 and SEQ ID NO:33 are each produced when the isolated nucleic acid molecule is expressed in a microorganism (e.g., nucleotides 1-2246 of SEQ ID NO:58). In another example, a preferred isolated nucleic acid molecule of the present invention can include the nucleotide sequence of SEQ ID NO:86. In another example, a preferred<br><br>
isolated nucleic acid molecule of the present invention can inciuae me nucieouae sequences of SEQ ID NO:31, SEQ ID NO:33 and SEQ ID NO:35, preferably linked such that the proteins encoded by the nucleotide sequences of SEQ ID NO:31, SEQ ID NO:33 and SEQ ID NO:35 are each produced when the isolated nucleic acid molecule is expressed in a microorganism {e.g., SEQ ID NO:58).<br>
In another embodiment, an isolated nucleic acid molecule of the present invention comprises a nucleotide sequence which is at least about 60-65%, preferably at least about 70-75%, more preferable at least about 80-85%, and even more preferably at least about 90-95% or more identical to a nucleotide sequence set forth as SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID NO 31, SEQ ID NO:33, SEQ ID NO:88, SEQ ID NO:35 or SEQ ID NO:37. In another embodiment, an isolated nucleic acid molecule hybridizes under stringent conditions to a nucleic acid molecule having a nucleotide sequence set forth as SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID NO 31, SEQ ID NO:33, SEQ ID NO:88, SEQ ID NO:35 or SEQ ID NO:37. Such stringent conditions are known to those skilled in the art and can be found in Current Protocols in Molecular Biology; John Wiley &amp; Sons, N.Y. (1989), 6.3.1-6.3.6. A preferred, non-limiting example of stringent {e.g. high stringency) hybridization conditions are hybridization in 6X sodium chloride/sodium citrate (SSC) at about 45°C, followed by one or more washes in 0.2 X SSC, 0.1% SDS at 50-65°C. Preferably, an isolated nucleic acid molecule of the invention that hybridizes under stringent conditions to the sequence of SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID NO 31, SEQ ID NO:33, SEQ ID NO:88, SEQ ID NO:35 or SEQ ID NO:37 corresponds to a naturally-occurring nucleic acid molecule. As used herein, a "naturally-occurring" nucleic acid molecule refers to an RNA or DNA molecule having a nucleotide sequence that occurs in nature.<br>
A nucleic acid molecule of the present invention (e.g., a nucleic acid molecule having the nucleotide sequence of SEQ ID NO:23, SEQ ID NO:25, SEQ ID N0.27, SEQ ID NO:29, SEQ ID NO 31, SEQ ID NO:33, SEQ ID NO:88, SEQ ID NO:35 or SEQ ID NO:37 can be isolated using standard molecular biology techniques and the sequence information provided herein. For example, nucleic acid molecules can be isolated using standard hybridization and cloning techniques (e.g., as described in Sambrook, J., Fritsh, E. F., and Maniatis, T. Molecular Cloning: A Laboratory Manual. 2nd, ed, Cold Spring Harbor Laboratory, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989) or can be isolated by the polymerase chain reaction using synthetic oligonucleotide primers designed based upon the sequence of SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID NO 31, SEQ ID NO:33, SEQ ID NO:88, SEQ ID NO:35 or SEQ ID NO:37. A nucleic acid of the invention can be<br><br>
amplified using cDNA, mRNA or alternatively, genomic DNA, as a template and appropriate oligonucleotide primers according to standard PCR amplification techniques. In another preferred embodiment, an isolated nucleic acid molecule of the invention comprises a nucleic acid molecule which is a complement of the nucleotide sequence shown in SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID NO:33, SEQ ID NO 31, SEQ ID NO:33, SEQ ID NO:88, SEQ ID NO:35.<br>
Additional panC nucleic acid sequences include those that comprise the nucleotide sequence of SEQ ID NO:25, encode a homologue of the polypeptide having the amino acid sequence set forth in SEQ ID NO:26 (e.g., encode a polypeptide having at least 30%, 40%, 50%, 60%, 70%, 80%, 90%, 95% or more identity to the polypeptide having the amino acid sequence as set forth in SEQ ID NO:26 and a substantially identical activity as said polypeptide), hybridize under stringent conditions to all or a portion of a nucleic acid molecule having the nucleotide sequence of SEQ ID NO:25 or to all or a portion of a nucleic acid molecule that encodes a polypeptide having the amino acid sequence of SEQ ID NO:26, or are complementary to apanC nucleotide sequence as set forth herein.<br>
Aditional panD nucleic acid sequences include those that comprise the nucleotide sequence of SEQ ID NO:27, encode a homologue of the polypeptide having the amino acid sequence set forth in SEQ ID NO:28 (e.g., encode a polypeptide having at least 30%, 40%, 50%, 60%, 70%, 80%, 90%, 95% or more identity to the polypeptide having the amino acid sequence as set forth in SEQ ID NO:28 and a substantially identical activity as said polypeptide), hybridize under stringent conditions to all or a portion of a nucleic acid molecule having the nucleotide sequence of SEQ ID NO:27 or to all or a portion of a nucleic acid molecule that encodes^ polypeptide having the amino acid sequence of SEQ ID NO:28, or are complementary taapanD nucleotide sequence as set forth herein.<br>
Additional panE nucleic acid sequences include those that comprise the nucleotide sequence of SEQ ID NO:29, encode a homologue of the polypeptide having the amino acid sequence set forth in SEQ ID NO:30 (e.g., encode a polypeptide having at least 30%, 40%, 50%, 60%, 70%, 80%, 90%, 95% or more identity to the polypeptide having the amino acid sequence as set forth in SEQ ID NO:30 and a substantially identical activity as said polypeptide), hybridize under stringent conditions to all or a portion of a nucleic acid molecule having the nucleotide sequence of SEQ ID NO:29 or to all or a portion of a nucleic acid molecule that encodes a polypeptide having the amino acid sequence of SEQ ID NO:30, or are complementary to apanE nucleotide sequence as set forth herein.<br><br>
Additional ilvB nucleic acid sequences are those that comprise the nucleotide sequence of SEQ ID NO:31, encode a homologue of the polypeptide having the amino acid sequence set forth in SEQ ID NO:32 {e.g., encode a polypeptide having at least 30%, 40%, 50%, 60%, 70%, 80%, 90%, 95% or more identity to the polypeptide having the amino acid sequence as set forth in SEQ ID NO:32 and a substantially identical activity as said polypeptide), hybridize under stringent conditions to all or a portion of a nucleic acid molecule having the nucleotide sequence of SEQ ID NO:31 or to all or a portion of a nucleic acid molecule that encodes a polypeptide having the amino acid sequence of SEQ ID NO:32, or are complementary to an ilvB nucleotide sequence as set forth herein.<br>
Additional ilvN nucleic acid sequences are those that comprise the nucleotide sequence of SEQ ID NO:33, encode a homologue of the polypeptide having the amino acid sequence set forth in SEQ ID NO:34 (e.g., encode a polypeptide having at least 30%, 40%, 50%, 60%, 70%, 80%, 90%, 95% or more identity to the polypeptide having the amino acid sequence as set forth in SEQ ID NO:34 and a substantially identical activity as said polypeptide), hybridize under stringent conditions to all or a portion of a nucleic acid molecule having the nucleotide sequence of SEQ ID NO:33 or to all or a portion of a nucleic acid molecule that encodes a polypeptide having the amino acid sequence of SEQ ID NO:34, or are complementary to an ilvN nucleotide sequence as set forth herein.<br>
Additional ilvC nucleic acid sequences include those that comprise the nucleotide sequence of SEQ ID NO:35, encode a homologue of the polypeptide having the amino acid sequence set forth in SEQ ID NO:36 (e.g., encode a polypeptide having at least 30%, 40%, 50%, 60%, 70%, 80%, 90%, 95% or more identity to the polypeptide having the amino acid sequence as set forth in SEQ ID NO:36 and a substantially identical activity as said polypeptide), hybridize under stringent conditions to all or a portion of a nucleic acid molecule having the nucleotide sequence of SEQ ID N0.35 or to all or a portion of a nucleic acid molecule that encodes a polypeptide having the amino acid sequence of SEQ ID NO:36, or are complementary to an ilvC nucleotide sequence as set forth herein.<br>
Additional HvD nucleic acid sequences include those that comprise the nucleotide sequence of SEQ ID NO:37, encode a homologue of the polypeptide having the amino acid sequence set forth in SEQ ID NO:38 (e.g., encode a polypeptide having at least 30%, 40%, 50%, 60%, 70%, 80%, 90%, 95% or more identity to the polypeptide having the amino acid sequence as set forth in SEQ ID NO:38 and a substantially identical activity as said polypeptide), hybridize under stringent conditions to all or a portion of a nucleic acid molecule having the nucleotide sequence of SEQ ID NO:37 or<br><br>
to all or a portion of a nucleic acid molecule that encodes a polypeptide having the amino acid sequence of SEQ ID NO:38; or are complementary to an ilvD nucleotide sequence as set forth herein.<br>
Additional alsS nucleic acid sequences include those that comprise the nucleotide sequence of SEQ ID NO:86, encode a homologue of the polypeptide having the amino acid sequence set forth in SEQ ID NO:87 (e.g., encode a polypeptide having at least 30%, 40%, 50%, 60%, 70%, 80%, 90%, 95% or more identity to the polypeptide having the amino acid sequence as set forth in SEQ ID NO:87 and a substantially identical activity as said polypeptide), hybridize under stringent conditions to all or a portion of a nucleic acid molecule having the nucleotide sequence of SEQ ID NO:86 or to all or a portion of a nucleic acid molecule that encodes a polypeptide having the amino acid sequence of SEQ ID NO:87, or are complementary to an alsS nucleotide sequence as set forth herein.<br>
In another embodiment, an isolated nucleic acid molecule is or includes a coaX gene, or portion or fragment thereof. In one embodiment, an isolated coaX nucleic acid molecule or gene comprises the nucleotide sequence as set forth in SEQ ID NO: 19 (e.g., comprises the B. subtilis coaX nucleotide sequence). In another embodiment, an isolated coaX nucleic acid molecule or gene comprises a nucleotide sequence that encodes the amino acid sequence as set forth in SEQ ID NO:9 (e.g., encodes the B. subtilis CoaX amino acid sequence). In yet another embodiment, an isolated coaX nucleic acid molecule or gene encodes a homologue of the CoaX protein having the amino acid sequence of SEQ ID NO:9. As used herein, the term "homologue" includes a protein or polypeptide sharing at least about 30-35%, preferably at least about 35-40%, more preferably at least about 40-50%, and even more preferably at least about 60%, 70%, 80%, 90% or more identity with the amino acid sequence of a wild-type protein or polypeptide described herein and having a substantially equivalent functional or biological activity as said wild-type protein or polypeptide. For example, a CoaX homologue shares at least about 30-35%, preferably at least about 35-40%, more preferably at least about 40-50%, and even more preferably at least about 60%, 70%, 80%, 90% or more identity with the protein having the amino acid sequence set forth as SEQ ID NO:9 and has a substantially equivalent functional or biological activity (i.e., is a functional equivalent) of the protein having the amino acid sequence set forth as SEQ ID NO:9 (e.g., has a substantially equivalent pantothenate kinase activity). In a preferred embodiment, an isolated coaX nucleic acid molecule or gene comprises a nucleotide sequence that encodes a polypeptide as set forth in any one of SEQ ID NO:7, SEQ ID NO:8, SEQ ID NO:10, SEQ ID NO:l 1, SEQ ID NO:12, SEQ ID NO:13, SEQ ID NO: 14, SEQ ID NO: 15, SEQ ID NO: 16, SEQ ID NO: 17, SEQ ID NO: 18, SEQ ID<br><br>
NO:74 or SEQ ID NO:75. In another embodiment, an isolated coaX nucleic acid molecule hybridizes to all or a portion of a nucleic acid molecule having the nucleotide sequence set forth in SEQ ID NO: 19 or hybridizes to all or a portion of a nucleic acid molecule having a nucleotide sequence that encodes a polypeptide having the amino acid sequence of any of SEQ ID NOs:7-l 8, 74 or 75. Such hybridization conditions are known to those skilled in the art and can be found in Current Protocols in Molecular Biology, Ausubel et al.9 eds., John Wiley &amp; Sons, Inc. (1995), sections 2,4 and 6. Additional stringent conditions can be found in Molecular Cloning: A Laboratory Manual, Sambrbok et aL9 Cold Spring Harbor Press, Cold Spring Harbor, NY (1989), chapters 7, 9 and 11. A preferred, non-limiting example of stringent hybridization conditions includes hybridization in 4X sodium chloride/sodium citrate (SSC), at about 65-70°C (or hybridization in 4X SSC plus 50% formamide at about 42-50°C) followed by one or more washes in IX SSC, at about 65-70°C. A preferred, non-limiting example of highly stringent hybridization conditions includes hybridization in IX SSC, at about 65-70°C (or hybridization in 1X SSC plus 50% formamide at about 42-50°C) followed by one or more washes in 0.3X SSC, at about 65-70°C. A preferred, non-limiting example of reduced stringency hybridization conditions includes hybridization in 4X SSC, at about 50-60°C (or alternatively hybridization in 6X SSC plus 50% formamide at about 40-45°C) followed by one or more washes in 2X SSC, at about 50-60°C. Ranges intermediate to the above-recited values, e.g., at 65-70°C or at 42-50°C are also intended to be encompassed by thepresent invention. SSPE (IX SSPE is 0.15 M NaCl, lOmM NaH2P04, and 1.25 mM EDTA, pH 7.4) can be substituted for SSC (IX SSC is 0.15 M NaCl and 15 mM sodium citrate) in the hybridization and wash buffers; washes are performed for 15 minutes each after hybridization is complete. The hybridization temperature for hybrids anticipated to be less than-50 base pairs in length should be 5-10°C less than the melting temperature (Tm) of the hybrid, where Tm is determined according to the following equations. For hybrids less than 18 base pairs in length, Tm(°C) = 2(# of A + T bases) + 4(# of G + C bases). For hybrids between 18 and 49 base pairs in length, Tm(°C) = 81.5 + 16.6(logio[Na+]) + 0.41(%G+C) - (600/N), where N is the number of bases in the hybrid, and [Na+] is the concentration of sodium ions in the hybridization buffer ([Na+] for IX SSC = 0.165 M). It will also be recognized by the skilled practitioner that additional reagents may be added to hybridization and/or wash buffers to decrease non-specific hybridization of nucleic acid molecules to membranes, for example, nitrocellulose or nylon membranes, including but not limited to blocking agents (e.g., BSA or salmon or herring sperm carrier DNA), detergents (e.g., SDS), chelating agents (e.g., EDTA), Ficoll, PVP and the like. When using nylon membranes, in particular, an additional preferred, non-limiting example of<br><br>
stringent hybridization conditions is hybridization in 0.25-0.5M NaF^PO^ 7% SDS at about 65°C, followed by one or more washes at 0.02M NaH2P04, 1% SDS at 65°C, see e.g., Church and Gilbert (1984) Proc. Natl Acad ScL USA 81:1991-1995, (or, alternatively, 0.2X SSC, 1% SDS). In another preferred embodiment, an isolated nucleic acid molecule comprises a nucleotide sequence that is complementary to a coaX nucleotide sequence as set forth herein (e.g., is the full complement of the nucleotide sequence set forth as SEQ ID NO: 19).<br>
In another preferred embodiment, an isolated nucleic acid molecule is or includes a coaA gene, for example, a Bacillus (e.g., B. subtilis) coaA gene, or portion or fragment thereof. Exemplary isolated coaA nucleic acid molecules and/or genes include (1) an isolated coaA nucleic acid molecule or gene comprising the nucleotide sequence as set forth in any one of SEQ ID NOs:20-22; (2) an isolated coaA nucleic acid molecule or gene comprising a nucleotide sequence that encodes the amino acid sequence as set forth in SEQ ID NO:3; (3) an isolated coaA nucleic acid molecule or gene comprising a nucleotide sequence which encodes a CoaA homologue (e.g., a polypeptide having an amino acid sequence at least about 30-35%, preferably at least about 35-40%, more preferably at least about 40-50%, and even more preferably at least about 60%, 70%, 80%, 90% or more identical to the amino acid sequence set forth as SEQ ID NO:3 and having a substantially equivalent enzymatic activity; (4) an isolated coaA nucleic acid molecule or gene comprising a nucleotide sequence that encodes a polypeptide as set forth in any one of SEQ ID NO:l, SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:5 or SEQ ID NO:6; (5) an isolated nucleic acid molecule that hybridizes under stringent conditions to all or a portion of a nucleic acid molecule having the nucleotide sequence set forth in SEQ ID NO:20, SEQ ID NO:21 or SEQ ID N0:22 or hybridizes to all or a portion of a nucleic acid molecule having a nucleotide sequence that encodes a polypeptide having the amino acid sequence of SEQ ID NO:3; and (6) an isolated nucleic acid molecule comprising a nucleotide sequence that is complementary to a coaA nucleotide sequence as set forth herein (e.g., is the full complement of the nucleotide sequence set forth in SEQ ID NO:20, SEQ ID NO:21 or SEQ ID NO:22).<br>
A nucleic acid molecule of the present invention (e.g., a coaA" nucleic acid molecule or gene or a coaA nucleic acid molecule or gene), can be isolated using standard molecular biology techniques and the sequence information provided herein. For example, nucleic acid molecules can be isolated using standard hybridization and cloning techniques (e.g., as described in Sambrook, J., Fritsh, E. F., and Maniatis, T. Molecular Cloning: A Laboratory Manual. 2nd, ed, Cold Spring Harbor Laboratory, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989) or can be isolated by the polymerase chain reaction using synthetic oligonucleotide primers designed<br><br>
based upon the coaX or coaA nucleotide sequences set forth herein, or flanking sequences thereof. A nucleic acid of the invention (e.g., a coaX nucleic acid molecule or gene or a coaA nucleic acid molecule or gene), can be amplified using cDNA, mRNA or alternatively, chromosomal DNA, as a template and appropriate oligonucleotide primers according to standard PCR amplification techniques.<br>
Yet another embodiment of the present invention features mutant coaX and coaA nucleic acid molecules or genes. The phrase "mutant nucleic acid molecule" or "mutant gene" as used herein, includes a nucleic acid molecule or gene having a nucleotide sequence which includes at least one alteration (e.g., substitution, insertion, deletion) such that the polypeptide or protein that may be encoded by said mutant exhibits an activity that differs from the polypeptide or protein encoded by the wild-type nucleic acid molecule or gene. Preferably, a mutant nucleic acid molecule or mutant gene (e.g., a mutant coaA or coaX gene) encodes a polypeptide or protein having a reduced activity (e.g., having a reduced pantothenate kinase activity) as compared to the polypeptide or protein encoded by the wild-type nucleic acid molecule or gene, for example, when assayed under similar conditions (e.g., assayed in microorganisms cultured at the same temperature). A mutant gene also can encode no polypeptide or have a reduced level of production of the wild-type polypeptide.<br>
As used herein, a "reduced activity" or "reduced enzymatic activity" is one that is at least 5% less than that of the polypeptide or protein encoded by the wild-type nucleic acid molecule or gene, preferably at. least 5-10% less, more preferably at least 10-25% less and even more preferably at least 25-50%, 50-75% or 75-100% less than that of the polypeptide or protein encoded by the wild-type nucleic acid molecule or gene. Ranges intermediate to the above-recited values, e.g., 75-85%, 85-90%, 90-95%, are also intended to be encompassed by the present invention. As used herein, a "reduced activity" or "reduced enzymatic activity" also includes an activity that has been deleted or "knocked out" (e.g., approximately 100% less activity than that of the polypeptide or protein encoded by the wild-type nucleic acid molecule or gene). Activity can be determined according to any well accepted assay for measuring activity of a particular protein of interest. Activity can be measured or assayed directly, for example, measuring an activity of a protein isolated or purified from a cell. Alternatively, an activity can be measured or assayed within a cell or in an extracellular medium. For example, assaying for a mutant coaA gene or a mutant cooA'gene (i.e., said mutant encoding a reduced pantothenate kinase activity) can be accomplished by expressing the mutated gene in a microorganism, for example, a mutant microorganism which expresses pantothenate kinase in a temperature-sensitive manner, assaying the mutant gene for the ability to complement a temperature sensitive (Ts) mutant for<br><br>
pantothenate kinase activity. A coaX mutant gene or coaA mutant gene that encodes a "reduced pantothenate kinase activity" is one that complements the Ts mutant less effectively than, for example, a corresponding wild-type coo^gene or coaA gene.<br>
It will be appreciated by the skilled artisan that even a single substitution in a nucleic acid or gene sequence (e.g., a base substitution that encodes an amino acid change in the corresponding amino acid sequence) can dramatically affect the activity of an encoded polypeptide or protein as compared to the corresponding wild-type polypeptide or protein. A mutant nucleic acid or mutant gene {e.g., encoding a mutant polypeptide or protein), as defined herein, is readily distinguishable from a nucleic acid or gene encoding a protein homologue, as described above, in that a mutant nucleic acid or mutant gene encodes a protein or polypeptide having an altered activity, optionally observable as a different or distinct phenotype in a microorganism expressing said mutant gene or nucleic acid or producing said mutant protein or polypeptide (i.e., a mutant microorganism) as compared to a corresponding microorganism expressing the wild-type gene or nucleic acid or producing said mutant protein or polypeptide. By contrast, a protein homologue has an identical or substantially similar activity, optionally phenotypically indiscernable when produced in a microorganism, as compared to a corresponding microorganism expressing the wild-type gene or nucleic acid. Accordingly it is not, for example, the degree of sequence identity between nucleic acid molecules, genes, protein or polypeptides that serves to distinguish between homologues and mutants, rather it is the activity of the encoded protein or polypeptide that distinguishes between homologues and mutants: homologues having, for example, low (e.g., 30-50% sequence identity) sequence identity yet having substantially equivalent functional activities, and mutants, for example-sharing 99% sequence identity yet having dramatically different or altered functional activities.-Exemplary homologues are set forth in Figure 20 (i.e., CoaA homologues) and in Figure 23 (i.e., CoaX homologues). Exemplary mutants are described in Examples XV and XVIII herein.<br>
VIII.    Recombinant Nucleic Acid Molecules and Vectors<br>
The present invention further features recombinant nucleic acid molecules (e.g., recombinant DNA molecules) that include nucleic acid molecules and/or genes described herein (e.g., isolated nucleic acid molecules and/or genes), preferably Bacillus genes, more preferably Bacillus subtilis genes, even more preferably Bacillus subtilis pantothenate kinase genes (e.g., coaX genes or coaA genes), pantothenate biosynthetic genes (e.g., genes encoding pantothenate biosynthetic enzymes, for example, panB genes encoding ketopantoate hydroxymethyltransferase,/?an£ genes encoding<br><br>
ketopantoate reductase, panC genes encoding pantothenate synthetase, and/or partD genes encoding aspartate-a-decarboxylase) and/or isoleucine-valine (i/v) biosynthetic genes {e.g., ilvBNor alsSgenes encoding acetohydroxyacid synthetase, ilvC genes encoding acetohydroxyacid isomeroreductase and/or ilvD genes encoding dihydroxyacid dehydratase).<br>
The present invention further features vectors (e.g., recombinant vectors) that include nucleic acid molecules (e.g., isolated or recombinant nucleic acid molecules and/or genes) described herein. In particular, recombinant vectors are featured that include nucleic acid sequences that encode bacterial gene products as described herein, preferably Bacillus gene products, more preferably Bacillus subtilis gene products, even more preferably Bacillus subtilis pantothenate biosynthetic gene products {e.g., pantothenate biosynthetic enzymes, for example, ketopantoate hydroxymethyltransferase, ketopantoate reductase, pantothenate synthetase, and/or aspartate-a-decarboxylase) and/or isoleucine-valine biosynthetic gene products (e.g., acetohydroxyacid synthetase, acetohydroxyacid isomeroreductase and/or dihydroxyacid dehydratase).<br>
The term "recombinant nucleic acid molecule" includes a nucleic acid molecule (e.g., a DNA molecule) that has been altered, modified or engineered such that it differs in nucleotide sequence from the native or natural nucleic acid molecule from which the recombinant nucleic acid molecule was derived (e.g., by addition, deletion or substitution of one or more nucleotides). Preferably, a recombinant nucleic acid molecule (e.g., a recombinant DNA molecule) includes an isolated nucleic acid molecule or gene of the present invention (e.g., an isolated coaX, coaA, pan or ilv gene) operably linked to regulatory sequences.<br>
The term "recombinant vector" includes a vector (e.g., piasmid, phage, phasmid, virus, cosmid or other purified nucleic acid vector) that has been altered, modified or engineered such that it contains greater, fewer or different nucleic acid sequences than those included in the native or natural nucleic acid molecule from which the recombinant vector was derived. Preferably, the recombinant vector includes a coaX, coaA, pan or ilv gene or recombinant nucleic acid molecule including such coaX, coaA, pan or ilv gene, operably linked to regulatory sequences, for example, promoter sequences, terminator sequences and/or artificial ribosome binding sites (RBSs), as defined herein.<br>
The phrase "operably linked to regulatory sequence(s)" means that the nucleotide sequence of the nucleic acid molecule or gene of interest is linked to the regulatory sequence(s) in a manner which allows for expression (e.g., enhanced, increased, constitutive, basal, attenuated, decreased or repressed expression) of the<br><br>
nucleotide sequence, preferably expression of a gene product encoded by the nucleotide sequence (e.g., when the recombinant nucleic acid molecule is included in a recombinant vector, as defined herein, and is introduced into a microorganism).<br>
The term "regulatory sequence" includes nucleic acid sequences which affect (e.g., modulate or regulate) expression of other nucleic acid sequences. In one embodiment, a regulatory sequence is included in a recombinant nucleic acid molecule or recombinant vector in a similar or identical position and/or orientation relative to a particular gene of interest as is observed for the regulatory sequence and gene of interest as it appears in nature, e.g., in a native position and/or orientation. For example, a gene of interest can be included in a recombinant nucleic acid molecule or recombinant vector operably linked to a regulatory sequence which accompanies or is adjacent to the gene of interest in the natural organism (e.g., operably linked to "native" regulatory sequences, for example, to the "native" promoter). Alternatively, a gene of interest can be included in a recombinant nucleic acid molecule or recombinant vector operably linked to a regulatory sequence which accompanies or is adjacent to another (e.g., a different) gene in the natural organism. Alternatively, a gene of interest can be included in a recombinant nucleic acid molecule or recombinant vector operably linked to a regulatory sequence from another organism. For example, regulatory sequences from other microbes (e.g., other bacterial regulatory sequences, bacteriophage regulatory sequences and the like) can be operably linked to a particular gene of interest.<br>
In one embodiment, a regulatory sequence is a non-native or non-naturally-occurring sequence (e.g., a sequence which has been modified, mutated, substituted, derivatized, deleted including sequences which are chemically synthesized). Preferred regulatory sequences include promoters, enhancers, termination signals, anti-termination signals and other expression control elements (e.g., sequences to^which repressors or inducers bind and/or binding sites for transcriptional and/or translational regulatory proteins, for example, in the transcribed mRNA). Such regulatory sequences are described, for example, in Sambrook, J., Fritsh, E. F., and Maniatis, T. Molecular Cloning: A Laboratory Manual. 2nd, ed.t Cold Spring Harbor Laboratory, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989. Regulatory sequences include those which direct constitutive expression of a nucleotide sequence in a microorganism (e.g., constitutive promoters and strong constitutive promoters), those which direct inducible expression of a nucleotide sequence in a microorganism (e.g., inducible promoters, for example, xylose inducible promoters) and those which attenuate or repress expression of a nucleotide sequence in a microorganism (e.g., attenuation signals or repressor sequences). It is also within the scope of the present invention to regulate expression of a gene of interest by removing or deleting regulatory sequences. For<br><br>
example, sequences involved in the negative regulation of transcription can be removed such that expression of a gene of interest is enhanced.<br>
In one embodiment, a recombinant nucleic acid molecule or recombinant vector of the present invention includes a nucleic acid sequence or gene that encodes at least one bacterial gene product (e.g., a pantothenate biosynthetic enzyme, an isoleucine-valine biosynthetic enzyme, or a CoaA biosynthetic enzyme, for example CoaA or CoaX) operably linked to a promoter or promoter sequence. Preferred promoters of the present invention include Bacillus promoters and/or bacteriophage promoters (e.g., bacteriophage which infect Bacillus). In one embodiment, a promoter is a Bacillus promoter, preferably a strong Bacillus promoter (e.g., a promoter associated with a biochemical housekeeping gene in Bacillus or a promoter associated with a glycolytic pathway gene in Bacillus). In another embodiment, a promoter is a bacteriophage promoter. In a preferred embodiment, the promoter is from the bacteriophage SP01. In a particularly preferred embodiment, a promoter is selected from the group consisting of Pi5, P26 or Pveg, for example, the promoters set forth in SEQ ID NO:39, SEQ ID NO:40 or SEQ ID NO:41. Additional preferred promoters include tef(the translational elongation factor (TEF) promoter) andpyc (the pyruvate carboxylase (PYC) promoter), which promote high level expression in Bacillus (e.g., Bacillus subtil is). Additional preferred promoters, for example, for use in Gram positive microorganisms include, but are not limited to, the amyE promoter or phage SP02 promoters. Additional preferred promoters, for example, for use in Gram negative microorganisms include, but are not limited to tac, trpt tett trp-tet, Ipp, lac, lpp-lac&gt; laclq, T7, T5, T3, gal, trc, ara, SP6, X-PR or X-PL.<br>
In another embodiment, a recombinant nucleic acid molecule or recombinant vector of the present invention includes a terminator sequence orterminator sequences (e.g., transcription terminator sequences). The term "terminator sequences" includes regulatory sequences which serve to terminate transcription of a gene. Terminator sequences (or tandem transcription terminators) can further serve to stabilize mRNA (eg-&gt; by adding structure to mRNA), for example, against nucleases.<br>
In yet another embodiment, a recombinant nucleic acid molecule or recombinant vector of the present invention includes sequences which allow for detection of the vector containing said sequences (i.e., detectable and/or selectable markers), for example, sequences that overcome auxotrophic mutations, for example, ura3 or ilvE, fluorescent markers, and/or colorimetric markers (e.g., /acZ/(3-galactosidase), and/or antibiotic resistance genes (e.g., amp or tet).<br><br>
In yet another embodiment, a recombinant nucleic acid molecule or recombinant vector of the present invention includes an artificial ribosome binding site (RBS). The term "artificial ribosome binding site (RBS)" includes a site within an mRNA molecule (e.g., coded within DNA) to which a ribosome binds (e.g., to initiate translation) which differs from a native RBS (e.g., a RBS found in a naturally-occurring gene) by at least one nucleotide. Preferred artificial RBSs include about 5-6, 7-8, 9-10, 11-12, 13-14, 15-16, 17-18, 19-20, 21-22, 23-24,25-26, 27-28,29-30 or more nucleotides of which about 1-2, 3-4, 5-6, 7-8,9-10, 11-12,13-15 or more differ from the native RBS (e.g., the native RBS of a gene of interest). Preferably, nucleotides which differ are substituted such that they are identical to one or more nucleotides of an ideal RBS (e.g., SEQ ID NO:44, SEQ ID NO:45, SEQ ID NO:46, SEQ ID NO:47 or SEQ ID NO:48), when optimally aligned for comparisons. Artificial RBSs can be used to replace the naturally-occurring or native RBS associated with a particular gene. Artificial RBSs preferably increase translation of a particular gene. Preferred artificial RBSs (e.g., RBSs for increasing the translation oipanB, for example, of B. sitbtilis panB) are depicted in Table IA (e.g., SEQ ID NO:49 and SEQ ID NO:50).<br><br><br>
Additional preferred artificial RBSs (e.g., RBSs for increasing the translation of panD, for example, of B. subtilis panD) are depicted in Table 1C (e.g., SEQ ID NO:55, SEQ ID NO:56 and SEQ ID NO:57). The predicted amino acid sequence at the C-terminus of the PanC protein is shown. The start codon for PanD translation is<br>
underlined.<br><br>
Accordingly, in one embodiment, a vector of the present invention includes an artificial RBS as set forth in SEQ ID NO:49 or SEQ ID NO:50. Jn another embodiment, a vector of the present invention includes an artificial RBS as set forth in SEQ ID NO:51, SEQ ID NO:52, SEQ ID NO:53 or SEQ ID NO:54. In yet another embodiment, a vector of the present invention includes an artificial RBS as set forth in SEQ ID NO:55, SEQ ID NO:56 or SEQ ID NO:57.<br>
In another embodiment, a recombinant vector of the present invention includes sequences that enhance replication in bacteria (e.g., replication-enhancing sequences). In one embodiment, replication-enhancing sequences are derived from E. coli. In another embodiment, replication-enhancing sequences are derived from pBR322 (e.g., sequences included within the pBR322 derived portion of any of the pAN vectors as set forth in the Figures , i.e., the Not I-Not I sequences from about 5.0 kB to 9.0 kB of the vector depicted in Figure 3A).<br><br>
In yet another embodiment, a recombinant vector of the present invention includes antibiotic resistance genes. The term "antibiotic resistance genes" includes sequences which promote or confer resistance to antibiotics on the host organism (e.g., Bacillus), In one embodiment, the antibiotic resistance genes are selected from the group consisting of cat (chloramphenicol resistance) genes, let (tetracycline resistance) genes, erm (erythromycin resistance) genes, neo (neomycin resistance) genes and spec (spectinomycin resistance) genes. Recombinant vectors of the present invention can further include homologous recombination sequences (e.g., sequences designed to allow recombination of the gene of interest into the chromosome of the host organism). For example, amyE sequences can be used as homology targets for recombination into the host chromosome.<br>
Preferred vectors of the present invention include, but are not limited to, vectors set forth in Figures 2-15, 17, 19, 22, 25 and 26. It will further be appreciated by one of skill in the art that the design of a vector can be tailored depending on such factors as the choice of microorganism to be genetically engineered, the level of expression of gene product desired and the like.<br>
IX.       Isolated Proteins<br>
Another aspect of the present invention features isolated proteins (e.g., isolated pantothenate biosynthetic enzymes and/or valine-isoleucine biosynthetic enzymes and'or isolated CoA biosynthetic enzymes, for example isolated CoaA or CoaX). In one embodiment, proteins (e.g., isolated pantothenate biosynthetic enzymes and/or valine-isoleucine biosynthetic enzymes and/or isolated CoaA biosynthetic enzymes, for example isolated CoaA or CoaX) are produced by recombinant DNA techniques and can be isolated from microorganisms of the present invention by an appropriate purification scheme using standard protein purification techniques. In another embodiment, proteins are synthesized chemically using standard peptide synthesis techniques.<br>
An "isolated" or "purified" protein (e.g., an isolated or purified biosynthetic enzyme) is substantially free of cellular material or other contaminating proteins from the microorganism from which the protein is derived, or substantially free from chemical precursors or other chemicals when chemically synthesized. In one embodiment, an isolated or purified protein has less than about 30% (by dry weight) of contaminating protein or chemicals, more preferably less than about 20% of contaminating protein or chemicals, still more preferably less than about 10% of contaminating protein or chemicals, and most preferably less than about 5% contaminating protein or chemicals.<br><br>
In a preferred embodiment, the protein or gene product is derived from Bacillus (e.g., is Bacillus-derived). The term "derived from Bacillus" or "Bacillus-derived" includes a protein or gene product which is encoded by a Bacillus gene. Preferably, the gene product is derived from a microorganism selected from the group consisting of Bacillus subtilis, Bacillus lentimorbus, Bacillus lentus, Bacillus firmus, Bacillus pantothenticus, Bacillus amyloliquefaciens, Bacillus cereus, Bacillus circularise Bacillus coagulans, Bacillus licheniformis, Bacillus megaterium, Bacillus pumilus, Bacillus thuringiensis, and other Group 1 Bacillus species, for example, as characterized by 16S rRNA type (Priest, supra). In another preferred embodiment, the protein or gene product is derived from Bacillus brevis or Bacillus stearothermophilus. In another preferred embodiment, the protein or gene product is derived from a microorganism selected from the group consisting of Bacillus licheniformis, Bacillus amyloliquefaciens, Bacillus halodurans, Bacillus subtilis, and Bacillus pumilus. In a particularly preferred embodiment, the protein or gene product is derived from Bacillus subtilis (e.g., is Bacillus subtil is-derived). The term "derived from Bacillus subtilis" or "Bacillus subtilis-derived" includes a protein or gene product which is encoded by a Bacillus subtilis gene. In yet another preferred embodiment, the protein or gene product is encoded by a Bacillus gene homologue (e.g., a gene derived from a species distinct from Bacillus but having significant homology to a Bacillus gene of the present invention, for example, a Bacillus pan gene or Bacillus ilv gene).<br>
Included within the scope of the present invention are bacterial derived proteins or gene products and/or Bacillus-derived proteins or gene products (e.g., B. subtilis-derived gene products) that are encoded by naturally-occurring bacterial and/or Bacillus genes (e.g., B. subtilis genes), for example, the genes identified by the present inventors, for example, Bacillus or B. subtilis coaX genes, coaA genes, parrgenes and/or ilv genes. Further included within the scope of the present invention are bacterial-derived proteins or gene products and/or itac/7/ws-derived proteins or gene products (e.g., B. subtMs-derived gene products) that are encoded bacterial and/or Bacillus genes (e.g., B. subtilis genes) which differ from naturally-occurring bacterial and/or Bacillus genes (e.g., B. subtilis genes), for example, genes which have nucleic acids that are mutated, inserted or deleted, but which encode proteins substantially similar to the naturally-occurring gene products of the present invention. For example, it is well understood that one of skill in the art can mutate (e.g., substitute) nucleic acids which, due to the degeneracy of the genetic code, encode for an identical amino acid as that encoded by the naturally-occurring gene. Moreover, it is well understood that one of skill in the art can mutate (e.g., substitute) nucleic acids which encode for conservative amino acid substitutions. It is further well understood that one of skill in the art can substitute, add or delete<br><br>
amino acids to a certain degree without substantially affecting the function of a gene product as compared with a naturally-occurring gene product, each instance of which is intended to be included within the scope of the present invention.<br>
In a preferred embodiment, an isolated protein of the present invention (e.g., an isolated pantothenate biosynthetic enzyme and/or an isolated isoleucine-valine biosynthetic enzyme and/or an isolated CoaA biosynthetic enzymes, for example isolated CoaA or CoaX) has an amino acid sequence shown in SEQ ID NO;3, SEQ ID NO:9, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO;28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, SEQ ID NO:36, SEQ ID NO:38 or SEQ ID NO:87. In other embodiments, an isolated protein of the present invention is a homologue of the at least one of the proteins set forth as SEQ ID NO:3, SEQ ID NO:9, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, SEQ ID NO:36, SEQ ID NO:38 or SEQ ID NO:87 (e.g., comprises an amino acid sequence at least about 30-40% identical, preferably about 40-50% identical, more preferably about 50-60% identical, and even more preferably about 60-70%, 70-80%, 80-90%, 90-95% or more identical to the amino acid sequence of SEQ ID NO:3, SEQ ID NO:9, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, SEQ ID NO:36, SEQ ID NO:38 or SEQ ID NO:87, and has an activity that is substantially similar to that of the protein encoded by the amino acid sequence of SEQ ID NO:3, SEQ ID NO:9, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, SEQ ID NO:36, SEQ ID NO:38 or SEQ ID NO:87, respectively.<br>
To determine the percent homology of two amino acid sequences or of two nucleic acids, the sequences are aligned for optimal comparison purposes (e.g., gaps can be introduced in the sequence of a first amino acid or nucleic acid sequence for optimal alignment with a second amino or nucleic acid sequence). When a position in the first sequence is occupied by the same amino acid residue or nucleotide as the corresponding position in the second sequence, then the molecules are identical at that position. The percent identity between the two sequences is a function of the number of identical positions shared by the sequences (i.e., % identity = # of identical positions/total # of positions x 100), preferably taking into account the number of gaps and size of said gaps necessary to produce an optimal alignment.<br>
The comparison of sequences and determination of percent homology between two sequences can be accomplished using a mathematical algorithm. A preferred, non-limiting example of a mathematical algorithm utilized for the comparison of sequences is the algorithm of Karlin and Altschul (1990) Proc. Natl Acad. ScL USA 87:2264-68, modified as in Karlin and Altschul (1993) Proc. Natl. Acad ScL USA 90:5873-77. Such<br><br>
an algorithm is incorporated into the NBLAST and XBLAST programs (version 2.0) of Altschul et al (1990)7. Mol Biol 215:403-10. BLAST nucleotide searches can be performed with the NBLAST program, score = 100, wordlength = 12 to obtain nucleotide sequences homologous to nucleic acid molecules of the invention. BLAST protein searches can be performed with the XBLAST program, score = 50, wordlength = 3 to obtain amino acid sequences homologous to protein molecules of the invention. To obtain gapped alignments for comparison purposes, Gapped BLAST can be utilized as described in Altschul et al (1997) Nucleic Acids Research 25(17):3389-3402. When utilizing BLAST and Gapped BLAST programs, the default parameters of the respective programs (e.g., XBLAST and NBLAST) can be used. See http://www.ncbi.nlm.nih.gov. Another preferred, non-limiting example of a mathematical algorithm utilized for the comparison of sequences is the algorithm of Myers and Miller (1988) Comput Appl Biosci. 4:11-17. Such an algorithm is incorporated into the ALIGN program available, for example, at the GENESTREAM network server, IGH Montpellier, FRANCE (http://vega.igh.cnrs.fr) or at the ISREC server (http://www.ch.embnet.org). When utilizing the ALIGN program for comparing amino acid sequences, a PAM120 weight residue table, a gap length penalty of 12, and a gap penalty of 4 can be used.<br>
In another preferred embodiment, the percent homology between two amino acid sequences can be determined using the GAP program in the GCG software package (available at http://www.gcg.com), using either a Blossom 62 matrix or a PAM250 matrix, and a gap weight of 12,10, 8, 6, or 4 and a length weight of 2,3, or 4. In yet another preferred embodiment, the percent homology between two nucleic acid sequences can be accomplished using the GAP program in the GCG software package (available at http://www.gcg.com), using a gap weight of-50 and a length weight of 3.<br>
X	Biotransformations and Byconversions<br>
Another aspect of the present invention includes biotransformation processes which feature recombinant microorganisms (e.g., mutant microorganisms) and/or isolated CoA, pantothenate or isoleucine-valine biosynthetic enzymes described herein. The term "biotransformation process", also referred to herein as "byconversion processes", includes biological processes which result in the production (e.g., transformation or conversion) of any compound (e.g., intermediate or product) which is upstream of a CoA, pantothenate or isoleucine-valine biosynthetic enzyme to a compound (e.g., substrate, intermediate or product) which is downstream of said CoA, pantothenate or isoleucine-valine biosynthetic enzyme.<br>
&gt;<br><br>
In one embodiment, the invention features a biotransformation process for the production of a panto-compound comprising contacting a microorganism which overexpresses at least one pantothenate biosynthetic enzyme with at least one appropriate substrate or precursor under conditions such that said panto-compound is produced and recovering said panto-compound.- In a preferred embodiment, the invention features a biotransformation process for the production of pantoate comprising contacting a microorganism which overexpresses ketopantoate reductase (the panE gene product) with an appropriate substrate (e.g., ketopantoate) under conditions such that pantoate is produced and recovering said pantoate. In another preferred embodiment, the invention features a biotransformation process for the production of pantothenate comprising contacting a microorganism which overexpresses ketopantoate reductase and pantothenate synthetase with appropriate substrates (e.g., ketopantoate and p-alanine) under conditions such that pantothenate is produced and recovering said pantothenate. In yet another preferred embodiment, the invention features a biotransformation process for the production of pantothenate comprising contacting a microorganism which overexpresses ketopantoate hydroxymethyltransferase, ketopantoate reductase and pantothenate synthetase with appropriate substrates (e.g., a-ketoisovalerate and p-alanine) under conditions such that pantothenate is produced and recovering said pantothenate. Preferred recombinant microorganisms for carrying out the above-described biotransformations include pantothenate kinase mutants. Conditions under which pantoate or pantothenate are produced can include any conditions which result in the desired production of pantoate or pantothenate, respectively.<br>
In yet another embodiment, the present invention includes a method of producing p-alanine that includes culturing a microorganism which overexpresses aspartate-a-decarboxylase under conditions such that p-alanine is produced. Preferably, the aspartate-a-decarboxylase-overexpressing microorganism has a mutation in a nucleic acid sequence encoding a pantothenate biosynthetic enzyme selected from the group consisting of ketopantoate hydroxymethyltransferase, ketopantoate reductase and pantothenate synthetase.<br>
The invention further features a method of producing p-alanine that includes contacting a composition comprising aspartate with an isolated Bacillus aspartate-a-decarboxylase enzyme under conditions such that p-alanine is produced (e.g., an in vitro synthesis method).<br>
The microorganism(s) and/or enzymes used in the biotransformation reactions are in a form allowing them to perform their intended function (e.g., producing a desired compound). The microorganisms can be whole cells, or can be only those portions of the cells necessary to obtain the desired end result. The microorganisms can be<br><br>
suspended (e.g., in an appropriate solution such as buffered solutions or media), rinsed (e.g., rinsed free of media from culturing the microorganism), acetone-dried, immobilized (e.g., with polyacrylamide gel or k-carrageenan or on synthetic supports, for example, beads, matrices and the like), fixed, cross-linked or permeablized (e.g., have permeablized membranes and/or walls such that compounds, for example, substrates, intermediates or products can more easily pass through said membrane or wall).<br>
Purified or unpurified CoA biosynthetic enzyme(s) (e.g., CoaA and/or CoaX), pantothenate biosynthetic enzyme(s) and/or valine-isoleucine biosynthetic enzyme(s) can also be used in biotransformation reactions. The enzyme can be in a form that allows it to perform its intended function (e.g., obtaining the desired compound). For example, the enzyme can be in free form or immobilized. Purified or unpurified CoA biosynthetic enzyme(s), pantothenate biosynthetic enzyme(s) and/or valine-isoleucine biosynthetic enzyme(s) can be contacted in one or more in vitro reactions with appropriate substrate(s) such that the desired product is produced.<br>
With respect to at least the above-described methodologies (e.g., the production methodologies of the present invention), at least one aspect of the invention features the folowing: embodiments is which the methods do not use microorganisms of the genus Corynebacterium and/or microorganisms of the genus Escherichia; embodiments in which the methods do not use microorganisms selected from the group consiting of Escherichia coli and Corynebacterium glutamicum; embodiments in which the methods do not use gram negative microorganisms; embodiments in which the microorganisms utilized do not include, express or produce nucleic acid molecules, genes or proteins (e.g., biosynthetic emzymes) derived from microorganisms of the genus Corynebacterium and/or microorganisms of the genus Escherichia; embodiments in which the microorganisms to not include, express or produce nucleic acid molecules, genes or proteins (e.g., biosynthetic emzymes) derived from microorganisms selected from the group consisting of Escherichia coli and Corynebacterium glutamicum.<br>
XL       Screening Assays<br>
Because CoA is an essential factor in bacteria, proteins (e.g., enzymes) involved in the biosynthesis of CoA provide valuable tools in the search for novel anti-biotics. In particular, the CoaX protein is a valuable target for identifying bacteriocidal compounds because it bears no resemblance in primary sequence to mammalian pantothenate kinase enzymes. Accordingly, the present invention also provides a method (also referred to herein as a "screening assay") for identifying modulators, i.e., candidate or test compounds or agents (e.g., peptides, peptidomimetics, small molecules or other drugs)<br><br>
which bind to CoaX, or have a stimulatory or inhibitory effect on, for example, coaX expression or CoaX activity.<br>
In one embodiment, the invention provides assays for screening candidate or test compounds which are capable of binding to CoaX proteins or a biologically active portion thereof. In another embodiment, the invention provides assays for screening candidate or test compounds which modulate the activity of CoaX proteins or biologically active portions thereof. The test compounds of the present invention can be obtained using any of the numerous approaches in combinatorial library methods known in the art, including: biological libraries; spatially addressable parallel solid phase or solution phase libraries; synthetic library methods requiring deconvolution; the 'one-bead one-compound' library method; and synthetic library methods using affinity chromatography selection. The biological library approach is limited to peptide libraries, while the other four approaches are applicable to peptide, non-peptide oligomer or small molecule libraries of compounds (Lam, K.S. (1997) Anticancer Drug Des. 12:145).<br>
Examples of methods for the synthesis of molecular libraries can be found in the art, for example in: DeWitt et al (1993) Proc. Natl. Acad Sci. U.S.A. 90:6909; Erb et al (1994) Proc. Natl. Acad. Sci. USA 91:11422; Zuckermann et al (1994). J. Med. Chem. 37:2678; Choet al (1993) Science 261:1303; Carrell et al (1994) Angew. Chem. Int. Ed Engl 33:2059; CarelU/ al (1994) Angew. Chem. Int. Ed. Engl 33:2061; and in Gallop et al (1994) J. Med. Chem. 37:1233. Libraries of compounds may be presented in solution (e.g., Houghten (1992) Biotechniques 13:412-421), or on beads (Lam (1991) Nature 354:82-84), chips (Fodor (1993) Nature 364:555-556), bacteria (Ladner USP 5,223,409), spores (Ladner USP '409), piasmids (Cull et al. (1992) Proc Natl Acad Sci USA 89:1865-1869) or on phage (Scott and Smith (1990) Science 249:386-390); (Devlin (1990) Science 249:404-406); (Cwirla et al (1990) Proc. Natl. Acad. ScL 87:6378-6382); (Felici(1991) J. Mol Biol 222:301-310); (Ladner supra.).<br>
In one embodiment, an assay is a microorganism-based assay in which a recombinant microorganism which expresses a CoaX protein or biologically active portion thereof is contacted with a test compound and the ability of the test compound to modulate CoaX activity is determined. Determining the ability of the test compound to modulate CoaX activity can be accomplished by monitoring, for example, intracellular phosphopanthoate or CoA concentrations or secreted pantothenate concentrations (as compounds that inhibit CoaX will result in a buildup of pantothenate in the test microorganism). CoaX substrate can be labeled with a radioisotope or enzymatic label such that modulation of CoaX activity can be determined by detecting a conversion of labeled substrate to intermediate or product. For example, CoaX substrates can be<br><br>
labeled with 32p, 14C, or 3H, either directly or indirectly, and the radioisotope detected by direct counting of radioemmission or by scintillation counting. Determining the ability of a compound to modulate CoaX activity can alternatively be determined by detecting the induction of a reporter gene (comprising a CoA-responsive regulatory element operatively linked to a nucleic acid encoding a detectable marker, e.g., luciferase), or detecting a CoA-regulated cellular response.<br>
In yet another embodiment, a screening assay of the present invention is a cell-free assay in which a CoaX protein or biologically active portion thereof is contacted with a test compound in vitro and the ability of the test compound to bind to or modulate the activity of the CoaX protein or biologically active portion thereof is determined. In a preferred embodiment, the assay includes contacting the CoaX protein or biologically active portion thereof with known substrates to form an assay mixture, contacting the assay mixture with a test compound, and determining the ability of the test compound to modulate enzymatic activity of the CoaX on its substrates.<br>
Screening assays can be accomplished in any vessel suitable for containing the<br>
microorganisms, proteins, and/or reactants. Examples of such vessels include microtiter<br>
plates, test tubes, and micro-centrifuge tubes. In more than one embodiment of the<br>
above assay methods of the present invention, it may be desirable to immobilize either<br>
CoaX protein or a recombinant microorganism expressing CoaX protein to facilitate<br>
separation of products and/or substrates, as well as to accommodate automation of the<br>
assay. For example, glutathione-S-transferase/CoaX fusion proteins can be adsorbed<br>
onto glutathione sepharose beads (Sigma Chemical, St. Louis, MO) or glutathione<br>
derivatized microtiter plates. Other techniques for immobilizing proteins on matrices<br>
(e.g., biotin-conjugation and streptavidin immobilization or antibody conjugation) can<br>
also be used in the screening assays of the invention.	-<br>
In another embodiment, modulators of CoaX expression are identified in a method wherein a cell is contacted with a candidate compound and the expression of coaX mRNA or CoaX polypeptide in the cell is determined. The level of expression in the presence of the candidate compound is compared to the level of expression in the absence of the candidate compound (or to a suitable control, for example, an appropriate buffer control or standard). The candidate compound can then be identified as a modulator of coaX mRNA or CoaX polypeptide expression based on this comparison.<br>
This invention further pertains to novel agents identified by the above-described screening assays. Accordingly, it is within the scope of this invention to further use an agent identified as described herein in an appropriate animal model. For example, an CoaX modulating agent identified as described herein (e.g., an anti-bactericidal<br><br>
compound) can be used in an infectious animal model to determine the efficacy, toxicity, or side effects of treatment with such an agent.<br>
This invention is further illustrated by the following examples which should not be construed as limiting. The contents of all references, patents, patent applications (including U.S. Patent Application Serial No. 09/400,494, filed September 21, 1999 (pending), provisional U.S. Patent Application Serial No. 60/210,072, filed June 7, 2000, provisional U.S. Patent Application Serial No. 60/221,938, filed July 28, 2000 and provisional U.S. Patent Application Serial No. 60/227,860, filed August 24,2000, to which this application relates) and published patent applications cited throughout this application are incorporated herein by reference.<br><br>
EXAMPLES<br>
General Methodology:<br>
Strains. Bacillus subtilis strains of the present invention are generally derived from either of two strains. The first is variously named "168", "1 Al", or "RL-1". The genotype is trpC2. This strain was derived from the wild type "Marburg" strain by mutagenesis and has been the basis of much of the molecular biology work done on B. subtilis. The second strain is PY79, a prototrophic derivative of 168 that was made Trp+ .by transduction from the wild type strain W23.<br>
Media. Standard minimal medium for B. subtilis is comprised of 1 x Spizizen salts and 0.5% glucose. Standard solid "rich medium" is Tryptone Blood Agar Broth (Difco), and standard liquid "rich medium" is VY, a mixture of veal infusion broth and yeast extract. For testing production of pantothenate in liquid test tube cultures, an enriched form of VY, called "Special VY" or "SVY" is used. For batch fermentations, SVYG and PFMG are used. The compositions of these media are given below.<br>
VY, a rich liquid medium: 25 g Difco Veal Infusion Broth, 5 g Difco Yeast Extract, 1L water (autoclave).<br>
TBAB, a rich solid medium: 33 g Difco Tryptone Blood Agar Broth, 1L water (autoclave).<br>
MIN, a minimal medium: 100 ml 10 x Spizizen salts; 10 ml 50% glucose; 2 ml 10% arginine HO*; 10 ml 0.8% tryptophan**; water to 1 liter. (*In some cases, arginine is omitted or replace by sodium glutamate at 0.04% final concentration. In general, B. subtilis grows faster in minimal medium when-certain amino acids, such as arginine, glutamine, glutamate, or proline, are added as an auxiliary nitrogen source. ) (**For strains that are tryptophan auxotrophs, tryptophan is routinely added to most minimal media.)<br>
10 xSpizizen Salts: 174gK2HP04-3H20; 20 g(NH
FeCl3 Solution: 4 g FeCl3-6H20; 197 g Na3Citrate-2H20; water to 1 liter (filter sterilize)<br>
Trace Elements Solution: 0.15 g Na2Mo04-2H20; 2.5 g H3B03; 0.7 g CoCl2*6H20; 0.25 g CuS04-5H20; 1.6 g MnCl2-4H20; 0.3 g ZnS04-7H20; water to 1 liter<br>
(filter sterilize).<br><br>
SVY, Special VY, a supplemented* rich medium for testing pantothenate production in test tube cultures: 25 g Difco Veal Infusion Broth; 5 g Difco yeast extract; 5 g sodium glutamate; 2.7 g ammonium sulfate; 740 ml water (autoclave); add 200 ml 1 M potassium phosphate, pH 7.0; 60 ml 50% glucose. (*For testing pantothenate production in liquid SVY test tube cultures, Na a-ketoisovalerate and/or p-alanine can be added to a concentration of 5 g/L from filter-sterilized stocks.)<br>
PFMG, a yeast extract based medium used infermentors: 20 g Amberex 1003™ yeast extract; 5 g sodium glutamate, 2 g ammonium sulfate; 5 g tryptophan; 10 g KH2P04; 20 g K2HP04-3H20; 1 g MgCl2-6H20; 0.1 g CaCl2-2H20; 1 g sodium citrate; 0.01 g FeS04-7H20; 1 ml trace elements solution; 20 g glucose; add water to 1 L. Glucose or other sugars are fed as needed. Feed solutions can contain minerals, defined or food grade nutrients.<br>
PFy a chemically defined pantothenate free medium for testing pantothenate auxotrophy: 100 ml 10 x Spizizen Salts; 100 ml 1 x Difco Pantothenate Assay Medium; 10 ml 50% glucose; water to 1 liter.<br>
For pantothenate auxotrophs, 1 mM Na pantothenate is added to both minimal and rich media, since there is generally not enough pantothenate in rich media to support B. subtilis pan mutants. Amino acids are at 100 mg per liter, when used. Selection for antibiotic resistance is done with 5 mg/L chloramphenicol, 100 mg/L spectinomycin HC1,15 mg/L tetracycline HO, or 1 mg/L erythromycin plus 25 mg/L lincomycin.<br>
Pantothenate Assays: Biological assay. The indicator organism, Lactobacillus plantarum, requires pantothenate for growth, and responds to low concentrations (jxg/L). Thus, using serial dilutions, a wide range of concentrations can be assayed. Commercially available medium (e.g., Pantothenate Assay Medium (PAM), Difco), can be used. However, Difco PAM supplemented with pantothenate does not support growth to the same level as obtainable using a fresh-mixed version of Pantothenate Assay Medium (FM-PAM), made up of the individual components as specified by Difco, which is accordingly, routinely used instead of the commercial product.<br>
Before assaying B. subtilis culture supernatants, the B, subtilis cells must be either removed or killed. B. subtilis culture supernatants give approximately the same pantothenate titer when the supernatants are autoclaved as when they are sterile filtered. Accordingly, routine procedures involve autoclaving samples for 5 minutes prior to the biological assay.<br><br>
Pantothenate Assays : HPLC assay. Pantothenic acid production is measured by HPLC with a detector wavelength of 197 nm and a reference at 450 nm. The procedure is a modification of one recommended by Hewlett-Packard for water soluble vitamins. Samples of culture broth are diluted into an equal volume of 60% acetronitrile (ACN), centrifiiged and filtered. Typically a further 10-fold dilution before analysis brings the final dilution to 20-fold. Higher concentrations of product are diluted further. Compounds are separated on a CI 8 Phenomenex 5\x Aqua 250 x 4.6 mm column with 5% acetronitrile (ACN) in 50 mM Na phosphate buffer at pH 2.5. An ACN gradient from 5% to 95% washes the column between every sample. The area of the pantothenate peak is proportional to the concentration between 5 to 1000 mg/L. Other panto-compounds are also separated and quantitated by this method.<br>
Amino Acid Analysis: HPLC assay. Amino acids present in the fermentation medium and throughout the fermentation are measured by HPLC with a detector wavelength of 338 nm and a reference at 390 nm. The procedure is a modification of one recommended by Hewlett-Packard for amino acid analysis. Samples of culture broth are prepared identically as for the panto-compound analysis. Compounds are separated on a CI8 Hypersil 5\x ODS 200 x 2.1 mm column. Solvent A is 20 mM Na acetate buffer at pH 7.2. Solvent B contains 40% ACN and 40% methanol.   A gradient from 100% Solvent A to 100% Solvent B separates amino acids and washes the column between every sample.<br>
Batch Fermentations. Pantothenate producing strains are grown in stirred tank ferrnentors, for example, in CF3000 Chemap 14 liter vessels with 10 liter working volumes. Computer control and data collection is by commercial software, for example, B. Braun Biotech MFCS software. Fermentations can be batch processes but are preferably sugar-limited, fed batch processes. Some media components (e.g. of SVYG and PMFG) are added to the fermentor and sterilized in place. Portions of the media are sterilized separately and added to the ferrnentors aseptically. This procedure is well known to those familiar with the art. Additional nitrogen sources in feeds are sterilized separately and added to the carbon source after cooling.<br>
i	The initial sugar in the medium is consumed in approximately 6 hours.<br>
Afterwards, glucose or other sugars are fed with the possible addition of minerals, and defined or food grade nutrients. Alternatively, feeds are scheduled based on a consensus profile of nutritional requirements from samples taken from earlier fermentations.<br><br>
After inoculation, agitation is set at a relatively low speed, e.g. 200 rpm. When the dissolved oxygen (p02) falls to 30%, computer control automatically adjusts the agitation to maintain a dissolved oxygen concentration between 25 and 30% p02.<br>
EXAMPLE I: Enhanced Production of a Panto-Compound Using Bacteria OverexpressingpanBCD Gene Products.<br>
This Example describes the cloning of the B subtilis panBCD operon and the generation of microorganisms overexpressing the panBCD gene products.<br>
To clone the B. subtilis panBCD operon, a plasmid library of B. subtilis GP275 (a derivative of 168) genomic DNA was transformed in E. coli BM4062 (birA*), and temperature resistant clones were selected at 42°C. By comparison of restriction maps to the genome sequence, one particular clone was deduced to contain the B. subtilis birA gene and the adjacent panBCD genes. This plasmid was named pAN201.<br>
To overexpress the panBCD operon and produce pantothenate, the native promoter of the panBCD operon was replaced by either of two strong, constitutive promoters derived from the B. subtilis bacteriophage SP01. These two promoters are named P26 and Pl5. In addition, either of two artificial ribosome binding sites (RBSs) were used to replace the native panB RBS. These two artificial RBSs (set forth as SEQ ID NO:49 and SEQ ID NO:50) were predicted to increase translation of panBCD; their sequences are shown in Table 1A Three such engineered panBCD expression cassettes were built into circular plasmids capable of replicating in E. coli. Other features of the plasmids include a strong rho-independent transcription terminator from the E. coli ribosomal RNA transcription unit, called T1T2, a Gram-positive chloramphenicol resistance gene (cat), derived from pC194, and a pair of Noil restriction sites at the junctions between the E. coli replicon and the segment intended Tor integration into B. subtilis. Three plasmids of this series, pAN004, pAN005, and pAN006 were constructed. pAN004 contains the P26 promoter, RBS1, and a low copy E. coli replicon. pAN005 contains the Pj$ promoter, which in our experience is not as strong as P26, RBS1, and the low copy replicon. pAN006 contains the P26 promoter, RBS2, and a medium copy replicon.<br>
The three panBCD expression cassettes contained in the above-mentioned three plasmids were all ligated to a DNA fragment consisting of sequences that naturally occur immediately upstream from the native panB gene and integrated in single copy by homologous recombination into the panBCD locus of B. subtilis strains RL-1 and PY79, replacing the wild-type operon. This was accomplished in two steps. First a deletion-substitution that replaced about two thirds of the panB coding region with a Gram-<br><br>
positive spectinomycin resistance gene (spec) was integrated atpanB to yield Spec", pantothenate auxotrophs. These intermediate strains were than transformed with the panBCD expression cassettes of pAN004, pAN005, and pAN006 after ligating them to a DNA fragment containing chromosomal sequences just upstream of panB. Selection of the incoming cassette was for pantothenate prototrophy. The resulting strains were named PA221, PA222 and PA223 (from RL-1), and PA235, PA232 and PA233 (from PY79), respectively. An example of a plasmid that contains the joined upstream sequence that is in the integrated strain in PA221 is pAN240 (see Figure 2), The nucleotide sequence of pAN240 is set forth as SEQ ID NO:76.<br>
Polymerase chain reaction using appropriate primers was used to verify the correct chromosomal structures of these engineered strains. When extracts of strain PA221 were examined by SDS-PAGE, two proteins were found to be overexpressed. One protein had an apparent molecular weight of 29,000 and the other protein appeared to be 39,000 daltons. The 29,000 dalton bands is presumably PanB (predicted molecular weight of 29,761). The larger protein band presumably represents PanC (predicted size 31,960 daltons).<br>
The ability of these strains to produce pantothenate in test tube cultures was assesed as follows. Each strain was grown in SVY medium supplemented with 5 g/L a-ketoisovalerate (a-KIV) and 5 g/L P-alanine, to ensure that these precursors were not limiting. Culture supernatants were autoclaved and assayed using the bioassay. Relative to the parent strains, RL-1 and PY79, the engineered strains produced about 8-to 30-fold more pantothenate, attaining 1 g/L pantothenate in some cases.<br><br><br>
The P26 promoter was about 3- to 4-fold more effective than the Pi5 promoter, while RBS1 and RBS2 were roughly equivalent. Plasmids such as pAN004, pAN005, pAN006 can also be recombined as circles into the B. subtilis wild type panBCD locus by Campbell-type (single crossover) integration, selecting for chloramphenicol resistance at 5 mg/L. Strains obtained in this fashion produce about the same amount of pantothenate as strains PA221, PA222, and PA223, respectively. pAN004 containing the P26 promoter, RBS1 and a low copy E. coli replicon, is depicted schematically in Figure 3A. The nucleotide sequence of plasmid pAN004 is set forth as SEQ ID NO:93. pAN006 containing the P26 promoter, RBS2 and a medium copy £*. coli replicon, is depicted schematically in Figure 3B. The nucleotide sequence of plasmid pAN006 is set forth as SEQ ID NO:94. The nucleotide sequence of panBCD is set forth as SEQ ID NO:59 and the predicted amino acid sequences of PanB, PanC and PanD are set forth as SEQ ID NO:24, SEQ ID NO:26 and SEQ ID NO:28, respectively. Methods for manipulating Bacilli are described, for example, in Harwood, C.R. and Cutting, S.M. (editors), Molecular Biological Methods for Bacillus (1990) John Wiley &amp; Sons, Ltd., Chichester, England, the content of which is incorporated herein by reference.<br><br>
EXAMPLE II:   Enhanced Production of a Panto-Compound Using Bacteria Overexpressing the panEl Gene Product - Ketopantoate Reductase.<br>
This Example describes the cloning of the B subtilis panEl gene and the generation of microorganisms overexpressing the panEl gene product.<br>
Pan" B. subtilis strains {e.g., B. subtilis mutants blocked in the synthesis of pantothenic acid) had previously been isolated, one of which was reported to be affected in ketopantoate reductase activity (Baigori et al. (1991) J. Bacterial. 173:4240-4242). However, the mutations in# these strains were incorrectly mapped to thtpurE-tre interval . of the B. subtilis genetic map which does not contain the panE or panBCD genes. Furthermore as shown below, a panE mutant does not have a Pan" phenotype as the ilvC gene product can substitute for the panE gene product in B. subtilis as in other bacterial strains such as E. coli. More recently, the S. typhimuruim panE gene has been located and determined to be allelic to apbA9 a gene required for anaerobic purine biosynthesis (Frodyma et al (1998)7. Biol. Chem. 273:5572-5576). E. coli carries a highly homologous gene at the same map location. Identification of the panE genes in E. coli and S. typhimurium was complicated by the fact that the ilvC gene product, acetohydroxy acid isomeroreductase, is also capable of carrying out the ketopantoate reductase reaction. As a result, pantothenate auxotrophy is not obtained unless both panE and ilvC are mutated.<br>
To identify the B. subtilis panEl gene, the B. subtilis genome was searched using the protein sequence of E. coli or S. typhimurium ApbA (PanE), and two open reading frames were identified having homology to ApbA, named ylbQ and ykpB. These genes were renamed panEl and panE2, due to their proposed function in pantothenate biosynthesis. Both panEl and panE2 were cloned as PCR products generated from RL-1 genomic DNA as a template. Both genes were disrupted by either a spectinomycin resistance gene (spec) or a chloramphenicol resistance gene (cat). The interrupted genes were each integrated by double crossover into PY79 to give PA240 (ApanEl::spec) and PA241 (ApanE2::cat). Neither of these strains were pantothenate auxotrophs when tested on pantothenate-free (PF) plates, although PA240 containing ApanElr.spec grew slightly more slowly on TBAB without added pantothenate than with a 1 mM pantothenate supplement. By comparison, a ApanB::spec strain does not produce single colonies on TBAB, presumably because B. subtilis has no active uptake system for pantothenate.<br>
It was hypothesized that the B. subtilis gene, /7vC, could function for panE as had been shown for E. coli. Accordingly, the panEl and panE2 disruptions were introduced into a strain, CU550, which is reported to be trpC2 ilvC4 leuC124. Both the single<br><br><br>
Thus, mutating both panEl and ilvC results in pantothenate auxotrophy, while mutating only panEl does not, similar to what has been reported for E.coli and S. typhimurium.<br>
Next, the quantitative effect of panEl and panE2 knockouts in a pantothenate overproducing strain (PA235 described herein) was examined. The panEl and panE2 disruptions were introduced into PA235, either singly or together to produce PA245 (ApanEl ::spcc\ PA248 (ApanE2::c3i) and PA244 (A/ww£7::cat, ApanE2::spec). The effect of each mutation on pantothenate production was then tested in liquid test tube cultures.<br><br><br>
Thus, deletion analysis indicated that the panEl gene contributes to over 90% of the pantothenate production, while deletion of panE2 did not have a significant effect on pantothenate production. It is therefore concluded that panEl accounts for most, but not necessarily all, of the ketopantoate reductase activity in B. subtilis. The rest of the ketopantoate reductase activity is predicted to be supplied by ilvC.<br>
Having identified panEl as an important gene for pantothenate production, increased panEl expression was tested to determine whether it could enhance pantothenate production in strains such as PA221 or PA235. The panEl coding sequence was installed downstream of the P26 promoter and RBS2 in a vector, pOTP61, designed to integrate and amplify at either the bpr locus (a non-essential protease gene) or at the locus of the cloned insert. The resulting plasmid, pAN236 (Figure 4) was transformed into PA221, selecting for resistance to tetracycline at 15 mg/L. The nucleotide sequence of pAN236 is set forth as SEQ ID NO:77. One transformant, named PA236 was chosen for further study.<br>
PA236 was shown to overexpress a protein of about 31,000 daltons, which is close to the expected molecular weight of 33,290 daltons for panEl protein. Briefly, whole cell extracts were prepared from PY79, RL-1, PA221, PA221/pOTP61and PA236 (2 samples). Cell extracts were separated by gel electrophoresis and the gels were coomassie stained to visualize proteins. In cells engineered to overexpress panE (PA236-1 and PA236-2), a band was visible having an approximate molecular weight of -31,000 daltons (as compared to molecular weight markers). Moreover, PA221 and PA236 expressed increased levels of a -29,000 dalton band, corresponding to the panB<br><br>
gene product, and a -39,000 dalton band, presumably corresponding the panC gene product. Furthermore, E. coli transformed with pAN006 (Figure 3B) expressed bands correlating to the panB and panC gene products and E. coli transfected with PAN236 expressed a -31,000 dalton band corresponding to thepanE gene product.<br>
Next, PA236 was compared to PA221 carrying the empty vector pOTP61 for pantothenate production in liquid test tube cultures supplemented with 5 g/L p-alanine and 5 g/L a-KIV.<br><br>
Overexpression of panEl caused a two-fold increase in pantothenate production when compared to the parent strain (e.g., to slightly over 2 g/L) whereas overexpression of panE2 resulted in a strain that produced about 35% less" pantothenate than the parent strain. The panEl nucleotide sequence and predicted amino acicTsequence are set forth as SEQ ID NO:29 and SEQ ID NO:30.<br>
EXAMPLE III:  Enhanced Production of a Panto-Compound by Culturing Bacteria Overexpressing/?ait£7 or pan BCD in the Presence of Valine.<br>
The ability of valine to function as a media supplement (e.g., as a substitute for a-KTV) in strains engineered to overexpress thepanBCD operon and panEl was evaluated. Valine is closely related to a-KIV by transamination, is less expensive than a-KIV, and is commercially available in kilogram quantities. Valine was substituted for a-KIV in the standard liquid test tube cultures in SVY medium. The concentration of i    valine was varied from 5 to 50 g/L. Although valine at 5 g/L was slightly less effective<br><br>
than a-KIV in promoting pantothenate production, valine at 10 or 20 g/L equaled or surpassed 5 g/L a-KIV in promoting pantothenate production.<br>
EXAMPLES IV-X Generation of Microorganisms Capable of Producing Pantothenate in a Precursor-Independent Manner<br>
B. subtilis strains such as PA221 and PA235 (engineered to overexpress panBCD) and PA236 (engineered to overexpress panBCD andpanEI) need to be fed a-ketoisovalerate (a-KIV) (or valine) and aspartate (or p-alanine) to achieve maximal pantothenate production, as both these precursors are limiting for pantothenate synthesis. Accordingly, manipulated microorganisms were designed to eliminate the need to feed limiting precursors of pantothenate biosynthesis in the production of pantothenate. These strains are also useful in the production of various pantothenate biosynthetic pathway intermediates.<br>
EXAMPLE IV:   Generation of Microorganisms Capable of Producing Pantothenate in an Aspartate- (or p-Alanine) Independent Manner<br>
The panD gene was cloned into B. subtilis expression vector pOTP61 to construct pAN423 (Figure 5). The nucleotide sequence of pAN423 is set forth as SEQ ID NO:78. The Not\ restriction fragment containing panD was isolated from pAN423, self ligated and used to transform PA221. Transformants resistant to Tet15, Tet30, and Tet60 were isolated and saved for further analysis.<br>
Six of the pAN423 transformants plus two control transformants were grown in<br>
SVY containing 5 g/1 a-KIV with and without 10 g/1 aspartate and then assayed for<br>
pantothenate production (Table 6).	~~<br><br><br><br>
The pAN423 transformants produced at least twice the amount of pantothenate as the controls (i.e., to a level at or near that which was obtained in earlier experiments by the addition of p-alanine to the culture medium). The data also show that in the absence of added aspartate, transformants containing additional copies of the panD gene expression cassette produce more pantothenate than the control transformants. One of the transformants, 423#l-5, produced about five times as much pantothenate as the controls. These results indicated that increased levels of PanD protein "pull" the conversion of available aspartate towards p-alanine, and that increasing panD gene expression can result in enhancement of pantothenate production both in the presence and absence of added aspartate.<br>
Transformant 423#l-5 was re-named strain PA401 and studied further in shake flask fermentations. The shake flask medium was SVY with maltose instead of SVY with glucose. Results of shake flask experiments agreed well with test tube experiments during the first 24 hours. In shake flask experiments without the addition of p-alanine, PA401 produced approximately 1,5 g/1 of pantothenate in 24 hours. Addition of p-alanine to the culture medium did not further improve pantothenate titers (Table 7), indicating that with this strain and these fermentation conditions, p-alanine is not limiting pantothenate production. In fact, when no p-alanine is fed, one can observe that PA401 is secreting p-alanine in significant amounts into the medium.<br><br><br>
Each value represents the average of duplicate 250 ml baffled flasks containing 50 ml of medium, incubated at 37°C with shaking (200 rpm). Base Medium: SVY with 10 g/1 a-KIV, 30 g/1 maltose 2% Inoculum: SVY with Tet    grown 24 hours.<br>
EXAMPLE V: Engineering the panD gene for Further Increased Synthesis of Aspartate Decarboxylase and Enhanced Production of Pantothenate<br>
This Example describes the generation of improved ribosome binding sites (RBSs) in the panD gene to increase the translation of panD mRNA.<br>
Increasing the translation of the panD gene mRNA by generation of synthetic panD RBSs<br>
The RBS (SEQ ID NO:88) used to express panD in pAN423 is a synthetic RBS and has been used to successfully produce other proteins in B. subtilis at a high level. However, it contains six mismatches when aligned to theJ'ideal" B. subtilis RBS (SEQ ID NO:45) (e.g., an RBS having a sequence which is complementary to the 16S RNA sequence within the B.subtilis ribosome). (See e.g., Table IB, mismatches in bold). Two new RBSs were designed to more closely mimic the ideal RBS. These synthetic RBSs, named new design A (NDA) and new design B (NDB) (also referred to herein as RBS3 and RBS4), are set forth as SEQ ID NO:51 and SEQ ID NO:52 and are aligned with the ideal RBS in Table IB.<br>
Oligonucleotides corresponding to the top and bottom strands of each new RBS were synthesized, annealed, then used to replace the RBS in pAN420, generating plasmids pAN426 and pAN427. These constructions are illustrated in Figure 6. The presence of the NDA and NDB RBS in pAN426 and pAN427 was confirmed by DNA sequence analysis. Next, the panD genes from pAN426 and pAN427 were transferred to B. subtilis expression vector pOTP61 as shown in Figure 7, creating pAN428 and pAN429. The nucleotide sequence of pAN429 is set forth as SEQ ID NO:79.<br><br>
Noil restriction fragments lacking the E. coli vector sequences were isolated from pAN428 and pAN429, self-ligated, and used to transform strain PA221 to resistance to Tet* . Four isolates resistant to Tet60 were picked from each transformation and assayed for pantothenate and p-alanine production along with PA221 transformed with the empty vector (pOTP61) and PA221 transformed with pAN423 (strain PA401) (see Table 8).<br><br>
When grown in medium supplemented with a-KJV at 5 g/1 (a-KIV5), the pAN428-l transformant and all four of the pAN429 transformants produced more<br><br>
pantothenate than did PA401, suggesting that these transformants contain higher levels of aspartate decarboxylase activity. When grown in medium supplemented with a-KIV5 and Asp10 none of the pAN428 or pAN429 transformants produced more pantothenate than PA401. However, the pAN428-l transformant and all four of the pAN429 transformants produced significantly more P-alanine than did PA401. It is possible that the excess p-alanine produced from added aspartate causes inhibition of pantothenate production. Alternatively, p-alanine may accumulate because pantoate is limiting in these strains.<br>
The strains that produced the highest level of p-alanine, the pAN428-l and pAN429-4 transformants, were renamed PA402 and PA403, respectively. These two strains were grown in SVY medium supplemented with various intermediates and reassayed for pantothenate and P-alanine production. PA221 and PA401 were included as controls. The results of the assays are presented in Table 9.<br><br><br>
UND: Below the limits of detection.<br>
When grown in medium supplemented with either <x-kiv5 or pantoate5 pa402 and pa403 produced significantly more pantothenate than did pa401. as before even though p-alanine pa401 when grown in medium supplemented with a-kiv5 asp10 they not produce a proportional increase pantothenate. however plus both about increase.></x-kiv5>
It can be concluded from these experiments that the improved NDA and NDB panD ribosome binding sites, engineered into pAN428 and pAN429, respectively, lead to increased levels of aspartate decarboxylase activity.<br>
Increasing the translation of the panD gene mRNA by generation of synthetic panD RBSs within the panBCD operon<br>
The native B. subtilis panD gene ribosome binding site (RBS) (SEQ ID NO:43), which is found in the P26panBCD operon cassette present in PA221 (and in other engineered pantothenate production strains described herein), is shown in Table 1C aligned with the ideal ribosome binding site (SEQ ID NO:47). The alignment shows mismatches between the native B. subtilis panD gene RBS, which is located within the coding sequence for PanC, and the the ideal RBS. Three new RBSs (within the P26 panBCD operon cassette) were generated to increase translation of the panD gene mRNA and to yield increased synthesis of aspartate decarboxylase. These synthetic RBSs (termed NDI, NDII, and NDIII, also referred to herein as RBS5, RBS6 and RBS7, respectively) are set forth as SEQ ID NO:55, SEQ ID NO:56 and SEQ ID NO:57, respectively) and are included in Table 1C, It should be noted that although changes in the panD RBS within the panBCD operon also changes the C-terminal amino acid sequence of the PanC protein encoded by that operon, an alignment of known and suspected PanC protein amino acid sequences showed that the sequence of the last nine amino acids of the B. subtilis PanC protein could be altered without affecting any conserved amino acid residues indicating that such changes should not reduce pantothenate synthetase activity or expression. The new RBSs were synthesized and incorporated into the P^ panBCD operon expression cassette as follows.<br>
First, PCR primers were designed to contain the following elements: (1) a nucleic acid sequence encoding the first five amino acids of PanD up to and including a unique BsiWL restriction site that had been previously introduced into panD by PCR; (2)<br><br>
a stop codon for panC, (3) at least one synthetic RBS; and (4) 30-39 bp of nucleic acid sequence having 100% identity with panC upstream of the panD RBS. The primers were named TP102, TP103, and TP104 and contain the NDI, NDII, and NDIII ribosome binding sites, respectively. These three primers were used in conjunction with the 5' primer TP101, which hybridizes near the start codon of panC, in three independent PCR reactions to generate the NDI, NDII, and NDIII PCR products. The PCR products were purified, digested with Xbal, then cloned into plasmid vector pASK-1 BA3 which had been digested with Xbal and Smal. The resulting plasmids were named pAN431, pAN432, and pAN433. The construction of pAN431 is illustrated in Figure 8 and is representative of all three plasmid constructions. The presence of the desired synthetic panD gene RBS in each new plasmid was confirmed by DNA sequencing.<br>
Next, the modified panC genes containing the new panD RBSs were joined with the panD gene utilizing the unique BsiWl restriction site. This was accomplished by isolating the appropriate Nsil-BsiWl restriction fragments from pAN431, pAN432, and pAN433 and ligating them with a 2395 bp Nsil-BsiWl restriction fragment from pAN420, which supplied the BsiWl-modifiedpanD gene. These constructions resulted in plasmids pAN441, pAN442, and pAN443, respectively. A representative construction (pAN441) is illustrated in Figure 9. The nucleotide sequence of pAN443 is set forth as SEQ ID NO:80.<br>
The new panD gene RBSs were then substituted into the P2$panBCD operon expression cassette as follows. First, a deletion-insertion mutation which removes the region of panC containing the panD RBS was created. This was constructed by digesting pAN430 with a mixture of BspEl and Bglll and_recovering the 4235 bp fragment which is now missing the 3* end ofpanC and the 5f end_of panD. This fragment was ligated with an Aval-BamRl restriction fragment from plasmid pECC4, which contains the chloramphenicol acetyl transferase (cat) gene. The 5' extension produced by Aval digestion is compatible with that produced by BspEl while the Bglll and BamHl extensions are also compatible. The resulting plasmid was named pAN440, and its construction is illustrated in Figure 10.<br>
The resulting deletion-insertion mutation was crossed into the P26P&amp;nBCD operon via homologous recombination by transforming PA221 with linearized pAN440 and selecting for resistance to chloramphenicol on Cam5 plates containing 1 mM pantothenate. Several transformants were tested, and were all found to require 1 mM pantothenate for growth, as expected. Two of these transformants were remaned PA408A and PA408B and were assayed for pantothenate production. Neither strain synthesized measurable quantities of pantothenate, even when grown in medium<br><br>
containing pantoate and p-alanine at 5 g/1, indicating that the strains are deficient in pantothenate synthetase activity. Next, the newpanD RBSs were crossed into the P26 panBCD operon by transforming PA408 with linearized pAN441, pAN442, and pAN443 plasmid DNA and selecting for growth on TBAB plates without pantothenate supplementation. A transformation with linearized pAN430 (including the native panD RBS) was included as a control and was expected to give rise to transformants identical to PA221 described herein. Four isolates from each transformation were assayed for pantothenate and p-alanine production in SVY medium supplemented with various intermediates (Tables 10 and 11).<br><br><br><br>
As expected from previous experiments using PA221, none of the transformants that contained the native panD RBS produced measurable quantities of pantothenate when grown in medium supplemented with pantoate. However, nine of the twelve transformants expected to contain modified panD RBSs produced significant quantities of pantothenate (160-230 mg/1) under these conditions, indicating that they possess elevated levels of aspartate decarboxylase activity. When grown in medium supplemented with both pantoate and aspartate, these same nine transformants produced approximately four times more pantothenate than those with the native panD RBS. In addition, these nine transformants accumulated measurable quantities of P-alanine (230-410 mg/1). All transformants produced roughly equivalent quantities of pantothenate when grown in medium containing pantoate and P-alanine, demonstrating that each contains a functional pantothenate synthetase.<br><br>
These data demonstrate that the synthetic panD RBSs are about four times more effective than the native panD RBS in directing translation of the panD gene mRNA and evidence the utility of such synthetic RBSs in enhancing pantothenate production. Additional approaches to increasing pantothenate production can include, for example, increasing the half-life of the panD gene mRNA, increasing the strength of the promoter for panD transcription and/or increasing the stability of the PanD protein.<br>
EXAMPLE VI: Construction of Strains Containing an Integrated P26 panEl Cassette without an Antibiotic Resistance Gene.<br>
Example II describes the identification of the B. subtilis panEl gene that encodes the enzyme responsible for the majority of the ketopantoate reductase activity in B. subtilis. PA236 (containing the pAN236 plasmid) produced about twice as much pantothenate (2 g/1) as its parent strain, PA221 (1 g/1) in 24 hour SVY test tube cultures. PA236 was presumed to contain an amplified (-3 copies) integrated pAN236 plasmid based on selection for tetracycline resistance (the tetR gene product being encoded on the pAN236 plasmid in addition to the P 26 panEl cassette). Also useful in the methodologies of the present invention are strains that contain a single integrated unamplifiable copy of P26 panEl at the panEl locus, for example, without an antibiotic resistance gene in the strain. Such a strain was generated as follows.<br>
A plasmid named pAN251 was derived from pAN236 by inserting additional chromosomal sequences just upstream and just downstream from the P26 panEl cassette. These additional sequences, which provide homology to allow integration of the P26 panEl cassette dX panEl by double crossover, were obtained by PCR from chromosomal DNA as a template. pAN251 is shown in Figure 11. The nucleotide sequence of pAN251 is set forth as SEQ ID NO:81.<br>
Next, a strain was constructed which allowed selection for the incoming P26 panEl cassette. The strain included the following three components: (1) ?26panBCD\ (2) ApanEl\ and (3) i/vC", since both panEl and ilvC must be mutated to have a Pan" phenotype. The starting strain was CU550 (trpC29 ilvC4,leuC124). The P26panBCD cassette from PA221 chromosomal DNA was introduced in two steps to create strain PA290. Next, ApanEl::spec was transformed into PA290, using chromosomal DNA from strain PA240, to give strain PA294 (trpC2, ilvC4, leuC124, P26panBCD, ApanEl::spec), which is a strict pantothenate auxotroph. Finally, PA294 was transformed with plasmid pAN251, selecting for pantothenate prototrophy, to give strain PA303. This strain was expected to have the genotype trpC27 ilvC4, leuC124, P26<br><br>
panBCD, P^eponEL PA303 was checked for the correct chromosomal structure at the panEX locus by PCR using primers that flank the P26 insertion just upstream of panEl. The PCR product from PA303 was of the expected size, with a concomitant loss of the PCR product from the wild type panEl gene, consistent with having obtained the desired double crossover event. Furthermore, PA303 was tetracycline sensitive, which is also consistent with the desired double crossover event, as opposed to a Campbell-type single crossover of the plasmids into the chromosome. The trp, ilv, and leu auxotrophies from the parent strain were all maintained in PA303.<br>
*        In 24 hour liquid SVY test tube cultures, PA303 produced almost the same level of pantothenate as positive control PA236, and about twice as much as PA221, which does not contain engineered panEl as indicated in Table 12.<br><br>
EXAMPLE VII: Generation of Microorganisms Capable of Producing Pantothenate in an a-KIV (or Valine) Independent Manner<br>
a-ketoisovalerate (a-KIV) is a rate limiting intermediate for pantothenate production in certain strains deregulated for pantothenate synthesis. Addition of either a-KIV or valine at 5 g/1 increases pantothenate production about 5-fold in test tube cultures with strains such as PA221. In order to alleviate the need to feed either a-KIV or valine, strains were engineered that have an increased capacity to synthesize a-KIV.<br>
a-KIV is produced in B. subtilis from pyruvate by the sequential action of three enzymes encoded by four genes, ilvB and ilvN9 z/vC, and ilvD. In a wild type B, subtilis, three of the genes (ilvB9 ilvN, and ilvC) are the first three genes of the large ilv-leu operon. The fourth gene necessary for a-KIV synthesis, i/vD, is located by itself elsewhere on the chromosome. The B. subtilis ilv-leu operon is thought to be regulated<br><br>
only by leucine levels. Feeding of exogenous leucine reduces transcription of the ilv-leu operon by about 13-fold, probably by ah attenuation mechanism (Grandoni et al (1992) J. Bactehoi 174: 3212-3219). The only known feedback regulation in the ilv-leu pathway is the inhibition of the leuA gene product by leucine.<br>
As a first step to deregulate the synthesis of a-KIV, a copy of the ilvBNC region from the wild type B. subtilis ilv-leu operon was isolated by PCR, and installed adjacent to the P26 promoter and RBS2 on a vector, pOLL8, that was designed to integrate a single P26 expression cassette by double recombination at the amyE locus. The amyE gene encodes a nonessential a-amylase, and is a useful locus for installing expression cassettes. The resulting plasmid, pAN267, is illustrated in Figure 12. The nucleotide sequence of pAN267 is set forth as SEQ ID NO:82. pAN267 readily gave stable transformants by double crossover at the amyE locus of B. subtilis strains, as described in detail below.<br>
Construction of pantothenate overproducing strains that are leucine prototrophs<br>
Initially, a B. subtilis strain containing ilvC4 and ApanEl was used to introduce a single copy of P'26 panEl into the chromosome without using an antibiotic resistance gene. The double mutant was required to select for the incoming P26panEl cassette because a ApanEl mutation alone does not result in pantothenate auxotrophy. A strain named CU550 was obtained containing ilvC4 to be used as a basis for this type of strain construction. However, CU550 also contains a closely linked leuC124 mutation, so all strains derived from CU550 required leucine. Having shown that the combination of P26 panBCD and P2&amp;panEl was favorable for pantothenate production, the next step was to reassemble this combination of two cassettes in a leucine prototroph.<br>
Accordingly, the two cassettes were combined in two different strain backgrounds, RL-1 and PY79. To introduce chromosomal P26panEl into the PY79 and RL-1 strain backgrounds without using an antibiotic resistance gene, a strategy was used that did not rely on ilvC4. (The strategy took advantage of the observation that the ApanEl mutation causes a pantothenate bradytrophy, manifested by relatively small colonies on TBAB (rich) plates). First, ApanBr.cat and ApanE::spec were introduced into both strain backgrounds. Next, the resulting strains were transformed simultaneously with DNA from two strains, PA221 (P26panBCD) and PA303 {P26 panEl), selecting for Pan+ on TBAB plates. Colonies of two distinct sizes grew on the selective plates, with the larger size comprising about 2% of the colonies. The larger colonies were presumed to represented co-transformants that received both P26 panBCD and P26panEl, and that the smaller colonies had received only P26 panBCD. Consistent<br><br>
with this prediction, the larger colonies had lost both Camr and Specr, while the smaller colonies had lost only the cat gene, and retained the spec gene. Furthermore, a representative derivative of PY79 named PA327, and a representative derivative of RL-1, named PA328, both produced the elevated levels of pantothenate in test tube cultures which was about 1.6 to 1.7 g/1 (Table 13).<br><br>
Thus, PA327 and PA328 were concluded to contain both P26ponBCD and P26 panEl, and were used for further constructions as described below. PCR analysis confirmed the presence of the two cassettes.<br>
Installation of a stable P26 ilvBNC cassette into two lineages ofpantothenate overproducing strains<br>
Having constructed PA327 and PA328, derivatives of PY79 and RL-1 that contain P26 panBCD and P26panEU and that are Leu+, the next step was to introduce stable copies of P26 ilvBNC. This was accomplished by transforming PA327 and PA328 with plasmid pAN267, selecting for Specr. Screening by PCR showed that about 85% of the obtained transformants contain P26 ilvBNC integrated at amyE by double crossover. One transformant of PA327, named PA340, and one transformant of PA328, named PA342, were chosen for further study.<br>
In test tube cultures grown in SVY medium plus 5 g/1 p-alanine but without added a-KIV, both PA340 and PA342 gave the expected increase in pantothenate production over that of PA327 and PA328, to about 1.3 to 2 g/1 (Table 14).<br><br><br>
The two new strains also gave a slight increase in valine secretion, indicating that the HvBNC genes had been deregulated. However, when the same strains were grown with 5 g/1 a-KIV added, a further increase in pantothenate production occurred from PA342, suggesting that a-KIV was still rate limiting in this strain background. Similar results, only with more growth and hence higher pantothenate levels, were seen in shake flask cultures (Table 15).<br>
Table 15.	Pantothenate and valine production by PA 340 and PA 342, both<br>
containing P26 HvBNC in 24 hour shake flask cultures grown in SVYwith 5 g/l ^-alanine and with or without 5 g/l a-KIV.<br><br><br>
EXAMPLE VIII: IncreasingpanD Copy Number in Strains Engineered to Overproduce pan El and the ilvBNC Gene Products Enhances Pantothenate Production<br>
Experiments where p-alanine was fed to cultures of engineered B. subtilis strains consistently showed that P-alanine was a rate limiting intermediate in pantothenate synthesis. The effect of adding additional copies of panD on pantothenate production in PA340 and PA342 was examined. Strains PA340 and PA342 were transformed with chromosomal DNA isolated from PA401 with selection on plates containing 15 ng/ml of tetracycline (Tet15 plates). Transformants derived from each parent were patched onto Tet60 plates to identify those which were likely to contain multiple copies of the expression cassette. Twelve transformants from each transformation which grew on Tet60 were streaked for single colonies on this medium and then assayed in SVY medium test tube cultures for pantothenate production. One transformant from each group was found to produce greater than 300 mg/1 pantothenate in 24 hours. These two transformants were saved and named PA404 (PA340 strain background) and PA405 (PA342 strain background). Both strains were resistant to spectinomycin, indicating that the P26 ilvBNC expression cassette was still present at amyE. PCR analysis of chromosomal DNA isolated from each strain confirmed that the deregulated panEl gene had also been retained.<br>
Next, PA404 and PA405 were evaluated in shake flask cultures which were grown in SVY medium containing maltose as the carbon source and supplemented with various intermediates. The cultures were grown for 24 and 48 hours and then assayed for pantothenate, p-alanine, and valine production. The results of this experiment are presented in Table 16. Analogous shake flask culture data for the parent strains (PA340 and PA342) are included in the tables for comparison.<br><br><br><br>
In the absence of any medium supplementation, PA404 and PA405 made four tc five times more pantothenate in 24 hours compared to their isogenic parent strains (Table 16). The supply of p-alanine was clearly limiting in the parent strains PA340 and PA342. Addition of amplified P26panD greatly increased the supply of p-alanine.<br>
EXAMPLE IX: Deregulation of the B. subtilis UvD Gene Enhances Pantothenate Production<br>
To deregulate expression of the UvD gene, standard procedures (described above) were used to integrate the constitutive P26 promoter and an artificial ribosome binding site, RBS2, just upstream of the UvD coding region. The UvD gene maps by itself, unlinked to the ilvBNC operon. First, a 2.4 kb region of the RL-1 chromosome that contains the UvD coding region and 730 bp of upstream sequence was cloned by PCR into a low copy (about 15 per E. coli cell) vector called pOK12, to give plasmid pAN257, shown in Figure 13.<br>
Taking advantage of a natural £coRI site just upstream of the native UvD gene promoter, and a natural Ncol site at the UvD start codon, an artificial sequence containing P2&amp; and RBS2 was inserted into pAN257 to give pAN263 (Figure 14). The nucleotide sequence of pAN263 is set forth as SEQ ID NO:83. In parallel with this construction, the cat gene was also inserted into pAN257, between the same upstream EcoRl site and a Bgltt site in the middle of the UvD coding region, to give pAN261, which is deleted for a large portion of the UvD gene (Figure 15). Using pAN261 and pAN263, the P26 UvD cassette could then be installed in the B. subtilis chromosome in two steps. In the first step, pAN261 is introduced by transformation, selecting for chloramphenicol resistance, and then confirming an Ilv" phenotype. In the second step, pAN263 is introduced, selecting for Ilv+, checking for chloramphenicol sensitivity, and confirming correct local structure by PCR.<br>
pAN261 was first transformed into strain RL-1 (highly competent) to give strain PA343 (AilvDr.caf), and then chromosomal DNA from PA343 was used to transform PA340 and PA342 to Ilv" auxotrophy, yielding strains named PA348 and PA349, respectively. Chromosomal DNA is inherently more efficient them monomelic plasmid<br><br>
in transforming B. subtilis. Similarly, pAN263 DNA was transformed into PA343 (moderately competent) to give strain PA345 (P26 HvD\ and then PA345 chromosomal DNA was used to transform PA348 and PA349 to Ilv+ prototrophy, yielding strains PA374 and PA354, respectively.<br>
As predicted, PA374 and PA354 gave further increases in pantothenate production, to about 2.5 to 2.9 g/1, in test tube cultures grown in SVY plus 5 g/1 p-alanine (Table 17).<br>
Table 17. Pantothenate and valine production by PA374 and PA354, containing P26<br>
ilvD, and controls, in 24 hr test tube cultures grown in SVYwith 5 g/lfi-alanine and with or without 5 g/l a-KIV.<br><br>
In the absence of added p-alanine, strains PA374 and PA354 produced only about 0.2 g/1 pantothenate in test tube cultures, indicating that PanD activity is significantly rate limiting.<br>
To alleviate this limitation, the amplifiable P26panD cassette from strain PA401 was installed. PA401 chromosomal DNA was transformed into PA374 and PA354, selecting for Tetr at 15 mg/1, to yield strains PA377 and PA365, respectively. After transformants were obtained, the strains were streaked on plates containing 30 and 60 mg/1 tetracycline to reamplify the copy number of the P26panD cassette integrated at the bpr locus. In test tube cultures grown in SVY without a-KIV or p-alanine, a substantial improvement in pantothenate titers over those of PA374 and PA354 was obtained (Tables 18 and 19).<br><br><br><br>
aspartate synthesis is suppressed by glucose. Strains PA377 and PA365 have been evaluated in 10 liter fermentors, where they typically produce above 20 g/1 pantothenate in 48 hours without supplemental p-alanine and a-KIV or valine, described in detail below.<br>
EXAMPLE X: 10 liter Fermentations of Pantothenate-Producing Microbes<br>
Engineering of the P26 UvBNC and P26 UvD cassettes to give strains PA342 and PA354 allowed the production of 22 and 26 g/1 of pantothenate, respectively, without the addition of valine or a-KIV to the fermentation medium (Table 20). At 48 hours, both strains had secreted about 0.5 g/1 of valine into the medium.<br><br>
Pantothenate synthesis in fermentors<br>
With the addition of the PjspanD cassette to strains PA354 and PA374 to create strains PA365 and PA377, neither p-alanine nor a-KIV needed to be added to the fermentors. Strain PA365 produced 21 g/1 pantothenate in 48 hours and 27 g/1 in 72 hours with no precursors added to the medium (Table 20). PA377 was somewhat better, producing 18 g/1 of pantothenate in 36 hours, 22 g/1 in 48 hours, and 31 g/1 in 72 hours). Valine was measured at 0.85 and 1.5 g/1 for strains PA365 and PA377, respectively, at<br><br>
48 hours in SVYG medium. Strain PA377 maintained valine between 1 -1.5 g/1 throughout most of the fermentation and P-alanine between 0.2 and 0.5 g/1.<br>
Strain PA377 was further evaluated in 10-liter fermentors in yeast extract based PFMG medium. Pantothenate yields in PFMG and SVYG medium were similar. In PFMG, PA377 produced 19 g/1 of pantothenate in 36 hours, 25 g/1 in 48 hours, and 29 g/1 in 72 hours. In SVYG, PA377 produced 18 g/L pantothenate in 36 hours, 22 g/L in 48 hours and 31 g/L in 72 hours (Table 20).<br>
EXAMPLE XI: Converting Strain PA377 to a Tryptophan Prototroph<br>
PA377 (Trp") was transformed to Trp+ using chromosomal DNA from PY79 to give strain PA824. After re-amplification of the P26panD casette, PA824 was compared to PA377 for pantothenate production in test tube cultures grown in SVY glucose with or without 5 g/L p-alanine (Table 21).<br><br>
The Trp-f strains grew to slightly higher densities than PA377. In the absence of exogenous p-alanine, all of the strains produced similar levels of pantothenate, while with the addition of p-alanine, the Trp+ derivatives produced somewhat more pantothenate.<br>
Fermenior studies with PA824<br>
PA824 was evaluated in CF3000 Chemap 14 liter vessels with 10 liter working volumes. Formulations for two of the media used in the fermentors are given in Tables 22 and 23.<br><br><br><br><br><br>
between 0.2 and 1.0 g/L by continuous feeding of a 60% glucose solution. The variable rate feed pump was computer controlled and linked to the dissolved oxygen concentration [pC^] in the tank by an algorithm. When the [p02]fell to 30%, computer control began to automatically adjust the agitation rate to maintain a dissolved oxygen concentration between 25 and 30% [p(&gt;2]. Computer control and data recording were by Braun MFCS software.<br>
In one study, PA284 was grown in fermentors at two temperatures (40°C and 43°C) in the medium described in Table 22. Results of two experiments demonstrated that the highest pantothenate titers at early time points were produced at 43°C. The cell mass approached 150 optical density units at OD600 and 56 hours at 43°C, and the pantothenate titers were 21 g/L, 28 g/L and 36 g/L at 36,48 and 72 hours respectively. In the parallel fermentation at 40°C, the cell mass approached 120 optical density units at OD600 and 56 hours, and the pantothenate titers were 18 g/L, 26 g/L and 37 g/L at 36, 48 and 72 hours, respectively.<br>
In another study, PA824 was grown in a fermentor at 43 °C in the medium described in Table 23. The cell mass exceeded 160 optical density units at OD600 and 36 hours, and the pantothenate titers were 23 g/L, 34 g/L, 37 g/L and 40 g/L at 24, 36, 48 and 60 hours, respectively. In other fermentations, increasing the amount of trace elements in the glucose feed (e.g., increasing the concentration of SM from IX to 2X) resulted in even higher titers of pantothenate.<br>
EXAMPLE XII: Identification and characterization of the B. subtil is coaA gene product<br>
The annotated version of the B. subtilis genome sequence available on the "Subtilist" web site contains no gene labeled as coaA. However~a homology search using the protein sequence of K coli pantothenate kinase as a query sequence gave a good match with B. subtilis gpneyqjS, which is annotated as "unknown; similar to pantothenate kinase." This gene appears to be the penultimate gene in an operon containing five open reading frames (Figure 18). Two of the open reading frames encode proteins which are similar to D-serine dehydratase and to "ketoacyl reductase"; the other two have no known homologies. For the open reading frame corresponding to coaA, there are three possible start codons; each having a possible ribosome-binding site (RBS) associated with it. The three potential coaA ORFs were named coaAl, coaA2, and coaA3, from longest to shortest.<br><br>
All three potential coaA open reading frames were cloned along with their respective RBSs by PCR followed by ligation into expression plasmid pAN229.<br>
pAN229 is a low copy vector in E. coli that provides expression from the SP01 phage Pis promoter and can integrate by single crossover at bpr with tetracycline selection. A representative resulting plasmid, pAN281, is shown in Figure 19.<br>
To determine if the cloned putative coaA ORFs actually encode a pantothenate kinase activity, several isolates of all three plasmids were transformed into the E. coli strain YH1, that contains the coaA15(Ts) allele. Transformants were streaked to plates incubated at 30° and 43°C to test for complementation of the temperature sensitive allele. All isolates of all three coaA variants, except for one isolate of pAN282, complemented well at 43 °C, indicating that all three plasmid constructs encode an active pantothenate kinase. Accordingly, it can be concluded that the B. subtilisyqjS open reading frame codes for an active pantothenate kinase.<br>
EXAMPLE XIII: Deletion of the coaA gene from the B. subtil is genome<br>
The coaA gene of B. subtilis iyqjS) was deleted from the chromosome of a B. subtilis strain by conventional means. The majority of the coaA coding sequence was deleted from a plasmid clone and replaced by a chloramphenicol resistance gene (cat\ while leaving approximately 1 kb of upstream and downstream sequence to allow homologous recombination within the chromosome, to give plasmid pAN296 (see Figure 17). pAN296 was then used to transform a B. subtilis strain (PY79), selecting for chloramphenicol resistance. The majority of transformants result from a double crossover event that effectively substitutes the cat gene for the coaA gene. The transformed strain containing the coaA deletion - cat insertion grew normally due the presence of a second B. subtilis pantothenate kinase encoding gene described herein.<br>
EXAMPLE XIV: Identification and characterization of a second B. subtilis gene encoding pantothenate kinase activity<br>
As described in detail in the instant specification, in order to maximize pantothenate production, it is necessary to restrict the flow of pantothenate toward Coenzyme A (CoA), for example, by reducing the activity of pantothenate kinase, the first enzyme in the pathway from pantothenate to CoA. After finding that deletion of the coaA gene from the chromosome of 5. subtilis is not a lethal event (see Example XIII), it was concluded that B. subtilis must contain a second gene that encodes an active pantothenate kinase, since pantothenate kinase is an essential enzyme activity.<br><br>
A second pantothenate kinase-encoding gene was identified by complementing the E. coli strain YH1 {coaA15(Ts)) with a B. subtilis gene bank and selecting for transformants that were able to grow at 43 °C. Found among the transformants were two families of plasmids that had overlapping restriction maps within each family, but not between the families. As expected, the restriction map of one family was identical to that predicted from the B. subtilis genome sequence for the homologue of the E. coli coaA gene (which we named coaA also, see above) and surrounding sequences. The other family had a restriction map that was completely non-overlapping with the first.<br>
DNA sequencing of the ends of the cloned inserts from the second family showed that the clones came from a region of the B. subtilis chromosome that includes the 3' end of the ftsli gene, the 5' end of the sul gene, and all of the yacB, yacC, yacD, cysK, pabB, pabA andpabC genes. None of the open reading frames of these cloned inserts showed homology to any known pantothenate kinase sequences, either prokaryotic or eukaryotic.<br>
Several deletions were created through the B. subtilis genomic sequences in the cloned inserts. Each deletion was tested for complementation of the E. coli temperature sensitive pantothenate kinase. In particular, a deletion that removed all DNA between a Stu I site in the cloning vector and a Swa I site in the yacC gene, leaves yacB as the only intact open reading frame in the cloned insert (see Figure 21). This deleted plasmid still complemented the E. coli pantothenate kinase mutant. However, another deletion that removed DNA from the Swa I site in yacC through a Bstl 1071 site in the (already truncated)/fs// gene, could not complement the E. coli pantothenate kinase mutant. From these results, it was concluded that the yacB opea reading frame was responsible for the complementation activity. To confirm that yacB is a pantothenate kinase gene, theyacB ORF plus 112 base pairs of downstream flanking sequence was amplified by PCR in two independent reactions and cloned downstream of a constitutive promote to give plasmids pAN341 and pAN342 (Figure 22). Both pAN341 and pAN342 complemented the defect in YH1 at 44°C, while a control plasmid, which has the same backbone, but expresses panBCD instead of yacB did not. This confirmed that the yacB open reading frame was responsible for the complementation of YH1.<br>
As such, a novel gene that encodes pantothenate kinase activity in B. subtilis has been discovered that is not related by homology to any previously known pantothenate kinase gene. This gene has been renamed coaX, as a second, alternative gene that encodes an enzyme that catalyzes the first step in the pathway from pantothenate to CoaA. Deletion of coaX by methods described above for deleting coaA, in conjunction<br><br>
with reduction in the activity of the CoaA enzyme, provides a means to reduce pantothenate kinase activity to the desired level.<br>
Several homologues of the B. subtilis coaX gene were identified by homology searching of various publically available databases using the published yacB (coaX) open reading frame sequence and predicted amino acid sequence (as set forth in SEQ ID NOs:84 and 85 respectively). In two cases (Mycobacterium tuberculosis and Streptomyces coelicolor) the homologous coaX genes are adjacent to, or almost adjacent to, pantothenate biosynthetic genes, consistent with these homologs having a role in pantothenate metabolism. The CoaX proteins show no homology to the CoaA family of pantothenate kinases, nor to the eukaryotic family of pantothenate kinases exemplified by PanK of Saccharomyces cerevisiae.<br>
Alignment of the amino acid sequences of several bacterial CoaX homologs with the amino acid sequence predicted from translating the B. subtilis yacB ORF described in the published B. subtilis genome sequence revealed that the CoaX proteins from other bacteria contained additional amino acid residues at their carboxy-terminal ends. Moreover, these extensions beyond the end of the predicted amino acid for the £. subtilis gene product contained two relatively well conserved segments of sequence.<br>
Translation of nucleotide sequences just downstream from the stop codon of the B. subtilis yacB ORF in a different reading frame revealed the existence of amino acid sequences very similar to the carboxy-terminal extensions of the other bacterial CoaX proteins. It is thus believed that an error exists in the published DNA sequence of the B, subtilis yacB ORF sequence that causes a frame shift leading to an artifactual downstream amino acid sequence and premature termination.<br>
The PCR-generated sequences of B. subtilis CoaX in pAN341 and pAN342 (described above) contain enough downstream flanking sequence to encode the putative carboxy-terminal extension described above, which is consistent with the result that the clones were functional in the complementation assay. However when the 3' PCR primer was positioned to include only the shorter yacB ORF predicted from the published sequence, but not to include the putative carboxy-terminal extension, then the resulting plasmids, pAN329 and pAN330 (similar in structure to pAN341 and pAN342; see Figure 22), did not complement the defect in YH1. This result supports the notion that the published yacB coding sequence contains a frame-shift error, and that the carboxy-terminal end of CoaX is necessary for pantothenate kinase activity. The predicted correct nucleotide sequence for B. subtilis coaX'xs set forth as SEQ ID NO: 19 and the translated amino acid sequence is set forth as SEQ ID NO:9. A multiple<br><br>
sequence alignment of the CoaX amino acid sequences of B. subtilis and 11 homologues thereof is set forth in Figure 23.<br>
EXAMPLE XV: Generation of mutant coaA genes encoding pantothenate kinase having reduced or temperature sensitive activities<br>
This Example describes strategies for modifying the coaA gene (i.e., by introducing point mutations) to reduce the activity of pantothenate kinase after coaX is deleted from the genome.<br>
Cloning and sequencing of the temperature sensitive allele of the E. coli coaA gene.<br>
Two E. coli strains, each exhibiting a different mutant CoaA phenotype, were obtained from the E. coli Genetic Stock Center. Strain DV62 contains the coaA 15(Ts) allele, and DV79 contains the coaA16(Fr) mutation. DV62 is temperature sensitive at 43°C and produces a pantothenate kinase that is temperature sensitive. DV79 was obtained by reversion of DV62 to temperature resistance, and it produces a temperature stable, feedback resistant pantothenate kinase activity. Since the DNA sequences of these alleles are not available in the literature, the coaA genes from the two mutant strains were cloned by PCR and sequenced, in addition to a coaA gene from a strain that is wild type at the coaA locus, MM294. The PCR primer at the 5' end was designed to include the start codon plus four bases upstream, and added an arbitrarily chosen ribosome binding site (RBS). The three PCR generated fragments were each iigated between the Xbal and BamHI sites of pAN229 to give pAN284 (from coaA 15(Ts)\ pAN285 (from wild type coaA), and pAN286 (from coaA16(Fr)\ pAN229 is a low copy &amp; coli vector that provides expression from the Pis promoter and that can integrate by single crossover at bpr in B. subtilis with tetracycline selection.<br>
All three plasmids were transformed into the E. coli strain YH1 for complementation testing. All three plasmids complemented the temperature sensitive coaA mutation in E. coli YH1. It is presumed that the coaA15(Ts) gene in pAN284 is probably significantly overexpressed relative to the normal chromosomal gene, such that the overproduction compensates for the temperature sensitive defect. Complementation of a defect by overproduction is a well-documented phenomenon in E. coli.<br>
The coaA coding regions from pAN284~285rand 286 were^ubdoned into pGEM7 to give pAN306, 307, and 308, respectively, for DNA sequencing. As expected, the DNA sequence of the insert in pAN307 (from wild type coaA) matched the coaA sequence from the E. coli genome database (GenBank™). The sequence from pAN306 contains a single base change that causes a S176L substitution (i.e., a Ser -&gt;<br><br>
Leu substitution in the amino acid sequence set forth as SEQ ID NO:2). Interestingly, the DNA sequence of the pAN308 insert, derived from the feedback resistant strain, was identical to that derived from its temperature sensitive parent (represented in pAN306). This is in accord with the genetic data that indicates that the reversion of the temperature sensitive mutation occurred at a second site unlinked to the coaA gene.<br>
The S176L mutation, predicted to cause the temperature sensitive defect in E. coli pantothenate kinase, changed a serine residue that is conserved in all known or suspected bacterial coaA encoded pantothenate kinases, including that of B. subtilis (see SEQ ID NO:3 and refer to alignment). Based on this, a serine to leucine change at the homologous residue in the B. subtilis pantothenate kinase is predicted to result in either a temperature sensitive enzyme or one which is less active. Accordingly, to produce a mutant B. subtilis coaA gene, this specific change was introduced into the B. subtilis coaA gene. The mutant version is installed in the chromosome of a B. subtilis strain deleted for coaX9 for example, and the recombinant microorganism is checked for temperature sensitivity (e.g., reduced growth at 43°C). The mutation is then installed into a pantothenate overproducing strain, preferably a strain deleted for the above mentioned coaX gene by standard methods to give strains favorable for pantothenate production in B. subtilis, i.e.y a strain that has reduced pantothenate kinase activity under typical fermentation conditions.<br>
Additional coaA point mutations resulting in reduced pantothenate kinase activity<br>
Of course it is expected that many other point mutations or combinations of more than one point mutation in B. subtilis coaA will also lead to reduced activity. Appropriate mutations can be generated by mutagenic polymerase chain reaction and in vitro recombination, and identified by screening for alleles that poorly complement the E. coli coaA15(Ts) mutant An example of such a mutation of this type is a tyrosine to histidine substitution at amino acid 181 of B. subtilis coaA, generated by mutagenic polymerase chain reaction (see SEQ ID NO:3 and first line of the alignment of Figure 24).<br>
Isolate pAN282A was derived from the middle-sized B. subtilis coaA open reading frame described inixample XIL-pAN282A complemented the E:-eoIi coaA15(Ts) mutant very poorly, but nonetheless at a level that was detectable above background. As was done for the E. coli coaA clones, the open reading frame from pAN282A was subcloned into pGEM7 to give pAN303. The DNA sequence of the insert in pAN303 showed a single base change that led to a tyrosine to histidine amino<br><br>
acid change at the tyrosine corresponding to Y181 of SEQ ID NO:3. This tyrosine residue is conserved in all bacterial coaA genes/homologues present in GenBank (Figure 24). This tyrosine residue and the serine that is altered in the E. coli temperature sensitive pantothenate kinase described above are separated by only three amino acid residues in a region which is highly conserved in bacterial pantothenate kinases whereas the DNA sequence of a second isolate of the middle-sized open reading frame, from pAN282B, was identical to the wild type sequence from the B. subtilis genome sequencing project. The single base change found in pAN303 probably occurred during PCR amplification of the coaA gene. If this variant of coaA2 has sufficient residual biological activity in B. subtilis, it may be useful in the future for providing reduced pantothenate kinase activity.<br>
A preferred plasmid that can serve as a basis for mutagenizing the coaA open reading frame is pAN294 (see e.g., Figure 25 and Example XII). Briefly, mutagenic PCR is performed using pAN294 as a template and variants of coaA having reduced pantothenate kinase activity are screened as described above. Alternatively, mutations such as the one isolated in pAN282A can be installed into pAN294. The desired mutation is then introduced into the chromosome of a B. subtilis strain by transformation with the appropriate pAN294 derivative and selected for chloramphenicol resistance at 5 mg/L. Among the resulting transformants will be isolates that contain the desired mutation.<br>
In a similar fashion, mutations that induce the activity of the CoaX enzyme can be generated and identified, and such mutations used for optimizing pantothenate production by reducing CoA production as described above.<br>
EXAMPLE XVI:   Deleting the second pantothenate kinase gene, coaXgene from B. subtilis<br>
With thelcnbwledgegained above concerning"the existence and nature of cooV; one can create a deletion of the coaX open reading frame from the B. subtilis chromosome that will remove the encoded activity, and that will not adversely affect the expression of the genes downstream from coaX. In such a deleted strain, the coaA gene will be the only gene that encodes pantothenate kinase.<br><br>
To delete the cooJf gene from B. subtilis, plasmid pAN336 (SEQ ID NO:92), which contains upstream and downstream homology for double crossover, was constructed with a kanamycin resistance gene replacing most of the coaXORF (Figure 26). Strain PY79 was transformed to kanamycin resistance by pAN336, and an isolate confirmed to have resulted from a double crossover by PCR was named PA876. As predicted, deletion of coaX by itself is not lethal for B. subtilis. Furthermore, chromosomal DNA from PA876 would not transform competent PA861 (PY79 JcoaA ::cat) to kanamycin resistance. These results indicate that it is the combination of AcoaAwcat and AcocQCw kan that is lethal for B. subtilis, confirming that B. subtilis contains two unlinked genes that encode pantothenate kanase, coaA and coaX, and that either gene alone is capable of supplying sufficient pantothenate kinase for a normal rate of growth.<br>
EXAMPLE XVII:    Construction of a plasmid designed to allow directed mutagenesis of the B. subtilis coaA gene<br>
In order to easily introduce mutated coaA genes into the B. subtilis chromosome, it was necessary to install an antibiotic resistance gene adjacent to the coaA gene. This was accomplished by joining together in the vector pGEM5 three DNA fragments: (1) a 3.4 kb DNA sequence containing 2.5 kb of genomic sequence upstream from coaA and the coaA open reading frame(s); (2) a 1.1 kb DNA sequence containing a chloramphenicol resistance gene (cat); and (3) a 1.4 kb DNA sequence comprising a region downstream from the operon that contains coaA.  The resulting plasmid, named pAN294, effectively replaces the open reading frame yq}T_ (the open reading frame just downstream from coaA) with the cat gene, with enough homology flanking both sides of the cat gene to allow double recombination into the B. subtilis chromosome (Figure 25). pAN294 was transformed into B. subtilis strain PY79, selecting for chloramphenicol resistance at 5 mg/1 to give strains PA836 and PA837, which are presumably identical. PA836 and 837 were checked by diagnostic PCR to show that the cat gene had integrated by double crossover, as opposed to single crossover. PA836 and PA837 grow normally, leading to the conclusion that the open reading frame yqjT is not essential (/.e, iheyqjT open reading frame could be deleted from strains PA836 and PA837 with no significant effect on growth or pantothenate production). Thus, variant alleles (/.£., mutations) of the coaA gane can be introduced into pAN294 and the resulting plasmids can be used to introduce the variant alleles into the chromosome of, for example, a B. subtilis strain.<br><br>
EXAMPLE XVIII: Generation of mutant coaX genes encoding pantothenate kinase having reduced or temperature sensitive activities<br>
Mutant coaX genes are generated by introducing point mutations into the gene and testing the resulting mutants for the ability to complement the E. coli YH1 strain as described in Example XII. Preferred mutations in the coaX gene seqeunces are those that encode a substitution of a residue conserved among CoaX sequences from a variety of bacterial sources {e.g., a conserved residue set forth in Figure 23). Alternatively, random mutations in the coaX gene sequence are generated by mutagenic PCR and in vitro recombination and identified by screening for alleles that poorly complement the E. coli coaAl5(Ts) mutant.<br>
Mutants so generated (Le.y mutants having reduced coaX activity) can be further engineered such that the endogenous coaA gene is deleted (as described in Example XIII). CoaX reduced-activity mutants can also be further engineered to contain reduced-activity CoaA gene products as described in Example XV.<br>
EXAMPLE XIX: Enhanced Production of Panto-Compounds Using Bacteria Having Deletions in One or More Pantothenate Biosynthetic Enzymes<br>
If the desired panto-compound is not pantothenate, then an appropriate deletion of one or more of the pantothenate biosynthetic genes from a pantothenate overproducing strain will provide a strain that produces said desired panto-compound. In this example, the desired panto-compound is pantoate. Starting with, for example, strain PA236, PA313 or PA824 either one or both of the panC andpanD genes is deleted. In another example, ketopantoate is the desired panto-compound. Starting with, for example strain PA244, PA245 or PA824 one, two or all of the UvC9panEJ9 panC andpanD genes are deleted from the starting strain. If p-alanine is the desired panto-compound, then panB and /wwCx^ui be^deleted,preferablyuLa fashion that leaves an in frame fusion of a small portion of the 5' end of panB with a small portion of the 3' end ofpanC, from the strain PA221, PA235, PA245, or PA313. In all of the above-mentioned examples, the panto-compound producing strain will be a pantothenate auxotroph. Accordingly, the growth medium requires sufficient pantothenate for adequate growth. Vectors designed to overexpress panD as described above are then transformed into the above strains to fiirther enhance (^-alanine production.<br>
The above-mentioned deletions are accomplished by methods well-known to those skilled in the art, for example, by insertion of an antibiotic resistance gene and removing sufficient sequence from the target gene(s) to inactivate said target gene(s).<br><br>
Alternatively, removal of targeted sequences is accomplished without simultaneous introduction of an antibiotic resistance gene in said target gene and then introduced by<br>
congression (co-transformation with any other appropriate selectable DNA sequence) followed by screening for the loss of function of said target gene by replica plating.<br>
Table 24 : Strains (and corresponding phenotypes) for panto-compound production<br>
Name Pheno    Drug       pan BCD     panE locus     UvD      amyE locus       bpr locus       Parent<br>
type    resist.	locus	locus   •<br>
PA221    Trp-	P26panBCD<br>
PA222	"                Pl5 panBCD                                                                                             RL-1<br>
PA235	P26panBCD<br>
PA236	PH panBCD	P76 panEl                                                                    PA221<br>
PA327	Trp-                  P26panBCD	P26panEJ                                                                   PA221<br>
PA328	Trp-                  P26panBCD	P26panEt                                                                   PA235<br>
PA340	Trp-       Spc    P26panBCD	P26panEl	P26UvBNC                               PA327<br>
PA342	Trp-       Spc    P26panBCD	P26panEl	P26itvBNC                               PA328<br>
PA354	Trp-       Spc    P26panBCD	P26panEl   P26dvD  P26ilvBNC                              PA342<br>
PA365    Tip-      Spc,    P26panBCD    P26panEl   P26iivD  P26ilvBNC    P26panD423     PA354<br>
Tet<br>
PA374    Trp-       Spc    P26panBCD    P26panEl   P26iivD  P26ilvBNC	PA340<br>
PA377    Trp-      Spc,    P26panBCD    P26panEl   P26ilvD  P26iivBNC    P26panD423     PA374<br>
Tet<br>
PA40I    Trp-	P26panBCD	P26panD423     PA221<br>
PA402    Trp-	P26panBCD	P26panD428     PA221<br>
PA403    Trp-	P26panBCD	P26panD429     PA22I<br>
PA404    Trp-      Spc,    P26panBCD    P26panEl	P26iIvBNC    P26panD423     PA340<br>
Tet<br>
PA405    Trp-      Spc,    P26panBCD    P26panEl	P26UvBNC   J&gt;26panD423     PA342<br>
Tet<br>
PA65I    Trp-       Spc    P26panBC*D P26panE1  P26itvD  P26itvBNC	PA374<br>
PA284	Spc,    P26'panBCD   P26panEl   P26ilvD  P26ihBNC    P26panD423     PA377<br>
Tet<br>
Equivalents     Those skilled in the art will recognize, or be able to ascertain using no more than routine experimentation, many equivalents to the specific embodiments of the invention described herein. Such equivalents are intended to be encompassed by the following claims.<br><br><br><br><br>
WE CLAIM :<br>
1.	A method of producing a panto-compound such as ketopantoate, pantoate, li-alanine and pantothenate characterized in that it comprises culturing a recombinant microorganism belonging to genus selected from the group consisting of Bacillus, Cornyebacterium, Lactobacillus, Lactococci and Streptomyces which over expresses at least one Bacillus pantothenate biosynthetic enzyme under standard conditions such that the panto-compound is produced.<br>
2.	The method as claimed in claim 1, wherein the microorganism overexpresses at least one Bacillus subtilis pantothenate biosynthetic enzyme.<br>
3.	The method as claimed in claim 1 or 2, wherein the pantothenate biosynthetic enzyme is selected from the group consisting of ketopantoate hydroxymethyltransferase, pantothenate synthetase, aspartate-a-decarboxylase and ketopantoate reductase.<br>
4.	The method as claimed in any one of claims 1 to 3, wherein the microorganism overexpresses at least two pantothenate biosynthetic enzymes.<br>
5.	The method of any one of claims 1 to 3, wherein the microorganism overexpresses at least three pantothenate biosynthetic enzymes.<br>
6.	A method of producing a panto-compound comprising culturing a ketopantoate reductase-overexpressing(KPAR-O) microorganism under standard conditions such that the panto-compound is produced.<br>
7.	The method as claimed in claim 6, wherein the panto-compound is pantothenate or pantoate.<br><br>
The method as claimed in claim 6 or 7, wherein the ketopantoate reductase is bacterial-derived.<br>
The method as claimed in claim 6 or 7, wherein the ketopantoate reductase is derived from Bacillus.<br>
The method as claimed in claim 6 or 7, wherein the ketopantoate reductase is derived from Bacillus subtilis.<br>
The method as claimed in any one of claims 6 to 10, wherein the KPAR-0 microorganism further overexpresses at least one pantothenate biosynthetic enzyme in addition to overexpressing ketopantoate reductase.<br>
The method as claimed in claim 11, wherein the KPAR-0 microorganism further overexpresses at least one of ketopantoate hydroxymethyltransferase, pantothenate synthetase and aspartate-a-decarboxylase.<br>
The method of claim 1 for producing pantothenate in a manner independent of precursor feed comprising culturing an aspartate-a-decarboxylase-overexpressing (AaD-O) microorganism having a deregulated isoleucine-valine(ilv) pathway under standard conditions such that pantothenate is produced.<br>
A method of producing at least 2 g/L pantothenate in a manner independent of aspartate or (3-alanine feed comprising culturing an aspartate-a-decarboxylase-overexpressing (AaD-O) microorganism under standard conditions such that pantothenate is produced.<br><br>
A method of producing at least 2 g/L pantothenate in a manner independent of valine or a-ketoisovalerate feed comprising culturing a microorganism having a deregulated isoleucine-valine(ilv) biosynthetic pathway under standard conditions such that pantothenate is produced.<br>
A method of producing at least 30 g/L pantothenate in a manner independent of aspartate or (3-alanine feed comprising culturing an aspartate-a-decarboxylase-overexpressing (AaD-O) microorganism under standard conditions such that pantothenate is produced.<br>
A method of producing at least 30 g/L pantothenate in a manner independent of valine or a-ketoisovalerate feed comprising culturing a microorganism having a deregulated isoleucine-valine(ilv) biosynthetic pathway under standard conditions such that pantothenate is produced.<br>
The method as claimed in any one of claims 13 to 17, wherein the microorganism overexpresses acetohydroxyacid synthetase or is transformed with a vector comprising an ilvBN nucleic acid sequence or an alsS sequence.<br>
The method as claimed in any one of claims 13 to 17, wherein the microorganism overexpresses acetohydroxyacid isomeroreductase or is transformed with a vector comprising an ilvC nucleic acid sequence.<br>
The method as claimed in any one of claims 13 to 17, wherein the microorganism overexpresses dihydroxyacid dehydratase or is transformed with a vector comprising an ilvD nucleic acid sequence.<br>
The method as claimed in claim 20, wherein the microorganism overexpresses aspartate-a-decarboxylase or is transformed with a vector comprising a panD nucleic acid sequence.<br><br>
The method as claimed in claim 1, wherein the microorganism is transformed with a vector comprising a panBCD nucleic acid sequence or a vector comprising a panEI nucleic acid sequence.<br>
The method as claimed in any one of claims 13 to 15 and 18 to 22, wherein pantothenate is produced at a level selected from the group consisting of a level greater than lOg/L, a level greater than 20g/L and a level greater than 40g/L.<br>
The method as claimed in claim 18, wherein the microorganism overexpresses acetohydroxyacid synthetase derived from Bacillus or is transformed with a vector comprising an ilvBN nucleic acid sequence or an alsS nucleic acid sequence derived from Bacillus.<br>
The method as claimed in claim 19, wherein the microorganism overexpresses acetohydroxyacid isomeroreductase derived from Bacillus or is transformed with a vector comprising an ilvC nucleic acid sequence derived from Bacillus.<br>
The method as claimed in claim 20, wherein the microorganism overexpresses dihydroxyacid dehydratase derived from Bacillus or is transformed with a vector comprising an ilvD nucleic acid sequence derived from Bacillus.<br>
The method as claimed in claim 21, wherein the microorganism overexpresses aspartate-a-decarboxylase derived from Bacillus or is transformed with a vector comprising a panD nucleic  acid sequence derived from Bacillus.<br>
The method as claimed in claim 22, wherein the vector comprises a panBCD nucleic acid sequence or a panEI nucleic acid sequence derived from Bacillus.<br><br>
A method of producing [3-alanine .comprising culturing an aspartate-a-decarboxylase-overexpressing (AaD-O) microorganism under standard conditions such that (3-alanine is produced.<br>
The method as claimed in claim 29, wherein the AaD-0 microorganism has a mutation in a nucleic acid sequence encoding a pantothenate biosynthetic enzyme selected from the group consisting of ketopantoate hydroxymethyltransferase, ketopantoate reductase and pantothenate synthetase.<br>
A method of producing (3-alanine comprising contacting a composition comprising aspartate with an isolated Bacillus aspartate-a-decarboxylase enzyme under standard conditions such that (3-alanine is produced.<br>
The method as claimed in claim 1, for enhancing production of a panto-compound comprising culturing a mutant microorganism having a mutant coaX gene under conditions such that the panto-compound production is enhanced.<br>
The method as claimed in claim 32, wherein said recombinant microorganism has a mutant coaA gene.<br>
The method as claimed in any one of claims 32and 33, wherein said panto-compound is selected from the group consisting of ketopantoate, pantoate or pantothenate.<br>
The method as claimed in any one of claims 32 and 33, wherein said panto-compound is pantothenate.<br>
The method as claimed in any one of claims 32 and 33, wherein said panto-compound is produced at a level selected from the group consisting of a level greater thanl lOg/L, a level greater than 20g/L and a level greater than 40g/L.<br><br>
The method as claimed in claim 1 for enhancing production of a panto-compound comprising culturing a microorganism that has a deregulated pantothenate biosynthetic pathway and that also has a mutation that results in reduced pantothenate kinase activity under conditions such that the panto-compound production is enhanced.<br>
The method as claimed in any one of claims 1 to 37, wherein the microorganism is Bacillus subtilis.<br>
The method as claimed in any one of claims 1 to 13, 32 to 35, 37 and 38, wherein the panto-compound is produced at a level greater than 2 g/L.<br>
A recombinant microorganism which overexpresses at least one Bacillus pantothenate biosynthetic enzyme.<br>
The recombinant microorganism as claimed in claim 40, which overexpresses at least one Bacillus subtilis pantothenate biosynthetic enzyme.<br>
The recombinant microorganism as claimed in claim 40 or 41, wherein the pantothenate biosynthetic enzyme is selected from the group consisting of ketopantoate hydroxymethyltransferase, pantothenate synthetase, aspartate-a-decarboxylase and ketopantoate reductase.<br>
The recombinant microorganism as claimed in any one of claims 40 to 42, wherein the pantothenate biosynthetic enzyme is ketopantoate reductase.<br>
A recombinant microorganism which overexpresses aspartate-a-decarboxylase and has a deregulated isoleucine-valine(ilv) biosynthetic pathway.<br>
A recombinant microorganism having a mutant coaX gene, said mutant coaX gene encoding reduced pantothenate kinase activity in said microorganism.<br><br>
A recombinant microorganism comprising a vector comprising an isolated coaX gene.<br>
A recombinant microorganism that overproduces a panto-compound, the microorganism having a deregulated pantothenate biosynthetic pathway and having at least one mutation that results in a decrease in the capacity of the microorganism to synthesize Coenzyme A (CoA).<br>
The recombinant microorganism as claimed in claim 47, having at least one mutation that results in a reduced level of pantothenate kinase activity.<br>
The recombinant microorganism as claimed in any one of claims 40 to 48, which is Gram positive.<br>
The recombinant microorganism as claimed in claim 49, belonging to a genus selected from the group consisting of Bacillus, Cornyebacterium, Lactobacillus, Lactococci and Streptomyces.<br>
The recombinant microorganism as claimed in claim 50 belonging to the genus Bacillus.<br>
The recombinant microorganism as claimed in claim 51 which is Bacillus subtilis.<br>
A recombinant microorganism selected from the group consisting of PA221, PA235, PA236, PA313,PA410, PA402, PA403,PA411, PA412, PA413, PA303, PA327, PA328, PA401, PA340, PA342, PA404, PA405, PA374, PA354, PA365, PA377, PA651 and PA824.<br><br>
A recombinant vector selected from the group consisting of pAN004, pAN005, pAN006, pAN236, pAN423, pAN428, pAN429, pAN441, pAN442, pAN443, pAN251, pAN267, pAN256, pAN257, pAN263, pAN240, pAN294, pAN296, pAN3 3 6, pAN3 41 and pAN3 42 for production of panto-compounds comprising a nucleic acid sequence which encodes at least one Bacillus pantothenate biosynthetic enzyme operably linked to regulatory sequences.<br>
The vector as claimed in claim 54, comprising a nucleic acid sequence which encodes at least one Bacillus subtilis pantothenate biosynthetic enzyme.<br>
The vector as claimed in claim 55, wherein the nucleic acid sequence encodes at least one of ketopantoate hydroxymethyltransferase, pantothenate synthetase, aspartate-a-decarboxylase and ketopantoate reductase.<br>
A recombinant vector comprising at least one nucleic acid sequence selected from the group consisting of SEQ IDNO : 23, SEQ IDNO : 25, SEQ IDNO : 27, SEQ ID NO : 29 and SEQID NO : 59.<br>
The vector as claimed in claim 55, wherein the nucleic acid sequence encodes ketopantoate reductase.<br>
A vector comprising a mutant coaX gene, said mutant encoding a pantothenate kinase enzyme having reduced activity.<br>
A vector comprising an isolated coaX gene.<br>
A vector comprising an isolated Bacillus coaX gene.<br>
A vector comprising an isolated Bacillus subtilis coaX gene.<br><br>
The vector as claimed in any one of claims 57 and 60 to 62, which further comprises regulatory sequences.<br>
The vector as claimed in any one of claims 54 to 56, 58 and 63, wherein the regulatory sequences comprise a constitutively active promoter.<br>
The vector as claimed in claim 64, wherein the constitutively active promoter comprises PVeg (SEQ IDNO : 41), P15 (SEQ IDNO : 39) or P26 (SEQ IDNO :<br>
40) sequences.<br>
The vector as claimed in claim 54, wherein the regulatory sequences comprise at least one artificial ribosome binding site (RBS).<br>
The vector as claimed in claim 66, wherein the artificial RBS comprises a nucleotide sequence selected from the group consisting of SEQ IDNO : 49, SEQ IDNO : 50, SEQ ID NO :51, SEQ ID NO : 52, SEQ ID NO : 53, SEQ IDNO : 54, SEQ ID NO : 55, SEQ ID NO : 56 and SEQ IDNO : 57.<br>
A recombinant microorganism comprising the vector as claimed in claim 57 or 64.<br>
An isolated nucleic acid molecule which encodes at least one Bacillus pantothenate biosynthetic gene.<br>
The isolated nucleic acid molecule as claimed in claim 69, which encodes at least one Bacillus subtilis pantothenate biosynthetic gene.<br>
The isolated nucleic acid molecule as claimed in claim 69 to 70, which encodes ketopantoate reductase.<br><br>
An isolated Bacillus pantothenate biosynthetic enzyme polypeptide.<br>
An isolated Bacillus subtilis pantothenate biosynthetic enzyme polypeptide.<br>
An isolated Bacillus ketopantoate reductase polypeptide.<br>
An isolated Bacillus subtilis ketopantoate reductase polypeptide.<br>
An isolated Bacillus aspartate-a-decarboxylase polypeptide.<br>
An isolated Bacillus subtilis aspartate-a-decarboxylase polypeptide.<br>
An isolated nucleic acid molecule comprising a mutant coaX gene.<br>
An isolated nucleic acid molecule comprising a coaX gene.<br>
An isolated pantothenate kinase protein encoded by a coaX gene.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItNTYyLWNoZS1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-562-che-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItNTYyLWNoZS1hc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-562-che-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItNTYyLWNoZS1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-562-che-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItNTYyLWNoZS1jb3JyZXNwb25kZW5jZSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-562-che-correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItNTYyLWNoZS1jb3JyZXNwb25kZW5jZSBwby5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-562-che-correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItNTYyLWNoZS1kZXNjcmlwdGlvbiBjb21wbGV0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-562-che-description complete.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItNTYyLWNoZS1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-562-che-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItNTYyLWNoZS1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-562-che-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItNTYyLWNoZS1mb3JtIDE5LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-562-che-form 19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItNTYyLWNoZS1mb3JtIDI2LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-562-che-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItNTYyLWNoZS1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-562-che-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItNTYyLWNoZS1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-562-che-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItNTYyLWNoZS1wY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-562-che-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="222498-reverse-conducting-gct-gate-commutated-thyristor-and-application-of-such-a-gct.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="222500-colored-pigments-and-a-process-for-preparing-the-same.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>222499</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2002/562/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>47/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Nov-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>14-Aug-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>18-Apr-2002</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>BASF AKTIENGESELLSCHAFT</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>D-67056, LUDWIGSHAFEN,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>YOCUM ROGERS R</td>
											<td>4 ORCHARD LANE, LEXINGTON, MA 02420,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>PATTERSON THOMAS A</td>
											<td>89 CHURCH STREET, NORTH ATTLEBORO MA 02760,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>HERMANN THERON</td>
											<td>18 CHILHOWIE DRIVE, KINNELON, NJ 07405,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>PERO JANICE G</td>
											<td>20 SOLOMON PIERCE ROAD, LEXINGTON MA 02420,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 9/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US00/25993</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-09-21</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>US 09/400494</td>
									<td>1999-09-21</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/221,836</td>
									<td>2000-07-28</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>60/210,072</td>
									<td>2000-06-07</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>4</td>
									<td>60/227,860</td>
									<td>2000-08-24</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/222499-methods-and-microorganisms-for-production-of-panto-compounds by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:24:50 GMT -->
</html>
