<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/222568-process-for-producing-recombinant-insulin-from-novel-fusion-proteins by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:15:36 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 222568:&quot;PROCESS FOR PRODUCING RECOMBINANT INSULIN FROM NOVEL FUSION PROTEINS&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;PROCESS FOR PRODUCING RECOMBINANT INSULIN FROM NOVEL FUSION PROTEINS&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>This invention relates to a DNA encoding a fusion protein of formula (I) [Y]-[Xl]-[B-chain]-[X2]-[Linker]-[X3]-[A-chain] (I) wherein Y is a leader peptide sequence for the expression and secretion of the protein, comprising at least one amino acid residue; XI is an amino acid sequence which is cleavable enzymatically or chemically; B-chain is the amino acid sequence of insulin B chain; X2 is an amino acid sequence which is cleavable enzymatically; Linker is a linker sequence comprising at least one amino acid residue; X3 is an amino acid sequence which is cleavable enzymatically; and A- chain is the amino acid sequence of insulin A chain, and to a method for producing insulin using an expression system containing the DNA in high efficiency and high yield.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>BACKGROUND OF THE INVENTION<br>
1.	Technical Field<br>
The present invention relates to a DNA encoding a novel fusion protein which is used for preparing recombinant insulin.    More specifically, the present invention relates to the use of the DNA for the preparation of insulin from the fusion protein, which is obtained by the expression of the DNA, through the action of thrombin and carboxypeptidase B.<br>
2.	Background Art<br>
Insulin is a hormone secreted by the B cells of the islet of Langerhans in the pancreas when an animal ingests food, which is the most important hormone for storage or use of sugars, amino acids and fatty acids and for maintaining the blood sugar homeostasis.   Although blood sugar, namely glucose in blood, is an essential energy source for a living body, if the blood sugar homeostasis is not maintained, then serious conditions may develop.   Increased blood sugar level causes the excretion of sugar in the urine, resulting in loss of glucose, i.e. onset of a so-called diabetes.    If this condition continues for long periods, complications can develop in the tissues of a living body.   On the other hand, decreased blood sugar level leads to an insufficient supply of the energy source, resulting in imperilment of life.   Homeostasis of the blood sugar level is maintained by. balancing factors that act to increase the blood sugar level (e.g., glucagon, growth hormone, cortisol, catecholamine) with factors that act to decrease the blood sugar level.   Insulin is the only hormone which can decrease the blood sugar level.   Hence, the reduction in secretion functions resulted from some causes and, as a consequence, insufficient supply of insulin can induce insulin-dependent diabetes mellitus (IDDM).    For patients suffering from such a disease, insulin is an indispensable drug.<br>
Human insulin is a polypeptide comprising an A chain with 21 amino acids and a B chain with 30 amino acids, which has one intrachain disulfide bond in the A chain and two disulfide bonds which link between the A chain and the B chain.    Insulin is initially biosynthesized as "preproinsulin" on ribosomes in B cells of the islets of Langerhans of the pancreas.    Preproinsulin is a linear molecule comprising a signal peptide with 24 amino acids (SP), a B chain (B), a C-peptide with 31 amino acids (C) and an A chain (A) linked in the order as represented by the formula "SP-B-C-A".    Upon transport to the endoplasmic reticulum, the signal peptide is cleaved out from the preproinsulin to produce "proinsulin (B-C-A)".   Proinsulin forms disulfide bonds in the endoplasmic reticulum, thereby taking on a three-dimensional structure.    Proinsulin is cleaved with a prohormone-converting enzyme PCI/3 at the B-C junction and then cleaved with a converting enzyme PC2 at the C-A junction.    Finally, N-terminal two basic amino acid residues of the C-peptide, which remain at the G-terminus of the B-chain when cleaved with PCI/3, are cut out with carboxypeptidase H.   In this manner, insulin is produced.<br>
Methods for producing therapeutic insulin have been initially developed using extracts from the pancreas of animals such as bovine and pig.    However, human insulin is different in amino acid composition from bovine insulin (at two positions in A chain and one position in B chain) and porcine insulin (at one position in B chain).    Therefore, adverse effects (e.g., allergy) are inevitable in the use of bovine or porcine insulin in human bodies.    Methods for semi-synthesis of human insulin from porcine insulin have been developed which utilize the transpeptidation reaction with trypsin.    However, recombinant insulin produced by genetic recombinant techniques has currently gone mainstream due to its low production cost and good production efficiency.<br>
For the production of recombinant insulin, a number of methods have been developed.    For example, the method developed by Eli Lilly Corp. is known, which method comprises expressing A chain and B chain separately using Escherichia coli; and mixing the A chain and the B chain in vitro to form the disulfide bridges, thereby linking them via the disulfide bonds (.TP-B- 63-18960).    This method, however, is poor in production efficiency.    Then, Eli Lilly Corp. has developed an improved method which comprises expressing proinsulin; forming the disulfide bonds in vi1ro\ and then cleaving out the C-peptide from the product with trypsin and carboxypeptidase B, thereby producing insulin (JP-B-1-48278 and Japanese Patent No. 2634176).<br>
Another method was developed by Novo Nordisk Corp., which method comprises expressing miniproinsulin comprising a B chain and an A chain linked via two basic amino acid residues, in yeast; and then treating the miniproinsulin with trypsin in vitro, thereby producing insulin (JP-B-7-121226 and JP-B-8-8871, and Japanese Patent No. 2553326).   This method has such advantages that the disulfide bonds are formed during the expression and secretion of the miniproinsulin and that the miniproinsulin can be isolated and purified readily because it is secreted into a cultured medium.<br>
Development of new recombinant insulin-production methods has been continued positively.   Hoechst Corporation developed a method comprising expressing a new-type insulin derivative or preproinsulin in E. coli; forming the disulfide bonds in vitro; and then treating the product with lysylendopeptidase or clostripain/carboxypeptidase B, thereby producing insulin (JP-A-2-195896, JP-A-2-225498, JP-A-2-233698, JP-A-3-169895, JP-A-4-258296, JP-A-6-228191, and JP-A-7-265092).   Recently, a method has been developed by BIO-TECHNOLOGY GENERAL CORPORATION, in which a fusion protein comprising superoxide dismutase (SOD) linked with proinsulin is expressed in E. coli to increase both the expression efficiency and the disulfide bond-forming efficiency, and proinsulin is converted into insulin with trypsin and carboxypeptidase B (WO 96/20724).   Thus, there are a number of approaches for recombinant insulin production, and further improvement has been made in expression efficiency, disulfide bond-forming efficiency and conversion into insulin.<br>
As the hosts for the production of recombinant proteins, a wide variety of hosts have been used including microorganisms, animals and plants.   Among them, microorganisms are most frequently used due to their easy-to-handle property and good applicability for industrial use, and Escherichia coli and yeast are especially known.   Recently, an expression system with Bacillus brevis has been known for recombinant proteins (see Japanese Patent No. 2082727; JP-A-62-201583; Yamagata, H. et al., J. Bacteriol. 169:1239-1245, 1987; Juzo Udaka, Nihon Nogei Kagaku-shi 61, 669-676, 1987; Takao, M. et al., Appl. Microbiol. Biotechnol. 30:75-80, 1989; Yamagata, H. et al., Proc. Natl. Acad. Sci. USA 86:3589-3593, 1989).<br>
The object of the present invention is to develop an expression system and a production method for insulin which have a high yield and a production efficiency equal to or better than those<br>
of the existing recombinant insulin production systems. That is, the object of the present invention is to develop a novel method for converting an insulin precursor into insulin, an environment where the disulflde bonds necessary for insulin activity can be formed, and an expression system with high yield.<br>
SUMMARY OF THE INVENTION<br>
In one aspect of the present invention, there is provided a DNA encoding a fusion protein of formula (I):<br>
(Formula Removed)<br>
wherein<br>
Y is a leader peptide sequence for expression and secretion of the protein, comprising at least one amino acid residue;<br>
XI is an amino acid sequence which is cleavable enzymatically or chemically; B-chain is the amino acid sequence of insulin B chain; X2 is an amino acid sequence which is cleavable enzymatically; Linker is a linker sequence comprising at least one amino acid residue; X3 is an amino acid sequence which is cleavable enzymatically; and A-chain is the amino acid sequence of insulin A chain, and<br>
wherein the Y, XI, B-chain, X2, Linker, X3 and A-chain are ligated in the order indicated in formula (I).<br>
In another aspect of the present invention, the present invention is provide a DNA encoding a fusion protein of formula (II):<br>
(Formula Removed)<br>
wherein:<br>
B-chain is the amino acid sequence of insulin B chain; X2 is an amino acid sequence which is cleavable enzymatically; Linker is a linker sequence comprising at least one amino acid residue; X3 is an amino acid sequence which is cleavable enzymatically; and A-chain is the amino acid sequence of insulin A chain, and<br>
wherein the B-chain, X2, Linker, X3 and A-chain are ligated in the order indicated in formula (II).<br>
In one embodiment of the present invention, the amino acid sequence XI, X2 or X3, which is used for enzymatic cleavage of a fusion protein, is a sequence cleavable with thrombin.    For example, the amino acid sequence which is cleavable with thrombin is the following sequence:<br>
XI = GlySerLeuGlnProArg (SEQ ID NO:1);<br>
X2 = ArgGlyHisArgPro (SEQ ID NO:2); or<br>
X3 = ProArg.<br>
In another embodiment of the present invention, the linker sequence has the following amino acid sequence:<br>
GluAlaGluAspLeuGlnValGlyGlnValGluLeuGlyGlyGlyProGlyAlaGlySer<br>
LeuGlnProLeuAlaLeuGluGlySerLeuGln   (SEQ ID NO:3).<br>
In still another embodiment of the present invention, the leader peptide sequence may comprise N-terminal 9 amino acid residues (i.e., the amino acid positions 1 to 9) of the protein MWP which is one of the cell wall proteins (CWPs) derived from a bacterium belonging to the genus Bacillus.   In this case, the DNA may comprise a CWP signal peptide attached at the 5' end of the DNA.<br>
An example of the DNA of the present invention is a DNA comprising a nucleotide sequence encoding an amino acid sequence shown in SEQ ID NO:21.    More specifically, the DNA comprises a nucleotide sequence shown in SEQ ID NO:20.<br>
In another aspect of the present invention, there is provided a DNA comprising a DNA sequence which comprises a promoter region required for the expression of a recombinant protein in a prokaryote or eukaryote, the DNA sequence being attached at the 5' end of the DNA defined above.<br>
In an embodiment of the present invention, the DNA sequence which comprises a promoter region is derived from a bacterium belonging to the genus Bacillus, and is preferably derived from the CWP from a bacterium belonging to the genus Bacillus.<br>
In another aspect of the present invention, there is provided a vector containing the DNA defined above.<br>
In still another aspect of the present invention, there is provided a host cell transformed with<br>
the vector.    The host cell is preferably a bacterium belonging to the genus Bacillus, such as Bacillus brevis.<br>
In still another aspect of the present invention, there is provided a method for producing insulin, wherein the method comprises:<br>
culturing the host cell or bacterium in a culture medium, to express a fusion protein encoded by a DNA of interest in the host cell or bacterium; collecting the fusion protein; and<br>
subjecting the fusion protein to an enzymatic cleavage treatment to isolate insulin. In this aspect, an example of the DNA is a DNA comprising a nucleotide sequence shown in SEQ ID NO:21, and an example of the fusion protein is a protein comprising an amino acid sequence shown in SEQ ID NO:22.<br>
In the method, the expressed fusion protein may be separated and purified from the host cell or the bacteria or the cultured medium.   According to an embodiment of the present invention, the enzymatic cleavage may be performed with thrombin and carboxypeptidase B.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
FIG. 1 illustrates the conversion of a fusion protein MWPmp9-GSLQPR-Bchain-RGHRP-Linker-PR-Achain into insulin.<br>
FIG. 2 is the amino acid sequence of a fusion protein MWPmp9-GSLQPR-Bchain-RGHRP-Linker-PR-Achain and the nucleotide sequence encoding the same.<br>
FIG. 3 is the schematic diagram depicting a process of introducing a fusion DNA into a Bacillus brevis expression vector (pNU211R2L5).<br>
FIG. 4 is the photograph of the electrophoresis of the media after culture of transformants: lane 1, a marker peptide; lane 2, a negative control (i.e., a transformant with only plasmid pNU211R2L5 and without a foreign protein); and lane 3, a transformant MWPmp9-GSLQPR-Bchain-RGHRP-Linker-PR-Achain.<br>
FIG. 5 shows the chromatogram of the fusion protein MWPmp9-GSLQPR-Bchain-RGHRP-Linker-PR-Achain isolated and purified by XL chromatography.<br>
FIG. 6 shows the chromatogram of the fusion protein MWPmp9-GSLQPR-Bchain-<br>
RGHRP-Linker-PR-Achain isolated and purified by HPLC.<br>
FIG. 7 shows the peptide mapping of ITOHAM insulin (the present invention) and Novolin (Novolin 40, which is a commercially available insulin, sold by Novo Nordisk Pharma).<br>
FIG. 8 shows the elution patterns of ITOHAM insulin (the present invention) and Novolin.<br>
FIG. 9 shows the time courses of the plasma glucose levels after administration of ITOHAM insulin (the present invention) and Novolin.<br>
FIG. 10 shows the time courses of the plasma insulin levels after administration of ITOHAM insulin (the present invention) and Novolin.<br>
DETAILED DESCRIPTION<br>
The DNA of the present invention has the structure represented by the above formula (I) or (II).   The DNA of formula (I) comprises a leader peptide sequence for expression and secretion of a fusion protein of interest, comprising at least one amino acid residue (Y); an amino acid sequence which is cleavable enzymatically or chemically (XI); a amino acid sequence of insulin B chain (B-chain); an amino acid sequence which is cleavable enzymatically (X2); a linker sequence comprising at least one amino acid residue (Linker); an amino acid sequence which is cleavable enzymatically (X3); and an amino acid sequence of insulin A chain (A-chain), wherein the Y, XI, B-chain, X2, Linker, X3, and A-chain are ligated in this order.   The DNA of formula (II) comprises a amino acid sequence of insulin B chain (B-chain); an amino acid sequence which is cleavable enzymatically (X2); a linker sequence comprising at least one amino acid residue (Linker); an amino acid sequence which is cleavable enzymatically (X3); and an amino acid sequence of insulin A chain (A-chain), wherein the B-chain, X2, Linker, X3, and A-chain are ligated in this order.    The presence of the leader peptide sequence enables to secrete the expression product outside the host cell, while the absence thereof provides the accumulation of the expression product within the host cell.<br>
In the examples described below, for establishment of a method for converting the fusion protein into insulin with thrombin and carboxypeptidase B, a novel modified proinsulin is designed in which thrombin-cleavage sites are provided between the insulin B chain and the linker peptide and between the linker peptide and the insulin A chain.    The modified proinsulin is then linked at its N-terminus, with the N-terminal 9-amino acids of a cell wall protein (CWP) from Bacillus brevis as a<br>
leader peptide, and a third thrombin-cleavage site which enables the cleavage of the modified proinsulin from the leader peptide is further linked immediately after the leader peptide, thereby providing an environment for forming disulfide bonds and enabling the expression of the modified proinsulin in Bacillus brevis.   In this manner, the linear artificial fusion protein which comprises a leader peptide, a thrombin-cleavage site, insulin B chain, a thrombin-cleavage site, a linker peptide, a thrombin-cleavage site, and insulin A chain in the order, can be designed.   At first, a DNA encoding the corresponding fusion protein was prepared and inserted into a suitable expression vector.    The vector is then introduced into a suitable host cell.    The transformed host cell is cultured to express the DNA, thereby producing the fusion protein.    The fusion protein is then enzymatically cleaved with thrombin and carboxypeptidase B.    Thus, insulin having the desired primary structure and biological activity, which are identical with those of the naturally occurring insulin, can be produced.<br>
The present invention will be illustrated in more detail hereinafter.<br>
The leader peptide (Y), which comprises at least one amino acid residue required for the expression of a protein of interest, includes the known MBP (Maina, C.V. et al., Gene 74:365-373, 1988), GST (Smith, D.B., et al., Gene 67:31-40,1988), TRX (LaVallie, E.R. et al., Bio/Technology 11:187-193,1993), DsbA (Collins-Racie, L.A. et al., Bio/Technology 13:982-987, 1995) and LamB (Benson, S.A. et al., Cell 32:1325-1335,1983), which are derived from E. coli; and a factor derived from yeast (Brake, A.J., Yeast Genetic Engineering, p269-280, 1989).    In particular, the leader sequence is often required, for E. coli, when a protein of interest is secreted into the periplasm or, for yeast, when the protein is secreted into the culture medium.   According to an embodiment of the present invention, a preferred leader peptide comprises the N-terminal 9 amino acids of a CWP from a bacterium belonging to the genus Bacillus (hereinafter, sometimes referred to as "Bacillus bacterium").    The CWP useful for the present invention includes, but is not limited to, those derived from Bacillus brevis strain 47 (PERM P-7224; JP-A- 60-58074 and JP-A-62-201589), strain HPD 31 (PERM BP-1087; JP-A- 4-278091).    Specifically, the following sequences may be used (where the brackets show references).<br>
MWPmp9: AlaGluGluAlaAlaThrThrThrAla<br>
(SEQ ID NO:4; J. Bacteriol., 169:1239-1245, 1989)<br>
OWPmp9: AlaProLysAspGlylleTyrlleGly<br>
(SEQIDNO:5;J. Bacteriol., 170:176-186, 1988)<br>
HWPmp9: AlaGluAspThrThrThrAlaProLys<br>
(SEQ.IDNO:6; J. Bacteriol., 172:1312-1320, 1990)<br>
The number of amino acid residues from the N-terminus of the CWP is not necessary to be 9, as long as the fusion protein can be expressed.    For example, a sequence comprising N-terminal 1-50 amino acid residues of the CWP from a Bacillus bacterium may be used.    The leader peptide is not always be needed if the fusion protein portion behind insulin B chain of the fusion protein, namely [B-chain]-[X2]-[Linker]-[X3]-[A-chain], is linked with the 3' end of a DNA containing the promoter region of an expression system to enable the expression of the fusion protein portion. When the fusion protein contains a leader peptide derived from a CWP, it is preferred that the leader peptide is linked to a CWP (particularly MWP) signal peptide at its 5' end.   The information regarding MWP sequences can be found in Yamagata, H. et al., J. Bacteriol., 169:1239-1245, 1987 or Tsuboi, A. et al., J. Bacteriol., 170:935-945, 1988, and one can refer to them.    The signal peptide generally helps to direct an expressed and translated protein to a cell membrane and to secrete the protein extracellularly.   A secreted protein is advantageous because it can be more readily isolated and purified compared to non-secreted protein.<br>
For the fusion protein, the enzymatic cleavage of XI includes the use of an enzyme of which no cleavage site is contained in insulin B chain or insulin A chain, such as factor Xa, tlirombin and enterokinase.   When a Gly or Ser residue is attached to the N-terminus of insulin B chain, if such a residue has no influence to the activity of the resulting insulin, then TEV protease may be used.    On the other hand, the chemical cleavage of the XI may include a selective cleavage at the C-terminus of methionine (J. Biol. Chem., 237:1856-1860,1962) and a selective cleavage at the C-terminus of tryptophan (Methods in Enzymol., 91:318-324, 1983).   According to a preferred embodiemnt of the present invention, all of XI to X3 may be cleaved simultaneously, in which the enzyme used is thrombin and the amino acid sequences of XI to X3 are as follows: XI = GlySerLeuGlnProArg (SEQ ID NO:1); X2 = ArgGlyHisArgPro (SEQ ID NO:2); and X3 = ProArg. In this case, XI, X2 and X3 with other amino acid sequences may also be employed, provided that the intended cleavage with thrombin can be achieved.    For example, in the above amino acid<br>
sequences, the following substitutions are possible for such cleavage: for XI and X3, Ser = Val, Glu, Phe, Asp, Pro, lieu, Gly, Lys, Arg, Ala, Gin, Asn or Leu, Leu = Arg, Val, Phe, Asp, Gly, Leu, His, lieu, Met, Thr or Lys, Gin = Gin, Phe, Tyr, Gly, lieu, Asn, Ala, Arg, Thr, Ser, Leu, Val or Cys, Pro = Ala or Val, and Arg = Lys (Chang, J-Y, Eur. J. Biochem., 151:217-224, 1985; Kawabata, S. et al., Eur. J. Biochem., 172:17-25, 1988); and for X2, Arg = Lys, Gly = Thr, lieu, His, Ser, Ala, Phe, Val, Asn, Asp, Leu or Pro, His = Pro, Tip, Cys, Gin, Thr, Ser, Val, Leu, Ala, Phe or Gly, Arg = Val, Pro, Glu, Asn, Asp, Ser, Met, Lys, Ala, Gin, Gly, Tip or Thr, Pro = Val, Thr, Leu, Ser, Asp, Gly, Tyr, lieu, Asn, Arg, His or Glu (Chang, J-Y, Eur. J. Biochem., 151:217-224, 1985).<br>
The linker comprising at least one amino acid residue is usually present between functional domains in a protein and can joint the domains without no influence on the functions of the domains. In the present invention, the linker is located between insulin B chain and insulin A chain via each enzymatically cleavable sequence, and serves for the formation of disulfide bonds between insulin B chain and insulin A chain and for the readily expression of the fusion protein.    The linker may comprise at least one amino acid residue and any kind of amino acid(s) may be used, provided that it exerts the same function.   According to a preferred embodiment of the present invention, the linker may preferably comprise the C peptide of proinsulin.   According to an embodiment of the present invention, the linker comprises the following sequence:<br>
GluAlaGluAspLeuGlnValGlyGlnValGluLeuGlyGlyGlyProGlyAlaGlySer<br>
LeuGlnProLeuAlaLeuGluGlySerLeuGln	(SEQ ID NO:3).<br>
In the present invention, the DNA encoding the fusion protein is expressed in the form linked with the 3' end of a DNA containing a promoter region for an expression system employed. Examples of the promoter include bacteriophage XpL promoter, T7 promoter, E. coli tip-lac promoter (Maniatis, T. et al., Molecular Cloning 2nd ed., A Laboratory Manual, Cold Spring Harbor Laboratory, 1989), yeast PRBI promoter (BIO/TECHNOLOGY 9:183-187,1991), GAPDH promoter (BIO/ TECHNOLOGY 12:381-384, 1994), viral LTR promoter, SV40 promoter (Maniatis, T. et al., Molecular Cloning 2nd ed., A Laboratory Manual, Cold Spring Harbor Laboratory, 1989). According to an embodiment of the present invention, the fusion protein is linked with the 3' end of a DNA sequence containing a promoter region derived from a Bacillus bacterium.    The available promoter includes, but is not limited to, MWP promoter derived from Bacillus brevis strain 47 (JP-<br>
A- 1-58950 and JP-A-7-108224), HWP promoter derived from Bacillus brevis strain HPD31 (JP-A-4-278091 and JP-A-6-133782).<br>
The DNA of the present invention may be prepared by any combination of the techniques known in the art.   For example, constituent DNA sequences of the DNA may be prepared separately by chemical synthesis methods or cloning methods and ligated in sequence with a ligase, and the resulting DNA sequence may be subjected to polymerase chain reaction (PCR) in combination, thereby producing the intended DNA.    The details of the specific preparation method for the DNA will be appreciated with reference to the examples below.    In the preparation of the DNA, conventional techniques may be employed, such as those described in Maniatis, T. et al., Molecular Cloning 2nd ed., A Laboratory Manual, Cold Spring Harbor Laboratory, 1989; and Innis, M.A. et al., PCR Protocols, A guide to methods and applications, Academic Press, 1990.<br>
The DNA encoding human insulin comprising insulin B chain, C peptide and A chain may be obtained from a commercially available human pancreas-derived mRNA using commercially available cDNAl st-strand synthesis kit (Pharmacia) or the like.    When short-strand DNAs (as primers) can be synthesized based on known DNA sequences using a commercially available DNA synthesizer, the DNA fragments encoding the B chain, the C peptide and the A chain may be amplified by a conventional PCR method.    In this case, the PCR may be performed for 20 cycles or more under the following conditions: DNA denaturation (e.g., at 94°C for 30 sec.-l min.); annealing to the primer (e.g., at about 45-60°C for 30 sec.-l min.); and elongation reaction (e.g., at 72°C for 30 sec. or more).<br>
The present invention also provides a vector containing the DNA.    The vector that can be used in the present invention is needed to have at least the following properties: it has an appropriate insertion site (i.e., a restriction site) to which the DNA of the present invention can be inserted; it can express the DNA in a host cell; and it is autonomously replicable in the host cell.    The vector generally contains a promoter, in which the promoter is operably linked with the upstream of the DNA of interest.    The vector may contain an origin of replication and a terminator sequence, and may also contain a selectable marker such as a drug resistance gene or an auxotrophy-complement gene.    The vector of the present invention is preferably a plasmid which is replicable in a Bacillus<br>
bacterium.    Examples of the plasmid include, but are not limited to, pNU200, pHY500 (Proc. Natl.<br>
Acad. Sci. USA, 86:3589-3593, 1989),pHY4831 (J. Bacteriol., 169:1239-1245, 1987),pNU100 (Appl. Microbiol. Biotechnol., 30:75-80,1989),pNU211 (J. Biochem., 112:488-491, 1992), pNU211R2L5 (Japanese Patent Application Laid-open No. 7-170984), pHY700 (Japanese Patent Application Laid-open No. 4-278091), pHT210 (Japanese Patent Application laid-open No. 6-133782), pHTl 10R2L5 (Appl. Microbiol. Biotechnol., 42:358-363, 1994).   According to an embodiment of the present invention, an expression vector pNU-mPINS can be prepared by the construction process as illustrated in FIG. 3.<br>
The present invention also provides a host cell transformed with the vector as defined above. The host cell may be prokaryotic cells (e.g., bacteria) or prokaryotic cells (e.g., fungi, yeasts, animal cells, plant cells), and preferably a Bacillus bacterium.    Examples of Bacillus bacterium as the host include, but are not limited to, Bacillus brevis strain 47 (PERM P-7224; JP-A- 60-58074 and JP-A-62-201589), strain 47K (JP-A- 2-257876), strain 31 OK (JP-A- 6-296485) and strain HPD31 (PERM BP-1087; JP-A- 4-278091).    The recombinant bacterium Bacillus brevis strain 47-5Q transformed with an expression vector pNU-mPINS has been deposited under the terms of the Budapest Treaty on April 20, 1999 at the National Institute of Bioscience and Human-technology, Agency of Industrial Science and Technology, Japan (1-3, Higashi 1-chome, Tsukuba-shi, Ibaragi-ken, Japan) under the accession No. PERM BP-6706.<br>
The expression vector prepared as mentioned above is then introduced into a competent host cell, preferably a Bacillus bacterium cell.   The bacterium cell is cultured in a suitable culture medium under the conditions which allow the expression of the DNA to produce the recombinant fusion polypeptide of interest extracellularly or intracellularly, preferably extracellularly.   The recombinant fusion polypeptide is then collected and purified.    The introduction of the expression vector into the host cell may be performed by any conventional method such as electroporation (Methods in Enzymol., 217:23-33, 1993).    The purification of the fusion polypeptide may be performed by an appropriate combination of any conventional methods such as solvent extraction, ultrafiltration, ammonium sulfate fractionation, HPLC, gel filtration chromatography, ion exchange chromatography, affinity chromatography, hydrophobic interaction chromatography, electrophoresis and isoelectric focusing.<br>
The fusion polypeptide may be treated with a protease and/or a peptidase which can cleave<br>
the fusion polypeptide enzymatically, such as thrombin and carboxypeptidase B as used in the examples below, resulting in the production of insulin.    As illustrated in FIG. 1, at first, thrombin cleaves between the leader peptide (Y) and the B-chain, between the B-chain and the linker and between the linker and the A chain under suitable conditions.   Preferable conditions for the specific cleavage with thrombin are as follows: pH 7.5-8.5 (preferably with Tris buffer); temperature of 3-6°C, preferably 4°C; the substrate:enzyme ratio = 5:1 to 125:1 (molar ratio), more preferably 25:1; a time period of 1-24 hours.   Next, carboxypeptidase B removes the Arg residue remaining at the C terminus of the B-chain, thus producing insulin (see FIG. 1).   The enzymes may be used in amounts sufficient to cause the cleavage of the fusion protein.<br>
According to the present invention, insulin can be obtained by culturing a transformed Bacillus bacterium prepared as described above to accumulate a fusion protein containing an insulin sequence outside the cell, and then cleaving the collected fusion protein.<br>
The recombinant insulin prepared in this manner has the same amino acid sequence, disulfide bridges and biological activity as those of the naturally occurring insulin and, therefore, is useful as a therapeutic medicament for insulin-dependent diabetes mellitus.<br>
EXAMPLES<br>
The present invention will be illustrated in detail in the following examples.    These examples, however, should not be construed to limit the scope of the invention.<br>
In the preparation of a DNA encoding a fusion protein, a method was employed in which DNA fragments amplified by PCR reaction were ligated by a ligation reaction with a DNA ligase. In the specification, the term "MWPsp" means a MWP-derived signal peptide, the term "MWPmp9" means N-terminal 9 amino acid residues of the mature MWP.<br>
EXAMPLE 1<br>
Construction of Vector (pmPINS) Having MWPsp-MWPmp9-GSLOPR-Bchain-RGHRP-Linker-PR-Achain Fusion DNA Integrated Therein<br>
(1)     Preparation of DNA fragment for M WPsp-M WPmp9<br>
a.	Template DNA:<br>
A genomic DNA extracted from Bacillus brevis strain 47-5 Q by a known method (Molecular Cloning 2nd ed., A Laboratory Manual, Cold Spring Harbor Laboratory, 1989): 840 ng<br>
b.	Primers:<br>
Forward primer: 5'-GTCGTTAACAGTGTATTGCT-3'    (SEQ ID N0:7)<br>
Reverse primer: 5'-AGCTGTAGTAGTTGCTGC-3'    (SEQ ID N0:8)<br>
These primers were chemically synthesized based on the nucleotide sequence of MWP<br>
determined by Yamagata, H. et al. (J. Bacteriol., 169:1239-1245, 1987) and Tsuboi, A. et al. (J.<br>
Bacteriol., 170:935-945, 1988) and added at the final concentration of 0.1 nM.<br>
c.	Taq DNA polymerase:<br>
A commercially available product (GIBCO BRL) (5U) was added.<br>
d.	Other materials:<br>
Tris-HCl (final concentration: 20mM, pH 8), MgCl2 (final concentration: 2.5 mM), dNTPs (dATP, dGTP, dCTP, dTTP; final concentration: 50 uM for each) were added.<br>
Materials "a" to "d" were mixed in a 0.5-ml tube so that the total volume of the reaction solution became 100 \il.    PCR reaction was then performed according to a conventional manner (Innis, M.A. et al., PCR Protocols, A guide to methods and applications, Academic Press, 1990) under the conditions: denaturation temperature: 94°C - 1 min.; annealing temperature: 50°C - 1 min.; DNA chain elongation temperature: 72°C - 1 min.; for 30 cycles.    After the PCR was completed, the reaction solution was concentrated with phenol, and then applied to a 0.8% agarose gel to perform electrophoresis under conventional conditions.    The agarose gel was treated using Ultrafree C3H (Millipore) to collect a PCR product (i.e., a DNA fragment for MWPsp-MWPmp9) therefrom.    The collected PCR product was extracted with phenol, precipitated with ethanol and then dried in vacua.    The dried product was dissolved in an appropriate volume of distilled water.<br>
and then subjected to blunting reaction using DNA blunting kit (Takara Shuzo Co., Ltd.) according to the instructions by the manufacturer.<br>
(2)	Preparation of DNA fragment for proinsulin<br>
A blunt-ended DNA fragment for proinsulin was prepared in the same manner as mentioned in (1), except the following things.<br>
•	As the template DNA, a plasmid vector having human preproinsulin DNA integrated<br>
therein (10 ng) was used.   This recombinant plasmid vector was prepared in the following manner.<br>
Human pancreatic cDNA was synthesized from a commercially available human pancreatic mRNA<br>
(CLONTECH) using 1 st strand cDNA synthesis kit (Pharmacia) according to the instructions by the<br>
manufacturer.    PCR was performed using the cDNA as a template and a forward primer: 5'-<br>
ATGGCCCTGTGGATGCGCC-3' (SEQ ID NO:9) and a reverse primer: 5'-<br>
CTAGTTGCAGTAGTTCTCC-3' (SEQ ID NO: 10) both synthesized based on the nucleotide<br>
sequence of human preproinsulin gene determined by Bell, G.I. et al. (Nature, 282:525-527, 1979)<br>
under the conditions: conditions: 94°C - 1 min.; 60°C - 1 min.; 72°C - 1 min.; for 35 cycles.    The<br>
PCR product, i.e., human preproinsulin DNA, was cloned into pGEM-T vector (Promega).<br>
•	As the primers, a forward primer: 5'-TTTGTGAACCAACACCTG-3' (SEQ ID NO:11)<br>
and a reverse primer: 5'-CTAGTTGCAGTAGTTCTCC-3' (SEQ ID NO: 10) were used.<br>
•	The following PCR conditions were employed: denaturation temperature: 94°C - 1 min.;<br>
annealing temperature: 47°C - 1 min.; DNA chain elongation temperature:    72°C - 30 sec.; for 25<br>
cycles).<br>
(3)	preparation of DNA fragment for GSLQPR-Bchain-R<br>
A blunt-ended DNA fragment for GSLQPR-Bchain-R was prepared in the same manner as mentioned in (1), except the following things.   The resulting DNA fragment was subjected to phosphorylation reaction using T4 polynucleotide kinase (Nippon Gene Co., Ltd.) according to the instructions by the manufacturer to prepare a phosphorylated DNA fragment for GSLQPR-Bchain-R.<br>
•	As the template DNA, the proinsulin PCR product prepared in (2) (10 ng) was used.<br>
•	As the primers, a forward primer: 5'-<br>
GGTTCCTTGCAACCTCGTTTTGTGAACCAACACCTG-3' (SEQ ID NO: 12) and a reverse primer: 5'-GCGGGTCTTGGGTGTGTA-3' (SEQ ID NO: 13) were used.<br>
•	The following PCR conditions were employed: denaturation temperature: 94°C - 1 min.;<br>
annealing temperature: 47°C - 1 min.; DNA chain elongation temperature:    72°C - 30 sec.; for 25<br>
cycles).<br>
(4)	Preparation of DNA fragment for Linker<br>
A blunt-ended DNA fragment for Linker was prepared in the same manner as mentioned in (1), except the following things.<br>
•	As the template DNA, the proinsulin PCR product prepared in (2) (10 ng) was used.<br>
•	As the primers, a forward primer: 5'-GAGGCAGAGGACCTGCAG-3' (SEQ ID NO: 14)<br>
and a reverse primer: 5'-CTGCAGGGACCCCTCCAG-3' (SEQ ID NO:15) were used.<br>
•	The following PCR conditions were employed: denaturation temperature: 94°C - 1 min.;<br>
annealing temperature: 55°C - 1 min.; DNA chain elongation temperature:    72°C - 30 sec.; for 25<br>
cycles).<br>
(5)	Preparation of DNA fragment for GHRP-Linker<br>
A blunt-ended DNA fragment for GHRP-Linker was prepared in the same manner as mentioned in (4), except the following things.   The resulting DNA fragment was subjected to phosphorylation reaction using T4 polynucleotide kinase (Nippon Gene Co., Ltd.) according to the instructions by the manufacturer to prepare a phosphorylated DNA fragment for GHRP-Linker.<br>
•	As the template DNA,, the PCR product (the DNA fragment for Linker) prepared in (4)<br>
(lOng) was used.<br>
•	As the forward primer, a primer: 5'-<br>
GGTCACCGTCCAGAGGCAGAGGACCTGCAGGTGGGG-3' (SEQ ID NO: 16) was used.<br>
•	The following PCR conditions were employed: denaturation temperature: 94°C - 1 min.;<br>
annealing temperature: 55°C - 1 min.; DNA chain elongation temperature:    72°C - 30 sec.; for 25<br>
cycles).<br>
(6)	Preparation of DNA fragment for Achain<br>
A blunt-ended DNA fragment for Achain was prepared in the same manner as mentioned in (1), except the following things.<br>
•	As the template DNA,, the PCR product for proinsulin prepared in (2) (10 ng) was used.<br>
•	As the primers, a forward primer: 5'-GGCATTGTGGAACAATGCTGT-3' (SEQ ID<br>
NO:17) and areverse primer: 5'-CTAGTTGCAGTAGTTCTCCAGCTGGTA-3' (SEQ ID N0:18)<br>
were used.<br>
•	The following PCR conditions were employed: denaturation temperature: 94°C - 1 min.;<br>
annealing temperature: 55°C - 1 min.; DNA chain elongation temperature:    72°C - 30 sec.; for 25<br>
cycles).<br>
(7)	Preparation of DNA fragment for PR-Achain<br>
A blunt-ended DNA fragment for PR-Achain was prepared in the same manner as mentioned in (6), except the following things.    The resulting DNA fragment was subjected to phosphorylation reaction using T4 polynucleotide kinase (Nippon Gene Co., Ltd.) according to the instructions by the manufacturer to prepare a phosphorylated DNA fragment for PR-Achain.<br>
•	As the template DNA,, the PCR product (i.e., the DNA fragment for Achain) prepared in<br>
(6)(10ng) was used.<br>
•	As the forward primer, a primer: 5'-CCACGTGGCATTGTGGAACAATGCTGT-3'<br>
SEQ ID NO: 19) was used.<br>
•	The following PCR conditions were employed: denaturation temperature: 94°C - 1 min.;<br>
annealing temperature: 55°C - 1 min.; DNA chain elongation temperature:    72°C - 30 sec.; for 25<br>
cycles).<br>
(8)	Preparation of fusion DNA for MWPsp-MWPmp9-GSLQPR-Bchain-R<br>
A blunt-ended fusion DNA fragment for MWPsp-MWPmp9-GSLQPR-Bchain-R was prepared in the same manner as mentioned in (1), except the following things.<br>
•	As the template DNA,, a product prepared by mixing appropriate amounts of the DNA<br>
fragment for MWPsp-MWPmp9 prepared in (1) and the DNA fragment for GSLQPR-Bchain-R<br>
prepared in (3) and reacting the mixture using a DNA ligation kit (Takara Shuzo Co., Ltd.) at 16°C for 30 minutes was used.<br>
•	As the reverse primer, a primer: 5'-GCGGGTCTTGGGTGTGTA-3' (SEQ ID NO: 13)<br>
was used.<br>
•	The following PCR conditions were employed: denaturation temperature: 94°C - 1 min.;<br>
annealing temperature: 47°C - 1 min.; DNA chain elongation temperature:   72°C - 30 sec.; for 25<br>
cycles).<br>
The PCR product was phosphorylated using T4 polynucleotide kinase (Nippon Gene Co., Ltd.) according to the instructions by the manufacturer.   The phosphorylated PCR product was digested with Hinc II using a DNA ligation kit (Takara Shuzo Co., Ltd.) and integrated into a vector (STRATAGENE, Blue Script SK-).   E. coli strain DH5a was transformed with the vector according to a known method (Molecular Cloning 2nd ed., A Laboratory Manual, Cold Spring Harbor Laboratory, 1989), and the vector (i.e., the plasmid DNA) was then isolated from the transformant.    The plasmid DNA was sequenced using a forward primer (Ml 3 forward primer) or a reverse primer (Ml 3 reverse primer) to confirm the presence of the fusion DNA for MWPsp-MWPmp9-GSLQPR-Bchain-R.    The second round PCR was performed using a vector having the fusion DNA for MWPsp-MWPmp9-GSLQPR-Bchain-R integrated therein as a template DNA and a forward primer: 5'-GTCGTTAACAGTGTATTGCT-3' (SEQ ID NO:7) and a reverse primer: 5'-GCGGGTCTTGGGTGTGTA-3' (SEQ ID NO: 13) in the same manner as mentioned above, thereby producing a blunt-ended fusion DNA for MWPsp-MWPmp9-GSLQPR-Bchain-R.<br>
(9)      Preparation of fusion DNA for MWPsp-MWPmp9-GSLQPR-Bchain-RGHRP-Linker<br>
A blunt-ended fusion DNA for MWPsp-MWPmp9-GSLQPR-Bchain-RGHRP-Linker was prepared in the same manner as mentioned in (8), except the following things.<br>
•	As the template DNA, for the first round PCR, a product prepared by mixing appropriate<br>
amounts of the fusion DNA for MWPsp-MWPmp9-GSLQPR-Bchain-R prepared in (8) and the<br>
DNA fragment for GHRP-Linker prepared in (5) and then reacting the mixture using a DNA ligation<br>
kit (Takara Shuzo Co., Ltd.) at 16°C for 30 minutes was used.<br>
•	As the reverse primer, a primer: 5'-CTGCAGGGACCCCTCCAG-3' (SEQ ID NO: 15)<br>
was used.<br>
(10) Preparation of a vector having fusion DNA for MWPsp-MWPmp9-GSLQPR-Bchain-RGHRP-Linker-PR-Achain integrated therein<br>
A vector having a fusion DNA for MWPsp-MWPmp9-GSLQPR-Bchain-RGHRP-Linker-PR-Achain integrated therein (pmPINS) was prepared in the same manner as mentioned in (8), except the following things.<br>
•	As the template DNA, for the first round PCR, a product prepared by mixing appropriate<br>
amounts of the fusion DNA for MWPsp-MWPmp9-GSLQPR-Bchain-RGHRP-Linker prepared in<br>
(9) and the DNA fragment for PR-Achain prepared in (7) and then reacting the mixture using a DNA<br>
ligation kit (Takara Shuzo Co., Ltd.) at 16°C for 30 minutes was used.<br>
•	As the reverse primer for the first round PCR, a primer: 5'-<br>
CTAGTTGCAGTAGTTCTCCAGCTGGTA-3' (SEQ ID NO: 18) was used.<br>
•	The following PCR conditions were employed: denaturation temperature: 94°C - 1 min.;<br>
annealing temperature: 50°C - 1 min.; DNA chain elongation temperature:    72°C - 1 min.; for 25<br>
cycles).<br>
EXAMPLE 2 Expression and Secretion of Fusion DNA<br>
(1)	Nucleotide sequence of fusion DNA and amino acid sequence effusion protein encoded by the<br>
fusion DNA<br>
The nucleotide sequence of the fusion DNA prepared in Example 1 and the amino acid sequence of a fusion protein encoded by the fusion DNA are shown in FIG. 2.<br>
(2)	Expression and secretion of fusion DNA<br>
The expression of a fusion protein encoded by the fusion DNA prepared in Example 1 was performed.    The fusion DNA was integrated into an expression vector as illustrated in FIG. 3.<br>
Specifically, the vector pmPINS into which the fusion DNA was integrated was digested<br>
with ApaL I and Hind III.   The digestion product was subjected to electrophoresis on a 0.8% agarose gel, and a gel portion containing the fusion DNA was excised.    Appropriate amounts of the excised fusion DNA and an expression vector for Bacillus brevis (pNU211R2L5; JP-A- 7-170984) which had been digested with ApaL I and Hind III were mixed, and the mixture was reacted using a DNA ligation kit (Takara Shuzo Co., Ltd.) at 16°C for 30 minutes, thereby the fusion DNA was integrated into the expression vector.   Thus, an expression vector pNU-mPINS having the fusion DNA integrated therein was prepared.   Bacillus brevis strain 47-5 (PERM BP-1664) was transformed with the expression vector according to a known method (Methods in Enzymol., 217:23-33,1993), and then plated on T2 agar culture medium [polypeptone (1%), meat extract (0.5%), yeast extract (0.2%), uracil (0.1 mg/ml), glucose (1%), erythromycin (10 fag/ml), agar (1.5%); pH 7] to collect the transformants.<br>
The transformants were cultured in T2 culture medium (a medium having the same composition as T2 agar culture medium except that agar was eliminated) at 37°C for 1 day, and plasmid DNA was isolated therefrom by a known method (Molecular Cloning 2nd ed., A Laboratory Manual, Cold Spring Harbor Laboratory, 1989).    The plasmid DNA was treated with ApaL I and Hind III to confirm the presence of the fusion DNA integrated therein.    With respect to the transformant that had been confirmed the presence of the fusion DNA integrated therein, expression and secretion of a fusion protein encoded by the integrated fusion DNA were performed.    A cell suspension which had been cultured in T2 culture medium at 37°C for 1 days was added to a culture medium [polypeptone (3%), yeast extract (0.4%), glucose (3%), MgSCv 7H2O (0.01%), MnSCv 4H2O (0.001%), erythromycin (10 ug/ml); pH 8] at a ratio of 1/1000 (by volume), and then cultured with shaking at 30°C for 4 days.<br>
After the cultivation, the culture medium was centrifuged at 15,000 rpm for 2 min. to give a culture supernatant.    The culture supernatant was subjected to electrophoretic protein analysis by a known method (Laemmli, U.K., Nature, 227:680-685, 1970).   That is, the culture supernatant (18 ul) was added with buffer 1 [125 mM Tris-HCl (pH 6.8), 20% glycerol, 4% SDS, 10% 2-mercaptoethanol] (2 ul), and the mixed solution was boiled for 5 min.    The mixed solution was added with buffer 2 [250 mM Tris-HCl (pH 6.5), 50% glycerol, 0.5% BPB] (4 ul).    The resulting<br>
mixed solution was electrophoresed on a 15/25% SDS polyacrylamide gel (DAIICHI PURE<br>
CHEMICALS, Co., Ltd.) (electrophoresis buffer: 100 mM Tris, 100 mM Tricine, 0.1% SDS). After the electrophoresis, the gel was subjected to Coomassie staining to determine the presence of the expression and secretion of the fusion protein.   As shown in FIG. 4, for the culture medium of the cells transformed with pNU-mPINS containing the fusion DNA, a band (marked with an arrow) corresponding to the fusion protein was detected (lane 3); whereas for the culture medium of the cells having the vector without the fusion DNA, such a band was not detected (lane 2).<br>
EXAMPLE 3 Conversion into Insulin<br>
(1)    Isolation and purification of fusion protein, M WPmp9-GSLQPR-Bchain-RGHRP-Linker-PR-Achain<br>
Cells transformed with pNU-mPINS was cultured at 37°C for 1 day.   An aliquot (50 ul) of the cell suspension was added to a culture medium [polypeptone (3%), yeast extract (0.4%), glucose (3%), MgSCV 7H2O (0.01%), MnSCv 4H2O (0.001%), erythromycin (10 ug/ml); pH 8]    (50 ml) The mixed culture medium was charged dividedly into six 500-ml conical flasks, and then cultured with shaking at 30°C for 4 days.    The culture medium was centrifuged at 9,000 rpm for 20 min. The supernatant was dialyzed against a buffer [20 mM Na-PO4, 150 mM, pH 8] at 4°C, and then centrifuged at 10,000 rpm for 20 min.    The supernatant was applied to a Ni-chelating column (Pharmacia; 5x10 cm) to elute the fusion protein of interest with the buffer supplemented with 60 rnM imidazole.    The elution fraction was added with EDTA and benzamidine (1 mM each) and kept at 4°C.    Thereafter, urea (final concentration: 1 M) and cystein (final concentration: 1 mg/ml) were further added to the elution fraction.   The mixed solution was adjusted to pH 10.8 with IN NaOH and stirred at the same temperature for 1 hour.   The solution was dialyzed against a buffer [20 mM Tris, 1 mM EDTA; pH 8.0].   The dialysate was added with urea (final concentration: 1 M) and 2-propanol (final concentration: 20%) and then applied to Q-Sepharose XL column (Pharmacia; 1.6 x 10 cm).    The column was sufficiently equilibrated with a buffer [20 mM Tris, 1 mM EDTA, 1 M urea, 20% 2-propanol; pH 8] and then eluted with the buffer supplemented under gradient with 1 M NaCl.    In FIG. 5, the elution pattern is shown.    The elution fractions eluted with 160 mM - 200<br>
mM NaCl (shown by the arrow) were combined, and adjusted to pH 3 with IN HC1.    The solution<br><br>
was concentrated using an ultrafilter (fractionation molecular weight: 3,000), and then applied to Vydac214TP54 (CYPRESS; C4 column, 4.6 x 250 mm) for purification by HPLC.    The column was equilibrated with 25% acetonitrile and 0.1% TFE solution, and then eluted under gradient with 33% acetonitrile and 0.1 % TFE solution.    In FIG. 6, the elution pattern is shown.    The fractions eluted with 30-31% acetonitrile (shown by the arrow) were centrifuged and concentrated to dryness. The resulting product was used in the subsequent cleavage experiment.<br>
(2)     Conversion into insulin and purification thereof<br>
The fusion protein prepared in (1) as a dry product, MWPmp9-GSLQPR-Bchain-RGHRP-Linker-PR-Achain, was dissolved in a appropriate amount of 0.1% TFA, and then added with 0.1M Tris buffer (pH 8) to the final concentration of 20 nmol/ml.    The resulting solution was cooled to 4°C, and then added with a thrombin solution (250 nmol/ml) at the substrate:enzyme ratio of 25:1 (molar base).    After 9 hours, a appropriate amount of 10% TFA was added to the reaction solution to adjust to pH 2, thereby terminating the reaction.    The thrombin used was thrombin of JP grade (ITOHAM FOODS INC.) which had been re-purified using Macro Prep CM (Bio Rad) and Lysine Sepharose 4B (Pharmacia).<br>
To isolate insulin-Arg, which had an Arg residue at the C terminus of the B chain cleaved with thrombin, by reversed phase HPLC, the reaction-terminated solution after the thrombin treatment was applied to Mightysil RP4 column (Cica-MERCK; 20 x 250 mm), the column was equilibrated with 25% acetonitrile and 0.1% TFA solution and then eluted under gradient with 35% acetonitrile and 0.1% TFA solution.    The fractions eluted with 30-31% acetonitrile were centrifuged and concentrated to dryness.   The resulting dry product was used in the subsequent experiment.<br>
The dry product insulin-Arg was dissolved in a appropriate amount of 0.1% TFA and then added with 0.1M Tris buffer (pH 8) to the final concentration of 1 mg/ml.   The resulting solution was added with a carboxypeptidase B solution (Sigma; 4.7 mg/ml) at the substrate:enzyme ratio   of 500:1 (molar base), and the reaction solution was allowed to react at 25°C for 12 hours.    The reaction solution was added with a appropriate amount of 10% TFA to adjust to pH 2, thereby terminating the reaction.    To isolate insulin from the reaction-terminated solution, the reversed<br><br>
phase HPLC was performed in the same manner as mentioned for the isolation of the insulin-Arg.<br>
(3)    Amino acid analysis of insulin<br>
At first, the total amino acid of the insulin was analyzed.    The insulin obtained in (2) (about 2 nmoles) was added with 6N HC1 (200 p.!) and 5% phenol (20 uJ).   The resulting reaction solution was deaerated and the tube containing the reaction solution was sealed.    The reaction solution was allowed to react at 110°C for 24 hours to cause hydrolysis reaction, and then dried. The resulting dry product was dissolved in 0.0IN HC1 (100 jal) and filtrated on a 0.2-u.m filter.   An aliquot (50 u.1) of the filtrate was analyzed using Hitachi amino acid analyzer Model L-8500 (HITACHI, Ltd.).<br>
Next, the cysteic acid content of the insulin was analyzed.   The insulin obtained in (2) (about 2 nmoles) was dissolved in a formic acid/methanol (5:1) mixed solution (40 u,l), and cooled to -20°C.   The cooled solution was added with a 99% formic acid/30% aqueous hydrogen peroxide (19:1) mixed solution (400 u.1) which had been cooled to -20°C, and then allowed to react at -20°C for 4 hours.   After the reaction was completed, distilled water (3 ml) was added to the reaction solution and lyophilized.   The resulting dry product was hydrolyzed in the same manner as mentioned above, and then analyzed.<br>
The analytical values for Val determined in the total amino acid analysis and the cysteic acid analysis were compared, and the analytical value for cysteic acid was converted into the analytical value for cysteine of the total amino acid analysis.    As shown in Table 1, the amino acid ratio of the insulin of the present invention was almost consistent with that of the naturally occurring insulin.<br>
Table 1<br><br>
(Table Removed)<br>
 (4)    Peptide mapping of insulin<br>
The insulin obtained in (2) (hereinafter, referred to as "ITOHAM insulin") and a commercially available insulin, Novolin 40, (Novo Nordisk Pharma)(hereinafter, referred to as "Novolin") (5 nmoles each) were separately dissolved in 0.1 M ammonium hydrogen carbonate (50 u.1) and 2 mM EDTA solution (pH 7.8; 50 ul).    The resulting solution was added with an aqueous V8 protease solution (Wako Pure Chemical Industries, Ltd.; 2 ^ig/rnl) (1.35 ul).    The reaction solution was allowed to react at 25°C for 24 hours, and 1% TFA solution was added thereto to adjust to pH 2, thereby terminating the reaction.   The reaction-terminated solution was applied to Vydac218TP54 column (4.6 x 250 mm; CIS column), equilibrated with 5% acetonitrile and 0.1% TFA solution, and then eluted under gradient with 35% acetonitrile and 0.1% TFA solution.    In FIG. 7, the elution pattern is shown.    ITOHAM insulin and Novolin showed similar elution patterns to each other.    Therefore, it is concluded that the both kinds of insulin have the similar disulfide bridging forms.<br>
EXAMPLE 4 Biological Activity of Insulin<br>
Novolin (1.2 ml) was treated with the same manner as in Example 3    (2) using Wakocil-II 5C18 AR Prep (Wako Pure Chemical Industries, Ltd.; 20 x 250 mm) to give insulin fractions.   The insulin fractions of ITOHAM insulin obtained in Example 3 (2) and the insulin fractions of Novolin were analyzed using Vydac218TP54 (4.6 x 250 mm; C18 column).    For each insulin sample, an aliquot was taken from the fractions so that the insulin contents for both kinds of insulin calculated based on the main peak areas became same, and then dried.    As shown in FIG. 8, the elution pattern of Novolin had sub-peaks which were assumed to indicate the presence of polymeric materials. The sub-peak level (i.e., total sub-peak area) of Novolin was 1.23 times greater than that of IHOHAM insulin.<br>
ITOHAM insulin and Novolin thus obtained were separately dissolved in a solution containing 0.1% BSA, 0.9% NaCl and 0.1% phenol solution at the final concentration of 1 unit/ml (calculated based on the assumption that each insulin was 26 units/mg).    The resulting solution (0.5 ml) was subcutaneously injected to the back of a Japanese Wister rabbit_(Kbs:JW, 20.-2.5 kg). After the injection, blood was collected from the anterior auricular vein of the rabbit over a period of time.   Each blood sample (0.45 ml) was added with a mix of glycolysis-inhibiting agents (NaF: 12.5 mg/ml, heparin-Na: 125 units/ml, EDTA-2Na: 48mg/ml) (0.05 ml) and fully mixed.   The mixed solution was centrifuged at 3,000 rpm for 15 min. at 5°C, and the supernatant was used as a plasma sample.    For each plasma sample, the plasma glucose level was determined using a biochemical automated analyzer (CIBA-CORNING; Express PLUS), and the plasma insulin level was determined using EIA kit (Wako Pure Chemical Industries, Ltd.).    The time course of the plasma glucose level and the time course of the plasma insulin level are shown in FIG. 9 and FIG. 10, respectively.    Both ITOHAM insulin and Novolin showed the decrease in plasma glucose level after the injection of insulin, demonstrating that both kinds of insulin acted to lower blood sugar. With respect to the plasma insulin level, both kinds of insulin showed the similar courses.    The slight lower plasma glucose level and the slight higher plasma insulin level of Novolin than those of<br>
ITOHAM insulin are considered to be resulted from the increment of the sub-peaks in Novolin<br>
which were confirmed by the analysis by HPLC above.<br>
As mentioned above, according to the present invention, a novel fusion protein convertible into insulin can be expressed and secreted in a high level in a Bacillus expression system.    By treating the fusion protein of the present invention with thrpmbin and carboxypeptidase B, insulin having the same amino acid composition and biological activity as naturally occurring insulin can be obtained.<br><br><br><br><br><br><br>
We claim:<br>
1.	A method for producing insulin, wherein the method comprises:<br>
(1)	culturing a microorganism host cell, wherein the host cell has been transformed<br>
with a vector comprising a DNA encoding a fusion protein of formula (I):<br>
[Yl-[Xl]-[B-chain]-[X2]-[Linker]-[X3]-[A-chain]    (I)<br>
wherein<br>
Y is a leader peptide sequence for expression and secretion of the protein, comprising the N-terminal 9 amino acid residues of the protein MWP which is one of the cell wall proteins (CWPs) from a bacterium belonging to the genus Bacillus;<br>
XI is an amino acid sequence which is cleavable with thrombin;<br>
B-chain is the amino acid sequence of insulin B chain;<br>
X2 is an amino acid sequence which is cleavable with thrombin;<br>
Linker is a linker sequence comprising at least one amino acid residue;<br>
X3 is an amino acid sequence which is cleavable with thrombin; and<br>
A-chain is the amino acid sequence of insulin A chain, and<br>
wherein the Y, XI, B-chain, X2, Linker, X3 and A-chain are ligated in the order indicated in formula (I), and wherein a CWP signal peptide DNA is attached at the 5' end of the DNA, in a culture medium, to express the fusion protein encoded by the DNA in the host cell;<br>
(2)	collecting the fusion protein; and<br>
(3)	subjecting the fusion protein to an enzymatic cleavage treatment using thrombin<br>
and carboxypeptidase B, to isolate insulin.<br>
2.	The method as claimed in claim 1, wherein the amino acid sequences XI, X2 and X3,<br>
which are used for enzymatic cleavage of a fusion protein, consist of the following sequences:<br>
XI = GlySerLeuGlnProArg (SEQ ID NO: 1): X2 = ArgGlyHisArgPro (SEQ ID NO: 2); and X3 = ProArg.<br>
3.	The method as claimed in claim 1, wherein the linker sequence comprises the<br>
following amino acid sequence:<br>
GluAlaGluAspLeuGlnValGlyGlnValGluLeuGlyGlyGlyProGlyAlaGlySer LeuGlnProLeuAlaLeuGluGlySerLeuGln (SEQ ID NO:3)<br>
4.	The method as claimed in claim 1, wherein the DNA comprises a nucleotide sequence<br>
encoding the amino acid sequence shown in SEQ ID NO:21.<br>
5.	The method as claimed in claim 4, wherein the DNA comprises the nucleotide<br>
sequence shown in SEQ ID NO:20.<br>
6.	The method as claimed in claim 1, wherein the vector comprises a promoter for<br>
expression of a recombinant protein in a prokaryote.<br>
7.	The method as claimed in claim 6, wherein the promoter is from a bacterium belonging<br>
to the genus Bacillus.<br>
8.	The method as claimed in claim 7, wherein the promoter is a CWP promoter from a<br>
bacterium belonging to the genus Bacillus.<br>
9.	The method as claimed in claim 1, wherein the host cell is a bacterium.<br>
10.	The method as claimed in claim 1, wherein the bacterium belongs to the genus<br>
Bacillus.<br>
11.	The method as claimed in claim 10, wherein the bacterium is Bacillus brevis.<br>
12.	The method as claimed in claim 1, wherein the expressed fusion protein is separated<br>
and purified from the host cell or from the cultured medium.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
		<br>
		<div class="pull-left">
			<a href="222567-method-of-hot-rolling-thin-steel-strip-produced-by-direct-casting.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="222569-1r-2s-4r-2-2-n-n-dimethylamino-2-phenyl-1-7-7-tri-methyl-bicyclo-2-2-1-heptane-and-pharmaceutically-acceptable-acid-addition-salts-thereof.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>222568</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/01035/DEL</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>36/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>05-Sep-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>18-Aug-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>08-Nov-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>ITOHAM FOODS INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>657-0037, 2-1, BINGO-CHO 3-CHOME, NADA-KU, KOBE-SHI, HYOGO 657-0037, JAPAN.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>OKA, SHUSAKU</td>
											<td>C/O CENTRAL RESEARCH INSTITUTE, IOHAM FOODS INC. 1-2, KABOGAOKA, MORIYA-MACHI, KITASOMA-GUN, IBARAKI 302/0104, JAPAN.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>HIGASHIKUNI, NAOHIKO</td>
											<td>C/O ENTRAL RESEARCH INSTITUTE, ITOHAM FOODS INC. 1-2, KUBOGAOKA, MORIYA-MACHI, KITASOMA-GUN, IBARAKI 302-0104, JAPAN.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>KUDO, TOSHIYAKI</td>
											<td>C/O ENTRAL RESEARCH INSTITUTE, ITOHAM FOODS INC. 1-2, KUBOGAOKA, MORIYA-MACHI, KITASOMA-GUN, IBARAKI 302-0104, JAPAN.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>SATO, SEIJI</td>
											<td>C/O ENTRAL RESEARCH INSTITUTE, ITOHAM FOODS INC. 1-2, KUBOGAOKA, MORIYA-MACHI, KITASOMA-GUN, IBARAKI 302-0104, JAPAN.</td>
										</tr>
										<tr>
											<td>5</td>
											<td>KONDO, MASAKI</td>
											<td>C/O ENTRAL RESEARCH INSTITUTE, ITOHAM FOODS INC. 1-2, KUBOGAOKA, MORIYA-MACHI, KITASOMA-GUN, IBARAKI 302-0104, JAPAN.</td>
										</tr>
										<tr>
											<td>6</td>
											<td>WAKI, YOSHIHIRO</td>
											<td>C/O ENTRAL RESEARCH INSTITUTE, ITOHAM FOODS INC. 1-2, KUBOGAOKA, MORIYA-MACHI, KITASOMA-GUN, IBARAKI 302-0104, JAPAN.</td>
										</tr>
										<tr>
											<td>7</td>
											<td>YUKI, HIROTAKI</td>
											<td>C/O ENTRAL RESEARCH INSTITUTE, ITOHAM FOODS INC. 1-2, KUBOGAOKA, MORIYA-MACHI, KITASOMA-GUN, IBARAKI 302-0104, JAPAN.</td>
										</tr>
										<tr>
											<td>8</td>
											<td>WATANABE, SHIGEAKI</td>
											<td>C/O ENTRAL RESEARCH INSTITUTE, ITOHAM FOODS INC. 1-2, KUBOGAOKA, MORIYA-MACHI, KITASOMA-GUN, IBARAKI 302-0104, JAPAN.</td>
										</tr>
										<tr>
											<td>9</td>
											<td>HIRAKI, YUSUKE</td>
											<td>C/O ENTRAL RESEARCH INSTITUTE, ITOHAM FOODS INC. 1-2, KUBOGAOKA, MORIYA-MACHI, KITASOMA-GUN, IBARAKI 302-0104, JAPAN.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/17</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/JP00/02736</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-04-26</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>11/124877</td>
									<td>1999-04-30</td>
								    <td>Japan</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/222568-process-for-producing-recombinant-insulin-from-novel-fusion-proteins by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:15:37 GMT -->
</html>
