<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/222791-a-method-for-measuring-dna-polymerization-and-applications-of-the-method by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:40:09 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 222791:A METHOD FOR MEASURING DNA POLYMERIZATION AND APPLICATIONS OF THE METHOD</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR MEASURING DNA POLYMERIZATION AND APPLICATIONS OF THE METHOD</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method for measuring DNA-dependent DNA polymerisation in a biological sample, is described. The method comprises the steps of providing a primer with a single stranded short specific sequence, which is unable to base pair internally, bound to a solid phase; contacting the primer construct with a reaction mixture containing a single stranded deoxynucleotide template with a part of the sequence complementary to the primer and the four deoxynucleoside triphosphates, one of which is modified so that it is specifically recognized by a labeled antibody; adding a biological sample comprising the DNA polymerase, such as retrovirus reverse transcriptase (RT), to the mixture; allowing the polymerase reaction to proceed; incubating the immobilized reaction product with the labeled antibody; detecting the amount of bound labeled antibody; and measuring the amount of incorporated modified deoxynucleoside triphosphate, as a measure of the DNA polymerisation, which may be used drug susceptibility testing. A commercial package is also disclosed.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>       FORM 2 <br>
THE PATENTS ACT 1970  <br><br>
[39 OF 1970]<br>
COMPLETE SPECIFICATION<br>
[See Section 10 ; rule 13]]<br>
"A    METHOD     FOR    MEASURING     DNA     POLYMERIZATION    AND APPLICATIONS OF THE METHOD"<br>
CAVIDI TECH AB, a Swedish corporation of Uppsala Science Park, SE-751 83 Uppsala, Sweden,<br>
The following specification particularly describes the nature of the invention and the manner in which it is to be performed:-<br><br>
WO 02/103039	PCT/SE02/01155<br>
1<br>
method for measuring DNA polymerization and applications of the method.<br>
The present invention relates to a method for measuring DNA polymerization and applications of the method. More precisely, the invention relates to a method for measuring <br>
5     DNA-dependent DNA polymerization. Background<br>
The latest decades have experienced a rapid development of new methods for measurement of reverse transcription, RNA dependent DNA polymerization. The more complex issue concerning methods for quantification of DNA dependent DNA <br>
10     	polymerization has so far received much less attention.<br>
The classic DNA polymerase activity assays involve use of DNAse treated DNA ("activated DNA") as primer / template and incorporation of radiolabeled nucleotides into DNA (Aposhian and Kornberg 1962). Measurement of acid preciptable radioactivity allows calculation of the amount of nucleotides incorporated and the number of enzyme units <br>
15     present. However, use of radioactivity is currently restricted and discouraged in many<br>
laboratories and there is due to this a general trend away from radioactivity-based techniques.<br>
For DNA polymerases a commercial assay based on ELISA detection of<br>
digoxigenine labeled nucleotides incorporated in newly made DNA is available (Roche<br>
Molecular Biochemicals Cat. no 1468120, US5635350). This assay is hampered by the use of<br>
20         two different nucleotide substrate analogues with bulky groups, digoxigenine as label and<br>
            biotin for product immobilization. As a result the polymerization reaction velocity and<br>
             subsequent detection sensitivity is reduced. The utilization of substrate analogues with highly<br>
              deviating kinetic properties makes this system less relevant for studies of drug susceptibility<br>
              of different polymerases.<br>
25	Another more attractive alternative method is a fluorescence-based assay for<br>
DNA polymerase holoenzyme, based on the specific reaction of the dye PicoGreen with double-stranded DNA (Seville et all 1996). The latter process has recently been modified to make it suitable for a broader range of different DNA polymerizing enzymes (Tveit and Kristensen 2001). This assay is technically simple and based on the utilization of natural <br>
30     nucleotides. The detection sensitivity is, however, still in the same range as the classic<br>
radioactive DNA polymerase assay and the applications described demonstrates a detection range of 0.05-0.5 U DNA polymerase / sample.<br><br>
WO 02/103039	PCT/SE02/01155<br>
2<br>
HIV therapy today is based on multidrug therapy. The regimens are based on combinations of all three types of drugs available: nucleoside analogues, non-nucleoside analogues and protease inhibitors. The strategy is to minimize the probability for a mutant virus to survive.<br>
The reverse transcriptase (RT) inhibitors are either nucleoside analogues or non-nucleoside analogues. The non-nucleoside inhibitors bind to a hydrophobic pocket in the RT enzyme close to, but not contiguous with,, the active site. HTV-1 replication is inhibited allosterically by displacing the catalytic aspartate residues relative to the polymerase binding site.<br>
The nucleoside inhibitors used today terminates the DNA chain elongation as they lack a 3'-hydroxyl group. Prolonged therapy with nucleoside inhibitors commonly leads to the development of resistant virus. This process is associated with the gradual appearance of mutations in the virus pol gene, each leading to defined amino acid substitutions (for a review see Vandamme et al 1998). The effects of these substitutions at the enzymatic levels is complicated and includes enhancement of a primitive DNA editing function. This reaction is nucleotide dependent and produces dinucleoside polyphosphate and an extendible DNA 3'end .(Arion et al 1998, Meyer et al 1999).<br>
The HTV-l RT as well as other reverse transcriptases perform three different enzymatic reactions: RNA-dependent DNA polymerization, DNA-dependent DNA polymerization, and degradation of RNA in the DNA-RNA hybrid (RNase H). The HTV • reverse transcriptase, encoded by the pol gene, is a heterodimer consisting of a p66 and a p51 subunit. Both RNA-dependent DNA polymerization and DNA-dependent DNA polymerization are performed by the same active site localized in the p66 subunit (for a review see Goff 1990). The reaction mechanism of these drugs has mainly been defined according to their action on the RNA-dependent DNA polymerization reaction. The effect on the DNA-dependent DNA polymerization reaction is comparatively less studied.<br>
Provided that the reaction mechanism and the active metabolized drug is known and available, phenotypic virus drug susceptibility could be determined at the enzyme level. Depending on the capacity of the enzyme assays and the virus isolation techniques used, the drug sensitivity testing can theoretically be done either on supematants from virus culture propagation, the primary virus isolation or on virus preparations recovered directly from the patients. Conventional RT activity assay is performed by utilizing an artificial template-primer construction and labeled deoxynucleoside triphosphate as nucleotide substrate. The template / primer pair poly(rA-) /oligo(dT) is the most efficient and most used combination for<br><br>
WO 02/103039	PCT/SE02/01155<br>
3<br>
..determination of HTV as well as for other retroviral RTs. A drawback of this type of assay ""when drug.sensitivity testing is concerned, is that only non-nucleoside analogues or analogues that can base pair with rA can be tested. Analogues to the other nucleotide bases will require an assay based on a variable polymer template. RNA polymers containing pyrimidine bases are notoriously sensitive to RNases and in practice not compatible with biological samples. It would therefore be advantageous to base a polymerase assay intended for drag sensivity testing on a variable DNA template, provided that the assay system gives results that correlate with those from inhibition of reverse transcription and classic phenotypic drug resistance tests.<br>
HTV therapy used today is only one example of the potency of DNA polymerase inhibitors. The current situation concerning resistance development among bacteria and other microorganisms motivates the search for new classes of antimicrobial drugs. DNA polymerases are one. of the major targets during this effort. As such there is a great demand for technically simple polymerase assays, which do not cause potential environmental hazards and can be applied for drug screening towards a broad range of microbial DNA polymerase isozymes. The toxicity of the drug leads found must further be evaluated towards the corresponding mammalian DNA polymerases.<br>
Quantification of proliferation associated polymerases such as polymerase -a and -5 can be used for monitoring cell proliferation. It may be mentioned in this context that the serum levels of thymidine kinase, another cell proliferation associated enzyme, are currently used for prognosis and classification of malignant disease (US4637977). Phosphorylation of thymidine is just one of the two intracellular synthetic pathways, which provides thymidine triphosphate for DNA synthesis. Measurement of the DNA polymerase itself has the potential to give a more correct estimation of total DNA synthesis compared to thymidine kinase activity or thymidine incorporation.<br>
Description of the invention<br>
The present invention provides a non-radioactive DNA polymerase assay in microtiter plate format enabling colorimetric or fluorimetric product detection.<br>
In a preferred embodiment it utilizes, as nucleoside triphosphate substrate, 5-bromodeoxyuridine 5'-triphosphate (BrdUTP). The difference in Van der Waals' radius between the 5' position bromine in BrdUTP and the 5' position methyl group in thymidine triphosphate is minimal (1.95 A compared to 2.0 A) and the enzyme kinetic properties of<br><br>
WO 02/103039	PCT/SE02/0M55<br>
4<br>
! these two nucleotides are quite similar. The method can be aromatized and have a detection range down to 3nU polymerase activity /sample.<br>
One of the applications of the present invention is drug susceptibility testing. All anti-retroviral drugs approved hitherto interfere with the enzymatic reaction of either the viral<br>
5     	protease or the RT. There are in addition candidate drugs in the pipeline which affect the function of the retroviral integrase.<br>
In particular, the presentinvention provides a procedure to measure a broad range of different DNA-dependent DNA polymerases. It has proved suitable even for studies of highly processive DNA polymerase systems in spite of the comparatively short template used. The <br>
10     	usefulness for determination of the activity of bacterial polymerase I and m, mammalian DNA polymerase a, p and y, a proliferation associated polymerase activity in human serum and DNA dependent DNA polymerization by HTV RT is demonstrated, but the method can be used for studies of virtually all viral and cellular DNA polymerases. One of the features that distinguish the present DNA polymerase assay from previous art is its outstanding sensitivity, <br>
15     	which makes detection of down to 3nU E. coli DNA polymerase I activity feasible.<br>
Thus, one aspect of the invention is directed to a method for measuring DNA dependent DNA polymerisation in a biological sample, comprising the steps of<br>
		a)  providing a primer with a single stranded short specific sequence, which<br>
         is unable to base pair internally, bound to a solid phase,<br>
20	b)  contacting the primer construct with a reaction mixture containing a<br>
	single stranded deoxy nucleotide template with a part of the sequence<br>
complementary to the primer and the four deoxy nucleoside<br>
triphosphates, one of which is modified so that it is specifically<br>
recognized by a labeled antibody<br>
25	c)   adding a biological sample comprising the DNA polymerase to the<br>
mixture of b),<br>
d)	allowing the polymerase reaction to proceed,<br>
e)	incubating the immobilized reaction product resulting from d) with the labeled antibody,<br>
30	f)   detecting the amount of bound labeled antibody with the aid of the label<br>
used, and <br>
g)  measuring the amount of incorporated modified deoxynucleotide, as a measure of the DNA polymerization, with the aid of the label of the bound antibody.<br><br>
WO 02/103039	PCT/SE02/01155<br>
In an embodiment the DNA polymerization is made by a retrovirus reverse transcriptase (RT), such as human immunodeficiency virus (HTV) RT.<br>
In another embodiment the modified deoxy nucleoside triphosphate is <br>
5        -5     	bromodeoxyuridine 5'-triphosphate (BrdUTP) and the labeled antibody is an alkaline phosphatase (Ap) conjugated anti-BrdU monoclonal antibody.<br>
In a preferred embodiment of the method accordmg to the invention, the measured DNA polymerisation is used for drug susceptibility testing.<br>
The drug susceptibility testing is performed to evaluate if a certain drug is <br>
10     	effective in a mammalian individual, and the result may be used for selecting drug treatment therapy for that individual. In practice, the individual will be subjected to testing at several points of time to monitor the development of drug treatment in said individual.<br>
The invention is also directed to a commercial package comprising written and/or data carrier instructions for measuring DNA-dependent DNA polymerisation according the. <br>
15     invention. The package will comprise at least the following items:<br>
a)	a primer with a single stranded short specific sequence, which is unable to base pair internally, bound to a solid phase,<br>
b)	a single stranded deoxynucleotide template with a part of the sequence complementary to the primer in a),<br>
20 	c)  the four deoxynucleoside triphosphates, one of which is modified so that<br>
it is specifically recognized by a labeled antibody, and d)  the labelled antibody that recognizes the modified deoxynucleoside triphosphate in c). The invention will now be illustrated by the following unlimiting description of <br>
25     embodiments and drawings of the invention.<br>
The teachings of the cited literature is incorporated herein by reference.<br>
Short descriptions of the drawings.<br>
Figure 1. demonstrates the effects of variation in template sequence on 30     inhibition of DNA polymerase III with TMAU.<br>
Figure 2. exemplifies the detection sensitivity of the DNA polymerase assay. HTV-l wild type RT (♦), mammalian DNA polymerase {3 (•) and E. coli DNA polymerase I<br><br>
WO 02/] 03039	PCT/SE02/01155<br>
6<br>
Figure 3. demonstrates the ability of the DNA polymerase assay to measure inhibition by the dideoxy analogues to all the four DNA bases. Symbols: ddATP (■), ddGT («*), ddCTP (•) and ddTP (A).<br>
Figure 4. shows that the biochemical mechanism underlying resistance to the <br>
5         antiviral drug tenofovir is based on enhancement of an ATP dependent phosphoroiysis  <br>
reaction. Symbols: HTV-1 wild type RT in standard reaction solution (O). HTV-1 wild type RT in reaction solution with ATP (•), HTV-1 mutant RT in standard reaction solution (O), HTV-1 mutant RT in reaction solution with ATP( ♦).<br>
Figure 5. exemplifies determination of susceptibility to the antiviral drug <br>
10      Nevirapine using RTs isolated from plasma from HTV infected individuals. Symbols: (□), (■) <br>
and (A) RTs from infected individuals (•) a control consisting of recombinant HTV-1 wild type RT, (♦) a control consisting of mutated (LI 001) recombinant HIV-1 RT with intermediate Nevirapine resistance.<br>
15     Description of embodiments<br>
Production of primer coated microliter plates<br>
l-ethyl-3-(3-dimetylamino-propyl)carboimide hydrochloride (final concentration 10 mg / ml) was added to a 100 mM l-Methylimidazole buffer (pH 7.0) and the mixture was used to dilute the primer construct to a final concentration of 1 ug /ml. 100 ul of the primer solution was<br>
20     aliquoted to each well of a microtiter plate consisting of Nalge Nunc<br>
NucleoLink® transparent strips (Cat no 248259). The plates were incubated 6-8 hours at 37 °C, washed thoroughly in 2M NaOH with 2 mM Ethylenediaminetetraacetic acid (EDTA) and soaked in three 5 L vials with water. Residual fluid in the wells was removed by tapping the plates upside down on absorbing cloth or paper. The plates were allowed to dry for 30 min at<br>
25     room temperature and finally frozen for storage at - 20 °C. Protocol for DNA-polymerase assay.<br>
The DNA-polymerase assay is based on a short primer with a specific sequence that is covalently bound to the wells of a 96 well microtiter plate. The reaction mixture contains a single stranded deoxynucleotide template with a part of the sequence <br>
30     complementary to the primer and the four deoxynucleoside triphosphates. Thymidine<br>
triphosphate is, however, replaced by 5-bromodeoxyuridine 5'-triphosphate (BrdUTP). The amount of bromodeoxyuridine monophosphate (BrdUMP) incorporated into DNA during the<br><br>
,   WO 02/103059	PCT/SE02/01155<br>
7<br>
polymerase reaction, is detected with an alkaline phosphatase (Ap) conjugated anti-BrdU monoclonal antibody. An Ap substrate, 4-methyiumbelliferyl phosphate, is used for fluorimetric product detection.<br>
100 ul of DNA polymerase reaction mixture was added to each well of the primer coated microtiter plates. The samples were diluted in DNA polymerase base buffer and the polymerase reaction was initiated by transferring 50 ul sample dilution to each well on the plate. The microtiter plate was incubated at 33 °C and reaction was terminated after indicated times by washing the plate in 3 mM borate buffer (pH 8.9) with 1.5 % (v/v)octophenoxypolyethoxyethanol (Triton X-100). Usually two incubation times, 4 hours and over night (16 hours), were used to check the linearity of the polymerization reaction. The plates were washed thoroughly in 2M NaOH with 2 mM EDTA and soaked in three 5 L vials with water.<br>
Next the plates were incubated for 90 minutes at 33 °C with 100 u.1 alkaline phosphatase (Ap) conjugated anti-BrdU monoclonal antibody diluted to 4.8 ug/ml in 25 mM (bis(2-HyaVoxyethyl]iminotris[hydroxymemyl]methane;2a-bis[2-Hydroxyemyl]amino-2-[hydroxymethyl]-1,3-propanediol) (Bis Tris) buffer (pH 7.2) with 50 mM NaCl, 37.5 mM (NTL^SO^ 1 mg/ ml Dextran sulphate, 1 % TritonX-100 and 25 mg/ ml Sigma non-fat dried milk.<br>
The plates were thereafter washed again in 3 mM borate buffer (pH 8.9) with 1.5 % (v/v) TritonX-100 to remove unbound labeled antibody. The alkaline phosphatase activity was determined using 4-methylumbelliferyl phosphate substrate dissolved in Tris-buffer (pH 8.9). Fluorescence was read at 460 nm with a Wallac Victor 2 reader at defined intervals (excitation 355 nm). Protocol for determination of inhibition of second strand synthesis on variable DNA template.<br>
The inhibition studies were performed in a modified DNA polymerase assay. The drugs were serially diluted in five steps and 25 ul aliquots were transferred to each well in the microtiter plate, mixed with 100 ul DNA polymerase reaction mixture and the enzyme reaction was initiated by addition of 25 ul enzyme dilution. Non-nucleoside analogues were studied at standard reaction conditions while the concentration of all four deoxynucleoside triphosphates (dNTP) were reduced to luM in the studies of dNTP competing inhibitors. The polymerase reaction was allowed to proceed over-night (16-24 hours at 33°C). Thereafter the reaction was terminated by a wash of the plate. The IC50 value was defined as the concentration of drug giving 50% inhibition of the polymerase activity studied.<br><br>
WO 02/103039	PCT/SE02/O1155<br>
8 Protocol for determination of RT activity,<br>
A modification of the colorimetric RT assay (Cavidi® Lenti RT activity kit), available from Cavidi Tech, Uppsala, Sweden was used for the determination of the level of RT activity in the virus preparations studied. The method has been described (Ekstrand at al 1996). In short, poly(rA) covalently bound to the wells of a 96 well microtiter plate serves as template for the incorporation of 5-bromodeoxyuridine 5'-triphosphate (BrdUTP) during the reverse transcription step at 33°C. The amount of bromodeoxyuridine monophosphate (BrdUMP) incorporated into DNA, is detected with an alkaline phosphatase (Ap) conjugated anti-BrdU monoclonal antibody. An Ap substrate, 4-methylumbelliferyl phosphate, is finally used for fiuorimetrie detection . Protocol for determination of inhibition of reverse transcription.<br>
The inhibition studies were performed in a modified Cavidi HS-kit Lenti RT assay. The inhibitors were serially diluted in five steps and 25 ul aliquots were transferred, to each well in the microtiter plate, mixed with 100 ul RT reaction mixture and the enzyme reaction was initiated by addition of 50 ul enzyme dilution. The final nucleoside triphosphate substrate (BrdUTP) concentration was 16 uM and the primer (odT22) amount 12 ng per well. The RT reaction was allowed to proceed overnight (16-24 hours ait 33°C). Thereafter the reaction was terminated by a wash of the plate. The IC50 value was defined as the concentration of drug giving 50% inhibition of the RT activity studied.<br>
Protocol for isolation of viral RT from material which contains RT blocking antibodies, based on destruction of soluble cellular enzymes followed by isolation of viral RT from mini columns.<br>
1) Label the 4.5ml plastic tubes to be used. Place them in a Nalgene box. Add 1 ml of sample (e.g. EDTA plasma from HTV infected individuals) to each labeled tube. Add 100 u.1 of a 66 mM solution of 5,5'-dithiobis-(2-nitrobenzoic acid) in buffered water, vortex and incubate the samples for one hour at room temperature.<br>
The activity of the free plasma enzymes is destroyed during this procedure while the enzymes contained within the virions remain intact. The virions can then be purified from 5,5'-dithiobis-(2-nitrobenzoic acid), enzyme activity blocking antibodies and other substances that may interfere with quantification of viral RT by several separation procedures. The protocol below is based on the use of Fractogel® EMD TMAE Hicap gel. 2)   Suspend the separation gel carefully and transfer 1500 u.1 gel slurry to each sample pre-treatment tube.<br><br>
WO 02/103039	PCT/SE02/01155.<br>
9<br>
3)  Incubate the samples with the gel slurry for 90 minutes at room temperature with the tubes lying down horizontally on an orbital shaker.<br>
4)	Label the desired amount of 10 ml plastic mini columns to identify the samples being<br>
analyzed. Mount the columns in a column washing device i.e. a Supelco Visiprep solid<br>
phase extraction vacuum manifold. Transfer the contents in the binding tubes to their<br>
corresponding columns. Before transfer vortex the tube briefly to evenly distribute the gel.<br>
5)	When all the columns are filled, apply the vacuum and suck the gels dry. Turn off the<br>
vacuum and start the washing by filling each column with 9 ml buffer A. When all<br>
columns have been filled, apply the vacuum and suck the gels dry.<br>
6)	  Repeat step 5 three more times, giving a total of four washes. Suck the gels dry after each<br>
wash. After sucking the gels dry after the fourth wash, turn off the vacuum and proceed to<br>
step 7.<br>
The washing step removes unbound RT blocking antibodies and 5,5'-dithiobis-(2-nitrobenzoic acid) from the system.<br>
7)	Add to all dry gels 9 ml of conditioning buffer (B). After one minute apply vacuum and suck the gels dry.<br>
8)	Repeat step 7. Before turning off the vacuum check that all conditioning buffer (B) has been removed from all gels.<br>
9)	Lift off the upper part of the column wash device. Mount the tube holder with the labeled tubes into a clean container. Refit the upper part of the device. Control that the small tubings from each column go down in their corresponding tubes.<br>
10)	Add 600 p.1 lysis buffer (C) to each column. Let the buffer stand in the column for five minutes. Then apply the vacuum slowly and suck the gels dry. This will in each tube give approximately 600 p.1 of virus lysate from the connected gel.<br>
The RT activity recovered in the lysates from step 10 are essentially free from RT blocking antibodies, drugs and cellular polymerase activity, and can be quantified with a sensitive RT activity assay, i.e. the Cavidi HS-kit Lenti RT, which is based on the method described by Ekstrand et al [7]. 25 ul lysate obtained according to the current protocol is sufficient for determination of the RT activity in the sample. The Remaining 575 p.1 sample should be frozen at -70 °C or below for later use in the drug sensitivity test.<br><br>
WO 02/103039	PCT/SE02/01155<br>
10<br>
Note: RT enzymes that are not sensitive to cystein modifying agents e.g. wild type HTV 1 RT can optionally be assayed in the presence of up to 5 mM 5,5'-dithiobis-{2-nitrobenzoic acid). Sensitive enzymes such as MULV RT and RT from certain therapy resistant HTV 1 strains (containing e.g. the mutation Yl 81C) on the other hand require addition of a sulfhydryl <br>
5     reducing agent i.e. cystein or cysteamine to the lysis buffer. Materials Primer / Template   <br>
for DNA polymerase assay.<br>
The primer sequence is 18 bases 5'-GTC-CCT-GTT-CCG-GCG-CCA-3' (SEQ<br>
ID NO: 12) and linked at the 5' end to a primary amine by a C6 spacer arm.<br>
10	The template construct contains three parts with different functions. From the 5'<br>
end: A (A)n polymer used to amplify the BrdU signal, a variable part (GTCA)m to obtain a polymerase reaction that is dependent on the four deoxynucleosi.de triphosphates and a sequence complementary to the primer.<br>
In the experiments included in the examples n=12 and m=5, if not otherwise is stated . <br>
15     Nucleosides, enzyme inhibitors and antiviral drugs<br>
ddATP, 2'3',dideoxyadenosine triphosphate; ddGTP, 2'3' dideoxyguanosine triphosphate; ddCTP, 2'3' dideoxy cytidine triphosphate; ddTTP, 2'3' dideoxy thymidine triphosphate. TMAU, 6-([3,4-trimethylene]anilino)uracil<br>
20     Tenofovir, (R)-9-(2-phosphonylmethoxy-propyl)adenine; Nevirapine, (1 l-cyclopropyl-5,11-dihydro-4-methyl-6H-dipyrido[3,2-b:2',3'-fj[l,4]diazepin-6-one) (NVP); and Efavirenz, (-)6-chloro-4-cyclopropylethynyl-4-trifluoromethyl-l,4-dihydro-2H-3,1 -benzoxazin-2-one) (EFV). Enzymes<br>
DNA polymerase I (E. coli) was purchased from Amersham Bioscience.<br>
25     Recombinant DNA polymerase III from Staphylococus aureus was produced as described (Brown et al 1998). Mammalian DNA polymerase a (calf thymus) and (3 (human) were purchased from CHTMERx (Milwaukee). DNA polymerase y was purified from beef heart as described (Pileur et al 2000).<br>
NNRTI resistant mutant forms of HIV-1 RT were produced (LI 001, K103N, <br>
30      L100I/K103N, Y181C). As template for the mutations was used the pETRT expression <br>
vector, which was constructed from the BH10 isolate. Mutations were generated using commercial site-directed mutagenesis kits, QuikChange (Stratagene). The mutations were verified by DNA sequence analysis. The mutated and native forms of RT were isolated as previously described (Lindberg et al 2002).<br><br>
WO 02/103039	PCT/SE02/01155<br>
11<br>
The procedure for production of recombinant HIV-1 RTs with AZT specific mutations were similar but the mutations were introduced into the RT-coding region from the HXB2-D isolate. • Plasma samples from HIV infected individuals.<br>
Plasma samples from treatment naive patients or from patients treated with ordinary combination therapy were selected retrospectively. The amount of HIV-1 RNA in each sample was measured by standard HIV 1 RNA PCR (Cobas, Roche Diagnostica). Serum samples from patients with lympho proliferative disorders were obtained from the Department of Internal Medicine, Uppsala University, Akademiska sjukhuset, Uppsala Separation gel: e.g. Fractogel® EMD TMAE or Fractogel® EMD TMAE Hicap in 314 mM (2-(N-Morpholino)ethanesulfonic acid) (MES) pH 5.1,413 mM Potassium iodide and<br>
Heparin 0.5 mg / ml.<br>
Mini columns, e.g. Biorad Poly-Prep® (7311553)<br>
Mini column washing device, i.e. Supelco Visiprep solid phase extraction vacuum manifold.<br>
Plastic tubes, e.g. Nunc 4.5 ml cryogenic tubes.<br>
Microliter plates with immobilised prA, i.e. Nalge Nunc NucleoLinck®<br>
Cysteine modifying agent, e.g. 66 mM 5,5'-dithiobis-(2-nitrobenzoic acid) in water buffered<br>
with 0.87 M Tris(hydroxymethyl)aminomethane (pH 8.3).<br>
Mild sulfhydryl reducing agent, e.g. 33 mM cysteamine in water.<br>
Buffers used:<br>
A)	Wash buffer: 20 mM MES pH 5.4, 500 mM Potassium acetate (KAc)<br>
B)	Conditioning buffer. An RT assay compatible buffer e.g, 50 mM (N-(2-Hydroxyethylpiperazine-N'-(2-ethanesulfonic acid) (Hepes) pH 7.6, KAc 25 mM, magnesium chloride (MgCh) 20 mM, EthyleneGlycol-bis(P-arninoethyl Ether) N,N,N',N'-Tetraacetic Acid (EGTA) 0.2 mM, spermine 2 mM and heat inactivated bovine serum,albumin (BSA) 0.5 mg / ml.<br>
C)	Lysis buffer: An RT assay compatible buffer including a'detergent e.g. 1.25 % Polyoxyethylene 4 Lauryl Ether (Brij 30), 13 ng / ml odT^ and the same components as in the conditioning buffer (B). A sulfhydryl reducing agent, i.e. 0.2 mM cysteamine is optionally added when processing viruses with RT that are sensitive to SH oxidation / modification.<br><br>
WO 02/103039	PCT/SE02/01155<br>
12<br>
RT Reaction mixture: (N-(2-Hydroxyethylpiperazine-N'-(2-ethanesulfonic acid) (Hepes) 11.7 mM pH 7.6, BrdUTP 28.3 uM, odT22 120 ng/ml, MgCl2 4 mM, dextrane sulphate 0.05 g/1,<br>
spermine 2 mM, Triton-X 100 0.5 %(v/v), EthyleneGIycol-bis(|3-aminoethyl Ether) N,N,N',N'-Tetraacetic Acid (EGTA) 0.2 mM and bovine serum albumin (BSA) 0.5 mg/ml. <br>
5<br>
DNA polymerase /and Retro DNA polymerase base buffer. Hepes 50 mM pH 8.0, MgCl2 8<br>
mM, dextrane sulphate 1.5 ug/1, spermine ImM, Triton-X 100 0.5 %(v/v), EGTA 0.2 mM, dithiothreitol (DTT) 1.5 mM and bovine serum albumin (BSA) 0.5 mg/ml.<br>
10     DNA polymerase III base buffer. (2-(N-Morpholino)ethanesulfonic acid) (MES) 40 mM pH 6.8, Potassium acetate (KAc) 40mM, MgCl2 10 mM, spermine 2 mM,<br>
polyoxyethylenesorbitan monolaureate (Tween 20) 0.5 %(v/v), EDTA 0.1 mM, dithiothreitol 1 mM and bovine serum albumin (BSA) 50 ug/ml.<br>
15 DNA polymerase j3base buffer. 3-[( 1,1 -dimethyl-2-hydroxyethyl)amino]-2-hydroxy-1 -propanesulfonic acid (AMPSO) 20 mMpH 8.3, MgCl21 mM, Spermidine3 mM, BSA1 ug/ml, EDTA10 μM, DTT 0.1 mM, Tween 20 0.01%.<br>
DNA polymerase reaction mixture. DNA polymerase base buffer reinforced with 24 uM <br>
20     BrdUTP, 49.5 uM dGTP, 49.5 uM dATP, 49.5 uM dCTP and 500 ng template / ml.<br>
Retro DNA polymerase reaction mixture with ATP. The DNA polymerase /and Retro DNA polymerase reaction mixture was reinforced with 3.2 mM ATP and pH was adjusted to 7.1.<br>
25     Examples<br>
Example 1. Utilization of different templates for second strand synthesis by HTV RT.<br>
Two step dilution series of indicated template constructs starting from 200 ng / ml were  included in each well of mictrotiter plates with immobilized primer according to "Production of primer coated microtiter plates ". 100 fg recombinant HTV 1 RT was added to <br>
30     	each well and the duration of the RT reaction was 18 hours. The polymerase activity on each template was determined according to "Protocol for DNA-polymerase assay" using DNA polymerase /and Retro DNA polymerase base buffer. 50 mM NaCl or alternatively 100 mM was used during the binding of anti-BrdU monoclonal antibody. The activities found were<br><br>
WO 02/103039	PCT/SE02/01155<br>
13<br>
  plotted towards concentration of template and the maximal signal achieved for each template type were calculated from the plateau value of respective graph.<br>
The results are summarized in Table 1. A clear correlation was found between the length of the A-tail and maximal signal obtainable in the polymerase assay. The length of the A-tail did also affect the ability of the antibody used for product detection to bind at increased ionic strength.<br>
Example 2. Effects of template sequence on inhibition of DNA polymerase III with TMAU.<br>
6-anilinouracils are selective inhibitors of DNA polymerase III from gram-positive bacteria. The anilinouracil molecule inhibits its polymerase LI target by sequestering it into an inactive DNA-drug-protein complex (Tarantino et al 1990). The drug TMAU can be regarded as an analogue to GTP. The inhibitory capacity of indicated concentrations of TMAU towards 1.25 ng / well recombinant DNA polymerase IK was determined according to "Protocol for determination of inhibition of second strand synthesis on variable DNA template" utilizing either (CTGA)6-A12 (SEQ ID NO: 10) (■) or (CTG)6-A3 (SEQ ID NO: 11) (A,) as template. The polymerase reaction time was one hour and the GTP concentration in the DNA polymerase m reaction mixture was reduced to 2.5 uM. The polymerase activities obtained on indicated template were recalculated into % of the activities found with the same polymerase incubated in absence of inhibitor.<br>
The results are depicted in Figure. 1. This highly specific inhibitor exhibited varying inhibitory capacity depending on the sequence of the template used. The current invention provides a system were the DNA template used easily can be changed to account for the specific conditions required by the enzyme or inhibitor studied.<br>
Example 3. Detection sensitivity of the DNA polymerase assay.<br>
The activity of serial dilutions of: HTV 1 wild type RT (♦), mammalian DNA polymerase P (•) and E. coli DNA polymerase I (A) were measured according to "Protocol for DNA-polymerase assay" utilizing base buffers optimized for indicated enzyme. The polymerase reaction time used was over-night (18 hours). The results are depicted in Figure 2. Each of the three enzyme preparations displayed a linear relationship between amount of enzyme used and amount of product recovered. The assay background was 3800 rfu/ hour. Utilizing double background as cut-off value for significant signal detection it was possible to<br><br>
WO 02/1031)39	PCT/SE-02/0U55<br>
14<br>
i detect down to 1 OntX HIV 1 wild type RT, 6 nil mamalian DNA polymerase p* and 3nU E. coli DNA polymerase I.<br>
Example 4. The activity of five DNA polymerases in different assay systems.<br>
The activity of serial dilutions of DNA polymerase a, DNA polymerase (3, DNA <br>
5     	polymerase y, serum from a patient suffering from non-Hodgkin's lymphoma and HTV-l RT were measured according to "Protocol for DNA-polymerase assay" and "Protocolfor determination of RT activity" using reaction solutions based on indicated polymerase base buffers. The polymerase reaction time used was 2 hours. The activities found were recalculated into % of the activity on variable DNA template at the optimal reaction<br>
10     conditions for each enzyme. The results are shown in Table 2. Each of the five en2yme preparations investigated had their individual preferences concerning optimal reaction conditions. DNA polymerase a and the human serum polymerase, however, displayed a similar pattern. Only HTV-l RT and DNA. polymerase y gave a significant activity in the reverse transcriptase assay.<br>
15     Example 5. Demonstration of the ability of the DNA polymerase assay to measure inhibition by the dideoxy analogues to all the four DNA bases.<br>
The inhibitory capacity of indicated concentrations of ddATP (■), ddGTT (♦), ddCTP (•) and ddTTP (A) towards the activity of 80 fg recombinant wild type HTV 1 RT was determined according to ilProtocolfor determination of inhibition of second strand<br>
20     	synthesis on variable DNA template". The polymerase activity at each inhibitor concentration was recalculated into % of the activity of a control in absence of inhibitor. The results are depicted in Figure 3.<br>
The polymerase reaction utilizing the template (CTGA)6-Ai2 (SEQ ID NO: 10) was found sensitive to inhibition by the dideoxy analogues to all the four DNA bases. The <br>
25     IC50 values found varied from 20 nM for ddCTP to 80 nM for ddATP.<br>
Example 6. Comparison of the effect of non-nucleoside inhibitors on first and second strand DNA synthesis by HIV 1 RT.<br>
The effects of three non-nucleoside inhibitors on indicated recombinant HTV 1 RT were determined according to "Protocol for determination of inhibition of second strand<br>
30    	 synthesis on variable DNA template" using DNA polymerase /and Retro DNA polymerase base buffer and "Protocol for determination of inhibition of reverse transcrivtion "<br><br>
WO 02/103039	PCT/SE02/01155<br>
15<br>
    respectively. The duration of the polymerization reactions were 19 hours for the second strand synthesis on (GTGA)6-Ai2 (SEQ ID NO: 10) and 2 hours for the first strand synthesis on prA respectively. The RT activities obtained were recalculated into % of the activities found with the same RT incubated in absence of inhibitor.<br>
The results are summarized in Table 3. Both assay systems had the capacity to distinguish between resistant (Y181C, V179D) and sensitive RT enzymes. The IC50 values for any of the inhibitors was not significantly affected of a five fold difference in the amount of enzyme used in any of the two assay systems. Further there was no significant difference between the IC50 values achieved by measuring inhibition of first or second strand DNA synthesis.<br>
Example 7. Demonstration of the biochemical mechanism underlying resistance to the antiviral drug tenofovir.<br>
Prolonged therapy of HTV infected individuals with nucleoside analogues leads to the development of resistant virus. This process is associated with the gradual appearance of mutations in the viral pol gene. The effects of these substitutions at the enzymatic levels are complicated and include enhancement of a primitive DNA editing function. This reaction is nucleotide dependent and produces dinucleoside polyphosphate and an extendible DNA 3'end.<br>
The effects of serial dilutions of tenofovir triphosphate on 4 pg /well of indicated recombinant HIV 1 RT were determined according to "Protocol for determination of inhibition of second strand synthesis on variable DNA template" utilizing standard reaction solution based on DNA polymerase /and Retro DNA polymerase base buffer and the same reaction solution supplemented with ATP. The duration of the polymerization reactions was 19 hours. The RT activities obtained were recalculated into % of the activity of the same RT incubated in absence of inhibitor. Figure 4 shows the resuts.<br>
Symbols: HIV-1 wild type RT in standard DNA polymerase reaction solution (O). HTV-1 wild type RT in DNA polymerase reaction solution with ATP (•), T69S-&gt;-SS/L210W/T215Y HTV-1 mutant RT in standard DNA polymerase reaction solution (O), T69S-&gt;SS/L210W/T215Y HTV-1 mutant RT in DNA polymerase reaction solution with <br>
ATP(^).<br>
The results depicted in figure 4 demonstrate that the difference in drug susceptibility between wild type and mutant RT increased approximately 1.0-fold. when. 	a ..<br><br>
WO 02/103039	PCT/SE02/01155<br>
16<br>
reaction solution with capacity to support an ATP dependent phosphorolysis reaction was used.<br>
Example 8. Determination of the susceptibility to Nevirapine using plasma derived RT.<br>
One ml samples of plasma from 3 HTV infected individuals from Stockholm, <br>
5    . 		Sweden were processed according to "Protocol for isolation of viral RT from material which contains RT blocking antibodies, based on destruction of soluble cellular enzymes followed by isolation of viral RT from mini columns" Each plasma RT and two control enzymes were titrated towards a set of serial dilutions of Nevirapine according to "Protocol for determination of inhibition of second strand synthesis on variable DNA template " using<br>
10      "DNA polymerase yand Retro DNA polymerase base buffer". See Figure 5.<br>
Symbols: (□) RT from patient 1 having 140 000 genome copies /ml (■ )RT from patient 2 having 180 000 genome copies /ml, (^ ) RT from patient 3 having 390 000 genome copies /ml,.(•) a control consisting of recombinant HTV-1 wild type RT, (♦) and a control consisting of the recombinant HIV-1 mutant RT (LI 001), with intermediate Nevirapine<br>
15      resistance.<br>
The IC50 values towards Nevirapine found for the patient RTs varied from 0.7 to 1.2 uM. To be compared with 0.5 u.M for the control wild type RT and &gt;10 uM for the mutant RT with intermediate Nevirapine resistance (LI001).<br>
20      Example 9. Detection of a DNA polymeras activity in serum from patients with lymphoproliferative disorders.<br>
Serum from four patients suffering from non-Hodgkin's lymphoma and from six healthy blood donors were serially diluted in DNA polymerase III base buffer. The polymerase activity at each dilution step were measured according to "Protocolfor DNA-<br>
25     	polymerase assay" using two and six hours polymerase reaction time.<br>
The DNA polymerase activity / p.1 serum sample and hour polymerase assay was calculated at the dilution range were there was a linear relationship between the amount of product formed and the amount of plasma added to the assay (see table 4).<br>
Each serum sample from the patients with non-Hodgkin's lymphoma contained <br>
30       	individual amounts of DNA polymerase activity, with similar properties as DNA-polymerase a <br>
(see table 3). The amount of activity ranged from approximately 2 to 190 times the average value found among healthy blood donors.<br><br>
WO 02/103039<br><br>
PCT/SE02/01155<br><br>
17<br>
finable 1. Utilization of different templates for second strand synthesis by HIV RT.<br><br>
Template used<br><br>
Signal at 100 mM<br>
Maximal signal*   Background   NaCT(%o.f<br>
(rfu / hour)	(rfu/hour)       maximal signal)<br><br>
SEQ ID NO:<br><br><br><br>
10<br>
15<br><br><br>
(CTGA)5	89444	1629	20	SEQ ID NO: 1<br>
(CTGA)5-A	63694	1958	25	SEQ ID NO: 2<br>
(CTGA)5-AA	184421	613	33	SEQ ID NO: 3<br>
(CTGA)5-AAAA	239688	2565	44	SEQ TD NO: 4<br>
(CTGA)5-A8	198894	1544	58	SEQ ED NO: 5<br>
(CTGA)6	60627	1285	22	SEQ ID NO: 6<br>
(CTGA)6-AAA	83555	1009	ND	SEQ ID NO: 7<br>
(CTGA)6-A5	119326	914	ND	SEQ ID NO: 8<br>
(CTGA)6-A9	202668	963	ND	SEQ ID NO: 9<br>
(CTGA)6-A12	226062	726	ND	SEQ ID NO: 10<br>
(CTG)6-A3	124334	814	ND	SEQ DD NO: IT<br><br>
20<br>
* Two step dilution series of indicated template starting from 200 ng / ml were included in  each  well of mictrotiter plates with immobilized primer. 100 fg recombinant HIV 1 RT was<br>
added to each well and the duration of the RT reaction was 18 hours. 50 mM was used during the binding of anti-BrdU monoclonal antibody. The activities found were plotted towards concentration of template and the maximal signal achieved for each template was calculated. 3 RT reaction conditions and calculations was. identical as in "maximal signal", but 100 mM <br>
25     NaCl was used during the antibody binding.<br><br>
WO 02/103039	PCT/SE02/01155<br>
18<br>
Table 2, Apparent activity of four mammalian DNA polymerases and HIV RT in different assay systems<br>
Activity of DNA polymerase isozyme in indicated assay (% *)<br>
5     Base buffer	DNA. poi cc DNA pol J3        DNA. poi y    serum DNAp       HTV RT<br>
0 33 100 2175<br>
Pol IE	100	17	0	100<br>
DNA pol (3	40	100	5	30<br>
DNA poly3	1	13	100	9<br>
10     prA/Lentib	0	0	10	0<br>
·	% of activity on variable DNA template with optimal base buffer for indicated isozyme a DNA polymerase y and Retro DNA polymerase base buffer. b Activity in RT assay with prA / odT as primer template. <br>
15<br><br>
WO 02/103039<br><br>
PCT/SE02/0115:<br><br>
19<br>
     Table 3. Comparison of the effect of non-nucleoside inhibitors on first and second strand DNA synthesis by HIV 1 RT.<br><br>
10<br>
15<br>
20<br>
25<br><br>
	RT enzyme	ICsofun	l) on template*<br>
5     Inhibitor	Type	amount (fg)	(CTGA)6	-Ai2       prA<br>
Nevirapine	wt	100	2.0	2.2<br>
	wt.	20	2.0	2.5<br>
	Y181C	100	&gt;500	200<br>
3	Y181C	20	&gt;500	180<br>
	V179D	100	8	7<br>
	V179D	20	12	8<br>
Efavirenz	wt	100	0.04	0.02<br>
	wt	20	0.03	0.02<br>
	Y181C	100	0.12	■ 0.15<br>
	Y181C	20	.0.14	0.13<br>
	V179D	100	0.5	0.7<br>
I	V179D	20	0.4	0.6<br>
Foscamet	wt	100	0.5	0.7<br>
	wt	20	0.5	0.6<br>
* The effects of three non-nucleoside inhibitors on indicated recombinant HTV 1 RT were determined according to "Protocol for determination of inhibition of second strand synthesis on variable DNA template'1 and "Protocol for determination of inhibition of reverse transcription " respectively. The duration of the polymerization reactions were 19 hours for . the second strand synthesis on (CTGAVA12 (SEQ ED NO: 10) and 2 hours for the first strand synthesis on prA respectively. The RT activities obtained were recalculated into % of the activities found with the same RT incubated in absence of inhibitor.<br><br>
WO 02/103039	PCT/SE02/01155<br>
20<br>
Table 4. Detection of a DNA polymerase activity in serum from patients with lymphoproliferative disorders.<br>
5	Polymerase activity (rfu /hour /u.1. serum<br>
Serum code       Serum origin        / hour polymerase reaction))<br><br>
Tl	NHL patient3	158521<br>
T2	NHL patient	1305<br>
T3	NHL patient	3405<br>
T4	NHL patient	16619<br>
B1-B6	Blood donors	841±180<br>
a Patient suffering from non-Hodgkin's lymphoma.<br>
15     * NS, not significant.<br>
Serum from four patients suffering from non-Hodgkin's lymphoma and from six healthy blood donors were serially diluted in DNA polymerase III base buffer. The polymerase activity at each dilution step were measured according to "Protocol for DNA-polymerase assay". The DNA polymerase activities tabulated was calculated from the dilution range were <br>
20     	there was a linear relationship between the amount of product formed and the amount of plasma added. The assay background 854 flu/ h has been subtracted from all values.<br><br>
WO 02/103039	PCT/SE02/01155<br>
21<br>
  References<br>
Aposhian HV, Komberg A. J.Biol.Chem. 1962, 237, 519<br>
Arion D, Kaushik N, McCormick S, Borkow G, Parniak MA. Phenotypic mechanism of HTV-1 <br>
5    	 resistance to 3'-azidc-3'-decxythymidine (AZT): increased polymerization processivity and enhanced sensitivity to pyrophosphate of the mutant viral reverse transcriptase. Biochemistry. 1998 Nov 10;37(45):15908-17.<br>
   Barnes MH, Leo CJ, Brown NC. DNA polymerase III of Gram-positive eubacteria is a zinc <br>
10     	metalloprotein conserving an essential finger-like domain. Biochemistry. 1998 Nov 3 ;37(44): 15254-60<br>
Eberie J, Seibl R, Kessler, C, Konig, Bemhard B. Reagents and kits for determining polymerase  activity. 1997 US5635350: <br>
15<br>
Ekstrand DH, Awad RJ, Kallander, CF, Gronowitz JS. A sensitive assay for the quantification   of reverse transcriptase activity based on the use of carrier-bound template and non-radioactive-product detection, with special reference to human-immunodeficiency-virus isolation. Biotechnol Appl Biochem. 1996 Apr;23 (Pt 2):95-105. <br>
20<br>
Goff, S.P. (1990) Retrovirus reverse transcriptase: Synthesis, Structure, and Function. Review. J Acquir Imm Defic Syndr 3: 817-831.<br>
Gronowitz; J S, Kallander, C F R Method of determining dTk isoenzyme activity and the use 25     thereof. 1987 US4637977:<br>
Meyer PR, Matsuura SE, Mian AM, So AG, Scott WA. Related Articles. A mechanism of AZT resistance: an increase in nucleotide-dependent primer unblocking by mutant HTV-1 reverse transcriptase. Mol Cell. 1999 Jul;4(l):35-43 <br>
30<br>
Pileur F, Toulme JJ, Cazenave C. Eukaryotic ribonucleases HI and HJJ. generate characteristic hydrolytic patterns on DNA-RNA hybrids: further evidence that mitochondrial RNase H is an RNase HJJ. Nucleic Acids Res. 2000 Sep 15;28(18):3674-83<br><br>
WO 02/10.3039	PCT/SE02/01155<br>
22<br>
Seville M, West AB, Cull MG, McHenry CS. Fluorometric assay for DNA polymerases and   • reverse transcriptase.Biotechniques. 1996 Oct;21(4):664, 666, 668, 670, 672.<br>
5      Tarantino PM Jr, Zhi C, Wright GE, Brown NC. Related Inhibitors of DNA polymerase HI as novel antimicrobial agents against gram-positive eubacteria. Antimicrob Agents Chemother. 1999 Aug;43(8): 1982-7.<br>
Tveit H, Kristensen T. Fluorescence-based DNA polymerase assay. Anal Biochem. 2001 Feb <br>
10      l;289(l):96-8.<br>
Vandamme AM, Van Vaerenbergh K, De Clercq E. Anti-human immunodeficiency virus drug combination strategies. Antivir Chem Chemother. 1998 May;9(3): 187-203.<br>
15<br><br>
WO 02/103039	PCT/SE02/01155<br>
23<br>
Claims<br>
I. A method for measuring DNA-dependent DNA polymerisation in a biological sample,<br>
comprising the steps of<br>
5	a) providing a primer with a single stranded short specific sequence, which<br>
is unable to base pair internally, bound to a solid phase,<br>
b)	contacting the primer construct with a reaction mixture containing a<br>
single stranded deoxynucleotide template with a part of the sequence<br>
complementary to the primer and the four deoxynucleoside triphosphates,<br>
10	one of which is modified so that it is specifically recognized by a labeled<br>
antibody,<br>
c)	adding a biological sample comprising the DNA polymerase to the mixture of b),<br>
d)	allowing the polymerase reaction to proceed,<br>
15	e) incubating the immobilized reaction product resulting from d) with the<br>
labeled antibody,<br>
f)	detecting the amount of bound labeled antibody with the aid of the label used, and<br>
g)	measuring the amount of incorporated modified deoxynucleoside<br>
20	triphosphate, as a measure of the DNA polymerization, with the aid of<br>
the label of the bound antibody.<br>
2.	The method according to claim 1, wherein the DNA polymerase is a retrovirus reverse<br>
transcriptase (RT).<br>
25<br>
3.	The method according to claim 2, wherein the retrovirus RT is human immunodeficiency<br>
virus (fflV)RT.<br>
4.	The method according to any one of claims 1 - 3, wherein the modified deoxynucleoside<br>
30	triphosphate is 5-bromodeoxyuridine 5'-triphosphate (BrdUTP) and the labeled antibody<br>
is an alkaline phosphatase (Ap) conjugated anti-BrU monoclonal antibody.<br>
5.	The method according to any one of claims 1-4, wherein the measured DNA<br>
polymerisation is used for drug susceptibility testing."<br><br>
WO 02/103039	PCT/SE02/01155<br>
24<br>
6. Commercial package containing written and/or data carrier instructions for measuring<br>
DNA-dependent DNA polymerisation according to any one or claims 1 - 4, comprising<br>
a) a primer with a single stranded short specific sequence, which is unable to<br>
base pair internally, bound to a solid phase,<br>
  5	b) a single stranded deoxynueleotide template with a part of the sequence<br>
complementary to the primer in a),<br>
c)	the four deoxynucleoside triphosphates, one of which is modified so that<br>
it is specifically recognized by a labeled antibody, and<br>
d)	the labelled antibody that recognizes the modified deoxynucleoside<br>
10	triphosphate in c).<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWFic3RyYWN0LSgyOS0yLTIwMDgpLmRvYw==" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-abstract-(29-2-2008).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWFic3RyYWN0LSgyOS0yLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-abstract-(29-2-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWNhbmNlbGxlZCBwYWdlKDI5LTItMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-cancelled page(29-2-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWNsYWltcyhncmFudGVkKS0oMjktMi0yMDA4KS5kb2M=" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-claims(granted)-(29-2-2008).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWNsYWltcyhncmFudGVkKS0oMjktMi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-claims(granted)-(29-2-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWNsYWltcy5kb2M=" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-claims.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWNvcnJlc3BvbmRlbmNlKDEzLTUtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-correspondence(13-5-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1NVU1OUC0yMDAzLUNPUlJFU1BPTkRFTkNFKDMxLTUtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1113-MUMNP-2003-CORRESPONDENCE(31-5-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWNvcnJlc3BvbmRlbmNlKGlwbyktKDE2LTctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-correspondence(ipo)-(16-7-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWNvcnJlc3BvbmRlbmNlLXJlY2VpdmVkLTAxMDQwNC5wZGY=" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-correspondence-received-010404.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWNvcnJlc3BvbmRlbmNlLXJlY2VpdmVkLTEwMDMwNC5wZGY=" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-correspondence-received-100304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWNvcnJlc3BvbmRlbmNlLXJlY2VpdmVkLTEzMDQwNC5wZGY=" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-correspondence-received-130404.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWNvcnJlc3BvbmRlbmNlLXJlY2VpdmVkLTE0MDEwNC5wZGY=" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-correspondence-received-140104.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWNvcnJlc3BvbmRlbmNlLXJlY2VpdmVkLTIyMDUwNC5wZGY=" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-correspondence-received-220504.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWNvcnJlc3BvbmRlbmNlLXJlY2VpdmVkLnBkZg==" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-correspondence-received.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWRyYXdpbmcoMjktMi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-drawing(29-2-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWZvcm0gMTMoMjktMi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-form 13(29-2-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWZvcm0gMTgoMjYtNS0yMDA2KS5wZGY=" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-form 18(26-5-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWZvcm0gMWEoMjktMi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-form 1a(29-2-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWZvcm0gMihncmFudGVkKS0oMjktMi0yMDA4KS5kb2M=" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-form 2(granted)-(29-2-2008).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWZvcm0gMihncmFudGVkKS0oMjktMi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-form 2(granted)-(29-2-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1NVU1OUC0yMDAzLUZPUk0gMjYoMzEtNS0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">1113-MUMNP-2003-FORM 26(31-5-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWZvcm0gMygxNi0zLTIwMDQpLnBkZg==" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-form 3(16-3-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWZvcm0gMygyOS0xLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-form 3(29-1-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWZvcm0gMyg1LTEyLTIwMDMpLnBkZg==" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-form 3(5-12-2003).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWZvcm0gMyg3LTQtMjAwNCkucGRm" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-form 3(7-4-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWZvcm0gNSgyOS0yLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-form 5(29-2-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWZvcm0gNSg1LTEyLTIwMDMpLnBkZg==" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-form 5(5-12-2003).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWZvcm0tMWEucGRm" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-form-1a.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWZvcm0tMi5kb2M=" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-form-2.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWZvcm0tMy0wNzA0MDQucGRm" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-form-3-070404.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWZvcm0tcGN0LWlwZWEtNDA5KDUtMTItMjAwMykucGRm" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-form-pct-ipea-409(5-12-2003).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLWZvcm0tcGN0LWlzYS0yMTAoNS0xMi0yMDAzKS5wZGY=" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-form-pct-isa-210(5-12-2003).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLXBvd2VyIG9mIGF1dGhvcml0eSgyOS0yLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-power of authority(29-2-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTExMy1tdW1ucC0yMDAzLXBvd2VyIG9mIGF1dGhvcml0eSg1LTEyLTIwMDMpLnBkZg==" target="_blank" style="word-wrap:break-word;">1113-mumnp-2003-power of authority(5-12-2003).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QxLmpwZw==" target="_blank" style="word-wrap:break-word;">abstract1.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="222790-emulsifiable-myco-chemical-herbicide-concentrate.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="222793-an-inclusion-complex-of-bupropion-hydrochloride-with-beta-cyclodextrin.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>222791</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1113/MUMNP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>39/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>26-Sep-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>25-Aug-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>05-Dec-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CAVIDI TECH AB</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>UPPSALA SCIENCE PARK, SE-75183 UPPSALA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>CLAS KALLANDER</td>
											<td>HUSBYGARD,HUSBYBORG, SE-755 92 UPPSALA.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>INGVAR PETTERSSON</td>
											<td>STAGNELIUSGATAN 3, SE-75427 UPPSALA.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>SIMON GRONOWITZ</td>
											<td>NORBYVAGEN 51, SE-75239 UPPSALA.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>XINGWU SHAO</td>
											<td>RACKARBERGSGATAN 96:230, SE-75232 UPPSALA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C 12 Q 1/48</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/SE02/01155</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-06-14</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/297,773</td>
									<td>2001-06-14</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/222791-a-method-for-measuring-dna-polymerization-and-applications-of-the-method by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:40:10 GMT -->
</html>
