<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/222898-methods-of-inducing-the-expression-of-bone-morphogenetic-proteins-bmps-and-transforming-growth-factor-a-proteins-tgf-as-in-cells by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:23:23 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 222898:METHODS OF INDUCING THE EXPRESSION OF BONE MORPHOGENETIC PROTEINS (BMPS) AND TRANSFORMING GROWTH FACTOR-â PROTEINS (TGF-âS) IN CELLS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHODS OF INDUCING THE EXPRESSION OF BONE MORPHOGENETIC PROTEINS (BMPS) AND TRANSFORMING GROWTH FACTOR-â PROTEINS (TGF-âS) IN CELLS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention discloses a method of inducing the expression of one or more bone morphogenic proteins or transforming growth factor-ß proteins in a cell, the method comprising: transfecting a cell with an isolated nucleic acid comprising a nucleotide sequence encoding a LIM mineralization protein (LMP) operably linked to a promoter, wherein the LMP is chosen from the group consisting of Human LMP-1 (HLMP-1), truncated versions of Human LMP (HLMP- 1s), HLMP-3 or combinations thereof.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>METHODS OF INDUCING THE EXPRESSION OF BONE<br>
MORPHOGENETIC PROTEINS (BMPs) AND TRANSFORMING<br>
GROWTH FACTOR-P PROTEINS (TGF-ßs).IN CELLS<br>
Field of the Invention<br>
The field of the invention relates generally to methods for transfecting cells<br>
with genetic material. More specifically, the field of the invention relates to<br>
methods of inducing the expression, of one or more bone morphogenetic proteins<br>
(BMPs) and/or transforming growth, factor-p proteins (TGF-ps) by transfecting a<br>
cell with a nucleic acid encoding a LIM mineralization protein (LMP).<br>
Background of the Technology<br>
Osteoblasts are thought to differentiate from pluripotent mesenchymal stem<br>
cells. The maturation of an osteoblast results in the secretion of an extracellular<br>
matrix which can mineralize and form bone. The regulation of this complex<br>
process is not well understood but is thought to involve a group of signaling<br>
glycoproteins known as bone morphogenetic proteins (BMPs). These proteins<br>
have been shown to be involved with embryonic dorsal-ventral patterning, limb<br>
bud development, and fracture repair in adult animals. B. L. Hogan, Genesjfe<br>
Develop., 10,1580 (1996). This group of transforming growth factor-beta<br>
superfamily secreted proteins has a spectrum of activities in a variety of cell types<br>
at different stages of differentiation; differences in physiological activity between<br>
these closely related molecules have not been clarified (D. M. Kingsley, Trends<br>
fienet,, 10,16(1994)).<br>
The effects of BMP-6, BMP-2 and BMP-4 on induction of rat calvarial<br>
osteoblast differentiation have been investigated ( Boden, et al., Endocrinology,<br>
137,3401 (1996)). In cultures of fetal rat calvaria that require BMP or<br>
glucocorticoid for initiation of differentiation, glucocorticoid provided a ten-fold<br>
induction of BMP-6 mRNA and protein expression which enhanced osteoblast<br>
differentiation (Boden, et al., Endocrinology. 138,2920 (1997)).<br>
BMPs have been investigated for the stimulation of bone formation in vivo.<br>
Despite the early successes achieved with BMPs and other extracellular signalling<br>
molecules, there are disadvantages to their use. For example, relatively large doses<br>
of purified BMPs are required to enhance the production of new bone, thereby<br>
increasing the expense of such treatment methods. Furthermore, extracellular<br>
proteins such as BMP are susceptible to degradation following their introduction<br>
into a host animal.<br>
Intracellular signals or regulatory molecules may also play a role in the<br>
osteinductive pathway. One class of intracellular regulatory molecules are the LIM<br>
proteins, which possess a characteristic structural motif known as the LIM domain.<br>
The LIM domain is a cysteine-rich structural motif composed of two special zinc<br>
fingers that are joined by a 2-amino acid spacer. Some proteins have only LIM<br>
domains, while others contain a variety of additional functional domains. LIM<br>
proteins form a diverse group, which includes transcription factors and cytoskeletal<br>
proteins. The primary role of LIM domains appears to be in mediating protein-<br>
protein interactions, through the formation of dimers with identical or different<br>
LIM domains, or by binding distinct proteins.<br>
In LIM homeodomain proteins (proteins having both LIM domains and a<br>
homeodomain sequence) the LIM. domains function as negative regulatory<br>
elements. LIM homeodomain.proteins are involved in the control of cell lineage<br>
determination and the regulation of differentiation, although LIM-only proteins<br>
may have similar roles;: LIM-only proteins are also implicated in the control of cell<br>
proliferation since several genes encoding such proteins are associated with<br>
oncogenic chromosome translocations.<br>
Humans and other mammalian species are prone to diseases or injuries that<br>
require the processes of bone repair and/or regeneration. Treatment of fractures,<br>
for example, would be improved by new treatment regimens that could stimulate<br>
the natural bone repair mechanisms, thereby reducing the time required for the<br>
fractured bone to heal. Individuals with systemic bone disorders, such as<br>
osteoporosis, could benefit from treatment regimens that produce systemic<br>
formation of new bone tissue. Such, treatment regimens could reduce the incidence<br>
of fractures arising from the loss of bone mass that is characteristic of the disease.<br>
It would be desirable to utilize treatment regimens that use an intracellular<br>
signaling molecule to induce .new bone formation. Gene therapy techniques make<br>
it possible to introduce nucleotide fragments encoding intraceUular signals that<br>
mediate bone formation into osteogenic precursor cells (cells involved in bone<br>
formation) or peripberal.blood leukocytes,. Gene therapy offers a number of<br>
potential advantages: (1) lower costs associated with production of a target therapy<br>
protein; (2) greater efficacy, compared to extracellular treatment regiments, due to<br>
the ability to achieve prolonged expression of the intracellular signal; (3) effect on<br>
target cells is not as limited fay virtue of the limited number of receptors available<br>
to interact with the therapy protein; (4) transfected osteoprogenitor cells can be<br>
delivered directly to the site where localized bone formation is required; and (5)<br>
therapy can be provided systemically, inducing systemic bone formation and<br>
providing a treatment regimen for osteoporosis and other metabolic bone diseases.<br>
Humans and other mammals are subject to intervertebral disc degeneration,<br>
-with associated low back pain, disc herniations, and spinal stenosis. Disc<br>
degeneration is associated with, a progressive loss of proteoglycan matrix -which<br>
may cause the disc to be more susceptible to bio-mechanical injury and<br>
degeneration. It would therefore be desirable to have a method of stimulating<br>
proteoglycan and/or collagen, synthesis by the appropriate cells, such as, for<br>
example, cells of the nucleous pulposus, cells of the annulus fibrosus, and cells of<br>
the intervertebral disc.<br>
SUMMARY OF THE INVENTION<br>
According to one aspect of the invention, a method of inducing the<br>
expression of one or more bone morphogenetic proteins or transforming growth<br>
factor-ß proteins (TGF-ps) in a cell is provided. The method includes transfecting<br>
a cell with an isolated nucleic acid comprising a nucleotide sequence encoding a<br>
LIM mineralization protein operably linked to a promoter. The expression of one<br>
or more proteins selected from the group consisting of BMP-2, BMP-4, BMP-6,<br>
BMP-7, TGF-ß1 and combinations thereof can be induced according to the method<br>
of the invention. The isolated nucleic acid according to this aspect of the invention<br>
can be a nucleic acid which can hybridize under standard conditions to a nucleic<br>
acid molecule complementary to the full length of SEQ. ID NO: 25; and/or a<br>
nucleic acid molecule which can hybridize under highly stringent conditions to a<br>
nucleic acid molecule complementary to the full length of SEQ. ID NO: 26. The<br>
cell can be any somatic cell such including, but not limited to, buffy coat cells,<br>
stem cells and intervertebral disc cells.<br>
According to a second aspect of the invention, a cell which overexpresses<br>
one or more bone morphogenetic proteins or transforming growth factor-fJ proteins<br>
is provided. The cell can be a cell which overexpresses one or more proteins<br>
selected from the group consisting of BMP-2, BMP-4, BMP-6, BMP-7, TGF-ß1<br>
and combinations thereof The cell can be a buffy coat cell, an intervertebral disc<br>
cell, a mesenchymal stem cell or apluripotential stem cell. An implant comprising<br>
a cell as set forth above and a carrier material is also provided. Also provided<br>
according to the invention is a method of inducing bone formation in a mammal<br>
comprising introducing a cell or an implant as set forth above, into the mammal and<br>
a method of treating intervertebral disc disease in a mammal comprising<br>
introducing a cell as set forth above into an intervertebral disc of the mammal.<br>
Additional advantages and novel features of the invention will be set forth<br>
in part in the description that follows, and in part will become more apparent to<br>
those skilled in the art upon examination of the following or upon learning by<br>
practice of the invention.<br>
/'ACCOMPANYING<br>
BRIEF DESCRIPTION OF THE£)RAWINGS<br>
Fig. 1 is a graph illustrating production of sulfated glycosaminoglycan<br>
(sGAG) after expression of HLMP-1 in rat intervertebral disc cells transfected with<br>
the indicated multiplicities of infection (MOIs);<br>
' Fig. 2 is a chart showing the dose response of rat intervertebral disc cells six<br>
days after infection with different MOI of AdHLMP-1;<br>
Fig. 3 is a chart showing the expression of Aggrecan and BMP-2 mRNA by<br>
AdHLMP-1 transfected rat intervertebral disc cells six days following transfection<br>
with an MOI of 250 virions/cell;<br>
Fig. 4A is a chart showing HLMP-1 mRKA expression 12 hours after<br>
infection with Ad-hLMP-1 at different MOIs;<br>
Fig. 4B is a chart showing the production of sGAG in medium from 3 to 6<br>
days after infection;<br>
Fig. 5 is a chart showing time course changes of the production of sGAG;<br>
Fig. 6A is a chart showing gene response to LMP-1 over-expression in rat<br>
annulus fibrosus cells for aggrecan<br>
Fig. 6B is a chart showing gene response to LMP-1 over-expression in rat<br>
annulus fibrosus cells for BMP-2;<br>
Fig. 7 is a graph showing the tune course of HLMP-1 mRNA levels in rat<br>
annulus fibrosus cells after infection with AdLMP-1 at MOI of 25;<br>
Fig. 8 is a chart showing changes in mRNA levels of BMPs and aggrecan in<br>
response to HLMP-1 over-expression;<br>
Fig. 9 is a graph showing the time course of sGAG production enhancement<br>
in response to HLMP-1 expression;<br>
Fig. 10 is a chart showing that the LMP-1 mediated increase in sGAG<br>
production is blocked fay noggin;<br>
Fig. 11 is a graph showing the effect of LMP-1 on sGAG in media after day<br>
6 of culture in monolayer.<br>
Figs. 12A-12D are photomicrographs of immunohistochemical staining for<br>
LMP-1 protein in A549 cells;<br>
Figs. 13A-13F are photomicrographs of immunohistochemical staining of<br>
A549 cells 48 hours after infection with AdLMP-1 (upper panels) or Adpgal (lower<br>
panels);<br>
Figs. 14A-14D are photomicrographs of immunohistochemical staining of<br>
A549 cells 48 hours after infection with either AdLMP-1 (upper panels) or AdPgai<br>
(lower panels);<br>
Figs. 15A-15D are photomicrographs of immunohistochemical staining for<br>
the leukocyte surface marker CD45 in human buffy coat cells infected with<br>
AdLMP-1 (upper panels) or Adpgal (lower panels) excised at 3 days (Figs. 1.5A<br>
and 15C) or 5 days (Figs. 15B and 15D) following implantation with a collagen<br>
matrix subcutaneously on the chest of an athymic rat;<br>
Figs. 16A-16D are photomicrographs of immunohistochemical staining for<br>
BMP-4 in human buffy coat cells infected with AdLMP-1 (upper panels) or Adpgal<br>
(lower panels) excised at 3 days (Figs. 16A and 16C) or 5 days (Figs. 16B and<br>
16D) following implantation with a collagen matrix subcutaneously on the chest of<br>
an athymic rat;<br>
Figs. 17A-17D are photomicrographs of immunohistochemical staining for<br>
BMP-7 in human buffy coat cells infected with AdLMP-1 (upper panels) or Adpgal<br>
(lower panels) excised at 3 days (Figs. 3.7A and 17C) or 5 days (Figs. 17B and<br>
17D) following implantation with a collagen matrix subcutaneously on the chest of<br>
an athymic rat;<br>
Fig. 18 is a high power photomicrograph of immunohistochemical staining<br>
for BMP-7 in human buffy coat cells infected with AdLMP-1 excised at 14 days<br>
following implantation with a collagen matrix subcutaneously on the chest of an<br>
athymic rat;<br>
Figs. 19A-19D are photomicrographs of human buffy coat cells infected<br>
with AdLMP-1 (upper panels) or Adßgal (lower panels) excised at 1 day<br>
(Figs. 19A and 19C) or 3 days (Figs. 19B and 19D) following implantation in a<br>
collagen matrix subcutaneously on the chest of an athymic rat;<br>
Figs. 20A and 20B are high power photomicrographs of human buffy coat<br>
cells infected with AdLMP-1 or AdPgal excised at 1 day following implantation in<br>
a collagen matrix subcutaneously on the chest of an athymic rat;<br>
Figs. 21A-21J are photomicrographs of human buffy coat cells infected<br>
with AdLMP-1 (upper panels-Figs. 21A-21E) or Adßgal (lower panels-Figs. 21F-<br>
21J) excised at various time points following implantation with a collagen, matrix<br>
subcutaneously on the chest of an athymic rat; and<br>
Figs. 22A-22.C are high power photomicrographs of human buffy coat cells<br>
infected with AdLMP-1. excised at various time points following implantation with<br>
a collagen matrix subcutaneously on the chest of an athymic rat.<br>
DETAILED DESCRIPTION OF THE PREFERRED EMBODIMENTS<br>
LMP-1 is a novel LIM domain protein associated with early osteoblast<br>
differentiation. LMP-1 transcripts are first detectable in mesenchymal cells<br>
adjacent to the hypertrophic cartilage cells in developing embryonic long bones just<br>
before osteoblasts appear at the center of the cartilage anlage (see Boden, et al,<br>
"LMP-1, A LM-Domain Protein, Mediates BMP-6 Effects on Bone Formation",<br>
Endocrinology. 139, 5125-5134 (1998)). The LMP-1 protein is a member of the<br>
heterogeneous family of LIM domain proteins, many of which are involved with<br>
growth and: differentiation in a variety of cell types. However, the precise<br>
mechanisms of action of LIM-domain proteins remain poorly understood. See<br>
Kong, et al., "Muscle LIM Protein Promotes Myogenesis by Enhancing the Activity<br>
of MyoD.", Mol.Cell.Biol.. 17,4750-4760 (1997); Sadler, et al., "Zyxin and<br>
cCRP: Two Interactive LIM Domain Proteins Associated with the Cytoskeleton", J.<br>
Cell Biol., 119,1573-1587 (1992); Salgia, et al., "Molecular Cloning of Human<br>
Paxillin, a Focal Adhesion Protein Phosphorylated by P210(BCCR/ABL)", J. Biol.<br>
Chem., 270,5039-5047 (1995); and Way, et al., "Mec-3, A Homeobox-Containing<br>
Gene that Specifies the Differentiation of the Touch Receptor Neurons in C.<br>
Elegans", Cell, 54, 5-16 (1988).<br>
Although LMP-1 is a LIM domain protein, it has recently been shown that<br>
the LIM domains themselves are not necessary for osteoblast differentiation (see<br>
Liu, et al., "Overexpressed LIM Mineralization Proteins do not Require LIM<br>
Domains to Induce Bone", J. Bone Mm. Res., 17, 406-414 (2002)). LMP-1 is<br>
thought to be a potent intracellular signalling molecule that is capable, at very low<br>
doses, of inducing osteoblast differentiation in vitro and de novo bone formation in<br>
vivo (Boden, et al., Endocrinology. 139, 5125-5134 (1998)).<br>
Results from two separate experimental systems indicate that LMPr1<br>
induces the expression of several BMPs. BMP-4 and BMP-7 can be detected as<br>
early as 48 hours after insertion of the LMP-1 cDNA in vitro and 72 hours in vivo.<br>
In vivo studies showed that most of the implanted buffy coat cells expressing LMP-<br>
1 survived for less than a week in vivo, but there was evidence of an influx of host<br>
cells that differentiated into bone forming cells. Results also indicate that LMP-1<br>
induces membranous bone formation without a clear cartilage interphase, which is<br>
common with many of the BMPs.<br>
The inventors have also shown that cells treated with AdLMP-1 produced<br>
LMP-1, BMP-2, and to lesser extent BMP-6 and,TGF-ß1 protein in vitro. BMP-4<br>
and BMP-7 are additional secreted osteoinductive factors induced by LMP-1.<br>
Antisense oligonucleotide experiments performed by the inventors indicate that<br>
BMP-4 and BMP-7 are necessary for intracellular LMP-1 to exert its<br>
osteoinductive effects on other cells.<br>
The A549 experiments described below show that the BMPs were not<br>
induced by the adenovirus itself nor were the BMPs expressed in untreated the<br>
cells. These experiments also show that two proteins not related to osteoblast<br>
differentiation (i.e., type H collagen and MyoD) were not induced by LMP-1.<br>
A549 lung carcinoma cells were chosen because the A549 cells, unlike .<br>
osteoblasts, have no basal expression of BMPs.<br>
The use of buffy coat cells from ordinary venous blood for ex vivo gene<br>
therapy is relatively new. See Viggeswarapu, et al., "Adenoviral Delivery of LIM<br>
Mineralization Protein-1 Induces New-Bone Formation in vitro and in v/vo", J.<br>
Bone Joint Sure. Amu 83-A, 364-376 (2001). To determine how long buffy coat<br>
cells transfected with LMP-1 cDNA survive in vivo and enhance synthesis,<br>
secretion and activity of BMPs, the CD-45 antigen was used.(see Kurtin, et al.,<br>
"Leukocyte Common Antigen-A Diagnostic Discriminant Between Hematopoietic<br>
and Nonhematopoietic Neoplasms in Paraffin Sections using Monoclonal<br>
Antibodies: Correlation with Immunologic Studies and Ultrastructural<br>
Localization", Hum.Pathol.. 16.353-365 (1985); and Pulido, et al., "Comparative<br>
Biochemical and Tissue Distribution Study of Four Distinct CD45 Antigen<br>
Specificities", J. Immunol.. 140,3851-3857 (1988). The number of cells<br>
specifically reacting with the anti-CD-45 primary antibody decreased progressively<br>
and was minimal by 10 days following implantation. The loss of anti-CD-45<br>
staining, the dropout of cells in the center of the implant by seven days, and the<br>
centripetal pattern of bone formation all suggested that the transplanted cells,<br>
including those expressing the LMP-1 cDNA, did not survive for long periods of<br>
time, suggesting that LMP-expressing cells may participate indirectly in the bone<br>
formation process through induction of secreted factors that subsequently recruit<br>
host progenitor cells and modulate their differentiation into mature osteoblasts.<br>
Evidence indicates that LMP-1 starts a cascade of events, including secretion of<br>
several osteoinductive proteins (BMPs). Therefore LMP-1 is an ideal therapeutic<br>
candidate as LMP-1 can exert significant effects without being expressed in many<br>
cells persist for long periods of time in vivo.<br>
The inventors have demonstrated bone induction by ex vivo gene transfer<br>
of LMP-1 cDNA to peripheral blood buffy coat cells implanted ectopically. The<br>
present invention therefore relates to the transfection of non-osseous cells with<br>
nucleic acids encoding LIM mineralization proteins. The inventors have<br>
discovered that transfection of non-osseous cells such as intervertebral disc cells<br>
with nucleic acids encoding LIM mineralization proteins can result in the increased<br>
synthesis of proteoglycan, collagen and other intervertebral disc components and<br>
tissue. The invention therefore provides a method for treating intervertebral disc<br>
disease associated with the loss of proteoglycan, collagen, or other intervertebral<br>
disc components.<br>
The inventors previously isolated a LIM Mineralization Protein (LMP)<br>
cDNA sequence (10-4/RLMP) from stimulated rat calvarial osteoblast cultures<br>
(SEQ. ID NO: 1, SEQ. ID NO: 2) (U.S. Patent No. 6,300,127). The gene has been<br>
cloned, sequenced and assayed for its ability to enhance the efficacy of bone<br>
mineralization in vitro. The protein, RLMP, has been found to affect the<br>
mineralization of bone matrix as well as the differentiation of cells into the<br>
osteoblast lineage. Unlike other known cytokines such as bone mineralization<br>
proteins (BMPs), RLMP is not a secreted protein, but is instead an intracellular<br>
signaling molecule. Thereforei smaller quantities of protein can lead to-.<br>
intracellular signaling amplification and more specificity for in vivo applications.<br>
Suitable clinical applications include enhancement of bone repair in fractures, bone<br>
defects, bone grafting, and normal homeostasis in patients presenting with<br>
osteoporosis.<br>
The amino acid sequence of a corresponding human protein, named human<br>
LMP-1 ("HLMP1"), has also been cloned, sequenced and deduced (U.S. Patent No.<br>
6,300,127). The human protein demonstrated enhanced efficacy of bone<br>
nuneralizatin in vitro and in vivo.<br>
A truncated (short) version of HLMP-1, termed HLMP-ls, has also been<br>
characterized by. the inventors (see U.S. Patent No. 6,300,127). This protein is the<br>
result of a point mutation that produces a stop codon and gives a truncated protein.<br>
HLMP-ls is fully functional when expressed in cell culture and in vivo.<br>
Using PCR analysis of a human heart cDNA library, two alternative splice<br>
variants (HLMP-2 and HLMP-3) have been identified (U.S. Patent Application<br>
Serial No. 09/959,578, filed April 28,2000). The nucleotide sequences of these<br>
proteins differ from HLMP-1 in the 325-444 base pair region of the HLMP-1<br>
sequence. The HLMP-2 sequence has a 119 base pair deletion and an insertion of<br>
17 base pairs in this region. Compared to HLMP-1, the nucleotide sequence<br>
encoding HLMP-3 has no deletions, but it does have the same 17 base pairs<br>
insertion as HLMP-2, these base pairs being inserted at a position corresponding to<br>
base pair 444 in the HLMP-1 sequence,<br>
LMP regulates or influences a number of biological processes, so different<br>
splice variants of LMP are expected to have different biological functions in<br>
mammals, such as growth, differentiation, and/or regeneration of various tissues.<br>
For example, some form of LMP is expressed not only in bone, but also in muscle,<br>
tendons, ligaments, spinal cord, peripheral nerves, and cartilage.<br>
The present invention provides a method of stimulating proteoglycan or<br>
collagen synthesis, or both, in a mammalian cell by providing an isolated nucleic<br>
acid comprising a nucleotide sequence encoding LIM mineralization protein<br>
operably linked to a promoter, transfecting the isolated nucleic acid sequence into a<br>
mammalian cell capable of producing proteoglycan; and expressing the nucleotide<br>
sequence encoding LIM mineralization protein, thereby stimulating proteoglycan<br>
synthesis. The mammalian cell can be a non-osseous cell, such as an intervertebral<br>
disc cell, a cell of the annulus fibrosus, or a cell of the nucleus pulposus.<br>
Transfection can occur either ex vivo or in vivo by direct injection of virus or naked<br>
DNA, such as, for example, a plasmid. In certain embodiments, the virus is a<br>
recombinant adenovirus, preferably AdHLMP-1.<br>
Another embodiment of the invention comprises a non-osseous mammalian<br>
cell comprising an isolated nucleic acid sequence encoding a LIM mineralization<br>
protein. The non-osseous mammalian cell may be a stem cell (e.g., a pluripotential<br>
stem cell or a mesenchymal stem cell) or an intervertebral disc cell, preferably a<br>
cell of the nucleus.pulposus or a cell of the annulus fibrosus..<br>
In a different aspect, the invention is directed to a method of expressing an<br>
isolated nucleotide sequence encoding LIM mineralization protein in a non-osseous<br>
mammalian cell, comprising providing an isolated nucleic acid comprising a<br>
nucleotide sequence encoding LIM mineralization protein operably linked to a<br>
promoter; transfecting said isolated nucleic acid sequence into a non-osseous<br>
mammalian cell; and expressing said nucleotide sequence encoding LIM<br>
mineralization protein. The non-osseous mammalian cell may be a stem cell or an<br>
intervertebral disc cell (e.g., a cell of the nucleus pulposus or annulus fibrosus).<br>
Transfection may occur either ex vivo or in vivo by direct injection of virus or<br>
naked DNA, such as, for. example, a plasmid. The virus can be a recombinant<br>
adenovirus, preferably AdHLMP-1.<br>
In yet another embodiment, the invention is directed to a method of treating<br>
intervertebral disc disease by reversing, retarding or slowing disc degeneration by<br>
providing an isolated nucleic acid comprising a nucleotide sequence encoding LIM<br>
mineralization protein operably linked to a promoter; transfecting the isolated<br>
nucleic acid sequence into a mammalian cell capable of producing proteoglycan;<br>
and stimulating proteoglycan synthesis in the cell by expressing the nucleotide<br>
sequence encoding LIM mineralization protein, thereby reversing or inhibiting disc<br>
degeneration. The disc disease may produce lower back pain, disc hemiation. or<br>
spinal stenosis, and the method can ameliorate these symptoms accordingly. The<br>
mammalian cell may be a non-osseous cell, such as a stem cell or an intervertebral<br>
disc cell (e.g., a cell of the annulus fibrosus, or a cell of the nucleus pulposus).<br>
Transfection may occur either ex vivo or in vivo by direct injection of virus<br>
or naked DNA, such as, for example, a plasmid. In certain embodiments, the virus<br>
is a recombinant adenovirus, preferably AdHLMP-1.<br>
The present invention relates to novel mammalian LIM proteins, herein<br>
designated LIM mineralization proteins, or LMPs. The invention relates more<br>
particularly to human LMP, known as HLMP or HLMP-1, or alternative splice<br>
variants of human LMP, which are known as HLMP-2 or HLMP-3. The inventors<br>
have discovered that these proteins enhance bone mineralization in mammalian<br>
cells grown in vitro. When produced in mammals, LMP also induces bone<br>
formation in vivo.<br>
Ex vivo transfection of bone marrow cells, osteogenic precursor cells,<br>
peripheral blood cells, and stem cells (e.g., pluripotential stem cells or<br>
mesenchymal stem cells) with nucleic acid that encodes a LIM mineralization<br>
protein (e.g., LMP or HLMP), followed by reimplantation of the transfected cells in<br>
the donor, is suitable for treating a variety of bone-related disorders or injuries. For<br>
example, one can use this method to augment long bone fracture repair, generate<br>
bone in segmental defects, provide a bone graft substitute for fractures, facilitate<br>
tumor reconstruction or spine fusion, and provide a local treatment (by injection)<br>
for weak or osteoporotic bone, such as in osteoporosis of the hip, vertebrae, or<br>
wrist. Transfection with LMP or HLMP-encoding nucleic acid is also useful in<br>
percutaneous injection of transfected marrow cells to accelerate the repair of<br>
fractured long bones; treatment of delayed union or non-union of long bone<br>
fractures or pseudoarthrosis of spine fusions, and for inducing new bone formation<br>
in avascular necrosis of the hip or knee.<br>
In addition to ex vivo methods of gene therapy, transfection of a<br>
recombinant DNA vector comprising a nucleic acid sequence that encodes LMP. or<br>
HLMP can be accomplished in vivo. When a DNA fragment that encodes LMP or<br>
HLMP is inserted into an appropriate viral vector such as, for example, an<br>
adenoviral vector, the viral construct can be injected directly into the site were<br>
endochondral bone formation is desired. By using a.direct, percutaneous injection<br>
to introduce the LMP or HLMP sequence, stimulation of bone formation can be<br>
accomplished without surgical intervention either to obtain bone marrow cells (to<br>
transfect ex vivo) or to reimplant them into Hie patient at the site where new bone is<br>
required. Alden, et al.(Neurosurgical Focus (1998)) have demonstrated the utility<br>
of a direct injection method of gene therapy using a BMP-2 cDNA cloned into an<br>
adenovirus vector.<br>
It is also possible to carry out in vivo gene therapy by directly injecting into<br>
an appropriate body site, a naked, or un-encapsulated, recombinant plasmid<br>
comprising a nucleic acid sequence that encodes HLMP. In this embodiment of the<br>
invention, transfection occurs when the naked plasmid DNA is taken up, or<br>
internalized, by the target cells. As in the case of in vivo gene therapy using a viral<br>
construct, direct injection of naked DNA offers the advantage that little or no<br>
surgical intervention is required. Direct gene therapy, using naked DNA that<br>
encodes the endothelial cell mitogen VEGF (vascular endothelial growth factor),<br>
has been successfully demonstrated in human patients by Baumgartner<br>
(Circulation. 97,12,1114-1123 (1998)).<br>
For intervertebral disc applications, ex, vivo transfection can be<br>
accomplished by harvesting cells from an intervertebral disc, transfecting the cells<br>
with nucleic acid encoding LMP in vitro, and introducing the cells into an<br>
intervertebral disc. The cells may be harvested from, or introduced back into, the<br>
intervertebral disc by any means known to those of skill in the art, such as, for<br>
example, any surgical techniques appropriate for the spine. In one embodiment, the<br>
cells are introduced into the intervertebral disc by injection.<br>
Also according to the invention, stem cells (e.g., pluripotential stem cells or<br>
mesenchymal stem cells) can be transfected with nucleic acid encoding a LIM<br>
Mineralization Protein ex vivo and introduced into the intervertebral disc, for<br>
example, by injection.<br>
The cells transfected ex vivo can also be combined with a carrier to form an<br>
intervertebral disc implant. The carrier comprising the transfected cells can then be<br>
implanted into the intervertebral disc of a subject Suitable carrier materials have<br>
been previous described. (See, for example, Helm, et al., "Bone Graft Substitutes<br>
for the Promotion of Spinal Arthrodesis", Neurosurg Focus. 10 (4) (2001). The<br>
carrier preferably comprises a biocompatible porous matrix such as a demineralized<br>
bone matrix (DBM), a biocompatible synthetic polymer matrix, or a protein matrix.<br>
Suitable proteins include, for example, extracellular matrix proteins such as<br>
collagen. Cells transfected with LMP ex vivo can be incorporated into the carrier .<br>
(i.e., into the pores of the porous matrix) prior to implantation.<br>
Similarly, for intervertebral disc applications where the cells are transfected<br>
in vivo, the DNA may be introduced into the intevertebral disc using any suitable<br>
method known to those of skill in the art. In one embodiment, the nucleic acid is<br>
directly injected into the intervertebral space.<br>
Since adenovirus does not incorporate into the genome of infected cells,<br>
transient expression of LMP is achieved when an adenovirus vector is used to<br>
deliver LMP to osteogenic cells. Transient expression, however, is sufficient to<br>
achieve the objects of the invention. Stable expression of LMP, however, can be "<br>
achieved by use of a vector that incorporates into the genome of the target cell.<br>
Retroviral vectors, for example, are suitable for this purpose.<br>
Stable expression of LMP is particularly, useful for treating, various systemic<br>
bone-related disorders, such as osteoporosis and osteogenesis imperfecta. For this<br>
embodiment of the invention, a regulatable promoter can be combined with the<br>
polynucleotide sequence of LMP for incorporation into the viral vector. Such a<br>
promoter can comprise a sequence that is controlled by exposure to an exogenous<br>
inducing agent such as, for example, tetracycline.<br>
Using this approach, stimulation of systemic new bone formation is<br>
achieved by administering an effective amount of the exogenous inducing agent<br>
Once the desired bone mass is achieved, administration of the exogenous inducing<br>
agent can be discontinued. This process may be repeated as needed to replace bone<br>
loss, for example, as a consequence of osteoporosis.<br>
Antibodies specific for HLMP are particularly suitable for use in methods<br>
for assaying the osteoinductive or bone-forming potential of patient cells, providing<br>
a means for identifying patients at risk for slow or impaired bone repair. Also,<br>
HLMP-specific antibodies are suitable for use in marker assays to identify risk<br>
factors in bone degenerative diseases, such as, for example, osteoporosis.<br>
Following well known and conventional methods, the gene therapy vectors<br>
of the present invention are prepared by ligation of polynucleotide sequences that<br>
encode LMP to nucleic acid sequences comprising cloning or expression vectors.<br>
Preferred vectors provide a means to both clone and express the DNA sequence of<br>
LMP. Methods needed to construct and analyze these recombinant vectors are well<br>
known to those of skill in the art of molecular biology, and are described, for<br>
example, in Sambrook, et al., Molecular Cloning: A Laboratory Manual. 2nd<br>
edition, Cold Spring Harbor Press, (1988), Davis, et al., Basic Methods in<br>
Molecular Biology. Elsevier (1986), and Ausubel, et al., Current Protocols in<br>
Molecular Biology, Wiley Interscience (1988).<br>
The polymerase chain reaction which provides a means for amplifying the<br>
LMP cDNA sequence has been described in U.S. Patent No. 4,800,159 (Mullis, et<br>
al). Kits for DNA amplification are commercially available, and comprise the<br>
necessary enzymes and related reagents to prepare multiple copies of a cDNA<br>
sequence from a sample, of limited quantity.<br>
A LIM mineralization protein expression vector can comprise any<br>
polynucleotide sequence that provides a template for expression of a LIM<br>
Mineralization Protein having bone forming activity. Conservative amino acid<br>
substitutions, or other modifications, such as the occurrence of an ammo-terminal<br>
methionine residue, are also within the scope of the present invention, as these<br>
substitutions and modifications are well within the skill of one in the art.<br>
A ribosomal binding site related to the host expression system of choice is<br>
ligated to the 51 end of the chimeric LMP coding sequence, forming a synthetic<br>
gene which can be inserted into an expression vector. A regulatable promoter, for<br>
example, the E. coli lac promoter, can also be provided for expression of the<br>
chimeric coding sequences. Other suitable regulatable promoters include, for<br>
example, trp, tac, recA, T7 and lambda promoters.<br>
DNA encoding LMP can be transfected into recipient cells by any means<br>
known to those of skill in the art such as, for example, calcium phosphate<br>
precipitation, DEAE-Dextran, electroporation or protoplast fusion, to form stable<br>
transformants. Calcium phosphate precipitation can be performed according to the<br>
method of Graham, et al.C Virology. 52,456 {1913)). Briefly, an aliquot of 40-50<br>
micrograms of DNA, with salmon sperm or calf thymus DNA as carrier, is used per<br>
0.5 x 106 cells plated on a 100 mm dish. The DNA is admixed with 0.5 ml of 2X<br>
Hepes solution (280 mM NaCl, 50 mM Hepes and 1.5 mM Na2HPO4, pH 7.0), to<br>
which an equal volume of 2x CaCl2 (250 mM CaCl2 and 10 mM Hepes, pH 7.0) is<br>
added. A white granular precipitate, appearing after 30-40 minutes, is evenly<br>
distributed dropwise on the cells, which are allowed to incubate for 4-16 hours at<br>
37°C. The medium is removed and the cells exposed to 15% glycerol in PBS for 3<br>
minutes. After the glycerol is removed, the cells are fed with Dulbecco's Minimal<br>
Essential Medium (DMEM) containing 10% fetal bovine serum.<br>
DNA can also be transfected using: the DEAE-Dextran methods of<br>
Kimura, et al. (Virology. 49:394 (1972)) and Sompayrac, et al.'.(Proc. Natl. Acad.<br>
Sci. USA. 78, 7575 (1981)), the electroporation method of Potter ( Proc. Natl. Acad.<br>
Sci. USA. 81,7161 (1984)), or the protoplast fusion method described by Sandri-<br>
Goddin, et al. (Molec. Cell.Biol.. 1, 743 (1981)).<br>
The present invention also includes nucleic acid molecules that hybridize<br>
under standard conditions to any of the nucleic acid sequences encoding the LIM<br>
mineralization proteins of the invention, or sequences complementary thereto.<br>
"Standard hybridization conditions" will vary with the size of the probe, the<br>
background and the concentration of the nucleic acid reagents, as well as the type of<br>
hybridization. For example, in situ, Southern blot, or hybrization of DNA-KNA<br><br>
hybrids (Northern blot) can be used. The determination of "standard hybridization<br>
conditions" is within the level of skill in the art. Such conditions are described, for<br>
example, in U.S. Patent 5,580,775 (Fremeau, et ah), by Southern in J. Mol. Biol..<br>
98:503 (1975), by Alwine, et al., Meth. EnzvmoL 68:220 (1979), and Sambrook, et<br>
al., Molecular Cloning: A Laboratory Manual. 2nd edition, Cold Spring Harbor<br>
Press, 7.19-7.50(1989).<br>
One preferred set of standard hybrization conditions provides<br>
prehybridization at 42°C for2 hours in 50% formamide, 5X SSPE (150 nM NaCl,<br>
10 mMNaH2PO4 (pH 7.4],:1 mM EDTA {pH 8.0])l 5XDenhardt's solution-(20mg<br>
Ficoll, 20 mg polyvinylpyrrolidone and 20 mg BSA per 100 ml water), 10% dextran<br>
sulphate, 1% SDS and 100 micrograms/ml salmon sperm DNA. A 32P- labeled ¦<br>
cDNA probe is added, and hybridization is allowed to proceed for 14 hours.<br>
Afterward, the blot is washed twice with 2X SSPE, 0.1 % SDS for 20 minutes at<br>
22°C, 0.1X SSPE, ,0.1 %SDS at 65°C for 1 hour. The blot is then dried and exposed<br>
to x-ray film for 5 days in the presence of an intensifying screen.<br>
Under "highly stringent conditions," a probe will hybridize to its target<br>
sequenceif those two sequences are substantially identical. As with standard<br>
hybridization conditions, highly stringent conditions are determined according to the<br>
specific hybridization purposes of one of skill in the art.<br>
According to one aspect of the present invention,.an isolated nucleic acid<br>
molecule comprising a nucleic acid sequence encoding a LIM mineralization protein<br>
is provided. The nucleic acid molecule according to the invention can be a molecule<br>
which hybridizes under standard conditions to a nucleic acid molecule<br>
complementary to the full length of SEQ. ID NO: 25, which hybridizes under highly<br>
stringent conditions to a nucleic acid molecule complementary to the full length of<br>
SEQ. ID NO: 26, or which hybridizes to both. More specifically, the isolated<br>
nucleic acid molecule according to the invention can encode HLMP-1, HLMP-ls,<br>
RLMP, HLMP-2, or HLMP-3.<br>
Another aspect of the invention includes the proteins encoded by the nucleic<br>
acid sequences. In still another embodiment, the invenrionrelates to the<br>
identification of such proteins based on anti-LMP antibodies. In this embodiment,<br>
protein samples are prepared for Western blot analysis by lysirig cells and separating<br>
the proteins by polyacrylamide gel electrophoresis (SDS-PAGE). Proteins are<br>
transferred to nitrocellulose by electroblotting as described by Ausubel, et al.,<br>
Current Protocols in Molecular Biology, John Wiley and Sons (1987). After<br>
blocking the filter with instant nonfat dry milk (1 gm in 100 ml PBS), anti-LMP<br>
antibody is added to the filter and incubated for 1 hour at room temperature. The<br>
filter is washed thoroughly with phosphate buffered saline (PBS) and incubated with<br>
horseradish peroxidase (HRPO)-antibody conjugate for 1 hour at room temperature.<br>
The filter is again washed thoroughly with PBS and the antigen bands are identified<br>
by adding diaminobenzidine (DAB).<br>
Monospecific antibodies are the reagent of choice in the present invention,<br>
and are specifically used to analyze patient cells for specific characteristics<br>
associated with the expression of LMP. "Monospecific antibody" as used herein is<br>
defined as a one or more antibody species with homogenous binding characteristics<br>
for IMP. "Homogeneous binding" as used herein refers to the ability of the<br>
antibody species to bind to a specific antigen or epitope, such as those associated<br>
with LMP, as described above. Monospecific antibodies to IMP are purified from<br>
mammalian antisera containing antibodies reactive against LMP or are prepared as<br>
monoclonal antibodies reactive with LMP using the technique described by Kohler,<br>
et al. (Nature. 256,495-497 (1975). The LMP-specific antibodies are produced by<br>
immunizing animals such as, for example, mice, rats, guinea pigs, rabbits, goats or<br>
horses, with an.appropriate concentration of LMP either with or without an immune<br>
adjuvant. Pre-immune serum collected prior to the first immunization. Each animal<br>
receives about 0.1 mg to about 1000 mg of LMP, and, if desired, an acceptable<br>
immune adjuvant. Acceptable adjuvants include, but are not limited to, Freund's<br>
complete, Freund's incomplete, alum-precipitate, water in oil emulsion containing<br>
Corynebacterium parvum, and tRNA adjuvants. The initial immunization consists<br>
of LMP in, preferably, Freund's complete adjuvant injected at multiple sites either<br>
subcutaneously.(SC), intraperitoneally (IP) or both. Each animal is bled at regular<br>
intervals, preferably, weekly, to determine antibody titer. The animals may or may<br>
not receive booster injections following the initial immunization. Those animals<br>
receiving booster injections are generally given an equal, amount of the antigen in<br>
Freund's incomplete adjuvant by the same route. Booster injections are given at<br>
about three week intervals until maximal titers are obtained. At about 7 days after<br>
each booster immunization or about weekly after a single immunization, the animals<br>
are bled, the serum collected, and aliquots are stored at about -20°C.<br>
Monoclonal antibodies (mAb) reactive with LMP are prepared by<br>
immunizing inbred mice, preferably Balb/c mice, with LMP. The mice are<br>
immunized by the IP or SC route with about 0.1 mg to about 10 mg, preferably<br>
about 1 mg, of LMP in about 0.5 ml buffer or saline incorporated in an equal<br>
volume of an acceptable adjuvant, as described above. Freund's complete adjuvant<br>
is preferred. The mice receive an initial immunization on day 0 and are rested for<br>
about 3-30 weeks. Immunized mice are given one or more booster immunizations<br>
of about 0.1 to about 10 mg of LMP in a buffer solution such as phosphate buffered<br>
saline by the intravenous (TV) route. Lymphocytes from antibody-positive mice,<br>
preferably splenic lymphocytes, are obtained by removing the spleens from<br>
immunized mice by standard procedures known in the art. Hybridoma cells are<br>
produced by mixing the splenic lymphocytes with an appropriate fusion partner,<br>
preferably myeloma cells, under conditions which will allow the formation of stable<br>
hybridomas. Fusion partners may include, but are not limited to: mouse myelomas<br>
P3/NSl/Ag 4-1; MPC-11; S-194 and Sp 2/0, with Sp 2/0 being preferred. The<br>
antibody producing cells and myeloma cells are fused in polyethylene glycol, about<br>
1,000 molecular weight, at concentrations from about 30% to about 50%. Fused<br>
hybridoma cells are selected by growth in hypoxanthine, thymidine and aminopterin<br>
(HAT) in supplemented Dulbecco's Modified Eagles Medium (DMEM) by<br>
procedures known in the art. Supernatant fluids are collected from growth positive<br>
wells on about days 14,18, and 21, and are screened for antibody production by an<br>
immunoassay such as solid phase immunoradioassay (SPIRA) using LMP as the<br>
antigen. The culture fluids are also tested in the Ouchterlony precipitation assay to<br>
determine the isotype of the mAb. Hybridoma cells from antibody positive wells<br>
are cloned by a technique such as the soft.agar technique described by MacPherson,<br>
(Soft Agar Techniques: Tissue Culture Methods and Applications. Kruse and<br>
Paterson (eds.), Academic Press (1973)) or Harlow, et al., Antibodies: A Laboratory<br>
Manual, Cold Spring Laboratory (1988).<br>
. Monoclonal antibodies may also be produced in vivo by injection of<br>
pristane- primed Balb/c mice, approximately 0.5 ml per mouse, with about 2x106 to<br>
about 6x106 hybridoma cells about 4 days after priming. Ascites fluid is collected at<br>
approximately 8-12 days after cell transfer and the monoclonal antibodies are<br>
purified by techniques known in the art<br>
In vitro production in anti-LMP mAb is carried out by growing the<br>
hydridoma cell line in DMEM containing about 2% fetal calf serum to obtain<br>
sufficient quantities of the specific mAb. The mAb are purified by techniques<br>
known in the art.<br>
Antibody titers of ascites or hybridoma culture fluids are determined by<br>
various serological or immunological assays, which include, but are not limited to,<br>
precipitation, passive agglutination, enzyme-linked immunosorbent antibody<br>
(ELISA) technique and radioimmunoassay (RIA) techniques. Similar assays are<br>
used to detect the presence of the LMP. in body fluids or tissue and cell extracts.<br>
It is readily apparent to those skilled in the art that the above described<br>
methods for producing monospecific antibodies may be utilized to produce<br>
antibodies specific for polypeptide fragments of LMP, full-length nascent LMP<br>
polypeptide, or valiants or alleles thereof.<br>
In another embodiment, the invention is directed to alternative splice<br>
valiants of HLMP-1. PCR analysis of human heart cDNA revealed mRNA for two<br>
HLMP alternative splice variants, named HLMP-2 and HLMP-3, that differ from<br>
HLMP-1 in a region between base pairs 325 and 444 in the HLMP-1 sequence. The<br>
HLMP-2 sequence has a 119 base pair deletion and an insertion of 17 base pairs in<br>
this region. These changes preserve the reading frame, resulting in a 423 amino acid<br>
protein, which compared to HLMP-1, has a net loss of 34 amino acids (40 amino<br>
acids deleted plus 6 inserted amino acids). HLMP-2 contains the c-terminal LIM<br>
domains that are present in HLMP-1.<br>
Compared to HLMP-1, HLMP-3 has no deletions, but it does have the same<br>
17 base pair insertion at position 444. This insertion shifts the reading frame,<br>
causing a stop codon at base pairs 459-461. As a result, HLMP-3 encodes a protein<br>
of 153 amino acids. This protein lacks the c-terminal LIM domains that are present<br>
in HLMP-1 and HLMP-2. The predicted size of the proteins encoded by HLMP-2<br>
and HLMP-3 was confirmed by western blot analysis.<br>
PCR analysis of the tissue distribution of the three splice variants revealed<br>
that they are differentially expressed, with specific isoforms predominating in<br>
different tissues. HLMP-1 is apparently the predominant form expressed in<br>
leukocytes, spleen, lung, placenta, and fetal liver. HLMP-2 appears to be the<br>
predominant isoform in skeletal muscle, bone marrow, and heart tissue. HLMP-3,<br>
however, was not the predominant isoform in any tissue examined.<br>
Over-expression of HLMP-3 in secondary rat osteoblast cultures induced<br>
bone nodule formation (287±56) similar to the effect seen for glucicorticoid (272±7)<br>
and HLMP-1 (232±200). Since HLMP-3 lacks the C-terminal LIM domains, there<br>
regions are not required for osteoinductive activity.<br>
Over-expression of HLMP-2, however, did not induce nodule formation<br>
(11±3). These data indicate that the amino acids encodedby the deleted 1,19 base...<br>
pairs are necessary for osteoinduction. The data also indicate that the distribution of<br>
HLMP splice variants can be important for tissue-specific function. Surprisingly,<br>
the inventors have shown that HLMP-2 inhibits steroid-induced osteoblast<br>
formation in secondary rat osteoblast cultures. Therefore, HLMP-2 may have<br>
therapeutic utility in clinical situations where bone formation is not desirable.<br>
On July 22,1997, a sample of 10-4/RLMP in a vector designated<br>
pCMV2/RLMP (which is vector pRc/CMV2 with insert 10-4 clone/RLMP) was<br>
deposited with the American Type Culture Collection (ATCC), 12301 Parklawn<br>
Drive, Rockville, MD 20852. The culture accession number for that deposit is<br>
209153. On March 19,1998, a sample of the vector pHis-A. with insert HLPM-ls<br>
was deposited at the American Type Culture Collection ("ATCC"). The culture<br>
accession number for that deposit is 209698. On April 14,2000, samples of<br>
plasmids pHAhLMP-2 (vector pHisA with cDNA insert derived from human heart<br>
muscle cDNA with HLMP-2) and pHAhLMP-3 (vector pHisA with cDNA insert<br>
derived from human heart muscle cDNA with HLMP-3) were deposited with the<br>
ATCC, 10801 University Blvd., Manassas, VA, 20110-2209, USA, under the<br>
conditions of the Budapest treaty. The accession numbers for these deposits are<br>
PTA-1698 and PTA-1699, respectively. These deposits, as required by the<br>
Budapest Treaty, will be maintained in the ATCC for at least 30 years and will be<br>
made available to the public upon the grant of a patent disclosing them. It should be<br>
understood that the availability of a deposit does not constitute a license to practice<br>
the subject invention in derogation of patent rights granted by government action.<br>
In assessing the nucleic acids, proteins, or antibodies of the invention,<br>
enzyme assays, protein purification, and other conventional biochemical methods<br>
are employed. DNA and RNA are analyzed by Southern blotting and Northern<br>
blotting techniques, respectively. Typically, the samples analyzed are size<br>
fractionated by gel electrophoresis. The DNA or RNA in the gels are then<br>
transferred to nitrocellulose or nylon membranes. The blots, which are replicas of<br>
sample patterns in the gels, were then hybridized with probes. Typically, the probes<br>
are radio-labeled, preferably with 32P, although one could label the probes with<br>
other signal-generating molecules known to those in the art. Specific bands of<br>
interest can then be visualized by detection systems, such as autoradiography.<br>
For purposes of illustrating preferred embodiments of the present invention,<br>
the following, non-limiting examples are included. These results demonstrate the<br>
feasibility of inducing or enhancing the formation of bone using the LIM<br>
mineralization proteins of the invention, and the isolated nucleic acid molecules<br>
encoding those proteins.<br>
EXAMPLE 1: Calvarial Cell Culture<br>
Rat calvarial cells, also known as rat osteoblasts ("ROB"), were obtained<br>
from 20-day pre-parturition rate as previously described by Boden, et al.<br>
(Endocrinology. 137, 8,3401-3407 (1996)). Primary cultures were grown to<br>
confluence (7 days), trypsinized, and passed into 6-well plates (1 x 10s cells/35 mm<br>
well), as first subculture cells. The subculture cells, which were confluent at day 0,-<br>
were grown for an additional 7 days. Beginning on day 0, media were changed and<br>
treatments. (Trm and/or BMPs) were applied, under a laminar flow hood, every 3 or<br>
4 days. The standard.culture protocol was as follows: days 1-7, MEM, 10% FBS,<br>
50 µg/ml ascorbic acid, ± stimulus; days 8-14, BGJb medium, 10% FBS,- 5mM P-<br>
GlyP (as a source of inorganic phosphate to permit mineralization). Endpoint<br>
analysis of bone nodule formation and osteocalcin secretion was performed, at<br>
day 14. The dose of BMP was chosen as 50 ng/ml based on pilot experiments in<br>
this system that demonstrated a mid-range effect on the dose-response curve for all<br>
BMPs studied.<br>
EXAMPLE 2: Antisense Treatment And Cell Culture<br>
To explore the potential functional role of LMP-1 during membranous bone<br>
formation, we synthesized an antisense oligonucleotide to block LMP-1 mRNA<br>
translation and treated secondary osteoblast cultures that were undergoing<br>
differentiation initiated by glucocorticoid. inhibition of RLMP expression was<br>
accomplished with a highly specific antisense oligonucleotide (having no significant<br>
homologies to known rat sequences) corresponding to a 25 bp sequence spanning the<br>
putative translational start site (SEQ. ID NO: 42). Control cultures either did not<br>
receive oligonucleotide or they received sense oligonucleotide. Experiments were<br>
performed in the presence (preincubation) and absence of lipofecfamine. Briefly, 22<br>
µg of sense or antisense RLMP oligonucleotide was incubated in MEM for 45<br>
minutes at room temperature. Following that incubation, either more MEM or pre-<br>
incubated lipofectamine/MEM (7% v/v; incubated 45 minutes at room temperature)<br>
was added to achieve an oligonucleotide concentration of 0:2 µM The resulting<br>
mixture was incubated for 15 minutes at room temperature. Oligonucleotide<br>
mixtures were then mixed with MEM/Ascorbate/±Trm to achieve a final<br>
oligonucleotide concentration of 0.1 µM.<br>
Cells, were incubated with the medium (stimulus) in the presence or absence<br>
of the appropriate oligonucleotides. Cultures originally incubated with lipofectamine<br>
were re-fed after 4 hours of incubation (37°C; 5% CO2) with media containing<br>
neither lipofectamine nor oligonucleotide. Oligonucleotide levels were maintained<br>
by feeding cultures every 24 hours.<br>
LMP-1 antisense oligonucleotide inhibited mineralized nodule formation and<br>
osteocalcin secretion in a dose-dependent manner, similar to the effect seen with<br>
BMP-6 oligonucleotide. The LMP-1 antisense block in osteoblast differentiation<br>
cotild not be rescued by addition of exogenous BMP-6, while the BMP-6 antisense<br>
oligonucleotide inhibition was reversed with addition of BMP-6, confirming the<br>
upstream position of LMP-1 relative to BMP-6 in the osteoblast differentiation<br>
pathway. LMP-1 antisense oligonucleotide also inhibited spontaneous osteoblast<br>
differentiation in primary rat osteoblast cultures.<br>
EXAMPLE 3: Quantitation of Mineralized Bone Nodule Formation'<br>
Cultures of ROBs prepared according to Examples 1 and 2 were fixed<br>
overnight in 70% ethanol and stained with von Kossa silver stain. A semi-automated<br>
computerized video image analysis system was used to quantitate nodule count and<br>
nodule area in each well (Boden, etaL, Endocrinology. 137,8,3401-3407 (1996)).<br>
These values were then used to calculate the area per nodule values. The automated<br>
process was validated against a manual counting technique, demonstrating a<br>
correlation coefficient of 0.92 (p 
standard error of the mean (S.E.M.) calculated from 5 or 6 wells at each condition.<br>
Each experiment was repeated at least twice using cells from different calvarial<br>
preparations.<br>
EXAMPLE 4: Quantitation of Osteocalcin Secretion<br>
Osteocalcin levels in the culture media were measured using a competitive<br>
radioimmunoassay with a monospecific polyclonal antibody (Pab) raised by the<br>
inventors against the C-terminal nonapeptide of rat osteocalcin as described by<br>
Nanes, et al. (Endocrinology. 127:588 (1990)). Briefly, 1 microgram of nonapeptide<br>
was iodinated witli 1 mCi 125I-Na by the lactoperoxidase method. Tubes containing<br>
200 gl of assay buffer (0.02 M sodium phosphate, 1 mM EDTA, 0.001 % thimerosal,:<br>
0.025% BSA) received media taken from cell cultures or osteocalcin standards<br>
(0 -12,000 fmole) at 100 gl/tube in assay buffer. The Pab (1:40,000; 100<br>
microliters) was then added, followed by the iodinated peptide (12,000 cpm; 100<br>
microliters). Samples tested for non-specific binding were prepared similarly but<br>
contained no antibody.<br>
Bound and free PAbs were separated by the addition of 700 microliters goat<br>
antirabbit IgG, followed by incubation for 18 hours at 4°C: After samples were<br>
centrifuged at 1200 rpm for 45 minutes, the supernatants were decanted and the<br>
precipitates counted in a gamma counter. Osteocalcin: values were reported in<br>
fmole/100 microliters, which, was then converted to pmole/ml medium (3-day<br>
production) by dividing those values by 100. Values were expressed as the mean ±<br>
S.E.M. of triplicate determinations for 5-6 wells for each condition. Each experiment<br>
was confirmed at least two times using cells from different calvarial preparations.<br>
EXAMPLE 5: Effect of Trm and RLMP on Mineralization In Vitro<br>
There was little apparent effect of either the sense or antisense<br>
oligonucleotides on the overall production of bone nodules in the non-stimulated cell<br>
culture system. When ROBs were stimulated with Trm, however, the antisense<br>
oligonucleotide to RLMP inhibited mineralization of nodules by &gt; 95%. The<br>
addition of exogenous BMP-6 to the oligonucleotide-treated cultures did not rescue<br>
the mineralization of RLMP-antisense-treated nodules.<br>
Osteocalcin has long been synonymous with bone mineralization, and<br>
osteocalcin levels have been correlated with nodule production and mineralization.<br>
The RLMP-antisense oligonucleotide significantly decreases osteocalcin production,<br>
but the nodule count in antisense-treated cultures does not change significantly. In<br>
this case, the addition of exogenous BMP-6 only rescued the production of<br>
osteocalcin in RLMP-antisense-treated cultures by 10-15%. This suggests that the<br>
action of RLMP is downstream from, and more specific than, BMP-6.<br>
EXAMPLE 6: Harvest and Purification of RNA<br>
Cellular RNA from.duplicate wells of ROBs (prepared according to<br>
Examples 1 and 2 in 6-well culture dishes) was harvested using 4M guanidine<br>
isothiocyanate (GIT) solution to yield statistical triplicates. Briefly, culture<br>
supernatant was aspirated from the wells, which were then overlayed with 0.6 ml of<br>
GIT solution per duplicate well harvest. After adding the GIT solution, the plates<br>
were swirled for 5-10 seconds. Samples were saved at -70 °C for up to 7 days before<br>
further processing.<br>
RNA was purified by a slight modification of standard methods according to<br>
Sambrook, et al. (Molecular Cloning: a Laboratory Manual).. Chapter 7.19,2nd<br>
Edition, Cold Spring Harbor Press (198S&gt;)). Briefly, thawed samples received 60<br>
microliters 2.0 M sodium acetate (pH 4.0), 550 microliters phenol (water saturated)<br>
and 150 microliters chloroform:soatnyl alcohol (49:1). After vortexing, the samples<br>
were centrifuged (10000 x g; 20 minutes; 4 °C), the aqueous phase transferred to a<br>
fresh tube, 600 microliters isopropanol added, and the RNA precipitated overnight at<br>
-20°C.<br>
Following the overnight incubation, samples were centrifuged (10000 x g; 20<br>
minutes) and the supernatant was gently aspirated. The pellets were resuspended in<br>
400 microliter DEPC-treated water, extracted once with phenolxhloroform (1:1),<br>
extracted with chloroform:isoamyl alcohol (24:1) and precipitated overnight at -20°C<br>
after addition of 40 microliters sodium acetate (3.0 M; pH 5.2) and 1.0 ml absolute<br>
ethanol. To recover the cellular KNA, the samples were centrifuged<br>
(10000 x g; 20 min), washed once with 70% ethanol, air dried for 5-10 minutes and<br>
resuspended in 20 µl of DEPC-treated water. KNA concentrations were calculated<br>
from optical densities determined by spectrophotometry.<br>
EXAMPLE 7: Reverse Transcription-Polymerase Chain Reaction<br>
Heated totalRNA (5 micrograms in 10.5 microliters total volume DEPC-H2O<br>
at 65°C for 5 minutes) was added to tubes containing 4 microliters 5X MMLV-RT<br>
buffer, 2 microliters dNTPs, 2 microliters dT17 primer (10 pmol/ml), 0.5 microliter<br>
RNAsin (40 U/ml) and 1 microliter MMLV-RT (200 imits/microliter). Samples were<br>
incubated at 37°C for 1 hour, tben at 95°C for 5 minutes to inactivate the MMLV-<br>
RT. Samples were then diluted by addition of 80 microliters of water.<br>
Reverse-transcribed samples (5 microliters) were subjected to polymerase-<br>
chain reaction using standard methodologies (50 microliters total volume). Briefly,<br>
samples were added to tubes containing water and appropriate amounts of PCR<br>
buffer, 25 mM MgCl2, dNTPs, forward and reverse primers for glyceraldehyde 3-<br>
phosphate dehydrogenase (GAP) and/or BMP-6,32P-dCTP, and Taq polymerase.<br>
Unless otherwise noted, primers were standardized to run consistently at 22 cycles<br>
(94°C, 30"; 58°C, 30"; 72°C, 20"),<br>
EXAMPLE 8: Quantitation of RT-PCR Products by Polyacrylamide Gel<br>
Electrophoresis (PAGE) and Phosphorlmager Analysis<br>
RT-PCR products received 5 microliters/tube loading dye, were mixed,<br>
heated at 65°C for 10 min and centrifuged. A ten (10) microliter sample from each<br>
reaction was subjected to PAGE (12% polyacrylamide:bis; 15 V/well; constant<br>
current) under standard conditions. Gels were then incubated in gel preserving buffer<br>
(10% v/v glycerol, 7% v/v acetic acid, 40% v/v methanol, 43% deionized water) for<br>
30 minutes, dried (80°C) in vacuo for 1-2 hours and developed with an<br>
electronically-enhanced phosphoresence imaging system for 6-24 hours. Visualized<br>
bands were analyzed. Counts per band were plotted graphically.<br>
EXAMPLE 9: Differential Display PCR<br>
RNA was extracted from cells stimulated with glucocorticoid (Trm, 1 :nM).<br>
Heated, DNase-treated total RNA (5 micrograms in 10.5 microliters total volume in<br>
DEPC- H2O at 65 °C for 5 minutes) was reverse transcribed as described in Example.<br>
7, but H-Tn M (SEQ. ID. NO: 4) was used as the MMLV-RT primer. The resulting<br>
cDNAs were PCR-amplified as described above, but with various commercial primer<br>
sets (for example, H-TnG (SEQ. ID NO: 4) and H-AP-10 (SEQ. ID NO: 5);<br>
GenHunter Corp, Nashville, TN). Radio-labeled PCR products were fractionated by<br>
gel electrophoresis on a DNA sequencing gel. After electrophoresis, the resulting<br>
gels were dried in vacuo and autoradiographs were exposed overnight. Bands<br>
representing differentially-expressed cDNAs were excised from the gel and<br>
reamplified by PCR using the method described by Conner, et al. (Proc. Natl. Acad.<br>
Sci. USA. 88,278 (1983)). The products of PCR reamplification were cloned into<br>
the vector PCR-11 (TA cloning kit; Invitrogen, Carlsbad, CA).<br>
EXAMPLE 10: Screening of a UMR106 Rat Osteosarcoma Cell cDNA Library<br>
A UMR 106 library (2.5 x 1010 pfu/ml) was plated at 5 x 104 pfu/ml onto agar<br>
plates (LB bottom agar) and the plates were incubated overnight at 37°C. Filter<br>
membranes were overlaid onto plates for two minutes. Once removed, the filters<br>
were denatured, rinsed, dried and UV cross-linked. The filters were then incubated<br>
in pre-hyridization buffer (2X PIPES [pH 6.5], 5% formamide, 1% SDS and 100<br>
fig/ml denatured salmon sperm DNA) for 2 h at 42°C. A 260 base-pair<br>
radio-labeled probe (SEQ. ID NO: 3 • 32P labeled by random priming) was added to<br>
the entire hybridization mix/filters, followed by hybridization for 18 hours at 42°C.<br>
The membranes were washed once at room temperature (10 min, 1 x SSC, 0.1%<br>
SDS) and three times at 55°C (15 min, 0.1 x SSC, 0.1 % SDS).<br>
After they were washed, the membranes were analyzed by autoradiographyas<br>
described above
with a second filter for four minutes to minimize spurious positives. Plaque-purified<br>
clones were rescued as lambda SK(-) phagemids. Cloned cDNAs were sequenced as<br>
described below.<br>
EXAMPLE 11: Sequencing of Clones<br>
Cloned cDNA inserts were sequenced by standard methods. Briefly,<br>
appropriate concentrations of termination mixture, template and reaction mixture<br>
were subjected to an appropriate cycling protocol (95°C, 30 s; 68°C, 30 s; 72°C, 60<br>
s; x 25).' Stop mixture was added to terminate the sequencing reactions. After<br>
heating at 92°C for 3 minuteS, the samples were loaded onto a denaturing 6%<br>
polyacrylamide sequencing gel (29:1 acrylamide:bisacrylamide). Samples were<br>
electrophoresed for about 4 hours at 60 volts, constant current.. After electrophoresis,<br>
the gels were dried in vacuo and autoradiographed.<br>
The autoradiographs were analyzed manually. The resulting sequences were<br>
•screened against the databases maintained by the National Center for Biotechnology<br>
Information (NIH, Bethesda, MD; hftp://vvww.ncbi.nlm.nih.gov/) using the BLASTN<br>
program set with default parameters. Based on the sequence data, new sequencing<br>
primers were prepared and the process was repeated until the entire gene had been<br>
sequehced. All sequences were confirmed a minimum of three times in both<br>
orientations.<br>
Nucleoti.de and amino acid sequences were also analyzed using the PCGENE<br>
software package (version 16.0). Percent homology values for nucleotide sequences<br>
were calculated by the program NALIGN, using the following parameters: weight of<br>
non-matching nucleotides, 10; weight of non-matching gaps, 10; maximum number<br>
of nucleotides considered, 50; and minimum number of nucleotides considered, 50.<br>
For amino acid sequences, percent homology values were calculated using<br>
P ALIGN. A value of 10 was selected for both the open gap cost and the unit gap<br>
cost.<br>
EXAMPLE 12: Cloning of RLMP cDNA<br>
The differential display PCR amplification products described in Example 9<br>
contained a major band of approximately 260 base pairs. This sequence was used to<br>
screen a rat osteosarcoma (UMR106) cDNA library. Positive clones were subjected<br>
to nested primer analysis to obtain the primer sequences necessary for amplifying the<br>
full length cDNA. (SEQ. ID NOs: 11,12,29,30 and 31). One of those positive<br>
clones selected for further study was designated clone 10-4.<br>
Sequence analysis of the full-length cDNA in clone 10-4, determined by<br>
nested primer analysis, showed that clone 10-4 contained the original 260 base-pair<br>
fragment identified by differential display PCR. Clone 10-4 (1696 base pairs;<br>
SEQ ID NO: 2) contains an open reading frame of 1371 base pairs encoding a protein<br>
having 457 amino acids (SEQ. ID NO: 1). The termination codon, TGA, occurs at<br>
nucleotides 1444-1446. The polyadenylation signal at nucleotides 1675-1680, and<br>
adjacent poly(A)+tail, was present in the 3'noncoding region. There were two<br>
potential N-glycosylation sites, Asn-Lys-Thr and Asn-Arg-Thr, at amino acid<br>
positions 113-116 and 257-259 in SEQ. ID NO: 1, respectively. Two potential<br>
cAMP- and cGMP-dependent protein kinase phosphorylation sites, Ser and Thr, were<br>
found at amino acid positions 191 and 349, respectively. There were five potential .<br>
protein kinase C phosphorylation sites, Ser or Thr, at amino acid positions 3,115,<br>
166,219,442. One potential ATP/GTP binding site motif A (P-loop), Gly-Gly-Ser-<br>
Asn-Asn-Gly-Lys-Thr, was determined at amino acid positions 272-279.<br>
In addition, two highly conserved putative LIM domains were found at amino<br>
acid positions 341-391 and 400-451. The putative LIM domains in this<br>
newly identified rat cDNA clone showed considerable homology with the LIM<br>
domains of other known LIM proteins. However, the overall homology with other<br>
rat LIM proteins was less than 25%. RLMP (also designated 10-4) has 78.5% amino<br>
acid homology to the human enigma protein (see U.S. Patent No. 5,504,192), but<br>
only 24.5% and 22.7% amino acid homology to its closest rat homologs, CLP-36 and<br>
RIT-18, respectively.<br>
EXAMPLE 13: Northern Blot Analysis of RLMP Expression<br>
Thirty micrograms of total RNA from ROBs, prepared according to Examples<br>
1 and 2, was size fractionated by formaldehyde gel electrophoresis in 1 % agarose<br>
flatbed gels and osmotically transblotted to nylon membranes. The blot was probed<br>
with a 600 base pair EcoRI fragment of full-length 10-4 cDNA labeled with 32P-<br>
dCTP by random priming:<br>
Northern blot analysis showed a 1.7 kb mRNA species that hybridized with<br>
the RLMP probe. RLMP mRNA was up-regulated approximately 3.7-fold in ROBs<br>
after 24 hours exposure to BMP-6. No up-regulation of RMLP expression was seen<br>
in BMP-2 or BMP-4-stimulated ROBs at 24 hours.<br>
EXAMPLE 14: Statistical Methods<br>
For each reported nodule/osteocalcin result, data from 5-6 wells from a<br>
representative experiment were used to calculate the mean ± S.E.M. Graphs may be<br>
shown with data normalized to the maximum value for each parameter to allow<br>
simultaneous graphing of nodule counts, mineralized areas and osteocalcin.<br>
For each reported RT-PCR, RNase protection assay or. Western blot analysis,<br>
data from triplicate samples of representative experiments, were used to determine<br>
the mean ± S.E.M. Graphs may be shown normalized to either day 0 or negative<br>
controls and expressed as fold-increase above control values.<br>
Statistical significance was evaluated using a one-way analysis of variance<br>
with post-hoc multiple comparison corrections of Bonferroni as appropriate<br>
(D. V. Huntsberger, "The Analysis of Variance", Elements of Statistical Variance. P.<br>
Billingsley (ed.), Allyn &amp; Bacon Inc., Boston, MA, 298-330 (1977) and SigmaStat<br>
Jandel Scientific, Corte Madera, CA). Alpha levels for significance were defined as<br>
p
EXAMPLE 15: Detection of Rat L1M Mineralization Protein by Western Blot<br>
Analysis<br>
Polyclonal antibodies were prepared according to the methods of<br>
England, et al. (Biochim-Biophys. Acta, 623,171 (1980)) and Timmer, et al., (I<br>
Biol. Chem.. 268,24863 (1993)).<br>
HeLa cells were transfected with pCMV2/RLMP. Protein was harvested<br>
from the transfected cells according to the method of Hair, et al. (Leukemia<br>
Research. 20,1 (1996)). Western Blot Analysis of native RLMP was performed as<br>
described by Towbin, et al. (Proc. Natl. Acad. Sci. USA. 76:4350 (1979)).<br>
EXAMPLE 16: Synthesis of the Rat LMP-Unique (RLMPU) derived Human<br>
PCRproduct<br>
Based on the sequence of the rat LMP-1 cDNA, forward and reverse PCR<br>
primers (SEQ. ID NOS: 15 and 16) were synthesized and a unique 223 base-pair<br>
sequence was PCR amplified from the rat LMP-1 cDNA. A similar PCR product<br>
was isolated from human MG63 osteosarcoma cell cDNA with the same PCR<br>
primers.<br>
RNA was harvested from MG63 osteosarcoma cells grown in T-75 flasks.<br>
Culture supernatant was removed by aspiration and the flasks were overlayed with<br>
3.0 ml of GIT solution per duplicate, swirled for 5-10 seconds, and the resulting<br>
solution was transferred to 1.5 ml eppendorf tubes (6 tubes with 0.6 ml/tube). RNA<br>
was purified by a slight modification of standard methods (Sambrook, et al.,<br>
Molecular Cloning: A Laboratory Manual. Chapter 7, page 19, Cold Spring Harbor<br>
Laboratory Press (1989) and Boden, et al., Endocrinology. 138,2820-2828 (1997).<br>
Briefly, the 0.6 ml samples received 60 microliters 2.0 M sodium acetate (pH 4.0),<br>
550 microliters water saturated phenol and 150 microliters chloroform:isoamyl<br>
alcohol (49:1). After addition of those reagents, the samples were vortexed,<br>
centrifuged (10000 x g; 20 min; 4C) and the aqueous phase transferred to a fresh<br>
tube. Isopropanol (600 microliters) was added and the RNA was precipitated<br>
overnight at -20°C. The samples were centrifuged (10000 x g; 20 minutes) and the<br>
supernatant was gently aspirated. The pellets were resuspended in 400 microliters of<br>
DEPC-treated water, extracted once with phenolxhloroform (1:1), extracted with<br>
chloroform:isoamyl alcohol (24:1) and precipitated overnight at -20 °C in 40<br>
microliters sodium acetate (3.0 M; pH 5.2) and 1.0 ml absolute ethanol. After<br>
precipitation, the samples were centrifuged (10000 x g; 20 mm), washed once with<br>
70% ethanol, air dried for 5-10 minutes and resuspended in 20 microliters of DEPC-<br>
treated water. RNA concentrations were derived from optical densities.<br>
Total RNA (5 micrograms in 10.5 microliters total volume in DEPC-H2O)<br>
was heated at 65 °C for 5 minutes, and then added to tubes containing 4 microliters<br>
5X MMLV-RT buffer, 2 microliters dNTPS, 2 microliters dT17 primer (10 pmol/ml),<br>
0.5 microliter RNAsin (40 U/ml) and 1 microliter MMLV-RT (200 units/microliter).<br>
The reactions were incubated at 37°C for 1 hour. Afterward, the MMLV-RT was<br>
inactivated by heating at 95°C for 5 minutes. The samples were diluted by addition<br>
of 80 microliters water.<br>
Transcribed samples (5 microliters) were subjected to polymerase-chain<br>
reaction using standard methodologies (50 microliters total volume), such as those<br>
described by Boden, et al. (Endocrinology, 138,2820-2828 (1997) and Ausubel, et<br>
al.("Quantitation of Rare DNAs by the Polymerase Chain Reaction", Current<br>
Protocols in Molecular Biology, Chapter 15.31-1, Wiley &amp; Sons, Trenton, NJ<br>
(1990)). Briefly, samples were added to tubes containing water and appropriate<br>
amounts of PCR buffer (25 mM MgCl2, dNTPs, forward and reverse primers (for<br>
RLMPU; SEQ. ID NOS: 15 and 16)), 32P-dCTP, and DNA polymerase. Primers<br>
were designed to run consistently at 22 cycles for radioactive band detection and 33<br>
cycles for amplification of PCR product for use as a screening probe (94°C, 30 sec,<br>
58°C, 30 sec; 72°C, 20 sec).<br>
Sequencing of the agarose gel-purified MG63 osteosarcoma-derived PCR<br>
product gave a sequence more than 95% homologous to;the RLMPU PCR product.<br>
That sequence is designated HLMP unique region (BLMPU; SEQ. ID NO: 6).<br>
EXAMPLE 17: Screening of reverse-transcriptase-derived MG63 cDNA<br>
Screening was performed with PCR using specific primers (SEQ. ID NOS: 16<br>
and 17) as described in Example 7. A 717 base-pair MG63 PCR product was agarose<br>
gel purified and sequenced with the given primers (SEQ. ID NOs: 12,15,16, 17,18,<br>
27 and 28). Sequences were confirmed a minimum of two times in both directions.<br>
The MG63 sequences were aligned against each other and then against the full-length<br>
rat LMP cDNA sequence to obtain a partial human LMP cDNA sequence (SEQ. ID<br>
NO: 7).<br>
EXAMPLE 18: Screening of a HumanHeart cDNA Library<br>
Based on Northern blot experiments, it was determined that LMP-1 is<br>
expressed at different levels by several different tissues, including human heart<br>
muscle. A human heart cDNA library was therefore examined. The library was<br>
plated at 5 x 104 pfu/ml onto agar plates (LB bottom agar) and plates were grown<br>
overnight at 37°C. Filter membranes were overlaid onto the plates for two minutes.<br>
Afterward, the filters denatured, rinsed, dried, UV cross-linked and incubated in pre-<br>
hyridization buffer (2X PIPES [pH 6.5]; 5% formamide, 1 % SDS, 100 g/ml<br>
denatured salmon sperm DNA) for 2 h at 42 °C. A radio-labeled, LMP-unique, 223<br>
base-pair probe (32P, random primer labeling; SEQ ID NO: 6) was added and<br>
hybridized for 18 h at 42°C. Following hybridization, the membranes were, washed<br>
once at room temperature (10 min, 1 x SSC, 0.1 % SDS) and three times at 55°C (15<br>
min, 0.1 x SSC, 0.1 % SDS). Double-positive plaque-purified heart library clones,<br>
identified by autoradiography, were rescued as lambda phagemids according to the<br>
manufacturers' protocols (Stratagene, La Jolla, CA).<br>
Restriction digests of positive clones yielded cDNA inserts of varying sizes.<br>
Inserts greater than 600 base-pairs in. length were selected for initial screening by<br>
sequencing. Those inserts were sequenced by standard, methods as described in<br>
Example 11.<br>
One clone, number 7, was also subjected to automated sequence analysis<br>
using primers corresponding to SEQ. ID NOS: 11-14,16 and 27. The sequences<br>
obtained by these methods were routinely 97-100% homologous. Clone 7 (Partial<br>
Human LMP-1 cDNA from a heart library; SEQ. ID NO: 8) contained a sequence<br>
that was more than 87% homologous to the rat LMP cDNA sequence in the<br>
translated region.<br>
EXAMPLE 19: Determination of Full-Length Human LMP-1 cDNA<br>
Overlapping regions of-the MG63 human osteosarcoma cell cDNA<br>
sequence and the human heart cDNA clone 7 sequence were used to align those two<br>
sequences and derive a complete human cDNA sequence of 1644 base-pairs.<br>
NALIGN, a program in the PCGENE software package, was used to align the two<br>
sequences. The overlapping regions of the two sequences constituted approximately<br>
360 base-pairs having complete homology except for a single nucleotide substitution<br>
at nucleotide 672 in the MG63 cDNA (SEQ. ID NO: 7) with clone 7 having an "A"<br>
instead of a "G" at the corresponding nucleotide 516 (SEQ.. ID NO: 8).<br>
The two aligned sequences were joined using SEQIN, another subprogram of<br>
PCGENE, using the "G" substitution of the MG63 osteosarcoma cDNA clone. The<br>
resulting sequence is shown in SEQ. ID NO: 9, Alignment of the novel human-<br>
derived sequence with the rat LMP-1 cDNA was accomplished with NALIGN. The<br>
full-length human LMP-1 cDNA sequence (SEQ. ID NO: 9) is 87.3% homologous .<br>
to the translated portion of rat LMP-1 cDNA sequence.<br>
EXAMPLE 20: Determination of Amino Acid Sequence of Human LMPri<br>
The putative amino acid sequence of human LMP-1 was determined with the<br>
PCGENE subprogram TRANSL. The open reading frame in SEQ. ID NO: 9 encodes •<br>
a protein comprising 457 amino acids (SEQ. ID NO: 10). Using the PCGENE<br>
subprogram Palign, the human LMP-1.amino acid sequence was found to be 94.. 1%<br>
homologous to the rat LMP-1 amino acid sequence.<br>
EXAMPLE 21: Determination of the 5 Prime Untranslated Region of the<br>
Human LMPcDNA<br>
MG63 5' cDNA was amplified by nested RT-PCR of MG63 total RNA<br>
using a 5' rapid amplification of cDNA ends (5' RACE) protocol. This method<br>
included first strand cDNA synthesis using a lock-docking oligo (dT) primer with<br>
two degenerate nucleotide positions at the 3' end (Chenchik, et al.,<br>
CLONTECHniques, X:5 (1995); Borson, et al., PC Methods Applic.. 2,144 (1993)).<br>
Second-strand synthesis was performed according to the method of Gubler, et al.<br>
(Gene. 2,263 (1983)), with a cocktail of Escherichia coli DNA polymerase 1, RNase<br>
H, and E. coli DNA ligase. After creation of blunt ends with T4 DNA polymerase,<br>
double-stranded cDNA was ligated to the fragment<br>
(5'-CTAATACGACTCACTATAGGGCTCGAGCGGCCGCCCGGGCAGGT-3')<br>
(SEQ. ID NO: 19). Prior to RACE, the adaptor-ligated cDNA was diluted to a<br>
concentration suitable, for Marathon RACE reactions (1:50). Adaptor-ligated double-<br>
stranded cDNA was then ready to be specifically cloned.<br>
First-round PCR was performed with the adaptor-specific oligonucieotide, 5'-<br>
CCATCCTAATACGACTCACTATAGGGC- 31 (API) (SEQ. ID NO: 20),as sense<br>
primer and a Gene Specific Primer (GSP) from the unique region described in<br>
Example 16 (HLMPU). The second round of PCR was performed using a nested<br>
primers GSP1-HLMPU (antisense/reverse primer) (SEQ. ID NO: 23) and GSP2-<br>
HLMPUF (SEQ. ID NO: 24) (see Example 16; sense/forward primer). PCR was<br>
performed using a commercial kit (Advantage cDNA PCR core kit; CloneTech<br>
Laboratories Inc., Palo Alto, CA) that utilizes an antibody-mediated, but otherwise<br>
standard, hot-start protocol. PCR conditions for MG63 cDNA included an initial<br>
hot-start denaturation (94°C, 60 sec) followed by. 94°C, 30 sec; 60°C, 30 sec; 68°C,<br>
4 min; 30 cycles. The firstround PCR product was approximately 750 base-pairs in<br>
length whereas the nested PCR product was approximately 230 base-pairs. The first-<br>
round PCR product was cloned into linearized pCR 2.1 vector (3.9 Kb). The inserts<br>
were sequenced in both directions using M13 Forward and. Reverse primers (SEQ. ID<br>
NO: 11; SEQ. ID NO: 12).<br>
EXAMPLE 22: Determination of Full-length Human LMP-1 cDNA with<br>
5 Prime UTR<br>
Overlapping MG63 human osteosarcoma cell cDNA 5'-UTR sequence<br>
(SEQ. ED NO: 21), MG63 717 base-pair sequence (Example 17; SEQ. ID NO: S) and<br>
human heart cDNA clone 7 sequence (Example 18) were aligned to derive a novel<br>
human cDNA sequence of 1704 base-pairs (SEQ. ID NO: 22). The alignment was<br>
accomplished with NALIGN, (both PCGENE and Omiga 1.0; Intelligenetics). Over-<br>
lapping sequences constituted nearly the entire 717 base-pair region (Example; 17) .<br>
with 100% homology. Joining of the aligned sequences was accomplished with.<br>
SEQESf.<br>
EXAMPLE 23: Construction of LIM Protein Expression Vector<br>
The construction of pHIS-5ATG LMP-1s expression vector was carried out<br>
with the sequences described in Examples 17 and 18. The 717 base-pair clone<br>
(Example 17; SEQ. ID NO: 7) was digested with Clal and EcoRV. A small fragment<br>
(-250 base-pairs) was gel purified. Clone 7 (Example 18; SEQ. ID NO: 8) was<br>
digested with Clal and Xbal and a 1400 base-pair fragment was gel purified. The<br>
isolated 250 base-pair and 1400 base-pair restriction fragments were ligated to form a<br>
fragment of ~ 1650 base-pairs.<br>
Due to the single nucleotide substitution in Clone 7 (relative to the 717 base-<br>
pair PCR sequence and the original rat sequence) a stop codon at translated base-pair<br>
672 resulted. Because of this stop codon, a truncated (short) protein was encoded,<br>
hence the name LMP-1s. This was the construct used in the expression vector (SEQ.<br>
ID NO: 32). The full length cDNA sequence with 5' UTR (SEQ. ID NO: 33) was<br>
created by alignment of SEQ. ID NO: 32 with the 5' RACE sequence (SEQ. ID NO:<br>
21). The amino acid sequence of LMP-ls (SEQ.ID NO: 34) was then deduced as a<br>
223 amino. acid protein, and confirmed by Western blot (as in Example 15) to run at<br>
the predicted molecular weight of -23.7 lcD.<br>
The pHis-ATG.vector (Invitrogen, Carlsbad, CA) was digested with EcoRV<br>
and Xbal. The vector was recovered and the 650 base-pair restriction fragment was<br>
then ligated into the linearized pHis-ATG. The ligated product was cloned and<br>
amplified. The pHis-ATG-LMP-ls Expression vector, also designated pHIS-A with<br>
insert HLMP-1s, was purified by standard methods.<br>
EXAMPLE 24: Induction of Bone Nodule Formation and Mineralization In vitro<br>
with LMP Expression Vector<br>
Rat calvarial cells were isolated and grown in secondary culture according to<br>
Example 1. Cultures were either unstimulated or stimulated with glucocorticoid (GC)<br>
as described in Example 1. A modification of the Superfect Reagent (Qiagen,<br>
Valencia, CA) transfection protocol was used to transfect 3 micrograms/well of each<br>
vector into secondary rat calvarial osteoblast cultures according to Example 25.<br>
Mineralized nodules were visualized by Von Kossa staining, as described in<br>
Example 3. Human LMP-1s gene product over-expression alone induced bone<br>
nodule formation (~203 nodules/well) in vitro. Levelsof nodules were<br>
approximately 50% of those induced by the GC positive control (~412 nodules/well).<br>
Other positive controls included thr pHisA-LMP-Rat expression vector (~152<br>
nodules/well) and the pCMV2/LMP~Rat-Fwd Expression vector (-206<br>
nodules/well), whereas the negative controls included the pCMV2/LMP-Rat-Rev.<br>
expression vector (~2 nodules/well) and untreated (NT) plates (-4 nodules/well).<br>
These data demonstrate that the human cDNA was at least as osteoinductive as the<br>
rat cDNA. The effect was less than that observed with GC stimulation, most likely<br>
due to sub-optimal doses of Expression vector.<br>
EXAMPLE 25: LMP-Induced Cell Differentiation In Vitro and In Vivo<br>
The rat.LMP cDNA in clone 10-4 (see Example 12) was excised from the<br>
vector by double-digestion with NotI and Apal overnight at 37°C. Vector pCMV2<br>
MCS (Invitrogen, Carlsbad, CA) was digested with the same restriction'enzymes.<br>
Both the linear cDNA fragment from clone 10-4 and pCMV2 were gel purified,<br>
extracted and ligated with T4 ligase. The ligated DNA was gel purified, extracted<br>
and used to transform E. coli JM109 cells for amplification. Positive agar colonies<br>
were selected, digested with Notl and Apal and the restriction digests were examined<br>
by gel electrophoresis. Stock cultures were prepared of positive clones.<br>
A reverse vector was prepared in analogous fashion except that the restriction<br>
enzymes used were Xbal and HindIII. Because these restriction enzymes were used,<br>
the LMP cDNA fragment from clone 10-4 was inserted into pRc/CMV2 in the<br>
reverse (that is non-translatable) orientation. The recombinant vector produced was<br>
designated pCMV2/RLMP.<br>
An appropriate volume of pCMV10-4 (60 nM iinal concentration is optimal<br>
[3micrograms]; for this experiment a range of 0-600 nM/well [0-30 microgram /well]<br>
final concentration is preferred) was resuspended in Minimal Eagle Media (MEM) to<br>
450 microliter final volume and vortexed for 10 seconds. Superfect was added (7.5<br>
microliter/ml final solution), the solution was vortexed for 10 seconds and then<br>
incubated at room temperature for 10 minutes. Following this incubation, MEM<br>
supplemented with 10% FBS (1 ml/well; 6 ml/plate) was added and mixed by<br>
pipetting.<br>
The resulting solution was then promptly pipetted (1 ml/well) onto washed<br>
ROB cultures. The cultures were incubated for 2 hours at 37 °C in a humidified<br>
atmosphere containing 5% CO2. Afterward, the cells were gently washed once with<br>
sterile PBS and the appropriate normal incubation medium was added.<br>
Results demonstrated significant bone nodule formation in all rat cell cultures<br>
which were induced with pCMV10-4. For example, pCMV10-4 transfected cells<br>
produced 429 nodules/well. Positive control cultures, which were exposed to Trm,<br>
produced 460 nodules/well. In contrast, negative controls, which received no<br>
treatment, produced 1 nodule/well. Similarly, when cultures were transfected with<br>
pCMV10-4 (reverse), no nodules were observed.<br>
For demonstrating de novo bone formation in vivo, marrow was aspirated<br>
from the hind limbs of 4-5 week old normal rats (mu/+; heterozygous for recessive<br>
athymic condition). The aspirated marrow cells were washed in alpha MEM,<br>
centrifuged, and RBCs were lysed by resuspending the pellet in 0.83%NH4Gl in<br>
10 mM Tris (pH 7.4)...The remaining marrow cells were washed 3x with MEM and<br>
transfected for 2 hours with 9 microgram of pCMV-LMP1s (forward or reverse<br>
orientation) per 3 x 106 cells. The transfected cells were then washed 2X with<br>
MEM and resuspended at a concentration of 3 x 107 cells/ml.<br>
The cell suspension (100 microliter) was applied via sterile pipette to a sterile<br>
2 x 5 mm type I bovine collagen disc (Sulzer Orthopaedics., Wheat Ridge, CO). The<br>
discs were surgically implanted subcutaneously on the skull, chest, abdomen or<br>
dorsal spine of 4-5 week old athymic rats (mu/mu). The animals were sacrificed at<br>
3-4 weeks, at which time the discs or surgical areas were excised and fixed in 70%<br>
ethanol. The fixed specimens were analyzed by radiography and undecalcified<br>
histologic examination was performed on 5 micrometer thick sections stained with<br>
Goldner Trichrome. Experiments were also performed using devitalized (guanidine<br>
extracted) demineralized bone matrix (Osteotech, Shrewsbury, NJ) in place of<br>
collagen discs.<br>
Radiography revealed a high level of mineralized bone formation that<br>
conformed to the form of the original collagen disc containing LMP-ls transfected<br>
marrow cells. No mineralized bone formation was observed in the negative control<br>
(cells transfected with a reverse-oriented version of the LMP-ls cDNA that did not<br>
code for a translated protein), and absorption of the carrier appeared to be well<br>
underway.<br>
Histology revealed new bone trabeculae lined with osteoblasts in the LMP-ls<br>
transfected implants. No bone was seen along with partial resorption of the carrier in<br>
the negative controls.<br>
Radiography of a further experiment in which 18 sets (9 negative control<br>
pCMV-LMP-REV &amp; 9 experimental pCMV-LMP-1s) of implants were added to<br>
sites alternating between lumbar and thoracic spine in athymic rats demonstrated 0/9<br>
negative control implants exhibiting bone formation (spine fusion) between<br>
vertebrae. All nine of the pCMV-LMP-ls treated implants exhibited solid bone<br>
fusions between vertebrae.<br>
EXAMPLE 26: The Synthesis of pHIS-5' ATG LMP-ls Expression Vector from the<br>
sequences Demonstrated in Examples 2 and 3<br>
The 717 base-pair clone (Example 17) was digested with ClaI and EcoRV<br>
(New England Biologicals, city, MA), A small fragment (~250 base pairs) was gel<br>
purified. Clone No. 7 (Example 18) was digested with Clal and XbaI. A 1400 base-<br>
pair fragment was gel purified from that digest. The isolated 250 base-pair and 1400<br>
base-pair cDNA fragments were ligated by standard methods to form a fragment of<br>
~1650 bp. The pEEs-A vector (Invitrogen) was digested with EcoRV and XbaI. The<br>
linearized vector was recovered and ligated to the chimeric 1650 base-pair cDNA<br>
fragment. The ligated product was cloned and amplified by standard methods, and<br>
the phis-A-5' ATG LMP-ls expression vector, also denominated as the vector pHis-A<br>
with insert HLMP-1s, was deposited at the ATCC as previously described.<br>
EXAMPLE 27: The Induction of Bone Nodule Formation and Mineralization In<br>
Vitro With pHis-5' ATG LMP-1s Expression Vector<br>
Rat calvarial cells were isolated and grown in secondary culture according to<br>
Example 1. Cultures were either unstimulated or stimulated with glucocorticoid<br>
(GC) according to Example 1. The cultures were transfected with 3 micrograms of<br>
recombinant pHis-A vector DNA/well as described in Example 25. Mineralized<br>
nodules were visualized by Von Kossa staining according to Example 3.<br>
Human LMP-1s gene product overexpression alone (i.e., without GC<br>
stimulation) induced significant bone nodule formation (~203 nodules/well) in vitro.<br>
This is approximately 50% of the amount of nodules produced by cells lo exposed to<br>
the GC positive control (~412 nodules/well). Similar results were obtained with<br>
cultures transfected with pHisA-LMP-Rat Expression vector (~152 nodules/well) and<br>
pCMV2/LMP-Rat-Fwd (~206 nodules/well). In contrast, the negative control<br>
p'CMV2/LMP-Rat-Rev yielded (~2 nodules/well), while approximately 4<br>
nodules/well were seen in the untreated plates. These data demonstrate that the<br>
human LMP-1 cDNA was at least as osteoinductive as the rat LMP-lcDNA in this<br>
model system. The effect in this experiment was less than that observed with GC<br>
stimulation; but in some the effect was comparable.<br>
EXAMPLE 28: LMP Induces Secretion of a Soluble Osteoinductive Factor<br>
Overexpression of RLMP-1 or HLMP-ls in rat calvarial osteoblast cultures as<br>
described in Example 24 resulted in significantly greater nodule formation than was<br>
observed in the negative control. To study the mechanism of action of LIM<br>
mineralization protein conditioned medium was harvested at different time points,<br>
concentrated to 10 X, sterile filtered, diluted to its original concentration in medium<br>
containing fresh serum, and applied for four days to untransfected cells.<br>
Conditioned media harvested from cells transfected with RLMP-1 or HLMP-<br>
ls at day 4 was approximately as effective in inducing nodule formation as direct<br>
overexpression of RLMP-1 in transfected cells. Conditioned media from cells<br>
transfected with RLMP-1 or HLMP-1 in the reverse orientation had no apparent<br>
effect on nodule formation. Nor did.conditioned media harvested from LMP-1<br>
transfected cultures before day 4 induce nodule formation. These data indicate that<br>
expression of LMP-1 caused the synthesis and/or secretion of a soluble factor, which<br>
did not appear in culture medium in effective amounts until 4 days post transfection.<br>
Since overexpression of rLMP-1 resulted in fhe secretion of an osteoinductive<br>
factor into the medium, Western blot analysis was used to determine if LMP-1<br>
protein was present in the medium. The presence of RLMP-1 protein was assessed<br>
using antibody specific for LMP-1 (QDPDEE) and detected by conventional means.<br>
LMP-1 protein was found only in the cell layer of the culture and not detected in the<br>
medium.<br>
Partial purification of the osteoinductive soluble factor was accomplished by<br>
standard 25% and 100% ammonium sulfate cuts followed by DE-52 anion exchange<br>
batch chromatography (100 mM or 500 mM NACl). All activity was observed in the<br>
high ammonium sulfate, high NaCl fractions. Such localization is consistent with the<br>
possibility of a single factor being responsible for conditioning the medium.<br>
EXAMPLE 29: Gene Therapy In Lumbar Spine Fusion Mediated by Low Dose<br>
Adenovirus<br>
This study determined the optimal dose of adenoviral delivery of the LMP-1<br>
cDNA (SEQ. ID NO: 2) to promote spine fusion in normal., that is, immune<br>
competent, rabbits.<br>
A replication-deficient human recombinant adenovirus was constructed with<br>
the LMP-1 cDNA (SEQ. ID NO: 2) driven by a CMV promoter using the Adeno-<br>
Quest™ Kit (Quantum Biotechnologies, Inc., Montreal). A commercially available<br>
(Quantum Biotechnologies, Inc., Montreal) recombinant adenovirus containing the<br>
beta-galactosidase gene was used as a control.<br>
Initially, an in vitro dose response experiment was performed to determine<br>
the optimal concentration of adenovirus-delivered LMP-1 ("AdV-LMP-1") to induce<br>
bone differentiation in rat calvarial osteoblast cultures using a 60-minute transduction<br>
with a multiplicity of infection ("MOI") of 0.025,0.25,2.5, or 25 plaque-forming<br>
units (pfu) of virus per cell. Positive control cultures were differentiated by a 7-day<br>
exposure to 109 M glucocorticoid ("GC"). Negative control cultures were left<br>
untreated. On day 14, the number of mineralized bone nodules was counted after<br>
von Kossa staining of the cultures, and the level of osteocalcin secreted into the<br>
medium (pmol/mL) was measured by radioimmunoassay (mean ± SEM).<br>
The results of this experiment are shown in Table I, below. Essentially no<br>
spontaneous nodules formed in the untreated negative control cultures. The data<br>
show that a MOI equal to 0.25 pfu/cell is most effective for osteoinducing bone<br>
nodules, achieving a level comparable to the positive control (GC). Lower and<br>
higher doses of adenovirus were less effective.<br>
In vivo experiments were men performed to determine if the optimal in vitro<br>
dose was capable of promoting intertransverse process spine fusions in skeletally<br>
mature New Zealand white rabbits. Nine rabbits were anesthetized and 3 cc of bone<br>
marrow was aspirated from the distal femur through the intercondylar notch using an<br>
18 gauge needle. The buffy coat was then isolated, a 10-minute transduction with<br>
AdV-LMP-1 was performed, and the cells were returned to the operating room for<br>
implantation. Single level posterolateral lumbar spine arthrodesis was performed<br>
with decortication of transverse processes and insertion of carrier (either rabbit<br>
devitalized bone matrix or a collagen sponge) containing 8-15 million autologous<br>
nucleated buffy coat cells transduced with either AdV-LMP-1 (MOI = 0.4) or AdV-<br>
BGal (MOI=0.4). Rabbits were euthanized after 5 weeks and spine fusions were<br>
assessed by manual palpation, plain x-rays, CT scans, and undecalcified histology.<br>
The spine fusion sites that received AdV-LMP-.t induced solid, continuous<br>
spine fusion masses in all nine rabbits. In contrast, the sites receiving AdV-BGal, or<br>
a lower dose of AdV-LMP-1 (MOI = 0.04) made little or no bone and resulted in<br>
spine fusion at a rate comparable to the carrier alone (
consistent as evaluated by manual palpation, CT scan, and histology. Plain<br>
radiographs, however, sometimes overestimated the amount of bone that was present,<br>
especially in the control sites. LMP-1 cDNA delivery and bone induction was<br>
successful with both of the carrier materials tested. There was no evidence of<br>
systemic or local immune response to the adenovirus vector.<br>
These data demonstrate consistent bone induction in a previously validated<br>
rabbit spine fusion model. The protocol of using autogenous bone marrow cells with<br>
intraoperative ex vivo gene transduction (10 minutes) is a more clinically feasible<br>
procedure than other methods that call for overnight transduction or cell expansion<br>
for weeks in culture. In addition, the most effective dose of recombinant adenovirus<br>
(MOI=0.25) was substantialfy lower than doses reported in other gene therapy<br>
applications (MOI40-500). The inventors believe this is due to the fact that LMP-1<br>
is an intracellular signaling molecule and may have powerful signal amplification<br>
cascades. Moreover, the observation that the same concentration of AdV-LMP-1 that<br>
induced bone in cell culture was effective in vivo was also surprising given the usual<br>
required increase in dose of other growth factors when translating from cell culture to<br>
animal experiments. Taken together, these observations indicate that local gene<br>
therapy using adenovirus to deliver the LMP-1 cDNA is possible and the low dose<br>
required will likely minimize the negative effects of immune response to the<br>
adenovirus vector.<br>
EXAMPLE 30: Use of Peripheral Venous Blood Nucleated Cells (Buffv Coat) for<br>
Gene Therapy With LMP-1 cDNA To Make Bone<br>
In four rabbits we performed spine fusion surgery as above (Example 29}<br>
except the transduced cells were the huffy coat from venous blood rather than bone<br>
marrow. These cells were transfected with AdLMP or pHIS-LMP plasmid and had<br>
equivalent successful results as when bone marrow cells were used. This discovery<br>
of using ordinary venous blood cells for gene delivery makes gene therapy more<br>
feasible clinically since it avoids painful marrow harvest under general anesthesia<br>
and yields two times more cells per milliliter of starting material.<br>
EXAMPLE 31: Isolation of Human LMP-1 Splice Variants<br>
Intron/Exon mRNA transcript splice variants are a relatively common<br>
regulatory mechanism in signal-transduction and cellular/tissue development. Splice<br>
variants of various genes have been shown to alter protein-protein, protein-DNA,<br>
protein-RNA, and protein-substrate interactions. Splice variants may also control<br>
tissue specificity for gene expression allowing different forms (and therefore<br>
functions) to be expressed in various tissues. Splice variants are a common<br>
regulatory phenomenon in cells. It is possible that the LMP splice variants may<br>
result in effects in other tissues such as nerve regeneration, muscle regeneration, or<br>
development of other tissues.<br>
To screen a human heart cDNA library for splice variants of the HLMP-1<br>
sequence, a pair of PCR primer corresponding to sections of SEQ. ID NO: 22 was<br>
prepared. The forward PCR primer, which was synthesized using standard<br>
techniques, corresponds to nucleotides 35-54 of SEQ. ID NO: 22, It has the<br>
following sequence:<br>
5' GAGCCGGCATCATGGATTCC 3' (SEQ. ID NO; 35)<br>
The reverse PCR primer, which is the reverse complement of nucleotides<br>
820-839 in SEQ. ID NO: 22, has the following sequence<br>
5' GCTGCCTGCACAATGGAGGT 3' (SEQ. ID NO: 36)<br>
The forward and reverse PCR primers were used to screen human heart<br>
cDNA (ClonTech, Cat No. 7404-1) for sequences similar to HLMP-1 by standard<br>
techniques, using a cycling protocol of 94°C for 30 seconds, 64°C for 30 seconds, and<br>
72°C for 1 minute, repeated 30 times and followed by a 10 minute incubation at 72°C.<br>
The amplification cDNA sequences were gel-purified and submitted to the Emory<br>
DNA Sequence Core Facility for sequencing. The clones were sequenced using<br>
standard techniques and the sequences were examined with PCGENE (intelligenetics;<br>
Programs SEQUIN and NALIGN) to determine homology to SEQ. ID NO: 22. Two<br>
homologous nucleotide sequences with putative alternative splice sites compared to<br>
SEQ. ID NO: 22 were then translated to their respective protein products with<br>
Intelligenetic's program TRANSL.<br>
One of these two novel human cDNA sequences (SEQ. ID NO: 37) comprises<br>
1456 bp:<br>
CGACGCAGAG CAGCGCCCTG GCCGGGCCAA GCAGGAGCCG GCATCATGGA TTCCTTCAAG 60<br>
GTAGTGCTGG AGGGGCCAGC ACCTTGGGGC TTCCGGCTGC AAGGGGGCAA GGACTTCAAT 120<br>
GTGCCCCTCT CCATTTCCCG GCTCACTCCT GGGGGCAAAG CGGCGCAGGC pCGGAGTGGCC 180<br>
GTGGGTGACT GGGTGCTGAG CATCGATGGC GAGAATGCGG GTAGCCTCAC ACACATCGAA 240<br>
GCTCAGAACA AGATCCGGGC CTGCGGGGAG CGCCTCAGCC TGGGCCTCAG CAGGGCCCAG 300<br>
CCGGTTCAGA GCAAACCGCA GAAGGTGCAG ACCCOTGACA AACAGCCGCT CCGACCGCTG 360<br>
GTCCCAGATG CCAGCAAGCA GCGGCTGATG GAGAACACAG AGGACTGGCG GCCGCGGCCG 420<br>
GGGACAGGCC AGTCGCGTTC CTTCCGCATC CTTGCCCACC TCACAGGCAC CGAGTTCATG 480<br>
CAAGACCCGG ATGAGGAGCA CCTGAAGAAA TCAAGCCAGG TGCCCAGGAC AGAAGCCCCA 540<br>
GCCCCAGCCT CATCTACACC CCAGGAGCCC TGGCCTGGCC CTACCGCCCC CAGCCCTACC.600<br>
AGCCGCCCGC CCTGGGCTGT GGACCCTGCG TTTGCCGAGC GCTATGCCCC GGACAAAACG 660<br>
AGCACAGTGC TGACCCGGCA- CAGCCAGCCG GCCACGCCCA C(3CCGCTGCA GAGCCGCACC 720<br>
TCCATTGTGC AGGCAGCTGC CGGAGGGC3TG CCAGGAGGGG GCAGCAACAA CGGCAAGACT 780<br>
CCCGTGTGTC ACCAGTGCCA CAAGGTCATC CGGGGCCGCT AGCTGGTGGC GTTGGGCCAC 840<br>
GCGTACCACC CGGAGGAGTT TGTGTGTAGC CAGTGTGGGA AGGTCCTGGA AGAGGGTGGC 900<br>
TTCTTTGAGG AGAAGGGCGC CATCTTCTGC CCACCATGCT ATGACGTGCG CTATGCACCC 960<br>
AGCTGTGCCA AGTGCAAGAA GAAGATTACA GGCGAGATCA TGCACGCCCT GAAGATGACC 1020<br>
TGGCACGTGC ACTGCTTTAC CTGTGCTGCC TGCAAGACGC CCATCCGGAA CAGGGCCTTC 1080<br>
TACATGGAGG AGGGCGTGCC CTATJGCGAG CGAGACTATG AGaAGATGTT TGGCACGA3A 1140<br>
TGCCATGGCT GTGACTTCAA GATCGACGCT GGGGACCGCT TCCTGGAGGC CCTGGGCTTC 1200<br>
AGCTGGCATG ACACCTGCTT CGTCTGTGCG ATATGTCAGA TCAACCTGGA AGGAAAGACC 1260<br>
TTCTACTCCA AGAAGGACAG GCCTCTCTGC. AAGAGCCATG CCTTCTCTCA TGTGTGAGCC 1320<br>
CCTTCTGCCC ACAGCTGCCG CGGTGGCCCC TAGCCTGAGG GGCCTGGAGT CGTGGCCCTG 1380<br>
CATTTCTGGG TAGGGCTGGC AATGGTTGCC TTAACCCTGG CTCCTGGCCC GAGCCTGGGC 1440<br>
TCCCGGGCCC TGCCCA 1456<br>
Reading frame shifts caused by the deletion of a 119 bp fragment (between<br>
X) and the addition of a 17 bp fragment (underlined) results in a truncated gene<br>
product having the following derived amino acid sequence (SEQ. ID NO: 38):<br>
Met Asp Ser Phe Lys Val Val Leu Glu Gly Pro Ala Pro Trp Gly Phe<br>
15 10 15<br>
Arg Leu Gln Gly Gly Lys Asp Phe Asn Val Pro Leu Ser lie Ser Arg<br>
20 25 30<br>
Leu Thr Pro Gly Gly Lys Ala Ala Gln Ala Gly Val Ala Val Gly Asp<br>
35 40 45<br>
Trp Val Leu Ser lie Asp Gly Glu Asn Ala Gly Ser Leu Thr His lie<br>
50 55 60<br>
Glu Ala Gln Asn Lys lle Arg Ala Cys Gly Glu Arg Leu Ser Leu Gly<br>
65 70 75 80<br>
Leu Ser Arg Ala Gln Pro Val Gln Asn Lya Pro Gln Lys Val Gln Thr<br>
85 90 95<br>
Pro Ssp Lvs Gln Pro Leu Arg Pro Leu ,Val Pro Asp Ala Ser Lys Gln 100 . 105 110<br>
Arg Leu Met Glu Asn Thr Glu Asp Trp Arg Pro Arg Pro Gly Thr Gly<br>
115 120 125<br>
Gln Ser Arg Ser Phe Arg lle Leu Ala His Leu Thr Gly Thr Glu Phe<br>
130 135 140<br>
Met Gln Asp Pro Asp Glu Glu His Leu Lys Lys Ser Ser Gln Val Pro<br>
145 150 155 160<br>
Arg Thr Glu Ala Pro Ala Pro Ala Ser Ser Thr Pro Gln Glu Pro Trp<br>
165 170 175<br>
Pro Gly Pro Thr Ala Pro Ser Pro Thr Ser Arg Pro Pro Trp Ala Val<br>
180 185 190<br>
Asp Pro Ala Phe Ala Glu Arg Tyr Ala Pro Asp Lys Thr Ser Thr Val<br>
195 200 205<br>
Leu Thr Arg His Ser Gln Pro Ala Thr Pro Thr Pro Leu Gln Ser Arg<br>
210 215 220<br>
Thr Ser lle Val Gln Ala Ala Ala Gly Gly Val Pro Gly Gly Gly Ser<br>
225 230 . 235 240<br>
Asn Asn Gly Lys Thr Pro Val Cys His Gln Cys His Gln Val lie Arg<br>
245 ' 250 255<br>
Ala Arg Tyr Leu Val Ala Leu Gly His Ala Tyr His Pro Glu Glu Phe<br>
260 265 270<br>
Val Cys Ser Gln Cys Gly Lys Val Leu Glu Glu.Gly Gly Phe Phe Glu<br>
275 280 285<br>
Glu Lys Gly Ala He Phe Cys Pro Pro Cys Tyr Asp Val Arg Tyr Ala<br>
290 295 300<br>
Pro Ser Cys Ala Lys Cys Lys Lys Lys lie Thr Gly Glu lie Met His<br>
305 310 315 320<br>
Ala Leu Lys Met Thr Trp His Val Leu Cys Phe Thr Cys Ala Ala Cys<br>
325 330 335<br>
Lys Thr Pro He Arg Asn Arg Ala Phe Tyr Met Glu Glu Gly Val Pro<br>
340 345 350<br>
Tyr Cys Glu Arg Asp Tyr Glu lys Met Phe Gly Thr Lys Cys Gln Trp<br>
355 360 365<br>
Cys Asp Phe Lys He Asp Ala Gly Asp Arg Phe Leu Glu Ala Leu Gly<br>
370 375 380<br>
Phe Ser Trp His Asp Thr Cys Phe Val Cya Ala lie Cys Gln lie Asn<br>
385 390 395 400<br>
Leu Glu Gly Lys Thr Phe Tyr Ser Lys Lys Asp Arg Pro Leu Cys Lys<br>
405 410 415<br>
Ser His Ala Phe Ser His Val<br>
420<br>
This 423 amino acid protein demonstrates 100% homology to the protein<br>
shown in SEQ. ED NO. 10, except for the sequence in the highlighted area (amino acids<br>
94-99), which are due to the nucleotide changes depicted above.<br>
The second novel human heart cDNA sequence (SEQ. ID NO: 39) comprises<br>
1575 bp:<br>
CGACGCAGAG CAGCGCCCTG GCCGGGCCAA GCAGGAGCCG GCATCATGGA TTCCTTCAAG 60<br>
GTAGTGCTGG AGGGGCCAGC ACCTTGGGGC TTCCGGCTGC AAGGGGGCAA GGACTTCAAT 120<br>
GTGCCCCTCT CCATTTCCCG GCTCACTCCT GGGGGCAAAG CGGCGCAGGC CGGAGTGGCC 180<br>
GTGGGTGACT GGGTGCTGAG CATCGATGGC GAGAATGCGG GTAGCCTCAC ACACATCGAfl 240<br>
GCTCAGAACA AGATCCGGGC CTGCGGGGAG CGCCTCAGCC TGGGCCTCAG CAGGGCCCAG 300<br>
CCGGTTCAGA GCAAACCGCA GAAGGCCTCC GCCCCCGCCG CGGACCCTCC GCGGTACACC 360<br>
TTTGCACCCA GCGTCTCCCT CAACAAGACG GCCCGGCCCT TTGGGGCGCC CCCGCCCGCT 420<br>
GACAGCGCCC CGCAACAGAA TGGgTGCAGA CCCCTGGACAA ACAGCCGCTC CGACCGCTGG 480<br>
TCCCAGATGC CAGCAAGCAG CGGCTGATGG AGAACACAGA GGACTGGCGG CCGCGGCCGG 540<br>
GGACAGGCCA GTCGCGTTCC TTCCGCATCC TTGCCCACCT CACAGGCACC GAGTTCATGC 600<br>
AAGACCCGGA TGAGGAGCAC CTGAAGAAAT CAAGCCAGGT GCCCAGGACA GAAGCCCCAG 660<br>
CCCCAGCCTC ATCTACACCC CAGGAGCCCT GGCCTGGCCC TACCGCCCCC AGCCCTACCA 720<br>
GCCGCCCGCC CTGGGCTGTG GACCCTGCGT TTGCCGAGCG CTAIGCCCCG GACAAAACGA 780<br>
GCACAGTGCT GACCCGGCAC.AGCCAGCCGG CCACGCCCAC GCCGCTGCAG AGCCGCACCT 840<br>
CCATTGTGCA GGCAGCTGCC GGAGGGGTGC CAGGAGGGGG CAGCAACAAC GGCAAGACTC 900<br>
CCGTGTGTCA CCAGTGCCAC AAGGTCATCC GGGGCCGCTA CCTGGTGGCG TTGGGCCACG 960<br>
CGTACCACCC GGAGGAGTTT GTGTGTAGCC AGTGTGGGAA GGTCCTGGAA GAGGGTGGCT 1020<br><br>
TCTTTGAGGA GAAGGGCGCC ATCTTCTGCC CACCATGCTA TGACGTGCGC TATGCACCCA 1080<br>
GCTGTGCCAA GTGCAAGAAG AAGATTACAG GCGAGATCAT GCACGCCCTG AAGATGACCT 1140<br>
GGCACGTGCA CTGCTTTACC TGTGCTGCCT GCAAGACGCC CATCCGGAAC AGGGCCTTCT 1200<br>
ACATGGAGGA GGGCGTGCCC TATTGCGAGC GAGACTATGA GAAGATGTTT GGCACGAAAT 1260<br>
GCCATGGCTG TGACTTCAAG ATCGACGCTG GGGACCGCTT CCTGGAGGCC CTGGGCTTCA 1320<br>
GCTGGCATGA CACCTGCTTC GTCTGTGCGA TATGTCAGAT CAACCTGGAA GGAAAGACCT 1380<br>
TqTACTCCAA GAAGGACAGG CCTCTCTGCA AGAGCCATGC CTTCTCTCAT GTGTGAGCCC 1440<br>
CTTCTGCCCA CAGCTGCCGC GGTGGCCCCT AGCCTGAGGG GCCTGGAGTC GTGGCCCTGC 1500<br>
ATTTCTGGGT AGGGCTGGCA ATGGTTGCCT TAACCCTGGC TCCTGGCCCG AGCCTGGGCT 1560<br>
CCCGGGCCCT GCCCA 3.575<br>
Reading frame shifts caused by the addition of a 17 bp fragment (bolded,<br>
italicized and underlined) results in. an early translation stop codon at position 565-567<br>
(underlined).<br>
The derived amino acid sequence (SEQ. ID NO: 40) consists of 153 amino<br>
acids:<br>
Met Asp Ser Phe Lys Val Val Leu Glu Gly Pro Ala Pro Trp Gly Phe<br>
1 5 10 15<br>
Arg Leu Gln Gly Gly Lys Asp Phe Asn Val Pro Leu Ser Ile Ser Arg<br>
20 25 30<br>
Leu Thr Pro Gly Gly Lys Ala Ma Gln Ala Gly Val Ala Val Gly Asp<br>
35 40 45<br>
Trp Val Leu Ser Ile Asp Gly Glu Asn Ala Gly Ser Leu Thr His Ile<br>
50 55 60<br>
Glu Ala Gln Asn Lys Ile Arg Ala Cys Gly Glu Arg Leu Ser Leu Gly<br>
65 70 75 80<br>
Leu Ser Arg Ala Sln Pro Val Gln Ser Lys Pro Gln Lys Ala Ser Ala<br>
85 90 95<br>
Pro Ala Ala Asp gro Pro Arg Tvr Thr Phe Ala Pro Ser Val Sex Len<br>
100 105 110<br>
Asn Lvs Thr Ala Arq Pro Phe Glv Ala Pro Pro Pro Ala Asp Ser Ala<br>
115 120 125<br>
Pro 61n Gln Asn Slv Cys Arq Pro Leu Thr Asn Ser Arq Ser AsP Arq<br>
130 135 140<br>
Trp Ser Gln Mat Pro Ala Ser Ser Glv<br>
145 150<br><br>
This protein demonstrates 100% homologyto SEQ. ID NO: 10 until amino<br>
acid 94, where the addition of the 17 bp fragment depicted in the nucleotide sequence<br>
results in a frame shift. Over amino acids 94-153, the protein is not homologous to<br>
SEQ. ID NO: 10. Amino acids 154-457 in SEQ. ID NO: 10 are not present due to the<br>
early stop codon depicted in nucleotide sequence.<br>
EXAMPLE 32: Genomic HLMP-1 Nucleotide Sequence<br>
Applicants have identified the genomic DNA sequence encoding HLMP-1,<br>
including putative regulatory elements associated with HLMP-1 expression. The entire<br>
genomic sequence is shown in SEQ. ID. NO: 41. This sequence was derived from<br>
AC023788 (clone RP11-564G9), Genome Sequencing Center, Washington University<br>
School of Medicine, St Louis, MO.<br>
The putative promoter region for HLMP-1 spans nucleotides 2,660-8,733 in<br>
SEQ. ID NO: 41. This region comprises, among other things, at least ten potential<br>
glucocorticoid response elements ("GREs") (nucleotides 6148-6153, 6226-6231, 6247-<br>
6252, 6336-6341,6510-6515, 6552-6557, 6727-6732,6752-6757,7738-7743, and<br>
8255-8260), twelve potential Sma-2 homologues to Drosophila decapentaplegic<br>
("SMAD") binding element sites (nucleotides 3569-3575,4552-4558,4582-4588,<br>
5226-5232,6228-6234,6649-6655, 6725-6731, 6930-6936,7379-7384,7738-7742,<br>
8073-8079, and 8378-8384), and three TATA boxes (nucleotides 5910-5913,6932-<br>
6935, and 7380-7383). the three TATA boxes, all of the GREs, and eight of the<br>
SMAD binding elements ("SBEs") are grouped in the region spanning nucleotides<br><br>
5,841-8,733 in SEQ. ID NO: 41. These regulatory elements can be used, for example,<br>
to regulate expression of exogenous nucleotide sequences encoding proteins involved<br>
in the process of bone formation. This would permit systemic adrninistration of<br>
therapeutic factors or genes relating to bone formation and repair, as well as factors or<br>
genes associated with tissue differentiation and development.<br>
In addition to the putative regulatory elements, 13 exons corresponding<br>
to the nucleotide sequence encoding HLMP-1 have been identified. These exons span<br>
the following nucleotides in SEQ. ID NO: 41:<br>
Exon 1 8733-8767<br>
Exon 2. 9790-9895<br>
Exon 3 13635-13787<br>
Exon 4 13877-13907<br>
Exon 5 14387-14502<br>
Exon 6 15161-15297<br>
Exon 7 15401-15437<br>
Exon 8 16483-16545<br>
Exon 9 16689-16923<br>
Exon 10 18068-18248<br>
Exon 11 221-17-22240<br>
Exon 12 . 22323-22440<br>
Exon 13 22575-22911<br><br>
In HLMP-2 there is another exon (Exon 5A), which spans nucleotides 14887-<br>
14904.<br>
EXAMPLE 33: Expression of HLMP-1 in Intervertebral Disc Cells<br>
LIM mineralization protein-1 (LMP-1) is an intracellular protein that can<br>
direct cellular differentiation in osseous and non-osseous tissues. This example<br>
demonstrates that expressing human LMP-1 ("HLMP-1") in intervertebral disc cells<br>
increases proteoglycan synthesis and promotes a more chondrocytic phenotype. In<br>
addition, the effect of HLMP-1 expression on cellular gene expression was<br>
demonstrated by measuring Aggrecan and BMP-2 gene expression.<br>
Lumbar intervertebral disc cells were harvested from Sprague-Dawley rats by gentle<br>
enzymatic digestion.and cultured in monolayer in DMEM/F12 supplemented with 10%<br>
FBS. These cells were then split into 6 well plates at approximately 200,000 cells per<br>
well and cultured for about 6 days until the cells reached approximately 300,000 cells<br>
per well. The culture media was changed to 1 % FBS. DMEM/F12 and this was<br>
considered Day 0.<br>
Replication deficient Type 5 adenovirus comprising a HLMP-1 cDNA<br>
operably linked to a cytomegalovirus ("CMV") promoter has been previously<br>
described, for example, in U.S. Patent No. 6,300,127. The negative control adenovirus<br>
was identical except the HLMP-1 cDNA was replaced by LacZ cDNA. For a positive<br>
control, uninfected cultures were incubated in the continuous presence of BMP-2 at a<br>
concentration of 100 nanograms/milliliter.<br>
On Day 0, the cultures were infected with adenovirus for 30 minutes at 37°C<br>
in 300 microliters of media containing 1 % FBS. Fluorescence Activated Cell Sorter<br><br>
("FACS") analysis of cells treated with adenovims containing the green fluorescent<br>
protein ("GFP") gene ("AdGFP") was performed to determine the optimal dose range<br>
for expression of transgene. The cells were treated with adenovirus containing the<br>
human LMP-1 cDNA (AdHLMP-1) (atMOIs of 0,100,300,1000, or 3000) or with<br>
adenovirus containing the LacZ marker gene (AdLacZ MOI of 1000) (negative<br>
control). The culture media was changed at day 3 and day 6 after infection.<br>
Proteoglycan production was estimated by measuring the sulfated<br>
glycosaminoglycans (sGAG) present in the culture media (at day 0, 3, and 6) using a<br>
di-methyl-methylene blue ("DMMB") calorimetric assay.<br>
For quantification of Aggreean and BMP-2 mRNA, cells were harvested at<br>
day 6 and the mRNA extracted by the Trizol technique. The mRNA was converted to<br>
cDNA using reverse-transcriptase and used for real-time PCR, which allowed the<br>
relative abundance of Aggrecan and BMP-2 message to be determined. Real time<br>
primers were designed and tested for Aggrecan and BMP-2 in previous experiments.<br>
The Cybergreen technique was used. Standardization curves were used to quantitate<br>
mRNA abundance.<br>
For transfected cells, cell morphology was documented with a light<br>
microscope. Cells became more rounded with AdHLMP-1 (MOI 1000) treatment, but<br>
not with AdLacZ treatment AdLacZ infection did not significantly change cell<br>
morphology.<br>
FACS analysis of rat disc cells infected with ADGFP at MOI of 1000 showed<br>
the highest percentage cells infected (45%).<br>
There was a dose dependent increase between sGAG production and<br>
Adh LMP-1 MOI. These data are seen in Fig. 1, which shows the production of sGAG<br>
after over-expressing HLMP-1 at different MOIs in rat disc cells in monolayer cultures.<br>
The results have been normalized today 0 untreated cells. Error bars represent the<br>
standard error of the mean. As shown in Fig. 1, the sGAG production observed at day<br>
3 was relatively minor, indicating a lag time between transfection and cellular<br>
production of GAG. Treatment with AdLacZ did not significantly change the sGAG<br>
production. As also shown in Fig. 1, the optimal dose of AdhLMP-1 was at a MOI of<br>
1000, resulting in a 260% enhancement of sGAG production over the untreated ,<br>
controls at day 6. -Higher or lower-doses of AdhLMP-1 lead to a diminished response.<br>
The effect of AdhLMP-1 dosage (MOI) on sGAG production is further<br>
illustrated in Fig. 2. Fig. 2 is a chart showing rat disc sGAG levels at day 6 after<br>
treatment with AdhLMP-1 at different MOIs. As can be seen from Fig. 2, the optimal<br>
dose of AdhLMP-1 was at a MOI of 1000.<br>
Aggrecan and BMP-2 mRNA production is seen in Fig. 3. This figure<br>
demonstrates the increase in Aggrecan and BMP-2 mRNA after over-expression of<br>
HLMP-1. Real-time PCR, of mRNA extracted from rat disc cells at day 6 was<br>
performed comparing the no-treatment ("NT") cells with cells: treated with ADhLMP-1<br>
at a MOI of 250. The data in Fig. 3 are represented as a percentage increase over the<br>
untreated sample. As illustrated in Fig. 3, a significant increase in Aggrecan and BMP-<br>
2 mRNA was noted following AdhLMP-1 treatment. The increase in BMP-2<br>
expression suggests that BMP-2 is a down-stream gene that mediates HLMP-1<br>
stimulation of proteoglycan synthesis.<br>
These data demonstrate that transfection with AdhLMP-1 is effective in<br>
increasing proteoglycan synthesis of intervertebral disc cells. The dose of virus leading<br>
to the highest transgene expression (MOI1000) also leads to the highest induction of<br>
sGAG, suggesting a correlation between HLMP-1 expression and sGAG induction.<br>
These data indicate that HLMP-1 gene therapy is a method of increasing proteoglycan<br>
synthesis in the intervertebral disc, and that HLMP-1 is a agent for treating disc<br>
disease.<br>
Fig. 4A is a chart showing HLMP-1 mRNA expression 12 hours after<br>
infection with Ad-hLMP-1 at different MOls. In Fig..4A, exogenous LMP-1<br>
expression was induced with different doses (MOI) of the Ad-hLMP-1 virus and<br>
quantitated with realtime PCR. The data is normalized to HLMP-1 mRNA levels from<br>
Ad-LMP-1 MOI 5 for comparison purposes. No HLMP-4 was detected in negative<br>
control groups, the no-treatment ("NT") or Ad-LacZ treatment ("LacZ"). HLMP-1<br>
mRNA levels in a dose dependent fashion to reach a plateau of approximately 8 fold<br>
with a MOI of 25 and 50.<br>
Fig. 4B is a chart showing the production of sGAG in medium from 3 to 6<br>
days after infection. DMMB assay was used to quantitate total sGAG .production<br>
between days 3 to 6 after infection. The data in Fig. 4B is normalized to the control<br>
(i.e., no treatment) group. As can be seen from Fig. 4B, there was a dose dependent<br>
increase in sGAG. with the peak of approximately three fold increase above control<br>
reached with a MOI of 25 and 50. The negative control, Ad-LacZ at a MOI of 25, lead<br>
to no increase in sGAG, In Fig. 4B, each result is expressed as mean with SD for three<br>
samples.<br>
Fig. 5 is a chart showing time course changes of the production of sGAG. As<br>
can be seen from Fig. 5, on day 3 sGAG production increased significantly at a MOI of<br>
25 and 50. On day 6 there was a dose dependent increase in sGAG production in<br>
response to AdLMP-1. The plateau level of sGAG increase was achieved at a MOI of<br>
25. As can also be seen from Fig. 5, treatment with AdLacZ ("LacZ") did not<br>
t<br>
significantly change the sGAG production. Each result is expressed as mean with SD<br>
for six to nine samples. In Fig. 5, "**" indicates data points for which the P value is 
0.01 versus the untreated control.<br>
Figs. 6A and 6B are charts showing gene response to LMP-1 over-expression<br>
in rat annulus fibrosus cells for aggrecan and BMP-2, respectively. Quantitative<br>
realtime PCR was performed on day 3 after infection with Ad-LMP-1 ("LMP-1") at a<br>
MOI of 25. As can be seen from Figs. 6A and 6B, the gene expression of aggrecan and<br>
BMP-2 increased significantly after infection with Ad-LMP-1 compared to the<br>
untreated control ("NT"). Further, treatment with AdLacZ ("LacZ") at a MOI of 25 did<br>
not significantiy change the gene expression of either aggrecan or BMP-2 compared to<br>
the untreated control. In Figs. 6A and 6B, each result is expressed as mean with SD for<br>
six samples. In Figs. 6A and 6B, "**" indicates data points for which the P value is P<br>
Fig. 7 is a graph showing the time course of HLMP-1 mRNA levels in rat<br>
annuius fibrosus cells after infection.with AdLMP-1 at a MOI of 25. The data is<br>
expressed as a fold increase above a MOI of 5 of AdLMP-1 after standardization using<br>
18S and replication coefficient of over-expression LMP-1 primer. As can be seen from<br>
Fig. 7, HLMP-1 mRNA was upregulated significantly as early as 12'hours after<br><br>
infection. Further, there was a marked increase of expression levels between day 1 and<br>
day 3. Each result in Fig. 7 is expressed as mean with SD for six samples.<br>
Fig. 8 is a chart showing changes in mRNA levels of BMPs and aggrecan in<br>
response to HLMP-1 over-expression. The mRNA levels of BMP-2, BMP-4, BMP-6,<br>
BMP7, and aggrecan were determined with realtime-PCR at. different time points after<br>
infection with Ad-hLMP-1 at a MOI of 25. As can be seen from Fig. 8, BMP-2 mRNA<br>
was upregulated significantly as early as 12 hours after infection with AdLMP-1. On<br>
the other hand, Aggrecan mRNA was not upregulated until 3 day-after infection. Each<br>
result is expressed as mean with SD for six samples. In Fig. 8, "**" indicates data<br>
points for which the P value is 
control.<br>
Fig. 9 is a graph showing the time course of sGAG production enhancement<br>
in response to HLMP-1 expression. For the data in Fig. 9, rat annulus cells were<br>
infected with Ad-hLMP-1 at a MOI of 25. The media was changed every three days<br>
after infection and assayed for sGAG with the DMMB assay. This data shows that<br>
sGAG production reaches a plateau at day 6 and is substantially maintained at day 9.<br>
Fig. 10 is a chart showing the effect of noggin (a BMP antagonist) on LMP-1<br>
mediated increase in sGAG production. As seen in Fig. 10, infection of rat annulus<br>
cells with Ad-LMP-1 at a MOI of 25 led to a three fold increase in sGAG produced<br>
between day 3 and day 6. This increase was blocked by the addition of noggin (a BMP<br>
antagonist) at concentration of 3200 ng/ml and 800 ng/m. As shown in Fig. 10,<br>
however, noggin did not significantly alter sGAG production in uninfected cells. As<br>
can also be seen in Fig. 10, stimulationwith rhBMP-2 at 100 ng/ml led to a 3 fold<br>
increase in sGAG production between day 3 and day 6 after addition of BMP-2.<br>
Noggin at 800 ng/ml also blocked this increase-<br>
Fig. 11 is a chart showing the effect of LMP-1 on sGAG in media after day 6<br>
of culture in monolayer. The data points are represented as fold increase above<br>
untreated cells. As shown in Fig. 11, LMP-1 with the CMV promoter when delivered<br>
by the AAV vector is also effective in stimulating glycosaminoglycan synthesis by rat<br>
disc cells in'monolayer.<br>
TABLE 2: Primer Sequences for RT-PCR &amp; Real-time PCR of SYBR Green<br>
____________Primer____________________________Sequence_____________<br>
Aggrecan (forward) AGGATGGCTTCCACCAGTGC<br>
Aggrecan (reverse) TGCGTAAAAGACCTCACCCTCC<br>
BMP-2 (forward) CACAAGTCAGTGGGAGAGC<br>
BMP-2 (reverse) GCTTCCGCTGTTTGTGTTTG<br>
GAPDH (forward) ACCACAGTCCATGCCATCAC<br>
GAPDH (reverse) TCCACCACCCTGTTGCTGTA<br>
GAPDH in Table 2 denotes glyceraldehyde phosphate dehydrogenase<br>
TABLE 3: Primer and Probe sequences for Real-time PCR of TaqMan®<br>
____________Primer___________________________ Sequence___________________<br>
Overexpression LMP-1 (forward) AATACGACTCACTATAGGGCTCGA<br>
Overexpression LMP-1 (reverse) GGAAGCCCCAAGGTGCT<br>
Overexpression LMP-1 (probe) FAM-AGCCGGCATCATGGATTCCTTCAA-TAMRA<br>
TaqMan® Ribosomal RNA Control Reagents (Part number 4308329, Applied<br>
Biosystems, Foster City, CA, U.S.A.) were used for the forward primer, reverse primer<br>
and probe of 18S ribosomal RNA (rRNA) gene.<br>
Mechanism of Bone Formation - Evidence for Induction of Multiple BMPs<br>
Animal and in vitro studies have demonstrated a striking and consistent bone-<br>
forming effect with ex vivo gene transfer of the DM Mineralization Protein-1 (LMP-1)<br>
cDNA using relatively low doses of adenoviral or plasmid vectors (Boden, et al.,<br>
"Volvo Award in Basic Sciences: Lumbar Spine Fusion by Local Gene Therapy with a<br>
cDNA Encoding a Novel Osteoinductive Protein (LMP-1)", Spine. 23,2486-2492<br>
(1998)). Little is known, however, about the mechanism of action of LMP-1, how long<br>
the transduced cells survive, or which osteoinductive growth factors and cells<br>
participate in the induction of new bone and osteoblast differentiation (see Boden, et<br>
al., "LMP-1, A LIM-Domain Protein, Mediates BMP-6 Effects on Bone Formation",<br>
. Endocrinology. 139,5125-5134 (1998) and Boden, et al., Spine. 23,2486-2492<br>
(1998)). Furthermore, the mechanism of bone formation in vivo (i.e., endochondral vs.<br>
membranous) has not been determined. Understanding the mechanism of LMP-1<br>
action would be helpful for optimal control of LMP-1 induced bone formation in. the<br>
clinical setting and to further the understanding of intracellular signaling pathways<br>
involved with osteoblast differentiation.<br>
LMP-1 is a member of the heterogeneous LIM domain family of proteins and<br>
is the first member to be directly associated with osteoblast differentiation (Kong, et<br>
al., "Muscle LIM Protein Promotes Myogenesis by Enhancing the Activity of MyoD,"<br>
Mol.Cell.Biol.. 17,4750-4760 (1997). LMP-1 was identified in messenger<br>
ribortucleic acid (mRNA) from rat calvarial osteoblasts stimulated by glucocorticoid<br>
and later isolated from an osteosarcoma complementary deoxyribonucleic acid (cDNA)<br>
library (Boden, et al., Endocrinology, 139, 5125-5134 (1998)). Unlike BMPs which<br>
are extracellular proteins that act through cell surface receptors, LMP-1 is thought to be<br>
an intracellular signaling molecule that is directly involved in osteoblast differentiation<br>
(Boden, et al., Spine. 20,2626-2632 (1995); Cook, et al.., "Effect of Recombinant<br>
Human Osteogenic Protein-1 on Healing of Segmental Defects in Non-Human<br>
Primates", J. Bone Joint Sure.. 77-A, 734-750 (1995); Schimandle, et al.,<br>
"Experimental Spinal Fusion with Recombinant Human Bone Morphogenetic Protein-<br>
2 (rhBMP-2)". Spine. 20,1326-1337 (1995)). Thus, the therapeutic use of LMP-1 may<br>
involve gene transfer of its cPNA. On the basis of its association with bone<br>
development and the results of suppression and over-expression experiments, LMP is<br>
considered to induce secretion of soluble factors that convey its osteoinductive. activity,<br>
and to be a critical regulator of osteoblast differentiation, and maturation in vitro and in<br>
vivo.<br>
Described below are studies conducted to: 1) to identify candidates for the<br>
secreted osteoinductive factors induced by LMP-1; 2) to describe the histologic<br>
sequence and type of bone formation induced by LMP-1; and 3) to determine how long<br>
the implanted cells overexpressing LMP-1 survive in vivo<br>
In the present study, human lung carcinoma (A549) cells were used to<br>
determine if LMP-1 overexpression would induce expression of bone morphogenetic<br>
. proteins in vitro. Cultured A549 cells were infected with recombinant replication<br>
deficient human adenovirus type 5 containing the LMP-1 or LacZ cDNA. Cells were<br>
analyzed using immunohistochemistry after 48 hours. Finally, 1.6 athymic rats received<br>
subcutaneous implants consisting of collagen discs loaded with human buffy coat cells<br>
that were infected with one of the above two viruses. Rats were euthanized at intervals<br>
and explants analyzed by histology and immunohistochemistry.<br>
Materials and Methods<br>
Phase 1: Detection of LMP-1 induced osteoinducu've factors in vitro.<br>
The human LMPr-1 cDNA with the human cytomegalovirus promoter was cloned into a<br>
transfer vector and subsequently was transferred into a recombinant replication<br>
deficient (El, E3 deleted) adenovirus as previously described.<br>
Human lung carcinoma cells (A549) are known for their high infectivity by<br>
human Type 5 adenovirus. These cells were seeded at a density of 50,000 cells/cm2 on<br>
2 well chamber slides (Nalge Nunc International, Naperville, IL) and were propagated<br>
in F12 Kaighn's medium (Gibco BRL), supplemented with 10% fetal bovine serum<br>
(FBS), and grown in a humidified 5 % CO2 incubator at 37 °C,<br>
The A549 cells were infected for 30 minutes at 37 °C on chamber at a<br>
multiplicity of infection (MOI) of 10 pfu/cell. Medium with 10 % FBS was added and<br>
the cells were grown for 48 hours at 37 °C. The cells were infected with either<br>
AdLMP-1 (active LMP) or AdLacZ (Adpgal-adenoviral control) each driven by the<br>
human cytomegalovirus promoter (Boden, et al. Endocrinology. 139,5125-5134<br>
(1998) and Boden, et al., Spine. 23,2486-2492 (1998))). As an additional negative<br>
control, some cells were not infected with adenovirus (no treatment control). After 48<br>
hours, the cells on chamber slides were fixed for 2 minutes in 50 % acetone/50 %<br>
methanol, and then were analyzed by immunohistochemistry (described below) using<br>
antibodies specific for LMP-1, BMP-2, BMP-4, BMP-6, BMP-7, TGF-ßl, MyoD, and<br>
Type II collagen.<br>
Phase 2: Histologic sequence of bone formation in vivo<br>
The experimental protocol was reviewed and approved by the Institutional<br>
Animal Care and Use Committee and the Human Investigation Committee. Rabbit or<br>
human peripheral blood (3 mL) was obtained by venipuncture and the buffy-coat cells<br>
were isolated by simple centrifugation at 1200 x g for 10 minutes. The cells were<br>
counted, and 1 x 106 cells were infected with adenovirus (AdLMP-1 or AdLacZ) at an<br>
MOI of 4.0 pfu/cell for ten minutes at 37 °C. After infection, the cells were<br>
resuspended in a final volume of 80 µl and applied to a 7 mm X 7 mm X 3 mm<br>
collagen disc (bovine type I collagen).<br>
Sixteen athymic rats that were 4-5 weeks old were obtained (Harlan,<br>
Indianapolis, IN) and housed in sterile conditions. Rats were anesthetized by inhalation<br>
of 1-2 % isoflurane. Four 10 mm skin incisions were made on the chest of athymic<br>
rats, pockets were developed by blunt dissection, and a collagen disc containing cells<br>
was implanted into each pocket Implants consisted of a collagen disc loaded with<br>
buffy coat cells infected with either AdUMP-1 (2 per rat) or AdLacZ (2 per rat). The<br>
skin was closed with resorbable suture. Each animal was sacrificed at one, three, five,<br>
seven, ten, fourteen, twenty-one and twenty-eight days after implantation, and explants<br>
were analyzed by histology and immnohistochernistry.<br>
The specimens were fixed for 24 hours in 10 % neutral buffered formalin.<br>
The specimens were prepared for undecalcified or decalcified sectioning. The<br>
specimens for undecalcified sections were dehydrated through graded strengths of<br>
ethanol and embedded in paraffin. The specimens at 21 and 28 days after implantation<br>
were decalcified with 10 % ethylenediaminetetraacetic acid (EDTA) solution for 3 to 5<br>
days. After decalcification, the specimens were dehydrated<br>
through graded strengths of ethanol and embedded in paraffin. Specimens were<br>
sectioned at a thickness of 5 micrometers on a microtome (Reichert Jung GmbH,<br>
Heidelberg, Germany). Sections were subjected to hematoxylin and eosin staining,<br>
Goldner's trichrome staining, and immimohistochemical study using antibodies<br>
specific for BMP-4, BMP-7, CD-45 and type I collagen.<br>
Preparation of Primary Antibodies<br>
Anti-LMP-1 Antibody: The anti-LMP-1 antibody is an affinity-purified rabbit<br>
polyclonal antibody mapping within an internal region of human LMP-1, and reacts<br>
with LMP-1 of rabbit and human origin. This antibody was used for the identification<br>
of LMP-1 protein at a dilution of 1:500 or 1:1000.<br>
Anti-BMP-2,Anti-BMP-4, Anti-BMP-6, Anti-BMP-7 and Anti-TGF-0<br>
Antibodies: Polyclonal goat anti-BMP-2, anti-BMP-4, anti-BMP-6, anti-BMP-7, and<br>
anti-TGF-pi antibodies (Santa Cruz Biotechnology, Inc., Santa Cruz, California) cross-<br>
react with mouse, rat and human BMPs. The anti-BMP-2, anti-BMP-4 and anti-BMP-6<br>
antibodies were raised against an epitope mapping at the amino terminus of BMP-2,<br>
BMP-4 and BMP-6 of human origin. The anti-BMP-7 antibody was an affinity-<br>
purified goat polyclonal antibody mapping within an internal region of human BMP-7.<br>
The anti-TGF-µl antibody was an affinity purified goat polyclonal antibody mapping at<br>
the carboxy terminus of the precursor form of human TGF-µl. These antibodies were<br>
used at a dilution of 1:100 and 1: 500 or 1:1000.<br>
Anti-CD45 Antibody: A monoclonal mouse anti-human leukocyte common<br>
antigen (LCA), CD-45 antibody (purified IgG 1, kappa; DAKO Co., Carpinteria,<br>
California) consists of two antibodies, PD7/26 and 2B11, directed against different<br>
epitopes. The PD7/26 was derived from human peripheral blood lymphocytes<br>
maintained on T-cell growth factor. The 2B11 was derived from neoplastic cells<br>
isolated from T-cell. lymphoma or leukemia. Both antibodies bound to lymphocytes<br>
and monocytes at the 94-96 % range when tested by immnofluorescence. ha the<br>
present study, this antibody was used at a dilution of 1:100 for the identification of<br>
human leukocytes.<br>
Anti-Collagen Type I Antibody: A monoclonal anti-type I collagen antibody<br>
(mouse IgG 1 isotype; Sigma Chemical Co., Saint Louis, Missouri) was derived from<br>
the collagen type I hybridoma produced by the fusion of mouse myeloma cells and<br>
splenocytes from BALB/c mice immunized with bovine skin type I collagen. The<br>
antibody reacts with human, bovine, rabbit, deer, pig and rat type I collagen, and was<br>
used at a dilution of 1:100-<br>
. Anti-Collagen Type II Antibody: A polyclonal rabbit anti-type II collagen<br>
antibody (Santa Cruz Biotechnology, Inc., Santa Cruz, CA) was raised against an<br>
epitope corresponding to theamino terminus of the alpha 1, chain of human type II<br>
collagen. The antibody reacts with type II collagen alpha 1 chain of mouse, rat, and<br>
human origin and was used at a dilution of 1:1000.<br>
Anti-MyoD Antibody: A polyclonal rabbit anti-MyoD antibody (Santa Cruz<br>
Biotechnology, Inc., Santa Cruz, CA) was raised against an epitope corresponding to<br>
amino acids 1-318 representing full length MyoD protein of mouse origin. The<br>
antibody reacts with MyoD (and not myogenin, Myf-5, or Myf-6) of mouse, rat, and<br>
human origin and was used at a dilution of 1:1000.<br>
Imrnunohistochemical staining<br>
The staining procedure was performed using the labeled streptavidin-biotin<br>
method (LSAB method). A kit (Universal LSAB Kit, Peroxidase; DAKO Co.,<br>
Carpinteria, California) was used for immunostaining with antibodies against LMP-1,<br>
BMP-2, BMP-4, BMP-6, BMP-7, TGF-pi, CD-45, MyoD, type I collagen,<br>
and type II collagen. Appropriate biotinylated secondary antibodies were used<br>
depending on the animal in which the primary antibody was raised. Endogenous<br>
peroxidase was blocked with methanol containing 0.3 % hydrogen peroxide.<br>
Specimens were incubated with phosphate buffered saline (PBS) containing either 5 %<br>
normal rabbit serum or 5 % normal goat serum, and 1 %• bovine serum albumin for 15<br>
minutes at room temperature to avoid nonspecific binding and then with the<br>
appropriate concentrations of primary antibodies at 4 "C overnight in a humidified<br>
chamber. After washing with PBS three times for 5 minutes, followed by incubation<br>
with biotinylated secondary antibody and streptavidin-biotin-peroxiadase complex in a<br>
humidified chamber for 10 minutes at room temperature, color was developed using<br>
3,3'-diaminobenzidine tetrachloride (DAB; DAKO Co., Carpinteria, California).<br>
Finally, the sections were counterstained by hematoxylin. As negative controls each<br>
primary antibody was incubated at room temperature for 3 hours with the<br>
corresponding blocking peptide (Santa Cruz Biotechnology, Inc., Santa Cruz,<br>
California) (1:40 dilution) prior to incubation with the specimens. In some<br>
experiments primary antibody alone or secondary antibody alone were used<br>
as additional negative controls.<br>
Results<br>
Phase 1: Detection of LMP-1 induced osteoinductive factors in vitro.<br>
The A549. cells infected with AdLMP-1 showed strong intracellular staining<br>
for LMP-1 protein as shown in Figs. 12A-12D. Figs. 12A-12D are photomicrographs of<br>
immunohistochemical staining for LMP-1 protein in A549 cells 48 hours after infection<br>
with AdLMP-1 (Fig. 12A), Adpgal (Fig. 12C), or untreated cells (Fig. 12D). As can be<br>
seen from Figs. 12A, 12C and 32D, a specific intracellular reaction was seen in cells<br>
infected with AdLMP-1 (Fig. 12A) but not in either control (Figs. 12C and 12D). The<br>
possibility of non-specific reaction was eliminated since pre-exposure of the primary<br>
antibody to a blocking peptide eliminated the positive intracellular staining (Fig. 12B).<br>
The photomicrographs of Figs. 12A-12D were taken at original magnifications of X132.<br>
Strong staining for BMP-2, BMP-4 and BMP-7 was observed in the AdLMP-<br>
1 treated cells, especially in the cytoplasm, as shown in Figs. 13A-13F. Figs. 13A-13F<br>
are photomicrographs of immunohistochemical staining of A549 cells 48 hours after<br>
infection with AdLMP-1 (upper panels - Figs. 13A, 13B and 13C) or Adpgal (lower<br>
panels - Figs. 13D, 13E, and 13F). In AdLMP-1 treated cells there was specific<br>
intracellular staining for BMP-2 (Fig. 13A), BMP-4 (Fig. 13B), and BMP-7 (FIB. 13C)<br>
which was not present in AdPgal treated cells (Figs. 13D, 13E, and 13F, respectively).<br>
The photomicrographs of Figs. 13-13F<br>
were taken at original magnifications of X132.<br>
The cells treated with AdLMP-1 also stained positive with anti-BMP-6 and<br>
t<br>
anti-TGF-ßl antibodies as shown in Figs. 3A-3D. Figs. 14A-14D are photomicrographs<br>
of immunohistochemical staining of A549 cells 48 hours after infection with either<br>
AdLMP-1 (upper panels - Figs. 14A and 14B) or Adpgal (lower panels - Figs. 14C and<br>
14D). In AdLMP-1 treated cells there was specific intracellular staining for BMP-6<br>
(Fig. 14A) and TGF-ßl (Fig. 14B) which was not present in AdPgal treated cells (Figs.<br>
14C and 14D, respectively). However, the reactions were somewhat less intense than<br>
that seen with other BMPs. In both the Adpgal infected and the "untreated controls, the<br>
cells had no specific reaction for LMP-1, any of the BMPs, or TGF-PL A blocking<br>
peptide for each antibody confirmed that the reaction was specific. There was no<br>
specific reaction with the anti-type IT collagen or anti-MyoD antibodies (data not<br>
shown). The photomicrographs of Figs. 14A-14D were taken at an original<br>
magnification of X132.<br>
Phase 2: Histologic sequence of bone formation in vivo<br>
Histological Examination - Immunohistochemical Staining.<br>
Immunolocalization of leukocytes. At one and three days after implantation,<br>
cells stained by anti-CD-45 antibody were abundantly present in buffy coat preparations<br>
within both the AdLMP-1 (active) and Adpgal (control) treated implants as shown in<br>
Figs. 15A-15D.<br>
Figs. 15A-15D are photomicrographs of immunohistochemical staining for<br>
the leukocyte surface marker CD45 in human buffy coat cells infected with AdLMP-1<br>
(upper panels - Figs. 15A and 15B) or Adpgal (lower panels - Figs. 15C and 15D)<br>
excised at 3 days (Figs. 15A and 15C) or 5 days (Figs. 15B and 15D) following<br>
implantation with a collagen matrix subcutaneously on the chest of an athymic rat. The<br>
number of cells with specific staining for CD45 antigen decreased rapidly in both<br>
treatment groups. This observation suggests that the implanted human cells did not<br>
survive very long and the bone formation likely depended on influx of host cells. The<br>
number of cells staining with the specific anti-human-CD-45 reaction decreased, after :<br>
Day 3, especially in the center of the implants. Positive staining still was observed in<br>
. the periphery of the implant at five days, but ten days after implantation there were few<br>
cells staining for anti-CD-45. The pattern of decreased staining was the same in active<br>
and control implants. The photomicrographs of Figs. 15A-15D were taken at an<br>
original magnification of X132.<br>
Immunolocalization of BMPs. In the AdLMP-1 treated implants three and<br>
five days after implantation, immunohistochemistry revealed strong BMP-4 (Figs. 16A-<br>
16D) and BMP-7 (Figs. 17A-17D) staining within cells on the collagen fibers.<br>
Figs. 16A-16D are photomicrographs of immunohistochemical staining for<br>
BMP-4 in human buffy coat cells infected with AdLMP-1 (upperpanels-Figs. l6Aand<br>
16B) or Adßgal (lower panels - Figs. 16C and 16D) excised at 3 days (Figs. 5A and 5C)<br>
or 5 days (Figs. 5B and 5D) following implantation with a collagen matrix<br>
subcutaneously on the chest of an athymic rat. In AdLMP-1 treated cells there was<br>
specific intracellular staining for BMP-4 which was not<br>
present in Adpgal treated cells. The photomicrographs of Figs. 16A-16D were taken at<br>
an original magnification of X132.<br>
Figs. 17A-17D are photomicrographs of immunohistochemical staining for<br>
BMP-7 in human buffy coat cells infected with AdLMP-1 (upper panels - Figs. 17A and<br>
17B) or Adpgal (lower panels - Figs. 17C and 17D) excised at 3 days (Figs. 17A and<br>
17C) or 5 days (Figs. 17B and 17D) following implantation with a collagen matrix<br>
subcutaneously on the chest of an atbymic rat In AdLMP-1 treated cells there was<br>
specific intracellular staining for BMP-7 which was not present in Adpgal treated cells.<br>
The photomicrographs of Figs. 17A-17D were taken at an original magnification of<br>
X132.<br>
As can be seen from Figs. 16A-16D and 17A-17D., there was no specific<br>
staining for BMP-4 or BMP-7 in cells on the Adpgal (control) implants. Moreover, the<br>
strong staining with anti-BMP-4 and anti-BMP-7 antibodies was also seen at each time<br>
point beyond 10 days in the AdLMP-1 implants. Strong staining for BMP-4 and BMP-<br>
7 was observed in two temporal phases; the first phase was in a limited number of buffy<br>
coat cells in the early days (i.e., three and five days after implantation) and the second<br>
was seen after Day 10 in osteoblast-like cells surrounded by matrix that most likely<br>
were responding cells rather than transplanted buffy coat cells as shown in Fig. 18.<br>
Fig. 18 is a high power photomicrograph of immunohistochemical staining<br>
for BMP-7 in human buffy coat cells infected with AdLMP-1 excised at 14 days<br>
following implantation with a collagen matrix subcutaneously on the chest of an<br>
athymic rat There is more abundant staining for BMP-7 compared with earlier time<br>
points which is now associated with most of the cells in. close proximity to the<br>
formation of new bone matrix. The photomicrographs of Fig. 18 was taken at an<br>
original magnification of X66.<br>
Immunolocalization of Type 1 collagen: Strong staining for anti-type I<br>
collagen antibody was observed in the AdLMP-1 implants seven, ten, fourteen, twenty-<br>
one and twenty-eight days after implantation. At the early time points, the specific<br>
reaction was seen adjacent to osteoblast-like cells and on the periphery of the cells<br>
hemselves. There was minimal staining for type I collagen in the control implants<br>
treated with Adpgal.<br>
Hematoxylin and Eosin &amp; Goldner's Trichrome Staining..<br>
Results were the same whether using rabbit or human buffy coat cells. To<br>
avoid duplication, the following description and corresponding illustrations will be for<br>
the human donor cells. At one and three days after implantation; the Ad-LMP implants<br>
had increased numbers of cells at the edge of the implant as shown in Figs. 19A-19D.<br>
Figs. 19A-19D are photomicrographs of human, buffy coat cells infected with<br>
AdLMP-1 (upper panels - FIGS 19A and 19B) or Adpgal (lower panels - Figs., 19C and<br>
19D) excised at 1 day (Figs. 19A and 19C) or 3 days (Figs. 19B and 19D) following<br>
implantation in a collagen matrix subcutaneously on the chest of an athymic rat the<br>
density of cells on the periphery of the implant was greater in the AdLMP-1 implant at<br>
both time points suggesting migration of host cells. The photomicrographs of Figs.<br>
19A-19D were taken at an original magnification of X33 using Goldner trichrome.<br>
In the AdPgal controls, fewer cells were seen at the periphery at the same<br>
time point (i.e., one and three days after implantation). These observations suggest that<br>
host cells migrated into the implants with cells expressing LMP-1 as shown in Figs.<br>
20A and 20B. These cells were a mixture of monocytes and polymorphonuclear<br>
leukocytes. Figs. 20A and 20B are high power photomicrographs of human buffy coat<br>
cells infected with AdLMP-1 or Adpgal excised at 1 day following implantation in a<br>
collagen matrix subcutaneously on the chest of an athymic rat. As shown in Fig. 20A,<br>
there were relatively few cells (arrow) on the periphery of the collagen (C) implants<br>
containing cells infected with Adpgal. Buffy coat cells and red cell ghosts could be<br>
seen in the center of the implant As shown in Fig. 20B, the density of nucleated cells<br>
on the periphery of the collagen (C) implant was greater in the AdLMP-1 implant<br>
suggesting migration of host cells from the surrounding soft tissues. The cells included<br>
monocytes, polymorphonuclear cells, and histiocyte appearing cells. The<br>
photomicrographs of<br>
Figs. 20A and 20B were taken at original magnifications of X100 (Fig. 20A) and XI60<br>
(Fig. 20B) using hematoxylin and eosin.<br>
Figs. 21A-21J are photomicrographs of human buffy coat cells infected with<br>
AdLMP-1 (upper panels - Figs. 21A-21E) or Adpgal (lower panels - Figs. 21F-21 J)<br>
excised at various time points following implantation with a collagen matrix<br>
subcutaneously on the chest of an athymic rat. The progression of membranous bone<br>
formation was evident with mineralized matrix seen by day 7 (Fig. 21C). No bone<br>
formation was seen in implants containing cells infected with Adpgal (Figs. 21F-21 J).<br>
The photomicrographs of Figs. 21A-21J were taken at original magnifications of X33<br>
using Goldner trichrome.<br>
As shown in Fig. 21A-21E, there were less buffy coat cells associated with<br>
the collagen fibers over time, and the number of cells surviving in the center of the<br>
Adpgal treated implants was diminished by five days after implantation (Fig. 21C).<br>
Figs. 22A-22C are high power photomicrographs of human buffy coat cells<br>
infected with AdLMP-1 excised at various time points following implantation with a<br>
collagen matrix subcutaneously on the chest of an athymic rat As can be seen from<br>
Fig. 22A, new mineralized bone matrix (B) was visible adjacent to osteoblast-like cells<br>
(arrows) between collagen fibers (C) at the periphery of the AdLMP-1 implants seven<br>
days after implantation. There was rapid mineralization of the matrix surrounding<br>
osteoblast-like cells (arrowheads) without classic osteoid seams and without any<br>
specific orientation. As can be seen from Fig. 22B, mature new bone'had formed in the<br>
spaces located throughout the AdLMP-1 implants and most of the collagen scaffold was<br>
resorbed by day 28. Osteoblasts (arrowheads) were seen covering surfaces of osteoid<br>
and newly-formed bone while osteoclasts (OC) could be seen remodeling the primary<br>
woven bone (B). Finally, as can be seen from Fig. 22C, hematopoietic marrow tissue<br>
was also seen forming within the bone (B) including a marrow stroma (S) and blood<br>
vessels (V). The photomicrographs of Figs. 22A-22C were taken at original<br>
magnifications of X160 using Goldner trichrome.<br>
As can be seen from Fig. 22A, new bone matrix was visible adjacent to<br>
osteoblast-like cells between collagen fibers at the periphery of the AdLMP-1 implants<br>
seven days after implantation. There was rapid mineralization of the surrounding<br>
matrix without classic osteoid seams without any specific orientation. The lack of<br>
organized bone orientation was not surprising given the fact that these were<br>
subcutaneous implants that were not significantly loaded. More abundant osteoblast-<br>
like cells were observed in the AdLMP-1 implants ten days after implantation and were<br>
growing into the voids between the collagen fibers. By fourteen days after implantation,<br>
osteoblast-like cells occupied the central region of the AdLMP-1 implants. In contrast,<br>
fibroblast-like cells were filling the voids of the collagen in the Adpgal treated implants.<br>
Twenty-one days after implantation, new bone matrix was mineralized and was forming<br>
in most or all of the central regions of the AdLMP-1 implants. Mature new bone had<br>
formed in the spaces located in the most central regions of the AdLMP-1 implants<br>
twenty-eight days after implantation. Osteoblasts were seen covering surfaces of<br>
osteoid and newly-formed bone while osteoclasts could be seen remodeling the primary<br>
woven bone (Fig. 22B). Hematopoietic marrow tissue was also seen forming within the<br>
bone (Fig. 22C). In the Adpgal treated controls, the implanted collagen was mostly<br>
resorbed by day 28 and was replaced with fibrous tissue.<br>
As set forth above, in vitro experiments with A549 cells showed that<br>
AdLMP-1 infected cells express elevated levels of BMP-2, BMP-4, BMP-6, BMP-7 and<br>
TGF-J31 protein. Human bufiy coat cells infected with AdLMP-1 also demonstrated<br>
increased levels of BMP-4 and BMP-7 protein 72 hours after ectopic implantation in<br>
athymic rats, confirming the in vitro hypothesis.<br>
Based on the results of the above study, it has therefore been shown that the<br>
osteoinductive properties of LMP-1 involve the synthesis of several BMPs and the<br>
recruitment of host cells which differentiate and participate in direct membranous bone<br>
formation. Accordingly, gene therapy with the LMP-1 cDNA may provide an<br>
alternative to implantation of large doses of single BMPs to induce new bone formation.<br>
According to the invention, a method of inducing the expression of one or<br>
more bone morphogenetic proteins or transforming growth factor-p proteins (TGF-<br>
Ps) in a cell is provided. The method includes transfecting a cell with an isolated<br>
nucleic acid comprising a nucleotide sequence encoding a LIM mineralization<br>
protein operably linked to a promoter. The expression of one or more proteins<br>
selected from the group consisting of BMP-2, BMP-4, BMP-6, BMP-7, TGF-ßl and<br>
combinations thereof can be induced according to the invention. The isolated<br>
nucleic acid can be a nucleic acid which can hybridize under standard conditions to a<br>
nucleic acid molecule complementary to the full length of SEQ. ID NO: 25; and/or a<br>
nucleic acid molecule which can hybridize under highly stringent conditions to a '<br>
nucleic acid molecule complementary to the full length of SEQ. ID NO: 26. The cell<br>
can be a buffy coat cell, a stem cell (e.g., a mesenchymal stem cell or a pluripotential<br>
stem cell) or an intervertebral disc cell (e.g., a cell of the nucleus pulposus or a cell<br>
of the annulus fibrosus). The cell can be transfected ex vivo or in vivo. For example,<br>
the cell can be transfected in vivo by direct injection of the nucleic acid into an<br>
intervertebral disc of a mammal.<br>
The LIM mineralization protein encoded by the nucleotide sequenc e can<br>
be RLMP, HLMP-1, HLMP-1s, HLMP-2, or HLMP-3. The promoter can be a<br>
cytomegalovirus promoter. According to one embodiment of the invention, the LIM<br>
mineralization protein is an LMP-1 protein. The nucleic acid can be in a vector (e.g.,<br>
an expression vector such as a plasmid). The vector can also be a virus such as an<br>
adenovirus or a retrovirus. An exemplary adenovirus that can be used according to<br>
the invention is AdLMP-1.<br>
According to a second aspect of the invention, a cell which overexpresses<br>
one or more bone morphogenetic proteins or transforming growth factor-P proteins is<br>
provided. The cell can be a cell which overexpresses one or more proteins selected<br>
from the group consisting of BMP-2, BMP-4, BMP-6, BMP-7, TGF-ß1 and<br>
combinations thereof- The cell can be a buffy coat cell, an intervertebral disc cell, a<br>
mesenchymal stem cell or a pluripotential stem cell. An implant comprising a cell as<br>
set forth above and a carrier material is also provided. Also provided according to<br>
the invention is a method of inducing bone formation in a mammal comprising<br>
introducing a cell or an implant as set forth above into the mammal and a method of<br>
treating intervertebral disc disease in a mammal comprising introducing a cell as set<br>
forth above into an intervertebral disc of the mammal.<br>
Overexpression of a bone morphogenetic protein or a transforming growth<br>
factor-P protein in the context of the invention refers to a cell which expresses that<br>
protein at a level greater than normally present in that particular cell (e.g., expression<br>
of the protein is at a level greater than the level in a cell which has not been<br>
transfected with a nucleic acid comprising a nucleotide sequence encoding a LIM<br>
mineralization protein operably linked to a promoter). The cell can be a cell which<br>
normally expresses one or more of the bone morphogenetic proteins or transforming<br>
growth factor-p proteins. The cell can also be a cell which does not normally express<br>
one or more of the bone morphogenetic proteins or transforming growth factor-p<br>
proteins.<br>
While the foregoing specification teaches the principles of the present<br>
invention, with examples provided for the purpose of illustration, it will be<br>
appreciated by one skilled in the art from reading this disclosure that various changes<br>
in form and detail can be made without departing from the true scope of the<br>
invention.<br>
We Claim:<br>
1. A method of inducing the expression of one or more bone morphogenic proteins or<br>
transforming growth factor-ß proteins in a cell, the method comprising: transfecting a cell with<br>
an isolated nucleic acid comprising a nucleotide sequence encoding a LIM mineralization protein<br>
(LMP) operably linked to a promoter, wherein the LMP is chosen from the group consisting of<br>
Human LMP-1 (HLMP-1), truncated versions of Human LMP (HLMP-1s), HLMP-3 or<br>
combinations thereof.<br>
2. The method as claimed in claim 1, wherein the LIM mineralization protein induces<br>
expression of one or more proteins selected from the group consisting of bone morphogenic<br>
protein-2 (BMP-2), bone morphogenic protein-4 (BMP-4), bone morphogenic protein-6 (BMP-<br>
6), bone morphogenic protein-7 (BMP-7), transforming growth factor-beta-1 (TGF-ß1), and<br>
combinations thereof.<br>
3. The method as claimed in claim 1, wherein the LMP is HLMP-1.<br>
4. The method as claimed in claim 1, wherein the cell is chosen from among the group<br>
consisting of a stem cell, an intervertebral disc cell, a cell of the nucleus pulposus, a cell of the<br>
annulus fibrosus, and a buffy coat cell.<br>
5. The method as claimed in claim 1, wherein transfecting a cell with an isolated nucleic<br>
acid comprises infection of the cell with a recombinant adenoviral vector comprising an isolated<br>
nucleic acid which encodes a protein chosen from the group consisting of HLMP-1, HLMP-ls,<br>
HLMP-3 or combination thereof.<br>
6. The method as claimed in claim 1, wherein the bone morphogenic protein is BMP-4.<br>
7 The method as claimed in claim 1, wherein the bone morphogenic protein is BMP-7.<br>
8. The method as claimed in claim 1, wherein the transfected cell comprises an intervertebral<br>
implant.<br>
The invention discloses a method of inducing the expression of one or more bone morphogenic<br>
proteins or transforming growth factor-ß proteins in a cell, the method comprising: transfecting a<br>
cell with an isolated nucleic acid comprising a nucleotide sequence encoding a LIM<br>
mineralization protein (LMP) operably linked to a promoter, wherein the LMP is chosen from<br>
the group consisting of Human LMP-1 (HLMP-1), truncated versions of Human LMP (HLMP-<br>
1s), HLMP-3 or combinations thereof.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk4OC1LT0xOUC0yMDA1LUZPUk0tMjcucGRm" target="_blank" style="word-wrap:break-word;">1988-KOLNP-2005-FORM-27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk4OC1rb2xucC0yMDA1LWdyYW50ZWQtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">1988-kolnp-2005-granted-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk4OC1rb2xucC0yMDA1LWdyYW50ZWQtYXNzaWdubWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">1988-kolnp-2005-granted-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk4OC1rb2xucC0yMDA1LWdyYW50ZWQtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">1988-kolnp-2005-granted-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk4OC1rb2xucC0yMDA1LWdyYW50ZWQtY29ycmVzcG9uZGVuY2UucGRm" target="_blank" style="word-wrap:break-word;">1988-kolnp-2005-granted-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk4OC1rb2xucC0yMDA1LWdyYW50ZWQtZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">1988-kolnp-2005-granted-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk4OC1rb2xucC0yMDA1LWdyYW50ZWQtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">1988-kolnp-2005-granted-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk4OC1rb2xucC0yMDA1LWdyYW50ZWQtZXhhbWluYXRpb24gcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">1988-kolnp-2005-granted-examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk4OC1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">1988-kolnp-2005-granted-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk4OC1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSAxOC5wZGY=" target="_blank" style="word-wrap:break-word;">1988-kolnp-2005-granted-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk4OC1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">1988-kolnp-2005-granted-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk4OC1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">1988-kolnp-2005-granted-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk4OC1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSA2LnBkZg==" target="_blank" style="word-wrap:break-word;">1988-kolnp-2005-granted-form 6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk4OC1rb2xucC0yMDA1LWdyYW50ZWQtZ3BhLnBkZg==" target="_blank" style="word-wrap:break-word;">1988-kolnp-2005-granted-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk4OC1rb2xucC0yMDA1LWdyYW50ZWQtcmVwbHkgdG8gZXhhbWluYXRpb24gcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">1988-kolnp-2005-granted-reply to examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk4OC1rb2xucC0yMDA1LWdyYW50ZWQtc2VxdWVuY2UgbGlzdGluZy5wZGY=" target="_blank" style="word-wrap:break-word;">1988-kolnp-2005-granted-sequence listing.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk4OC1rb2xucC0yMDA1LWdyYW50ZWQtc3BlY2lmaWNhdGlvbi5wZGY=" target="_blank" style="word-wrap:break-word;">1988-kolnp-2005-granted-specification.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="222897-a-6-amino-2-phenyl-substituted-pyrazine-compound.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="222899-a-pharmaceutical-composition-for-topical-administration.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>222898</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1988/KOLNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>35/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>29-Aug-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>27-Aug-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>06-Oct-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>WARSAW ORTHOPEDIC, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>2500 SILVEUS CROSSING, WARSAW. INDIANA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>MCKAY WILLIAM</td>
											<td>3870 MCELRIE COVE, MEMPHIS, TN 38133</td>
										</tr>
										<tr>
											<td>2</td>
											<td>BODEN SCOTT</td>
											<td>2842 CRAVEY DRIVE, ATLANTA, GA 30345</td>
										</tr>
										<tr>
											<td>3</td>
											<td>YOON SANGWOOK</td>
											<td>2431 VALHALLA DRIVE, ATLANTA, GA 30345</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12P 21/06</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2004/007616</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-03-07</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>10/382,844</td>
									<td>2003-03-07</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/222898-methods-of-inducing-the-expression-of-bone-morphogenetic-proteins-bmps-and-transforming-growth-factor-a-proteins-tgf-as-in-cells by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:23:24 GMT -->
</html>
