<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/222988-a-method-for-identity-testing-by-probe-elongation-mediated-analysis-of-variable-sites-in-the-hla-gene-in-a-genome by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:08:26 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 222988:&quot;A METHOD FOR IDENTITY TESTING BY PROBE ELONGATION-MEDIATED ANALYSIS OF VARIABLE SITES IN THE HLA GENE IN A GENOME&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A METHOD FOR IDENTITY TESTING BY PROBE ELONGATION-MEDIATED ANALYSIS OF VARIABLE SITES IN THE HLA GENE IN A GENOME&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention is related to a method for identity testing by probe elongation-mediated analysis of variable sites in the HLA gene in a genome by providing sets of different cognate probes, the cognate probes in a set capable of annealing to one or more amplicons, said amplicons generated by amplifying regions of the genome using asymmetric polymerase chain reaction, and said cognate probes further capable of being elongated with a detectably labeled nucleotide if, following annealing, the probe&#x27;s interrogation site is complementary to the aligned nucleotide in the amplicon, wherein said probes are designed such that the aligned nucleotide is complementary to a nucleotide at a variable site; and designating for each set of amplicons, one strand (either sense or antisense) for the probe elongation-mediated analysis of the variable sites, depending on which strand has a greater degree of complementarity to its cognate probe in the terminal elongation initiation region of the cognate probe.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>A METHOD FOR IDENTITY TESTING BY PROBE ELONGATION-MEDIATED ANALYSIS OF VARIABLE SITES IN THE HLA GENE IN A GENOME<br>
This application claims priority from U. S. Provisional Application Serial No. 60/329,427 filed October 14,2001, U. S. Provisional Application Serial No. 60/329,620, filed October 15,2001, U. S. Provisional Application Serial No. 60/329, 428, filed October 14,2001 and U. S. Provisional Application Serial No. 60/329,619 filed October 15,2001. All the above- referenced applications are expressly incorporated herein by reference.<br>
FIELD OF THE INVENTION<br>
The present invention generally relates to molecular diagnostics and genetic typing or profiling. The invention relates to methods, processes and probes for the multiplexed analysis of highly polymorphic genes. The invention also relates to the molecular typing and profiling of the Human Leukocyte Antigen (HLA) gene complex and the Cystic Fibrosis Conductance Trans-membrane Regulator gene (CFTR) and to compositions, methods and designs relating thereto.<br>
BACKGROUND OF THE INVENTION<br>
The ability to efficiently, rapidly and unambiguously analyze polymorphisms in the nucleic acid sequences of a gene of interest plays an important role in the development of molecular diagnostic assays, the applications of which includes genetic testing, carrier screening, genotyping or genetic profiling, and identity testing. For example, it is the objective of genetic testing and carrier screening to determine whether mutations associated with a particular disease<br><br><br>
ai c present in a gene of interest. The analysis of polymorphic loci, whether or not these co npn.se mutations known to cause disease, generally provides clinical benefit, as for example in i he context of pharmacogenomic genotyping or in the context of HLA molecular typing, in winch the degree of allele matching in the HLA loci of transplant donor and prospective re ipient is determined in context of allogeneic tissue and bone marrow transplantation.<br>
UK- multiplexed analysis of polymorphisms while desirable in facilitating the analysis of a high volume of patient samples, faces a considerable level of complexity which will likely increase as new polymorphisms, genetic markers and mutations are identified and must be included in lh analysis. The limitations of current methods to handle this complexity in a multiplexed fo mat of analysis so as to ensure reliable assay performance while accommodating high sample volume . and the consequent need for novel methods of multiplexed analysis of polymorphisms an. I mutations is the subject of the present invention. By way of example, the genetic loci ei odmg Cystic Fibrosis Transmembrane Conductance (CFTR) channel and Human Leukocyte Aiiiigens (HLA) are analyzed by the methods of the invention.. Cystic fibrosis (CF) is one of the m &gt;s; common recessive disorders hi Caucasians with a rate of occurrence in the US of 1 in 2000 li\ c births. About 4% of the population carry one of the CF mutations. The CFTR gene is highly variable: more than 900 mutations have been identified to date (see hi p www.genet.sickkids.on.ca/cftr. which is incorporated herein by reference). The cluiracterization of the CFTR gene provides the key to the molecular diagnosis of CF by fivilitating the development of sequence-specific probes (Rommens et al, 1989; Riordan, et al., 1VS9: Kerem et al., 1989, each of which is incorporated herein by reference). The National Institutes of Health (NEH) - sponsored consensus development conference recommended carrier sc cenmg for CFTR mutations for adults with a positive family history of CF (NIH 1997). The a mmitiee on carrier screening of the American College of Medical Genetics (ACMG) has IT ommended for use in general population carrier screening a pan-ethnic mutation panel that includes a set of 25 disease-causing CF mutations with an allele frequency of &gt;0.1% in the gi-neral population of United States ( see http://www.faseb.org/genetics/acmg, which is ii irporuted herein by reference). The mutations in the ACMG panel also include the most d mmon mutations in Ashkenazi Jewish and African-American populations.<br><br>
Several methods have been described for the detection of CFTR mutations including the following: : denaturing gradient gel electrophoresis (Devoto et al., 1991); single strand co librmation polymorphism analysis (Plieth et al., 1992); RPLP (Friedman et al., 1991); an plificaiion with allele-specific primers (ASPs) (Gremonesi et al., 1992), and probing with a] 1 ,;le specific oligonucleotides (ASO) (Saiki et al., 1986). A widely used method involves PCR an i pi ification followed by blotting of amplified target strands onto a membrane and probing of sli nds with oligonucleotides designed to match either the normal ("wild type") or mutant configuration. Specifically, multiplex PCR has been used in conjunction with ASO hybridization in this dot blot format to screen 12 CF mutations (Shuber et al, 1993). In several in :ances. arraysof substrate-immobilized oligonucleotide probes were used to facilitate the defection of known genomic DNA sequence variations (Saiki, RK et al, 1989) in a "reverse do blor format An array of short oligonucleotides synthesized in-situ by pi &gt;iolithographicprocesses was used to detect known mutations in the coding region of the Ci TR gene (Cronin, MT., et al, 1996). Primer extension using reverse transcriptase has been reported as a method for detecting the A508 mutation in CFTR (Pastinen, T., 2000). This approach was described as early as 1989 (Wu, D. Y. et al, Proc. Natl. Acad. Sci. USA. 86:2757-T /i (1989), Newton, C. R. et al, Nucleic Acids Res. 17:2503-2506 (1989)). As further discussed hi ein below, while providing reasonable detection in a research laboratory setting, these m thods require significant labor, provide only slow turnaround, offer only low sample throughput, and hence require a high cost per sample.<br>
Li connection with the spotted microarrays, several methods of spotting have been described, al&gt; &gt;ng with many substrate materials and methods of probe immobilization. However, the spotted ai ays of current methods exhibit not only significant array-to-array variability but also significant spot-to-spot variability, an aspect that leads to limitations in assay reliability and sensitivity. In addition, spotted arrays are difficult to miniaturize beyond their current spot di nensions of typically 100 um diameter on 500 um centers, thereby increasing total sample v
 <br>
dc'cribed The complexity of array fabrication, however, limits routine customization and coi nhines considerable expense with lack of flexibility for diagnostic applications.<br>
Tlv major histocompatibility complex (MHC) includes the human leukocyte antigen (HLA) gene complex, located on the short arm of human chromosome six. This region encodes cell-surface pr&gt; &gt;ieins which regulate the cell-cell interactions underlying immune response. The various HLA C1..SS I loci encode 44,000 dalton polypeptides which associate with p-2 microglobulin at the cell sin lace and mediate the recognition of target cells by cytotoxic T lymphocytes. HLA Class n loci en&gt; ode cell surface heterodimers, composed of a 29,000 dalton and a 34,000 dalton polypeptide wl ich mediate the recognition of target cells by helper T lymphocytes. HLA antigens, by presenting foreign pathogenic peptides to T-cells in the context of a "self protein, mediate the initiation of an immune response. Consequently, a large repertoire of peptides is desirable bo ause it increases the immune response potential of the host. On the other hand, the co respondingly high degree of immunogenetic polymorphism represents significant difficulties in tilouansplantation, with a mismatch in HLA loci representing one of the main causes of allograft rejection. The degree of allele matching in the HLA loci of a donor and prospective recipient is a major factor in the success of allogeneic tissue and bone marrow transplantation.<br>
Tl o HI,A-A, HLA-B, and HLA-C loci of the HLA Class I region as well as the HLA-DRB, H i A-DQB, HLA-DQA, HLA-DPB and HLA-DPA loci of the HLA Class H region exhibit an ex i emely high degree of polymorphism. To date, the WHO nomenclature committee for factors ol the HLA system has designated 225 alleles of HLA A (HLA A*0101, A*0201, etc.), 444 al I eles of HLA-B, and 111 alleles of HLA-C, 358 HLA-DRB alleles, 22 HLA-DQA alleles, 47 H i A-DQB alleles, 20 HLA-DPA alleles and 96 HLA-DPB alleles (See IMGT/HLA Sequence D.uabase. http://www3.ebi.ac.uk:80/imgt/hla/index.html) and Schreuder,G.M.Th. et al. Tissue Ai itigens. 54:409-437 (1999)), both of which are hereby incorporated by reference.<br>
HLA typing is a routine procedure that is used to determine the immunogenetic pi ifilc of transplant donors. The objective of HLA typing is the determination of the patient's al: Ic configuration at the requisite level of resolution, based on the analysis of a set of designated polymorphisms within the genetic locus of interest. Increasingly, molecular typing of HLA is the<br><br>
nieU'od of choice over traditional serological typing, because it eliminates the requirement for viable cells, offers higher allelic resolution, and extends HLA typing to Class n for which serology has not been adequate(Erlich, H. A. et al, Immunity. 14:347-356 (2001)).<br>
One method currently applied to clinical HLA typing uses the polymerase chain reaction (PCR) in conjunction with sequence-specific oligonucleotide probes (SSO or SSOP), which are allowed to hybridize to amplified target sequences to produce a pattern as a basis for HLA typing.<br>
The availability of sequence information for all available HLA alleles has permitted the design of sequence-specific oligonucleotides (SSO) and allele-specific ollgonucleotides (ASO) for the characterization of known HLA polymorphisms as well as for sequencing by hybridization (Saiki, R.K. Nature 324:163-166 (1986), Cao, K. et al, Rev Inimunogenetics, 1999:1:177-208).<br>
Li one embodiment of SSO analysis, also referred to as a "dot blot format", DNA samples are ex iracted from patients, amplified and blotted onto a set of nylon membranes in an 8x12 grid fomiat. One radio-labeled oligonucleotide probe is added to each spot on each such membrane; following hybridization, spots are inspected by autoradiography and scored either positive (1) 01 negative (0). For each patient sample, the string of 1's and O's constructed from the analysis ol all membranes defines the allele configuration. A multiplexed format of SSO analysis in the "reverse dot blot format" employs sets of oligonucleotide probes immobilized on planar supports (Saiki, R. et al, Lmnunologjcal Rev. 167:193-199 (1989), Erlich, H. A. Eur. J. Immunogenet. IS: 33-55 (1991)).<br>
Another method of HLA typing uses the polymerase-catalyzed elongation of sequence-specific primers (SSPs) to discriminate between alleles. The high specificity of DNA polymerase generally endows flu's method with superior specificity. In the SSP method, PCR amplification is performed with a specific primer pair for each polymorphic sequence motif or pair of motifs and a DNA polymerase lacking 3' -&gt; 5' exonuclease activity so that elongation (and hence amplification) occurs only for that primer whose 3' terminus is perfectly complementary<br><br>
("matched") to the template. The presence of the corresponding PCR product is ascertained by gel electrophoretic analysis. An example of a highly polymorphic locus is the 280 nt DNA fr.igment of the HLA class n DR gene which features a high incidence of polymorphisms<br>
HIA typing based on the use of sequence-specific probes (SSP), also referred to as phototyping (] &gt;upont, B. Tissue Antigen. 46: 353-354 (1995)), has been developed as a commercial ttv-hnology that is in routine use for class I and class H typing (Bunce, M. et al, Tissue Antigens. 4i.:355-367 (1995), Krausa, P and Browning, M.J., Tissue Antigens. 47:237-244 (1996), Bunce, M et al, Tissue Antigens. 45:81-90 (1995)). However, the requirement of the SSP methods of the prior art for extensive gel electrophoretic analysis for individual detection of amplicons represents a significant impediment to the implementation of multiplexed assay formats that can ai hieve high throughput. This disadvantage is overcome by the methods of the present invention.<br>
In the context of elongation reactions, highly polymorphic loci and the effect of non-designated polymorphic sites as interfering polymorphisms were not considered in previous applications, especially in multiplexed format. Thus, there is a need to provide for methods, a mpositions and processes for the multiplexed analysis of polymorphic loci that would enable tlie detection of designated while accommodating the presence of no-designated sites and without interference from such non-designated sites. SUMMARY OF THE INVENTION<br>
The present invention provides methods and processes for the concurrent iii eiTogation of multiple designated polymorphic sites in the presence of non-designated p lymorphic sites and without interference from such non-designated sites.. Sets of probes are provided which facilitate such concurrent interrogation. The present invention also provides methods, processes, and probes for the identification of polymorphisms of the HLA gene cc-mplex and the CFTR gene.<br>
The specificity of methods of detection using probe extension or elongation is ini i insically superior to that of methods using hybridization, particularly in a multiplexed format, because the discrimination of sequence configurations no longer depends on differential hybridization but on the fidelity of enzymatic recognition. To date, the overwhelming majority<br><br>
oJ applications of enzyme-mediated analysis use single base probe extension. However, probe el« Migation, in analogy to that used in the SSP method of HLA typing, offers several advantages foi the multiplexed analysis of polymorphisms, as disclosed herein. Thus, single nucleotide as wi 11 as multi-nucleotide polymorphisms are readily accommodated. The method, as described he ein, is generally practiced with only single label detection, accommodates concurrent as well as i msecutive interrogation of polymorphic loci and incorporates complexity in the probe design.<br>
Ore aspect of this invention provides a method of concurrent determination of nucleotide composition at designated polymorphic sites located within one or more target nucleotide sc , uences. This method comprises the following steps: (a) providing one or more sets of probes, en h probe capable of annealing to a subsequence of the one or more target nucleotide sequences located within a range of proximity to a designated polymorphic site; (b) contacting the set of probes with the one or more target nucleotide sequences so as to permit formation of hybridization complexes by placing an interrogation site within a probe sequence in direct all -;mnent with the designated polymorphic site; (c) for each hybridization complex, determining tli presence of a match or a mismatch between the interrogation site and a designated polymorphic site; and (d) determining the composition of the designated polymorphic site.<br>
Another aspect of this invention is to provide a method of sequence-specific amplification of as ,ay signals produced in the analysis of a nucleic acid sequence of interest in a biological simple. This method comprises the following steps: (a) providing a set of immobilized probes capable of forming a hybridization complex with the sequence of interest; (b) contacting said set ol immobilized probes with the biological sample containing the sequence of interest under conditions which permit the sequence of interest to anneal to at least one of the immobilized pi &gt;bes to form a hybridization complex; (c) contacting the hybridization complex with a prlvnierase to allow elongation or extension of the probes contained within the hybridization complex; (d) converting elongation or extension of the probes into an optical signal; and (e) re 
 <br>
el. -ngation probe capable of alignment of the interrogation site of the probe with a designated pi lynioiphic site; (b) further determining a complete set of degenerate probes to accommodate al i uoii designated as well as non-selected designated polymorphic sites while maintaining al • -nment of the interrogation site of the probe with the designated polymorphic site; and (c) a Hieing the degree of degeneracy by removing all tolerated polymorphisms.<br>
Oi e aspect of this invention is to provide a method for identifying polymorphisms at one or m
A' (Other aspect of this invention is to provide a method for determining a polymorphism at one 01 more designated sites in a target polynucleotide sequence. This method comprises providing a uobe set for the designated sites and grouping the probe set in different probe subsets ac ording to the terminal elongation initiation of each probe.<br>
Another aspect of this invention is to provide a method for the concurrent interrogation of a multiplicity of polymorphic sites comprising the step of conducting a multiplexed elongation as ay by applying one or more temperature cycles to achieve linear amplification of such target.<br>
Y t another aspect of this invention is to provide a method for the concurrent interrogation of a multiplicity of polymorphic sites. This method comprises the step of conducting a multiplexed el ngation assay by applying a combination of annealing and elongation steps under temperature-ceil rolled conditions.<br>
A; lother aspect of this invention is to provide a method of concurrent interrogation of nucleotide composition at S polymorphic sites, PS := {c P(s); 1 <s s located within one or more ci niguous target sequences said method assigning to each cp of a limited set possible v ues by performing the following steps: providing designated immobilized oli probes also known as elongation probe capable annealing in></s>
 <br>
preferred alignment to a subsequence of the target located proximal to a designated polymorphic site, the preferred alignment placing an interrogation site within the probe sequence in direct juxtaposition to the designated polymorphic site, the probes further containing a terminal elongation initiation (TEI) region capable of initiating an elongation or extension reaction; (b) permitting the one or more target sequences to anneal to the set of immobilized oligonucleotide probes so as form probe-target hyrbdization complexes; and (c) for each probe-target hybridization complex, calling a match or a mismatch in composition between interrogation site and corresponding designated polymorphic site.<br>
Other objects, features and advantages of the invention will be more clearly understood when taken together with the following detailed description of an embodiment which will be understood as being illustrative only.<br>
STATEMENT OF THE INVENTION<br>
In accordance with the present invention it is associated to a method for identity testing by probe elongation-mediated analysis of variable sitesin the HLA gene in a genome by providing sets of different cognate probes characterized in that the cognate are probes in a set capable of annealing to one or more amplicons, said amplicons generated by amplifying regions of the genome using asymmetric polymerase chain reaction, and said cognate probes further capable of being elongated with a detectably labeled nucleotide if, following annealing, the probe's interrogation site is complementary to the aligned nucleotide in the amplicon, wherein said probes are designed such that the aligned nucleotide is complementary to a nucleotide at a variable site; and designating for each set of amplicons, one strand (either sense or antisense) for the probe elongation-mediated analysis of the variable sites, depending on which strand has a greater degree of complementarity to its cognate probe in the terminal elongation initiation region of the cognate probe.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Fig. 1a is an illustration of probe sets designed to interrogate designated sites in HLA-DR and an internal control.<br>
Fig. 1b is an illustration of a staggered primer design.<br>
Fig. 2 is an illustration of a modification of allele binding pattern based on tolerance effect.<br>
Fig. 3 is an illustration of the use of linked primer structure to separate the anchoring sequence and polymorphism detection sequence.<br>
Fig. 4 shows simulated ambiguity in allele identification due to allele combination.<br>
Fig. 5 shows one method for decreasing the ambiguity in allele identification that arises from allele combination.<br>
Fig. 6 is an illustration of a combination of hybridization and elongation<br><br><br>
Fig. 7  shows a model reaction using synthetic oligonucleotides as targets.<br>
Fig. 8   shows results obtained using testing real patient sample in an eMAP format.<br>
Fig. 9  shows results obtained from eMAP primer extension for DR locus.<br>
Fig. 10 shows results obtained from eMAP for DR locus.<br>
Fig. 11 shows results obtained from eMAP for A locus Exon 3.<br>
Fig. 12 shows results obtained from eMAP SSP for A locus Exon 3 and is an example of tolerance for the non-designated polymorphism.<br>
Fig. 13 is an illustration of bead immobilized probe elongation of variable mutant sites. Fig. 14 is an illustration of PCR using primers immobilized on the surface of beads. Fig. 15 is an illustration of elongation of multiple probes using combined PCR products.<br>
Fig. 16 is an illustration of results for probe elongation of a multiplexed CF mutation. Fig. 16a is an illustration of probe elongation using a synthetic target. Fig. 16b is an illustration of probe elongation using beads in a PCR reaction.<br>
Fig. 17 is an illustration of one-step elongation with temperature-controlled cycling results.<br>
Fig. 18 is an illustration of primer elongation with labeled dNTP and three other unlabeled dNTPs.<br><br>
Fig. 19 is an illustration of primer elongation with labeled ddNTP and three other unlabeled dNTPs.<br>
Fig. 20 is an illustration of primer elongation, where four unlabeled dNTPs are used for elongation and the product is detected by a labeled oligonucleotide probe which hybridizes to the extended unlabeled product.<br>
Fig.21 is an illustration of a primer extension in which a labeled target and four unlabeled dNTPs are added. This illustration which shows that only with the extended product can the labeled target be retained with the beads when high temperature is applied to the chip.<br>
Fig. 22 is an illustration of linear amplification where sequence specific probes are immobilized.<br>
Fig. 23 _ is an illustration of the utilization of hairpin probes.<br>
Fig. 24 is an illustration of applying this invention to the analysis of cystic fibrosis and Ashkenazi Jewish disease mutations.<br>
DI .TAILED DESCRIPTION OF THE INVENTION<br>
This invention provides compositions, methods and designs for the multiplexed analysis of highly polymorphic loci; that is, loci featuring a high density of specific ('designated") polymorphic sites, as well as interfering non-designated polymorphic sites. The multiplexed analysis of such sites thus generally involves significant overlap in the sequences<br>
0	i probes directed to adjacent sites on the same target, such that probes designed for any specific<br>
01	designated site generally also will cover neighboring polymorphic sites. The interference in<br>
tl i  analysis of important genes including CFTR and HLA has not been addressed in the prior art.<br>
'I o exemplify the methods of the methods of the invention, the HLA gene complex and the<br><br>
CJ TR gene are analyzed.<br>
II present invention provides compositions and methods for the parallel or multiplexed analysis ot polymorphisms ("MAP") in nucleic acid sequences displaying a high density of polymorphic si; v In a given nucleic acid sequence, each polymorphic site comprises a difference comprising 01 or more nucleotides.<br>
11 s invention provides methods and compositions for the concurrent interrogation of an entire SL of designated polymorphisms within a nucleic acid sequence. This invention provides c
TV methods and compositions of this invention are useful for improving the reliability and ac macy of polymorphism analysis of target regions which contain polymorphic sites in addition to :hc polymorphic sites designated for interrogation. These non-designated sites represent a sc i rcc o( interference in the analysis. Depending on the specific assay applications, one or more pi lies ofdiffering composition may be designated for the same polymorphic site, as elaborated ii ,(-venil Examples provided herein. It is a specific objective of the present invention to provide compositions and methods for efficient, rapid and unambiguous analysis of polymorphisms in<br><br>
gi lies of interest.  This analysis is useful in  molecular diagnostic assays,   such as those dt signed, for example, for genetic testing, carrier screening, genotyping or genetic profiling, identity testing, paternity testing and forensics.<br>
Preparation of target sequences may be carried out using methods known in the art. In a non-lii i liting example, a sample of cells or tissue is obtained from a patient. The nucleic acid regions containing target sequences (e.g., Exons 2 and 3 of HLA) are then amplified using standard tc hniques such as PCR (e.g., asymmetric PCR).<br>
Pnihes for detecting polymorphic sites function as the point of initiation of a polymerase-cuialyzed elongation reaction when the composition of a polymorphic site being analyzed is ci 'inplementary ("matched") to that of the aligned site in the probe. Generally, the probes of the ii i \-ention should be sufficiently long to avoid annealing to unrelated DNA target sequences. In crriain embodiments, the length of the probe may be about 10 to 50 bases, more preferably about 1: to 25, and more preferably 18 to 20 bases. Probes may be immobilized on the solid supports vi i linker moieties using methods and compositions well known in the art.<br>
As used herein, the term "nucleic acid" or "oligonucleotide" refers to deoxyribonucleic acid or ribonucleic acid in a single or double-stranded form. The term also covers nucleic-acid like structures with synthetic backbones. DNA backbone analogues include phosphodiester, pliosphorothioate, phosphorodithioate, methylphosphonate, phosphoramidate, alkyl pliosphotriester, sulfamate, 3'-thioacetal, methylene(methylimino), 3'-N-carbamate, morpholino c;i i hamate, and peptide nucleic acids (PNAs). See Oligonucleotides and Analogues, A Practical Approach (Editor: F. Eckstein), IRL Press at Oxford University Press (1991); Antisense St i ategies, Annals of the New York Academy of Sciences, vol. 600, Eds.; Baserga and Denhardt (&gt; VAS 1992); Milligan, J. Med. Chem., vol. 36, pp. 1923-1937; Antisense Research and A ^plications (1993, CRC Press). PNAs contain non-ionic backbones, such as N-2(2-aminoethyl) glvcine units. Phosphorothioate linkages are described in WO 97/03211; WO 96/39159; and Mala, Toxicol. Appl. Pharmacol. 144:189-197 (1997). Other synthetic backbones encompassed b* (lie term include methyl-phosphonate linkages or alternating methylphosphonate and pl..isphodiester linkages (Strauss-Soukup, Biochemistry, 36: 8692-8698 (1997), and<br><br>
b
A' used herein, the term "hybridization" refers to the binding, duplexing, or hybridizing of a mii-leic acid molecule preferentially to a particular nucleotide sequence under stringent co uiiiions. The term "stringent conditions" refers to conditions under which a probe will h\ iridize preferentially to the corresponding target sequence, and to a lesser extent or not at all to &gt;ther sequences. A "stringent hybridization" is sequence dependent, and is different under di: icrent conditions. An extensive guide to the hybridization of nucleic acids may be found in, e.;1. Tijssen, Laboratory Techniques in Biochemistry and Molecular Biology, Elsevier, NY (1' &gt;()3). Generally, highly stringent hybridization and wash conditions are selected to about 5°C Icv.ver than the thermal melting point (Tm) for the specific sequence at a defined ionic strength an. I pH. The Tm is the temperature (under defined ionic strength and pH) at which 50% of the tai 'jet sequence hybridizes to a perfectly matched probe. Very stringent conditions are selected b\ conducting the assay at a temperature set to be equal to the Tm for a particular probe. An ex unplc of highly stringent wash condition is 0.15 M NaCl at 72°C for about 15 minutes. An e.x unple of stringent wash conditions is a 0.2xSSC wash at 65°C for 15 minutes. See Sambrook, Molecular Cloning: A Laboratory Manual (2nd Ed), vol. 1-3 (1989).<br>
A used herein, the term "designated site" is defined as a polymorphic site of interest (i.e., a po ym orphic site that one intends to identify) on a given nucleic acid. The term "non-designated sii " refers to any polymorphic site that co-exists with a designated site or sites on a given nucleic aoil bul is not of interest.<br>
As used herein, the term "correlated designated sites" refers to polymorphic sites with correlated Of urrences. Typically, each member of such a set of polymorphic sites must be identified in 01 itr to identify the allele to which the set belongs.<br>
As used herein, the term "selected designated site" refers to a polymorphic site of interest on a gi\ t-ii nucleic acid that also overlaps with the 3'end of a probe sequence of this invention. A "i MI-selected designated site" refers to a polymorphic site of interest that does not overlap with<br><br>
a "  end of a probe sequence of this invention.<br>
As used herein, an "interfering non-designated site" refers to a non-designated polymorphic site th; i is within 1-5 bases from the 3' end of a probe sequence of this invention. A "non-interfering no11-designated site" refers to anon-designated site that is greater than 5 bases from the 3' end of ;t probe sequence of this invention. The non-interfering non-designated site may be closer to tlu 5' end of the probe sequence than to the 3' end.<br>
hi eitain embodiments, the probes of this invention comprise a "terminal elongation initiation" re;-ion (also referred to as a "TEI" region) and a Duplex Anchoring ("DA") region. The TEI region refers a section of the probe sequence, typically the three or four 3' terminal positions of thi" probe. The TEI region is designed to align with a portion of the target nucleic acid sequence at i designated polymorphic site so as to initiate the polymerase-catalyzed elongation of the prlie. The DA region, typically comprises the remaining positions within the probe sequence an I is preferably designed to align with a portion of the target sequence in a region located close (\\ i thin 3-5 bases) to the designated polymorphism.<br>
A; used herein, the term a "close range of proximity" refers to a distance of between 1-5 bases al»ng a given nucleic acid strand. A 'Yange of proximity" refers to a distance within 1-10 bases al' &gt;ng a given nucleic acid strand. The term "range of tolerance" refers to the total number of mismatches in the TEI region of a probe hybridized to a target sequence that still permits annealing and elongation of the probe. Typically, more than 2 mismatches in the TEI region of a 1 \bridized probe is beyond the range of tolerance.<br>
Tlv tenns ."microspheres", "microparticles", "beads", and "particles" are herein used in 101 changeably. The composition of the beads includes, but is not limited to, plastics, ceramics, glass, polystyrene, methylstyrene, acrylic polymers, paramagnetic materials, thoria sol, carbon gi iphite, titanium dioxide, latex or cross-linked dextrans such as sepharose, cellulose, nylon, ci  'iii nanometers (e.g., 100 nm) to millimeters (e.g., 1 mm), with beads from about 0.2 micron<br><br>
to ;ihout 200 microns being preferred, more preferably from about 0.5 to about 5 micron being particularly preferred.<br>
Tins invention provides for the concurrent interrogation of a set of designated polymorphic sites within one or more target strands by first annealing a set of immobilized sequence specific ol ;onucleotide probes to target nucleic acid strands and by probing the configuration of d in idization complex ("duplex"). The probe's 3' terminus is placed at or near the designated sii within the target and polymerase-catalyzed probe elongation is initiated if the 3' terminal pi the composition matches (i.e., is complementary to) that of the target at the interrogation site. A described herein, the probe may be designed to anneal in a manner such that the designated site is within a range of proximity of the 3' terminus.<br>
In one embodiment of the invention, two or more probes may be provided for interrogation of a specific designated site. The probes are designed to take into account the possibility of p'-lymorphisms or mutations at the interrogation site and non-designated polymorphic sites \vi thin a certain range of proximity of the designated polymorphic site. In this context, the term "l Milymorphism" refers to any variation in a nucleic acid sequence, while the term "mutation" iv: ers to a sequence variation in a gene that is associated or believed to be associated with a pi icnotype. In a preferred embodiment, Ihis multiplicity of probe sequences contains at least one pi &gt;he that matches the specific target sequence in all positions within the range of proximity to ensure elongation.<br>
Li   eriain embodiments, the invention discloses compositions and methods for the parallel ini rroiviiion of S polymorphic sites selected from a target sequence of length N by a set of L ^<br><br>
S ('I igonucleotide primers.<br>
In accordance with the requirements of specific assay applications, one or more probes of dill'cring composition maybe designated for the same polymorphic site, as elaborated in several Ex.-nnples provided herein.<br>
Eae of given interrogation site composition assigns to that site one of two values, namely niii died, numerically represented by 1, or non-matched, numerically represented by 0. In HLA molecular typing, the resulting binary string of length L identifies an allele to a desired typing resolution.<br>
In . preferred embodiment, the interrogation step uses the extension of the designated probe. This reaetion, catalyzed by a polymerase, produces an extended hybridization complex by adding to the probe sequence one or more nucleoside triphosphates in the order reflecting the sequence of the target sequence in the existing hybridization complex. In order for this extension reaction to proceed, a designated primer of length M must contain a terminal extension initiation region of len,;th M*<m herein also referred to as terminal extension initiation sequence tei seij.ience which contains the interrogation site. proceeds if composition of des ignated site matches that designated polymorphic></m>
Methods of the prior art of detecting successful extension have been described which involve the use labeled deoxy nucleoside triphosphates (dNTPs) or dideoxy nucleoside triphosphates (ddN'TPs). The present invention also discloses novel methods of providing optical signatures for del ction of successful extension eliminating the need for labeled dNTPs orddNTPs, an advantage arit nig from the reduction in the efficiency of available polymerases in accommodating labeled dN : I's orddNTPs.<br><br>
However, the density of polymorphic sites in highly polymorphic loci considered in connection wi i i ihe present invention makes it likely that designated primers directed to selected polymorphic sili •&gt;. when annealing to the target subsequence proximal to the designated polymorphic site, will ovvrlap adjacent polymorphic sites.<br>
That is, an oligonucleotide probe, designed to interrogate the configuration of the target at one of UK selected polymorphic sites, and constructed with sufficient length to ensure specificity and tlii mal stability in annealing to the correct target subsequence, will align with other nearby po! morphic sites. These interfering polymorphic sites may include the non-designated sites as well as non-selected designated sites in the target sequence.<br>
In i multiplexed SSP reaction carried out in solution, the partial overlap between designated pn hes directed to nearby selected polymorphisms may lead to mutual competition between probes for the same target. The present invention significantly reduces this complication by way of probe immobilization.<br>
As with multiplexed differential hybridization generally, the mismatch in one or more positions between a designated probe and target may affect the thermal stability of the hybridization coi.iplex. That is, any set of annealing conditions applied to the entire reaction mixture may on \luce varying degrees of annealing between probe and target and may affect the outcome of the subsequent probe extension reaction, thereby introducing ambiguities in the assay which may require subsequent resequencing.<br>
Noii-designated polymorphic sites located in immediate proximity to the interrogation site near or;it the 3' terminus of the designated probe are particularly deleterious to the effectiveness of the pn lie's 1 El sequence in initiating the extension reaction.<br>
Th
In ;i preferred embodiment yielding optimal discriminating power, the interrogation site is pn v'ided at the probe's 3' terminus. Given a probe sequence of length M designated for a selected<br><br>
site s* in the representation PM(S«) := (cp(m); 1 £ m ^ M}, the index m increasing in the primer's 5' t
implicated.<br>
Under these circumstances as they are anticipated in the multiplexed analysis of highly polymorphic loci, the advantage of enhanced specificity afforded by the application of a pohinerase-catalyzed extension reaction is greatly diminished or lost as a result of complications aris ng from "sub-optimal" annealing conditions closely related to those limiting the performance<br>
of .^ &gt;&gt;O analysis.<br>
In i mnection with the optimization of the design of multiple probe sequences sharing the same interrogation site composition for any given designated polymorphic site, it will be useful to consider the concept of tolerance of interfering polymorphisms. Considering without limitation of generality the example of the single nucleotide polymorphism, a shift in alignment of s* away froi i the 3' terminus to positions M-l, M-2, ..., M-m* leads to a gradually diminished dist riminatory power. That is, when the designated polymorphic site is aligned with an interior probe position, m*, the extension reaction no longer discriminates between match and mismatch. Conversely, in the preferred embodiment of placing the interrogation site at the probe's 3' terminus, the deleterious effect of nearby non-designated polymorphisms on the effectiveness of the xiension reaction likewise decreases with distance from the 3' terminus. That is, non-desmiaied polymorphisms aligned with position between 1 and m* will not affect the extension re;i
Thi terminal sequence of length M-m*+l within the probe is herein referred to as the TEI sequence of a given primer. In general, 1 
Tht present invention accommodates the presence of interfering polymorphic sites within the len; • th of a designated probe sequence by taking into account these known sequence variations in the«lesign of multiple probes. In particular, the number of alternate probe sequence configurations to I provided for given probe length M is significantly reduced as a result of the existence of a 11: i sequence of length M-m*+1. That is, in Older to ensure effective discriminatory power of the extrusion reaction, it is sufficient to restrict the anticipatory alternate probe sequence<br><br>
configurations to the length of the TEI sequence. In a preferred embodiment, all possible aid mative sequences are anticipated so that one of these alternate probe sequences will match the tai -el in all of the positions m*, m*+l,... M-l, M.<br>
Prling or probe pooling. Complete probe sequence pooling reduces the coding complexity to that of ilie original design in which no anticipatory probe sequences were provided. Partial pooling als. i is possible.<br>
li certain preferred embodiments, the polymerase used in probe elongation is a DNA polymerase tli a lacks 3' to 5' exonuclease activity. Examples of such polymerases include T7 DNA polymerase, T4 DNA polymerase, ThermoSequenase and Taq polymerase. When the target nucleic acid sequence is KNA, reverse transcriptase may be used, hi addition to polymerase, ni K-,leoside triphosphates are added, preferably all four bases. For example dNTPs, or analogues, may be added, hi certain other embodiments, ddNTPs may be added. Labeled nucleotide ai alogues, such as Cye3-dUTP may also be used to facilitate detection.<br>
Pnor art methods for detecting successful elongation have been described which use labeled d
T; .is invention provides methods and compositions for accurate polymorphism analysis of highly polymorphic target regions. As used herein, highly polymorphic sequences are those containing, \\a thin a portion of the sequence contacted by the probe, not only the designated or interrogated polymorphic site, but also non-designated polymorphic sites which represent a potential source o; error in the analysis. Analogous considerations pertain to designs, compositions and methods<br><br>
of multiplexing PCR reactions. In a preferred embodiment, covering sets of PCR probes composed of priming and annealing subsequences are displayed on encoded microparticles to pi i iduce bead-displayed amplicons by probe elongation. Assemblies of beads may be formed on plimar substrates, prior to or subsequent to amplification to facilitate decoding and imaging of<br>
pi' 'bos.<br>
in me embodiment, this invention provides probes that are designed to contain a 3' terminal "priming" subsequence, also referred to herein as a Terminal Elongation Initiation (TEI) region, an : an annealing subsequence, also referred to herein as a Duplex Anchoring (DA) region. The T) I region typically comprises the three or four 3' terminal positions of a probe sequence. The Tl , region is designed to align with a portion of the target sequence at a designated polymorphic suv so as to initiate the polymerase-catalyzed elongation of the probe. Probe elongation indicates a i .,-rfect match in composition of the entire TEI region and the corresponding portion of the tai :i't sequence. The DA region, comprising remaining positions within the probe sequence, is pi\ ferab ly designed to align with a portion of the target sequence in a region located close (within 3- bases) to the designated polymorphism. The duplex anchoring region is designed to ensure specific and strong annealing, and is not designed for polymorphism analysis. As described he- dn, the DA and TEI regions may be located immediately adjacent to one another within the pi lie 01 may be linked by a molecular tether. The latter approach permits flexibility in the pi. L-einent of DA region so as to avoid non-designated polymorphisms located immediately ad acent to the designated site. The composition and length of the DA region are chosen to fat ilitate the formation of a stable sequence-specific hybridization complex ("duplex"), while ad nmmodating (i.e., taking into account) the presence of one or more non-designated poi vraoqAisms located in that region of the target. The length of the annealing subsequence is ch "?en tc&gt; minimize cross-hybridization by minimizing sequence homologies between probe and non-selected subsequences of the target. The length of the annealing subsequence generally ex
Tl elongation reaction provides high specificity in detecting polymorphisms located within the Tl i region. For non-designated polymorphisms in the DA region, the elongation reaction will pii oeed at a level either comparable to, or lower than that of the perfect match under certain<br><br>
conditions. This is referred to as the tolerance effect of the elongation reaction. Tolerance is iinli/ed in the design of probes to analyze designated and non-designated polymorphisms as described in examples herein.<br>
The density of polymorphic sites in the highly polymorphic loci considered in certain ei; ihodmients of this invention makes it likely that probes directed to designated polymorphic A ill overlap adjacent polymorphic sites, when annealing to a target subsequence proximal to i he designated polymorphic site. That is, an oligonucleotide probe designed to interrogate the configuration of the target at a selected designated polymorphic site, and constructed with sufficient length to ensure specificity and thermal stability in annealing to the correct target subsequence will align with nearby polymorphic sites. These interfering polymorphic sites may iii'.Rule non-designated sites in the target sequence as well as designated but not selected polymorphic sites<br>
Specifically, non-designated polymorphisms as contemplated in the present invention may in 01 fere with duplex formation, thereby interfering with or completely inhibiting probe eli mgation. In one embodiment, the present invention provides designs of covering probe sets to ai ornmodate such non-designated polymorphisms. A covering probe set contains probes for concurrently interrogating a given multiplicity of designated polymorphic sites within a nucleic acid sequence. A covering probe set comprises, for each site, at least one probe capable of an icaling to the target so as to permit, on the basis of a subsequent elongation reaction, as ignment of one of two possible values to that site: "matched" (elongation) or "unmatched", (no elongation).<br>
TK- covering probe set associated with each designated site may contain two or more probes di i ferine in one or more positions, also referred to herein as a degenerate set. hi certain en .nodiments, the probe sequence may contain universal nucleotides capable of forming a base-pan with any of the nucleotides encountered in DNA. In certain embodiments, probes maybe all iched to encoded microparticles, and specifically, two or more of the probes in a covering set 01 u generate sel may be attached to the same type of microparticle. The process of attaching t\\ • • or inure probes to a microparticle or bead is referred to as "probe pooling".<br><br>
The design of covering probe sets is described herein in connection with elongation-mediated multiplexed analysis of polymorphisms in two representative areas of genetic analysis: (1): the si .-ring of multiple uncorrelated designated polymorphisms and mutations, as in the case of m i itation analysis for CF and Ashkenazi Jewish (AJ) disease carrier screening, and (2) the scoring ol ;i con-elated set of polymorphisms as in the case of HLA molecular typing. In the first instance, thi covering set for the entire multiplicity of mutations contains multiple subsets, each subset bei ng associated with one designated site. In such a case, two or more probes are provided to ascertain heterozygosity. For the purpose of general SNP identification and confirmatory sequencing, degenerate probe sets can be provided to contain up to four labeled (e.g., bead-displayed) probes per polymorphic site. In the second instance, the covering set contains subsets constructed to minimize the number of probes in the set, as elaborated herein. The set of designated probes is designed to identify allele-specific sequence configurations on the basis of tlv elongation pattern.<br>
While this method of accommodating or identifying non-designated polymorphic sites is especially useful in connection with the multiplexed elongation of sequence specific probes, it al-D may be used in conjunction with single base extension of probes, also known as mini-scijuencing (see e.g., Pastinen, et al. Genome Res. 7: 606-614 (1997), incorporated herein by<br>
reference).<br>
The elongation-mediated method of analysis of the present invention, unlike the single-base probe extension method , may be used to detect not only SNPs, but also to detect other types of polymorphisms such as multiple (e.g., double, triple, etc.) nucleotide polymorphisms, as well as insertions and deletions commonly observed in the typing of highly polymorphic genetic loci such as HLA. hi these complex systems, sequence-specific probe elongation in accordance with thv methods of this invention, simplifies the detection step because two or more probes are pinvided for each polymorphic target location of interest and the detection step is performed 01: iy to determine which of the two or more probes was elongated , rather than to distinguish Ix . \\ ten two extended probes, as in the case of single-base probe extension Thus, although the methods of this invention accommodate the use of multiple fluorophore or chromophore labels<br><br>
in the detection step, a single universal label generally will suffice for the sequence specific pi i ibe elongation. This is in contrast to single-base extension methods whose application in a multiplexed format requires at least two fluorophore or chromophore labels.<br>
DNA methylation: In certain embodiments, methods and compositions for determining the mi thylation status of DNA are provided. Cytosine methylation has long been recognized as an in i portant factor in the silencing of genes in mammalian cells. Cytosine methylation at single CpG dinucleotides within the recognition sites of a number of transcription factors is enough to blck binding and related to several diseases. eMAP can be used to determine the methylation status of genomic DNA for diagnostic and other purposes. The DNA is modified by sodium bi; ulfite treatment converting unmethylated Cytosines to Uracil. Following removal of bisulfite an I completion of the chemical conversion, this modified DNA is used as atemplate for PCR. A pair of probes is designed, one specific for DNA that was originally methylated for the gene of interest, and one specific for unmethylated DNA. eMAP is performed with DNA polymerase and one labeled dNTP and unlabeled mixture of 3 dNTPs or ddNTPs. The elongated product on thv specific bead surface can indicate the methylation status.<br>
Si'inclive Sequencing: In certain other embodiments of this invention, selective sequencing (also re i erred to as "sequencing") is used for concurrent interrogation of an entire set of designated polymorphisms within a nucleic acid sequence in order to determine the composition at each such sii 
Cystic Fibrosis Carrier Screening - One practical application of this invention involves the an i lysis of a set of designated mutations within the context of a large set of non-designated<br><br>
m niations and polymorphisms in the Cystic Fibrosis Transmembrane Conductance (CFTR) gene. Ejicli of the designated mutations in the set is associated with the disease and must be im lependently scored. In the simplest case of a point mutation, two encoded probes are provided to ensure alignment of their respective 3' termini with the designated site, with one probe aniicipating the wild-type, and the other anticipating the altered ("mutated") target sequence.<br>
However, to ensure elongation regardless of the specific target sequence configuration em ountered near the designated site, additional probes are provided to match any of the possible 01 likely configurations, as described in several Example herein. In a preferred embodiment, the covering probe set is constructed to contain probes displaying TEI sequences corresponding to all known or likely variations of the corresponding target subsequence. This ensures elongation in the presence of otherwise elongation-inhibiting non-designated polymorphisms located within a ; ingc of proximity of the designated site.<br>
In certain embodiments, the identification of the specific target configuration encountered in the non-designated sites is not necessary so long as one of the sequences provided in the covering pi he set matches the target sequence sufficiently closely to ensure elongation,and thus matches th target sequence exactly within the TEI region. In this case, all or some of the covering probes si iimg the same 3' terminus may be assigned the same code In a preferred embodiment, such pi •! ics may be associated with the same solid support ("probe pooling"). Probe pooling reduces tl o number of distinguishable solid supports required to represent the requisite number of TEI sequences. In one particularly preferred embodiment, solid supports are provided in the form of a et or array of distinguishable microparticles which may be decoded in-situ. Inclusion of additional probes in the covering probe set to identify additional polymorphisms in the target region is a useful method to elucidate haplotypes for various populations.<br>
HI.A - Another application of this invention involves the genetic analysis of the Human 1 i ikocyte Antigen (HLA) complex, allowing the identification of one or more alleles within n. 'ions of HLA encoding class I HLA antigens (preferably HLA-A, HLA-B, HLA-C or any combination thereof) and class n HLA antigens (preferably including HLA-DR, HLA-DQ, HLA-Di' or any combination thereof). Class I and n gene loci also maybe analyzed simultaneously.<br><br>
In contrast to the independent scoring of multiple uncorrelated designated mutations, identification of alleles (or groups of alleles) relies on the scoring of an entire set of elongation reactions. Each of these reactions involves one or more probes directed to a member of a selected set of designated polymorphic sites. The set of these elongation reactions produces a characteristic elongation signal pattern, hi a preferred embodiment, a binary pattern is produced, aligning a value of "1" to matching (and hence elongated) probes, and a value of "0" to non-e!
Tl total number of probes required for HLA typing depends on the desired resolution. The term "ri solution" is used here to indicate the degree of alleh'c discrimination. Preferably, the method 01 this invention allows typing of an HLA allele that is sufficient to distinguish different antigen groups. For example, A*01 and A*03 are different antigen groups that have to be distinguished in clinical applications. The National Marrow Donor Program (NMDP) recommended a panel fo molecular typing of the donors. The low-to-medium resolution required by the NMDP panel ni ;iiis that different antigen groups should be distinguished at all times. Further, at least some oJ the alleles within one group should be distinguished, though not necessarily all alleles. In certain embodiments, the present invention allows typing of the HLA allele to a low to medium re oiution, as defined by the NMDP standard (www.NMDPresearch.org), incorporated herein b} ieTerence.<br>
W ith such resolution, A*01, A*03 etc., will always be identified. A*0101 and A*0102 may not be necessarily distinguishable. For the SSO method, the current NMDP panel contains 30 probes foi I1LA-A; 48 for HLA-B and 31 for HLA-DR-B. High resolution HLA typing refers to the sii i lation when most of the alleles will be identified within each group. In this case, A*0101 and A 0102 will be distinguished. To reach such resolution, approximately 500 to 1000 probes will b
 <br>
I	his invention also provides strategies for designating sites and for designing probe sets for<br>
such designated sites in order to produce unique allele assignments based on the elongation<br>
reaction signal patterns. The design of covering probes explicitly takes into account the distinct<br>
respective functions of TEI and DA regions of each probe.<br>
A covering set of probes associated with a given designated site is constructed to contain subsets. Each subset in turn contains probes displaying identical TEI regions. A mismatch in a single p. -<iiion witliin the tei region or a mismatch in three more positions within da pi eludes elongation. accordingly elongation of two probes displaying such differences composition generally will produce distinct patterns. all can be multiplexed parallel reaction as long they are individually encoded. p i cf erred embodiment encoding is accomplished by attaching to color-encoded beads.></iiion>
Pi obes displaying identical TEI subsequences and displaying DA subsequences differing in not in Me than two positions generally will produce elongation reactions at a yield (and hence signal inicnsity) either comparable to, or lower than that of a perfect match. In the first case which indicates tolerance of the mismatch, the set of alleles matched by the probe in question will be e\ paneled to include alleles that display the tolerated mismatched sequence configurations within the DA region. In the second case, indicating only partial tolerance, three approaches are described herein to fiirther elucidate the allele matching pattern, hi the first approach, probes di splaying one or two nucleotide polymorphisms in their respective DA regions are included in tli covering set. Information regarding the target sequence is obtained by quantitatively c
i place the DA region farther away from the TEI region in order to avoid target polymorphisms.<br>
In the third approach, probes are optionally pooled in such cases offering only a modest expansion of the set of matched alleles.<br>
I1	certain embodiments of this invention probes preferably are designed to be complementary<br>
to certain target sequences that are known to correlate with allele combinations within the HLA<br>
gi lie locus. Known polymorphisms are those that have appeared in the literature or are available<br><br>
from a searchable database of sequences (e.g., www.NMDProcessing.org'). In certain ei i ilxxliments, the HLA gene of interest belongs to HLA class I group, (e.g., HLA-A, HLA-B or 1~] I A-C or combination thereof). In certain other embodiments, the HLA gene of interest belongs to the HLA class II group, (e.g., DR, DQ , DP or combination thereof). The HLA class I and cl.iss 11 loci may be examined in combination and by way of concurrent interrogation. Pi i ibes previously employed in the SSP/gel method also may be used in this invention. Pi teiably, the probes set forth in Bunce et al., Tissue Antigen, 46: 355-367 (1995) and/or Bunce ei il,. Tissue Antigen, 45:81-90(1995), (each of which are hereby incorporated by reference) are UMX! in preparing the probes for this invention. The probe sequences or HLA sequence in lormation provided in WO 00/65088; European Application No. 98111696.5; WO 00/70006; ai"! bilich et al., Immunity, 14: 347-356 (2001), (each of which are hereby incorporated by reference) may be used in designing the probes for this invention.<br>
The complexity of an encoded bead array is readily adjusted to accommodate the requisite typing re • solution. For example, when 32 types of beads are used for each of four distinct subarrays, a to1 ;il of 128 probes will be available to attain a medium level of resolution for HLA class I and cl.iss LI typing in a multiplexed elongation reaction. Analogously, with 128 types of beads and fb 11 subarrays, or 64 types of beads and 8 subarrays, a total of 512 probes will be available to an am a high resolution of HLA class I and class II typing in a multiplexed elongation reaction.<br>
Ti o encoded bead array format is compatible with high throughput analysis. For example, ci i tain embodiments of this invention provide a carrier that accommodates multiple samples in a format that is compatible with the dimensions of 96-well microplates, so that sample distribution may be handled by a standard robotic fluid handling apparatus. This format can accommodate multiple encoded bead arrays mounted on chips and permits the simultaneous completion of multiple typing reactions for each of multiple patient samples on a single multi-ci' ip canierln a 96-well carrier testing 128 types per patient, more than 10, 000 genotypes can be di u tmined at a rate of throughput that is not attainable by current SSP or SSO methodology.<br>
In certain embodiments of this invention, the elongation reaction can be combined with a hybridization reaction to correlate subsequences on the same DNA target strand, a<br><br>
capability referred to herein as "phasing". Phasing resolves ambiguities in allele assignment ai i si ug from the possibility that a given elongation pattern is generated by different combinations of alleles. Similarly, phasing is useful in the context of haplotying to assign polymorphisms to th same DNA strand or chromosome.<br>
In certain embodiments of this invention, the annealing and elongation steps of the elongation reaction can be combined as a one-step reaction. Furthermore, means to create continuous or discrete temperature variations can be incorporated into the system to accommodate multiple op lima! conditions for probes with different melting temperatures in a multiplexed reaction.<br>
In certain embodiments of this invention, encoded bead arrays are formed on solid substrates. 1 iicse solid substrates may comprise any suitable solid material, such as glass or semiconductor, tli ii lias sufficient mechanical strength and can be subjected to fabrication steps, if desired. In si no embodiments, the solid substrates are divided into discrete units known as "chips". Chips comprising encoded bead arrays maybe processed individually or in groups, if they are loaded in in a multichip carrier. For example, standard methods of temperature control are readily applied to set the operating temperature of, or to apply a preprogramed sequence of temperature cl mges to, single chips or to multichip carriers. Further, chips may be analyzed with the direct in,aiding capability of Random Encoded Array Detection ("READ"), as disclosed in P
In one embodiment, the invention provides a method for polymorphism analysis in which each<br><br>
target nucleic acid sequence is used as a template in multiple elongation reactions by applying OJK or more "annealing-extending-detecting-denaturing" temperature cycles. This method achieves linear amplification with in-situ detection of the elongation products. This additional capability obviates the need for a first step of sequence-specific amplification of a polynucleotide sample<br>
In i grati on of assay procedure and signal amplification by way of cycling not only simplifies and ac derates the completion of genetic analysis, but also eliminates the need to develop, test and inipienieiil multiplexed PCR procedures. The methods of this invention also provide a high-tlnoughput fonnat for the simultaneous genetic analysis of multiple patient samples.<br>
Si v;eral embodiments of this invention are provided for the multiplexed elongation of sequence-sp cific probes to permit simultaneous evaluation of a number of different targets. In certain en iiodiments, oligonucleotide probes are immobilized on a solid support to create dense patterns ol probes on a single surface, e.g., silicon or glass surface. In certain embodiments, pj&gt; synthesized oligonucleotide probes are immobilized on a solid support, examples of which in. I ude silicon, chemically modified silicon, glass, chemically modified glass or plastic. These so ul supports may be in the form of microscopic beads. The resolution of the oligonucleotide at iy is determined by both spatial resolution of the delivery system and the physical space requirements of the delivered nucleotide solution volume. [See Guo, et al., Nucleic Acids Res. 22 5456-5465 (1994); Fahy, et al., Nucleic Acid Res. 21:1819-1826 (1993); Wolf, et al., Nuc. A; uls Res. 15: 2911-2926 (1987); and Ghosh, et al., Nuc. Acids Res. 15: 5353-5372 (1987).]<br>
Tins invention provides methods for multiplexed assays. In certain embodiments, sets of eliigation probes are immobilized on a solid phase in a way that preserves their identity, e.g., b\ spatially separating different probes and/or by chemically encoding the probe identities. One 01 more solution-borne targets are then allowed to contact a multiplicity of immobilized probes in i he annealing and elongation reactions. This spatial separation of probes from one another by immobilization reduces ambiguities in identifying elongation products. Thus, this invention ol crs advantages over the existing PCR-SSP method, which is not adaptable to a high tlv oughput fonnat because of (i) its requirement for two probes for each PCR amplification; (ii) tli v onipetition between overlapping probes for the highly polymorphic genes, such as HLA, in<br><br>
a multiplexed homogeneous reaction; and (iii) the difficulty in distinguishing between specific pi "ducts in such a multiplexed reaction..<br>
In ;i preferred embodiment, probes are attached, via their respective 5' termini, to encoded miaopailicles ("beads") having a chemically or physically distinguishable characteristic that in, quc'y identifies the attached probe. Probes capture target sequences of interest contained in a niution that contacts the beads. Elongation of the probe displayed on a particular bead pi &gt; «luces an optically detectable signature or a chemical signature that may be converted into an opi ically detectable signature. In a multiplexed elongation reaction, the optical signature of each p;n ticipating bead uniquely corresponds to the probe displayed on that bead. Subsequent to the pi "lii: elongation step, one may determine the identity of the probes by way of particle id limitation and detection, e.g., by flow cytometry.<br>
In certain embodiments, beads may be arranged in a planar array on a substrate before the el. -neation step. Beads also may be assembled on a planar substrate to facilitate imaging after the eli &gt;n gat ion step. The process and system described herein provide a high throughput assay format permitting the instant imaging of an entire array of beads and the simultaneous genetic analysis o! multiple patient samples.<br>
Tl o array of beads may be a random encoded array, in which a chemically or physically di unguishable characteristic of the beads within the array indicates the identity of oluonucleotide probes attached to the beads. The array may be formed according to the READ format<br>
Tl c bead array may be prepared by employing separate batch processes to produce application-si ri'.ic substrates (e.g., a chip at the wafer scale). Beads that are encoded and attached to oliuonucleotide probes (e.g., at the scale of about 10s beads/100 pi suspension) are combined \vi ill a substrate (e.g., silicon chip) and assembled to form dense arrays on a designated area of tin substrate. In certain embodiments, the bead array contains 4000 beads of 3.2 pjm diameter and has a dimension of 300 [im by 300 p.m. With beads of different size, the density will vary. M ilnplc bead arrays also can be formed simultaneously in discrete fluid compartments<br><br>
maintained on the same chip. Such methods are disclosed in U.S. Application Serial No. 10 192,3 51, filed July 9, 2002, which is incorporated herein by reference in its entirety.<br>
Brad arrays may be formed by the methods collectively referred to as "LEAPS", as described in US Patent No. 6,251,69) and PCT International Application No. PCT/USOO/25466), both of vvl ich are incorporated herein by reference.<br>
Tlie substrate (e.g., a chip) used in this invention may be in the form of a planar electrode p.i • tamed in accordance with the interfacial patterning methods of LEAPS. For example, the smstrate may be patterned with oxide or other dielectric materials to create a desired configuration of impedance gradients in the presence of an applied AC electric field. Patterns m.;y be designed so as to produce a desired configuration of AC field-induced fluid flow and corresponding particle transport. Substrates may be patterned on a wafer scale by using sc 11 icouductor processing technology. In addition, substrates may be compartmentalized by di H tiling a thin film of a UV-patternable, optically transparent polymer to affix to the substrate a uesired layout of fluidic conduits and compartments. These conduits and compartments confine fluid in one or several discrete compartments, thereby accommodating multiple samples on a given substrate.<br>
B '.;ad arrays may be prepared using LEAPS by providing a first planar electrode that is in substantially parallel to a second planar electrode ("sandwich" configuration) with the two eh ctrodes being separated by a gap and containing a polarizable liquid medium, such as an ek ctrolyte solution. The surface or the interior of the second planar electrode may be patterned wiih the interfacial patterning method. The beads are introduced into the gap. When an AC voltage is applied to the gap. the beads form a random encoded array on the second electrode (c ;;., a "chip").<br>
In mother embodiment of LEAPS, an array of beads may be formed on a light-sensitive electrode (c- -... a "chip") Preferably, the sandwich configuration described above is also used with a pi ;iar light sensitive electrode and another planar electrode. Once again, the two electrodes are st j .urated by the a gap and contain an electrolyte solution. The functionalized and encoded beads<br><br>
an introduced into the gap. Upon application of an AC voltage in combination with light, the be uls form an array on the light-sensitive electrode.<br>
In ulain embodiments of the present invention, beads maybe associated with a chemically or pi, &gt; sically distinguishable characteristic. This may be provided, for example, by staining beads wi' h sets of optically distinguishable tags, such as those containing one or more fluorophore or cb'omophore dyes spectrally distinguishable by excitation wavelength, emission wavelength, e&gt; iled-state lifetime or emission intensity. The optically distinguishable tags may be used to si; n beads in specified ratios, as disclosed, for example, in Fulwyler, US 4,717,655 (Jan 5, 1("<x staining may also be accomplished by swelling of particles in accordance with methods k>\vn to those skilled in the art, (Molday, Dreyer, Rembaum &amp; Yen, J. Mol Biol 64, 75-88 (1 '75); L. Bangs, "Uniform latex Particles, Seragen Diagnostics, 1984). For example, up to tv- -Ive types of beads were encoded by swelling and bulk staining with two colors, each in i vidually in four intensity levels, and mixed in four nominal molar ratios. Alternatively, the m i hods of combinatorial color encoding described in International Application No. PCT/US 98 10719 (incorporated by reference in its entirety) can be used to endow the bead arrays with optically distinguishable tags. In addition to chemical encoding, beads may also be rendered in gnetic by the processes described in PCT/US0/20179.<br>
Ir. iddition to chemical encoding with dyes, beads having certain oligonucleotide primers may bt spatially separated ("spatial encoding"), such that the location of the beads provides in i ormation as to the identity of the beads. Spatial encoding, for example, can be accomplished wiihin a single fluid phase in the course of array assembly by usingLight-controlled F • rirokinetic Assembly of Particles near Surfaces (LEAPS). LEAPS can be used to assemble pi.inar bead arrays in any desired configuration in response to alternating electric fields and/or in accordance with patterns of light projected onto the substrate.<br>
1 I APS can be used to create lateral gradients in the impedance at the interface between a silicon cl p and a solution to modulate the electrohydrodynamic forces that mediate array assembly. Fl rtncal requirements are modest: low AC voltages of typically less than 10VPP are applied ai. i oss a lluid gap between two planar electrodes that is typically lOOfjm. This assembly process<br><br>
is rapid and it is optically programmable: arrays containing thousands of beads are formed within si -i-onds under an applied electric field. The formation of multiple subarrays can also occur in m iltiple fluid phases maintained on a compartmentalized chip surface.<br>
5	i i bsequent to the formation of an array, the array may be immobilized. For example, the bead<br>
arrays may be immobilized, for example, by application of a DC voltage to produce random<br>
ei coded arrays. The DC voltage, set to typically 5-7 V (for beads in the range of 2-6um and for<br>
a 'ap size of 100-150 um) and applied for 
di 'ped silicon substrate would form the anode, causes the array to be compressed to an extent<br>
facilitating contact between adjacent beads within the array and simultaneously causes beads to<br>
b&gt; moved toward the region of high electric field in immediate proximity of the electrode surface.<br>
0 ice in sufficiently close proximity, beads are anchored by van der Waals forces mediating<br>
pi ysical adsorption. This adsorption process is facilitated by providing on the bead surface a<br>
population of "tethers" extending from the bead surface; polylysine and streptavidin have been<br>
ui-od for this purpose.<br>
In certain embodiments, the particle arrays may be immobilized by chemical means, e.g, by foi ming a composite gel-particle film. In one exemplary method for forming such gel-composite piirticle films, a suspension of microparticles is provided which also contains monomer, cr isslinker and initiator for in-situ gel formation. The particles are assembled into a planar<br>
6	;embly on a substrate by using LEAPS.  AC voltages of 1 -20 VM in a frequency range from<br>
lt'0'sofhertzto several kilohertz are applied between the electrodes across the fluid gap. In the<br>
presence of the applied AC voltage, polymerization of the fluid phase is triggered after array<br>
assembly by thermally heating the cell to ~ 40-45°C using an infra-red (IR) lamp or<br>
pltotoinitiating the reaction using a mercury lamp source. The resultant gel effectively entraps the<br>
p. i. 11 c le array.   Gels may be composed of a mixture of acrylamide and bisacrylamide of varying<br>
m niomer concentrations from 20% to 5% (acrylamide : bisacrylamide = 37.5 : 1, molar ratio),<br>
bm any other low viscosity water soluble monomer or monomer mixture may be used as well.<br>
i heroically immobilized functionalized microparticle arrays prepared by this process may be<br>
»&gt; d for a variety of bioassays, e.g., ligand receptor binding assays.<br><br>
Iii one ev: 'tiple, thermal hydrogels are formed using azodiisobutyramidine dihydrochloride as a ilienii initiator at a low concentration to ensure that the overall ionic strength of the p
•o<br>
solution.<br>
In certain embodiments, the particle arrays may be immobilized by mechanical means. For evunple, an array of microwells maybe produced by standard semiconductor processing methods ii the low impedance regions of a silicon substrate. Particle arrays maybe formed using such si: uctures. In certain embodiments LEAPS mediated hydrodynamic and ponderomotive forces ar utilized to transport and to accumulate particles on the hole arrays. The AC field is then sv itched off and particles are trapped into microwells and thus mechanically confined. Excess bi ids are removed leaving behind a spatially ordered random bead array on the substrate surface.<br>
Substrates (e.g., chips) can be placed in one or more enclosed compartments that permit samples ar.'l reagents to be transported in and out of the compartments through fluidic interconnection. R :actions can also be performed in an open compartment format such as a microtiter plate. R 1'ients may be pipetted on top of the chip by robotic liquid handling equipment, and multiple samples may be processed simultaneously. Such a format accommodates standard sample processing and liquid handling for the existing microtiter plate format and integrates sample pi icessing and array detection.<br>
In I'.criain embodiments of this invention, encoded beads are assembled on the substrate surface, bin nol in an array. For example, by spotting bead suspensions into multiple regions of the si.. isiraU; and allowing beads to settle under gravity, assemblies of beads can be formed on the siiiisti ate In contrast to the bead arrays formed by LEAPS, these assemblies generally assume di udered configurations of low-density or non-planar configurations involving stacking or cl. mpmg of beads, thereby preventing imaging of affected beads. However, the combination of sp 'lial and color encoding attained by spotting mixtures of chemically encoded beads into a<br><br>
multiplier jf discrete positions on the substrate still allows multiplexing.<br>
In certain embodiments, a comparison of an image of an array after the assay with a tecoded image of the array can be used to reveal chemically or physically distinguishable characteristics, as well as the elongation of probes. This comparison can be achieved by using, for example, an optical microscope with an imaging detector and computerized image capture ai .1 analysis equipment. The assay image of the array is taken to detect the optical signature that ir licates the probe elongation. The decoded image is taken to determine the chemically and/or pi ysically distinguishable characteristics that uniquely identify the probe displayed on the bead surface. In this way, the identity of the probe on each particle in the array may be identified by a iistinguishable characteristic.<br>
In i age analysis algorithms may be used in analyzing the data obtained from the decoding and the a.*- ;ay images. These algorithms may be used to obtain quantitative data for each bead within an ai -:iy. The analysis software automatically locates bead centers using a bright-field image of the ai ay as a template, groups beads according to type, assigns quantitative intensities to individual bi ids, rejects "blemishes" such as those produced by "matrix" materials of irregular shape in se um samples, analyzes background intensity statistics and evaluates the background-corrected m
Pi obe elongation may be indicated by a change in the optical signature, or a change in chemical si i nature which may be converted to a change in optical signature, originating from the beads di playing elongated probes, for example. Direct and indirect labeling methods well known in th art are available for this purpose. Direct labeling refers to a change in optical signature re ilting from the elongation; indirect labeling refers to a change introduced by elongation which re. uires one or more additional steps to produce a detectable optical signature. In certain en ihodiments, fluorophore or chromophore dyes may be attached to one of the nucleotides added<br>
a; an ingredient of probe elongation, such that probe elongation changes the optical signature of iicads by changing, for example, fluorescence intensities or by providing other changes in<br>
t!   optical signatures of beads displaying elongation products.<br><br>
EXAMPLES<br>
Tl io present invention will be better understood from the Examples which follow. It should be understood that these examples are for illustrative purposes and are not to be construed as limiting this invention in any manner.<br>
EXAMPLE 1 - Staggered Probe Design for Multiplexed SSP Analysis Probes for each polymorphism are immobilized on a solid phase carrier to provide a format in which multiple concurrent annealing and extension reactions can proceed with minimal mutual interference. Specifically, this method provides a design which accommodates overlapping probes, as ill .strated in Fig. 1. In this example, we consider three alleles: allele A, allele B and allele C. P ones 1 and 2 detect SNPs that are aligned with their respective 3" termini while probes 3 and 4 detect two-nucleotide polymorphisms that are aligned with their respective 3' termini. The polymorphic sites targeted by probes 1 and 2 are located five nucleotides upstream of those la jcted by probes 3 and 4. This design permits each probe to bind its corresponding target and p* mils elongation to proceed when there is a perfect match at the designated polymorphic site. 1: HIS, probes 1 and 3 match allele A, probe 2 and possibly probe 3 match allele B, and probes 1 ,ind 4 match allele C<br>
EXAMPLE 2 : Probe Design for HLA Typing<br>
Ti i design probes for the analysis of the polymorphic region ranging from base 106 to base 125 ol i he DRB gene, twenty-two different types of sequences for the 20 base long fragment were located in the DRB database. These are listed in the table below:<br><br>
7<br>
104 26 l<br>
3<br><br>
DRB1*0101<br>
DRB1*03011<br>
DRB1*04011<br>
DRB1*0434<br>
DRB1*07011<br><br>
TTCTTGTGGCAGCTTAAGTT TTCTTGGAGTACTCTACGTC TTCTTGGAGCAGGTTAAACA TTCTTGGAGCAGGTTAAACC TTCCTGTGGCAGGGTAAGTA<br><br>
1<br>
28<br>
1<br>
1<br>
1<br>
1<br>
2<br>
1<br>
1<br>
18<br>
9<br>
1<br>
1<br>
16<br>
1<br>
6<br>
14<br><br>
DRB1*07012<br>
DRB1*0801<br>
DRB1*0814<br>
DRB1*0820<br>
DRB1*0821<br>
DRB1*09012<br>
DRB1*10011<br>
DRB1*1122<br>
DRB1*1130<br>
DRB1*15011<br>
DRB3*01011<br>
DRB3*0102<br>
DRB3*0104<br>
DRB3*0201<br>
DRB3*0212<br>
DRB4*01011<br>
DRB5*01011<br><br>
TTCCTGTGGCAGGGTAAATA TTCTTGGAGTACTCTACGGG TTCTTGGAGTACTCTAGGGG TTCTTGGAGTACTCTACGGC TTCTTGGAGTACTCTATGGG TTCTTGAAGCAGGATAAGTT TTCTTGGAGGAGGTTAAGTT TTCTTGGAGCAGGCTACACA TTCTTGGAGTTCCTTAAGTC TTCCTGTGGCAGCCTAAGAG TTCTTGGAGCTGCGTAAGTC TTCTTGGAGCTGTGTAAGTC TTCTCGGAGCTGCGTAAGTC TTCTTGGAGCTGCTTAAGTC TTCTTGCAGCTGCTTAAGTC TTCTTGGAGCAGGCTAAGTG TTCTTGCAGCAGGATAAGTA<br><br>
Tl first column contains the number of alleles sharing the sequence listed in third column, the se
 <br>
104<br><br>
DRB1*03011<br><br>
TTCTTGGAGTACTCTACGTC<br><br><br><br>
1 9<br>
1 1 16<br><br>
DRB1*1130<br>
DRB3*01011<br>
DRB3*0102<br>
DRB3*0104<br>
DRB3*0201<br><br>
TTCTTGGAGTgCctTAaGTC TTCTTGGAGctgcgTAaGTC TTCTTGGAGctgTgTAaGTC TTCTcGGAGctgcgTAaGTC TTCTTGGAGctgctTAaGTC<br><br><br><br>
DRB3*0212<br><br>
TTCTTGcAGctgctTAaGTC<br><br>
7 1 2<br><br>
DRB1*0101<br>
DRB1*09012<br>
DRB1*10011<br><br>
TTCTTGTGGCAGCTTAAGTT TTCTTGaaGCAGgaTAAGTT TTCTTGgaGGAGgTTAAGTT<br><br><br><br>
26<br>
1<br><br>
DRB1*04011 DRB1*1122<br><br>
TTCTTGGAGCAGGTTAAACA TTCTTGGAGCAGGcTAcACA<br><br><br><br>
DRB1*0434<br><br>
TTCTTGGAGCAGGTTAAACC<br><br><br><br>
14<br><br>
DRB1*07011 DRB5*01011<br><br>
TTCCTGTGGCAGGGTAAGTA TTCtTGcaGCAGGaTAAGTA<br><br><br><br>
DRB1*07012<br><br>
TTCCTGTGGCAGGGTAAATA<br><br><br><br>
28<br><br>
DRB1*0801<br><br>
TTCTTGGAGTACTCTACGGG<br><br><br><br>
DRB1*0814<br>
DRB1*0821<br><br>
TTCTTGGAGTACTCTAgGGG TTCTTGGAGTACTCTAtGGG<br><br><br><br>
DRB1*0820<br><br>
TTCTTGGAGT ACTCTACGG C<br><br><br><br>
18<br><br>
DRB1*15011<br><br>
TTCCTGTGGCAGCCTAAGAG<br><br><br><br>
10   6<br><br>
DRB4*01011<br><br>
TTCTTGGAGCAGGCTAAGTG<br><br>
• or sequences in the same group, variations between the first sequence of the group and the i\ ^ ace indicated in lower case. Three probe sequences are used to illustrate the application of o r probe design rules. The first sequence in the first group is selected as probe el; the 6th ?, i|tience in the first group is selected as probe e2; and the first group in the 7th sequence is Si acted as probe e3.<br>
P u- to requirement for perfect complementarity of the target and the probe's TEI region, s, plaices in group 2 to group 10 do not produce elongation products for el and e2. Similarly,<br><br>
se. uences in groups other than the 7th group do not produce elongation products for e3. Each ui up is distinctive from the others with respect to elongation reaction patterns.<br>
F
E: .-.ept for the target matching el, the remaining 5 sequences only differ from e2 by one or two<br>
n rleotides as indicated below:<br><br>
16<br>
1<br>
9<br>
1 1 1<br><br>
DRB3*0201<br>
DRB1*1130<br>
DRB3*01011<br>
DRB3*0102<br>
DRB3*0104<br>
DRB3*0212<br><br>
1,2	M<br>
TTCTTGGAGCTGCTTAAGTC e2<br>
TTCTTGGAGtTcCTTAAGTC	a<br>
TTCTTGGAGCTGCgTAAGTC	b<br>
TTCTTGGAGCTGtgTAAGTC	c<br>
TTCTcGGAGCTGCgTAAGTC	d<br>
TTCTTGcAGCTGCTTAAGTC	e<br><br>
T --i: sequences are cross-reactive. When targets for sequences b and e, which differ from e2 by o t: base at respective positions M-7 and M-14 anneal to probe e2, the non-designated pt lyniorphism(s) in the annealing region will be tolerated and the elongation reaction will pi &gt;ceed to substantially the same degree as for perfectly matched sequences. When targets for si inences a, c, and d, which differ from e2 by two nucleotides anneal to probe e2, the el nuation reaction will exhibit only partial tolerance of the non-designated polymoprhism(s). 0 c approach to improve on this situation is to provide separate probes for a, c, and d, then qi mtjtaiively analyze the yield of elongation products by analyzing signals intensitities to Hi'utily the correct sequences. An alternative would be to bridge the non-designated p
 <br>
For (he sequences in the 7th group, the other two sequences will be partially tolerated by the e3 pi-. he 1 hese three sequences may be pooled. The e2 probe will yield elongation products for •3d allcles instead of 28 alleles.<br>
F^ \MP1 R 3 : Utilizing Mismatch Tolerance  To ModifyAllele Binding Patterns<br>
Pi In- DR-13e, GGACATCCTGGAAGACGA, was used to target the bases 281-299 of the DRB gc-iio. Thirty-four alleles, including allele DRB1*0103, are perfectly matched to this sequence. Tims, in the binding pattern, 13e is positive for theses 34 alleles (that is, 13e will yield elMigalion products with these 34 alleles). Several additional alleles display the same TEI ix ion but display non-designated polymorphisms in their respective annealing regions. For e&gt;&gt; nnple, five alleles, such as DRB 1*0415, contain T in instead of A in position 4 while four all les. such as DRB1* 1136, contain C in the that position. Due to mismatch tolerance in the an i icali ng region, target sequences complementary to these nine alleles will produce elongation re. icl ion patterns similar to that of the perfectly matched sequence. The result is shown in Fig. 2. TO-3 and TO-4 are completely complementary sequences to allele *0415 and *1136, re .pcctively.<br>
DRB 1*0103	GACATCCTGGAAGACGA	34 alleles<br>
DRB 1*0415	GACTTCCTGGAAGACGA	5 alleles<br>
DRB 1*1136	GACCTCCTGGAAGACGA	4 alleles<br>
FA AMPLE 4 :    Design of Linker Structure in the Probes to Bridge Non-designated Polymorphisms<br>
A illustrated in Fig. 3, an anchor sequence is derived from conserved sequence regions to ei; uue specific and strong annealing. It is not designed for polymorphism detection. For that purpose, a shorter sequence for polymorphism detection is attached to the anchoring sequence Iv \vnv of a neutral chemical linker. The shorter length of the sequence designed for pi -i vinorphism detection will limit potential interference to non-designated polymorphisms in the ii. .nediute vicinity of the designated site and thus decreases the number of possible sequence C( ubinations required to accommodate such interfering polymorphisms This approach avoids<br><br>
highly dense polymorphic sites in certain situations. For example, it would be possible to di; tinguish between the sequences listed in Example 3 using a probe which takes into account thi additional polymorphism^). Illustrative designs of the linker and the sequences are listed<br>
be low.<br>
I iiikei 13-5	AGCCAGAAGGAC/Spacer 18/spacer 18/GGAAGACGA<br>
linker 13-8	AGCCAGAAGGAC/Spacer 18/spacer 18/AGACGA<br>
linker 13-11	AGCCAGAAGGAC/Spacer 18/spacer 18/CGA<br>
EXAMPLE 5: PHASING<br>
Tl o present invention also is useful in reducing ambiguities that arise when two or more allele cr nhinations can produce the same reaction pattern. In a simulated situation shown in Figs. 4 , nd 5, allele Awhich matches - and hence produces an elongation product with - Probe 1 and Pi•• &gt;be 3, and allele B, which matches Probe 2 and Probe 4 when present in the same multiplexed reaction, generate the same total reaction pattern as does the combination of allele C which m tches Probe 1 and 2, and allele D which matches Probe 3 and and Probe 4.. Such ambiguity c; !x reduced or eliminiated by using the detection methods provided in this invention to an ilyze the elongation product of Probe 1 by hybridization using a labeled detection probe that is It-signed to target the same polymorphic site as Probe 3. If the result of the analysis is positive, only one allele combination, namely combination 1, is possible because Probe 1 and Probe 3 are a? ocialed with the same allele. The detection probe can be labeled by using any of the methods cl' closed in this invention or methods known in the art. If this identification detection step is pi ••formed together with the multiplexed elongation reaction detection, different labels are used fo: (he elongation detection and probe hybridization detection as shown in the Fig. 5.<br>
In ihis method, the ambiguity is resolved by assigning two or more polymorphisms to the same "p.iase" using elongation in conjunction with hybridization. Phasing is rapidly emerging as an in norlant concern for haplotype analysis in other genetic studies designed in the art. More pi -lies can be included by reacting them with the target sequentially, or they can be arranged in tli same reaction with different labels for detection.<br><br>
Tlic capability of combining probe elongation and hybridization reactions is demonstrated in ex pcriments using a sample sequence from HLA-B exon 3. The result is shown in Fig. 6. A pi' 'be SB3 P was elongated in the reaction and the elongated product was detected using a labeled DNA probe. For the two samples presented in Fig. 6A and 6B, SB 127r and SB3P, and SB285r and SB3P are in the same phase, respectively.<br>
EXAMPLE 6 - Model HLA Typing Reaction using Random Encoded Probe Arrays<br>
T
In the model reaction system, two pairs of probes were synthesized to contain SNPs at their re :votive 3' termini. The respective sequences were as follows::<br>
SSP13: AAGGACATCCTGGAAGACG; SSP24: AAGGACATCCTGGAAGACA;<br><br>
SSP16: ATAACCAGGAGGAGTTCC SSP36: ATAACCAGGAGGAGTTCG.<br>
Ti	probes were biotinylated at the 5' end; a 15-carbon triethylene glycol linker was inserted<br>
b
in	iutilization on the subsequent reactions. For each probe, coupling to encoded beads was<br>
pi	tunned using 50 ul of bead suspension. Beads were washed once with 500 ul of 20 mM Tris/<br>
p!	7.4. 0.5M NaCl {buffer C) and resuspended in 300 ul of that buffer. 2.5 ul of a 100 fiM<br>
si	mion of probe were added to the bead suspension and allowed to react for 30 min at room<br>
It	iperaiure. Beads were then washed three times with 20 mM Tris/pH7.4, 150 mM NaCl, 0.01%<br>
ti i	on and stored in 20mM Tris/pH 7.4,150 mM NaCl.<br>
T . following synthetic targets of 33 bases in length were provided:<br>
TA16:	GTCGAAGCGCAGGAACTCCTCCTGGTT ATGG AA<br>
TA36:	GTCGAAGCGCACGAACTCCTCCTGGTTATAGAA<br>
T A13:	GGCCCGCTCGTCTTCCAGGATGTCCTTCTGGCT<br>
TA24:	GGCCCGCTTGTCTTCCAGGATGTCCTTCTGGCT<br>
Targets were allowed to react with four probes (SSP13, SSP24, SSP16, SSP36) on the chip. An aliquot of 10 ul of a 100 nM solution of the target in annealing buffer of 0.2 M NaCl, 0.1% T: :ion X-100,10 mM Tris/pH 8.0,0.1 mM EDTA was applied to the chip and allowed to react fi 15 min at 30 °C. The chip was then washed once with the same buffer and was then covered \v ;li an extension reaction mixture including: lOOnM of TAMRA-ddCTP (absorption/emission: 5 1/580) (PerkinElmer Bioscience, Boston, MA), 10 uM dATP-dGTP-dTTP, ThermoSequenase (A mersham, Piscataway, NJ) in the associated buffer supplied by the manufacturer. The reaction \\ -- allowed to proceed for 5 min at 60 °C, and the chip was then washed in HiO. Decoding and a; .ty images of the chip were acquired using a Nikon fluorescence E800 microscope with an ai i) mated filter changer containing hydroxy coumarin, HQ narrow band GFP and HQ Cy3 filters it hiiic, green decoding images and for the assay image, respectively. An Apogee CCD KX85<br><br>
(Apogee Instruments, Auburn, CA) was used for image acquisition. In each reaction, only the perfectly matching target was extended producing, in the case of the SNPs tested here, di.1 Tirnir ;on between matching and non-matching targets in the range from 13-fold to 30-fold; tin is ilii ated in Fig. 7 for TA13.<br>
E\ AMPLE 7 - HLA-DR Typing of Patient Sample<br>
A ' "&gt;N A sample extracted from a patient was processed using a standard PCR protocol. The fol mwing primers were used for general DR amplification:<br>
forward primer:         GATCCTTCGTGTCCCCACAGCACG<br>
reverse primer:	GCCGCTGCACTGTGAAGCTCTC.<br>
TV: • PCR protocol was as follows: one cycle of 95 °C for 7 min, 35 cycles of 95 °C for 30 sec, 60 "C for 30 sec and 72 °C for 1 min and one cycle of 72 °C for 7 min.<br>
Tb PCR product, 287 bases in length and covering the DR locus, was denatured at 100 °C for 5 . ,in, chilled on ice and mixed with annealing buffer as described in Example 6 for the model re;;ciion. An aliquot of lOul was applied to each chip and reacted at 40 °C for 15 min. The el
Tl • multiplexed extension of sequence-specific probes using the PCR product produced from th patient sample produced results in accordance with the probe design. Of the four probes tested in parallel (SSP13, SSP16, SSP24, SSP36), SSP13 was elongated while the SNP probe SSP24 only showed background binding as did the unrelated SSP16 and SSP36 probes. As illustrated in Fig. 8, the multiplexed elongation of SSP significantly enhanced the discrimination be • ween matching and non-matching SNPs from approximately two-fold for an analysis based on the hybridization of matching and non-matching sequence-specific oligonucleotide probes to al <e></e><br>
EXAMPLE 8 : Group-Specific Amplification<br>
Pi-mers for group-specific amplification (GSA) are most frequently used when multiplexed Ir &gt;i idization with SSOs yields ambiguous assignments of heterozygous allele combinations. In su li a situation, GSA primers are selected to amplify selected sets of specific alleles so as to re' nove ambiguities, a labor-intensive additional assay step which delays the analysis. Using the m thods of the present invention, preferably an embodiment of displaying probe? on random ei iled bead arrays, GSA primers may be incorporated as probes into the multiplexed reaction tlr reby eliminating an entire second step of analysis. F AMPLE 9 : Analysis of HLA- DR, -A and -B Loci using Cell Lines Pi -!&gt;es (or the elongation-mediated multiplexed analysis of HLA-DR, HLA-A and HLA-B were di igned and tested using standard cell lines. The probes were derived from SSP probes pi \ iously reported in the literature (Bunce, M. et al, Tissue Antigens. 46:355-367 (1995), K nisa, P and Browning, M.J., Tissue Antigens. 47: 237-244 (1996), Bunce, M. et al, Tissue Avtigens. 45:81-90(1995)).<br>
The probes used for DR were:<br>
Si J:   ACGGAGCGGGTGCGGTTG<br>
SI 3:   GCTGTCGAAGCGCACGG<br>
Sill: CGCTGTCGAAGCGCACGTT<br>
Si. i v&gt;: GITATGGAAGTATCTGTCCAGGT<br>
SI 23: ACGTTTCTTGGAGCAGGTTAAAC<br>
SI • 32: CGTTTCCTGTGGCAGGGTAAGTATA<br>
S! ~3: TCGCTGTCGAAGCGCACGA<br>
Si 36: CGTTTCTTGGAGTACTCTACGGG<br>
Si 3i&gt;: TCTGCAGTAGGTGTCCACCA<br>
SK45: CACGTTTCTTGGAGCTGCG<br>
SU46: GGAGTACCGGGCGGTGAG<br>
SIMS: GTGTCTGCAGTAATTGTCCACCT<br>
SK52: CTGTTCCAGGACTCGGCGA<br>
SI 57: CTCTCCACAACCCCGTAGTTGTA<br>
Si 58: CGTTTCCTGTGGCAGCCTAAGA<br>
SI 60: CACCGCGGCCCGCGC<br>
Si:(.7: GCTGTCGAAGCGCAAGTC<br>
Si 71: (iCTGTCGAAGCGCACGTA<br>
N i (i    AAAAAAAAAAAAAAAAAA<br><br>
Sme of the probes have a SNP site at their respective 3' termini, for example: SR3 and SR33 ((, and A, respectively); SRI 1, SR67 and SR71 (T,C, and A, respectively). In addition, probes SR3 and 33 are staggered at the 3'-endwith respect to probes the group of SR11, 67 and 71 by<br>
one base.<br>
S :•' 2       GCTGTCGAAGCGCACGG S i • 3 3      TCGCTGTCGAAGCGCACGA S :i      CGCTGTCGAAGCGCACGTT<br>
S (//      GCTGTCGAAGCGCAAGTC S ":      GCTGTCGAAGCGCACGTA<br>
K action conditions were as described in Example 7 except that the annealing temperature was 5 "C instead of 40 °C, and the extension temperature was 70 °C instead of 60 °C. Doubles', .inded DNA was used as in Example 7. Single-stranded DNA generated better results under<br>
i -uncut conditions. Single-stranded DNA was generated by re-amplifying the initial PCR product ir. the same PCR program with only one of the probes. Results for two cell lines, W51 and SI '0010. are shown in Fig. 9 and Fig. 10. NEG , a negative control, was coupled to a selected<br>
type of bead. Signal intensity for other probes minus NEG was considered to be real signal for th probe and the values were plotted in the figures. The Y axis unit was the signal unit from the camera used in the experiment. The distinction between the positive and negative probes was unambiguous for each sample, hi particular, and in contrast to the situation typically encountered ii SSO analysis, it was not necessary to make comparisons to other samples to determine a rr'iablc threshold for each probe.<br>
'] lie probes used for HLA-A were:<br>
S vl)	CACTCCACGCACGTGCCA<br>
s \r	GCGCAGGTCCTCGTTCAA<br>
S\Q	CTCCAGGTAGGCTCTCAA<br>
S. \ R	(TCCAGGTAGGCTCTCTG<br>
S\X	GCCCGTCCACGCACCG<br>
SAZ	GGTATCTGCGGAGCCCG<br>
S \,VP	CATCCAGGTAGGCTCTCAA<br>
S \8	GCCGGAGTATTGGGACGA<br>
S\ 13	TGGATAGAGCAGGAGGGT<br><br>
S A16  GACCAGGAGACACGGAATA<br>
Ri suits for A locus exon 3, shown in Figs. 11 and Fig. 12, also were unambiguous. Fig. 12 ; • i so shows an example of the mismatch tolerance for a non-designated polymorphism. That is, \\hileallele0201, displaying C instead of A at position M-18, is not perfectly matched to<br>
pi &gt;he S AAP, the elongation reaction nonetheless proceeded because the polymerase detected<br>
a perfect match   for the designated polymorphism at the probe's 3' end and tolerated the<br>
mismatch at position M-18..<br>
Tie probes used for HLA-B were:<br>
S i.: 20   CCGCGCGCTCCAGCGTG SI • 2 46   CCACTCCATGAGGTATTTCC SM229   CTCCAACTTGCGCTGGGA S': 272   CGCCACGAGTCCGAGGAA S1
Experiments using these probes for typing HLA-B exon 2 were performed using reference cell lires. As with HLA-A, unambiguous results (not shown here) were obtained.<br>
I	N: AMPLE 10 : CF Mutation Analysis - Probe and Array Design for Probe Elongation<br>
This Example describes the design and application of a planar array of probes, displayed on c
Ti ic CFTR gene sequence from Genebank (\vw\v.ncbi.nlm.nih.gov) was used to design sixteen-<br>
II	T probes for the multiplexed analysis of the 25 CFTR mutations in the ACMG-CF mutation<br>
p.'iid. Probe sequences were designed using PROBE 3.0 (http://www.genome.wi.mit.edu) and<br><br>
aliened with respective exon sequences (http://searcnlauncher.bcm.tmc.edu/seq-sc iich/alignrnent.html). Oligonucleotides were designed to comprise 15 to 21 nucleotides, with a 0-50% G+C rich.base composition and synthesized to contain a 5' biotin TEG (Synthegen 1.. K to handle small deletions, the variable sequence of the TEI region was placed at or within 3 positions of the probe's 3' terminus. Probe compositions are listed in the table below.<br>
A ombination of 17 either pure blue or blue-green stained beads were used with CF mutation at ilysis. The 48 base long Human fi-actin gene (Accession #X00351) was synthesized and used ii: each reaction as an internal positive control. Sixteen base long complementary probes were in hided on each array. The CFTR gene sequence from Genebank (www.ncbi.nlm.nih.gov) was u; ;d fin probe design for analysis of 25 CFTR mutations in the ACMG-CF mutation panel. The pi 'he sequences were designed by PROBE 3;0 (http://www.genome.wi.mit.edu'). Each probe si iiicncc was aligned with respective exon sequences (http://searchlauncher.bcm.tmc.edu/seq-ii uctvalignment.html). Oligonucleotides were synthesized with a 5' biotin TEG (Synthegen TX) ai 1 coupled on the surface of beads in presence of 0.5 M NaCl. Beads were immobilized on the si luce of a chip by LEAPS.<br>
E CON MUTATIONS SEQUENCE<br>
3	G85E	CCCCTAAATATAAAAAGATTC<br>
G85E-X	CCCCTAAATATAAAAAGATTT<br>
4	1148	ATT CTC ATC TCC ATT CCA A<br>
1148-X	ATT CTC ATC TCC ATT CCA G<br>
621+1G&gt;T	TGT GTG CAA GGA AGT ATT AC<br>
62H1G&gt;T-X	TGT GTG CAA GGA AGT ATT AA<br>
R117H	TAG ATA AAT CGC GAT AGA GC<br>
Rl 17H-X	TAG ATA AAT CGC GAT AGA GT<br>
5	71 l-t-lG&gt;T	TAA ATC AAT AGO TAG ATA C<br>
TAA ATC AAT AGG TAC ATA A<br><br>
R334W	ATG GTG GTG AAT ATT TTC CG<br>
R334 W-X	ATG GTG GTG AAT ATT TTC CA<br>
R347P	ATT GCC GAG TGA CCG CCA TGC<br>
R347P-X	ATT GCC GAG TGA CCG CCA TGG<br>
1078delT	CAC AGA TAA AAA CAC CAC AAA<br>
1078delT-X	CAC AGA TAA AAA CAC CAC AA<br>
1078delT-X-2	CAC AGA TAA AAA CAC CAC A<br><br>
A455E A455E-X<br><br>
TCC AGT GGA TCC AGC AAC CG TCC AGT GGA TCC AGC AAC CT<br><br><br><br>
508<br>
1507<br>
F508<br><br>
CAT AGG AAA CAC CAA AGA T CAT AGG AAA CAC CAA A CAT AGG AAA CAC CAA T<br><br><br><br>
1717-1OA<br>
1717-1OA<br>
55 ID<br>
551D-X<br>
R553<br>
R553X<br>
R560<br>
R560X<br>
G542<br>
G542X<br><br>
CTG CAA ACT TGG AGA TGT CC CTG CAA ACT TGG AGA TGT CT TTC TTG CTC GTT GAC TTC TTG CTC GTT GAT TAAAGAAATTCTTGCTCG TAAAGAAATTCTTGCTCA ACCAATAATTAGTTATTCACC ACCAATAATTAGTTATTCACG GTGTGATTCCACCTTCTC C GTGTGATTCCACCTTCTC A<br><br><br><br>
IM-12<br><br>
1898 1898-X<br><br>
AGG TAT TCA AAG AAC ATA C AGG TAT TCA AAG AAC ATA T<br><br><br><br>
2183deLA<br><br>
TGT CTG TTT AAA AGA TTG T<br><br>
2183deLA-X<br><br>
TGT CTG TTT AAA AGA TTG C<br><br><br><br>
r i i4B<br><br>
2789 2789-X<br><br>
CAA TAG GAC ATG GAA TAC CAA TAG GAC ATG GAA TAC T<br><br><br><br>
]'  T 16<br><br>
3120 3120-X<br><br>
ACT TAT TTT TAC ATA C ACT TAT TTT TAC ATA T<br><br><br><br>
D1152 D1152<br><br>
ACT TAC CAA GCT ATC CAC ATC ACT TAC CAA GCT ATC CAC ATG<br><br><br><br>
3849-4-10kbC&gt;T-WTl 3849+10kbOT-Ml<br><br>
CCT TTC Agg GTG TCT TAC TCG CCT TTC Agg GTG TCT TAC TCA<br><br><br><br>
R1162<br>
R1162-X<br>
3659delC-WTl<br>
3659delCX-Ml<br>
3659delC-WT2<br>
3659delCX-M2<br><br>
AAT GAA CTT AAA GAC TCG AAT GAA CTT AAA GAC TCA GTA TGG TTT GGT TGA CTT GG GTA TGG TTT GGT TGA CTT  GTA GTA TGG TTT GGT TGA CTT GGT A GTA TGG TTT GGT TGA CTT   GTA<br><br><br><br>
W1282 W1282-X<br><br>
ACT CCA AAG GCT TTC CTC CT CCA AAG GCT TTC CTT<br><br><br><br>
N1303K N1303K-X<br><br>
TGT TCA TAG GGA TCC AAG TGT TCA TAG GGA TCC AAC<br><br><br><br>
Actin<br><br>
AGG ACT CCA TGC CCA G<br><br>
Prbes were attached, in the presence of 0.5 M NaCl, to differentially encoded beads , stained eil i icr pure blue or blue-green Beads were immobilized on the surface of a chip using LEAPS. A vnthetic 48 base Human B-actin gene (Accession #X00351) was included in each reaction<br>
as  M internal positive control.<br>
A i i/i Design - In a preferred embodiment, the 25 CF mutations were divided into four different gi ips so as to minimize sequence homologies between members of each group. . That is, mi iialions were sorted into separate groups so as to minimize overlap between probe sequences in .my such group and thereby to minimize cross-hybridization under conditions of multiplexed an lysis.. Each group, displayed on color-encoded beads, was assembled into a separate array. (! suits for this 4-chip array design are described in the following Example). Alternative robust ai designs also are disclosed herein.<br>
FA \MPLE 11: Multiplexed CF Mutation Analysis by Probe Elongation Using READ<br>
Ci omic DNA, extracted from several patients, was amplified with corresponding probes in a m H iplcx PCR (mPCR) reaction using the method described in L. McCurdy, Thesis, Mount Sinai Si tool of Medicine, 2000, which is incorporated by reference. This mPCR reaction uses cb -iierie primers tagged with a universal sequence at the 5' end. Antisense primers were pi &gt;sphorylated at the 5' end (Synthegen, TX). Twenty eight amplificationcycles were performed us ng a Perkin Elmer 9600 thermal cycler, each cycle comprising a 10 second denaturation step at &gt;4°C with a 48 second ramp, a 10 second annealing step at 60 °C with a 36 second ramp and a -i) second extension step at 72 °C with a 38 second ramp, each reaction (50 ^1) containing 500 n; genomicDNA, IX PCR buffer (10 mMTris HCL, 50mMKCL)0.1%TritonX-100), 1.5 mM N in 2, 200 uM each of PCR grade dNTPs and 5 units Taq DNA polymerase. Optimal probe c
I' R products were amplified with antisense 5'-phosphorylated primers. To produce single-si indcd DNA templates, PCR reaction products were incubated with 2.5 units of exonuclease<br><br>
in IX buffer at 37 °C for 20 min, followed by enzyme inactivation by heating to 75 °C for 10 nun. Under these conditions, the enzyme digests one strand of duplex DNA from he 5'-p]i &gt;sphorylated end and releases S'-phosphomononucleotides (J. W. Little, et al., 1967). Single-su ;inded targets also can be produced by other methods known in the art.<br>
Si igle or pooled PCR products (20 ng each) were added to an annealing mixture containing 10 m 1 Tris-HCL(pH 7.4) ImM EDTA, 0.2 MNaCl, 0.1% Triton X-100. The annealing mixture \\ is placed in contact with the encoded array of bead-displayed CF probes (of Example 10) and in. ubated at 37-55 °C for 20 minutes. The extension mixture - containing 3 U of Thermo Si i|uenase (Amersham Pharmacia Biotech NJ), IX enzyme buffer with either Fluorescein-labeled o I AMRA-labeled deoxynucleotide (dNTP) analogs (NEN Life Sciences) and 1 \wiole of each t\ •(. of unlabeled dNTP - was then added, and the elongation reaction was allowed to proceed fo 3 minutes at 60 °C. The bead array was washed with deionized, sterilized water (dsHaO) for 515 minutes. An image containing the fluorescence signal from each bead within the array was re orded using a fluorescence microscope equipped with a CCD camera. Images were analyzed t( determine the identity of each of the elongated probes. The results are shown in Fig. 15.<br>
E x'AMPLE 12:  Use of Covering Probes<br>
S veral SNPs have been identified within exon 10 of the CFTR gene. The polymorphisms in e on 10 are listed at the end of this Example. The following nine SNPs have been identified in tl,,; sequence of A508, the most common mutation in the CFTR gene (http://snp.cshl.org):<br>
dbSNP213450 A/G dbSNP 180001 C/T dbSNP 1800093 G/T 1648 A/G<br>
dbSNPl 00092 C/G dbSNP1801178A/G dbSNPlS00094A/G dbSNP 1800095 G/A<br><br>
lobes are designed to accommodate all possible SNPs are synthesized and coupled to color-ei (&gt;dcd beads. The primers for target amplification (described in Example 11) are also modified to :;kc into account all possible SNPs. The PCR-amplified target mediates the elongation of ti i.inally matched probes. The information collected from the analysis is twofold: iii ntification of mutations and SNPs.<br>
E - ON 10 POLYMORPHISMS<br>
cactgtagct gtactacctt ccatctcctc aacctattcc aactatctga atcatgtgcc (•  cttctctgtg aacctctatc ataatacttg tcacactgta ttgtaattgt ctcttttact 1.   ucccttgta tcttttgtgc atagcagagt acctgaaaca ggaagtattt taaatatttt<br>
1	uiiak-aaatg agttaataga atctttacaa ataagaatat acacttctgc ttaggatgat<br>
2	aattggaggc aagtgaatcc tgagcgtgat ttgataatga cctaataatg atgggtttta<br>
? ' tttccagact tcaCttctaa tgAtgattat gggagaactg gagccttcag agggtaaaat 3' 1 taagcacagt ggaagaartt cattctgttc tcagttttcc tggattatgc ctggcaccat 4    laaagaaaat AtCAtctTtg gtgtttccta tgatgaatat agatacagaa gcgtcatcaa<br>
4	i agcatgccaa ctagaAgagG taagaaacta tgtgaaaact ttttgattat gcatatgaac<br>
5	ccttcacact acccaaatta tatatttggc tccatartca atcggttagt ctacatatat<br>
6
6'   tgctttaagaagcttgcaaacacatgaaat aaatgcaatt tattttttaa ataatgggtt<br>
7  ; catttgatca caataaatgc attttatgaa atggtgagaa ttttgttcac tcattagtga<br>
7 I gacaaacgtc tcaatggtta tttatatggc atgcatatag tgatatgtgg t<br>
t X. AMPLE 13: CF Mutation Analysis - On-Bead Probe Elongation with Model System<br>
] 13 provides an overview of detection of CF gene mutation R117H. The target was n nlified by PCR as described in Example 11. Two 17-base probes variable at their 3' ends \ .;c immobilized on color coded beads. The target nucleic acid sequence was added along v diTAVIRA-labeleddCTP, unlabeled dNTPs and thermostable DNA polymerase.<br><br>
C-i 'inplementary 17-mer oligonucleotide probes variable at the 3' end were were synthesized by a commercial vendor (Synthegen TX) to contain 5' biotin attached by way of a 12-C spacer (1 niin-TEG) and were purified by reverse phase HPLC. Probes were immobilized on color ei :oded beads. Probes were attached to color- encoded beads. A synthetic 48-mer ohuonucleotide also was provided to contain either A,T,C or G at a designated variable site, c( i1 responding to a cystic fibrosis gene mutation at exon 4 (Rl 17H).<br>
1 i M of synthetic target was added to an annealing mixture containing 10 mM Tris-HCL (pH 7 •'•) ImM EDTA, 0.2 M NaCl, 0.1% Triton X-100. The annealing mixture was placed in c
E V AMPLE 14: CF Mutation Analysis - PCR with Bead-Tagged Primers and Integrated<br>
T&gt; 'tection<br>
T us example illustrates probe elongation on the surface of beads in suspension , followed by a; emhly of and immobilization of beads on the surface of a chip for image analysis. 0: it'onucleotides corresponding to CFTR gene mutation Rl 17H were designed with variable 3' eih.ls (Fig. 14) and were synthesized to contain a 5' biotin-TEG with a 12 C spacer (Synthegen, Ti-xas). The probes were attached to blue stained beads as follows: 2 (oM of probe were added to .1 head solution in IX TE (100 mM Tris-HCl, 10 mM EDTA), 500 mM NaCl and reacted for<br><br>
4r niin at room temperature. Beads were washed with IX TE, 150 mM of NaCl for 3X, and su ponded in 50 u.1 of the same solution. One u,l of each type of bead was added to PCR mix containing IX buffer (100 mM Tris-HCl, pH. 9.0,1.5 mM MgCl2, 500 mM KC1), 40 jiM Cy5-lal clod dCTP (Amersham Pharmacia Biotech NJ), and 80 uM of the other three types of dNTPs, ai ! ? 11 of Taq DNA polymerase (Amersham Pharmacia Biotech NJ). Wild type complementary target (40 ng) was added to the PCR mix just before amplification. Eleven c\ les of PCR amplification were performed in a Perkin Elmer 9600 thermal cycler, each cycle consisting of denaturation for 30 s at 90°C, annealing for 30 s at 55 °C, and elongation at 72 °( for 20 s After amplification, beads were washed four times by centrifugation in IX TE hi for and placed on the chip surface. Images were recorded as in previous Examples and analyzed using the software described in WO 01/98765. The results show specific amplification fi •- beads coupled with the wild-type probe, but no amplification for beads coupled with the m .tant probe. The results are shown in Figure 16b.<br>
Ti is example demonstrates the integration of multiplexed PCR using bead-tagged probes with subsequent assembly of beads on planar surfaces for instant imaging analysis. In a preferred en ihodiment, a microfluidically connected multicompartment device may be used for template amplification as described here. For example, aplurality of compartments capable of permitting temperature cycling and housing, in each compartment, one mPCR reaction producing a subset 01 .ill desired amplicons may be used as follows: (1) perform PCR with different probe pairs in e;. li of four compartments, using encoded bead-tagged primers as described in this Example; (21 following completion of all PCR reactions, pool the amplicon-displaying beads; (3) assemble random array; and (4) record image and analyze the data. Array assembly maybe accomplished b&gt; one of several methods of the prior art including LEAPS.<br>
F. , SIMPLE 15: CF Mutation Analysis - One-Step Annealing and Elongation in Temperature-controlled Reactor<br>
G
 <br>
coi ccnti ation was determined by spectrophotometric analysis. Single or pooled PCR products (2i ng each) were added to an annealing mixture containing 10 mM Tris-HCL (pH 7.4) ImM Eli I A, 0.2 M NaCl, 0.1% Triton X-100. The annealing mixture was mixed with elongation mi Hire containing 3 U of Thermo Sequenase (Amersham Pharmacia Biotech, NJ), IX enzyme bu :cr with either fluorescein-labeled or TAMRA-kbeled deoxynucleotide (dNTP) analogs (NEN Li Sciences) and 1-10 umole of each type of unlabeled dNTP and placed in contact with an aii ' of oligonucleotide probes displayed on a color-encoded array. Oligonucleotides were dt itmed and synthesized as in previous Examples. The annealing- and elongation reactions were<br>
a! &gt;',vcd to proceed in a temperature controlled cycler. The temperature steps were as follows: thi c minutes each at 65 °C, 60 °C, 55 °C, 50 °C and 45 °C, with a ramp between temperatures<br>
o less than 30 seconds. The bead array was then washed with dsH^O for 5 to 15 min. and an ini !-c containing the fluorescence signal from each bead within the array was recorded using a i uorescence microscope equipped with a CCD camera. Images were analyzed to determine th identity of each of the elongated probes. Typical results are shown in Fig. 17.<br>
EXAMPLE 16: Pooling of Covering Probes<br>
T
W Id-type probe sequence:<br>
Oligo 1:	"G" at position 20, "C" at 10, and "T" at 15.<br>
Oligo 2:	"G" at position 20, "C" at 10, and "G" at 15.<br>
Oligo 3:	"G" at position 20, "A" at 10, and "T" at 15.<br>
Oligo 4:	"G" at position 20, "A" at 10, and "G" at 15.<br>
M lant Probe Sequence:<br>
Oligo 1:	"T" at position 20. "C" at 10, and "T" at 15.<br>
Oligo 2:	"T" at position 20, "C" at 10, and "G" at 15.<br><br>
Oligo 3:	"T" at position 20, "A" at 10, and "T" at 15.<br>
Oligo 4:	"T" at position 20, "A" at 10, and "G" at 15.<br>
All of the probes were pooled and attached to a single type of color-coded bead using protocols of previous Examples..  When single-stranded target is added to these beads displaying pooled pi  bus, one of the probes will yield elongation product as long as it is perfectly aligned with tli   designated polymorphism.<br>
EXAMPLE  17:     Designated   Polymorphisms  in  Heterozygous   and  Homozygous Configurations:<br>
Ti distinguish between heterozygous and homozygous configurations, the design of the previous E* ample is augmented to contain a second set of probes to permit the identification of the C/A dc -iiinaled polymorphism   aligned with the probes'    3'ends, and to permit calling of he erozygous versus homozygous mutations.<br>
A, in the previous example, two non-designated polymorphic sites are anticipated at positions K' (OA) and 15 (T/G). A summary of the design follows:<br>
Oligo 1:	"C" at position 20, "C" at 10, and 'T" at 15.<br>
Oligo 2:	"C" at position 20, "C" at 10, and "G" at 15.<br>
Oligo 3:	"C" at position 20, "A" at 10, and "T" at 15.<br>
Oligo 4:	"C" at position 20, "A" at 10, and "G" at 15.<br>
Set --2:<br>
Oligo 5:	"A" at position 20, "C" at 10, and "T" at 15.<br>
Oligo 6:	"A" at position 20, "C" at 10, and "G" at 15.<br>
Oligo 7:	"A" at position 20, "A" at 10, and"T" at 15.<br>
Oligo 8:	"A" at position 20, "A" at 10, and "G" at 15.<br><br>
(	i.iionuclcotides from set ft] are pooled and attached to a single type of color (e.g. green) coded<br>
b	.i.i   using protocols of previous Examples. Oligonuclcotides from set # 2 were pooled and<br>
;r	; idled to a scond type of color (e.g. orange) coded bead using protocols of previous Examples.<br>
H	ads were pooled and immobilized on the surface of chip as described earlier. Next, target was<br>
ii	induced, and   on-chip reactions   performed as described in previous Examples. If probes<br>
.	i izreen beads only are elongated, the individual has a normal (or wild-type) allele. If probes<br>
0	orange beads only  are elongated, the individual is homozygous for the mutation. I If probes<br>
o- • ;j.rcen as well as ongan beads are elongated, the individual is heterozygous for that allele. This<br>
(1	sivn is useful for the identification of known and unknown mutations.<br>
1	\ VYU'LE 18 - Confirmatory Sequencing ("Resequencing")<br>
1	i design of the present invention can be used for re-sequencing of a specific area. This test can<br>
b	'.;si:d when on-chip probe elongation reaction requires confirmation, as in the case of reflex<br>
t&gt;	i; lor 1506V, I507V, F508C and 7T in the CF mutation panel. The sequence in question, here<br>
2	bases to 30 bases in length , is sequenced on-chip by multiplexed interrogation of all variable<br>
si os. This is accomplished by designing specific probes for ambiguous locations, and by probe<br>
-i	loling as described in Examples 16 and 17.<br>
I	WV1I'LL19: Elongation with One Labeled dNTP and Three Unlabeled dNTPs.<br>
H   way of incorporating at least one labeled dNTP, all elongation products are detected in realtime and identified by their association with coded solid phase carriers. Using assay conditions<br>
(1	-.Tibcd in connection with Examples 6 and 7, tetramethylrhodamine-6-dCTP and unlabeled<br>
i	P d IT P and dGTP were provided in an elongation reaction to produce a fluorescently<br>
1.	•
1,	-• L\i dUTP and Cy5-labeled dUfP) maybe used. Similarly, any other labeled dNTP can be<br>
II	.\\. The length of the elongation product depends on the amount of labeled dXTP tolerated by<br>
tl	UNA polymerase. Available enzymes generally exhibit a higher tolerance for strand-<br>
it	'ditving moieties such as biolin and digoxigenin which may then be reacted in a second step<br>
^	i: hilvk-d avidins or antibodies to accomplish indirect labeling of elongation procucts. When<br><br>
u.Mtig these small molecules, elongation products measuring several hundred bases in length are<br>
produced.<br>
EXAMPLE 20 : Extension with One Labeled ddNTP, Three Unlabeled dNTPs.<br>
TAMRA-labeled ddCTP may be incorporated to terminate the extension reaction, as illustrated in Pig. 19. On-chip reactions using TAMRA-labeled ddCTP were performed as described in E amples 6 and 7. In a reaction mixture containing TAMRA-ddCTP and unlabeled dTTP, dATP a? I dGTP, following annealing of the target to the matching probe, the extension reaction tei initiates when it completes the incorporation of the first ddCTP. This may occur with the very first base incorporated, producing a single base extension product, or it may occur after a number ol unlabeled dNTPs have been incorporated.<br>
E \ AMPLE 21 : Elongation with Four Unlabeled dNTPs, Detection by Hybridization of Lubeled Probe<br>
Probes are elongated using a full set of four types of unlabeled dNTPs, producing, under these "n itive" conditions for the polymerase, elongation products measuring several hundred bases in length, limited only by the length of the annealed template and on-chip reaction conditions. Tl c elongation product is detected, following denaturation at high temperature, in a second step by hybridization with a labeled oligonucleotide probe whose sequence is designed to be complementary to a portion of the elongation product This process is illustrated in Fig. 20.<br>
EXAMPLE 22: Elongation with Four Unlabeled dNTPs, Detection via Labeled Template<br>
A with standard protocols in routine use in multiplexed hybridization assays, the DNA target to ie analyzed can itself be labeled in the course of PCR by incorporation of labeled probes. Under conditions such as those described in Examples 6 and 7, a labeled target is annealed to probes. Matching probes are elongated using unlabeled dNTPs. Following completion of the eh 'iigation reaction, detection is performed by setting the temperature (Tdet) to a value above the milling temperature (Too,,.,^,,) of the complex formed by target and non-matched probe , but<br><br>
below the melting temperature (Tmatch) of the complex formed by target and matched, and hence elongated , probe. The latter complex, displaying a long stretch of duplex region,will be si; nificantly more stable than the former so that (Tnon-match) col is directly compatible with methods of polymorphism analysis by hybridization of sc i ence-specific oligonucleotides.<br>
EXAMPLE 23 : Real-time On-chip Signal Amplification<br>
A ..iiulaid temperature control apparatus used with a planar geometry such as that illustrated in Fi 12 permits the application of programmed temperature profiles to a multiplexed extension 01 -&gt;SPs. Under conditions of Examples 6 and 7, a given template mediates the elongation of one probe in each of multiple repeated "denature-anneal-extend" cycles. In the first cycle, a tai uet molecule binds to a probe and the probe is elongated or extended, hi the next cycle, the t;i: -:et molecule disassociates from the first probe in the "denature" phase (at a typical ter.iperature of 95 °C), then anneals with another probe molecule in the "anneal" phase (at a typical temperature of 55 °C) and mediates the extension of the probe in the "extend" phase (at a upical temperature of 72 °Q. In N cycles, each template mediates the extension of N probes, a i rotocol corresponding to linear amplification (Fig. 30). In a preferred embodiment of this in cation, in which planar arrays of encoded beads are used to display probes in a multiplexed ex ension reaction, a series of temperature cycles is applied to the reaction mixture contained bi-1 \veen two planar, parallel substrates. One substrate permits direct optical access and direct ii; '.uing of an entire array of encoded beads. The preferred embodiment provides for real-time<br><br>
atuplifieation by permitting images of the entire bead array to be recorded instantly at the completion of each cycle.<br>
Gcnomic, mitochondrial or other enriched DNA can be used for direct detection using on-chip lii icar amplification without sequence specific amplification. This is possible when an amount 01 DNA sufficient for detection is provided in the sample.. In the bead array format, if 104 fjnorophores are required for detection of signal from each bead, 30 cycles of linear amplification will reduce the requisite number to ~300. Assuming the use of 100 beads of the requisite type within the array , the requisite total number of fluorophores would be ~105, a number typically available in clinical samples. For example, typical PCR reactions for clinical m.ilecular typing of HLA are performed with 0.1 to 1 ng of genomic DNA. One jig of human gniomic DNA corresponds to approximately 10"18 moles, thus, 6xl05 copies of the gene of in i crest This small amount of sample required by the miniaturized bead array platform and on-cliip amplification makes the direct use of pre-PCR samples possible. This not only simplifies siimple preparation but, more importantly, eliminates the complexity of multiplexed PCR, frequently a rate limiting step in the development of multiplexed genetic analysis.<br>
EXAMPLE 24: Construction of a Probe Library for Designated and Unselected Polymorphisms for CF Mutation Analysis<br>
T.I increase the specificity of elongation probes and avoid false positives, elongation probes were designed to accommodate all known polymorphisms present in a target sequence. In addition, PCR primers were designed taking into consideration designated and non-designated polymorphisms.<br>
The G/C mutation at position 1172. of R347P on Exon 7 within the CFTR gene, one of 25<br>
m uiations within the standard population carrier screening panel for cystic fibrosis, was selected<br>
a.
n: nation panel for general population carrier screening (http://www.faseb.org/genetics/acmg).<br>
A polymorphism G/T/A	at the same site has been reported<br>
(l::ip://\vww.genet.sickkids.on.ca/cftr). and in addition, non-designated polymorphisms have<br><br>
beui reported at positions 1175, 1178,1186,1187 and 1189. All of these polymorphisms can interfere with desired probe elongation.<br>
Th
5'	3'<br>
No rrnal Target Sequence for Elongation:       Gca Tgg Cgg tea ctC GgC a<br>
Degenerate Elongation Probe Set:	Ngt Ycc Ycc agt gaY RcY t<br>
3'	5'<br>
wl ie N = a, c, g or t; R (puRines) = a or g and Y (pYrimidines) = c or t, implying a degeneracy<br>
of i 28 for the set.<br>
Punier Pooling for Mutation Analysis - The principal objective in the construction of a degenerate set is to provide at least one probe sequence to match the target sequence sufficiently closely to ensure probe annealing and elongation. While this is always attainable in principle by providing the entire set of possible probe sequences associated with the designated po. vinorphism, as in the preferred mode of constructing covering sets, the degree of degeneracy of hat set, 128 in the example, would lead to a corresponding reduction in assay signal intensity by i wo orders of magnitude if all probes were to be placed onto a single bead type for complete probe pooling. Splitting pools would improve the situation by distributing the probe set over mi; Itiple bead types, but only at the expense of increasing array complexity.<br>
Fii st. the probe pool was split into a minimum of two or more pools, each pool providing the complementary composition, at probe position M (i.e., the probe's 3' terminus), for each of the p( sihle compositions of the designated polymorphic site. In the example, four such pools are r&amp; lined for a positive identification of the designated target composition. Next, non-designated polymorphic sites were examined successively in the order of distance from the designated site.<br><br>
A i long these, positions within the TEI region are of special importance to ensure elongation. Tl at is, each pool is constructed to contain all possible probe compositions for those non-dt ,ignated sites that fall within the TEI region.. Finally, as with the construction of degenerate pi•' &gt;bes for cloning and sequencing of variable genes, the degeneracy of the set is minimized by pi cing neutral bases such as inosine into those probe positions which are located outside the Tl I region provided these are known never to be juxtaposed to G in the target In the example, non-designated polymorphisms in probe positions M-16 and M-l 8 qualify. That is, the minimal degeneracy of each of the four pools would increase to four, producing a corresponding reduction in ;ignal intensity. As an empirical guideline, signal reduction preferably will be limited to a f;: -nr of eight.<br>
In total, four pools, each uniquely assigned to one bead type and containing eight degenerate pi &gt;he sequences, will cover the target sequence. These sequences are analogous to those shown bi ioxv for pools variable at M:<br><br>
Probe pool for CF mutation R347P<br>
R347P    Cgt	Ace	Gcc	agt	gaG	GgC<br>
3(                                                         5'<br>
POOL 1 Cgt	Ace	Gcc	agt	gaG	Igl<br>
Cgt	Ace	Gcc	agt	gaC	Igl<br>
Cgt	Ace	Ccc	agt	gaG	Igl<br>
Cgt	Ace	Ccc	agt	gaC	Igl<br>
Cgt	Tec	Gcc	agt	gaG	Igl<br>
Cgt	Tec	Gcc	agt	gaC	Igl<br>
Cgt	Tec	Ccc	agt	gaG	Igl<br>
Cgt	Tec	Ccc	agt	gaC	Igl<br>
POOL 2 Ggt	Ace	Gcc	agt	gaG	Igl<br>
Ggt	Ace	Gcc	agt	gaC	Igl<br>
Ggt	Ace	Ccc	agt	gaG	Igl<br>
Ggt	Ace	Ccc	agt	gaC	Igl<br>
Ggt	Tec	Gcc	agt	gaG	Igl<br>
Ggt	Tec	Gcc	agt	gaC	Igl<br>
Ggt	Tec	Ccc	agt	gaG	Igl<br>
Ggt	Tec	Ccc	agt	gaC	Igl<br>
POOL 3 Agt	Ace	Gcc	agt	gaG	Igl<br>
Agt	Ace	Gcc	agt	gaC	Igl<br>
Agt	Ace	Ccc	agt	gaG	Igl<br>
Agt	Ace	Ccc	agt	gaC	Igl<br>
Agt	Tec	Gcc	agt	gaG	Igl<br>
Agt	Tec	Gcc	agt	gaC	Igl<br>
Agt	Tec	Ccc	agt	gaG	Igl<br>
Agt	Tec	Ccc	agt	gaC	Igl<br>
POOL 4 Tgt	Ace	Gcc	agt	gaG	Igl<br>
Tgt	Ace	Gcc	agt	gaC	Igl<br>
Tgt	Ace	Ccc	agt	gaG	Igl<br><br>
Tgt     Ace    Ccc    agt    gaC    Igl<br>
Tgt	Tec	Gcc	agt	gaG	Igl<br>
Tgt	Tec	Gcc	agt	gaC	Igl<br>
Tgt	Tec	Ccc	agt	gaG	Igl<br>
Tgt	Tec	Ccc	agt	gaC	Igl<br>
Li general, the type of non-designated polymorphisms on the antisense strand may differ from th. 11 on the sense strand, and it may then be advantageous to construct degenerate probe sets for tin; antisense strand. As with the construction of degenerate elongation probes, degenerate hybridization probe sets may be constructed by analogous rules to minimize the degeneracy.<br>
E SAMPLE 25: "Single Tube" CF Mutation Analysis by eMAP<br>
Tl is example is concerned with methods and compositions for performing an eMAP assay, \\ herein the annealing and elongation steps occur in the reactor. This embodiment is useful because it obviates the need for sample transfer between reactors as well as purification or ex traction procedures, thus simplifying the assay and reducing the possibility of error. A non-lii liting exemplary protocol follows.<br>
Genomic DNA extracted from several patients was amplified with corresponding primers in a multiplex PCR (mPCR) reaction. The PCR conditions and reagent compositions were as<br>
follows.<br>
PRIMER DESIGN: Sense primers were synthesized without any modification and antisense pi•; mers with "Phosphate" at the 5' end. Multiplex PCR was performed in two groups. Group one amplification includes exon 5,7, 9,12, 13, 14B, 16, 18 and 19. Amplifications for group 2 Deludes primers for exon 3, 4, 10, 11, 20, 21 and intron 19. The 5' phosphate group modification on exon 5, 7, and 11 was included on forward primer to use antisense target for pi'»be elongation. While sense target was used for all other amplicons by placing phosphate gi nip on reverse primer.<br>
/' A' Master mix composition<br><br>
F
»nts	Volume ( ul)<br>
K<x pcr buffer></x>
2
dr I Ps (2.5 mM)	2.0<br>
P i . m er mi x      (Multiplex 1 Ox)	1.5<br>
T
d(:H2O	1.5<br>
DVA ____ 3X)<br>
Total	10<br>
PCR Cycling<br>
9- °C   5 min, 94 °C 10 sec., 60 °C  10 sec., 72 °C 40 sec 7.   (    5 min.,   Number of cycles:   28-35<br>
The reaction volume can be adjusted according to experimental need. Amplifications are pi i formed using a Perkin Elmer 9600 thermal cycler. Optimal primer concentrations were dt ermined for each primer pair. Following amplifications, 5 ul of the product was removed for g&gt; electrophoresis. Single stranded DNA targets were generated as follows: Two microliters ol exonuclease was added to 5 ul of PCR product, incubated at 37 °C for 15 minutes and enzyme \\.is denatured at 80 °C for 15 minutes. After denaturation, 1 ul of 10X exonuclease buffer was aiiiled with 1 ul of X exonuclease (5 U/ul) and incubated at 37 °C for 20 minutes and the reaction v. s stopped by heating at 75 °C for 10 minutes.<br>
0	V CHIP ELONGATION<br>
Wild type and mutant probes for 26 CF mutations were coupled on the bead surface ami assembled on the chip array. The probes were also divided into two groups. A third<br>
gi &gt;i:p was assembled for reflex test including 5T/7T/9T polymorphisms.<br>
fci.mgation Group 1, total 31 groups on the chip surface.<br>
BI ad cluster #	Mutation<br>
1	G85E-WT<br>
2	G85E-M<br>
3	621+1G&gt;T-WT<br>
4	621+1GXT-M<br>
5	R117H-WT<br><i r117h-m></i>
7	P Actin<br><br>
8	I148T-WT<br>
9	I148T-M<br>
K	508-WT<br>
1J	F508<br>
II	1507<br>
1	G542X-WT<br>
^	G542X-M<br>
15	G551D-WT<br>
\
T	R553X-WT<br>
1&gt;	R553X-M<br>
1'	BIOTIN<br>
2(	1717-1G&gt;A-WT<br>
21	1717-1G&gt;A-M<br>
2:	R560T-WT<br>
2?	R560T-M<br>
2-	3849+10kbT-WT<br>
2-	3849+10kbT-M<br>
2'	W1282X-WT<br>
2	W1282X-M<br>
2:-	N1303K-WT<br>
2
3
mgation Group 2, total 28 groups on the chip surface.<br><br>
(Table Remove)<br>
19	3120+1OA-WT<br>
2(&gt;	3120+1G&gt;A-WT<br>
21	R1162X-WT<br>
22	R1162X-M<br>
23	3659delC-WT<br>
24	3659delC-M<br>
25	D1152-WT<br>
2(	D1152-M<br>
2'	OLIGO-C<br>
mi'CR group 2:<br>
El'ingation Group 3, total 6 groups<br>
Cl ister.#	Mutation<br>
1	(3 Actin.<br>
I	Oligo C<br>
5T<br>
3	7T<br>
4	9T<br>
5	Biotin<br>
Elongation reaction buffer has been optimized for use in uniplex and/or multiplex target el
EXAMPLE 26: CF Mutation Analysis - Single tube single chip-One Step Elongation.<br>
Probes for 26 CF mutations and controls were coupled on the surface of 51 types of b ,; s. Probe coupled beads were assembled on the surface of a single chip. Genomic DNA was e\ racted from several patients and was amplified with corresponding primers in a multiplexed<br>
P( :R (mPCR) reaction, as described in the previous example. Following amplification, single sti anded DNA products were produced using X exonuclease. Single or pooled PCR products ( Siig) were added to a reaction mixture containing reaction buffer, deoxynucleotide (dNTP) analogs (NEN Life Sciences), each type of unlabeled dNTP, and DNA polymerase (Amersham Pharmacia Biotech, NJ). The annealing/elongation reaction was allowed to proceed in a tci i iperature controlled cycler. The temperature steps were as follows: 20 minutes at 53°C, and 3 minutes at 60 °C. The bead array was then washed with dsHaO containing 0.01% SDS for 5 to 15 minutes. An image containing the fluorescent signal form each bead within the array was recorded using a fluorescence microscope and a CCD camera. Images were analyzed to determine th identity of each of the elongated probes. The composition of bead chip containing 26 CF mutations is provided below.<br>
Elongation Group 4, total 51 groups<br>
Oi.ster-	Mutation<br><br>
2N	2184del<br>
(Table Remove)A-M<br>
2'	2789+5G&gt;A-WT<br>
3<i>A-M<br>
31	3120+1G-WT<br>
3	3120+1G-WT<br>
3	D1152-WT<br>
3	D1152-M<br>
3	R1162-WT<br>
3
3	OLIGO-C<br>
3.v	W1282X-WT<br>
3"	W1282-M<br>
4
4	N1303-M<br>
4,	R334-WT<br>
4.	R334-M<br>
4:	1078delT-WT<br>
4:	1078delT-M<br>
4'	3849-10kb-WT<br>
4	3849-10kb-M<br>
4'	1717-1G&gt;A-WT<br>
5A-WT<br>
5!	Biotin<br>
L\ AMPLE 27: Identification of Three or More Base Deletions and/or Insertions by e.MAP:<br>
Elongation was used to analyze mutations with more than 3 base deletions or insertions.<br>
Probes were designed by placing mutant bases 3-5 base before 3' end. The wild type probes<br>
\vi:re designed to either include or exclude mutant bases (terminating before mutations). The<br>
following	is an example of mutations caused by a deletion of ATCTC and/or insertion of<br>
A<igta. the probe designs are as follows:></igta.>
1.	WT1		ATCTCgca<br>
2.	WT2	<br>
3.	Ml		gca (deletion only)<br>
4         M2- 	AGGTAgca (deletion and insertion)<br>
Wild type probes were either coupled on the surface of differentially encoded beads or pooled as described in this invention. Probes for mutation 1 (Ml: deletion) and 2 (M2: insertion) were coupled on different beads. Both wild type probes provide similar<br><br>
information, while the mutant probes can show the type of mutation identified in a specific<br>
sample.<br>
EXAMPLE 28. Hairpin Probes<br>
In certain embodiments of this invention, bead-displayed priming probes form hairpin srucnires. A hairpin structure may include a sequence fragment at the 5' end that is complementary to the TEI region and the DA sequence, as shown in Fig. 23. During a competitive hybridization reaction, the hairpin structure opens whenever the DA region preferentially hybridizes with the target sequence. Under this condition, the TEI region will align \\ ith the designated polymorphic site and the elongation reaction will occur. The competitive n. nne of the reaction can be used to control tolerance level of probes.<br>
EXAMPLE 29 Analysis of Cystic Fibrosis and Ashkenazi Jewish Disease Mutations by Multiplexed Elongation of Allele Specific Oligonucleotides Displayed on Custom Bead Arrays<br>
A novel assay for the high throughput multiplexed analysis of mutations has been evaluated for ACMG+ panel of Cystic Fibrosis mutations. In addition, an Ashkenazi Jewish disease panel also has been developed to detect common mutations known to cause Tay-Sachs, Cmavan, Gaudier, Niemann-Pick, Bloom Syndrome, Fancomi Anemia, Familial Dysautonomia, aril mucolipodosis IV.<br>
In elongated-mediated multiplexed analysis of polymorphisms (eMAP), allele specific ol igonucleotides (ASO) containing variable 3' terminal sequences are attached to color-encoded bi ads which are in turn arrayed on silicon chips. Elongation products for normal and mutant so|iiences are simultaneously detected by instant imaging of fluorescence signals from the entire ai '
In this example, several hundred clinical patient samples were used to evaluate ACMG ('I bead chips. As shown in Fig. 24, the assay correctly scored all of the mutations identified by sl.mdard DNA analysis.<br>
In summary, a multiplexed elongation assay comprising customized beads was used to<br><br>
study mutations corresponding to ACMG+ and Ashkenazi disease panels. The customized beads can be used for DNA and protein analysis. The use of these customized beads are advantageous foi several reasons including (1) instant imaging - the turnaround time for the assay is within two hen us (2) automated image acquisition and analysis (3) miniaturization, which means low reagent consumption, and (4) the beadchips are synthesized using wafer technology, so that millions of chips can be mass-produced, if desired.<br><br><br><br><br>
WE CLAIM:<br>
1.	A method for identity testing by probe elongation-mediated analysis of variable<br>
sites in the HLA gene in a genome:<br>
by providing sets of different cognate probes,charactrerized in that are the cognate probes in a set capable of annealing to one or more amplicons, said amplicons generated by amplifying regions of the genome using asymmetric polymerase chain reaction, and said cognate probes further capable of being elongated with a detectably labeled nucleotide if, following annealing, the probe's interrogation site is complementary to the aligned nucleotide in the amplicon, wherein said probes are designed such that the aligned nucleotide is complementary to a nucleotide at a variable site;<br>
and designating for each set of amplicons, one strand (either sense or antisense) for the probe elongation-mediated analysis of the variable sites, depending on which strand has a greater degree of complementarity to its cognate probe in the terminal elongation initiation region of the cognate probe.<br>
2.	The method as claimed in claim 1 wherein the probe's interrogation site is at the 3' terminus of the probe.<br>
3.	The method as claimed in claim 1 wherein the different sets of cognate probes are associated with differently encoded microparticles,such as herein described.<br>
4.	The method as claimed in claim 1 wherein the probes in a set includes four different types of probes, each with a different interrogation site nucleotide.<br>
5.	The method as claimed in claim 1 wherein encoding of probes is by associating probes with different sequences to carriers, including beads, having different optical signatures.<br>
6.	The method as claimed in claim 6 wherein the encoding is with color.<br><br>
7. A method for identity testing by probe elongation-mediated analysis of variable sites in the HLA gene in a genome substantially as herein described with reference to the foregoing description, examples and the accompanying drawing.<br><br><br></i></x></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEyOTEtZGVsbnAtMjAwNC1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">01291-delnp-2004-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEyOTEtZGVsbnAtMjAwNC1hc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">01291-delnp-2004-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEyOTEtREVMTlAtMjAwNC1DbGFpbXMtKDI4LTA3LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">01291-DELNP-2004-Claims-(28-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEyOTEtZGVsbnAtMjAwNC1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">01291-delnp-2004-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEyOTEtZGVsbnAtMjAwNC1jb3JyZXNwb25kZW5jZS1vdGhlcnMtKDE1LTAyLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">01291-delnp-2004-correspondence-others-(15-02-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEyOTEtREVMTlAtMjAwNC1Db3JyZXNwb25kZW5jZS1PdGhlcnMtKDI4LTA3LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">01291-DELNP-2004-Correspondence-Others-(28-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEyOTEtZGVsbnAtMjAwNC1jb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">01291-delnp-2004-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEyOTEtZGVsbnAtMjAwNC1kZXNjcmlwdGlvbiAoY29tcGxldGUpLTIwLTA4LTIwMDgucGRm" target="_blank" style="word-wrap:break-word;">01291-delnp-2004-description (complete)-20-08-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEyOTEtZGVsbnAtMjAwNC1kZXNjcmlwdGlvbiAoY29tcGxldGUpLTI4LTA3LTIwMDgucGRm" target="_blank" style="word-wrap:break-word;">01291-delnp-2004-description (complete)-28-07-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEyOTEtZGVsbnAtMjAwNC1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">01291-delnp-2004-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEyOTEtZGVsbnAtMjAwNC1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">01291-delnp-2004-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEyOTEtZGVsbnAtMjAwNC1mb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">01291-delnp-2004-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEyOTEtZGVsbnAtMjAwNC1mb3JtLTE4LSgxNS0wMi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">01291-delnp-2004-form-18-(15-02-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEyOTEtZGVsbnAtMjAwNC1mb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">01291-delnp-2004-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEyOTEtZGVsbnAtMjAwNC1mb3JtLTMtKDE1LTAyLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">01291-delnp-2004-form-3-(15-02-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEyOTEtREVMTlAtMjAwNC1Gb3JtLTMtKDI4LTA3LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">01291-DELNP-2004-Form-3-(28-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEyOTEtZGVsbnAtMjAwNC1mb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">01291-delnp-2004-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEyOTEtZGVsbnAtMjAwNC1mb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">01291-delnp-2004-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEyOTEtZGVsbnAtMjAwNC1ncGEucGRm" target="_blank" style="word-wrap:break-word;">01291-delnp-2004-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEyOTEtZGVsbnAtMjAwNC1wY3QtMjIwLnBkZg==" target="_blank" style="word-wrap:break-word;">01291-delnp-2004-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEyOTEtZGVsbnAtMjAwNC1wY3QtNDA5LnBkZg==" target="_blank" style="word-wrap:break-word;">01291-delnp-2004-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEyOTEtZGVsbnAtMjAwNC1wY3QtNDE2LnBkZg==" target="_blank" style="word-wrap:break-word;">01291-delnp-2004-pct-416.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEyOTEtZGVsbnAtMjAwNC1wY3QtNDI4LnBkZg==" target="_blank" style="word-wrap:break-word;">01291-delnp-2004-pct-428.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEyOTEtZGVsbnAtMjAwNC1wY3QtcmVxdWVzdCBmb3JtLnBkZg==" target="_blank" style="word-wrap:break-word;">01291-delnp-2004-pct-request form.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEyOTEtZGVsbnAtMjAwNC1wY3Qtc2VhcmNoIHJlcG9ydC5wZGY=" target="_blank" style="word-wrap:break-word;">01291-delnp-2004-pct-search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI5MS1ERUxOUC0yMDA0LUFic3RyYWN0ICgxOS0xMi0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">1291-DELNP-2004-Abstract (19-12-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI5MS1ERUxOUC0yMDA0LUFic3RyYWN0LSgyMC0wOC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1291-DELNP-2004-Abstract-(20-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI5MS1ERUxOUC0yMDA0LUNsYWltcyAoMTktMTItMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">1291-DELNP-2004-Claims (19-12-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI5MS1ERUxOUC0yMDA0LUNsYWltcy0oMjAtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1291-DELNP-2004-Claims-(20-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI5MS1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlLU90aGVycyAoMTktMTItMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">1291-DELNP-2004-Correspondence-Others (19-12-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI5MS1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTktMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1291-DELNP-2004-Correspondence-Others-(19-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI5MS1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjAtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1291-DELNP-2004-Correspondence-Others-(20-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI5MS1ERUxOUC0yMDA0LURyYXdpbmdzICgxOS0xMi0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">1291-DELNP-2004-Drawings (19-12-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI5MS1ERUxOUC0yMDA0LUZvcm0tMS0oMjAtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1291-DELNP-2004-Form-1-(20-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI5MS1kZWxucC0yMDA0LWZvcm0tMTMtKDE5LTA4LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">1291-delnp-2004-form-13-(19-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI5MS1ERUxOUC0yMDA0LUZvcm0tMi0oMjAtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1291-DELNP-2004-Form-2-(20-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI5MS1ERUxOUC0yMDA0LUdQQS0oMjAtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1291-DELNP-2004-GPA-(20-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI5MS1ERUxOUC0yMDA0LVBldGl0aW9uLTEzNy0oMTktMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1291-DELNP-2004-Petition-137-(19-08-2008).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="222987-a-pyrimidine-compound-of-formula-i.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="222989-process-for-the-preparation-of-aminated-alkyloxfuranone-compounds.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>222988</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>01291/DELNP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>44/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>31-Oct-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>29-Aug-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>13-May-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>BIOARRAY SOLUTIONS, LTD.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>35 TECHNOLOGY DRIVE, SUITE 100, WARREN, NJ 07059, U.S.A.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>LI, ALICE, XIANG</td>
											<td>5 ESTATE DRIVE, ITHACA, NY 14850,USA.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>HASHMI, GHAZALA</td>
											<td>53 AGOSTINA DRIVE, HOLMDEL, NJ 07733,USA.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>SEUL, MICHAEL</td>
											<td>84 PLEASANT AVENUE, FANWOOD, NJ 07023, USA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>G01N</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US02/33012</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-10-15</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/329,428</td>
									<td>2001-10-15</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/364,416</td>
									<td>2002-03-14</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>60/329,427</td>
									<td>2001-10-15</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>4</td>
									<td>60/329,619</td>
									<td>2001-10-15</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>5</td>
									<td>60/329,620</td>
									<td>2001-10-15</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/222988-a-method-for-identity-testing-by-probe-elongation-mediated-analysis-of-variable-sites-in-the-hla-gene-in-a-genome by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:08:27 GMT -->
</html>
