<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/223176-a-novel-negative-sense-single-stranded-rna-virus by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:37:06 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 223176:A NOVEL NEGATIVE-SENSE SINGLE STRANDED RNA VIRUS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A NOVEL NEGATIVE-SENSE SINGLE STRANDED RNA VIRUS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to an isolated essentially mammalian negative-sense single stranded RNA virus (MPV) belonging to the sub-family Pneumovirinae of the family Paramyxoviridae and identifiable as phylogenetically corresponding to the genus Metapneumovirus.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
A virus causing respiratory tract illness in susceptible mammals.<br>
The invention relates to the field of virology.<br>
In the past decades several etiological agents of mammalian disease, in particular of respiratory tract illnesses (RTI), in particular of humans, have been identified7. Classical etiological agents of RTI with mammals are respiratory syncytial viruses belonging to the genus Pneumovirus found with humans (hRSV) and ruminants such as cattle or sheep (bRSV and/or oRSV). In human RSV differences in reciprocal cross neutralization assays, reactivity of the G proteins in immunological assays and nucleotide sequences of the G gene are used to define 2 hRSV antigenic subgroups. Within the subgroups the aa sequences show 94 % (subgroup A) or 98% (subgroup B) identity, while only 53% aa sequence identity is found between the subgroups. Additional variability is observed within subgroups based on monoclonal antibodies, RT-PCR assays and RNAse protection assays. Viruses from both subgroups have a worldwide distribution and may occur dixring a single season. Infection may occur in presence of pre-existing immunity and the antigenic variation is not strictly required to allow re-infection. See for example Suliender, W.M., Respiratory Syncytial Virus Genetic and Antigenic Diversity, Clinical Microbiology Reviews, 2000. 13(1): p. 1-15; Collins, P.L., Mcintosh, K. and Chanock, R.M., Respiratory syncytial virus. Fields virology, ed. B.N. Knipe, Howley, P.M. 1996, Philadelphia: Uppencott-raven. 1313-1351; Johnson, P.R., et al., The G glycoprotein of human respiratory syncytial viruses of subgroups A and B: extensive sequence divergence between antigenically related proteins, Proc Natl Acad Sci USA, 1987. 84(16): p. 5625-9; Collins, P.L., The molecular Biology of Human Respiratory Syncytial Virus (RSV) of the Genus Pneumovirus, in The Paramyxoviruses, D.W. Kingsbury, Editor. 1991, Plenum Press: New York. p. 103-153.<br>
Another classical Pneumovirus is the pneumonia virus of mice (PVM). in<br><br>
general only found with laboratory mice. However, a proportion of the illnesses observed among mammals can still not be attributed to known pathogens.<br><br><br>
The invention provides an isolated single stranded RNA virus (MPV) belonging to the sub-family Pneumovirinae of the family Paramyxoviridae and identifiable as phylogenetically corresponding to the genus Metapneumovirus, Said virus is identifiable as phylogeneticaUy corresponding to the genus Metapneumouirus by determining a nucleic acid sequence of said virus and testing it in phylogenetic analyses, for example wherein maximum likelihood trees are generated using 100 bootstraps and 3 jumbles and finding it to be more closely phylogeneticaUy corresponding to a virus isolate deposited as 1-2614 with CNCM, Paris than it is corresponding to a essentiaUy avian virus isolate of avian pneumovirus (APV) also known as turkey rhinotracheitis virus (TRTV), the aetiological agent of avian rhinotracheitis. For said phylogenetic analyses it is most useful to obtain the nucleic acid sequence of a non-MPV as outgroup to be compared with, a very useful outgroup isolate can be obtained from avian pneumovirus serotype C (APV-C), as is for example demonstrated in figure 5 herein.<br>
Although phylogenetic analyses provides a convenient method of identifying a virus as an MPV several other possibly more straightforward albeit somewhat more course methods for identifying said virus or viral proteins or nucleic acids firom said virus are herein also provided. As a rule of thumb an MPV can be identified by the percentages of a homology of the virus, proteins or nucleic acids to be identified in comparison with isolates, viral proteins, or nucleic acids identified herein by sequence or deposit. It is generaUy known that virus species, especiaUy RNA virus species, often constitute a quasi species wherein a cluster of said viruses displays heterogeneity among its members. Thus it is expected that each isolate may have a somewhat different percentage relationship with one of the various isolates as provided herein.<br>
When one wishes to compare with the deposited virus 1-2614, the invention<br>
provides an isolated essentiaUy mammalian negative-sense single stranded RNA<br>
		*â– <br>
virus (MPV) belonging to the sub-family Pneumovirinae of the fanuly<br>
Param5rxoviridae and identifiable as phylogeneticaUy corresponding to the genus<br>
Metapneumovirus by determining an amino acid sequence of said virus and<br>
determining that said amino acid sequence has a percentage amino acid homology to<br>
a virus isolate deposited as 1-2614 with CNCM, Paris which is essentiaUy higher than<br>
the percentages provided herein for the L protein, the M protein, the N protein, the P<br>
protein, or the F protein, in comparison with APV-C or, likewise, an isolated<br><br>
essentially mammalian negative-sense single stranded RNA virus (MPV) belonging to the sub-family Pneumovirinae of the family Paramyxoviridae is provided as identifiable as phylogenetically corresponding to the genus Metapneumovirus by determining a nucleic acid sequence of said virus and determining that said nucleic acid sequence has a percentage nucleic acid identity to a virus isolate deposited as I-2614 with CNCM, Paris which is essentially higher than the percentages identified herein for the nucleic acids encoding the L protein, the M protein, the N protein, the P protein, or the F protein as identified herein below in comparison with APV-C.<br>
Again as a rule of thumb one may consider an MPV as belonging to one of the two serological groups of MPV as identified herein when the isolates or the viral proteins or nuclear acids of the isolates that need to be identified have percentages homology that fall within the bounds and metes of the percentages of homology identified herein for both separate groups, taking isolates 00-1 or 99-1 as the respective isolates of comparison. However, when the percentages of homology are smaller or there is more need to distinguish the viral isolates from for example APV-C it is better advised to resort to the phylogenetic analyses as identified herein.<br>
Again one should keep in mind that said percentages can vary somewhat when other isolates are selected in the determination of the percentage of homology.<br>
With the provision of this MPV, the invention provides diagnostic means and methods and therapeutic means and methods to be employed in the diagnosis and/or treatment of disease, in particular of respiratory disease, in particular of mammals, more in particular in humans. However, due to the , albeit distant, genetic relationship of the essentially mammalian MPV with the essentially avian APV, in particular with APV-C, the invention also provides means and methods to be employed in the diagnosis and treatment of avian disease. In virology, it is most advisory that diagnosis and/or treatment of a specific viral infection is performed with reagents that are most specific for said specific virus causing said infection. In this case this means that it is preferred that said diagnosis and/or treatment of an MPV infection is performed with reagents that are most specific for MPV. This by no means however excludes the possibility that less specific, but sufficiently cross-reactive reagents are used instead, for example because they are more easily available and sufficiently address the task at hand. Herein it is for example provided to perform virological and/or serological diagnosis of MPV infections in mammals with reagents derived from APV, in particular with reagents derived from APV-C, in<br><br>
the detailed description herein it is for example shown that serological diagnosis of MPV infections in mammals can be achieved by using an ELISA specifically designed to detect APV antibodies in birds. A particular useful test for this purpose is an ELISA test designed for the detection of APV antibodies (e.g in serum or egg yolk), one commercialy available version of which is known as APV-Ab SVANOVIR Â© which is manufactured by SVANOVA Biotech AB, Uppsal Science Park Glunten SE-751 83 Uppsala Sweden. The reverse situation is also the case, herein it is for example provided to perform virological and/or serological diagnosis of APV infections in mammals with reagents derived from MPV, in the detailed description herein it is for example shown that sufficiently trustworthy serological diagnosis of APV infections in birds can be achieved by using an ELISA designed to detect MPV antibodies. Considering that antigens and antibodies have a lock-and-key relationship, detection of the various antigens can be achieved by selecting the appropriate antibody having sufficient cross-reactivity. Of course, for reljdng on such cross-reactivity, it is best to select the reagents (such as antigens or antibodies) under guidance of the amino acid homologies that exist between the various (glyco)proteins of the various viruses, whereby reagents relating to the most homologous proteins will be most useful to be used in tests relying on said cross-reactivity.<br>
For nucleic aciddetection, it is even more straightforward, instead of designing primers or probes based on heterologous nucleic acid sequences of the various viruses and thus that detect differences between the essentially mammalian or avian Metapneumoviruse,  it suffices to design or select primers or probes based on those stretches of virus-specific nucleic acid sequences that show high homology. In general, for nucleic acid sequences, homology percentages of 90% or higher guarantee sufficient cross-reactivity to be relied upon in diagnostic tests utilizing stingent conditions of hybridisation.<br>
The invention for example provides a method for virologically diagnosing a MPV infection of an animal, in particular of a mammal, more in particular of a human being, comprising determining in a sample of said animal the presence of a viral isolate or component thereof by reacting said sample with a MPV specific nucleic acid a or antibody according to the invention, and a method for serologically diagnosing an MPV infection of a mammal comprising determining in a sample of said mammal the presence of an antibody specifically directed against an MPV or<br><br>
component molecule or fragment thereof or an antigen according to the invention. The invention also provides a diagnostic kit for diagnosing an MPV infection comprising an MPV, an MPV-specific nucleic acid, proteinaceous molecule or fragment thereof, antigen and/or an antibody according to the invention, and preferably a means for detecting said MPV, MPV-specific nucleic acid, proteinaceous molecule or fragment thereof, antigen and/or an antibody, said means for example comprising an excitable group such as a fluorophore or enzymatic detection system used in the art (examples of suitable diagnostic kit format comprise IF, ELISA, neutralization assay, RT-PCR assay). To determine whether an as yet unidentified virus component or synthetic analogue thereof such as nucleic acid, proteinaceous molecule or fragment thereof can be identified as MPV-specific, it suffices to analyse the nucleic acid or amino acid sequence of said component, for example for a stretch of said nucleic acid or amino acid, preferably of at least 10, more preferably at least 25, more preferably at least 40 nucleotides or amino acids (respectively), by sequence homology comparison with known MPV sequences and with known non-MPV sequences APV-C is preferably used) using for example phylogenetic analyses as provided herein. Depending on the degree of relationship with said MPV or non-MP\ sequences, the component or synthetic analogue can be identified.<br>
The invention also provides method for virologically diagnosing an MPV infection of a mammal comprising determining in a sample of said mammal the presence of a viral isolate or component thereof by reacting said sample with a cross-reactive nucleic acid derived from APV (preferably serotype C) or a cross-reactive antibody reactive with said APV, and a method for serologically diagnosing an MPV infection of a mammal comprising determining in a sample of said mammal the presence of a cross-reactive antibody that is also directed against an APV or<br>
component thereof by reacting said sample with a proteinaceous molecule or<br>
â€¢ fragment thereof or an antigen derived from APV. Furthermore, the invention<br>
provides the use of a diagnostic kit initially designed for AVP or AVP-antibody detection for diagnosing an MPV infection, in particular for detecting said MPV infection in humans.<br>
The invention also provides method for virologically diagnosing an APV infection in a bird comprising determining in a sample of said bird the presence of a viral isolate or component thereof by reacting said sample with a cross-reactive<br><br>
nucleic acid derived from MPV or a cross-reactive antibody reactive with said MPV. and a method for serologically diagnosing an APV infection of a bird comprising determining in a sample of said bird the presence of a cross-reactive antibody that is also directed against an MPV or component thereof by reacting said sample with a proteinaceous molecule or fragment thereof or an antigen derived from MPV, Furthermore, the invention provides the use of a diagnostic kit initially designed for MPV or MPV-antibody detection for diagnosing an APV infection, in particular for detecting said APV infection in poultry such as a chicken, duck or turkey.<br>
As said, with treatment, similar use can be made of the cross-reactivity found, in particular when circumstances at hand make the use of the more homologous approach less straightforward. Vaccinations that can not wait, such as emergency vaccinations against MPV infections can for example be performed with vaccine-preparations derived from APV(preferably type C) isolates when a more homologous MPV vaccine is not available, and, vice versa, vaccinations against APV infections can be contemplated with vaccine preparations derived from MPV, Also, reverse genetic techniques make it possible to generate chimeric APV-MPV virus constructs that are useful as a vaccine, being sufficiently dissimilar to field isolates of each of the respective strains to be attenuated to a desirable level. Similar reverse genetic techniques will make it also possible to generate chimeric paramyxovirus-metapneumovirus constructs, such as RSV-MPV or PI3-MPV constructs for us in a vaccine preparation. Such constructs are particularly useful as a combination vaccine to combat respiratory tract illnesses.<br>
The invention thus provides a novel etiological agent, an isolated essentially mammalian negative-sense single stranded RNA virus (herein also called MPV) belonging to the subfamily Pneumovirinae of the family Paramyxoviridae but not identifiable as a classical pneumovirus, and belonging to the genus Metapneumovirus, and MPV-specific components or synthetic analogues thereof. Mammalian viruses . resembling metapneumoviruses, i.e. metapneumoviruses isolatable from mammals that essentially function as natural host for said virus or cause disease in said mammals, have until now not been found. Metapneumoviruses, in general thought to be essentially restricted to poultry as natural host or aetiological agent of disease, are also known as avian pneumoviruses. Recently, an APV isolate of duck was described (FR 2 801 607), further demonstrating that APV infections are essentially restricted to birds as natural hosts.<br><br>
The invention provides an isolated mammalian <br>
called MPV) comprising a gene order and amino acid sequence distinct from that of the genus Pneumovirus and which is closely related and considering its phylogenetic relatedness likely belonging to the genus Metapneumovirus within the subfamily Pneumovirinae of the family Pararnyxoviridae. Although until. now, . metapneumoviruses have only been isolated from birds, it is now shown that related, albeit materially distinct, viruses can be identified in other animal species such as mammals. Herein we show repeated isolation of MPV from humans, whereas no such reports exists for APV. Furthermore, unlike APV, MPV essentially does not or only httle replicates in chickens and turkeys where it easily does in cynomolgous macaques. No reports have been found on replication of APV in mammals. In addition, whereas specific anti-sera raised against MPV neutralize MPV, anti-sera raised against APV A, B or C do not neutralize MPV to the same extent, and this lack of full cross reactivity provides another proof for MPV being a different metapneumovirus. Furthermore, where APV and MPV share a similar gene order, the G and SH proteins of MPV are largely different from the ones known of APV in that they show no significant sequence homologies on both the amino acid or nucleic acid level. Diagnostic assays to discriminate between APV and MPV isolates or . antibodies directed against these different viruses can advantageously be developed based on one or both of these proteins (examples are IF, ELISA, neutralization assay, RT-PCR assay). However, also sequence and/or antigenic information obtained firom the more related N, P, M, F and L proteins of MPV and analyses of sequence homologies with the respective proteins of APV, can also be used to discriminate between APV and MPV. For example, phylogenetic analyses of sequence information obtained firom MPV revealed that MPV and APV are two different viruses. In particular, the phylogenetic trees show that APV and MPV are two different lineages of virus. We have also shown that MPV is circulating in the human popidation for at least 50 years, therefore interspecies transmission has probably taken place at least 50 years ago and is not an everyday event.Since MPV CPE was virtually indistinguishable from that caused by hRSV or hPIV-1 in tMK or other cell cultures, the MPV may have well gone unnoticed until now. tMK (tertiary monkey kidney cells, i.e. MK cells in a third passage in cell culture) are preferably used due to their lower costs in comparison to primary or secondary culltures. The CPE is, as well as with some of the classical Paramyxoviridae, characterized by syncytium formation<br><br>
after which the cells showed rapid internal disruption, followed by detachment of the cells from the monolayer. The cells usually (but not always) displayed CPE after three passages of virus from original material, at day 10 to 14 post inoculation, somewhat later than CPE caused by other viruses such as hRSV or hPIV-1.<br>
Classically, as devastating agents of disease, paramyxoviruses account for many animal and human deaths worldwide each year. The Paramyxoviridae form a family within the order of Mononegavirales (negative-sense single stranded RNA viruses), consisting of the sub-familys Paramyxovirinae and Pneumovirinae, The latter sub-family is at present taxonomically divided in the genera Pneumovirus and Metapneumovirus ^ Human respiratory syncytial virus (hRSV), the type species of the Pneumovirus genus, is the single most important cause of lower respiratory tract infections during infancy and early childhood worldwide2. Other members of the Pneumovirus genus include the bovine and ovine respiratory syncytial viruses and pneumonia virus of mice (PVM).<br>
Avian pneumovirus (APV) also known as turkey rhinotracheitis virus (TRTV), the aetiological agent of avian rhinotracheitis, an upper respiratory tract infection of turkeys3, is the sole member of the recently assigned Metapneumovirus genus, which, as said was until now not associated with infections, or what is more, with diseaseof mammals. Serological subgroups of APV can be differentiated on the basis of nucleotide or amino acid sequences of the G glycoprotein and neutralization tests using monoclonal antibodies that also recognize the G glycoprotein. Within subgroups A, B and D the G protein shows 98.5 to 99.7% aa sequence identity within subgroups while between the subgroups only 31,2- 38% aa identity is observed. See for example Collins, M.S., Gough, KE. and Alexander, D.J., Antigenic differentiation of avian pneumovirus isolates using polyclonal antisera and mouse monoclonal antibodies. Avian Pathology, 1993. 22: p. 469-479.; Cook, J.K.A.. Jones, B.V.. Ellis, M.M., Antigenic differentiation of strains of turkey rhinotracheitis virus using monoclonal antibodies. Avian Pathology, 1993. 22: p. 257-273; Bayon-Auboyer, M.H., et al., Nucleotide sequences of the F, L and G protein genes of two non-A/non-B avian pneumoviruses (APV) reveal a novel APV subgroup. 3 Gen Virol, 2000. 81(Pt 11); p. 2723-33; Seal, B.S., Matrix protein gene nucleotide and predicted amino acid sequence demonstrate that the first US avian pneumovirus isolate is distinct from European strains. Virus Res, 1998. 58(1-2): p. 45*52; Bayon-Auboyer, M.H., et al., Comparison of F-, G- and N-based RT-PCR protocols with conventional virological<br><br>
procedures for the detection and typing of turkey rhinotracheitis virus. Arch Virol, .999.144(6): p. 1091-109; Juhasz, K. and A.J. Easton, Extensive sequence variation in he attachment (G) protein gene of avian pneumovirus: evidence for two distinct Fubgroups. J Gen Virol, 1994. 75(Pt 11): p. 2873-80.<br>
A further serotype of APV is provided in WOOO/20600, which describes the Colorado isolate of APV and compared it to known APV or TRT strains with in vitro serum neutralization tests. First, the Colorado isolate was tested against monospecific polyclonal antisera to recognized TRT isolates. The Colorado isolate was not neutralized by monospecific antisera to any of the TRT strains. It was, however, neutralized by a hyperimmune antiserum raised against a subgroup A strain. This antiserum neutralized the homologous virus to a titre of 1:400 and the Colorado isolate to a titer of 1: 80. Using the above method, the Colorado isolate was then tested against TRT monoclonal antibodies. In each case, the reciprocal neutralization titer was 
The Colorado strain of APV does not protect SPF chicks against challenge with either a subgroup A or a subgroup B strain of TRT virus. These results suggest that the Colorado isolate may be the first example of a farther serotype of avian pneumovirus, as also suggested by Bayon-Auboyer et al (J. Gen. Vir. 81:2723-2733 (2000),<br>
In a preferred embodiment, the invention provides an isolated MPV taxonomicaHy corresponding to a (hereto unknown mammalian) metapneumovirus comprising a gene order distinct from that of the pneumoviruses within the subfamily Pneumovirinae of the family Paramyxoviridae, The classification of the two genera is based primarily on their gene constellation; metapneumoviruses generally lack non-structural proteins such NSl or NS2 ( see also Randhawa et al., J. Vir. 71:9849-9854 (1997) and the gene order is different from that of pneumoviruses (RSV: â€¢3.NSl.NS2-N-P-M-SH.G-F-M2-L-5', APV: '3-N.P-M-F-M2-SH-G-L-5') 4,56,. MPV as<br>
provided by the invention or a virus isolate taxonomicaHy corresponding therewith is<br>
*<br>
upon EM analysis revealed by paramyxovirus-like particles. Consistent with the classification, MPV or virus isolates phylogenetically corresponding or taxonomically corresponding therewith are sensitive to treatment with chloroform; are cultured optimally on tMK cells or cells functionally equivalent thereto and are essentially<br><br>
trypsine dependent in most cell cultures. Furthermore, the typical CPE and lack of haemagglutinating activity with most classically used red blood cells suggested that a virus as provided herein is, albeit only distantly, related to classical pneumoviruses such as RSV. Although most paramyxoviruses have haemagglutinating acitivity, most of the pneumoviruses do not 13-An MPV according to the invention also contains a second overlapping ORF (M2-2) in the nucleic acid fragment encoding the M2 protein, as in general most other pneumoviruses such as for example also demonstrated in Ahmadian et al.. J. Gen. Vir. 80:2011-2016 (1999)<br>
To find further viral isolates as provided by the invention it suffices to test a sample, optionally obtained from a diseased animal or human, for the presence of a virus of the sub-family Pneumovirinae, and test a thus obtained virus for the presence of genes encoding (functional) NSl or NS2 or essentially demonstrate a gene order that is different from that of pneumoviruses such as RSV as already discussed above. Furthermore, a virus isolate phylogenetically corresponding and thus taxonomically corresponding with MPV may be found by cross-hybridisation experiments using nucleic acid from a here provided MPV isolate, or in classical cross-serology experiments using monoclonal antibodies specifically directed against and/or antigens and/or immunogens specifically derived from an MPV isolate.<br>
Newly isolated viruses are phylogenetically corresponding to and thus taxonomically corresponding to MPV when comprising a gene order and/or amino acid sequence sufficiently similar to our prototypic MPV isolate(s), or are structurally corresponding therewith, and show close relatedness to the genus Metapneumovirus within the subfamily Pneumovirinae. The highest amino sequence homology, and defining the structural correspondence on the individual protein level, between MPV and any of the known other viruses of the same family to date (APV subtype C) is for matrix 87%, for nucleoprotein 88%, for phosphoprotein 68%, for fusionprotein 81% and for parts of the polymerase protein 56-64%, as can be deduced when comparing the sequences given in figure 6 with sequences of other viruses, in particular of AVP-C. Individual proteins or whole virus isolates with, respectively, higher homology to these mentioned maximum values are considered phylogenetically corresponding and thus taxonomically corresponding to MPV, and comprise a nucleic acid sequence structurally corresponding with a sequence as shown in figure 6. Herewith the invention provides a virus phylogenetically corresponding to the deposited virus.<br><br>
It should be noted that, similar to other viruses, a certain degree ot variation is found. between different isolated essentially mammalian negative-sense single stranded RNA virus isolates as provided herein. In phylogenetic trees, we have identified at least 2 genetic clusters of virus isolates based on comparitive sequence analyses of parts of the L, M, N and F genes. Based on nucleotide and amino-acid differences in the viral nucleic acid or amino acid sequences (the viral sequences), and in analogy to other pneumoviruses such as RSV, these MPV genotypes represent subtypes of MPV. Within each of the genetic clusters of MPV isolates, the percentage identity at the nucleotide level was found to be 94-100 for L, 91-100 for M, 90-100 for N and 93-100 for F and at the amino acid level the percentage identity was found to be 91-100 for L, 98-100 for M, 96-100 for N and 98-100 for F. A further comparison can be found in figures 18 to 28. The minimum percentage identity at the nucleotide level for the entire group of isolated essentially mammahan negative-sense single stranded RNA virus as provided herein (MPV isolates) identified so far was 81 for L and M, 83 for N . and 82 for F. At the amino acid level, this percentage was 91 for L and N, 94 for M, and 95 for F. The viral sequence of a MPV isolate or an isolated MPV F gene as provided herein for example shows less than 81%nucleotide sequence identity or less than 82%(amino acid sequence identity with the respective nucleotide or amino acid sequence of an APV-C fusion (F) gene as for example provided by Seal et al., Vir. Res. 66:139147 (2000).<br>
Also, the viral sequence of a MPV isolate or an an isolated MPV L gene as provided herein for example shows less than 61% nucleotide sequence identity or less than 63% amino acid sequence identity with the respective nucleotide or amino acid sequence of an APV-A polymerase gene as for example provided by Randhawa et al.Â»., J. Gen. Vir. 77:3047-3051 (1996).<br>
Sequence divergence of MPV strains around the world may be somewhat higher, in analogy with other viruses. Consequently, two potential genetic clusters are identified by analyses of partial nucleotide sequences in the N, M, F and L ORFs of 9 virus isolates. 90-100% nucleotide identity was observed within a cluster, and 81-88% identity was observed between the clusters. Sequence information obtained on more virus isolates confirmed the existence of two genotypes. Virus isolate ned/00/01 as prototype of cluster A, and virus isolate ned/99/01 as prototype of cluster B have been used in cross neutralization assays to test whether the genotypes are related to different serotypes or subgroups. From these data we conclude that essentially<br><br>
mammalian virus isolates displaying percentage amino acid homology higher than 64 for L, 87 for M. 88 for N, 68 for P, 81 for F 84 for M2-1 or 58 for M2-2 to isolate 1-2614 may be classified as an isolated essentially mammalian negative-sense single stranded RNA virus as provided herein. In particular those virus isolates in general that have a minimum percentage identity at the nucleotide sequence level with a prototype MPV isolate as provided herein of 81 for L and M, 83 for N and/or 82 for F are members of the group of MPV isolates as provided herein . At the amino acid level, these percentage are 91 for L and N, 94 for M, and/or 95 for F. When the percentage amino acid sequence homology for a given virus isolate is higher than 90 for L and N, 93 for M, or 94 for F, the virus isolate is similar to the group of MPV isolates displayed in figure 5. When the percentage amino acid sequence homology for a given virus isolate is higher than 94 for L, 95 for N or 97 for M and F the virus isolate can be identified to belong to one of the genotype clusters represented in figure 5. It should be noted that these percentages of homology, by which genetic clusters are defined, are similar to the degree of homology found among genetic clusters in the corresponding genes of RSV.<br>
In short, the invention provides an isolated essentially mammalian negative-sense single stranded RNA virus (MPV) belonging to the sub-family Pneumovirinqe of the family Paramyxouiridae and identifiable as phylogenetically corresponding to the genus Metapneumovirus by determining a nucleic acid sequence of a suitable fragment of the genome of said virus and testing it in phylogenetic tree analyses wherein maximum likelihood trees are generated using 100 bootstraps and 3 jumbles and finding it to be more closely phylogenetically corresponding to a virus isolate deposited as 1-2614 with CNCM, Paris than it is corresponding to a virus isolate of avian pneumovirus (APV) also known as turkey rhinotracheitis virus (TRTV), the aetiological agent of avian rhinotracheitis.<br>
Suitable nucleic acid genome fragments each useful for such phylogenetic tree analyses are for example any of the RAP-PCR fragments 1 to 10 as disclosed herein in the detailed description, leading to the various phylogenetic tree analyses as disclosed herein in figures 4 or 5. Phylogenetic tree analyses of the nudeoprotein (N), phosphoprotein (P), matrixprotein (M) and fusion protein (F) genes of MPV revealed the highest degree of sequence homology with APV serotype C, the avian pneumovirus found primarily in birds in the United States<br><br>
In a preferred embodiment, the invention provides an isolated essentially mammalian negative-sense single stranded RNA virus (MPV) belonging to the subfamily Pneumovirinae of the family Paramyxoviridae and identifiable as phylogenetically corresponding to the genus Metapneumovirus by determining a nucleic acid sequence of a suitable fragment of che genome of said virus and testing it in phylogenetic tree analyses wherein maximum likelihood trees are generated using 100 bootstraps and 3 jumbles and finding it to be more closely phylogenetically corresponding to a virus isolate deposited as 1-2614 with CNCM, Paris than it is corresponding to a virus isolate of avian pneumovirus (APV) also known as turkey rhinotracheitis virus (TRTV), the aetiological agent of avian rhinotracheitis, wherein said suitable fragment comprises an open reading frame encoding a viral protein of said virus.<br>
A suitable open reading frame (ORF) comprises the ORF encoding the N protein. When an overall amino acid identity of at least 91%, preferably of at least 95% of the analysed N-protein with the N-protein of isolate 1-2614 is found, the analysed virus isolate comprises a preferred MPV isolate according to the invention. As shown, the first gene in the genomic map of MPV codes for a 394 amino acid (aa) protein and shows extensive homology with the N protein of other pneumoviruses. The length of the N ORF is identical to the length of the N ORF of APV-C (Table 5) and is smaller than those of other paramyxoviruses (Barr et al., 1991). Analysis of the amino acid sequence revealed the highest homology with APV-C (88%), and only 7-11% with other paramyxoviruses (Table 6).<br>
Barr et al (1991) identified 3 regions of similarity between viruses belonging to the order Mononegavirales: A, B and C (Figure 8). Although similarities are highest within a virus family, these regions are highly conserved between virus familys. In all three regions MPV revealed 97% aa sequence identity with APV-C, 89% with APV-B, 92 with APV-A, and 66-73% with RSV and PVM. The region between aa residues 160 and 340 appears to be highly conserved among metapneumoviruses and to a somewhat lesser extent the Pneumovirinae (Miyahara et aL, 1992; Li et aL, 1996; Barr et al, 1991). This is in agreement with MPV being a metapneumovirus, this particular region showing 99% similarity with APV C.<br>
Another suitable open reading frame (ORF) useful in phylogenetic analyses comprises the ORF encoding the P protein. When an overall amino acid identity of at least 70%, preferably of at least 85% of the analysed P-protein with the P-protein of<br><br>
isolate 1-2614 is found, the analysed virus isolate comprises a preferred MPV isolate according to the invention. The second ORF in the genome map codes for a 294 aa protein which shares 68% aa sequence homology with the P protein of APV-C, and only 22-26% with the P protein of RSV (Table 6). The P gene of MPV contains one substantial ORF and in that respect is similar to P from many other paramyxoviruses (Reviewed in Lamb and Kolakofsky, 1996; Sedlmeier et al, 1998). In contrast to APV A and B and PVM and similar to RSV and APV-C the MPV P ORF lacks cysteine residues. Ling (1995) suggested that a region of high similarity between all pneumoviruses (aa 185-241) plays a role in either the RNA synthesis process or in maintaining the structural integrity of the nucleocapsid complex. This region of high similarity is also found in MPV (Figure 9) especifically when conservative substitutions are taken in account, showing 100% similarity with APV-C, 93 % with APV-A and B, and approximately 81% with RSV. The C-terminus of the MPV P protein is rich in glutamate residues as has been described for APVs (Ling et al., 1995).<br>
Another suitable open reading frame (ORF) useful in phylogenetic analyses comprises the ORF encoding the M protein. When an overall amino acid identity of at least 94%, preferably of at least 97% of the analysed M-protein with the M-protein of isolate 1-2614 is found, the analysed virus isolate comprises a preferred MPV isolate according to the invention. The third ORF of the MPV genome encodes a 254 aa protein, which resembles the M ORFs of other pneumoviruses. The M ORF of MPV has exactly the same size as the M ORFs of other metapneumoviruses (Table 5) and shows high aa sequence homology with the matrix proteins of APV (76-87%) lower homology with those of RSV and PVM (37-38%) and 10% or less homology with those of other paramyxoviruses (Table 6). Easton (1997) compared the sequences of matrix proteins of all pneumoviruses and found a conservedhexapeptide at residue 14 to 19 that is also conserved in MPV (Figure 10). For RSV, PVM and APV small secondary ORFs within or overlapping with the major ORF of M have been identified (52 aa and 51 aa in bRSV, 75 aa in RSV, 46 aa in PVM and 51 aa in APV) (Yu et al., 1992; Easton et al., 1997; Samal et al., 1991; Satake et al, 1984). We noticed two small ORFs in the M ORF of MPV. One small ORF of 54 aa residues was foimd within the major M ORF , starting at nucleotide 2281 and one small ORF of 33 aa residues was found overlapping with the major ORF of M starting at nucleotide 2893 (data not shown). Similar to the secondary ORFs of RSV and APV there is no significant<br><br>
homology between these secondary ORFs and secondary ORFs of the other pneumoviruses, and apparent start or stop signals are lacking. In addition, evidence for the synthesis of proteins corresponding to these secondary ORFs of APV and RSV has not been reported.<br>
Another suitable open reading frame (OEF) useful in phylogenetic analyses comprises the ORF encoding the F protein. When an overall amino acid identity of at least 95%, preferably of at least 97% of the analysed F-protein with the F-protein of isolate I-2614 is found, the analysed virus isolate comprises a preferred MPV isolate according to the invention. The F ORF of MPV is located adjacent to the M ORF, which is characteristic for members of the Metapneumovirus genus. The F gene of MPV encodes a 539 aa protein, which is two aa residues longer than F of APV-C (Table 5). Analysis of the aa sequence revealed 81% homology with APV-C, 67% with APV-A and B, 33-39% with pneumovirus F proteins and only 10-18% with other paramyxoviruses (Table 6). One of the conserved features among F proteins of paramyxoviruses, and also seen in MPV is the distribution of cysteine residues (Morrison, 1988; Yu et al, 1991). The metapneumoviruses share 12 cysteine residues in Fl (7 are conserved among all paramyxoviruses), and two in F2 (1 is conserved among all paramyxoviruses). Of the 3 potential N-linked glycosylation sites present in the F ORF of MPV, none are shared with RSV and two (position 66 and 389) are shared with APV. The third, unique, potential N-linked glycosylation site for MPV is located at position 206 (Figure 11). Despite the low sequence homology with other paramyxoviruses, the F protein of MPV revealed typical fusion protein characteristics consistent with those described for the F proteins of other Paramyxoviridae family members (Morrison, 1988). F proteins of Paramyocoviridae members are synthesized as inactive precursors (FO) that are cleaved by host cell proteases which generate ammo terminal F2 subunits and large carboxy terminal Fl subunitss.,The proposed<br>
cleavage site (Collins et al., 1996) is conserved among all members of the<br>
*<br>
Paramyxoviridae family. The cleavage site of MPV contains the residues RQSR. Both arginine (R) residues are shared with APV and RSV, but the glutamine (Q) and serine (S) residues are shared with other paramyxoviruses such as human parainfluenza virus type 1, Sendai virus and morbilliviruses (data not shown). The hydrophobic region at the amino terminus of Fl is thought to function as the membrane fusion domain ani shows high sequence similarity among paramyxoviruses and morbilliviruses and to a lesser extent the pneumoviruses<br><br>
(Morrison, 1988). These 26 residues (position 187-163, Figure 11) are conserved between MPV and APV-C, which is in agreement with this region being highly conserved among the metapneumoviruses (Naylor etal., 1998; Seal etal., 2000).<br>
As is seen for the F2 subunits of APV and other paramyxoviruses, MPV revealed a deletion of 22 aa residues compared with RSV (position 107-128, Figure 11). Furthermore, for RSV and APV, the signal peptide and anchor domain were found to be conserved within subtypes and displayed high variability between subtypes (Plows et al., 1995; Naylor et al, 1998). The signal peptide of MPV (aa 10-35, Figure 11) at the amino terminus of F2 exhibits some sequence similarity with APV-C (18 out of 26 aa residues are similar) and less conservation with other APVs or RSV. Much more variabliity is seen in the membrane anchor domain at the carboxy terminus of Fl, although some homology is still seen with APV-C.<br>
Another suitable open reading frame (ORF) useful in phylogenetic analyses comprises the ORF encoding the M2 protein. When an overall amino acid identity of at least 85%, preferably of at least 90% of the analysed M2-protein with the M2-protein of isolate 1-2614 is found, the analysed virus isolate comprises a preferred MPV isolate according to the invention. M2 gene is unique to the Pneumovirinae and two overlapping ORFs have been observed in all pneumoviruses. The first major ORF represents the M2-1 protein which enhances the processivity of the viral polymerase (Collins et al., 1995; Collins, 1996) and its readthrough of intergenic regions (Hardy et al, 1998; Fearns et al, 1999). The M2-1 gene for MPV, located adjacent to the F gene, encodes a 187 aa protein (Table 5), and reveals the highest (84%) homology with M2-1 of APV-C (Table 6). Comparison of all pneumovirus M2-1 proteins revealed the highest conservation in the amino-terminal half of the protein (Collins et al, 1990; Zamora et al, 1992; Ahmadian et al, 1999), which is in agreement with the observation that MPV displays 100% similarity with APV-C in the first 80 aa residues of the protein (Figure 12A). The MPVM2-1 protein contains 3 cysteine residues located within the first 30 aa residues that are conserved among all pneunoviruses. Such a concentration of cysteines is firequently found in zinc-binding proteins (Ahmadian et al, 1991; Cuesta et al., 2000),<br>
The secondary ORFs (M2-2) that overlap with the M2-1 ORFs of pneumoviruses are conserved in location but not in sequence and are thought to be involved in the control of the switch between virus RNA replication and transcription (Collins et al,<br><br>
1985; Elango et oZ., 1985; Baybutt et al, 1987; Collins et al., 1990; ling et aL, 1992; Zamora et al, 1992; Alansari et al., 1994; Ahmadian et al, 1999; Bermingham et al, 1999). For MPV, the M2-2 ORF starts at nucleotide 512 in the M2-1 ORF (Figure 7), which is exactly the same start position as for APV-C. The length of the M2-2 ORFs are the same for APV-C and MPV, 71 aa residues (Table 5). Sequence comparison of the M2-2 ORF (Figure 12B) revealed 56% aa sequence homology between MPV and APV-C and only 26-27%aa sequence homology between MPV and APV-A and B (Table 6).<br>
Another suitable open reading frame (ORF) useful in phylogenetic analyses comprises the ORF encoding the L protein. When an overall amino acid identity of at least 91%, preferably of at least 95% of the analysed L-protein with the L-protein of isolate I-2614 is found, the analysed virus isolate comprises a preferred MPV isolate according to the invention. In analogy to other negative strand viruses, the last ORF of the MPV genome is the RNA-dependent RNA polymerase component of the replication and transcription complexes. The L gene of MPV encodes a 2005 aa protein, which is 1 residue longer than the APV-A protein (Table 5). The L protein of MPV shares 64% homology with APV-A, 42-44% with RSV, and approximately 13% with other paramyxoviruses (Table 6). Poch et al. (1989; 1990) identified six conserved domains within the L proteins of non-segmented negative strand RNA viruses, from which domain III contained the four core polymerase motifs that are thought to be essential for polymerase function. These motifs (A, B, C and D) are well conserved in the MPV L protein: in motifs A, B and C: MPV shares 100% similarity with all pneumoviruses and in motif D MPV shares 100 % similarity with APV and 92% with RSVs. For the entire domain III (aa 625- 847 in the L ORF), MPV shares 83% identity with APV, 67-68% with RSV and 26-30% with other paramyxoviruses (Figure 15). In addition to the polymerase motifs the pneiunovirus L proteins contain a sequence which conforms to a consensus ATP binding motif K(X)2iGEGAGN(X)2oK (Stec, 1991).The MPV L ORF contains a similar motif as APV, in which the spacing of the intermediate residues is off by one: K(x)22GEGAGN(X)i9 K<br>
A much preferred suitable open reading frame (ORF) useful in phylogenetic analyses comprises the ORF encoding the SH protein. When an overall amino acid identity of at least 30%, preferably of at least 50%, more preferably of at least 75% of the analysed SH-protein with the SH-protein of isolate 1-2614 is found, the analysed virus isolate comprises a preferred MPV isolate according to the invention. The gene<br><br>
located adjacent to M2 of MPV encodes a 183 aa protein (Figure 7). Analysis of the nucleotide sequence and its deduced amino acid sequence revealed no discernible homology with other RNA virus genes or gene products. The SH ORF of MPV is the longest SH ORF known to date (Table 5). The composition of the aa residues of the SH ORF is relatively similar to that of APV, RSV and PVM, with a high percentage of threonine and serine (22%, 18%, 19%, 20.0%, 21% and 28% serine/threonine content for MPV, APV, RSV A, RSV B, bRSV and PVM respectively). The SH ORF of MPV contains 10 cysteine residues, whereas APV SH contains 16 cysteine residues. All pneumoviruses have similar numbers of potential N-glycosylation sites (MPV 2, APV l,RSV2,bRSV3,PVM4).<br>
The hydrophobicity profiles for the MPV SH protein and SH of APV and RSV revealed similar structural characteristics (Figure 13B). The SH ORFs of APV and MPV have a hydrophylic N-terminus (aa 1-30), a central hydrophobic domain (aa 30-53) which can serve as a potential membrane spanning domain, a second hydrophobic domain around residue 160 and a hydrophilic C-terminus. In contrast, RSV SH appears to lack the C-terminal half of the APV and MPV ORFs. In all pneumovirus SH proteins the hydrophobic domain is flanked by basic amino acids, which are also found in the SH ORF for MPV (aa 29 and 54).<br>
Another much preferred suitable open reading frame (ORF) useful in phylogenetic analyses comprises the ORF encoding the G protein. When an overall amino acid identity of at least 30%, preferably of at least 50%, more preferably of at least 75% of the analysed G-protein with the G-protein of isolate 1-2614 is found, the analysed virus isolate comprises a preferred MPV isolate according to the invention. The G ORF of MPV is located adjacent to the SH gene and encodes a 236 amino acid protein. A secondary small ORF is found immediately following this ORF, potentially coding for 68 aa residues (pos. 6973-7179,), but lacking a start codon. A third major ORF, in a different reading frame, of 194 aa residues (fragment 4, Figure 7) is overlapping with both of these ORFs, but also lacks a startcodon (nucleotide 6416-7000). This major ORF is followed by a fourth ORF in the same reading frame (nt 7001-7198), possibly coding for 65 aa residues but again lacking a start codon. Finally, a potential ORF of 97 aa residues (but lacking a startcodon) is found in the third reading frame (nt 6444-6737, Figure 1). Unlike the first ORF, the other ORFs do not have apparent gene start or gene end sequences (see below). Although the 236 aa residue G ORF probably represents at least a part of the MPV attachment protein it can not be<br><br>
excluded that the additional coding sequences are expressed as separate proteins or as part of the attachment protein through some RNA editing event. It should be noted that for APV and RSV no secondary ORFs after the primary G ORF have been identified but that both APV and RSV have secondary ORFs within the major ORP of G. However, evidence for expression of these ORFs is lacking and there is no homology between the predicted aa sequences for different viruses (ling et ah, 1992). The secondary ORFs in MPV G do not reveal characteristics of other G proteins and whether the additional ORFs are expressed requires further investigation. BLAST analyses with all. four ORFs revealed no discernible homology at the nucleotide or aa sequence level with other known virus genes or gene products. This is in agreement with the low sequence homologies found for other G proteins such as hRSV A and B (53%) (Johnson etal., 1987) and APV A and B (38%) (Juhasz et al, 1994). Whereas most of the MPV ORFs resemble those of APV both in length and sequence, the G ORF of MPV is considerably smaller than the G ORF of APV (Table 5). The aa sequence revealed a serine and threonine content of 34%, which is even higher than the 32% for RSV and 24% for APV. The G ORF also contains 8.5% proline residues, which is higher than the 8% for RSV and 7% for APV. The unusual abimdance of proline residues in the G proteins of APV, RSV and MPV has also been observed in glycoproteins of mucinous origin where it is a major determinant of the proteins three dimensional structure (Collins etal., 1983; Wertz et al., 1985; Jentoft, 1990). The number of potential N-linked glycosylation sites in G of MPV is similar to other pneumoviruses: MPV has 5, whereas hRSV has 7, bRSV has 5, and APV has 3 to 5.<br>
The predicted hydrophobicity profile of MPV G revealed characteristics similar to the other pneumoviruses. The amino-terminus contains a hydrophylic region followed by a short hydrophobic area (aa 33-53) and a mainly hydrophilic carboxy terminus (Figure 14B). This overall organisation is consistent with that of an anchored t3^e II transmembrane protein and corresponds well with these regions in the G protein of APV and RSV. The G ORF of MPV contains only 1 cysteine residue in contrast to RSV and APV (5 and 20 respectively).<br>
According to classical serological analyses as for example known from Francki, R.I.B., Fauquet, CM., Knudson, D.L., and Brown, F., Classification and nomenclature of viruses. Fifth report of the international Committee on Taxonomy of<br><br>
Viruses. Arch Virol, 1991. Supplement 2: p. 140-144. an MPV isolate is also identifiable as belonging to a serotype as provided herein, being defined on the basis of its immunological distinctiveness, as determined by quantitative neutralization with animal antisera (obtained from for example ferrets or guinnea pigs as provided in the detailed description). Such a serotype has either no cross-reaction with others or shows a homologous-to heterologous titer ratio &gt;16 in both directions. If neutralization shows a certain degree of cross-reaction between two viruses in either or both directions (homologous-to-heterologous tier ration of eight or 16), distinctiveness of serotype is assumed if substantial biophysical/biochemical differences of DNA's exist. If neutralization shows a distinct degree of cross-reaction between two viruses in either or both directions (homologous-to-heterologous tier ration of smaller than eight), identity of serotype of the isolates under study is assumed. As said, useful prototype isolates, such as isolate 1-2614, herein also known as MPV isolate 00-1, are provided herein,<br>
A further classification of a virus as an isolated essentially mammalian negative-sense single stranded RNA virus as provided herein can be made on the basis of homology to the G and/or SH proteins. Where in general the overall amino acid sequence identity between APV (isolated from birds) and MPV (isolated from humans) N, P, M, F, M2 and L ORFs was 64 to 88 percent, and nucleotide sequence homology was also found between the non-coding regions of the APV and MPV genomes, essentially no discernable amino acid sequence homology was found between two of the ORFs of the human isolate (MPV) and any of the ORFs of other paramyxoviruses. The amino acid content, hydrophobicity profiles and location of these ORFs in the viral genome show that they represent G and SH protein analogues. The sequence homology between APV and MPV, their similar genomic organization (3'-N-P-M-F-M2-SH-G-L-5') as well as phylogenetic analyses provide further evidence for the proposed classification of MPV as the first mammalian metapneumovirus.New MPV isolates are for thus example identified as such by virus isolation and characterisation on tMK or other cells, by RT-PGR and/or sequence analysis followed by phylogenetic tree analyses, and by serologic techniques such as virus neutralisation assays, indirect immunofluorescence assays, direct immunofluorescence assays, PACs analyses or other immunological techniques. Preferably these techniques are directed at the SH and/or G protein analogues.<br><br>
For example the invention provides herein a method to identify further isolates of MPV as provided herein, the method comprising inoculating a essentially MPV-uninfected or specific-pathogen-free guinea pig or ferret (in the detailed description the animal is inoculated intranasally but other ways of inoculation such as intramuscular or intradermal inoculation, and using an other experimental animal, is also feasible) with the prototype isolate 1-2614 or related isolates. Sera are collected from the animal at day zero, two weeks and three weeks post inoculation. The animal specifically seroconverted as measured in virus neutralisation (VN) assay and indirect IFA against the respective isolate 1-2614 and the sera from the seroconverted animal are used in the immunological detection of said further isolates.<br>
As an example, the invention provides the characterisation of a new member in the family oÂ£ Paramyxoviridae, a human metapnetunovirus or metapneumovirus-like virus (since its final taxonomy awaits discussion by a viral taxonomy committee the MPV is herein for example described as taxonomically corresponding to APV) (IVIPV) which may cause severe RTI in humans. The clinical signs of the disease caused by MPV are essentially similar to those caused by hESV, such as cough, myalgia, vomiting, fever, broncheolitis or pneumonia, possible conjunctivitis, or combinations thereof. As is seen with hRSV infected children, especifically very young children may require hospitalisation. As an example an MPV which was deposited January 19, 2001 as 1-2614 with CNCM, Institute Pasteur, Paris or a virus isolate phylogenetically corresponding therewith is herewith provided. Therewith, the invention provides a virus comprising a nucleic acid or functional fragment phylogeneticEilly corresponding to a nucleic acid sequence shown in figure 6a, 6b, 6c, or structurally corresponding therewith. In particular the invention provides a virus characterised in that after testing it in phylogenetic tree analyses wherein maximum likelihood trees are generated using 100 bootstraps and 3 jumbles it is found to be more closely phylogenetically corresponding to a virus isolate deposited as 1-2614 with CNCM, Paris than it is related to a virus isolate of avian pneumovirus (APV) also known as turkey rhinotracheitis virus (TRTV), the aetiological agent of avian rhinotracheitis. It is particularly useful to use an AVP-C virus isolate as outgroup in said phylogenetic tree analyses, it being the closest relative, albeit being an essentially non-mammalian virus.<br>
We propose the new human virus to be named human metapneumovirus or metapneumovirus-like virus (MPV) based on several observations. EM analysis<br><br>
revealed paramyxovirus-like particles. Consistent with the classification, MFV appeared to be sensitive to treatment with chloroform. MPV is cultured optimal on tMK cells and is trypsine dependent. The clinical symptoms caused by MPV as well as the typical CPE and lack of haemagglutinating activity suggested that this virus is closely related to hRSV. Although most paramyxoviruses have haemaglutinating acitivity, most of the pneumoviruses do not <br>
As an example, the invention provides a not previously identified paramyxcovirus from nasopharyngeal aspirate samples taken from 28 children suffering from severe RTI. The clinical symptoms of these children were largely similar to those caused by hRSV. Twenty-seven of the patients were children below the age of five years and half of these were between 1 and 12 months old. The other patient was 18 years old. All individuals suffered from upper RTI, with synptoms ranging from cough, myalgia, vomiting and fever to broncheohtis and severe pneumonia. The majority' of these patients were hospitalised for one to two weeks.<br>
The virus isolates from these patients had the paramyxovirus morphology in negative contrast electron microscopy but did not react with specific antisera against known human and animal paramyxoviruses. They were all closely related to one another as determined by indirect immunofluorescence assays (IFA) with sera raised against two of the isolates. Sequence analyses of nine of these isolates revealed that the virus is somewhat related to APV. Based on virological data, sequence homology as well as the genomic organisation we propose that the virus is a member of Metapneumovirus genus. Serological surveys showed that this virus is a relatively-common pathogen since the seroprevalence in the Netherlands approaches 100% of humans by the age of five years. Moreover, the seroprevelance was found to be equally high in sera collected from humans in 1958, indicating this virus has been circulating in the human population for more than 40 years. The identification of this proposed new member of the Metapneumovirus genus now also provides for the development of means and methods for diagnostic assays or test kits and vaccines or serum or antibody compositions for viral respiratory tract infections, and for methods to test or screen for antiviral agents useful in the treatment of MPV infections.<br>
To this extent, the invention provides among others an isolated or recombinant nucleic acid or virus-specific functional fragment thereof obtainable from a virus according to the invention. In particular, the invention provides primers and/or probes suitable for identifying an MPV nucleic acid.<br><br>
Furthermore, the invention provides a vector comprising a nucleic acid according to the invention. To begin with, vectors such as plasmid vectors containing (parts of) the genome of MPV, virus vectors containing (parts of) the genome of MPV. (For example, but not limited to other paramyxoviruses, vaccinia virus, retroviruses, baculovirus), or MPV containing (parts of) the genome of other viruse or other pathogens are provided. Furthermore, a number of reverse genetics techniques have been described for the generation of recombinant negative strand viruses, based on two critical parameters. First, the production of such virus relies on the replication of a partial or full-length copy of the negative sense viral KNA (vRNA) genome or a complementary copy thereof (cRNA). This vRNA or cRNA can be isolated from infectious virus, produced upon in-vitro transcription, or produced in cells upon transfection of nucleic acids. Second, the production of recombinant negative strand virus relies on a functional polymerase complex. Typically, the polymerase complex of pneumoviruses consists of N, P, L and possibly M2 proteins, but is not necessarily limited thereto. Polymerase complexes or components thereof can be isolated from virus particles, isolated from cells expressing one or more of the components, or produced upon transfection of specific expression vectors.<br>
Infectious copies of MPV can be obtained when the above mentioned vRNA, cRNA, or vectors expressing these RNAs are replicated by the above mentioned polymerase complex 16.17,18,19.20,21.22 Yot the generation of minireplicons or, a reverse genetics system for generating a full-length copy comprising most or all of the genome of MPV it suffices to use 3'end and/or 5'end nucleic acid sequences obtainable from for example APV (Randhawa et al., 1997) or MPV itself.<br>
Also, the invention provides a host cell comprising a nucleic acid or a vector according to the invention. Plasmid or viral vectors containing the polymerase components of MPV (presumably N, P, L and M2, but not necessarily limited thereto) are generated in prokaryotic cells for the expression of the components in relevant cell types (bacteria, insect cells, eukaryotic cells). Plasmid or viral vectors containing full-length or partial copies of the MPV genome will be generated in prokaryotic cells for the expression of viral nucleic acids in-vitro or in-vivo. The latter vectors may contain other viral sequences for the generation of chimeric viruses or chimeric virus proteins, may lack parts of the viral genome for the generation of replication defective virus, and may contain mutations, deletions or insertions for the generation of attenuated viruses.<br><br>
infectious copies of MPV (being wild type, attenuated, replication-defective or himeric) can be produced upon co-expression of the polymerase components ccording to the state-of-the-art technologies described above.<br>
n addition, eukaryotic cells, transiently or stably expressing one or more full-length r partial MPV proteins can be used. Such cells can be made by transfection (proteins ir nucleic acid vectors), infection (viral vectors) or transduction (viral vectors) and nay be useful for complementation of mentioned wild type, attenuated, replication-lefective or chimeric viruses.<br>
A chimeric virus may be of particular use for the generation of recombinant vaccines protecting against two or more viruses 23.24,26  For example, it can be invisaged that a MPV virus vector expressing one or more proteins of RSV or a RSV vector expressing one or more proteins of MPV will protect individuals vaccinated with such vector against both virus infections, A similar approach can be envisaged or PIS or other paramyxoviruses. Attenuated and replication-defective viruses may 36 of use for vaccination purposes with live vaccines as has been suggested for other viruses 25.26.<br>
In a preferred embodiment, the invention provides a proteinaceous molecule or metapneumovirus-specific viral protein or functional fragment thereof encoded by a nucleic acid according to the invention. Useful proteinaceous molecules are for example derived from any of the genes or genomic fragments derivable from a virus according to the invention. Such molecules, or antigenic fragments thereof, as provided herein, are for example useful in diagnostic methods or kits and in pharmaceutical compositions such as sub-unit vaccines. Particularly useful are the F, SH and/or G protein or antigenic fragments thereof for inclusion as antigen or subunit immunogen, but inactivated whole virus can also be used. Particulary useful are also those proteinaceous substances that are encoded by recombinant nucleic acid fragments that are identified for phylogenetic analyses, of course preferred are those that are within the preferred bounds and metes of ORFs useful in phylogenetic analyses, in particular for eliciting MPV specific antibodies, whether in vivo (e.g. for protective puposes or for providing diagnostic antibodies) or in vitro (e.g. by phage display technology or another technique useful for generating synthetic antibodies).<br>
Also provided herein are antibodies, be it natural polyclonal or monoclonal, or synthetic (e.g. (phage) library-derived binding molecules) antibodies that specificallyreact with an antigen comprising a proteinaceous molecule or MPV-<br><br>
specific functional fragment thereof according to the invention. Such antibodies are useful in a method for identifying a viral isolate as an MPV comprising reacting said viral isolate or a component thereof with an antibody as provided herein. This can for example be achieved by using purified or non-purified MPV or parts thereof (proteins, peptides) using ELISA, RIA, FACS or similar formats of antigen detection assays (Current Protocols in Immunology). Alternatively, infected cells or cell cultures may be used to identify viral antigens using classical immunofluorescence or immunohistochemical techniques.<br>
Other methods for identifying a viral isolate as a MPV comprise reacting said viral isolate or a component thereof with a virus specific nucleic acid according to the invention, in particular where said mammalian virus comprises a human virus.<br>
In this way the invention provides a viral isolate identifiable with a method according to the invention as a mammalian virus taxonomically corresponding to a negative-sense single stranded RNA virus identifiable as likely belonging to the genus Metapneumovirus within the sub-family Pneumovirinae of the family Paramyxoviridae,<br>
The method is useful in a method for virologically diagnosing an MPV infection of a mammal, said method for example comprising determining in a sample of said mammal the presence of a viral isolate or component thereof by reacting said sample with a nucleic acid or an antibody according to the invention. Examples are further given in the detailed description, such as the use of PCR (or other amplification or hybridisation techniques well known in the art) or the use of immunofluorescence detection (or other immunological techniques known in the art)<br>
The invention also provides a method for serologically diagnosing a MPV infection of a mammal comprising determining in a sample of said mammal the presence of an antibody specifically directed against a MPV or component thereof by reacting said sample with a proteinaceous molecule or fragment thereof or an antigen according to the invention<br>
Methods and means provided herein are particularly useful in a diagnostic kit for diagnosing a MPV infection, be it by virological or serological diagnosis. Such kits or assays may for example comprise a virus, a nucleic acid, a proteinaceous molecule or fragment thereof, an antigen and/or an antibody according to the invention. Use of a virus, a nucleic acid, a proteinaceous molecule or fragment thereof, an antigen and/or an antibody according to the invention is also provided for the<br><br>
production of a pharmaceutical composition, for example for the treatment or prevention of MPV infections and/or for the treatment or prevention of respiratory tract illnesses, in particular in humans. Attenuation of the virus can be achieved by established methods developed for this purpose, including but not limited to the use of related viruses of other species, serial passages through laboratory animals or/and tissue/cell cultures, site directed mutagenesis of molecular clones and exchange of genes or gene fragments between related viruses.<br>
A pharmaceutical composition comprising a virus, a nucleic acid, a proteinaceous molecule or fragment thereof, an antigen and/or an antibody according to the invention can for example be used in a method for the treatment or prevention of a MPV infection and/or a respiratory illness comprising providing an individual with a pharmaceutical composition according to the invention. This is most useful when said individual comprises a human, especifically when said human is below 5 years of age, since such infants and young children are most likely to be infected by a . human MPV as provided herein. Generally, in the acute phase patients will suffer from upper respiratory symptoms predisposing for other respiratory and other diseases. Also lower respiratory illnesses may occur, predisposing for more and other serious conditions.<br>
The invention also provides method to obtain an antiviral agent useful in the treatment of respiratory tract illness comprising establishing a cell culture or experimental animal comprising a virus according to the invention, treating said culture or animal with an candidate antiviral agent, and determining the effect of said agent on said virus or its infection of said culture or animal. An example of such an antiviral agent comprises a MPV-neutralising antibody, or functional component thereof, as provided herein, but antiviral agents of other nature are obtained as well. The invention also provides use of an antiviral agent according to the invention for the preparation of a pharmaceutical composition, in particular for the preparation of a pharmaceutical composition for the treatment of respiratory tract illness, especifically when caused by an MPV infection, and provides a pharmaceutical composition comprising an antiviral agent according to the invention, useful in a method for the treatment or prevention of an MPV infection or respiratory illness, said method comprising providing an individual with such a pharmaceutical composition.<br><br>
The invention is further explained in the detailed description without limiting it thereto.<br><br>
Figure legends<br>
Figure lA comprises table 1: Percentage homology found between the amino acid sequence of isolate 00-1 and other members of the Pneumovirinae, Percentages (xlOO) are given for the amino acid sequences of N, P, M, F and two EAP-PCR fragments in L (8 and 9/10). Accession numbers used for the analyses are described in the materials and methods section.<br>
Fig IB comprises table 2: Seroprevalence of MPVin humans categorised by age group using immunofluorescence and virus neutralisation assays..<br>
Fig. 2: Schematic representation of the genome of APV with the location and size of the fragments obtained with RAP-PCR and RT-PCR on virus isolate 00-1. Fragments 1 to 10 were obtained using RAP-PCR. Fragment A was obtained with a primer in RAP-PCR fragment 1 and 2 and a primer designed based on alignment of leader and trailer sequences of APV and RSV6. Fragment B was obtained using primers designed in RAP-PCR fragment 1 and 2 and RAP-PCR fragment 3. Fragment C was obtained with primers designed in RAP-PCR fragment 3 and RAP-PCR fragment 4,5,6 and 7.<br>
For all phylogenetic trees, (figures 3-5) DNA sequences were aligned using the ClustalW software package and maximum likelihood trees were generated using the DNA-ML software package of the Phylip 3.5 program using 100 bootstraps and 3 jumbles15. Previously published sequences that were used for the generation of phylogenetic trees are available from Genbank under accessions nvunbers : For all ORFs:hRSV:NC001781;bRSV:NC001989; Furthe FORF: PVM, D11128; APV-A, D00850; APV-B, Y14292; APV-C, AF187152; For the N ORF: PVM. D10331; APV-A, U39295; APV-B, U39296; APV-C, AF176590; For the M ORF: PMV.U66893; APV-A, X5S639; APV-B, U37586; APV-C, AF262571; For the P ORF: PVM, 09649; APV-A, U22110, APV-C, AF176591. Phylogenetic analyses for the nine different virus isolates of MPV were performed with APV strain C as outgroup. Abbreviations used in figures: hRSV: human RSV; bRSV: bovine RSV; PVM: pneumonia virus of mice; APV-A,B,and C: avian pneumovirus type A, B and C. Fig. 3 Comparison of the N, P, M and F ORF's of members of the subfamily Pneumovirinae and virus isolate 00-1. The alignment shows the amino acid sequence<br><br>
of the complete N, P, M and F proteins and partial L proteins of virus isolate 00-1. Amino acids that differ between isolate 00-1 and the other viruses are shown, identical amino acids are represented by periods, gaps are represented as dashes. Numbers correspond to amino acid positions in the proteins. Accession numbers used for the analyses are described in the materials and methods section. APV-A, B or C; Avian Pneumovirus type A, B or C, b-or KRSV: bovine or human respiratory syncytial virus, PVM: pneumonia virus of mice. L8: fragment 8 obtained with RAP-PCR located in L, L9/10: consensus of fragment 9 and 10 obtained with RAP-PCR, located in L. For the P alignment, no APV-B sequence was available from the Genebank, For the L allignment only bRSV, hRSV and APV-A sequences were available.<br>
Fig. 4: Phylogenetic analyses of the N, P, M, and F ORFs of members of the genus Pneumovirinae and virus isolate 00-1. Phylogenetic analysis was performed on viral sequences from the following genes: F (panel A), N (panel B), M (panel C), and P (panel D). The phylogenetic trees are based on maximum likelyhood analyses using 100 bootstraps and 3 jumbles. The scale representing the number of nucleotide changes is shown for each tree.<br>
Fig. 5: Phylogenetic relationship for parts of the F (panel A), N (panel B), M (panel C) and L (panel D) ORFs of nine of the primary MPV isolates with APV-C, it's closest relative genetically. The phylogenetic trees are based on maximum likelyhood analyses. The scale representing the number of nucleotide changes is shown for each tree. Accesion numbers for APV-C: panel A: D00850; panel B: U39295; panel C: X58639; and panel D: U65312-<br><br>
Fig. 6A: Nucleotide and amino acid sequence information from the 3' end  of the genome of MPV isolate 00-l.ORF's are given. N: ORF for nucleoprotein; P: OEF for phosphoprotein; M: ORF for matrix protein; F: ORF for fusion protein; GE: gene end; 3S: gene start.<br>
Fig. 6B and C: Nucleotide and amino acid sequence information from obtained fragments in the polymerase gene (L) of MPV isolates 00-1. Positioning of the fragments in L is based on protein homologies with APV-C (accession nimiber U65312). The translated fragment 8 (Fig. 6B.) is located at amino acid number 8 to 243, and the consensus of fragments 9 and 10 (Fig. 6C) is located at amino acid number 1358 to 1464 of the APV-C L ORF.<br>
Figure 7<br>
Genomic map of MPV isolate 00-1. The nucleotide positions of the start and stop codons are indicated under each ORF.. The double hues which cross the L ORF indicate the shortened representation of the L gene. The three reading frames below the map indicate the primary G ORF (nt 6262-6972) and overlapping potential secondary ORFs.<br>
Figure 8:<br>
Alignment of the predicted amino acid sequence of the nucleoprotein of MPV with those of other pneimioviruses. The conserved regions identified by Barr (1991) are represented by boxes and labelled A, B, and C. The conserved region among pneimioviruses (Li, 1996) is shown gray shaded. Gaps are represented by dashes, periods indicate the positions of identical amino acid residues compared to MPV.<br>
Figure 9:<br>
Amino acid sequence comparison of the phosphoprotein of MPV with those of other pneumoviruses. The region of high similarity (Ling, 1995) is boxed, and the glutamate rich region is grey shaded. Gaps are represented by dashes and periods indicate the position of identical amino acid residues compared to MPV,<br><br>
Figure 10:<br>
Comparison of the deduced amino acid sequence of the matrix protein of MPV with those of other pneumoviruses. The conserved hexapeptidesequence (Easton, 1997) is grey shaded. Gaps are represented by dashes and periods indicate the position of identical amino acid residues relative to MPV.<br>
Figure 11:<br>
Allignment of the predicted amino acid sequence of the fusion protein of MPV with those of other pneumoviruses. The conserved cysteine residues are boxed, iV-linked glycosylation sites are underlined, the cleavage site of FO is double underlined, the fusion peptide, signal peptide and membrane anchor domain are shown grey shaded. Gaps are represented by dashes and periods indicate the position of identical amino acids relative to MPV.<br>
Figure 12<br>
Comparison of amino acid sequence of the M2 ORFs of MPV with those of other pneumoviruses. The alignment of M2-1 ORFs is shown in panel A, with the conserved amino terminus (Collins, 1990; Zamora, 1999) shown grey shaded. The three conserved cysteine residues are printed bold face and indicated by #. The alignment of M2-2 ORFs is shown in panel B. Gaps are represented by dashes and periods indicate the position of identical amino acids relative to MPV.<br>
Figure 13<br>
Amino acid sequence analyses of the SH ORF of MPV. (A) Amino acid sequence of the SH ORFof MPV, with the serine and threonine residues grey shaded, cysteine residues in bold face and the hydrophobic region double underlined. Potential N- linked glycosylation sites are single underlined. Numbers indicate the positions of the basic amino acids flanking the hydrophobic domain. (B) Alignment of the hydrophobicity plots of the SH proteins of MPV, APV-A and hRSV-B. The procedure of Kyte and Doolittle (1982) was used with a window of 17 amino acids. Arrows indicate a strong hydrophobic domain. Positions within the ORF are given on the X-axis.<br><br>
Figure 14<br>
Amino acid sequence analyses of the G ORF of MPV. (A) Amino acid sequence of the G ORF of MPV, with serine, threonine and proline residues grey shaded, the cysteine residue is in bold face and the hydrophobic region double underlined. The potential N-linked glycosylation sites are single underlined. (B) Alingment of the hydrophobicity plots of the G proteins of MPV, APV-A and hRSV-B. The procedure of Kyte and Doolittle (1982) was used with a window of 17 amino acids. Arrows indicate the hydrophobic region, and positions within the ORF are given at the X-axis.<br>
Figure 15<br>
Comparison of the amino acid sequences of a conserved domain of the pol5anerase gene of<br>
MPV and other paramyxoviruses. Domain III is shown with the four conserved<br>
polymerase motifs (A, B, C, D) in domain III (Poch 1998, 1999) boxed.<br>
Gaps are represented by dashes and periods indicate the position of identical amino acid<br>
residues relative to MPV. hPIV3: human parainfluenza virus type 3; SV: sendai virus;<br>
hPIV-2: human parainfluenza virus type 2; NDV: New castle disease virus; MV: measles<br>
virus;; nipah: Nipah virus.<br>
Figure 16:<br>
Phylogenetic analyses of the M2-1 and L ORFs of MPV and selected paramyxoviruses. The M2-1 ORF was ahgned with the M2-1 ORFs of other members of the genus Pneumovirinae (A) and the L ORF was aligned with L ORFs members of the genus pneumovirinae and selected other paramyxoviruses as described in the legends of figure 15 (B). Phylogenetic trees were generated by maximum likelihood analyses using 100 bootstraps and 3 jumbles. The scale representing the number of nucleotide changes is shown for each tree. Numbers in the trees represent bootstrap values based on the consensus trees.<br>
Figure 17:<br>
Noncoding sequences of hMPV isolate 00-1. (A) The noncoding sequences between the ORFs and at the genomic termini are shown in the positive sense. From left to right, stop codons of indicated ORFs are shown, followed by the noncoding sequences, the gene start signals and start codons of the indicated subsequent ORFs. Numbers indicate the first position of start and stop codons in the hMPV map. Sequences that<br><br>
display similarity to published gene end signals are underlined and sequences that display similarity to UAAAAAU/A/C are represented with a line above the sequence. (B) Nucleotide sequences of the genomic termini of hMPV. The genomic termini of hMPV are aligned with each other and with those of APV. Underlined regions represent the primer sequences used in RT-PCR assays which are based on the 3' and 5' end sequences of APV and RSV (Randhawa etal., 1997; Mink et al, 1991). Bold italicalized nucleotides are part of the gene start signal of the N gene. Le: leader, Tr: trailer.<br>
Figure 18:<br>
Comparison of two prototypic hMPV isolates with APV-A and APV-C; DNA similarity<br>
matrices for nucleic acids encoding the various viral proteins.<br>
Figure 19:<br>
Comparison of two prototypic hMPV isolates with APV-A and APV-C; protein<br>
similarity matrices for the various viral proteins.<br>
Figure 20:<br>
Amino acid alignment of the nucleoprotein of two prototype hMPVisolates<br>
Figure 21:<br>
Amino acid alignment of the phosphoprotein of two prototype hMPV isolates<br>
Figure 22:<br>
Ammo acid alignment of the matrix protein of two prototype hMPV isolates<br>
Figure 23:<br>
Amino acid alignment of the fusion protein of two prototype hMPVisolates<br>
Figure 24:<br>
Amino acid alignment of the M2-1 protein of two prototype hMPV isolates<br>
Figure 25:<br>
Amino acid alignment of the M2-2 protein of two prototype hMPV isolates<br><br>
Figure 26:<br>
Amino acid alignment of the short hydrophobic protein of two prototype hMPV<br>
isolates<br>
Figure 27:<br>
Amino acid alignment of the attachement glycoprotein of two prototype hMPV<br>
isolates<br>
Figure 28:<br>
Amino acid alignment of the N-terminus of the polymerase protein of two prototype<br>
hMPV isolates<br>
Figure 29: Residts of RT-PCR assays on throat and nose swabs of 12 guinea pigs inoculated with ned/00/01 and/or ned/99/01.<br>
Figure 30A: IgG response against ned/00/01 and 'ned/99/01 for guinea pigs infected with ned/00/01 and re-infected with ned/00/01 (GP 4, 5 and 6) or ned/99/01 (GP 1 and 3).<br>
Figure SOB: IgG response against ned/00/01 and ned/99/01 for guinea pigs infected with ned/99/01 and re-infected with either ned/00/01 (GP's 8 and 9) or with ned/99/01 (GP's 10, 11, 12).<br>
Figure 31: Specificity of the ned/00/01 and ned/99/01 ELISA on sera taken from guinea pigs infected with either ned/00/01 or ned/99/01.<br>
Figure 32: Mean IgG response against ned/00/01 and ned/99/01 EUSA of 3 homologous (00-1/00-1), 2 homologous (99-1/99-1). 2 heterologous (99-1/00-1) and 2 heterologous (00-1/99-1) infected gxiinea pigs.<br>
Figure 33: Mean percentage of APV inhibition of hMPV infected guinea pigs.<br><br>
Figure 34: Virus neutralisation titers of ned/00/01 and ned/99/01 infected guinea pigs against ned/00/01, ned/99/01 and APV-C.<br>
Figure 35: Results of RT-PCR assays on throat swabs of cynomolgous macaques inoculated (twice) with ned/00/01.<br>
Figure 36 A (top two panels):<br>
IgA, IgM and IgG response against ned/00/01 of 2 cynomologous macaques<br>
(re)infected with ned/00/01.<br>
Figure 36B (bottom panels)<br>
IgG response against APV of 2 cynbomologous macaques infected with ned/00/01.<br>
Figure 37: Comparison of the use of the hMPV ELISA and the APV inhibition ELISA for the detection of IgG antibodies in human sera.<br><br>
Detailed description<br>
Virus isolation and characterisation<br>
From 1980 till 2000 we found 28 unidentified virus isolates from patients with severe i*espiratory disease. These 28 unidentified virus isolates grew slowly in tMK cells, poorly in VERO cells and A549 cells and could not or only little be propagated in MDCK or chicken embryonated fibroblast cells. Most of these virus isolates induced CPE after three passages on tMK cells, between day ten and fourteen. The CPE was virtually indistinguishable from that caused by hRSV or hPIV in tMK or other cell cultures, characterised by sjTicytium formation after which the cells showed rapid internal disruption, followed by detachment of the cells from the monolayer. The cells usually (sometimes later) displayed CPE after three passages of virus from original material, at day 10 to 14 post inoculation, somewhat later than CPE caused by other viruses such as hRSV or hPIV.<br>
We used the supernatants of infected tA'IK cells for EM analysis which revealed the presence of paramyrovirus-like virus particles ranging from 150 to 600 nanometer, with short envelope projections ranging from 13 to 17 nanaometer. Consistent with the biochemical properties of enveloped viruses such as the Paramyxoviridae, standard chloroform or ether treatment resulted in &gt;10* TCID50 reduction of infectivity for tMK cells. Virus-infected tMK cell culture supernatants did not display heamagglutinating activity with turkey, chicken and guinea pig erythrocytes. During culture, the virus replication appeared to be trypsine dependent on the cells tested. These combined virological data allowed that the newly identified virus was taxonomically classified as a member of the Paramyxoviridae family.<br>
We isolated RNA from tMK cells infected with 15 of the unidentified virus isolates for reverse transcription and polymerase chain reaction (RT-PCR) analyses using primer-sets specific for Paramyxovirinae hPIV 1-4, sendai virus, simian virus type 5, New-Castle disease virus, hESV, morbilli, mumps, Nipah, Hendra, Tupaia and Mapuera viruses. RT-PCR assays were carried out at low stringency in order to detect potentially related viruses and RNA isolated from homologous virus stocks were used as controls. Whereas the available controls reacted positive with the respective virus-specific primers, the newly identified virus isolates did not react with any primer set, indicating the virus was not closely related to the viruses tested.<br><br>
We used two of the virus-infected tMK cell culture supematants to inoculate iinea pigs and ferrets intranasaly. Sera were collected from these animals at day eno, two weeks and three weeks post inoculation. The animals displayed no clinical symptoms but all seroconverted as measured in virus neutralisation (VN) assays and idirect IFA against the homologous viruses. The sera did not react in indirect IFA ith any of the known paramyxoviruses described above and with PVM. Next, we greened the so far unidentified virus isolates using the guinea pig and ferret pre-nd post-infection sera, of which 28 were clearly positive by indirect IFA with the ost-infection sera suggesting they were serological closely related or identical.<br>
RAP PCR<br>
To obtain sequence information on the unknown virus isolates, we used a andom PCR amplification strategy known as RAP-PCR1o. To this end, tMK cells were infected with one of the virus isolates (isolate 00-1) as well as with hPIV-1 which served as a control After both cultures displayed similar levels of CPE, virus n the culture supernatants was purified on continuous 20-60% sucrose gradients. the gradient fractions were inspected for virus-like particles by EM, and RNA was solated from the fraction containing approximately 50% sucrose, in which nucleocapsids were observed. Equivalent amounts of RNA isolated from both virus Tactions were used for RAP-PCR, after which samples were run side by side on a 3% NuSieve agarose gel. Twenty differentially displayed bands specific for the identified virus were subsequently purified from the gel, cloned in plasmid pCR2.1 Invitrogen) and sequenced with vector-specific primers. When we used these lequences to search for homologies against sequences in the Genbank database using he BLAST software (www.ncbi.nlm.nih.gov/BLAST/) 10 out of 20 fragments displayed resemblance to APV/TRTV sequences.<br>
These 10 fragments were located in the genes coding for the nucleoprotein (N; fragment 1 and 2), the matrix protein (M; fragment 3), the fusion protein (F; fragment 4, 5, 6, 7,) and the polymerase protein (L; fragment 8,9,10) (Pig.2). We next designed PCR primers to complete the sequence information for the 3' end of the viral genome based on our RAP PCR fragments as well as published leader and trailer sequences for the Pneumovirinae Â®. Three fragments were amplified, of which fragment A spanned the extreme 3' end of the N open reading frame (ORF), fragment B spanned the phosphoprotein (P) ORF and fragment C closed the gap between the M<br><br>
and F ORFs (Fig. 2). Sequence analyses of these three fragments revealed the absence of NSl and NS2 OEFs at the extreme 3' end of the viral genome and positioning of the F ORF immediately adjacent to the M ORF. This genomic organisation resembles that of the metapneumovirus APV, which is also consistent with the sequence homology. Overall the translated sequences for the N, P, M and F ORFs showed an average of 30-33% homology with members of the genus Pneumovirus and 66-68% with members of the genus Metapneumovirus. For the SH and G OEF's no discernable homology was found with members of either of the genera. The amino acid homologies found for N showed about 40% homology with hRSV and 88% with APV-C, its closest relative genetically, as for example can be deduced by comparing the amino acid sequence of figure 3 with the amino add sequence of the respective N proteins of other viruses. The amino acid sequence for P showed about 25% homology with hRSV and about 66-68% with APV-C, M showed about 36-39% with hRSV and about 87-89% with APV-C, F showed about 40% homology with hRSV and about 81% with APV-C, M2-1 showed about 34-36% homology with pneumoviruses and 84-86 % with APV-C, M2-2 showed 15-17% homology with pneumoviruses and 56% with APV-C and the fragments obtained in L showed an average of 44% with pneumoviruses and 64% with APV-C .<br>
Phylogeny<br>
Although BLAST searches using nucleotide sequences obtained from the unidentified virus isolate revealed homologies primarily with members of the Pneumovirinae, homologies based on protein sequences revealed some resemblance with other paramyxoviruses as well (data not shown). As an indication for the relation between the newly identified virus isolate and members of the Pneumovirinae, phylogenetic trees were constructed based on the N, P, M and F ORFs of these viruses. In all four phylogenetic trees, the newly identified virus isolate was most closely related to APV (Fig.4). From the four serotypes of APV that have been described S APV serotype C, the metapneumovirus found primarily in birds in the USA, showed the closest resemblance to the newly identified virus. It should be noted however, that only partial sequence information for APV serotype D is available.<br>
To determine the relationship of our various newly identifi.ed virus isolates, we constructed phylogenetic trees based on sequence information obtained from eight to<br><br>
nine isolates (8 for F, 9 for N, M and L). To this end, we used ET-PCR with primers designed to amplify short fragments in the N, M, F and L ORFs, that were subsequently sequenced directly. The nine virus isolates that were previously found to be related in serological terms (see above) were also found to be closely related genetically. In fact, all nine isolates were more closely related to one another than to APV. Although the sequence information used for these phylogenetic trees was limited, it appears that the nine isolates can be divided in two groups, with isolate 94-1, 99-1 and 99-2 clustering in one group and the other six isolates (94-2; 93-1; 93-2; 93-3; 93-4; 00-1) in the other (Fig.5).<br>
Serpprevalence<br>
To study the seroprevalence of this virus in the human population, we tested sera from humans in different age categories by indirect IFA using tMK cells infected with one of the unidentified virus isolates. This analysis revealed that 25% of the children between six and twelve months had antibodies to the virus, and by the age of five nearly 100% of the children were seropositive. In total 56 serum samples tested by indirect IFA were tested by VN assay. For 51 (91%) of the samples the results of the VN assay (titre &gt;8) coincided with the results obtained with indirect IFA (titre&gt;32). Four samples that were found positive in IFA, were negative by VN test (titre 
IFA conducted with 72 sera taken from humans in 1958 (ages ranging from 8-99 years) 1.27 revealed a 100% seroprevalence, indicating the virus has been circulating in the human population for more than 40 years. In addition a number of these sera were used in VN assays to confirm the IFA data (table 2).<br>
Genetic analyses of the N, M, P and F genes revealed that MPV has higher sequence homology to the recently proposed genus Metapneumovirinae (average of 63 %) as compared to the genus Pneumovirinae (average of 30 %) and thus demonstrates a genomic organisation similar to and resembling that of APV/TRTV. In contrast to the genomic organisation of the RSVs C3-NS1-NS2-N-P-M-SH-G-F-M2-L-50. metapneumoviruses lack NSl and NS2 genes and have a different positioning of the genes between M and L C3-N-P-M-F-M2-SH-G-L-5'). The lack of ORFs between the M and F genes in our virus isolates and the lack of NSl and NS2 adjacent to to N, and<br><br>
the high amino acid sequence homology found with APV are reasons to propose the classification of MPV isolated from humans as a first member of the Metapneumovirus genus of mammalian, in particular of human origin,<br>
Phylogenetic analyses revealed that the nine MPV isolates from which sequence information was obtained are closely related. Although sequence information was limited, they were in fact more closely related to one another than to any of the avian metapneumoviruses. Of the four serotypes of APV that have been described, serotype C was most closely related to MPV based on the N, P, M and F genes. It should be noted however that for serotype D only partial sequences for the F gene were available from Genbank and for serotype B only M, N and F sequences were available. Our MPV isolates formed two clusters in phylogenetic trees. For both hRSV and APV different genetic and serological subtypes have been described. Whether the two genetic clusters of MPV isolates represent serogical subgroups that are also functionally different remains unknown at presentOur serological surveys showed that MPV is a common human pathogen. The repeated isolation of this virus from clinical samples from children with severe RTI indicates that the chnical and economical impact of MPV may be high. New diagnostic assays based on virus detection and serology will allow a more detailed analysis of the incidence and clinical and economical impact of this viral pathogen.<br>
The slight differences between the IFA and VN results (5 samples) maybe due to the fact that in the IFA only IgG serum antibodies were detected whereas the VN assay detects both classes and sub-classes of antibodies or differences may be due to the differences in sensitivity between both assays. For IFA a cut off value of 16 is used, whereas for VN a cut off value of 8 is used.<br>
On the other hand, differences between IFA versus VN assay may also indicate possible differences between different serotypes of this newly identified virus. Since MPV seems most closely related to APV, we speculate that the human virus may have originated from birds. Analysis of serum samples taken firom humans in 1958 revealed that MPV has been widespread in the human population for more then 40 years indicating that a tentative zoonosis event must have taken place long before 1958.<br><br>
Materials and Methods<br>
Specimen collection<br>
Over the past decades our laboratory has collected nasopharyngeal aspirates from children suffering from RTI, which are routinely tested for the presence of viruses. All nasopharyngeal aspirates were tested by direct immmunofluorescence assays (DIF) using fluorescence labelled antibodies against influenza virus types A, and B, hESV and human parainfluenza virus (hPIV) types 1 to 3. The nasopharyngeal aspirates were also processed for virus isolation using rapid shell vial techniques on various celllines including VERO cells, tertiary cynomolgous monkey kidney (tMK) cells, human endothelial lung (HEL) cells and marbin dock kidney (MDCK) cells. Samples showing cytophatic effects (CPE) after two to three passages, and which were negative in DIF, were tested by indirect immunofluorescence assays (IFA) using virus specific antibodies against influenza virus types A, B and C, hRSV types A and B, measles virus, mumps virus, human parainfluenza virus (hPIV) types 1 to 4, sendai virus, simian virus type 5, and New-Castle disease virus. Although for many cases the aetiological agent could be identified, some specimens were negative for all these viruses tested.<br>
Direct Immunofluorescence Assay (DIF)<br>
Nasopharyngeal aspirate samples from patients suffering from RTI were used for DEF and virus isolation as described14.15 Samples were stored at -70 'C. In brief, nasopharyngeal aspirates were diluted with 5 ml Dulbecco MEM (Bio Whittaker, Walkersville, MD) and thoroughly mixed on a vortex mixer for one minute. The suspension was thus centrifuged for ten minutes at 840 x g. The sediment was spread<br>
on a multispot slide (Nutacon, Leimuiden, The Netherlands), the supernatant was<br>
â€¢<br>
used for virus isolation. After drying, the cells were fixed in aceton for 1 minute at room temperature. After washing the slides were incubated for 15 minutes at 37 oC with commercial available FITC-labelled virus specific anti-sera such as influenza A and B, hRSV and hPIV 1 to 3 (Dako, Glostrup, Denmark). After three washings in PBS and one in tap water, the slides were included in a glycerol/PBS solution (Citifluor, UKC, Canterbury, UK) and covered. The slides were analysed using a Axioscop fluorescence microscope (Carl Zeiss B.V, Weesp, the Netherlands.<br><br>
Virus isolation<br>
For virus isolation tMK cells (RIVM, Bilthoven, The Netherlands) were cultured in 24 well plates containing glass slides (Costar, Cambridge, UK), with the medium described below supplemented with 10% fetal bovine serum (BioWhittaker, Vender, Belgium). Before inoculation the plates were washed with PBS and supplied with Eagle's MEM with Hanks* salt (ICN, Costa mesa, CA) of which half a litre was supplemented with 0.26 gram HaHCOa. 0.025 M Hepes (Biowhittaker), 2 mM L-glutamine (Biowhittaker), 100 units penicilline, 100 Î¼g streptomycine (Biowhittaker), 0.5 gram lactalbumine (Sigma-Aldrich, Zwijndrecht, The Netherlands), 1.0 gram D-glucose (Merck, Amsterdam, The Netherlands), 5.0 gram peptone (Oxoid, Haarlem, The Netherlands) and 0.02% trypsine (life Technologies, Bethesda, MD). The plates were inoculated with supernatant of the nasopharyngeal aspirate samples, 0,2 ml per well in triplicate, followed by centrifuging at 840x g for one hour. After inoculation the plates were incubated at 37 Â°C for a maximum of 14 days changing the medium once a week and cultures were checked daily for CPE. After 14 days cells were scraped from the second passage and incubated 14 days. This step was repeated for the third passage. The glass slides were used to demonstrate the pr6sence of the virus by indirect IFA as described below.<br>
Animal immunisation<br>
Ferret and guinea pig specific antisera for the newly discovered virus were generated by experimental intranasal infection of two specific pathogen free ferrets and two guinea pigs, housed in separate pressurised glove boxes. Two to three weeks later all the animals were bled by cardiac puncture, and their sera were used as reference sera. The sera were tested for all previous described viruses with indirect IFA as described below.<br>
Antigen detection by indirect IFA<br>
We performed indirect IFA on sUdes containing infected tMK cells. After washing with PBS the slides were incubated for 30 minutes at 37 **C with virus specific anti-sera. We used monoclonal antibodies in DIF against influenza A, B and C, hPIV type 1 to 3 and hESV as described above. For hPIV type 4, mumps virus, measles virus, sendai virus, simian virus type 5, New-Castle Disease virus polyclonal antibodies<br><br>
(RIVM) and ferret and guinea pig reference sera were used. After three washings with PBS and one wash with tap water, the slides were stained with a secondary-antibodies directed against the sera used in the first incubation. Secondary antibodies for the polyclonal anti sera were goat-anti-ferret (KPL, Gvdlford, UK, 40 fold diluted), mouse-anti-rabbit (Dako, Glostrup, Denmark, 20 fold diluted), rabbit-anti-chicken (KPL, 20 fold dilution) and mouse-anti-guinea pig (Dako, 20 fold diluted). Slides were processed as described for DEF.<br>
Detection of antibodies in humans by indirect IFA<br>
For the detection of virus specific antibodies, infected tMK cells were fixed with cold acetone on coverslips, washed with PBS and stained with serum samples at a 1 to 16 dilution. Subsequently, samples were stained with FITC-labelled rabbit anti human antibodies 80 times diluted in PBS (Dako). Slides were processed as described above.<br>
Virus culture of MPV<br>
Sub-confluent mono-layers of tMK cells in media as described above were inoculated with supernatants of samples that displayed CPE after two or three passages in the 24 well plates. Culttures were checked for CPE daily and the media was changed once a week. Since CPE differed for each isolate, all cultures were tested at day 12 to 14 with indirect IFA using ferret antibodies against the new virus-isolate. Positive cultures were freeze-thawed three times, after which the supematants were clarified by low-speed centrifugation, aliquoted and stored frozen at -70 oC. The 50% tissue culture infectious doses (TCID50) of virus in the culture supernatants were determined as described16.<br>
Virus neutralisation assay<br>
VN assays were performed with serial two-fold dilutions of human and animal sera starting at an eight-fold dilution. Diluted sera were incubated for one hour with 100 TCID50 of virus before inoculation of tMK cells grown in 96 well plates, after which the plates were centrifuged at 840 x g. The media was changed after three and six days and IFA was conducted with ferret antibodies against MPV 8 days after inoculation. The VN titre was defined as the lowest dilution of the serum sample resulting in negative IFA and inhibition of CPE in cell cultures.<br><br>
Virus characterisation<br>
Haemagglutination assays and chloroform sensitivity tests were performed as describedÂ®'. For EM analyses, virus was concentrated from infected cell culture supernatants in a micro-centrifuge at 4 Â°C at 17000 x g, after which the pellet was resuspended in PBS and inspected by negative contrast EM. For RAP-PCR, virus was concentrated from infected tMK cell supernatants by ultra-centrifugation on a 60% sucrose cussion (2 hours at 150000 x g, 4 Â°C). The 60% sucrose interphase was subsequently diluted with PBS and layered on top of a 20-60% continuous sucrose gradient which was centrifuged for 16 hours at 275000 x g at 4 Â°C. Sucrose gradient fractions were inspected for the presence of virus-like particles by EM and poly-acrylamide gel electrophoresis followed by silver staining. The approximately 50% sucrose fractions that appeared to contain nucleocapsids were used for RNA isolation and RAP-PCR.<br>
RNA isolation<br>
RNA was isolated from the supernatant of infected cell cultures or sucrose gradient fractions using a High Pure RNA Isolation kit according to instructions from the manufacturer (Roche Diagnostics, Almere, The Netherlands).<br>
RT-PCR<br>
Virus-specific oligonucleotide sequences for RT-PCR assays on known paramyxoviruses are described in addenda 1. A one-step RT-PCR was performed in 50Î¼l reactions containing 50 mM Tris.HCl pH 8.5, 50 mM NaCl, 4 mM MgCl2, 2 mM dithiotreitol, 200Î¼M each dNTP, 10 units recombinant RNAsin (Promega, Leiden, the Netherlands), 10 units AMV RT (Promega, Leiden, The Netherlands), 5 units Amplitaq Gold DNA polymerase (PE Biosystems, Nieuwerkerk aan de Ijssel, The Netherlands) and 5 pi RNA. Cycling conditions were 45 min. at 42  Â°C and 7 min. at 95 Â°C once, 1 min at 95 Â°C, 2 min. at 42 'C and 3 min. at 72  Â°C repeated 40 times and 10 min. at 72  Â°C once.<br>
RAP'PCR<br>
RAP-PCR was performed essentially as described10. The oligonucleotide sequences are described in addenda 2. For the RT reaction, 2Î¼l RNA was used in a 10Î¼l<br><br>
reaction containing 10 ng/Î¼l oligonucleotide, 10 mM dithiotreitol, 500 Î¼m each dNTP, 25 mM Tris-HCl pH 8,a, 75 mM KCl and 3 mM MgCl2. The reaction mixture was incubated for 5 min. at 70 Â°C and 5 min. at 37 oC, after which 200 units Superscript RT enzyme (LifeTechnologies) were added. The incubation at 37 oC was continued for 55 min. and the reaction terminated by a 5 min. incubation at 72 oC. The RT mixture was diluted to give a 50 Î¼l PCR reaction containing 8 ng/Î¼l oHgonucleotide, 300 Î¼m each dNTP, 15 mM Tris-HCL pH 8.3, 65 mM KCl, 3.0 mM MgCl2 and 5 units Taq DNA polymerase (PE Biosystems). Cycling conditions were 5 min. at 94 Â°C, 5 min. at 40 Â°C and 1 min. at 72 oC once, followed by 1 min. at 94oC, 2 min. at 56 Â°C and 1 min. at 72 Â°C repeated 40 times and 5 min. at 72oC once. After RAP-PCR, 15 ^1 the RT-PCR products were run side by side on a 3% NuSieve agarose gel (FMC BioProducta, Heerhugowaard, The Netherlands). Differentially displayed fragments specific for MPV were purified from the gel with Qiaquick Gel Extraction kit (Qiagen, Leusden, The Netherlands) and cloned in pCR2.1 vector (Invitrogen, Groningen, The Netherlands) according to instructions from the manufacterer.<br>
Sequence analysis<br>
RAP-PCR products cloned in vector pCR2.1 (Invitrogen) were sequenced with M13-specific oligonucleotides. DNA fragments obtained by RT-PCR were purified from agarose gels using Qiaquick Gel Extraction kit (Qiagen, Leusden, The Netherlands), and sequenced directly with the same oligonucleotides used for PCR. Sequence analyses were performed using a Dyenamic ET terminator sequencing kit (Amersham Pharmacia Biotech, Roosendaal, The Netherlands) and an ABI 373 automatic DNA sequencer (PE Biosystem). All techniques were performed according to the instructions of the manufacturer.<br><br>
Generating genomic fragments of MPV by RT-PCR<br>
To generate PCR fragments spanning gaps A, B and C between the RAP-PCR fragments (Fig.2) we used RT-PCR assays as described before on RNA isolated from virus isolate 00-1. The following primers were used; For fragment A: TRl designed in the leader: (5'-<br>
AAAGAATTCACGAGAAAAAAACGC-3') and Nl designed at the 3'end of the RAP-PCR fragments obtained in N (5'-CTGTGGTCTCTAGTCCCACTTC-3') For fragment B: N2 designed at the 5'end of the RAP-PCR fragments obtained in N: (5'-CATGCAAGCTTATGGGGC-3') and Ml designed at the 3'end of the RAP-PCR fragments obtained in M: (5'-CAGAGTGGTTATTGTCAGGGT-3'). For fragment C: M2 designed at the 5' end of the RAP-PCR fragment obtained in M: (5'-GTAGAACTAGGAGCATATG-3') and Fl designed at the 3'end of the RAP-PCR fragments obtained in F: (5'-TCCCCAATGTAGATACTGCTTC-3'). Fragments were purified from the gel, cloned and sequenced as described before.<br>
RT-PCR for diagnosing MPV.<br>
For the amplification and sequencing of parts of the N, M, F and L ORFs of nine of the MPV isolates, we used primers N3 (5'-GCACTCAAGAGATACCCTAG -3') and N4 (5'-AGACTTTCTGCTTTGCTGCCTG-3'), amplifying a 151 nucleotide fragments, M3 (5'-CCCTGACAATAACCACTCTG.3') and M4 (6'-GCCAACTGATrTGGCTGAGCTC-3') amplifying a 252 nucleotide firagment, F7 (5'-TGCACTATCTCCTCTTGGGGCTrTG-3') and F8 (5'-<br>
TCAAAGCTGCTTGACACTGGCC-3') amplifying a 221 nucleotide firagment and L6 (5'-CATGCCCACTATAAAAGGTCAG-30 and L7 (5'-CACCCCAGTCTTTCTTGAAA-3') amplifying a 173 nucleotide firagment respectively. RT-PCR, gel purification and direct sequencing were performed as described above. Furthermore, probes used were:<br>
Probe used in M: 5'-TGC TTG TAG TTC CCA AAG-3' Probe used in N: 5'-TAT TTG AAC AAA AAG TGT-3' Probe used in L: 5'-TGGTGTGGGATATTAACAG-3'<br><br>
Phylogenetic analyses<br>
For all phylogenetic trees, DNA sequences were alligned using the ClustalW software package and maximum likelihood trees were generated using the DNA-ML software package of the Phylip 3.5 program using 100 bootstraps and 3 jumbles15. Previously published sequences that were used for the generation of phylogenetic trees are available from Genbank under accessions numbers : For all ORFs: hESV: NC001781; bRSV: NC001989; FortheFORF: PVM, D11128; APV-A, D00850;APV-B.Y14292; APV-C AF187152; For the N ORF: PVM, D10331; APV-A, U39295; APV-B, U39296; APV-C, AF176590; For the M ORF: PMV,U66893; APV-A, X58639; APV-B, U37586; APV-C, AF262571; For the P ORF: PVM, 09649; APV-A, U22110, APV-C, AF176591. Phylogenetic analyses for the nine different virus isolates of MPV were performed with APV strain C as outgroup.<br>
Abbreviations used in figures: hRSV: human RSV; bRSV: bovine RSV; PVM: pneumonia virus of mice; APV-A, B,and C: avian pneumovirus typ A, B and C.<br>
Examples of methods to identify MPV<br>
Specimen collection<br>
In order to find virus isolates nasopharyngeal aspirates, throat and nasal swabs, broncheo alveolar lavages preferably from mammals such as humans, carnivores (dogs, cats, musteliits, seals etc.), horses, ruminants (cattle, sheep, goats etc.), pigs, rabbits, birds (poultry, ostriches, etc) should be examined. From birds cloaca swabs and droppings can be examined as welL Sera should be collected for immunological assays, such as ELISA and virus neutralisation assays. Collected virus specimens were diluted with 5 ml Dulbecco MEM medium (BioWhittaker, Walkersville, MD) and thoroughly mixed on a vortex mixer for one minute. The suspension was thus centrifuged for ten minutes at 840 x g. The sediment was spread on a multispot slide (Nutacon, Leimuiden, The Netherlands) for immunofluorescence techniques, and the supernatant was used for virus isolation.<br><br>
Virus isolation<br>
For virus isolation tMK cells (RIVM, Bilthoven, The Netherlands) were cultured in 24 well plates containing glass slides (Costar, Cambridge, UK), with the medium described below supplemented with 10% fetal bovine serum (BioWhittaker, Vender, Belgium). Before inoculation the plates were washed with PBS and supplied with Eagle's MEM with Hanks' salt (ICN, Costa mesa, CA) supplemented with 0.52/liter gram NaHCO3. 0.025 M Hepes (Biowhittaker), 2 mM L-glutamine (Biowhittaker), 200 units/liter penicilline, 200 Î¼g/liter streptomycine (Biowhittaker), Igram/liter lactalbumine (Sigma-Aldrich, Zwijndrecht, The Netherlands), 2.0 gram/liter D-glucose (iVIerck, Amsterdam, The Netherlands), 10 gram/liter peptone (Oxoid, Haarlem, The Netherlands) and 0.02% trypsine (Life Technologies, Bethesda, MD). The plates were inoculated with supernatant of the nasopharyngeal aspirate samples, 0,2 ml per well in triplicate, followed by centrifuging at 840x g for one hour. After inoculation the plates were incubated at 37 Â°C for a maximiim of 14 days changing the medium once a week and cultures were checked daily for CPE. After 14 days, cells were scraped from the second passage and incubated for another 14 days. This step was repeated for the third passage. The glass slides were used to demonstrate the presence of the virus by indirect IFA as described below.<br>
CPE was generally observed after the third passage, at day 8 to 14 depending on the isolate. The CPE was virtually indistinghuisable from that caused by hESV or hPIV in tMK or other cell cultures. However, hRSV induces CPE starting around day 4. CPE was characterised by syncytia formation, after which the cells showed rapid internal disruption, followed by detachment of cells from the monolayer. For some isolates CPE was difficult to observe, and IFA was used to confirm the presence of the virus in these cultures.<br>
Virus culture of MPV<br>
Sub-confluent monolayers of tMK cells in media as described above were inoculated with supernatants of samples that displayed CPE after two or three passages in the 24 well plates. Cultures were checked for CPE daily and the media was changed once a week. Since CPE differed for each isolate, all cultures were tested at day 12 to 14 with indirect IFA using ferret antibodies against the new virus isolate. Positive cultures were freeze-thawed three times, after which the supernatants were clarified<br><br>
by low-speed centrifugation, aliquoted and stored frozen at -70 oC. The 50% tissue culture infectious doses (TCID50) of virus in the culture supernatants were determined following established techniques used in the field18.<br>
Virus characterisation<br>
Haemagglutination assays and chloroform sensitivity tests were performed following well established and described techniques used in the field. For EM analyses, virus was concentrated from infected cell culture supernatants in a micro-centrifuge at 4 C at 17000 X g, after which the pellet was resuspended in PBS and inspected by negative contrast EM.<br>
Antigen detection by indirect IFA<br>
Collected specimens were processed as described and sediment of the samples was spread on a multispot slide. After during, the cells were fixed in aceton for 1 minute at room temperature.<br>
Alternatively, virus was cultured on tMK cells in 24 well slides containing glass slides. These glass slides were washed with PBS and fixed in aceton for 1 minute at room temperature.<br>
After washing with PBS the slides were incubated for 30 minutes at 37o'C with polyclonal antibodies at a dilution of 1:50 to 1:100 in PBS. We used immunised ferrets and guinea pigs to obtain polyclonal antibodies, but these antibodies can be raised in various animals, and the working dilution of the polyclonal antibody can vary for each immunisation. After three washes with PBS and one wash with tap water, the slides were incubated at 37Â°C for 30 minutes with FITC labeled goat-anti-ferret antibodies (KPL, Guilford, UK, 40 fold diluted). After three washes in PBS and one in tap water, the shdes were included in a glycerol/PBS solution (Citifluor, UKC, Canterbury, UK) and covered. The shdes were analysed using an Axioscop fluorescence microscope (Carl Zeiss B.V., Weesp, the Netherlands). Detection of antibodies in humans, mammals, ruminants or other animals by indirect IFA<br>
For the detection of virus specific antibodies, infected tMK cells with MPV were fixed with acetone on coverslips (as described above), washed with PBS and incubated 30 minutes at 37oC with serum samples at a 1 to 16 dilution. After two washes with PBS and one with tap water, the shdes were incubated 30 minutes at 37oC with FITC-<br><br>
labelled secondary antibodies to the species used (Dako). Slides were processed as described above.<br>
Antibodies can be labelled directly with a fluorescent dye, which will resvdt in a direct immune fluorescence assay. FITC can be replaced with any fluorescent dye.<br>
Animal immunisation<br>
Ferret and guinea pig specific antisera for the newly discovered virus were generated by experimental intranasal infection of two specific pathogen free ferrets and two guinea pigs, housed in separate pressurised glove boxes. Two to three weeks later the animals were bled by cardiac puncture, and their sera were used as reference sera. The sera were tested for all previous described viruses with indirect IFA as described below. Other animal species are also suitable for the generation of specific antibody preparations and other antigen preparations may be used.<br>
Virus neutralisation assay (VN assay)<br>
VN assays were performed with serial two-fold dilutions of human and animal sera starting at an eight-fold dilution. Diluted sera were incubated for one hour with 100 TCID50 of virus before inoculation of tMK cells grown in 96 weU plates, after which the plates were centrifuged at 840 x g. The same culture media as described above was used. The media was changed after three and six days, and after 8 days IFA was performed (see above). The VN titre was defined as the lowest dilution of the serum sample resulting in negative IFA and inhibition of CPE in cell cultures.<br><br>
RNA isolation<br>
RNA was isolated from the supernatant of infected cell cultures or sucrose gradient fractions using a High Pure RNA Isolation kit according to instructions from the manufacturer (Roche Diagnostics, Almere, The Netherlands). RNA can also be isolated following other procedures known in the field {Current Protocols in Molecular Biology).<br>
RT-PCR<br>
A one-step RT-PCR was performed in 50 jxl reactions containing 50 mM Tris.HCl pH<br>
8.5, 50 mM NaCl, 4 mM MgCb, 2 mM dithiotreitol, 200 jiM each dNTP, 10 units<br>
recombinant RNAsin (Promega, Leiden, the Netherlands), 10 units AMY RT<br>
(Promega, Leiden, The Netherlands), 5 units Amplitaq Gold DNA polymerase (PE<br>
Biosystems, Nieuwerkerk aan de Ijssel, The Netherlands) and 5 Î¼l RNA. Cycling<br>
conditions were 45 min. at 42 Â°C and 7 min. at 95Â°C once, 1 min at 95 Â°C, 2 min. at<br>
42 Â°C and 3 min. at 72 Â°C repeated 40 times and 10 min. at 72Â°C once.<br>
Primers used for diagnostic PCR:<br>
In the nucleoprotein: N3 (5'-GCACTCAAGAGATACCCTAG -3') and N4 (5'-<br>
AGACTTTCTGCTTTGCTGCCTG-3'). amplifying a 151 nucleotide fragment.<br>
In the matrixprotein: M3 (5'-CCCTGACAATAACCACTCTG-30 and M4 (5*-<br>
GCCAACTGATTTGGCTGAGCTC-30 amplifying a 252 nucleotide fragment<br>
In the polymerase protein: L6 (5'-CATGCCCACTATAAAAGGTCAG-3') and L7 (5'-<br>
CACCCCAGTCTTTCTTGAAA-3') amplifying a 173 nucleotide fragment.<br>
Other primers can be designed based on MPV sequences, and different buffers and<br>
assay conditions may be used for specific purposes.<br>
Sequence analysis<br>
Sequence analyses were performed using a Dyenamic ET terminator sequencing kit (Amersham Pharmacia Biotech, Roosendaal, The Netherlands) and an ABI 373 automatic DNA sequencer (PE Biosystem). All techniques were performed according to the instructions of the manufacturer. PCR fragments were sequenced directly with the same oligonucleotides used for PCR, or the fragments were purified from the gel with Qiaquick Gel Extraction kit (Qiagen, Leusden, The Netherlands) and cloned in<br><br>
pCR2.1 vector (Invitrogen, Groningen, The Netherlands) according to instructions from the manufacturer and subsequently sequenced with Ml3-specific oligonucleotides.<br>
Oligonucleotides used for analysing the 3'end of the genome (absence of NSl/NS2), Primer TRl (5*-AAAGAATTCACGAGAAAAAAACGC-30 was designed based on published sequences of the trailer and leader for hRSV and APV, published by Randhawa (1997) and primer Nl (5-CTGTGGTCTCTAGTCCCACTTC-3') was designed based on obtained sequences in the N protein. The RT-PCR assay and sequencing was performed as described above.<br>
The RT-PCR gave a product of appi'oximately 500 base pairs which is to small to contain information for two ORFS, and translation of these sequences did not reveal anORF.<br>
Detection of antibodies in humans, mammals, ruminants or other animals by ELISA<br>
In Paramyxoviridae, the N protein is the most abundant protein, and the immune response to this protein occurs early in infection. For these reasons, a recombinant source of the N proteins is preferably used for developing an ELISA assay for detection of antibodies to MPV. Antigens suitable for antibody detection include any MPV protein that combines with any MPV-specific antibody of a patient exposed to or infected with MPV virus. Preferred antigens of the invention include those that predominantly engender the immune response in patients exposed to MPV, which therefore, typically are recognised most readily by antibodies of a patient. Particularly preferred antigens include the N, F and G proteins of MPV. Antigens used for immunological techniques can be native antigens or can be modified versions thereof. Well known techniques of molecular biology can be used to alter the amino acid sequence of a MPV antigen to produce modified versions of the antigen that may be used in immunologic techniques.<br>
Methods for cloning genes, for manipulating the genes to and from expression vectors, and for expressing the protein encoded by the gene in a heterologous host are well-known, and these techniques can be used to provide the expression vectors, host cells, and the for expressing cloned genes encoding antigens in a host to produce<br><br>
recombinant antigens for use in diagnostic assays. See for instance; molecular cloning, A laboratory manual and Current Protocols in Molecular Biology, A variety of expression systems may be used to produce MPV antigens. For instance, a variety of expression vectors suitable to produce proteins in E.Coli, B,subtilis, yeast, insect cells and mammalian cells have been described, any of which might be used to produce a MPV antigen suitable to detect anti-MPV antibodies in exposed patients.<br>
The baculoviruB expression system has the advantage of providing necessary processing of proteins, and is therefor preferred. The system utilizes the polyhedrin promoter to direct expression of MPV antigens. (Matsuura et al. 1987, J.Gen.Virol. 68: 1233-1250).<br>
Antigens produced by recombinant baculo-viruses can be used in a variety of immunological assays to detect anti-MPV antibodies in a patient. It is well established, that recombinant antigens can be used in place of natural virus in practically any immunological assay for detection of virus specific antibodies. The assays include direct and indirect assays, sandwich assays, solid phase assays such as those using plates or beads among others, and liquid phase assays. Assays suitable include those that use primary and secondary antibodies, and those that use antibody binding reagents such as protein A. Moreover, a variety of detection methods can be used in the invention, including colorimetric, fluorescent, phosphorescent, chemUuminescent, luminescent and radioactive methods.<br>
Example 1 of indirect anti-MPV IgG EIA using recombinant N protein<br>
An indirect IgG EIA using a recombinant N protein (produced with recombinant baculo-virus in insect (Sf9) cells) as antigen can be performed. For antigen preparation, Sf9 cells are infected with the recombinant baculovirus and harvested 3-7 days post infection. The cell suspension is washed twice in PBS, pH 7.2, adjusted to a cell density of 5.0X 10ceUs/ml, and freeze-thawed three times. Large cellular debris is pelleted by low speed centrifugation (500 x g for 15 min.) and the supernatant is collected and stored at -70Â°C until use. Uninfected cells are processed similarly for negative control antigen.<br>
100 Î¼1 of a freeze-thaw lysate is used to coat microtiter plates, at dilutions ranging from 1:50 to 1:1000. An uninfected cell lysate is run in duplicate wells and serves as a<br><br>
negative control. After incubation overnight, plates are washed twice with PBS/0.05%Tween,Test sera are diluted 1:50 to 1:200 in ELISA buffer (PBS. supplemented to 2% with normal goat sera, and with 0.5% bovine serum albumine and 0.1% milk), followed by incubation wells for 1 hour at 37Â°C.<br>
Plates are washed two times with PBS/0.05%Tween. Horseradish peroxidase labelled goat anti-human (or against other species) IgG, diluted 1:3000 to 1:5000 in ELISA buffer, added to wells, and incubated for 1 hour at 37'o, The plates are then washed t\vo times with PBS/0.05%Tweeri and once with tap water, incubated for 15 minutes at room temperature with the enzyme substrate TMB, 3,3',5,5' tetramethylbenzidine, such as that obtained from Sigma, and the reaction is stopped with 100 Î¼l of 2 M phosphoric acid. Colorimetric readings are measured at 450 nm using an automated microtiter plate reader.<br>
Example 2: Capture anti-MPV IgM EIA using a recombinant nucleoprotein<br>
A capture IgM EIA using the recombinant nucleoprotein or any other recombinant protein as antigen can be performed by modification of assays as previously described by Erdman et al (1990) J.CUn.Microb. 29: 1466-1471.<br>
Affinity purified anti-human IgM capture antibody (or against other species), such as that obtained from Dako, is added to wells of a microtiter plate in a concentration of 250 ng per well in 0.1 M carbonate buffer pH 9.6. After overnight incubation at room temperature, the plates are washed two times with PBS/0.05% Tween. 100Î¼l of test serum diluted 1:200 to.l:1000 in ELISA buffer is added to triplicate wells and incubated for 1 hour at 37Â°C. The plates are then washed two times with in PBS/0.05%Tween.<br>
The freeze-thawed (infected with recombinant virus) Sf21 cell lysate is diluted 1:100 to 1: 500 in ELISA buffer is added to the wells and incubated for 2 hours at 37'C. Uninfected cell lysate serves as a negative control and is rim in duplicate wells. The plates are then washed three times in PBS/0.05% Tween and incubated for 1 hour at 37oC with 100 Î¼l of a polyclonal antibody against MPV in a optimal dilution in ELISA buffer. After 2 washes with PBS/0.05% Tween, the plates are incubated<br><br>
with horseradish peroxide labeled secondary antibody (such as rabbit anti ferret), and the plates are incubated 20 minutes at 37oC.<br>
The plates are then washed five times in PBS/0/05% Tween, incubated for 15 minutes at room temperature with the enzyme substrate TMB, 3,3',5,5' tetramethylbenzidine, as, for instance obtained from "Sigma", and the reaction is stopped with 100 Î¼l of 2M phosphoric acid. Colormetric readings are measured at 450 nm using automated microtiter plate reader.<br>
The sensitivities of the capture IgM EIAs using the recombinant nucleoprotein (or other recombinant protein) and whole MPV virus are compared using acute-and convalescent-phase serum pairs form persons with clinical MPV virus infection. The specificity of the recombinant nucleoprotein capture EIA is determined by testing serum specimens from healthy persons and persons with other paramyxovirus infections.<br>
Potential for EIAs for using recombinant MPV fusion and glycoprotein proteins produced by the baculovirus expression.<br>
The glycoproteins G and F are the two transmembraneous envelope glycoproteins of the MPV virion and represent the major neutralisation and protective antigens. The expression of these glycoproteins in a vector virus system sych as a baculovirus system provides a source of recombinant antigens for use in assays for detection of MPV specific antibodies. Moreover, their use in combination with the nucleoprotein, for instance, further enhances the sensitivity of enzyme immunoassays in the detection of antibodies against MPV.<br>
A variety of other immunological assays {Current Protocols in Immunology) may be used as alternative methods to those described here.<br>
In order to find virus isolates nasopharyngeal aspirates, throat and nasal swabs, broncheo alveolar lavages and throat swabs preferable from but not limited to humans, carnivores (dogs, cats, seals etc.), horses, ruminants (cattle, sheep, goats etc.), pigs, rabbits, birds (poultry, ostridges, etc) can be examined. From birds, cloaca and intestinal swabs and droppings can be examined as well. For all samples,<br><br>
serology (antibody and antigen detection etc.), virus isolation and nucleic acid detection techniques can be performed for the detection of virus. Monoclonal antibodies can be generated by immunising mice (or other animals) with purified MPV or parts thereof (proteins, peptides) and subsequently using established hybridoma technology (Current protocols in Immunology). Alternatively, phage display technology can be used for this purpose (Current protocols in Immunology). Similarly, polyclonal antibodies can be obtained from infected humans or animals, or from immunised humans or animals (Current protocols in Immunology),<br>
The detection of the presence or absence of NSl and NS2 proteins can be performed using western-blotting, IFA, immuno precipitation techniques using a variety of antibody preparations. The detection of the presence or absence of NSl and NS2 genes or homologues thereof in virus isolates can be performed using PCR with primer sets designed on the basis of known NSl and/or NS2 genes as well as with a variety of nucleic acid hybridisation techniques.<br>
To determine whether NSl and NS2 genes are present at the 3' end of the viral genome, a PCR can be performed with primers specific for this 3' end of the genome. In our case, we used a primer specific for the 3* untranslated region of the viral genome and a primer in the N ORF. Other primers may be designed for the same purpose. The absence of the NS1/NS2 genes is revealed by the length and/or nucleotide sequence of the PCR product. Primers specific for NSl and/or NS2 genes may be used in combination with primers specific for other parts of the 3' end of the viral genome (such as the untranslated region or N, M or F ORFs) to allow a positive identification of the presence of NSl or NS2 genes. In addition to PCR, a variety of techniques such as molecular cloning, nucleic acid hybridisation may be used for the same purpose.<br>
Example 3: Different serotypes/subgroups of MPV<br>
Two potential genetic clusters are identified by analyses of partial nucleotide sequences in the N, M, F and L ORFs of 9 virus isolates, 90 -100% nucleotide identity was observed within a cluster, and 81-88% identity was observed between the clusters. Sequence information obtained on more virus isolates confirmed the existence of two genotypes. Virus isolate ned/00/01 as prototype of cluster A, and<br><br>
virus isolate ned/99/01 as prototype of cluster B have been used in cross neutralization assays to test whether the genotypes are related to different serotypes or subgroups.<br>
Results<br>
Using RT-PCR assays with primers located in the polymerase gene, we identified 30 additional virus isolates from nasopharyngeal aspirate samples. Sequence information of parts of the matrix and polymerase genes of these new isolates together with those of the previous 9 isolates were used to construct phylogenetic trees (Figure 16). Analyses of these trees confirmed the presence of two genetic clusters, with virus isolate ned/00/00-1 as the prototype virus in group A and virus isolate ned/99/01 as the prototype virus in group B. The nucleotide sequence identity within a group was more than 92%, while between the clusters the identity was 81-85%.<br>
Virus isolates ned/00/01 and ned/99/01 have been used to inoculate ferrets to raise virus-specific antisera. These antisera were used in virus neutralization assays with both viruses.<br><br><br>
For isolate 00-1 the titer differs 32 (64/2) fold For isolate 99-1 the titer differs 16 (64/4) fold<br>
In addition, 6 guinea pigs have been inoculated with either one of the viruses (ned/00/01 and ned/99/01). RT-PCR assays on nasopharyngeal aspirate samples showed virus replication from day 2 till day 10 post infection. At day 70 post infection the guinea pigs have been challenged with either the homologous or the heterologous virus, and for in all four cases virus replication has been noticed.<br><br>
Virus neutralization assays with anti sera after the first challenge showed essentially the same results as in the VN assays performed with the ferrets (&gt; 16-fold difference in VN titer).<br>
The results presented in this example confirm the existence of two genotypes, which correspond to two serotypes of MPV, and show the possibility of repeated infection with heterologous and homologous virus<br><br>
Example 4: Further sequence determination<br>
This example describes the further analysis of the sequences of MPV open reading frames (ORFs) and intergenic sequences as well as partial sequences of the genomic termini.<br>
Sequence analyses of the nucleoprotein (N), phosphoprotein (P), matrixprotein (M) and fusion protein (F) genes of IVIPV revealed the highest degree of sequence homology with APV serotype C, the avian pneumovirus found primarily in birds in the United States. These analyses also revealed the absence of non-structural proteins NSl and NS2 at the 3'end of the viral genome and positioning of the fusion protein immediately adjacent to the matrix protein. Here we present the sequences of the 22K (M2) protein, the small hydrophobic (SH) protein, the attachment (G) protein and the polymerase (L) protein genes, the intergenic regions and the trailer sequence. In combination with the sequences described previously the sequences presented here complete the genomic sequence of MPV with the exception of the extreme 12-15 nucleotides of the genomic termini and establish the genomic organisation of MPV. Side by side comparisons of the sequences of the MPV genome with those of APV subtype A, B and C, RSV subtype A and B, PVM and other paramyxoviruses provides strong evidence for the classification of MPV in the Metapneumovirus genus.<br>
Be suits<br>
Sequence strategy<br>
MPV isolate 00-1 (van den Hoogen et al., 2001) was propagated in tertiary monkey kidney (tMK) cells and RNA isolated from the supernatant 3 weeks after inoculation was used as template for RT-PCR analyses. Primers were designed on the basis of the partial sequence information available for MPV 00-1 (van den Hoogen et al., 2001) as well as the leader and trailer sequences of APV and BSV (Raudhawa et al, 1997; Mink et aL, 1991). Initially, fragments between the previously obtained products,<br><br>
ranging in size from 500 bp to 4 Kb in length, were generated by RT-PCR amplification and sequenced directly. The genomic sequence was subsequently confirmed by generating a series of overlapping RT-PCE fragments ranging in size from 500 to 800 bp that represented the entire MPV genome. For all PCR fragments, both strands were sequenced directly to minimize amplification and sequencing errors. The nucleotide and amino acid sequences were used to search for homologies with sequences in the Genbank database using the BLAST software (www.ncbi.nlm.nih.gov/BLAST). protein names were assigned to open reading frames (ORFs) based on homology with known viral genes as well as their location in the genome. Based on this information, a genomic map for MPV was constructed (Figure 7). The MPV genome is 13378 nucleotides in length and its organization is similar to the genomic organization of APV. Below, we present a comparison between the ORFs and non-coding sequences of MPV and those of other paramyxoviruses and discuss the important similarities and differences.<br>
The nucleoprotein (N) gene<br>
As shown, the first gene in the genomic map. of MPV codes for a 394 amino acid (aa) protein and shows extensive homology with the N protein of other pneumioviruses. The length of the N ORF is identical to the length of the N ORP of APV-C (Table 5) and is smaller than those of other paramyxoviruses (Barr et aL, 1991). Analysis of the amino acid sequence revealed the highest homology with APV-C (88%), and only 7-11% with other paramyxoviruses (Table 6).<br>
Barr et al (1991) identified 3 regions of similarity between viruses belonging to the order Mononegavirales: A, B and C (Figure 8). Although similarities are highest within a virus family, these regions are highly conserved between virus familys. In all three regions MPV revealed 97% aa sequence identity with APV-C, 89% with APV-B, 92% with APV-A, and 66-73% with RSV and PVM. The region between aa residues 160 and 340 appears to be highly conserved among metapneumoviruses and to a somewhat lesser extent the Pneumovirinae (Miyahara et al, 1992; Li et al, 1996; Barr et aL, 1991). This is in agreement with MPV being a metapneumovirus, showing 100% similarity with APV C.<br>
The phosphoprotein (P) gene<br><br>
The second ORF in the genome map codes for a 294 aa protein which shares 68% aa sequence homology with the P protein of APV-C, and only 22-26% with the P protein of RSV (Table 6). The P gene of MPV contains one substantial ORF and in that respect is similar to P from many other paramyxoviruses (Reviewed in Lamb and Kolakofsky, 1996; Sedlmeier et al, 1998).<br>
In contrast to APV A and B and PVM and similar to RSV and APV-C the MPV P ORF lacks cysteine residues. ling (1995) suggested that a region of high similarity between all pneumoviruses (aa 185-241) plays a role in either the RNA synthesis process or in maintaining the structural integrity of the nucleocapsid complex. This region of high similarity is also found in MPV (Figure 9) especifically when conservative substitutions are taken in accoimt, showing 100% similarity with APV-C, 93 % with APV-A and B, and approximately 81% with RSV. The C-terminus of the MPV P protein is rich in glutamate residues as has been described for APVs (Ling et aU 1995).<br>
The matrix (M) protein gene<br>
The third ORF of the MPV genome encodes a 254 aa protein, which resembles the M ORFs of other pneuraoviruses. The M ORF of MPV has exactly the same size as the M ORFs of other metapneumoviruses (Table 5) and shows high aa sequence homology with the matrix proteins of APV (78-87%), lower homology with those of RSV and PVM (37-38%) and 10% or less homology with those of other paramyxoviruses (Table<br>
6).<br>
Easton (1997) compared the sequences of matrix proteins of all pneumoviruses and found a conserved heptadpeptide at residue 14 to 19 that is also conserved in MPV (Figure 10). For RSV, PVM and APV small secondary ORFs within or overlapping with the major ORF of M have been identified (52 aa and 51 aa in bRSV, 75 aa in RSV, 46 aa in PVM and 51 aa in APV) (Yu et al, 1992; Easton et al, 1997; Samal et al., 1991; Satake et al, 1984). We noticed two small ORFs in the M ORF of MPV. One small ORF of 54 aa residues was found within the major M ORF (fragment 1, Figure 7), starting at nucleotide 2281 and one small ORF of 33 aa residues was found overlapping with the major ORF of M starting at nucleotide 2893 (fragment 2, Figure 7). Similar to the secondary ORFs of RSV and APV there is no significant homology between these secondary ORFs and secondary ORFs of the other pneumoviruses, and apparent start or stop signals are lacking. In addition, evidence for the synthesis of<br><br>
proteins corresponding to these secondary ORFs of APV and RSV has not been reported.<br>
The fusion protein (F) gene<br>
The F ORF of MPV is located adjacent to the M ORF, which is characteristic for members of the Metapneumovirus genus. The F gene of MPV encodes a 539 aa protein, which is two aa residues longer than F of APV-C (Table 5). Analysis of the aa sequence revealed 81% homology with APV-C, 67% with APV-A and B, 33-39% with pneumovirus F proteins and only 10-18% with other paramyxoviruses (Table 6). One of the conserved features among F proteins of paramyxoviruses, and also seen in MPV is the distribution of cysteine residues (Morrison, 1988; Yu et a/., 1991). The metapneumoviruses share 12 cysteine residues in Fl (7 are conserved among all paramyxoviruses), and two in F2 (1 is conserved among all paramyxoviruses). Of the 3 potential N-linked glycosylation sites present in the F ORF of MPV, none are shared with RSV and two (position 74 and 389) are shared with APV The third, unique, potential A^-linked glycosylation site for MPV is located at position 206 (Figure 11).<br>
Despite the low sequence homology with other paramyxoviruses, the F protein of MPV revealed typical fusion protein characteristics consistent with those described for the F proteins of other Paramyxoviridae family members (Morrison, 1988). F proteins oi Paramyxoviridae members are synthesized as inactive precursors (FO) that are cleaved by host ceU proteases which generate amino terminal F2 subunits and large carboxy terminal Fl subunits. The proposed cleavage site (Collins et al., 1996) is conserved among all members of the Paramyxoviridae family. The cleavage site of MPV contains the residues RQSR. Both arginine (R) residues are shared with APV and RSV, but the glutamine (Q) and serine (S) residues are shared with other paramyxoviruses such as human parainfluenza virus type 1, Sendai virus and morbilliviruses (data not shown).<br>
The hydrophobic region at the amino terminus of Fl is thought to function as the membrane fusion domain and shows high sequence similarity among paramyxoviruses and morbilliviruses and to a lesser extent the pneumoviruses (Morrison, 1988). These 26 residues (position 137-163, Figure 11) are conserved between MPV and APV-C, which is in agreement with this region being highly conserved among the metapneumoviruses (Naylor et aL, 1998; Seal et aL, 2000),<br><br>
'CCLLC^tA   A<br>
As is seen for the F2 subunits of APV and deletion of 22 aa residues compared with RSV (position 107-128, Figure 11). Furthermore, for RSV and APV, the signal peptide and anchor domain were found to be conserved within subtypes and displayed high variability between subtypes (Plows et aJ., 1995; Naylor et aL, 1998). The signal peptide of MPV (aa 10-85, Figure 11) at the amino terminus of F2 exhibits some sequence similarity with APV-C (18 out of 26 aa residues are similar), and less conservation with other APVs or RSV. Much more variability is seen in the membrane anchor domain at the carboxy terminus of Fl, although some homology is still seen with APV-C.<br>
The 22K (M2) protein<br>
The M2 gene is unique to the Pneumovirinae and two overlapping ORFs have been observed in all pneumoviruses. The first major ORF represents the M2-1 protein which enhances the processivity of the viral polymerase (Collins et ah, 1995; Collins, 1996) and its readthrough of intergenic regions (Hardy et al, 1998; Fearns et al, 1999). The M2-1 gene for MPV, located adjacent to the F gene, encodes a 187 aa protein (Table 5), and reveals the highest (84%) homology with M2-1 of APV-C (Table 6). Comparison of all pneumovirus M2-1 proteins revealed the highest conservation in the amino-terminal half of the protein (Collins et al, 1990; Zamora etal., 1992; Ahmadian etal., 1999), which is in agreement with the observation that MPV displays 100% similarity with APV-C in the first 80 aa residues of the protein (Figure 12A). The MPV M2-1 protein contains 3 cysteine residues located within the first 30 aa residues that are conserved among all pneumoviruses. Such a concentration of cysteines is frequently found in zinc-binding proteins (Ahmadian et aL, 1991; Cuesta et al., 2000).<br>
The secondary ORFs (M2-2) that overlap with the M2-1 ORFs of pneumoviruses are conserved in location but not in sequence and are thought to be involved in the control of the switch between virus RNA replication and transcription (Collins et aL, 1985; Elango et aL, 1985; Baybutt et al, 1987; Collins et al., 1990; ling et aL, 1992; Zamora et aL, 1992; Alansari et aL, 1994; Ahmadian et aL, 1999; Bermingham et aL, 1999). For MPV, the M2-2 ORF starts at nucleotide 512 in the M2-i ORF (Figure 7), which is exactly the same start position as for APV-C. The length of the M2-2 ORFs<br><br>
are the same for APV-C and MPV, 71 aa residues (Table 5). Sequence comparison of the M2-2 ORF (Figure 12B) revealed 64% aa sequence homology between MPV and APV-C and only 44-48% aa sequence homology between MPV and APV-A and B (Table 6).<br>
The small hydrophobic protein (SH) ORF<br>
The gene located adjacent to M2 of hMPV probably encodes a 183 aa SH protein (Pig. 1 and 7). There is no discernible sequence identity between this ORF and other RNA virus genes or gene products. This is not surprising since sequence similarity between pneumovirus SH proteins is generally low. The putative SH ORF of hMPV is the longest SH ORF known to date (Table 1). The aa composition of the SH ORF is relatively similar to that of APV, RSV and PVM, with a high percentage of threonine and serine residues (22%, 18%, 19%, 20.0%, 21% and 28% for hMPV, APV, RSV A, RSV B, bRSV and PVM respectively). The SH ORF of hMPV contains 10 cysteine residues, whereas APV SH contains 16 cysteine residues. The SH ORF of hMPV contains two potential A'-linked glycosylation sites (aa 76 and 121), whereas APV has one, RSV has two or three and PVM has four.<br>
The hydrophilicity profiles for the putative hMPV SH protein and SH of APV and RSV revealed similar characteristics (Fig. 7B). The SH ORFs of APV and hMPV have a hydrophilic N-terminus, a central hydrophobic domain which can serve as a potential membrane spanning domain (aa 30-53 for hMPV), a second hydrophobic domain (aa 155-170) and a hydrophilic C-terminus. In contrast, RSV SH appears to lack the C-terminal part of the APV and hMPV ORFs. In all pneumovirus SH proteins the hydrophobic domain is flanked by basic aa residues, which are also found in the SH ORF for hMPV (aa 29 and 54).<br>
The attachment glycoprotein (G) ORF<br>
The putative G ORP of hMPV is located adjacent to the putative SH gene and encodes a 236 aa protein (nt 6262-6972, Fig. 1). A secondary small ORF is foimd immediately following this ORF, potentially coding for 68 aa residues (nt 6973-7179) but lacking a start codon. A third potential ORF in the second reading frame of 194 aa residues is overlapping with both of these ORFs but also lacks a start codon (nt 6416-7000). This ORF is followed by a potential fourth ORF of 65 aa residues in the same reading frame (nt 7001-7198), again lacking a start codon. Finally, a potential ORF of 97 aa<br><br>
residues (but lacking a start codon) is found in the third reading frame (nt 6444-6737, Fig. 1). Unlike the first ORF, the other OKFs do not have apparent gene start or gene end sequences (see below). Although the 236 aa G ORF probably represents at least a part of the hMPV attachment protein it can not be excluded that the additional coding sequences are expressed as separate proteins or as part of the attachment protein through some RNA editing event. It should be noted that for APV and RSV no secondary ORFs after the primary G ORF have been identified but that both APV and RSV have secondary ORFs within the major ORF of G. However, evidence for expression of these ORFs is lacking and there is no sequence identity between the predicted aa sequences for different viruses (Ling et aL, 1992). The secondary ORFs in hMPV G do not reveal characteristics of other G proteins and whether the additional ORFs are expressed requires further investigation. BLAST analyses with all ORFs revealed no discernible sequence identity at the nucleotide or aa sequence level with other known virus genes or gene products. This is in agreement with the low percentage sequence identity found for other G proteins such as those of hRSV A and B (53%) (Johnson et al,. 1987) and APV A and B (38%) (Juhasz and Easton, 1994).<br>
Whereas most of the hMPV ORFs resemble those of APV both in length and sequence, the putative G ORF of 236 aa residues of hMPV is considerably smaller than the G ORF of APV (Table 1). The aa sequence revealed a serine and threonine content of 34%, which is even higher than the 32% for RSV and 24% for APV. The putative G ORF also contains 8.5% proline residues, which is higher than the 8% for RSV and 7% for APV. The unusual abundance of proline residues in the G proteins of APV, RSV and hMPV has also been observed in glycoproteins of mucinous origin where it is a major determinant of the proteins three dimensional structure (Collins and Wertz, 1983; Wertz et al, 1985; Jentoft, 1990). The G ORF of hMPV contains five potential iV-linked glycosylation sites, whereas hRSV has seven, bRSV has five and APV has three to five.<br>
The predicted hydrophilicity profile of hMPV G revealed characteristics similar to the other pneumoviruses. The N-terminus contains a hydrophilic region followed by a short hydrophobic area (aa 33-53 for hMPV) and a mainly hydrophilic C-terminus (Fig. 8B). This overall organization is consistent with that of an anchored type II transmembrane protein and corresponds well with these regions in the G protein of APV and RSV. The putative G ORF of hMPV contains only 1 cysteine residue in<br><br>
contrast to RSV and APV (5 and 20 respectively). Of note, only two of the four secondary ORFs in the G gene contained one additional cysteine residue and these four potential ORFs revealed 12-20% serine and threonine residues and 6-11% proline residues.<br>
The polymerase gene (L)<br>
In analogy to other negative strand viruses, the last ORF of the MPV genome is the RNA-dependent RNA polymerase component of the replication and transcription complexes. The L gene of MPV encodes a 2005 aa protein, which is 1 residue longer than the APV-A protein (Table 5). The L protein of MPV shares 64% homology with APV-A, 42-44% with RSV, and approximately 13% with other paramyxoviruses (Table 6). Poch et aL (1989; 1990) identified six conserved domains within the L proteins of non-segmented negative strand RNA viruses, from which domain III contained the four core polymerase motifs that are thought to be essential for polymerase function. These motifs (A, B, C and D) are well conserved in the MPV L protein: in motifs A, B and C; MPV shares 100% similarity with all pneumoviruses and in motif D MPV shares 100 % similaritywith APV and 92% with RSVs. For the entire domain III (aa 627- 903 in the L ORF), MPV shares 77% identity with APV, 61-62% with RSV and 23-27% with other paramyxoviruses (Figure 15). In addition to the polymerase motifs the pneumovirus L proteins contain a sequence which conforms to a consensus ATP binding motif K(X)21GEGAGN(X)2oK (Stec, 1991).The MPV L ORF contains a similar motif as APV, in which the spacing of the intermediate residues is off by one: K(x)22GEGAGN(X)19 K<br>
Phylogenetic analyses<br>
As an indicator for the relationship between MPV and members of the Pneumovirinae, phylogenetic trees based on the N, P, M, and F ORFs have been constructed previously (van den Hoogen et al, 2001) and revealed a close relationship between MPV and APV-C. Because of the low homology of the MPV SH and G genes with those of other paramyxoviruses, reliable phylogenetic trees for these genes can not be constructed. In addition, the distinct genomic organization between members of the Pneumovirus and Metapneumovirus genera make it impossible to generate phylogenetic trees based on the entire genomic sequence. We therefore only constructed phylogenetic trees for the M2 and L genes in addition to those previously<br><br>
published. Both these trees confirmed the close relation between APV and MPV within the Pneumovirinae subfamily (Figure 16).<br>
MPV non-coding sequences<br>
The gene junctions of the genomes of paramyxoviruses contain short and highly conserved nucleotide sequences at the beginning and end of each gene (gene start and gene end signals), possibly playing a role in initiation and termination of transcription (Curran et aL, 1999). Ck3mparing the intergenic sequences between all genes of MPV revealed a consensus sequence for the gene start signal of the N, P, M, F, M2 and G: GGGACAAGU (Figure 17A), which is identical to the consensus gene start signal of the metapneumoviruses (Ling et al, 1992; Yu et aL, 1992; Li et al., 1996; Bayon-Auboyer et al, 2000). The gene start signals for the SH and L genes of MPV were found to be slightly different from this consensus (SH: GGGAUAAAU, L: GAGACAAAU). For APV the gene start signal of L was also found to be different from the consensus: AGGACCAAT (APV-A) (Randhawa et al,. 1996) and GGGACCAGT (APV-D) (Bayon-Auboyer et al, 2000).<br>
In contrast to the similar gene start sequences of MPV and APV, the consensus gene end sequence of APV, UAGUUAAUU (Randhawa et aL, 1996). could not be found in the MPV intergenic sequences. The repeated sequence found in most genes, except the G-L intergenic region, was U AAAAA U/A/C, which could possibly act as gene end signal. However, since we sequenced viral RNA rather than mRNA, definitive gene end signals could not be assigned and thus requires further investigation. The intergenic regions of pneumoviruses vary in size and sequence (Curran et aL, 1999; Blumberg et aL, 1991; Collins et al., 1983;). The intergenic regions of MPV did not reveal homology with those of APV and RSV and range in size from 10 to 228 nucleotides (Figure 17B). The intergenic region between the M and F ORFs of MPV contains part of a secondary ORF, which starts in the primary M ORF (see above). The intergenic region between SH and G contains 192 nucleotides, and does not appear to have coding potential based on the presence of numerous stop-codons in all three reading frames. The intergenic region between G and L contains 241 nucleotides, which may include additional ORFs (see above). Interestingly, the start of the L ORF is located in these secondary ORFs. Whereas the L gene of APV does not start in the preceding G ORF, the L ORF of RSV also starts in the preceding M2 gene. At the 3' and 5'extremities of the genome of paramyxoviruses short extragenic<br><br>
region are referred to as the leader and trailer sequences, and approximately the first 12 nucleotides of the leader and last 12 nucleotides of the trailer are complementary, probably because they each contain basic elements of the viral promoter (Curran et aUl., 1999; Blumberg et al, 1991; Mink et al, 1986). The 31eader of MPV and APV are both 41 nucleotides in length, and some homology is seen in the region between nucleotide 16 and 41 of both viruses (18 out of 26 nucleotides) (Figure 17B). As mentioned before the first 15 nucleotides of the MPV genomic map are based on a primer sequence based on the APV genome. The length of the 5'trailer of MPV (188 nucleotides) resembles the size of the RSV 5'trailer (155 nucleotides), which is considerably longer than that of APV (40 nucleotides). Alignments of the extreme 40 nucleotides of the trailer of MPV and the trailer of APV revealed 21 out of 32 nucleotides homology, apart from the extreme 12 nucleotides which represent primer sequences based on the genomic sequence of APV. Our sequence analyses revealed the absence of NS1 and NS2 genes at the 3'end of the genome and a genomic organisation resembling the organisation of metapneumoviruses (3*-N-P-M-F-M2-SH-G-L-o'). The high sequence homology found between MPV and APV genes further emphasises the close relationship between these two viruses. For the N, P, M, F, M2-1 and M2-2 genes of MPV an overall amino acid homology of 79% is found with APV-C. In fact, for these genes APV-C and MPV revealed sequence homologies which are in the same range as sequence homologies found between subgroups of other genera, such as RSV- A and B or APV-A and B. This close relationship between APV-C and MPV is also seen in the phylogenetic analyses which revealed MPV and APV-C always in the same branch, separate from the branch containing APV-A and B. The identical genomic organisation, the sequence homologies and phylogentic analyses are all in favour of the classification of MPV as the first member in the Metapneumovirus genus that is isolatable from mammals. It should be noted that the found sequence variation between different virus isolates of MPV in the N, M, F and L genes revealed the possible existence of different genotypes (van den Hoogen etal., 2001). The close relationship between MPV and APV-C is not reflected in the host range, since APV infects birds in contrast to MPV (van den Hoogen et ah, 2001), This difference in host range may be determined by the differences between the SH and G proteins of both viruses that are highly divergent. The SH and G proteins of MPV did not reveal significant aa sequence homology with SH and G proteins of any other virus. Although the amino acid content and hydrophobicity plots are in favour of<br><br>
defining these ORFs as SH and G, experimental data are required to assess their function. Such analyses will also shed light on the role of the additional overlapping ORFs in these SH and G genes. In addition, sequence analyses on the SH and G genes of APV-C might provide more insight in the function of the SH and G proteins of MPV and their relationship with those of APV-C. The noncoding regions of MPV were found to be fairly similar to those of APV. The 3'leader and 6' trailer sequences of APV and MPV displayed a high degree of homology. Although the lengths of the intergenic regions were not always the same for APV and MPV, the consensus gene start signals of most of the ORFs were found to be identical. In contrast, the gene end signals of APV were not found in the MPV genome. Although we did find a repetitive sequence (U AAAAA U/A/C) in most intergenic regions, sequence analysis of viral mRNAs is required to formally delineate those gene end sequences. It should be noted that sequence information for 15 nucleotides at the extreme 3'end and 12 nucleotides at the extreme 5'end is obtained by using modified rapid amplification of cDNA ends (RACE) procedures. This technique has been proven to be successful by others for related viruses (Randhawa, J.S. et al., Rescue of synthetic minireplicons establishes the absence of the NSl and NS2 genes from avian pneumovirus. J. Virol, 71, 9849-9854 (1997); Mink, M.A., et al. Nucleotide sequences of the 3* leader and 5* trailer regions of human respiratory syncytial virus genomic RNA, Virology 185, 615-24 (1991).) To determine the sequence of the 3;- vRNA leader sequence, a homopolymer A tail is added to purified vRNA using poly-A-polymerase and the leader sequence subsequently amplified by PCR using a poly-T primer and a primer in the N gene. To determine the sequence of the 5' vRNA trailer sequence, a cDNA copy of the trailer sequence is made using reverse transcriptase and a primer in the L gene, followed by homopolymer dG tailing of the cDNA with terminal transferase. Subsequently, the trailer region is amplified using a poly-C primer and a primer in the L gene. As an alternative strategy, vRNA is Ugated to itself or synthetic linkers, after which the leader and trailer regions are amplified using primers in the L and N genes and hnker-specific primers. For the 5* trailer sequence direct dideoxynucleotide sequencing of purified vRNA is also feasible (Randhawa, 1997). Using these approaches, we can analyse the exact sequence of the ends of the hMPV genome.. The sequence information provided here is of importance for the generation of diagnostic tests, vaccines and antivirals for MPV and MPV infections.<br><br>
Materials and Methods<br>
Sequence analysis<br>
Virus isolate 00-1 was propagated to high titers (approximately 10,000 TCID50/ml) on tertiary monkey kidney cells as described previously (van den Hoogen et al, 2001). Viral RNA was isolated from supernatants from infected cells using a High Pure RNA Isolating Kit according to instructions from the manufacturer (Roch Diagnostics, Almere, The Netherlands). Primers were designed based on sequences published previously (van den Hoogen et aL, 2001) in addition to sequences published for the leader and trailer of APV/RSV (Randhawa et al, 1997; Mink et al., 1991) and are available upon request. RT-PCR assays were conducted with viral RNA, using a one-tube assay in a total volume of 50[il with 50 mM Tris pH 8.5, 50 mM NaCl, 4.5 mM MgCl2, 2 mM DTT, 1Î¼M forward primer, 1Î¼M reverse primer, 0.6 mM dNTPs, 20 units RNAsin (Promega, Leiden, The Netherlands), 10 U AMV reverse transcriptase (Promega, Leiden, The Netherlands), and 5 units Taq Polymerase (PE Applied Biosystems, Nieuwerkerk aan de IJssel, The Netherlands). Reverse transcription was conducted at 42Â°C for 30 minutes, followed by 8 minutes inactivation at 95Â°C. The cDNA was amplified during 40 cycles of 95Â°C, 1 min.; 42Â°C, 2 min.72Â°C, 3 min. with a final extension at 72Â°C for 10 minutes. After examination on a 1% agarose gel, the RT-PCR products were purified from the gel using a Qiaquick Gel Extraction kit (Qiagen, Leusden, The Netherlands) and sequenced directly using a Dyenamic ET terminator sequencing kit (Amersham Pharmacia Biotech, Roosendaal, the Netherlands) and an ABI 373 automatic DNA sequencer (PE Applied Biosystem, Nieuwerkerk aan den IJssel, the Netherlands), according to the instructions of the manufacturer.<br>
Sequence alignments were made using the clustal software package available in the software package of BioEdit version5.0.6. (http://jwbrown.mbio.ncsu.edu/Bioedit// bioedit.html; Hall, 1999).<br>
Phylogenetic analysis<br>
To construct phylogenetic trees, DNA sequences were aligned using the ClustalW software package and maximum likelihood trees were generated using the DNA-ML<br><br>
software package of the Phylip 3.5 program using 100 bootstraps and 3 jumbles. Bootstrap values were computed for consensus trees created with the consense package (Felsenstein, 1989).<br>
The MPV genomic sequence is available from Genbank under accession number AF371337. All other sequences used here are available from Genbank under accession numbers AB046218 (measles virus, all ORFs), NC-001796 (human parainfluenza virus type 3, all ORFs), NC-001552 (Sendai virus, all ORFs), X57559 (human parainfluenza virus type 2, all ORFs), NC-002617 (New Castle Disease virus, all ORFs), NC-002728 (Nipah virus, all ORFs), NG-001989 (bRSV, all ORFs), M11486 (hRSV A, all ORFs except L), NC-001803 (hRSV, L ORF). NC-001781 (hRSV B, all ORFs), D10331 (PVM, N ORF), U09649 (PVM, P ORF), U66893 (PVM, M ORF), U66893 (PVM, SH ORF), D11130 (PVM, G ORF), D11128 (F ORF). The PVM M2 ORF was taken from Ahmadian (1999), AF176590 (APV-C, N ORF), U39295 (APV-A, N ORF), U39296 (APV-B, N ORF), AF262571 (APV-C, M ORF), U37586 (APV-B, M ORF), X58639 (APV-A, M ORF), AF176591 (APV-C, P ORF), AF325443 (APV-B, P ORF), U22110 (APV-A, P ORF), AF187152 (APV-C, F ORF), Y14292 (APV-B, F ORF), D00850 (APV-A. F ORF), AF176592 (APV-C, M20RF), AF35650 (APV-B, M2 ORF), X63408 (APV-A, M2 ORF), U65312 (APV-A, L ORF), S40185 (APV-A, SH ORF).<br><br>
Table 5: Lengths of the ORFs of MPV and other paramyxoviruses.<br><br>
Footnotes:<br>
1. length in amino acid residues.<br>
2 . sequences not available<br>
3.	others: human parainfluenza virus type 2 and 3, Sendai virus, measles virus,<br>
nipah virus, phocine distemper virus, and New Castle Disease virus.<br>
4.	. ORF not present in viral genome<br><br>
Table 6: Amino acid sequence identity between the ORFs of MPV and those of other paramyxoviruses^<br><br>
Footnotes:<br>
1.	No sequence homologies were found with known G and SH proteins and were<br>
thus excluded<br>
2.	Sequences not available.<br>
3.	See Hst in table 5, footnote 3.<br>
4.	ORF absent in viral genome.<br><br>
References<br>
Current Protocols in Molecular Biology, volume 1-3 (1994-1998). Ed. by Ausubel, F.M., Brent, R., Kinston, R.E., Moore, D.D., Seidman, J.G., Smith, J.A. and StruH, K Published by John Wiley and sons. Inc., USA-Current Protocols in Immunology, volume 1-3. Ed. by CoHgan, J.E., Kruisbeek, AM., Margulies, D.H., Shevach, E.M. and Strobe, W. Published by John Wiley and sons, Inc., USA.<br>
Sambrook et al. Molecular cloning, a laboratory manual, second ed., vol. 1-3. (Cold Spring Harbor Laboratory, 1989).<br>
Fields, Virology. 1996, Vol. 1-2 3rd. Edition, Ed. by: Fields, B.N., Knipe, D.M. and Howley, P.M.    Lippincott-Raven, Philadelpia, USA.<br>
1.	Pringle, C.R. Virus taxonomy at the Xith international congress of virology, Sydney, Australia 1999. Arch, Virol. 144/2, 2065-2070 (1999).<br>
2.	Domachowske, J.B. &amp; Rosenberg, H.F. Respiratory syncytial virus infection; immime response, immunopathogenesis, and treatment. Chn. Microbio. Rev. 12(2), 298-309 (1999). Review.<br>
3.	Giraud, P., Bennejean, G., Guittet, M. &amp; Toquin, D. Turkey rhinotracheitis in France: prehminary investigations on a cihostatic virus. Vet Rec. 119, 606-607 (1986).<br>
4.	Ling, R., Easton, A. J. &amp; Pringle, C.R. Sequence analysis of the 22K, SH and G genes of turkey rhinotracheitis virus and their intergenic regions reveals a gene order different from that of other pneumoviruses. J. Gen,ViroL 73,1709-1715 (1992).<br>
5.	Yu, Q., Davis, P. J., Li, J. &amp; Cavanagh, D. Cloning and sequencing of the matrix protein (M) gene of turkey rhinotracheitis virus reveal a gene order different from that of respiratory syncytial virus. Virology 186, 426-434 (1992).<br>
6.	Randhawa, J.S., Marriott, AC, Pringle, C.R. &amp; Easton, A J. Rescue of synthetic minirepHcons estabhshes the absence of the NSl and NS2 genes from avian pneumovirus, J.Virol. 71, 9849-9854 (1997).<br><br>
7.	Evans, A.S. In: Viral Infections of Humans. Epidemiology and control 3th edn. (ed. Evans, A.S) 22-28 (Plenum Publishing Corporation, New York, 1989).<br>
8.	Osterhaus, A.D.M.E., Yang, H., Spijkers, H.E.M., Groen, J., Teppema, J.S. &amp; van Steenis, G. The isolation and partial characterization of a highly pathogenic herpesvirus fi:om the Harbor Seal (Phoca vitulina). Arch.of Virol 86, 239-251 (1985).<br>
9.	K.B. Chua et al Nipah virus: a recently emergent deadly paramyxovirus. Science 288, 1432-1435 (2000).<br><br>
10.	Welsh, J., Chada, K, Dalai, S.S, Cheng, R., Ralph, D. &amp; McClelland, M. Arbitrarily primed PCR fingerprinting of RNA. NAR. 20, 4965-4970 (1992).<br>
11.	Bayon-Auboyer, M., Arnauld, C, Toquin, D. &amp; Eterradossi, N. Nucleotide sequences of the F, L and G protein genes of two non-A/non-B avian pneumoviruses (APV) reveal a novel APV subgroup. J, of Gen. Virol 81, 2723-2733 (2000).<br>
12.	Mulder, J. &amp; Masurel, N. Pre-epidemic antibody against 1957 strain of asiatic influenza in serum of older people hving in The Netherlands. The Lancet, april 19, 810-814 (1958).<br>
13.	Pringle, C.R. In: The Paramyxoviruses. 1th edn.(ed. D.W. Kingsbury) 1-39 ( Plenum Press, New York, 1991).<br>
14.	Rothbarth, P.H., Groen, J., Bohnen, A.M., Groot, de R., &amp; Osterhaus, A.D.M.E. Influenza virus serology-a comparative study. J.of Virol Methods 78,163-169 (1999).<br>
15.	Brandenbinrg, A.H., Groen, J., van Steensel-MoU, H.A., Claas, E.J.C., Rothbarth, P.H., Neijens, H.J. &amp; Osterhaus, A.D.M.E. Respiratory syncytial virus specific serum antibodies in infants under six months of age: limited serological response upon infection. J.Med.Virol 52, 97-104 (1997).<br>
16.	Lennette, D.A. et al. In: Diagnostic procedures for vii^al, rickettsial, and chlamydial infections. 7th edn. (eds. Lennette, E.H., Lennette, DA. &amp; Lennette, E.T.) 3-25; 37-138; 431-463; 481-494; 539-563 (American public health association, Washington, 1995).<br><br>
15.	Felsenstein, J. Department of Genetics, Universtity of Washington. HttD://evolution.genetic8. washington.edu/phyhp.html<br>
16.	Schnell et al EMBO J 13, 4195-4203,1994<br>
17.	Collins, P.L., ffill, M.G.. Camargo, E., Grosfeld, Hâ€ž Chanock, R.M. &amp; Murphy, B.R. Production of infectious human respiratory syncytial virus from cloned cDNA confirms an esential role for the transcription elongation factor from the 5' proximal<br><br>
open reading frame of the M2 mRNA in gene expression and provides a capability for vaccine development. PNAS 92, 11563-11567 (1995).<br>
18.	Hoffinann, E., Neumann, G., Kawakao, Y., Hobom, G. &amp; Webster. R.G. A DNA transfection system for generation of influenza virus from eight plasmids. PNAS 97, 6108-6113 (2000).<br>
19.	Bridgen, A., EUiot, R.M. Rescue of a segmented negative-strand virus entirely from cloned complementary DNAs. PNAS 93,15400-15404 (1996).<br>
20.	Palese, P., Zheng, H., Engelhardt, O.Gâ€ž Pleschka, S. &amp; Garcia-Sastre, A. Negative-strand RNA viruses: genetic engineering and applications. PNAS 93, 11354-11358(1996).<br>
21.	Peeters, B.P., de Leeuw, O.S., Koch, G. &amp; Gielkens, A.L. Rescue of Newcastle disease virus from cloned cDNA: evidence that cleavabiHty of the fusion protein is a major determinant for virulence. J.Virol 73, 5001-5009 (1999).<br>
22.	Durbin, A.P., Hall, S.L., Siew, J.W., Whitehead, S.S., Collins, P.L. &amp; Murphy. B.R. Recovery of infectious human parainfluenza virus type 3 from cDNA. Virology 235, 323-332 (1997).<br>
23.	Tao, T., Durbin, A.P., Whitehead, S.S., Davoodi, F.. Collins, P.L. &amp; Murphy, B.R. Recovery of a fully viable chimeric human parainfluenza virus (PIV) type 3 in which the hemagglutinin-neuraminidase and fusion glycoproteins have been replaced by those of PIV type 1. J. Virol 72, 2955-2961 (1998).<br>
24.	Durbin, A.P., Skiadopoulos, M.H., McAidiffe, J.M., Riggs, J.M., Surman, S.R., Collins, P.L. &amp; Murphy, B.R. Human parainfluenza virus type 3 (PIV3) expressing the hemagglutinin protein of measles virus provides a potential method for immunization against measles virus and PIV3 in early infancy. J.Virol 74, 6821-6831 (2000).<br>
25.	Skiadopoulos, M.H., Durbin, A.P., Tatem, J.M., Wu, S.L., Paschalis, M., Tao. T., Collins, P.L. &amp; Murphy, B.R. Three amino acid substitutions in the L protein of the human parainfluenza virus type 3 cp45 live attenuated vaccine candidate contribute to its temperature-sensitive and attenuation phenotypes. J". Virol 72, 1762-1768 (1998).<br>
26.Teng, N., Whitehead, S.S., Bermingham, A.. St.Claire, M.. EUdns, W.R., Murphy, B.R. &amp; CoUins, P.L. J.Virol 74, 9317-9321 (2000).<br><br>
27. Masurel, N. Relation between Hong Kong virus and former human A2 isolates and the A/EQU12 virus in human sera collected before 1957. The Lancet May S, 907-910 (1969).<br>
Further references used with example 4.<br>
AHMADIAN, G., CHAMBERS, P., and EASTON, A. J. (1999). Detection and characterisation of proteins encoded by the second ORF of the M2 gene of pneimiovixuses. J Gen Virol 80y 2011-6.<br>
ALANSARI, H., and POTGIETER, L. N. (1994). Molecular cloning and sequence analysis of the phosphoprotein, nucleocapsid protein, matrix protein and 22K (M2) protein of the ovine respiratory syncytial virus. J Gen Virol 75, 3597-601.<br>
BARR, J., CHAMBERS, P., PRINGLE, C. R., and EASTON, A. J. (1991). Sequence of the major nucleocapsid protein gene of pneumonia virus of mice: sequence comparisons suggest structural homology between nucleocapsid proteins of pneumoviruses. paramyxoviruses, rhabdoviruses and filoviruses. J Gen Virol 72, 677-85.<br>
BAYBUTT, H. N., and PRINGLE, C. R. (1987). Molecular cloning and sequencing of the F and 22K membrane protein genes of the RSS-2 strain of respiratory s3aiC3ddal virus. J Gen Virol 68, 2789-96.<br>
BAYON-AUBOYER, M. H., ARNAULD, C, TOQUIN, D., and ETERRADOSSI, N. (2000).<br>
Nucleotide sequences of the F, L and G protein genes of two non-A/non-B avian pneumoviruses (APV) reveal a novel APV subgroup. J Gen Virol 81, 2723-33.<br>
BERMINGHAM, A., and COLLINS, P. L. (1999). The M2-2 protein of human respiratory syncytial virus is a regulatory factor involved in the balance between RNA repHcation and transcription. Proc Natl Acad Sci USA 96,11259-64.<br>
BLUMBERG, B.M., CHAN, J., AND UDEM, S.A. (1991). Function of Paramyxovirus 3' and 5'end sequences: In therory and practice. In "the Paramyxoviruses" (D.<br>
0<br>
Kingsbury, Ed.), pp. 235-247. Plenum, New York. COLHNS, P. L., and WERTZ, G. W. (1983). cDNA cloning and transcriptional mapping of<br>
nine polyadenylylated RNAs encoded by the genome of human respiratory<br>
syncytial virus. Proc Natl Acad Sci [7 S A 80, 3208-12. COLHNS, P, L., and WERTZ, G. W. (1985). The envelope-associated 22K protein of human<br>
respiratory syncytial virus: nucleotide SEQUENCE of the mRNA and a related<br>
polytranscript. J Virol 54, 65-71.<br><br>
COLLINS, P.L., DICKENS, L.E., BUCKLER-WHITE, A., OLMSTED, RA-, SPRIGGS, M.K.,<br>
CAMARGO, E., AND COEUNGH, KV.W. (1986). Nudeotide sequences for the gene junctions of human respiratory syncytial virus reveal distinctive features of intergenic structure and gene order. Proc Natl Acad SciU SA 83, 4594-98. COLLINS, P. L., HILL, M. G., and JOHNSON, P. R. (1990). The two open reading frames of the 22K mRNA of human respiratory syncytial virus: sequence comparison of antigenic subgroups A and B and expression in vitro. J Gen Virol 71, 3015-20.<br>
COLLINS, P. L., HILL, M. G., CAMARGO, E.. GROSFELD, H., CHANOCK, R, M., and MURPHY.<br>
B. R. (1995). Production of infectious human respiratory syncytial virus from<br>
cloned cDNA confirms an essential role for the transcription elongation factor<br>
from the 5* proximal open reading frame of the M2 mRNA in gene expression and<br>
provides a capability for vaccine development. Proc Natl Acad Sci C7 S A 92,<br>
11563-7. COLLINS, P. L,, MCINTOSH, K. AND CHANOCK, R.M. (1996). "Respiratory syncytial virus."<br>
In: Fields virology (B. N. Knipe, Howley, P.M., Ed.) Lippencott-Raven,<br>
Philadelphia. COOK, J, K. (2000). Avian rhinotracheitis. Rev Sci Tech 19, 602-13. CUESTA, L, GENG, X., ASENJO, A., AND ViLLANUEVA, N. (2000). Structural<br>
phosphoprotein M2-1 of the human respiratory sjrncytial virus is an RNA binding<br>
protein. J. Gen. Virol 74, 9858-67. CURRAN, J,, AND KOLAKOFSKY, D. (1999). Replication of paramyxoviruses. Adv. Virus<br>
Res, 50, 403-422. EASTON, A. J., and CHAMBERS, P. (1997). Nucleotide sequence of the genes encoding the<br>
matrix and small hydrophobic proteins of pneumonia virus of mice. Virus Res 48,<br>
27-33. ELANGO, N., SATAKE, M., and VENKATESAN, S. (1985). mRNA sequence of three<br>
respiratory syncytial virus genes encoding two nonstructural proteins and a 22K<br>
structural protein. J Virol 55,101-10. FEARNS, R., and COLLINS, P. L. (1999). Role of the M2-1 transcription antitermination<br>
protein of respiratory sjrncytial virus in sequential transcription. J Virol 73, 5852-64.<br>
FELSENSTEIN, J. (1989). "PHYLIP-Phylogeny Inference Package (Version 3.2. Cladistics 5).".<br><br>
GIRAUD, P., BENNEJEAN, G., GUITTET, M., and TOQUIN, D. (1986). Turkey rhinotracheitis<br>
in France: preliminary investigations on a dliostatic virus. Vet Rec 119, 606-7. HaU, T.A. (1999). BioEdit: a user-friendly biological sequence alignment editor and<br>
analysis program for Windows 95/98/NT. NucL Acids, Symp. Ser, 41, 95-98. HARDY, R. W., and WERTZ, G. W. (1998). The product of the respiratory syncytial virus<br>
M2 gene ORFl enhances readthrough of intergenic jvmctLons during viral<br>
transcription. J Virol 72, 520-6. HORVATH, C. M., and LAMB, R. A. (1992). Studies on the fusion peptide of a<br>
paramyxovirus fusion glycoprotein: roles of conserved residues in cell fusion. J<br>
Virol 66, 2443-55. JENTOFT, N. (1990). Why are proteins 0-glycosylated? Trends Biochem Sci 15, 291-4. JOHNSON, P. R., JR., OLMSTED, R. A., PRINCE, G. A., MURPHY, B. R., ALLING, D. W.,<br>
WALSH, E. E., and COLLINS, P. L. (1987). Antigenic relatedness between<br>
glycoproteins of human respiratory syncytial virus subgroups A and B: evaluation<br>
of the contributions of F and G glycoproteins to immunity. J Virol 61, 3163-6. JUHASZ, K., and EASTON, A. J. (1994). Extensive sequence variation in the attachment<br>
(G) protein gene of avian pneumovirus: evidence for two distinct subgroups. J Gen<br>
Virol 75, 2873-80 Kyte, J. and Doolittle, R.F. (1982). A Simple Method for Displaying the Hydrophobic<br>
Character of a Protein. J. Mol BioL 157, 105-142. LAMB, R. A., AND KOLAKOFSKY, D. (1996). "Paramyxoviridae: the viruses and their<br>
replication'*. In: Fields virology (B. N. Knipe, Howley, P.M., Ed.) Lippencott-<br>
Raven, Philadelphia. Ll, J., LING, R., RANDHAWA, J. S., SHAW, K, DAVIS, P. J., JUHASZ, K., PRINGLE, C. R.. EASTON, A. J., and CAVANAGH, D. (1996). Sequence of the nucleocapsid protein gene of<br>
subgroup A and B avian pneunoviruses. Virus Res 41,185-91. LING, R., EASTON, A. J., and PRINGLE, C. R. (1992). Sequence analysis of the 22K, SH<br>
and G genes of turkey rhinotracheitis virus and their intergenic regions reveals a<br>
gene order different from that of other pneumoviruses. J Gen Virol 73,1709-15.<br>
LING, R., DAVIS, P. J., Yu, Q., WOOD, C. M., PRDSGLE, C. R.. CAVANAGH, D., and EASTON,<br>
A. J. (1995). Sequence and in vitro expression of the phosphoprotein gene of avian pneumovirus. Virus Res 36, 247-57.<br><br>
MARRIOT, A.C., SMITH, J.M., AND EASTON, A. (2001). Fidelity of leader and trailer sequence usage by the respiratory syncytial virus and avian pneumovirus replication complexes. J. Virol, 75, 6265-72.<br>
MINK, M. A, STEC, D. S., and COLLINS, P. L. (1991). Nucleotide sequences of the 3'<br>
leader and 5' trailer regions of human respiratory syncytial virus genomic RNA. Virology 185, 615-24.<br>
MIYAHARA, K, KITADA, S., YOSHIMOTO, M., MATSIMURA, H., KAWANO, M., KOMADA, H., TSURUDOME, M., KUSAGAWA, S., NiSHIO, M., and lTO, Y. (1992). Molecular evolution of human paramsxoviruses. Nucleotide sequence analyses of the human parainfluenza type 1 virus NP and M protein genes and construction of phylogenetic trees for all the human paramyxoviruses. Arch Virol 124, 255-68,<br>
MORRISON, T. G. (1988). Structure, function, and intracellular processing of paramyxovirus membrane proteins. Virus Res 10,113-35.<br>
NAYLOR, C. J., BRITTON, P., and CAVANAGH, D. (1998). The ectodomains but not the transmembrane domains of the fusion proteins of subtypes A and B avian pneumovirus are conserved to a similar extent as those of human respiratory syncytial virus. J Gen Virol 79, 1393-8.<br>
PLOWS, D. J., and PRINGLE, C. R. (1995). Variation in the fusion glycoprotein gene of human respiratory syncytial virus subgroup A. Virus Genes 11, 37-45.<br>
POCH, 0., BLUMBERG, B. M., BOUGUELERET, L., and TORDO, N. (1990). Sequence<br>
comparison of five polymerases (L proteins) of unsegmented negative-strand RNA viruses: theoretical assignment of fimctional domains. J Gen Virol 71, 1153-62.<br>
POCH, 0., SAUVAGET, L, DELARUE, M., andTORDO, N. (1989). Identification of four conserved motifs among the RNA-dependent polymerase encoding elements. Embo J 8, 3867-74.<br>
RANDHAWA, J. s., MARRIOTT, A. C, PRINGLE, C. R., and EASTON, A J. (1997). Rescue of<br>
synthetic minireplicons establishes the absence of the NSl and NS2 genes from avian pneumovirus. J Virol 71, 9849-54.<br>
RANDHAWA, J. S., WILSON, S. D., TOLLEY, K P., CAVANAGH, D., PRINGLE, C. R., and<br>
EASTON, A. J. (1996). Nucleotide sequence of the gene encoding the viral polymerase of avian pneumovirus. J Gen Virol 77, 3047-51. SAMAL, S. K., and ZAMORA, M. (1991). Nucleotide sequence analysis of a matrix and<br>
small hydrophobic protein dicistronic mRNA of bovine respiratory syncytial virus<br><br>
demonstrates extensive sequence divergence of the small hydrophobic protein from that of human respiratory syncytial virus. J Gen Virol 72,1715-20. SATAKE, M., and VENKATESAN, S. (1984). Nucleotide sequence of the gene encoding<br>
respiratory syncytial virus matrix protein. J Virol 50, 92-9. SEAL, B. S., SELLERS, H. S., and MEINERSMANN, R. J. (2000). Fusion protein predicted<br>
amino acid sequence of the first US avian pneiunovirus isolate and lack of<br>
heterogeneity among other US isolates. Virus Res 66,139-47. SEDLMEIER, R,, and NEUBERT, W. J. (1998). The replicative complex of paramyxoviruses:<br>
structure and function. Adv Virus Res 50,101-39. STEC, D.S., HILL, M.G., 3RD, AND COLLINS, P.L. (1991). Sequence analysis of the<br>
polymerase L gene of human respiratory syncytial virus and predicted phylogeny<br>
of nonsegmented negative-strand viruses. Virology 183, 273-87 VAN DEN HOOGEN, B. G., DE JONG, J. C, GROEN, J., KUIKEN, T., DE GROOT, R.,<br>
POUCHIER, R. A., and OSTERHAUS, A. D. (2001). A newly discovered human<br>
pneumovirus isolated from young children with respiratory tract disease. Nat Med<br>
7(6), 719-24. VIRUS TAXONOMY (2000). Seventh report of the international Committee on Taxonomy<br>
of Viruses.<br>
WERTZ, G. W., COLLINS, P. L., HUANG, Y., GRUBER, C, LEVINE, S., and BALL, L. A.<br>
(1985). Nucleotide sequence of the G protein gene of human respiratory syncytial virus reveals an unusual type of viral membrane protein. JFVoc Natl Acad Sci USA 82, 4075-9.<br>
Yu, Q., DAVIS, P. J., BARRETT, T., BINNS, M. M., BOURSNELL, M. E., and CAVANAGH, D.<br>
(1991). Deduced amino acid sequence of the fusion glycoprotein of turkey rhinotracheitis virus has greater identity with that of human respiratory syncytial virus, a pneumovirus, than that of paramyxoviruses and morbillivirixses. J Gen Virol 72, 75-81. Yu, Q., DAVIS, P. J., Ll. J., and CAVANAGH, D. (1992). Cloning and sequencing of the matrix protein (M) gene of turkey rhinotracheitis virus reveal a gene order different from that of respiratory syncytial virus. Virology 186, 426-34. ZAMORA, M., and SAMAL, S. K. (1992). Sequence analysis of M2 mRNA of bovine respiratory syncytial virus obtained from an F-M2 dicistronic mRNA suggests structural homology with that of human respiratory syncytial virus. J Gen Virol 73, 737-41.<br><br>
Primers used for RT-PCR detection of known paramyxo-viruses. Primers for hPTV-l to 4, mumps, measles, Tupaia, Mapuera and Hendra are developed in house and based on aUignments of available sequences. Primers for New Castle Disease Virus are taken from Seal, J., J. et al; CM. Microb., 2624-2630, 1995. Primers for Nipah and general paramyxovirus-PCR are taken from: Chua, KB., et al; Science, 288 26 may 2000<br>
Virus	primers	located in protein<br>
HPIV-1	fwd	5'-TGTTGTCGAGACTATTCCAA-3'	HN<br>
Rev	5'-TGTTG(T/A)ACCAGTTGCAGTCT.3'<br>
HPIV-2	Fwd	5'-TGCTGGTTCTATTGAGAAACGCC-3' N<br>
Rev	5'-GGTGAC/T TC(T/C)AATAGGGCCA-3'<br>
HPIV-3	Fwd	5'-CTCGAGGTTGTCAGGATATAG-3'	HN<br>
Rev	5'-GTTTGGGAGTTGAACACAGTT-3'<br>
HPIV-4	Fwd	5'-TTC(A/G)GTTTTAGCTGCTTACG-3'    N<br>
Rev	5'-AGGCAAATCTCTGGATAATGC-3'<br>
Mumps	Fwd	5'-TCGTAACGTCTCGTGACC-3'	SH<br>
Rev	5'-GGAGATCTTTCTAGAGTGAG-3'<br>
NOV	Fwd	5'-CCTTGGTGAiTCTATCCGIAG-3'	F<br>
Rev	5'-CTGCCACTGCTAGTTGiGATAATCC-3'<br>
Tupaia	Fwd	5'-GGGCTTCTAAGCGACCCAGATCTTG-3'        N<br>
Rev	5'-GAATTTCCTTATGGACAAGCTCTGTGC-3'<br>
Mapuera       Fwd	5'-GGAGCAGGAACTCCAAGACCTGGAG-3'       N<br>
Rev:	5'-GCTCAACCTCATCACATACTAACCC-3'<br>
Hendra	Fwd	5'-GAGATGGGCGGGCAAGTGCGGCAACAG-3' N<br>
Rev	5'-GCCTTTGCAATCAGGATCCAAATTTGGG-3'<br>
Nipah	Fwd	5'-CTGCTGCAGTTCAGGAAACATCAG-3'	N<br>
Rev	5'-ACCGGATGTGGTCACAGAACTG-3'<br>
HRSV	Fwd	S'-TTTGTTATAGGCATATCATTG-y	F<br>
Rev	5'-TTAACCAGCAAAGTGTTA-3'<br>
Measles	Fwd	5'-TTAGGGCAAGA6ATGGTAAGG-3'	N<br>
Rev	5'-TTATAACAATGATGGAGGG-3'<br>
General Paramyxoviridae :<br><br>
Fwd    5'-CATTAAAAAGGGCACAGACGC-3' Rev     5'-TGGACATTCTCCGCAGT-3' Primers for RAP-PCR:<br>
ZFl: 5'-CCCACCACCAGAGAGAAA-3' ZF4: 5'-ACCACCAGAGAGAAACCC-3' ZF7: 5'-ACCAGAGAGAAACCCACC-3' ZFIO: 5'-AGAGAGAAACCCACCACC-3" ZF13: 5'-GAGAAACCCACCACCAGA-3' ZF16: 5'-AAACCCACCACCAGAGAG-3'<br>
CSl: 5'-GGAGGCAAGCGAAGGCAA-3' CS4: 5'-GGCAAGCGAACGGAAGGA-3' CS7: 5'-AAGCGAACGCAAGGAGGC-3' CS10:5'-CGAACGCAAGGAGGCAAG-3' OS 13:5'-ACGCAAGGAGGCAAGCGA-3' CS16:5'-CAAGGAGGCAAGCGAACG-3'<br>
20 fragments successfully purified and sequenced:<br>
10 fragments found with sequence homology in APV<br>
Fragment 1	ZF 7,     335 bp	N gene<br>
Fragment 2	ZF 10,   235 bp	N gene<br>
Fragments	ZF 10,   800bp	M gene<br>
Fragment 4	CS   1,1250 bp	F gene<br>
Fragment 5	CS 10,   400 bp	F gene<br>
Fragment 6	CS 13, 1450 bp	F gene<br>
Fragment 7	CS 13, 750 bp	F gene<br>
Fragment 8	ZF   4, 780 bp	L gene (protein level)<br>
Fragment 9	ZF 10, 330 bp	L gene (protein level)<br>
Fragment 10	ZF10,   250bp	L gene (protein level)<br>
Primers used for RAP-PCR amplification of nucleic acids from the prototype isolate.<br><br>
Example 6<br>
Further exploration of the two subtypes of hMPV<br>
Based on phylogenetic analysis of the different isolates of hMPV obtained so far, two genotypes have been identified with virus isolate 00-1 being the prototype of genotype A and isolate 99-1 the prototype of genotype B.<br>
We hypothesise that the genotypes are related to subtypes and that re-infection with viruses from both subgroups occur in the presence of pre-existing immimity and the antigenic variation may not be strictly required to allow re-infection. Furthermore, hMPV appears to be closely related to avian pneumovirus, a virus primarily found in poultry. The nucleotide sequences of both viruses show high percentages of homology, with the exception of the SH and G proteins. Here we show that the viruses are cross-reacting in tests, which are based primarily on the nucleoprotein and matrixprotein, but they respond differently in tests, which are based on the attachment proteins. The differences in virus neutralisation titers provide further proof that the two genotypes of hMPV are two different serotypes of one virus, where APV is a different virus.<br>
The cross reaction between the two serotypes and the cross reaction between APV an<br>
hMPV<br>
Methods<br>
Protocol for IgG , IgA and IgM antibody detection for hMPV:<br>
The indirect IgG EIA for hMPV was performed in microtitre plates essentially as<br>
described previously (Rothbarth, P.H. et al., 1999; Influenza virus serology-a<br>
comparative study. J. of Vir. Methods 78 (1999) 163-169.<br>
Briefly, concentrated hMPV was solubilized by treatment with 1% Triton X-100 an<br>
coated for 16 hr at room temperature into inicrotitre plates in PBS after<br>
determination of the optimal working dilution by checkerboard titration.<br>
Subsequently, 100 ul volumes of 1:100 diluted human serum samples in EIA buffer<br>
were added to the wells and incubated for 1 h at 37C. Binding of human IgG was<br>
detected by adding a goat anti-human IgG peroxidase conjugate (Biosource, USA).<br><br>
Adding TMB as substrate developed plates and OD was measured at 450 nm. the results were expressed as the S(ignal)/N(egative) ratio of the OD. A serum was considered positive for IgG, if the S/N ratio was beyond the negative control plus three times the standard.<br>
hMPV antibodies of the IgM and IgA classes were detected in sera by capture EIA essentially as described previously (Rothbarth, P.H et al. 1999; Influenza virus serolgy-a comparative study. J. Vir. methods 78 (1999) 163-169. For the detection of IgA and IgM commercially available microtiter plates coated with anti human IgM or IgA specific monoclonal antibodies were used. Sera were diluted 1:100 and after incubation of 1 hr at 37C, an optimal working dilution of hMPV is added at each well (100 ul). Incubated 1 hr 37C. After washing polyclonal anti hMPV labeled with peroxidase was added, the plate was incubated 1 hr 37C. Adding TMB as substrate developed plates and OD was measured at 450 nm. the results were expressed as the S(ignal)/N(egative) ratio of the OD, A serum was considered positive for IgG, if the S/N ratio was beyond the negative control plus three times the standard.<br>
AVP antibodies were detected in an AVP inhibition assay. Protocol for APV inhibition test is included the APV-Ab SVANOVIR Â® enzyme immunoassay which is manufactured by SVANOVA Biotech AB, Uppsal Science Park Glunten SE-751 83 Uppsala Sweden. The results were expressed as the S(ignal)/N(egative) ratio of the OD. A serum was considered positive for IgG, if the S/N ratio was beyond the negative control plus three times the standard.<br>
1,   Guinea pigs<br>
A.  (re) infection of guinea nigs with both subtypes of hMPV<br>
Virus isolates ned/00/01 (subtype A) and ned/99/01 (subtype B) have been used to inoculate 6 guinea pigs per subtype (intratracheal, nose and eyes). 6 GFs infected with hMPV 00-1 (10e6,5 TCID50) 6 GP's infected with hMPV 99-1 (10e4,l TCID50)<br><br>
54 Days after the primary infection, the guinea pigs have been inoculated with the homologous and heterologous subtypes (10e4 TCID50/ml):<br>
2	guinea pigs; 1st infection 00-1; 2nd 99-1 (heterologous)<br>
3	guinea pigs: lst infection 00-1; 2nd 00-1 (homologous)<br><br>
2	guinea pigs: 1st  infection 99-1; 2nd 00-1 (heterologous)<br>
3	guinea pigs: lst infection 99-1; 2nd 99-1 (homologous)<br>
Throat and nose swabs have been collected for 12 days (1st infection) or 8 days (2nd infection) post infection, and have been tested for presence of the virus by RT-PCR assays.<br>
Results of RT-PCR assav: Figure 29<br>
Summary of results: guinea pigs inoculated with virus isolate ned/00/01 show infection of the upper respiratory tract day 1 to 10 post infection. Guinea pigs inoculated with ned/99/01 show infection of the upper respiratory tract day 1 to 5 post'infection. Infection with ned/99/01 appears to be less severe than infection with ned/00/01. A second inoculation of the guinea pigs with the heterologous virus results in re-infection in 3 out of<br>
4	guinea pigs and with the homologous virus in 2 out of 6 guinea pigs. No or only little<br>
clinical symptoms were noted in those animals that became re-infected, and no clinical<br>
symptoms were seen in those animals that were protected against the re-infections,<br>
demonstrating that even with wild-type virus, a protective effect of the furst infection is<br>
evident, showing the possible use of heterologous (and of course homologues) isolates as a<br>
vaccine, even in an unattenuated form.<br>
Both subtypes of hMPV are able to infect guinea pigs, although infection with subtype B (ned/99/01) seems less severe (shorter period of presence of the virus in nose and throat) than infection with subtype A (ned/00/01). This may be due to the higher dose given for subtype A, or to the lower virulence of subtype B.<br>
Although the presence of pre-existing immunity does not completely protect against re-infection with both the homologous and heterologous virus, the infection appears to be less prominent in that a shorter period of presence of virus was noted and not all animals became virus positive.<br><br>
B. Serology of guinea pies infected with both subtypes of hMPV<br>
At day 0, 52, 70, 80, 90, 110,126 and 160 sera were collected from the guinea pigs and tested at a 1:100 dilution in a whole virus ELISA against ned/00/01 and ned/99/01 antigen.<br>
Figure 30 A and B: IgG response against ned/00/01 and ned/99/01 for each individual<br>
guinea pig<br>
Figure 31: Specificity of the ned/00/01 and ned/99/01 ELISA. Only data from<br>
homologous reinfected guinea pigs have been used.<br>
Figure 32: Mean IgG response against ned/00/01 and ned/99/01 ELISA of 3<br>
homologous (00-1/00-1), 2 homologous (99-1/99-1), 2 heterologous (99-1/00-1) and 2<br>
heterologous (00-1/99-1) infected guinea pigs.<br>
SummEuy of results:<br>
Only a minor difference in response to the two different ELISA's is observed. Whole virus ELISA against 00-1 or 99-1 cannot be used to discriminate between the two subtypes.<br>
C.Reactivity of sera raised against hMEPV in guinea nigs with APV antigen<br>
Sera collected from the infected guinea pigs have been tested with an APV inhibition ELISA<br>
Figure 33: Mean percentage of APV inhibition of hMPV infected guinea pigs.<br>
Summary of results:<br>
Sera raised against hMPV in guinea pigs, react in the APV inhibition test in a same<br>
manner as they react in the hMPV IgG ELISA's.<br>
â€¢ 4<br>
Sera raised against ned/99/01 reveal a lower percentage of inhibition in the APV inhibition ELISA than sera raised against ned/00/01. Guinea pigs infected with<br><br>
ned/99/01 might have a lower titer (as is seen in the hMPV ELISA's) or the cross-reaction of ned/99/01 with APV is less than that of ned/00/01. Nevertheless, the APV-Ab inhibition ELISA can be used to detect hMPV antibodies in guinea pigs.<br>
(D. Virus neutralisation assays with sera raised against hMPV in guinea pigs.<br>
Sera collected at day 0, day 52, 70 and 80 post infection were used in a virus (cross) neutralisation assay with ned/00/01, ned/99/01 and APV-C. Starting dilution was 1 to 10 and 100 TCID50 virus per well was used. After neutralisation, the virus was brought on tMK cells, 15 min. centrifuged at 3500 RPM, after which the media was refreshed.<br>
The APV tests were grown for 4 days and the hMPV tests were grown for 7 days. Cells were fixed with 80% aceton, and IFAs were conducted with monkey-anti hMPV fito labeled. Wells that were negative in the staining were considered as the neutralising titer. For each virus a 10-log titration of the virus stock and 2 fold titration of the working solution was included.<br>
Figure 34: Virus neutralisation titers of ned/00/01 and ned/99/01 infected guinea pigs against ned/00/01, ned/99/01 and APV-C<br>
2.   Cynomologous macaques<br>
A. (re) infection of cynomologous macaques with both subtypes of hMPV<br>
Virus isolates ned/00/01 (subtype A) and ned/99/01 (subtype B) (1.5 TCID50) have been used to inoculate 2 cynomologous macaques per subtype (intratracheal, nose and eyes). Six months after the primary infection, the macaque have been inocvdated for the second time with ned/00/01. Throat swabs have been collected for 14 days (1" infection) or 8 days (2nd infection) post infection, and have been tested for presence of the virus by RT-PCR assays.<br>
Figure 35: Results of RT-PCR assays on throat swabs of cynomolgous macaques inoculated (twice) with ned/00/01.<br>
Summary of results:<br><br>
Summary of results: cynomologous macaques inoculated with virus isolate ned/00/01 show infection of the upper respiratory tract day 1 to 10 post infection. Clinical symptoms included a suppurative rhinitis. A second inoculation of the macaques with the homologous virus results in re-infection, as demonstrated by PCR, however, no clinical symptoms were seen.<br>
B. Serology on sera collected of hMPV infected cynomologous macaques.<br>
From the macaques which received ned/00/01 sera were collected during 6 months after the primary infection (re-infcetion occurred at day 240 for monkey 3 and day 239 for monkey 6).<br>
Sera were used to test for the presence of IgG antibodies against either ned/00/01 or APV, and for the presence against IgA and IgM antibodies against ned/00/01.<br>
Results: Figure 36A<br>
IgA, IgM and IgG response against ned/00/01 of 2 cynomologous macaques<br>
(re)infeGted with ned/00/01.<br>
Figure 36B<br>
IgG response against APV of 2 cynbomologous macaques infected with ned/00/01.<br>
Summary of results:<br>
Two macaques have been succesfully infected with ned/00/01 and in the presence of antibodies against ned/00/01 been reinfected with the homologous virus. The response to IgA and IgM antibodies shows the raise in IgM antibodies after the first infection, and the absence of it after the reinfection. IgA antibodies are only detected after the re-infection, showing the immediacy of the immune response after a first infection. Sera raised against hMPV in macaques which were tested in an APV inhibition ELISA show a similar response as to the hMPV IgG ELISA.<br>
Discussion/conclusion<br>
hMPV antibodies in cynomologous macaques are detected with the APV inhibition<br>
ELISA with a similar sensitivity as with an hMPV ELISA, and therefore the APV<br><br>
inhibition EIA is suitable for testing human samples for the presence of hMPV antibodies.<br>
C.Virus (cross) neutralisation assays with sera collected from hMPV infected cynomologous macaques<br>
Summary of results: The sera taken from day 0 to day 229 post primary infection show only low virus neutralisation titers against ned/00/01 (0-80), the sera taken after the secondary infection show high neutralisation titers against ned/00/01: &gt;1280. Only sera taken after the secondary infection show neutralisation titers against ned/99/01 (80-640), and none of the sera neutralise the APV C virus.<br>
There is no cross reaction between APV-C and hMPV in virus (cross)neutralisation assays, where there is a cross reaction between ned/00/01 and ned/99/01 after a boost of the antibody response.<br>
3. Humans<br>
Sera of patients ranging in age of 20 years of age have previously been<br>
tested in IFA and virus neutralisation assays against ned/00/01. (See tabel 1 of<br>
patent).<br>
Here we have tested a number of these sera for the presence of IgG, IgM and IgA<br>
antibodies in an ELISA against ned/00/01, and we tested the samples in the APV<br>
inhibition ELISA.<br>
Results: Figure 37 Comparison of the use of the hMPV ELISA and the APV inhibition ELISA for the detection of IgG antibodies in human sera, there is a strong correlation between the IgG hMPV test and the APV-Ab test, therefore the APV-Ab test is essentially able to detect IgG antibodies to hmPV in humans.<br><br>
4. Poultry<br>
96 chickens have been tested in both the APV inhibition ELISA and the ned/00/01 ELISA for the presence of IgG antibodies against APV.<br>
Summary of results: Both the hMPV ELISA and the APV inhibition ELISA detect antibodies against APV (data not shown).<br>
Summary of results.<br>
We found two genotypes of hMPV with ned/00/01 being the prototype of subgroup A and ned/99/01 the prototype of subgroup B.<br>
"According to classical serogical analyses (as for example known Francki, R.I.B., Fauquet, CM., Knudson, D.L., and Brown, F., Classification and nomenclature of viruses. Fifth report of the international Committee on Taxonomy of Viruses. Arch Virol, 1991. Supplement 2: p. 140-144), two subtypes can be defined on the basis of its immunological distinctiveness, as determined by quantitative neutralization assays with animal antisera. Two distinct serotypes have either no cross-reaction with eachother or show a homologous-to heterologous titer ratio &gt;16 in both directions. If neutralization shows a certain degree of cross-reaction between two viruses in either or both directions (homologous-to-heterologous titer ration of eight or 16), distinctiveness of serotype is assumed if substantial biophysical/biochemical differences of DNA's exist. If neutralization shows a distinct degree of cross-reaction between two viruses in either or both directions (homologous-to-heterologous titer ration of smaller than eight), identity of serotype of the isolates under study is assumed."<br>
For RSV it is known that re-infection occurs in the presence of pre-existing immunity (both homologous and heterologous). Infection of guinea pigs and cynomologous macaques with both the homologous and heterologous serotypes of hMPV revealed that this is also true for hMPV. In addition, IgA and IgM ELISA's against hMPV<br><br>
revealed the reaction of IgA antibodies only occurs after re-infection. Sera raised against hMPV or APV respond in an equal way in APV and hMPV ELISAa. From the nucleotide sequence comparisons, it is known that the viruses show about 80% amino acid homology for the N, P, M, and F genes. In ELISA's the N and M proteins are the main antigens to react. Virus neutralisation assays (known to react against the surface glycoproteins G, SH and F) show a difference between the two different sera. Although APV en hMPV cross react in ELISAs, phylogenetic analyses of the nucleotide sequences of hMPV and APV, the differences in virus neutralisation titers of sera raised against the two different viruses, and the differences in host usage again reveal that APV-C and hMPV are two different viruses. Based on the results we speculate that hMPV infection in mammals is possible a result of a zoonotic event from birds to mammals. But the virus has adapted in such a way (i.e. the G and SH proteins) that a return (from mammals to birds) zoonotic event seems unlikely, considering the presence of AVP in birds.<br><br>
Addendum<br>
Background information on Pneumovirinae<br>
The family of Paramyxoviridae contains two subfamilys: the Paramyxovirinae and the Pneumovirinae, The subfamily Pneumovirinae consists of two genera: Pneumovirus and Metapneumovirus. The genus Pneumovirus contains the human, bovine, ovine and caprine respiratory syncytial viruses and the pneumonia virus of mice (PVM). The genus Metapneumovirus contains the avian pneumoviruses (APV, also referred to as TRTV).<br>
The classification of the genera in the subfamily Pneumovirinae is based on classical virus characteristics, gene order and gene constellation. Viruses of the genus Pneumovirus are unique in the family of Paramyxoviridae in having two nonstructural proteins at the 3'end of the genome (3'-NSl-NS2-N-P-M-SH-G-F-M2-L" 5'). In contrast, viruses in the genus Metapneumovirus lack the NSl and NS2 genes and the organisation of genes between the M and L coding regions is different: 3'-N-P-M-F-M2-SH-G-L-5'.<br>
All members of the subfamily Paramyxovirinae have haemagluttinating activity, but this function is not a defining feature for the subfamily Pneumovirinae, being absent in RSV and APV but present in PMV. Neuraminidase activity is present in members of the genera Paramyxovirus and Rubulavirus (subfamily Paramyxovirinae) but is absent in the genus Morbillivirus (subfamily Paramyxovirinae) and the genera Pneumovirus and Metapneumovirus (subfamily Pneumovirinae), A second distinguishing feature of the subfamily Pneumovirinae is the apparent limited utilization of alternative ORFs within mRNA by RSV, In contrast, several members of the subfamily Paramyxovirinae, such as Sendai and Measles viruses, access alternative ORFs within the mRNA encoding the phosphoprotein (P) to direct the synthesis of a novel protein.<br>
The G protein of the Pneumovirinae does not have sequence relatedness or structural similarity to the HN or H proteins of Paramyxovirinae and is only approximately half the size of their chain length. In addition, the N and P proteins are smaller than their counterparts in the Paramyxovirinae and lack unambigous sequence homology. Most nonsegmented negative stranded RNA viruses have a single matrix (M) protein.<br><br>
Members of the subfamily Pneumovirinae are an exception in having two such<br>
proteins, M and M2. The M protein is smaller than its Paramyocovirinae counterparts<br>
and lacks sequence relatedness with Paramyxovirinae.<br>
When grown in cell cultures, members of the subfamily Pneumovirinae show typical<br>
cytopathic effects; they induce characteristic syncytia formation of cells.<br>
(Collins, 1996).<br>
The subfamily Pneumovirinae, genus Pneumovirus<br>
hRSV is the type-species of the genus Pneumovirus and is a major and widespread cause of lower respiratory tract illness during infancy and early childhood (Selwyn, 1990). In addition, hRSV is increasingly recognised as an important pathogen in other patient groups, including immune compromised individuals and the elderly. RSV is also an important cause of community-acquired pneumonia among hospitalised adults of all ages (Englund, 1991; Falsey, 2000; Dowell, 1996). Two major antigenic types for RSV (A and B) have been identified based on differences in their reactivity with monoclonal and polyclonal antibodies and by nucleic acid sequence analyses (Anderson, 1985; Johnson, 1987; Sullender, 2000)". In particular the G protein is used in distinguishing the two subtypes. RSV-A and B share only 53% amino acid sequence homology in G, whereas the other proteins show higher homologies between the subtypes (table 1) (Collins, 1996). Detection of RSV infections has been described using monoclonal and polyclonal antibodies in immunofluorescence techniques (DIF, IFA), virus neutralisation assays and ELISA or RT-PCR assays (Rothbarth, 1988; Van Milaan. 1994; Coggins, 1998). Closely related to hRSV are the bovine (bRSV), ovine (oRSV) and caprine RSV (oRSV), from which bRSV has been studied most extensively. Based on sequence homology with hRSV, the ruminant RSVs are classified within the Pneumovirus genus, subfamily Pneumovirinae (Collins, 1996). Diagnosis of ruminant RSV infection and subtyping is based on the combined use of serology, antigen detection, virus isolation and RT-PCR assays (Uttenthal, 1996;Valarcher, 1999; Oberst, 1993; Vilcek, 1994).<br>
Several analyses on the molecular organisation of bRSV have been performed using human and bovine antisera, monoclonal antibodies and cDNA probes. These analyses revealed that the protein composition of hRSV and bRSV are very similar and the<br><br>
genomic organisation of bRSV resembles that of hRSV. For both bRSV and hRSV, the G and F proteins represent the major neutralisation and protective antigens. The G protein is highly variable between the hRSV subtypes and between hRSV and bRSV (53 and 28% respectively) (Prozzi, 1997; Lerch, 1990). The F proteins of hRSV and bRSV strains present comparable structural characteristics and antigenic relatedness. The F protein of bRSV shows 80-81% homology with hRSV. while the two hRSV subtypes share 90% homology in F (Walravens, K 1990). Studies based on the use of hRSV and bRSV specific monoclonal antibodies have suggested the existence of different antigenic subtypes of bRSV, Subtypes A, B, and AB are distinguished based on reaction patterns of monoclonal antibodies specific for the G protein (Furze, 1994; Prozzi, 1997; Elvander, 1998). The epidemiology of bRSV is very similar to that of hRSV. Spontaneous infection in young cattle is frequently associated with severe respiratory signs, whereas experimental infection generally results in milder disease with slight pathologic changes (Elvander, 1996). RSV has also been isolated from naturally infected sheep (oRSV) (LeaMaster, 1983) and goats (cRSV) (Lehmkuhl, 1980). Both strains share 96% nucleotide sequence with the bovine RSV and are antigenically crossreacting. Therefore, these viruses are also classified within the Pneumovirus genus.<br>
A distinct member of the subfamily Pneumovirinae, genus Pneumovirus is the Pneumonia virus of mice (PVJ/I),<br>
PVM is a common pathogen in laboratory animal colonies, particularly those containing atymic mice. The naturally acquired infection is thought to be asymptomatic, though passage of virus in mouse lungs resulted in overt signs of disease ranging from an upper respiratory tract infection to a fatal pneumonia (Richter, 1988; Weir, 1988).<br>
Restricted serological crossreactivity between the nucleocapsid protein (N) and the phosphoprotein (P) of PVM and hRSV has been described but none of the external proteins show cross-reactivity, and the viruses can be distinguished from each other in virus neutralisation assays (Chambers, 1990a; Gimenez, 1984; ling, 1989a). The glycoproteins of PVM appear to differ from those of other paramyxoviruses and resemble those of RSV in terms of their pattern of glycosylation. They differ, however, in terms of processing. Unlike RSV, but similar to the other paramyxoviruses, PVM has haemagglutinating activity with murine erythrocytes, for which the G protein<br><br>
appears to be responsible since a monoclonal antibody to this protein inhibits haemagglutination (Ling, 1989b).<br>
The genome of PVM resembles that of hRSV, including two nonstructural proteins at its 3'end and a similar genomic organisation (Chambers, 1990a; Chambers, 1990b). The nucleotide sequences of the PVM NS1/NS2 genes are not detectably homologous with those of hRSV (Chambers, 1991). Some proteins of PVM show strong homology with hRSV (N: 60%, and F: 38 to 40%) while G is distinctly different (the amino acid sequence is 31 % longer) (Barr, 1991; Barr, 1994; Chambers, 1992). The PVM P gene, but not that of RSV or APV, has been reported to encode a second ORF, representing a unique PVM protein (Collins, 1996). New PVM isolates are identified by virus isolation, heamagglutination assays, virus neutrahsation assay and various immunofluorescence techniques.<br><br>
Avian pneumoviruses (APV) has been identified as the aetiological agent of turkey rhinotracheitis (McDougall, 1986; Collins, 1988) and is therefore oftenreferred to as<br><br>
turkey rhinotracheitis virus (TRTV). The disease is an upper respiratory tract infection of tunkeys, resulting in high morbidity and variable, but often high, mortality. In turkey hens, the virus can also induce substantial reductions in egg production. The same virus can also infect chickens, but in this species, the role of the virus as a primary pathogen is less clearly defined, although it is commonly associated with swollen head syndrome (SHS) in breeder chicken (Cook, 2000). The virions are pleiomorphic, though mainly spherical, with sizes ranging from 70 to<br>
600 nm and the nucleocapsid, containing the linear, non-segmented, negative-sense RNA genome, shows helical symmetry (Collins, 1986; Giraud, 1986). This morphology resembles that of members of the family Paramyxoviridae, Analyses of the APV-encoded proteins and RNAs suggested that of the two subfamilys of this family (Paramyxovirinae and Peumovirinae), APV most closely resembled the Pneurnovirinae (Collins, 1988; Ling, 1988; Cavanagh, 1988).<br>
APV has no non-structural proteins (NSl and NS2) and the gene order (3'-N-P-M-F-M2-SH-G-L-5') is different from that of mammalian pneumoviruses such as RSV. APV has therefore recently been classified as the type species for the new genus Metapneumovirus (Pringle, 1999).<br>
Differences in neutralisation patterns, ELISA and reactivity with monoclonal antibodies have revealed the existence of different antigenic types of APV. Nucleotide sequencing of the G gene led to the definition of two virus subtypes (A and B), which share only 38% amino acid homology (Collins, 1993; Juhasz, 1994). An APV isolated from Colorado, USA (Cook, 1999), was shown to cross-neutralize poorly with subtype A and B viruses and based on sequence information was designated to a novel subtype, C (Seal, 1998; Seal 2000). Two non-A/non-B APVs were isolated in France, and were shown to be antigenically distinct from subtypes A, B and C. Based on amino acid sequences of the F, L and G genes, these viruses were classified again as a novel subtype, D (Bayon-Auboyer, 2000).<br>
Diagnosis of APV infection can be achieved by virus isolation in chicken or turkey tracheal organ cultures (TOCs) or in Vero cell cultures. A cytopathic effect (CPE) is generally observed after one or two additional passages. This CPE is characterised by scattered focal areas of cell rounding leading to synctyial formation (Buys, 1989). A number of serology assays, including IF and virus neutralisation assays have been developed. Detection of antibodies to APV by ELISA is the most commonly used method (O'Loan, 1989; Gulati, 2000), Recently, the polymerase chain reaction (PCR)<br><br>
has been used to diagnose APV infections. Swabs taken from the oesophagus can be used as the starting material (Bayon-Auboyer, 1999; Shin, 2000)<br>
Alansari, H. and Potgieter, L.N.D. 1994. Nucleotide and predicted amino acid sequence analysis of the ovine respiratory syncytial virus non-structural IC and IB<br>
genes and the small hydrophobic protein gene. J.Gen.Virol. 75: 401-404.<br>
Alansari, H,, Duncan R.B., Baker, J.C. and Potgieter, L.N. 1999. Analysis of ruminant respiratory syncytial virus isolates by RNAse protection of the G glycoprotein transcripts. J.Vet.Diagn.Invest. 11: 215-20<br>
Anderson, L.J, Hierholzer, J.C, Tsou, C, Hendry, R.M., Fernic, B.F., Stone, Y. and Mcintosh, K. 1985. Antigenic characterisation of respiratory syncytial virus strains with monoclonal antibodies. J. Inf. Dis. 151: 626-633.<br>
Barr, J., Chambers, Pringle, C.R., Easton, A. J. 1991. Sequence of the major nucleocapsid protein gene of pneumonia virus of mice: sequence comparisons suggest structural homology between nucleocapsid proteins of pneumoviruses, paramyxoviruses, rhabdoviruses and filoviruses. J.Gen.Virol. 72: 677-685.<br>
Barr, J., Chambers, P., Harriott, P., Pringle, C.R. and Easton, A.J. 1994. Sequence of the phosphoprotein gene of pneumonia virus of mice: expression of multiple proteins from two overlapping rading frames. J. Virol. 68: 5330-5334,<br>
Bayon-Auboyer, M.H., Jestin, V., Toquin, D., Cherbonnel, M. and Eterradosi, N. 1999. Comparison of F-, G- and N-based RT-PCR protocols with conventional virological procedures for the detection and typing of turkey rhinotracheitis virus. Arch.Vir. 144: 1091-1109.<br>
Bayon-Auboyer, M.H., Arnauld, C, Toquin, D., and Eterradossi, N. 2000. Nucleotide sequences of the F, L and G protein genes of two non-A/non-B avian pneumoviruses (APV) reveal a novel APV subgroup. J.Gen.Virol. 81: 2723-2733.<br><br>
Buys, S.B., Du Preez, J.H. and Els, H.J. 1989. The isolation and attenuation of a virus causing rhinotracheitis in turkeys in South AMca. Onderstepoort J.Vet.Res. 56:<br>
87-98.<br>
Cavanagh, D. and Barrett, T. 1988. Pneumovirus-like characteristics of the mRNA and proteins of turkey rhinotracheitis virus. Virus Res. 11: 241-256.<br>
Chambers, P., Pringle, C.R. and Easton, A. J. 1990a. Molecular cloning of pneumonia virus of mice. J. Virol. 64: 1869-1872.<br>
Chambers, P., Matthews, D.A, Pringle, C.R. and Easton, A. J. 1990b. The nucleotide sequences of intergenic regions between nine genes of pneumonia virus of mice establish the physical order of these genes in the viral genome. Virus Res. 18: 263-270.<br>
Chambers, P., Pringle, C.R., and Easton, A.J. 1991. Genes 1 and 2 of pneumonia virus of mice encode proteins which have little homology with the IC and IB proteins of human respiratory syncytial virus. J.Gen.Vir. 72: 2545-2549.<br>
Chambers, P. Pringle CR, Easton AJ. 1992. Sequence analysis of the gene encding the fusion glycoprotein of pneumonia virus of mice suggests possible conserved secondary structure elements in pramyxovirus fusion glycoproteins, J. Gen. Virol. 73:1717-1724.<br>
Coggins, W.B., Lefkowitz, E.J. and Sullender, W.M. 1998. Genetic variability among group A and group B respiratory syncytial viruses in a children's hospital. J. Clin. Microbiol. 36: 3552-3557.<br>
Collins, M.S. and Gough, R,E., Lister, S A., Chettle, N. and Eddy, R. 1986. Further characterisation of a virus associated with turkey rhiotracheitis. Vet. Rec. 119: 606.<br>
Collins, M.S. and Gough, R.E. 1988. Characterisation of a virus associated with turkey rhinotracheitis. J. Gen.Virol. 69: 909-916.<br><br>
Collins, M.S., Gough, R.B., and Alexander, D.J. 1993. Antigenic differentiation of avian pneumovirus isolates using polyclonal antisera and mouse monoclonal antibodies. Avian Pathology 22: 469-479.<br>
Collins, P.L., Mcintosh, K, Chanock, R.M. 1996. Respiratory syncytial virus. P. 1313-1351. In: B.N. Fields, D.M. Knipe, and P.M. Howley (ed.). Fields virology, 3rd ed., vol. 1 Lippincott-Raven, Philadelphia, Pa, USA-Cook,, J.K.A., Huggins, M.B., Orbell, S.J. and Senne, DA. 1999. Preliminiary antigenic characterization of an avian pneumovirus isolated from commercial turkeys in Colorado, USA. Avian pathol. 28: 607-617.<br>
Cook, J.KA. 2000. Avian rhinotracheitis. Rev. Sci.tech. off int. Epiz. 19 : 602-613.<br>
Dowell, S.F., Anderson, L.J.,Gary, H.E., Erdman, D.D., Plouffe, J.F., File, T.M., Marston, B.J. and Breiman, R.F. 1996. Respiratory syncytial virus is an important cause of community-acqviired lower respiratory infection among hospitalized adults. J. Infect .Dis.174: 456-462.<br>
Elvander, M. 1996. Severe respiratory disease in dairy cows caused by infection with bovine respiratory syncytial virus. Vet Rec. 138: 101-105.<br>
Elvander, M., Vilcek, S., Baule, C, Uttenthal, A., Ballagi-Pordany, A. and Belak, S.<br>
1998.<br>
Genetic and antigenic analysis of the G attachment protein of bovine respiratory<br>
syncytial virus strains. J. Gen. Virol. 79: 2939-2946.<br>
Englund, JA., Anderson, L.J., and Rhame, F.S, 1991. Nosocomial transmission of respiratory syncytial virus in immunocompromised adults. J. Clin. Microbiol. 29:115-119.<br>
Falsey, A.R. and Walsh, E.E. 2000. Respiratory syncytial virus infection in adults. Clin. Microb. Rev. 13: 371-84.<br><br>
Furze, J., Wertz, G,, Lerch, R. and Taylor, G. 1994. Antigenic heterogeneity of the attachment protein of bovine respiratory syncjiial virus. J. Gen. Virol. 75: 363-370.<br>
Gimenez, H.B., Cash, P. and Melvin, W.T. 1984. Monoclonal antibodies to human respiratory Syncytial virus and their use in comparison of different virus isolates. J. Gen. Virol. 65: 963-971.<br>
Gulati, B.R., Cameron, KT., Seal, B.S, Goyal, S.M., Halvorson, DA. and Njenga, M.K 2000.<br>
Development of a highly sensitive and specific enzyme-linked immunosorbent assay based on recombinant matrix protein for detection of avian pneumovirus antibodies. J, Clin. Microbiol. 38: 4010-4.<br>
Johnson, P.R., Spriggs M.K, Ohmsted, R.A. and Collins, P.L. 1987, The G glycoprotein of human respiratory syncytial virus subgroups A and B: extensive sequence divergence between antigenically related proteins. Proc. Natl. Acad. Sci. USA 84: 5625-5629.<br>
Juhasz, K. and Easton, A-J. 1994. Extensive sequence variation in the attachment (G) protein gene of avian pneummovirus: evidence for two distinct subgroups. J. Gen. Virol. 75: 2873-2880.<br>
LeaMaster, B.R., Evermann, J.Fâ€ž Mueller, M.K, Prieur, M.K and Schlie. J.V. 1983. Serologic studies on naturally occurring respiratory Syncytial virus and Haemophilus sommus infections in sheep. American Association of Veterinary Laboratory Diagnosticians 26: 265-276.<br>
Lehmkuhl, H.D., Smith, M.H., Cutlip, R.C, 1980. Morphogenesis and structiure of caprine respiratory syncytial virus. Arch.Vir. 65: 269-76.<br>
Lerch, R.A., Anderson, K and Wertz, G.W. 1990. Nucleotide sequence analysis and expression from recombinant vectors demonstrate that the attachment protein G of bovine respiratory syncytial virus is distinct from that of human respiratory synctial virus. J.Virol. 64: 5559-5569.<br><br>
Ling, R. and Pringle, C.R. 1988. Turkey rhinotracheitis virus: in vivo and in vitro polypeptide synthesis. J. Gen. Virol, 69: 917-923.<br>
Ling, R. and Pringle, C.R. 1989a. Polypeptides of pneumonia virus of mice. I. Immunological cross-reactions and post-translational modifications. J. Gen. Virol. 70: 1427-1440.<br>
Ling, R. and Pringle, C.R. 1989b. Polypeptides of pneumonia virus of mice. II. Characterization of the glycoproteins. J.Gen.Virol. 70: 1441-1452.<br>
McDougall, J.S. and Cook, J.KA. 1986. Turkey rhinotracheitis: preliminary investigations. Vet. Rec. 118: 206-207.<br>
Oberst, R.D., M.P. Hays, K.J. Heimessy, L.C. Stine, J.F. Evermann, and KeUing, C.L. 1993. Characteristic differences in reverse transcription polymerase chain reaction products of ovine, bovine and human respiratory syncytial viruses. J. Vet. Diagn. Investig. 5: 322-328.<br>
O'Loan, C.J., AUan, G., Baxter-Jones, C. and McNulty, M.S. 1989. An improved ELISA and serum neutralisation test for the detection of turkey rhinotracheitis virus antibodies. J. Virol. Meth. 25: 271-282.<br>
Paccaud, M.F. and Jacquier, C, 1970. A respiratory syncytial virus of bovine origin. Arch. Ges.Virusforsch. 30: 327-342.<br>
Pringle, C.R. 1999 Virus taxonomy at the Xith international congress of virology, Sydney, Australia 1999. Arch, Virol. 144/2: 2065-2070.<br>
Prozzi, D., Walravens, K, Langedijk, J.P,M, Daus, F., Kramps, J.A and Letesson, J.J. 1997. Antigenic and molecular analysis of the variability of bovine respiratory syncytial virus G glycoprotein. J. Gen. Virol. 78: 359-366.<br><br>
Randhawa, j.S., Marriott, A-C, Pringle, C.R., and A.J. Fasten 1997. Rescue of synthetic minireplicons establish the absence of the NSl and NS2 genes from avian pneumoviruses. J. Virol. 71; 9849-9854.<br>
Richter,C.B., Thigpen, J.E., Richter, C.S. and Mackenzie, J.M. 1988. Fatal pneumonia with terminal emaciation in nude mice caused by pneumonia vrius of mice. Lab. Anim. Sci. 38: 255-261.<br>
Rothbarth, P.H., Habova, J.J. and Masurel, N. 1988. Rapid diagnosis of infections caused by respiratory syncytial virus. Infection 16:252.<br>
Seal, B.S. 1998. Matrix protein gene nucleotide and predicted amino acid sequence demonstrate that the first US avian pneumovirus isolate is distinct form European strans. Virus Res. 58, 45-52.<br>
Seal, B.S., Sellers, H.S., Meinersmann, R.J. 2000. Fusion protein predicted amino acid sequence of the first US avian pneumovirus isolate and lack of heterogeneity among other US isolates. Virus Res. 66: 139-147.<br>
Selwyn, B.J. 1990. The epidemiology of acute respiratory tract infection in young childi'en: comparison findings from several developing countries. Rev. Infect. Dis. 12: S870-S888.<br>
Shin, H.J., Rajashekara, G., Jirjis, F.F., Shaw, D.P., Goyal, S.M., Halvorson, D.A. and Nagaraja, KV 2000. Specific detection of avian pneumovirus (APV) US isolates by RT-PCR. Arch. Virol. 145:1239-1246.<br>
Sullender, W.M. 2000. Respiratory syncytial virus genetic and antigenic diversity. Clin. Microb. Rev. 13: 1-15.<br>
Trudel, M., Nadon, F., Sinnard, C., Belanger, F., Alain, R., Seguin, C. and Lussier, G. 1989. Comparison of caprine, human and bovine strains of respiratory syncytial virus. Arch. Vir. 107: 141-149,<br><br>
Uttenthal, A., Jensen, N.P.B. and Blom, J.Y. 1996. Viral aetiology of enzootic pneumonia in Danish dairy herds, diagnostic tools and epidemiology. Vet. Rec. 139, 114-117.<br>
Valarcher, J., Bourhy, H., Gelfi, J. and Schelcher, F. 1999. Evaluation of a nested reverse transcription-PCR assay based on the nucleoprotein gene for diagnosis of spontaneous and experimental bovine respiratory syncytial virus infections, J. Clin. Microb. 37: 1858-1862<br>
Van Milaan, A.J., Sprenger, J.J., Rothbarth, P.H., Brandenburg, A.H., Masurel, N. and Claas, E.G. 1994. Detection of respiratory syncytial virus by RNA-polymerase chain reaction and differentiation of subgroups with oligonucleotide probes. J.MedVirol. 44:80-87.<br>
Vilcek, S, Elvander, M., Ballagi-Pordany, A., and Belak, S. 1994. Development of nested PCR assays for detection of bovine respiratory sjnicytial virus in clinical samples. J. Glin. Microb. 32: 2225-2231.<br>
Walravens, K, Kettmann, R., Gollard, A., Goppe, P. and Burny, A.. 1990. Sequence comparison between the fusion protein of human and bovine respiratory syncytial viruses. J. Gen. Virol. 71: 3009-3014.<br>
Weir, E.G., Brownstein, D.G., Smith, A.L. and Johnson, E.A. 1988. Respiratory disease and wasting in athymic mice infected with pneumonia virus of mice. Lab. Anim. Sci. 34: 35-37.<br><br><br><br><br><br>
WE CLAIM:<br>
1.	An isolated essentially mammalian negative-sense single stranded RNA virus<br>
(MPV) belonging to the sub-family Pneumovirinae of the family Paramyxoviridae and identifiable as phylogenetically corresponding to the genus Metapneumovirus.<br>
2.	An isolated negative-sense single stranded RNA virus (MPV) belonging to the subfamily Pneumovirinae of the family Paramyxoviridae and identifiable as phylogenetically corresponding to the genus Metapneumovirus by determining a nucleic acid sequence of said virus and testing it in phylogenetic tree analyses wherein maximum likelihood trees are generated using 100 bootstraps and 3 jumbles and finding it to be more closely phylogenetically corresponding to a virus isolate deposited as 1-2614 with CNCM, Paris than it is corresponding to a virus isolate of avian pneumovirus (APV) also known as turkey rhinotracheitis virus (TRTV), the aetiological agent of avian rhinotracheitis.<br>
3.	The virus according to claim 2 wherein said avian pneumovirus comprises APV type C(APV-C).<br>
4.	The virus according to claim 1 to 3 wherein said nucleic acid sequence comprises an open reading frame (ORF) encoding a viral protein of said virus.<br>
5.	The virus according to claim 4 wherein said open reading frame is selected from the group of ORFs encoding the N, P, M, and F proteins.<br>
6.	The virus according to claim 5 wherein said open reading frame is selected from the group of ORFs encoding the SH or G proteins.<br>
7.	The virus according to anyone of claims 1 to 6 comprising a nucleic acid or functional fragment phylogenetically corresponding to a sequence shown in figure 6.<br><br>
8.	The virus according to anyone of claims 1 to 7 comprising an MPV isolate deposited as 1-2614 with CNCM, Institute Pasteur, Paris or a virus isolate phylogenetically corresponding therewith.<br>
9.	The virus according to claim 8 isolatable from a human with respiratory tract illness.<br>
10.	An isolated or recombinant nucleic acid or MPV-specific functional fragment<br>
thereof obtainable from a virus according to anyone of claims 1 to 9.<br>
11.	A vector comprising a nucleic acid according to claim 10.<br>
12.	A host cell comprising a nucleic acid according to claim 10 or a vector according to claim 11.<br>
13.	An isolated or recombinant proteinaceous molecule or MPV-specific functional fragment thereof encoded by a nucleic acid according to claim 10.<br>
14.	An antigen comprising a proteinaceous molecule or MPV-specific fimctional<br>
fragment thereof according to claim 13.<br>
15.	An antibody specifically directed against an antigen according to claim 14.<br>
16.	A method for identifying a viral isolate as an MPV comprising reacting said viral isolate or a component thereof with an antibody according to claim 15.<br>
17.	A method for identifying a viral isolate as an MPV comprising reacting said viral isolate or a component thereof with a nucleic acid according to claim 10.<br><br>
18.	A method according to claim 16 or 17 wherein said MPV comprises a human MPV.<br>
19.	A viral isolate identifiable with a method according to anyone of claims 16 to 18 as a mammalian negative-sense single stranded RNA virus within the sub-family Pneumovirinae of the family Paramyxoviridae and identifiable as phylogenetically corresponding to the genus Metapneumovirus.<br>
20.	A method for virologically detecting an MPV in a sample comprising determining in the sample the presence of a viral isolate or component thereof by reacting said sample with a nucleic acid according to claim 10 or an antibody according to claim 15.<br>
21.	A method for serologically detecting an MPV in a sample comprising determining in the sample the presence of an antibody specifically directed against an MPV or component thereof by reacting said sample with a proteinaceous molecule or fragment thereof according to claim 13 or an antigen according to claim 14.<br>
22.	A diagnostic kit for diagnosing an MPV infection comprising a virus according to anyone of claims 1  to 9, a nucleic acid according to claim 10, a proteinaceous molecule or fragment thereof according to claim 13, an antigen according to claim 14 and/or an antibody according to claim 15.<br>
23.	A pharmaceutical composition comprising a virus according to any one claims 1 to 9, a nucleic acid according to claim 10, a vector according to claim 11, a host cell according to claim 12, a proteinaceous molecule or fragment thereof according to claim 13, an antigen according to claim 14, or an antibody according to claim 15.<br><br>
24.	A method to obtain an antiviral agent comprising establishing a cell culture<br>
comprising a virus according to any one of claims 1 to 9, treating said culture with a<br>
candidate antiviral agent, determining the effect of said agent on said virus or its<br>
infection of said culture, and selecting an anitviral agent with an antiviral effect.<br>
25.	An antiviral agent obtainable according to the method according to claim 24.<br>
26.	A pharmaceutical composition comprising an antiviral agent according to claim<br>
25.<br>
27.	A method for virologically detecting an MPV in a sample comprising determining in the sample the presence of a viral isolate or component thereof by reacting said sample with a nucleic acid or an antibody specifically reactive with a component of an avian pneumovirus (APV), said nucleic acid or antibody being cross-reactive with a component MPV.<br>
28.	A method for serologically detecting an MPV in a sample comprising determining in the sample the presence of an antibody directed against an MPV or component thereof by reacting said sample with a proteinaceous molecule or fragment thereof or antigen derived from an APV isolate or component thereof, said molecule, fragment or antigen selected for being essentially homologous with a component of MPV.<br>
29.	A method for virologically detecting an APV in a sample comprising determining in the sample the presence of a viral isolate or component thereof by reacting said sample with a nucleic acid according to claim 10 or an antibody according to claim 15 said nucleic acid or antibody being cross-reactive with a component of APV.<br><br>
30.	A method for serologically detecting an APV in a sample comprising determining<br>
in the sample the presence of an antibody specifically directed against an APV or<br>
component thereof by reacting said sample with a proteinaceous molecule or fragment<br>
thereof according to claim 13 or an antigen according to claim 14, said molecule,<br>
fragment or antigen selected for being essentially homologous with a component of<br>
APV.<br>
31.	A diagnostic test kit designed to detect APV specific antibodies for the detection of an antibody directed against MPV.<br>
32.	The diagnostic test kit according to claim 31 wherein said test comprises an enzyme immune assay (EIA).<br><br>
33.	A method for the detection of an antibody directed against MPV in a sample comprising testing said sample in a diagnostic test designed to detect APV specific antibodies.<br>
34.	A method according to claim 33 wherein said test comprises an enzyme immune assay (ETA).<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI3Mi1jaGVucC0yMDAzIGFic3RyYWN0IGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">1272-chenp-2003 abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI3Mi1jaGVucC0yMDAzIGNsYWltcyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1272-chenp-2003 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI3Mi1jaGVucC0yMDAzIGRlc2NyaXB0aW9uIChjb21wbGV0ZSkgZHVwbGljYXRlLTEucGRm" target="_blank" style="word-wrap:break-word;">1272-chenp-2003 description (complete) duplicate-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI3Mi1jaGVucC0yMDAzIGRlc2NyaXB0aW9uIChjb21wbGV0ZSkgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">1272-chenp-2003 description (complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI3Mi1jaGVucC0yMDAzIGRyYXdpbmdzIGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">1272-chenp-2003 drawings duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI3Mi1jaGVucC0yMDAzLWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1272-chenp-2003-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI3Mi1jaGVucC0yMDAzLWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1272-chenp-2003-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI3Mi1jaGVucC0yMDAzLWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1272-chenp-2003-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI3Mi1jaGVucC0yMDAzLWNvcnJlc3BvbmRuZWNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">1272-chenp-2003-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI3Mi1jaGVucC0yMDAzLWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">1272-chenp-2003-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI3Mi1jaGVucC0yMDAzLWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1272-chenp-2003-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI3Mi1jaGVucC0yMDAzLWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">1272-chenp-2003-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI3Mi1jaGVucC0yMDAzLWZvcm0gMjYucGRm" target="_blank" style="word-wrap:break-word;">1272-chenp-2003-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI3Mi1jaGVucC0yMDAzLWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">1272-chenp-2003-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI3Mi1jaGVucC0yMDAzLWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">1272-chenp-2003-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI3Mi1jaGVucC0yMDAzLW90aGVyIGRvY3VtZW50cy5wZGY=" target="_blank" style="word-wrap:break-word;">1272-chenp-2003-other documents.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI3Mi1jaGVucC0yMDAzLXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1272-chenp-2003-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="223175-a-patterned-fabric.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="223177-sequentially-operated-twin-filaments-electric-bulb.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>223176</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1272/CHENP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>47/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Nov-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>05-Sep-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>13-Aug-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>VIRONOVATIVE B.V</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>BURGEMEESTER OUDLAAN 50, NL-3062 PA ROTTERDAM,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>DE JONG, JAN, CORNELIUS</td>
											<td>KRUGERLAAN 205, NL-2806 EH GOUDA,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>FOUCHIER, RONALDUS, ADRIANUS, MARIA</td>
											<td>ESENBURGSINGEL 44A, NL-3021 AR ROTTERDAM,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>VAN DEN HOOGEN, BERNADETTA, GERARDA</td>
											<td>ESSENBURGSSINGEL 44A, NL-3021 AR ROTTERDAM,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>OSTERHAUS, ALBERTUS, DOMINICUS, MARCELLINUS, ERASMUS</td>
											<td>DR BREVEESTRAAT 16, NL-3981 CH BUNNIK,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>GROEN, JAN</td>
											<td>ANHOLTKADE 2, NL-3446 BM WOERDEN,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K14/135</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/NL02/00040</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-01-18</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>01203985.5</td>
									<td>2001-10-18</td>
								    <td>EUROPEAN UNION</td>
								</tr>
								<tr>
									<td>2</td>
									<td>01200213.5</td>
									<td>2001-01-19</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/223176-a-novel-negative-sense-single-stranded-rna-virus by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:37:07 GMT -->
</html>
