<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/223190-a-novel-fungal-lipoxygenase by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:34:46 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 223190:A NOVEL FUNGAL LIPOXYGENASE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A NOVEL FUNGAL LIPOXYGENASE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The inventors have found a novel fungal lipoxygenase from Magnaporthe salvinli and determined its sequence. They have sequenced the gene and cloned it into E. coli and deposited the clone. Oligonucleotide probes based on the sequence information are useful for screening a eukaryotic library to obtain a lipoxygeriase. The lipoxygenase is useful in baking and in a detergent.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
LIPOXYGENASE<br>
FIELD OF THE INVENTION<br>
The present invention relates to a lipoxygenase and a polynucleotide encoding it.<br>
BACKGROUND OF THE INVENTION<br>
Lipoxygenase (EC 1.13.11.12) is an enzyme that catalyzes the oxygenation of polyunsaturated fatty acids such as linoleic acid, linolenic acid and arachidonic acid, which contain a cis,cis-l ,4-pentadiene unit and produces hydroperoxides of these fatty acids. The enzyme is widely distributed in plants and animals. A number of lipoxygenase genes have been isolated from various plant and mammalian sources.<br>
On the other hand, only a limited number of microbial lipoxygenases are known, and no lipoxygenase gene of microbial origin has been described. Su and Oliw, J. Biological Chemistry. 273 (21), 13072-79 (1998) describe a lipoxygenase from Gaeumannomyces graminis.<br>
SUMMARY OF THE INVENTION<br>
The inventors have found a novel fungal lipoxygenase and determined its sequence, which can be used for the production of the enzyme in industrial scale. They have cloned the gene into E. coli and deposited the clone.<br>
Accordingly, the invention provides a lipoxygenase which is:<br>
a)	a polypeptide encoded by a DNA sequence cloned into plasmid pUC19 present in Escherichia coli deposited as DSM 14139,<br>
b)	a polypeptide having an amino acid sequence as the mature peptide shown in SEQ ID NO: 1, or which can be obtained therefrom by substitution, deletion, and/or insertion of one or more amino acids,<br>
c)	an analogue of the polypeptide defined in (a) or (b) which:<br>
i) has at least 50 % homology with said polypeptide,<br>
ii) is immunologically reactive with an antibody raised against said<br>
polypeptide in purified form, iii) is an allelic variant of said polypeptide, or<br>
d)	a polypeptide encoded by DNA that hybridizes under low stringency conditions<br>
with a complementary strand of<br>
i) the DNA sequence cloned into plasmid pUC19 present in Escherichia coli deposited as DSM 14139 or<br>
ii) the DNA sequence of SEQ ID NO: 1 encoding the mature polypeptide or a subsequence thereof having at least 100 nucleotides.<br>
The invention also provides a polynucleotide which comprises:<br><br>
a)	the partial DNA sequence encoding a mature lipoxygenase cloned into a plasmid present in Escherichia coli DSM 14139.<br>
b)	the partial DNA sequence encoding a mature lipoxygenase shown in SEQ ID NO:<br>
1,<br>
c)	an analogue of the sequence defined in a) or b) which encodes a lipoxygenase<br>
and<br>
i) has at least 60 % homology with said DNA sequence, or<br>
ii) hybridizes at high stringency with a complementary strand of said DNA sequence or a subsequence thereof having at least 100 nucleotides,<br>
iii) is an allelic variant thereof, or<br>
d)	a complementary strand of a), b) or c).<br>
Other aspects of the invention provide a nucleic acid construct and a recombinant expression vector comprising the polynucleotide, a recombinant host cell comprising the construct or the vector, and a method of producing a lipoxygenase by cultivating the cell. Further, the Invention provides a method of screening a eukaryotic library to obtain a lipoxygenase and an oligonucleotides probe useful for screening. Finally, the invention provides use of the lipoxygenase in baking and in a detergent.<br>
DETAILED DESCRIPTION OF THE INVENTION Genomic DNA source<br>
A lipoxygenase gene of the invention may be derived from a filamentous fungus, e.g. an Ascomycota, particulariy Magnaporthaceae, such as a strain of Magnaporthe, particularly Magnaporthe salvinii Cattaneo (Mycologia 64 (1), 110 (1972)). The species is also known under the synonyms Curvulaha sigmoidea, Helminthosporium sigmoideum, Leptosphaeha salvinii, Nal<ataea sigmoidea sderotium oryzae and vakrabeeja sigmoidea. an example is the strain a samnii ifo></ataea>
Altematively, the gene may be isolated from Pyricularia, e.g. P. oryzae or P. ghsea, e.g. P. oryzae IFO 30517. The IFO strains are available on commercial terms from Institute for Fermentation, Osaka (IFO), 17-85, Juso-honmachi 2-chome, Yodogawa-ku, Osaka 532-8686, Japan.<br>
The lipoxygenase gene may be isolated from these organisms using probes designed on the basis of the DNA sequences in this specification.<br>
A strain of Escherichia coli containing a lipoxygenase gene from M. salvinii IFO 6642 was deposited by the inventors under the terms of the Budapest Treaty with the DSMZ -Deutsche Sammmlung von Microorganismen und Zellkutturen GmbH, Mascheroder Weg 1b, D-38124 Braunschweig DE, Germany. The deposit date was 28 February 2001, and the accession number was DSM 14139.<br><br>
Production of lipoxygenase by cultivation of transformant<br>
The lipoxygenase of the invention may be produced by transforming a suitable host ceil with a DNA sequence encoding the lipoxygenase, cultivating the transformed organism under conditions permitting the production of the enzyme, and recovering the enzyme from the<br>
culture.<br>
The host organism may be a eukaryotic cell, in particular a fungal cell, such as a yeast cell or a filamentous fungal cell, e.g. a strain of Aspergillus, Fusarium, Trichoderma or Saccharomyces, particularly A. niger, A. oryzae, F. graminearum, F. sambudnum, F. cerealis or S. cerevlsiae. The production of the lipoxygenase in such host organisms may be done by the general methods described in EP 238,023 (Novo Nordisk), WO 96/00787 (Novo Nordisk) or EP 244,234 (AIko).<br>
Properties of LOX<br>
The lipoxygenase of the invention is able to oxidize a wide range of substrates containing a c/s-c/s-pentadienyl moiety. Thus, it acts on polyunsaturated fatty acids such as linoleic acid (18 carbon atoms, 2 double bonds), linolenic acid (18:3), arachidonic acid (20:4), eicosapentaenoic acid (EPA, 20:5) and docosahexaenoic acid (DHA, 22:6). It also acts on substrates other than fatty acids, such as methyl linoleate and probably also triglycerides. The enzyme has a very low Michaelis constant (KM) for linoleic acid and a high specificity (Vmax/Ku) towards this substrate.<br>
The lipoxygenase from M. salvinii is a 9-lipoxygenase, i.e. it oxidizes the double bond between carbon atoms 9 and 10 in linoleic acid and linolenic acid.<br>
The lipoxygenase from M. salvinii has optimum activity around pH 7, and it is highly active over a broad pH range 3-12, having more than 50 % of optimum activity in the range pH 6-11. It is stable after overnight incubation at pH 5-11.<br>
The native lipoxygenase from M. saMnii has optimum activity at 50-60°C. It is quite active at 40-60°C, and the activity begins to decline at 70 °C. The lipoxygenase is stable after 30 minutes incubation at pH 7 at temperatures up to 50°C.<br>
The reaction rate for recombinant lipoxygenase (expressed in A. oryzae) increases nearly ten times at the optimal temperature for catalysis compared to the rate obtained at room temperature. The maximum reaction rate is obtained at 67.5°C. A steep decrease in rate constant is seen above the temperature optimum. It is believed that glycosylation renders the recombinant enzyme more stable towards heat than the wild-type enzyme.<br>
The recombinant lipoxygenase is quite stable at temperatures up to 50°C for at least one hour. The activity drops in a linear fashion at higher temperatures between 50-60°C, and no activity is detected after incubations above 60°C for one hour. No activity loss is detected during incubation at temperatures below 45°C.<br><br>
Frozen solutions of the lipoxygenase lose some activity during storage. With addition of 10 % glycerol there is no discemible activity loss after two weeks storage at -20°C, and the enzyme survived repeated cycles of thaw-freeze without loss of activity.<br>
The lipoxygenase of the invention has good stability in the presence of anionic surfactants Thus, the lipoxygenase from M. salvinii is stable in the presence of 400 ppm of LAS (linear alkyl-benzene sulfonate).<br>
Use of lipoxygenase<br>
The lipoxygenase can be used for green flavor synthesis, e.g. nonenal from 9-hydoperoxide of linolenic acid. The synthesis may be done in analogy with Whitehead et al.1995, Cereal foods world 40(4), 193-197 and US 4769243.<br>
The lipoxygenase can also be used for plant hormone synthesis as described in JP H11-29410.<br>
Also the lipoxygenase is a good oxidant of carotenoids, so it can be used for bleaching of foodstuffs such as flour, oil or marine food including carotenoids or carotenoid-<br>
iike pigments.<br>
The oxidation activity can be utilized for cross-linking of protein, oil. starch, fiber and mixture of these. Cross-linking of chemical compounds can be utilized for synthesis of polymer to give plastic fiber or plastic resin. It can be used for bleaching as a detergent for phenolic, carotenoid or fatty stains or dinginess. Or it can be used for bleaching of waste water or textile dye.<br>
Lipoxygenase can be used for bleaching of plant or marine food materials containing of carotenoids. Thus it could be used for bleaching of flour for bread, noodle or pasta, or bleaching of fish meat or fish oil containing astaxanthin.<br>
It also can be used for cross-linking of protein, oil, starch, plant-fiber or mixture of these in presence of fatty acid, oil or fats. It is useful to change the texture or physical properties of foodstuff or to control of flavor for fat and oil, or to produce polymers made of natural staff beside food use. Cross-linked compounds can be chemical compounds, e.g. phenolic, carbonyl, carboxyl or amide compounds or mixture of these. It could be used for synthesis of plastic fiber or resin.<br>
Other usages of lipoxygenase can be the synthesis of flavor compound such as hexanal or hexenal together as synergy effect of hydroperoxide lyase. Or in case plant material is used as the source of above two enzymes, lipoxygenase can be added to it to improve the yield of flavor compound. The similar can be done for synthesis of plant or animal hormones.<br>
Finally it can be used as bleaching agent. It can be used in detergents for bleaching Of phenolic, carotenoid, fatty stains or dinginess of clothes. Or it can be used for bleaching of<br><br>
textile dye or dye for pulp industry in waste water or changing of dye texture. Recombinant expression vector<br>
The expression vector of the invention typically includes control sequences encoding<br>
a promoter, operator, ribosome binding site, translation initiation signal, and, optionally, a<br>
    selectable marker, a transcription terminator, a repressor gene or various activator genes. The<br>
vector may be an autonomously replicating vector, or it may be integrated into the host cell<br>
genome.<br>
Production by cultivation of transformant<br>
The lipoxygenase of the invention may be produced by transforming a suitable host cell with a DNA sequence encoding the lipoxyenase, cultivating the transformed organism under conditions pemiitting the production of the enzyme, and recovering the enzyme from the culture.<br>
The host organism may be a eukaryotic cell, in particular a fungal cell, such as a yeast  cell or a filamentous fungal cell, e.g. a strain of Aspergillus, Fusahum, Thchoderma or Saccharomyces, particularly A. niger, A. oryzae, F. graminearum, F. sambudnum, F. cerealls or S. cerevisiae. The production of the lipoxygenase in such host organisms may be done by the general methods described in EP 238,023 (Novo Nordisk), WO 96/00787 (Novo Nordisk) or EP 244,234 (AIko).<br>
The enzyme can be purified in one step by cation-exchange chromatography to homogeneity.<br>
Nucleotide probe<br>
A nucleotide probe may be designed on the basis of the DNA sequence of SEQ ID NO: 1 or the polypeptide sequence of SEQ ID NO; 2, particularly the mature peptide part. The probe may be used in screening for LOX-encoding DNA as described below.<br>
A synthetic oligonucleotide primer may be prepared by standard techniques (eg, as described in Sambrook J, Fritsch EF, Maniatis T (1989) Molecular cloning: a laboratory manual (2nd edn.) Cold Spring Harbor Laboratory. Cold Spring Harbor, New Yoric) on the basis of the mature part of the amino acid sequence in SEQ ID NO: 2 or the con-esponding part of the DNA sequence. It may be a degenerate probe and will typically contain at least 20 nucleotides.<br>
Screening of eukaryotic DNA library<br>
A polypeptide with lipoxygenase activity may be obtained by a method comprising:<br>
a)	preparing a eukaryotic DNA library,<br>
b)	screening the library to select DNA molecules which hybridize to the probe<br><br>
described above,<br>
c)	transforming host cells with the selected DNA molecules,<br>
d)	cultivating the transformed host cells to express polypeptides encoded by the DNA<br>
molecules, and<br>
e)	assaying the expressed polypeptides to select polypeptides having lipoxygenase<br>
activity.<br>
The eukaryotic DNA library may be prepared by conventional methods. It may include genomic DNA or double-stranded cDNA derived from suitable sources such as those<br>
described above.<br>
Molecular screening for DNA sequences may be carried out by polymerase chain<br>
reaction (PCR) followed by hybridization.<br>
In accordance with well-known procedures, the PCR fragment generated in the molecular screening may be isolated and subcloned into a suitable vector. The PCR fragment may be used for screening DNA libraries by e.g. colony or plaque hybridization.<br>
Hybridization<br>
The hybridization is used to indicate that a given DNA sequence is analogous to a nucleotide probe corresponding to a DNA sequence of the invention. The hybridization may be done at low, medium or high stringency. One example of hybridization conditions is described in detail below.<br>
Suitable conditions for detenrnining hybridization between a nucleotide probe and a homologous DNA or RNA sequence involves presoaking of the filter containing the DNA fragments or RNA in 5 x SSC (standard saline citrate) for 10 min, and prehybridization of the filter in a solution of 5 x SSC (Sambrook et al. 1989), 5 x Denhardt's solution (Sambrook et al. 1989), 0.5 % SDS and 100µg/ml of denatured sonicated salmon sperm DNA (Sambrook et al. 1989), followed by hybridization in the same solution containing a random-primed (Feinberg, A. P. and Vogelstein, B. (1983) Anal. Biochem. 132:6-13), 32P-dCTP-labeled (specific activity &gt; 1 X 109 cpm/µg) probe for 12 hours at approx. 45°C. The filter is then washed two times for 30 minutes in 2 x SSC, 0.5 % SDS at a temperature of at least 55°C, particularty at least 60°C, more particularty at least 65°C, e.g. at least 70°C, or at least 75°C.<br>
Molecules to which the oligonucleotide probe hybridizes under these conditions are detected using an x-ray film-Alignment and homology<br>
The lipoxygenase and the nucleotide sequence of the invention may have homologies to the disclosed sequences of at least 75 % or at least 85 %, particularly at least 90 % or at least 95 %, e.g. at least 98 %-<br><br>
For purposes of the present invention, alignments of sequences and calculation of homology scores were done using a Needleman-Wunsch alignment (i.e. global alignment), useful for both protein and DNA alignments. The default scoring matrices BLOSUM50 and the identity matrix are used for protein and DNA alignments respectively. The penalty for the first residue in a gap is -12 for proteins and -16 for DNA, while the penalty for additional residues in a gap is -2 for proteins and -4 for DNA. Alignment is from the FASTA package version v20u6 (W. R. Pearson and D. J. Lipman (1988), "Improved Tools for Biological Sequence Analysis". PNAS 85:2444-2448, and W. R. Pearson (1990) "Rapid and Sensitive Sequence Comparison with FASTP and FASTA", Methods in Enzymology, 183:63-98).<br>
EXAMPLES Materials and Methods<br>
Molecular cloning techniques are described in Sambrook et al. (1989).<br>
The following commercial plasmids and E. coli strains were used for sub-cloning and DNA library construction:<br>
pT7Blue (Novagen)<br>
pUC19 (TOYOBO, Japan)<br>
E. coli JM109 (TOYOBO, Japan)<br>
E coli DH12S (GIBCO BRL, Life Technologies, USA)<br>
Labeling and detection of hybridization probe was done using DIG-labeling and detection Kit (Boehringer Manheim). Nylon membrane Hybond-N+ (Amersham, England) was used for DNA transfer for colony hybridization.<br>
Soybean lipoxygenase (type l-B) (cat.# L7315) and astaxanthin (cat.# A-9335) was supplied by Sigma, b-carotene (cat.# 031-05533) were supplied by Wako.<br>
Media and buffer solution<br>
COVE-ar: per liter 342.3 g sucrose, 20 ml COVE salt solution, 10 mM acryl amide, 15 mM CsCb, 30 g Agar nobie (Difco)<br>
COVE2-ar: per liter 30 g sucrose, 20 ml COVE salt solution, 10 mM acrylamide, 30 g Agar noble (Difco)<br>
COVE salt solution: per liter 26 g KCI, 26 g MgS04-7H20, 76 g KH2PO4, 50ml Cove trace metals.<br>
Cove trace metals: per liter 0.04 g NaB4O7-10H2O, 0.4 g CUSO4-5H2O, 1.2 g FeS04-7H2O, 0.7 g MnS04-H20, 0.7 g Na2Mo02-2H20, 0.7 g ZnS04-7H20.<br>
AMG trace metals: per liter 14.3 g ZnS04-7H20, 2.5 g CUSO4-5H2O, 0.5 g NiCb. 13.8 g FeS04, 8.5 g MnS04, 3.0 g citric acid.<br>
YPG: per liter 4 g yeast extract, 1 g KH2PO4, 0.5 g MgS04-7H20, 15 g glucose, pH<br><br>
STC: 0.8 M Sorbitol, 25 mM Tris pH 8, 25 mWI CaCb.<br>
STPC: 40% PEG4000 in STC buffer.<br>
Cove top agarose: per liter 342.3 g sucrose, 20 ml COVE salt solution, 10 mM Acetamide, 10 g low melt agarose.<br>
MS-9: per liter 30 g soybean powder, 20 g glycerol, pH 6.0.<br>
MDU-2Bp: per liter 45 g maltose-IH2O, 7 g yeast extract, 12 g KH2PO4,1 g MgS04-7H2O, 2 g K2SO4, 5 g Urea, 1 g NaCI, 0.5 ml AMG trace metal solution pH 5.0.<br>
Host organism<br>
Aspergillus oryzae BECh2 is described in WO 00/39322. It is a mutant of JaL228<br>
(described in WO98/123000), which is a mutant of IF04177.<br>
Transformation of A. oryzae<br>
Aspergillus oryzae strain BECh2 was inoculated in 100 ml of YPG medium and incubated at 32°C for 16 hours with stining at 80 rpm. Grown mycelia was collected by filtration followed by washing with 0.6 M KCI and re-suspended in 30 ml of 0.6 M KCI containing Glucanex® (Novozymes) at the concentration of 30 µl/ml. The mixture was incubated at 32°C with the agitation at 60 rpm until protoplasts were formed. After filtration to remove the remained mycelia, protoplasts were collected by centrifugation and washed with STC buffer twice. The protoplasts were cournted with a hematitometer and re-suspended in a solution of STC:STPC:DMSO (8:2:0.1) to a final concentration of 1.2 x 107 protoplasts/ml. About 4 µg of DNA was added to 100 pi of protoplast solution, mixed gently and incubated on ice for 30 minutes. 1 µl STPC buffer was added to the mixture and incubated at 37°C for another 30 minutes. After the addition of 10 ml of Cove top agarose pre-warmed at 50°C, the reaction mixture was poured onto COVE-ar agar plates. The plates were incubated at 32°C for 5 days.<br>
SDS-PAGE<br>
SDS polyacrylamide electrophoresis was carried out using the commercialized gel PAGEL AE6000 NPU-7.5L (7.5T%) with the apparatus AE-6400 (Atto, Japan) following the provided protocol. 15 pi of sample was suspended in 15 µlof 2x conc, of sample loading buffer (100 mM Tris-HCI (pH 6.8), 200 mM Dithiothreitol, 4% SDS, 0.2% Bromophenol blue and 20% glycerol) and boiled for 5 minutes. 20 pi of sample solution was applied to a polyacrylamide gel, and subjected for electrophoresis in the running buffer (25 mM Tris, 0.1% SDS, 192 mM Glycine) at 20 mA per gel. Resulting gel was stained with SYPRO Orange and detected by molecular Imager FX (BIO-RAD)<br><br>
Assays for lipoxygenase activity<br>
Spectrophotometric assay<br>
Lipoxygenase activity was determined spectrophotometrically at 25°C by following the formation of hydroperoxides with the absorbance at 234 nm. To 0.98 ml of the buffer (50 mM KH2P04/NaHP04, pH 7.0), lOµI of substrate solution (lOmM linolenic acid dispersed with 0.2% Tween20) was added and the reaction was started by the addition of 10 pi of enzyme solution. One unit causes an increase in absorbance at 234 nm of 0.001/min.<br>
FOX assay<br>
The assay was initiated by the addition of 20 ^l enzyme solution to 80 \i\ of 50 mM each buffer containing 0.7 mM linolenic acid dispersed with 0.02% of Tween 20 using Hiscotron, and incubated for 10 min. The assay was temriinated by the addition of 900 \x\ of FOX reagent: sulfuric acid (25 mM), xylenol orange(100 µM), iron(ll) sulfate (100 µM), butylated hydroxyloluen (4 mM) in methanol:water (9:1). Blanks contained only substrate solution during the incubation, but enzyme solution was added after the addition of FOX reagent. The yellow color of acidified xylenol orange was converted to a blue color by the lipid hydroperoxide-mediated oxidation of Fe2+ ions with the dye. Absorbance of the Fe3+complex at 620 nm was measured 1 hour after the addition of FOX reagent.<br>
Bleaching assay<br>
Bleaching effect by lipoxygenase was examined spectrophotometrically at 25°C by following the absortaance at 470 nm. The pigment solution was prepared as follows. 150 ul of stock pigment solution (1mg each pigment in 1ml chloroform) was evaporated to be dry. Then 30 ml of the buffer (50 mM KH2P04/NaHP04, pH 7.0) with 0.3% of Tween 20 was added slowly and the pigment was dissolved. To 0.98 ml of the pigment solution. 10µl of substrate solution (lOmM linolenic acid dispersed with 0.2% of Tween20) was added and the reaction was started by the addition of 10 pi of enzyme solution.<br>
Example 1: Cloning of genomic LOX gene from M, salvinii<br>
Genomic DNA from Magnaporthe saMnii was digested with Sac I and separated on 1.0% agarose gel. Around 2.5 kbp of DNA digestion was recovered from the gel and ligated with BAP treated pUC19 linearized by Sac I. Ligation mixture was transformed into E. coli DH12S to construct a partial genomic library. It was screened, and a lipoxygenase-positive E. coli colony was isolated and the plasmid, termed pSG28, was recovered. The plasmid pSG28 contained a 2.5 kbp Sac\ genpmic fragment that contained the presumed LOX homologMe<br><br>
equence. The sequence of 1973 bp out of 2.5 kbp is shown as SEQ. ID 1<br>
Introns were identified and are indicated in SEQ ID NO: 1. The splice sites were redicted as described in S.M. Hebsgaard et al., Nucleic Acids Research, 1996. Vol. 24, No.<br>
17,3439-3452.<br>
The presumed open reading frame consisted of 1851 bp, and the deduced amino acid sequence corresponded to 617 amino acids, shown as SEQ ID NO: 2. The molecular<br>
■nass was estimated as 67500 Da.<br>
The E. coli DH12S harboring plasmid pSG28 was deposited at DSMZ as DSM 14139<br>
with the accession date 2001-02-28.<br>
Example 2: Expression of Af. saMnii LOX in A, oryzae<br>
Construction of expression plasmid<br>
The partial genomic sequence of M. saMnii genomic gene was amplified by PCR using pSG28 as a template. Primer 3 and 4 (SEQ ID NO: 3 and 4) were designed to make BamH I and Xho I sites at both ends of the PCR product (nucleotides 4-9 of primer 3 and 5-10 of primer 4, respectively). PCR reaction mbdure comprised of 2,5 mM dNTP, 30 pmol each of primer 3 and 4, 5 units of LA taq polymerase (Tai^ara) and supplied GC buffer I. Reaction condition was shown below. LA taq polymerase was added to the reaction mixture after step 1.<br><br><br>
gene is deficient and can be complemented with S. cerevisiae Ura3. Resulting plasmid was termed pSG30.<br>
Expression of M. salvinii LOX in A. oryzae<br>
A. oryzae BECh2 was transformed with the plasmid pSG30 and selection positive transformants were isolated. Transformants were grown on COVE 2 -ar at 32°C for 5 days and inoculated to 100 ml of MS-9 shaking flask. After the cultivation with vigorous agitation at 32°C for 1 day, 3 ml of each culture was transferred to 100 ml of MDU-2Bp in shaking flask to cultivate at 32°C for 3 days. Culture broth was centrifuged at 3500 rpm for 10 minutes and<br>
supernatant was collected.<br>
Lipoxygenase activities of the supernatant were determined spectrophotometrlcally as described before. Positive transformants showed about 100,000U/ml culture broth while untransformed A, oryzae BECh2 showed no activity. Culture supernatant was also subjected to SDS-PAGE analysis. Positive transformants showed 80-100kDa smear band which indicated the protein was heavily glycosylated. Untransformed A oryzae BECh2 did not show any significant bands.<br><br><br><br>
Example 4: Temperature dependence of lipoxygenase activity<br>
The effect of temperature on the M. salvinii lipoxygenase was studied by 10 min incubation at pH 7.0.<br>
Temperature     Relative Activity (%)<br>
25	501<br>
40	90LO<br><br><br><br><br>
The results show that M. salvinii LOX bleaches the pigment solutions. Soybean LOX showed little effect on bleaching.<br><br><br><br><br><br>
WE  CLAIM<br>
1.	A lipoxygenase which is:<br>
a)	a polypeptide encoded by a DNA sequence cloned into plasmid pUC19 present in Escherichia coil deposited as DSM 14139,<br>
b)	a polypeptide having an amino acid sequence as the mature peptide shown in SEQ ID NO: 1, or which can be obtained therefrom by substitution, deletion, and/or insertion of one or more amino acids,<br>
c)	an analogue of the polypeptide defined in (a) or (b) which:<br>
i) has at least 50 % homology with said polypeptide,<br>
ii) is immunologically reactive with an antibody raised against said<br>
polypeptide in purified fomi,<br>
iii) is an allelic variant of said polypeptide, or<br>
d)	a polypeptide encoded by DNA that hybridizes under low stringency conditions with a<br>
complementary strand of<br>
i) the DNA sequence cloned into plasmid pUC19 present in Escherichia coli deposited as DSM 14139 or<br>
ii)the DNA sequence of SEQ ID NO: 1 encoding the mature polypeptide or a subsequence thereof having at least 100 nucleotides.<br>
2.	The lipoxygenase of claim 1 which is derived from a filamentous fungus, e.g. an Ascomycota, such as a strain of Magnaporthe, particulariy M. sa/vinii, more particulariy strain IFO 6642.<br>
3.	DNA comprising a nucleic acid sequence which encodes the lipoxygenase of claim 1 or 2.<br>
4.	A polynucleotide which comprises:<br><br>
a)	the partial DNA sequence encoding a mature lipoxygenase cloned into a plasmid present in Escherichia coli DSM 14139,<br>
b)	the partial DNA sequence encoding a mature lipoxygenase shown in SEQ ID NO: 1,<br>
c)	an analogue of the sequence defined in a) or b) which encodes a lipoxygenase and<br>
i) has at least 60 % homology with said DNA sequence, or<br>
ii) hybridizes at high stringency with a complementary strand of said DNA sequence or a subsequence thereof having at least 100 nucleotides, iii) is an allelic variant thereof, or<br>
d)	a complementary strand of a), b) or c).<br><br>
5.	A nucleic acid construct comprising the nucleic acid sequence of claim 3 or 4<br>
operably linked to one or more control sequences capable of directing the expression<br>
of the lipoxygenase in a suitable expression host.<br>
6.	A recombinant expression vector comprising the nucleic acid construct of<br>
claim 5, a promoter, and transcriptional and translational stop signals.<br>
7.	A recombinant host cell comprising the nucleic acid construct of claim 5 or the vector of claim 6.<br>
8.	A method for producing a lipoxygenase comprising cultivating the host cell of claim 7 under conditions conducive to production of the lipoxygenase, and recovering the lipoxygenase.<br>
9.	An oligonucleotide probe which consists of at least 20 nucleotides and which encodes a partial polypeptide sequence of SEQ ID No.2.<br>
10.	A detergent composition comprising a surfactant and the lipoxygenase of claim<br>
l or 2.<br>
11.	The detergent composition of the preceding claim wherein the surfactant is<br>
anionic.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0OC1jaGVucC0yMDAzIGFic3RyYWN0IGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">1648-chenp-2003 abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0OC1jaGVucC0yMDAzIGNsYWltcyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1648-chenp-2003 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0OC1jaGVucC0yMDAzIGRlc2NyaXB0aW9uIChjb21wbGV0ZSkgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">1648-chenp-2003 description (complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0OC1jaGVucC0yMDAzLWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1648-chenp-2003-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0OC1jaGVucC0yMDAzLWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1648-chenp-2003-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0OC1jaGVucC0yMDAzLWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1648-chenp-2003-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0OC1jaGVucC0yMDAzLWNvcnJlc3BvbmRuZWNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">1648-chenp-2003-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0OC1jaGVucC0yMDAzLWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">1648-chenp-2003-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0OC1jaGVucC0yMDAzLWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">1648-chenp-2003-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0OC1jaGVucC0yMDAzLWZvcm0gMjYucGRm" target="_blank" style="word-wrap:break-word;">1648-chenp-2003-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0OC1jaGVucC0yMDAzLWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">1648-chenp-2003-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0OC1jaGVucC0yMDAzLWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">1648-chenp-2003-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0OC1jaGVucC0yMDAzLW90aGVyIGRvY3VtZW50cy5wZGY=" target="_blank" style="word-wrap:break-word;">1648-chenp-2003-other documents.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0OC1jaGVucC0yMDAzLXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1648-chenp-2003-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="223189-5-phenylpyrimidines-their-preparation-intermediates-for-their-preparation-and-a-composition-comprising-the-same.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="223191-constant-velocity-coupling-and-control-system-therefor.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>223190</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1648/CHENP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>47/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Nov-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>05-Sep-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>17-Oct-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVOZYMES A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>Krogshoejvej 36, DK-2880 Bagsvaerd,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SUGIO, Akiko</td>
											<td>1604 Hillcrest drive Apt. U-21, Manhattan, KS 66502,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>TAKAGI, Shinobu</td>
											<td>Maehara-nishi 1-31-1-708, Funabashi, Chiba 274-0825,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A21D8/04</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/DK2002/000251</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-04-18</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>PA 2001 00631</td>
									<td>2001-04-20</td>
								    <td>Denmark</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/223190-a-novel-fungal-lipoxygenase by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:34:47 GMT -->
</html>
