<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/223232-a-method-for-genetically-modifying-an-endogenous-gene-or-chromosomal-locus-in-eukaryotic-cells by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:27:48 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 223232:&quot;A METHOD FOR GENETICALLY MODIFYING AN ENDOGENOUS GENE OR CHROMOSOMAL LOCUS IN EUKARYOTIC CELLS&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A METHOD FOR GENETICALLY MODIFYING AN ENDOGENOUS GENE OR CHROMOSOMAL LOCUS IN EUKARYOTIC CELLS&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method for engineering and utilizing large DNA vectors to target, via homologous recombination, and modify, in any desirable fashion, endogenous genes and chromosomal loci in eukaryotic cells. These large DNA targeting vectors for eukaryotic cells, terrned LTVECs, are derived from fragments of cloned genomic DNA larger than those typically used by other approaches intended to perform homologous targeting in eukaryotic cells. Also provided is a rapid and convenient method of detecting eukaryotic cells in which the LTVEC has correctly targeted and modified the desired endogenous genes(s) or chromosomal locus (loci) as well as the use of these cells to genetale organising bearing; the genetic modification.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>METHODS OF MODIFYING EUKARYOTIC CELLS<br>
This application claims priority to U.S. Patent Application No. 09/732,234, filed December 7, 2000 and U.S. Provisional Application No. 60/244,665, filed October 31,2000. Throughout this application various publications are referenced. The disclosures of these publications in their entireties are hereby incorporated by reference into this application.<br>
Field of the Invention<br>
The field of this invention is a method for engineering and utilizing large DNA vectors to target, via homologous recombination, and modify, in any desirable fashion, endogenous genes and chromosomal loci in eukaryotic cells. These large DNA targeting vectors for eukaryotic cells, termed LTVECs, are derived from fragments of cloned genomic DNA larger than those typically used by other approaches intended to perform homologous targeting in eukaryotic cells. The field of the invention further provides for a rapid and convenient method of detecting eukaryotic cells in which the LTVEC has correctly targeted and modified the desired endogenous gene(s) or chromosomal locus(loci). The field also encompasses the use of these cells to generate organisms bearing the genetic modification, the organisms, themselves, and methods of use thereof.<br>
Introduction<br>
The use of LTVECs provides substantial advantages over current methods. For example, since these are derived from DNA fragments larger than those currently used to generate targeting vectors, LTVECs can be more rapidly and conveniently generated from available libraries of large genomic DNA fragments (such as BAG and PAC libraries) than targeting vectors made using current technologies. In addition, larger modifications as well as modifications spanning larger genomic regions can be more conveniently generated than using current technologies.<br><br>
Furthermore, the present invention takes advantage of long regions of homology to increase the targeting frequency-of "hard to target" loci, and also diminishes the benefit, if any, of using isogenic DNA in these targeting vectors.<br>
The present invention thus provides for a rapid, convenient, and streamlined<br>
•,   ^ method for systematically modifying virtually all the endogenous genes and<br>
chromosomal loci of a given organism.<br>
Background of the Invention<br>
Gene targeting by means of homologous recombination between homologous exogenous DNA and endogenous chromosomal sequences has proven to be an extremely valuable way to create deletions, insertions, design mutations, correct gene mutations, introduce transgenes, or make other genetic modifications in mice. Current methods involve using standard targeting vectors, with regions of homology to endogenous DNA typically totaling less than 10-20 kb, to introduce the desired genetic modification into mouse embryonic stem (ES) cells, followed by the injection of the altered ES cells into mouse embryos to transmit these engineered genetic modifications into the mouse germline (Smithies et al., Nature, 317:230-234, 1985; Thomas et al., Cell, 51:503-512,1987; Koller et al., Proc Natl Acad Sci USA, 86:8927-8931,1989; Kuhn et al., Science, 254:707-710,1991; Thomas et al., Nature, 346:847-850,1990; Schwartzberg et al., Science, 246:799-803,1989; Doetschman et al., Nature, 330:576-578,1987; Thomson et al., Cell, 5:313-321,1989; DeChiara et al., Nature, 345:78-80,1990; U.S. Patent No. 5,789,215, issued August 4,1998 in the name of GenPharm International) In these current methods, detecting the rare ES cells in which the standard targeting vectors have correctly targeted and modified the desired endogenous gene(s) or chromosomal locus (loci) requires sequence information outside of the homologous targeting sequences contained within the targeting vector. Assays for successful targeting involve standard Southern, blotting or long PCR (Cheng, et aL, Nature, 369:684-5,1994; Foord and Rose, PCR Methods Appl, 3:5149-61,1994; Ponce and Micol, Nucleic Acids Res, 20:623,1992; U.S. Patent No. 5,436,149 issued to Takara Shuzo Co., Ltd.) from sequences outside the targeting vector and spanning an entire homology arm. (see Definitions); thus, because of size considerations that limit these methods, the size of the<br><br>
homology arms are restricted to less than 10-20 kb in total (Joyner, The Practical Approach Series, 293,1999).<br>
The ability to utilize targeting vectors with homology arms larger than those used in current methods would be extremely valuable. For example, such<br>
* -•	S<br>
targeting vectors could be more rapidly and conveniently generated from available libraries containing large genomic inserts (e.g. BAG or PAC libraries) than targeting vectors made using current technologies, in which such genomic inserts have to be extensively characterized and trimmed prior to use. In addition, larger modifications as well as modifications spanning larger genomic regions could be more conveniently generated and in fewer steps than using current technologies. Furthermore, the use of long regions of homology could increase the targeting frequency of "hard to target" loci in eukaryotic cells, since the targeting of homologous recombination in eukaryotic cells appears to be related to the total homology contained within the targeting vector (Deng and Capecchi, Mol Cell Biol, 12:3365-71,1992). In addition, the increased targeting frequency obtained using long homology arms could diminish any potential benefit that can be derived from using isogenic DNA in these targeting vectors.<br>
The problem of engineering precise modifications into very large genomic fragments, such as those cloned in BAG libraries, has largely been solved through the use of homologous recombination in bacteria (Zhang, et al., Nat Genet, 20:123-8,1998; Yang, et al., Nat Biotechnol, 15:859-65,1997; Angrand, et al., Nucleic Acids Res, 27:el6,1999; Muyrers, et al., Nucleic Acids Res; 27:1555-7, 1999; Narayanan, et al., Gene Ther, 6:442-7,1999), allowing for the construction of vectors containing large regions of homology to eukaryotic endogenous genes or chromosomal loci. However, once made, these vectors have not been generally useful for modifying endogenous genes or chromosomal loci via homologous recombination because of the difficulty in detecting rare correct targeting events when homology arms are larger than 10-20 kb (Joyner, The Practical Approach Series, 293,1999). Consequently, vectors generated using bacterial homologous recombination from BAG genomic fragments must still be extensively trimmed prior to use as targeting vectors (Hill et al., Genomics, 64:111-3, 2000). Therefore, there is still a need for a rapid and convenient methodology that makes possible the use of targeting vectors containing large<br><br>
regions of homology so as to modify endogenous genes or chromosomal loci in eukaryotic cells.<br>
In accordance with the present invention/ Applicants provide novel methods that enables the use of targeting vectors containing large regions of homology so as to modify endogenous genes or crirornosomailociln eukaryotic cells via homologous recombination. Such methods overcome the above-described limitations of current technologies. In addition, the skilled artisan will readily recognize that the methods of the invention are easily adapted for use with any genomic DNA of any eukaryotic organism including, but not limited to, animals such as mouse, rat, other rodent, or human, as well as plants such as soy, corn and wheat.<br>
Summary of the Invention<br>
In accordance with the present invention, Applicants have developed a novel, rapid, streamlined, and efficient method for creating and screening eukaryotic cells which contain modified endogenous genes or chromosomal loci. This novel methods combine, for the first time:<br>
1.	Bacterial homologous recombination to precisely engineer a desired<br>
genetic modification within a large cloned genomic fragment, thereby creating a<br>
large targeting vector for use in eukaryotic cells (LTVECs);<br>
2.	Direct introduction of these LTVECs into eukaryotic cells to modify the<br>
endogenous chromosomal locus of interest in these cells; and<br>
3.	An analysis to determine the rare eukaryotic cells in which the<br>
targeted allele has been modified as desired, involving an assay for modification<br>
of allele (MOA) of the parental allele that does not require sequence information<br>
outside of the targeting sequence, such as, for example, quantitative PCR.<br>
A preferred embodiment of the invention is a method for genetically modifying an endogenous gene or chromosomal locus in eukaryotic cells, comprising: a) obtaining a large cloned genomic fragment containing a DNA sequence of interest; b) using bacterial homologous recombination to genetically modify the large cloned genomic fragment of (a) to create a large targeting vector for use in the eukaryotic cells (LTVEC); c) introducing the LTVEC of (b) into the<br><br>
eukaryotic cells to modify the endogenous gene or chromosomal locus in the cells; and d) using a quantitative assay to detect modification of allele (MOA) in the eukaryotic cells of (c) to identify those eukaryotic cells in which the endogenous gene or chromosomal locus has been genetically modified.<br>
Another embodiment of the invention is a method whefelrvthe genetic modification to the endogenous gene or chromosomal locus comprises deletion of a coding sequence, gene segment, or regulatory element; alteration of a coding sequence, gene segment, or regulatory element; insertion of a new coding sequence, gene segment/ or regulatory element; creation of a conditional allele; or replacement of a coding sequence or gene segment from one species with an homologous or orthologous coding sequence from a different species.<br>
An alternative embodiment of the invention is a method wherein the alteration of a coding sequence, gene segment, or regulatory element comprises a substitution, addition, or fusion, wherein the fusion comprises an epitope tag or bifunctional protein.<br>
Yet another embodiment of the invention is a method wherein the quantitative assay comprises quantitative PCR,.comparative genomic hybridization, isothermal DNA amplification, quantitative hybridization to an immobilized probe, Invader Probes®, or MMP assays®, and wherein the quantitative PCR comprises TaqMan® Molecular Beacon, or Eclipse™ probe technology.<br>
Another preferred embodiment of the invention is a method wherein the eukaryotic cell is a mammalian embryonic, stem cell and in particular wherein the embryonic stem cell is a mouse, rat, or other rodent embryonic stem cell.<br>
Another preferred embodiment of the invention is a method wherein the endogenous gene or chromosomal locus is a mammalian gene or chromosomal locus, preferably a human gene or chromosomal locus or a mouse, rat, or other rodent gene or chromosomal locus.<br><br>
An additional preferred embodiment is one in which the LTVEC is capable of accommodating large DNA fragments greater than 20 kb, and in particular large DNA fragments greater than 100 kb.<br>
Another preferred embodiment is a genetically modified endogenous gene or chromosomal locus that is produced by the method of the invention.<br>
Yet another preferred embodiment is a genetically modified eukaryotic cell that is produced by the method of the invention.<br>
A preferred embodiment of the invention is a non-human organism containing the genetically modified endogenous gen&amp; or chromosomal locus produced by the method of the invention.<br>
Also preferred in a non-human organism produced from the genetically modified eukaryotic cells or embryonic stem cells produced by the method of the invention.<br>
A preferred embodiment is a non-human organism containing a genetically modified endogenous gene or chromosomal locus, produced by a method comprising the steps of: a) obtaining a large cloned genomic fragment containing a DNA sequence of interest; b) using bacterial homologous recombination to genetically modify the large cloned genomic fragment of (a) to create a large targeting vector (LTVEC) for use in embryonic stem cells; c) introducing the LTVEC of (b) into the embryonic stem cells to modify the endogenous gene or chromosomal locus in the cells; d) using a quantitative assay to detect modification of allele (MOA) in the embryonic stem cells of (c) to identify those embryonic stem cells in which the endogenous gene or chromosomal locus has been genetically modified; e) introducing the embryonic stem cell of (d) into a blastocyst; and f) introducing the blastocyst of (e) into a surrogate mother for gestation.<br>
An additional preferred embodiment of the invention is a non-human organism containing a genetically modified endogenous gene or chromosomal locus, produced by a method comprising the steps of: a) obtaining a large cloned<br><br>
1<br>
genomic fragment containing a DNA sequence of interest; b) using bacterial homologous recombination to genetically modify the large cloned genomic fragment of (a) to create a large targeting vector for use in eukaryotic cells (LTVEC); c) introducing the LTVEC of (b).into the eukaryotic-cells to genetically modify the endogenous gene or chromosomal locus in the cells; d) using a<br>
* -•         .^<br>
quantitative assay to detect modification of allele (MOA)'in'ihe eukaryotic cells of (c) to identify those eukaryotic cells in which the endogenous gene or chromosomal locus has been genetically modified; e) removing the nucleus from tiie eukaryotic cell of (d); f) introducing the nucleus of (e) into an oocyte; and g) introducing the oocyte of (f) into 'a surrogate mother for gestation.<br>
Yet another preferred embodiment is a non-human organism containing a genetically modified endogenous gene or chromosomal locus, produced by a method comprising the steps of: a) obtaining a large cloned genomic fragment containing a DNA sequence of interest; b) using bacterial homologous recombination to genetically modify the large cloned genomic fragment of (a) to create a large targeting vector for use in eukaryotic cells (LTVEC); c) introducing the LTVEC of (b) into the eukaryotic cells to genetically modify the endogenous gene or chromosomal locus in the cells; d) using a quantitative assay to detect modification of allele (MOA) in the eukaryotic cells of (c) to identify those eukaryotic cells in which the endogenous gene or chromosomal locus has been genetically modified; e) fusing the eukaryotic cell of (d) with another eukaryotic cell; f) introducing the fused eukaryotic cell of (e) into a surrogate mother for gestation.<br>
In preferred embodiments, the non-human organism is a mouse, rat, or other rodent; the blastocyst is a mouse, rat, or other rodent blastocyst; the oocyte is a mouse, rat, or other rodent oocyte; and the surrogate mother is a mouse, rat, or other rodent.<br>
Another preferred embodiment is one in which the embryonic stem cell is a mammalian embryonic stem cell, preferably a mouse, rat, or other rodent embryonic stem cell.<br><br>
An additional preferred embodiment is the use of the genetically mocLinea eukaryotic cells of the invention for the production of a non-human organism, and in particular, the use of the genetically modified embryonic stem cell of "the invention for the production of a non-human organism.<br>
A preferred embodiment of the invention is a method for genetically modifying an endogenous gene or chromosomal locus of interest in mouse embryonic stem cells, comprising: a) obtaining a large cloned genomic fragment greater than 20 kb which contains a DNA sequence of interest, wherein the large cloned DNA fragment is homologous to the endogenous gene or chromosomal locus; b) using bacterial, homologous recombination to genetically modify the large cloned genomic fragment of (a) to create a large targeting vector for use in the mouse embryonic stem cells, wherein the genetic modification is deletion of a coding sequence, gene segment, or regulatory element; c) introducing the large targeting vector of (b) into the mouse embryonic stern cells to modify the endogenous gene or chromosomal locus in the cells; and d) using a quantitative assay to detect modification of allele (MOA) in the mouse embryonic stem cells of (c) to identify those mouse embryonic stem cells in which the endogenous gene or chromosomal locus has been genetically modified, wherein the quantitative assay is quantitative PCR. Also preferred is a genetically modified mouse embryonic stem cell produced by this method; a mouse containing a genetically modified endogenous gene or chromosomal locus produced by this method; and a mouse produced from the genetically modified mouse embryonic stem cell.<br>
Another preferred embodiment is a mouse containing a genetically modified endogenous gene or chromosomal locus of interest, produced by a method comprising the steps of: a) obtaining a large cloned genomic fragment greater than 20 kb which contains a DNA sequence of interest, wherein the large cloned DNA fragment is homologous to the endogenous gene or chromosomal locus; b) using bacterial homologous recombination to genetically modify the large cloned genomic fragment of (a) to create a large targeting vector for use in the mouse embryonic stem cells, wherein the genetic modification is deletion of a coding sequence, gene segment, or regulatory element; c) introducing the large targeting vector of (b) into the mouse embryonic stem cells to modify the<br><br>
endogenous gene or chromosomal locus in the cells; and d) using a quantitative assay to detect modification of allele (MOA) in the mouse embryonic stem cells of (c) to identify those mouse embryonic stem cells in which the endogenous gene or chromosomal locus has been genetically modified, wherein the quantitative assay is quantitative PCR; e) introducing the mouse embryonic stem cell of (d) into a blastocyst; and f) introducing the blastocyst of (e) into a surrogate mother for gestation.<br>
Also preferred is the use of the genetically modified mouse embryonic stem, cell described above for the production of a mouse.<br>
Also preferred are methods wherein about 1-5 /ig of large targeting vector DNA is introduced into about 1 x 107 eukaryotic cells.<br>
Brief Description of the Figures<br>
Figure 1: Schematic diagram of the generation of a typical LTVEC using<br>
bacterial homologous recombination.<br>
(hbl = homology box 1; hb2 = homology box 2; RE = restriction enzyme site).<br>
Figure 2: Schematic diagram of donor fragment and LTVEC for mouse OCRIO. (hbl = homology box 1; lacZ = fi-galactosidase ORE; SV40 polyA = a DNA fragment derived from Simian Virus 40, containing a polyadenylation site and signal; PGKp = mouse phosphoglycerate kinase (PGK) promoter; EM7 - a bacterial promoter; neo = neomycin phosphotransferase; PGK polyA = 3' untranslated region derived from the PGK gene and containing a polyadenylation site and signal; hb2 = homology box 2)<br>
Figure 3A-3D: Sequence of the mouse OCRIO cDNA, homology box 1 (hbl), homology box 2 (hb2), and TaqMan® probes and primers used in a quantitative PCR assay to detect modification of allele (MOA) in ES cells targeted using the mOCRlO LTVEC.<br>
hbl: base pairs 1 to 211 hb2: base pairs 1586 to 1801<br><br>
TaqMan® probe and corresponding PCR primer set derived from mOCRlO<br>
exon 3:<br>
TaqMan® probe: nudeotides 413 to 439 - upper strand<br>
Primer ex3-5': nudeotides 390 to 410 - upper strand<br>
Primer ex3-3': nudeotides 445 to 461 - lower strand<br>
TaqMan® probe and corresponding PCR primer set derived from mOCRlO<br>
exon 4:<br>
TaqMan® probe: nudeotides 608 to 639 - upper strand<br>
Primer ex4-5': nudeotides 586 to 605 - upper strand<br>
Primer ex4-3': nudeotides 642 to 662 - lower strand<br>
Definitions<br>
A "targeting vector" is a DNA construct that contains sequences "homologous" to endogenous chromosomal nucleic acid sequences flanking a desired genetic modification(s). The flanking homology sequences, referred to as "homology arms", direct the targeting vector to a specific chromosomal location within the genome by virtue of the homology that exists between the homology arms and the corresponding endogenous sequence and introduce the desired genetic modification by a process referred to as "homologous recombination".<br>
"Homologous" means two or more nucleic acid sequences that are either identical or similar enough that they are able to hybridize to each other or undergo intermolecuiar exhange.<br>
"Gene targeting" is the modification of an endogenous chromosomal locus by the insertion into, deletion of, or replacement of the endogenous sequence via homologous recombination using a targeting vector.<br>
A "gene knockout" is a genetic modification resulting from the disruption of the genetic information encoded in a chromosomal locus.<br>
A "gene knockin" is a genetic modification resulting from the replacement of the genetic information encoded in a chromosomal locus with a different DNA sequence.<br>
A "knockout organism" is an organism in which a significant.proportion of the organism's cells harbor a gene knockout.<br>
A "knockin organism" is an organism in which a significant proportion of the organism's cells harbor a gene knockin.<br>
A "marker " or a "selectable marker" is a selection marker that allows for the isolation of rare transfected cells expressing the marker from the majority of treated cells in the population. Such marker's gene's include, but are not limited to, neomycin phosphotransferase and hygromycin B phosphotransferase, or fluorescing proteins such as GFP.<br>
An "ES cell" is an embryonic stem cell. This cell is usually derived from the inner cell mass of a blastocyst-stage embryo.<br>
An "ES cell clone" is a subpopulation of cells derived from a single cell of the ES cell population following introduction of DNA and subsequent selection.<br>
A "flanking DNA" is a segment of DNA that is collinear with and adjacent to a particular point of reference.<br>
"LTVECs" are large targeting vectors for eukaryotic cells that are derived from fragments of cloned genomic DNA larger than those typically used by other approaches intended to perform homologous targeting in eukaryotic cells.<br>
A "non-human, organism" is an organism that is not normally accepted by the public as being human.<br>
"Modification of allele" (MOA) refers to the modification of the exact DNA sequence of one allele of a gene(s) or chromosomal locus (loci) in a genome. This modification of allele (MOA) includes, but is not limited to, deletions/<br>
substitutions, or insertions of as little as a single nucleotide or deletions of many kilbbases spanning a gene(s) or chromosomal locus (loci) of interest, as well as any and all possible modifications between these two extremes.<br>
"Orthologous" sequence refers to a sequence from one sp.ecies that is the functional equivalent of that sequence in another species'.    /<br>
The description and examples presented infra are provided to illustrate the subject invention. One of skill in the art will recognize that these examples are provided by way of illustration only and are not included for the purpose of limiting the invention.<br>
Detailed Description of the Invention<br>
Applicants have developed a novel, rapid, streamlined, and efficient method for creating and screening eukaryotic cells which contain modified endogenous genes or chromosomal loci. In these cells, the modification may be gene(s) knockouts, knocldrvs, point mutations, or large genomic insertions or deletions or other modifications. By way of non-limiting example, these cells may be embryonic stem cells which are useful for creating knockout or knockin organisms and in particular, knockout or knockin mice, for the purpose of determining the function of the gene(s) that have been altered, deleted and/or inserted.<br>
The novel .methods described herein combine, for the first time:<br>
1.	Bacterial homologous recombination to precisely engineer a desired<br>
genetic modification within a large cloned genomic DNA fragment, thereby<br>
creating a large targeting vector for use in eukaryotic cells (LTVECs);<br>
2.	Direct introduction of these LTVECs into eukaryotic cells to modify the<br>
corresponding endogenous gene(s) or chromosomal locus(loci) of interest in<br>
these cells; and<br>
3.	An analysis to determine the rare eukaryotic cells in which the<br>
targeted allele has been modified as desired, involving a quantitative assay for<br>
modification of allele (MOA) of the parental allele.<br>
It should be emphasized that previous methods to detect successful homologous recombination in eukaryotic cells cannot be utilized in conjunction with the LTVECs of Applicants' invention because of the long homology arms present in the LTVECs. Utilizing a LTVEC to deliberately modify endogenous<br>
genes or chromosomal loci in eukaryotic cells via homologous recombination is<br>
",    j made possible by the novel application of an assay to deter-mine the rare<br>
eukaryotic cells in which the targeted allele has been modified as desired, such assay involving a quantitative assay for modification of allele (MOA) of a parental allele, by employing, for example, quantitative PCR or other suitable quantitative assays for MOA.<br>
The ability to utilize targeting vectors with homology arms larger than those used in current methods is extremely valuable for the following reasons:<br>
1. Targeting vectors are more rapidly and conveniently generated from available libraries containing large genomic inserts (e.g. BAC or PAC libraries) than targeting vectors made using previous technologies, in which the genomic inserts have to be extensively characterized and "trimmed" prior to use (explained in detail below). In addition, minimal sequence information needs to be known about the locus of interest, i.e. it is only necessary to know the approximately 80-100 nucleotides that are required to generate the homology boxes (described in detail below) and to generate probes that can be used in quantitative assays for MOA (described in detail below).<br>
2. Larger modifications as well as modifications spanning larger genomic regions are more conveniently generated and in fewer steps than using previous technologi.es. For example, the method of the invention makes possible the precise modification of large loci that cannot be accommodated by traditional plasmid-based targeting vectors because of their size limitations. It also makes possible the modification of any given locus at multiple points (e.g. the introduction of specific mutations at different exons of a multi-exon gene) in one step, alleviating the need to engineer multiple targeting vectors and to perform multiple rounds of targeting and screening for homologous recombination in ES cells.<br>
3. The use of long regions of homology (long homology arms) increase the targeting frequency of "hard to target" loci in eukaryotic cells, consistent with previous findings that targeting of homologous recombination in<br>
eukaryotic cells appears to be related to the total homology contained within the targeting vector.<br>
4,	The increased targeting frequency obtained using long homology<br>
arms apparently diminishes the benefit/ if any, from using isogenic DNA in    ,<br>
these targeting vectors.	<br>
5,	The application of quantitative MOA assays for screening eukaryotic<br>
cells for homologous recombination not only empowers the use of LTVECs as<br>
targeting vectors (advantages outlined above) but also reduces the time for<br>
identifying correctly modified eukaryotic cells from the typical several days to a<br>
few hours. In addition, the application of quantitative MOA does not require<br>
the use of probes located outside the endogenous gene(s) or chromosomal<br>
locus(loci) that is being modified, thus obviating the need to know the sequence<br>
flanking the modified gene(s) or locus(loci). This is a significant improvement in<br>
the way the screening has been performed in the past and makes it a much less<br>
labor-intensive and much more cost-effective approach to screening for<br>
homologous recombination events in eukaryotic cells.<br>
Methods<br>
Many of the techniques used to construct DNA vectors described herein are standard molecular biology techniques well known to the skilled artisan (see e.g., Sambrook,}., E. F. Fritsch And T. Maniatis. Molecular Cloning: A Laboratory Manual, Second Edition, Vols 1, 2, and 3,1989; Current Protocols in Molecular Biology, Eds. Ausubel et al., Greene Publ. Assoc., Wiley Interscience, NY). All DNA sequencing is done by standard techniques using an ABI 373A DNA sequencer arid Taq Dideoxy Terminator Cycle Sequencing Kit (Applied Biosystems,, Inc., Foster City, CA).<br>
Step 1. Obtain a large genomic DNA clone containing the gene(s) or chromosomal locus (loci) of interest.<br>
A Gene(s) or locus(loci) of interest can be selected based on specific criteria, such as detailed structural or functional data, or it can be selected in the absence of such detailed information as potential genes or gene fragments become predicted through the efforts of the various genome sequencing projects.<br>
Importantly, it should be noted that it is not necessary to know the complete sequence and gene structure of a gene(s) of interest to apply the method of the subject invention to produce LTVECs. In fact, the only sequence information that is required is approximately 80-100 nucleotides so as to obtain the genomic<br>
clone of interest as well as to generate the homology boxes used in making the<br>
LTVEC (described in detail below) and to make probes for use in quantitative MOA assays.<br>
Once a gene(s) or locus(loci) of interest has been selected, a large genomic clone(s) containing this gene(s) or locus(loci) is obtained. This clone(s) can be obtained in any one of several ways including, but not limited to, screening suitable DNA libraries (e.g. BAG, PAC, YAC, or cosmid) by standard hybridization or PCR techniques, or by any other methods familiar to the skilled artisan.<br>
Step 2. Append homology boxes 1 and 2 to a modification cassette and generation of LTVEC.<br>
Homology boxes mark the sites of bacterial homologous recombination that are used to generate LTVECs from large cloned genomic fragments (Figure 1). Homology boxes are short segments of DNA, generally double-stranded and at least 40 nucleotides in length, that are homologous to regions within the large cloned genomic fragment flanking the "region to be modified". The homology boxes are appended to the modification cassette, so mat following homologous recombination in bacteria, the modification cassette replaces the region to be modified (Figure 1). The technique of creating a targeting vector using bacterial homologous recombination can be performed in a variety of systems (Yang et ah, Nat Biotechnol, 15:859-65,1997; Muyrers et al, Nucleic Acids Res, 27:1555-7, 1999; Angrand et al., Nucleic Acids Res, 27:el6,1999; Narayanan et al., Gene Ther, 6:442-7,1999; Yu, et al., Proc Natl Acad Sci U S A, 97:5978-83, 2000). One example of a favored technology currently in use is ET cloning (Zhang et al., Nat Genet, 20:123-8,1998; Narayanan et al., Gene Ther, 6:442-7,1999) and variations of this technology (Yu, et al., Proc Natl Acad Sci USA, 97:5978-83, 2000). ET refers to the recE (Hall and Kolodner, Proc Natl Acad Sci USA, 91:3205-9, 1994) and recT proteins (Kusano et al., Gene, 138:17-25,1994) that carry out the<br>
homologous recombination reaction. RecE is an exonuclease that trims one strand of linear double-stranded DNA (essentially the donor DNA fragment described infra) 5' to 3', thus leaving behind a linear double-stranded fragment with a 3" single-stranded overhang. This single-stranded overhang is coated by<br>
recT protein, which has single-stranded DNA (ssDNA) binding activity (Kovall<br>
* -   tj and Matthews, Science, 277:1824-7,1997). ET cloning is performed using E. coli<br>
. that transiently express the E. coli gene products of recE and recT (Hall and Kolodner, Proc Natl Acad Sci USA, 91:3205-9,1994; Clark et al., Cold Spring Harb Symp Quant Biol, 49:453-62,1984; Noirot and Kolodner, J Biol Chem, 273:12274-80,1998; Thresher et al., J Mol Biol, 254:364-71,1995; Kolodner et al., Mol Microbiol, 11:23-30,1994; Hall et al., J Bacteriol, 175:277-87,1993) and the bacteriophage lambda (A.) protein Agam (Murphy, J Bacteriol, 173:5808-21,1991; Poteete et al., J Bacteriol, 170:2012-21,1988). The Xgam protein is required for protecting the donor DNA fragment from degradation by the recBC exonuclease system (Myers and Stahl, Annu Rev Genet, 28:49-70,1994) and it is required for efficient ET-doning in recBC+ hosts such as the frequently used E. coli strain DHlOb.<br>
The region to be modified and replaced using bacterial homologous recombination can range from zero nucleotides in length (creating an insertion into the original locus) to many tens of kilobases (creating a deletion and/or a replacement of the original locus). Depending on the modification cassette, the modification can result in the following:<br>
(a)	deletion of coding sequences, gene segments, or regulatory elements;<br>
(b)	alteration(s) of coding sequence, gene segments, or regulatory elements<br>
including substitutions, additions, and fusions (e.g. epitope tags or creation of<br>
bifunctional proteins such as those with GFP);<br>
(c)	insertion of new coding regions, gene segments, or regulatory elements,<br>
such as those for selectable marker genes or reporter genes or putting new<br>
genes under endogenous transcriptional control;<br>
(d)	creation of conditional alleles, e.g. by introduction of loxP sites flanking the<br>
region to be excised by Cre recombinase (Abremski and Hoess, J Biol Chem,<br>
259:1509-14,1984), or FRT sites flanking the region to be excised by Flp<br>
recombinase (Andrews et al., Cell, 40:795-803,1985; Meyer-Leon et al., Cold<br>
Spring Harb Symp Quant Biol, 49:797-804,1984; Cox, Proc Natl Acad Sci USA, 80:4223-7,1983); or<br>
(e) replacement of coding sequences or gene segments from one species with orthologous coding sequences from a different species, e.g. replacing a murine<br>
generic locus with the orthologous human genetic locus to engineer a mouse<br>
where that particular locus has been 'humanized',<br>
Any or all of these modifications can be incorporated into a LTVEC. A specific, non-limiting example in which an endogenous coding sequence is entirely deleted and simultaneously replaced with both a reporter gene as well as a selectable marker is provided below in Example 1, as are the advantages of the method of the invention as compared to previous technologies.<br>
Step 3 (optional). Verify that each LTVEC has been engineered correctly.<br>
Verify that each LTVEC has been engineered correctly by:<br>
a.	Diagnostic PCR to verify the novel junctions created by the introduction of<br>
the donor fragment into the gene(s) or chromosomal locus(loci) of interest. The<br>
PCR fragments thus obtained can be sequenced to further verify the novel<br>
junctions created by the introduction of the donor fragment into the gene(s) or<br>
chromosomal locus(loci) of interest.<br>
b.	Diagnostic restriction enzyme digestion to make sure that only the desired<br>
modifications have been introduced into the LTVEC during the bacterial<br>
homologous recombination process.<br>
c.	Direct sequencing of the LTVEC, particularly the regions spanning the site of<br>
the modification to verify the novel junctions created by the introduction, of the<br>
donor fragment into the gene(s) or chromosomal locus(loci) of interest.<br>
Step 4. Purification, preparation, and linearization of LTVEC DNA for introduction into eukaryotic cells.<br>
a. Preparation of LTVEC DNA:<br>
Prepare miniprep DNA (Sambrook, J., E. F. Fritsch And T. Maniatis. Molecular<br>
Cloning: A Laboratory Manual, Second Edition, Vols 1, 2, and 3,1989; Tillert and<br>
Neilan, Biotechniques, 24:568-70, 572,1998;<br>
http:/V www.qiagen.com/Hterature/handbooks/plkrxiird/<br>
plm_399.pdf) of the selected LTVEC and re-transform the miniprep LTVEC'<br>
DNA into E. call using electroporation (Sambrook, J., E. F. Fritsch and T.<br>
Maniatis, Molecular Cloning: A Laboratory Manual, Second Edition, Vols 1,2,<br>
and 3,1989). This step is necessary to get rid of the plasmid'encoding the recombinogenic proteins that are utilized for the bacterial homologous recombination step (Zhang et al., Nat Genet, 20:123-8,1998; Narayanan et al., Gene Ther, 6:442-7,1999). It is useful to get rid of this plasmid (a) because it is a high copy number plasmid and may reduce the yields obtained in the large scale LTVEC preps; (b) to eliminate the possibility of inducing expression of the recombinogenic proteins; and (c) because it may obscure physical mapping of the LTVEC. Before introducing the LTVEC into eukaryotic cells, larger amounts of LTVEC DNA are prepared by standard methodology<br>
(http://www.qiagen.com/literature/handbooks/plk/plklow.pdf; Sambrook, J., E. F. Fritsch And T. Maniatis. Molecular Cloning: A Laboratory Manual, Second Edition, Vols 1,2, and 3,1989; Tillett and Neilan, Biotechniques, 24:568-70, 572, 1998). However, this step can be bypassed if a bacterial homologous recombination method that utilizes a recombinogenic prophage is used, i.e. where the genes encoding the recombinogenic proteins are integrated into the bacterial chromosome (Yu, et al., Proc Natl Acad Sci U S A, 97:5978-83, 2000), is used.<br>
b. Linearizing the LTVEC DNA:<br>
To prepare the LTVEC for introduction into eukaryotic cells, the LTVEC is preferably linearized in a manner that leaves the modified endogenous gene(s) or chromosomal locus(loci) DNA flanked with long homology arms. This can be accomplished by linearizing the LTVEC, preferably in the vector backbone, with any suitable restriction enzyme that digests only rarely. Examples of suitable restriction enzymes include NotI, Pad, Sfil, SrfL Swal, FseL etc. The choice of restriction enzyme may be determined experimentally (i.e. by testing several different candidate rare cutters) or, if the sequence of the LTVEC is known, by analyzing the sequence and choosing a suitable restriction enzyme based on the analysis. In situations where the LTVEC has a vector backbone containing rare sites such as CosN sites, then it can be cleaved with enzymes<br>
recognizing such sites, for example X terminase (Shizuya et al., Proc Natl Acad Sci USA, 89:8794-7, 1992; Becker and Gold, Proc Natl Acad Sci USA, 75:4199-203, 1978; Rackwitz et al., Gene, 40:259-66,1985).<br>
Step 5. Introduction of LTVEC into eukaryotic cells and selection of cells where<br>
successful introduction of the LTVEC has taken place.<br>
LTVEC DNA can be introduced into eukaryotic cells using standard methodology, such as transfection mediated by calcium phosphate, lipids, or electroporation (Sambrook,}., E. F. Fritsch And T. Maniatis. Molecular Cloning: A Laboratory Manual, Second Edition, Vols 1, 2, and 3,1989). The cells where the LTVEC has been introduced successfully can be selected by exposure to selection agents, depending on the selectable marker gene that has been engineered into the LTVEC. As a non-limiting example, if the selectable marker is the neomycin phosphotransferase (neo) gene (Beck, et al., Gene, 19:327-36, 1982), then cells that have taken up the LTVEC can be selected in G418-containing media; cells that do not have the LTVEC will die whereas cells that have taken up the LTVEC will survive (Santerre, et al., Gene, 30:147-56,1984). Other suitable selectable markers include any drug that has activity in eukaryotic cells (Joyner, The Practical Approach Series, 293,1999), such as hygromycin B (Santerre, et al, Gene, 30:147-56,1984; Bernard, et al., Exp Cell Res, 158:237-43,1985; Giordano and McAllister, Gene, 88:285-8,1990), Blasticidin S (Izumi, et al., Exp Cell Res, 197:229-33,1991), and other which are familiar to those skilled in the art.<br>
Step 6. Screen for homologous recombination events in eukaryotic cells using quantitative assay for modification of allele (MOA).<br>
Eukaryotic cells mat have been successfully modified by targeting the LTVEC into the locus of interest can be identified using a variety of approaches that can detect modification of allele within the locus of interest and that do not depend on assays spanning the entire homology arm or arms. Such approaches can include but are not limited to:<br>
(a) quantitative PCR using TaqMan® (Lie and Perropoulos, Curr Opin Biotechnol, 9:43-8,1998);<br><br>
(b)	quantitative MOA assay using molecular beacons (Tan, et al., Chemistry,<br>
6:1107-11,2000)<br>
(c)	fluorescence in situ hybridization FISH (Laan, et al., Hum Genet, 96:275-80,<br>
1995) or comparative genomic hybridization (CGH) (Forozan, et al., Trends   .<br>
Genet, 13:405-9,1997; Thompson and Gray, J Cell Biochem Suppl, 139-43,1993;<br>
Houldsworth and Chaganti, Am J Pathol, 145:1253-60,1994);<br>
(d)	isothermic DNA amplification (Lizardi, et al., Nat Genet, 19:225-32,1998;<br>
Mitra and Church, Nucleic Acids Res, 27:e34,1999);<br>
(e)	quantitative hybridization to an immobilized probe(s ) (Southern, J. Mol.<br>
Biol. 98: 503,1975; Kafatos FC; Jones CW; Efstratiadis A, Nucleic Acids Res<br>
7(6):1541-52,1979);<br><br>
(f)	Invader Probes® (Third Wave Technologies);<br>
(g)	Eclipse™ and Molecular Beacon probes (Synthetic Genetics); and<br>
(h) MMP assays (High Throughput Genomics)<br>
Applicants provide herein an example in which TaqMan® quantitative PCR is used to screen for successfully targeted eukaryotic cells. In this non limiting example, TaqMan® is used to identify eukaryotic cells which have undergone homologous recombination wherein a portion of one of two endogenous alleles in a diploid genome has been replaced by another sequence. In contrast to traditional methods, in which a difference in restriction fragment length spanning the entire homology arm or arms indicates the modification of one of two alleles, the quantitative TaqMan® method will detect the modification of one allele by measuring the reduction in copy number (by half) of the unmodified allele. Specifically, the probe detects the unmodified allele and not the modified allele. Therefore, the method is independent of the exact nature of the modification and not limited to the sequence replacement described in this example. TaqMan is used to quantify the number of copies of a DNA template in a genomic DNA sample, especially by comparison to a reference gene (Lie and Perropoulos, Curr Opin Biotechnol, 9:43-8,1998). The reference gene is quantitated in the same genomic DNA as the target gene(s) or locus(loci). Therefore, two TaqMan® amplifications (each with its respective probe) are performed. One TaqMan® probe determines the "Ct" (Threshold Cycle) of the reference gene, while the other probe determines the Ct of the region of the targeted gene(s) or locus(loci) which is replaced by successful targeting. The Ct<br><br>
is a quantity that reflects the amount of starting DNA for each of the TaqMan® probes, i.e. a less abundant sequence requires more cycles of PCR to reach the threshold cycle. Decreasing by half the number of copies of the template sequence for a TaqMan® reaction will result in an increase of about one Ct unit. TaqMan® reactions in cells where one allele of the target_gene(s) or locus(loci)<br>
*<br>
has been replaced by homologous recombination will result in an increase of one Ct for the target TaqMan® reaction without an increase in the Ct for the reference gene when compared to DNA from non-targeted cells. This allows for ready detection of the modification of one allele of the gene(s) of interest in eukaryotic cells using LTVECs.<br>
As stated above, modification of allele (MOA) screening is the use of any method that detects the modification of one allele to identify cells which have undergone homologous recombination. It is not a requirement that the targeted alleles be identical (homologous) to each other, and in fact, they may contain polymorphisms, as is the case in progeny resulting from crossing two different strains of mice. In addition, one special situation that is also covered by MOA screening is targeting of genes which are normally present as a single copy in cells, such as some of the located on the sex chromosomes and in particular, on the Y chromosome. In this case, methods that will detect the modification of the single targeted allele, such as quantitative PCR, Southern blottings, etc., can be used to detect the targeting event. It is clear that the method of the invention can be used to generate modified eukaryotic cells even when alleles are polymorphic or when they are present in a single copy in the targeted cells.<br>
Step 8. Uses of genetically modified eukaryotic cells.<br>
(a)	The genetically modified eukaryotic cells generated by the methods<br>
described in steps 1 through 7 can be employed in any in vitro or in vivo assay,<br>
where changing the phenotype of the cell is desirable.<br>
(b)	The genetically modified eukaryotic cell generated by the methods described<br>
in steps 1 through 7 can also be used to generate an organism carrying the<br><br>
genetic modification. The genetically modified organisms .can be generated by several different techniques including but not limited to:<br>
1. Modified embryonic stem (ES) cells such as the frequently used rat and mouse ES cells. ES cells can be used to create genetically modified rats or mice<br>
by standard blastocyst injection technology or aggregation techniques<br>
 (Robertson, Practical Approach Series, 254,1987; Wood, et aL, Nature, 365:87-9, 1993; Joyner, The Practical Approach Series, 293,1999), tetraploid blastocyst injection (Wang, et al,, Mech Dev, 62:137-45,1997), or nuclear transfer and. cloning (Wakayama, et al., Proc Natl Acad Sci USA, 96:14984-9,1999). ES cells derived from other organisms such as rabbits (Wang, et al., Mech Dev, 62:137-45,1997; Schoonjans, et al., Mol Reprod Dev, 45:439-43,1996) or chickens (Pain, et al., Development, 122:2339-48,1996) or other species should.also be amenable to genetic modification(s) using the methods of the invention.<br>
2.	Modified protoplasts can be used to generate genetically modified<br>
plants (for example see US patent 5,350,689 'Zea mays plants and transgenic Zea<br>
mays plants regenerated from protoplasts or protoplast-derived cells", and US<br>
patent 5,508,189 "Regeneration of plants from cultured guard cell protoplasts"<br>
and references therein),<br>
3.	Nuclear transfer from modified eukaryotic cells to oocytes to generate<br>
cloned organisms with modified allele (Wakayama, et al., Proc Natl Acad Sci U S<br>
A, 96:14984-9,1999; Baguisi, et al., Nat Biotechnol, 17:456-61,1999; Wilmut, et al.,<br>
Reprod Fertil Dev, 10:639-43,1998; Wilmut, et al., Nature, 385:810-3,1997;<br>
Wakayama, et al., Nat Genet, 24:108-9, 2000; Wakayama, et al., Nature, 394:369-<br>
74,1998; Rideout, et al., Nat Genet, 24:109-10,2000; Campbell, et al., Nature,<br>
380:64-6,1996).<br><br>
4.	Cell-fusion to transfer the modified allele to another cell, including<br>
transfer of engineered chromosome(s), and uses of such cell(s) to generate<br>
organisms carrying the modified allele or engineered chromosome(s) (Kuroiwa,<br>
et al., Nat Biotechnol, 18:1086-1090,2000).<br>
5.	The method of the invention are also amenable to any other<br>
approaches that have been used or yet to be discovered.<br>
While many of the techniques used in practicing the individual steps of the methods of the invention are familiar to the skilled artisan, Applicants contend that the novelty of the method of the invention lies in the unique combination<br><br>
of those steps and techniques coupled with the never-before-described method of introducing a LTVEC directly into eukaryotic cells to modify a chromosomal locus, and the use of quantitative MOA assays to identify eukaryotic cells which have been appropriately modified. This novel combination represents a<br>
significant improvement over previous technologies for creating organisms<br>
possessing modifications of endogenous genes or chromosomal loci.<br>
Example's Example 1: Engineering mouse ES cells bearing a deletion of the QCRIO gene.<br>
a. Selection of a large genomic DNA clone containing mOCRlO. A Bacterial Artificial Chromosome (BAG) clone carrying a large genomic DNA fragment that contained the coding sequence of the mouse OCR10 (mOCRlO) gene was obtained by screening an arrayed mouse genomic DNA BAG library (Incyte Genomics) using PCR. The primers employed to screen this library were derived from the mOCRlO gene cDNA sequence.<br>
Two primer pairs where used:<br>
(a)	OCR10.RAA (S'-AGCTACCAGCTGCAGATGCGGGCAG -3') and<br>
OCRlO.PVIrc (S'-CTCCCCAGCCTGGGTCTGAAAGATGACG-S1) which<br>
amplifies a 102 bp DNA; and<br>
(b)	OCR10.TDY (5-GACCTCACTTGCTACACTGACTAC-3') and OCRlO.QETrc<br>
(S'-ACTTGTGTAGGCTGCAGAAGGTCTCTTG-S1) which amplifies a 1500 bp<br>
DNA.<br>
This mOCRlO BAG contained approximately 180 kb of genomic DNA including the complete mOCRlO coding sequence. This BAG done was used to generate an LTVEC which was subsequently used to delete a portion of the coding region of mOCRlO while simultaneously introducing a reporter gene whose initiation codon precisely replaced the initiation codon of OCR10, as well as insertion of a selectable marker gene useful for selection both in E. coli and mammalian cells following the reporter gene (Figure 2). The reporter gene (in this non-limiting<br><br>
example LacZ, the sequence of which is readily available to the skilled artisan.), encodes the E. coli fi-galactosidase enzyme. Because of the position of insertion of LacZ (its initiating codon is at the same position as the initiation codon of mOCR'10) the expression of lacZ should mimic that of mOCRlO, as has been   , observed in other examples where similar replacements with LacZ were<br>
t<br>
performed using previous technologies (see "Gene trap strategies in. ES cells", by W Wurst and A. Gossler, in Joyner, The Practical Approach Series, 293,1999) The LacZ gene allows for a simple and standard enzymatic assay to be performed that can reveal its expression patterns in situ, thus providing a surrogate assay that reflects the normal expression patterns of the replaced gene(s) or chromosomal locus(loci).<br>
b. Construction of donor fragment and generation of LTVEC. The modification cassette used in the construction of the rnOCRlO LTVEC is the lacZ-SV40 polyA-PGKp-EM7-neo-PGI
To construct the mOCRlO LTVEC, first a donor fragment was generated consisting of a mOCRlO homology box 1 (hbl) attached upstream from the<br><br>
LacZ gene in the modification cassette and a mOCRlO homology box 2 (hb2) attached downstream of the neo-PGK polyA sequence in the modification cassette (Figure 2), using standard recombinant genetic engineering technology. Homology box 1 (hbl) consists of 211 bp of untranslated sequence immediately<br>
upstream of the initiating methionine of the mOCRlO open reading frame<br>
 (mOCRlO ORF) (Figure 3A-3D). Homology box 2 (hb2) consists of last 216 bp of the mOCRlO ORF, ending at the stop codon (Figure 3A-3D).<br>
Subsequently, using bacterial homologous recombination (Zhang, et al., Nat Genet, 20:123-8,1998; Angrand, et al., Nucleic Acids Res, 27:el6,1999; Muyrers, et al., Nucleic Acids Res, 27:1555-7,1999; Narayanan, et al., Gene Ther, 6:442-7, 1999; Yu, et al., Proc Natl Acad Sci USA, 97:5978-83, 2000), this donor fragment was used to precisely replace the mOCRlO coding region (from initiation methionine to stop codon) with the insertion cassette, resulting in construction of the mOCRlO LTVEC (Figure 2). Thus, in this mOCRlO LTVEC, the mOCRlO coding sequence was replaced by the insertion cassette creating an approximately 20 kb deletion in the mOCRlO locus while leaving approximately 130 kb of upstream homology (upstream homology arm) and 32 kb of downstream homology (downstream homology arm).<br>
It is important to note that LTVECs can be more rapidly and conveniently generated from available BAG libraries than targeting vectors made using previous technologies because only a single bacterial homologous recombination step is required and the only sequence information required is mat needed to generate the homology boxes. In contrast, previous approaches for generating targeting vectors using bacterial homologous recombination require that large targeting vectors be "trimmed" prior to their introduction in ES cells (Hill et al., Genomics, 64:111-3, 2000). This trimming is necessary because of the need to generate homology arms short enough to accommodate the screening methods utilized by previous approaches. One major disadvantage of the method of Hill et al. is that two additional homologous recombination steps are required simply for trimming (one to trim the region upstream of the modified locus and one to trim the region downstream of the modified locus). To do this, substantially more sequence information is needed, including sequence information spanning the sites of trimming.<br>
In addition, another obvious advantage, illustrated by the above example, is that a very large deletion spanning the mOCRlO gene (approximately 20 kb) can be easily generated in a single step. In contrast using previous technologies/ to accomplish the same task may require several steps and may involve marking die regions upstream and downstream of'the coding sequences with loxP sites in order to use the Cre recombinase to remove the sequence flanked by these sites after introduction of the modified locus in eukaryotic cells. This may be unattainable in one step/ and thus may require the construction of two targeting vectors using different selection markers and two sequential targeting events in ES cells/ one to introduce the loxP site at the region upstream of the coding sequence and another to introduce the loxP site at the region downstream of the coding sequence. It should be further noted that the creation of large deletions often occurs with low efficiency using the previous targeting technologies in eukaryotic cells/ because the frequency of achieving homologous recombination may be low when using targeting vectors containing large deletion flanked by relatively short homology arms. The high efficiency obtained using the method of the invention (see below) is due to the very long homology arms present in the LTVEC that increase the rate of homologous recombination in eukaryotic cells.<br>
c. Verification/ preparation, and introduction of mOCRlO LTVEC DNA into ES cells.<br>
The sequence surrounding the junction of the insertion cassette and the homology sequence was verified by DNA sequencing. The size of the mOCRlO LTVEC was verified by restriction analysis followed by pulsed field gel electrophoresis (PFGE) (Cantor, et al./ Annu Rev Biophys Biophys Chem, 17:287-304,1988; Schwartz and Cantor, Cell, 37:67-75,1984). A standard large-scale plasmid preparation of the mOCRlO LTVEC was done, the plasmid DNA was digested with the restriction enzyme NotI, which cuts in the vector backbone of the mOCRlO LTVEC, to generate linear DNA. Subsequently the Linearized DNA was introduced into mouse ES cells by electroporation (Robertson, Practical Approach Series, 254,1987; Joyner, The Practical Approach Series, 293,1999; Sambrook, et al., Sambrook, J., E. F. Fritsch and'T. Martian's. Molecular Cloning: A Laboratory Manual, Second Edition, Vols I/ 2, and 3,<br>
1989), ES cells successfully transfected with the mOCR.10 LTVEC were selected for in G418-con.tai.ning media using standard selection methods (Robertson, Practical Approach Series, 254,1987; Joyner, The Practical Approach Series, 293, 1999).<br>
d. Identification of targeted ES cells clones using a quantitative modification of allele (MOA) assay.<br>
To identify ES cells in which one of the two endogenous mOCRlO genes had been replaced by the modification cassette sequence, DNA from individual ES cell clones was analyzed by quantitative PCR using standard TaqMan® methodology as described (Applied Biosystems, TaqMan® Universal PCR Master Mix, catalog number P/N 4304437; see also http://www.pebiodocs.com/pebiodocs/04304449.pdf). The primers and TaqMan® probes used are as described in Figure 3A-3D. A total of 69 independent ES cells clones where screened and 3 were identified as positive, i.e. as clones in which one of the endogenous mOCRlO coding sequence had been replaced by the modification cassette described above.<br>
Several advantages of the MOA approach are apparent:<br>
(i) It does not require the use of a probe outside the locus being modified, thus<br>
obviating the need to know the sequence flanking the modified locus.<br>
(ii) It requires very little time to perform compared to conventional Southern<br>
blot methodology which has been the previous method of choice (Robertson,<br>
Practical Approach Series, 254,1987, Joyner, The Practical Approach Series, 293,<br>
1999), thus reducing the time for identifying correctly modified cells from the<br>
typical several days to just a few hours.<br>
This is a significant improvement in the way screening has been performed in the past and makes it a much less labor-intensive and more cost-effective approach to screening for homologous recombination events in eukaryotic cells.<br>
Yet another advantage of the method of the invention is that it is also superior to previous technologies because of its ability to target difficult loci. Using previous technologies, it has been shown that for certain loci the frequency of successful targeting may by as low as 1 in 2000 integration events, perhaps even<br>
lower. Using the method of the invention, Applicants have demonstrated that such difficult loci can be targeted much more efficiently using LTVECs that contain long homology arms (i.e. greater than those allowed by previous technologies). As the non-limiting example described above demonstrates, the<br>
Applicants have targeted the OCR10 locus, a locus that has previously proven<br>
*'   ,J recalcitrant to targeting using conventional technology. "Using the method of<br>
the invention, Applicants have shown that they have obtained successful targeting in 3 out of 69 ES cells clones in which them.OCR.10 LTVEC (containing more than 160 kb of homology arms, and introducing a 20 kb deletion) had integrated, whereas using previous technology for ES cell targeting (Joyner, The Practical Approach Series, 293,1999) using a plasmid-based vector with homology arms shorter than 10-20 kb while also introducing a deletion of less than 15 kb, no targeted events were identified among more than 600 integrants of the vector. These data clearly demonstrate the superiority of the method of the invention over previous technologies.<br>
Example 2: Increased targeting frequency and abrogation of the need to use isogenic DNA when LTVECs are used as the targeting vectors.<br>
As noted above, the increased targeting frequency obtained using long homology arms should diminish the benefit, if any, derived from using genomic DNA in constructing LTVECs that is isogenic with (i.e. identical in sequence to) the DNA of the eukaryotic cell being targeted. To test this hypothesis, Applicants have constructed several LTVECs using genomic DNA derived from the same mouse substrain as the eukaryotic cell to be targeted (presumably isogenic), and a large number of other LTVECs using genomic DNA derived from mouse substrains differing from that of the eukaryotic cell to be targeted (presumably non-isogenic). The non-isogenic LTVECs exhibited an average targeting frequency of 6% (ranging from 1-20%, Table 1), while the isogenic LTVECs exhibited as average targeting frequency of 3% (ranging from 2-5%), indicating that the rate of successful targeting using LTVECs does not depend on isogenicity.<br>
(Table Remove) <br>
Example 3: Detailed description of the TaqMan®-based MOA for identification of targeted ES clones<br>
ES cell clones that have taken up the LTVEC and incorporated,.it into the genome at the targeted locus by homologous recombination, are identified by a modification of allele (MOA) assay that uses real-time quantitative PCR to discern the difference between targeted ES cell clones, in which one of the two targeted alleles is modified, and non-targeted ES cell clones, in which both alleles remain unmodified. The MOA assay consists of a primary and a secondary screen. The primary screen contains the following steps: (1) growth.of LTVECV transfected ES cell clones on gelatin-coated 96-well plates; (2) isolation of genomic DNA from each ES cell clone; (3) use of each genomic DNA sample as a template in 8 separate quantitative PCRs on two 384-well plates in which 2 of the PCRs employ a target-locus-specific primer set that hybridyzes to DNA sequences at one end of the genomic fragment targeted for deletion ('upstream PCR'), 2 of the PCRs employ a target-locus-specific primer set that hybridyzes to DNA sequences at the other end of the genomic fragment targeted for deletion ('downstream PCR'), 4 of the PCRs employ primer sets that recognize four non-targeted reference loci ('reference PCRs')/ and each PCR includes a fluorescent probe (for example a TaqMan® [ABI], Eclipse™, or Molecular Beacon probe [Synthetic Genetics]) that recognizes the amplified sequence and whose fluorescence signal is directly proportional to the amount of PCR product; (4) running the PCRs in a device that combines a thermocycler with a fluorescence detector (for example the ABI 7900HT) that quantifies the accumulation of amplification products during the PCR and determines the threshold cycle (Of), the point in the PCR at which the fluorescence signal is detectable above background noise; (5) for each ES cell clone DNA sample, calculation of the difference in the CT values (ACT) between the upstream PCRs and each of the four reference PCRs and between the downstream PCRs and each of the four reference PCRs to create 8 tables of 96 ACj values; (6) normalization of the ACT values to positive values; (7) calculation of the median ACT value for each target-reference comparison table; (8) determination of a confidence score by use of a computer program that examines the eight ACT<br><br>
tables and calculates the number of times a given ES cell done DNA sample produces a ACT value within the tolerance ranges 0.5 to 1.5, 0.25 to 1.5,0.5 to 2.0, 0.25 to 2.0, 0.5 to 3.0 and 0.25 to 3.0 cycles greater than the median ACT (examples of computer programming languages suitable for creating or writing such a program include visual basics, Java, or any other computer'<br>
*<br>
programming language familiar to the skilled artisan); (.9) plotting the values and their medians for each of the eight ACT tables as histograms; and (10) identification of correctly targeted ES cell clone candidates from an inspection of the confidence scores and the ACT histograms. In a preferred example, the ACT value for the candidate targeted clone falls within 0.5 to 1.5 cycles greater than the median in 8 out of 8 reference comparisons.<br>
Candidate clones identified by the MOA assay primary screen are confirmed or rejected in a secondary screen, which contains the following steps: (1) use of the genomic DNA from each of the positive candidate ES cell clones, from a larger number of negative clones, and from genomic DNA copy-number standards from mice that carry one or two copies of the LTVEC LacZ-Neo cassette per diploid genome as templates in 8 separate quantitative PCRs on two 384-well plates in which 1 reaction is an upstream PCR (as in the primary screen), one reaction is a downstream PCR (as in the primary screen), 4 reactions are reference PCRs with two reference loci that are different from those used in the primary screen, one reaction is a PCR with primers and a probe that are specific for the LacZ gene of the LTVEC, and one reaction is a PCR with primers and a probe that are specific for the Neo gene of the LTVEC; (2) running the PCRs in a quantitative PCR device, as in the primary screen; (3) calculation, as in the primary screen, of the ACT values between the upstream PCR and each of the two reference PCRs, between the downstream PCRs and each of the two reference PCRs, between the LacZ PCR and each of the two reference PCRs, and between the Neo PCR and each of the two reference PCRs to create eight ACT tables; (4) normalization of the ACT values to positive values; (5) calculation of the median value for each ACT table; (6) calculation of confidence scores as in the primary screen; and (7) plotting the values and their medians for each of the eight ACT tables as histograms.<br>
From an inspection of the confidence scores and the ACT histograms for both the primary and secondary screens, correctly targeted ES clone candidates are either confirmed or rejected. In a preferred example, the ACT value for the candidate targeted clone falls within 0.5 to 1.5 cycles greater than the median in 12 out of 12 reference comparisons from .the combined prjmar,y and secondary<br>
*<br>
screens.<br>
To score the number of copies of the LTVEC per diploid genome in the confirmed, correctly targeted ES clones, their ACT values from the comparisons of the LacZ and Neo PCRs with the two reference PCRs are compared with the ACT values for the LacZ-Neo copy number standards. Each ES cell clone is scored as having 1, 2 or greater than 2 copies of the LTVEC. For each modified allele project, ES cell clones are screened in groups of 96 (usually fewer than 288 total clones) until 3 clones that score positive in the MOA assay and have a single copy of the LacZ-Neo cassette are identified.<br>
Example 4: Use of FISH to identify correctly targeted LTVECs in. ES cells.<br>
Using the LTVEC technology described herein, Applicants knocked out the SM22alpha gene in ES cells. SM22alpha is a 22-kDa smooth muscle cell (SMC) lineage-restricted protein that physically associates with cytoskeletal actin filament bundles in contractile SMCs. The targeted ES cells were then subjected to standard fluorescence in situ hybridization (FISH) on metaphase chromosomal spreads to verify that the gene was appropriately targeted. The experiment was performed with two probes: 1) an SM22alpha gene probe consisting of the unmodified SM22alpha BAC clone used to generate the LTVEC and 2) a LacZ and Neomycin DNA probe which detects only the gene modification made by the targeting event (insertion of LacZ and Neo gene cassettes). Metaphase chromosomal spreads were prepared from cells and hybridization was performed simultaneously with both probes which were labeled with different colored fluorophores to allow detection of hybridization of each probe within the same spread. A non-targeted ES cell line was analyzed in parallel as a control. As expected, in the control spreads, two alleles of SM22alpha were detected on homologous chromosomal arms,<br>
but there was no hybridization of the LacZ-Neo probe. As in controls, in targeted ES cell spreads two alleles were also detected at the same chromosomal location and on homologous chromosomes/ but double-labeling with the LacZ-Neo probe was apparent on one of the two chromosomes indicating co-localization of the SM22alpha and LacZ-Neo DNA sequences at that allele of SM22alpha. Importantly, no SM22alpha or LacZ-Neo gene sequences were detected at inappropriate locations in the spreads. Lack of extra integration of SM22alpha gene sequences and co-localization of LacZ-Neo with SM22alpha in one chromosome of a homologous pair strongly suggests that correct targeting of LacZ-Neo to one of the SM22alpha alleles via homologous recombination had occurred.<br>
Example 5: Lowering the amount of DNA used to electroporate ES cells improves targeting efficiency.<br>
Standard methods for targeted modification of genes in mouse embryonic stem (ES) cells typically employ 20 to 40 jiig of targeting vector in the electroporation procedure. Applicants have discovered that with LTVECs, electroporation with much lower amounts of DNA — in the range of about 1 to 5 /ig per 1 x 107 cells — doubles the frequency of correctly targeted homologous recombination events while greatly reducing the number of secondary/ non-homologous insertion events. This dear improvement in targeting efficiency is important because it significantly reduces the number of ES cells clones that need to be screened to find several positive clones with a correctly targeted, single-copy modification. The associated benefits are reduced cost and increased throughput.<br>
Example 6: Use of the method of the invention to create MA61 knockout mice to study muscle atrophy<br>
MA61, also called MAFbx, is a recently discovered ubiquitin ligase that is up-regulated in various conditions of muscle atrophy (See U.S. Provisional Application No. 60/264,926, filed January 30,2001, U.S. Provisional Application No. 60/311,697, filed August 10, 2001, and U.S. Provisional Application (serial number not yet known), filed October 22, 2001, all assigned to Regeneron<br>
Pharmaceuticals, Inc., each of which is incorporated herein in its entirety by reference). To further study the biological significance of this gene in muscle atrophy, knockout mice were created using the method of the invention as follows.<br>
First, to obtain a large cloned genomic fragment containing:the MA61 gene, a<br>
Bacterial Artificial Chromosome (BAG) library was screened with primers derived from the MA61 cDNA sequence. The BAG done thus obtained was then used to create a Large Targeting Vector for Eukaryotic Cells (LTVEC) as follows. A modification cassette containing a 5' homology box/lacZ gene/polyA/PGK promoter/neo/polyA/3' homology box was engineered. The homology boxes were appended to mark the sites of bacterial homologous recombination during the generation of the LTVEC. The LacZ is a reporter gene that was positioned such that its initiating codon was at the same position as the initiating codon of MA61. Following homologous recombination in bacteria, the modification cassette replaced the MA61 gene. Thus, a MA61 LTVEC was created wherein the MA61 coding sequences in the BAG done was replaced by the modification cassette engineered as described supra. LTVEC DNA was then prepared, purified, and linearized for introduction into eukaryotic cells as described infra.<br>
A MA61 LTVEC DNA miniprep was prepared (Sambrook, J., E. F. Fritsch And T. Maniatis. Molecular Cloning: A Laboratory Manual, Second Edition, Vols 1, 2, and 3,1989; Tillett and Neilan, Biotechniques, 24:568-70,572,1998; http://www.qiagen.com/literature/handbooks/plkmini/ plm_399.pdf) and re-transformed into E. coli using electroporation (Sambrook, J., E. F. Fritsch and T. Maniatis, Molecular Cloning: A Laboratory Manual, Second Edition, Vols 1, 2, and 3,1989) in order to get rid of the plasmid encoding the recombinogenic proteins that are utilized for the bacterial homologous recombination step (Zhang et al., Nat Genet, 20:123-8,1998; Narayanan et al., Gene Ther, 6:442-7,1999). Before introducing the MA61 LTVEC into eukaryotic cells, larger amounts of MA61 LTVEC were prepared by standard methodology (http://www.qiagen.com/literarure/handboolcs/plk/plklow.pdf; Sambrook, J., E. F. Fritsch And T. Maniatis. Molecular Cloning: A Laboratory Manual, Second<br>
Edition, Vols 1, 2, and 3,1989; Tillett and Neilan, Biotechniques, 24:568-70, 572, 1998).<br>
Next, to prepare the MA61 LTVEC for introduction into eukaryotic cells, the<br>
MA61 LTVEC was linearized. This was accomplished by digesting with die<br>
restriction enzyme NotI which leaves the modified endogenous gene(s) or<br>
chromosomal locus(loci) DNA flanked with long homology arms.<br>
The MA61 LTVEC was then introduced into eukaryotic cells using standard electroporation methodology (Sambrook, ]., E. F. Fritsch And T. Maniatis. Molecular Cloning: A Laboratory Manual, Second Edition, Vols 1, 2, and 3, 1989)). The cells in which the MA61 LTVEC was introduced successfully were selected by exposure to a selection agent. Because the selectable marker used in the modification cassette was die neomycin phosphotransferase (neo) gene (Beck, et al., Gene, 19:327-36,1982), the cells that had taken up the MA61 LTVEC were selected in a medium containing G418; cells that do not have the MA61 LTVEC died whereas cells that have taken up the MA61 LTVEC survived (Santerre, et al., Gene, 30:147-56,1984).<br>
Eukaryotic cells that have been successfully modified by targeting the MA61 LTVEC into the MA61 locus were identified with the quantitative PCR method TaqMan® (Lie and Petropoulos, Curr Opin Biotechnol, 9:43-8,1998).<br>
Finally, the genetically modified ES cells were used to create genetically modified, in this case knock out, mice by standard blastocyst injection technology. Thus created were the MA61 knock-outs, mice in which the MA61 gene had been deleted.<br>
Both these knock out mice and wild-type (WT) mice were exposed to atrophy-inducing conditions, created by denervating the mice, and levels of atrophy compared. First, the sciatic nerve was isolated in the mid-thigh region of the right hind limb and transected in mice. Transection of the sciatic nerve leads to denervation and, over a fourteen daay period, to atrophy in the muscles of the lower limb, specifically the tibialis anterior and gastrocnernius muscles, over a 14-day period. At 7 and 14 days following the denervation, animals were<br>
sacrificed by carbon dioxide inhalation.   Then the tibialis anterior (TA) and gastrocnemius complex (GA) were removed from the right (denervated) and left (intact) hind limbs, weighed/ and frozen at a fixed length in liquid nitrogen cooled isopentane. The amount of atrophy was assessed by comparing the    , weight of the muscles from the denervated limb with the weight of the muscles from the non-denervated limb.<br>
Muscle atrophy was assessed 7 and 14 days following transection of the right sciatic nerve. The wet weights of the right/ denervated muscles were compared to the wet weights of the left, non-denervated muscles. The rightleft comparisons are given in <br>
(Table Remove)<br><br>
At 7 and 14 days, the muscles from the knock mice showed significantly (p<.001 less atrophy than the muscles from wild type mice. difference between knock out and mice was greater at days days. while continued to showed no additional atrophy.></.001>
In summary, the approach of creating LTVECs and directly using them as targeting vectors combined with MOA screening for homologous recombination events in ES cells creates a novel method for engineering genetically modified loci that is rapid, inexpensive and represents a significant improvement over the tedious, time-consuming methods previously in use. It thus opens the possibility of a rapid large scale in vivo functional genomics<br>
analysis of essentially any and all genes in an organism's genome in a fraction of the time and cost necessitated by previous methodologies.<br>
Although the foregoing invention has been described in some detail by way of<br>
illustration and examples, it will be readily apparent to those of ordinary skill in<br>
the art that certain changes and modifications may be made "'to the teachings of<br>
the invention without departing from the spirit or scope of the appended claims.<br><br><br>
Claims<br>
1.	A method for genetically modifying an endogenous gene or chromosomal<br>
locus of interest in eukaryotic cells, comprising:<br>
a)	obtaining a large cloned genomic fragment containing a DNA<br>
sequence of interest;<br>
b)	using bacterial homologous recombination to genetically modify the<br>
large cloned genomic fragment of (a) to create a large targeting vector for use in<br>
the eukaryotic cells (LTVEC);<br>
c)	introducing the LTVEC of (b) into the eukaryotic cells to modify the<br>
endogenous gene or chromosomal locus in the cells; and<br>
d)	using a quantitative assay to detect modification of allele (MOA) in<br>
the eukaryotic cells of (c) to identify those eukaryotic cells in which the<br>
endogenous gene or chromosomal locus has been genetically modified.<br><br>
2.	The method of claim 1 wherein the large cloned genomic fragment<br>
containing a DNA sequence is homologous to the endogenous gene or<br>
chromosomal locus of interest.<br>
3.	The method of claim 1 or 2 wherein the genetic modification to the<br>
endogenous gene or chromosomal locus comprises deletion of a coding<br>
sequence, gene segment, or regulatory element; alteration of a coding sequence,<br>
gene segment, or regulatory element; insertion of a new coding sequence, gene<br>
segment, or regulatory element; creation of a conditional allele; or replacement<br>
of a coding sequence or gene segment from one species with an homologous or<br>
orthologous coding sequence from the same or a different species.<br>
4.	The method of claim 3 wherein the alteration of a coding sequence, gene<br>
segment, or regulatory element comprises a substitution, addition, or fusion.<br><br>
5.	The method of claim 4 wherein the fusion comprises an epitope tag or<br>
bifunctional protein.<br>
6.	The method of any one of the preceding claims wherein the quantitative<br>
assay comprises quantitative PCR, FISH, comparative genomic hybridization,<br>
isothermic DNA amplification,quantitative hybridization to an immobilized<br>
probe, Invader Probes®, or MMP assays®.<br>
7.	The method of claim 6 wherein the quantitative PCR comprises TaqMan®,<br>
Molecular Beacon, or Eclipse™ probe technology.<br>
8.	The method of any one of the preceding claims wherein the eukaryotic cell is<br>
a mammalian embryonic stem cell.<br>
9.	The method of claim 8 wherein the embryonic stem cell is a mouse, rat, or<br>
other rodent embryonic stem cell.<br><br>
10.	The method of any one of the preceding claims wherein the endogenous<br>
gene or chromosomal locus is a mammalian gene or chromosomal locus.<br>
11.	The method of claim 10 wherein the endogenous gene or chromosomal<br>
locus is a human gene or chromosomal locus.<br>
12.	The method of claim 10 wherein the endogenous gene or chromosomal<br>
locus is a mouse, rat, or other rodent gene or chromosomal locus.<br>
13.	The method of any one of the preceding claims wherein the LTVEC is<br>
capable of accommodating large DNA fragments greater than 20 kb.<br><br>
14.	The method of claim 13 wherein the LTVEC is capable of accommodating<br>
large DNA fragments greater than 100 kb.<br>
15.	A method for genetically modifying an endogenous gene or chromosomal<br>
locus of interest in mouse embryonic stem cells, comprising:<br><br>
a)	obtaining a large cloned genomic fragment greater than 20 kb which<br>
contains a DNA sequence of interest, wherein the large cloned DNA fragment<br>
is homologous to the endogenous gene or chromosomal locus;<br>
b)	using bacterial homologous recombination to genetically modify the<br>
large cloned genomic fragment of (a) to create a large targeting vector for use in<br>
the mouse embryonic stem cells, wherein the genetic modification is deletion of<br>
a coding sequence, gene segment, or regulatory element;<br>
c)	introducing the large targeting vector of (b) into the mouse embryonic<br>
stem cells to modify the endogenous gene or chromosomal locus in the cells;<br>
and<br>
d)	using a quantitative assay to detect modification of allele (MOA) in<br>
the mouse embryonic stem cells of (c) to identify those mouse embryonic stem<br>
cells in which the endogenous gene or chromosomal locus has been genetically<br>
modified, wherein the quantitative assay is quantitative PCR.<br><br>
16.	The method of any one of claims 1 to 15, wherein about 1-5 ng of the large<br>
targeting vector of (c) is introduced to about 1 x 107 cells.<br>
17.	A genetically modified endogenous gene or chromosomal locus produced<br>
by the method of any one of claims 1 to 16.<br>
18.	A genetically modified eukaryotic cell produced by the method of any one<br>
of claims 1 to 14 or 16.<br><br>
19.	A genetically modified cell according to claim 18 which is an embryonic<br>
stem cell.<br>
20.	A genetically modified mouse embryonic stem cell produced by the method<br>
of claim 15.<br>
21.	A non-human organism containing a genetically modified endogenous<br>
gene or chromosomal locus produced by the method of any one of claims 1 to<br>
16.<br>
22.	A mouse containing a genetically modified endogenous gene or<br>
chromosomal locus produced by the method of claim 15.<br>
23.	A non-human organism produced from the genetically modified eukaryotic<br>
cell of claim 18 or 19.<br>
24.	A mouse produced from the genetically modified mouse embryonic stem<br>
cell of claim 20.<br>
25.	A non-human organism containing a genetically modified endogenous<br>
gene or chromosomal locus of interest, produced by a method comprising the<br>
steps of:<br><br>
a)	obtaining a large cloned genomic fragment containing a DNA<br>
sequence of interest;<br>
b)	using bacterial homologous recombination to genetically modify the<br>
large cloned genomic fragment of (a) to create a large targeting vector (LTVEC)<br>
for use in embryonic stem cells;<br>
c)	introducing the LTVEC of (b) into the embryonic stem cells to modify<br>
the endogenous gene or chromosomal locus in the cells;<br><br>
d)	using a quantitative assay to detect modification of allele (MOA) in<br>
the embryonic stem cells of (c) to identify those embryonic stem cells in which<br>
the endogenous gene or chromosomal locus has been genetically modified;<br>
e)	introducing the embryonic stem cell of (d) into a blastocyst; and<br>
f)	introducing the blastocyst of (e) into a surrogate mother for gestation.<br>
26.	A mouse containing a genetically modified endogenous gene or<br>
chromosomal locus of interest, produced by a method comprising the steps of:<br>
a)	obtaining a large cloned genomic fragment greater than 20 kb which<br>
contains a DNA sequence of interest, wherein the large cloned DNA fragment<br>
is homologous to the endogenous gene or chromosomal locus;<br>
b)	using bacterial homologous recombination to genetically modify the<br>
large cloned genomic fragment of (a) to create a large targeting vector for use in<br>
the mouse embryonic stem cells, wherein the genetic modification is deletion of<br>
a coding sequence, gene segment, or regulatory element;<br>
c)	introducing the large targeting vector of (b) into the mouse embryonic<br>
stem cells to modify the endogenous gene or chromosomal locus in the cells;<br>
and<br>
d)	using a quantitative assay to detect modification of allele (MOA) in<br>
the mouse embryonic stem cells of (c) to identify those mouse embryonic stem<br>
cells in which the endogenous gene or chromosomal locus has been genetically<br>
modified, wherein the quantitative assay is quantitative PCR;<br>
e)	introducing the mouse embryonic stem cell of (d) into a blastocyst;<br>
and<br>
f)	introducing the blastocyst of (e) into a surrogate mother for gestation.<br>
27.	A non-human organism containing a genetically modified endogenous<br>
gene or chromosomal locus, produced by a method comprising the steps of:<br>
a) obtaining a large cloned genomic fragment containing a DNA sequence of interest;<br><br>
b)	using bacterial homologous recombination to genetically modify the<br>
large cloned genomic fragment of (a) to create a large targeting vector for use in<br>
eukaryotic cells (LTVEC);<br>
c)	introducing the LTVEC of (b) into the eukaryotic cells to genetically<br>
modify the endogenous gene or chromosomal locus in the cells;<br>
d)	using a quantitative assay to detect modification of allele (MOA) in<br>
the eukaryotic cells of (c) to identify those eukaryotic cells in which the<br>
endogenous gene or chromosomal locus has been genetically modified;<br>
e)	removing the nucleus from the eukaryotic cell of (d);<br>
f)	introducing the nucleus of (e) into an oocyte; and<br>
g)	introducing the oocyte of (f) into a surrogate mother for gestation.<br>
28.	A non-human organism containing a genetically modified endogenous<br>
gene or chromosomal locus, produced by a method comprising the steps of:<br>
a)	obtaining a large cloned genomic fragment containing a DNA<br>
sequence of interest;<br>
b)	using bacterial homologous recombination to genetically modify the<br>
large cloned genomic fragment of (a) to create a large targeting vector for use in<br>
eukaryotic cells (LTVEC);<br>
c)	introducing the LTVEC of (b) into the eukaryotic cells to genetically<br>
modify the endogenous gene or chromosomal locus in the cells;<br>
d)	using a quantitative assay to detect modification of allele (MOA) in<br>
the eukaryotic cells of (c) to identify those eukaryotic cells in which the<br>
endogenous gene or chromosomal locus has been genetically modified;<br>
e)	fusing the eukaryotic cell of (d) with another eukaryotic cell; and<br>
f)	introducing the fused eukaryotic cell of (e) into a surrogate mother for<br>
gestation.<br>
29.	The non-human organism of any one of claims 25, 27 or 28 wherein the<br>
non-human organism is a mouse, rat, or other rodent.<br><br>
30.	The non-human organism of any one of claims 25 to 29 which is produced<br>
by a method as further defined in any one of claims 2 to 16.<br>
31.	The use of the genetically modified cell of claim 18 or 19 for the production<br>
of a non-human organism.<br><br>
32.	The use of the genetically modified mouse embryonic stem cell of claim<br>
20 for the production of a mouse.<br>
33.	A method according to claim 1 or 15 which is substantially as<br>
hereinbefore described with reference to any one of the foregoing Examples.<br>
34.	A genetically modified endogenous gene or chromosomal locus<br>
according to claim 17 which is substantially as hereinbefore described with<br>
reference to any one of the foregoing Examples.<br>
35.	A genetically modified cell according to claim 18 or 20 which is<br>
substantially as hereinbefore described with reference to any one of the<br>
foregoing Examples.<br>
36.	A non-human organism according to any one of claims 21 to 28 which is<br>
substantially as hereinbefore described with reference to any one of the<br>
foregoing Examples.<br>
37.	Use according to claim 31 or 32 which is substantially as hereinbefore<br>
described with reference to any one of the foregoing Examples.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA2MjktZGVsbnAtMjAwMy1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">00629-delnp-2003-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA2MjktZGVsbnAtMjAwMy1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">00629-delnp-2003-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA2MjktZGVsbnAtMjAwMy1jb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">00629-delnp-2003-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA2MjktZGVsbnAtMjAwMy1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">00629-delnp-2003-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA2MjktZGVsbnAtMjAwMy1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">00629-delnp-2003-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA2MjktZGVsbnAtMjAwMy1mb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">00629-delnp-2003-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA2MjktZGVsbnAtMjAwMy1mb3JtLTE4LnBkZg==" target="_blank" style="word-wrap:break-word;">00629-delnp-2003-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA2MjktZGVsbnAtMjAwMy1mb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">00629-delnp-2003-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA2MjktZGVsbnAtMjAwMy1mb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">00629-delnp-2003-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA2MjktZGVsbnAtMjAwMy1mb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">00629-delnp-2003-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA2MjktZGVsbnAtMjAwMy1ncGEucGRm" target="_blank" style="word-wrap:break-word;">00629-delnp-2003-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA2MjktZGVsbnAtMjAwMy1wY3QtMTAxLnBkZg==" target="_blank" style="word-wrap:break-word;">00629-delnp-2003-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA2MjktZGVsbnAtMjAwMy1wY3QtMjEwLnBkZg==" target="_blank" style="word-wrap:break-word;">00629-delnp-2003-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA2MjktZGVsbnAtMjAwMy1wY3QtMjIwLnBkZg==" target="_blank" style="word-wrap:break-word;">00629-delnp-2003-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA2MjktZGVsbnAtMjAwMy1wY3QtNDAxLnBkZg==" target="_blank" style="word-wrap:break-word;">00629-delnp-2003-pct-401.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA2MjktZGVsbnAtMjAwMy1wY3QtNDAyLnBkZg==" target="_blank" style="word-wrap:break-word;">00629-delnp-2003-pct-402.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA2MjktZGVsbnAtMjAwMy1wY3QtNDA5LnBkZg==" target="_blank" style="word-wrap:break-word;">00629-delnp-2003-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA2MjktZGVsbnAtMjAwMy1wY3QtNDE2LnBkZg==" target="_blank" style="word-wrap:break-word;">00629-delnp-2003-pct-416.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjI5LURFTE5QLTIwMDMtQWJzdHJhY3QtKDI2LTAyLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">629-DELNP-2003-Abstract-(26-02-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjI5LURFTE5QLTIwMDMtQ2xhaW1zLSgyNi0wMi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">629-DELNP-2003-Claims-(26-02-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjI5LURFTE5QLTIwMDMtQ29ycmVzcG9uZGVuY2UtT3RoZXJzLSgyNi0wMi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">629-DELNP-2003-Correspondence-Others-(26-02-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjI5LURFTE5QLTIwMDMtRHJhd2luZ3MtKDI2LTAyLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">629-DELNP-2003-Drawings-(26-02-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjI5LURFTE5QLTIwMDMtRm9ybS0xLSgyNi0wMi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">629-DELNP-2003-Form-1-(26-02-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjI5LURFTE5QLTIwMDMtRm9ybS0yLSgyNi0wMi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">629-DELNP-2003-Form-2-(26-02-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjI5LURFTE5QLTIwMDMtRm9ybS0zLSgyNi0wMi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">629-DELNP-2003-Form-3-(26-02-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjI5LURFTE5QLTIwMDMtRm9ybS01LSgyNi0wMi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">629-DELNP-2003-Form-5-(26-02-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjI5LURFTE5QLTIwMDMtR1BBLSgyNi0wMi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">629-DELNP-2003-GPA-(26-02-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjI5LURFTE5QLTIwMDMtUENULTIxMC5wZGY=" target="_blank" style="word-wrap:break-word;">629-DELNP-2003-PCT-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjI5LWRlbG5wLTIwMDMtcGN0LTMzMS5wZGY=" target="_blank" style="word-wrap:break-word;">629-delnp-2003-pct-331.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjI5LURFTE5QLTIwMDMtUENULTQwOS5wZGY=" target="_blank" style="word-wrap:break-word;">629-DELNP-2003-PCT-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjI5LURFTE5QLTIwMDMtUGV0aXRpb24tMTM3LSgyNi0wMi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">629-DELNP-2003-Petition-137-(26-02-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjI5LURFTE5QLTIwMDMtUGV0aXRpb24tMTM4LSgyNi0wMi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">629-DELNP-2003-Petition-138-(26-02-2008).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="223231-3r-4r-trans-3-4-diarylchroman-derivatives-and-a-process-for-the-preparation-thereof.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="223233-a-method-of-producing-a-feed-for-aquatic-creatures-and-the-feed-obtained-thereof.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>223232</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>00629/DELNP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>29/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>26-Sep-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>09-Sep-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>24-Apr-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>REGENERON PHARMACEUTICALS, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>777 OLD SAW MILL RIVER ROAD, TARRYTOWN, NEW YORK 10591, U.S.A.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ANDREW J. MURPHY</td>
											<td>10 NEWTON COURT, CROTON-ON-HUDSON, NEW YORK 10520, U.S.A.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>DAVID FRENDEWEY</td>
											<td>564 FIRST AVENUE, APT. 22C, NEW YORK, NEW YORK 10016, U.S.A.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>ARIS N. ECONOMIDES</td>
											<td>52 NORTH WASHINGTON STREET, TARRYTOWN, NEW YORK 10591, U.S.A.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>DAVID M. VALENZUELA</td>
											<td>529 GIORDANO DRIVE, YORKTOWN HEIGHTS, NEW YORK 10598, U.S.A.</td>
										</tr>
										<tr>
											<td>5</td>
											<td>GEORGE D. YANCOPOULOS</td>
											<td>1519 BAPTIST CHURCH ROAD, YORKTOWN HEIGHTS, NEW YORK 10598, U.S.A.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/85</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US01/45375</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-10-31</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>09/732,234</td>
									<td>2000-12-07</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/244,665</td>
									<td>2000-10-31</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/223232-a-method-for-genetically-modifying-an-endogenous-gene-or-chromosomal-locus-in-eukaryotic-cells by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:27:49 GMT -->
</html>
