<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/223357-novel-fusion-proteins-of-mage-family-and-encoding-nucleic-acid-sequences-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:07:05 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 223357:NOVEL FUSION PROTEINS OF MAGE FAMILY AND ENCODING NUCLEIC ACID SEQUENCES THEREOF</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">NOVEL FUSION PROTEINS OF MAGE FAMILY AND ENCODING NUCLEIC ACID SEQUENCES THEREOF</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A tumour-associated antigen derivative from the MAGE family wherein the derivative is selected from the group: • MAGE antigen comprising derivatised thiol residues; • a recombinantly expressed MAGE fusion protein wherein the fusion partner is selected from protein D or fragment thereof; NS 1 protein from influenza or a fragment thereof; or C-lyta from Streptococcus pneumoniae or fragment thereof.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
The present invention relates to protein derivatives, comprising a tumor-associated antigen, that find utility in cancer vaccine therapy. In particular the derivatives of the invention include fusion proteins comprising an antigen encoded by the family of MAGE genes (e.g. MAGE-3, MAGE-1), linked to an immunological fusion partner which provides T helper epitopes, such as, for example the lipidated form of protein D from Haemophilus influenzae B; chemically modified MAGE proteins wherein the antigen's disulphide bridges are reduced and the resulting thiols blocked and genetically modified MAGE proteins provided with an affinity tag and/or genetically modified to prevent disulphide bridge formation. Methods are also described for purifying MAGE proteins and for formulating vaccines for treating a range of cancers, including, but not limited to Melanoma, breast, bladder, lung, NSCLC, head and squamous cell carcinoma, colon carcinoma and oesophagus carcinoma.<br>
Antigens encoded by the family of MAGE genes are predominately expressed on melanoma cells (including malignant melanoma) and some other cancers including NSCLC (non small cell lung cancer), head and neck squamous cell carcinoma, bladder transitional cell carcinoma and oesophagus carcinoma, but are not detectable on normal tissues except in the testis and the placenta (Gaugler, 1994; Weynants, 1994; Patard. 1995). MAGE-3 is expressed in 69% of melanomas (Gaugler, 1994), and can also be detected in 44% of NSCLC (Yoshimatsu 1988), 48% of head and neck squamous cell carcinoma, 34% of bladder transitional cell carcinoma 57% of oesophagus carcinoma 32% of colon cancers and 24% of breast cancers (Van Pel. 1995); Inoue, 1995 Fujie 1997; Nishimura 1997). Cancers expressing MAGE proteins are known as Mage associated tumours.<br>
The immunogenicity of human melanoma cells has been elegantly demonstrated in experiments using mixed cultures of melanoma cells and autologous lymphocytes. These culture often generate specific cytotoxic T lymphocytes (CTLs) able to lyse exclusively the autologous melanoma cells but neither autologous fibroblasts,   nor   autologous   EBV-transformed   B   lymphocytes   (Knuth,    1984;<br><br>
Anichini, 1987). Several of the antigens recognised on autologous melanoma cells by these CTL clones are now identified, including those of the MAGE family.<br>
The first antigen which could be defined through its recognition by specific CTLs on autologous melanoma cells is termed MZ2-E (Van den Eynde, 1989) and is encoded by the gene MAGE-1 (Van der Bruggen, 1991). CTLs directed against MZ2-E recognise and lyse MZ2-E positive melanoma ceils from autologous as well as from other patients provided that these cells have the HLA.A1 allele.<br>
The MAGE-1 gene belongs to a family of 12 closely related genes, MAGE 1, MAGE 2. MAGE 3, MAGE 4, MAGE 5. MAGE 6, MAGE 7 , MAGE 8, MAGE 9, MAGE 10, MAGE 11, MAGE 12, located on chromosome X and sharing with each other 64 to 85% homology in their coding sequence (De Plaen, 1994).  These are sometimes known as MAGE AL MAGE A2, MAGE A3, MAGE A4, MAGE A5, MAGE A6, MAGE A7, MAGE A8, MAGE A9, MAGE A 10, MAGE All. MAGE A 12 (The MAGE A family).  Two other groups of proteins are also pan of the MAGE family although more distantly related.  These are the MAGE B and MAGE C group. The MAGE B family includes MAGE Bl (also known as MAGE Xpl, and DAM 10), MAGE B2 (also known as MAGE Xp2 and DAM 6) MAGE B3 and MAGE B4 - the Mage C family currently includes MAGE CI and MAGE C2.  In general terms, a MAGE protein can be defined as containing a core sequence signature located towards the C-tenninal end of the protein (for example with respect to MAGE Al a 309 amino acid protein, the core signature corresponds to amino acid 195-279).<br>
The consensus pattern of the core signature is thus described as follows wherein x represents any amino acid, lower case residues are conserved (conservative variants allowed) and upper case residues are perfectly conserved.<br>
Core sequence signature<br>
LixvL(2x) I (3x)g(2x)apEExiWexl (2x)m(3-4x)Gxe (3-4x)gxp (2x) lit: (3x) VqexYLxYxqVPxsxP (2x) yeFLWGprA (2x) Et (3<br>
x) kv<br>
Conservative substitutions are well known and are generally set up as the default scoring matrices in sequence alignment computer programs.  These<br><br>
programs include PAM250 (Dayhoft M.O. et al, (1978), "A model of evolutionary changes in proteins", In "Atlas of Protein sequence and structure" 5(3) M.O. Dayhoft (ed.), 345-352), National Biomedical Research Foundation, Washington, and Blosum 62 (Steven Henikoft and Jorja G. Henikoft (1992), "Amino acid substitution matricies from protein blocks"), Proc. Natl. Acad. Sci. USA 89 (Biochemistry): 10915-10919.<br>
In general terms, substitution within the following groups are conservative substitutions, but substimtions between groups are considered non-conserved.  The groups are:<br>
i)	Aspartate/asparagine/glutamate/glutamine<br>
ii)        Serine/threonine<br>
iii)       Lysine/arginine<br>
iv)       Phenylalanine/tyrosine/tryptophane<br>
v)        Leucine/isoleucine/valine/methionine<br>
vi)       Glycine/alanine<br>
In general and in the context of this invention, a MAGE protein will be approximately 50% identical in this core region with amino acids 195 to 279 of MAGE Al.<br>
Several CTL epitopes have been identified on the MAGE-3 protein. One such epitope, MAGE-3.Al, is a nonapeptide sequence located between amino acids 168 and 176 of the MAGE-3 protein which constitutes an epitope specific for CTLs when presented in association with the MHC class I molecule HLA.A1. Recently two additional CTL epitopes have been identified on the peptide sequence of the MAGE-3 protein by their ability to mount a CTL response in a mixed culture of melanoma cells and autologous lymphocytes. These two epitopes have specific binding motifs for the HLA.A2 (Van der Bruggen. 1994) and HLA.B44 (Herman, 1996) alleles respectively.<br>
The present invention provides MAGE protein derivatives.  Such derivatives are suitable for use in therapeutic vaccine formulations which are suitable for the treatment of a range of tumour types.<br><br>
In one embodiment of the present invention, the derivative is a fusion proteins comprising an antigen from the MAGE protein family linked to a heterologous partner. The proteins may be chemically conjugated, but are preferably expressed as recombinant fusion proteins allowing increased levels to be produced in an expression system as compared to non-fused protein. Thus the fusion partner may assist in providing T helper epitopes(immunological fusion partner), preferably T helper epitopes recognised by humans, or assist in expressing the protein (expression enhancer) at higher yields than the native recombinant protein. Preferably the fusion partner will be both an immunological fusion partner and expression enhancing partner.<br>
In a preferred form of the invention, the immunological fusion partner is derived from protein D, a surface protein of the gram-negative bacterium, Haemophilus influenza B (W091/18926). Preferably the protein D derivative comprises approximately the first 1/3 of the protein, in particular approximately the first N-terminal 100-110 amino acids. Preferably the protein D derivative is lipidated. Preferably the first 109 residues of the Lipoprotein D fusion partner is included on the N-terminus to provide the vaccine candidate antigen with additional exogenous T-cell epitopes and increase expression level in E-coli (thus acting also as an expression enhancer). The lipid tail ensures optimal presentation of the antigen to antigen presenting cells.<br>
Other fusion partners include the non-structural protein from influenzae virus, NS1 (hemagglutinin). Typically the N terminal 81 amino acids are utilised, although different fragments may be used provided they include T-helper epitopes.<br>
In another embodiment the immunological fusion partner is the protein known as LYTA. Preferably the C terminal portion of the molecule is used.  Lyta is derived from Streptococcus pneumoniae which synthesize an N-acetyl-L-alanine amidase, amidase LYTA, (coded by the lytA gene {Gene, 43 (1986) page 265-272} an autolysin that specifically degrades certain bonds in the peptidoglycan backbone. The C-terminal domain of the LYTA protein is responsible for the affinity to the choline or to some choline analogues such as DEAE. This property has been exploited for the development of E.coli C-LYTA expressing plasmids useful for expression of fusion proteins.  Purification of hybrid proteins containing the C-<br><br>
LYTA fragment at its amino terminus has been described {Biotechnology: 10, (1992) page 795-798}.  As used herein a preferred embodiment utilises the repeat portion of the Lyta molecule found in the C terminal end starting at residue 178. A particularly preferred form incorporates residues 188 - 305.<br>
The immunological fusion partners noted above are also advantageous in aiding expression.  In particular, such fusions are expressed at higher yields than native recombinant MAGE proteins.<br>
Such constructs in a clinical setting have been shown by the present inventors to be able to treat melanoma. In one case, a patient with stage IV melanoma was cleared of metasties after two doses of unadjuvanted lipo D 1/3 MAGE 3 His protein.<br>
Accordingly, the present invention in the embodiment provides fusion proteins comprising a tumour-associated antigen from the MAGE family linked to an immunological fusion partner. Preferably the immunological fusion partner is protein D or fragment thereof, most preferably lipoprotein D. The MAGE proteins are preferably MAGE Al or MAGE A3. The Lipoprotein D part preferably comprises the first 1/3 of Lipoprotein D.<br>
The proteins of the present invention preferably are expressed in E. coli. In a preferred embodiment the proteins are expressed with an affinity tag, such as for example, a histidine tail comprising between 5 to 9 and preferably six histidine residues. These are advantageous in aiding purification.<br>
The present invention also provides a nucleic acid encoding the proteins of the present invention. Such sequences can be inserted into a suitable expression vector and used for DNA/RNA vaccination or expressed in a suitable host. Microbial vectors expressing the nucleic acid may be used as vaccines. Such vectors include for example, poxvirus, adenovirus, alphavirus, listeria and monarphage.<br>
A DNA sequence encoding the proteins of the present invention can be synthesized using standard DNA synthesis techniques, such as by enzymatic ligation as described by D.M. Roberts et al in Biochemistry 1985, 24, 5090-5098, by chemical synthesis, by in vitro enzymatic polymerization, or by PCR technology<br><br>
uiiiising for example a heat stable polymerase,  or by a combination of these techniques.<br>
Enzymatic polymerisation of DNA may be carried out in vitro using a DNA polymerase such as DNA polymerase I (Klenow fragment) in an appropriate buffer containing the nucleoside triphosphates dATP, dCTP, dGTP and dTTP as required at a temperature of 10°-37°C, generally in a volume of 50µl or less.   Enzymatic ligation of DNA fragments may be carried out using a DNA ligase such as T4 DNA ligase in an appropriate buffer, such as 0.05M Tris (pH 7.4), 0.01M MgCl2, 0.01M dithiothreitol, ImM spermidine, ImM ATP and O.lmg/mi bovine serum albumin, at a temperature of 4°C to ambient, generally in a volume of 50ml or less.    The chemical synthesis of the DNA polymer or fragments may be carried out by conventional phosphotriester, phosphite or phosphoramidite chemistry, using solid phase techniques such as those described in 'Chemical and Enzymatic Synthesis of Gene Fragments - A Laboratory Manual' (ed. H.G. Gassen and A. Lang), Veriag Chemie,. Weinheim (1982), or in other scientific publications, for example M.J. Gait, H.W.D. Matthes, M. Singh, B.S. Sproat, and R.C. Titmas. Nucleic Acids Research, 1982, 10, 6243; B.S. Sproat. and W. Bannwarth, Tetrahedron Letters, 1983, 24, 5771; M.D. Matteucci and M.H. Caruthers,  Tetrahedron Letters, 1980, 21, 719; M.D. Matteucci and M.H. Caruthers, Journal of the American Chemical Society, 1981, 103, 3185; S.P. Adams et al, Journal of the American Chemical Society, 1983, 105, 661; N.D. Sinha. J. Biernat, J. McMannus, and H. Koester, Nucleic Acids Research,  1984,  12, 4539; and H.W.D. Matthes et al, EMBO Journal, 1984,3, 801.<br>
The process of the invention may be performed by conventional recombinant techniques such as described in Maniatis et al.y Molecular Cloning - A Laboratory Manual; Cold Spring Harbor, 1982-1989.<br>
In particular, the process may comprise the steps of :<br>
i)	preparing a replicable or integrating expression vector<br>
capable, in a host cell, of expressing a DNA polymer comprising a nucleotide sequence that encodes the protein or an immunogenic derivative thereof;<br>
ii)        transforming a host cell with said vector;<br><br>
iii)      culturing said transformed host cell under conditions<br>
permitting expression of said DNA polymer to produce said protein; and<br>
iv)       recovering said protein.<br>
The term 'transforming' is used herein to mean the introduction of foreign DNA into a host cell. This can be achieved for example by transformation, transfection or infection with an appropriate plasmid or viral vector using e.g. conventional techniques as described in Genetic Engineering; Eds. S.M. Kingsman and A.J. Kingsman: Blackwell Scientific Publications; Oxford. England, 1988. The term 'transformed' or 'transformant' will hereafter apply to the resulting host cell containing and expressing the foreign gene of interest.<br>
The expression vectors are novel and also form pan of the invention.<br>
The replicable expression vectors may be prepared in accordance with the invention, by cleaving a vector compatible with the host cell to provide a linear DNA segment having an intact replicon, and combining said linear segment with one or more DNA molecules which, together with said linear segment encode the desired product, such as the DNA polymer encoding the protein of the invention, or derivative thereof, under iigating conditions.<br>
Thus, the DNA polymer may be preformed or formed during the construction of the vector, as desired.<br>
The choice of vector will be determined in part by the host cell, which may be prokaryotic or eukaryotic but are preferably E. Coli or CHO cells. Suitable vectors include piasmids, bacteriophages, cosmids and recombinant viruses.<br>
The preparation of the replicable expression vector may be carried out conventionally with appropriate enzymes for restriction, polymerisation and ligation of the DNA, by procedures described in, for example, Maniatis et al cited above.<br>
The recombinant host ceil is prepared, in accordance with the invention, by transforming a host cell with a replicable expression vector of the invention under transforming conditions. Suitable transforming conditions are conventional and are described in, for example, Maniatis et aL cited above, or aDNA Cloning" Vol. II, D.M. Glover ed., IRL Press Ltd, 1985.<br><br>
The choice of transforming conditions is determined by the host ceil. Thus, a bacterial host such as E. coii may be treated with a solution of CaCI2 (Cohen et a/.. Proc. Nat. Acad. Sci., 1973, 69, 2110) or with a solution comprising a mixture of RbCl, MnCl2, potassium acetate and glycerol, and then with 3-[N-morpholino]-propane-sulphonic acid. RbCl and glycerol. Mammalian cells in culture may be transformed by calcium co-precipitation of the vector DNA onto the cells. The invention also extends to a host cell transformed with a replicable expression vector of the invention.<br>
Culturing the transformed host cell under conditions permitting expression of the DNA poiymer is carried out conventionally, as described in. for example, Maniatis et al and "DNA Cloning" cited above. Thus, preferably the cell is supplied with nutrient and cultured at a temperature below 50°C.<br>
The product is recovered by conventional methods according to the host cell and according to the localisation of the expression product (intracellular or secreted into the culture medium or into the cell periplasm). Thus, where the host cell is bacterial, such as E. coli it may, for example, be lysed physically, chemically or enzymaticaily and the protein product isolated from the resulting lysate. Where the host cell is mammalian, the product may generally be isolated from the nutrient medium or from cell free extracts. Conventional protein isolation techniques include selective precipitation, adsorption chromatography, and affinity chromatography including a monoclonal antibody affinity column.<br>
The proteins of the present invention are provided either soluble in a liquid form or in a lyophilised form.<br>
It is generally expected that each human dose will comprise 1 to 1000 µg of protein, and preferably 30 - 300 µg.<br>
The present invention also provides pharmaceutical composition comprising a protein of the present invention in a pharmaceutically acceptable excipient. A preferred vaccine composition comprises at least Lipoprotein D - MAGE-3. Such vaccine may optionally contain one or more other tumor-associated antigen. For example other members belonging to the MAGE and GAGE families. Suitable other tumour associated antigen include MAGE-1, GAGE-1 or Tyrosinase proteins.<br><br>
Vaccine preparation is generally described in Vaccine Design ("The subunit and adjuvant approach" (eds. Powell M.F. &amp; Newman M.J). (1995) Plenum Press New York). Encapsulation within liposomes is described by Fullenon. US Patent 4,235,877.<br>
The proteins of the present invention are preferably adjuvanted in the vaccine formulation of the invention. Suitable adjuvants include an aluminium salt such as aluminium hydroxide gel (alum) or aluminium phosphate, but may also be a salt of calcium, iron or zinc, or may be an insoluble suspension of acvlated tyrosine, or acylated sugars, cationically or anionically derivatised polysaccharides, or polyphosphazenes. Other known adjuvants include CpG containing oligonucleotides. The oligonucleotides are characterised in that the CpG dinucleotide is unmethylated. Such oligonucleotides are well known and are described in, for example WO96/02555.<br>
In the formulation of the inventions it is preferred that the adjuvant composition induces an immune response preferentially of the TH1 type. Suitable adjuvant systems include, for example a combination of monophosphoryi lipid A, preferably 3-de-O-acylated monophosphoryi lipid A (3D-MPL) together with an aluminium salt.  CpG oligonucleotides also preferentially induce a TH1 response.<br>
An enhanced system involves the combination of a monophosphoryi lipid A and a saponin derivative particularly the combination of QS21 and 3D- MPL as disclosed in WO 94/00153, or a less reactogenic composition where the QS21 is quenched with cholesterol as disclosed in WO 96/33739.<br>
A particularly potent adjuvant formulation involving QS21 3D-MPL &amp; tocopherol in an oil in water emulsion is described in WO 95/17210 and is a preferred formulation.<br>
Accordingly in one embodiment of the present invention there is provided a vaccine comprising a protein of the present invention, more preferably a Lipoprotein D (or derivative thereof) - MAGE-3 adjuvanted with a monophosphoryi lipid A or derivative thereof.<br>
Preferably the vaccine additionally comprises a saponin, more preferably QS21.<br><br>
Preferably the formulation additional comprises an oil in water emulsion and tocopherol. The present invention also provides a method for producing a vaccine formulation comprising mixing a protein of the present invention together with a pharmaceutically acceptable excipient, such as 3D-MPL.<br>
In one aspect of the invention there is provided a process for purifying a recombinantiy produced MAGE-protein. The process comprises solubiiising the protein, for example in a strong chaotropic agent (such as for example, urea, guanidium hydrochloride), or in a Zwirterionicnic detergent, e.g. (Empigen BB - n-dodecyl- N,N-dimethyigIycine), reducing the protein's intra and inter molecular disulphide bonds, blocking the resulting thiols to prevent oxidative recoupling, and subjecting the protein to one or more chromatographic steps.<br>
Preferably, the blocking agent is an alkylating agent. Such blocking agents include but are not limited to alpha haloacids or alpha haloamides. For example iodoacetic acid and iodoacetamide which results in carboxymethyiation or carboxyamidation (carbamidomethylation) of the protein. Other blocking agents may be used and are described in the literature (See for example, The Proteins Vol II Eds H neurath, RL Hill and C-L Boeder, Academic press 1976, or Chemical Reagents for Protein modification Vol I eds. RL Lundblad and CM Noyes, CRC Press 1985). Typical examples of such other blocking agents include N -ethylmaleimide, chloroacetyl phosphate, O-methylisourea and acrylonitrile. The use of the blocking agent is advantageous as it prevents aggregation of the product, and ensure stability for downstream purification.<br>
In an embodiment of the invention the blocking asents are selected to induce<br>
■_&gt;   ._,<br>
a stable covalent and irreversible derivative (eg alpha halo acids or alpha haloamides). However other blocking agents maybe selected such that after purification the blocking agent may be removed to release the non derivatised protein.<br>
MAGE proteins having derivatised free thiol residues are new and form an aspect of the invention. In particular carboxyamidated or carboxymethylated derivatives are a preferred embodiment of the invention.<br>
In a preferred embodiment of the invention the proteins of the present invention is provided with an affinity tag, such as CLYTA or a polyhistidine tail.<br><br>
In such cases the protein after the blocking step is preferably subjected to affinity<br>
chromatography.  For those proteins with a polyhistidine tail, immobilised metal ion<br>
affinity chromatography (IMAC) may be performed.   The metal ion, may be any<br>
suitable ion for example zinc, nickel, iron, magnesium or copper, but is preferably       zinc or nickel.  Preferably the IMAC buffer contain a zwitterionic detergent such as<br>
Empigen BB (hereinafter Empigen) as this results in lower levels of endotoxin in<br>
the final product. If the protein is produced with a Clyta part, the protein may be purifed by<br>
exploiting its affinity to choline or choline analogues such as DEAE.    In an      embodiment of the invention the proteins are provided with a polyhistidine tail and<br>
a Clyta pan.    These may purified in a simple two step affinity chromatographic<br>
purification schedule.<br>
The  invention will be further described  by reference  to  the  following<br>
examples:<br>
EXAMPLE I:<br>
Preparation of the recombinant E. coli strain expressing the fusion protein Lipoprotein D-MAGE-3~His (LPD l/3-MAGE-3-ffis or LpD MAGE-3-His)<br>
1. The E. Coli expression system:<br>
For the production of Lipoprotein D the DNA encoding protein D has been cloned into the expression vector pMG 81. This plasmid utilizes signals from lambda phage DNA to drive the transcription and translation of inserted foreign genes. The vector contains the lambda PL promoter PL, operator OL and two utilization sites (NutL and NutR) to relieve transcriptional polarity effects when N protein is provided (Gross et al., 1985. Mol. &amp; Cell. Biol. 5:1015). Vectors containing the PL promoter, are introduced into an E. coli lysogenic host to stabilize the plasmid DNA. Lysogenic host strains contain replication-defective lambda phage DNA integrated into the genome (Shatzman et al., 1983; In Experimental Manipulation of Gene Expression. Inouya (ed) pp 1-14. Academic<br><br>
Press NY). The lambda phage DNA directs the synthesis of the cl repressor protein which binds to the OL repressor of the vector and prevents binding of RNA polymerase to the PL promoter and thereby transcription of the inserted gene. The cl gene of the expression strain AR58 contains a temperature sensitive mutation so that PL directed transcription can be regulated by temperature shift, i.e. an increase in culture temperature inactivates the repressor and synthesis of the foreign protein is initiated. This expression system allows controlled synthesis of foreign proteins especially of those that may be toxic to the cell (Shimataka &amp; Rosenberg, 1981. Nature 292:128).<br>
2.	The E. Coli strain AR58:<br>
The AR58 lysogenic E. coli strain used for the production of the LPD-MAGE-3-His protein is a derivative of the standard NIH E.coli K12 strain N99 (F-su- galK2, lacZ- thr-). It contains a defective lysogenic lambda phage (gaiE::TN10, 1 Kil- cI857 DHl). The Kil- phenotype prevents the shut off of host macromolecuiar synthesis. The cI857 mutation confers a temperature sensitive lesion to the cl repressor. The DHl deletion removes the lambda phage right operon and the hosts bio, uvr3, and chlA loci. The AR58 strain was generated by transduction of N99 with a P lambda phage stock previously grown on an SA500 derivative (galE::TN10, 1 Kil- cI857 DHl). The introduction of the defective lysogen into N99 was selected with tetracycline by virtue of the presence of a TN10 transposon coding for tetracyclin resistance in the adjacent galE gene. N99 and SA500 are E.coli K12 strains derived from Dr. Martin Rosenberg's laboratory at the National Institutes of Health.<br>
3.	Construction of the vector designed to express the recombinant protein<br>
LPD-MAGE-3-His:<br>
The rationale was to express MAGE 3 as a fusion protein using the N-terminal third of the lipidated protein D as fusion partner connected at the N-<br><br>
terminus of MAGE-3 and a sequence of several histidine residues (His tail) placed at its C-terminus.<br>
Protein D is a lipoprotein (a 42 kDa immunoglobulin D binding protein exposed on the surface of the Gram-negative bacterium Haemophilus influenzae). The protein is synthesized as a precursor with an 18 amino acid residue signal sequence, containing a consensus sequence for bacterial lipoprotein (WO 91/18926).<br>
When the signal sequence of a lipoprotein is processed during secretion, the Cys (at position 19 in the precursor molecule) becomes the amino terminal residue and is concomitantly modified by covalent attachment of both ester-linked and amide-linked fatty acids.<br>
The fatty acids linked to the amino-terminal cysteine residue then function as membrane anchor.<br>
The plasmid expressing the fusion protein was designed to express a precursor protein containing the 18 amino acids signal sequence and the first 109 residues of the processed protein D, two unrelated amino acids (Met and Asp), amino acid residues 2 to 314 of MAGE-3, two Gly residues functioning as a hinge region to expose the subsequent seven His residues.<br>
The recombinant strain thus produces the processed lipidated His tailed fusion protein of 432 amino acid residues long (see Figure 1), with the amino acids sequence described in ID No 1 and the coding sequence is described in ID No2.<br>
4.   Cloning strategy for the generation of the LPD-MAGE-3-His fusion protein (vector pRJT14477):<br>
A cDNA plasmid (from Dr Thierry Boon from the Ludwig Institute) containing the coding sequence for MAGE-3 gene (Gaugler B et al, 1994), and the vector PRIT 14586, containing the N terminal portion of the Lipo-D-1/3 coding sequence (prepared as outlined in Figure 2) were used. The cloning strategy included the following steps (Figure 3).<br>
a) - PCR amplification of the sequences presented in the piasmid cDNA MAGE 3 using the oligonucleotide sense: 5' gc gcc atg gat ctg gaa cag cgt agt cag<br><br>
cac tgc aag cct, and the oligonucleotide antisense: 5' gcg tct aga tta atg gtg atg gtg atg gtg atg ace gcc etc ttc ccc etc tct caa); this amplification leads to the following modifications at the N terminus: changing of the first five codons to E. coli codon usage, replacement of the Pro codon by an Asp codon at position 1, installation of an Ncol site at the 5' extremity and finally addition of two 2 Gly codons and the 7 His codon followed by an Xbal site at the C-terminus.<br>
b)	- Cloning into the TA cloning vector of invitrogen of the above amplified fragment and preparation of the intermediate vector pRIT14647.<br>
c)	- Excision of the Ncol Xbal fragment from plasmid pRIT 14647 and cloning into the vector pRIT 14586.<br>
d)	- Transformation of the host strain AR58.<br>
e)	- Selection and characterization of the E. coli strain transformants<br>
containing the plasmid pRIT 14477, expressing the LPD-MAGE-3-His fusion<br>
protein.<br>
EXAMPLE II:<br>
Preparation of the LPD1/3 - MAGE-3 -His antigen:<br>
1.  Growth and induction of bacterial strain - Expression of LPD1/3 -MAGE-3 -His:<br>
Cells of AR58 transformed with plasmid pRIT14477 were grown in 2 litre flasks, each containing 400 mL of LY12 medium supplemented with yeast extract (6.4 g/L) and kanamycin sulphate (50 mg/L). After incubation on a shaking table at 30°C for 8 +/- 1 h, a small sample was removed from each flask for microscopic examination. The contents of the two flasks were pooled to provide the inoculum for the 20 litre fermentor.<br><br>
The inoculum (about 800 mL) was added to a pre-sterilised 20 litre (total volume) fermentor containing 7 litres of medium, supplemented with 50 mg/L of kanamycin sulphate. The pH was adjusted to and maintained at 6.8 by the periodic addition of NH4OH (25 % v/v), and the temperature was adjusted to and maintained at 30°C. The aeration rate was adjusted to and maintained at 12 litres of air/min and the dissolved oxygen tension was maintained at 50 % of saturation by feedback control of the agitation speed. The over-pressure in the fermentor was maintained at 500 g/cm2 (0.5 bar).<br>
The fed-batch cultivation was carried out by controlled addition of a carbon feed solution. The feed solution was added at an initial rate of 0.04 mL/min, and increased exponentially during the first 42 hours to maintain a growth rate of<br>
0.1 h-1.<br>
After 42 hours, the temperature in the fermentor was rapidly increased to 39°C, and the feeding speed was maintained constant at 0.005 mL/g DCW/min during the induction phase for an additional 22-23 hours, during which time intracellular expression of LPD-MAGE-3-His reached a maximum level.<br>
Aliquots (15 mL) of broth were taken at regular intervals throughout the growth/induction phases and at the end of the fermentation to follow the kinetics of microbial growth and intracellular product expression and in addition, to provide samples for microbial identification/purity tests.<br>
At the end of fermentation, the optical density of the culture was between 80 and 120 (corresponding to a cell concentration of between 48 and 72 g DCW/L), and the total liquid volume was approximately 12 litres. The culture was rapidly cooled to between 6 and 10°C, and the cells of ECK32 were separated from the culture broth by centrifugation at 5000 x g at 4°C for 30 minutes. The concentrated cells of ECK32 were quickly stored in plastic bags and immediately frozen at -80°C.<br><br>
2. Extraction of the protein:<br>
The frozen concentrated cells of ECK32 were thawed to 4°C before being re-suspended in ceil disruption buffer to a final optical density of 60 (corresponding to a cell concentration of approximately 36 g DCW/L).<br>
The cells were disrupted by two passes through a high-pressure homogeniser (1000 bar). The broken cell suspension was centrifuged (x 10 OOOg at 4°C for 30 minutes) and the pellet fraction was washed twice with Triton X100 (1% w/v) + EDTA (1 mM), followed by a wash with phosphate buffered saline (PBS) -f Tween 20 (0.1% v/v) and finally a wash with PBS. Between each washing stage, the suspension was centrifuged at x 10 OOOg for 30 minutes at 4°C, the supernatant was discarded and the pellet fraction was retained.<br>
EXAMPLE HI:<br>
Characterisation of fusion Protein Lipo D - MAGE 3:<br>
1.  Purification:<br>
LPD-MAGE-3-His was purified from the cell homogenate using a sequence of steps described below:<br>
a)	- Solubilisation of the washed pellet fraction from cell disruption,<br>
b)	- Chemical reduction of intra- and inter-protein disulphide bonds followed<br>
by blocking of thiol groups to prevent oxidative re-coupling,<br>
c)	- Microfiltration of the reaction mixture for the removal of particulates<br>
and reduction of endotoxins,<br>
d)	- Capture and primary purification of LPD-MAGE-3-His by exploitation<br>
of the affinity interaction between the polyhistidine tail and zinc-loaded Chelating Sepharose,<br>
e)	- Removal of contaminant proteins by anion exchange chromatography.<br><br>
The purified LPD-MAGE 3-His was subjected to a number of polishing stages:<br>
0 - Buffer exchange/urea removal by size exclusion chromatography using<br>
Superdex 75, g) - In-process filtration,<br>
h) - Buffer exchange/desalting by size exclusion chromatography using Sephadex G25.<br>
Each of these steps is described in more detail below:<br>
1.1)	- Solubilisation of cell homogenate pellet<br>
The pellet fraction from the final washing stage (as described above) was re-solubilised overnight in 800 mL of a solution of guanidine hydrochloride (6M) and sodium phosphate (0.1 M, pH 7.0) at 4°C.<br>
1.2)	- Reduction and carboxymethylation<br>
The solubilised material (a pale yellow, turbid suspension) was flushed with argon to purge any remaining oxygen, and a stock solution of 2-mercaptoethanoi (14M) was added to provide a final concentration of 4.3M (which corresponded to 0.44 mL of 2-mercaptoethanol per mL of solution).<br>
The resulting solution was divided and transferred into two glass flasks which were both heated to 95 °C in a water bath. After 15 minutes at 95 °C, the flasks were removed from the water bath and allowed to cool, whereupon the contents were pooled into a foil-covered beaker (5 L), placed on ice, and solid iodoacetamide added with vigorous mixing to provide a final concentration of 6M (which corresponded to 1.11 g of iodoacetamide per mL of solution). The mixture was held on ice in the dark for 1 hour to ensure complete solubilisation of iodoacetamide, before being neutralised (maintaining vigorous mixing and continuous pH monitoring) by the addition of approximately 1 litre of sodium hydroxide (5 M) to give a final pH of 7.5-7.8.<br>
The resulting mixture was maintained on ice in the dark for a further 30 minutes, after which time the pH was re-adjusted to pH 7.5-7.8.<br><br>
1.3)	- Microfiltration<br>
The mixrure was micro filtered in an Amicon Proflux M12 tangential-flow unit equipped with a Minikros hollow fibre cartridge (ref. No. M22M-600-01N; area 5.600 cm2, 0.2 (am). The permeate was retained for subsequent chromatographic purification.<br>
1.4)	- Metal (Zn 2+) chelate chromatography (IMAC)<br>
Metal chelate chromatography was performed with Chelating Sepharose FF (Pharmacia Biotechnology Cat. No. 17-0575-01) packed into a BPG 100/500 column (Pharmacia Biotechnology Cat No. 18-1103-01). The dimensions of the packed bed were: diameter 10 cm; cross-sectional area 79 cm2; bed height 19 cm; packed volume 1,500 mL. The empty column was sanitised with sodium hydroxide (0.5M), then washed with purified water.<br>
The support (delivered in 20 % v/v ethanol) was washed with purified water (8 litres) on a Buchner funnel (under vacuum) and charged with zinc by passing at least 15 litres of a solution of ZnCl2 (0.1M). Excess zinc was removed by washing the support with 10 litres of purified water, until the pH of the outlet liquid reached the pH of the ZnCl: solution (pH 5.0). The support was then equilibrated with 4 litres of a solution containing guanidine hydrochloride (6M) and sodium phosphate (0.1M, pH7.0).<br>
The permeate from microfiltration, containing LPD-MAGE-3-Hjs, was mixed with the support (batch binding), before loading and packing the BPG column with the solution containing guanidine hydrochloride (6M) and sodium phosphate (0.1M, pH 7.0).<br>
The next stages of metal chelate chromatography were conducted at an eluent flow rate of 60 mL/min. The column was washed, first with the solution containing guanidine hydrochloride (6M) and sodium phosphate (0.1M, pH 7.0), then with the solution containing urea (6M) and sodium phosphate (0.1M, pH 7.0), until the column eluent attained zero absorbance at OD280 nm (baseline).<br>
The semi-pure LPD-MAGE-3-His protein fraction was eluted with 2 column volumes of a solution containing urea (6M), sodium phosphate (0.1M, pH 7.0) and imidazole (0.5M). The conductance of this fraction was approximately 16 mS/cm.<br><br>
1.5)	- Anion exchange chromatography<br>
Before continuing with anion exchange chromatography, [he conductance of the semi-pure LPD-MAGE-3-His protein fraction was reduced to approximately ~ mS/cm by dilution with a solution containing urea (6M) and Tris-HCl (20 mM, pH<br>
8.0).<br>
Anion exchange chromatography  was  performed using Q-Sepharose  FF<br>
(Pharmacia Biotechnology, Cat. No. 17-0510-01) packed in a BPG 200/500 column<br>
(Pharmacia Biotechnology Cat. No. 18-1103-11).   The dimensions of the packed<br>
bed were: diameter 10 cm; cross-sectional area 314 cm2; bed height 9 cm; packed<br>
volume 2,900 mL.<br>
The column was packed (with 20 % v/v ethanol) and washed with 9 litres of purified water at an eluent flow rate of 70 mL/min. The packed column was sanitised with 3 litres of sodium hydroxide (0.5M), washed with 30 litres of purified water, then equilibrated with 6 litres of a solution containing urea (6M) and Tris-HCl (20 mM, pH 8.0). The diluted, semi-purified LPD-MAGE-3-His was loaded onto the column and then washed with 9 litres of a solution containing urea (6M), Tris-HCl (20 mM, pH 8.0), EDTA (ImM) and Tween (0.1 %), until the absorbance (280 nm) of the eluent fell to zero.<br>
A further washing step was performed with 6 litres of a solution containing urea (6M) and Tris-HCl (20 mM, pH 8.0).<br>
The purified LPD-MAGE-3-His was eiuted from the column with a solution containing urea (6M), Tris-HCl (20 mM, pH 8.0) and NaCl (0.25M).<br>
1.6)	- Size exclusion chromatography<br>
The removal of urea from purified LPD-MAGE-3-His and the buffer exchange were both achieved by size exclusion chromatography. This was performed using Superdex 75 (Pharmacia Biotechnology Cat. No. 17-1044-01) packed in an XK 50/100 column (Pharmacia Biotechnology Cat. No. 18-8753-01). The dimensions of the packed bed were: diameter 5 cm; cross-sectionai area 19.6 cm2; bed height 90 cm; packed volume 1,800 mL.<br><br>
The column was packed in ethanol (20 %) and washed with 5 litres of purified water at an effluent flow rate of 20 mL/min. The column was sanitised with 2 litres of sodium hydroxide (0.5M), washed with 5 litres of purified water, then equilibrated with 5 litres of phosphate-buffered saline containing Tween 80 (0.1 % v/v).<br>
The purified LPD-MAGE-3-His fraction (maximum 500 mL/ desalting run) was loaded onto the column at an eluent flow rate of 20 mL/min. The desalted purified LPD-MAGE-3-His was eluted from the column with 3 litres of PBS containing Tween 80 (0.1 % v/v).<br>
The fraction containing LPD-MAGE-3-His eiuted at the void volume of the column.<br>
1.7)	- In-process filtration<br>
The bulk LPD-MAGE-3-His from size exclusion chromatography was filtered through a 0.22 jim membrane in a laminar flow hood (class 10.000). The filtered bulk was frozen at -80 °C and stored until the desalting step.<br>
1.8)	- Desalting chromatography<br>
Since the osmolality of the final bulk should be less than 400 mOsM, a further buffer exchange step was required to reduce the salt concentration. This was performed by a desalting chromatographic step using Sephadex G25 (Pharmacia Biotechnology Cat. No. 17-0033-02) packed in a BPG 100/950 column (Pharmacia Biotechnology Cat. No. 18-1103-03). The dimensions of the packed bed were: diameter 10 cm; cross-sectional area 78.6 cm2; bed height 85 cm; packed volume 6,500 mL.<br>
The Sephadex G25 was hydrated with 7 litres of purified water and allowed to swell overnight at 4 °C. The gel was then packed in the column with pure water at an eluent flow rate of 100 mL/min.<br>
The column was sanitised with 6 litres of sodium hydroxide (0.5M), then equilibrated with 10 litres of a solution containing sodium phosphate (10 mM, pH 6.8), NaCl (20 mM) and Tween 80 (0.1 % v/v).<br><br>
The purified LPD-MAGE-3-His fraction (maximum 1500 mL/desalting run) was loaded onto the column at an eluent flow rate of 100 mL/min, The desalted purified LPD-MAGE-3-His fraction eluted at the void volume of the column, was sterile filtered through a 0.22 µm membrane and stored at -80°C. The final bulk protein is thawed to +4°C before being aiiquoted into vials and freeze-dried in a lactose excipient (3.2%).<br>
2.	Analysis on Coomassie-stained SDS-polyacrylamide gels:<br>
The LPD-MAGE-3-His purified antigen was analysed by SDS-PAGE on a 12.5% acrylamide gel in reducing conditions.<br>
The protein load was 50 µg for Coomassie blue staining and 5 µg for silver nitrate staining. Clinical lot 96K19 and pilot lot 96J22 were analyzed. One major band corresponding to a molecular weight of 60kDa was visualised. Two minor additional bands of approximately 45kDa and 35 kDa were also seen.<br>
3,	Western Blot Analysis:<br>
The peptides revealed by SDS-PAGE analysis of the LPD-MAGE-3-His protein were identified by Western blot using mouse monoclonal antibodies. These antibodies were developed in-house using a purified preparation of the MAGE-3-His protein (this protein does not contain the LPD part of the LPD-MAGE-3-His).<br>
Two monoclonal antibody preparations (Mab 22 and Mab 54) have been selected on the basis of their suitability for Western blot analysis and used in the identity test for lot release. Figure 4 shows the band patterns obtained for lots 96K19 and 96J22 after staining with Mabs 32 and 54. Six hundred (600) ng of protein were resolved on a 12.5% SDS-PAGE, transferred to a nylon membrane, reacted with Mabs 32 and 54 (60 µg/ml) and revealed with anti-mouse antibodies coupled to peroxidase.<br>
The 60 kDa and 30 kDa peptide detected by SDS-PAGE are revealed by both Mabs.<br><br>
EXAMPLE IV:<br>
L  Vaccine preparation using LPD-MAGE-3-His protein:<br>
The vaccine used in these experiments is produced from a recombinant DNA, encoding a Lipoprotein D l/3-MAGE-3-His, expressed in E. coii from the strain AR58, either adjuvanted or not. As an adjuvant, the formulation comprises a mixture of 3 de -O-acylated monophosphoryi lipid A (3D-MPL) and QS21 in an oil/water emulsion. The adjuvant system SBAS2 has been previously described WO 95/17210.<br>
3D-MPL: is an immunostimulant derived from the lipopoiysaccharide (LPS) of the Gram-negative bacterium Salmonella minnesota. MPL has been deacylated and is lacking a phosphate group on the lipid A moiety. This chemical treatment dramatically reduces toxicity while preserving the immunostimuiant properties (Ribi, 1986). Ribi Immunochemistry produces and supplies MPL to SB-Biologicals. Experiments performed at Smith Kline Beecham Biologicals have shown that 3D-MPL combined with various vehicles strongly enhances both the humoral and a TH1 type of cellular immunity.<br>
QS21: is a natural saponin molecule extracted from the bark of the South American tree Quillaja saponaria Molina. A purification technique developed to separate the individual saponines from the crude extracts of the bark, permitted the isolation of the particular saponin, QS21, which is a triterpene glycoside demonstrating stronger adjuvant activity and lower toxicity as compared with the parent component. QS21 has been shown to activate MHC class I restricted CTLs to several subunit Ags, as well as to stimulate Ag specific lymphocytic proliferation (Kensil, 1992). Aquila (formally Cambridge Biotech Corporation) produces and supplies QS21 to SB-Biologicals.<br>
Experiments performed at SmithKline Beecham Biologicals have demonstrated a clear synergistic effect of combinations of MPL and QS21 in the induction of both humoral and TH1 type cellular immune responses.<br><br>
The oil/water emulsion is composed an organic phase made of of 2 oils (a tocopherol and squaiene), and an aqueous phase of PBS containing Tween 80 as emulsifier.   The emulsion comprised 5% squaiene 5% tocopherol 0.4% Tween 80 and had an average particle size of 180 nm and is known as SB62 (see WO 95/17210).<br>
Experiments performed at SmithKline Beecham Biologicals have proven that the adjunction of this O/W emulsion to 3D-MPL/QS21 (SBAS2)further increases the immunostimulant properties of the latter against various subunir antigens.<br>
2.	Preparation of emuision SB62 (2 fold concentrate):<br>
Tween 80 is dissolved in phosphate buffered saiine (PBS) to give a 2% solution in the PBS. To provide 100 ml two fold concentrate emuision 5g of DL alpha tocopherol and 5ml of squaiene are vortexed to mix thoroughly. 90ml of PBS/Tween solution is added and mixed thoroughly. The resulting emulsion is then passed through a syringe and finally microfluidised by using an MHOS microfluidics machine. The resulting oil droplets have a size of approximately 180 nm.<br>
3.	Preparation of Lipoprot. Dl/3 - MAGE-3-His QS21/3D MPL oil in water<br>
(SBAS2) formulation:<br>
The adjuvant is formulated as a combination of MPL and QS2L in an oil/water emulsion. This preparation is delivered in vials of 0.7 ml to be admixed with the lyophilised antigen (vials containing from 30 to 300 peg antigen).<br>
The composition of the adjuvant diluent for the lyophilised vaccine is as follows :<br><br><br>
The final vaccine is obtained after reconstitution of the lyophilised LPD-MAGE-3-His preparation with the adjuvant or with PBS alone.<br>
The adjuvants controls without antigen were prepared by replacing the protein by PBS.<br>
4. Vaccine antigen: Fusion protein Lipoprotein Dl/3 - MAGE-3-His:<br>
Lipoprotein D is a lipoprotein exposed on the surface of the Gram-negative bacteria Haemophilus influenzae.<br><br>
The inclusion of the first 109 residues of the processed protein D as fusion partner is incorporated to provide the vaccine antigen with a T-cell epitopes. Besides the LPD moiety, the protein contains two unrelated amino acids (Met and Asp), amino acid residues 2 to 314 of Mage-3, two Gly residues functioning as hinge region to expose the subsequent seven His residues.<br>
EXAMPLE V:<br>
L Immunogeaicity of LPD-MAGE-3-His in mice and monkeys:<br>
In order to test the antigenicity and immunogenicity of the human MAGE-3 protein, the candidate vaccine was injected into 2 different mouse strains (C57BL/6 and Balb/C), varying in their genetic background and MHC alleles. For both mouse strains, potential MHC class-I and MHC class-II peptide motifs were theoretically predicted for the MAGE pan of the LPD-MAGE-3-His fusion protein.<br>
a)	- Immunization protocol:<br>
5 mice of each strain were injected twice at 2 weeks interval in the foot pad with 5 µg of LPD-MAGE-3-His, formulated or not in SBAS2 at l/10th of the concentration used in human settings.<br>
b)	- Proliferation assay:<br>
Lymphocytes were prepared by crushing the spleen or the popliteal lymph nodes from the mice, 2 weeks after the last injection. 2 x 105 cells were placed in triplicate in 96 well plates and the cells were re-stimulated in vitro for 72 hours with different concentrations (1- 0.1 jig/ml) of His-Mage 3 as such or coated onto latex micro-beads.<br>
An increased MAGE-3 specific lymphoproliferative activity was observed with both spleen cells (see Figures 5 and 7) and lymph node ceils (see Figures 6 and 8) from either C57BL/6 or Balb/C mice injected with the LPD-MAGE-3-His<br><br>
protein, as compared with the lymphoprohferative response of mice having received the SBAS-2 formulation alone or PBS.<br>
Moreover, a significant higher proliferative response was obtained with lymphocytes from mice immunized with LPD-MAGE-3-His in the adjuvant SBAS2<br>
(see Figures 6 and 8).<br>
c) - Conclusion:<br>
LPD-MAGE-3-His is immunogenic in mice, and this immunogenicity can be increased by the use of the SBAS2 adjuvant formulation.<br>
2. Antibody response:<br>
a)	- Immunization Protocol:<br>
Balb/c or C57BL/6 mice were immunized by 2 intra foot pad injections at 2 weeks interval with either PBS. or SBAS2, or 5 pG of LPD-MAGE-3-His, or 5 fiG of LPD-MAGE-3-His + SBAS2.<br>
Three and five animals were used in the control groups and in the tested groups respectively.<br>
b)	- Indirect ELISA:<br>
Two weeks after the second injection, individual sera were taken and submitted to an indirect ELISA.<br>
2 fiGIvcl of purified His MAGE 3 was used as coated antigen. After saturation during 1 hour at 37°C, in PBS + 1% newborn calf serum, the sera were serially diluted (starting at 1/1000) in the saturation buffer and incubated overnight at 4°C, or 90 minutes at 37°C. After washing in PBS/Tween 20,01%, Biotinylated goat anti-mouse total IgG (1/1000) or goat anti-mouse IgGl, IgG2a, IgG2b antisera (1/5000) were used as second antibodies. After 90 minutes incubation at 37°C. Streptavidin coupled to peroxidase was added, and TMB (tetra-methyl-benzidine peroxide) was used as substrate. After 10 minutes the reaction was blocked by addition of H:S04 0.5M, and the O.D. was determined.<br><br>
c) - Results;<br>
Figure 9 compares between the different groups of mice (N = 5/group), the relative mean midpoint titer of the sera, which consists in the mean dilution needed to reach the midpoint of the curves.<br>
These results show that in both mouse strains tested, a weak Ab response is mounted after 2 injections of LPD-MAGE-3-His alone, but that higher anti-MAGE 3 Ab concentrations are generated when LPD-MAGE-3-His is injected in the presence of SBAS2. Thus, only 2 injections of LPD-MAGE-3-His + SBAS2, at 2 weeks interval, are sufficient to generate the high Ab response observed.<br>
The better Ab response observed in the Balb/c mice as compared with the response obtained in the C57BL/6 mice can be explained by differences in haplotypes or in background between these 2 strains, even though the Ab titre achieved in C57BL/6 mice is also higher after injections of LPD-MAGE-3-His -r SBAS2 than after injections with LPD-MAGE-3-His alone.<br>
The Ig subclasses-specific anti-MAGE-3 responses after vaccinations in the different groups of mice can be seen on the figures 10 and 11, which give a comparison of the mean midpoint dilution of the sera.<br>
Neither IgA, nor IgM were detected in any of the serum samples even from the mice vaccinated with LPD-MAGE-3-His in the adjuvant SBAS2.<br>
On the contrary, the total IgG level was slightly higher in the sera from mice vaccinated with LPD-MAGE-3-His alone, and significantly increased in the sera of animals injected with LPD-MAGE-3-His in SBAS2.<br>
The analysis of the different IgG-subclasses concentrations show that a mixed Ab response was induced in the mice, since the levels of all IgG subclasses tested (IgGl, IgG2a, IgG2b) were higher in mice vaccinated with the adjuvanted Ag than in mice injected with the Ag or the adjuvant alone.<br>
The nature of this mixed Ab response after vaccination with LipoD-MAGE 3 in the presence of SBAS2 seems however to depend on the mouse strain, since IgGl and IgG2b were predominantly found in the sera of Balb/c and C57BL/6 mice respectively.<br><br>
3.        Immunogenicity of Lipoprotein D 1/3 MAGE-3 - His + SBAS2 adjuvant in Rhesus monkeys<br>
Three groups of five Rhesus (Macaca Mulatta) animals were selected.   RTS,S and<br>
gp l20 were used as positive control.<br>
Groups:<br>
Group 1	right leg: RTS,S/SBAS2<br>
left leg: GP120/SBAS2<br>
Group 2	right leg: RTS,S/SB26T<br>
left leg: GP120/SB26T<br>
Group 3	right leg: LipoDl/3 Mage 3 His/SBAS2<br>
The animals received vaccine at day 0 and were boosted at day 28. and 84 and bled to determine their antibody response to both the MAGE 3 and protein D component. The vaccines were administered intramuscularly as a bolus injection (0.5ml) in the posterior pan of the right leg.<br>
Small blood samples were taken every 14 days. Unheparinized blood samples of 3 ml were collected from the femoral vein, were allowed to clot for at least 1 hour and centrifuged at room temperature for 10 minutes at 2500 rpm.<br>
Serum was removed, frozen at -20°C and sent for determination of the antibody levels by specific Elisa.<br>
96well microplates (maxisorb Nunc) were either coated with 5 µg of His Mage 3 or Protein D overnight at 4°C. After 1 hour saturation at 37°C with PBS NCS 1 %, serial dilution of the rabbit sera were added for IH 30 at 37°C (starting at 1/10), after 3 washings in PBS Tween, anti rabbit biotinylated serum (Amersham ref RPN 1004 lot 88) was added (1/5000). Plates were washed and peroxydase couple streptavidin (1/5000) was added for 30 minutes at 37°C. After washing, 50 µl TMB (BioRad) was added for 7 minutes and the reaction was stopped with H2S04 0.2M, OD was measured at 450 run. Midpoint dilutions were calculated by SoftmaxPro.<br><br>
Small blood samples were taken every 14 days to follow the kinetic of the antibody response to Mage 3 by ELISA. The results indicates that after one injection of LPD1/3 Mage 3 His + SBAS2, the Mage 3 specific total Ig titer was low, a clear boost was seen in 3 out of 5 animals after a second and a third injection of LipoDl/3 Mage 3 + adjuvant in the same monkeys. The poor responders remained negative even after 3 injections. 28 days post II or post III, the antibody titers has returned to basal levels. The subclass of these antibodies was determined as predominantly IgG and not IgM. The switch to IgG suggests that a T helper response has been triggered . The Protein D specific antibody response, although weaker, is exactly parallel to the Mage 3 antibody response.<br>
EXAMPLE VI:<br>
1.        LPD - MAGE 1 His<br>
In an analogous fashion - LPD - MAGE 1-His was prepared. The amino acid and DNA sequences are depicted in SEQUENCE ID Nos. 3 and 4. The resulting protein was purified in an analogous manner to the LPD-MAGE-3-His protein. Briefly, the cell culture were homogenated and treated with 4M guanidine HC1 and 0.5 M beta mercaptoethanol in the presence of 0.5% Empigen detergent. The product was filtered and the permeate treated with 0.6 M iodoacetamidb. The carboxyamidated fractions was subjected to IMAC (zinc Chealate-sepharose FF) chromatography. The column was first equilbrated and washed with a solution containing 4M guanidine. HC1 and sodium phosphate (2GmM, pH7.5) and 0.5% Empigen, then the column was washed with a solution containing 4M urea in sodium phosphate (20mM. pH7.5) 0.5% Empigen buffer. The protein was eluated in the same buffer, but with increasing concentration of Imidazole (20mM, 400mM and 500 mM).<br>
The eluate was diluted with 4M Urea. The Q-sepharose column was equilabrated and washed with 4M Urea in 20mM phosphate buffer (pH7.5) in the presence of 0.5% Empigen.   A second wash was performed in the same buffer, but<br><br>
devoid of the detergent.   The protein eluated in the same buffer but with increasing Imidazole (150mM, 400mM, 1M).  The eluate was ultra filtered.<br>
EXAMPLE VII:<br>
Construction of the expression plasmid pRIT14426 and transformation of the host strain AR58 to produce NS1 - MAGE -3 His:<br>
Protein design:<br>
The design of the fusion protein NSl,-MAGE-3-His to be expressed in E. coli is described in figure 12.<br>
The primary structure of the resulting protein has the sequence set forth in ID No. 5.<br>
The coding sequence(ID No. 6) corresponding to the above protein design was placed under the control of λpL promoter in a E. coli expression piasmid.<br>
The cloning strategy for the generation of NS1-MAGE-3-His fusion protein:<br>
The starting material was a cDNA piasmid received from Dr Tierry Boon from the Ludwig Institute, containing the coding sequence for MAGE-3 gene and the vector PMG81, containing the 81aa of NS, (Non structural protein) coding region from Influenza.<br>
The cloning strategy outlined in figure 13 included the following steps:<br>
a)    PCR amplification of the sequences presented in the piasmid cDNA MAGE-3 using the oligonucleotide sense: 5' gc gcc atg gat ctg gaa cag cgt agt cag cac tgc aag cct, and the oligonucleotide antisense: 5' gcg tct aga tta atg gtg atg gtg atg gtg atg ace gcc etc ttc ccc etc tct caa.<br>
This amplification leads to the following modifications at the N terminus: changing of the first five codons to the E. coli codon usage, replacement of the<br><br>
Pro codon by an Asp codon at position 1, installation of an Ncol site at the 5'; extremity and finally addition of the 2 Gly codons and the 7 His codon followed by an Xbal site at the C-terminus.<br>
b)	Cloning into the TA cloning vector of invitrogen of the above amplified fragment and preparation of the intermediate vector pRIT14647<br>
c)	Excision of the Ncol Xbal fragment form plasmid pRITl4647 and cloning into the vector pRIT PMG81<br>
d)	Transformation of the host strain AR58<br>
e)	Selection and characterization of the E. coli strain transformants containing the piasmid- pRIT 14426 (see figure 14) expressing the NSl-MAGE-3-His fusion protein<br>
Characterization of the recombinant NSrMAGE-3-His (pRIT14426):<br>
Bacteria were grown on LB Medium supplemented with 50 µg/ml kanamycin at 30 °C. When the culture had reached OD= 0.3 (at 620 ran), heat induction was achieved by raising the temperature to 42 °C.<br>
After 4 hours induction, cells were harvested, resuspended in PBS and lysed (by disintegration) by pressing three times in the French press. After centrifugation (60 minutes at 100 ,000 g), pellet supernatant and total extract were analyzed by SDS-PAGE. Proteins were visualized in Coomassie Bl stained gels where the fusion protein represented about 1 % of the total E. coli proteins. The recombinant protein appeared as a single band with an apparent MW of 44,9 K. The fusion protein was identified by Western Blot analysis using anti-NS 1 monoclonal,<br>
EXAMPLE VIII:<br>
Purification of NS1-MAGE 3-His (E. Coli) for Rabbit/Mice Immunization.<br><br>
Purification Scheme:<br>
The following purification scheme was used to purify the antigen:<br>
Lysis of cells + centrifugation<br>
T Antigen solubilisation + centrifugation<br>
T Ni2-p-NTA agarose<br>
Concentration<br><br>
Prep cell<br><br>
TCA precipitation and PBS solubilisation<br>
a.	Lysis<br>
Bacterial cells (23g) were lysed in 203 ml of a 50 mM P04 pH7 buffer by Rannie (homogeniser) and the lysate was centrifuged in a J A 20 rotor at 15.000 rpm during 30 minutes. The supernatant was discarded.<br>
b.	Antigen solubilisation<br>
1/3 of the pellet was resolubilised O/N at 4°C in 34 ml of 100 mM P04 - 6 M GuHCl pH7.  After centrifugation in a JA 20 rotor at 15,000 rpm for 30 minutes, the pellet was discarded and the supernatant was further purified by IMAC.<br>
c.	affinity chromatography:  Ni2+-NTA agarose (Qiagen)<br>
Column volume: 15 ml (16 mm x 7.5 cm)<br><br>
Packing buffer: 0.1 M P04- 6 M GuHCl pH7 Sample buffer: idem<br>
Washing buffer: 0.1 M P04- 6 M GuHCl pH7<br>
0.1 M P04 - 6 M urea pH7<br>
Elution:  imidazol gradient (0-&gt;250 mM) in 0.1 M P04 buffer pH7 supplemented with 6 M urea.<br>
Flow rate: 2 ml/min<br>
a.	Concentration:<br>
Antigen positive fractions of the MAC eluate (160 ml) were pooled and concentrated to 5 ml in an Amicon stirred cell on a Filtron membrane (type Omega cut-off 10,000). The purity at this stage is about 70% as estimated by SDS-PAGE.<br>
b.	Preparative electrophoresis (Prep Cell Biorad)<br>
2.4 ml of the concentrated sample was boiled in o.8 ml reducing sample buffer and loaded on a 10% acrylamide gel. The antigen was eluted in a Tris-Glycine buffer pH 8.3 supplemented with 4% SDS and Nst -MAGE 3 His positive fractions were pooled.<br>
a.   TCA precipitation:<br>
The antigen was TCA precipitated and after centrifugation in a JA 20 rotor at 15,000 rpm for 20 minutes, the supernatant was discarded. The pellet was resolubilised in PBS buffer pH 7.4.<br>
The protein is soluble in PBS after freeze/thaw does not show any degradation when stored for 3 hours at 37° C and has an apparent molecular weight of approximately 50.000 Daltons as determined bv SDS (12.5% PAGE).<br><br>
EXAMPLE IX:<br>
Preparation of the E. coli strain expressing a fusion protein CLYTA-MAGE-1-His tail<br>
1.	Construction of the expression plasmid pRIT14613 and transformation of<br>
the host strain AR58:<br>
Protein design:<br>
The design of the fusion protein Clyta-Mage-1-His to be expressed in E. coii is described in figure 15.<br>
The primary structure of the resulting protein has the sequence set forth in sequence ID No. 7.<br>
The coding sequence (see SEQUENCE ID No. 8) corresponding to the above protein design was placed under the control of X pL promoter in a E. coli expression plasmid.<br>
Cloning:<br>
The starting material was the vector PCUZ1 that contains the 117 C-terminal codons of the LytA coding region from Streptococcus pneumoniae and the rector pRIT14518, in which we have previously subcloned the MAGE-1 gene cDNA from a plasmid received from Dr Thierry Boon from the Ludwig Institute.<br>
The cloning strategy for the expression of CLYTA-Mage-1-His protein (see oudine in Figure 16) included the following steps:<br>
2.	Preparation of the CLYTA-Mage-1-His coding sequence module:<br>
a) The first step was a PCR amplification, destined to flank the CLYTA sequences with the Ndel-AflIII restriction sites. The PCR amplification was done   using    the   plasmid   PCUZlas    template    and    as    primers    the<br><br>
oligonucleotide sense: 5? tta aac cac ace tta agg agg ata taa cat atg aaa ggg gga art gta cat tea gac , and the oligonucleotide antisense: 5' GCC AG A CAT GTC CAA TTC TGG CCT GTC TGC CAG. This leads to the amplification of a 378 nucleotides long CLYTA sequence.<br>
b)	The second step was linking of CLYTA sequences to the MAGE-1-His sequences, to generate the coding sequence for the fusion protein. This step included the excision of a Ndel-AfIII Clyta fragment and insertion into the vector pRIT14518 previously opened by Ndel and Ncol (Ncol and AflTTT compatible) restriction enzymes and gave rise to the plasmid pRIT14613.<br>
c)	Transformation of the host strain AR58<br>
d)	Selection and characterization of the E. coli transformant (KAN resistant) containing the plasmid pRIT14613. (See figure 16)<br>
1.  Characterization of the recombinant protein CLYTA-MAGE-1-His (pRIT14613):<br>
Bacteria were grown on LB Medium supplemented with 50 /zg/mi kanamvcin at 30 °C. When the culture had reached OD= 0.3 (at 620 nm), heat induction was achieved by raising the temperature to 38 °C.<br>
After 4 hours induction, cells were harvested, resuspended in PBS and iysed (by disintegration) by one shot . After centrifugation, pellet supernatant and total extract were analyzed by SDS-PAGE. Proteins were visualized in Coomassie Bl stained gels, where the fusion protein represented about 1 % of the total E. coli proteins. The recombinant protein appeared as a single band with an apparent MW of about 49 kD. The fusion protein was identified by Western Blot analysis using anti-Mage-1 polyclonal antibodies.<br><br>
Reconstitution of the expression unit composed by the long X pL promoter (useful for Nalidixic acid induction) and the CLYTA-Mage-1 coding sequence pRIT14614):<br>
A EcoRI-NCO, restriction fragment containing the long PL promoter and a pan of CLYTA sequences was prepared from piasmid pRIT DVA6 and inserted between the EcoRI-NCOI sites of piasmid pRIT14613.<br>
The recombinant piasmid pRIT14614 was obtained.<br>
The recombinant piasmid pRIT14614 (see figure 17) encoding the fusion protein CLYTA-Mage-1-His was used to transform E. coli AR120. A Kan resistant candidate strain was selected and characterized.<br>
Characterization of the recombinant protein:<br>
Bacteria were grown on LB Medium supplemented with 50mg/ ml kanamycin at 30 °C. When the culture had reached 0D= 400 (at 620nm) Nalidixic acid was added to a final concentration of 60 mg/ ml.<br>
After 4 hours induction, cells were harvested, resuspended in PBS andJysed by desintegration (disintegration CLS "one shot" type). After centrifugation, pellet supernatant and total extract were analyzed by SDS-PAGE. Proteins were visualized in Coomassie Bleu stained gels, where the fusion protein represented about 1 % of the total E. coli proteins. The fusion protein was identified by Western blot analysis using rabbits anti-Mage-1 polyclonal antibodies. The recombinant protein appeared as a single band with an apparent MW of about 49 kD.<br><br>
EXAMPLE X: CLYTA - MAGE-3-HIS<br>
A:  Tumour rejection recombinant antigen: a fusion protein CLYTA -Mage-3-His where the C-lyt A fusion partner lead to expression of a soluble protein, act as affinity tag and provides a useful T-helper.<br>
Preparation of the E. coii strain expressing a fusion protein CLYTA-Mage-3-His tail<br>
Construction of the expression plasmid pRIT 14646 and transformation of the hose strain AR 120:<br>
Protein design:<br>
The design of the fusion protein Clyta-Mage-3-His to be expressed in E. coii is<br>
described in fisure 18.<br>
The primary structure of the resulting protein has the sequence described in SEQUENCE ID No.9: and the coding sequence in sequence ID No. 10<br>
The coding sequence corresponding to the above protein design was placed under the control of λ pL promoter in a E. coli expression plasmid.<br>
Cloning:<br>
The starting material was the vector PCUZ1 that contains the 117 C-terminal codons of the LytA coding region from Streptococcus pneumoniae, described in Gene 43, (1986) p. 265-272 and the vector pRIT14426, in which we have previously subcloned the MAGE-3 gene cDNA from a plasmid received from Dr Tierry Boon from the Ludwig Institute.<br><br>
The cloning strategy for the expression of CLYTA-MAGE-3-His protein (see outline in Figure 19) included the following steps:<br>
1- Preparation of the CLYTA-MAGE-3-His coding sequence module:<br>
1.1. The first step was a PCR amplification, destined to flank the CLYTA sequences with the Aflll and AfUII restriction sites. The PCR amplification was done using the piasmid PCUZlas template and as primers the oligonucleotide sense: 5* tta aac cac ace tta agg agg ata taa cat atg aaa ggg gga att gta cat tea gac , and the oligonucleotide antisense: 5' ccc aca tgt cca gac tgc tgg cca att ctg gec tgt ctg cca gtg . This leads to the amplification of a 427 nucleotides long CLYTA sequence. The above amplified fragment was cloned into the TA cloning vector of Invitrogen to get the intermediate vector pRIT14661<br>
1.2 The second step was linking of CLYTA sequences to the MAGE-3-His sequences, to generate the coding sequence for the fusion protein. This step included the excision of a Afl II-AfMII Clyta fragment and insertion into the vector pRIT14426 previously opened by Afl Hand Ncol (Ncol and Aflll compatible) restriction enzymes and gave rise to the piasmid pRTT14662.<br>
2.- Reconstitution of the expression unit composed by the long λ pL promoter (useful for Nalidixic acid induction) and the CLYTA-Mage-3 coding sequence:<br>
A BgUI - Xbal restriction fragment containing the short pL promoter and the CLYTA-Mage-3-His coding sequences was prepared from piasmid pRIT14662. and inserted between the BgUI - Xbal sites of piasmid TCM67 (a pBR322 derivative containing the resistance to ampicillin, and the long X pL promoter, described in the<br>
international application PCT/EP92/01827 ). The piasmid pRIT14607 was obtained.<br>
The recombinant piasmid pRTT 14607 encoding the fusion protein Clyta-Mage-3 His was used to transform E. coli AR 120 (Mott et al.1985, Proc. Natl. Acad. Sci, 82. 88).  An ampicillin resistant candidate strain was selected and characterized.<br><br>
3. Preparation of plasmid pRIT 14646:<br>
Finally a plasmid similar to pRIT 14607 but having the Kanamycin selection was constructed (pRIT 14646)<br>
Characterization of the recombinant protein:<br>
Bacteria were grown on LB Medium supplemented with 50mg/ ml kanamycin at 30°C. When the culture had reached 0D = 400 (at 600nm) Nalidixic acid was added to a final concentration of 60?g/ ml.<br>
After 4 hours induction , cells were harvested, resuspended in PBS and lysed by desintegration (desintegration CLS "one shot" type). After centrifugation, pellet supernatant and total extract were analyzed by SDS-PAGE. Proteins were visualized in Coomassie Bleu stained gels, where the fusion protein represented about 1 % of the total E. coli proteins. The fusion protein was identified by Western blot analysis using rabbits anti-Mage-3 polyclonal antibodies . The recombinant protein appeared as a single band with an apparent MW of about 58 kD.<br>
EXAMPLE XI:<br>
Purification of the recombinant protein CLYTA-Mage-3 His:<br>
The recombinant bacteria AR120 (pRIT 14646) were grown in a 20 Litters fennentor under fed-batch conditions at 30°. The expression of the recombinant protein was induced by adding Nalidixic acid at a final concentration of 60 ?g/ml. Cells were harvested at the end of fermentationand and lyzed at 60 OD/600 by two passages through a French Press disrupter (20 000 psi). Lysed cells were pelleted 20 min at 15 000 g at 4 °C. Supernatant containing the recombinant protein was loaded onto exchange DEAE Sepharose CL6B resin (Pharmacia) pre-equilibrated in 0.3M NaCl, 20 mM Tris HC1 pH 7.6 Buffer A. After a column wash with buffer A,  fusion protein was eluted by 2   %  choline in (Buffer A).  Positive antigen<br><br>
fractions, as revealed by Western blotting analysis using an anti Mage-3 antibody,<br>
were  pooled.   DEAE-eluted  antigen   was   brought  to  0.5   %   Empigen  BB   (a<br>
zwitterionic detergent) and to 0.5 M NaCl before loading onto an Ion Metal Affinity<br>
chromatography column preequilibrated in 0.5 % Empigen BB? 0.5 M NaCL 50<br>
mM phosphate buffer pH 7.6 (Buffer B).<br>
EMAC column was washed with buffer B until 280 nm absorbency reached the base<br>
line. A second wash in buffer B without Empigen BB (Buffer C) in order to<br>
eliminate the detergent was executed before Antigen elution by an Imidazole<br>
gradient 0-250mM Imidazole in buffer C.<br>
0.090-0.250 M Imidazole fractions were pooled, concentrated on a 10 kDa Filtron<br>
omega membrane before dialysis versus PBS buffer.<br>
CONCLUSION:<br>
We have demonstrated that the fused protein LPD-MAGE3-His is immunogenic in mice, and that this immunogenicity (the proliferative response and antibody response) can be further increased by the use of the adjuvant described above. Purification can be enhanced by derivatising the thiols that form disulphide bonds.<br>
We have also demonstrated that a better antibody response was triggered by the vaccination with the LPD-MAGE-3-His in the presence of the adjuvant. The predominant isotype found in the serum of C57BL/6 being IgG2b suggesting that a TH1 type immune response was raised.<br>
In the human, clinical setting a patient treated with LPD-MAGE3-His in an unadjuvanted formulation was cleared of melanoma.<br><br>
SEQUENCE LISTING<br>
(1) GENERAL INFORMATION (i) APPLICANT: SmithKline Beecham Biologicais (ii) TITLE OF THE INVENTION: Vaccine<br>
(iii) NUMBER OF SEQUENCES: 10<br>
(iv) CORRESPONDENCE ADDRESS:<br>
{A) ADDRESSEE: SmithKline 3eecham<br>
(B)	STREET: 2 New Horizons Court, Great West Road, B<br>
(C)	CITY: Middx<br>
(D)	STATE:<br>
-  (E) COUNTRY: UK (F) SIP:  TW8 9Ep<br>
(v) COMPUTER READABLE FORM:<br>
(A)	MEDIUM TYPE: Diskette<br>
(B)	COMPUTER: IBM Compatible<br>
(C)	OPERATING SYSTEM: DOS<br>
(D)	SOFTWARE: FastSEQ for Windows Version 2.0<br>
(vi) CURRENT APPLICATION DATA:<br>
(A)	APPLICATION NUMBER:<br>
(B)	FILING DATE:<br>
(C)	CLASSIFICATION:<br>
(vii) PRIOR APPLICATION DATA:<br>
(A)	APPLICATION NUMBER:<br>
(B)	FILING DATE:<br>
(viii) ATTORNEY/AGENT INFORMATION;<br>
(A)	NAME:  Dalton, Marcus J<br>
(B)	REGISTRATION NUMBER:<br>
(C)	REFERENCE/DOCKET NUMBER: B45126<br>
(ix) TELECOMMUNICATION INFORMATION:<br>
(A)	TELEPHONE: 0181 9756348<br>
(B)	TELEFAX: 0131 9756177<br>
(C)	TELEX:<br>
(2) INFORMATION FOR SEQ ID NO:l:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 452 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:1:<br>
Met Asp Pro Lys Thr Leu Ala	Leu Ser	Leu	Leu Ala Ala Gly Val Leu<br>
15	10	15<br>
Ala Gly Cys Ser Ser His Ser	Ser Asn	Met	Ala Asn Thr Gln Met Lvs<br>
20	25	30<br>
Ser Asp Lys lle lle lle Ala	His Arg	Gly	Ala Ser Gly Tyr Leu Pro<br><br>
35	40	45<br>
Glu His Thr Leu Glu Ser Lvs  Ala Leu Ala PIlH Ala Gln Gln Ala Asp<br>
50	55	60<br>
Tyr Leu Glu Gln Asp Leu Ala Met Thr Lys Asp Giy Arg Leu Val Val<br>
65	70	75	80<br>
lie His Asp His PIlH Leu ASD Glv Leu Thr ASD Val Ala Lys Lys PIlH<br>
85               90	95<br>
Pro His Arg His Arg Lys ASD Gly Arg Tyr Tyr Val Ile Asp PIlH Thr<br>
100	105	110<br>
Leu Lys Glu IlH Gln Ser Leu Glu Met Thr Glu Asn PIlH Gliu Thr Met<br>
115	120	125<br>
Asp Leu Glu Gln Arg Ser Gln His Cys Lys Pro Glu Glu Gly Leu Glu<br>
130	'     135	140<br>
Ala Arg Gly Glu Ala Leu Gly Leu Val Gly Ala Gln Ala Pro Ala Thr<br>
145	150	155	160<br>
Glu Glu Gln Giu Ala Ala Ser Ser Ser Ser Thr Leu Val Glu Val Thr<br>
165	170	175<br>
Leu Gly Glu Val Pro Ala Ala Glu Ser Pro Asp Pro Pro Gln Ser Pro<br>
180	185	190<br>
Gln Gly Ala Ser Ser Leu Pro Thr Thr Met Asn Tyr Pro Leu TrD Ser<br>
195	200	205<br>
Gln Ser Tyr Glu ASG Ser Ser Asn Gln Glu Glu Glu Gly Pro Ser Thr<br>
210	'     215	220<br>
PIlH Pro ASD Leu Glu Ser Glu PIlH Gln Ala Ala Leu Ser Arg Lys Val<br>
225       "	230	235	240<br>
Ala Glu Leu Val His PIlH Leu Leu Leu Lys Tyr Arg Ala Arg Glu Pro<br>
245	250	255 .<br>
Val Thr Lys Ala Glu Met Leu Gly Ser Val Val Gly Asn Trg Gln Tvr<br>
260	265	270<br>
PIlH PIlH Pro Val IlH PIlH Ser Lys Ala Ser Ser Ser Leu Gln Leu Val<br>
275	280	285<br>
PIlH Gly IlH Glu Leu Met Glu Val Asp Pro IlH Gly His Leu Tyr IlH<br>
290	295	300<br>
PIlH Ala Thr Cys Leu Gly Leu Ser Tyr ASD Glv Leu Leu Gly ASD Asn<br>
305	310	315	 320<br>
Gln IlH Met Pro Lvs Ala Gly Leu Leu IlH lie Val Leu Ala Ile lie<br>
325	330	335<br>
Ala Arg Glu Gly ASD Cys Ala Pro Glu Glu Lys IlH Trp Glu Glu Leu<br>
340   "	345	350<br>
Ser Val Leu Glu Val PIlH Glu Gly Arg Glu Asp Ser IlH Leu Gly ASD<br>
355	360	365<br>
Pro Lys Lys Leu Leu Thr Gln His PIlH Val Gln Glu Asn Tvr Leu Glu<br>
370	375	380<br>
Tyr Arg Gln Val Pro Gly Ser ASD Pro Ala Cys Tyr Glu PIlH Leu TrD<br>
385	390	395	400<br>
Gly Pro Arg Ala Leu Val Glu Thr Ser Tyr Val Lys Val Leu His His<br>
405	410	415<br>
Met Val Lys IlH Ser Gly Gly Pro His IlH Ser Tvr Pro Pro Leu His<br>
420	425	"      430<br>
Glu Trp Val Leu Arg Glu Gly Glu Glu Thr Ser Gly Gly His His His<br>
435	440	445<br>
His His His 450<br>
(2) INFORMATION FOR SEQ ID NO:2:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 1353 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br><br>
(xi)    SEQUENCE   DESCRIPTION:   SEQ   ID  NO:2:<br>
ATGGATCCAA AAACT77AGC  CCTTTCTTTA  TTAGCAGCTG  GCGTACTAGC  AGGTTGTAGC	60<br>
AGCCATTCAT  CAAATATGGC   GAATACCCAA  ATGAAATCAG  ACAAAATCAT   TATTGCTCAC	120<br>
CGTGGTGCTA  GCGG7TATTT  ACCAGAGCAT  ACGTTAGAAT   CTAAAGCACT   TGCGTTTGCA	180<br>
CAACAGGCTG ATTATTTAGA  GCAAGATTTA  GCAATGACTA AGGATGGTCG  TTTAGTGGTT	24 0<br>
ATTCACGATC  ACTTTTTAGA  TGGCTTGACT   GATGTTGCGA AAAAATTCCC  ACATCGTCAT	300<br>
CGTAAAGA.TG  GCCGTTACTA  TGTCATCGAC  TTTACCTTAA AAGAAATTCA AAGTTTAGAA	360<br>
ATGACAGAAA ACTTTGAAAC  CATGGATCTG  GAACAGCGTA GTCAGCACTG  CAAGCCTGAA	420<br>
GAAGGCCTTG AGGCCCGAGG AGAGGCCCTG   GGCCTGGTGG  GTGCGCAGGC  TCCTGCTACT	4 80<br>
GAGGAGCAGG AGGCTGCCIC  CTCCTCTTCT  ACTCTAGTTG AAGTCACCCT   GGGGGAGG7G	54 0<br>
CCTGCTGCCG AGTCACCAGA  TCCTCCCCAG  AGTCCTCAGG  GAGCCTCCAG  CC7CCCCACT	600<br>
ACCATGAACT ACCCTCTCTG  GAGCCAATCC  TATGAGGACT  CCAGCAACCA AGAAGAGGAG	660<br>
GGGCCAAGCA  CC7TCCCTGA  CC7GGAGTCC   GAG77CCAAG  CAGCAC7CAG  7AGGAAGG7G	720<br>
GCCGAAT7GG  77CA7T77C7  GC7CCTCAAG   TA7CGAGCCA GGGAGCCGGT  CACAAAGGCA	780<br>
GAAA7GC7GG GGAG7G7CG7  CGGAAA7TGG  CAGTA77TCT 77CCTGTGAT  CTTCAGCAAA	84 0<br>
GCT7CCAG77   CC77GCAGC7   GGTC777GGC  ATCGAGC7GA 7GGAAG7GGA  CCCCA7CGGC	900<br>
CAC7TG7ACA TC777GCCAC  CTGCC7GGGC  CTCTCCTACG ATGGCC7GCT  GGG7GACAAT	960<br>
CAGA7CA7GC  CCAAGGCAGG  CC7CCTGATA A7CGTCC7GG  CCATAA7CGC AAGAGAGGGC	1020<br>
GAC7G7GCCC  CTGAGGAGAA AA7C7GGGAG  GAGC7GAG7G  TG7TAGAGGT  GT7TGAGGGG	1080<br>
AGGGAAGACA G7A7C7TGGG  GGA7CCCAAG  AAGC7GC7CA CCCAACA77T  CG7GCAGGAA	1140<br>
AAC7ACC7GG AG7ACCGGCA  GG7CCCCGGC  AG7GATCCTG  CA7G7TA7GA A77CC7G7GG	1200<br>
GG7CCAAGGG  CCC7CG77GA AACCAGCTAT   GTGAAAGTCC  TGCACCATAT   GGTAAAGA7C	1260<br>
AG7GGAGGAC  C7CACAT77C  C7ACCCACCC   CTGCA7GAGT  GGGT7T7GAG  AGAGGGGGAA	1320<br>
GAGGGCGG7C  A7CACCATCA  CCA7CACCA7   7AA	1353<br>
(2)    INFORMATION   FOR  SEQ   ID   NO: 3:<br>
(i)   SEQUENCE  CHARACTERISTICS:<br>
(A)	LENG7H:   1341  base  pairs<br>
(B)	TYPE:   nucleic  acid<br>
(C)	S7RANDEDNESS:   single<br>
(D)	TOPOLOGY:   linear<br>
(ii)   MOLECULE  TYPE:   cDNA<br>
(xi)   SEQUENCE  DESCRIPTION:   SEQ   ID NO: 3:<br>
ATGGATCCAA AAACTTTAGC  CCTTTCTTTA TTAGCAGCTG  GCGTACTAGC  AGGTTGTAGC	60<br>
AGCCATTCAT  CAAATATGGC  GAATACCCAA ATGAAATCAG ACAAAATCAT  TATTGCTCAC	120<br>
CGTGGTGCTA GCGGTTATTT ACCAGAGCAT  ACGTTAGAAT  CTAAAGCACT  TGCGTTTGCA	180<br>
CAACAGGCTG  ATTATTTAGA GCAAGATTTA  GCAATGACTA AGGATGGTCG  TTTAGTGGTT	'24 0<br>
ATTCACGATC ACTTTTTAGA TGGCTTGACT  GATGTTGCGA AAAAATTCCC  ACATCGTCAT	300<br>
CGTAAAGATG GCCGTTACTA TGTCATCGAC  TTTACCTTAA AAGAAATTCA AAGTTTAGAA	360<br>
ATGACAGAAA ACTTTGAAAC  CATGGGCTCT   C7GGAACAGC  GTAGTCTGCA  CTGCAAGCCT	4 20<br>
GAGGAAGCCC  TTGAGGCCCA ACAAGAGGCC  CTGGGCCTGG  TGTGTGTGCA GGCTGCCACC	4 80<br>
TCCTCCTCCT  CTCCTCTGGT  CCTGGGCACC  CTGGAGGAGG TGCCCACTGC  TGGGTCAACA	540<br>
GATCCTCCCC AGAGTCCTCA GGGAGCCACC  GCCTTTCCCA CTACCATCAA CTTCACTCGA	600<br>
CAGAGGCAAC  CCAGTGAGGG  TTCCAGCAGC   CGTGAAGAGG  AGGGGCCAAG   CACCTCTTGT	660<br>
ATCCTGGAGT  CCTTGTTCCG AGCAGTAATC  ACTAAGAAGG  TGGCTGATTT   GGTTGGTTTT	7 20<br>
CTGCTCCTCA AATATCGAGC  CAGGGAGCCA GTCACAAAGG  CAGAAATGCT  GGAGAGTGTC	7 80<br>
ATCAAAAATT ACAAGCACTG  TTTTCCTGAG  ATCTTCGGCA AAGCCTCTGA GTCCTTGCAG	840<br>
CTGGTCTTTG  GCATTGACGT  GAAGGAAGCA  GACCCCACCG  GCCACTCCTA TGTCCTTGTC	900<br>
ACCTGCCTAG  GTCTCTCCTA TGATGGCCTG   CTGGGTGATA ATCAGATCAT  GCCCAAGACA	960<br>
GGCTTCCTGA TAATTGTCCT  GGTCATGATT  GCAATGGAGG  GCGGCCATGC  TCCTGAGGAG	1020<br>
GAAATCTGGG AGGAGCTGAG  TGTGATGGAG  GTGTATGATG  GGAGGGAGCA CAGTGCCTAT	1080<br>
GGGGAGCCCA GGAAGCTGCT  CACCCAAGAT   TTGGTGCAGG AAAAGTACCT  GGAGTACCGG	114 0<br>
CAGGTGCCGG  ACAGTGATCC   CGCACGCTAT   GAGTTCCTGT  GGGGTCCAAG  GGCCCTCGCT	1200<br>
GAAACCAGCT  ATGTGAAAGT  CCTTGAGTAT  GTGATCAAGG  TCAGTGCAAG  AGTTCGCTTT	1260<br>
TTCTTCCCAT  CCCTGCGTGA AGCAGCTTTG  AGAGAGGAGG AAGAGGGAGT  CGGCGGTCAT	1320<br>
CACCATCACC  ATCACCATTA A	1341<br>
(2)    INFORMATION   FOR  SEQ   ID   NO: 4 :<br><br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 4 66 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:4:<br>
Met ASD Pro Lvs Thr Leu Ala Leu Ser Leu Leu Ala Ala Gly Val Leu<br>
1    "	'        5	10	15<br>
Ala Gly Cvs   Ser Ser His Ser Ser Asn Met Ala Asn Thr Gln Met Lvs<br>
20	25	30<br>
Ser Asp Lys Ile Ile Ile Ala His Arg Gly Ala Ser Gly Tyr Leu Pro<br>
35	40	45<br>
Glu His Thr Leu Glu Ser Lys Ala Leu Ala PIlH Ala Gln Gln Ala Asp<br>
50	55	60<br>
Tvr Leu Glu Gln Asp Leu Ala Met Thr Lys Asp Gly Arg Leu Val Val<br>
65	" 70	75	80<br>
IlH His ASD His PIlH Leu Asp Gly Leu Thr Asp Val Ala Lys Lys PIlH<br>
85	90	95<br>
Pro His Arg His Arg Lys ASD Gly Arg Tyr Tyr Val IlH ASD PIlH Thr<br>
100	105	110<br>
Leu Lys Glu Ile Gln Ser Leu Glu Met Thr Glu Asn PIlH Glu Thr Met<br>
115	120	125<br>
Gly Ser Leu Glu Gln Arg Ser Leu His Cvs   Lvs Pro Glu Glu Ala Leu<br>
130	135	140<br>
Glu Aia Gln Gln Glu Ala Leu Gly Leu Val Cys Val Gln Ala Ala Thr<br>
145	150	155	160<br>
Ser Ser Ser Ser Pro Leu Val Leu Gly Thr Leu Glu Glu Val Pro Thr<br>
165	170	175<br>
Ser<br>
185<br>
Sin<br>
Ala Gly Ser Thr Asp Pro Pro Gln Ser Pro Gln Gly Ala Ser Ala PIlH<br>
180	185	190<br>
Pro Thr Thr IlH Asn PIlH Thr Arg Gln Arg Gln Pro Ser Glu Gly Ser<br>
195	200	205<br>
Ser Ser Arg Glu Glu Glu Gly Pro Ser Thr Ser Cys IlH Leu Glu Ser<br>
210	215	220<br>
Leu PIlH Arg Ala Val IlH Thr Lys Lys Val Ala Asp Leu Val Gly PIlH<br>
225	230	235	240<br>
Leu Leu Leu Lvs Tyr Arg Ala Arg Glu Pro Val Thr Lys Ala Glu Met<br>
245	250	255<br>
Leu Glu Ser Val IlH Lys Asn Tyr Lys His Cys PIlH Pro Glu IlH PIlH<br>
260	265	270<br>
Gly Lys Ala Ser Glu Ser Leu Gln Leu Val PIlH Gly IlH Asp Val Lys<br>
275	280	285<br>
Glu Ala Asp Pro Thr Gly His Ser Tyr Val Leu Val Thr Cys Leu Glv<br>
290	295	300<br>
Leu Ser Tyr Asp Gly Leu Leu Gly ASD Asn Gln IlH Met Pro Lys Thr<br>
305	310	315	320<br>
Gly PIlH Leu IlH IlH Val Leu Val Met IlH Ala Met Glu Gly Gly His<br>
325	330	335<br>
Ala Pro Glu Glu Glu IlH Trp Glu Glu Leu Ser Val Met Glu Val Tyr<br>
340	345	350<br>
Asp Gly Arg Glu His Ser Ala Tyr Gly Glu Pro Arg Lys Leu Leu Thr<br>
355	360	365<br>
Gln Asp Leu Val Gln Glu Lys Tyr Leu Glu Tyr Arg Gln Val Pro Asp<br>
370	375	380<br>
Ser Asp Pro Ala Arg Tyr Glu PIlH Leu Trp Gly Pro Arg Ala Leu Ala<br>
385	390	395	400<br>
Glu Thr Ser Tyr Val Lys Val Leu Glu Tyr Val IlH Lys Val Ser Ala<br>
405	410	415<br>
Arg Val Arg PIlH PIlH PIlH Pro Ser Leu Arg Glu Ala Ala Leu Arg Glu<br>
420	425	430<br><br>
Glu Glu Glu Gly Val Gly Gly His His His His His His His<br>
435   "	440	445<br>
(2) INFORMATION FOR SEQ ID NO:5:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 404 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein (xi) SEQUENCE DESCRIPTION: SEQ ID NO:5: Met Aso PrC Asn Thr Val Ser Ser PIlH Gln Val ASD CVS PIlH Leu TrD<br>
1<br>
j.<br>
0	15<br>
His Val Ara Lys Arg Val Ala ASD Gln Glu Leu Gly ASD Ala Pro PIlH<br>
20	25	30<br>
Leu ASD Arc Leu Arg Arg Asp Gln Lys Ser Leu Arg Glv Arg Glv Ser<br>
35"	40	45"<br>
Thr Leu Glv Leu ASD Ile Glu Thr Ala Thr Arg Ala Gly Lvs Gln Ile<br>
50   "	55	60<br>
Val Glu Arg Ile Leu Lvs Glu Glu Ser Asp Glu Ala Leu Lys Met Thr<br>
65	70	75	80<br>
Met ASD Leu Glu Gln Arg Ser Gln His Cys Lys Pro Glu Glu Gly Leu<br>
85	90	95<br>
Glu Ala Arg Gly Glu Ala Leu Gly Leu Val Gly Ala Gln Ala Pro Ala<br>
100	 105       "	110<br>
Thr Glu Glu Gln Glu Ala Ala Ser Ser Ser Ser Thr Leu Val Glu Val<br>
115	120	125<br>
Thr Leu Gly Glu Val Pro Ala Ala Glu Ser Pro ASD Pro Pro Gln Ser<br>
130	135	140<br>
Pro Gln Gly Ala Ser Ser Leu Pro Thr Thr Met Asn Tyr Pro Leu TrD<br>
145	150	155      -	160<br>
Ser Gln Ser Tyr Glu ASD Ser Ser Asn Gln Glu Glu Glu Gly Pro Ser<br>
165   *	170	175<br>
Thr PIlH Pro Asp Leu Glu Ser Glu PIlH Gln Ala Ala Leu Ser Arg Lys<br>
180	185	190<br>
Val Ala Glu Leu Val His PIlH Leu Leu Leu Lys Tyr Arg Ala Arg Glu<br>
195	200	205<br>
Pro Val Thr Lvs Ala Glu Met Leu Gly Ser Val Val Glv Asn Trg Gln<br>
210      *	215	220<br>
Tvr PIlH PIlH Pro Val Ile PIlH Ser Lvs Ala Ser Ser Ser Leu Gln Leu<br>
225	230	235	240<br>
Val PIlH Gly Ile Glu Leu Met Glu Val ASD Pro IlH Gly His Leu Tvr<br>
245	250	255<br>
IlH PIlH Ala Thr Cys Leu Glv Leu Ser Tyr Asp Gly Leu Leu Glv ASD<br>
260	265	270<br>
Asn Gln IlH Met Pro Lys Ala Gly Leu Leu IlH IlH Val Leu Ala IlH<br>
275	280	285<br>
IlH Ala Arg Glu Gly Asp Cvs  Ala Pro Glu Glu Lvs IlH Trp Glu Glu<br>
290	"    295	300<br>
Leu Ser Val Leu Glu Val PIlH Glu Glv Arg Glu Asp Ser Ile Leu Gly<br>
305	310	315	320<br>
Asp Pro Lys Lys Leu Leu Thr Gln His PIlH Val Gln Glu Asn Tyr Leu<br>
325	330	335<br>
Glu Tvr Arg Gln Val Pro Gly Ser ASD Pro Ala Cys Tvr Glu PIlH Leu<br>
340	345	"  350<br>
Trp Gly Pro Arg Ala Leu Val Glu Thr Ser Tyr Val Lvs Val Leu His<br>
355	360	365<br>
His Met Val Lys Ile Ser Gly Glv Pro His IlH Ser Tyr Pro Pro Leu<br>
370	375	380<br>
His Glu Trp Val Leu Arg Glu Gly Glu Glu Gly Gly His His His His<br><br>
385<br>
His His His<br><br>
390<br><br>
395<br><br>
400<br><br>
(2) INFORMATION FOR SEQ ID NO: 6:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 1212 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
Cii) MOLECULE TYPE: cDNA<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 6:<br><br>
ATGGATCCAA	ACACTGTGTC	AAGCTTTCAG	GTAGATTGCT	TTCTTTGGCA	TGTCCGCAAA	60<br>
CGAGTTGCAG	ACCAAGAACT	AGGTGATGCC	CCATTCCTTG	ATCGGCTTCG	CCGAGATCAG	120<br>
AAATCCCTAA	GAGGAAGGGG	CAGCACTCTT	GGTCTGGACA	TCGAGACAGC	CACACGTGCT	180<br>
GGAAAGCAGA	TAGTGGAGCG	GATTCTGAAA	GAAGAATCCG	ATGAGGCACT	TAAAATGACC	240<br>
ATGGATCTGG	AACAGCGTAG	TCAGCACTGC	AAGCCTGAAG	AAGGCCTTGA	GGCCCGAGGA	300<br>
GAGGCCCTGG	GCCTGGTGGG	TGCGCAGGCT	CCTGCTACTG	AGGAGCAGGA	GGCTGCCTCC	360<br>
TCCTCTTCTA	CTCTAGTTGA	AGTCACCCTG	GGGGAGGTGC	CTGCTGCCGA	GTCACCAGAT	420<br>
CCTCCCCAGA	GTCCTCAGGG	AGCCTCCAGC	CTCCCCACTA	CCATGAACTA	CCCTCTCTGG	480<br>
AGCCAATCCT	ATGAGGACTC	CAGCAACCAA	GAAGAGGAGG	GGCCAAGCAC	CTTCCCTGAC	540<br>
CTGGAGTCCG	AGTTCCAAGC	AGCACTCAGT	AGGAAGGTGG	CCGAATTGGT	TCATTTTCTG	600<br>
CTCCTCAAGT	ATCGAGCCAG	GGAGCCGGTC	ACAAAGGCAG	AAATGCTGGG	GAGTGTCGTC	. 660<br>
GGAAATTGGC	AGTATTTCTT	TCCTGTGATC	TTCAGCAAAG	CTTCCAGTTC	CTTGCAGCTG	720<br>
GTCTTTGGCA	TCGAGCTGAT	GGAAGTGGAC	CCCATCGGCC	ACTTGTACAT	CTTTGCCACC	780<br>
TGCCTGGGCC	TCTCCTACGA	TGGCCTGCTG	GGTGACAATC	AGATCATGCC	CAAGGCAGGC	840<br>
CTCCTGATAA	TCGTCCTGGC	CATAATCGCA	AGAGAGGGCG	ACTGTGCCCC	TGAGGAGAAA	900<br>
ATCTGGGAGG	AGCTGAGTGT	GTTAGAGGTG	TTTGAGGGGA	GGGAAGACAG	TATCTTGGGG	960<br>
GATCCCAAGA	AGCTGCTCAC	CCAACATTTC	GTGCAGGAAA	ACTACCTGGA	GTACCGGCAG	1020<br>
GTCCCCGGCA	GTGATCCTGC	ATGTTATGAA	TTCCTGTGGG	GTCCAAGGGC	CCTCGTTGAA	1080<br>
ACCAGCTATG	TGAAAGTCCT	GCACCATATG	GTAAAGATCA	GTGGAGGACC	TCACATTTCC	1140<br>
TACCCACCCC	TGCATGAGTG	GGTTTTGAGA	GAGGGGGAAG	AGGGCGGTCA	TCACCATCAC	1200<br>
CATCACCATT	AA					1212<br>
(2) INFORMATION FOR SEQ ID NO: 7<br>
i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 445 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 7:<br>
Met Lys Gly Gly IlH Val His Ser ASD Gly Ser Tyr Pro Lys Asp Lys<br>
1	5	10	15<br>
PIlH Glu Lys IlH Asn Gly Thr Tro Tvr Tyr PIlH ASD Ser Ser Gly Tyr<br>
20	"25	     30<br>
Met Leu Ala ASD Arg Trp Arg Lys His Thr Asp Gly Asn Trp Tvr Trp<br>
35	40	45<br>
PIlH Asp Asn Ser Gly Glu Met Ala Thr Gly Trp Lvs Lys IlH Ala Asp<br>
50	55	60<br>
Lys Trp Tyr Tyr PIlH Asn Glu Glu Glv Ala Met Lys Thr Gly Trp Val<br>
65	70	75	80<br>
Lys Tyr Lys Asp Thr TrD Tyr Tyr Leu ASD Ala Lvs Glu Gly Ala Met<br>
85	90	95<br>
Val Ser Asn Ala PIlH IlH Gln Ser Ala ASD Gly Thr Gly Trp Tyr Tvr<br>
100	105	110<br><br>
Leu Lvs Pro ASD Gly Thr Leu Ala Asp Arg Pro Glu Leu Asp Met Gly<br>
115	120	125<br>
Ser Leu Glu Gln Arg Ser Leu Hls Cys Lys Pro Glu Glu Ala Leu Glu<br>
130	135	140<br>
Ala Gln Gln Glu Ala Leu Gly Leu Val Cys  Val Gln Ala Ala Thr Ser<br>
145	150	155	160<br>
Ser Ser Ser Pro Leu Val Leu Gly Thr Leu Glu Glu Val Pro Thr Ala<br>
165	170	175<br>
Gly Ser Thr ASD Pro Pro Gln Ser Pro Gln Gly Ala Ser Ala PIlH Pro<br>
180	185	190<br>
Thr Thr IlH Asn PIlH Thr Arg Gln Arg Gln Pro Ser Glu Gly Ser Ser<br>
195	200	205<br>
Ser Arg Glu Glu Glu Glv Pro Ser Thr Ser Cvs  Ile Leu Glu Ser Leu<br>
210	215	220<br>
PIlH Arg Ala Val IlH Thr Lys Lys Val Ala Asp Leu Val Glv PIlH Leu<br>
225	230	235	"     240<br>
Leu Leu Lys Tyr Arg Ala Arg Glu Pro Val Thr Lys Ala Glu Met Leu<br>
245	250	255<br>
Glu Ser Val IlH Lys Asn Tyr Lys His Cys PIlH Pro Glu IlH PIlH Gly<br>
260	265 .	270<br>
Lys Ala Ser Glu Ser Leu Gln Leu Val PIlH Gly IlH Asp Val Lys Glu<br>
275	280	285<br>
Ala Asp Pro Thr Gly His Ser Tvr Val Leu Val Thr Cys Leu Gly Leu<br>
290	295	300<br>
Ser Tyr Asp Gly Leu Leu Gly Asp Asn Gln Ile Met Pro Lys Thr Gly<br>
305	310	315	320<br>
PIlH Leu IlH IlH Val Leu Val Met IlH Ala Met Glu Gly Gly His Ala<br>
325	330	335<br>
Pro Glu Glu Glu IlH Trp Glu Glu Leu Ser Val Met Glu Val Tyr ASD<br>
340	345	350<br>
Gly Arg Glu His Ser Ala Tyr Gly Glu Pro Arg Lys Leu Leu Thr Gln<br>
355	360	365<br>
ASD Leu Val Gln Glu Lys Tvr Leu Glu Tyr Arg Gln Val Pro Asp Ser<br>
370	375	380<br>
Asp Pro Ala Arg Tyr Glu PIlH Leu Trp Gly Pro Arg Ala Leu Ala Glu<br>
385	390	395	400<br>
Thr Ser Tyr Val Lys Val Leu Glu Tyr Val IlH Lys Val Ser Ala Arg<br>
405	410	415<br>
Val Arg PIlH PIlH PIlH Pro Ser Leu Arg Glu Ala Ala Leu Arg Glu Glu<br>
420	425	430<br>
Glu Glu Gly Val Gly Gly His His His His His His His<br>
435	     440	445<br>
(2) INFORMATION FOR SEQ ID NO: 8:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 1338 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br><xi sequence description: seq id no:8:></xi>
ATGAAAGGGG GAATTGTACA TTCAGACGGC TCTTATCCAA AAGACAAGTT TGAGAAAATC	60<br>
AATGGCACTT GGTACTACTT TGACAGTTCA GGCTATATGC TTGCAGACCG CTGGAGGAAG	120<br>
CACACAGACG GCAACTGGTA CTGGTTCGAC AACTCAGGCG AAATGGCTAC AGGCTGGAAG	180<br>
AAAATCGCTG ATAAGTGGTA CTATTTCAAC GAAGAAGGTG CCATGAAGAC AGGCTGGGTC	24 0<br>
AAGTACAAGG ACACTTGGTA CTACTTAGAC GCTAAAGAAG GCGCCATGGT ATCAAATGCC	300<br>
TTTATCCAGT CAGCGGACGG AACAGGCTGG TACTACCTCA AACCAGACGG AACACTGGCA	360<br>
GACAGGCCAG AATTGGACAT GGGCTCTCTG GAACAGCGTA GTCTGCACTG CAAGCCTGAG	420<br>
GAAGCCCTTG AGGCCCAACA AGAGGCCCTG GGCCTGGTGT GTGTGCAGGC TGCCACCTCC	4 80<br>
TCCTCCTCTC CTCTGGTCCT GGGCACCCTG GAGGAGGTGC CCACTGCTGG GTCAACAGAT                 540<br><br><br><br><br><br>
REFERENCES:<br>
-	Anichini A., Fossati G., Parmiani G. Immunol. Today, 8: 385 (1987).<br>
-	De Plaen E., Arden K., Traversari C., et al. Immunogenetics, 40: 360 (1994).<br>
-	Gaugler B.. Van den Eynde B., van der Bruggen P., et al. /. Exp. Med., 179: 921 (1994).<br>
-	IlHrman J., van der Bruggen P., Immanuel F., et al. Immunogenetics, 43: 377 (1996).<br>
-	Inoue H., Mori M., Li J., et al. Int. J. Cancer, 63: 523 (1995).<br>
-	Kensil C.R., Soitysik S., Patel U., et al. in: Channock R.M. , Glnsburg H.S., Brown F., et al., (eds.), Vaccines 92,<br>
(Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.), 36^0: (1992).<br>
-	Knuth A., Danowski B., Oettgen H.F., et al. Proc. Natl. Acad. Sci. USA, 81: 3511 (1984).<br>
-	Patard J.J., Brasseur F., Gil-Diez S., et al. Int. J. Cancer, 64: 60 (1995).<br>
-	Ribi E., et al. in: Levine L., Bonventre P.F., Morello J., et al. (eds)., American Society for Microbiology, Washington DC, Microbiology 1986, 9-13; (1986).<br>
-	Van den Eynde B., Hainaut P.. IlHrin M. et al. Int. J. Cancer, 44: 634 (1989).<br><br>
-	Van der Bruggen P., Traversari C, Chomez P., et al. Science, ISA: 1643 (1991).<br>
-	Van der Bruggen P., Bastin J.. Gajewski T., et al. Eur. J. Immunol., 24: 3038 (1994).<br>
-	Van Pel A., van der Bruggen P., Couiie P.G. , et al., Immunol. Rev., 145: 229 (1995).<br><br>
-	Weynants P., LetIlH B., Brasseur F., et al. Int. J. Cancer, 56: 826 (1994).<br>
-	Nishimura S, Fujita M, Terata N, Tani T, Kodama M, Itoh K, Nihon Rinsho Meneki Gakkai Kaishi 1997, Apr, 20 (2): 95-101.<br>
-	Fuijie T et al, Ann Oncol 1997 Apr, 8 (4): 369-72.<br><br><br><br><br><br><br>
We claim:<br>
1.	A tumour-associated antigen derivative from the MAGE family wherein the<br>
derivative is selected from the group:<br>
•	MAGE antigen comprising derivatised thiol residues;<br>
•	a recombinantly expressed MAGE fusion protein wherein the fusion partner is selected from protein D or fragment thereof ; NS 1 protein from influenza or a fragment thereof ; or C-lyta from Streptococcus pneumoniae or fragment thereof.<br><br>
2.	An antigen as claimed in claim 1 wherein the derivatised free thiols are carboxyamidated or carboxymethylated.<br>
3.	An antigen as claimed in claims 1 and 2 wherein the antigen comprises an affinity tag.<br>
4.	An antigen as claimed in any of claims 1 to 3 wherein the protein D or fragment thereof is lipidated.<br>
5.	An antigen as claimed in any of claims 1 to 4 wherein the MAGE protein is selected from the group:<br>
MAGE Al, MAGE A2, MAGE A3, MAGE A4, MAGE A5, MAGE A6, MAGE A7, MAGE A8, MAGE A9, MAGE AlO, MAGE All, MAGE A12, MAGE Bl, MAGE B2, MAGE 33, MAGE B4, MAGE C1 and MAGE C2.<br>
6.	A nucleic acid sequence encoding a fusion protein as claimed in any of claims 1 to 5.<br>
7.	A vector comprising a nucleic acid of claim 6.<br>
8.	A microbial host transformed with a nucleic acid of claim 6 or a vector of claim 7.<br>
9.	A vaccine containing a protein as claimed in any of claims 1 to 5.<br>
10.	A vaccine as claimed in claim 9 additionally comprising an adjuvant, and/or immunostimulatory cytokine or chemokine.<br>
11.	A vaccine as claimed in claim 9 or 10 wherein the protein is presented in an oil in water emulsion vehicle.<br><br>
12.	A vaccine as claimed in claim 10 or 11 wherein the adjuvant comprises 3D-<br>
MPL, QS21 or a CpG oligonucleotide.<br>
13.	A vaccine as claimed herein additionally comprising one or more other<br>
antigens.<br>
14.	A vaccine as claimed herein for use in medicine.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMDIyNC1jaGUgYWJzdHJhY3QtZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2000-0224-che abstract-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMDIyNC1jaGUgY2xhaW1zLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-0224-che claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMDIyNC1jaGUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKS1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-0224-che description (complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMDIyNC1jaGUgZHJhd2luZ3MtZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2000-0224-che drawings-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMjI0LWNoZS1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-224-che-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMjI0LWNoZS1jbGFpbXMgLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2000-224-che-claims .pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMjI0LWNoZS1jb3JyZXNwb25kZW5jZSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-224-che-correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMjI0LWNoZS1jb3JyZXNwb25kZW5jZSBwby5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-224-che-correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMjI0LWNoZS1kZXNjcmlwdGlvbiBjb21wbGV0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-224-che-description complete.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMjI0LWNoZS1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-224-che-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMjI0LWNoZS1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-224-che-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMjI0LWNoZS1mb3JtIDE4LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2000-224-che-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMjI0LWNoZS1mb3JtIDI2LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2000-224-che-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMjI0LWNoZS1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-224-che-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMjI0LWNoZS1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-224-che-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMjI0LWNoZS1vdGhlciBkb2N1bWVudHMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-224-che-other documents.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMjI0LWNoZS1wY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-224-che-pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtMjI0LWNoZS1wcmlvcml0eSBkb2N1bWVudHMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-224-che-priority documents.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="223356-a-process-for-the-preparation-of-cyclopropane-carboxylic-acids-and-intermediates-therefor.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="223358-a-nucleic-acid-or-functional-fragment-thereof-encoding-proteinaceous-molecule-and-a-method-for-producing-a-flavour-employing-the-same.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>223357</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2000/224/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>47/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Nov-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>09-Sep-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>27-Jul-2000</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SMITHKLINE BEECHAM BIOLOGICALS SA</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>RUE DE L&#x27;INSTITUT 89, B-1330 RIXENSART,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>CABEZON SILVA, TERESA</td>
											<td>SMITHKLINE BEECHAM BIOLOGICALS SA, RUE DE L&#x27;INSTITUT 89, B-1330 RIXENSART,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>COHEN, JOSEPH</td>
											<td>SMITHKLINE BEECHAM BIOLOGICALS SA, RUE DE L&#x27;INSTITUT 89, B-1330 RIXENSART,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>SLAOUI MONCEF, MOHAMED</td>
											<td>SMITHKLINE BEECHAM BIOLOGICALS SA, RUE DE L&#x27;INSTITUT 89, B-1330 RIXENSART,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>VINALS BASSOLS, CARLOTA</td>
											<td>SMITHKLINE BEECHAM BIOLOGICALS SA, RUE DE L&#x27;INSTITUT 89, B-1330 RIXENSART,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/12</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP99/00660</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>1999-02-02</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>9802543.0</td>
									<td>1998-02-05</td>
								    <td>U.K.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>9802650.3</td>
									<td>1998-02-06</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/223357-novel-fusion-proteins-of-mage-family-and-encoding-nucleic-acid-sequences-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:07:06 GMT -->
</html>
