<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/223392-modified-free-living-microbes-vaccine-compositions-and-methods-of-use-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:00:48 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 223392:MODIFIED FREE-LIVING MICROBES, VACCINE COMPOSITIONS AND METHODS OF USE THEREOF</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">MODIFIED FREE-LIVING MICROBES, VACCINE COMPOSITIONS AND METHODS OF USE THEREOF</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>There is disclosed a vaccine comprising modified microbes, wherein the nucleic acid of the microbes has been modified by reaction with nucleic acid targeted compound that reacts directly with the nucleic acid so that the microbes are attenuated for proliferation, wherein the microbes are bacteria, protozoa, or fungi, and wherein the modified microbes express an antigen at a level sufficient for the vaccine to induce an immune response to the antigen in a host upon administration of the vaccine to the host.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>MODIFIED FREE-LIVING MICROBES, VACCINE COMPOSITIONS AND<br>
METHODS OF USE THEREOF<br>
RELATED APPLICATIONS<br>
[0001] This application claims the priority benefit of U.S. Provisional Application<br>
No. 60/446,051, filed February 6, 2003, U.S. Provisional Application No. 60/449,153, filed<br>
February 21, 2003, U.S. Provisional Application No. 60/490,089, Field July 24, 2003, U.S.<br>
Provisional Application No. 60/511,869, Field October 15, 2003, the U.S. Provisional<br>
Application entitled "Listeria Attenuated for Entry into Non-Phagocytic Cells, Vaccines<br>
comprising the Listeria, and Methods of Use Thereof" filed February 2, 2004, the contents<br>
of each of which, are hereby incorporated by reference into the present disclosure.<br>
FIELD OF THE INVENTION<br>
[0001] The present invention relates generally to vaccine compositions and<br>
immunotherapy. In particular, the present invention relates to vaccine compositions<br>
comprising a population of a modified free-living microbe that can be used to deliver a<br>
particular antigen to an individual. In such compositions, the vaccine is directed against the<br>
microbe itself or against a heterologous antigen that has been incorporated into the<br>
microbe.) The present invention also relates to the use of the modified microbes to load<br>
and to induce the activation and maturation of antigen-presenting cells, such as dendritic<br>
cells.<br>
BACKGROUND OF THE INVENTION<br>
[0002] A variety of vaccines have been developed for clinical use, mostly targeting<br>
the prevention of infectious diseases caused by viruses, bacteria and parasites. Vaccines<br>
can be prepared from live attenuated microbes, inactivated (killed) microbes, or<br>
components of the microbes themselves. Live attenuated microbes contain genetic<br>
alterations, such as deletion of virulence factors, resulting in a less virulent microbe. For<br>
inactivated vaccines, a microbe may be chemically or physically inactivated. Ideally, such<br>
vaccines cannot cause an infection but are still able to stimulate a desired immune<br><br>
response. Examples of inactivated vaccines include polio and influenza viruses, and<br>
bacterial vaccines against cholera and pertussis, although live attenuated -vaccines are an<br>
option for polio, influenza, and cholera as well. In order to elicit the desired immune<br>
response, it is important that the inactivated microbe comprises the appropriate antigens<br>
prior to inactivation. It has been observed in some cases that inactivating the microbe<br>
results in a significantly reduced immune response because de novo gene expression by an<br>
infecting microbe is required to stimulate an optimal immune response. This is particularly<br>
important for intracellular bacteria. Methods that have been used to inactivate bacteria<br>
include the use of acetone, alcohol, formalin, glutaraldehyde, parafonnalclehyde, or phenol,<br>
heating, or ultraviolet irradiation [Pace et al., Vaccine 16(16):1563 (1998)].<br>
[0003] In addition to using microbial vaccines to prevent infectious diseases caused<br>
by the microbe itself, the microbes can be modified to contain heterologo us nucleic acid<br>
sequences that-encode a certain protein or antigen. Such recombinant microbes are used as<br>
delivery vehicles and may be used as vaccines to stimulate an immune response to the<br>
heterologous antigens. These recombinant vaccines have been shown to be effective in<br>
animal models. An oral vaccine of live attenuated Salmonella modified to express<br>
Plasmodium berghei circumsporozite antigen has been shown to protect mice against<br>
malaria [Aggarwal et al., J Exp Med 172(4): 1083 (1990)]. Similarly, U.S. Patent No.<br>
6,051,237 describes a live recombinant form of Listeria monocytogenes that grows and<br>
spreads and expresses a tumor-specific antigen for use as a cancer vaccine. While such<br>
recombinant vaccines may be effective, each microbe strain must be genetically modified<br>
to provide the vaccine. It would therefore be desirable to develop a method of producing a<br>
safe and effective microbial vaccine that can be applied to any microbe, whether or not the<br>
microbe comprises recombinant antigens. Dendritic cell (DC)-based immunotherapy has<br>
been widely investigated and demonstrated to provide a clinical benefit for the treatment of<br>
a. wide range of tumor types. A variety of strategies are presently being developed to<br>
isolate and generate autologous dendritic cells (DC), and subsequently load them with<br>
antigen or peptides ex vivo prior to patient vaccination. Recent advances in the<br>
understanding of immune mechanisms have, in addition to efficient antigen loading,<br>
highlighted the importance of the activation and maturation state of DC used for<br>
vaccination on the efficacy of cancer immunotherapy. Whereas immature DC are more<br>
effective in the uptake and processing of antigen, activated/mature DC lose this capacity,<br>
yet are more potent at presenting antigen to naive T lymphocytes in the context of MHC<br>
molecules. In fact, mature DCs have been found to be potent antigen presenting cells<br><br>
(APC) to induce primary T lymphocyte responses, overcoming peripheral T cell tolerance<br>
and enhance anti-tumor immunity. Despite the development of a variety of methods to<br>
load and to stimulate the activation and maturation of DC mat has led to encouraging<br>
clinical data, there still are not standard efficient and cost effective methods for combining<br>
antigen loading with DC activation and maturation.<br>
SUMMARY OF THE INVENTION<br>
[0004] The invention involves a free-living microbe, in which the proliferation of<br>
the microbe is attenuated while maintaining sufficient microbial gene expression, wherein<br>
the attenuation can be controlled in a dose-dependent manner. The invention includes<br>
methods for this attenuation of the free-living microbe. The invention includes vaccine<br>
compositions comprising these attenuated microbes. The present invention also provides<br>
novel uses of modified microbes, and attenuated Listeria, in particular, to load and to<br>
induce the activation and maturation of antigen-presenting cells, such as dendritic cells, in<br>
vitro or ex vivo. The resulting antigen-presenting cells are useful in vaccines and<br>
immunotherapy. Ia particular embodiments, the provided vaccines and immunotherapy are<br>
directed against cancer.<br>
[0005] In one aspect, the invention provides a vaccine comprising a free-living<br>
microbe, wherein the nucleic acid of the microbe (e.g., genomic nucleic acid) is modified<br>
so that the microbe is attenuated for proliferation. In some embodiments, the attenuation of<br>
the proliferation of the microbe is controllable in a dose-dependent manner. In some<br>
embodiments, microbial gene expression in the microbe is substantially unaffected by<br>
attenuation of the proliferation of the microbe. In some embodiments, the microbe in the<br>
vaccine expresses an antigen at a sufficient level to induce an immune response to the<br>
antigen in an individual upon administration of the vaccine to the individual. In some<br>
embodiments, the nucleic acid has been modified by reaction with a nucleic acid targeted<br>
compound (alternatively termed a nucleic acid "targeting" compound) which reacts directly<br>
with the nucleic acid. In one embodiment, the nucleic acid target compound is an alkylator<br>
such as ?-alanine, N-(acridin-9-yl), 2-[bis(2-chloroethyl)amino]ethyl ester. In other<br>
embodiments, the nucleic acid targeted compound is a psoralen compound (e.g., 4'-(4-<br>
ammo-2-oxa)butyl-4,5',8-trimethylpsoralen, also referred to herein as "S-59") activated by<br>
UVA irradiation. In some embodiments, the microbe in the vaccine comprises a genetic<br><br>
mutation that attenuates the ability of the microbe to repair its nucleic acid that has been<br>
modified. In some embodiments, the microbe is a bacterium, such as Bacillus anthracis or<br>
Listeria monocytogenes. In some embodiments, the microbe comprises a heterologous<br>
nucleic acid sequence encoding an antigen. In some embodiments, the vaccine farther<br>
comprises a pharmaceutically acceptable carrier and/or an adjuvant. The invention, further<br>
provides a method of preventing or treating a disease in a host, comprising administering to<br>
the host an effective amount of the vaccine. The invention also provides a method of<br>
inducing an immune response in a host to an antigen comprising administering to the host<br>
an effective amount of the vaccine, wherein the microbe expresses the antigen.<br>
[0006] In another aspect, the invention provides a vaccine comprising a free-living<br>
microbe (e.g., a bacterium) which is defective with respect to at least one DNA repair<br>
enzyme. In some embodiments, the free-living microbe comprises a genetic mutation in<br>
one or more gene selected from the-group consisting of phrB, uvrA, uvrB, uvrC, uvarD and<br>
recA, or in a functional equivalent of one or more gene selected from the group consisting<br>
of phrB, uvrA, uvrB, uvrC, uvrD and recA. In some embodiments, the microbe comprises<br>
genetic mutations in both uvrA and uvrB (or in functional equivalents of both uvrA and<br>
uvrB, depending upon the genus and species of the microbe). In some embodiments, the<br>
microbe is defective with respect to RecA (or the functional equivalent of RecA, depending<br>
upon the genus and species of the microbe). In some embodiments, the microbe comprises<br>
a heterologous nucleic acid sequence encoding an antigen (for instance, a cancer antigen, or<br>
an infectious disease antigen foreign to the microbe). In some embodiments, the va ccine<br>
further comprises a pharmaceutically acceptable carrier or an adjuvant. The invention<br>
further provides method of preventing or treating a disease in a host, comprising<br>
administering to the host an effective amount of the vaccine. The invention also provides a<br>
method of inducing an immune response in a host to an antigen comprising administering<br>
to the host an effective amount of the vaccine wherein the microbe expresses the antigen.<br>
[0007] In another aspect, the invention provides an isolated mutant Listeria strain,<br>
such as a mutant Listeria monocytogenes strain, comprising a genetic mutation that<br>
attenuates its ability to repair its nucleic acid. In some embodiments, the mutant Listeria<br>
strain is defective with respect to at least one DNA repair enzyme (such asUvrA and/or<br>
UvrB). In some embodiments, the mutant Listeria strain comprises a genetic mutation in<br>
the uvrA gene and/or the uvrB gene. In some embodiments, the mutant strain is the Listeria<br>
monocytogenes actA/uvrAB' strain deposited with the American Type Culture Collection<br>
(ATCC) and identified by accession number PTA-5563. In other embodiments, the strain<br><br>
is a mutant of the Usteria monocytogenes actA'luvrAB' strain deposited with the American<br>
Type Culture Collection (ATCC) and identified by accession number PTA-5563, wherein<br>
the mutant of the deposited strain is defective with respect to UvrA, UvrB., and ActA. The<br>
invention further provides vaccines and professional antigen-presenting cells comprising<br>
the mutant Listeria strain. Methods of using the modified Listeria strain to induce immune<br>
responses and to prevent or treat disease are also provided.<br>
[00081 In another aspect, the invention provides an isolated mutant Bacillus<br>
anthracis strain, comprising a genetic mutation that attenuates its ability to repair its<br>
nucleic acid. In some embodiments, the mutant strain is defective with respect to at least<br>
one DNA repair enzyme (such asUvrA and/or UvrB). In some embodiments, the mutant<br>
strain comprises a genetic mutation in the uvrA gene and/or the uvrB gene. In some<br>
embodiments, the mutant strain is attenuated with respect to RecA. In some embodiments,<br>
the mutant strain comprises a genetic mutation in the recA gene. In some embodiments, the<br>
mutant strain comprises one or more mutations in the lef gene, cya gene, ox both genes, that<br>
decreases the toxicity of the strain. The invention further provides vaccines and<br>
professional antigen-presenting cells comprising the mutant strain. Methods of using the<br>
modified Bacillus anthracis strain to induce immune responses and to prevent or treat<br>
disease are also provided<br>
[0009] In one embodiment, the invention includes a vaccine comprising bacteria<br>
that has been reacted with a psoralen compound and UVA light, wherein the proliferation<br>
of the bacteria is attenuated. In one embodiment, the bacterial expression is sufficiently<br>
active after the psoralen. modification such that the psoralen attenuated bacteria can<br>
continue to express a protein antigen, wherein when the bacteria is administered to an<br>
individual, an immune response to the antigen is elicited. In one embodiment, the desired<br>
immune response is to the bacteria itself. In one embodiment, the bacteria, are a<br>
recombinant strain that expresses a heterologous protein antigen, wherein when the bacteria<br>
is administered to an individual, an immune response to the heterologous antigen is<br>
elicited. Such a vaccine comprising a heterologous antigen may be designed to treat or<br>
prevent a variety of diseases including infectious diseases, autoimmune diseases, allergies,<br>
cancers, and other hyperproliferative diseases.<br>
[0010] For the treatment or prevention of infectious diseases, the disease causing<br>
agent may be prepared according to the methods of the invention to be used as the vaccine.<br>
In one embodiment, a vaccine can be prepared from a microbe of the invention comprising<br>
a heterologous antigen from the disease-causing agent, such as a virus, bacterium or<br><br>
parasite. Such a vaccine may provide a level of benefit when the health risk of receiving<br>
the bacterial vector is significantly less than the risks associated with possible infection, by<br>
the infectious agent. A heterologous vaccine for the treatment or prevention of infections<br>
disease that is attenuated by the methods of the present invention may have other benefits<br>
as well. First, it may not be possible to prepare an attenuated live vaccine or a killed<br>
vaccine directly from the infectious agent itself. Second, if a live vaccine is required, it<br>
may not be possible to otherwise attenuate the infectious agent and still maintain an<br>
appropriate immune response.<br>
[0011] Another possibility is that the antigen inserted into the bacterial vector does<br>
not stimulate an immune response in an individual in the absence of the innate immune<br>
response induced by the bacterial vector. For example, diseases in which autologous cells<br>
proliferate improperly may contain antigens that do not typically stimulate an immune<br>
response. It may be useful to fight such diseases by finding-a way to stimulate such an<br>
immune response against an autologous antigen. In one embodiment the proliferating cells<br>
express or over express an antigen at higher levels than on a normal cell so that the immune<br>
response is largely specific to the proliferating cells. Diseases that may be treated with<br>
such a vaccine include, but are not limited to, autoimmune diseases, allergies, cancers and<br>
other hyperproliferative cellular diseases. In another embodiment, the vaccine may target a<br>
product of the disease or a disease related target rather than the diseased cells themselves.<br>
For example, tumors may be treated with a vaccine targeting vascular endothelial growth<br>
factor (VEGF), which is essential for generation of new blood vessels required to feed<br>
tumor cells. The VEGF is peripheral to the tumor cells themselves but is prevalent in areas<br>
of tumor growth and is a viable vaccine target that could potentially limit the growth of the<br>
tumor cells. Another example is a vaccine that comprises an antigen that will elicit a<br>
response to a disease related protein, such as the proteins that cause the amyloid plaques<br>
characteristic of Alzheimer's disease or Creutzfeldt-Jakob disease. Similarly, the vaccine<br>
may target proteins involved in autoimmune or allergic responses. The vaccine may<br>
comprise an idiotype antigen that can elicit a response to the specific antibodies or cells,<br>
such as 3-cells or T cells, causing an autoimmune or allergic response.<br>
[0012] In one embodiment, the invention includes a vaccine composition<br>
comprising a free-living microbial population in which the microbial nucleic acid is<br>
modified, so that the proliferation of the microbial population is attenuated, wherein the<br>
microbial gene expression is substantially unaffected. In one embodiment, the microbial<br>
gene expression is substantially unaffected so that an antigen is expressed at a level<br><br>
sufficient to stimulate an immune response upon administration of the microbial population<br>
to an individual, In one embodiment, the proliferation of the microbial population is<br>
attenuated by at least about 0.3 log, also at least about 1 log, about 2 log, about 3 log, about<br>
4 log, about 6 log, or at least about 8 log. In another embodiment, the proliferation of the<br>
microbial population is attenuated by about 0.3 to &gt; 10 log, about 2 to &gt; 10 log, about 4 to<br>
&gt;10 log, about 6 to &gt;10 log, about 0.3-8 log, about 0.3-6 log, about 0.3-5 log, about 1-5 log,<br>
or about 2-5 log. In one embodiment, the expression of an antigen by the microbial<br>
population is at least about 10%, about 25%, about 50%, about 75%, or at least about 90%<br>
of the expression of the antigen by a microbial population in which the microbial nucleic<br>
acid is not modified. In one embodiment, the antigen expressed is an antigen, from the<br>
microbe itself. In one embodiment, the microbe comprises a heterologous nucleic acid<br>
sequence encoding an antigen. In one embodiment, the antigen is a disease associated<br>
antigen. In one embodiment, the antigen is associated with a disease selected-from the<br>
group consisting of infectious diseases, autoimmune diseases, allergies, cancers, and other<br>
hyperproliferative diseases. In one embodiment, the antigen is a tumor associated antigen.<br>
In one embodiment, the tumor antigen is selected from the group consisting of<br>
differentiation antigens, tissue-specific antigens, developmental antigens, tumor-associated<br>
viral antigens, cancer-testis antigens, embryonic antigens, oncoprotein antigens, over-<br>
expressed protein antigens and mutated protein antigens. In one embodiment, the tumor<br>
antigen is selected from the group consisting of mesothelin, Spl7, gp100, EphA2, PR3,<br>
PAGE-4, TARP, and SP AS-1. In one embodiment, the microbial nucleic acid is modified<br>
by a method selected from the group consisting of exposing the microbe to radiation and<br>
reacting the microbe with a nucleic acid targeted compound that causes the modification of<br>
the microbial nucleic acid. In a preferred embodiment, the microbial nucleic acid is<br>
modified by reacting the microbial population with a nucleic acid targeted compound that<br>
reacts directly with the nucleic acid. In one embodiment, the nucleic acid targeted<br>
compound is targeted to the nucleic acid by a mode selected from the group consisting of<br>
intercalation, minor groove binding, major groove binding, electrostatic binding, and<br>
sequence-specific binding. In one embodiment, the nucleic acid targeted compound<br>
comprises a nucleic acid alkylator. In a preferred embodiment, the nucleic acid targeted<br>
compound is ?-alanine, N-(acridin-9-yl), 2-[bis(2-chloroethyl)amino]ethyl ester. In one<br>
embodiment, the nucleic acid targeted compound that reacts directly with the nucleic acid<br>
reacts upon activation of the compound by irradiation, preferably by UVA irradiation. In<br><br>
one embodiment, the nucleic acid targeted compound activated by UVA irradiation is a<br>
psoralen. In a preferred embodiment, the psoralen is 4'-(4-amino-2-oxa)butyl-4,5',8-<br>
trimethylpsoralen. In one embodiment, the nucleic acid targeted compound indirectly<br>
causes the modification of the nucleic acid. In one embodiment, the nucleic acid targeted<br>
compound indirectly causes modification upon activation by irradiation, preferably by<br>
UVA irradiation. In one embodiment, the microbe comprises a genetic mutation. In one<br>
embodiment, the genetic mutation results in the attenuation of the ability of the microbe to<br>
repair microbial nucleic acid that has been modified. In one embodiment, the genetic<br>
mutation is in the gene selected from the group consisting of phrB, uvrA, uvrB, uvrC, uvrD<br>
and recA, or their functionally equivalent genes, depending on the genus and species of the<br>
microbe. In one embodiment, the genetic mutation is in more than one of the genes<br>
selected from the group consisting of phrB, uvrA, uvrB, uvrC, uvrD and recA, or their<br>
functionally equivalent genes, depending on the genus and species of the microbe. In an<br>
embodiment where the mutation is in the recA gene, whether alone or in combination with<br>
one or more other mutations, the recA mutation is a conditional mutation. In one<br>
embodiment, the genetic mutation results in the attenuation in tie activity of at least one of<br>
a DNA repair enzyme selected from the group consisting of PhrB, UvrA, UvrB, UvrC,<br>
UvrD and RecA. In one embodiment, attenuation in the activity of RecA is conditional. In<br>
a further embodiment, the microbes containing these mutations are modified by reaction<br>
with a psoralen activated by UVA irradiation. In a preferred embodiment, the psoralen is<br>
4'-4-amino-2-oxa)butyl-4,5',8-trimethylpsoralen. In one embodiment, the microbe is<br>
selected from the group consisting of bacteria, protozoa and fungi. In one embodiment, the<br>
microbe is a bacteria. In one embodiment, the bacteria is a mycobacteria. In one<br>
embodiment, the mycobacteria is Mycobacterium tuberculosis. In one embodiment, the<br>
bacteria is an intracellular bacteria. In one embodiment, the intracellular bacteria is<br>
Bacillus anthracis. In one embodiment, the intracellular bacteria is Yersinia pestis. In a<br>
preferred embodiment, the bacteria is a Listeria, preferably Listeria monocytogenes. In one<br>
embodiment, the Listeria comprises a mutation that results in the attenuation of the ability<br>
of the Listeria to invade non-phagocytic cells without significantly affecting the uptake of<br>
the Listeria by plaagocytic cells. In one embodiment, the Listeria mutation is in an<br>
internalin gene(s). In one embodiment, the Listeria mutation is in the gene selected from<br>
the group consisting of inlA, inlB, and any gene encoding an in ternalin. In one<br>
embodiment, the Listeria monocytogenes comprises a genetic mutation in both the inlA and<br>
inIB genes. In one embodiment, the Listeria comprises a mutation that results in the<br><br>
attenuation of the ability of the Listeria to escape the phagolysosome of an infected cell. In<br>
one embodiment, the Listeria mutation is in the hly gene. Jn one embodiment, the Listeina<br>
comprises a mutation that results in the attenuation of the polymerization of actin by the<br>
Listeria. In one embodiment, the Listeria mutation is in the actA gene. In one<br>
embodiment, the Listeria monocytogenes comprises more than one mutation. In a preferred<br>
embodiment, the Listeria mutations are in both the actA and inlB genes, preferably deletion<br>
mutations in both the actA and inlB genes.<br>
[0013] In one embodiment, the invention includes a vaccine comprising a microbial<br>
population in which the microbial nucleic acid is modified by reaction with a nucleic acid<br>
targeted compound that reacts directly with the nucleic acid so that the proliferation of the<br>
microbial population is attenuated, wherein the microbial gene expression is substantially<br>
unaffected, and wherein the microbe of the population comprises a heterologous nucleic<br>
acid sequence encoding a tumor antigen. In one embodiment, the microbial gene<br>
expression is substantially unaffected so that the tumor antigen is expressed at a level<br>
sufficient to stimulate an immune response upon administration of the microbe to an<br>
individual. In one embodiment, the proliferation of the microbial population is attenuated<br>
by at least about 0.3 log, also at least about 1 log, about 2 log, about 3 log, about 4 log,<br>
about 6 log, or at least about 8 log. In another embodiment, the proliferation of the<br>
microbial population is attenuated by about 0.3 to &gt; 10 log, about 2 to &gt;10 log, about 4 to<br>
&gt;10 log, about 6 to &gt;10 log, about 0.3-8 log, about 0.3-6 log, about 0.3-5 log, about 1-5 log,<br>
or about 2-5 log. In one embodiment, the expression of the tumor antigen by the microbial<br>
population is at least about 10%, about 25%, about 50%, about 75%, or at least about 90 %<br>
of the expression of the tumor antigen by a microbial population in which the microbial<br>
nucleic acid is not modified. In one embodiment, the tumor antigen is selected from the<br>
group consisting of differentiation antigens, tissue-specific antigens, developmental<br>
antigens, tumor-associated viral antigens, cancer-testis antigens, embryonic antigens,<br>
oncoprotein antigens, over-expressed protein antigens aad mutated protein antigens. In one<br>
embodiment, the tumor antigen, is selected from the group consisting of mesothelin, Spl7,<br>
gp100, EphA2, PIG, PAGE-4, TARP, and SPAS-1. In one embodiment, the nucleic acid<br>
targeted compound comprises an alkylator. In one embodiment, the alkylator is selected<br>
from the group consisting of mustards, mustard intermediates and mustard equivalents. Jn<br>
one embodiment, the nucleic acid targeted compound comprises a nucleic acid targeting<br>
group selected from the group consisting of intercalators, minor groove binders, major<br>
groove binders, electrostatic binders, and sequence-specific binders. In one embodiment,<br><br>
the nucleic acid targeted compound is ?-alanine, N-(acridin-9-yl), 2-[bis(2-<br>
chloroeutyl)amino]ethyl ester. In one embodiment, the nucleic acid targeted compound<br>
reacts directly with the nucleic acid upon activation of the compound. In one embodiment,<br>
the activation of the compound is by irradiation. In one embodiment, the irradiation is<br>
UVA irradiation. In a preferred embodiment, the nucleic acid targeted compound is a<br>
psoralen compound activated by UVA irradiation. In a preferred embodiment, the psoralen<br>
is 4'-(4-ammo-2-oxa)butyl-4,5',8-trimethylpsoralen. In one embodiment, the microbe of<br>
the population comprises a genetic mutation. In one embodiment, the genetic mutation<br>
results in the attenuation of the ability of the microbe to repair microbial nucleic acid that<br>
has been modified. In one embodiment, the genetic mutation is in the gene selected from<br>
the group consisting of phrB, uvrA, uvrB, uvrC, uvrD and recA, or their functionally<br>
equivalent genes, depending on the genus and species of the microbe. In one embodiment,<br>
the genetic mutation is in more than one of the genes selected from the group consisting of<br>
phrB, uvrA, uvrB, uvrC, uvrD and recA, or their functionally equivalent genes, depending<br>
on the genus and species of the microbe. In an embodiment where the mutation is in the<br>
recA gene, whether alone or in combination with one or more other mutations, the recA<br>
mutation is a conditional mutation. In one embodiment, the genetic mutation results in the<br>
attenuation in the activity of at least one of a DNA repair enzyme selected from the group<br>
consisting of PhrB, UvrA, UvrB, UvrC, UvrD and RecA. In one embodiment, attenuation<br>
in the activity of RecA is conditional. In a further embodiment, the microbes containing<br>
these mutations are modified by reaction with a psoralen activated by UVA irradiation. In<br>
a preferred embodiment, the psoralen is 4'-(4-amino-2-oxa)buryl-4,5',8-trimethaylpsoralen.<br>
In one embodiment, the microbe is selected from the group consisting of bacteria, protozoa<br>
and fungi. In one embodiment, the microbe is a bacteria. In one embodiment, the bacteria<br>
is an intracellular bacteria. In a preferred embodiment, the bacteria is a Listeria, preferably<br>
Listeria monocytogenes. In one embodiment, the Listeria comprises a mutation that results<br>
in the attenuation of the ability of the Listeria to invade non-phagocytic cells without<br>
significantly affecting the uptake of the Listeria by phagocytic cells. In one embodiment,<br>
the Listeria mutation is in an internalin gene(s). In one embodiment, the Listerna mutation<br>
is in the gene selected from the group consisting of inlA, inlB, and any gene encoding an<br>
internalin. In one embodiment, the Listeria monocytogenes comprises a genetic mutation<br>
in both the inlA and inlB genes. In one embodiment, the Listeria comprises a mutation<br>
that results in the attenuation of the ability of the Listeria to escape the phagolysosome of<br><br>
an infected cell. In one embodiment, the Listeria mutation is in the hly gene. In one<br>
embodiment, the Listeria comprises a mutation that results in the attenuation of the<br>
polymerization of actin by the Listeria. In one embodiment, the Listeria mutation is in the<br>
actA gene. In one embodiment, the Listeria monocytogenes comprises more than one<br>
mutation. In a preferred embodiment, the Listeria mutations are in both the actA and inlB<br>
genes, preferably deletion mutations in both the actA and inlB genes. In a preferred<br>
embodiment, the Listeria monocytogenes actA/inlB deletion mutant further comprises a<br>
deletion mutation in the uvrAB gene.<br>
[0014] In one embodiment, the invention includes a vaccine comprising a Listeria<br>
monocytogenes population in which the listerial nucleic acid is modified by reaction with a<br>
psoralen activated by UVA irradiation so that the proliferation of the listerial population is<br>
attenuated, wherein the listerial gene expression is substantially unaffected, and wherein<br>
the Listeria monocyiogenes comprises a heterologous nucleie-acid sequence encoding a<br>
tumor antigen. In a preferred embodiment, the psoralen is 4'-(4-amino-2-oxa)butyl-4,5',8-<br>
trimethylpsoralen. In one embodiment, the listerial gene expression is substantially<br>
unaffected so that the tumor antigen is expressed at a level sufficient to stimulate an<br>
immune response upon administration of the Listeria to an individual. In one embodiment,<br>
the proliferation of the listerial population is attenuated by at least about 0.3 log, also at<br>
least about 1 log, about 2 log, about 3 log, about 4 log about 6 log, or at least about 8 log.<br>
In another embodiment, the proliferation of the listerial population is attenuated by about<br>
0.3 to &gt; 10 log, about 2 to &gt;10 log, about 4 to &gt;10 log, about 6 to &gt;10 log, about 0.3-8 log,<br>
about 0.3-6 log, about 0.3-5 log, about 1-5 log, or about 2-5 log. In one embodiment, the<br>
expression of the tumor antigen by the listerial population is at least about 10%, about 25%,<br>
about 50%, about 75%, or at least about 90% of the expression of the tumor antigen by a<br>
listerial population in which the listerial nucleic acid is not modified. In one embodiment,<br>
the tumor antigen is selected from the group consisting of differentiation antigens, tissue-<br>
specific antigens, developmental antigens, tumor-associated viral antigens, cancer-testis<br>
antigens, embryonic antigens, oncoprotein antigens, over-expressed protein antigens and<br>
mutated protein antigens. In one embodiment, the tumor antigen is selected, from the group<br>
consisting of mesothelin, Spl7, gp100, EphA2, PR3, PAGE-4, TARP, and SPAS-1. In one<br>
embodiment, the Listeria monocytogenes comprises a genetic mutation. In one<br>
embodiment, the genetic mutation results in the attenuation of the ability of the Listeria<br>
monocytogenes to repair nucleic acid that has been modified. In one embodiment, the<br>
genetic mutation is in the gene selected from the group consisting of phrB, uvrA, uvrB,<br><br>
uvrC, uvrD and recA. In one embodiment, the genetic mutation is in more than one of the<br>
genes selected from the group consisting of phrB, uvrA, uvrB, uvrC, uvrD and recA. In an<br>
embodiment where the mutation is in the recA gene, whether alone or in combination with<br>
one or more other mutations, the recA mutation is a conditional mutation. In one<br>
embodiment, the genetic mutation results in the attenuation in the activity of at least one of<br>
a DNA repair enzyme selected from the group consisting of PhrB, UvrA, UvrB, UvrC,<br>
UvrD and RecA. In one embodiment; attenuation in the activity of RecA is conditional. In<br>
one embodiment, the genetic mutation results in the attenuation of the ability of the Listeria<br>
monocytogenes to invade non-phagocytic cells without significantly afifecting the uptake of<br>
the Listeria monocytogenes by phagocytic cells. In one embodiment, the genetic mutation<br>
is in an intemalin gene(s). In one embodiment, the Listeria mutation is in the gene selected<br>
from the group consisting of inlA, inlB, and any gene encoding an intemalin. In one<br>
embodiment, the Listeria monocytogenes comprises a genetic mutation, in both the inlA and<br>
inlB genes. In one embodiment, the Listeria comprises a mutation that results in the<br>
attenuation of the ability of the Listeria to escape the phagolysosome of an infected cell. In<br>
one embodiment, the Listeria mutation is in the hfy gene. In one embodiment, the genetic<br>
mutation results in the attenuation of the polymerization of actin by the Listeria. In one<br>
embodiment, the Listeria mutation is in the act A gene. In one embodiment, the Listeria<br>
monocytogenes comprises more than one mutation. In a preferred embodiment, the Listeria<br>
mutations are in both the actA and inlB genes, preferably deletion mutations in both the<br>
actA and inlB genes. La a preferred embodiment, the Listeria monocytogenes actA/inlB<br>
deletion mutant further comprises a deletion mutation in the uvrAB gene.<br>
[0015] In another, aspect, the invention provides a professional antigen-presenting<br>
cell (e.g., a dendritic cell) comprising a free-living microbe, wherein the nucleic acid of the<br>
microbe is modified so that the microbe is attenuated for proliferation. In some<br>
embodiments, the attenuation of the proliferation of the microbe is controllable in a dose-<br>
dependent manner. In some embodiments, microbial gene expression in the microbe is<br>
substantially unaffected by attenuation of the proliferation of the micro be. In some<br>
embodiments, the microbe in the vaccine expresses an antigen at a sufficient level to induce<br>
an immune response to the antigen in an individual upon administration of the vaccine to<br>
the individual. In some embodiments, the nucleic acid has been modified by reaction with<br>
a nucleic acid targeted compound which reacts directly with the nucleic acid. In one<br>
embodiment, the nucleic acid target compound is an alkylator such as ?-alanine, N-(acridin-<br><br>
9-yl), 2-[bis(2-chloroethyl)amino]ethyl ester. In other embodiments, the nucleic acid<br>
targeted compound is a psoralen compound (e.g., 4-(4-amino-2-oxa)butyl-4,5',8-<br>
trimethylpsoralen, also referred to herein as "S-59") activated by UVA irradiation. In some<br>
embodiments, the microbe in the vaccine comprises a genetic mutation that attenuates the<br>
ability of the microbe to repair its nucleic acid that has been modified. In some<br>
embodiments, the microbe is a bacterium. In some embodiments, the microbe comprises a<br>
heterologous nucleic acid sequence encoding an antigen. The invention also provides a<br>
vaccine comprising the antigen-presenting cell. The invention further provides a method of<br>
preventing or treating a disease in a host, comprising administering to the host an effective<br>
amount of the antigen-presenting cell. The invention also provides a method of inducing<br>
an immune response in a host to an antigen comprising administering to the host an<br>
effective amount of the antigen-presenting cell, wherein the microbe expresses the antigen.<br>
The invention further provides a method of activating naive T cells ex vivo and/or in vitro<br>
(not mutually exclusive), comprising contacting the narve T cells with the professional<br>
antigen-presenting cell under suitable conditions and for a sufficient time to activate the<br>
naive T-cells.<br>
[0016] In another aspect, the invention provides an isolated professional antigen-<br>
presenting cell (e.g., a dendritic cell) comprising a fiee-living microbe (e.g., a bacterium)<br>
which is defective with, respect to at least one DNA repair enzyme. In some embodiments,<br>
the microbe comprises a genetic mutation is in one or more gene selected from the group<br>
consisting of phrB, uvrA, uvrB, uvrC, uvrD and recA, ox in a functional equivalent of one<br>
or more gene selected from the group consisting of phrB, uvrA, uvrB, uvrC, uvrD and<br>
recA. For instance, the microbe may comprise genetic mutations in both uvrA and uvrB, or<br>
in functional equivalents of both uvrA and uvrB (depending on the genus and species of the<br>
microbe). In some embodiments, the antigen-presenting cell is defective with respect: to<br>
RecA, or the functional equivalent of RecA. In some embodiments, the microbe comprises<br>
a heterologous nucleic acid sequence encoding an antigen. Methods of preventing or<br>
treating a disease in a host, comprising administering to the host an effective amount of the<br>
antigen-presenting cell are also provided, as are methods of inducing an immune response<br>
in a host to an antigen comprising administering to the host an effective amount of the<br>
antigen-presenting cell, wherein the microbe expresses the antigen.<br>
[0017] In another aspect, the invention provides a method of loading professional<br>
antigen-presenting cells with an antigen comprising contacting the professional antigen-<br>
presenting cells (in vitro or in vivo) with a free-living microbe that comprises a nucleic acid<br><br>
sequence encoding the antigen, under suitable conditions and for a sufficient time to load<br>
the professional antigen-presenting cells, wherein the nucleic acid of the microbe is<br>
modified (e.g., has been modified by reaction with a nucleic acid targeted compound that<br>
reacts directly with the nucleic acid) so that the microbe is attenuated for proliferation.<br>
[0018] In still another aspect, the invention provides a method of activating and/or<br>
maturing professional antigen-presenting cells comprising contacting the professional<br>
antigen-presenting cells (in vitro or in vivo) with a free-living microbe that comprises a<br>
nucleic acid sequence encoding art antigen, under suitable conditions and for a sufficient<br>
time to load the professional antigen-presenting cells, wherein the nucleic acid of the<br>
microbe is modified (e.g., has been modified by reaction with a nucleic acid targeted<br>
compound that reacts directly with, the nucleic acid) so that the microbe is attenuated for<br>
proliferation.<br>
[0019] In yet another aspect, the invention provides a method of preventing or<br>
treating a disease in a host, comprising the following steps, (a) loading professional<br>
antigen-presenting cells with an antigen by contacting the cells with a free-living microbe<br>
that comprises a nucleic acid sequence encoding an antigen, wherein the nucleic acid of the<br>
microbe is modified so that the microbe is attenuated for proliferation; and (b)<br>
administering an effective amount of a composition comprising the loaded professional<br>
antigen-presenting cells to the host. In some embodiments, the microbe has been, modified<br>
by reaction with a nucleic acid targeted compound that reacts directly with the nucleic acid.<br>
[0020] In still another aspect, the invention provides a method of loading<br>
professional antigen-presenting cells, such as dendritic cells, with an antigen(s), comprising<br>
contacting the cells in vitro or ex vivo with a modified microbe expressing the antigen,<br>
under suitable conditions and for a time sufficient to load the antigen-presenting cells. In<br>
some embodiments, proliferation of the microbe is attenuated. In some embodiments, the<br>
microbe maintains sufficient gene expression to effect antigen presentation by the cell,<br>
even though proliferation of the microbe is attenuated. The antigen presentation can be<br>
MHC class I presentation or MHC class II presentation.<br>
[0021] In another aspect, the invention provides a method of activating and/or<br>
maturing antigen-presenting cells (for instance, dendritic cells) comprising contacting the<br>
antigen-presenting cells in vitro or ex vivo with a modified microbe under suitable<br>
conditions and for a time sufficient to activate the dendritic cells and/or to allow the<br>
antigen-presenting cells to mature. In one embodiment, proliferation of the microbe is<br>
attenuated. In another embodiment, the microbe maintains sufficient gene expression to<br><br>
effect activation and/or maturation of the cell, even though proliferation of the microbe is<br>
attenuated.<br>
[0022] In still another aspect, the invention provides a method of inducing an<br>
immune response to an antigen, comprising administering to the host an effective amount<br>
of an immunogenic composition comprising an antigen presenting cell (such as a dendritic<br>
cell) presenting the antigen, wherein the antigen-presenting cell comprises a modified<br>
microbe. In one embodiment, proliferation of the microbe has been attenuated. In another<br>
embodiment, the microbe maintains sufficient gene expression to effect antigen<br>
presentation by the cell, even though proliferation of the microbe is attenuated. In one<br>
embodiment, the immune response is a CD8+ T-cell response. In another embodiment, the<br>
immune response is a CD4+ T-cell response.<br>
[0023] In yet another aspect, the invention provides a method of inducing an<br>
immune response to an antigen, comprising the following steps: (a) contacting antigen-<br>
presenting cells (such as dendritic cells) in vitro or ex vivo with Listeria expressing the<br>
antigen under suitable conditions and for a time sufficient to load the antigen-presenting<br>
cells with the antigen and to effect activation and/or maturation of the antigen-presenting<br>
cells; and (b) administering an effective amount of the antigen-presenting cells to the host.<br>
In one embodiment, proliferation of the microbe is attenuated. In another embodiment, the<br>
microbe which is contacted with the antigen-presenting cells maintains sufficient gene<br>
expression to effect both presentation of the antigen on the antigen-presenting cell and<br>
activation and/or maturation, of the antigen-presenting cell, even though proliferation of the<br>
microbe is attenuated. In one embodiment, the immune response is a CD8+ T-cell<br>
response. In another embodiment, the immune response is a CD4+ T-cell response.<br>
[0024] In another aspect, the invention provides an ex vivo or in vitro professional<br>
antigen-presenting cell comprising a modified microbe, wherein proliferation of the<br>
microbe is attenuated. In another embodiment, the modified microbe maintains sufficient<br>
gene expression to effect antigen presentation by the dendritic cell, even though,<br>
proliferation of the microbe is attenuated. In one embodiment, the antigen-presenting cell<br>
is a dendritic cell.<br>
[0025] In yet another aspect, the invention provides a vaccine comprising an<br>
antigen-presenting cell (such, as a dendritic cell), wherein the antigen-presenting cell<br>
comprises a modified microbe. In one embodiment, the microbe is Listeria. In one<br>
embodiment, proliferation of the Listeria has been attenuated. In another embodiment, the<br><br>
Listeria maintains sufficient gene expression to effect antigen presentation on the cell, even<br>
though proliferation of the Listeria is attempted.<br>
[0026] In a still further aspect, the invention provides a pharmaceutical composition<br>
comprising an antigen-presenting cell (such as a dendritic cell) and a pharmaceutically<br>
acceptable carrier, wherein the antigen-presenting cell comprises modified Listeria. In one<br>
embodiment, proliferation of the Listeria has been attenuated. In another embodiment, the<br>
Listeria maintains sufficient gene expression to effect antigen presentation by the cell, even<br>
though proliferation of the Listeria is attenuated.<br>
[0027] In some embodiments of each of the aforementioned aspects, the modified<br>
microbe is a modified Listeria. In additional embodiments of each of the aforementioned<br>
aspects, the Listeria is Listeria monocytogenes. In still further embodiments, the Listeria<br>
comprises a mutation in one or more genes selected from the group consisting of phrB,<br>
uvrA, uvrB, uvrC, uvrD, and recA. For instance, in any of the aforementioned aspects, the<br>
Listeria optionally comprises a mutation in uvrAB. In alternative embodiments, the<br>
Listeria optionally comprises both a mutation in uvrAB and actA.<br>
[0028] In other embodiments of each of the aforementioned aspects, the attenuation<br>
of the Listeria has been effected by exposure of the Listeria to a cross-linking agent. In<br>
some embodiments of each of the aforementioned aspects, the cross-linking agent is ?-<br>
alanine, N-(acridin-9-yl), 2-[bis(2-chloroethyl)amino]ethyl ester. In other embodiments of<br>
each of the aforementioned aspects, the cross-linking agent is a psoralen derivative and the<br>
Listeria is exposed to UVA light. In some embodiments of each of the aforementioned<br>
aspects, the cross-linking agent is 4'-(4-amino-2-oxa)butyl-4,5',8-trimethylpsoralen (also<br>
referred to herein as "S-59").<br>
DRAWINGS<br>
[0029] Figure 1 shows the attenuation of wild-type Listeria DP-L4056 containirxg<br>
OVA antigen as a function of psoralen S-59 concentration (2 J/cm2 UVA) along with the<br>
measurement of OVA antigen presentation to a dendritic cell line. The bacterial log titer<br>
and % of antigen presented relative to untreated (data is for 100 Listeria per DC 2.4 cell)<br>
are plotted vs. nM S-59.<br>
[0030] Figure 2 shows the attenuation of wild-type Listeria DP-L4056 (2A) and<br>
LLO- mutant DP-L4027 (2B) containing OVA antigen as a function of alkylator compound<br><br>
I concentration along with the measurement of OVA antigen presentation to a dendritic cell<br>
line. The bacterial log titer and % of antigen presented relative to untreated (data is for 1<br>
Listeria per DC 2.4 cell) are plotted vs. uM compound I.<br>
[0031] Figure 3 shows a comparison of the inactivation of wild type E. coli to<br>
repair deficient mutant CSR 603 (uvrA recA phr mutant) as a function of S-59<br>
concentration (2 J/cm2 UVA). The bacterial log titer is plotted vs. nM S-59 (log scale).<br>
[0032] Figure 4 shows the mean tumor volunxe as a function of days post implant of<br>
B16 OVA tumors into C57B1/6 mice that are vaccinated at days 3, 7, and 14. The vaccines<br>
tested are with and without S-59 treatment.<br>
[0033] Figure 5 shows the percent survival as a function of days post implant of<br>
B16 OVA tumors into C57B1/6 mice that are vaccinated at days 3, 7, and 14. The vaccines<br>
tested are with and without S-59 treatment.<br>
[0034] Figure 6 shows flow cytometry results showing a population of spleen cells<br>
that are TNF-? and IFN-? positive from mice vaccinated with wild type Listeria with and<br>
without OVA expression, with and without S-59 UVA treatment (PCT). Figure 6A shows<br>
the population of cells specific for LLO190-210. Figure 6B shows the population of cells<br>
specific for OVA.<br>
[0035] Figure 7 shows ELISPOT results showing the number of EFN-? spot forming<br>
cells per 2 x 105 spleen cells upon stimulation with either SL8, LLO190-201, or LLO296-304,<br>
from mice vaccinated with the indicated wild type Listeria strains with or without S-59<br>
UVA treatment (PCT).<br>
[0036] Figure 8 shows the attenuation of Listeria strains with and without deletion<br>
of uvrAB. The log titer is plotted vs. nM concentration of psoralen S-59 used (6 J/cm2)<br>
Figure 8A, strains DP-L401 V(L461T LLO mutant) and wild type (DP-L4056). Figure SB,<br>
strains DP-L4017 and DP-L4029 (AactA).<br>
[0037] Figure 9 shows the attenuation of DP-L4029 (AactA) Listeria strain<br>
containing OVA antigen as a function of psoralen S-59 concentration along with the<br>
measurement of OVA antigen presentation to a dendritic cell line. The parent strain ( in<br>
this case, AactA; 9A, 9C) is compared to the strain with a uvrAB deletion (AuvrAB; 9B,<br>
9D). The bacterial log titer and % of antigen presentation relative to untreated are plotted<br>
vs. nM S-59. Figures 9A, 9B, dosed with 0.5 J/cm2UVA, washed Listeria once, dosed<br>
again with 5.5 J/cm2 UVA, antigen presentation measured at 1 Listeria per DC 2.4 cell-<br>
Figures 9C, 9D, Listeria was grown in the presence of S-59, then dosed with 6 J/cm2 UVA,<br><br>
antigen presentation measured at 10 Listeria per DC 2.4 cell. (Expanded plots of the data<br>
are also provided in Figures 9C and 9D.)<br>
[0038] Figure 10 shows polyacrylamide gels of 35S methionine/cysteine<br>
incorporated into protein synthesized by S-59/UVA treated Listeria monocytogenes strains<br>
DP-L4029 (AactA) and DP-L4029 uvrAB (AactAAuvrAB).<br>
[0039] Figure 11 shows the ELISPOT assay for spleen cells from anice vaccinated<br>
with 59/UVA treated (two methods) Listeria monocytogenes strains DT-LA029(AactA)-<br>
OVA or AactAAuvrAB-OVA, stimulated with OVA specific antigen SL8, LLO specific<br>
antigens LLO 190 and LLO 296. Figure 11A shows spot forming colonies on plates<br>
stimulated with OVA specific antigen, Figure 11B plots the IFN-? spot forming cells per 2<br>
x 105 spleen cells for all three antigens.<br>
[0040] Figure 12 shows the Intracellular Cytokine Staining (ICS) assay for spleen<br>
cells from mice vaccinated with S-59/UVA treated (two methods) Listeria monocytogenes<br>
strains VP-IA029(AactA)-OVA or AactAAwrAJS-OVA, stimulated with OVA derived T<br>
cell epitope SL8 (12A), LLO specific class II antigen LLO190-201 (12B), or LLO specific<br>
class I antigen LLO296-304 (12C). The S-59/UVA. treated Listeria are marked "PCT" (stands<br>
for photochemical treatment) in the figure.<br>
[0041] Figure 13 shows the number of colony forming units isolated per spleen<br>
(13A) or liver (13B) from mice vaccinated with S-59/UVA treated (two methods) Listeria<br>
monocytogenes strains GP-IA029(AactA) or AactAAuvrAB and challenged, with wild type<br>
Listeria monocytogenes thirty days after vaccination.<br>
[0042] Figure 14 shows the number of colony forming units isolated per spleen<br>
(14A) or liver (14B) from mice vaccinated with S-59/UVA treated (grown, with psoralen,<br>
then UVA treated) Listeria monocytogenes strains DP-LA029(AactA) or DIP-L4029<br>
AactAAuvrAB (1x, 3x, or 5x vaccination) and challenged with wild type Listeria<br>
monocytogenes thirty days after vaccination.<br>
[0043] Figure 15 shows the antibody titer of Listeria specific antibodies from serum<br>
of mice vaccinated with S-59/UVA treated (grown with psoralen, then UVA treated)<br>
Listeria monocytogenes strains DP-LA029(AactA) or AactAAuvrAB (lx, 3x, or 5x<br>
vaccination).<br>
[0044] Figure 16 shows the percent survival (10 days post challenge) of mice<br>
vaccinated with S-5 9/UVA treated (grown with psoralen, then UVA treated) Listeria<br>
monocytogenes strains DP-L4029(AactA) or AactAAuvrAB (1x, 3x, or 5x vaccination) and<br><br>
challenged with 20 x LD50 or 100 x LD50 wild type Listeria monocytogenes thirty days<br>
after vaccination.<br>
[0045] Figure 17 shows the results of an ICS assay for spleen cells from mice<br>
vaccinated with S-59/UVA treated (grown with psoralen, then UVA treated) Listeria<br>
monocytogenes strains DP-L4029(AocA)-OVA AH1A5 or AactAAuvrAB-OVA AH1A5,<br>
stimulated with antigens LLO91, AH1, AH1A5, or cells P815 or CT2 6 cells.<br>
[0046] Figure 18 shows the results of an ELISPOT assay showing plates with spot<br>
forming colonies for spleen cells from mice vaccinated with S-59/UVA treated (grown with.<br>
psoralen, then UVA treated) Listeria monocytogenes strains DP-L402 9(AaclA)-OVA<br>
AH1A5 or AactAAuvrAB-OVA AH1A5, stimulated with. AH1A5 (18A) or AH1 (18B)<br>
antigen.<br>
[0047] Figure 19 shows lungs from mice with established CT26 lung tumors given<br>
a therapeutic vaccination with S-59/UVA treated DP-L4029, with or without a AuvrAB<br>
mutation (19A). The number of lung metastases are plotted for each vaccine strain (19B).<br>
The survival of the remaining mice is plotted in Figure 19C.<br>
[0048] Figure 20 shows mice with established CT26 tumors were given therapeutic<br>
vaccination with Listeria monocytogenes AactA, AactA AH1-A5, AactAAuvrAB AH1-A5<br>
and AcictAAinlB AH1-A5. The AuvrAB strain was either no treatment, heat-killed (HK) or<br>
S-59 UVA (PCT) treated. The lungs harvested from a subset of the mice are shown in<br>
Figure 20A, with the number of lung metastases in each group plotted in Figure 20B.<br>
Survival of the remaining mice is plotted in Figure 20C (parent strain) and 20D (AuvrAB<br>
strain).<br>
[0049] Figure 21A shows fluorescent microscopy images of DC 2.4 cells infected<br>
by wild type Listeria monocytogenes uvrzAB mutant that has been S-59/UVA treated,<br>
showing merged image (both Listeria and actin positive) and Rhodamine image (only actin<br>
positive). Figure 21B is a plot of the percentage of the Listeria monocytogenes that is in<br>
the cytoplasm for wild type and AuvrAB strains (live, heat-killed or S-59 UVA treated)<br>
compared to LLO.<br>
[0050] Figure 22 shows a negative image photomicrograph of Gram stained<br>
Listeria monocytogenes wild-type and AuvrAB strains that have been S-59/UVA treated.<br>
[0051] Figure 23 shows the target cell populations following injection into mice<br>
vaccinated with the indicated Listeria strains or vehicle control. The reduced levels of<br>
antigen-specific target cells relative to non-specific target cells indicates in vivo<br><br>
cytotoxicity of T cells in response to the vaccination. Figure 23 A shows results for AH1-<br>
A5 expressing vaccines with, vaccination at days 0 (also 1 and 2 for S-59 UVA treated<br>
strains). (The top row in 23 A and 23B shows results for mice vaccinated with the indicated<br>
vaccines for AH1 target cells. The bottom row shows results for mice vaccinated with the<br>
indicated vaccines for AH1-A5 target cells.) Figure 23B has a repeat vaccination at day 14-<br>
(15 and 16 for S-59 UVA treated) and Figure 23C looks at an CTVA specific response.<br>
[0052] Figure 24 shows the attenuation of Bacillus anthracis Sterne strain with and<br>
without deletion of uvrAB. The log titer is plotted vs. nM concentration of psoralen S-59<br>
present during growth and UVA irradiation (6 J/cm2).<br>
[0053] Figure 25 shows Listeria uvrAB are more susceptible to S-59/UVA light<br>
inactivation. Listeria were grown to mid-log phase, washed in PBS, incubated for 5 min<br>
with varying concentrations of S-59 and illuminated at 2.1 J/cm2 of UVA light The<br>
viability of Listeria was assessed by growth on BHI agar plates. (A) Representative BHI<br>
agar plates of Listeria treated at 100 nM S-59. Heat-killed Listeria served as control; (B)<br>
Viability of Listeria treated at varying concentrations of S-59 to form colonies on BHI agar<br>
plates.<br>
[0054] Figure 26 shows that S-59/UVA treated, non-viable Listeria uvrAB retain<br>
their metabolic activity and the expression of their genomic repertoire. (A) Metabolic<br>
activity determined in a MTT assay of S-59/UVA inactivated Listeria urvAB. Live and<br>
heat-killed Listeria uvrAB served as control; (B) Quantification of the metabolic activity of<br>
inactivated Listeria uvrAB strain determined in a MTT assay.<br>
[0055] Figure 27 shows that fully inactivated Listeria uvrAB retain their capacity to<br>
infect DC and to escape from the phagolysosome. The murine DC line, DC2.4, grown on<br>
coverslips was infected at an MOI of 1 for 30 min at 37°C. Extracellular bacteria were<br>
carefully removed by several washes and infected cells were incubated for 5 hrs at 37°C in<br>
the presence of gentamicin to prevent growth of extracellular bacteria. DC2.4 cells were<br>
fixed with 3.5% formaldehyde and then stained with rabbit anti-Zisteria antibody, detected<br>
with a goat-anti-rabbit FITC secondary antibody. Actin was detected with Phalloidin-<br>
rhodamine and the nucleus was visualized using DAPI.<br>
[0056] Figure 28 shows that fully inactivated Listeria uvr-AB efficiently load<br>
antigen into the MHC class I pathway of murine bone marrow-derived DC (BM-DC). Day<br>
5 BM-DC were infected with, a MOI of 100 for 30 min at 37°C. Extracellular bacteria were<br>
removed by several washes. Infected BM-DC were co-incubated with B3Z overnight and<br>
activation was determined by hydrolysis of the chromogenic substrate CPRG (absorbance).<br><br>
[0057] Figure 29 shows that Listeria infected human immature monocyte-derived<br>
DC upregulate activation (29A) and maturation markers (29B) as well as secrete pro-<br>
inflammatory cytokines (29C). DCs were infected with Listeria at different MOI for 1<br>
hour. Infected DCs were cultured for additional 24 hours in the presence of gentamicin to<br>
prevent the growth of extracellular bacteria. Pheanotypic changes were determined by flow<br>
cytometry. Cytokine levels were determined from cell supernatants using the Cytometric<br>
bead array kit (Pharmingen).<br>
[0058] Figure 30 shows that S-59/UVA inactivated Listeria uvrAB OVA induce<br>
OVA-specific immunity in vivo. Female C57BL/6 mice were administered intravenously<br>
with 1x108 CFU of S-59/UVA inactivated Listeria uvrAB OVA. The S-59/UVA<br>
inactivated parent Listeria strain and heat-killed Zisteria served as control. Seven days<br>
later, spleens were harvested and OVA-specific CD8+ T cell responses were assessed by<br>
IFlSPy ELISPOT. (A) Representative ELISPOT wells are shown; (B) OVA-specific<br>
immunity assessed by ELISPOT. Spleen cells of vaccinated mice were cultured with or<br>
without OVA25 7-264 peptide.<br>
[0059] Figure 31 shows the primary amino acid sequence of the heterologous<br>
antigen LLO-OVA/PR3 (SEQ ID NO:48). The Sgure also shows the OVA H-2 Kb epitope<br>
(SEQ ID NO:49) and the PR3 HLA A-2 restricted class I epitope (a.k.a. PR1) (SEQ ID<br>
NO:50).<br>
[0060] Figure 32 shows the compound 4'-(4-amino-2-oxa)butyl-4,5',8-<br>
trimethylpsoralen (S-59).<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
[0061] The present invention involves modified free-living microbes and the use of<br>
modified free-living microbes in vaccine compositions, wherein the nucleic acid of the<br>
microbe is modified so that proliferation of the microbe is attenuated. In some<br>
embodiments, th.e microbial gene expression is substantially unaffected by the<br>
modification. The present invention also involves the use of the modified microbes for<br>
antigen loading and induction of the activation/maturation of antigen presenting cells<br>
(APCs), in vitro or ex vivo. The antigen may be either an antigen produced naturally by the<br>
modified microbe, or may be a heterologous antigen expressed by a recombinant microbe.<br>
The resulting antigen presenting cells are suitable for use in vaccine compositions and for<br><br>
immunotherapy. The immune response stimulated by administration of the resulting<br>
vaccine compositions may be a CD4+ or a CD8+ immune response.<br>
[0062] One such modified microbe is Listeria monocytogenes. The inventors have<br>
engineered Listeria to be particularly sensitive to inactivation by psoralens, a group of<br>
compounds that form irreversible cross-links in the genomes of bacteria after illumination<br>
with ultraviolet A (UVA) light, so that they are non-viable. (See Example 3, below.) The<br>
attenuation of proliferation of wild-type and modified Listeria while maintaining<br>
expression of model antigens has now been shown (see Example 1-2 and 11, below). The<br>
modified Listeria is also shown to provide an anti-tumor response (Examples 4 and 14-16,<br>
below) and induce antigen-specific T-cell responses (Example 5) and in vivo cytotoxic<br>
responses (Example 20). Listeria is rapidly phagocytosed by DC and transported into the<br>
phagolysosomal compartment. This encounter results in the phenotypic maturation of the<br>
DC and subsequent secretion of a broad profile of immunostimulatory cytolrines, including<br>
IFN-?, IL-12, and TNF-?. The inventors have now demonstrated that infection of<br>
immature DC with recombinant Listeria results in rapid DC activation/maturation, together<br>
with. MHC class I-restricted presentation of an encoded heterologous antigen.<br>
Additionally, degradation of Listeria vaccines within the phagolysosome results in<br>
presentation of encoded antigen via the MHC class It pathway. (See Examples, below)<br>
[0063] Another such modified microbe is Bacillus anthracis. The inventors have<br>
also engineered attenuated strain of Bacillus anthracis which are particularly sensitive to<br>
inactivation by psoralens (see Example 21, below).<br>
[0064] Accordingly, the invention provides a vaccine comprising a free-living<br>
microbe, wherein the nucleic acid, of the microbe is modified so that the microbe is<br>
attenuated for proliferation. In some embodiments, the attenuation of the proliferation of<br>
the microbe is controllable in a dose-dependent manner. In some embodiments, microbial<br>
gene expression in the microbe is substantially unaffected by attenuation of the<br>
proliferation of the microbe. In some embodiments, the microbe in the vaccine expresses<br>
an antigen at a sufficient level to induce an immune response to the antigen in an individual<br>
upon administration of the vaccine to the individual. In some embodiments, the nucleic<br>
acid has been modified by reaction with a nucleic acid targeted compound which reacts<br>
directly with the nucleic acid. In one embodiment, the nucleic acid target compound is an<br>
alkylator such as p-alanine, N-(acridin-9-yl), 2-[bis(2-chloroethyl)amino]ethyl ester. In<br>
other embodiments, the nucleic acid targeted compound is a psoralen compound (e.g., 4'-<br><br>
(4-amino-2-oxa)butyl-4,5',8-trimethylpsoralen, also referred to herein as "S-59") activated<br>
by UVA irradiation. In some embodiments, the microbe in the vaccine comprises a genetic<br>
mutation that attenuates the ability of the microbe to repair its nucleic acid that has been<br>
modified. In some embodiments, the microbe is a bacterium, such as Bacillus anthracis or<br>
Listeria monocytogenes. In some embodiments, the microbe comprises a heterologous<br>
nucleic acid sequence encoding an antigen. In some embodiments, the vaccine further<br>
comprises a pharmaceutically acceptable carrier and/or an adjuvant The invention farther<br>
provides a method of preventing or treating a disease in a host, comprising administering to<br>
the host an effective amount of the vaccine. The invention also provides a method of<br>
inducing an immune response in a host to an antigen comprising administering to the host<br>
an effective amount of the vaccine, wherein the microbe expresses the antigen.<br>
[0065] The invention also provides an isolated mutant Listeria strain, such as a<br>
mutant Listeria monocytogenes strain, comprising a genetic mutation that attenuates its-<br>
ability to repair its nucleic acid. In some embodiments, the mutant Listeria strain is<br>
defective with respect to at least one DNA repair enzyme (such as UvrA and/or UvrB). In<br>
some embodiments, the mutant Listeria strain comprises a genetic mutation in the uvrA<br>
gene and/or the uvrB gene. In some embodiments, the mutant strain is the Listeria<br>
monocytogenes AactA/AuvrAB strain deposited with the American Type Culture Collection<br>
(ATCC) and identified by accession number PTA-5563. In other embodiments, the strain<br>
is a mutant of the Listeria monocytogenes AactA/AuvrAB strain deposited with the<br>
American Type Culture Collection (ATCC) and identified by accession number PTA-5563,<br>
wherein the mutant of the deposited strain is defective with respect to UvrA, UvrB, and<br>
ActA. The invention further provides vaccines and professional antigen-presenting cells<br>
comprising the mutant Listeria strain. Methods of using the modified Listeria strain to<br>
induce immune responses and to prevent or treat disease are also provided.<br>
[0066] The invention provides an isolated mutant Bacillus anthracis strain,<br>
comprising a genetic mutation that attenuates its ability to repair its nucleic acid. In some<br>
embodiments, the mutant strain is defective with respect to at least one DNA repair enzyme<br>
(such asUvrA and/or UvrB). In some embodiments, the mutant strain comprises a genetic<br>
mutation in the uvrA gene and/or the uvrB gene. In some embodiments, the mutant strain is<br>
attenuated with respect to RecA. m some embodiments, the mutant strain comprises a<br>
genetic mutation in the recA gene. In some embodiments, the mutant strain comprises one<br>
or more mutations in the lef gene, cya gene, or both genes, that decreases the toxicity of the<br><br>
strairn. The invention further provides vaccines and professional antigen-presenting cells<br>
comprising the mutant strain. Methods of using the modified Bacillus anthracis strain to<br>
induce immune responses and to prevent or treat disease are also provided.<br>
[0067] In addition, the invention provides a professional antigen-presenting cell<br>
(e.g., a dendritic cell) comprising a free-living microbe, wherein the nucleic acid of the<br>
microbe is modified so that the microbe is attenuated for proliferation. In some<br>
embodiments, the attenuation of the proliferation of the microbe is controllable in a dose-<br>
dependent manner. In some embodiments, microbial gene expression in the microbe is<br>
substantially unaffected by attenuation of" the proliferation of the microbe. In some<br>
embodiments, the microbe in the vaccine expresses an antigen at a sufficient level to induce<br>
an immune response to the antigen in an individual upon administration of the vaccine to<br>
the individual. In some embodiments, the nucleic acid has been modified by reaction with<br>
a nucleic acid targeted compound which reacts directly with the nucleic acid. In one<br>
embodiment,the nucleic acid target compound is an alkylator such as ?-alanine, N-(acridin-<br>
9-yl), 2-[bis(2-chloroethyl)amino]ethyl ester. In other embodiments, the nucleic acid<br>
targeted compound is a psoralen compound activated by UVA irradiation. In some<br>
embodiments, the microbe in the vaccine comprises a genetic mutation that attenuates the<br>
ability of the microbe to repair its nucleic acid that has been modified. In some<br>
embodiments, the microbe is a bacterium. In some embodiments, the microbe comprises a<br>
heterologous nucleic acid sequence encoding an antigen. The invention also provides a<br>
vaccine comprising the antigen-presenting cell. The invention further provides a method of<br>
preventing or treating a disease in a host, comprising administering to the host an effective<br>
amount of the antigen-presenting cell. The invention also provides a method of inducing<br>
an immune response in a host to an antigen comprising administering to the host an<br>
effective amount of the antigen-presenting cell, wherein the microbe expresses the antigen.<br>
The invention further provides a method of activating naive T cells ex vivo or in vitro,<br>
comprising contacting the naive T cells with the professional antigen-presenting cell under<br>
suitable conditions and for a sufficient time to activate the naive T-cells.<br>
[0068] The invention provides a method of loading professional antigen-presenting<br>
cells with an antigen comprising contacting the professional antigen-presenting cells with a<br>
free-living microbe that comprises a nucleic acid sequence encoding the antigen, under<br>
suitable conditions and for a sufficient time to load the professional antigen-presenting<br><br>
cells, wherein the nucleic acid of the microbe is modified so that the microbe is attenuated<br>
for proliferation.<br>
[0069] The invention also provides a method of activating and/or maturing<br>
professional antigen-presenting cells comprising contacting the professional antigen-<br>
presenting cells with, a free-living microbe that comprises a nucleic acid sequence encoding<br>
an antigen, under suitable conditions and for a sufficient time to load the professional<br>
antigen-presenting cells, wherein the nucleic acid of the microbe is modified so that the<br>
microbe is attenuated, for proliferation.<br>
[0070] The invention further provides a method of preventing or treating a disease<br>
in a host, comprising the following steps, (a) loading professional antigen-presenting cells<br>
with an antigen by contacting the cells with a free-living microbe that comprises a nucleic<br>
acid sequence encoding an antigen, wherein the nucleic acid of the microbe is modified so<br>
that the microbe is attenuated for proliferation, and (b) administering an effective amount<br>
of a composition comprising the loaded professional antigen-presenting cells to the frost<br>
[0071] The invention also provides a method of loading antigen-presenting cells,<br>
such as dendritic cells, with an antigen, comprising contacting the cells in vitro or ex vivo<br>
with a modified microbe expressing the antigen, under suitable conditions and for a time<br>
sufficient to load the antigen-presenting cells.<br>
[0072] The invention provides a method of activating and/or maturing antigen-<br>
presenting cells comprising contacting the antigen-presenting cells in vitro or ex vivo with a<br>
modified microbe under suitable conditions and for a time sufficient to effect activation<br>
and/or maturation of the dendritic cells and/or to allow the antigen-presenting cells to<br>
mature.<br>
[0073] The invention provides a method of inducing an immune response to an<br>
antigen, comprising administering to the host an effective amount of an immunogenic<br>
composition comprising an antigen presenting cell presenting the antigen, wherein the<br>
antigen-presenting cell comprises a modified microbe.<br>
[0074] In addition, the invention provides a method of inducing an immune<br>
response to an antigen, comprising the following steps: (a) contacting antigen-presenting<br>
cells in vitro or ex vivo with Listeria expressing the antigen under suitable conditions and<br>
for a time sufficient to load the antigen-presenting cells with the antigen and to effect<br>
activation and/or maturation of the antigen-presenting cells; and (b) administering an.<br>
effective amount of the antigen-presenting cells to the host In one embodiment,<br>
proliferation of the microbe is attenuated.<br><br>
[0075] The invention also provides an ex vivo or in vitro professional antigen-<br>
presenting cell comprising a modified microbe, wherein proliferatioa of the microbe is<br>
attenuated.<br>
[0076] Additionally, the invention provides a vaccine comprising an antigen-<br>
presenting cell, wherein the antigen-presenting cell comprises a modified microbe and a<br>
pharmaceutical composition comprising a antigen-presenting cell and a pharmaceutically<br>
acceptable carrier, wherein the antigen-presenting cell comprises Listeria.<br>
Microbe-Based Vaccines<br>
[0077] The present invention involves modified free-living microbes and the use of<br>
modified free-living microbes in a vaccine composition, wherein the nucleic acid of the<br>
microbe is modified so that proliferation of the microbe is attenuated. In some<br>
embodiments, the microbial gene expression is substantially unaffected by the<br>
modification.<br>
[0078] It has been observed that killed microbial vaccines are often inferior to live<br>
attenuated rnicrobial vaccines [Lauvau et al., Science 294:1735-1739 (2001)]. In<br>
completely killed microbes, the de novo microbial gene expression is essentially<br>
eliminated. Therefore, the modification of the microbial nucleic acid to an appropriate<br>
level such that proliferation is attenuated while maintaining a sufficient level of microbial<br>
gene expression may be more effective than a killed microbial vaccine and provides an<br>
approach to vaccine preparation that can be applied to any microbial vector, whether the<br>
vaccine targets the prevention of infectious disease caused by the microbial vector, or the<br>
vector is used to deliver a heterologous antigen. It is to be understood that the use of the<br>
term microbes as it relates to all embodiments of the present invention is intended to mean<br>
free-living microbes and is not intended to include viruses. Such a microbe-based vaccine<br>
may be used to deliver a specific antigen to an individual. In one embodiment, the vaccine<br>
delivers more than one antigen. Such vaccines are designed to stimulate an immune<br>
response to one or more antigens, resulting in an individual who is immunized against the<br>
antigen or antigens. The immune response that is generated can be either an antibody<br>
mediated response, a cell mediated response, or both. The term vaccine is intended to<br>
encompass a. preventative vaccine, i.e. one that is given to stimulate an immune response so<br>
that if the individual subsequently is exposed to the antigen in nature, the pre-formed<br>
immune response will increase the individual's ability to fight off the agecnt or cells carrying<br>
the antigen. The term vaccine is also intended, to encompass a therapeutic vaccine, i.e. one<br><br>
that is given to an individual who already has a disease associated, with the vaccine antigen<br>
wherein the vaccine can elicit an immune response or boost the in dividual's existing<br>
immune response to the antigen to provide an increased ability to fight the agent or cells<br>
carrying the antigen. This includes an immune response to a diseased cell, such as a cancen<br>
cell, as well as an immune response to a disease associated protein such as a prion. In one<br>
embodiment, the free-living microbe is selected from the group consisting of bacteria,<br>
protozoa, and fungi. In one embodiment, the tree-living microbe is a bacteria selected from<br>
the group consisting of Gram positive bacteria, Gram negative bacteria, intracellular<br>
bacteria and mycobacteria.<br>
The present invention includes various levels of modification of the nucleic acid of<br>
microbes. It is understood that the metabolism of the microbial nucleic acid occurs in<br>
several ways. Replication of the microbe involves the copying of the DNA of the entire<br>
microbial genome in order to replicate the microbe and the subsequent partitioning of the<br>
DNA molecules into separate cells, i.e. the cell divides with the resulting cells both having<br>
a complete copy of the DNA of the microbial genome. Microbial nucleic acid metabolism<br>
also involves the combination of transcription of DNA into RNA and translation of RNA to<br>
produce proteins. The transcription of the microbial genome involves the copying of<br>
portions of the DNA of the microbial genome into RNA, either messenger or transfer RNA_<br>
The translation of the messenger RNA involves the reading of this RNA in order to<br>
produce a specific protein or portion of a protein. In the present invention the nucleic acid<br>
of a population of microbes is modified to a desired extent based upon the nature of the<br>
microbe and its intended use. In some embodiments, the desired extent of modification is<br>
such that replication of the microbe's genome is significantly attenuated while the<br>
production of proteins remains sufficiently active (i.e. the microbes is metabolically active).<br>
It is to be understood that whatever the nature of the modification, the level of modification,<br>
can be represented in terms of the number of modifications on average per base pair of the<br>
microbial genome. For example, if the modification is due to covalent binding of a<br>
compound to the nucleic acid (adducts), the modification can be represented in terms of the<br>
average number of base pairs between adducts. The microbes of the invention can be<br>
modified to levels of about 1 modification per 104-108 base pairs, also about 1 modification<br>
per 104-107, also about 1 modification per 105-107, or about 1 modification per 105-106 bases<br>
pairs. In one embodiment, the level of modification is adjusted to the minimum amount<br>
required to block DNA replication in the microbial population, such that the population<br>
shows no observable proliferation, while maintaining sufficient activity of transcription and<br><br>
translation of individual genes (i.e. maintains some metabolic activity) to achieve a safe<br>
and effective vaccine.<br>
[0079] In one aspect, the invention provides a vaccine comprising a free-living<br>
microbe, wherein the nucleic acid of the microbe is modified so that the microbe is<br>
attenuated for proliferation. In some embodiments, the attenuation of the proliferation of*<br>
the microbe is controllable in a dose-dependent manner. In some embodiments, microbial<br>
gene expression in the microbe is substantially unaffected by attenuation of the<br>
proliferation of the microbe. In some embodiments, the microbe in the vaccine expresses<br>
an antigen at a sufficient level to induce an immune response to the antigen in an individual<br>
upon administration of the vaccine to the individual. In some embodiments, the nucleic<br>
acid has been modified by reaction with a nucleic acid targeted compound which reacts<br>
directly with the nucleic acid. In one embodiment, the nucleic acid target compound is an<br>
alkylator such as ?-alanine, N-(acridin-9-yl), 2-[bis(2-chloroethyl)amino]ethyl ester. In<br>
other embodiments, the nucleic acid targeted compound is a psoralen compound (e.g., 4'-<br>
(4-amino-2-oxa)butyl-4,5',8-trimethylpsoralen, also referred to herein as "S-59") activated<br>
by UVA irradiation. In some embodiments, the microbe in the vaccine comprises a genetic<br>
mutation that attenuates the ability of the microbe to repair its nucleic acid that has been<br>
modified. In some embodiments, the microbe is a bacterium, such as Bacillus anthracis or<br>
Listeria monocytogenes. In some embodiments, the microbe comprises a heterologous<br>
nucleic acid sequence encoding an antigen. In some embodiments, the vaccine further<br>
comprises a phannaceutically acceptable carrier and/or an adjuvant. The invention further<br>
provides a method of preventing or treating a disease in a host, comprising administering to<br>
the host an effective amount of the vaccine. The invention also provides a method of<br>
inducing an immune response in a host to an antigen comprising administering to the host<br>
an effective amount of the vaccine, wherein the microbe expresses the antigen.<br>
[0080] The invention further provides vaccines comprising a mutant Listeria<br>
monocytogenes strain or a mutant Bacillus anthracis strain, wherein the mutant Listeria<br>
monocytogenes strain or Bacillus anthracis strain comprises a genetic mutation that<br>
attenuates its ability to repair its nucleic acid.<br>
Antigen-presenting cell vaccines.<br>
[0081] The present invention involves modified free-living microbes and the use of<br>
modified free-living microbes in the preparation of vaccine compositions based on antigen-<br><br>
presenting cells, wherein the nucleic acid of the microbe is modified so that proliferation of<br>
the microbe is attenuated. In some embodiments, the microbial gene expression of the<br>
modified microbe is substantially unaffected.<br>
[0082] In one embodiment of the invention, the antigen-presenting cells used in the<br>
vaccines are professional antigen presenting cells. Professional antigen-presenting cells<br>
include macrophages, dendritic cells and B cells. Other professional antigen-presenting<br>
cells include monocytes, marginal zone Kupffer cells, microglia, Langerhans' cells,<br>
interdigitating dendritic cells, follicular dendritic cells, and T cells. In one embodiment, the<br>
professional antigen-presenting cells are dendritic cells. In another embodiment, the<br>
professional antigen-presenting cells are macrophages or dendritic cells (DCs). In one<br>
embodiment the antigen-presenting cells axe human cells.<br>
[0083] In one embodiment, immature antigen-presenting cells, such as DCs, are<br>
isolated from a patient and infected with a modified microbe expressing an antigen. The<br>
resulting, loaded, antigen-presenting cells are then transferred back into the patient as an<br>
autologous APC vaccine, thereby inducing either a CD4+ or a CD8+ imnxune response.<br>
[0084] Accordingly, one example of a method of preparing and using an antigen-<br>
presenting cell vaccine of the invention is as follows: Immature DCs are isolated from<br>
colon cancer patients and infected with S-59/UVA-inactivated, non-viable, metabolically<br>
active recombinant Listeria-CEA vaccines. DC Infection with Listeria results in efficient<br>
loading of CEA tumor antigen into the MHC class I and II pathways. Listeria infection<br>
stimulates DC to undergo rapid activation and maturation, critical for DC to become potent<br>
APCs capable of inducing primary T cell responses in vivo. Mature DC upregulate the<br>
expression of CD83, co-stimulatory molecules such as CD80, CD86, as well as MHC<br>
molecules. Listeria vaccine-loaded DCs are washed and infused back into the patient as an<br>
autologous DC vaccine to stimulate a CEA-specific T cell response.<br>
[0085] Particular embodiments ate exemplified in the specific Examples listed<br>
below. It is understood, however, that the general methods and techniques described herein<br>
may be more broadly applied to a wide variety of modified microbes, antigens, and<br>
diseases. One of ordinary skill in the art will be able to readily adapt the teachings<br>
described herein.<br>
[0086] In an alternative embodiment, immature antigen-presenting cells, such as<br>
DCs, are infected in vitro with a modified microbe expressing an antigen. The resulting,<br>
loaded, antigen-presenting cells are then used to prime a T-cell population which is then<br><br>
transferred into the patient, thereby inducing either a CD4+ or a CD8+ immune response to<br>
the antigen.<br>
[0087] In another aspect, the invention provides a professional antigen-presenting<br>
cell (e.g., a dendritic cell) comprising a free-living microbe, wherein the nucleic acid of the<br>
microbe is modified so that the microbe is attenuated for proliferation. In some<br>
embodiments, the attenuation of the proliferation of the microbe is controllable in a dose-<br>
dependent manner. In some embodiments, microbial gene expression in the microbe is<br>
substantially unaffected by attenuation of the proliferation of the microbe. In some<br>
embodiments, the microbe in the vaccine expresses an antigen at a sufficient level to induce<br>
an immune response to the antigen in an individual upon administration of the vaccine to<br>
the individual. In some embodiments, the nucleic acid has been modified by reaction with<br>
a nucleic acid targeted compound which reacts directly with the rrucleic acid. In one<br>
embodiment,the nucleic acid target compound is an alkylator such as ?-alanine, N-(acridin-<br>
9-yl), 2-[bis(2-chloroethyl)amino]ethyl ester. In other embodiments, the nucleic acid<br>
targeted compound is a psoralen compound activated by UVA irradiation. In some<br>
embodiments, the microbe in the vaccine comprises a genetic mutation that attenuates the<br>
ability of the microbe to repair its nucleic acid that has been modified. In some<br>
embodiments, the microbe is a bacterium. In some embodiments, the microbe comprises a<br>
heterologous nucleic acid sequence encoding an antigen. The invention also provides a<br>
vaccine comprising the antigen-presenting cell. The invention further provides a method of<br>
preventing or treating a disease in a host, comprising administering to the host an effective<br>
amount of the antigen-presenting cell. The invention also provides a method of inducing<br>
an immune response in a host to an antigen comprising administering to the host an<br>
effective amount of the antigen-presenting cell, wherein the microbe expresses the antigen.<br>
The invention further provides a method of activating naive T cells ex vivo or in vitro,<br>
comprising contacting the naive T cells with the professional antigen-presenting cell under<br>
suitable conditions and for a sufficient time to activate the naive T-cells.<br>
Attenuation of microbial replication.<br>
[0088] The present invention involves the modification of microbial nucleic acid in<br>
order to attenuate replication, of the microbe. This attenuation in replication can be used to<br>
increase the level of safety upon administration of the microbes to individuals. The ability<br>
of a microbe to proliferate can be measured by culturing a population of microbes under<br><br>
conditions that provide normal growth. The normal growth of a population of microbes is<br>
considered to be the growth, of microbes having no modifications to the nucleic acid of the<br>
microbe. The modification of the microbial genome will result in some attenuation so that<br>
the microbe will not undergo normal growth. Some microbes will form colonies that can<br>
be counted on solidified growth medium. Attenuation of the replication of the microbe can<br>
thus be measured as a reduction in the number of colony forming units (CFU). A stock<br>
solution of the microbe colony will be serially diluted until the number of colony forming<br>
units can be easily measured (e.g. 50-500 CFU). Typically, dilutions are 10-fold and the<br>
number of colonies counted for one or more of the diluted samples is used to estimate the<br>
log titer of the sample. For example, an aliquot of diluted microbe stock is plated on<br>
growth media and the resulting colonies are counted. The colony forming units per mL<br>
(CFU/mL) of the dilution is calculated, and the colony forming units per mL of the original<br>
stock (known as the titer) is calculated from the dilution. The log number is known as the<br>
log titer. As an example, 24 colony forming units on plating a 0.2 mL aliquot of a 1 x105<br>
dilution gives a 1.2 x 107 titer, or 7.08 log titer stock. The attenuation can be measured as<br>
the comparison of microbial titer prior to modification of the microbial nucleic acid to that<br>
after modification of the microbial nucleic acid. The log of the ratio of the titer of<br>
unmodified microbe to the titer of microbe after modification, represents the log attenuation<br>
(or simply the difference in log titer of the two). For example, if an unmodified microbe<br>
titer measures 1.2xl07and a modified microbe titer measures 4.3x102, the resulting level of<br>
attenuation is 4.45 log. This method can be used to assess the attenuation of any microbe,<br>
whether pathogenic or non-pathogenic. For some microbes, rather than measuring the<br>
growth of the microbe directly, a plaque assay that measures the microbe by its ability to<br>
kill infected cells can be used. For example, certain intracelhular bacteria can be grown on<br>
a lawn of mammalian cells that it can infect. After appropriate incubation conditions, the<br>
lawn can be observed for plaques (clear areas in the cell layer that represent killed cells).<br>
The above calculations are similar, where the number of plaque forming units is substituted<br>
for colony forming units to assess attenuation of the number of plaque forming units by<br>
modification of the nucleic acid of the microbe. For embodiments of the invention, the<br>
desired amount of attenuation can range from a two-fold reduction to much greater levels<br>
of attenuation, including a level where essentially no proliferation is observed, depending<br>
on the desired level of safety and the intended application of the microbe. A two-fold<br>
attenuation in replication would be observed if for a given dilution, there are half as many<br>
colonies (or plaques) in the population of a microbe where the nucleic acid is modified as<br><br>
there are in an unmodified population of the microbe (about 0.3 log attenuation). En some<br>
embodiments, the attenuation is at least about 0.3 log, about 1 log, about 2 log, about 3 log,<br>
about 4 log about 5 log, about 6 log, or at least about 8 log. In some embodiments, the<br>
attenuation is in the range of about 0.3 to &gt; 10 log, about 2 to &gt;10 log, about 4 to &gt;10 log,<br>
about 6 to &gt; 10 log, about 0.3-8 log, also about 0.3-7 log, also about 0.3-6 log, also about<br>
0.3-5 log, also about 0.3-4 log, also about 0.3-3 log, also about 0.3-2 log, also about 0.3-1<br>
log. In some embodiments, the attenuation is in the range of about 1 to &gt;10 log, 1-8 log, 1-<br>
6 log, also about 2-6 log, also about 2-5 log, also about 3-5 log. In one embodiment: of the<br>
invention, the attenuation results in essentially complete inactivation (e.g. where no<br>
colonies or plaques are observed to the limit of detection), wherein the microbial gene<br>
expression is sufficiently active. Such a population of microbes can be achieved by<br>
titrating the concentration of the agent used to modify the microbial nucleic acid to find the<br>
lowest concentration at which no colonies or plaques are observed at the limit of detection.<br>
[0089] In the case of pathogenic microbes, it is also possible to assess the<br>
attenuation in terms of biological effects of the microbe. For example, the pathogenicity of<br>
a microbe can be assessed by measurement of the median lethality (LD50) in mice or other<br>
vertebrates. The LD50 is the amount (e.g. CFU) of microbe injected into the vertebrate that<br>
would result in the death of half of the population of the vertebrate. The LD50 values can<br>
be compared for modified and unmodified microbes as a measure of the amount of<br>
attenuation. For example, if an unmodified population of microbes has an LD50 of 103<br>
microbes and the population of microbes in which the nucleic acid has been modified has<br>
an LD50 of 105 microbes, the microbe has been attenuated so that its LD50 is increased 100-<br>
fold, or by 2 log. In some embodiments, the LD50 is 2-fold to 1000-fold higher. In some<br>
embodiments, an attenuated strain is used that already has a relatively high LD50. In such<br>
cases, the modified microbes increase in LD50 will be limited by how much material can be<br>
infused without causing harm. For example, the LD50 of a heat killed organism would not<br>
be much higher than about 1-5 x 109 simply because of the loading of biological material<br>
into the mice and/or the inflammatory reaction to the bacterial wall components. The<br>
degree of attenuation may also be measured qualitatively by other biological effects, such<br>
as the extent of tissue pathology or serum, liver enzyme levels. Typically, alanine<br>
aminotransferase (ALT), aspartate aminotransferase (AST), albumin, and billirubin. levels<br>
in the serum are determined at a clinical laboratory for mice injected with microbes of the<br>
present invention. Comparisons of these effects in mice or other vertebrates would be<br>
made for unmodified and modified micro be as a way to assess the attenuation of the<br><br>
microbe. In addition to measuring the effects of the microbes on the tissues, the amount of<br>
viable microbe that can be recovered from infected tissues such as liver or spleen as a<br>
function of time could also be used as a measure of attenuation by comparing these values<br>
in mice injected with unmodified vs. modified microbes.<br>
Expression of proteins by microbes of the invention.<br>
[0090] The modification of the nucleic acid of the microbe, in addition to<br>
attenuating proliferation of the microbe, is controlled so that microbial gene expression is<br>
substantially unaffected. To be substantially unaffected, the microbial gene expression<br>
need not be completely active upon modification of the nucleic acid. It is only necessary<br>
that in a population of a microbe in which, the nucleic acid is modified to attenuate<br>
replication, microbial gene expression is sufficiently active to provide an adequate level of<br>
expression of the desired protein by the microbe. An adequate level of expression depends<br>
to some extent on the intended use of the microbe. For example, if the microbe contains a<br>
particular antigen that is to be used as a vaccine, adequate expression would be determined<br>
as the minimum level of expression that provides an effective protective or therapeutic<br>
immune response to the vaccine. The microbial gene expression can also be assessed by<br>
both in vitro and in vivo methods in order to assess whether such a vaccine might provide<br>
an effective immune response. In general, a population of a microbe in which the nucleic<br>
acid has been modified can be compared to an unmodified population of the microbe with<br>
respect to a particular antigen.<br>
[0091] One possibility is to measure the presentation of the antigen, of interest by an<br>
antigen presenting cell that has been mixed with a population of the microbe. The<br>
microbes may be mixed with a suitable antigen presenting cell or cell line, for example a<br>
dendritic cell, and the antigen presentation, by the dendritic cell to a T cell that recognizes<br>
the antigen can be measured. If the microbes are expressing the antigen at a sufficient<br>
level, it will be processed into peptide fragments by the dendritic cells and presented in the<br>
context of MHC class I or class II to CD8+ or CD4+ T cells, respectively. For the purpose<br>
of detecting the presented antigen, a T cell clone or T cell line responsive to the particular<br>
antigen may be used. The T cell may also be a T cell hybridoma, where the T cell is<br>
immortalized by fusion with a cancer cell line. Such T cell hybridomas, T cell clones, or T<br>
cell lines can comprise either CD8+ or CD4+ T cells. The antigen presenting cell can<br>
present to either CD8+ or CD4+ T cells, depending on the pathway by which the antigens<br>
are processed. CD8+ T cells recognize antigens in the context of MHC class I while CD4+<br>
T cells recognize antigens in the context of MHC class EL The T cell will be stimulated by<br><br>
the presented antigen through specific recognition by its T cell receptor, resulting in the<br>
production of certain proteins, such as IL-2 or interferon-y (DFN-?), that can be<br>
quantitatively measured (for example using an ELISA assay). Alternatively, a hybridoma<br>
can be designed to include a reporter gene, such as ?-galactosidase, that is activated upon<br>
stimulation of the T cell hybridoma by the presented antigens. The increase in the<br>
production of ?-galactosidase can be readily measured by its activity on a substrate, such<br>
as chlorophenohred-?-D-galactopyranoside, which results in a color change. The color<br>
change can be directly measured as an indicator of specific antigen presentation (Examples<br>
1, 2 and 11). Additional in vitro and in vivo methods for assessing the antigen expression<br>
of microbial vaccines of the present invention can be found in Example 5. It is also<br>
possible to directly measure the expression of a particular protein by microbes of the<br>
present invention. For example, a radioactively labeled amino acid can be added to a cell<br>
population and the amount of radioactivity incorporated into a particullar protein can be<br>
determined. The proteins synthesized by the cell population can be is olated, for example<br>
by gel electrophoresis or capillary electrophoresis, identified as the protein of interest, e.g.<br>
by binding with an antibody-specific for the protein, and the amount of radioactivity can be<br>
quantitatively measured to assess the expression level of the particular protein.<br>
Alternatively, the proteins can be expressed without radioactivity and detected by various<br>
methods, such as an ELISA assay or by gel electrophoresis and Western blot with detection<br>
using an enzyme linked antibody or fluorescently labeled antibody.<br>
[0092] While it is possible that the modification of the microbial nucleic acid<br>
reduces the level of protein expression as compared to an unmodified microbe, it is to be<br>
understood that this may still provide an effective vaccine. It is the combination of<br>
attenuation of proliferation with adequate protein expression that is important in some<br>
embodiments of the invention. The efficacy of a vaccine is generally related to the dose of<br>
antigen that can be delivered by the microbe, and in some instances, some level of active<br>
gene expression by the microbe is necessary. The attenuation of replication of the microbe<br>
may be several log while the microbial gene expression is still sufficiently maintained. If<br>
the same dose of an attenuated microbe is compared to that of an unmodified microbe, the<br>
resulting antigen expression (as assessed by the methods discussed above) in the attenuated<br>
microbe population is at least about 1%, about 5%, about 10%, about 25%, about 50%,<br>
about 75% or at least about 90% of the antigen expression in the unmodified microbe<br>
population. Since there may be several log attenuation in replication, the dose of the<br><br>
modified microbe may be safely increased by up to several log, resulting in an equivalent<br>
or greater amount of the antigen presented by the attenuated microbes relative to<br>
unmodified microbes upon vaccination.<br>
[0093] In some embodiments, a heterologous nucleic acid sequence encoding a<br>
protein may be codon-optimized to match the codon preference of the bacterial host<br>
expressing the protein. In addition, the sequence encoding a signal peptide fused to the<br>
expressed protein may also be codon-optimized to match the codon preference of the<br>
bacterial host. In preferred embodiments, the bacterial host is Listeria and either or both, of<br>
the heterologous protein encoding sequence and the sequeace encoding a signal peptide<br>
may be codon-optimized. For further information on codon optimization of antigens and<br>
signal sequences in Listeria, see U.S. application Serial No- 60/532,598, incorporated by<br>
reference herein.<br>
Microbial nucleic acid modification<br>
[0094] The nucleic acid of a population of a microbe can be modified by a variety<br>
of methods. The nucleic acid of the microbe can be modified by physical means, e.g.<br>
irradiation with ultraviolet light or ionizing radiation. Ionizing radiation, such as x-rays or<br>
y-rays, may be used to cause single-strand or double-strand breaks in the nucleic acid.<br>
Ultraviolet radiation may be used to cause pyrimidine dimears in the nucleic acid. The<br>
appropriate dose of radiation is determined by assessing the effects of the radiation on<br>
replication and protein expression as detailed above.<br>
[0095] The nucleic acid of the microbe can also be modified by chemical means,<br>
e.g. by reaction with a nucleic acid targeted compound. In one embodiment, the microbe is<br>
treated with a nucleic acid targeted compound that can modify the nucleic acid such that<br>
the proliferation of the microbe is attenuated, wherein the nxicrobial population is still able<br>
to express a desired protein antigen to a degree sufficient to elicit an immune response.<br>
The nucleic acid targeted compound is not limited to a particular mechanism of modifying<br>
the nucleic acid. Such compounds modify the nucleic acid either by reacting directly with<br>
the nucleic acid (i.e. all or some portion of the compound covalently binds to the nucleic<br>
acid), or by indirectly causing the modification of the nucleic acid (e.g. by causing oxygen<br>
damage via generation of singlet oxygen or oxygen radicals, by generating radicals of the<br>
compound that cause damage, or by other mechanisms of reduction or oxidation of the<br>
nucleic acid). Enediynes are an example of a class of compounds that form radical species<br>
that result in the cleavage of DNA double strands [Nicolaou et al., Proc. Natl. Acad. Sci.<br><br>
USA, 90:5881-5888 (1993)]. Compounds that react directly with the nucleic acid may<br>
react upon activation of the compound, for example upon radiation of the compound.<br>
Compounds that react indirectly to cause modification of the nucleic acid may require<br>
similar activation to generate either an activated species of the compound or to generate<br>
some other active species. While not being limited to the means for activation of nucleic<br>
acid targeted compounds, one embodiment of the invention includes the use of<br>
photoactivated compounds that either react directly with the nucleic acid or that generate a<br>
reactive species such as a reactive oxygen species (e.g. singlet oxygen) which then reacts<br>
with the nucleic acid.<br>
[0096] The nucleic acid targeted compounds preferentially modify nucleic acids<br>
without significantly modifying other components of a biological sample. Such<br>
compounds provide adequate modification of the nucleic acid without significantly altering<br>
ordamaging cell membranes, proteins; and lipids. Such compounds may modify these<br>
other cell components to some degree that is not significant. These cell components such<br>
as cell membranes, proteins and lipids are not significantly altered if their biological<br>
function, is sufficiently maintained. In the case of treating a microbe with a nucleic acid<br>
targeted, compound, the nucleic acid modification is such that the replication, of the microbe<br>
is attenuated while the cell membranes, proteins and lipids of the microbe are essentially<br>
unaffected such that microbial gene expression, is active (e.g. the enzymes required for this<br>
are not significantly affected), and the surface of the microbe maintains essentially the<br>
same antigenicity as a microbe that has not been treated with the compound. As a result,<br>
such compounds are useful in preparing an inactivated microbe for use as a vaccine since<br>
the proliferation of the microbe is sufficiently attenuated while maintaining sufficient<br>
antigenicity or immunogenicity to be useful as a vaccine. Because the compounds<br>
specifically modify nucleic acids, the modification can be controlled to a desired level so<br>
that replication is attenuated while maintaining a sufficient level of protein expression. The<br>
modification can be controlled by varying the parameters of the reaction, such as<br>
compound concentration, reaction media, controlling compound activation factors such as<br>
light dose or pH, or controlling compounds that cause oxygen damage by coatrolling the<br>
oxygen concentration (either physically, e.g. by degassing, or chemically, by use of oxygen<br>
scavengers). A nucleic acid targeted compound is any compound that has a tendency to<br>
preferentially bind nucleic acid, i.e. has a measurable affinity for nucleic acid- Such<br>
compounds have a stronger affinity for nucleic acids than for most other components of a<br>
biological sample, especially components such as proteins, enzymes, lipids and<br><br>
membranes. The nucleic acid targeting provides specificity for the modification of mucleic<br>
acids without significantly affecting other components of the biological sample, such, as the<br>
machinery for gene transcription and protein translation.<br>
[0097J Compounds can be targeted to nucleic acids in a number of modes.<br>
Compounds which bind by any of the following modes or combinations of them are<br>
considered nucleic acid targeted compounds. Intercalation, minor groove binding, major<br>
groove binding, electrostatic binding (e.g. phosphates backbone binding), and sequence-<br>
specific binding (via sequence recognition in the major or minor groove) are all non-<br>
covalent modes of binding to nucleic acids. Compounds that include one or more of these<br>
modes of binding will have a high affinity for nucleic acids. While the invention is not<br>
limited to the following compounds, some examples of compounds having these modes of<br>
binding to nucleic acid are as follows: intercalators are exemplified by acridities, acricones,<br>
' proflavin, acriflavine, actinomycins, anthracyclinones, beta-rhodomycin A, daunamycin,<br>
thiaxanthenones, miracil D, anthramycin, mitomycin, ecbinomycin, quinomycin, triostin,<br>
diacridines, ellipticene (including dimers, trimers and analogs), norphilin A, fluorenes and<br>
flourenones, fluorenodiamines, quinacrine, benzacridines, phenazines, phenanthradines,<br>
phenothiazines, chlorpromazine, phenoxazines, benzothiazoles, xanthenes and thio-<br>
xanthenes, anthraquinories, anthrapyrazoles, benzothiopyranoindoles, 3,4-benzpyrene,<br>
benzopyrene diol epoxidie, 1-pyrenyloxirane, benzanthracene-5,6-oxide, benzodipyrones,<br>
benzothiazoles, quinolones, chloroquine, quinine, phemylquinoline carboxamides,<br>
furocoumarins (e.g. psoralens, isopsoralens, and sulfur analogs thereof), ethidium salts,<br>
propidium, coralyne, ellipticine cation and derivatives, polycyclic hydrocarbons and their<br>
oxirane derivatives, and echinimycin; minor groove binders are exemplified by distamycin,<br>
mitomycin, netropsin, other lexitropsins, Hoechst 33258 and other Hoechst dyes, DAPI<br>
(4',6'-diamidine-2-phenylindole), berenil, and triarylmethane dyes; major groove binders<br>
are exemplified by aflatoxins; electrostatic binders are exemplified by spermine,<br>
spennidine, and other polyamines; and sequence-specific binders are exemplified by<br>
nucleic acids or analogues which bind by such sequence-specific interactions as triple helix<br>
formation, D-loop formation, and direct base pairing to single stranded targets. Other<br>
sequence-specific binding compounds include poly pyrrole compounds, poly pyrrrole<br>
imidazole compounds, cyclopropylpyrroloindole compounds and related minor groove<br>
binding compounds [Wemmer, Nature Structural Biology, 5(3): 169-171 (1998), Wurtz et<br>
al., Chemistry &amp; Biology 7(3): 153-161 (2000), Anthoney et al., Am. J. Pharmacogenomics<br>
1(1):67-81 (2001)].<br><br>
[0098] In addition to targeting nucleic acids, the compounds are also able to react<br>
with the nucleic acid, resulting in covalent binding to the nucleic acid. Nucleic acid<br>
alkylators are a class of compounds that can react covalently with nucleic acid and include,<br>
but are not limited to, mustards (e.g. mono or bis haloethylamine groups, and mono<br>
haloethylsulfide groups), mustard equivalents (e.g. epoxides, alpha-halo ketones) and<br>
mustard intermediates (e.g. aziridines, aziridiniums and their sulfur analogs),<br>
methanesulphonate esters, and nitroso ureas. The nucleic acid alkylators typically react<br>
with a nucleophilic group on the nucleic acid. It is the combination of the nucleic acid<br>
alkylating activity and the nucleic acid targeting ability of these compounds that gives them<br>
the ability to covalently react specifically with nucleic acids, providing the desired<br>
modification of the nucleic acid of microbes for use in the present invention. The<br>
specificity of these compounds may be further enhanced by the use of a quencher that will<br>
not enter the microbe. Such a quencher will quench reactions with the surface of the<br>
microbe while still allowing the nucleic acid targeted compounds to react with the<br>
microbial nucleic acid. A discussion of such quenching can be found in US Patent number<br>
6,270,952, the disclosure of which is hereby incorporated by reference. The modification<br>
of the microbial nucleic acid can be controlled by adjusting the compound concentration<br>
and reaction conditions. The appropriate concentration and reaction conditions are<br>
determined by assessing their effects on replication and protein expression as detailed<br>
above. The compounds used in the present invention are effective at concentrations of<br>
about 10 pM to 10 mM, also about 100 pM to 1 mM, also about 1 nM to 10 uM, also about<br>
1-500 nM, also about 1-200 nM or about 1-100 nM. A discussion of nucleic acid targeted,<br>
nucleic acid reactive compounds for specific reaction with nucleic acids, in particular<br>
microbial nucleic acids, can be found in US patents 6,143,490 and 6,093,725, the<br>
disclosures of which are hereby incorporated by reference.<br>
[0099] The nucleic acid can be modified by using a nucleic acid targeted compound<br>
that requires activation with radiation in order to cause the nucleic acid modification. Such<br>
compounds are targeted to nucleic acids as discussed above. These compounds include,<br>
but are not limited to, acridines, acridones, anthyrl derivatives, alloxazines (e.g. riboflavin),<br>
benzotriazole derivatives, planar aromatic diazo derivatives, planar aromatic cyan.o<br>
derivatives, toluidines, flavines, phenothiazines (e.g. methylene blue), furocoumarins,<br>
angelicins, psoralens, sulfur analogs of psoralens, quinolones, quinolines, quinoxalines,<br>
napthyridines, fluoroquinolones, anthraquinones, and anthracenes. Many of these<br><br>
compounds are used as DNA photocleavage agents [Da Ros et al., Current Pharmaceutical<br>
Design 7:1781 (2001)]. While the invention is not limited to the method of activation of<br>
the nucleic acid targeted compounds, typically, the compounds can be activated with light<br>
of particular wavelengths. The effective wavelength of light depends on the nature of the<br>
compound and can range anywhere from approximately 200 to 1200 nm. For some of<br>
these compounds, activation causes modification of the nucleic acid without direct binding<br>
of the compound to the nucleic acid, for example by generating reactive oxygen species in<br>
the vicinity of the nucleic acid. For some of these compounds, activation results in binding<br>
of the compound directly to the nucleic acid (i.e. the compound binds covalently). Some of<br>
these compounds can react with the nucleic acid to form an interstrand crosslink. Psoralens<br>
are an example of a class of compounds that crosslink nucleic acids. These compounds are<br>
typically activated with UVA light (320-400 nm). Psoralen compounds for use in the<br>
present invention are exemplified in US patents 6,133,460 and 5,593,823-, the disclosures of<br>
which are hereby incorporated by reference. Again, it is the combination of nucleic acid<br>
targeting and the ability to modify the nucleic acid upon activation that provide specific<br>
reactivity with nucleic acids. The modification of the microbial nucleic acid can be<br>
controlled by adjusting the compound concentration, reaction conditions and light dose.<br>
The appropriate concentration and light dose are determined by assessing their effects on<br>
replication and protein expression as detailed above. In addition to compound<br>
concentration and level of light exposure, the reaction is affected by the conditions under<br>
which the sample is dosed with UVA light. For example, the required overall<br>
concentration for irradiating a population of microbes in a buffered media is going to vary<br>
from a population that is cultured in a growth media (e.g. BHI, Triptase Soy Broth). The<br>
photoreaction may be affected by the contents of the growth media, which may interact<br>
with the psoralen, thereby requiring a higher overall concentration of the psoralen. In<br>
addition, the effective dosing of the microbes may depend on the growth phase of the<br>
organism and the presence or absence of compound during the growth phase. In one<br>
embodiment, the population of microbes comprises growth media during the psoralen UVA<br>
treatment. In one embodiment, the psoralen is added to the population of microbes, the<br>
population is cultured to grow the microbes in the presence of psoralen and growth media,<br>
and the UVA treatment is performed at some point in the growth phase of the microbes. In<br>
one embodiment, the population is grown to an OD of 0.5-1 (1 x 1 07 to 1x 109 CFU/mL) in<br>
the presence of the psoralen prior to irradiation with an appropriate dose of UVA light<br>
Psoralen compounds are effective at concentrations of about 10 pM to 10 mM, also about<br><br>
100 pM to 1 mM, also about 1 nM to 10 uM, also about 1-500 nM, also about 1-200 nM or<br>
about 1-100 nM, with the UVA light dose ranging from about 0.1 -100 J/cm2, also about<br>
0.1-20 J/cm2, or about 0.5-10 J/cm2, 0.5-6 J/cm2 or about 2-6 J/cm2. In one embodiment,<br>
the microbe is treated in the presence of growth media at psoralen concentrations of about<br>
10 pM to 10 mM, also about 1-5000 nM, also about 1-500 nM, also about 5-500 nM, or<br>
about 10-400 nM. In one embodiment, the microbe treated in the presence of growth<br>
media is grown to an OD of 0.5-1 in the presence of psoralen at concentrations of about 10<br>
pM to 10 mM, also about 1-5000 nM, also about 1 -500 nM, also about 5-500 nM, or about<br>
10-400 nM. Following the growth to an OD of 0.5-1, the microbe population is irradiated.<br>
with UVA light at a dose ranging from about 0.1 - 100 J/cm2, also about 0.1-20 J/cm2, or<br>
about 0.5-10 J/cm2, 0.5-6 J/cm2 or about 2-6 J/ cm2.<br>
Microbes containing heterologous nucleic acid sequences<br>
[0100]' Microbes can be alfered to include a heterologous nucleic acid sequence that<br>
can be expressed by the microbe. The heterologous sequence can encode at least one<br>
specific protein antigen. The microbes may be altered by methods known to one skilled in<br>
the art [Sambrook and Russell, Molecular Clonings A Laboratory Manual, Third Edition,<br>
Cold Spring Harbor Laboratory Press, (2000)]. The microbes can be altered to contain one<br>
or more sequences that encode one or more antigens. The heterologous nucleic acid<br>
sequence encoding a specific antigen is not limited to an exact nucleic acid sequence but is<br>
of a sequence that is sufficient to provide the expression of an antigen that will elicit the<br>
desired immune response when administered to an individual. The heterologous sequence<br>
can be expressed as an antigen related to a particular disease. The microbe expressing such<br>
antigens can be used as a vaccine, wherein the vaccine may be used as a preventative<br>
treatment or a therapeutic treatment Diseases that can be treated by such vaccines include<br>
infectious diseases, autoimmune diseases, allergies, cancers and other hyperproliferative<br>
diseases.<br>
[0101] The microbes of the invention may be altered to contain a heterologous<br>
nucleic acid sequence encoding a specific tumor antigen. A large number of tumor specific<br>
antigens that are recognized by T cells have been identified [Renkvist et al., Cancer<br>
Immunol Innumother 50:3-15 (2001)]. These tumor antigens may be differentiation<br>
antigens (e.g., PSMA, Tyrosinase, gp100), tissue-specific antigens (e.g. PAP, PSA),<br>
developmental antigens, tumor-associated viral antigens (e.g. HPV 16 E7), cancer-testis<br>
antigens (e.g. MAGE, BAGE, NY-ESO-1), embryonic antigens (e.g. CEA, alpha-<br><br>
fetoprotein), oncoprotein antigens (e.g. Ras, p53), over-expressed protein antigerxs (e.g.<br>
ErbB2 (Her2/Neu), MUC1), or mutated pro tein antigens. The tumor antigens that may be<br>
encoded by the heterologous nucleic acid sequence include, but are not limited to, 707-AP,<br>
Annexin II, AFP, ART-4, BAGE, p-catenin/m, BCL-2, bcr-abl, bcr-abl p190, bcr-abl p210,<br>
BRCA-1, BRCA-2, CAMEL, CAP-1, CASP-8, CDC27/m, CDK-4/m, CEA, CT9, CT10,<br>
Cyp-B, Dek-cain, DAM-6 (MAGE-B2), DAM-10 (MAGE-B1), ELF2M, EphA2, ETV6-<br>
AML1, G250, GAGE-1, GAGE-2, GAGE-3, GAGE-4, GAGE-5, GAGE-6, GAOE-7B,<br>
GAGE-8, GnT-V,, gp100, HAGE, HER2/neu, HLA-A*0201-R170I, HPV-E7, HSP70-2M,<br>
HST-2, hTERT, hTRT, iCE, inhibitors of apoptosis (e.g. survivin), KIAA0205, LAGE,<br>
LAGE-1, LDLR/FUT, MAGE-1, MAGE-2, MAGE-3, MAGE-6, MAGE-A1, MAGE-A2,<br>
MCAGE-A3, MAGE-A4, MAGE-A6, MAGE-A10, MAGE-A12, MAGE-B5, MAGE-B6,<br>
MAGE-C2, MAGE-C3, MAGE-D, MART-1, MART-1/Melan-A, MC1R, MDM-2,<br>
mesothelin, Myosin/m, MUC1, MUC2, MUM-1, MUM-2, MUM3, neo-polyA-<br>
polymerase, NA88-A, NY-ESO-1, NY-ESO-1a (CAG-3), PAGE-4, PAP, Proteinase 3<br>
(PR3), P15, pl90, Pml/RARa, PRAME, PSA, PSM, PSMA, RAGE, RAS, RCAS 1, RU1,<br>
RU2, SAGE, SART-1, SART-2, SART-3, SP17, SPAS-1, TEL/AML1, TP17m, Tyrosinase,<br>
TARP, TRP-1 (gp75), TRP-2, TRP-2/INT2, WT-1, and alternatively translated NY-ESO-<br>
ORF2 and CAMEL proteins, derived from the NY-ESO-1 and LAGE-1 genes. The<br>
microbes of the present invention encompass any tumor antigen that can elicit a tumor-<br>
specific immune response, including antigens yet to be identified. The microbes may be<br>
altered to contain more than one heterologous sequence encoding more than one tumor<br>
antigen. Preferred antigens include mesothelin [Argani et al., Clin Cancer Res. 7(12):3862-<br>
8 (2001)], Spl7 [Lim et al., Blood. 97(5):1508-10 (2001)], gp100 [Kawakami et al., Proc.<br>
Natl. Acad. Sci. USA 91:6458 (1994)], PAGE-4 [Brinkmann et al., Cancer Res.<br>
59C7):1445-8 (1999)], TARP [Wolfgang et al., Proc. NatL Acad. Sci. USA 97(17):9437-42<br>
(2000)], EphA2 [Tatsumi et al., Cancer Res. 63(15):4481-9 (2003)], PR3 [Muller-Berat et<br>
al., Clin. Immunol. Immunopath. 70(l):51-9 
Publication Number 20020150588].<br>
[0102] In one embodiment of the invention, the heterologous antigen expressed by<br>
the modified microbe is CEA. CEA is a 180-kDA membrane intercellular adhesion<br>
glycoprotein that is over-expressed in a significant proportion of human tumors, including<br>
90% of colorectal, gastric, and pancreatic, 70% of non-small cell lung caner, and 50% of<br>
breast cancer (Hammarstrom, Semin. Cancer Biol., 9:67-81). A variety of<br><br>
immunotherapeutics such as anti-idiotype monoclonal antibody mimicking CEA (Foon et<br>
al., Clin. Cancer Res., 87:982-90 (1995), or vaccination using a recombinant vaccinia virus<br>
expressing CEA (Tsang et al., J. Natl. Cancer Inst., 87:982-90 (199:5)) have been<br>
investigated, unfortunately, however, with limited success. Nonetheless, investigators have<br>
identified a HLA*0201-restricted epitope, CAP-l(CEA605-613), that is recognized by<br>
human T cell lines that were generated from vaccinated patients. Vaccination of patients<br>
with DC pulsed with this epitope foiled to induce clinical responses (Morse et al., Clin.<br>
Cancer Res., 5:1331-8 (1999)). Recently, a CEA605-613 peptide agonist was identified<br>
with a heteroclitic aspartate to asparagine substitution at position 610 (CAP1-6D).<br>
Although this amino acid substitution did not alter MHC binding affinity of this peptide,<br>
the use of the altered peptide ligand CAPL) resulted in improved generation of CEA-<br>
specific cytotoxic T lymphocytes (CTL) in vitro. CAPl-6D-specific CTL maintained their<br>
ability to recognize and4yse tumor cells expressing native GEA (Zaremba et al., Cancer<br>
Res., 57:4570-7 (1997); Salazar et al., Int. J. Cancer, 85:829-38 (2000)). Fong et al.<br>
demonstrated induction of CEA-specific immunity in patients with colon cancer vaccinated<br>
with Flt3-ligand expanded DC incubated with this APL. Encouragingly, 2 of 12 patients<br>
after vaccination experienced dramatic tumor regressions that correlated with the induction<br>
of peptide-MHC tetramer+ T cells (Fong et al., Proc. Natl. Acad. Sci. U.S.A., 98:8809-14<br>
(2001)). Taken together, this work provides significant validation for CEA-targeted<br>
immunotherapy for colorectal cancer.<br>
[0103] In another embodiment, the heterologous antigen expressed by the modified<br>
microbe is proteinase-3 or is derived from proteinase-3. For instance, in one embodiment,<br>
the antigen, comprises the HLA-A2.1 -restricted peptide PR1 (aa 169-177; VLQELNVTV<br>
(SEQ ID NO:50)). Information on proteinase-3 and/or the PR1 epitope is publicly<br>
available in the following references: US Patent No. 5,180,819, Molldrem, et al., Blood,<br>
90:2529-2534 (1997); Molldrem et al., Cancer Research, 59:2675-2681 (1999); Molldrem,<br>
et al., Nature Medicine, 6:1018-1023 (2000); and Molldrem et al., Oncogene, 21: 8668-<br>
8673 (2002).<br>
[0104] Accordingly, in some embodiments, the modified microbe comprises a<br>
nucleic acid molecule encoding an antigen such as mesothelin, SPA S-1, proteinase-3,<br>
EphA2, SP-17, gp100, PAGE-4, TARP, Her-2/neu, WT-1, NY-ESO-1, PSMA, K-ras, or<br>
CEA, or an antigen derived from one of those proteins. In some embodiments, the<br>
modified microbe comprises a nucleic acid molecule encoding aa antigen such as<br>
mesothelin, SPAS-1, proteinase-3, SP-17, gp100, PAGE-4, TARP, WT-1, NY-ESO-1 or<br><br>
CEA, or an antigen derived from one of those proteins. In some embodiments, the<br>
modified microbe comprises a nucleic acid molecule encoding human mesothelin, or an<br>
antigen derived from human mesothelin. In other embodiments, the modified microbe<br>
comprises a nucleic acid molecule encoding human EphA2, or derived from human EphA2.<br>
[0105] The microbes of the invention may be altered to contain a heterologous<br>
nucleic acid sequence encoding a specific infectious disease antigen. In one embodiment,<br>
the antigen is derived from a human or animal pathogen. The pathogen is optionally a<br>
virus, bacterium, fungus, or a parotozoan. For instance, the antigen may be a viral or fungal<br>
or bacterial antigen.<br>
[0106] For instance, the antigen may be derived from Human Immunodeficiency<br>
virus (such as gp 120, gp 160, gp41, gag antigens such as p24gag and p55gag, as well as<br>
proteins derived from the pol, env, tat, vif, rev, nef, vpr, vpu and LTR regions of HIV),<br>
Feline Immunodeficiency virus, or human or animal herpes viruses. In one embodiment,<br>
the antigen is derived from herpes simplex virus (HSV) types 1 and 2 (such as gD, gB, gH,<br>
Immediate Early protein such as ICP27), from cytomegalovixus (such as gB and gH), from<br>
Human Metapneumovirus, from Epstein-Barr virus or from Varicella Zoster Virus (such as<br>
gpl, II or HI). (See, e. g. Chee et al. (1990) Cytomegaloviruses (J. K. McDougall, ed.,<br>
Springer Verlag, pp. 125-169; MEcGeoch et al. (1988) J. Gen. Virol. 69:1531-1574; U.S.<br>
Pat. No. 5,171,568; Baer et al. (1984) Nature 310: 207-211; and Davison et al. (1986) J.<br>
Gen. Virol. 67:1759-1816.)<br>
[0107] In another embodiment, the antigen is derived from a hepatitis virus such as<br>
hepatitis B virus (for example, Hepatitis B Surface antigen), hepatitis A virus, hepatitis C<br>
virus, delta hepatitis virus, hepatitis E virus, or hepatitis G virus. See, e. g., WO 89/04669;<br>
WO 90/11089; and WO 90/14436. The HCV genome encodes several viral proteins,<br>
including El and E2. See, e. g., Houghton et al., Hepatology 14:381-388(1991).<br>
[0108] An antigen that is a viral antigen is optionally derived from a virus from any<br>
one of the families Picornaviridae (e. g., polioviruses, rhinoviruses, etc.); Caliciviridae;<br>
Togaviridae (e. g., rubella virus, dengue virus, etc.); Flaviviridae; Coronaviridae;<br>
Reoviridae (e. g., rotavirus, etc.); Bimaviridae; Rhabodoviridae (e. g., rabies virus, etc.);<br>
Orthomyxoviridae (e. g., influenza virus types A, B and C, etc.); Filoviridae;<br>
Paramyxoviridae (e. g., mumps virus, measles virus, respiratory syncytial virus,<br>
parainfluenza virus, etc.); Bunyaviridae; Arenaviridae; Retrovuradae (e. g., HTLV-I;<br>
HTLV-11; HIV-1 (also known as HTLV-111, LAV, ARV, hTXR, etc.)), including but not<br>
limited to antigens from the isolates HIVI1lb, HTVSF2, HTVXAV, HTVLAI, HTVMN);<br><br>
HIV-1CM235, HIV-1; HIV-2, among others; simian immunodeficiency virus (SIV);<br>
Papillomavirus, the tick-borne encephalitis viruses; and the like. See, e. g. Virology, 3rd<br>
Edition (W. K. Joklik ed. 1988); Fundamental Virology, 2.sup.nd Edition (B. N. Fields and<br>
D. M. Knipe, eds. 1991), for a description of these and other viruses.<br>
[0109] In some alternative embodiments, the antigen is derived from bacterial<br>
pathogens such as Mycobacterium, Bacillus, Yersinia, Salmonella, Neisseria, Borrelia (for<br>
example, OspA or OspB or derivatives thereof), Chlamydia, or Bordetella (for example,<br>
P.69, PT and FHA), or derived from parasites such as plasmodium or Toxoplasma. In one<br>
embodiment, the antigen is derived from Mycobacterium tuberculosis (e.g. ESAT-6, 85A,<br>
85B, 72F), Bacillus anthracis (e.g. PA), or Yersiniapes-tis (e.g. F1, V). In addition,<br>
antigens suitable for use in the present invention can be obtained or derived from known<br>
causative agents responsible for diseases including, but not limited to, Diptheria, Pertussis,<br>
Tetanus, Tuberculosis, Bacterial or Fungal Pneumonia, Otitis Media, Gonorrhea, Cholera,<br>
Typhoid, Meningitis, Mononucleosis, Plague, Shigellosis or Salmonellosis, Legionaire's<br>
Disease, Lyme Disease, Leprosy, Malaria, Hookworm, Onchocerciasis, Schistosomiasis,<br>
Trypanosonuasis, Leislunaniasis, Giardia, Amoebiasis, Filariasis, Borelia, and Tricrainosis.<br>
[0110] The microbes of the invention may be altered to contain a heterologous<br>
nucleic acid sequence encoding an autoimmune disease-specific antigen. In a T cell<br>
mediated autoimmune disease, a T cell response to self antigens results in the autoimmune<br>
disease. The type of antigen for use in treating an autoimmune disease with the vaccines of<br>
the present invention might target the specific T cells responsible for the autoimmune<br>
response. For example, the antigen may be part of a T cell receptor, the idiotype, specific<br>
to those T cells causing an autoimmune response, wherein the antigen incorporated into a<br>
vaccine of the invention would elicit an immune response specific to those T cells causing<br>
the autoimmune response. Eliminating those T cells would be the therapeutic mechanism<br>
to alleviating the autoimmune disease. Another possibility would be to incorporate an<br>
antigen that will result in an immune response targeting the antibodies that are generated to<br>
self antigens in an autoimmune disease or targeting the specific B cell clones that secrete<br>
the antibodies. For example, an idiotype antigen may be incorporated into the microbe that<br>
will result in an anti-idiotype immune response to such B cells and/or the antibodies<br>
reacting with self antigens in an autoimmune disease. Autoimmune diseases that may be<br>
treatable with vaccine microbes of the present invention, include, but are not limited to,<br>
rheumatoid arthritis, multiple sclerosis, Crohn's disease, lupus, myasthenia gravis, vitiligo,<br>
scleroderma, psoriasis, pemphigus vulgaris, fibromyalgia, colitis and diabetes. A similar<br><br>
approach, may be taken for treating allergic responses, where the antigens incorporated into<br>
the vaccine microbe target either T cells, B cells or antibodies that are effective in<br>
modulating the allergic reaction. In some autoimmune diseases, such as psoriasis, the<br>
disease results in hyperproliferative cell growth with expression of antigeas that may be<br>
targeted as well. Such an antigen that will result in an immune response to&gt; the<br>
hyperproliferative cells is considered.<br>
[0111] Rather than targeting the malfunctioning cells of a disease, the microbes of<br>
the present invention comprise antigens that target unique disease associated protein<br>
structures. One example of this is the targeting of antibodies, B cells or T cells using<br>
idiotype antigens as discussed above. Another possibility is to target unique protein<br>
structures resulting from a particular disease. An example of this would be to incorporate<br>
an antigen that will generate an immune response to proteins that cause the amyloid<br>
plaques-observed in diseases such as-Alzheirner's disease, Creutzfeldt Jakob disease (CJD)<br>
and Bovine Spongiform Encephalopathy (BSE). While this approach may only provide for<br>
a reduction in plaque formation, it may be possible to provide a curative vaccine in the case<br>
of diseases like CJD. This disease is caused by an infectious form of a prion protein. The<br>
vaccine incorporates an antigen to the infectious form of the prion protein such that the<br>
immune response generated by the vaccine may eliminate, reduce, or control the infectious<br>
proteins that cause CJD.<br>
Microbes containing mutations<br>
[0112] In one embodiment, the invention includes a vaccine comprising a microbe<br>
wherein the nucleic acid of the microbe is modified so that the proliferation of the microbe<br>
is attenuated, wherein the microbial population is still able to express a desired antigen to<br>
an extent that is sufficient to elicit an immune response, and wherein the microbe is further<br>
attenuated by at least one genetic mutation. The mutation in the microbe may affect a<br>
variety of features of the microbe. In some cases, the mutation affects the ability of the<br>
microbe to invade certain cells. For example, certain intracellular bacteria can invade a<br>
variety of cell types depending on receptors present on the bacteria. The mutation may<br>
alter the expression of certain receptors so that the bacteria is taken up by some cell types<br>
but not others. As an example of this, Listeria is typically taken up by phaugocytic cells and<br>
also actively invades non-phagocytic cells (e.g. hepatic cells). A mutation, of Listeria may<br>
be used in which the invasion of non-phagocytic cells is significantly reduced or eliminated<br>
while the uptake by phagocytic cells is sufficiently active. Such a mutation may provide<br>
for a better immune response as the vaccine would be preferentially taken up by phagocytic<br><br>
cells, which are important in presenting the bacterial antigens to the immune system. It is<br>
understood that the mutation can be to any gene that results in an attenuation of the ability<br>
of the microbe to invade certain cell types, and that this is exemplified by mutations to<br>
internalin genes in Listeria (e.g. inlA, inlB). Similar genes may exist (e.g. invasin genes in<br>
Salmonella, Bacillus anthracis, and Yersinia) in other bacteria, and mutations in these<br>
genes are encompassed by the present invention. The mutation might impact other features<br>
of the microbe, such as a virulence factor or a gene that allows for growth and spreading,<br>
thereby reducing the virulence of the microbe. For example, a mutation in the actA gene of<br>
Listeria causes a deficiency in the polymerization of host cell actin, which inhibits the<br>
ability of the Listeria to spread, to other cells. A mutation in the hly gene of Listeria<br>
(listeriolysin (LLO) protein) impacts the ability of the Listeria to escape the phagolysosome<br>
of an infected cell. A mutation in either the plcA or plcB genes of Listeria impacts the<br>
ability of the Listeria to spread, from cell to cell. A mutation in the yop gene of Yersinia<br>
affects the ability of the Yersinia to prevent phagocytosis by macxophages. In another<br>
embodiment the genetic mutation attenuates the expression of certain antigens, for<br>
example, antigens that would normally result in an immune response to the microbe itself.<br>
Such a mutation may be useful if the microbe is used as a vaccine comprising a<br>
heterologous antigen in order to stimulate a strong immune response to the heterologous<br>
antigen but with a reduced immune response to the delivery microbe compared to the non-<br>
mutated microbe. In one embodiment, the microbe is attenuated by a mutation in more<br>
than one gene. In one embodiment, one of the mutations is in an internalin gene of Listeria<br>
or a similar gene in other bacteria. In one embodiment, the mutation is in one or more of<br>
an internalin gene of Listeria or similar gene in other bacteria. In one embodiment, one of<br>
the mutations is in the actA gene. In one embodiment, the microbe comprises Listeria<br>
monocytogenes with mutations in the actA gene and one or more internalin genes. In a<br>
preferred embodiment, the Listeria monocytogenes comprises a mutation in the actA gene<br>
and the inlB gene, preferably the Listeria monocytogenes comprises an actA/inlB deletion<br>
mutant (which is alternatively referred to herein as either AactAAZnlB or actA'inlET). The<br>
sequences of a variety of Listeria genes including those described herein are found in<br>
Genbank accession no. NC_00321O.<br>
[0113] The microbe might contain a mutation that significantly reduces the ability<br>
of the microbe to repair modifications to their nucleic acid. Such a mutation could be in<br>
any of a variety of genes that are involved in the DNA repair mechanisms of microbes<br><br>
[Aravind et al., Nucleic Acids Research 27(5):1223-1242 (1999)]. Microbes that are<br>
deficient in their ability to repair damage to their nacleic acid provide an added level of<br>
safety and efficacy to the use of the microbes of the present invention. Using the<br>
appropriate repair deficient mutants, the microbes are exquisitely sensitive to nucleic acid<br>
modification. The micleic acid of the microbes may be modified to a lesser degree yet still<br>
ensure the desired amount of attenuation of proliferation. This provides a larger Avindow of<br>
efficacy in which to operate so that the expression of the microbial nucleic acid is sufficient<br>
to generate the desired proteins. In the case where cie novo antigen expression is required,<br>
this provides a vaccine that will elicit an effective immune response. It also provides an<br>
added level of safety as the level of attenuation of proliferation achieved can not be<br>
compromised by repair of the modified nucleic acid. In another embodiment, the genetic<br>
mutation alters the susceptibility of the microbe to treatment with a nucleic acid targeted<br>
compound; for example by altering the permeability of thermicrobe to the compound or by<br>
altering the ability of the compound to access and bind the microbial nucleic acid. Such<br>
mutations may also impact the efficacy of the process of attenuating proliferation, while<br>
leaving microbial gene expression substantially unaffected.<br>
[0114] To illustrate the advantages of using a repair deficient mutant, one can<br>
consider the mechanism of the attenuation of microbial proliferation. The microbial<br>
nucleic acid is modified either by strand breakage or pyrimidine dimers, or by chemical<br>
modifications such as monoadducts or crosslinks. If the mechanisms for repair of these<br>
modifications are intact, a certain number of modifications will be required in order to<br>
achieve sufficient attenuation of proliferation. The greater the modification of nucleic acid,<br>
the greater the reduction in protein expression. Even, though the levels of modification<br>
required to attenuate proliferation are much lower than the levels required to stop protein<br>
expression, protein expression will still be reduced to some extent, possibly to an<br>
unacceptable level. The use of repair deficient mutants significantly reduces the levels<br>
needed to attenuate proliferation such that a lower nrodification level will result in adequate<br>
attenuation of proliferation. Since the nucleic acid modification is much lower, the<br>
expression of proteins will be less affected, providing for a higher level of expression of the<br>
protein of interest. Such repair deficient mutants may be particularly useful in the<br>
preparation of vaccines, such as vaccines to the microbe itself, where the safety of the<br>
vaccine can be increased by a slight modification of "the nucleic acid, leaving a sufficiently<br>
high level of protein expression, in particular the antigen to which the immune response is<br>
targeted. In one embodiment the repair deficient mutant lacks the ability to make PhrB (a<br><br>
photolyase), which repairs pyrimidine dimers. For example, the mutation may be in the<br>
phrB gene, or a functionally equivalent gene, depending on the genus and species of the<br>
microbe. Such a mutant could be used in conjuction with ultraviolet irradiation (e.g. UVB,<br>
UVC) of the microbe to produce pyrimidine dimers in the microbial nucleic acid. In one<br>
embodiment the repair deficient mutant is unable to repair interstand crosslinks. Such<br>
mutants include, but are not limited to, mutations in uvr genes, i.e. uvrA, uvrB, uvrC, and<br>
uvrD genes as well as recA genes, or functionally equivalent genes, depending on the<br>
genus and species of the microbe. The mutations may be in one or more of these genes.<br>
These mutations result in attenuation in the activity of the corresponding enzymes UvrA<br>
(an ATPase), UvrB (a helicase), UvrC (a nuclease), UvrD (a helicase II) and RecA (a<br>
recombinase). These mutants would be used in conjunction with a crosslinking compound,<br>
such as a psoralen. Since the microbial nucleic acid is crosslinked in some locations, and<br>
these Crosslinks can not be repaired, the microbe is unable to replicate as the original<br>
strands of nucleic acid can not be separated. Since they can not be repaired, very few<br>
crosslinks are needed, the microbial nucleic acid is for the most part accessible for<br>
transcription, and protein expression is not altered significantly. In a preferred<br>
embodiment, a population of repair deficient microbial mutants that are unable to repair<br>
interstrand crosslinks are suitably crosslinked such that essentially every microbe in the<br>
population contains at least one crosslink, such that attenuation of replication, is essentially<br>
complete, wherein the microbial gene expression of the population is sufficiently active. In<br>
one embodiment, a mutation in the recA gene is a conditional mutation. In such a<br>
mutation, the mutation in the recA gene results in the attenuation in the activity of recA<br>
only under certain conditions (i.e. non-permissive conditions), such as a suitable pH or<br>
temperature of the microbial population. A. microbe comprising a conditional recA<br>
mutation can be cultured under permissive conditions in order to grow sufficient levels of<br>
the microbe and then placed under non-permissive conditions for treatment to modify the<br>
nucleic acid, then stored under non-permissive conditions such that the nucleic acid<br>
damage is not adequately repaired. As an example of this, a recA temperature sensitive<br>
mutant is grown at 30 °C, where it grows well, and is treated to modify the nucleic acid at<br>
42 °C, which is non-permissive for recA such that it is very sensitive to treatment, such as<br>
psoralen crosslinking. While the treated microbe may be stored under non-permissive<br>
conditions, it is possible that upon vaccination, the conditions may permit expression of<br>
recA, resulting in some repair and presenting a safety issue. It is possible to construct the<br>
microbe such that the recA is under the control of the lac repressor, such that growth of the<br><br>
strain can be induced by isopropyl-?-D-tbiogalactopyranoside (IPTG) when growth is<br>
desirable, prior to the inactivation. and/or immunization steps. The possibility of recA<br>
expression can then be eliminated for the inactivation and/or immunization steps by<br>
withholding further IPTG from the strain and/or eliminating IPTG from, the strain's<br>
environment.<br>
[0115] In one embodiment, the microbe comprises at least one mutation that<br>
significantly reduces the ability of the microbe to repair modifications to their nucleic acid<br>
in combination with at least one mutation not related to repair mechanisms. The mutation<br>
that is not related to repair mechanisms may affect a variety of features of the microbe,<br>
such as the ability of the microbe to invade certain cells, a mutation in a virulence factor or<br>
a gene that allows for growth and spreading, or a mutation that attenuates the expression of<br>
certain antigens. Such mutations are discussed above and include, but axe not limited to,<br>
mutations in internalin genes (e.g. inlB), actA gene, hty gene, pIcA gene, or pleB gene of<br>
Listeria, invasion genes (e.g. Salmonella, Bacillus anthracis, and Yersinid) or the yop gene<br>
of Yersinia. In one embodiment, the microbe comprises Listeria monocytogenes having a<br>
mutation in the actA gene. In one embodiment, the Listeria monocytogenes comprises a<br>
mutation in the actA gene and in an internalin gene. In one embodiment, the Listeria<br>
monocytogenes comprises an actA mutation and a uvrAB mutation, preferably actA/uvrAB<br>
deletion mutations (which may be referred to as either AactAAuvrAB or actA'uvrABT). In<br>
one embodiment, fixe Listeria monocytogenes comprises an actA mutation, an inlB<br>
mutation, and a uvrAB mutation, preferably actA/inlB/uvrAB deletion mutations. In some<br>
other embodiments, the microbe comprises Bacillus anthracis having a uvrAB mutation,<br>
such as a deletion.<br>
[0116] In another embodiment, the invention provides an isolated mutant Listeria<br>
strain, such as a mutant Listeria monocytogenes strain, comprising a genetic mutation that<br>
attenuates its ability to repair its nucleic acid. In some embodiments, the mutant Listeria<br>
strain is defective with respect to at least one DNA repair enzyme (such asUvrA and/or<br>
UvrB). In some embodiments, the mutant Listeria strain comprises a genetic mutation in<br>
the uvrA gene and/or the uvrB gene. In some embodiments, the mutant strain is the Listeria<br>
monocytogenes AactAAuvrAB strain deposited with the American Type Culture Collection<br>
(ATCC) and identified by accession number PTA-5563. In other embodiments, strain is a<br>
mutant of the Listeria monocytogenes AactAAuvrAB strain deposited with the American<br><br>
Type Culture Collection (ATCC) and identified by accession number PTA-5563, wherein<br>
the mutant of the deposited strain is defective with respect to UvrA, UvrB, and ActA.<br>
[0117] In some embodiments, the invention provides a free-living microbe which is<br>
defective with respect to at least one DNA repair enzyme (relative to wild type). In some<br>
embodiments, the microbe that is defective with respect to at least one DNA repair enzyme<br>
is attenuated for DNA repair relative to wild type. In some embodiments, the capacity of<br>
th.e microbe for DNA repair is reduced by at least about 10%, at least about 25%, at least<br>
about 50%, at least about 75%, or at least about 90% relative to wild type. Methods for<br>
assessing the ability of a microbe to effect DNA repair are well known to those of ordinary<br>
skill in the art In some embodiments, the microbe is defective with respect to one or more<br>
of the following enzymes: PhrB, UvrA, UvrB, UvrC, UvrD, and RecA. In some<br>
embodiments, the microbe is defective with respect to UvrA, UvrB, or both enzymes. In<br>
some embodiments, the microbe is defective with respect to RecA, or a functional<br>
equivalent of Rec A. In some embodiments, the microbe comprise a genetic mutation in<br>
one or more gene selected from the group consisting of phrB, uvrA, uvrB, uvrC, uvrD and<br>
recA, or in a functional equivalent of one or more gene selected from the group consisting<br>
of phrB, uvrA, uvrB, uvrC, uvrD and recA. In some embodiments, the microbe comprises<br>
genetic mutations in both uvrA and uvrB, or in functional equivalents of both uvrA and<br>
uvrB. In some embodiments, the microbe comprises a genetic mutation in recA. In some<br>
embodiments, the microbe is a bacterium. For instance, in some embodiments, the microbe<br>
is Mycobacterium tuberculosis, Listeria monocytogenes, or Bacillus anthracis.<br>
[0118] The invention also provides an isolated mutant Listeria monocytogenes<br>
strain, comprising a genetic mutation that attenuates its ability to repair its nucleic acid. In<br>
some embodiments, the mutant strain is defective with respect to at least one DNA repair<br>
enzyme (such asUvrA and/or UvrB). In some embodiments, the mutant strain comprises a<br>
genetic mutation in the uvrA gene and/or the uvrB gene. In some embodiments, the uvrA<br>
gene, the uvrB gene, or both genes are deleted. In some embodiments, the mutant strain is<br>
attenuated with respect to RecA. In some embodiments, the mutant strain comprises a<br>
genetic mutation in the recA gene. In some embodiments, the mutant microbe is the<br>
Listeria monocytogenes actA'/uvrAB' strain deposited with the American Type Culture<br>
Collection (ATCC) and identified by accession mimber PTA-5563, or a mutant of the<br>
deposited strain which is defective with respect to UvrA, UvrB, and ActA.<br>
[0119] The invention also provides an isolated mutant Bacillus anthracis strain,<br>
comprising a genetic mutation that attenuates its ability to repair its nucleic acid. In some<br><br>
embodiments, the mutant strain is defective with respect to at least one DNA repair enzyme<br>
(such asUvrA and/or UvrB). In some embodiments, the mutant strain comprises a genetic<br>
mutation in the uvrA gene and/or the uvrB gene. In some embodiments, the uvrA gene<br>
(SEQ ID NO: 18), the uvrB gene (SEQ ID NO: 19), or both genes are deleted. In some<br>
embodiments, the mutant strain is attenuated with respect to RecA. In some embodiments,<br>
the mutant strain comprises a genetic mutation in the recA gene. In some embodiments, the<br>
mutant strain comprises a mutation in the recA gene that makes expression of the recA<br>
protein temperature sensitive. In some alternative embodiments, a mutant strain of B.<br>
anthracis is constructed which is under control of the lac repressor (inducible by IPTG),<br>
permitting expression of recA during growth, but not during inactivation (such as with S-<br>
59/UVA) and/or post-immunization. In some embodiments, the mutant strain comprises<br>
one or more mutations in the lef gene, cya gene, or both genes, that decreases the toxicity of<br>
the strain.<br>
[0120] As with any microbe of the invention, the modification of the DNA of the<br>
repair deficient (e. g. uvr deficient) bacteria with, psoralen can be controlled by adjusting the<br>
compound concentration, reaction conditions and light dose. The appropriate<br>
concentration, reaction conditions and light dose are determined by assessing their effects<br>
on replication and protein expression as detailed above. The use of repair deficient mutants<br>
provides an additional level of control of proliferation while maintaining adequate protein<br>
expression such that the parameters of concentration, reaction conditions and light dose can<br>
be adjusted over a wider range of conditions to provide a suitable population of microbes.<br>
For example, there will be a broader range of nucleic acid modification density over which<br>
proliferation can be completely inhibited without significantly affecting protein expression.<br>
The minimum level of modification required to completely inhibit repair deficient strains is<br>
much less than for non-repair deficient strains (see Examples 3, 7, 11, and 21). As a result,<br>
the modification level can be higher than the minimum level required to stop proliferation<br>
(ensuring complete inactivation) yet still be below a level that is detrimental to protein<br>
expression. Thus, while the invention is effective for non-repair deficient strains, uvr<br>
deficient strains provide greater flexibility in preparing a desirable population of microbes<br>
that would be effective as a vaccine. Psoralen compounds are effective at concentrations of<br>
about 10 pM to 10 mM, also about 100 pM to 1 mM, also about 1 nM to 10 ?M, also about<br>
1-500 nM, also about 1-200 nM or about 1-100 nM, with the UVA light dose ranging from<br>
about 0.1 - 100 J/cm2, also about 0.1-20 J/cm2, also about 0.5-10 J/cm2, or about 0.5-6<br><br>
J/cm2 or about 2-6 J/cm2 . In one embodiment, the microbe is treated in the presence of<br>
growth media atpsoralen concentrations of" about 10 pM to 10 mM, also about 1-5000 nM,<br>
also about 1-500 nM, also about 5-500 nM, or about 10-400 nM. In one embodiment, the<br>
microbe treated in the presence of growth media is grown to an OD of 0.5-1 in the presence<br>
of psoralen at concentrations of about 10 pM to 10 mM, also about 1-5000 nM, also about<br>
1-500 nM, also about 5-500 nM, or about 10-400 nM. Following the growth to an OD of<br>
0.5-1, the microbe population is irradiated with UVA light at a dose ranging from about 0.1<br>
- 100 J/cm2, also about 0.1-20 J/cm2, or about 0.5-10 J/cm2, 0.5-6 J/cm2 or about 2-6 J/ cm2.<br>
[0121] In order to generate primarily psoralen crosslinks in any microbe,<br>
particularly uvr deficient mutant bacteria, it is possible to dose the psoralen and UVA light<br>
initially to form adducts and follow this with a second dose of UVA light alone to convert<br>
some or most of the monoadducts to crosslinks. The psoralen photochemistry is such that<br>
absorption of a photon of appropriate energy will first forma monoadduct. Absorption of<br>
an additional photon will convert this monoadduct to a crosslink when a furan side<br>
monoadduct is appropriately situated in the DNA double helix [Tessman et al.,<br>
Biochemistry 24:1669-1676 (1985)]. The sample can be dosed with a lower UVA dose at a<br>
desired concentration of psoralen and the unreacted psoralen can be removed, e.g. by<br>
washing, dialysis or ultrafiltration of the bacteria. The bacteria containing psoralen adducts<br>
(monoadducts and crosslinks) can be further dosed with UVA light to convert some or most<br>
of the monoadducts to crosslinks without resulting in significant additional adducts to the<br>
bacteria. This allows for the controlled addition of a low number of psoralen adducts with<br>
the initial light dose, then converting a substantial number of any monoad-ducts to crosslink<br>
with the second dose. This provides for mo dification of the microbial genome at<br>
sufficiently low levels wherein a majority of the adducts formed will be crosslinks. This is<br>
particularly effective for blocking replication with uvr deficient mutants. In such<br>
embodiments, psoralen compounds are effective at concentrations of about 10 pM to 10<br>
mM, also about 100 pM to 1 mM, also about 1-500 nM, also about 1-200 nM or about 1-<br>
100 nM, with the UVA light dose ranging from about 0.1 - 10 J/cm2, also about 0.1-2<br>
J/cm2, or about 0.5-2 J/cm2. Following removal of most of the unreacted psoralen by<br>
washing, dialysis or ultrafiltration of the bacteria, the bacteria may be dosed with UVA<br>
light ranging from 0.1 - 100 J/cm2, also about 0.1-20 J/cm2, or about 0.5-10 J/cm2 or about<br>
2-6 J/cm2.<br>
Vaccine compositions and in vivo efficacy<br><br>
[0122] Vaccine compositions of the invention comprise a microbe in which the<br>
microbial nucleic acid is modified and/or comprise an antigen-presenting cell which has<br>
been antigen-loaded and/or activated/matured by infection with a microbe in which the<br>
microbial nucleic acid is modified so that the proliferation of the microbe is attenuated,<br>
wherein the microbial gene expression is substantially unaffected, as discussed above. The<br>
vaccine compositions of the present invention can be used to stimulate an immune response<br>
in an individual. The formulations can be administered to an individual by a variety of<br>
administration routes. Methods of administration of such, a vaccine composition are known<br>
in the art, and include oral, nasal, intraveneous, intradennal, intraperitoneal, intramuscular,<br>
intralymphatic and subcutaneous routes of administration. The vaccine compositions may<br>
further comprise additional components known in the art to improve the immune response<br>
to a vaccine, such as adjuvants or T cell co-stimulatory molecules. The invention also<br>
includes medicamentscomprising the pharmaceutical compositions of the invention. An<br>
individual to be treated with such vaccines, is any vertebrate, preferably a mammal,<br>
including domestic annuals, sport animals, and primates, including humans. The vaccine<br>
may be administered as a prophylactic, where the individual is vaccinated in order to<br>
immunize the individual against a particular disease. While the vaccine can be given to any<br>
individual, in some instances, such as with cancer vaccines, the individual treated might be<br>
limited to those individuals at higher risk of developing a cancer. The vaccine may also be<br>
administered as a therapeutic, where the individual having a. particular disease is vaccinated<br>
in order to improve the immune response to the disease or a disease related protein. In this<br>
embodiment, the vaccine may result in a lessening of the physical symptoms associated<br>
with the disease. For example, with cancer vaccines, the vaccination may result in stopping<br>
the growth of a tumor, preferably a lessening of the mean tumor volume, more preferably<br>
elimination of any tumors. In one embodiment, the mean tumor volume decreases by at<br>
least about 5%, also about 10%, also about 25%, also about 50%, also about 75%, also<br>
about 90% or about 100%. Similarly, the vaccination may result in stopping the metastases<br>
of a tumor, preferably resulting in a reduction in the number of tumor metastases. An<br>
additional effect of a cancer vaccine would be an extension of the median survival of the<br>
individual. In humans, the median survival may be extended by at least about 3 months,<br>
also at least about 6 months, or at least about 12 months.<br>
[0123] Vaccine formulations are known in the art and include numerous additives,<br>
such as preservatives, stabilizers, adjuvants, antibiotics, and other substances.<br>
Preservatives, such as thinxerosal or 2-phenoxy ethanol, are added to slow or stop the<br><br>
growth of bacteria or fungi resulting from inadvertent contamination, especially as might<br>
occur with vaccine vials intended for multiple uses or doses. Stabilizers, such as lactose or<br>
monosodium glutamate (MSG), are added to stabilize the vaccine formulation against a<br>
variety of conditions, such as temperature variations or a freeze-drying process. Adjuvants,<br>
such as aluminum hydroxide or aluminum phosphate, are added to increase the ability of<br>
the vaccine to trigger, enhance, or prolong an immune response. Additional materials, such<br>
as cytokines, chemokines, and bacterial nucleic acid sequences, like CpG, are also potential<br>
vaccine adjuvants. Antibiotics, such as neomycin and streptomycin, are added to prevent<br>
the potentially harmful growth of germs. Vaccines may also include a suspending fluid<br>
such as sterile water or saline. Vaccines may also contain small amounts of residual<br>
materials from the manufacturing process, such as cell or bacterial proteins, egg proteins<br>
(from vaccines that are produced in eggs), DNA or UNA, or formaldehyde from a<br>
toxoiding process;<br>
[0124] The efficacy of the vaccines can be evaluated in an individual, for example<br>
in mice. A mouse model is recognized as a model for efficacy in humans and is useful in<br>
assessing and defining the vaccines of the present invention. The mouse model is used to<br>
demonstrate the potential for the effectiveness of the vaccines in any individual. Vaccines<br>
can be evaluated for their ability to provide either a prophylactic or therapeutic effect<br>
against a particular disease. For example, in the case of infectious diseases, a population of<br>
mice can be vaccinated with a desired amount of the appropriate vaccine of the invention,<br>
where the microbe expresses an infectious disease associated antigen. This antigen can be<br>
from the delivery microbe itself or can be a heterologous antigen. The mice can be<br>
subsequently infected with the infectious agent related to the vaccine antigen and assessed<br>
for protection, against infection. The progression of the infectious disease can be observed<br>
relative to a control population (either non vaccinated or vaccinated with vehicle only or a<br>
microbe that does not contain the appropriate antigen).<br>
[0125] In the case of cancer vaccines, tumor cell models are available, where a<br>
tumor cell line expressing a desired tumor antigen can be injected into a population of mice<br>
either before (therapeutic model) or after (prophylactic model) vaccination with a microbe<br>
of the invention containing the desired tumor antigen. Vaccination with a microbe<br>
containing the tumor antigen can be compared to control populations that are either not<br>
vaccinated, vaccinated with vehicle, or with a microbe that expresses an irrelevant antigen.<br>
In addition, the relative efficacy of the vaccines of the invention can be compared to a<br>
population of microbe in which the microbial nucleic acid has not been modified. The<br><br>
effectiveness of the vaccine in such models can be evaluated in terms of tumor volume as a<br>
function of time after tumor injection or in terms of survival populations as a function of<br>
time after tumor injection (e.g. Example 4). In one embodiment, the tumor volume in mice<br>
vaccinated with nucleic acid modified microbe is about 5%, about 10%, about 25%, about<br>
50%, about 75%, about 90% or about 100% less than the tumor volume in mice that are<br>
either not vaccinated or are vaccinated with vehicle or a microbe that expresses an<br>
irrevelant antigen. In another embodiment, this differential in tumor volume is observed at<br>
least about 10, about 17, or about 24 days following the implant of the tumors into the<br>
mice. In one embodiment, the median, survival time in the mice vaccinated with nucleic<br>
acid modified microbe is at least about 2, about 5, about 7 or at least about 10 days longer<br>
than in mice that are either not vaccinated or are vaccinated with vehicle or a microbe that<br>
expresses an irrelevant antigen. In addition to an effective immune response to the<br>
vaccines of the present invention, the modified microbes provide an addled level of safety<br>
such that a higher dose of the microbe may be administered relative to the corresponding<br>
unmodified microbe. In one embodiment of the invention, the vaccination with the nucleic<br>
acid modified microbe is done at a dose of microbes that is the same as the dose of the<br>
corresponding unmodified microbe. In another embodiment; the vaccination of nucleic<br>
acid modified microbe is safely dosed at a level that is at least about 2, about 5, about 10,<br>
about 102, about 103, or at least about 104 fold higher than the vaccination dose of the<br>
corresponding unmodified microbe, wherein the resulting tumor volume and median<br>
survival times discussed above are observed for the nucleic acid modified microbe.<br>
Methods of use<br>
[0126] A variety of methods of using the modified microbes, antigen-presenting<br>
cells, vaccines, and pharmaceutical compositions described herein are provided by the<br>
present invention. For instance, methods of using the modified microbes, antigen-<br>
presenting cells, vaccines, and pharmaceutical compositions described herein to induce<br>
immune responses and/or to treat or prevent disease are provided. Method of using the<br>
modified microbes and/or mutant strain to prepare vaccines and other compositions are also<br>
provided.<br>
[0127] For instance, in one aspect, the invention provides a method of inducing an<br>
immune response in a host to an antigen, comprising administering to the host an effective<br>
amount of a composition comprising a free-living microbe that expresses the antigen,<br>
wherein the nucleic acid of the microbe is modified so that the microbe is attenuated for<br><br>
proliferation. In some embodiments, the composition comprising the microbe is a vaccine.<br>
In some embodiments, the composition comprising the microbe is a professional antigen-<br>
presenting cell. The antigen may be heterologous or autologous to the microbe as<br>
described above. In some embodiments, the nucleic acid of the microbe has been modified<br>
by reaction with a nucleic acid targeted compound that reacts directly with the nucleic acid.<br>
[0128] The invention also provides a method of inducing an immune response in a<br>
host to an antigen, comprising administering to the host an effective amount of a<br>
composition comprising a mutant strain of Listeria monocytogenes that expresses the<br>
antigen, wherein the mutant strain comprises a genetic mutation that attenuates its ability to<br>
repair its nucleic acid. The antigen may be a Listerial or non-Listerial antigen. In some<br>
embodiments, the nucleic acid of the Listeria has been modified so that the microbe is<br>
attenuated for proliferation (e.g., by S-59/UVA treatment).<br>
[0129] The invention also provides a method of inducing an immune response in a<br>
host to an antigen, comprising administering to the host an effective amount of a<br>
composition comprising a mutant strain of Bacillus anthracis that expresses the antigen,<br>
wherein the mutant strain comprises a genetic mutation that attenuates its ability to repair<br>
its nucleic acid. In some embodiments, the nucleic acid of the Bacillus has been modified<br>
so that the microbe is attenuated for proliferation (e.g., by S-59/UVA treatment).<br>
[0130] The invention also provides a method of preventing or treating a disease in a<br>
host, comprising administering to the host an effective amount of a composition comprising<br>
a free-living microbe, wherein the nucleic acid of the microbe is modified so that the<br>
microbe is attenuated for proliferation. In some embodiments, the composition comprising<br>
the microbe is a vaccine. In some embodiments, the composition comprising the microbe<br>
is a professional antigen-presenting cell.<br>
[0131] The invention also provides a method of preventing or treating a disease in a<br>
host, comprising administering to the host an effective amount of a composition comprising<br>
a mutant strain of Listeria monocytogenes, wherein the mutant strain comprises a genetic<br>
mutation that attenuates its ability to repair its nucleic acid. In some embodiments, the<br>
nucleic acid of the Listeria has been modified so that the microbe is attenuated for<br>
proliferation (e.g., by S-59/UVA treatment). In some embodiments, the disease is an<br>
infectious disease. In other embodiments, the disease is cancer.<br>
[0132] The invention also provides a method of preventing or treating disease in a<br>
host, comprising administering to the host an effective amount of a composition comprising<br>
a mutant strain of Bacillus anthracis, wherein the mutant strain comprises a genetic<br><br>
mutation that attenuates its ability to repair its nucleic acid. In some embodiments, the<br>
nucleic acid of the Bacillus has been modified so that the microbe is attenuated for<br>
proliferation (e.g., by S-59/UVA treatment).<br>
[0133] The invention also provides a free-living microbe for medical use, wherein<br>
the nucleic acid of the microbe is modified so that the microbe is attenuated for<br>
proliferation and/or the microbe is defective with respect to a DNA repair enzyme. It is<br>
understood, that medical use encompasses both therapeutic and preventative medical<br>
applications (e.g., for use as a vaccine). In some embodiments, the microbe has been<br>
modified by reaction with a nucleic acid targeted compound that reacts directly with the<br>
nucleic acid so that the microbe is attenuated for proliferation. In some embodiments, tfcie<br>
microbe is Listeria monocytogenes ox Bacillus anthr cis.<br>
[0134] In other aspects, the invention provides a professional antigen-presenting<br>
cell for medical Use, wherein the antigen-presenting-cell comprises a free-living microbe,<br>
wherein the nucleic acid of the microbe is modified so that the microbe is attenuated for<br>
proliferation and/or the microbe is defective with respect to a DNA repair enzyme. In some<br>
embodiments, the microbe has been modified by reaction with a nucleic acid targeted<br>
compound that reacts directly with the nucleic acid so that the microbe is attenuated for<br>
proliferation. In some embodiments, the microbe is Listeria monocytogenes or Bacillus<br>
anthracis.<br>
 [0135] The invention also provides a mutant Listeria monocytogenes strain for<br>
medical use, wherein the mutant Listeria monocytogenes strain comprises a genetic<br>
mutation that attenuates its ability to repair its nucleic acid.<br>
[0136] In addition, the invention provides a mutant Bacillus anthracis strain for<br>
medical use, wherein the mutant Bacillus anthracis strain comprises a genetic mutation that<br>
attenuates its ability to repair its nucleic acid.<br>
[0137] The invention further provides the use of a free-living microbe, wherein the<br>
nucleic acid has been modified so that the microbe is attenuated for proliferation, for the<br>
manufacture of a medicament for a disease unrelated, and/or not caused by the free-living<br>
microbe. In some embodiments, the disease is cancer. In some embodiments, the disease<br>
is an infectious disease unrelated to the free-living microbe.<br>
[0138] The invention further provides the use of a free-living microbe for the<br>
manufacture of a medicament for a disease unrelated, and/or not caused the microbe,<br>
wherein the microbe is defective with respect to at least one DNA repair enzyme. In some<br><br>
embodiments, the disease is cancer. In some embodiments, the disease is an infectious<br>
disease unrelated to the microbe.<br>
[0139] Additionally, the invention provides the use of a professional antigen-<br>
pxesenting cell, wherein the antigen-presenting cell comprises a free-living microbe,<br>
wherein the nucleic acid of the microbe is modified so that the microbe is attenuated for<br>
proliferation and/or wherein the microbe is defective with respect to at least one DNA<br>
repair enzyme, for the manufacture of a medicament for a disease unrelated and/or not<br>
caused by the free-living microbe. In some embodiments, the disease is cancer. In some<br>
embodiments, the disease is an infectious disease unrelated to the free-living microbe.<br>
[0140] The invention further provides the use of a mutant strain of Listeria<br>
monocytogenes, wherein the mutant Listeria monocytogenes strain comprises a genetic<br>
mutation that attenuates its ability to repair its nucleic acid, for the manufacture of a<br>
medicament for a disease" unrelated and/or not caused by Listeria monocytogenes. In some<br>
embodiments, the disease is cancer. In some embodiments, the disease is an infectious<br>
disease unrelated to the Listeria monocytogenes.<br>
[0141] In another aspect, the invention provides a method of activating naive T<br>
cells, comprising contacting the naive T cells with a professional antigen-presenting cell<br>
under suitable conditions and for a sufficient time to activate the naive T cells, wherein the<br>
antigen-presenting cell comprises a free-living microbe, wherein the nucleic acid of the<br>
microbe is modified so that the microbe is attenuated for proliferation. The contacting step<br>
of this method may be performed either in vitro or in vivo. Suitable conditions and a<br>
sufficient time for activating the naive T-cells would be known to one of ordinary skill of<br>
the art. In addition, examples of such conditions are provided in the specific Examples,<br>
below.<br>
[0142] A method of loading professional antigen-presenting cells with an antigen is<br>
also provided. The method comprises contacting the professional antigen-presenting cells<br>
with a free-living microbe that comprises a nucleic acid sequence encoding the antigen,<br>
under suitable conditions and for a sufficient time to load the professional antigen-<br>
presenting cells (e.g., dendritic cells), wherein the nucleic acid of the microbe is modified<br>
so that the microbe is attenuated for proliferation and/or the microbe is defective with<br>
respect to at least one DNA repair enzyme. The contacting step of the method may be<br>
performed in vitro, ex vivo, or in vivo. The antigen may be heterologous or autologous to<br>
the microbe. Suitable conditions and a sufficient time for loading antigen-presenting cells<br><br>
would generally be known to one of ordinary skill of the art In addition, examples of such<br>
conditions are provided in the specific Examples, below.<br>
[0143] In another aspect, the invention provides a method of activating and/or<br>
maturing professional antigen-presenting cells comprising contacting the professional<br>
antigen-presenting cells (in vitro, ex vivo, and/or in vivo) with a free-living microbe that<br>
comprises a nucleic acid sequence encoding an antigen, under suitable conditions and for a<br>
sufficient time to activate and/or bring to maturation the professional antigen-presenting<br>
cells, wherein the nucleic acid of the microbe is modified so that the microbe is attenuated<br>
for proliferation. The contacting step of the method may be performed either in vitro or in<br>
vivo. The antigen may be heterologous or autologous to the microbe. Suitable conditions<br>
and a sufficient time for activating antigen-presenting cells and/or bringing antigen-<br>
presenting cells to maturation would generally be known to one of ordinary skill of the art.<br>
In addition, examples of such conditions are provided in the specific Examples, below<br>
[0144] In another aspect, the invention provides a method of preventing or treating<br>
a disease in a host, comprising the following steps, (a) loading professional antigen-<br>
presenting cells with an antigen by contacting the cells with a free-living microbe that<br>
comprises a nucleic acid sequence encoding an antigen, wherein the nucleic acid of the<br>
microbe is modified so that the microbe is attenuated for proliferation; and (b)<br>
administering an effective amount of a composition comprising the loaded professional<br>
antigen-presenting cells to the host.<br>
[0145] In still another aspect, the invention provides a method of inducing an<br>
immune response to an antigen in a host, comprising the following steps, (a) loading<br>
professional antigen-presenting cells with the antigen by contacting the cells with a free-<br>
living microbe that comprises a nucleic acid sequence encoding the antigen, wherein the<br>
nucleic acid of the microbe is modified so that the microbe is attenuated for proliferation;<br>
and (b) administering an effective amount of a composition comprising the loaded<br>
professional antigen-presenting cells to the host.<br>
Kits<br>
[0146] The invention further provides kits (or articles of manufacture) comprising<br>
the modified microbes and mutant strains of the present invention.<br>
[0147] In one aspect, the invention provides a kit comprising both (a) a composition<br>
comprising a mutant Listeria monocytogenes strain comprising a genetic mutation that<br>
attenuates its ability to repair its nucleic acid, a mutant Bacillus anthracis strain comprising<br><br>
a genetic mutation that attenuates its ability to repair its nucleic acid, or a free-living<br>
microbe, wherein the nucleic acid of the microbe is modified so that the microbe is<br>
attenuated for proliferation; and (b) instructions for the use of the composition in the<br>
prevention or treatment of a disease in a host In some embodiments, the instructions are<br>
on a label. In other embodiments, the instructions are on an insert contained within the Hat.<br>
[0148] In another aspect, the invention provides a kit comprising both (a) a<br>
composition comprising a mutant Listeria monocytogenes strain comprising a genetic<br>
mutation that attenuates its ability to repair its nucleic acid, a mutant Bacillus anthracis<br>
strain comprising a genetic mutation that attenuates its ability to repair its nucleic acid, or a<br>
free-living microbe, wherein, the nucleic acid of the microbe is modified so that the microbe<br>
is attenuated for proliferation; and (b) instructions for the administration of the composition<br>
to a host. In some embodiments, the instructions are on a label. In other embodiments, the<br>
instructions are on an insert contained within the kit<br>
[0149] In another aspect, the invention provides a kit comprising both (a) a<br>
composition comprising a mutant Listeria monocytogenes strain comprising a genetic<br>
mutation that attenuates its ability to repair its nucleic acid, a mutant Bacillus anthracis -<br>
strain comprising a genetic mutation that attenuates its ability to repair its nucleic acid, or a<br>
free-living microbe, wherein, the nucleic acid of the microbe is modified so that the microbe<br>
is attenuated for proliferation; and (b) instructions for selecting a host to which the<br>
composition is to be administered. In some embodiments, the instructions are on a label.<br>
In other embodiments, the instructions are on an insert contained within the kit.<br>
[0150] In some embodiments of each of the aforementioned aspects, the<br>
composition is a vaccine. In some embodiments of each of the aforementioned aspects, the<br>
composition is a professional-antigen-presenting cell. In some embodiments of each of the<br>
aforementioned aspects, the nucleic acid of the free-living microbe has been modified by<br>
reaction with a nucleic acid targeted compound that reacts directly with the nucleic acid. In<br>
some embodiments, the microbe has been s-59/UVA treated. In some embodiments, the<br>
microbe is defective with respect to a DNA repair enzyme.<br>
Additional embodiments of the invention<br>
[0151] In one embodiment, the invention includes a vaccine composition<br>
comprising a free-living microbe in which the microbial nucleic acid is modified so that the<br>
proliferation of the microbe is attenuated and/or comprising an antigen-presenting cell that<br>
has been antigen-loaded and/or activated or matured through infection with a free-living<br><br>
microbe in which the microbial nucleic acid is modified so that the proliferation of the<br>
microbe is attenuated, wherein the microbial gene expression is substantially unaffected. In<br>
one embodiment, the microbial gene expression is substantially unaffected so that an<br>
antigen is expressed at a level sufficient to stimulate an immune response upon<br>
administration of the microbe to an individual tn one embodiment, the proliferation of the<br>
microbe is attenuated by at least about 0.3 log, also at least about 1 log, about 2 log, about 3<br>
log, about 4 log, about 6 log, or at least about 8 log. In another embodiment, the<br>
proliferation of the microbe is attenuated by about 0.3 to &gt; 10 log, about 2 to &gt; 10 log, about<br>
4 to &gt;10 log, about 6 to &gt;10 log, about 0.3-8 log, about 0.3-6 log, about 0.3-5 log, about 1-5<br>
log, or about 2-5 log. In one embodiment, the expression of an antigen by the microbe is at<br>
least about 10%, about 25%, about 50%, about 75%, or at least about 90% of the<br>
expression of the antigen by a microbe in which the microbial nucleic acid is not modified.<br>
In one embodiment, the antigen expressed is an antigen from the microbe itself. In one<br>
embodiment, the microbe comprises a heterologo us nucleic acid sequence encoding an<br>
antigen. In one embodiment, the antigen is a disease associated antigen. In one<br>
embodiment, the antigen is associated with a disease selected from the group consisting of<br>
infectious diseases, autoimmune diseases, allergies, cancers, and other hyperproliferative<br>
diseases. In one embodiment, the antigen is a tumor associated antigen. In one<br>
embodiment, the tumor antigen is selected from the group consisting of differentiation<br>
antigens, tissue-specific antigens, developmental antigens, tumor-associated viral antigens,<br>
cancer-testis antigens, embryonic antigens, oncoprotein antigens, over-expressed protein<br>
antigens and mutated protein antigens. In one embodiment, the tumor antigen is selected<br>
from the group consisting of mesothelin, Spl7, gp100,, PR3, PAGE-4, TARP, WT-1, NY-<br>
ESO-1 and SPAS-1. In one embodiment, the micarobial nucleic acid is modified, by a<br>
method selected from the group consisting of exposing the microbe to radiation and<br>
reacting the microbe with a nucleic acid targeted compound that causes the modification of<br>
the microbial nucleic acid. In a preferred embodiment, the microbial nucleic acid is<br>
modified by reacting the microbe with a nucleic aoid targeted compound that reacts directly<br>
with the nucleic acid. In one embodiment, the nucleic acid targeted compound is targeted<br>
to the nucleic acid by a mode selected from the group consisting of intercalation, minor<br>
groove binding, major groove binding, electrostatic binding, and sequence-specific binding.<br>
In one embodiment, the nucleic acid targeted compound comprises a nucleic acid allylator.<br>
In a preferred embodiment, the nucleic acid targeted compound is ?-alanine, N-(acridin-9-<br><br>
yl), 2-[bis(2-chloroethyl)amino]ethyl ester. In one embodiment, the nucleic acid targeted<br>
compound that reacts directly with the nucleic acid reacts upon activation of the compound<br>
by irradiation, preferably by UVA irradiation. In one embodiment, the nucleic acid<br>
targeted compound activated by UVA irradiation is a psoralen. In a preferred embodiment,<br>
the psoralen is 4'-(4-amino-2-oxa)butyl-4,5',8-trimethylpsoralen. In orxe embodiment, the<br>
nucleic acid targeted compound indirectly causes the modification of the nucleic acid. In<br>
one embodiment, the nucleic acid targeted compound indirectly causes modification upon<br>
activation by irradiation, preferably by UVA irradiation. In one embodiment, the microbe<br>
comprises a genetic mutation. In one embodiment, the genetic mutation results in the<br>
attenuation of the ability of the microbe to repair microbial nucleic acid, that has been<br>
modified. In one embodiment, the genetic mutation is in the gene selected from the group<br>
consisting of phrB, uvrA, wvrB, uvrC, uvrD and recA, or their functionally equivalent<br>
genes, depending on the genus and species of the microbe. In one embodiment, the<br>
mutation is in one or more of the genes selected from the group consisting of phrB, uvrA,<br>
wvrB, uvrC, uvrD and recA, or their functionally equivalent genes. In one embodiment, the<br>
genetic mutation results in the attenuation in the activity of a DNA repair enzyme selected<br>
from the group consisting of PhrB, UvrA, UvrB, UvrC, UvrD and RecA. In a further<br>
embodiment, the microbes containing these mutations are modified by reaction with a<br>
psoralen activated by UVA irradiation. In a preferred embodiment, the psoralen is 4'-(4-<br>
amino-2-oxa)butyl-4,5',8-trunethylpsoralen. In one embodiment, the microbe is selected<br>
from the group consisting of bacteria, protozoa and fungi. In one embodiment, the microbe<br>
is a bacteria. In one embodiment, the bacteria is an intracellular bacteria. In a preferred<br>
embodiment, the bacteria is a Listeria, preferably Listeria monocytogenes. In one<br>
embodiment, the Listeria comprises a mutation that results in the attenuation of the ability<br>
of the Listeria to invade non-phagocytic cells without significantly affecting the uptake of<br>
the Listeria by phagocytic cells. In one embodiment, the Listeria mutation is in an<br>
internalin gene(s). In one embodiment, the Listeria mutation is in the gene selected from<br>
the group consisting of inlA, inlB, and any gene encoding an internalin. In one<br>
embodiment, the Listeria monocytogenes comprises a genetic mutation in both the inlA and<br>
inlB genes. In one embodiment, Has Listeria comprises a mutation that results in the<br>
attenuation of the ability of the Listeria to escape the phagolysosome of an infected cell. In<br>
one embodiment, the Listeria mutation, is in the hly gene. In one embodiment, the Listeria<br>
comprises a mutation that results in the attenuation of the polymerization, of actin by the<br>
Listeria. In a preferred embodiment, the Listeria mutation is in the actA gene. In one<br><br>
embodiment, the Listeria comprises mutations in the actA gene and one or more internalin<br>
genes. In a preferred embodiment, the Listeria comprises a mutation in the actA gene and<br>
the inlB gene, preferably the Listeria comprises an actA/inlB deletion mutant In a<br>
preferred embodiment, the Listeria monocytogenes actA/inIB deletion mutant further<br>
comprises a deletion mutation in the uvrAB gene.<br>
[0152] In one embodiment, the invention includes a vaccine composition<br>
comprising a bacteria in which the bacterial nucleic acid is modified so that the<br>
proliferation of the bacteria is attenuated, wherein the bacterial gene expression is<br>
substantially unaffected. In one embodiment, the bacterial gene expression is substantially<br>
unaffected so that an antigen is expressed at a level sufficient to stimulate an immune<br>
response to the bacteria upon administration of the bacteria to an individual. In one<br>
embodiment, the proliferation of the bacteria is attenuated by at least about 0.3 log, also at<br>
-least about 1 log, about 2 log, about 3 log, about 4 log, about 6 log, or at least about 8 log.<br>
In another embodiment, the proliferation of the microbe is attenuated by about 0.3 to &gt; 10<br>
log, about 2 to &gt;10 log, about 4 to &gt;10 log, about 6 to &gt;10 log, about 0.3-8 log, about 0.3-6<br>
log, about 0.3-5 log, about 1-5 log, or about 2-5 log. In one embodiment, the expression of<br>
an antigen by the bacteria is at least about 10%, about 25%, about 50%, about 75%, or at<br>
least about 90% of the expression of the antigen by a bacteria in which the bacterial nucleic<br>
acid is not modified. In one embodiment, the bacterial nucleic acid is modified by a<br>
method selected from the group consisting of exposing the bacteria to radiation and<br>
reacting the bacteria with a nucleic acid targeted compound that causes the modification of<br>
the bacterial nucleic acid. In a preferred embodiment, the bacterial nucleic acid is modified<br>
by reacting the bacteria with a nucleic acid targeted compound that reacts directly with the<br>
nucleic acid. In one embodiment, the nucleic acid targeted compound is targeted to the<br>
nucleic acid by a mode selected from the group consisting of intercalation, minor groove<br>
binding, major groove binding, electrostatic binding, and sequence-specific binding. In one<br>
embodiment, the nucleic acid targeted compound comprises a nucleic acid alkylator. In a<br>
preferred embodiment, the nucleic acid targeted compound is ?-alanine, N-(acridin-9-yl),<br>
2-[bis(2-chloroethyl)amino] ethyl ester. In one embodiment, the nucleic acid targeted<br>
compound that reacts directly with the nucleic acid reacts upon activation of the compound<br>
by irradiation, preferably by UVA irradiation. In one embodiment; the nucleic acid<br>
targeted compound activated by UVA irradiation is a psoralen. In a preferred embodiment,<br>
the psoralen is 4'-(4-amino-2-oxa)butyl-4,5',8-trimethylpsoralen. In one embodiment, the<br><br>
nucleic acid targeted compound indirectly causes the modification of the nucleic acid. In<br>
one embodiment, the nucleic acid targeted compound indirectly causes modification upon<br>
activation by irradiation, preferably by UVA irradiation. In one embodiment, the bacteria<br>
comprises a genetic mutation. In one embodiment, the genetic mutation results in the<br>
attenuation, of the ability of the bacteria to repair bacterial nucleic acid mat has been<br>
modified. In one embodiment, the genetic mutation is in the gene selected from the group<br>
consisting of phrB, uvrA, uvrB, uvrC, uvrD and recA, or their functionally equivalent<br>
genes, depending on the genus and species of the bacteria. In one embodiment, the<br>
mutation is in one or more of the genes selected from the group consisting of phrB, uvrA,<br>
uvrB, uvrC, uvrD and recA, or their functionally equivalent genes. In one embodiment, the<br>
genetic mutation results in the attenuation in the activity of a DNA repair enzyme selected<br>
from the group consisting of PhrB, UvrA, UvrB, UvrC, UvrD and RecA. In a preferred<br>
embodiment, the bacteria containing these mutations are modified by reaction with a<br>
psoralen activated by UVA irradiation. In a preferred embodiment, the psoralen is 4'-(4-<br>
aniino-2-oxa)butyl-4,5',8-trimemylpsoralen. In one embodiment, the bacteria is selected<br>
from the group consisting of Gram positive bacteria, Gram negative bacteria, intracellular<br>
bacteria and mycobacteria. In one embodiment, the bacteria is selected from the group<br>
consisting of Bacillus anthracis, Cholera, Bordetella pertussis, Corynebacterium<br>
diphtheriae, E. coli, Borrelia burgdorferi (Lyme), Streptococcus pneumoniae, Salmonella,<br>
Staphylococcus sp., Mycobacterium tuberculosis, Brucella abortus, Brucella melitensis,<br>
Haemophilus influenzae, Neisseria meningitides, Tersiniapestis, Shigella sp., Franci sella<br>
tulraensis, and Streptococcus pyogenes. In one embodiment, the bacteria is a<br>
mycobacteria. In one embodiment, the mycobacteria is Mycobacterium tuberculosis. In<br>
one embodiment, the Mycobacterium tuberculosis comprises a uvrAB deletion mutation. In<br>
one embodiment, the Mycobacterium tuberculosis comprises a conditional recA mutation.<br>
In one embodiment, the bacteria is an intracellular bacteria. In one embodiment, the<br>
intracellulax bacteria is Bacillus anthracis. In one embodiment, the Bacillus anthracis<br>
comprises a uvrAB deletion mutation. In one embodiment, the Bacillus anthracis<br>
comprises a. conditional recA mutation. In one embodiment, the intracellular bacteria, is<br>
Yersiniapestis. In one embodiment, the Yersinia pestis comprises a uvrAB deletion<br>
mutation. In one embodiment, the Yersiniapestis comprises a conditional recA mutation.<br>
[0153] The invention includes medicaments comprising the above compositions<br>
and methods of use of the above compositions, such as vaccination of an individual. In one<br>
embodiment, the invention includes a method of using a vaccine of the present invention<br><br>
comprising administering the vaccine to an individual. In one embodiment, the vaccination<br>
is performed by administration of the vaccine by a route selected from the group consisting<br>
of oral, nasal, intraveneous, intradermal, intraperitoneal, intramuscular, intralymphatic, and<br>
subcutaneous. In one embodiment, the vaccine is administered using a prophylactic<br>
regimen to an individual having no signs of the disease against which, the vaccine is<br>
targeted. In one embodiment, the vaccine is administered using a therapeutic regimen to an<br>
individual having symptoms of the disease against which the vaccine is targeted. In one<br>
embodiment, the vaccine comprises a tumor antigen targeting a cancer and the therapeutic<br>
vaccination results in a lessening of the symptoms of the cancer. In one embodiment, the<br>
mean, tumor volume in a vaccinated individual decreases by at least about 5%, also about<br>
10%, also about 25%, also about 50%, also about 75%, also about 90% or about 100%. In<br>
one embodiment, the vaccine is administered to a mouse using either a prophylactic or<br>
therapeutic regimen, wherein the mouse is a model system that can be implanted-with<br>
tumor cells in order to establish tumors in the mice, wherein the vaccines contains at least<br>
one antigen of the implanted tumor. The tumors are implanted in the mice either after<br>
(prophylactic regimen) or before (therapeutic regimen) the vaccine is administered to the<br>
mice. In one embodiment, the mean tumor volumes in mice vaccinated using either a<br>
prophylactic or a therapeutic regimen are less than the tumor volumes in similar mice that<br>
are either not vaccinated, or are vaccinated with a similar vaccine vehicle that expresses an<br>
irrelevant antigen (control mice). In one embodiment, the mean tumor volumes in the<br>
vaccinated mice is at least about 5%, about 10%, about 25%, about 50% , about 75%, about<br>
90% or about 100% less than the mean tumor volumes in the control mice. In one<br>
embodiment, the median survival time of the mice vaccinated using either a prophylactic or<br>
a therapeutic regimen is at least about 2, about 5, about 7 or at least about 10 days longer<br>
than in the control mice.<br>
[0154] In one embodiment, the invention includes a method of making a vaccine<br>
composition comprising treating a microbial population so that the microbial nucleic acid is<br>
modified so that the proliferation of the microbial population is attenuated, wherein the<br>
microbial gene expression is substantially unaffected. In another embodiment, the<br>
invention includes a method of making a vaccine composition comprising treating a<br>
microbial population so that the microbial nucleic acid is modified so that the proliferation<br>
of the microbial population is attenuated, wherein the microbial gene expression is<br>
substantially unaffected, and then using that microbial population to load, an antigen-<br>
presenting cell with antigen and induce activation/maturation of the antigen-presenting cell.<br><br>
In one embodiment, the microbial population is treated by irradiation. In one embodiment,<br>
the microbial population is treated by reacting with a nucleic acixi targeted compound that<br>
indirectly causes the modification of the nucleic acid. In a further embodiment, the nucleic<br>
acid targeted compound is activated by irradiation, wherein activation of the compound<br>
causes the indirect modification of the nucleic acid. In a further embodiment, activation of<br>
the nucleic acid targeted compound results in a reactive oxygen species that modifies the<br>
nucleic acid. In one embodiment, the microbial population is treated by reacting with a<br>
nucleic acid targeted compound that reacts directly with the nucleic acid. In one<br>
embodiment, the nucleic acid targeted compound is reacted at a concentration of about 10<br>
pM to 10 mM, also about 100 pM to 1 mM, also about 1-500 nM, also about 1-200 nM or<br>
about 1-100 nM. In one embodiment, the nucleic acid targeted compound comprises an<br>
alkylator. In one embodiment, the alkylator is selected from the group consisting of<br>
mustards, mustard intermediates and mustard equivalents. In one embodiment, the nucleic<br>
acid targeted compound comprises a nucleic acid targeting group selected from the group<br>
consisting of intercalators, minor groove binders, major groove binders, electrostatic<br>
binders, and sequence-specific binders. In one embodiment, the mucleic acid targeted<br>
compound reacts directly with the nucleic acid upon activation of the compound. In one<br>
embodiment, the activation of the compound is by irradiation. In one embodiment, the<br>
irradiation is UVA irradiation. In a preferred embodiment, the nucleic acid targeted<br>
compound is a psoralen compound activated by UVA irradiation. In one embodiment, the<br>
psoralen compound is at a concentration of about 10 pM to 10 mM, also about 100 pM to 1<br>
mM, also about 1-500 nM, also about 1-200 nM or about 1-100 nM, and the UVA<br>
irradiation is at a dose of about 0.1 - 100 J/cm2, also about 0.1-20 J/cm2, or about 0.5-5<br>
J/cm2 or about 2-4 J/cm2. In one embodiment, the proliferation of the microbial population<br>
is attenuated by at least about 0.3 log, also at least about 1 log, about 2 log, about 3 log,<br>
about 4 log, about 6 log, or at least about 8 log. In another embodiment, the proliferation of<br>
the microbial population is attenuated by about 0.3 to &gt; 10 log, at&gt;out 2 to &gt;10 log, about 4<br>
to &gt;10 log, about 6 to &gt;10 log, about 0.3-8 log, about 0.3-6 log, about 0.3-5 log, about 1-5<br>
log, or about 2-5 log. In one embodiment, the expression of an antigen by the microbial<br>
population is at least about 10%, about 25%, about 50%, about 75%, or at least about 90%<br>
of the expression of the antigen by a microbial population that has not been treated to<br>
modify the nucleic acid. In one embodiment, the antigen expressed is an antigen from the<br>
microbe itself. In one embodiment, the microbe is Mycobacteriurn tuberculosis and the<br>
antigen is from Mycobacteriurn tuberculosis. In one embodiment, the microbe is Bacillus<br><br>
anthracis and the antigen is from Bacillus anthracis. In one embodiment, the microbe<br>
comprises a heterologous nucleic acid sequence encoding an antigen. In one embodiment,<br>
the antigen is a disease associated antigen. In one embodiment, the antigen is associated<br>
with a disease selected from the group consisting of infectious diseases, autoimmune<br>
diseases, allergies, cancers, and other hyperproliferative diseases. In one embodiment, the<br>
antigen is a tumor associated antigen. In one embodiment, the tumor antigen is selected,<br>
from the group consisting of differentiation antigens, tissue specific antigens,<br>
developmental antigens, tumor-associated viral antigens, cancer-testis antigens, embryonic<br>
antigens, oncoprotein antigens, over-expressed protein antigens and mutated protein<br>
antigens. In one embodiment, the tumor antigen is selected from the group consisting of<br>
mesothelin, Spl7, gp100, PR3, PAGE-4, TARP, WT-1, NY-ESO-1 and SPAS-1. In one<br>
embodiment, the microbe comprises a genetic mutation. In one embodiment, the genetic<br>
mutation results in the attenuation of the ability of the microbe to repair microbial nucleic<br>
acid that has been modified. In one embodiment, the genetic mutation is in the gene<br>
selected from the group consisting of phrB, uvrA, uvrB, uvrC, uvrD and recA, or their<br>
functionally equivalent genes, depending on the genus and species of the microbe. In one<br>
embodiment, the genetic mutation is in one or more of" the genes selected from the group<br>
consisting of phrB, uvrA, uvrB, uvrC, uvrD and recA, or their functionally equivalent<br>
genes. In one embodiment, the genetic mutation results in the attenuation in the activity of<br>
a DNA repair enzyme selected from the group consisting of PhrB, UvrA, UvrB, UvrC,<br>
UvrD and RecA. In a further embodiment, microbes having these mutations are treated<br>
with a psoralen activated by UVA irradiation. In one embodiment, the microbe is selected,<br>
from the group consisting of bacteria, protozoa and fungi. In one embodiment, the micro be<br>
is a bacteria. In one embodiment, the bacteria is an intracellular bacteria. In a preferred<br>
embodiment, the bacteria is a Listeria, preferably Listeria monocytogenes. In one<br>
embodiment, the Listeria comprises a mutation that results in the attenuation of the ability<br>
of the Listeria to invade non-phagocytic cells without significantly affecting the uptake of<br>
the Listeria by phagocytic cells. In one embodiment, the Listeria mutation is in an<br>
internalin gene(s). In one embodiment, the Listeria mutation is in the gene selected from<br>
the group consisting of inlA, inlB, and any gene encoding an internalin. In one<br>
embodiment, the Listeria monocytogenes comprises a genetic mutation in both the inlA and<br>
inlB genes. In one embodiment, the Listeria comprises a mutation that results in the<br>
attenuation of the ability of the Listeria to escape the pnagolysosome of an infected cell. In<br>
one embodiment, the Listeria mutation is in the hly gene. In one embodiment, the Listener<br><br>
comprises a mutation that results in the attenuation of the polymerization of actin by the<br>
Listetia. In a preferred embodiment, the Listeria mutation is in the act A gene. In one<br>
embodiment, the Listeria comprises mutations in. the actA gene and one or more internalin<br>
genes. In a preferred embodiment, the Listeria comprises a mutation in the actA gene and<br>
the inlB gene, preferably the Listeria comprises sax actA/inlB deletion mutant. In a<br>
preferred embodiment, the Listeria monocytogerzes actA/inlB deletion mutant further<br>
comprises a deletion mutation in the uvrAB gene.<br>
EXAMPXES<br>
EXAMPLE 1<br>
Psoralen treatment of Listeria strains providing attenuation of proliferation while<br>
maintaining expression of OVA antigen<br>
[0155] Several strains of Listeria monocytogenes that have been modified to<br>
express ovalbumin, a heterologous chicken OVA antigen, were reacted with 4'-(4-amino-2-<br>
oxa)butyI-4,5',8-trimethylpsoralen (S-59 prepared from solid (Ash-Stevens, Riverview, MT)<br>
as 3 mM solution by Ben Venue, Cleveland, OHE (see US patent 5,399,719)) and UVA light<br>
(320-400 nm). The resulting Listeria were assayed to assess the reduction in the log titer of<br>
viable Listeria as well as for the expression of title OVA antigen by the Listeria. The<br>
Listeria strains were provided by Dr. Dan Portaoy at the University of California, Berkeley<br>
and modified to contain the OVA antigen as discussed in Example 8. These were DP-<br>
L4056 (wild-type), DP-L4029 (10403S AactA, phage cured deletion mutation in the Act A<br>
gene, see Skoble et al., Journal of Cell Biology, 150:527-537 (2000) and Lauer et al.,<br>
Journal of Bacteriology 184(15):4177-4186(2002)), DP-L4364 (10403S AlplA, deletion<br>
mutation in phospholipase A gene) and DP-L4017 (10403S hlyL46IT, point anutation in the<br>
hemolysin gene, see Glomski et al., Journal of Cell Biology 156(6): 1029-L 038, (2002)).<br>
The strains were grown in BHI medium (Brain Heart Infusion, Fisher Scientific) at 37 °C at<br>
300 rpm to a concentration of approximately 1 x 109 CFU/mL (to an absorbance at 600 nm<br>
of 0.5). A 1.0 ml, aliquot of each strain was transferred to duplicate 15 mL tubes. Each<br>
tube was centrifixged at 4 °C for 20 minutes at 2300 x g, the supernatant removed and 5 mL<br>
of PBS (phosphate buffered saline, Hyclone) with 1% BSA with and witho ut the S-59 was<br>
added to the duplicate tubes (1 x 108 CFU/mL). The S-59 was added at a concentration of<br><br>
100 nM. Samples were placed in 6 well culture plates and UVA irradiated at a dose of<br>
approximately 2 J/cm2 (FX1019 irradiation device, Baxter Fenwal, Round Lake, IL). Each<br>
sample was then transferred to a 15 mL tube, centrifuged as above, and the supernatant<br>
removed. These were washed with 5 mL of PBS, centrifiiged and the supernatant removed<br>
and the final bacterial pellet was suspended in 0.5 mL of PBS. A 100 ?L sample of each<br>
was used to determine the bacterial titer by serial dilution. Each, dilution was plated onto<br>
an LB (Luria-Bertani, Q-Biogene, Carlsbad, CA) plate and incubated overnight at 37 °C<br>
and the colonies were counted to measure the bacterial titer.<br>
[0156] The antigen presentation of the bacterial samples was assessed using a<br>
murine DC 2.4 cell line (dendritic cell line from the Dana Farber Cancer Institute, see Shen<br>
et al., J Immunol 158(6):2723-30 (1997)) and a B3Z T cell hybridoma (obtained from Dr.<br>
Shastri, University of California, Berkeley). The B3Z is a lacZ indwcible CD8+ T cell<br>
hybridoma thatrexpresses a P-galactosidase gene upon recognition of OVA antigen in<br>
context of MHC class I molecules. The metabolism of CPRG (chlorophenolred-?-D-<br>
galactopyranoside, Calbiochem, La Jolla, CA), a substrate for the ?-galactosidase, was used<br>
to assess the level of ?-galactosidase produced, which is directly coxrelated to the amount<br>
of OVA. antigen presented by the DC 2.4 cells. The DC 2.4 cells arxd the B3Z T cell hybrid<br>
were maintained in RPMI 1640 culture medium (RPMI, Invitrogen) with 10% FBS (fetal<br>
bovine serum, HyClone ). The DC 2.4 cells were transferred in 200 ?L aliquots to the<br>
wells of a 96 well culture plate (1 x 105 DC 2.4 per well). The bacterial samples were<br>
serially diluted 50 ?L stock to 450 fiL PBS down to 1 x 105 CFU/mX (S-59 treated samples<br>
are CFU equivalents, i.e. it is the number of colony forming units prior to S-59 treatment).<br>
A 20 ?L aliquot of each dilution is transferred to a well containing the DC 2.4 cells to give<br>
approximately 1 x 104, 1 x 105, 1 x 106, 1 x 107, or 1 x 108 CFU/mL. In addition, a 20 ?L<br>
aliquot of PBS only was added as a negative control. The samples were incubated for 1<br>
hour at 3 7 °C in 5% CO2. The plate was washed three times with PBS to remove<br>
extracellular bacteria. A 200 ?L aliquot of B3Z T cells (1 x 105 cell) and 100 ?g/mL<br>
Gentamycin (Sigma) was added to each well. As a positive control, 100 nM SL8 OVA257-<br>
264peptide (SL8 OVA antigen, SHNFEKL, SEQ ID NO: 1, Invitrogen, San Diego, CA) was<br>
added to a well containing 1 x 105 each, of the DC 2.4 and B3Z cells. The sampes were<br>
incubated, overnight at 37 °C in 5% CO2. The plate was centrifiiged for 3 minutes at 400 x<br>
g and eaoh well washed with 250 ?L of PBS. A 100 uL aliquot of PBS containing 100 ?M<br>
2-mercaptoethanol, 9 mM MgCl2, 0.125% Igepal CA-630<br><br>
((Octaphenoxy)polyethoxyethanol, Sigma), and 0.15 mM CPRG was added to each well.<br>
The samples were incubated at 37 °C for at least 4 hours. The absorbance was measured at<br>
595 nm with a reference measurement at 655 nm using a plate reader. The results for the<br>
bacterial titer and the antigen presentation of S-59 treated relative to the untreated (100<br>
bacteria per DC 2.4) is given in Table 1. The results indicate that at a level of 100 bacterial<br>
cells added per DC 2.4, the antigen presentation is approximately 55-85% of the untreated<br>
sample. Since the bacterial titer was reduced by approximately 104, this indicates that<br>
sufficient antigen presentation is maintained with considerable attenuation of the<br>
proliferation of the Listeria.<br>
Table 1 Log attenuation and antigen presentation of Listeria strains expressing OVA<br>
antigen treated with 100 nM psoralen S-59 and 2 J/cm2 UVA light.<br><br>
* As percent of untreated, measured at 100 Listeria per DC 2.4 cell.<br>
[0157] A similar procedure was done using the DP-L4056 "wild-type strain. The<br>
bacteria was treated with 100,200,400, 800 or 1000 nM S-59, the remaining titer<br>
determined and the antigen presentation measured as detailed above. The results for the<br>
bacterial titer and antigen presentation (100 Listeria per DC 2.4 cell) are shown in Table 2<br>
and plotted in Figure 1. This data indicates that the antigen presentation is significant over<br>
a broad range of attenuation in the Listeria growth, including presentation of antigen with<br>
complete inhibition of proliferation (i.e. to the limit of detection).<br>
Table 2 Log attenuation and antigen presentation of Listeria strain DP-L4056<br>
expressing OVA antigen treated with varying concentrations of psoralen S-<br>
59 and 2 J/cm2 UVA light.<br><br><br><br>
EXAMPLE 2<br>
DNA targeted alkylator treatment of Listeria strains providing attenuation of proliferation<br>
while maintaining expression of OVA antigen.<br>
[0158] A procedure was done similarly to Example 1 only using the compound. P-<br>
alanine, N-(acridin-9-yl), 2-[bis(2-chloroethyl)amino]ethyI ester (Compound 1, ChemSyn,<br>
Harrisbriville, MO, see US patent 6,093,725). The Listeria strains used were DP-L405 6<br>
and DP-L4017. Compound 1 (1 mM in acidic BBS (blood bank saline), 135 ?l of 1.48 M<br>
H3PO4 per 100 ?L BBS) was added to 5 mL of bacteria at 1 x 108 CFU/mL to<br>
concentrations of 0, 0.5, 1, 2, 5, and 10 ?M and the samples incubated for 2 hours at room<br>
temperature. After the incubation, the bacterial titer and antigen presentation was assessed<br>
as per Example 1. For the antigen presentation, the Listeria strains were diluted to 1 x 102,<br>
1 x 103, 1 x 104, 1 x 105,1 X 106, or 1 x 107 CFU/mL. The log titer, log attenuation and<br>
antigen presention. as a percent of untreated (1 Listeria per DC 2.4) as a function of<br>
compound 1 concentration is given in Table 3 and Figures 2A,B- The results indicate that<br>
compound 1 is also effective, e.g. at 1 ?M, at providing sufficient antigen presentation with<br>
considerable attenuation of the proliferation of the Listeria.<br>
Table 3 Log attenuation and antigen presentation of Listeria strains treated with<br>
varying concentrations of compound 1.<br><br><br>
* As percent of untreated, measured at 1 Listeria per DC 2.4 cell.<br>
EXAMPLE 3<br>
Comparison of attenuation of proliferation by psoralen treatment of uvrAB mutant vs. wild-<br>
type Escherichia coli.<br>
[0159] The psoralen treatment of a mutant Escherichia coli (E. coli) strain that is<br>
deficient in the ability to repair nucleic acid damage was compared to a wild-type strain. E.<br>
coli strains AB1157 (wild-type) and CSR 603 (uvrA, recA, phr mutant obtained from Dr.<br>
Aziz Sancar, University of North Carolina, see Harm, Mutation Research 60:263-270<br>
(1979)). This example compares me attenuation of AB1157 vs mutant CSR603 grown in 3<br>
mL of LB media with streptomycin overnight at 37 °C on an orbital shaker at 250 rpm. A 2<br>
mL aliquot of this was added to 100 mL, of LB media at 30 °C andplaced on the shaker for<br>
approximately 5 hours, until the absorbance at 600 mn was 0.9 OD, approximately 1 x 109<br>
CFU/mL. For each strain, approximately 0.5 mL of the bacterial stock was added to a 15<br>
mL tube and centrifuged at 4 °C for 20 minutes at 2300 x g. The supernatant was removed<br>
and each, pellet was suspended in 5 mL of PBS containing 0, 1, 10, 100, and 1000 nM of<br>
psoralen S-59. Each sample was transferred to a 6 well culture plate and irradiated as per<br>
Example 1. The samples were serially diluted and the titer determined as per Example 1.<br>
The results are shown in Table 4 and Figure 3. The results indicate that psoralen treatment<br>
of the uvrABC mutant results in greater attenuation in the proliferation of the bacteria<br>
(lower titer remaining) for a given psoralen concentration.<br><br><br>
EXAMPLE 4<br>
Therapeutic vaccination, of mice using Listeria strains with and without S-59 treatment.<br>
[0160] In order to assess the utility of S-59 treated Listeria as a vaccine, a C57B1/6<br>
mouse melanoma tumor model was used. C57B1/6 mice (Charles River, Hollister, CA)<br>
were shaved and implanted subcutaneously with 2 x 105 B16.F10.Mo520.10 cells (B16-<br>
OVA expressing melanoma cells obtained from Dr. Kenneth Rock, University of<br>
Massucbesetts, see Mandl et al., Proc Natl Acad Sci USA 95:8216 (1998)) in 100 ?L of<br>
HBSS. Listeria monocytogenes strains DP-L4056 and DP-L4017 containing the OVA<br>
antigen were prepared with, or without S-59 treatment (20 nM S-59 UVA dosed as per<br>
Example 1). In addition, the wild-type strain DP-L4056 without the OVA antigen was used<br>
as a control. The log titer of the S-59 treated samples was determined to assess the log<br>
attenuation due to the psoralen treatment (Table 6). The Listeria were suspended in HBSS<br>
(Hanks Balanced Salt Medium, Gibco) and groups of 10-12 mice were vaccinated three<br>
times with a 100 ?L intraperitoneal injection of each strain, as well as a group injected with<br>
HBSS vehicle. The vaccination dose (total CFU per vaccination) for the various strains is<br>
indicated in Table 6. The doses corresponded to 0.1 LD50 for the non S-59 treated Listeria<br>
and the maximum possible dose for the S-59 treated Listeria. The vaccination was done at<br>
3, 10 and 17 days after tumor implant. The mice were observed for palpable tumors. Once<br>
observed, the opposing diameters of the tumors were measured twice a week. If the tumor<br>
measured 20 mm in any direction, the mice were sacrificed. The mean tumor volume as a<br>
function of days post B16-OVA implant are shown in Figure 4 and Table 5. The percent<br>
survival of mice per group is plotted in Figure 5 and the median survival is given in Table<br>
6. This example shows that high doses of S-59 treated Listeria strains can be safely given<br>
to mice, resulting in a good anti tumor response.<br>
Table 5 Tumor volume at days post implant for mice implanted with B16-OVA and<br>
vaccinated with the identified Listeria strains.<br><br><br><br>
Table 6 Vaccination dose and median survival fox Listeria strains with and without<br>
20 nM S-59 treatment (2 J/cm2 UVA).<br><br>
* Value is average of three preparations.<br>
EXAMPLE 5<br>
Assessment of antigen-specific immune responses after vaccination.<br>
[0161] The vaccines of the present invention can. be assessed using a variety of in<br>
vitro and in vivo methods. These methods are exemplified using a Listeria based vaccine<br>
but can be used to evaluate the potential efficacy of any microbial based vaccine of the<br>
present invention.<br>
[0162] Some assays involve the analysis of antigen-specific T cells from the<br>
spleens of mice that have been vaccinated. C57B1/6 mice are vaccinated, for example<br>
intraperitoneal injection of 0.1 LD50, with a Listeria-OV A strain, where the Listeria may be<br>
treated to attenuate proliferation (e.g. S-59 treatment). Seven days after the vaccination,<br>
the spleen cells of the mice are harvested (typically 3 mice per group) by placing the<br>
spleens into ice cooled RPMI 1640 medium and preparing a single cell suspension from<br>
this. As an alternative, the lymph nodes of the mice could be similarly harvested, prepared<br>
as a single cell suspension and substituted for the spleen cells in the assays described<br>
below. Typically, spleen cells are assessed for intravenesous or intraperitoneal<br><br>
administration of the vaccine while spleen cells and cells from lymph nodes are assessed<br>
for intramuscular, subcutaneous or intradermal administration of the vaccine.<br>
[0163] Unless otherwise noted, all antibodies used in these examples can be<br>
obtained from Pharmingen, San Diego, CA.<br>
[0164J ELISPOT Assay:<br>
[0165] A Listeria strain having an OVA antigen is assessed for the quantitative<br>
frequency of antigen-specific T cells generated upon immunization in a mouse model using<br>
an EIISPOT assay. The antigen-specific T cells evaluated are OVA specific CD8+ or<br>
LLO specific CD8+ or CD4+ T cells. This OVA antigen model assesses the immune<br>
response to a heterologous tumor antigen inserted into the vaccine and could be substituted<br>
with any antigen of interest. The LLO antigen is specific to Listeria, and could be<br>
substituted for an appropriate antigerx for any microbial vector used as the vaccine vehicle.<br>
The specific T cells are assessed by detection of cytokine release (e.g. IFN-?) upon<br>
recognition of the specific antigen. PVDF-based 96 well plates (BD Biosciences, San Jose,<br>
CA) are coated overnight at 4°C with an anti-murine IFN-? monoclonal antibody (mAb R4;<br>
5 ?g/mL). The plates are washed and blocked for 2 hours at room temperature with 200 ?L<br>
of complete RPMI. Spleen cells from, vaccinated mice (or non vaccinated control mice) are<br>
added at 2 x 105 cells per well and incubated for 20 to 22 hours at 37°C in the presence of<br>
various concentrations of peptides ranging from about 0.01 to 10 ?M. The peptides used<br>
are either SL8, an MHC class I epitope for OVA, LLO190 (NEKYAQAYPNVS, SEQ ID<br>
NO:2, Invitrogen) an MHC class II epitope for listeriolysin O (Listeria antigen), or LLO296<br>
(VAYGRQVYL, SEQ ID NO:3), an. 1VIHC class I epitope for listeriolysin 6. After<br>
washing, the plates are incubated with, secondary biotinylated antibodies specific for IFN-?<br>
(XMG1.2) diluted in PBS to 0.5?g/mL. After incubation at room temperature for 2 hours,<br>
the plates are washed and incubated for 1 hour at 37 °C with a 1 nm gold goat anti-biotin<br>
conjugate (GAB-1; 1:200 dilution; Ted Pella, Redding, CA) diluted in PBS containing 1 %<br>
BSA. After thorough washing, the plates are incubated at room temperature for 2 to 10<br>
minutes with substrate (Silver Enhancing Kit; 30 ?L/well; Ted Pella) for spot development<br>
The plates are then rinsed with distilled water to stop the substrate reaction. Aiter the plates<br>
have been air-dried, spots in each well are counted using an automated ELISPOT plate<br>
reader (CTL, Cleveland, OH). The cytokine response is expressed as the number of IFN-?<br>
spot-forming cells (SFCs) per 106 spleen cells for either the OVA specific T cells or the<br>
Listeria specific T cells.<br><br>
Intracellular Cytokine Staining Assay (1CS):<br>
[0166] In order to further assess the number of antigen-specific CD8+ or CD4+ T<br>
cells and correlate the results with, those obtained from ELISPOT assays, ICS is performed<br>
and the cells evaluated by flow cytometry analysis. Spleen cells from vaccinated and<br>
control groups of mice are incubated with SL8 (stimulates OVA specific CD8+ cells) or<br>
LLO190 (stimulates LLO specific CD4+ cells) for 5 hours in the presence of Brefeldin A<br>
(Pharmingen). The Brefeldin A inhibits secretion of the cytokines pro duced upon<br>
stimulation of the T cells. Spleen, cells incubated with an irrelevant MHC class I peptide<br>
are used as controls. PMA (phorbol-12-myristate-13-acetate, Sigma) 20 ng/mL and<br>
ionomyein (Sigma) 2 ?g/mL stimulated spleen cells are used as a positive control for IFN-?<br>
and TNF-? intracellular cytokine staining. For detection of cytoplasmic cytokine<br>
expression, cells are stained with FITC-anti-CD4 mAb (RM 4-5) and PerCP-anti-CD8 mAb<br>
(53-6.7), fixed and permeabilized -with Cytofix/CytbPenn solution (Phamingen), and<br>
stained with PE-conjugated anti-TNF-a mAb (MP6-XT22) and APC-conjugated anti-IFN-?<br>
mAb (XMG1.2) for 30 minutes on ice. The percentage of cells expressing intracellular<br>
IFN-? and/or TNF-? was determined by flow cytometry (FACScalibur, Becton Dickinson,<br>
Mountain View, CA) and data analyzed using CELLQuest software (B ecton Dickinson<br>
Immurtocytometry System). As the fluorescent labels on the various antibodies can all be<br>
distinguished by the FACScalibur, the appropriate cells are identified by gating for those<br>
CD8+ and CD4+ that are stained with either or both of the anti-IFN-y or anti-TNF-?. This<br>
method can also be used to determine the immunogenicity of microbial vaccines, wherein a<br>
dendritic cell population, or another antigen presenting cell such as a macrophage<br>
population, is incubated with the microbial vector. The resulting antigen presenting cells<br>
are injected into the feet of the mice and the cell population from the lymph nodes is<br>
assessed for T cells as above.<br>
Cytokine Expression of Stimulated Spleen Cells:<br>
[0167] The level of cytokine secretion by the spleen cells of mice can also be<br>
assessed for control and vaccinated C57B1/6 mice. Spleen cells are stimulated for 24 hours<br>
with SL8 or LLO190. Stimulation with irrelevant peptide HSV-gB2 (Invitrogen,<br>
SSEEFARL, SEQ ID NO:4) is used as a control. The supernatants of the stimulated cells<br>
are collected and the levels of T helper-1 and T helper 2 cytokines are determined using an<br>
EIISA assay (eBiosciences, CO) or a Cytometric Bead Array Kit (Pharmingen).<br><br>
[0168] Assessment of Cytotoxic T cell Activity:<br>
[0169] The OVA specific CD8+ T cells can be farther evaluated by assessing their<br>
cytotoxic activity, either in vitro or directly in C57B1/6 mouse in vivo. The CD8+ T cells<br>
recognize and lyse their respective target cells in an antigen-specific manner. In vitro<br>
cytotoxicity is determined using a chromium release assary. Spleen cells of naive and<br>
Listeria-OVA (internal) vaccinated mice are stimulated at a 10:1 ratio with either irradiated<br>
EG7.OVA cells (EL-4 tumor cell line transfected to express OVA, ATCC, Manassas, VA)<br>
or with 100 nM SL8, in order to expand the OVA specific T cells in the spleen cell<br>
population. After 7 days of culture, the cytotoxic activity of the effector cells is determined<br>
in a standard 4-hour 51Cr-release assay using EG7.OVA or SL8 pulsed EL-4 cells (ATCC,<br>
Manassas, VA) as target cells and EL-4 cells alone as negative control. The YAC-1 cell<br>
line (ATCC, Manassas, VA) is used as targets to determine NEC cell activity, in order to<br>
distinguish the activity due to T cells from that due to NK cells. The percentage of specific<br>
cytotoxicity is calculated as 100 x (experimental release - spontaneous release) / (maximal<br>
release - spontaneous release). Spontaneous release is determined by incubation of target<br>
cells without effector cells. Maximal release is detennined by lysing cells with 0.1% Triton<br>
X-100. Experiments are considered valid for analysis if spontaneous release is 
maximal release.<br>
[0170] For the assessment of cytotoxic activity of OVA-specific CD8+ T cells in<br>
vivo, spleen cells from naive C57B1/6 mice are split into two equivalent aliquots. Each,<br>
group is pulsed with a specific pepride, either target (SL8&gt; or control (HSV-gB2), at 0.5<br>
?g/mL for 90 minutes at 37 °C Cells are then washed 3 times in medium, and twice in<br>
PBS + 0.1 % BSA. Cells are resuspended at 1 x 107 per ml in warm PBS + 0.1% BSA 
mL or less) for labeling with, carboxyfluorescein diacetate succinimidyl ester (CFSE,<br>
Molecular Probes, Eugene, OR). To the target cell suspen.sion, 1.25 ?L of a 5mM stock: of<br>
CFSE is added and the sample mixed by vortexing. To the control cell suspension, a ten-<br>
fold dilution of the CFSE stock is added and the sample mixed by vortexing. The cells are<br>
incubated at 37 °C for 10 minutes. Staining is stopped by addition of a large volume<br>
(&gt;40 mL) of ice-cold PBS. The cells are washed twice at room temperature with PBS, then<br>
resuspended and counted. Each cell suspension is diluted to 50 x 106 per mL, and 100 ?L<br>
of each population is mixed and injected via the tail vein of either naive or vaccinated mice.<br>
After 12-24 hours, the spleens are harvested and a total of 5 x 106 cells are analyzed by<br>
flow cytometry. The high (target) and low (control) fluoresscent peaks are enumerated, and<br><br>
the ratio of the two is used to establish the percentage of target cell lysis. The in vivo<br>
cytotoxicity assay permits the assessment of lytic activity of antigen-specific T cells<br>
without the need of in vitro re-stimulation. Furthermore, this assays assesses the T cell<br>
function in their native environment<br>
EXAMPLE 6<br>
ELISPOT and ICS analysis of spleen cells from mice vaccinated with Listeria DP-L4056<br>
with and without S-59. treatment.<br>
[0171] Listeria strain DP-L4056 with or without the OVA antigen was prepared<br>
with or without S-59 treatment and used to vaccinate C57B1/6 mice as per Example A<br>
(HBSS control as well), with the exception that administration was intraveneous. The<br>
vaccination was done on naive mice at the doses indicated in Tables 7 and 8. The spleens<br>
were harvested at 12 days post vaccination. The spleens were assessed by ICS and<br>
ELISPOT assays as per Example 5. In addition, the LD50 was assessed for these Lisrteria.<br>
The ICS assay results for both LLO190 specific CD4+ T cells and OVA specific CD8+ T<br>
cells, in terms of percent of cells positive for both TNT-a and IFN-?, are given in Table 7<br>
and Figures 6A,B. The ELISPOT assays, in terms of IFN-? SFC per 2 x 105 spleen cells is<br>
given in table 8 and Figure 7. These results indicate that the S-59 treated sample with OVA<br>
stimulates an OVA specific response when dosed at 100-fold excess of the non S-59 treated<br>
sample. While the positive OVA specific response is not observed at lower doses, this still<br>
provides an increased safety margin as the S-59 treated sample was attenuated by 4 log. In<br>
addition, the LDso was 103-fold higher for the S-59 treated relative to the untreated sample,<br>
indicating that even dosing at 100-fold higher levels, there is a 10-fold level of safety<br>
relative to the untreated Listeria.<br>
Table 7 Percent of spleen cells that are bom TNTF-? and IFN-? positive for mice<br>
vaccinated with DP-L4056 with or without OVA, with or without S-59<br>
treatment.<br><br><br><br>
Table 8 IFN-? SFC per 106 spleen cells for mice vaccinated with DP-L4-056 with or<br>
without OVA, with or without S-59 treatment.<br><br>
EXAMPLE 7<br>
Construction of pKSVT-dlBsrFI uvrAB for deletion of uvrAB<br>
from Listeria by allelic exchange.<br>
[0172] A mutant strain of Listeria unable to repair damage to DNA induced by<br>
treatment with psoralen and UVA light was created by substantially deleting the ultraviolet<br>
light resistance (uvr) AB gene (uvrAB) in Listeria. These mutants are knowa as DNA<br>
repair mutants, or alternatively, nucleoude excision repair (NER) mutants. Deletion of<br>
uvrAB from Listeria was accomplished by allelic exchange [Camilli et al., Molecular<br>
Microbiology 8:143-147 (1993)]. As an example that uvrAB could be deleted from any<br>
Listeria strain, uvrAB was deleted from the Listeria monocytogenes strains shown in Table<br>
9.<br><br>
Table 9. Parent Listeria monocytogenes strains used for deletion of uvrAB by allelic<br>
exchange.<br><br>
[0173] The uvrA and uvrB genes encode 2 of the 3 proteins of the ABC<br>
excinuclease complex required for nucleotide-excision repair (NER) in Listeria and other<br>
bacterial strains of DNA damage inflicted by UV and other agents. The uvrA and uvrB<br>
genes comprise the same operon in the Listeria genome, and were thus deleted together in<br>
the Listeria strains shown in Table 9. The uvrA gene maps from Listeria nts. 2562547 to<br>
2565461 (SEQ ID NO:5), and the uvrB gene maps from Listeria nts. 2565469 to 2567459<br>
(SEQ ID NO:6 )[Glaser et. al., Science 294:849-852 (2001)]. To delete uvrAB by allelic<br>
exchange, the uvrAB gene was first amplified by PCR, using forward and reverse primers<br>
that were approximately 900 base pairs (bps) upstream and downstream, respectively, of<br>
uvrAB. The Listeria uvrAB amplicon was generated using PCH primers Lm-2561677F<br>
(SEQ ID NO:7) and Lm-2568330R (SEQ ID NO:8) and DP-L4056 as template, and was<br>
6654 base bps long, encompassing Listeria nts. 2561677-2568330 (SEQ ID NO:9).<br>
Listeria wild-type strain DP-L4056 was cultured overnight at 30 °C in Brain Heart Infusion<br>
broth (BHI, Difco), and 10 ?L of a washed bacterial suspension (prepared by centrifugation<br>
of the 3 ml overnight culture, re-suspension of the bacterial pellet in 5 ml PBS, re-<br>
centrifugation, and followed by a final re-suspension of' the Listeria pellet in 1 ml of PBS),<br>
was added to a PCR reaction having a final volume of 100 ?L, that also contained 0.2 ?M<br>
each of Lm-2561677F and Xm-2568330R primers, 2 ?L pf Vent DNA polymerase (New<br>
England Biolabs), together with deoxynucleotide triphospates, buffer, and MgSO4<br>
according to the recommendations of the supplier. Successful PCR was confirmed by<br>
0.8% agarose gel electrophoresis in TAE buffer, as demonstrated by the presence of a<br>
distinct 6654 bp band following staining with ethidium bromide and visualization by<br>
illumination with UV light. The amplicon product was purified from the PCR reaction<br><br>
using GeneClean (Qbiogene, Carlsbad, CA), in a final volume of 50?L. Subsequently, the<br>
amplicon was inserted into the pCR2.1-TOPO vector (Invitrogen, Carlsbad, CA), using 5<br>
?L of the purified uvrAB amplicon in the ligation mixture. Restriction enzymes were<br>
purchased from New England Biolabs (Beverly, MA). The correct construction of the<br>
pCR2.1-TOPO-uvrAB plasmid was verified by digestion with BsrFI (New England<br>
Biolabs), followed by 1% agarose/TAE electrophoresis, yielding fragments of 4612/1388,<br>
1094,181, 886 and 2424 base pairs.<br>
[0174] The pCR2.1-TOPO-uvrAB plasmid was used subsequently to generate a<br>
plasmid for allelic exchange, in which nts 2562709 to 2567320 of uvrAB (4612 bps) were<br>
deleted. All restriction enzymes and T4 DNA ligase for recombinant plasmid construction<br>
were obtained from New England Biolabs. To accomplish the deletion of uvrAB sequence,<br>
one aliquot of the pCR-TOPO/uvrAB plasmid (approximately 2 ?g) was digested with<br>
HindBT, BsrFI, and BgIII and the 1092 base pair fragment was purified by 1% agarose/TAE<br>
gel electrophoresis and GeneClean. In parallel, a second aliquot (approximately 2 ?g) was<br>
digested with Xhol, BsrFI, and BgUI enzymes, and the 1050 base pair fragment was<br>
purified by 1 % agarose/TAE gel electrophoresis and GeneClean. The two 1092 bp and<br>
1050 bp fragments containing compatible BsrFI ends were ligated together and the 2142 bp<br>
ligation product was purified using GeneClean. One portion of the 2142 bp ligation<br>
product was digested with PstI and the 1486 bp fragment was purified by 1% agarose/TAE<br>
gel electrophoresis and GeneClean. A second portion of the 2142 bp ligation product was<br>
digested with Kpnl and PstI, and the 622 bp fragment was purified by 1% agarose/TAE gel<br>
electrophoresis and GeneClean. The parent plasmid vector for allelic excliange, pKS V7<br>
[Camilli et al., Molecular Microbiology 8:143-147 (1993)], was digested with Kpnl and<br>
PstI and treated with Calf Intestinal Alkaline Phosphatase (CIAP, New England Biolabs)<br>
and the 622 bp fragment having Kpnl and PstI compatible ends was inserted into the<br>
pKSV7 plasmid vector to give pKSV7-K/P-338. Subsequently, the 1486 bp fragment<br>
having PstI compatible ends was inserted into the vector construct pKSV7-K/P-338 that<br>
was digested with PstI and treated with CIAP. Insertion of the 1486 bp construct in the<br>
correct orientation was determined by digestion with Kpnl and HindIII to yield fragment<br>
sizes of 1253 bp, 865 bp, and 6.9 kb. This plasmid contract is known as pKSV7-dIBsrFI<br>
uvrAB. The Listeria dlBsrFI uvrAB portion of the pKS V7 recombinant plasmid was<br>
sequenced to verify the fidelity of the Listeria sequence and the precise deletion in the<br>
uvrAB gene of nts. 2562709 to 2567320 (deletion from uvr coordinates nts. 2562547 to<br><br>
2567459). The deleted region is SEQ ID NO: 10, with the amplicon sequence remaining<br>
given in SEQ ID NO: 11.<br>
[0175] The uvrAB genes of Listeria strains DP-L4056, IDP-L4017, and DP-L4029<br>
(Table 9) were deleted by allelic exchange using plasmid pKS V7-dlBsrFI uvrAB, as<br>
described previously [Camilli et al., Molecular Microbiology 8 :143-147 (1993)]. The<br>
plasmid pKSV7-dlBsrFI uvrAB was introduced into Listeria strains DP-L4056, DP-L4017,<br>
and DP-L4029 by electroporation. The Listeria strains were nxade competent for<br>
electroporation by first growing a 10 mL overnight culture from an isolated bacterial<br>
colony in BHI with shaking at 37°C. A log-phase culture of ea.cn strain was then derived<br>
by innoculating 2 mL. of the overnight culture into 100 mL 0.51M sucrose/BHI (sterile<br>
filtered) in a 250 mL flask. The culture was grown to mid-log phase by shaking at 37°C for<br>
2-3 hours until a bacterial density of OD6oo=0.2 was reached. Subsequently, the culture<br>
was treated to generate bacteria lacking the peptidoglycan cell wall, known as spheroplastsr<br>
100 ?L of a penicillin G stock solution (10 mg/mL, sterile filtered) was added to the mid-<br>
log phase culture, followed, by shaking at 37°C for 2 hours. The spheroplast culture was<br>
pelleted in a 100 mL centrifuge bottle, resuspended with 45 mL. of an ice-cold<br>
HEPES/sucrose stock solution (lmM HEPES, pH 7.0 / 0.5M sucrose, sterile filtered), and<br>
pelleted again by centrifugation. The bacterial pellet was resuspended with 20 mL of<br>
HEPES/sucrose, transferred to a 40 mL centrifuge tube, pelleted, and resuspended in 10 mL<br>
HEPES/sucrose. 100 uL of a 10 mg/mL lysozyme stock was then added to the bacterial<br>
solution, mixed thoroughly, and the culture was incubated 15 minutes at 37 °C without<br>
shaking, but was inverted gently twice at 5 minute intervals. The lysozyme-treated culture<br>
was then centrifuged at 5000 rpm (3000xg) for 10 min. at 4°C, and resuspended in 10 mL<br>
HEPES/sucrose; this process was repeated twice, with careful and thorough resuspension<br>
each time. The final step to yield electrocompetent Listeria was to resuspend the bacterial<br>
pellet in 500 ?L HEPES/sucrose.<br>
[0176] For electroporation, 2 ?g of pKSV7-dlBsrFI uvrAB plasmid DNA was<br>
added to 10 uL of DP-L4O56, DP-L4017, and DP-L4029 electrocompetent Listeria, and the<br>
solution was added to a 0.1 cm cuvette. The cuvettes were placed in the electroporation<br>
device that was set at 1 K.V, 400 ohms, and 25 ?FD, and then pulsed. This typically<br>
resulted in a time constant of about 5 milli-seconds. The cells were immediately added to 1<br>
mL BHI/sucrose media that was pre-warmed to 30°C, and then incubated for 1 hour at<br>
30°C, without shaking. Following the incubation period, the bacteria were pelleted,<br><br>
resuspended in 200 ml of BHI broth, and the suspension was plated on BHI-agar containing<br>
10 ug/mL chloramphenicol (BHI/CM10). The plates were then incubated overnight at<br>
30°C, after which colonies corresponding to pKSVT-dlBsrFI uvrAB plasmid transfonnants<br>
were visible. The plasmid pKSV7 contains a temperature-sensitive Listeria replicon, and<br>
thus the plate must be incubated at 30°C in order to visualize chloramphenicol-resistant<br>
colonies.<br>
[0177] Allelic exchange of the native uvrAB gene in Listeria strains DP-L405 6, DP-<br>
L4017, and DP-L4029 transformed by electroporation with pKSV7-dlBsrFI uvrAB plasmid<br>
containing a 4612 bp deletion in uvrAB was accomplished in two steps comprised of<br>
plasmid integration, followed by plasmid excision (including native uvrAB) and curing, as<br>
described previously [Camilli et al., Molecular Microbiology 8:143-147(1993)]. Two<br>
isolated chloratnphenicol-resistant colonies resulting flora each of the Listeria strains DP-<br>
L4056, DP-L4O17, and DP-L4029 electroporated with pKSV7-dlBsrFI uvrAB plasmid<br>
DNA were selected and then each selected colony was streaked onto fresh BHI/CM1O<br>
plates, and incubated overnight at 30°C. The next day, a colony was selected from each<br>
plate and used to inoculate 10 mL of BHI/CM10 contained in a 250 mL flask, which was<br>
then incubated overnight at 30°C with shaking.. 10 ?L of each of the overnight cultures<br>
were then used to inoculate 10 mL of fresh BHI/CM10 media (1:1000 dilution), which,<br>
were then grown at 41 °C with shaking until the cultures reached stationary phase.<br>
Following sampling of 10 ?L, a plasmid preparation was performed with the remaining<br>
overnight Listeria culture to insure the presence of the pKSV7-dlBsrFI uvrAB plasmid<br>
DNA. Once at stationary phase, 10 ?L of the cultures were used to inoculate 10 mL of<br>
fresh BHI/CM10 media, which was pre-warmed to 41 °C, and then incubated overnight at<br>
41°C, with shaking. A sample was then taken from each 41 °C overnight culture, and used<br>
to streak for isolated colonies on BHI/CM10 plates, that were pre-warmed at 41 °C. As the<br>
plasmid pKSV7 contains a temperature-sensitive Listeria replicon, incubation at 41°C<br>
selects for colonies arising from integration through homologous recombination of the<br>
pKS V7-dlBsrFI uvrAB plasmid with the native uvrAB gene in the Listeria genome, and<br>
thus amplification and expression of the chloramphenicol drug-resistance marker through<br>
bacterial cell growth and division. At this point, the L-isteria strains are merodiploid for<br>
the uvrAB gene, comprised of the native uvrAB gene and the 4612 bp deleted uvrAB gene,<br>
arising from integration by homologous recombination, of the pKSV7-dIBsrFI uvrAB<br>
plasmid.<br><br>
[0178] For excision and curing of tfcie native uvrAB gene with the pKS V7 plasmid, a<br>
single colony was selected from each of the BHI/CMIO plates incubated at 41°C from the<br>
step above, and used to inoculate 10 ml of BHI media without chloramphenicol, and then<br>
incubated at 30°C overnight with shaking. The cultures were then diluted 1:1000 in 10 mL<br>
of fresh BHI media without chloramphenicol, and then incubated at 30°C overnight with<br>
shaking for 6 hours. This step was repeated, twice. Upon passage of the nxerodiploid<br>
intermediate strains for several generations without drug-selective pressures at a temperature<br>
permissive for the pKSV7 plasmid replicon, spontaneous excision of the integrated pKSV7<br>
plasmid occurs, and eventual curing of the plasmid from the bacterium previously.<br>
[Camilli et al., Molecular Microbiology 8:143-147 (1993)]. Following the third 1:1000<br>
dilution and 6-hour incubation period, a sample from each culture was taken, and used to<br>
streak for isolated colonies on BHI plates (without chloramphenicol), and incubated<br>
overnight at 37°C. One hundred isolated colonies were selected from each BHI plate with<br>
a toothpick, and a 5 mM-long streak was made with each colony first on a BHI/CM10<br>
plate, followed by a BHI plate. Plates were marked using a grid so that each matched pair<br>
of BHI/CM10 and BHI plates were replicates. The BHI/CM10 and BHI replicate plates<br>
were incubated overnight at 37°C. Approximately 5% of the colonies replica-plated on<br>
BHI/CMIO and BHI plates originating from the two cMoramphenicol-resistant colonies<br>
from electroporation with DP-L4056, DP-L4-017, or DP-L4029 with pKSV7-dlBsrFl uvrAB<br>
plasmid DMA were drug sensitive (i.e., growth only on BHI plates). These drug-sensitive<br>
colonies represented candidates containing tfcie 4612 bp deletion in uvrAB. ZEach of the<br>
drug-sensitive colonies were re-streaked for isolated colonies on both BID/CM10 and BHI<br>
plates and incubated overnight at 37°C, to erasure that candidate clones were both pure and<br>
drug sensitive. The chloramphenicol-sensitive clones were subjected to PCR using<br>
primers Lna-2561677F and Lm-25683 30R (ibid), in order to identify clones that also<br>
contained the uvrAB deletion. The amplicon size of clones with the native vtvrAB gene was<br>
6654 bps, and the amplicon size of the deleted uvrAB gene was 2042 bps; about 50% of the<br>
chloramphenicol-sensitive clones also contained the deleted uvrAB gene. Two clones of the<br>
uvrAB deleted strains derived from DP-L4056, DP-L4017, and DP-L4029 were selected for<br>
further characterization. Glycerol stocks (30 °C overnight culture diluted 1: 1 with sterile<br>
LB/40% glycerol) were made for each uvrAB mutant strain and stored at-80 °C. These<br>
strains are known as shown in Table 10. VP-IA029uvrAB (actA/uvrAB) was deposited<br>
with ATCC on October 3,2003, assigned PTA-5563.<br><br><br>
[0179] To demonstrate the increased sensitivity to attenuation with S-59 psoralen<br>
and UVA light, a preparation of 1 x 109 CFU of the uvrAB mutant Listeria strains shown in<br>
Table 10 were treated either with 2,20,100 and 500 nM S-59 (both clones of IA017/uvrAB<br>
and L4056/uvrAB) or 2,10,20, and 100 nM S-59 (clone 1 of L401 HuvrAB and both clones<br>
of \A019luvrAB), UVA irradiated at a dose of 6 J/cm2(FX1019), and tested for viability by<br>
plating; dilutions on BHI plates, exactly as described in Example 1. The results of tins<br>
study are shown in Table 11 Av-B (log attenuation as a function of S-59 dose) and Figure<br>
8A-B (log titer remaining as a. function of S-59 dose). The results demonstrate clearly that<br>
the DNA NER repair mutant strains shown in Table 10 were dramatically more susceptible<br>
to photochemical attenuation -with psoralen and UVA irradiation, as compared to the parent<br>
strains. This data provides utxequivocal evidence that significantly and substantially lower<br>
levels of S-59 psoralen can be used to inactivate uvrAB mutant bacteria to the same extent,<br>
as compared to their isogenic counterpart.<br>
Table 11 A. Log attenuation of Listeria monocytogenes strains after irradiation<br>
(6 J/cm2 UVA) at indicated S-59 concentration.<br><br><br><br>
Table 1 IB. Log attenuation of Listeria monocytogenes strains after irradiation<br>
(6 J/cm2 UVA) at indicated S-59 concentration.<br><br>
[0180] The uvrAB mutant strains can be used directly as a parent strain in which to<br>
incorporate expression cassettes encoding heterologous antigens relevant to malignant or to<br>
infectious disease. In this configuration, following photochemical attenuation with S-59<br>
and UVA light, the bacterium retains its ability to program MHC class I-restricted<br>
responses, because while the ability to replicate its DNA has been abrogated via cross-<br>
linking, the ability to express its genetic complement remains essentially intact.<br>
Furthermore, as a result of the requirement of significantly fewer DNA cross-links to<br>
inactivate ttvrAB mutants, in the context of the population of bacterial genomes comprising<br>
a vaccine dose, the expression of any one gene will not be significantly affected, due to the<br>
low level of DNA crosslinking resulting in essentially no interruption of expression, at that<br>
given gene. Finally, the UVKAB mutation can be combined with any other attenuating<br>
mutation(s), in order to derive a safe and efficacious vaccine platform combining both<br>
photochemical and genetic attenuation. In the composition described herein, the uvrA,<br>
uvrB, or uvrC genes, or any Listeria gene involved in NER, alorxe, or in any combination,<br>
can be mutated such that a functional form of the protein is not expressed. These<br>
compositions can be used as an approach for deriving a safe and efficacious vaccine<br>
derived from a selected bacterial pathogen, in order to protect against challenge with the<br>
wild-type pathogen in vaccinated individuals. Alternatively, these compositions can be<br>
used as an approach for deriving a safe and efficacious recombinant vaccine platform for<br><br>
the expression, of heterologous antigens relevant to any selected infectious or malignant<br>
disease.<br>
EXAMPLES<br>
Insertion of antigen expression cassettes into the genomes of selected Listeria strains by<br>
allelic exchange or by a site-specific integration vector<br>
[0181] The strains described in Example 7, any selected Listeria strain, or any<br>
bacterial strain, can be modified further to express a lieterologous protein or antigen<br>
relevant to malignant or infectious disease. Expression of the heterologous protein can be<br>
via a plasmid containing a replicon that is compatible with the selected host bacterium so<br>
that the plasmid. is stably maintained. Alternatively, a prokaryotic expression cassette can<br>
be integrated stably into the genome of the host bacterium using a variety of methods,<br>
including allelic exchange as described in Example T, or with vectors that integrate<br>
randomly or site-specifically, that are derived from selected transposons or bacfceriophage.<br>
[0182] -As an example, the derivation of recornbinant Listeria monocytcrgenes<br>
derived fromth.e uvrAB nucleotide excision repair (MER) mutant strains described in<br>
Example 7, by utilizing a site-specific integration vector known as pPL2, which, is derived<br>
from the listeriophage PSA (Phage from ScottA), is described here [Lauer et aL., J.<br>
Bacteriol. 184:4177-4186 (2002)]. Specifically, the pPL2 integration vectoris engineered<br>
to express the chicken ovalbumin (OVA) model antigen as a fusion partner with the amino-<br>
terminal half of the Listeriolysin O (LLO) protein that includes the secretion signal and.<br>
PEST sequence [Decatur and Portnoy, Science 290:9 92-995 (2000)], but lacking hemolysin<br>
activity, that is fused in-frame with OVA. Expression of the truncated LLO-OVTA fusion<br>
protein is driven by the hly promoter, which is a prfA-dependent promoter that dives the<br>
expression of Listeria virulence genes, including LLO. This vector is known as-<br>
PPL2/LLOSS-PEST-OVA. The pPL2 vector integrates within the tRNAArg gene off Listeria in<br>
such a manner that the native sequence of the tRNA gene is restored upon successful<br>
integration, thus keeping its native expressed function intact.<br>
[0183] Xhe first step in the construction of PPX2/LLOSS-PEST-OVA is to amplify the<br>
hly promoter and LLOSS-PEST sequence together from DP-L4056 wild-type Lister-ia genomic<br>
DNA, by PCR using the primer pair of forward primer Kpnl-LLO nts. 1257-1276 (SEQ ID<br>
NO:12) and reverse primer XhoI-LLO1665R (SEQ ID NO:13). The 426 bp amplicon is<br>
purified with GeneClean, digested with Kpnl and Xhol, and ligated into pPL2 plasmid,<br><br>
which is prepared by digesting with Kpnl and Xhol, treatment with calf intestinal alkaline<br>
phosphatase (CIAP), and purified with GeneClean. Correct plasnxids containing the LLOss.<br>
PEST sequence are verified by digesting with Kpnl and Xhol and 1% agarose/TAB<br>
electrophoresis, yielding DNA fragments of 418 bps and 6112 bps. This intermediate<br>
plasmid DNA construct is known as PPL2/LLOSS-PEST.<br>
[0184] The OVA sequence can be amplified by PCR from, any number of plasmids<br>
that are used by those in the art, including pDP3616 plasmid DNA from DP-E3616 E. coli<br>
[Biggins et al., Mol. Molbiol. 31:1631-1641 (1999)], using the primer pair of forward<br>
primerXhol-NcolOVA cDNA. nts. 174-186 (SEQ ID N0:14) and reverse primer XhoI-<br>
Notl-HindIII(SEQ ID N0:15).<br>
[0185] The 1013 bp amplicon is purified with GeneClean, digested with Xhol and<br>
Notl, and ligated into PPL2/LLOSS-PEST plasmid, prepared by digesting with Xhol and NotI,<br>
treatment with CIAP; and purified with Gene Clean. The correct plasmid construct<br>
containing the LLOSS-PEST and OVA sequence is verified by digesting with Kpnl, Xhol, and<br>
Notl and 1% agarose/TAE electrophoresis, yielding DNA fragments of 994 bps, 1560 bps,<br>
and 6039 bps. The precise expected sequence of the LLO and OVA regions of plasmid<br>
pPL2/LLOss-PEST-OVA is confirmed by sequencing.<br>
[0186] The pPL2/IXOss-PEsT-OVA plasmid is incorporated into the tRNAArg gene in<br>
the genome of selected Listeria uvrAB mutant strains described in. Example 7, exactly<br>
according to the methods as described previously [Lauer et. al., J. IBacteriol. 184, 4177-<br>
4186 (2002)]. Briefly, the plasmid pPL2/LLOss-pEST-OVA is first introduced into the E.<br>
coli host strain SM10 (Simon et. al., Bio/Technology 1:784-791 (1983)] by electroporation<br>
or by chemical means. Subsequently, the pPL2/LLOss-PEST-OVA plasmid is transferred<br>
from transformed SM10 to the selected Listeria strains by conjugation. Following<br>
incubation on drug-selective BHI agar plates containing 7.5 ?g of chloramphenicol per ml<br>
and 200 ?g of streptomycin per ml as described, selected colonies are purified by passaging<br>
3 times on plates with the same composition. To verify integration of the pPL2 vector at<br>
the phage attachment site, individual colonies are picked and screened by PCR using the<br>
primer pan- of forward primer NC16 (SEQ ID NO: 16) and reverse primer PL95 (SEQ ID<br>
NO: 17). Selected colonies having the PPL2/LLOSS-PEST-OVA plasmid incorporated into the<br>
tRNAArg gene in the genome of selected Listeria uvrAB mutant strains will yield a<br>
diagnostic DNA amplicon of 4-99 bps.<br><br>
[0187] The ability of the recombinant Listeria uvrAB mutants harboring a stable<br>
integrant of pPL2/LLOss-PEST-OVA to be taken up by antigen presenting cells and<br>
subsequently program presentation of OVA via the MHC class I pathway is tested, using<br>
the cloned C57Bl/6-derived dendritic cell line DC2.4, as described in Example 1.<br>
Presentation of OVA peptide by DC2.4 cells on class I molecules following phagocytosis<br>
of Listeria is measured after incubation with B3Z cells, also as described in Example 1.<br>
These procedures verify that the recombuaant Listeria strains are functional, and can be<br>
used further as described in the Examples contained herein.<br>
[0188] Thus, this example provides instructions for introducing a prokaryotic<br>
expression cassette encoding any desirable antigen(s) related to selected infectious and<br>
malignant diseases into DNA repair mutant Listeria strains containing a deletion within the<br>
uvrAB gene. The said recombinant Listeria strains can be inactivated by treatment with<br>
psbralens as described in Example 1 and can be used subsequently for a variety of<br>
applications, including, for example, proplaylactic and therapeutic vaccines for infectious<br>
and malignant disease.<br>
EXAMPLE 9<br>
Bacterial Vaccines derived from nucleotide-excision repair (NEP) mutants<br>
[0189] The examples described in this patent illustrate the efficacy of vaccine<br>
compositions utilizing genomic inactivatiom through photochemical treatment of the<br>
recombinant delivery platform encoding antigens related to infectious and malignant<br>
disease. According to this composition, while the genomes are inactivated and cannot<br>
separate during replication, the transcriptioinal profile remains largely intact, thus resulting<br>
in antigen expression de novo in the vaccinated individual, and optimal induction of<br>
pathogen-specific immune responses, including CD8+ cytotoxic T cells (CTL).<br>
Furthermore, as described in Example 7, by utilizing a vaccine platform in this composition<br>
in which the DNA nucleotide excision repair (NER) machinery has been inactivated by any<br>
number of means, including by engineered genetic deletion, the sensitivity to<br>
photochemical inactivation in these mutants is dramatically increased.<br>
[0190] As a result of the requirement of significantly fewer DNA cross-links to<br>
inactivate the DNA repair mutants, in the context of the population of bacterial genomes<br>
comprising a vaccine dose, the expression of any one gene will not be significantly<br><br>
affected, due to the low level of DNA crosslinking resulting in essentially no inteiruption<br>
of expression, at that given gene.<br>
[0191] Thus, the overall utility of gene-based vaccines utilizing bacterial platforms<br>
derived from pathogens can be increased dramatically by combining photochemical<br>
inactivation with a vector defective in NER, While the inactivated vaccine cannot caaise<br>
disease, it still retains its efficient ability to induce potent immunity, including T-cell<br>
mediated cellular immunity, specific for the vector-expressed heterologous antigens.<br>
Furthermore, the uvrAB mutation can be combined with any other attenuating mutation(s),<br>
in order to derive a safe and efficacious vaccine platform combining both photochemical<br>
and genetic attenuation.<br>
[0192] Significantly, these compositions can be used as an approach for deriving a<br>
safe and efficacious vaccine derived from a selected bacterial pathogen, in order to protect<br>
against challenge with the wild-type pathogen in. vaccinated-individuals. According to-this<br>
application, it is not feasible in many cases to derive a safe and efficacious vaccine that is<br>
derived from an attenuated viable form of the pathogen, as the possibility for reactivity and<br>
disease pafhogenesis in particular individuals receiving the vaccine remain high. While<br>
subunit or inactivated vaccines related to a selected bacterial pathogen might be safe, on the<br>
other hand, these vaccines are often not efficacious because they do not efficiently elicit the<br>
breadth, depth, and durability of pathogen-specific immune responses that are required to<br>
protect the vaccinated individual against challenge with the wild-type form of the said<br>
pathogen. Thus, it is well known in the art that there is a clear need for vaccine<br>
compositions that combine safety with an efficient ability to elicit the type of immune<br>
responses in vaccinated individuals that are protective.<br>
[0193] As such, mutants in the nucleotide-excision repair (NER) pathway of<br>
pathogenic microbes provide a composition that can be used for safe and efficacious<br>
vaccines that elicit protection against challenge in immunized individuals with amounts of<br>
the said microbe that are sufficient to cause disease in non-vaccinated individuals. NEH is<br>
catalyzed by an ATP-dependent nuclease made of three subunits, known as the ABC<br>
excinuclease, and encoded by the genes uvrA, uvrrB, and uvrC. Mutations in any one or<br>
more than one of the three uvr genes results in cells, including microbes of pathogenic<br>
organisms, extremely sensitive to photochemical inactivation utilizing psoralens and UVA<br>
light.<br>
[0194] As an example, mutation of the uvr genes of Bacillus anthracis (B.<br>
anthracis), fee etiological agent of Anthrax, is provided. The current acellular anthrax<br><br>
vaccines that are licensed for human use are based on sterile culture supematants of<br>
attenuated B. anihracis adsorbed on alum hydroxide (U.S. vaccine), or precipitated with<br>
alum phosphate (U.K. vaccine). It is well known that these vaccines are rather weak,<br>
requiring at least six immunizations for protection as well as annual boosters.<br>
[0195] In the composition described herein, the uvrA, uvrB, or uvrC genes, or any<br>
B. anthracis gene involved in NER, alone, or in any combination, is mutated such, that a<br>
functional form of the protein is not expressed.<br>
[0196] As an example, mutation in the uvrA, uvrB, or uvrC genes, or any 25.<br>
anthracis gene involved in NER, can be performed, for example, by allelic exchange, as<br>
described in Example 7. While the uvr genes of B. anthracis have not been identified<br>
through targeted deletion and characterization of the phenorypes of the resulting mutant<br>
strains, the uvr genes can be identified through a homology search with the genomes of<br>
related organisms'in whose uvr genes are known. For example, the genome of B.<br>
anthracis, that is, the main chromosome and the two virulence plasmids can be compared<br>
with Bacillus Subtilis (B. Subtilis), a related bacterium from the same genera as B.<br>
anthracis. The genomic scaffold representing the main chromosome of the Florida B.<br>
anthracis isolate (Read et al. 2002. Science 296, 2028-2033) has a GenBank accession<br>
number of AAAC010000001. B. subtilis has a GenBank accession number of NC_000964.<br>
The B. subtilis uvrA gene encompasses nts. 3609064 to 3611997, and the B. subtilis uvrB<br>
gene encompasses nts. 3612005-3613990. A BLAST search was performed using the B.<br>
subtilis uvrA and uvrB coding sequences against the B. anthracis sequence. This analysis<br>
identified a region of 72% sequence identity in the genome of B. anthracis that corresponds<br>
to the ttvrA and uvrB genes of this organism. The B. anthracis uvrA gene maps from<br>
226021-228783, and bears 72% sequence homology to the B. subtilis uvrA gene<br>
(2082/2867 identical sequence homology alignment). The B. anthracis uvrB gene maps<br>
from 228864-230771, and bears 72% sequence homology to the B. subtilis uvrB gene<br>
(1401/1925 identical sequence homology alignment). Thus, the B. anthracis uvrsSLB genes<br>
include nts. 226021 to 230771 of the main chromosome of B. anthracis.<br>
[0197] Deletion of the B. anthraczs uvrAB genes, including nts. 226021 to 230771<br>
of the main bacterial chromosome can be accomplished according to the methods described<br>
in Example 7 for the deletion of uvrAB genes in L. monocytogenes. Briefly, this region and<br>
approximately 1000 bps both upstream and downstream of the B. anthracis genome are<br>
amplified by PCR, and subsequently cloned into the pKSV7 allelic exchange plasrnid<br>
vector. As an alternative, a Bacillus genera-specific or B. anthracis-specific temperature-<br><br>
sensitive (ts) replicon may be substituted for the Listeria ts replicon present in the pKSV7<br>
allelic exchange plasmid vector. Using convenient restriction endonuclease recognition<br>
sites mapping specifically within the uvrAB region, any part of th.e uvrA, uvrB, or all of the<br>
uvrAB genes sequence are deleted. Finally, the allelic exchange plasmid is introduced into<br>
B. anthracis and NER mutants are selected as described in Example 7. Any selected B.<br>
anihracis strain can be used as a parent strain for derivation of the NER-defective vaccine,<br>
including, for example, the following strains: Ames, Vollum, Al.a/10, Al.b/23, A2/29,<br>
A3 .a/34, A3.b/57, A4/69, B/80, Asterne, VN41A1, Dames, NNRL Al, and DNH1.<br>
Additionally, other attenuating mutations can be incorporated into the genome of the<br>
selected NER mutant B. anthracis strain, to enable vaccine compositions combining<br>
photochemical with genetic inactivation. Such B. anthracis vaccine compositions are able<br>
to induce immune responses against known correlates of anthrax immunity and protection,<br>
including lethal factor (LF), edema factor (EF), and protective antigen (PA). Additionally,<br>
as a result that the expression profile of the NER mutant vaccine composition remains<br>
intact, immune responses against other unknown correlates of anthrax immunity and<br>
protection, including those expressed from the two virulence plasrmids pXOl and pXO2<br>
and the main chromosome are also induced.<br>
[0198] The compositions described herein, using B. anthracis as an example<br>
utilizing NER mutants as a component of vaccine, can be used in either a prophylactic or a<br>
therapeutic immunization setting against all three types of anthrax according to the route of<br>
infection, including cutaneous, gastrointestinal and respiratory. Furthermore it can be<br>
appreciated that the approach for generating NER mutants of B. crnthracis to derive a safe<br>
and efficacious vaccine can be adopted to derive safe and efficacious vaccines for any<br>
microbial pathogen that utilizes NER.<br>
EXAMPLE 10<br>
Use of vaccines of the invention for the in vivo treatment of human cancers.<br>
[0199] As an example of the treatment or prevention of a human cancer, a vaccine<br>
comprising a microbial population in which the microbial nucleic acid is modified so that<br>
the proliferation of the microbial population is attenuated, wherein the microbial gene<br>
expression is substantially unaffected, is administered to an individual. The microbe can be<br>
prepared following the protocols of examples 7 and 8, wherein any desired prokaryotic<br><br>
expression, cassettes encoding human tumor antigenCs) are incorporated into the microbe,<br>
by utilizing, for example the pPL2 integration vector described in Example 8, or any<br>
modifications thereof, or by any methods that are common to those in the art. The resulting<br>
population may be formulated in crude, or preferably purified form. They may be prepared<br>
as a liquid suspension, or may be freeze-dried and resuspended in a suitable carrier for<br>
administration. In addition, they may be formulated "with additives such as preservatives<br>
(e.g. thirnerosal, 2-phenoxy ethanol), stabilizers (e.g. lactose, monosodium glutamate),<br>
adjuvants (e.g. aluminum hydroxide, aluminum phosphate, cytokines), antibiotics (e.g.<br>
neomycin, streptomycin) or other substances. Formulations may be resuspended or diluted.<br>
in a suitable diluent such as sterile water, saline, isotonic buffered saline (e.g. phosphate<br>
buffered to physiological pH), or other suitable diluent.<br>
[0200] The vaccine may be administered by a variety of routes, including oral,<br>
nasal, intraveneous, intradermal, intraperitoneal, intramuscular, intralymphatic and<br>
subcutaneous routes, as well as by any route that is relevant for any given malignant or<br>
infectious disease. An. effective amount of the vaccine will be administered to an<br>
individual for treatment. For a therapeutic treatment, an effective amount is a dose that will<br>
result in the desired immune response, wherein the immune response either slows the<br>
growth of the targeted tumors, reduces the size of the tumors, or preferably eliminates the<br>
tumors completely. Trxe administration of Ihe vaccine may be repeated at appropriate<br>
intervals, and may be administered simultaneously at multiple distinct sites in the<br>
vaccinated individual. For the prophylactic treatment, an effective amount is a dose that<br>
will result hi a protective immune response such that ttie likelihood of an individual to<br>
develop the cancer is significantly reduced. The vaccination regimen may be comprised of<br>
a single dose, or may be repeated at suitable intervals until a protective immune response is<br>
established.<br>
[0201] The therapeutic treatment of an individxxal may be started on an individual<br>
who has been diagnosed with a cancer as an initial treatment, or may be used in<br>
combination with other treatments. For example, individuals who have had tumors<br>
surgically removed or who have been treated with radiation therapy or by chemotherapy<br>
may be treated with the vaccine in order to reduce or eliminate any residual tumors in the<br>
individual, or to reduce the risk of a recurrence of the cancer. The prophylactic treatment<br>
of an individual would be started on an individual who has an increased risk of contracting<br>
certain cancers, either due to environmental conditions or genetic predisposition.<br><br>
EXAMPLE 11<br>
Antigen presentation of Listeria strain DP-L4029 with and without uvrAB mutation<br>
following S-59 psoralen UVA treatment.<br>
[0202J Th.e Listeria strain DP-L4029 uvrAB mutant clone 1 of Example 7 was<br>
modified to express the OVA antigen using the procedure of Example 8. This strain and<br>
DP-L4029 modified to express OVA were treated with the psoralen S-59 at various<br>
concentrations. The Listeria strains were grown overnight at 37 °C and a 2 mL aliquot was<br>
diluted into 100 mL of BHI and grown approximately 4 hours at 37 °C to an OD600 of 0.5<br>
(approximately 1 xlO9 CFU/mL). A 5 mL aliquot of each. Listeria strain was added to a 15<br>
mL tube and centrifuged for 20 minutes at 2300 x g, the supernatant removed, and the<br>
bacteria resuspended in 5 mL of PBS resulting in approximately 1 x 109 CFU/mL. For the<br>
uvrAB mutant strain, 3 mM S-59 stock was diluted 33.3 ?L to 10 mL PBS to give a 10 ?M<br>
solution, and appropriate aliquots of this was added, to the Listeria to final concentrations of<br>
10,20, 30, 40,50, 60, 70, 80, 90, and 100 nM, while for the DP-L4029, S-59 was added to<br>
final concentrations of 100,200,400, 800, and 100 0 nM in a final volume of 5 mL. These<br>
were transferred to a 6 well culture plate and irradiated for a dose of 0.5 J/cm2 (FX101 9<br>
UVA device). The samples were transferred to 15 rmL tubes, 5 mL PBS was added, and<br>
they were centrifuged for 20 minutes at 2300 x g to wash out unreacted psoralen. The<br>
supernatant was removed and the bacteria resusperxded in 5 mL PBS and transferred to new<br>
6 well plates. These were irradiated at a UVA dose of 5.5 J/cm2 in order to convert<br>
psoralen monoadducts to crosslinks. A sample of each, Listeria strain was also heat killed<br>
by treating at 72 °C for 3 hours. The log titer and OVA antigen presentation were assessed<br>
as per Example 1. The results for the S-59 treated samples are found in Table 12A and<br>
Figures 9A and 9B (antigen presentation at 1 Listeria per DC 2.4 cell, calculated without<br>
subtracting background levels). The results for both, heat lolled strains showed a titer<br>
below the limit of detection (complete inactivation) and the heat killed bacteria did not<br>
present OVA antigen in the B3Z assay. The results indicate that the uvrAB mutant shows<br>
very strong antigen presentation even with attenuation of proliferation to the limit of<br>
detection where the non uvrAB mutant strain shows a greater reduction in the antigen<br>
presentation as a function of attenuation of proliferation (to approximately background<br>
levels with essentially complete inactivation). This demonstrates that the uvrAB mutant<br>
retains MHC class I presentation in the context of psoralen attenuated Listeria and should<br><br>
provide a vaccine with an effective immune response and significantly increased level of<br>
safety.<br>
Table 12A Log attenuation and OVA antigen presentation of Listeria strains UVA<br>
treated with varying concentrations of psoralen S-59.<br><br>
* As percent of untreated, measured at 1 Listeria per DC 2.4 cell.<br>
[0203] Another study was done using the same strains. In this study the Listeria<br>
were grown ia BHI at 37 °C overnight. These were diluted 1:50 into BHI and grown at 37<br>
°C at 300 rpm to an OD600 of 0.5, at which point 50 mL of solution was transferred to a<br>
clean flask and S-59 was added to the levels: indicated in Table 12B. These samples were<br>
incubated at 37 °C at 300 rpm for approximately 1 hour (OD600 approximately 1.0,<br>
approximately 1 x 109/mL). A 1 mL aliquot was removed to assess the titer and the<br>
remaining was transferred to a 150 mm Petri dish and irradiated at a dose of 6 J/cm2<br>
(FX1019). The titer post irradiation was determined for each sample and the OVA antigen<br>
presentation was assessed as above. The results are found in Table 12B and Figures 9C<br>
and 9D (antigen presentation at 10 Listeria per DC 2.4 cell, calculated without subtracting<br>
backeround levels). The results indicate that for the parent strain, the antigen presentation<br><br>
is at background levels where there is essentially complete inactivation whereas for the<br>
ttvrAB mutant, there is an approximately 10-fold range of S-59 concentration over which<br>
there is essentially complete inactivation along with adequate antigen presentation.<br>
Table 12B Log attenuation and OVA antigen presentation of Listeria strains UVA<br>
treated with varying concentrations of psoralen S-59 present during growth<br>
of the bacteria.<br><br>
* As percent of untreated, measured at 10 Listeria per DC 2.4 cell.<br>
EXAMPLE 12<br>
Protein synthesis in S-59/UVA treated Listeria monocytogenes DP-L4029 compared to DP-<br>
L4029 uvrAB.<br>
[0204] Listeria monocytogenes DP-L4029 and DP-L4029uvrAB were grown in BHI<br>
at 37 °C overnight. These were diluted 1:50 into BHI and grown at 37 °C at 300 rpm to an<br><br>
OD600 of 0.5, at which point 50 mL of solution was transferred to a clean flask and S-59<br>
was added to a level of 2500 nM for the 4029 and 200 nM for the 4029 uvrAB mutant<br>
strain. These samples were incubated at 37 °C at 300 rpm for approximately 1 hour<br>
(OD600 approximately 1.0). A 1 mL aliquot was removed to assess tfhe titer and the<br>
remaining was transferred to a 150 mm Petri dish and irradiated at a dose of 6 J/cm2<br>
(FX1019). The titer post irradiation was determined for each sample to assess the level of<br>
inactivation, resulting in essentially complete inactivation. It has been determined that this<br>
treatment is approximately the lowest S-59 dose that provides inactivation to the limit of<br>
detection for both strains. For each strain, 1 x 1010 bacteria, based on the OD600 vs. titer<br>
CFU/mL growth curve, was transferred to a 15 mL centrifuge tube. The sample was<br>
centrifuged at 4 °C for 20 minutes at 2300 x g, the supernatant removed and the pellet<br>
washed with 50 mL of PBS. This was repeated for a total of three washes. The final pellet<br>
' was suspended in 2 mL of DMEM without methionine or cysteine (Gibco) and incubated at<br>
37 °C in 5% CO2 incubater with shaking for 30 minutes. The samples were centrifuged in<br>
2 mL centrifuge tubes at 1600 rpm for 2 minutes, the supernatant removed and 2 mL of<br>
DMEM without methionine or cysteine was added. An 80 ?Ci aliquot of 35S methionine-<br>
cysteine was added (Perkin Elmer Life Sciences) and the sample incubated at 37 °C in 5%<br>
CO2 incubater with shaking for 30 minutes. The samples were centrifuged as above and<br>
the supernatant removed. A 50 uX, aliquot of each supernatant were loaded in adjacent<br>
lanes onto an SDS-PAGE gel (Tnvitrogen, NuPage 4-12% Bis-Tris gel) and run at 100 volts<br>
for approximately 1.5 hours. The gel was fixed with 10% acetic acid and 30% ethanol,<br>
th.en soaked in enhancer (Enlightning, NEN Life Sciences) for 15 mirjurtes. The gel was<br>
dried for 3 hours at 80 °C and the bands visualized by exposure to X-ray film. The results<br>
for two studies are shown in Figure 10, indicating considerable protein synthesis in the<br>
wvrAB mutant strain while the parent strain shows limited protein synthesis.<br>
EXAMPLE 13<br>
Comparison of S-59/UVA inactivation with or without S-59 present during growth of<br>
Listeria.<br>
[0205] Two inactivation methods were compared with respect to inactivation of<br>
Listeria monocytogenes strains. In the first method, the Listeria was grown in BHI at 37 °C<br>
at 300 rpm overnight, then diluted 1:50 into BHI and grown at 37 °C at 300 rpm to an<br><br>
OD600 of 0,7-1.00. These were centrifuged and suspended in PBS with 1% BSA to a level<br>
of 1 x 109/mL. S-59 was added to a level of 120 nM for the parent strain and 30 nM for the<br>
uvrAB mutant strain. The samples were incubated on ice for approximately 60 minutes,<br>
then transferred to a 150 mm Petri dish and irradiated at a dose of 6 J/cm2 (FX1019). In the<br>
second method, the Listeria was similarly grown to an OD600 of 0.5, at which point 50 mL<br>
of solution was transferred to a clean flask and S-59 was audded to a level of 2500 nM for<br>
the parent strain and 200 nM for the uvrAB mutant strain. These samples were incubated at<br>
37 °C at 300 rpm for approximately 1 hour (OD600 approximately 1.0, approximately 1 x<br>
109/mL). A 1 mL aliquot was removed to assess the liter and the remaining was transferred,<br>
to a 150 nun Petri dish, and irradiated as per the first method. The titer post irradiation was<br>
determined for each sample, resulting in essentially complete inhibition of proliferation for<br>
all samples (&gt;8 log inactivated). In a study done with DP-X4029 vs. DP-IA029uvrAB, the<br>
entire sample containing approximately 1 x 1011 bacteria treated by the second method, the<br>
entire sample was plated, indicating approximately 9 log kill for the parent strain and &gt;10<br>
log kill for the uvrAB mutant. The results on four different; preparations of Listeria are<br>
given in Table 13.<br>
Table 13 Inactivation of Listeria monocytogenes actA and actAuvrAJEf with S-<br>
59/UV.A, measurement of entire sample to assess log titer inactivation.<br><br>
[0206] In one study, the two methods were compared using Listeria monocytogenes<br>
DP-L4029-OVA and DP-L4029 uvrAB-OVA, Samples were prepared as above and were<br>
centrifuged for 20 minutes at 2300 x g, the supernatant renroved and the bacteria washed<br>
once with PBS. After centrifuging and removing the PBS wash, the final pellet was<br><br>
resuspended in 8 % DMSO in PBS, then quickly frozen in cryo-vials, either using liquid<br>
nitrogen or dry ice, and stored at -80 °C. Sets of three mice (C57B1/6) were injected<br>
intraveneously with 1 x 108 Listeria in 200 ?L {frozen stock diluted approximately 1:40<br>
into HBSS). In addition to the S-59/UVA treated strains, injections were made with live<br>
and heat killed DP-L4029 uvrAB-OVA, as well as HBSS control. For the comparison of<br>
the two S-59 methods, mice were injected at day 0. For the samples prepared by the<br>
second method, additional sets of mice were injected again either on days 2 and 3 or on<br>
days 2,3,4, and 5. All mice were sacrificed on day 7 post vaccination and the spleens<br>
removed for analysis. The spleen cells were assessed for an OVA specific immune<br>
response by ELISPOT assay as described in Example 5, stimulating the cell population<br>
with SL8 (OVA specific). The results are shown in figure 11A, indicating that the Listeria<br>
prepared by the second method, for both the parental strain and the uvrAB mutant, result in<br>
a more potentrOVA specific immune response than for strains prepared by the first: method.<br>
The ELISPOX assay was also done with stimulation using LLO class II antigen LLO 190,<br>
or class I antigen LLO296. The ELISPOT results comparing all three antigens is shown in<br>
Figure 1 IB, indicating that the LLO specific CD 4+ response is similar to the OVA. specific<br>
response. The spleen cells were also assessed by ICS as described in Example 5,<br>
stimulating with either SL8, LLO190, or LLO296. The results are shown in Figures 12A-<br>
C, indicating a stronger immune response for both OVA and LLO in the second method.<br>
The data also demonstrates the improved response for the uvrAB strain over the parent<br>
strain. In both, strains, additional vaccination on successive days results in improved<br>
response to both OVA and LLO antigens (1 vs. 3 days).<br>
[0207] In another study, DP-L4029 and DP-L4029 uvrAB strains are assessed for<br>
their ability to provide protective immunity against a wild type challenge in mice. Balb/c<br>
mice were vaccinated in groups as described in Table 14 with HBSS, DP-L4056 wild type<br>
(+/- heat killed), DP-L4027 (LLO deletion), DP-L4029 S-59/UVA treated (first and second<br>
methods as above), DP-L4029mv4B S-59AJVA. treated (first and second methods as<br>
above). Twenty-seven days after the vaccination, three mice per group were challenged<br>
with 2xLD50 and six mice per group with 100xLD50 of wild type Listeria monocytogenes.<br>
Three days post challenge, the mice challenged with 2xLD50 were sacrificed and the spleen<br>
and liver were isolated and cultured for growth of Listeria. The spleen or liver from each<br>
mouse was homogenized in sterile distilled water with 0.5% Triton X-100 (Sigma). Serial<br>
10-fold dilutions were plated on BHI agar plates containing streptomycin (50 ?g/mL) and<br><br>
incubated at 37 °C overnight. The number of colony forming units per spleen or liver was<br>
determined as an indication of immunity to the wild type challenge. Figures 13A,B show<br>
that S-59/UVA treated samples give approximately 3 log reduction in CFU per organ<br>
compared to HBSS (non-vaccinated) controls, with the samples prepared by the second<br>
method showing more reduction in CFU than those prepared with the first method. In<br>
addition, the treated uvrAB mutant strain shows slightly better CFU reduction than the<br>
treated parent strain. While the CFU reduction is not as good as vaccination with wild<br>
type, the S-59/UVA treated strains show some efficacy for reduction in CIFU, which<br>
generally correlates with protective immunity. The six mice challenged with 100xLD50<br>
were monitored for survival for ten days, with only the mice vaccinated with wild type<br>
Listeria surviving.<br>
Table 14 Dosing of Baib/c mice for assessment of protective immunity comparing<br>
two S-59/UVA methods.<br><br>
[0208] An additional study was done in Balb/c mice using HBSS, DP-L4056 wild<br>
type (+/- heat killed), DP-L4027 (LLO deletion), DP-L4406actA (actA/inlB deletion double<br>
mutant, deposited on October 3, 2003, ATCC number PTA-5562) DP-L4O29 +S-59/UVA<br>
(second method), DP-IA029uvrAB +/- S-59/UVA treated (second method only) or + heat<br>
killed, where vaccination was done daily for 1, 3, or 5 days for S-59 and heat killed strains.<br>
The dosing is summarized in Table 15. Twenty-nine days post the first vaccination, three<br>
mice from each group were challenged with 20x LD50, and six from each group were<br>
challenged with 100x LD50 of wild type Listeria monocytogenes. These mice were<br>
monitored for survival for ten days. Thirty-two days post first vaccination, three additional<br><br>
mice from each group were challenged with 2xLD50 of wild type and three days later<br>
sacrificed and the spleen and liver were isolated and cultured for growth of Listeria. In<br>
addition, the anti-Listeria antibody titer of the mice sera was assessed by doing an ELISA<br>
assay. Frozen, ground Listeria in a sodium bicarbonate buffer was plated and incubated<br>
with serum from the vaccinated mice with serial dilutions, then bound antibody was<br>
detected with goat anti-mouse antibody conjugated to HRP. An HRP substrate was added<br>
and the level of antibody determined by quantitatively measuring the color change. These<br>
were compared to naive mice to assess Listeria specific antibody, where a sample was<br>
considered positive for Listeria if greater than one standard deviation above the<br>
measurement of a naive serum sample. The CFU per spleen or liver results are shown in<br>
Figures 14A,B, the sati-Listeria antibody titer is shown in Figure 15, and the survival<br>
results are shown in Figure 16. This study demonstrates good CFU reduction and<br>
protective immunity of the S-59 treated uvrAB strain with 3 or 5 vaccinations, approaching<br>
that of the untreated uvrAB strain, and is nearly as effective as the wild type strain.<br>
Table 15 Dosing of Balb/c mice for assessment of protective immunity, multiple<br>
vaccinations with S-59/UVA treated strains.<br><br><br><br>
EXAMPLE 14<br>
Demonstration of breaking of immune tolerance using S-59/UVA treated strains in a mouse<br>
model.<br>
[0209] DP-L4029 and DP-L4029 uvrAB strains expressing Gp-70-AH1A5 and<br>
OVA were S-59/UVA treated according to the second method of Example 13. Gp-70 is an<br>
autologous mouse antigen that is expressed by CT-26 tumor cells. The AH1A5 is a singles<br>
base mutation of the natural sequence which has been shown to induce an immune response<br>
when expressed in live strains (AH1 peptide is SPSYVYHQF (SEQ ID NO:20), AH1A5<br>
peptide is SPSYAYHQF (SEQ ID NO:21)). In a prophylactic immunization study, Balb/c<br>
mice were vaccinated intraveneously (100 ?L) in groups of 8 mice according to Table 16<br>
(day 7 post the first set of vaccinations, 3 mice per group were sacrificed and the spleens<br>
harvested). At day 21 post initial vaccination, the remaining 5 mice per group were<br>
injected intraveneously with 1 x 105 CT-26 colon epithelial tumor cells (ATCC) and<br>
monitored for survival.<br>
Table 16 Vaccine strains and treatment regimen.<br><br><br>
[0210] The T cell population of the harvested spleen cells was assessed by ICS<br>
according to Example 5, using LLO91, AH1, AH1/A5 peptides or P815 and CT26 cells<br>
(completely inactivated with 150 mM S-59 and 3 J/cm2 UVA) to stimulate the cells. The<br>
P815 cells serve as a negative control for CT26 whole cell stimulation, as the 1815 does<br>
not express gp70 antigen. The results are shown in Figure 17, indicating that the treated<br>
uvrAB mutants result in an AH1A5 or AH1 specific response that can be improved with<br>
additional vaccinations. The cells were also assessed by ELISPOT assay according to<br>
Examples. The cells were stimulated with either AH1A5 or AH1 peptides. The results are<br>
shown in Figure 18 A, B indicating an immune response to both the AH1A5 and the AH1<br>
with the uvrAB mutant strains.<br>
EXAMPLE 15<br>
Therapeutic vaccination of mice using psoralen attenuated Listeria strains with uvrAB<br>
deletion.<br>
[0211] Using C57B1/6 mice, B16.F10.MO5.10.H3 (OVA+, this is a subclone of the<br>
cells used in Example 4 which have increased homogeneity for OVA expression)<br>
melanoma tumor cells were injected into the mice (1 x 106 in 100 \xL HBSSIV7) to establish<br>
lung metastases. Listeria monocytogenes strains DP-L4029-OVA, DP-L4027-OVA, DP-<br>
L4038-OVA (ac(A/4-6lT double mutant), and DP-L4029uvrAB-OVA were used for<br>
vaccinating groups often mice. The DP-IA029uvrAB-OVA strain was used with and<br>
without S-59 treatment (&gt;8 log kill by first method of Example 13) and heat killed DP-<br>
L4029-OVA was used as a control along with HBSS only. The mice were vaccinated (100<br>
p.L IV in HBSS) on day 3 post tumor implant with the dose given in Table 16. Thirty days<br>
post tumor implant, five mice per group were sacrificed and the lungs harvested. The<br>
number of metastases per lung were counted. The remaining five mice per group were<br>
monitored for survival. The number of lung mestastases and median survival days are<br>
indicated in Table 17. The lungs for the actA', czctA' OVA, and actA'uvrAB' OVA S-<br>
59/UVA treated and heat killed are shown in Figure 19A, number of lung meta stases<br>
plotted in Figure 19B, and the survival is plotted in Figure 19C. This data shows mat the S-<br>
59/UVA treated uvrAB mutant can be administered as a therapeutic vaccine, resulting in<br><br>
significantly reduced lung metastases and extended survival compared to non-yaccinated,<br>
heat killed control, or DP-L4029 without OVA.<br>
Table 17 Therapeutic vaccination of mice in an OVA lung tumor model.<br><br>
EXAMPLE 16<br>
Therapeutic vaccination with S-59 inactivated Listeria strains expressing gp70 mouse<br>
antigen.<br>
[0212] Using Balb/c mice, CT26 tumor cells (which express AH1) modified to<br>
express a human antigen (the human antigen being irrelevant for this experiment) were<br>
injected into the mice (2 x 10s in 100 ?L FV in HBSS) to establish lung metastases.<br>
Listeria monocytogenes strains DP-L4D29, DP-L4029-AH1A5, DP-L4029M-V/-
and DP-L4406actA-AH1A5(actA/inlBdouble mutant) were used for vaccinating groups of<br>
thirteen mice. The AH1A5 strains also express the OVA antigen. The DP-LA029uvrAB-<br>
AH1A5 strain was used without treatment, heat killed, or S-59 treated (per second method<br>
of Example 13). The mice were vaccinated (100 ?L HBSS IV) beginning 4 days after the<br>
tumor implant according to Table 18. Nineteen days post tumor implant, three mice per<br>
group were sacrificed and the lungs harvested. The number of metastases per lung were<br>
counted. The remaining ten mice per group were monitored for survival. The results for<br>
the lung metastases are shown in Figure 20 A (lung pictures) and 20B (number of lung<br><br>
metastases plotted) and survival is indicated in Table 18 and Figure 20C (AactA samples)<br>
and 20D (AactAAuvrAB samples). The AH1A5 antigen is endogenous to the mice, such<br>
that any immunization effect would be breaking immune tolerance in the mice. The results<br>
indicate that the S-59 treated uvrAB mutant strain is able to break tolerance in the mice,<br>
resulting in significantly reduced lung metastases and extended survival. The therapeutic<br>
effect is improved when the vaccine is dosed over three days compared to a single<br>
vaccination (total dose delivered over three days is equal to the single day).<br>
Table 18 Therapeutic vaccination of mice using Listeria modified to express AH1A5.<br><br><br>
EXAMPLE 17<br>
Evaluation of S-59/UVA treated Listeria monocytogenes localizatioa in dendritic cells<br>
using fluorescence microscopy.<br>
[0213] The uptake and distribution of Listeria monocytogenes within an antigen<br>
presenting cell was evaluated by fluorescence microscopy. The dendritic cell line DC 2.4<br>
was cultured on coverslips in a Petri dish at 5 x 105 cells per dish in complete RPMI media,<br>
RPMI-1640 (Gibco) supplemented with 10% FBS (Hyclone), lx Non-Essential Amino<br>
Acids (Cellgro), 5 x 1041.U. Penecillin/5 x 104 ?g Streptomycin (Irvine Scientific), 2 mM<br>
L-glutarnine (Irvine Scientific) and 1 nM Sodium Pyruvate (Sigma), and incubated<br>
overnight at 37 °C (this could be done similarly with other cell lines, e.g. macrophage<br>
J774). Stationary phase cultures of Listeria strains (DP-L4056, DP-L4027 (LLO-) and DP-<br>
L4056uvrAB) were prepared by seeding 3 mL of BHI media with a bacterial colony and<br>
growing at 30 °C overnight.<br>
[0214] The overnight cultures of Listeria were diluted 1:20 in fresh BHI media<br>
and stationary phase cultures at 30°C were grown to an OD600 of 0.5-0.6. Approximately 1<br>
mL of the overnight cultures for the DP-L4056 and DP-L4056uvrAB strains were also heat<br>
killed at 72 °C for 3-4 hrs. Frozen stocks of psoralen inactivated DP-L4-056 and DP-<br>
L4056uvrAB Listeria, prepared according to the second method of Example 13, were<br>
thawed and allowed to recover in stationary phase at 37° C for 1 hr. Prior to infection,<br>
OD600 readings of all Listeria preparations were obtained, the number of DC 2.4 cells per<br>
coverslip were counted and Multiplicity of Infection (MOI, number of bacteria per DC 2.4<br>
cell) for each strain were calculated. Fresh log phase cultures were used to infect cells at an<br>
MOI of 5, heat-killed cultures were used at an MOI of 20 and S-59/UVA treated strains at<br>
an MOI of 10.<br>
[0215] The coverslips were transferred to a 24 well dish and washed 3 times with<br>
RPMI lacking Pen/Strep and appropriate dilutions of the Listeria strains to give the desired<br>
MOI were incubated with the cells in Pen/Strep free media for 30 mins at 37° C. The<br>
coverslips were then washed 3 times with Pen/Strep free media and incubated at 37° C for<br>
another 30 mins. At the end of the incubation, the coverslips were washed and incubated in<br>
media with 50?g/ml Gentamycin for 4 hrs at 37 °C. Coverslips were then washed in PBS<br>
and fixed in 3.5% formaldehyde/PBS for 15 mins at room temperature. Post fixation,<br><br>
coverslips were washed/penneablized with TBS-Tx buffer (25 mM Tris-HCl pH 8.0, 150<br>
mM NaCl, 0.1% Triton X-100) and blocked in 1 % BSA/TBS-Tx for 15 mins at room<br>
temperature. Coverslips were stained with Rabbit anti-Listeria O antigen anti-serum<br>
(Difco) for 30 minutes at room temperature and washed in TBS-Tx buffer. Samples were<br>
then stained with Fluorescein labeled anti-rabbit secondary antibody (Vector Laboratories)<br>
and actin was stained with Rhodamine-Phalloidio (Molecular Probes). Coverslips were<br>
washed in TBS-Tx and mounted on slides in VectaShield+DAPI-hardset (Vector<br>
Laboratories) in order to stain for cell nuclei. Slides were allowed to dry for at least 8 hrs<br>
and cells were visualized on a Nikon TE300-U inverted microscope. Images were taken<br>
using a CCD Hamamatsu C4742-95-12NR camera and analyzed using Image-Pro software<br>
from Phase 3 Imaging Systems.<br>
[0216] Three images were taken for each field; one using a UV-2E/C filter<br>
(CHROMA Technology Corp, visualizes DAPI/nuclei), a second with a HYQ TRITC filter<br>
(CHROMA Technology Corp, visualizing actin) and a third using a B-1A (HYQ-FITC)<br>
filter (CHROMA Technology Corp, visualizing Listeria). The three images were then<br>
merged to determine if staining for Listeria co-localizes with staining for actin. Listeria<br>
that were unable to escape the phagolysosome appear green while those that were able to<br>
escape into the cytosol were able to nucleate actira and therefore appeared yellow due to the<br>
co-localization of actin (red) and Listeria (green). In order to quantitate the percentage of<br>
Listeria that was able to escape the lysosome, the total number of Listeria in the field weare<br>
counted and the number of Listeria that appeared yellow were determined by counting<br>
yellow bacteria or by confirming the presence of the actin firom the rhodamine image (see<br>
Figure 21 A). The number of Listeria that escaped the phagolysosome were divided by the<br>
total number of Listeria counted and the percentage of phagolysosomal escape was<br>
calculated, as reported in Table 19 and represented in Figure 2IB. The results indicate that<br>
the heat killed strains and the S-59/UVA treated wild type strain behave like the LLO"<br>
strain, i.e. can not escape the phagolysosome, while the uvrAB mutant that is S-59/UVA<br>
treated shows substantial ability to escape the phagolysosome.<br>
Table 19 Percentage of Listeria escaping the phagolysosome for DP-L4056 (+/- S-<br>
59/UVA, heat killed), DP-L4027, and DP-L4056uvrAB (+/- S-59/UVA,<br>
heat lolled).<br><br><br><br>
EXAMPLE 18<br>
Visualization of S-59 UVA treated Listeria monocytogenes uvrAR strains using Gram<br>
stain.<br>
[0217] Wild-type and uvrAB strains of Listeria monocytogenes were grown<br>
to an OD600 of 0-5. at which point 50 mL of solution was transferred fo a. clean flask and S-<br>
59 was added to a level of 2500 nM for the wild-type strain and 200 nM.for the uvrAB<br>
mutant strain. These samples were incubated at 37 °C at 300 rpm for approximately 1 hour<br>
(OD600 approximately 1.0, approximately 1 x 109/mL). A 1 mL aliquot was removed to<br>
assess the liter and the remaining was transferred to a 150 mm Petri dish and irradiated at a<br>
UVA dose of 6 J/cm2 (FX-1019), resulting in &gt; 8 log inactivation for both strains. The<br>
treated strains were stored frozen as described in Example 13. These were thawed, and<br>
diluted 1:10 into BHI medium in a 15 mL tube at a concentration of approximately 1-2 x<br>
109 per mL. These were incubated at 37 °C at 300 rpm and aliquots were removed at 0, 2,<br>
4, 6, 8 hours and overnight (approximately 18 hours). The aliquots were spread on. glass<br>
slides (approximately 50 ?L) and allowed to air dry. The smear was heat fixed by passing<br>
through a flame three times, then allowed to cool before Gram staining using Fisher Gram<br>
Stain Set (catalog # 282-407). The slides were viewed on a microscope and photographed<br>
and the negative images are shown in Figure 22. This clearly demonstrates the unique<br>
nature of the treated repair deficient strain, which shows chains indicating gene expression<br>
but is not able to divide such that the bacteria do not proliferate.<br><br>
EXAMPLE 19<br>
Construction of additional mutant Listeria strains.<br>
[0218] Preparation of mutant Listeria strains. Listeria strains were derived from<br>
10403S (Bishop et al., J. Immunol 139:2005 (1987)). Listeria strains with in-frame<br>
deletions of the indicated genes were generated by SOE-PCR and allelic exchange with<br>
established methods (Camilli, et al, Mot. Microbiol. 8:143 (1993)). The mutant strain LLO<br>
L461T (DP-L4017) was described in Glomski, et al, J. Cell. Biol. 156:1029 (2002),<br>
incorporated by reference herein. The actA' mutant (DP-L4029) is the DP-L3078 strain<br>
described in Skoble et al., J. of Cell Biology, 150: 527-537 (2000), incorporated by<br>
reference herein in its entirety, which has been cured of its prophage. (Prophage curing is<br>
described in (Lauer et al., J. Bacteriol. 184:4177 (2002); U.S. Patent Publication No.<br>
2003/0203472).) The LLO mutant(DP-L4027) (Lauer et al., J. of Bacteriology, 184:4177-<br>
4186 (2002)), and LLO A26 (DP-L4042) (Decatur et al, Science 290:992 (2000)) were also<br>
described previously. Construction of an actd'uvrAB' strain is described in the copending<br>
U.S. provisional application 60/446,051, filed February 6, 2003, as LA029/uvrAB (see, e.g.<br>
Example 7 of that application). DP-L4029uvrAB (actA~/uvrAB~) was deposited with ATCC<br>
October 3, 2003, assigned PTA-5563.<br>
[02 1 9] Construction of pKSV7-dl inlBfor deletion of inlB front Listeria by allelic<br>
exchange. Deletion of inlB from Listeria DP-L4029 (or from other selected mutant strains<br>
or from wild-type Listeria) can be effected by allelic exchange, as described by Camilli et<br>
al., Mol. Microbiol. 8:143-147 (1993). Overlapping PCR can be used to prepare the<br>
construct used in the allelic exchange procedure. The source of the internalin B gene is the<br>
sequence listed as Genbank accession number AL591975 (Listeria nuonocytogenes strain<br>
EGD, complete genome, segment 3/12; inlB gene region: nts. 97008-98963) and/or the<br>
sequence listed as Genbank accession no. NC_003210 (Listeria monocytogenes strain<br>
EGD, complete genome, inlB gene region; nts. 457008-458963) both of which are<br>
incorporated by reference herein in their entirety.<br>
[0220] In the primary PCR reactions, approximately 1000 bps of sequence<br>
upstream and downstream from th.e Listeria inlB gene 5' and 3' ends, respectively, are<br>
amplified using the following template and primers:<br>
Template: DP-L4056 orDP-L4029 genomic DNA<br>
Primer pair /(For amplification of region upstream from 5' end of inlB):<br><br>
Lm-9603 1F: 5'-GTTAAGTTTCATGTGGACGGCAAAG (SEQ ID<br>
NO:22) (Tm: 72°C)<br>
Lm-(3' inlB-R +) 97020R: 5'-<br>
AGGTCTTTTTCAGTTAACTATCCTCTCCTTGATTCTAGTTAT<br>
(SEQ ID NO:23) (Tm: 114°C)<br>
(The underlined sequence complementary to region downstream of InlB<br>
carboxy terminus.)<br>
(Amplicon Size (bps): 1007)<br>
Primer pair 2 (For amplification of region downstream from 3' end of inlB):<br>
Lm-(5' inlB-F +) 9891 IF: 5'-<br>
CAAGGAGAGGATAGTTAACTGAAAAAGACCTAAAAAAGA<br>
AGGC (SEQ ID NO:24) (Tm: 118°C) (The underlined sequence is<br>
complementary to region upstream of InlB amino terminus.)<br>
Lm-9997OP: 5'-TCCCCTGTTCCTATAATTGTTAGCTC (SEQ ID<br>
NO:2S) (Tm: 74°C)<br>
(Amplicorx size (bps): 1074)<br>
[0221] In the secondary PCR reaction, the primary PCR amplicons are fused<br>
through overlapping PCR, taking advantage of complementarity between reverse primer<br>
from pair 1 and the forward primer of pair 2. This results in precise deletion of inlB coding<br>
sequence: nts. 97021-98910=1889 bps. The following template and primers were utilized<br>
in the secondary PCR reaction:<br>
Template: Cleaned primary PCR reactions<br>
Primer pair:<br>
Lm-96043F: 5'-GTGGACGGCAAAGAAACAACCAAAG (SEQID<br>
NO:26)(Tm: 74°C)<br>
Lm-99964R: 5'-GTTCCTATAATTGTTAGCTCATTTTTTTC (SEQ ID<br>
NO:27)(Tm: 74°C)<br>
(Amplicon size (bps): 2033)<br>
[0222] A protocol for completing the construction process is as follows:<br>
[0223] The primary PCR reactions (3 temperature cycle) are performed using Vent<br>
DNA polymerase (NEB) and 10 ?l of a washed 30°C Listeria DP-L4056 OR DP-L4029<br>
overnight culture. The expected size of Listeria amplicons by 1% agarose gel (1007 bps<br>
and 1074 bps). The primary PCR reactions are gel purified and the DNA eluted with<br>
GeneClean (BIO 101).<br><br>
[0224] A secondary PCR reaction is performed, utilizing approximately equal<br>
amounts of each primary reaction as template (ca. 5 ?l). The expected size of the Listeria<br>
amplicon from the secondary PCR reaction is verified by 1% agarose gel (2033 bps).<br>
Adenosine residue are added at the 3' ends of Listeria dl inlB amplicon with Taq<br>
polymerase.<br>
[0225] The Listeria dl MB amplicon is then inserted into a pCR2.1-TOPO vector.<br>
The pCR2.1-TOPO-dl inlB plasmid DNA is digested with Xhol and Kpnl and the 2123 bp<br>
fragment is gel purified. The KpruTXhoI 2123 bp fragment is inserted into a pKSV7 vector<br>
that has been prepared by digestion with Kpnl and XSoI and treatment with CIAP (pKS V7-<br>
dlinlB). [Correct?] Tlie fidelity of dl MB sequence in pKSV7-dl MB is then verifiedL<br>
The inlB gene is deleted from desired Listeria strains by allelic exchange with pKSV7—dl<br>
MB plasmid.<br>
[0226] Construction of antigen-expressing strains. Mutant Listeria strains<br>
expressing a truncated form of a model antigen ovalbumin (OVA), the immunodominant<br>
epitope from mouse colorectal cancer (CT26) known, as AH1 (SPSYVYHQF; SEQ ED<br>
NO:20), and the altered epitope AH1-A5 (SPSYAYHQF; SEQ ID NO:21; Slansky et al,<br>
Immunity, 13:529-538 (2000)) were prepared. The p]?L2 integrational vector (Lauer et al.,<br>
J. Bacteriol 184:4177 (2002); U.S. Patent Publication No. 2003/0203472) was used to<br>
derive OVA and AH1-.A5/OVA recombinantListeriac strains containing a single copy<br>
integrated into an innocuous site of the Listeria genome.<br>
[0227] Construction of OVA-expressingListeria (DP-L4056). An antigen<br>
expression cassette consisting of hemolysin-deleted I_LO fused with truncated OVA an.d<br>
contained in the pPL2 integration vector (pPL2/LLO-OVA) is first prepared. The Listeria-<br>
OVA vaccine strain is derived by introducing pPL2/LLO-OVA into the phage-cured L.<br>
monocytogenes strain DP-L4056 at the PSA (Phage from ScottA) attachment site tRNAArg-<br>
attBB'.<br>
[0228] PCR is used to amplify the hemolysin-deleted LLO using the following<br>
template and primers:<br>
Source: DP-L4O56 genomic DNA<br>
Primers:<br>
Forward (Kpnl-LLO nts. 1257-1276):<br>
5 '-CTCTGGTACCTCCTTTGATTAGTATATTC (SEQ ID NO: 28)<br>
(Tm: LLO-spec: 52°C. Overall: 80°C.)<br>
Reverse (BamHI-XhoI-LLO nts. 2811-2792):<br><br>
5 '-CAATGGATCCCTCGAGATCATAATTTACTXCATCCC<br>
(SEQ ID NO:29)<br>
(Tm: LLO-spec: 52°C. Overall: 102°C.)<br>
[0229] PCR is also used to amplify the truncated OVA using the following template<br>
and primers:<br>
Source: pDP3616 plasmid DNA from DP-E3616 E. coli (Higgins et al., Mol.<br>
Molbiol. 31:1631-1641 (1999)).<br>
Primers:<br>
Forward (Mol-Ncol OVA cDNA nts. 174-186):<br>
5'-ATTTCTCGAGTCCATGGGGGGTTCTCATCATC (SEQ ID<br>
NO:30)<br>
(Tm: OVA-spec:6O°C. Overall: 88°C.)<br>
-Reverse (XhoI-Notl-HindIII):-<br>
5'-GGTGCTCGAGTGCGGCCGCAAGCTT (SEQ ID NO:31)<br>
(Tm: Overall: 82°C)<br>
[0230] One protocol for completing the construction process involves first cutting<br>
the LLO amplicon with Kpnl and BamHI and inserting the KpnI/BamHI vector into the<br>
pPL2 vector (pPL2-LLO). The OVA amplicon is then cut with Xhol and Notl and inserted<br>
into the pPL2-LLO which has been cut with XhoI/Notl. (Note: The pPL2 vector does not<br>
contain any Xhal sites; pDP-3616 contains one Xhol site, that is exploited in the OVA<br>
reverse primer design.) The construct pPL2/XLO-OVA is verified by restriction analysis<br>
(KpnI-LLO-XhoI-OVA-NotI) and sequencing. The plasmid pPL2/LLO-OVA is introduced<br>
into E. coli by transformation, followed by introduction and integration into Listeria (DP-<br>
L4056) by conjugation, exactly as described by Lauer et al. (or into another desired strain<br>
of Listeria, such as an inlBT mutant or an inlBactA' double mutant).<br>
[0231] A description of the insertion of an antigen expression cassette that<br>
expresses OVA can also be found in Example 8 of the U.S. provisional application entitled<br>
"Free-Living Microbe Based Vaccine Compositions", US Serial No. 60/511,869, filed<br>
October 15, 2003.<br>
[0232] Construction of Listeria strains expressing AH1/OVA or AHZ-A5/OVA. To<br>
prepare Listeria expressing either the AH1/OVA or the AH1-A5/OVA antigen sequences,<br>
inserts bearing the antigen are first prepared from oligonucleotides and then ligated into the<br>
vector pPL2-ULO-OVA (prepared as described above).<br><br>
[0233] The following oligorrucleotides are used in preparation of the AH1 or AH1-<br>
A5 insert<br>
AH1 epitope insert (Clal-PstI compatible ends):<br>
Top strand oligo (AHC1 Top):<br>
5'-CGATTCCCCTAGTTATGTTTACCACCAATTTGCTGCA (SEQ ID<br>
NO:32)<br>
Bottom strand oligo (AH1 Bottom):<br>
5'-GCAAATTGGTGGTAAACATAACTAGGGGAAT (SEQ ID NO:33)<br>
AH1-A5 epitope insert (ClaI-AvaII compatible ends):<br>
The sequence of the AH1-A5 epitope is SPSYAYHQF (SEQ ID NO:21)<br>
(5'-AGT CCA AGT TAT GCA TAT CAT CAA TTT-3' (SEQ ID<br>
NO:34)).<br>
Top: 5'-CGATAGTCCAAGTTATGCATATCATCAATTTGC (SEQ ID<br>
NO:35)<br>
Bottom: 5'-GTCGCAAATTGATGATATGCATAACTTGGACTAT<br>
(SEQ ID NO:36)<br>
[O234] The oligonucletide pair for a given epitope are mixed together at an<br>
equimolar ratio, heated at 95 °C for 5 min. The oligonucleotide mixture is then allowed to<br>
slowly cool. The annealed oligonucleotide pairs are then ligated at a 200 to 1 molar ratio<br>
with pPL2-LLO/OVA plasmid prepared by digestion with the relevant restriction enzymes.<br>
Tlie identity of the new construct can be verified by restriction analysis and/or sequencing.<br>
[0235] The plasmid can then be introduced into E. coli by transformation, followed<br>
by introduction and integration into Listeria (DP-L4056) by conjugation, exactly as<br>
described by Lauer et al. (or into another desired strain of Listeria, such as an inlB mutant<br>
or an inlB'actA' double mutant).<br>
EXAMPLE 20<br>
Assessment of in vivo cytotoxic activity in mice vaccinated with Listeria monocytogenes.<br>
[0236] A series of studies were done to assess the ability of vaccinated mice<br>
to lyse antigen specific target cells in vivo. In the first study, Balb/c mice were vaccinated<br>
either intraveneously (IV) with Listeria monocytogenes strains DP-L4029 (actA), DP-<br>
L4029 expressing AH1/A5, and DP-3L4029 uvrAB' expressing AH1/A5. The AH1/A5<br>
expressing strains were also treated with S-59 UVA according to the second method of<br><br>
Example 13. The Listeria constructs expressing AH1-A5 also express LLO and OVA.<br>
Vaccinations were done on day 0, for all groups and additionally on days 1 and 2 for the S-<br>
59 treated strains at the dose (0. 1 LD50) indicated in Table 20. For each strain and control,<br>
two groups of 3 mice were vaccinated. A target cell population was prepared by harvesting<br>
the spleens of 20 naive Balb/c mice in RPMI1640 medium. The cells were dissociated and<br>
the red cells lysed. The white blood cells were counted and split into four equal<br>
populations. Each group was pulsed with a specific peptide, either target AH1<br>
(SPSYVYHQF (SEQ ID NO:20), fiom SynPep, Dublin, CA), target AH1-A5<br>
(SPSYAYHQF (SEQ ID NO:21), SynPep), or two populations pf control ((3-gal,<br>
TPHPARIGL (SEQ ID NO:37)), at 0.5 ug/mL for 90 minutes at 37 °C. Cells were then<br>
washed 3 times in medium., and twice in PBS + 0.1% BSA. Cells were resuspended at<br>
1 x 107 per mL in warm PBS + 0.1 % BSA (10 mL or less) for labeling with<br>
carboxyfluorescein diacetate succinimidyl ester (CFSE, Molecular Probes, Eugene, OR).<br>
To the target cell suspensions, 1.25 ?L of a 5mM stock of CFSE was added and the sample<br>
mixed by vortexing. To the control cell suspensions, a ten-fold dilution of the CFSE stock<br>
was added and the sample mixed by vortexing. The cells were incubated at 37 °C for<br>
10 minutes. Staining was stopped by addition of a large volume (&gt;40 mL) of ice-cold PBS.<br>
The cells were washed twice at room temperature with PBS, then resuspended and counted.<br>
Each cell suspension was diluted to 50 x 106 per mL, and 100 ?L of each population was<br>
mixed and injected via the tail vein of either naive or vaccinated mice on day 6. For each<br>
strain or control, the group of 3 mice was injected with P-gal and AH-1 or P-gal and AH1-<br>
A5. After 12-24 hours, the spleens were harvested and a total of 5 x 106 cells were<br>
analyzed by flow cytometry. The high (target) and low (control) fluorescent peaks were<br>
enumerated, and the ratio of the two was used to establish the percentage of target cell lysis<br>
relative to the HBSS control population. The results are shown in Table 20 and Figure<br>
23A. (The tables in this example show averages for three mice, while the figures are<br>
representative histograms from individual mice for the indicated samples (not necessarily<br>
the same mouse).) The vaccination using the S-59 treated stains shows a slightly better<br>
response to AH1 for the uvrAB mutant and a significantly higher response to AH1-A5 for<br>
the uvrAB mutant relative to the S-59 treated actA strain.<br><br>
Table 20 In vivo cytotoxicity of Balb/c mice vaccinated as indicated on day 0, also<br>
days 2 and 3 for S-59 treated strains.<br><br>
[0237] This study was repeated with an additional vaccination at day 14 for<br>
all groups and additionally days 15 and 16 for the S-59 treated strains. The labeled target<br>
cells were injected on day 20. The results are shown in Table 21 and Figure 23B. The<br>
response to the S-59 treated uvrAR mutant can be significantly improved with a boost<br>
vaccination, this is not the case for the S-59 treated actA' strain.<br>
Table 21 In vivo cytotoxicity of Balb/c mice vaccinated as indicated on day 0 and 14,<br>
also days 2, 3, 15, and 16 for S-59 treated.<br><br>
[0238] A similar study was done using actA', actA" expressing OVA and<br>
actA'uvrAB' expressing OVA, including with and without S-59 treatment for the OVA<br>
expressing strains. This study used C57B1/6 mice. Groups of 6 mice were vaccinated day<br>
0 (also 1 and 2 for S-59 treated) and three of each group was injected with labeled target<br>
cells on day 6. The remaining mice were vaccinated day 14 (also 15 and 16 for S-59<br>
treated) and injected with labeled target cells on day 20. In this study, the naive target<br><br>
spleen cells were pulsed with p-gal (low CPSE) or SL8 (high CFSE). The results are<br>
shown in Table 22 and Figure 23C. Again, the response to the S-59 treated uvrAB' mutants<br>
is enhanced significantly with a boost vaccination.<br>
Table 22 In vivo cytotoxicity of Balb/c mice vaccinated as indicated.<br><br>
EXAMPLE 21<br>
S-59/UVA treatment of Bacillus anthracis with and without uvrAB deletion.<br>
[0239] The allelic exchange methods detailed in Examples 7- 9 and Camilli et al.,<br>
Molecular Micro., 8:143-147 (1993) were "used to modify the Bacillus anthracis Sterne<br>
strain. The virulence of this strain is attenuated (pX01+, pX02").<br>
[0240] The uvrAB gene from Bacillus anthracis was identified (Genbank accession<br>
number AEO17040, Bacillus anthracisAm.es strain, section 17 of 18 of the complete<br>
genome, uvrAB genes coding sequence: nts. 212613-217471) and a plasmid based on<br>
pKSV7 with the uvrAB gene deletion was constructed (pKSV7-dl uvrAB) using Splice<br>
Overlap Extension (SOE) PCR and the steps described below:<br>
[0241] Primary PCR reactions: Approximately 1000 bps of sequences upstream and<br>
downstream from the B. anthracis uvrAB genes 5' and 3' ends, respectively, "were<br>
amplified.<br>
Template: B. anthracis Sterne genomic DNA<br>
Primer pair 1: Amplification of region 1000 bp upstream from 5' end of uvrB.<br>
(Amplicon Size (bps): 1029)<br>
Ba-225099F: 5'-CTGTGCTTTGCGAATGGAAAGAAGC (SEQ ID<br>
NO:38) (Tm: 74°C)<br><br>
Ba-(3' uvrA-R +) 226109R:<br>
5'-OTTITCATTCATACACTTAGACAAGCGTTGGCTTTTGCA<br>
CTTC (SEQ IE) NO:39) (Tm: 120°C) (Underlined sequence is<br>
complementary to region downstream of uvrA carboxy terminus.) or<br>
Ba-226109R: 5'-GACAAGCGTTGGCTTTTGCACTTC (SEQ ID<br>
NO:40) (Tm: 72°C).<br>
Primer pair 2: Amplification of region downstream from 3' end of uvrA.<br>
(Amplicon size (bps): 990)<br>
Ba-(3' uvrA-R +) 230779F:<br>
5'-CAAAAGCCAACGCTTGTCTAAGTGTATGAATGAAAACC<br>
GAGTGG (SEQ ID NO:41) (Tm: 126°C) (Underlined sequence is<br>
complementary to region upstream of uvrB amino terminus.) or Ba-<br>
230779F: 5'-AAGTGTATGAATGAAAACCGAGTGG (SEQ ID<br>
NO:42) (Tm: 70°C)<br>
Ba-231769R: 5'-CATATAAAGGTTCCACAATTGCCTTTTC (SEQ ID<br>
NO:43) (Tm: 76°C)<br>
[0242] Secondary PCR reaction: Fusion of primary PCR amplicons through SOE<br>
PCR, taking advantage of complementarity between reverse primer of pair 1 and the<br>
forward primer of pair 2. Results in precise deletion of uvrAB coding sequence: nts.<br>
226110-230779=4670 bps.<br>
Template: Cleaned primary PCR reactions<br>
Primer pair: (Amplicon size (bps): 1973)<br>
Ba-225118F: S'-GAAGCAGAAATGAAGCCAATACTCAATC (SEQ ID<br>
NO:44) (Tm: 78°C)<br>
Ba-231761R: 5'-GGTTCCACAATTGCCTTTTCAATAATC (SEQ ID<br>
NO:45) (Tm: 74°C)<br>
[0243] Construction: Primary PCR reactions (3 temperature cycle) are performed<br>
using Vent DNA polymerase (NEB) and Sterne strain genomic DNA. Four primary PCR<br>
reactions are performed both with and without primers used for splice overlap extension<br>
(SOE). (If reactions containing Ba-(3, uvrA-R +) 226109R or Ba-(3' uvrA-R +) 226109R<br>
primers did not yield significant amplicon product, then these primers on amplicons from<br>
reactions with Ba-225099F/Ba-226109R or Ba-230779F/ Ba-231769R primer pairs were<br>
used.) The expected size of anthracis primary amplicons by 1% agarose gel (1029 bps and<br><br>
990 bps) was verified. The reaction was cleaned with S6 columns (BioRad) or GeneClean<br>
(BIO 101).<br>
[0244] The secondary PCR reaction was performed, utilizing approximately equal<br>
amounts of each primary reaction as template (ca. 5 ?l) were performed. The expected size<br>
of the Listeria amplicon. from, secondary PCR reaction by 1% agarose gel (1973 bps) was<br>
verified.<br>
[0245] The anthracis dl uvrAB amplicon was inserted into pCR2.1-Blunt II-TOPO<br>
vector. TheplasmidpCR2.1-TOPO-dl uvrAB plasmid DNA was digested with KpnI and<br>
PstI and gel-purify 2033 bp fragment. The KpnIIPstI 2033 bp fragment was inserted into<br>
pKS V7 vector, that had been prepared by digestion with Kpnl and PstI and treatment with<br>
CIAP (pKSV7-dl uvrAB). The fidelity of dl uvrAB sequence in pKSV7-dl uvrAB was<br>
verified.<br>
[0246] The uvrAB genes weretleleted from B. anthracis Sterne by allelic-exchange<br>
with pKSV7-dl uvrAB plasmid. The plasmid pKSV7-dl uvrAB was introduced into the B.<br>
anthracis Sterne strain by electroporation selecting for chloramphenicol resistance. The<br>
electroporation was done using a freezing step that significantly increased the frequency of<br>
electroporation. B, anthracis culture was grown O/N in 3 ml BHI 0.5% glycerol shaking at<br>
37°C. 0.5 ml culture was transferred to 50 ml BHI 0.5 % glycerol (OD600=0.1) in 500 ml E-<br>
flask. The sample was incubated at 200 rpm 37°C. (or 0.1-0.2 ml to 25 ml BHI 0.5%<br>
glycerol in 250 ml flask:). At OD600- 0.6-0.8 (approx 1 hour 45 min), bugs were collected<br>
in 500 ml disposable sterile filter apparatus. The bugs were washed 3 x 25 ml each with<br>
cold electroporation buffer (1 mM HEPES 10% glycerol pH 7.4). The cells were<br>
resuspended in 1/20 original volume (2.45 ml of e-poration buffer for 50 ml culture) and<br>
kept on ice. 1- ?g (1 to 5 ?l of miniprep) of "very clean" unmethylated plasmid DNA to 0.2<br>
ml cells suspension in a 0.2 cm gap e-poration cuvette (control=no DNA). The sample was<br>
then kept on ice for 15 rain. The cells were then pulsed at 25 ?FD, 200u,2.5 kV (or,<br>
alternatively, 0.4 ml cells were pulsed in 0.4cm cuvette at 400H). Time constant was<br>
approximately 4-5 msec. Immediately after pulse, 1 ml BGGM (BHI containing 10%<br>
glycerol, 0.4% glucose and 10 mM MgC12) was added. The cells are transfered to a sterile<br>
polyprop. tube and incubated 37°C 1 1/2 hour, shaking. The cells are pelleted, resuspended<br>
in 200 ?l BGGM and plated on selective media.<br>
[0247] The pK SV7-dl uvrAB was integrated into the B. anthracis chromosome at<br>
41° C. pKSV7-rf/ uvrAB was allowed to excise and cure at the permissive temperature,<br><br>
resulting in chloramphenicol sensitive colonies. PCR primers were designed to detect the<br>
deletion on the chromosome. 20% of the chloramphenicol sensitive colonies harbored the<br>
deletion in the B anthrads chromosome. PCR analysis of the uvrAB strain indicated<br>
retention of the pXO 1 virulence plasmid.<br>
[0248] Two uvrAB' clones (clone 8 and clone 32A) were S-59-treated, along with<br>
the parent strain, by growing in BHI at 37 °C at 300 rpm to an OD600 of 0.3, at which point<br>
50 mL of solution was transferred to a clean flask and S-59 was added to the concenfrations<br>
indicated in Table 23. These samples were incubated at 37 °C at 300 rpm with vigorous<br>
shaking for approximately 1 hour (OD600 approximately 1.0, approximately 1 x 109/mL) - A<br>
1 mL aliquot was removed to assess the titer and the remaining was transferred to a 150<br>
mm Petti dish and irradiated at a UVA dose of 6 J/cm2 (FX-1019), resulting in a six-log<br>
reduction in titer, as compared to the parental strain, as indicated in Table 23, below, and<br>
Figure 24. This demonstrates a sensitivity to psoralen treatmentin B. anthrads feat is<br>
similar to that observed for Listeria monocytogenes uvrAB' strains.<br>
Table 23 Attenuation of Bacillus anthrads Sterne strain vs. uvrAB" mutant with<br>
psoralen S-59/UVA treatment.<br><br>
EXAMPLE 22<br>
Use of vaccines of the invention for the in vivo treatment of human cancers.<br><br>
[0249] As an example of the treatment or prevention of a human cancer, a vaccine<br>
comprising an. antigen-presenting cell that has been, loaded and activated by infection by a<br>
microbe modified so that the proliferation of the microbe is attenuated, wherein the<br>
microbial gene expression is substantially unaffected, is administered to an individual. The<br>
microbe can be prepared following the protocols of examples 4 and 5, wherein, any desired<br>
prokaryotic expression cassettes encoding human tamor antigen(s) are incorporated into the<br>
microbe, by utilizing, for example the pPL2 integration vector described in Example 8, or<br>
any modifications thereof, or by any methods that are common to those in the art.<br>
Antigen-presenting cells (APCs) are then loaded an.d activated with the modified microbes<br>
using methods such as those outline herein.<br>
[0250] The resulting APC vaccine may be formulated in crude, or preferably<br>
purified form. The vaccine composition may be prepared as a liquid suspension. In<br>
addition, they may be formulated with additives such as preservatives (e.g. thimerosal, 2-<br>
phenoxy ethanol), stabilizers (e.g. lactose, monosoctium glutamate), adjuvants (e.g.<br>
aluminum hydroxide, aluminum phosphate, cytokines), antibiotics (e.g. neomycin,<br>
streptomycin) or other substances. Formulations may be resuspended or diluted, in a<br>
suitable diluent such as sterile water, saline, isotonic buffered saline (e.g. phosphate<br>
buffered to physiological pH), or other suitable diluent.<br>
[0251] The vaccine may be administered by a variety of routes, including oral,<br>
nasal, intraveneous, intradermal, intraperitoneal, intramuscular, intralymphatic and<br>
subcutaneous routes, as well as by any route that is relevant for any given maligvnant or<br>
infectious disease. An effective amount of the vaccine will be administered to an<br>
individual for treatment. For a therapeutic treatment, an effective amount is a dose that will<br>
result in the desired immune response, wherein the immune response either slows the<br>
growth of the targeted tumors, reduces the size of the tumors, or preferably eliminates the<br>
tumors completely. The administration of the vaccine may be repeated at appropriate<br>
intervals, and may be administered simultaneously- at multiple distinct sites in tine<br>
vaccinated individual. For the prophylactic treatment, an effective amount is a dose that<br>
will result in a protective immune response such that the likelihood of an individual to<br>
develop the cancer is significantly reduced. The vaccination regimen may be comprised of<br>
a single dose, or may be repeated at suitable intervals until a protective immune response is<br>
established.<br>
[0252] The therapeutic treatment of an individual may be started on an individual<br>
who has been, diagnosed with a cancer as an initial treatment, or may be used in<br><br>
combination with other treatments. For example, individuals who have had tumors<br>
surgically removed or who have been treated with radiation therapy or by chemotherapy<br>
may be treated with the vaccine in order to reduce or eliminate any residual tumors in the<br>
individual, or to reduce the risk of a recurrence of the cancer. The proptiylactic treatment<br>
of an individual would be started on. an individual who has an increased risk of contracting<br>
certain cancers, either due to environmental conditions or genetic predisposition.<br>
EXAMPLE 23<br>
Generation of recombinant tumor Ag-secreting vaccines based on attenuated strains of<br>
Listeria.<br>
[0253] Chicken ovalbumin (OVA) fused with a truncated form of Listeriolysin O<br>
(LLO) to facilitate antigen secretion, and MHC class I processing was used as a model<br>
antigen in studies to evaluate the immunogenicity of selected attenuated Listeria strains.<br>
The tumor antigen expression cassette was incorporated site-specifically into an innocuous<br>
site on the chromosome of a panel of attenuated Listeria strains with the proprietary pPL2<br>
integration vector. The recombinant Listeria strains expressed and secreted the predicted<br>
modified LLO-OVA fusion protein as determined by Western blot analysis (data not<br>
shown). The growth of each of these recombinants in liquid broth culture as well as the<br>
intracellular growth kinetics was also indistinguishable from its parent, furthermore, the<br>
recombinant OVA-expressing strains were shown to have an IV LD50 that was within a<br>
factor of two of the unmodified parental strains (Table 24).<br>
Table 24. Selected strains of Listeria monocytogenes<br><br>
[0254] The integration vectors facilitate rapid derivation of multiple recombinant<br>
Listeria vaccine candidates. A single construct can be mated in parallel into any number of<br>
unique genetic backgrounds to rapidly create isogenic strains.<br><br>
EXAMPLE 24<br>
Psoralen-induced DNA crosslinking to produce non-viable but metabolically active<br>
Listeria.<br>
[0255] To ensure safety for a Listeria-based ex vivo antigen delivery platform, in<br>
addition to using genetically attenuated Listeria, the inventors have engineered a Listeria<br>
strain that can be fully inactivated through treatment with psoralen., yet is metabolically<br>
active and thus retains its ability to infect cells, escape from the pnagolysosome and<br>
promote the presentation of encoded antigens via the class I pathway. The engineered<br>
Listeria strain is exquisitely sensitive to inactivation with psoralerts, a group of compounds<br>
that form irreversible crosslinks in the genomes of bacteria after illumination with UVA<br>
light, so thar they are incapable of multiplying. Mutant strains of Listeria unable to repair<br>
psoralen-mediated DNA damage were created by deleting the ultraviolet light resistance<br>
(uvr) AB gene (uvrAB), which is required for nucleotide-excision xepair in. Listeria and<br>
other bacteria (Sancar et al., Ann. Rev. Biochem., 57:29-67 (1988)). The psoralen S-59 is<br>
one of a number of Cerus compounds used in the DNA crosslinking technology known as<br>
Helinx (Lin, L., Psoralen photochemical treatment of platelets, Science and Medicine,<br>
1998; Hei, et al., Transfusion, 39:239-48 (1999)). At a psoralen concentration that<br>
inactivates the Listeria uvnAB deletion mutants to the limit of detection, the parental, non-<br>
mutant strain having intact DNA repair mechanisms is more than four logs less sensitive to<br>
UVA light inactivation (Figure 25B). S-59/UVA inactivated Listeria uvrAB maintained<br>
their mitochondrial activity as determined in a MTT assay and retained their capacity to<br>
express their genomic repertoire, as determined by 35S-methionine-labeled pulse-chase<br>
experiments (Figure 26 and Figure 10). S-59/UVA inactivated Listeria uvrAB but not the<br>
inactivated parental strain demonstrated continued expression of their genetic repertoire.<br>
The expression level of the inactivated parental strain was significantly diminished,<br>
indicating that S-59/UVA treatment at the S-59 concentrations required for full inactivation<br>
significantly decreases expression of Listeria gene products, including likely the expression<br>
of an encoded tumor antigen.<br>
EXAMPLE 25<br>
Non-viable Listeria uvrAB retain the ability to infect DC and escape from the<br>
phagolysosome.<br><br>
[0256] In addition to preserving metabolic activity, it is important to demonstrate<br>
efficient antigen loading into the MHC class I pathway by S-59/UVA inactivated Listeria<br>
uvrAB strains upon infection of dendritic cells (DC). The inventors have now demonstrated<br>
that infection of DC with a Listeria mutant (DP-L4027) mat is unable to escape from the<br>
phagolysosome by virtue of deletion of the hly geae that encodes LLO eliminates<br>
presentation of antigens in context of MHC class I.<br>
[0257] To demonstrate escape from the phagolysosome of Listeria uvrAB infected<br>
DC, the inventors took advantage of the fact the cytoplasmic Listeria are surrounded by<br>
host-cell actin filaments, so-called "actin clouds", that can be visualized by fluorescences<br>
microscopy. Actin polymerization on the Listeria suxface is mediated by the bacterial<br>
ActA protein and other host cell factors. A murine DC cell line, DC2.4, was infected at<br>
multiplicity of infection (MOI) of 1 with Listeria wt, S-59/UVA inactivated Listeria uvr-AB<br>
and Listeria ALLO (DP-L4027) for 30 minutes at 37oC. Extracellular bacteria were<br>
removed carefully by several washes and DC were incubated for an additional 5 hours art<br>
37°C in the presence of gentamicin to prevent the growth of extracellular bacteria. DC2.4<br>
infected with wild-type Listeria or fully inactivated Listeria uvrAB demonstrated typical<br>
actin clouds or actin comet tails, typical for cytoplasmic localization of Listeria (Figure<br>
27). However, in DC2.4 cells infected with the Listeria LLO null mutant no co-<br>
localization of actin and Listeria could be observed, indicating that the bacteria were unable<br>
to escape from the phagolysosome. This result demonstrated that these DNA repair<br>
mutants retain the ability to escape from the phagolysosome and enter the cytosol of the<br>
infected cell, where antigen can be secreted, a requisite step for direct presentation via ttxe<br>
MHC class I pathway. See also Example 17, above, and Figure 21.<br>
EXAMPLE 2S<br>
Non-viable Listeria uvrAB efficiently load antigen into the MHC class I pathway of"<br>
infected dendritic cells (DC).<br>
[0258] Due to the unique ability of S-59 psoralen inactivated Listeria uvrAB to<br>
escape the phagolysosome within the infected cell, gene products secreted by cytosolic<br>
Listeria are processed and presented via the MHC class I pathway. To test the ability of S-<br>
59/UVA inactivated Listeria uvrAB to load antigen into the MHC class I pathway of DC,<br>
DC2.4 cells were infected at a multiplicity of infection (MOI) of 1 with the OVA-<br><br>
expressing Listeria strain, L4029 uvrAB OVA, inactivated with different concentrations of<br>
S-59. The parental Listeria OVA and heat-killed Listeria uvrAB OVA served as controls.<br>
Presentation of OVA peptide by DC2.4 on class I molecules following phagocytosis of<br>
Listeria was measured after incubation with B3Z cells. B3Z is a LacZ-inducible CD8+ T-<br>
cell hybridoma specific for the OVA257-264 (SL8) epitope presented on the murine Kb class I<br>
molecule (Sanderson, Int. Immunol, 6:369-76 (1994)). Class I-resnicted presentation of<br>
SL8 to B3Z cells results in the induction of P-gal synthesis by B3Z. The amount of p-gal<br>
produced can be measured by the hydrolysis of the chromogenic substeate CPRG- and is an<br>
indication, of the amount of SL8/ Kb complexes presented on the surface of APCs. As<br>
shown in Figure 9A and 9B, S-59/UVA inactivated Listeria uvrAB OVA strain, but not the<br>
cognate parental strain, maintained its capacity to load antigen into the MHC class I<br>
pathway independent of its ability to multiply. (This is the same data as described in<br>
Example 11.) Even at full inactivationusing'S-59 concentrations of 70 to 100 nM, more<br>
than 90% of B3Z activation was maintained. In contrast, the parental List&amp;ria OVA strain<br>
with intact DNA repair lost its ability to activate the B3Z T-cell hybridoma when higher<br>
concentrations of S-59 were used for inactivation. In contrast to the Lister-ia uvrAB OVA<br>
strain, B3.Z activation and the ability of th.e parental Listeria OVA strain to form colonies<br>
on BHI agar plates was closely correlated, suggesting that only viable List&amp;ria OVA are<br>
capable of infecting DC2.4 cells and loading antigen into the MHC class I pathway.<br>
Moreover;, heat-killed Listeria uvrAB 0VA did not result in B3Z activation. This result<br>
demonstrates the capacity of Listeria to load antigen into the MHC class I pathway can be<br>
unlinked from the requirement for multiplication using S-59/UVA inactivated Listeria,<br>
modified to prevent their capacity to repair psoralen-mediated DNA damage.<br>
[0259] To test the ability of Listeria uvrAB OVA to load antigen irxto the MHC<br>
class I pathway of primary DC, immature murine BM-DC were infected with fully<br>
inactivated S-59/UVA treated Listeria uvrzAB OVA. Viable Listeria uvrAB OVA, the<br>
parental strain and L4027 served as controls. As shown in Figure 28, BM-DC infected<br>
with the OVA-expressing but not the parent strains stimulated the B3Z cells in vitro. No<br>
significant difference between the live and non-viable S-59/UVA treated Listeria uvrAB<br>
mutant strain (L4029 uvrAB OVA) was observed, suggesting that MHC class I molecules<br>
of primary DC are efficiently loaded with L,isteria-derived peptides following the escape of<br>
the bacteria from the phagolysosome into the cytosol in spite of the inability of Listeria<br>
uvrAB to multiply. Importantly, Listeria actA OVA inactivated by heat-killing did not<br>
result in any significant presentation of OVA peptides in the MHC class I pathway<br><br>
suggesting that incubation of DC with heat-killed bacteria does not result in any significant<br>
antigen loading of MHC class I molecules.<br>
EXAMPLE 27<br>
Listeria directly infect and activate human DC.<br>
[0260] For the development of a potent antigen delivery platform it is widely<br>
thought that activation/maturation of DC is required in addition to efficiently delivering<br>
antigen into the MHC class I pathway. In situ, immature DC reside in peripheral tissues<br>
where they continuously take up and process antigen, but it is the encounter of an activation<br>
stimulus, such as that which bacteria provide, that initiates the activation/maturation<br>
process, leading to modulation of chemokine receptors and migration of DC to the T cell<br>
area of the draining lymph node. We assessedthe potency of wild-type Listeria (L4056) to<br>
induce phenotypic maturation and cytokine production of human monocyte-derived DC.<br>
As shown in Figure 29, encounter of human immature DC with Listens led to up-<br>
regulation of the activation markers, CD86 and HLA-DR (Fig. 29A), as well as the<br>
maturation marker, CD83 (Fig. 29B). Furthermore, the exposure of hixman immature DC<br>
to Listeria increased their immime-stimulatory capacity as shown by their ability to secrete<br>
high levels of pro-inflammatory cytokines, such as IL-12p70 and TNF-a (Fig. 29C).<br>
EXAMPLE 28<br>
S-59/UVA inactivated Listeria uvrAB OVA induce OVA-specific immunity in vitro:<br>
[0261] We assessed the potency of the S-59/UVA inactivated Listeria uvrAB OVA<br>
vaccine to induce OVA-specific immunity in vivo. Female C57BL/6 mice were vaccinated<br>
intravenously with 1 x 108 particles of S-59/UVA inactivated Listeria zivrAB OVA. The<br>
induction of OVA-specific immunity was assessed 7 days post vaccination. Strikingly,<br>
mice that received S-59/UVA inactivated Listeria uvrAB OVA but not the parent Listeria<br>
OVA strain mounted a significant OVA-specific CD8+ T cell response, as shown in Figure<br>
30. Furthermore, vaccination of mice with heat-killed Listeria uvrAB OVA did not result<br>
in. the induction of OVA-specific immunity.<br><br>
EXAMPLE 29<br>
Construction of two recombinant attenuated Listeria actA/uvrAB strains expressing full-<br>
length CEA containing either the native (CAP1) or the enhanced agonist cytotoxic T<br>
lymphocyte epitope (CAP1-6D).<br>
[0262] CEA is a 180 kDa large protein found in adenocarcinomas of endodermally<br>
derived digestive system, epithelium and fetal colon. CE-A is attached to the membrane of<br>
cells by a GPI-anchor. The protein contains 7 immunogLobulin-like domains and the C-<br>
terminus demonstrates homology with the Non-specific Cross-reacting protein, NCA, a<br>
member of the carcinoembryonic antigen gene family. We propose to construct the full-<br>
length CEA containing either the HLA*A0201-restricted. CEA native T cell epitope CAP 1<br>
(YLSGANLNL) (SEQ ID NO:51)or the enhancer agonist cytotoxic T lymphocyte peptide<br>
CAP1^6D (YLSGADLNL) (SEQ ID NO: 52) (Zaremba-et air, Cancer Res., 57:4570-7<br>
(1997)) that has been demonstrated to be more potent at inducing CEA-specific immunity<br>
in cancer patients (Table 25) (Fong et aL, Proc. Natl. Acad. Sri. U.S.A., 98:8809-14<br>
(2001)).<br><br>
[0263] The CEA tumor antigen expression plasmids are constructed on the pPL2<br>
backbone, a vector that integrates site-specifically into the Listeria genome (Lauer et aL, J.<br>
Bacteriol., 184:4177-86 (2002)). The two plasmids will toe constructed so that the secretion<br>
signal and PEST elements derived from Listeria LLO are fused genetically with full-length<br>
CEA cDNA. Starting from the 5' end of the gene construct, the fusion protein will consist<br>
of the N-terminal region of LLO to promote bacterial secretion fused to CEA. Precise<br>
linkage of the domains will be accomplished by overlapping PCR. The fidelity of all<br>
plasmid constructs will be confirmed by DNA sequencing.<br>
EXAMPLE 30<br>
Derivation of two attenuated recombinant Listeria strains containing pPL2 CEAwt and<br>
pPL2 CEA-610D integrated into Listeria strain L4029 uvrAB (AactA, AuvrAB), and verify<br>
expression and secretion of CEA. antigens.<br><br>
[0264] Integration of the pPL2-CEA constructs adjacent to the tRNAArg gene in the<br>
genome of the Listeria strain L4029 uvrAB are accomplished as described previously by<br>
Lauer et al., J. Bacteriol., 184:4177-86 (2002). Briefly, plasmids are first introduced into<br>
E.coli strain SM10 by transformation, and then introduced into the desired strain of Listeria<br>
by conjugation. Listeria trans-conjugants are selected by chloramphenicol (pPLZ) and<br>
streptomycin (Listeria strain) selective media; the efficiency of this process is<br>
approximately 1 x 10-4. To ensure purity of trans-conjugants, and to ensure integration of<br>
the pPL2 backbone into the bacterial chromosome, a limited number of candidate colonies<br>
are passaged three times by streaking onto fresn selective media. Precise integration of the<br>
CEA construct into the Listeria genome are confirmed by colony-PCR.<br>
[0265] Antigen expression and secretion of the LLO-CEA fusion protein are<br>
determined by Western blotting of whole cell tysates; and TCA precipitated bacterial<br>
culture fluids. An LLO-specific rabbit polyclonal antibody and a CEA-specific monoclonal<br>
antibody are used to verify expression and secretion of the LLO-CEA fusion protein from<br>
recombinant Listeria. One can compare the biological properties of the recombinant<br>
Listeria strains expressing CEA to their respective parent strain. The growth kinetics in<br>
brain heart infusion (BHT) broth following inoculation by dilution of a stationary- phase<br>
culture 1:100 into fresh media are determined. In the past we have expressed proteins of<br>
similar or larger size in Listeria. However, recombinant protein expression of mammalian<br>
gene products in bacteria might pose a challenge dependent on each individual piotein. If<br>
CEA expression in Listeria poses a problem, one canl construct Listeria strains that express<br>
either fragments of CEA or the T cell mini-epitope. The HLA*A0201-restricted. CEA<br>
native T cell epitope CAP1 (YLSGANLNL) (SEQ ID NO:51) or the enhancer agonist<br>
cytotoxic T lymphocyte peptide CAP1-6D (YLSGADLNL) (SEQ ID NO:52) will be<br>
embedded in-frame within Ovalbumin (OVA) of our existing expression constructs,<br>
whereby the secretion signal and PEST elements derived from Listeria LLO are fused<br>
genetically with OVA. Expression and immunogenicity of T cell mini-epitopes are<br>
conserved as previously demonstrated with the gp70 T cell mini-epitopes, AH1 and AH1-<br>
A5 and B16 Trpl, Trp2, and gp100 (data not shown).<br>
EXAMPLE 31<br><br>
Establishment of conditions that fully inactivate Listeria actA/uvrAB CEA strains by S-<br>
59/UVA. treatment, yet retain optimal metabolic activity, tumor antigen, expression,<br>
infection of antigen presenting cells and phagolysosomal escape.<br>
[0266] Metabolic activity as a result of gene expression is best preserved with a<br>
minimal number of crosslinks. On can readily establish conditions for the minimal amount<br>
of S-59/UVA. treatment that fully inactivates Listeria actA/uvrAB CEA vaccines, leaving<br>
antigen expression levels intact An example of inactivation conditions is the addition of S-<br>
59 psoralen to 200 nM in a log-phase culture of OD600 = 0.5, followed by iaactivation with<br>
6 J/m2 of UVA light when the culture reaches an optical density of one. Inactivation<br>
conditionsare optimized by varying concentrations of S-59, UVA dose, the time of S-59<br>
exposure prior to UVA treatment as well as varying the time of treatment during bacterial<br>
growth of the Listeria actA/uvrAB CEA strain. The parental Listeria strain is used as -<br>
control. Inactivation of Listeria (log-kill) is determined by the inability of the bacteria to<br>
form colonies on BHI (Brain heart infusion) agar plates. In addition, one can confirm the<br>
expression of CEA and virulence factors, such as LLO and p60, of S-59/UVA. inactivated<br>
Listeria using 35S-pulse-chase experiments to determine the synthesis and secretion of<br>
newly expressed proteins post S-59 / UVA inactivation. Expression of LLO aond p60 using<br>
35S-metabolic labeling can be routinely determined. S-59/UVA inactivated Listeria<br>
actA/uvrAB CEA will be incubated for 1 hour in the presence of 35S-Methionine. Antigen<br>
expression and secretion of the LLO-CEA fusion protein, endogenous LLO, atnd p60 will<br>
be determined of both whole cell lysates, and TCA precipitation of bacterial culture fluids.<br>
LLO-, p60- and CEA-specific monoclonal antibodies will be used for irnmuno-<br>
precipitation to verify the continued expression and secretion from recombinaoit Listeria<br>
post inactivation. The expression level of S-59/UVA inactivated Listeria actM/uvrAB CEA<br>
will be compared to our current Listeria-OVA. vaccine strain that results in the induction of<br>
potent antigen-specific T cell responses. One can select S-59/UVA conditions that lead to<br>
reproducible full inactivation with limited affect on expression levels of the assessed gene<br>
products.<br>
EXAMPLE 32<br>
Establishment of a protocol and vaccine strain for infection of human immature dendritic<br>
cells (DC) with inactivated (S-59/UVA) Listeria actA/uvrAB CEA vaccines, that results in<br>
efficient presentation of CEA in context of MHC Class I.<br><br>
[0267] Optimal conditions for ex vivo infection of DC are determined based on the<br>
results of three independent assays: (1) change in phenotype and cytokine profile of human<br>
immature DC upon infection, (2) the potency of Listeria-infected DC to induce an<br>
allogeneic T lymphocyte response, and (3) the potency of Listeria actA/uvrAB CEA<br>
infected DC to stimulate a CEA-specific HLA*A0201-restricted T cell lin-e in vitro.<br>
[0268] I. Determination and comparison of the phenotype and cytokine secretion<br>
profile of human immature DC infected with live and fully inactivatedListeria-CEA strains.<br>
Comparison of the activation of Listeria-infected human DC with commonly used<br>
activation signals such as LPS, INF-?, and ?-CD40.<br>
[0269] One can characterize and optimize the efficiency of S-59/UVA inactivated<br>
Listeria a'ctA/uvrAB'CEK strains to infecfand activate primary human DC_Human DC are<br>
enriched from immobilized peripheral blood as previously described (Fong et al., J. Virol.,<br>
76:11033-41 (2002). Briefly, PBMC are obtained by centrifugation over Ficoll-Hypaque<br>
(Pharmacia, Uppsala, Sweden), and then monocytes are depleted by density centrifugation<br>
through Percoll (Pharmacia) as previously described Mayordomo et al., Next. Med., 1:1297-<br>
302 (1995). Monocyte-depleted PBMC are incubated in RPMI1640 (BioWhittaker,<br>
Walkersville, MD) supplemented with 10% pooled human AB serum without the addition<br>
of exogenous cytokines. After a 24-h culture in. a humidified incubator at 37°C with 10%<br>
CO2, DC are further enriched from lymphocytes by centrifugation through a 15% (w/v)<br>
metrizamide gradient (Sigma, St. Louis, MO). The phenotype of the enriched DC<br>
population is verified by flow cytometry (HLA-DR expression and lack of CD3, CD14,<br>
CD19, and CD56 expression) and dextran uptake. To assess the infectivity of DC with<br>
Listeria, DC is incubated at different MOI with S-59/UVA inactivated LisZeria actA/uvrAB<br>
CEA strains for one hour. Live Listeria will be used as comparison. After extensive<br>
washes to remove any extracellular Listeria, infected DC is further incubated in the<br>
presence of 50 ?g/mL gentamicin to kill extracellular bacteria. Phenotypics changes upon<br>
infection of DC with Listeria LactALxcvrAB CEA strains are assessed by determining cell<br>
surface expression of CD80, CD83, CD86, and MHC class II using flow cytometry at<br>
different time points post infection. Expression, of T helper-1 and T helper-2 type<br>
cytokines is measured from the supernatant of infected DC cultures using Che Cytometric<br>
Bead Array Kit (Tharmingen). Infection and activation conditions are coittpared to<br>
commonly used stimuli such as LPS, TNF-a, and a-CD40. Infection conditions are<br><br>
selected that result in potent and consistent stimulation and activation of human DC in vitro<br>
as well as secretion of cytokines that are most similar to the parental live Listeria strain. If<br>
the overall infectrvity of DC isolated from peripheral blood without the use of cytokines is<br>
low, infection of DC prior to the density gradient centrifugation will be assessed.<br>
Moreover, additional sources of DC such as monocyte-derived DC will be assessed for<br>
their infectivity for non-viable and live Listeria. Briefly, human monoeytes are enriched<br>
using negative selection and suspended in medium (RPMI-1640+10% PCS) at 1 x 106<br>
cells/mL, supplemented with 1000 U/mL GrM-CSF and 1000 U/mL IL-
culture, the phenotype of the in vitro cultured DC population is verified by flow cytometry<br>
and dextran uptake. The phenotypic change as well as the cytokine secretion pattern of<br>
monocyte-derived DC upon Listeria infection is assessed as described previously.<br>
[0270] 2. Determination and comparison of the stimulatory potency of human<br>
immature DC infected with live or fully inactivated Listeria-CBA strain_s to activate<br>
allogeneic T cells in vitro.<br>
[0271] To address the stimulatory capacity of the Listeria-infected DC population,<br>
one can determine their ability to stimulate primary allo-reactive T cells in mixed leukocyte<br>
reactions (MLR).' It is widely believed that the relative potency of an AJPC to elicit immune<br>
responses in vivo, which depends on their activation/maturation state, is reflected by their<br>
capacity to stimulate an allogeneic T cell response in vitro (Jung et al., Immunity, 17:211<br>
(2002)). Briefly, DC are isolated and infected with fully inactivated Lisleria actAluvrAB<br>
CEA. The phenotype of the infected cell population will be verified by flow cytometry.<br>
Various numbers of irradiated (3000 rad) E&gt;C are co-cultured with 5 x 1O4 allogeneic<br>
responders in 96-well U-bottom plates (Costar, Cambridge, MA). PBMC from random<br>
donors are used as responders. After 6 days, the cultures are pulsed witfa. 1 ?Ci of [3H]<br>
thymidine for 18 hours. Cells are harvested, onto glass fiber sheets and the incorporation of<br>
[3H] thymidine will be determined by measuring the radioactivity on the scintillation<br>
counter. The stimulatory capacity of DC infected with non-viable Listeria is compared to<br>
DC infected with live Listeria as well as DC activated using stimuli sucb. as LPS, TNF-?,<br>
and a-CD40.<br>
[0272] 3. Assessment of the potency of Listeria-in£ecte&amp; human. DC to activate a<br>
CEA-specific HLA.*A0201-restricted T cell line in vitro. Comparison of immature human<br>
DC infected with either live or fully inactivated Listeria to peptide-pulsed DC.<br><br>
[0273] Phenotypic changes, cytokine secretions profile as well as the allo-<br>
stimulatoiy capacity of DC represent an indirect measure for the potency of DC to<br>
stimulate an antigen-specific T cell response in- vivo. The potency of DC to present the<br>
recombinant tumor antigen expressed by the frilly inactivated Listeria actA/uvrAB CEA<br>
strain is assessed on the basis of activation of a. CEA-specific HLA*A0201 -restricted T cell<br>
line generated by L. Fong (unpublished data). Briefly, DC is isolated from peripheral<br>
blood of HLA*A0201 positive donors as described in Milestone 3-1. Various numbers of<br>
irradiated (3000 rad) DC, infected under optimal conditions, are co-cultured with 5 x: 104<br>
CEA-specific HLA.*A0201-restricted T cells in 96-well U-bottom plates (Costar,<br>
Cambridge, MA). After 24 hours, cell supematants are collected. T cell activation is<br>
measured on the basis of IFN-7, GM-CSF, or DL-2 secretion. Secreted cytokines are<br>
determined using commercially available Cytornetric Bead Array kits (Pharmingen). The<br>
stimulatory capacity of DC infected with non-viable Listeria-is compared to DC infected<br>
with live Listeria as well as DC activated using stimuli such as LPS, TNF-a, and <x-cd40.></x-cd40.>
EXAMPLE 33<br>
Confirmation of potency of Listeria-loaded primary human DC to prime CEA-specific<br>
immunity in vitro and select the lead Listeria strain for further development.<br>
[0274] To confirm that S-59/UVA inactivated Listeria-vaSocted DC are capab-le of<br>
priming naive CEA-specific CD8+ T cell response in vitro, human immature DC, infescted<br>
under the established optimal conditions -with Lzsteria actA/uvrAB CEA, are used to<br>
stimulate naive T cells in vitro. The lead Listerfa actA/uvrAB CEA strain containing -either<br>
the native or altered T cell epitope is selected based on its potency to induce naive CEA-<br>
specific T cell responses as determined by three independent assays: (1) [3H] thymidine<br>
incorporation of the DC-primed T cell cultures; 
specific T cell cultures, measured in a 5ICr release assay; and (3) the frequency of CELA-<br>
specific T cells determined by peptide:MHC tetramer staining. Optimal infection is<br>
confirmed by phenotypic changes of DC, assessed by determining cell surface expression<br>
of CD80, CD83, CE&gt;86, and MHC class II using flow cytometry, as well as the cytolriaie<br>
profile secreted by the infected DC. For the induction of primary T cell responses, a<br>
constant number of CD45RA+ T lymphocytes (2. x 105/well) is co-incubated with varying<br>
numbers of irradiated (3,000 R) Listeria-loadsd DC for 7 days in 96-well, round-bottom<br><br>
microtiter plates. After 6 days, the cultures are pulsed with 1 uCi of [3H] thymidine for 18<br>
hours. Cells are harvested onto glass fiber sheets and the incorporation of [3H] thymidine<br>
will be determined by measuring the radioactivity on the scintillation counter<br>
(Wallac, Turku, Fmland). Furthermore, induction of CEA-specificT cells is assessed in a<br>
cytotoxic T cell assay. Briefly, 5 x 106 CD45RA+ T lymphocytes are cultured in parallel<br>
with irradiated (3,000 R) Listeria-hadscL DC at a 10:1 ratio in 24-well plates (Costar) at 5 x<br>
106 cells/1.5 ml of media. The cytotoxic activity of the T cells is assessed in a standard 4-<br>
hour51Cr-release assay after 7 days. Briefly, the target cell lines SW403, SW1417, A375,<br>
and T2 are incubated in 250 uCi of [51Cxj for 2 hours. During this labeling step, T2 cells<br>
are also incubated without or with the HLA*0201-restricted target peptides OAP1 and<br>
CAP1-6D. The target cell lines are washed three times with RPMI and plated in triplicate<br>
with at least 5,000 targets/well in 96-well U-bottom plates (Costar). Effector cells will be<br>
co-incixbafed with the 51Cr-Iabeled target cells at the* described effector/target ratios. After<br>
a 4-h culture, supernatants are harvested and counted in a Microbeta counter (Wallac,<br>
Turku, Finland). Percent specific lysis will be calculated by the formula:<br>
100% x (experimental release - spontaneous release) / (maximum release - spontaneous<br>
release). Maximum release is determined by lysis of target cells in PBS containing 0.5%<br>
Triton IX-100 (Sigma). Lastly, one can determine the frequency of CEA-spescific T cells<br>
post in vitro priming using MHC/tetramers presenting CAP1 or CAP1-6D, as described<br>
previously (Fong et al, Proc. Natl. Acad. Sci. U.S.A., 98:8809-14 (2001)). Cryopreserved<br>
CD45RA+ T cells obtained before in vitro priming is analyzed in parallel with the in vitro<br>
primed T cell cultures. A total of 1 x 1O6 cells will be stained with the corresponding<br>
HLA*AX)201 phycoerythrin-labeled MHC/tetrarnerfor 30 mim at room temperature.<br>
Antibodies to CD8 (used for positive gate) and to CD4, CD14, CD19, and CD56 (negative<br>
"dump" gate) are added at the recommended concentrations and incubated for 30 mim at<br>
4°C. Following the staining, samples are washed twice and analyzed with four-color flow<br>
cytometry. We have established the background for tetramer staining previously.<br>
20 vohmteer blood donors were assessed with the same methodology and had<br>
0.30% dfc 0.18% and 0.27% ± 0.14% to CEA605-6i3 and 610D tetramers, respectively (Fong<br>
et al., Rroc. Natl. Acad. Sci. U.S.A., 98:8809-14 (2001)).<br>
EXAMPLE 34<br>
Use of proteinase 3 or PR1 as a heterologous antigen.<br><br>
[0275] Although some of the procedures outlined in the specific examples above<br>
describe the use of CEA antigens as the antigen expressed by the modified Listeria, one of<br>
ordinary skill in the art will readily recognize that similar procedures may be used to<br>
prepare a modified Listeria that expresses a different antigen, such as a proteinase-3 or a<br>
proteinase-3 derived antigen, to infect dendritic cells in vitro or ex vivo in order to effect<br>
loading and activation/maturation. One of ordinary skill in the art will also recognize that<br>
the resulting DC vaccines may then be administered to an animal or patient to induce an<br>
immune response to proteinase-3 and/or PR1.<br>
[0276] For instance, the IA029-uvrAB Listeria strain described in the Examples<br>
above may be modified with a -vector comprising a pPL2 vector backbone or the like<br>
encoding the proteinase-3 gene and/or the PR1 epitope to integrate the antigen-expressing<br>
sequence into the genome of the Listeria. In one example, the PR1 antigen could be<br>
expressed as'panrfcf a fusion protein such as an LLO-0VA/PR1 fusion protein comprising a<br>
truncated LLO sequence fused to OVA in which the PR1 epitope has "been embedded. The<br>
sequence of such an antigenic protein (LLO-OVA/PR3) that could be expressed by the<br>
modified Listeria is shown in Figure 31.<br>
EXAMPLE 35<br>
Measuring the ability of mutant Listeria to escape the phagolysosome and promote Class I<br>
antigen presentation<br>
[0277] An exemplary protocol for assessing the ability of a particular candidate<br>
mutant Listeria to escape the phagolysosome of an antigen-presenting cell and promote<br>
Class I antigen presentation by the cell is as follows: First, DC2.4 cells are grown on<br>
coverslips. The cells are then infected with the desired Listeria strain (MOI=100). At 0.5<br>
hpi, the cells rinsed to wash away free Listeria. At 1 hpi, gentamicin is added at 50 ug/mL.<br>
At 5 hpi, coverslips are washed and fixed for in 3.5% formaldehyde. The coverslips are<br>
blocked, stained with rabbit sati-Listeria antibody (Difco), and detected with a goat-anti-<br>
rabbit FITC secondary (Vector Labs). Actin is detected with Phalloidin-rhodamine<br>
(Molecular Probes). The coverslips are mounted with Vectamount+E&gt;API (Vector Labs)<br>
and examined. See also Example 17 and Example 25, above.<br><br>
EXAMPLE 36<br>
Generation of Human Monocyte-Derived Dendritic Cells and Infection with Listeria<br>
Vaccines<br>
[0278] An outline of an exemplary protocol for generation of human monocyte-<br>
derived dendritic cells and infection with Listeria vaccines is presented below:<br>
[0279] Materials: Human peripheral blood (buffy coat from blood donor<br>
preferred); Ficoll-Hypaque (Amersham); dPBS w/o Ca, Mg (MediaTech); RPMI-1640 w/<br>
L-Glutamine (MediaTech); Fetal Bovine Serum, Defined, Heat Inactivated (HyClone);<br>
Human GM-CSF (R&amp;D Systems)-stock solution made at 500U/uX, and stored at -20°;<br>
Human IL-4 (R&amp;D Systems)-stocJc solution made at 200U/uL and stored at -20°; Costar<br>
24-well plates (Fisher).<br>
[0280] Monocyte Isolation Media (MIM): To ihake Solution 1 (Tso§motic Percoll),<br>
50 mL of NaCl solution (500 mL dH20,43.84 g NaCI (1.5M))is added to 450 mL Percoll<br>
and mixed. Solution 2 (PBS/Citrate) is prepared by mixing 1000 mL dH2O,205.6 mg<br>
NaH2PO4*2H2O (1.49mM), 1.30 g Na2HPO4 (9.15 mM)), 8.18 g NaCl (139.97 mM), and<br>
3.82g C6H5Na3O7*2H2O (13 mM) and bringing the pH to 7.2. 250 mL of isosmotic percoll<br>
is then mixed with 250 mL of PBS/citrate. The solution is sterile filtered and stored at 4°<br>
[0281] Culture media: RPMI-1640 W. GlutaMax (Gibco) -+ 10% Fetal Calf Serum<br>
(Defined, heat inactivated FCS from HyClone is used).<br>
[0282] Methods: Ficoll and MIM are wanned to room temperature. 20mLof<br>
Ficoll into into each of 2 50 mL conical tubes. Blood is diluted 2 fold with dPBS and<br>
mixed well. 25 mL of blood is layered on top of Ficoll in each tube. The tubes are<br>
centrifuged at 400 xg for 30 minutes at 18-20°.<br>
[0283] The monuclear interface is carefully harvested from the gradient, and placed<br>
into a clean 50 mL tube. The remainder of the tube is filled with dPBS. The tube is<br>
centrifuged at 100 xg for 15 minutes. This pellets the lymphocytes and monocytes, but<br>
leave the platelets suspended. The supernatant is aspirated. The steps of filling the<br>
remainder of the tube with dPBS, centrifuging, and aspirating are repeated two more times,<br>
for a total of 3 washes.<br>
[0284] The pellet is resuspended in 20 mL of dPBS. The suspension is layered onto.<br>
20 mL of MIM. The sample is centrifuged at 400 xg for 35 miautes at room temperature.<br><br>
The monocytes are harvested from the interface and transferee! into a clean tube contain ing<br>
culture media. If culturing DC to use with bacteria;, do not use antibiotics.)<br>
[0285] \ The sample is centrifuged at 400 xg for 10 minutes and the supernatant<br>
aspirated. The pellet is washed 4x in dPBS. After the final wash, cell pellet is resusperxded<br>
in RPMI-1640 +10% FCS. The sample is then counted on hemacytometer using Trypan<br>
Blue or using automated counter. The cell suspension is diluted to 1 x 106 cells per mL_<br>
For each mL of cell suspension, 500U GM-CSF and 200UIL-4 is added (1 uL per mL of<br>
each if stocks were made as described above). 1 mJL per well is plated into Costar 24-well<br>
plates. The plates are placedat 37°C, 5% CO2, 10094 humidity for 48 hours.<br>
[0286] On second day, feeding media for dendritic cells is made up. This consists<br>
of 0.5mL culture media (warm to 37° before use) per well cultured, with 500U/mL GM—<br>
CSF and 200U/mL DL-4. 0.5 mL from the top of eaxh well is aspirated and replaced with<br>
6.5 mL of fresh feeding media. Platesare placed at 37°~C, 5% CO2,100% humidity for -48~<br>
additional hours. Feeding is repeated on day 4. On. day 5, cells are ready for use. The cells<br>
should always be kept in GM-CSF and IL-4 containing media, or they will revert to<br>
macrophages. The dendritic cells are examined phenotypically on the cytometer looking at<br>
HLA-DR, CDla, CD83, and CD86.<br>
[0287] Listeria Infection of Human DC:<br>
[0288] The day 5 dendritic cells (DC) are pelleted and resuspended in fresh media<br>
with GM-CSF and LL-4 at 2x106 cells per mL. 500 uL of suspension is aliquoted to each<br>
well of a 24 well plate. Maturation stimuli or bacteria is added in 500 uL. 1 ug of LPS is<br>
used for maturation control. (1000U of IFN-y or 1 ?g of sCD40L can be added to augment<br>
this response.) For Listeria infection, between 10-100 Listeria per DC is used. Cells are<br>
infected for 1 hour, then extracellular bacteria is washed away and cells are resuspended in<br>
media containing 50?g/mL gentamicin. sCD40L will can be added to enhance DC survival<br>
and promote greater IL-12p70 release. lOOOU/mL EFN- ? can be added to augment<br>
maturation and IL-12p70 secretion. The DC are examined phenotypically on the cytomester<br>
looking at HLA-DR, CDla, CD83, and CD86.<br><br>
EXAMPLE 37<br>
Asporagenic B. anthracis vaccine strains<br>
[0289] The spoIIE in-frame deletion. The spoIIE region of B. anthracis is<br>
identified by homology to the same gene ia. B. subtilis. In order to isolate an in-frame<br>
deletion of B. anthracis SpoIIE, the spoIIE gene is first amplified by PCR and cloned it into<br>
pCR-Blunt II-TOPO (Invitrogen). Next, rrxost of the spoIIE gene is deleted by u-sing the<br>
technique of gene splicing by overlap extension (SOE) (Horton et al., Biotechniques 8:528-<br>
35 (1990)). This in-frame deleted spoIIE gene is cloned into the shuttle vector pKSV7,<br>
which carries a chloramphenicol-resistance gene and cannot replicate at 42°C (Smith et al.,<br>
Biochimie, 74:705-11 (1992)): pKSV7 containing the deleted spoIIE gene is then<br>
electroporated into B. anthracis, and cells are grown at 42°C in the presence of<br>
chloramptienicol to select for strains in which the plasmid has integrated by homologous<br>
recombination into the spoIIE gene. Furthex growth at 30°C without chloramphenicol<br>
selection allows excision and loss of the plasmid. Chloramphenicol-sensitive strains should<br>
be found at about 1%, and about half of them should contain the deleted spoIIE allele<br>
(Camilli et al., (1993)). The presence of the deletion is confirmed by PCR and S outhern<br>
blot analyses.<br>
[0290] The spoIIE/uvrAB double d&amp;letion strain. Starting with the spoIIE deletion<br>
strain, an in-frame deletion of the uvrA and uvrB genes is made. Once again, the genes of<br>
interest are amplified and and cloned into pCR-Blunt II-TOPO. Then we shall delete most<br>
of the UVKA and uvrB genes by the SOE teclinique. This in-frame deleted uvrAB region is<br>
cloned into pKSV7, and the construct is electroporated into the B. anthracis spoIIE deletion<br>
strain. Chloramphenicol-resistance is selected at 42°C in order to select for the integration<br>
of the plasmid into the uvrAB region. Growth at 30°C without drug selection is allowed in<br>
order to encourage the growth of segregants that have lost the plasmid. Chloramphenicol-<br>
sensitive colonies are picked and tested by 2PCR for loss of the uvrAB region, and that loss<br>
will be confirmed by Southern blot analysis.<br>
EXAMPLE 38<br>
A temperature sensitive recA mutant of B. anthracis<br>
[0291] To generate a temperature sensitive recA mutant of B. anthracis which<br>
grows well at 30C and is very sensitive to psoralen at 42C, a mutation is made in. B.<br><br>
anthracis which is analogous to the V246M mutation of the temperature sensitive recA<br>
mutant of E. coli, recA44 (Kawashima et at., 193:288-92 (1984)). To make the B.<br>
anthracis mutant, the sequence 245KWKNK250 (SEQ ID NO:46), which is conserved<br>
between E. coli and B. anthracis, is mutated. The V246M mutation is introduced into the<br>
cloned B. anthracis recA gene by mismatched oligonucleotide mutagenesis, using the<br>
Stratagene Quick Change kit The mutations are confirmed by sequence analysis, and the<br>
mutated gene is transferred into pKS V7, in order that they can be introduced into the<br>
chromosome of B. anthracis spoIIE uvrAB by allelic exchange. Alternatively, the recA<br>
gene from the B. anthracis strains is deleted and replaced with the re=cA44(ts) allele of E.<br>
coli. (It is known that B. anthracis recA functions in E. coli (Ko et al., J. Bacteriol<br>
184:3917-22 (2002)).)<br>
EXAMPLE 39<br>
Introduction of mutations in the active sites of B. anthracis antigens<br>
[0292] The lethal factor mutation H686A inactivates its protease activity, and the<br>
edema factor mutations K346Q and KL353Q (together) inactivate its axienyl cyclase activity<br>
(Brossier et al., Infect. Immun., 68:1781-6 (2000)). These mutations are introduced into B.<br>
anthracis strains to be used in vaccines, such as the spoIIE uvrAB and spoIIE uvrAB recAts<br>
strains. The lef (lethal factor) and cya. (edema factor, adenyl cyclase) genes are cloned and<br>
mutagenized with the Quick Change Kit (Stratagene) to create the nmtant genes. The<br>
mutant genes are then transferred to pKSV7 and finally introduced into the host pX01<br>
plasmid by allelic exchange.<br>
EXAMPLE 40<br>
The use of SOS regulatory sequences for expressing protective antigen at high levels<br>
[0293] Cheo et al (Cheo et al., J. Bacteriol., 175:5907-15 (1993)) have shown that<br>
the consensus sequence GAACN4GTTC (SEQ ID NO:47) defines the LexA repressor site<br>
for genes in the SOS response of B. subtilis. A similar consensus sequence upstream of the<br>
promoters for the B. anthracis recA and uvrAB genes, which are part of the SOS regulon,<br>
is to be located. To make a B. anthracis strain that expresses protective antigen at high<br>
levels, the protective antigen gene is put under the control of the SOS regulatory sequence<br>
and introduced it into B. anthracis spoIIE uvrAB strain, so that treatment with psoralen will<br><br>
cause high levels of protective antigen to be made. In order to insert this artificial gene into<br>
the B. anthracis chromosome, an integration vector, such as pPL2, is used (Lauer et al, J.<br>
Bacteriol, 184:4177-86 (2002)). The gene of interest, in this cases the protective antigen<br>
gene under control of a promoter, is inserted in the multicloning site. The plasmid is mated<br>
from E. coli into B. anthracis strains. Since it cannot replicate in. gram-positive bacteria, it<br>
can only be maintained by integration into the chromosome. The current pPL2 vector<br>
contains a pbage integrase and phage attachment site from L. monocytogenes, and<br>
therefore, inust be modified by removing the L- monocytogenes phage integrase gene and<br>
phage attachment site and replacing them with similar elements from a phage of B.<br>
anthracis, siich as gamma phage (Brown et al., J. Infect Dis., 96:34-9 (1955)). Also, the<br>
pPL2 vector typically contains chloramphenicol-resistance geaes for selection. Since drag<br>
resistance genes are undesirable for vaccine work, they are removed. One of the drug<br>
resistance genes has been'replaced by the gene for D-alanine racemase, which synthesizes<br>
D-alanine and allows a D-alanine auxotroph to grow on rich medium without the addition<br>
of D-alanine. The other drug resistance gene is replaced by the gene for glutamine<br>
synthetase, Avhich synthesizes glutamine and allows growth of a glutamine synthetase<br>
mutant bacterium on rich medium without glutamine.<br>
EXAMPLE 41<br>
Exemplary nrutant B. anthracis strains<br>
[0294] A variety of different mutant B. anthracis strains are prepared using<br>
combinations of the methods described in the Examples, above. Exemplary mutant B.<br>
anthracis strains to be used in vaccine compositions are listed in Table 26.<br><br><br><br>
1NER, nucleotide excision repair<br>
3Conditional recA strains under the control of a lad repressible promoter will also be derived<br>
4HR, homologous recombination<br>
EXAMPLE 42<br>
Characterization of protein expression levels, including protective antigen and capsule, in<br>
psoralen-inactivated B. anthracis strains<br>
[0295] To show that inactivated B. anthracis strains can still metabolize, the cells<br>
are incubated in minimal medium with bicarbonate (Thome et al., J. Gen.<br>
Microbiol.,17:505-16 (1957)). After such, incubation the cells are removed by<br>
centrifugation and save the supernatant. The supernatant is subjected to SDS-<br>
polyacrylamide gel electrophoresis. After staining with Coomassie Blue, protective axitigen<br>
stands out, and its presence is confirmed by Western blot analysis (Brossier et al., Infect.<br>
Immun. 68:5731-4 (2000)) and by mass spectometry. In addition, mass spectrometry is<br>
used to identify the other proteins that are excreted under these conditions, using the<br>
methods described in Lenz et al., Proc. Natl. Acad. Sci. U.S.A., 100:12432-12437 (2003).<br>
In order to assess whether polyglutamate capsule is made under these conditions, pXO2,<br>
which encodes the genes for capsule synthesis, is introduced into the strains by traosduction<br><br>
and (Green et al., Infect. Immun., 49:291-7 (1985). Capsule is measured by rocket<br>
immunoelectrophoresis (Uchida et al., Mol. Microbiol, 23:1229-40 (1997)).<br>
EXAMPLE 43<br>
Characterization of the humoral and mucosal responses in Swiss Webster and A/J mice<br>
immunized with attenuated B. anthracis strains<br>
[0296] Mouse Immunization. Mice are injected with the S-59AUVA vaccines by the<br>
intramuscular (IM) or the subcutaneous (SC) routes to determine which route of<br>
immunization results in the best bacterial-specific humoral and cellular responses.<br>
Intranasal (IN) immunization of mice is also tested to assess mucosal responses induced by<br>
the candidate vaccines. IN immunization "with 5 ul of a designated vaccine preparation into<br>
each nare of lightly anesthetized mice is performed as described previously (Boyafca, et al.,<br>
J. Immunol., 170: 5636-43 (2003)) Mice are immunized with 0.1 LD50 doses of the<br>
candidate vaccines. Any of the eight S-59/UVA inactivated vaccine candidates in which a<br>
median lethality level is not observed is given at an initial dose of 108 particles. Mice that<br>
are immunized by more than one route are not injected with a combined dose that exceeds<br>
the 0.1 LD50 dose, or is greater than 108.pax-ticles. Mice given multiple immunizations<br>
receive consistent vaccine doses with all injections. As immunization, on three consecutive<br>
days with S-59/UVA inactivated Listeria wvrAB resulted in increased Immoral and cellular<br>
immunity as compared to a single immunization, the same strategy is used with the B.<br>
anthracis strain vaccines. Mice are also given booster immunizations at 14 days and 28<br>
days following the primary immunization.<br>
[0297] Quantification of antibodies to PA, LF, EF, capsule, and whole bacteria.<br>
The mucosal and antibody responses in mice immunized with the various vaccine<br>
candidates are characterized. Sera is taken from the retroorbital plexus prior to<br>
immunization as well as 1 week after each immunization (We have IACUUC approval for<br>
a maximum of five retroorbital plexus procedures per mouse with blood sampling not to<br>
exceed one per week, and to be taken from alternating eyes). Saliva and nasal washes for<br>
measurement of IgA levels are performed at the time of sacrifice one week after the final<br>
imxnunization. The durability of the humoral and mucosal immunity induced by the<br>
candidate vaccines at 45 days after the final immunization is also characterized. Humoral<br>
and. mucosal responses against PA, capsule, and vegetative bacteria (Sterne strain) are<br>
determined by enzyme-linked immunosorbent assays (ELISAs), as published previously<br><br>
(Ballard et al., Proc. Natl. Acad. Sci. U.S.A, 93:12531-4 (1996); Rhie et al, Proc. Natl.<br>
AcadSci. U.SA., 100:10925-30 (2003)). Briefly, Immulon 96-well Maxisorp plates<br>
(Nalge Nunc) are first coated by 5ug purified PA, LF, EF, B SA conjugated with poIy-y-D-<br>
glutamic acid (PGA) capsule prepared as described previously.(Rhie et al., Proc. Natl.<br>
Acad. Sci. U.SA, 100:10925-30 (2003)), or with S-59 psoralen/UVA inactivated bacteria<br>
ground under liquid nitrogen using a mortar and pestle in 50xnM carbonate buffer (pH9.6)<br>
at 4°C for 16 h, and blocked with TSTA buffer (50mMTris QpH 7.6], 142 mM NaCl, 0.05%<br>
sodium azide, 0.05% Tween 20,2% bovine serum albumin). Serial two-fold dilutions of<br>
mouse plasma or mucosal secretions are added to the 96-well plates coated with PA, PGA-<br>
BSA, or Sterne respectively. Binding of Abs to the immobilized antigens is determined by<br>
incubation with isotype-specific peroxidase goat anti-mouse u, y, or a H chain-specific<br>
antibodies from Southern Biotechnology Associates (Birmingham, AL). Biotinylated rat<br>
anti-mouse y 1 (clone Gl-7.3), 7 2a (clone Rl 9-15), y 2b (clone R12-3), or y 3 (clone R40-<br>
82) H chain-specific mAbs (BD PharMingen, San Diego, CA) and streptavidin-conjugated<br>
peroxidase are used for IgG Ab subclass analysis (Cole, J. Bacteriol., 107:846-52 (1971);<br>
Cole et al., Basic Life Sci., 5B:487-95 (1975)). The colorinxetric reaction is developed by<br>
addition of ABTS substrate (Sigma-Aldrich, St. Louis, MO). End-point titers are expressei.<br>
as the reciprocal Iog2 dilution giving OD415 &gt; two standard deviations above those<br>
obtained with control, non-immunized mice.<br>
[0298] Enzyme-linked immunospot (ELISPOT) assay for the detection of Ig-<br>
secreting cells. The frequency of PA-specific Ig-secreting lymphocytes is determined by<br>
ELISPOT analysis (Boyaka et al., J. Immunol, 170:5636-43 (2003)). Briefly, spleens or<br>
cervical lymph nodes of vaccinated and control mice are rapidly dissected out and placed in-<br>
ice-cooled RPMI1640 medium and single cell suspensions are prepared. 96-well PVDF-<br>
based plates (BD Biosciences, San Jose) are coated overnight with 2.5 ug/ml purified PA<br>
(List Biological Laboratories, Campbell, CA). The plates are washed, blocked for 2 hrs at<br>
37°C with 200 ul complete RFMI, and serial dilutions of cell suspensions are added to 96-<br>
well plates. Cells are incubated on the plates for 6 hours at 37°C in 5% CO2. Antigen-<br>
specific Antibody Forming Cells (AFC) are detected with isotype-specific biotin-labeled<br>
anti-mouse u, y, or a H chain-specific antibodies (Southern Biotechnology Associates).<br>
After incubation at RT for 2 h, the plates are washed, and goat anti-biotin: lnm"Gold<br>
conjugate (GAB1; Ted Pella) is added for 1 hour at RT. Aftex extensive washing, 30 ul of<br>
the silver substrate (Silver Enhancing Kit; Ted Pella) is added into each well and the spot<br>
development is monitored. Spots in each well are counted using an automated ELISPOT<br><br>
plate reader (CTL, Cleveland). The humoral response is expressed as the number of<br>
antibody forming cells per 106 spleen or lymph node cells.<br>
[0299] Toxin Neutralization Assays. Neutralizing antibodies induced in mice<br>
immunized with the vaccine candidates are evaluated for the ability to protect the J774<br>
macrophage cell line from lethal toxin (PA+LF) (Mock et al., Annu. Rev. Microbiol.,<br>
55:647-71 (2001); Boyakaetal. (2003); Rhie et al. (2003)) Briefly, J774 cells (ATCC,<br>
Manassus, VA) are added to 96-well flat-bottom plates (Nunc) at 5 X104 cells/well and<br>
incubated for 12 hours at 37° C in 5% CO2. Test serum or mucosal secretions axe serially<br>
diluted two-fold in TSTA buffer. PAandLF (400 ng/ml PA and 40 ng/ml LP) are added to<br>
the antiserum dilutions. After incubation for 1 hour the antiserum/lethal toxin complex<br>
mixture is added to the cell suspension and incubated for an additional 5 hours. Cell<br>
viability is monitored by the MTT assay (absorbance measured at 540nm). Assays are<br>
performed in triplicate with a negative control (normal serum) and a positive control<br>
(MAbs, 14B7 and 1G3) (Mikesell et al., InfectImmun., 39:371-6 (1983); Starnfeach et al.,<br>
Nature, 9, (2003)) included in each plate. The mean and standard deviation of each<br>
triplicate sample dilution is calculated. The endpoint is expressed as the highest: serum<br>
dilution exhibiting 50% neutralization of the anthrax toxin as compared to normal control<br>
serum.<br>
EXAMPLE 44<br>
Characterization of the PA-, LF-, and EF-specific CD4+ T cell-mediated responses in A/J<br>
mice vaccinated with modified B. anthracis<br>
[0300] T cell Proliferation. CD4+ T cell proliferation are determined firom PBMC,<br>
spleen and. lymph node cells of vaccinated and naive A/J mice. Spleen and cervical lymph<br>
nodes are dispersed to obtain single cell suspensions as previously described (Boyaka et al.,<br>
/. Immunol, 162:122-8 (1999); Lillardet al., J. Immunol, 166:162-169 (200O; Little et<br>
aL, Infect. Immun., 65:5171-5 (1997)). CD4-+- T cells are isolated by negative selection<br>
using the Mouse CD4+ T cell isolation kit from Miltenyi Biotec (Auburn, CA) .. Purified<br>
CD4+ T cells from individual mouse spleens, from pooled lymph nodes or PBlMCs are<br>
cultured at 4 x 106 cells/ml and stimulated with varying concentrations of PA, X,F or EF in<br>
the presence of T-cell-depleted, non-dividing syngeneic naive spleen feeder cells (8 X 106<br>
cells/ml) in complete RPMI (RPMI supplemented with 10% FBS, 10mMHepes, 2 mM L-<br>
glutamine, 1mM Sodium Pyruvate, non-essential amino acids, 23.8 mM Sodium<br><br>
Bicarbonate, 5 X 10-5 M ?-Mercapthoethanol, 100 U/ml penicillin and 100 Ug/ml<br>
streptomycin). The replication of splenic feeder cells is arrested by brief photochemical<br>
treatment with S-59 psoralen. Cultures are incubated for 4 days at 37°C and 5% CO2 prior<br>
to addition of 0.5 uCi of tritiated thymidine ([3H]TdR) for the final 18 to 20 hours. The<br>
cells are harvested onto glass fiber sheets and the amount of incorporated thymidine is<br>
determined by measuring the radioactivity on the scintillation counter (Wallac, Turku,<br>
Finland).<br>
[0301] Analysis of PA-, EF- or LF-induced cytoUne responses. CD4+ T cells are<br>
isolated by negative selection using the Mouse CD4+ T cell isolatiom kit from Miltenyi<br>
Biotec (Auburn, CA). Purified CE&gt;4+ T cells from spleens or lymph node of individual<br>
mice are cultured in round-bottom 96-well plates at 1 X 105 cells/well and stimulated with<br>
varying concentrations of PA, LF or EF in the presence of T cell-depleted, non-dividing<br>
syngeneic naive spleen feeder cells (1 X 105 cells/well) in complete RPMI. The T cell-<br>
depleted spleen feeder cells are arrested by a brief photochemical treatment with S-59. T<br>
cell cultures are incubated for 2 days at 37°C and 5% CO2. Expression of T helper-1 and T<br>
lxelper-2 cytokines is determined from supernatants of antigen-stimialated CD4+ T cells<br>
using the Thl/Th2 Cytometric Bead Array kit (BD Phanningen, San. Diego, CA).<br>
EXAMPLE 44<br>
Characterization of the extent of protection against spore and letrxal toxin challenge in<br>
Swiss Webster and All mice at 45 days post last immunization dose with modified B.<br>
anthracis vaccines<br>
[0302] Protection of mice against lethal toxin challenge. IVfice immunized with<br>
selected candidate vaccines are challenged by tail vein injection with, lethal toxin, as<br>
described previously (Price et al., Infect. Immun., 69:4509-15 (2001 ); Rhie et al. (2003)).<br>
Lethal toxin is prepared by mixing recombinant PA and LF recombLnant proteins (List<br>
Biological Laboratories, Campbell, CA) as described (Rhie et al. (2003)). The lethal toxin<br>
IV LDsoper mouse is approximately 12 ug of PA mixed with 6 ug of LF. The median<br>
lethality in mice of freshly prepared lethal toxin is determined by tail vein injection over a<br>
0.1 -10 LD50 dose range of the published values. The protection studies will likely include<br>
lethal toxin challenge over a range of 5-10 times the LD50 dose. In this model, unprotected<br>
mice succumb within 24 h. Initially, death by anthrax is confirmed in selected mice by<br>
plating blood on tryptic soy agar and incubating overnight at 37° C. Plates are observed<br><br>
for colonies with 2-3rum typical anthracis-tike "ground glass" appearance. All mice<br>
treated with lethal toxin are monitored daily, and experiments are terminated after 2 weeks<br>
and all protected mice are sacrificed.<br>
[0303] Spore preparation. Sterne strain spores are prepared as described (Barnard<br>
and Friedlander, 1999). Briefly, single colonies are inoculated into 5 ml of FA medium.<br>
(3.3% tryptone, 2% yeast extract [dialyzed overnight against water], 0.2% L-histidine,<br>
0.8% Na2HPO4) 0.4% KH2PO4, 0.74% NaCl) contained in a 100-ml bottle and shaken for 5<br>
h at 37°C. One-tenth-milliliter aliquots are spread on L agar plates, and incubated at 37C'C.<br>
Bacterial lawns are scraped from the plates, washed extensively with sterile water, heat<br>
shocked for 30 min at 60°C, washed with water, purified on 58% Renografin-76 (Bristol-<br>
Myers Squibb, Princeton, N.J.) in water, as previously described (Palucka et al., Nature<br>
Medicine, 5:868-870 (1999)), and washed once more with water. The spores are then<br>
sedimented to a pellet at 10,000 x g and resuspended in 1% phenol in water. This yield of<br>
this process has been published to range from 0.5 x 109 to 5.0 x 109 spores per plate.<br>
[0304] Protection of mice against lethal spore challenge. The LD50 value of heat-<br>
shocked Sterne strain spores given by intramuscular (IM) injection is determined over a.<br>
dose range of 103 to 108 spores. To evaluate protection in vaccinated mice against<br>
inhalation anthrax, crxallenge experiments are also performed by intratrachial (IT) spore<br>
administration, as described previously (Brook et al., J. Med. Microbiol., 50:702-11<br>
(2001)). Briefly, the tongue of immobilized and anesthetized mice are gently pulled<br>
outward and laterally with forceps, and the vaccine is delivered using a syringe fitted with a<br>
blunt 1.5 inch 22-gau.ge needle bent at a gentle angle, approximately 1 inch from the tip.<br>
We anticipate that the Sterne strain LD50 value admiaiistered by IM or IT routes is<br>
approximately 103 in A/J mice, and up to 10-fold higher in Swiss Webster mice. The<br>
protection studies include up to 100 LD50 dose spore challenge. All mice treated with<br>
spores are monitored daily, and experiments are terminated after 2 weeks and all protected<br>
mice are sacrificed. In all challenge experiments, the mean time to death is determined in<br>
non-surviving cohorts.<br><br>
EXAMPLE 45<br>
Protective immunity of the Listeria vaccines against challenge -with Vaccinia expressing<br>
OVA model antigen in mice<br>
[0305] The vaccines of the invention show protective immunization against a<br>
Listeria challenge. To further illustrate the ability to immunize and protect against a<br>
pathogen, Listeria vaccines with or without OVA antigen and -with and without S-59 UVA<br>
treatment (second method of Example 13, above) are used to imrnunize against another<br>
microbe, e.g. vaccinia virus that expresses the OVA antigen (W-OVA). This will<br>
demonstrate that an antigen specific immunization against other microbes can be achieved<br>
with a Listeria vaccine.<br>
[0306] Vaccinia virus (WR strain) expressing OVA is obtained from La Jolla<br>
Research Institute and prepared in. Vero cells using Opticell chambers (BioCrystal, OH).<br>
Opticell chambers are seeded with. Vero cells in Eagle's Minimal Essential Medium<br>
supplemented with L-glutamine, P/S (penicillin/sterptomycin), NTEAA (non-essential amin-o<br>
acids), NaHCO3, and 10% FBS. When the cells are at approximately 75% confluence, the<br>
growth medium is removed and replaced with fresh medium containing approximately 1 x<br>
105 PFU/mL of W-OVA. When the monolayers show &gt;50% cytopathic effect, cells and<br>
supernatants are harvested, subjected to three freeze-thaw cycles-, clarified and stored at -8C<br>
°C. The titer is determined by a plaque assay on Vero-76 cells. Ovalbumin expression by<br>
the vaccinia is confirmed by a western blot analysis prior to injection into the mice.<br>
[0307] C57B1/6 mice (3 per group) are vaccinated IV according to Table 27 and<br>
challenged on day 7 with IP injection of 1 x 107 PFU of the W-OVA. On day 12, the mice<br>
are euthanized and the ovaries are harvested and observed for gross pathology. The ovaries<br>
are also assayed for vaccinia plaque forming units. Paired ovaries from individual mice are<br>
homogenized in 1 mL of buffer, frozen (liquid nitrogen) and tha wed (37 °C) three cycles,<br>
with vortexing between each cycle, then stored at -80 °C. To assay, samples are thawed,<br>
centrifuged at 4 °C to remove debris, and serially diluted for application to Vero cells.<br>
Vero-76 cells are plated in 6-well tissue culture plates. When the cell monolayers reach a<br>
confluence of about 70-85%, the medium is aspirated from each, well and the cells are<br>
inoculated with 1 mL of the appropriate dilution of ovary homogenate preparation. After at<br>
least one hour, the medium is aspirated and replaced with 3 mL of 1:1 2x growth<br>
medium: 1.5% agarose. Plaques are enumerated after 3-4 days o f culture.<br><br>
Table 27 Vaccination of mice with IListeria monocytogenes expressing OVA and<br>
challenged with vaccinia expressing OVA.<br><br>
*Initial dose, day 1 and 2 doses are 10-fold lower. All doses 100 ?L HBSS.<br>
[0308] All publications, patents, patent applications, and accession, numbers<br>
(including both polynucleotide and polypeptide sequences) cited herein are hereby<br>
incorporated by reference in their entirety for all purposes to the same extent as if each<br>
individual publication, patent or patent application were specifically and individually<br>
indicated to be so incorporated by reference.<br><br>
WE CLAIM:<br>
1. A vaccine comprising modified microbes, wherein the nucleic acid of the microbes has been<br>
modified by reaction with nucleic acid targeted compound that reacts directly with the nucleic acid so<br>
that the microbes are attenuated for proliferation, wherein the microbes are bacteria, protozoa, or<br>
fungi, and wherein the modified microbes express an antigen at a level sufficient for the vaccine to<br>
induce an immune response to the antigen in a host upon administration of the vaccine to the host.<br>
2. The vaccine as claimed in claim 1, wherein the nucleic-acid targeted compound is a nucleic<br>
acid alkylator.<br>
3. The vaccine as claimed in claim 2, wherein the nucleic acid alkylator is p-alanine, N-(acridin-<br>
9-yl), 2-[bis(2-chloroethyl)amino]ethyl ester.<br>
4. The vaccine as claimed in claim 1, wherein the nucleic acid targeted compound is activated<br>
by irradiation.<br>
5. The vaccine as claimed in claim 4, wherein the nucleic acid targeted compound is a psoralen<br>
compound activated by UVA irradiation.<br>
6. The vaccine as claimed in claim 5, wherein the nucleic acid targeted compound is 4'- (4-<br>
amino-2-oxa)butyl-4,5',8-trimethylpsoralen.<br>
7. The vaccine as claimed in claim 1, wherein the microbes comprise a genetic mutation that<br>
attenuates the ability of the microbe to repair their nucleic acid that has been modified.<br>
8. The vaccine as claimed in claim 7, wherein the microbes are defective with respect to a DNA<br>
repair enzyme.<br>
9. The vaccine as claimed in claim 8, wherein the genetic mutation is in one or more gene<br><br>
selected from the group consisting of phrB, uvrA, uvrB, uvrC, uvrD and recA, or in a functional<br>
equivalent of one or more gene selected from the group consisting of phrB, uvrA, uvrB, me, uvrD<br>
and recA.<br>
10. The vaccine as claimed in claim 9, wherein the microbes comprise genetic mutations in both<br>
uvrA and uvrB, or in functional equivalents of both uvrA and uvrB.<br>
11. The vaccine as claimed in claim 8, which is defective with respect to RecA, or the functional<br>
equivalent of RecA.<br>
12. The vaccine as claimed in claim 1, wherein the microbes are bacteria.<br>
13. The vaccine as claimed in claim 12, wherein the bacteria are Mycobacterium tuberculosis.<br>
14. The vaccine as claimed in claim 12, wherein the bacteria are Bacillus anthracis.<br>
15. The vaccine as claimed in claim 12, wherein the bacteria are Listeria monocytogenes.<br>
16. The vaccine as claimed in claim 15, wherein the bacteria comprise at least one mutation in<br>
both uvrA and uvrB.<br>
17. The vaccine as claimed in claim 16, wherein the bacteria comprise a mutation in the actA<br>
gene, the inlB gene, or both genes.<br>
18. The vaccine as claimed in claim 1, wherein the microbes comprise a heterologous nucleic<br>
acid sequence encoding an antigen.<br>
19. The vaccine as claimed in claim 1, wherein the vaccine further comprises a pharmaceutically<br>
acceptable carrier or an adjuvant.<br>
20. Isolated professional antigen-presenting cells comprising modified microbes, wherein the<br>
nucleic acid of the microbes has been modified by reaction with a nucleic acid targeted compound<br><br>
that reacts directly with the nucleic acid so that the microbes are attenuated for proliferation, wherein<br>
the microbes are bacteria, protozoa or fungi and wherein the modified microbes express an antigen at<br>
a level sufficient for the antigen-presenting cells to induce an immune response to the antigen in a<br>
host upon administration of the antigen-presenting cells to the host.<br>
21. The professional antigen-presenting cell as claimed in claim 20, which is a dendritic cell.<br>
22. The professional antigen-presenting cell as claimed in claim 20, wherein the nucleic-acid<br>
targeted compound is a nucleic acid alkylator.<br>
23. The professional antigen-presenting cell as claimed in claim 22, wherein the nucleic acid<br>
alkylator is P-alanine, N-(acridin-9-yl), 2-[bis(2-chloroethyl)amino]ehyI ester.<br>
24. The professional antigen-presenting cell as claimed in claim 20, wherein the nucleic acid<br>
targeted compound is activated by irradiation.<br>
25. The professional antigen-presenting cell as claimed in claim 24, wherein the nucleic acid<br>
targeted compound is a psoralen compound activated by UVA irradiation.<br>
26. The professional antigen-presenting cell as claimed in claim 25, wherein the nucleic acid<br>
targeted compound is 4'-(4-amino-2-oxa)butyl-4,5',8-trimethylpsoralen.<br>
27. The professional antigen-presenting cell as claimed in claim 20, wherein the microbes<br>
comprise a genetic mutation that attenuates the ability of the microbes to repair their nucleic acid that<br>
has been modified.<br>
28. The professional antigen-presenting cell as claimed in claim 27, wherein the microbes are<br>
defective with respect to a DNA repair enzyme.<br>
29. The professional antigen-presenting cell as claimed in claim 28, wherein the genetic mutation<br>
is in one or more gene selected from the group consisting of phrB, uvrA, uvrB, uvrC, uvrD and recA,<br>
or in a functional equivalent of one or more gene selected from the group consisting of phrB, uvrA,<br><br>
uvrB, uvrC, uvrD and recA.<br>
30. The professional antigen-presenting cell as claimed in claim 29, wherein the microbes<br>
comprise genetic mutations in both uvrA and uvrB, or in functional equivalents of both uvrA and<br>
uvrB.<br>
31. The professional antigen-presenting cell as claimed in claim 29, wherein the microbes are<br>
defective with respect to RecA, or'a functional equivalent of Rec A.<br>
32. The professional antigen-presenting cell as claimed in claim 21, wherein the microbes are<br>
bacteria.<br>
33. The professional antigen-presenting cell as claimed in claim 32, wherein the bacteria are<br>
Mycobacterium tuberculosis.<br>
34. The professional antigen-presenting cell as claimed in claim 32, wherein the bacteria are<br>
Listeria monocytogenes.<br>
35. The professional antigen-presenting cell as claimed in claim 30, wherein the bacteria<br>
comprise at least one mutation in both uvrA and uvrB.<br>
36. The professional antigen-presenting cell as claimed in claim 20, wherein the bacteria<br>
comprise a heterologous nucleic acid sequence encoding an antigen.<br>
37. A vaccine comprising the professional antigen-presenting cell as claimed in claim 20.<br>
38. The vaccine as claimed in claim 15, wherein the bacteria comprise a genetic mutation that<br>
attenuates their ability to repair their nucleic acid.<br>
39. The vaccine as claimed in claim 38, wherein the bacteria are defective with respect to at least<br>
one DNA repair enzyme.<br><br>
40. The vaccine as claimed in claim 38, wherein the bacteria are attenuated with respect to UvrA,<br>
UvrB, or both UvrA and UvrB.<br>
41. The vaccine as claimed in claim 40, wherein the bacteria comprise a genetic mutation in the<br>
uvrA gene, the uvrB gene, or both the uvrA and uvrB genes.<br>
42. The vaccine as claimed in claim 38, containing a pharmaceutically acceptable carrier or<br>
adjuvant, such as herein described.<br>
43. The professional antigen-presenting cell comprising the strain as claimed in claim 34,<br>
wherein the Listeria are defective with respect to at least one DNA repair enzyme.<br>
44. The vaccine as claimed in claim 14, wherein the bacteria comprise a genetic mutation that<br>
attenuates ability to repair their nucleic acid.<br>
45. The vaccine as claimed in claim 44, wherein the bacteria are defective with respect to at least<br>
one DNA repair enzyme.<br>
46. The vaccine as claimed in claim 45, wherein the bacteria are attenuated with respect to UvrA,<br>
UvrB, or both UvrA and UvrB.<br>
47. The vaccine as claimed in claim 46, wherein the bacteria comprise a genetic mutation in the<br>
uvrA gene, the uvrB gene, or both the uvrA and uvrB genes.<br>
48. The vaccine as claimed in claim 44, wherein the bacteria comprise one or more mutations in<br>
the lef gene, the cya gene, or both genes that decreases the toxicity of the strain.<br>
49. The vaccine as claimed in claim 8, wherein the microbes comprise a heterologous nucleic<br>
acid sequence encoding an antigen.<br>
50. The vaccine as claimed in claim 8, wherein the vaccine contains a pharmaceutically<br>
acceptable carrier or an adjuvant, such as herein described.<br><br>
51. The antigen-presenting cell as claimed in claim 30, wherein the microbes are bacteria.<br>
52. The antigen-presenting cell as claimed in claim 30, wherein the microbes comprise a<br>
heterologous nucleic acid sequence encoding an antigen.<br>
53. The vaccine as claimed in claim 18, wherein the antigen is a tumor antigen or an infectious<br>
disease antigen.<br>
54. The professional antigen-presenting cell as claimed in claim 36, wherein the antigen is a<br>
tumor antigen or an infectious disease antigen.<br>
55. A vaccine as claimed in claims 1, 12 or 15, which is capable of being used for preventing or<br>
treating a disease in a host or for inducing an immune response in a host.<br>
56 A professional antigen-presenting cell as claimed in claims 20, 32 or 34, which is capable of<br>
being used for preventing or treating a disease in a host or for inducing an immune response in a<br>
host.<br>
57. The vaccine as claimed in claim 12, wherein the bacteria are Salmonella.<br>
There is disclosed a vaccine comprising modified microbes, wherein the nucleic acid of the<br>
microbes has been modified by reaction with nucleic acid targeted compound that reacts directly with<br>
the nucleic acid so that the microbes are attenuated for proliferation, wherein the microbes are<br>
bacteria, protozoa, or fungi, and wherein the modified microbes express an antigen at a level<br>
sufficient for the vaccine to induce an immune response to the antigen in a host upon administration<br>
of the vaccine to the host.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1NC1LT0xOUC0yMDA1LSgwMy0wMS0yMDEyKS1GT1JNLTI3LnBkZg==" target="_blank" style="word-wrap:break-word;">1654-KOLNP-2005-(03-01-2012)-FORM-27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1NC1rb2xucC0yMDA1LWdyYW50ZWQtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">1654-kolnp-2005-granted-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1NC1rb2xucC0yMDA1LWdyYW50ZWQtYXNzaWdubWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">1654-kolnp-2005-granted-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1NC1rb2xucC0yMDA1LWdyYW50ZWQtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">1654-kolnp-2005-granted-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1NC1rb2xucC0yMDA1LWdyYW50ZWQtY29ycmVzcG9uZGVuY2UucGRm" target="_blank" style="word-wrap:break-word;">1654-kolnp-2005-granted-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1NC1rb2xucC0yMDA1LWdyYW50ZWQtZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">1654-kolnp-2005-granted-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1NC1rb2xucC0yMDA1LWdyYW50ZWQtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">1654-kolnp-2005-granted-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1NC1rb2xucC0yMDA1LWdyYW50ZWQtZXhhbWluYXRpb24gcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">1654-kolnp-2005-granted-examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1NC1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">1654-kolnp-2005-granted-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1NC1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">1654-kolnp-2005-granted-form 13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1NC1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSAxOC5wZGY=" target="_blank" style="word-wrap:break-word;">1654-kolnp-2005-granted-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1NC1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">1654-kolnp-2005-granted-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1NC1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">1654-kolnp-2005-granted-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1NC1rb2xucC0yMDA1LWdyYW50ZWQtZ3BhLnBkZg==" target="_blank" style="word-wrap:break-word;">1654-kolnp-2005-granted-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1NC1rb2xucC0yMDA1LWdyYW50ZWQtbGV0dGVyIHBhdGVudC5wZGY=" target="_blank" style="word-wrap:break-word;">1654-kolnp-2005-granted-letter patent.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1NC1rb2xucC0yMDA1LWdyYW50ZWQtcmVwbHkgdG8gZXhhbWluYXRpb24gcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">1654-kolnp-2005-granted-reply to examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1NC1rb2xucC0yMDA1LWdyYW50ZWQtc3BlY2lmaWNhdGlvbi5wZGY=" target="_blank" style="word-wrap:break-word;">1654-kolnp-2005-granted-specification.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="223391-method-for-transmitting-radio-frequency-signals-on-low-voltage-networks-and-an-associated-arrangement.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="223393-an-arrangement-in-a-spinning-machine-for-condensing-a-fibre-strand.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>223392</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1654/KOLNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>37/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>12-Sep-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>10-Sep-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>18-Aug-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CERUS CORPORATION</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>2411 STANWELL DRIVE, CONCORD, CA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>DUBENSKY, THOMAS, W., JR.</td>
											<td>15 KING AVENUE, PIEDMONT, CA 94611</td>
										</tr>
										<tr>
											<td>2</td>
											<td>BROCKSTEDT, DIRK, G.</td>
											<td>864 YORK STREET, APARTMENT 2, OAKLAND, CA 94610</td>
										</tr>
										<tr>
											<td>3</td>
											<td>BAHJAT, KEITH</td>
											<td>5208 CLOVIS COURT, CONCORD, CA 94521</td>
										</tr>
										<tr>
											<td>4</td>
											<td>HEARST, JOHN, E.</td>
											<td>101 SOUTHHAMPTON, BERKELEY, CA 94707</td>
										</tr>
										<tr>
											<td>5</td>
											<td>COOK, DAVID</td>
											<td>1975 MARION CT., LAFAYETTE, CA 94549</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A61K 39/04, 39/07</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2004/003671</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-02-06</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/446,051</td>
									<td>2003-02-06</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/490,089</td>
									<td>2003-07-24</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>60/541,515</td>
									<td>2004-02-02</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>4</td>
									<td>60/449,153</td>
									<td>2003-02-21</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>5</td>
									<td>60/511,869</td>
									<td>2003-10-15</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/223392-modified-free-living-microbes-vaccine-compositions-and-methods-of-use-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:00:49 GMT -->
</html>
