<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/223538-a-filamentous-fungus-useful-for-the-production-of-heterologous-polypeptides by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:31:52 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 223538:A FILAMENTOUS FUNGUS USEFUL FOR THE PRODUCTION OF HETEROLOGOUS POLYPEPTIDES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A FILAMENTOUS FUNGUS USEFUL FOR THE PRODUCTION OF HETEROLOGOUS POLYPEPTIDES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>ABSTRACT 627/MAS/97 &quot;An Aspersillus oryzae ccH useful for the production of heterologQus polypeptides&quot; The present invention relates to an Aspergillus oryzae cell usefijl for the production of heterologous polypeptides, having been modified by recombinant DNA technology in a manner by which the expression of alkaline proteases have been completely or partially Inactivated. The Invention also encompasses processes for the production of proteins of interest in high yields by using the fungi of the invention. The invention furthermore relates to methods for producing such fungi and DNA constructs to be used in these methods.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
FIELD  OF  THE   INVENTION<br>
The present invention relates to fungi, which do not produce alkaline proteases. The fungi of the invention are useful as hosts for the production of proteins susceptible to proteolytic degradation by the proteases usually produced, and the invention  consequently encompasses processes for the production of proteins of interest in high yields by using the fungi of the invention. The invention also comprises methods for producing such fungi and DNA constructs to toe used in these methods.<br>
 BACKGROUND OF THE INVENTION<br>
Fungi, and especially filamentous fungi, are widely used com¬mercially because of their ability to secrete remarkably high levels of proteins<br>
	Among the filamentous fungi species belonging to the<br>
genus Aspergillus have a long history of commercial use for the production of endogenous and lately also heterologous proteins.<br>
One disadvantage with most microorganisms used for the production of proteins is the inherent production of proteases<br>
   which may lead to degradation by proteolysis of a protein product of interest.<br>
Various ways of avoiding this have been envisaged. Among other solutions it has been suggested to delete or disrupt the genes encoding the various proteases.<br>
	WO 9000192  (Genencor Inc.,  US)  describes a mutant<br>
filamentous fungi suitable (more specifically an Aspergillus awamori) for producing heterologous polypeptides having a gene coding for an aspartic proteinase non-revertibly inactivated or elirainated.  It was shown that the production yield of the<br>
 heterologous polypeptide, bovine chymosin, was increased using said mutant host.<br>
EP 574 347 describes an Aspergillus host defective in a<br><br>
serine protease of the subtilisln type.<br>
Unfortunately fungi produce a high number of proteases. Consequently, strains of filamentous fungi exhibiting no or very low levels of protease production are still needed.<br>
SUMMARY OF THE INVENTION<br>
The object of the present invention is to provide a filamentous fungus suitable as host cell for the production of increased    yields of heterologous polypeptides.<br>
The present invention relates to such fungi, wherein at least one gene encoding alkaline protease activity has been com¬pletely or partially inactivated by recombinant DNA technology.<br>
According to the invention this may be done by modifying<br>
    said alkaline protease gene(s) of the fungus in question in such<br>
a manner that it is deleted,  inactivated or expressed with<br>
significantly reduced functionality or without functionality at<br>
all.<br>
In a specific embodiment the alkaline protease gene is  the alp  gene shown in sEQ ID No 1 and described by Murakami K et al., (1991), Agric. Biol. Chem. 55, pp. 2807-2811.<br>
Furthermore,  the  invention  relates  to methods  for producing such fungi, wherein the inactivation of the alkaline gene(s) is(are) obtained by substitution, deletion or insertion     of extra DNA in the alkaline protease gene.<br>
The invention also relates to processes for producing heterologous polypeptides in fungi of the invention. Especially contemplated are secreted polypeptides, whereby a fungal host cell modified and optionally transformed with a DNA  construct comprising at least a DNA sequence coding for the polypeptide or gene product of interest, is cultivated in a suitable growth medium at appropriate conditions and the desired gene product is recovered and purified.<br>
The inventors found that the fungi of the invention     produces such secreted polypeptides in a much improved yield.<br>
Furthermore, the invention relates to polypeptide products produced by the above methods.<br><br>
Lastly, the invention relates to DNA constructs comprising the DNA sequence of the alp gene of Aspergillus oryzae intended for use in the above mentioned methods.<br><br>
BRIEF DESCRIPTION OF THE DRAWING<br>
The invention is described in further detail in the following parts of the specification with reference to the Examples and the    drawing, wherein<br>
Fig. 1 shows the steps involved in the construction of HowBloi, Fig. 2 shows the steps involved in the construction of pJaL 212, Fig. 3 show the steps involved in the construction of A.   oryzae<br>
    Jal 125 (alp deleted), Fig. 4 shows pSX320, and<br>
Fig. 5 shows the production of 45 kD endoglucanase in a host strain of the invention (A. oryzae JaL 125) having the alp gene deleted  in  comparison  with  a  corresponding  strain  still<br>
 comprising the alp  gene.<br>
DEFINITIONS<br>
 In the present specification the following definitions are used: The expression alpD  means a strain in which the alp  gene is deleted.<br>
The expression alp~  means a strain which does not produce a  functional polypeptide i.e. Alp    protein having  lost the  functionality.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
As indicated the present invention relates in its first aspect to fungi, having been modified by recombinant DNA technology in a  manner by which the expression of one or more of the alkaline proteases have been completely or partially inactivated.<br><br>
Senetlc Hodificatlons<br>
The fungi of the invention may be modified using standard recombinant DNA technology, known to the person skilled in the art. The gene sequence responsible for the production of alkaline proteases may be inactivated or eliminated entirely.<br>
In other words a fungi of the invention exereses the alkaline protease in reduced levels, or expresses no alkaline protease at all, or expresses alkaline protease(s) with lost functionality.<br>
In a particular embodiment, the said inactivation is obtained modification in the structural or regulatory regions encoding the alkaline protease(s) in question, for instance, by interfering with the regulation of the expression signals regula¬ting the expression of the alkaline protease gene.<br>
Known and useful techniques include, but are not limited to, specific or random mutagenesis, PCR generated mutagenesis, site specific DNA deletion, insertion andor substitution, gene disruption or gene replacement techniques, anti-sense techniques, or a combination thereof.<br>
Mutagenesis may be performed using a suitable physical or chemical mutagenizing agent. Examples of a physical or chemical mutagenizing agent suitable for the present purpose includes ultraviolet (UV) irradiation, hydroxylamine, N-methyl-N'-nitro-N-nitrosoguanidine (MNNG), 0-methyl hydroxylamine, nitrous acid, ethyl methane sulphonate fEMS), sodium bisulfite, formic acid, and nucleotide analogues. When such agents are used, the mutagenesis is typically performed by incubating the cell to be mutagenized in the presence of the mutagenizing agent of choice under suitable conditions for the mutagenesis to take place, and selecting for mutated cells having a significantly reduced production of alkaline protease(s).<br>
Modification may also be accomplished by introduction, substitution or removal of one or more nucleotides in the alkaline protease encoding sequence or a regulatory element required for the transcription or translation thereof. Nucleo¬tides may, e.g., be inserted or removed so as to result in the introduction of a stop codon, the removal of the start codon  or a<br><br>
change of the open reading frame. The modification or inactivation of the structural sequence or a regulatory element may be accomplished by site-directed mutagenesis or PCR generated mutagenesis in accordance with methods known in the art. Although  in principle, the modification may be performed in vivo, i.e. directly on the cell carrying the alkaline protease gene, it is presently preferred to conduct the modification in vitro.<br>
A convenient way to inactivate or reduce the alkaline protease production of a fungi of choice is based on the<br>
 principles of gene interruption. This method involves the use of a DNA seguence corresponding to the endogenous gene or gene fragment which it is desired to destroy. Said DNA seguence is in vitro mutated to a defective gene and transformed into the fungi. By homologous recombination, the defective gene replaces the<br>
 endogenous gene or gene fragment. It may be desirable that the defective gene or gene fragment encodes a marker which may be used for selection of transformants in which gene encoding the alkaline protease has been modified or destroyed.<br>
Methods  for  deleting  or  disrupting  a  gene  are<br>
 specifically described in Miller et al. (1985), Mol. Cell. Biol, 5, pp. 1714-1721; WO 9000192 (Genencor); May G., (1992), Applied Molecular Genetics of Filamentous Fungi pp. 1-25, Eds. J.R. Kinghorn and G, Turner; Blackie Academic and Professional; and in G. Turner, (1994), Vectors for genetic manipulation, pp. 641-665,<br>
 Eds. S.D. Martinelli and J.R. Kinghorn; Elsevier.<br>
Alternatively, the modification andor inactivation of the DNA sequence may be performed by use of established anti-sense techniques using a nucleotide sequence complementary to the alkaline protease encoding sequence, e.g.   the alp  gene nucleotide<br>
  sequences presented as SEQ ID NO. 1.<br>
The anti-sense technology and how to employ it is described in detail in the aforementioned US Patent No. 5,190,931 (University of New York).<br>
Owing  to  genetic  modification,  the  fungi  of  the<br>
 invention expresses significantly reduced levels of alkaline proteases. In a preferred embodiment the level of alkaline protease expressed by the fungus is reduced more than about 50%,<br><br>
preferably more than about 8 5%, more preferred more than about<br>
9 0%, most preferred more than about 95%. In a most preferred<br>
embodiment the product expressed by the host cell is essentially<br>
free of any alkaline protease activity.<br>
$	The fungi of the  invention may have the alkaline<br>
protease gene replaced with a selective marker, such as pyrG gene, argrB gene, sC  gene, or the hph  gene.<br>
In a specific embodiment of the invention the alkaline protease gene in the a.lp  gene shown in SEQ ID NO 1.<br>
	Alp is a serine protease with highest activity in the<br>
neutral to alkaline pH. According to the nucleotide sequence the mature Alp is a subtilisin-type serine protease consisting of 282 amino acids and a prepro region of 121 amino acids in the N-terrainal vicinity. Comparison of the sequence between Alp and<br>
 other subtilisin-type serine proteases suggest that three residues in Alp (Asp41, His72; and Ser228) form the catalytic site. That Ser228 is part of the catalytic site is supported by chancing the Ser228 to an alanine residue (Tatsumi et al. , (1991), Agri. Biol. Chem. 55, pp. 3099-3loi.<br>
The Host Fungi<br>
According to the invention the fungus preferably belongs to a<br>
genus selected from the group comprising Aspergillus,    Trxcbo-<br>
derma, Humicola,   Candida,  Acremonium,  Fusarium,   and Penicillium<br>
	Among these genera species selected from the group<br>
comprising  A. oryzae,   A.   niger,   A.   awamori,   A.   phoenicis,   A.<br>
japonicus,   A.   foetidus,   A.   nidulans,   T.   reesei, T.   harzianum,   H.<br>
insulens,      H.      lanuginosa,      F.      gramineairum,      F.      solani,      P.<br>
chrysogenum,   and others are preferred.<br>
	Normally the fungus of the invention is transformed in<br>
order to make the fungus capable of producing the desired<br>
product.<br>
Methods for transforming fungi are well known in the<br>
art,  cf. e.g.     EP 0 184 438 A2  (Gist-Brocades N.V.)  and EP  application no. 87103806 (Novo Nordisk AS) and.<br>
For indigenous products this is of course not necessary,<br>
but in order to increase the production it may be an advantage to<br><br>
provide for multiple copies of the gene encoding the protein of interest to be incorporated into the fungi of the invention (i.e. the host cell).<br>
    Method for Producing the Host Fungus<br>
As indicated the invention also is meant to encompass the method for producing the fungus of the first aspect of the invention, wherein said inactivation of the alkaline protease gene is obtained by     i}    cloning of homologues of the alkaline protease gene from a fungus in question, ii)   producing a DNA construct comprising the alkaline protease gene wherein an  internal part has been  substituted, deleted, or extra DNA has been inserted,  iii)  transforming said fungus with the construct, and iv)   selecting transformants from which<br>
1)	no alkaline protease activity can be determined, or<br>
2)	a reduced level of alkaline protease activity can be<br>
determined, or<br>
       3) alkaline protease(s) having lost its function can be obtained.<br>
The term "reduced" alkaline protease activity means that the activity is less that the corresponding not mutated fungus,<br>
	The alkaline protease activity, including Alp activity,<br>
can be assay with the synthetic substrate, N-succinyl-Ala-Ala-Pro-Phe-p-nitroanilide (Sigma Chemical Co.). The activity is measured at 25=0 with X mM substrate in 20 mM Tris-HCl (pH Q.Q] in a total volume of 1 ml. The release of p-nitroanilin is<br>
 monitored at 410 nra (Ramesh et al. (1994) Infection and Immunity, Jan. 79-85).<br>
Homologs of the alkaline protease gene in question in step i) may be cloned either by cross-hybridization with one already known genes or by complementation of alkaline protease<br>
   mutants.<br>
In the case where no alkaline protease activity can be determined it may be due to deletion of essential parts of<br><br>
alkaline protease gene(s) of the fungus.<br>
If a reduced level of alkaline protease activity can be determined it may be due to that not all alkaline protease genes have  been  inactivated  or  to  the  expression  of  alkaline  protease(s) with reduced functionality.<br>
In connection with this method information provided by the cloning can be used to make DNA constructs that can be in¬tegrated into the alkaline protease gene, and even replace it with another gene, such as the pyrG gene, argB  gene, sC  gene, or    the hph  gene.<br>
If the gene is the alp  gene the isolated transformants may be alp"  or alpD.<br>
Also included is the method for producing the fungi, wherein the inactivation is obtained by using anti-sense tech- nology. Such a method comprising<br>
i)    construction of an expression plasmid which gives rise to<br>
synthesis of a RNA molecule complementary to the BIRNA<br>
transcribed from the alkaline protease gene,<br>
ii)   transformation of the host fungus with said expression<br>
	plasmid and a suitable marker, either on separate plasmids<br>
or on the same plasmid, iii)  selection of transformants using said marker, and iv)   screening transformants for strains exhibiting a reduction in the synthesis of the alkaline protease product.<br>
A further aspect of the invention is meant to comprise DNA constructs for use in the above mentioned methods.<br>
In respect of the former method said DNA constructs may comprise the alkaline protease gene,  such as the alp    gene,  wherein an internal part has been substituted, deleted, or extra DNA has been inserted.<br>
The DNA construct may furthermore also comprise DNA sequences encoding a polypeptide product of interest, such as those mentioned later.  In respect of the latter anti-sense method the DNA construct may comprise an inverted DNA sequence of the alkaline protease gene connected to a functional promoter, whereby the<br><br>
mRNA is at least partially complementary to inRNA produced from the alkaline protease gene.<br>
The Process A further aspect of the invention relates to a process for the production of heterologous polypeptides, preferably a secreted gene product, whereby a fungus according to the invention is cultivated in a suitable growth medium at appropriate conditions and the desired gene product is recovered and purified.<br>
	When producing polypeptidase, e.g.   enzymes,   the choice of<br>
fermentation pH will often depend on factors such the host cell to be used, the growth medium, the stability of the polypeptide etc. Consequently, even though the fungi of the invention may be used for any fermentation process carried out at any pH, it is<br>
 advantageous that the fermentation process is carried out at a pH when acidic andor neutral proteases produced by the host cell is essentially inactive or at least significantly reduced.<br>
In other word removal of aspartic proteases such as described in WO 9000192  (Genencor) would not have the same<br>
 advantageous effect on the production yield.<br>
If the fermentation process is carried out at e.g. pH 5 to 11, such as 6 to 10.5, 7 to 10, or S to 9.5 acidic proteases (such as aspartic and serine proteases) and for pHs above 7 also neutral proteases will not have their optimal activity and may<br>
 even be inactive. At such pHs alkaline proteases in unmodified host cell may be active (at least to some degree) and may be the main reason for degradation of the polypeptide product of interest. Consequently, in such cases the inactivation of gene(s) encoding alkaline protease(s)  is especially advantageous  as<br>
isr' acidic proteases, and at higher pHs also neutral proteases, do not exhibit maximal activity.<br>
Inactivation of alkaline protease gene(s) of the invention is also especially advantageous for certain host, as the acidic, neutral and alkaline protease activity varies in different fungi.<br>
 For instance are the alkaline protease activity in Aspergillus oryzae  higher than in Aspergillus niger.<br>
Examples of enzymes being fermented at the above pHs<br><br>
include Endoglucanases, Phytase and Protein Disulfide Isomerases.<br>
The heterologous polypeptide<br>
In the context of the present invention the desired polypeptide  is a "heterologous polypeptide" which means a polypeptide which is not native to the fungus of the invention, or a native polypeptide in which modifications have been made to alter the native sequence (i.e. variants), or a native polypeptide whose expression is quantitatively altered as a result of a manipula-<br>
 tion of a native regulatory sequence required for the expression of the native polypeptide, such as a promoter, a ribosome binding site, etc., or other manipulation of the fungi of the invention by recombinant DNA techniques.<br>
The heterologous polypeptide expressed by the fungus of<br>
 the invention may also be a precursor protein, i.e. a zymogen, a hybrid protein, a protein obtained &amp;s a pro sequence or pre-pro sequence, or in ummaturated form.<br>
In another preferred embodiment the product is an enzyme, in  particular  a  glycosidase  enzyme, e.g.      an  amylase,  in<br>
 particular an a-amylase or a b-amylase, a cellulase, in particular an endD-l,4-b-glucanase (EC 3.2.1.4) or an endo-i,3-b-glucanase (3.2.1.6), a xylanase, in particular an endo-l,4-b-glucanase (EC 3.3.1.8) or a xylan-endo-l,3-b-xylosidase (EC 3.2.1,32), an   a-galactosidase (EC 3.2.1.22},   a polygalacturonase<br>
 (EC 3.2.1.15, also known as pectinases), a cellulose-1,4-b-cellobiosidase (EC 3.2.1.91, also Xnown as cellobiohydrolases), an endoglucanase, in particular an endo-l,6-_-glucanase (EC 3.2.1.75), an endo-l,2-b-glucanase (EC 3.2.1.71), an endo-l,3-b-glucanase (EC 3.2.1.39) or an  endo-l,3-a-glucanase (EC 3.2.1.59).<br>
 In a preferred embodiment, the product is a eucaryotic polypeptide, such as insulin, growth hormone, glucagon, somatostatin, interferon, PDGF, factor VII, factor VIII, urokinase, EPO, chyroosin, tissue plasminogen activator, serum albumin, or TPO (thrombopoetin),<br>
	In another preferred embodiment, the product is a protein<br>
of fungal origin. In a preferred embodiment, the product is a fungal enzyme, e.g.   an  amylolytic enzyme, such as an a-amylase, a<br><br>
b-amylase, a glucoamylase, a b-galactosidase, a phytase j ', a cellulytic enzyme, a lipolytic enzyme, a xylanolytic enzyme, a proteolytic enzyme, an oxidoreductase, such as a peroxidase or a laccase, a pectinase, a cutinase, or a protein disulfide  isomerase.<br>
In yet another preferred embodiment, the product is a bacterial protein. In a preferred embodiment, the product is a bacterial enzyme, in particular an amylolytic enzyme, e.g. an amylolytic enzyme, such as an a-amylase, a b-amylase, a  glucoamylase, a b-galactosidase, a cellulytic enzyme, a lipolytic enzyme, or a proteolytic enzyme.<br>
Preferred hybrid polypeptides are prochymosin and pro-trypsin-like proteases.<br>
The invention is explained in further detail  in the    Examples given below. These should, however, not in any way be construed as limiting the scope of the invention as defined in the appended claims.<br>
   EXAMPLES<br>
Materials and Methods<br>
strains<br><br><br>
Aspergillus  oryzae  IFO 4177;<br><br>
JaL125:<br>
Genes<br>
 alp:<br><br>
available from Institute for Fermention, Osaka; 17-25 Juso Hammachi 2-Chome Yodogawa_ku, Osaka, Japan-<br>
The construction of this strain is described in the Examples.<br>
This gene codes for alkaline protease shown in SEQ ID NO 1,<br><br>
pyrG:<br><br>
This gene codes for orotidine-5'-phosphate decaroxylase, an enEyme involved in the biosyn¬thesis of uridine.<br><br><br><br><br><br><br><br>
Flasmid<br>
pS02:   Prepared according to Example 1,<br>
pS05:   Prepared from pS02 according to Example i.<br>
pSRe403: Prepared according to Example 1.<br>
pSX320:  The construction of this plasmid is described in EP<br>
application no. 0 531 372B1. pToC90:  A subclone of p3SR2, harboring the amdS gene from<br>
Aspergillus nidulans  as a 2.7 kb Xbal fragment [Corrick<br>
et al., GENE 1987 53 63-71], on a pUC19 vector<br>
[Yannisch-Perron et al., GENE 1985 33 103-119],<br>
prepared as described in WO 9117243.<br>
Bluescript SK-,  M-A. Alting-Mees and Short,J,M.;<br>
Nucleic Acids Res. 1989 17 9494-9494.<br>
pUClS:  Viera and Messing J. Meth, Enzymol. 1987 153 3-11. pUCliS;  Viera and Messing J. Meth. Enzymol. 1987 153 3-11.<br><br><br><br><br><br>
EXAMPLE 1<br>
Genomic Deletion of Aspergillus oryzae AlXaline Protease The alp gene was deleted by a one step gene replacement method  (G. May in "Applied Molecular Genetics of Filamentous Fungi" (1992) pp. 1-25. Eds. J. R. Kinghorn and G. Turner; Blackie Academic and Professional) . As marker was used the A. oryza.e pyrG gene,   the A.   orysae  strain used was a pyrG~  minus strain.<br>
 Cloning of the Aspercrillus oryzae pyrG gene<br>
The A. oryzae pyrG gene was cloned by cross-hybridization with the A- niger  pyrG  gene [W, van Hartingsveldt et al., Mol. Gen.<br><br>
Genet 1987 206 71-75]. A lambda library of partial SauIIIA digested A.oryzae IFO 4177 DNA was probed at low stringency with a 1 kb DNA fragment from the A. niger pyrG gene. DNA from one positive clone was subcloned into a pUCHS vector- The resultant  plasmid, pS02, was shown to contain the pyrG gene by complementation of an A.   niger pyrG  mutant.<br>
Construction of an Aspergillus oryzae  pyrG minus strain<br>
A pyrG   deletion plasmid, pS05, containing about i kb of pyrG<br>
 flanking sequences on each end was constructed from the plasmid pS02. A. oryzae IFO 4177 was transformed with this construct and transformants were selected by resistance to 5-fluoro-orotic acid, a phenotype characteristic of pyrG mutants. One transformant, HowBioi, were shown by Southern analysis to have<br>
 the expected deletion at the pyrG locus. Being a pyrG mutant HowBloi requieres uridine for growth. HowBlOl can be transformed with the wt pyrG gene by selection for ability to grow without uridine.<br>
The steps involved in the construction of HowBiOl is<br>
   illustrated in Fig. i.<br>
Construction of an Aspergillus oryzae alp minus strain An aspergillus oryzae alp minus strain was  constructed by transforming A.oryzae    strain HowBioi with the alp    deletion    plasmid pJaL212.<br>
Construction of the alkaline protease disruption plasmid pJaL212 An A.oryzae  IPO 4177 genomic library was established by partially digesting genomic DNA with Sau3A. Fragments were ligated into<br>
 pUC19 digested with BamHi. The library was screened using degenerate oligo nucleotides matching known protein sequence fragments of the alkaline protease. A plasmid containing a 3.9 kb Sau3A fragment was isolated in this way. This plasmid was named pSRe403.<br>
 pSRe403 was digested with SacII, and the 5.6 kb fragment was isolated and religated to form pSRe 403_SacII. This plasmid was partially digested with Hind III, ends was filled in with<br><br>
Klenow polymerase and religated in order to remove the Hind III site in pUC 19. The resulting plasmid was termed pJaL173. pJaL173 was digested with Hindiri, and the 3.8 kb Hindiri fragment, containing the pyrG   gene, of pS02 was inserted into    pJaL173 to form pJaL212. The construction is outline in Fig. 2.<br>
Isolation of an A.   oryzae alp  minus strain<br>
A.    oryzae   HowBlOl was transformed with the 6.8 kb Saci-SphI<br>
fragment of pJaL212 using standard procedures. This fragment<br>
5 consists of 1.5 kb of the alp promoter, the pyrG gene and 1.5 kb of the 3' end of the alp gene. Transf ormants were selected by their ability to grow without the addition of uridine. After reisolation chromosomal DNA was prepared from 12 transformants, the DNA was cut with PstI and analysed by Southern analysis with<br>
 a 599 bp PstlSacII fragment from pJaL173 containing part of the alp gene as a radioactive labelled probe, strains which carry a deletion of the alp gene are easily recognized by that the wild type PstI band on 1.0 kb is shifted to a 1.9 kb band in the transformant.<br>
	In four transf ormants was the 1.0 kb PstI band shifted to<br>
the 1.9 kb band. One of these transformants was named JaL125.<br>
The steps involved in the construction of JaLl25  is illustrated in Fig. 3.<br>
X<br>
EXAMPLE 2<br>
Production of 45 kP endoqlucanase in A. oryzae strain JaLl25<br>
A. oryzae strains IFO 4177 and JaL125 was transformed with the &gt;tf plasmid pSX320 (Fig. 4), which is a fungal expression plasmid for the 45 kD endoglucanase from Humicola insolens, by co-transformation with pToC90. Construction of the plasmid pSX320 is described in EP patent application no. 0 531 372.<br>
Transformants was selected for growth on minimal medium i
One transformant from each strains was selected and grown<br><br>
at 30'C in shake flasks containing 100 ml Maltodextrin, Soybeanmeal, and Peptone for 5 days. Samples of the fermentation broth were taken every day and applied to SDS-Page and Coomassie Blue stained. The expression yield of the 45kD i endoglucanase was quantified from the Coomassie Blue stained SDS-page by scanning and quatitation of the bands with the gel scanning system Biolmage from Biolmage System UK. The results are shown in Fig. 5. From these results it can been seen that the 45 kD endoglucanase is mush more stable (slower  degradation) in the strains deleted in the alp gene. Where in the wt. strain containing the alp gene is the 45 kD endoglucanase degrated very fast after 48 hours-<br><br><br><br><br><br><br><br><br><br>
WE CLAIM:<br>
1.	An Aspergillus oryzae cell useful for the production of heterologous polypepti¬des, having been modified by recombinant DNA technology in a manner by which an endogenous alkaline protease activity, encoded in the alp gene with the sequence shown in SEQ ID NO 1 or a DNA sequence having at least 80% identity thereof, has been completely or partially inactivated.<br>
2.	The cell according to claim 1, wherein said inactivatlon is obtained by nucleotide deletion, insertion or substitution in a sequence encoding an alkaline protease gene product.<br>
3.	The cell according to claim 1. wherein said inaclivalion is obtained by interfer¬ing with the regulation of the expression signals regulating the expression of the alkaline protease gene.<br>
4.	fhe cell according to claim 1. wherein said inactivatlon is obtained by using anti-sense technology.<br>
5.	A recombinant DNA comprising a modification of the DNA sequence encoding the alp gene of A. oryzae shown in SEQ ID NO 1. or a DNA sequence having at least 80% identity thereof, wherein a part of the gene, such as an internal part, has been substituted, deleted or extra DNA has been inserted within the gene.<br>
6.	A method for producing an Aspergillus oryzae cell according to any of claims 1 to 4. wherein the inactivatlon of the alkaline protease gene is obtained by:<br><br>
i)	cloning of the alkaline protease gene  in question from a fungus of<br>
interest,<br>
ii) producing a DNA construct comprising the alkaline protease gene, wherein a part of the gene, such as an internal part, has been substituted, deleted, or extra DNA has been inserted within the gene,<br>
iii)      transforming said fungus with the DNA constructs, and<br>
iv)       isolating transformants from which:<br>
1)	no alkaline protease activity can be determined;<br>
2)	a reduced level of alkaline protease activity can be determined; or<br>
3)	alkaline protease(s) having lost its function can be obtained,<br>
7.	A method for producing a cell according to any of claims 1 to 4, wherein the<br>
inactivation of the alkaline protease gene is obtained by using anti-sense<br>
technology, which method comprises:<br>
i)        construction of an expression plasmid resuUing in the synthesis of an<br>
RNA  molecule  complementary to the  mRNA  transcribed  from the<br>
alkaline protease gene; ii)       transformation of the host fungal cell with said expression plasmid and a<br>
suitable marker: iii)      selection of transformants using said marker; and iv)       screening  selected  transformants  a reduction in the   synthesis  of the<br>
alkaline protease product in question.<br>
8,	A method for producing a cell according to any of claims 1 to 4, wherein said<br>
fungus is modified to produce lower levels of alkaline protease than the<br>
wild-type by a process comprising transforming a parent cei! of said fungus with<br>
a DNA construct capable of causing reduced production of a functional alkaline<br>
protease when integrated into the genome of said fungus.<br><br>
9.	The method according to claims 6 to 8, wherein said alkaline protease gene is an alp gene comprising the sequence shown in SKQ [D NO \ or a DNA sequence having at least 80% identity thereof.<br>
10.	A method for producing heterologous polypeptides comprising cultivating a fungal host cell according to any of claims 1 to 4 that has been transformed with a DNA sequence encoding a desired gene producl.<br>
11.	The method according to claim 10. wherein said polypeptide is secreted into the extracellular medium by said fungal host eel).<br>
12.	The method according to claim II. wherein said polypeptide is recovered and purified from said culture medium.<br>
13.	fhe method according to any of claims 10 to 12, wherein the fermentation is carried out at pH 5 to 11, preferably 6 to 10.5, such as 7 to iO, especially 8 to 9,5.<br>
[4. The method according to any of claims 10 to 13, wherein said desired gene product is an enzyme selected from the group comprising a glycosidase enzyme. e.g., an amylase, in particular, an a-amylase or a p-amylase; a cellulase, in particular, an endo-l,4-p-glucanase or an endo-1.3-p-glucanase; a xylanase. in particular, an endo-l.4-p-xylanasc or a xylan-endo-l,3-p-xylosidase; an a-galactosidase; a polygalacturonase; a cellulose 1.4-p-cellobiosidase; and an endoglucanasc, in particular, an endo-),6-p-glucanase, an endo-),2'(i-glucanase, an endo-1,3-[l-glueanase or an endo-1,3-a-glucanase.<br><br>
13. the method according to any of claims 10 to 13, wherein said desired gene product is a eukaryotic polypeptide selected from the group comprising insulin, growth hormone, glucagon, somatostatin, interferon, PDGF, factor VII, factor VIII, urokinase, tPA, EPO, and TPO.<br>
16.	The method according to any of claims 10 to 13, wherein said heterologous<br>
polypeptide is a protein of fungal origin.<br>
17.	The method according to claim 16. wherein the product is a fungal enyme<br>
selected from the group comprising an amylolytic enzyme, such as an a-<br>
amylase, a p-amylase. a glucoamylase, a p-gaiactosidasc, a phytase, a ccllulytic<br>
enzyme, a lipolytic enzyme, a xylanolytic enzyme, a proteolytic enzyme, an<br>
oxidoreductase, such as a peroxidase or a laccase, a pectinase, a cutinase, and a<br>
protein disulfide isomerase.<br>
18.	Ihc method according lo any of claims 10 to 13. wherein said heterologous<br>
polypeptide of bacterial origin.<br>
19.	The method according to claim 18, wherein the bacterial protein is an enzme<br>
selected from the group comprising an amylolytic enzyme, a glucoamylase. a p-<br>
galaclosidase, a cellulytic cnzy'me, a lipolytic enzynie. and a proteolytic enzyme.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyNy1tYXMtMTk5NyBhYnN0cmFjdC1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">0627-mas-1997 abstract-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyNy1tYXMtMTk5NyBhYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">0627-mas-1997 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyNy1tYXMtMTk5NyBhc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">0627-mas-1997 assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyNy1tYXMtMTk5NyBjbGFpbXMtZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">0627-mas-1997 claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyNy1tYXMtMTk5NyBjbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">0627-mas-1997 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyNy1tYXMtMTk5NyBjb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">0627-mas-1997 correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyNy1tYXMtMTk5NyBjb3JyZXNwb25kZW5jZS1wby5wZGY=" target="_blank" style="word-wrap:break-word;">0627-mas-1997 correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyNy1tYXMtMTk5NyBkZXNjcmlwdGlvbiAoY29tcGxldGUpLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">0627-mas-1997 description (complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyNy1tYXMtMTk5NyBkZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">0627-mas-1997 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyNy1tYXMtMTk5NyBkcmF3aW5ncy1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">0627-mas-1997 drawings-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyNy1tYXMtMTk5NyBkcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">0627-mas-1997 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyNy1tYXMtMTk5NyBmb3JtLTE5LnBkZg==" target="_blank" style="word-wrap:break-word;">0627-mas-1997 form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyNy1tYXMtMTk5NyBmb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">0627-mas-1997 form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyNy1tYXMtMTk5NyBmb3JtLTI2LnBkZg==" target="_blank" style="word-wrap:break-word;">0627-mas-1997 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyNy1tYXMtMTk5NyBmb3JtLTQucGRm" target="_blank" style="word-wrap:break-word;">0627-mas-1997 form-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyNy1tYXMtMTk5NyBmb3JtLTYucGRm" target="_blank" style="word-wrap:break-word;">0627-mas-1997 form-6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyNy1tYXMtMTk5NyBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">0627-mas-1997 others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyNy1tYXMtMTk5NyBwZXRpdGlvbi5wZGY=" target="_blank" style="word-wrap:break-word;">0627-mas-1997 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="223537-glyphosate-formulations-containing-etheramine-surfactants.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="223539-formulation-comprising-electrolyzed-water.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>223538</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>627/MAS/1997</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>47/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Nov-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>12-Sep-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>25-Mar-1997</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVOZYMES A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>KROGSHOJVEJ 36, DK-2880 BAFSVAERD,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>JAN LEHMBECK</td>
											<td>C/O. NOVO NORDISK A/S, NOVO ALLE, DK-2880 BAGSVAERD,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N1/15</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0354/96</td>
									<td>1996-03-27</td>
								    <td>Denmark</td>
								</tr>
								<tr>
									<td>2</td>
									<td>0555/96</td>
									<td>1996-05-09</td>
								    <td>Denmark</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/223538-a-filamentous-fungus-useful-for-the-production-of-heterologous-polypeptides by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:31:53 GMT -->
</html>
