<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/223554-an-invitro-method-of-determining-the-status-of-sepsis-using-biomarket-profiles by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:28:41 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 223554:AN INVITRO METHOD OF DETERMINING THE STATUS OF SEPSIS USING BIOMARKET PROFILES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">AN INVITRO METHOD OF DETERMINING THE STATUS OF SEPSIS USING BIOMARKET PROFILES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a kit for in vitro diagnosis of sepsis in an individual comprising two to twenty antibodies are present in the kit, wherein at least two biomarkers are identified.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
DIAGNOSIS OF SEPSIS OR SIRS USING BIOMARKER PROFILES<br>
[0001]       The present application claims priority to United States Provisional Patent Application Serial No. 60/425,322, filed November 12, 2002, and to United States Provisional Patent Application Serial No. 60/511,644, filed October 17, 2003, both of which are herein incorporated by reference in their entirety.<br>
FIELD OF THE INVENTION<br>
[0002]       The present invention relates to methods of diagnosing or predicting sepsis or its stages of progression in an individual. The present invention also relates to methods of diagnosing systemic inflammatory response syndrome in an individual.<br>
BACKGROUND OF THE INVENTION<br>
[0003]	Early detection of a disease condition typically allows for a more effective<br>
therapeutic treatment with a correspondingly more favorable clinical outcome. In many cases, however, early detection of disease symptoms is problematic; hence, a disease may become relatively advanced before diagnosis is possible. Systemic inflammatory conditions represent one such class of diseases. These conditions, particularly sepsis, typically result from an interaction between a pathogenic microorganism and the host's defense system that triggers an excessive and dysregulated inflammatory response in the host. The complexity of the host's response during the systemic inflammatory response has complicated efforts towards understanding disease pathogenesis. (Reviewed in Healy, Annul. Pharmacother. 36: 648-54 (2002).) An incomplete understanding of the disease pathogenesis, in turn, contributes to the difficulty in finding diagnostic biomarkers. Early and reliable diagnosis is imperative, however, because of the remarkably rapid progression of sepsis into a life-threatening condition.<br>
[0004]       Sepsis follows a well-described time course, progressing from systemic inflammatory response syndrome ("SIRS")-negative to SIRS-positive to sepsis, which may then progress to severe sepsis, septic shock, multiple organ dysfunction ("MOD"), and ultimately death. Sepsis also may arise in an infected individual when the individual subsequently develops SIRS. "SIRS" is commonly defined as the presence of two or more of the following parameters: body temperature greater than 38°C or less than 36°C; heart rate greater than 90 beats per minute; respiratory rate greater than 20 breaths per minute; Pco2 less than 32 mm Hg; and a white blood cell count either less than 4.0x109 cells/L or greater than<br><br>
12.0xl09 cells/L, or having greater than 10% immature band forms. "Sepsis" is commonly defined as SIRS with a confirmed infectious process. "Severe sepsis" is associated with MOD, hypotension, disseminated intravascular coagulation ("DIG") or hypoperfusion abnormalities, including lactic acidosis, oliguria, and changes in mental status. "Septic shock" is commonly defined as sepsis-induced hypotension that is resistant to fluid resuscitation with the additional presence of hypoperfusion abnormalities.<br>
[0005]       Documenting the presence of the pathogenic microorganisms clinically significant to sepsis has proven difficult. Causative microorganisms typically are detected by culturing a patient's blood, sputum, urine, wound secretion, in-dwelling line catheter surfaces, etc. Causative microorganisms, however, may reside only in certain body microenvironments such that the particular material that is cultured may not contain the contaminating microorganisms. Detection may be complicated further by low numbers of microorganisms at the site of infection. Low numbers of pathogens in blood present a particular problem for diagnosing sepsis by culturing blood. In one study, for example, positive culture results were obtained in only 17% of patients presenting clinical manifestations of sepsis. (Rangel-Frausto et aL, JAMA 273: 117-23 (1995).) Diagnosis can be further complicated by contamination of samples by non-pathogenic microorganisms. For example, only 12.4% of detected microorganisms were clinically significant in a study of 707 patients with septicemia. (Weinstein et al.9 Clinical Infectious Diseases 24: 584-602 (1997).)<br>
[0006]       The difficulty in early diagnosis of sepsis is reflected by the high morbidity and mortality associated with the disease. Sepsis currently is the tenth leading cause of death in the United States and is especially prevalent among hospitalized patients in non-coronary intensive care units (ICUs), where it is the most common cause of death. The overall rate of mortality is as high as 35%, with an estimated 750,000 cases per year occurring in the United States alone. The annual cost to treat sepsis in the United States alone is in the order of billions of dollars.<br>
[0007]       A need, therefore, exists for a method of diagnosing sepsis sufficiently early to allow effective intervention and prevention. Most existing sepsis scoring systems or predictive models predict only the risk of late-stage complications, including death, in patients who already are considered septic. Such systems and models, however, do not predict the development of sepsis itself. What is particularly needed is a way to categorize<br><br>
those patients with SIRS who will or will not develop sepsis. Currently, researchers will typically define a single biomarker that is expressed at a different level in a group of septic patients versus a normal (i.e., non-septic) control group of patients. United States Patent Application Serial No. 10/400,275, filed March 26,2003, the entire contents of which are hereby incorporated by reference, discloses a method of indicating early sepsis by analyzing time-dependent changes in the expression level of various biomarkers. Accordingly, optimal methods of diagnosing early sepsis currently require both measuring a plurality of biomarkers and monitoring the expression of these biomarkers over a period of time.<br>
[0008]       There is a continuing urgent need in the art to diagnose sepsis with specificity and sensitivity, without the need for monitoring a patient over time. Ideally, diagnosis would be made by a technique that accurately, rapidly, and simultaneously measures a plurality of biomarkers at a single point in time, thereby minimizing disease progression during the time required for diagnosis.<br>
SUMMARY OF THE INVENTION [0009]       The present invention allows for accurate, rapid, and sensitive prediction and diagnosis of sepsis through a measurement of more than one biomarker taken from a biological sample at a single point in time. This is accomplished by obtaining a biomarker profile at a single point in time from an individual, particularly an individual at risk of developing sepsis, having sepsis, or suspected of having sepsis, and comparing the biomarker profile from the individual to a reference biomarker profile. The reference biomarker profile may be obtained from a population of individuals (a "reference population") who are, for example, afflicted with sepsis or who are suffering from either the onset of sepsis or a particular stage in the progression of sepsis. If the biomarker profile from the individual contains appropriately characteristic features of the biomarker profile from the reference population, then the individual is diagnosed as having a more likely chance of becoming septic, as being afflicted with sepsis or as being at the particular stage in the progression of sepsis as the reference population. The reference biomarker profile may also be obtained from various populations of individuals including those who are suffering from SIRS or those who are suffering from an infection but who are not suffering from SIRS. Accordingly, the -   -present invention allows the clinician to determine, inter alia, those patients who do not have SIRS, who have SIRS but are not likely to develop sepsis within the time frame of the investigation, who have sepsis, or who are at risk of eventually becoming septic.<br><br>
[0010]       Although the methods of the present invention are particularly useful for detecting or predicting the onset of sepsis in SIRS patients, one of ordinary skill in the art will understand that the present methods may be used for any patient including, but not limited to, patients suspected of having SIRS or of being at any stage of sepsis. For example, a biological sample could be taken from a patient, and a profile of biomarkers in the sample could be compared to several different reference biomarker profiles, each profile derived from individuals such as, for example, those having SIRS or being at a particular stage of sepsis. Classification of the patient's biomarker profile as corresponding to the profile derived from a particular reference population is predictive that the patient falls within the reference population. Based on the diagnosis resulting from the methods of the present invention, an appropriate treatment regimen could then be initiated.<br>
[0011]       Existing methods for the diagnosis or prediction of SIRS, sepsis or a stage in the progression of sepsis are based on clinical signs and symptoms that are nonspecific; therefore, the resulting diagnosis often has limited clinical utility. Because the methods of the present invention accurately detect various stages of sepsis, they can be used to identify those individuals who might appropriately be enrolled in a therapeutic study. Because sepsis may be predicted or diagnosed from a "snapshot" of biomarker expression in a biological sample obtained at a single point in time, this therapeutic study may be initiated before the onset of serious clinical symptoms. Because the biological sample is assayed for its biomarker profile, identification of the particular biomarkers is unnecessary. Nevertheless, the present invention provides methods to identify specific biomarkers of the profiles that are characteristic of sepsis or of a particular stage in the progression of sepsis. Such biomarkers themselves will be useful tools in predicting or diagnosing sepsis.<br>
[0012]       Accordingly, the present invention provides, inter alia, methods of predicting the onset of sepsis in an individual. The methods comprise obtaining a biomarker profile at a single point in time from the individual and comparing the individual's biomarker profile to a reference biomarker profile. Comparison of the biomarker profiles can predict the onset of sepsis in the individual with an accuracy of at least about 60%. This method may be repeated again at any time prior to the onset of sepsis.<br>
[0013]       The present invention also provides a method of diagnosing sepsis in an individual having or suspected of having sepsis comprising obtaining a biomarker profile at a single<br><br>
point in time from the individual and comparing the individual's biomarker profile to a reference biomarker profile. Comparison of the biomarker profiles can diagnose sepsis in the individual with an accuracy of at least about 60%. This method may be repeated on the individual at any time.<br>
[0014]       The present invention further provides a method of determining the progression (i.e., the stage) of sepsis in an individual having or suspected of having sepsis. This method comprises obtaining a biomarker profile at a single point in time from the individual and comparing the individual's biomarker profile to a reference biomarker profile. Comparison of the biomarker profiles can determine the progression of sepsis in the individual with an accuracy of at least about 60%. This method may also be repeated on the individual at any time.<br>
[0015]       Additionally, the present invention provides a method of diagnosing SIRS in an individual having or suspected of having SIRS. This method comprises obtaining a biomarker profile at a single point in time from the individual and comparing the individual's biomarker profile to a reference biomarker profile. Comparison of the biomarker profiles can diagnose SIRS in the individual with an accuracy of at least about 60%. This method may also be repeated on the individual at any time.<br>
[0016]       In another embodiment, the invention provides, inter alia, a method of determining the status of sepsis or diagnosing SIRS in an individual comprising applying a decision rule. The decision rule comprises comparing (i) a biomarker profile generated from a biological sample taken from the individual at a single point in time with (ii) a biomarker profile generated from a reference population. Application of the decision rule determines the status of sepsis or diagnoses SIRS in the individual. The method may be repeated on the individual at one or more separate, single points in time.<br>
[0017]       The present invention further provides, inter alia, a method of determining the status of sepsis or diagnosing SIRS in an individual comprising obtaining a biomarker profile from a biological sample taken from the individual and comparing the individual's biomarker profile to* a reference biomarker profile. A single such comparison is capable of classifying the individual as having membership in the reference population. Comparison of the biomarker profile determines the status of sepsis or diagnoses SIRS in the individual.<br><br>
[0018]       The invention further provides, inter alia, a method of determining the status of sepsis or diagnosing SIRS in an individual comprising obtaining a biomarker profile from a biological sample taken from the individual and comparing the individual's biomarker profile to a reference biomarker profile obtained from biological samples from a reference population. The reference population may be selected from the group consisting of a normal reference population, a SIRS-positive reference population, an infected/SIRS-negative reference population, a sepsis-positive reference population, a reference population at a particular stage in the progression of sepsis, a SIRS-positive reference population that will be confirmed as having sepsis by conventional techniques after about 0-36 hours, a SIRS-positive reference population that will be confirmed as having sepsis by conventional techniques after about 36-60 hours, and a SIRS-positive reference population that will be confirmed as having sepsis by conventional techniques after about 60-84 hours. A single such comparison is capable of classifying the individual as having membership in the reference population, and the comparison determines the status of sepsis or diagnoses SERS in the individual.<br>
[0019]       In yet another embodiment, the present invention provides, inter alia, a method of determining the status of sepsis or diagnosing SIRS in an individual. The method comprises comparing a measurable characteristic of at least one biomarker between a biomarker profile obtained from a biological sample from the individual and a biomarker profile obtained from biological samples from a reference population. Based on this comparison, the individual is classified as belonging to or not belonging to the reference population. The comparison, therefore, determines the status of sepsis or diagnoses SIRS in the individual. The biomarkers, in one embodiment, are selected from the group of biomarkers shown in any one of TABLES 15-23 and 26-50.<br>
[0020]       In a further embodiment, the present invention provides, inter alia, a method of determining the status of sepsis or diagnosing SIRS in an individual comprising selecting at least two features from a set of biomarkers in a profile generated from a biological sample of an individual. These features are compared to a set of the same biomarkers in a profile generated from biological samples from a reference population. A single such comparison is capable of classifying the individual as having membership in the reference population with an accuracy of at least about 60%, and the comparison determines the status of sepsis or diagnoses SIRS in the individual.<br><br>
[0021]       The present invention also provides, inter alia, a method of determining the status of sepsis or diagnosing SIRS in an individual comprising determining the changes in the abundance of at least two biomarkers contained in a biological sample of an individual and comparing the abundance of these biomarkers in the individual's sample to the abundance of these biomarkers in biological samples from a reference population. The comparison is capable of classifying the individual as having membership in the reference population, and the comparison determines the status of sepsis or diagnoses SIRS in the individual.<br>
[0022]       In another embodiment, the invention provides, inter alia, a method of determining the status of sepsis in an individual, comprising determining changes in the abundance of at least one, two, three, four, five, 10 or 20 biomarkers as compared to changes in the abundance of the at least one, two, three, four, five, 10 or 20 biomarkers for biological samples from a reference population that contracted sepsis and one that did not. The biomarkers are selected from the group consisting of the biomarkers listed in any one of TABLES 15-23 and 26 - 50. Alternatively, the abundance of the at least one, two, three, four, five, 10 or 20 biomarkers may be compared to the abundance of the at least one, two, three, four, five, 10 or 20 biomarkers.<br>
[0023]       The present invention further provides, inter alia, a method of isolating a biomarker, the presence of which in a biological sample is diagnostic or predictive of sepsis. This method comprises obtaining a reference biomarker profile from a population of individuals and identifying a feature of the reference biomarker profile that is predictive or diagnostic of sepsis or one of the stages in the progression of sepsis. This method further comprises identifying a biomarker that corresponds with the feature and then isolating the biomarker.<br>
[0024]       In another embodiment, the present invention provides a kit comprising at least one, two, three, four, five, 10 or all of the biomarkers selected from the group consisting of the biomarkers listed in any one of TABLES 15-23 and 26 - 50.<br>
[0025]       In another embodiment, the reference biomarker profile may comprise a combination of at least two features, preferably five/10, or 20 or more, where the features are characteristics of biomarkers in the sample. In this embodiment, the features will contribute to the prediction of the inclusion of an individual in a particular reference population. The relative contribution of the features in predicting inclusion may be determined by a data<br><br>
analysis algorithm that predicts class inclusion with an accuracy of at least about 60%, at least about 70%, at least about 80%, at least about 90%, about 95%, about 96%, about 97%, about 98%, about 99% or about 100%. In one embodiment, the combination of features allows the prediction of the onset of sepsis about 24, about 48, or about 72 hours prior to the actual onset of sepsis, as determined using conventional techniques.<br>
[0026]       In yet another embodiment the reference biomarker profile may comprise at least two features, at least one of which is characteristic of the corresponding biomarker and where the feature will allow the prediction of inclusion of an individual in a sepsis-positive or SIRS-positive population. In this embodiment, the feature is assigned a p-value, which is obtained from a nonparametric test, such as a Wilcoxon Signed Rank Test, that is directly related to the degree of certainty with which the feature can classify an individual as belonging to a sepsis-positive or SIRS-positive population. In another embodiment, the feature classifies an individual as belonging to a sepsis-positive or SIRS-positive population with an accuracy of at least about 60%, about 70%, about 80%, or about 90%. In still another embodiment, the feature allows the prediction of the onset of sepsis about 24, about 48, or about 72 hours prior to the actual onset of sepsis, as determined using conventional techniques.<br>
[0027]      "In yet another embodiment the present invention provides an array of particles, with capture molecules attached to the surface of the particles that can bind specifically to at least one, two, three, four, five, 10 or all of the biomarkers selected from the group consisting of the biomarkers listed in any one of TABLES 15 - 23 and 26 - 50.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
[0028]       FIGURE 1 illustrates the progression of SIRS to sepsis. The condition of sepsis consists of at least three stages, with a septic patient progressing from severe sepsis to septic shock to multiple organ dysfunction.<br>
[0029]       FIGURE 2 shows the relationship between sepsis and SIRS. The various sets shown in the Venn diagram correspond to populations of individuals having the indicated condition.<br>
[0030] FIGURE 3 shows the natural log of the ratio in average normalized peak intensities for about 400 ions for a sepsis-positive population versus a SIRS-positive population.<br><br>
[0031]       FIGURE 4 shows the intensity of an ion having an m/z of 437.2 Da and a retention time on a C18 reverse phase column of 1.42 min in an ESI-mass spectrometer profile. FIGURE 4A shows changes in the presence in the ion in various populations of individuals who developed sepsis. Clinical suspicion of sepsis in the sepsis group occurred at "time 0," as measured by conventional techniques. "Time -24 hours" and "time -48 hours" represent samples taken about 24 hours and about 48 hours, respectively, preceding the clinical suspicion of the onset of sepsis in the sepsis group.   Individuals entered the study at "Day 1." FIGURE 4B shows the presence of the same ion in samples taken from populations of individuals who did not develop sepsis at time 0.<br>
[0032]       FIGURE 5 is a classification tree fitted to data from time 0 in 10 sepsis patients and 10 SIRS patients, showing three biomarkers identified by electrospray mass spectrometry that are involved in distinguishing sepsis from SIRS.<br>
[0033]       FIGURE 6 shows representative LC/MS and LC/MS/MS spectra obtained on plasma samples, using the configuration described in the examples.<br>
[0034]       FIGURES 7A and 7B show proteins that are regulated at higher levels in plasma up to 48 hours before conversion to sepsis.<br>
[0035]       FIGURES 8A and 8B show proteins that are regulated at lower levels in plasma up to 48 hours before conversion to sepsis.<br>
DETAILED DESCRIPTION OF THE PREFERRED EMBODIMENTS<br>
[0036]       The present invention allows for the rapid, sensitive, and accurate diagnosis or prediction of sepsis using one or more biological samples obtained from an individual at a single time point ("snapshot") or during the course of disease progression. Advantageously, sepsis may be diagnosed or predicted prior to the onset of clinical symptoms, thereby allowing for more effective therapeutic intervention.<br>
[0037]       "Systemic inflammatory response syndrome," or "SIRS," refers to a clinical response to a variety of severe clinical insults, as manifested by two or more of the following conditions within a 24-hour period:<br>
body temperature greater than 38°C (100.4°F) or less than 36°C (96.8°F);<br>
•	heart rate (HR) greater than 90 beats/minute;<br><br>
respiratory rate (RR) greater than 20 breaths/minute, or PCo2 less than 32 mm Hg, or requiring mechanical ventilation; and<br>
white blood cell count (WBC) either greater than 12.0 x 109/L or less than 4.0 x 109/L or having greater than 10% immature forms (bands).<br>
[0038]       These symptoms of SIRS represent a consensus definition of SIRS that may be modified or supplanted by an improved definition in the future. The present definition is used to clarify current clinical practice and does not represent a critical aspect of the invention.<br>
[0039]       A patient with SIRS has a clinical presentation that is classified as SIRS, as defined above, but is not clinically deemed to be septic. Individuals who are at risk of developing sepsis include patients in an ICU and those who have otherwise suffered from a physiological trauma, such as a burn or other insult. "Sepsis" refers to a SERS-positive condition that is associated with a confirmed infectious process. Clinical suspicion of sepsis arises from the suspicion that the SIRS-positive condition of a SIRS patient is a result of an infectious process. As used herein, "sepsis" includes all stages of sepsis including, but not limited to, the onset of sepsis, severe sepsis and MOD associated with the end stages of sepsis.<br>
[0040]       The "onset of sepsis" refers to an early stage of sepsis, i.e., prior to a stage when the clinical manifestations are sufficient to support a clinical suspicion of sepsis. Because the methods of the present invention are used to detect sepsis prior to a time that sepsis would be suspected using conventional techniques, the patient's disease status at early sepsis can only be confirmed retrospectively, when the manifestation of sepsis is more clinically obvious. The exact mechanism by which a patient becomes septic is not a critical aspect of the invention. The methods of the present invention can detect changes in the biomarker profile independent of the origin of the infectious process. Regardless of how sepsis arises, the methods of the present invention allow for determining the status of a patient having, or suspected of having, sepsis or SIRS, as classified by previously used criteria.<br>
[0041]  ,  ."Severe sepsis" refers to sepsis associated with organ dysfunction, hypoperfusion abnormalities, or sepsis-induced hypotension. Hypoperfusion abnormalities include, but are not limited to, lactic acidosis, oliguria, or an acute alteration in mental status. "Septic shock" refers to sepsis-induced hypotension that is not responsive to adequate intravenous fluid<br><br>
challenge and with manifestations of peripheral hypoperfusion. A "converter patient" refers to a SIRS-positive patient who progresses to clinical suspicion of sepsis during the period the patient is monitored, typically during an ICU stay. A "non-converter patient" refers to a SIRS-positive patient who does not progress to clinical suspicion of sepsis during the period the patient is monitored, typically during an ICU stay.<br>
[0042]       A "biomarker" is virtually any biological compound, such as a protein and a fragment thereof, a peptide, a polypeptide, a proteoglycan, a glycoprotein, a lipoprotein, a carbohydrate, a lipid, a nucleic acid, an organic or inorganic chemical, a natural polymer, and a small molecule that are present in the biological sample and that may be isolated from, or measured in, the biological sample. Furthermore, a biomarker can be the entire intact molecule, or it can be a portion thereof that may be partially functional or recognized, for example, by an antibody or other specific binding protein. A biomarker is considered to be informative if a measurable aspect of the biomarker is associated with a given state of the patient, such as a particular stage of sepsis. Such a measurable aspect may include, for example, the presence, absence, or concentration of the biomarker in the biological sample from the individual and/or its presence as part of a profile of biomarkers. Such a measurable -aspect of a biomarker is defined herein as a "feature," A feature may also be a ratio of two or more measurable aspects of biomarkers, which biomarkers may or may not be of known identity, for example. A "biomarker profile" comprises at least two such features, where the features can correspond to the same or different classes of biomarkers such as, for example, a nucleic acid and a carbohydrate. A biomarker profile may also comprise at least three, four, five, 10,20, 30 or more features. In one embodiment a biomarker profile comprises hundreds, or even thousands, of features. In another embodiment, the biomarker profile comprises at least one measurable aspect of at least one internal standard.<br>
[0043]       A "phenotypic change" is a detectable change in a parameter associated with a given state of the patient. For instance, a phenotypic change may include an increase or decrease of a biomarker in a bodily fluid, where the change is associated with sepsis or the onset of sepsis. A phenotypic change may further include a change in a detectable aspect of a given state of the patient that is not a change in a measurable aspect of a biomarker. For example, a change in phenotype may include a detectable change in body temperature, respiration rate, pulse, blood pressure, or other physiological parameter. Such changes can be determined via clinical observation and measurement using conventional techniques that are<br><br>
well-Known to the skilled artisan. As used herein, "conventional techniques" are those techniques that classify an individual based on phenotypic changes without obtaining a biomarker profile according to the present invention.<br>
[0044]       A "decision rule" is a method used to classify patients. This rule can take on one or more forms that are known in the art, as exemplified in Hastie et al9 in "The Elements of Statistical Learning," Springer-Verlag (Springer, New York (2001)), herein incorporated by reference in its entirety. Analysis of biomarkers nTthe complex mixture of molecules within the sample generates features in a data set. A decision rule may be used to act on a data set of features to, inter alia, predict the onset of sepsis, to determine the progression of sepsis, to diagnose sepsis, or to diagnose SIRS.<br>
[0045]       The application of the decision rule does not require perfect classification. A<br>
classification may be made with at least about 90% certainty, or even more, in one<br>
embodiment. In other embodiments, the certainty is at least about 80%, at least about 70%,<br>
or at least about 60%. The useful degree of certainty may vary, depending on the particular<br>
method of the present invention. "Certainty" is defined as the total number of accurately<br>
classified individuals divided by the total number of individuals subjected to classification.<br>
As used herein, "certainty" means "accuracy." Classification may also be characterized by<br>
its "sensitivity." The "sensitivity" of classification relates to the percentage of sepsis patients<br>
who were correctly identified as having sepsis. "Sensitivity" is defined in the art as the<br>
number of true positives divided by the sum of true positives and false negatives. In contrast,<br>
the "specificity" of the method is defined as the percentage of patients who were correctly<br>
identified as not having sepsis. That is, "specificity" relates to the number of true negatives<br>
divided by the sum of true negatives and false positives. In one embodiment, the sensitivity<br>
and/or specificity is at least 90%, at least 80%, at least 70% or at least 60%. The number of<br>
features that may be used to classify an individual with adequate certainty is typically about<br>
four. Depending on the degree of certainty sought, however, the number of features may be<br>
more or less, but in all cases is at least one. In one embodiment, the number of features that<br>
may be used to classify an individual is optimized to allow a classification of an individual<br>
with high certainty.	-  -<br>
[0046]       "Determining the status" of sepsis or SIRS in a patient encompasses classification of a patient's biomarker profile to (1) detect the presence of sepsis or SIRS in the patient, (2)<br><br>
predict the onset of sepsis or SIRS in the patient, or (3) measure the progression of sepsis in a patient. "Diagnosing" sepsis or SIRS means to identify or detect sepsis or SIRS in the patient. Because of the greater sensitivity of the present invention to detect sepsis before an overtly observable clinical manifestation, the identification or detection of sepsis includes the detection of the onset of sepsis, as defined above. That is, "predicting the onset of sepsis" means to classify the patient's biomarker profile as corresponding to the profile derived from individuals who are progressing from a particular stage of SIRS to sepsis or from a state of being infected to sepsis {i.e., from infection to infection with concomitant SIRS). "Detecting the progression" or "determining the progression" of sepsis or SIRS means to classify the biomarker profile of a patient who is already diagnosed as having sepsis or SIRS. For instance, classifying the biomarker profile of a patient who has been diagnosed as having sepsis can encompass detecting or determining the progression of the patient from sepsis to severe sepsis or to sepsis with MOD.<br>
[0047]       According to the present invention, sepsis may be diagnosed or predicted by obtaining a profile of biomarkers from a sample obtained from an individual. As used herein, "obtain" means "to come into possession of" The present invention is particularly useful in predicting and diagnosing sepsis in an individual who has an infection, or even sepsis, but who has not yet been diagnosed as having sepsis, who is suspected of having sepsis, or who is at risk of developing sepsis. In the same manner, the present invention may be used to detect and diagnose SIRS in an individual. That is, the present invention may be used to confirm a clinical suspicion of SIRS. The present invention also may be used to detect various stages of the sepsis process such as infection, bacteremia, sepsis, severe sepsis, septic shock and the like.<br>
[0048]       The profile of biomarkers obtained from an individual, le.t the test biomarker profile, is compared to a reference biomarker profile. The reference biomarker profile can be generated from one individual or a population of two or more individuals. The population, for example, may comprise three, four, five, ten, 15,20, 30,40, 50 or more individuals. Furthermore, the reference biomarker profile and the individual's (test) biomarker profile that -are compared in the methods of the present invention may be generated from the same individual, provided that the test and reference profiles are generated from biological samples taken at different time points and compared to one another. For example, a sample may be obtained from an individual at the start of a study period. A reference biomarker profile<br><br>
taken from that sample may then be compared to biomarker profiles generated from subsequent samples from the same individual. Such a comparison may be used, for example, to determine the status of sepsis in the individual by repeated classifications over time.<br>
[0049]       The reference populations may be chosen from individuals who do not have SIRS ("SIRS-negative"), from individuals who do not have SIRS but who are suffering from an infectious process, from individuals who are suffering from SIRS without the presence of sepsis ("SIRS-positive"), from individuals who are suffering from the onset of sepsis, from   * individuals who are sepsis-positive and suffering from one of the stages in the progression of sepsis, or from individuals with a physiological trauma that increases the risk of developing sepsis. Furthermore, the reference populations may be SIRS-positive and are then subsequently diagnosed with sepsis using conventional techniques. For example, a population of SIRS-positive patients used to generate the reference profile may be diagnosed with sepsis about 24,48,72, 96 or more hours after biological samples were taken from them for the purposes of generating a reference profile. In one embodiment, the population of SIRS-positive individuals is diagnosed with sepsis using conventional techniques about 0-36 hours, about 36-60 hours, about 60-84 hours, or about 84-108 hours after the biological samples were taken. If the biomarker profile is indicative of sepsis or one of its stages of progression, a clinician may begin treatment prior to the manifestation of clinical, symptoms of sepsis. Treatment typically will involve examining the patient to determine the source of the infection. Once locating the source, the clinician typically will obtain cultures from the site of the infection, preferably before beginning relevant empirical antimicrobial therapy and perhaps additional adjunctive therapeutic measures, such as draining an abscess or removing an infected catheter. Therapies for sepsis are reviewed in Healy, supra.<br>
[0050]       The methods of the present invention comprise comparing an individual's biomarker profile with a reference biomarker profile. As used herein, "comparison" includes any means to discern at least one difference in the individual's and the reference biomarker profiles. Thus, a comparison may include a visual inspection of chromatographic spectra, and a comparison may include arithmetical or statistical comparisons of values assigned to the features of the profiles. Such statistical comparisons include, but are not limited to, applying a decision rule. If the biomarker profiles comprise at least one internal standard, the comparison to discern a difference in the biomarker profiles may also include features of these internal standards, such that features of the biomarker are correlated to features of the<br><br>
internal standards. The comparison can predict, inter alia, the chances of acquiring sepsis or SIRS; or the comparison can confirm the presence or absence of sepsis or SIRS; or the comparison can indicate the stage of sepsis at which an individual may be.<br>
[0051]       The present invention, therefore, obviates the need to conduct time-intensive assays over a monitoring period, as well as the need to identify each biomarker. Although the invention does not require a monitoring period to classify an individual, it will be understood that repeated classifications of the individual, i.e., repeated snapshots, may be taken over time until the individual is no longer at risk. Alternatively, a profile of biomarkers obtained from the individual may be compared to one or more profiles of biomarkers obtained from the same individual at different points in time. The artisan will appreciate that each comparison made in the process of repeated classifications is capable of classifying the individual as having membership in the reference population.<br>
[0052]       Individuals having a variety of physiological conditions corresponding to the various stages in the progression of sepsis, from the absence of sepsis to MOD, may be distinguished by a characteristic biomarker profile. As used herein, an "individual" is an animal, preferably a mammal, more preferably a human or non-human primate. The terms "individual," "subject" and "patient" are used interchangeably herein. The individual can be normal, suspected of having SIRS or sepsis, at risk of developing SIRS or sepsis, or confirmed as having SIRS or sepsis. While there are many known biomarkers that have been implicated in the progression of sepsis, not all of these markers appear in the initial, preclinical stages. The subset of biomarkers characteristic of early-stage sepsis may, in fact, be determined only by a retrospective analysis of samples obtained from individuals who ultimately manifest clinical symptoms of sepsis. Without being bound by theory, even an initial pathologic infection that results in sepsis may provoke physiological changes that are reflected in particular changes in biomarker expression. Once the characteristic biomarker profile of a stage of sepsis, for example, is determined, the profile of biomarkers from a biological sample obtained from an individual may be compared to this reference profile to determine whether the test subject is also at that particular stage of sepsis.<br>
[0053]       The progression of a population from one stage of sepsis to-another, or from normalcy (i.e., a condition characterized by not having sepsis or SIRS) to sepsis or SIRS and vice versa, will be characterized by changes in biomarker profiles, as certain biomarkers are<br><br>
expressed at increasingly higher levels and the expression of other biomarkers becomes down-regulated. These changes in biomarker profiles may reflect the progressive establishment of a physiological response in the reference population to infection and/or inflammation, for example. The skilled artisan will appreciate that the biomarker profile of the reference population also will change as a physiological response subsides. As stated above, one of the advantages of the present invention is the capability of classifying an individual with a biomarker profile from a single biological sample as having membership in a particular population. The artisan will appreciate, however, that the determination of whether a particular physiological response is becoming established or is subsiding may be facilitated by a subsequent classification of the individual. To this end, the present invention provides numerous biomarkers that both increase and decrease in level of expression as a physiological response to sepsis or SIRS is established or subsides. For example, an investigator can select a feature of an individual's biomarker profile that is known to change in intensity as a physiological response to sepsis becomes established. A comparison of the same feature in a profile from a subsequent biological sample from the individual can establish whether the individual is progressing toward more severe sepsis or is progressing toward normalcy.<br>
[0054]       The molecular identity of biomarkers is not essential to the invention. Indeed, the present invention should not be limited to biomarkers that have previously been identified. (See, e.g., U.S. Patent Application Serial No. 10/400,275, filed March 26, 2003.) It is, therefore, expected that novel biomarkers will be identified that are characteristic of a given population of individuals, especially a population in one of the early stages of sepsis. In one embodiment of the present invention, a biomarker is identified and isolated. It then may be used to raise a specifically-binding antibody, which can facilitate biomarker detection in a variety of diagnostic assays. For this purpose, any immunoassay may use any antibodies, antibody fragment or derivative capable of binding the biomarker molecules (e.g., Fab, Fv, or scFv fragments). Such immunoassays are well-known in the art. If the biomarker is a protein, it may be sequenced and its encoding gene may be cloned using well-established techniques.<br>
[0055]       The methods of the present invention may be employed to screen, for example, patients admitted to an ICU. A biological sample such as, for example, blood, is taken immediately upon admission. The complex mixture of proteins and other molecules within<br><br>
the blood is resolved as a profile of biomarkers. This may be accomplished through the use of any technique or combination of techniques that reproducibly distinguishes these molecules on the basis of some physical or chemical property. In one embodiment, the molecules are immobilized on a matrix and then are separated and distinguished by laser desorption/ionization time-of-flight mass spectrometry. A spectrum is created by the characteristic desorption pattern that reflects the mass/charge ratio of each molecule or its fragments. In another embodiment, biomarkers are selected from the various mRNA species obtained from a cellular extract, and a profile is obtained by hybridizing the individual's mRNA species to an array of cDNAs. The diagnostic use of cDNA arrays is well known in the art. (See, e.g., Zou, et al9 Oncogene 21:4855-4862 (2002).) In yet another embodiment, a profile may be obtained using a combination of protein and nucleic acid separation methods.<br>
[0056]       The invention also provides kits that are useful in determining the status of sepsis or diagnosing SIRS in an individual. The kits of the present invention comprise at least one biomarker. Specific biomarkers that are useful in the present invention are set forth herein. The biomarkers of the kit can be used to generate biomarker profiles according to the present invention. Examples of classes of compounds of the kit include, but are not limited to, proteins, and fragments thereof, peptides, polypeptides, proteoglycans, glycoproteins, lipoproteins, carbohydrates, lipids, nucleic acids, organic and inorganic chemicals, and natural and synthetic polymers. The biomarker(s) may be part of an array, or the biomarker(s) may be packaged separately and/or individually. The kit may also comprise at least one internal standard to be used in generating the biomarker profiles of the present invention. Likewise, the internal standards can be any of the classes of compounds described above. The kits of the present invention also may contain reagents that can be used to detectably label biomarkers contained in the biological samples from which the biomarker profiles are generated. For this purpose, the kit may comprise a set of antibodies or functional fragments thereof that specifically bind at least two, three, four, five, 10, 20 or more of the biomarkers set forth in any one of the following TABLES that list biomarkers. The antibodies themselves may be detectably labeled. The kit also may comprise a specific biomarker binding component, such as an aptamer. If the biomarkers comprise a nucleic acid, the kit may provide an oligonucleotide probe that is capable of forming a duplex with the biomarker or with a complementary strand of a biomarker. The oligonucleotide probe may be detectably labeled.<br><br>
[0057]       The kits of the present invention may also include pharmaceutical excipients, diluents and/or adjuvants when the biomarker is to be used to raise an antibody. Examples of pharmaceutical adjuvants include, but are not limited to, preservatives, wetting agents, emulsifying agents, and dispersing agents. Prevention of the action of microorganisms can be ensured by the inclusion of various antibacterial and antifungal agents, for example, paraben, chlorobutanol, phenol sorbic acid, and the like. It may also be desirable to include isotonic agents such as sugars, sodium chloride, and the like. Prolonged absorption of an injectable pharmaceutical form can be brought about by the inclusion of agents which delay absorption such as aluminum monostearate and gelatin.<br>
Generation of Biomarker Profiles<br>
[0058]       According to one embodiment, the methods of the present invention comprise obtaining a profile of biomarkers from a biological sample taken from an individual. The biological sample may be blood, plasma, serum, saliva, sputum, urine, cerebral spinal fluid, cells, a cellular extract, a tissue sample, a tissue biopsy, a stool sample and the like. The reference biomarker profile may be obtained, for example, from a population of individuals selected from the group consisting of SIRS-negative individuals, SERS-positive individuals, individuals who are suffering from the onset of sepsis and individuals who already have sepsis. The reference biomarker profile from individuals who already have sepsis may be obtained at any stage in the progression of sepsis, such as infection, bacteremia, severe sepsis, septic shock or MOD.<br>
[0059]       In one embodiment, a separation method may be used to create a profile of biomarkers, such that only a subset of biomarkers within the sample is analyzed. For example, the biomarkers that are analyzed in a sample may consist of mRNA species from a cellular extract, which has been fractionated to obtain only the nucleic acid biomarkers within the sample, or the biomarkers may consist of a fraction of the total complement of proteins within the sample, which have been fractionated by chromatographic techniques. Alternatively, a profile of biomarkers may be created without employing a separation method. For example, a biological sample may be interrogated with a labeled compound that forms a specific complex mth a biomarker in the sample, "where the intensity of the label in the specific complex is a measurable characteristic of the biomarker. A suitable compound for forming such a specific complex is a labeled antibody. In one embodiment, a biomarker is measured using an antibody with an amplifiable nucleic acid as a label. In yet another<br><br>
embodiment, the nucleic acid label becomes amplifiable when two antibodies, each conjugated to one strand of a nucleic acid label, interact with the biomarker, such that the two nucleic acid strands form an amplifiable nucleic acid.<br>
[0060]       In another embodiment the biomarker profile may be derived from an assay, such as an array, of nucleic acids, where the biomarkers are the nucleic acids or complements thereof. For example, the biomarkers may be ribonucleic acids. The biomarker profile also may be obtained using a method selected from the group consisting of nuclear magnetic resonance, nucleic acid arrays, dot blotting, slot blotting, reverse transcription amplification and Northern analysis. In another embodiment, the biomarker profile is detected immunologically by reacting antibodies, or functional fragments thereof, specific to the biomarkers. A functional fragment of an antibody is a portion of an antibody that retains at least some ability to bind to the antigen to which the complete antibody binds. The fragments, which include, but are not limited to, scFv fragments, Fab fragments and F(ab)2 fragments, can be recombinantly produced or enzymatically produced. In another embodiment, specific binding molecules other than antibodies, such as aptamers, may be used to bind the biomarkers. In yet another embodiment, the biomarker profile may comprise a measurable aspect of an infectious agent or a component thereof. In yet another embodiment, the biomarker profile may comprise measurable aspects of small molecules, which may include fragments of proteins or nucleic acids, or which may include metabolites.<br>
[0061]       Biomarker profiles may be generated by the use of one or more separation methods. For example, suitable separation methods may include a mass spectrometry method, such as electrospray ionization mass spectrometry (ESI-MS), ESI-MS/MS, ESI-MS/(MS)n (n is an integer greater than zero), matrix-assisted laser desorption ionization time-of-flight mass spectrometry (MALDI-TOF-MS), surface-enhanced laser desorption/ionization time-of-flight mass spectrometry (SELDI-TOF-MS), desorption/ionization on silicon (DIOS), secondary ion mass spectrometry (SIMS), quadrupole time-of-flight (Q-TOF), atmospheric pressure chemical ionization mass spectrometry (APCI-MS), APCI-MS/MS, APCI-(MS)n, atmospheric pressure photoionization mass spectrometry (APPI-MS), APPI-MS/MS, and APPI-(MS)n. Other mass spectrometry methods may include, inter alia, quadrupole, fourier transform mass spectrometry (FTMS) and ion trap. Other suitable separation methods may include chemical extraction partitioning, column chromatography, ion exchange chromatography, hydrophobic (reverse phase) liquid<br><br>
chromatography, isoelectric focusing, one-dimensional polyacrylamide gel electrophoresis (PAGE), two-dimensional polyacrylamide gel electrophoresis (2D-PAGE) or other chromatography, such as thin-layer, gas or liquid chromatography, or any combination thereof. In one embodiment, the biological sample may be fractionated prior to application of the separation method.<br>
[0062]       Biomarker profiles also may be generated by methods that do not require physical separation of the biomarkers themselves. For example, nuclear magnetic resonance (NMR) spectroscopy may be used to resolve a profile of biomarkers from a complex mixture of molecules. An analogous use of NMR to classify tumors is disclosed in Hagberg, NMR Biomed. 11: 148-56 (1998), for example. Additional procedures include nucleic acid amplification technologies, which may be used to generate a profile of biomarkers without physical separation of individual biomarkers. (See Stordeur et al9 J. Immunol Methods 259: 55-64 (2002) mdTanetal,Proc. Nat'l Acad Set USA 99: 11387-11392 (2002), for example.)<br>
[0063]       In one embodiment, laser desorption/ionization time-of-flight mass spectrometry is used to create a profile of biomarkers where the biomarkers are proteins or protein fragments that have been ionized and vaporized off an immobilizing support by incident laser radiation. A profile is then created by the characteristic time-of-flight for each protein, which depends on its mass-to-charge ("m/z") ratio. A variety of laser desorption/ionization techniques are known in the art. (See, e.g., Guttman et al, Anal. Chem. 73: 1252-62 (2001) and Wei et al, Natute.399: 243-46 (1999).)<br>
[0064]       Laser desorption/ionization time-of-flight mass spectrometry allows the generation of large amounts of information in a relatively short period of time. A biological sample is applied to one of several varieties of a support that binds all of the biomarkers, or a subset thereof, in the sample. Cell lysates or samples are directly applied to these surfaces in volumes as small as 0.5 |xL, with or without prior purification or fractionation. The lysates or sample can be concentrated or diluted prior to application onto the support surface. Laser desorption/ionization is then used to generate mass spectra of the sample, or samples, in as little as three hours.<br>
[0065]       In another embodiment, the total mRNA from a cellular extract of the individual is assayed, and the various mRNA species that are obtained from the biological sample are used<br><br>
as biomarkers. Profiles may be obtained, for example, by hybridizing these mRNAs to an array of probes, which may comprise oligonucleotides or cDNAs, using standard methods known in the art. Alternatively, the mRNAs may be subjected to gel electrophoresis or blotting methods such as dot blots, slot blots or Northern analysis, all of which are known in the art. (See, e.g., Sambrook et al in i;Molecular Cloning, 3rd ed.," Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York (2001).) mRNA profiles also may be obtained by reverse transcription followed by amplification and detection of the resulting cDNAs, as disclosed by Stordeur et al, supra, for example. In another embodiment, the profile may be obtained by using a combination of methods, such as a nucleic acid array combined with mass spectroscopy.<br>
Use of a Data Analysis Algorithm<br>
[0066]       In one embodiment, comparison of the individual's biomarker profile to a reference biomarker profile comprises applying a decision rule. The decision rule can comprise a data analysis algorithm, such as a computer pattern recognition algorithm. Other suitable algorithms include, but are not limited to, logistic regression or a nonparametric algorithm that detects differences in the distribution of feature values (e.g., a Wilcoxon Signed Rank Test). The decision rule may be based upon one, two, three, four, five, 10,20 or .more features. In one embodiment, the decision rule is based on hundreds or more of features. Applying the decision rule may also comprise using a classification tree algorithm. For example, the reference biomarker profile may comprise at least three features, where the features are predictors in a classification tree algorithm. The data analysis algorithm predicts membership within a population (or class) with an accuracy of at least about 60%, at least about 70%, at least about 80% and at least about 90%.<br>
[0067]       Suitable algorithms are known in the art, some of which are reviewed in Hastie et al, supra. Such algorithms classify complex spectra from biological materials, such as a blood sample, to distinguish individuals as normal or as possessing biomarker expression levels characteristic of a particular disease state. While such algorithms may be used to increase the speed and efficiency of the application of the decision rule and to avoid investigator-bias, one of ordinary skill in the art will realize that computer-based algorithms are not required to carry out the methods of the present invention.<br><br>
[0068]       Algorithms may be applied to the comparison of biomarker profiles, regardless of the method that was used to generate the biomarker profile. For example, suitable algorithms can be applied to biomarker profiles generated using gas chromatography, as discussed in Harper, "Pyrolysis and GC in Polymer Analysis/' Dekker, New York (1985). Further, Wagner et al.Anal Chem. 74: 1824-35 (2002) disclose an algorithm that improves the ability to classify individuals based on spectra obtained by static time-of-flight secondary ion mass spectrometry (TOF-SIMS). Additionally, Bright et a!., J. Microbiol. Methods 48: 127-38 (2002) disclose a method of distinguishing between bacterial strains with high certainty (79-89% correct classification rates) by analysis of MALDI-TOF-MS spectra. Dalluge, Fresenius J. Anal Chem. 366: 701-11 (2000) discusses the use of MALDI-TOF-MS and liquid chromatography-electrospray ionization mass spectrometry (LC/ESI-MS) to classify profiles of biomarkers in complex biological samples.<br>
Biomarkers<br>
[0069]       The methods of the present invention can be carried out by generation of a   -biomarker profile that is diagnostic or predictive of sepsis or SIRS. Because profile generation is sufficient to carry out the invention, the biomarkers that constitute the profile need not be known or subsequently identified.<br>
[0070]       Biomarkers that can be used to generate the biomarker profiles of the present invention may include those known to be informative of the state of the immune system in response to infection; however, not all of these biomarkers may be equally informative. These biomarkers can include hormones, autoantibodies, soluble and insoluble receptors, growth factors, transcription factors, cell surface markers and soluble markers from the host or from the pathogen itself, such as coat proteins, lipopolysaccharides (endotoxin), lipoteichoic acids, etc. Other biomarkers include, but are not limited to, cell-surface proteins such as CD64 proteins; CD1 lb proteins; HLA Class II molecules, including HLA-DR proteins and HLA-DQ proteins; CD54 proteins; CD71 proteins; CD86 proteins; surface-bound tumor necrosis factor receptor (TNF-R); pattern-recognition receptors such as Toll-like receptors; soluble markers such as interleukins IL-1, IL-2, IL-4, IL-6, IL-8, IL-10, EL-11, IL-12, IL-13, and IL-18; tumor necrosis factor alpha (TNF-a); neopterin; C-reactive protein (CRP); procalcitonin (PCT); 6-keto Fla; thromboxane B2; leukotrienes B4, C3, C4, C5, D4 and E4; interferon gamma (IFNy); interferon alpha/beta (IFN a/p); lymphotoxin alpha (LTa); complement components (C); platelet activating factor (PAF); bradykinin; nitric<br><br>
oxide (NO); granulocyte macrophage-colony stimulating factor (GM-CSF); macrophage inhibitory factor (MIF); interleukin-1 receptor antagonist (IL-lra); soluble tumor necrosis factor receptor (sTNFr); soluble interleukin receptors sIL-lr and sIL-2r; transforming growth factor beta (TGFP); prostaglandin E2 (PGE2); granulocyte-colony stimulating factor (G-CSF); and other inflammatory mediators. (Reviewed in Oberholzer et al, Shock 16: 83-96 (2001) and Vincent et al in "The Sepsis Text," Carlet et al, eds. (Kluwer Academic Publishers, 2002). Biomarkers commonly and clinically associated with bacteremia are also candidates for biomarkers useful for the present invention, given the common and frequent occurrence of such biomarkers in biological samples. Biomarkers can include low molecular weight compounds, which can be fragments of proteins or nucleic acids, or they may include metabolites. The presence or concentration of the low molecular weight compounds, such as metabolites, may reflect a phenotypic change that is associated with sepsis and/or SIRS. In particular, changes in the concentration of small molecule biomarkers may be associated with changes in cellular metabolism that result from any of the physiological changes in response to SIRS and/or sepsis, such as hypothermia or hyperthermia, increased heart rate or rate of respiration, tissue hypoxia, metabolic acidosis or MOD. Biomarkers may also include RNA and DNA molecules that encode protein biomarkers.<br>
[0071]       Biomarkers can also include at least one molecule involved in leukocyte modulation, such as neutrophil activation or monocyte deactivation. Increased expression of CD64 and CD 1 lb is recognized as a sign of neutrophil and monocyte activation. (Reviewed in Oberholzer et al, supra and Vincent et al, supra.) Among those biomarkers that can be useful in the present invention are those that are associated with macrophage lysis products, as well as markers of changes in cytokine metabolism. (See Gagnon et al, Cell 110: 119-31 (2002); Oberholzer, et. al, supra; Vincent, et. al, supra,)<br>
[0072]       Biomarkers can also include signaling factors known to be involved or discovered to be involved in the inflammatory process. Signaling factors may initiate an intracellular cascade of events, including receptor binding, receptor activation, activation of intracellular kinases, activation of transcription factors, changes in the level of gene transcription and/or translation, and changes in metabolic processes, etc. The -signaling molecules and the processes activated by these molecules collectively are defined for the purposes of the present invention as "biomolecules involved in the sepsis pathway." The relevant predictive biomarkers can include biomolecules involved in the sepsis pathway.<br><br>
[0073]       Accordingly, while the methods of the present invention may use an unbiased approach to identifying predictive biomarkers, it will be clear to the artisan that specific groups of biomarkers associated with physiological responses or with various signaling pathways may be the subject of particular attention. This is particularly the case where biomarkers from a biological sample are contacted with an array that can be used to measure the amount of various biomarkers through direct and specific interaction with the biomarkers (e.g., an antibody array or a nucleic acid array). In this case, the choice of the components of the array may be based on a suggestion that a particular pathway is relevant to the determination of the status of sepsis or SIRS in an individual. The indication that a particular biomolecule has a feature that is predictive or diagnostic of sepsis or SIRS may give rise to an expectation that other biomolecules that are physiologically regulated in a concerted fashion likewise may provide a predictive or diagnostic feature. The artisan will appreciate, however, that such an expectation may not be realized because of the complexity of biological systems. For example, if the amount of a specific mRNA biomarker were a predictive feature, a concerted change in mRNA expression of another biomarker might not be measurable, if the expression of the other biomarker was regulated at a post-translational level. Further;, the mRNA expression level of a biomarker may be affected by multiple converging pathways that may or may not be involved in a physiological response to sepsis.<br>
[0074]       Biomarkers can be obtained from any biological sample, which can be, by way of example and not of limitation, blood, plasma, saliva, serum, urine, cerebral spinal fluid, sputum, stool, cells and cellular extracts, or other biological fluid sample, tissue sample or tissue biopsy from a host or patient. The precise biological sample that is taken from the individual may vary, but the sampling preferably is minimally invasive and is easily performed by conventional techniques.<br>
[0075]       Measurement of a phenotypic change may be carried out by any conventional technique. Measurement of body temperature, respiration rate, pulse, blood pressure, or other physiological parameters can be achieved via clinical observation and measurement. Measurements of biomarker molecules may include, for example, measurements that indicate the presence, concentration, expression level, or any other value associated with a biomarker molecule. The form of detection of biomarker molecules typically depends on the method used to form a profile of these biomarkers from a biological sample. For instance,<br><br>
biomarkers separated by 2D-PAGE are detected by Coomassie Blue staining or by silver staining, which are well-established in the art.<br>
Isolation of Useful Biomarkers<br>
[0076]       It is expected that useful biomarkers will include biomarkers that have not yet been identified or associated with a relevant physiological state. In one aspect of the invention, useful biomarkers are identified as components of a biomarker profile from a biological sample. Such an identification may be made by any well-known procedure in the art, including immunoassay or automated microsequencing.<br>
[0077]       Once a useful biomarker has been identified, the biomarker may be isolated by one of many well-known isolation procedures. The invention accordingly provides a method of isolating a biomarker that is diagnostic or predictive of sepsis comprising obtaining a reference biomarker profile obtained from a population of individuals, identifying a feature of the reference biomarker profile that is predictive or diagnostic of sepsis or one of the stages in the progression of sepsis, identifying a biomarker that corresponds with that feature, and isolating the biomarker. Once isolated, the biomarker may be used to raise antibodies that bind the biomarker if it is a protein, or it may be used to develop a specific oligonucleotide probe, if it is a nucleic acid, for example.<br>
[0078]       The skilled artisan will readily appreciate that useful features can be further characterized to determine the molecular structure of the biomarker. Methods for characterizing biomolecules in this fashion are well-known in the art and include high-resolution mass spectrometry, infrared spectrometry, ultraviolet spectrometry and nuclear magnetic resonance. Methods for determining the nucleotide sequence of nucleic acid biomarkers, the amino acid sequence of polypeptide biomarkers, and the composition and sequence of carbohydrate biomarkers also are well-known in the art.<br>
Application of the Present Invention to SIRS Patients<br>
[0079]       In one embodiment, the presently described methods are used to screen SIRS patients who are particularly at risk for developing sepsis. A biological sample is taken from a SIRS-positive patient, and a profile of biomarkers in the sample is compared to a reference profile from SIRS-positive individuals who eventually progressed to sepsis. Classification of the patient's biomarker profile as corresponding to the reference profile of a SIRS-positive population that progressed to sepsis is diagnostic that the SIRS-positive patient will likewise<br><br>
progress to sepsis. A treatment regimen may then be initiated to forestall or prevent the progression of sepsis.<br>
[0080]       In another embodiment, the presently described methods are used to confirm a clinical suspicion that a patient has SIRS. In this case, a profile of biomarkers in a sample is compared to reference populations of individuals who have SIRS or who do not have SIRS. Classification of the patient's biomarker profile as corresponding to one population or the other then can be used to diagnose the individual as having SIRS or not having SIRS.<br>
Examples<br>
[0081]       The following examples are representative of the embodiments encompassed by the present invention and in no way limit the subject embraced by the present invention.<br>
Example 1: Identification of small molecule biomarkers using quantitative liquid chromaltography/electrospray ionization mass spectrometry (LC/ESI-MS)<br>
1.1.      Samples Received and Analyzed<br>
[0082]       Reference biomarker profiles were established for two populations of patients. The first population ("the SIRS group") represented 20 patients who developed SHIS and who entered into the present study at "Day 1," but who did not progress to sepsis duringtheir hospital stay. The second population (;ithe sepsis group") represented 20 patients who likewise developed SIRS and entered into the present study at Day 1, but who progressed to sepsis at least several days after entering the study. Blood samples were taken approximately every 24 hours from each study group. Clinical suspicion of sepsis in the sepsis group occurred at "time 0," as measured by conventional techniques. "Time -24 hours" and "time -48 hours" represent samples taken about 24 hours and about 48 hours, respectively, preceding the clinical suspicion of the onset of sepsis in the sepsis group. That is, the samples from the sepsis group included those taken on the day of entry into the study (Day 1), about 48 hours prior to clinical suspicion of sepsis (time -48 hoxirs), about 24 hours prior to clinical suspicion of sepsis (time -24 hours), and on the day of clinical suspicion of the onset of sepsis (time 0). In total, 160 blood samples were analyzed: 80 samples from the 20 patients in the sepsis group and 80 samples from the 20 patients in the SIRS .group.<br><br>
1.2.	Sample Preparation<br>
[0083]       In plasma, a significant number of small molecules may be bound to proteins, which may reduce the number of small molecules that are detected by a pattern-generating method. Accordingly, most of the protein was removed from the plasma samples following the release of small molecules that may be bound to the proteins. Appropriate methods to remove proteins include, but are not limited to, extraction of the plasma with ice-cold methanol, acetonitrile (ACN), butanol, or trichloroacetic acid (TC A), or heat denaturation and acid hydrolysis. In this example, plasma was extracted with ice-cold methanol. Methanol extraction was preferred because it resulted in the detection of the highest number of small molecules. 50 juL from each plasma sample were mixed with 100 |a.L ice-cold 100% methanol, giving a final volume percent of methanol of 67%. The solution was vortexed for 60 seconds. The samples were then incubated at 4°C for 20 minutes, and proteins were precipitated by centrifugation at 12,000 rpm for 10 minutes. The supernatant was removed, dried, and resuspended in 50 ^L water. Prior to LC/MS analysis, two low molecular weight molecules, sulfachloropyridazine and octadecylamine, were added to the extracted plasma samples. These molecules served as internal standards to normalize ion intensities and retention times. Sulfachloropyridazine has a m/z of 285.0 Da, determined by MS, and elutes at 44% ACN, determined by LC; octadecylamine has a m/z of 270.3 Da and elutes at 89% ACN.<br>
1.3.	LC/ESI-MS Analysis<br>
[0084]       10 \iL of the resuspended supernatant was injected onto a 2.1 x 100 mm Cis Waters Symmetry LC column (particle size -3.5 ^m; interior bore diameter = 100 A). The column was then eluted at 300 nL/minute at a temperature of 25°C with a three-step linear gradient of ACN in 0.1% formic acid. For t = 0 - 0.5 minutes, the ACN concentration was 9.75% to 24%; for t = 0.5 - 20 minutes, the ACN concentration was 24% to 90.5%; and for t = 20-27 minutes, the ACN concentration was 90.5% to 92.4%. The aforementioned experimental conditions are herein referred to as "LC experimental conditions." Under LC experimental conditions, sulfachloropyridazine eluted at 44% ACN with a retention time of 6.4 minutes, and octadecylamine eluted at 89% ACN with a retention time of 14.5 minutes. Samples that were fractionated by LC were then subjected to ESI-MS using an Agilent MSD 1100 quadrupole mass spectrometer that, was connected in tandem to the LC column (LC/ESI-MS). Mass spectral data were acquired for ions with a mass/charge ratio (m/z)<br><br>
ranging from 100 or 150 - 1000 Da in positive ion mode with a capillary voltage of 4000 V. The LC/ESI-MS analyses were performed three times for each sample. The data may be expressed as the m/z in Daltons and retention time in minutes (as "m/z, retention time") of each ion, where the retention time of an ion is the time required for elution from a reverse phase column in a linear ACN gradient. To account for slight variations in the retention time for run to run, however, the data also may be represented as the m/z and the percentage of ACN at which the ion elutes from a Cjg column, which represent inherent properties of the ions that will not be affected greatly by experimental variability. The relationship between retention time and the percent ACN at elution is expressed by the following equations:<br><br>
[0085]       The values for these parameters nevertheless should be understood to be approximations and may vary slightly between experiments; however, ions can be recognized reproducibly, especially if the samples are prepared with one or more internal standards. In the data shown below, the m/z values were determined to within ± 0.4 m/z, while the percent ACN at which the ions elute is determined to within ± 10%.<br>
1.4.      Data Analysis and Results<br>
[0086]       Several hundred spectral features were analyzed from each plasma sample. Similar features were aligned between spectra. The choice of alignment algorithm is not crucial to the present invention, and the skilled artisan is aware of various alignment algorithms that can be used for this purpose. In total, 4930 spectral features were analyzed. For the purpose of this Example, a "feature" is used interchangeably with a "peak" that corresponds to a particular ion. Representative peaks from samples obtained from five different individuals are shown in TABLE 1. The first column lists in parentheses the m/z and percentage of ACN at elution for each ion, respectively. The remaining columns are normalized intensities of the corresponding ions from each patient, which were determined by normalizing the intensities to those of the two internal standards. Over 400 peaks bad an average normalized intensity higher than 0.1.<br><br><br>
[0087]       Various approaches may be used to identify ions that inform a decision rule to distinguish between the SIRS and sepsis groups. In this Example, the methods chosen were (1) comparing average ion intensities between the two groups, and (2) creating classification trees using a data analysis algorithm.<br>
1.4.1.   Comparing Average Ion Intensities<br>
[0088]       Comparison of averaged ion intensities effectively highlights differences in individual ion intensities between the SIRS and sepsis patients. Over 1800 normalized ion intensities were averaged separately for the sepsis group and the SIRS group. Ions having an average normalized intensity of less than 0.1 in either the sepsis group or the SIRS group were analyzed separately from-those ions having a normalized intensity greater than 0.1 in profiles from both groups. The ratios of average normalized intensities for approximately 400 ions having a normalized intensity greater than 0.1 were determined for the sepsis group<br><br>
versus the SIRS group. A distribution of relative intensity ratios of these ions is shown in FIGURE 3.<br>
[0089]       Using this method, 23 ions, listed in TABLE 2, were observed that displayed an intensity at least three-fold higher in samples from patients with sepsis than patients with SIRS (see FIGURE 3, where the natural log of the ion intensity ratio is greater than about 1.1) and that were present in at least half of the patients with sepsis and generally in about a third or a quarter of the patients having SIRS. In this context, the "presence" of a biomarker means that the average normalized intensity of the biomarker in a particular patient was at least 25% of the normalized intensity averaged over all the patients. While these ions, or subsets thereof, will be useful for carrying out the methods of the present invention, additional ions or other sets of ions will be useful as well.<br><br><br><br>
[0090]       Subsets of these biomarkers were present in at least three-fold higher intensities in a majority of the sepsis-positive population. Specifically, at least 12 of these biomarkers were found at elevated levels in over half of the sepsis-positive population, and at least seven biomarkers were present in 85% of the sepsis-positive population, indicating that combinations of these markers will provide useful predictors of the onset of sepsis. All the biomarkers were at elevated levels with respect to the SIRS-positive population, as shown in TABLE 3.<br><br><br><br>
[0091]       The two ions listed in TABLE 4 were observed to have an average normalized intensity three-fold higher in the SIRS population than in the sepsis population. (See FIGURE 3, where the natural log of the ion intensity ratio is less than about- 1.1.)<br><br>
[0092]       Thirty-two ions having an average normalized intensity of greater than 0.1 were identified that exhibited at least a three-fold higher intensity in the sepsis group versus the SIRS group. These ions are listed in TABLE 5A. Likewise, 48 ions having an average normalized intensity of less than 0.1 were identified that had a three-fold ratio of intensity higher in the sepsis group versus the SIRS group. These ions are listed in TABLE 5B. (A negative retention time reflects the fact that retention times are normalized against internal standards.)<br><br><br><br><br><br><br>
[0093]       Thus, the reference biomarker profiles of the invention may comprise a combination of features, where the features may be intensities of ions having a m/z of about 100 or 150 Da to about 1000 Da as determined by electrospray ionization mass spectrometry in the.positive mode, and where the features have a ratio of average normalized intensities in a sepsis-positive reference population versus a SIRS-positive reference population of about 3:1 or higher. Alternatively, the features may have a ratio of average normalized intensities in a sepsis-positive reference population versus a SIRS-positive reference population of about<br><br>
1:3 or lower. Because these biomarkers appear in biomarker profiles obtained from biological samples taken about 48 hours prior to the onset of sepsis, as determined by conventional techniques, they are expected to be predictors of the onset of sepsis.<br>
1.4.2.	Changes in Feature Intensity Over Time<br>
[0094]       The examined biomarker profiles displayed features that were expressed both at increasingly higher levels and at lower levels as individuals progressed toward the onset of sepsis. It is expected that the biomarkers corresponding to these features are characteristics of the physiological response to infection and/or inflammation in the individuals. For the reasons set forth above, it is expected that these biomarkers will provide particularly useful predictors for determining the status of sepsis or SIRS in an individual. Namely, comparisons of these features in profiles obtained from different biological samples from an individual are expected to establish whether an individual is progressing toward severe sepsis or whether SIRS is progressing toward normalcy.<br>
[0095]       Of the 23 ions listed in TABLE 2,14 showed a maximum intensity in the time -48 hours population, eight showed a maximum intensity in the time -24 hours population, and one showed a maximum intensity in the time 0 population. A representative change in the intensity of a biomarker over time in biological samples from the sepsis group is shown in FIGURE 4A, while the change in the intensity of the same biomarker in biological samples from the SIRS group is shown in FIGURE 4B. This particular ion, which has a m/z of 437.2 Da and a retention time of 1.42 min, peaks in intensity in the sepsis group 48 hours prior to the conversion of these patients to sepsis, as diagnosed by conventional techniques. A spike in relative intensity of this ion in a biological sample thus serves as a predictor of the onset of sepsis in the individual within about 48 hours.<br>
1.4.3.	Cross-Validation<br>
[0096]       A selection bias can affect the identification of features that inform a decision rule, when the decision rule is based on a large number of features from relatively few biomarker profiles. (See Ambroise et al, Proc. Nat 7 Acad. Set USA 99: 6562-66 (2002).) Selection bias may occur when data are used to select features, and performance then is estimated conditioned on the selected features with no consideration made for the variability in the selection process. The result is an overestimation of the classification accuracy. Without compensation for selection bias, classification accuracies may reach 100%, even<br><br>
when the decision rule is based on random input parameters. (Id.) Selection bias may be avoided by including feature selection in the performance estimation process, whether that performance estimation process is 10-fold cross-validation or a type of bootstrap procedure. (See, e.g., Hastie etal, supra, dX 7.10 - 7.11, herein incorporated by reference.)<br>
[0097]       In one embodiment of the present invention, model performance is measured by ten-fold cross-validation. Ten-fold cross-validation proceeds by randomly partitioning the data into ten exclusive groups. Each group in turn is excluded, and a model is fitted to the remaining nine groups. The fitted model is applied to the excluded group, and predicted class probabilities are generated. The predicted class probabilities can be compared to the actual class memberships by simply generating predicted classes. For example, if the probability of sepsis is, say, greater than 0.5, the predicted class is sepsis.<br>
[0098]       Deviance is a measure comparing probabilities with actual outcomes. As used herein, "deviance" is defined as:<br><br>
where P is the class probability for.the specified class. Deviance is minimized when class probabilities are high for the actual classes. Two models can make the same predictions for given data, yet a preferred model would have a smaller predictive deviance. For each of the ten iterations in the ten-fold cross-validation, the predicted deviance is calculated for the cases left out of the model fitting during that iteration. The result is 10 unbiased deviances. Typically, these 10 deviances are summed to create a general summary of model performance (i.e., accuracy) on the total data set. Because in fact 10 different models were fit, cross-validation does not prove the performance of a specific model. Rather, the 10 models were generated by a common modeling process, and cross-validation proved the performance of this process. An eleventh model arising from this process will likely have predictive performance similar to those of the first 10. Use of a ten-fold cross-validation typically results in a model performance of less than 100%, but the performance obtained after ten-fold cross-validation is expected to reflect more closely a biologically meaningful predictive accuracy of the decision rule, when applied to biomarker profiles obtained from samples outside of the training set.<br><br>
1.4.4.   Classification Tree Analysis [0099]       One approach to analyze this data is to use a classification tree algorithm that searches for patterns and relationships in large datasets. A "classification tree" is a recursive partition to classify a particular patient into a specific class (e.g., sepsis or SIRS) using a series of questions that are designed to accurately place the patient into one of the classes. Each question asks whether a patient's condition satisfies a given predictor, with each answer being used to guide the user down the classification tree until a class into which the patient falls can be determined. As used herein, a "predictor" is the range of values of the features— in this Example, ion intensities—of one ion having a characteristic m/z and elution profile from a Cis column in ACN. The "condition" is the single, specific value of the feature that is measured in the individual's biomarker profile. In this example, the "class names" are sepsis and SIRS. Thus, the classification tree user will first ask if a first ion intensity measured in the individual's biomarker profile falls within a given range of the first ion's predictive range. The answer to the first question may be dispositive in determining if the individual has SIRS or sepsis. On the other hand, the answer to the first question may further direct the user to ask if a second ion intensity measured in the individual's biomarker profile falls within a given range of the second ion's predictive range. Again, the answer to the second question may be dispositive or may direct the user further down the classification tree until a patient classification is ultimately determined.<br>
[00100]     A representative set of ion intensities collected from sepsis and SIRS populations at time 0 was analyzed with a classification tree algorithm, the results of which are shown in FIGURE 5. In this case, the set of analyzed ions included those with normalized intensities of less than 0.1. The first decision point in the classification tree is whether the ion having a m/z of about 448.5 Daltons and a percent ACN at elution of about 32.4% has a normalized intensity of less than about 0.0414. If the answer to that question is "yes," then one proceeds down the left branch either to another question or to a class name. In this case, if the normalized intensity were less than about 0.0414, then one proceeds to the class name of "SIRS," and the individual is classified as SIRS-positive, but sepsis-negative. If the answer were "no," then one proceeds down the right branch to the next decision point, and so on until a class name is reached. In this example, three decision points were used to predict a class name for an individual. While a single decision point may be used to classify patients as SIRS- or sepsis-positive, additional decision points using other ions generally improved the accuracy of the classification. The skilled artisan will appreciate that many different<br><br>
classification trees are possible from large datasets. That is? there are many possible combinations of biomarkers that can be used to classify an individual as belonging to a SIRS population or a sepsis population, for example.<br>
1.4,5. Multiple Additive Regression Trees<br>
[00101]     An automated, flexible modeling technique that uses multiple additive regression trees (MART) was used to classify sets of features as belonging to one of two populations. A MART model uses an initial offset which specifies a constant that applies to all predictions, followed by a series of regression trees. Its fitting is specified by the number of decision points in each tree, the number of trees to fit, and a "granularity constant" that specifies how radically a particular tree can influence the MART model. For each iteration, a regression tree is fitted to estimate the direction of steepest descent of the fitting criterion. A step having a length specified by the granularity constant is taken in that direction. The MART model then consists of the initial offset plus the step provided by the regression tree. The differences between the observed and predicted values are recalculated, and the cycle proceeds again, leading to a progressive refinement of the prediction. The process continues either for a predetermined number of cycles or until some stopping rule is triggered.<br>
[00102]     The number of splits in each tree is a particularly meaningful fitting parameter. If each tree has only one split, the model looks only at one feature and has no capability for combining two predictors. If each tree has two splits, the model can accommodate two-way interactions among features. With three trees, the model can accommodate three-way interactions, and so forth.<br>
[00103]     The value of sets of features in predicting class status was determined for data sets with features and known class status (e.g., sepsis or SIRS). MART provides a measure of the contribution or importance of individual features to the classification decision rule. Specifically, the degree to which a single feature contributes to the decision rule upon its selection at a given tree split can be measured to provide a ranking of features by their importance in determining the final decision rule. Repeating the MART analysis on the same data set may yield a slightly different ranking of features, especially with respect to those features that are less important in establishing the decision rule. Sets of predictive features and their corresponding biomarkers that are useful for the present invention, therefore, may vary slightly from those set forth herein.<br><br>
[00104]     One implementation of the MART technology is found in a module, or "package," for the R statistical programming environment (see Venables et al.9 in Modem Applied Statistics with S9 4   ed. (Springer, 2002); www.r-project.org). Results reported in this document were calculated using R versions 1.7.0 and 1.7.1. The module implementing MART, written by Dr. Greg Ridgeway. is called "gbm" and is also freely available for download (see www.r-project.org). The MART algorithm is amenable to ten-fold cross-validation. The granularity parameter was set to 0.05, and the gbm package's internal stopping rule was based on leaving out 20% of the data cases at each marked iteration. The degree of interaction was set to one, so no interactions among features were considered. The gbm package estimates the relative importance of each feature on a percentage basis, which cumulatively equals 100% for all the features of the biomarker profile. The features with highest importance, which together account for at least 90% of total importance, are reported as potentially having predictive value. Note that the stopping rule in the fitting of every MART model contributes a stochastic component to model fitting and feature selection. Consequently, multiple MART modeling runs based on the same data may choose slightly, or possibly even completely, different sets of features. Such different sets convey the same predictive information; therefore, all the sets are useful in the present invention. Fitting MART models a sufficient number of times is expected to produce all the possible sets of predictive features within a biomarker profile. Accordingly, the disclosed sets of predictors are merely representative of those sets of features that can be used to classify individuals into populations.<br>
1.4.6.   Logistic Regression Analysis [00105]     Logistic regression provides yet another means of analyzing a data stream from the LC/MS analysis described above. "Peak intensity" is measured by the height of a peak that appears in a spectrum at a given m/z location. The absence of a peak at a given m/z location results in an assigned peak intensity of "0." The standard deviations (SD) of the peak intensities from a given m/z location are then obtained from the spectra of the combined SIRS and sepsis populations. If there is no variation in peak intensity between SIRS and sepsis populations (i.e., the SD = 0), the peak intensity is not considered further. Before regression analysis, peak intensities are scaled, using methods well-known in the art. Scaling algorithms are generally described in, Hastie et al , supra, at Chapter 11.<br><br>
[00106]     This feature-selection procedure identified 26 input parameters (i.e., biomarkers) from time 0 biomarker profiles, listed in TABLE 6. Although input parameter are ranked in order of statistical importance, lower ranked input parameters still may prove clinically valuable and useful for the present invention. Further, the artisan will understand that the ranked importance of a given input parameter may change if the reference population changes in any way.<br><br>
[00107]     Using this same logistic regression analysis, biomarkers can be ranked in order of importance in predicting the onset of sepsis using samples taken at time -48 hours. The<br>
feature-selection process yielded 37 input parameters for the time -48 hour samples as shown<br>
*<br>
in TABLE 7.<br><br><br>
1.4.7.   Wilcoxon Signed Rank Test Analysis<br>
[00108]     In yet another method, a nonparametric test such as a Wilcoxon Signed Rank Test can be used to identify individual biomarkers of interest The features in a biomarker profile are assigned a "p-value," which indicates the degree of certainty with which the.biomarker can be used to classify individuals as belonging to a particular reference population. Generally, a p-value having predictive value is lower than about 0.05. Biomarkers having a low p-value can be used by themselves to classify individuals. Alternatively, combinations<br><br>
of two or more biomarkers can be used to classify individuals, where the combinations are chosen on the basis of the relative p-value of a biomarker. In general, those biomarkers with lower p-values are preferred for a given combination of biomarkers. Combinations of at least three, four, five, six, 10,20 or 30 or more biomarkers also can be used to classify individuals in this manner. The artisan will understand that the relative p-value of any given biomarker may vary, depending on the size of the reference population.<br>
[00109]     Using the Wilcoxon Signed Rank Test, p-values were assigned to features from biomarker profiles obtained from biological samples taken at time 0, time -24 hours and time -48 hours. These p-values are listed in TABLES 8, 9 and 10, respectively.<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
[00110]     A nonparametric test (e.g., a Wilcoxon Signed Rank Test) alternatively can be used to find p-values for features that are based on the progressive appearance or disappearance of the feature in populations that are progressing toward sepsis. In this form of the test, a baseline value for a given feature first is measured, using the data from the time of entry into the study (Day 1 samples) for the sepsis and SIRS groups. The feature intensity in sepsis and SIRS samples is then compared in, for example, time -48 hour samples to determine whether the feature intensity has increased or decreased from its baseline value. Finally, p-values are assigned to the difference from baseline in a feature intensity in the sepsis populations versus the SIRS populations. The following p-values, listed in TABLES 11-13, were obtained when measuring these differences from baseline in p-values.<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
Example 2: Identification of protein biomarkers using quantitative liquid chromatography-mass spectrometry/mass spectrometry (LC-MS/MS)<br>
2.1.      Samples Received and Analyzed<br>
[00111]     As above, reference biomarker profiles were obtained from a first population representing 15 patients ("the SIRS group") and a second population representing 15 patients who developed SIRS and progressed to sepsis ("the sepsis group"). Blood was withdrawn from the patients at Day 1, time 0, and time -48 hours. In this case, 50 - 75 |xL plasma samples from the patients were pooled into four batches: two batches of five and 10 individuals who were SIRS-positive and two batches of five and 10 individuals who were sepsis-positive. Six samples from each pooled batch were further analyzed.<br>
2.2      Sample Preparation<br>
[00112]     Plasma samples first were immunodepleted to remove abundant proteins, specifically albumin, transferrin, haptoglobulin, anti-trypsin, IgG, and IgA, which together constitute approximately 85% (wt%) of protein in the samples. Immunodepletion was performed with a Multiple Affinity Removal System column (Agilent Technologies, Palo Alto, California), which was used according to the manufacturer's instructions. At least 95% of the aforementioned six proteins were removed from the plasma samples using this system. For example, only about 0.1% of albumin remained in the depleted samples. Only an estimated 8% of proteins left in the samples represented remaining high abundance proteins, such as IgM and a-2 macroglobulin. Fractionated plasma samples were then denatured, reduced, alkylated and digested with trypsin using procedures well-known in the art. About 2 mg of digested proteins were obtained from each pooled sample.<br><br>
23.      Multidimensional LC/MS<br>
[00113]     The peptide mixture following trypsin digestion was then fractionated using LC columns and analyzed by an Agilent MSD/trap ESI-ion trap mass spectrometer configured in an LC/MS/MS arrangement. One mg of digested protein was applied at 10 pL/minute to a micro-flow Cig reverse phase (RP1) column. The RP1 column was coupled in tandem to a Strong Cation Exchange (SCX) fractionation column, which in turn was coupled to a Cis reverse phase trap column. Samples were applied to the RP1 column in a first gradient of 0 -10% ACN to fractionate the peptides on the RP1 column. The ACN gradient was followed by a 10 mM salt buffer elution, which further fractionated the peptides into a fraction bound to the SCX column and an eluted fraction that was immobilized in the trap column. The trap column was then removed from its operable connection with the SCX column and placed in operable connection with another Cis reverse phase column (RP2). The fraction immobilized in the trap column was eluted from the trap column onto the RP2 column with a gradient of 0 - 10 % ACN at 300 nL/minute. The RP2 column was operably linked to an Agilent MSD/trap ESI-ion trap massspectrometer operating at a spray voltage of 1000 - 1500 V. This cycle (RPl-SCX~Trap-RP2) was then repeated to fractionate and separate the remaining peptides using a total ACN% range from 0-80% and a salt concentration up to 1M. Other suitable configurations for LC/MS/MS may be used to generate* biomarker profiles that are useful for the invention. Mass spectra were generated in an m/z range of 200 - 2200 Da. Data dependent scan and dynamic exclusion were applied to achieve higher dynamic range. FIGURE 6 shows representative biomarker profiles generated with LC/MS and LC/MS/MS.<br>
2.4.      Data Analysis and Results<br>
[00114]     For every sample that was analyzed in the MS/MS mode, about 150,000 spectra were obtained, equivalent to about 1.5 gigabytes of information. In total, some 50 gigabytes of information were collected. Spectra were analyzed using Spectrum Mill v 2.7 software (© Copyright 2003 Agilent Technologies, Inc.). The MS-Tag database searching algorithm (Millennium Pharmaceuticals) was used to match MS/MS spectra against a National Center for Biotechnology Information (NCBI) database of human non-redundant proteins. A cutoff score equivalent to 95% confidence was used to validate the matched peptides, which were then assembled to identify proteins present in the samples. Proteins that were detectable using the present method are present in plasma at a concentration of-1 ng/mL, covering a dynamic range in plasma concentration of about six orders of magnitude.<br><br>
[00115]     A semi-quantitative estimate of the abundance of detected proteins in plasma was obtained by determining the number of mass spectra that were "positive" for the protein. To be positive, an ion feature has an intensity that is detectably higher than the noise at a given m/z value in a spectrum. In general, a protein expressed at higher levels in plasma will be detectable as a positive ion feature or set of ion features in more spectra. With this measure of protein concentration, it is apparent that various proteins are differentially expressed in the SIRS group versus the sepsis group. Various of the detected proteins that were 6iup-regulated" are shown in FIGURES 7A and 7B, where an up-regulated protein is expressed at a higher level in the sepsis group than in the SIRS group. It is clear from FIGURE 7A that the level at which a protein is expressed over time may change, in the same manner as ion # 21 (437.2 Da, 1.42 min), shown in FIGURE 4. For example, the proteins having GenBank Accession Numbers AAH15642 and NP_000286, which both are structurally similar to a serine (or cysteine) proteinase inhibitor, are expressed at progressively higher levels overtime in sepsis-positive populations, while they are expressed at relatively constant amounts in the SIRS-positive populations. The appearance of high levels of these proteins, and particularly a progressively higher expression of these proteins in an individual over time, is expected to be a predictor of the onset of sepsis. Various proteins that were down-regulated in sepsis-positive populations overtime are shown in FIGURES 8A and 8Bv The expression of some of these proteins, like the unnamed protein having the sequence shown in GenBank Accession Number NPJ)79216, appears to increase progressively or stay at relatively high levels in SIRS patients, even while the expression decreases in sepsis patients. It is expected that these proteins will be biomarkers that are particularly useful for diagnosing SIRS, as well as predicting the onset of sepsis.<br>
Example 3: Identification of Biomarkers Using an Antibody Array<br>
3.1.      Samples Received and Analyzed<br>
[00116]     Reference biomarker profiles were established for a SIRS group and a sepsis group. Blood samples were taken every 24 hours from each study group. Samples from the sepsis group included those taken on the day of entry into the study (Day 1), 48 hours prior to clinical suspicion of 5epsis (time -48 hours), and on the day of clinical suspicion of the onset of sepsis (time 0). In this example, the SIRS group and sepsis group analyzed at time 0 contained 14 and 11 individuals, respectively, while the SIRS group and sepsis group analyzed at time -48 hours contained 10 and 11 individuals, respectively.<br><br>
3.2.      Multiplex Analysis<br>
[00117]     A set of biomarkers in each sample was analyzed simultaneously in real time.<br>
using a multiplex analysis method as described in U.S. Patent No. 5,9.81,180 ("the '180<br>
patent"), herein incorporated by reference in its entirety, and in particular for its teachings of<br>
the general methodology, bead technology, system hardware and antibody detection. The<br>
immunoassay described in the ; 180 patent is representative of a type of immunoassay that<br>
could be used in the methods of the present invention. Furthermore, the biomarkers used<br>
herein are not meant to limit the scope of available biomarkers used in the methods of the<br>
present invention. For this analysis, a matrix of microparticles was synthesized, where the<br>
matrix consisted of different sets of microparticles. Each set of microparticles had thousands<br>
of molecules of a distinct antibody capture reagent immobilized on the microparticle surface<br>
and was color-coded by incorporation of varying amounts of two fluorescent dyes. The ratio<br>
of the two fluorescent dyes provided a distinct emission spectrum for each set of	*  .<br>
microparticles, allowing the identification of a microparticle within a set following the pooling of the various sets of microparticles. U.S. Patents No. 6,268,222 and No. 6,599,331 also are incorporated herein by reference in their entirety, and in particular for their teachings of various methods of labeling microparticles for multiplex analysis.<br>
[00118]     The sets of labeled beads were pooled and were combined with a plasma sample from an individual used in the study. The labeled beads were identified by passing them single file through a flow device that interrogated each microparticle with a laser beam that excited the fluorophore labels. An optical detector then measured the emission spectrum of each bead to classify the beads into the appropriate set Because the identity of each antibody capture reagent was known for each set of microparticles, each antibody specificity was matched with an individual microparticle that passes through the flow device. U.S. Patent No. 6,592,822 is also incorporated herein by reference in its entirety, and in particular for its teachings of multi-analyte diagnostic system that can be used in this type of multiplex analysis.<br>
[00119]     To determine the amount of analyte that bound a given set of microparticles, a -   reporter molecule was added such that it formed a complex with the antibodies bound to their respective analyte. In the present example, the reporter molecule was a fluorophore-labeled secondary antibody. The fluorophore on the reporter was excited by a second laser having a different excitation wavelength, allowing the fluorophore label on the secondary antibody to<br><br>
be distinguished from the fluorophores used to label the microparticles. A second optical detector measured the emission from the fluorophore label on the secondary antibody to determine the amount of secondary antibody complexed with the analyte bound by the capture antibody. In this manner, the amount of multiple analytes captured to beads could be measured rapidly and in real time in a single reaction.<br>
3.3.      Data Analysis and Results<br>
[00120]     For each sample, the concentrations of analytes that bound 162 different antibodies were measured. In this Example, each analyte is a biomarker, and the concentration of each in the sample can be a feature of that biomarker. The biomarkers were analyzed with the various 162 antibody reagents listed in TABLE 14 below, which are commercially available from Rules Based Medicine of Austin, Texas. The antibody reagents are categorized as specifically binding either (1) circulating protein biomarker components of blood, (2) circulating antibodies that normally bind molecules associated with various pathogens (identified by the pathogen that each biomarker is associated with, where indicated), or (3) autoantibody biomarkers that are associated with various disease states.<br>
TABLE 14<br>
(1) Circulating serum components<br>
Alpha-Fetoprotein<br>
Apolipoprotein Al<br>
Apolipoprotein CHI<br>
Apolipoprotein H<br>
p-2 Microglobulin<br>
Brain-Derived Neurotrophic Factor<br>
Complement 3<br>
Cancer Antigen 125<br>
Carcinoembryonic Antigen (CEA)<br>
Creatine Kinase-MB<br>
Corticotropin Releasing Factor<br>
C Reactive Protein<br>
Epithelial Neutrophil Activating Peptide-78 (ENA-78)<br>
Fatty Acid Binding Protein<br>
Factor VII<br>
Ferritin<br>
Fibrinogen<br>
Growth Hormone<br>
Granulocyte Macrophage-Colony Stimulating Factor<br>
Glutathione S-Transferase<br><br>
Intercellular adhesion molecule 1 (ICAM 1)<br>
Immunoglobulin A<br>
Immunoglobulin E<br>
Immunoglobulin M<br>
Interleukin-10<br>
Interleukin-12p40<br>
Interleukin-12p70<br>
Interleukin-13<br>
Interleukin-15<br>
Interleukin-16<br>
Interleukin-18<br>
Interleukin-la<br>
Interleukin-lp<br>
Interleukin-2<br>
Interleukin-3<br>
Interleukin-4<br>
Interleukin-5<br>
Interleukin-6<br>
Interleukin-7<br>
Interleukin-8<br>
Insulin<br>
Leptin<br>
Lipoprotein (a)<br>
Lymphotactin<br>
Macrophage Chemoattractant Protein-1 (MCP-1)<br>
Macrophage-Derived Chemokine (MDC)<br>
Macrophage Inflammatory Protein-1(5 (MIP-ip)<br>
Matrix Metalloproteinase-3 (MMP-3)<br>
Matrix Metalloproteinase-9 (MMP-9)<br>
Myoglobin<br>
Prostatic Acid Phosphatase<br>
Prostate Specific Antigen, Free<br>
Regulated on Activation, Normal T-cell Expressed and Secreted (RANTES)<br>
Serum Amyloid P<br>
Stem Cell Factor<br>
Serum glutamic oxaloacetic transaminase (SGOT)<br>
Thyroxine Binding Globulin<br>
Tissue inhibitor of metalloproteinase 1 (TIMP 1)<br>
Tumor Necrosis Factor-a (TNF-a)<br>
Tumor Necrosis Factor-p (TNF-p)<br>
Thrombopoietin<br>
Thyroid Stimulating Hormone (TSH)  *<br>
von Willebrand Factor<br><br>
(2) Antibodies that bind the indicated pathogen marker<br>
Adenovirus<br>
Bordetella pertussis<br>
Campylobacter jejuni<br>
Chlamydia pneumoniae<br>
Chlamydia trachomatis<br>
Cholera Toxin<br>
Cholera Toxin (subunit B)<br>
Cytomegalovirus<br>
Diphtheria Toxin<br>
Epstein-Barr Virus-Viral Capsid Antigen<br>
Epstein Barr Virus Early Antigen<br>
Epstein Barr Virus Nuclear Antigen<br>
Helicobacter pylori<br>
Hepatitis B Core<br>
Hepatitis B Envelope<br>
Hepatitis B Surface (Ad)<br>
Hepatitis B Surface (Ay)<br>
Hepatitis C Core<br>
Hepatitis C Non-Structural 3<br>
Hepatitis C Non-Structural 4<br>
Hepatitis C Non-Structural 5<br>
Hepatitis D<br>
Hepatitis A<br>
Hepatitis E Virus (or£2 3KD)<br>
Hepatitis E Virus (orf2 6KD)<br>
Hepatitis E Virus (orf3 3KD)<br>
Human Immunodeficiency Virus-1 p24<br>
Human Immunodeficiency Virus-1 gpl20<br>
Human Immunodeficiency Virus-1 gp41<br>
Human Papilloma Virus<br>
Herpes Simplex Virus-1/2<br>
Herpes Simplex Virus-1 gD<br>
Herpes Simplex Virus-2 gG<br>
Human T-Cell Lymphotropic Virus 1/2<br>
Influenza A<br>
Influenza A H3N2<br>
Influenza B<br>
Leishmania donovani<br>
Lyme Disease Virus<br>
Mycobacteria pneumoniae<br>
Mycobacteria tuberculosis<br>
Mumps Virus<br>
Parainfluenza 1<br>
Parainfluenza 2<br>
Parainfluenza 3<br><br>
Polio Virus<br>
Respiratory Syncytial Virus Rubella Virus Rubeola Virus Streptolysin 0 (SLO) Trypanosoma cruzi Treponema pallidum 15KD Treponema pallidum p47 Tetanus Toxin Toxoplasma Varicella zoster<br>
(3) Autoantibodies<br>
Anti-Saccharomyces cerevisiae antibodies (ASCA) Anti-p-2 Glycoprotein Anti-Centromere Protein B Anti-Collagen Type 1 Anti-Collagen Type 2 Anti-Collagen Type 4 Anti-Collagen Type 6 . Anti-Complement C1 q Anti-Cytochrome P450 Anti-Double Stranded DNA (ds DNA) Anti-Histone Anti-Histone HI Anti-Histone H2a Anti-Histone H2b Anti-Histone H3 Anti-Histone H4<br>
Anti-Heat Shock Cognate Protein 70 (HSC 70) Anti-Heat Shock Protein 32 (HO) Anti-Heat Shock Protein 65 Anti-Heat Shock Protein 71 Anti-Heat Shock Protein 90 a Anti-Heat Shock Protein 90 p Anti-Insulin<br>
Anti-Histidyl-tRNA Synthetase (JO-1) Anti-Mitochondrial<br>
Anti-Myeloperoxidase (perinuclear autoantibodies to neutrophil cytoplasmic antigens) Anti-Pancreatic Islet Cells (Glutamic Acid Decarboxylase Autoantibody) Anti-Proliferating Cell Nuclear Antigen (PCNA) Polymyositis-1 (PM-1)<br>
Anti-Proteinase 3 (cytoplasmic autoantibodies to neutrophil cytoplasmic antigens) Anti-Ribosomal P Anti-Ribonuclear protein (RNP) Anti-Ribonuclear protein (a)<br><br>
Anti-Ribonuclear protein (b)<br>
Anti-Topoisomerase I (Scl 70)<br>
Anti-Ribonucleoprotein Smith Ag (Smith)<br>
Anti-Sjogren*s Syndrome A (Ro) (SSA)<br>
Anti-Sjogreris Syndrome B (La) (SSB)<br>
Anti-T3<br>
Anti-T4<br>
Anti-Thyroglobulin<br>
Anti-Thyroid microsomal<br>
Anti-tTG (Tissue Transglutaminase, Celiac Disease)<br>
[00121]     Various approaches may used to identify features that can inform a decision rule to classify individuals into the SIRS or sepsis groups. The methods chosen were logistic regression and a Wilcoxon Signed Rank Test.<br>
3.3.1.  Analysis of the Data Using Logistic Regression<br>
[00122]     Quantitative results from the biomarker immunoassays were analyzed using logistic regression. The top 26 biomarkers for the time 0 populations, which comprise a pattern that distinguishes SIRS from sepsis, are listed in TABLE 15. For the time -48 hours population, the top 14 biomarkers, which comprise a pattern that distinguishes SIRS from sepsis, are listed in TABLE 16. The data in Tables 15 &amp; 16 demonstrate those biomarkers that comprise the patterns that distinguish the SIRS and sepsis groups.<br><br><br><br><br><br>
3.3.2,   Analysis of the Data Using a Wilcoxon Signed Rank Test<br>
[00123]     A Wilcoxon Signed Rank Test also was used to identify individual protein biomarkers of interest. Biomarkers listed in TABLE 14 were assigned a p-value by comparison of sepsis and SIRS populations at a given time, in the same manner as in Example 1.4.7., TABLES 8-10, above. For this analysis, the sepsis and SIRS populations at time 0 (TABLE 17) constituted 23 and 25 patients, respectively; the sepsis and SIRS populations at time -24 hours (TABLE 18) constituted 25 and 22 patients, respectively; and the sepsis and SIRS populations at time -48 hours (TABLE 19) constituted 25 and 19 patients, respectively.<br><br><br><br>
*   {00124]     In addition, p-values were based on the progressive appearance or disappearance of the feature in populations that are progressing toward sepsis, in the same manner as in Example 1.4.7., TABLES 11-13. For this analysis, the population sizes were the same as<br><br>
shown immediately above, except that the sepsis and SIRS populations at time -48 hours constituted 22 and 18 patients, respectively.<br><br>
3.33.  Analysis of the Data Using Multiple Adaptive Regression Trees (MART) [001251     Data from protein biomarker profiles obtained from time 0 samples were analyzed using MART, as described above in Example 1.4.5. In this analysis, the time 0 hours sepsis<br><br>
population consisted of 23 patients and the SIRS population consisted of 25 patients. Features corresponding to all 164 biomarkers listed in TABLE 14 were analyzed. The fitted model included 24 trees, and the model allowed no interactions among the features. Using ten-fold cross-validation, the model correctly classified 17 of 25 SIRS patients and 17 of 23 sepsis patients, giving a model sensitivity of 74% and a specificity of 68%. The biomarkers are ranked in order of importance, as determined by the model, in TABLE 23. All features with zero importance are excluded. Markers indicated with a sign of "1" were expressed at progressively higher levels in sepsis-positive populations as sepsis progressed, while those biomarkers with a sign of "-1" were expressed at progressively lower levels.<br><br>
Example 4: Identification of Biomarkers Using SELDI-TOF-MS<br>
4.1,      Sample Preparation and Experimental Design<br>
[00126]     SELDI-TOF-MS (SELDI) provides yet another method of determining the status of sepsis or SIRS in an individual, according to the methods of the invention. SELDI allows a non-biased means of identifying predictive features in biomarker profiles from biological samples. A sample is ionized by a laser beam, and the m/z of the ions is measured. The biomarker profile comprising various ions then may be analyzed by any of the algorithms described above.<br><br>
[00127]     A representative SELDI experiment using a WCX2 sample platform, or "chip/' is described. Each type of chip adsorbs characteristic biomarkers; therefore, different biomarker profiles may be obtained from the same sample, depending on the particular type of chip that is used. Plasma (500 (aL) was prepared from blood collected in a PPTTM Vacutainer    tube (Becton, Dickinson and Company, Franklin Lakes, New Jersey) per conventional protocol. The plasma was divided into 100 uL aliquots and was stored at -S0°C. The WCX-2 chip (Ciphergen Biosystems, Inc., Fremont, California) was prepared in a Ciphergen bioprocessor according to the manufacturer protocol, using a Biomek 2000 robot (Beckman Coulter). One WCX-2 chip has eight binding spots. The spots on the chip were successively washed twice with 50 |iL of 50% acetonitrile for 5 minutes, then with 50 jiL of 10 mM of HC1 for 10 minutes, and finally with 50 (J.L of de-ionized water for 5 minutes. After washing, the chip was conditioned twice with 50 p.L of WCX2 buffer for 5 minutes before the introduction of plasma samples. Wash buffers for WCX2 chips, and for other chip types, including H50, IMAC and SAX2/Q10 chips, are given in TABLE 24.<br><br>
[00128]     To each spot on the conditioned WCX-2 chip, 10 \xL of the plasma sample and 90 \iL of WCX-2 binding buffer (20 mM ammonium acetate and 0.1% Triton X-100, pH 6) were added. After incubation at room temperature for 30 minutes with shaking, the spots were washed twice with 100 \iL of the WCX-2 binding buffer, followed by two washes with 100 ]j.L of de-ionized water. The chip was then dried and spotted twice with 0.75 JIL of a saturated solution of matrix materials, such as a-cyanohydroxycinnamic acid (99%) (CHCA) or sinapic acid (SPA), in a 50% acetonitrile, 0.5% TFA aqueous solution. The chips with bound plasma proteins were then read by SELDI-TOF-MS using the experimental conditions shown in TABLE 25.<br><br>
TART /P. &gt;*<br>
[00129]     TABLES 26 - 49 show p-values for SELDI experiments conducted on plasma samples under the conditions indicated in TABLE 25. In each table, the type of chip is shown, which is WOX-2, H50, Q10 or IMAC. For each chip, experiments were performed with either a CHCA matrix, an SPA matrix at high energy (see TABLE 25), or an SPA matrix at low energy. Further, for each matrix, samples from time 0 hours, time -24 hours, and time -48 hours were analyzed. The p-values determined for the listed ions were determined using a nonparametric test, which in this case was a Wilcoxon Signed Rank Test. Only ions with a corresponding p-value of less than 0.05 are listed (blank boxes in the TABLES below indicate those ions in the sample having a p-value not less than 0.05). Finally, in each sample, p-values were assigned to the difference from baseline in a feature intensity in the sepsis populations versus the SIRS populations, which are labeled in the TABLES below as "p-values for features differenced from baseline" (as in Example 1.4.7., supra).  The m/z values listed in the TABLES have an experimental error of about ± 2%.<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
[00130]     MART analysis was performed on the data from SELDI analysis set forth in TABLES 26 - 49, as described at Example 1.4.5., supra. TABLE 50 shows the results of two SELDI experiments from time 0 samples in which the accuracy of the classification meets or exceeds about 60%.<br><br>
[00131]     Having now fully described the invention with reference to certain representative ..embodiments and details, it will be apparent to one of ordinary skill in the art that changes and modifications can be made thereto without departing from the spirit or scope of the invention as set forth herein.<br><br><br>
What is claimed is:<br>
1.	A method of determining the status of sepsis "in an individual, comprising:<br>
(a)	obtaining a first biomarker profile from a first biological sample taken from the individual; and<br>
(b)	comparing the individual's first biomarker profile to a reference biomarker profile obtained from a reference population;<br>
wherein a single such comparison is capable of classifying the individual as belonging to or not belonging to the reference population, and wherein the comparison determines the status of sepsis in the individual.<br>
2.	A method of determining the status of sepsis in an individual, comprising:<br>
(a)	obtaining a first biomarker profile at a single point in time from the individual; and<br>
(b)	comparing the individual's first biomarker profile to a reference biomarker profile;<br>
wherein the comparison of the biomarker profiles determines the status of sepsis in the individual with an accuracy of at least about 60%.<br>
3.	A method of determining the status of sepsis in an individual, comprising comparing (i) a first biomarker profile generated from a first biological sample taken from the individual at a single point in time with (ii) a reference biomarker profile generated from a reference population, wherein the comparison comprises applying a decision rule that determines the status of sepsis in the individual.<br>
4.	A method of determining the status of sepsis in an individual, comprising:<br><br>
(a)	obtaining a first biomarker profile from a first biological sample taken from the individual; and<br>
(b)	comparing the individual's first biomarker profile to a reference biomarker profile obtained from biological samples from a reference population,<br>
wherein the reference population is selected from the group consisting of a normal reference population, a SIRS-positive reference population, an infected/SIRS-negative reference population, a sepsis-positive reference population, a reference population at a stage in the progression of sepsis, a SIRS-positive reference population confirmed as having sepsis by<br><br>
conventional techniques after about 0-36 hours, a SIRS-positive reference population confirmed as having sepsis by conventional techniques after about 36 - 60 hours, and a SIRS-positive reference population confirmed as having sepsis by conventional techniques after about 60 - 84 hours, and wherein a single such comparison is capable of classifying the individual as belonging to or not belonging to the reference population, and wherein the comparison determines the status of sepsis in the individual.<br>
5.	A method of determining the status of sepsis in an individual, comprising comparing a measurable characteristic of at least one biomarker between (i) a first biomarker profile obtained from a first biological sample from the individual and (ii) a biomarker profile obtained from biological samples from a reference population, wherein the comparison classifies the individual as belonging or not belonging to the reference population, and wherein the comparison determines the status of sepsis in the individual.<br>
6.	A method of determining the status of sepsis in an individual, comprising:<br><br>
(a)	selecting at least two features from a set of biomarkers in a first biomarker profile generated from a first biological sample of an individual; and<br>
(b)	comparing the features to a set of the same biomarkers in a reference biomarker profile generated from biological samples from a reference population,<br>
wherein a single such comparison is capable of classifying the individual as belonging to or not belonging to the reference population with an accuracy of at least about 60%, and wherein the comparison determines the status of sepsis in the individual.<br>
7.	A method of determining the status of sepsis in an individual, comprising:<br>
(a)	determining an abundance or changes in an abundance of at least two biomarkers in a first biomarker profile obtained from a first biological sample from the individual, and<br>
(b)	comparing the abundance or the changes in the abundance of the at least two biomarkers in the individual's first biomarker profile to an abundance or* changes in an abundance of these biomarkers in a reference biomarker profile obtained from biological samples from a reference population,<br><br>
wherein the comparison is capable of classifying the individual as belonging to or not belonging to the reference population, and wherein the comparison determines the status of sepsis in the individual<br>
8.	A method of determining the status of sepsis in an individual, comprising determining an abundance or a change in an abundance of at least one biomarker of a first biomarker profile obtained from a first biological sample from the individual as compared to an abundance or change in an abundance of the at least one biomarker of a reference biomarker profile obtained from biological samples from a (i) SIRS-positive reference population that contracted sepsis and (ii) a SIRS-positive reference population that did not, wherein the biomarkers are selected from the group consisting of the biomarkers listed in any one of TABLES 15 - 23 and 26 - 50.<br>
9.	The method of claim 2, wherein the individual's first biomarker profile is from a first biological sample from the individual, and the reference biomarker profile is from biological samples taken from the reference population.<br>
10.	The method of any one of claims 1 and 3-9, wherein the biological sample is selected from the group consisting of blood, saliva, serum, plasma, urine, stool, cerebral spinal fluid, cells, a cellular extract, a tissue sample, and a tissue biopsy.<br>
11.	The method of claim 1, further comprising:<br><br>
(a)	obtaining a second biomarker profile from a second biological sample taken from the individual; and<br>
(b)	comparing the individual's second biomarker profile to the reference biomarker profile;<br>
wherein the second comparison is capable of classifying the individual as belonging to or not belonging to the reference population, and wherein the second comparison determines the status of sepsis in the individual.<br>
12.	The method of any one of claims 1 and 3 - 9, further comprising repeating the method<br>
at least once, wherein a separate biomarker profile is obtained from the individual from a<br>
separate biological sample taken each time the method is repeated.<br><br>
13.	The method of claim 12. wherein the biological samples from the individual are taken about 24 hours apart.<br>
14.	The method of any one of claims 1 - 8, wherein the determining the status of sepsis in the individual comprises predicting the onset of sepsis in the individual.<br>
15.	The method of claim 14. wherein the onset of sepsis is predicted at least about 24 hours prior to the determination of sepsis in the individual using conventional techniques.<br>
16.	The method of claim 14. wherein the onset of sepsis is predicted at least about 48 hours prior to the determination of sepsis in the individual using conventional techniques.<br>
17.	The method of claim 14. wherein the onset of sepsis is predicted at least about 96 hours prior to the determination of sepsis in the individual using conventional techniques.<br>
18.	The method of any one of claims 1 - 8, wherein the determining the status of sepsis in the individual comprises determining the progression of sepsis in the individual.<br>
19.	The method of any one of claims 1-8, wherein the determining the status of sepsis in the individual comprises diagnosing sepsis in the individual.<br>
20.	The method of any one of claims 1 - 2 and 4-8, wherein the comparison comprises applying a decision rule.<br>
21.	The method of claim 3 or claim 20, wherein applying the decision rule comprises using a data analysis algorithm.<br>
22.	The method of claim 21, wherein the data analysis algorithm comprises the use of a classification tree.<br>
23.	The method of claim 21, wherein the data analysis algorithm is nonparametric.<br><br>
24.	The method of claim 23. wherein the data analysis algorithm detects differences in a<br>
distribution of feature values.<br>
25.	The method of claim 24. wherein the nonparametric algorithm comprises using a -Wilcoxon Signed Rank Test.<br>
26.	The method of claim 21. wherein the data analysis algorithm comprises using a multiple additive regression tree.<br>
27.	The method of claim 21. wherein the data analysis algorithm is a logistic regression.<br>
28.	The method of claim 21. wherein the data analysis algorithm comprises at least two input parameters.<br>
29.	The method of claim 28. wherein the data analysis algorithm comprises at least five input parameters.<br>
30.	The method of claim 29. wherein the data analysis algorithm comprises at least ten input parameters.<br>
31.	The method of claim 30, wherein the data analysis algorithm comprises at least twenty input parameters.<br>
32.	The method of claim 21, wherein the data analysis algorithm uses at least two of the features set forth in any one of TABLES 15-23 and 26 - 50 as input parameters.<br>
33.	The method of claim 20, wherein the decision rule determines the status of sepsis in the individual with an accuracy of at least about 60%.<br>
34.	The method of claim 33, wherein the decision rule determines the status of sepsis in the individual with an accuracy of at least about 70%.<br><br>
35.	The method of claim 34, wherein the decision rule determines the status of sepsis in the individual with an accuracy of at least about 80%.<br>
36.	The method of claim 35, wherein the decision rule determines the status of sepsis in the individual with an accuracy of at least about 90%.<br>
37.	The method of claim 33, wherein the determination of the status of sepsis in the individual is made at least about 48 hours prior to clinical suspicion that the individual had sepsis, as determined using conventional techniques.<br>
38.	The method of claim 33, wherein the decision rule has been subjected to ten-fold cross-validation.<br>
39.	The method of any one of claims 1-8, wherein the reference biomarker profile is obtained from a population comprising a single individual.<br>
40.	The method of any one of claims 1-8, wherein the reference biomarker profile is obtained from a population comprising at least two individuals.<br>
41.	The method of claim 40, wherein the reference biomarker profile is obtained from a population comprising at least 20 individuals.<br>
42.	The method of any one of claims 1-3 and 5-8, wherein the reference biomarker profile is obtained from a population selected from the group consisting of a normal reference population, a SIRS-positive reference population, an infected/SIRS-negative reference population, a sepsis-positive reference population, a reference population at a stage in the progression of sepsis, a SIRS-positive reference population confirmed as having sepsis by conventional techniques after about 0-36 hours, a SIRS-positive reference population confirmed as having sepsis by conventional techniques after about 36 - 60 hours, and a SIRS-positive reference population confirmed as having sepsis by conventional techniques after about 60 - 84 hours.<br><br>
43.	The method of any one of claims 1 and 3-9, further comprising comparing a second biomarker profile from the individual with a reference biomarker profile, wherein the second biomarker profile is obtained from a second biological sample taken from the individual.<br>
44.	The method of claim 43, wherein the second biological sample from the individual is taken about 24 hours after the first biological sample is taken from the individual.<br>
45.	The method of claim 43. wherein the second biomarker profile is compared to a different reference biomarker profile than the first biomarker profile.<br>
46.	The method of any one of claims 1 and 3-9, wherein the individual's first biomarker profile and the reference biomarker profile comprise a measurable aspect of at least one nucleic acid.	*  .<br>
47.	The method of claim 46, wherein the nucleic acid is an mRNA.<br>
48.	The method of any one of claims 1 - 8, wherein the individual's first biomarker profile and the reference biomarker profile comprise a measurable aspect of at least one polypeptide.<br>
49.	The method of claim 48, wherein measurement of said measurable aspect comprises contacting the at least one polypeptide with an antibody or a functional fragment thereof that specifically binds the at least one polypeptide.<br>
50.	The method of claim 49, wherein said antibody or a functional fragment thereof is detectably labeled.<br>
51.	The method of claim 50, wherein the label is an amplifiable nucleic acid.<br>
52.	The method of claim 49, wherein the at least one polypeptide is present in blood.<br>
53.	The method of claim 49, wherein the at least one polypeptide is a cell surface protein.<br><br>
54.	The method of claim 49, wherein the at least one polypeptide is a component of a<br>
pathogen.<br>
55.	The method of claim" 49, wherein the at least one polypeptide is an antibody that binds a component of a pathogen.<br>
56.	The method of claim 49, wherein the at least one polypeptide is an autoantibody.<br>
57.	The method of any one of claims 1 and 3-9, comprising contacting proteins from the biological sample obtained from the individual with an array of antibodies, wherein the antibodies of the array are immobilized.<br>
58.	The method of any one of claims 1 and 3-9, wherein said biological sample is fractionated prior to said obtaining of said individual's first biomarker profile.<br>
59.	The method of any one of claims 1-8, wherein at least one separation method is used to obtain said individual's first biomarker profile.<br>
60.	'    The method of claim 59, wherein at least two separation methods are used to obtain<br>
said individual's first biomarker profile.<br>
61.	The method of claim 59, wherein said at least one separation method comprises mass spectrometry.<br>
62.	The method of claim 61, wherein said mass spectrometry is selecting from the group consisting of electrospray ionization mass spectrometry (ESI-MS), ESI-MS/MS, ESI-MS/(MS)n, matrix-assisted laser desorption ionization time-of-flight mass spectrometry (MALDI-TOF-MS), surface-enhanced laser desorption/ionization time-of-flight mass spectrometry (SELDI-TOF-MS), desorption/ionization on silicon (DIOS), secondary ion mass spectrometry (SIMS), quadrupole time-of-flight (Q-TOF), atmospheric pressure chemical ionization mass spectrometry (APCI-MS), APCI-MS/MS, APCI-(MS)n, atmospheric pressure photoionization mass spectrometry (APPI-MS), APPI-MS/MS, and<br><br>
APPI-(MS)n, quadrupole mass spectrometry, fourier transform mass spectrometry (FTMS), and ion trap mass spectrometry, where n is an integer greater than zero.<br>
63.	The method of claim 62. wherein the at least one separation method comprises SELDI-TOF-MS.<br>
64.	The method of claim 59. wherein the at least one separation method is selected from the group consisting of chemical extraction partitioning, ion exchange chromatography, reverse phase liquid chromatography, isoelectric focusing, one-dimensional polyacrylamide gel electrophoresis (PAGE), two-dimensional polyacrylamide gel electrophoresis (2D-PAGE), thin-layer chromatography, gas chromatography, liquid chromatography, and any combination thereof.<br>
65.	The method of claim 59, wherein at least two different separation methods are used to obtain said individual's biomarker profile.<br>
66.	The method of any one of claims 1 - 8, wherein said individual's first biomarker profile and reference biomarker profile comprise a measurable aspect of an infectious agent or a component thereof.<br>
67.	The method of claim 66, wherein said component is selected from the group consisting of a viral coat protein, a lipopolysaccharide and lipoteichoic acid.<br>
68.	The method of any one of claims 1-8, wherein said individual's first biomarker profile and said reference biomarker profile comprise a measurable aspect of a biomarker that is informative of the state of the immune system in response to infection.<br>
69.	The method of any one of claims 1-8, wherein said individual's first biomarker profile and said reference biomarker profile comprise a measurable aspect of a biomarker selected from the group consisting of hormones, autoantibodies, growth factors, transcription factors, cell surface markers, and soluble proteins derived from cells.<br><br>
70.	The method of any one of claims 1-8, wherein said individual's first biomarker profile and said reference biomarker profile comprise a measurable aspect of a biomarker associated with bacteremia.<br>
71.	The method of any one of claims 1-8, wherein said individual's first biomarker profile and said reference biomarker profile comprise a measurable aspect of a biomarker associated with macrophage lysis.<br>
72.	The method of any one of claims 1-8, wherein said individual's first biomarker profile and said reference biomarker profile comprise a measurable aspect of a biomarker associated with a sepsis pathway.<br>
73.	The method of any one of claims 1 — 8, wherein said individual's first biomarker profile and said reference biomarker profile comprise a measurable aspect of an autoantibody.<br>
74.	The method of any one of claims 1 - 8, wherein said individual's first biomarker profile and said reference biomarker profile comprise a measurable aspect of a biomarker associated with a physiological condition selected from the group consisting of tissue hypoxia, multiple organ dysfunction, and metabolic acidosis.<br>
75.	A method of predicting the onset of sepsis in an individual, comprising:<br><br>
(a)	measuring an aspect of at least two features in a biomarker profile, wherein the biomarker profile comprises at least two biomarkers selected from the set of biomarkers set forth in any one of TABLES 15-23 and 26 - 50; and<br>
(b)	comparing the measured aspect of said at least two features with the value of a corresponding aspect of the same at least two features in a reference popxilation,<br>
wherein a single such comparison is capable of classifying the individual as belonging to or not belonging to the reference population, and wherein the comparison predicts the onset of sepsis in the individual.<br><br>
76.	The method of claim 75, wherein said prediction of the onset of sepsis is made about 12-36 hours prior to the onset of sepsis, where the onset of sepsis is determined by conventional techniques.<br>
77.	The method of claim 75, wherein said prediction of the onset of sepsis is made about 36 - 60 hours prior to the onset of sepsis, where the onset of sepsis is determined by conventional techniques.<br>
78.	The method of claim 75. wherein said prediction of the onset of sepsis is made about 60.- 84 hours prior to the onset of sepsis, where the onset of sepsis is determined by conventional techniques.<br>
79.	A method of diagnosing SIRS in an individual, comprising:<br><br>
(a)	obtaining a first biomarker profile from a first biological sample taken from the individual; and<br>
(b)	comparing the individual's first biomarker profile to a reference biomarker profile obtained from a reference population,<br>
wherein a single such comparison is capable of classifying the individual as belonging to or not belonging to the reference population, and wherein the comparison diagnoses SIRS in the individual.<br>
80.	A method of diagnosing SIRS in an individual, comprising:<br>
(a)	obtaining a biomarker profile at a single point in time from the individual; and<br>
(b)	comparing the individual's biomarker profile to a reference biomarker profile, wherein the comparison of the biomarker profiles can diagnose SIRS in the individual with an accuracy of at least about 60%.<br>
81.	A method of diagnosing SIRS in an individual, comprising comparing (i) a first<br>
biomarker profile generated from a first biological sample taken from the individual at a<br>
single point in time with (ii) a reference biomarker profile generated from a reference<br>
population, wherein the comparison comprises applying a decision rule that determines the<br>
status of SIRS in the individual.<br><br>
82.	A method of diagnosing SIRS in an individual, comprising:<br>
(a)	obtaining a first biomarker profile from a first biological sample taken from the individual; and<br>
(b)	comparing the individual's first biomarker profile to a reference biomarker profile obtained from biological samples from a reference population,<br>
wherein the reference population is selected from the group consisting of a normal reference population, a SIRS-positive reference population, and an infected/SIRS-negative reference population, a sepsis-positive reference population, a reference population at a stage in the progression of sepsis, a SIRS-positive reference population confirmed as having sepsis by conventional techniques after about 0-36 hours, a SIRS-positive reference population confirmed as having sepsis by conventional techniques after about 36 - 60 hours, and a SIRS-positive reference population confirmed as having sepsis by conventional techniques after about 60 - 84 hours, and wherein a single such comparison is capable of classifying the individual as belonging to or not belonging to the reference population, and wherein the comparison diagnoses SIRS in the individual.<br>
83.	A method of diagnosing SIRS in an individual, comprising comparing a measurable characteristic of at least one biomarker between (i) a first biomarker profile obtained from a first biological sample from the individual and (ii) a biomarker profile obtained from biological samples from a reference population, wherein the comparison classifies the individual as belonging or not belonging to the reference population, and wherein the comparison diagnoses SIRS in the individual.<br>
84.	A method of diagnosing SIRS in an individual, comprising:<br><br>
(a)	selecting at least two features from a set of biomarkers in a biomarker profile generated from a first biological sample of an individual; and<br>
(b)	comparing the features to a set of the same biomarkers in a biomarker profile generated from biological samples from a reference population,<br>
wherein a single such comparison is capable of classifying the individual as belonging to or not belonging to the reference population with an accuracy of at least about 60%, and wherein the comparison diagnoses SIRS in the individual.<br><br>
85.	A method of diagnosing SIRS in an individual, comprising:<br>
(a)	determining an abundance or change in an abundance of at least two biomarkers contained in a first biological sample of an individual, and<br>
(b)	comparing the abundance or change in an abundance of the biomarkers in the individual's biological sample to an abundance of these biomarkers in biological samples from a reference population,<br>
wherein the comparison is capable of classifying the individual as belonging to or not belonging to the reference population, and wherein the comparison diagnoses SIRS in the individual.<br>
86.	A method of diagnosing SIRS in an individual, comprising determining the abundance or a change in abundance of at least one biomarker obtained from a biological sample from the individual as compared to an abundance or change in an abundance of the at least one biomarker obtained from biological samples from a normal reference population, wherein the biomarkers are selected from the group consisting of the biomarkers listed in any one of TABLES 15 - 23 and 26 - 50.<br>
87.	A method of isolating a biomarker, wherein said biomarker can be used to generate a biomarker profile to diagnose or predict sepsis, said method comprising:<br><br>
(a)	obtaining a reference biomarker profile, said reference biomarker profile obtained from a population of individuals;<br>
(b)	identifying a feature of said reference biomarker profile, wherein said feature is predictive or diagnostic of sepsis or one of the stages of sepsis;<br>
(c)	identifying a biomarker that corresponds with said feature; and<br>
(d)	isolating said biomarker.<br>
88.	A biomarker profile comprising at least two features that contribute to the<br>
classification of an individual as belonging to a reference population with an accuracy of at<br>
least about 60%, based on a comparison with the reference population, wherein the reference<br>
population is selected from the group consisting of a normal reference population, a SIRS-<br>
positive reference population, an infected/SIRS-negative reference population, a sepsis-<br>
positive reference population, a reference population at a stage in the progression of sepsis, a<br>
SIRS-positive reference population confirmed as having sepsis by conventional techniques<br><br>
after about 0-36 hours, a SIRS-positive reference population confirmed as having sepsis by conventional techniques after about 36 - 60 hours, and a SIRS-positive reference population confirmed as having sepsis by conventional techniques after about 60 - 84 hours.<br>
89.	A kit, comprising at least two biomarkers selected from the group consisting of the<br>
biomarkers listed in any one of TABLES 15-23 and 26 - 50.<br>
90.	A kit, comprising a set of antibodies or functional fragments thereof that specifically<br>
bind at least two biomarkers selected from the group consisting of the biomarkers listed in<br>
any one of TABLES 15 - 23 and 26 - 50.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTIwMC1jaGVucC0yMDA1IGFic3RyYWN0IGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">1200-chenp-2005 abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTIwMC1jaGVucC0yMDA1IGNsYWltcyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1200-chenp-2005 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTIwMC1jaGVucC0yMDA1IGRlc2NyaXB0aW9uIChjb21wbGV0ZSkgZHVwbGljYXRlIDIucGRm" target="_blank" style="word-wrap:break-word;">1200-chenp-2005 description (complete) duplicate 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTIwMC1jaGVucC0yMDA1IGRlc2NyaXB0aW9uIChjb21wbGV0ZSkgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">1200-chenp-2005 description (complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTIwMC1jaGVucC0yMDA1LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1200-chenp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTIwMC1jaGVucC0yMDA1LWFzc2lnbmVtZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">1200-chenp-2005-assignement.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTIwMC1jaGVucC0yMDA1LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1200-chenp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTIwMC1jaGVucC0yMDA1LWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1200-chenp-2005-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTIwMC1jaGVucC0yMDA1LWNvcnJlc3BvbmRuZWNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">1200-chenp-2005-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTIwMC1jaGVucC0yMDA1LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">1200-chenp-2005-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTIwMC1jaGVucC0yMDA1LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1200-chenp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTIwMC1jaGVucC0yMDA1LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">1200-chenp-2005-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTIwMC1jaGVucC0yMDA1LWZvcm0gMjYucGRm" target="_blank" style="word-wrap:break-word;">1200-chenp-2005-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTIwMC1jaGVucC0yMDA1LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">1200-chenp-2005-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTIwMC1jaGVucC0yMDA1LWZvcm0xOC5wZGY=" target="_blank" style="word-wrap:break-word;">1200-chenp-2005-form18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTIwMC1jaGVucC0yMDA1LXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1200-chenp-2005-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="223553-an-invitro-method-of-predicting-the-prognosis-of-a-biological-condition-in-animal-tissue.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="223555-an-unsaturated-sulfone-compound.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>223554</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1200/CHENP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>47/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Nov-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>12-Sep-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>10-Jun-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>BECTON, DICKINSON AND COMPANY</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1 BECTON DRIVE, FRANKLIN LAKES, NEW JERSEY 07417-1880</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>IVEY, RICHARD</td>
											<td>11 SHADY RIDGE COURT, PARKTON, MD 21120</td>
										</tr>
										<tr>
											<td>2</td>
											<td>GENTLE, THOMAS</td>
											<td>710 OAKWOOD DRIVE, RED LION, PA 17356,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MOORE, RICHARD</td>
											<td>5537 WOOL MILL ROAD, GLENVILLE, PA 17329,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>TOWNS, MICHAEL</td>
											<td>3 FERNS COURT, TIMONIUM, MD 21093,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>BACHUR, NICHOLAS</td>
											<td>750 MONKTON ROAD, MONKTON, MD 21111,</td>
										</tr>
										<tr>
											<td>6</td>
											<td>ROSENSTEIN, ROBERT</td>
											<td>4273 BRIGHT BAY WAY, ELLICOTT CITY, MD 21042,</td>
										</tr>
										<tr>
											<td>7</td>
											<td>NADEAU, JAMES</td>
											<td>10311 CROMWELL COURT, ELLICOTT CITY, MD 21042,</td>
										</tr>
										<tr>
											<td>8</td>
											<td>GOLDENBAUM, PAUL</td>
											<td>3931 BRITTANY LANE, HAMPSTEAD, MD 21074,</td>
										</tr>
										<tr>
											<td>9</td>
											<td>SHI, SONG</td>
											<td>1916 BILLY BARTON CIRCLE, REISTERSTOWN, MD 21136,</td>
										</tr>
										<tr>
											<td>10</td>
											<td>COPERTINO, DONALD</td>
											<td>710 HUNTER WAY, CATONSVILLE, MD 21228,</td>
										</tr>
										<tr>
											<td>11</td>
											<td>GARRETT, JAMES</td>
											<td>1002 LICTHFIELD ROAD, BALTIMORE, MD 21239,</td>
										</tr>
										<tr>
											<td>12</td>
											<td>TICE, GREGORY</td>
											<td>3 WESTON COURT, LUTHERVILLE, MD 21093,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>G01N</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US03/36019</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-11-12</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/511,644</td>
									<td>2003-10-17</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/425,322</td>
									<td>2002-11-12</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/223554-an-invitro-method-of-determining-the-status-of-sepsis-using-biomarket-profiles by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:28:42 GMT -->
</html>
