<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/223517-isolated-dna-sequence-capable-of-driving-expression-of-an-associated-nucleotide-sequence by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:36:03 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 223517:ISOLATED DNA SEQUENCE CAPABLE OF DRIVING EXPRESSION OF AN ASSOCIATED NUCLEOTIDE SEQUENCE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">ISOLATED DNA SEQUENCE CAPABLE OF DRIVING EXPRESSION OF AN ASSOCIATED NUCLEOTIDE SEQUENCE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Isolated DNA sequence capable of driving expression of an associated nucleotide sequence Isolated DNA sequence capable of driving expression of an associated nucleotide sequence, wherein said DNA sequence comprises: a) a nucleotide sequence selected from the group consisting of SEQ ID NO: 1, SEQ ID NO: 2, SEQ ID NO: 3, SEQ ID NO: 4, SEQ ID NO: 5 SEQ ID NO: 6, SEQ ID NO: 19 and SEQ ID NO: 20; or b) the complement of a DNA sequence which hybridizes under stringent conditions to any one of SEQ ID NO: 1, SEQ ID NO: 2, SEQ ID NO: 3, SEQ ID NO: 4, SEQ ID NO: 5 or SEQ ID NO: 6, said hybridisation being carried out at a temperature of 65°C in double strength citrate buffered saline containing 0.1% SDS followed by rinsing at the same temperature in a one tenth strength citrate buffered saline solution containing 0.1% SDS.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present invention relates to novel DNA sequences that function as promoters of transcription of associated nucleotide sequences in plants. More specifically, this invention relates to novel promoters that confer constitutive expression to an associated nucleotide sequence of interest <br>
In the field of agriculture there exists a continuous desire to modify plants according to one's needs. One way to accomplish this is by using modern genetic engineering techniques. For example, by introducing a gene of interest into a plant, the plant can be specifically modified to express a desirable phenotypic trait. For this, plants are transformed most commonly with a heterologous gene comprising a promoter region, a coding region ,and a termination region. When genetically engineering a heterologous gene for expression in plants, the selection of a promoter is a critical factor. While it may be desirable to express certain genes only in response to particular stimuli or confined to specific cells or tissues, other genes are more desirably erased constitutively, i.e. throughout the plant at all times and in  „QJ2Lans. In the past, the 35S promoter from Cauliflower Mosaic Virus 35S promoter) has been widely used for constitutive expression of heterologous genes in plants. There are, however, occasions where it is desirable to use alternative promoters. Therefore, it is a major objective of the present invention to provide such   for expression of a nucleotide sequence of interest in plants. The invention also provides recombinant DNA molecules, expression vectors and transgenic plants comprising the promoters of the present invention.<br>
The present invention thus provides:<br>
a DNA sequence capable’ of driving expression.of an associated nucleotide sequence, wherein said DNA sequence is obtainable from the genome of Cestrum yellow leaf curling virus (CmYLCV). Preferred is a DNA sequence which is obtainable from the CmYLCV full-length transcript promoter and comprises the nucleotide sequence depicted in SEQ ID NO:1. In particular, DNA sequences are provided, wherein<br>
•	said DNA sequence comprises the nucleotide sequence depicted in SEQ ID N0:2<br>
•	said DNA sequence comprises the nucleotide sequence depicted in SEQ ID N0:3<br><br>
•	said DNA sequence comprises the nucleotide sequence depicted in SEQ ID NO:4<br>
•	said DNA sequence comprises the nucleotide sequence depicted in SEQ ID NO:5<br>
•	said DNA sequence comprises the nucleotide sequence depicted in SEQ ID NO:6<br>
•	said DNA sequence hybridizes under stringent conditions to any one of SEQ ID N0:1, SEQ ID N0:2, SEQ ID NO:3, SEQ ID NO:4, SEQ ID NO:5 or SEQ ID NQ:6, in particular wherein the DNA sequence as mentioned hereinbefore comprises the nucleotide sequence depicted in SEQ ID NQ:19 or SEQ ID NO:20.<br>
The invention further provides DNA sequences comprising a consecutive stretch of at least 50 nt, preferably of between about 400 bases and about 650 bases, more preferably of between about 200 bases and about 400 bases and most preferably of about 350 bases in length of SEQ ID NO:1, wherein said DNA sequences are capable of driving expression of an associated nucleotide sequence.<br>
In a particular embodiment of the invention said consecutive stretch of at least 50 nt, preferably of between about 400 bases and about 650 bases, more preferably of between about 200 bases and about 400 bases and most preferably of about 350 bases in length has at least 75%, preferably 80%, more preferably 90% and most preferably 95% sequence identity with a corresponding consecutive stretch of at least 50 nt, preferably of between about 400 bases and about 650 bases, more preferably of between about 200 bases and about 400 bases and most preferably of about 350 bases in length of SEQ ID NQ:1. The invention further provides recombinant DNA molecules comprising a full-length transcript promoter region isolated from Cestrum yellow leaf curling virus. In addition, the invention provides recombinant DNA molecules and DNA expression cassettes comprising a DNA sequence according to the invention operably linked to a nucleotide sequence of interest. The invention also provides DNA expression vectors comprising said recombinant DNA and expression cassettes, respectively.<br>
In particular, recombinant DNA molecules and DNA expression cassettes are provided wherein the nucleotide sequence of interest comprises a coding sequence, particularly wherein<br>
•	the coding sequence encodes a desirable phenotypic trait<br>
•	the coding sequence encodes a selectable or screenable marker gene<br>
•	the coding sequence encodes a protein conferring antibiotic resistance, virus resistance, insect resistance, disease resistance, or resistance to other pests, herbicide tolerance, improved nutritional value, improved performance in an industrial process or altered reproductive capability<br><br>
•	the coding sequence encodes a protein which confers a positive selective advantage to cells that have been transformed with said coding sequence<br>
•	the coding sequence encodes a protein which confers a metabolic advantage to cells that have been transformed with said coding sequence consisting of being able to metabolize a compound, wherein said compound is mannose or xylose or a derivative or a precursor of these, or a substrate of the protein, or is capable of being metabolized by cells transformed with said coding sequence into such a substrate<br>
•	the coding sequence encodes an enzyme selected from the group of xyloisomerases, phosphomanno-isomerase, mannose-6-phosphate isomerase, mannose-1-phosphate isomerase, phosphomanno mutase, mannose epimerase, mannose or xylose phosphatase, mannose-6-phosphatase, mannose-1-phosphatase and mannose or xylose permease<br>
•	the coding sequence encodes a phosphomanno isomerase<br>
•	the coding region is non-translatable<br>
•	the non-translatable coding region is from a viral gene, in particular from TSWV, more particularly from the TSWV NP gene<br>
•	the coding sequence encodes commercially valuable enzymes or metabolites in the plant<br>
•	the coding sequence is in antisense orientation<br>
The invention also provides DNA expression vectors comprising a DNA sequence or a recombinant DNA molecule as mentioned hereinbefore. In a particular embodiment of the invention, said DNA expression vector is pNOV2819 or pNOV2819. Further are provided DNA expression vectors comprising a first DNA sequence according to the invention operably linked to a nucleotide sequence of interest, and a second ONA sequence according to the invention operably linked to a nucleotide sequence of interest. In a specific embodiment of the invention the DNA expression vectors as mentioned hereinbefore are capable of altering the expression of a viral genome.<br>
In an even more specific embodiment, the DNA expression vector as mentioned hereinbefore comprises a first DNA sequence capable of expressing in a cell a sense RNA fragment of said viral genome or portion thereof and a second DNA sequence capable of expressing in a cell an antisense RNA fragment of said viral genome or portion thereof, wherein said sense RNA fragment and said antisense RNA fragment are capable of forming a double-stranded RNA. Expression vectors as mentioned hereinbefore are provided wherein<br>
•	said virus is selected from the group consisting of tospoviruses, potyviruses,<br>
potexviruses, tobamoviruses, luteovlruses, cucumoviruses, bromoviruses,<br><br>
closteorviruses, tombusviruses and furoviruses<br>
•	said DNA sequences comprises a nucleotide sequence derived from a viral coat protein gene, a viral nucleocapsid protein gene, a viral replicase gene, or a viral movement protein gene or portions thereof<br>
•	said DNA sequence is derived from tomato spotted wilt virus (TSWV)<br>
•	said DNA is derived from a nucleocapsid protein gene<br>
The invention further provides<br>
host cells stably transformed with a DNA sequence, a recombinant DNA molecule or a DNA expression vector according to the invention. In particular, wherein<br>
•	the host cell is a bacterium<br>
•	the host cell is a plant cell<br>
•	the host cell is a plant cell selected from the group consisting of rice, maize, wheat,<br>
,    barley, rye, sweet potato, sweet corn, bean, pea, chicory, lettuce, cabbage, cauliflower, broccoli, turnip, radish, spinach, asparagus, onion, garlic, pepper, celery, squash, pumpkin, hemp, zucchini, apple, pear, quince, melon, plum, cherry, peach, nectarine, apricot, strawberry, grape, raspberry, blackberry, pineapple, avocado, papaya, mango, banana, soybean, tomato, sorghum, sugarcane, sugar-beet, sunflower, rapeseed, clover, tobacco, carrot, cotton, alfalfa, potato, eggplant, cucumber, Arabidopsis thaliana, and woody plants such as coniferous and deciduous trees, but particularly rice, maize, wheat, barley, cabbage, cauliflower, pepper, squash, melon, soybean, tomato, sugar-beet, sunflower or cotton, rice, maize, wheat, Sorghum bicolor, orchardgrass, sugar beet and soybean cells<br>
•	the host cell is a plant cell from a dicotyledonous plant<br>
•	the host cell is a plant cell from a dicotyledonous plant selected from the group consisting of soybean, cotton, tobacco, sugar beet and oilseed rape,<br>
•	the host cell is a plant cell from a monocotyledonous plant<br>
•	the host cell is a plant cell from a monocotyledonous plant selected from the group consisting of maize, wheat, sorghum, rye, oats, turf grass, rice, and barley.<br>
In addition, plants and the progeny thereof including seeds are provided stably transformed with a DNA sequence, a recombinant DNA molecule or a DNA expression vector according to the invention. In particular, wherein<br>
•	the plant is selected from the group consisting of rice, maize, wheat, barley, rye, sweet<br>
potato, sweet corn, bean, pea, chicory, lettuce, cabbage, cauliflower, broccoli, turnip,<br>
radish, spinach, asparagus, onion, garlic, pepper, celery, squash, pumpkin, hemp.<br><br>
zucchini, apple, pear, quince, melon, plum, cherry, peach, nectarine, apricot, strawberry, grape, raspberry, blackberry, pineapple, avocado, papaya, mango, banana, soybean, tomato, sorghum, sugarcane, sugar-beet, sunflower, rapeseed, clover, tobacco, carrot, cotton, alfalfa, potato, eggplant, cucumber, Arabidopsis thaliana, and woody plants such as coniferous and deciduous trees, but particularly rice, maize, wheat, barley, cabbage, cauliflower, pepper, squash, melon, soybean, tomato, sugar-beet, sunflower or cotton, rice, maize, wheat, Sorghum bicolor, orchardgrass, sugar beet and soybean. The present invention further discloses<br>
•	the use of the DNA sequence according to the invention to express a nucleotide sequence of interest<br>
•	a method of producing a DNA sequence according to the invention, wherein the DNA is produced by a polymerase chain reaction wherein at least one oligonucleotide used comprises a sequence of nucleotides which represents a consecutive stretch of 15, 18, 20, 22, 24 or more base pairs of SEQ ID N0:1.<br>
In order to ensure a clear and consistent understanding of the specification and the claims,<br>
the following definitions are provided:<br>
CmYLCV: Cestrum yellow leaf curling virus.<br>
DNA shuffling: DNA shuffling is a method to rapidly, easily and efficiently introduce<br>
rearrangements, preferably randomly, in a DNA molecule or to generate exchanges of DNA<br>
sequences between two or more DNA molecules, preferably randomly. The DNA molecule<br>
resulting from DNA shuffling is a shuffled DNA molecule that is a non-naturally occurring<br>
DNA molecule derived from at least one template DNA molecule.<br>
Expression: refers to the transcription and/or translation of an endogenous gene or a<br>
transgene in plants. In the case of antisense constructs, for example, expression may refer<br>
to the transcription of the antisense DNA only.<br>
Functionally equivalent sequence: refers to a DNA sequence which has promoter activity<br>
substantially similar to the CmYLCV full-length transcript promoter or parts thereof and<br>
which under stringent hybridizing conditions hybridizes with the said promoter sequences.<br>
Gene: refers to a coding sequence and associated regulatory sequence wherein the coding<br>
sequence is transcribed into RNA such as mRNA, rRNA, tRNA, snRNA, sense RNA or<br>
antisense RNA. Examples of regulatory sequences are promoter sequences, 5'- and 3'-<br>
untranslated sequences and termination sequences. Further elements that may be present<br>
are, for example, introns.<br><br>
Gene of interest: refers to any gene which, when transferred to a plant, confers upon the plant a desired characteristic such as antibiotic resistance, virus resistance, insect resistance, disease resistance, or resistance to other pests, herbicide tolerance, improved nutritional value, improved performance in an industrial process or altered reproductive capability. The "gene of interest" may also be one that is transferred to plants for the production of commercially valuable enzymes or metabolites in the plant. Heterologous as used herein means of different natural or of synthetic origin. For example, if a host cell is transformed with a nucleic acid sequence that does not occur in the untransformed host cell, that nucleic'acid sequence is said to be heterologous with respect to the host cell. The transforming nucleic acid may comprise a heterologous promoter, heterologous coding sequence, or heterologous termination sequence. Alternatively, the transforming nucleic acid may be completely heterologous or may comprise any possible ^ combination of heterologous and endogenous nucleic acid sequences. Leader region: region in a gene between transcription start site and translation start site. Marker gene: refers to a gene encoding a selectable or screenable trait. Operablv linked to/associated with: a regulatory DNA sequence is said to be "operably linked to" or "associated with" a DNA sequence that codes for an RNA or a protein if the two sequences are situated such that the regulatory DNA sequence affects expression of the coding DNA sequence.<br>
Plant: refers to any plant, particularly to seed plants<br>
Plant cell: structural and physiological unit of the plant, comprising a protoplast and a cell wall. The plant ceil may be in form of an isolated single cell or a cultured cell, or as a part of higher organized unit such as, for example, a plant tissue, or a plant organ. Plant material: refers to leaves, stems, roots, flowers or flower parts, fruits, pollen, pollen tubes, ovules, embryo sacs, egg cells, zygotes, embryos, seeds, cuttings, cell or tissue cultures, or any other part or product of a plant<br>
Promoter: refers to a DNA sequence that initiates transcription of an associated DNA sequence. The promoter region may also include elements that act as regulators of gene expression such as activators, enhancers, and/or repressors and may include all or part of the 5' non-translated leader sequence.<br>
Recombinant DNA molecule: a combination of DNA sequences that are joined together using recombinant DNA technology.<br><br>
Recombinant DNA technology: procedures used to join together DNA sequences as described, for example, in Sambrook et al., 1989, Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press.<br>
Screenable marker gene: refers to a gene whose expression does not confer a selective advantage to a transformed cell, but whose expression makes the transformed cell phenotypically distinct from untransformed cells.<br>
Selectable marker gene: refers to a gene whose expression in a plant cell gives the cell a selective advantage. The selective advantage possessed by the cells transformed with the selectable marker gene may be due to their ability to grow in the presence of a negative selective agent, such as an antibiotic or a herbicide, compared to the growth of non-transformed cells. The selective advantage possessed by the transformed cells, compared to non-transformed cells, may also be due to their enhanced or novel capacity to utilize an added compound as a nutrient, growth factor or energy source. Selectable marker gene also refers to a gene or a combination of genes whose expression in a plant cell in the presence of the selective agent, compared to the absence of the selective agent, has a positive effect on the transformed plant cell and a negative effect on the un-transformed plant cell, for example with respect to growth, and thus gives the transformed plant cell a positive selective advantage.<br>
Sequence identity: the percentage of sequence identity is determined using computer programs that are based on dynamic programming algorithms. Computer programs that are preferred within the scope of the present invention include the BLAST (Basic Local Alignment Search Tool) search programs designed to explore all of the available sequence databases regardless of whether the query is protein or DNA. Version BLAST 2.0 (Gapped BLAST) of this search tool has been made publicly available on the Internet (currently http://www.ncbi.nlm.nih.gov/BLAST/). It uses a heuristic algorithm, which seeks local as opposed to global alignments and is therefore able to detect relationships among sequences, which share only isolated regions. The scores assigned in a BLAST search have a well-defined statistical interpretation. Said programs are preferably run with optional parameters set to the default values.<br>
Transformation: refers to the introduction of a nucleic acid into a cell. In particular, it refers to the stable integration of a DNA molecule into the genome of an organism of interest TSWV: tomato spotted wilt virus<br>
The present invention relates to DNA sequences obtainable from the genome of the<br><br>
Cestrum yellow leaf curling virus (CmYLCV). Preferred are DNA sequences which are obtainable from the CmYLCV full-length transcript promoter which are capable of driving expression of an associated nucleotide sequence of interest. DNA sequences comprising functional and/or structural equivalents thereof are also embraced by the invention. The present invention thus relates to DNA sequences that function as promoters of transcription of associated nucleotide sequences. The promoter region may also include elements that act as regulators of gene expression such as activators, enhancers, and/or repressors and may include the 5' non-translated leader sequence of the transcribed mRNA. In a preferred embodiment of the invention, said DNA sequence confers constitutive expression to an associated nucleotide sequence. Constitutive expression means that the nucleotide sequence of interest is expressed at all times and in most tissues and organs. When tested in association with a GUS or CAT reporter gene in transient expression assays, the DNA sequence according to the invention confers high level of expression of the GUS or CAT reporter gene. Thus, the DNA sequence according to the invention is useful for high level expression of an associated nucleotide sequence of interest, which preferably is a coding sequence. It is known to the skilled artisan that the associated coding sequence of interest can be expressed in sense or in antisense orientation. Further, the coding sequence of interest may be of heterologous or homologous origin with respect to . the plant to be transformed. In case of a homologous coding sequence, the DNA sequence according to the invention is useful for ectopic expression of said sequence. In one particular embodiment of the invention expression of the coding sequence of interest under control of a DNA sequence according to the invention suppresses its own expression and that of the original copy of the gene by a process called cosuppression.<br>
The promoters of the present invention can be obtained from Cestrum yellow leaf curling virus (CmYLCV) genomic DNA, which can be isolated, for example, from infected Cestrum parqui plants. CmYLCV-infected Cestrum parqui plants are identified by their leaf mosaic and crinkled leaf malformation. This DNA is then sequenced and analyzed. Sequence comparison with sequences in the database shows that the genomic organization of the CmYLCV is related to the one of other members of the Caulimovirus family. CmYLCV contains '8 open reading frames. Sequence comparison of the CmYLCV gene products shows that gene I product is involved in intracellular movements: gene A encodes a small protein of unknown function, gene B codes for the aphid transmission factor; gene C encodes a virion associated protein; the gene IV is the major capsid protein; gene V codes<br><br>
for a reverse transcriptase and the gene VI activates in.-trans the translation of viral genes on full-length transcript. Of particular interest is the so-called CmYLCV full-length transcript promoter.<br>
It is known to the skilled artisan that various regions of the CmYLCV full-length transcript promoter can be used according to the invention. One particular embodiment of the invention is the CmYLCV full-length transcript promoter region shown in SEQ ID N0:1, called the CmpD promoter. This sequence contains 350 bp of CmYLCV full-length transcript promoter and 320 bp of the CmYLCV full-length transcript 5' non-translated leader sequence. This sequence is obtained by sequencing the CmYLCV genomic DNA. The promoter and leader sequences respectively, are identified by comparison to sequences in the database. A variant of SEQ ID N0:1, called CmpE, wherein the CmYLCV full-length transcript 5' non-translated leader sequence is 318 bp instead of 320 bp in length, is shown inSEQIDNO:19.<br>
One preferred embodiment of the invention is the DNA sequence depicted in SEQ ID N0:2, called the CmpC promoter. The CmpC promoter is a fragment of the sequence shown in SEQ ID NO:1 and contains 346 bp of CmYLCV full-length transcript promoter, corresponding to base 5 to 350 of SEQ ID NO:1. This DNA sequence is obtainable by PCR with genomic DNA from Oestrum yellow leaf curling virus or from Oestrum yellow leaf curling virus-infected Oestrum parqui plants using forward primer Cmp1 (SEQ ID NO:13) and reverse primer CmpC2 (SEQ ID NO:14). The putative TATA-box of the CmpC promoter is located from base 308 to base 315 of SEQ ID N0:2. The CmpC promoter contains at least 3 putative enhancer regions. Enhancer region 1 having the nucleotide sequence CAAT is located from base 232 to base 235 of SEQ ID N0:2, and enhancer region 2 also having the nucleotide sequence CAAT is located from base 243 to base 246 of SEQ ID NO:2. Enhancer region 3 is located from base 246 to base 253 of SEQ ID NO:2 and has the nucleotide sequence TGACGTAA.<br>
Another preferred embodiment of the invention is the DNA shown in SEQ ID NQ:3, called the CmpS promoter. The CmpS promoter also is a fragment of the sequence shown in SEQ ID NO: 1 and contains the 346 bp of SEQ ID N0:2 plus 54 bp from the leader region of the CmYLCV full-length transcript promoter. The leader region is a nucleotide sequence preceding the coding region which is transcribed but not translated into protein. It is known to the skilled artisan that the leader region can contain regulatory elements with important functions in gene expression. The CmpS promoter is obtainable by PCR with genomic DNA from Oestrum yellow leaf curling virus or from Oestrum yellow leaf curling virus-infected<br><br>
Oestrum parqui plants using fonrt/ard primer Cmpi (SEQ ID NO: 13) and reverse primer CmpS2 (SEQ ID N0:15). The CmpS promoter contains at least 2 putative enhancer regions in the leader region. Enhancer region 4 (GAGAGA) is located at base 354 to base 359 of SEQ ID N0:3 and enhancer region 5 (GAGAGAGA) is located at base 368 to base 375 of SEQ ID N0:3.<br>
Yet another preferred embodiment of the invention is the sequences depicted in SEQ ID N0:4, called the CmpL promoter. As the preceding promoters, the CmpL promoter is a fragment of the sequence shown in SEQ ID NO: 1 and contains the 346 bp of SEQ ID N0:2 plus 288 bp of the CmYLCV full-length transcript 5' non-translated leader sequence. The CmpL promoter is obtainable by PCR with genomic DNA from Oestrum yellow leaf curling virus or from Oestrum yellow leaf curling virus-infected Oestrum parqui plants using forward primer Cmpi (SEQ ID N0:13) and reverse primer CmpL2 (SEQ ID NO:16). A variant of ,SEQ ID N0:4, called CmpF, wherein the CmYLCV full-length transcript 5' non-translated leader sequence is 286 bp instead of 288 bp in length, is shown in SEQ ID NO:20. It is known to the skilled artisan that the nucleotide sequences shown in SEQ ID NOs:1, 2, 3, 4, 19 and 20 can be extended at their 5-' and 3'-ends with homologous or heterologous nucleotide sequences. For example, the 5'-end of SEQ ID N0s:1, 2, 3, 4, 19 and 20 can be extended by all or part of the nucleotides shown in SEQ ID NO:6. SEQ ID NO:6 is naturally found upstream of SEQ ID NOs: 2, 3, 4 and 20 and contains part of ORF VI of CmYLCV (base 1 to base 100 of SEQ ID NO:6) as well as part of the CmYLCV full-length transcript promoter (base 101 to base 104 of SEQ ID NO:6). Likewise, the 3'-ends of SEQ ID NOs:2, 3, 4 and 20 can be extended by all or part of the nucleotides shown in SEQ ID N0:5, representing the nucleotide sequence naturally found at the 3'-end of SEQ ID N0s:4 and 20 and comprising part of the CmYLCV full-length transcript leader sequence.<br>
As described above, the DNA sequences of the invention can be obtained, for example, by PCR with genomic DNA from Oestrum yellow leaf curling virus or from Oestrum yellow leaf curling virus-infected Cesfrum pargu; plants. Alternatively, the DNA sequences of the invention can be obtained from any other virus of the Oaulimovirus family comprising homologues of the DNA sequence of the invention using sequence specific primers. It is apparent to the skilled artisan that, based on the nucleotide sequences shown in SEQ ID N0:1 to SEQ ID NO:6 and SEQ ID Nos 19 and 20, any primer combination of interest can be chosen to PCR amplify DNA fragments of various lengths that can be used according to the invention. The invention thus includes fragments derived from the<br><br>
CmYLCV full-length transcript promoters that function according to the invention, i.e. are capable of conferring expression of an associated nucleotide sequence. This can be tested by generating such promoter fragments, fusing them to a selectable or screenable marker gene and assaying the fusion constructs for retention of promoter activity. Such assays are within the ordinary skill of the person skilled in the art. Preferred DNA fragments of the invention are of at least about 50 bases, preferably of between about 400 bases and about 650 bases, more preferably of between about 200 bases and about 400 bases and most preferably of about 350 bases in length.<br>
It is also clear to the skilled artisan that mutations, insertions, deletions and/or substitutions of one or more nucleotides can be introduced into the DNA sequences of SEQ ID NOs:1 , 2, 3, 4, 5 and 6 or longer or shorter fragments derived from the sequence information thereof using methods known in the art. In addition, an unmodified or modified nucleotide ^sequence of the present invention can be varied by shuffling the sequence of the invention. To test for a function of variant DNA sequences according to the invention, the sequence of interest is operably linked to a selectable or screenable marker gene and expression of the marker  gene   is  tested   in   transient   expression   assays  with   protoplasts  or  in   stably transformed plants. It is known to the skilled artisan that DNA sequences capable of driving expression of an associated nucleotide sequence are build in a modular way. Accordingly, expression levels from shorter DNA fragments may be different than the one from the longest fragment and may be different from each other. For example, deletion of a down-regulating upstream element will lead to an increase in the expression levels of the associated nucleotide sequence while deletion of an up-regulating element will decrease the expression levels of the associated nucleotide sequence. It is also known to the skilled artisan that deletion of development-specific or a tissue-specific element will lead to a temporally or spatially altered expression profile of the associated nucleotide sequence. Embraced by the present invention are also functional equivalents of the promoters of the present invention, i.e. nucleotide sequences that hybridize under stringent conditions to any one of SEQ ID N0:1, SEQ ID N0:2, SEQ ID NO:3, SEQ ID N0:4, SEQ ID NO:5 or SEQ ID NO:6, such as, for example, the sequences shown in SEQ ID N0:19 and SEQ ID NO:20. A stringent hybridization is performed at a temperature of 65°C, preferably 60°C and most preferably 55°C in double strength (2X) citrate buffered saline (SSC) containing 0.1% SDS followed by rinsing of the support at the same temperature but with a buffer having a reduced SSC concentration. Such reduced concentration buffers are typically one tenth strength SSC (0.1X SSC) containing 0.1% SDS, preferably 0.2X SSC containing 0.1% SSC<br><br>
and most preferably half strength SSC (0.5X SSC) containing 0.1% SDS. In fact, functional equivalents to all or part of the CmYLCV full-length transcript promoter from other organisms can be found by hybridizing any one of SEQ ID NO:1, SEQ ID NQ:2, SEQ ID N0:3, SEQ ID N0:4, SEQ ID N0:5, SEQ ID N0:6, SEQ ID N0:19 or SEQ ID NO:20 with genomic DNA isolated from an organism of interest, particularly from another Caulimovirus. The skilled artisan knows how to proceed to find such sequences as there are many ways known in the art to identify homologous sequences in other organisms. Such newly identified DNA molecules then can be sequenced and the sequence can be compared to any one of SEQ ID N0:1, SEQ ID NQ:2, SEQ ID N0:3, SEQ ID N0:4, SEQ ID N0:5, SEQ ID NO:6, SEQ ID NO:19 or SEQ ID NO:20 and tested for promoter activity. Within the scope of the present invention are DNA molecules having at least 75%, preferably 80%, more preferably 90%, and most preferably 95% sequence identity to the nucleotide sequence of any one of SEQ ID Nos:1, 2, 3, 4 or 6 over a length of at least 50 nucleotides. The percentage of sequence identity is determined using computer programs that are based on dynamic programming algorithms. Computer programs that are preferred within the scope of the present invention include the BLAST (Basic Local Alignment Search Tool) search programs designed to explore all of the available sequence databases regardless of whether the query is protein or DNA. Version BLAST 2.0 (Gapped BLAST) of this search tool has been made publicly available on the Internet (currently<br>
http://www.ncbi.nlm.nih.gov/BLAST/). It uses a heuristic algorithm which seeks local as opposed to global alignments and is therefore able to detect relationships among sequences which share only isolated regions. The scores assigned in a BLAST search have a well-defined statistical interpretation. Said programs are preferably run with optional parameters set to the default values.<br>
It is another object of the present invention to provide recombinant DNA molecules comprising a DNA sequence according to the invention operably linked to a nucleotide sequence of interest. The nucleotide sequence of interest can, for example, code for a ribosomal RNA, an antisense RNA or any other type of RNA that is not translated into protein. In another preferred embodiment of the invention the nucleotide sequence of interest is translated into a protein product. The nucleotide sequence associated with the promoter sequence may be of homologous or heterologous origin with respect to the plant to be transformed. The sequence may also be entirely or partially synthetic. Regardless of the origin, the associated DNA sequence will be expressed in the transformed plant in accordance with the expression properties of the promoter to which it is linked. In case of<br><br>
homologous nucleotide sequences associated with the promoter sequence, the promoter according to the invention is useful for ectopic expressiori of said homologous sequences. Ectopic expression means that the nucleotide sequence associated with the promoter sequence is expressed in tissues and organs and/or at times where said sequence may not be expressed under control of its own promoter. In one particular embodiment of the invention, expression of nucleotide sequence associated with the promoter sequence suppresses its own expression and that of the original copy of the gene by a process called cosuppression.<br>
In a preferred embodiment of the invention the associated nucleotide sequence may code for a protein that is desired to be expressed throughout the plant at all times and in most tissues and organs. Such nucleotide sequences preferably encode proteins conferring a desirable phenotypic trait to the plant transformed therewith. Examples are nucleotide sequences encoding proteins conferring antibiotic resistance, virus resistance, insect .resistance, disease resistance, or resistance to other pests, herbicide tolerance, improved nutritional value, improved performance in an industrial process or altered reproductive capability. The associated nucleotide sequence may also be one that is transferred to plants for the production of commercially valuable enzymes or metabolites in the plant. Embraced by the present invention are also selectable or screenable marker genes, i.e. genes comprising a DNA sequence of the invention operably linked to a coding region encoding a selectable or screenable trait.<br>
Examples of selectable or screenable marker genes are described below. For certain target species, different antibiotic or herbicide selection markers may be preferred. Selection markers used routinely in transformation include the npt\\ gene which confers resistance to kanamycin, paromomycin, geneticin and related antibiotics (Vieira and Messing, 1982, Gene 19: 259-268; Bevan et a!., 1983, Nature 304:184-187) the bacterial aadA gene (Goidschmidt-Clermont, 1991, Nucl. Acids Res. 19: 4083-4089), encoding aminoglycoside 3'-adenylyltransferase and conferring resistance to streptomycin or spectinomycin, the hph gene which confers resistance to the antibiotic hygromycin (Blochlinger and Diggelmann, 1984, Mol. Cell. Biol. 4: 2929-2931), and the dhfrgene, which confers resistance to methotrexate (Bourouis and Jarry, 1983, EMBO J. 2: 1099-1104). Other markers to be used include a phosphinothricin acetyltransferase gene, which confers resistance to the herbicide phosphinothricin (White et al.,-1990, Nucl. Acids Res. 18: 1062; Spencer et al. 1990, Theor. Appl. Genet. 79: 625-631), a mutant EPSP synthase gene encoding glyphosate resistance (Hinchee et al., 1988, Bio/Technology 6: 915-922), a mutant acetolactate synthase (ALS)<br><br>
gene which confers imidazolione or sulfonylurea resistance (Lee et al., 1988, EMBO J. 7: 1241-1248), a mutant psbA gene conferring resistance to atrazine (Smeda et al., 1993, Plant Physiol. 103: 911-917), or a mutant protoporphyrinogen oxidase gene as described in EP 0 769 059. Identification of transformed cells may also be accomplished through expression of screenable marker genes such as genes coding for chloramphenicol acetyl transferase (CAT), p-glucuronidase (GUS), luciferase (LUC), and green fluorescent protein (GFP) or any other protein that confers a phenotypically distinct trait to the transformed cell. Selection markers resulting in positive selection, such as a phosphomannose isomerase gene, as described in patent application WO 93/05163, are also used. Other genes to be used for positive selection are described in WO 94/20627 and encode xyioisomerases and phosphomanno-isomerases   such   as   mannose-6-phosphate   isomerase   and   mannose-1-phosphate isomerase; phosphomanno mutase; mannose epimerases such as those which .convert carbohydrates to mannose or mannose to carbohydrates such as glucose or galactose; phosphatases   such   as   mannose  or  xylose   phosphatase,   mannose-6-phosphatase  and mannose-1-phosphatase, and permeases which are involved in the transport of mannose, or a derivative, or a precursor thereof into the cell. The agent which reduces the toxicity of the compound to the cells is typically a glucose derivative such as methyl-3-O-glucose or phloridzin. Transformed cells are identified without damaging or killing the non-transformed cells in the population and without co-introduction of antibiotic or herbicide  resistance genes. As described in WO 93/05163, in addition to the fact that the need for antibiotic or herbicide resistance genes is eliminated, it has been shown that the positive selection method Is often far more efficient than traditional negative selection.<br>
Therefore, the promoters of the present invention are preferably operably linked to a nucleotide sequence which encodes a protein which comprises a region which: (a) encodes a protein which is involved in the metabolism of the compound, and/or (b) regulates the activity of the gene encoding, the protein, wherein the compound is mannose or xylose or a derivative or a precursor of these, or a substrate of the protein, or is capable of being metabolized by the transformed cells into such a substrate. Said nucleotide sequence may encode a mannose-6-phosphate isomerase and the compound may be mannose. Alternatively, the nucleotide sequence may encode a phosphosugar-isomerase, a phosphosugar-mutase such as phosphomanno mutase, a phosphatase such as mannose-6-phosphatase, a sugar-epimerase, a sugar-permease, a phosphosugar-permease or a xylose isomerase, and the compound may be mannose, mannose-6-phosphate, D-mannoseamine or xylose. In a preferred embodiment of the invention, the promoters of the invention are operably linked to<br><br>
the coding region of the E. co//manA gene (Joersbo and Okkels, 1996, Plant Cell Reports 16:219-221, Negrotto et al., 2000, Plant Cell Reports 19:798-803), encoding a phosphomannose isomerase (PMI), and a Nos (nopaline synthetase) terminator obtained from Agrobacterium tumefaciens T-DNA (Depicker et al., J. Mol. Appl. Genet. 1 (6), 561-573 (1982)). The skilled artisan knows that the Nos terminator may be substituted for by any other terminator that functions in plants. The promoter of the invention may be the CmpD, CmpC, CmpS, CmpL, CmpB, CmpA, CmpE or CmpF promoter, depicted in SEQ ID Nos 1 to 6 and 19 to 20, or any promoter derived therefrom. In an even more preferred embodiment of the invention, the promoter is the CmpS promoter depicted in SE!Q ID N0:3.<br>
The promoters of the present invention may also be used to provide resistance or tolerance to viruses by operatively linking the promoters of the invention with coding regions of genes from , viruses to be controlled. For virus control of negative strand viruses such as tomato spotted wilt virus (TSWV), the viral nucleocapsid protein (NP) and the movement protein (MP) gene sequences can be used, and for viruses belonging to the Alpha-like supergroup of viruses such as TMV and CMV, RNA-dependent RNA-polymerase (RdRp) and movement protein (MP) gene sequences can be used. For viruses belonging to the Picorna-like supergroup of viruses, including potyviruses, any viral sequence may be linked to the promoters of the invention. Finally, for all other viruses, including DNA viruses, RNA-dependent RNA-polymerase (RdRp) or replicase-associated gene sequences may be used. Such constructs for virus control can be constructed as exemplified in Example 8 and Example 9 of WO 00/68374. It is within the ordinary skill of the person skilled in the art to replace the promoters disclosed in WO 00/68374 with the promoters of the present invention. Following the teaching of WO 00/68374 the skilled artisan also knows how to prepare constructs comprising the promoters of the invention op'erably linked to a viral sequences as mentioned hereinbefore to confer resistance or tolerance to said virus.<br>
Instead of expressing double stranded viral RNA in transgenic plants, as disclosed in WO 00/68374, the viral RNA can also be expressed as a non-translatable RNA plus-sense viral RNA molecule as described, for example, in US 558302 or in Examples 12 and 13 of the present invention.<br>
It is a further objective of the invention to provide recombinant expression vectors comprising a DNA sequence of the invention fused to an associated nucleotide sequence of<br><br>
interest. In these vectors, foreign DNA can be inserted into a polylinker region such that these exogenous sequences can be expressed in a suited host cell which may be, for example, of bacterial or plant origin. For example, the plasmid pBI101 derived from the Agrobacterium tumefaciens binary vector pBINI 9 allows cloning and testing of promoters using beta-glucuronidase (GUS) expression signal (Jefferson et al, 1987, EMBO J 6: 3901-3907). The size of the vector is 12.2 kb. It has a low-copy RK2 origin of replication and confers kanamycine resistance in both bacteria and plants. There are numerous other expression vectors known to the person skilled in the art that can be used according to the invention.<br>
It is a further objective of the present invention to provide transgenic plants comprising the recombinant DNA sequences of the invention. The invention thus relates to plant cells, to plants derived from such cells, to plant material, to the progeny and to seeds derived from ^such plants, and to agricultural products with improved properties obtained by any one of the transformation methods described below. Plants transformed in accordance with the present invention may be monocots or dicots and include, but are not limited to, rice, maize, wheat, barley, rye, sweet potato, sweet corn, bean, pea, chicory, lettuce, cabbage, cauliflower, broccoli, turnip, radish, spinach, asparagus, onion, garlic, pepper, celery, squash, pumpkin, hemp, zucchini, apple, pear, quince, melon, plum, cherry, peach, nectarine, apricot, strawberry, grape, raspberry, blackberry, pineapple, avocado, papaya, mango, banana, soybean, tomato, sorghum, sugarcane, sugar-beet, sunflower, rapeseed, clover, tobacco, carrot, cotton, alfalfa, potato, eggplant, cucumber, Arabidopsis thaliana, and woody plants such as coniferous and deciduous trees. Preferred plants to be transformed are rice, maize, wheat, barley, cabbage, cauliflower, pepper, squash, melon, soybean, tomato, sugar-beet, sunflower or cotton, but especially rice, maize, wheat. Sorghum bicolor, orchardgrass, sugar beet and soybean. The recombinant DNA sequences of the invention can'be introduced into the plant cell by a number of well-known methods. Those skilled in the art will appreciate that the choice of such method might depend on the type of plant which is targeted for transformation, i.e., monocot or dicot. Suitable methods of transforming plant cells include microinjection (Crossway et al., 1986, Bio Techniques 4:320-334), electroporation (Riggs and Bates, 1986, Proc. Natl. Acad. Sci., USA 83:5602-5606), Agrobacterium-mediated transformation (Hinchee et al., 1988, Bio/Technology 6:915-922; EP 0 853 675), direct gene transfer (Paszkowski et al., 1984, EMBO J. 3:2717-2722), and ballistic particle acceleration using devices available from Agracetus, Inc., Madison, Wisconsin and Dupont, Inc., Wilmington, Delaware (see, for example, US Patent<br><br>
No. 4,945,050 and McCabe et al., 1988, Bio/Technology'6:923-926). The cells to be transformed may be differentiated leaf cells, embryogenic cells, or any other type of cell. In the direct transformation of protoplasts, the uptake of exogenous genetic material into a protoplast may be enhanced by the use of a chemical agent or an electric field. The exogenous material may then be integrated into the nuclear genome. The previous work is conducted in dicot tobacco plants, which resulted in the foreign DNA being incorporated and transferred to progeny plants (Paszkowski et al., 1984, EMBO J. 3:2712-2722; Potrykus et al., 1985, Mol. Gen. Genet 199:169-177). Monocot protoplasts, for example, of Triticum monococcum, Lolium multiflorum (Italian rye grass), maize, and Black Mexican sweet corn, are transformed by this procedure. An additional preferred embodiment is the protoplast transformation method for maize as disclosed in EP 0 292 435, as well as in EP 0 846 771. For maize transformation also see Koziel et al., 1993, Bio/Technology 11:194-200. ,Transformation of rice can be carried out by direct gene transfer techniques utilizing protoplasts or particle bombardment. Protoplast-mediated transformation is described for Japonica-types and Indica-types (Zhang et al., 1988, Plant Cell Rep., 7:379-384; Shimamoto et al., 1989, Nature 338:274-276; Datta et al., 1990, Bio/Technology 8:736-740). Both types described above are also routinely transformable using particle bombardment (Christou et al., 1991, Bio/Technology 9:957-962). Patent application No. EP 0 332 581 describes techniques for the generation, transformation and regeneration of Pooideae protoplasts. These techniques allow the transformation of all Pooideae plants including Dactylis and wheat. Furthermore, wheat transformation is described in patent application No. EP 0 674 715; and by Weeks et al., 1993 (Plant Physiol. 102:1077-1084). The thus-constructed plant expression vector can, for example, be introduced into the calli of rice according to the conventional plant transformation method, and the differentiation of roots and leaves is induced therefrom, and then, can be transferred to a flowerpot for cultivation, thereby obtaining the transformed rice.<br>
The plants resulting from transformation with the DNA sequences or vectors of the present invention will express a nucleotide sequence of interest throughout the plant and in most tissues and organs.<br>
The genetic properties engineered into the transgenic plants described above are passed on by sexual reproduction or vegetative growth and can thus be maintained and propagated in progeny plants. Generally said maintenance and propagation make use of known agricultural methods developed to fit specific purposes such as tilling, sowing or harvesting. Specialized processes such as hydroponics or greenhouse technologies can also be<br><br>
applied. Use of the advantageous genetic properties of the transgenic plants according to the invention can further be made in plant breeding that aims at the development of plants with improved properties such as tolerance of pests, herbicides, or stress, improved nutritional value, increased yield, or improved structure causing less loss from lodging or shattering. The various breeding steps are characterized by well-defined human intervention such as selecting the lines to be crossed, directing pollination of the parental lines, or selecting appropriate progeny plants. Depending on the desired properties different breeding measures are taken. The relevant techniques are well known in the art and include but are not limited to hybridization, inbreeding, backcross breeding, multiline breeding, variety blend, interspecific hybridization, aneuploid techniques, etc. Hybridization techniques also include the sterilization of plants to yield male or female sterile plants by mechanical, chemical or biochemical means. Cross pollination of a male sterile plant with ^pollen of a different line assures that the genome of the male sterile but female fertile plant will uniformly obtain properties of both parental lines. Thus, the transgenic plants according to the invention can be used for the breeding of improved plant lines that for example increase the effectiveness of conventional methods such as herbicide or pesticide treatment or allow to dispense with said methods due to their modified genetic properties. Alternatively new crops with improved stress tolerance can be obtained that, due to their optimized genetic "equipment", yield harvested product of better quality than products that were not able to tolerate comparable adverse developmental conditions.<br>
It is another objective of the present invention to provide DNA sequences that can be used to express a nucleotide sequence of interest in a desired organism. This organism can be a bacterium, a plant orany other organism of interest.<br>
Furthermore, the disclosure of SEQ ID N0s:1 to 6 enables a person skilled in the art to design oligonucleotides for polymerase chain reactions which attempt to amplify DNA fragments from templates comprising a sequence of nucleotides characterized by any continuous sequence of 15 and preferably 20 to 30 or more base pairs in SEQ ID Nos:1, 2, 3, 4, 5 or 6. Said nucleotides comprise a sequence of nucleotides which represents 15 and preferably 20 to 30 or more base pairs of SEQ ID Nos:1, 2, 3, 4, 5 or 6. Polymerase chain reactions performed using at least one such oligonucleotide and their amplification products constitute another embodiment of the present invention.<br>
BRIEF DESCRIPTION OF THE SEQUENCES IN THE SEQUENCE LISTING<br>
SEQ ID N0:1	CmpD<br><br>
SEQ ID N0:2	CmpC<br>
SEQ ID NO:3	CmpS<br>
SEQ ID N0:4	CmpL<br>
SEQ ID N0:5	CmpB<br>
SEQ ID NO:6	CmpA<br>
SEQ ID NO:7	Si forward primer<br>
SEQ ID N0:8	82 reverse primer<br>
SEQ ID NQ:9	GUS forward primer<br>
SEQIDNO:10	GUS reverse primer<br>
SEQ ID N0;11	CAT fonvard primer<br>
SEQ ID NO: 12	CAT reverse primer<br>
SEQ ID NO: 13	Cmpi forward primer<br>
^SEQ1DN0:14	CmpC2 reverse primer<br>
SEQIDN0:15	CmpS2 reverse primer<br>
SEQIDNO:16	CmpL2 reverse primer<br>
SEQIDN0:17	PA fonward primer<br>
SEQ ID NO:18	PA reverse primer<br>
SEQIDNO:19	CmpE<br>
SEQ ID NO:20	CmpF<br>
SEQ ID NQ:21	NOS terminator fonward primer<br>
SEQ ID NO:22	NOS terminator reverse primer<br>
SEQ ID NO:23	TSWV NP gene forward primer<br>
SEQ ID NO:24	TSWV NP gene reverse primer<br>
SEQ ID NO:25	CmYLCV promoter fonward primer<br>
SEQ ID NO:26	CmYLCV promoter reverse primer<br>
SEQ ID NO:27	AGLINK multicloning site<br>
SEQ ID NO:28	BIGLINK multicloning site<br>
SEQ ID NO:29	Cmpf-B primer<br>
SEQ ID NO:30	CmpS-B primer<br>
SEQ ID N0:31	pNOV2804; binary vector, contains a promoterless PMI-nos<br>
terminator expression cassette<br>
SEQ ID NO:32	pNOV3604; vector for transformation by biolistics, contains £<br>
promoterless PMI-nos terminator expression cassette<br>
SEQ ID NO:33	CmpMr reverse primer<br><br>
SEQ ID NO:34	CmpMf fonvard primer<br>
SEQ ID NO:35	CmpMrs reverse primer<br>
SEQ ID NO:36	CmpMfs fon^/ard primer<br>
SEQ ID NO:37	synGFPI; plant optimized GFP gene with intron<br>
SEQ ID NO:38	GIG; GUS gene with intron<br>
SEQ ID NO:39	Cmp-C reverse primer<br>
SEQ ID NO:40	CmpS-synGFPI-nos expression cassette<br>
SEQ ID N0:41	CmpS-GIG-nos expression cassette<br>
SEQ ID NO;42	CmpC-synGFPI-nos expression cassette<br>
SEQ ID NO:43	CmpC-GIG-nos expression cassette<br>
SEQ ID NO:44	pNOV2117 vector<br>
SEQ ID NO:45	pNOV4200 vector<br>
SEQ ID NO:46	ZmUbi-GFP-35S term expression cassette<br>
SEQ ID NO:47	ZmUbi-GIG-nos expression cassette<br>
SEQ ID NO:48	Ubq3(At)-synGFPl-nos expression cassette<br>
SEQ ID NO:49	Ubq3(At)-GIG-nos expression cassette<br>
EXAMPLES<br>
Standard recombinant DNA and molecular cloning techniques used here are well known in the art and are described, for example, by Sambrook et al., 1989, "Molecular Cloning", Cold Spring Harbor, Cold Spring Harbor Laboratory Press, NY and by Ausubel et al. ,1994, "Current protocols in molecular biology", John Wiley and Sons, New York.<br>
Example 1: Virus cloning<br>
1. DNA extraction	Virus genomic DNA is extracted from Cestrum yellow leaf curling<br>
virus-infected Cestrum parqui plants. Five grams of infected leaves are homogenized in 30 ml of grinding buffer (0.2 M Tris pH 7.0, 0.02 M EDTA, 2 M Urea) for 60 sec at maximum speed in a Brinkman Poiytron homogenizer with PT10 probe and gently shaken at 4°C overnight with Triton X-100 (2% final concentration). Virus is purified from crude homogenate by low-speed centrifugation (10,000 rpm for 20 min in a Sorvall SS-34 rotor) and from the obtained supernatant by high-speed centrifugation (27,000 rpm for 2 hr in a Beckman SW-28 rotor) through a sucrose cushion (3 ml of 15% sucrose). The pellet-containing virus is then resuspended in 0.1 M Tris, pH 7.4, 2.5 mM MgCb. DNase I (Sigma)<br><br>
and RNase A (Sigma) are added to a concentration of 10 mg/ml each. After 30 min at 37°C, the reaction is stopped with the addition of EDTA to 10 mM. Virus DNA is isolated from CmYLCV particles by treating with protease K (E. Merck, 0.5 mg/ml final concentration) in the presence of 1% SDS at 65°C for 15 min. The viral DNA is then purified by phenol extraction and by ethanol precipitation. The virus DNA contained in the final ethanol precipitate is dissolved in water.<br>
2. DNA amplification and cloning Hundred nanograms of the obtained DNA are used as a template for PCR amplification in a 50 pi reaction volume containing 10 pM of each primer, 25 mM each dNTP, 5 pi of pfu reaction buffer (Stratagene) and 2.5 units of pfu turbo DNA polymerase (Stratagene). A S1 forward primer (gaccacaaacatcagaag, SEQ ID NO:7) and a S2 reverse primer (caaacttattgggtaatc, SEQ ID NO:8) are used for the PCR reaction. ^Cycling parameters for PCR are: 1x (94°C for 1 min); 30x (94°C for 1 min, 50°C for 1 min, 72°C for 1 min); 1x (72°C for ^0 min) Each single DNA fragment is cloned in pPCR-Script™ Amp SK(+) plasmid (Stratagene) according to the manufacturer's instructions.<br>
Example 2: DNA Sequencing<br>
Sequencing of the DNA virus clones is carried out using the automated ABI PRISM 377 DNA sequencer (Perkin Elmer) and ABI PRISM dRhodamine terminator cycle sequencing kit (Perkin Elmer) according to the manufacturer's instructions. The described S1 primer and S2 primers (Example 1) are used for the sequencing reactions as well as the universal M13-20 and Reverse primers.<br>
Example 3: Construction of plasmids for transient expression<br>
All the PCR reactions are carried out with Pfu polymerase (Stratagene) as described in Example 1.<br>
1. Reporter genes amplification The beta-glucuronidase (GUS) and the chloramphenicol acetyltransferase (CAT) reporter genes are amplified. For GUS gene amplification GUS fonfl/ard (cagggtaccactaaaatcacc, SEQ ID NO:9) and GUS reverse (aggggatccccaattcccc, SEQ ID NO:10) oligonucleotides are used at the annealing temperature of 60°C. CAT fonrt/ard (aggggtaccatcgatatggag, SEQ ID N0:11) and Cat reverse (ttaggatccgccctgccac, SEQ ID N0:12) oligonucleotides are annealed at 62°C. The fonward primers are designed to contain a Kpn\ recognition site and the reverse ones a BamH\ site.<br><br>
2. CmYLCV promoter amplification	Three different promoter fragments are selected.<br>
CmpC (SEQ ID N0:2), CmpS (SEQ ID NO:3), and CmpL (SEQ ID NO:4). Cmpi forward primer (cttctagacaaagtggcagac, SEQ ID NO:13) is used for all the three amplifications at the annealing temperature of 52°C. It is modified (shown in bold) from the original sequence to contain a Xba\ recognition site. The CmpC2 reverse primer (ttggtaccttaacaatgaggg, SEQ ID N0:14) is used for CmpC fragment amplification and the CmpS2 reverse primer (ctacttctaggtaccttgctc, SEQ ID N0:15) is used for the CmpS fragment. Both of them are modified to contain a Kpn\ recognition site. The CmpL2 reverse primer (ttggtaccttaacaatgaggg, SEQ ID N0:16) is used for CmpL fragment amplification and it is modified to contain a Cla\ restriction site.<br>
,3. Polyadenylation signal amplification The polyadenylation signal fragments is amplified from the cauliflower mosaic virus (CaMV) infectious clone (Franck et al.. Cell 21 (1), 285-294, 1980). PA (polyadenylation signal amplification) fon^/ard (ggggatccccagtctctctc, SEQ ID NO:17) and PA reverse (gtgaattcgagctcggta, SEQ ID N0:18) oligonucleotides are used for PCR and annealed at 60°C. The forward primer is designed to contain a BamHl recognition site and the reverse one an EcoR\ site.<br>
4. Produced constructs<br>
•	pCmpCG (CmpC promoter fragment + GUS gene + polyA)<br>
•	pCmpSG (CmpS promoter fragment + GUS gene + polyA)<br>
•	pCmpCC (CmpC promoter fragment + CAT gene + polyA)<br>
•	pCmpSC (CmpS promoter fragment + CAT gene + polyA)<br>
•	pCmpLC (CmpL promoter fragment + CAT gene + polyA)<br>
The cassettes containing the promoter fragment, the reporter genes and the polyadenylation signal are inserted in a pUC19 vector (Stratagene) restricted with Xba\ and EcoR.<br>
5.Constructs used for comparison<br>
•	pCapG (Cap promoter fragment + GUS gene + polyA)<br>
•	pCapSG (Caps promoter fragment + GUS gene -i- polyA)<br>
•	pCapC (Cap promoter fragment + CAT gene + polyA)<br><br>
-box (base 7175 to base 7486 of the \/00141). These positions correspond<br>
Promoter fragments contained in these constructs £ re obtained from Cal\/1V (Franck et al., Cell 21 (1), 285-294, 1980, see GenBank accession number V00141). GUS, CAT and polyA fragments are identical to those used for the CmYL[3V constructs. Cap corresponds to the fragment from base -227 to base + 33 from TAT/-box (base 7175 to base 7438 of the CaMV genome, see GenBank accession number \ 00141) and CapS corresponds to the fragment from base -227 to base + 82 from TATfi-CaMV genome, see GenBank accession number<br><br>
approximately to those chosen for the CmYLCV fragments.<br>
Example 4: Transient expression experiments<br>
1. Suspension cultures and protoplast preparation<br>
Orychophragmus violaceus. Suspension cultures a|e maintained in 40 ml of MS medium '(Murashige and Skoog, Physiol Plant 15, 474-497, sucrose, and 0.1 mg/ml 2,4D. Protoplasts are isolalted from 4- to 5-day-oid-cultures. Cell walls are digested at 26°C for 1 hr in 0.1% pectolyase Y23 (Seishin Pharmaceutical Co., Japan), 1% cellulase Onozuka RIO (Yakult Honsha Co., Japan), 0.4 M D-mannitol, and 0.1% MES, pH 5.5. Protoplasts are filtered through<br>
electroporation (EP) solution (10 mM HEPES, 150 rjiM NaCI, 5 mM CaCb, 0.2 M mannitol, pH7.1).<br>
at 26°C in a solution of 0.5% driselase 10000), 3.85 mM CaClz, 6 mg/l NAA,<br>
Nicotians plumbaginifolia. Plants are maintained axdnically on RPf\/l2 medium (Blonstein et al., Mol Gen Genet 211, 252-259, 1988) plus 7g1 bacto agar, pH 5.6.. For protoplasts preparation, leaves are cut and incubated overnight (Fluka), 0.25 mM PVP 10 (polyvinylpyrrolidone MW<br>
2 mg/l BAP, and 0.5 M sucrose, pH 5.7. Protoplasts are filtered through a 100 pm sieve.<br><br>
Sucrose solution (0.6 M sucrose, 0.1% MES, and<br><br>
15 mM CaCl2, pH 5.7) is added to the<br><br>
protoplast suspension for the first purification step, <ind the suspension is overlaid with w5></ind>
solution (150 mM NaCI, 125 mM CaCb, 5 mM KCI, 6 mM glucose; Menczel et al., Theor<br>
AppI Genet 59, 191-195, 1981). Protoplasts are thon washed once with W5 solution and<br>
finally with EP solution.<br>
Oryza  sativa.   Protoplasts  are  prepared  from  a   norphogenic  rice  suspension  culture<br>
established from O. sativa cv. Nipponbare as described (Datta et al., 1990, Bio/Technology<br>
8:736-740).<br><br>
2. Transient expression experiment Transfection by electroporation of 2x10^ Orychophragmus violaceus protoplasts in 0.66 ml EP buffer is carried out by discharging a 960 |JF capacitor through a distance of 4 mm of protoplast suspension. The capacitor is loaded at the 450 Volts. Electroporation is performed in the presence of 5-10 ^ig of plasmid DNA, then protoplasts are cultivated 16 to 24 hours at 25°C. Transfection of 2x10^ Nicotiana plumbaginifolia protoplast in 0.3 ml suspension is carried out in the presence of 0.3 ml PEG (40% polyetyieneglycole 6000) and 5-10 pg of plasmid DNA. Protoplasts are cultivated in 0.4 ml K3 medium (Godall et al., Methods Enzymol 181, 148-161, 1990) for 16 to 24 hours at 25°C and added with 10 nil W5 buffer before harvesting.<br>
Protein extracts are prepared by at least three cycles of freezing and thawing, and clarified by centrifugation.<br>
, Example 5: CAT and GUS assays<br>
For detection of CAT gene expression double antibody sandwich (DAS)-ELISA is carried out using a CAT ELISA kit (Boehringer) according to the manufacturer's instructions. CAT activity is measured with a Dynex MRXII apparatus. Samples for GUS assay are diluted in GUS buffer (0.05 M NaP04, pH 7, 0.01 M EDTA, 0.1% Triton-X-100, 0.1% Sarkosyl). The reaction is carried out in the presence of an equal amount of GUS-reaction buffer (100 ml/I lOx GUS buffer, 200 mg/l BSA, 705 mg/l 4-methylumbelliferyl-glucuronide) at 37°C and stopped with 2 M Ammediol. The activity is measured in a Titertek Fluoroskan II. Both CAT and GUS assay results are normalized to a standard clone value. Results obtained from ten different experiments show that the various CmYLCV promoter fragments function as highly active promoters.<br>
N. plumbaginifolia protoplast transient expression GUS reporter gene<br><br>
CmpCG	100<br>
CmpSG	86,9   +/-20%<br>
CapG	9      +/-20%<br>
CapSG	38,9   +/-15%<br><br>
CAT reporter gene<br><br>
CmpCC	100<br>
CmpSC	78      /-20%<br>
CapSC	89,5   +/-15%<br>
CmpLC	9       +/-20%<br>
O.violaceus protoplast transient expression<br>
GUS reporter gene<br><br>
CmpCG	100	<br>
CmpSG	84,6	+/-15%<br>
'CapG	9,6	+/-15%<br>
CapSG	40,7	+/-15%<br>
CAT reporter	gene	<br>
CmpCC	100	<br>
CmpSC	255	+/-20%<br>
CapSC	546	+/-15%<br>
CmpLC	10	+/-20%<br>
O. sativa protoplast transient expression GUS reporter gene<br><br>
CmpCG	100<br>
CmpSG	84,6   V-157o<br>
CapG	9,6   V-15%<br>
CapSG	40,7   V-15%<br><br>
Example 6: Preparation of Solutions and Media for Plant Regeneration and Transformation<br>
Culture media GM, CIM and SIM are the media described by Valvekens et al. (1988, Proc. Natl. Acad. Sci. USA. 85: 5536-5540).<br>
Culture medium GM contains the mineral salts of Murashige and Skoog (1962, Physiol. Plant. 15:473-497), 1.0 mg/l thiamine (stock 1 mg/ml), 0.5 mg/l pyridoxine HCI (stock 1 mg/ml), 0.5 mg/l nicotinic acid (stock 1 mg/ml), 0.5 g/l 2-(N-morpholino)ethanesulfonic acid (MES), 10 g/l sucrose, 8 g/l agar, with the pH adjusted to 5.8 with IN KOH. CIM contains the mineral salts and vitamins of B5 medium (Gamborg et al., 1968, Exp. Cell Res. 50:151-158), 0.5 g/l 2-(N-morpholino)ethanesulfonic acid (MES), 20 g/l glucose, 0.5 mg/l 2,4-dichlorophenoxyacetic acid (2,4-D) (stock 10 mg/mi in DMSO), 0.05 mg/l kinetin (stock 5 mg/ml in DMSO), pH 5.8. Solid CIM medium contains 8 g/l agar. SIM contains the mineral , salts and vitamins of 85 medium (Gamborg et al., 1968, supra), 0.5 g/l 2-(N-morpholino)ethanesulfonic acid (MES), 20 g/l glucose, 5 mg/l N6-(2-isopentenyl)adenine (2iP) (stock 20 mg/ml in DMSO), 0.15 mg/l indole-3-acetic-acid (lAA) (stock 1.5 mg/ml in DMSO), 8 g/l agar, pH 5.8. SIM V750 K100 is SIM medium supplemented with 750 mg/l vancomycin and 100 mg/l kanamycin. SIM V500 K100 is SIM medium supplemented with 500 mg/l vancomycin and 100 mg/l kanamycin. GM K50 is GM medium supplemented with 50 mg/i kanamycin.<br>
The culture media are all sterilized by autoclaving (20 min, 121°C). Vitamins are dissolved in water and added to media before autoclaving. Hormones are dissolved in dimethyl sulfoxide (DMSO). Antibiotics are dissolved in water and sterilized by filtration (0.22 pm). Hormones and antibiotics are added after autoclaving and cooling of the media to 65 C. In all cases 9-cm Petri dishes (Falcon, 3003) are used, except for GM and GM K50 which are usually poured into 15-cm Petri dishes (Falcon, 3025). Plates with solid media are dried before usage in laminar flow to remove condensate.<br>
Example 7: Arabidopsis strain and growth conditions<br>
Arabidopsis thaliana seeds ecotype Columbia (Col-0) wild type are purchased from Lehle Seeds, USA (1102 South Industrial Blvd. Suite D, Round Rock TX 78681, USA). Plants are grown at 22°C 16/8 hour light/dark cycle in pots in the mixture of 4 parts sand, 4 parts garden soil and 1 part agrilit.<br>
Example 8: Agrobacterium strain and culture<br><br>
Vector plasmids are introduced into recipient Agrobactenum tumefac/ens strain LBA4404 (Clontech) by triparental mating according to the protocol described by Walkerpeach and Velten ("Agrobacterium-mediated gene transfer to plant cells: Cointegrate and binary vector systems", in: Plant Molecular Biology Manual, 81: 1-19, 1994. Eds.: S.B. Gelvin, R.A., Schilperoort, Kluvers Acad. Publishers.). Mobilizing strain used is E. co//HB101 harboring conjugation plasmid pRK2013 (Ditta et al., 1980, Broad host range DNA cloning system from Gram-negative bacteria. Construction of gene bank of Rhizobium meliloti. Proc. Natl. Acad. Sci. USA 77: 7347-7351). Agrobacteria used for root transformation are grown in LB medium (Sambrook et al., 1989, "Molecular Cloning", Cold Spring Harbor, Cold Spring Harbor Laboratory Press, NY) without antibiotics at 28°C and 200 rpm.<br>
Example 9: Seed sterilization<br>
. Seeds are placed in 70% EtOH/0.05% Tween 20 for 1 minute in a 2 ml Eppendorf tube. 70% EtOH/0.05% Tween 20 is removed with a pipette and replaced with 5% NaOCI/0.05% Tween 20 for 15 minutes. Seeds are shaken regularly. The solution is removed in sterile conditions and the seeds are washed in sterile, distilled water 3 times for 10 minutes each. After the last wash seeds are keep in 0.5 - 1 ml water. Seeds can be used immediately or stored at 4°C for two - three weeks. Sterilized seeds (20-30) are transferred with forceps on GM medium in 15-cm Petri dishes. Seedlings are grown in vertically placed plates in a growth chamber (22°C; 16/8 hour light/dark cycle).<br>
Example 10: Transformation of root explants of Arabidopsis thaliana<br>
Roots of three-week-old seedlings are used in the transformation procedure. Roots should not be green or brown. Green parts of seedlings are removed with scalpel and forceps. Remaining roots are collected and approximately 5 entire root systems are placed per plate with solid CIM medium. Roots are pressed gently onto the surface of the plate to ensure full contact with the medium, but they should not be dipped into the agar. Roots are incubated for three days in a growth chamber (22°C; 16/8 hour light/dark cycle). Roots are then transferred to a sterile Petri dish with filter paper wetted with liquid CIM medium and cut with a scalpel in 0.5-1 cm pieces. Root explants are then transferred to a 50 ml sterile Falcon tube containing 10 ml of liquid CIM medium. To this, 0.5 ml of an overnight Agrobacterium culture (OD 0.6-1) is added and incubate for 1-2 minutes while shaking gently. The liquid is poured out of the tube through sterile metal screens (50 mesh, Sigma, S-0895), which are<br><br>
kept with forceps. Roots usually remain on the wall of the tube close to its edge. Then the root explants are transferred to a sterile Petri dish with filter paper and briefly blotted dry to remove excess of liquid. Root explants are put onto plates with solid GIM medium and incubated in a growth chamber for 2 days under dim light (1.5-2 klux). Slight traces of overgrowth with Agrobacterium should be visible after the period of cocultivation. Root explants are then transferred to sterile 50 ml Falcon tubes with 20 ml of liquid CIM medium, supplemented with 1000 mg/l vancomycin. The Falcon tubes are then gently vortexed to remove the Agrobacteria. The liquid is poured out of the tube as described above and the explants are briefly blotted dry on filter paper. Explants are then transferred to plates containing SIM V750 K100 medium. Roots should be in a close contact with the medium. The explants are incubated in a growth chamber in normal conditions for one week and then transferred to SIM V500 K100 medium and incubated for an additional week. Then the ^ amount of vancomycin is reduced to 250 mg/l. First shoots should appear at the end of the third week of cultivation on SIM media. Shoots are excised when 0.3-0.5-cm long, any residual callus is removed, and the shoots are transferred to 15-Gm plates containing GM K50 medium. Max. 3 shoots are placed per plate. To get more shoots, the remaining root explants can be transferred to fresh SIM plates supplemented with 125 mg/l vancomycin and 100 mg/J- kanamycin for additional two weeks. Rooted shoots can be transferred to soil to allow seed set. Shoots that do not root are transferred to Magenta jars (one per jar) containing GM medium to produce seeds in vitro.<br>
Seeds from individual transgenic plants are germinated on GM K50 medium in growth chamber for 2 weeks. Phenotypically normal kanamycin resistant seedlings, which form green true leaves and branched root system, are selected for further analyses.<br>
Example 11: Histochemical 13-glucuronidase (GUS) assay<br>
in vitro grown seedlings or plants grown in soil are used in GUS assays. Either whole seedlings or dissected organs are dipped into GUS staining solution. GUS staining solution contains 1 mM 5-bromo-4-chloro-3-indolyl glucuronide (X-Gluc, Duchefa, 20 mM stock in DMSO), 100 mM Na-phosphate buffer pH 7.0, 10 mM EDTA pH 8.0, and 0.1% Triton XI00. Tissue samples are incubated at 37°C for 1-16 hours. If necessary samples can be cleared with several washes of 70% EtOH to remove chlorophyll.<br>
Example 12: Construction of tobacco transformation vector pZU627<br>
All the PCR reactions are carried out with Platinum Pfx DNA Polymerase (Life<br><br>
Technologies) according to manufacturer's instructions. ■<br>
1 ■ NOS terminator amplification and cloning<br>
The NOS terminator is amplified from pBIN19 (Bevan etal 1984) using the NOS terminator forward primer (SEQ ID N0:21) and NOS terminator reverse primer (SEQ ID NO:22). The fon/vard primer is modified to contain a PstI site and the reverse primer is modified to contain a Hindlll site. The NOS terminator amplification product is cloned as PstI / Hindlll fragment in same sites of pBluescript (Stratagene) resulting in plasmid pZU400A.<br>
2.	Amplification of the tomato spotted wilt virus nucleocapsid protein (TSWV NP) gene<br>
amplification and cloning<br>
In order to obtain a non-translatable TSWV NP gene, modified TSWV NP gene primers are ,used to amplify the TSWV NP gene. The TSWV NP gene fonward primer introduces a BamHI, a SphI, a start and a stop codon (SEQ ID NO:23). The reverse primer is modified to introduce a PstI site (SEQ ID NO:24). The amplification product is cloned as BamHI / PstI fragment into the BamHI / PstI site of pZU400A upstream of and in same direction as the NOS terminator. In the resulting clone pZU400b the PstI site between the TSWV NP gene and the NOS terminator is removed using a T4 DNA polymerase treatment (Life Technologies) according to manufacturer's instructions. From the resulting clone denoted pZU400C, the TSWV NP gene and NOS terminator are cloned as a SphI / Hindlll fragment into the SphI / Hindlll site of shuttle vector pZO1560 resulting in plasmid pZU400. pZO1560 is a pBluescript derivative in which the original multicloning site is replaced by the AGLINK multicloning site (SEQ ID NO:27).<br>
3.	CmYLCV Promoter amplification and cloning<br>
The CmYLCV viral promoter is amplified using CmYLCV promoter fonward primer (SEQ ID NO:25) and CmYLCV promoter reverse primer (SEQ ID NO:26). The forward primer is modified to contain a SstI site and the reverse primer is modified to contain a PstI site. The amplified CmYLCV viral promoter is cloned as SstI / PstI fragment, in same direction as and upstream of the TSWV-N gene, into the SstI / PstI site of pZU400. This results in clone pZU625 that contains a viral gene cassette that produces a non-translatable TSWV NP RNA.<br>
4.	Cloning of viral gene cassette to pVictorHJNK<br><br>
The viral gene cassette is cloned as AscI / Pad fragment from pZU625 into the AscI / Pad site of pVictorHiNK vector pZU494. This results in the plant transformation vector pZU627. pVictorHiNK is a binary vector comprising an origin of replication (ORI) derived from Pseudomonas aeruginosa plasmid pVSI which is known to be highly stable in A. tumefaciens (Woh et al., 1984. Plasmid 11:206-220; Itoh and Haas, 1985. Gene 36;27-36). The pVSI ORI is only functional in Agrobacterium and can be mobilized by the helper plasmid pRK2013 from E.coli into A. tumefaciens by means of a triparental mating procedure (Ditta etal., 1980. Proc. Natl. Acad. Sci. USA 77:7347-7351). This binary vector also comprises a ColEI origin of replication which is functional in E. coli and derived from pUC19 (Yannisch-Perron et al., 1985. Gene 33:103-119). For maintenance in E.coli and A. tumefaciens this vector contains a bacterial resistance gene to spectomycin and streptomycin encoded by a 0.93 kb gene from transposon Tn7 .(Fling et al., 1985. Nucl. Acid Res. 13:7095) which functions as selection marker for maintenance of the vector in E. coli and A. tumefaciens. The gene is fused to the tac promoter for efficient bacterial expression (Amman et al., 1983. Gene 25: 167-178). The right and left T-DNA border fragments of 1.9 kb and 0.9 kb, respectively that comprise the 24 bp border repeats, are derived from the Ti-plasmid of the nopaline type A. tumefaciens sXrains pTil37 (Yadev et al., 1982. Proc. Natl. Acad. Sd. USA. 79:6322-6326). Subsequently the T-DNA region between the borders is modified by deleting the Ml 3 derived sequences and, to improve its cloning versatility, the BIGLINK multidoning site (SEQ ID NO:28) is introduced yielding pVictorHiNK.<br>
PVictorHiNK contains an NPTII gene cassette for selection of transformants during the plant transformation process. This gene cassette contains a NOS promoter, the NPTII gene and the NOS terminator, obtained from A. tumefaciens.<br>
Example 13: Plant transformation and screening for resistance<br>
1 ■ Transformation of binary vectors to plant material<br>
Methods to transfer binary vectors to plant material are well established and known to a person skilled in the art. Variations in procedures exist due to for instance differences in used Agrobacterium strains, different sources of explant material, differences in regeneration systems depending on as well the cultivar as the plant species used. Binary vector pZU627 is used in plant transformation experiments according to the following procedures. PZU627 is transferred by tri-parental mating to an acceptor Agrobacterium<br><br>
tumefaciens strain, followed by Southern blot analysis of the ex-conjugants for verification of proper transfer of the construct to the acceptor strain, inoculation and co-cultivation of axenic explant material with the recombinant Agrobacterium tumefaciens strain, selective killing of the Agrobacterium tumefaciens strain using the appropriate antibiotics, selection of transformed cells by growing on selective media containing kanamycine, transfer of plantlets to soil, assaying for the intactness of the integrated T-DNA by Southern blot analysis of isolated chromosomal DNA of the plant.<br>
2. Resistance of plants against TSWV infections<br>
Transformed plants are grown in the greenhouse under standard quarantine conditions in order to prevent any infection by pathogens. At a four-leaf stage the plants are infected with TSWV by mechanical inoculation. Tissue from plants systemically infected with TSWV is ,ground in 5 volumes of ice-cold inoculation buffer (10 mM phosphate buffer supplemented with 1% Na2S03) and rubbed in the presence of carborundum powder on the first two fully extended new leaves. The inoculated plants are monitored for symptom development during three weeks after inoculation. Plants containing pZU627 sequences do not show TSWV symptoms, whereas untransformed control plants show severe systemic TSWV symptoms within 7 days after inoculation. Resistant plants are self-pollinated and the seeds harvested. Progeny plants are analyzed for segregation of the inserted gene and subsequently re-screened for resistance against TSWV infection as described above.<br>
Example 14: Construction of CmpS-phosphomannose isomerase-nos constructs for plant transformation<br>
The CmpS promoter is PCR amplified using the Cmpf-B (cgc gga tec tgg cag aca aag tgg cag a; SEQ ID NO:29) and the CmpS-B (cgc gga tec tac ttc tag get act tg, SEQ ID NO:30) primers having flanking BamHI sites. The resulting PCR fragment is cloned into the BamHI site of pBluescript KS (+) to form pNOV4211.<br>
To create a binary vector for transformation via Agrobacterium tumefaciens, the CmpS promoter is excised from pNOV4211 using BamHI and inserted into BamHI-digested pNOV2804 (SEQ ID N0:31) upstream of the PMI gene, and the resulting vector called pNOV2819. pNOV2804 (SEQ ID N0:31) is a binary vector with VS1 origin of replication, a copy of the Agrobacterium y\rG gene in the backbone, and a promoterless Pfvll-nos terminator expression cassette between the left and right borders of T-DNA. PMI (phosphomannose isomerase) is the coding region of the E.coli manA gene (Joersbo and<br><br>
Okkels, 1996, Plant Cell Reports 16:219-221, Negrotto efal., 2000, Plant Cell Reports 19:798-803). The nos (nopaline synthase) terminator is obtained from Agrobacterium tumefaciens T-DNA (Depicker et al., 1982, J. Moi. Appl. Genet. 1 (6), 561-573. The phosphomannose isomerase coding region and the nos terminator are located at nt 290 to nt 1465 and nt 1516 to 1789 respectively, of pNOV2804 (SEQ ID NO:31). To create a vector for biolistic transformation, the CmpS promoter is excised from pNOV4211 using BamHI and Inserted into BamHI-digested pNOV3604 (SEQ ID NO:32) to form pNOV2820, thus creating a PMI expression cassette driven by the CmpS promoter. pNOV3604 is a vector for preparing biolistic fragments and is based on pUCI 9 with an ColEI origin of replication and a beta-lactamase gene conferring ampicillin-resistance. As pNOV2804, pNOV3604 also contains a promoterless PMI-nos terminator expression cassette (see above). The phosphomannose isomerase coding region and the nos ■terminator are located at nt 42 to nt 1217 and nt 1268 to 1541 respectively, of pNOV3604 (SEQ ID NO:32).<br>
Binary vector pNOV2819 is used for transformation via Agrobacterium tumefaciens and vector pNO\/2820 is used for biolistic transformation.<br>
Example 15: Construction of CmpC-GUS plasmids for stable expression in planta.<br>
1 .Construction of the binary vector<br>
The vector pCambia 1302 (Cambia, Canberra, Australia; Roberts, C.S., Rajagopal, S., Smith, L., Nguyen, T., Yang, W., Nugroho, S., Ravi, K.S. Cao, M.L., Vijayachandra, K., et al. A comprehensive set of modular vectors for advanced manipulation and efficient transformation of plants. Presented at the Rockefeller Foundation Meeting of the International Program on Rice Biotechnology, Malacca, Malaysia, September 15-19, 1997) is chosen and modified for the experiments: The CaMV 35S promoter in front of the GFP gene is removed by digestion at position 9788 with HindlH endonucfease and at position 1 with Ncol endonuclease and the vector is relegated at the two compatible ends. Digestion with Xhol endonuclease at positions 7613 and BstXI at position 9494 excise the hygromycin gene and the CaMV 35S promoter in front of it. The 35S promoter sequences are eliminated from the vector to exclude any risk of homology-mediated gene silencing . The bar gene driven by the 1' promoter (Mengiste et al., Plant J, 1997, 12(4): 945-948) is introduced in the EcoRI site at position 9737 as a selectable marker. The obtained vector is named pCamBar.<br><br>
2. Produced constructs<br>
The cassettes CmpCG and CapG described in Example 3 are inserted in the pCamBar vector between Xba\ site at position 9764 and EcoRI at position 9737 to obtain the pCamBarCmpCG and the pCamBarCapG plasmids, respectively. The two constructs are used to transform Agrobacterium tumefaciens cells.<br>
Example 16: Stable expression in Arabidopsis thaliana<br>
1.	Plant production and transfection '<br>
Arabidopsis tfialiana (Columbia 0) wt seeds are sown, cold-treated 3 days at 4 °C in the dark to synchronize germination and transferred to growth room (22°C/ 24h light). Germinated plants are infiltrated when they have produced a maximum number of ^unopened buds. The infiltration is carried out according to the protocol described by Clough and Bent, Plant J. 16: 735-743, 1987.<br>
2.	Selection of transgenic plants<br>
Seedlings obtained from T1-generation seeds are selected by spraying with the BASTA herbicide (Pluss-Staufer AG/SA) (150 mg/l) every five days for three times. Twenty pCamBarCmpCG and nine pCamBarCapG resistant plants are collected after selection. They are grown under the described condition to obtain the T2-generation seeds. These seeds undergo the same procedure as described above for the wt, the T2 seedlings are selected by BASTA spraying, and the resistant mutants analyzed for GUS-gene expression.<br>
3.	Histochemical GUS expression in stably transformed Arabidopsis plants Histochemical GUS staining is done as described in Example 11. The analysis is performed in 15 ml culture tubes by immersion of the transformed seedlings in the X-g(uc solution, application of 130 mBar pressure for 10 min and incubation at 37°C overnight. Results obtained from Arabidopsis plants transformed with pCamBarCmpCG indicate a constitutive expression of the GUS reporter gene driven by the CmpC promoter in all vegetative organs.<br>
4.	Quantitative enzymatic GUS assays in stably transformed Arabidopsis plants A. Sample preparation<br>
Four plants per transgenic line and one leaf per each of these plants are selected for the test. The four leaves are collected in the same eppendorf tube, frozen with liquid nitrogen<br><br>
and ground to fine powder with disposable grinders. This is diluted in 150 \i\ GUS buffer (O.OSfVI NaP04, pH7, 0.01 M EDTA, 0.1% Triton-X-100, 0.1% Sarkosyl), incubated 5 min at 37°C and spun in a microcentrifuge for 5 min at maximum speed. The supernatant is transferred to a new eppendorf tube and these samples are used for the enzymatic test. Protein in these plant extracts is estimated according to the Bradford method (Bradford, M.M. Anal. Biochem. 72: 248-254, 1976) using BSA as a standard. The presence of the promoter and GUS gene in the mutant lines is confirmed by PCR analysis using the extracted samples as a template.<br>
B. Fluorometric GUS assay<br>
The fluorometric GUS assay is done as described in Example 5. For detection of GUS gene expression samples are diluted in GUS buffer. The reaction is carried out in the presence of ,an equal amount of GUS-reaction buffer at 37°C and stopped with 2 M Ammediol. The activity is measured in a Titertek Fluoroskan II.<br>
Fifteen out of the twenty selected lines transformed with pCamBarCmpCG and eight out of nine lines transformed with pCamBarCapG show GUS enzymatic activity in leaves. The level of activity in several CmpCG transformed lines is comparable to CapG. However the results are variable for the different transgenic lines due to the difference in their genotypes (number of loci and number of transgene insertions per locus).<br>
Example 17: Construction of CmpC promoter variants<br>
All the PCR reactions are carried out with Pfu polymerase (Stratagene) as described in Example 1.<br>
1. DNA amplification and cloning   Two variants, pCmpMG and pCmpMsG, of the pCmpCG<br>
plasmid (Example 3) are produced. The former deletes the sequence "GTGGTCCC" in the<br>
CmpC promoter and the latter mutates it to the sequence "GTGCTCGC".<br>
To obtain pCmpMG three PCR products are amplified:<br>
CmpMI  using the Cmpi forward primer (see Example 3, SEQ ID NO:13), the CmpMr<br>
reverse primer (CCATCGTGGTATTTGGTATTG, SEQ ID NO:33) and the pCmpCG plasmid<br>
as a template.<br>
CmpM2 using the CmpMf forward primer (CAATACCAAATACCACGATGG; SEQ ID NO:34),<br>
the CmpC2 reverse primer (see Example 3; SEQ ID NQ:14) and the pCmpCG plasmid as a<br>
template.<br><br>
CmpM using the Cmpi forward primer, the CmpC2 reverse primer and an equimolar mix of CmpM1 and CmpM2 PCR products as a template.<br>
To obtain pCmpMsG three PCR product are amplified:<br>
CmpMsl using the Cmpi forward primer, the CmpMrs reverse primer<br>
(CGTGGTAGCGAGCACTTTGGT, SEQ ID NO:35) and the pCmpCG plasmid as a<br>
template.<br>
CmpMs2  using the  CmpMfs forward  primer (AAGTGCTCGCTACCACGATGG,  SEQ ID<br>
NO:36), the Cmp2 reverse primer and the pCmpCG plasmid as a template.<br>
CmpsM using the Cmpi forward primer, the Cmp2 reverse primer and an equimolar mix of<br>
CmpMsl and CmpMs2 PCR products as a template.<br>
^To obtain the pCmpMG and the pCmpMsG plasmids the original pCmpCG plasmid is restricted with Xba\ and BamH\ endonucleases to eliminate the CmpC fragment and to insert in place of it the CmpM and the CmpMs PCR products.<br>
Example 18: Transient expression experiments witli pCmplVI and pCmpiVIs constructs<br>
1.	Suspension cultures and protoplast preparation<br>
Suspension cultures and protoplasts are produced as described in Example 4.<br>
2.	GUS assays<br>
Transfection and protein extract preparation are carried out as described in the Example 4, and the GUS assay is carried out as described in Example 5.<br>
GUS assay results are obtained from the average of ten different experiments and normalized to a standard clone value. The deletion of the "GTGGTCCC" sequence in the CmpCG construct reduces expression of the GUS reporter gene to about 50% of the GUS reporter gene expression from CmpC in all protoplast species (see below). The effect induced by the "GTGCTCGC" mutation depends on the plant species. In O. violaceus and O. sativa protoplasts the level of GUS gene expression is decreased to 65.2% and 73.6%, respectively. In N. plumbaginifolia protoplasts the CmpMs promoter activity is similar to the CmpM promoter.<br>
N. plumbapinifolia<br><br>
CmpCG	100<br>
CmpMG	52        V-12%<br>
CmpMsG	53.7    V-16%<br>
CapG	28.3     V-14%<br>
O. violaceus<br><br>
CmpCG	100<br>
CmpMG	55.2        Vl2%<br>
CmpMsG	73.6        ^/-10%<br>
CapG	5           V-  1%<br>
O. sativa<br><br>
CmpCG	100<br>
CmpMG	59.6  ■ V-21%<br>
CmpMsG	65.2    V-20%<br>
CapG	42.5    V-16%<br>
ixample 19: Construction of plant transformation vectors containing S'-promoter ragments operably linked to GFP or GUS reporter genes<br>
. Promoter amplification and cloning<br>
^ chimeric gene is constructed that includes a DNA sequence from the CmYLCV full-length -anscript promoter (SEQ ID N0:1) fused to the plant optimized GFP with intron (synGFPI, ;EQ ID NO:37) or GUS reporter gene with intron (GIG; SEQ ID NO:38) sequence. The GIG lene contains the ST-LS1 intron from Solanum tuberosum at nt 385 to nt 576 of SEQ ID JO: 38 (obtained from Dr. Stanton Gelvin, Purdue University, and described in Jarasimhulu, et al 1996, Plant Cell, 8: 873-886.). SynGFPI is a plant optimized GFP gene ito which the ST-LS1 intron is introduced (nt 278 to nt 465 of SEQ ID NO:37). For the )romoters, the CmYLCV genomic DNA is used as a template for the polymerase chain<br><br>
reaction (PCR). Gene specific primers are designed to amplify the DNA sequence. A gene<br>
fragment corresponding to CmpC (SEQ ID N0:2) is isolated using the Cmpf-B fonvard<br>
primer     (SEQ     ID     NO:29)	in     combination     with     the     5'     to     3'     primer<br>
CGCGGATTGCTCCCTTAACAATGAGG (SEQ ID NO:39) as reverse primer. A gene<br>
fragment corresponding to CmpS (SEQ ID NO:3) is isolated using the Cmpf-B (SEQ ID<br>
NO:29) forward primer in combination with the CmpS-B reverse primer (SEQ ID NO:30). All<br>
three primers contain the BamHI restriction enzyme recognition site CGCGGA at their 5'<br>
end. All PCR reactions are carried out with Pfu polymerase according to manufacturers<br>
recommendations (Stratagene). A thermocycler (DNA Engine, [\yiJResearch, Inc. Waltham,<br>
MA USA) is used to amplify the promoter fragments using the following PCR conditions:<br>
[(94°C:10s):(94°C:10s/56°C:30s/72°C:1min)X20]:(72°C:1.5m)].	Following      restriction<br>
digestion with BamHI the promoter fragments are Iigated into pUC-based vectors with the GIG gene or synGFPI gene to create the promoter-reporter gene fusions operatively linked with a nos terminator.<br>
2. Produced promoter-reporter cassettes<br>
•	CmpS promoter fragment + synGFPI gene + nos (SEQ ID NO:40)<br>
•	• CmpS promoter fragment + GIG gene + nos (SEQ ID N0:41)<br>
•	CmpC promoter fragment + synGFPI gene + nos (SEQ ID NO:42)<br>
•	CmpC promoter fragment + GIG gene + nos (SEQ ID NQ:43)<br><br>
SEQ ID NO	Promoter	Gene (synGFPI or GIG)	terminator<br>
40	nt 1 to nt 402	nt411 font 1331	nt 1343 font 1618<br>
41	nt 1 to nt 405	nt 425 to nt 2425	nt 2479 to nt 2751<br>
42	nt 1 to nt 354	nt 380 font 1292	nt 1304 font 1577<br>
43	nt 1 tont354'	nt 399 to nt 2399	nt 2453 to nt 2725<br>
3. Subcloninq into Aqrobacterium binary vector<br>
The promoter-reporter cassettes (SEQ ID NO:40 to 43) containing the promoter fragment, the reporter gene and the nos terminator are inserted into binary vector pNOV2117 for maize and binary vector pNQV4200 for torriato transformation. pNOV2117 (SEQ ID NO:44) is a binary vector with VS1 origin of replication, a copy of the Agrobacterium virG gene in<br><br>
the backbone, and a Maize Ubiquitin promoter- PMI gene-nos terminator expression cassette between the left and right borders of T-DNA. PMI (phosphomannose isomerase) is the coding region of the E.coli manA gene (Joersbo and Okkels, 1996, Plant Cell Reports 16:219-221, Negrotto et al., 2000, Plant Cell Reports 19:798-803). The nos (nopaline synthase) terminator is obtained from Agrobacterium tumefaciens T-DNA (Depicker et al., 1982, J. Mol. Appl. Genet. 1 (6), 561-573). The maize ubiquitin promoter, the phosphomannose isomerase coding region and the nos terminator are located at nt 31 to nt 2012, nt 2109 to nt 3212 and nt 3273 to 3521 respectively, of pN0V2117 (SEQ ID NO:44). The reporter-promoter cassettes are inserted closest to the right border. The selectable marker expression cassette in the binary vectors is closest to the left border. For tomato transformation, pNOV4200 (SEQ ID NO:45) is used as the binary vector, which contains the VS1 origin of replication, a copy of the Agrobacterium v'nQ gene in the backbone, and a hygromycin selectable marker cassette between the left and right border sequences. The hygromycin selectable marker cassette comprises the Arabidopsis Ubiquitin 3 promoter (Ubq3(At), Callis et al., J. Biol. Chem. 265:12486-12493 (1990)) operably linked to the gene encoding hygromycin resistance (denoted here as "HYG", synthetic hygromycin B phosphotransferase gene from E.coli, Patent; JP 1986085192-A 1 30-APR-1986) and the nos terminator (Depicker et al., J. Mol. Appl. Genet. 1 (6), 561-573 (1982). The Arabidopsis ubiquitin promoter, HYG gene and nos terminator are located at nt 162 to nX 11494, n\ 1897 to nt 2939 and nt 2939 to nt 3236, respectively of pNOV4200 (SEQ ID NO:45). The reporter-promoter cassettes (SEQ ID NO:40 to 43) are inserted closest to the right border. The selectable marker expression cassette, in the binary vectors is closest to the left border.<br>
Shown below are the orientations of the selectable marker and promoter-reporter cassettes in the binary vector constructs.<br>
4. Binary Vector Constructs:<br>
•	NOV4215 (RB CmpS promoter fragment + synGFPI gene + nos - ZmUbi -i- PMI gene + nos LB)<br>
•	NOV4216 (RB nos -i- synGFPI gene + CmpS promoter fragment - ZmUbi + PMI gene+ nos LB)<br>
.   NOV4217 (RB CmpS promoter fragment + synGFPI gene + nos - Ubq3(At) + HYG gene + nos LB)<br><br>
.   NOV4220 (RB CmpS promoter fragment + GIG gene + nos - Ubq3(At) + HYG gene + nos LB)<br>
•	NOV4224 (RB CmpC promoter fragment + GIG gene + nos - ZmUbi + HYG gene + nos LB)<br>
•	NOV4226 (RB CmpC promoter fragment + synGFPI gene + nos - ZmUbi + PMI gene + nos LB)<br>
Additional cassettes are constructed with known promoters to be used for comparison. To this end, a promoter cassette, ZmUbi-GFP-35S term (SEQ ID NO:46), comprising the maize Ubiquitin promoter (Christensen et al. Plant Molec. Biol. 12: 619-632 (1989)) operatively linked with synGFP and the 35S terminator (35S term) is constructed and a promoter cassette comprising the maize ubiquitin promoter operatively linked with the GIG gene and Jhe nos terminator (NOV 3612, SEQ ID NO:47) is also constructed The ZmUbi promoter, GFP gene and 35S terminator are located at nt 1 to nt 2019, nt 2020 to nt 2751 and nt 2876 to nt 2949, respectively of SEQ ID NO:46. The ZmUbi promoter, GIG gene and nos terminator are located at nt 1 to nt 1982, nt 2015 to nt 4015 and nt 4069 to nt 4341, respectively of SEQ ID NO:47. The ZmUbi-GIG nos cassette (SEQ ID NO: 47) is cloned into a pUC-based vector. The ZmUbi- GFP-nos cassette (SEQ ID NO:46 ) is cloned into the pN0V2117 binary vector for maize transformation, as described above. Promoter cassettes comprising the Arabidopsis UbiquitinS promoter (Ubq3(At)) operatively linked to synGFPI and the nos terminator (SEQ ID NO:48) and the Arabidopsis Ubiquitin3 promoter operatively linked to GIG gene and the nos terminator'(SEQ ID NO:49) are constructed. The Ubiqutin3 promoter, synGFPI and the nos terminator are located at nt 1 to nt 1332, nt 1738 to nt 2658 and nt 2670 to nt 2943, respectively of SEQ ID NO: 48. The Ubiqutin3 promoter, GIG gene and the nos terminator are located at nt 1 to nt 1335, nt 1746 to nt 3746 and nt 3800 to nt 4072, respectively of SEQ ID NO: 49. As described above, the promoter cassettes are cloned into the pNOV4200 binary vector containing a hygromycin selectable marker cassette for tomato transformation. Shown below are the orientations of the selectable marker and promoter-reporter cassettes in the binary vector constructs.<br>
5. Binary vector constructs<br>
.   NOV2110 (RB ZmUbi Promoter -f- synGFP gene + 35S term - ZmUbi + PMI gene + nos LB)<br><br>
•   NOV4209 (RB Ubq3(At) promoter + synGFPI gene + hos - Ubq3(At) + HYG gene + nos<br>
LB) .   NOV4208 (RB Ubq3(At) promoter + GUS -Intron -GUS gene + nos - Ubq3(At) + HYG<br>
gene + nos LB)<br>
Example 20: Agrobacterium-medlaled transformation of maize<br>
Transformation of immature maize embryos is performed essentially as described in Negrotto et al., (2000) Plant Cell Reports 19: 798-803. For this example, all media constituents are as described in Negrotto et al., supra. However, various media constituents described in the literature may be substituted.<br>
1.	Transformation plasmids and selectable marker<br>
.The genes used for transformation are cloned into a vector suitable for maize transformation as described in Example 19. Vectors used contain the phosphomannose isomerase (PMI) gene (Negrotto et al. (2000) Plant Cell Reports 19: 798-803).<br>
2.	Preparation of Agrobacterium tumefaciens<br>
Agrobacterium strain LBA4404 (pSBI) containing the plant transformation plasmid is grown on YEP (yeast extract (5 g/L), peptone (lOg/L), NaCI (5g/L),15g/l agar, pH 6.8) solid medium for 2 to 4 days at 28°C. Approximately 0.8X 10^ Agrobacteria are suspended in LS-inf media supplemented with 100 |aM acetosyringone (As) (Negrotto et a/.,(2000) Plant Cell Rep 19: 798-803). Bacteria are pre-induced in this medium for 30-60 minutes.<br>
3.	Inoculation<br>
Immature embryos from A188 or other suitable maize genotypes are excised from 8 -- 12 day old ears into liquid LS-inf + 100 |j.M As. Embryos are rinsed once with fresh infection medium. Agrobacterium solution is then added and embryos are vortexed for 30 seconds and allowed to settle with the bacteria for 5 minutes. The embryos are then transferred scutellum side up to LSAs medium and cultured in the dark for two to three days. Subsequently, between 20 and 25 embryos per petri plate are transferred to LSDc medium supplemented with cefotaxime (250 mg/l) and silver nitrate (1.6 mg/1) and cultured in the dark for 28°C for 10 days.<br><br>
4. Selection of transformed cells and regeneration of transformed plants Immature embryos producing embryogenic callus are transferred to LSD1M0.5S medium. The cultures are selected on this medium for 6 weeks with a subculture step at 3 weeks. Surviving calli are transferred either to LSD1M0.5S medium to be bulked-up or to Regl medium. Following culturing in the light (16 hour light/ 8 hour dark regiment), green tissues are then transferred to f^eg2 medium without growth regulators and incubated for 1-2 weeks. Plantiets are transferred to Magenta GA-7 boxes (Magenta Corp, Chicago 111.) containing Reg3 medium and grown in the light. Plants that are PCR positive for the promoter-reporter cassette are transferred to soil and grown in the greenhouse.<br>
Example 21: Stable transformation of tomato<br>
Transformation of tomato is performed essentially as described in Meissner et al., The Plant Journal 12: 1465-1472,1997.<br>
1.	Transformation plasmids and selectable marker<br>
The genes used for transformation are cloned into a binary vector suitable for tomato transformation as described in Example 19. Vectors contain the hygromycin resistance gene for selection.<br>
2.	Preparation of Agrobactenum tumefaciens<br>
Agrobactenum strain GV3101 containing the plant transformation piasmid is grown on LB (yeast extract (5 g/L), peptone (lOg/L), NaCI (5g/L), 15g/l agar, pH 6.8) liquid media for 2 days at 22°C. Approximately 0.8X 10^ Agrobacteria are suspended in KCMS inoculation media (MS salts (4.3g/l), Myo-lnositol, Thiamine (1.3 ug/ml), 2,4-D (0.2 ug/ml), Kinetin O.lug/ml and 3% Sucrose, pH 5.5) supplemented with 100 \iM acetosyringone. Bacteria are pre-induced in this medium for 60 minutes.<br>
3.	Seeds sterilization and germination.<br>
Seeds of Micro-Tom cultivar of tomato (Meissner et al., The Plant Journal 12: 1465-1472,1997) and ZTV-840 cultivars are soaked in 20% bleach solution with Tween for 20 min. Seeds are then washed three times with sterile water and plated on TSG media {V2 MS saltes, 1% Sucrose, 1% PhytoAgar, pH 5.8) in Magenta boxes (Magenta Corp, Chicago III) for germination. Cotyledons are excised and cotyledon petioles are cut into 5 mm segments<br><br>
7 to 10 days after germination.<br>
4.	Inoculation<br>
Seven to ten day old cotyledons and cotyledonal petioles segments are incubated for 24 hrs on tobacco BY-2 feeder cells plates (MS salts, Myo-inisitol 0.1 mg/ml, Casein Hydrolysate 0.2 mg/ml, Thiamine -HCI 1.3 |.ig/ml, 3% Sucrose, pH 5.5) prior to inoculation with Agrobacterium.<br>
Cotyledons are dipped into liquid KCMS Agrobacterium suspension for 5 minutes. Between 20 and 25 cotyledons are then transferred abaxial side upward on the same feeder cell plates and cultured in the dark for two to three days.<br>
5.	Selection of transformed cotyledons and petiols and regeneration of transformed plants<br>
'Cotyledons and petiole fragments are transferred onto selection media TSM-1 (4.3g/l MS<br>
salts, 1X 85 Vitamins, 2% Sucrose, 1% Glucose, 1 ^ig/ml Zeatin, 0.05 |ag/ml lAA, 5 |ig/ml Hygromicine and 250 j^g/ml Carbenicillin, pH 5.8) two to three days after inoculation with Agrobacterium in the light. Cotyledons and petiole segments producing embryogenic callus are transferred to TSM-2 medium (MS Salts 4.3 g/l, 1X MS Vitamins, 2% Sucrose, 1 ug/ml Zeatin, 5 j^g/ml Hygromicine and 250 ^g/ml Carbenicillin, pH 5.8). Surviving call! forming plantlets are transferred to Magenta GA-7 boxes (Magenta Corp, Chicago III.) containing TRM medium (4.3 g/l MS salt, IX MS Vitamins, 2% Sucrose pH 5.8) and grown in the light. After rootformation primary transformants are transferred to soil.<br>
Example 22: Green Fluorescent Protein (GFP) fluorometric assay<br>
For detection of synGFP gene expression the fluorometric assay is performed. Leaf discs of stably transformed maize and tomato are frozen on dry ice. Frozen tissue is throughly ground and mixed with 300 }.il of GFP Extraction Buffer (10 mM Tris-HCI (pH 7.5), 100 mM NaCI, 1 mM MgCI2, 10 mM DTT and 0.1% Sarcosyl). The extracts are separated from the leaf debris by centrifugation followed by transfer into 96 well black plate with clear bottom (Costar 3615). Relative fluoresence units (RFU) are measured by Spectra Fluor Plus Plate Reader at 465 nm excitation and 512 nm emission. GFP activity is normalized to total protein measured using BCA Assay (according to Pierce).<br>
Example 23: Expression analysis of stably transformed maize and tomato plants<br><br>
1 ■ Zea mays transgenic To plants<br>
GFP  expression  is   10  to  20  times  higher under the  control  of the  CmpS  promoter (NOV4215, NOV4216) compared to the ZmUbi promoter (NOV2110).   These results are obtained from leaf samples of 42 independent To lines representing a total of 93 To plants. 2. GFP Elisa assay<br>
A quantitative sandwich immunoassay is carried out for the detection of green fluorescent protein (GFP). Two commercially available polyclonal antibodies are used. The goat anti-GFP lAP (Rockland, #600-1-1-215) is immunoaffinity purified and the rabbit anti-GFP antibody (Torrey Pines Biolabs, #TP401) is protein A purified. The GFP protein in the plant extract is captured onto the solid phase microtiter by the rabbit antibody. Then a "sandwich" is formed between the solid phase rabbit antibody, the GFP protein, and the secondary goat antibody that is in the well. After a wash step, where unbound secondary antibody is .removed, the bound antibody is detected using an alkaline phosphatase-labeled antibody. Substrate for the enzyme is added and color development is measured by reading the absorbance of each well. The reading is then fit to a standard curve to plot the GFP concentrations versus the absorbance<br>
synGFP reporter gene<br><br>
Binary vector Cassette	GFP Activity (Average)<br>
RFU/mf protein	ELISA (Average)<br>
ng GFP/mg protein<br>
NOV4215(CmpS)	4722. 3	657.1<br>
NOV4216(CmpS)	2795.8	246.1<br>
NOV2110(ZmUbi)	272.0	41.4<br>
2. Lycopersicon esculentum To plants<br>
The CmpS fragment of CmYLCV promoter (NOV 4217) results in synGFP expression level<br>
that is significantly higher then the expression level of Ubq3 promoter of Arabidopsis (NOV<br>
4209).<br>
synGFP reporter gene<br>
The results are obtained from evaluating of intensity of synGFP fluorescence by microscopy<br><br>
in callus and primary shoots of 5 independent stably transformed callus lines transformed with the NOV4217 cassette and 2 independent stably transformed callus lines transformed with the NOV4209 cassette.<br><br>
Binary vector Cassette (Promoter)	Callus ID	GFP Fluorescence Intensity<br>
NOV4217(CmpS)	NOV4217-1	+++++<br>
NOV4217(CmpS)	NOV4217-2	++++<br>
NOV4217(CmpS)	NOV4217-3	++++<br>
NOV4217(CmpS)	NOV4217-4	+++<br>
NOV4217(CmpS)	NOV4217-5	+++<br>
.NOV4209 (Ubq3 (At))	NOV4209-1	+<br>
NOV4209 (Ubq3 (At))	NOV4209-2	+<br>
Gus-lntron-GUS reporter gene<br>
The results are obtained from evaluating of intensity of GUS staining in 11 independent stably transformed callus lines containing the CmpS promoter-reporter cassettes (lines NOV4220-1 through NOV4220-11). Two stably transformed callus lines containing CmpC promoter fragment of CmYLCV promoter (NOV4224-1, NOV4224-2) are evaluated. For comparison two stably transformed callus lines containing the Arabidopsis Ubiquitin 3 (Ubq3) promoter (NOV4208-1, NOV4208-2) are evaluated.<br><br>
Binary    vector    cassette (Promoter)	Callus ID	GUS Staining<br>
NOV4220 (CmpS)	NOV4220-1	++++<br>
NOV4220 (CmpS)	NOV4220-2	+++<br>
NOV4220 (CmpS)	NOV4220-3	++++<br>
NOV4220 (CmpS)	NOV4220-4	+++++<br>
NOV4220 (CmpS)	NOV4220-5	+++++<br><br>
NOV4220 (CmpS)	NOV4220-6	+++<br>
NOV4220 (CmpS)	NOV4220-7	++++<br>
NOV4220 (CmpS)	NOV4220-8	+++++<br>
NOV4220 (CmpS)	NOV4220-9	++++<br>
NOV4220 (CmpS)	NOV4220-10	+++<br>
NOV4220 (CmpS)	NOV4220-11	++++<br>
NOV4224 (CmpC)	NOV4224-1	++++<br>
NOV4224 (CmpC)	NOV4224-2	+++<br>
NOV4208 (Ubq3 (At))	NOV4208-1	+<br>
NOV4208 (Ubq3(At))	NOV4208-2	++<br>
Example 24: Transient expression experiments in maize<br>
1.	Embryos preparation<br>
Immature embryos from A188 or other suitable maize genotypes are excised from 8-12 day old ears into liquid 2DG4 + 0.5d (Duncan's D medium modified to contain 20mg/l glucose and supplemented with 10g/l mannose) and cultured for 1 to 2 days.<br>
2.	Plasmolysis<br>
Immature embryos were incubated in 12% sucrose solution for 3-4 hour prior bombardment. Embryos are arranged in 8-10 mm circles on a plate.<br>
3. Bombardment<br>
Transfection by Particle Bombardment of Maize embryos is performed in the presence of 5 (jg of plasmid DNA at 650 psi as described (Wright et a!, 2001, Plant Cell Reports, In Press) In case an 'in press' reference won't do, here are the details lifted from the paper: Fragment DNA is precipitated onto gold microcarriers (
 <br>
tissue. . Embryos are cultivated on the plates for 48 hrsin the dark at 25°C. Protein extracts are prepared by lysing cells in GUS lysis buffer and clarified by centrifugation.<br>
Example 25: GUS assays<br>
For detection of GUS gene expression the histochemical and chemiluminescent assays are performed.<br>
1.	Histochemical B-glucuronidase (GUS) assay<br>
Maize embryos and tomato in vitro callus lines are used in GUS assays. Either whole embryos or small pieces of callus are dipped into GUS staining solution. GUS staining solution contains 1 mM 5-bromo-4-chloro-3-indolyl glucuronide (X-Gluc, Duchefa, 20 mM stock in DMSO), 100 mM Na-phosphate buffer pH 7.0, 10 mM EDTA pH 8.0, and 0.1% . Tritpn X100. Tissue samples are incubated at 37°C for 1-16 hours. If necessary, samples are cleared with several washes of 70% EtOH to remove chlorophyll<br>
2.	B-qlucuronidase (GUS) chemilluminescent assay<br>
For quantitive analysis of Gus expression the GUS chemilluminescent assay is performed using GUS-Light Kit (Tropix. Inc) according to manufacturer's instructions. The activity is measured in a Luminometer.<br>
The GUS results with construct pCmpS and pCmpMG (Example 17) are normalized to a comparison clone value (pNOV3612, Example 19). Results obtained from two different experiments show that the 8 bp deletion in CmYLCV promoter (pCmpMG, Example 17) does not significantly affect the expression levels of the GUS reporteras compared to pCmpS.<br>
Transient expression in Z.mays embryos<br>
GUS Activity at 48 and 72 Hours after transfection<br><br>
Construct ID	GUS Activity   RFU/mg protein 48hr                                72hr<br>
pCmpS	103.7	51.7<br>
pCmpMG	81.4	70.8<br>
ZmUbi(pNOV3612)	52.6	70.2<br><br><br><br>
WE CLAIM:<br>
1.	Isolated DNA sequence capable of driving expression of an associated nucleotide<br>
sequence  wherein said DNA sequence comprises:<br>
a)	a nucleotide sequence selected from the group consisting of SEQ ID NO: 1, SEQ ID NO: 2, SEQ ID NO: 3, SEQ ID NO: 4, SEQ ID NO: 5, SEQ ID NO: 6, SEQ ID NO: 19 and SEQ ID NO: 20; or<br>
b)	the complement of a DNA sequence which hybridists under stringent conditions to any one of SEQ ID NO: 1, SEQ ID NO: 2, SEQ ID NO: 3, SEQ ID NO: 4, SEQ ID NO: 5 or SEQ ID NO: 6, said hybridisation being carried out at a temperature of 65°C in double strength citrate buffered saline containing 0.1% SDS followed by rinsing at the same temperature in a one tenth strength citrate buffered saline solution containing 0.1% SDS.<br><br>
2.	The DNA sequence according to claim 1, wherein said DNA sequence comprises a nucleotide sequence selected from the group consisting of SEQ ID NO: 2, SEQ ID NO:<br>
3.	SEQ ID NO: 4 and SEQ ID NO: 20 and wherein said DNA sequence additionally comprises at its 3'-end the nucleotide sequence depicted in SEQ ID NO: 5.<br><br>
3.	The DNA sequence according to claim 1 or claim 2 additionally comprising at its 5'-end the nucleotide sequence depicted in SEQ ID NQ: 6.<br>
4.	DNA sequence comprising a consecutive stretch of at least 50 nt of SEQ ID NO: 1, wherein said DNA sequence is capable of driving expression of an associated nucleotide sequence.<br><br>
5.	DNA sequence according to claim 4 wherein said consecutive stretch of at least 50 nt has at least 70% sequence identity with a consecutive stretch of at least 50 nt of SEQ ID N0:1.<br>
6.	Recombinant DNA molecule comprising a DNA sequence according to any one of claims 1 to 5 operably linked to a nucleotide sequence of interest.<br>
7.	The recombinant DNA molecule according to claim 6, wherein the nucleotide<br>
sequence of interest comprises a coding sequence.<br>
8.	The recombinant DNA molecule according to claim 7, wherein the coding sequence encodes a desirable phenotypic trait.<br>
9.	The recombinant DNA molecule according to claim 8, wherein the coding sequence encodes a protein which confers a positive selective advantage to cells that have been transformed with said coding sequence.<br>
10.	The recombinant DNA molecule according to claim 8 or 9, wherein the coding<br>
sequence encodes a protein which confers a metabolic advantage to cells that have been<br>
transformed with said coding sequence consisting of being able to metabolise a<br>
compound, wherein said compound is mannose or xylose or a derivative or a precursor of<br>
these, or a substrate of the protein, or is capable of being metabolised by cells<br>
transformed with said coding sequence into such a substrate.<br><br>
11.	The recombinant DNA molecule according to claim 10, wherein said coding sequence encodes an enzyme selected, from the group of xyloisomerases, phosphomanno-isomerase, mannose-6-phospbate isomerase, mannose-1-phosphate isomerase, phosphomanno mutase, mannose epimerase, mannose or xylose phosphatase, mannose-6-phosphatase, mannose-1-phosphatase and mannose or xylose permease.<br>
12.	The recombinant molecule according to claim 11, wherein the coding sequence encodes a phosphomanno isomerase.<br>
13.	The recombinant DNA according to claim 7, wherein the coding region is non-translatable.<br>
14.	The recombinant DNA according to claim 13, wherein the non-translatable coding region is from a viral gene.<br>
15.	The recombinant DNA according to claim 14, wherein the viral gene is derived from TSWV, in particular from the TSWV NP gene.<br>
16.	The recombinant DNA molecule according to claim 7, wherein the coding sequence is in antisense orientation.<br>
17.	A DNA expression vector comprising a DNA sequence according to any one of claims 1 to 5 or a recombinant DNA molecule of any one of claims 6 to 16.<br><br>
18.	The DNA expression vector according to claim 17, wherein said DNA expression vector is pNOV2819.<br>
19.	The DNA expression vector according to claim 17, wherein said DNA expression vector is pNOV2820.<br>
20.	A DNA expression vector comprising a first DNA sequence according to any one of claims 1 to 5 operably linked to anucleotide sequence of interest, and a second DNA sequence according to any one of claims 1 to 5 linked to a nucleotide sequence of interest.<br>
21.	A DNA expression vector according to claim 20 capable of altering the expression of a viral genome.<br>
22.	The DNA expression vector according to claim 21 comprising a first DNA sequence capable of expressing in a cell a sense RNA fragment of said viral genome or portion thereof and a second DNA sequence capable of expressing in a cell an antisense RNA fragment of said viral genome or portion thereof, wherein said sense RNA fragment and said antisense RNA fragment are capable of forming a double-stranded RNA.<br>
23.	The DNA expression vector according to claim 22, wherein said virus is selected from the group consisting of tospoviruses, potyviruses, potexviruses, tobamoviruses, luteoviruses, cucumoviruses, bromoviruses, closteoviruses, tombusviruses and ftiroviruses.<br><br>
24.	The DNA expression vector according to claim 23, wherein said DNA sequences<br>
comprises a nucleotide sequence derived from a viral coat protein gene, a viral<br>
nucleocapsid protein gene, a viral replicase gene, or a viral movement protein gene or<br>
portions thereof.<br>
25.	The DNA expression vector according to claim 24, wherein said DNA sequence is<br>
derived from tomato spotted wilt virus (TSWV).<br>
26.	The DNA expression vector according to claim 25, wherein said DNA is derived<br>
from a nucleocapsid protein gene.<br>
27.	A host cell stably transformed with a DNA sequence according to any one of claims 1<br>
to 5 or a recombinant DNA molecule of any one of claims 6 to 16 or a DNA expression<br>
vector according to anyone of claims 17 to 26, wherein said host cell is a bacterial cell.<br>
28.	A method of producing a DNA sequence according to claim 1, wherein the DNA is<br>
produced by a polymerase chain reaction wherein at least one oligonucleotide used<br>
comprises a sequence of nucleotides which represents a consecutive stretch of 15 or more<br>
base pairs ofSEQ ID NO: 1.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDE1MjgtY2hlIGFic3RyYWN0IGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-01528-che abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDE1MjgtY2hlIGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-01528-che abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDE1MjgtY2hlIGNsYWltcyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-01528-che claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDE1MjgtY2hlIGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-01528-che claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDE1MjgtY2hlIGNvcnJlc3BvbmRlbmNlIG90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-01528-che correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDE1MjgtY2hlIGNvcnJlc3BvbmRlbmNlIHBvLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-01528-che correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDE1MjgtY2hlIGRlc2NyaXB0aW9uIChjb21wbGV0ZSkgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-01528-che description (complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDE1MjgtY2hlIGRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-01528-che description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDE1MjgtY2hlIGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-01528-che form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDE1MjgtY2hlIGZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-01528-che form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDE1MjgtY2hlIGZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-01528-che form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDE1MjgtY2hlIGZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-01528-che form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDE1MjgtY2hlIGZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-01528-che form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDE1MjgtY2hlIHBjdCBzZWFyY2ggcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-01528-che pct search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDE1MjgtY2hlIHBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-01528-che pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDE1MjgtY2hlIHBldGl0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-01528-che petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="223516-a-method-and-a-device-for-the-production-of-a-cellulose-fiber.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="223518-a-method-of-manufacturing-an-optical-information-medium.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>223517</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2002/1528/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>47/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Nov-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>12-Sep-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>24-Sep-2002</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SYNGENTA PARTICIPATIONS AG</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>SCHWARZWALDALLE 215, CH-4058 BASEL,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>STAVOLONE, LIVA</td>
											<td>LANGACKERSTRASSE 49, CH-8057 ZURICH,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>KONONOVA, MARIA</td>
											<td>7920 STRAWBERRY MEADOWS STREET, RALEIGH, NC-27613,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>HOHN, THOMAS</td>
											<td>HANGSTRASSE 35, CH-4144 ARLESHEIM,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>DE HAAN PETRUS, THEODORUS</td>
											<td>VINCENT VAN GOGHLAAN 46, 2343 RR OEGSTGEEST,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>LIGON, HOPE, THOMPSON</td>
											<td>3616 SOUTH POINTE DRIVE, APEX, NC-27539,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/82</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP01/03408</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-03-26</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>01101802.5</td>
									<td>2001-01-26</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>0010486.9</td>
									<td>2000-04-28</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>0007427.8</td>
									<td>2000-03-27</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>4</td>
									<td>60/272076</td>
									<td>2001-02-28</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/223517-isolated-dna-sequence-capable-of-driving-expression-of-an-associated-nucleotide-sequence by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:36:04 GMT -->
</html>
