<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/223535-isolated-pectate-lyase-from-bacillus-licheniformis by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:32:28 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 223535:ISOLATED PECTATE LYASE FROM BACILLUS LICHENIFORMIS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">ISOLATED PECTATE LYASE FROM BACILLUS LICHENIFORMIS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>ABSTRACT 2654/MAS/98 An isolated pectate lyase which is i) a polypeptide produced by Bacillus licheniformis, ATCC 14580, or ii) a polypeptide comprising an amino acid sequence as shown in positions 1-341 of SEQ ID N0:8, or iii) an analogue of the polypeptide defined in i) or ii) which is at least 70% identical to said polypeptide, or is derived from said polypeptide by substitution, deletion or addition of one or several amino acids, or is immunologically reactive with a polyclonal antibody raised against said polypeptide in purified form.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
PECTIN DEGRADING ENZYMES FROM Bacillus  licheniformis<br>
The present invention relates to a pectin degrading enzyme preparation; preferably to microbial pectin degrading enzymes, more specifically to microbial enzymes exhibiting pectin degrading activity as their major enzymatic activity in the neutral and alkaline pH ranges, especially to cloned pectin degrading enzymes derived from Bacillus licheniformis;   to a method of producing such enzymes; and to methods for using such enzymes in the textile, detergent and cellulose fiber processing industries.<br>
BACKGROUND OF THE INVENTION<br>
Pectin polymers are important constituents of plant cell walls. Pectin is a hetero-polysaccharide with a backbone composed of alternating homogalacturonan (smooth regions) and rhamnogalacturonan (hairy regions). The smooth regions are linear polymers of 1,4-linked alpha-D-galacturonic acid. The galacturonic acid residues can be methyl-esterified on the carboxyl group to a varying degree, usually in a non-random fashion with blocks of polygalacturonic acid being completely methyl-esterified.<br>
Pectinases can be classified according to their preferential substrate, highly methyl-esterified pectin or low methyl-esterified pectin and polygalacturonic acid (pectate), and their reaction mechanism, beta-elimination or hydrolysis. Pectinases can be mainly endo-acting, cutting the polymer at random sites within the chain to give a mixture of oligomers, or they may be exo-acting, attacking from one end of the polymer and producing monomers or dimers. Several pectinase activities acting on the smooth regions of pectin are included in the classification of enzymes provided by the Enzyme Nomenclature (1992) such as pectate lyase (EC 4.2.2.2), pectin lyase (EC 4.2.2.10), polygalacturonase (EC 3.2.1.15), exo-<br>
polygalacturonase (EC 3.2.1.67), exo-polygalacturonate lyase (EC 4.2.2.9) and exo-poly-alpha-galacturonosidase (EC 3.2.1.82).<br>
Pectate lyases have been cloned from different bacterial genera such as Erwinia,   Pseudomonas,   Klebsiella  and Xanthomonas. Also from Bacillus subtilis   (Nasser et al. (1993) FEES 335:319-<br><br>
326) and Bacillus  sp. YA-14 (Kim et al. (1994) Biosci. Biotech. Biochem. 58:947-949) cloning of a pectate lyase has been described. Purification of pectate lyases with maximum activity in the pH range of 8-10 produced by Bacillus pumilus   (Dave and Vaughn (1971) J. Bacteriol. 108:166-174), B. polymyxa   (Nagel and Vaughn (1961) Arch. Biochem. Biophys. 93:344-352), B. stearotherinophilus (Karbassi and Vaughn (1980) Can. J. Microbiol. 26:377-384), Bacillus  sp. (Hasegawa and Nagel (1966) J. Food Sci, 31:838-845) and Bacillus  sp. RK9 (Kelly and Fogarty (1978) Can. J. Microbiol. 24:1164-1172) has been reported, however, no publication was found on cloning of pectate lyase encoding genes from these organisms. All the pectate lyases described require divalent cations for maximum activity, calcium ions being the most stimulatory.<br>
WO 98/453 93 discloses detergent compositions containing protopectinase with remarkable detergency agains muddy soilings.<br>
Generally, pectinase producing microorganisms exhibit a broad range of pectin degrading or modifying enzymes. Often the microorganisms also produce cellulases and/or hemicellulases and complex multi-component enzyme preparations from such microorganisms may be difficult to optimise for various applications, they even may contain enzymes with detrimental effect. Thus, it is an object of the present invention to provide a pectin degrading enzyme exhibiting only the desired effects e.g. in detergents or different industrial processes.<br>
SUMMARY OF THE INVENTION<br>
The inventors have now found several pectin degrading enzymes, especially alkaline pectin degrading enzymes, which are endogeneous to a bacterial strain of the genus Bacillus,   more specifically to the strain Bacillus licheniformis,   and have succeeded in identifying DNA sequences encoding such enzymes.<br>
Accordingly, in a first aspect the present invention relates to an enzyme preparation consisting essentially of a pectin degrading enzyme derived from or endogeneous to a strain of Bacillus licheniformis  or highly related Bacillus  species, the enzyme preferably being a pectate lyase (EC 4.2.2.2), a pectin lyase (EC 4.2.2.10) or a polygalacturonase (EC 3.2.1.15) .<br><br>
The DNA sequences of two pectate lyases of the invention are listed in the sequence listing as SEQ ID No. 3 and 7, respectively, and the deduced amino acid sequences are listed in the sequence listing as SEQ ID No. 4 and 8, respectively, it is believed that these novel enzyme will be classified according to the Enzyme Nomenclature in the Enzyme Class EC 4.2.2.2. However, it should be noted that the enzyme of the invention also exhibits catalytic activity on pectin (which may be esterified) besides the activity on pectate and polygalacturonides conventionally attributed to enzymes belonging to EC 4.2.2.2.<br>
The DNA sequence of a pectin lyase of the invention is listed in the sequence listing as SEQ ID No. 1 and the deduced amino acid sequence is listed in the sequence listing as SEQ ID No. 2. It is believed that this novel enzyme will be classified according to the Enzyme Nomenclature in the Enzyme Class EC 4.2.2.10.<br>
The DNA sequence of a polygalacturonase of the invention is listed in the sequence listing as SEQ ID No. 5 and the deduced amino acid sequence is listed in the sequence listing as SEQ ID No. 6. It is believed that this novel enzyme will be classified according to the Enzyme Nomenclature in the Enzyme Class EC 3 .2 .1.15.<br>
Accordingly, in a second aspect, the present invention relates to a pectate lyase which is i) a polypeptide produced by BacilluB  licheniformis,   ATCC 14580, or ii) a polypeptide comprising an amino acid sequence as shown in positions 1-341 of SEQ ID NO:8, or iii) an analogue of the polypeptide defined in i) or ii) which is at least 70% homologous with said polypeptide, is derived from said polypeptide by substitution, deletion or addition of one or several amino acids, or is immunologically reactive with a polyclonal antibody raised against said polypeptide in purified form.<br>
Within one aspect, the present invention provides an isolated polynucleotide molecule selected from the group consisting of (a) polynucleotide molecules encoding a polypeptide having pectate lyase activity and comprising a sequence of nucleotides as shown in SEQ ID NO: 7 from nucleotide 1 to nucleotide 1026; (b) species homologs of (a) ; (c) polynucleotide molecules that encode a polypeptide having<br><br>
pectate lyase activity that is at least 70% identical to the amino acid sequence of SEQ ID NO: 8 from amino acid residue 1 to amino acid residue 341; (d) molecules complementary to (a), (b) or (c); and (e) degenerate nucleotide sequences of {a), (b), (c) or (d).<br>
The plasmid pSJ1678 comprising the polynucleotide molecule (the DNA sequence) encoding the pectate lyase of the present invention as represented by the amino acid sequence SEQ ID NO:8 has been transformed into a strain of the Escherichia  coli  which was deposited by the inventors according to the Budapest Treaty on the International Recognition of the Deposit of Microorganisms for the Purposes of Patent Procedure at the Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg lb, D-38124 Braunschweig, Federal Republic of Germany, on 25 September 1997 under the deposition number DSM 11789.<br>
In a third aspect, the present invention relates to a pectate lyase which is i) a polypeptide produced by Bacillus licheniformis,   ATCC 14580, or ii) a polypeptide comprising an amino acid sequence as shown in positions 1-221 of SEQ ID N0:4, or iii) an analogue of the polypeptide defined in i) or ii) which is at least 70% homologous with said polypeptide, is derived from said polypeptide by substitution, deletion or addition of one or several amino acids, or is immunologically reactive with a polyclonal antibody raised against said polypeptide in purified form.<br>
Within one aspect, the present invention provides an isolated polynucleotide molecule selected from the group consisting of (a) polynucleotide molecules encoding a polypeptide having pectate lyase activity and comprising a sequence of nucleotides as shown in SEQ ID NO: 3 from nucleotide 1 to nucleotide 666; (b) species homologs of (a); (c) polynucleotide molecules that encode a polypeptide having pectate lyase activity that is at least 70% identical to the amino acid sequence of SEQ ID NO: 4 from amino acid residue 1 to amino acid residue 221; (d) molecules complementary to (a), (b) or (c); and (e) degenerate nucleotide sequences of (a), (b), (c) or (d).<br>
In a fourth aspect, the present invention relates to a<br><br>
pectin lyase which is i) a polypeptide produced by Bacillus licheniformis,   ATCC 14580, or ii) a polypeptide comprising an amino acid sequence as shown in positions 1-494 of SEQ ID NO:2, or iii) an analogue of the polypeptide defined in i) or ii) which is at least 70% homologous with said polypeptide, is derived from said polypeptide by substitution, deletion or addition of one or several amino acids, or. is immunologically reactive with a polyclonal antibody raised against said polypeptide in purified form.<br>
Within one aspect, the present invention provides an isolated polynucleotide molecule selected from the group consisting of (a) polynucleotide molecules encoding a polypeptide having pectin lyase activity and comprising a sequence of nucleotides as shown in SEQ ID NO: 1 from nucleotide 1 to nucleotide 1485; (b) species homologs of (a); (c) polynucleotide molecules that encode a polypeptide having pectin lyase activity that is at least 70% identical to the amino acid sequence of SEQ ID NO: 2 from amino acid residue 1 to amino acid residue 494; (d) molecules complementary to (a), (b) or {c); and (e) degenerate nucleotide sequences of (a), (b), (c) or (d).<br>
The plasmid pSJ1678 comprising the polynucleotide molecule (the DNA sequence) encoding the pectin lyase of the present invention as represented by the amino acid sequence SEQ ID NO:2 has been transformed into a strain of the Escherichia  coli  which was deposited by the inventors according to the Budapest Treaty on the International Recognition of the Deposit of Microorganisms for the Purposes of Patent Procedure at the Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg lb, D-38124 Braunschweig, Federal Republic of Germany, on 23 February 1998 under the deposition number DSM 12031.<br>
In a fifth aspect, the present invention relates to a polygalacturonase which is i) a polypeptide produced by Bacillus licheniformis,   ATCC 14580, or ii) a polypeptide comprising an amino acid sequence as shown in positions 1-415 of SEQ ID NO:6, or iii) an analogue of the polypeptide defined in i) or ii) which is at least 70% homologous with said polypeptide, is derived from said polypeptide by substitution, deletion or addition of one or several amino acids, or is immunologically<br><br>
reactive with a polyclonal antibody raised against said polypeptide in purified form.<br>
Within one aspect, the present invention provides an isolated polynucleotide molecule selected from the group consisting of (a) polynucleotide molecules encoding a polypeptide having polygalacturonase activity and comprising a sequence of nucleotides as shown in SEQ ID NO: 5 from nucleotide 1 to nucleotide 1248; (b) species homologs of (a); (c) polynucleotide molecules that encode a polypeptide having polygalacturonase activity that is at least 70% identical to the amino acid sequence of SEQ ID NO: 6 from amino acid residue 1 to amino acid residue 415; (d) molecules complementary to (a), (b) or (c); and (e) degenerate nucleotide sequences of (a), (b), (c) or (d) .<br>
The plasmid pSJ1678 comprising the polynucleotide molecule (the DNA sequence) encoding the polygalacturonase of the present invention as represented by the amino acid sequence SEQ ID NO:2 has been transformed into a strain of the Escherichia  coli  which was deposited by the inventors according to the Budapest Treaty on the International Recognition of the Deposit of Microorganisms for the Purposes of Patent Procedure at the Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg lb, D-38124 Braunschweig, Federal Republic of Germany, on 23 February 1998 under the deposition number DSM 12030.<br>
Within other aspects of the invention there is provided an expression vector comprising the following operably linked elements: a transcription promoter; a DNA segment selected from the group consisting of (a) polynucleotide molecules encoding a polypeptide having pectate lyase activity and comprising a sequence of nucleotides as shown in SEQ ID NO: 7 from nucleotide 1 to nucleotide 1026, or as shown in SEQ ID NO: 3 from nucleotide 1 to nucleotide 666; or encoding a polypeptide having pectin lyase activity and comprising a sequence of nucleotides as shown in SEQ ID NO: 1 from nucleotide 1 to nucleotide 1485; or encoding a polypeptide having polygalacturonase activity and comprising a sequence of nucleotides as shown in SEQ ID NO: 5 from nucleotide 1 to nucleotide 1248; (b) species homologs of (a); (c) polynucleotide molecules that encode a polypeptide<br><br>
having pectate lyase activity that is at least 70% identical to the amino acid sequence of SEQ ID NO: 8 from amino acid residue 1 to amino acid residue 341 or to the amino acid sequence of SEQ ID NO: 4 from amino acid residue 1 to amino acid residue 221; or encode a polypeptide having pectin lyase activity that is at least 70%- identical to the amino acid sequence of SEQ ID NO: 2 from amino acid residue 1 to amino acid residue and 494; encode a polypeptide having polygalacturonase activity that is at least 70% identical to the amino acid sequence of SEQ ID NO: 6 from amino acid residue 1 to amino acid residue 415; (d) degenerate nucleotide sequences of (a), (b), or (c); and a transcription terminator.<br>
Within yet another aspect of the present invention there is provided a cultured cell into which has been introduced an expression vector as disclosed above, wherein said cell expresses the polypeptide encoded by the DNA segment.<br>
A further aspect of the present invention provides an isolated polypeptide having pectate lyase activity selected from the group consisting of (a) polypeptide molecules comprising a sequence of amino acid residues as shown in SEQ ID NO:8 from amino acid residue 1 to amino acid residue 341; (b) polypeptide molecules comprising a sequence of amino acid residues as shown in SEQ ID NO:4 from amino acid residue 1 to amino acid residue 221; and (c) species homologs of (a) or (b).<br>
Another aspect of the present invention provides an isolated polypeptide having pectin lyase activity selected from the group consisting of (a) polypeptide molecules comprising a sequence of amino acid residues as shown in SEQ ID NO:2 from amino acid residue 1 to amino acid residue 4 94; and (b) species homologs of (a).<br>
Another aspect of the present invention provides an isolated polypeptide having polygalacturonase activity selected from the group consisting of (a) polypeptide molecules comprising a sequence of amino acid residues as shown in SEQ ID NO:6 from amino acid residue 1 to amino acid residue 415; and (b) species homologs of (a) .<br>
Within another aspect of the present invention there is provided a composition comprising a purified polypeptide<br><br>
according to the invention in combination with other polypeptides.<br>
Within another aspect of the present invention there are provided methods for producing a polypeptide according to the invention comprising culturing a cell into which has been introduced an expression vector as disclosed above, whereby said cell expresses a polypeptide encoded by the DNA segment and recovering the polypeptide.<br>
The novel enzyme of the present invention is useful for the treatment of cellulosic material, especially cellulose-containing fiber, yarn, woven or non-woven fabric, treatment of mechanical paper-making pulps or recycled waste paper, and for retting of fibres. The treatment can be carried out during the processing of cellulosic material into a material ready for garment manufacture or fabric manufacture, e.g. in the desizing or scouring step; or during industrial or household laundering of such fabric or garment.<br>
Accordingly, in further aspects the present invention<br>
relates to a detergent composition comprising an enzyme having<br>
substantial pectin degrading activity; and to use of the enzyme<br>
of the invention for the treatment of cellulose-containing<br>
fibers, yarn, woven or non-woven fabric.		N<br>
The enzymes of the invention are very effective for use in an enzymatic scouring process in the preparation of cellulosic material e.g. for proper response in subsequent dyeing operations. Further, it is contemplated that detergent compositions comprising the novel enzymes are capable of removing or bleaching certain soils or stains present on laundry, especially soils and spots resulting from galactan or arabinogalactan containing food, plants, and the like. It is also contemplated that treatment with detergent compositions comprising the novel enzyme can prevent binding of certain soils to the cellulosic material. The enzymes of the invention are also useful as ingredients in hard surface cleaning compositions having the effect of removing or assisting in removing certain soils or stains from hard surfaces in need of cleaning.<br>
DEFINITIONS<br><br>
Prior to discussing this invention in further detail, the following terms will first be defined.<br>
The term "ortholog" (or "species homolog") denotes a polypeptide or protein obtained from one species that has homology to an analogous polypeptide or protein from a different species.<br>
The term "paralog" denotes a polypeptide or protein obtained from a given species that has homology to a distinct polypeptide or protein from that same species.<br>
The term "expression vector" denotes a DNA molecule, linear or circular, that comprises a segment encoding a polypeptide of interest operably linked to additional segments that provide for its transcription.  Such additional segments may include promoter and terminator sequences, and may optionally include one or more origins of replication, one or more selectable markers, an enhancer, a polyadenylation signal, and the like. Expression vectors are generally derived from plasmid or viral DNA, or may contain elements of both. The expression vector of the invention may be any expression vector that is conveniently subjected to recombinant DNA procedures, and the choice of vector will often depend on the host cell into which the vector it is to be introduced. Thus, the vector may be an autonomously replicating vector, i.e. a vector which exists as an extra-chromosomal entity, the replication of which is independent of chromosomal replication, e.g. a plasmid. Alternatively, the vector may be one which, when introduced into a host cell, is integrated into the host cell genome and replicated together with the chromosome(s) into which it has been integrated.<br>
The term "recombinant expressed" or "recombinantly expressed" used herein in connection with expression of a polypeptide or protein is defined according to the standard definition in the art. Recombinantly expression of a protein is generally performed by using an expression vector as described immediately above.<br>
The term "isolated", when applied to a polynucleotide molecule, denotes that the polynucleotide has been removed from its natural genetic milieu and is thus free of other extraneous or unwanted coding sequences, and is in a form suitable for use within genetically engineered protein production systems.  Such<br><br>
isolated molecules are those that are separated from their natural environment and include cDNA and genomic clones. Isolated DNA molecules of the present invention are free of other genes with which they are ordinarily associated, but may include naturally occurring 5' and 3' untranslated regions such as promoters and terminators.  The identification of associated regions will be evident to one of ordinary skill in the art (see for example, Dynan and Tijan, Nature li£.:774-78, 1985) .  The term "an isolated polynucleotide" may alternatively be termed "a cloned polynucleotide".<br>
When applied to a protein/polypeptide, the term "isolated" indicates that the protein is found in a condition other than its native environment.  In a preferred form, the isolated protein is substantially free of other proteins, particularly other homologous proteins (i.e. "homologous impurities" (see below)). It is preferred to provide the protein in a greater than 4 0% pure form, more preferably greater than 6 0% pure form.<br>
Even more preferably it is preferred to provide the protein in a highly purified form, i.e., greater than 80% pure, more preferably greater than 95% pure, and even more preferably greater than 99% pure, as determined by SDS-PAGE.<br>
The term "isolated protein/polypeptide may alternatively be termed "purified protein/polypeptide".<br>
The term "homologous impurities" means any impurity (e.g. another polypeptide than the polypeptide of the invention) which originate from the homologous cell where the polypeptide of the invention is originally obtained from.<br>
The term "obtained from" as used herein in connection with a specific microbial source, means that the polynucleotide and/or polypeptide is produced by the specific source, or by a cell in which a gene from the source have been inserted.<br>
The term "endogeneous to" as used herein in connection with a specific microbial source, means that a polypeptide is produced by the specific source due to the presence in the source of a native gene, ie a gene which has not been recombinantly inserted into a cell of the source.<br>
The term "operably linked", when referring to DNA segments, denotes that the segments are arranged so that they function in concert for their intended purposes, e.g. transcription<br><br>
initiates in the promoter and proceeds through the coding segment to the terminator<br>
The term "polynucleotide" denotes a single- or double-stranded polymer of deoxyribonucleotide or ribonucleotide bases read from the 5' to the 3' end.  Polynucleotides include RNA and DNA, and may be isolated from natural sources, synthesized in vitro,   or prepared from a combination of natural and synthetic molecules.<br>
The term "complements of polynucleotide molecules" denotes polynucleotide molecules having a complementary base sequence and reverse orientation as compared to a reference sequence. For example, the sequence 5' ATGCACGGG 3' is complementary to 5' CCCGTGCAT 3'.<br>
The term "degenerate nucleotide sequence" denotes a sequence of nucleotides that includes one or more degenerate codons (as compared to a reference polynucleotide molecule that encodes a polypeptide) . Degenerate codons contain diffe'rent triplets of nucleotides, but encode the same amino acid residue (i.e., GAU and GAC triplets each encode Asp).<br>
The term "promoter" denotes a portion of a gene containing DNA sequences that provide for the binding of RNA polymerase and initiation of transcription.  Promoter sequences are commonly, but not always, found in the 5' non-coding regions of genes.<br>
The term "secretory signal sequence" denotes a DNA sequence that encodes a polypeptide (a "secretory peptide") that, as a component of a larger polypeptide, directs the larger polypeptide through a secretory pathway of a cell in which it is synthesized.  The larger peptide is commonly cleaved to remove the secretory peptide during transit through the secretory pathway.<br>
The term "pectin" denotes pectate, polygalacturonic acid, and pectin which may be esterified to a higher or lower degree.<br>
The term "pectin degrading enzyme" or "pectinase" denotes a pectinase enzyme defined according to the art where pectinases are a group of enzymes that cleave glycosidic linkages of pectic substances mainly poly(1,4-alpha-D-galacturonide and its derivatives (Sakai et al., 1993).<br>
Preferably a pectinase of the invention is a pectinase<br><br>
which is a pectinase enzyme which catalyzes the random cleavage of alpha-l,4-glycosidic linkages in pectic acid also called polygalacturonic acid by transelimination such as the enzyme class polygalacturonate lyase (EC 4.2.2.2) (PGL) also known as poly(1,4-alpha-D-galacturonide) lyase also known as pectate lyase. Also preferred is a pectinase enzyme which catalyzes the random hydrolysis of alpha-1,4-glycosidic linkages in pectic acid such as the enzyme class polygalacturonase (EC 3.2.1.15) (PG) also known as endo-PG. Also preferred is a pectinase enzyme such as polymethylgalcturonate lyase (EC 4.2.2.10) (PMGL), also known as Endo-PMGL, also known as poly(methyoxygalacturonide)lyase also known as pectin lyase which catalyzes the random cleavage of alpha-l,4-glycosidic linkages of pectin.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
HOW TO USE A SEQUENCE OF THE INVENTION TO GET OTHER RELATED SEQUENCES: The disclosed sequence information herein relating to a polynucleotide sequence encoding a pectate lyase of the invention can be used as a tool to identify other homologous pectate lyases. For instance, polymerase chain reaction (PCR) can be used to amplify sequences encoding other homologous pectate lyases from a variety of microbial sources, in particular of different Bacillus  species.<br>
ASSAY FOR ACTIVITY TEST<br>
A polypeptide of the invention having pectate lyase activity may be tested for pectate lyase activity according to standard test procedures known in the art, such as by applying a solution to be tested to 4 mm diameter holes punched out in agar plates<br>
containing 0.2% AZCL galactan (Megazyme, Australia).<br>
POLYNUCLEOTIDES:<br>
Within preferred embodiments of the invention an isolated polynucleotide of the invention will hybridize to similar sized regions of SEQ ID No, 1, 3, 5 or 7, respectively, or a sequence complementary thereto, under at least medium stringency conditions.<br><br>
In particular polynucleotides of the invention will hybridize to a denatured double-stranded DNA probe comprising either the full sequence shown in positions 1-1485 of SEQ ID N0:1, in positions 1-666 of SEQ ID NO:3, in positions 1-1248 of SEQ ID NO:5 or in positions 1-1026 of SEQ ID NO:7, or any probe comprising a subsequence of SEQ ID N0:1, 3, 5 or 7, respectively, having a length of at least about 100 base pairs under at least medium stringency conditions, but preferably at high stringency conditions as described in detail below. Suitable experimental conditions for determining hybridization at medium, or high stringency between a nucleotide probe and a homologous DNA or RNA sequence involves presoaking of the filter containing the DNA fragments or RNA to hybridize in 5 x SSC (Sodium chloride/Sodium citrate, Sambrook et al. 1989) for 10 min, and prehybridization of the filter in a solution of 5 x SSC, 5 X Denhardt's solution (Sambrook et al. 1989), 0.5 % SDS and 100 /g/ml of denatured sonicated salmon sperm DNA (Sambrook et al. 1989), followed by hybridization in the same solution containing a concentration of lOng/ml of a random-primed (Feinberg, A. P. and Vogelstein, B. (1983) Anal. Biochem. 132:6-13), 32P-dCTP-labeled (specific activity higher than 1 x 109 cpm//g ) probe for 12 hours at ca. 45°C. The filter is then washed twice for 30 minutes in 2 x SSC, 0.5 % SDS at least 60°C (medium stringency), still more preferably at least 6 5°C (medium/high stringency), even more preferably at least 70°C (high stringency), and even more preferably at least 75°C (very high stringency).<br>
Molecules to which the oligonucleotide probe hybridizes under these conditions are detected using a x-ray film.<br>
As previously noted, the isolated polynucleotides of the present invention include DNA and RNA.  Methods for isolating DNA and RNA are well known in the art. DNA and RNA encoding genes of interes can be cloned in Gene Banks or DNA libraries by means of methods known in the art.<br>
Polynucleotides encoding polypeptides having pectate lyase activity of the invention are then identified and isolated by, for example, hybridization or PCR.<br>
The present invention further provides counterpart polypeptides and polynucleotides from different bacterial<br><br>
strains (orthologs or paralogs). Of particular interest are pectin degrading polypeptides from gram-positive alkalophilic strains, including species of Bacillus.<br>
Species homologues of polypeptides exhibiting pectin degrading activity of the invention can be cloned using information and compositions provided by the present invention in combination with conventional cloning techniques. For example, DNA can be cloned using chromosomal DMA obtained from a cell type that expresses the protein. Suitable sources of DNA can be identified by probing Northern blots with probes designed from the sequences disclosed herein. A library is then prepared from chromosomal DNA of a positive cell line. A DNA encoding an polypeptide having pectin degrading activity of the invention can then be isolated by a variety of methods, such as by probing with a complete or partial DNA or with one or more sets of degenerate probes based on the disclosed sequences. A DNA can also be cloned using the polymerase chain reaction, or PCR (Mullis, U.S. Patent 4,683,202), using primers designed from the sequences disclosed herein. Within an additional method, the DNA library can be used to transform or transfect host cells, and expression of the DNA of interest can be detected with an antibody (mono-clonal or polyclonal) raised against the pectate lyases, pectin lyase or polygalacturonase cloned from B.licheniformis,   ATCC 14580, expressed and purified as described in Materials and Methods and the Examples, or by an activity test relating to a polypeptide having pectin degrading activity. Similar techniques can also be applied to the isolation of genomic clones.<br>
The polypeptide encoding part of the DNA sequence cloned into plasmid pSJ1678 present in Escherichia  coli  DSM 1178 9 and in Escherichia  coli  DSM 12030 and in Escherichia  coli  DSM 12031 and/or an analogue DNA sequence of the invention may be cloned from a strain of the bacterial species Bacillus licheniformis, preferably the strain ATCC 14580, producing the enzyme with pectin degrading activity, or another or related organism as described herein.<br>
Alternatively, the analogous sequence may be constructed on the basis of the DNA sequence obtainable from the plasmid present in Escherichia coli  DSM 11789, in Escherichia  coli  DSM<br><br>
12030 or in Escherichia  coli  DSM 12031, e.g be a sub-sequence thereof, and/or by introduction of nucleotide substitutions which do not give rise to another amino acid sequence of the pectin degrading enzyme encoded by the DNA sequence, but which corresponds to the codon usage of the host organism intended for production of the en2yme, or by introduction of nucleotide substitutions which may give rise to a different amino acid sequence (i.e. a variant of the pectin degrading enzyme of the invention).<br>
Based on the sequence information disclosed herein a full length DNA sequence encoding a pectinase of the invention and comprising the DNA sequence shown in SEQ ID No 1, 3, 5 or 7, respectively, may be cloned.<br>
Cloning of is performed by standard procedures known in the art such as by,<br>
preparing a genomic library from a Bacillus  strain;<br>
plating such a library on suitable substrate plates;<br>
identifying a clone comprising a polynucleotide sequence of the invention by standard hybridization techniques using a probe based on SEQ ID No 1, 3, 5 or 7, respectively; or by identifying a clone from said Bacillus licheniformis  ATCC 14580 genomic library by an Inverse PCR strategy using primers based on sequence information from SEQ ID No 1, 3, 5 or 7, respectively. Reference is made to M.J. MCPherson et al. ("PCR A practical approach" Information Press Ltd, Oxford England) for further details relating to Inverse PCR.<br>
Based on the sequence information disclosed herein (SEQ ID Nos 1, 2, 3, 4, 5, 6, 7 and 8) is it routine work for a person skilled in the art to isolate homologous polynucleotide sequences encoding homologous pectinases of the invention by a similar strategy using genomic libraries from related microbial organisms, in particular from genomic libraries from other strains of the genus Bacillus  such as Bacillus subtilis.<br>
Alternatively, the DNA encoding the pectin degrading enzyme of the invention may, in accordance with well-known procedures, conveniently be cloned from a suitable source, such as any of the below mentioned organisms, by use of synthetic oligonucleotide probes prepared on the basis of the DNA sequence obtainable from the plasmid present in Escherichia<br><br>
coli  DSM 11789, in Escherichia coli  DSM 12030 or in Escherichia coli  DSM 12031.<br>
Accordingly, the polynucleotide molecule of the invention may be isolated from Escherichia coli, DSM 11789, Escherichia coli, DSM 12030, or Escherichia coli, DSM 12031, in which the plasmid obtained by cloning such as described above is deposited. Also, the present invention relates to an isolated substantially pure biological culture of the strain Escherichia coli, DSM 11789, Escherichia coli, DSM 12030, and Escherichia coli,   DSM 12031, respectively.<br>
POLYPEPTIDES:<br>
The sequence of amino acids no. 1-221 of SEQ ID No 4 is a mature pectate lyase sequence. The sequence of amino acids no. 1-341 of SEQ ID No 8 is a mature pectate lyase sequence. The sequence of amino acids no. 31-494 of SEQ ID No 2 is a mature pectin lyase sequence, positions 1-30 are the prosequence. The sequence of amino acids no. 1-415 of SEQ ID No 6 is a mature polygalacturonase sequence.<br>
The present invention also provides pectate lyase polypeptides that are substantially homologous to the polypeptides of SEQ ID NO:2, 4, 6 and 8, respectively, and their species homologs (paralogs or orthologs. The term "substantially homologous" is used herein to denote polypeptides having at least 60%, preferably at least 70%, more preferably at least 8 5%, and even more preferably at least 90%, sequence identity to the sequence shown in SEQ ID NO:2, 4, 6 and 8, respectively, or their orthologs or paralogs.  Such polypeptides will more preferably be at least 95% identical, and most preferably 98% or more identical to the sequence shown in SEQ ID NO:2, 4, 6 and 8, respectively, or its orthologs or paralogs. Percent sequence identity is determined by conventional methods, by means of computer programs known in the art such as GAP provided in the GCG program package (Program Manual for the Wisconsin Package, Version 8, August 1994, Genetics Computer Group, 575 Science Drive, Madison, Wisconsin, USA 53711) as disclosed in Needleman, S.B. and Wunsch, CO., (1970), Journal of Molecular Biology, 48, 443-453, which is hereby incorporated by reference in its entirety. GAP is used with the following settings for<br><br>
polypeptide sequence comparison: GAP creation penalty of 3.0 and GAP extension penalty of 0.1.<br>
Sequence identity of polynucleotide molecules is determined by similar methods using GAP with the following settings for DNA sequence comparison: GAP creation penalty of 5.0 and GAP extension penalty of 0.3.<br>
The present invention is based in part upon the discovery of several novel polynucleotide sequences obtained from a Bacillus  licheniformis  strain which encode polypeptide sequences which have homology to other microbial amino acid pectinase sequences.<br>
The new Bacillus licheniformia  pectinases have been designated:<br>
I; pectate lyase (EC 4.2,2.2) II: pectate lyase (EC 4.2.2.2) III: pectin lyase (EC 4.2.2.10) IV:     polygalacturonase   (EC 3.2.1.15)<br>
The novel pectinase polypeptide sequences of the present invention were initially identified by querying representative search sequences, specifically amino acid sequences of known pectinases, to the Bacillus licheniformis  sequence database to identify homologous sequences to pectinases.<br>
Using a conventional percent sequence identity program as described in further details herein (vide infra)   the amino acid sequence identity of the amino acid sequences of the appended SEQ ID NOS: 2 (III), 4 (I), 6 (IV), 8 (II) to the closesst prior art known pectinases are shown in tables 1, 2 and 3 below.<br>
Table 1:<br>
Homology between amino acid sequences for pectate lyases (I) Most homologous pectin degrading protein to the pectat lyase (I) of this invention is the B.sp.   strain KSM-P15 found in the International Patent Application published as WO 98/453 93 as SEQ ID N0:1 which is 58.8% homologous to the protein sequence SEQ ID NO: 4 .<br>
Table 2 :<br>
Homology between amino acid sequences for pectate lyases (II)<br><br>
The sequence used for simularity are the protein sequences from TREMBLREL with the locus listed in the scheme. II: Amino acid sequence of the invention (SEQ ID N0:8) B:  O084 54.sp_bacteria Amycolata sp. Pectate lyase C: q00893.sp_fungi Glomerella cingulata<br><br>
Homology between amino acid aequences for pectin lyases (III)<br>
Most homologous pectin lyase proteins to the pectin lyase of this invention are the B.subtilis  pectin lyase 034819 and the B.subtilis  pectin lyase P94449, both protein sequences found in the TREMBL data base (EMBL/GENBANK/DDBJ DATA BANKS). They are 55% and 56% homologous to the protein sequence SEQ ID NO: 2.<br>
Table 4;<br>
Homology between amino acid sequences for polygalacturonases<br>
The sequences used for simularity are the protein sequences from SWISSPROT with the locus listed in the scheme. IV:   Amino acid sequence of the invention (SEQ ID NO:6) B:  p27644.swissprot_bacteria Agrobacterium tumefaciens C: p20041.swissprot_bacteria Burkholdaria solanacearum<br><br>
The enzyme preparation of the invention is preferably derived from a microorganism, preferably from a bacterium, an archea or a fungus, especially from a bacterium such as a bacterium belonging to Bacillus,   preferably to an alkalophilic Bacillus  strain which may be selected from the group consisting of the species BacilluB licheniformis  and highly related Bacillus  species in which all species preferably are at least<br><br>
95%, even more preferably at least 98%, homologous to Bacillus licheniformis  based on aligned 16S rDNA sequences.<br>
Substantially homologous proteins and polypeptides are characterized as having one or more amino acid substitutions, deletions or additions. These changes are preferably of a minor nature, that is conservative amino acid substitutions (see Table 2) and other substitutions that do not significantly affect the folding or activity of the protein or polypeptide; small deletions, typically of one to about 30 amino acids; and small amino- or carboxyl-terminal extensions, such as an amino-terminal methionine residue, a small linker peptide of up to about 20-25 residues, or a small extension that facilitates purification (an affinity tag), such as a poly-histidine tract, protein A (Nilsson et al., EMBO J. 1:1075, 1985; Nilsson et al., Methods Enzymol. 198:3, 1991. See, in general Ford et al.,<br>
pyptein Expression ar;id Purifigatxon Z-  95-107, 1991, which is<br>
incorporated herein by reference. DNAs encoding affinity tags are available from commercial suppliers (e.g., Pharmacia Biotech, Piscataway, NJ; New England Biolabs, Beverly, MA).<br>
However, even though the changes described above preferably are of a minor nature, such changes may also be of a larger nature such as fusion of larger polypeptides of up to 3 00 amino acids or more both as amino- or carboxyl-terminal extensions to a Pectate lyase polypeptide of the invention.<br><br><br><br>
In addition to the 20 standard amino acids, non-standard amino acids (such as 4-hydroxyproline, 6-W-methyl lysine, 2-aminoisobutyric acid, isovaline and a-methyl serine) may be substituted for amino acid residues of a polypeptide according to the invention. A limited number of non-conservative amino acids, amino acids that are not encoded by the genetic code, and unnatural amino acids may be substituted for amino acid residues. "Unnatural amino acids" have been modified after protein synthesis, and/or have a chemical structure in their side chain(s) different from that of the standard amino acids. Unnatural amino acids can be chemically synthesized, or preferably, are commercially available, and include pipecolic acid, thiazolidine carboxylic acid, dehydroproline, 3- and 4-methylproline, and 3,3-dimethylproline.<br>
Essential amino acids in the pectate lyase polypeptides of the present invention can be identified according to procedures known in the art, such as site-directed mutagenesis or alanine-scanning mutagenesis (Cunningham and Wells, Science 244: 1081-1085, 1989). In the latter technique, single alanine mutations are introduced at every residue in the molecule, and the resultant mutant molecules are tested for biological activity {i.e pectate lyase activity) to identify amino acid residues that are critical to the activity of the molecule. See also, Hilton et al., J. Biol. Chem. 221:4699-4708, 1996. The active site of the enzyme or other biological interaction can also be determined by physical analysis of structure, as determined by such techniques as nuclear magnetic resonance, crystallography, electron diffraction or photoaffinity labeling, in conjunction with mutation of putative contact site amino acids.  See, for example, de Vos et al., Science 2S5.:306-312, 1992; Smith et al. , J. Mol. Biol. 221:899-904, 1992; Wlodaver et al., FEBS Lett. 309:59-64, 1992.  The identities of essential amino acids can<br><br>
also be inferred from analysis of homologies with polypeptides which are related to a polypeptide according to the invention.<br>
Multiple amino acid substitutions can be made and tested using known methods of mutagenesis, recombination and/or shuffling followed by a relevant screening procedure, such as those disclosed by Reidhaar-Olson and Sauer (Science 241:53-57, 1988), Bowie and Sauer (Proc. Natl. Acad. Sci. USA 86:2152-2156, 1989), W095/17413, or WO 95/22625. Briefly, these authors disclose methods for simultaneously randomizing two or more positions in a polypeptide, or recombination/shuffling of different mutations (W095/17413, W095/22625), followed by selecting for functional a polypeptide, and then sequencing the mutagenized polypeptides to determine the spectrum of allowable substitutions at each position. Other methods that can be used include phage display (e.g., Lowman et al., Biochem. iQ.:10B32-10837, 1991; Ladner et al., U.S. Patent No. 5,223,409; Huse, WIPO Publication WO 92/06204) and region-directed mutagenesis (Derbyshire et al. , Gene 4i.:145, 1986; Ner et al. , DNA 2:127, 1988) .<br>
Mutagenesis/shuffling methods as disclosed above can be combined with high-throughput, automated screening methods to detect activity of cloned, mutagenized polypeptides in host cells. Mutagenized DNA molecules that encode active polypeptides can be recovered from the host cells and rapidly sequenced using modern equipment. These methods allow the rapid determination of the importance of individual amino acid residues in a polypeptide of interest, and can be applied to polypeptides of unknown structure.<br>
Using the methods discussed above, one of ordinary skill in the art can identify and/or prepare a variety of polypeptides that are substantially homologous to residues 31 to 494 of SEQ ID NO: 2 (the mature protein), residues 1 to 221 of SEQ ID NO: 4, residues 1 to 415 of SEQ ID NO: 6, and residues 1 to 341 of SEQ ID NO: 8 and retain the pectate lyase activity of the wild-type protein.<br>
The present invention relates in one aspect to a pectin lyase enzyme exhibiting protopectinase activity and having the amino acid sequence of positions 31-494 SEQ ID No:2 or an amino sequence derived therefrom by deletion, replacement or addition<br><br>
of one or more amino acid residues (hereinafter referred to as mutation) provided that the protopectinase and pectin lyase are not deactivated and the mutation conserves arginine at the 377th position and arginine at the 383rd position of SEQ ID No:2. Also, the degree of mutation is not particularly limited provided that the above described arginine in the 377th position and the 3 83rd position are conserved. Preferably, 56% or higher homology exists between such mutation variants of the native or parent pectin lyase enzyme, calculated on the partial sequence corresponding to amino acid positions 31 to 375 of SEQ ID No:2. More preferably, the homology is 70% or higher, particularly 80% or higher.<br>
The pectin degrading enzyme of the invention may, in addition to the enzyme core comprising the catalytically domain, also comprise a cellulose binding domain (CBD), the cellulose binding domain and enzyme core {the catalytically active domain) of the enzyme being operably linked. The cellulose binding domain (CBD) may exist as an integral part of the encoded enzyme, or a CBD from another origin may be introduced into the pectin degrading enzyme thus creating an enzyme hybrid. In this context, the term "cellulose-binding domain" is intended to be understood as defined by Peter Tomme et al. "Cellulose-Binding Domains: Classification and Properties" in "Enzymatic Degradation of Insoluble Carbohydrates", John N. Saddler and Michael H. Penner (Eds.), ACS Symposium Series, No. 618, 1996. This definition classifies more than 120 cellulose-binding domains into 10 families (I-X), and demonstrates that CBDs are found in various enzymes such as cellulases, xylanases, mannanases, arabinofuranosidases, acetyl esterases and chitinases. CBDs have also been found in algae, e.g. the red alga Porphyra purpurea  as a non-hydrolytic polysaccharide-binding protein, see Tomme et al., op.cit. However, most of the CBDs are from cellulases and xylanases, CBDs are found at the N and C termini of proteins or are internal. Enzyme hybrids are known in the art, see e.g. WO 90/00609 and WO 95/16782, and may be prepared by transforming into a host cell a DNA construct comprising at least a fragment of DNA encoding the cellulose-binding domain ligated, with or without a linker, to a DNA sequence encoding the pectin<br><br>
degrading enzyme and growing the host cell to express the fused gene. Enzyme hybrids may be described by the following formula:<br>
CBD - MR - X wherein CBD is the N-terminal or the C-terminal region of an amino acid sequence corresponding to at least the cellulose-binding domain; MR is the middle region (the linker), and may be a bond, or a short linking group preferably of from about 2 to about 100 carbon atoms, more preferably of from 2 to 4 0 carbon atoms; or is preferably from about 2 to to about 100 amino acids, more preferably of from 2 to 40 amino acids; and X is an N-terminal or C-terminal region of the pectin degrading enzyme of the invention.<br>
Preferably, the enzyme of the present invention has its maximum catalytic activity at a pH of at least 8, more preferably of at least 8.5, more preferably of at least 9, more preferably of at least 9.5, more preferably of at least 10, even more preferably of at least 10.5, especially of at least 11; and preferably the maximum activity of the enzyme is obtained at a<br>
temperature of at least 50°C, more preferably of at least 55°C.<br>
PROTEIN PRODUCTION:<br>
The polypeptides of the present invention, including full-length proteins, fragments thereof and fusion proteins, can be produced in genetically engineered host cells according to conventional techniques. Suitable host cells are those cell types that can be transformed or transfected with exogenous DNA and grown in culture, and include bacteria, fungal cells, and cultured higher eukaryotic cells. Bacterial cells, particularly cultured cells of gram-positive organisms, are preferred. Gram-positive cells from the genus of Bacillus are especially preferred, such as Bacillus subtilis.   Bacillus lentus,   Bacillus brevis,   Bacillus stearothermophilus,   Bacillus alkalophilus, Bacillus amyloliquefaciens,   Bacillus coagulans.   Bacillus  cir-culans,   Bacillus lautus,   Bacillus  thuringiensis,   Bacillus agaradhaerens,   or in particular Bacillus licheniformis.  ATCC 14580 is the type strain of Bacillus  licheniformis.<br>
Techniques for manipulating cloned DNA molecules and<br>
introducing exogenous DNA into a variety of host cells are<br>
disclosed by Sambrook et al., Molecular Cloning;	A Laboratory<br><br>
Mamial/ 2nd ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989; Ausubel et al. (eds.). Current Protocols in Molecular Biology, John Wiley and Sons, Inc., NY, 1987; and Bacillus subtilis and Other Gram-Positive Bacteria, Sonensheim et al., 1993, American Society for Microbiology, Washington D.C., which are incorporated herein by reference.<br>
In general, a DNA sequence encoding a pectate lyase of the present invention is operably linked to other genetic elements required for its expression, generally including a transcription promoter and terminator within an expression vector. The vector will also commonly contain one or more selectable markers and one or more origins of replication, although those skilled in the art will recognize that within certain systems selectable markers may be provided on separate vectors, and replication of the exogenous DNA may be provided by integration into the host cell genome.  Selection of promoters, terminators, selectable markers, vectors and other elements is a matter of routine design within the level of ordinary skill in the art. Many such elements are described in the literature and are available through commercial suppliers.<br>
To direct a polypeptide into the secretory pathway of a host cell, a secretory signal sequence (also known as a leader sequence, prepro sequence or pre sequence) is provided in the expression vector. The secretory signal sequence may be that of the polypeptide, or may be derived from another secreted protein or synthesized de novo.   Numerous suitable secretory signal sequences are known in the art and reference is made to (Bacillus subtilis and Other Gram-Positive Bacteria, Sonensheim et al., 1993, American Society for Microbiology, Washington D.C.; and Cutting, S. M.(eds.) "Molecular Biological Methods for Bacillus", John Wiley and Sons, 1990) for further description of suitable secretory signal sequences especially for secretion in a Bacillus host cell. The secretory signal sequence is joined to the  DNA sequence in the correct reading frame. Secretory signal sequences are commonly positioned 5' to the DNA sequence encoding the polypeptide of interest, although certain signal sequences may be positioned elsewhere in the DNA sequence of interest (see, e.g., Welch et al., U.S. Patent No. 5,037,743; Holland et al., U.S. Patent No. 5,143,830).<br><br>
Transformed or transfected host cells are cultured according to conventional procedures in a culture medium containing nutrients and other components required for the growth of the chosen host cells. A variety of suitable media, including defined media and complex media, are known in the art and generally include a carbon source, a nitrogen source, essential amino acids, vitamins and minerals. Media may also contain such components as growth factors or serum, as required. The growth medium will generally select for cells containing the exogenously added DNA by, for example, drug selection or deficiency in an essential nutrient which is complemented by the selectable marker carried on the expression vector or co-transfected into the host cell.<br>
The polypeptides of the present invention may also be produced by fermenting a wildtype strain belonging to the genus Bacillus, preferably a strain which may be selected from the group consisting of the species Bacillus licheniformis  and highly related Bacillus  species in which all species are at least 95% homologous to Bacillus  licheniformis  based on aligned 16S rDNA sequences. A specific and highly preferred example is Bacillus  licheniformis,   ATCC 14580.<br>
Further, the polypeptides of the present invention may be produced by fermenting a mutant or a variant derived from the above mentioned strain. Such a mutant may be obtained by using conventional techniques and selection for mutants giving higher pectinase activity.<br>
The fermentation may be carried out by cultivation of the strain under aerobic conditions in a nutrient medium containing carbon and nitrogen sources together with other essential nutrients, the medium being composed in accordance with the principles of the known art. The medium may be a complex rich medium or a minimal medium. The nitrogen source may be of inorganic and/or organic nature. Suitable inorganic nitrogen sources are nitrates and ammonium salts. Among the organic nitrogen sources quite a number are used regularly in fermentations. Examples are soybean meal, casein, corn, corn steep liquor, yeast extract, urea and albumin. Suitable carbon sources are carbohydrates or carbohydrate containing materials. Preferable the nutrient medium contains pectate.<br><br>
polygalacturonic acid and/or pectin esterified to a higher or lower degree as carbon source and/or inducer of pectinase production. Alternatively, the medium contains a pectin rich material such as soybean meal, apple pulp or citrus peel.<br>
Since the Bacillus  species of this invention are alkalophilic the cultivation is preferably conducted at alkaline pH values such as at least pH 8 or at least pH 9, which can be obtained by addition of suitable buffers such as sodium carbonate or mixtures of sodium carbonate and sodium bicarbonate after sterilisation of the growth medium.<br>
It is contemplated that fermentation of a wildtype strain or mutant in a suitable medium can result in a yield of at least 0.5 g of pectinase protein per litre of culture broth or even at least 1 g/1 or 2 g/1.<br>
PROTEIN ISOLATION:<br>
When the expressed recombinant polypeptide is secreted the polypeptide may be purified from the growth media. Preferably the expression host cells are removed from the media before purification of the polypeptide {e.g.   by centrifugation).<br>
When the expressed recombinant polypeptide is not secreted from the host cell, the host cell are preferably disrupted and the polypeptide released into an aqueous "extract" which is the first stage of such purification techniques. Preferably the expression host cells are removed from the media before the cell disruption {e.g.   by centrifugation).<br>
The cell disruption may be performed by conventional techniques such as by lysozyme digestion or by forcing the cells through high pressure. See (Robert K. Scobes, Protein Purification, Second edition, Springer-Verlag) for further description of such cell disruption techniques.<br>
Whether or not the expressed recombinant polypeptides (or chimeric polypeptides) is secreted or not it can be purified using fractionation and/or conventional purification methods and media.<br>
Ammonium sulfate precipitation and acid or chaotrope extraction may be used for fractionation of samples. Exemplary purification steps may include hydroxyapatite, size exclusion,<br><br>
FPLC and reverse-phase high performance liquid chromatography. Suitable anion exchange media include derivatized dextrans, agarose, cellulose, polyacrylamide, specialty silicas, and the like. PEI, DEAE, QAE and Q derivatives are preferred, with DEAE Fast-Flow Sepharose (Pharmacia, Piscataway, NJ) being particularly preferred. Exemplary chromatographic media include those media derivatized with phenyl, butyl, or octyl groups, such as Phenyl-Sepharose FF (Pharmacia), Toyopearl butyl 650 (Toso Haas, Montgomeryvilie, PA) , Octyl-Sepharose (Pharmacia) and the like; or polyacrylic resins, such as Amberchrom CG 71 (Toso Haas) and the like. Suitable solid supports include glass beads, silica-based resins, cellulosic resins, agarose beads, cross-linked agarose beads, polystyrene beads, cross-linked polyacrylamide resins and the like that are insoluble under the conditions in which they are to be used. These supports may be modified with reactive groups that allow attachment of proteins by amino groups, carboxyl groups, sulfhydryl groups, hydroxyl groups and/or carbohydrate moieties. Examples of coupling chemistries include cyanogen bromide activation, N-hydroxysuccinimide activation, epoxide activation, sulfhydryl activation, hydrazide activation, and carboxyl and amino derivatives for carbodiimide coupling chemistries. These and other solid media are well known and widely used in the art, and are available from commercial suppliers.<br>
Selection of a particular method is a matter of routine design and is determined in part by the properties of the chosen support. See, for example. Affinity Chromatography; Principles &amp; Methods, Pharmacia LKB Biotechnology, Uppsala, Sweden, 1988.<br>
Polypeptides of the invention or fragments thereof may also be prepared through chemical synthesis. Polypeptides of the invention may be monomers or multimers; glycosylated or non-glycosylated; pegylated or non-pegylated; and may or may not include an initial methionine amino acid residue.<br>
Accordingly, in a further aspect, the present invention also relates to a method of producing the enzyme preparation of the invention, the method comprising culturing a microorganism capable of producing the pectate lyase under conditions permitting the production of the enzyme, and recovering the enzyme from the culture. Culturing may be carried out using<br><br>
conventional fermentation techniques, e.g. culturing in shake flasks or fermentors with agitation to ensure sufficient aeration on a growth medium inducing production of the pectate lyase enzyme. The growth medium may contain a conventional N-source such as peptone, yeast extract or casamino acids, a reduced amount of a conventional C-source such as dextrose or sucrose, and an inducer such as xyloglucan or composit plant substrates such as cereal brans (e.g. wheat bran or rice husk). The recovery may be carried out using conventional techniques, e.g. separation of bio-mass and supernatant by centrifugation or filtration, recovery of the supernatant or disruption of cells if the enzyme of interest is intracellular, perhaps followed by further purification as described in EP 0 406 314 or by crystallization as described in WO 97/15660,<br>
In yet another aspect, the present invention relates to an isolated pectate lyase having the properties described above and which is free from homologous impurities, and is produced using conventional recombinant techniques.<br>
TRANSGENIC PLANTS<br>
The present invention also relates to a transgenic plant, plant part or plant cell which has been transformed with a DNA sequence encoding the pectin degrading enzyme of the invention so as to express and produce this enzyme in recoverable quantities. The enzyme may be recovered from the plant or plant part. Alternatively, the plant or plant part containing the recombinant enzyme may be used as such.<br>
The transgenic plant can be dicotyledonous or monocotyledonous, for short a dicot or a monocot. Examples of monocot plants are grasses, such as meadow grass {blue grass, Poa), forage grass such as festuca, lolium, temperate grass, such as Agrostis, and cereals, e.g. wheat, oats, rye, barley, rice, sorghum and maize (corn).<br>
Examples of dicot plants are tobacco, legumes, such as lupins, potato, sugar beet, pea, bean and soybean, and cruciferous (family Brassicaceae), such as cauliflower, oil seed rape and the closely related model organism Arabidopsis thaliana.<br>
Examples of plant parts are stem, callus, leaves, root.<br><br>
fruits, seeds, and tubers. In the present context, also specific plant tissues, such as chloroplast, apoplast, mitochondria, vacuole, peroxisomes and cytoplasm are considered to be a plant part. Furthermore, any plant cell, whatever the tissue origin, is considered to be a plant part.<br>
Also included within the scope of the invention are the progeny of such plants, plant parts and plant cells.<br>
The transgenic plant or plant cell expressing the enzyme of the invention may be constructed in accordance with methods known in the art. In short the plant or plant cell is constructed by incorporating one or more expression constructs encoding the enzyme of the invention into the plant host genome and propagating the resulting modified plant or plant cell into a transgenic plant or plant cell.<br>
Conveniently, the expression construct is a DNA construct which comprises a gene encoding the enzyme of the invention in operable association with appropriate regulatory sequences required for expression of the gene in the plant or plant part of choice. Furthermore, the expression construct may comprise a selectable marker useful for identifying host cells into which the expression construct has been integrated and DNA sequences necessary for introduction of the construct into the plant in question (the latter depends on the DNA introduction method to be used).<br>
The choice of regulatory sequences, such as promoter and terminator sequences and optionally signal or transit sequences is determined, eg on the basis of when, where and how the enzyme is desired to be expressed. For instance, the expression of the gene encoding the enzyme of the invention may be constitutive or inducible, or may be developmental, stage or tissue specific, and the gene product may be targeted to a specific tissue or plant part such as seeds or leaves. Regulatory sequences are eg described by Tague et al. Plant, Phys., 86, 506, 1988.<br>
For constitutive expression the 35S-CaMV promoter may be used (Franck et al., 1980. Cell 21: 285-294). Organ-specific promoters may eg be a promoter from storage sink tissues such as seeds, potato tubers, and fruits (Edwards &amp; Coruzzi, 1990. Annu. Rev. Genet. 24: 275-303), or from metabolic sink tissues such as meristems (Ito et al., 1994. Plant Mol. Biol. 24: 863-878), a<br><br>
seed specific promoter such as the glutelin, prolamin, globulin or albumin promoter from rice (Wu et al,, Plant and Cell Physiology Vol. 39, No. 8 pp. 885-889 (1998)), a Vlcia  faba promoter from the legumin B4 and the unknown seed protein gene from Vicia  faba  described by Conrad U. et al, Journal of Plant Physiology Vol. 152, No. 6 pp. 708-711 (1998), a promotter from a seed oil body protein (Chen et al., Plant and cell physiology vol. 39, No. 9 pp. 935-941 (1998), the storage protein napA promoter from Brassica napus, or any other seed specific promoter known in the art, eg as described in WO 91/14772. Furthermore, the promoter may be a leaf specific promoter such as the rbcs promoter from rice or tomato (Kyozuka et al., Plant Physiology Vol. 102, No. 3 pp. 991-1000 (1993), the chlorella virus adenine methyltransferase gene promoter (Mitra, A. and Higgins, DW, Plant Molecular Biology Vol. 26, No. 1 pp. 85-93 (1994), or the aldP gene promoter from rice (Kagaya et al., Molecular and General Genetics Vol. 248, No. 6 pp. 668-674 (1995), or a wound inducible promoter such as the potato pin2 promoter (Xu et al. Plant Molecular Biology Vol. 22, No. 4 pp. 573-588 (1993) .<br>
A promoter enhancer element may be used to achieve higher expression of the enzyme in the plant. For instance, the promoter enhancer element may be an intron which is placed between the promoter and the nucleotide sequence encoding the enzyme. For instance, Xu et al. op clt  disclose the use of the first intron of the rice actin 1 gene to enhance expression.<br>
The selectable marker gene and any other parts of the expression construct may be chosen from those available in the art.<br>
The DNA construct is incorporated into the plant genome according to conventional techniques known in the art, including Agrojbacteriuffl-mediated transformation, virus-mediated transformation, micro injection, particle bombardment, biolistic transformation, and electroporation (Gasser et al. Science, 244, 1293; Potrykus, Bio/Techn. 8, 535, 1990; Shimamoto et al. Nature, 338, 274, 1989).<br>
Presently, Agrrobacterium tumefaciens mediated gene transfer is the method of choice for generating transgenic dicots (for review Hooykas &amp; Schilperoort, 1992. Plant Mol.<br><br>
Biol. 19: 15-38), however it can also be used for transforming monocots, although other transformation methods are generally-preferred for these plants. Presently, the method of choice for generating transgenic monocots is particle bombardment (microscopic gold or tungsten particles coated with the transforming DNA) of embryonic calli or developing embryos (Christou, 1992. Plant J. 2: 275-281; Shimamoto, 1994. Curr. Opin. Biotechnol. 5: 158-162; Vasil et al., 1992. Bio/Technology 10: 667-674) . An alternative method for transformation of monocots is based on protoplast transformation as described by Omirulleh S, et al., Plant Molecular biology Vol. 21, No. 3 pp. 415-428 (1993) .<br>
Following transformation, the transformants having incorporated the expression construct are selected and regenerated into whole plants according to methods well-known in the art.<br>
ENZYME PREPARATION<br>
In the present context, the terra **enzyme preparation" is intended to mean either be a conventional enzymatic fermentation product, possibly isolated and purified, from a single species of a microorganism, such preparation usually comprising a number of different enzymatic activities; or a mixture of monocomponent enzymes, preferably enzymes derived from bacterial or fungal species by using conventional recombinant techniques, which enzymes have been fermented and possibly isolated and purified separately and which may originate from different species, preferably fungal or bacterial species; or the fermentation product of a microorganism which acts as a host cell for expression of a recombinant pectate lyase, but which microorganism simultaneously produces other enzymes, e.g. pectin lyases, proteases, or cellulases, being naturally occurring fermentation products of the microorganism, i.e. the enzyme complex conventionally produced by the corresponding naturally occurring microorganism.<br>
The pectin degrading enzyme preparation of the invention may further comprise one or more enzymes selected from the group consisting of proteases, cellulases (endo-P-1,4-glucanases), p-<br><br>
glucanases (endo-P-1,3(4)-glucanases), lipases, cutinases, peroxidases, laccases, amylases, glucoamylases, pectinases, reductases, oxidases, phenoloxidases, ligninases, pullulanases, arabinanases, hemicellulases, mannanases, xyloglucanases, xylanases, pectin acetyl esterases, rhamnogalacturonan acetyl esterases, polygalacturonases, rhamnogalacturonases, galactanases, pectat lyases, pectin lyases, pectin methylesterases, cellobiohydrolases, transglutaminases; or mixtures thereof. In a preferred embodiment, one or more or all enzymes in the preparation is produced by using recombinant techniques, i.e. the enzyme(s) is/are mono-component enzyme(s) which is/are mixed with the other enzyme(s) to form an enzyme preparation with the desired enzyme blend.<br>
IMMUNOLOGICAL CROSS-REACTIVITY:<br>
Polyclonal antibodies (which are monospecific for a given enzyme protein) to be used in determining immunological cross-reactivity may be prepared by use of a purified pectin degrading enzyme. More specifically, antiserum against the pectin degrading enzyme of the invention may be raised by immunizing rabbits (or other rodents) according to the procedure described by N. Axelsen et al. in: A Manual of Quantitative Immunoelectrophoresis, Blackwell Scientific Publications, 1973, Chapter 23, or A. Johnstone and R. Thorpe, Immunochemistry in Practice, Blackwell Scientific Publications, 1982 (more speci¬fically p. 27-31). Purified immunoglobulins may be obtained from the antisera, for example by salt precipitation ((NH4)2 SO) , followed by dialysis and ion exchange chromatography, e.g.   on DEAE-Sephadex. Immunochemical characterization of proteins may be done either by Outcherlony double-diffusion analysis (O. Ouchterlony in: Handbook of Experimental Immunology (D.M. Weir, Ed.), Blackwell Scientific Publications, 1967, pp. 655-706), by crossed Immunoelectrophoresis (N. Axelsen et al., supra, Chap¬ters 3 and 4), or by rocket Immunoelectrophoresis (N. Axelsen et al., Chapter 2).<br>
Use in the detergent or cleaning industry<br>
In further aspects, the present invention relates to a detergent composition comprising the pectin degrading enzyme or<br><br>
enzyme preparation of the invention, to a process for machine treatment of fabrics comprising treating fabric during a washing cycle of a machine washing process with a washing solution containing pectin degrading enzyme or enzyme preparation of the invention, and to cleaning compositions, including laundry, hard surface cleaner, personal cleansing and oral/dental compositions, comprising a pectin degrading enzyme or enzyme preparation of the invention providing superior cleaning performance, i.e. superior stain removal.<br>
Without being bound to this theory, it is believed that the mannanase of the present invention is capable of effectively degrading or hydrolysing any soiling or spots containing galatomannans and, accordingly, of cleaning laundry comprising such soilings or spots.<br>
The cleaning compositions of the invention must contain at least one additional detergent component. The precise nature of these additional components, and levels of incorporation thereof will depend on the physical form of the composition, and the nature of the cleaning operation for which it is to be used.<br>
The cleaning compositions of the present invention preferably further comprise a detergent ingredient selected from a selected surfactant, another enzyme, a builder and/or a bleach system.<br>
The cleaning compositions according to the invention can be liquid, paste, gels, bars, tablets, spray, foam, powder or granular. Granular compositions can also be in "compact" form and the liquid compositions can also be in a "concentrated" form.<br>
The compositions of the invention may for example, be formulated as hand and machine dishwashing compositions, hand and machine laundry detergent compositions including laundry additive compositions and compositions suitable for use in the soaking and/or pretreatment of stained fabrics, rinse added fabric softener compositions, and compositions for use in general household hard surface cleaning operations. Compositions containing such carbohydrases can also be formulated as sanitization products, contact lens cleansers and health and beauty care products such as oral/dental care and personal cleaning compositions.<br><br>
When formulated as compositions for use in manual dishwashing methods the compositions of the invention preferably contain a surfactant and preferably other detergent compounds selected from organic polymeric compounds, suds enhancing agents, group II metal ions, solvents, hydrotropes and additional enzymes.<br>
When formulated as compositions suitable for use in a laundry machine washing method, the compositions of the invention preferably contain both a surfactant and a builder compound and additionally one or more detergent components preferably selected from organic polymeric compounds, bleaching agents, additional enzymes, suds suppressors, dispersants, lime-soap dispersants, soil suspension and anti-redeposition agents and corrosion inhibitors. Laundry compositions can also contain softening agents, as additional detergent components. Such compositions containing carbohydrase can provide fabric cleaning, stain removal, whiteness maintenance, softening, colour appearance, dye transfer inhibition and sanitization when formulated as laundry detergent compositions.<br>
The compositions of the invention can also be used as detergent additive products in solid or liquid form. Such additive products are intended to supplement or boost the performance of conventional detergent compositions and can be added at any stage of the cleaning process.<br>
If needed the density of the laundry detergent compositions herein ranges from 400 to 1200 g/litre, preferably 500 to 950 g/litre of composition measured at 20°C.<br>
The "compact" form of the compositions herein is best reflected by density and, in terms of composition, by the amount of inorganic filler salt; inorganic filler salts are conventional ingredients of detergent compositions in powder form;  in conventional detergent compositions, the filler salts are present in substantial amounts, typically 17-3 5% by weight of the total composition. In the compact compositions, the filler salt is present in amounts not exceeding 15% of the total composition, preferably not exceeding 10%, most preferably not exceeding 5% by weight of the composition. The inorganic filler salts, such as meant in the present compositions are selected from the alkali and alkaline-earth-metal salts of sulphates and<br><br>
chlorides. A preferred filler salt is sodium sulphate.<br>
Liquid detergent compositions according to the present invention can also be in a "concentrated form", in such case, the liquid detergent compositions according the present invention will contain a lower amount of water, compared to conventional liquid detergents. Typically the water content of the concentrated liquid detergent is preferably less than 40%, more preferably less than 3 0%, most preferably less than 20% by weight of the detergent composition.<br>
Suitable specific detergent compounds for use herein are selected from the group consisting of the specific compounds as described in WO 97/0162 9 which is hereby incorporated by reference in its entirety,<br>
Mannanase may be incorporated into the cleaning compositions in accordance with the invention preferably at a level of from 0.0001% to 2%, more preferably from 0.0005% to 0.5%, most preferred from 0.001% to 0.1% pure enzyme by weight of the composition.<br>
The cellulases usable in the present invention include both bacterial or fungal cellulases. Preferably, they will have a pH optimum of between 5 and 12 and a specific activity above 50 CEVU/mg (Cellulose Viscosity Unit). Suitable cellulases are disclosed in U.S. Patent 4,435,307, J61078384 and W096/02653 which discloses fungal cellulase produced from Humicola insolens,   Trichoderma,   Thielavia  and Sporotrichum,   respectively. EP 739 982 describes cellulases isolated from novel Bacillus species. Suitable cellulases are also disclosed in GB-A-2075028 ,-GB-A-2095275; DE-OS-22 47 832 and W095/26398.<br>
Examples of such cellulases are cellulases produced by a strain of Humicola  insolens   (Humicola grisea var.   thermoidea), particularly the strain Humicola  insolens,   DSM 1800. Other suitable cellulases are cellulases originated from Humicola insolens  having a molecular weight of about 50kD, an isoelectric point of 5.5 and containing 415 amino acids; and a ~43kD endo-beta-l,4-glucanase derived from Humicola insolens,   DSM 1800; a preferred cellulase has the amino acid sequence disclosed in PCT Patent Application No. WO 91/17243. Also suitable cellulases are the EGIII cellulases from Trichoderma longihrachiatum  described in WO94/21801. Especially suitable cellulases are the cellulases<br><br>
having color care benefits. Examples of such cellulases are the cellulases described in W096/29397, EP-A-0495257, WO 91/17243, W091/17244 and WO91/21801. Other suitable cellulases for fabric care and/or cleaning properties are described in WO96/34092, W09S/17994 and W095/24471.<br>
Said cellulases are normally incorporated in the detergent composition at levels from 0.0001% to 2% of pure enzyme by weight of the detergent composition.<br>
Preferred cellulases for the purpose of the present invention are alkaline cellulases, i.e. enzyme having at least 25%, more preferably at least 40% of their maximum activity at a pH ranging from 7 to 12. More preferred cellulases are enzymes having their maximum activity at a pH ranging from 7 to 12. A preferred alkaline cellulase is the cellulase sold under the<br>
tradename Carezyme® by Novo Nordisk A/3.<br>
Amylases (a and/or fi) can be included for removal of carbohydrate-based stains. W094/D2597, Novo Nordisk A/S published February 03, 1994, describes cleaning compositions which incorporate mutant amylases. See also  WO95/10603, Novo Nordisk A/S, published April 20, 1995. Other amylases known for use in cleaning compositions include both a- and P-amylases.  a-Amylases are known in the art and include those disclosed in US Pat. no. 5,003,257; EP 252,666; WO/91/00353; PR 2,676,456; EP 285,123; EP 525,610; EP 368,341; and British Patent specification no. 1,296,839 (Novo). Other suitable amylases are stability-enhanced amylases described in W094/18314, published August 18, 1994 and WO96/05295, Genencor, published February 22, 1996 and amylase variants having additional modification in the immediate parent available from Novo Nordisk A/S, disclosed in WO 95/10603, published April 95. Also suitable are amylases described in EP 277 216, W095/26397 and W096/23873 {all by Novo Nordisk).<br>
Examples of commercial a-amylases products are Purafect Ox<br>
Am® from Genencor and Termamyl®, Ban® ,Fungamyl® and Duramyl , all available from Novo Nordisk A/S Denmark. W095/2 63 97 describes other suitable amylases : a-amylases characterised by having a specific activity at least 25% higher than the specific<br>
activity of Termamyl® at a temperature range of 25°C to 55°C and<br><br>
at a pH value in the range of 8 to 10, measured by the Phadebas<br>
® a-araylase activity assay. Suitable are variants of the above enzymes, described in W096/23873 (Novo Nordisk). Other amylolytic enzymes with improved properties with respect to the activity level and the combination of thermostability and a higher activity level are described in W095/35382.<br>
Preferred amylases for the purpose of the present invention are the amylases sold under the tradename Termamyl, Duramyl and<br>
Maxamyl and or the a-amylase variant demonstrating increased thermostability disclosed as SEQ ID No. 2 in W096/23873,<br>
Preferred amylases for specific applications are alkaline amylases, ie enzymes having an enzymatic activity of at least 10%, preferably at least 25%, more preferably at least 40% of their maximum activity at a pH ranging from 7 to 12. More preferred amylases are enzymes having their maximum activity at a pH ranging from 7 to 12.<br>
The amylolytic enzymes are incorporated in the detergent compositions of the present invention a level of from 0.0001% to 2%, preferably from 0.00018% to 0.06%, more preferably from 0.00024% to 0.048% pure enzyme by weight of the composition.<br>
The term xyloglucanase encompasses the family of enzymes described by Vincken and Voragen at Wageningen University [Vincken et al (1994) Plant Physiol., 104, 99-107] and are able to degrade xyloglucans as described in Hayashi et al (1989) Plant. Physiol, Plant Mol. Biol., 40, 139-168. Vincken et al demonstrated the removal of xyloglucan coating from cellulase of the isolated apple cell wall by a xyloglucanase purified from Trichoderma  viride   (endo-IV-glucanase). This enzyme enhances the enzymatic degradation of cell wall-embedded cellulose and work in synergy with pectic enzymes. Rapidase LIQ+ from Gist-Brocades contains an xyloglucanase activity.<br>
This xyloglucanase is incorporated into the cleaning compositions in accordance with the invention preferably at a level of from 0.0001% to 2%, more preferably fromO.0005% to 0,5%, most preferred from 0.001% toO.l % pure enzyme by weight of the composition.<br>
Preferred xyloglucanases for specific applications are alkaline xyloglucanases, ie enzymes having an enzymatic activity<br><br>
of at least 10%, preferably at least 25%, more preferably at least 40% of their maximum activity at a pH ranging from 7 to 12. More preferred xyloglucanases are enzymes having their maximum activity at a pH ranging from 7 to 12.<br>
The above-mentioned enzymes may be of any suitable origin, such as vegetable, animal, bacterial, fungal and yeast origin. Origin can further be mesophilic or extremophilic (psychrophilic, psychrotrophic, thermophilic, barophilic, alkalophilic, acidophilic, halophilic, etc). Purified or non-purified forms of these enzymes may be used. Nowadays, it is common practice to modify wild-type enzymes via protein or genetic engineering techniques in order to optimise their performance efficiency in the cleaning compositions of the invention. For example, the variants may be designed such that the compatibility of the enzyme to commonly encountered ingredients of such compositions is increased. Alternatively, the variant may be designed such that the optimal pH, bleach or chelant stability, catalytic activity and the like, of the enzyme variant is tailored to suit the particular cleaning application.<br>
In particular, attention should be focused on amino acids sensitive to oxidation in the case of bleach stability and on surface charges for the surfactant compatibility. The isoelectric point of such enzymes may be modified by the substitution of some charged amino acids, e.g. an increase in isoelectric point may help to improve compatibility with anionic surfactants. The stability of the enzymes may be further enhanced by the creation of e.g. additional salt bridges and enforcing metal binding sites to increase chelant stability.<br>
Use in the textile and cellulosic fiber processing industries<br>
The pectin degrading enzyme of the present invention can be used alone or in combination with other carbohydrate degrading enzymes (for instance arabinanase, xyloglucanase, mannanase) for biopreparation of fibers or for cleaning of fibers in combination with detergents. Cotton fibers consist of a primary cell wall layer containing pectin and a secondary layer containing mainly cellulose. Under cotton preparation or cotton refining part of the primary cell wall will be removed. The<br><br>
present invention relates to either help during cotton refining by removal of the primary cell wall. Or during cleaning of the Dotton to remove residual pectic substances and prevent graying of  the textile.<br>
In the present context, the term "cellulosic material" is intended to mean fibers, sewn and unsewn fabrics, including knits, wovens, denims, yarns, and toweling, made from cotton, cotton blends or natural or manmade cellulosics (e.g. orig¬inating from xylan-containing cellulose fibers such as from wood pulp) or blends thereof. Examples of blends are blends of cotton or rayon/viscose with one or more companion material such as wool, synthetic fibers (e.g. polyamide fibers, acrylic fibers, polyester fibers, polyvinyl alcohol fibers, polyvinyl chloride fibers, polyvinylidene chloride fibers, polyurethane fibers, polyurea fibers, aramid fibers), and cellulose-containing fibers (e.g. rayon/viscose, ramie, hemp, flax/linen, jute, cellulose acetate fibers, lyocell).<br>
The preparation of the present invention is useful in the cellulosic fiber processing industry for the pretreatment or retting of fibers from hemp, flax or linen.<br>
The processing of cellulosic material for the textile industry, as for example cotton fiber, into a material ready for garment manufacture involves several steps: spinning of the fiber into a yarn; construction of woven or knit fabric from the yarn and subsequent preparation, dyeing and finishing operations. Woven goods are constructed by weaving a filling yarn between a series of warp yarns; the yarns could be two different types. Knitted goods are constructed by forming a network of interlocking loops from one continuous length of yarn. The cellulosic fibers can also be used for non-woven fabric.<br>
The preparation process prepares the textile for the proper response in dyeing operations. The sub-steps involved in preparation are<br>
a.	Desizing (for woven goods) using polymeric size like<br>
e.g. starch, CMC or PVA is added before weaving in order to<br>
increase the warp speed; This material must be removed before<br>
further processing.<br>
b.	Scouring, the aim of which is to remove non-cellulosic<br><br>
material from the cotton fiber, especially the cuticle (mainly consisting of waxes) and primary cell wall (mainly consisting of pectin, protein and xyloglucan). A proper wax removal is necessary for obtaining a high wettability, being a measure for obtaining a good dyeing. Removal of the primary cell wall -especially the pectins - improves wax removal and ensures a more even dyeing. Further this improves the whiteness in the bleaching process. The main chemical used in scouring is sodium hydroxide in high concentrations, up to 70 g/kg cotton and at high temperatures, 80-95°C; and<br>
c. Bleaching; normally the scouring is followed by a bleach using hydrogen peroxide as the oxidizing agent in order to obtain either a fully bleached (white) fabric or to ensure a clean shade of the dye.<br>
A one step combined scour/bleach process is also used by the industry. Although preparation processes are most commonly employed in the fabric state; scouring, bleaching and dyeing operations can also be done at the fiber or yarn stage.<br>
The processing regime can be either batch or continuous with the fabric being contacted by the liquid processing stream in open width or rope form. Continuous operations generally use a saturator whereby an approximate equal weight of chemical bath per weight of fabric is applied to the fabric, followed by a heated dwell chamber where the chemical reaction takes place. A washing section then prepares the fabric for the next processing step. Batch processing generally takes place in one processing bath whereby the fabric is contacted with approximately 8 -15 times its weight in chemical bath. After a reaction period, the chemicals are drained, fabric rinsed and the next chemical is applied. Discontinuous pad-batch processing involves a saturator whereby an approximate equal weight of chemical bath per weight of fabric is applied to the fabric, followed by a dwell period which in the case of cold pad-batch might be one or more days.<br>
Woven goods are the prevalent form of textile fabric construction. The weaving process demands a "sizing" of the warp yarn to protect it from abrasion. Starch, polyvinyl alcohol (PVA), carboxymethyl cellulose, waxes and acrylic binders are examples of typical sizing chemicals used because of availability and cost. The size must be removed after the<br><br>
reaving process as the first step in preparing the woven goods. Phe sized fabric in either rope or open width form is brought in contact with the processing liquid containing the desizing igentB. The desizing agent employed depends upon the type of size to be removed. For PVA sizes, hot water or oxidative processes are often used. The most common sizing agent for rotton fabric is based upon starch. Therefore most often, woven ::otton fabrics are desized by a combination of hot water, the snzyme a-amylase to hydrolyze the starch and a wetting agent or surfactant. The cellulosic material is allowed to stand with the Resizing chemicals for a "holding period" sufficiently long to accomplish the desizing. The holding period is dependent upon the type of processing regime and the temperature and can vary from 15 minutes to 2 hours, or in some cases, several days. Typically, the desizing chemicals are applied in a saturator<br>
bath which generally ranges from about 15°C to about 55°C. The fabric is then held in equipment such as a "J-box" which provides sufficient heat, usually between about 55°C and about<br>
100°C, to enhance the activity of the desizing agents. The chemicals, including the removed sizing agents, are washed away from the fabric after the termination of the holding period.<br>
In order to ensure a high whiteness or a good wettability and resulting dyeabillty, the size chemicals and other applied chemicals must be thoroughly removed. It is generally believed that an efficient desizing is of crucial importance to the following preparation processes: scouring and bleaching.<br>
The scouring process removes much of the non-cellulosic compounds naturally found in cotton. In addition to the natural non-cellulosic impurities, scouring can remove dirt, soils and residual manufacturing introduced materials such as spinning, coning or slashing lubricants. The scouring process employs sodium hydroxide or related causticizing agents such as sodium carbonate, potassium hydroxide or mixtures thereof. Generally an alkali stable surfactant is added to the process to enhance solubilization of hydrophobic compounds and/or prevent their redeposition back on the fabric. The treatment is generally at a high temperature, 80°C - 100°C, employing strongly alkaline solutions, pH 13-14, of the scouring agent. Due to the non-<br><br>
specific nature of chemical processes not only are the impurities but the cellulose itself is attacked, leading to damages in strength or other desirable fabric properties. The softness of the cellulosic fabric is a function of residual natural cotton waxes. The non-specific nature of the high temperature strongly alkaline scouring process cannot discriminate between the desirable natural cotton lubricants and the manufacturing introduced lubricants. Furthermore, the conventional scouring process can cause environmental problems due to the highly alkaline effluent from these processes. The scouring stage prepares the fabric for the optimal response in bleaching. An inadequately scoured fabric will need a higher level of bleach chemical in the subsequent bleaching stages.<br>
The bleaching step decolorizes the natural cotton pigments and removes any residual natural woody cotton trash components not completely removed during ginning, carding or scouring. The main process in use today is an alkaline hydrogen peroxide bleach. In many cases, especially when a very high whiteness is not needed, bleaching can be combined with scouring.<br>
In the examples below it is shown that the scouring step can be carried out using the pectate lyase or pectate lyase<br>
preparation of the present invention a temperature of about 50°C<br>
- 80°C and a pH of about 7-11, thus substituting or supplementing the highly causticizing agents. An optimized enzymatic process ensures a high pectin removal and full wettability.<br>
Degradation or modification of plant material<br>
The enzyme or enzyme preparation according to the invention is preferably used as an agent for degradation or modification of plant cell walls or any pectin-containing material originating from plant cells walls due to the high plant cell wall degrading activity of the enzyme of the invention.<br>
The pectin degrading enzyme of the present invention may be used alone or together with other enzymes like glucanases, pectinases and/or hemicellulases to improve the extraction of oil from oil-rich plant material, like soy-bean oil from soy-beans, olive-oil from olives or rapeseed-oil from rape-seed or sunflower<br><br>
oil from sunflower.<br>
The pectin degrading enzyme of the present invention may be used for separation of components of plant cell materials. Of particular interest is the separation of sugar or starch rich plant material into components of considerable commercial interest (like sucrose from sugar beet or starch from potato) and components of low interest (like pulp or hull fractions). Also, of particular interest is the separation of protein-rich or oil-rich crops into valuable protein and oil and invaluable hull fractions, The separation process may be performed by use of methods known in the art.<br>
The pectin degrading enzyme of the invention may also be used in the preparation of fruit or vegetable juice in order to increase yield, and in the enzymatic hydrolysis of various plant cell wall-derived materials or waste materials, e.g. from wine or juice production, or agricultural residues such as vegetable hulls, bean hulls, sugar beet pulp, olive pulp, potato pulp, and the like.<br>
The plant material may be degraded in order to improve different kinds of processing, facilitate purification or extrac¬tion of other component than the galactans like purification of pectins from citrus, improve the feed value, decrease the water binding capacity, improve the degradability in waste water plants, improve the conversion of plant material to ensilage, etc.<br>
By means of an enzyme preparation of the invention it is possible to regulate the consistency and appearence of processed fruit or vegetables. The consistency and appearence has been shown to be a product of the actual combination of enzymes used for processing, i.e. the specificity of the enzymes with which the pectate lyase of the invention is combined. Examples include the production of clear juice e.g. from apples, pears or berries; cloud stable juice e.g. from apples, pears, berries, citrus or tomatoes; and purees e.g. from carrots and tomatoes.<br>
The pectin degrading enzyme of the invention may be used in modifying the viscosity of plant cell wall derived material. For instance, the pectin degrading enzyme may be used to reduce the viscosity of feed which contain galactan and to promote processing of viscous galactan containing material. The viscosity<br><br>
reduction may be obtained by treating the galactan containing plant material with an enyme preparation of the invention under suitable conditions for full or partial degradation of the galactan containing material.<br>
The pectin degrading enzyme can be used e.g. in combination with other enzymes for the removal of pectic substances from plant fibres. This removal is essential e.g. in the production of textile fibres or other cellulosic materials. For this purpose plant fibre material is treated with a suitable amount of the pectin degrading enzyme of the invention under suitable conditions for obtaining full or partial degradation of pectic substances associated with the plant fibre material.<br>
Animal feed additive<br>
Pectin degrading enzyme of the present invention may be used for modification of animal feed and may exert their effect either in vitro (by modifying components of the feed) or in vivo. The pectin degrading enzyme is particularly suited for addition to animal feed compositions containing high amounts of arabinogalactans or galactans, e.g. feed containing plant material from soy bean, rape seed, lupin etc. When added to the feed the pectin degrading enzyme significantly improves the in vivo break-down of plant cell wall material, whereby a better utilization of the plant nutrients by the animal is achieved. Thereby, the growth rate and/or feed conversion ratio (i.e. the weight of ingested feed relative to weight gain) of the animal is improved. For example the indigestible galactan is degraded by pectate lyase, e.g. in combination with 6-galactosidase, to galactose or galactooligomers which are digestible by the animal and thus contribute to the available energy of the feed. Also, by the degradation of galactan the pectate lyase may improve the digestibility and uptake of non-carbohydrate feed constituents such as protein, fat and minerals.<br>
For further description reference is made to PCT/DK 96/00443 and a working example herein.<br>
Wine and juice processing<br>
The enzyme or enzyme preparation of the invention may be used for de-pectinization and viscosity reduction in vegetable or<br><br>
fruit juice, especially in apple or pear juice. This may be accomplished by treating the fruit or vegetable juice with an enzyme preparation of the invention in an amount effective for degrading pectin-containing material contained in the fruit or vegetable juice.<br>
The enzyme or enzyme preparation may be used in the treatment of mash from fruits and vegetables in order to improve the extractability or degradability of the mash. For instance, the enzyme preparation may be used in the treatment of mash from apples and pears for juice production, and in the mash treatment of grapes for wine production.<br>
DETERMINATION OF CATALYTIC ACTIVITY OF PECTIN DEGRADING ENZYMES The viscosity assay APSU<br>
APSU units: The APSU unit assay is a viscosity measurement using the substrate polygalacturonic acid with no added calcium.<br>
The substrate 5% polygalacturonic acid sodium salt (Sigma P-1879) is solubilised in 0.1 M Glycin buffer pH 10. The 4 ml substrate is preincubated for 5 min at 40 'C. The enzyme is added (in a volume of 250pil) and mixed for 10 sec on a mixer at maximum speed, it is then incubated for 20 min at 40°C. For a standard curve double determination of a dilution of enzyme concentration in the range of 5 APSU/ml to above 100 APSU/ml with minimum of 4 concentrations between 10 and 60 APSU per ml The viscosity is measured using a MIVI 600 from the company Sofraser, 45700 Villemandeur, France. The viscosity is measured as mV after 10 sec.<br>
For calculation of APSU units a enzyme standard dilution as described above was used for obtaining a standard curve. The GrafPad Prism program, using a non linear fit with a one phase exponential decay with a plateau, was used for calculations. The plateau plus span is the mV obtained without enzyme. The plateau is the mV of more than 100 APSU and the half reduction of viscosity in both examples was found to be 12 APSU units with a standard error of 1.5 APSU.<br>
The lyase assay (at 235 nm)<br>
For determination of the S-elimination an assay measuring the increase in absorbance at 235 nm was carried out using the<br><br>
substrate 0.1% polygalacturonic acid sodium salt (Sigma P-1879) solubilised in 0.1 M Glycin buffer pH 10. For calculation of the catalytic rate an increase of 5.2 Absorbency at 235 units per min corresponds to formation of 1 /xmol of unsaturated product {Nasuna and Starr (1966) J. Biol. Chem. Vol 241 page 5298-5306; and Bartling, Wegener and Olsen (1995) Microbiology Vol 141 page 873-881) .<br>
Steady state condition using a 0.5 ml cuvette with a l cm light path on a HP diode array spectrophotometer in a temperature controlled cuvette holder with continuous measurement of the absorbency at 235 nm. For steady state a linear increase for at least 200 sec was used for calculation of the rate. It was used for converted to formation jiimol per min product.<br>
Endpoint analysis - Transelimination at 235 nm for Pectate Lyases (high Calcium method: 1 itM Calcium in the final incubation mixture)<br>
In this method, the substrate and enzyme is incubated for 20 min at 37°C followed by measurement at 235 nm of the formation of double bounds. Finally, the rate of the degradation is calculated based on the molar extinction coefficient in terms of Trans Units.<br>
Procedure:<br>
Mixing of 0,5 ml enzyme dilution with 0,5 ml 2*substrate solution.<br>
Substrate :Polygalactoronic acid from Sigma  P-1879 lot 77H37B4<br>
Buffer 2x : 0.IM Glycin pH 10 + 2.0 mmol CaCl<br>
Stop reagent: 0.02 M H3PO4<br>
Temperature of incubation 3 7°C<br>
Reaction time 20 min.<br>
Extinction coefficient of the transelimination 0.0052 /imol cm"<br>
Enzyme diluted in ion-free water to 0.5 to 5 APSU per ml. Main value in duplicate 0.5 ml. The 2% w/v substrate in 2x buffer is mixed with 0.5 ml diluted enzyme Both pre-incubated 5 min on water bath at 37°C. Incubate for 20 min. Stop using 5 ml stop reagent and mix. Blank mix enzyme and stop reagent first<br><br>
and then ad substrate all in the same volume.<br>
Enzyme        0.5 ml<br>
Substrate     0.5 ml<br>
Stop	5 ml<br>
Total volume   6 ml<br>
Measure the absorbency at 23 5 nm in a 1 cm cuvette.<br>
Calculate the formation of transelimination per min using the extinction coefficient of 0.0052 /xmole cm-1<br>
Calculation: [(main plus main)/2 -Blank] 0,0052*6*2*Enzyme dilution/20min/1000ml = /imol per min.<br>
Endpoint analysis - Transelimination at 235 nm for Pectin Lyases at pH 9.0<br>
The method is carried out as described above for the Endpoint analysis - Transelimination at 23 5 nm for Pectate Lyases (high Calcium method) but using the following substrate and buffer:<br>
Substrate :Pectin Esterifired from Citrus from Sigma  P-9561 lot 125H0123.<br>
Buffer 2x : 0.IM Borate pH 9.0, 5mM EDTA.<br>
MATERIALS AHD METHODS Strains<br>
Bacillus  licheniformis  ATCC 14 580.<br>
B.subtilis  PL23D6. This strain is the B.subtilis DN1885 with disrupted apr and npr genes (Diderichsen, B., Wedsted, U. , Hedegaard, L., Jensen, B. R., SJ0holm, C. (1990) Cloning of aldB, which encodes alpha-acetolactate decarboxylase, an exoenzyme from Bacillus brevis. J. Bacterid., 172, 4315-4321) disrupted in the transcriptional unit of the known Bacillus subtilis  cellulase gene, resulting in cellulase negative cells. The disruption was performed essentially as described in ( Eds. A.L. Sonenshein, J.A. Hoch and Richard Losick (1993) Bacillus subtilis  and other Gram-Positive Bacteria, American Society for microbiology, p.618).<br>
Competent cells were prepared and transformed as described by Yasbin, R.E., Wilson, G.A. and Young, F.E. (1975) Transformation and transfection in lysogenic strains of Bacillus subtilis: evidence for selective induction of prophage in competent cells. J. Bacteriol, 121:296-304.<br><br>
Plasmids<br>
PMOL944:<br>
This plasmid is a pUBllO derivative essentially containing elements making the plasmid propagatable in Bacillus  suhtilis, kanamycin resistance gene and having a strong promoter and signal peptide cloned from the amyL gene of B.licheniformis ATCC14580. The signal peptide contains a SacII site making it convenient to clone the DNA encoding the mature part of a protein in-fusion with the signal peptide. This results in the expression of a Pre-protein which is directed towards the exterior of the cell.<br>
The plasmid was constructed by means of conventional genetic engineering techniques which are briefly described in the following.<br>
Construction of pMOL944:<br>
The pUBllO plasmid (McKenzie, T. et al., 1986, Plasmid 15:93-103) was digested with the unique restriction enzyme Neil. A PCR fragment amplified from the amyL promoter encoded on the plasmid pDNl98l (P.L. J0rgensen et al.,1990, Gene, 96, p37-4l.) was digested with Neil and inserted in the Neil digested pUBllO to give the plasmid pSJ2624. The two PCR primers used have the following sequences:<br>
#	LWN54 94 5'-GTCGCCGGGGCGGCCGCTATCAATTGGTAACTGTATCTCAGC -3"<br>
#	LWN54 95 5 ' - GTCGCCCGGGAGCTCTGATCAGGTACCAAGCTTGTCGACCTGCAGAA<br>
TGAGGCAGCAAGAAGAT -3'<br>
The primer #LWN5494 inserts a NotI site in the plasmid.<br>
The plasmid pSJ2624 was then digested with Sad and NotI and a new PCR fragment amplified on amyL promoter encoded on the pDN1981 was digested with Sad and NotI and this DNA fragment was inserted in the SacI-NotI digested pSJ2624 to give the plasmid pSJ2670.<br>
This cloning replaces the first amyL promoter cloning with the same promoter but in the opposite direction. The two primers used for PCR amplification have the following sequences: #LWN5 93 8  5 " -GTCGGCGGCCGCTGATCACGTACCAAGCTTGTCGACCTGCAGAATG AGGCAGCAAGAAGAT -3"<br><br>
#LWN5 93 9  5 "■ -GTCGGAGCTCTATCAATTGGTAACTGTATCTCAGC - 3 '<br>
The plasmid pSJ2670 was digested with the restriction enzymes PstI and Bell and a PCR fragment amplified from a cloned DNA sequence encoding the alkaline amylase SP722 (disclosed in the International Patent Application published as W095/26397 which is hereby incorporated by reference in its entirety) was digested with PstI and Bell and inserted to give the plasmid pMOL944. The two primers used for PCR amplification have the following sequence:<br>
#LWN-7864  5"- -AACAGCTGATCACGACTGATCTTTTAGCTTGGCAC-3'<br>
#LWW7901	5" -AACTGCAGCCGCGGCACATCATAATGGGACAAATGGG -3'<br>
The primer #LWN7901 inserts a SacII site in the plasmid.<br>
General molecular biolocfy methods<br>
Unless otherwise mentioned the DFA manipulations and transformations were performed using standard methods of molecular biology (Sambrook et al. (1989) Molecular cloning: A laboratory manual. Cold Spring Harbor lab., Cold Spring Harbor, NY; Ausubel, F. M. et al. (eds.) "Current protocols in Molecular Biology". John Wiley and Sons, 1995; Harwood, C. R., and Cutting, S. M. (eds.) "Molecular Biological Methods for Bacillus". John Wiley and Sons, 1990).<br>
Enzymes for DNA manipulations were used according to the specifications of the suppliers (e.g. restriction endonucleases, ligases etc. are obtainable from New England Biolabs, Inc.).<br>
Media<br>
TY (as described in Ausubel, F. M. et al. (eds.) "Current protocols in Molecular Biology". John Wiley and Sons, 1995). LB agar (as described in Ausubel, F. M. et al. (eds.) "Current protocols in Molecular Biology". John Wiley and Sons, 1995). LBPG is LB agar supplemented with 0.5% Glucose and 0.05 M potassium phosphate, pH 7.0 BPX media is described in EP 0 506 780 (WO 91/09129) .<br>
The following examples illustrate the invention.<br>
EXAMPLE 1<br><br>
Cloning, expression, purification and characterization of a pectate lyase (II) from Bacillus licbexiiformis<br>
Genomic DNA prepayati-on<br>
strain Bacillus licheniformis  ATCC 14 580 was propagated in liquid medium 3 as specified by ATCC (American Type Culture<br>
Collection, USA) . After 18 hours incubation at 37°C and 3 00 rpm, the cells were harvested, and genomic DNA isolated by the method described by Pitcher et al, (Pitcher, D. G., Saunders, N. A., Owen, R. J. (1989), Rapid extraction of bacterial genomic DNA with guanidium thiocyanate. Lett. Appl. Microbiol., 8, 151-156).<br>
The pectate lyase II (vide  supra,   represented by amino acid sequence SEQ ID NO:8 encoding DNA sequence of the invention was PCR amplified using the PCR primer set consisting of these two oligo nucleotides:<br>
Peel.B.lich.upper.SacII<br>
5'-CTA ACT GCA GCC QCG GCA GCT TCT GCC TTA AAC TCG GGC -3'<br>
Peel.B.lich.lower.NotI<br>
5'-GCG TTG AGA CGC QCG GCC GCT GAA TGC CCC GGA CGT TTC ACC -3'<br>
Restriction sites SacII and Notll are underlined.<br>
Chromosomal DNA isolated from B.licheniformis  ATCC 14580 as described above was used as template in a PCR reaction using Amplitaq DNA Polymerase (Perkin Elmer) according to manufacturers instructions. The PCR reaction was set up in PCR buffer (10 mM Tris-HCl, pH 8.3, 50 mM KCl, 1.5 mM MgCl,,, 0.01 % (w/v) gelatin) containing 200 /xM of each dNTP, 2.5 units of AmpliTaq polymerase (Perkin-Elmer, Cetus, USA) and 100 pmol of each primer<br>
The PCR reactions was performed using a DNA thermal cycler (Landgraf, Germany). One incubation at 94°C for 1 min followed by thirty cycles of PCR performed using a cycle profile of denaturation at 94°C for 30 sec, annealing at 60°C for 1 min, and extension at 72°C for 2 min. Five-fxl  aliquots of the ampli¬fication product was analysed by electrophoresis in 0.7 % agarose gels (NuSieve, FMC). The appearance of a DNA fragment size 1.0 kb indicated proper amplification of the gene segment.<br><br>
gubgionjng of PCR fragment<br>
Fortyfive-/il aliquots of the PCR products generated as described above were purified using QIAquick PCR purification kit (Qiagen, USA) according to the manufacturer's instructions. The purified DNA was eluted in 50 1 of lOmM Tris-HCl, pH 8.5. 5 /j,g of pMOL944 and twentyfive-/il of the purified PCR fragment was digested with SacII and NotI, electrophoresed in 0.8 % low gelling temperature agarose (SeaPlaque GTG, FMC) gels, the relevant fragments were excised from the gels, and purified using QIAquick Gel extraction Kit (Qiagen, USA) according to the manufacturer's instructions. The isolated PCR DNA fragment was then ligated to the SacII-NotI digested and purified pMOL944. The ligation was performed overnight at 16°C using 0.5 fj,g  of each DNA fragment, 1 U of T4 DNA ligase and T4 ligase buffer (Boehringer Mannheim, Germany).<br>
The ligation mixture was used to transform competent B.subtilis  PL23 06. The transformed cells were plated onto LBPG-10 /ig/ml of Kanamycin plates. After 18 hours incubation at 37°C several clones were restreaked on fresh agar plates and also grown in liquid TY cultures with 10 fig/  ml kanamycin and incubated overnight at 37°C. Next day 1 ml of cells were used to isolate plasmid from the cells using the Qiaprep Spin Plasmid Miniprep Kit #27106 according to the manufacturers recommendations for B.subtilis  plasmid preparations. This plasmid DNA was used as template for DNA sequencing.<br>
One clone containing the pectate lyase gene was kept, this clone was termed MB541.<br>
The DNA corresponding to the mature part of the pectate lyase was characterised by DNA sequencing by primerwalking, using the Taq deoxy-terminal cycle sequencing kit (Perkin-Elmer, USA), fluorescent labelled terminators and appropriate oligonucleotides as primers.<br>
Analysis of the sequence data was performed according to '  Devereux et al.    (1984) Nucleic Acids Res. 12, 387-395. The<br>
cloned DNA sequence was expressed in B.subtilis  and the protein that appeared in the supernatant corresponded to the mature protein represented in SEQ ID NO:8.<br><br>
Purification<br>
MB541 was grown in 25 x 200 ml BPX media with 10 /xg/ml of Kanatnycin in 500 ml two baffled shakeflasks for 5 days at 37°C at 300 rpm, whereby 3500 ml of culture broth was obtained. The pH was adjusted to 5.0 using acetic acid and 100 ml of cationic agent (C521) and 200 ml of anionic agent (A130) was added during agitation for flocculation. The flocculated material was separated by centrifugation using a Sorval RC 3B centrifuge at 10000 rpm for 30 min at 6°C. The resulting supernatant contained 370 APSU per ml in a total volume of 3600 ml.<br>
The supernatant was clarified using Whatman glass filters GF/D and C and finally concentrated on a filtron UF membrane with a cut off of 10 kDa. The total volume of 2000 ml was adjusted to pH 8.5. 50 gram of DEAE A-50 Sephadex (Pharmacia) was swelled in 2000 ml 50 mM Tris pH 8.5. Excess buffer was discarded and the clear concentrated enzyme solution was mixed with the slurry for 15 min. The enzyme was separated from the ion-exchange material by suction on a Buchner funnel. The resulting solution was concentrated on a filtron with a cut off of 10 kDa to a final volume of 800 ml.<br>
For obtaining a highly purified pectate lyase a final step using S-sepharose cation-exchange chromatography was carried out. 50 ml of the solution of 950 APSU per ml (see above) was adjusted to pH 5.0 using acetic acid. It was applied to a 50 ml column containing S-Sepharose (Pharmacia) equilibrated with a buffer of 50 mmol sodium acetate pH 5.0. The pectate lyase bound and was eluted using a gradient of 0.5 M sodium chloride.<br>
Characterisation<br>
The pure enzyme gave a single band in SDS-PAGE of 3 5 kDa and an isoelectric point of around 6.1.<br>
The protein concentration was determined using a molar extinction coefficient of 57750 (based on the amino acid composition deducted from the sequence).<br>
Using the assay of detection the formation of cleavage by the formation of a double bound which can be measured at 23 5 nm the following data were obtained<br>
1. (conditions: pH 10; glycine buffer; no calcium; polygalacturonic acid Sigma P-1879 as substrate) : 1 jxmol  per min<br><br>
per mg.<br>
2. (conditions: pH 10; glycine buffer; no calcium; DE 35 (35% esterified pectin) as substrate): 4 mol per min per mg.<br>
The pure enzyme was dialysed against EDTA at pH 8.0 (20 mM tris pH 8.0, and at pH 10 (20 mM Glycine pH 10) and the enzyme analysed in Circular dichroism, no differences was seen in the spectra with and with out EDTA.<br>
Differential Scanning Calorimetry DSC of the 4 samples showed that the enzyme was most stable at pH 8.0 with a melting temperature around 70°C in Tris pH 8.0 and 75°C after dialysis against EDTA. At pH 10 the enzyme melted at 55°C with and without EDTA.<br>
The catalytic activity of the pectate lyase is inhibited by the presence of EDTA during Incubation with substrate but the enzyme dialysed against EDTA was still active if EDTA was omitted during incubation with substrate. Divalent cation like Fe++, Li++, Mg++, Cu++, Mn++ has no effect on the catalytic activity.<br>
The P-transelimination activity (using the lyase assay at 235 nm) at different pH values was determined as steady state kinetic at 40°C using the following buffers: pH 6.0: Na-MES 0.IM pH 6.5, 7.0 &amp; 7.5:  Na-MOPS 0.IM pH 8.0 &amp; 8.5:  Tris 0.IM<br>
pH 9.0, 9.5, 10.0 Sc 10.5: Na-glycine 0. IM pH: 11-11.5: Na-Carbonate 0.IM<br>
MES: From SIGMA number M-8250 (2[N-Morpholino] ethane sulfonic acid).<br>
MOPS: From SIGMA number M-1254 (3-[N-Morpholino] propane sulfonic acid).<br>
Tris: From Merck No. 1.08382 Glycine from MERCK and sodium carbonate from Merck No. 6392.<br>
The relative activity (rate) is calculated as percentage of the optimum activity, the following result was obtained:<br><br><br><br>
Activity in detergents: Using commercial detergents instead of buffer and incubation for 20 min. at 40
Immunological properties; At the Danish company DAKO, rabbit polyclonal monospecific serum was raised against the highly purified pectate lyase using conventional techniques. The serum formed a nice single precipitate in agarose gels with the pectate lyase of the invention and only one precipitation arch against Bacillus licheniformis  crude products like Pulpzyme HC batch no. CKF0054 or batch no. CKN00009 from Novo Nordisk A/S.<br>
EXAMPLE 2<br>
Cloning, expression, purification and characterization of a<br>
pectate lyase (I) from Bacillus licheniformis<br><br>
genqmic DNA PyeparatiOD<br>
The strain ATCC 14580 (identical to Bacillus  lichenxformis  DSM 8721) was propagated in liquid medium 3 as specified by ATCC (American Type Culture Collection, USA) . After 18 hours incubation at 37°C and 300 rpm, the cells were harvested, and genomic DNA was isolated by the method described in Pitcher et al.    [Pitcher,   D.   G.,   Saunders,   N.  A.,   Owen,   R.   J;   Rapid extraction of bacterial genomic DNA with guanidium thiocyanate; Lett Appl Hicrobiol 1989 8 151-156].<br>
Definition of the sequencee of the invention<br>
The sequence is defined by the following two primers which can be used in a subseqiient PCR reaction for amplification of the entire open reading frame of the Pectate lyase of the invention:<br>
Pecl3.orf.PstI<br>
5'-CAC ATC TGC AGC ATG AAG AGA TTA GCA GGT ACG GTT ATT TTG TC-3'<br>
Pecl3.Licheniformis.lower.NotI<br>
5'- CTC ATC ATG CGG CC.G  CAG GGG CCT TTA TTT GCA ATC AGT G -3' .<br>
Restriction sites for cloning purposes are underlined.<br>
The entire orf can be cloned using the above primers in a PCR carried out as described in example 1. The appearance of a DNA fragment size 0.7 kb indicates proper amplification of the gene segment. This DNA fragment can be cloned in any vector suitable for cloning in E.coli,   B.Buhtilia  or other. The fragment is cloned as a Pstl-Noti fragment. When performing DNA sequencing on the PCR fragment or cloned fragment, the openreading frame of the DNA sequence thus appearing is shown in SEQ ID No. 4.<br>
Subcloning and expression of Pectate Ivass ix\  g.gyfetilig<br>
The Pectate lyase encoding DNA sequence of the invention was PCR amplified using the PCR primer set consisting of these two oligo nucleotides:<br>
Pecl3.Licheniformis.upper.PstI<br><br>
5'- CAC ATC TGC AGC CGC GGC AGO CGA GGT CGT TCA CAA AAC G - 3' Pecl3.Licheniformis.lower.NotI<br>
5'- CTC ATC ATQ CGG CCG  CAG GGG CCT- TTA TTT GCA ATC AGT G -3' Restriction sites PstI and NotI are underlined.<br>
Chromosomal DNA isolated from Bacillus licheniformis  as described above was used as template in a PCR reaction carried out as described in example 1. The appearance of a DNA fragment size 0.6 kb indicated proper amplification of the gene segment.<br>
Subcloning of PCR fragment was carried out as described in example 1 except that the purified PCR fragment was digested with PstI and NotI. Several clones were analysed by isolating plasmid DNA from overnight culture broth.<br>
One such positive clone was restreaked several times on agar plates as used above, this clone was called MB750. The clone MB750 was grown overnight in TY-10 /g/ml kanamycin at 37°C, and next day 1 ml of cells were used to isolate plasmid from the cells using the Qiaprep Spin Plasmid Miniprep Kit #27106 according to the manufacturers recommendations for B.subtilis  plasmid preparations. This DNA was DNA sequenced and revealed the DNA sequence corresponding to the mature part of the Pectate lyase, i.e. positions 85-666 bp of the appended SEQ ID NO:3. The derived protein sequence is listed in SEQ.N0:4.<br>
Fermentation and purification<br>
The clone MB 750 obtained as described above was grown in<br>
200ml BPX media with 10 9/1 of Kanamycin in 500ml two baffled<br>
shakeflasks for 5 days at 37°C at 300 rpm.<br>
Flocculation was done using cationic flocculation agent<br>
C521 (10% solution) and 0.1% solution of anionic agent A130: To<br>
1300 ml of fermentation medium at pH 6.0, 10 ml of C521 (10%)<br>
simultaneous with 20 ml of A130(0.1%) was added under stirring<br>
at room temperature. The flocculated material was separated by<br>
centrifugation using a Sorval RC 3B centrifuge at 10,000 rpm for<br>
30 minutes. The liquid was concentrated into 200 ml using<br>
filtron ultrafiltration with a MW cut off of 10 kDa. This<br>
product was used for application trials after stabilization<br>
using 40% Glycerol (batch #9845 containing 193 Trans Units per<br>
ml or 1.97 mg active pectate lyase per ml).<br><br>
Highly purified enzyme was obtained using Cationic chromatography S-Sepharose column at pH 5.5 using a 25 mM Sodium Acetate buffer) the enzyme eluted using a NaCl gradient, the final purification step was a size chromatography on a Superdex 200 column run in a 0.1 M Sodium acetate buffer.<br>
Charaterisation<br>
The pure enzyme has a MW of 22 kDa and a pi of 6.2. The temperature optimum (relative activity) at pH 10 is 40°C.<br>
The relative activity is higher than 50% between pH 9.5 and 10.5 at a temperataure of 40°C.<br>
DSC of the enzyme at pH 6.0 gave a unfolding temperature of 61°C.<br>
The N-terminal of the purified pectate lyase has the following sequence: AEWHKTIV {starting at position 29 of the amino acid sequence SEQ ID N0:4).<br>
This enzyme belongs to family 3 of glycosyl hydrolases.<br>
EXAMPLE 3<br>
Cloning, expression, purification and characterization of a<br>
pectin lyase (III) from Bacillus licheniformis<br>
Cloning of Bacillus  licheniformis  Pectin lyase encoding gene.<br>
Subcloning and expression of a B.licheniformis  Pectin lyase in B,8ubtili&amp;;<br>
The Pectin lyase encoding DNA sequence SEQ ID no. 1 of the invention was PCR amplified using the PCR primer set consisting of the following two oligo nucleotides: Pect.upper.PstI<br>
5'-CAT AAA TCT GCA GCC GCG GCA GCA AAC GAA GAT TAT CCG GAA C -3' Pect.lower.NotI<br>
5'-GAA AGG AAA AGC GGC CGC  CAA ATA TTG AAA AGT GAG CGC AAT GTC G-3'<br>
Restriction sites PstI and NotI are underlined.<br>
Chromosomal DNA isolated from Bacillus  licheniformis  as described above was used as template in a PCR reaction carried out as described in example 1. The appearance of a DNA fragment approximate size of 1.5 kb indicated proper amplification of the<br><br>
gene segment.<br>
Subcloning of PCR fragment<br>
The subcloning was carried out as described in example 1 except that the purified PCR fragment was digested with PstI and NotI. Several clones were analyzed by isolating plasmid DNA from overnight culture broth.<br>
One such positive clone was restreaked several times on agar plates as used above, this clone was called MB588. The clone MB588 was grown overnight in TY-lOg/ml Kanamycin at 37°C, and next day 1 ml of cells were used to isolate plasmid from the cells using the Qiaprep Spin Plasmid Miniprep Kit #27106 according to the manufacturers recommendations for B.subtilis plasmid preparations. This DNA was DNA sequenced and revealed the DNA sequence corresponding to the mature part of the Pectin lyase in the appended DNA sequence SEQ ID N0:1 and corresponding to the protein sequence in the appended protein sequence SEQ ID NO: 2.<br>
Fermentation and purification<br>
The clone MB588 obtained as described above was grown in 2 5 x 200ml BPX media with 10 /xg/ml of Kanamycin in 500ml two baffled shakeflasks for 5 days at 37°C at 300 rpm.<br>
Flocculation was done using cationic flocculation agent C521 (10% solution) and 0.1% solution of anionic agent A130: To 2000 ml of fermentation medium pH 6.0 20 ml of C521 (10%) simultaneous with 40 ml of A130 was added under stirring at room temperature. The flocculated material was separated by centrifugation using a Sorval RC 3B centrifuge at 10,000 rpm for 30 minutes. The supernatant was clarified using Whatman glass filter number F.<br>
The liquid was concentrated into 300 ml, using filtron ultrafiltration with a MW cut off of 10 kDa, containing 2,262,000 Trans Units on Pectin. The filtrate was adjusted to pH 5.5 using acetic acid and applied to S-Sepharose column equilibrated with 50 mM Sodium acetate pH 5.5. The pectin lyase bound and was eluted as a pure protein using a NaCl gradient.<br>
Characterisation<br><br>
The pure enzyme has a MW of 55 kDa and a pi of 9.3.<br>
The relative activity is higher than 50% between pH 8.5 and 9.3 at a temperature of 40°C (pectin is not stable esterified above 9.3).<br>
This enzyme belongs to family 1 of glycosyl hydrolases.<br>
EXAMPLE 4<br>
Construction and expression of fusion protein between Pectate<br>
lyase and CBD.<br>
The CBD encoding DNA sequence of the CipB gene from Clostridium  thermocellum  strain YS (Poole D M; Morag E; Lamed R; Bayer EA; Hazlewood GP; Gilbert HJ (1992) Identification of the cellulose-binding domain of the cellulosome subunit SI from Clostridium thermocellum YS, Pems Microbiology Letters Vol. 78 , No. 2-3 pp. 181-186 was PCR amplified using the PCR primer set consisting of the following two oligo nucleotides:<br>
CIPCBD.upper.PECL.Sail<br>
5' -CGA CAA TGT CGA CAA TGT AAA ATC AAT CGT CAA GCA AAA TGC CGG AGT CGG CAA AAT CCA GCG CAG ACC GCC AAC ACC GAC CCC GAC TTC ACC GCC AAG CGC AAA TAG ACC GGT ATC AGG CAA TTT G -3'<br>
CIPCBD.lower.NotI<br>
5'-GCG TTG AGA CGC GCG GCC  GCT ATA CCA CAC TGC CAC CGG GTT CTT<br>
TAC-3'<br>
Restriction sites Sail and NotI are underlined.<br>
Chromosomal DNA encoding the CBD can be obtained as described in Poole DM; Morag E; Lamed R; Bayer EA; Hazlewood GP<br>
Gilbert HJ (1992) Identification of the cellulose-binding domain of the cellulosome subunit SI from Clostridium thermocellum YS, Ferns Microbiology Letters Vol. 78 , No. 2-3 pp. 181-186 and was used as template in a PCR reaction carried out as described in example 1. The appearance of a DNA fragment approximate size of 0.5 kb indicated proper amplification of the gene segment.<br>
Subcloning of PCR fragment<br><br>
The subcloning was carried out as described in example 1 except that the purified PCR fragment was digested with Sail and Notl. Several clones were analyzed by isolating plasmid DNA from overnight culture broth.<br>
One such positive clone was restreaked several times on agar plates as used above, this clone was called MB914. The clone MB914 was grown overnight in TY-lO/ig/ml Kanamycin at 37°C, and next day 1 ml of cells were used to isolate plasmid from the cells using the Qiaprep Spin Plasmid Miniprep Kit #27106 according to the manufacturers recommendations for B.subtilis plasmid preparations. This DNA was DNA sequenced and revealed the DNA sequence corresponding to the fusionprotein of: Pectate lyase-linker-cbd as represented in SEQ ID N0:9 and in the appended protein sequence SEQ ID NO:10.<br>
Expression and detection of Pectate-lyase-cbd fusion protein<br>
MB914 was incubated for 20 hours in TY-medium at 37°C and<br>
250 rpm. 1 ml of cell-free supernatant was mixed with 200 pil  of 10% Avicel (Merck, Darmstadt, Germany) in Millipore H20. The mixture was left for i hour incubation at 0°C. After this binding of Pectatelyase-Linker-CBD fusion protein to Avicel the Avicel with bound protein was spun 5 min at 5000g. The pellet<br>
was resuspended in 100 \il  of SDS-page buffer, boiled at 95°C for<br>
5 min, spun at 5000g for 5 min and 25 |j.l was loaded on a 4-20% Laemmli Tris-Glycine, SDS-PAGE NOVEX gel (Novex, USA). The samples were electrophoresed in a Xcell™ Mini-Cell  (NOVEX, USA) as recommended by the manufacturer, all subsequent handling of gels including staining with comassie, destaining and drying were performed as described by the manufacturer.<br>
The appearance of a protein band of approx. 55 kDa, indicated expression in B.suJbtiiis of the Pectatelyase-Linker-CBD fusion encoded on the plasmid pMB914.<br>
EXAMPLE 5 &gt; Pectate Lyase Treatment of Cellulosic Material: Effect of Temperature on Pectin Removal ajid Wettability<br>
A 100% cotton woven twill fabric, desized Test Fabric #428U, representing a typical cellulosic material, was treated<br><br>
with an aqueous enzyme solution comprising the J3. licheniformis pectate lyase of example 1,   dosed at 9 APSU/g fabric at pH 9 and at a 15:1 liquor ratio. Treatment time was 2 hours and temperature varied between 35-75°C. The fabric was rinsed well after the enzyme treatment, dried and then dyed with Ruthenium Red. The dye uptake was measured spectrophotometrically and is a measure of the residual pectin on the fiber. The percentage of residual pectin was calculated using the dye uptake of the starting material as 100% residual pectin and that of fully chemically scoured and bleached fabric as 0%. Results are shown in Table 6. Further, the wettability (drop test - measuring the time in seconds for a drop of water to become absorbed by the fabric) was measured and compared to a no enzyme control. Results are shown in Table 7.<br><br>
EXAMPLE 6<br>
Pectate Lyase Treatment of Cellulosic Material:<br>
Effect of pH on Pectin Removal<br>
A 100% cotton woven twill fabric, desized Test Fabric #428U, representing a typical cellulosic material, was treated with an aqueous enzyme solution comprising the B.   licheniformis pectate lyase of example 1, dosed at 9 APSU/g fabric at a 15:1 liquor ratio. Treatment time was 2 hours and the temperature 55°C. pH was varied between 8-11. The fabric was rinsed well<br><br>
after the enzyme treatment, dried and then dyed with Ruthenium Red. The dye uptake is measured spectrophotometrically and is a measure of the residual pectin on the fiber. The percentage of residual pectin was calculated using the dye uptake of the starting material as 100% residual pectin and that of fully chemically scoured and bleached fabric as 0%. The results are<br><br>
EXAMPLE 7<br>
Use of the enzyme of the invention in detergents<br>
The purified enzyme obtained as described in example 1 (batch 9751) showed improved cleaning performance when tested at a level of 1 ppm in a miniwash test using a conventional commercial liquid detergent. The test was carried out under conventional North American wash conditions.<br><br>
LITERATURE<br>
Lever, M. (1972) A new reaction for colortnetric determination of carbohydrates. Anal. Biochein, 47, 273-279.<br>
N. C. Carpita and D. M. Gibeaut (1993) The Plant Journal 3:1-30.<br>
Diderichsen, B., Wedsted, U., Hedegaard, L., Jensen, B. R., Sj0holm, C. (1990) Cloning of aldB, which encodes alpha-acetolactate decarboxylase, an exoenzyme from Bacillus brevis. J. Bacteriol. 172:4315-4321.<br>
Sakai et al., Pectin, pectinase and protopectinase: production, properties and applications, pp 213-294 in: Advances in Applied Microbiology vol:3 9,1993.<br><br><br><br><br><br><br><br><br><br>
WE CLAIM<br>
1.	An isolated pectate lyase which is<br>
i)        a polypeptide produced by Bacillus licheniformis, ATCC 14580, or<br>
ii)      a polypeptide comprising an amino acid sequence as shown in positions 1-341<br>
of SEO ID N0:8, or<br>
iii)     an analogue of the polypeptide defined in i) or ii) which is at least 70%<br>
identical to said polypeptide, or is derived from said polypeptide by substitution,<br>
deletion or addition of one or several amino acids, or is immunologically reactive<br>
with a polyclonal antibody raised against said polypeptide in purified form.<br>
2.	An isolated polynucleotide molecule encoding a polypeptide having pectate<br>
lyase activity selected from the group consisting of:<br>
a)	polynucleotide molecules comprising a nucleotide sequence as shown in SEQ ID NO: 7 from nucleotide 1 to nucleotide 1026;<br>
b)	polynucleotide molecules that encode a polypeptide that is at least 70% identical to the amino acid sequence of SEQ ID NO: 8 from amino acid residue 1 to amino acid residue 341; and<br>
c)	degenerate nucleotide sequences of (a) or (b).<br><br>
3.	The isolated polynucleotide molecule according to claim 2 wherein the polynucleotide is DNA.<br>
4.	An expression vector comprising the following operably linked elements: a transcription promoter; a DNA segment selected from the group consisting of<br><br>
a)	polynucleotide molecules encoding a polypeptide having pectate lyase activity comprising a nucleotide sequence as shown in SEQ ID N0:7 from nucleotide 1 to nucleotide 1026,<br>
b)	polynucleotide molecules encoding a polypeptide having pectate lyase activity<br><br>
that is at least 70% identical to the amino acid sequence of SEQ ID N0:8 from<br>
amino acid residue 1 to amino acid residue 341,<br>
c)       degenerate nucleotide sequences of (a) or (b); and a transcription terminator.<br>
5.	A cultured microbial cell, selected from the group of bacteria and fungal cells, into which has been introduced an expression vector according to claim 3, wherein said cell expresses the polypeptide encoded by the DNA segment.<br>
6.	An isolated polypeptide selected from the group consisting of:<br><br>
a)	polypeptide molecules having pectate lyase activity and comprising an amino acid sequence as shown in SEQ ID NO: 8 from residue 1 to residue 341;<br>
b)	polypeptide molecules having pectate lyase activity and which are at least 70% identical to the amino acids of SEQ ID NO: 8 from amino acid residue 1 to amino acid residue 341;<br>
7.	An enzyme preparation comprising a purified polypeptide according to claim 6.<br>
8.	A method of producing a polypeptide having pectate lyase activity comprising culturing a cell into which has been introduced an expression vector according to claim 4, whereby said cell expresses a polypeptide encoded by the DNA segment; and recovering the polypeptide.<br>
9.	An isolated enzyme having pectate lyase activity, in which the enzyme is (i) free from homologous impurities, and (ii) produced by the method according to claim 8.<br>
10.	The preparation according to claim 7 which further comprises one or more enzymes   selected   from  the  group  consisting  of proteases,    cellulases<br><br>
(endoglucanases), B-glucanases, hemicellulases, lipases, peroxidases, laccases, a-amylases, glucoamylases, cutinases, pectinases, reductases, oxidases, phenoloxidases, ligninases, pullulanases, arabinosidases, mannanases, xyloglucanases, xylanases, pectin acetyl esterases, polygalacturonases, rhamnogalacturonases, galactanases, pectin lyases, other pectate lyases, pectin methylesterases, cellobiohydrolases, transglutaminases; or mixtures thereof.<br>
11.	A detergent composition comprising the enzyme preparation according to<br>
claim 7 or the enzyme according to claim 1 and a surfactant.<br>
12.	A method for improving the properties of cellulosic fibres, yam, woven or non-woven fabric in which method the fibres, yam or fabric is treated with an effective amount of the preparation according to claim 7 or an effective amount of the enzyme according to claim 1.<br>
13.	The method according to claim 12, wherein the enzyme preparation or the enzyme is used in a scouring process step.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1NC1tYXMtMTk5OCBhYnN0cmFjdCBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">2654-mas-1998 abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1NC1tYXMtMTk5OCBhYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">2654-mas-1998 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1NC1tYXMtMTk5OCBhc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">2654-mas-1998 assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1NC1tYXMtMTk5OCBjbGFpbXMgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">2654-mas-1998 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1NC1tYXMtMTk5OCBjbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">2654-mas-1998 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1NC1tYXMtMTk5OCBjb3JyZXNwb25kZW5jZSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">2654-mas-1998 correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1NC1tYXMtMTk5OCBjb3JyZXNwb25kZW5jZSBwby5wZGY=" target="_blank" style="word-wrap:break-word;">2654-mas-1998 correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1NC1tYXMtMTk5OCBkZXNjcmlwdGlvbiAoY29tcGxldGUpIGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">2654-mas-1998 description (complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1NC1tYXMtMTk5OCBkZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">2654-mas-1998 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1NC1tYXMtMTk5OCBmb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">2654-mas-1998 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1NC1tYXMtMTk5OCBmb3JtLTE5LnBkZg==" target="_blank" style="word-wrap:break-word;">2654-mas-1998 form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1NC1tYXMtMTk5OCBmb3JtLTI2LnBkZg==" target="_blank" style="word-wrap:break-word;">2654-mas-1998 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1NC1tYXMtMTk5OCBmb3JtLTQucGRm" target="_blank" style="word-wrap:break-word;">2654-mas-1998 form-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1NC1tYXMtMTk5OCBmb3JtLTYucGRm" target="_blank" style="word-wrap:break-word;">2654-mas-1998 form-6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1NC1tYXMtMTk5OCBvdGhlcnMtMS5wZGY=" target="_blank" style="word-wrap:break-word;">2654-mas-1998 others-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1NC1tYXMtMTk5OCBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">2654-mas-1998 others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjY1NC1tYXMtMTk5OCBwZXRpdGlvbi5wZGY=" target="_blank" style="word-wrap:break-word;">2654-mas-1998 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="223534-a-process-for-optical-resolution-of-racemic-1-arylethyl-acetates.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="223536-a-crosslinkable-and-non-aromatic-polyurethane-potting-compound.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>223535</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2654/MAS/1998</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>47/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Nov-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>12-Sep-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>24-Nov-1998</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVOZYMES A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>KROGSHOEJVEJ 36, DK-2880 BAGSVAERD,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ANDERSEN, LENE, NONBOE</td>
											<td>LAKESEVEJ 11, DK-3450 ALLEROED,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SCHULEIN, MARTIN</td>
											<td>WIEDEWEKTSGADE 51, DK-2100 COPENHAGEN 0,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>LANGE, NIELS, ERIK, KREBS</td>
											<td>STONECLIFF DRIVE APT 5, RALEIGH, NC 27615,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>BJORNVAD, MADS, ESELUND</td>
											<td>DR. ABILDSGAARDS ALLE 8, 3TH., DK-1955 FREDERIKSBERG,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N9/88</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>1344/97</td>
									<td>1997-11-24</td>
								    <td>Denmark</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/223535-isolated-pectate-lyase-from-bacillus-licheniformis by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:32:29 GMT -->
</html>
