<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/223553-an-invitro-method-of-predicting-the-prognosis-of-a-biological-condition-in-animal-tissue by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:28:53 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 223553:AN INVITRO METHOD OF PREDICTING THE PROGNOSIS OF A BIOLOGICAL CONDITION IN ANIMAL TISSUE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">AN INVITRO METHOD OF PREDICTING THE PROGNOSIS OF A BIOLOGICAL CONDITION IN ANIMAL TISSUE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a method of predicting the prognosis of a biological condition in animal tissue, wherein the expression of genes is examined and correlated to standards. The inventiob further relates to the treatment of the biological condition and an assay for predicting the prognosis. In particular, the invention concerns gene expression in epithelial tissue, such as urinary bladder under both normal and abnormal conditions.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Technical field of the invention<br>
The present invention relates to a method of predicting the prognosis of a biological condition in animal tissue, wherein the expression of genes is examined and correlated to standards. The invention further relates to the treatment of the biological condition and an assay for predicting the prognosis.<br>
Background<br>
The building of large databases containing human genome sequences is the basis for studies of gene expressions in various tissues during normal physiological and pathological conditions. Constantly (constitutively) expressed sequences as well as sequences whose expression is altered during disease processes are important for our understanding of cellular properties, and for the identification of candidate genes for future therapeutic intervention. As the number of known genes and ESTs build up in the databases, array-based simultaneous screening of thousands of genes is necessary to obtain a profile of transcriptional behaviour, and to identify key genes that either alone or in combination with other genes, control various aspects of cellular life. One cellular behaviour that has been a mystery for many years is the malignant behaviour of cancer cells. It is now known that for example defects in DNA repair can lead to cancer but the cancer-creating mechanism in heterozygous individuals is still largely unknown as is the malignant cell's ability to repeat cell cycles to avoid-apoptosis to escape the immune system to invade and metastasize and to escape therapy. There are indications in these areas and excellent progress has been made, buth the myriad of genes interacting with each other in a highly complex multidimensional network is making the road to insight long and contorted.<br>
Similar appearing tumors - morphologically, histochemically, microscopically - can be profoundly different. They can have different invasive and metastasizing properties, as well as respond differently to therapy. There is thus a need in the art for methods which distinguish tumors and tissues on factors different than those currently in ciinicai use. The malignant transformation from normal tissue to cancer is believed to be a multistep process, in which tumorsuppressor genes, that normally repress cancer growth show reduced gene expression and in which other genes that encode tumor promoting proteins (oncogenes) show an increased expression level. Several tumor suppressor genes have been identified up till now, as e.g. p16, Rbp p53 ( Nesrin Ozflren and Wafik S. El-Deiry, Introduction to cancer genes and growth control, In: DNA alterations in cancer, genetic and epigenetic changes, Eaton publishing, Melanie Ehrlich (ed) p. 1-43,<br><br>
2000.; and references therein). They are usually identified by their lack of expression or their mutation in cancer tissue.<br>
Other examinations have shown this downregulation of transcripts to be partly due to loss of genomic material ( loss of heterozygosity), partly to methyiation of promctorregions, and partly due to unknown factors ( Nesrin Ozoren and Wafik S. E!-Deiry, Introduction to cancer genes and growth control, in: DNA alterations in cancer, genetic and epigenetic changes, Eaton publishing, Meianie Ehrlich (ed) p. 1-43, 2C00.; and references therein).<br>
Several oncogenes are known, e.g. cyciinD1/PRAD1/BCL1, FGFs, c-MYC, BCL-2 all of which are genes that are amplified in cancer showing an increased level of transcript ( Nesrin Ozoren and Wafik S, EJ-Deiry, Introduction to cancer genes and growth control, In: DNA alterations in cancer, genetic and epigenetic changes, Eaton publishing, Meianie Ehrlich (ed) p. 1-43, 2000.; and references therein). Many of these genes are related to cell growth and directs the tumor cells to uninhibited growth. Others may be related to tissue degradation as they e.g. encode enzymes that break down the surrounding connective tissue.<br>
Bladder cancer is the fourth most common malignancy in males in the western countries (Pisani). The disease basically takes two different courses: one where patients have multiple recurrences of superficial tumors (Ta and T1), and one where the disease from the beginning is muscle invasive (T2+) and leads to metastasis. About 5-10% of patients with Ta tumors and 20-30% of the patients with T1 tumors will eventually develop a higher stage tumor (Wolf). Patients with superficial bladder tumors represent 75% of ail bladder cancer patients and no clinical useful markers identifying patients with a poor prognosis exists at present.<br>
The patients presenting isolated or concomitant Carcinoma in situ (CIS) lesions have a high risk of disease progression to a muscle invasive stage (Aithausen). The CIS lesions may have a widespread manifestation in the bladder (field disease) and are believed to be the most common precursors of invasive carcinomas (Spruck, Rosin). The ability to predict which tumours are likely to recur or progress would have great impact on the clinical management of patients with superficial disease, as it would be possible to treat high-risk patients more aggressively (e.g. radical cystectomy or adjuvant therapy). This approach is currently not possible, as no clinical useful markers exist that identify these patients. Although many prognostic markers have been investigated, the most important prognostic factors are still disease stage, dysplasia grade and especially the presence of areas with CIS (Anderstrom, Cummings, Cheng). The gold standard for detection of CIS is urine cytology and histopathologic analysis of a set of selected site biopsies removed during routine<br><br>
cytsocopy examinations; however these prccsciures are not sufficient sensitive, tmplementing routine cytoscopy examinations with 5-ALA fluorescence imaging of the tumours and pre-cancerous lesions (CIS lesions and moderate dysplasia lesions) may increase the sensitivity of the procedure (Kriegmar), however, increased detection sensitivity is still necessary in order to offer better treatment regiments to the individual patients.<br>
Summary of the invention<br>
The present invention relates to prediction of prognosis of a biological condition, in particular to the prognosis of cancer such as bladder cancer. It is known that individuals suffering from cancer, although their tumors macroscopically and microscopically are identical, may have very different outcome. The present inventors have identified new predictor genes to classify macroscopically and microscopically identical tumors into two or more groups, wherein In each group has a separate risk profile of recurrence, invasive growth, metastasis etc. as compared to the other group(s). The present invention relates to genotyping of the tissue, and correlating the result to standard expression ievel(s) to predict the prognosis of the biological condition.<br>
Accordingly, in one aspect the present invention relates to a method of predicting the prognosis of a biological condition in animal tissue,<br>
comprising collecting a sample comprising cells from the tissue and/or expression products from the cells,<br>
determining an expression level of at least one gene in said sample, said gene being selected from the group of genes consisting of gene No. 1 to gene No. 562,<br>
correlating the expression level to at least one standard expression level to predict the prognosis of the biological condition in the animal tissue.<br>
The genes No. 1 - gene No. 562 are found in table A described below herein.<br>
Animal tissue may be tissue from any animal, preferably from a mammal, such as a horse, a cow, a dog, a cat, and more preferably the tissue is human tissue. The biological condition may be any condition exhibit'ng gene expression different from normal tissue. In particular the biological condition relates to a malignant or premalignant condition, such as a tumor or cancer, in particular bladder cancer. By the term "collecting a sample comprising cells" is meant the sample is provided in a manner, so that the expression level of the genes may be determined.<br><br>
Furthermore, the invention relates to a method of determining the stage of a biological condition in animal tissue,<br>
comprising collecting a sample comprising ceils from the tissue,<br>
determining an expression level of at least one gene in said sample, said gene being selected from the group of genes consisting of geneNo 1 to gene No. 562,<br>
correlating the expression level of the assessed genes to at least one standard levei of expression determining the stage of the condition.<br>
The determination of the stage of the biological condition may be conducted prior to the method of predicting the method, or the stage of the biological condition may as such contain the information about the prognosis.<br>
The methods above may be used for determining single gene expressions, however the invention also relates to a method of determining an expression pattern of a bladder cell sample, comprising:<br>
collecting sample comprising bladder ceils and/or expression products from bladder cells,<br>
determining the expression level of at least one gene in the sample, said gene being selected from the group of genes consisting of gene No. 1 to gene No. 562, and obtaining an expression pattern of the bladder cell sample.<br>
Further, the invention relates to a method of determining an expression pattern of a bladder cell sample independent of the proportion of submucosal, muscle, or connective tissue ceils present, comprising:<br>
determining the expression of one or more genes in a sample comprising cells, wherein the one or more genes exclude genes which are expressed in the submucosai, muscie, or connective tissue, whereby a pattern of expression is formed for the sample which is independent of the proportion of submucosai, muscle, or connective tissue cells in the sample.<br>
The expression pattern may be used in a method according to this information, and accordingly, the invention also relates to a method of predicting the prognosis a biological condition in human bladder tissue comprising,<br><br>
collecting a sample comprising cells from the tissue,<br>
determining an expression pattern of the cells as defined in any of claims 43-54, correlating the determined expression pattern to a standard pattern, predicting the prognosis of the biological condition of said tissue<br>
as wel! as a method for determining the stage of a biological condition in animal tissue, comprising<br>
collecting a sample comprising cells from the tissue, determining an expression pattern of the cells as defined above, correlating the determined expression pattern to a standard pattern, determining the stage of the biological condition is said tissue.<br>
The invention further relates to a method for reducing celf tumorigenicity or malignancy of a cell, said method comprising<br>
-contacting a tumor cell with at least one peptide expressed by at least one gene selected from the group of genes consisting of gene Nos. 200-214, 233, 234, 235, 236, 244? 249, 251, 252, 255, 256, 259? 261, 262, 266, 268, 269, 273, 274, 275, 276, 277, 279, 280, 281, 282, 285, 286, 289, 293, 295, 296, 299, 301, 304, 306, 307, 308, 311, 312, 313, 314 , 320 , 322, 323, 325, 326, 327, 328 , 330, 331, 332, 333, 334, 338, 341, 342, 343, 345, 348, 349, 350, 351, 352, 353, 355, 357, 360, 361, 363, 366, 367, 370, 373, 374, 375, 376, 385, 386, 387, 389, 390, 392, 394, 398, 400, 401, 405, 406, 407, 408, 410, 411, 412, 414, 415, 416, 418, 424, 426, 428, 433, 434, 435, 436, 438, 439, 440, 441, 442, 443, 445, 446, 453, 460, 461, 463, 464, 465, 466, 467, 469, 470, 471, 472, 473, 475, 476, 477, 479, 480, 481, 482, 483, 485, 486, 487, 488, 490, 492, 494, 496, 497, 498 , 499, 503, 515, 516, 517, 521, 526, 527, 528, 530 ,532, 533, 537, 539, 540, 541, 542, 543, 545, 554, 557, 560 or<br>
obtaining at least one gene selected from the group of genes consisting of gene Nos200-214, 233, 234, 235, 236, 244, 249, 251, 252, 255, 256, 259, 261, 262, 266, 268, 269, 273, 274, 275, 276, 277, 279, 280, 281, 282, 285, 286, 289,,293, 295, 296, 299, 301, 304, 306, 307, 308, 311, 312, 313, 314 , 320 , 322, 323, 325, 326, 327, 328 , 330, 331, 332, 333, 334,<br><br>
338, 341, 342, 343, 345, 348, 349, 350, 351, 352, 353, 355, 357, 360, 361, 353, 366, 367. 370, 373, 374, 375. 376. 385, 386, 387, 389, 390, 392, 394, 398, 400, 401. 405. 406, 407.<br>
408,	410, 411, 412, 414, 415, 416, 418, 424, 426, 428, 433, 434, 435, 436, 438. 439, 440.<br>
441, 442, 443, 445, 446, 453, 460, 461, 463, 464, 465, 466, 467, 469, 470, 471, 472, 473.<br>
475, 476, 477, 479, 480, 481, 482? 483, 485, 486, 487, 488, 490, 492, 494, 496, 497, 498 .<br>
499, 503, 515, 516, 517, 521, 526, 527, 528, 530 ,532, 533, 537, 539, 540, 541, 542, 543T<br>
545, 554, 557, 560, and introducing said at least one gene into the tumor ceil in a manner<br>
ailowing expression of said gene(s), or<br>
obtaining at least one nudeotide probe capable of hybridising with at least one gene of a tumor ceil, said at least one gene being selected from the group of genes consisting of gene Nos. 1-199, 215-232, 237, 238, 239, 240, 241, 242, 243, 245, 246, 247, 248, 250, 253, 254, 257, 258, 260, 263, 264, 265, 267, 270, 271, 272, 278, 283, 284, 287, 288, 290, 291, 292, 294, 297, 298, 300, 302, 303, 305, 309, 310, 315, 316, 317, 318, 319, 321, 324, 329, 335, 336, 337, 339, 340, 344, 346, 347, 354, 356, 358, 359, 362, 364, 365, 368, 369, 371, 372, 377, 378, 379, 380, 381, 382, 383, 384, 388, 391, 393, 395, 396, 397, 399, 402, 403, 404,<br>
409,	413, 417, 419, 420, 421, 422, 423, 425, 427 ,429, 430, 431, 432, 437, 444, 447, 448,<br>
449, 450, 451, 452, 454, 455 ,456, 457, 458, 459, 462, 468, 474, 478, 484, 489, 491, 493,<br>
495, 500, 501, 502, 504, 505, 506, 507, 508, 509, 510, 511, 512, 513, 514, 518 , 519, 520,<br>
522, 523, 524, 525, 529, 531, 534, 535, 536, 538, 544, 546, 547, 548, 549, 550, 551, 552,<br>
553, 555, 556, 558, 559, 561, 562, and introducing said at least one nudeotide probe into<br>
the tumor eel! in a manner allowing the probe to hybridise to the at least one gene, thereby<br>
inhibiting expression of said at least one gene.<br>
In a further aspect the invention relates to a method for producing antibodies against an expression product of a cell from a biological tissue, said method comprising the steps of<br>
obtaining expression product(s) from at least one gene said gene being expressed as defined above,<br>
immunising a mammal with said expression product(s) obtaining antibodies against the expression product.<br>
The antibodies produced may be used for producing a pharmaceutical composition. Further, the invention relates to a vaccine capable of eliciting an immune response against at least one expression product from at least one gene said gene being expressed as defined above.<br>
The invention furthermore relates to the use of any of the methods discussed above for producing an assay for diagnosing a biological condition in anima! tissue.<br><br>
Also, the invention relates to the use of a peptide as defined above as an expression product and/or the use of a gene as defined above and/or the use of a probe as defined above for preparation of a pharmaceutical composition for the treatment of a biological condition in animal tissue.<br>
In yet a further aspect the invention relates to an assay for determining the presence or ab-sence of a biological condition in animal tissue, comprising<br>
at least one first marker capable of detecting an expression level of at least ore gene selected from the group of genes consisting of gene No. 1 to gene No. 562,<br>
In another aspect the invention relates to an assay for determining an expression pattern of a bladder cell, comprising at least a first marker and and/or a second marker, wherein the first marker is capable of detecting a gene from a first gene group as defined above, and the second marker is capable of detecting a gene from a second gene group as defined above.<br>
Drawings<br>
Description of figures:<br>
Figure 1 Hierarchical cluster analysis of tumor samples based on 3,197 genes that show large variation across all tumor samples. Samples with progression are marked Prog.<br>
Figure 2 Delineation of the 200 best marker genes. Genes that show higher levels of expression in the non-progression group are shown in the top and genes that show higher levels of expression in the progression group is shown in the bottom. Each column in the diagram represents a tumor sample and each row a gene. The 13 non-progressing samples are shown to the left and the 16 progressing samples are shown to the right in the diagram. The color saturation indicates differences in gene expression across the tumor samples; light color indicates up regulation compared the median expression and down regulation compared to the median expression of the gene is shown in dark color. Gene names of particular interesting genes are listed. Notable, non-group expression patterns were observed for two tumors (arrows). The tumor in the no progression group (150-6) showed a solid growth pattern, which is associated with a poor prognosis. No special tumor characteristics can help explain the gene expression pattern observed for the tumor in the progression group (825-3).<br>
Figure 3. Cross-validation performance using from 1 to 200 genes.<br><br>
Figure 4. Predicting progression in early stage bladder tumors, a, The 45-gene expression signature found to be optirral for progression prediction. Genes showing high expression in progressing samples are srow in the top and genes showing high expression in the non-progressing samples are snown in the bottom. Genes are listed according to how many cross-validation loops included the genes, b, The 45-gene expression signature in the 19 tumor test-set. The samples are listed according to the correlation to the average ncn-progression signature from the training set samples. The read punctuatec line separates samples with positive (left) and negative (right) correlation values. The white lines separates samples above and below :he correlation cutoff values of 0.1 and -0.1. The sample legend indicates no-progression (N) samples and progression (P) samples.<br>
Figure 5 Hierarchical cluster analysis of the metachronous tumor samples. Tight clustering tumors of different stage from the same patients are colored in grey.<br>
Figure 6 Two-way hierarchical clustering and multidimensional scaling analysis of gene expression data from 40 biadder tumour biopsies, a, Tumour cluster dendrogram based on the 1767 gene-set. CIS annotations following the sample names indicate concomitant carcinoma in situ. Tumour recurrence rates are shown to the right of the dendrogram as + and ++ indicating moderate and high recurrence rates, respectively, while no sign indicates no or moderate recurrence, b, Tumour cluster dendrogram based on 88 cancer related genes, c, 2D plot of multidimensional scaling analysis of the 40 tumours based on the 1767 gene-set. The colour code identifies the tumour samples from the cluster dendrogram (Fig. 1a). d, Two-way cluster analysis diagram of the 1767 gene-set. Each row in the diagram represents a gene and each column a tumour sample. The colour saturation represents differences in gene expression across the tumour samples; Igiht color indicates higher expression of the gene compared to the median expression and lower expression of the gene compared to the median expression shown in dark color. The colour intensities indicate degrees of gene-regulation. The sidebars to the right of the diagram represent gene clusters a-j and normal 1-3 in the left side indicate the three normal biopsies and normal 4 indicates the pool of biopsies from 37 patients.<br>
Figure 7 Enlarged view of the gene clusters a, c, f, and g. The dendrogram at the top is identical to Fig. 6a. a, Cluster of transcription factors and other nuclear associated genes, c, Cluster of genes involved in proliferation and cell cycle control, f, Gene expression pattern and corresponding area with squamous metaplasia in urothelial carcinoma. Tne light colour indicates genes up-regulated in samples 1178-1 and 875-1, the only two samples with squamous cell metaplasia, g, Cluster of genes involved in angiogenesis and-matrix remodelling.<br><br>
Figure 8. Hierarchical duster analysis results<br>
here we show expanded views of clusters a-j as identified in the 1767 gene-cluster. The tumour cluster dendrogram and colour bars on top of the clusters represents the same tumour cluster as shown in the paper. The four samples to the left are normal biopsies (normal 1-3) and a pooi of 37 normal biopsies (normal 4).<br>
Figure 8a. Molecular classification of tumour samples using 80 predictive genes in each cross-validation loop. Each classification is based on the closeness to the mean in the three classes. Samples marked with * were not used to build the classifier. The scale indicates the distance from the samples to the classes in the classifier, measured in weighted squared Euclidean distance.<br>
Figure 9 Number of classification errors vs. number of genes used in cross-validation loops.<br>
Figure 10 Expression profiles of the 71 genes used in the final classifier model.The tumors shown are the 33 tumors used in the cross validation scheme. The Ta tumors are shown to the left, the T1 tumors in the middle, and the T2 tumors to the right.<br>
Figure 11 Number of prediction errors vs. number of genes used in cross-validation loops.<br>
Figure 12 The expression profiles of the 26 genes that constitute our final prediction model. The genes are listed according to the degree of correlation with the recurrence and non-recurrence groups. Genes with highest correlations are found in the top and the bottom of the list.<br>
Figure 13 . Hierarchical cluster analysis of the gene expression in 41 TCC, 9 normal samples and 10 samples from cystectomy specimens with CIS lesions, a, Cluster dendrogram of all 41 TCC biopsies based on the expression of 5,491 genes, b, Cluster dendrogram of all superficial TCC biopsies based on the expression of 5,252 genes, c, Two-way cluster analysis diagram of the 41 TCC biopsies together with gene expressions in the normal and cystectomy samples (left columns). Each row represents a gene and each column represent a biopsy sample. Yellow indicates up-regulation compared to the median expression (black) of the gene and blue indicates down-regulation compared to the median expression. The colour saturation indicates degree of gene regulation. The sidebars to the right of the diagram represent gene-clusters 1-4; enlarged views of cluster 1 and 4 are shown to the right, with all gene symbols listed.<br><br>
Figure 14 . Delineation of the 100 best markers that separate TCC without CIS from TCC with concomitant CIS. a, The 50 best up-regulated marker genes in TCC without CIS are shown in the top and the 50 best up-regulated marker genes in TCC with CIS are shown '.n the bottom. The gene symbols are listed to the right of the diagram, b, Expression profiles of the 100 marker genes in 9 normal biopsies (left column), 5 histoiogically normal sampies adjacent to CIS lesions (middle column), and 5 biopsies with CIS fesions detected, (right column).<br>
Figure 15 Cross validation performance using all samples<br>
Figure 16 Expression profiles of the 16 genes in the CIS classifier, a, the expression of 'he 16 classifier genes in TCC with no surrounding CIS (left) and in TCC with surrounding CIS (right). The gene symbols cf the classifier genes are fisted together with the number of ihe times used in cross-validation loops, b, the expression of the 16 classifier genes in normal samples, in histoiogically normal samples adjacent to CIS lesions, and in biopsies with CIS lesions. The top dendrogram shows the sample clustering from hierarchical cluster analysis based on the 16 classifier genes. The genes appear in the same order as in 3a.<br>
Figure 17 Cross validation performance using half of the samples<br>
Figure 18 shows table B<br>
Figure 19 shows table C<br>
Figure 20 shows table D<br>
Figure 21 shows table E<br>
Figure 22 shows table F<br>
Figure 23 shows table G<br>
Figure 24 shows table H<br>
Detailed description of the invention<br>
As discussed above the present invention relates to the finding that it is possible to predic: the prognosis of a biological condition by determining the expression level of one or more genes from a specified group of genes and comparing the expression level to at feast one<br><br>
standard for expression levels. The present inventors nave :jemified 562 genes relevant for predicting the prognosis of a biological condition, in particular a cancer disease, such as bladder cancer.<br>
The following table A shows the genes relevant in this context. Whenever a gene is cited herein with reference to a gene No. the numbering refers to the genes of Table A.<br>
'able A<br><br><br>
Gene	GeneChip	Probeset	Unigene	Unigene<br>
#			Build	<br>
1	HUGeneFL	AB000220_at	168	Hs.171921<br>
2	HUGeneFL	AF000231 at	168	Hs.75618<br>
3	HUGeneFL	D10922 s at	168	Hs.99855<br>
4	HUGeneFL	D10925 at	168	Hs.301921<br>
5	HUGeneFL	D11086_at	168	Hs.84<br>
6	HUGeneFL	D11151 at	168	Hs.211202<br>
7	HUGeneFL	D13435_at	168	Hs.426142<br>
8	HUGeneFL	D13666 s at	168	Hs.136348<br>
9	HUGeneFL	D14520 at	168	Hs.84728<br>
10	HUGeneFL	D21878 at	168-	Hs.169998<br>
11	HUGeneFL	D26443_at	168	Hs .371369<br>
12	HUGeneFL	D42046 at	168	Hs.194665<br>
13	HUGeneFL	D45370 at	168	HsJ4120<br>
14	HUGeneFL	D49372 s at	168	Hs.54460<br>
15	HUGeneFL	D50495 at	168	Hs.224397<br>
16	HUGeneFL	D63135 at	168	Hs.27935<br>
17	HUGeneFL	D64053 at	168	Hs.198288<br>
18	HUGeneFL	D83920_at	168	Hs.440898<br>
19	HUGeneFL	D85131_s_at	168	Hs.433881<br>
20	HUGeneFL	D86062 s at	168	Hs.413482<br>
21	HUGeneFL	D86479 at	168	Hs.439463<br>
22	HUGeneFL	D86957 at	168	Hs.307944<br>
23	HUGeneFL	D86959 at	168	Hs.105751<br>
24	HUGeneFL	D86976 at	168	Hs.196914<br>
25	HUGeneFL	D87433 at	168	Hs.301989<br>
26	HUGeneFL	D87443 at	168	Hs.409862<br>
27	HUGeneFL	D87682 at	168	Hs. 134792<br>
28	HUGeneFL	D89077 at	168	Hs.75367<br>
29	HUGeneFL	D89377 at	168	Hs.89404<br>
30	HUGeneFL	D90279 s at	168	Hs.433695<br>
31	HUGeneFL	HG1996-	168	_<br>
		HT2044 at		<br>
32	HUGeneFL	HG2090-	168	<br>
		HT2152 s at		<br>
33	HUGeneFL	HG2463-	168	_<br>
		HT2559 at		<br>
34	HUGeneFL	HG3044-	168	___<br>
		HT3742 s at		<br>
35	HUGeneFL	HG3187-	168	...<br>
		HT3366 s at		<br>
36	HUGeneFL	HG3342-	168	<br>
		HT3519 s at		<br>
37	HUGeneFL	HG371-	168	<br>
		HT26388_s__a		<br>
38	HUGeneFL	HG4069-	168	<br>
		HT4339 s at		<br>
39	HUGeneFL	HG67-	168	<br>
		HT67 f at		<br>
40	HUGeneFL	HG907-	168	<br><br>
description<br>
sema domain, immunoglobulin domain (lg), short basic domain, secreted, (semaphorin)<br>
3C<br>
RA311A, member RAS onccgene family formyl peptide receptor-iike 1 chemokjne (C-C motif) receptor 1 interleukin 2 receptor, gamma (severe combined immunodeficiency) endothelin receptor type A phosphatidylinositoi glycan, class F osteoblast specific factor 2 (fasciclin I-Iike) Kruppel-like factor 5 (intestinal) bone marrow stromal cell antigen 1 solute carrier family 1 (glial high affinity gluta-mate transporter), member 3 DNA2 DNA replication helicase 2-like (yeast)<br>
adipose specific 2<br>
chemokine (C-C motif) ligand 11<br>
transcription elongation factor A (Sll), 2<br>
tweety homolog 2 (Drosophila)<br>
protein tyrosine phcsphatase, receptor type, R<br>
ficolin (collagen/fibrinogen domain containing)<br>
1<br>
MYC-associated zinc finger protein (purine-<br>
binding transcription factor)<br>
chromosome 21 open reading frame 33<br>
AE binding protein 1<br>
likeiy ortholog of mouse septin 8<br>
Ste20-related serine/threonine kinase<br>
minor histocompatibility antigen HA-1<br>
stabilin 1<br>
sorting nexin 19<br>
KIAA0241 protein<br>
Src-!ike-adaptor<br>
msh homeo box homolog 2 (Drosophila) collagen, type V, alpha 1<br><br>
Classifier stage<br>
stage stage stage<br>
stage<br>
stage stage stage stage stage stage<br>
stage stage stage stags stage stage stage<br>
stags<br>
stage stage<br>
stage stage stage stage stage stage stage stage stage stage<br>
stage stage stage stage stage stace<br>
stage stage stage<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
The methods according to the invention may be used for determining any" biological condition, wherein said condition leads to a change in the expression of at least ore geno, and preferably a change in a variety of genes.<br>
Thus, the biological condition may be any malignant or premaiignant condition, in particular in bladder, such as a tumor or an adenocarcinoma, a carcinoma, a teratoma, a sarcoma, and/or a lymphoma, anc/or carcinoma-in-situ, and/or dysplasia-ln-situ.<br>
The expression level may be determined as single gene approaches, i.e. wherein the determination of expression from one or two or a few genes is conducted. It is however preferred that information is obtained from several genes, so that an expression pattern is obtained.<br>
In a preferred embodiment expression from at least one gene from a first group is determined, said first gene group representing genes being expressed at a higher !evel in one type of tissue, i.e. tissue in one stage or one risk group, in combination with determination of expression of at least one gene from a second group, said second group representing genes being expressed at a higher level in tissue from another stage or from another risk group. Thereby the validity of the prediction increases, since expression levels from genes from more than one group are determined.<br>
However,- determination of the expression or a single gene whether belonging to the first group or second group is also within the scope of the present invention. In this case it is preferred that the single gene is selected among genes having a high change in expression level from normal ceils to biological condition cells.<br>
Another approach is determination of an expression pattern from a variety of genes, wherein the determination of the biological condition in the tissue relies on information from a variety of gene expression, i.e. rather on the combination of expressed genes than on the information from single genes.<br>
The following data presented herein relates to bladder tumors, and therefore the description has focused on the gene expression level as one way of identifying genes that lose or gain function in cancer tissue. Genes showing a remarkable downregulation (or complete loss) or upregulation (gene expression gained de novo) of the expression level - measured as the mRNA transcript, during the malignant progression in bladder from normal mucosa through Ta superficial tumors, and Carcinoa in situ (CIS) to T1, slightly invasive tumors, to T2, T3 and T4 which have spread to muscle or even further into lymph nodes or other organs are within the scope of the invention, as well as. genes gaining importance during the differentiation from normal towards malignancy.<br><br>
The present invention relates to a variety of genes identified either by an 5ST identification number ar.c/or by a gene identification number. 3oth type of identification numbers relates to identification numbers of UniGene database, NC3I, build 18.<br>
The various genes have been identified using Affymetrix arrays of the following product<br>
numbers:<br>
HUGeneFL (sold in 2000-2002)<br>
EOS HuO3 (customized Affymetric array)<br>
U133A (product #900367 sold in 2003)<br>
Stage of a biadder tumor indicates how jteep the tumor has penetrated. Superficial turners are termed Ta, and Carcinoma in situ (CIS), and T1, T2, T3 and T4 are used to describe increasing degrees of penetration into the muscle. The grade of a bladder tumor is expressed on a scale of i-IV (1-4) according to Bergkvist, A.; ijungquist, A.; Moberger, B."Classification of bladder tumours basodf on the cellular pattern. Preliminary report of a clinical-pathological study of 300 cases with a minimum follow-up of eight years", Acta Chir Scand., 1965, 130(4):371-8). The grade reflects the cytological appearance of the cells. Grade ! cells are almost normal. Grade II cells are slightly deviant. Grade 111 cells are cieariy abnormal. And Grade IV cells are highly abnormal. A special form of bladder malignancy is carcinoma-in-situ or dyplasia-in-situ in whish the altered cells are located in-situ.<br>
It is important to predict the prognosis of a cancer disease, as superficial tumors may require a less intensive treatment than invasive lumors. According to the invention the expression level of genes may be used to identify genes whose expression can be used to identify a certain stage and/or the prognosis of the disease. These "Classifiers" are divided into those which can be used to identify Ta, Carciroma in situ (CIS), T1, and T2 stages as well as those identifying risk of recurrence or progression. In one aspect of the invention measuring the transcript level of one or more of these genes may lead to a classifier that can add supplementary information to the information obtained from the pathological classification. For example gene expression levels that signify a T2 stage will be unfavourable to detect in a Ta tumor, as they may signal that the Ta tumor has the potential to become a T2 tumor. The opposite is probably also true, that an expression level that signify Ta will be favorable to have in a T2 tumor. In that way independent information may be obtained from pathological classification and a classification based on gene expression levels is made.<br>
In the present context a standard expression level is the level of expression of a gene in a standard situation, such as a standard Ta tumor or a standard T2 tumor. For use in the present invention standard expression levels is determined for each stage as well as for each<br><br>
group of progression, recurrence, and other prognostic indices, it is then possible to compare the result of a determination of the expression level from a gene of a given biological condition with a standard for each stage, progression, recurrence and other inaices to obtain a classification of the bioiogicat condition.<br>
Furthermore, in the present context a reference patterne refers to the pattern of expression levels seen in standard situations as discussed above, and reference patterns may be uses as discussed above for standard expression levels.<br>
It is known from the histopathological classification of bladder tumors that seme information is obtained from merely classifying into stage and grade of tumor. Accordingly, in one aspect, the invention relates to a method of predicting the prognosis of the biological condition by determining the stage of the biological condition, by determining an expression level of 2: least one gene, wherein said gene is selected from the group of genes consisting of gene No 1 to gene No. 562. In this aspect infornruation about the stage reveils directly information about the prognosis as well. An example hereof is when a bladder turnor is classified as for example stage T2, then the prognosis for the bladder tumor is obtained directly from the prognosis related generally to stage T2 tumors. In a preferred embodiment the genes for predicting the prognosis by establishing the stage of the tumor may be selected from gene selected from the group of genes consisting of gene No. 1 to gene No. 188. More preferably the genes for predicting the prognosis by establishing the stage of the tumor may be selected from gene selected from the group of genes consisting of gene Nos. 18, 39, 40, 55, 58, 79, 86, 87, 88, 91, 93, 103, 105, 106, 121, 123, 125, 126, 136, 137, 140, 149, 156, 158. 161, 165, 166, 167, 175, 184, 187, 188.<br>
It is preferred that the expresison level of more one gene is determined, such as the expression level of at least two genes, such as the expression level of at least three genes, such as the expression level of at least four genos, such as the expression level of at least five genes, such as the expression level of at least six genes, such as the expression level of at least seven genes, such as the expression level of at least eight genes, such as the expression level of at ieast nine genes, such as tie expression level of at least ten genes, such as the expression level of at (east 15 genes, isuch as the expression level of at least 20 genes, such as the expression levels of at least 25 genes, such as the expression levels of at feast 30 genes, such as the expression level of 2\2 genes.<br>
As discussed above, in relation to bladder cancer the stages of a bladder tumor are selected from bladder cancer stages Ta, Carcinoma in situ, T1, T2, T3 and T4. In an embodiment the determination of a stage comprises<br><br>
assaying at least the expression of Ta stage gene from a Ta stage gene group, at least one expression of a CIS gene, at least one expression of T1 stage gene from a T1 stage gene group, at least the expression of T2 stage gene *rom a T2 stage gene group, and more preferably assaying at least the expression of Ta stage gene from a Ta stage gene group, at least one expression of a CIS gene, at least one expression of T1 stage gene from a Tl stage gene group, at least the expression of T2 stage gene from a T2 stage gene group, at least the expression of T3 stage gene from a T3 stage gene group, at least the expression of T4 stage gene from a T4 stage gene group wherein at least one gene from each gene group is expressed in a significantly different amcunt in that stage than in one of the other stages.<br>
Preferably, the genes selected may be a (|ene from each gene group being expressed :n a significantly higher amount in that stage than in one of the other stages as compared to normal controls, see for example Table B below.<br>
The genes selected may be a gene from each gene group being expressed in a significantly lower amount in that stage than in one of the other stages.<br>
In another embodiment the present invention reiates to a method of predicting the prognosis of a biological condition by obtaining information in addition to the stage classification as such. As described above, by determining <jene expression levels that signify a t2 stage in tumor otherwise classified as ta the signal has potential to become tumor. opposite is also true an level will be favorable have lumor. present invention inventors shown some genes are relevant for obtaining this additional information.></jene>
Also, in one embodiment the present invertion reiates to a further method of predicting the prognosis of a biological condition by obtairing information in addition to the stage classification as such. Determination of squamous netaplasia in a tumor, in particular in a T2 stage tumor, is indicative of risk of progression. In particular the genes may be selected from gene selected from the group of genes consisting of gene No. 215 to gene No. 232, see also table H.<br>
It is preferred that the expresison level of more one gene is determined, such as the expression level of at least two genes, such as the expression level of at least three genes, such as the expression level of at least four genes, such as the expression level of at least five genes, such as the expression level of at least six genes, such as the expression level of at least seven genes, such as the expression evel of at least eight genes, such as the expression level of at least nine genes, such as ths expression level of at least ten genes, such as the expression level of at least 15 genes, su:h as the expression level of 18 genes.<br><br>
In another embodiment the invention relates to genes bearing information of recurrence of the biological condition as such. In particular the genes may be selected rcm gene selected from the group of genes consisting of gene Mo. 139 to gene No. 214. it is preferred to determine a "first expression ievei of at least one gene from a first gene group, wrerein the gene from the first gene group is selected fnom the group of genes wherein expression is increased \n case of recurrence, genes No. 189 to gene No. 199 (recurrence genes), and ce-termined a second expression level of at least one gene from a second gene group, wherein the second gene group is selected from Ihe group of genes wherein expression is increased in case of no recurrence, genes No. 200 to No. 214 (non-recurrence genes), and correiate the first expression level to a standard expression level for progressors, and/or the second expression ievei to a standard expression level for non-progressors to predict the prognosis of the biological condition in the animal tissue, see also table C.<br>
It is preferred that the expresison level of more one gene is determined, such as the expression level of at least two genes, such as the expression level of at least three genes, such as- -the expression ievei of at least four genes, such as the expression level of at least five genes, such as the expression level of at east six genes, such as the expression level of at least seven genes, such as the expression level of at least eight genes, such as the expression level of at least nine genes, such as he expression level of at least ten genes, such as the expression level of at least 15 genes, such as the expression level of at least 20 genes, such as the expression level of at least 25 Irenes, such as the expression level of 26 genes.<br>
Furthermore, in another embodiment the invention relates to genes bearing information of progression as such. In particular the geres may be selected from the group of genes of table D, more preferably selected from the group of genes consisting of gene No. 233 to gene No. 446. More preferably the genes may be selected from the group of genes Ncs. 255, 273, 279, 280, 281, 282 , 287, 295, 300, 311, 317, 320f 333, 346, 347, 349, 352, 36<i see table e.></i>
!t is preferred that the expfesison Ievei of more one gene is determined, such as the expression level of at least two genes, such as the expression ievei of at least three genes, such as the expression Ievei of at least four geness, such as the expression Ievei of at least five genes, such as the expression level of at least six genes, such as the expression level of at least seven genes, such as the expression eve! of at least eight genes, such as the expression level of at least nine genes, such as th<s expression level of at least ten genes such as the sich levels></s>
 <br>
30 genes, such as the expression level pt at least ^o genes, suui as me ^Ajji^saiun iÂ«vei ui at least 40 genes, such as the expression level of 45 genes.<br>
Furthermore, it is within the scope of the invention to predict the prognosis of a biological condition in animal tissue by determining the expression level of at least two genes, by<br>
determining a first expression level of at least one gene from a first gene group, wherein the gene from the first gene group s selected ?rom the group of gene Nos. 237, 238r 239, 240, 241, 242, 243, 245, 246, 247, 248, 250, 253/254, 257, 258, 260, 263, 26^, 265, 267, 270, 271, 272, 278, 283, 284, 287, 288, 290r 291, 292, 294, 297, 298, 300, 302, 303, 305, 309f 310, 315, 316, 317, 318, 319, 321, 324, 329, 335, 336, 337, 339,<br>
340,	344, 346, 347, 354, 356, 358, 359, 362, 364, 365, 368, 369, 371, 372, 377, 378.<br>
379, 380, 381, 382, 383, 384, 388, 391, 393T 395, 396, 397, 399f 402, 403, 404, 4C9,<br>
413, 417, 419, 420, 421, 422, 423, 425, 427 ,429, 430, 431, 432, 437, 444 (progresses<br>
genes), and<br>
determining a second expression level of at least one gene from a second gene group, wherein the second gene group is selected from the group of genes Nos. 233, 234, 235, 236, 244, 249, 251, 252, 255, 256, i59, 261, 262, 266, 268, 269, 273, 274, 275, 276, 277, 279, 280, 281,282, 285, 286, 289, 293, 295, 296, 299, 301, 304, 306, 307, 308, 311, 312, 313, 314 , 320 , 322, 323, 325, 326, 327, 328 , 330, 331, 332, 333, 334, 338,<br>
341,	342, 343, 345, 348, 349, 350, 351, 352, 353, 355, 357, 360, 361, 363, 366, 367,<br>
370, 373, 374, 375f 376, 385, 386, 387, 389, 390, 392, 394, 398, 400, 401, 405, 406,<br>
407, 408, 410, 411, 412, 414, 415, 416, 418, 424, 426, 428, 433, 434, 435, 436, 438,<br>
439, 440, 441,442, 443, 445, 446 (non-progressor genes), and<br>
correlating the first expression level to a standard expression level for progressors, and/or the second expression level to a standard expression level for non-progressors to predict the prognosis of the biological condition in the animal tissue.<br>
In particular the genes of the first group anil the second group for predicting the prognosis of a Ta stage tumor may be selected from gene selected from the group of progression/non-progession genes described above.<br>
In yet another embodiment the present invention offers the possibility to predict the presence or absence of Carcinoma in situ in the sane organ as the primary biological condition. An example hereof is for a Ta bladder tumor to predict, whether the bladder in addition to the Ta tumor comprises Carcinoma in situ (CIS). [The presence of carcinoma in situ in a bladder containing a superficial Ta tumor is a signal that the Ta tumor has the potential of recurrence<br><br>
and invasiveness. Accordingly, by predicting the presence of carcinoma in situ important information about the prognosis is obtained. In the present context, genes for predicting the presence of carcinoma in situ for a Ta stage tumor may be selected from gene selected frcm the group of genes consisting of gene No. 447 to gene No. 552. More preferably the genes are selected from the group of genes consisting of gene Mos 447, 448, 449, 450, 451, 452,<br>
453,	454, 455, 456, 457, 458, 459, 460, 461, 462, 453, 464, 465, 466, ^57, 468, 469, 470,<br>
471, 472, 473, 474, 475, 476, 477, 478, 479, 480 ,481, 482, 483 ,484, -85, 486, 487, 488,<br>
489, 490, 491, 492, 493, 494, 495 , 496, 497, 498, 499, 500, 501, 502, 503, 504, 505, 5C6.<br>
507, 508, 509, 510, 511, 512, 513, 514, 515r 516, 517 ,518 ,519, 520; 521, 522 ,523, 524.<br>
525, 526, 527, 528, 529, 530, 531, 532, 533, 534, 535, 536, 537, 538, 539, 540, 541, 542.<br>
543, 544, 545, 546, see table F, or from the group of genes consisting of gene Nos. 547.<br>
548, 549, 550, 551, 552. 553, 554, 555, 556, 557, 558, 559, 560, 561, 562, see table G.<br>
it is preferred that the expresison level of more one gene is determined, such as the expression level of at least two genes, such as the expression level of at least three genes, such as the expression level of at least four genes, such as the expression level of at least five genes, such as the expression level of at east six genes, such as the expression ieve! of at least seven genes, such as the expression ieve! of at least eight genes, such as the expression ievei of at least nine genes, such as the expression level of at least ten genes, such as the expression level of at least 15 genes, such as the expression level of at least 20 genes, such as the expression levels of at least 2p genes, such as the expression levels of at least 30 genes, such as the expression level of at least 35 genes, such as the expression,level of at least 40 genes, such as the expression eve! of at [east 45 genes, such as the expression level of at least 50 genes, such as 100 genes. In another embodiment the expression level of 16 genes are determined.<br>
It is also preferred to determine a first expression level of at least one gene from a first gene group, wherein the gene from the first gene group is selected from the group of genes wherein expression is increased in case o1 CIS, genes Nos. 447, 448, 449, 450, 451, 452,<br>
454,	455 ,456, 457, 458, 459, 462, 468, 474, 478, 484, 489, 491, 493, 495, 500, 501, 502,<br>
504, 505, 506, 507, 508, 509, 510, 511, 512, 513, 514, 518 , 519, 520, 522, 523, 524, 525,<br>
529, 531, 534, 535, 536, 538, 544, 546, 547, 548, 549, 550, 551, 552, 553, 555, 556, 558,<br>
559, 561, 562 (CIS genes), and determined a second expression level of at least one gene<br>
from a second gene group, wherein the second gene group is selected from the group of<br>
genes wherein expression is increased in case of no CIS, genes Nos. 453, 460, 461, 463,<br>
464, 465, 466, 467, 469, 470, 471, 472, 473, 475, 476, 477, 479, 480, 481, 482, 483, 485,<br>
486, 487, 488, 490, 492, 494, 496, 497, 49fi , 499, 503, 515, 516, 517, 521, 526, 527, 528r<br>
530 ,532, 533, 537, 539, 540, 541, 542, 543 545, 554, 557, 560 (non-CIS genes), and corre<br>
late the first expression level to a standard [expression level for CIS, and/or the second ex-<br><br>
pression level to a stanaard expression Ifevel for non-CIS to predict the prognosis of the biological condition in the animal tissue.<br>
It is preferred when determining the expression level of at least one gene frcrn a first group and at least one gene from a second group that the expression level of more than one genes from each group is determined. Thus, it is preferred that the expresison level of more one gene is determined, such as the expression level of at least two genes, such as the expression level of at least three genes, such as the expression level of at feast four genes, such as the expression level of at least five genes such as the expression tevel of at ieast six genes, such as the expression ievel of at ieast seven genes, such as the expression ievel of at least eight genes, such as the expression level of at least nine genes, such as the expression level of at least ten genes in each group.<br>
In one embodiment of the invention the stage of the biological condition has been determined before the prediction of prognosis The stage may be determined by any suitable means such as determined by histological examination of the tissue or by genotyping of the tissue, preferably by genotyping of the tissue such as described herein or as described in WO 02/02804 incorporated herein by reference.<br>
In another aspect the invention relates to a method of determining the stage of a biological condition in animal tissue, -<br>
comprising collecting a sample compris ng cells from the tissue,<br>
determining an expression level of at least one gene selected from the group of genes consisting of gene No. 1 to gene No. 562,<br>
correlating the expression level of the assessed genes to at least one standard level of expression determining the stage of the condition.<br>
In particular the expression level of at least one gene selected from the group of genes consisting of gene Nos. 1-457 and gene Nos. 459-535 and gene Nos. 537-562.<br>
Specific embodiments of determining the stage is as described above for predicting prognosis by determination of stage.<br>
In a preferred embodiment the expression level of at least two genes is determined by<br><br>
determining the expression cf at least a first stage gene from a first stage gene group and at least a second stage gene from a second stage gene group, wherein at least cne of said genes is expressed in said finst stage of the condition in a higher amount than n said second stage, and the other gene is a expressed in said first stage of the condition in a lower amount than in said second stage of the condition, and<br>
correlating the expression level of the assessed genes to a standard level of expression determining the stage of the condition<br>
In general, genes being downregulated for higher stage tumors as well as for progression and recurrence may be of importance as predictive markers for the disease as ioss of one or more of these may signal a poor outcome or an aggressive disease course. Furthermore, they may be important targets for therapy as restoring their expression level, e.g. by gene therapy, or substitution with those peptide products or small molecules with a similar biological effect may suppress the malignant growth.<br>
Genes that are up-regulated (or gained de novo) during the malignant progression of bladder cancer from normal tissue through Ta, T1, T2, T3 and T4 is also within the scope of the invention. These genes are potential oncogenes and may be those genes that create or enhance the malignant growth of the cells. Trie expression level of these genes may serve as predictive markers for the disease course and treatment response, as a high level may signal an aggressive disease course, and they may serve as targets for therapy, as blocking these genes by e.g. anti-sense therapy, or by biochemical means could inhibit, or slow the tumor growth.<br>
The genes used according to the invention show a sufficient difference in expression from one group to another and/or from one stage to another to use the gene as a classifier for the group and/or stage. Thus, comparison of an expression pattern to another may score a change from expressed to non-expressed, ar the reverse. Alternatively, changes in intensity of expression may be scored, either increases or decreases. Any significant change can be used. Typical changes which are more than 2-fold are suitable. Changes which are greater than 5-fold are highly suitable.<br>
The present invention in particular relates tc methods using genes wherein at least a twofold change in expression, such as at least a three-fold change, for example at least a four fold change, such as at least a five fold change, for example at least a six fold change, such as at least a ten fold change, for example al least a fifteen fold change, such as at least a twenty fold change is seen between two groups.<br><br>
As described above the invention relates! to the use of information of expression levels, in one embodiment the expression patterns is obtained, thus, the invention relates to a method of determining an expression pattern of a bladder cell sample, comprising:<br>
collecting sample comprising bladder celis and/or expression products from biadder cells,<br>
determining the expression level of at least one gene in the sample, said gene being selected from the group of genes consisting of gene No. t to^gene No, 562, and obtaining an expression pattern of the bladder ceil sample.<br>
The invention preferably include more than one gene in the pattern, according it is preferred to include the expression level of at least two genes, such as the expression level of at ieast three genes, such as the expression level of at least four genes, such as the expression level of at least five genes, such as the expression level of more than six genes.<br>
The expression pattern preferably relates l:o one or more of the group of genes discussed above with respect to prognosis relating to stage, SSC, progression, recurrence and/or CIS.<br>
In order to predict prognosis and/or stages t is preferred to determine an expression pattern of a cell sample preferably independent of the proportion of submucosal, muscle and connective tissue cells present. Expression is determined of one or more genes in a sample comprising cells, said genes being selected from the same genes as discussed above and shown in the tables.<br>
It is an object of the present invention that characteristic patterns of expression of genes can be used to characterize different types of tissiue. Thus, for example gene expression patterns can be used to characterize stages and grades of bladder tumors. Similarly, gene expression patterns can be used to distinguish cells ha ing a biadder origin from other cells. Moreover, gene expression of cells which routinely contaminate bladder tumor biopsies has been identified, and such gene expression can be* removed or subtracted from patterns obtained from bladder biopsies. Further, the gene expression patterns of single-cell solutions of bladder tumor cells have been found to be substantially without interfering expression of contaminating muscle, submucosal, and connective tissue cells than biopsy samples.<br>
The one or more genes exclude genes which are expressed in the submucosal, muscle, and connective tissue. A pattern of expression is formed for the sample which is independent of the proportion of submucosal, muscle, and connective tissue cells in the sample.<br><br>
In another aspect of the invention a method of determining an expression pattern of a call sample is provided. Expression is determined of one or more genes in a sample comprising cells. A first pattern of expression is thiereby formed for the sample. Genes which are expressed ;n submucosal, muscle, and connective tissue ceils are removed from the -"irst pattern of expression, forming a second pattern of expression which is independent of the proportion of submucosal, muscle, and coijinective tissue ceils in the sample.<br>
Another embodiment of the invention provides a method for determining an expression<br>
i pattern of a bladder mucosa or bladder cancer ceil. Expression is determined of one or mere<br>
genes in a sample comprising bladder mucosa or bladder cancer cells; the expression<br>
determined forms a first pattern of expression. A second pattern of expression which was<br>
i<br>
formed us;ng the one or more genes andja sample comprising predominantly submucosai.<br>
i<br>
muscle, and connective tissue cells, is Subtracted from the first pattern of expression, forming a third pattern of expression. Thejthird pattern of expression reflects expression of the bladder mucosa or bladder cancer cells independent of the proportion of submucosal, muscle, and connective tissue cells present; in the sample.<br>
In one embodiment the invention provides; a method to predict the prognosis of a bladder tumor as described above. A first pattern of expression is determined of one or more genes in a bladder tumor sample. The first pattern is compared to one or more reference patterns of expression determined for bladder tumors at different stages and/or in different groups. The reference pattern which shares maximum similarity with the first pattern is identified. The stage of the reference pattern with the maxjimum similarity is assigned to the bladder turner sample.<br>
Yet another embodiment the invention provides a method to determine the stage of a bladder tumor as described above. A first pattern of expression is determined of one or more genes in a bladder tumor sample. The first pattern is compared to one or more reference patterns of expression determined for bladder tumors at different stages. The reference<br>
pattern which shares maximum similarity wi|h the first pattern is identified. The stage of the<br>
i<br>
reference pattern with the maximum similarity is assigned to the bladder tumor sample.<br>
Since a biopsy of the tissue often contains more tissue material such as connective tissue than the tissue to be examined, when the tidsue to be examined is epithelial or mucosa, the invention also relates to methods, wherein the expression pattern of the tissue is independent of the amount of connective tisspe in the sample.<br>
Biopsies contain epithelial ceils that most oftejn are the,targets for the studies, and in addition many other cells that contaminate the epithelial cell fraction to a varying extent. The<br><br>
contaminants induce nistiocytes, endothelijai ceils, leukocytes, nerve ceils, muscle cells etc. Micro dissection is the method of choice fir DNA examination, but in tie case of expression studies this procedure is difficult due to RNf degradation during the procedure. The epithelium may be gently removed and the expression ;r. the remaining submucosa and underlyirg connective tissue (the bladder wall) monitcjred. Genes expressed at high or iow levels in the bladder wall should be interrogated when performing expression monitoring of the mucosa and tumors. A similar apcroach could be used f6r studies of epithelia in other organs.<br>
in one embodiment cf the invention normal! mucosa lining the-bladder iumen from bladders -or cancer is scraped off. Then biopsies is taken from the denuded submucosa and connective tissue, reaching approximately 5 mm into jthe bladder wall, and immediately disintegrated 'n<br>
guanidinium isothlocyanate. Total RNA may be extracted, pooled, and poly(A)+ mRNA may be<br>
i<br>
prepared from the cool followed by conversion to double-stranded cDNA and in vitro transcription into cRNA containing biotin-labpled CTP and UTP.<br>
Genes that are expressed and genes that are not expressed in bladder wall can both interfere with the interpretation of the expression in a biopsy, and should be considered when interpreting expression intensities in tumor fciopsies, as the bladder wall component of a biopsy<br>
varies in amount from biopsy to biopsy.<br>
i<br>
When having-determined the pattern of (jenes expressed in bladder wall components said pattern may be subtracted from a pattern obtained from the sample resulting in a third pattern related to the mucosa (epithelial) cells.<br>
In another embodiment of the invention a method is provided for determining an expression pattern of a bladder tissue sample independent of the proportion of submucosai, muscle and connective tissue cells present. A singlelcell suspension of disaggregated bladder tumor cells is isolated from a bladder tissue samjple comprising bladder tumor cells is isolated from a bladder tissue sample comprising blajdder cells, submucosai cells, muscle cells, and connective tissue cells. A pattern of expression is thus formed for the sample which s independent of the proportion of submucosai, muscle, and connective tissue cells in the bladder tissue sample.<br>
Yet another method relates to the elimination of mRNA from bladder wail components before determining the pattern, e.g. by filtration land/or affinity chromatography to remove mRNA related to the bladder wall.<br>
WorKing with tumor material requires biopjsies or body fluids suspected to comprise relevant cells. Working with RNA requires freshly frozen or immediately processed biopsies, cr<br><br>
chemical oretreatment of the bioosy. Apairt from the cancer tissue, biopsies do inevitably contain many different cell types, such as cells present in the blood, connective and muscie tissue, endotheiium etc. in the case of DMA studies, microdissection or laser capture sre methods of choice, however the time-dependent degradation of RNA makes it difficult ;c<br>
perform manipulation of the tissue for m6re than a few minutes. Furthermore, studies of<br>
expressed sequences may be difficult on ijhe few cells obtained via microdissection or laser capture, as these cells may have an expression pattern that deviates from the predominant pattern in a tumor due :o large intratumoralj heterogeneity.<br>
In the present context high density expression arrays may be used to evaluate the impact of bladder wall components in bladder tumor biopsies, and tested preparation of single ceil solutions as a means of eliminating the I contaminants. The results of these evaluations permit for the design of methods of evaluating bladder samples without the interfering<br>
background noise caused by ubiquitous contaminating submucosal, muscle, and connective<br>
tissue cells. The evaluating assays of the invention may be of any type.<br>
While high density expression arrays can be used, other techniques are aiso contemplated. These include other techniques for assaying for specific mRNA species, including RT-PCR and Northern Blotting, as well as techniques for assaying for particular protein products, such as ELISA, Western blotting, and enzyme assays. Gene expression patterns according to the present invention are determined byj measuring any gene product of a particular gereT including mRNA and protein. A pattern may be for one or more genes.<br>
RNA or protein can be isolated and assayed from a test sample using any techniques known in the art. They can for example be isolated from a fresh or frozen biopsy, from formafin-fixeti tissue, from body fluids, such as biood, plasma, serum, urine, or sputum.<br>
Expression of genes may in general be detected by either detecting mRNA from the ceils and/or detecting expression products, such as peptides and proteins.<br>
The detection of mRNA of the invention hay be a tool for determining the developments!<br>
stage of a ceil type which may be definable by its pattern of expression of messenger RNA. For example, in particular stages of cells,,high levels of ribosomal RNA are found whereas relatively low levels of other types of messenger RNAs may be found. Where a pattern is<br>
shown to be characteristic of a stage, said| stage may be defined by that particular pattern cf<br>
messenger   RNA  expression.   The   mF^NA  population   is   a   good   determinant  of  a<br>
developmental stage, and may be correlated with other structural features of the ceil. In this<br>
manner, cells at specific developmental jstages will be characterized by the intracelluiar environment, as well as the extracellular environment. The present invention also allows the<br><br>
combination of definitions based in part upon antigens and in part upon mRNA expression. In one embodiment, the two may be combined in a single incubation step. A particular incubation condition may be found which jis compatible with both hybridization recognition and non-hybridization recognition molecules. Tnus, e.g. an incubation condition may be selected which allows both specificity of j antibody binding and specificity of nucleic acid<br>
hybridization. This allows simultaneous performance of both types of interactions on a single<br>
matrix. Again, where developmental mRN|\ patterns are correlated with structural features.<br>
or with probes which are able to hybridiz^ to imracellular mRNA populations, a ceil sorter may be used to sort specifically those c^ells having desired mRNA population patterns.<br>
It is within the general scope of the presenjt invention to provide methods for the detection of mRNA. Such methods often involve sample extraction, PCR amplification, nucleic ac:d fragmentation and labeling, extension reactions, and transcription reactions.<br>
The nucleic acid (either genomic DNA or niRNA) may be isolated from the sample according to any of a number of methods well known to those of skill in the art. One of skill will appreciate that where alterations in the copy number of a gene are to be detected gencmic DNA is preferably isolated. Conversely, where expression levels of a gene or genes are to be detected, preferably RNA (mRNA) is isqlated.<br>
Methods of isolating total mRNA are well known to those of skill in the art.  In one<br>
embodiment, the total nucleic acid is isolated from a given sample using, for example, an<br>
acid guanidinium-phenol-chloroform extraction method and polyA.sup. and mRNA is isolated<br>
by oligo dT column chromatography or by jjsing (dT)n magnetic beads (see, e.g., Sambrook<br>
et al., Molecular Cloning; A Laboratory IVfTanuai (2nd ed.). Vols. 1-3, Cold Spring Harbor Laboratory, (1989), or Current Protocols irj Molecular Biology, F. Ausubel et al., ed. Greene<br>
Publishing and Wiley-lnterscience, New Ydrk (1987)).<br>
The sample may be from tissue and/or tjody fluids, as defined elsewhere herein. Before<br>
analyzing the sample, e.g., on an oligonucleotide array, it will often be desirable to perform one or more sample preparation operations upon the sample. Typically, these sample preparation operations will include such manipulations as extraction of intracellular material, e.g.,  nucleic acids from  whole cell  sahnples,  viruses,  amplification  of nucleic acids,<br>
fragmentation, transcription, labeling and/or extension reactions. One or more of these<br>
various operations may be readily incorporated into the device of the present invention.<br>
DNA extraction may be relevant under circumstances where possible mutations in the genes<br>
are to be determined in addition to the determination of expression of the genes.<br><br>
For those embodiments where whole cellsjur other tissue samples are being analyzed, it wiii typically be necessary to extract the nucleic acids from the cells or viruses, prior to continuing with the various sample preparation operations. Accordingly, following sampie collection, nucleic acids may be liberated fifom the collected ceils, viral coat etc. into a crude extract followed by additional treatments t6 prepare the sample for subsequent operations,<br>
such as denaturation of contaminating (IBNA binding) proteins, purification, filtration and<br>
desalting.<br>
Liberation of nucleic acids from the sample cells, and denaturation of DNA binding proteins may generally be performed by physica or chemical methods. For example, chemical methods generally employ lysing agents 1o disrupt the ceils and extract the nucleic acids from the cells, followed by treatment of the extract with chaotropic salts such as guanidinium<br>
isothiocyanate or urea :o denature any contaminating and potentially interfering proteins.<br>
Alternatively, physical methods may be used to extract the nucleic acids and denature DNA binding proteins, such as physical protrusions within microchannels or sharp edged particles piercing ceil membranes and extract their] contents. Combinations of such structures with piezoelectric elements for agitation can proi/ide suitable shear forces for lysis.<br>
More traditional methods of ceil extraction jmay also-be used, e.g., employing a channel with<br>
restricted cross-sectional dimension whiclji causes cell lysis when the sample is passed<br>
through the channel with sufficient flow pressure. Alternatively, cell extraction and denaturing<br>
of contaminating proteins may be carried out by applying an alternating electrical current to<br><br>
the sample. More specifically, the sample) of cells is flowed through a microtubular array<br>
while an alternating electric current is applied across the fluid flow. Subjecting cells to ultrasonic agitation, or forcing cells through microgeometry apertures, thereby subjecting the cells to high shear stress resulting in rupture are also possible extraction methods.<br>
j Following extraction, it will often be desrable to separate the nucleic acids from other<br>
elements of the crude extract, e.g. denatured proteins, cell membrane particles and salts. Removal of particulate matter is generally accomplished by filtration or flocculation. Further where chemical denaturing methods are usied, it may be desirable to desalt the sample prior to proceeding to the next step. Desalting a: the sample and isolation of the nucleic acid may<br>
generally be carried out in a single step, 4-g. by binding the nucleic acids to a solid phase<br>
i<br>
and washing away the contaminating saltk or performing gel filtration chromatography on the sample passing salts through dialysisj membranes. Suitable solid supports for nucleic<br>
acid binding include e.g. diatomaceousj earth or silica (i.e.,  glass wool). Suitable gel<br>
i exclusion media also well known in the art may be readily incorporated into the devices cf<br><br>
the present invention and is commercialljy available from, e.g., Pharmacia and Sigma Chemical.<br>
Alternatively, desalting methods may generally take advantage of the high electrophoretic mobility and negativity of DNA compared \o other elements. Electrophoretic methods may aiso be utilized in the purification of nuclei^ acids from other cell contaminants and debris. Upon application of an appropriate electric jfleld, the nucleic acids present in the sample will migrate toward the positive electrode anp become trapped on the capture membrane. Sample impurities remaining free of the mjembrane are then washed away by applying an appropriate fluid flow. Upon reversal of the voitage, the nucleic acids are released from the membrane in a substantially purer form. Farther, coarse filters may also be overlaid on the barriers to avoid any fouling of the barrier^ by particulate matter, proteins co nucleic acids, thereby permitting repeated use.<br>
In a similar aspect, the high electrophoretic mobility of nucleic acids with their negative charges, may be utilized to separate nucleic acids from contaminants by utilizing a short        .    .<br>
column of a gel or other appropriate matrices or gels which will slow or retard the flow of<br>
other contaminants while allowing the fastej" nucieic acids to pass.<br>
This invention provides nucleic acid affinity matrices that bear a large number of different nucleic acid affinity ligands allowing the jsimultaneous selection and removal of a large number of preselected nucleic acids from! the sample. Methods of producing such affinity matrices are also provided. In general the rjiethods involve the steps of a) providing a nucleic acid amplification template array comprising a surface to which are attached at least 50 oligonucleotides  having  different nucieicj acid  sequences,  and  wherein  each  different<br>
oligonucieotide is localized in a predetermined region of said surface, the density of said<br>
oligonucleotides is greater than about 60 different oligonucleotides per 1 cm.sup.2, and all of said different oligonucleotides have an identical terminal 3* nucleic acid sequence and an identical terminal 51 nucleic acid sequenceL b) amplifying said multiplicity of oligonucleotides<br>
to provide a pool of amplified nucleic acids; and c) attaching the pool of nucleic acids to a<br>
solid support.	<br>
For  example,   nucleic   acid   affinity  chfomatography   is   based   on   the   tendency  of<br>
complementary, single-stranded nucleic adds to form a double-stranded or duplex structure<br>
through complementary base pairing. A hucleic acid (either DNA or RNA) can easily be attached to a solid substrate (matrix) wh^re it acts as an immobilized ligand that interacts with and forms duplexes with complementary nucleic acids present in a solution contacted to the immobilized ligand. Unbound components can be washed away from the bound complex to either provide a solution lacking the target molecules bound to the affinity column, or to<br><br>
provide the isolated target molecules themselves. The nucleic acids captured in a hybrid<br>
duplex can be separated and released fromjthe affinity matrix by denaturaticn either through heat, adjustment of salt concentration, <v the use of a destabilizing agent such as formamide tween.tm.-20 denaturing ageht or sodium dodecyl sulfate></v>
Affinity columns (matrices) are typically used either to isolate a single nucleic acid typically<br>
by providing a single species of affinity iigaijid. Alternatively, affinity columns bearing a single<br>
affinity ligand (e.g. oiigo dt columns) have tyeen used to isolate a multiplicity of nucleic acics<br>
where the nucleic acids all share a common sequence (e.g. a polyA).<br>
The type of affinity matrix used depends on the purpose of the analysis. For example, where it is desired to analyze mRNA expression levels of particular genes in a complex nucleic acid sample (e.g., total mRNA) it is often desirable to eliminate nucleic acids produced by genes that are constitutively overexpressed and thereby tend to mask gene products expressed at characteristically lower levels. Thus, in one embodiment, the affinity matrix can be used to remove a number of preselected genel products .(e.g., actin, GAPDH, etc.). This is accomplished by providing an affinity j matrix bearing nucleic acid affinity ligands complementary to the gene products (e.g., mRNAs or nucleic acids derived therefrom) or to subsequences thereof. Hybridization of trie nucleic acid sample to the affinity matrix will result in duplex formation between the affinity ligands and their target nucleic acids. Upon elution of the sample from the affinity matrix, the matrix will retain the duplexes nucleic acids leaving a sample depleted of the overexpressed target nucleic acids.<br>
The affinity matrix can also be used to identify unknown mRNAs or cDNAs in a sample. Where the affinity matrix contains nucleic acids complementary to every known gene (e.g., in a cDNA library, DNA reverse transcribed from an mRNA, mRNA used directly or amplified. or polymerized from a DNA template) in a sample, capture of the known nucleic acids by the affinity matrix leaves a sample enriched for those nucleic acid sequences that are unknown. In effect, the affinity matrix is used to perform a subtractive hybridization to isolate unknown nucleic acid sequences. The remaining "unknown" sequences can then be purified and<br>
sequenced according to standard method.<br>
The affinity matrix can also be used to capture (isolate) and thereby purify unknown nucleic acid sequences. For example, an affinity matrix can be prepared that contains nucleic acid (affinity ligands) that are complementary to sequences not previously identified, or net previously known to be expressed in a particular nucleic acid sample. The sample is then hybridized to the affinity matrix and those sequences that are retained on the affinity matrix are "unknown" nucleic acids. The retainec! nucleic acids can be eluted from the matrix (e.g.<br><br>
at increasec temperature, increased destabilizing agent concentration, or decreased salt)<br>
and the nucleic acids can then be sequence^ according to standard methods.<br>
Similarly, th affinity rratrix can be used toj efficiently capture (isolate) a number of known<br>
nucleic acid sequences. Again, the matrix is prepared bearing nucleic acids complementary<br><br>
to those nucleic acids it is desired to isolate. The sample is contacted to the matrix under<br>
conditions where the complementary nuclelicacid sequences hybridize to the affinity ligands in the matrix. The non-hybridized material is washed off the matrix leaving the desired sequences bound. The hybrid duplexes ard then denatured providing a pool of the isolated nucleic acids. The different nucleic acid^ in the pool can be subsequently separated according to standard methods (e.g. ge! elec[trophoresis).<br>
As indicated above the affinity matrices can be used to selectively remove nucleic acids from virtually  any sample containing  nucleic aids  (e.g.  in  a  cDNA library,   DNA reverse<br>
transcribed from an mRNA, mRNA used djrectly or amplified, or polymerized from a DNA<br>
template, and so forth). The nucleic acid^ adhering to the column can be removed by washing with a low salt concentration buffej\ a buffer containing a destabilizing agent such as formamide, or by elevating the column temperature.<br>
in one particularly preferred embodiment, :he affinity matrix can be used in a method to enrich a sample for unknown RNA sequences (e.g. expressed sequence tags (ESTs)). The<br>
method involves first providing an affinity matrix bearing a library of oligonucleotide probes<br>
specific to known RNA (e.g., EST) sequences. Then, RNA from undifferentiated and/or unactivated ceils and RNA from differentiated or activated or pathological (e.g., transformed)<br>
or otherwise having a different metabolic state are separately hybridized against the affinity<br>
i matrices to provide two pools of RNAs lacking the known RNA sequences.<br>
In a preferred embodiment, the affinity matrix is packed into a columnar casing. The sample is then applied to the affinity matrix (e.g. Injected onto a column or applied to a column by a pump such as a sampling pump driven by an autosampler). The affinity matrix (e.g. affinity column) bearing the sample is subjected toj conditions under which the nucleic acid probes comprising the affinity matrix hybridize specifically with complementary target nucleic acids. Such conditions are accomplished by mantaining appropriate pH, salt and temperature<br>
conditions to facilitate hybridization as discussed above.<br>
For a number of applications, it may be desirable to extract and separate messenger RNA from cells, cellular debris, and other contaminants. As such, the device of the present invention may, in some cases, include a mRNA purification chamber or channel in general, such purification takes advantage of the poly-A tails on mRNA. In particular and as noted<br><br>
above, poly- T oiigonucieotides may be iijnmociiized within a chamoer or channel of ;he device to serve as affinity iigands for mRNA. Poiy-T oligonucleotides may be immobilized upon a solid support incorporated within the chamber or channel, or alternatively, may be immobilized upon the surface(s) of the | chamber or channel itself. Immobilization of oligonucleotides on the surface of the chambers or channels may be carried out by methoas described herein inducing, e.g., oxidation aid silanation of the surface followed by standard DMT synthesis of the oiigonucieotides.<br>
In operation, the lysed sample Is introduced to a high salt solution to increase the ionic strength for hybridization, whereupon the mRNA will hybridize to the immobilized poly-T. The mRNA bound to the immobilized poiy-T olicjonucieotides is then washed free in a low ionic strength buffer. The poy-T oiigonucieotides may be immobiiiized upon poroussurfaces, e.g.. porous silicon, zeolites silica xerogeis, scintejred particles, or other solid supports.<br>
Following sample preparation, the sample cfan be subjected to one or more different analysis operations. A variety of analysis operations may generally be performed, including size based analysis using, e.g., microcapillary electrophoresis, and/or sequence based analysis using, e.g., hybridization to an oligonucleotice array.<br>
In the latter case, the nucleic acid sample may be probed using an array of oligonucleotice probes. Oligonucleotide arrays generally include a substrate having a large number of positionally distinct oligonucleotide probes attached to the substrate. These arrays may be produced using mechanical or light directed synthesis methods which incorporate a combination of photolithographic methods and solid phase oligonucleotide synthesis methods.<br>
The basic strategy for light directed synthessis of oligonucleotide arrays is as follows. The surface of a solid support, modified with photosensitive protecting groups is illuminated through a photolithographic mask, yielding reactive hydroxyl groups in the illuminated regions. A selected nucleotide, typically in the form of a 3'-0-phosphoramidite-activated deoxynucleoside (protected at the 5' hydroxyl with a photosensitive protecting group), is then presented to the surface and coupling occurs at the sites that were exposed to light. Following capping and oxidation, the substrate is rinsed and the surface is illuminated through a second mask to expose additional hydroxyl groups for coupling. A second selectee nucleotide (e.g., 5'-protected, 3'-O-phosphoramidite-activated deoxynucleoside) is presented to the surface. The selective deprotection and coupling cycles are repeated until the desired set of products is obtained. Since photolithography is used the process can be readily miniaturized to generate high density arrays of oligonucleotide probes. Furthermore, the sequence of the oligonucleotides at each site is known. See Pease et ai. Mechanical<br><br>
synthesis methods are similar to the light! directed methods except involving mechanical<br>
direction of fluids for deprotection and additipn in the synthesis steps.<br>
For some embodiments, oligonucleotide arrjays may be prepared having all possible probes of a given length. The hybridization pattern Jrf the target sequence on the array may be used to reconstruct the target DNA sequence. Hj/bridization analysis of large numbers of probes can be used to sequence long stretches of pNA or provide an oiigonucleotide array which is specific and complementary to a particular nucleic acid sequence. Fcr example, !n particularly preferred aspects, the oligonucjeotide array wili contain oligonucieotide probes which are complementary to specific target sequences, and individual or multiple mutations of these. Such arrays are particularly useful in the diagnosis of specific disorders which are characterized by the presence of a particular nucleic acid sequence.<br>
Following sample collection and nucleic acid extraction, the nucleic acid portion of the sample is typically subjected to one or more preparative reactions. These preparative reactions include in vitro transcription, labeling, fragmentation, amplification and-other reactions. Nucleic acid amplification increases the number of copies of the target nucleic acid sequence of interest. A variety of amplification methods are suitable for use in the methods and device of the present invention, including for example, the poiymerase chain reaction method or (PCR), the ligase chain reaction (LCR), self sustained sequence replication (3SR), and nucleic acid based sequence amplification (NASBA).<br>
The latter two amplification methods invove isothermal reactions based on isothermal<br>
transcription, which produce both single stranded RNA (ssRNA) and double stranded DNA (dsDNA) as the amplification products in a ratio of approximately 30 or 100 to 1, respectively. As a result, where these latter methods are employed, sequence analysis may be carried out using either type of substrate, i.e. complementary to either DNA or RNA.<br>
Frequently, it is desirable to amplify the nucleic acid sample prior to hybridization. One of skill in the art will appreciate that whatever! amplification method is used, if a quantitative result is desired, care must be taken to us;e a method that maintains or controls for the relative frequencies of the amplified nucleic acids.<br>
PCR<br>
Methods of "quantitative" amplification are well known to those of skill in the art. Fcr example, quantitative PCR involves simultaneously co-amplifying a known quantity of a control sequence using the same primers, "his provides an internal standard that may be used to calibrate the PCR reaction. The high density array may then include probes specific to the internal standard for quantification of the amplified nucleic acid.<br><br>
Thus, in cne embodiment, this invention provides for a method of optimizing a probe set *"or detection of a particular gene. Generally, this method involves providing a high density array containing a multiplicity of probes of one or more particular length(s) that are complementary to subsequences of the mRNA transcribed by the target gene. In one embodiment the high density array may contain every probe of a particular length that is complementary to a particular mRNA. The probes of the high density array are then hybridized with their target nucleic acid alone and then hybridized with a high complexity, high concentration nucleic acid sample that does not contain the targets complementary to the probes. Thus, -'or example, where the target nucleic acid is an RNA, the probes are first hybridized with -heir target nucieic acid alone and then hybridised with RNA made from a cDNA library (e.g., reverse transcribed poIyA.sup.* mRNA) where the sense of the hybridized RNA is opposite that of the target nucleic acid (to insure that the high complexity sample does not contain targets for the probes). Those probes that siow a strong hybridization signal with their target and little or no cross-hybridization with the high complexity sample are preferred probes for use in the high density arrays of this invention.<br>
PCR amplification generally involves the use of one strand of the target nucleic acid sequence as a template for producing a i^rge number of complements to that sequence. Generally, two primer sequences complenentary to different ends of a segment of :he complementary strands of the target sequence hybridize with their respective strands of the target sequence, and in the presence of polymerase enzymes and nucleoside triphosphates, the primers are extended along the target sequence. The extensions are melted from the target sequence and the process is repeated, this time with the additional copies of the target sequence synthesized in the preceding steps. PCR amplification typically involves repeated cycles of denaturation, hybridization and extension reactions to produce sufficient amounts of the target nucleic acid. The first step of each cycle of the PCR involves the separation of the nucleic acid duplex formed by the primer extension. Once the strands are separated, the next step in PCR involves hybridizing the separated strands with primers that flank the target sequence. The primers are then extended to form complementary copies of the target strands. For successful PCR amDlification, the primers are designed so that the position at which each primer hybridizes along a duplex sequence is such that an extension<br>
product synthesized from one primer, whesn separated from the template (complement),<br>
serves as a template for the extension of the other primer. The cycle of denaturaticn, hybridization, and extension is repeated as many times as necessary to obtain the desired amount of amplified nucleic acid.<br>
In PCR methods, strand separation is ndrmally achieved by heating the reaction to a sufficiently high temperature for a sufficient Ime to cause the denaturation of the duplex but<br><br>
not to cause an irreversible denaturationi of the polyrnerase. Typical heat denaturation involves temperatures ranging from about 80.degree. C. to 105.degree. C. for times ranging from seconds to minutes. Strand separatio^, however, can oe accomplished by any suitable denaturing method including physical, cherricat, or enzymatic means. Strand separation may be induced by a heiicase, for example, or an enzyme capable of exhibiting helicase activity.<br>
in addition to PCR and IVT reactions, the methods and devices of the oresent invention are also applicable to a number of other reaction types,  e.g.,  reverse  transaction,  nick<br>
translation, and the like.  '	<br>
The nucleic acids in a sample will generally be labeled to facilitate detection in subsequent steps. Labeling may be carried out during the amplification, in vitro transcription or nick translation processes. In particular, amplification, in vitro transcription or nick translation may incorporate a label into the amplified or transcribed sequence, either through the use of labeled primers or the incorporation of labeled dNTPs into the amplified sequence. Hybridization between the sample nucleic acid and the oligonucleotide probes upon the array is then detected, using, e.g., epifluorekcence confocal microscopy. Typically, sample is<br>
mixed during hybridization to enhance hybridization of nucleic acids in the sample to nuclecc<br>
acid probes on the array.	<br>
In some cases, hybridized oligonucieotides may be labeled following hybridization. For example, where biotin labeled dNTPs are used in, e.g. amplification or transcription, streptavidin linked reporter groups may be used to label hybridized complexes. Such operations are readily integratable into the| systems of the present invention. Alternatively, the nucleic acids in the sample may be labeled following amplification. Post amplification labeling typically involves the covalent attachment of a particular detectable group upon the amplified sequences. Suitable labels or detectable groups include a variety of fluorescent or radioactive labeling groups well known in the art. These labels may also be coupled to the<br>
sequences using methods that are well known in the art.<br>
Methods for detection depend upon the label selected. A fluorescent label is preferred<br>
because of its extreme sensitivity and simplicity. Standard labeling procedures are used to determine the positions where interactions between a sequence and a reagent take place. For example, if a target sequence is labeled and exposed to a matrix of different probes, only those locations where probes do interact with the target will exhibit any signal. Alternatively, other methods may be used to scan the matrix to determine where interaction takes place. Of course, the spectrum of interactions may be determined in a temporal manner by<br>
repeated scans of interactions which occur at each of a multiplicity of conditions. However,<br><br>
instead of testing each individual interaction separately, a multiplicity of sequence interactions may be simultaneously determined en a matrix.<br>
Means of detecting laceied target (sample) i)iuc!eic acids hybridized to the probes of the high<br>
density array are known to those of skill in 'the art. Thus, for example, where a colorimetric label is used, simple visualization of the label is sufficient. Where a radioactive labeled probe is used, detection of the radiation (e.g with photographic film or a solid state detector) is sufficient.<br>
In a preferred emboaiirent, however, the target nucleic acids are labeled with a fluorescent label and the localization of the label on the probe array is accomplished with fluorescent microscopy. The hybricized array is excited with a light source at the excitation wavelength of the particular fluorescent label and the resulting fluorescence at the emission wavelength is detected, in a particularly preferred embodiment, the excitation light source is a laser appropriate for the excitation of the fluorescent label.<br>
The target polynuclectide may be labeled iby any of a number of convenient detectable markers. A fluorescent label is preferred because it provides a very strong signal with low background. It is also ootically detectable a\ high resolution and sensitivity through a quick scanning procedure. Other potential labeling moieties include, radioisotopes, chemiluminescent compounds, labeled binding proteins, heavy metal atoms, spectroscopic<br>
markers, magnetic labels, and linked enzymejs.<br>
Another method for labeling may bypass any label of the target sequence. The target may be exposed to the probes, and a double strand hybrid is formed at those positions only. Addition of a double strand specific reagent will detect where hybridization takes place. An intercalate dye such as ethidium bromide may be used as long as the probes themselves do not fold back on themselves to a significant extent forming hairpin loops. However, the length of the hairpin loops in short oligonucleotide probes would typically be insufficient to form a stable duplex.<br>
Suitable chromogens will include molecules and compounds which absorb light in a distinctive range of wavelengths so that a color may be observed, or emit light when irradiated with radiation of a particular wave length or wave length range, e.g., fluorescers. Biliproteins, e.g., phycoerythrin, may also serve as labels.<br>
A wide variety of suitable dyes are availabld, being primarily chosen to provide an intense color with minimal absorption by their surroundings. Illustrative dye types include quinoline dyes, triarylmethane.dyes, acridine dyes, alizarine dyes, phthaleins, insect dyes, azo dyes, anthraquinoid dyes, cyanine dyes, phenazathionium dyes, and phenazoxonium dyes.<br><br>
A wide variety of fluorescers may be employed either by themselves or in conjunction with quencher molecules. Fluorescers of interest fall -nto a variety of categories having certain primary functionalities. These primary functionalities include 1- and 2-aminonaphthalene, p,p'-diaminostiibenes, pyrenes, quaternary phenanthridine salts, 9-amincacridinesdiaminobenzophenone imines, anthracanes, oxacarbocyanine, merocyanine, 3-aminoequilenin, perylene, bis-benzoxazole, bis-p-oxazolyl benzene, 1,2-benzophenaz:n, retinoi, bis-3-aminopyridinium salts, hellebrigenin, tetracyciine, sterophenoi. benzimidzaolyiphenyiamine, 2-oxo-3-chroJnen, indoJe, xanthen, 7-hydroxycoumar:n, phenoxazine, salicyiate, strophanthidfn, porphyrins, triarylmethanes and flavin. Individual<br>
fluorescent compounds which have functionalities for Sinking or which can be modified :c<br>
incorporate such functionalities include, e.g., dansyl chloride; fluoresceins such as 3,3-dihydroxy-9-phenylxanthhydrol; rhodamlneisothiocyanate; N-phenyl 1-amino-3-sulfonatonaphthalene; N-phenyi 2-anpino-6-suifonatonaphthaiene; 4-acetamido---isothiocyanato-stilbene-2,2'-disulfonic acid; pyrene-3-sulfonic acid; 2-toluidinonaphthalene-5-sulfonate; N-phenyl, N-methyl 2-aminoaphtnalene-6-sulfonate; ethidium bromide; stebrine; auromine-0,2-(9'-anthroyi)palmitate; dansyl phosphatidylethanolamine; N,N'-dioctadecyi oxacarbocyanine; N.N'-dihexyl oxacarbocyanine; merocyanine, 4-(3'pyreny!)butyrate; d-3-aminodesoxy-equilenin; 12-(9'-anthroyl)steÂ£irate; 2-methylanthracene; 9-vinyianthracene; 2,2'-(vinylene-p-pheny!ene)bisbenzoxazole; p-bis&gt;2-(4-methy!-5-phenyl-oxazolyl)!benzene; 3-<br>
dimethylamino-1,2-benzophenazin; retinoi; bis(3'-aminopyridinium) 1,10-decandiyl diiodide;<br>
sulfonaphthylhydrazone of hellibrienin; chlbrotetracycline; N-(7-dimethylamino-4-methyl-2-<br><br>
oxo-3-chromenyl)nnaleimide;	enzimidazo(yl)-phenyl!maleimide;	N-(4-<br>
fluoranthyl)maleimide; bis(acid); resazarin; 4-chloro-7-nitro-2,1,3-benzooxadiazole; merocyanine 540; resorufln; rose bengal; and 2,4-diphenyl-3(2H)-furanone.<br>
Desirably, fluorescers should absorb light above about 300 nm, preferably about 350 nm, and more preferably above about 400 nm, usually emitting at wavelengths greater than about 10 nm higher than the wavelength of the light absorbed. It should be noted that the absorption and emission characteristics of the bound dye may differ from the unbound dye. Therefore, when referring to the various wavelength ranges and characteristics of the dyes, it is intended to indicate the dyes as employed and not the dye which is unconjugated and characterized in an arbitrary solvent.<br>
Fluorescers are generally preferred because: by irradiating a fluorescer with light, one can obtain a plurality of emissions. Thus, a single^ label can provide for a plurality of measurable events.<br><br>
Detectabie signal may also be provided by! chemiluminescent and bioluminescent sources. Chemiluminescent sources include a compDund which becomes electronically excited by a chemical -eaction and may then emit light which serves as the detectible signal cr donates energy to a fluorescent acceptor. A diverse number of families of compounds have beer found to provide chemiluminescence under a variety of conditions. One family of compounds is 2,3-dihydro-1,-4-phthalazinedione. The mbst popular compound is luminol, which is the 5-amino compound. Other members of the family include the 5-amino-5,7,8-trimeihoxy- anc the dimethylamino&gt;ca!benz analog. These compounds can be made to luminesce with alkaline hydrogen peroxide or calcium hypochlcrite and base. 'Another family of compounds is the 2,4,5-triphenylimidazoles, with lophins as the common name for the parent product. Chemiluminescent analogs include para-dimethylamino and -methoxy substituents. Chemiluminescence may also be obtained with oxalates, usually oxalyl active esters, e.g.. c-nitrophenyf and a peroxide, e.g., hydrogen peroxide, under basic conditions. Alternatively, iuciferins may be used in conjunctior with luciferase or lucigenins to provide bioluminescence.<br>
Spin labels are provided by reporter molecules with an unpaired electron spin which can be detected by electron spin resonance (ESR) spectroscopy. Exemplary spin labels include organic free radicals, transitional metal complexes, particularly vanadium, copper, iron, and manganese, and the iike. Exemplary spin labels include nitroxide free radicals.<br>
In addition, amplified sequences may be subjected to other post amplification treatments. For example, in some cases, it may be desirable to fragment the sequence prior to hybridization with an oiigonucleotide array, in order to provide segments which are more readily accessible to the probes, which avoic looping and/or hybridization to multiple probes. Fragmentation of the nucleic acids may generally be carried out by physical, chemical or enzymatic methods that are known in the art.<br>
Following the various sample preparation operations, the sample will generally be subjected to one or more analysis operations. Particularly preferred analysis operations include, e.g. sequence based analyses using an oiigonucleotide array and/or size based analyses using, e.g. microcapillary array electrophoresis.<br>
In some embodiments it may be desirable tc provide an additional, or alternative means for analyzing the nucleic acids from the sample<br>
Microcapillary array electrophoresis generally involves the use of a thin capillary or channel which may or may not be filled with a particular separation medium. Electrophoresis of a sample through the capillary provides a size based separation profile for the sample.<br><br>
Microcapillary array siectrcpnoresis generally provides a rapid method for size based sequencing, PCR product analysis and restriction fragment sizing. The high surface to volume ratio of these capillaries allows for tie application of higher electric fields across the capillary without substantial thermal variation across the capillary, consequently allowing  more rapid separations. Furthermore, when combined with confocal imaging methods these methods provide sensitivity in the range of attomoles, which is comparable to the sensitivity of radioactive sequencing methods.<br>
In many capillary electrophoresis methods, the capillaries e.g. fused silica capillaries or<br>
channels etched, machined or molded into planar substrates, are filled with an appropriate separation/sieving matrix. Typically, a variety of sieving matrices are known In the art may be used in the microcapiilary arrays. Examples of such matrices include, e.g. hydroxyethyi cellulose, poiyacrylamice and agarose. Gei matrices may be introduced and polymerized within the capillary channel. However, in some cases this may result in entrapment of bubbles within the channels which can interfere with sample separations. Accordingly, it is often desirable to place a preformed separat on matrix within the capillary channe((s), prior :o mating the planar elements of the capillary sortion. Fixing the two parts, e.g. through sonic welding, permanently fixes the matrix within the channel. Polymerization outside of the channels helps to ensure that no bubbles are formed. Further, the pressure of the welding<br>
process helps to ensure a void-free system.<br>
sized based analyses the capillary arrays may also be used in sequencing applications. In particular, ge! based sequencing techniques may be readily adapted for capillary array electrophoresis.<br>
In addition to detection of mRNA or as the sole detection method expression products from the genes discussed above may be detected as indications of the biological condition of the tissue. Expression products may be detected in either the tissue sample as such, or in a body fluid sample, such as blood, serum, plasma, faeces, mucus, sputum, cerebrospinai fluid, and/or urine of the individual.<br>
The expression products, peptides and -proteins, may be detected by any suitable technique<br>
known to the person skilled in the art.<br>
In a preferred embodiment the expression products are detected by means of specific antibodies directed to the various expression products, such as immunofluorescent and/or immunohistochemical staining of the tissue.<br><br>
immunchistochemical localization of expressed proteins may be carried out cy immunostaining of tissue sections from the single rumors to determine whicn ceils expressed the protein encoded by the transcript in question. The transcript levels may be used to select a group of proteins supposed to show variation :'rom sample to sample making a roucn correlation between the level of protein detected and the intensity of the transcript on :'r.e microarray possible.<br>
For example sections may be cut from paraffin-embedded tissue biccKs, mounted, ard deparaffinized by incubation at 80 CÂ° for 10 min. followed by immersion in heated oil at 50c Z for 10 min. (Estisol 312, Estichem A/S, Denmark) and rehydration. Antigen retrieval s achieved in TEG (TrisEDTA-Glycerol) buffer using microwaves at 900 W. The tissue sections may be cooled in the buffer for 15 min before a brief rinse in tap water. Endogenous peroxidase activity is blocked by incubating the sections with 1% H202 for 20 min. followed by three rinses in tap water, 1 min each. Th3 sections may then be soaked in PBS buffer r'cr 2 min. The next steps can be modified from the descriptions given by Oncogene Science Inc., in the Mouse immunohistochemistry Detection System, XHCO1 (UniTect, Uniondaie. NY, USA). Briefly, the tissue sections are incubated overnight at 4Â° C with primary antibccy (against beta-2 microgiobulin (Dako), cytokeratin 3, cystatin-C (both from Europa, US), junE, CD59, E-cadherin, apo-E, cathepsin E, vimentin, IGFII (all from Santa Cruz), followed by three rinses in PBS buffer for 5 min each. Afterwards, the sections are incubated with biotinylated secondary antibody for 30 min, rinsed three times with PBS buffer arc subsequently incubated with ABC (avidin-biotinlyiated horseradish peroxidase complex) fcr 30 min. followed by three rinses in PBS buffer.<br>
Staining may be performed by incubation with AEC (3-amino-ethyIcarbazoie) for 10 min. Tre tissue sections are counter stained with Mayers hematoxylin, washed in tap water for 5 min. and mounted with glycerol-gelatin. Positive and negative controls may be included in eac-staining round with all antibodies.<br>
In yet another embodiment the expression products may be detected by means cf conventional enzyme assays, such as ELISA methods.<br>
Furthermore, the expression products may be detected by means of peptide/protein chips capable of specifically binding the peptides and/or proteins assessed. Thereby EH expression pattern may be obtained.<br>
Assay<br><br>
In a turner aspect the invention relates jto an assay for predicting the prognosis of a<br>
biological condition in animal tissue, comprising<br>
at least one first marker capable of detecting an expression level of at least one gene selected from the group of genes consisting of gene No. 1 to gene No. 562.<br>
Preferably the assay njrther comprises mesms for correlating the expression level to at least one stancard expression level and/or at least one reference pattern.<br>
The means for correlating preferably includes one or more standard expression levels and/or reference patterns for use in comparing or correlating the expression levels or patterns obtained from a tumor under examination to the standards,<br>
Preferably the invention relates to an assa^ for determining an expression pattern of a bladder ceil, comprising at 'east a first marker and/or a second marker, wherein the first marker is capable of detecting a gene from a first gone group as defined above, and/or the second marker is capable of detecting a gene from a second gene group as defined above, correlating the first expression level and/or the second expression level to a standard level of the assessed genes to predict the prognosis of a biological condition in the animal tissue. The marker(s) are preferably specifically delecting a gene as identified herein.<br>
As described above, it is preferred to determine the expression level from more than one<br>
gene, and correspondingly, it is preferred to include more than one marker in the assay, such as at least two markers, such as at least three markers, such as at least four markers, such as at least five markers, such as at least six markers, such as at least seven markers, such as at least eight markers, such as at least nine markers, such as at least ten markers, such as at least 15 markers.<br>
When using markers for at least two different groups, it is preferred that the above number of markers relate to markers in each group.<br>
As discussed above the marker may be any nucleotide probe, such as a DNA, RNA, PNA, or LNA probe capable of hybridising to mRNA indicative of the expression level. The hybridisation conditions are preferably as described below for probes. In another embodiment the marker is an antibody capable of specifically binding the expression product in question.<br>
Patterns can be compared manually by a person or by a computer or other machine. An algorithm can be used to detect similarities; and differences. The algorithm may score and compare, for example, the genes which are expressed and the genes which are not<br><br>
axpressec. Alternatively, the algorithm mayj lock for changes in intensity or' expression of a particular gene and score changes in intensity between two samples. Similarities may be determined on the basis of genes which are expressed in both samples and genes which sre not expressed in both samples or on the basis of genes whose intensity of expression are numerically similar.<br>
Generally, the detection operation will be performed using a reader device external to the diagnostic device. However, it may be desirable in some cases to incorporate the data gathering operation into the diagnostic devic9 itself.<br>
The detection apparatus may be a fluorescence detector, or a spectroscopic detector, or another detector.<br>
Although hybridization is one type of specific interaction which is clearly useful for use in this mapping embodiment antibody reagents may also be very useful.<br>
Gathering data from the various analysis operations, e.g. oligonucieotide and/or microcapiiiary arrays will typically be carrisd out using methods known in the art. For example, the arrays may be scanned using lasers to excite fluorescently labeled targets 'hat have hybridized to regions of probe arrays mentioned above, which can then be imaged using charged coupled devices ("CCDs") for a wide field scanning of the array. Alternatively, another particularly useful method for gathering data from the arrays is through the use of laser confocal microscopy which combines the ease and speed of a readily automated process with high resolution detection.<br>
Following the data gathering operation, the data will typically be reported to a data analysis operation. To facilitate the sample analysis operation, the data obtained by the reader from the device will typically be analyzed using a digital computer. Typically, the computer will be appropriately programmed for receipt and storage of the data from the device, as well as fcr analysis and reporting of the data gathered, i.e., interpreting fluorescence data to determine the sequence of hybridizing probes, normalization of background and singie base mismatch hybridizations, ordering of sequence data in SBH applications, and the like.<br>
The invention aiso relates to a pharmaceutics I composition for treating a biological condition, such as bladder tumors.<br>
In one embodiment the pharmaceutical composition comprises one or more of the peptices being expression products as defined above, In a preferred embodiment, the peptides are bound to carriers. The peptides may suitably be coupled to a polymer carrier, for example a<br><br>
protein carrier, such as 3SA. Such formulations are well-known to the person skilled in ihe art.<br>
The pep:ices may be suppressor peptides normally lost or decreased in tumor tissue administered in order to stabilise tumors towards a less malignant stage, in another embodiment the peptides are onco-peptides capable of eliciting an immune response towards the tumor cells.<br>
in another embodiment the pharmaceutical composition comprises genetic material, either<br>
genetic material for substitution therapy, or fir suppressing therapy as discussed below.<br>
In a third embodiment the pharmaceutical composition comprises at least one antibody produced as described aoove.<br>
In the present context the term pharmaceutical composition is used synonymously with the term medicament. The medicament of the invention comprises an effective amount of one or more of the compounds as defined above, or a composition as defined above in combination with pharmaceutically acceptable additives. Such medicament may suitably be formulated for oral, percutaneous, intramuscular, intravenous, intracranial, intrathecal, intracerebroven-tricular, intranasal or pulmona! administration. For most indications a localised or substantially localised application is preferred.<br>
Strategies in formulation development of mecicaments and compositions based on the compounds of the present invention generally correspond to formulation strategies for any other protein-based drug product. Potential problems and the guidance required to overcome these problems are dealt with in several textbooks, e.g. "Therapeutic Peptides and Protein Formulation. Processing and Delivery Systems", Ed. A.K. Banga, Technomic Publishing AG. Basel, 1995.<br>
Injectables are usually prepared either as liquid solutions or suspensions, solid forms suitable for solution in, or suspension in, liquid prior to injection. The preparation may also be emulsified. The active ingredient is often mixed with excipients which are pharmaceuticaily acceptable and compatible with the active ingredient. Suitable excipients aret for example: water, saline, dextrose, glycerol, ethanol or the like, and combinations thereof. In addition, if desired, the preparation may contain minor amounts of auxiliary substances such as wetting or emulsifying agents, pH buffering agents, or which enhance the effectiveness or transportation of the preparation.<br>
Formulations of the compounds of the invention can be prepared by techniques known to the<br><br>
person skilled in the art. The formulations rfiay contain pharmaceuticaily acceptable carriers and excipients including microspheres, liposomes, microcapsules and nancparticies.<br>
The preparation may suitably be administered by injection, optionally at the site, where the active ingredient is to exert its effect. Additional formulations which are suitable for other modes of administration include suppositories, and in some cases, oral formulations. For suppositories, traditional binders and carriers include polyalkylene glycols or triglycerides. Such suppositories may be formed from mixtures containing the active ingredient(s) in the range of from 0.5% to 10%, preferably 1-2%. Oral formulations include such normaily employed excipients as, for example, pharmaceutical grades of mannitol, lactose, starch, magnesium stearate, sodium saccharine, cellulose, magnesium carbonate, and the (ike. These compositions take the form of solutions, suspensions, tablets, pills, capsules, sustained release formulations or powders and generally contain 10-95% of the active ingredient(s), preferably 25-70%.<br>
The preparations are administered in a mamer compatible with the dosage formulation, and in such amount as wiii be therapeutically effective. The quantity to be administered depends on the subject to be treated, including, e.g. the weight and age of the subject, the disease to be treated and the stage of disease. Suitable! dosage ranges are of the order of several hundred pg active ingredient per administration with a preferred range of from about 0.1 pg to 1000 pg, such as in the range of from abou; 1 pg to 300 jjg, and especially in the range of from about 10 pg to 50 fjg. Administration may be performed once or may be followed by subsequent administrations. The dosage will also depend on the route of administration and will vary with the age and weight of the subject to be treated. A preferred dosis would be in the interval 30 mg to 70 mg per 70 kg body weight.<br>
Some of the compounds of the present invention are sufficiently active, but for some of the others, the effect will be enhanced if the preparation further comprises pharmaceuticaily acceptable additives and/or carriers. Such additives and carriers will be known in the art. in some cases, it will be advantageous to include a compound, which promote delivery of the active substance to its target.<br>
In many instances, it will be necessary to administrate the formulation multiple times. Administration may be a continuous infusion, sich as intraventricular infusion or administration in more doses such as more times a day, dail/, more times a week, weekly, etc.<br><br>
Vaccines	<br>
In a further embodiment the present invention relates to a vaccine for the prophylaxis cr treatment of a biological condition comprishg at least one expression product from at least one gene said gene being expressed as defined above.<br>
The term vaccines is used with its normal meaning, i.e preparations of immunogenic material for administration to induce in the recipient an immunity to infection or intoxication by a given infecting agent. Vaccines may be administered by intravenous injection or through oral, nasal and/or mucosal administration. Vaccines may be either simple vaccines prepared from one species of expression products, such as proteins or peptides, or a variety of expression products, or they may be mixed vaccines containing two or more simple vaccines. They ars prepared in such a manner as not to destroy the immunogenic material, aithough the methods of preparation vary, depending on the vaccine.<br>
The enhanced immune response achieved according to the invention can be attributable to e.g. an enhanced increase in the level of imnunocjlobulins or in the level of T-cel!s including cytotoxic T-ceils will result in immunisation of at least 50% of individuals exposed to said immunogenic composition or vaccine, such as at least 55%, for example at least 60%, such as at least 65%, for example at least 70%, for example at least 75%, such as at least 80%. for example at least 85%, such as at least $&gt;0%, for example at least 92%, such as at least 94%, for example at feast 96%, such as at (east 97%, for example at least 98%, such as at least 98.5%, for example at least 99%, for example at least 99.5% of the individuals exposed to said immunogenic composition or vaccine are immunised.<br>
Compositions according to the invention may also comprise any carrier and/or adjuvant known in the art including functional equivalents thereof. Functionally equivalent carriers are capable of presenting the same immunogenic determinant in essentially the same steric conformation when used under similar conditions. Functionally equivalent adjuvants are capable of providing similar increases in'the effcacy of the composition when used under similar conditions.<br>
Therapy<br>
The invention further relates to a method of treating individuals suffering from the biologicai condition in question, in particular for treating a bladder tumor.<br>
Accordingly, the invention relates to a method for reducing cell turnorigenicity or malignancy of a cell, said method comprising contacting a tumor ceil with at least one peptide expressed by at least one gene selected from the group of genes consisting of gene No. 200-214, 233, 234, 235, 236, 244, 249,  251, 252, 255, 256, 259, 261, 262, 266, 268, 269, 273, 274, 275,<br><br>
275, 277. 279, 280. 281, 282, 285, 286, 2^9, 293, 295, 296, 299, 301, 3Ca. 306, 307, 308, 311,312. 313, 314, 320, 322,323, 325.326,327,328 ,330,331, 332, 333,334,338, 34V 342, 343, 345, 348, 349, 350, 351, 352, 3c3t 355, 357, 360, 361, 363, 365, 367, 370, 373, 374, 375, 376, 385, 386f 387, 389, 390, 392, 394, 398, 400, 401, 405, 4C6, 407, 408, 410,<br>
411, 412. 414, 415, 416, 418, 424, 426, 4^8. 433, 434, 435, 436, 438, 439, 440, 441, 4^2.<br>
443, 445, 446, 453. 460, 461, 463, 464, 4^35, 466, 467T 469, 470, 471, 472, 473, 475, 475. 477, 479, 480, 481 f ^82, 483, 435, 486, 487, 488, 490, 492, 494, 496, 497, 498 , 499, 503, 515, 516. 517, 521, 526, 527, 528F 530 ,532, 533. 537, 539, 540, 541, 5-2, 543, 545, 55-. 557, 560.<br>
In order to increase the effect several different peptides may be used simultaneously, SUCH as wherein the tumor cell is contacted with at least two different peptides.<br>
In one embodiment the invention re!a:es to a method of substitution therapy, ;e. administration of genetic material generally expressed in normal ceils, but lost or decreasea jn biological condition cells (tumor suppressors). Thus, the invention relates to a method for reducing cell tumorigenicity or malignancy of a cell, said method comprising<br>
obtaining at least one gene selected from the group of genes consisting of gene No. 200-214, 233, 234, 235, 236, 244, 249, 251, 252, 255, 256, 259, 261, 262, 266, 268, 269, 273, 274, 275, 276, 277, 279, 280, 281, 282, 235, 286, 289, 293, 295, 296, 299, 301, 304, 3C6. 307, 308, 311, 312, 313, 314 , 320 , 322, 323, 325, 326, 327, 328 , 330, 331, 332, 333, 334, 338, 341, 342, 343, 345, 348, 349, 350, 351, 352, 353, 355, 357, 360, 361, 363, 366, 367, 370, 373, 374, 375, 376, 385, 386, 387, 339, 390, 392, 394, 398, 400, 401, 405, 406, 407. 408, 410, 411, 412, 414, 415, 416, 418, 424, 426, 428, 433, 434, 435, 436, 438, 439, 4^0, 441, 442, 443, 445, 446, 453, 460, 461, 433, 464, 465, 466, 467, 469, 470, 471, 472, 473, 475, 476, 477, 479, 480, 481, 482, 483, 4135, 486, 487, 488, 490, 492, 494, 496, 497, 498 , 499, 503, 515, 516, 517, 521, 526, 527, 528, 530 ,532, 533, 537, 539, 540, 541, 542, 543, 545, 554, 557, 560,<br>
introducing said at least one gene into the tumor cell in a manner allowing expression of said gene(s).<br>
In one embodiment at least one gene is introduced into the tumor cell. In another embodiment at leasi two genes are introduced into the tumor cell.<br>
In one aspect of the invention small molecules that either inhibit increased gene expression or their effects or substitute decreased gene expression or their effects, are Introduced to the cellular environment or the cells. Application of small molecules to tumor cells may be<br><br>
performed by e.g. local application or intravenous injection or by oral ingestion. Small molecules have "he aoiiity to restore function of reduced gene expression in tumor or cancer tissue.<br>
In another aspect the invention relates io a therapy whereby genes (increase and/or decrease) generally are correlated to disease are inhibited by one or more of the following methods:<br>
A method for reducing ceil tumorigenicity or Tiafignancy of a cell, said method comprising<br>
obtaining at least one nucleotide probe capable of hybridising with at feast one gene of a tumor ceil, said at (east one gene being selected from the group of genes consisting of gsne Nos. 1-199, 215-232, 237, 238, 239, 240, 241, 242, 243, 245, 246, 247, 248, 250, 253, 254, 257, 258, 260, 263, 264, 265, 267, 270, 271, 272, 278, 283, 284, 287, 288, 290, 291, 292, 294, 297, 298, 300, 302, 303f 305, 309, 310, 315, 316, 317, 318, 319, 321, 324, 329, 335, 336, 337, 339, 340, 344, 346, 347, 354, 35(3, 358, 359, 362, 364, 365, 368, 369, 371, 372, 377, 378, 379, 380, 381, 382, 383, 384, 38!3, 391, 393, 395, 396, 397, 399, 402, 403, 404, 409, 413, 417, 419, 420, 421, 422, 423, 42!S, 427 ,429, 430, 431, 432, 437, 444, 447, 448, 449, 450, 451, 452, 454, 455 ,456, 457, 458, 459, 462, 468, 474, 478, 484, 489, 491, ^93, 495, 500, 501, 502, 504, 505, 506, 507, 506, 509, 510, 511, 512, 513, 514, 518 , 519, 520, 522, 523, 524, 525, 529, 531, 534, 535, 536, 538, 544, 546, 547, 548, 549, 550, 551, 552, 553, 555, 556, 558, 559, 561, 562,<br>
introducing said at (east one nucleotide probe into the tumor cell in a manner allowing the probe to hybridise to the at least one gene, thereby inhibiting expression of said at least cne gene. This method is preferably based on anti-sense technology, whereby the hybridisation of said probe to the gene leads to a down-reculation of said gene.<br>
In another preferred embodiment, the method for reducing cell tumorigenicity or malignancy of a cell is based on RNA interference, comprising small interfering RNAs (siRNAs) specifically directed against at least one gene being selected from the group of genes consisting of gene Nos. 1-199, 215-232, 237, 238, 239, 240, 241, 242, 243, 245, 246, 247, 248, 250, 253, 254, 257, 258, 260, 263, 264, 265, 267, 270, 271, 272, 278, 283, 284, 237, 288, 290, 291, 292, 294, 297, 298, 300, 302, 303, 305, 309, 310, 315, 316, 317, 318, 319, 321, 324, 329, 335, 336, 337, 339, 340, 344, 346, 347, 354, 356, 358, 359, 362, 364, 365, 368, 369, 371, 372, 377, 378, 379, 380, 381, 382, 383, 384, 388, 391, 393, 395, 396, 397, 399, 402, 403, 404, 409, 413, 417, 419, 420, 421, 422, 423, 425, 427 ,429, 430, 431, 432, 437, 444, 447, 448, 449, 450. 451, 452, 454, 455 ,456, 457, 458, 459, 462, 468, 474, 473, 484, 489, 491, 493, 495, 500, 501, 502, 504  505, 506, 507, 508, 509, 510, 511, 512, 513,<br><br>
514, 518 , 519, 520, 522, 523, 524, 525, 52:9, 531, 534, 535, 536, 538. 5--, 5^5, 547, 548, 549, 550, 551, 552, 553, 555, 556, 558, 559, 561, 562.<br>
The down-regulaticn may of course also be based on a probe capable of hybridising to regulatory components of the genes in quesion, such as promoters.<br>
The hybridization may be tested in vitro at conditions corresponding to in vivo conditions. Typically, hybridization conditions are of low to moderate stringency. These conditions favour specific interactions between completely complementary sequences, but allow some non-specific interaction between less than perfectly matched sequences to occur as well. After hybridization, the nucleic acids can be "washed" under moderate or high conditions of stringency to dissociate duplexes that are bound together by some non-specific interaction (the nucleic acids that form these duplexes are thus not completely complementary).<br>
As is known in the art, the optimal conditions for washing are determined empirically, often by gradually increasing the stringency. The parameters that can be changed to affect stringency include, primarily, temperature and salt concentration. In general, the lower the salt concentration and the higher the temperature the higher the stringency. Washing can be initiated at a low temperature (for example, 'oom temperature) using a solution containing a salt concentration that is equivalent to or lower than that of the hybridization solution. Subsequent washing can be carried out using progressively warmer solutions having the same sait concentration. As alternatives, the salt concentration can be lowered and the temperature maintained in the washing step, or the salt concentration can be lowered and the temperature increased. Additional parameters can also be altered. For example, use of a destabilizing agent, such as formamide, alters the stringency conditions.<br>
In reactions where nucleic acids are hybridized, the conditions used to achieve a given level of stringency will vary. There is not one sest of conditions, for example, that will allow duplexes to form between ail nucleic acids that are 85% identical to one another; hybridization also depends on unique features of each nucleic acid. The length of the sequence, the composition of the sequence (for example, the content of purine-like nucleotides versus the content of pyrimidine-like nucleotides) and the type of nucleic acid (for example, DNA or RNA) affect hybridization. An additional consideration is whether one of the nucleic acid&amp; is immobilized (for example on a filter).<br>
An example of a progression from lower to higher stringency conditions is the following, where the salt content is given as the relative abundance of SSC (a salt solution containing sodium chloride and sodium citrate; 2X SSC: is 10-fold more concentrated than 0.2X SSC). Nucleic acids are hybridized at 42Â°C in 2X SSC/0.1% SDS (sodium dodecylsulfate; a deter-<br><br>
gent) and then wasned in 0.2X SSC/0.1% SDS at room temperature (for conditions of low stringency); 0.2X SSC/0.1% SDS at 42Â°C (for conditions of moderate stringency); and 0.1X SSC at 38Â°C (for conditions of high stringency;. Washing can be carried out using only one of the conditions given, or each of the conditions can be used (for example, washing for 10-15 minutes each in the order listed above]. Any or all of the washes can be repeated. As mentioned above, optimal conditions will very and can be determined empirically.<br>
In another aspect a method of reducing 'umoregeneicity relates to the use of antibodies against an expression product of a cell from the biological tissue. The'antibodies may be produced by any suitable method, such as ;a method comprising the steps of<br>
obtaining expression product(s) from at least one gene said gene being expressed as defined above,<br>
immunising a mammal with said expression product(s) obtaining antibodies against the expression product.<br>
Use<br>
The methods described above may be used for producing an assay for diagnosing a biological condition in animal tissue, or for identification of the origin of a piece of tissue. Further, the methods of the invention may be used for prediction of a disease course and treatment response.<br>
Furthermore, the invention relates to the uses of a peptide as defined above for preparation of a pharmaceutical composition for the treatment of a biological condition in animal tissue.<br>
Furthermore, the invention relates to the uses of a gene as defined above for preparation of a pharmaceutical composition for the treatment of a biological condition in animal tissue.<br>
Also, the invention relates to the use of a probe as defined above for preparation of a pharmaceutical composition for the treatment of a biological condition in animal tissue.<br>
The genetic material discussed above for may be any of the described genes or functional parts thereof. The constructs may be introdjced as a single DNA molecule encoding all of the genes, or different DNA molecules having one or more genes. The constructs may be introduced simultaneously or consecutively, each with the same or different markers.<br><br>
The gene may be linked to the complex as SUCH or protected by any suitaole system normally used for transfection such as viral vectors or artificial viral envelope, iiposomes cr rni-cellas, wherein the system is linked to the complex.<br>
Numerous techniques for introducing DNA into eukaryotic cells are known :o the skilled artisan. Often this is done by means of vectors, and often in the form of nucleic acid encapsi-dated by a (frequently virus-like) proteinaceous coat. Gene delivery systems may be apolied to a wide range of clinical as well as experimental applications.<br>
Vectors containing useful elements such as selectabfe and/or amplifiabie markers, oro-moter/enhancer elements for expression in mammalian, particularly human, cells, and which may be used to prepare stocks of construct DNAs and for carrying out transfections are -veil known in the art. Many are commercially available.<br>
Various techniques have been developed for modification of target tissue and cells in vivo. A number of virus vectors, discussed below, are known which allow transfection and random integration of the virus into the host. See, for example, Dubensky et al. (1984) Proc. Matl. Acad. Set. USA 81:7529-7533; Kaneda et al., (1989) Science 243:375-373; Hiebert et al. (1989) Proc. Natl. Acad. Sci. USA 86:35Â£i4-3598; Hatzoglu et al., (1990) J. Biol. Chem. 265:17285-17293; Ferry et al. (1991) Proc. Natl. Acad. Sci. USA 88:8377-3381. Routes and modes of administering the vector include injection, e.g intravascularly or intramuscularly, inhalation, or other parenteral administration.<br>
Advantages of adenovirus vectors for human gene therapy include the fact that recombination is rare, no human malignancies are known to be associated with such viruses, the ace-novirus genome is double stranded DNA which can be manipulated to accept foreign genes of up to 7.5 kb in size, and live adenovirus is a safe human vaccine organisms.<br>
Another vector which can express the DNA molecule of the present invention, and is useful in gene therapy, particularly in humans, iis vaccinia virus, which can be rendered nonrepeating (U.S. Pat. Nos. 5,225,336; 5,204,243; 5,155,020; 4,769,330).<br>
Based on the concept of viral mimicry, artifical viral envelopes (AVE) are designed based on the structure and composition of a viral membrane, such as HIV-1 or RSV and used to deliver genes into cells in vitro and in vivo. See, for example, U.S. Pat. No. 5,252,348, Schreier H. et a!., J. Mol. Recognit, 1995, 8:59-62; Schreier H et al., J. Biol. Chern., 1994, 269:9090-9098; Schreier, H., Pharm. Acta Helv, 1994, 68:145-159; Chander, R et al. Life Sci.f 1992, 50:481-489, which references are hereby incorporated by reference in their entirety. The envelope is preferably produced in a two-step dialysis procedure where the "naked" enve-<br><br>
iope is :ormed initially, followed by unidired ionai insertion of the viral surface glycoprotein of interest. This process and the physical characteristics of the resulting AVE are described in detail by Chander ei al., (supra). Examples of AVE systems are (a) an AVE containing the HIV-1 surface glyccprotein gp160 (Chander et al., supra; Schreier et al.f 1995, supra) or giycosyl phosphatidvlinositoi (GPI)-Hnked cp120 (Schreier et al., 1994, supra), respectively, and (b) an AVE containing the respiratory syncytial virus (RSV) attachment (G) and fusion (F) giycoproteins (Stecenko, A. A. et al., Pharm. Pharmacol. Lett. 1:127-129 (1992)). Thus, vesicles are constructed which mimic the natural membranes of enveloped viruses in their ability to bind to and deliver materials to eel s bearing corresponding surface receptors.<br>
AVEs are used to deliver genes both by int'avenous injection and by instillation in the iungs. For example, AVEs are manufactured to m mic RSV, exhibiting the RSV F surface giycopro-tein which provides selective entry into epithelial cells. F-AVE are loaded with a plasmid coding for the gene of :nterestt (or a reporter gene such as CAT not present in mammalian tissue).<br>
The AVE system described herein in physically and chemically essentially identical to the natural virus yet is entirely "artificial", as it is constructed from phospholipids, cholesterol, and recombinant viral surface giycoproteins. Hence, there is no carry-over of viral genetic information and no danger of inadvertant viral infection. Construction of the AVEs in two independent steps allows for bulk production cf the plain lipid envelopes which, in a separate second step, can then be marked with the desired viral glycoprotein, also allowing for the preparation of protein cocktail formulations if desired.<br>
Another delivery vehicle for use in the present invention are based on the recent description of attenuated Shigeila as a DNA delivery system (Sizemore, D. R. et al., Science 270:299-302 (1995), which reference is incorporated by reference in its entirety). This approach exploits the ability of Shigellae to enter epithelial cells and escape the phagocytic vacuoie as a method for delivering the gene construct ini:o the cytoplasm of the target cell. Invasion with as few as one to five bacteria can result in expression of the foreign plasmid DNA delivered by these bacteria.<br>
A preferred type of mediator of nonvira! transfection in vitro and in vivo is cationic (ammonium derivatized) iipids. These positively charged lipids form complexes with negatively charged DNA, resulting in DNA charged neutralization and compaction. The complexes en-docytosed upon association with the cell membrane, and the DNA somehow escapes the endosome, gaining access to the cytoplasm. Cationic !ipid:DNA complexes appear highly stable under aorma! conditions. Studies of the cationic lipid DOTAP suggest the complex dissociates when the inner layer of the cell membrane is destabilized and anionic lipids frcm<br><br>
the inner layer displace DNA from the caiionic iipid. Several cationic lipids are available commercially. Two of these, DMRI and DC-cholesterol, have been used in human clinical trials. First generation cationic lipids are less; efficient than viral vectors. For delivery to lung, any inflammatory responses accompanying the liposome administration are reduced by changing the delivery mode to aerosol administration which distributes the dose more evenly.<br>
Drug screening<br>
Genes identified as changing in various stages of bladder cancer can be used as markers for drug screening. Thus by treating bladder cancer cells with test compounds or extracts, arc monitoring the expression of genes identified as changing in the progression of bladder cancers, one can identify compounds or extracts which change expression of genes to a pattern which is of an earlier stage or even of normal bladder mucosa.<br>
It is also within the scope of the invention to use small molecules in drug screening. The following are non-limiting examples illustrating the present invention. EXAMPLES<br>
Example 1<br>
Identification of a molecular signature defining disease progression in patients with<br>
superficial bladder carcinoma<br>
Patient samples<br>
Bladder tumor biopsies were obtained directly from surgery after removal of the necessary amount of tissue for routine pathology examination. The tumors were frozen at -80Â°C in a guanidinium thiocyanate solution for preservation of the RNA. Informed consent was obtained in all cases, and the protocols were spproved by the scientific ethical committee of Aarhus County. The samples for the no progression group were selected by the following criteria: a) Ta or T1 tumors with no prior higher stage tumors; b) a minimum follow up period of 12 months to the most recent routine cystoscopy examination of the bladder with no occurrence of tumors of higher stage. The samples for the progression group were selected by two criteria: a) Ta or T1 tumors with no prior h gher stage tumors; b) subsequent progression to a higher stage tumor, see Table 1.<br>
Table 1. Clinical data on all patients involved in the study<br><br><br><br><br>
Delineation of non-progressing tumors from progressing tumors<br>
To delineate non-progressing tumors from progressing tumors we now profiled a total of 29 bladder tumor samples; 13 early stage bladder tumor samples without progression (median follow-up time 35 months) and 16 early stage bladder tumor samples with progression (median time to progression 7 months). See Table 1 for description of patient disease courses. We analyzed gene expression changes between the two groups of tumors by hybridizing the labeled RNA samples to customized Affymetix GeneChips with 59,000 probe-sets to cover virtually the entire transcriptome (-95% coverage). Low expressed and non-varying probe-sets were eliminated from the data set and tho resulting 6,647 probe-sets that showed variation across the tumor samples were subjected to further analysis. These probe-sets represent 5,356 unique genes (Unigene clusters).<br>
Gene expression similarities between tumor b opsies<br>
We analyzed gene expression similarities between the tumor biopsies using unsupervised hierarchical cluster analysis (Fig. 1). This showed a notable distinction between the non-progressing and the progressing tumors wher using the 3,197 most varying probe-sets (s.d. &gt; 75) for clustering (4 errors; %2 test, P = 0.0001). Using other gene-sets based on different gene variation criteria demonstrated the same distinction between the tumor groups. Two of the samples that show later progression (825-3 and 112-2) were found in the non-progression branch of the cluster dendrogram and two of the non-progressing samples (815-1 and 150-6) were found in the progression branch. This distinct separation of the samples indicated a considerable biological difference between the two groups of tumors. Notably, the T1 tumors did not cluster separately from Ta tumors; however, they did form a sub-cluster in the progressing branch of the dendrogram. Based on this we decided to look for a general signature of progression disregarding pathologic staging of the tumors.<br><br>
Selection of the 100 most s:gnificantly up-regulated genes in each group using t-test statistics<br>
We delineated the non-progressing tumors from the progressing tumors by selecting the 1CC most significantly up-regulated genes in each group using t-test statistics (Fig. 2 and Table 2). Among the genes up regulated in the non-firogressing group we found the SERPINB5 and FAT tumor suppressor genes and the FGFR3 gene, which has been shown to be frequently mutated in superficial bladder tumors with low recurrence rates (van Rhijn et ai. 2001). Among the genes up regulated in the prcgressing group we found the PLK (Yuan et al. 1997), CDC25B (Galaktionov et al. 1991), ZDC2Q (Weinstein et at. t994) and MCM7 (Hiraiwa et al. 1997) genes, which are involved in regulating ceil cycle and cell proliferation. Furthermore, in this group we identified the WHSC1, DD96 and GRB7 genes, which have been predicted/computed (Gene Ontology) to bs involved in oncogenic transformation. Another interesting candidate in this group is the NRG1 gene, which through interaction with the HER2/HER3 receptors has been found to nduce differentiation of lung epithelial cells (Liu &amp; Kern 2002). The PPARD gene was also dentified as up regulated in the tumors that show later progression. Disruption of this gene was found to decrease tumorigenicity in colon cancer cells (Park et al. 2001). Furthermore, PFARD regulates VEGF expression in;bladder cancer cell lines (Fauconnet et al. 2002).<br><br><br><br><br><br><br><br><br><br><br><br><br><br>
Permutation analysis of 100 most significantly up-regulated genes in each group By permuting the sample labels 500 times we estimated the significance of the differentially expressed genes. The permutation analysis revealed that it was highly unlikely to find as good markers by chance, as similar godd markers were only found in 5% of the permutated data sets, see Table 2<br>
Molecular predictor of progression<br>
A molecular predictor of progression using a corrbination of genes may have higher prediction accuracy than when using single marker geres. Therefore, to identify the gene-set that gives the best prediction results using the lowest number of genes we built a predictor using the "leave one out" cross-validation approach, a;; previously described (Golub et at. 1999). Selecting the 100 best genes in each cross-validation loop gave the lowest number of prediction errors (5 errors, 83% correct classification) in our training set consisting of the 29 tumors (see Figure 3). As in our previous study we used a maximum likelihood classification approach. We selected a gene-expression signat jre consisting of those 45 genes that were present in 75% of the cross-validation loops, and these represent our optimal gene-set for progression prediction (Fig. 4a and Table 3).<br>
Many of these 45 genes were also found among the 200 best markers of progression, however, the cross-validation approach also identified other interesting markers of progression like BIRC5 (Survivin), an apoptosis inhibitor that hi up regulated in the tumors that show later progression. BIRC5 has been reported to be exprsssed in most common cancers (Ambrosini et al. 1997). To validate the significance of the 45-gene expression signature we used a test set consisting of 19 early stage bladder tumors (9 tumors with no progression and 10 tumors with later progression). Total RNA from these samples were amplified, labeled and hybridized to customized 60mer-oligonucleotide microarray glass slides and the relative expressions of the 45 classifier genes were measure J following appropriate normalization and background adjustments of the microarray data: Jhe independent tumor samples were clas-<br><br>
sified as non-progressing or progressing according to the degree of correlation to the average no progression profile from the training sampies {Fig. 3Jb). When applying no cutoff limits to the predictions the predictor identified 74% of the samples correctly. However, as done recently in a breast cancer study (van't Veer et a:. 2002), we applied correlation cutoff limits of 0.1 and -0.1 in order to disregard samples with really low correlation values and in this way we obtained 92% correct predictions of samples with correlation values above 0.1 or below-0.1. Although the test-set is limited in size the performance is notable and could be of clinical use.<br><br><br><br>
Permutation analysis of 45 genes<br>
Again permutation analysis revealed that for all of the 45 genes similar good markers were only found in 5% of the 500 permuted datasets (see Table 3).<br>
Expression profiling of metachrone higher stage timers<br>
Expression profiling of the metachrone higher stage tumors could provide important information on the degree of expression similarities between the primary and the secondary tumors. Tissues from secondary tumors were avsilabie from 14 of the patients with disease progression and these were also hybridized to the customized Affymetrix GeneChips.<br><br>
Hierarchical cluster analysis of all tumor simples based on the 3,213 most varying probe-sets snowed that tumors originating from the same patient in 9 of the cases clustered tightly together indicating a high degree of intra ir dividual similarity in expression profiles (Fig. 5). Notabie, one tight clustering pair of tumors was a Ta and a T2+ tumor ;patient 941). it was remarkable that Ta and T1 tumors and T1 or T2+ tumors from a single individual were more similar than e.g. Ta tumors from two individuals. There was no correlation between presence and absence of the right clustering of samples from the same patient and time interval to tumor progression. Tie tight clustering of the 9 tumor pairs probably reflects the monoclonal nature of many biadcer tumors (Sidransky 3t al. 1997). A set of genomic abnormalities !ike chromosomal gains 2nd losses characterize bladder tumors of different stages from single individuals (Primdahi et al, 2002), and su:h physical abnormalities could be one of the causes of the strong similarity of metachroious tumors. The fact that 5 of the tumor pairs clustered apart may be explained by an oligoclonal origin of these tumors.<br>
Customized GeneChio design, normalizatior and expression measures We used a customized Affymetrix QeneChiD (Eos HuO3) designed by Eos Biotech Inc., as described (Eaves et al. 2002). Approximately 45,000 mRNA/EST clusters and 6,200 predicted exons are represented by the 59,000 probesets on Eos HuO3 array. Data were normalized using protoccis and software developed at Eos Biotechnology, Inc. (WO0079465). An "average intensity" (Al) for each probese t was calculated by taking the trimean of probe intensities following background subtraction and normalization to a gamma distribution (Turkey 1977).<br>
cRNA preparation, array hybridization and scanning<br>
Preparation of cRNA from total RNA and subsequent hybridization and scanning of the customized GeneChip microarrays (Eos Hu03) were performed as described previously (Dyrskjotetal.2003).<br>
Custom oligonucleotide microarray procedures<br>
Three 60mer oligonudeotides were designed for each of the 45 genes using Array Designer 2.0. All steps in the customized oligonucleotide microarray analysis were performed essentially as described (Kruhoffer et al.) Each of the probes was spotted in duplicates and all hybridisations were carried out twice. The samples were labelled with Cy3 and a common reference pool was labelled with Cy5. The reference pool was made by pooling of cRNA generated from investigated samples and fronn universal human RNA. Following scanning of the glass slides the fluorescent intensities were quantified and background adjusted using SPOT 2.0 (Jain et al. 2002). Data were subsequently normalized using a LOWESS normalisation procedure implemented in the SMA p;3ckage to R. To select the best ofigonucieotide probe for each of the 45 genes, 13 of the samples from the training set were re-anaiysed on<br><br>
the custom oiigonucleotide microarray platform and the obtained expression ratios were compared to the expression levels from the ^ffymetrix GeneChips. The oligonucleotide probes with the highest correlation to the Affymetrix GeneChip probes were selected.<br>
Expression data analysis<br>
Before analysing the expression data from the Eos HuO3 GeneChips control probes were removed and only probes with Al levels above 100 in at least 8 experiments and with max/min equal to or above 1.6 were selected. This filtering generated a gene-set consisting of 6,647 probes for further analysis. Average lirkage hierarchical cluster analysis of the tumour samples was carried out using a modified Pearson correlation as similarity metric (Eisen et al. 1998). Genes and arrays were median centered and normalised to the magnitude of 1 before clustering. We used the GeneC uster 2.0 software for the supervised selection of markers and for performing permutation tests. The 45 genes for precicting progression were selected by t-test statistics and cross-validation performance as previously described (Dyrskjot et al. 2003) and independent samples were classified according to the correlation to the average no progression signature profile of the 45 genes.<br>
EXAMPLE 2<br>
Identifying distinct classes of bladder carcinoma using microarrays<br>
Patient disease course information - class discovery<br>
We selected tumours from the entire spectrum o: bladder carcinoma for expression profiling in order to discover the molecular classes of the disease. The tumours analysed are listed In Table 4 below together with the available patient disease course information.<br><br><br><br><br><br>
Group A: Ta gr2 tumours - no recurrence within 2 &gt;ears.<br>
Group B: Ta gr3 tumours - no prior T1 tumour and IO carcinoma in situ in random biopsies.<br>
Group C: Ta gr3 tumours - no prior T1 tumour but carcinoma in situ in random biopsies.<br>
Group D: Ta gr3 tumours - a prior T1 tumour ard carcinoma in situ in random biopsies.<br>
Group E: T1 gr3 tumours - no prior T2+ tumour. G:oup F: T2+ tumours gr3/4 -= only primary<br>
tumours.<br>
* Carcinoma in situ detected in selected site biopsies at previous, sampling or subsequent<br>
visits.<br>
Two-way hierarchical cluster analysis of tumor samples<br>
A two-way hierarchical cluster analysis of the tumour samples based on the 1767 gene-set (see class discovery using hierachical clustering) rsmarkabiy separated ail 40 tumours according to conventional pathological stages and grades with only few exceptions (Fig. 6a). We identified two main branches containing the superficial Ta tumours, and the invasive T1 and T2+ tumours. In the superficial branch two sut-clusters of tumours could be identified, one holding 8 tumours that had frequent recurrences and one holding 3 out of the five Ta grade 2 tumours with no recurrences. In the invasive branch, it was notable that four Ta grade 3 tumours clustered tightly with the muscle hvasive T2+ tumours. These four Ta tumours, from patients with no previous tumour histcry, showed concomitant CIS in the surrounding mucosa, indicating that this sub-fraction oFTa tumours has some of the more aggressive features found in muscfe invasive tumours. The stage T1 cluster could be separated into three sub-clusters with no clear clinical difference. The one stage T1 grade 3 tumour that clustered with the stage T2+ muscle invasive tumours was the only T1 tumour that showed a solid growth pattern, all others showing papillary growth. Nine out of ten T2+ tumours were found in one single cluster. The rema'kable distinct separation of the tumour groups according to stage, with practically no overlap between groups, was also demonstrated by multidimensional scaling analysis (Fig. 6c].<br><br>
In an attempt to reduce the number of genos needed for class prediction we identified rhose genes that were scored by the Cancer Genome Anatomy Project (at NCI) as belonging to cancer-related groups such as tumour suppressors, oncogenes, cell cycle, etc. These genes were then selected from the initial 1767 gene-set, and those 38 which showed largest variation (SD of the gene vector &gt;=4), were usod for hierarchical clustering of the tumour samples. The obtainedclusters was almost identical to the 1767 gene-set cluster dendrogram (Fig. ob), indicating that the tumour clustering does not simply reflect larger amounts of stromal components in the invasive tumour biopsies.<br>
The clustering of the 1767 genes revealed several characteristic profiles in which there was a distinct difference between the tumour groups (Fig. 6d; black lines identifying clusters 3 to<br>
j).<br>
Cluster a, shows a high expression level in all the Ta grade 3 tumours (Fig. 7a) and, as a novel finding, contains genes encoding 8 transcription factors as well as other nuclear genes related to transcripticnal activity. Cluster c contains genes that are up-regulated in both Ta grade 3 with high recurrence rate and CIS, in T2+ and some T1 tumours. This cluster shows a remarkable tight co-regulation of genes reated to ceil cycle control and mitosis (Fig. 7c). Genes encoding cyciins, PCNA as well as a number of centromere related proteins are present in this cluster. They indicate increased cellular proliferation and may form new targets for small molecule therapy (Seymour 1999). Cluster f shows a tight cluster of genes related to keratinisation (Fig. 7f). Two tumours (875-1 and 1178-1) had a very high expression of these genes and a re-evaluation of the pathology slides revealed that these were the only two samples to show squamous metaplasia. Thus, activation of this cluster of genes promotes the squamous metaplasia not infrequently seen by light microscopy in invasive bladder tumours. The genes in this cluster is listed in Table 5.<br><br><br><br>
Cluster g contains genes that are up-regulated in T2+ tumours and in the Ta grade 3 tumours with CIS that duster in the invasive branch (Fig. 7g). This cluster contains genes related :o angiogenesis and connective tissue such as iaminin, myosin, caldesmon, collagen, dystrcphin, fibronectir, and endogiin. The increased transcription of these genes may indicate a profound remodelling of the stroma that could reflect signalling from the tumour ceils, from infiltrating lymphocytes, or both. Some of these may also form new drug targets (Fox et al. 2001). It is remarkable that these genes ere those that most clearly separate the Ta grade 3 tumours surrounded by CIS from alt other Ta grade 3 tumours. The presence of adjacent CIS is usually diagnosed by taking a set of eight biopsies from different places in the bladder mucosa. However, the present data clearly indicate that analysis of stroma remodelling genes in the Ta tumours could eliminate this invasive procedure.<br>
The clusters b, d, e, h, i, and j contain genes related to nuclear proteins, ceil adhesion, growth factors, stromal proteins, immune system, and proteases, respectively (see Figure 8). ... A summary of the stage related gene expression is shown in Table 6.<br><br><br>
Class prediction of bladder tumours<br>
An oojective class prediction of bladder tumours based on a limited gene-set is clinically useful!. We therefore built a classifier usinq tumours correctly separated in the three main grcuDs as identified in the cluster dendrogram (Fig. 6a). We used a maximum likelihood classification method with a "leave one out" cross-validation scheme (Shipp et al. 20C2; van't Veer et at. 2002) in which one test tumour was removed from the set, and a set of predictive genes was selected from the remaining tumour sampies for classifying the test tumour. This process was repeated for all tumours. Predictive genes that showed the largest possible separation of the three groups were selecte i for classification, and each tumour was classified according to how close it was to the mean of the three groups (Fig. 3a).<br>
Classification of samples<br>
From the hierarchical cluster analysis of this samples (class discover/) we identified three major "molecular classes" of bladder carcinoma highly associated with the pathologic staging of the samples. Based on this finding we decided to build a molecular classifier that assigns tumcurs to these three "molecular classes". "*o build the classifier, we only used the tumours in which there was a correlation between the "molecular class" and the associated pathologic stage. Consequently, a T1 tumour clustering in the "molecular class;I of T2 tumours was not used to build-the classifier.<br>
The genes used in the classifier were those <jenes with the highest values of ratio variation between groups to within groups. high signify genes good group separaion performance. we calculated sum over squared distance from sample value mean and classified as belonging where ihe was smallest. if relative difference closest second compared were below classification failed bcth is refered classifier strength.></jenes>
Classifier performance<br>
The classifier performance was tested usin^ from 1-160 genes in cross-validation [oops. Figure 9 shows that the closest correlation to histopathology is obtained in the cross-validation model using from 69-97 genes. Based on this we chose the model using 80 genes for cross-validation as our final classifier model.<br>
Classifier model using 71 genes<br><br><br><br><br><br><br>
Test for significance of classifier<br>
To test the class separation performance cf the 71 selected genes we compared the B/W ratios with the similar ratios of all the genes calculated from permutations of the arrays. For each permutation we construct three pseudogroups, pseudo-Ta, pseudo-T1, and pseudo-T2, so that the proportion of samples from the three original groups is approximately the same in the three pseudogroups. We then calculate the ratio of the variation between the psudogroups to the variation within the pseudogroups for ali the genes. For 500 permutations we only two times had one gene for which the B/W value was higher than the lowest value for the original B/W values of the 71 selected genes (the two values being 25.28 and 25.93).<br>
The classifier performance was tested using from 1-160 genes in cross-validation loops, and a model using an 80 gene cross-validation scheme showed the best correlation to pathologic staging (p
Permutation analysis<br>
To test the class separation performance of the 71 selected genes we compared their performance to those of a permutated set of pseudo-Ta, T1 and T2 tumours. In 500 permutations we only detected two genes with a perormance equal to the poorest performing classifying genes.<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
p!es. We tested the strength of the predictive genes by performing 500 permutations of the arrays. This revealed that for most of our p-edictive genes we would only in a small number or the new pseudo-groups obtain at least as good predictors as in the real groups.<br>
3iological material<br>
56 bladder tumour biopsies were sampled Jrom patients following removal of the necessary amount of tissue for routine pathology examination. The tumours were frozen immediately after surgery and stored at -80Â°C in a guanidinium thiocyanate solution. All tumours were graded according to Bergkvist et al. 1965 and re-evaluated by a single pathologist. As normal urothelial reference samples we used a pool of biopsies (from 37 patients) as weil as :hree single bladder biopsies from patients with prostatic hyperpiasia or urinary incontinence, informed consent was obtained in ail cases and protocols were approved by the local scientific ethical committee,<br>
RNA purification and cRNA preparation<br>
Total RNA was isolated from crude tumour biopsies using a Polytron homogenisator asd the RNAzol B RNA isolation method (WAK-Che-nie Medical GmbH). 10 (ig total RNA was used as starting material for the cDNA preparation. The first and second strand cDNA synthesis was performed using the Superscript Choice System (Life Technologies) according to the manufacturers instructions except using an oiigo-aT primer containing a T7 RNA polymerase promoter site. Labelled cRNA was prepared using the BioArray High Yield RNA Transcript Labelling Kit (Enzo). Biotin labelled CTP and UTF (Enzo) were used in the reaction together with unlabeled NTP's. Following the IVT reaction the unincorporated nucleotides were removed using RNeasy coiumns (Qiagen).<br>
Array hybridisation and scanning<br>
15 [ig of cRNA was fragmented at 94Â°C for 35 min in a fragmentation buffer containing 40 mM Tris-acetate pH 8.1, 100 mM KOAc, 30 mM MgOAc. Prior to hybridisation, the fragmented cRNA in.a 6xSSPE-T hybridisation buffer (1 M NaCI, 10 mM Tris pH 7.6, 0.005% Triton), was heated to 95Â°C for 5 min and subsequently to 45Â°C for 5 min before loading onto the Affymetrix probe array cartridge (HuGeneFL). The probe array was then incubated for 16 h at 45Â°C at constant rotation (60 rpm). The washing and staining procedure was performed in the Affymetrix Ruidics Station. The probe array was exposed to 10 washes in 6xSSPE-T at 25Â°C followed by 4 washes in 0.5xSSPE-T at 50Â°C. The biotinylated cRNA was stained with a streptavidin-phycoerythrin conjugate, final concentration 2 ^gl\i\ (Molecular Probes, Eugene, OR) in 6xSSPE-T for 30 min at 25Â°G followed by 10 washes in 6xSSPE-T at 25Â°C. The probe arrays were scanned at 560 nm using a confocal laser-scanning microscope (Hewlett Packard GeneArray Scanner G2500A). The readings from the quantitative scanning were analysed by the Affymetrix Gene Expression Analysis Software. An antibody ampiifica-<br><br>
tion step followed using normai goat IgG ap blocking reagent, final concentration 0.1 mg/ml (Sigma) and biotinylated anti-streptavidin antibody (goat), final concentration 3 ug/ml (Vector Laboratories). This was followed by a staining step with a strep:avjdin-phycoerythrin conjugate, final concentration 2 \LQI\L\ (Molecular Probes, Eugene, OR) :n 6xSSPE-T for 30 min at 25Â°C and 10 washes in 6xSSPE-T at 25Â°C. The arrays were then subjected to a second scan under similar conditions as described above.<br>
Class discovery using hierarchical clustering<br>
All microarray results were scaled to a global intensity of 15Q ur.;:s using the Affymetrix Qe-neChip software. Other ways of array normalisation exist (Li and Hung 2001), however, using the dCHIP approach did not change trie expression profiles of the obtained classifier genes in this study (results not shown). For hierarchical cluster analysis and molecular classification procedures we used expression level ratios between tumours and the normal urothe-iium reference pool calculated using the comparison analysis implemented in the Affymetrix GeneChip software. In order to avoid expression ratios based on saturated gene-probes, we used the antibody amplified expression-dsta for genes with a mean Average Difference value across all samples below 1000 and tine non-amplified expression-data for genes with values equal to or above 1000 in mean Average Difference value across all samples. Consequently, gene expression levels across a I samples were either from the amplified or the non-amplified expression-data. We applied different filtering criteria to the expression data in order to avoid including non-varying and very low expressed genes in the data analysis. Firstly, we selected only genes that showed significant changes in expression levels compared to the normal reference pool in at least three samples. Secondly, only genes with at least three "Present* calls across all samples were selected. Thirdly, we eliminated genes varying less than 2 standard deviations ac-oss ail samples. The final gene-set contained 1767 genes following filtering. Two-way hierarchical agglomerative cluster analysis was performed using the Cluster software25. We used average linkage clustering with a modified Pearson correlation as similarity metric. Gen^s and arrays were median centred and normalised to the magnitude of 1 prior to cluster analysis. The TreeView software was used for visualisation of the cluster analysis results (E:isen et al. 1998). Multidimensional scaling was performed on median centred and normalised data using an implementation in the SPSS statistical software package.<br>
Tumour stage classifier<br>
We based the classifier on the log-transformed expression level ratios. For these transformed values we used a normal distribution with the mean dependent on the gene and the group (Ta, T1, and T2, respectively) and the /ariance dependent on the gene only. For each gene we calculated the variation within the groups (W) and the three variations between two groups (B(Ta/T1), B(Ta/T2), B(T1/T2)) and used the three ratios B/W to select genes. We<br><br>
selected those genes having a high value of 3(Ta/T1)/W, those genes having a high value of B(Ta/T2)/W, and those genes with a hie h value of B(T1/T2)/VV. To classify a sample, we calculated the sum over the genes of the squared distance from the sample value to the group mean, standardised by the variance. Thus, we got a distance to each of the three groups and the sampie was classified as belonging to the group in which the distance was smallest. When calculating these distances the group means and the variances were estimated from all the samples in the training s&gt;3t excluding the sample being classified.<br>
Recurrence prediction using a supervised learning method<br>
Average Difference values were generated using the Affymetrix GeneChip software and all values below 20 were set to 20 to avoid wry low and negative numbers. We only included genes that had a "Present* call in at least 7 samples and genes that showed intensity variation (Max-Min&gt;100, Max/Min&gt;2). The values were log transformed and rescaied. We used a supervised learning method essentially as described ( Shipp et al. 2002). Genes were selected using t-test statistics and cross-validation and sample classification was performed as described above.<br>
Immunohistochemistry<br>
Tumour tissue microarrays were prepared essentially as described (Kononen et al. 1998), with four representative 0.6 mm paraffin cores from each study case. Immunohistochemical staining was performed using standard highly sensitive techniques after "appropriate heat-induced antigen retrieval. Primary polyclonpl goat antibodies against Smad 6 (S-20) and cyclin G2 (N-19) were from Santa Cruz Biotechnology. Antibodies to p53 (monoclonal DO-7) and Her-2 (poiyclonal anti-c-erbB-2) were from Dako A/S. Ki-67 monoclonal antibody (MIBl) was from Novocastra Laboratories Ltd. Staining intensity was scored at four levels, Negative, Weak, Moderate and Strong by an experienced pathologist who considered both colour intensity and number of stained cells, and who was unaware of array results.<br>
EXAMPLE 3<br>
A molecular classifier detects carcinoma in situ expression signatures in tumors and<br>
normal urothelium of the bladder.<br>
Clinical samples<br>
Bladder tumour samples were obtained directly from surgery following removal of tissue for routine pathological examination. The samples were immediately submerged in a guadinium thiocyanate solution for RNA preservation and stored at -80Â° C. Informed consent was obtained in al! cases, and the protocols were approved by the scientific ethical committee of Aarhus County. Samples in the No-CIS grouD were selected based on the following criteria: a) Ta tumours with no CIS in selected sitei biopsies in all visits; b) no previous muscle invasive tumour. Samples in the CIS group were selected based on the criteria: a) Ta or T1<br><br>
tumours with CIS in selectee site biopsies in any visit (preferable Ta tumours with CIS in the sampling visit); b) no previous muscle invasive tumours. Normal biopsies were obtained from individuals witn prostatic hyperplasia or jrinarv incontinence. CIS and "normal" biopsies were obtained Tom cystecromy specimens directly following removal of the oiadder. A and was placed in the bladder for orientation and biopsies were taken from 8 positions covering the bladder surface. At each position, three biopsies were taken - two for pathologic examination and one in between these for RNA extraction for microarray expression profiling. The samples for RNA extraction were immediately transferred to the guadinium thiocyanate sciution and stored at -80Â° C until use. Samples used for RNA "extraction were assumed to have CIS if CIS was detected in both adjacent biopsies. The "normal" samples were assumed to be normal if both adjacent biopsies were normai.<br>
cRNA preparation, array hybridisation and scanning<br>
Purification of total RNA, preparation of ciRNA from cDNA and hybridisation and scanning were performed as previously described (Oyrskjot et ai. 2003). The labelled samples were hybridised to Affymetrix U133A GeneChips.<br>
Expression data analysis<br>
Following scanning all data were normalised using the RMA normalisation approach in the Bioconductor Affy package to R. Variation liters were applied to the data to eliminate non-varying and presumably non-expressed cenes. For gene-set 1 this was done by only including genes with a minimum expression above 200 in at !east 5 samples and genes with max/min expression intensities above or equal to 3. The filtering for gene-set 2 including only genes with a minimum expression of 200 in at least 3 samples and genes with max/min expression intensities above or equal to 3. Average linkage hierarchical cluster analysis was carried out using the Cluster software with a modified Pearson correlation as similarity metric (Eisen et al. 1998). We used the TreeView software for visualisation of the cluster analysis results (Eisen et al. 1998). Genes were log-transformed, median centred and normalised to the magnitude of 1 before clustering. We used GeneCIuster 2.0 (http://www-aenome.wi.mit.edu/cancer/software/qenecluiiter2/qc2.html) for the supervised selection of markers and for permutation testing. The algorithms used in the software are based on (Golub et al. 1999, Tamayo et al. 1999). Cassifiers for CIS detection were built using the same methods as described previously (Dyrskjot et al. 2003).<br>
Gene expression profiling<br>
We used high-density oligonucleotide microarxays for gene expression profiling of approximately 22,000 genes in 28 superficial bladder tumour biopsies (13 tumours with surrounding CIS and 15 without surrounding CIS) and in 13 invasive carcinomas. See table 19 for patient disease course descriptions. Furthermore, expression profiles were obtained<br><br><br><br>
' The tumour groups involved were  CC without CIS (1), TCC with CIS (2) ana invasive TCC (3).<br>
bThe numbers indicate the patient number followed by the clinic visit number. "CIS in selected.site biopsies in previous, present or subsequent visits to the clinic. ND: not determined. Molecular classification of the samples u:&gt;ing 25 genes in cross-validation loops.<br>
Hierarchical cluster analysis<br>
Following appropriate normalisation and expression intensity calculations we selected those genes that showed high variation across; the 41 TCC samples for further analysis. The filtering produced a gene-set consisting of 5,491 genes (gene-set 1} and two-way hierarchical duster analysis was performed based on this gene-set. The sample clustering showed a separation of the three groups of samples with oniy few exceptions (Fig. 14a). Superficial TCC with surrounding CIS clustered in the one main branch of the dendrogram, while the superficial TCC without CIS and the invasive TCC clustered in two separate sub-branches in the other main branch of the dendrogram. The only exceptions were that the invasive TCC samples 1044-1 and 1124-1 clustered in the CIS group and two TCC with CIS clustered in the invasive group (samples 1330-1 and 956-2). The only TCC without CIS that clustered in the CIS group was sample 1432-1. The distinct clustering of the tumour groups indicated a large difference in gene expression patterns.<br>
Hierarchical clustering of the genes (Fig. 14c) identified large clusters of genes characteristic for the each tumour phenotype. Cluster I showed a cluster of genes down-regulated in cystectomy biopsies, TCC with adjacent CIS and in some invasive carcinomas (Fig. 14c). There is no obvious functional relationsh p between the genes in this cluster. Cluster 2 showed a tight cluster of genes related to immunology and cluster 3 contained mostly genes expressed in muscle and connective tissue. Expression of genes in this cluster was observed in the normal and cystectomy sanples, in a fraction of the TCC with CIS and in the invasive tumours. Cluster 4 contained genes up-regulated in the cystectomy biopsies, TCC with adjacent CIS and in invasive carcinomas (Fig. 14c). This cluster includes genes involved in cell cycle regulation, cell proliferation and apoptosis. However, for most of the genes in this cluster there is not apparent functional relationship either. Comparisons of chromosomal location of the genes in th 5 clusters revealed no correlation between the observed gene clusters and chromosomal position of the identified genes. A positive correlation could have indicated chromosomal loss or gain or chromosomal inactivation by e.g. methylation of common promoter regions.<br>
To analyse the impact of surrounding CIS losions further we used the 28 superficial tumours<br>
only, and created a new gene set consisting of 5,252 varying genes (gede-set 2). Hierarchical cluster analysis of the tumour samples (Figure 13b) based on the new gene-set separated the samples according to the presence of CIS in the surrounding urothelium with<br><br>
only 1 excepiior. (P
Marker selection<br>
To delineate the tumours with surrounding CIS from the tumours without CIS we used t-test statistics to select the 50 most up-regulated genes in each group (Figure 15a). Permutation of the sample labels 500 times revealed that the 50 genes up-regutated in the CJS-group are highly significant differentially expressed aid unlikely to find by chance, as all markers were significant on a 5% confidence level. Consequently, in 500 random catasets it was only possible to select as good genes in less than 5% of the datasets. The 50 genes up-regulated in the no-CIS group showed a poorer performance in the permutation tests, as these were not significant on a 5% confidence level. See Table 20 for details. The relative expression of these 100 genes is 9 normal and 10 biopsies from cystectomies with CIS are shown in figure 15b. The no-CIS profile was found in all of the normal samples. However, all histologicaily normal sampies adjacent to the CIS lesions as well as the CIS biopsies showed the CIS profile.<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
found in the one main branch of the dendrogram and 3 of the 9 normal biopsies were found on the other main branch ("/2-test, P
 <br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
Tables Table B References<br>
Pisani, P., Parkin, D.M., Bray, F. &amp; Feriay J. Estimates of the worldwide mortality from 25<br>
cancers in 1990. IntJ Cancer 83, 18-29 (1-999)<br>
Wolf, H. et al. Bladder tumors. Treated natural history. Prog din Biol Res 221, 223-55 (1986).<br>
Althausen, A. F., Prout, G. R., Jr., and Daly, J. J. Non-invasive papillary carcinoma of the bladder associated with carcinoma in situ. Lj Uro!, 116: 575-580,1976.<br>
Spruck, C. HM 3rd, Ohneseit, P. F.f Gonzilez-Zuiueta. M., Esrig, D., Miyao, N., Tsai, Y. C,<br>
Lerner, S. P., Schmutte, C, Yang, A. S.,  Cote, R., and et al. Two molecular<br>
pathways to transitional cell carcinoma of the bladder. Cancer Res, 54: 784-788,<br>
1994. Rosin, M. P., Cairns, P., Epstein, J. i., Schoenberg, M. P., and Sidransky, D. Partial<br>
allelotype of carcinoma in situ of the human oladder. Cancer Res, 55; 5213-5216,<br>
1995. Anderstrom, C, Johansson, S., and Nilsson, S. The significance of lamina propria invasion<br>
on the prognosis of patients with bladder tumors. J Urol, 124:23-26,1980, Cummings, K. B. Carcinoma of the bladder: predictors. Cancer, 45:1849-1855,1980. Cheng, L, Cheville, J. C, Neumann, R. M.I Leibovich, B. C, Egan, K. S.t Spotts, B. E., and<br>
Bostwick, D. G. Survival of patients with carcinoma in situ of the urinary bladder.<br>
Cancer, 85:2469-2474, 1999. Kriegmair, M., Baumgartner, R., Lumper, w., Waidelich, R., and Hofstetter, A. Early clinical<br>
experience with 5-aminolevuIinic acid for the photodynamic therapy of superficial<br>
bladder cancer. Br J Urol, 77:667-671,1996.<br>
van 't Veer, LJ. et al. Gene expression profiling predicts clinical outcome of breast cancer. Nature 415, 530-6(2002).<br>
Dyrskjot, L. et al. Identifying distinct classes of bladder carcinoma using microarrays. Nat Genet 33, 90-6 (2003).<br>
van Rhijn, B.W. et al. The fibroblast growth fs ctor receptor 3 (FGFR3) mutation is a strong<br>
indicator of superficial bladder cance â  with low recurrence rate. Cancer Res 61,      -   -1265-8(2001).<br><br>
Yuan, J. et al. Polo-like kinase, a novel marker for cell'- ar proliferation. Am J Pathol 150, 1165-72(1997).<br>
Galaktionov, K. &amp; Beach, D. Specific activation of cdc25 tyrosine phosphatases by B-type cyclins: evidence for multiple roles of mitoticcvciins. Ceil 671 1181-94 (1991).<br>
Weinstein, J., Jacobsen, F.W., Hsu-Chen, .J.t Wu, T. &amp; 3aum, LG. A novel mammalian protein, p55CDCt present in dividing cells is associated with protein kinase activity arc has homology to the Saccharomycss cerevisiae cell division cycle proteins Cdc20 and Cdc4. Mol Cell Biol 14, 3350-63 (1994).<br>
Hiraiwa, A. et al. Immunolocaiization of hCI)C47 prote^ in normal and neoplastic human tissues and its relation to growth. ItiJ Cancer 74, 180-4 (1997).<br>
Liu, J. &amp; Kern, J.A. Neuregulin-1 activates tie JAK-STAT pathway and regulates lung epithelial cell proliferation. Am J Respir Cell Mol Bid 27, 306-13 (2002).<br>
Park, B.H., Vogelstein, B. &amp; Kinzler, K.W. Genetic disruption of PPARdeita decreases the tumorigenicity of human colon cancer ceils. Prcc NatlAcad Sci U S A 98, 2598-603 (2001).<br>
Fauconnet, S. et al. Differential regulation oF vascular endotheiiai growth factor expression by peroxisome proliferator-activatecl receptors n bladder cancer cells. J Biol Chem 277, 23534-43 (2002).<br>
Golub, T.R. et al. Molecular classification of cancer: class discovery and class prediction by gene expression monitoring. Science 286, 531-7 (1999).<br>
Ambrosini, G., Adida, C. &amp; Altieri, D.C. A novel anti-apoptosis gene, survivin, expressed in cancer and lymphoma. NatMedZ, 017-21 (1997).<br>
Sidransky, D. et al. Cional origin bladder cancer. N Engi J Med 326, 737-40 (1992).<br>
Primdahl, H. et al. Allelic imbalances in human bladder cancer: genome-wide detection with high-density single-nucleotide polymorphism arrays. J Nat! Cancer Inst 94, 216-23 (2002).<br>
Eaves, IA et al. Combining mouse congenis strains and microarray gene expression analyses to study a complex trait: the NOD model of type 1 diabetes. Genome Res 12, 232-43 (2002).<br><br>
Gnandour, G. &amp; Glynne, R. Method and apparatus for analysis of data from biomolecular arrays. International patent # WOO079465 (2CC0).<br>
Turkey, J. Exploratory Data Analysis, (Addison-Wesley, Reading, Mass, 1977).<br>
Kruhoffer, MM Magnusson, N.E., Aaboe, M.l Dyrskjot, L. &amp; Orntoft, T.F. Microarrays for gene expression profiling: Fabrication of pligonuclectide microarrays, Isolation of RNA, Fluorescent labelling of cRNA, Hybridisation, and Scanning, in Cell Biology - A laboratory handbook (ed. Celis, J.E.) (Tp appear in :he 4th edition of this book).<br>
Jain, A.N. at al. Fully automatic quantificaticn of microarray image data. Genome Res 12, 325-32 (2002).<br>
Â£:sen, M.B., Spellman, P.T., Brown, P.O. &amp; Botstein, D. Cluster analysis and display of genome-wide expression patterns. Proc NatlAcad Sci U S A 95,14863-8 (1998).<br>
Seymour, L. Novel anti-cancer agents in development: exciting prospects and new challenges. Cancer Treat.Rev. 25, 301-312 (199S).<br>
FoxT S. B., Gasparini, G.t &amp; Harris, A. L Anciogenesis: pathological, prognostic, and growth-factor pathways and their link to trial design and anticancer drugs. Lancet Oncol. 2, 278-289(2001).<br>
Shipp, M. A. et al. Diffuse large B-celi lymphoma outcome prediction by gene-expression profiling and supervised machine learning. Nai.Med. 8 , 68-74 (2002).<br>
Kerr, J. S., Slee, A. M.r &amp; Mousa, S. A. Smal; molecule alpha(v) integrin antagonists: novel anticancer agents. Expert.Opin.lnvestig.Drugs 9,1271-1279 (2000).<br>
Bergkvist, A., Ljungqvist, A.r &amp; Moberger, G. Classification of bladder tumours based on the cellular pattern. Preliminary report of a clinical-pathological study of 300 cases with a minimum follow-up of eight years. Acts Chir.Scand. 130, 371-378 (1965).<br>
Li, C. &amp; Hung, W. W. Model-based analysis of oligonucfeotide arrays: model validation, design issues and standard error application. Genome Biol. 2, RESEARCH0032 (2001).<br>
Kononen, J. et al. Tissue microarrays for high^throughput molecular profiling of tumor specimens. NatMed. 4, 844-847 (1998).<br><br>
Claims<br>
1.	A method of predicting the prognosis of a biological condition in animal tissue,<br>
comprising collect of a sample comprising ceils from the tissue and/or expression products from the cells.<br>
determining an expression level of a plurality of genes in the sample, said genes being selected from the group of genes consisting of gene No. 1 to gene No. 552,<br>
correlating the expression level to at least one standard expression level to predited the prognosis of the biological condition in the animal  issue.<br>
2.	The method of claim 1, wherein the animal tissue is selected from body organs.<br>
3.	The method of clam 2, wherein the animal tissue is selected from epithelial tissue in<br>
body organs.<br>
4.	The method of claim 3, wherein the animal tissue is selected from epithelial tissue in the<br>
urinary bladder.<br>
5.	The method according to claim 4, wherein the stage of the biological condition is selected from bladder cancer stages Ta. Carcinoma in situ (CIS),'Tl. T2, T3 and T4.<br>
5. The method according to claim 5, comprising determining at least the expression of a Ta stagÂ© gene from a Ta stage gene group, at leas: one T1 stage gene from a T1 stage gene group, at least a T2 stage gene from a T2 stage gene group, at feast a T3 stage gene from a T3 stage gene group, at least a T4 stage gene group from a T4 stage gene group, wherein at least one gene from each gene group is expressed in a significantly different amount in that stage than in one of ihe other stages.<br>
7.	The method according to claim 4f 5 or 6, wherein the stage is bladder cencer stage Ta.<br>
8.	The methcc accorcing to claim 4, where n the anin.ai tissue is mucosa.<br>
9.	The method of any of the preceding claims, where;n the biological condition is an adeno-<br>
carcinoma, a carcinoma, a teratoma, a sarcoma, and/or a lymphoma and/or csrcinoma-<br>
in-situ, and/or dysplasia-in-sltu.<br><br>
'0. The method of any of the preceding clains, where'r the ssmcle is a biopsy of the tissue or of metastasis crginating from said tissue,<br>
11.	The method according to any of the preceding claims, wherein the sample comprises<br>
substantially only ceils from said tissue.<br>
12.	The method according to claim 9, wherein the sancle comprises substantially only ceils<br>
from muccsa or tumors derived from saici mucosa cells.<br>
13.	The method according to any of the preceding claims, wherein ;he gere from the group<br>
of genes is selected individually from gere No. 1 :o gene No. 188 (stages).<br>
14.	The methcc according to any of the preceding claims 1-12, wherein the gene from :he<br>
group of genes is selected individually from gene No. 189 to gene No. 21- (recurrence).<br>
15.	The method according to any of the preceding claims 1-12, wnerein the gene from the<br>
group of genes is selected individually from gene No. 215 to gene No. 232 (SCC).<br>
16.	The method according to any of the preceding Claims 1-12, wherein the gene from the<br>
group of genes is selected individually from gene No. 233 to gene No. 446 (progression).<br>
17.	The method according to any of the preceding claims 1-12, wherein the gene from the<br>
group of genes is selected individually from gene No. 447 to gene No. 552 (CIS).<br>
18.	The methcc according to any of the preceding claims, wherein the expression level of at<br>
least two genes from the group of genes are determined.<br>
19.	The method according to any of the preceding claims, wherein the difference in expres<br>
sion level cf a gene from the gene group to the at least one standard expression level is<br>
at (east two-fold.<br>
20.	The method according to any of the preceding claims, wherein the expression level is<br>
determined by determining the mRNA of the cells.<br>
21.	The method according to any of the claims 1-19, wherein the expression level is<br>
a) determined by determining expression products, such as peptides, in the cells, or<br><br>
b) determined by determining expressiorji products, such as peotides, in the body fluids, such as blood, serum, plasma, faeces, mucus, sputtm, cerebrospinal fluid, and/or urine.<br>
22.	The method according :o any of the preceding claims, wherein the stage of the biological<br>
condition has been determined prior to the prediction of the prognosis.<br>
23.	The method according to claim 22, wherein the stage of the biological condition has<br>
been determined by<br><br>
a)	histoiogical examination cf the tissue or by gerctyping of the tissue, and/or<br>
b)	genotyping of the tissue.<br>
24.	The method accorcing to claim 23, wherein the stage of the biological condition has<br>
been determined bv<br>
determining the expression of at least a first stage gene from a first stage gene group and/or at least a second stage gene fron a second stage gene group, wherein at least one of said genes is expressed in said first stage of the condition in a higher amount than in saic seconc stage, and the other gene is a expressed in said first stage of the condition in a lower amount than in said second stage of the condition,<br>
correlating the expression level of the assessed genes to a standard levs. of expression determining the stage of the condition.<br>
'25. The method according to any of the preceding claims, wherein the expression level of at least two genes is determined, by<br>
determining a first expression level of at least one gene from a first gene group, wherein the gene from the first gene group is selected fron the group of genes No. 237, 233, 239, 240, 241, 242, 243, 245r 246, 247, 248, 250, 253r 254, 257, 253, 260, 263, 264, 265, 267, 270, 271, 272, 278. 253, 284, 237, 288] 290, 291, 292, 294, 297, 298, 300, 302, 303, 305, 309, 310, 315, 316, 317, 318. 319, 321, 324, 329, 335, 336, 337, 339, 340, 344, 346, 347, 3E4, 356, 358, 359, 362, 364, 365, 368T 369, 371r 372, 377f 378, 379, 380, 381r 352, 383, 334, 338, 391, 393, 3S5, 396, 397, 399, 402, 403, 404, 409, 413, 417, 419, 420, 421, 422, 423, 425, 427 ,429, 430r 431, 432, 437, 444 "(progressorgener), and<br><br>
determining a second exoression level 6f at leas: zne gene from 3 second gene group, wherein the second gene group is seiecied from re group of genes No. 233, 234, 235, 236, 24A, 249, 251, 252, 255, 256,1!59, 261, 262, 266, 268, 269, 273, 274; 275, 276, 277, 279, 280, 231, 232, 285, 236, 289, 293, 295, 296, 299, 301, 304, 306, 307, 3C8, 311. 312, 313, 314 , 320 , 322, 323, 325, 326, 327, 328 , 330, 331 r 332, 333, 334, 333, 341, 342, 343, 345, 343, 349, 350, 351, 352, 353, 355, 357, 360, 361, 363, 366, 367, 370, 373, ci74, 375, 576, 385, 386, 387, 389, 390, 392. 394, 393, 400, 401, 405, 406, 407, 408, 41C, -A11. 412, 414, 415, 415, 418, 424, 426, 428; 433, 434, 435, 436, 438t 439, 440, 441, 442, 443, 445, 446 (non-prcgrasscrgener), and<br>
correlating the first expression level tc a stancard exoression level for progressors, and/cr the second expression level to a standard exoression level for non-crogresscrs to predict the prognosis of the biological condition ir :ne animal tissue.<br>
26.	A method of determining the stage of a biological condition in animal tissue,<br>
comprising collecting a sample comprisir g ceils from the tissue,<br>
determining an expression level of a plurality of genes selected from the group of genes consisting of gene Mo 1 to gene No. 562<br>
correlating the expression level of the assessed cenes to at least one standard level of expression determining the stage of the condition,<br>
27.	The method according to claim 26, wherein the expression level of at least two genes is<br>
determined by<br>
determining the expression of at least a first stage gene from a first stage gene group and at least a second stage gene from a second stage gene group, wherein at least one of said genes is expressed In said first si age of the condition in a higher amount than in said second stage, and the other gene is a expressed in said first stage of the condition in a lower amount than in said second stsge of the condition, and<br>
correlating the expression level of the assessed genes to a standard level of expression determining the stage of the condition.<br>
28.	The methoc according to claim 26 or 27, wherein the stage is selected erom bladder<br>
cancer stages Ta, carcinoma in situ (CIS), T1, T2, 73 and T4.<br><br>
29.	The method accorar.g to claim 28, comprising determining at least the expression cf a<br>
Ta stage gene fron a Ta stage gene grcup, at leas: one T1 stage gene from a T1 stage<br>
gene group, at least a T2 stage gene fnsm a T2 3:sge gene group, at least a T3 stage<br>
gene from a T3 stage gene group, at least a T4 stage gene group from a 74 stage gene<br>
group, wherein at least one gene from oach gere group is expressed in a significantly<br>
different amount in that stags than in one of the an stages.<br>
30.	The method accorchg to claim 29, wherein a Ta stage gene is selected fncividuafly from<br>
the group of Table 3'.<br>
31.	The method accorar.g io claim 29, wherein a T1 s:age gene is selected individually from<br>
the group of Table 52.<br>
32.	The method according to claim 29, where in a T2 stage gene is selected individually from<br>
the group of Table 53.<br>
33.	The method according to any of claims :26-32, saic method comprising ere or more of<br>
the features defined In any of the claims 1-25.<br>
34.	A method of determining an expression pattern of s bladder cell sample, comprising:<br>
collecting sample comprising bladder colls and/or expression products from bladder cells.<br>
determining the expression level of a plurality of genes in the sample, said gene being selected from the group of genes consisting of gene No. 1 to gene No. 562, and obtaining an expression pattern of the bladder cell sample.<br>
35.	The method according to claim 34, wherein the expression level of at leas: two genes<br>
are determined.<br>
36.	The method according to claim 34, wheren the expression level of at least three genes<br>
are determined.<br>
37.	The method of claims 34-36, wherein the genes exclude genes which are expressed in<br>
the submucosal, muscle, or connective tissue, whereby a pattern of expression is formed<br>
for the sarrple which is independent of the proportion of submucosal, muscle, or connec<br>
tive tissue cells in trie sample.<br><br>
38.	the method of ciaim 37, comprising dstermininc the expression level of one or more<br>
genes in the sampie comprising predominantly suomucosal. muscle, and connective tis<br>
sue ceils, obtaining 3 second pattern, subtracting said second pattern from the expres<br>
sion pattern of the bladder ceil sample, forming a third pattern of expression, said third<br>
pattern of expression reflecting axpressbn of the tladder mucosa or bladder cancer cells<br>
independent of the proportion of submucosal, rrLscle, and connective tissue ceils pre<br>
sent in toe sample.<br>
39.	The method of any cf the preceding claims 34-33, wherein the sample is a biopsy of the<br>
tissue.<br>
40.	The method according to any of the preceding claim 34-39, wherein the sample is a cell<br>
suspension.<br>
41.	Tne method according to any of the preceding cairns 34-40, wherein the sample com<br>
prises substantially only cells from said lissue.<br>
42.	The method according to claim 41, wherein the sample comprises substantially only cells<br>
from mucosa.<br>
43.	A method of predicting the prognosis a biologic3i condition in human bladder tissue<br>
comprising,<br>
collecting a sample comprising cells from the tissue,<br>
determining an expression pattern of the cells as defined fn any of claims 33-42, correlating the determined expression pattern to a reference pattern, predicting the prognosis of the biological condition of said tissue.<br>
44.	A. method for determining the stage of a biological condition in animal tissue<br>
comprising,<br>
collecting a sample comprising cells from the tissue,<br>
determining an expression pattern of the cells as defined in any of claims 34-42,<br><br>
correlating the determined expression pattern to a reference pattern, determining the stage of the biological condition is said tissue.<br>
45. An assay for precicting the prognoses of a  biological condition  ir animal tissue, comprising<br>
at least one first marker capable of detecting an expression level of a plurality of genes selected from the croup of genes consistng of gere No. 1 to gene No. 552.<br>
4-6. The assay according to claim 45, wherein the maror is a nucieotide probe,<br>
47.	The assay according to claim 45, whereir the marker is an antibody.<br>
48.	The assay according to ciaim 45, comprising a: east a first marker and/or a second<br>
marker, wherein the first marker is capahie of detecting a gene from a first gene group<br>
as defined in claim 25, and/or the second marker's capable of detecting a gene from a<br>
second gene group as defined in claim 2J-.<br>
49.	The assay according to any of claims 4I5-48, said assay further comprising means for<br>
correlating the. expression level of the a plurality of genes to a standard expression level<br>
and/or a reference expression pattern.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA3Mi1jaGVucC0yMDA1IGFic3RyYWN0LWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">1072-chenp-2005 abstract-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA3Mi1jaGVucC0yMDA1IGNsYWltcy1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1072-chenp-2005 claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA3Mi1jaGVucC0yMDA1IGRlc2NyaXB0aW9uIChjb21wbGV0ZSktZHVwbGljYXRlLTEucGRm" target="_blank" style="word-wrap:break-word;">1072-chenp-2005 description (complete)-duplicate-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA3Mi1jaGVucC0yMDA1IGRlc2NyaXB0aW9uIChjb21wbGV0ZSktZHVwbGljYXRlLTIucGRm" target="_blank" style="word-wrap:break-word;">1072-chenp-2005 description (complete)-duplicate-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA3Mi1jaGVucC0yMDA1IGRlc2NyaXB0aW9uIChjb21wbGV0ZSktZHVwbGljYXRlLTMucGRm" target="_blank" style="word-wrap:break-word;">1072-chenp-2005 description (complete)-duplicate-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA3Mi1jaGVucC0yMDA1IGRlc2NyaXB0aW9uIChjb21wbGV0ZSktZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">1072-chenp-2005 description (complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA3Mi1jaGVucC0yMDA1IGRyYXdpbmdzLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">1072-chenp-2005 drawings-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA3Mi1jaGVucC0yMDA1LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1072-chenp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA3Mi1jaGVucC0yMDA1LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1072-chenp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA3Mi1jaGVucC0yMDA1LWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1072-chenp-2005-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA3Mi1jaGVucC0yMDA1LWNvcnJlc3BvbmRuZWNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">1072-chenp-2005-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA3Mi1jaGVucC0yMDA1LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">1072-chenp-2005-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA3Mi1jaGVucC0yMDA1LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1072-chenp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA3Mi1jaGVucC0yMDA1LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">1072-chenp-2005-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA3Mi1jaGVucC0yMDA1LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">1072-chenp-2005-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA3Mi1jaGVucC0yMDA1LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">1072-chenp-2005-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA3Mi1jaGVucC0yMDA1LW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1072-chenp-2005-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA3Mi1jaGVucC0yMDA1LXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1072-chenp-2005-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="223552-controlled-release-pharmaceutical-composition-comprising-melatonin.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="223554-an-invitro-method-of-determining-the-status-of-sepsis-using-biomarket-profiles.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>223553</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1072/CHENP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>47/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Nov-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>12-Sep-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>31-May-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>AROS APPLIED BIOTECHNOLOGY APS</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>C/O OSTJYSK INNOVATION, GUSTAV WIEDS VEJ 10, DK-8000 ARHUS C,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ORNTOFT, TORBEN, FALCK</td>
											<td>HELGESVEJ 19, DK-8230 AABYHOJ,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>ANDERSEN, THOMAS, THYKJAER</td>
											<td>TRANEBAERVEJ 26, DK-8220 BRABRAND,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>ANDERSEN, LARS, DYRSKJOT</td>
											<td>LILLE ELSTEDVEJ 157D, DK-8520 LYSTRUP,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>JENSEN, JENS, LEDET</td>
											<td>FLINTEBAKKEN, 146B, DK-8240 RISSKKOV,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/DK03/00750</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-11-03</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>PA 2002 01685</td>
									<td>2002-11-01</td>
								    <td>Denmark</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/223553-an-invitro-method-of-predicting-the-prognosis-of-a-biological-condition-in-animal-tissue by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:28:54 GMT -->
</html>
