<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/223521-novel-gene-derived-from-coryneform-bacteria by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:35:15 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 223521:NOVEL GENE DERIVED FROM CORYNEFORM BACTERIA</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">NOVEL GENE DERIVED FROM CORYNEFORM BACTERIA</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Abstract 251/MAS/95 &quot;NOVEL GENE DERIVED FROM CORYNEFORM BACTERIA An isolated gene derived from Coryneform bacteria and coding for a protein which has an activity to impart surfactant resistance to said bacteria, wherein the protein has an amino acid sequence from the 37&lt;sup&gt;th&lt;/sup&gt; methionine residue to the 543&lt;sup&gt;rd&lt;/sup&gt; leucine residue of Sequence ID No.l in the Sequence Listing.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Field of The Invention<br>
The present invention relates to a new gene of Coryneform bacteria which are used for the fermentative production of L-amino acids such as L-glutamic acid and L-lysine and other substances, and the use thereof.<br>
Description of the Prior Art<br>
When Coryneform bacteria are cultivated in a medium having a restricted amount of biotin, they produce a large amount of L-glutamic acid. On the other hand, when the Coryneform bacteria are cultivated in a medium containing an excess amount of biotin, they do not produce L-glutamic acid. However, it is known that if a surfactant or penicillin is added to a medium containing such an excess amount of biotin, the growth of the bacteria in this medium is inhibited and the bacteria produce a large amount of L-glutamic acid therein.<br>
To make Coryneform bacteria produce L-glutamic acid, any<br><br>
of the following means is effective.<br>
1.	The biotin concentration in the medium is made<br>
suboptimal. Refer to S. Okumura, T. Tsugawa, T. Tsunoda and<br>
A. Kitai, Nippon Nogeikagaku Kaishi, 36_, 197-203 (1962).<br>
2.	A surfactant is added to the medium containing a<br>
sufficient amount of biotin.  Refer to I. Shiio, H. Otsuka<br>
and N. Atsuya, J. Biochem., 5_3, 333-340 (1963); K. Takinami,<br>
H. Okada and T. Tsunoda, Agr. Biol. Chem., 27, 853-863<br>
(1963).<br>
3.	Penicillin is added to the medium containing a<br>
sufficient amount of biotin. Refer to U.S. Patent 3,080,297;<br>
Japanese Patent Publication No. 37-1695; M. Shibui, T.<br>
Kurima, S. Okabe and T. Osawa, Amino Acid and Nucleic Acid,<br>
17, 61-65 (1968).<br>
The mechanisms in these means have been considered in the following way.<br>
The essential factor for producing L-glutamic acid in a medium containing a restricted amount of biotin are as follows: Biotin acts as the coenzyme for acetyl-CoA-carboxylase for the production of fatty acids and, in addition, unsaturated fatty acids, such as oleic acid, and<br><br>
their derivatives have a substitutive effect for biotin. Therefore, biotin will have an influence on the composition of fatty acids constituting the cell membranes of the bacteria, thereby varying the permeability of L-glutamic acid through the cell membranes. Refer to I. Shiio, S. Otsuka and M. Takahashi, J. Biochem., 51, 56-62 (1962); I. Shiio, K. Narui, N. Yahaba and M. Takahashi, J. Biochem., 5_1, 109-111 (1962).<br>
The production of L-glutamic acid in a medium containing a surfactant or penicillin has also been considered along with the variation in the permeability of L-glutamic acid through the cytoplasmic membranes of the bacteria growing in the medium because of the variation in the structures of cell surfaces. Refer to I. Shiio, S. Otsuka and N, Katsuya, J. Biochem., 53, 333-340 (1963).<br>
As mentioned above, the production of L-glutamic acid has been discussed along with its permeability through cell membranes, but no knowledge there has heretofore been obtained directly verifying the relationship between the production of L-glutamic acid and the permeability thereof.<br>
There have been many unclear matters with regard to by<br><br>
what mechanisms the restriction of the biotin content in the medium or the addition of a surfactant or penicillin to the medium improves the ability of Coryneform bacteria to produce L-glutamic acid.<br>
In addition, no information has been available on the gene level, which will be an important key factor for clarifying these mechanisms.<br>
Detailed Description of the Invention<br>
The subject matter of the present invention is to clarify the mechanisms of L-glutamic acid-production by Coryneform bacteria, more precisely the functional mechanisms of the surfactant to be added in the system of L-glutamic acid production by Coryneform bacteria, and to breed and improve L-glutamic acid-producing Coryneform bacteria on the basis of the information thus obtained. It is also intended to apply the thus-obtained information to the production of substances other than L-glutamic acid by using Coryneform bacteria.<br>
Concretely, the subject matter of the present invention is to clarify, on the gene level, the mechanisms of L-<br><br>
glutamic acid-production by Coryneform bacteria, to isolate a gene of Coryneform bacteria that participates in surfactant resistance, and to apply the thus-obtained gene to the breeding of L-glutamic acid-producing Coryneform bacteria and to the production of L-glutamic acid and other substances by Coryneform bacteria.<br>
We, the present inventors have intensively studied so as to attain the above-mentioned subject matter and, as a result, have found the existence of a gene which wills participate in the L-glutamic acid-production by Coryneform bacteria (the gene is hereinafter referred to as dtsR gene, and the protein encoded by the gene is referred to as DTSR protein). In addition, we have found a valuable use for this gene. On the basis of these findings, we have completed the present invention.<br>
The  present  invention  includes  the   following embodiments:<br>
(1)	A gene derived from Coryneform bacteria and coding for a protein which has an activity to impart surfactant resistance to said bacteria.<br>
(2)	The gene according to (1), wherein the protein has an<br><br>
amino acid sequence comprising from the 37th methionine residue to the 543rd leucine residue shown by Sequence ID No. 1 in the Sequence Listing.<br>
(3)	The gene according (1), which has a nucleotide sequence comprising from base 467 to base 1987 shown by Sequence ID No. 1 in the Sequence Listing.<br>
(4)	A DNA fragment comprising the gene according to (1).<br>
(5)	A recombinant DNA to be obtained by ligating a vector functioning in Coryneform bacteria to the DNA fragment according to (4).<br>
(6)	A Coryneform bacterium harboring the recombinant DNA according to (5).<br>
(7)	A method for producing L-lysine comprising cultivating a Coryneform bacterium harboring the recombinant DNA according to (5) and capable of producing L-lysine in a liquid medium to produce and accumulate L-lysine in the culture.<br>
(8)	A DNA fragment comprising a nucleotide sequence having substitution, deletion, insertion, addition and/or inversion of one or more bases in the gene according to (1).<br><br>
(9)	A recombinant DNA to be obtained by ligating a vector functioning in Coryneform bacteria to the DNA fragment according to (8).<br>
(10)	A Coryneform bacterium having substitution, deletion, insertion, addition and/or inversion of one or more bases in the coding region or promoter region of the gene according to  (1)  on chromosome and  lacking surfactant resistance.<br>
(11)	A method for producing L-glutamic acid comprising cultivating a Coryneform bacterium having substitution, deletion, insertion, addition and/or inversion of one or more bases in the coding region or promoter region of the gene according to (1) on chromosome, lacking surfactant resistance, and capable of producing L-glutamic acid in a liquid medium to produce and accumulate L-glutamic acid in the culture.<br>
The present invention will be described in detail hereinunder.<br>
Coryneform bacteria as referred to herein are a group of microorganisms defined in Bergey's Manual of Determinative<br><br>
Bacteriology, 8th Ed., p. 599 (1974), which are aerobic, gram-positive, non-oxidative bacilli not having the ability to sporulate, and include bacteria which had been classified as bacteria belonging to the genus Brevibacterium but have now been unified into the group of Corynebacterium [see Int. J. Syst. Bacteriol., 41^, 255 (1981)] and also include bacteria of the genus Brevibacterium and Microbacterium which are closely related to the genus Corynebacterium. Of such Coryneform bacteria, those mentioned below, which are known as L-glutamic acid-producing bacteria, are most preferred for use in the present invention.<br>
Corynebacterium acetoacidophilum<br>
Corynebacterium acetoglutamicum<br>
Corynebacterium callunae<br>
Corynebacterium glutamicum<br>
Corynebacterium lilium (Corynebacteri<br>
Corynebacterium melassecola<br>
Brevibacterium divaricatum (Corynebacterium glutamicum)<br>
Brevibacterium   lactofermentum   (Corynebacterium<br>
glutamicum)<br>
Brevibacterium saccharolyticum<br>
i<br><br>
Brevibacterium immariophilium<br>
Brevibacterium roseum<br>
Brevibacterium flavum (Corynebacterium glutamicum)<br>
Brevibacterium thiogenitalis<br>
Specifically, the following strains of these bacteria are exemplified:<br>
Corynebacterium acetoacidophilum	ATCC 13870<br>
Corynebacterium acetoglutamicum	ATCC 15806<br>
Corynebacterium callunae	ATCC 15991<br>
Corynebacterium glutamicum	ATCC 13032<br>
Corynebacterium glutamicum	ATCC 13060<br>
Brevibacterium divaricatum	ATCC 14020<br>
Brevibacterium 1actofermentum	ATCC 13869<br>
Corynebacterium lilium	ATCC 15990<br>
Corynebacterium melassecola	ATCC 17965<br>
Brevibacterium saccharolyticum	ATCC 14066<br>
Brevibacterium immariophilium	ATCC 14068<br>
Brevibacterium roseum	ATCC 13825<br>
Brevibacterium flavum	ATCC 13826<br>
Brevibacterium thiogenitalis	ATCC 19240<br><br>
These strains can be obtained from the American Type Culture Collection (ATCC). A registration number has been assigned to each strain of bacteria. Based on the reference number, anyone can obtain the corresponding strain of bacteria from ATCC. The registration numbers of the strains of bacteria deposited in ATCC are described in the ATCC catalog.<br>
Surfactants as referred in the present invention function to accelerate the production of L-glutamic acid by Coryneform bacteria, like penicillin, in a medium containing an excess amount of biotin therein, and these include various nonionic surfactants, cationic surfactants and anionic surfactants [see K. Yamada, J. Takahashi and J. Nakamura, the Hakkokogaku Kaishi, 20, 348-350 (1962); K. Udagawa, S. Abe and I. Kinoshita, Hakkokogaku Kaishi, 40, 614-619 (1962)]. Of nonionic surfactants, Tween 60 (polyoxyethylene sorbitan monostearate) and Tween 40 (polyoxyethylene sorbitan monopalmitate) have a penicillin-like effect [see I. Shiio, S. Otsuka and N. Katsuya, J. Biochem., 53, 333-340 (1963)]. C3 to C18 free saturated fatty acids have the same effect by themselves [see K. Takinami, H. Okada and T. Tsunoda, Agr.<br><br>
Biol. Chem., 2j}, 114-118 (1964)].  Tween 40 was used in the examples of the present invention.<br>  Isolation of  a gene  from a Coryneform bacterium participating in surfactant resistance:<br>
To isolate a gene from a Coryneform bacterium participating in surfactant resistance, for example, the following process can be employed.<br>
(1)	A surfactant-sensitive mutant of a Coryneform bacterium which shows sensitivity with regard to surfactants is obtained;<br>
(2)	various fragments of a chromosomal DNA prepared from a wild strain of a Coryneform bacterium each are ligated to a vector that functions in Coryneform bacteria to produce various recombinant DNAs;<br>
(3)	the recombinant DNAs each are introduced into cells of the surfactant-sensitive mutant of a Coryneform bacterium by transformation;<br>
(4)	from the resulting transformants, strains which have lost the surfactant-sensitivity are selected;<br>
(5)	the recombinant DNAs  are recovered  from the<br><br>
thus-selected surfactant-insensitive transforraants; and<br>
(6) the structure of the chromosomal DNA fragment of the wild strain of a Coryneform bacterium ligated to the vector is analyzed.<br>
The chromosomal DNA fragment of the wild strain of a Coryneform bacterium thus obtained contains a gene derived from a Coryneform bacterium participating in surfactant resistance. This gene participates in the mechanism of the production of L-glutamic acid by Coryneform bacteria in a medium containing a surfactant. In addition, this gene also participates in the production of L-glutamic acid by said bacteria in a medium containing penicillin or containing a restricted amount of biotin.<br>
A surfactant-sensitive mutant of a Coryneform bacterium which shows sensitivity to surfactants means a mutant of a Coryneform bacterium which grows poorly in a medium containing a surfactant at such a concentration that does not have any influence on the growth of the wild strain of Coryneform bacteria in said medium. Regarding a surfactant of   polyoxyethylene   sorbitan   monopalmitate,    a<br><br>
surfactant-sensitive mutant of a Coryneform bacterium grows worse than the corresponding wild strain in a medium containing the surfactant at a concentration of from 0.1 to 1 mg/dl. On the contrary, the growth of a wild strain of a Coryneform bacterium is not affected by the presence of said surfactant at a concentration of from 0.1 to 1 mg/dl in the medium. Where such a mutant is cultivated in a medium containing an excess amount of biotin to produce L-glutamic acid therein by adding a surfactant, the necessary concentration of the surfactant to be added to the medium may be lower than that in the ordinary case. It is considered that the condition of the cells of the surfactant-sensitive mutant will be similar to that of the cells of the corresponding wild strain which are exposed to the surfactants.<br>
To obtain a surfactant-sensitive mutant of a Coryneform bacterium, the method described in Japanese Patent Laid-Open No. 50-126877 (Japanese Patent Publication No. 52-24593) can be employed.<br>
As one example of the surfactant-sensitive mutant of a Coryneform   bacterium,   mentioned   is   Brevibacterium<br><br>
lactofermentum (Corynebacterium glutamicum) AJ 11060. This mutant was deposited in the National Institute of Bioscience and Human-Technology, Agency of Industrial Science and Technology, Ministry of International Trade and Industry, under the deposition number FERM P-3678.<br>
To prepare various fragments of the chromosomal DNA of a wild strain of a Coryneform bacterium, the following process may be employed. The wild strain of a Coryneform bacterium is cultivated in a liquid medium, the cells grown therein are collected, and the chromosomal DNA is recovered from the collected cells according to the method of Saito et al. [see H. Saito and K. Miura, Biochem. Biophys., Acta 72, 619 (1963)]. The thus-recovered chromosomal DNA is cleaved with restriction enzymes. Four-base recognition enzymes are used as the restriction enzymes, and the cleavage is conducted to prepare various DNA fragments under the condition under which the DNA is incompletely decomposed.<br>
The vector functioning in Coryneform bacteria as referred to herein is, for example, a plasmid which is self-amplification in Coryneform bacteria. Specific examples of the vector are mentioned below.<br><br>
pAM 330	see Japanese Patent Laid-Open No. 58-67699<br>
pHM 1519	see Japanese Patent Laid-Open No. 58-77895<br>
pAJ 655	see Japanese Patent Laid-Open No. 58-192900<br>
pAJ 611	see Japanese Patent Laid-Open No. 58-192900<br>
pAJ 1844	see Japanese Patent Laid-Open No. 58-192900<br>
pCG 1	see Japanese Patent Laid-Open No. 57-134500<br>
pCG 2	see Japanese Patent Laid-Open No. 58-35197<br>
pCG 4	see Japanese Patent Laid-Open No. 57-183799<br>
pCG 11	see Japanese Patent Laid-Open No. 57-183799<br>
To prepare various recombinant DNAs by ligating the vector functioning in Coryneform bacteria and various fragments of the chromosomal DNA of a wild strain of Coryneform bacteria, the following process may be employed. The vector is first cleaved with a restriction enzyme. The restriction enzyme to be used for cleaving the vector is the same as that used for cleaving the chromosomal DNA, or is such one by which the vector is cleaved to give cross sections complementary to the cross sections of the fragment of the chromosomal DNA. The ligation of the vector and the DNA fragment is generally effected via a ligase, such as T4<br><br>
DNA ligase, etc.<br>
To introduce the recombinant DNA to the surfactant-sensitive mutant of a Coryneform bacterium, any known transformation methods can be employed. For instance, employable is a method of treating recipient cells with calcium chloride so as to increase the permeability of DNA through the cells, which has been reported for Escherichia coli K-12 [see Mandel, M. and Higa, A., J. Mol. Biol., 53, 159 (1970)]; and a method of preparing competent cells from cells which are at the growth phase followed by introducing the DNA thereinto, which has been reported for Bacillus subtllis [see Duncan, C.H., Wilson, G.A. and Young, F.E., Gene, 1_, 153 (1977)]. In addition to these, also employable is a method of making DNA-recipient cells into the protoplast or spheroplast which can easily take up recombinant DNAs followed by introducing the recombinant DNA into the cells, which is known to be applicable to Bacillus subtilis, actinomycetes and yeasts [see Chang, S. and Choen, S.N., Molec. Gen. Genet., 16JJ, 111 (1979); Bibb, M.J., Ward. J.M. and Hopwood, O.A., Nature, 274, 398 (1978); Hinnen, A., Hicks, J.B. and Fink, G.R.. Proc. Natl. Sci., USA, 75, 1929<br><br>
(1978)].<br>
The above-mentioned protoplast method for Bacillus subtilis can be employed in the present invention to obtain sufficiently high-level frequency.  In addition to this, however, also employable is a method of introducing a DNA into protoplast of cells of a Coryneform bacterium while the protoplast are kept in contact with either polyethylene glycol or polyvinyl alcohol and with divalent metal ions, as described in Japanese Patent Laid-Open No. 57-183799.  In this method,  carboxymethyl cellulose,  dextran,  Ficoll, Bruronik F68 (produced by Selva Co.), etc. may also be used, in place of polyethylene glycol or polyvinyl alcohol, so as to accelerate the introduction of DNA to the protoplast cells.   In the examples of the present invention, the transformation was conducted by an electric pulse method (see Japanese Patent Laid-Open No. 2-207791).<br>
The method for selecting the surfactant-insensitive strain from the transformants will be described below.<br>
A DNA fragment having a size of approximately from 4 to 6 kbp, which has been obtained by partially digesting the chromosomal DNA of a wild strain of a Coryneform bacterium<br><br>
with the restriction enzyme, Sau3AI, is ligated to a plasmid vector which is self-amplification both in Escherichia coli and in Coryneform bacteria to produce a recombinant DNA, and this recombinant DNA is introduced into the competent cells of Escherichia coli DH5 (produced by Takara Shuzo Co., Ltd.). The resulting transformants are cultivated to prepare a gene library of the wild strain of a Coryneform bacterium.<br>
By using the recombinant DNA in this gene library, Brevibacterium lactofermentum AJ 11060 is transformed. The resulting transformants are inoculated on a surfactant-free M-CM2G agar plate (containing 5 g of glucose, 10 g of polypeptone, 10 g of yeast extract, 5 g of NaCl, 0.2 g of DL-methionine, 15 g of agar and 4 mg of chloramphenicol in one liter of pure water, and having a pH of 7.2), on which about 40,000 colonies are formed. These colonies are replicated onto a M-CM2G plate containing 30 mg/liter of a surfactant, Tween 40, and the colonies growing well on this surfactant-containing M-CM2G plate are collected. Thus, surfactant-insensitive transformants are obtained.<br>
To recover the recombinant DNA from the thus-obtained surfactant-insensitive transformants, the same method as that<br><br>
employed for preparing the chromosomal DNA of the wild strain of a Coryneform bacterium may be employed. Briefly, the transformants are cultivated in a liquid medium, the cells are collected from the culture, and the recombinant DNA is recovered from them according to the method of Saito et al. [see H. Saito and K. Miura, Biochem. Biophys., Acta 7^2, 619 (1963)].<br>
The structure of the chromosomal DNA fragment of the wild strain of a Coryneform bacterium that has been ligated to the vector is analyzed as follows. The full-length nucleotide sequence of the chromosomal DNA fragment is determined by the dyedeoxy method which is an ordinary nucleotide sequencing method, and the structure of the DNA is analyzed to determine the positions of the enhancer, the promoter, the operator, the SD sequence, the leader peptide, the attenuator, the initiation codon, the termination codon, the open reading frame, etc.<br>
The  gene obtained  from  the Coryneform  bacterium<br>
participating in surfactant resistance is dtsR gene, which<br>
has a sequence ranging between the bases 467-469 (ATG) and<br>
bases 1985-1987th (CTG) shown by Sequence ID No. 1 in the<br>
i<br><br>
Sequence Listing. This gene codes for DTSR protein having an amino acid sequence ranging from the 37th methionine residue to the 543th leucine residue in the amino acid sequence shown by Sequence ID No. 1 in the Sequence Listing. From the analysis of the promotor consensus sequence in the promotor region of the gene, it is considered that the translation of dtsR gene starts at the ATG codon of the bases 467-469, but the translation may start at the ATG codon of the bases 359-361 and the DTSR protein encoded by the gene may have an amino acid sequence ranging from the 1st methionine residue to the 543rd leucine residue. In either case, the N-terminal methionine can be cleaved with an aminopeptidase in the expression of the dtsR gene in the cells.<br>
According to the search on data base, it has been confirmed that the dtsR gene having the nucleotide sequence shown by Sequence ID No. 1 and the DTSR protein encoded by this sequence are novel. It has been found that this protein is homologous to the protein described in Proc. Natl. Acad. Sci. USA, 83, 8049-8053 (1986); Proc. Natl. Acad. Sci. USA, 8^3, 4864-4869 (1986); and Gene, 122, 199-202 (1992), as propionyl-CoA-carboxylase (PCC) protein p subunit. However,<br><br>
none of  these  references  suggests  that  said protein participates in the production of L-glutamic acid.<br>
Propionyl-CoA-carboxylase is an enzyme that catalyzes one reaction in the metabolic pathway where a-ketoglutaric acid is converted into succinyl-CoA via 2-hydroxyglutaric acid, propionyl-CoA, D-methylmalonyl-CoA and L-methylmalonyl-CoA, and it seems that said metabolic pathway is a by-pass route for the reaction to be catalyzed by a-ketoglutaric acid dehydrogenase in the TCA cycle. In this connection, it should be specifically noted that propionyl-CoA-carboxylase is an enzyme needing biotin as a coenzyme, by which the production of L-glutamic acid in the surfactant-addition method is related to the production of L-glutamic acid in the biotin-restriction method.<br>  Preparation of a strain of a Coryneform bacterium having<br>
the recombinant DNA:<br>
The recombinant DNA containing the dtsR gene from a Coryneform bacterium participating in surfactant resistance, which is obtained in the foregoing item , is prepared in vitro, and is introduced into Coryneform bacteria.  By the<br><br>
introduction, the intracellular concentration of the DTSR protein in the cells is increased. The general means for this purpose is to enhance the intracellular expression of the dtsR gene or to increase the number of copies of the dtsR gene in the cells.<br>
To enhance the intracellular expression of the dtsR gene, the gene is ligated downstream of a strong promoter.<br>
As strong promoters functioning intracellularly in Coryneform bacteria, known are the lac promoter, tac promoter and Trp promoter derived from Escherichia coli [see Y. Morinaga, M. Tsuchiya, K. Miwa and K. Sano, J. Biotech., 5_, 305-312 (1087)]. In addition to these, the trp promoter derived from a Coryneform bacterium is also preferred (see Japanese Patent Laid-Open No. 62-195294).<br>
The DNA containing the dtsR gene and the DNA containing such a promoter are prepared separately, and these are ligated to each other in vitro. To cleave these DNAs and to ligate them together, restriction enzymes and a ligase are employed. The recombinant DNA to be obtained by the ligation is then introduced into cells of a Coryneform bacterium. For the introduction, the same process as that referred to in the<br><br>
foregoing item  can be employed.<br>
To introduce the recombinant DNA comprising a strong promoter and the dtsR gene into cells of a Coryneform bacterium, a vector functioning in the cells of the Coryneform bacterium must be used. When the vector referred to in item  is used in this step, the recombinant DNA is held outside the chromosome of the cell. In order to make the recombinant DNA hold onto the chromosomal DNA in the cell, a temperature-sensitive plasmid such as that described in Japanese Patent Laid-Open No. 5-7491 is used as the vector, and the cells are cultivated at a nonpermissible temperature to cause homologous recombination.<br>
The expression of the dtsR gene by the thus-obtained cells of the Coryneform bacterium having therein the recombinant DNA comprising a strong promoter and the dtsR gene is enhanced and the intracellular concentration of the DTSR protein in these cells is increased.<br>
When the DNA containing the dtsR gene is ligated to a multi-copy plasmid and introduced into cells of a Coryneform bacterium, the number of copies of the said gene in the cells is increased.  As examples of such a multi-copy plasmid,<br><br>
those mentioned in item  are referred to.<br>
Apart from the above-mentioned process, also employable is a process for causing homologous recombination by using, as the target sequence that exists in large numbers on the chromosomal DNA of cells of a Coryneform bacterium. As one example of the sequence much existing on the chromosomal DNA of cells of the Coryneform bacterium, mentioned is an insertion sequence existing at both ends of the transposal element of the cells of the Coryneform bacterium. Said sequence and a method of causing such homologous recombination by using said sequence are disclosed in International Laid-Open Pamphlet WO 93/18151.<br>
The expression of the dtsR gene by the thus-obtained cells of a Coryneform bacterium where the number of copies of the dtsR gene has been increased is enhanced, and the intracellular concentration of the DTSR protein in these cells is increased.<br>  Production of L-lysine by Coryneform bacteria having the recombinant DNA therein:<br>
Various artificial mutants have heretofore been known as<br><br>
L-lysine-producing bacteria. Using these as the hosts, the recombinant DNA of the present invention is introduced into them with the result that their L-lysine productivity is improved. Such artificial mutants are as follows: S-(2-aminoethyl)-cysteine (hereinafter referred to as "AEC")-resistant mutants; mutants requiring amino acids such as L-homoserine for their growth (see Japanese Patent Publication Nos. 48-28078 and 56-6499); mutants resistant to AEC and requiring amino acids such as L-leucine, L-homoserine, L-proline, L-serine, L-arginine, L-alanine, L-valine, etc. (see U.S. Patents 3,708,395 and 3,825,472); L-lysine-producing mutants resistant to DL-a-amino-6-caprolactam, a-amino-lauryl-lactam, aspartic acid analogues, sulfa drugs, quinoids and N-lauroyl-leucine, and L-lysine-producing mutants resistant to oxaloacetate decarboxylase or respiratory system enzyme inhibitors (see Japanese Patent Laid-Open Nos. 50-53588, 50-31093, 52-102498, 53-9394, 53-86089, 55-9783, 55-9759, 56-32995, 56-39778, Japanese Patent Publication Nos. 53-43591, 53-1833); L-lysine-producing mutants requiring inositol or acetic acid (see Japanese Patent Laid-ODen Nos.  55-9784.  56-8692}:<br><br>
L-lysine-producing mutants sensitive to fluoropyruvic acid or to temperatures of 34°C or higher (see Japanese Patent Laid-Open No. 53-86090); L-lysine-producing mutants of Brevibacterium or Corynebacterium resistant to ethylene glycol (see U.S. Patent 4,411,997).<br>
As specific examples, the following strains are referred to.<br>
Brevibacterium lactofermentum AJ 12031 (FERM BP-277);<br>
see Japanese Patent Laid-Open No. 60-62994.<br>
Brevibacterium lactofermentum ATCC 39134; see Japanese<br>
Patent Laid-Open No. 60-62994.<br>
Corynebacterium glutamicum AJ 3463 (FERM P-1987); see<br>
Japanese Patent Publication No. 51-34477.<br>
Brevibacterium lactofermentum AJ 12435 (FERM BP-2294);<br>
see U.S Patent 5,304,476.<br>
Brevibacterium lactofermentum AJ 12592 (FERM BP-3239);<br>
see U.S Patent 5,304,476.<br>
Corynebacterium glutamicum AJ 12596 (FERM BP-3242); see<br>
U.S Patent 5,304,476.<br>
In the cells of the Coryneform bacterium to be obtained<br>
by introducing the recombinant DNA of the present invention<br>
i<br><br>
into these L-lysine-producing bacteria according to the process in the foregoing item , the intracellular concentration of the DTSR protein is increased, and the resultant bacterium has the ability to produce a large amount of L-lysine.<br>
The medium to be used for the production of L-lysine may be ordinary medium containing carbon sources, nitrogen sources, inorganic ions and, if desired, other minor organic nutrients.<br>
As carbon sources, usable are saccharides such as glucose, lactose, galactose, fructose, hydrolysates of starch, etc.; alcohols such as ethanol, inositol, etc.; organic acids such as acetic acid, fumaric acid, citric acid, succinic acid, etc.<br>
As the nitrogen sources, usable are inorganic ammonium salts such as ammonium sulfate, ammonium chloride, ammonium phosphate, etc.; organic nitrogen compounds such as hydrolysates of soybeans, etc.; as well as ammonia gas, aqueous ammonia, etc.<br>
As the inorganic ions, small amounts of potassium phosphate, magnesium sulfate, iron ions, manganese ions, etc.<br><br>
are added to the medium.  It is desirable that suitable amounts of minor organic nutrients are added to the medium. As the minor organic nutrients,  usable are substances required for the growth of the cells, such as vitamin Blf etc., as well as yeast extract, etc.<br>
It is recommended that the cultivation is performed in the medium under an aerobic condition for 16 to 72 hours at a temperature ranging from 30 to 45°C. During the cultivation, the pH of the media is controlled to between 5 and 7. To adjust the pH, inorganic or organic, acidic or alkaline substances, ammonia gas, etc. can be used.<br>
The collection of L-lysine from the fermentation liquid may be effected by an ordinary ion-exchange method, a precipitation method and other known methods in combination.<br> Preparation of Coryneform bacteria in which the expression of dtsR gene on chromosomes is partially or thoroughly repressed:<br>
The dtsR gene has been obtained as a gene that imparts resistance to surfactants to a Coryneform bacteria, as mentioned in the foregoing item .  Therefore, it was<br><br>
expected that the strain with the amplified dtsR gene would no longer produce L-glutamic acid in a medium to which a surfactant was added at a concentration at which the wild strain of a Coryneform bacterium produces L-glutamic acid in the presence of an excess amount of biotin. In consideration of this, the effect of the amplification of the dtsR gene in the production of L-glutamic acid by adding a surfactant was investigated according to the method shown in the examples, which, as was expected, verified the noticeable inhibition of the production of L-glutamic acid. In addition, it was confirmed that the amplification of the dtsR gene also resulted in the diminution of the production of L-glutamic acid in the biotin-restricted method and in the penicillin-adding method in the presence of an excess amount of biotin. These results indicate that the dtsR gene does not only make the strain resistant to surfactants but also plays an important role in the production of L-glutamic acid. For these reasons, it was expected that if, opposite to the above, the expression of the dtsR gene is inhibited and the intracellular concentration or activity of the DTSR protein in the cells is lowered, the ability of the cells to<br><br>
produce L-glutamic acid might be improved and L-glutamic acid might be produced especially in a medium containing an excess amount of biotin without adding any biotin-inhibiting substance such as surfactants or antibiotics. In consideration of this, a strain in which the dtsR gene of the wild strain was disrupted was prepared and the ability of the thus-modified strain to produce L-glutamic acid was examined, resulting in the fact that the dtsR gene-lacking strain produced a large amount of L-glutamic acid even in a medium containing biotin at a high concentration at which the wild strain did almost not produce L-glutamic acid, as is demonstrated in the examples as described below.<br>
The strain in which the expression of the dtsR gene is repressed can be obtained by any chemical mutation method where mutants are derived by the use of mutagens or by a breeding method using recombinant DNA technique. Where the gene has already been obtained, the recombinant DNA technique is employed by which the gene can easily be disrupted by homologous recombination. The means of disrupting a gene by homologous recombination has already been established. For this, employable are a method of using a linear DNA and a<br><br>
method of using a temperature-sensitive plasmid.<br>
As the means of repressing the gene expression, for example, employable are site-specific mutation [see Kramer, W. and Frits, H.J., Methods in Enzymology, 154, 350 (1987)] or mutation with chemicals such as sodium hyposulfite, hydroxylamine, etc. [see Shortle, D. and Nathans, D., Proc. Natl. Acad. Sci. USA, 75, 270 (1978)], which brings about the substitution, deletion, insertion, addition or inversion of one or more nucleotides in the nucleotide sequence of the coding region or of the promoter region in the dtsR gene. The thus-modified or disrupted gene is substituted for the normal gene on the chromosome in the cell, thereby lowering or inactivating the activity of the genetic product, DTSR protein or lowering the transcription of the gene.<br>
In the site-specific mutation method, synthetic oligonucleotides are used, and according to this method, it is possible to apply any desired substitution, deletion, insertion, addition or inversion to only (a) limited base pair(s). To carry out this method, a plasmid having the intended gene which has been cloned and whpse nucleotide sequence has been determined, is first denatured to prepare<br><br>
a single-stranded DNA.  Next, a synthetic oligonucleotide complementary to the site to be mutated is prepared.  The synthetic oligonucleotide shall be such that it does not have a completely complementary sequence but may have any desired base  substitution,  deletion,  insertion,  addition  or inversion.  After this, the single-stranded DNA is annealed with the synthetic oligonucleotide having such a desired base substitution, deletion, insertion, addition or inversion. Then, this is formed into a complete double-stranded plasmid, using the Klenow fragment of DNA polymerase I and T4 ligase. The resulting plasmid is then introduced into the competent cell of Escherichia coli.     Some of these transformants thus obtained have a plasmid containing the gene that has the desired base substitution, deletion, insertion, addition or inversion fixed therein.  As a similar method by which the mutation, modification or disruption of the gene is possible, known is a recombinant PCR method [see PCR Technology, Stockton Press (1989)].<br>
The chemical mutation method is used to directly treat the DNA fragment containing the intended gene with sodium hyposulfite, hydroxylamine or the like chemical thereby<br><br>
randomly introducing the mutation having base substitution, deletion, insertion, addition or inversion into the DNA fragment.<br>
As the means of substituting the thus-obtained gene that has been modified or disrupted by the introduction of the mutation for the normal gene on the chromosome of cells of a Coryneform bacterium, employable is a method of using homologous recombination [see Experiments in Molecular Genetics, Cold Spring Harbor Laboratory Press (1972); Matsuyama, S. and Mizushima, S., J. Bacteriol., 162, 1196 (1985)]. The mechanism of homologous recombination is as follows. When a plasmid or the like having a sequence which is homologous to the sequence of the chromosome of a cell is introduced into the cell, a recombination occurs at the sites having the homologous sequence at a certain frequency whereby the complete plasmid introduced is incorporated into the chromosome of the cell. After this, when a further recombination occurs at the sites having the homologous sequence on the chromosome, the plasmid is removed from the chromosome. By the latter recombination, the gene into which the mutation has been  introduced is  fixed onto  the<br><br>
chromosome, depending on the recombined site, and the original normal gene is often removed from the chromosome along with the plasmid. It is therefore possible to select the thus-modified strain where the gene previously modified or disrupted by the introduction thereinto of the mutation having base substitution, deletion, insertion, addition or inversion has been substituted for the normal gene in the chromosome.<br>  Production of L-glutamic acid by Coryneform bacteria in which the expression of dtsR gene is repressed:<br>
For obtaining a mutant having a lowered concentration of DTSR protein from Coryneform bacteria having the ability to produce L-glutamic acid a wild strain or a mutant derived therefrom can be used as the starting strain. As examples of the mutant, the following are mentioned:<br>
Brevibacterium lactofermentum AJ 12745 (FERM-BP 2922); see Japanese Patent Laid-Open No. 3-49690. Brevibacterium lactofermentum AJ 12746 (FERM-BP 2923); see Japanese Patent Laid-Open No. 3-49690.<br><br>
Brevibacterium flavum AJ 12747 (FERM-BP 2924); see Japanese Patent Laid-Open No. 3-49690. Corynebacterium glutamicum AJ 12748 (FERM-BP 2925); see Japanese Patent Laid-Open No. 3-49690. Corynebacterium glutamicum ATCC 21492.<br>
The medium to be used for the production of L-glutamic acid may be ordinary medium containing carbon sources, nitrogen sources, inorganic ions and, if desired, other minor organic nutrients.<br>
As the carbon sources, usable are saccharides such as glucose, lactose, galactose, fructose, hydrolysates of starch, etc.; alcohols such as ethanol, inositol, etc.; organic acids such as acetic acid, fumaric acid, citric acid, succinic acid, etc.<br>
As the nitrogen sources, usable are inorganic ammonium salts such as ammonium sulfate, ammonium chloride, ammonium phosphate, etc.; organic nitrogen compounds such as hydrolysates of soybeans, etc.; as well as ammonia gas, aqueous ammonia, etc.<br>
As the inorganic ions, small amounts of potassium<br><br>
phosphate, magnesium sulfate, iron ions, manganese ions, etc. are added to the medium. It is desirable that suitable amounts of minor organic nutrients are added to the medium. As the minor organic nutrients, usable are substances required for the growth of the cells in the medium, such as vitamin B1# etc., as well as yeast extract, etc.<br>
It is recommended that the cultivation of the cells is carried out in the medium under an aerobic condition for 16 to 72 hours at a temperature ranging from 30 to 45°C. During the cultivation, the pH of the medium is kept at from 5 to 8. To adjust the pH, inorganic or organic, acidic or alkaline substances, ammonia gas, etc. can be used.<br>
Surfactants or penicillin may be added to the medium where the dtsR gene-disrupted strain thus obtained is cultivated, or the biotin concentration in the medium may be restricted. In this way, the yield of L-glutamic acid to be produced in the medium may be increased further.<br>
The collection of L-glutamic acid from the fermentation broth may be effected by an ordinary ion exchange method, precipitation method and other known methods in combination.<br><br>
Brief Description of the Drawings<br>
Fig. 1 shows the growth of A J 11060/pDTR6 (dtsR gene-amplified strain) and AJ 11060/pSAC4 (control) in a surfactant-free or surfactant-added medium.<br>
0 : pDTR6-introduced strain in surfactant-free medium<br>
□	: pSAC4-introduced strain in surfactant-free medium<br>
9 : pDTR6-introduced strain in surfactant-added medium<br>
■ : pSAC4-introduced strain in surfactant-added medium<br>
Fig. 2 shows the degree of surfactant resistance of ATCC<br>
13869/pSAC4, ATCC 13869/pKCX-KAE and ATCC 13869/pDTR6.<br>
0 : ATCC 13869/pSAC4<br>
A : ATCC 13869/pKCX-KAE<br>
A : ATCC 13869/pDTR6<br>
Fig. 3 shows the growth curve of AJ 11060/pDTR6 and A J 11060/pSAC4 in a medium containing 3 or 300 ug/liter of biotin.<br>
O : pDTR6-introduced strain in the presence of 300 ug/liter of biotin<br>
□	:  pSAC4-introduced strain in the presence of 300<br>
ug/liter of biotin • :  pDTR6-introduced strain in the presence of 3<br><br>
ug/liter of biotin ■:  pSAC4-introduced strain in the presence of 3 ug/liter of biotin<br>
Description of the Preferred Embodiment<br>
The present invention will be described in more detail by means of the following examples.<br>
Example 1 (Preparation of chromosomal DNA of Brevibacterium lactofermentum ATCC 13869 (a wild strain of Coryneform bacteria):<br>
Cells of Brevibacterium lactofermentum ATCC 13869 were inoculated in 100 ml of a T-Y medium [containing 1 % Bacto-tripton (by Difco), 0.5 % Bacto-yeast extract (by Difco) and 0.5 % NaCl; pH 7.2] and cultivated at 31.5°C for 8 hours to obtain a culture. This culture was subjected to centrifugation at 3,000 r.p.m. for 15 minutes to obtain 0.5 g of wet cells. From the wet cells, obtained was the chromosomal DNA according to Saito &amp; Miura method (see Biochem. Biophys. Acta. 72, 619, (1963)). Next, 60 ug of the chromosomal DNA and 3 units of a restriction enzyme, Sau3AI<br><br>
were separately mixed with 10 mM Tris-HCl buffer (containing 50 mM NaCl, 10 mM MgS04 and 1 mM dithiothreitol; pH 7.4), and the reaction was carried out at 37°C for 30 minutes. After the reaction, the reaction mixture was subjected to ordinary phenol extraction and ethanol precipitation to obtain 50 ug of a chromosomal DNA fragment of Brevibacterium lactofermentum ATCC 13869 digested with Sau3AI.<br>
Example 2 (Formation of  gene  library of  Brevibacterium<br>
lactofermentum ATCC 13869, using plasmid vector<br>
DNA):<br>
20 ug of a plasmid vector DNA (pSAC4) capable of<br>
self-amplification in both the cells of Escherichia coli and<br>
the cells of Coryneform bacteria and 200 units of a<br>
restriction enzyme, BamHI were mixed in 50 mM Tris-HCl buffer<br>
(containing 100 mM NaCl and 10 mM magnesium sulfate; pH 7.4)<br>
and reacted at 37CC for 2 hours to obtain a digested product,<br>
which was then subjected to ordinary phenol extraction and<br>
ethanol precipitation.  After this, the DNA fragment was<br>
dephosphorylated  with  bacterial  alkaline  phosphatase<br>
according to the method described in Molecular Cloning, 2nd<br><br>
Edition [J. Sambrook, E.F. Fritsch and T. Maniatis, Cold Spring Harbor Laboratory Press, p.1.56 (1989)] so as to prevent the re-binding of the plasmid vector-derived DNA fragment, and then the thus-dephosphorylated DNA fragment was subjected to ordinary phenol extraction and ethanol precipitation.<br>
One ug of this pSAC4 digested with BamHI, 1 ug of the chromosomal DNA fragment of Brevibacterium lactofermentum ATCC 13869 digested with Sau3AI, that had been obtained in Example 1, and 2 units of T4 DNA ligase (produced by Takara Shuzo Co., Ltd.) were added to 66 mM Tris-HCl buffer (pH 7.5) containing 66 mM magnesium chloride, 10 mM dithiothreitol and 10 mM ATP and reacted therein at 16 °C for 16 hours to conduct the ligation of the DNA. Next, cells of Escherichia coli DH5 were transformed with said DNA mixture by an ordinary method, and the resulting transformant cells were inoculated onto an L-agar medium containing 170 ug/ml of chloramphenicol to obtain about 20,000 colonies constituting a gene library.<br>
Example 3 (Recovery of recombinant DNA from gene library):<br>
From these approximately 20,000 colonies mentioned<br><br>
above, the recombinant DNA was recovered according to the above-mentioned Saito and Miura method.<br>
Example 4 (Transformation of Brevibacterium lactofermentum AJ 11060):<br>
The recombinant DNA mixture was divided into 50 batches, which were introduced into cells of Brevibacterium lactofermentum AJ 11060, a mutant more sensitive to surfactants, by ordinary transformation using electric pulse (see Japanese Patent Laid-Open No. 2-207791). The resulting transformant cells were inoculated onto a glucose-added L-agar medium and the cultivation was performed by static incubation at 31.5°C, whereby about 20,000 colonies of transformants were formed on the medium. Next, these transformant colonies were replicated to the same plate medium but containing 30 mg/liter of the surfactant, from which obtained were several strains that were resistant to the surfactant and grown on the plate medium.<br>
Example 5 (Measurement of surfactant resistance of strains having multi-copies of dtsR gene):<br><br>
The recombinant DNA was extracted from each of the several strains that had been grown on the above-mentioned plate medium with which Brevibacterium lactofermentum AJ 11060 was re-transformed. From the resulting transformants, one surfactant-resistant strain was selected. The recombinant DNA of this strain is referred to as pDTR6, and the gene which is carried by this plasmid and which has the ability to make the strain resistant to the surfactant is referred to as dtsR. The inhibition of the growth of AJ 11060/pDTR6, into which the plasmid had been introduced, in a surfactant(3 g/liter)-added liquid medium is not observed, i.e., the pDTR6-introduced strain can grow well even in the presence of a surfactant (see Fig. 1).<br>
Example 6 (Preparation of pDTR6):<br>
The plasmid was prepared from A J 11060/pDTR6 having the recombinant DNA, that had been obtained in the above, by an ordinary method, and this was introduced into Escherichia coli JM 109. Escherichia coli JM 109/pDTR6 was cultivated in 20 ml of a medium comprising 1 % of tryptone, 0.5 % of yeast extract and 0.5 % of NaCl, at 37°C for 24 hours, and 20 ml of<br><br>
the resulting culture was inoculated on one liter of a medium having the same composition as above and cultivation was performed at 37 °C for 3 hours. Then, 0.2 g of chloramphenicol were added thereto, and the cultivation was continued for 20 hours more at the same temperature. Next, the resulting culture was centrifuged at 3,000 r.p.m. for 10 minutes to obtain 2 g of wet cells. The obtained cells were suspended in 20 ml of 350 mM Tris-HCl buffer (pH 8.0) containing 25 % of sucrose, to which added were 10 mg of lysozyme (produced by Sigma Co.), 8 ml of 0.25 M EDTA solution (pH 8.0) and 8 ml of 20 % sodium dodecylsulfate solution. Then, the resulting suspension was heated at 60°C for 30 minutes to obtain a lysate. 13 ml of 5 M NaCl solution were added to the lysate, which was then treated at 4°C for 16 hours. After the treatment, this was centrifuged at 15,000 r.p.m. for 30 minutes. The supernatant thus obtained was subjected to ordinary phenol extraction and ethanol precipitation to obtain a DNA precipitate.<br>
The precipitate was dried under reduced pressure and then dissolved in 6 ml of 10 mM Tris-HCl buffer (pH 7.5) containing 1 mM EDTA, to which added were 6 g of cesium<br><br>
chloride and 0.2 ml of ethldlum bromide (19 mg/ml). Then, this was subjected to equilibrium density gradient centrifugation, using an ultracentrifugater, at 39,000 r.p.m. for 42 hours, by which the DNA was isolated. Next, ethidium bromide was removed from this, using n-butanol, and thereafter this was subjected to dialysis against 10 mM Tris-HCl (pH 7.5) containing 1 mM EDTA to obtain about 500 ug of a purified recombinant DNA, pDTR6. Escherichia coli JM109/pDTR6 has the number AJ 12967. This strain was deposited in National Institute of Bioscience and Human-Technology, Agency of Industrial Science and Technology, Ministry of International Trade and Industry, Japan, with the deposition number FERM P-14168, and then transferred the deposit under the Budapest Treaty with the deposition number FERM BP-4994.<br>
Example 7 (Analysis of nucleotide sequence of DNA having dtsR gene):<br>
The nucleotide sequence of the recombinant DNA obtained in Example 6 was determined. The sequencing was effected according to the Sanger method,  using a Taq DyeDeoxy<br><br>
Terminator Cycle Sequencing Kit (produced by Applied Biochemical Co.). The nucleotide sequence of the DNA thus obtained is shown by Sequence ID No. 1 in the Sequence Listing. From this nucleotide sequence, the open reading frame was estimated. The amino acid sequence of the product to be estimated from this nucleotide sequence is also shown in Sequence ID No. 1 in the Sequence Listing. From the analysis of the promoter consensus sequence in the promoter region, the open reading frame is considered to be from the base 467 (A) to the base 1987 (G) in Sequence ID No. 1 in the Sequence Listing, i.e., the translation of dtsR gene starts at bases 467-469 (ATG codon). The protein encoded by this open reading frame is referred to as DtSR protein. However, there is another initiation codon (ATG) at bases 359-361, and the open reading frame may be from the base 359 (A) to the base 1987 (G). It is well known that the N-terminal methionine residue of protein is deleted after translation by the action of peptidase. This is because the N-terminal methionine is derived from the translation initiation codon, ATG, and therefore has no relation to the intrinsic function of protein in most cases.  Also in the DTSR protein of the<br><br>
present invention, there is a probability that the methionine residue is deleted.<br>
The nucleotide sequence and the amino acid sequence were compared with known sequences with respect to the homology. The data bases used for the comparison were EMBL and SWISS-PROT. As a result, it has been confirmed that the gene represented by Sequence ID No. 1 in the Sequence Listing and the protein encoded by this are novel.<br>
Example 8 (Identification of dtsR gene as having the ability to impart surfactant resistance):<br>
The open reading frame was confirmed by the nucleotide sequencing, which suggested the existence of DTSR protein. In order to further confirm that the dtsR gene has the ability to impart surfactant resistance to Coryneform bacteria, a part of nucleotides in the region of the open reading frame of the dtsR gene was deleted in-frame from the DNA fragment of represented by Sequence ID No. 1 in the Sequence Listing. This DNA fragment was investigated as to whether or not it has an activity of imparting surfactant resistance to Coryneform bacteria.  Concretely, pDTR6 was<br><br>
digested with Xbal and Kpnl to obtain a fragment containing dtsR gene. This DNA fragment was ligated to a fragment of plasmid pHSG398 (obtained from Takara Shuzo Co., Ltd.) that had been treated with Xbal and Kpnl, using T4DNA ligase (produced by Takara Shuzo Co., Ltd.), to prepare plasmid pHSGX-K. The dtsR gene has two sites that are digested with Eco52I, at the bases 766 and 1366 in Sequence ID No. 1. pHSGX-K was completely digested with Eco52I and then self-ligated to prepare plasmid pHSGX-KAE from which 600 base pairs of the Eco52l fragments were deleted. The structure of the dtsR gene on this pHSGX-KAE was such that the center part was deleted in-frame. Next, in order to make pHSGX-KAE capable of self-amplification in Coryneform bacteria, the replication origin derived from a known plasmid, pHM1519, capable of self-amplification in Coryneform bacteria (see Miwa, K. et al., Agric. Biol. Chem. 48, 2901-2903 (1984); Japanese Patent Laid-open No. 5-7491) was introduced into the only one Kpnl cleavage site existing on pHSGX-KAE. Concretely, pHM1519 was digested with restriction enzymes, BamHl and Kpnl to obtain a gene fragment having a replication origin, the resulting fragment was made to have blunt ends,<br><br>
using a blunting kit produced by Takara Shuzo Co., Ltd, and then introduced into the Kpnl site of pHSGX-KAE, using a Kpnl linker (produced by Takara Shuzo Co., Ltd.), to obtain pKCX-KAE. As a control sample, the replication origin of pHM1519 was inserted into the Sail site of pHSG399, using Sail linker (produced by Takara Shuzo Co., Ltd.) to prepare pSAC4. These pKCX-KAE and pSAC4 prepared herein were separately introduced into cells of a wild strain of Coryneform bacteria, Brevibacterium lactofermentum ATCC 13869, according to the above-mentioned electric pulse method, and the resulting transformants were examined with respect to the degree of surfactant resistance. Concretely, the cells were cultivated in a M-CM2G liquid medium, to which from 0 to 10 mg/dl of polyoxyethylene sorbitan monopalmitate had been added, and the degree of the cell growth in the culture was measured. The biotin demand by AJ 11060/pDTR6 (dtsR gene-amplified strain) was reduced (see Fig. 2). As a result, it was verified that the deletion-type dtsR gene had lost the function to impart the surfactant resistance to the cells.<br><br>
Example 9 (Cultivation of dtsR gene-amplified strain by biotin-restricted method for production of L-glutamic acid):<br>
Brevibacterium lactofermentum AJ 11060/pDTR6 was cultivated to produce L-glutamic acid, according to the biotin-restricted method mentioned below. Concretely, A J 11060/pDTR6 was separately cultivated on an M-CM2G plate medium containing 4 mg/liter of chloramphenicol. Then, the grown cells were inoculated in a medium containing 80 g of glucose, 1 g of KH2P04, 0.4 g of MgS04.7H20, 30 g of (NH4)2S04, 0.01 g of FeS04.7H20, 0.01 g of MnS04.7H20, 15 ml of soybean hydrolysate, 200 ug of thiamine hydrochloride, 60 ug of biotin, 4 mg of chloramphenicol and 50 g of CaC03 in one liter of pure water (the pH of the medium having been adjusted to 7.0 with KOH) and cultivated therein at 31.5°C for 20 hours. The resulting culture was inoculated in the same medium as above but not containing biotin (this medium is hereinafter referred to as a "biotin-restricted medium"), at a concentration of 5 % by volume, and the cultivation was carried out at 31.5°C for about 20 hours.<br>
While   cultivating   AJ   11060/pDTR6   in   the<br><br>
biotin-restricted medium, the amount of biotin required by the unit weight of the cells grown in the medium was measured (see Fig. 3). As a control, AJ 11060/pSAC4 was cultivated in the same manner as above.<br>
The amounts of the grown cells of AJ 11060/pDTR6 were larger than that of the control, AJ 11060/pSAC4. Thus, it was noted that the biotin demand per the unit weight of the grown cells of AJ 11060/pDTR6 was lowered than that of the control, AJ 11060/pSAC4.<br>
Example 10     (Cultivation of dtsR gene-amplified strain by<br>
surfactant-added method for production of L-glutamic acid):<br>
Brevibacterium lactofermentum AJ 11060 can produce a large amount of L-glutamic acid even in a medium containing a high concentration of biotin, if a surfactant is added to the medium. However, since dtsR gene-amplified strains have elevated surfactant resistance, it was considered that the production of L-glutamic acid by these strains will be decreased in such a medium, even though a surfactant is added to the medium.   A J 11060/pSAC4 and A J 11060/pDTR6 was<br><br>
cultivated to produce L-glutamic acid, according to the surfactant-added method mentioned below. Concretely, the strains were separately cultivated on an M-CM2G plate medium containing 4 mg/liter of chloramphenicol. Then, the cells were inoculated onto a medium containing 80 g of glucose, 1 g of KH2P04, 0.4 g of MgS04.7H20, 30 g of (NH4)2S04, 0.01 g of FeS04.7H20, 0.01 g of MnS04.7H20, 15 ml of soybean hydrolysate, 200 ug of thiamine hydrochloride, 300 ug of biotin, 4 mg of chloramphenicol, 3.0 g of polyoxyethylene sorbitan monopalmitate and 50 g of CaC03 in one liter of pure water (the pH of the medium having been adjusted to 8.0 with KOH) and cultivated therein at 31.5°C for 20 hours.<br>
After the cultivation, the amount of L-glutamic acid produced and accumulated in each culture was measured. The yield obtained is shown in Table 1.<br><br><br>
Example 11     (Cultivation of dtsR gene-amplified strain by<br>
penicillin-added method for production of L-glutamic acid):<br>
AJ 11060/pSAC4 and AJ 11060/pDTR6 were cultivated to produce L-glutamic acid by the penicillin-added method in the same manner as in Example 10, except that 30 units of penicillin G were added to the medium in place of polyoxyethylene sorbitan monopalmitate. The yields obtained are shown in Table 2.<br><br>
Example 12     (Production of dtsR gene-disrupted strain):<br>
Since it has been found that the production of L-glutamic acid is decreased by the amplification of the dtsR gene, it was expected that the disruption of the dtsR gene would result in an increase in the yield of L-glutamic acid. The gene-disrupted strain was obtained by the homologous<br><br>
recombination method disclosed in Japanese Patent Laid-open No. 5-7491 using a temperature-sensitive plasmid, pHSC4. Concretely, the replication origin of pHSC4, which had been obtained from a plasmid capable of self-amplification in Coryneform bacteria and of which the self-amplification had become temperature-sensitive, was introduced into the Kpnl recognition site of pHSGX-KAE shown in Example 8, to form plasmid pKTCX-KAE.<br>
This pKTCX-KAE was introduced into a wild strain, Brevibacterium lactofermentum ATCC 13869 by an electric pulse method, and the dtsR gene on the chromosome of the strain was substituted by a deleted-type dtsR gene according to the method described in Japanese Patent Laid-open No. 5-7491. Concretely, ATCC 13869/pKTCX-KAE was cultivated by shaking in a M-CM2G liquid medium containing 50 ug/ml of oleic acid at 25°C for 6 hours, and the culture was inoculated onto an M-CM2G medium containing 5 ug/ml of chloramphenicol and 50 ug/ml of oleic acid. The plasmid-inserted strains formed colonies at 34 °C, and these were collected. From the thus-obtained strains, those that had become sensitive to chloramphenicol at 34 °C were selected by a replication method<br><br>
on the same medium. The chromosomes of these sensitive strains were obtained by an ordinary method, and the structure of the dtsR gene of each of these chromosomes was analyzed by the Southern hybridization method (Southern, E.M., J. Mol. Biol., 98_, 503 (1975)). Thus, the AE strain where the dtsR gene had been substituted by a deleted-type dtsR gene was obtained.<br>
Example 13     (L-Glutamic acid productivity of AE strain):<br>
Three strains, ATCC 13869, AJ 11060 and AE, were cultivated to produce L-glutamic acid in the manner mentioned below. Concretely, these strains were refreshed by cultivating on an M-CM2G plate medium, respectively. The thus-refreshed cells were inoculated into a medium containing 80 g of glucose, 1 g of KH2P04, 0.4 g of MgS04, 30 g of (NH4)2S04, 0.01 g of FeS04.7H20, 0.01 g of MnS04.7H20, 15 ml of soybean hydrolysate, 200 ug of thiamine hydrochloride, 300 ug of biotin, 1 g of Tween 80 (polyoxyethylene sorbitan monooleate) and 50 g of CaC03 in one liter of pure water (the pH of the medium having been adjusted to 8.0 with KOH), and the cultivation was carried out at 31.5°C for 20 hours.  The<br><br>
results are shown in Table 3.<br><br>
Example 14    (Production   of   L-lysine   by   dtsR<br>
gene-amplified strain):<br>
pDTR6 was introduced into an L-lysine-producing strain of Coryneform bacteria, Brevibacterium 1actofermenturn A J 12435 (FERM BP-2294), by an electric pulse method. The resulting transformant was cultivated in a medium mentioned below to produce L-lysine.<br>
Concretely, the strain was cultivated on an M-CM2G plate medium containing 4 mg/liter of chloramphenicol, and the thus-obtained cells were inoculated in a medium containing 100 g of glucose, 1 g of KH2P04, 0.4 g of MgS04, 30 g of (NH4)2S04, 0.01 g of FeS04.7H20, 0.01 g of MnS04.7H20, 15 ml of soybean hydrolysate, 200 ug of thiamine hydrochloride, 300 ug<br><br>
of biotin, 4 mg of chloramphenicol and 50 g of CaC03 in one liter of pure water (the pH of the medium having been adjusted to 7.0 with KOH) at 32°C for 40 hours. As a control, AJ 12435/pSAC4 was cultivated in the same manner as above. The accumulated amounts of L-lysine are shown in Table 4.<br><br>
[Advantages of the Invention]<br>
The dtsR gene of the present invention is a gene derived from Coryneform bacteria which are used for the production of L-glutamic acid by fermentation, and this gene plays an important role in the production of L-glutamic acid. Where this gene is amplified in L-lysine-producing Coryneform bacteria, the L-lysine productivity is improved. Where this gene is disrupted in L-glutamic acid-producing Coryneform<br><br>
bacteria, the L-glutamic acid productivity is improved.<br>
The AJ12435 Strain (FERM BP-2294) was deposited under the Budapest Treaty on February 20, 1989 in the National Institute of Bioscience and Human-Technology, Agency of Industrial Science and Technology (currently, International Patent Organism Depositary, National Institute of Advanced Industrial Science and Technology, Central 6, 1-1-1 Higashi, Tsukuba, Ibaraki 305-8566, Japan.<br><br>
[ SEQENCE LISTING ]<br>
Sequence ID No. :   1<br>
Sequence Length :   2855 bp<br>
Sequence Type :    Nucleic Acid<br>
Strandedness :     Double<br>
Topology :	Linear<br>
Molecular Type :   Genomic DNA Feature :<br>
Feature Key :	mat peptide<br>
Position : 467 - 1987<br>
Sequence:<br>
GATCTTGGAA CTCGACAGTT TTCACCGTCC AGTTTGGAGC GCCTGAGCTT GCAAGCTCCA    60<br>
GCAAGTCAGC ATTAGTGGAG CCTGTCACTT TTTCGTAAAT GACCTGGCCA AAGTCACCGT   120<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
We claim: 1. An isolated gene derived from Coryneform bacteria and coding for a protein which has an activity to impart surfactant resistance to said bacteria, wherein the protein has an amino acid sequence from the 37th methionine residue to the 543rd leucine residue of Sequence ID No.l in the Sequence Listing.<br>
2.	The isolated gene according to Claim 1, which has a nucleotide sequence from<br>
base 467 to base 1987 of Sequence ID No. 1 in the Sequence Listing.<br>
3.	An isolated DNA fragment comprising the gene according to Claim 1.<br>
4.	A recombinant DNA to be obtained by ligating a vector functioning in Coryneform bacteria to the DNA fragment according to Claim 3.<br>
5.	A Coryneform bacterium transformed with the recombinant DNA according to Claim 4.<br>
6.	A method for producing L-lysine comprising cultivating a Coryneform bacterium<br>
harboring the recombinant DNA according to Claim 4 and capable of producing L-<br>
lysine in a liquid medium in a known manner to produce and accumulate L-lysine in<br>
the culture.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxLW1hcy0xOTk1IGFic3RyYWN0IGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">251-mas-1995 abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxLW1hcy0xOTk1IGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">251-mas-1995 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxLW1hcy0xOTk1IGNsYWltcyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">251-mas-1995 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxLW1hcy0xOTk1IGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">251-mas-1995 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxLW1hcy0xOTk1IGNvcnJlc3BvbmRlbmNlIG90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">251-mas-1995 correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxLW1hcy0xOTk1IGNvcnJlc3BvbmRlbmNlIHBvLnBkZg==" target="_blank" style="word-wrap:break-word;">251-mas-1995 correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxLW1hcy0xOTk1IGRlc2NyaXB0aW9uIChjb21wbGV0ZSkgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">251-mas-1995 description (complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxLW1hcy0xOTk1IGRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">251-mas-1995 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxLW1hcy0xOTk1IGRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">251-mas-1995 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxLW1hcy0xOTk1IGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">251-mas-1995 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxLW1hcy0xOTk1IGZvcm0tMTkucGRm" target="_blank" style="word-wrap:break-word;">251-mas-1995 form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxLW1hcy0xOTk1IGZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">251-mas-1995 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxLW1hcy0xOTk1IGZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">251-mas-1995 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxLW1hcy0xOTk1IGZvcm0tNC5wZGY=" target="_blank" style="word-wrap:break-word;">251-mas-1995 form-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxLW1hcy0xOTk1IG90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">251-mas-1995 others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxLW1hcy0xOTk1IHBldGl0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">251-mas-1995 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="223520-processes-for-the-preparation-of-thiazolidinedione-derivatives-and-intermediates.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="223522-an-absorbent-product.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>223521</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>251/MAS/1995</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>47/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Nov-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>12-Sep-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>01-Mar-1995</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>AJINOMOTO CO., INC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>15-1, KYOBASHI 1-CHOME, CHUO-KU, TOKYO,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>YASUHIKO YOSHIHARA</td>
											<td>C/O AJINOMOTO CO., INC TECHNOLOGY AND ENGINEERING LABORATORIES, 1-1, SUZUKI-CHO, KAWASAKI-KU, KAWASAKI-SHI KANAGAWA,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>TSUYOSHI NAKAMATSU</td>
											<td>C/O AJINOMOTO CO., INC TECHNOLOGY AND ENGINEERING LABORATORIES, 1-1, SUZUKI-CHO, KAWASAKI-KU, KAWASAKI-SHI KANAGAWA,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>EIICHIRO KIMURA</td>
											<td>C/O AJINOMOTO CO., INC TECHNOLOGY AND ENGINEERING LABORATORIES, 1-1, SUZUKI-CHO, KAWASAKI-KU, KAWASAKI-SHI KANAGAWA,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>CHIZU ABE</td>
											<td>C/O AJINOMOTO CO., INC TECHNOLOGY AND ENGINEERING LABORATORIES, 1-1, SUZUKI-CHO, KAWASAKI-KU, KAWASAKI-SHI KANAGAWA,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>YOSHIO KAWAHARA</td>
											<td>C/O AJINOMOTO CO., INC TECHNOLOGY AND ENGINEERING LABORATORIES, 1-1, SUZUKI-CHO, KAWASAKI-KU, KAWASAKI-SHI KANAGAWA,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/31</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/223521-novel-gene-derived-from-coryneform-bacteria by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:35:16 GMT -->
</html>
