<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/223653-a-vaccine-comprising-a-salmonella-typhi-ty800-cell by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:09:06 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 223653:&quot;A VACCINE COMPRISING A SALMONELLA TYPHI Ty800 CELL&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A VACCINE COMPRISING A SALMONELLA TYPHI Ty800 CELL&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention features a bacterial cell the virulence of which is attenuated by a deletion of a portion of the PhoQ gene and bacterial cells having a deletion of the PhoQ gene and a deletion of the PhoP gene. The invention also features vaccines comprising such bacteria.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>SALMONELLA VACCINES<br>
Background of the Invention<br>
The invention relates to vaccines.<br>
This invention was made with Government support<br>
under Grant No. AI30479 and Grant No. 00917 awarded by<br>
the National Institutes of Health. The Government has<br>
certain rights in the invention.<br>
Enteric fevers and diarrheal diseases, e.g.,<br>
typhoid fever and cholera, are major causes of morbidity<br>
and mortality throughout the developing world, Hook et<br>
al., 1980, In Harrison's Principles of Internal Medicine,<br>
9th Ed., 641-848, McGraw Hill, New York. Traditional<br>
approaches to the development of vaccines for bacterial<br>
diseases include the parenteral injection of purified<br>
components or killed organisms. These parenterally<br>
administered vaccines require technologically advanced<br>
preparation, are relatively expensive, and are often,<br>
because of dislike for needle-based injections, resisted<br>
by patients. Live oral vaccine strains have several<br>
advantages over parenteral vaccines: low cost, ease of<br>
administration, and simple preparation.<br>
The development of live vaccines has often been<br>
limited by a lack of understanding of the pathogenesis of<br>
the disease of interest on a molecular level. Candidate<br>
live vaccine strains require nonrevertable genetic<br>
alterations that affect the virulence of the organism,<br>
but not its induction of an immune response. Work<br>
defining the mechanisms of toxigenesis of Vibrio cholerae<br>
has made it possible to create live vaccine strains based<br>
on deletion of the toxin genes, Mekalanos et al., 1983,<br>
Nature 306:551, Levine et al., 1988, Infect. Immun.<br>
5_6:161.<br>
Recent studies have begun to define the molecular<br>
basis of Salmonella typhimurium macrophage survival and<br>
virulence, Miller et al., 1989, Proc. Natl. Acad. Sci.<br>
USA 8.6:5054, hereby incorporated by reference.<br>
Salmonella typhimurium strains with mutations in the<br>
positive regulatory regulon phoP are markedly attenuated<br>
in virulence for BALB/c mice. The phoP regulon is<br>
composed of two genes present in an operon, termed phoP<br>
and phoQ. The phoP and phoQ gene products are highly<br>
similar to other members of bacterial two-component<br>
transcriptional regulators that respond to environmental<br>
stimuli and control the expression of a large number of<br>
other genes. A mutation at one of these phoP regulatory<br>
region regulated genes, pagC, confers a virulence defect.<br>
Strains with pagC, phoP, or phoQ mutations afford partial<br>
protection to subsequent challenge by wild-type S.<br>
typhimurium.<br>
Salmonella species cause a spectrum of clinical<br>
disease that includes enteric fevers and acute<br>
gastroenteritis, Hook et al., 1980, supra. Infections<br>
with Salmonella species are more common in<br>
immunosuppressed persons, Celum et al., 1987, J. Infect.<br>
Dis. 156:998. S. typhi, the bacterium that causes<br>
typhoid fever, can only infect man, Hook et al., 1980,<br>
supra. The narrow host specificity of S. typhi has<br>
resulted in the extensive use of S. enteriditis<br>
typhimurium infection of mice as a laboratory model of<br>
typhoid fever, Carter et al., 1984 J. Exp. Med. 139:1189.<br>
S. typhimurium infects a wider range of hosts, causing<br>
acute gastroenteritis in man and a disease similar to<br>
typhoid fever in the mouse and cow.<br>
Salmonella infections are acquired by oral<br>
ingestion. The organisms, after traversing the stomach,<br>
replicate in the small bowel, Hornik et al., 1970, N.<br>
Eng. J. Med. 283:686. Salmonella are capable of invasion<br>
of the intestinal mucosal cells, and S. typhi can pass<br>
through this mucosal barrier and spread via the Peyer's<br>
patches to the lamina propria and regional lymph nodes.<br>
Colonization of the reticuloendothelial cells of the host<br>
then occurs after bacteremia. The ability of S. typhi to<br>
survive and replicate within the cells of the human<br>
reticuloendothelial system is essential to its<br>
pathogenesis, Hook et al.,. 1980, supra, Hornick et al.,<br>
1970, supra, and Carter et al., 1984, supra.<br>
Immunity to Salmonella typhi involves humoral and<br>
cell-mediated immunity, Murphy et al., 1987, J. Infect.<br>
Dis. 156:1005, and is obtainable by vaccination, Edelman<br>
et al., 1986, Rev. Inf. Dis. 8.:324. Recently, human<br>
field trials demonstrated significant protective efficacy<br>
against S. typhi infection after intramuscular<br>
vaccination with partially purified Vi antigen, Lanata et<br>
al., 1983, Lancet 2.M41. Antibody-dependent enhancement<br>
of S. typhi killing by T cells has been demonstrated in<br>
individuals who received a live S. typhi vaccine,<br>
indicating that these antibodies may be necessary for the<br>
host to generate a cell-mediated immune response, Levine<br>
et al., 1987, J. Clin. Invest. 7_9:888. The cell-mediated<br>
immune response is important in typhoid immunity since<br>
killed vaccines that do not induce this immune response<br>
are not protective in man, Collins et al., 1972, Infect.<br>
Immun. 41:742.<br>
Summary of the Invention<br>
The invention features a bacterial cell having a<br>
mutation in the PhoQ gene, wherein said mutation<br>
attenuates virulence. In various preferred embodiments<br>
the mutation is a deletion; the cell has a mutation in<br>
the PhoP gene; and the bacterial cell is a Salmonella<br>
cell.<br>
In even more preferred embodiments the Salmonella<br>
cell is a Salmonella typhimurium cell; the Salmonella<br>
cell is a Salmonella enteriditis cell; the Salmonella<br>
cell is a Salmonella pylorum cell; the Salmonella cell is<br>
a Salmonella paratyphi A cell; the Salmonella cell is a<br>
Salmonella paratyphi B cell; the Salmonella cell is a<br>
Salmonella cholerasuis cell; the Salmonella cell is a<br>
Salmonella typhi cell.<br>
In other preferred embodiments the mutation in the<br>
PhoP gene is a deletion. In another preferred embodiment<br>
the mutation in the PhoP gene is a deletion of at least 5<br>
nucleotides; is a deletion of at least 10 nucleotides; is<br>
a deletion of at least 50 nucleotides; is a deletion of<br>
at least 100 nucleotides; is a deletion of at least 750<br>
nucleotides.<br>
In another preferred embodiment the mutation in<br>
the PhoQ gene is a deletion of at least 5 nucleotides; is<br>
a deletion of at least 10 nucleotides; is a deletion of<br>
at least 50 nucleotides; is a deletion of at least 100<br>
nucleotides; is a deletion of at least 750 nucleotides.<br>
In another preferred embodiment the deletion of<br>
the PhoQ gene and the deletion of the PhoP gene comprise<br>
a deletion of nucleotides 376 to 1322 of the PhoP/PhoQ<br>
locus.<br>
In another aspect, the invention features a<br>
vaccine which includes the bacterial cell of having a<br>
mutation in the PhoQ gene which mutation attenuates<br>
virulence. In a preferred embodiment the bacterial cell<br>
is TySOO.<br>
In other preferred embodiments, the PhoQ mutation<br>
is a null mutation and the PhoP mutation is a null<br>
mutation.<br>
The invention provides a Salmonella vaccine which<br>
does not cause transient bacteremia. In general, the<br>
invention features a bacterial cell, preferably a<br>
Salmonella cell, e.g., a S. typhi, S. enteritidis<br>
typhimurium, or S. cholerae-suis cell,' the virulence of<br>
which is attenuated by a first mutation in a PhoP regulon<br>
and a second mutation in an aromatic amino acid synthetic<br>
gene. As used herein, PhoP regulon is defined as a DNA<br>
which comprises a unit of Salmonella virulence gene<br>
expression characterized by two regulatory genes, phoP<br>
and phoQ, and structural genes, the expression of which<br>
is regulated by phoP and phoQ, e.g., phoP regulatory<br>
region repressed genes (prgr) or phoP regulatory region<br>
activated genes (pag) . Such a bacterial cell can be used<br>
as a vaccine to immunize a mammal against salmonellosis.<br>
The Salmonella cell may be of any serotype, e.g.,<br>
S. typhimurium, S. paratyphi A, S. paratyphi B, S.<br>
paratyphi C, S. pylorum, S. dublin, S. heidelberg, S.<br>
newport, S. Minnesota, S. infantis, S. virchov, or S.<br>
panama.<br>
The first mutation may be a non-revertable null<br>
mutation in the PhoP/PhoQ locus. Preferably, the<br>
mutation is a deletion of at least 100 nucleotides; more<br>
preferably, the mutation is a deletion of at least 500<br>
nucleotides; even more preferably, the mutation is a<br>
deletion of at least 750 nucleotides; and, most<br>
preferably, the mutation is a deletion of nucleotides 376<br>
to 1322 of the PhoP/PhoQ regulatory locus.<br>
The second mutation may be a non-revertable null<br>
mutation in an aroA locus or a non-revertable null<br>
mutation in an aroC/aroD locus, or another locus involved<br>
in the biosynthesis of aromatic amino acids.<br>
To further attenuate the virulence of the<br>
bacterial cell of the invention, the cell may contain yet<br>
another mutation, e.g., a deletion, in a non-aromatic<br>
amino acid synthetic gene, e.g., a mutation which renders<br>
the cell auxotrophic for a non-aromatic amino acid, e.g.,<br>
histidine. In preferred embodiments, the bacterial cell<br>
of the invention is a 5. typhi cell with the genotype<br>
AroA~, His", PhoP/PhoQ~, e.g., TyLH445.<br>
The invention may also include a Salmonella cell,<br>
the virulence of which is attenuated by the constitutive<br>
expression of a gene under the control of a two-component<br>
regulatory system. In preferred embodiments the<br>
constitutive expression is the result of a mutation at a<br>
component of the two-component regulatory system. In<br>
preferred embodiments the bacterial cell includes a<br>
second mutation which attenuates virulence.<br>
In yet other preferred embodiments of the vaccine<br>
the two-component regulatory system is the phoP<br>
regulatory region, and the gene under the control of the<br>
two-component system is a phoP regulatory region<br>
regulated gene, e.g., a prg gene, e.g., prgA, prgB, prgC,<br>
prgE, or prgH, or pag gene, e.g., pagC. In preferred<br>
embodiments constitutive expression is the result of a<br>
change or mutation, e.g., a deletion, (preferably a nonrevertible<br>
mutation) at the promoter of the regulated<br>
gene or of the phoP regulatory region, e.g., a mutation<br>
in the phoQ or the phoP gene, e.g., the PhoP0 mutation.<br>
In another aspect, the invention features a<br>
vaccine including a bacterial cell which is attenuated by<br>
decrease of expression of a virulence gene under control<br>
of a phoP regulatory region, e.g., a prg gene, e.g.,<br>
prgA, prgB, prgC, prgE, or prgH.<br>
In preferred embodiments of the vaccine the<br>
Salmonella cell includes a first mutation, e.g., a<br>
deletion, which attenuates virulence, e.g., a mutation in<br>
a phoP regulatory region gene, e.g., a mutation in the<br>
phoP or phoQ gene, e.g., PhoPc, or a mutation in a phoP<br>
regulatory region regulated gene, and a second mutation<br>
which attenuates virulence, e.g., a mutation in an<br>
aromatic amino acid synthetic gene, e.g., an aro gene, a<br>
mutation in a phoP regulatory region regulated gene,<br>
e.g., a mutation in a prg gene, e.g., prgA, prgB, prgC,<br>
prgE, or prgH, or pag locus, e.g., a pagC mutation.<br>
In yet other preferred embodiments the bacterial<br>
cell includes a first mutation in a phoP regulatory<br>
region gene and a second mutation in an aromatic amino<br>
acid synthetic gene, e.g, an aro gene.<br>
In another aspect, the invention features a<br>
vaccine, preferably a live vaccine, including a bacterial<br>
cell, the virulence of which is attenuated by a mutation,<br>
e.g., a deletion, in a gene under the control of a twocomponent<br>
regulatory system. In preferred embodiments<br>
the bacterial cell includes a virulence attenuating<br>
mutation in a second gene, e.g., in an aromatic amino<br>
acid synthetic gene, e.g., an aro gene.<br>
In yet other preferred embodiments of the vaccine<br>
the bacterial cell is a Salmonella cell, the twocomponent<br>
regulatory system is the phoP regulatory<br>
region, and the gene under its control is a prg gene,<br>
e.g. prgA, prgB, prgC, prgE, or prgH, or a pag gene,<br>
e.g., the page gene.<br>
In another aspect the invention features a<br>
vaccine, preferably a live vaccine, including a<br>
Salmonella cell e.g., a 5. typhi, S. enteritidis<br>
typhimurium, or S. cholerae-suis cell, including a first<br>
virulence attenuating mutation in an aromatic amino acid<br>
biosynthetic gene, e.g., an aro gene, and a second<br>
virulence attenuating mutation in a phoP regulatory<br>
region gene, e.g., a phoP~ mutation.<br>
In another aspect the invention features a<br>
bacterial cell, or a substantially purified preparation<br>
thereof, preferably a Salmonella cell, e.g., a S. typhi,<br>
S. enteritidis typhimurium, or S. cholerae-suis cell,<br>
which constitutively expresses a gene under the control<br>
of a two-component regulatory system and which includes a<br>
virulence attenuating mutation, e.g., a deletion, which<br>
does not result in constitutive expression of a gene<br>
under the control of the two-component regulatory system.<br>
In preferred embodiments the bacterial cell includes a<br>
mutation in a component of the two-component regulatory<br>
system.<br>
In preferred embodiments the bacterial cell is a<br>
Salmonella cell which expresses a phoP regulatory region<br>
regulated gene constitutively (the constitutive<br>
expression preferably caused by a mutation, preferably a<br>
non-revertible mutation, e.g., a deletion in the phoP<br>
regulatory region, e.g., a mutation in the phoQ or phoP<br>
gene, e.g., phoPc), and which includes a virulence<br>
attenuating mutation, preferably a non-revertible<br>
mutation, e.g., a deletion, preferably in an aromatic<br>
amino acid synthetic gene, e.g., an aro gene, or in a<br>
phoP regulatory region regulated gene, e.g., a prg gene,<br>
e.g., prgA, prgB, prgC, prgE, or prgH or pag gene, e.g.,<br>
page which does not result in the constitutive expression<br>
of a gene under the control of the phoP regulatory<br>
region.<br>
In another aspect, the invention features a<br>
bacterial cell, or a substantially purified preparation<br>
thereof, e.g., a Salmonella cell, e.g., a S. typhi cell,<br>
an S. enteritidis typhimurium or a S. cholerae-suis cell,<br>
including a virulence attenuating mutation in a gene<br>
regulated by a two-component regulatory system. In<br>
preferred embodiments the virulence attenuating mutation<br>
is in a phoP regulatory region regulated gene, e.g., a<br>
prg gene, e.g., prgA, prgB, prgC, prgE, or prgH or pag<br>
gene, e.g. , pagC.<br>
In preferred embodiments the bacterial cell<br>
includes a second mutation, e.g., in an aromatic amino<br>
acid synthetic gene, e.g., an aro gene, in a phoP<br>
regulatory region gene, e.g., the phoP or phoQ genes, or<br>
in a phoP regulating region regulated gene, e.g., a prg<br>
gene, e.g., prgA, prgB, prgC, prgE, or prgH or a pag<br>
gene, e.g., pagC, which attenuates virulence but which<br>
does not result in constitutive expression of a phoP<br>
regulatory region regulated gene.<br>
The invention also features a live Salmonella<br>
cell, or a substantially purified preparation thereof,<br>
e.g., a S. typhi, S. enteriditis typhimurium, or<br>
S. cholerae-suis cell, in which there is inserted into <br>
virulence gene, e.g., a gene in the phoP regulating<br>
region, or a phoP regulating region regulated gene, e.g.,<br>
a prg gene, e.-g., prgA, prgB, prgC, prgE, or prgH or a<br>
pag locus, e.g., pagC, a gene encoding a heterologous<br>
protein, or a regulatory element thereof.<br>
In preferred embodiments the live Salmonella cell<br>
carries a second mutation, e.g., an aro mutation, e.g.,<br>
an aroA mutation, e.g., aroA" or aroADEL407, that<br>
attenuates virulence.<br>
In preferred embodiments the DNA encoding a<br>
heterologous protein is under the control of an<br>
environmentally regulated promoter. In other preferred<br>
embodiments the live Salmonella cell further includes a<br>
DNA sequence encoding T7 polymerase under the control of<br>
an environmentally regulated promoter and a T7<br>
transcriptionally sensitive promoter, the TV<br>
transcriptionally sensitive promoter controlling the<br>
expression of the heterologous antigen.<br>
The invention also features a vector capable of<br>
integrating into the chromosome of Salmonella including:<br>
a first DNA sequence encoding a heterologous protein; a<br>
second (optional) DNA sequence encoding a marker e.g., <br>
selective marker, e.g., a gene that confers resistance<br>
for a heavy metal resistance or a gene that complements<br>
an auxotrophic mutation carried by the strain to be<br>
transformed; and a third DNA sequence, e.g., a phoP<br>
regulon encoded gene, e.g., a prg gene, e.g., prgA, prgB,<br>
prgC, prgE, or prgH or a pag locus, e.g., pagC, encoding<br>
a phoP regulatory region regulated gene product necessary<br>
for virulence, the third DNA sequence being mutationally<br>
inactivated.<br>
In other preferred embodiments: the first DNA<br>
sequence is disposed on the vector so as to mutationally<br>
inactivate the third DNA sequence; the vector cannot<br>
replicate in a wild-type Salmonella strain; the<br>
heterologous protein is under the control of an<br>
environmentally regulated promoter; and the vector<br>
further includes a DNA sequence encoding TV polymerase<br>
under the control of an environmentally regulated<br>
promoter and a 11 transcriptionally sensitive promoter,<br>
the T7 transcriptionally sensitive promoter controlling<br>
the expression of the heterologous antigen.<br>
In another aspect the invention includes a method<br>
of vaccinating an animal, e.g., a mammal, e.g., a human,<br>
against a disease caused by a bacterium, e.g.,<br>
Salmonella, including administering a vaccine of the<br>
invention.<br>
The invention also includes a vector including DNA<br>
which encodes the pagC gene product; a cell transformed<br>
with the vector; a method of producing the pagC gene<br>
product including culturing the transformed cell and<br>
purifying the pagC gene product from the cell or culture<br>
medium; and a purified preparation of the pagC gene<br>
product.<br>
In another aspect the invention includes a method<br>
of detecting the presence of Salmonella in a sample<br>
including contacting the sample with pagC encoding DNA<br>
and detecting the hybridization of the pagC encoding DNA<br>
to nucleic acid in the sample.<br>
The invention also includes a vector including DNA<br>
which encodes the prgH gene product; a cell transformed<br>
with the vector; a method of producing the prgH gene<br>
product including culturing the transformed cell and<br>
purifying the prgH gene product from the cell or culture<br>
medium; and a purified preparation of the prgH gene<br>
product.<br>
In another aspect the invention includes a method<br>
of detecting the presence of Salmonella in a sample<br>
including contacting the sample with prgH encoding DNA<br>
and detecting the hybridization of the prgH encoding DNA<br>
to nucleic acid in the sample.<br>
In another aspect the invention features a method<br>
of attenuating the virulence of a bacterium, the<br>
bacterium including a two-component regulatory system,<br>
including causing a gene under the control of the twocomponent<br>
system to be expressed constitutively. In<br>
preferred embodiments the bacterium is Salmonella, e.g.,<br>
S. typhi, S. enteritidis typhimurium, or S. choleraesuis,<br>
and the two-component system is the phoP regulatory<br>
region.<br>
In yet another aspect, the invention features a<br>
substantially pure DNA which includes the sequence given<br>
in SEQ ID NO:5 or a fragment thereof.<br>
The invention also includes a substantially pure<br>
DNA which includes a sequence encoding pagD, e.g.,<br>
nucleotides 91 to 354 of SEQ ID NO:5 (pagD open reading<br>
frame (ORF)) and degenerate variants thereof that encode<br>
a product with essentially the amino acid sequence given<br>
in SEQ ID NO:6, as well as the pagD ORF and its 5' noncoding<br>
region, nucleotides 4 to 814 of SEQ ID NO:15)<br>
which contains the pagD promoter. DNA in the region<br>
between the page ORF and the pagD ORF (nucleotides 4 to<br>
814 of SEQ ID NO:15), DNA which includes the page<br>
promoter (nucleotides 562 to 814 of SEQ ID NO:15), and<br>
DNA which includes the pagD promoter alone (nucleotides 4<br>
to 776 of SEQ ID NO:15) are also within the claimed<br>
invention.<br>
The invention also includes a substantially pure<br>
DNA which includes a sequence encoding envE, e.g.,<br>
nucleotides 1114 to 1650 of SEQ ID NO:5 (envE ORF) and<br>
degenerate variants thereof that encode a product with<br>
essentially the amino acid sequence given in SEQ ID NO:7.<br>
Another aspect of the invention features a<br>
substantially pure DNA which includes a sequence encoding<br>
msgA, e.g., nucleotides 1825 to 2064 of SEQ ID NO:5 (msgA<br>
ORF) and degenerate variants thereof which encode a<br>
product with essentially the amino acid sequence given in<br>
SEQ ID NO:8, as well as the msgA ORF with its 5' noncoding<br>
region, nucleotides 1510 to 1824 of SEQ ID NO:5<br>
which contains the msgA promoter. Also within the<br>
invention is a substantially pure DNA comprising the msgA<br>
promoter alone (nucleotides 1510 to 1760 of SEQ ID NO:5).<br>
In yet another aspect, the invention features a<br>
substantially pure DNA which includes a sequence encoding<br>
envF, e.g., nucleotides 2554 to 3294 of SEQ ID N0:5 (envF<br>
ORF) and degenerate variants thereof which encode a<br>
product with essentially the amino acid sequence given in<br>
SEQ ID NO:9, as well as the envF ORF with its 5' noncoding<br>
region, nucleotides 2304 to 2553 of SEQ ID NO:5<br>
containing the envF promoter.<br>
Also within the invention is a substantially pure<br>
DNA which includes the sequence given in SEQ ID NO:10 or<br>
a fragment thereof.<br>
The invention also includes a substantially pure<br>
DNA which includes a sequence encoding prgH, e.g.,<br>
nucleotides 688 to 1866 of SEQ ID NO:10 (prgH ORF) and<br>
degenerate variants thereof which encode a product with<br>
essentially the amino acid sequence given in SEQ ID<br>
NO:11, as well as the prgH ORF with its promoter region<br>
(nucleotides 1 to 689 of SEQ ID NO:10).<br>
The invention also includes a substantially pure<br>
DNA which includes a sequence encoding prgl, e.g.,<br>
nucleotides 1891 to 2133 of SEQ ID NO:10 (prgl ORF) and<br>
degenerate variants thereof which encode a product with<br>
essentially the amino acid sequence given in SEQ ID<br>
NO:12, as well as the prgl ORF with its promoter region<br>
(nucleotides 1 to 689 of SEQ ID NO:10.<br>
In another aspect, the invention features a<br>
substantially pure DNA which includes a sequence encoding<br>
prgJ, e.g., nucleotides 2152 to 2457 of SEQ ID NO:10<br>
(prgJ ORF) and degenerate variants thereof which encode a<br>
product with essentially the aittino acid sequence given in<br>
SEQ ID NO:13, as well as the prgJ ORF and its promoter<br>
region (nucleotides 1 to 689 of SEQ ID NO:10.<br>
In yet another aspect, the invention features a<br>
substantially pure DNA which includes a sequence encoding<br>
prgK, e.g., nucleotides 2456 to 3212 of SEQ ID NO:10<br>
(prgK ORF) and degenerate variants thereof which encode a<br>
product with essentially the amino acid sequence given in<br>
SEQ ID NO:14, as well as the prgK ORF with its promoter<br>
region (nucleotides 1 to 689 of SEQ ID NO:10.<br>
The invention also encompasses a bacterial cell<br>
the virulence of which is attenuated by a mutation, e.g.,<br>
a deletion, in one or more genes selected from the group<br>
consisting of pagD, pagE, pagF, pagG, pagH, pagl, pagj,<br>
pagK, pagL, pagM, pagN, pagP, envE, and envF. Also<br>
included is a bacterial cell which is attenuated by a<br>
mutation, e.g., a deletion, in one or more genes selected<br>
from the group consisting of page, pagD, pagJ, pagK,<br>
pagM, and msgA. A bacterial cell, the virulence of<br>
which is attenuated by a mutation, e.g., a deletion, in<br>
one or more genes selected from the group consisting of<br>
prgH, prgl, prgJ, and prgK is also within the claimed<br>
invention.<br>
Two-component regulatory system, as used herein,<br>
refers to a bacterial regulatory system that controls the<br>
expression of multiple proteins in response to<br>
environmental signals. The two-components referred to in<br>
the term are a sensor, which may, e.g., sense an<br>
environmental parameter and in response thereto promote<br>
the activation, e.g. by promoting the phosphorylation, of<br>
the second component, the activator. The activator<br>
affects the expression of genes under the control of the<br>
two-component system. A two-component system can<br>
include, e.g., a histidine protein kinase and a<br>
phosphorylated response regulator, as is seen in both<br>
gram positive and gram negative bacteria. In E. coli,<br>
e.g., 10 kinases and 11 response regulators have been<br>
identified. They control chemotaxis, nitrogen<br>
regulation, phosphate regulation, osmoregulation,<br>
sporulation, and many other cellular functions, Stock et<br>
al., 1989 Microbiol. Rev. 5_3:450-490, hereby incorporated<br>
by reference. A two-component system also controls the<br>
virulence of Agrobacterium tumefasciens plant tumor<br>
formation, Leroux et al. EMBO J 6_: 849-856, hereby<br>
incorporated by reference). Similar virulence regulators<br>
are involved in the virulence of Bordetella pertussis<br>
Arico et al., 1989, Proc. Natl. Acad. Sci. USA 86;6671-<br>
6675, hereby incorporated by reference, and Shigella<br>
flexneri, Bernardini et al., 1990, J. Bact. 172:6274-<br>
6281, hereby incorporated by reference.<br>
Environmentally regulated, as used herein refers<br>
to a pattern of expression wherein the expression of a<br>
gene in a cell depends on the levels of some<br>
characteristic or component of the environment in which<br>
the cell resides. Examples include promoters in<br>
biosynthetic pathways which are turned on or off by the<br>
level of a specific component or components, e.g., iron,<br>
temperature responsive promoters, or promoters which are<br>
expressed more actively in specific cellular<br>
compartments, e.g., in macrophages or vacuoles.<br>
A vaccine, as used herein, is a preparation<br>
including materials that evoke a desired biological<br>
response, e.g., an immune response, in combination with a<br>
suitable carrier. The vaccine may include live organism,<br>
in which case it is usually administered orally, or<br>
killed organisms or components thereof, in which case it<br>
is usually administered parenterally. The cells used for<br>
the vaccine of the invention are preferably alive and<br>
thus capable of colonizing the intestines of the<br>
inoculated animal.<br>
A mutation, as used herein, is any change (in<br>
comparison with the appropriate parental strain) in the<br>
DNA sequence of an organism. These changes can arise<br>
e.g., spontaneously, by chemical, energy e.g., X-ray, or<br>
other forms of mutagenesis, by genetic engineering, or as<br>
a result of mating or other forms of exchange of genetic<br>
information. Mutations include e.g., base changes,<br>
deletions, insertions, inversions, translocations or<br>
duplications.<br>
A mutation attenuates virulence if, as a result of<br>
the mutation, the level of virulence of the mutant cell<br>
is decreased in comparison with the level in a cell of<br>
the parental strain, as measured by (a) a significant<br>
(e.g., at least 50%) decrease in virulence in the mutant<br>
strain compared to the parental strain, or (b) a<br>
significant (e.g., at least 50%) decrease in the amount<br>
of the polypeptide identified as the virulence factor in<br>
the mutant strain compared to the parental strain.<br>
A non-revertible mutation, as used herein, is a<br>
mutation which cannot revert by a single base pair<br>
change, e.g., deletion or insertion mutations and<br>
mutations that include more than one lesion, e.g., a<br>
mutation composed of two separate point mutations.<br>
The phoP regulatory region, as used herein, is a<br>
two-component regulatory system that controls the<br>
expression of pag and prg genes. It includes the phoP<br>
locus and the phoQ locus.<br>
phoP regulatory region regulated genes, as used<br>
herein, refer to genes such as pag and prg genes.<br>
pag, as used herein, refers to a gene which is<br>
positively regulated by the phoP regulatory region.<br>
prg, as used herein, refers to a gene which is<br>
negatively regulated by the phoP regulatory region.<br>
An aromatic amino acid synthetic gene, as used<br>
herein, is a gene which encodes an enzyme which catalyzes<br>
a step in the synthesis of an aromatic amino acid. aroA,<br>
aroC, and aroD are examples of such genes in Salmonella.<br>
Mutations in these genes can attenuate virulence without<br>
the total loss of immunogenicity.<br>
Abnormal expressions, as used herein, means<br>
expression which is higher or lower than that seen in<br>
wild type.<br>
Heterologous protein, as used herein, is a protein<br>
that in wild type, is not expressed or is expressed from<br>
a different chromosomal site, e.g., a heterologous<br>
protein is one encoded by a gene that has been inserted<br>
into a second gene.<br>
Virulence gene, as used herein, is a gene the<br>
inactivation of which results in a Salmonella cell with<br>
less virulence than that of a similar Salmonella cell in<br>
which the gene is not inactivated. Examples include the<br>
phoP, pagC, prgH genes.<br>
A marker, as used herein, is gene product the<br>
presence of which is easily determined, e.g., a gene<br>
product that confers resistance to a heavy metal or a<br>
gene product which allows or inhibits growth under a<br>
given set of conditions.<br>
Purified preparation, as used herein, is a<br>
preparation, e.g., of a protein, which is purified from<br>
the proteins, lipids, and other material with which it is<br>
associated. The preparation is preferably at least 2-10<br>
fold purified.<br>
Constitutive expression, as used herein, refers to<br>
gene expression which is modulated or regulated to a<br>
lesser extent than the expression of the same gene in an<br>
appropriate control strain, e.g., a parental or in wildtype<br>
strain. For example, if a gene is normally<br>
repressed under a first set of conditions and derepressed<br>
under a second set of conditions constitutive expression<br>
would be expression at the same level, e.g., the<br>
repressed level, the derepressed level, or an<br>
intermediate level, regardless of conditions. Partial<br>
constitutive expression is included within the definition<br>
of constitutive expression and occurs when the difference<br>
between two levels of expression is reduced in comparison<br>
in what is seen in an appropriate control strain, e.g., a<br>
wild-type or parental strain.<br>
A substantially purified preparation of a<br>
bacterial cell is a preparation of cells wherein<br>
contaminating cells without the desired mutant genotype<br>
constitute less than 10%, preferably less than 1%, and<br>
more preferably less than 0.1% of the total number of<br>
cells in the preparation.<br>
The invention allows for the attenuation of<br>
virulence of bacteria and of vaccines that include<br>
bacteria, especially vaccines that include live bacteria,<br>
by mutations in two-component regulatory systems and/or<br>
in genes regulated by these systems. The vaccines of the<br>
invention are highly attenuated for virulence but retain<br>
immunogenicity, thus they are both safe and effective.<br>
The vectors of the invention allow the rapid<br>
construction of strains containing DNA encoding<br>
heterologous proteins, e.g., antigens. The heterologous<br>
protein encoding DNA is chromosomally integrated, and<br>
thus stable, unlike plasmid systems which are dependent<br>
on antibiotic resistance or other selection pressure for<br>
stability. Live Salmonella cells of the invention in<br>
which the expression of heterologous protein is under the<br>
control of an environmentally responsive promoter do not<br>
express the heterologous protein at times when such<br>
expression would be undesirable e.g., during culture,<br>
vaccine preparation, or storage, contributing to the<br>
viability of the cells, but when administered to humans<br>
or animals, express large amounts of the protein. This<br>
is desirable because high expression of many heterologous<br>
proteins in Salmonella can be associated with toxicity to<br>
the bacterium. The use of only a single integrated copy<br>
of the DNA encoding the heterologous protein also<br>
contributes to minimal expression of the heterologous<br>
protein at times when expression is not desired. In<br>
embodiments where a virulence gene, e.g., the page gene<br>
or the prgH gene, contains the site of integration for<br>
the DNA encoding the heterologous protein the virulence<br>
of the organism is attenuated.<br>
A substantially pure DNA, as used herein, refers<br>
to a nucleic acid sequence, segment, or fragment, which<br>
has been purified from the sequences which flank it in a<br>
naturally occurring state, e.g., a DNA which has been<br>
removed from the sequences which are normally adjacent to<br>
the fragment, e.g., the sequences adjacent to the<br>
fragment in the genome in which it naturally occurs. The<br>
term also applies to DNA which has been substantially<br>
purified from other components which naturally accompany<br>
the DNA, e.g., DNA which has been purified from proteins<br>
which naturally accompany it in a cell.<br>
Other features and advantages of the invention<br>
will be apparent from the following description of the<br>
preferred embodiments and from the claims.<br>
Description of the Preferred Embodiments<br>
The drawings will first be described.<br>
Drawings<br>
Fig. 1 is a graph of the survival of Salmonella<br>
strains within macrophages.<br>
Fig. 2 is a map of the restriction endonuclease<br>
sites of the pagC locus.<br>
Fig. 3 is a map of the DNA sequence of the pagC<br>
region (SEQ ID N0:l ) .<br>
Fig. 4 is a map of the location of prgH within the<br>
hil locus. The arrows indicate the direction of<br>
orientation of the neomycin promoter of Tn5B50 insertions<br>
within the hil locus and the direction of transcription<br>
of the prgHl::TnphoA fusion protein. Restriction<br>
endonuclease sites are represented by B, BamUl; H,<br>
HindiII; X, Xhol; S, Sad; V, £coRV.<br>
Fig. 5 is a DNA sequence from the prgH gene<br>
(plasmid pIBOl) (SEQ ID N0:3).<br>
Fig. 6 is a bar graph showing a comparison of the<br>
sensitivity of wild type (ATCC 14028), PhoP-null mutant<br>
(CS015) , and pagr: rTnphoA mutant strains to NP-1 defensin.<br>
The y-axis represents the Defensin Killing Index (OKI)<br>
which is a measure of bacteria killed on exposure to NP-<br>
1. The OKI is defined as the logarithmic function of the<br>
ratio of control bacteria to surviving bacteria incubated<br>
with NP-1 [DKI=log (CFU without NP-l/CFU with NP-1)].<br>
The individual bars represent the mean and standard error<br>
of five separate experiments. The x-axis indicates the<br>
allele mutated. The mean DKI for each of the pag::TnphoA<br>
strains tested was determined not be different from that<br>
of wild type Salmonella. (P
mutant was significantly different (P
Fig. 7 is a diagram showing a partial physical map<br>
of the restriction endonuclease sites of the pagC<br>
chromosomal region. The mouse 50% lethal doses (LD50) for<br>
strains with transposon insertions in pagD, envE, msgA,<br>
and pagC are shown above each gene. Horizontal arrows<br>
demonstrate the direction of transcription. Vertical<br>
arrows denote TnphoA insertions and the hollow triangle<br>
denotes a MudJ insertion. Below the chromosomal map is a<br>
representation of the DNA Insert in plasmid pCAA9, which<br>
was mutagenized with TnphoA and MudJ. Letter<br>
designations: A, AccI; C, Clal; E, EcoRI; H, //pal; P,<br>
PstI; and V, £coRV.<br>
Fig. 8 is a DNA sequence of the region upstream of<br>
pagC and the translation of each ORF. The Hpal and Clal<br>
sites at the beginning and end of the region are<br>
indicated. Shine-Delgarno regions are underlined and<br>
stem loop structures (potential Rho-independent<br>
terminators) are indicated with a line below and above<br>
the sequence. Arrow heads denote the location of the<br>
representative transposon insertion within each gene.<br>
Horizontal arrows in the pagD and msgA promoter regions<br>
mark the transcriptional start sites, and asterisks mark<br>
the -10 and -35 sequences. The consensus lipid<br>
attachment site in EnvF is enclosed in brackets. The<br>
pagD ORF begins at nucleotide 91 and ends at nucleotide<br>
354 of SEQ ID NO:5; the envE ORF begins at nucleotide<br>
1114 and ends at nucleotide 1650 of SEQ ID NO:5; the msgA<br>
ORF begins at nucleotid 1825 and ends at nucleotide 2064<br>
of SEQ ID NO:5; and the envF ORF begins at nucleotide<br>
2554 and ends at nucleotide 3294 of SEQ ID NO:5.<br>
Fig. 9 is a DNA sequence containing the prgH,<br>
prgl, prgJ, and prgK genes. The start codon (ATG) of<br>
each gene is underlined, and the stop codon is indicated<br>
with an asterisk. The prgH ORF begins at nucleotide 688<br>
and ends at 1866 of SEQ ID NO: 10; the prgrl ORF begins at<br>
nucleotide 1891 and ends at nucleotide 2133 of SEQ ID<br>
NO:10; the prgJ ORF begins at nucleotide 2152 and ends at<br>
nucleotide 2457 of SEQ ID NO:10; and the prgK ORF begins<br>
at nucleotide 2454 and ends at nucleotide 3212 of SEQ ID<br>
N0:10.<br>
Fig. 10 is a line graph showing the growth rates<br>
of the parent Salmonella strain (AroA-) and the vaccine<br>
strain (AroA-, PhoP-).<br>
Fig. 11 is a bar graph showing defensin<br>
sensitivity of mouse vaccine strains (S. typhimurium).<br>
Fig. 12 is a bar graph showing phoP activation as<br>
measured by LacZ activity using the PagBiLacZ recorder<br>
fusion construct.<br>
Fig. 13 is a bar graph showing defensin<br>
sensitivity of S. typhi vaccine strain TyLH445 compared<br>
to the AroA" parent strain.<br>
Fig. 14A is a graph showing the relative<br>
expression of constitutive expression (610 and 617) and<br>
phoP regulated (PagC and pagD) expression of AP fusion<br>
proteins.<br>
Fig. 14B is a graph showing the immune response to<br>
lipopolysaccharide (LPS).<br>
Fig. 14C is a graph showing the immune response to<br>
the model heterologous antigen, AP.<br>
Fig. 15 is a DNA sequence containing the pagC-pagD<br>
intergenic region. pagC translational start site (ATG on<br>
the opposite DNA strand) is underlined (nucleotides 1-3<br>
of SEQ ID NO:15). The pagC transcriptional start<br>
(nucleotide 562) is indicated with an arrow pointing<br>
left. The pagD translational start (ATG) is underlined<br>
(nucleotides 815-817 of SEQ ID NO:15). The pagD<br>
transcriptional start is indicated with an arrow pointing<br>
right (nucleotide 776).<br>
Strain Deposit<br>
Under the terms of the Budapest Treaty on the<br>
International Recognition of the Deposit of<br>
Microorganisms for the Purpose of Patent Procedure,<br>
deposit of the following materials has been made with the<br>
American Type Culture Collection (ATCC) of Rockville, MD,<br>
USA.<br>
Applicant's assignee, Massachusetts General<br>
Hospital, represents that the ATCC is a depository<br>
affording permanence of the deposit and ready<br>
accessibility thereto by the public if a patent is<br>
granted. All restrictions on the availability to the<br>
public of the material so deposited will be irrevocably<br>
removed upon the granting of a patent. The material will<br>
be available during the pendency of the patent<br>
application to one determined by the Commissioner to be<br>
entitled thereto under 37 CFR 1.14 and 35 U.S.C. §122.<br>
The deposited material will be maintained with all the<br>
care necessary to keep it viable and uncontaminated for a<br>
period of at least five years after the most recent<br>
request for the furnishing of a sample of the deposited<br>
plasmid, and in any case, for a period of at least thirty<br>
(30) years after the date of deposit or for the<br>
enforceable life of the patent, whichever period is<br>
longer. Applicants' assignee acknowledges its duty to<br>
replace the deposit should the depository be unable to<br>
furnish a sample when requested due to the condition of<br>
the deposit.<br>
PhoPc strain CS022 (described below) has been<br>
deposited with the American Type Culture Collection<br>
(Rockville, MD) and has received ATCC designation 55130.<br>
The plasmid, pIBOl, containing the prgH gene has<br>
been deposited on July 9, 1993 with the American Type<br>
Culture Collection (Rockville, MD) and has received ATCC<br>
designation ATCC 75496.<br>
Vaccine Having a phoP/phop Deletion<br>
The phoP/phoQ virulence regulatory genes of<br>
Salmonella typhimurium were deleted in Salmonella typhi<br>
Ty2 and the resultant strain (TySOO) was tested as a<br>
live, attenuated typhoid fever vaccine in human<br>
volunteers. Groups of 2-3 volunteers received single<br>
oral doses of 107, 108, 109, or 1010 colony forming units.<br>
Two volunteers receiving the largest dose had<br>
self-limited side effects; significantly, no bacteremias<br>
were detected. Ten of 11 volunteers had evidence of<br>
intestinal immune responses to the vaccine as measured by<br>
increases in S. typhi lipopolysaccharide-specifc<br>
IgA-secreting cells detected in peripheral blood samples.<br>
Humoral inunune responses were measured and compared to<br>
control vaccinees who received 4 oral doses of S. typhi<br>
Ty21a. In the most sensitive assays, 9/11 volunteers and<br>
5/8 Ty21a control vaccinees had evidence of IgG directed<br>
against S. typhi antigens. These experiments demonstrate<br>
that TySOO is safe and single oral doses are highly<br>
immunogenic in human volunteers.<br>
A major advantage of TySOO appears to be the large<br>
doses which may be delivered orally without development<br>
of bacteremias, suggesting that this strain may be<br>
particularly well suited for engineering as a live vector<br>
for delivery of heterologous antigens to the<br>
gastrointestinal lymphoid tissue.<br>
Typhoid fever remains a worldwide health problem<br>
[1]. Ty21a is an FDA-approved, live attenuated oral<br>
typhoid fever which confers approximately 60-70%<br>
protective efficacy in endemic areas (for review see<br>
Levine, Typhoid Fever Vaccines, in Vaccines, Plotkin et<br>
al., eds. W.B. Sanders: Philadelphia, p. 597-633). At<br>
least 3 doses of Ty21a are required to achieve such rates<br>
of efficacy (Levine, supra), and protection may be lower<br>
in naive populations (Hirschell et al., Eur. J. Clin.<br>
Microbiol. 4:925, 1986; Schwartz et al., Arch. Intern.<br>
Med. 150:349, 1990). Development of a live attenuated,<br>
single dose, oral typhoid fever vaccine could<br>
significantly improve public health.<br>
Creation of a defined genetic deletion of the<br>
phoP/phoQ genes in the aromatic amino acid auxotrophic S.<br>
typhi 514Ty (Ty2 DaroA) resulted in strain Ty445. This<br>
candidate vaccine strain was non-reactogenic in human<br>
volunteers receiving 2 oral doses of 5 x 1010 colony<br>
forming units (cfu) but poorly immunogenic, as only 2/14<br>
volunteers receiving 2 maximal doses developed<br>
significant serological responses to S. typhi antigens.<br>
An S. typhi Ty2 strain deleted only for the phoP/phoQ<br>
genes (TySOO) was constructed in an attempt to provide a<br>
more effective vaccine.<br>
Construction of TySOO<br>
S. typhi Ty2 was used as the parent strain for<br>
construction of TySOO. A defined chromosomal deletion of<br>
956 base pairs in the contiguous phoP and phoQ genes was<br>
created as previously described above (nucleotides<br>
376-1332 of the published sequence: Miller et al., Proc.<br>
Nat'l Acad. Sci. 86:5054, 1989). Briefly, a 2.2 kb<br>
fragment of S. typhimurium DNA internally deleted for 956<br>
base pairs (phoP/phoQ D956) was cloned into the suicide<br>
vector pCVD442 (Blofield et al., Mol. Micro, 5:1447,<br>
1991; Donnenberg et al., Infect. Immun. 59:4310, 19xx)<br>
and maintained in the permissive host E. coli SM10 1 pir.<br>
The resultant suicide vector was mobilized via conjugal<br>
transfer to the chromosome of Ty2, using plasmid-encoded<br>
ampicillin resistance and the leucine, threonine, and<br>
thiamine auxotrophies of E. coli SM10 1 pir for<br>
selection. Subsequently, selection against the<br>
vector-encoded sacB gene on sucrose agar plates was used<br>
as previously described (Donnenberg, supra) to identify<br>
S. typhi clones which had eliminated plasmid sequences<br>
via a second homologous recombination event,<br>
reconstituting either a wild type or deleted phoP/phoQ<br>
locus. Clones containing the phoP/phoQ D956 locus were<br>
easily recognized as white colonies on indicator plates<br>
containing the phosphatase substrate 5-bromo-4-choloro-3-<br>
indolyl phosphate, or BCIP (the nonspecific acid<br>
phosphatase of Salmonellae, encoded by the phoN gene, is<br>
regulated by the phoP/phoQ loci and is responsible for<br>
most of the blue coloration of S. typhi Ty2 on this<br>
indicator plate; Kier et al., J. Bacteriol. 138:155,<br>
1979) . Several white clones were characterized by<br>
Southern blotting of chromosomal DNA to demonstrate the<br>
anticipated chromosomal deletion in the phoP/phoQ genes<br>
and one was designated TySOO.<br>
Preparation of vaccine inocula<br>
Stock cultures of TySOO were stored at -70°C in<br>
20% glycerol and thawed for growth on Luria broth agar<br>
plates for indentification and confirmation of species<br>
(by agglutination with antisera) and phenotype on BCIP<br>
plates prior to growth of inocula. Multiple colonies<br>
identified as correct were subsequently inoculated into<br>
Luria broth and grown for 16 hours at 37°C on a rotary<br>
shaker. This culture was then suspended in 0.9% saline,<br>
standardized spectophotometrically, and diluted as needed<br>
in saline to attain the appropriate number of viable<br>
colony forming units (cfu) in 35 ml. A sample of each<br>
inoculum was diluted and plated for determination of the<br>
exact cfu delivered at each log dose.<br>
Human volunteers and study protocol<br>
Healthy adult men and women ages 18-49 with no<br>
prior history of typhoid fever vaccination were screened<br>
for health status with a complete medical history and<br>
physical exam. Laboratory screening tests included<br>
complete blood counts, chemistry panel, HIV, hepatitis<br>
and syphilis serologies, urinalysis, chest radiography<br>
and stool culture and ova and parasite exam. Volunteers<br>
were admitted to the General Clinical Research Center at<br>
Massachusetts General Hospital between April and<br>
September of 1995. Volunteers received a single oral<br>
inoculum on the day of admission (study day 0) and were<br>
followed in hospital for 14 subsequent days. Volunteers<br>
were examined daily by a physician and vital signs were<br>
taken every 6 hours. Blood was drawn for serologies at<br>
the screening visit and study days 0, 7, 14, 21, and 28.<br>
Blood was drawn routinely for peripheral blood<br>
raononuclear cell (PBMC) isolations on study days 0 and 7,<br>
and on days 4 and 10 for some individuals. Twelve<br>
volunteers enrolled in the study and received vaccine. A<br>
fourth volunteer in the highest dose group was excluded<br>
from further study when Salmonella brandenberg was<br>
isolated from two stool samples shortly after receiving<br>
the experimental vaccine. This organism had not been<br>
detected in a stool culture obtained at pre-study<br>
screening visit 10 days prior to admission, nor on the<br>
first 5 daily inpatient stool cultures. This volunteer<br>
but was formally excluded from the study thereafter, as<br>
evidence of active enteric infection was an exclusion<br>
criterion. This individual completed the inpatient<br>
observation period, felt completely well, but did not<br>
return for serology collections. This volunteer denied<br>
recent gastrointestinal illness but was presumably<br>
asymptomatically infected with S. brandenburg at some<br>
point in the past and intermittently excreting this<br>
organism at very low levels. Ty21a vaccinees were a group<br>
of microbiology laboratory workers. This group received<br>
4 Ty21a capsules (Swiss Serum and Vaccine Institute,<br>
Berne, Switzerland) on alternate days (1-6 x 109 cfu<br>
viable organisms and 5-50 x 109 non viable bacterial<br>
cells per capsule) as directed by the manufacturer.<br>
Bacteriology<br>
Bacteriology specimens were processed in the<br>
Clinical Microbiology Laboratory of Massachusetts General<br>
Hospital. A single daily stool culture was obtained from<br>
each volunteer. Stools were planted both as primary<br>
cultures and after overnight enrichment in selenite-F<br>
broth on Hektoen enteric agar and MacConkey agar plates.<br>
Quantitative stool cultures were performed on some<br>
volunteers by dissolution and serial dilution of 0.5-1.0<br>
g of stool in phosphate buffered saline and plating on<br>
selective media. On study days 2, 4, 6, 8, and 10, two<br>
sets of blood cultures were collected from each volunteer<br>
and inoculated into Bactec (Becton Dickinson, Sparks MD)<br>
aerobic and anaerobic blood culture bottles (5 ml<br>
inoculum per bottle). Blood cultures were held for 7<br>
days .<br>
Immunological assays<br>
ELISPOT analysis: Sterile ELISA plates with<br>
nitrocellulose membrane bottoms (Millititer HA,<br>
Millipore) were used in assays designed to enumerate the<br>
number of peripheral blood IgA angtibody-secreting cells<br>
(ASC) (Forrest et al., Lancet 1:81, 1988; Kantele et al. <br>
J. Infect. Dis. 153:1126, 1986) directed against S. typhi<br>
lipopolysaccharide (LPS). Membranes were incubated<br>
overnight with either 50 mM sodium carbonate buffer (pH<br>
10) alone, or buffer with LPS from S. typhi or Vibrio<br>
cholerae, the latter as a control LPS antigen (both at 20<br>
ug/ml, Sigma, St. Louis MO). Plates were then blocked<br>
with phosphate buffered saline (PBS) containing 2% bovine<br>
serum albumin (BSA) overnight and subsequently rinsed 3<br>
times with PBS. Peripheral blood mononuclear cells<br>
(PBMC) were isolated from 16 ml of citrated blood<br>
utilizing Vacutainer Cell Preparation Tubes<br>
(Becton-Dickinson, Franklin Lakes, NJ) as directed by the<br>
manufacuturer; 1-2 x 107 PBMC were consistently obtained<br>
per draw. PBMC were washed three times with PBS, counted,<br>
and suspended at known concentrations in RPMI tissue<br>
culture medium (Mediatech, Herndon, VA) containing 10%<br>
fetal calf serum and 1% penicillin/streptomycin solution<br>
(Sigma). Duplicate aliquots of 1 x 106 PBMC and serial<br>
5-fold dilutions of these aliquots were applied to<br>
membrane bottom wells treated with the various antigens<br>
and incubated for 4 hours in a tissue culture incubator.<br>
Non-adherent cells were removed by washing wells with PBS<br>
(3 times) and PBS containing 0.05% Tween 20 (Sigma) (3<br>
times). Goat anti-human IgA conjugated to peroxidase<br>
 (Kirkegaard and Perry Laboratories, Gaithersburg, MD) was<br>
applied at a dilution of 1:2500 in PBS- 2% BSA, incubated<br>
overnight at 4 °C, and plates were subsequently washed<br>
with PBS/0.05% Tween 20 (5 times) and the with PBS (three<br>
times). Membranes were developed for 15 minutes with<br>
3-amino-9-ethyl-carbazole substrate (AEC, Sigma) in 0.1 M<br>
acetate buffer at pH 5 as directed by the manufacturer.<br>
Dry membranes were then punched out of the ELISA wells<br>
using a cork borer and brown/red spots representing<br>
IgA-bearing cells specifically bound to antigen-coated<br>
membranes were counted at the most appropriate dilution<br>
under 20x magnification. Numbers reported represent the<br>
mean value for duplicate wells at an appropriate dilution<br>
for counting.<br>
Seroloqical analyses<br>
ELISAs were used to measure increases in serum IgG<br>
directed against either whole TySOO vaccine organisms or<br>
S. typhi O antigen (DIFCO, Detroit, MI). The same<br>
protocol was used to measure increases in serum IgA<br>
directed against S. typhi LPS (coating solution 5/jg/ml<br>
LPS) and serum. IgG directed against purified Vi antigen<br>
(gift of Dr. Tim Barnett, CDC, Atlanta, GA; coating<br>
solution 2 ug/ml). In each of these assays, a standard<br>
serum dilution of 1:80 was used, and each diluted serum<br>
sample was applied in parallel to wells containing either<br>
buffer alone or buffer containing antigen. Affinity<br>
purified, alkaline phosphatase-labelled goat antihuman<br>
IgG or antihuman IgA secondary antibodies (Kirkegaard and<br>
Perry Laboratories) were used. Specific optical density<br>
(OD) was determined for each serum sample by subtracting<br>
the OD value of the buffer well from the antigen well.<br>
Increases in specific OD from preimraune to peak values<br>
were obtained by subtracting the preimmune value from<br>
values on subsequent days. Significant optical density<br>
increases were those which exceed a statistically<br>
derived significant OD threshold (0.22 OD units for the<br>
whole cell IgG assay, 0.13 OD units for the O antigen IgG<br>
assay, and 0.07 OD units for the LPS IgA assay). These<br>
threshold values were determined by testing pre- and<br>
post-immune paired sera from 15 volunteers receiving a<br>
live attenuated cholera vaccine and are egual to the mean<br>
net increase in OD plus 3 standard deviations of these<br>
paired sera. To provide another measurement of<br>
serological response, endpoint dilution studies were<br>
performed using S. typhi LPS. In these studies, 5 /zg/ml<br>
S. typhi LPS was used for coating antigen wells, and sera<br>
were serially diluted 2-fold in PBS/2% BSA in a<br>
microtiter plate beginning at 1:40. The endpoint titer<br>
was defined as the highest serum dilution at which<br>
specific optical density exceeded 0.15 OD units, and more<br>
than a four fold increase in titer from preimmune to peak<br>
was deemed significant. The Widal tube test using H<br>
antigen was performed as directed by the manufacturer<br>
(DIFCO). Fishers exact T-test was used to generate p<br>
values for comparison between TySOO and Ty21a vaccinees.<br>
In vitro characterization of TySOO<br>
Chromosomal DNA of TySOO was demonstrated by<br>
restriction endonuclease digestion and Southern blotting<br>
to contain the anticipated chromosomal deletion in the<br>
phoP and phoQ genes. TySOO had a growth rate in Luria<br>
broth identical to that of the Ty2 parent strain as<br>
measured by serial determinations of OD600 values for<br>
cultures in logarithmic growth phase. TySOO was more<br>
sensitive to rabbit defensin NP-1 than Ty2, a phenotype<br>
of other PhoP/PhoQ null Salmonella strains (Miller et<br>
al., Infect. Immunol. 68: , 1990).<br>
Clinical response and side effects<br>
Following vaccination 9/11 volunteers remained<br>
completely well and had no temperature elevations or<br>
other adverse effects. Two volunteers receiving the<br>
highest dose of 4 x 1010 had transient side effects.<br>
Volunteer number 9 had a single temperature elevation of<br>
37.9°C approximately 20 hours after receiving the<br>
vaccine, associated with one loose stool and myalgias.<br>
He felt completely well within another 12 hours and had<br>
no other adverse effects for the duration of the study.<br>
Volunteer number 10 had a single temperature elevation to<br>
38°C also at 20 hours after vaccination, and also had an<br>
acute gastroenteritis-like picture with 10 loose to<br>
liquid stoods (grade 3-4) during the time period of 12-36<br>
hours following vaccination. Loose stools were<br>
associated with cramping; there was no vomiting or bloody<br>
stools. This syndrome resolved without therapy within 48<br>
hours after vaccination and the volunteer felt well for<br>
the remainder of the study. The excluded volunteer with<br>
nontyphoidal felt completely well throughout the hospital<br>
stay.<br>
Bacteriology<br>
No volunteer had blood cultures positive for the<br>
vaccine organism at any time in the study (10<br>
sets/volunteer). Limiting dilution studies using strain<br>
TySOO and sterile human blood demonstrated that the blood<br>
culture system utilized was very sensitive. Blood<br>
culture bottles consistently turned positive within 48<br>
hours after introduction of experimental inocula<br>
calculated to deliver 1 organism as measured by parallel<br>
plating on Luria broth agar plates.<br>
The vaccine organism was detected in stool<br>
cultures in 10/11 volunteers. Volunteer number 2 (1 of 2<br>
receiving the lowest dose) had no positive stool<br>
cultures. The duration of positive stool cultures ranged<br>
from 1-3 days after vaccination in 9 volunteers.<br>
Volunteer number 3 who received a dose of 6 x 108 cfu had<br>
positive cultures on days 1, 2, and 3 and then<br>
intermittently positive stool cultures only after<br><br>
selenite enrichment until day 15 of the study. This<br>
volunteer was completely asymptomatic during this time<br>
and received a 7 day course of oral ciprofloxacin to<br>
eradicate the vaccine organism, which was documented as<br>
successful by multiple negative followup stool cultures<br>
over the ensuing month. Quantitative stool cultures<br>
performed only on volunteers receiving the highest 2<br>
doses showed that vaccine organism burdens ranged from<br>
102-105 cfu/gram of stool (8 x 109 cfu cohort) to 102-107<br>
cfu/gram (4 x 1010 cfu cohort). The largest detectable<br>
numbers of vaccine organisms occurred on days 1 and 2<br>
after vaccination and decreased rapidly thereafter.<br>
Vaccine organisms recovered from volunteers retained<br>
their PhoP null phenotype on BCIP plates (see above).<br>
Immunological analyses<br>
All volunteers, except number 1, who received the<br>
lowest dose, had signficant numbers of IgA-bearing cells<br>
directed against S. typhi LPS detected in their<br>
peripheral blood on day 7 after vaccination (Table 1A).<br>
Control studies performed on day 0 and control wells<br>
containing buffer alone or V. cholerae LPS on days 0 and<br>
7 were negative in all cases (
highest inoculum of cells). Additional ELISPOT studies<br>
performed on days 4 and 10 for the 2 lower dose groups<br>
showed that IgA secreting cell numbers were maximal on<br>
day 7 (data not shown), as previously demonstrated for<br>
Ty21a (Kantele et al., J. Infect. Dis. 153:1126, 1986).<br>
The excluded volunteer excluded with complicating<br>
non-typhoidal salmonellosis had 9 IgA secreting cells<br>
detected on day 0 and 1,325 on day 7 after vaccination.<br>
Several serological studies were performed to<br>
examine humoral immune responses to different<br>
preparations of S. typhi antigens. ELISAs using standard<br>
1:80 dilutions of serum samples to measure increases in<br>
serum IgG against O antigen or whole vaccine organisms<br>
were positive in 9/11 volunteers over the entire range of<br>
doses in each assay. ELISA results for a control group<br>
that received 4 doses of Ty21a were positive in 3/8 or<br>
5/8 vaccinees, respectively in these assays. (Two<br>
control vaccinees, numbers 13 and 14, had high background<br>
OD values in these assays - one had been vaccinated with<br>
a parenteral typhoid fever vaccine more than 30 year<br>
ago). Peak IgG responses in these assays occurred on<br>
study days 14 or 21 days for both TySOO and Ty21<br>
vaccinees.<br>
Five of 11 volunteers had evidence of<br>
seroconversion by four-fold or greater increases in<br>
endpoint titers utilizing S. typhi LPS as an antigen, and<br>
6/11 seroconverted by four- fold or greater increases in<br>
the Widal tube agglutination test using H antigen.<br>
Considering seroconversion to either of these antigens<br>
evidence of a systemic immune response, 7/11 volunteers<br>
over the wide dose range given seroconverted after single<br>
dose vaccinations. In comparison, only 3/8 volunteers<br>
receiving 4 oral doses of Ty21a seroconverted by IgG<br>
endpoint titer vs. LPS (vs. Ty800, p = 0.65), and none<br>
had positive Widal tube agglutination tests (vs. TySOO,<br>
p=0.018).<br>
Serum IgA directed against S. typhi LPS has been<br>
demonstrated to increase significantly in some<br>
individuals after vaccination with live attenuated S.<br>
typhi vaccines (Forrest et al., J. Infect. Dis. 163:336,<br>
1991; Tacket et al., Vaccine :443, 1991). As<br>
IgA-ELISPOT studies were not performed for the Ty21a<br>
group, we measured serum IgA by ELISA to provide another<br>
measure of comparison between these 2 groups. Eight of<br>
11 experimental vaccinees and 4/8 Ty21a vaccines had<br>
significant increases in OD values reflecting serum IgA<br>
directed against S. typhi LPS (Table.1A; p = 0.38).<br>
Three of 11 volunteers (numbers 6, 10, and 11)<br>
also had measurable increases in serum IgG directed<br>
against purified Vi antigen. In this assay, Ty21a<br>
vaccinees were used as a control population, as Ty21 does<br>
not express the Vi antigen. Volunteers 6, 8, and 10 had<br>
peak specific OD increases of 0.56, 0.26 and 0.34 OD<br>
units respectively at a 1:80 dilution of sera, while the<br>
mean net increase +/- 3 SD for Ty21a vaccinees from day 0<br>
to day 21 was 0.02 +/- 0.04 OD units.<br>
The studies described above demonstrates that<br>
Ty800, a genetically defined phoP/phoQ-deleted S. typhi<br>
Ty2, is markedly attenuated in human volunteers and is a<br>
very promising candidate vaccine and vector strain.<br>
Importantly, no volunteer developed serious side effects,<br>
or bacteremias even at very high doses. A dose level<br>
which induced some toxicity was reached, but only one<br>
volunteer had a self- limited gastroenteritis-like<br>
adverse event, and lower doses were clearly still<br>
immunogenic. One volunteer appeared to have become<br>
colonized with the vaccine organism; others shed the<br>
organism only transiently after vaccination.<br>
Unfortunately, quantitative stool cultures were not<br>
performed on the volunteer with prolonged shedding, and<br>
because this volunteer received antibiotics to eliminate<br>
the recombinant organism, the true duration of<br>
colonization could not be assessed. This volunteer<br>
received only 108 cfu and volunteers receiving much<br>
larger doses did not have prolonged shedding, suggesting<br>
that colonization patterns may be more dependent upon<br>
host factors than vaccine-specific factors. It has been<br>
previously suggested that the presence or absence of<br>
secreted ABH carbohydrate determinants may modulate<br>
adherence of S. typhi to epithelial cells.<br>
After vaccination with Ty21a, .IgA production by<br>
PBMC in vitro is a very sensitive measure of<br>
inununological response which correlates more highly with<br>
intestinal IgA production than serum measurement of IgG<br>
or IgA (Forrest et al., Infect. Irmunol 60:2023, 1992).<br>
Ten of 11 (91%) volunteers receiving any single dose of<br>
TySOO had evidence of intestinal immune responses as<br>
measured by significant numbers S. typhi LPS-specific<br>
IgA-bearing peripheral blood mononuclear cells on day 7<br>
after vaccination. In previous studies, 3 doses of Ty21a<br>
have resulted in detection of IgA- bearing cells by<br>
ELISPOT in 56% (Tacket et al., J. Infect. Dis. 163:901,<br>
1991) to 80% (Kantele et al., J. Infect. Dis. 153:1126,<br>
1986) of individuals. S. typhi-specific IgA production<br>
by PBMC in vitro was detected in 93%-100% of volunteers<br>
after vaccination with Ty21a (Forrest et al., Lancet<br>
1:81, 1988; Forrest, Infect. Inununol. 60:2023, 1992).<br>
CVD908, a single dose, aromatic amino auxotrophic vaccine<br>
strain of S. typhi, induced S. typhi-specific antibody<br>
secreting cells (ASC) or in vitro IgA production by PBMC<br>
in 100% of volunteers (Gonzales et al., J. Infect. Dis.<br>
169:927, 1994; Tacket et al, Vaccine :443, 1992) at doses<br>
that resulted in bacteremias in some individuals. In our<br>
study and those of other detection of ASC cells appears<br>
to be an extremely sensitive measure of immunological<br>
response to oral attenuated S. typhi vaccines. TySOO<br>
induced more than 1000 ASC/106 PBMC in 6/11 volunteers<br>
and more than 2000 ASC/106 PBMC in those receiving the<br>
highest dose, larger numbers than previously reported in<br>
studies of attenuated live typhoid vaccines in which ASC<br>
numbers were specifically reported (Gonzales et al.,<br>
supra; Kantele et al., J. Infect. Dis. 153:1126, 1986).<br>
By the studies performed here, a single oral dose of<br>
TySOO appeared to induce comparable, if not superior<br>
increases in markers of intestinal immune responses to<br>
multiple oral doses of Ty21a. These results suggest that<br>
TySOO may be a particularly promising' vector strain for<br>
antigens relevant to the gastrointestinal tract, or where<br>
secretory IgA at other sites is important. Direct<br>
measurement of intestinal IgA and IgA at other mucosal<br>
sites would be useful in future studies of TySOO.<br>
The relative importance of intestinal and systemic<br>
serological responses in typhoid fever is debated (Levine<br>
et al., Rev, Inf. Dis. 11S3:S552, 1989). Systemic<br>
seroconversion has correlated with protection in endemic<br>
areas (Levine.et al., supra), and parenteral vaccines<br>
which do not generate intestinal immunity have been shown<br>
to have some protective efficacy in both endemic areas<br>
(Levine et al., supra) and presumed naive vaccinees<br>
Schwartz et al., Arch. Intern. Med. 150:349, 1994).<br>
Additionally, generation of systemic humoral immune<br>
responses is relevant to other antigens potentially<br>
engineered into a multivalent vector strain (e.g.<br>
tetanus or pertussis toxin fragments). Systemic humoral<br>
immune responses measured by a variety of techniques<br>
after single doses of TySOO were equal, and in some<br>
studies performed here, superior to those generated by 4<br>
doses of Ty21a. 6/11 volunteers seroconverted by the<br>
Widal test, compared with 0/8 Ty21a vaccinees. Lack of<br>
seroconversion in the Widal test has been reported in<br>
previous study of Ty21 (Tacket et al., J. Infect. Dis.<br>
163:901, 1991) and approximately 50% serconversion by<br>
Widal was seen in a recent study of volunteers receiving<br>
two oral doses of CVD908 expressing a malarial<br>
circumsporozoite antigen (Gonzales et al., supra).<br>
We found that the most sensitive assays for<br>
detecting increases in serum IgG were ELISAs against<br>
whole vaccine organisms or S. typhi O antigen. We used<br>
this technique because it had successfully documented<br>
serological responses to attenuated S. typhi strains at<br>
the Center for Vaccine Development (Gonzales et al.,<br>
supra; Tacket et al., Infect. Immun. 60:536, 1992; Tacket<br>
et al., Vaccine :443, 1992). Using these ELISAs, we<br>
found that even controls and volunteers with high<br>
baseline OD values in an endpoint dilution study<br>
(volunteers 6 and 8, and controls 13 and 14) mounted<br>
increases in specific OD units which exceeded by 3<br>
standard deviations the mean net increase of the control<br>
cholera-vaccinated population. For example, volunteer 8,<br>
who had large increases in IgA-secreting cells.but<br>
endpoint titers falling short of formally seroconverting<br>
had relatively large increases in optical densities<br>
ELISAs (0.6-0.8 OD units). This individual did not meet<br>
criteria for seroconversion by the less sensitive but<br>
presumably more specific endpoint dilution titer tests<br>
against S. typhi LPS (Table 1A); nevertheless it is<br>
likely that this volunteer had a true serological<br>
response to S. typhi. Similarly, controls 13 and 14 had<br>
high baseline titers in the LPS endpoint dilution study,<br>
and no increase after vaccination; these 2 control<br>
vaccinees did exceed the statistically significant<br>
threshold in the ELISA studies. We presume that these 4<br>
individuals with high baseline titers or OD values may<br>
have unknowningly had prior typhoid fever vaccines or<br>
exposure to cross-reacting bacterial antigens. Most<br>
likely, nontyphoidal salmonellosis, which has been<br>
demonstrated to impair response to Ty21a (Forrest et al.,<br>
J. Infect. Dis. 166:210, 1992) may be responsible for the<br>
high baseline values in serological tests and relatively<br>
poor serological responses in these individuals.<br>
In both groups there were individuals with very<br>
low baseline serology results in all assays (volunteers 1<br>
and 7 and controls 15 and 19) who were apparent<br>
serological nonresponders, presumably as a result of<br>
other unknown host factors. Volunteer 7 had no evidence<br>
of serological response in any assay, but still generated<br>
significant numbers of specific IgA-secreting cells,<br>
confirming that this is a very sensitive measure of<br>
immunological response to enteric vaccines. The results<br>
of the excluded volunteer with coexisting S. brandenburg<br>
infection are more difficult to interpret. Presumably<br>
the low level positive ELISPOT study on day 0 represent<br>
cells generated by the nontyphoidal infection which cross<br>
reacted with 5. typhi LPS. It is not possible to say<br>
what proportion of day 7 ELISPOT cells are attributable<br>
to the experimental vaccine.<br>
An unexpected finding in our study was the<br>
apparent systemic immune response to orally administered<br>
Vi antigen. There was no seroconversion to Vi antigen in<br>
a previous trial of 9 volunteers who received 3 oral<br>
doses of 5 x 109 cfu of Ty21a engineered to express this<br>
antigen, despite expected rates of seroconversion to<br>
other antigens (Tacket et al., J. Infect. Dis. 163:901,<br>
1991). Serological studies of Vi antigen responses have<br>
not been consistently reported in studies of other<br>
attenuated S. typhi vaccines, but occasional unquantified<br>
responses have been noted (Tacket et al., Infect. Immun.<br>
60:536, 1992), suggesting that the more immunogenic live<br>
oral vaccines may also engender responses to Vi. Two of<br>
the volunteers (numbers 6 and 10) in our study who<br>
responded to this antigen were not those with the most<br>
vigorous serological responses in other assays,<br>
suggesting that host factors may direct responses to<br>
individual antigens. In summary, we have demonstrated<br>
that TySOO is a promising new single-dose oral typhoid<br>
vaccine candidate which appears safe and is highly<br>
immunogenic. Although this study tested small numbers of<br>
volunteers at each dose, several important features of<br>
this vaccine strain are apparent. Large doses were<br>
well-tolerated and lower doses were immunogenic. There<br>
were no bacteremias in this study, making this vaccine<br>
strain more widely applicable to large populations than<br>
less attenuated strains. Very large numbers of vaccine<br>
specific IgA-secreting cells were generated, suggesting<br>
that this strain may be particularly useful in generating<br>
mucosal immune responses. phoP- activated genes, which<br>
; will be transcriptionally repressed in TySOO, have been<br>
demonstrated in vitro in S. typhimurium to inhibit<br>
antigen processing within murine macrophages (Wick et<br>
al., Mol. Micro. 16:465, 1995). We conclude that TySOO<br>
may be a particularly useful vaccine strain for both<br>
development of a single oral dose typhoid fever vaccine<br>
and for engineering an S. typhi vector capable of<br>
delivering heterologous antigens to the gastrointestinal<br>
immune system.<br>
Constitutive Expression of the PhoP Requlon Attenuates<br>
Salmonella Virulence and Survival within Macrophaqes<br>
The phoP constitutive allele (PhoP0) , pho-24,<br>
results in derepression of pag loci. Using diethyl<br>
sulfate mutagenesis of S. typhimurium LT-2, Ames and coworkers<br>
isolated strain TA2367 pho-24 (all strains,<br>
materials, and methods referred to in this section are<br>
described below), which contained a phoP locus mutation<br>
that resulted in constitutive production of acid<br>
phosphatase in rich media, Kier et al., 1979, J.<br>
Bacteriol. 138:155, hereby incorporated by reference.<br>
This phoP-regulated acid phosphatase is encoded by the<br>
phoN gene, a pag locus, Kier et al., 1979, supra, Miller<br>
et al., 1989, supra. To analyze whether the pho-24<br>
allele increased the expression of other pag loci the<br>
effect of the pho-24 allele on the expression of other<br>
pag loci recently identified as transcriptional (e.g.,<br>
pagA and pagB) and translational (e.g., pagC) fusion<br>
proteins that required phoP and phoQ for expression,<br>
Miller et al., 1989, supra, was determined. pag gene<br>
fusion strains, isogenic except for the pho-24 allele,<br>
were construpted and assayed for fusio^n protein activity.<br>
PhoPc derivatives of the pagA: :Mu dJ and pagJ3::Mu dJ<br>
strains produced 480 and 980 U, respectively, of /?-<br>
galactosidase in rich medium, an increase of 9- to 10-<br>
fold over values for the fusion strains with a wild-type<br>
phoP locus, see Table 1.<br>
(Table Removed)<br>
The pagC::TnphoA gene fusion produced 350 U of<br>
AP, an increase of three- to fourfold over that produced<br>
in strain CS119, which is isogenic except for the pho-24<br>
mutation, Miller et al., 1989, supra. These results<br>
compare with a ninefold increase in the acid phosphatase<br>
activity in strain CS022 on introduction of the pho-24<br>
allele. Therefore, these available assays for pag gene<br>
expression document that the pho-24 mutation causes<br>
constitutive expression of pag loci other than phoN.<br>
Identifications of protein species that are repressed as<br>
well as activated in the PhoP° mutant strain<br>
Whole-cell proteins of strain CS022 were analyzed<br>
to estimate the number of protein species that could be<br>
potentially regulated by the PhoP regulon. Remarkably,<br>
analysis by one-dimensional polyacrylamide gel<br>
electrophoresis of the proteins produced by strains with<br>
the PhoPc phenotype indicated that some protein species<br>
were decreased in expression when many presumptive pag<br>
gene products were fully induced by the pho-24 mutation.<br>
The proteins decreased in the PhoPc strain might<br>
represent products of genes that are repressed by the<br>
PhoP regulator. Genes encoding proteins decreased by the<br>
pho-24 allele are designated prg loci, for phoP-repressed<br>
genes. Comparison of wild-type, PhoP", and PhoPc mutant<br>
strain proteins shows that growth in LB medium at 37°C<br>
represents repressing conditions for pag gene products<br>
and derepressing conditions for prg gene products.<br>
To estimate the total number of potentially PhoPregulated<br>
gene products, the total cell proteins of wildtype<br>
and PhoPc mutant strains grown in LB were analyzed<br>
by two-dimensional gel electrophoresis. At least 40<br>
species underwent major fluctuation in expression in<br>
response to the pho-24 mutation.<br>
(Table Removed)<br><br>
Remarkably, strains with the single pho-24<br>
mutation were markedly attenuated for virulence in mice<br>
(Table 2). The number of PhoPc organisms (2 x 105) that<br>
killed 50% of BALB/c mice challenged (LD50) by the<br>
intraperitoneal (i.p.) route was near that (6 x 105) of<br>
PhoP~ bacteria, Miller et al., 1989, supra. The PhoPc<br>
strains had growth comparable to wild-type organisms in<br>
rich and minimal media. The PhoPc mutants were also<br>
tested for alterations in lipopolysaccharide, which could<br>
explain the virulence defect observed. Strain CS022 had<br>
normal sensitivity to phage P22, normal group B<br>
reactivity to antibody to O antigen, and a<br>
lipopolysaccharide profile identical to that of the<br>
parent strain, as determined by polyacrylamide gel<br>
electrophoresis and staining.<br>
Since the TA2367 pho-24 strain was constructed by<br>
chemical mutagenesis and could have another linked<br>
mutation responsible for its virulence defect revertants<br>
of the PhoPc were isolated to determine whether the pho-<br>
24 allele was responsible for the attenuation of<br>
virulence observed. Phenotype PhoPc revertants,<br>
identified by the normal levels of acid phosphatase in<br>
rich medium, were isolated among the bacteria recovered<br>
from the livers of mice infected with strain CS022. Six<br>
separate phenotypic revertants, designated CS122 to<br>
CS128, were found to be fully virulent (LD50 of less than<br>
20 organisms for BALB/c mice). The locus responsible for<br>
the reversion phenotype was mapped in all six revertants<br>
tested for virulence by bacteriophage P22 cotransduction<br>
and had linkage characteristics consistent with the phoP<br>
locus (greater than 90% linkage to purB) . These data<br>
indicate that these reversion mutations are not<br>
extragenic suppressors but are intragenic suppressors or<br>
true revertants of the pho-24 mutation. Thus, the<br>
virulence defect of PhoPc mutants is probably the result<br>
of a single revertible mutation in the phoP locus and not<br>
the result of a second unrelated mutation acquired during<br>
mutagenesis.<br>
Reversion frequency of the PhoPc phenotype<br>
The reversion frequency of the PhoP0 mutation in<br>
vivo in mice was investigated to assess whether reversion<br>
could reduce the LD50 of this strain. The presence of the<br>
revertants of strain CS022 was tested for by<br>
administering 105, 104, and 102 challenge organisms to<br>
each of eight animals by i.p. injection. On day 7, three<br>
animals died that received 106 PhoPc organisms. On that<br>
day, the livers and spleens of all animals were harvested<br>
and homogenized in saline. After appropriate dilution,<br>
t<br>
10% of the tissue was plated on LB plates containing the<br>
chromogenic phosphatase substrate XP. Revertants were<br>
identified by their lighter blue colonies compared with<br>
PhoPc bacteria and were confirmed by quantitative acid<br>
phosphatase assays. An estimated 107, 105, and 103<br>
organisms per organ were recovered from animals at each<br>
of the three respective challenge doses. Revertants were<br>
identified only at the highest dose and comprised 0.5 to<br>
1%, or. 105 organisms per organ, at the time of death. It<br>
is likely that revertants are able to compete more<br>
effectively for growth in these macrophage-containing<br>
organs, since strain CS022 is deficient in survival<br>
within macrophages (see below). However, revertants were<br>
not identified if fewer than 105 organisms were<br>
administered in the challenge dose, suggesting that the<br>
reversion frequency must be approximately 10~5. The<br>
reversion rate of the PhoPc phenotype for CS022 bacteria<br>
grown in LB is in fact 6xlO~4 when scored by the same<br>
colony phenotypes. The percentage of revertants<br>
recovered from animals near death suggests that pressure<br>
is applied in vivo that selects for revertants of the<br>
PhoPc phenotype and implies that the virulence defect<br>
observed could be much greater quantitatively for a<br>
strain with a nonrevertible PhoP° mutation.<br>
The PhoPc strain is deficient in survival within<br>
macrophages Because of the importance of<br>
survival within macrophages to Salmonella virulence<br>
Fields et al., 1986, Proc. Natl. Acad. Sci. USA 83:5189,<br>
hereby incorporated by reference, PhoPc bacteria were<br>
tested for this property. Strain CS022 was defective in<br>
.the ability to grow and persist in macrophages as<br>
compared with wild-type organisms (Fig. 1). In Fig. 1<br>
the survival of strain CS022 (PhoPc) (triangles) in<br>
cultured macrophages is compared with that of wild-type<br>
S. typhimurium ATCC 10428 (circles). The experiment<br>
shown is a representative one. The difference between<br><br>
the two strains at 4 and 24 hours is significant (P 
0.05). PhoP~ bacteria seemed to have a macrophage<br>
survival defect qualitatively similar to that of PhoPc<br>
bacteria but survived consistently better by two- to<br>
threefold in side-by-side experiments. The increased<br>
recovery of organisms that reverted to PhoPc phenotype in<br>
mouse organs rich in macrophage content is consistent<br>
with the reduced macrophage survival of PhoP° mutants in<br>
vitro.<br>
Use of the PhoPc strain as a live vaccine<br>
It has been previously reported that PhoP~ strains<br>
are useful as live vaccines in protecting against mouse<br>
typhoid, Miller et al., 1989, supra. The immunogenicity<br>
of PhoP° when used as live attenuated vaccines in mice<br>
was compared with the of PhoP~. This was done by<br>
simultaneous determination of survival, after graded<br>
challenge doses with the wild-type strain ATCC 10.428, in<br>
mice previously immunized with graded doses of the two<br>
live vaccine strains. CS015 phoP::TnlOd-Cam and CS022<br>
pho-24, as well as a saline control. The results<br>
obtained (Table 2) suggest the following conclusions: (i)<br>
small i.p. doses of the PhoP0 strain (e.g., 15 organisms)<br>
effectively protect mice from challenge doses as large as<br>
5xl05 bacteria (a challenge dose that represents greater<br>
than 104 i.p. LD50s) , (ii) large doses of PhoPc organisms<br>
given orally completely protect mice from an oral<br>
challenge consisting of 5xl07 wild-type bacteria (over<br>
200 oral wild-type LD50s) and (iii) by comparison, a large<br>
dose of PhoP" organisms (5xl05) does not provide similar<br>
protection. The reversion of the PhoP0 mutation in vivo<br>
somewhat complicates the analysis of the use of these<br>
strains as vaccines, since revertants of the CS022 strain<br>
(i.e., wild-type cells) could increase immunogenicity).<br>
However, we were unable to identify revertants by<br>
examining 10% of the available spleen and liver tissue<br>
from those mice that received 104 or fewer organisms.<br>
Strains. Materials and Methods<br>
The strains,, materials, and methods used in the<br>
PhoP regulon work described above are as follows.<br>
American Type Culture Collection (ATCC) strain<br>
14028, a smooth virulent strain of S. typhimurium, was<br>
the parent strain for all virulence studies. Strain<br>
TT13208 was a gift from Nang Zhu and John Roth. Strain<br>
TA2367 was a generous gift of Gigi Stortz and Bruce Ames,<br>
Kier et al., 1979, supra. Bacteriophage P22HT int was<br>
used in transductional crosses to construct strains<br>
isogenic except for phoP locus mutations, Davis et al.,<br>
1980, Advanced Bacterial Genetics, p. 78, 87. Cold<br>
Spring Harbor Laboratory, Cold Spring Harbor, NY, hereby<br>
incorporated by reference. Luria broth was used as rich<br>
medium, and minimal medium was M9, Davis et al., 1980,<br>
supra. The chromogenic phosphatase substrate 5-bromo-4-<br>
chloro-3indolyl phosphate (XP) was used to qualitatively<br>
access acid and AP production in solid media.<br>
Derivatives of S. typhimurium ATCC 10428 with the<br>
pho-24 mutation were constructed by use of strain TA2367<br>
as a donor of the purB gene in a P22 transductional cross<br>
with strain CS003 AphoP ApurB, Miller et al., 1989,<br>
supra. Colonies were then selected for the ability to<br>
grow on minimal medium. A transductant designated CS022<br>
(phenotype PhoPc) that synthesized 1,750 U of acid<br>
phosphatase in rich medium (a ninefold increase over the<br>
wild-type level in rich medium) was used in further<br>
studies.<br>
Derivatives of strains CS022 and CS023 pho-24<br>
phoN2 zxx::6251TnlOd-Cam, and acid phosphatase-negative<br>
derivative of CS02.2, containing pag gene fusions were<br>
constructed by bacteriophage P22 transductional crosses,<br>
using selection of TnphoA- or Mu dJ-encoded kanamycin<br>
resistance. Strains were checked for'the intact pag gene<br>
fusion by demonstration of appropriate loss of fusion<br>
protein activity on introduction of a phoPlOS::Tnl0d or<br>
phoP102::TnlOd-Cam allele.<br>
Assays of acid phosphatase, AP, and (3-<br>
galactosidase were performed as previously described,<br>
Miller et al., 1989, supra and are reported in units as<br>
defined in Miller, 1972, Experiments in molecular<br>
genetics, p. 352-355, Cold Spring Harbor Laboratory, Cold<br>
Spring Harbor, NY, hereby incorporated by reference.<br>
In the mouse virulence and vaccination studies<br>
bacteria grown overnight in Luria broth were washed and<br>
diluted in normal saline. The wild-type parent strain of<br>
CS022 (ATCC 10428) was used for all live vaccine<br>
challenge studies. This strain has a 50% lethal dose<br>
(LD50) for naive adult BALB/c mice of less than 20<br>
organisms when administered by intraperitoneal (i.p.)<br>
injection and 5xl04 when administered orally in NaHC03.<br>
Mice were purchased from Charles River Breeding<br>
Laboratories, Inc. (Wilmington, Mass.) and were 5 to 6<br>
weeks of age at initial challenge. All i.p. inoculations<br>
were performed as previously described, Miller et al.,<br>
1989, supra. Oral challenge experiments were performed<br>
with bacteria grown in LB broth and concentrated by<br>
centrifugation. The bacteria were resuspended in 0.1 M<br>
NaHCO3 to neutralize stomach acid, and administered as a<br>
0.5-ml bolus to animals under ether anesthesia. Colony<br>
counts were performed to accurately access the number of<br>
organisms administered. All challenge experiments were<br>
performed 1 month after i.p. inoculation and 6 weeks<br>
after oral challenge. Challenge inocula were<br>
administered by the same route as vaccinations. The care<br>
of all animals was under institutional guidelines as set<br>
by the animal are committees at the Massachusetts General<br>
Hospital and Harvard Medical School.<br>
Protein electrophoresis was performed as follows.<br>
One-dimensional protein gel electrophoresis was performed<br>
by the method of Laemrali, 1970, Nature 227 ;680, hereby<br>
incorporated by reference, on whole-cell protein extracts<br>
of stationary-phase cells grown overnight in Luria broth.<br>
The gels were fixed and stained with Coomassie brilliant<br>
blue R250 in 10% acetic acid-10% methanol. Twodimensional<br>
protein gel electrophoresis was performed by<br>
method of O'Farrell, 1975, J. Biol. Chem. .250:4007,<br>
hereby incorporated by reference, on the same whole-cell<br>
extracts. Isoelectric focusing using 1.5% pH 3.5 to 10<br>
ampholines (LKB Instruments, Baltimore, Md.) was carried<br>
out for 9,600 V h (700 V for 13 h 45 min). The final<br>
tube gel pH gradient extended from pH 4.1 to pH 8.1 as<br>
measured by a surface pH electrode (BioRad Laboratories,<br>
Richmond, Calif.) and colored acetylated cytochrome pi<br>
markers (Calbiochem-Behring, La Jolla, Calif.) run in an<br>
adjacent tube. The slab gels were silver stained, Merril<br>
et al., 1984, Methods Enzymol. 104:441. hereby<br>
incorporated by reference.<br>
In the macrophage survival assays experiments were<br>
performed as previously described, Miller et al., 1989,<br>
supra, by the method of Buchmeier et al., 1989, Infect.<br>
Immun. 5_7:1, hereby incorporated by reference, as<br>
modified from the method of Lissner et al, 1983, J.<br>
Immunol. 131:3006, hereby incorporated by reference.<br>
Stationary-phase cells were opsonized for 30 min in<br>
normal mouse serum before exposure to the cultured bone<br>
marrow-derived macrophages harvested from BALB/c mice.<br>
One hour after infection, gentamicin sulfate (8 /ng/ml)<br>
was added to kill extracellular bacteria. All time<br>
points were done in triplicate and repeated on three<br>
separate occasions.<br>
PhoPc Mutant Strains Are More Effective as Live Vaccines<br>
PhoPc mutant S. typhimurium are very effective<br>
when used as a live vaccine against mouse typhoid fever<br>
and are superior to PhoP~ bacteria. As few a 15 PhoPc<br>
bacteria protect mice against 105 LD50 (lethal doses 50%)<br>
of wild type organisms by the intraperitoneal route<br>
(Table 3). This suggests that pag gene products are<br>
important antigens for protective immunity against mouse<br>
typhoid. Preliminary results have documented that<br>
antigens recognized by serum of chronic typhoid carriers<br>
recognizes some phoP-regulated gene products of S. typhi.<br>
If protective antigens are only expressed within the<br>
host, then dead vaccines only grown in rich media may not<br>
induce an immune response against these proteins.<br>
The use of different S. typhimurium dead vaccine<br>
preparations containing different mutations in the phoP<br>
regulon was evaluated. As can be seen in Table 3 no dead<br>
cell preparations (even those containing mixtures of<br>
PhoP~ and PhoPc bacteria) are as effective vaccines as are<br>
live bacteria. This suggests that there are other<br>
properties of live vaccines that increase immunogenicity<br>
or that important non-PhoP-regulated antigens are not in<br>
these preparations. The only protection observed in any<br>
animals studied was at the lowest challenge dose for<br>
those immunized with PhoPc bacteria. This further<br>
suggests that phoP activated genes are important<br>
protective antigens.<br>
(Table Removed)<br>
BALB/c mice were immunized twice, 7 days apart, with 5x10<br>
formalin-killed bacteria. Three weeks after the second vaccination,<br>
mice were challenged with wild-type organisms at the two doses<br>
indicated. The numbers in parentheses indicate no survivors after<br>
challenge and mean number of days until death (*) Ratio of survivors<br>
to number challenged.<br>
phoPc indicates the constitutive unregulated expression of<br>
phoP-activated genes and lack of expression of phoP repressed genes.<br>
phoP' indicates a lack of expression of p^oP-activated genes<br>
and expression of phoP repressed genes.<br>
aroA PhoP Requlon Double Mutant Strains<br>
Recent efforts by Stocker, Levine, and colleagues<br>
have focused on the use of strains with auxotrophic<br>
mutations in aromatic amino acid and purine pathways as<br>
live vaccines, Hoseith et al., 1981, Nature 291:238,<br>
hereby incorporated by reference, Stocker, 1988, Vaccine<br>
6.: 141, hereby incorporated by reference, and Levine et<br>
al., 1987, J. Clin. Invest. 7__9:888, hereby incorporated<br>
by reference. Purine mutations were found to be too<br>
attenuating for immunogenicity, likely because purines<br>
are not available to the organism within the mammalian<br>
host, Sigwart et al., 1989, Infect. Immun. 57:1858.<br>
hereby incorporated by reference. Because auxotrophic<br>
mutations may be complemented by homologous recombination<br>
events with wild type copies donated from environmental<br>
organisms or by acquiring the needed metabolite within<br>
the host, it would seem prudent for live vaccines to<br>
contain a second attenuating mutation in a different<br>
virulence mechanism, (i.e., not just a second mutation in<br>
the same metabolic pathway). Additionally, in mice the<br>
aroA mutants have some residual virulence. Various<br>
strains with aroA mutations combined with phoP regulon<br>
mutations were investigated for virulence attenuation and<br>
immunogenicity. Table 4 demonstrates that a PhoP~ or<br>
PhoPc mutation further attenuates aroA mutant S.<br>
typhimurium by at least 100-fold and that, at least at<br>
high levels of vaccinating organisms, immunogenicity is<br>
retained. Strains with both a pagC~ and phoP0 phenotype<br>
are also further attenuated than either mutation alone.<br>
Therefore, phoP regulon mutations may increase the safety<br>
of aroA live vaccine preparations.<br>
(Table Removed)<br>
Salmonella typhi phoP Requlon Mutations<br>
The phoP regulon is at least partially conserved<br>
/<br>
in 5. typhi DNA hybridization studies as well as P22<br>
bacteriophage transductional crosses have documented that<br>
the phoP, phoQ, and page genes appear highly conserved<br>
between S. typhi and S. typhimurium mutations in these<br>
genes in S. typhi have been made.<br>
Salmonella Live Vaccines as Delivery Systems for<br>
Heterologous Antigens<br>
The vector used in the vaccine delivery system is<br>
a derivative of pJM703.1 described in Miller et al.,<br>
1988, J. Bact. 170:2575, hereby incorporated by<br>
reference. This vector is an R6K derivative with a<br>
deletion in the pir gene. R6K derivatives require the<br>
protein product of the pir gene to replicate. E. coli<br>
that contain the pir gene present as a lambda<br>
bacteriophage prophage can support the replication of<br>
this vector. Cells that do not contain the pir gene will<br>
not support the replication of the vector as a plasmid.<br>
This vector also contains the mob region of RP4 which<br>
will allow mobilization into other gram negative bacteria<br>
by mating from E. coli strains such as SMIOlambda pir,<br>
which can provide the mobilization function in trans.<br>
The pagC region is shown in Figs. 2 and 3. Fig. 2<br>
shows the restriction endonuclease sites of the pagC<br>
locus. The heavy bar indicates pagC coding sequence.<br>
The TnphoA insertion is indicated by a inverted triangle.<br>
The direction of transcription is indicated by the arrow<br>
and is left to right. The numbers indicate the location<br>
of endonuclease sites, in number of base pairs, relative<br>
to the start codon of predicted pagC translation with<br>
positive numbers indicating location downstream of the<br>
start codon and negative numbers indicating location<br>
upstream of the start codon. A is AccI, B is Bgll, C is<br>
Clal, D is Oral, E is EcoRI, H is //pal, N is Nrul, P is<br>
PstI, S is Sspl, T is Stult U is PvuII, V is £coRV, and<br>
II is Bgrlll. Fig. 3 shows the DNA sequence (Sequence<br>
I.D. No. 1) and translation of page::TnphoA. The heavy<br>
t<br>
underlined sequence indicates a potential ribosomal<br>
binding site. The single and double light underlines<br>
indicate sequences in which primers were constructed<br>
complementary to these nucleotides for primer extension<br>
of RNA analysis. The asterisk indicates the approximate<br>
start of transcription. The arrow indicates the<br>
direction of transcription. The boxed sequences indicate<br>
a region that may function in polymerase binding and<br>
recognition. The inverted triangle is the site of the<br>
sequenced TnphoA insertion junction. The arrow indicates<br>
a potential site for single sequence cleavage.<br>
3 kilobases of DNA containing the pagC gene (from<br>
the PstI restriction endonuclease site 1500 nucleotides<br>
5' to the start of page translation to the EcoRI<br>
restriction endonuclease site 1585 nucleotides downstream<br>
of pagC translation termination) were inserted into the<br>
pJM703.1 derivative discussed above. The pagC sequence<br>
from the Clal restriction endonuclease site 'was deleted<br>
(490 nucleotides) and replaced with a synthetic<br>
oligonucleotide polylinker that creates unique<br>
restriction endonuclease sites. DNA encoding one or more<br>
heterologous proteins, e.g., an antigen, can be inserted<br>
into this site. This creates a vector which allows the<br>
insertion of multiple foreign genes into the DNA<br>
surrounding pagC.<br>
The vector can be mobilized into Salmonella by<br>
mating or any other delivery system, e.g., heat shock,<br>
bacteriophage transduction or electroporation. Since it<br>
can not replicate, the vector can only insert into<br>
Salmonella by site specific recombination with the<br>
homologous DNA on both sides of the pagC gene. This will<br>
disrupt and inactivate the native pagC locus and replace<br>
it with the disrupted pagC DNA carried on the vector.<br>
Such recombination events can be identified by<br>
marker exchange and selective media if the foreign DNA<br>
inserted into the page locus confers a growth advantage.<br>
The insertion of antibiotic resistance genes for<br>
selection is less desirable as this could allow an<br>
increase in antibiotic resistance in the natural<br>
population of bacteria. Genes which confer resistance to<br>
substances other than antibiotics e.g., to heavy metals<br>
or arsenic (for mercury resistance, see Nucifora et al.,<br>
1989, J. Bact., 171:4241-4247, hereby incorporated by<br>
reference), can be used to identify transformants.<br>
Alternatively, selection can be performed using a<br>
Salmonella recipient strain that carries an auxotrophic<br>
mutation in a metabolic pathway and a vector that carries<br>
DNA that complements the auxotrophic mutation. Many<br>
Salmonella live vaccine prototypes contain mutations in<br>
histidine or purine pathways thus complementation of<br>
these metabolic auxotrophies can be used to select for<br>
integrants. (Purine mutations specifically have been<br>
shown to be too attenuated for use in man.) Further<br>
proof of marker exchange can be documented by loss of the<br>
ampicillin resistance (carried on the plasmid backbone)<br>
or by blot hybridization analysis.<br>
A gene useful for selection can be cloned by<br>
complementation of a vaccine strain with a metabolic<br>
auxotrophy. Specific examples include the cloning of the<br>
DNA encoding both purB and phoP by complementation of a<br>
strain deleted for function of both these- genes.<br>
Salmonella gene libraries have been constructed in a<br>
pLAFR cosmid vector (Frindberg et al., 1984, Anal.<br>
Biochem. 137:266-267, hereby incorporated by reference)<br>
by methods known to those skilled in the art. pLAFR<br>
cosmids are broad host range plasmids which can be<br>
mobilized into Salmonella from E. coli. An entire bank<br>
of such strains can be mobilized into' Salmonella vaccine<br><br>
strains and selected for complementation of an<br>
auxotrophic defect (e.g., in the case of purB growth on<br>
media without adenine). The DNA able to complement this<br>
defect is then identified and can be cloned into the<br>
antigen delivery vector.<br>
As discussed above heterologous genes can be<br>
inserted into the polylinker that is inserted into the<br>
page sequence of the vector. The heterologous genes can<br>
be under the control of any of numerous environmentally<br>
regulated promoter systems which can be expressed in the<br>
host and shut off in the laboratory. Because the<br>
expression of foreign proteins, especially membrane<br>
proteins (as are most important antigens), is frequently<br>
toxic to the bacterium, the use of environmentally<br>
regulated promoters that would be expressed in mammalian<br>
tissues at high levels but which could be grown in the<br>
laboratory without expression of heterologous antigens<br>
would be very desirable. Additionally, high expression<br>
of antigens in host tissues may result in increased<br>
attenuation of the organism by diverting the metabolic<br>
fuel of the organism to the synthesis of heterologous<br>
proteins. If foreign antigens are specifically expressed<br>
in host phagocytic cells this may increase the immune<br>
response to these proteins as these are the cells<br>
responsible for processing antigens.<br>
The promoter systems likely to be useful include<br>
those nutritionally regulated promoter systems for which<br>
it has been demonstrated that a specific nutrient is not<br>
available to bacteria in mammalian hosts. Purines,<br>
Sigwart et al., 1989, Infect. Immun., 5.7:1858 and iron,<br>
Finklestein et al., 1983, Rev. Infect. Dis. 5:S759, e.g.,<br>
are not available within the host. Promoters that are<br>
iron regulated, such as the aerobactin gene promoter, as<br>
well as promoters for biosynthetic genes in purine<br>
pathways, are thus excellent candidates for testing as<br>
promoters that can be shut down by growth in high<br>
concentrations of these nutrients. Other useful<br>
environmentally regulated Salmonella promoters include<br>
promoters for genes which encode proteins which are<br>
specifically expressed within macrophages, e.g., the DnaK<br>
and GroEL proteins, which are increased by growth at high<br>
temperature, as well as some phoP activated gene<br>
products, Buchmeier et al., 1990, Science 248:730. hereby<br>
incorporated by reference. Therefore, promoters such as<br>
the pagC 5' controlling sequences and the better<br>
characterized promoters for heat shock genes, e.g., GroEL<br>
and DnaK, will be expected to be activated specifically<br>
within the macrophage. The macrophage is the site of<br>
antigen processing and the expression of heat shock genes<br>
in macrophages and the wide conservation of heat shock<br>
genes in nature may explain the immunodominance of these<br>
proteins. A consensus heat shock promoter sequence is<br>
known and can be used in the vectors (Cowling et al.,<br>
1985, Proc. Natl. Acad. Sci. USA 82:2679, hereby<br>
incorporated by reference).<br>
The vectors can include an environmentally<br>
regulated T7 polymerase amplification system to express<br>
heterologous proteins. For example, the T7 polymerase<br>
gene (cloned by Stan Tabor and Charles Richardson, See<br>
Current Protocols in Molecular Biology ed. Ausubel et<br>
al., 1989, (page 3.5.1.2) John Wiley and Sons, hereby<br>
incorporated by reference) under control of an iron<br>
regulated promoter, can be included on the vectors<br>
described above. We have inserted the aerobactin gene<br>
promoter of E. coli with the sequence<br>
CATTTCTCATTGATAATGAGAATCATTATTGACATAATTGTTATTATTTTACG<br>
(SEQ ID NO:2), Delorenzo et al. J. Bact. 169:2624, hereby<br>
incorporated by reference, in front of the T7 polymerase<br>
gene and demonstrated iron regulation of the gene<br>
product. This version of the vector'will also include<br>
one or more heterologous antigens under the control of T7<br>
polymerase promoters. It is well known that RNA can be<br>
synthesized from synthetic oligonucleotide T7 promoters<br>
and purified T7 in vitro. When the organism encounters<br>
low iron T7 polymerase will be synthesized and high<br>
expression of genes with T7 promoters will be<br>
facilitated.<br>
page-fusion proteins in S. typhimurium<br>
Expression of heterologous antigens within<br>
macrophages under the control of phoP regulated promoters<br>
can be used as an effective method of both attenuating<br>
Salmonellae and enhancing immunogenicity of foreign<br>
antigens. As discussed above, the expression of Page is<br>
induced in antigen processing cell, i.e., a macrophage.<br>
Thus, expression of a heterologous antigen under the<br>
control of the page promoter is also likely to be<br>
inducible in macrophages.<br>
To evaluate the immune respone to a heterologous<br>
antigen expressed under the control of inducible pag<br>
promoters, mice were inoculated with bacteria which<br>
expressed the antigen, AP, under the control of the page<br>
or pagD regulatory sequences. Pagr-AP fusion proteins<br>
were produced in these bacteria from a single chromosomal<br>
copy of the gene encoding AP. The bacteria were<br>
generated utilizing two methods: TnphoA mutagenesis, and<br>
genetic engineering techniques using a suicide vector,<br>
both of which are described above.<br>
As a control, mice were innoculated with bacteria<br>
which expressed an AP fusion protein under the control of<br>
constitutive promoters. The constitutive promoter was<br>
completely independent of regulation by genes in the PhoP<br>
regulon. Two such strains of bacteria, Strain 610 and<br>
Strain 617, were constructed using methods described<br>
above. AP expression in Strain 610 was moderate, whereas<br>
AP expression in Strain 617 was high (see Fig. 14C).<br>
These strains were injected intraperitoneally into<br>
BABL/C mice. Serum samples were taken three weeks after<br>
inoculation. Normal mouse serum (MNS) was used as a<br>
control. Standard ELISA assays were used to test the<br>
sera for the presence of AP-specific antibodies. Sera<br>
was also tested for LPS-specific antibodies using S.<br>
typhimurium LPS. Antibodies directed to LPS were<br>
detected in all the murine sera tested, but only those<br>
strains in which AP was expressed as a Pag fusion protein<br>
from a single chromosomal gene copy engendered an immune<br>
response against the model heterologous antigen, AP (see<br>
Figs. 14A and Fig. 14B).<br>
Despite approximately 10-fold higher constitutive<br>
expression of the AP fusion in strain 617, only a minimal<br>
immune response to this antigen was noted after<br>
immunization with strain 617. In contrast, a strong<br>
response was observed in mice inoculated with strains<br>
which expressed the Pagr-AP fusion protein. These data<br>
indicate that phoP-regulation which results in in vivo<br>
induction of protein expression within macrophages<br>
increases the immunogenicity of heterologous antigens<br>
expressed under the control of the pag promoters. Any<br>
promoter which directs cell-specific, inducible<br>
expression of a protein in macrophages or other antigen<br>
presenting cells, e.g., pag described herein, can be used<br>
to increase the immunogencity of an antigen expressed in<br>
Salmonella.<br>
The pagC gene and pagC Gene Product<br>
Strains, materials, and methods<br>
The following strains, materials, and methods were<br>
used in the cloning of pagC and in the analysis of the<br>
gene and its gene product.<br>
Rich media was Luria broth (LB) and minimal media<br>
was M9, Davis et al., 1980, supra. The construction of<br>
S. typhimurium strain CS119 pagrCl: :TnphoA phoN2 zxx::6251<br>
TnlOd-Cam was previously described, Miller et al., 1989,<br>
supra. American Type Culture Collection (ATCC) S.<br>
typhimurium strain 10428 included CS018 which is isogenic<br>
/<br>
to CS119 except for phoPlOS::TnlOd, Miller et al., 1989,<br>
supra, CS022 pho-24, Miller et al., 1990, J. Bacteriol.<br>
172;2485-2490, hereby incorporated by reference, and<br>
CS015 phoP102::TnlOd-cam, Miller et al., 1989, supra.<br>
Other wild type strains used for preparation of<br>
chromosomal DNA included S. typhimurium LT2 (ATCC 15277),<br>
S. typhimurium Ql and S. drypool (Dr. J. Peterson U.<br>
Texas Medical Branch, Galveston), and Salmonella typhi<br>
Ty2 (Dr. Caroline Hardegree, Food and Drug<br>
Administration). pLAFR cosmids were mobilized from E.<br>
coli to S. typhimurium using the E. coli strain MM294<br>
containing pRK2013, Friedman et al., 1982, Gene 18:289-<br>
296, hereby incorporated by reference. AP activity was<br>
screened on solid media using the chromogenic phosphatase<br>
substrate 5-bromo-4-chloro-3-indolyl phosphate (XP) . AP<br>
assays were performed as previously described, Brickman<br>
et al., 1975, J. Mol. Biol. 9.6:307-316, hereby<br>
incorporated by reference, and are reported in units as<br>
defined by Miller, Miller, 1972, supra, pp. 352-355.<br>
One dimensional protein gel electrophoresis was<br>
performed by the method of Laemmli, 1970, Nature,<br>
227:680-685, hereby incorporated by reference, and blot<br>
hybridization using antibody to AP was performed as<br>
previously described, Peterson et al., 1988, Infect.<br>
Immun. 5J5:2822-2829, hereby incorporated by reference.<br>
Whole cell protein extracts were prepared, from saturated<br>
cultures grown in LB at 37°C with aeration, by boiling<br>
the cells in SDS-pagE sample buffer, Laemmli, 1970,<br>
supra. Two dimensional gel electrophoresis was performed<br>
by the method of O'Farrell, 1975, J. Biol, Chem.<br>
250:4007, hereby incorporated by reference. Proteins in<br>
the 10% polyacrylamide slab gels were visualized by<br>
silver staining, Merril et al., 1984, Methods in<br><br>
Enzymology, 104:441, hereby incorporated by reference.<br>
Chromosomal DNA was prepared by the method of<br>
Mekalanos, 1983, Cell, 3_5:253-263, hereby incorporated by<br>
reference. DNA, size fractionated in agarose gels, was<br>
transferred to nitrocellulose (for blot hybridization) by<br>
the method of Southern, 1975, J. Mol. Biol. 98:503-517,<br>
hereby incorporated by reference. DNA probes for<br>
Southern hybridization analysis were radiolabeled by the<br>
random primer method, Frinberg et al., 1984, supra.<br>
Plasmid DNA was transformed into E. coli and Salmonella<br>
by calcium chloride and heart shock, Mekalanos, 1983,<br>
supra, or by electroporation using a Genepulser apparatus<br>
(BioRad, Richmond, Ca.) as recommended by the<br>
manufacturer, Dower et al., 1988, Nucl. Acids Res.<br>
3J5: 6127-6145, hereby incorporated by reference. DNA<br>
sequencing was performed by the dideoxy chain termination<br>
method of Sanger et al., 1977, Proc. Natl. Acad. Sci.<br>
USA, 7j4: 54 63-5467, hereby incorporated by reference, as<br>
modified for use with SEQUENASE® (U.S. Biochemical,<br>
Cleveland, Ohio). Oligonucleotides were synthesized on<br>
an Applied Biosystems Machine and used as primers for<br>
sequencing reactions and primer extension of RNA.<br>
Specific primers unique to the two ends of TnphoA, one of<br>
which corresponds to the AP coding sequence and the other<br>
to the right IS50 sequence, were used to sequence the<br>
junctions of the transposon insertion.<br>
Construction of a S. typhimurium cosmid gene bank<br>
in pLAFR3 and screening for clones containing the wild<br>
type pagC DNA was performed as follows. DNA from S.<br>
typhimurium strain ATCC 10428 was partially digested<br>
using the restriction endonuclease Sau3A and then size<br>
selected on 10-40% sucrose density gradient. T4 DNA<br>
ligase was used to ligate chromosomal DNA of size 20-30<br>
kilobases into the cosmid vector pLAFR3, a derivative of<br>
pLAFRl, Friedman et al., 1982, Gene 18.:289-296, hereby<br>
incorporated by reference, that was digested with the<br>
restriction endonuclease BamEI. Cosmid DNA was packaged<br>
and transfected into E. coli strain DH5-a using extracts<br>
purchased from Stratagene, La Jolla, Ca. Colonies were<br>
screened by blot hybridization analysis.<br>
The analysis of proteins produced from cloned DNA<br>
by in vitro transcription/translation assays was analyzed<br>
as follows. These assays were performed with cell free<br>
extracts, (Amersham, Arlington Heights, Illinois), and<br>
were performed using conditions as described by the<br>
manufacturer. The resultant radiolabeled proteins were<br>
analyzed by SDS-pagE.<br>
RNA was purified from early log and stationary<br>
phase Salmonella cultures by the hot phenol method, Case<br>
et al., 1988, Gene 7.2:219-236, hereby incorporated by<br>
reference, and run in agarose-formaldehyde gels for blot<br>
hybridization analysis, Thomas, 1980, Proc. Natl. Acad.<br>
Sci. USA 22.' 5201, hereby incorporated by reference.<br>
Primer extension analysis of RNA was performed as<br>
previously described, Miller et al., 1986, Nuc. Acids.<br>
Res. ^4:7341-7360, hereby incorporated by reference,<br>
using AMV reverse transcriptase (Promega, Madison,<br>
Wisconsin) and synthesized oligonucleotide primers<br>
complementary to nucleotides 335-350 and 550-565 of the<br>
page locus.<br>
Identification of an 18 kDa protein missing in a page<br>
mutant of S. typhimurium<br>
page mutant strain CS119 was analyzed by two<br>
dimensional protein electrophoresis to detect protein<br>
species that might be absent as a result of the TnphoA<br>
insertion. Only a single missing protein species, of<br>
approximately 18 kD and pl-8.0, was observed when<br>
strains, isogenic except for their transposon insertions,<br>
were subjected to this analysis. This 18 kDa species was<br>
also missing in similar analysis of Salmonella strains<br>
with mutations phoP and phoQ. Though two-dimensional<br>
protein gel analysis might not detect subtle changes of<br>
protein expression in strain CS119, this suggested that a<br>
single major prote'in species was absent as a result of<br>
the pagC::TnphoA insertion.<br>
Additional examination of the 2-dimensional gel<br>
analysis revealed a new protein species of about 45 kDa<br>
that is likely the pagrC-Ap fusion protein. The pagC-AP<br>
fusion protein was also analyzed by Western blot analysis<br>
using antisera to AP and found to be similar in size to<br>
native AP (45 kDa) and not expressed in PhoP-S.<br>
typhimurium.<br>
Cloning of the pagC::TnphoA insertion<br>
Chromosomal DNA was prepared from S. typhimurium<br>
strain CS119 and a rough physical map of the restriction<br>
endonuclease sites in the region of the pagC::TnphoA<br>
fusion was determined by using a DNA fragment of TnphoA<br>
as a probe in blot hybridization analysis. This work<br>
indicated that digestion with the restriction<br>
endonuclease ecoRV yielded a single DNA fragment that<br>
included the pagC::TnphoA insertion in addition to<br>
several kilobases of flanking DNA. Chromosomal DNA from<br>
strain CS119 was digested with EcoRV (blunt end) and<br>
ligated into the bacterial plasmid vector pUC19 (New<br>
England Biolabs) that had been digested with the<br>
restriction endonuclease Smal (blunt end). This DNA was<br>
electroporated into the E. coli strain DH5-a (BRL) and<br>
colonies were plated onto LB agar containing the<br>
antibiotics kanamycin (TnphoA encoded and ampicillin<br>
(pUC19 encoded). A single ampicillin and kanamycin<br>
resistant clone containing a plasmid designated pSMlOO<br>
was selected for further study.<br>
A radiolabeled DNA probe from pSMlOO was<br>
constructed and used in Southern hybridization analysis<br>
of strain CS119 and its wild type parent ATCC 10428 to<br>
prove that the pagC::TnphoA fusion had been cloned. The<br>
probe contained sequences immediately adjacent to the<br>
transposon at the opposite end of the AP gene [Hpal<br>
endonuclease generated DNA fragment that included 186<br>
bases of the right IS50 of the transposon and 1278 bases<br>
of Salmonella DNA (Fig. 2) . As expected, the pSMlOO<br>
derived probe hybridized to an 11-12 kb AccI endonuclease<br>
digested DNA fragment from the strain containing the<br>
transposon insertion, CS119. This was approximately<br>
7.7kb (size of TnphoA) larger than the 3.9 kB AccJ<br>
fragment present in the wild type strain that hybridizes<br>
to the probe. In addition, a derivative of plasmid<br>
pSMlOO, pSMlOl (which did not allow expression of the<br>
pagC-PhoA gene fusion off the lac promoter), was<br>
transformed into phoP- (strain CsOlS) and phoN- (strain<br>
CS019) Salmonella strains and the cloned AP activity was<br>
found to be dependent on phoP for expression. Therefore<br>
we concluded that the cloned DNA contained the<br>
pagC::TnphoA fusion.<br>
The presence of the pagC gene was also<br>
demonstrated in other strains of S. typhimurium, as well<br>
as in S. typhi, and S. drypoo 1. All Salmonella strains<br>
examined demonstrated similar strong hybridization to an<br>
8.0 kb EcoRV and a 3.9 kb AccI restriction endonuclease<br>
fragment suggesting that pagC is a virulence gene common<br>
to Salmonella species.<br>
The pagC gene probe from nucleotides -46 (with 1<br>
as the first base of the methionine to 802 (PstI site to<br>
the Bglll site) failed to cross hybridize to DNA from<br>
Citrobacter freundii, Shigella flexneri, Shigella sonnei,<br>
Shigella dysenterial, Escherichia coli, Vibrio cholerae,<br>
Vibrio vulnificus, Yersenia entero colitica, and<br>
Klebsiella pneumonia.<br>
Cloning of the wild type pagC locus DNA and its<br>
complementation of the virulence defect of a 5.<br>
typhimurium pagC mutant<br>
The same restriction endonuclease fragment<br>
described above was used to screen a cosmid gene bank of<br>
wild type strain ATCC 10428. A single clone, designated<br>
pWPOGl, contained 18 kilobases of S. typhimurium DNA and<br>
hybridized strongly to the pagC DNA probe. pWPOGl was<br>
found to contain Salmonella DNA identical to that of<br>
pSMlOO when analyzed by restriction endonuclease analysis<br>
and DNA blot hybridization studies. Probes derived from<br>
pWP061 were also used in blot hybridization analysis with<br>
DNA from wild type and CS119 S. typhimurium. Identical<br>
hybridization patterns were observed to those seen with<br>
pSMlOO. pWP061 was also mobilized into strain CS119, a<br>
pagC mutant strain. The resulting strain had wild type<br>
virulence for BALB/c mice (a LD50 less than 20 organisms<br>
when administered by IP injection). Therefore the cloned<br>
DNA complements the virulence defect of a pagC mutant<br>
strain.<br>
Since, a wild type cosmid containing pagC locus<br>
DNA was found to complement the virulence defect of a<br>
pagC mutant S. typhimurium strain, it was concluded that<br>
the pagC protein is an 188 amino acid (18 kDa) membrane<br>
(see below) protein essential for survival within<br>
macrophages and virulence of S. typhimurium.<br>
Physical mapping of restriction endonuclease sites, DNA<br>
sequencing, and determination of the pagC gene product<br>
Restriction endonuclease analysis of plasmid pSMlOO and<br>
pWP061 was performed to obtain a physical map of the pagC<br>
locus, and, in the case of PSMlOO, to determine the<br>
direction of transcription (Fig. 2). DNA subclones were<br>
generated and the TnphoA fusion junctions were sequenced,<br>
as well as the Salmonella DNA extending from the Hpal<br>
site, 828 nucleotides 5' to the phoA fusion junction, to<br>
the EcoRI site 1032 nucleotides 3' to the TnphoA<br><br>
insertion (Fig. 2 and 3). The correct reading frame of<br>
the DNA sequence was deduced from that required to<br>
synthesize an active AP gene fusion. The deduced amino<br>
acid sequence of this open reading frame was predicted to<br>
encode a 188 amino acid protein with a predicted pI+8.2.<br>
This data were consistent with the 2-D polyacrylamide gel<br>
analysis of strain CS119 in which an 18 kDa protein of<br>
approximate pI+8.0 was absent. No other open reading<br>
frames, predicted to encode peptides larger than 30 amino<br>
acids, were found.<br>
The deduced amino acid sequence of the 188 amino<br>
acid open reading frame contains a methionine start codon<br>
33 amino acids from the fusion of page and AP (Fig. 3).<br>
This 33 amino acid pagC contribution to the fusion<br>
protein was consistent with the size observed in Western<br>
blot analysis and contains a hydrophobic N-terminal<br>
region, identified by the method of Kyle et al., 1982, J.<br>
Mol. Biol. 157;105-132, hereby incorporated by reference,<br>
that is a typical bacterial signal sequence, Von Heinje,<br>
1985, J. Mol. Biol. 184:99-105, hereby incorporated by<br>
reference. Specifically, amino acid 2 is a positively<br>
charged lysine, followed by a hydrophobic domain and<br>
amino acid 24 is a negatively charged aspartate residue.<br>
A consensus cleavage site for this leader peptide is<br>
predicted to be at an alanine residue at amino acid 23,<br>
Von Heinje, 1984, J. Mol. Biol. 173:243-251, hereby<br>
incorporated by reference. The DNA sequence also<br>
revealed a typical ribosomal binding site, Shine et al.,<br>
1974, Proc. Natl. Acad. Sci. USA 7JL: 1342-1346, hereby<br>
incorporated by reference, at 6-2 nucleotides 5' to the<br>
predicted start of translation (Fig. 3) nucleotides 717-<br>
723). This suggested that the open reading frame was, in<br>
fact, translated and further supported the assumption<br>
that this was the deduced amino acid 'sequence of the pagC<br>
protein interrupted by the TnphoA insertion (Fig. 3).<br>
In vitro synthesis of proteins by the cloned page locus<br>
To detect if other proteins were encoded by page<br>
and to determine the approximate size of the page gene<br>
product, an in vitro coupled transcription/translation<br>
analysis was performed. A 5.3 kilobase EcoRI fragment of<br>
pWP061 was inserted into pUC19 so that the page gene<br>
would not be expressed off the lac promoter. This<br>
plasmid was used in an in vitro coupled transcriptiontranslation<br>
assay. A single protein of approximately 22<br>
kilodaltons was synthesized by the cell free system. The<br>
size was compatible with this being the precursor of the<br>
pagC protein containing its leader peptide. These data<br>
further support the conclusion the single and the single<br>
pagC gene product had been identified.<br>
Identification of the page encoded RNA<br>
An approximately 1100 nucleotide RNA is encoded by<br>
pagC. The pagC gene is highly expressed by cells with a<br>
phoP constitutive phenotype of pag activation, as<br>
compared to wild type and phoP constitutive phenotype of<br>
pag activation, as compared to wild type and phoP~<br>
bacteria. In these blot hybridization experiments pagC<br>
is only detected in wild type cells grown in rich media<br>
during stationary growth. This result, coupled with<br>
previous work, Miller et al., 1989, supra, Miller et al.,<br>
1990, supra, demonstrates that pagC is transcriptionally<br>
regulated by the phoP gene products and is only expressed<br>
during early logarithmic phase growth in rich media by<br>
cells with a phoP constitutive phenotype.<br>
The size of-the pagC transcript is approximately<br>
500 nucleotides greater than that necessary to encode the<br>
188 amino acid protein. Primer extension analysis of<br>
Salmonella RNA using oligonucleotide primers specific for<br>
pagC sequence was performed to determine the approximate<br>
start site of transcription and to determine whether<br>
these nucleotides might be transcribed 5' or 3' to the<br>
188 amino acid page gene product. Primer extension<br>
analysis with an oligonucleotide predicted to be<br>
complementary to nucleotides 550-565 of pagC, 150<br>
nucleotides 5' to the predicted start codon, resulted in<br>
an approximately 300 nucleotide primer extension product.<br>
Therefore a primer further upstream was constructed<br>
complementary to nucleotides 335-350 of pagC and used in<br>
a similar analysis. A primer extension product of 180<br>
nucleotides was observed to be primer specific. This is<br>
consistent with transcription starting at nucleotide 170<br>
(Fig. 3). Upstream of the predicted transcriptional<br>
start, at nucleotides 153-160, a classic RNA polymerase<br>
binding site was observed with the sequence TATAAT at -12<br>
nucleotides as well as the sequence TAATAT at -10<br>
nucleotides. No complete matches were observed for the<br>
consensus RNA polymerase recognition site (TTGACA) 15-21<br>
nucleotides upstream from the -10 region. AT -39 (126-<br>
131) nucleotides (TTGGAA), -38 (127-132) nucleotides<br>
(TTGTGG), and -25 (135-140) nucleotides (TTGATT) are<br>
sequences that have matches with the most frequently<br>
conserved nucleotides of this sequence.<br>
Based on the above results transcription was<br>
predicted to terminate near the translational stop codon<br>
of the 188 amino acid protein (nucleotide 1295, Fig. 3).<br>
Indeed, a stem loop configuration was found at<br>
nucleotides 1309-1330 that may function as a<br>
transcription terminator. This was consistent with the<br>
lack of evidence of open reading frames downstream of the<br>
188 amino acid protein and the lack of synthesis of other<br>
transcription/translation using the cloned pagC DNA.<br>
This further suggests that the pagC::TnphoA insertion<br>
inactivated the synthesis of only a single protein.<br>
Similarity of page to Ail and Lorn<br>
A computer analysis of protein similarity using<br>
the National Biomedical Research Foundation/Protein<br>
Identification Resource, George et al., 1986, Nucleic<br>
Acids Res. 14.: 11-15, hereby incorporated by reference,<br>
protein sequence base was conducted to identify other<br>
proteins that had similarity to pagC in an attempt to<br>
find clues to the molecular function of this protein.<br>
Remarkably, page was found to be similar to a<br>
bacteriophage lambda protein, Lorn, that has been<br>
localized to the outer membrane in minicell analysis,<br>
Court et al., 1983, Lambda II, Hendrix, R.W. et al. ed.<br>
Cold Spring Harbor Laboratory (Cold Spring Harbor NY),<br>
pp. 251-277, hereby incorporated by reference, and<br>
demonstrated to be expressed by lambda lysogens of E.<br>
coli, Barondess, et al., 1990, Nature 346:871-874. hereby<br>
incorporated by reference. Recently, the deduced amino<br>
acid sequence of the cloned ail gene product of Y.<br>
enterocolitica was determined and found to also be<br>
similar to Lorn, Miller et al.y 1990b, J. Bacteriol.<br>
172;1062-1069. Therefore, a protein family sequence<br>
alignment was performed using a computer algorithm that<br>
establishes protein sequence families and consensus<br>
sequences, Smith et al., 1990, Proc. Natl. Acad. Sci.<br>
£2:118-122, hereby incorporated by reference. The<br>
formation of this family is indicated by the internal<br>
data base values of similarity between these proteins :<br>
page and Lorn (107.8), pagC and Ail (104.7), and Ail and<br>
Lorn (89.8). These same proteins were searched against<br>
314 control sequences in the data base and mean values<br>
and ranges were 39.3 (7.3-52.9) pagC, 37.4 (7.3-52.9)<br>
Ail, and 42.1 (7.0-61.9) Lorn. The similarity values for<br>
this protein family are all greater than 3.5 standard<br>
deviations above the highest score obtained for<br>
similarity to the 314 random sequences. No other<br>
similarities or other family members were found in the<br>
database. Regions of similarity are located not only in<br>
the leader peptide transmembrane domains but throughout<br><br>
the protein.<br>
pag Mutant Strains. Are Attenuated For Virulence<br>
Salmonella typhimurium strains of the invention<br>
with a pagC mutation were attenuated for virulence by<br>
least 1,000-fold.<br>
In addition pagC, other pag genes described herein<br>
may be useful in the development of live Salmonella<br>
vaccines. Mutations in phoP-activated genes could be<br>
used to construct attenuated, live Salmonella vaccines.<br>
In constructing multivalent Salmonella vectored vaccines,<br>
PhoP-activated promoters could increase immunogenecity by<br>
targeting foreign protein expression to antigen<br>
presenting macrophages.<br>
Identification of novel phoP-activated genes.<br>
To further analyze the role of phoP-activated<br>
genes in bacterial virulence, a bank of strains with<br>
active phoA gene fusions was generated by TnphoA<br>
mutagenesis. Strain CS019 was the parent strain for<br>
TnphoA mutagenesis because it has wild-type bacterial<br>
virulence and carries the phoN2 allele, which results in<br>
minimal background phosphatase activity. Strains with<br>
active phoA gene fusions were identified by blue colony<br>
phenotype after growth in agar containing XP. Such<br>
strains were then screened for decreased fusion protein<br>
activity on acquisition of the phoP12 allelle that<br>
results in a PhoP-null phenotype.<br>
Two thousand and sixty-four AP expressing strains<br>
were isolated and colony purified from two hundred and<br>
forty independent matings. Strains with AP activity were<br>
isolated at a frequency of 0.8% from the total pool of<br>
kanamycin resistant (TnphoA containing) bacteria. A<br>
total of fifty-four candidate pag::TnphoA insertions were<br>
isolated from the AP expressing strain bank, and fortynine<br>
of these were determined to have-greater than sixfold<br>
reduction in AP activity in the absence of<br><br>
functional phoP/phoQ. Therefore, approximately 2% of the<br>
colonies expressing AP were identified as pag-phoA gene<br>
fusions.<br>
Identification of thirteen unique pag loci.<br>
Three methods were used to determine whether the<br>
forty-nine TnphoA insertions defined unique pag loci.<br>
First, physical maps of the JE7coRI and HindJ.II restriction<br>
endonuclease sites 5' to the TnphoA insertions were<br>
defined. Second, linkage analysis to transposon<br>
insertions highly linked to known pag loci was performed.<br>
Third, strains determined to be unique by the above<br>
methods were screened for linkage to a bank of strains<br>
with transposon insertions of known chromosomal location.<br>
Blot hybridization analysis demonstrated that<br>
thirteen of the forty-nine strains had unique restriction<br>
endonuclease sites 5' to the TnphoA insertion. The<br>
numbers of strains with similar physical maps 5' to the<br>
TnphoA insertion ranged from 1-7. One of the thirteen<br>
physical maps was similar to that expected for an<br>
insertion in page and was noted in seven of the strains<br>
isolated as containing candidate pag::TnphoA insertions.<br>
Analysis of these seven strains indicated that only three<br>
of these were pagC::TnphoA insertions, since blot<br>
hybridization analysis with a fragment of pagrC as a probe<br>
and linkage analysis to transposon insertions highly<br>
linked to page indicated that four of these insertions<br>
were not in pagC. Another of the pag::phoA fusions,<br>
denoted pagP, had the physical 5' restrictionendonuclease<br>
map that would be expected for phoN.<br>
However, this insertion was determined not to be within<br>
phoN by linkage analysis and blot hybridization. A<br>
(Table Removed)<br>
transductional cross was performed between wild type<br>
bacteria and strain CS1247 containing pagP::TnphoA and<br>
zxx: : 6215Tnl0d-cajn. These transductants were selected on<br>
kanamycin, insuring the inheritance of the pagP::TnphoA<br>
which encodes kanamycin resistance. These colonies were<br>
then screened for choramphenicol resistance which would<br>
indicate linkage of zxx: 6215TnlOd-cajn to pagP. No<br>
linkage was found indicating that pagP was not linked to<br>
phoN. Blot hybridization using a portion of phoN as a<br>
probe was also performed on CS1247 and indicated that<br>
this strain contained a wild type phoN locus. Thirteen<br>
pag loci were defined and designated pagD-P.<br>
To further define the PhoP regulation of the 13<br>
pag::TnphoA fusion proteins, AP activity was assayed in<br>
strains isogenic except for the phoP locus. AP activity<br>
was assayed during bacterial growth in rich medium in<br>
logarithmic and stationary growth phase (Table 13). The<br>
dependence of an intact phoP locus for full expression<br>
remained constant for the different stages of growth;<br>
however, the relative amount of AP expression increased<br>
as growth was limited. The difference in expression of<br>
pag gene fusions varied from six to forty-eight fold when<br>
isogenic strains with a wild type and null phoP locus<br>
were compared.<br>
Of the five previously identified pag loci, only<br>
phoN, pagC, and pagA have known chromosomal locations.<br>
Linkage analysis of the 13 newly identified pag loci was<br>
performed using strains containing transposon insertions<br>
linked to pagC (AK3233 and AK3140), and to pagA (AK3255).<br>
Three pag::TnphoA insertions were foupd to be linked to<br>
AK3140 which is in a region near pagC at 24-25 minutes on<br>
the chromosome. These were designated pagD, pagE, and<br>
pagF. PagD::TnphoA was similarly linked to the<br>
transposon insertion of AK3233 (83%) and AK3140 (33%) as<br>
was previously reported for pagC. The TnphoA insertion<br>
of this strain has been further defined and is<br>
divergently transcribed from page. pagE and pagF<br>
exhibited different linkage to the insertions of AK3233<br>
and AK3140 than pagC and pagD suggesting a significantly<br>
different chromosomal location. The pagE::TnphoA<br>
insertion is 39% linked to the transposon insertion of<br>
AK3233 and 99.1% linked to that of AK3140, while<br>
pagF::TnphoA is 31% linked to the insertion of AK3140 but<br>
not to that of AK3233. These different linkages in<br>
addition to the physical maps of the restriction<br>
endonuclease sites 5' to the TnphoA insertion indicated<br>
that these were new pagr loci. Therefore, three new pag<br>
loci were found in the region of 25 minutes, one of which<br>
is highly linked to the previously defined pagC.<br>
Linkage analysis was then performed using a group<br>
of defined random Tnl0Al6Al7 insertions on the ten<br>
strains with TnphoA insertions of no known location. Of<br>
these ten pag::TnphoA alleles only two demonstrated<br>
linkage to the bank of TnlOAl6Al7 insertions. The<br>
pagG::TnphoA insertion was demonstrated to have 97%<br>
linkage to the transposon insertion of AK3258 located at<br>
approximately 30 minutes. The pag::TnphoA insertion,<br>
designated pagH, exhibited 23% linkage to the insertion<br>
of AK3091. The linkage to the transposon insertion of<br>
AK3091 was similar to linkage previously demonstrated for<br>
prgj? (26%) . Therefore, this chromosomal region contains<br>
both PhoP-activated and repressed genes. This TnAl6Al7<br>
insertion was analyzed using pulse field gradient<br>
electrophoresis of chomosomal DNA from AK3091 digested<br>
with the restriction endonuclease Xbal and Blnl. These<br>
data indicate that the transposon insertion of AK3091 was<br>
located in the region of 20-25 minutes and that pagH and<br>
prg£ are located in this region of the chromosome.<br><br>
(Table Removed)<br>
gtrains with pag:TnphoA insertions have wild type<br>
sensitivity to the rabbit NP-1 defensin<br>
S. Typhimurium strains with null mutations in the<br>
phoP operon have increased sensitivity to a variety of<br>
cationic antimicrobial peptides including defensins,<br>
magainins, and protamine. The defensins are a family of<br>
mammalian peptides present in the granules of<br>
neutrophils, lung macrophages, and intestinal Paneth<br>
cells. Resistance to these peptides may contribute to<br>
bacterial virulence and the ability to colonize mucosal<br>
surfaces. Strains with pag::TnphoA insertions were<br>
tested for sensitivity to the highly active rabbit<br>
defensin NP-1. None of the strains with single<br>
pag::TnphoA insertions demonstrated increased sensitivity<br>
to NP-1 defensin (see Fig. 6). Thus despite the<br>
demonstrated sensitivity of PhoP-null mutants to rabbit<br>
defensin NP-1, no defined mutations in pag loci were<br>
associated with sensitivity to defensins.<br>
Four strains with pag::TnphoA insertions demonstrate<br>
marked attenuation for mouse virulence<br>
To further define whether these new pag loci<br>
contributed to mouse virulence, the 13 strains with pag<br>
transposon insertions were screened in vivo. Mice were<br>
injected intraperitoneally with approximately 100<br>
organisms. Four strains with transposon insertions in<br>
pagD, pagJ, pagK, and pagM demonstrated attenuated<br>
virulence. Mice injected with these strains all survived<br>
and showed no signs of systemic infections, such as<br>
hepatosplenomegaly and scruffiness (piloerection due to<br>
fever). These four strains were subjected to further<br>
virulence testing by intraperitoneal injection of<br>
multiple doses of organisms in a total of ten mice on two<br>
separate occasions. The mean LD50 was determined from<br>
these subsequent injections and is lasted in Table 14.<br>
One of these strains, containing the pagD::TnphoA<br>
insertion, has a LD50 10,000 fold greater than wild-type<br>
S. typhimurium. The other three strains were also<br>
markedly attenuated for mouse virulence with LD50 values<br>
greater than 1000-10,000 times that of wild type<br>
organisms. These data indicated that the PhoP-regulated<br>
loci, pagD, pagJ, pagK, and pagM, when mutated, result in<br>
attenuation of bacterial virulence.<br>
pag::TnphoA strains attenuated for mouse virulence have<br>
reduced survival within macrophages.<br>
Since PhoP mutant Salmonella are deficient in<br>
survival within macrophages, strains containing mutations<br>
in pag genes that had attenuated mouse virulence were<br>
tested for reduced viability within macrophages. As<br>
shown Table 14, all strains with pag mutations<br>
demonstrated significantly reduced survival within<br>
macrophages. Decreased intracellular survival of pag<br>
mutants was not observed until a time when pag are<br>
predicted to be maximally expressed.<br>
Four strains with mutations in the pagC, pagD,<br>
pagJ, pagK and pagM loci were found to be attenuated for<br>
mouse virulence and survival within macrophages. Strains<br>
with mutations in these five pag all had varying degrees<br>
of virulence attenuation. Strains with a mutation in<br>
pagJ had a virulence defect comparable to that observed<br>
for pagC mutants (greater than 1000 x the LD50 of wild<br>
type organisms). The pagD: :Tr\phoA insertion resulted in<br>
the greatest attenuation of virulence, comparable to that<br>
of a PhoP null mutation (greater than 10,000 x the LD50 of<br>
wild type organisms). pagK and pagM mutants had<br>
virulence attenuation that was intermediate between the<br>
pagJ and pagD mutants. The cumulative effect of deletion<br>
of pagC, pagD, pagJ, pagK, and pagM, if additive and<br>
similar to the<br>
(Table Removed)<br><br>
The 50% lechai dose was determined by incrapenconeal injection of :en mice per<br>
dilucicn using che mechcd of Reed and Ntuer.ch ( 3 1 ) .<br>
i h e Nfacrophage Surv'ivai Index (MSI) '.vas cecerrGir.ed bv dividing che mean<br>
SaJcicr.si'z CFU recovered from rr.-acropnage culcures i'perfcrrned in t r i p l i c a t e ) <br>
hours after che addicion of gencamicin' by che mean Salmonella CPU recovered from<br>
macrophages 1 hour after gencamicin was added.<br>
!-6 -Cier ec al., L979, J- Bacceriol., 138:155-51<br>
25 Miller ec al., 1989, Proc. JTacl. Acad. Sci. USA, 36:5054-53<br>
attenuation observed with TnphoA insertions, may be much<br>
greater than that observed by deletion of phoP alone.<br>
The observation that many of these genes are somewhat<br>
expressed in stationary phase in the absence of PhoP<br>
suggests that functional Pag proteins could be produced<br>
in vivo in the absence of PhoP. One virulence gene pagM<br>
is significantly expressed in the absence of PhoP, though<br>
it may still require PhoP/PhoQ for induction within<br>
macrophage phagosomes. This data suggests that deletion<br>
of pag gene products could lead to greater virulence<br>
attenuation than deletion of the regulatory proteins.<br>
Salmonella envelope proteins as virulence factors:<br>
Defensin senstivity<br>
Based on the methods used to identify pag loci,<br>
i.e., translational gene fusions to phoA, and the<br>
observation that the page gene fusions produce AP, it has<br>
now been discovered that many pag encode bacterial<br>
envelope proteins. No strains have been found with<br>
single pag mutations that confer sensitivity to defensins<br>
or other cationic peptides. The data suggest that an<br>
alteration of the bacterial envelope as a result of the<br>
change in synthesis of the entire aggregate of envelope<br>
proteins mediated by PhoP/PhoQ may be important to S.<br>
typhimurium virulence.<br>
Defensins are small amphipathic cationic peptides<br>
of approximately 30-35 amino acids in length whose antimicrobial<br>
action involves penetration and disruption of<br>
membranes, possibly by forming selective anionic<br>
channels. Though defensins are largely found in<br>
neutrophils and Paneth cells these or other related<br>
molecules likely contribute to non-oxidative killing of<br>
phagocytosed bacteria by macrophages. Though it remains<br>
possible that a single unidentified pag encodes a protein<br>
responsible for defensin resistance, it seems more likely<br>
that a cumulative effect of expression of several pag<br>
encoded envelope proteins could result in resistance to<br>
defensins. An aggregate change in a large number of<br>
bacterial envelope proteins could alter the membrane<br>
charge, electrical potential, or lipid content such that<br>
defensin interaction with bacterial membranes could be<br>
changed.<br>
Identification of transcriptional units linked to page.<br>
To identify genes upstream of page, E. coli<br>
carrying plasmid pWPL17 containing 2.8kb of DNA 5' to<br>
pagC (Table 15 and Fig. 7) was mutagenized with the<br>
transposons MudJ and TnphoA, and strains with AP or /3-<br>
galactosidase activity were identified on chromogenic<br>
substrates. In addition, as part of an effort to<br>
identify additional PhoP-activated genes, random<br>
mutagenesis of the Salmonella chromosome with TnphoA was<br>
performed, and strains with AP activity were screened for<br>
TnphoA insertions linked to the TnlOAl6Al7 of strain<br>
AK3233, which is 75% linked to pagC. Several strains<br>
that contained plasmids with active MudJ or TnphoA<br>
generated gene fusions were identified. In addition, two<br>
strains were identified that contained active chromosomal<br>
TnphoA insertions closely linked to pagrC. Physical maps<br>
of the restriction endonuclease sites surrounding the<br>
transposon insertions in strains with active plasmid or<br>
chromosomal lacZ and phoA gene fusions were performed to<br>
determine the relationship of the transposon insertions<br>
to pagC. This analysis revealed that several regions of<br>
the DNA were transcribed oppositely to pagC (Fig. 7).<br>
Several TnphoA insertions that resulted in active phoA<br>
gene fusions were identified. These data indicated that<br>
pagrC-linked genes encoded membrane or secreted proteins.<br>
(Table Removed)<br>
pBR322 containing a 2.8 Kb Hpal fragment from pWP061 This worx<br>
pV/PL17 containing ^ InphoA insertion in envF Tnis work<br>
pir-dependent suicide vector (34)<br>
pGP704 containing the cloned envF::phoA gene fusion Tnis work<br>
Cosrrud clone containing the pagC region (36)<br>
a MSI (macrophage survival index) is calculated by dividing the number of surviving organisms ai 24<br>
hours post-infecccn by the number of cell associated organisms present after the 30 minute infection.<br>
D MGH, Massachusetts General Hospital. ATCC, American Type Culture Collection, FDA, Food and<br>
Drug Administration: VRI. Virus Research Institute<br>
2elden ec a.i., 1989, I n f e c t . Irroun., 5 7 : 1 - 7<br>
 Miller ec a i . , 1989, ?roc. Nad. Acad. Sci. USA, 86:5054-58<br>
 Miller ec a l . , 1988, J. Bacceriol., 170:2575-83<br>
 Pulkkinen ec a l . , 1991, J. B a c t e r i o l . , 173:86-93<br>
Genes linked to page  encode four novel proteins.<br>
To further analyze the genes defined by transposon insertions, the DNA sequence of this region was determined (Fig. 8).  DNA containing this region was cloned; 4 kb of DNA between the Hpal  site 737bp upstream of the start codon of pagC  to a ClaI site far upstream was sequenced.  The DNA sequence of the fusion junctions of all TnphoA  and MudJ gene fusions was also determined. Based on these data, the correct reading frame of each gene was determined.  The DNA sequence data revealed four ORFs predicted to be transcribed and translated based on the data derived from the TnphoA and MudJ insertions. All ORFs revealed typical ribosome binding sites 6 to 11 bases from the predicted start of translation.  The translation of the ORF immediately upstream and oppositely transcribed to pagC, pagD, indicates that a short envelope protein of 87 amino acids (unprocessed) is encoded.  It is followed by a second ORF (envE)   which encodes an envelope protein of 178 amino acids (unprocessed).  This ORF is followed by a structure that could function as a Rho-independent transcriptional terminator (see Fig. 8) .  The third ORF, msgA   (macrophage survival gene), encodes a small protein similar in size to that of the first gene product (79 amino acids) and is also followed by a structure that could function as a Rho-independent transcriptional terminator (see Fig. 8). The DNA sequence predicts that this protein is composed of several charged residues with a large number of negatively charged amino acids residing at the carboxy terminus.  The predicted protein product does not contain a structure resembling a signal sequence at its amino terminus nor any hydrophobic stretches; therefore, the third ORF is unlikely to encode an envelope protein.  The final ORF (envF)    encodes an envelope protein of 278 amino acids (unprocessed).  A computer search of known protein<br><br>
motifs revealed that EnvF contains a consensus<br>
prokaryotic membrane lipid attachment site and,<br>
therefore, is likely to be a lipoprotein (see Fig. 8 for<br>
consensus site location).<br>
The predicted proteins produced by pagD, envE, and<br>
envF contain a typical bacterial signal sequence<br>
structure. In addition, hydrophobic profiles confirmed<br>
the hydrophobic nature of the amino-termini of these<br>
proteins. The EnvE and EnvF proteins also contain<br>
hydrophobic stretches that could function as membrane<br>
spanning domains. The G+C content of the genes in this<br>
region are: page, 43.4%; pagD, 42.1%; envE, 45.9%; msgA,<br>
46.8%; and envF, 40.5%, which is considerably lower than<br>
the average G+C content of S. typhimurium (52%). A<br>
complete search of the database with the predicted<br>
protein sequences of these four ORFs showed no<br>
significant similarities. Strains containing three<br>
distinct TnphoA insertions and one MudJ insertion, each<br>
located in one of the four genes, were chosen for further<br>
characterization.<br>
A gene pagD, oppositely transcribed to page, is<br>
positively regulated by PhoP/PhoQ<br>
Representative strains with transposon insertions<br>
were examined to evaluate whether genes transcribed<br>
oppositely to pagC were increased in synthesis in the<br>
presence of PhoP. To accurately determine if these genes<br>
were PhoP regulated, it was necessary to recombine<br>
plasmid insertions onto the Salmonella chromosome. Upon<br>
replacement of the wildtype gene with the gene containing<br>
the transposon insertion, P22HTint lysates made on these<br>
strains-were transduced into a PhoP deleted (PhoP~)<br>
strain and AP or /3-galactosidase levels were monitored.<br>
One of these transposon generated gene fusions<br>
demonstrated a significant increase in activity between<br>
PhoP" and WT backgrounds, while the other insertions<br>
showed no PhoP regulation (Table 16).<br>
(Table Removed)<br>
The pagD gene is adjacent to and divergently transcribed<br>
from pagC.<br>
The representative transposon insertion in envF<br>
was unable to be recombined onto the chromosome, likely<br>
due to an insufficient amount of homologous DNA<br>
downstream of the transposon. In order to examine the<br>
possibility of PhoP regulation of the envF gene, a region<br>
upstream of this gene through and including the phoA gene<br>
of the TnphoA transposon was cloned as a 3-kb Pvul<br>
(blunt-ended)-Xhol fragment into the EcoRV-Sall sites of<br>
the suicide vector pGP704. This clone was mated into<br>
Salmonella strain CS019, and ampicillin-resistant<br>
recombinants were selected (creating a strain designated<br>
envF::pGPP2). A phoPlOS::TnlOd-Tet mutation was<br>
transduced into this strain to create an isogenic pair<br>
differing only in the ability to produce a functional<br>
PhoP protein. As shown in Table 16, the introduction of<br>
the phoPlOS::TnlOd-Tet had no effect on the AP levels of<br>
these two strains, demonstrating that envF is not a PhoPactivated<br>
gene.<br>
Transposon insertions in pagC-linked genes attenuate<br>
virulence and cause reduced survival within macrophaqes<br>
Since transposon insertions in page significantly<br>
increase the LD50 of S. typhimurium in BALB/c mice,<br>
strains containing transposon insertions linked to page<br>
were evaulated for attenuation of mouse virulence. As<br>
shown in Fig. 7, while the transposon insertion in envE<br>
had no affect on strain virulence, a TnphoA insertion in<br>
pagD and the MudJ insertion 1.8 kb downstream in msgA<br>
attenuate S. typhimurium virulence by greater than 300<br>
fold as compared to wild-type organisms (LD50
organisms). These data suggested that these two loci are<br>
essential to virulence.<br>
To examine the survival capabilities of those<br>
strains having a virulence defect, 5. typhimurium<br>
containing insertions in either pagD or msgA were used to<br>
infect bone marrow-derived macrophages. The results,<br>
shown in Table 15, demonstrate a macrophage survival<br>
defect for these two strains. The survival defect is<br>
greater for the pagD insertion (MSI=0.002) compared with<br>
the msgA insertion (MSI=0.01), and both defects are equal<br>
to or greater than that of the PhoP~strain (MSI=0.01).<br>
Transposon insertions in this gene could not be<br>
recombined onto the chromosome. Thus, it was necessary<br>
to demonstrate that the virulence and macrophage survival<br>
defects of msgA was not due to a polar effect of the MudJ<br>
insertion on envF transcription. Therefore, pGPP2 was<br>
recombined into the msgA::MudJ strain and AP activity of<br>
this strain was compared to that of CSO19 containing the<br>
recombinant pGPP2. This data (shown in Table 16)<br>
demonstrates that the transcription of the envF gene is<br>
unaffected by the msgA::MudJ insertion and is transcribed<br>
from its own promoter. However, it is possible that<br>
under different environmental conditions, other promoters<br>
may be activated that could place msgA and envF on the<br>
same transcript.<br>
Determination of the msgA and pagD transcriptional start<br>
sites<br>
The 5' regions of these genes were examined to<br>
define the transcriptional start sites of msgA and pagD.<br>
Oliogonucleotides complimentary to the 5' end of each ORF<br>
or upstream region were used in a primer extension<br>
analysis. The results of this analysis revealed that the<br>
pagD transcript begins 39 bases upstream of the<br>
translational start. The predicted -10 (TTCCAT) and -35<br>
(TTGAAT) regions were found to be similar to the known<br>
consensus sequences for E. coli promoters. The pagD<br>
transcript was detected only in PhoPc Salmonella RNA and<br>
not in PvNA from PhoP~ Salmonella. The msgA<br>
transcriptional start was found to begin 58 bases<br>
upstream of the translational start and contain predicted<br>
-10 (CAAAAC) and -35 (TTACGT) sequences. These regions<br>
do not conform well to consensus -10 and -35 sequences;<br>
however, the cDNA from this transcript was easily<br>
detected using various primers in primer extensions of<br>
both PhoP° and PhoP~ RNA and appears to produce an<br>
abundant RNA.<br>
Distribution of pagD and msgA genes in the<br>
Enterobacteriaceae and in two G+C content organisms<br>
The G+C content of the pagC chromosomal region is<br>
much lower than the average G+C content of Salmonella.<br>
The gene encoding the PhoP-regulated acid phosphatase of<br>
S. typhimurium (phoN) also has a low G+C content (39%),<br>
and DNA homologous to phoN was found only in two low G+C<br>
organisms of several genera tested. The DNAs of several<br>
members of the Enterobacteriaceae and two low G+C<br>
organisms were examined for similarity to pagD and msgA<br>
by blot hybridization. PCR fragments highly specific to<br>
each ORF were labeled and used as probes. This analysis<br>
demonstrated hybridization at high stringency to all<br>
Salmonella species examined as well as Shigella sonnei,<br>
Shigella flexneri, Klebsiella pneumonias and Citrobacter<br>
freundii. No hybridization was seen to the low G+C<br>
organisms Morganella morganii or Providencia stuartii.<br>
Identical hybridization patterns were seen with probes<br>
specific for both genes indicating that these genes are<br>
also linked in organisms other than Salmonella.<br>
A virulence gene cluster required for Salmonells<br>
typhimurium survival within macrophage macrophages<br>
Four genes upstream and oppositely transcribed to<br>
the pagC gene of Salmonella typhimurium have now been<br>
identified. Three genes (pagD, envE and envF) are<br>
predicted to be envelope proteins based on the isolation<br>
of active TnphoA insertions in these loci and the<br>
presence of a typical signal sequence at the amino-'<br>
terminus of each protein. None of the four proteins<br>
possess significant homology to any protein in the<br>
database.<br>
Only the gene immediately upstream of page and<br>
oppositely transcribed (pagD) was determined to be PhoP<br>
regulated. Transposon insertions in this gene greatly<br>
attenuate virulence and the ability of the organism to<br>
survive within murine macrophages. The transcription of<br>
several pag (including page) has been shown to be induced<br>
when Salmonella are within macrophage phagosome. In<br>
addition, analysis of proteins produced by Salmonella<br>
after infection of macrophage-derived cell lines indicate<br>
that pag products are induced and that pagC may be among<br>
the most abundant gene products induced upon macrophage<br>
infection. Since pagD is required for macrophage<br>
survival, it is likely that the transcription of this<br>
gene also will be induced within macrophage phagosomes.<br>
The pagD protein is small (87 amino acids, unprocessed)<br>
and has no strong hydrophobic domains; therefore, it is<br>
likely that it is a periplasraic or secreted protein.<br>
Transposon insertions in the msgA gene were found<br>
to have an effect on mouse virulence and macrophage<br>
survival. It is likely that this gene may also be<br>
induced within acidified macrophage phagosomes as are<br>
other genes necessary for macrophage survival. If this<br>
gene is induced by the macrophage environment, its<br>
expression (as well as other genes necessary for<br>
macrophage survival) may be controlled by a regulatory<br>
system separate from the PhoP/PhoQ system.<br>
These pagC-linked genes do not appear to form an<br>
operon. Because none of the genes downstream of pagD are<br>
PhoP regulated, they appear not be transcribed from the<br>
pagD promoter. The presence of a potential<br>
transcriptional terminator at the end of the envE gene<br>
makes it unlikely that msgA is co-transcribed with envE.<br>
The data suggest that the msgA: :Mud'J insertion is not<br>
polar on envF, which suggests that envF has its own<br>
promoter. Additionally, a potential transcriptional<br>
terminator following msgA as well as a 493 bp intergenic<br>
region makes it unlikely that these genes are cotranscribed.<br>
Primer extension analysis of these genes<br>
confirms that all four genes are transcribed from their<br>
own promoter.<br>
The other two genes identified in this region,<br>
envE and envF, appear to produce membrane proteins that<br>
contain characteristic membrane spanning regions. The<br>
envF gene product is likely to be a lipoprotein based on<br>
the presence of a consensus lipid attachment site, and is<br>
likely to play a role in Salmonella virulence.<br>
The low G+C content of the genes in the pagC<br>
region suggests that they may have been acquired by<br>
horizontal transmission. Southern blot analysis of low<br>
G+C organisms probed with the msgA or pagD genes showed<br>
no homology, but this does not eliminate the possibility<br>
that they were acquired from another low G-l-C content<br>
organism. The possibility also exists that these genes<br>
reside on a mobile genetic element acquired from another<br>
source. The msgA and pagD probes hybridized in identical<br>
patterns to some members of the Enterobacteriaceae other<br>
than Salmonella. However, the page gene has been shown<br>
to be unique to Salmonella species. This may indicate<br>
that the products of the genes upstream of page do not<br>
form a "complex with PagC or that their functions do not<br>
require PagC interaction. Alternatively, because<br>
proteins that have homology to PagC exist in other<br>
Enterobacteriaceae (in the absence of any DNA homology),<br>
a PagC homolog may be linked to msgA and pagD in other<br>
species which was not detected by the DNA hybridization<br>
experiments.<br>
pagC/pagD promoter region; expression of heterologous<br>
proteins<br>
pagC and pagD are divergently transcribed and are<br>
both PhoP activated. Other divergently transcribed,<br>
regulated genes are known in the art (Beck et al., 1988,<br>
Microbiol. Rev. 52:318-326), e.g., the Klebsiella<br>
pneumonias pulA-malX region (Chapon et al., 1985, J.<br>
Bacteriol. 164:639-645). Transcription of most of such<br>
genes require accessory proteins, such as CAP, in<br>
addition to the regulator to activate transcription.<br>
These two genes are divergently transcribed, and their<br>
promoters are arranged back-to-back. A region of 134 bp<br>
exists between transcriptional start sites of these<br>
genes, which is similar to the intergenic region between<br>
page and pagD. The pulA-malK promoter region is<br>
predicted to contain two MalT (the regulatory protein of<br>
this system) binding sites, one for each gene. Other<br>
MalT-activated genes require the CAP protein for<br>
expression, but the pulA and malX genes do not, possibly<br>
because of the high local concentration of the MalT<br>
regulator. Since the region between the transcriptional<br>
start sites of pagC and pagD (the predicted -35<br>
sequences) is only 137bp (nucleotides 562 to 776 of SEQ<br>
ID NO:15), it is likely that only PhoP binding sites<br>
exist in the intergenic region, and that binding of one<br>
or more" phosphorylated PhoP molecules positively<br>
regulates both genes. This pagC/pagD intergenic region<br>
which contains the divergent promoters can be used to<br>
construct vectors to express two heterologous proteins,<br>
one in each direction.<br>
prg genes<br>
As discussed above, phoP/phoQ constitutive<br>
mutations (phenotype PhoPc) increase the expression of<br>
pag and repress the synthesis of apprdximately 20<br>
proteins encoded by phoP-repressed genes (prg). PhoP0<br>
bacteria are attenutated for mouse virulence suggesting<br>
that prg are virulence genes.<br>
By use of the transposon, TnphoA, five unlinked<br>
prg loci were identified. In general, media conditions<br>
(starvation) that activate pag expression repress prg<br>
expression. One prg locus, prgH, was demonstrated to<br>
contribute to mouse virulence by both the oral and the<br>
intraperitoneal route. Both PrgH as well as PhoPc mutant<br>
S. typhimurium were found to be defective in induction<br>
of endocytosis by epithelial cells. Identification and<br>
mutation of such virulence genes will be useful in<br>
vaccine development.<br>
Nucleotide sequence of the prg H, prgl, prgJ, and prgK<br>
genes SEQ ID NO:10 represents the nucleotide sequence of<br>
a 5100-bp Hindlll fragment that contains the<br>
hyperinvasive hil locus. Four ORFS encoding four prg<br>
genes are located within this DNA (see Fig. 9). The ATG<br>
start codon is underlined; the asteriks indicate the<br>
positions of the prgH, prgl, prgJ, and prgK stop codons.<br>
These prg loci are required for bacterial invasion of<br>
epithelial cells, full mouse virulence, and<br>
transepithelial neutrophil migration. A bacteria<br>
attenuated by a mutation in one or more of these loci can<br>
be used to vaccinate individuals against infection by the<br>
wild type pathogen.<br>
Strains, materials and methods.<br>
All bacterial strains used in the characterization<br>
of prg genes are listed in Table 5.<br>
Strain<br>
genotype or description<br>
(Table Removed)<br>
SM10 (pRT291) contains plasmid pRT291 (Tnp^oA) derived from<br>
pRK290 selecting for Tcr and Kmr.<br>
KM294 (pPHlJI) contains Gmr plasmid pPHUI, which is<br>
incompatible with pRK290<br>
VV42(pWKSHS) contains plasmid pWKSHS, a derivative of<br>
pSClOl that contains a 5.1 kb Hindlll<br>
fragment of hil DNA including prgH<br>
V.Bajaj and C.Lee<br>
 (49)<br>
(51)<br>
(19) Kukral et al., Journal of Bacteriology, 169:1787-<br>
1793, 1987<br>
(41) .Sanderson et al., Microbiological Reviews, 52:485-<br>
532, 1988<br>
(13) Galan et al., Infection and Immunity, 59:3116-<br>
3121, 1990<br>
(2) Aliabadi et al., Journal of Bacteriology, 165:780-<br>
786, 1986<br>
(45) Stone et al., Journal of Bacteriology, 174:3945-<br>
3952, 1992<br>
Bacteria were grown as follows: Luria-Bertani (LB)<br>
broth was used as rich medium. Antibiotics were used in<br>
the following concentrations in growth media or agar:<br>
ampicillin 100 ^g/ml (Ap) , chloramphenicol 25 p.g/ml (Cm) ,<br>
gentamicin 30 ng/ml (Gm) , kanamycin 45 ng/Tnl (Km) , and<br>
tetracycline 25 /^g/ml (Tc) . The chromogenic substrate<br>
5-bromo-4-chloro-3-indolyl-phosphate (p-toluidine salt)<br>
(XP) was used to detect phosphatase activity on agar at a<br>
final concentration of 40 /jg/ml. p-nitrophenyl<br>
phosphate (p-NPP) was used as a substrate for<br>
quantitative measurement of AP activity. Media was<br>
buffered to various pH ranges with 1 M sodium citrate. E<br>
media (Vogel-Bonner minimal) was prepared as described by<br>
Davis et al., 1980, Advanced Bacterial Genetics: A<br>
Manual for Genetic Engineering. Cold Spring Harbor<br>
Laboratory, Cold Spring Harbor, N. Y. Nitrogen-, carbon-<br>
, and phosphate free medium (N~C~P~) was prepared as<br>
described by Kier et. al., 1977, J. Bacteriol. 130:399,<br>
herein incorporated by reference.<br>
This starvation medium was supplemented with 0.04%<br>
(wt/vol) glucose as the carbon source, 10 mM NH4C1 as the<br>
nitrogen source, and 1 mM NaH2PO4.H2O as the phosphate<br>
source. The carbon concentration is one log less than<br>
described by Kier et al., supra.<br>
AP activity of strains isogenic except for<br>
mutations in the phoP locus was measured in cultures<br>
grown from a single colony inoculum under various oxygen<br>
tensions with or without shaking at 37°C. Anaerobic<br>
cultures were grown in an anaerobic chamber (Coy<br>
Laboratories Products, Inc.) with a gas mixture of 80%<br>
N2, 10% O2, and 10% CO2 at 37°C. For acid regulation,<br>
aliquots of mid-logarithmic cultures were removed to<br>
measure initial pH and AP activity. 1M sodium citrate<br>
(pH &gt;6.0) or 1M citric acid (pH 4.7) were added to<br>
equivalent amounts of culture to a final concentration of<br>
50 mM citrate. Cultures were grown aerobically for two<br>
hours at 37°C and then pH and AP measurements were taken.<br>
AP activity was measured as described previously<br>
(Michaelis et al., 1983, J. Bacteriol. 154:366-374.<br>
herein incorporated by reference). AP units were<br>
calculated by the following formula: units = {OD420/[time<br>
(minutes) x volume x OD600]} x 1000 as defined by Miller<br>
for /3-galactosidase (Miller et al., 1972, Experiments in<br>
molecular genetics, p. 352-355. Cold Spring Harbor<br>
Laboratory, Cold Spring Harbor, NY.).<br>
Standard bacterial genetic techniques were used to<br>
study prg loci. Bacteriophage P22HTint-mediated<br>
transduction was performed as according to methods known<br>
in the art. TnphoA mutagenesis was performed using a<br>
broad host range plasmid (pRT291) to deliver TnphoA<br>
(Taylor et al., 1989, J. Bacteriol. 171:1870, herein<br>
incorporated by reference). Transpositions of TnphoA<br>
into Salmonella DNA were identified by use of the<br>
incompatibility plasmid pPHlJI (Taylor et al., supra).<br>
Screening for phoP-repressed genes was performed using<br>
CS031, the donor strain of the pho-24 allele. CS031 was<br>
constructed by a P22 bacteriophage transductional cross<br>
between strains AD154 and CS022 which contains the<br>
purB::TnlO allele and the pho-24 allele, respectively.<br>
The linkage of pho-24 and purB::TnlO was 70%, similar to<br>
the linkage of purB to other phoP alleles. Therefore,<br>
when P22 bacteriophage transductional crosses were<br>
performed between CS031 and the strains containing active<br>
gene fusions to phoA, strains could be screened for loss<br>
of fusion protein activity on acquisition of tetracycline<br>
resistance. Initial screening involved detection of loss<br>
of AP activity in approximately 70% of colonies that<br>
acquired tetracycline resistance, as they were presumed<br>
to contain the pho-24 allele. In addition, controls<br>
were performed using strain AD154 that contains the same<br>
purB::TnlO allele linked to a phoP null allele, phoP12.<br>
Plasmid DNA was transformed into S. typhimurium strain<br>
LB5010 by the calcium chloride and heat shock procedure<br>
(Maclachlan et al., 1985, J. Bacteriol. 161:442) .<br>
Isolation of strains with TnphoA insertions in phoPrepressed<br>
genes<br>
Constitutive mutations in the phoP locus<br>
(phenotype PhoPc) that result in increased expression of<br>
pag in an unregulated fashion also markedly attenuate S.<br>
typhimurium virulence and survival within macrophages.<br>
The virulence defect of PhoPc strains can be explained by<br>
their decreased expression of approximately 20<br>
polypeptides encoded by phoP-repressed genes (prg).<br>
A PhoP~PhoN~ strain (IB001) was constructed by a<br>
P22 transductional cross between CS019 and CS003. IB001<br>
was then mutagenized with TnphoA (so that background acid<br>
phosphatase, encoded by phoN, would not interfere with<br>
the measurement of fusion protein activity on alteration<br>
of the phoP locus) and 1800 individual blue colonies with<br>
PhoA fusion protein activity were isolated on LB agar<br>
plates containing XP. These colonies were the result of<br>
18 separate matings with approximately 20 pools in each.<br>
These strains were tested for reduction of fusion protein<br>
activity on acquisition of the pho-24 allele (CS031),<br>
which resulted in a PhoPc phenotype. AP assays were then<br>
performed on strains isogenic except for the phoP locus.<br>
The PhoPc phenotype was confirmed in these strains<br>
by preparation of whole cell protein extracts and SDSPAGE<br>
analysis. All strains with a PhoPc phenotype<br>
demonstrated the expected distinctive pattern of protein<br>
expression in PhoP0 strains, i.e. repressed protein<br>
species of specific sizes.<br>
Eight strains were identified with gene fusions to<br>
phoP-repressed genes. As shown in Table 6, the synthesis<br>
of most prg::TnphoA fusion proteins was fully repressed<br>
by the pho-24 allele. While two loci had complete<br>
repression of fusion protein activity, others<br>
demonstrated only partial repression. The expression of<br>
pag in PhoPc strains is 5-10 fold less than that observed<br>
after bacteria are phagocytosed by macrophages suggesting<br>
that the degree of repression of some prg loci may be<br>
greater when pag are maximally activated within acidified<br>
macrophage phagosomes.<br>
Lower values for prgB -phoA fusions in strains<br>
with a wildtype phoP locus (Table 7B) compared to PhoP~<br>
strains (Table 7) may represent some degree of repression<br>
in the presence of PhoP,<br>
(Table Removed)<br>
In Table 6, a comparison of the effect of phoP<br>
locus mutations on Prg-PhoA fusion protein activity is<br>
made. PhoP" indicates that the strain assayed contains<br>
the phoP12 allele (CS030) and PhoPc indicates the strain<br>
assayed contains the pho-24 allele (CS031). Values were<br>
calculated from stationary phase cultures. The numbers<br>
denote representative values of experiments performed on<br>
three separate occasions and represent activity in units<br>
of AP as defined above.<br>
(Table Removed)<br><br>
Table 7 demonstrates the effects of environmental<br>
conditions on the in vitro regulation of prg loci.<br>
Table 7A shows the effect of starvation on prg and<br>
pag expression. Starvation medium (N~"C~P~) (17) contained<br>
0.04% glucose, 10 mM NH4Cl, and 1 mM NaH2PO4.H2O. The<br>
fusion protein activity for starvation media was measured<br>
after 48 hours of growth (OD600 = 0.5) while that in rich<br>
media (LB) was measured in late-logarithmic growth (OD600<br>
= 1.0).* All cultures were grown aerobically.<br>
. Table 7B shows the effect of oxygen tension on<br>
expression of phoP-activated and phoP-repressed genes.<br>
Expression in rich medium is compared under aerobic<br>
conditions at stationary phase (OD60Q &gt; 1.4),<br>
microaerophilic (OD600 - °-8), and strict anaerobic<br>
conditions with 80% N2, 10% O2, and 10% C02 (OD500 = 0.6)<br>
after 24 hours of growth.*<br>
Table 7C shows the effect of pH on the expression of<br>
fusion protein activity of prg and pag loci. Expression<br>
was measured from cultures grown to logarithmic growth<br>
(OD600 =0.5) in LB media buffered to various pHs with<br><br>
sodium citrate. All the numbers represent activity in<br>
units of AP as defined above.<br>
Chromosomal location of prg::TnphoA loci<br>
prg::TnphoA linkage analysis was performed to a<br>
bank of strains with randomly spaced TnloAl6Al7<br>
insertions to determine chromosomal locations and<br>
whether prg::TnphoA alleles were unlinked loci. The<br>
prg::TnphoA insertions were in five distinct linkage<br>
groups. Three alleles, prgEl-3: rTnphoA were identically<br>
linked to the TnlOAl6Al7 insertion of AK3091(26%) and two<br>
other alleles, prgBl-2::TnphoA were similarly linked to<br>
the TnlOAl6Al7 insertion of AK3190 (94%), AK3249 (89%),<br>
and AK3186 (50%). Another allele, prgH1::TnphoA, was<br>
found to be 37% linked to the Tn!OAl6Al7 insertion of<br>
strain AK3304. The other two prg alleles did not<br>
demonstrate linkage to the bank of strains tested. The<br>
chromosomal- DNA of these two strains was analyzed by<br>
Southern hybridization analysis using a portion of TnphoA<br>
as a probe, and a rough physical map of the sites located<br>
adjacent to the TnphoA insertion was determined. These<br>
alleles, prgA and prgC, had different restriction<br>
endonuclease sites surrounding the TnphoA insertions. In<br>
addition, the repression of prgA and prgC fusion protein<br>
activity in strains with the pho-24 mutation was<br>
different; prgC was completely repressed, while prgA was<br>
only partially repressed indicating that these loci are<br>
different. Thus, five unlinked loci encoding envelope<br>
proteins repressed in the PhoPc phenotype were<br>
identified.<br>
Though three prg loci were identified that were<br>
linked to transposon insertions, none of the TnlOAl6Al7<br>
insertions had a known map location. The physical map<br>
location of two of these transposon insertions, AK3249<br>
and AK3304, was analyzed using Xjbal restriction<br>
endonuclease digestion and pulse field gel<br>
electrophoresis (PFGE). Since TnlOAl6Al7 contains a<br>
single Xbal site, these TnlOAl6Al7 insertions can be<br>
assigned to a specific Xbal fragment of known map<br>
location (Liu et al., 1992, J. Bacteriol. 174:16622).<br>
AK3249 was assigned to 28-32 min, while AK3304 was<br>
assigned to either end of the 58-70 minute fragment.<br>
Further P22 transduction to known markers in those<br>
regions was performed. The TnlOAl6Al7 insertion of<br>
strain AK3249 and prgBl::TnphoA were found not to be<br>
linked to the TnlO insertion of strain TN3061 (6% linked<br>
to dcp), which has a transposon insertion at 28 min, or<br>
to the ompD::Tn5 insertion of strain SH7782 at 32 min.<br>
prgHl::TnphoA was found to be very weakly linked to the<br>
sr!202::TnlO insertion of strain TT520 (
minutes. These data indicate that prg are unlinked on<br>
the Salmonella chromosome, consistent with the function<br>
of PhoP/PhoQ as global regulators.<br>
The chromosomal location of TnphoA insertions in<br>
phoP-repressed genes (prg::TnphoA) was determined by<br>
linkage analysis to a bank of strains with TnlOAl6Al7<br>
insertions (Kukral et al., 1987, J. Bacteriol. 169:1787,<br>
herein incorporated by reference). Cells with TnphoA<br>
insertions were spread on LB agar plates containing 10<br>
Mg/ml tetracycline and 40 /jg/ml XP. Then P22 lysates<br>
grown on strains with TnlOAl6Al7 insertions were spotted<br>
onto plates with a multiprong inoculator. After<br>
overnight inoculation, plates were reviewed for linkage<br>
by looking for mixed blue and white colonies. Linkage<br>
was confirmed and quantitated by carrying out individual<br>
transductional crosses between the TnlOAl6Al7 containing<br>
strains and the strain with the TnphoA insertion. After<br>
selection for the TnlOAl6Al7 encoded tetracycline<br>
resistance, strains were scored for loss of blue color<br>
and TnphoA encoded kanamycin resistance. Some TnphoA<br>
strains were found to be linked to TnlOAl6Al7 strains<br>
with no known map location. Two of these TniOAl6Al7<br>
insertions were physically mapped using PFGE following<br>
Xbal restriction endonuclease digestion. Based on<br>
physical mapping, linkage analysis to other transposon<br>
insertions by P22 bacteriophage transduction was<br>
determined as necessary.<br>
Chromosomal DNA was prepared as described by<br>
Mekalanos, 1983, Cell 3.5:253, herein incorporated by<br>
reference, using Proteinase K instead of Pronase.<br>
Purification of plasmid DNA was performed by standard<br>
methods. Restriction endonuclease digestion was<br>
performed according to the recommendations of the<br>
manufacturer (New England Biolabs). DNA, size<br>
fractionated in agarose gels, was transferred to<br>
Genescreen Plus membranes (New England Nuclear/Dupont,<br>
Boston, MA) for blot hybridization by the method of<br>
Southern well known in the art. DNA probes were purified<br>
from agarose gels by the freeze-squeeze method (Tautz et<br>
al., 1983, Anal. Biochem. 132:14) and radiolabelled with<br>
[32P]dCTP by the random primer method (Feinberg et al.,<br>
1983, Anal. Biochem. 132:6) .<br>
Cloning genes from Tnpho A fusions<br>
The gene encoding prgH has been cloned using<br>
methods described below. The plasmid, pIBOl, containing<br>
the prgH gene has been deposited with the American Type<br>
Culture Collection on July 9, 1993 (Rockville, MD) and<br>
has received ATCC designation ATCC 75496. Fig. 5 shows<br>
the partial DNA sequence of prgH (SEQ ID NO: 3). Fig. 9<br>
shows the location and sequence of the entire prgH gene.<br>
The genes described herein which have been<br>
identified by ThphoA insertion can be cloned using<br>
methods known in the art (Beattie et al., 1990, J.<br>
Bacteriol. 172:6997). Chromosomal For example, DNA of<br>
each strain containing a prg::TnphoA -gene fusion is<br>
digested with a restriction enzyme such as BamHl which<br>
cuts at a single site in TnphoA maintaining the fusion<br>
junction, phoA sequences and the neo gene. Similarly, a<br>
plasmid such as pUC19 is digested with the same enzyme.<br>
Digested chromosomal and plasmid DNA are ligated<br>
overnight at 15°C and transformed into competent E. coli.<br>
Transformations are plated on LB agar containing<br>
ampicillin and kanamycin to select for the bla gene of<br>
pUC19 and the neo gene of TnphoA. The chromosomal DNA<br>
containing the prg::TnphoA gene fusion can then be<br>
sequenced using standard methodology described above,<br>
such as the SEQUENASE® (United States Biochemical) kit.<br>
Universal primer (United States Biochemical)<br>
corresponding to DNA sequences in the plasmid or TnphoA<br>
primer (5'-AATATCGCCCTGAGCA-3') fSEO ID NO: 4)<br>
corresponding to bases 71 to 86 of TnphoA can be used as<br>
primers.<br>
To clone the wild type gene, a fragment of<br>
chromosomal DNA flanking TnphoA sequences can be used to<br>
screen a cosmid gene bank of wild type Salmonella strain<br>
ATCC 10428 using methods described above for cloning wild<br>
type pagC.<br>
Environmental regulation of prg loci<br>
Since PhoP/PhoQ are environmentally responsive<br>
regulators, the effects of different growth conditions on<br>
prg::TnphoA expression were tested. The growth rate of<br>
strains with prgr::TnphoA insertions was comparable to<br>
wild-type organisms under all conditions. The expression<br>
of all prgr loci was maximal in late logarithmic growth<br>
phase when bacteria were grown in rich (LB) media. An<br>
example of this is the comparison of values of<br>
prg/f: : TnphoA expression in Table 7A (rich media and<br>
stationary growth) and Table 7C (pH 7.0, log phase).<br>
Since the expression of pag loci was maximal in<br>
starvation (which only reaches a maxi'mal OD600 = 0.5) and<br>
stationary growth phase, this was consistent with a<br>
reciprocal relationship between the expression of pag and<br>
prg. Further analysis of prg loci expression under<br>
starvation conditions confirmed this reciprocal<br>
relationship (Table 7A). prgH expression was repressed<br>
(Table 7A) and other prg were minimally affected under<br>
starvation conditions, in contrast to the induction of<br>
pag expression when bacteria were starved (Table 7A).<br>
Because of its role in bacterial-mediated<br>
endocytosis (BME), the effect of oxygen tension in rich<br>
medium on pag and prg expression was also tested (Table<br>
7B). Different but not reciprocal regulation of pag and<br>
prg loci was found on growth at different oxygen<br>
tensions. Though pagA and pagB loci were minimally<br>
affected by growth at different oxygen tensions, the pagC<br>
virulence locus was approximately 5 fold repressed when<br>
bacteria were grown anaerobically as compared to aerobic<br>
growth (Table 7B). Variability was also noted in the<br>
expression of prg loci in response to growth conditions<br>
in the absence of oxygen. One loci, prgH, was repressed<br>
three-fold in anaerobic growth, while another locus,<br>
prgB, was induced almost 50-fold when grown anaerobically<br>
(Table 7B). Other prg loci had minimal change in fusion<br>
protein expression as a result of different oxygen<br>
tensions in the growth media.<br>
Low pH conditions also had a variable effect on<br>
prg expression (Table 7C). The expression of pagC fusion<br>
protein activity was induced under acid conditions as<br>
previously known. When bacteria were grown to midlogarithmic<br>
growth, no significant induction of the<br>
relative repression of prgH expression was noted in<br>
media of low pH, while prgB expression was induced on<br>
exposure of bacteria to low pH (Table 7C). Hence, loci<br>
maximally expressed under diverse environmental<br>
conditions can all be repressed by the PhoP° phenotype.<br>
Acid sensitivity was tested by the method of<br>
Foster et. al., 1990, J. Bacteriol. 171:771, herein<br>
incorporated by reference. Strains were grown<br>
aerobically in E media and 0.4% glucose at 37°C to an<br>
OD600 of 0.5. The pH of the bacterial culture was<br>
decreased to near 3.3 by the addition of 1 M hydrochloric<br>
acid. An aliquot was taken immediately (t0), the<br>
remainder of the culture was incubated further at 37°C<br>
with subsequent aliquots removed at 40 min (t40) and 80<br>
min (tao&gt; time-points. The pH of the cultures remained<br>
near 3.3. The aliquots were diluted 1:10 in cold PBS,<br>
washed and resuspended in normal saline prior to plating<br>
serial dilutions for colony forming units.<br>
prgH is a virulence locus for S. typhimurium<br>
Since the PhoPc phenotype resulted in virulence<br>
attenuation and repressed the synthesis of approximately<br>
20 proteins, the virulence of strains with single<br>
mutations in prg loci was tested (Table 8). Strains with<br>
prg::TnphoA insertions were screened for virulence<br>
defects by intraperitoneal injection of approximately 150<br>
organisms into BALB/c mice. Controls were also performed<br>
with wild-type bacteria. A significantly longer time<br>
course of clinical disease progression was observed with<br>
a prg mutant strain compared to wild type bacteria. Mice<br>
injected intraperitoneally with strains containing the<br>
prgHl::TnphoA insertion developed clinical signs of<br>
typhoid fever, such as a "scruffy "phenotype (fever and<br>
piloerection) and hepatosplenomegaly in approximately 10-<br>
14 days, compared to approximately 24 hours for the wild<br>
type bacteria. Despite the extended time course of<br>
disease development, all the mice eventually died.<br>
Disease progression of mice injected with other strains<br>
containing prg: \1r\phoA insertions showed a similar<br>
pattern of illness to that of wild type bacteria.<br>
Table 8<br>
(Table Removed)<br><br>
Table 8 shows the effect of the prgHl::TnphoA<br>
mutation on Salmonella mouse virulence. Strains were<br>
isogenic and administered by intraperitoneal injection<br>
and oral inoculation in 35 day old BALB/c mice. The<br>
number of animals used at bacterial dilutions near the<br>
LD50 for each allele is listed in parentheses. The LD50<br>
determinations were repeated on three separate occasions.<br>
Further testing of the LD50 of strains containing<br>
prgH mutations was performed. pzgH mutants were<br>
determined to have an LD50 of approximately 60 organisms<br>
compared to a value or 
to the difficulty in accurately delivering organisms in<br>
small doses to mice, a strain with a mutation in both<br>
prgH and phoP was constructed. The PrgH~PhoP~ strain had<br>
greater than a 10 fold increase in LD50 compared to CS015,<br>
an isogenic PhoP" strain (Table 8). The combined effect<br>
of the two mutations further documented that the<br>
prgHl: rTnphoA mutation attenuated S. typhimurium<br>
virulence and indicated that mutations which affected two<br>
phases of PhoP/PhoQ regulated gene expression were<br>
additive in their effect on virulence. Strains with<br>
prgHl::TnphoA insertions were also tested for virulence<br>
when administered by the oral route. A 10 fold decrease<br>
in virulence (increase in LD50) was observed (Table 8).<br>
Further analysis of the efficiency of strains with<br>
prgHl::TnphoA insertions in crossing the mucosal barrier<br>
was tested by competition experiments with wild-type<br>
bacteria. During the first 72 hours after oral<br>
inoculation with mutant bacteria, no prgH1::TnphoA<br>
mutants were recovered from the bloodstream of mice<br>
compared to control experiments in which organisms were<br>
routinely isolated from the blood of mice inoculated with<br>
wild type bacteria. Other strains with prg mutations<br>
were also tested for virulence defects by the oral route,<br>
but no significant change in virulence was observed.<br>
Mouse virulence studies were carried out as<br>
follows. Bacteria were grown aerobically at 37°C to<br>
stationary phase, washed with LB, and diluted in normal<br>
saline. 35 days old (16-18g) female BALB/c mice were<br>
purchased from the Charles River Breeding Laboratories,<br>
Inc. (Wilmington, MA). Diluted bacterial samples in<br>
saline were injected intraperitoneally with an inoculum<br>
of 0.1-0.15 ml. Bacteria were administered orally as a<br>
0.5 ml bolus to mice fasted for 2 hours, via a 2 inch<br>
straight, 18 gauge stainless steel animal oral feeding<br>
needle (Harvard Apparatus, Inc., South Natick, MA) under<br>
mild 2-bromo-2-chloro-l,1,1-trifluoroethane (Halothane)<br>
anesthesia. The number of organisms administered was<br>
guantitated by plating for cfu/ml on LB agar. Mouse 50%<br>
lethal dose (LD50) values were determined by standard<br>
methods (Reed and Muench, 1938, Amer. J. Hygiene 27:493).<br>
The LD50 determinations were repeated on three separate<br>
occasions. Competition assays were performed after<br>
bacteria were administered orally to mice as above.<br>
Bacteremia was assessed on days 1-4 from tail bleeds or<br>
intracardiac punctures with 50 fil of blood plated<br>
immediately and after growth in LB broth at 37°C<br>
overnight. Spleen and intestinal harvests were performed<br>
on days 1-6 with organs homogenized in 3 mis of 0.9%<br>
sodium chloride. Samples and cultures were plated in<br>
serial dilutions. S. typhimurium was confirmed by<br>
characteristic growth (black colonies) on Hektoen-enteric<br>
agar (Difco Laboratories) and by the macroscopic slide<br>
agglutination test with Salmonella rabbit serum Group B<br>
(Antigens 4, 5, 12) (Fisher Scientific).<br>
Mutations in oxygen-induced genes do not affect mouse<br>
virulence<br>
Both prgH and pagC loci were shown to be repressed<br>
by anaerobic growth and required for full virulence, thus<br>
suggesting that a shift from anaerobic to aerobic<br>
conditions might serve as a general signal for induction<br>
of virulence genes. Strains with mutations in oxygeninducible<br>
loci (Aliabadi et al., 1986, J. Bacteriol.<br>
165:780) were constructed. ATCC14028S derivatives with<br>
oxiA, oxiC, and oxiE mutations were made (termed CS032,<br>
CS033, CS034, respectively). These strains were as<br>
virulent as wild type bacteria. Though these gene<br>
fusions could still mark operons containing virulence<br>
genes, this data suggests that these loci are not<br>
essential to full virulence and that oxygen induction is<br>
mutants have normal survival within macrophages<br>
Since the PhoPc phenotype resulted in a defect in<br>
bacterial survival within macrophages, the effect of this<br>
mutation on the synthesis of a prgtf-encoded protein was<br>
tested. A strain with the prgHl::TnphoA insertion was<br>
tested for intracellular survival within bone marrowderived<br>
macrophages from BALB/c mice and J774.2 cells, a<br>
macrophage derived cell line. No defect in intracellular<br>
survival was observed. A strain with a prgBl::TnphoA<br>
insertion was also tested and found not to have a defect<br>
in survival within macrophages.<br>
Assays to determine bacterial survival within<br>
macrophages were performed as described by Buchmeier al.,<br>
1989, Infect. Immun. 52:1, herein incorporated by<br>
reference. Bacteria grown to stationary-phase were<br>
opsonized for 30 minutes in normal mouse serum before<br>
exposure to cultured bone marrow-derived macrophages<br>
harvested from BALB/c mice. One hour after infection,<br>
gentamicin 10 /ig/ml was added to kill extracellular<br>
bacteria. All time points (1, 4, and 24 hr) were done in<br>
triplicate and repeated on three separate occasions.<br>
Cultured bone marrow macrophages were harvested<br>
from BALB/c mice purchased from the Charles River<br>
Breeding Laboratories. J774.2 macrophages were cultured<br>
in Dulbecco's mirtimal essential medium with 10% fetal<br>
bovine serum (DMEM/10%FBS).<br>
prg::TnphoA insertions do not suppress the phenotypes of<br>
PhoP mutants<br>
Several phenotypes of phoP mutants, including<br>
defensin and acid sensitivity as well as mouse virulence<br>
attenuation, were tested for suppression on addition of a<br>
prgr: : TnphoA mutation. To test the eibility of a phoP<br>
mutation to suppress the synthesis of prg products, PhoP<br>
mutant strains isogenic except for prg: :TnphoA mutations<br>
were constructed and tested for mouse virulence, where<br>
suppression would involve an increase in virulence, or<br>
decreased acid and defensin sensitivity. prg::TnphoA<br>
insertions had no effect on the virulence phenotypes of<br>
PhoP~ bacteria. These results indicate that the<br>
prg::TnphoA mutations tested did not suppress the PhoP<br>
null phenotype as single mutations.<br>
PrgH and PhoPc mutants are defective in bacterialmediated<br>
endocytosis by cultured epithelial cells<br>
The BME of prg::TnphoA and PhoPc S. typhimurium<br>
strains was tested. The following observations<br>
(described herein) suggested that prg genes may be<br>
involved in bacterial-mediated uptake by eucaryotic<br>
cells: prgHl::TnphoA was shown to be located at 59' on<br>
the bacterial chromosome, a location where other genes<br>
essential to invasion are clustered; prgH mutants were<br>
shown to be defective in competition with wild type<br>
organisms on reaching the bloodstream of mice in the<br>
first 72 hours after oral ingestion; and the expression<br>
of one prg locus, prgB, was dramatically induced under<br>
anaerobic growth conditions. Strains with prgH and pho-<br>
24 mutations had a significant reduction (p-value 
in their ability to induce uptake by Madin-Darby canine<br>
kidney (MDCK) polarized epithelial cells compared to<br>
wild-type bacteria. Other prg strains with TnphoA<br>
insertions did not demonstrate a statistically<br>
significant defect in BME by epithelial cells (Table 9).<br>
The adherence of strains defective in BME was unaffected<br>
by the prgH::TnphoA insertion when determined by cellassociated<br>
cfu/ml before the administration of gentamicin<br>
(Table 9) and by microscopy.<br>
To assay bacterial adherence and uptake of<br>
bacteria by epithelial cells, bacterial strains were<br>
grown at 37°C without shaking (microaerophilie) to a<br>
final density of approximately 2xl08 colony forming units<br>
(cfu)/ml. Assays were performed by seeding 105 MDCK<br>
cells/well in 24-multiwell tissue culture plates. Cells<br>
were incubated overnight at 37°C in 5% CO2/95% air<br>
atmosphere in DMEM/10%FBS without antibiotics until &gt;80%<br>
confluent. The adherence and invasion assays were<br>
carried out according to the protocol of Lee and Falkow,<br>
1990, Proc. Natl. Acad. Sci. USA ET7:4304, herein<br>
incorporated by reference.<br>
(Table Removed)<br>
In Table 9, the effect of prg::TnphoA insertions<br>
on Salmonella-mediated endocytosis by MDCK epithelial<br>
cells is shown. Microaerophilically grown bacterial<br>
strains were assessed for changes in adherence and<br>
invasion. Adherence was determined as the percentage of<br>
bacteria adhered to the cells after centrifugation and 30<br>
minute 4°C incubation/ total number of bacteria added to<br>
each well. Invasion was determined as the percentage of<br>
bacteria that had invaded after a two hour incubation<br>
with gentamicin/ total number of bacteria added to each<br>
well. There was no difference between S. typhimurium<br>
wildtype and S. enteritidis CDC5 wildtype strains with<br>
respect to adherence and invasion freguency. The<br>
asterisk (*) represents statistical significance by<br>
variance analysis of the invasion data done in triplicate<br>
compared to wild-type (p-value 
The confluent MDCK monolayers were washed three<br>
times with PBS, then 0.9 ml of cold DMEM/10%FBS was added<br>
to each well. Bacteria were washed in LB and resuspended<br>
in an eguivalent volume of DMEM/10%FBS. Approximately<br>
5xl07 bacteria were added/well. The plates were spun at<br>
500 rpm at 4°C for 10 minutes, then incubated at 4°C for<br>
30 minutes. Adherent bacteria were recovered by washing<br>
the plates three times with phosphate-buffered saline<br>
(PBS), lysing the epithelial cells in 0.5 ml of 1%<br>
Triton-X-100/PBS, and plating for cfu/ml on LB agar. A<br>
morphologic assessment of adherence was also performed by<br>
staining bacterially infected epithelial cell monolayers<br>
grown overnight on coverslips for 7 minutes in 1 /ig/ml 4'<br>
6-diamidino-2-phenylindole (DAPI). These DAPI stained<br>
coverslips were examined by both fluorescent and phase<br>
contrast microscopy using a Leitz Laborlux 12 microscope.<br>
Invasion or bacterial-mediated endocytosis (BME)<br>
was assessed by allowing bacteria to adhere as described<br>
above. Plates containing bacteria and epithelial cells<br>
were incubated for two hours at 37°C in a 5% CO2/95% air<br>
atmosphere. Each well was washed three times with PBS to<br>
remove bacteria not associated with cells. DMEM/10%FBS<br>
supplemented with 10 /ig/ml gentamicin was then added to<br>
kill extracellular bacteria. After 90 minutes of<br>
incubation, the cell monolayers were washed three times<br>
with PBS and the viable intracellular bacteria were<br>
released by vigorously pipetting with 0.5 ml of 1% Triton<br>
X-100/PBS. An invasion deficient Salmonella enteritidis<br>
mutant and an invasive clinical wild-type isolate of S.<br>
enteritidis were used as controls for BME. Viable<br>
bacteria were quantitated by plating for cfu/ml on LB<br>
agar medium. All assays were done in triplicate and<br>
repeated at least three times.<br>
MDCK epithelial cells were used between passage<br>
40-58 to maximize bacterial adherence and invasion.<br>
Epithelial cell lines were cultured in DMEM/10% FBS and<br>
1% penicillin/streptomycin solution at 37°C in a 5% C02<br>
atmosphere.<br>
To assay bacterial defensin sensitivity, NP-1<br>
defensin was purified from rabbit peritoneal neutrophils<br>
according to methods known in the art (Selsted et al.,<br>
1985, J. Biol. Chem. 260:4579 ; Selsted et al., 1984,<br>
Infect. Iromun. 4.5:655). Typically, 105 bacteria in 0.5%<br>
tryptone in 100 /nl volume were exposed to 50-100 /ig of<br>
defensin/ml at 37°C for 2 hours. The reactions were<br>
stopped by diluting the reaction in 0.9% NaCl.<br>
Appropriate dilutions were plated to determine the cfu/ml<br>
of surviving bacteria. Assays were performed in<br>
duplicate at least twice for each strain. Appropriate<br>
assays with sensitive (PhoP~) and resistant (wild-type)<br>
strains were performed as controls.<br>
Mapping of prgH<br>
The location of prgH relative to other invasion<br>
loci at 59 minutes was determined using linkage analysis.<br>
P22 transduction linkage analysis indicated that the<br>
TnlOAl6Al7 of strain AK3304 had similar linkage to invA<br>
(40%) and prgH- (37%) ; however, invA was not linked to<br>
sorbital. The prgHl::TnphoA insertion was found to be<br>
linked (99.6%) to the transposon insertion of EE517, a<br>
strain with a 8.5 kilobase deletion adjacent to the Tn5B50-<br>
378 insertion of hil.<br>
A physical map of the restriction endonuclease<br>
sites surrounding the TnphoA insertion of strain IB037<br>
was made (Fig. 4) revealing no similarities to the known<br>
restriction endonuclease map of the invA-E region.<br>
Plasmids containing the cloned inv and hil DNA were then<br>
used as probes in Southern hybridization analysis of<br>
chromosomal DNA from wild type ATCC10428s and IB040<br>
bacteria containing the prgHl::TnphoA insertion. When a<br>
plasmid which contains other invasion loci highly linked<br>
to invA-E (invH, invF, and part of invG) was used as a<br>
probe, no differences in hybridization pattern was found<br>
between wild type bacteria and strain IB040 indicating<br>
that prgH was not located within the inv region.<br>
However, when a plasmid containing a 5 kb region<br>
immediately downstream of the Tn5B50-380 insertion of<br>
hil was used as a probe, the prgHl::TnphoA insertion was<br>
demonstrated to be located within this region. By use of<br>
the known restriction map of the hil locus (Lee et al.,<br>
1992, Proc. Natl. Acad. Sci. USA £9:1847) and the known<br>
restriction endonuclease sites of TnphoA, the physical<br>
map of this area and the relationship of prgHl::TnphoA<br>
within it were further defined (Fig. 4). The<br>
prgHl::TnphoA insertion was oriented so that the<br>
direction of transcription of the phoA fusion protein was<br>
opposite to that of the Tn5B50 insertions that confer the<br>
hil phenotype and contain a constitutive neomycin<br>
promoter that is transcribed out of the transposon (Fig.<br>
4). Althought prgH was found to be located within the<br>
hil locus, this gene is unique in that it is oppositely<br>
transcribed and unlike any other genes identified within<br>
the hil locus, prgH is regulated by the phoP regulon.<br>
Since it was possible that a protein whose<br>
expression was altered by the Tn5B50-380 insertion might<br>
alter the expression of prgH, strains containing both<br>
insertions were constructed and the prgH-phoA fusion<br>
protein activity compared under different environmental<br>
conditions. When bacteria were starved or grown<br>
anaerobically, derepression of fusion protein activity<br>
was observed. Table 11 shows the effect of the TnSBSO-<br>
380 insertion on expression of prgH fusion protein<br>
activity.<br>
(Table Removed)<br>
This data demonstrates that the Tn5B50-380<br>
insertion increased prgH expression, even though prgH<br>
transcription was opposite to that of the Tn5B50-380<br>
encoded neomycin promoter. Starvation (repressing<br>
conditions for prg) indicates that bacteria were grown<br>
aerobically for 48 hours in starvation medium (N~C~P~)<br>
containing 0.04% glucose, 10 mM NH4C1, and 1 mM<br>
NaH2PO4.H2O. LB (aerobic) indicates that bacteria were<br>
grown in Luria-Bertani broth (rich media) to late<br>
logarithmic growth (nonrepressing conditions) (OD600 &gt;1.0<br>
). LB (anaerobic) indicates that bacteria were grown<br>
under strict anaerobic conditions for 24 hours (OD600 =<br>
0.6). All the numbers represent activity in units of AP<br>
as described above.<br>
To rule out the possibility that the BME defect of<br>
the prgH mutant was an artifact of the PhoA fusion<br>
protein produced, complementation analysis was performed<br>
with a plasmid (pWKSHS) containing a 5.1 kb Hindlll<br>
fragment which included the hil and prgH loci. The<br>
plasmid was crossed into PrgH (IB040) and PhoPc (CS022)<br>
mutant bacteria to create strains IB043 and IB044,<br>
respectively. The BME phenotype of the PrgH mutant was<br>
similar to wild-type with the same plasmid insertion.<br>
The BME phenotype of the PhoPc mutant was not complemented<br>
by this plasmid. These results indicate that a gene<br>
product altered in synthesis as a result of the<br>
prgH::TnphoA insertion was necessary for BME.<br>
Using a strain with a phoP/phoQ locus mutation<br>
that constitutively simulates the environmental<br>
activation of pag (phenotype PhoP°), five unique phoPreprossed<br>
loci encoding envelope proteins were defined.<br>
phoP-repressed genes (prg) were found to be widely spaced<br>
*<br>
on the chromosome and the expression of prg loci was<br>
repressed under starvation conditions, when pag loci<br>
were induced (Table 10).<br>
Table 10<br>
(Table Removed)<br><br>
PrgH was shown to lie between two Tn5B50<br>
insertions that confer the hil phenotype. Since deletion<br>
mutants in this region have been demonstrated to also<br><br>
have defects of BME, and the BME defect of prgH mutants<br>
can be complemented with a plasmid containing this locus,<br>
it is possible that a protein not synthesized as a result<br>
of the prgHl::TnphoA insertion promotes BME (Fig. 4).<br>
Contrary to the expectation that genes essential<br>
to the hil phenotype would be induced under<br>
microaerophilic conditions similar to what was found for<br>
prgB, prgH expression was maximal during aerobic growth<br>
and the Tn5B50-380 insertion, which results in a hil<br>
phenotype, derepressed expression of prgH. In addition,<br>
the direction of transcription predicted by the<br>
prgH1::TnphoA insertion is opposite to that of the<br>
Tn5B50-380 encoded neomycin promoter associated with the<br>
hil phenotype suggesting that a regulatory protein<br>
interrupted by or transcribed from the Tn5B50-380<br>
insertion affects the expression of prgH.<br>
In view of the observation that pWKSHS, a plasmid<br>
containing prgH (hil), did not complement PhoPc bacteria<br>
for BME, it is possible that other invasion genes may<br>
also be regulated by PhoP/PhoQ. If prgH was expressed<br>
from pWKSH5, despite the presence of the pho-24 mutation,<br>
this suggest that other genes repressed as part of the<br>
PhoPc phenotype are necessary for BME.<br>
The identification and characterization of prgH<br>
has shown that PhoP/PhoQ oppositely regulate factors<br>
necessary for bacteria to enter or to survive within<br>
mammalian cells, further documenting the importance of<br>
gene regulation to bacterial virulence. The<br>
identification of prg loci can be used to study the<br>
regulation of bacterial genes after infection of<br>
mammalian cells. Understanding the regulation of<br>
virulence genes, such as prgH can also be used to<br>
attenuated pathogenic bacteria for the development of new<br>
live vaccines for typhoid fever.<br>
Role of prg genes in virulence<br>
The prg locus, prgH, was found to contribute to<br>
mouse virulence when S. typhimurium was administered by<br>
both the oral and intraperitoneal routes. PrgH as well<br>
as PhoPc mutants were further found to be defective in<br>
bacterial-mediated uptake by epithelial cells suggesting<br>
that an inability to cross epithelial barriers might<br>
contribute to the attenuation of virulence observed.<br>
Competition studies in mice after oral ingestion of<br>
bacteria further supported that prgH mutants were<br>
defective in transcytosis across the intestinal<br>
epithelial barrier. Therefore, at least two phases of<br>
PhoP/PhoQ regulated protein expression essential to<br>
bacterial virulence have been defined. In one phase, prg<br>
expression promotes bacterial mediated endocytosis by<br>
epithelial cells (Table 10), while in another phase, pag<br>
expression promotes survival within macrophages.<br>
Systemic pathogens, such as Salmonella, may<br>
encounter more complex and varied environments than may<br>
be encountered by mucosal pathogens. The achievement of<br>
intermediate states of pag and prg expression could be<br>
essential to virulence at some stage of the infectious<br>
cycle. Consistent with this concept was the lack of<br>
uniformity observed in the expression of pag and prg on<br>
growth at different oxygen tensions and pH conditions.<br>
These data may also indicate that not all regulation of<br>
pag and prg is mediated directly through PhoP and PhoQ.<br>
Given the function of PhoP as a transcriptional<br>
regulator, it is likely that prg loci repression occurs<br>
at the level of transcription.<br>
The approach of defining genes repressed by the<br>
pho-24 mutation has led to the discovery of at least one<br>
virulence locus, prgH, which can be mutated to attentuate<br>
the bacteria for vaccine purposes.<br>
Attenuation of Bacterial Virulence by Constitutive<br>
Expression of Two-component Regulatory Systems<br>
The virulence of a bacterium can be attenuated by<br>
inducing a mutation which results in the constitutive<br>
expression of genes under the control of a two-component<br>
regulatory system or by inducing a mutation that<br>
inactivates a gene under the control of the two-component<br>
systems. A balance between the expression of the genes<br>
under the control of the two-component system, e.g.,<br>
between pag and prg gene expression, and possibly between<br>
two-component system regulated genes and other genes, is<br>
necessary for full virulence. Mutations that disrupt<br>
this balance, e.g., mutations that cause the constitutive<br>
expression of a gene under the control of the twocomponent<br>
system, or a mutation that inactivates a gene<br>
under the control of the two-component system, e.g., the<br>
pag gene, reduce virulence.<br>
Constitutive mutations in two-component<br>
regulators can be identified by the use of a strain<br>
containing a recorder gene fusion to a gene regulated by<br>
the two-component system. Such gene fusions would most<br>
typically include DNA encoding the lacZ gene or AP fused<br>
to a gene under the control of the two-component system.<br>
Strains containing fusions that are (as compared to wild<br>
type or parental strains) highly expressed in an<br>
unregulated fashion, i.e., constitutive, can be detected<br>
by increased color on chromogenic substrates for the<br>
enzymes. To detect constitutive mutations a cloned<br>
virulence regulator could be mutagenized e.g., by passage<br>
through an E. coli strain defective in DNA repair or by<br>
chemical mutagenesis. The rautated DNA for the regulator<br>
would then be transferred to the strain containing the<br>
gene fusion and constitutive mutations identified by the<br>
high gene fusion expression (blue color in the case of a<br>
lacZ fusion grown on media containing X-gal).<br>
Constitutive mutations in a component of a two-component<br>
regulatory system could also be made by in vitro<br>
mutagenesis after other constitutive mutations have been<br>
sequenced and a specific amino acid change responsible<br>
for the constitutive phenotype identified. Putting<br>
several amino acid changes that all result in a PhoP<br>
constitutive phenotype would result in a decreased<br>
frequency of reversion by spontaneous base changes. A<br>
constitutive mutation could also be constructed by<br>
deletion of the portion of the amino terminus of the<br>
phospho-accepting regulator which contains the<br>
phosphoacceptor domain e.g., deletion of sequences<br>
encoding amino acids amino terminal to amino acid 119 in<br>
the phoP gene or deletion of analogous phospho accepting<br>
sequences in genes of other two-component regulatory<br>
systems. This could result in a conformational change<br>
similar to that induced by phosphorylation and result in<br>
increased DNA binding and transcriptional activation.<br>
Attenuation of virulence: d_el_ejti_on in the phoP/phoQ<br>
regulon As discussed above, the PhoP regulon is<br>
essential to full virulence of Salmonella. This regulon<br>
is composed of two genes, PhoP and PhoQ located in an<br>
operon, and over 40 genes they positively and negatively<br>
regulate (pag and prg, respectively).<br>
PhoP null S. typhimurium mutants have been<br>
demonstrated to be markedly attenuated and also effective<br>
vaccine strains when studied in the BALB/c mouse model of<br>
typhoid fever. This phenotype is likely the result of<br>
multiple, phoP-activated virulence genes, as transposon<br>
insertions in multiple different phoP-activated genes<br>
have been independently demonstrated to decrease S.<br>
typhimurium virulence. S. typhimurium mutants deleted<br>
for genes essential to eiromatic amino acids (aroA null or<br>
aroC/aroD null mutants) are also markedly attenuated in<br>
the mouse model. However, testing of aroC/aroD mutants<br>
in humans has shown that although these strains are<br>
immunogenic, bacteremias and side effects such as fever<br>
have been noted at doses as low as 105 to 107 organisms<br>
administered as a single oral dose (Hone et al., J. Clin.<br>
Invest. 90:412-420). It has now been found<br>
that a large deletion in a global regulator of Salmonella<br>
virulence, i.e., the PhoP/PhoQ operon, significantly<br>
decreases the virulence of the bacteria. This mutation,<br>
the result of a 1 kB deletion of DNA within the phoP/phoQ<br>
locus, was initially made in S. typhimurium and<br>
subsequently transferred via homologous recombination to<br>
S. typhi. In order to confer an even greater margin of<br>
safety in construction of this vaccine, it was created in<br>
a strain background deleted for genes essential to<br>
aromatic amino acids and carrying the histidine G46<br>
mutation, a mutation rendering the organism auxotrophic<br>
for histidine. The resulting strain, S. typhi TyLH445,<br>
offers several advantages over existing vaccine<br>
candidates, most, notably, imnunogenicity without<br>
transient bactermia .<br>
Use<br>
The Salmonella cells of the invention are useful<br>
as sources of immunological protection against diseases,<br>
e.g., typhoid fever and related diseases, in an animal,<br>
e.g., a mammal, e.g., a human, in particular as the basis<br>
of a live-cell vaccine capable of colonizing the<br>
inoculated animal's intestine and provoking a strong<br>
immune reaction. Appropriate dosages and conditions of<br>
administration of such a live, attenuated vaccine are<br>
known in the art, e.g., as described in Holem et al.,<br>
Acute Enteric Infections in Children, New Prospects for<br>
Treatment and Prevention, (1981) Elsevier/North-Holland<br>
biomedical Press, Ch. 26, pp. 443 et seg. (Levine et<br>
al.), hereby incorporated by reference, and are described<br>
in the examples below.<br>
Advantages<br>
One advantage of the invention is that the<br>
bacterial cells are attenuated as a result of a<br>
mutation(s), i.e., the phoP/phoQ operon, that directly<br>
affect a virulence pathway. Another advantage is that<br>
the bacterial cells have mutations in two completely<br>
different attenuating genes, i.e., the aromatic amino<br>
acid synthesis pathway (Aro), and in an operon important<br>
to Salmonella virulence (PhoP/Q). As a result, the<br>
bacteria appear to be extremely attenuated; doses as high<br>
as 1 x io9 cfu appear to be very safe. Other vaccines<br>
under development, such as CVD 908, have caused some<br>
systemic symptoms, e.g., fever or bacteremia, at doses as<br>
low as 1 x io7 cfu.<br>
In addition to the phoP/phoQ deletion and the<br>
AroA-mutation, the bacterial cells of the invention may<br>
also contain a histidine mutation to further alternate<br>
virulence, although absence of the histidine mutation may<br>
improve immunogenicity. The bacterial cells of the<br>
invention are the most promising vaccine candidates to<br>
date because they are strongly immunogenic and safe,<br>
i.e., extremely attenuated.<br>
Construction of vaccine strain<br>
The bacterial cells of the invention were made by<br>
deleting approximately 1 kb of DNA in the phoP/phoQ<br>
regulon.<br>
PhoP/phoQ deleted suicide vectors were constructed<br>
using methods known in the art. A DNA fragment<br>
containing the phoP/phoQ locus was obtained by PCR using<br>
wild type S. typhimurium chromosomal DNA as a template.<br>
PCR primers flanking the phoP/phoQ locus were engineered<br>
to contain terminal restriction enzyme recognition sites,<br>
e.g., recognition site for EcoRI, to facilitate<br>
subsequent cloning. Following amplification, the PCR<br>
product was digested with £coRI and cloned into the EcoRI<br>
site in the polylinker of a high copy vector. The<br>
plasmid containing the phoP/phoQ DNA fragment was named<br>
pLH356.<br>
Sequence analysis and restriction mapping of the<br>
phoP/phoQ locus revealed four Hpal sites within the<br>
locus; no Hpal sites were found in the vector. To create<br>
an internal deletion within the phoP/phoQ locus, pLH356<br>
DNA was cut to completion with Hpal, and religated, to<br>
yield with an internal deletion from nucleotides 376-1322<br>
(pLH418). This deletion was confirmed by restriction<br>
digestion of the plasmid.<br>
A DNA fragment containing the internally deleted<br>
phoP/phoQ locus was excised from pLH418 using the<br>
SacI/SphI restriction sites within the polylinker region<br>
of the vector. This fragment was cloned into compatible<br>
sites in the plasmid CVD442, which carries the sacB gene<br>
to allow positive selection for allelic exchange. The<br>
resulting suicide vector was called pLH423.<br>
pLH423 was transformed into E. coli lambda pir<br>
SY327, and subsequently into E. coli lambda pir SM10<br>
(strain LH425). E. coli strain LH425 was mated with<br>
S. typhimurium strain CS019. Single recombinants<br>
carrying plasmid sequences integrated onto the<br>
S. typhimurium chromosome were selected by plating on<br>
agar containing ampicillin and chloramphenicol (Strain<br>
LH428) .- These strains were confirmed to be ampicillin<br>
resistant and sucrose sensitive, i.e., death on 20%<br>
sucrose plates containing no NaCl when incubated at 30°C.<br>
These data confirm the integration of plasmid sequences<br>
into the Salmonella chromosome.<br>
A P22 bacteriophage lysate was made from strain<br>
LH428; phage particles were concentrated 20x by high<br>
speed centrifugation and transduced into S. typhi strain<br>
522Ty2 (a strain with a deletion in the aroA gene, and<br>
the G646 mutation which renders the organism auxotrophic<br>
for histidine). Single recombinant S. typhi organisms<br>
were selected by plating on LB plates supplemented with<br>
aromatic amino acids, cystine, histidine, and ampicillin<br>
(strain LH435).<br>
Strain LH453 was grown with aromatic amino acids,<br>
cystine, and histidine (but without ampicillin) to mid<br>
logarithmic growth phase. Serial dilutions were plated<br>
on LB 20% sucrose plates lacking NaCl, and on LB plates<br>
lacking NaCl. The number of bacteria that grew on plates<br>
without sucrose was greater than the number that grew on<br>
sucrose-supplemented plates by a factor of three logs.<br>
These data suggest that many colonies lost plasmid<br>
sequences containing the sacB gene.<br>
Multiple colonies from the sucrose selection were<br>
picked and confirmed to be ampicillin sensitive and<br>
sucrose resistant. Chromosomal DNA from approximately 10<br>
colonies was purified and subjected to Southern blot<br>
analysis, utilizing the 2.3 kb fragment of wild type<br>
phoP/phoQ as a probe.<br>
Southern blotting revealed the loss of two Hpal<br>
sites and an XmnI site known to be within the 1 kb<br>
deleted fragment of phoP/phoQ in several strains. One of<br>
these strains was designated TyLH445.<br>
In vitro evaluation of TyLH445<br>
-TyLH445 was extensively characterized in vitro<br>
using standard clinical microbiological tests. The<br>
nutritional requirements of TyLH445 were evaluated.<br>
TyLH445 did not grow on M-9 plates unless supplemented<br>
with aromatic amino acid mix, cystine (S. typhi grows<br>
better with cystine), and histidine. These data<br>
confirmed that TyLH445 was AroA-, His-.<br>
TyLH445 was found to agglutinate with polyclonal<br>
serum against Salmonella and polyclonal serum against S.<br>
typhi Vi antigen. Group D agglutination was found to be<br><br>
variable, perhaps due to excess Vi antigen. TyLH445 was<br>
also found to be indole negative (as are all<br>
Salmonellae), and to produce very little hydrogen sulfide<br>
(as do many S. typhi). Biochemical testing utilizing<br>
both the VITEK system as well as the BBL Crystal Enteric<br>
organism identification system was also carried out.<br>
These data indicated that the TyLH445 strain was S.<br>
typhi.<br>
Growth characteristics of TyLH445 were also<br>
evaluated. TyLH445 was found to grow just as quickly as<br>
its parent, 522Ty2, (phoP/phoQ locus intact). Growth in<br>
vitro was measured in aromatic amino<br>
acid/histidine/cystine-supplemented Luria broth at 37°C.<br>
Growth curves of the parent and vaccine strain were found<br>
to be essentially identical (see Fig. 10).<br>
Standardized clinical testing methods were use to<br>
determine antibiotic sensitivity. TyLH445 and the parent<br>
strain, 522Ty2, were found to be sensitive to ampicillin,<br>
trimethoprim-sulfamethoxazole, ciprofloxacin,<br>
aminoglycosides, and third generation cephalosporins. No<br>
difference in zone sizes was detected between the parent<br>
and vaccine strains, suggesting that no other antibiotic<br>
resistance mechanisms, e.g., modification of antibiotic<br>
transport systems, or modification of the cell wall of<br>
the bacterium, were affected by introduction of the<br>
mutated phoP/phoQ locus into S. typhi.<br>
The phoP/phoQ Hpal deletion mutants were tested<br>
for defensin sensitivity, a phenotype of PhoP null<br>
mutants. Defensin sensitivity assays were performed as<br>
follows.<br>
Liquid cultures of strains to be tested were grown<br>
overnight. Cultures were then diluted 1:200, and grown<br>
to an optical density (OD600) of approximately 0.2, after<br>
which the cells were diluted to concentration of<br>
approximately 1 x 105 organisms per 0.05 ml.<br>
Two reactions were carried out for each strain:<br>
(1) vehicle alone (0.01% acetic acid in sterile water)<br>
and (2) defensin NP-1 solution (70 ug/ml in 0.01% acetic<br>
acid). An equal volume of bacterial suspension in<br>
tryptone was added and the test tubes were incubated on a<br>
roller at 37°C for 2 hours. The final volume in each<br>
reaction tube was 0.1 ml, making the final concentration<br>
of defensin 35 ug/ml.<br>
Defensin is inactivated by the high salt and high<br>
protein concentration present in bacterial growth media,<br>
e.g. LB broth. Thus, defensin activity was stopped by<br>
adding 900 ul of Luria broth to each tube. Serial<br>
dilutions of each tube were plated and cfu/ml was<br>
determined for both the control tube and treatment tube<br>
for each strain. Results were expressed as log of<br>
bacteria killed for each strain. Typically, 1.0-1.5 log<br>
of wild type bacteria were killed. PhoP null mutants<br>
generally exhibit 2-4 logs of killing. Since strains<br>
with slower growth rates appear less susceptible to<br>
defensin killing, the growth rate of each strain tested<br>
in the defensin sensitivity assay was measured. Strains<br>
with similar growth rates were compared in the defensin<br>
sensitivity assay.<br>
The Hpal deletion was evaluated both in an S.<br>
typhimurium background and in the S. typhi background.<br>
In both backgrounds, the deletion mutation conferred<br>
sensitivity to rabbit defensin NP-1 at a concentration of<br>
35 ug/ml. See Fig. 11 and Fig. 13. The difference<br>
between PhoP+ and Hpal deleted PhoP null mutants was less<br>
pronounced in the S. typhi strain, an effect that may<br>
reflect the slower growth rate of the less hardy 5. typhi<br>
strain compared to the S. typhimurium strain which lacks<br>
the additional auxotrophies.<br>
The state of phoP activation in bacteria with the<br>
tfpal phoP/phoQ deletion was tested utilizing a LacZ<br>
recorder gene fused to phoP-activated gene B (pagB).<br>
Since the efficiency of transduction utilizing P22 in<br>
5. typhi is low, these studies were performed in S.<br>
typhimurium rather than S. typhi. PhoP activation was<br>
found to be 40-60 Miller units (Miller et al., 1972,<br>
Experiments in Molecular Genetics, Cold Spring Harbor<br>
Laboratory, Cold Spring Harbor, N.Y., pp. 352-355) in the<br>
presence of an intact phoP/phoQ locus, and just barely<br>
detectable in strains with the Hpal deletion (3cfusee<br>
Fig. 12) .<br>
In vivo evaluation of S. typhimurium Hpa~L deleted strain<br>
As S. typhi strains are not pathogenic for mice,<br>
the Hpal phoP/phoQ deletion mutation was evaluated in<br>
both wild type and aroA- S. typhimurium. Female BALB/c<br>
mice were injected intraperitoneally with various<br>
dilutions of S. typhimurium LH430, a wild type S.<br>
typhimurium carrying the Hpal. deletion. The LD50 of this<br>
strain was determined to be between 8.2 x 105 and 8.2 x<br>
106. (All mice receiving 8.2 x 105 cfu survived, and all<br>
receiving 8.2 x 106 died.) These data are consistent<br>
with the LD50 data obtained with strains harboring<br>
transposon insertions at the phoP/phoQ locus.<br>
Immunogenicity of the Hpal phoP/phoQ deletion was<br>
evaluated in S. typhimurium aroA::tet (LH481), a strain<br>
comparable to the human vaccine strain. Mice were<br>
inoculated intraperitoneally with 2.3 x 105 and 2.3 x 106<br>
cfu of LH481 (4 mice per vaccine dose), and challenged 30<br>
days later with 30 x the LD50 of wild type organisms. All<br>
mice but one mouse survived. The mouse that died was in<br>
the group that received the lower vaccine dose. No<br>
animal receiving the higher vaccine dose became ill.<br>
EXAMPLE 4: Phase I study human studies<br>
The vaccine strain was administered to human<br>
volunteers at doses of 1 x io5 to 1 x 1010 cfu/single oral<br>
dose. Two volunteers received each dose; 3 volunteers<br>
were given a dose of 1 x io8 cfu/ml. Volunteers were<br>
evaluated at various time points following administration<br>
of the vaccine.<br>
Safety<br>
To detect the presence of the vaccine strain in<br>
patient blood, Bactec blood cultures were performed in<br>
duplicate on days 4, 6, 8, 10, 12 after taking vaccine.<br>
Bacteremia was not detected in any of the volunteers.<br>
Thirteen adult human volunteers have received<br>
escalating single oral doses of this new attenuated<br>
typhoid fever vaccine. No individuals have had side<br>
effects of any sort. Specifically, there have been no<br>
fevers, no gastrointestinal symptoms, and no<br>
constitutional symptoms. Volunteers have been subjected<br>
to serial blood cultures on a preset schedule after<br>
receiving the oral vaccine 2 sets of BACTEC blood<br>
cultures performed on each of days 4, 6, 8, 10 and 12<br>
after receiving the vaccine, and no positive blood<br>
cultures have been noted. Volunteers have been followed<br>
up at 2 months after receiving the vaccine, and no late<br>
symptoms have been reported.<br>
Colonization<br>
Stool samples were tested for the presence of the<br>
vaccine strain TyLH455 using methods known in the art.<br>
Primary stool was evaluated for the presence of the<br>
vaccine strain on culture plates. In some cases, it was<br>
necessary to enrich stool samples for the vaccine strain<br>
by incubating the stool overnight in BBL Selenite F broth<br>
supplemented with Aro/His/Cystine in order to detect the<br>
bacteria. This medium is somewhat inhibitory for E. coli<br>
and but promotes Salmonella growth.<br>
Volunteers have been colonized for various time<br>
periods from 1-6 days after receiving the vaccine. With<br>
the highest doses (109 or 1010) volunteers have had<br>
positive primary culture plates in the initial 1-3 days<br>
post vaccination, whereas at lower doses, only selenite<br>
enrichment broth cultures (selective medium for<br>
Salmonella which inhibits other enterics) have been<br>
positive for the vaccine organism. No volunteer studied<br>
thus far has had prolonged carriage of the vaccine<br>
organism at 2 months of followup.<br>
(Table Removed)<br>
Measured by whole cell and LPS ELISAs and Widal<br>
test vs. H flagellar antigen. Sera analyzed at 1:40 and<br>
higher dilutions in all tests.<br>
** One of these volunteers has received a booster dose<br>
of 1010 organisms, given one month after the primary<br>
inoculation (serologies pending).<br>
Immunogenicity<br>
Induction of an immune response to the vaccine<br>
strain was measured by standard ELISA assays. Sera was<br>
collected from volunteers 0, 7, 14, 21, and 28 days after<br>
receiving a single oral dose of the vaccine. ELISA<br>
assays were carried out using whole bacteria TyLH445 and<br>
S. typhi LPS (SIGMA, St. Louis, MO) as antigens. Day 0<br>
serum from each volunteer was used as an internal<br>
negative control. Convalescent sera from patients<br>
previously infected with wild type S. typhi (most from<br>
Mexico) were used as positive controls.<br>
Several volunteers had documented seroconversion<br>
at 21 days after receiving the vaccine, as measured by<br>
ELISA in which IgG antibodies directed against whole<br>
vaccine organisms or against S. typhi LPS were detected.<br>
Sera taken from patients prior to administration of the<br>
vaccine (pre-immune sera) were tested and the data used<br>
to establish a baseline. Patient sera taken at various<br>
time points after vaccination were considered positive if<br>
the test results were 0.2 ELISA OD units greater than<br>
that of the preinunune serum.<br>
Other Embodiments<br>
Other embodiments, e.g., strains of Salmonella<br>
which contain only a deletion in the phoP/phoQ regulatory<br>
locus to attenuate virulence, and strains which, in<br>
addition to a phoP related mutation or genetic<br>
alteration, also contain an attenuating mutation in<br>
another gene, e.g., cya gene (adenylate cyclase) or crp<br>
gene (adenylate cyclase receptor), are also within the<br>
claims."<br>
(Table Removed)<br><br><br><br><br>
WE CLAIM:<br><br>
1.	A vaccine comprising a Salmonella typhi Ty800 cell, said cell comprising a mutation in the PhoQ gene, wherein said mutation attenuates virulence and does not result in the insertion of an antibiotic resistance gene into said PhoQ gene.<br>
2.	A vaccine comprising a Salmonella typhi Ty800 cell substantially as hereinbefore described with reference to the foregoing examples.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDQ5LWRlbC0yMDAwLWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">449-del-2000-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDQ5LURFTC0yMDAwLUNsYWltcy0oMDQtMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">449-DEL-2000-Claims-(04-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDQ5LURFTC0yMDAwLUNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">449-DEL-2000-Claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDQ5LURFTC0yMDAwLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDQtMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">449-DEL-2000-Correspondence-Others-(04-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDQ5LURFTC0yMDAwLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDUtMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">449-DEL-2000-Correspondence-Others-(05-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDQ5LURFTC0yMDAwLUNvcnJlc3BvbmRlbmNlLU90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">449-DEL-2000-Correspondence-Others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDQ5LWRlbC0yMDAwLWNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">449-del-2000-correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDQ5LURFTC0yMDAwLURlc2NyaXB0aW9uIChDb21wbGV0ZSktMDQtMDktMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">449-DEL-2000-Description (Complete)-04-09-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDQ5LURFTC0yMDAwLURlc2NyaXB0aW9uIChDb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">449-DEL-2000-Description (Complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDQ5LURFTC0yMDAwLURyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">449-DEL-2000-Drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDQ5LURFTC0yMDAwLUZvcm0tMS0oMDQtMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">449-DEL-2000-Form-1-(04-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDQ5LURFTC0yMDAwLUZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">449-DEL-2000-Form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDQ5LWRlbC0yMDAwLWZvcm0tMTMtKDA0LTA5LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">449-del-2000-form-13-(04-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDQ5LURFTC0yMDAwLUZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">449-DEL-2000-Form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDQ5LURFTC0yMDAwLUZvcm0tMi0oMDQtMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">449-DEL-2000-Form-2-(04-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDQ5LURFTC0yMDAwLUZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">449-DEL-2000-Form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDQ5LURFTC0yMDAwLUZvcm0tMy0oMDQtMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">449-DEL-2000-Form-3-(04-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDQ5LURFTC0yMDAwLUZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">449-DEL-2000-Form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDQ5LURFTC0yMDAwLUZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">449-DEL-2000-Form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDQ5LURFTC0yMDAwLUdQQS0oMDQtMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">449-DEL-2000-GPA-(04-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDQ5LWRlbC0yMDAwLWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">449-del-2000-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDQ5LURFTC0yMDAwLVBldGl0aW9uLTEzOC5wZGY=" target="_blank" style="word-wrap:break-word;">449-DEL-2000-Petition-138.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDQ5LURFTC0yMDAwLVBldGl0aW9uLU90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">449-DEL-2000-Petition-Others.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="223652-power-management-for-subscriber-identity-module.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="223654-process-for-producing-3r-5s-e-7-2-cyclopropyl-4-4-fluorophenyl-quinolon-3-yl-3-5-dihydroxyhept-6-enic-acid-esters.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>223653</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>449/DEL/2000</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>40/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>03-Oct-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>19-Sep-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>24-Apr-2000</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>PRESIDENT AND FELLOWS OF HARVARD COLLEGE</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>17 QUINCY STREET, CAMBRIDGE, MASSACHUSETTS 02139, U.S.A.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>JOHN J. MEKALANOS</td>
											<td>78 FRESH POND PLACE, CAMBRIDGE, MASSACHUSETTS 02138, U.S.A.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SAMUEL I. MILLER</td>
											<td>HEALTH SCIENCES BUILDING, SEATTLE, WASHINGTON 98195, U.S.A.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A61K35/74</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>08/565,861</td>
									<td>1995-12-01</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/223653-a-vaccine-comprising-a-salmonella-typhi-ty800-cell by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:09:07 GMT -->
</html>
