<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/223692-a-recombinant-cell-comprising-a-nucleic-acid-sequence-encoding-an-exogenous-extremozyme-and-a-process-for-producing-the-cell-and-the-extremozyme by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:01:31 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 223692:A RECOMBINANT CELL COMPRISING A NUCLEIC ACID SEQUENCE ENCODING AN EXOGENOUS EXTREMOZYME AND A PROCESS FOR PRODUCING THE CELL AND THE EXTREMOZYME</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A RECOMBINANT CELL COMPRISING A NUCLEIC ACID SEQUENCE ENCODING AN EXOGENOUS EXTREMOZYME AND A PROCESS FOR PRODUCING THE CELL AND THE EXTREMOZYME</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a recombinant cell comprising a nucleic acid sequence encoding an exogenous extremozyme wherein the recombinant cell is a Pseudomonad.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br><br>
OVER-EXPRESSION OF EXTREMOZYA'IE GENES IN PSEUDOMONADS AND<br>
CLOSELY RELATED BACTERIA<br>
BACKGROUND Enzymes have long found use as biocataiysts in industrial and household processes and, more recently, in medical applications. For example, enzymes are commonly employed in traditional industrial biotechno logical processes such as the catalytic liquefaction of com starch (e.g., by amylase enzymes), in household processes such as catalytic stain removal (e.g., by subtilisins and other protease enzymes), and in medical applications such as catalytic thrombolysis for the in vivo dissolution of clots (e.g., by urokinase enzymes). It is widely recognized that enzymes having increased stability under the conditions present in the intended use, a feature typically described in lermsof the half-life of the erizyme's activity under such conditions, have ^eater desirability than those with lesser stability. It is also widely recognized as desirable for the enzyme to exhibit a maximal degree of catalytic activity under the conditions of use, a feature referred to as the enzyme's "opdma" (stated in the plural to reflect that the maximum possible !evel(s) of an enzyme's catalytic activity can vary widi different environmental parameters, e.g., temperature, salinity, pH, etc.). This means that it is most desirable for an enzyme to exhibit both high stability and catalytic optima under the conditions of the intended use.<br>
Many intended uses for enzymes have been proposed wherein the environmental conditions include high or low temperature, high or low pH, high salinity, and other conditions that deviate substantially from the environmental parameters supporting more common living things; among such "more common" biotic conditions are, e.g., temperatures of about 20-60°C,pH of about 6.0-7.5, and salinity below about 3.5% (w/v). [n order to attempt to fiilfill these proposed uses, "extremozymes" have been suggested, Extremozymes are generally considered to be enzymes having significant catalytic activities under extreme environmental conditions, and typically often exhibiting high stability to and catalytic optima under such extreme conditions.<br>
Examples of proposed uses in which extremozymes could offer particular advantages include, e.g,, those listed in Table 2 of M.W.W. Adams &amp; R.M. Kelly, Finding and Using<br><br>
Hyperthermophilic Enzymes, TIBTECH 16:329-332 {1998). Such proposed applications<br>
have contemplated the use of extremozymes in:<br>
i.   Molecular biology, e.g.: employing hyperthermophilic DNA polymerases in the<br>
Polymerase Chain Reaction (PCR); use of extremophilic DNA Hgases in genetic<br>
engineering; extremophilic proteases for use in research;<br>
2.	Starch hydrolysis and processing, e.g.: using alpha-amylases, beta-amylases, glucoamylases, alpha-glucosidases, pullulanases, amylopullulanase, cyclomaltodextrin glucanotransferases, glucose isomerases, and xylose isomerases to produce such products as oligosaccharides, maltose, glucose syrups, high iructose syrups;<br>
3.	Chemical synthesis, e.g.: ethanol production; production of aspartame by thermolysin; production of chiral intermediates for synthesis of pharmaceutical active ingredients; use of other proteases, lipases, and glycosidases having high stability at high temperatures or in organic solvents;<br>
4.	Cellulose and gum degradation and processing, e.g.: paper and pulp bleaching by xylanases; cellobiohydrolases, beta-glucosidases and beta-glucanases for cellulose hydrolysis; thermostable cellulases and glucanases for degradation of biological gums used in oil recovery;<br>
5.	Food and feed processing, e.g.: pectinases, cellulases, and chitinases; galactosidases for lactose hydrolysis; and phytases for dephosphorylation of phytate in animal feed during high temperature processing;<br>
6.	Medical treatments and diagnostic devices and kits, e.g.: peroxidases, phosphatases, oxidases, carboxylases, and dehydrogenases;<br>
7.	Detergents and household products, e.g.: thermophilic proteases, alkalophilic proteases; and, alkaline amylases; and<br>
8.	Other industrial applications, e.g.: biomining and bio-leaching of minerals, bioremediation, remediation of radioactive wastes, antioxidation systems.<br>
The main recognized source for extremozymes is the diverse group of organisms known as extremophiies. Extremophiles are organisms that have been discovered to thrive under extreme environmental conditions, e.g., in or near deep sea hydrothermal vents, hot spnngs, high-salinity lakes, exposed desert surfaces, glaciers and ice packs. Members of this group of organisms include representatives from within each of the following categories, e.g.: prokaryotes including archaea and bacteria, and eukaryotes mcluding fungi and yeasts,<br><br>
lichens, protists and protozoa, algae and mosses, tardigrades and fish. Because organisms of this group naturally thrive under environmental extremes, they are viewed as a source of aaturaily occurring extremozymes. Accordingly, a number of extremozymes from extremophiles have been isolated and tested, and found to have the desired advantageous properties of high stability and catalytic optima under proposed, extreme conditions of use. However, while the industry has anxiously awaited the expected widespread commercialization of extremozymes, this has not been forthcoming.<br>
The; problem is that extremophiles have been found either impossible to culture, or at least too difficult to culture on a commercially significant enough scale to permit cost-effective isolation of extremozymes in sufficient quantity for marketing purposes. As a result, genetic engineering has been tried wherein extremozyme genes, isolated from extremophiles, have been transformed into and expressed in common expression host organisms. Chief among these expression host organisms are E. coli and Bacillus subtilis. Yet, these expression hosts, which have been found so reliable in producmg commerciaJ quantities of non-extremozyme proteins, have so far been unreliable at producing, or unable to produce, commercial quantities of extremozymes. Thus, at best, in spite of the wealth of potential applications for extremozymes, their use has been limited to specialized, small-scale applications such as thermostable DKA pol&gt;'merases for use in research; significant industrial scale use has not yet been achieved because of the lack of a commercially viable, industrial scale extremozyme expression system.<br>
Many examples of such attempts at expression of heterologous extremozyme genes have leen reported in E. coli hosts, and occasionally in Bacillus hosts, and the expression levels ire typically poor, /. e. less than 5% total cell protein. Representative examples include, e.g.: yDon^et al.,\nAppl. Envir. Microbiol. 63&gt;(9)^S69-^Sie iSt^ \991) {Pyrococcus furiosus<br>
imylopuUulanase expressed in E. coli at 10-28mg/L, i.e. about 1.4% total cell protein (tcp)); I. Leveque ei al., in FEMSMicrobiol. Lett. 186(1):67-71 (May 1, 2000) {Thermococcus '.ydrothermalis alpha-amyiase expressed in E. coli at less than 5% tcp, as estimated from IDS-PAGE); A. Linden et al., in J. Chromatog. B Biomed. Sci. Appl 737(l-2):253-9 (Jan<br>
4,	2O0O) (Fyrococciis woesei alpha-amylase expressed in E. coti at 0.4% tcp, as calculated :om data presented therein); and C Pire et al-, in FEMS Microbiol Lett. 200(2):22I-27 (Jun<br>
5,	2001) (25-40 mg/L yield of a halophilic glucose dehydrogenase expressed in E. coli).<br><br>
Two exceptions to the rule of poor expression of extremozymes are reported, both for hyperthermophilic dehydrogenases expressed in E. coli, at levels of 50% top and 15% tcp, respectively. See, H. Connans ex al, in Biotech. Biocng. 64(l);38-45 (Jul 5,1999) {Haloferax volcanii dihydrolipoamide dehydrogenase expressed in E. coli at 50% tcp); and J. Diruggiero &amp; F.T. Robb, in Appt. Environ, Microbiol. 61(I):159-164 (Jan 1995) (Pyrococcus furiosus gJutamate dehydrogenase expressed in £. coH at 15% tcp). However, even these examples fail to provide a commercially viable, industrial scale extremozyme expression system for the following reasons.<br>
First, the £. coU host cells used in the expression systems reported by Connans and Diruggiero grow on a rich medium, which can support a maximum cell density of about 2 g/L (maximum biomass accumulation stated in terms of dry cell weight). At such a low cell density, even an expression level of 50% tcp (total cell protein), results in a yield far too low for uidustiial scale production. For example, with a maximum biomass of 2 g/L, the total cell protein content is approximately 1 g/L; thus, at a 50% tcp expression level, only about 0.5 g/L of the extremozyme would be expressed. An expression system providing a total productivity of only about 0.5 ^L extiemozyme is far too low to be considered capable of industrial scale production. This is especially highlighted when considered in light of the bulk quantities of extremozymes required to enable market supply for the majority of proposed industrial processing and household product uses (most of which are premised on large-scale, mass production).<br>
Second, the largest scale of fermentation reported by either of the Connaris and Diruggiero references is a one-liter (1 L) fermentation, which is far too low to be considered "industrial scale" fermentation. Generally, the lowest limit for any cognizable industrial scale fermentation is about 10 L, though for most purposes this is still considered a small "seed-scale" fermentor. However, some, small-scale commercial uses can be provided by 5 L or 10 L fermentation if the total productivity of the expression system is sufficiently high. Common "seed-scale" fermentors also include 20 L and 40 L fermentors; common "pilot-scale" fermentors can range from about 50 L to 200 L, 250 L, and even 500 L in volume. Typical industrial scale productions are done in fermentors having a volume of 1,000 L and above; even 10,000 L and 50,000 L fermentors are not uncommon.<br><br>
Thus, scaling up a 1 L fermentation-scaie expression system to industrial scale fermentation is not a trivia! matter. Scaling it up in such as way as to provide industrial scale enzyme production is typically quite a challenge, and especially 50 when starting ■■^.'i&amp;i a low-productivity expression system such as report'^d in the Connaris and Diruggiero references. Nor do these references provide anv ';-ggestion or guidance as to how to attempt or accomplish such a scale-up v/iin the expression systems they describe.<br>
Third, the use errich media, e.g., LB medium and others, requires expensive ad_ditives such as peptones and yeast extracts, a fact that makes industrial scale production significantly cost disadvantaged. In fact, for most proposed uses in which extremozymes could replace existing industrial enzymes, this cost disadvantage would make it too expensive to supply extremozymes to the market for industrial use.<br>
Hence, the biotechnology industry continues to lack a commercially viable, industrial scale extremozyme expression system.<br>
SUMMARY OF THE INVENTION<br>
The present invention provides novel means for overexpression of extremozymes. native to extremophilic organisms, on a commercial scale. In a more specific aspect, the invention teaches commercial scale production of these extremozymes by overexpression in host cell species selected from Pseudomonads and closely related bacteria.<br>
These extremozyme expression systems according to the present invention are capable of overexpressing the extremozymes at high levels, at greater than 5% total cell protein, ^eater than 30% total cell protein, and still higher levels. These extremozyme expression systems according to the present invention are capable of obtaining high cell densities, with a dry weight biomass of greater than 20 q/L and even greater than 80 g/L, and are capable of maintaining high levels of extremozyme expression at these high cell densities, thereby providing a high level of total productivity of extremozyme. These extremozyme expression systems according to the present invention are also capable of industrial scale Fermentation, at or above the 10-Liter scale, while maintaining high levels of total jroductivity. In addition, the extremozyme expression systems according to the present<br><br>
iiivcimuii iciaui iiiese aoiiities when grown on simple, inexpensive media, such as carbon source-supplemented mineral salts media.<br>
The present invention also provides:<br>
A recombinant bactenal host cell genetically engineered to contain an expression vector operative therein, the expression vector containing a nucleic acid containing an exogenous extremozyme coding sequence operabiy linked to a control sequence, said host cell being capable of overexpressing said coding sequence, so as to produce said extremozyme at a total productivity of at least 1 g/L, when grown on a medium under conditions permitting expression, characterized in that the bacterial host cell is selected from the Pseudomonads and closely related bacteria.<br>
An extremozyme overexpression system comprising a recombinant bacterial host cell, an expression vector operative in said host cell, the expression vector containing a nucleic acid containing an exogenous extremozyme coding sequence operabiy linked to a conn-o! sequence, said expression system being capable of overexpressing said coding sequence so as to produce said extremozyme at a total productivity of at least ! g/L when grown on a medium under conditions permitting expression, characterized in that the bacterial host cell is selected from the Pseudomonads and closely related bacteria.<br>
A process for overexpressing an extremozyme at a total productivity of at least 1 g/L, comprising the steps of. providing (a) a bacterial host ceil selected from the Pseudomonads and closely related bacteria, (b) an expression vector operative in said host cell and containing a nucleic acid containing an exogenous extremozyme coding sequence operabiy linked to a control sequence, and (c) a medium; transforming said expression vector into said bacterial host cell to form a lecombinant bacterial host cell; and growing said recombinant bacterial host cell on the medium under conditions peroaitting expression; and optionally lysing the host cell and separating, isolating, or purifying the extremozyme therefrom.<br>
A method for overexpressing an extremozyme, at a total productivity of at least 1 g/L, comprising: (I) transforming an expression vector, containing a nucleic acid containing an exogenous exfremozyme coding sequence operabiy linked to a control sequence, into a bacterial host cell selected from the Pseudomonads and closely related bacteria to produce a<br><br>
recombinant bacterial host cell; and (2) growing said recorabinam bacterial host cell on a medium under conditions permitting expression; and optionally lysing the host cell and separating, isolating, or purifying the extremozyme therefrom.<br>
Use, in a method for overexpressing an extremozyme at a total productivity of at least I g/L from a recombinant bacterial host cell grown on a medium under conditions permitting expression, of a recombinant bacterial host cell selected from the Pseudomonads and closely related bacteria.<br>
A commercial kit for overexpressing an extremozyme at a total productivity of at least 1 g/L, comprising: a quantity' of abacterial host cell selected from the Pseudomonads and closely related bacteria; a quantity of an expression vector operative in said bacterial host cell and containing a control sequence; instructions for inserting into said expression vector a nucleic acid containing an exogenous extremozyme coding sequence, so as to operably hnk the coding sequence to the control sequence, thereby preparing the expression vector; instructions for subsequently transforming said expression vector into said bacteria! host cell 10 form a recombinant bacterial host cell; and instructions for growing said recombinant bacterial host cell on a medium under conditions permitting expression; and optionally, a quantity of said medium; and optionally, a quantity of an inducer for a regulated promoter vhere said control sequence utilizes said regulated promoter.<br>
\ commercial kit for overexpressing an extremozyme at a total productivity of at least 1 '/L, comprising: a quantity of a bacterial host cell selected from the Pseudomonads and ;losely related bacteria; a quantity of an expression vector operative in said bacterial host ell and containing a control sequence and an exogenous extremozyme coding sequence perably linked thereto; instructions for transforming said expression vector into said acterial host cell to form a recombinant bacterial host cell; and instructions for growing aid recombinant bacterial host eel! on a medium under conditions permitting expression; fid optionally, a quantity of said medium; and optionally, a quantity of an inducer for a jgulated promoter where said control sequence utilizes said regulated promoter.<br>
jiy of the above wherein the extremozyme is a hydrolase. Any of the above wherein the (tremozyme is a cellulase or amylase; or a peptidase. Any of the above wherein the (tremozyme is an amylase; or a serine endopeptidase or aspaitic endopeptidase. Any of<br><br>
the above wherein the extremozyme is an alpha-amylase; or a pyrolysin or ihermopsin. The extremozyme expressed according to any of the above. Use of an extremozyme expressed according to any of the above in a biocatalytic process.<br>
Any of the above wherein the host cell is a Pseudomonas species. Any of the above wherein the host cell is a fluorescent Pseudomonas species. Any of the above wherein the host cell is Pseudomonas fluorescens.<br>
Any of the above wherein the expression vector is RSFIOIO or a derivative thereof ,Any of the above wherein the heterologous extremozyme promoter is Pfgc-<br>
Any of the above wherein the extremozym-; is expressed in an inclusion body widiin the host cell and the inclLKion body is soiiibilized. Any of the above wherein the extremozyme is refolded using a refoldtng step.<br>
BRIEF DESCRIPTION OF DRAWINGS<br>
Figure 1 presents a plasmid map of an RSFlOlO-based expression vector useful in expressing extremozyme genes according to the present invention.<br>
DETAILED DESCRIPTION 0¥ PREFERRED EMBODIMENTS<br>
The present invention provides a commercial scale production system for extremozymes in which Pseudomonads and closely related bacteria aie used as host cells to over-express the<br>
extremozymes.<br>
Pseudomonas spp. have previously been use as expression systems. See, e.g., US Patent So. 5,055,294 to Gilroy and US Patent No. 5,128,130 to Gilroy et al.\ US Patent No. j,281,532 to Rammler et al; US Patent Nos. 5,527,883 and 5,840,554 to Thompson et al; JS Patent Nos. 4,695,455 and 4,861,595 to Barnes etai; US Patent No. 4,755,465 to Gray '.t al; and US Patent No. 5,169,760 to Wilcox. However, in none of these references has it )een suggested that Pseudomonads and closely related bacteria would be particularly .dvanlageous at over-expressing extremozymes in commercial quantities, as defined by the iresent invention.<br><br>
GLOSSARY<br>
A and An<br>
As used herein and in the appended claims, the singular forms "a", "an", and "the" include<br>
both singular and plural referents unless the context clearly dictates otherwise. Thus, for<br>
example, reference to "a host cell" literally defines both those embodiments employing only<br>
a single host cell and those employing a plurality of such host cells.<br>
In and On<br>
As used herein in regard to growing organisms by use of a growth medium, the oraanisms<br>
may be said to be grown "in" or "on" the medium. In the expression systems of the present<br>
invention, the medium is a liquid medium. Thus, in this context, the terms "in" and "on" are<br>
used synonymously with one another to indicate growth of the host cells in contact with the<br>
medium and generally within the bulk of the medium, though some incidental cell growth<br>
at, in, or upon the surface of the medium is also contemplated.<br>
Comprising<br>
As used herein, the term "comprising" means that the subject contains the elements<br>
enumerated following the term "comprising" as well as any other elements not so<br>
enumerated. In this, the term "comprising" is to be construed as a broad and open-ended<br>
term; thus, a claim to a subject "comprising" enumerated elements is to be construed<br>
inclusively, i.e. construed as not limited to the enumerated elements. Therefore, the term<br>
"comprising" can be considered synonymous with terms such as, e.g., "having,"<br>
"containing," or "including."<br>
The invention, as described herein, is spoken of using the tenns "comprising" and "characterized in that" However, words and phrases having narrower meanings than Ihese are also useful as substitutes for these open-ended terms in describing, defining, or claiming the invention more narrowly. For example, as used herein, the phrase "consisting of means that the subject contains the enumerated elements and no other elements. In this, the phrase "consisting of is to be construed as a narrow and closed-ended term. Therefore, the term "consisting of can be considered synonymous with, e.g.: "containing only" or "having solely".<br><br>
Depositories<br>
ACAM - Australian Collection of Antarctic Microorganisms, Cooperative Research Centre<br>
for Antarctic And Southern Ocean Environment, University of Tasmania, GPO Box 252C,<br>
Hobart, Tasmania 7001, Australia.<br>
ATCC - American T&gt;pe Culture Collection, 10801 University Boulevard, Manassas, VA 20110-2209, U.S.A.<br>
NCIMB - National Collection of Industrial and Marine Bacteria, National Collections of Industrial, Food and Marine Bacteria, 23 Machar Drive, Aberdeen, AB24 3RY, Scotland. UQM - Culture Collection, Department of Microbiology, University of Queensland, St. Lucia, Queensland 4067, Australia.<br>
General Materials &amp; Methods<br>
Unless otherwise noted, standard techniques, vectors, control sequence elements, and other expression, system elements known, in the field of molecular biology are used for nucleic acid manipulation, transformation, and expression. Such standard techniques, vectors, and elements can be found, for example, in: Ausubel ei al (eds.). Current Protocols in Molecular Biology (1995) (John Wiley &amp; Sons); Sambrook, Friisch, &amp; Maniatis (eds.). Molecular Cloning (1989) (Cold Spring Harbor Laboratory Press, NY); Berger &amp; Kimmel, Methods in Enzymology 152: Guide to Molecular Cloning Techniques (1987) (Academic Press); andBukhari etal. (eds.), DMA Insertion Elements, Plasmids andEpisomes (1977) (Cold Spring Harbor Laboratory Press, NY).<br>
X-gal means 5-bromo-4-chloro-3-indolyl-beta-D-galactoside<br>
IPTG means Isopropylthio-beta-D-galactoside<br>
ORF means Open reading frame.<br>
tcp and %tcp<br>
As used herein, the term "tcp" means "total cell protein" and is a measure of the<br>
approximate mass of expressed cellular protein per liter of culture. As used herein, the term<br>
"%tcp" means "percent total cell protein" and is a measure of the fraction of total cell<br>
protein that represents the relative amount of a given protein expressed by the ceil.<br><br>
txogenous and Heterologous<br>
The term "exogenous" means "from a source external to" a given cell or molecule. The term "heterologous" means "from a source different from" a given cell or molecule. In the present application, as is common use in the art. these two terms are used interchangeably, as synonyms. Both of these terms are used herein to indicate that a given object is foreign to the cell or molecule, i.e. not found in nature in the cell or not found in nature with or connected to the molecule.<br><br>
* - Xerophilic is defined by a dimensionless quantity known as "water potential": ^w "^ [(Vapor Pressure of Water in Liquid Solution)/(Vapor Pressure of Pure<br>
Water)], wherein "Liquid Solution" indicates any aqueous medium or aqueous environment, whether intracellular or extraceilular.<br>
Extremophiles are thus defined as those organisms that readily survive or thrive under extracellular environmental conditions falling within these listed parameters. Extremophilic enzymes, or extremozymes, are likffwise defined with reference to the conditions defined in Table 1, and these may be either intracellular or extracellular condidons.<br>
In some cases, chemophilic (e.g., metalophilic) and radiophilic conditions are also recognized in the ait as classes of extremophilic conditions, although these depend on the type of chemical {e.g., a specific metal or a organic compound) and the type of radiation, and thus no uniform definition is included in the present definition of "extremophilic."<br>
Enzymes<br>
As used herein, the terra "enzymes" includes;<br>
I. Oxidoreductases (lUBMB EC 1: including, e.g., monooxygenases, cytochromes,<br>
dioxygenases, dehydrogenases, metalloreductases, ferredoxins, thioredoxins);<br><br>
2.	Transferases (lUBMB EC 2: including, e.g., glycosyltransferases, alkyltranstcrases, acyltransferases, carboxyltransferases, fatty acyl synthases, kinases, RNA and DNA polymerases, reverse transcriptases, nucleic acid integrases);<br>
3.	Hydrolases (lUBMB EC 3: including, e.g., glycosylases, glycosidases, glucohydrolases, glucanases, amylases, cellulases, peptidases and proteases, nucleases, phosphatases, lipases, nucleic acid recombinases);<br>
4.	Lyases (lUBMB EC 4: including, e.g.., decarboxylases, RUBISCOs, adenylate cyclases);<br>
5.	Isomerases (RJEMB EC 5: including, e.g., racemases, epimerases, mutases, topo-isomerases, gyrases, foldases); and<br>
6.	Ligases (lUBMB EC 6; including, e.g., carboxylases, acyl synthetases, peptide synthetases, nucleic acid ligases).<br>
Extremozymes<br>
A wide range of extremozymes are known in the art. See, e.g., references 11-20. As used<br>
herein, the term "extremozyme" means an enzyme exhibiting an optimum of at least one<br>
catalytic property under at least one extremophilic condition as defined in Table !, and<br>
encoded by either:  [} nucleic acid obtained fi-om an extremophilic organism; or 2) nucleic<br>
acid obtained from an extremophilic organism and flirther altered by mutagenesis and/or<br>
recombination as described below. In a preferred embodiment, the extremophilic organism<br>
will be an extremophilic Archaeon, extremophilic bacterium, or extremophilic eukaryote.<br>
Particularly preferred extremophilic eukaryotes include extremophilic fimgi and<br>
extremophilic yeasts. In a paiticularly preferred embodiment, the organism will be an<br>
extremophilic Archaeon or an extrernophilic bacteria.<br>
Whether the extremozyme-encoding nucleic acid is native or altered, the codons of the coding sequence(s) of the nucleic acid may be optimized according to the codon usage frequency of a host cell in which it is to be expressed. The catalytic property in which the optimum is exhibited may be, e.g.: catalytic activity per se or enzymatic throu^put; a metric such as K^, kcat. ^i, ^ii, or V^ax; or stability (catalytic half-life) under conditions of use or proposed use. In addition, the term "extremozyme," as used herein in reference to extremozyme expression systems of the present invention, is restricted to those extremozymes that are heterologous to a selected host ceil chosen for expression thereof<br><br>
Nucleic acids encoding extremozymes may be obtained, eg,, directly from environmental samples using techniques commonly available in the art, e.g., the techniques described in: US Patent Nos. 5,958,672, 6,057,103, and 6,280,926 to Short; US Patent No. 6,261,842 to and WO 01/81567 of Handelsman ei al: US Patent No. 6,090,593 to Fleming &amp; Sayler; or in L. Diels etal, Use of DNA probes and plasmid capture in a search for new interesting environmental genes, Sci. ofihe Total Environ. 139-140:471-8 (Nov 1, 1993). In addition, the techniques described in the following references may also be used; S. Jorgensen et al., in J. Biol. C/iew. 272(26): 16335-42 (Jun 27, 1997); EP Patent No. 577257B1 to Laderman &amp;. Anfmsen; EP Patent No. 579360B1 to Asada ei ai; EP 648843A1 of Taguchi et ai.\ WO 98/45417 of Zeikus et al.; US Patent No. 6,100,073 to Deweer &amp; Amory; and G. Dong et al., inAppl. Environ. Microbiol. 63(9):3577-S4 (Sep 1997).<br>
Once obtained, the extremozyme-encoding nucleic acids may be altered and expressed to obtain an extremozyme exhibiting improvement in or toward a desired catalytic property. Such aheration may be accomplished by use of one or more rounds of nucleic acid mutagenesis and/or recombination, resulting in fomiation of a library comprising altered nucleic acids, followed by (or, if desired when using multiple rounds, regularly or intennittently alternating with) expression of the library and screening of the resuUing-<br>
enzymes. The nucleic acid mutagenesis and recombination technique(s) selected may be in vitro techniques or in vivo or in cyto techniques, and may be random techniques (random mutagenesis, random recombination) or directed techniques {e.g., oHgonucleotide-directed mutagenesis, site-directed recombination). Many such mutagenesis and recombination techniques are commonly known in the art. For example, any of the techniques described in U.S. Patent Nos. 5,850,696.5,965,408, or 6,171,820 to Short; in US Patent Nos. 5,605,793 and 5,811,238 to Stemmer et al\ and WO 98/42832 to Arnold et al. may be used; in addition, mutagenesis may be performed by use of the technique, Error-Prone PCR (also referred to as Low-Fidelity PCR).<br>
In a preferred embodiment, the extremozyrae will be selected from among any of the classes, lUBMB EC 1-6. In a preferred embodiment, the extremozyme will be selected from among any of the classes, lUBMB EC 2-6. In a preferred embodiment, the extremozyme will be selected &amp;om among any of the classes, lUBMB EC 2-5. In a preferred embodiment, the exQ-emozyme will be selected from among either of the classes.<br><br>
lUBMB EC 2-3. In a preferred embodiment, the extremozyme will be selected from among any of the enzymes within lUBMB EC 3, i.e. extremophilic hydrolases. In a preferred embodiment, the extremozyme will be selected from among any of the enzymes within lUBMB EC 3.1-3.8. In a preferred embodiment, the extremozyme wiii be selected from among any of the enzymes within lUBMB EC 3.1-3.2. In a preferred embodiment, the extremozyme will be selected from among any of the enzymes within lUBMB EC 3,2, i.e. extremophilic glycosylases. In a preferred embodiment, the extremozyme will be selected from among any of the enzymes within lUBMB EC 3.2.1, i.e. extremophilic glycosidases. In a preferred embodiment, the extremozyme will be selected from among any of the following enzymes within lUBMB EC 3.2.1: amylases, amyloglucosidases, and glucoamylases; cellulases, cellobiohydroiases, endoglucanases, and hemiceilulases; and beta-glucosidases. In a preferred embodiment, the extremozyme will be selected from among any of the following enzymes within lUBMB EC 3.2.1: amylases and cellulases. In a preferred embodiment, the extremozyme will be selected from among any of the amylases within lUBMB EC 3.2.1, i.e. extremophilic amylases. In a preferred embodiment, the extremozyme will be selected from among any of the alpha-amylases within lUBMB EC 3.2,1 (i.e., the enzymes of lUBMB EC 3.2.1.1), thus, the extremophilic alpha-amylases.<br>
In a preferred embodiment, the extremozyme will be selected from among any of the enzymes within lUBMB EC 3.4. In a preferred embodiment, the extremozyme will be selected from among any of the enzymes within lUBMBEC 3.4.21 or 3.4.23, i.e. extremophilic serine peptidases and exfremophilic aspartic endopeptidases. In a prefened embodiment, the extremozyme will be selected from among any of the following enzymes within lUBMB EC 3.4.21 and 3.4.23: pyrolysins and thennopsins.<br>
In a preferred embodiment, the extremozyme is at least one of: hyperthermophilic, psychrophilic, acidophilic, alkalophilic, and halophilic. In a preferred embodiment, the extremozyme is at least one of: hyperthermophilic, psychrophilic, acidophilic, and alkalophilic. In a preferred embodiment, the extremozyme is at least one of:<br>
hyperthermophilic, acidophilic, and alkalophilic. In a preferred embodiment, the extremozyme is at leasfhyperthermophilic. Particularly preferred are at least hyperthermophilic extremozymes.<br><br>
in the extremozyme expression systems of the present invention, the extremozyme-encoding nucleic acid will be operably linked to a control sequence, and optionally other element(s), to form an expression construct (also called an "expression cassette"), and the resulting expression construct will be inserted into an expression vector; alternatively, the expression cassette can be constructed within the vector by inserting the elements of the expression cassette into the vector in any other series of steps. The expression vector will be then be transformed into a bacterial host cell according to the present invention, followed by expression of the exCremozyme.<br>
Vectors<br>
A great many bacterial vectors are known in the art as useful for expressing proteins in the<br>
Gram(-) Proteobacteria, and these may be used for expressing the extremozymes according<br>
to the present invention. Such vectors include, e.g., plasmids, cosmids, and phage<br>
expression vectors. Examples of useful plasmld vectors include the expression plasmids<br>
pMB9, pBR312, pBR322, pML122, RK2, RK6, and RSFlOlO. Other examples of such<br>
useftjl vectors include those described by, e.g.: N Hayase, in Appl. Envir. Microbiol.<br>
60(9):3336-42 (Sep 5994); AA Lushnikov e; ai, mBasicLife Sci. 30:657-62 (1985); S<br>
Graupner &amp; W Wackemagel, m Biomolec. Eng. 17(1):11-16. (Oct 2000); HP Schweizer, in<br>
Curr. Opin. Biotech. 12(5)-.439-45 (Oct 2001); M Bagdasarian &amp; KN Timmis, in Curr.<br>
Topics Microbiol Immunol. 96:47-67 (1982); T Ishii et al, in FEMS Microbiol Lett.<br>
116(3):307-13 (Mar 1, 1994); IN Olekhnovich &amp; YK Fomichev, in Gene 140(l):63-65 (Mar<br>
n, 1994); MTsuda &amp; T Nakazawa, in Gene 136(l-2):257'62 (Dec 22,1993); C Nieto et<br>
al., in Gene 87(l):I45-49 (Mar 1, 1990); JD Jones &amp; N Gutterson, in Gene 61(3):299-306<br>
(1987); M Bagdasarian et ai, in Gem 16(l-3):237-47 (Dec 1981); HP Schweizer et al, in<br>
Genet. Eng. (NY) 23:69-81 (2001); P Mukhopadhyay et al, in J. Bad. n2(l):477-80 (Jan<br>
1990); DO Wood etal, in J. Bact. I45(3):144S-51 (Mar 1981); and R Holtwick etai, in<br>
Microbiology 147(Pt 2):337'44 (Feb 2001).<br>
Further examples of useful Pseudomonas expression vectors include those listed in Table 2.<br><br><br>
The expression plasraid, RSF1010, is described, e.g., by F Heffron ei al., in Proc Nat'l Acad. Sci. USA 72(9):3623-27 (Sep 1975), and by K Nagahari &amp; K Sakaguchi, in J. Bad. 133(3):1527-29(Mar 1978). Ptasmid RSFlOlO and derivatives thereof are particularly useful vectors in the present invention. Exemplary, useful derivatives of RSFlOIO, which are known in the art, include, e.g., plCT212, pKT214, pKT23I and related plasmids, and pMYClOSO and related plasmids (see, e.g., US Patent Nos. 5,527,883 and 5,840,554 to Thompsone/a/.), sucha, e.g.,pMYCiS03. Other particularly useful vectors include those described in US Patent No. 4,680,264 to Puhler et aL<br>
In a preferred embodiment, an expression piasmid is used as the expression vector. In a preferred embodiment, RSF i 010 or a derivative thereof is used as the expression vector. In a preferred embodiment, pMYClOSO or a derivaUve thereof, or pMYClSOS or a derivative thereof, is used as the expression vector.<br>
Control Sequences<br>
The term "control sequence" is defined herein as the set of all elements which are necessary,<br>
and optionally other elements that are advantageous, for the expression of an extremozyme<br>
in the host cells according to the present invention. Each control sequence element may be<br>
native or foreign to the nucleic acid encoding the extremozyme and may be native or<br>
foreign to the host cell. Such control sequence elements include, but are not limited to:<br>
promoters; transcriptional enhancers; ribosome binding sites (also called "Shine Delgamo<br>
sequences"); translational enhancers (see, e.g., US Patent No. 5,232,840 to Olins); leader<br>
pep tide-encoding sequences, e.g., for targeting peptides or secretion signal peptides, pro-<br><br>
peptide-coding sequences; transcriptional and translational start and stop sienals, poiyadenylation signals; and transcription terminators.<br>
At a minimum, the control sequence(s) will include a promoter, a ribosome binding site, and transcriptional and translational start and stop signals and a transcription terminator. The control sequence elements, vector, and extremozyrae coding sequence may be attached to, or extended to add, linkers or tails for the purpose of introducing specific sequences (e.g., restriction sites) facilitating assembly (e.g., via ligation, recombination, or PCR overlap extension) of the control sequence elements with the coding sequence(s) of the nucleic acid encoding an extremozyme, and with the vector. The term "operably linked," as used herein, refers to any configuration in which the elements of the control sequence are covalently attached to the coding sequence in such disposition(s), relative to the coding sequence, that in and by actloit of the host cell, the control sequence can direct the expression of the coding sequence.<br>
Promoters<br>
The promoter may be any nucleic acid sequence that exhibits transcri.ptional activity in the<br>
host cell of choice, and may be a native, mutant, truncated, or hybrid promoter, native promoters may be obtained from polypeptide-encoding genes that are either native or heterologous to the host cell. If desired, the nucleic acid containing the promoter may remain linked to a ribosome binding site found attached thereto, and optionally to at least part of the coding sequence controlled thereby, as found in its native configuration, (This native coding sequence or portion thereof, if retained, will be attached to the extremozyme coding sequence, ultimately resulting in expression of an extremozyme-fusion protein.)<br>
Any of the many promoters known in the art as capable of directing transcription in the host cells of the present invention may be selected for use therein. See, e.g., Sambrook et al. (1989), supra. The promoter selected may be either a constimtive promoter or a regulated promoter, provided that where the extremozyme is expressed intracellularly {i.e., where it is not secreted or otherwise delivered to a point beyond the host cell's cytoplasm) a constitutive promoter is preferably not used.<br>
Where a regulated promoter is selected, it may be either a positively or negatively regulated promoter. A positively regulated promoter is one that is regulated, via transcriptional<br><br>
activation by an activator protein, to begin transcribing mRNA upon induction. A negatively regulated promoter is one that is repressed by a repressor protein and which permits transcription of mRNA only upon de-repression upon induction. Either a reversibly-inducible or irreversibly-inducible regulated promoter may be selected.<br>
Where a positively regulated promoter is used, the expression system will also contain, or will be genetically engineered to contain, a gene encoding an activator protein therefor, which gene is expressed, preferably constitutively expressed, in the host cell. The activator protein-encoding gene is preferably contained within the host cell chromosome, or it may be contained on the same vector as, or a different vector irom, the vector containing the extremozyme-encoding nucleic acid}. Many such positively regulated promoters and positively regulated promoter-activator protein combinations are know in the art. For example, see: US Patent Nos. 5,670,350, 5,686,283, and 5,710,031 to Gaffney el al.; US Patent No. 5,686,282 lo Lam et al.; Alhri^l el a!., in Annual Rev. Gener. 23:311-336 {1989);BourreteM/.,in.4nnwa/^ev. Biochem. 60:401-441 (1991); and Mekalanos,/. Bad. 174:1-7(1992).<br>
Examples of positively regulated promoters include, e.g.; the "meta promoter" (Pf^) &amp;om ^e meta operon of the toluene-catabolic-pathway-encoding plasmid pWWO of Pseudomonas putida (seeN Hugouvieux-Cotte-Pattate/ai., in J. Bad. 172(12):6651-60 (Dec 1990)); and the araB promoter, which is inducible by addition of L-arabinose which interacts with the activator (the product of the araC gene), as described in US Patent No. 5,028,530.  .<br>
Where a negatively regulated promoter is used, the expression system will also contain, or will be genetically engineered to contain, a gene encoding a repressor protein therefor, which gene is expressed, preferably constitutively expressed, in the host cell. The repressor-protein-encoding gene may be contained on the same vector as, or a different vector from, the vector containing the extremozyme-encoding nucleic acid (or it may be contained within the host cell chromosome). Examples of useful repressors, and genes encoding them, include those described in US Patent Nos. 5,210,025 and 5,356,796 to Keller.<br>
Many negatively regulated promoters and negatively regulated promoter-represser combinations are well known in the art. Examples of preferred negatively regulated<br><br>
promoters include the E. coli tryptophan promoter (Ptjp), the E. coli Sactose promoter (P]ac) and derivatives thereof (e.g., the tac, tacll, and trc promoters, Pt^c- PtacU' ^'^ ^trc described in US Patent No. 4,551,433 to DeBoer), the phage T7 promoter (P77), lambda phage promoters (e.g., ^pL, XpR), and the recA promoter from Rhodobacier capsuhtus. All of the Pjac, Ptac PfacII' PtrC' ^nd P77 promoters are repressed by the lac repressor (lad).<br>
Where a regulated promoter is used, at an appropriate time during the host cell growth cycle, an inducer will be added to activate or de-repress the regulated promoter. Many positively regulated promoter-activator protein-inducer combinations and many negatively regulated promotev-repressor protein-inducer combinations, effective in the host cells of the present invention are well known in the art. For example, in the case of Pm, benzoate will serve as<br>
an inducer; and in the case of Pjac. Ptac- PtacII' Ptrc- ^nd P-^-j, one preferred inducer is IPTG. Also see Table 2. Where an extremozyme is expressed intracellularly within the host cell, preferably the inducer for the regulated promoter will be added upon, or shortly prior to, achievement of maximum host cell proliferation, i.e. maximum "cell density." Especially preferred is to add the inducer at about the mid-log phase of cell proliferation.<br>
In a preferred embodiment of the present invention, a regulated promoter is selected. In a preferred embodiment, a positively regulated promoter is selected, preferably P^. In a<br>
preferred embodiment, a negatively regulated promoter is selected, preferably Pjac- In a preferred embodiment, a negatively regulated promoter is selected for use in an intracellular extremozyme expression system according to the present invention. In a preferred embodiment, the negatively regulated promoter is Vi^^ and the promoter-rqjressor-inducer<br>
combination in which the regulated promoter is utilized will be Ptax:-lacMPTG.<br>
A secreted protein expression system can use either constitutive or regulated promoters. In a secreted protein expression system, either an extremozyme or an extremozyme-fusion protein is secreted from the host ceil. A regulated promoter for a secreted protein expression system can be selected from, e.g., any of those regulated promoters described above. A constitutive promoter for a secreted protein expression system can be selected from among any of the large number of constitutive promoters known in the art as effective<br><br>
for protein expression in the host cells of the present invention. A particularly useful constitutive promoter is the neomycin phosphotransferase 11 promoter (Pnptll) obtained   . from transposon Tn5. See, e.^., DWBauer&amp; A Collraer, Wo/. Plant Microbe Interact. 10{3):369-79 (Apr 1997); and C Casavant el al, A novel genetic system to iirect programmed, high-level gene expression in natural environments, Abstracts of the 99th American Society for Microbiology General Meeting (held May 30 - June 3, 1999 in Chicago, IL, USA). In a preferred embodiment of a secreted protein expression system, a constitutive promoter is used; in a preferred embodiment of a secreted protein expression system, Pnptll is used as the promoter for the extremozyme-encoding nucleic acid.<br>
Other Elements and Methods<br>
Other elements may also be included within the expression system according to the present<br>
invention. For example, a tag sequence that facilitates identi6cation, separation,<br>
purification, or isolation of an extremozyme expressed as a fusion protein therewith can be<br>
encoded by a coding sequence attached to the coding sequence of the extremozyme. In a<br>
preferred embodiment of the present invention, where use of a tag sequence is desired, the<br>
;ag sequence is a hexa-histidine peptide and the extremozyme coding sequence is fused to a<br>
lexa-histidine-encoding sequence. Similarly, the extremozyme may be expressed as a<br>
iusion protein with a whole or partial viral structural protein, e.g., a viral (or phage) coat<br>
jrotein, by attaching all or part of the viral coat protein coding sequence to the coding<br>
lequence of the extremozyme.<br>
'urthermore, one or more marker genes or reporter genes may be used in the expression ystem to verify expression of the extremozyme. Many such usefiil marker or reporter genes re known in the art. See, e.g., US Patent No. 4,753,876 to Hemming et al, and DL Day et (., in J. Bact. 157(3):937-39(Mar 1984). In a preferred embodiment, the marker gene is elected from among the antibiotic resistance-conferring marker genes. In a preferred mbodiment, the marker gene is selected from among the tetracycline and kanamycin isistance genes. In a preferred embodiment, a reporter gene is selected from among those ncoding: (1) fluorescent proteins (e.g., GFP); (2) colored proteins; and (3) fluorescence- or 3lor-facilitating or -inducing proteins, the latter class (3) including, e.g., luminases and eta-gaiactosidese genes. Beta-gaiactosidases hydrolze X-gal to create a blue-colored srivative.<br><br>
Further examples of methods, vectors, and translation and transcription elements, and other elements useftil in the present invention are described in, e.g.: US Patent No. 5,055,294 to Gilroy and US Patent No. 5,128,130 to GUroy ei al.\ US Patent No. 5,281,532 to Rammler etai; US Patent Nos, 4,695,455 and 4,861,595 to Barnes e; a/.; US Patent No. 4,755,465 to Gray et ai\ and US Patent No. 5,169,760 to Wilcox.<br>
Host Cells<br>
Whether in native or altered form, the extremezyme-encoding nucleic acids will be over-expressed, according to the present invention, in bacterial host cells selected from Pseudomonads and closely related bacteria. The "Pseudomonads and closely related bacteria," as used herein, is co-extensive with the group defined herein as "Gram(-) Proteobacteria Subgroup 1." " Gram{-) Proteobacteria Subgroup 1" is more specifically defined as the group of Proteobacteria belonging to the families and/or genera described as falling within that taxonomic "Pari" named "Gram-Negative Aerobic Rods and Cocci" by R.E. Buchanan and N.E, Gibbons (eds.), Bergey's Manual of Determinative Bacteriology, pp. 217-289 (8th ed., 1974) (The Williams &amp; Wilkins Co., Baltimore, MD, USA) (hereinafter "Bergey (1974)"). Table 1 presents the families and genera of organisms listed in this taxonomic "Part."<br><br><br>
"Gram(-) Proteobacleria Subgroup 1" contains all Proteobacteria classified thereunder, as well as all Proteobacteria that would be classified thereunder according to the criteria used in forming that taxonomic "Part." As a result, "Gram(-) Proteobacteria Subgroup I" excludes, e.g.: all Gram-positive bacteria; those Gram-negative bacteria, such as the Enterobacteriaceae, which fall under others of the 19 "Parts" of this Bergey (1974) taxonomy; the entire "Family V. Halobacteriaceae" of this Bergey (1974) "Part," which family has since been recognized as being a non-bacterial family of Archaea; and the genus, Thermits, listed within this Bergey (1974) "Part," which genus which has since been recognized as being a non-Proteobacterial genus of bacteria.<br>
Also in accordance with this definition, "Gram{-) Proteobacteria Subgroup 1" further includes those Proteobacteria belonging to (and previously called species of) the genera and families defined in this Bergey (1974) "Fart," and which have since been given other Proteobacterial taxonomic names, in some cases, these re-namings resulted in the creation of entirely new Proteobacterial genera. For example, the genera Acidovorax,<br>
Brevundimonas, Burkholderia, Hydrogenophaga, Oceanimonas, Ralstonia, and<br>
-22-<br><br>
^tenotropnomonas, were created by regrouping organisms belonging to (and previously called species of) the genus Pseudomonas as defined in Bergey (1974). Likewise, e.g., the genus Sphingomonas (and the genus Blastomonas, derived therefrom) was created by regrouping organisms belonging to (and previously called species oO the genus Xanthomonas as defined in Bergey (1974). Similarly, e.g., the genus Acidomonas was created by regrouping organisms belonging to (and previously called species of) the genus Acetobacler as defined in Bergey (1974). Such subsequently reassigned species are also included within "Gram(-) Proteobacteria Subgroup 1" as defined herein.<br>
In other cases, Proteobacteriat species falling within the genera and families defined in this Bergey (1974) "Part" were simply reclassified under other, existing genera of Proteobacteria. For example, in the case of the genus Pseudomonas, Pseudomonas enalia (ATCC 14393), Pseudomonas nigrifaciens (ATCC 19375), and Pseudomonasputrefaciens (ATCC 8071) have since been reclassified respectively as Aileromonas haloplanktis, Atleromonas nigrifaciens, and Alteromonas putrefaciens. Similarly, e.g., Pseudomonas acidovorans (ATCC 15668) and Pseudomonas testosteroni (ATCC 11996) have since been reclassified as Comamonas acidovorans and Comamonas testosteroni, respectively; and Pseudomonas nigrifaciens (ATCC 19375) and Pseudomonaspiscicida (ATCC 15057) have since been reclassified respectively as Pseudoalteromonas nigrifaciens and Pseudoalteromonas piscicida. Such subsequendy reassigned Proteobacterial species are also included within "Gram{-) Proteobacteria Subgroup 1" as defined herein.<br>
Likewise in accordance with this definition, "Gram(-) Proteobacteria Subgroup 1" further includes Proteobacterial species that have since been discovered, or that have §ince been reclassified as belonging, within the Proteobacterial families and/or genera of this Bergey (1974) "Part." In regard to Proteobacterial families, "Gram(-) Proteobacteria Subgroup 1" also includes Proteobacteria classified as belonging to any of the families: Pseudomonadaceae, Azotobacieraceae (now often called by the synonym, the "Azotobacter group" olPseudomonadaceae), Rhizohiaceae, and Methylomonadaceae (now often called by the synonym, "Methylococcaceae"). Consequently, in addition to those genera otherwise described herein, further Proteobacterial genera falling within "Gram(-) Proteobacteria Subgroup 1" include-. 1) Azotobacter group bacteria of the genus Azorhizophilus\ 2) Pseudomonadaceae family bacteria of the genera Cellvibrio, Oligella, and Teredinibacter,<br><br>
3) Rhizobiaceae family bacteria of the genera Chelatobacter, Ensifer, Liberibacter {also called "Candidatus Liberibacter"), and Sinorhizobium\ and 4) Methylococcaceae family bacteria of the genera Methylobacter, Methylocaldum, Meihyiomicrobium, Methylosarcina, and Methylosphaera.<br>
In a preferred embodiment, the host cell is selected from "Gram(-) Proteobacteria Subgroup 1," as defined above.<br>
In a preferred embodiment, the host cell is selected from "Gram(-) Proteobacteria Subgroup 2." "Gram(-) Proteobacteria Subgroup 2" is defined as the group of Proteobacteria of the following genera (with the total numbers of cataiog-Iisted, publicly-available, deposited strains thereof indicated in parenthesis, ail deposited at ATCC, except as otherwise indicated): Acidomonas (2); Acetobacter (93); Gluconobacter (37); Brevundimonas (23); Beijerinckia (13); Derxia (2); Brucella (4); Agrobacterium (79); Chelatobacter (2); Ensifer (3); Rhizobium (144); Sinorhizobium (24); Blastomonas (1); Sphingomonas (27); Alcaligenes (88); Bordetella (43); Burkholderia (73); Ralstonia (33); Acidovorax (20); Hydrogenophaga (9); Zoogloea (9); Methylobacter (2); Melhylocaldum (1 at NCIMB); Methylococcus (2); Methylomicrobium (2); Methylomonas (9); Methylosarcina (1); Methylosphaera; Azomonas (9); Azorhizophilus (5); Azotobacter (64); Cellvibrio (3); Oligella (5); /'^eU£/omonaj(!I39); Francisella {^)\ Xanthomonas {219); Stenotrophomonas (50); and Oceanimonas (4).<br>
Exemplary host cell species of "Gram(-) Proteobacteria Subgroup 2" include, but are not limited to the following bacteria (with liie ATCC or other deposit numbers of exemplary strain(s) thereof shown in parenthesis): Acidomonas methanoUca (ATCC 43581); Acetobacter aceti (ATCC 15973); Gluconobacter oxydans (ATCC 19357); Brevundimonas diminuta (ATCC 11568); Beijerinckia indica (ATCC 9039 and ATCC 19361); Z)erria gummosa (ATCC 15994); 5ruce//a melitensis (ATCC 23456), firuce//a aftorm^ (ATCC 23448); Agrobacterium tumefaciens (ATCC 23308), Agrobacterium radiobacler (ATCC 19358), Agrobacterium rhizogenes (ATCC 11 325); Chelatobacter heintzii (ATCC 29600); Ensifer adhaerens (ATCC 33212); Rhizobium leguminosarum (ATCC 10004); Sinorhizobium/redii (ATCC 35423); Blastomonas natatoria (ATCC 35951); Sphingomonas paucimobilis (ATCC 29S37); Alcaligenes faecalis (ATCC 8750); Bordetella pertussis<br><br>
(ATCC 9797); Burkholderia cepacia (ATCC 25416); Rabtonia picketiii (ATCC 27511); Acidovoraxfacilis (ATCC 11228); Hydrogenophaga Jlava (ATCC 33667); Zoogloea ramigera (ATCC 19544); Methylohacter luieus (ATCC 49S78); Methyhcaldum gracile (NCIMB 11912); Methyhcoccus capsulatus (ATCC 19069); Methylomicrobium agile (ATCC 35068); Methylomonas methanica (ATCC 35067); Methylosarcinafibrala (ATCC 700909); Methylosphaera hansonii (ACAM 549); Azomonas agilis (ATCC 7494); Aiorhizophilus paspali (ATCC 23%33)\ Azotobacier chroococcum (ATCC 9043); Cellvihrio mi^liis (UQM 2601); OUgella urelhmlis (ATCC 17960); Pseudomonas aeruginosa (ATCC 10145), Pseudomonasfluorescens (ATCC 35858); Francisella tularensis (ATCC 6223); Stenotrophomonas maltophilia (ATCC \ 3637): Xanthomonas campestris (ATCC 33913); and Oceanimonas doudoroffii (ATCC 27123).<br>
In a preferred embodiment, the host eel! is selected from "Gram(-) Proteobacteria Subgroup 3." "Gram(-) Proteobacteria Subgroup 3" is defmed as the group of Proteobacteria of the following genera; Srevundimorias; Agrohacierium; Rhizobium; Sinorhizobium; Blastomonas; Sphingomonas; Alcaligenes; Burkholderia; Ralstonia; Acidovorax; Hydrogenophaga; Methylohacter; Methyhcaldum; Methyhcoccus; Methylomicrobium; Methylomonas; Methylosarcina; Methylosphaera; Azomonas; Azorhizophilus; Azotobacter; Cellvihrio; OUgella; Pseudomonas; Teredinibacter; Francisella; Stenotrophomonas; Xanthomonas; and Oceanimonas.<br>
In a preferred embodiment, the hostcell is selected from "Gram(-) Proteobacteria Subgroup 4." "Gram(-) Proteobacteria Subgroup 4" is defined as the group of Proteobacteria of the following genera: Brevundimonas; Blastomonas; Sphingomonas; Burkholderia; Ralstonia; Acidovorax; Hydrogenophaga; Methylohacter; Methyhcaldum; Methyhcoccus; Methylomicrobium; Methylomonas; Methylosarcina; Methylosphaera; Azomonas; Azorhizophilus; Azotobacler; Cellvihrio; OUgella; Pseudomonas; Teredinibacter, Francisella; Stenotrophomonas; Xanthomonas; and Oceanimonas.<br>
In a preferred embodiment, the host eel! is selected from "Gram(-) Proteobacteria Subgroup 5." "Gram(-) Proteobacteria Subgroup 5" is defined as the group of Proteobacteria of the following genera: Methylohacter; Methyhcaldum; Methyhcoccus; Methylomicrobium; Methylomonas; Methylosarcina; Methylosphaera; Azomonas; Azorhizophilus; Azohbacter;<br><br>
Cellvibrio; Oligella; Pseudomonas; Teredinibacier; Francisella; Slenotrophomonas; Xanthomonas; and Oceanimonas.<br>
In a preferred embodiment, the host cell Is selected from "Gram{-) Proteobacteria Subgroup 6." "Gram(-) Proteobacteria Subgroup 6" is defmed as the group of Proteobacteria of the following genera: Brevundimonas; Blaslomonas; Sphingomonas; Burkholderia; Ralstonia; Acidovorax., Hydrogenophaga: AzQmonas\ Azorhizophilus; Azotobacter; Cellvibrio; Oligella; Pseudomonas; Teredinibacier; Stenotrophomonas; Xanthomonas; and Oceanimonas.<br>
In a preferred embodiment, the host cell is selected from "Gram(-) Proteobacteria Subgroup 7." "Gram(-) Proteobacteria Subgroup 7" is defined as the group of Proteobacteria of the following genera: Azomonas; Azorhizophilus; Azotobacter; Cellvibrio; Oligella; Pseudomonas; Teredinibacter; Stenotrophomonas; Xanthomonas; and Oceanimonas.<br>
In a preferred embodiment, the host cell is selected from "Gram(-) Proteobacteria Subgroup 8," "Gram(-) Proteobacteria Subgroup 8" is defmed as the group of Proteobacteria of the following genera: Brevundimonas; Blastomonas; Sphingomonas; Burkholderia; Ralstonia; Acidovorax; Hydrogenophaga; Pseudomonas; Stenotrophomonas; Xanthomonas; and Oceanimonas.<br>
In a preferred embodiment, the host ceil is selected from "Gram{-) Proteobacteria Subgroup 9." "Gram(-) Proteobacteria Subgroup 9" is defmed as the group of Proteobacteria of the following genera: Brevundimonas; Burkholderia; Ralstonia; Acidovorax; Hydrogenophaga: Pseudomonas; Stenotrophomonas; and Oceanimonas.<br>
In a preferred embodiment, the host cell is selected from "GTam(-) Proteobacteria Subgroup 10." "Gram(-) Proteobacteria Subgroup 10" is defined as the group of Proteobacteria of the following genera: Burkholderia; Ralstonia; Pseudomonas; Stenotrophomonas; and Xanthomonas.<br>
In a preferred embodiment, the host cell is selected from "Gram(-) Proteobacteria Subgroup H." "Gram(-) Proteobacteria Subgroup U" is defined as the group of Proteobacteria of the genera: Pseudomonas; Stenotrophomonas; and Xanthomonas.<br><br>
In a preferred embodiment, the host cell is selected from "Gram{-) Proteobacteria Subgroup 12." "Gr3m(-) Proteobacteria Subgroup 12" is defined as the groupof ProteobacVeriaof the following genera: Burkholderia; Rablonia; Pseudomonas.<br>
In a preferred embodiment, the host cell is selected from "Gram(-) Proteobacteria Subgroup 13." "Gram(-) Proteobacteria Subgroup 13" is defined as the group of Proteobacteria of the following genera: Burkholderia; Rablonia; Pseudomonas; and Xanlhomonas.<br>
In a preferred embodiment, the host cell is selected from "Gram(-) Proteobacteria Subgroup 14." "Gram(-) Proteobacteria Subgroup 14" is defuiedasthe group of Proteobacteria of the following genera: Pseudomonas and Xanlhomonas.<br>
In a preferred embodiment, the host cell is selected from "Gram(-) Proteobacteria Subgroup 15." "Gram(-) Proteobacteria Subgroup 15" is definedas the group of Proteobacteria of the genus Pseudomonas.<br>
In a preferred embodiment, the host cell is selected from "Gram(-) Proteobacteria Subgroup 16." "Gram(-) Proteobacteria Subgroup 16" is defined as the group of Proteobacteria of the following Pseudomonas species (with the ATCC or other deposit numbers of exemplary sfrain(s) shown in parenthesis): Pseudomonas abietaniphila (ATCC 700689); Pseudomonas aeruginosa (ATCC 10145); Pseudomonas alcaligenes (ATCC 14909); Pseudomonas anguilliseptica (ATCC 33660); Pseudomonas citronelloUs (ATCC 13674); Pseudomonas Jlavescens (ATCC 51555); Pseudomonas mendocina (ATCC 25411); Pseudomonas nilroreducem (ATCC 33634); Pseudomonas oleovorans (ATCC 8062); Pseudomonas pseudoalcaligenes (ATCC 17440); Pseudomonas resinovorans (ATCC 14235); Pseudomonas straminea (ATCC 33636); Pseudomonas agarici (ATCC 25941); Pseudomonas alcaliphila; Pseudomonas alginovora; Pseudomonas andersonii; Pseudomonas asplenii (ATCC 23835); Pseudomonas azelaica (ATCC 27162); Pseudomonas beijerinckii (ATCC 19372); Pseudomonas horealis; Pseudomonas boreopolis (ATCC 33662); Pseudomonas brassicacearum; Pseudomonas butanovora (ATCC 43655); Pseudomonas cellulosa (ATCC 55703); Pseudomonas aurantiaca (ATCC 33663); Pseudomonas chlororaphis (ATCC 9446, ATCC 13985, ATCC 17418, ATCC 17461); Pseudomonas fragi (ATCC 4973); Pseudomonas lundensis (ATCC 49968): Pseudomonas<br><br>
laelrolens (ATCC 4683); Pseudomonas cissicola (ATCC 33616); Pseudomonas coronafaciens; Pseudomonas dilerpcniphila\ Pseudomonas elongata (ATCC 10144); Pseudomonas Jlectens (ATCC 12775); Pseudomonas azotoformans; Pseudomonas brenneri-Pseudomonas cedrella; Pseudomonas corrugata (ATCC 29736); Pseudomonas extremorientalis; Pseudomonas fluorescens (ATCC 35858); Pseudomonas gessardii', Pseudomonas Ubanensis; Pseudomonas mandelii (ATCC 700871); Pseudomonas marginalis (ATCC 10844/- Pseudomonas migulae; Pseudomonas mucidolens (ATCC 4685); Pseudomonas orientalis; Pseudomonas rhodesiae; Pseudomonas synxantha (ATCC 9890); Pseudomonas tolaasii (ATCC 3361S); Pseudomonas veronii (ATCC 700474); Pseudomonas frederikshergensis; Pseudomonas gemculata (ATCC 19374); Pseudomonas gingeri; Pseudomonas graminis; Pseudomonas grimontii; Pseudomonas halodenitriftcans; Pseudomonas halophiia; Pseudomonas hibiscicola (ATCC 19867); Pseudomonas hulliensis (ATCC 14670); Pseudomonas hydrogenovora; Pseudomonas jessenii (ATCC 700870); Pseudomonas kilonensis; Pseudomonas lanceolata (ATCC 14669); Pseudomonas lini; Pseudomonas marginata (ATCC 25417); Pseudomonas mephitica (ATCC 33665), Pseudomonas denitrificans (ATCC 19244); Pseudomonas pertucinogena (ATCC 190); Pseudomonas pictorum (ATCC 23328); Pseudomonas psychrophila; Pseudomonas fulva (ATCC 31418); Pseudomonas monteilii (ATCC 700476); Pseudomonas mosselii; Pseudomonas oryzihabilans (ATCC 43272); Pseudomonas plecoglossicida (ATCC 700383); Pseudomonasputida (ATCC 12633); Pseudomonas reaclans; Pseudomonas spinosa (ATCC 14606); Pseudomonas balearica; Pseudomonas luteola (ATCC 43273); Pseudomonas stutzeri (ATCC 17588); Pseudomonas amygdali (ATCC 33614); Pseudomonas avellanae (ATCC 700331); Pseudomonas caricapapayae (ATCC 33615); Pseudomonas cichorii (ATCC 10S57); PseudomonasJicusereclae (ATCC 35104); Pseudomonas Juscovaginae; Pseudomonas meliae (ATCC 33050); Pseudomonas syringae (ATCC 19310); Pseudomonas viridijlava (ATCC 13223); Pseudomonas ihermocarboxydovorans (ATCC 35961); Pseudomonas thermotolerans; Pseudomonas thivervalensis; Pseudomonas vancouverensis (ATCC 700688); Pseudomonas wisconsinensis; and Pseudomonas xiamenensis.<br>
In a preferred embodiment, the host cell is selected from "Gram(-) Proteobacteria Subgroup 17." "Gram(-) Proteobacteria Subgroup 17" is defined as the group of Proteobacteria known in the art as the "fluorescent Pseudomonads" including those belonging, e.g., to the<br><br>
following Pseudomonas species: Pseudomonas azotoformans; Pseudomonas brenneh; Pseudomonas cedrella; Pseudomonas cornigata; Pseudomonas extremorientalis\ Pseudomonas Jluorescens; Pseudomonas gessardii; Pseudomonas libanensis; Pseudomonas mandelii; Pseudomonas marginalis; Pseudomonas migulae; Pseudomonas mucidolens; Pseudomonas orientalis; Pseudomonas rhodesiae; Pseudomonas synxanlha; Pseudomonas tolaasii; and Pseudomonas veronii.<br>
In a preferred embodiment, the host cell is selected from "Gram(-) Proteobacteria Subgroup 18." "Gram(-) Proteobacteria Subgroup 18" is defined as the group of all subspecies, varieties, strains, and other sub-special units of the species Pseudomonas Jluorescens, including those belonging, e.g., to the following (with the ATCC or other deposit numbers of exemplary strain(s) shown in parenthesis): Pseudomonas fluorescens biotype A, also called biovar I or biovar I (ATCC MiSlS); Pseudomonas Jluorescens hio^s'Q, also called biovar 2 or biovar II {ATCC 17816); Pseudomonas Jluorescens biotype C, also called biovar 3 or biovar III (ATCC 17400); Pseudomonas Jluorescens biotype F, also called biovar 4 or biovar IV (ATCC 12983); Pseudomonas Jluorescens biotype G, also called biovar 5 or biovar V (ATCC \151S)\ and Pseudomonas Jluorescens s\xh^'p.cellulosa(^C\UH 10462).<br>
In a preferred embodiment, the host cell is selected from "Gram(-) Proteobacteria Subgroup 19." "Gram(-) Proteobacteria Subgroup 19" is defined as the group of all strains of Pseudomotms Jluorescens biotype A. A particularly preferred strain of this biotype is P. fluorescens strain MBlOl (see US Patent No. 5,169,760 to Wilcox), and derivatives thereof<br>
In a particularly preferred embodiment, the host cell is selected from "Gram(-) Proteobacteria Subgroup 1." In a particularly preferred embodiment, the host cell is selected from "Gram(-) Proteobacteria Subgroup 2.'* In a particularly preferred imbodiment, the host cell is selected from "Gram(-) Proteobacteria Subgroup 3." In a Darticularly preferred embodiment, the host cell is selected from "Gram(-) Proteobacteria Subgroup 5." In a particularly preferred embodiment, the host cell is selected from 'Gram(-) Proteobacteria Subgroup 7." In a particularly preferred embodiment, the host cell s selected from "Gram(-) Proteobacteria Subgroup 12." In a particularly preferred :mbodiment, die host cell is selected from "Grain(-) Proteobacteria Subgroup 15." In a particularly preferred embodiment, the host cell is selected from "Gram(-) Proteobacteria<br><br>
Subgroup 17." In a particularly preferred embodiment, the host cell is selected from "Gram(-) Proteobacteria Subgroup 18." In a particularly preferred embodiment, the host cell is selected from "Gram(-) Proteobacteria Subgroup 19."<br>
Transformation<br>
Transformation of the host cells with the vector(s) may be performed using any<br>
transformation methodology known in the art, and the bacterial host cells may be<br>
transformed as intact cells or as protoplasts {i.e. including cyfoplasts). Exemplary<br>
transfomiation methodologies include poration methodologies, e.g., electroporation,<br>
protoplast fiJsion, bacterial conjugation, and divalent cation treatment, e.g., calcium chloride<br>
treatment or CaCl/Mg2+ treatment,<br>
Fermentation<br>
As used herein, the term "fermentation" includes both embodiments in which literal<br>
fermentation is employed and embodiments in which other, non-fermentative culture modes<br>
are employed. Fermentation may be performed at any scale, in a preferred embodiment.<br>
The fennentation medium may be selected from among rich media, minimal media, and<br>
mineral salts media; a rich medium may be used, but is preferably avoided. In a preferred<br>
embodiment either a minimal medium or a mineral salts medium is selected.<br>
In a preferred embodiment, a minimal medium is selected. In a preferred embodiment, a<br>
mineral salts medium is selected. Mineral salts media are particularly preferred.<br>
Mineral salts media consist of mineral salts and a carbon source such as, e.g., glucose, sucrose, or glycerol. Examples of mineral salts media include, e.g., M9 medium, Pseudomonas medium (ATCC 179), Davis and Mingioli medium (see, BD Davis &amp; ES MingioH, inT; 5acr. 60:17-28 (1950)). The mineral salts used to make mineral salts media include those selected from among, e.g., potassium phosphates, ammonium sulfate or chloride, magnesium sulfate or chloride, and trace minerals such as calcium chloride, borate, and sulfates of iron, copper, manganese, and zinc. No organic nitrogen source, such as peptone, tryptone, amino acids, or a yeast extract, is included in a mineral salts medium. Instead, an inorganic nitrogen source is used and this may be selected from among, e.g., ammonium salts, aqueous ammonia, and gaseous ammonia. A preferred mineral salts medium will contain glucose as the carbon source. In comparison to mineral salts media, minimal media also contain mineral salts and a carbon source, but are fiirther supplemented<br><br>
with, e.g., low levels of amino acids, vitamins, peptones, or other ingredients, though these are added at very minimal levels.<br>
The extremozyme expression system according to the present invention can be cultured in any fermentation format. For example, batch, red-batch, semi-continuous, and continuous fermentation modes may be employed herein.<br>
The expression systems according to the present invention are useful for extremozyme expression at any scale (i.e. volume) of fermentation. Thus, e.g., microliter-scale, centiliter scale, and decihter scale fermentation volumes may be used; and I Liter scale and larger fermentation volumes can be used. In a preferred embodiment, the fermentation volume will be at or above 1 Liter. In a preferred embodiment, the fermentation volume will be at or above 5 Liters. In a preferred embodiment, the fermentation volume will be at or above 10 Liters. In a preferred embodiment, the fermentation volume will be at or above 15 Liters. In a preferred embodiment, the fermentation volume will be at or above 20 Liters. In a preferred embodiment, the fermentation volurne will be at or above 25 Liters. In a preferred embodiment, the fermentation volume will be at or above 50 Liters. In a preferred embodiment, the fermentation volume will be at or above 75 Liters. In a preferred embodiment, the fermentation volume will be at or above 100 Liters. In a preferred embodiment, the fermentation volume will be at or above 150 Liters, In a preferred embodiment, the fermentation volume will be at or above 200 Liters. In a preferred embodiment, the fermentation volume will be at or above 250 Liters. In a preferred embodiment, the fermentation volume will be at or above 500 Liters. In a preferred embodiment, the fermentation volume will be at or above 750 Liters. In a preferred embodiment, the fermentation volume will be at or above 1,000 Liters. In a prefened embodiment, fhe fermentation volume will be at or above 2,000 Liters. In a preferred embodiment, the fermentation volume will be at or above 2,500 Liters. In a preferred embodiment, the fermentation volume will be at or above 5,000 Liters. In a preferred embodiment, the fermentation volume will be at or above 10,000 Liters. In a preferred embodiment, the fermentation volume will be at or above 50,000 Liters. In a particularly preferred embodiment, the femientation volume will be at or above 10 Liters,<br><br>
[n the present invention, growth, culturing, and/'or fermentation of the host cells is performed within a temperature range of about 4°C to about 55°C, inclusive. Thus, e.g., the terms "growth" (and"grow," "growing"), "culmring" (and "culture"), and "fermentation" (and "ferment," "fermenting"), as used herein in regard to the host ceils of the present invention, inherently and necessarily means "growth," "culturing," and "fermentation," within a temperature range of about 4°C to about 55^C, inclusive. In addition, "growth" is used to mdicate both biological states of active cell division and/or enlargement, as well as biological states in which a non-dividing and/or non-enlarging cell is being meiaboIicaUy sustained, the latter use of the terai "growth" being synonymous with the term "maintenance."<br>
In addition, growth "under conditions permitting expression" when used in regard to the •ecombinant bacterial host cells and expression systems of the present invention, is defined \erein to mean; (I) growth of the recombinant bacterial host cells per se, whei-^ the promoter used in the control sequence operably linked to the extremozyme coding sequence s a constimtive promoter; and (2) where the promoter used in the control sequence operably inked to the extremozyme coding sequence is a regulated promoter, (a) growtli of the ecombinant bacterial host cells in the presence of (i.e. in contact with) an inducer therefor, ind (b) growth of the recombinant bacterial host cells in the absence of an inducer therfor, bllowed by addition of such an inducer to the system, thereby causing contact between the ell and the inducer.<br>
iiocatalyst Preparation<br>
)nce expressed, the extremozymes can thetv be separated, isolated, and/or purified using any<br>
rotein recovery and/or protein purification methods known in the art. For example, where<br>
le extremozyme is expressed intracellularly, the host cell can be lysed by standard physical,<br>
hemical, or enzymatic means, see, e.g., P. Prave et al. (eds.), Fundamentals of<br>
•doiechnology (1987) (VCH Publishers, New York) (especially Section 8.3), following by<br>
eparation of the proteins, e.g., by any one or more of microfiltration, ultrafiltration, gel<br>
;ltration, gel purification (e.g., by PAGE), affinity purification, chromatography (e.g., LC,<br>
(PLC, FPLC), and the like. Alternatively, variations of these commonly known protein<br>
icovery and protein purification methods can be used which capitalize on the specific<br>
roperties of these enzymes. For example, it has been reported that hyperthentiophilic<br><br>
enzymes can be easily separated from cellular materials by heating which resuspends the extremozymes while causing precipitation of the other cellular proteins and materials; this method is particularly preferred for use with hyperthermophilic enzymes herein.<br>
Where the extremozyme is secreted from the host cell, it can be directly separated, isolated, and/or purified from the medium. Where the extremozyme is expressed in the host cell as, or as part of, an insolubEe inclusion body, the inclusion body will be solubilized to pemiit recovery of functional enzymes. For example, the host ceils can be lysed to obtain such inclusion bodies therefrom, and then solubilized; alternatively, some extremozyme inclusion bodies can be directly extracted from the host cell by solubilization in cyto without use of a cell lysis step. In either embodiment, such solubilization may result in some degree of unfolding of the expressed extremozyme. Where solubilization results in unfolding of the expressed extremozyme, a refolding step will preferably follow the solubihzation step-Various techniques for solubilizing and refolding the enzymes and other proteins expressed in inclusion bodies are known in the art. See, for example: E De Bemardez Clark, Protein refolding for indiastrial processes, Curr. Opin- in Bioiechnol. n(2):202-07 (Apr 1, 2001); MM Carrio &amp; A Villaverde, Protein aggregation as bacterial inclusion bodies is reversible, FEBS Lett. 489(1 );29-33 (Jan 26, 2001); R Rudolph &amp; H Lilie, In vitro folding of inclusion body proteins,/^45£5 y. 10:49-56 (1996); B ¥isc\\QT et aL,m Bioiechnol. Bioeng. 41:3-13 (1993) (refolding of eukaryotic proteins expressed in E. coli)\ G. Dong et al., in Appl. Envir. Microbiol 63(9):3569-3576 (Sep 1997) (refolding of an exfremophihc amylase enzyme); A Yamagata et al, in Nud. Acids Res., 29(22):4617-24 (Nov 15, 2001) (urea denaturation to solubilize a heterologous, thermophilic RecJ exonuclease enzyme, followed by refolding to obtain an active enzyme); and C Pire et al., in FEMS Microbiol Lett. 200(2):221-27 (Jun 25j 2001) (refolding of an archaeai halophilic glucose dehydrogenase expressed in E. coli).<br>
The exfremozyme expressed according to the present invention can be used in a biocatalytic process, such as described above. Preferred biocatalytic processes are industrial biocatalytic processes. Once separated, isolated, or purified, the extremozymes can then be used to perform biocatalysis, e.g., in free-enzyme or immobilized-enzyme bioreactors, e.g., in place of current industrial enzymes. Alternatively, once the extremozyme has been expressed (or while it is being expressed) by the host cell, it can be used in cyio for biocatalysis. For<br><br>
example, the cell can be used as a biocatalytic unit, e.g., in a whole-cell bioreactor, whether a free-cell or immobilized-cell bioreactor; in this format, the extremozyme can be expressed intracellularly or on die cell surface or can be secreted or odierwise exported from the cell. In a preferred embodiment using this format, the extremozyme is expressed either intracellularly or on the celt surface. The resulting enzyme or whole-cell bioreactor can itself be a batch, fed-batch, semi-continuous, or continuous bioreactor.<br>
Expression Levels<br>
The expression systems according to the present invention express extremozymes at a level<br>
at or above 5% tcp. In a preferred embodiment, the expression level will be at or above 8% tcp. In a preferred embodiment, the expression level will be at or above 10% tcp. In a preferred embodiment, the expression level will be at or above 15% tcp. In a preferred embodiment, the expression level will be at or above 20% tcp. In a preferred embodiment, the expression level will be at or above 25% tcp. In a preferred embodiment, the expression level will be at or above 30% tcp. In a preferred embodiment, the expression level will be at or above 40% tcp. In a prefen-ed embodiment, the expression level will be at or above 50% tcp.<br>
In a preferred embodiment, the expression level will be at or below 35% tcp. In a preferred embodiment, the expression level will be at or below 40% tcp. In a preferred embodiment, the expression level will be at or below 45% tcp. In a preferred embodiment, the expression level will be at or below 50% tcp. In a preferred embodiment, the expression level will be at or below 60% tcp. In a preferred embodiment, the expression level will be at or below 70% tcp. In a preferred embodiment, the expression level will be at or below 80% tcp.<br>
In a preferred embodiment, the expression level will be between 5% tcp and 80% top. In a preferred embodiment, frie expression level will be between 8% tcp and 70% tcp, inclusive. In a preferred embodiment, the expression level will be between 10% tcp and 70% tcp, inclusive. In a preferred embodiment, the expression level will be between 15% tcp and 70% tcp, inclusive. In a particularly preferred embodiment, the expression level will be between 20% tcp and 70% tcp, inclusive.<br><br>
Cell Density<br>
The expressions systems according to the present invention provide a cell density, i.e. a<br>
maximum cell density, of at least about 20 g/L (even when grown in mineral salts media);<br>
the expressions systems according to the present invention likewise provide a cell density of<br>
at least about 70 g/L. as stated in terms of biomass per volume, the biomass being measured<br>
as dry cell weight.<br>
In a preferred embodiment, the ceil density will be at least 20 §/L. In a preferred embodiment, the cell density will be at least 25 g/L. In a preferred embodiment, the cell density will be at least 30 g'L. In a preferred embodiment, the cell density will be at least 35 g/L. In a preferred embodiment, the cell density will be at least 40 g/L. In a preferred embodiment, the cell density will be at least 45 g/L. In a preferred embodiment, the cell density will be at least 50 g/L. In a preferred embodiment, the cell density will be at least 60 g/L. !n a preferred embodiment, the cell density will be al least 70 g/L. In a prefeixed embodiment, the cell density will be at least 80 g/L. In a preferred embodiment, the cell density will be at least 90 g/L. In a preferred embodiment, the cell density will be at least 100 g/L. In a preferred embodiment, the cell density will be at least 110 g/L. In a preferred embodiment, the cell density will be at least 120 g/L. In a preferred embodiment, the cell density will be at least 130 g^L. In a preferred embodiment, the cell density will be at least 140 g/L. In a preferred embodiment, the cell density will be at least 150 g/L.<br>
In a preferred embodiment, the cell density will be at or below 150 g/L. In a preferred embodiment, the cell density will be at or below 140 g/L. In a preferred embodiment, the cell density will be at or below 130 g/L. In a preferred embodiment, the cell density will be at OT below 120 g/L. In a preferred embodiment, the cell density will be at or below 110 g/L. In a preferred embodiment, the cell density will be at or below 1 GO g/L. In a preferred embodiment, the cell density will be at or below 90 g/L. In a preferred embodiment, the cell density will be at or below 80 g/L. In a preferred embodiment, the cell density will be at or below 75 g/L. In a preferred embodiment, the cell density will be at or below 70 g/L.<br>
In a preferred embodiment, the cell density will be between 20 g/L and 150 g/L, inclusive. In a preferred embodiment, the cell density will be between 20 g/L and 120 g/L, inclusive. In a preferred embodiment, the cell density will be between 20 g/L and 80 gL, inclusive. In a preferred embodiment, the cell density will be between 25 g/L and 80 g/L, inclusive. In a<br><br>
preferred embodiment, the ceil density will be between 30 g/L and 80 g/L, inclusive.	In a<br>
preferred embodiment, the cell density will be between 35 g/L and 80 g/L, inclusive,	in a<br>
prefened embodiment, the cell density will be between 40 g/L and 80 g/L, inclusive.	In a<br>
preferred embodiment, the cell density will be between 45 g/L and 80 g/L, inclusive.	In a<br>
preferred embodiment, the cell density will be between 50 g/L and 80 g/L, inclusive.	In a<br>
preferred embodiment, the cell density will be between 50 g/L and 75 g/L, inclusive.	In a<br>
preferred embodiment, the cell density will be between 50 g/L and 70 g/L, inclusive.	In a<br>
particularly preferred embodiment, the cell density will be at least 40 g/L. In a particularly<br>
preferred embodiment, the cell density will be "between 40 g/L and 80 g^L.<br>
Total Productivity<br>
In the expression systems according to the present invention, the total productivity, /. e. the<br>
total extremozyme productivity, is at least 1 g/L. The factors of cell density and expression<br>
level are selected accordingly. In a preferred embodiment, the total productivity will be at<br>
least 2 g/L. In a preferred embodiment, the total productivity will be at least 3 g/L. In a<br>
preferred embodiment, the total productivity will be at least 4 g/L. In a preferred<br>
embodiment, the total productivity will be at least 5 g/L. In a preferred embodiment, the<br>
total productivity will be at least 6 g/L. In a preferred embodiment, the total productivity<br>
will be at least 7 g/L. In a preferred embodiment, the total productivity will be at least 8<br>
g/L. In a preferred embodiment, the total productivity will be at least 9 g/L. In a preferred<br>
embodiment, the total productivity will be at least 10 g/L.<br>
In a particularly preferred embodiment, the expression system will have an extremozyme expression level of at least 5% tcp and a cell density of at least 40 g/L, when grown (i.e. within a temperature range of about 4°C to about 55°C, inclusive) in a mineral salts medium. In a particularly preferred embodiment, the expression system will have an extremozyme expression level of at least 5% tcp and a cell density of at least 40 g/L, when grown (i.e. within a temperature range of about 4°C to about 55°C, inclusive) in a mineral salts medium at a fermentation scale of at least 10 Liters.<br><br>
EXAMPLES<br>
Example 1. Extremophilic Cellulase<br>
Example 1 A. Construction of Pseudomonas Jluorescens strains expressing Thermotoga maritima and Pyrococcus fiiriosus ceiluiases.<br>
Methods<br>
Molecular Biology techniques were as described in Ausube! et al. (eds.). Current Protocols<br>
in Molecular Biology {\995) (John Wiley &amp; Sons); Sambrook, Fritsch, &amp;Maniatis (eds.),<br>
Molecular Cloning (1989) (Cold Spring Harbor Laboratory Press, NY).<br>
Expression Cassettes<br>
The parent plasmid pMYCl803 is a derivative of pTJS260 (see US Patent :NO. 5,169,760 to<br>
Wilcox), carrying a regulated tetracycline resistance marker and, the rephcation and<br>
mobilization loci from RSFl 010 plasmid. (pMYC1803 is a source for many derivative<br>
plasmids useful in expression extremozymes according to the present invention. Most such<br>
derivatives differ from pMYCl 803 primarily around the ORF in order to introduce<br>
convenient restriction sites for cloning different exogenous genes.)<br>
The Thermotoga maritima cellulase gene (0.94 kb encoding the 314 aa, 38kD cellulase) and the Pyrococcus furiosus endoglucanase gene (0.90 kb encoding the 500 aa, 34kD endoglucanase) were PCR-amplified using primers designed to introduce a Spel site at the N-terminal end, along with the franslational start site of the ORF in pMYC1803, and a.A7ioI site at the C-terminus of the coding sequences of the genes. The 5pel --^ol fragment of the respective PCR products were independently inserted into pMYC1803 at tiie corresponding sites such that the enzyme genes replaced an exogenous gene already present therein hence, their expression was initiated from the tac promoter. The resulting constructs, pMYC1954 and pDOW2408, in E. coli JM109 was screened by restriction digests and qualitative enzyme assays and then, alkaline lysis miniprep plasmid DNA's of the correct constructs were elecfroporated into P. Jluorescens MB214.<br><br>
Host Strain Pseudomonas fluorescens MB214<br>
M.B214 is a derivative of MBl 01 (a wild-type prototrophic P. fluorescens strain), derived by a procedure wherein (he lacIZYA operon (deleted of the /acZpromoter region) had been integrated into the chromosome to provide a host background where derivatives of the lac<br>
promoter can be regulated by lactose or IPTG. MBl 01 is Lac" whereas MB214 \- Lac"^ However, MBlOl can be rendered Lac+ by iritroducing an E. coli lad genf on a plasmid into the strain.<br>
Example IB. Expression of Extremophilic Cellulases<br>
Seed cultures were produced as follows. P. fluorescens MB214 transforaiants were inoculated into 2-5 mL of Luria-Bertani Broth ("l,Li'% supplemented widi 15p,g/mL telracyciine HCl, in 15 ml Faicun tubes a^u growth for 16-20h, at 32°C, 300rpm. 1 mL of the seed culture (in LB) was placJ into 50 mL of the Terrific Broth (TB) medium (see Table 4), supplemented witli IS^g/mL tetracycline HCl, in 250ml bottom baffled shake-flasks, and incubated for 5h at 32°C, 300rpm. Induction was performed by the addition of<br><br><br><br>
The cellulases were expressed at levels above 8% tcp in both shake-flask and high cell density fermentor cultures. The cellulases were separated and tested foi activity and were found to be active.<br>
Example 2. Extremophilic Amylases<br>
Alpha-amylase genes from a Thermococcal and a Sulfolobus solfataricus source were PCR amplified and cloned onto pMYC 1803 as in Example 1, so that they became operably linked to a control sequence including the P^ac Promoter in, an RSFlOlO-based vector also<br>
carrying a tetracycline resistance marker, as shown in Figure I. The resulting constructs<br>
were transformed into Lad"*" P. Jluorescens MBSOl. The resulting recombinant host cells were cultured in 10 L fermentors by growth in a mineral salts medium (supplemented with tetracycline and fed wi,th glucose or glycerol). The transformants were grown in fed-batch fermentation cultures, ultimately to ceil densities providing biomasses within the range of about 20 g/L to more than 70 g/L (dry cell weight). The gratuitous inducer of the Pj^^^.<br>
promoter, IPTG, was added to induce expression. Thereupon, the amylases were expressed (i. e. over-expressed) to a level within the range of about 5% tcp to more than 30% tcp. Thus, total productivity ranged fi^om about 2 g/L to over 10 g/L, offering a yield above 100 g of extremozyme &amp;om a single 10 L batch. After host cell lysis, the extremozymes were purified by microfiltration followed by ultrafiltration. The resulting enzymes were characterized and further tested for starch liquefaction activity and found to be active, hyperthennophilic, and acidophilic.<br>
Example 3. ExtremopMic Proteases<br>
Pyrococcus furiosus and Sulfolobus acidocaldarius protease genes respectively encode pyrolysin (lUBMB EC 3.4.21.-), a serine protease active at 1 IS^'C and pH 6.5-10.5, and themiopsin (lUBMB EC 3.4.23.42), an acid protease operating optimally at 90°C and pH 2.0, respectively. These genes were PCR amplified and cloned onto pMYCl803 as in Example 1, so that they became operably linked to a control sequence including the P^ac<br><br>
promoter in an RSFlOlO-based vector also carrying a tetracycline resistance marker, as<br>
shown in Figure 1. The resulting constructs were transformed mtohacl'^ P. Jluorescem MB2I4. The resulting recombinant host cells were cultured in 10 L fermentors by growth in a mineral salts medium (supplemented with tetracycline and fed with glucose or glycerol). The transformants were grown in fed-batch fermentation cultures, ultimately to cell densities providing biomasses within the range of about 20 g/L to more than 70g/L (dry cell weight). Upon induction with IPTG, the proteases were expressed to levels within the range of about 5% tcp to more than 30% tcp. Thus, total productivity ranged from about 1 g/L to over 10 g/L, offering a yield above 100 g of extremozyme from a single 10 L batch.<br>
References<br>
1.	C. Nieto et al., Cloning vectors, derived from a naturally occurring plasmid of Pseudomonas savastanoi, specifically tailored for genetic manipulations in Pseudomonas, Gene 87:145-149 (1990).<br>
2.	M. Bagdasanan et ai, Molecular and functional analysis of the broad host range plasmid RSFIOIO and construchon of vectors for gene cloning in Gram-negative bacteria, in Microbial Drug Resistance: Proceedings of the Third Int'l Symp., pp.\S3'97 (Tokyo, 1982).<br>
3.	J.P. Fuerste el ai. Molecular cloning of the plasmid RP4 primase region in a multi-host-range tacP expression vector, Gene 48:119-131 (1986).<br>
4.	E. Brunschwig &amp; A. Darzins, A two-component T7 system for the overexpression of genes in Pseudomonas aeruginosa. Gene 111:35-41 (1992).<br>
5.	R. Leemans ei al., A broad-host-range expression vector based on the pL promoter of coliphageX: regulated syiXhesis of human inierleukin2 in Srwinia and Serralia species, J.Bact. 169:1899-1904(1987).<br>
6.	J.L. Ramos et ai. Broad-host-range expression vectors containing manipulated meta-cleavage pathway regulatory elements of the TOL plasmid, FEBSLett. 226:241-246 (1988).<br>
7.	N.T. Keen et al. Improved broad-host-range plasmids for DNA cloning in Gram-negative bacteria, Gene 70:191-197 (1988).<br>
8.	J.M. Blatny et al. Construction and use of a versatile set of broad-hosl-range cloning and expression vectors based on the RK2 replicon. Appl. Env. Microbiol. 63:370-379 (1997); and J.M. Blatney et al. Improved broad-host-range RK2 vectors for high and low regulated gene expression levels in Gram-negative bacteria, P/ajm/t^ 38:35-51 (1997).<br>
9.	A.A. Watson et ai. Construction of improved vectors for protein production in Pseudomonas aeruginosa, Gene 172:163'164 (1996).<br>
10.	H.P. Schweizer ej o/., Vector design and host systems fox Pseudomonas, ini.'K. Setlow (ed.). Genetic Engineering, vol. 23 (2001) (Kluwer Plenum Press, New York).<br>
11.	J. Eichler, Bioiechnological uses of archaeal extremozymes, Biotech. Adv. 19:261-78 (2001).<br><br>
12.	M.C. Srinivasan ei al. Production and application of enzymes stable to and active under<br>
extreme environments: an overview, Proc. Indian Nat'! Sci. Acad. 65{Pt. B): 143-62.<br>
(1999).<br>
13.	G.A. Sellek &amp; IB. Chaudhuri, Biocatalysis in organic media using enzymes from<br>
	extremophiles, Enz. Microb. Technol. 25:471-82 (1999).<br>
14.	K.O. Stetter, Extremophiles and their adaptation to hot environments, FEES Lett. 452:22-25 (1999).<br>
15.	M.W.W. Adams et al., Extremozymes: Expanding the limits of biocatalysis, Biohechnology 13:662-68. (1995).<br>
16.	B.C. Demirjian e/a/., Enzymes from extremophiles, Curr. Opin. Chem. Biol. 5:144-51 (2001).<br>
17.	E. Leveque et al.. Thermophilic archaeal amylolytic enzymes, Enz. Microb. Technol. 26-.3-14. (2000).<br>
18.	F. Niehaus et at, Extremophiles as a source of novel en2ymes for industrial application, Appi Microbiol. Biotech. 5V.711-29 (1999).<br>
19.	C. Vieille &amp; G.J. Zeikus, Hyperthennophilic enzymes: Sources, uses, and molecular mechanisms foT thermostability, Microbiol. Mol. Biol. Rev. 65:1-43 (2001).<br>
20.	D.W. Hough &amp; M.J. Danson, Extremozymes, Curr. Opin. Chem. Biol. 3:39-46 (1999).<br>
It is to be understood that the preferred embodiments described above are merely exemplary of the present invention and that the terminology used therein is employed solely for the purpose of illusfrating these preferred embodiments; thus, the preferred embodiments selected for the above description are not intended to limit the scope of the present invention. The invention being thus described, other embodiments, alternatives, variadons, and obvious alterations will be apparent to those skilled in the art, using no more than routine experimentation, as equivalents to those preferred embodiments, methodologies, protocols, vectors, reagents, elements, and combinadons particularly described herein. Such equivalents are to be considered within the scope of the present invention and are not to be regarded as a departure from the spirit and scope of the present invention. All such equivalents are intended to be included within the scope of the following claims, the true scope of the invention thus being defined by the following claims, as construed under the doctrine of equivalents or like doctrine(s) applicable in the present jurisdiction.<br><br><br><br>
WE CLAIM:<br>
1.	A recombinant cell comprising a nucleic acid sequence encoding an exogenous exlremozyme wherein the recombinant cell is a Pseudomonad.<br>
2.	The recombinant cell as claimed in claim 1 wherein the recombinant cell expresses an extremozyme protein.<br>
3.	The recombinant cell as claimed in claim 2, wherein the extremozyme is expressed at a productivity level of at least 1 gram per liter (g/L).<br>
4.	The recombinant cell as claimed in claim 1 wherein the host cell is a Pseudomonas fluorescens cell.<br>
5.	The recombinant cell as claimed in claim 1 wherein the extremo2yme is selected from the group consisting of hyperthermophilic, psychrophilic, acidophilic, alkalophilic, and halophilic extremozymes.<br>
6.	A recombinant cell comprising a nucleic acid sequence encoding an exogenous extremozyme wherein the recombinant cell is selected from Gram (-) Proteobacteria Subgroup consisting of 1,2, 3, 5, 7, 12, 15, 17 and 18.<br>
7.	The recombinant cell as claimed in claim 6 wherein the recombinant cell expresses an extremozyme protein.<br>
8.	The recombinant cell as claimed in claim 7, wherein the extremozyme is expressed at a productivity level of at least 1 gram per liter (g/L),<br><br>
9.	The recombinant cell as claimed in claim 6 wherein the extremozyme is selected<br>
from the group consisting of hypeithermophilic, psychrophilic, acidophilic,<br>
alkaiophilic, and halophilic extremozyme.<br>
10.	A process for producing a recombinant cell expressing an extremozyme<br>
comprising:<br>
a)	providing a host cell selected from a Pseudomonad;<br>
b)	transforming the host cell with a nucleic acid including a coding sequence for an exogenous extremozyme;<br>
c)	growing the transformed host cell under conditions permitting expression of the extremozyme,<br><br>
11.	The process as claimed in claim 10 wherein the sequence of the exogenous extremozyme is altered to improve a catalytic property of the extremozyme.<br>
12.	The process as claimed in claim 10, wherein the expression of the extremozyme is at a productivity Ie\'e] of at least 1 gram per liter (g/L).<br>
13.	The process as claimed in claim 10 wherein the recombinant cell expresses the extremozyme at an expression level of at least 5% total cell protein.<br>
14.	A process for producing an extremozyme comprising: providing a recombinant cell comprising a nucleic acid sequence encoding for an exogenous extremozyme wherein the recombinant cell is a Pseudomonad; and growing the recombinant cell under conditions permitting expression of the extremozyme, wherein the extremozyme is isolated.<br><br>
15.	The process as claimed in claim 14 wherein the extremozyme is expressed at a productivity of at least I gram per liter (g/L).<br>
16.	The process as claimed in claim 15 wherein the productivity is at least 4 g/L.<br>
17.	The process as claimed in claim 15 wherein the productivity is 6 g/L.<br>
18.	The process as claimed in claim 15 wherein the productivity is at least 10 g/L.<br>
19.	The process as claimed in claim 14 wherein the cell is grown to a cell density of at least 40 grams per liter.<br>
20.	The process as claimed in claim 14 wherein the cell is grown in a fermentation scale of at least 10 liters.<br>
21.	The process as claimed in claim 14 wherein the cell is grown in a mineral salts medium.<br>
22.	The process as claimed in claim 14 wherein the host cell is a Pseudomonas fluorescens cell.<br>
23.	The process as claimed in claim 14 wherein the extremozyme is purified.<br>
24.	The process as claimed in claim  14 wherein the extremozyme undergoes a<br>
refolding step.<br><br>
25.	The process as claimed in claim 14, wherein the extremozyme is selected from the group consisting of hyperthermophihc, psychrophihc, acidophilic, alkalophilic, and halophilic extremozyme.<br>
26.	A process for producing an extremozyme comprising: providing a recombinant cell comprising a nucleic acid sequence encoding an exogenous extremozyme wherein the recombinant cell is selected from the group consisting of Gram (-) Proteobacteria Subgroup 1, 2, 3, 5, 7, 12, 15, 17 arid 18; and growing the recombinant cell under conditions permitting expression of the extremozyme.<br>
27.	The process as claimed in claim 26 wherein the extremozyme is expressed at a productivity of at least 1 gram per liter (g/L),<br>
28.	The process as claimed in claim 27 wherein the productivity is at least 4 g/L.<br>
29.	The process as claimed in claim claim 27 wherein the productivity is at least 6 g/L.<br>
30.	The process as claimed in claim 2? wherein the productivity is at least 10 g/L.<br>
31.	The process as claimed in claim 26 wherein the recombinant cell expresses the extremozyme at an expression level of at least 5% total cell protein.<br>
32.	The process as claimed in claim 26 wherein the cell is grown to a cell density of at least 40 grams per liter.<br><br>
33.	The process as claimed in claim 26 wherein the cell is grown in a fermentation scale of at least 10 liters,<br>
34.	The process as claimed in claim 26 wherein the cell is grown in a mineral sahs medium.<br>
35.	The process as claimed in claim 26, wherein the extremozyme is selected from the group consisting of hyperthermophilic, psychrophilic, acidophilic, alkalophilic, and halophilic extremozyme.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2Ni1jaGVucC0yMDA0IGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1766-chenp-2004 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2Ni1jaGVucC0yMDA0IGNsYWltcy1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1766-chenp-2004 claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2Ni1jaGVucC0yMDA0IGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1766-chenp-2004 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2Ni1jaGVucC0yMDA0IGNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1766-chenp-2004 correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2Ni1jaGVucC0yMDA0IGNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">1766-chenp-2004 correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2Ni1jaGVucC0yMDA0IGRlc2NyaXB0aW9uKGNvbXBsZXRlKS1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1766-chenp-2004 description(complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2Ni1jaGVucC0yMDA0IGRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">1766-chenp-2004 description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2Ni1jaGVucC0yMDA0IGRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1766-chenp-2004 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2Ni1jaGVucC0yMDA0IGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">1766-chenp-2004 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2Ni1jaGVucC0yMDA0IGZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">1766-chenp-2004 form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2Ni1jaGVucC0yMDA0IGZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">1766-chenp-2004 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2Ni1jaGVucC0yMDA0IGZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">1766-chenp-2004 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2Ni1jaGVucC0yMDA0IGZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">1766-chenp-2004 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2Ni1jaGVucC0yMDA0IG90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1766-chenp-2004 others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2Ni1jaGVucC0yMDA0IHBjdCBzZWFyY2ggcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">1766-chenp-2004 pct search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2Ni1jaGVucC0yMDA0IHBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1766-chenp-2004 pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2Ni1jaGVucC0yMDA0IHBldGl0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">1766-chenp-2004 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="223691-a-method-for-producing-continuously-molded-bodies-from-a-molding-material.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="223693-a-non-human-animal-medicine-and-a-process-for-preparing-the-same.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>223692</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1766/CHENP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>47/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Nov-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>19-Sep-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>06-Aug-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>DOW GLOBAL TECHNOLOGIES, INC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>WASHINGTON STREET, 1790 BUILDING, MIDLAND, MICHIGAN 48674,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>CHEW, LAWRENCE, C</td>
											<td>5590 WILLOWMERE LANE, SAN DEIGO, CA 92130-4802,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>TALBOT, HENRY, W</td>
											<td>4111 STURGEON COURT, SAN DIEGO, CA 92130,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>LEE, STACEY, L</td>
											<td>13831 PASEO CARDIEL, SAN DIEGO, CA 92129,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12P21/06</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US03/04288</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-02-13</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>US02/04294</td>
									<td>2002-02-13</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/223692-a-recombinant-cell-comprising-a-nucleic-acid-sequence-encoding-an-exogenous-extremozyme-and-a-process-for-producing-the-cell-and-the-extremozyme by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:01:32 GMT -->
</html>
