<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/223740-a-recombinant-dna-molecule-useful-as-a-promoter-in-dicot-as-well-as-monocot-plant-cells by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 04:51:55 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 223740:A RECOMBINANT DNA MOLECULE USEFUL AS A PROMOTER IN DICOT AS WELL AS MONOCOT PLANT CELLS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A RECOMBINANT DNA MOLECULE USEFUL AS A PROMOTER IN DICOT AS WELL AS MONOCOT PLANT CELLS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention relates to an artificial promoter which is characterised in that it comprises a 5 chimeric molecule of recombinant DNA which, once introduced into plant cells of any class, promotes high expression levels of any DNA molecule that is fused to the 3&#x27; end thereof. The basic genetic elements of the inventive promoter molecule are as follows: a promoter nucleus with a consensus TATA box followed by an Exon/Intron/Exon region and a translational activity-potentiating element, all of which are produced artificially. 10 Transcriptional expression-regulating elements can be inserted upstream of the promoter in order to provide the expression with the specific time-response capacity of organ or tissue. The artificial genetic elements designed can be functionally inserted between any active promoter in plant cells and any DNA sequence in order to increase the transcription/translation levels of the latter. 15</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>i<br>
FORM 2<br>
THE PATENTS ACT, 1970 (39 of 1970)<br>
&amp;<br>
The Patents Rules, 2003 PROVISIONAL / COMPLETE SPECIFICATION<br>
	  (See section 10 and rule 13)	<br>
1. TITLE OF THE INVENTION :<br>
"ARTIFICIAL PROMOTER FOR THE EXPRESSION OF   DNA SEQUENCES<br>
IN VEGETAL CELLS"<br><br>
2. APPLICANT (S)<br>
(a)	NAME<br>
(b)	NATIONALITY<br>
(c)	ADDRESS<br><br>
CENTRO DE INGENIERIA GENETICA Y BIOTECNOLOGIA<br>
Cuba<br>
Ave. 31, entre 158 y 190, Cubanacan, Playa, C. Habana 10600, Cuba<br><br>
3. PREAMBLE TO THE DESCRIPTION<br><br>
PROVISIONAL<br>
The following specification describes the invention<br><br>
COMPLETE V<br>
The  following  specification  particularly describes  the invention and the manner in which it is to be performed.<br><br>
4.  DESCRIPTION (Description shall start from next page)<br>
5. CLAIMS (not applicable for provisional specification. Claims should start with the preamble â€” "I/we claim" on separate page)<br>
6. DATE AND SIGNATURE (to be given at the end of last page of specification)<br>
7. ABSTRACT OF THE INVENTION (to be given along with complete specification on separate page)<br><br>
W
 <br>
!
 <br>
JWfWrofOV QS<br>
/3 7/or<br><br>
2<br>
ARTIFICIAL PROMOTOR FOR THE EXPRESSION OF DNA SEQUENCES IN<br>
VEGETAL CELLS<br>
Field of tbe invention<br>
5      This invention is related with biotechnology and more specifically with Plant Genetic<br>
Engineering. In particular, chimerical DNA constructs are given, which shows a high<br>
transcription/translation promoter activity of any nucleotide sequence fused to them in<br>
dicot and monocot plant cells, which allows obtaining transgenic plants with higher<br>
expression levels of genes and DNA sequences of interest.<br>
10<br>
Previous Art<br>
Plant genetic engineering is a technology that has demonstrated to be very productive for<br>
basic   investigation  and  commercial  production  of new  biotechnological  products.<br>
(Hammond J. Curr. Top. Microbiol. Immunol 1999, 240:1-19; Simoens C. and Van<br>
15      Montagu M. Reproduction Update 1995,1:523-542).<br>
The selection of promoter signals that warranty the adequate expression in terms of strength and temporal or spatial specificity of gene or DNA sequence introduced in plant genetically manipulated by the means of molecular biology techniques is very importat for the success of plant genetic engineering. That is why in the last two decades multiples<br>
20     efforts have been dedicated to the search of promoters and signals able to ensure the expression each transgene required. Thus, promoters of different origin (plant, viral, Ti or Ri Agrobacterium or chimerical) have been evaluated and employed in transgenic plant production. The constitutive promoters more widely used in plant genetic manipulation have been the<br>
25 Cauliflower Mosaic Virus (CaMV) 35 S ARN promoter (Odell J.T; Nagy F; Chua N.H. Nature 1985, 313:810-812); nopaline synthetase gene (nos) promoter from A tumefaciens Ti plasmid (An G; Costa M.A; Mitra A; Ha S; Marton L. Plant Phisiol. 1986, 88:547-552), rice actin-1 gene promoter (McElroy D; Zhang W; Cao J; Wu R. Plant Cell 1990, 2:163-171) and maize ubiquitin-1 gene promoter (Christensen A.H; Sharrock R.A; Quail P.H.<br>
30 Plant Mol. Biol. 1992, 18:675-689). However, each of these natural expression systems have limitations, mainly because its expression levels are not enough high in any class of plants; for example, the promoter expression is low in dicot plant cells and almost undetectable in monocots, while the expression of CaMV 35 S, the most widely used promoter, is much stronger in tobacco cells than in monocot plants (Topfer R; Maas C;<br>
35      Horicke-Grandpierre C; Schell J; Steinbiss H.H. Methods Enzymol. 1993, 217:67-78;<br><br>
3<br>
Mitsuhara I; Ugaki M; Hirochika H; Ohshima M; Murakami T; Gotoh Y; et al. Plant Cell Physiol. 1996, 37:49-59). Similarly, rice actin-1 and maize ubiquitin-1 promoters are highly efficient promoting transcription of its downstream genes in monocot plant cells, but its promoter activity in tobacco cells is low (Schledzewski K; Mendel R.R. Transgenic S     Research 1994, 3:249455).<br>
In order to increase heterologous protein expression levels in transgenic plants, a variety of chimerical promoters where natural promoters are combined with transcription or translation enhancers have been designed. Among these enhancer elements we can mention Omega translational enhancer from Tobacco Mosaic Virus (TMV) (Gallie D.R;<br>
10 Sleat D.E; Watts J.W; Turner P.C; Wilson T.M.A. Nucleic Acids Res. 1987, 15:3257-3273), translational enhancer from Tobacco Etch Virus (TEV) (Carrington J.C; Freed D.D. J. Virol. 1990, 64:1590-1597), promoter transcriptional enhancers from octopine synthase (Fromm H; Katagiri F; Chua N.H. Plant Cell 1989, 1:977-984), mannopine synthase (Comai L; Moran P; Maslyar D. Plant Mol Biol. 1990, 15:373-381) and CaMV 35S<br>
15 promoter (Kay R; Chan A; Daly M; McPherson J. Science 1987, 236:1299-1302); and natural exons and introns, e.g., maize alcohol dehydrogenase intron 1 (Callis J; Fromm M; Walbot V. Genes Devel. 1987, 1:1183-1200; Last D.I; Brettell R.I.S; Chamberlaine D.A; Chaudhury A.M; Larkin P.J; et al. Theor Appl. Gen. 1991, 81:581-588), the first exon/intron from maize sucrose synthase (Maas C; Laufs J; Grant S; Korfhage C; Werr W.<br>
20 Plant Mol. Biol. 1991, 16:199-207), the first exon/intron from rice Actin-1 gene (McElroy D; Blowers A.D; Jenes B; Wu R. Mol. Gen. Genet. 1991, 231:150-160), etc. That gave rise to promoters like 2X35S, Mac, Emu and others (EP0459643; EP0651812), which are strong mainly in a specific class of plant cells, dicots or monocots. (Schledzewski K; Mendel R.R. Transgenic Research 1994, 3:249-255).<br>
25 The development of strong promoters which can be employed to express genes in both dicot and monocot plant cells has been and is a relevant topic of many laboratories, not just for the scientific challenge it represents or the savings that implies to have a unique genetic construction to transform diverse classes of plants, but also to have their own expression systems, which make easier biotechnological products production and commercialization.<br>
30 The synthetic promoter claimed in the patent application W09943838 claimed in a sequence from the TATA box till the transcription initiation site with an elevated GC content (64 % or higher), fused in its 5' end to transcription enhancer sequences from 35S, maize ubiquitin-1 and octopine synthase promoters. In the other hand, in order to look for expression not only in dicot and monocot plants, but also avoiding sequence homologous-<br><br>
4<br>
dependent gene silencing (Park Y.D; Papp I; Moscone E; Iglesias V; Vaucheret H; Matzke A; Matzke M.A. Plant J. 1996, 9:183-194), patent application WO0058485 claims an artificial promoter derived from the combination of sequences coming from two plant viruses genomes: Commelina Yellow Mottle Virus (CoYMV) and Cassava Vein Mossaic 5     Virus (CsVMV), and also enhancer sequences from 35S promoter.<br>
Mechanisms permitting that different genetic elements enhance transcription or translation of nucleotide sequences are not still clear. For example, it has been reported that leader sequences from many RNA viruses can enhance translation of different messenger RNAs (mRNA), independently of the presence of cap (m7G (5') ppp (5') N) fused to the 5'end<br>
10 (Sleat D.E; Wilson T.M.A. 1992. Plant virus genomes as sources of novel functions for genetic manipulations. In: T.M.A. Wilson &amp; J.W. Davies (Eds), Genetic engineering with plant viruses. CRC Press, Inc. p.55-113; Gallie D.R; Sleat D.E; Watts J.W; Turner P.C; Wilson T.M. Nucleic Acids Res. 1987,15:8693-8711). However, with the exception of the RNA secondary structure of all these viral leaders is not complex, is not determined<br>
15 another common element between its nucleotide sequences responding for its translational enhancer properties.<br>
Particularly, it has been reported that translation enhancement of TMV Omega fragment is due to the presence of at least one copy of the octamer ACATTTAC, which is repeated in its sequence, and a 25-base (CAA)n region that is considered a critic motif (two copies of<br>
20 (CAA)n region are enough to confer high enhancer levels) (Gallie D.R; Walbot V. Nucleic Acids Res. 1992, 20:4631-4638). However, a 28-base, CA-rich region from Potato X Virus (PVX) leader, have not shown translation enhancer activity "per se" (Pooggin M.M; Skryabin K.G. Mol. Gen. Genet. 1992, 234:329-331), while it is reported that CCACC pentanucleotide present in the CA region of PVX leader, might have pairing interactions<br>
25      with the 3'end of 18S rRNA (Tomashevskaya O.L; Solovyev A.G; Karpova O.V; Fedorkin O.N; Rodionova N.P; Morozov S.Y; Atabekov J.G. J. Gen. Virol. 1993, 74:2717-2724) It has been determined that some viral leaders have sequence elements involved in translation enhancer activities,  such as the CCTTTAGGTT sequence conserved in carlavirus leaders like Potato Virus S (PVS) (Turner R; Bate N; Tewell D; Foster G.D.<br>
30 Arch. Virol. 1994, 134:321-333) and the so-called internal control regions type 2 (ICR2) motif (GGTTCGANTCC), which is found in 27-base repeated regions in the RNA3 leader of Alfalfa Mosaic Virus (A1MV), needing two to reach optimal translation levels (van der Vossen E.A.G; Neeleman L; Bol J.F. Nucleic Acids Res. 1993, 21:1361-1367).<br><br>
3<br>
In the case of the TEV leader two regions called CIRE-1 and CIRE-2, between nucleotides 28 to 65 and 66 to 118, respectively, have been identified as responsible for the translation enhancement properties of this 148 bp viral leader (Niepel M; Gallie D.R. J. Virol. 1999, 73:9080-9088). However, inside ORE regions it has not been defined an specific element 5      considered critic for the enhancer activity of these viral leader.<br>
As we referred above, introns of natural origin and its adjacent sequences have been also widely employed to enhance different gene expression systems, especially when the intron is near of the 5'end of the gene. However, an intron-mediated enhancement of gene expression (IME), depending on factors like intron origin, exonic flanking regions and<br>
10 cellular type, has been reported. A strong IME of the expression has been observed mainly in monocot plant cells, but in dicots it commonly does not exceed 2 to 5-fold. Molecular mechanisms behind IME have not been completely disclosed (Simpson G.G; Filipowicz W. Plant Mol. Biol. 1996, 32:1-41; Schuler M.A. 1998. Plant pre-MRNA splicing. In: J. Bailey-Serres &amp; D.R. Gallie (Eds), A look beyond transcription: mechanisms determining<br>
15 MRNA stability and translation in plants. American Society of Plant Physiologists. P. 1-19; Lorkovic Z.J; Kirk D.A.W; Lambermon M.H.L; Filipowicz W. Trends in Plant Science. 2000, 5:160-167).<br>
IME expression variations observed between monocot and dicot plant cells can be due to the known differences in the requirements needed for an adequate pre-mRNA processing<br>
20 in different classes plants cells. In fact, in monocot plant cells, but not in dicot plant cells, the presence of AU-rich segments in the intron sequence is not indispensable to its processing; monocot cells can process introns with high GC-content (more than 50 %) and complex secondary structures (hairpin-loops), which indicates that dicot plant cells are unable to process introns with complex secondary structures (Goodall G.J; Filipowicz W.<br>
25 The EMBO Journal 1991, 10:2635-2644; Lorkovic Z.J; Kirk D.A.W; Lambermon M.H.L; Filipowicz W. Trends in Plant Science. 2000, 5:160-167). These reasons can explain, at least partially, why current systems employing IME to artificially enhance nucleotide sequence expression, are class-specific.<br>
30     Statement of Invention<br>
The present invention relates to novel promoter and its expression vectors. These promoters have been developed to function as "universal promoters" for any biological system for transformation of useful genes, proteins, peptides etc. The application of these promoters can be extended even to development of novel vaccines, etc. Till date the<br><br>
6<br>
developed promoters have had major drawbacks in becoming unstable over a period of time in biological systems, as they became inefficient over a period of time or were lost from the biological systems. The construction of the presently developed promoters is so unique, that they do not loose there efficiency at any point of time in the biological 5 systems. The vectors are designed by studying in-dept attributes of different sequences; and further modifying and aligning these sequences in specific order to achieve the unexpected results from these vectors. The present study has taken the inventors unprecedented number of years of hard work and extraordinary skills to achieve the results. It is after much experimentation and human endeavor the inventors arrived at such<br>
10 an invention, which could overcome the existing drawbacks of earlier known promoters. In other words, the use of these promoters is non-obvious, novel and inventive use and not conventional use. It is a use, which a person skilled in art has arrived after conscientious experimental planning and much of human interference and thus is not a mere known use. The very novelty lies in selecting, designing and combining the appropriate sequences to<br>
15 achieve the distinctiveness in developed vectors. In fact any other person skilled in this art will have high appreciation for such a high quality and exceptional degree of work.<br>
Detailed description of the invention<br>
The expression promoter sequence proposed in this patent application provides a set of 20 distinctive characteristics: 1) it is universally functional as it is active in dicot as well as in monocots plants cells, permitting the obtaining of transgenic plants of any class with high expression levels of the genes and DNA sequences of interest; 2) it is based on the combination of artificially assembled genetic elements, increasing mRNA levels not only by IME, but also promoting its translation; 3) the lack of long DNA fragments with 25 identical sequence to natural or viral genes in this promoter minimize the risk of RNA-mediated homologous gene silencing and the possibility of the appearance of new viral races or strains as a result of in planta homologous recombination; 4) the GC content of the sequence from the TATA box to the transcription initiation site must not necessarily to be high; 5) the versatility of our promoter sequence permits to insert in its 5' end 30 transcription regulatory elements, which confers temporal, organ or tissue-specificity to the expression; 6) artificial genetic elements comprising it can be also functionally inserted between any promoter active in plant cells and any DNA sequence to increase its transcription/translation.<br><br>
7<br>
Chimerical exon/intron/exon region design with a high enhancement activity of mRNA accumulation in any class of plant cells, and its functional integration with an artificial translation enhancer constitute two essential components of the current patent application, because these elements permits us to efficiently express any DNA sequence of interest in 5     plant cells.<br>
It is important to clear that when we call a region, molecule or DNA sequence is artificial or chimerical, we are referring to its designed and synthesized in vitro, thus it is not any genetic element with identical primary DNA structure, although small fragments of its sequence could have a natural origin.<br>
10 To design an intron with its correspondent adjacent exonic sequences, able to promote IME of expression, we studied which sequence motifs and genetic components were common to plant introns with reported transcription enhancer activity. At the same time, we had to resolve the challenge of achieving an adequate and efficient processing of this intron in dicot and monocot plants, independently of its GC content.<br>
15 When the widely used as transcription enhancers promoters rice Actin-1, maize ubiquitin-1 and sucrose synthase-1 intron sequences are compared, it can be detected common and repeated sequence motifs in all of them (Figure 1). There is not demonstrated the responsibility of any of these motifs for the IME of gene expression these introns confers, but high conservation levels of the CTCC motif (or its homologous sequences CTC, TCC<br>
20 and TC) in these regions and in the 5' regions of the TATA boxes in a variety of plant promoters, indicates the possibility that it can favor the binding of transcriptional factors, which can promote RNA-polymerase II activity. At the same time, C and A-rich sequences abundance and conservation in the first exons (regions remaining as non translational mRNA untranslated leaders) and in the viral leaders with reported translational enhancer<br>
25 activity, indicates that such sequences can promote the stability of the resulting MRNA and its ability to be translated.<br>
It is appropriated to emphasize that none of the above explained theories has a complete scientific demonstration, thus it is not obvious that the construction of an artificial intron with its adjacent exonic sequences, containing the repeated sequence motifs specified,<br>
30 results in a region that promotes high transcription levels and accumulation of mRNA; however, the results of our work indicates this.<br>
From the mentioned comparison study between the different introns (with its correspondent exons), we decided to design an artificial exon/intron/exon region, which combines rice actin-1 and maize ubiquitine-1 intron/exon sequence fragments rich in the<br><br>
8<br>
motifs that we consider relevant of the IME of gene expression. In order to achieve this goal, we had to take into account that the resulting artificial intron must be efficiently processed in dicot plat cells, so the increase of gene expression can take place in these class of plants too. Nevertheless, we find out that the intron sequences we used as prime 5 material have a high GC content, complex secondary structures with abundant hairpin-loops, and the sequence of its AG acceptor 3' splicing site is some different from the branch point consensus sequence, thus these introns can be difficultly processed in dicot plant cells. In order to simplify the secondary structure of the exon/intron/exon designed by us, so it<br>
10 can be processed in any class of plant cells, we decided to make some punctual changes in its sequence and to insert sequences UUUUUAU-like, which activates its processing (Gniadkowski M; Hemmings-Mieszczak M; Klahre U; Liu H.X; Filipowicz W. Nucleic Acids Res. 1996, 24:619-627). Besides, our chimerical sequence was fused to the second exon and inserted into maize actin-1 gene second intron (IVS2), taking advantage of its<br>
15 efficient processing in dicot (e.g. tobacco) (Goodall G.J; Filipowicz W. The EMBO Journal 1991, 10:2635-2644). The putative secondary structure of each artificial exon/intron/exon variant was studied by computing methods, using the PCFOLD 4.0 program (Zuker M. Meth. Enzymology 1989, 180:262-288). The artificial exon/intron/exon sequence created for us will be called ART from this point.<br>
20 As it was already mentioned, the second relevant component of this patent application is an artificial translational enhancer that was fused downstream to chimerical exon/intron in order to increase gene expression levels.<br>
The artificial translational enhancer was designed from analysis of sequence and secondary structure formed on some viral leaders. From this analysis we conclude that there are three<br>
25 essential elements in the translational enhancers: 1) low complexity of secondary structures; 2) C and A-rich sequence segments; 3) motifs with up to 83% of homology with the consensus HCAYYY sequence (H= C 6 U 6 A; Y= C 6 U, see Table 1), exposed and frequently repeated and/or in the hairpin of structures with a low melting point tails.<br>
30 Table 1. Structurally conserved sequences in some RNA virus leader fragments (H=C/U/A; Y=C/U).<br><br>
Viral leader	Sequence<br>
TMV	ACAUUUAC<br>
TEV(CIRE-l)	GCAUUCUA<br>
TEV (CIRE-2)	UCAUUUCU<br><br>
9<br><br>
PVS	ACCUUUAG<br>
A1MV(RNA3)	UAAUUCG<br>
A1MV(RNA3)	ACUUUUC<br>
PVX	CCAAUUG<br>
BMV	AACAUCGG<br>
RSV	CCAUUCA<br>
Consensus	HCAYYY<br>
From the premises above pointed out, we designed an artificial translational enhancer in which sequences HCAYYY-like, each one in hairpin-loop structures, were inserted inside a 45-base, C and A-rich sequence. This artificial translational enhancer have no more than 5 55 % of homology with RNA viral leaders which provided theoretical premises employed in its confection, and there is no even a sequence segment with more than 6 nucleotides with 100% of homology. That is why we can affirm that our translational enhancer is not a derivative of any translational enhancer previously reported or protected (EP 0270611), neither has sequences directly derived from them.<br>
10 In order to make easier its manipulation and the fusion of genes of interest, restriction sites were added to our translational enhancer. Finally, before to fuse the new translational enhancer to the artificial Exon/Intron we created, its functionality was in vivo, showing the same capacity to enhance expression of a chimerical gene when compared with TMV Omega fragment. The artificial translational enhancer created for us will be called Eureka.<br>
15      (Figure 2).<br>
To construct the promoter sequence claimed in this patent, a core promoter formed by a consensus TATA box (Joshi C.P. Nucleic Acids Res. 1987, 15:6643-6653) was firstly designed which was fused to CaMV 35 S -24 to -4 region (from the transcription initiation site), followed by actin-1 -5 to +27 promoter region, which provides the transcription<br>
20 initiation site and a C and A-rich region. The maize ubiquitine-1 region, from +26 to +72 from the transcription initiation site, was fused downstream, providing AC- and TC-rich regions, yielding the first artificial exon, which was linked to maize actin-1 second exon, 12 bases before the 5' splicing site of its IVS2 intron and including itself. Bases change, adding or deletions were made around this joining according with the predictions the<br>
25 computing method in order to avoid putative secondary structures, which can affect RNA maturation. The artificial intron designed for us, is constituted by the first 54 bases of IVS2 intron, fused to 37 bases from a 5' region of the maize ubiquitine-1 first intron, corresponding to the bases +89 to + 126 from the transcription initiation site, followed by 375 bases from rice actin-1 first intron (from the position +103 to +477, from this gene<br><br>
10<br>
transcription initiation site), fused to 33 bases from maize ubiquitine-1 intron 3' end (from the position +1051 to 1083 from the transcription initiation site), linked to the second half of the actin-1 IVS2 intron (from the position -52 to +5 from its 3' processing initiation site), and to a 29-base chimerical sequence containing restriction sites and a translation 5 initiation consensus sequence. (LUtcke H.A; Chow K.C; Mickel F.S; Moss K.A; Kern H.F; Scheele G.A. The EMBO Journal. 1987, 6:43-48). The sequence of the artificial exon/intron/exon ART created for us is shown in figure 3. Once the efficient processing of the artificial exon/intron/exon constructed was tested by transient expression in both tobacco and rice cells, the translational enhancer Eureka was  fused to its 3'end,<br>
10 appreciating in figure 4 the final structure of the promoter sequence object of this invention (PARTE promoter).<br>
It must be highlighted that the enhancer element ART designed by us showed a higher efficiency as gene expression enhancer than the commonly used rice actin-1 gene first exon/intron/exon; Eureka fragment is an additional enhancer to its activity.<br>
15 In this work it can be for the first time achieved two artificial, very efficient genetic elements, enhancing the expression of any DNA sequence in transgenic plant cells of any class, which demonstrate the validity of the theoretical precepts we are based on. It is also for the first time that an artificial promoter with an AT content lower than 52 % is adequately processed in dicot plant cells, promoting a high IME of the expression. The<br>
20 construction of a completely artificial, highly efficient translational enhancer, with low homology with ARN viral leaders, is also novel.<br>
Different transcription enhancer sequences were fused 5' to the promoter sequence object of this invention. Thus, rice actin-1 region from -43 to -310 (from the transcription initiation site) was fused to the 5' end of the promoter PARTE, as shown in figure 5, to<br>
25 form the promoter region APARTE, which also has as-1-like transcription enhancer elements (Benfey P.N; Chua N.H. Science, 1990, 250:959-966) and a 556-base fragment from the 5' region of the maize ubiquitin-1 promoter (from -299 to -855 from the transcription initiation site), to finally obtain the U3ARTE promoter, which structure is showed in Figure 6.<br>
30 Many promoter sequence variants were constructed from the described genetic elements (see figure 7), and all of they demonstrate its functionality by the means of in vivo assays, proving the synergic effect over the gene expression all the enhancer and activator regions employed.<br><br>
11<br>
The as-\ element employed in our constructions as a transcription enhancer (see fig 6) has a innovative design, because it has less than 50 % of homology with the octopine synthase palindromic enhancer (Ellis J.G; Llewellyn DJ; Walker J.C; Dennis E.S; Peacock W.J. EMBO J. 1987, 6:3203-3208; EP0278659), is not identical (less than 85 % of identity) to 5 any of this type of sequence variants claimed in a study done by Ellis et al (Bouchez D; Tokuhisa J.G; Llewellyn D.J; Dennis E.S; Ellis J.G. EMBO J. 1989, 8:4197-4204; USPat. 5,837,849) and the TGACG motifs found in it are found in an unique flank sequence context. Although the rice actin-1 gene has been described and used to express different genes<br>
10 (McElroy D; Zhang W; Cao J; Wu R. Plant Cell 1990, 2:163-171; WO9109948), it is important to emphasize that our work reveals the transcription enhancer activity of its 5' region and its use as heterologous expression enhancer. Similarly, although the use of the maize ubiquitine-1 5' transcription enhancer region has been claimed (EP0342926), the 556-base fragment employed for us does not contains the "heat shock" elements defined as<br>
15 essentials for this enhancer functionality (it is found between the positions -188 to -214 of this promoter sequence), thus it is original and does not obviate the transcription enhancer activity of the ubiquitine sequence used by us.<br>
The PARTE promoter was also fused to a small, 214-base fragment corresponding to the -31 to -245 region from the transcription initiation site of the rice gluB-\ gene (Takaiwa F;<br>
20     Oono K; Kato A. Plant Mol. Biol. 1991, 16:49-58) to form the new promoter region GARTE (fig 8) Transient expression assays demonstrate that the new artificial promoter GARTE is highly efficient to express DNA sequences in seeds endosperm. Thus, we conclude that functional combination of these chimerical 5' transcription enhancers has novelty, and greatly useful to produce genetic elements allowing the<br>
25 achievement of high expression levels of DNA sequences in plant cells, independently of the class they belong to.<br>
Obviously, other 5'regulator regions from different promoters can be eys-fused to the object of this invention in order to achieve high expression levels and/or to confer temporal, organ or tissue specificity to the expression.<br>
30 For somebody experienced in genetic engineering techniques, what is here described would make possible the use ART and Eureka enhancers, in combination with any transcription promoter element in plant cells, to enhance transcription/translation of any DNA sequence in monocot or dicot plant cells. Similarly, theoretical precepts obtained from our observations, leading us to the design of ART and Eureka, can be employed by<br><br>
12<br>
someone with experience in molecular biology techniques to construct new DNA expression enhancer sequences in plant cells.<br>
Considering the rising development of plant genetic engineering in the last two decades, it is obvious that the promoter object of this invention, joined to any gene and a transcription 5 terminator sequence, can be inserted in a plant cell genetic transformation vector, and by the means of the use of well-established, efficient techniques, obtaining transgenic plants able to express the gene of interest.<br>
In this patent application, a genetic transformation vector refers to a DNA molecule (purified or contained inside a bacterial cell or a virus), which serves as a carrying vehicle<br>
10     to introduce in a plant cell any DNA fragment previously inserted in it.<br>
The present invention relates to novel promoter and its expression vectors. These promoters have been developed to function as "universal promoters" for any biological system for transformation of useful genes, proteins, peptides etc. The application of these promoters can be extended even to development of novel vaccines, etc. Till date the<br>
15 developed promoters have had major drawbacks in becoming unstable over a period of time in biological systems, as they became inefficient over a period of time or were lost from the biological systems. The construction of the presently developed promoters is so unique, that they do not loose there efficiency at any point of time in the biological systems. The vectors are designed by studying in-dept attributes of different sequences;<br>
20 and further modifying and aligning these sequences in specific order to achieve the unexpected results from these vectors. The present study has taken the inventors unprecedented number of years of hard work and extraordinary skills to achieve the results. It is after much experimentation and human endeavor the inventors arrived at such an invention, which could overcome the existing drawbacks of earlier known promoters.<br>
25 In other words, the use of these promoters is non-obvious, novel and inventive use and not conventional use. It is a use, which a person skilled in art has arrived after conscientious experimental planning and much of human interference and thus is not a mere known use. The very novelty lies in selecting, designing and combining the appropriate sequences to achieve the distinctiveness in developed vectors. In fact any other person skilled in this art<br>
30     will have high appreciation for such a high quality and exceptional degree of work.<br>
Accordingly, the main embodiment of the present invention provides a promoter comprising of:<br><br>
13<br>
(a)	A core promoter,<br>
(b)	5' transcription regulator element, and<br>
(c)	An enhancer element<br>
5     Another embodiment of the present invention provides a core promoter comprises of TATA box and CaMV 35 S promoter.<br>
Still another embodiment of the present invention provides a 5' transcription regulator element comprises having SEQ ID NO. 6 and 8.<br>
10<br>
One more embodiment of the present invention provides an enhancer element is having<br>
SEQ ID No. 1.<br>
One more embodiment of the present invention provides SEQ ID Nos. 6 and 8 wherein 15      said sequences comprises of artificial exon/intron/exon regions have SEQ ID Nos. 10, 11 and 19.<br>
Another embodiment of the present invention provides a promoter wherein the promoter<br>
controls the gene expression in plants.<br>
20<br>
Still another embodiment of the present invention provides a promoter wherein the said<br>
promoter expresses under biotic stress, abiotic stress, wounding, in seed endosperm, and<br>
other environmentally and artificial induced conditions.<br>
25 In another embodiment of the present invention provides an promoter wherein said promoter expresses both in dicots and monocot plants.<br>
One more embodiment of the present invention provides an artificial promoter,wherein<br>
first and second exon region comprises of motifs with high C and A content.<br>
30<br>
Another embodiment of the present invention provides an artificial promoter, wherein<br>
second exon comprises of at least 83% homology with motif HCAYYY (H=C or T or A ;<br>
Y= C or T)<br>
35 Still another embodiment of the present invention provides an artificial promoter, wherein first exon and intron region comprises of sequences repeated with motif CTCC and /or homologous sequences CTC, TCC and TC.<br><br>
14<br>
One more embodiment of the present invention provides an artificial promoter, wherein<br>
the said promoter expresses in all parts/organs/cells/tissues of both monocot and dicot<br>
plants.<br>
5<br>
Another embodiment of the present invention provides an artificial promoter, wherein the<br>
said promoter is expressed in plasmid vector system selected from group comprising of<br>
Eureka-pBS, pBS-AcUc, pBSAcAcUc, pBS-ART, pPARTE, pBS-EURGUSint, pBPF-<br>
EURGGUSint, pBPFa7-GUSint, pBPFATGUSint, pBPFARTEGUSint, pC-ARTGUSint,<br>
10     pC-ARTEGUSint,      pBPFA19ARTGUSint,     pBPFA19ARTEGUSint,      pAlPARTE,<br>
pASPAlPARTE,  pASPAPARTE,     p2Al PARTE,  pBS-Ubil,  pBS-Ubi2,  pU3ARTE,<br>
pGARTE,   pC-APARTEGUSint,   pC-2AlPARTEGUSint,   pC-2APARTEGUSint,   pC-<br>
U3ARTEGUSint, pC-ACTIF, pGARTEGUSint, and pGluGUSint.<br>
15      Another embodiment of the present invention provides a plasmid vectors, wherein said plasmid vectors systems express both in monocot and dicot plants.<br>
One more embodiment of the present invention provides a plasmid vectors wherein said plasmid vectors express in all the parts, tissues, cells, organs of both monocot and dicot 20     plants.<br>
Still another embodiment of the present invention provides a plasimd vectors, wherein said<br>
plasmid vectors expresses under biotic stress, abiotic stress, wounding, in seed endosperm,<br>
and other environmentally and artificial induced conditions.<br>
25<br>
Another embodiment of the present invention provides a process of preparing an artificial<br>
promoter, said method comprising the steps of:<br>
(a) isolating the core promoter TATA and CaMV,<br>
30	(b) designing and creating 5'translational regulator element having SEQ ID<br>
Nos. NO. 6 and 8,<br>
(c)	assembling the sequences of step (a) and (b) in a vector system,<br>
(d)	designing and assembling a enhancer having SEQ ID No. 1 in a vector system,<br>
35	(e) constructing a promoter by isolating sequences of step (c) and (d),<br>
(f) inserting the assembly obtained in step (e) into vector system, and<br><br>
15<br>
(g) studying the expression of artificial promoter of step (f) by inserting in various expression vector systems.<br>
Description of the drawings.<br>
5      Figure 1. Rice Actin-1 (Act), maize ubiquitine-1 (Ubi) and maize sucrose synthase (Shrun) gene sequences from the transcription initiation site. In uppercase is shown the first exon and in lowercase the first intron 5' region and the localization of the repeated and common sequence motifs are underlined. Figure 2. Eureka artificial translational enhancer sequence, where its relevant elements and<br>
10     restriction endonucleases recognition sites are shown.<br>
Figure 3. ART Exon/Intron/Exon artificial sequence, showing the origin of each of its component fragments (lowercase: artificial intron; the bases inserted to create UUUUUAU-like sequences are double-underlined; simply underlined are marked some relevant recognition sites for restriction endonucleases).<br>
15 Figure 4. Primary structure of this invention object (PARTE promoter), showing the core promoter (lowercase italic) fused to the ART Exon/Intron/Exon region (intron bases in lowercase, exon's in uppercase) and to the artificial translational enhancer EUREKA. Some relevant recognition sites for restriction endonucleases are underlined; TATA box is double-underlined and translation initiation codon is in bold.<br>
20 Figure 5. Primary structure of the APARTE promoter, showing rice actin-1 5' regulatory region (region from -43 to -310 from the transcription initiation site, in italics uppercase) fused to PARTE promoter (in italics lowercase the promoter, in lowercase the intron; in uppercase the exons). Underlined are marked some relevant recognition sites for restriction endonucleases; TATA box is double-underlined and the translation initiation codon is in<br>
25     bold.<br>
Figure 6. Primary structure of the U3 ARTE promoter, showing its component elements: -299 to -855 region from the maize ubi-l gene transcription initiation site, in uppercase; as-1 -like transcription enhancer, in bold uppercase; region from -43 to -310 from the transcription initiation site of the rice act-l gene, in italics uppercase; PARTE promoter in<br>
30 lowercase (TATA box is double underlined, ART intron is in italics and the translation initiation site is simple underlined).<br><br>
16<br><br>
2APARTE; H: U3ARTE.E ART Exon/Intron/Exon;<br>
Figure 7. Promoter variants with the enhancer elements object of this invention. A: 35SEureka; B: 35SART;_C: 35SARTE; D: PARTE; E: APARTE; F: 2A1PARTE; G:<br>
35S Promoter (1.3Kb)jfiJ Translational enhancer Eureka;<br>
33 artificial promoter core; |	| rice actin-1 gene 5' activation<br>
5      region (-43 a -221); [jj rice actin-1 gene 5'activation region (-226 a -310); M ASP (as-l-like enhancer); H| maize ubiquitine-1 promoter 5'activation region (-299 a -855). Figure 8. Primary structure of GARTE promoter, showing its component elements: rice g/wB-1 gene region from -31 to -245 from the transcription initiation site, in italics uppercase; PARTE promoter (promoter is in italics lowercase, intron in lowercase; exons<br>
10     are in uppercase; some relevant restriction sites are underlined; TATA-box is double underlined; translation initiation codon is in bold). Figure 9. pUC-GUSint map. Figure 10. pBPFQ (omega) 7 map. Figure 11. pBPFA19-linker map.<br>
15      Figure 12. Comparative demonstration by X-Gluc hystochemical dyeing of ART and EUREKA elements functionality in rice cells by the means of transient expression of different  genetic   constructions  harboring  GUSint  gene,   introduced  by  accelerated microprojectils bombardment. Microorganism Deposits<br>
20 The plasmids pC-EURGUSint; pC-ARTEGUSint; pGARTEGUSint y pC-U3ARTEGUSint were deposited under the Budapest treated for the protection of Microorganisms in the Belgian Coordinated Collection of Microorganism, Plasmid Collection (BCCM/LMBP) Universiteit Gent, Tiers-Schell-Van Montagu' building, Technologiepark 927, B-9052 Gent-Zwijnaarde, Belgica. Plasmids pC-EURGUSint; pC-<br>
25 ARTEGUSint; pGARTEGUSint with the access numbers LMBP 4727; LMBP 4725; LMBP 4728 respectively and with the date May 19, 2003 and pC-U3ARTEGUSint with the number 4791 of November 25, 2003.<br>
EXAMPLES 30     Example No.l: Construction of the constituting elements of a new chimerical system for the expression of DNA sequences in plant cells<br>
All the synthesized DNA fragments were created with sticking ends to different type-II restriction endonucleases restriction sites in order to make easier its correct cloning.<br><br>
17<br>
a)	Eureka translational enhancer cloning.<br>
The 86 base pairs (bp) DNA fragment corresponding to translational enhancer EUREKA (SEQ ID NO: 1), was cloned into the vector pBluescript II SK (Stratagene, USA) previously digested with the restriction enzymes Pst I and Sac I, taking advantage of the 5 sticking ends for both enzymes included in the design of the synthetic DNA fragment. The resulting plasmid was named pBS-Eureka.<br>
b)	Assemble of the artificial Exon/Intron/Exon region ART.<br>
The artificial Exon/Intron/Exon region ART was constructed by cloning, assembling, one behind the other, DNA fragments designed. Firstly, the DNA synthetic fragment named<br>
10 P35AcU (SEQ ID NO: 2), which contains the core promoter, the first Exon and part of the artificial Intron, was cloned into the pBluescript II SK vector digested with Eco RI and Spe I restriction enzymes to obtain the plasmid pBS-AcU. After that, such plasmid was digested with Spe I and Sac I, inserting in it the DNA synthetic fragment I-U/Ac (SEQ ID NO: 3), which codes for part of the artificial intron. That is the way the pBS-AcUAc<br>
15     plasmid was obtained.<br>
Next, the DNA synthetic fragment I-Ac/U (SEQ ID NO: 4), harboring the end of the artificial Intron, was inserted into the pBS-AcUAc plasmid digested with the restriction enzymes Bam HI and Sac I, to produce the plasmid pBS-AcUAcU. When the fragment IniT (SEQ ID NO: 5) was inserted into the Spel I Sad-digested pBS-<br>
20 AcUAcU, the artificial Exon/Intron/Exon ART was completed (SEQ ID NO: 6), conforming the plasmid pBS-ART, which primary structure between the restriction sites EcoBI and Sacl of the pBluescript II vector is shown in the sequence SEQ ID NO: 7.<br>
c)	PARTE promoter construction.<br>
To construct the promoter sequence object of this invention (PARTE promoter), the DNA 25      fragment containing the core promoter and the Exon/Intron/Exon region ART (without its<br>
3' region) was obtained from pBS-ART plasmid by an Xhol I Pstl digestion, inserting it<br>
into the pBS-Eureka plasmid digested with the same enzymes. Thus, we achieved the<br>
plasmid pPARTE (Figure 4, Figure 7D), which sequence between the EcoBJ and Sacl sites<br>
is shown in sequence SEQ ID NO: 9. 30     Example No.2: Demonstration of Eureka and ART enhancer elements functionality<br>
in plant cells.<br>
a)  Translational enhancer Eureka functionality in tobacco cells.<br>
To verify the enhancer power of Eureka in tobacco and rice cells, a series of auxiliary<br>
genetic constructions was made.<br><br>
18<br>
The reporter gene uidh with potato ST-LS1 gene IV2 intron inserted into the Sna BI site (GUSint), was obtained by an Nco I / Sac I digestion of plasmid pUC-GUSint (Figure 9) and cloned in the same sites of plasmid pBS-Eureka, giving rise to the vector Pbs-EURGUSint. This one was further digested with the restriction enzymes Pstl, Sail and 5 treated with S-l Nuclease to obtain plasmid pBS-AEURGUSint; which was Xho I / Kpn I digested to obtain a DNA fragment containing the Eureka enhancer fused to GUSint gene, that was inserted in the pBPFfi (omega) 7 vector (Figure 10). Thus we obtained the pBPF-EURGUSint vector (Figure 7A), having GUSint gene expression under the control of CaMV 35S promoter (1.3 kb version), Eureka enhancer and Agrobacterium tumefaciens<br>
10     nos gene transcription termination signals (tNOS).<br>
As a control to evaluate the expression of the construction pBPF-EURGUSint, the plasmid pBPFQ(omega)-GUSint was constructed, cloning GUSint gene, obtained from the plasmid pUC-GUSint by a Sail I Klenow and Kpnl digestion, between the pBPFfi (omega) 7 vector Smal and Kpnl sites. This plasmid is similar to pBPF-EURGUSint except for the<br>
15     presence of the translational enhancer Q (omega) controlling GUSint instead of Eureka. Another control plasmid was constructed by eliminating the enhancer omega of plasmid pBPFQ(omega)-GUSint by Xho I - Nco I digestion, treatment with Klenow and plasmid self-ligation, obtaining pBPF-GUSint vector. Plasmids pBPFQ(omega)-GUSint, pBPF -GUSint, y pBPF-EURGUSint were Hindlll<br>
20 digested to obtain the cassettes for GUSint expression in plants; these were cloned into //mt/III-digested binary vector pCAMBIA2300, giving rise to binary vectors pC-Q(omega)7GUSint, pC-GUSint y pC-EURGUSint, respectively.<br>
Binary plasmids obtained were introduced into A. tumefaciens strain LBA4404, we proceed to assay functionality of the enhancer Eureka by the means of a transient<br>
25 expression experiment in NTl tobacco cells, following the protocol described by An et al (An G. Plant Physiol. 1985, 79:568-570) with some modifications. After four days co-culturing tobacco cells with Agrobacterium carrying each of the binary vectors, the cells were collected and processed as described by Jefferson (Jefferson R.A. 1988. Plant reporter genes: the GUS gene fusion system. In: J.K. Setlow (Ed), Genetic Engineering.<br>
30 Vol.10, Plenum Publishing Corporation. P.247-263) to determine its P-glucuronidase (GUS) activity. Each experiment was repeated three times, with 5 replicas per construction each time. The results are shown in the following table.<br><br>
19<br>
Table 2. Demonstration of the functionality of the translational enhancer Eureka in tobacco cells.<br><br>
Experiment	GUS Activity (Pm 4-1	MU/min/mg total proteins)	Eureka / Q(omega)rate	Eureka/Q(omega)rate media<br>
	Cell control	pC-GUSint	pC-Q(omega) 7GUSint	PC-EURGUSint		<br>
I	0.31+0.01	1.93+1.17	7.1512.26	7.50+2.60	1.04	1.0010.33<br>
II	1.1010.28	2.1110.18	8.5611.60	11.2212.80	1.31	<br>
III	0.79+0.19	4.8411.66	33.213.6	21.116.1	0.64	<br>
5 As it can be seen in results showed in Table 2, there are not significant differences between enhancer activities in Eureka when compared with that of TMV Omega leader sequence, which demonstrate that it has been for the first time achieved a completely artificial, efficient translational enhancer. This also confirm our theoretical precepts, showing that it is possible to construct a genetic element with significant properties to enhance the<br>
10 translation of DNA sequences fused downstream in 3' direction end by combining regions of sequences rich in C and A with sequences homologous to the motif HCAYYY (H=C/T/A; Y=C/T).<br>
b)  Functionality of the artificial Exon/Intron/Exon ART in tobacco cells. To test the functionality of the ART element in tobacco cells, it was firstly obtained the<br>
15 GUSint fragment from plasmid pUC-GUSint digesting it Nco I - Sac I, and it was cloned into pBS-ART digested with the same enzymes, to obtain plasmid pBS-ARTGUSint. Next, this plasmid was Sail - Bglll digested and treated with SI-Nuclease, obtaining pBS-AARTGUSint, which was digested Xhol - Kpnl to obtain the ARTGUSint fragment, and cloned  into  the  vector pBPFfi(omega)7-GUSint digested with the  same  enzymes,<br>
20     obtaining the plasmid pBPFARTGUSint (Figure 7B), where GUSint expression is under the control of CaMV 35S promoter (1.3 kb version), the artificial Exon/Intron/Exon region ART and A. tumefaciens nos gene transcription termination signals (tNOS). From the plasmid pBS-AARTGUSint was obtained by Xhol I Pstl digestion the band containing the element ART, which was fused into the pBS-EURGUSint vector digested<br>
25      with the same enzymes to obtain the plasmid pBS-ARTEGUSint. From this one was obtained by Xhol I Kpnl digestion a DNA fragment to get the GUSint gene under the signals of the genetic elements ART and Eureka, introduced into pBPFQ(omega)7 vector equally digested to produce pBPFARTEGUSint plasmid (Figure 7C). Plasmids pBPFARTGUSint and pBPFARTEGUSint were Hindlll digested to obtain<br>
30     cassettes for GUSint expression in plants; these were cloned into the binary vector<br><br>
20<br>
pCAMBIA2300,  resulting  in binary plasmids pC-ARTGUSint y pC-ARTEGUSint,<br>
respectively.<br>
Following   the   introduction   of the   binary  plasmids   obtained   into  Agrobacterium<br>
tumefaciens strain LBA 4404, we proceed to assay functionality of the translational<br>
5      enhancer Eureka in transient expression assays, using NTl tobacco cells as described in<br>
section (a) of this example. The results obtained are shown in Table 3.<br>
Table 3. Demonstration of functionality of the genetic elements ART and Eureka in tobacco cells.<br><br>
Experiment	GUS Activity (Pm 4-	VlU/min/mg total proteins)	35SART / 35S Rate	35SARTE/ 35S Rate<br>
	Cell Control	pC-GUSint	pC-ARTGUSint	pC-ARTEGUSint		<br>
I	1.13Â±0.27	4.39+0.96	5.26+1.69	26.313.5	1.2	6.0<br>
II	1.77Â±0.58	6.11Â±2.45	12.9212.36	32.016.1	2.1	5.2<br>
III	0.69Â±0.30	2.4610.77	3.9411.14	13.512.8	1.6	5.5<br>
MediaÂ±SD		1.6310.33	5.5710.29<br>
10<br>
Results of ART functionality evaluation in tobacco cells revealed that the artificial intron<br>
is correctly processed and possess expression enhancer activity in dicot plant cells. Our<br>
experimental results also showed that positive synergism on expression levels obtained<br>
form   the   interaction   between   ART   and   Eureka   elements   in   the   construction<br>
15 pBPFARTEGUSint, where there is an 5-fold increase of the expression ability of the known natural promoter CaMV 35S.<br>
It is also proved than the artificial genetic elements designed (ART and Eureka) can be functionally inserted between any promoter active in plant cells (CaMV 35S promoter, in this   case)   and   any   other   DNA   sequence   (GUSint   in   our   case)   increasing  its<br>
20     transcription/translation.<br>
c)  Functionality of the enhancer elements Eureka and ART in rice cells. A set of new constructions was made in order to prove functionality of our artificial enhancers in monocot plant cells. First, the Xhol - Kpnl fragment from the plasmid pBPFARTGUSint, harboring the uidA (gus) gene fused in its 5' end to the artificial<br>
25 Exon/Intron ART, was inserted into pBPFA19-linker vector (Figure 11) digested with the same restriction enzymes, to form the plasmid pBPFA19ARTGUSint. In this plasmid, rice actin-1 Exon/Intron/Exon region present in pBPFA19-linker has been substituted by the artificial element ART, remaining as the other regulatory elements the chimerical promoter A 19 (where quadruplicated octopine synthase as-1-like enhancer is fused to CaMV 35S<br>
30      promoter (400 bp version)) and the tNOS transcription terminator signal.<br><br>
21<br>
Similarly, the Xhol - Kpnl band from plasmid pBS-ARTEGUSint was cloned into the pBPFA19-linker vector to obtain the construction pBPFA19ARTEGUSint. A construction used as a control, pBPFA 19GUSint, was made by cloning GUSint fragment from plasmid pUC-GUSint into the Ncol I Sad digested pBPFA19-linker vector. Another control 5     plasmid used was pBPFfi(omega)-GUSint.<br>
Qualitative evaluation of the ability of ART and Eureka to enhance gene expression in rice cells was carried out by assaying transient expression in callus of variety indica Perla. Calli were obtained from mature seeds previously sterilized with sodium hipoclorite and alcohol, and cultured for 21 days in the dark in N6-2 media: N6 salts and vitamins (Chu<br>
10 C.C et al. Scientia Sinic 1975, 18:659); O.lg/L myo-inositol; 1 g/L casein hidrolisate; 2mg/L 2,4 D; 30g/L Sucrose; 3g/L Phytagel, Ph 5.7). The transformation was performed by micro-projectile bombardment: before the bombardment the calli were sub cultivated in N6-2 media supplemented with 0.4 M Manitol for osmotic pre-treatment. 1 |am spherical gold particles (BioRad) were used as micro-projectiles for bombardment following<br>
15 published protocols (Russell D.R., Wallace K.M., Bathe J.H., et al. Plant Cell Rep. 1993, 12:165-169). Transformation was performed employing the PDS-1000/He system (BioRad). For the bombardment 30 callus were placed at the center of the plate and the conditions were: 1100 psi of pressure, at the distance of 6 cm, one shoot per plate. After bombardment, calli remained in the same osmotic media for 16 hours at the dark, to be<br>
20 then sub cultivated 2 days in N6-2 media without Manitol. GUS activity is revealed with X-Gluc by hystochemical method (Jefferson R.A. 1988. Plant reporter genes: the GUS gene fusion system. In: J.K. Setlow (Ed), Genetic Engineering. Vol.10, Plenum Publishing Corporation. P.247-263). Evaluation was performed by counting blue points and zones in each callus in a stereomicroscope (Figure 12). Table 4 shows the results obtained after 4<br>
25     experiments with 3 replicas each.<br>
Table 4. ART and Eureka functionality comparative demonstration in rice cells.<br><br>
Experiment	Callus % with blue zones and points<br>
	pBPEQ(omega)-GUSint	pBPFA19GUSint	pA19ARTGUSint	pA19ARTEGUSint<br>
I	40	60	100	100<br>
II	43	80	93	97<br>
III	33	70	87	90<br>
IV	27	83	93	90<br>
MediaÂ±SD	36Â±6	73Â±8	93+3	94Â±4<br><br>
22<br>
As it can be seen, these experimental results also confirm functionality of ART and Eureka as gene expression enhancer elements in monocot plant cells. It is important to highlight that in our assays, IME activity developed by the artificial Exon/Intron/Exon ART (pA19ARTGUSint), was higher than that observed for rice actin-1 gene first 5 Exon/Intron/Exon (pBPFA19GUSint), which is a genetic element with recognized gene expression enhancer ability. Additionally, although in our experiments a significant difference between the results obtained for constructions pA19ARTGUSint and pA19ARTEGUSint can not be appreciated, it is remarkable in figure 12 that the presence of Eureka fragment in the last construction strongly increases expression, because the size<br>
10 and intensity of blue colored zones after X-Gluc hystochemical staining of calli bombarded with pA19ARTEGUSint (Figure 12).<br>
Concluding, the present example shows the functionality of the artificial genetic elements ART and Eureka as gene expression enhancers in any kind of plant cells. Besides, it was demonstrated that these enhancer elements are highly efficient, increasing expression<br>
15 levels independently of the promoter that they are fused to. Finally, it was also demonstrated that ART and Eureka could be combined for synergistically enhance even more the expression of downstream genes.<br>
It is anew shown that ART and Eureka can be functionally inserted between any plant active promoter (e.g. A19) and any DNA sequence (GUSint gene) to increase its<br>
20      transcription/translation.<br>
Example No. 3: PARTE expression system variants employing different 5' transcription enhancer regions.<br>
a)	Addition of rice actin-1 5' transcription activator region to PARTE promoter.<br>
To cys-fase the 5' transcription enhancer region from rice actin-1 gene to PARTE 25 promoter, pPARTE plasmid was digested with the enzymes EcoRl and EcoRY, inserting in it the synthetic DNA fragment En-Ac 1 (from -43 to -221 from the rice actin-1 gene transcription initiation site; SEQ ID NO: 10), with extremes that ligate with those enzymes. The resulting plasmid, pAlPARTE, was Eco RV and Hind III digested to insert the synthetic DNA fragment En-Ac2 (from -226 to -310 from the rice actin-1 transcription 30 initiation site; SEQ ID NO: 11), completing actin-1 gene promoter 5'activator region and producing the plasmid pAPARTE (Figure 5, Figure 7E), which nucleotide sequence between the restriction sites Hindlll and Sad is shown in sequence SEQ ID NO: 12.<br>
b)	Addition of as-1 -like transcription enhancer sequences to APARTE promoter.<br><br>
23<br>
Plasmid pAl PARTE was Nrul and Sail digested to insert in it the DNA synthetic fragment called ASP (SEQ ID NO: 13), which possessed sticking ends to the mentioned restriction enzymes and codifies for an <zs-l-like transcription enhancer sequence producing the construction paspaalparte. this plasmid was sal i digested treated with si nuclease and then pst to obtain approximately bp dna fragment later cloned into vector pal parte nrul pstl finally paspalparte which has asp inserted in site rice actin-1 activation region. by digesting it ecorv xhol a second paspal giving rise></zs-l-like>
10 p2AlPARTE (in sequence SEQ ID NO: 14, the nucleotide sequence is shown between the restriction sites Kpnl and Sad), and in Figure 7F its structure.<br>
As it was explained above for pAl PARTE, an En-Ac2 fragment was also inserted into pASPAl PARTE to obtain the construction pASPAPARTE, where a second ASP enhancer was inserted by digesting it EcoRV - Sail, to finally obtain the vector p2APARTE (Figure<br>
15      7G). Its nucleotide sequence between the sites Sail and Sacl is shown in SEQ ID NO: 15.<br>
c)	U3ARTE promoter construction.<br>
Firstly, it was amplified by Polymerase Chain Reaction (PCR), using the primers Oli-Ul (SEQ ID NO: 16) and OH-U2 (SEQ ID NO: 17), an approximately 395 bp DNA fragment, corresponding to the region from -299 to -680 from maize ubi-l gene transcription<br>
20 initiation site. The amplified fragment was digested with the restriction enzymes Kpnl and Xhol (both sites were included inside the primer) and cloned into similarly processed pBluescript II SK vector, obtaining the construction pBS-Ubil. After that, a synthetic DNA fragment (En-U2), which codifies for maize ubi-\ gene from -680 to -855 (sequence SEQ ID NO: 18), was cloned into the Ncol / Kpnl digested pBS-Ubil vector, resulting in<br>
25 the construction pBS-Ubi2, which contains maize ubiquitine-1 gene 5' activator region (from -299 to -855, sequence SEQ ID NO: 19).<br>
The 5' transcription activator cloned region from maize ubiquitine-1 gene does not contain the "heat shock" box was obtained from the plasmid pBS-Ubi2 by an Xho I - Kpn I digestion, and cw-inserted 5' to promoter 2APARTE by Sail - Kpnl digestion of the<br>
30 vector, to obtain the so called construct pU3ARTE (Figure 6, Figure 7H). The sequence of the vector pU3ARTE between the sites Kpn I and Sac I is shown in sequence SEQ ID NO: 20.<br>
d)	GARTE promoter construction.<br><br>
25<br>
the 5' transcription regulatory regions fused to PARTE. It must be highlighted that with<br>
our genetic constructions we reached superior expression levels in dicot plant cells than<br>
that achieved when expression is controlled by the natural promoter CaMV 35S.<br>
b)  Assays on rice.<br>
5      The binary vectors pC-APARTEGUSint, pC-2A 1 PARTEGUSint, pC-2APARTEGUSint y<br>
pC-U3ARTEGUSint were bombarded on rice calli as described in Example 2 section (c) in<br>
order to carry out a transient evaluation of the activity of the PARTE promoter different<br>
variants. The control plasmid, pActl-F (McElroy D; Zhang W; Cao J; Wu R. Plant Cell<br>
1990, 2:163-171), has the gus gene expression under the control of the rice actin-1 gene<br>
10     promoter and the tNOS terminator. The expression cassette was extracted from these<br>
plasmid by Kpnl - Xbal digestion and inserted into the binary plasmid pCAMBIA 2300<br>
digested with the same restriction enzymes to produce the vector pC-ActlF.<br>
The bombardment experiments were performed three times with three replicas for each<br>
construction to be evaluated. The results obtained are shown in the following table:<br>
15<br>
Table 6. Functionality of different variants of PARTE promoter expression system in rice cells.<br><br>
Experiment		% of calli with blue zones and dots	<br>
	pC-ActlF	pC-APARTEGUSint	pC-2A1PARTEGUSint	pC-2APARTEGUSint	pC-U3ARTEGUSint<br>
I	67	73	92	100	100<br>
II	63	88	95	91	92<br>
III	81	85	90	87	100<br>
MediaÂ±SD	70Â±4	82Â±6	92Â±2	93+5	97Â±4<br>
20 The results shown in this table certify the functionality of the different variants of PARTE promoter in monocot plant cells, achieving expression levels superior to that of the natural promoter of rice actin-1 gene. Therefore, usefulness of the object of the present invention as an efficient genetic tool to achieve high expression levels of DNA sequences placed in cys under its control is confirmed.<br>
25     c)  Assays on rice seeds.<br>
To evaluate tissue specificity of GARTE promoter in rice cell endosperms, the Sail I Klenow - Pstl fragment of about 2.5 Kb from pBPFARTEGUSint vector, containing the Eureka fragment fused to GUSint gene with nos terminator (tNOS), was cloned into<br><br>
26<br>
pGARTE vector Xbal I Klenow - Pstl digested, giving as a result the construction pGARTEGUSint.<br>
It was also constructed a control plasmid, where the GARTE promoter in pGARTEGUSint is substituted by Xho I - Nco I digestion, by a seed endosperm-specific, highly efficient 5      rice gluteline B-l promoter obtained from Sail - Ncol digestion of pGEM-T-GluB-1 plasmid. Thus, we obtained pGluGUSint.<br>
Evaluation of GARTE promoter and its comparison with that of GluBl, was carried out according to Y-S. Hwang (Hwang Y-S; McCullar Cass; Huang N. Plant Science. 2001, 161:1107-1116) by bombarding immature  endosperms (8-9 days  after polinization)<br>
10 isolated from the ear cariopsis of greenhouse cultured rice variety indica Perla. Fluorometric assay to determine GUS activity in endosperms was performed according to Jefferson (Jefferson R.A. 1988. Plant reporter genes: the GUS gene fusion system. In: J.K. Setlow (Ed), Genetic Engineering. Vol.10, Plenum Publishing Corporation. P.247-263), 24 hours after bombardment with gold micro-particles covered by plasmidic DNA to be<br>
15      evaluated. The results of the GUS activity obtained in two independent experiments with 5 replicas each, are shown in Table 7. Table 7. Functionality of GARTE promoter in rice seed endosperms.<br><br>
Experiment	GUS activity (Pm 4-MU/hr/mg total proteins)	GARTE/GluB-1 rate<br>
	pGluGUSint	pGARTEGUSint	<br>
I	34Â±9	79+22	2.3<br>
II	27Â±5	52Â±13	1.9<br>
20 These results confirm that the chimerical promoter GARTE, based on artificial elements designed for us, is highly efficient to express genes in seed endosperms; although GLU sequence inserted on the GARTE promoter is able to confer specificity to the expression, it 'per se' does not guarantee high levels, which depends also on other elements conforming the promoter (Takaiwa F; Yamanouchi U; Yoshihara T; Washida H; Tanabe F; Kato A;<br>
25      Yamada K. Plant Mol Biol. 1996, 30:1207-1221).<br>
The showed data reaffirm that the insertion of regulatory regions upstream to the element object of this invention permits its use to efficiently conduce the expression of any DNA sequence with development-, organ- or tissue-specificity. To somebody experienced in molecular biology, it is obvious that GluB-1 promoter regulatory sequences inserted into<br>
30 the GARTE promoter can be successfully substituted for regulatory sequences responding to biotic stress (pathogen attack, for example), abiotic factors (e.g., wounding, extremely<br><br>
27<br>
high or low temperatures, salinity, drought, the presence of some chemicals), oxidative stress, different organ and tissue development stages, etc.<br>
It is also evident that DNA sequences cloned under the regulatory regions object of this invention can be introduced into plant cells and stably inserted by the means of known 5 biological or physic-chemical transformation methods and that, from these genetically modified cells it is possible to regenerate fertile plants in which DNA sequences will conveniently express according to the promoter variant which they are fused to. Thus, the present invention reveals its potentiality to contribute to the production of transgenic plants with greater levels of resistance to pests, diseases, a variety of stresses, greater agricultural 10 yields or highly efficient producing compounds with medical or industrial applications, among other uses.<br><br>
28<br>
SEQUENCE LISTING<br> Centro de Ingenierla Genetica y Biotecnologia<br>
5   Artificial promoter for the expression of DNA sequences in plant cells<br> Artificial promoter<br>
10    0000<br> 2002-11-18<br> 22<br>
15   Patentln Ver. 2.1<br> 1  86  DNA 20   Artificial sequence<br><br> Artificial sequence description: Translational enhancer Eureka.<br>
25<br> 1<br>
gaaacaaatt gaacatcatt ctatcaatac aacacaaaca caacacaact caatcattta 60<br>
tttgacaaca caactaaaca accatg	86<br>
30<br>	2<br>	198<br>	DNA<br><br>
35<br><br> Artificial sequence description: Synthetic fragment P35AcU.<br>
40    2<br>
gaattctata tataggaagt tcatttcatt tggagccccc caaccctacc accaccacca 60<br>
ccaccacctc ctccttcaca caacacacac acaacagatc tcccccatcc tccctcccgt 120<br>
cgcgccgcgc aacacctggt aagatggctg tgcgctcaga tatatatagt gatatgcact 180<br>
acaaagatca taactagt	198<br>
45<br> 3  231  DNA 50   <br><br> Artificial sequence description: Synthetic fragment I-U/Ac.<br>
55<br> 3<br>
ctagaccgcc gcctcccccc ccccccctct ctaccttctc tctttctttc tccgtttttt 60<br>
ttttccgtct cgtctcgatc tttggccttg gtagtttggg ggcgagaggc ggcttcgtcg 120<br>
cccagatcgg tgcgcgtttt tttatttgga ggggcgggat ctcgcggctg ggtctcggcg 180<br>
60  tgcggccgga ttctcgcggg gaatggggct ctcggatgtg gatccgagct c	231<br><br>
29<br> 4  255 5    DNA<br> Artificial sequence<br><br> Artificial sequence description: 10       Synthetic fragment I-Ac/U.<br> 4<br>
gatctgatcc gccgttgttg ggggagatat ggggcgttta aaatttcgcc atgctaaaca 60<br>
agatcaggaa gaggggaaaa gggcactatg gtttaatttt tatatatttc tgctgctgct 120<br>
15  cgtcaggatt agatgtgctt gatctttctt tcttcttttt gtgggtagaa tttgaatccc 180<br>
tcagcattgt tcatcggtag tttttctttt gtcgatgctc accctgttgt ttggtgtttt 240<br>
tatactagtg agctc	255<br>
20    5  93  DNA  Artificial sequence<br>
25   <br> Artificial sequence Description: Synthetic fragment IniT.<br> 5<br>
30  ctagtggcta tcctgacacg gtctctttgt caaatatctc tgtgtgcagg tataactgca 60<br>
ggaaacaaca acaataacca tggtctagag etc	93<br> 6 35    692  DNA <br> 40   Artificial sequence description: Artificial Exon/Intron/Exon ART.<br> 6<br>
accaccacca ccaccaccac ctcctccttc acacaacaca cacacaacag atctccccca 60<br>
45 tcctccctcc cgtcgcgccg cgcaacacct ggtaagatgg ctgtgcgctc agatatatat 120 agtgatatgc actacaaaga tcataactag accgccgcct cccccccccc ccctctctac 180 cttctctctt tctttctccg tttttttttt ccgtctcgtc tcgatctttg gccttggtag 240 tttgggggcg agaggcggct tcgtcgccca gatcggtgcg cgttttttta tttggagggg 3 00 cgggatctcg cggctgggtc tcggcgtgcg gccggattct cgcggggaat ggggctctcg 3 60<br>
50 gatgtggatc tgatccgccg ttgttggggg agatatgggg cgtttaaaat ttcgccatgc 420 taaacaagat caggaagagg ggaaaagggc actatggttt aatttttata tatttctgct 480 gctgctcgtc aggattagat gtgcttgatc tttctttctt ctttttgtgg gtagaatttg 540 aatccctcag cattgttcat cggtagtttt tcttttgtcg atgctcaccc tgttgtttgg 600 tgtttttata ctagtggcta tcctgacacg gtctctttgt caaatatctc tgtgtgcagg 660<br>
55   tataactgca ggaaacaaca acaataacca tg	692<br>	7<br>	750<br>
60   	DNA<br><br><br>
30<br><br> Artificial sequence description:<br>
pBS-ART vector sequence between the restriction sites EcoRI and 5  Sad.<br> 7<br>
gaattctata tataggaagt tcatttcatt tggagcccec eaaccctacc accaecacca 60<br>
ccaccacctc ctccttcaca caacacacac acaacagatc tcccccatcc tccctcccgt 120<br>
10 cgcgccgcgc aacacctggt aagatggctg tgcgctcaga tatatatagt gatatgcact 18 0 acaaagatca taactagacc gccgcctccc cccccccccc tctctacctt ctctctttct 240 ttctccgttt tttttttccg tctcgtctcg atctttggcc ttggtagttt gggggcgaga 300 ggcggcttcg tcgcccagat cggtgcgcgt ttttttattt ggaggggcgg gatctcgcgg 360 ctgggtctcg gcgtgcggcc ggattctcgc ggggaatggg gctctcggat gtggatctga 420<br>
15 tccgccgttg ttgggggaga tatggggcgt ttaaaatttc gccatgctaa acaagatcag 480 gaagagggga aaagggcact atggtttaat ttttatatat ttctgctgct gctcgtcagg 540 attagatgtg cttgatcttt ctttcttctt tttgtgggta gaatttgaat ccctcagcat 600 tgttcatcgg tagtttttct tttgtcgatg ctcaccctgt tgtttggtgt ttttatacta 660 gtggctatcc tgacacggtc tctttgtcaa atatctctgt gtgcaggtat aactgcagga 720<br>
20  aacaacaaca ataaccatgg tctagagctc	750<br> 8  757 25    DNA<br> Artificial sequence<br> 30    Artificial sequence description: Artificial Exon/Intron/Exon ARTE,<br> 8<br>
accaecacca ccaccaccac ctcctccttc acacaacaca cacacaacag atctccccca 60<br>
35 tcctccctcc cgtcgcgccg cgcaacacct ggtaagatgg ctgtgcgctc agatatatat 120 agtgatatgc actacaaaga tcataactag accgccgcct cccccccccc ccctctctac 180 cttctctctt tctttctccg tttttttttt ccgtctcgtc tcgatctttg gecttggtag 240 tttgggggcg agaggegget tcgtcgccca gatcggtgcg cgttttttta tttggagggg 3 00 egggatcteg cggctgggtc tcggcgtgcg geeggattet cgcggggaat ggggctctcg 360<br>
40 gatgtggatc tgatccgccg ttgttggggg agatatgggg cgtttaaaat ttcgccatgc 420 taaacaagat caggaagagg ggaaaagggc actatggttt aatttttata tatttctgct 480 gctgctcgtc aggattagat gtgettgate tttctttctt ctttttgtgg gtagaatttg 540 aatccctcag cattgttcat cggtagtttt tcttttgtcg atgctcaccc tgttgtttgg 600 tgtttttata ctagtggcta tcctgacacg gtctctttgt caaatatctc tgtgtgcagg 660<br>
45 tataactgea ggaaacaaat tgaacatcat tctatcaata caacacaaac acaacacaac 720<br>
tcaatcattt atttgacaac acaactaaac aaccatg	757<br>	9<br>
50   	815<br>	DNA<br>	Artificial sequence<br> 55   Artificial sequence description:<br>
pPARTE vector sequence between the restriction sites EcoRI and SacI.<br> 9 60  gaattctata tataggaagt tcatttcatt tggagcccec eaaccctacc accaecacca 60 ccaccacctc ctccttcaca caacacacac acaacagatc tcccccatcc tccctcccgt 120<br><br>
31<br><br>
cgcgccgcgc aacacctggt aagatggctg tgcgctcaga tatatatagt gatatgcact 18 0<br>
acaaagatca taactagacc gccgcctccc cccccccccc tctctacctt ctctctttct 240<br>
ttctccgttt tttttttccg tctcgtctcg atctttggcc ttggtagttt gggggcgaga 300<br>
ggcggcttcg tcgcccagat cggtgcgcgt ttttttattt ggaggggcgg gatctcgcgg 360<br>
5 ctgggtctcg gcgtgcggcc ggattctcgc ggggaatggg gctctcggat gtggatctga 42 0<br>
tccgccgttg ttgggggaga tatggggcgt ttaaaatttc gccatgctaa acaagatcag 480<br>
gaagagggga aaagggcact atggtttaat ttttatatat ttctgctgct gctcgtcagg 540<br>
attagatgtg cttgatcttt ctttcttctt tttgtgggta gaatttgaat ccctcagcat 600<br>
tgttcatcgg tagtttttct tttgtcgatg ctcaccctgt tgtttggtgt ttttatacta 660<br>
10 gtggctatcc tgacacggtc tctttgtcaa atatctctgt gtgcaggtat aactgcagga 720<br>
aacaaattga acatcattct atcaatacaa cacaaacaca acacaactca atcatttatt 780<br>
tgacaacaca actaaacaac catggtctag agctc	815<br>
15    10  184  DNA  Artificial sequence<br>
20   <br> Artificial sequence description: Synthetic fragment En-Acl.<br> 10<br>
25  atcaccgtga gttgtccgca ccaccgcacg tctcgcagcc aaaaaaaaaa aaagaaagaa 60<br>
aaaaaagaaa aagaaaaaac agcaggtggg tccgggtcgt gggggccgga aaagcgagga 120<br>
ggatcgcgag cagcgacgag gccggccctc cctccgcttc caaagaaacg ccccccatca 180<br>
attc	184<br>
30<br> 11<br> 94<br> DNA<br>
35<br> Artificial sequence<br><br> Artificial sequence description: Synthetic fragment En-Ac2.<br>
40    11<br>
aagcttgata tccatagcaa gcccagccca acccaaccca acccaaccca ccccagtgca 60<br>
gccaactggc aaatagtctc cacaccccgg cact	94<br>
45    12<br> 1087<br> DNA<br> Artificial sequence<br>
50   <br> Artificial sequence description:<br>
pAPARTE vector sequence between the restriction sites Hindlll y Sad.<br>
55    12<br>
aagcttgata tccatagcaa gcccagccca acccaaccca acccaaccca	ccccagtgca 60<br>
gccaactggc aaatagtctc cacaccccgg cactatcacc gtgagttgtc	cgcaccaccg 120<br>
cacgtctcgc agccaaaaaa aaaaaaagaa agaaaaaaaa gaaaaagaaa	aaacagcagg 18 0<br>
tgggtccggg tcgtgggggc cggaaaagcg aggaggatcg cgagcagcga	cgaggccggc 24 0<br>
60  cctccctccg cttccaaaga aacgcccccc atcaattcta tatataggaa	gttcatttca 300<br>
tttggagccc cccaacccta ccaccaccac caccaccacc tcctccttca	cacaacacac 360<br><br>
33<br> Artificial sequence<br><br> Artificial sequence description: 5       p2APARTE vector sequence between the restriction sites Sail and Sad.<br> 15<br>
gtcgactgac gcttcgaatg acgcacatgc catccatagc aagcccagcc caacccaaqc 60<br>
10 caacccaacc caccccagtg cagccaactg gcaaatagtc tccacacccc ggcactatca 120 ccgtgagttg tccgcaccac cgcacgtctc gcagccaaaa aaaaaaaaag aaagaaaaaa 180 aagaaaaaga aaaaacagca ggtgggtccg ggtcgtgggg gccggaaaag cgaggaggat 24 0 cgctgacgct tcgaatgacg cacatgcccg agcagcgacg aggccggccc tccctccgct 300 tccaaagaaa cgccccccat caattctata tataggaagt tcatttcatt tggagccccc 360<br>
15 caaccctacc accaccacca ccaccacctc ctccttcaca caacacacac acaacagatc 420 tcccccatcc tccctcccgt cgcgccgcgc aacacctggt aagatggctg tgcgctcaga 480 tatatatagt gatatgcact acaaagatca taactagacc gccgcctccc cccccccccc 540 tctctacctt ctctctttct ttctccgttt tttttttccg tctcgtctcg atctttggcc 600 ttggtagttt gggggcgaga ggcggcttcg tcgcccagat cggtgcgcgt ttttttattt 660<br>
20 ggaggggcgg gatctcgcgg ctgggtctcg gcgtgcggcc ggattctcgc ggggaatggg 720 gctctcggat gtggatctga tccgccgttg ttgggggaga tatggggcgt ttaaaatttc 780 gccatgctaa acaagatcag gaagagggga aaagggcact atggtttaat ttttatatat 840 ttctgctgct gctcgtcagg attagatgtg cttgatcttt ctttcttctt tttgtgggta 900 gaatttgaat ccctcagcat tgttcatcgg tagtttttct tttgtcgatg ctcaccctgt 960<br>
25 tgtttggtgt ttttatacta gtggctatcc tgacacggtc tctttgtcaa atatctctgt 1020 gtgcaggtat aactgcagga aacaaattga acatcattct atcaatacaa cacaaacaca 1080 acacaactca atcatttatt tgacaacaca actaaacaac catggtctag agctc     1135<br>
30    16  31  DNA  Artificial sequence<br>
35   <br> Artificial sequence description:<br>
1.<br> 16<br>
40  gaaggtaccg ccatggtcta aaggacaatt g	31<br> 17  27 45    DNA<br> Artificial sequence<br><br> Artificial sequence description: 50        Oligonucleotidic primer 01i-U2.<br> 17<br>
ctcctcgagg gcgtttaaca ggctggc	27<br><br>
55<br>
60<br><br>	18<br>	186<br>	DNA<br>	Artificial sequence<br><br><br>
31<br><br>
10<br>
15<br><br><br>
cgcgccgcgc	aacacctggt	aagatggctg	tgcgctcaga	tatatatagt	gatatgcact	180<br>
acaaagatca	taactagacc	gccgcctccc	cccccccccc	tctctacctt	ctctctttct	240<br>
ttctccgttt	tttttttccg	tctcgtctcg	atctttggcc	ttggtagttt	gggggcgaga	300<br>
ggcggcttcg	tcgcccagat	cggtgcgcgt	ttttttattt	ggaggggcgg	gatctcgcgg	360<br>
ctgggtctcg	gcgtgcggcc	ggattctcgc	ggggaatggg	gctctcggat	gtggatctga	420<br>
tccgccgttg	ttgggggaga	tatggggcgt	ttaaaatttc	gccatgctaa	acaagatcag	480<br>
gaagagggga	aaagggcact	atggtttaat	ttttatatat	ttctgctgct	gctcgtcagg	540<br>
attagatgtg	cttgatcttt	ctttcttctt	tttgtgggta	gaatttgaat	ccctcagcat	600<br>
tgttcatcgg	tagtttttct	tttgtcgatg	ctcaccctgt	tgtttggtgt	ttttatacta	660<br>
gtggctatcc	tgacacggtc	tctttgtcaa	atatctctgt	gtgcaggtat	aactgcagga	720<br>
aacaaattga	acatcattct	atcaatacaa	cacaaacaca	acacaactca	atcatttatt	780<br>
tgacaacaca	actaaacaac	catggtctag	agctc			815<br> 10						<br> 184						<br> DNA						<br> Artificial sequence				<br><br><br><br>
20<br><br><br> Artificial sequence description: Synthetic fragment En-Acl.<br><br><br><br>
25<br><br> 10<br>
atcaccgtga gttgtccgca ccaccgcacg tctcgcagcc aaaaaaaaaa aaagaaagaa 60<br>
aaaaaagaaa aagaaaaaac agcaggtggg tccgggtcgt gggggccgga aaagcgagga 12 0<br>
ggatcgcgag cagcgacgag gccggccctc cctccgcttc caaagaaacg ccccccatca 180<br>
attc	184<br><br><br><br>
30<br>
35<br>
40<br><br> 11<br> 94<br> DNA<br> Artificial sequence<br><br> Artificial sequence description: Synthetic fragment En-Ac2.<br> 11<br>
aagcttgata tccatagcaa gcccagccca acccaaccca acccaaccca ceccagtgca 60<br>
gccaactggc aaatagtctc cacaccccgg cact	94<br><br><br><br>
45<br>
50<br><br> 12<br> 1087<br> DNA<br> Artificial sequence<br><br> Artificial sequence description:<br>
pAPARTE vector sequence between the restriction sites Hindlll y Sacl.<br><br><br><br>
55<br>
60<br><br> 12<br>
aagcttgata<br>
gccaactggc<br>
cacgtctcgc<br>
tgggtccggg<br>
cctccctccg<br>
tttggagccc<br><br>
tccatagcaa aaatagtctc agccaaaaaa tcgtgggggc cttccaaaga cccaacccta<br><br>
gcccagccca cacaccccgg aaaaaaagaa cggaaaagcg aacgcccccc ccaccaccac<br><br>
acccaaccca cactatcacc agaaaaaaaa aggaggatcg atcaattcta caccaccacc<br><br>
acccaaccca gtgagttgtc gaaaaagaaa cgagcagcga tatataggaa tcctccttca<br><br>
ceccagtgca 60 cgcaccaccg 120 aaacagcagg 180 cgaggccggc 240 gttcatttca 300 cacaacacac 360<br><br>
32<br><br>
acacaacaga tctcccccat cctccctccc gtcgcgccge gcaacacctg gtaagatggc 420<br>
tgtgcgctca gatatatata gtgatatgca ctacaaagat cataactaga ccgccgcctc 480<br>
cccccccccc cctctctacc ttctctcttt ctttctccgt tttttttttc cgtctcgtct 540<br>
cgatctttgg ccttggtagt ttgggggcga gaggcggctt cgtcgcccag atcggtgcgc 600<br>
5 gtttttttat ttggaggggc gggatctcgc ggctgggtct cggcgtgcgg ccggattctc 660<br>
gcggggaatg gggctctcgg atgtggatct gatccgccgt tgttggggga gatatggggc 720<br>
gtttaaaatt tcgccatgct aaacaagatc aggaagaggg gaaaagggca ctatggttta 780<br>
atttttatat atttctgctg ctgctcgtca ggattagatg tgcttgatct ttctttcttc 840<br>
tttttgtggg tagaatttga atccctcagc attgttcatc ggtagttttt cttttgtcga 900<br>
10 tgctcaccct gttgtttggt gtttttatac tagtggctat cctgacacgg tctctttgtc 960<br>
aaatatctct gtgtgcaggt ataactgcag gaaacaaatt gaacatcatt ctatcaatac 1020<br>
aacacaaaca caacacaact caatcattta tttgacaaca caactaaaca accatggtct 1080<br>
agagctc	1087<br>
15<br> 13<br> 31<br> ADN<br>
20<br> Artificial sequence<br><br> Artificial sequence description: Synthetic fragment ASP.<br>
25    13<br>
gtcgactgac gcttcgaatg acgcacatgc c	31<br> 14 30    1065  DNA  Artificial sequence<br> 35   Artificial sequence description:<br>
p2AlPARTE vector between the restriction sites Kpnl and SacI.<br> 14<br>
ggtaccgggc cccccctcga ctgacgcttc gaatgacgca catgccatca ccgtgagttg 60<br>
40 tccgcaccac cgcacgtctc gcagccaaaa aaaaaaaaag aaagaaaaaa aagaaaaaga 120 aaaaacagca ggtgggtccg ggtcgtgggg gccggaaaag cgaggaggat cgctgacgct 180 tcgaatgacg cacatgcccg agcagcgacg aggccggccc tccctccgct tccaaagaaa 240 cgccccccat caattctata tataggaagt tcatttcatt tggagccccc caaccctacc 300 accaccacca ccaccacctc ctccttcaca caacacacac acaacagatc tcccccatcc 360<br>
45 tccctcccgt cgcgccgcgc aacacctggt aagatggctg tgcgctcaga tatatatagt 420 gatatgcact acaaagatca taactagacc gccgcctccc cccccccccc tctctacctt 480 ctctctttct ttctccgttt tttttttccg tctcgtctcg atctttggcc ttggtagttt 540 gggggcgaga ggcggcttcg tcgcccagat cggtgcgcgt ttttttattt ggaggggcgg 600 gatctcgcgg ctgggtctcg gcgtgcggcc ggattctcgc ggggaatggg gctctcggat 660<br>
50 gtggatctga tccgccgttg ttgggggaga tatggggcgt ttaaaatttc gccatgctaa 720 acaagatcag gaagagggga aaagggcact atggtttaat ttttatatat ttctgctgct 780 gctcgtcagg attagatgtg cttgatcttt ctttcttctt tttgtgggta gaatttgaat 840 ccctcagcat tgttcatcgg tagtttttct tttgtcgatg ctcaccctgt tgtttggtgt 900 ttttatacta gtggctatcc tgacacggtc tctttgtcaa atatctctgt gtgcaggtat 960<br>
55  aactgcagga aacaaattga acatcattct atcaatacaa cacaaacaca acacaactca 1020<br>
atcatttatt tgacaacaca actaaacaac catggtctag agctc	1065<br> 15 60    1135  DNA<br><br>
33<br> Artificial sequence<br><br> Artificial sequence description: 5        p2APARTE vector sequence between the restriction sites Sail and Sacl.<br> 15<br>
gtcgactgac gcttcgaatg acgcacatgc catccatagc aagcccagcc caacccaacc	60<br>
10  caacccaacc caccccagtg cagccaactg gcaaatagtc tccacacccc ggcactatca	120<br>
ccgtgagttg tccgcaccac cgcacgtctc gcagccaaaa aaaaaaaaag aaagaaaaaa	180<br>
aagaaaaaga aaaaacagca ggtgggtccg ggtcgtgggg gccggaaaag cgaggaggat	24 0<br>
cgctgacgct tcgaatgacg cacatgcccg agcagcgacg aggccggccc tccctccgct	300<br>
tccaaagaaa cgccccccat caattctata tataggaagt tcatttcatt tggagccccc	360<br>
15  caaccctacc accaccacca ccaccacctc ctccttcaca caacacacac acaacagatc	420<br>
tcccccatcc tccctcccgt cgcgccgcgc aacacctggt aagatggctg tgcgctcaga	480<br>
tatatatagt gatatgcact acaaagatca taactagacc gccgcctccc cccccccccc	540<br>
tctctacctt ctctctttct ttctccgttt tttttttccg tctcgtctcg atctttggcc	600<br>
ttggtagttt gggggcgaga ggcggcttcg tcgcccagat cggtgcgcgt ttttttattt	660<br>
20  ggaggggcgg gatctcgcgg ctgggtctcg gcgtgcggcc ggattctcgc ggggaatggg	720<br>
gctctcggat gtggatctga tccgccgttg ttgggggaga tatggggcgt ttaaaatttc	780<br>
gccatgctaa acaagatcag gaagagggga aaagggcact atggtttaat ttttatatat	840<br>
ttctgctgct gctcgtcagg attagatgtg cttgatcttt ctttcttctt tttgtgggta	900<br>
gaatttgaat ccctcagcat tgttcatcgg tagtttttct tttgtcgatg ctcaccctgt	960<br>
25  tgtttggtgt ttttatacta gtggctatcc tgacacggtc tctttgtcaa atatctctgt	1020<br>
gtgcaggtat aactgcagga aacaaattga acatcattct atcaatacaa cacaaacaca	1080<br>
acacaactca atcatttatt tgacaacaca actaaacaac catggtctag agctc	1135<br>
30    16  31  DNA  Artificial sequence<br>
35   <br> Artificial sequence description: 1.<br> 16<br>
40  gaaggtaccg ccatggtcta aaggacaatt g	31<br> 17  27 45    DNA<br> Artificial sequence<br><br> Artificial sequence description: 50        Oligonucleotidic primer 01i-U2.<br> 17<br>
ctcctcgagg gcgtttaaca ggctggc	2 7<br><br>
55<br>
60<br><br>	18<br>	186<br>	DNA<br>	Artificial sequence<br><br><br>
34<br> Artificial sequence description: Synthetic fragment En-U2.<br> 18<br>
ggtaccgagc attgcatgtc taagttataa aaaattacca catatttttt ttgtcacaet 60<br>
tgtttgaagt gcagtttatc tatctttata catatattta aactttactc tacgaataat 120<br>
ataatctata gtacaacaat aatatcagtg ttttagagaa tcatataaat gaacagttag 180<br>
acatgg	186<br><br>
10<br> 19<br> 563<br> DNA<br>
15<br> Artificial sequence<br><br> Artificial sequence description:<br>
Maize ubiquitine-1 gene derived transcriptional enhancer sequence (region from -299 a -855) .<br>
20<br> 19<br>
ggtaccgagc attgcatgtc taagttataa aaaattacca catatttttt	ttgtcacaet 60<br>
tgtttgaagt gcagtttatc tatctttata catatattta aactttactc	tacgaataat 120<br>
ataatctata gtacaacaat aatatcagtg ttttagagaa tcatataaat	gaacagttag 180<br>
25  acatggtcta aaggacaatt gagtattttg acaacaggac tctacagttt	tatcttttta 240<br>
gtgtgcatgt gttctccttt ttttttgcaa atagcttcac ctatataata	cttcatccat 300<br>
tttattagta catccattta gggtttaggg ttaatggttt ttatagacta	atttttttag 360<br>
tacatctatt ttattctatt ttagectcta aattaagaaa actaaaactc	tattttagtt 420<br>
tttttattta ataatttaga tataaaatag aataaaataa agtgactaaa	aattaaacaa 480<br>
30  atacccttta agaaattaaa aaaactaagg aaacattttt cttgtttcga	gtagataatg 540<br>
ccagcctgtt aaacgccctc gac	563<br> 20 35    1692  DNA  Artificial sequence<br>
Secuencia  40   Artificial sequence description:<br>
pU3ARTE vector sequence between the restriction sites Kpnl and Sad.<br> 20<br>
45  ggtaccgagc attgcatgtc taagttataa aaaattacca catatttttt ttgtcacaet 60 tgtttgaagt gcagtttatc tatctttata catatattta aactttactc tacgaataat 120 ataatctata gtacaacaat aatatcagtg ttttagagaa tcatataaat gaacagttag 180 acatggtcta aaggacaatt gagtattttg acaacaggac tctacagttt tatcttttta 240 gtgtgcatgt gttctccttt ttttttgcaa atagcttcac ctatataata cttcatccat 300<br>
50 tttattagta catccattta gggtttaggg ttaatggttt ttatagacta atttttttag 360 tacatctatt ttattctatt ttagectcta aattaagaaa actaaaactc tattttagtt 420 tttttattta ataatttaga tataaaatag aataaaataa agtgactaaa aattaaacaa 480 atacccttta agaaattaaa aaaactaagg aaacattttt cttgtttcga gtagataatg 540 ccagcctgtt aaacgccctc gaetgacget tcgaatgacg cacatgccat ccatagcaag 600<br>
55 cccagcccaa cccaacccaa cccaacccac cccagtgcag ccaactggca aatagtctcc 660 acaccccggc actatcaccg tgagttgtcc gcaccaccgc acgtctcgca gecaaaaaaa 720 aaaaaagaaa gaaaaaaaag aaaaagaaaa aacagcaggt gggtccgggt cgtgggggcc 780 ggaaaagega ggaggatege tgaegctteg aatgaegcac atgcccgagc agegacgagg 84 0 ccggccctcc ctccgcttcc aaagaaaege cccccatcaa ttctatatat aggaagttca 900<br>
60 tttcatttgg agccccccaa ccctaccacc accaccacca ccacctcctc cttcacacaa 960 cacacacaca acagatctcc cccatcctcc ctcccgtcgc gccgcgcaac acctggtaag 1020<br><br>
35<br><br>
atggctgtgc	gctcagatat	atatagtgat	atgcactaca aagatcataa ctagaccgcc	1080<br>
gcctcccccc	ccccccctct	ctaccttctc	tctttctttc tccgtttttt ttttccgtct	1140<br>
cgtctcgatc	tttggccttg	gtagtttggg	ggcgagaggc ggcttcgtcg cccagatcgg	1200<br>
tgcgcgtttt	tttatttgga	ggggcgggat	ctcgcggctg ggtctcggcg tgcggccgga	1260<br>
5  ttctcgcggg	gaatggggct	ctcggatgtg	gatctgatcc gccgttgttg ggggagatat	132 0<br>
ggggcgttta	aaatttcgcc	atgctaaaca	agatcaggaa gaggggaaaa gggcactatg	1380<br>
gtttaatttt	tatatatttc	tgctgctgct	cgtcaggatt agatgtgctt gatctttctt	1440<br>
tcttcttttt	gtgggtagaa	tttgaatccc	tcagcattgt tcatcggtag tttttctttt	1500<br>
gtcgatgctc	accctgttgt	ttggtgtttt	tatactagtg gctatcctga cacggtctct	1560<br>
10  ttgtcaaata	tctctgtgtg	caggtataac	tgcaggaaac aaattgaaca tcattctatc	1620<br>
aatacaacac	aaacacaaca	caactcaatc	atttatttga caacacaact aaacaaccat	1680<br>
ggtctagagc	tc	1692<br>
15    21  223  DNA  Artificial sequence<br>
20   <br> Artificial sequence description: Synthetic fragment GLU.<br> 21<br>
25  ctcgagatac atattaagag tatggacaga catttcttta acaaactcca tttgtattac 60<br>
tccaaaagca ccagaagttt gtcatggctg agtcatgaaa tgtatagttc aatcttgcaa 120<br>
agttgccttt ccttttgtac tgtgttttaa cactacaagc catatattgt ctgtacgtgc 180<br>
aacaaactat atcaccatgt atcccaagat gcttttttaa ttc	223<br>
30<br> 22<br> 1032<br> DNA<br>
35<br> Artificial sequence<br><br> Artificial sequence description:<br>
pGARTE vector sequence between the restriction sites Xhol and Sad.<br>
40   22<br>
ctcgagatac atattaagag tatggacaga catttcttta acaaactcca tttgtattac 60 tccaaaagca ccagaagttt gtcatggctg agtcatgaaa tgtatagttc aatcttgcaa 120 agttgccttt ccttttgtac tgtgttttaa cactacaagc catatattgt ctgtacgtgc 180 aacaaactat atcaccatgt atcccaagat gcttttttaa ttctatatat aggaagttca 240<br>
45 tttcatttgg agccccccaa ccctaccacc accaccacca ccacctcctc cttcacacaa 300 cacacacaca acagatctcc cccatcctcc ctcccgtcgc gccgcgcaac acctggtaag 360 atggctgtgc gctcagatat atatagtgat atgcactaca aagatcataa ctagaccgcc 420 gcctcccccc ccccccctct ctaccttctc tctttctttc tccgtttttt ttttccgtct 480 cgtctcgatc tttggccttg gtagtttggg ggcgagaggc ggcttcgtcg cccagatcgg 54 0<br>
50 tgcgcgtttt tttatttgga ggggcgggat ctcgcggctg ggtctcggcg tgcggccgga 600 ttctcgcggg gaatggggct ctcggatgtg gatctgatcc gccgttgttg ggggagatat 660 ggggcgttta aaatttcgcc atgctaaaca agatcaggaa gaggggaaaa gggcactatg 72 0 gtttaatttt tatatatttc tgctgctgct cgtcaggatt agatgtgctt gatctttctt 780 tcttcttttt gtgggtagaa tttgaatccc tcagcattgt tcatcggtag tttttctttt 840<br>
55  gtcgatgctc accctgttgt ttggtgtttt tatactagtg gctatcctga cacggtctct 900<br>
ttgtcaaata tctctgtgtg caggtataac tgcaggaaac aaattgaaca tcattctatc 960<br>
aatacaacac aaacacaaca caactcaatc atttatttga caacacaact aaacaaccat 1020<br>
ggtctagagc tc	1032<br>
60<br><br>
36<br>
We Claim:<br>
1.	An artificial promoter comprising of:<br>
5	(a) A core promoter,<br>
(b)	5' transcription regulator element, and<br>
(c)	An enhancer element<br>
2.	An artificial promoter as claimed in claim 1, wherein core promoter comprises of<br>
10	TATA box and CaMV 3 5 S promoter.<br>
3.	An artificial promoter as claimed in claim 1, wherein 5' transcription regulator<br>
element comprises having SEQ ID NO. 6 and 8.<br>
15	4.   An artificial promoter as claimed in claim 1, wherein enhancer element is having<br>
SEQ ID No. 1.<br>
5.	An artificial promoter as claimed in claim 3, wherein SEQ ID Nos. 6 and 8<br>
comprises of artificial exon/intron/exon regions have SEQ ID Nos. 10, 11 and 19.<br>
20<br>
6.	An artificial promoter as claimed in claim 1, wherein the promoter controls the<br>
gene expression in plants.<br>
7.	An artificial promoter as claimed in claim 6, wherein the promoter expresses under<br>
25                 biotic   stress,   abiotic   stress,   wounding,   in   seed   endosperm,   and   other<br>
environmentally and artificial induced conditions.<br>
8.	An artificial promoter as claimed in claim 1, wherein said promoter expresses both<br>
in dicots and monocot plants.<br>
30<br>
9.	An artificial promoter as claimed in claim 5, wherein first and second exon region<br>
comprises of motifs with high C and A content.<br>
10.	An artificial promoter as claimed in claim 5, wherein second exon comprises of at<br>
35	least 83% homology with motif HCAYYY (H=C or T or A ; Y= C or T)<br>
11.	An artificial promoter as claimed in claim 5, wherein first exon and intron region<br>
comprises of sequences repeated with motif CTCC and /or homologous sequences<br>
CTC, TCC and TC.<br>
40<br>
12.	An artificial promoter as claimed in claim 1, wherein the said promoter expresses<br>
in all parts/organs/cells/tissues of both monocot and dicot plants.<br>
13.	An artificial promoter as claimed in claim 1, wherein the said promoter is<br>
45	expressed in plasmid vector system selected from group comprising of Eureka-<br>
pBS, pBS-AcUc, pBSAcAcUc, pBS-ART, pPARTE, pBS-EURGUSint, pBPF-<br>
EURGGUSint, pBPFQ7-GUSint, pBPFATGUSint, pBPFARTEGUSint, pC-<br>
ARTGUSint, pC-ARTEGUSint, pBPFA19ARTGUSint, pBPFA19ARTEGUSint,<br>
pAlPARTE, pASPAlPARTE, pASPAPARTE, p2AlPARTE, pBS-Ubil, pBS-<br>
50	Ubi2, pU3ARTE, pGARTE, pC-APARTEGUSint, pC-2AlPARTEGUSint, pC-<br><br>
37<br>
2APARTEGUSint, pC-U3ARTEGUSint, pC-ACTIF, pGARTEGUSint, and pGluGUSint.<br>
14.	An artificial promoter as claimed in claim 13, wherein said plasmid vectors<br>
5	systems express both in monocot and dicot plants.<br>
15.	An artificial promoter as claimed in claim 13, wherein said plasmid vectors express<br>
in all the parts, tissues, cells, organs of both monocot and dicot plants.<br>
10	16. An artificial promoter as claimed in claim 13, wherein said plasmid vectors<br>
expresses under biotic stress, abiotic stress, wounding, in seed endosperm, and other environmentally and artificial induced conditions.<br>
17.	A process of preparing an artificial promoter as claimed in claim 1, said method<br>
15	comprising the steps of:<br>
(a)	isolating the core promoter TATA and CaMV,<br>
(b)	designing and creating 5'translational regulator element having SEQ ID Nos. NO. 6 and 8,<br>
20	(c) assembling the sequences of step (a) and (b) in a vector system,<br>
(d)	designing and assembling a enhancer having SEQ ID No. 1 in a vector system,<br>
(e)	constructing a promoter by isolating sequences of step (c) and (d),<br>
(f)	inserting the assembly obtained in step (e) into vector system, and<br>
25	(g) studying the expression of artificial promoter of step (f) by inserting in<br>
various expression vector systems.<br>
18.	A process as claimed in claim 17, wherein SEQ ID Nos. 6 and 8 comprises of<br>
artificial exon/intron/exon regions have SEQ ID Nos. 10, 11 and 19.<br>
30<br>
19.	A process as claimed in claim 17, wherein promoter controls the gene expression in<br>
plants.<br>
20.	A process as claimed in claim 17, wherein promoter expresses under biotic stress,<br>
35	abiotic stress, wounding, in seed endosperm, and other environmentally and<br>
artificial induced conditions.<br>
21.	A process as claimed in claim 17, wherein said promoter expresses both in dicots<br>
and monocot plants.<br>
40<br>
22.	A process as claimed in claim 17, wherein the said promoter expresses in all<br>
parts/organs/cells/tissues of both monocot and dicot plants.<br>
23.	A process as claimed in claim 17, wherein the said promoter is expressed in<br>
45                 plasmid vector system selected from group comprising of Eureka-pBS, pBS-AcUc,<br>
pBSAcAcUc, pBS-ART, pPARTE, pBS-EURGUSint, pBPF-EURGGUSint,<br>
pBPFQ7-GUSint, pBPFATGUSint, pBPFARTEGUSint, pC-ARTGUSint, pC-<br>
ARTEGUSint, pBPFA19ARTGUSint, pBPFA19ARTEGUSint, pAl PARTE,<br>
pASPAl PARTE,    pASPAPARTE,	p2Al PARTE,    pBS-Ubil,    pBS-Ubi2,<br>
50	plBARTE,     pGARTE,    pC-APARTEGUSint,    pC-2AlPARTEGUSint,    pC-<br><br>
38<br><br>
2APARTEGUSint,   pC-U3ARTEGUSint,   pC-ACTIF,   pGARTEGUSint,   and pGluGUSint.<br>
24.	A process as claimed in claim 17, wherein said plasmid vectors systems express<br>
5                both in monocot and dicot plants.<br>
25.	A process as claimed in claim 17, wherein said plasmid vectors express in all the<br>
parts, tissues, cells, organs of both monocot and dicot plants.<br>
10	26. A process as claimed in claim 17, wherein said plasmid vectors expresses under<br>
biotic   stress,   abiotic   stress,   wounding,   in   seed   endosperm,   and   other environmentally and artificial induced conditions.<br>
27.	An artificial promoter substantially as herein described with reference to the<br>
15	foregoing examples and accompanying drawings.<br>
28.	A process of preparing an artificial promoter as herein described with reference to<br>
the foregoing examples and accompanying drawings.<br>
k<br>
20<br>
Dated this 16th day of July, 2005<br><br>
25<br><br>
RAJESHWARI HARIHARAN OF K &amp; S PARTNERS ATTORNEY FOR THE APPLICANT(S)<br><br>
39<br>
ABSTRACT<br>
The invention relates to an artificial promoter which is characterised in that it comprises a 5 chimeric molecule of recombinant DNA which, once introduced into plant cells of any class, promotes high expression levels of any DNA molecule that is fused to the 3' end thereof. The basic genetic elements of the inventive promoter molecule are as follows: a promoter nucleus with a consensus TATA box followed by an Exon/Intron/Exon region and a translational activity-potentiating element, all of which are produced artificially. 10 Transcriptional expression-regulating elements can be inserted upstream of the promoter in order to provide the expression with the specific time-response capacity of organ or tissue. The artificial genetic elements designed can be functionally inserted between any active promoter in plant cells and any DNA sequence in order to increase the transcription/translation levels of the latter.<br>
15<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtYWJzdHJhY3QoMTEtNC0yMDA4KS5kb2M=" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-abstract(11-4-2008).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtYWJzdHJhY3QoMTEtNC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-abstract(11-4-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtYWJzdHJhY3QuZG9j" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-abstract.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtY2FuY2VsbGVkIHBhZ2VzKDExLTQtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-cancelled pages(11-4-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtY2xhaW1zKGdyYW50ZWQpLSgxMS00LTIwMDgpLmRvYw==" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-claims(granted)-(11-4-2008).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtY2xhaW1zKGdyYW50ZWQpLSgxMS00LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-claims(granted)-(11-4-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtY2xhaW1zLmRvYw==" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-claims.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtY29ycmVzcG9uZGVuY2UoMTEtNC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-correspondence(11-4-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtY29ycmVzcG9uZGVuY2UoaXBvKS0oMS03LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-correspondence(ipo)-(1-7-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtY29ycmVzcG9uZGVuY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtY29ycmVzcG9uZGVuY2UtcmVjZWl2ZWQtMDIxMjA1LnBkZg==" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-correspondence-received-021205.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtY29ycmVzcG9uZGVuY2UtcmVjZWl2ZWQtMDkxMjA1LnBkZg==" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-correspondence-received-091205.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtY29ycmVzcG9uZGVuY2UtcmVjZWl2ZWQucGRm" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-correspondence-received.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZHJhd2luZygxMS00LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-drawing(11-4-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZm9ybSAxKDE5LTctMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-form 1(19-7-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZm9ybSAxKDItMTItMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-form 1(2-12-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZm9ybSAxMygxOS03LTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-form 13(19-7-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZm9ybSAxOCg5LTEyLTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-form 18(9-12-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZm9ybSAyKGdyYW50ZWQpLSgxMS00LTIwMDgpLmRvYw==" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-form 2(granted)-(11-4-2008).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZm9ybSAyKGdyYW50ZWQpLSgxMS00LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-form 2(granted)-(11-4-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZm9ybSAyNigxMS00LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-form 26(11-4-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZm9ybSAyNigxNi03LTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-form 26(16-7-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZm9ybSAyNigzLTgtMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-form 26(3-8-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZm9ybSAzKDE2LTctMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-form 3(16-7-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZm9ybSAzKDItMTItMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-form 3(2-12-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZm9ybSA1KDE5LTctMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-form 5(19-7-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZm9ybS0xMy5wZGY=" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZm9ybS0xOC5wZGY=" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZm9ybS0yLmRvYw==" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-form-2.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZm9ybS0yLnBkZg==" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZm9ybS0yNi5wZGY=" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZm9ybS1wY3QtaWItMzA2LnBkZg==" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-form-pct-ib-306.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZm9ybS1wY3QtaXBlYS00MDgucGRm" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-form-pct-ipea-408.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZm9ybS1wY3QtaXBlYS00MDkucGRm" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-form-pct-ipea-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZm9ybS1wY3QtaXBlYS00MTYucGRm" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-form-pct-ipea-416.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtZm9ybS1wY3QtaXNhLTIxMCgxOS03LTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-form-pct-isa-210(19-7-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtcGN0LXNlYXJjaCByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-pct-search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtcGV0aXRpb24gb2YgdW5kZXIgcnVsZSAxMzcoMTAtNC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-petition of under rule 137(10-4-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzk5LW11bW5wLTIwMDUtcGV0aXRpb24gb2YgdW5kZXIgcnVsZSAxMzgoMTAtNC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">799-mumnp-2005-petition of under rule 138(10-4-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QxLmpwZw==" target="_blank" style="word-wrap:break-word;">abstract1.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="223739-a-process-for-the-manufacture-of-commercial-grade-nodular-semi-nodular-cast-iron-alloy.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="223741-a-radio-transmission-method.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>223740</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>799/MUMNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>06/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>06-Feb-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>22-Sep-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>19-Jul-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CENTRO DE INGENIERIA GENETICA Y BIOTECNOLOGIA</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>AVE. 31, ENTER 158 Y 190, CUBANACAN , PLAYA, C. HABANA 10600</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SELMAN-HOUSEIN SOSA, GULLERMO</td>
											<td>CALLE 186 ENTER 31 Y 33, NO. 3115, APTO 6A, CUBANACAN, PLAYA, 12100 GIUDAD DE LA HABANA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SALAZAR RODRIGUEZ ALBERTO</td>
											<td>BARRUTIA NO 160 ENTRE PALLARE Y CRISTOBAL GUARDIA, GUANABACOA 11100 CIUDAD DE LA HABANA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>ABREU REMEDIOS DAYMI</td>
											<td>EDIFICIO 29 APTO 10 ENTRE GARAITA Y. CARRETERA CENTRAL, REPARTO ROMAN,60200 SANCTI SPIRITUS,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>RAMOS GONZALEZ OSMANY</td>
											<td>CALLE 170 ENTRE 51 Y 59, 5137,LISA 13500, CIUDAD DE LAD HABANA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/82</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/CU2003/000018</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-12-19</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2002-0337</td>
									<td>2002-12-27</td>
								    <td>Cuba</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/223740-a-recombinant-dna-molecule-useful-as-a-promoter-in-dicot-as-well-as-monocot-plant-cells by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 04:51:56 GMT -->
</html>
