<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/224083-an-isolated-dna-fragments by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:43:16 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 224083:AN ISOLATED DNA FRAGMENTS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">AN ISOLATED DNA FRAGMENTS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>An isolated DNA fragments The present invention is related to a transgene system, wherein harmful effects of genes spreading from one organism to another can be contained. According to the invention, the target gene is split into at least two gene fragments which are compartmentalized into different membrane bound organelles. Each gene fragment expresses a protein fragment as an inactive protein and these separately expressed fusion proteins are reconstituted into an active protein in vivo. Compartmentalization of the gene fragments allows the target protein to be reconstituted in a desired location and prevents the transmission of a functional gene to other organisms. The above strategy may be applied successfully in plants as well as animals.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM 2<br>
THE PATENTS ACT, 1970<br>
(39 of 1970)<br>
&amp;<br>
THE PATENTS RULES, 2003<br>
COMPLETE SPECIFICATION (See section 10, rule 13)<br>
'AN ISOLATED DNA FRAGMENTS'<br>
NEW ENGLAND BIOLABS INC., of 32 Tozer Road, Beverly, MA 01915, U.S.A. and BOSTON BIOMEDICAL RESEARCH INSTITUTE, 25 Grove Street, Watertown, Massachusetts 02472, U.S.A.<br>
The following specification particularly describes the invention and the manner in which it is to be performed.<br><br>
FIELD OF THE INVENTION<br>
The present invention provides isolated DNA sequences and a method of reconstituting a target protein.<br>
BACKGROUND OF THE INVENTION<br>
In the past few years, agriculture in the United States has been revolutionized by the introduction of transgenic crops that are resistant to specific diseases, insects, herbicides or have improved nutritional value. At the same time, much concern has been expressed around the world that these genetically modified (GM) agricultural products may be harmful to the consumer and that the transgenes could be transferred to related plant species so as to generate insect- or herbicide-resistant"superweeds" (Ferber, D., Science 286: 1662 (1999)) or consumed by other organisms to their detriment (Losey, et al., Nature 399: 214 (1999)). Whereas there is little scientific basis to the fear of harmful effects of "GM foods", the possibility that transgenes are transferred to other plants and thereby have an adverse ecological impact is not entirely unfounded (Bergelson, et al.. Nature 395: 25 (1998)). Such transfer could occur either by pollination of closely related species or by the transfer of gene fragments to unrelated plants by viral or plasmid vectors whose transmission may be mediated by plant-associated fungi, bacteria or insects.<br>
There have been a number of techniques discussed for the prevention of transgene spread, however these procedures either are designed to have a negative impact on the new hybrid plant (Gressel, Trends Biotechnol., 17: 361-366 (1999)), as in the case of tandem constructs or will not eliminate the possiblity of spread by horizontal gene transfer (Bertollaand Simonet, Res. Microbiol., 150: 375-384 (1999)).<br>
In this disclosure, we propose a new type of transgene that allows efficient protein expression but does not require a gene coupling approach and has a significantly lower chance of spread by horizontal gene transfer.<br><br>
SUMMARY OF THE INVENTION<br>
In accordance with the present invention, there is disclosed a new type of transgene system that allows efficient protein expression in a target host such as a plant, but avoids the undesirable result of the migration of the transgene into related host systems and/or to the environment via the pollen. The methods described herein can also be applied to the expression of virtually any protein of interest (e. g. a toxic protein) in eukaryotic (yeast, insect, mammalian cells, etc.) and prokaryotic (E. coli, etc.) organisms.<br>
In each case, the target gene is split into at least two segments, each can be fused to a portion of an intein coding sequence. Each fusion gene is expressed as an inactive protein and these separately expressed fusion proteins are reassembled into an active form. Compartmentalization of the gene fragments allows the target protein to be reconstituted in a desired location and can prevent the transmission of a functional gene to other organisms.<br>
It should be noted that although the present invention is specifically exemplified in agriculture and plant biotechnology, the approach proposed here has a much broader scope and can be applied to any gene expressed in any organism for the prevention of its accidental transfer to another organism.<br>
STATEMENT OF THE INVENTION<br>
Accordingly, the present invention relates to an isolated DNA fragment representing a gene of interest and encoding for a target protein being spliced into fragments at a pre-determinded site, and each split sites being fused to an intein in frame, wherein the fused sequences are located at a distant location in the genome or a separated chromosome.<br>
DETAILED DESCRIPTION ACCOMPANYING DRAWINGS<br>
Figure 1 A-Protein Splicing Mechanism. Protein splicing is a post-translational processing event involving the excision of an internal protein segment, the intein, from a precursor protein with the concomitant ligation of the flanking N-and C- terminal regions (the<br><br>
exteins). Sequence alignment reveals that there are highly conserved residues at the two splice junctions: a cysteine or serine residue at the N-terminus of the intein, His-Asn at the C-terminus of the intein, and Cys, Ser or Thr as the first residue of the C-terminal extein. These conserved splice junction residues are directly involved in the catalysis of peptide bond cleavage and ligation of the protein splicing reactions. The chemical mechanism of protein splicing with an intein which has cysteine residues at its N-terminus and adjacent to its C-terminus is shown in Figure 1: Step 1- Formation of a linear thioester intermediate by an N-S acyl rearrangement of Cysl at the N-terminus of the intein; Step 2-Formation of a branche intermediate by transesterification involving attack by the Cys immediately following the C- terminus of the intein on the thioester formed in Step 1; Step 3-Excision of the intein by peptide bond cleavage coupled to succinimide formation involving the intein C-terminal Asn residue; Step 4-Spontaneous S-N acyl rearrangement of the transitory ligation product from a thioester to a stable amide bond. Protein splicing involving other inteins presumably proceeds by four analogous chemical steps, except that the Cys residues shown in Figure 1 can be replace by Ser or Thr, so that Steps 1 and 4 are N-0 and O-N acyl shifts, respectively.<br>
Figure 1B-Cartoon of protein splicing.<br>
Figure 2-Trans-Splicing.<br>
Figure 2A-The association of the N-terminal and C- terminal intein fragments aligns the two splice junctions for the fusion of the N-and C-extein sequences. The splicing reaction presumably occurs via the same splicing pathway as the cis-splicing pathway proposed previously.<br>
Figure 2B-Alternatively, in the absence of splicing the intein could facilitate the association of the two extein sequences with the subsequent generation of enzymatic activity. This has been termed intein-mediated complementation.<br>
Figure 3-Ssp DnaE intein gene arrangement in Synechocystis sp PCC6803. The genome of the blue-green algae Synechocystis sp PCC6803 contains the split dnaE gene with the<br><br>
fragments located 745 kb apart. The naturally occurring trans-splicing intein fuses the two gene product fragments to produce an active polymerase.<br>
Figure 4A-Splitting of a target gene. A target gene can be split into two fragments with partial intein genes fused at the C-and N-terminal portions. These split genes can be placed into plant chromosomes so that the following expression can be reconstituted.<br>
Figure 4B-Containment of a trans-gene. The gene of interest, in this case an herbicide resistance gene, is divided into two fragments (target N and target C) and an intein (INn and INc) is fused to each partial gene. The two gene fusions are placed on separate, remote locations on the genome. One of these may be in the chloroplast, the other in the nuclear genome. The chloroplast located transgene is transcribed and translated in the chloroplast while the nuclear transgene is transcribed in the nucleus and translated in the cytoplasm. Following translation of the nuclear gene it is transported into the chloroplast with the help of chloroplast transit peptide where it can associate with the other gene fragment using the intein as either an association or splicing element.<br>
Figure 5-Trans-splicing of acetolactate synthase (ALS) in E. coli strain ER2744. The target gene is split by intein fragments (INn and INc) and expressed as two inactive partial proteins. Protein trans-splicing produces an active target protein product in host cells.<br>
Figure 6-Sequence alignment for acetolactate synthase (ALS) genes (SEQ ID NO: 42, SEQ ID NO: 43, SEQ ID NO: 44, SEQ ID NO: 45 and SEQ ID NO: 46). The gap region for E. coli acetolactate synthase II (ALSII) is underlined. The arrow indicates the split site for E. coli ALSII. The star indicates the split site for maize ALS.<br>
Figure 7-Plate assay showing that ALSIIm-14 renders E. coli ER2744 resistant to valine and herbicide, SM. E. coli ER2744 cells were transformed with plasmid DNA expressing ALSII protein (1), ALSIIm (2), ALSIIm-14 (3) and plated on M9 medium containing 0.3 mM IPTG, with 100 pg/ml of valine (a), or with 100 pg/ml valine and 50 pg/ml SM (b). The plate assay was performed at 30°C for 50 hours.<br><br>
Figure 8-Production of recombinant ALSIIm-14 through Ssp DnaE intein mediated trans-splicing. 2u of whole cell extract, from cells transformed with expression plasmids for control (lane 1), ALSII (lane 2), ALSIIm (N)-INn (lane 3), ALSIIm (C)-INc (lane 4), ALSIIm (N)-INn and ALSIIm (C)-INc (lane 5), was run on an SDS-polyacrylamide (12%) gel, transferred to a S&amp;S nitrocellulose membrane, and probed with antiserum against ALSII N-terminus (Figure 8A) or against ALSII C- terminus (Figure 8B). (Figure 8C) The efficiency of trans- splicing is temperature sensitive. Western blot was performed using a antiserum against ALSIIm N-terminus. Protein extract was made from cells transformed with expression plasmids for control E. coli extracts contain a non¬specific protein (the top band) that reacts with antiserum: (lane 1), ALSII (lane 2), ALSIIm (N)- INn and ALSIIm (C)-INc (lane 3 to lane 6). The cell culture temperature is 37°C for lane 1 to lane 3,30°C for lane 4,25°C for lane 5, and 15°C for lane 6.<br>
Figure 9-Assays for acetolactate synthase II (ALSII) Activity.<br>
Figure 9A-Co-expression of ALSIIm (N)-INn and ALSIIm (C)-INc rescued cell growth on a valine plus herbicide added plate. E. coli ER2744, transformed with expression plasmids for ALSII (1), ALSIIm (2), ALSIIm (N)-INn and ALSIIm (C)-INc (3), ALSIIm (N)- INn (4), ALSIIm (C)-INc (5), ALSIIm (N) and ALSIIm (C) (6), were plated on M9 medium at 37°C (a), 37°C with 100 IJg/ml valine (b), 30°C with 100 pg/ml valine (c), and 30°C with 100 IJg/ml valine and 50 pg/ml sulfometuron methyl (SM) (d). Plates contained 0.3mM IPTG.<br>
Figure 9B-Co-expression of ALSIIm (N)-INn and ALSIIm- (C)-INc rescued cell growth in valine and herbicide added medium. E. coli ER2744, transformed with expression plasmids for fusion proteins as indicated under graph, was cultured in M9 medium (0.3mM IPTG), with or without 100 pg/ml valine and 50 pg/ml sulfometuron methyl (SM) as indicated. OD600 was taken to determine the cell growth rate after cells were cultured for 40 hours at 30°C.<br>
Figure 9C-The time course study on the growth rate of cells expressing ALSIIm (N)-INn and ALSIIm (C)-INc. E. coli ER2744, transformed with the expression plasmids for<br><br>
proteins as indicated, was cultured at 30°C in M9 medium (0.3mM IPTG) with the addition of 100 pg/ml valine. The cell density was determined by measuring OD600 at several time points as indicated.<br>
Figure 10-Western blot detection of trans-splicing product, maize ALS-14.2 pi of whole cell lysate, from E. coli ER2744 cells transformed with expression plasmids for control (lane 1) (please note the antibody reacts with a non-specific protein in E. coli), cALS (lane 2), cALS (N)-INn (lane 3), cALS (C) INc (lane 4), cALS (N)-INn and cALS (C)-INc (lane 5), was run on a 12% SDS polyacrylamide gel, transferred to a S&amp;S Nitracellulose membrane and probed with antiserum against cALS N-terminus (A) or cALS C-terminus (B). cALS indicates corn/maize ALS protein.<br>
Figure 11-Plating Assay for Ssp DnaE intein Cis-splicing Constructs. Plasmids pCE182DnaE, pCE215DnaE, pCE235DnaE, and pCE267DnaE encode for the 5-enolpyruvyl- 3-phosphoshikimate synthetase (EPSPS) protein with the full length Ssp DnaE intein inserted at amino acid positions 182, 215,235 and 267, respectively. These were transformed into ER2799 E. coli cells (which require the EPSPS protein for viability in M9 minimal media), and plated on M9 minimal plates. Following incubation at 37°C overnight, individual clones on each plate were picked and stripped onto a single M9 minimal plate. This master plate was then incubated at 37°C overnight or RT for 2-3 days. As a control the pCYB3 plasmid was used as it carries no EPSPS gene, and there is no growth on the selection plate. pC+E2, a plasmid which contains the full length wild type EPSPS containing a ProlOlSer mutation, grows on M9 selection plate and also confers glyphosate resistance.<br>
Figure 12-Plating Assay for the Ssp DnaE intein Trans- splicing Constructs at Positions 215 and 235.<br>
The activity of each 5-enolpyruvyl-3-phosphoshikimate synthetase (EPSPS) trans-splicing construct was assayed by co-transforming the matching constructs into E. coli ER2799 cells and plating on an M9 selection plate. pCYB3 or pKYBl (New England Biolabs, Inc., Beverly, MA), which has no EPSPS gene present, was used to provide<br><br>
ampicillin or kanamycin resistance when testing the activity of each half of the EPSPS gene.<br>
The plasmids used were: pC+E2, which contains the full length EPSPS mutant gene; p215EN2, which has the first 215 amino acids of EPSPS fused to the N-terminal splicing domain of the Ssp DnaE intein; p235EN2, which has the first 235 amino acids of EPSPS fused to the N-terminal splicing domain of the Ssp DnaE intein; pEPS#28, which contains amino acids 216-427 of the EPSPS gene fused to the C-terminal splicing domain of the Ssp DnaE intein; pEPS#29, which contains amino acids 236-427 of the EPSPS gene fused to the C- terminal splicing domain of the Ssp DnaE intein; pEPS#33, which has the first 235 amino acids of EPSPS fused to a splicing defective N-terminal domain of the Ssp DnaE intein; pEPS#37, which has amino acids 236-427 of EPSPS fused to a splicing defective C-terminal domain of the Ssp DnaE intein; pEPS#34, which has the first 235 amino acids of EPSPS, but no intein fragment; and pEPS#36, which has amino acids 236- 427 of EPSPS and no intein fragment. These plasmids were co-transformed, in various combinations, into ER2799 E. coli cells, and plated on both LB plates and M9 plates, each plate was supplemented with 100 pg/mL ampicillin and 50 pg/mL kanamycin and 0.3 mM IPTG. Individual clones were picked from each LB plate and stripped on one M9 selection plate following incubation at 37°C overnight or RT for 2-3 days.The M9 minimal media selection plate contained 100 pg/mL ampicillin and 50 pg/mL kanamycin and 0.3 mM IPTG. The combinations used were: WT, pC+E2 and pKYB; 215NC, p215EN2 and pEPS#28; 215C, pEPS#28 and pCYB3; 235NC- Dead, pEPS#33 and pEPS#37; 235NC, p235EN2 and pEPS#29; 235N, p235EN2 and pKYBl; 235C, pEPS#29 and pCYB3; 235N-215C, p235EN2 and pEPS#28; and 235 complement, pEPS#34 and pEPS#36.<br>
Figure 13-Glyphosate Resistance Liquid Assay for 235 Trans-splicing Constructs. The plasmid constructs were as described in Figure 12. The combinations used were: WT, pC+E2 and pKYB; 235NC-Dead, pEPS#33 and pEPS#37; 235NC, p235EN2 and pEPS#29; 235N, p235EN2 and pKYBl; 235C, pEPS#29 and pCYB3; and 235 complement, pEPS#34 and pEPS#36. These plasmids were co-transformed into ER2799<br><br>
E. coli cells and plated on LB plates, supplemented with 100 pg/mL ampicillin and 50 pg/mL kanamycin; pCYB3/pKYB were co-transformed into E. coli ER2744, and plated on the LB plate, supplemented as described previously. A preculture was prepared for each transformation by inoculating the fresh colony into LB medium containing 100 ug/mL ampicillin and 50 ug/mL kanamycin at 30°C for overnight. Equal amounts of pre¬culture (10-11 uL depending on the cell density) was inoculated into freshly- made M9 minimal medium containing 100 ug/ml of ampicillin, 50 aug/ml of kanamycin and 0.3 mM IPTG in the absence or presence of different amounts of glyphosate. The growth of each construct was measured by OD at 600 nm. Figure 13A, growth at 37°C. Figure 13B, growth at 30°C.<br>
Figure 14-Growth of the cis-splicing 235 construct in M9 liquid minimal media. A plasmid with the full length Ssp DnaE intein inserted into position 235 of 5-enolpyruvyl-3- phosphoshikimate synthetase (EPSPS) was constructed. Two plasmid vectors were created (pCE235 DnaE and pEPS#31), one with a splicing competent Ssp DnaE intein (235 cis) and another with a splicing incompetent intein (235 dead). These plasmids were co-transformed with pKEB12 into ER2799 E. coli cells and plated on LB plates supplemented with 100 pg/mL ampicillin and 50 ug/mL kanamycin. A preculture was prepared for each transformation by inoculating the fresh colony into LB medium at 30°C for overnight. Equal amounts of pre-culture (10-11, depending on the cell density) was inoculated into freshly-made M9 minimal medium containing 100 ug/ml of ampicillin, 50 ug/ml of kanamycin and 0.3 mM IPTG. The cell density was determined at various times using the OD at 600 nm.<br>
Figure 15 is a table that shows the sites in the 5- enolpyruvyl-3-phosphoshikimate synthetase (EPSPS) protein that allow a 5 amino acid insertion and still result in active protein.<br>
Figure 16 is a table that shows the sites in the 5- enolpyruvyl-3-phosphoshikimate synthetase (EPSPS) protein where a 5 amino acid insertion results in inactive protein.<br><br>
Figure 17 is a map of pIH976. Circular double stranded DNA with a multiple cloning site. The restriction enzyme sites are indicated. Restriction sites with parenthesis are not unique. Ptac represents tac promoter. Origin of replication is ori. This plasmid has tetracyclin drug resistant marker (Tetr).<br>
Figure 18 is a map of pAGR3. Circular double stranded DNA (SEQ ID NO: 76) with a multiple cloning site. The restriction enzyme sites are indicated below. Ptac represents Tac promoter. Origin of replication is ori. This plasmid has ampicillin drug resistant marker (ampr). Lac operator and ribosome binding sites are indicated. Plasmid pAGR3 is an expression vector which includes several elements: (1) a synthetic tac promoter coupled to a symmetric synthetic lac operator sequence; (2) a lac ribosome binding site; (3) a polylinker for cloning with the ATG within the Ncol site being about seven nucleotides downstream of the ribosome binding site; (4) a copy of the laclq gene to provide repression of the tac promoter; (5) the replication origin from pBR322; (6) ampicillin resistance gene; and (7) a four-fold copy of the ribosomal transcription terminator upstream of the tac promoter. The transcription terminators lower the basal level of transcription by reducing read-through transcription from upstream promoters.<br>
Figure 19 Trans-splicing of two unrelated gene products in E. coli using the Ssp DnaE intein as splice element.<br>
Figure 19A Plasmid pIHaadE-N represents aadA gene (in black) fused to the N-terminal splicing domain of the Ssp DnaE intein (INn in grey). Plasmid pAGRE-CsmGFP plasmid represents the C-terminal splicing domain of the Ssp DnaE intein (INc in grey) and smGFP (in black). The calculated molecular mass for each of the partners is indicated below in kDa. The arrow indicates a trans-splicing event resulting in a aadA-smGFP (57 kDa) fusion protein.<br>
Figure 19B Ampicillin and spectinomycin sulphate selection of pIHaadE-N and pAGRE-CsmGFP plasmid in E. coli cells. E coli were transformed with the plasmids indicated on the right side. Colony numbers are indicated on top.<br><br>
Figure 19C Expression and detection of hybrid aadA- smGFP protein through trans-splicing. Western blot analysis of E. coli cell extracts expressing the constructs as indicated above the figure, using a monoclonal smGFP specific antibody. The relative positions of biotinylated MW markers (76,57,46, 37,28 and 20) are in kDa. The protein bands corresponding to aadA-srnGFP hybrid as well as INc-smGFP are indicated.<br>
Figure 20 is a map of pNCT 114/224. Circular double stranded DNA with a multiple cloning site capable of targeting gene/ (s) to predetermined locus. The restriction enzyme sites are indicated. PpsbA and TpsbA represents photosynthetic polypeptide Dl gene promoter and terminator respectively. Origin of replication is ori. This plasmid has ampicillin drug resistant marker (ampr). The homologous recombination sequences are indicated as left border (or£22$-ssb for pNCT114 and 16SrDNA-trnaV for pNCT224) and right boarder (orfl244 for PNCT114 and rps7/12for pNCT224). CS represents the cloning sites.<br>
Figure 21 Plant promoter PpsbA activity in E. coli and Trans-splicing of aadA and smGFP.<br>
Figure 21A Plasmid p115ag/p225ag represents aadA gene (in black) fused to the Ssp DnaE intein N-terminal domain (INn in grey) and the Ssp DnaE intein C-terminal domain (INc in grey) fused to smGFp (in black). Both the hybrid genes are transcribed in opposite directions. The calculated molecular mass for each of the partner is indicated below in kDa. Arrow indicates a trans-splicing event resulting in a fused aadA-srnGFP (57 kDa) protein.<br>
Figure 21B Ampicillin and spectinomycin sulphate selection of pll5ag and p225ag plasmid in E. coli cells. E coli were transformed with the plasmids indicated on the right side. Colony identities are indicated on top. The digit after the plasmid is the isolate number. A plus symbol ("+") indicates the growth of the plasmid with the indicated antibiotics.<br><br>
Figure 21C Expression and detection of hybrid aadA- smGFP protein through trans-splicing. Western blot analysis of E. coli cell extracts expressing the constructs as indicated above the figure, using a monoclonal smGFP specific antibody. The relative positions of biotinylated MW markers are to the left in kDa. The protein bands corresponding to aad-smGFP hybrid as well as INc-smGFP are indicated.<br>
Figure 22 Splicing in cis in plant cytoplasm. 5- enolpyruvyl-3-phosphoshikimate synthetase (EPSPS) and acetolactate synthase (ALS) genes are inserted in to the binary vector pBI121. The amino and carboxy terminal fragments of EPSPS or ALS are indicated in black. The Ssp DnaE intein (Intein) gene is flanked on either side by EPSPS/ALS fragment. Right and left boarder of the Agrobacterium is indicated as LB and RB. CaMV 35S promoter, NOS promoter (PNOS) and NOS terminator (TNOS) are indicated.<br>
Figure 23 Nuclear transfer vector pBITPEC or pBITPECsmGFP. This binary vector has the CaMV35S promoter driving the rubisco3A transit peptide (TP) that is fused to the Ssp DnaE intein C-terminal splicing domain (INc). Genes to be cloned for organelle transport are indicated after INc. In case of pBITPECsmGFP the smGFP gene is cloned in to the multiple cloning site.<br>
Figure 24 is the psbA promoter (PpsbA) sequence (SEQ ID NO: 59).<br>
Figure 25 is the psbA terminator (TpsbA) (SEQ ID NO: 60).<br>
Figure 26 is the Rubisco3 transit peptide (SEQ ID NO: 61). Nucleotides in lower case represent codon optimized units.<br>
Figure 27 is the chloroplast gene targeting vector (pNCTl 14) (SEQ ID NO: 62). Features of pNCT114 include: (1) vector backbone: pLITMUS28; (2) Inserted in BssHII to BsiWI the left border. (orf228-ssb, 1210 bp) chloroplast genome targeting fragment; (3) inserted in Avrll to Kpnl the right border, (orfl244,1550 bp) chloroplast genome targeting<br><br>
fragment; and (4) addition of PpsbA and TpsbA between BsiWI and PstI, whereas the other pair is between Avrll and Ncol site.<br>
Figure 28 is chloroplast gene targeting vector (pNCT224) (SEQ ID NO: 63). Features of pNCT114 include: (1) vector backbone: pLITMUS28; (2) Inserted in BssHII to BsiWI the left border, (16SrDNA-trnaV, 1680 bp) chloroplast genome targeting fragment; (3) inserted in Avrll to Kpnl the right border, (rps7/12,1310 bp) chloroplast genome targeting fragment; and (4) addition of PpsbA and TpsbA between BsiWI and PstI, whereas the other pair is between Avrll and Ncol site.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
Protein splicing involves the excision of an intervening sequence from a polypeptide with the concomitant joining of the flanking sequences to yield a new polypeptide (Chong, et al., J. Biol. Chem., 271: 22159-22168 (1996)), as illustrated in Figure 1A and 1B. The elucidation of the mechanism of protein splicing has led to a number of intein-based applications (Comb, et al., U. S. Patent No. 5,496,714; Comb, et al., U. S. Patent No. Camarero and Muir, J. Amer. Chem. Soc, 121: 5597-5598 (1999); Chong, et al., Gene, 192: 271-281 (1997), Chong, et al., Nucleic Acids Res., 26: 5109-5115 (1998); Chong, et al., J. Biol. Chem., 273: 10567-10577 (1998); Cotton, et al., J. Am. Chem. Soc, 121: 1100-1101 (1999); Evans, et al., J. Biol. Chem., 274: 18359-18363 (1999); Evans, et al., J. Biol. Chem., 274: 3923-3926 (1999); Evans, et al., Protein Sci., 7: 2256- 2264 (1998); Evans, et al., J. Biol. Chem., 275: 9091-9094 (2000); Iwai and Pluckthun, FEBS Lett. 459: 166-172 (1999); Mathys, et al., Gene, 231: 1-13 (1999); Mills, et al., Proc. Natl. Acad. Sci. USA 95: 3543-3548 (1998); Muir, et al., Proc. Natl. Acad. Sci. USA 95: 6705-6710 (1998); Otomo, et al., Biochemistry 38: 16040-16044 (1999); Otomo, et al., J. Biolmol. NMR 14: 105-114 (1999); Scott, et al., Proc. Natl. Acad. Sci. USA 96: 13638-13643 (1999); Severinov and Muir, J. Biol. Chem., 273: 16205-16209 (1998); Shingledecker, et al., Gene, 207: 187-195 (1998); Southworth, et al., EMBO J. 17: 918-926 (1998); Southworth, et al., Biotechniques, 27: 110-120 (1999); Wood, et al., Nat. Biotechnol., 17: 889-892 (1999); Wu, et al., Proc. Natl. Acad. Sci. USA 95: 9226-9231 (1998a); Wu, et al., Biochim Biophys Acta 1387: 422-432 (1998b); Xu, et al., Proc. Natl.<br><br>
Acad. Sci. USA 96: 388-393 (1999); Yamazaki, et al., J. Am. Chem. Soc, 120: 5591-5592(1998)).<br>
Protein splicing in trans has recently been described both in vivo and in vitro (Shingledecker, et al., Gene 207: 187 (1998), Southworth, et al., EMBO S. 17: 918 (1998); Mills, et al., Proc. Natl. Acad. Sci. USA, 95: 3543-3548 (1998); Lew, et al., J. Biol. Chem., 273: 15887-15890 (1998); Wu, et al., Biochim. Biophys. Acta 35732: 1 (1998b), Yamazaki, et au.,-7. Am. Chem. Soc. 120: 5591 (1998), Evans, et al., J. Biol. Chem. 275: 9091 (2000); Otomo, et al., Biochemistry 38: 16040- 16044 (1999); Otomo, et al., J. Biolmol. NMR 14: 105-114 (1999); Scott, et al., Proc. Natl. Acad. Sci. USA 96: 13638-13643 (1999)) and provides the opportunity to express a protein as two inactive fragments that subsequently can undergo ligation to form a functional product (Figure 2).<br>
Trans-protein splicing also occurs naturally in Synechocystis sp PCC6803 (Wu, H., et al., Proc. Natl, Acad. Sci. 95: 9226 (1998)), where it is essential for forming a functional DNA polymerase 111 by joining two fragments of the DnaE protein, encoded by two genes separated by 750 kb of chromosomal DNA (Figure 3).<br>
These observations led the present inventors to investigate whether a functional gene product could be generated by splitting the gene of interest into two fragments and fusing an intein fragment to each partial target gene. Expression of the two protein fragments followed by trans- splicing, intein mediated complementation, or protein complementation would generate an active form of the target protein (Figure 4). In this scenario the target gene fragments can be located anywhere in the host genome, including being widely separated in the nucleus, chloroplast, mitochondria, plasmids, bacterial artificial chromosomes, yeast artificial chromosomes, or any combination of these. Furthermore, by placing the gene fragments into different organelles or plasmids, such as one half in the nucleus and the other half in the chloroplast or mitochondria of a plant, the transfer of both gene halves, needed to reconstitute the fully active target protein, for example, to a distant relative by pollination or by horizontal gene transfer via a bacterial, fungal, or viral vector would be virtually eliminated. This would greatly reduce and<br><br>
possibly eliminate the risk of the spread of a transgene outside of its relevant environment.<br>
Two examples of splitting a target gene and reconstituting activity using a protein splicing element are described below. The two genes investigated were mutant forms of the acetolactate synthase (ALS) gene from Escherichia coli and the 5-enolpyruvyl-3-phosphoshikimate synthetase (EPSPS) gene from Salmonella typhimurium, which confer resistance to the sulfonylurea and glyphosate herbicides, respectively. Both enzymes are involved in the biosynthesis of protein building blocks. ALS is the first common enzyme in the biosynthesis of branched-chain amino acids (LaRossa and Schloss, J. Biol. Chem., 259: 8753-8757 (1984); Chaleff and Ray, Science, 223: 1148-1151 (1984); Falco and Dumas, Genetics, 109: 21-35 (1985)) while EPSPS is required in the synthesis of aromatic amino acids (Stalker, et al., J. Biol. Chem. 260: 4724-4728 (1985)). Inhibition of these enzymes by chemical compounds can lead to the death of the organism.<br>
The commonly used sulfonylurea herbicides (SU), such as sulfometuron methyl (SM) (Short and Colburn, Toxicol Ind. Health, 15: 240-275 (1999)), block the growth of bacteria, yeast and higher plants by inhibiting acetolactate synthase (ALS) (EC 4.1.3.18). In order to generate herbicide resistant plants, there was a great effort in identifying a mutant ALS gene which permits growth in the presence of SM. The mutations which render bacteria and yeast resistant to SM were the first to be reported (Hill, et al., Biochem. J., 335: 653- 661 (1998)). Subsequently, similar point mutations were confirmed in the ALS genes isolated from naturally occurring resistant crops, corn, cocklebur and tobacco (Lee, et al., EMBO J., 7: 1241-1248 (1988); Bernasconi et al., J. Biol. Chem., 270: 17381-17385 (1995)). Some of these SU tolerant crops, such as corn ICI8532 IT and Pioneer 3180 IR have been commercialized.<br>
In Example I below, the herbicide resistant gene was split and an intein fragment fused in-frame to each partial gene. The split gene was determined to confer resistance to the herbicide SM in E. coli. E. coli was used as a model system since it contains the active ALSI and acetolactate synthase III (ALSIII) enzymes, but not an active ALSII. ALSI and ALSIII are the two isoforms of ALS genes in E. coli which are crucial for the synthesis of<br><br>
valine, isoleucine and leucine (DeFelice, et al., Ann. Microbiol. (Paris) 133A: 251-256 (1982)). Their activity is sensitive to valine feedback inhibition. Therefore, by saturating the growth medium with valine, ALSI and III will be inhibited and the cells will stop growing. By introducing a recombinant ALSII into E. coli cells, their growth will be rescued since ALSII is resistant to valine inhibition. This feature makes E. coli strain ER2744 a good in vivo model system for investigating the activity of the E. coli ALSII gene genetically modified by a linker insertion or a trans-splicing intein element.<br>
The second herbicide resistant gene tested was the aroA gene from Salmonella typhimurium that has a C301 to T mutation (Stalker, et al., J. Biol. Chem. 260: 4724 (1985)). This encodes the 5-enolpyruvyl-3-phosphoshikimate synthetase (EPSPS) (EC 2.5.1.19) protein with a ProlOl to Ser change and is known to confer resistance to the herbicide glyphosate (commonly marketed as Round-Up@). In this embodiment, an N-terminal fragment of the EPSPS gene was fused to the N- terminal splicing domain of the Ssp DnaE intein and the C- terminal fragment of the EPSPS gene was fused to the C-terminal splicing domain of the Ssp DnaE intein. In order to determine the sites in the EPSPS protein that would tolerate the insertion of an intein, a linker scanning experiment was performed (Biery, et al, Nucleic Acids Res., 28: 1067-1077 (2000)) (GPS-LS from New England Biolabs, Inc., Beverly, MA) that randomly inserted 5 amino acids throughout the protein sequence. Inteins were inserted into those sites found to be tolerant of amino acid insertion. Trans-splicing constructs were then created that placed the gene fusion encoding the N-terminal fragment of EPSPS fused to the N-terminal domain of the Ssp DnaE intein on one plasmid and the C-terminal portion of EPSPS fused to the C-terminal splicing domain of the Ssp DnaE intein on another plasmid. For example the EPSPS protein could be split at the site corresponding to Gly235. The two plasmids were co-transformed into E. coli cells which lacked a functional EPSPS protein and cell growth on M9 minimal media in the presence or absence of the herbicide glyphosate was observed.<br>
The activity of both the split ALS and the split EPSPS herbicide resistant genes were observed whether the intein was unmodified or had its catalytic residues changed, thus<br><br>
eliminating trans-splicing activity. This indicated that although splicing would generate a covalently attached protein product, it is not necessary to do so in every situation. The intein in this manifestation would work as an affinity domain to bring the two protein fragments together and in the correct orientation. In these experiments the presence of the intein was absolutely required for activity of the split proteins. This is based on the observation that both the split ALS and EPSPS genes without an intein fusion were not able to allow E. coli growth on the appropriate herbicides.<br>
In one embodiment of the invention, two gene fragments, fused to an intein splicing domain, are introduced independently into nuclear chromosomes, using selectable markers such as resistance to an antibiotic or other growth inhibitors to verify gene transfer. Independent transfer of the two fusion genes will assure a remote location on the plant genome, probably on separate chromosomes, thus excluding the possibility that both genes could be acquired by a single virus or plasmid vector for transfer to other organisms. If so desired, the remote location of the two genes can be assured by targeting to specific sites by homologous recombination with known DNA sequences.<br>
In another embodiment, one of the two fusion proteins is transformed into the cell nucleus and the other into chloroplasts, so as to eliminate virtually any chance of gene transfer to related plants by any conceivable mechanism, including cross-pollination of related species, since only inactive fragments of the gene would be present in the pollen. The gene fragments in chloroplast are maternally transmitted and cannot be transmitted through pollen. The same consideration would apply to gene fragments expressed in mitochondria.<br>
This technology may also be applied to non-plant systems. By way of example, a transgene to be compartmentalized could be split and an intein fused to the gene fragments. In the case of bacteria, the split genes are preferably placed far apart on the bacterial chromosome using standard chromosomal transformation techniques. As a further control measure the gene segments may also be arranged in opposite orientations. Another manifestation of this method is to split a target transgene in two and fuse to the appropriate intein domains prior to insertion of the split gene into a eukaryotic cell to<br><br>
prevent the transgene's activity from being spread to the environment or neighboring cells. The split gene is also placed far apart on the eucaryotic chromosome or placed on separate chromosomes. Furthermore, the gene fragments may be located in separate organelles such as the nucleus and mitochondria. The gene fragment in mitochondria is maternally transmitted.<br>
One application of the present invention is in preventing the spread of complete transgenes to the environment from transgenic plants. This is accomplished by splitting the transgene fusions into two or more fragments and fusing these to intein fragments. The partial transgene fusions are located in separate compartments, such as one portion in the nuclear DNA and the second portion in the chloroplast DNA.<br>
Following expression of the partial genes, the protein fragments are directed to the site of activity where they associate to reconstitute the target protein activity. Only the transgene fragment present in the nucleus is spread through pollen since the chloroplast DNA is passed to the next generation only maternally. This will vastly reduce the spread of the complete transgene to the environment.<br>
Another advantage of the present invention is that the host cells expressing only one inactive fusion protein species of a protein can be handled safely, thereby reducing the risk of exposing humans and the environment to the target protein, which may be a toxin, etc. Also, splitting a target gene into two separate loci greatly reduces the chance of transferring the entire protein coding sequence into other organisms through DNA carriers (plasmid, virus, cosmid, etc.) or other means (cell fusion, etc.). One hypothetical case is to express a toxic gene, for example the diphtheria toxin. The diphtheria toxin protein is an extremely toxic protein to human and animal cells and needs to be handled extremely carefully. This protein has been tested in preclinical and clinical phase I trials for use as a drug to eradicate tumor cells (Kelley, Proc. Natl. Acad. Sci. USA 85 (11): 3980-3984 (1988); Alexander, Neuron 3 (1): 133-139 (1989); Maxwell, et al., Cancer Res. 51 (16) 4299-4304 (1991); Madshus, J. Biol. Chem., 269 (26): 17723-17729 (1994); Murphy and vanderSpeck, Semin Cancer Biol. 6 (5): 259-267 (1995); Rozemuller and Rombouts, Leukemia, 12 (5): 710-717 (1998); Veggeberg, Mol. Med. Today 4 (3): 93<br><br>
(1998); Kreitman, Current Opin. Immunol., 11 (5): 570-578 (1999); Vallera, et al., Protein Eng. 12 (9): 779- 785 (1999)). Therefore it would be advantageous to split the diphtheria toxin gene into two intein fusion DNA segments and express them in two different bacteria or yeast strains. The two fusion proteins can be mixed, when it is needed, to assemble the toxin.<br>
Thirdly, by compartmentalizing at least one of the fragments of the target gene into an organelle that is subject to maternal inheritance (e. g., chloroplasts or mitochondria), the genetic transfer of the functional gene to related organisms through processes such as cross-pollination can be avoided.<br>
The invention described may also be utilized as a means for expressing any gene of interest in transgenic animals. Transgenic animal models have been widely used as a scientific tool to conduct biomedical studies or to produce desired proteins. Transgenic mice and other transgenic animals, such as transgenic fish, frog, rat, cow, pig, etc. have been shown to express human genes (or a foreign gene) for research and commercial purposes, such as production of a vaccine or therapeutic agent, or used as an animal model for human disease (Alexander, Neuron 3 (1): 133-139 (1989); Groner, et al., J. Physiol. 84 (1): 53-77 (1990); Patil, et al., Neuron 4 (3): 437-447 (1990); Aloe, et al.. Growth Factors 9 (2): 149-155 (1993); Aguzzi, et al, Brain Pathol. 4 (1) 3-20 (1994); Groner, et al., Biomed. Pharmacother. 48 (5-6) 231-240 (1994); Schorderet, Experientia 51 (2): 99-105 (1995)). One of the concerns is that the transgenic animal may acquire an undesired foreign gene and pass it on to the next generation and thereafter. This would result in genetically altered animal strains, which may have unforeseen social and ethical consequences. In accordance with the present invention, such a transgene can be split into two inactive fusion DNA fragments. One of them could be genetically integrated into an animal genome and the other fragment could be supplie by a DNA carrier (such as virus, etc.) which cannot be incorporated into the genome. Therefore, when one fusion protein from the animal and the other from the DNA carrier co- express, the fusion proteins will reassemble, trans-splice and produce an active protein. This gene arrangement can<br><br>
prevent animals from acquiring an intact foreign gene, thereby avoiding genetic contamination.<br>
The compartmentalization of two gene fragments is an extension of trans-splicing. The protein in question is divided into fragments and the appropriate split genes separated onto the same or different DNA molecules. For example, the genes for the two halves of the DnaE protein from Synechocystis sp PCC 6803 with the Ssp DnaE or Ssp DnaB intein splicing domains fused to the appropriate fragments (Wu, et al., Proc. Natl. Acad. Sci. USA, 95: 9226-9231 (1998a); Wu, et al., Biochim Biophys Acta 1387: 422-432 (1998b)) could be divided so that one half is in the nucleus and the second half is in the mitochonria of a specified organism.<br>
In carrying out the present invention, one must employ one or more of the following methods:<br>
(1)	identifying a suitable split site on the target transgene;<br>
(2)	the methodology for splitting the gene into two or more fragments and fusing each fragment to a split intein;<br>
(3)	the methodology for successfully generating the split gene product into a functional enzyme or protein;<br>
(4)	the methodology for screening the host cell for active gene'product or organism;<br>
(5)	location of split gene sequences in the relevant cellular compartment;<br>
(6)	a method of splitting the target gene into more than two fragments;<br>
(7)	use of protein complementation to present transgene spread; and<br>
(8)	introduction of the transgene-<br>
9) A method for identification of a suitable split site on any transgene<br><br><br>
One preferred method for identifying a split site on the transgene is based on the structural analysis of the protein of interest or its analogs and by sequence homology. This approach involves studying the known biochemical and X-ray, NMR or related structural information in order to determine a preferable intein insertion site and/or sites to divide the protein into fragments. In particular, one should determine which are the pertinent reactive amino acid residues and their spacing and spatial arrangement within the protein. If possible, it may be ideal to split the target gene so that catalytic amino acids are distributed onto each fragment. This will increase the likelihood that neither fragment will have activity alone. The protein split site may be anywhere in the protein, but initial sites for testing should be loops or linkers present between secondary motifs such as beta sheets or alpha helices. The first loops chosen should not be part of the catalytic site, although the eventual split site may be located there. As a first trial, the preferred split site would be a loop or linker region between two folding domains within a protein. This increases the possibility that the protein fragments will fold properly when expressed separately.<br>
If no biochemical or structural information is available for the protein of interest, then the alignment of similar protein sequences from different organisms or of similar protein sequences from the same organism may be informative. The protein alignment could be by sequence comparison by traditional methods or using any of a variety of computer programs such as GCG (Genetics Computer Groups, Madison, WI.) Regions of high conservation between similar proteins in all likelihood represent areas of general importance and splitting the protein in a region of high conservation should be reserved for later testing. Instead one should determine regions of low conservation, preferably regions that also vary in amino acid number that lie between regions of high conservation. The low conservation indicates that there is a low probability of a catalytic residue being present and the variation in amino acid residue length indicates that the exact spacing between the conserved domains may not be dictated by this stretch of amino acids. These properties would be advantageous for a site of intein insertion and splitting a target protein.<br><br>
Also, when choosing the site to insert an intein in the protein of interest one should test sites that possess amino acid residues favorable for the splicing activity of the intein being tested. Preferably a site in the target protein that was similar or identical to the naturally occurring extein residues of the intein under investigation could be chosen. Alternatively, residues known to facilitate proficient splicing may be inserted together with the intein. In this case, following the splicing reaction these residues would be present in the sequence of the spliced product and may alter the activity of the target protein. The effect of these extra residues on the target protein should be tested by inserting the extra amino acids into the target protein and checking for the desired property or activity. Another preferred method is based on systematic scanning of a protein of interest by random linker insertion. Linker scanning can be performed by many methods (Gustin, et al. Methods Mol. Biol. 130: 85-90 (2000); Hobson, et al. Methods Mol. Biol. 57: 279-285 (1996); Biery, NucleicAcids Res. 28: 1067-1077 (2000)). This protocol generates a library of genes with extra stretches of DNA randomly inserted throughout. When this library is translated it produces a set of proteins with extra amino acid residue (s) inserted in different positions. The library is then screened for the desired property of the target protein. For example, if the target protein confers resistance to an herbicide then the library is screened to determine which of the proteins with the extra amino acid residues can allow growth of the target organism in the presence of an herbicide. A list of sites in a protein that can tolerate extra amino acids is created. If structural or biochemical information is available, this list can be compared with the known information. An ideal case would involve choosing a split site that tolerates the extra amino acid insertion and is present in a linker or loop region and results in catalytic residues being located on different fragments. If no structural information is available then one would preferably begin by splitting the gene at the tolerant site closest to the middle of the target protein and continue testing split sites outward from there until the desired activity can be reconstituted. In both methods a preferred insertion site would also posses the native extein sequence for the intein being used, although this is not required. The fusion proteins may have optimized amino acid residues at the splice junctions that allow for a functional product to be assessed.<br><br>
(2) A method for splitting a gene and fusing each gene fragment in-frame to a split intein coding sequence<br>
Once a site to split a gene of interest has been determined (see above), then the target gene is split into two or more fragments using common genetic techniques (Sambrook, et al., Molecular Cloning : A Laboratory Manual, 2nd Edition, Cold Spring Harbor Laboratory, NY: Cold Spring Harbor Laboratory Press (1989)). For example, PCR primers, with appropriate restriction sites, may be designed so that one corresponded to the start of the target gene and the other to the sequence at the split site. Another set of PCR primers may be designed that correspond to the split site and the other end of the target gene. The two target gene fragments are then amplified by PCR (Sambrook, et al., supra) and cloned into a plasmid vector with the same unique cloning sites present in the PCR primers. Once cloned into separate vectors, intein fragments would be fused to the target genes. In one method, the C-terminal end of DNA coding for an N- terminal portion of the target protein would be fused to the N-terminal end of the DNA coding for an N-terminal portion of the intein, and-in a separate fusion-the N-terminal end of DNA coding for a C-terminal portion of the target protein would be fused to the C-terminal end of DNA coding for a C- terminal portion of the intein.<br>
These gene fragment fusions are then transferred to the same or separate expression vectors and transformed into bacterial or eucaryotic cells, existing as single or multicellular organisms, to screen for the desired activity of the target protein. It should be noted that the gene fragments in question could be cloned using restriction sites within or external to the intein gene present either naturally or added by mutation. Also, recombination sites may be used instead of restriction enzyme sites for the movement of the gene by recombination. The gene or gene fragments may then be transferred and/or expressed from a plasmid vector, a viral genome or the genome of a bacterial, eucaryotic, or archeal organism. One preferred method is to utilize a naturally occurring trans-splicing intein, for example the intein from the dnaE gene of Synechocystis species PCC6803 (Wu, et al., Proc. Natl. Acad. Sci. USA 95: 9226-9231 (1998)). However, any of the known inteins could be used (See InBase at http://www. neb. com/neb/frame_tech.<br><br>
html; Perler, et al., Nucleic Acids Res., 28; 344-345 (2000)). This would involve splitting the full length intein in order to generate the desired affinity or trans-splicing domains. One method would be to split the full length intein in the linker region between the blocks B and F of the protein splicing domains (Petrokovski, Protein Sci. 7: 64-71 (1998); Perler, et al., Nucleic Acids Res. 25: 1087-1093 (1997); Perler, et al., Nucleic Acids Res., 28: 344-345 (2000)).<br>
(3) Creating a functional protein from expressed split fragments<br>
The next step is to use an intein as an affinity domain to facilitate complementation and reconstitution of the N-and C- terminal halves of a protein into a functional enzyme. The sites to determine protein splitting would be as described in (1) above and the cloning of the target gene fragments and the addition of the intein domains as described in (2). In this case the intein fragments need not cause splicing of the two protein fragments to reconstitute enzyme activity. In one preferred embodiment, the intein domains would be mutated to abolish the possibility of splicing activity and would act only as a facilitator of protein complementation. The intein splicing activity could be abolished by mutating the amino acid residues involved in the splicing reaction (Xu, et al., EMBOJ. 15: 5146-5153 (1996); Chong, et al., J. Biol. Chem. 271: 22159- 22168 (1996); Chong, et al., Biochem. Biophys Res. Commun., 259: 136-140 (1999); Chong, et al., Gene, 192: 271-281 (1997); Chong, et al., Nucleic Acids Res., 26: 5109-5115 (1998); Chong, et al., J. Biol. Chem., 273: 10567-10577 (1998); Chong and Xu, J. Biol. Chem., 272: 15587-15590 (1997); Evans, et al., J. Biol. Chem., 274: 18359-18363 (1999); Evans, et al., 1. Biol. Chem., 274: 3923-3926 (1999), Evans, et al., Protein Sci., 7: 2256-2264 (1998); Evans, eta L, J. Biol.<br>
Chem., 275: 9091-9094 (2000); Mathys, et al, Gene, 231: 1-13 (1999); Paulus, Chem. Soc. Rev., 27: 375-386 (1998); Perler, et al., Nucleic Acids Res., 25: 1087-1093 (1997); Pietrokovski, et al., Protein Sci., 3: 2340-2350 (1994); Pietrokovski, et al., Protein Sci., 7: 64-71 (1998), Scott, Proc. Natl. Acad. Sci. USA, 96: 13638-13648 (1999), Shingledecker, et al., Arch Biochem. Biophys. 375: 138-144 (2000); Southworth, et al., Biotechniques 27: 110-120 (1999); Telenti, et al, J. Bacterid., 179: 6378-6382 (1997); Wood, et al.,<br><br>
Nat. Biotechnol., 17; 889- 892 (1999); Wu, et al., Biochim Biophys Acta 1387: 422-432 (1998b); Wu, et al, Proc. Natl. Acad. Sci. USA 95: 9226-9231 (1998a)).<br>
In another embodiment the intein affinity domain could retain its normal catalytic residues. Furthermore, the intein may be comprised of a deletion or mutant form such that it is significantly smaller or larger or contains non-native amino acid residues when compared to its original primary sequence. The deletion forms of the intein could be created by sequentially decreasing the size of the intein either at the gene level or proteolytically and then testing for affinity activity. The affinity activity could be tested by using the split herbicide resistant gene and fusing the new deletion mutant to the appropriate herbicide resistant gene fragments and looking for growth on the herbicide in question. Mutants of the intein fragment could be formed by error prone PCR, linker scanning, site directed mutagenesis, or by mutagenic compounds and the activity of the intein fragments tested as described above. Note the herbicide resistance gene could be substituted by a drug resistance gene, green fluorescent protein or any selectable marker. The affinity of the intein fragments could also be tested by immobilizing one fragment on a solid support and testing for the binding of the second fragment to the first fragment.<br>
(4) A method of screening for constructs producing active proteins of interest in a suitable host cell or organism<br>
The screen for the target gene activity will vary with the target gene but could be by in vitro assay following expression and purification or in a crude cell lysate or in vivo by determining protein activity by cell phenotype, such as viability, morphology, sensitivity, or insensitivity to a drug or compound, appearance, or ability to bind or not bind a specific molecule or compound. One preferred method is to use E. coli as host cells to test, for example, herbicide resistant activity of the re-assembled product of a split gene. The E. coli cells must be sensitive to the herbicide in question. The target gene fragments, with the intein fusion, is present on a plasmid or plasmids and is transformed into E. coli cells using standard techniques.<br><br>
The gene fusions are expressed either constitutively or by an inducible promoter. E. coli are then tested for growth under selection conditions, i. e. in the presence of herbicide, in both the presence or absence of the appropriate gene fragments. Growth in the presence of the gene fragments indicates the reconstitution of the target protein activity. The E. coli cells could be substituted with any bacterial, archaea, or eucaryotic cell type (either single or multicellular) as well as a virus by employing techniques well known in the art.<br>
Furthermore, both of the target gene fragments could be present in the genome of the organism, or one fragment could be present in the genome and the other in a plasmid or some other vector. The target protein fragments could be expressed in one organism together or separately and added to another cell type for assay. The fusion could be tested directly in plant cells or other multicellular organisms by placing the transgene fragments in the host organisms nuclear, chloroplast, or mitochondrial genome and determining if the desired activity is present. The target gene or protein fragments could be delivered by a bacterial, fungal, viral, micellar, mechanical (biolistic) or similar vector to the cell type or organism to be tested.<br>
(5)	Location of split genes<br>
The present invention also comprises location of the split target gene sequences in different cellular compartments, different locations on the chromosome, or different vectors. One preferred method is to position the two split gene sequences in the nucleus, chloroplast, mitochondria, bacterial artificial chromosome, yeast artificial chromosome, plasmid, preferably not both in any one of the aforementioned. Location of fragments can be accomplished in accordance with standard molecular biology techniques. In order to reconstitute the gene product from its fragments, the appropriate gene fragments must be fused to a targeting/localization sequence so that their protein products are transported into a cellular compartment (e. g., the chloroplasts) where functional reconstitution can occur.<br>
(6)	A method of splitting the target gene into two or more fragments<br><br>
The present invention also embodies methods for splitting the target gene into two or more fragments and reconstituting the desired activity by trans-splicing, intein mediated complementation or protein complementation of all the necessary fragments. For example, inteins with differing affinities could be attached to the target protein fragments so that they ressemble the active protein, in a manner described previously (Otomo, et al., Biochemistry, 38: 16040- 16044 (1999); Otomo, et al., J. Biomol. NMR, 14: 105-114 (1999)). In this case each fragment could be located far apart in the chromosome, on a separate chromosome or in multiple locations as described above, except that the number of locations could match the number of fragments the protein was divided into.<br>
(7)	Use protein complementation in the prevention of transgene spread<br>
This protocol uses the natural complementation activity of two protein fragments to reconstitute the desired protein property. The two genes encoding the protein halves may be located in the nucleus, chloroplast, mitochondria, bacterial artificial chromosome, yeast artificial chromosome, plasmid or any combination of those organelles or vectors. Following expression, both protein fragments may be targeted to the site of protein action and the desired protein property generated by complementation of the protein fragments. Protein complementation has been reported previously (Rossi, et al.. Trends Cell Biol. 10: 119-122 (2000)) and so makes a viable alternative to using an intein as a complementation domain. The procedures necessary to carry out this experiment are similar to what has already been discussed except no intein fusion is used. A site to split a target gene is determined as described in (1). The transgene fragments are cloned as described in (2), except that an intein is not used as a fusion partner. The screening for activity of the split protein is conducted as described in (4).<br>
(8)	Introducing a transgene into an organism by viral infection<br>
In yet another embodiment, the two transgene fragments, either intein fusions or not, may be packaged into separate viral particles. These viruses co-infect an organism and both transgenes are expressed. The desired protein property is generated following protein splicing, intein mediated complementation, or protein complementation. One preferred<br><br>
method comprises choosing the split site, clone the fragments and check for activity in trans as described in (1), (2), and (4). The appropriately split transgene or transgene-intein fusions are packaged into adenovirus. The adenoviruses containing the appropriate transgenes can be introduced into a subject organism and upon transfection introduce the two gene fragments so that the target protein activity can be expressed.<br>
BRIEF DESCRIPTION OF THE EXAMPLES<br>
In Example I, we demonstrate a method of splitting a herbicide resistant gene by an intein. We show how to select potential split sites in the E. coli herbicide resistant gene encoding for acetolactate synthase (ALS) based on the sequence homology analysis and the crystal structure of the protein of interest or its analog. The DNA fragment encoding for the N-terminal 327 amino acid residues of the ALS protein was fused in frame to the N-terminal 123 amino acids of the Ssp DnaE intein while the DNA fragment encoding for the C- terminal 221 amino acid residues was fused in frame to the C- terminal 36 amino acids of the Ssp DnaE intein. A plasmid vector bearing one of the fusion genes was expressed as an inactive ALS protein fragment. When both fusion gene vectors were introduced into the same host cell and co- expressed, the two inactive fusion proteins underwent trans- splicing to produce a functional enzyme in vivo, conferring herbicide resistance to the E. coli host cells. This approach may be applied to selection of suitable sites in any gene for fusion to an intein sequence.<br>
In Example II, we demonstrate how to choose a split site in the maize ALS gene based on the sequence homology of the maize ALS gene and its E. coli counterpart, ALSII gene. The DNA encoding the N-terminal 397 amino acid residues of the maize ALS gene was fused in-frame to the DNA sequence encoding the N-terminal 123 amino acids of the Ssp DnaE intein while the DNA fragment encoding the C-terminal 241 amino acid residues was fused in frame to the DNA encoding the C-terminal 36 amino acids of the Ssp DnaE intein. We show that, when the two fusion genes were co-expressed, the two fusion proteins underwent trans-splicing to produce a protein product of expected size for the mature protein.<br><br>
In Example III, we demonstrate a method of identifying potential split sites in a mutant S. typhimurium aroA gene encoding 5-enolpyruvyl-3-phosphoshikimate synthetase (EPSPS) based on transposon random linker insertion. Two sites at amino acid positions 215 and 235 of EPSPS among all 42 potential sites were chosen to split the EPSPS gene. The DNA fragment encoding the N-terminal 215 or 235 amino acid residues of the EPSPS protein was fused in-frame to the N- terminal 123 amino acids of the Ssp DnaE intein while the DNA fragment encoding the C-terminal 212 or 192 amino acid residues of EPSPS was fused in-frame to the DNA encoding the C-terminal 36 amino acids of the Ssp DnaE intein. When only introducing half of the EPSPS gene with or without the intein fused and the two complement halves without intein into ER2799, the EPSPS was expressed as a non-functional protein. However, when introducing both the halves of EPSPS fused with both active or inactive intein halves into ER2799. the EPSPS was expressed as a functional protein and confers resistance to the herbicide glyphosate indicating that the N- and C-terminal halves of the Ssp DnaE intein facilitate the complementation and reconstitution of the N-and C-terminal halves of the EPSPS protein by bringing the EPSPS halves in close proximity.<br>
In Example IV, we describe a method in which two unrelated gene products such as aminoglycoside-3- acetyltransferase (enzyme responsible for metabolism of drug spectinomycin or streptomycin) and Aequorea victoria soluble modified green fluorescent protein could be trans-spliced to one hybrid protein in E. coli cell. Both the genes are located on two different plasmids with respective trans-splicing elements from Ssp DnaE intein. The plasmids have two independent mechanisms of expression. This hybrid protein confers resistance to spectinomycin sulphate.<br>
In Example V, we describe a method in which two unrelated genes, such as aadA (encodes for aminoglycoside- 3-acetyltransferase) and smGFP (soluble modified green fluorescent protein), could be located on a single E. binary vector under the transcriptional and translational control by a chloroplast promoter (PpsbA). Both the genes when expressed are capable of producing a hybrid aminoglycoside-3-acetyltransferase-soluble modified green fluorescent protein. Thus this method allows for<br><br>
rapid trans- splicing screening of protein/protein fragments before introducing to the plant cells using promoter that could be recognized both by E. coli and plant cellular machinary.<br>
In Example VI, we describe a method in which a cis- splicing construct containing two fragments of either 5- enolpyruvyl-3-phosphoshikimate synthetase (EPSPS) or acetolactate synthase (ALS) genes along with a Ssp DnaE intein is capable of splicing into a mature protein in plant cytoplasm. This experiment will enforce the idea of cis/trans- splicing in the cytoplasm. This technique would be useful for proteins, which need specific modification for activity/folding in cytoplasmic environment. A part of the target protein gene with necessary transport signal and splicing element will be placed in an organelle for cytoplasmic transport in the form of a pre-cursor polypeptide.<br>
In Example VII, Section 1, we describe a method in which two unrelated genes, such as aadA (encodes for aminoglycoside-3-acetyltransferase) and smGFP (soluble modified green fluorescent protein), could be located on the chloroplast genome and produce a hybrid protein via protein trans-splicing. Success in this method will lead to compartmentalization of protein/protein fragments and trans- splicing of the functional protein. Also transformation of several separated genes in one vector to form a multifunctional protein simplifying engineering of novel characters.<br>
In Example VII, Section 2, we describe a method in which two unrelated genes/gene fragments could be localized in two different compartments in plant cell, such as chloroplast and nucleus and express the respective protein/polypeptide. The nuclear encoded component is tripartite with a chloroplast transit peptide which will help the protein fragment to be synthesized in cytoplasm and migrate in to the chloroplast for the trans-splicing event to occur. The chloroplast half will be as an integrated component in the circular genome of the organelle. The resulting plants will not be able to transfer the novel character of the newly introduced transgene to any closely related species.<br><br>
The present invention is further illustrated by the following Examples. These Examples are provided to aid in the understanding of the present invention and are not construed as a limitation thereof.<br>
The references cited above and below are hereby incorporated by reference.<br>
EXAMPLE I<br>
Production of Functional Herbicide-resistant Acetolactate Synthase in E. coli by Protein Trans-splicing<br>
In this Example we demonstrate a method to split the gene which encodes E. coli acetolactate synthase II (ALSII; EC 4.1.3.18; acetohydroxyacid synthase), possessing a <br><br><br> herbicide-resistant mutation (Yadav et al, Proc. Natl. Acad. Sci. USA, 83: 4418-4422 (1986); HilletalBiochem335: 653- 661 (1998)), by fusion with Ssp DnaE intein coding sequences (EvansetalBiolChem2759091-9094 (2000); Scott, et al.. pro. Natl. Acad. Sci. USA, 96: 13638-13643 (1999)). We were able to reconstitute a functionally active ALSII enzyme through protein trans-splicing in the bacterium E. coli ER2744 (fhuA2 glnV44 el4-rfbDl? relAl? endAl spoTl? thi-1 A (mcrC- mrr) 114:: IS 10 lacZ:: T7 genel) (Figure 5). First, we show how to select a potential split site in the acetolactate synthase II gene based on the analysis of its sequence and structure homology. Then we show how to design and carry out experiments to analyze the protein trans-splicing activity of the split ALS protein and how to assay the enzymatic activity of reconstituted ALS. We demonstrate that the two portions of the ALS fusion protein, produced from two separate plasmid vectors, undergo trans-splicing to produce a protein product of expected size for the mature protein. Furthermore, co-expression of the split ALS gene fragments conferred resistance to a herbicide in the E. coli ER2744. This method may be applied to the production of any protein of interest utilizing trans-splicing inteins.<br>
1. Cloning of wild-type E. coli ALSII and generation of its herbicide resistant mutant<br><br>
The initial step is to clone the wild type ALSII and to create a herbicide resistant ALSII mutant carrying Alanine26 to Valine substitution (Yadav et al, Proc. Natl. Acad. Sci. USA, 83: 4418-4422 (1986); Hill et al., Biochem. J., 335: 653-661 (1998)). E. coli strain MI 162, containing an enzymatic active copy ALSII, was obtained from CGSC, E. coli Genetic Stock Center (Yale University, New Haven, CT). Genomic DNA was extracted from E. coli strain M1l62 using QIAamp Tissue Kit (Qiagen, Inc., Studio City, CA). DNA Polymerase Chain Reaction (PCR) was performed on the E. coli DNA sample to clone the full length ALSII using primers 5'-GGACGGGGAACTAA CTATG-3' (SEQ ID NO: 1) and 5'-CCACGATGACGCACCACGCG-3' (SEQ ID NO: 2) and Vents DNA Polymerase (New England Biolabs, Beverly, MA). The ALSII coding sequence was further amplified using primers 5'GGAGGGGGCATATGAATGGCGCACAGT GGG-3' (SEQ ID NO: 3) and 5'-GGGGGGTCATGATAATTTCTCCAAC -3' (SEQ ID NO: 4) and cloned into Ndel and PstI sites of pTYBl plasmid (New England Biolabs, Beverly, MA), creating a vector, pALSII. A shorter construct, pTYBT-ALSII, was obtained by the removal of a 3-kb non-essential sequence from pALSII by restriction digestion with Pmel and BstZ172 followed by self ligation. The herbicide resistant mutation, Alanine26 to Valine, was introduced in pTYBT-ALSII by site-directed mutagenesis using Quickchange Site-Directed Mutagenesis kit (Stratagene, La Jolla, CA). The mutagenesis primers were 5'-CCGGGTGGCG TAATTATGCCGGTTTACG-3' (SEQ ID NO: 5) and 5'-CGTAAACCG GCATAATTACGCCACCCGG-3' (SEQ ID NO: 6). The mutated ALSII (ALSIIm) coding sequence generated by partial Ndel and PstI digestion of pTYBT-ALSIIm was ligated with pTYBl to produce an ALSIIm expression vector, pALSIIm.<br>
2. Selection of Split Site<br>
One preferred method for identifying a suitable split site within any gene, is to analyze the sequence homology of a family of proteins and to examine its protein structure or the structure of its homologues (Ibdah et al., Biochemistry, 35: 16282-16291 (1996)). Sequence alignment and structure comparison suggest that the ALS genes of bacteria, yeast and higher plants share highly conserved regions (Figure 6, only partial sequence<br><br>
alignment is shown here). Still, there are highly variable regions present in the proteins, such as the region around amino acid residues Q327 and C328 in the isoform II of the E. coli acetolactate synthase (Figure 6). E. coli ALSII has a 10 amino acid gap in this region compared to other homologues and the flanking sequence has less homology among ALS genes from different species (Figure 6). Furthermore, analysis of the crystal structure of a homologue, pyruvate oxidase, suggests that Q327 and C328 are likely to be located in a linker structure between two intra-molecular domains, away from the catalytic core (Ibdah et al., Biochemistry, 35: 16282-16291 (1996)). We reasoned, therefore, that ALSII split by an intein at this region may retain the necessary flexibility to allow efficient protein trans- splicing. In addition, insertion of a foreign protein sequence into this location may have less or no effect on the structure of the catalytic domain of ALSII and its enzymatic activity. Thus amino acid residues Q327 and C328 were selected as one of the split sites for E. coli ALSII (indicated by an arrow, Figure 6).<br>
3.	E. coli assay system<br>
The isoform II of the E. coli acetolactate synthase that possesses the mutation Ala26Val. referred to as ALSIIm, confers resistance to sulfonylurea herbicides (SU), such as sulfometuron methyl (SM), in E. coli strain ER2744. E. coli ER2744 strain was employed as an in vivo model system for assessing the activity of the herbicide resistant E. coli ALSII gene, genetically modified by a linker insertion between Q327 and C328. E. coli ER2744 is derived from wild type E. coli K12 that contains the active ALSI and ALSIII enzymes, but not an active ALSII. ALSI and ALSIII are two isoforms of ALS genes in E. coli, which are crucial for the synthesis of valine, isoleucine and leucine (LaRossa and Schloss, J. Biol. Chem. 259: 8753- 8757 (1984)). Their activity is sensitive to the valine feedback inhibition. Therefore, by saturating the growth medium with 100 pg/ml valine (Sigma, St. Louis, MO), ALSI and III will be inhibited and the cells will stop growing. By introducing a recombinant herbicide resistant ALSII (ALSIIm) into E. coli cells, their growth will be rescued since ALSII is resistant to valine inhibition.<br>
4.	Generation of a modified herbicide resistance ALS gene<br><br>
Inteins often require certain amino acid residues flanking its N-and C-termini to achieve optimal splicing or <br><br><br> trans-splicing activity. For example, the intein from the dnaE<br><br><br> gene of Synechocystis species PCC6803 spliced efficiently when 5 native residues were present at both its N-and C- termini, while deletion of these residues inhibited splicing activity to various extents (Evans et al., J. Biol. Chem. 275: 9091-9094 (2000)). Inclusion of these optimal amino acid residues at the splice junctions may be required for proficient splicing activity. The resulting product may therefore possess these residues at the ligation junction of two protein sequences. Thus, for each intein insertion site, it is necessary to assess if these extra amino acid residues will have an adverse effect on the activity of the product.<br>
ALSIIm-14 was constructed by insertion of a synthetic DNA linker (New England Biolabs, Beverly, MA), encoding the following 14 amino acid residues (NH2-LEKFAEYCFNKSTG- COOH (SEQ ID NO: 7)), into the ALSIIm coding sequence between Q327 and C328A. The herbicide resistance activity of ALSIIm-14 was examined using E. coli ER2744 host cells transformed by the plasmid expressing ALSIIm-14 protein. E. coli ER2744 cells transformed with plasmids expressing the wild type ALSII and herbicide resistant ALSII (ALSIIm) were used as controls.<br>
Plate assays were conducted to examine the capability of ALSIIm-14 to rescue E. coli ER2744 from valine (100 ug/ml) or valine plus herbicide SM (50 pg/ml, Supelco Park, Bellefonte, PA) saturated M9 minimum medium plate (Sambrook et al., (1989)). The M9 medium contains 2 pg/ml Thiamin, 2 mM MgS04,0.1 mM Cal2,0.2% glucose, 50 ug/ml of kanamycin, 100 1Jg/ml of ampicillin and 0.3 mM IPTG. For the plating assay, 100 pi of 25 mg/ml Valine with or without 50 u of 25 Ug/ml Sulfometuron methyl (SM) was spread on M9 selection plate. To assay bacterial growth, overnight cultures were streaked on M9 plates with or without valine and/or SM. The plates were incubated at various temperatures (as indicated in Figure 7) for 48 to 72 hrs before the pictures were taken. On the plate supplemented with valine, cells expressing either ALSII, ALSIIm or ALSIIm-14 were able to grow (Figure 7-a). However, when both valine and SM was applied, only strains expressing herbicide-resistant ALSIIm or ALSIIm-14 were able to grow (Figure<br><br>
7-b). These in vivo results demonstrated that ALSIIm with 14 amino acid residues inserted at the proposed split site, rescued E. coli ER2744 growth in the presence of valine and SM. Therefore, ALSIIm-14 is functionally active and the 14 amino acid insertion does not affect its enzymatic activity.<br>
5. Construction of ALSII-Intein fusion genes<br>
Next, the E. coli ALSIIm gene was split and fused in- frame to the N-and C-terminal halves of the Ssp DnaE intein coding regions. The fusion genes were created using two compatible E. coli expression vectors, pMEBlO and pKEBl, which are capable of co-expressing two intein fusion genes in the same E. coli host cell, as previously described by Evans et al. (J. Biol. Chem. 275: 9091-9094 (2000)). The DNA sequence encoding for an N-terminal fragment of 327 amino acids of the herbicide resistant ALSII (ALSIIM) gene was fused in frame to the coding region for the 7 amino acid residues flanking the N- terminus of the Ssp DnaE intein, followed by the intein N- terminal 123 amino acid residues (INn) (Figure 5). The DNA sequence encoding the C-terminal 221 amino acid residues of ALSIIm was fused in frame to the DNA sequence encoding the C-terminal 36 amino acid residues of the Ssp DnaE intein (INc) and the 7 amino acid residues flanking the C-terminus of the intein (Figure 5). ALSII N-terminal fragment was amplified from pALSIIm using primers 5'-GGGGGTCATGAATGGCGCACAG TGGG-3' (SEQ ID NO: 10) and S'-GCGCGCTCGAGTTGATTTAACGG CTGCTGTAATG-3' (SEQ ID NO: 11). The amplified fragment was digested and cloned into the Ncol and Xhol sites of pMEB16, which contains the sequence encoding the N-terminal 123 amino acid residues of the Ssp DnaE intein. The resulting vector pEA (N) expresses a fusion protein composed of the ALSIIm N-terminal fragment and the DnaE N-terminal fragment (ALSIIm (N)- INn). The ALS II C-terminal fragment was amplified using primers 5'-GCGCGACCGGTTGTGACTGGCA GCAACACTGC-3' (SEQ ID NO: 12) and 5'-GGGGGGCTGCAGTCA TGATAATTTCTCCAAC-3' (SEQ ID NO: 13). The fragment was digested with Agel and PstI and then cloned into the Agel and PstI sites of pMEB9. The resulting plasmid pEA (C) expresses a fusion protein composed of the Ssp DnaE intein C-terminal fragment and the ALSII C-terminal fragment (ALSIIm (C)-INc). A lkb<br><br>
Xbal-PstI fragment containing AESIlm (C)-INc fusion gene was subcloned from pEA (C) into the Xbal and PstI sites of pKEBl plasmid to produce a kanamycin resistant expression vector pKEC3. When pEA (N) and pKEC3 were co-expressed in E. coli ER2744, it was predicted that trans-splicing of the two fusion proteins would result in ligation of the two split halves of the E. coli ALSIIm, with 14 amino acids present at the ligation junction.<br>
6. Characterization of protein trans-splicing activity<br>
To determine whether ALSII-DnaE intein fusion proteins are able to trans-splice in E. coli cells to produce ALSIIm-14, western blots were performed using rabbit antiserum specifically against either the N-or C-terminal fragment of ALSII.<br>
Two rabbit antisera were raised against peptides derived from the N-terminal and C-terminal regions of ALSII, respectively (COVANCE). These two peptides are 1) NH2-CAQ WVVHALRAQGVNTVFGYG-COOH (SEQ ID NO: 8) derived from the ALSII N-terminal sequence (amino acid residues Ala4 to Tyr23) and 2) NH2-CVWPLVPPGASNSEMLEKLS-COOH (SEQ ID NO: 9) derived from the ALSII C-terminal sequence (amino acid residues Val 530V to Ser548). A single bacterial colony was inoculated in LB medium supplemented with 100 ug/ml of ampicillin for 4 hrs at 37°C. Then it was induced by addition of IPTG to 0.3 mM final concentration. Cells were further cultured for 2-16 hours at 15 C. 20, ul of cell culture was removed, mixed with 3XSDS loading buffer (New England Biolabs, Beverly, MA), boiled for 5 minutes and 2 u ! was) oaded to 12% Tris-glycine gel (Novex, San Diego, CA). Subsequently proteins were transferred to a nitrocellulose membrane and blocked with 5% dry milk for one hour at room temperature (Sambrook. et al, Molecular Cloning, (1989)). Immunoblotting was performed using antiserum (1: 20000 dilution) overnight at 4°C in the presence of 1% dry milk. Blots were then washed three times for 15 minutes each and incubated with 1: 10000 diluted HRP-conjugated anti-rabbit secondary antibody for 1 hour at room temperature. The reactions were visualized with Chemiluminescent Western Detection kit (New England Biolabs, Beverly, MA).<br><br>
In control cells cultured at 15°C, expression of ALSII (Figure 8A, 8B &amp; 8C, lane 2) was recognized specifically by both antibodies. In cells bearing a single ALSII-intein fusion vector and another control vector to confer both ampicillin and kanamycin resistance only ALS (N)-INn or ALS (C)-INc protein was detected by anti-ALS (N) or anti-ALS (C) serum (Figure 8A, lane 3, Figure 8B, lane 4). When ALS (N)-INn and ALS (C)-INc were co-expressed a 60 kD band, as expected for the spliced product ALSIIm-14, reacted with antibodies raised against the N-terminus and C-terminus of ALSII (Figure 8A &amp; 8B, lane 5). This band of AISIIm-14, as predicted, exhibited a slightly higher molecular weight than native ALSII. The data indicated that trans-splicing occurred between the two ALSII-intein fusion proteins. A non-specific protein reacting with anti- ALS (N) was observed (Figure 8A and Figure 8C, lane 1 to lane 5).<br>
Trans-splicing activity of the Ssp DnaE intein was previously shown to be temperature sensitive (Evans et al., J. Biol. Chem. 275: 9091-9094 (2000)). The temperature sensitivity of trans-splicing of the ALSII-Ssp DnaE intein proteins were examined by western blot analysis using an antiserum against ALSII N-terminal fragment (Figure 8C). Cells were transformed by plasmids expressing ALSII, or both ALSllm (N)-INn and ALSIIm (C)-INc. Expression of the ALSII proteins were induced at 37°C for 3 hours. Co-expression of ALSIIm (N)-INn and ALSIIm (C)-INc was induced at 37°C for 3 hours, 30°C for 3 hours, 25°C for 6 hours, or 15°C for 16 hours. Cell extracts were treated with SDS sample buffer and denatured at 95°C to 100°C for 5 minutes and then subjected to electrophoresis on a 12% SDS-PAGE. A western blot was probed using an antiserum raised against the ALSII N-terminal fragment. Figure 8C includes the following samples: cells with no ALSII (lane 1, control), ALSII (lane 2), ALSIIm (N)-INn and ALSIIm (C)-INc (lane 3 to lane 6). The cell culture temperature are 37°C for lane 1 to lane 3,30°C for lane 4,25°C for lane 5, and 15°C for lane 6.<br>
In cells grown at 37°C, ALSIIm-14 was not detectable (Figure 8C, lane 3). However, in cells cultured at 30°C, the spliced product was observed with a significant amount of N-terminal fusion protein accumulation (Figure 8C, lane 4). In cells cultured at 25°C and 15°C (Figure 8C, lane 5 and 6), only the spliced product was detected, indicating a<br><br>
complete conversion of the N-terminal fusion protein to the spliced product. The ALSIIm (C)-INc protein was produced in excess under all the expression conditions. The data demonstrated that the Ssp DnaE intein was capable of mediating trans- splicing of the N-and C-terminal ALSIIm protein segments to form ALSIIm-14. The splicing reaction was inhibited when the experiment was conducted at 37°C. Splicing appeared to be more efficient when cells were cultured at 15°C-25°C rather than at 30°C.<br>
7. Herbicide resistance in cells bearing the split ALS gene<br>
The next step was to determine whether the spliced product, as the result of trans-splicing of the ALSIIm (ALSIIm (N)-INn and ALSIIm (C)-INc) fusion proteins, would render E. coli ER2744 resistant to valine and SM. The first experiment was to test the effect of co-expression of ALSIIm (N)-INn and ALSIIm (C)-INc fusion proteins on cell growth in valine saturated M9 minimum medium. In a plating assay (see Section 4.), all transformed cells grew well on M9 medium in the absence of valine (Figure 9A-a). However, only ALSII and its herbicide resistant mutant ALSIIm rescued the cells growth at both 30°C and 37°C (Figure 9A-b, 9A-c) in the presence of valine. Significantly, co-expression of ALSII (N)-INn and ALSII (C)-INc rescued cell growth at 30°C (Figure 9A-c) or lower temperatures (data not shown) from a valine plate. Furthermore, expression of ALSIIm or ALSIIm (N)-INn and ALSIIm (C)-INc rescued cells from additional herbicide inhibition (Figure 9A-d). Moreover, transformation of wild type ALSII could not rescue cell growth from herbicide inhibition (Figure 9A-d). The control cells which expressed either ALSII (N)-INn or ALSII (C)-INc alone did not grow on a valine plate (Figure 9A- b, 9A-c); neither did the co-expression of native ALSII N-and C-terminal segments which were not fused to the intein (Figure 9A-b, 9A-c). The data indicates that co-expression of ALSIIm (N)-INn and ALSIIm (C)-INc fragments is required for trans-splicing and generating a functional ALSII, which can rescue cell growth from valine and herbicide inhibition.<br>
A quantitative liquid culture assay was performed to verify the results obtained from the plating assay. The liquid assay was performed as follows. A single colony was used to inoculate LB medium supplemented with kanamycin and ampicillin at 37°C for 4 hours.<br><br>
Expression was induced by 0.3 mM IPTG and cell cultures were shifted to 30°C for another 2hrs. Then, 200 uL of equivalent OD600 8. 0 was spun down, washed one time with M9 medium and resuspended in 200 u ! M9 medium. 40 pl was aliquote into 2ml of appropriate culture medium and grown for 24-72 hours before its OD6O0 was measured. The concentration for valine is 100 ug/ml and for SM is 50 ug/ml. At 30°C, all transformed cells grew equally well in M9 minimum medium (Figure 9B). In valine saturated M9 medium, wild type ALS allowed cells to grow, but no growth was observed when SM was added. However, the expression of ALSIIm or co-expression of ALSIIm (N)-INn and ALSIIm (C)-INc allowed cells to grow in valine M9 medium, as well as medium containing SM. In control experiments, ALSIIm (N)-INn or ALSIIm (C)-INc alone or co-expression of ALSIIm N-and C-terminus not fused to the intein, did not rescue cell growth in valine containing medium. This data is in agreement with that from the plating assay. To further compare the growth kinetics for trans-splicing mediated cell growth to wild type ALSII mediated cell growth, a time course study was performed (Figure 9C). Data showed that ALSII expressing cells have the fastest growth rate followed by ALSIIm expressing cells. ALS (N)-INn and ALS (C)-INc transformed cells have slower growth rates compared to ALSII wild type expressing cells, but not significantly less than the growth rate of ALSIIm expressing cells. Cells expressing split ALSII, with no fusion to the intein, have very slow growth. Therefore, from the plating and liquid assay we have demonstrated that the Ssp DnaE can mediate ALSII trans-splicing, which results in a functionally herbicide-resistant ALSIIm-14 in vivo.<br>
In conclusion, the data indicated that the two ALS-intein fusion proteins, produced from two different loci, underwent trans-splicing in a temperature-dependent manner to form a full length, functional ALSIIm protein. E. coli host cells possessing both ALSIIm fusion gene fragments showed the herbicide resistance phenotype.<br>
EXAMPLE II<br>
Trans-splicing of a maize Acetolactate Synthase in E. coli<br><br>
In this Example, we demonstrate a method to produce a full length maize acetolactate synthase by protein trans- splicing in E. coli. We demonstrate how to choose a split site in the maize ALS gene based on the sequence homology of the maize ALS gene and its E. coli counterpart, the ALSII gene. We show that, when the split maize ALS-intein fusion genes were co-expressed, the two fusion proteins underwent trans-splicing to produce a protein product of expected size for the mature maize ALS protein.<br>
1.	Selection of split site.<br>
It is important to demonstrate the trans-splicing of other herbicide resistant genes, such as the maize acetolactate synthase (cALS) gene, of which the herbicide resistant mutant form has been utilized to genetically modify plants (Bernasconi et al., J. Biol. Chem. 270: 17381-17385 (1995)). One preferred method for the identification of a suitable split site within any gene is to analyze the trans-splicing activity of a homologous gene from a different organism. We have described in Example I, that E. coli ALSII gene, after being split between Q327 and C328, can be reconstituted by the trans-splicing activity of the Ssp DnaE intein in vivo. Sequence alignment between E. coli ALSII and maize ALS was conducted to search for the region in the maize ALS gene corresponding to the split site of the E. coli ALSII gene. The result suggests that Serine397 and Threonine398 align with the split site (Glutamine327 and Cysteine328) of E. coli ALSII. Splitting the maize ALS between Serine397 and Threonine398, as indicated by a star (Figure 6), may yield two maize ALS-intein fusion proteins which would be capable of proficient splicing.<br>
2.	Cloning of the maize ALS gene<br>
Reverse transcriptase polymerase chain reaction (RT- PCR) was carried out to clone the maize ALS cDNA.<br>
Total RNA was isolated from corn leaves using the RNAqueous kit (Ambion, Inc., Texas). The RNA was then used for first strand cDNA synthesis using the reverse primer 3-3 (5'-AT CAGTACACAGTCCTGCCATC-3' (SEQ ID NO: 14)) and Superscript Reverse Transcriptase (LTI-GIBCOBRL, Rockville, MD). The first strand cDNAs were<br><br>
then treated with RNaseH (LTI-GIBCO BRL, Rockville, MD) before being used as a template in a PCR reaction. The PCR reaction was carried out using Expand Long Template PCR system (Boehringer Mannheim, Germany). The primers used in this reaction were Reverse Primer 3-3 and cALS 5-4 primer (5'GAGACAGCCGCCG CAACCAT-3' (SEQ ID NO: 15)).<br>
An aliquot of the PCR product was electrophoresed on an agarose gel and a band of approximately 2 kb was observed. This fragment was cloned into the TOPO 2.1 vector (Invitrogen, San Diego, CA, manufacturer's protocol) to make PALS1. The sequence of pCALSl was confirmed using Ml3 forward and reverse primers.<br>
3. Construction of the maize ALS-intein fusions<br>
The DNA encoding for the N -terminal 397 amino acid residues of the maize ALS gene<br>
was	amplified	by	PCR	using	forward	primer	5'-<br>
GGGCCCATATGGCCACCGCCGCCGCCGCG-3' (SEQ ID NO: 16), reverse primer 5'-GGGCCCTCGAGGCTTCCTTC AAGAAGAGC-3' (SEQ ID NO: 17), and the template PALS1 (Sambrook et al., Molecular Cloning, (1989)). A 1.2 kb PCR product was cloned into TOPO-blunt vector (Invitrogen, San Diego, CA manufacturer's protocol), resulting in TOPO-cALS (N). Then TOPO-cALS (N) was digested with Ndel and Xhol. A 1.2 kb digested DNA fragment was recovered from low melting agarose gel and fused in-frame to the DNA sequence encoding the N-terminal 123 amino acids of the Ssp DnaE intein, resulting in a vector (MEBlO-cALS (N) which expresses the N-terminal cALS-intein fusion protein, cALS (N)-IN-n. A DNA fragment encoding for the C-terminal 241 amino acid residues of the maize ALS gene was PCR amplified using forward primer 5'-GGGCCACCGGTACATCAAAGAAGAGCTTG-3' (SEQ ID NO: 18), reverse primer 5"-GGGGCTGCATTCAGTACACAGTCCTGC CATC-3' (SEQ ID NO: 19), and the template pCALS4. A 0.8 kb PCR product was cloned into a TOPO-blunt vector (see protocol above), TOPO-cALS (N). TOPO-cALS (N) was then digested with Agel and Pstl. A 700 bp DNA fragment was recovered from low melting agrose gel and was fused in frame to the DNA encoding the C-terminal 36 amino acids of the Ssp DnaE intein, resulting in a vector MEB9-CALS (C). MEB9-cALS (C) was further cut by Xbal and Pstf<br><br>
and released a 1 kb fragment. This 1 kb fragment was cloned into pKEBl vector to create a kanamycin resistant expression vector for the cALS-intein C-terminal fusion protein, cALS (C)-INc. The same extra 7 amino acids, NH2-LEKFAEY-COOH (SEQ ID NO: 20) and NH2-CFNKSTG-COOH (SEQ ID NO: 21) were also present at the junctions of the N-and C-terminal cALS-intein fusion proteins, respectively.<br>
4. Trans-splicing of the maize ALS-intein fusion proteins<br>
Both ALS-intein fusion fragments, cALS (N)-INn and cALS (C)-INc, described in Section 3, were co-expressed in E. coli ER2744 under the same conditions as described in Example I, Section 6. A western blot was performed to detect the trans-splicing product (Method, see Example I, Section 6). On the blot, a fragment of 69 kD, which corresponds to the size of the wild type cALS (Figure 10A, and Figure 10B, lane 2), was detected in both fusion proteins expressed cells and was recognized by rabbit antisera specifically raised against two peptides derived from N-and C-terminal sequence of maize ALS (Figure 10A and Figure 10B, lane 5). A non-specific protein reacting with antiserum against N-terminal of maize ALS was observed (Figure 10A, lane 1 to lane 5). The peptides used to raise antibodies are 1) ALS-N peptide corresponding to the sequence from Lys66 to Ala85, NH2- CKGADILVESLERCGVRDVFA-COOH (SEQ ID NO: 22), and 2) ALS-C peptide corresponding to the sequence from Ile619 to Tyr638, NH2-CI PSGGAFKDMILDGDGRTVY.COOH (SEQ ID NO: 23). The full length CALS species was not detected in cells expressing either N-or C-terminal fusion protein (Figure 10A and Figure 10B, lane 3 and lane 4). This demonstrated that split maize ALS, like E, coli ALSII, when fused with the Ssp DnaE intein, was also able to perform trans-splicing to produce the full length ALS.<br>
In conclusion, the maize ALS gene was split by the Ssp DnaE intein and cloned into two separate plasmid vectors. When both the fusion gene vectors were introduced into the same host cell and co-expressed, the two fusion proteins underwent trans-splicing to produce a full length cALS. Although a functional assay is needed to determine the activity of the spliced maize ALS protein in plants, it does raise the possibility of successfully spi'itti'ng a plant herbicide resistant or disease resistant genes into two<br><br>
inactive gene segments. These two gene fragments can be confined into two separate cellular compartments, such as the chloroplast and nucleus, or two separate loci on the chromosomes, or two separate DNA vector. This novel mode of gene expression may greatly lessen the chance of spreading an intact active transgene into other species.<br>
EXAMPLE III<br>
The present Example details the feasibility of splitting the aroA gene and regenerating the desired protein activity using an intein. The experiment consisted of dividing the gene encoding the mutant aroA gene at various positions and fusing the gene encoding the N-terminal splicing domain of the Ssp DnaE intein (INn) to the gene encoding the N-terminal fragment of the EPSPS protein. Concurrently, the gene encoding the C-terminal splicing domain of the Ssp DnaE intein (INc) was fused to the gene encoding the C-terminal fragment of the EPSPS protein. When the fusion genes were placed on to two separate plasmids and co-transformed and co-expressed in the same bacterial cell it was demonstrated that those bacterial cells were resistant to the herbicide glyphosate.<br>
The cloning of Salmonella typhimurium aroA gene that confers resistance to glyphosate<br>
1. Creation of plasmid pEPS#l<br>
The Salmonella typhimurium aroA gene with the C301 to T mutation was acquired from the American Type Culture Center in the form of a cosmid in the bacteria Salmonella choleraesuis subsp choleraesuis (ATCC No. 39256). The modified aroA gene was amplified from the cosmid by the polymerase chain reaction using primers EPSP#1 (5'-GGATC CTAAGAAGGAGATATACCCATGGAATCCCTGACGTTACA-3' (SEQ ID NO: 24)) and EPSP#2 (5'-GTCGACGCTCTCCTGCAGTTAGGCAGGC GTACTCATTC-3' (SEQ ID NO: 25). The PCR product was inserted into the Stul site of the plasmid LITMUS 28 (New England Biolabs, Inc., Beverly, MA). Following transformation and plasmid preparation, sequencing revealed an unexpected mutation (C103 to G) which was reverted using Stratagene's (La Jolla, CA) Quick Change Site<br><br>
Directed Mutagenesis Kit and primers EPSP#10 (5'-GCl 1 IGCTCCTGGCGGCTACCTTGTGGT AAAACCGC-3* (SEQ ID NO: 26)) and EPSP#11 (5'-GCGGTTTTAC CACAAGGTAAAGCCGCCAGGAGCAAAGC-3' (SEQ ID NO: 27)). Sequencing of DNA from the resulting colonies revealed that the unexpected mutation had been reverted to the expected C. This plasmid was termed pEPS#8 and used as the acceptor plasmid in the subsequent transposition linker scanning reactions.<br>
2.	Description of ER2799, an E. coli strain used to test the aroA gene constructs<br>
An E. coli strain that has the aroA gene deleted from its chromosome was acquired from the Yale E. coli stock center (E. coli strain AB2829, CGSC#2829, ID#8215). This strain was made hsdR-and named ER2799. Because ER2799 lacks the aroA gene, which is necessary for aromatic amino acid synthesis, it does not grow on M9 minimal media. This strain is used to test the various aroA gene constructs to see if the new aroA gene can rescue the bacteria and allow growth on minimal media either in the presence or absence of glyphosate.<br>
3.	Finding a site to split the aroA target gene by transposon based linker scanning<br>
The first step in performing this experiment was to determine the sites in the 5-enolpyruvyl-3-phosphoshikimate synthetase (EPSPS) protein which could allow insertion of an intein in cis. In cis refers to the fact that the complete intein is inserted into the complete EPSPS protein. However, it was not known which portions of the EPSPS protein itself would be tolerant to extra amino acid residues. So to determine where the EPSPS protein could tolerate amino acid insertions a new technology, the GYPSE'-LYS kit (available from New England Biolabs, Inc., Beverly, MA), was used to randomly insert 5 amino acid residues throughout the EPSPS protein sequence. An expression plasmid library was constructed with the EPSPS gene with the randomly inserted 5 amino acids. This library was transformed into E. coli strain ER2799 and applied to plates containing M9 minimal media. ER2799 lacks the aroA gene and will not grow on M9 minimal plates unless an active EPSPS gene is supplied by plasmid transformation. The<br><br>
ER2799 E. coli that grew following transformation with the library should contain an EPSPS protein that is active with the 5 amino acid insertion. These were sequenced to determine the position of the 5 amino acid insertion and 42 unique sites were discovered in the EPSPS protein that allowed growth of ER2799 on M9 minimal plates (Figure 15). Furthermore, another 19 unique sites were found that did not tolerate a 5 amino acid insertion (Figure 16).<br>
4.	Transposition Reaction<br>
The reaction was performed by adding 6 NI of 20 ng/pl pEPS#8 (target DNA), 1.5 u ! of 20 ng/pl Pmel donor DNA, 3 NI of distille water, 3, ul of tOX GPS@-LS buffer and 1.5 NI of Tn*ABC and mixing for 15 min at 37°C. 1 D. ! of Start Solution was added and the reaction incubated at 37°C for 1 hour and 20 min. The reaction was stopped by heat inactivation for 15 min at 75°C. Following cooling the reaction mixture to room temperature and dialysis against water for 2 hours the reaction mixture was transformed into freshly-made ER2685 (fhuA2 gInV44 el4-rfbDl? relAl? endAl spoTl? thi-1 A (mcrC- mrr) 114:: IS10 A (lacI-lacA) 200 F'proA+B+IacIq Dl (lacZ) Ml 5 zzf: TnlO (TetR)) cells by electroporation. The cells were incubated for 1 hour at 37°C and then plated onto LB plates containing ampicillin and kanamycin. Cell growth was allowed to proceed at 37°C overnight. It was discovered that 10 u ! of reaction mixture gave over 10,000 colonies (enough to cover all possible transposon insertion sites, 2840 sites in pEPS#8, 3.3 times) following transformation.<br>
5.	Isolating the DNA fragment (3.0kb) containing the EPSPS gene plus transposon<br>
All the transformants from the transposition reaction were recovered using LB medium and 66% of the cells were saved at-70°C by adding 20% glycerol. The rest were grown in 500 ml of LB liquid medium containing 100 ug/ml ampicillin, and 50 ug/ml kanamycin at 37°C overnight. The cells were harvested by centrifugation and the plasmid DNA was purified (508 ug total) using a Qiagen Midi kit (Qiagen, Studio City, CA). The 3.0kb aroA gene-Transposon DNA fragment was released by digesting the DNA (58 p9) with<br><br>
PstI, Ncol and Ahdl and isolated by gel-purification using agarase following ethanol precipitation (4 p9 DNA was recovered).<br>
6.	Cloning the aroA gene-Transposon 3.0 kb fragment into the pCYB3 vector<br>
The gel-purified 3.0 kb aroA gene-Transposon DNA fragment was ligated into the Ncol to PstI sites of pCYB3 (5.2kb), and transformed into ER2685 by electroporation after drop dialysis for 2 hours. The electroporated cells were incubated for 1 hour in LB medium. 250 NI of this cell suspension was plated onto LB plates containing 100 ug/mL ampicillin and 50 pg/mL kanamycin while another 5.5 ml was inoculated into 1 liter of LB liquid medium with 100 pg/mL ampicillin and 50 ug/mL kanamycin and grown at 37°C overnight. The plasmid DNA library containing the transposon within the aroA gene was isolated by Qiagen (Studio City, CA) Midi kit (750 p9).<br>
7.	Screening the library EPSPS protein that is active with the 5 amino acid linker<br>
105 ng of the library DNA was digested with Pmel to remove the transposon from the aroA gene. This leaves 15 bases (or 5 amino acid residues) at the transposon insertion site. A 7 kb fragment was recovered (in a final volume of 400 ut EB), self-ligated (86 IJI out of 400 pi 7 kb fragment in a 100 pi rxn), transformed (30 pi of the 100 rxn) into E. coli strain ER2799 and plated onto both LB and M9 minimal plates, each containing 100 ug/mL ampicillin in the presence of 0.3 mM 1PTG. Following incubation at 37°C overnight ca. 20% of the original cells survived on M9 minimal plates as compared to the LB plates. Individual colonies that grew on M9 minimal media plates were analyzed by Dral digestion and DNA sequencing to confirm the site of linker insertion site into the aroA gene.<br>
42 different insertion sites were identified among 72 active individual clones that can tolerate 5 amino acid residues inserted into the aroA gene and 19 different insertion sites were identified among 39 inactive clones that can not grow on M9 minimal media selection plates (see Figure 15 and Figure 16). Plasmids pCE-5-22, pCE-5-21, pCE- 5-35 and pCE-5-23 were the active clones that have 5 amino acid residues incorporated into<br><br>
the EPSPS protein (aroA gene product) at positions 182,215,235 and 267, respectively. These four sites were chosen for further studies.<br>
Construction of Ssp DnaE Cis-and Trans-splicing vectors<br>
1. Creation of vectors pCE182DnaE, pCE215DnaE, pCE235DnaE, and pCE267DnaE for Cis-Splicing<br>
This involved inserting an intein into the sites in the target protein that were discovered to tolerate 5 amino acid insertions.<br>
Four sites were chosen for further study (positions 182, 215,235, and 267). The full length Ssp DnaE intein was inserted into these sites and the EPSPS-intein fusion was tested for its ability to permit ER2799 cells to grow on M9 minimal plates. All four sites were found to grow on M9 plates, indicating that the EPSPS protein could tolerate the intein inserted at these positions (see Figure 11 and Figure 14).<br>
CE182 or CE215, which was the linear DNA of pCE-5-22 or pCE-5-21 with the exception that the five amino acid linker at 182 or 215 has been removed, was generated by polymerase chain reaction (PCR) from templates pCE-5-22 or pCE-5-21 using primers 5'-GCCCCTAAAGACACAATTATTCGCG-3' (SEQ ID NO: 28) and 5'-CAGCGGCGCCGTCATCAGCAGAGCG-3' (SEQ ID NO: 29) for CE182 or 5'-GCGAACCACCACTACCAACAATT TG-3' (SEQ ID NO: 30) and 5'-TATCTCCACGCCAAAGGTTTTCATT- 3' (SEQ ID NO: 31) for CE215. The Ssp DnaE intein gene containing two native N-extein residues and three native C- extein residues was amplified by PCR from pMEB8 (Evans, et al., J. Biol. Chem., 275: 9091 (2000)) using primers 5'-GAATAT TGCCTGTCTTTTGGT-3' (SEQ ID NO: 32) and 5'-GTTAAAGCAGTT AGCAGCGAT-3* (SEQ ID NO: 33). The resultant PCR fragment was phosphorylated with T4 polynucleotide kinase, purified by QIAquick column (Qiagen, Inc., Studio City, CA) and ligated into CE182 or CE215 to generate pCE182DnaE or pCE215DnaE, respectively.<br><br>
The Ssp DnaE intein gene containing four native N- extein residues and three native C-extein residues was amplified by PCR from pMEB8 using primers 5'-TGCTGAATATTG CCTGTCTTTTGG-3' (SEQ ID NO: 34) and 5'-CCGTTAAAGCAGTTAG CAGCGATAGC-3' (SEQ ID NO: 35). The resultant PCR fragment was purified by QIAquick column (Qiagen Inc., Studio City, CA) and ligated into the gel-purified, Pmel cut pCE-5-35 or pCE-5- 23 vector DNA to generate pCE235DnaE or pCE267DnaE, respectively.<br>
2. Creation of vectors p215EN2/pEPS#28 and p235EN2/pEPS#29 for Trans-Splicing:<br>
Two plasmids were constructed with compatible origins of replication. The N-terminus of the appropriate EPSPS protein was fused to the N-terminus of the N-terminal Ssp DnaE splicing domain (INn) and inserted into one plasmid. The remaining C-terminal portion of the appropriate EPSPS protein was fused to the C-terminus of the C-terminal splicing domain of the Ssp DnaE intein (INc). This fusion was inserted into the second plasmid. The plasmids were co-transfected into ER2799 by electroporation. Expression of the fusion protein was under the control of an IPTG inducible pTac promoter. The transformed cells grew on M9 minimal plates, liquid M9 minimal media, or liquid M9 minimal media supplemented with glyphosate (Figures 11,12,13 and 14). This indicated that the protein halves could generate an active EPSPS protein when co-expressed in the same cell.<br>
The 0.6 kilobase Xhol to PstI fragment of pMEB4 was gel-purified using the QIAquick extraction kit and ligated into the Xhol to PstI sites in the pCYB3 (New England Biolabs, Inc., Beverly, MA) vector to generate pCENl. The Ncol site between the Ssp DnaE intein and the chitin-binding domain (CBD) was removed by Pad and Sapl digestion of pCEN2 followed by T4 DNA polymerase treatment and self-ligation to generate plasmid pCEN2. This vector contains the N-terminal 123 amino acid residues of the Ssp DnaE intein (INn) under the control of pTac promoter and confers resistance to ampicillin.<br><br>
p215EN2 or p235EN2 were constructed by ligating the Ncol to Kpnl fragment of pCE215DnaE or pCE235DnaE into the same sites of pCEN2. p215EN2 or p235EN2 has the N- terminus of EPSPS (residues 1-215 for p215EN2,l-235 for p235) fused to the INn<br>
The Ncol to Fspl fragment of pCYB3 was ligated into the Ncol to Dral sites of pKEBl to generate pKEB12 (NEB#1282). A sample of pKEB12 plasmid transformed in E. coli strain ER2566 has been deposited under the terms and conditions of the Budapest Treaty<br>
with the American Type Culture Collection on May	, 2000 and received ATCC<br>
Patent Accession No.	. This vector has the C-terminal 36 amino acid residues<br>
of the Ssp DnaE intein (INn) fused to CBD and confers resistance to kanamycin. pEPS#28 and pEPS#29 were constructed by ligating the Bg/II to PstI fragment of pCE215DnaE and pCE235DnaE into the same sites of pKEB12. pEPS#28 or pEPS#29 has the C- terminus of EPSPS (residues 216-427 for pEPS#28,236-427 for pEPS#29) replacing the CBD in pKEB12 and attached to the C-terminus of INc.<br>
3. Creation of the EPSPS complementary construct pEPS#34 and pEPS#36.<br>
When the EPSPS protein fragments, lacking the intein domains, were co-expressed in ER2799 cells, the cells failed to grow on M9 minimal plates, liquid M9 minimal media, or liquid M9 minimal media supplemented with glyphosate (Figure 12 and Figure 13). This indicated that EPSPS activity was absolutely dependent on the presence of both intein halves.<br>
DNA encoding the N-terminus of the EPSPS protein, residues 1-235. (EPS235N) was amplified by PCR from pCE235DnaE using primers 5'-GGATCCTAAGAAGGAGATATACCC ATGGAATCCCTGACGTTACA-3' (SEQ ID NO: 36) and S'-GATATC CTGCAGTTAACCTGGAGAGTGATACTGTTGACC-3' (SEQ ID NO: 37). The resultant PCR product was purified using a QIAquick PCR kit, digested with Ncol and Pstl, purified from an agarose gel using the QIAquick extraction kit and ligated into the Ncol to Pstl sites of plasmid pCYB3 to generate pEPS#34.<br><br>
Plasmid pEPS#36 was created by amplifying DNA encoding the C-terminus of EPSPS, residues 236-427, (EPS235C) by PCR from pC+E2 using primers 5"-GATATCCCATG GGACGCTATCTGGTCGAGGGCGATG-3' (SEQ ID NO: 38) and 5'-GT CGACGCTCTCCTGCAGTTAGGCAGGCGTACTCATTC-3' (SEQ ID NO: 39). The resultant PCR product was purified using the QIAquick PCR kit, digested with Ncol and PstI, purified from agarose gel and ligated into the Ncol to PstI sites of plasmid pKEB12. Two extra residues Met-Gly were also incorporated at the N-terminus of EPS235C due to the Ncol site for cloning.<br>
4.	Creation of Vectors Containing the Cis or Trans "dead"Ssp DnaE intein at<br>
position 235 (pEPS#31, pEPS#33, pEPS#37).<br>
Interestingly, trans-splicing was not required for activity, because if three of the most highly conserved catalytic residues of the Ssp DnaE intein were changed to alanine the co-transformed ER2799 cells still grew. This event demonstrates that the intein can act as an affinity domain to bring the two EPSPS intein fragments together (Figure 12 and Figure 13).<br>
The Ssp DnaE intein gene containing four native N- extein residues and three native C-extein residues was amplified by PCR from pMEB8 using primers 5'-TGCTGAATATGC GCTGTCTTTTGGTACCGAA-3' (SEQ ID NO: 40) and 5'-CCGTTAAA CGCCGCAGCAGCGATAGCGCC-3' (SEQ ID NO: 41). The resultant PCR fragment was purified by QIAquick column (Qiagen Inc., Studio City, CA) and ligated into the Pmel site of plasmid pCE- 5-35 to generate pEPS#31. This Ssp DnaE intein contains three mutations, Cysl e Ala/Cys+1 
5.	Methods of Assaying EPSPS Activity<br>
Plating assay for EPSPS activity. The presence of a functional EPSPS protein could be determined in vivo using E. coli strain ER2799, which lacks an endogenously active EPSPS   (see   above).   ER2799   cells   alone   fail   to   grow   on   M9   minimal   plates<br><br>
(supplemented with 0.3 mM IPTG). In the following description when M9 minimal plates are mentioned they also contain 0.3 mM IPTG. Plasmid pC+E2, which contains the full length wild type EPSPS gene with a C301 to T mutation, is able to rescue growth of ER2799 on the M9 minimal plates when introduced by transformation.<br>
Assaying the Ssp DnaE cis-splicing constructs. Plasmids pCE182DnaE, pCE215DnaE, pCE235DnaE, pCE267DnaE (0.05 pg of each) were transformed into E. coli ER2799 cells by electroporation (Sambrook, et al., Molecular Cloning : A Laboratory Manual, 2nd Edition, Cold Spring Harbor Laboratory, NY: Cold Spring Harbor Laboratory Press (1989)), see Fig. 11. 0.8 mL of LB media was added to the transformed cells and these were incubated at 37°C for 1 hour with shaking. 200 pL of this solution was plated onto either LB or M9 minimal plates supplemented with 0.1 mg/mL ampicillin (Sambrook, et al., Molecular Cloning : A Laboratory Manual, 2nd Edition, Cold Spring Harbor Laboratory, NY: Cold Spring Harbor Laboratory Press (1989)). The plates were incubated for varying length of time and at various temperatures. The most commonly used being overnight at 37°C.<br>
Assaying the Ssp DnaE trans-splicing constructs. The activity of each EPSPS trans construct was assayed by co- transforming the constructs to be tested into ER2799 and plating on either an M9 minimal plate, containing 0.3 mM IPTG, or an LB plate in which both were supplemented with 0.1 mg/mL ampicillin and 0.05 mg/mL kanamycin. In cases where only one plasmid contained the EPSPS gene or a portion of the EPSPS gene the complementary antibiotic resistance was supplie by co-transforming the E. coli with either pCYB3 or pKYBl (New England Biolabs, Beverly, MA), which has no EPSPS gene present.<br>
The plasmids used were: pC+E2, p215EN2, p235EN2, pEPS#28, pEPS#29, pEPS#33, pEPS#37, pEPS#34, and pEPS#36. These plasmids were co-transformed (Sambrook, et al., Molecular Cloning : A Laboratory Manual, 2nd Edition, Cold Spring Harbor Laboratory, NY: Cold Spring Harbor Laboratory Press (1989)) using 0.1 ug of the appropriate plasmids, in various combinations, into ER2799 E. coli cells, and plated on both LB plates and M9 minimal media plates, each containing 100 pg/mL ampicillin and<br><br>
50 ug/mL kanamycin. The M9 minimal plate also contained 0.3 mM IPTG. Individual clones were picked from each LB plate and stripped on one M9 minimal media selection plate following incubation at 37oC overnight or RT for 2-3 days. The combinations used were: WT, pC+E2 and pKYBl (New England Biolabs, Beverly, MA); 215NC, p215EN2 and PEPS#28; 215C, pEPS#2$ and pCYB3; 235NC-Dead, pEPS#33 and pEPS#37; 235NC, P235EN2 and PEPS#29; 235N, p235EN2 and pKYBl; 235C, pEPS#29 and pCYB3; 235N-215C, p235EN2 and pEPS#28; and 235 complement, pEPS#34 and yEPS#36 (see Figure 12\.<br>
Determination of ER2799 growth in liquid culture in the presence or absence of glyphosate. The testing of glyphosate resistance for the 235 trans constructs was made using plasmid combinations as follows; WT, pC+E2 and pKYBl; 235NC-Dead, pEPS#33 and pEPS#37; 235NC, p235EN2 and pEPS#29; 235N, p235EN2 and pKYBl; 235C, pEPS#29 and pCYB3; and 235 complement, pEPS#34 and pEPS#36. These plasmids were co-transformed into ER2799 E. coli cells as described above and plated onto LB plates containing 100 ug/mL ampicillin and 50 ug/mL kanamycin. As a control, pCYB3/pKYB were co-transforrned into E. coli strain ER2744, and plated on an LB plate containing 100 ug/mL ampicillin and 50 ug/mL kanamycin. A preculture was prepared for each transformation by inoculating the fresh colony into LB medium supplemented with 100 pg/mL ampicillin and 50 ug/mL kanamycin at 30°C for overnight. Equal amounts of pre-culture (10-llpL depending on the cell density) was inoculated into freshly-made M9 minimal medium containing 100 ug/ml of ampicillin, 50 pg/ml of kanamycin and 0.3 mM IPTG in the absence or presence of different amounts of glyphosate. The growth of each construct was measured by OD at 600 nm, see Figure 13.<br>
Growth of the cis 235 construct in M9 liquid minimal media. Two plasmid vectors one with a splicing competent Ssp DnaE intein (235 cis) and another with a splicing incompetent intein (235 dead), pCE235DnaE and pEPS#31, respectively, were transformed into separate ER2799 E. coli cells and plated on LB plates supplemented with 100 pg/mL ampicillin and 50 pg/mL kanamycin. A preculture was prepared for each transformation by inoculating the fresh colony into LB medium supplemented with 100<br><br>
ug/mL ampicillin and 50 IJg/mL kanamycin at 30°C for overnight. Equal amounts of pre-culture (10-llpL depending on the cell density) was inoculated into freshly-made M9 minimal medium containing 100 ug/ml of ampicillin, 50 gug/ml of kanamycin and 0.3 mM IPTG. The cell density was determined at various times using the OD at 600 nm (see FigureM). The Ncol to Kpnl fragment of pEPS#31 was ligated into the same sites in plasmid pCEN2 to generate pEPS#33. Plasmid pEPS#37 was created by cloning the BgUI to PstI fragment of pEPS#31 into the same sites in plasmid pKEB12.<br>
EXAMPLE IV<br>
Trans-splicing of two unrelated gene products aminoglycoside-3-acetyltransferase (aadA) and soluble modified green fluorescent protein (smGFP), to give rise to a functional hybrid protein in E. coli.<br>
Aminoglycoside-3-acetyltransferase gene was fused to Ssp DnaE intein N-fragment (INn). The C fragment of the Ssp DnaE intein (INc) was fused to the smGFP gene. The fusion proteins could be translated as individual polypeptides from the respective constructs. These fusion protein coding DNA sequences were cloned into either pIH976 (Figure 17) or pAGR3 (Figure 18) plasmids. Both the plasmids (pIHaadE-N (pIH976 containing aadA and nonterminal) and pAGRE- CsmGFP (pAGR3 containing INc and smGFP)) were co- transformed in to E. coli (Figure 19A). The transformed E coli were resistant to spectinomycin/streptomycin sulfate (Figure 19B). The cell extracts were made after 16 hrs of growth. The proteins in the extract was separated on SDS tris glycine gel and blotted on to a PVDF membrane. This membrane was probed with anti GFP monoclonal antibodies. Trans-splicing was observed in E. coli extracts, where both the plasmids were introduced. As a result of trans-splicing the fusion product had a molecular mass identical with the calculated cumulative mass of both the proteins (Figure 19C).<br>
The following protocol describes the production of cassettes, pIHaadE-N (Aminoglycoside-3-acetyltransferase  gene  fused  to  DNA  encoding  INn),  pAGRE-<br><br>
CsmGFP (DNA encoding INc was fused to smGFP gene), Western blotting and detection.<br>
Polymerase chain reaction (PCR) was used for cloning of the open reading frames<br>
(ORFs) in to the desired plasmids. The reaction contains Vents DNA polymerase buffer<br>
supplemented with 2 mM magnesium sulfate, 200 pM dNTPs, 1 uM of each primer and<br>
100 ng plasmid DNA in a total volume of 50psi with 2 units of Vents DNA polymerase.<br>
Between 10 to 20 rounds of amplification were carried out using a Perkin- Elmer gene<br>
amp PCR 2400 system (Emeryville, CA). The following primers used for amplification<br>
of	the	aadA	gene	(aadA	forward	primer:<br>
GCCTTAATTAACCATGAGGGAAGCGGTGATCGC CG (SEQ ID NO: 47), rada<br>
reverse primer: TGCGGTCGACTTTGC CGACTACCTTGGTGATCTC (SEQ ID NO:<br>
48). PCR products were purified using a PCR purification kit (QIAquick PCR<br>
purification) from Qiagen (Valencia, CA). Purified PCR products were digested by Pac I<br>
and Sal I restriction enzymes and cloned in to pNEB193 (New England Biolabs, Inc.,<br>
Beverly, MA) plasmid. The clone containing the aadA gene was named pNEBaad3.<br>
Similar protocol was used for amplification and cloning of the smGFP gene using<br>
specific	primers	(smGFP	forward	primer:<br>
CCCAAGCTTGGCGCCATGAGTAAAGGAGAAGAACTTTTCAC (SEQ ID NO: 49) and smGFP reverse primer: GCGACCGGTTTATTTGTATAG TTCATCCATGCCATG (SEQ ID NO: 50) into pLITMUS 28 (New England Biolabs, Inc., Beverly, MA). The clone containing the smGFP gene was named psmGFP7. Sequences for both aadA and smGFP genes were verified by DNA sequencing.<br>
The Intein from the dnaE gene of Synechocystis species PCC6803 was PCR amplified. The amino terminal part of the intein (amino acids 1-123) is referred to as INn and the carboxy terminal as INc (amino acids 124-159). Both INn and INc fragments were cloned into pLITMUS 28 and pNEB193 respectively. The primer pairs for amplification of INn and INc are listed (INn forward primer: AGGGAATTCGTCGACAAATTTG CTGA ATATTGCCTGTCT (SEQ ID NO: 51), INn reverse primer: GGCCTCGAGTTATTTAATTGTCCCAGCGTCAAGTAATG (SEQ ID NO: 52), INc<br><br>
forward primer: AGCTTTGTTTAAACCATGGTTAAAG TTATCGGTCGTAGATC<br>
(SEQ	ID	NO:	53),	INc	reverse	primer:<br>
CAGCGTCGACGGCGCCGTGGGATTTGTTAAAGCAGTTAGCAGC (SEQ ID NO: 54)). The plasmids containing the INn and INc fragments were pLitDnaE-Nl and pNEBDnaE-C2 respectively.<br>
Fusion constructs of intein fragments and either aadA or smGFP gene products were made in the following way: BamHI and Sa'i fragment (800bp) from pNEBaad3 was ligated into BamHI-Sall digested pLitDnaE-Nl to give rise to pAENl. In a similar way, the 150 bp insert (pNEBIN-c digested with PstI and KasI) was ligated into PstI and KasI digested pLit SmGFP5 to give rise to pGFPEC. Plasmid pAEN contains aadA gene in frame with INn and pGFPEC contain smGFP gene in frame with INc.<br>
The fused genes were PCR amplified and cloned into E. coli expression vectors. The inserts of pAEN and pGFPEC were cloned into pIH976 (Ncol and Sad site) and pAGR3 (EcoRI and SacII sites) vectors. The primers are listed (aadA- INn forward primers: CATGCCATGGGGGAAGCGGTGATCGC CGAAG (SEQ ID NO: 55), aadA-INn reverse primers: ACGCG AGCTCTTATTTAATTGTCCCAGCGTCAAGTAATG (SEQ ID NO: 56), INc-smGFP forward primer: CGAATTCTATGGTTAAAGTTATCGG TCGTAGATC (SEQ ID NO: 57), INc-smGFP reverse primer: AG CCCGCGGTTATTTGTATAGTTCATCCATGCCATG (SEQ ID NO: 58)). The E. coli expression plasmids were pIH976-aadE-N and pAGR-Nc-smGFP, under the control of Ptac promoter of the host. Either of the plasmids or both together were transformed into E. coli ER1992 (New England Biolabs, Inc., Beverly, MA) and plated on LB agar-Ampicillin plates as well as LB agar ampicillin and spectinomycin plates.<br>
For Western blotting, E. coli cell extracts were mixed with SDS loading dye with 1 mM DTT, boiled at 95°C for 5 min and loaded on a 10-20% Tris-glycine-SDS gradient gel. The proteins were blotted on an Immobilin-P membrane and probed with an anti-GFP monoclonal antibody (Roche Molecular Biochemicals, Indianapolis, IN) followed by chemiluminescent detection of the GFP and aadA-GFP fusion protein.<br><br>
1<br><br>
EXAMPLE V<br>
Utilization of plant promoters in E. coli for trans-splicing of two unrelated gene product, aminogIycoside-3- acetyltransferase (aadA) and soluble modified green fluorescent protein (smGFP) to give a functional hybrid protein.<br>
The above DNA fragments were cloned downstream of the chloroplast specific promoter PpsbA (SEQ ID NO: 59). A terminator sequence of the same gene (TpsbA (SEQ ID NO: 60) was placed down stream of the cloned gene. The two genes were expressed in opposite direction to avoid read through. The plant promoters were functional upon transformation in to E. coli and trans-spliced products (aadA- smGFP fusion protein, 57 kDa) were observed in Western blot assay using anti GFP antibodies. Thus chloroplast specific promoters are functional in E. coli and could be used for gene expression studies.<br>
The following protocol describes the production of a E. co/i/plant shuttle vector (pNCTl 14/pNCT224) that is capable of homologous recombination of a transgene (s) in vivo.<br>
A shuttle vector consists of elements that will make it functional in both E. coli as well as plant cell. Plasmid pLITMUS28 (New England Biolabs, Inc., Beverly, MA) is the backbone for the pNCT114 and pNCT224 gene targeting vector. The vector DNA comprises, at least (1) two DNA sequence homologous to the plastid genome (also referred as targeting sequence/fragment), (2) one or more promoter element, (3) transcription terminator elements, and (4) one or more selectable/drug resistance (non-lethal) marker gene.<br>
Promoter element (PpsbA) DNA sequences were PCR amplified from genomic DNA<br>
extracted from 7 days old tobacco seedlings using the CTAB method as described by<br>
Murray and Thompson (Nucleic Acids Res., 16: 4321-4325 (1980)). The primers used for<br>
amplification	are	listed	(PpsbA	forward	primer:<br>
AACTGCAGGAATAGATCTACATACACCTTGG (SEQ ID NO: 64), PpsbA reverse primer: CCGCTCGAGCTTAATTAAGGTAA AATCTTGGTTTATTTAATC (SEQ ID<br><br>
NO: 65)). Similarly the terminator sequence (TpsbA) was amplified by PCR and cloned. The primers used for amplification are listed (TpsbA forward primer: GCGACCGGTGATCCTGGCCTAGTCTATAGGAGG (SEQ ID NO: 66), TpsbA reverse primer: AGGCCTAGGAGAATACT CAATCATGAATAAATGC (SEQ ID NO: 67)). A vector with a psbA promoter and terminator DNA sequence allows genes to be cloned in between these for expression of the protein. The targeting DNA sequences were amplified and inserted outside of the promoter and terminator in a flanking manner (Figure 20), thus facilitating homologous recombination of the trans- gene at a predetermined loci. pNCTIH contains 16SrDNA- trnaV and rps7/12 targeting sequence (SEQ ID NO: 61), whereas, pNCT224 contains orf228-ssb as left border and orfl244 as right border (SEQ ID NO: 62). The following primers were used for PCR amplification of the targeting sequences.<br>
Primers for pNCTl 14<br>
Left border forward primer: TTGGCGCGCTTGACGATATAGCAATTTTGCTTGG (SEQ ID NO: 68)<br>
Left border reverse primer: TTGCGTACGATTTATCTCAGATTAGATGGTCTAG (SEQ ID NO: 69)<br>
Right border forward primer: TTGCCTAGGCGTATTGATAATGCCGTCTTAACCAG (SEQ ID NO: 70)<br>
Right border reverse primer: AGGGGTACCGAATTCAAGATTCTAGAGTCTAGAG (SEQ ID NO: 71)<br>
Primers	for	pNCT224	Left	border	forward	primer:<br>
TTGGCGCGCAATTCACCGCCGTATGGCTGACCGG (SEQ ID NO: 72)<br>
Left border reverse primer: TTGCGTACGCCTTTGACTTAGGATTAGTCAGTTC (SEQ ID NO: 73)<br><br>
Right border forward primer: TTGCCTAGGGTCGAGAAACTCAACGCCACTATTC (SEQ ID NO: 74)<br>
Right border reverse primer: AGGGGTACCATCACGATCTTATATATAAGAAGAAC (SEQ ID NO: 75)<br>
A detailed diagram for pNCT 114/224 is in Figure 20A. Both the plasmids contain two promoters and two terminator DNA fragments. For directional cloning, unique restriction enzyme sites are incorporated. Plasmid pNCT114 and pNCT224 have unique restriction enzyme sites (Pmel-Agel and Pacl-Xhol sites). Insert from plasmid pAEN (aadA gene in frame with INn) was obtained by digesting with Pacl-Xhol and pGFPEC (smGFP in frame with INc) was obtained by digesting with Pmel-Agel and ligated sequentially into pNCTl 14 or pNCT224. The plasmids are designated as p!15ag and p225ag (Figure 21 A). The plasmids were transformed into E. coli and selected with ampicillin and spectinomycin (Figure 2IB). The cell extracts were made from overnight cultures and separated on 10-20% Tris-glycine-SDS gradient gel. The proteins were blotted on an Immobilin-P membrane and probed with an anti-GFP monoclonal antibodies (Roche Molecular Biochemicals, Indianapolis, IN) followed by chemiluminescent detection of the GFP and aadA-GFP fusion proteins (Figure 21C).<br>
EXAMPLE VI<br>
Cis-splicing of the EPSPS and ALS gene products in plant cytoplasm expressed from a DNA cassette integrated into molecular DNA<br>
The introduction of DNA into plant nuclei has been achieved in many different ways, such as, electroporation, polyethylene glycol mediated, Agrobacterium mediated, microinjection and biolistic transformation. In accordance with the present invention, one should determine if the plant cytoplasm will mediate protein-splicing event in cis or trans. This will be a prerequisite for further trans-splicing technologies in plants. This technique will be useful if the target protein needs specific cytoplasmic modification for activity. Either of the above techniques may be employed to introduce the EPSPS and/or ALS<br><br>
gene cassettes into tobacco or any other suitable plant tissue or cells. The general cassette consists of: (1) Drug selection/degrading marker gene such as kanamycin or any other suitable selection marker; (2) a strong promoter element such as 35sCMV (cauliflower mosaic virus); and (3) right and left border T DNA repeats of Agrobacterium. Such a cassette could be introduced into plants either by a biolistic process or by Agrobacterium mediated gene transfer (Horsch, et al.. Nature 227: 1229-1231 (1985)). The cassette is based on pBI121 gene transfer vector (Jefferson, et al., EM80 J., 6: 3901-3907 (1987)). The design of the final cassette is illustrated in Figure 22.<br>
In the biolistic process, the transforming DNA is coated on the surface of fine gold particles and introduced into the plant cell by a particle accelerator gun (PDSIOOO/He gun, Biorad, Richmond, CA). For Agrobacterium mediated gene transfer the transforming DNA cassette is introduced into the bacteria. The Agrobacterium harboring the cassette is allowed to be in contact with a disk or tissue section from tobacco or other suitable plant leaves. This facilitates the transfer of the DNA cassette to the plant nuclei. In either of the above approaches, the DNA finally gets integrated into the plant nuclei. The putative transformed cells are used for marker gene (drug) selection. The plants regenerated in presence of the selected drugs are strong transgenic candidates. After the plants are mature, the cell extracts will be taken and mixed with SDS loading dye with 1 mM DTT, boiled at 95°C for 5 min and loaded on a 10-20% Tris-glycine-SDS gradient gel.<br>
The separated proteins will be blotted on an Immobilin-P membrane and probed with an anti-ALS or EPSPS antibody. PCR may then be performed to determine if the gene has integrated in a predictable fashion without rearrangement.<br>
This technique would be useful for proteins that need specific modification for activity/folding in cytoplasmic environment. A part of the target protein gene with necessary transport signal and splicing elements will be placed in an organelle for cytoplasmic transport in the form of a precursor polypeptide.<br><br>
These plants are allowed to grow in the greenhouse till they mature and the seeds will be collecte. The collecte seeds are then germinated and Fl plants tested for herbicide resistance. A small-scale trial may be done to see whether or not the segregation pattern of the introduced transgenes follows a Mendelian inheritance pattern. Integration into nuclear DNA would yield Mendelian inheritance, whereas integration into chloroplast DNA woule yield non-Mendelian maternal inheritance.<br>
EXAMPLE VII<br>
Trans-splicing of a split gene, such as EPSPS/ALS or of two unrelated gene products, such as aminoglycoside-3- acetyltransferase (aadA) and soluble modified green fluorescent protein (smGFP), to give a functional hybrid protein in plant chloroplast.<br>
The aim of these experiments is to investigate if trans- splicing is feasible in plant chloroplasts. Plant chloroplasts are similar to bacteria with respect to their transcription and translation machinery. In Examples IV-VI, we have used the naturally occurring intein from the dnaE gene of the Synechocystis species PCC6803, which is a cyanobacterium. Cyanobacteria are photosynthetic bacteria which are similar to plant chloroplasts. Thus it should be possible for inteins to splice or trans-splice in plant chloroplasts. These proposed experiments are in two sections: Section 1, To demonstrate the trans-splicing event of two unrelated gene products aadA and smGFP in plant chloroplasts, where both genes are integrated in chloroplast genome; and Section 2, Trans- splicing in chloroplast, where the smGFP gene cassette is integrated into the nuclear genome and the translated protein containing a transit peptide (rubisco 3A-INc-smGFP) is imported into the chloroplast for the reaction to proceed. The chloroplast will have aadA gene fused to INn fragment. The detailed protocol is narrated below.<br>
To demonstrate trans-splicing of two unrelated gene products, aminoglycoside-3-acetyltransferase (aadA) and soluble modified green fluorescent protein (smGFP) in chloroplast, upon transcription and translation in chloroplasts.<br><br><br><br>
We Claim:<br>
1.	An isolated DNA fragment comprising a portion of an ESPS or an ALS gene wherein the ESPS or ALS gene has been split into 2 portions at a split site in the gene and the two portions of the gene can be reconstituted by transplicing.<br>
2.	An isolated DNA fragment as claimed in claim 1, wherein the DNA fragment is selected from the group consisting of the DNA encoding amino acids 1-235 or portions thereof; SEQ ID NO: 24, SEQ ID NO: 25, SEQ ID NO: 28, SEQ ID NO: 29, SEQ ID NO: 30, SEQ ID NO: 31, SEQ ID NO: 36, SEQ ID NO: 37, SEQ ID NO: 38 and SEQ ID NO: 39.<br>
3.	An isolated DNA fragment as claimed in claim 1, wherein gene of interest<br>
is ALS gene.<br>
4.	An isolated DNA fragment of claim 1, wherein the DNA fragment is selected from the group consisting of SEQ ID NO: 10, SEQ ID NO:ll, SEQ ID NO:12, SEQ ID NO:13, SEQ ID NO:17, SEQ ID NO:17, SEQ ID NO:18 and SEQ ID NO:19.<br>
5.	An isolated DNA fragment as claimed in claim 1, wherein the said fragment is expressed either in E. coli or in maize.<br>
6.	An isolated DNA fragments substantially such as herein described with figures, examples and foregoing description.<br>
Dated this on 3rd May 2005<br>
[GOUTAM BHATTACHARYYA]<br>
OF K&amp;S PARTNERS<br>
AGENT FOR THE APPLICANT(S)<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc5LW11bW5wLTIwMDUtYWJzdHJhY3QoMTctMS0yMDA3KS5kb2M=" target="_blank" style="word-wrap:break-word;">379-mumnp-2005-abstract(17-1-2007).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc5LW11bW5wLTIwMDUtYWJzdHJhY3QoMTctMS0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">379-mumnp-2005-abstract(17-1-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc5LW11bW5wLTIwMDUtY2FuY2VsbGVkIHBhZ2VzKDE3LTEtMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">379-mumnp-2005-cancelled pages(17-1-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc5LW11bW5wLTIwMDUtY2xhaW1zKGdyYW50ZWQpLSgxNy0xLTIwMDcpLmRvYw==" target="_blank" style="word-wrap:break-word;">379-mumnp-2005-claims(granted)-(17-1-2007).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc5LW11bW5wLTIwMDUtY2xhaW1zKGdyYW50ZWQpLSgxNy0xLTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">379-mumnp-2005-claims(granted)-(17-1-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc5LU1VTU5QLTIwMDUtQ09SUkVTUE9OREVOQ0UoMjAtMDYtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">379-MUMNP-2005-CORRESPONDENCE(20-06-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc5LW11bW5wLTIwMDUtY29ycmVzcG9uZGVuY2UoMjktMTAtMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">379-mumnp-2005-correspondence(29-10-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc5LW11bW5wLTIwMDUtY29ycmVzcG9uZGVuY2UoaXBvKS0oMjktOS0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">379-mumnp-2005-correspondence(ipo)-(29-9-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc5LW11bW5wLTIwMDUtZHJhd2luZygxNy0xLTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">379-mumnp-2005-drawing(17-1-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc5LW11bW5wLTIwMDUtZm9ybSAxKDI1LTktMjAwNikucGRm" target="_blank" style="word-wrap:break-word;">379-mumnp-2005-form 1(25-9-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc5LW11bW5wLTIwMDUtZm9ybSAxKDYtNS0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">379-mumnp-2005-form 1(6-5-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc5LW11bW5wLTIwMDUtZm9ybSAxOCg3LTExLTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">379-mumnp-2005-form 18(7-11-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc5LW11bW5wLTIwMDUtZm9ybSAyKGdyYW50ZWQpLSgxNy0xLTIwMDcpLmRvYw==" target="_blank" style="word-wrap:break-word;">379-mumnp-2005-form 2(granted)-(17-1-2007).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc5LW11bW5wLTIwMDUtZm9ybSAyKGdyYW50ZWQpLSgxNy0xLTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">379-mumnp-2005-form 2(granted)-(17-1-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc5LW11bW5wLTIwMDUtZm9ybSAzKDMtNS0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">379-mumnp-2005-form 3(3-5-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc5LW11bW5wLTIwMDUtZm9ybSA1KDMtNS0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">379-mumnp-2005-form 5(3-5-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc5LW11bW5wLTIwMDUtcG93ZXIgb2YgYXV0aG9yaXR5KDI1LTktMjAwNikucGRm" target="_blank" style="word-wrap:break-word;">379-mumnp-2005-power of authority(25-9-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc5LU1VTU5QLTIwMDUtU0VRVUVOQ0UgTElTVElORygyMC0wNi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">379-MUMNP-2005-SEQUENCE LISTING(20-06-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QxLmpwZw==" target="_blank" style="word-wrap:break-word;">abstract1.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="224082-implanting-device-for-inserting-implantable-objects-benath-the-skin.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="224084-compounds-of-the-formulae-iii-iii-a-and-iii-b-and-preparation-process-thereof.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>224083</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>379/MUMNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>06/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>06-Feb-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>29-Sep-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>06-May-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>BOSTON BIOMEDICAL RESEARCH INSTITUTE</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>25 GROVE STREET, WATERTOWN, MASSACHUSETTS 02472,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>MING-QUN XU</td>
											<td>40 CRESCENT ROAD, HAMILTON, MASSACHUSETTS 01982,USA.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>MING-QUN XU</td>
											<td>40 CRESCENT ROAD, HAMILTON, MASSACHUSETTS 01982,USA.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>SRIHARSA PRADHAN</td>
											<td>6 BEVERLEY COMMONS, ESSEX #1, BEVERLEY, MASSACHUSETTS 01915, USA.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>LUO SUN</td>
											<td>SU46 PLUM STREET, HAMILTON, MASSACHUSETTS 01982, USA</td>
										</tr>
										<tr>
											<td>5</td>
											<td>LIXIN CHEN</td>
											<td>60 RANTOUL STREET #409, BEVERLY, MASSACHUSETTS 01915, USA.</td>
										</tr>
										<tr>
											<td>6</td>
											<td>INCA GHOSH</td>
											<td>10 COTTAGE AVENUE, SOMERVILLE, MASSACHUSETTS 02144, USA.</td>
										</tr>
										<tr>
											<td>7</td>
											<td>THOMAS C. EVANS</td>
											<td>68 ALBION STREET, 1ST FLOOR, SOMERVILLE, MASSACHUSETTS 02143, USA.</td>
										</tr>
										<tr>
											<td>8</td>
											<td>DONALD G. COMB</td>
											<td>9 PROCTOR STREET, MANCHESTER, MASSACHUSETTS 01944, USA</td>
										</tr>
										<tr>
											<td>9</td>
											<td>HENRY PAULUS</td>
											<td>85 EAST INDIA ROW, BOSTON, MASSACHUSETTS 02110, USA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/10</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-05-23</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/224083-an-isolated-dna-fragments by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:43:17 GMT -->
</html>
