<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/224268-a-modified-phytase-and-a-process-for-the-preparation-of-the-same by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:00:17 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 224268:A MODIFIED PHYTASE AND A PROCESS FOR THE PREPARATION OF THE SAME</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A MODIFIED PHYTASE AND A PROCESS FOR THE PREPARATION OF THE SAME</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A modified phytase with a specific activity improved over the specific activity of the corresponding unmodified phytase characterized in that the amino acid sequence of the unmodified phytase has been changed at the position corresponding to position 27 of the phytase&#x27; of Aspergillus niger to an amino acid selected from the group consisting of Ala, Val, Leu, lIe, Thr, Asn and Gly, with the provisio that if the unmodified phytase is a phytase of Aspergillus terreus, position 27 of the amino acid sequence is changed to an 8:ffiino acid selected from the group consisting of Ala, Val, Ile, Thr, Asn and Gly.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Phytases (nryo-inositol hexakisphosphate phosphohydrolases; EC 3.1.3.8) are enzymes that hydrolyze phytate (nvyo-inositol hexakisphosphate) to myo-inositol and inorganic phosphate and are known to be valuable feed additives.<br>
A phytase was first described in rice bran in 1907 [Suzuki et al., Bull. Coll. Agr. Tokio Imp. Univ. 7, 495 (1907)] and phytases from Aspergillus species in 1911 [Dox and Golden, J. Biol. Chem. 10, 183-186 (1911)]. Phytases have also been found in wheat bran, plant seeds, animal intestines and in microorganisms [Howsen and Davis, Enzyme Microb. Technol. 5, 377-382 (1983), Lambrechts et al., Biotech. Lett. 14., 61-66 (1992), Shieh and Ware, Appl. Microbiol. 16,1348-1351 (1968)].<br>
The cloning and expression of the phytase from Aspergillus niger (ficuum) has been described by Van Hartingsveldt et al., in Gene, 127. 87-94 (1993) and in European Patent Application, Publication No. (EP) 420 358 and from Aspergillus niger var. awamori by Piddington et al., in Gene 133, 55-62 (1993).<br>
Cloning, expression and purification of phytases with improved properties have been disclosed in EP 684 313. However, since there is a still ongoing need for further improved phytases, especially with respect to the activity properties, it is an object of the present invention to provide the following:<br>
i) A process for the production of a modified phytase with improved activity properties characterized therein that the following steps are effected:<br>
a)	the three dimensional structure of the phytase to be modified and, optionally of another phytase with activity properties which are more favorable than the ones of the phytase to be modified is/are computer modelled on the basis of the three dimensional structure of the phytase of Aspergillus niger (ficuum);<br>
b)	the structure of the active sites of the phytase to be modified and of the phytase with the more favorable activity properties are compared<br><br>
and those amino acid residues in both active sites which are different are identified;<br>
c)	a DNA sequence coding for a modified phytase is constructed by changing the nucleotides coding for at least one of the amino acids by which both active sites differ;<br>
d)	integrating such a DNA sequence into a vector capable of expression in a suitable host cell;<br>
e)	transforming a suitable host cell by the DNA sequence of c) or the vector of d), growing said host cell under suitable growth conditions and isolating the modified phytase from the host cell or the culture medium by methods known in the state of the art; or<br>
ii) a process as described under i) wherein the phytase to be modified is of eukaryotic, preferably fungal, more preferably Aspergillus, e.g. Aspergillus fumigatus origin; or<br>
iii) a process as described under i) or ii) wherein the phytase with more favorable activity properties is of eukaryotic, preferably fungal, more preferably Aspergillus, e.g. Aspergillus niger or Aspergillus terreus (Aspergillus terreus cbs 116.46 or 9A1) origin; or<br>
iv) a process as described under i), ii) or iii) wherein the phytase to be modified is a phytase of Aspergillus fumigatus and the phytase with the more favorable activity properties is the Aspergillus terreus phytase or the phytase of Aspergillus niger.<br>
In this context it should be mentioned that another possibil y of producing phytases with improved properties is by isolating phytases from the same organism, like for example the Aspergillus ficuum, but different strains which can be found in nature and have been deposited by any of the known depository authorities. Their amino acid sequences can be determined by cloning their corresponding DNA sequences by methods as described, e.g. in European Patent Application No. (EP) 684 313. Once such sequences have been defined they can be modeled on the basis of the three-dimensional structure of the A. niger phytase and the active sites of both sequences can be compared to find out whether such phytase should have improved activity properties (see Example 8) or both active site sequences can<br><br>
be compared directly and than tested for increased and/or improved activity by the assays described in the present application.<br>
It is furthermore an object of the present invention to provide a modified phytase which is obtainable by a process as described above.<br>
It is in general an object of the present invention to provide a phytase which has been modified in a way that its activity property is more favorable than the one of the non-modified phytase, specifically such a phytase characterized therein that the amino acid sequence of the non-modified phytase has been changed by deletion, substitution and/or addition of one or more amino acids, more specifically such a phytase wherein changes have been made at at least one position which is homologous to one of the following positions of the amino acid sequence of the phytase of Aspergillus (A.) niger (see Fig.l): 27, 66, 71,103,140,141,188, 205, 234, 235, 238, 274, 277, 282, 340 and/or 424, preferably 27, 66,140, 205, 274, 277, 282 and/or 340, and even more specifically such a phytase which is the phytase of eukaryotic, preferably fungal, more preferably Aspergillus and most preferably Aspergillus fumigatus, origin.<br>
It is furthermore an object of the present invention to provide such a phytase wherein at position 27 or at least at position 27 a change occurs, preferably a phytase wherein the amino acid at position 27 is replaced by one selected from one of the following groups:<br>
a)	Ala, Val, Leu, He; or<br>
b)	Thr or<br>
c)	Asn; and furthermore such a phytase wherein in addition to position 27 a change occurs also at position 66 or wherein in addition to position 27 a change occurs also at position 140 and/or at positions 274 and/or 277.<br>
It is also an object of the present invention to provide a phytase as specified above which is characterized by at least one of the following mutations: Q27L, Q27N, Q27T, Q27I, Q27V, Q27A, Q27G, S66D, S140Y, D141G, A205E, Q274L, G277D, G277K, Y282H and/or N340S.<br>
It is furthermore an object of the present invention to provide phytase muteins which are resistant against degradation by proteases of fungal, preferably Aspergillus and most preferably Aspergillus niger (ficuum) origin. Such muteins are characterized therein that at least at one of the following positions (which refers to the homologous position in the amino<br><br>
acid sequence of A. niger), namely position 130 or 129 and 130, preferably of the Aspergillus fumigatus or 167, 168 preferably of the A. nidulans phytase amino acid sequence, the amino acid which is present in the wild type sequence has been replaced against another amino acid which is known to change the protease sensitivity, e.g. in the case of A. fumigatus at position 130 from "S" to "N" and at position 129 from "R" to "L" and in case of A. nidulans at position 167 from "K" to "G" and at position 168 from R to Q. Such positions can be also combined with those providing for improved activity properties.<br>
In this context "improved activity property" means any type of improvement of the activity of the mutated phytase as compared to the non mutated. This could mean for example a higher specific activity, preferably at least two fold or more preferably at least 3 to 4 fold higher in an assay known in the state of the art to measure phytase activity, see e.g. in EP 684 313 or described in the examples of the present application. Furthermore this could mean a different substrate specificity determined in an assay known in the state of the art or as described e.g. in the specific examples of the present invention. This could also mean a maximum of the specific activity at a different more favorable pH or a broad pH optimum ("improved pH profile") determined by an assay as known in the state of the art or as described e.g. in the examples. Finally this could also mean any combination of such properties.<br>
"Homologous" in the context of the present invention means the best fit of the primary, preferably also secondary and most preferably also tertiary structure of the phytase to be modified and the phytase of Aspergillus niger. How such best fit can be obtained is described in detail in Example 1 of the present invention. Figure 1 gives an example of such best fit for the phytase amino acid sequences of Aspergillus fumigatus and Aspergillus terreus aligned on the basis of the Aspergillus niger amino acid sequence which latter sequence is also used as the reference to which the positions of the other sequences, e.g. the ones named before, are referred to. Furthermore the modified Aspergillus fumigatus phytase with the Q27L mutation, means nothing else than the phytase of Aspergillus fumigatus wherein at position 27 according to the assignment as defined above (which is in fact position 23 of the Aspergillus fumigatus amino acid sequence) the naturally occurring glutamine ("Q" refers to the standard UPAC one letter amino acid code) has been replaced by leucine ("L"). All muteins of the present invention are<br><br>
designated in this way independent from wether they are protease resistant muteins or muteins with improved activity properties.<br>
It is furthermore an object of the present invention to provide a DNA-sequence comprising a DNA sequence coding for a phytase as described above, a vector, preferably an expression vector, comprising such a DNA sequence, a host cell which has been transformed by such a DNA sequence or vector, a process for the preparation of a phytase of the present invention wherein the host cell as described before is cultured under suitable culture conditions and the phytase is isolated from such host cell or the culture medium by methods known in the art, and a food or feed composition comprising a phytase of the present invention.<br>
In this context it should be noted that it is also an object of the present invention to provide a DNA sequence which codes for a phytase carrying at least one of the specific mutations of the present invention and which hybridizes under standard conditions with the DNA sequences of the specific modified phytases of the present invention or a DNA sequence which, because of the degeneracy of the genetic code does not hybridize but which codes for a polypeptide with exactly the same amino acid sequence as the one encoded by the DNA sequence to which it does not hybridize or a DNA sequence which is a fragment of such DNA sequences which maintains the activity properties of the polypeptide of which it is a fragment.<br>
"Standard conditions" for hybridization mean in the context the conditions which are generally used by a man skilled in the art to detect specific hybridization signals and which are described, e.g. by Sambrook et al., "Molecular Cloning", second edition, Cold Spring Harbor Laboratory Press 1989, New York, or preferably so called stringent hybridization and non-stringent washing conditions or more preferably so called stringent hybridization and stringent washing conditions a man skilled in the art is familiar with and which are described, e.g. in Sambrook et al. (s.a.).<br>
It is furthermore an object of the present invention to provide a DNA sequence which can be obtained by the so called polymerase chain reaction method ("PCR") by PCR primers designed on the basis of the specifically described DNA sequences of the present invention. It is understood that the so obtained DNA sequences code for phytases with at least the same mutation as the ones from which they are designed and show comparable activity properties.<br><br>
The principles of the polymerase chain reaction (PCR) method are outlined e.g. by White et al., Trends in Genetics, 5, 185-189 (1989), whereas improved methods are described e.g. in Innis et al. [PCR Protocols: A guide to Methods and Applications, Academic Press, Inc. (1990)].<br>
DNA sequences of the present invention can be constructed starting from genomic or cDNA sequences coding for phytases known in the state of the art [for sequence information see references mentioned above, e.g. EP 684 313 or sequence data bases, for example like Genbank (Intelligenetics, California, USA), European Bioinformatics Institute (Hinston Hall, Cambridge, GB), NBRF (Georgetown University, Medical Centre, Washington DC, USA) and Vecbase (University of Wisconsin, Biotechnology Centre, Madison, Wisconsin, USA) or disclosed in the figures by methods of in vitro mutagenesis [see e.g. Sambrook et al., Molecular Cloning, Cold Spring Harbor Laboratory Press, New York]. A widely used strategy for such "site directed mutagenesis", as originally outlined by Hurchinson and Edgell [J. Virol. 8, 181 (1971)], involves the annealing of a synthetic oligonucleotide carrying the desired nucleotide substitution to a target region of a single-stranded DNA sequence wherein the mutation should be introduced [for review see Smith, Annu. Rev. Genet. 19_, 423 (1985) and for improved methods see references 2-6 in Stanssen et al., Nucl. Acid Res., Y7_, 4441-4454 (1989)]. Another possibility of mutating a given DNA sequence which is also preferred for the practice of the present invention is the mutagenesis by using the polymerase chain reaction (PCR). DNA as starting material can be isolated by methods known in the art and described e.g. in Sambrook et al. (Molecular Cloning) from the respective strains. For strain information see, e.g. EP 684 313 or any depository authority indicated below. Aspergillus niger [ATCC 9142], Myceliophthora thermophila [ATCC 48102], Talaromyces thermophilus [ATCC 20186] and Aspergillus fumigatus [ATCC 34625] have been redeposited on March 14, 1997 according to the conditions of the Budapest Treaty at the American Type Culture Cell Collection under the following accession numbers: ATCC 74337, ATCC 74340, ATCC 74338 and ATCC 74339, respectively. It is however, understood that DNA encoding a phytase to be mutated in accordance with the present invention can also be prepared on the basis of a known DNA sequence, e.g. as shown in Fig. 6 in a synthetic manner and described e.g. in EP 747 483 by methods known in the art.<br><br>
Once complete DNA sequences of the present invention have been obtained they can be integrated into vectors by methods known in the art and described e.g. in Sambrook et al. (s.a.) to overexpress the encoded polypeptide in appropriate host systems. However, a man skilled in the art knows that also the DNA sequences themselves can be used to transform the suitable host systems of the invention to get overexpression of the encoded polypeptide. Appropriate host systems are for example fungi, like Aspergilli, e.g. Aspergillus niger [ATCC 9142] or Aspergillus ficuum [NRRL 3135] or like Trichoderma, e.g. Trichoderma reesei or yeasts, like Saccharomyces, e.g. Saccharomyces cerevisiae or Pichia, like Pichia pastoris, or Hansenula polymorpha, e.g. H. polymorpha (DSM5215). A man skilled in the art knows that such microorganisms are available from depository authorities, e.g. the American Type Culture Collection (ATCC), the Centraalbureau voor Schimmelcultures (CBS) or the Deutsche Sammlung fur Mikroorganismen und Zellkulturen GmbH (DSM) or any other depository authority as listed in the Journal "Industrial Property" [(1991) 1, pages 29-40]. Bacteria which can be used are e.g. E. coli, Bacilli as, e.g. Bacillus subtilis or Streptomyces, e.g. Streptomyces lividans (see e.g. Anne and Mallaert in FEMS Microbiol. Letters 114. 121 (1993). E. coli, which could be used are E. coli K12 strains e.g. M15 [described as DZ 291 by Villarejo et al. in J. Bacteriol. 120, 466-474 (1974)], HB 101 [ATCC No. 33694] or E. coli SG13009 [Gottesman et al., J. Bacteriol. 148, 265-273 (1981)].<br>
Vectors which can be used for expression in fungi are known in the art and described e.g. in EP 420 358, or by Cullen et al. [Bio/Technology 5, 369-376 (1987)] or Ward in Molecular Industrial Mycology, Systems and Applications for Filamentous Fungi, Marcel Dekker, New York (1991), Upshall et al. [Bio/Technology 5,1301-1304 (1987)] Gwynne et al. [Bio/Technology 5, 71-79 (1987)], Punt et al. [J. Biotechnol. 17,19-34 (1991)] and for yeast by Sreekrishna et al. [J. Basic Microbiol. 28_, 265-278 (1988), Biochemistry 28, 4117-4125 (1989)], Hitzemann et al. [Nature 293, 717-722 (1981)] or in EP 183 070, EP 183 071, EP 248 227, EP 263 311. Suitable vectors which can be used for expression in E. coli are mentioned, e.g. by Sambrook et al. [s.a.] or by Fiers et al. in Procd. 8th Int. Biotechnology Symposium" [Soc. Franc, de Microbiol., Paris (Durand et al., eds.), pp. 680-697 (1988)] or by Bujard et al. in Methods in Enzymology, eds. Wu and Grossmann, Academic Press, Inc. Vol. 155, 416-433 (1987) and Sttiber et al. in Immunological Methods, eds. Lefkovits and Pernis, Academic Press, Inc., Vol. rV, 121-152 (1990). Vectors which could be used for expression in Bacilli are known in the art and<br><br>
described, e.g. in EP 405 370, Procd. Natl. Acad. Sci. USA 81, 439 (1984) by Yansura and Henner, Meth. Enzymol. 185, 199-228 (1990) or EP 207 459. Vectors which can be used for the expression in H. Polymorpha are known in the art and described, e.g. in Gellissen et al., Biotechnology 9, 291-295 (1991).<br>
Either such vectors already carry regulatory elements, e.g. promotors, or the DNA sequences of the present invention can be engineered to contain such elements. Suitable promotor elements which can be used are known in the art and are, e.g. for Trichoderma reesei the cbhl- [Haarki et al., Biotechnology 7, 596-600 (1989)] or the pkil-promotor [Schindler et al., Gene 130. 271-275 (1993)], for Aspergillus oryzae the amy-promotor [Christensen et al., Abstr. 19th Lunteren Lectures on Molecular Genetics F23 (1987), Christensen et al., Biotechnology 6, 1419-1422 (1988), Tada et al., Mol. Gen. Genet. 229, 301 (1991)], for Aspergillus niger the glaA- [Cullen et al., Bio/Technology 5, 369-376 (1987), Gwynne et al., Bio/Technology 5, 713-719 (1987), Ward in Molecular Industrial Mycology, Systems and Applications for Filamentous Fungi, Marcel Dekker, New York, 83-106 (1991)], alcA- [Gwynne et al., Bio/Technology 5, 718-719 (1987)], sucl- [Boddy et al., Curr. Genet. 24, 60-66 (1993)], aphA- [MacRae et al., Gene 71, 339-348 (1988), MacRae et al., Gene 132,193-198 (1993)], tpiA- [McKnight et al., Cell 46,143-147 (1986), Upshall et al., Bio/Technology 5, 1301-1304 (1987)], gpdA- [Punt et al., Gene 69, 49-57 (1988), Punt et al., J. Biotechnol. 17, 19-37 (1991)] and the pkiA-promotor [de Graaff et al., Curr. Genet. 22, 21-27 (1992)]. Suitable promotor elements which could be used for expression in yeast are known in the art and are, e.g. the pho5-promotor [Vogel et al., Mol. Cell. Biol., 2050-2057 (1989); Rudolf and Hinnen, Proc. Natl. Acad. Sci. 84, 1340-1344 (1987)] or the gap-promotor for expression in Saccharomyces cerevisiae and for Pichia pastoris, e.g. the aoxl-promotor [Koutz et al., Yeast 5, 167-177 (1989); Sreekrishna et al., J. Basic Microbiol. 2£, 265-278 (1988)], or the FMD promoter [Hollenberg et al., EPA No. 0299108] or MOX-promotor [Ledeboer et al., Nucleic Acids Res. 13, 3063-3082 (1985)] for H. polymorpha.<br>
Accordingly vectors comprising DNA sequences of the present invention, preferably for the expression of said DNA sequences in bacteria or a fungal or a yeast host and such transformed bacteria or fungal or yeast hosts are also an object of the present invention.<br>
Once such DNA sequences have been expressed in an appropriate host cell in a suitable medium the encoded phytase can be isolated either from the<br><br>
medium in the case the phytase is secreted into the medium or from the host organism in case such phytase is present intracellularly by methods known in the art of protein purification or described, e.g. in EP 420 358. Accordingly a process for the preparation of a polypeptide of the present invention characterized in that transformed bacteria or a host cell as described above is cultured under suitable culture conditions and the polypeptide is recovered therefrom and a polypeptide when produced by such a process or a polypeptide encoded by a DNA sequence of the present invention are also an object of the present invention.<br>
Phytases of the present invention can be also expressed in plants according to methods as described, e.g. by Pen et al. in Bio/Technology H, 811-814 (1994) or in EP 449 375, preferably in seeds as described, e.g. in EP 449 376.<br>
For example, a DNA sequence encoding a phytase of the present invention can be placed under the control of regulatory sequences from the gene encoding the 12S storage protein cruciferin from Brassica napus. The construct is thereafter subcloned into a binary vector such as pMOG23 (in E. coli K-12 strain DH5a, deposited at the Centraal Bureau voor Schimmel-cultures, Baarn, The Netherlands under accession number CBS 102.90). This vector is introduced into Agrobacterium tumefaciens which contains a disarmed Ti plasmid. Bacterial cells containing this contruct are co-cultivated with tissues from tobacco or Brassica plants, and transformed plant cells are selected by nutrient media containing antibiotics and induced to regenerate into differentiated plants on such media. The resulting plants will produce seeds that contain and express the DNA contruct. Or the phytase-encoding DNA sequence can be placed under the control of regulatory sequences from the 35S promoter of Cauliflower Mosaic Virus (CaMV). The contruct is thereafter subcloned into a binary vector. This vector is then introduced into Agrobacterium tumefaciens which contains a disarmed Ti plasmid. Bacterial cells containing this construct are cocultivated with tissues from tobacco or Brassica plants, and transformed plant cells are selected by nutrient media containing antibiotics and induced to regenerate into differentiated plants on such media. The resulting plants contain and express the DNA construct constitutively.<br>
The plant or plant part containing phytase can be used directly for the preparation of a feed composition or can be extracted from plants or plant organs by methods known in the art. Accordingly it is also an object of the<br><br>
present invention to provide a process for the production of the phytases of the present invention in plants or plant organs, like seeds, the phytases when produced by such methods, the transformed plants and plant organs, like seeds itself.<br>
Once obtained the polypeptides of the present invention can be characterized regarding their properties which make them useful in agriculture any assay known in the art and described e.g. by Simons et al. [Br. J. Nutr. 64, 525-540 (1990)], Schoner et al. [J. Anim. Physiol, a. Anim. Nutr. 66, 248-255 (1991)], Vogt [Arch. Gefliigelk. 56, 93-98 (1992)], Jongbloed et al. [J. Anim. Sci., 70,1159-1168 (1992)], Perney et al. [Poultry Sci. 72, 2106-2114 (1993)], Farrell et al., [J. Anim. Physiol, a. Anim. Nutr. 69, 278-283 (1993), Broz et al., [Br. Poultry Sci. 35, 273-280 (1994)] and Diingelhoef et al. [Animal Feed Sci. Technol. 49, 1-10 (1994)] can be used.<br>
In general the polypeptides of the present invention can be used without being limited to a specific field of application for the conversion of inositol polyphosphates, like phytate to inositol and inorganic phosphate.<br>
Furthermore the polypeptides of the present invention can be used in a process for the preparation of compound food or feeds wherein the components of such a composition are mixed with one or more polypeptides of the present invention. Accordingly compound food or feeds comprising one or more polypeptides of the present invention are also an object of the present invention. A man skilled in the art is familiar with their process of preparation. Such compound foods or feeds can further comprise additives or components generally used for such purpose and known in the state of the art.<br>
It is furthermore an object of the present invention to provide a process for the reduction of levels of phytate in animal manure characterized in that an animal is fed such a feed composition in an amount effective in converting phytate contained in the feedstuff to inositol and inorganic phosphate.<br>
Before describing the present invention in more detail a short explanation of the enclosed Figures is given below.<br><br>
Figure Legends<br>
Figure 1:      Primary sequence alignment of A. niger (ficuum),A. terreus cbsll6.46 and A. fumigatus [ATCC 13073] phytase. Stars show identical residues within the active site and rectangles, non-identical residues within the active site.<br>
Figure 2:      pH optima curves. Specific activity of wild-type and mutant A. fumigatus phytases is plotted against pH of incubation. Filled squares represent A. fumigatus wild-type phytase; Open triangles represent A. fumigatus Q27L mutant; Filled circles represent A. fumigatus Q27L, Q274L mutant; Open squares represent A. fumigatus Q27L, Q274L, G277D mutant.<br>
Figure 3:      Substrate specificities of wild-type and mutant A. fumigatus phytases. (A) wild-type; (B) Q27L single mutant; (C) Q27L, Q274L, G277D triple mutant. The following substrates were used: (1) phytic acid; (2) p-nitrophenyl phosphate; (3) fructose-1,6-bisphosphate; (4) fructose-6-phosphate; (5) glucose-6-phosphate; (6) ribose-5-phosphate; (7) ot-glycerophosphate; (8) P-glycerophosphate; (9) 3-phosphoglycerate; (10) phosphoenol-pyruvate; (11) AMP; (12) ADP; (13) ATP.<br>
Figure 4:      Complete coding sequence and encoded amino acid sequence of the Aspergillus nidulans phytase.<br>
Figure 5:      Complete coding sequence and encoded amino acid sequence of Talaromyces thermophilus phytase.<br>
Figure 6:      Complete coding sequence and encoded amino acid sequence of Aspergillus fumigatus [ATCC 13073] phytase.<br>
Figure 7:      Complete coding sequence and encoded amino acid sequence of Aspergillus terreus CBS 116.46 phytase.<br>
v<br>
Figure 8:      Crystallographic data of the structure of the Aspergillus niger phytase.<br>
Figure 9:      Substrate specificities of wild-type and mutant A. fumigatus<br>
phytase (N1-N6). Substrates 1 to 13 are as indicated for Figure 3.<br><br>
Figure 10:    pH optima curves of further mutant A. fumigatus phytases (Nl-N6). All activity values were standardized (maximum activity = 1.0).<br>
Figure 11a: Stereo picture of the three-dimensional fold of A. niger<br>
(A. ficuum; NRRL 3135) phytase. The active site is indicated with a circle and the catalytically essential amino acid residues Arg 58 and His 59 are shown in ball-and-stick representation. This figure was prepared with the programs "MOLSCRIPT" [Kraulis, P.J., J. Appl. Cryst. 24, 946-950 (1991)] and "RASTER3D" [Merritt, E.A. &amp; Murphy, M.E.P., Acta Cryst., 869-873(1994)].<br>
Figure lib:  Topological sketch, using the same scheme as in (a). The five disulphide bridges are shown as black zigzag lines together with the sequence numbers of the cysteine residues involved. The (3-strands are defined with the sequence numbers A: 48-58,<br>
B: 134-138, C: 173-177, D: 332-337, E: 383-391, and F: 398-403. The a-helices are defined with the sequence numbers a: 66-82, b: 88-95, c: 107-123, d: 141-159, e: 193-197, f: 200-210, g: 213-223, h: 231-246, i: 257-261, j: 264-281, k: 290-305,1: 339-348, m: 423-429, and n: 439-443. The asterisk at the C-terminal end of (3-strand A marks the location of the catalytically essential amino acid residues Arg 58 and His 59.<br>
Figure 12:    Stereo picture of the active site of A. ficuum (ATCC 13073) phytase with a hypothetical binding mode of the substrate phytate. In this model, the bound crystal water molecules were removed and the protein atom positions were held fixed, except for small adaptations of the side chain torsion angles of Lys 68 in order to interact with the substrate. All the conserved amino acid residues Arg 58, His 59, Arg 62, Arg 142, His 338 and Asp 339 form hydrogen bonds to the scissile 3-phosphate group of phytate, as indicated with lines of small dots. His 59 is in a favorable position to make a nucleophilic attack at the scissile phosphorous, indicated with a line of larger dots, and Asp 339 is in a position to protonate the leaving group.<br>
Figure 13:    Construction of the basic plasmids pUC18-AfumgDNA and pUCl8-AfumcDNA for   """ J-—-----J -mutagenesis.<br><br>
Figure 14a:  Primer sets A-N used for site directed mutagenesis.<br>
Figure 14b:   Primer sets O-T used for site directed mutagenesis.<br>
Figure 15:    Construction of plasmids pgDNATl-pgDNAT7.<br>
Figure 16:    Construction of plasmids pgDNANl-pgDNAN6.<br>
Figure 17a: Construction of plasmids pcTl - pcT7.<br>
Figure 17b: Construction of plasmids pcTl-Avrll, pcTl-S66D and pcTl-S140Y-D141G<br>
Figure 17c: Construction of plasmids pcDNA-N27, -T27, -127, -V27, -A27, -G27.<br>
Figure 18:    Construction of plasmids pcNl- pcN6.<br>
Figure 19:    Plasmid pAfum-Tl for the expression of mutein Tl in Aspergillus niger.<br>
Figure 20:    pH optima curves. Specific activity of wild-type and mutant A. fumigatus phytases is plotted against pH of incubation. Open triangles: A. fumigatus [ATCC 13073] wild-type phytase; Open rhombs: A. fumigatus Q27G phytase; Filled squares: A. fumigatus Q27N phytase; Filled triangles: A. fumigatus Q27V phytase; Open squares: A. fumigatus Q27A phytase; Filled circles: A. fumigatus Q27I phytase; Open circles: A. fumigatus Q27T phytase; Dashed line: A. fumigatus Q27L phytase.<br>
Figure 21:    Substrate specificities of wild-type and mutant A. fumigatus [ATCC 13073] phytases. The used substrates 1-13 are the same as mentioned in Figure 3.<br>
The specific activities of the different phytases with any one of the 13 substrates tested are given in the following order (from left to right): A. fumigatus wild-type phytase, A. fumigatus Q27N phytase, A. fumigatus Q27T phytase, A. fumigatus Q27L phytase, A. fumigatus Q27I phytase, A. fumigatus Q27V phytase, A. fumigatus Q27A phytase, A. fumigatus Q27G phytase.<br><br>
Figure 22:    pH optima curves. Specific activity of wild-type and mutant A. fumigatus [ATCC 13073] phytases is plotted against pH of incubation.<br>
Filled rhombs: A. fumigatus wild-type phytase; Filled squares: A. fumigatus Q27L single mutant; Open circles: A. fumigatus Q27L-S66D double mutant; Filled triangles: A. fumigatus Q27L-S140Y-D141G triple mutant.<br>
Figure 23:   Natural variation of phytases in different isolates of A.<br>
fumigatus [ATCC 13073]. The predicted protein sequences are shown and compared to that of the phytase from A. fumigatus strain ATCC 13073. Only the amino acids which differ from those in #13073 are shown.<br>
Figure 24:   pH dependent specific activity of phytases isolated from two<br>
different A. fumigatus wildtype strains. Open squares: wild-type strain ATCC 13073; Filled circles: strain ATCC 32239.<br>
Figure 25: Substrate specificities of phytases isolated from two different A. fumigatus wildtype strains. Black bars: wild-type strain ATCC 13073; White bars: strain ATCC 32239.<br>
Figure 26:   Construction of plasmids pc-S130N, pc-R129L-S130N, pc-K167G-R168Q.<br>
Examples<br>
Example 1<br>
Homology Modeling of A. fumigatus and A. terreus cbs!16.46 phytase<br>
The amino acid sequences of A. fumigatus [ATCC 13073] (see Figure 1) and A. terreus cbsl 16.46 phytase (see Figure 1) were compared with the sequence of A. niger (ficuum) phytase (see Figure 1) for which the three-dimensional structure had been determined by X-ray crystallography. Crystallographic data are given in Figure 8.<br>
A multiple amino acid sequence alignment of A. niger (ficuum) phytase, A. fumigatus phytase and A. terreus cbsll6.46 phytase was calculated with the program "PILEUP" (Prog. Menu for the Wisconsin Package, version 8, September 1994, Genetics Computer Group, 575 Science<br><br>
Drive, Madison Wisconcin, USA 53711). The three-dimensional models of A. fumigatus phytase and A. terreus cbsll6.46 phytase were built by using the structure of A. niger (ficuum) phytase as template and exchanging the amino acids of A. niger (ficuum) phytase according to the sequence alignment to amino acids of A. fumigatus and A. terreus cbsll6.46 phytases, respectively. Model construction and energy optimization were performed by using the program Moloc (Gerber and Miiller, 1995). C-alpha positions were kept fixed except for new insertions/deletions and in loop regions distant from the active site.<br>
Only small differences of the modelled structures to the original crystal structure could be observed in external loops. Furthermore the different substrate molecules that mainly occur on the degradation pathway of phytic acid (tfryo-inositol-hexakisphosphate) by Pseudomonas sp. bacterium phytase and, as far as determined, by A. niger (ficuum) phytase (Cosgrove, 1980; Fig. 1) were constructed and forged into the active site cavity of each phytase structure. Each of these substrates was oriented in a hypothetical binding mode proposed for histidine acid phosphatases (Van Etten, 1982). The scissile phosphate group was oriented towards the catalytically essential His 59 to form the covalent phosphoenzyme intermediate. The oxygen of the substrate phosphoester bond which will be protonated by Asp 339 after cleavage was orientated towards the proton donor. Conformational relaxation of the remaining structural part of the substrates as well as the surrounding active site residues was performed by energy optimization with the program Moloc.<br>
Based on the structure models the residues pointing into the active site cavity were identified. More than half (60%) of these positions were identical between these three phytases, whereas only few positions were not conserved (see Figure 1). This observation could be extended to four additional phytase sequences (A. nidulans, A. terreus 9A1, Talaromyces thermophilus, Myceliophthora thermophila).<br>
The results coming from sequence alignment and structural informa¬tion including favourable enzyme-substrate interactions were combined to define the positions for mutational analysis which are shown in Table 1.<br><br><br><br>
created with primer #40 downstream of the TGA termination codon of the gene.<br>
The PCR fragment (approx. 1450 bp) was subsequently cloned into the Smal site of pUC18 using the "sure clone Kit" (Boehringer Mannheim s.a.) according to the supplier's recommendations. The resulting plasmid was named pUC18-AfumgDNA.<br>
pUC18-AfumcDNA: This plasmid lacks the intron (small gap letters in Figure 6) of the A. fumigatus phytase gene and was constructed as outlined in Figure 13. Briefly, using primers Fum28 and Fumll the 5' end of exon 2 was amplified by PCR (see below), digested with Ncol and EagI (new restriction site introduced with primer Fum28) and ligated together with the linker coding for exon 1 made of primers Fum26 and Fum27 into the Xbal and Ncol sites of pUC18-AfumgDNA, thereby resulting in plasmid pUC18-AfumcDNA.<br>
Fum28:<br>
5' ATATATCGGCCGAGTGTCTGCGGCACCTAGT 3' EagI<br>
Fumll:<br>
5' TGAGGTCATCCGCACCCAGAG 3'<br>
Fum26:<br>
5' CTAGAATTCATGGTGACTCTGACTTTCCTGCTTTCGGCGGCGTATCT GCTTTCC 3'<br>
Fum27:<br>
5'   GGCCGGAAAGCAGATACGCCGCCGAAAGCAGGAAAGTCAGAGTC ACCATGAATT 3'<br>
PCR reaction to get 5' end of exon 2 of the A. fumigatus phytase:<br>
2 ul	template: pUC18-AfumgDNA (20 ng)<br>
1 ul	dNTP's-mix   (Boehringer Mannheim s.a.)<br>
5 ul	lOx Buffer<br>
1 ul	Taq polymerase (Boehringer Mannheim s.a.)<br>
1.9 ul	Fumll (=10 pmol)<br><br><br>
In total 35 cycles with the temperature profile: 95°C for 30 sec/56°C for 30 sec/ 72°C for 45 sec were made. The amplified fragment (approx. 330 bp) was extracted once with an equal volume of phenol/chloroform (1:1). To the recovered aqueous phase 0.1 volume of 3 M sodium acetate, pH 4.8 and 2.5 volumes of ethanol were added. The mixture was centrifuged for 10 min at 12000 g and the pellet resuspended in 20 (J.1 of H2O. Subsequently, the purified fragment was digested with Ncol and EagI and processed as outlined above.<br>
Example 3<br>
Construction of muteins of the phytase of Aspergillus fumigatus for expression in A. niger<br>
To construct all muteins for the expression in A. niger, plasmid pUC18-AfumgDNA was used as template for site-directed mutagenesis. Mutations were introduced using the "quick exchange™ site-directed mutagenesis kit" from Stratagene (La Jolla, CA, USA) following the manufacturer's protocol and using the corresponding primers (Figure 14). All mutations made are summarized in Table 1A and B wherein Tl to T7 and Nl to N6, respectively, refer to the muteins and "Mutation" to the amino acids replaced at such position. For example T5 refers to a mutein with a double mutation: L at position 27 for Q and L at position 274 for Q. The primer sets (A-H) used to introduce the corresponding mutations are shown in Figure 14a. The newly introduced amino acid is shown in bold and the subscript indicates the position in the mature Aspergillus fumigatus enzyme concerning to the numbering of the A. niger amino acid sequence. Figures 15 and 16 outline the scheme for the construction of different plasmids pgTl-pgT7 and pgNl-pgN6 encoding the muteins carrying only one mutation (T1-T4; N1-N3) or more mutations (T5-T7; N4-N6). Clones harboring the desired mutations were identified by DNA sequence analysis as known in the art. The mutated phytases were verified by complete sequencing of the genes.<br><br>
Example 4<br>
Construction of muteins of the phytase of Aspergillus fumigatus for expression in Saccharomyces cerevisiae<br>
Construction of plasmids pcTl - pcT7 (Figure 17a) and pcNl - pcN6 (Figure 18), respectively, encoding the muteins T1-T7 and N1-N6 for the expression in S. cerevisiae was basically done as outlined in Example 3. Instead of using pUC18-AfumgDNA as the basic construct to introduce the mutations, plasmid pUC18-AfumcDNA was used (Figure 13).<br>
The plasmids pcDNA-N27, -G27, -V27, -A27, -127 and -T27 encoding the muteins N27, G27, V27, A27, 127 and T27 were constructed as follows:<br>
A silent restriction site for Avrll was introduced into plasmid pcTl by site directed mutagenesis as described in Example 3 using primer set I (Figure 14a; Figure 17b). The A. fumigatus phytase gene fragment Avrll/Xhol was then replaced by the linker fragment harbouring the desired mutations (Figure 17c). Each linker fragment was generated by annealing of the respective pairs of synthesized polynucleotides (Fig. 14b; sense and antisense strand; 90 ng each) for 3 min at 70 °C in 9 ul distilled water.<br>
Construction of plasmids pcTl-S66D and pcTl-S140Y-D141G encoding the A fumigatus Q27L-S66D double mutant and the A. fumigatus Q27L-S140Y-D141G triple mutant was basically carried out as described in Example 3. Plasmid pcTl, harbouring the mutation coding for Q27L, was used as template for site directed mutagenesis together with the corresponding primer sets J and K (Figure 14a; Figure 17b).<br>
All mutations were verified by DNA sequence analysis of the entire gene.<br>
Example 5<br>
Expression in Aspergillus niger<br>
The genes encoding the aforementioned A. fumigatus wild-type phytase and muteins (Fig. 16) were isolated with BspHI and EcoRV from plasmids pgDNATl~pgDNAT7 and pgDNANl-pgDNAN6 and ligated into the Ncol site downstream of the glucoamylase promoter df Aspergillus niger (glaA) and the EcoRV site upstream of the Aspergillus nidulans tryptophan C terminator<br><br>
{trpC) (Mullaney et al., 1985). The resulting expression plasmids had in addition the orotidine-5'-phosphate decarboxylase gene (pyr4) of Neurospora crassa as selection marker. Figure 19 shows an example for such an expression plasmid carrying the gene encoding mutein Tl (van den Hondel et al., 1991). The basic expression plasmid described above corresponds basically to the pGLAC vector described in example 9 of EP 684 313. Transformation of Aspergillus niger and expression of the muteins was done as described in EP 684 313.<br>
The supernatant was concentrated by way of ultrafiltration in Amicon 8400 cells (PM30 membranes) and ultrafree-15 centrifugal filter devices (Biomax-30K, Millipore).<br>
The concentrate (typically 1.5-5 ml) was desalted in aliquots of 1.5 ml on a Fast Desalting HR 10/10 column (Pharmacia Biotech), with 10 mM sodium acetate, pH 5.0, serving as elution buffer. The desalted A. fumigatus samples were directly loaded onto a 1.7 ml Poros HS/M cation exchange chromatography column (PerSeptive Biosystems, Framingham, MA, USA). A. terreus cbsll6.46 [CBS 220.95] phytase was directly loaded onto a 1.7 ml Poros HQ/M anion exchange chromatography column. In both cases, phytase was eluted in pure form by way of a sodium chloride gradient.<br>
References:<br>
Mullaney, E. J., J. E. Hamer, K. A. Roberti, M. M. Yelton, and W. E. Timberlake. 1985. Primary structure of the trpQ gene from Aspergillus nidulans. Mol. Gen. Genet. 199:37-45.<br>
Van den Hondel, C. A. M. J. J., P. J. Punt, and R. F. M. van Gorcom. 1991. Heterologous gene expression in filamentous fungi. In: More gene manipulations in fungi, pp. 396-428. Bennett, J. W. and Lasure, L. L. (eds.). Academic Press Inc., San Diego, CA.<br>
Example 6<br>
Expression in Saccharomyces cerevisiae<br>
The intron less genes encoding the A. fumigatus wild-type phytase and the different muteins (Fig. 17/18) mentioned above were isolated from the respective plasmids pUC18-AfumcDNA, pcDNATl - pcDNAT7 and pcDNANl - pcDNAN6 with EcoRI and EcoRV and subcloned either between the blunt ended Xhol and the EcoRI sites of plasmid pYES2 (Invitrogen, San<br><br>
Diego, CA, USA) or the shortened GAPFL (glyceraldehyde-3-phosphate dehydrogenase) promoter and the PH05 terminator as described by Janes et al. (1990). Transformation of Saccharomyces cerevisiae strains, e.g. INVScl (Invitrogen, San Diego, CA, USA) was done according to Hinnen et al. (1978). Single colonies harbouring the phytase gene under the control of the GAPFL promoter were picked and cultivated in 5ml selection medium (SD -uracil) (Sherman et al., 1986) at 30° C under vigorous shaking (250 rpm) for 1 day. The preculture was then added to 500 ml YPD medium (Sherman et al., 1986) and cultivated under the same conditions. After four days cell broth was centrifuged (7000 rpm, GS3 rotor, 15 min. 5° C) and the supernatant was collected. Induction of the GAL1 promotor (plasmid pYES2 from Invitrogen, San Diego, CA, USA) was done according to the manufacturers instructions. Purification of the muteins was as described in example 5 (s.a.).<br>
References:<br>
Janes, M., B. Meyhack, W. Zimmermann and A. Hinnen. 1990. The influence of GAP promoter variants on hirudine production, avarage plasmid copy number and cell growth in Saccharomyces cerevisiae. Curr. Genet. 18: 97-103<br>
Hinnen, A., J.B. Hicks and G.R. Fink. 1978. Proc. Natl. Acad. Sci. USA 75: 1929-1933<br>
Sheman, J.P., Finck, G.R. and Hicks, J.B. (1986). Laboratory Course Manual for Methods in Yeast Genetics. Cold Spring Harbor University Press.<br>
Example 7<br>
Determination of phytase activity and substrate specificity<br>
Phytase activity was measured in an assay mixture containing 0.5% phytic acid (~5 mM), 200 mM sodium acetate, pH 5.0. After 15 min incubation at 37°C, the reaction was stopped by addition of an equal volume of 15% trichloroacetic acid. The liberated phosphate ions were quantified by mixing 100 JJ.1 of the assay mixture with 900 |xl H20 and 1 ml of 0.6 M H2S04, 2% ascorbic acid and 0.5% ammonium molybdate. Standard solutions of potassium phosphate were used as reference.<br><br>
In case of pH optimum curves, purified enzymes were diluted in 10 mM sodium acetate, pH 5.0. Incubations were started by mixing aliquots of the diluted protein with an equal volume of 1% phytic acid (-10 mM) in a series of different buffers: 0.4 M glycine/HCl, pH 2.5; 0.4 M acetate/NaOH, pH 3.0, 3.5, 4.0, 4.5, 5.0, 5.5; 0.4 M imidazole/HCl, pH 6.0, 6.5; 0.4 M Tris/HCl, pH 7.0, 7.5, 8.0, 8.5, 9.0. Control experiments showed that pH was only slightly affected by the mixing step. Incubations were performed for 15 min at 37°C as described above.<br>
For determination of the substrate specificities of wild-type and mutant A. fumigatus phytases, phytic acid in the assay mixture was replaced by 5 mM-concentrations of the respective phosphate compounds. The activity tests were performed as described above.<br>
Protein concentrations were calculated from the OD at 280 nm, using theoretical absorption values calculated from the known protein sequences with the DNA* software (DNASTAE, Inc., Madison, Wisconsin, USA). An absorption of 1.0 OD at 280 nm corresponds to 0.94 mg/ml A. fumigatus phytase and 0.85 mg/ml of A. terreus cbsll6.46 phytase.<br>
pH profiles of Aspergillus fumigatus mutants Tl (Q27L), T5 (Q27L, Q274L) and T6 (Q27L, Q274L, G277D) have drastically changed compared to the wild-type A. fumigatus phytase (see Figure 2). All mutants showed equal pH profiles. Increase in specific activity at pH 5.0 of the muteins as compared to the wild-type phytase of Aspergillus fumigatus is shown in Table 2. Enzyme activities were measured under standard assay conditions at pH 5.0. Several individual measurements (n: number of assays) were averaged.<br>
The pH profile of A. fumigatus phytase mutant Q27A resembles the pH profile of A. fumigatus wild-type phytase over nearly the whole pH range (Figure 20). Whereas the specific activity of wild-type phytase is decreasing at pH values below pH 4.0, the specific activity of the phytase mutant Q27A remains nearly constant down to pH 2.9.<br>
The single amino acid exchanges Q27L, Q27I, Q27V or Q27T have remarkably increased the specific activity over the whole pH range, especially between pH 5.0 and 7.5 (Figure 20). Maximum values are reached at pH 6.5. In addition, mutation Q27T caused the highest specific activity values for phytic acid at low pH (pH 3.0-5.0).<br><br>
Higher specific activities are also gained by the single mutations Q27G or Q27N, between pH 2.5 and 7.0, with maximum values at pH 6.0 (Figure 20). The specific activity decreases at pH values below 3.5.<br>
All single mutants still show a broad substrate specificity which is comparable to that of A. fumigatus wild-type phytase (Figure 21). Some of the mutants show significantly higher specific activities than other mutants for selected substrates, e. g., the Q27T mutant for p-nitrophenyl phosphate and ATP, or the Q27G mutant for phosphoenolpyruvate.<br>
As shown in Figure 22 the combination of mutation Q27L with S66D or S140Y and D141G led to a shift of the pH profile towards lower pH. The maximum specific activity gained by the single mutation Q27L is further increased by the additional amino acid exchanges.<br>
As shown in Figure 3, Aspergillus fumigatus phytase mutant Tl (Q27L) showed no difference in substrate specificity compared to the triple mutant T6 (Q27L, Q274L, G277D).<br>
The pH profiles of the muteins Nl-6, except N2 show significant differences compared to the wild-type phytase (Fig. 10). Whereas the pH profile of mutein N4 is expanded towards lower pH, the profiles of muteins N3 to N6 are shifted towards lower pH. The muteins N5, N6 reach maximum activity already at pH 3.0.<br>
The muteins Nl to N6 show in almost all cases a drastic reduction in specific activity for all tested substrates, except for phytic acid (Fig. 9). Specific activity for phytic acid remained unchanged compared to the wild-type phytase, whereas mutant N3 and N6 show a tendential higher activity (Fig. 19).<br><br>
t<br>
r <br>
Example 8<br>
As an alternative approach to obtain phytases with modified characteristics and to get a better idea about the natural variation found in phytase characteristics within a certain species, naturally occurring variants of A. fumigatus phytase were analysed. Phytase genes were obtained from six different isolates of A. fumigatus. The amino acid sequence of phytase from two of the A. fumigatus isolates (ATCC 26934 and ATCC 34625) showed no difference to the original amino acid sequence of<br><br>
wild-type A. fumigatus phytase ATCC 13073. Phytase from three other isolates had one or two amino acid substitutions, none of which directly affected the active site. Enzymatic characteristics remained unaffected by these substitutions (not shown). The phytase of isolate of A. fumigatus (ATCC 32239) differed in 13 positions in the signal sequence and 51 positions in the mature part of the protein compared to the original wild-type A. fumigatus phytase (ATCC 13073). Several of these substitutions affect variable amino acids of the active site cavity. This resulted in an increase in specific activity with phytic acid as substrate (47 U/mg, standard enzyme assay) and in loss of enzymatic activity above pH 7 (Fig. 24). Also in this case, the specific activity against phytic acid was increased relative to the specific activities with other substrates (Fig. 25).<br>
Example 9<br>
Construction of plasmids pc-S130N, pc-R129L-S130N, pc-K167G-R168Q encoding A. fumigatus [ATCC 13073] phytase S130N single mutant and R129L-S130N double mutant and A. nidulans phytase K167G-R168Q double mutant was basically carried out as described in Example 3. Plasmid pUC18-AfumcDNA was used as template for site directed mutagenesis together with the corresponding primer sets L, M and N (Figure 14a; Figure 26).<br>
All mutations were verified by DNA sequence analysis of the entire gene.<br>
Example 10<br>
When expressed in A. niger and stored as concentrated culture supernatants at 4°C, the phytases from A. fumigatus, A. nidulans displayed tendency to undergo proteolytic degradation. N-terminal sequencing of fragments suggested that cleavage occured between amino acids S130-V131 and K167-R168 or R168-A169, respectively. Compared with 3D structure of A. niger phytase revealed that all cleavage sites are found within surface-exposed loop structures and are therefore accessible to proteases.<br>
Site-directed mutagenesis at protease-sensitive sites of A. fumigatus phytase (S130N, R129L-S130N) and A. nidulans phytase (K167G-R168Q) yielded mutant proteins with considerably reduced susceptibility to proteolysis.<br><br>
In contrast to expression in A. niger, proteolytic degradation was not observed when the phytases were expressed in Hansenula polymorpha.<br><br><br>
WE CLAIM :<br>
1.	A modified phytase with a specific activity improved over the specific activity of the corresponding unmodified phytase characterized in that the amino acid sequence of the unmodified phytase has been changed at the position corresponding to position 27 of the phytase of Aspergillus niger to an amino acid selected from the group consisting of Ala, Val, Leu, He, Thr, Asn and Gly, with the provisio that if the unmodified phytase is a phytase of Aspergillus terreus, position 27 of the amino acid sequence is changed to an amino acid selected from the group consisting of Ala, Val, He, Thr, Asn and Gly.<br>
2.	The phytase as claimed in claim 1 wherein the phytase is the phytase of Aspergillus.<br>
3.	The phytase as claimed in claim 2 wherein the phytase is the phytase of Aspergillus fumigatus.<br>
4.	The phytase as claimed in any of claims 1 to 3, wherein in addition to position 27 a substitution occurs also at position 66.<br>
5.	The phytase as claimed in any of claims 1 to 3 wherein in addition to position 27 a substitution occurs also at position 140.<br>
6.	The phytase as claimed in any one of claims 1 to 5 which is characterized by at least one of the following mutations: Q27L, Q27N, Q27T, Q27I, Q27V, Q27A, Q27G, S66D, S140Y, D141G, A205E, Q274L, G277D, G277K, Y282H, and/or<br>
N340S.<br>
7.	A DNA sequence comprising a DNA sequence coding for a phytase as claimed in any one of claims 1 to 6.<br>
8.	A vector comprising the DNA sequence as claimed in claim 7.<br>
9.	The vector as claimed in claim 8 which is an expression vector.<br>
10.	A microbial host cell which has been transformed by a DNA sequence as claimed in claim 7 or the vector as claimed in claim 8 or 9.<br><br>
11.	A process for the preparation of a phytase as claimed in any one of claims 1 to 6<br>
wherein the host cell as claimed in claim 10 is cultured under suitable culture<br>
conditions and the phytase is isolated from such host cells or the cultured medium<br>
by methods known in the art.<br>
12.	A food or feed composition comprising a phytase as claimed in any one of claims<br>
1 to 6.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LW1hcy05OCBhYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">558-mas-98 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LW1hcy05OCBjbGFpbXMgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">558-mas-98 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LW1hcy05OCBjbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">558-mas-98 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LW1hcy05OCBjb3JyZXNwb25kZW5jZSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">558-mas-98 correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LW1hcy05OCBjb3JyZXNwb25kZW5jZSBwby5wZGY=" target="_blank" style="word-wrap:break-word;">558-mas-98 correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LW1hcy05OCBkZXNjcmlwdGlvbiAoY29tcGxldGUpIGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">558-mas-98 description (complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LW1hcy05OCBkZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">558-mas-98 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LW1hcy05OCBkcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">558-mas-98 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LW1hcy05OCBmb3JtLTE5LnBkZg==" target="_blank" style="word-wrap:break-word;">558-mas-98 form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LW1hcy05OCBmb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">558-mas-98 form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LW1hcy05OCBmb3JtLTI2LnBkZg==" target="_blank" style="word-wrap:break-word;">558-mas-98 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LW1hcy05OCBmb3JtLTQucGRm" target="_blank" style="word-wrap:break-word;">558-mas-98 form-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LW1hcy05OCBmb3JtLTYucGRm" target="_blank" style="word-wrap:break-word;">558-mas-98 form-6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LW1hcy05OCBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">558-mas-98 others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU4LW1hcy05OCBwZXRpdGlvbi5wZGY=" target="_blank" style="word-wrap:break-word;">558-mas-98 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="224267-a-method-for-allocating-power-for-power-control-bits-in-a-cellular-network.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="224269-high-sensitivity-resist-compositions-for-electron-based-lithography.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>224268</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>558/MAS/1998</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>47/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Nov-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>10-Oct-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>17-Mar-1998</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>ROCHE VITAMINS LTD.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>124, CH-4070 BASEL, SWITZERLAND</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>DIRK KOSTREWA</td>
											<td>1 PETER-THUMB-STRASSE, D-79100 FREIBURG,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>LUIS PASAMONTES</td>
											<td>197 BASLERSTRASSE, CH-4632 TRIMBACH,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>ANDREA TOMSCHY</td>
											<td>47 KRAFTWERKSTRASSE, D-79639 GRENZACH-WYHLEN,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>ADOLPHUS VAN LOON</td>
											<td>17 WALDSHUTERSTRASSE, CH-4310 RHEINFELDEN,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>KURT VOGEL</td>
											<td>41 EULERSTRASSE, CH-4051 BASEL,</td>
										</tr>
										<tr>
											<td>6</td>
											<td>MARKUS WYSS</td>
											<td>9, ROTACKERSTRASSE, CH-4410 LIESTAL,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/155</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>97810175.6</td>
									<td>1997-03-25</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/224268-a-modified-phytase-and-a-process-for-the-preparation-of-the-same by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:00:18 GMT -->
</html>
