<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/224299-an-aspergillus-host-cell-for-the-expression-of-a-heterologous-protein-product-and-a-method-for-producing-the-protein by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:53:05 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 224299:AN ASPERGILLUS HOST CELL FOR THE EXPRESSION OF A HETEROLOGOUS PROTEIN PRODUCT AND A METHOD FOR PRODUCING THE PROTEIN</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">AN ASPERGILLUS HOST CELL FOR THE EXPRESSION OF A HETEROLOGOUS PROTEIN PRODUCT AND A METHOD FOR PRODUCING THE PROTEIN</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>ABSTRACT 537/MAS/96 &quot;An aspergillus host cell for the expression of a heterologous protein product and a method for producing the protein&quot; An Aspefgillus host cell for the expression of a heterologous protein product, which cell has been genetically modified in order to express significantiy reduced levels of an Aspergillus neutrul metalloprolease. Npll having optimal proleolylic activity in the range of pH &amp;#946;-8. as compsred to a parental cell.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br><br>
TECHNICAL FIELD<br>
The present invention relates to novel host cells and to methods of producing proteins. More specifically the invention relates to a host cell useful for the expression of heterologous proteins, which host cell has been genetically modified n order to express significantly reduced levels of a metalloprotease. Moreover the nvention relates to a method of producing a heterologous protein, which method comprises cultivating the host cell in a suitable growth medium, followed by recovery of the desired protein.<br>
BACKGROUND ART<br>
The use of recombinant host cells in the expression of heterologous proteins has in recent years greatly simplified the production of large quantities of commercially valuable proteins, which otherwise are obtainable only by purification rom their native sources. Currently, there is a varied selection of expression systems rom which to choose for the production of any given protein, including eubacterial and eucaryotic hosts. The selection of an appropriate expression system often not only depends on the ability of the host cell to produce adequate yields of the protein n an active state, but also to a large extent may be governed by the intended end use of the protein.<br>
One problem frequently encountered is the high level of proteolytic enzymes produced by a given host cell or in the culture medium. It has been suggested that one could provide host organism deprived of the ability of producing specific proteolytic compounds. For example. International Patent Application WO 90/00192 describes filamentous fungal hosts Incapable of excreting enzymatically active aspartic proteinase, and EP 574 347 describes Aspergillus hosts defective in a serine protease of the subtilisin-type.<br><br>
Metalloproteases have been isolated from a number of eucaryotic sources. Neutral metalloproteases, i.e. metalloproteases having optimal activity at neutral pH, isolated from strains of Aspergillus also have been reported. Neutral metalloproteases have been classified into two groups, Npl and Npll [Sekine; Aaric. Biol. Chem. 1972 36 207-216]. Recently the nucleotide sequence of a neutral metalloprotease II cDNA from Aspergillus oryzae have been disclosed [Tatsumi H, Murakami S, Tsuji R F, Ishida Y, Murakami K, Masaki A, Kawabe H, Arimura H, Nakano E and Motai l-{; Mol. Gen. Genet. 1991 228 97-103]. The nucleotide sequence of a neutral metalloprotease I cDNA from Aspergillus oryzae have never been disclosed.<br>
Although metalloproteases have been reported, their role in relation to reducing the stability of the products obtained from these organisms have never been described.<br>
SUMMARY OF THE INVENTION<br>
According to the present invention it has now been found that metalloproteases may reduce significantly the stability of the product obtained by a cell.<br>
Accordingly, the present invention provides a host cell useful for the expression of a heterologous protein product, which cell has been genetically modified in order to express significantly reduced levels of a metalloprotease, as compared to the parental cell.<br>
In another aspect, the invention provides a method of producing a heterologous protein product in a host cell of the invention, which method comprises introducing into the host cell a nucleic acid sequence encoding the protein, cultivating the host cell in a suitable growth medium, and isolating the heterologous protein product.<br>
By the method of the invention, the proteolytic action arising from metalloproteases have been significantly reduced, thereby improving the stability of the protein obtained by the method. Moreover, the protein obtained by the method<br><br>
of the invention can be obtained as a precursor protein, i.e. a zymogen, a hybrid protein, a protein obtained as a pro sequence or pre-pro sequence, or in un-maturated form.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
The present invention is further illustrated by reference to the accompanying drawing, in which:<br>
Fig. 1 shows a map of plasmid pS02, cf. Example 2;<br>
Fig. 2 shows the construction of Aspergillus oryzae strain HowB101, cf. Example 2;<br>
Fig. 3 shows the construction of plasmid pJaL335, cf. Example 2;<br>
Fig. 4 shows the construction of plasmid pJaL399, cf. Example 2;<br>
Fig. 5 shows the construction of plasmid pJaL218, cf. Example 4; and<br>
Fig. 6 shows a map of plasmid pToC56, cf. Example 5.<br>
DETAILED DISCLOSURE OF THE INVENTION<br>
Host Cells<br>
The present invention provides a host cell useful for the expression of heterologous proteins, which cell, when compared to the parental cell, has been genetically modified in order to express significantly reduced levels of a metalloÂ¬protease.<br>
The parental cell is the source of said host cell. It may be a wild-type cell. Alternatively, besides a decrease in metalloprotease level, it may be genetically altered in another respect.<br>
In order to produce the desired protein, the host cell of the invention obviously must hold structural (i.e. regions comprising the coding nucleotide sequences) and regulatory (i.e. regions comprising nucleotide sequences necessary for e.g. transcription, translation and termination) genetic regions necessary for the<br><br>
expression of the desired product. The nature of such structural and regulatory regions greatly depends on the product and the host cell in question. The genetic design of the host cell of the invention may be accomplished by the person skilled in the art, using standard recombinant DNA technology for the transformation or transfection of a host cell [vide e.g. Sambrook etal.; Molecular Cloning. Cold Spring Harbor, NY, 1989].<br>
Preferably, the host cell is modified by methods known in the art for introduction of an appropriate cloning vehicle, i.e. a plasmid or a vector, comprising a DNA fragment encoding the desired product. The cloning vehicle may be introduced into the host cell either as an autonomously replicating plasmid or integrated into the chromosome. Preferably the cloning vehicle comprises one or more structural regions operably linked to one or more appropriate regulatory regions.<br>
The structural regions are regions holding nucleotide sequences encoding the desired product. The regulatory regions include promoter regions comprising transcription and translation control sequences, terminator regions comprising stop signals, and polyadenylation regions. The promoter, i.e. a nucleotide sequence exhibiting a transcriptional activity in the host cell of choice, may be one derived from a gene encoding an extracellular or an intracellular protein, preferably an enzyme, such as an amylase, a glucoamylase, a protease, a lipase, a cellulase, a xylanase, a oxidoreductase, a pectinase, a cutinase, or a glycolytic enzyme. Examples of suitable promoters for transcription in a fungal host cell are promoters derived from the gene encoding Aspergillus oryzae TAKA amylase, Aspergillus niger neutral a-amylase, Aspergillus niger acid stable a-amylase, Aspergillus niger or Aspergillus awamsii glucoamylase (g\uA), Aspergillus niger aceXamidase, Aspergillus oryzae alkaline protease, y\sperg/7/L/s oryzae triose phosphatase isomerase, Rhizopus meihei aspartic proteinase, and Rhizopus meihei lipase. Preferred are the Aspergillus oryzae TAKA-amylase and Aspergillus awamsii gluA promoters.<br>
The cloning vehicle may also comprise a selectable marker, e.g. a gene, the product of which complements a defect in the host cell, or one which conÂ¬fers antibiotic resistance, such as ampicillin, kanamycin, chloramphenicol or tetracycline resistance. Examples of Aspergillus selection markers include amdS,<br><br>
pyrG, argB, niaD and sC, a marker giving rise to hygromycin resistance. Preferred for use In an Aspergillus host cell are the amdS and pyrG markers of Aspergillus nidulans or Aspergillus oryzae. A frequently used mammalian marker is the dihydrofolate reductase (DHFR) gene. Furthermore, selection may be accomplished by co-transformation.<br>
The procedures used to ligate the DNA construct of the invention, the promoter, terminator and other elements, respectively, and to insert them into suitable cloning vehicles containing the information necessary for replication, are well known to persons skilled in the art [vide e.g. Sambrook et al.; Molecular Cloning, Cold Spring Harbor, NY, 1989].<br>
The host cell of the invention may be any host cell conventionally used for heterologous expression of proteins.<br>
Preferably, the host cell of the invention is a yeast or a filamentous fungus capable of producing a desired protein. In particular, the yeast cell may be a strain of Saccharomyces. preferably Saccharomyces cerevisiae. In particular, the filamentous fungus may be a strain selected from the group consisting of Acremonium, Aspergillus, Candida, Cocliobolus, Endothia, Fusarium, Humicola, Neurospora, Rhizomucor, Rhizopus, Thermomyces, Trichoderma, Podospora, Pyricularia, or Penicillium.<br>
In a preferred embodiment, the filamentous fungus is a strain selected from the group consisting of Aspergillus oryzae, Aspergillus niger, Aspergillus nidulans, Aspergillus awamori, Aspergillus phoenicis, Aspergillus japonicus, Aspergillus foetus, Fusarium graminearum, Fusarium oxysporum, Fusarium solani, Humicola grisea, Neurospora crassa, Penicillium chrysogenum, Rhizomucor meihei, Trichoderma reesel, or Trichoderma viride.<br>
Products<br>
The desired end product, i.e. the heterologous protein expressed by the host cell of the invention, may be any eubacterial or eucaryotic protein.<br>
As defined herein, a "heterologous protein product" Is a protein which<br>
I is not native to the host cell, or a native protein in which modifications have been<br>
made to alter the native sequence, or a native protein whose expression is<br><br>
quantitatively altered as a result of a manipulation of a native regulatory sequence required for the expression of the native protein, such as a promoter, a ribosome binding site, etc., or other manipulation of the host cell by recombinant DNA techniques.<br>
Owing to the absence of metalloprotease, the heterologous protein expressed by the host cell may also be a precursor protein, i.e. a zymogen, a hybrid protein, a protein obtained as a pro sequence or pre-pro sequence, or in un-maturated form. In a preferred embodiment the product is an enzyme.<br>
In a more specific embodiment, the product is an eucaryotic enzyme, such as insulin, growth hormone, glucagon, somatostatin, interferon, PDGF, factor VII, factor VIII, urokinase, EPO, chymosin, tissue plasminogen activator, or serum albumin.<br>
In another preferred embodiment, the product is an enzyme of fungal,<br>
of yeast, or of bacterial origin.<br>
;	Preferably the enzyme is a glycosidase enzyme, e.g. an amylase, in<br>
particular an a-amylase (EC 3.2.1.1), a -amylase (EC 3.2.1.2), a glucan 1,4-a-<br>
glucosidase (EC 3.2.1.3), a cellulase (EC 3.2.1.4), an endo-1,3(4)-;8-glucanase (EC<br>
3.2.1.6), an endo-1,4-i3-glucanase (EC 3.2.1.8), a polygalacturonase (EC 3.2.1.15),<br>
an a-glucosidase (EC 3.2.1.20), a i3-glucosidase (EC 3.2.1.21), an a-galactosidase<br>
) (EC 3.2.1.22), a )8-galactosidase (EC 3.2.1.23), a xylan-endo-1,3-)3-xylosidase (EC<br>
3.2.1.32), an endo-1,3--glucanase (EC 3.2.1.39), an endo-1,3-a-glucanase (EC<br>
3.2.1.59), an endo-1,2--glucanase (EC 3.2.1.71), an endo-1,6-6-glucanase (EC<br>
3.2.1.75), a cellulose-1,4-)8-cellobiosidase (EC 3.2.1.91, also known as<br>
cellobiohy drolases).<br>
j	In another preferred embodiment the enzyme is a lipolytic enzyme, in<br>
particular a lipase, an esterase, a phospholipase, or a lyso-phospholipase.<br>
In a third preferred embodiment the enzyme is a phytase, in particular a 3-phytase (EC 3.1.3.8) or a 6-phytase (EC 3.1.3.26).<br>
In a fourth preferred embodiment the enzyme is a proteolytic enzyme.<br>
âo	In a fifth preferred embodiment the enzyme is an oxidoreductase, such<br>
as a peroxidase or a laccase, a pectinase, or a cutinase.<br><br>
Preferred hybrid polypeptides are prochymosin and pro-trypsin-like proteases.<br>
Metalloproteases<br>
In the context of this invention a metalloprotease is a proteolytic enzyme containing a catalytic zinc metal center which participates in the hydrolysis of the peptide backbone. The active zinc center differentiates these proteases from calpains, whose activities are dependent upon the presence of calcium. Confirmation of a protease as a metalloprotease is loss of proteolytic activity accomplished by removal of the zinc center. The zinc center can be removed with 1,10-phenanthroline (1 mM). After titration with Zn* (0.1-100 /iM), proteolytic activity is restored.<br>
In a preferred embodiment, the metalloprotease contemplated in the context of this invention is a Fusarium metalloprotease, preferably a Fusarium oxysporum metalloprotease. In a most preferred embodiment, the metalloprotease is a Fusarium oxysporum p45 metalloprotease having the amino acid sequence presented as SEQ ID NO: 2, or a sequence homologous hereto.<br>
In another preferred embodiment, the metalloprotease contemplated in the context of this invention is a neutral metalloprotease, which is a metalloÂ¬protease possessing optimal proteolytic activity in the neutral pH region, i.e. in the range of about pH 6-8, preferably the range of about pH 6.5-7.5, around pH 7.<br>
More particularly, the metalloprotease contemplated in the context of this invention is a neutral Aspergillus metalloprotease of group Npl or Npll.<br>
In a preferred embodiment, the metalloprotease is an Aspergillus oryzae Neutral Metalloprotease I (Npl) encoded by a cDNA comprising the partial nucleotide sequence presented as SEQ ID NO: 4, or a sequence homologous hereto.<br>
The degree of homology may be determined as the degree of identity between the two sequences indicating a derivation of the first sequence from the second. The homology may suitably be determined by means of computer programs, by methods known in the art, e.g. by comparing 50 bp continuous sequences. As defined herein, the protein encoded by a homologous cDNA sequence exhibits a degree of homology of at least 70% homology, preferably more<br><br>
than 80% homology, more preferred more than 90% homology, most preferred more than 95% homology, with the sequence in question.<br>
The gene encoding the metalloprotease may be identified by screening by hybridization for nucleic acid sequences coding for all of, or part of, the metalloprotease, e.g. by using synthetic oligonucleotide probes, that may be prepared on the basis of a cDNA sequence, e.g. the nucleotide sequences presented as SEQ ID NO: 1 and SEQ ID NO: 4, or on the basis of the amino acid sequence of the metalloprotease, in accordance with standard techniques [vide e.g. Sambrook et al.\ Molecular Cloning. Cold Spring Harbor, NY, 1989].<br>
Genetic Modifications<br>
The host cell of the invention, genetically modified in order to express significantly reduced levels of a metalloprotease, may be modified using standard recombinant DNA technology, known to the person skilled in the art. The gene sequence responsible for the production of metalloprotease may be inactivated or eliminated entirely.<br>
In a particular embodiment, the host cell of the invention is one genetically modified at the structural or regulatory regions encoding the metalloÂ¬protease. Known and useful techniques include, but are not limited to, specific or random mutagenesis, PCR generated mutagenesis, site specific DNA deletion, insertion and/or substitution, gene disruption or gene replacement techniques, anti-sense techniques, or a combination thereof.<br>
Mutagenesis may be performed using a suitable physical or chemical mutagenizing agent. Examples of a physical or chemical mutagenizing agent suitable for the present purpose includes ultraviolet (UV) irradiation, hydroxylamine, N-methyl-N'-nitro-N-nitrosoguanidine (MNNG), 0-methyl hydroxylamine, nitrous acid, ethyl methane sulphonate (EMS), sodium bisulfite, formic acid, and nucleotide analogues. When such agents are used, the mutagenesis is typically performed by incubating the cell to be mutagenized in the presence of the mutagenizing agent of choice under suitable conditions for the mutagenesis to take place, and selecting for mutated cells having a significantly reduced production of metalloprotease.<br><br>
Modification may also be accomplished by introduction, substitution or removal of one or more nucleotides in the metalloprotease encoding sequence or a regulatory element required for the transcription or translation thereof. Nucleotides may, e.g., be inserted or removed so as to result in the introduction of a stop codon, the removal of the start codon or a change of the open reading frame. The modification or inactivation of the structural sequence or a regulatory element may be accomplished by site-directed mutagenesis or PCR generated mutagenesis in accordance with methods known in the art. Although in principle, the modification may be performed in vivo, i.e. directly on the cell carrying the metalloprotease gene, it is presently preferred to conduct the modification in vitro.<br>
A convenient way to inactivate or reduce the metalloprotease production of a host cell of choice is based on the principles of gene interruption. This method involves the use of a DNA sequence corresponding to the endogenous gene or gene fragment which it is desired to destroy. Said DNA sequence is in vitro mutated to a defective gene and transformed into the host cell. By homologous recombination, the defective gene replaces the endogenous gene or gene fragment. It may be desirable that the defective gene or gene fragment encodes a marker which may be used for selection of transformants in which gene encoding the metalloprotease has been modified or destroyed.<br>
Alternatively, the modification or inactivation of the DNA sequence may be performed by use of established anti-sense techniques using a nucleotide sequence complementary to the metalloprotease encoding sequence, e.g. the nucleotide sequences presented as SEQ ID NO: 1 and SEQ ID NO: 4.<br>
Owing to genetic modification, the host cell of the invention expresses significantly reduced levels of metalloproteases. In a preferred embodiment the level of metalloprotease expressed by the host cell is reduced more than about 50%, preferably more than about 85%, more preferred more than about 90%, most preferred more than about 95%. In a most preferred embodiment, the product expressed by the host cell is essentially free of any metalloprotease activity.<br><br>
Methods of Producing Proteins<br>
In another aspect, the invention provides a method of producing proteins (i.e. polypeptides and/or proteins), which method comprises cultivating the host cell of the invention in a suitable growth medium, followed by recovery of the desired product.<br>
By the method of the invention, the proteolytic action of metallo-proteases have been significantly reduced, thereby improving the stability of the product obtained. Moreover, owing to the absence of metalloprotease, the heterologous protein expressed by the host cell may be obtained as a precursor protein, i.e. a zymogen, a hybrid protein, a protein obtained as a pro sequence or pre-pro sequence, or in unmaturated form.<br>
The broth or medium used for culturing may be any conventional medium suitable for growing the host cell in question, and may be composed according to the principles of the prior art. The medium preferably contain carbon and nitrogen sources and other inorganic salts. Suitable media, e.g. minimal or complex media, are available from commercial suppliers, or may be prepared according to published receipts, e.g. the American Type Culture Collection (ATCC) Catalogue of strains.<br>
After cultivation, the protein is recovered by conventional method for isolation and purification proteins from a culture broth. Well-known purification procedures include separating the cells from the medium by centrifugation or filtraÂ¬tion, precipitating proteinaceous components of the medium by means of a salt such as ammonium sulphate, and chromatographic methods such as e.g. ion exchange chromatography, gel filtration chromatography, affinity chromatography, etc.<br>
EXAMPLES<br>
The invention is further illustrated with reference to the following examples which are not intended to be in any way limiting to the scope of the invention as claimed.<br><br>
Materials and Methods<br>
Strains	<br>
Aspergillus oryzae IFO 77, available from Institute for Fermentation, Osaka, 17-25 Juso Hammachi 2-Chome Yodogawa-Ku, Osaka, Japan.<br>
Fusarium oxysporum DSM 2672, deposited according to the Budapest Treaty on the International Recognition of the Deposit of Microorganisms for the Purposes of Patent Procedure at Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH (DSM), Mascheroder Weg 1 b, DE-3300 Braunschweig, Germany, on 6 June 1983.<br>
Escherichia coli DH5a, Hanahan D, J. Mol. Biol. 1983 166 557.<br>
Genes<br>
Npl, which gene encodes Neutral Metalloprotease I. Npll, which gene encodes Neutral Metalloprotease II. pyrG: which gene encodes orotidine-5'-phosphate decarboxylase, an enzyme involved in the biosynthesis of uridine.<br>
Plasmids<br>
pUC118; Yanish-Perron et a!., 1985 Gene 33 103.<br>
pJaL389; Construction of this plasmid from cosmid 3E8 is described in Example 1.<br>
pS02; Construction of this plasmid is described in Example 2.<br>
pJers4; A subclone of pS02.<br>
pJaL335; Construction of this plasmid from pS02 is described in Example 2.<br>
pS05; Construction of this plasmid from pS02 is described in Example 2.<br>
pJaL198; Construction of this plasmid from pJaL198 is described in Example 3.<br>
pjal_218; Construction of this plasmid from pJaL218 is described in Example 4.<br><br>
p3SR2; Kelly J M and Hynes M J, EMBO Journal 1985 4 475-479. pToC90; A subclone of p3SR2.<br>
pToC56; Construction of this plasmid is described in EP 238,023 B. pToC65; Construction of this plasmid is described in EP 531 372 B. pCR"!!; Available from Invitrogen Corporation, San Diego, CA, USA.<br>
EXAMPLE 1<br>
Gioning of Aspergillus oryzae Neutral Metalloprotease I (Npl)<br>
Construction of a Cosmid Library of Aspergillus oryzae<br>
The library was essentially constructed according to the instruction from the supplier (Stratagene) of the "SuperCosI Cosmid Vector Kit".<br>
Genomic DNA of Aspergillus oryzae IFO 4177 was prepared from protoplasts made by standard procedures [cf. e.g. Christensen etal.. Biotechnology 1989 6 1419-1422]. After isolation of the protoplasts these were pelleted by centrifugation at 2500 rpm for 5 minutes in a Labofugeâ¢ T (Heto), the pellet was suspended in 10 mM NaCI, 20 mM Tris-HCI (pH 8.0), 1 mM EDTA, 100 MQ/mi Proteinaseâ¢ K and 0.5% SDS, as described in the manual from the Supercos 1 Cosmid Vector Kit, as was the rest of the DNA preparations.<br>
The size of the genomic DNA was analyzed by electrophoresis using the CHEF-gel apparatus from Biorad. A 1 % agarose gel was run for 20 hours at 200 volt with a 10-50 second pulse. The gel was stained by ethidium bromide and photographed. The DNA was 50-100 kb in size. The DNA was partially restricted by Sau3A. The size of the restricted DNA was 20-50 kb determined the same way.<br>
The CsCI gradient banded SuperCosI vector was prepared according to the supplier's manual, as was ligation and packaging. After titration of the library, all of the packaging mix from one ligation and packaging was transfected into the host cells XLI-Blue MR and plated on 50 /zg/ml ampicillin LB plates. Approx. 3800 colonies were obtained. Cosmid preparation from 10 colonies showed that they all had inserts of expected size. The colonies were picked individually and inoculated in microtiter plate wells with 100 1 LB (100 /xg/ml ampicillin) and incubated at 37Â°C Dver night. 100 ii\ of 50% glycerol was added to each well, and the whole library was<br><br>
frozen at -80Â°C. A total of 3822 colonies were stored. This represents the Aspergillus oryzae genome approx. 4.4 times.<br>
Cloning Fusarium oxysporum p45 Metalloprotease Gene Purification<br>
Fusarium oxysporum DSM 2672 broth is centrifuged at 9000 rpm for 10 minutes and the supernatant is filtered through a 0.45 nn filter. 200 ml of filtrate is concentrated down to 10 ml on an Amicon cell (PM 10 membrane) and Centriprep-0 (Amicon). 5 ml of concentrate is diluted to 100 ml and pH adjusted to 5 with acetic acid and run on a 1 ml Mono-S column in the following buffer: 0.1 IVI borate, 10 mM DMG, 2 mM calcium chloride, pH 5.2, in a gradient of 0 to 0.5 M sodium chloride over 70 minutes. After 10 minutes of wash in the above identified buffer at a flow rate of 1 ml/minute, 1.5 ml fractions are collected and concentrated on Centricon-10 (Amicon).<br>
Gel filtration using Superose-12 (HR 10/30, Pharmacia) is performed in 0.1 M borate, 10 mM DMG, 2 mM CaCI, pH 6.5, flow rate 0.4 ml/minute. 0.4 ml fractions are collected. 200 ii\ samples are injected.<br>
Proteolytic Enzyme Assay<br>
Metalloprotease activity is measured as released trypsin activity from the pro-trypsin-like protease from the strain Fusarium oxysporum DSM 2672, after a 30-60 minutes pre-incubation at 25Â°C in 0.1 M TRIS, 2 mM CaCI, pH 7 (at lower pH, 100 mM borate, 10 mM DMG, 2 mM CaCIa is used). The tryptic activity is measured in microtiter plates, 100 /zl samples are mixed with 100 /zl of substrate (stock: 87 mg/ml L-BAPNA (Sigma) in DMSO, diluted 50-fold in buffer), and the absorption at 405 nm is measured using a Thermomax reader from Molecular Devices.<br>
SDS-PAGE and Electro Blotting onto PVDF<br>
SDS-PAGE (10-27%, Novex) is run according to the manufacturer's instructions. Samples to be run are pre-incubated with PMSF before adding sample buffer. Electro blotting onto pro-blot membranes (Applied Biosystems) is performed<br><br>
in 3 mM NaCOa. 10 mM NaHCOa, 20% MeOH, pH 9.9, at 30 V for 2 hours using the blotting module from Novex. The pro-blot is stained as described by Applied Biosystems.<br>
lEF-overlay<br>
Isoelectric focusing (lEF) is run on an Amphollne PAG-plate (PharmaÂ¬cia), pH 3.5 to 9.5, and stained according to the manufacturer's instructions. The gel to be overlaid is first equilibrated for 15 minutes in 0.1 M TRIS, 2 mM CaClg, pH 8.1, and then overlaid with 10 ml 1% agarose, 0.1 M TRIS, 2 mM CaCI, pH 8.1, added 300 ii\ L-BAPNA (Sigma) stock and 500 /il pro-trypsin-like Fusarium oxysporum DSM 2672 protease (~ 0.25 mg/ml).<br>
Amino Acid Analysis and Amino Acid Sequencing<br>
Microwave facilitated vapor phase hydrolysis of lyophilized samples is performed using the MDS-2000 hydrolysis station (CEM). 6 N HCI containing 1% phenol (scavenger) is used for creating the vapor phase. Hydrolysis time is 20 minutes at 70 psi (~ 148Â°C). Hydrolysed samples are lyophilized and redissolved in 20 jul of 500 pmol//il sarcosine and norvaline as internal standard. The analysis is done using the AminoQuant from Hewlett-Packard according to the manufacturer's instructions. 1 /il of sample is injected. Amino acid sequencing is performed using the 476A Protein Sequencer from Applied Biosystems according to the manufacÂ¬turer's instructions. Premixed buffers are used for the oniine-HPLC.<br>
Purification of p45 from Fusarium oxysporum Broth<br>
The p45 metailoprotease is purified from concentrated and filtered fermentation broth by cation-exchange chromatography (Mono-S) followed by gel filtration on Superose 12. Fractions from Mono-S are selected by assaying for metailoprotease activity as released trypsin-like activity from pro-trypsin-like Fusarium oxysporum DSM 2672 protease.<br>
Metailoprotease containing fractions from the Superose-12 column are identified by the same assay procedure as for the Mono-S fractions. The purified metailoprotease appears as a single band on SDS-PAGE at 45 kDa. Two isoforms<br><br>
of the metalloprotease are observed in lEF (pH 3.5-9.5) at respectively pl 8.4 and 8.7.<br>
Results from amino acid analysis indicate that this metalloprotease (p45) has the N-terminal amino acid sequence shown in the Sequence Listing as SEQ ID NO: 3.<br>
Cloning of Fusarium oxysporum p45 Metalloprotease Gene and Characterization of Recombinant p45<br>
A portion of the Fusarium oxysporum p45 metalloprotease gene is first cloned by PCR. One primer is designed using the N-terminal protein sequence (SEQ I ID NO: 3), and a reverse primer is designed from an internal metalloprotease peptide sequence (residues 483-515 of SEQ ID NO: 1). PCR is performed using the DNA primers and genomic DNA isolated from Fusarium oxysporum. Genomic DNA is isolated as follows.<br>
Approximately 15 g wet weight Fusarium oxysporum is grown in MY50 medium (50 g/l maltodextrin, 2 g/l Mg2S04, 10 g/l KHPO, 2 g/l citric acid, 10 g/l yeast extract, 2 g/l urea, 2 g/l K2S04, 0.5 ml trace metal solution, adjusted to pH 6 with 5 N NaOH) at 30Â°C. Mycelia are suspended in 16 ml TE (10 mM TRIS, 1 mM EDTA, pH 8.0), split into two tubes, and approx. 12 g of 0.45-0.52 mm glass beads (Thomas Scientific) are added to each tube. The samples are alternately vortexed and iced for 30 second intervals until a noticeable viscosity breakdown occurs. The samples are alternately vortexed two additional 30 second intervals. 2.5 ml 20% SDS is added to each sample. The samples are mixed by inversion, incubated 10 minutes at room temperature, and mixed again. Samples are spun 8 minutes at 3.5 K at room temperature. Supernatants are combined in a 50 ml polypropylene tube. The sample is extracted with an equal volume of TE equilibrated with phenol:chloroform:isoamyl alcohol (25:24:1) (P/C/l extracted), then centrifuged 10 minutes at 10,000 rpm at 4Â°C. The supernatant is treated with 300 n\ 10 mg/ml Proteinase Kfor 30 minutes at 25Â°C. The DNA is P/C/l extracted as described above, ethanol precipitated and dissolved in 5 ml TE. The sample is treated with 150 /zl 10 mg/l RNAase A for 15 minutes at 65Â°C, then 15 minutes at 25Â°C. The sample is treated again with Proteinase K (100 /il 10 mg/ml for 1.5 hours at 25Â°C) and P/C/l<br><br>
extracted twice and ethanol precipitated. Tine DNA is pooled onto a bent pasteur pipet and transferred to 5 ml 80% ethanol. The sample is spun 3 minutes at 10,000 rpm. The DNA pellet is dried briefly, then dissolved in 1 ml TE.<br>
PCR is used to clone a portion of the Fusarium oxysporum p45 gene.<br>
5 50-100 ng Fusarium oxysporum genomic DNA is mixed with approx. 100 pmoles each of the synthetic PCR primer DNAs in 1XTaq buffer (Boehringer Mannheim) and a concentration of 100 /il each of dGTP, dATP, dTTP and dCTP in a volume of 50 /il. Taq DNA polymerase (Boehringer Mannheim), 1-5 units, is added, and the PCR incubations are, 95Â°C for 5 minutes, then 35 cycles of [95Â°C for 30 seconds; 50Â°C<br>
I for 1 minute; and 72Â°C for 1 minute].<br>
The PCR reaction produces two DNA fragments of approx. 1.0 and 1.3 kb in length. These fragments are isolated by gel electrophoresis, purified, cloned into an E. coli replicating plasmid, and sequenced using standard methods known in the art of molecular biology. The 1.0 kb DNA fragment is found to contain Fusarium oxysporum p45 gene sequences by a comparison of the translations of the DNA with amino acid sequences obtained from direct protein sequencing. Therefore, this 1.0 kb PCR generated DNA fragment is used as a probe to clone the entire metalloprotease gene from a Fusarium oxysporum genomic DNA library.<br>
A genomic library in lambda phage is prepared from the Fusarium oxysporum genomic DNA using methods such as those described by Sambrook et al. [Sambrook et al.; Molecular Clonina. Cold Spring Harbor, NY, 1989]. A total of 50 nQ genomic DNA is digested in a volume of 200 /zl containing 10 mM TRIS, pH 7.5, 50 mM NaCI, 7 mM MgClg, 7 mM 2-mercaptoethanol, and 4 units restriction enzyme Sau3A for one minute at 25Â°C. Partially digested DNA of molecular size 10-20 kb is isolated by agarose gel electrophoresis, followed by electroelution into dialysis membrane and concentration using an Elutip-D column (Schleicher and Schuell). One /ig of lambda arms of phage of EMBL4, that had been cut with restriction enzyme BamHI and treated with phosphatase (Clonetech), is ligated with 300-400 ng Sau3A cut genomic DNA in a volume of 25 /il under standard conditions jy [cf. Sambrook et al.; Molecular Clonina. Cold Spring Harbor, NY, 1989]. Lambda phage are prepared from this litigation mix using a commercially available kit (Gigapack Gold II, Stratagene), following the manufacturers instructions. The plating<br><br>
of approx. 18,000 recombinant lambda phage, and the production of filter lifts (to N+ filters, Amersham) were performed using standard techniques [cf. Sambrook etal.; Molecular Cloning. Cold Spring Harbor, NY, 1989]. The filters are processed for hybridization with a Genious Kit for non-radioactive nucleic acids detection (Boehringer Mannheim) using the instructions provided by the manufacturer. The DNA used as p45 probe is the 1.0 kb PCR fragment obtained as described above. The probe is labelled by PCR incorporation of dioxigenin (DIG) using a DIG labelling kit and the instructions supplied by the manufacturer. Fifteen ng of the 1.0 kb p45 fragment is mixed in 1X Taq Buffer (Boehringer Mannheim) with 100 pmoles each N-terminal primer and internal reverse primer, and 1-5 units Taq polymerase (Boehringer Mannheim) in a total volume of 80 n\. Reaction conditions were 95Â°C for 3 minutes, then 35 cycles of [95Â°C for 30 seconds; 50Â°C for 1 minute; and 72Â°C for 1 minute], and 72Â°C for 5 minutes. The filter hybridizations using the DIG labelled probe and the wash conditions were performed using the instructions provided by the Genious Kit manufacturer.<br>
Hybridizing phages are detected with an alkaline phosphatase-conjugated anti-dioxigenin antibody visualized with Lumiphos 530 as described by the manufacturer (Boehringer Mannheim). DNA preparations are made from the positive lambda clones using the Qiagen Lambda Midi Kit (QiAGEN, Inc.). DNA from one preparation is digested with restriction enzyme EcoRI and a 6.3 kb fragment is subcloned into plasmid pUC118. DNA sequence analysis of portions of this subclone identified the entire coding region of the p45 gene, cf. SEQ ID NO: 1.<br>
Cloning p45 Metalloprotease cDNA<br>
Total RNA and poly-A RNA is prepared from Fusarium oxysporum according to the previous published protocols [Chirgwin et al., Biochemistry. 1988 18 5294-5299; Aviv and Leder. Proc. Natl. Acad. Sci.. USA. 1972 69 1408-1412; Sambrooketal.; Molecular Cloning. Cold Spring Harbor, NY, 1989] with the following modifications.<br>
Specifically, mycelia is ground in liquid nitrogen to a fine powder and<br>
then resuspended, with stirring, in a lysis buffer containing 4 M guanidinium<br>
' thiocyanate, 0.5% Na-laurylsarcosine, 25 mM Na-citrate and 0.1  M 2-mercapto-<br><br>
ethanol, pH 7.0, for 30 minutes at room temperature. Cell debris is removed by low speed (5000 rpm for 30 minutes) centrifugation. Typically, the poly-A RNA fraction is isolated using oligo (dT) cellulose obtained from Boehringer Mannheim.<br>
The poly-A RNA is used to generate cDNA using the hairpin/RNaseH method [Sambrook et al.; Molecular Cloning. Cold Spring Harbor, NY, 1989]. Specifically, 5 /zg poly-A RNA in 5 n\ water is heated at 70Â°C, then placed on ice. A total reaction mix of 50 n\ is prepared containing the poIy-A RNA, 50 mM TRIS. pH 8.3, 75 mM KCI, 3 mM MgCI, 10 mM DTT, 1 mM each dGTP, dATP, dTTP and dCTP, 40 units RNasin, 10 /zg oligo (dT12-18) primer, and 1000 units Superscript II RNase H- reverse transcriptase (Bethesda Research Laboratories). The mix is incubated at 45Â°C for one hour. Then 30 /xl of 10 mM TRIS, pH 7.5. 1 mM EDTA, 40 lig glycogen carrier (Boehringer Mannheim), 0.2 volumes 10 M ammonium acetate, and 2.5 volumes ethanol were added to precipitate the nucleic acids. After centrifugation, the pellet is resuspended in 20 mM TRIS, pH 7.4, 90 mM KCI, 4.6 mM MgCIa, 10 mM ammonium sulphate, 16 /iM )9NAD\ 100 nM each dGTP, dATP, dTTP and dCTP, 44 units E. coli DNA polymerase I, 6.25 units RNaseH, and 10.5 units DNA ligase. Second strand DNA synthesis is performed in this solution at 16Â°C for 3 hours. The DNA is concentrated by ethanol precipitation and the pellet is resuspended in 30 n\ of 30 mM Na-acetate, pH 4.6, 300 mM NaCI, 1 mM ZnS04, 0.35 mM DTT, 2% glycerol, and 30 units Mung Bean nuclease (Bethesda Research Laboratories) at 30Â°C for 30 minutes. The DNA solution is neutralized with 70 /il 10 mM TRIS, pH 7.5,1 mM EDTA, phenol extracted, and ethanol precipitated. The pellet is treated with 7.5 units T4 polymerase (Invitrogen) at 25Â°C for 15 minutes in 50 ti\ buffer (20 mM TRIS-acetate, pH 7.9, 10 mM Mg-acetate, 50 mM K-acetate, 1 mM DTT, 0.5 mM each dGTP, dATP, dTTP and dCTP). The reaction is stopped by addition of EDTA to 20 mM followed by phenol extraction and ethanol precipitation. The result of this procedure is double stranded cDNA with blunt ends suitable for attachment of DNA linkers and cloning Into any vector.<br>
The cDNA with EcoRI linkers is size fractionated on an agarose gel to obtain cDNAs of molecular size 0.7 kb or greater. The cDNA is recovered from the gel by electroelution and purified by phenol extraction and ethanol precipitation. The size fractionated cDNA is used to construct a lambda cDNA library.The cDNA is<br><br>
cloned into lambda ZIPLOX arms (Gibco BRL). Full length cDNA clones are identified using a 467 bp dioxigenin labeled fragment as probe (bp 336-803 of the genomic clone) with the techniques of plaque lifts and DNA hybridization as previously described. Full length cDNA is recovered in plasmid pZL1 as described by the [manufacturer (strains and plasmid from Bibco BRL).<br>
The full length cDNA is sequenced and compared with the sequence of the genomic DNA. The genomic DNA is 2052 bp in length and contains three introns. The predicted coding region of pre-pro p45 metailoprotease consists of a putative 18 amino acid signal sequence, a 226 amino acid pro-region, and a 388 amino acid mature region, as shown in SEQ ID NO: 1.<br>
Preparation of Fusarium oxysporum p45 Metailoprotease Probe<br>
A clone from the above cDNA library was selected and designated pDM115. Plasmid pDM115 contains a 1.76 kb fragment of Fusarium oxysporum cDNA, that encodes part of the p45 gene. This plasmid was digested with Sail and the fragments were separated on a 1% agarose gel. The 1.5 kb fragment was cut out and DNA eluted. This fragment was labelled with 32-P-dATP by random-primed labeling and used for either Southern or colony lift probing.<br>
Screening Aspergillus oryzae Library with Fusarium oxysporum p45 Probe<br>
The individually frozen colonies in the library were inoculated onto LB-plates (100/ig/ml ampicillin) by using a multipin device with 6 times 8 pins fitting into half a microtiter dish. Plates were made containing colonies from all clones in the library. The plates were incubated at 37Â°C over night. Sterilized Whatman 540 filters cut to the size of a petri dish were placed upon the colonies which were incubated for two more hours at 37Â°C. The filters were transferred to LB plates containing 200/ig/ml of chloramphenicol and the plates were incubated over night at 37Â°C.The next day the filters were washed twice in 0.5 M NaOH for 5 minutes, then twice in 0.5 M Tris-HCI (pH 7.4) for 5 minutes and then twice in 2 x SSC for 5 minutes. The filters were wet with ethanol and air dried.<br>
The filters were hybridized with the 1.5 kb 'P labelled DNA fragment from  pDM115 containing the protease gene from Fusarium oxysporum. The<br><br>
hybridization was carried out for 16 hours at 65Â°C in 10 x Denhart, 5 x SSC, 0.02 M EDTA, 1% SDS, 0.15 mg/ml polyA, and 0.05 mg/ml yeast tRNA. After hybridization the filters were washed in 2 x SSC, 0.1% SDS at 65Â°C twice and placed on X-ray films. Three colonies showed hybridization to the probe, namely 3E8, 3C1 and 2A5, I the names refer to their position in the library.<br>
Characterization of the Cosmid Clones<br>
By restriction analysis it was establised that two of the three cosmid clones (3E8 and 3C1) contained inserts which were derived from the same region of the Aspergillus oryzae genome.<br>
3 /zg of cosmid DNA was digested with EcoRI and fractionated by agarose gel electroferase. The DNA was transferred to Immobilan-N membrane filters and hybridized with the 1.5 kb radiolabelled probe from pDM115. The probe hybridized to a 4 kb EcoRI fragment in both cosmid clones. The 4.0 kb EcoRI fragment was chosen for further analysis.<br>
5 Cloning of Npl into the Plasmid pToC65 and its Sequence<br>
Plasmid pToC65 was digested with Sad and treated with bacterial alkaline phosphatase to remove the 5'-phosphate groups according to the manufacturers instructions. Afterwards it was phenol extracted and precipitated.<br>
The 5.5 kb Sad fragment from cosmid clone 3E8 containing the ) Aspergillus oryzae Npl gene was isolated by gel electrophoresis and purified.<br>
The two fragments were mixed together and ligated. After transformations of E. coli, the colonies carrying the correct plasmid were identified by restriction enzyme digestion of mini-plasmid preparations. This plasmid was called pJaL389.<br>
Comparison of DNA sequence analysis of portions of this subclone to other known Npl gene sequences was used to identify that the subclone contains the coding region of the Aspergillus oryzae Npl gene.<br><br>
EXAMPLE 2<br>
Genomic Disruption of Aspergillus oryzae Neutral Metalioprotease Npl<br>
In order to generate strains of Aspergillus oryzae that are specifically deficient in the production of Npl, a gene replacement strategy as described by Miller et al.\ Mol. Cell. Biol. 1985 5 1714-1721, was employed. Below, these experiments are described in more details.<br>
Cloning of the Aspergillus oryzae pyrG gene<br>
The Aspergillus oryzae pyrG gene was cloned by cross hybridization with the Aspergillus niger pyrG gene [W. van Hartingsveldt et al.; Mol. Gen. Genet 1987 206 71-75]. A lambda library of partial SaulllA digested Aspergillus oryzae IFO 4177 DNA was probed at low stringency with a 1 kb DNA fragment from the Aspergillus niger pyrG gene. DNA from a positive clone was subcloned into a pUC118 vector. The resultant plasmid, pS02, was shown to contain the pyrG gene by complementation of an Aspergillus niger pyrG - mutant, cf. Fig. 1.<br>
Construction of an Aspergillus oryzae pyrG Minus Strain<br>
A pyrG deletion plasmid, pS05, containing about 1 kb of pyrG flanking sequences on each end, was constructed from the plasmid pS02. The strain Aspergillus oryzae IFO 4177 was transformed with this construct and transformants were selected by resistance to 5-fluoro-orotic acid, a phenotype characteristic of pyrG mutants.<br>
One transformant, HowBI 01, was shown by Southern analysis to have the expected deletion at the pyrG locus. Being a pyrG mutant, HowB101 requires uridine for growth. HowBI 01 can be transformed with the wt pyrG gene by selection for ability to grow without uridine.<br>
The steps involved in the construction of HowBI 01 are illustrated in Fig. 2.<br><br>
Construction of Plasmid pJaL335<br>
In order to amplify a 431 bp fragment located 479 nucleotides upstream from the 5' end of the Aspergillus oryzae pyrG genet, the two following oligonucleotides were made:<br>
Primer A: GGAGGAAGATCTCTCTGGTACTCTTCGATCTC: SEQ ID NO: 5; and Primer B: GGAGGAGAATTCAAGCTTCTTCTACATCACAGTTTGAAAGC: SEQ ID NO: 6. The underlined part corresponds to the Aspergillus oryzae pyrG gene sequence.<br>
The 5' ends of the primers were for facilitating cloning (Primer A contains a Bglll restriction endonuclase site, and primer B contains a EcoRI and a I Hindlll restriction endonuclase site).<br>
Plasmid pS02 was used as template in the PCR reaction. Amplification was performed in 100 /xl volumes containing 2.5 units Taq-polymerase, 100 ng of pS02, 50 mM KCI, 10 mM Tris-HCI pH 8.0,1.5 mM MgCI, 250 nM of each dNTP, and 10 pmol of each of the two primers described above.<br>
Amplification was carried out in a Perkin-Elmer Cetus DNA Termal 480, and consisted of one cycle of 3 minutes at 94Â°C, followed by 25 cycles of 1 minutes at 94Â°C, 30 seconds at 55Â°C, and 1 minutes at 72Â°C. The PCR reaction produced one DNA fragment of 430 bp in length. This fragment was digested with Bglll and EcoRI, and isolated by gel electrophoresis. It was purified and cloned into the corresponding site in plasmid pS02. The resulting plasmid was called pJaL335. The construction of pJaL335 is illustrated in Fig. 3.<br>
Construction of Disruption Plasmid pJaL399<br>
Plasmid pJaL389 was digested with Ball, and treated with Klenow polymerase to make the ends blunt. The 7.1 kb fragment was isolated by gel electrophoresis, and purified. This DNA fragment was then treated with bacterial alkaline phosphatase to remove the 5' phosphate groups according to the manufacturer's instructions and phenol extracted and precipitated.<br>
Plasmid pJaL335 was digested with Hindlll, and treated with Klenow polymerase to make the ends blunt. The 3.5 kb fragment encoding the Aspergillus oryzae pyrG gene was isolated by gel electrophoresis and purified.<br><br>
The two fragments were mixed together and ligated. After transformations of E. coli, the colonies carrying the correct plasmids were identified by restriction enzyme digestion of mini-plasmid preparations. The construction of pJaL399 is illustrated in Fig. 4.<br>
pJaL399 holds a pToC65 vector containing a fragment which carries the Npl gene flanked by Sad sites, and where the central 1.1 kb Ball fragment has been replaced by an 3.5 kb DNA fragment encoding the Aspergillus oryzae pyrG gene.<br>
Transformation of Aspergillus oryzae<br>
15 Mg of plasmid pJaL399 are digested to completion by Sad. The completeness of the digest is checked by running an aliquot on a gel and the remainder of the DNA is phenol extracted, precipitated and resuspended in 10 8,25 111 of sterile water.<br>
The transformation of Aspergillus oryzae HowBIOI host strain is preformed by the protoplast method [Christensen et al.; Biotechnology 1988 61419-1422]. Typically, Aspergillus oryzae mycelia is grown in a rich nutrient broth. The mycelia is separated from the broth by filtration. Novozymeâ¢ (available from Novo Nordisk A/S, Denmark) is added to the mycelia in an osmotically stabilizing buffer such as 1.2 M MgS04 buffered to pH 5.0 with sodium phosphate. The suspension is incubated for 60 minutes at 37Â°C with agitation. The protoplasts are filtered through mira-cloth to remove mycelial debris. The protoplasts are harvested and washed twice with STC (1.2 M sorbitol, 10 mM CaClj, 10 mM Tris-HCI pH 7.5). Finally, the protoplasts are resuspended in 200-1000 /zl STC.<br>
For transformation 5 HQ DNA is added to 100/il protoplast suspension. 200 Ml PEG solution (60% PEG 4000, 10 mM CaCI, 10 mM Tris-HCI pH 7.5) was added, and the mixture is incubated for 20 minutes at room temperature. The protoplasts are harvested and washed twice with 1.2 M sorbitol. The protoplasts are finally resuspended in 200 /xl 1.2 M sorbitol, plated on selective plates (minimal medium + 10 g/l Bacto-Agar (Difco), and incubated at 37Â°C. After 3-4 days of growth at 37Â°C, stable transformants will appear as vigorously growing and sporulating colonies.<br><br>
Identification of Gene Disruption<br>
From the stable colonies, individual spores are streaked on fresh minimal plates. Single colonies are selected and restreaked to give pure cultures. These are used to inoculate 10 ml of liquid YPIVI medium (1% yeast extrat, 1% peptone, 2% maltose). After 18 hours at 30Â°C and shaking at 180 rpm, the mycelia is han/ested on filter paper. Mycelia is then transferred to an 2 ml eppendorf tube and freeze dried.<br>
After freeze drying DNA is prepared from the individual mycelia by grinding the mycelia to a fine powder with a pestle in the tube. This powder is resuspended in 0.5 ml of 50 mM EDTA pH 8.0, 0.2% SDS, 1 /xl DEP, by vortexing. After incubation at 65Â°C for 20 minutes, 0.1 ml 5 M KAc pH 6.5, is added and the solution is mixed and incubated on ice for 5 minutes. The cell debris is separated from the DNA solution by centrifugation at 20.000 rpm for 5 minutes. 0.4 ml supernatant is precipitated with 0.3 ml isopropanol and centrifugated at 20.000 rpm &gt; for 10 minutes. The DNA pellet is redisolved in 100 jul of sterile TE buffer containing 0.1 mg/ml RNAaseA.<br>
3 ng of each DNA is digested with Ball, fractionated by agarose gel electroferase, transferred to Immobilan-N membrane filters. The filters were hybridized with the 5.5 kb P labelled DNA Sad fragment from pJaL389 containing I the Npl protease gene. Strains which carry a disruption of the Npl gene are recognized by lacking the 1.1 kb Ball hybridizing fragment as well as having altered mobility of the other two flanking fragments.<br>
EXAMPLE 3<br>
Cloning of Aspergillus oryzae Neutral Metalloprotease II (Npll) f Construction of pJaL198<br>
From the published cDNA nucleotide sequence encoding Aspergillus oryzae Npll [Tatsumi etal.; Mol. Gen. Genet. 1991 228 97-103], two oligonuclotides were designed so that the encoding part of the Npll gene was amplified in a PCR reaction.<br><br>
A primer (CTAGGATCCAAGGCATTTATGCGTGTCACTACTCTC: SEQ ID NO: 7) was constructed so that the 3' end of the nucleotide sequence correÂ¬sponds to the N-terminal part of the Npll gene (underlined), and the 5'-end is for facilitating cloning (contains a BamHI restriction endonuclease site).<br>
A primer (CTACTCGAGTTAGCACTTGAGCTCGATAGC: SEQ ID NO: 8) was constructed so that the 3' end of the nucleotide sequence corresponds to the C-terminal part of the Npll gene (underlined), and the 5'-end is for facilitating cloning (contains a Xhol restriction endonuclease site).<br>
Genomic DNA from Aspergillus oryzae IFO 4177 was used as template in the PCR reaction. Amplification reaction was performed in 100 /il volumes containing 2.5 units Taq-polymerase, 100 ng o Aspergillus oryzae genomic DNA, 50 mM KCI, 10 mM Tris-HCI pH 8.0. 1.5 mM MgCI. 250 nM of each dNTP, and 100 pM of each of the two primers described above.<br>
Amplification was carried out in a Perkin-Elmer Cetus DNA Termal 480, and consisted of one cycle of 3 minutes at 94Â°C, followed by 25 cycles of 1 minutes at 94Â°C, 30 seconds at 55Â°C, and 1 minutes at 72Â°C. The PCR reaction produces one DNA fragment of approx. 1.1 kb in length. This fragment was isolated by gel electrophoresis, purified, cloned into the vector pCRâ¢ll (Invitrogen Corporation), and sequenced using standard methods known in the art of molecular biology. The resulting plasmid was called pJaL198.<br>
EXAMPLE 4<br>
Genomic Disruption of Npll<br>
Construction of JaL121<br>
In order to generate strains o Aspergillus oryzae that were specifically deficient in the production of Npll, a gene replacement strategy as described by Miller et al.\ Mol. Cell. Biol. 1985 5 1714-1721, was employed.<br>
Cloning Aspergillus oryzae pyrG Gene<br>
The Aspergillus oryzae pyrG gene was cloned by cross hybridization with the Aspergillus r)iger pyrG gene [W. van Hartingsveldt et al.; Mol. Gen. Genet.<br><br>
1987 206 71-75]. A lambda library of partial SaulllA digested Aspergillus oryzae IFO 4177 DNA was probed at low stringency with a 1 kb DNA fragment from the Aspergillus niger pyrG gene. DNA from a positive clone was subcloned into a pUC118 vector. The resultant plasmid, pS02, was shown to contain the pyrG gene by complementation of an Aspergillus niger pyrG - mutant, cf. Fig. 1.<br>
Construction of an Aspergillus oryzae pyrG Minus Strain<br>
A pyrG deletion plasmid, pS05, containing about 1 kb of pyrG flanking sequences on each end, was constructed from the plasmid pS02. The strain Aspergillus oryzae IFO 4177 was transformed with this construct, and transformants were selected by resistance to 5-fluoro-orotic acid, a phenotype characteristic of pyrG mutants. One transformant, HowBIOI, was shown by Southern analysis to have the expected deletion at the pyrG locus. Being a pyrG mutant, HowBIOI requires uridine for growth. HowBIOI can be transformed with the wt pyrG gene by selection for ability to grow without uridine.<br>
The steps involved in the construction of HowBI 01 are illustrated in Fig. 2.<br>
Construction of Disruption Plasmid pJaL218<br>
Plasmid pJaL198 is digested with BstEII and treated with Klenow polymerase to make the ends blunt. The 4.9 kb fragment was isolated by gel electrophoresis and purified. This DNA fragment was then treated with bacterial alkaline phosphatase to remove the 5' phosphate groups, according to the manufacturers instructions, phenol extracted and precipitated.<br>
Plasmid pJers4 was digested with Hindlll and treated with Klenow polymerase to make the ends blunt. The 1.8 kb fragment encoding the Aspergillus oryzae pyrG gene was isolated by gel electrophoresis and purified.<br>
The two fragments were mixed and ligated. After transformations of Â£ coll DH5a, the colonies carrying the correct plasmids are identified by restriction enzyme digestion of mini-plasmid preparations. The construction of pJaL218 is illustrated in Fig. 5.<br><br>
pJal_218 consists of the pCRâ¢ll vector containing a fragment which carries the Npll gene flanked by EcoRI sites, in which the central BstEII fragment has been replaced by a 1.8 kb DNA fragment encoding the Aspergillus oryzae pyrG gene.<br>
Transformation Aspergillus oryzae<br>
15 /ig of plasmid pJal_218 is digested to completion by EcoRI. The completeness of the digest was checked by running an aliquot on a gel. The remainder of the DNA was phenol extracted, precipitated and resuspended in 10 MI of sterile water.<br>
The transformation of Aspergillus oryzae HowBIOI host strain was performed by the protoplast method [Christensen et al.; Biotechnology 1988 61419-1422]. Typically, Aspergillus oryzae mycelia was grown in a rich nutrient broth. The mycelia was separated from the broth by filtration. Novozymeâ¢ (available from Novo Nordisk A/S, Denmark) was added to the mycelia in an osmotically stabilizing buffer, 1.2 M MgS04, sodium phosphate buffer pH 5.0. The suspension was incubated for 60 minutes at 37Â°C with agitation. The protoplast was filtered through Miracloth to remove mycelial debris. The protoplast was harvested and washed twice with STC (1.2 M sorbitol, 10 mM CaCI, 10 mM Tris-HCI pH 7.5). The protoplast was finally resuspended in 200-1000 M' STC.<br>
For transformation, 5 iig DNA was added to 100 /xl protoplast suspension. 200 /il PEG solution (60% PEG 4000,10 mM CaCI, 10 mM Tris-HCI pH 7.5) was added, and the mixture is incubated for 20 minutes at ambient temperature. The protoplast was harvested and washed twice with 1.2 M sorbitol. The protoplast was finally resuspended 200 /il 1.2 M sorbitol, plated on selective plates (minimal medium -i- 10 g/l Bacto-Agar (Difco), and incubated at 37Â°C.<br>
After 3-4 days of growth at 37Â°C, stable transformants appear as vigorously growing and sporulating colonies.<br>
Identification of Gene Disruptions<br>
From stable colonies, individual spores are streaked on fresh minimal plates. Single colonies are selected and restreaked to give pure cultures.<br><br>
Thirty-three transformants were screened to see if the transformed DNA fragment had integrated by a double overcross into the corresponding gene on the chromosome by PCR. PCR reaction and genomic DNA from the transformants was performed as described above.<br>
The primers used were CCCTTCTTTCCAAACCG (SEQ ID NO: 9), which is located 5' from the encoding region of the Npll gene, and pyrG-5' (GGGTGAGCCACTGCCTC; SEQ ID NO: 10), which is specific for the pyrG gene. One transformant yielded the expected PCR product on 1.1 kb.<br>
From Southern blots, where genomic DNA from the transformant and from Aspergillus oryzae was digested with EcoRI, fractionated by agarose gel electrophoresis, transferred to Immobilan-N membrane filters, and probed with the 1.1 kb EcoRI fragment from pJaL198 containing the Npll gene, it was found that the wild-type band on 3.8 kb was shifted to a 10 kb band in the transformant. This proves that the transformed DNA was integrated into the Npll gene in multiple copies. The strain was designated JaL121.<br>
EXAMPLE 5<br>
Production of Chymosin in JaL121<br>
Aspergillus oryzae strain JaL121 was transformed with the plasmid pToC56 (cf. Fig. 6), which is a fungal expression plasmid for the mammalian enzyme I chymosin, by co-transformation with pToC90. The construction of plasmid pToC56  is described in EP 98 993 A.<br>
Transformants were selected for growth on minimal medium containing 10 mM acetamide, and screened for the presence of pToC56 by the ability to produce chymosin. A transformant was grown in shake flasks for 4 days at 30Â°C in a medium containing maltodextrin, soybean meal and peptone. A transformant of pToC56 in Aspergillus oryzae IFO 4177 was grown together with the JaL121 transformant.<br>
Each day, fermentation broth samples were collected and applied to SDS-Page and Western blotting. The blotting membrane was incubated with<br><br>
chymosin specific rabbit antibody, followed by goat rabbit antibody coupled to peroxidase.<br>
Staining of the membrane showed that on the first and second day of fermentation, the supernatants from transformants oi Aspergillus oryzae IFO 4177 contained small amounts of chymosin, or degradation products thereof. Later on no chymotrypsin was detected. In contrast, transformants of JaL121 contained at least ten times of full size chymosin. The amount of chymosin in the supernatants increased for the first two-three days and then remained constant.<br><br>
SEQUENCE LISTINGS<br>
INFORMATION FOR SEQ ID NO: 1:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 2052 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: DNA (genomic) (vi) ORIGINAL SOURCE:<br>
(A)	ORGANISM: Fusarium oxysporum<br>
(B)	STRAIN: DSM 2672<br>
(C)	INDIVIDUAL ISOLATE: p45 (ix) FEATURE:<br><br>
(A)	NAME/KEY: mat_peptide<br>
(B)	L0CATI0N:785..2049 (ix) FEATURE:<br><br>
(A)	NAME/KEY: sig_peptide<br>
(B)	LOCATION:55..784 (ix) FEATURE:<br><br>
(A)	NAME/KEY: intron<br>
(B)	L0CATI0N:364..415 (ix) FEATURE:<br><br>
(A)	NAME/KEY: intron<br>
(B)	L0CATI0N:802..854 (ix) FEATURE:<br><br>
(A)	NAME/KEY: intron<br>
(B)	LOCATION:1821..1868<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 1:<br>
ATGCGTTTCT CCGACTCTCT CCTCCTCATC GGCCTATCCA GCCTCGCTGG TGCTCATCCC	60<br>
AGCAGAAGGG CTCCTAATCC TTCACCGCTG AGCAAGCGTG GCCTCGACCT GGAAGCTTTT	120<br>
AAGCTTCCTC CCATGGCCGA GTACGTTCCT CAGGACGAGG TTCCTGATGA TGTCAGTGCC	180<br>
AAGGTCGTCA CCAAGCGCGC TGATTACACC GAGACTGCCA AGGACTTGGT TAAGTCGACT	240<br>
TTCCCCAAGG CTACTTTCCG TATGGTCACG GATCACTATG TTGGTAGCAA CGGAATTGCG	300<br>
CATGTAAACT TTAAGCAGAC TGTCAACGGT ATTGATATCG ACAATGCTGA TTTCAACGTC	360<br>
AACGTGGGTA TTCTCAAGAC TTTGGGGAGT TTGGAATGTG CTGACATGGA TACAGATTGG	420<br><br>
CGCT6AC66C GAG6TCTTCT CCTACGGAAA CAGCTTCTAC GAGG6CAAGA TTCCCGGTCC	480<br>
TCTTACCAAG CGTGACGAGA AAGACCCCGT CGACGCTCTC AAGGACACCG TTGATGTTCT	540<br>
TTCTCTCCCC GTTGAGGCTG ACAAGGCCAA GGCTGAGAAG AAGAGCAAGA ACCACTACAC	600<br>
CTTCACTGGT ACCAAGGGTA CCGTCA6CAA GCCCGAGGCT AA6CTCACCT ACCTTGTTGA	660<br>
TGAGAACAAG GAGCTCAAGC TCACATGGAG AGTTGAGACT GATATTGTTG ACAACTGGCT	720<br>
GTTGACTTAT GTCAATGCTG CCAAGACTGA TGAGGTTGTT GGTGTTGTT6 ACTAC6TCAA	780<br>
TGAGGCGACA TACAAGGTCT AGTACGTATT TCCATAAATT GACGATTGGG AAAGAATTGA	840<br>
CC6TTGTATT ATAGTCCTTG GGGTGTCAAT GATCCCTCCA AGG6ATCTCG CTCCACTGTT	900<br>
GAGAACCCCT GGAATCTCGC GGCCTCCGAG TTCACCTGGC TCAGCGACG6 CTCAAACAAC	960<br>
TACACCACAA CCC6CGGGAA CAATGGAATT GCACAGGTGA ATCCTTCAGG GGGCTCCACG	1020<br>
TATCTGAACA ATTACC6TCC TGATAGCCCG TCGCTGAAGT TCGAGTATGA TTACTCCACC	1080<br>
AGCACCACTA CACCCACCAC CTACCGCGAT GCTTCCATC6 CTCAGCTTTT CTACACAGCC	1140<br>
AACAAGTACC ACGACCTCCT CTACCTTCTT GGCTTTACCG AACAGGCTGG TAACTTCCAG	1200<br>
ACCAACAACA ATGGCCAGGG TG6TGTAGGA AACGATATGG TTATCCTCAA CGCTCAGGAC	1260<br>
GGAAGCGGCA CCAACAACGC CAACTTCGCT ACACCCGCTG ACGGTCAGCC CGGCCGCATG	1320<br>
CGAATGTATC TCTGGACATA CAGCACACCC CAGCGTGACT GCAGTTTCGA CGCTGGCGTT	1380<br>
GTTATCCACG AGTACACTCA CGGTCTCTCC AACCGTCTCA CAGGTG6CCC TGCCAACTC6	1440<br>
GGTTGTCTTC CCGGTGGTGA ATCCGGTGGC ATGGGTGAGG GCTGGGGTGA CTTCATGGCT	1500<br>
ACTGCCATTC ACATCCAATC CAAGGATACC CGC6CTAGCA ACAAGGTCAT GGGTGACTGG	1560<br>
GT6TACAACA ACGCAGCTGG TATCCGAGCT TATCCTTACA GTACAAGCCT TACCACTAAC	1620<br>
CCTTACACTT ACAA6AGTGT TAACAGTCTC AGTGGAGTCC ATGCTATTGG TACTTACTGG	1680<br>
GCTACTGTTC TGTAT6AG6T TATGTGGAAC CTCATCGACA AGCATGGGAA GAATGATGCG	1740<br>
GATGAGCCCA AATTCAACAA CGGCGTTCCT ACAGATGGCA AATATCTTGC TATGAAGTTA	1800<br>
GTAGTGGATG GCATGTCGCT GTAAGTTGTC CCTTGGATTT GTAGGAGTTC TTATCTAACG	1860<br>
TTTAATAGGC AACCTTGCAA CCCCAACATG GTCCAGGCCC GAGACGCCAT CATC6ACGCC	1920<br>
GACACCGCTC TTACCAAGGG AGCTAACAAG TGCGAGATCT GGAAGGGCTT TGCCAAGCGT	1980<br>
GGTCTTGGAA CTGGTGCCAA GTATAGTGCT TCCAGCCGTA CTGAGAGCTT TGCTCTTCCT	2040<br>
TCTGGATGTT AA	2052<br><br>
INFORMATION FOR SEQ ID NO: 2:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LEN6TH: 388 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	T0P0L06Y: linear (ii) MOLECULE TYPE: protein (vi) 0RI6INAL SOURCE:<br><br>
(A)	ORGANISM: Fusarium oxysporum<br>
(B)	STRAIN: DSM 2672<br>
(C)	INDIVIDUAL ISOLATE: p45<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 2:<br>
Ala Thr Tyr Lys Val Tyr Pro Trp Gly Val Asn Asp Pro Ser Lys Gly<br>
15	10	15<br>
Ser Arg Ser Thr Val Glu Asn Pro Trp Asn Leu Ala Ala Ser Glu Phe<br>
20	25	30<br>
Thr Trp Leu Ser Asp Gly Ser Asn Asn Tyr Thr Thr Thr Arg Gly Asn<br>
35	40	45<br>
Asn Gly He Ala Gin Val Asn Pro Ser Gly Gly Ser Thr Tyr Leu Asn<br>
50	55	60<br>
Asn Tyr Arg Pro Asp Ser Pro Ser Leu Lys Phe Glu Tyr Asp Tyr Ser<br>
65	70	75	80<br>
Thr Ser Thr Thr Thr Pro Thr Thr Tyr Arg Asp Ala Ser He Ala Gin<br>
85	90	95<br>
Leu Phe Tyr Thr Ala Asn Lys Tyr His Asp Leu Leu Tyr Leu Leu Gly<br>
100	105	110<br>
Phe Thr Glu Gin Ala Gly Asn Phe Gin Thr Asn Asn Asn Gly Gin Gly<br>
115	120	125<br>
Gly Val Gly Asn Asp Met Val He Leu Asn Ala Gin Asp Gly Ser Gly<br>
130	135	140<br>
Thr Asn Asn Ala Asn Phe Ala Thr Pro Ala Asp Gly Gin Pro Gly Arg<br>
145	150	155	160<br>
Met Arg Met Tyr Leu Trp Thr Tyr Ser Thr Pro Gin Arg Asp Cys Ser<br>
165	170	175<br>
Phe Asp Ala Gly Val Val He His Glu Tyr Thr His Gly Leu Ser Asn<br>
180	185	190<br><br>
Arg Leu Thr Gly Gly Pro Ala Asn Ser Gly Cys Leu Pro Gly Gly Glu<br>
195	200	205<br>
Ser Gly Gly Met Gly Glu Gly Trp Gly Asp Phe Met Ala Thr Ala He<br>
210	215	220<br>
His He Gin Ser  Lys Asp Thr Arg Ala Ser Asn Lys Val Met Gly Asp<br>
225	230	235	240<br>
Trp Val Tyr Asn Asn Ala Ala Gly He Arg Ala Tyr Pro Tyr Ser Thr<br>
245	250	255<br>
Ser Leu Thr Thr Asn Pro Tyr Thr Tyr Lys Ser Val Asn Ser Leu Ser<br>
260	265	270<br>
Gly Val His Ala He Gly Thr Tyr Trp Ala Thr Val Leu Tyr Glu Val<br>
275	280	285<br>
Met Trp Asn Leu He Asp Lys His Gly Lys Asn Asp Ala Asp Glu Pro<br>
290	295	300<br>
Lys Phe Asn Asn Gly Val Pro Thr Asp Gly Lys Tyr Leu Ala Met Lys<br>
305	310	315	320<br>
Leu Val Val Asp Gly Met Ser Leu Gin Pro Cys Asn Pro Asn Met Val<br>
325	330	335<br>
Gin Ala Arg Asp Ala He He Asp Ala Asp Thr Ala Leu Thr Lys Gly<br>
340	345	350<br>
Ala Asn Lys Cys Glu He Trp Lys Gly Phe Ala Lys Arg Gly Leu Gly<br>
355	360	365<br>
Thr Gly Ala Lys Tyr Ser Ala Ser Ser  Arg Thr Glu Ser Phe Ala Leu<br>
370	375	380<br>
Pro Ser Gly Cys 385<br>
INFORMATION FOR SEQ ID NO: 3:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 14 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear (ii) MOLECULE TYPE: peptide<br>
(v) FRAGMENT TYPE: N-terminal (vi) ORIGINAL SOURCE:<br>
(A) ORGANISM: Fusarium oxysporum<br><br>
(B)	STRAIN: DSM 2672<br>
(C)	INDIVIDUAL ISOLATE: p45<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 3:<br>
Ala Thr Tyr Lys Val Tyr Pro Trp Gly Val Asn Asp Pro Ser<br>
1	5	10<br>
INFORMATION FOR SEQ ID NO: 4:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 747 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: DNA (genomic) (vi) ORIGINAL SOURCE:<br>
(A)	ORGANISM: Aspergillus oryzae<br>
(B)	STRAIN: IFO 4177<br>
(C)	INDIVIDUAL ISOLATE: Npl<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 4:<br>
GCGTGGGGGA TGAATGACCC GACGGAGGGC	CCTCGCACCG	TCATCAGCGA TCCATGGGAT	60<br>
TC6TCC6CAT CTGC6TTCAC CTGGATCAGT	GACGGAGAAA	ACAACTATAC CACAACTCGC	120<br>
I GGCAACAACG GTATCGCGCA GTCGAACCCT	ACC6GTGGAT	CGCAGTACTT GAAGAACTAC	180<br>
C66CCTGATA GCCCCGATTT GAAATTCCAA	TACCCCTATT	CGTTCAACGC CACACCCCCA	240<br>
GAGTCCTATA TTGATGCGTC TATCACTCAG	CTTTTCTACA	CTGCCAACAC GTACCACGAT	300<br>
CTACTCTACA CTCTGGGCTT CAACGAG6A6	GCCGGTAATT	TCCAGTACGA TAACAATGGA	360<br>
AAAGGAGGTG CT66AAACGA CTACGTGATC	CTCAATGCTC	AGGACGGTTC TGGCACCAAT	420<br>
i AACGCCAACT TCGCTACGCC CCCGGATGGA	CAGCCCGGCC	GCATGCGCAT GTACATATGG	480<br>
ACCGAGTCCC AGCCTTACCG TGACGGCTCC	TTCGAGGCTG	GTATTGTGAT TCACGAGTAT	540<br>
ACTCACG6CC GTATGTATCC CTTATGAACC	CCAAGTAAGG	CAGTCTGAAC TAACACCAC6	600<br>
GCACACAGTC TCTAACCGGC TCACTGGAGG	ACCCGCTAAC	TCTCGCTGTT TGAATGTCCT	660<br>
TGAATCCGGC GGAATGG6TG AAGGTTGGGG	AGACTTCATG	GCCACGGTAT TTCGGCTCAA	720<br>
) GGTCGGCGAT TCTCACTTCG ATCCTTT	747<br><br>
[NFORMATION FOR SEQ ID NO: 5:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 32 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear (ii) MOLECULE TYPE: cDNA<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 5:<br>
SGAGGAAGATCTCTCTGGTACTCTTCGATCTC<br>
INFORMATION FOR SEQ ID NO: 6:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 41 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear (ii) MOLECULE TYPE: cDNA<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 6:<br>
GGAGGAGAATTCAAGCTTCTTCTACATCACAGTTTGAAAGC<br>
INFORMATION FOR SEQ ID NO: 7:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 36 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear (ii) MOLECULE TYPE: cDNA<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 7:<br>
CTAGGATCCA AGGCATTTAT GCGTGTCACT ACTCTC<br>
INFORMATION FOR SEQ ID NO: 8:<br>
(i) SEQUENCE CHARACTERISTICS: (A) LENGTH: 30 base pairs<br>
hh<br><br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear (ii) MOLECULE TYPE: cDNA<br>
(XI) SEQUENCE DESCRIPTION: SEQ ID NO: 8: ;TACTCGAGT TAGCACTTGA GCTCGATAGC<br>
[NFORMATION FOR SEQ ID NO: 9:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 17 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear (ii) MOLECULE TYPE: cDNA<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 9:<br>
:CCTTCTTTC CAAACCG<br>
[NFORMATION FOR SEQ ID NO: 10:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 17 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear (ii) MOLECULE TYPE: cDNA<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 10:<br>
GGGTGAGCCA CTGCCTC<br><br><br>
We claim:<br>
1.	An   Aspergillus host cell  for the expression of a heterologous protein product, which cell has been genetically modified in order to express significantly reduced levels of an Aspergillus neutral metalloprotease, Npll, having optimal proteolytic activity in the range of pH 6-8, as compared to a parental cell.<br>
2.	The host cell according to claim 1, which is a strain selected from the group consisting of Aspergillus oryzae, Aspergillus niger, AspergilIus nidulans, Aspergillus awamori, Aspergillus phoenicis, Aspergillus japonicus, Aspergillus foetus.<br>
3.	The host cell according to any of claims 1-2, which has been genetically modified at the structural or regulatory regions encoding the metalloprotease.<br>
4.	The host cell according to claim 3, which has been genetically modified by specific or random mutagenesis. PCR generated mutagenesis, site specific DNA deletion, insertion and/or substitution, gene disruption or gene replacement techniques, anti-sense techniques, or a combination thereof.<br>
5.	T he host cell according to any of claims 1-4, in which cell the level of expressed metalloprotease is reduced more than about 50%, preferably more than about 85%, more preferred more than about 90%, most preferred more than about 95%.<br>
6.	The host cell according to any of claims 1-4, which cell is essentially free of any metalloprotease activity.<br>
7.	A method of producing a heterologous protein product in the host cell of claim 1, which method comprises,<br><br>
(a)	introducing into said host cell a nucleic acid sequence encoding said protein product;<br>
(b)	cultivating in a suitable growth medium, the host cell of step (a); and<br>
(c)	isolating said heterologous protein product.<br><br>
8.	The method according to claim 7, in which the host cell is a strain selected from the group consisting of Aspergillus oryzae, Aspergillus niger, Aspergillus nidulans, Aspergillus awamori, Aspergillus phoenicis, Aspergillus japonicus, Aspergillus foetus.<br>
9.	The method according to any of claim 7-8, in which the host cell has been genetically modified at the structural or regulatory regions encoding the metalloprotease.<br><br>
10.	The method according to claim 9, in which the host cell has been genetically modified by specific or random mutagenesis, PCR generated mutagenesis, site specific DNA deletion, insertion and/or substitution, gene disruption or gene replacement techniques, anti-sense techniques, or a combination thereof.<br>
11.	The method according to any of claims 7-10, in which the level of metalloprotease expressed by the host cell is reduced more than 50%, preferably more than 85%. more preferred more than 90%, most preferred more than 95%.<br>
12.	The method according to any of claims 7-10, in which the product expressed by the host cell is essentially free of any metalloprotease activity.<br>
13.	The method according to any of claims 7-12, in which the protein product is an eucaryotic enzyme, such as insulin, growth hormone, glucagon, somatostatin, interferon, PDGF, factor VII, factor VIII, urokinase, EPO, chymosin, tissue plasminogen activator, or serum albumin.<br>
14.	The method according to any of claims 7-13, in which the protein product is a protein of fungal origin.<br>
15.	The method according to claim 14, in which the protein product is a fungal enzyme, in particular an amytolytic enzyme, such as an alpha-amylase, a beta-amylase, a glucoamylase, a beta-galactosidase, a cellulytic enzyme, a lipolytic enzyme, a xylanolyfc enzyme, a proteolytic enzyme, an oxidoreductase, such as a peroxidase or a laccase, a pectinase, or a cutinase.<br>
16.	The method according to any of claims 7-15, in which the protein product is a bacterial protein.<br><br>
17.	The method according to claim 16, in which the protein product is a bacterial enzyme, in<br>
particular an amylolytic enzyme, such as an alpha-amylase, a beta-amylase, a<br>
glucoamylase, a beta-galactosidase, a cellulytic enzyme, a lipolytic enzyme, a xylanolytic<br>
enzyme, a proteolytic enzyme, an oxidoreductase, such as a peroxidase or a iaccase, a<br>
pectinase, or a cutinase.<br>
18.	The method according to any of claim 7- 17, in which the protein product is a precursor<br>
protein, i.e. a zymogen, a hybrid protein, a protein obtained as a pro sequence or pre-pro<br>
sequence, or in unmaturated form.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTM3LW1hcy0xOTk2IGFic3RyYWN0IGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">537-mas-1996 abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTM3LW1hcy0xOTk2IGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">537-mas-1996 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTM3LW1hcy0xOTk2IGFzc2lnbm1lbnQucGRm" target="_blank" style="word-wrap:break-word;">537-mas-1996 assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTM3LW1hcy0xOTk2IGNsYWltcyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">537-mas-1996 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTM3LW1hcy0xOTk2IGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">537-mas-1996 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTM3LW1hcy0xOTk2IGNvcnJlc3BvbmRlbmNlIG90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">537-mas-1996 correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTM3LW1hcy0xOTk2IGNvcnJlc3BvbmRlbmNlIHBvLnBkZg==" target="_blank" style="word-wrap:break-word;">537-mas-1996 correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTM3LW1hcy0xOTk2IGRlc2NyaXB0aW9uIChjb21wbGV0ZSkgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">537-mas-1996 description (complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTM3LW1hcy0xOTk2IGRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">537-mas-1996 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTM3LW1hcy0xOTk2IGRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">537-mas-1996 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTM3LW1hcy0xOTk2IGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">537-mas-1996 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTM3LW1hcy0xOTk2IGZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">537-mas-1996 form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTM3LW1hcy0xOTk2IGZvcm0tMTkucGRm" target="_blank" style="word-wrap:break-word;">537-mas-1996 form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTM3LW1hcy0xOTk2IGZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">537-mas-1996 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTM3LW1hcy0xOTk2IGZvcm0tNC5wZGY=" target="_blank" style="word-wrap:break-word;">537-mas-1996 form-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTM3LW1hcy0xOTk2IGZvcm0tNi5wZGY=" target="_blank" style="word-wrap:break-word;">537-mas-1996 form-6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTM3LW1hcy0xOTk2IHBldGl0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">537-mas-1996 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="224298-a-method-of-identifying-a-pharmacologically-active-substance-and-a-dna-template-for-the-same.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="224300-an-enzyme-exhibiting-aminopeptidase-activity-and-a-dna-construct-comprising-a-dna-sequence-encoding-such-enzyme.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>224299</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>537/MAS/1996</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>47/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Nov-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>10-Oct-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>02-Apr-1996</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVOZYMES A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>KROGSHOEJVEJ 36, DK-2880 BAGSVAERD,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>LEHMBECK, JAN</td>
											<td>C/O. NOVO NORDISK A/S, NOVO ALLE, DK 2880 BAGSVAERD,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/80</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0284/95</td>
									<td>1995-03-20</td>
								    <td>Denmark</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/224299-an-aspergillus-host-cell-for-the-expression-of-a-heterologous-protein-product-and-a-method-for-producing-the-protein by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:53:06 GMT -->
</html>
