<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/224315-a-hirudin-precursor-comprising-a-signal-sequence-for-the-production-of-leu-hirudin by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:49:21 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 224315:A HIRUDIN PRECURSOR COMPRISING A SIGNAL SEQUENCE FOR THE PRODUCTION OF LEU-HIRUDIN</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A HIRUDIN PRECURSOR COMPRISING A SIGNAL SEQUENCE FOR THE PRODUCTION OF LEU-HIRUDIN</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A hirudin precursor comprising a signal sequence for the production of Leu-hirudin The present invention relates to a hirudin precursor comprising a signal sequence selected from the group comprising the signal sequences of the outer membrane protein of Serratia marcescens having SEQ ID No.25, the oprF protein of Pseudomonas fluorescens having SEQ ID No.26, and the fumarate reductase of Shewanella putrifaciens having SEQ ID No.28, onto which there is C-terminal attachment of the sequence of Leu-hirudin.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Signal sequences for preparing Leu-hirudin by secretion by E. coli into the culture medium<br>
The leech-derived product Refludan   shows good therapeutic properties in<br>
clinical trials (The Lancet, Vol. 353, p. 429 - 438 ). This leads to the conclu¬sion that larger amounts of the product are likely to be required in future.<br><br>
The   biological   active   ingredient   in   the   product   is   [Leu ,   Thr ]-63-<br>
desulfatohirudin which is described in European patent 0 324 712 and is called "Leu-hirudin" for short hereinafter.<br>
European patent 0 448 093 describes a process for preparing hirudin. The preferred embodiment of the patent comprises a hirudin whose N-terminal amino acid consists of alanine. Fusion of this hirudin to the signal sequence of a-cyclodextrin glycosyltransferase (CGTase) and transformation of an expression vector encoding this fusion protein, as described in the patent, into an E. coli secretor mutant make it possible to prepare Ala-hirudin with crude yields of more than 2 grams per liter. European patent 0 549 915 describes variants of Ala-hirudin with improved stability. Preparation of these variants using the E. coli secretor system results in yields of several grams per liter. The yields are thus distinctly higher than described by Dodt et al. for the hirudin variant HV1 ( FEBS LETTERS vol. 202 373-377 1986). A negligible increase in the yield compared therewith is described in US patent 5,573,929 by expressing the expression cassette via a pUC vector in a known manner in place of the pBR322-d6rived vector of Dodt et al. Bender et al. (Appl. Microbiol Blotechnol 34, p. 203 -207 1990) describe the secretion of Thr-hirudin, which is described in European patent 0 171 024, by Streptomyces lividans. However, the yields compared with the yields mentioned in European patents 0 448 093 and 0 549 915 are once again distinctly less. This also applies to expression in E. coli as found by P. de Taxis du Poet et al. for secretion of the hirudin variant HV1 via the signal sequence Ompa of E. coli. The authors find yields of 300 mg/ I hirudin in the periplasm and about 40 mg/l in the cell supernatant. The expression in insect cell systems, which is also described in the article, is low (400 ig/l).<br>
Yields  which   can   be  achieved  with   the  yeast   expression   systems Hansenula polymorpha or Pichia pastoris come closest to the yields<br><br>
described in European patents 0 448 093 and 0 549 915, in contrast to the levels achieved with S. cerevisiae.<br>
Rosenfeld etal. ( Protein Expression and Purification 8 , 476 - 482, 1996 ) describe the expression and secretion of hirudin by the yeast Pichia pastoris. Yields of about 1.5 g/l of culture broth are achieved in this case. A similar order of magnitude can be achieved with the yeast Hansenula polymorpha (Appl. Microbiol. Biotechnol. 44, 377 - 385 1995). However, a considerable disadvantage of such expression systems is that the fermen¬tation times are distinctly longer than for the E. co//system. It would thus be advantageous if Leu-hirudin could, like Ala-hirudin, be prepared by secretion by E. coli.<br>
However, this is not possible with the system described in European patent 0 448 093. For this reason, it is proposed in the patent to extend the Leu-hirudin sequence by the tripeptide Ala-Thr-Arg to produce a pre-Leu-hirudin which is finally converted after reaction with trypsin into the native active ingredient Leu-hirudin. Following this proposal in a shaken flask experiment results in crude yields which are distinctly worse than described for Ala-hirudin. Thus, at first sight, no distinct advantage is clearly evident compared with later yeast expression systems.<br>
The object on which the present invention was based was accordingly to prepare a fusion protein in which the combination of signal sequence and Leu-hirudin permits direct processing to Leu-hirudin and subsequent sec¬retion of native Leu-hirudin in high yields by E. coli. This is prerequisite for developing a process having an advantageous effect on the costs of production of Refludan both in the fermentation and in the subsequent purification owing to the improved initial hirudin concentration. It has now been found, surprisingly, that signal sequences which permit direct secretion of Leu-hirudin by E. coli exist, and that in this case the secretion is in fact observed to be more efficient than described in European patent 0 448 093. It is thus possible to develop a process which makes large amounts of Leu-hirudin available without great expenditure. The invention relates thereto.<br>
In order to find advantageous signal sequences, a method of PCR-assisted signal sequence screening is introduced. This method uses the DNA encoding the protein of interest as template and a defined reverse PCR<br><br>
primer, and variable forward primers which permit the synthesis of a DNA section which encodes a signal sequence coupled to a gene of interest. The reaction proceeds as shown in the scheme depicted in Figure 1. It is clear to the skilled worker that the number of reaction steps may vary according to the length of the signal sequence to be synthesized. Short signal sequences can be prepared with one reaction step, and longer sequences with two, three or more reactions. In addition, the number of reactions also depends on the apparatus used to synthesize the oligo¬nucleotides used as primers. The signal peptide gene fusion synthesized in this way can then be cleaved specifically with the enzymes recognizing restriction sites 1 and 2 and be inserted into a correspondingly opened expression vector. The system becomes of general significance when hirudin is chosen as the gene of interest. Variable selection of the N-terminal amino acid of hirudin is moreover possible. Although this has a certain effect on the binding of hirudin to thrombin (change in the binding constant), the inhibitory effect of hirudin in relation to the thrombin activity remains measurable.<br>
The patent EP-B1 0 448 093 describes the secretion of hirudin into the culture supernatant. The hirudin concentration therein can be determined directly via the well-known thrombin inhibition assay. The hirudin concen¬tration is a direct measure of the efficiency of secretion and thus of the elimination of the signal sequence. The patent describes, however, that, for example, hirudin starting with the amino acid leucine cannot be efficiently released into the supernatant via the signal sequence of the CGTase. It is now possible, using the method described above, to search for signal sequences which effectively permit this. It is now possible in a similar way to investigate the secretion of hirudins starting with one of the other 19 amino acids. This results in each case in a spectrum of signal sequences which permit in a model way the efficient processing of the carboxy-terminal amino acid of the signal peptide and the peptide residue attaching thereto. It is thus possible to make a preselection of signal peptides for efficient secretion of any desired protein into the periplasm and thus increase the chances of developing an advantageous process for preparing a protein. The invention likewise relates thereto. The process can be expedited or automated by shaking the transformation mixture of ligand mixture and competent cells as liquid culture in a selection medium overnight and, the next day, inoculating with an aliquot of the cells as described in example 11 medium which contains inducer to carry out the<br><br>
induction, but centrifuging most of the culture and freezing out the cell pellet. If hirudin activity is found on expression, the corresponding expression plasmid can be re-isolated from the cells, linearized and separated by gel electrophoresis from any autoligation products. The linear plasmid DNA is then religated and transformed anew into the host strain. It is then possible for individual colonies to be isolated and tested for their expression efficiency. It is possible to proceed in this case in such a way that the process meets the criteria of pharmaceutical approval.<br>
A further advantage of the procedure is that it is easy to investigate different variants of a signal peptide, as arise in the course of evolution by exchange of amino acids between individual species, side by side for their ability to secrete a hirudin efficiently.<br>
The process is also advantageous by comparison with the use of computer programs as described by Nielsen et al. (Protein Engineering 10, 1-6, 1997), with the aid of which it is possible to predict cleavage sites between signal sequence and a protein of interest. However, it is found that the predictions made therewith are not correct in every case, so that advan¬tageous combinations may easily be overlooked. In addition, there is no relation between the prediction of correct processing and the yield which can actually be achieved.<br>
One aspect of the invention is a hirudin precursor comprising a signal sequence selected from the group comprising the signal sequences of the outer membrane protein of Serratia marcescens, the oprF protein of Pseudomonas fluorescens, the lamb B protein of Escherichia coli (encoded by the lambda receptor (lamB) gene), and the fumarate reductase of Shewanella putrifaciens, preferably selected from the group comprising the signal sequence of the outer membrane protein of Serratia marcescens and the fumarate reductase of Shewanella putrifaciens, onto which there is C-terminal attachment of the sequence of Leu-hirudin.<br>
Another aspect of the invention is a process for preparing Leu-hirudin, in which a hirudin precursor as described above occurs as intermediate, wherein<br>
(a)      an expression plasmid comprising a DNA sequence coding for the hirudin precursor is prepared;<br><br>
(b)	the expression plasmid from (a) is expressed in a suitable E. coli cell;<br>
(c)	the hirudin precursor is secreted from E. coli and simultaneously processed; and<br>
(d)	the Leu-hirudin is isolated directly from the culture medium.<br>
Likewise an aspect of the invention is the use of a hirudin precursor as described above for preparing Leu-hirudin, preferably In a process as described above.<br>
A further aspect of the invention is a process for finding a suitable signal peptide for secretory expression of any desired protein in E. coli, wherein<br>
(a)	hirudin or a hirudin derivative which has an antithrombotic effect and which has a defined amino acid aax at its N terminus which is connected N-terminally to a signal peptide to be tested is expressed in E. coli;<br>
(b)	the expression rate is determined by measuring the hirudin activity in the culture supernatant;<br>
(c)	steps (a) and (b) are repeated with various signal peptides;<br>
(d)	a suitable signal peptide is selected by comparing the expression rates represented by the hirudin activities found in step (b).<br>
Likewise an aspect of the invention is the use of hirudin or a hirudin derivative which has an antithrombotic effect and which has a defined amino acid aax at its N terminus for finding a signal peptide which makes it possible to secrete efficiently a precursor protein consisting of the signal peptide and any other desired protein with the N-terminal amino acid aax with simultaneous elimination of the signal peptide from E. coli, in particular where aax is leucine.<br>
A further aspect of the invention is a process for preparing any desired protein by secretory expression in E. coli, wherein<br>
(a) a suitable signal peptide is found by the process for finding a suitable signal peptide;<br><br>
(b)	a nucleic acid construct coding for a precursor protein consisting of<br>
the suitable signal peptide from (a) and the desired protein is<br>
expressed in E. coli; and<br>
(c)	the desired protein is isolated from the culture supernatant,<br>
5<br>
in particular where the N-terminal amino acid of the desired protein is leucine, and the expression takes place via a nucleic acid construct in which the sequence comprising the signal peptide codes for a signal peptide selected from the group comprising the outer membrane protein of 10 Serratia marcescens, the oprF protein of Pseudomonas fluorescens, the lamb B protein of Escherichia coli, and the fumarate reductase of Shewanella putrifaciens.<br>
The synthesis of signal sequences which permit efficient synthesis and 15 secretion of Leu-hirudin is to be described by way of example. Likewise described is the synthesis of other signal sequences which did not lead to the objective or gave worse results in relation to the yield. The examples are intended in this connection to explain the concept of the invention on the basis of the selection of signal sequences on the basis of Leu-hirudin 20     but not to be restricted thereto.<br>
The processes described can be used for the purification of Refludan; this is described, for example, in example 11.<br>
25 Example 1: Synthesis of a fusion gene coding for a fusion protein consisting of Leu-hirudin and the signal sequence of the outer membrane protein from Serratia marcescens<br>
The expression plasmid used is the vector pJF118 which is described in 30     European patent 0 468 539, in Figure 1, because this is identical in its basic structure to the vector pCM7053 described in European patent 0 448 093.<br>
The template used is the plasmid pK152 which is mentioned in Example 1 35     in European patent 0 448 093 and which harbors the hirudin sequence corresponding to European patent 0 171 024.<br>
The membrane protein has been described by Braun, G. and Cole, S.T: (Mol. Gen. Genet. 195, 321-328 , 1984).<br><br>
To synthesize the required DNA section, three oligonucleotide sequences are prepared.<br><br>
The primer hybridizes with the region 227 -210 bp of the hirudin gene depicted in Table 1.<br>
Primer smompafi has the sequence:<br><br>
The primer hybridizes with nucleotides 1-19 of the hirudin sequence depicted in Table 1. The hybridizing part of the primer sequence is symbolized by small letters. The remainder of the sequence hybridizes with the region 229 bp - 263 bp of the sequence published by Braun, G. and Cole, S.T. (Mol. Gen. Genet. 195, 321-328, 1984).<br>
Primer smompaf2 has the sequence:<br><br>
The primer sequence hybdridizes from the 13 bp position onwards with the 201 bp - 245 bp sequence published by Braun and Cole and thus overlaps with the primer sequence smompaf2. The 1-12 position of the primer contains a recognition site for the restriction enzyme EcoRl and, adjoining, 6 T nucleotides in order to make recognition by the enzyme possible.<br>
In a standard PCR (such as, for example, 94°C: 10",50°C: 30",72°C: 45", 25 cycles) with DNA of the plasmid pK152, which harbors the sequence described in Table 1, as template, and the primers hirrev and smompafi, the hirudin sequence is extended by the bacterial partial signal sequence. The reaction product is then reacted in a second PCR as template with the primers hirrev and smompaf2 under the same conditions. The reaction<br><br>
product is a DNA fragment which codes for a fusion protein which consists of the hirudin sequence extended by the desired signal sequence. At the 5' end there is the recognition site for the restriction enzyme EcoRI and at the 3' end there is the recognition site for the enzyme /-//ndlll.<br>
The reaction product from the second PCR is reacted in a double-digestion mixture with the two restriction enzymes and is inserted as EcoRI/H/'/idlll fragment into the vector DNA, which has been opened with the two enzymes, in a T4 DNA ligase reaction. Competent cells of the E. coli strain Mel 061 or the secretor mutant WCM100 are transformed with the ligation mixture and grown under selection pressure on ampicillin-containing plates. The next morning, expression as described in Example 6 is then compared with Ala-hirudin expression using the E. co//strain WCM100 / pCM7053. It is found that the expression obtained is about 1.5 times better than in the comparative test.<br>
Example 2: Synthesis of the fusion protein of Leu-hirudin and the signal sequence of the oprF gene product from Pseudomonas fluorescens<br>
Construction takes place in accordance with the scheme described in Example 1 with the exception that, in place of the primers smompafi / f2, there is use of two new primers which, in terms of their specificity for the hirudin gene and the sequence for recognition by the restriction enzyme EcoRI, have the same characteristics as the smompa primers but code for the required signal sequence of the oprF gene (De, E. et ai: FEMS Microbial. Lett. 127, 267-272 , 1995).<br>
Primer pfufi has the sequence:<br><br>
Primer pfuf2 has the sequence:<br><br>
In this case, the primer pfufi is used in accordance with Example 1 in PCR1 and primer pfuf2 is used correspondingly in PCR2. The expression is carried out by comparison with Ala-hirudin expression using the E. coli<br><br>
strain WCM100 / pCM7053. It is found that the expression obtained is about 1.1 times better than in the comparative test. After fractionation by gel electrophoresis in the SDS-PAGE system, the hirudin band is isolated and the N-terminal sequence of the hirudin is determined. It is found that the sequence is completely intact and starts with the amino acid leucine. This result is surprising because the program for identifying the putative signal peptidase recognition site predicts an extension of the hirudin by valine.<br>
Example 3: Synthesis of the fusion protein of Leu-hirudin and the signal sequence of the lamB gene product from E. coli<br>
Construction takes place in accordance with the scheme described in Example 1 with the exception that, in place of the primers smompafi / f2, there is use of two new primers which, in terms of their specificity for the hirudin gene and the sequence for recognition by the restriction enzyme EcoRI, have the same characteristics as the smompa primers but code for the required signal sequence of the lamb gene (Clement, J.M. and Hofnung, M.: Cell 27, 507-514, 1981).<br>
Primer lambbfl has the sequence:<br><br>
Primer Iambbf2 has the sequence:<br>
5'ttttttgaat tcATGATGAT TACTCTGCGC AAACTTCCTC TGGCGGTTGC CGTCGCAGC 3'<br>
[SEQ ID NO: 7]<br>
In this case, the primer lambbfl 1 is used in accordance with Example 1 in PCR1 and the primer Iambbf2 is correspondingly used in PCR2. The expression is carried out by comparison with the Ala-hirudin expression using the E. co//strain WCM100 / pCM7053. It is found that the expression obtained is at the same level as in the comparative test. After fractionation by gel electrophoresis in the SDS-PAGE system, the hirudin band is isolated, and the N-terminal sequence of the hirudin is determined. It is found that the sequence is completely intact and starts with the amino acid leucine. This result is surprising because the program for identifying the<br><br>
putative signal peptidase recognition site does not predict correct processing of hirudin.<br>
Exannple 4: Synthesis of the fusion protein of Leu-hirudin and the signal sequence of the precursor of fumarate reductase flavoprotein subunit from Shewanella putrefaciens<br>
Construction takes place in accordance with the scheme described in Example 1 with the exception that, in place of the primers smompafi / f2, there is use of two new primers which, in terms of their specificity for the hirudin gene and the sequence for recognition by the restriction enzyme EcoRI, have the same characteristics as the smompa primers but code for the required signal sequence from Shewanella putrefaciens (Pealing S.L. et al.: Biochemistry 31, 12132-12140, 1992). Since the publication describes only the protein sequence, the amino acid sequence is translated in accordance with the codon tables into a DNA sequence so that the sequence which emerges for the<br>
primer spfccfl is as follows:<br><br>
Primer spfccf2 has the sequence:<br>
5'ttttttgaat tcATGAAAAA AATGAACCTG GCTGTTTGCA TCGCTACCCT GATGGGTACC 3'<br>
[SEQ ID NO: 9]<br>
In this case, the primer spfccfl is used in accordance with Example 1 in PCR1 and primer spfccf2 is used correspondingly in PCR2. The expression is carried out by comparison with Ala-hirudin expression using the E. coli strain WCM100 / pCM7053. It is found that the expression obtained is about 1.5 times better than in the comparative test. After fractionation by gel electrophoresis in the SDS-PAGE system, the hirudin band is isolated and the N-terminal sequence of the hirudin is determined. It is found that the sequence is completely intact and starts with the amino acid leucine. This result is surprising because the program for identifying the putative signal peptidase recognition site predicts processing on the carboxyl side of cysteine in position 6 of the hirudin sequence.<br><br>
Example 5: Synthesis of the fusion protein of Leu-hirudin and the signal sequence of the (3-lactamase precursor from pBR322<br>
Construction takes place in accordance with the scheme described in Example 1 with the exception that, in place of the primers smompafi / f2, there is use of two new primers which, in terms of their specificity for the hirudin gene and the sequence for recognition by the restriction enzyme EcoRI, have the same characteristics as the smompa primers but code for the required signal sequence of the p-lactamase precursor protein (Sutcliffe J.G.; Cold Spring Harbor Symp. Quant. Biol. 43:77-90 (1978)).<br>
Primer blatf 1 has the following sequence:<br><br>
Primer blatf2 has the sequence:<br><br>
In this case, the primer blatf 1 is used in accordance with Example 1 in PCR1 and primer blatf2 is used correspondingly in PCR2. The expression is carried out by comparison with Ala-hirudin expression using the E. coli strain WCM100 / pCM7053. It is found that the expression yield obtained is only 50%-90% of the yield obtained in the comparative test. After fractionation by gel electrophoresis in the SDS-PAGE system, the hirudin band is isolated and the N-terminal sequence of the hirudin is determined. It is found that the sequence is completely intact and starts with the amino acid leucine. This result was predicted by the program for identifying the putative signal peptide recognition site.<br>
Example 6: Synthesis of the fusion gene of Leu-hirudin and the signal sequence of the precursor of alkaline phosphatase from E. coli<br>
Construction takes place in accordance with the scheme described in Example 1 with the exception that, in place of the primers smompafi / f2, there is use of two new primers which, in terms of their specificity for the<br><br>
hirudin gene and their sequence for recognition by the restriction enzyme EcoRI, have the same characteristics as the smompa primers but code for the required signal sequence of the alkaline phosphatase protein from E. coli (Shuttieworth, H., Taylor, J. and Minton, N. Nucleic Acids Res. 14 (21), 8689(1986)).<br>
Primer linkphoafl has the following sequence:<br><br>
Primer linkphoaf2 has the sequence:<br><br>
The two primers are optimized in terms of the codon choice for E. coli, i.e. they do not correspond entirely to the natural sequence of the starting gene.<br>
In this case, the primer linkphoafl is used in accordance with Example 1 in PCR1 and primer linkphoaf2 is used correspondingly in PCR2. The expression is carried out by comparison with Ala-hirudin expression using the E. CO//strain WCM100 / pCM7053. It is found that the expression yield obtained is only a fraction of the yield obtained in the comparative test. After fractionation by gel electrophoresis in the SDS-PAGE system, the hirudin band is isolated and the N-terminal sequence of the hirudin is determined. It is found that the sequence is completely intact and starts with the amino acid leucine. This result was predicted by the program for identifying the putative signal peptidase recognition site. However, the poor yield is surprising.<br>
Example 7: Synthesis of the fusion gene of Leu-hirudin and the signal sequence of the precursor of the alkaline phosphatase from E. fergusonii<br>
Construction takes place in accordance with the scheme described in Example 1 with the exception that, in place of the primers smompafi / f2, there is use of two new primers which, in terms of their specificity for the hirudin gene and their sequence for recognition by the restriction enzyme<br><br>
EcoRI, have the same characteristics as the smompa primers but code for the required signal sequence of the alkaline phosphatase protein from E. fergusonii (Du Bose, R.F. and HartI, D.L. Mol. Biol. Evol. 7, 547-577 (1990)).<br>
This signal sequence differs at five positions from the alkaline phosphatase from E. coli.<br>
Primer fergusf 1 has the following sequence:<br><br>
Primer fergusf2 has the sequence:<br><br>
The two primers are optimized in terms of the codon choice for E. coli, i.e. they do not correspond entirely to the natural sequence of the starting gene. In this case, the primer fergusf 1 is used in accordance with Example 1 in PCR1 and primer fergusf2 is used correspondingly in PCR2. The expression is carried out by comparison with Ala-hirudin expression using the E. CO//strain WCM100 / pCM7053. It is found that the expression yield obtained is only a fraction of the yield obtained in the comparative test. It is a further approximately 50% lower than observed with the construct of signal peptide from E. coli alkaline phosphatase and Leu-hirudin.<br>
Example 8: Synthesis of the fusion gene of Leu-hirudin and the signal sequence of the precursor of cyclodextrin glucanotransferase from Paenibacillus macerans<br>
Construction takes place In accordance with the scheme described in Example 1 with the exception that, in place of the primers smompafi / f2, there is use of two new primers which, in terms of their specificity for the hirudin gene and their sequence for recognition by the restriction enzyme EcoRI, have the same characteristics as the smompa primers but code for the required signal sequence of the cyclodextrin glucanotransferase gene from   Paenibacillus  macerans  (Takano,  T.,   Fukuda,   M.,   Monma,   M.,<br><br>
Kobayashi, S., Kainuma, K. and Yamane, K. J. Bacteriol. 166, 1118-1122 (1986)).<br>
Primer baccdgf 1 has the following sequence:<br><br>
Primer baccdgf2 has the sequence:<br><br>
In this case, the primer baccdgfl is used in accordance with Example 1 in PCR1 and primer baccdgf2 is used correspondingly in PCR2. The expression is carried out by comparison with Ala-hirudin expression using the E. CO//strain WCM100 / pCM7053. It is found that the expression yield obtained is about one quarter of the yield obtained in the comparative test. The synthesized hirudin behaves like Leu-hirudin in the thrombin inhibition assay. This means that the signal peptide has been correctly processed. This does not correspond to the expectation from the theoretical analysis, which indicated an extension of 8 amino acids or, alternatively, a truncation by two amino acids at the N terminus.<br>
Example 9: Synthesis of the fusion gene from Leu-hirudin and the signal sequence of the E. coli PCFO20 fimbrillin precursor protein (fotA)<br>
Construction takes place in accordance with the scheme described in Example 1 with the exception that, in place of the primers smompafi / f2, there is use of two new primers which, in terms of their specificity for the hirudin gene and their sequence for recognition by the restriction enzyme EcoRI, have the same characteristics as the smompa primers but code for the required signal sequence of the E. coli PCFO20 fimbrillin precursor protein (Viboud, G.I., Jonson, G., Dean-Nystrom, E. and Svennerholm, A.M. Infect. Immun. 64 (4), 1233-1239 (1996)).<br>
Primer pcf1-ala has the following sequence:<br><br><br>
Primer p-pcf2 has the sequence:<br>
5' TTTTGGGAAT TCATGAAAAA GAGAATTATG TCTGTGGCTG TGGTTTCAGC TTTAGTAAGG   3' [SEQIDNO.: 19]<br>
In this case, the primer pcfl-ala is used in accordance with Example 1 in PCR1 and the primer p-pcf2 is used correspondingly in PCR2. The expression is carried out by comparison with the Ala-hirudin expression using the E. co//strain WCM100 / pCM7053. It is found that the expression yield obtained is about 40% of the yield obtained in the comparative test.<br>
Example 10: Synthesis of the fusion gene of Leu-hirudin and the signal sequence of S. typhimurium outer membrane protein (fimD)<br>
Construction takes place in accordance with the scheme described in Example 1 with the exception that, in place of the primers smompafi / f2, there is use of two new primers which, in terms of their specificity for the hirudin gene and their sequence for recognition by the restriction enzyme EcoRI, have the same characteristics as the smompa primers but code for the required signal sequence of the S. typhimurium outer membrane protein ( Rioux, C.R., Friedrich, M.J. and Kadner, R.J.; J. Bacteriol. 172 (11), 6217-6222 (1990)).<br>
Primer styfimf 1 has the following sequence:<br><br><br>
In this case, the primer styfimfl is used in accordance with Example 1 in PCR1 and the primer styfimf2 is used correspondingly in PCR2. The expression is carried out by comparison with the Ala-hirudin expression using the E. coil strain WCM100 / pCIVITOSS. It is found that the expression yield obtained is about 10% of the yield obtained in the comparative test.<br>
Example 11: Expression in E. coli<br>
The example describes expression of hirudin. For this purpose, 1-5 ml portions of LB medium which contains 25 mg/ml ampicillin and 0.5-2 mM IPTG (isopropyl p-D-thiogalactopyranoside) are inoculated with cells of a transformant and shaken at 220 rpm in an incubating shaker at 28°C for about 20 hours. Subsequently, after determination of the optical density, the cell suspension is centrifuged and hirudin is determined in the clear supernatant.<br>
Expression of the Ala-hirudin described in European patent 0 448 093 via the plasmid pCM7053 in the secretor mutant WCM100 described in the patent is carried out in parallel with expression of Refludan. This makes direct comparison of the expression rate possible.<br>
Expression on a larger scale can take place as described in US patent 5,616,476. Refludan can then be purified by the methods described in Examples 5 and 6 in this patent.<br>
Example 12 : Determination of the hirudin concentration<br>
Determination of the hirudin concentration is carried out by the method of GrieBbach et al. (Thrombosis Research 37, 347-350 , 1985). For this purpose, defined amounts of a Refludan standard are included in the series of measurements to construct a calibration plot. It is thus possible to state the yield directly in mg/l.<br>
Table 1: Hirudin-encoding DNA sequence with translation into amino acids<br>
1   CTTACGTATACTGACTGCACTGAATCTGGTCAGAACCTGTGCCTGTGCGAAGGATCTAAC  60 LTYTDCTESGQNLCLCEGSN<br><br><br><br><br><br><br><br>
WE CLAIM:<br>
1.	A hirudin precursor comprising a signal sequence selected from the group<br>
comprising the signal sequences of the outer membrane protein of Serratia<br>
marcescens having SEQ ID No.25, the oprF protein of Pseudomonas<br>
fluorescens having SEQ ID No.26, and the fumarate reductase of Shewanella<br>
putrifaciens having SEQ ID No.28, onto which there is C-terminal attachment<br>
of the sequence of Leu-hirudin.<br>
2.	The hirudin precursor as claimed in claim 1, where the signal sequence is<br>
selected from the group comprising the signal sequence of the outer membrane<br>
protein of Serratia marcescens having SEQ ID No.25, and the fumarate<br>
reductase of Shewanella putrifaciens having SEQ ID No. 28.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDM5NS1jaGUgYWJzdHJhY3QgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0395-che abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDM5NS1jaGUgYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-0395-che abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDM5NS1jaGUgY2xhaW1zIGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-0395-che claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDM5NS1jaGUgY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0395-che claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDM5NS1jaGUgY29ycmVzcG9uZGVuY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0395-che correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDM5NS1jaGUgY29ycmVzcG9uZGVuY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-0395-che correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDM5NS1jaGUgZGVzY3JpcGl0aW9uKGNvbXBsZXRlZCkgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0395-che descripition(completed) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDM5NS1jaGUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKSBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-0395-che description (complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDM5NS1jaGUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-0395-che description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDM5NS1jaGUgZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-0395-che drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDM5NS1jaGUgZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0395-che form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDM5NS1jaGUgZm9ybS0xMy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-0395-che form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDM5NS1jaGUgZm9ybS0xOS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-0395-che form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDM5NS1jaGUgZm9ybS0yNi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-0395-che form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDM5NS1jaGUgZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0395-che form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDM5NS1jaGUgZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0395-che form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDM5NS1jaGUgb3RoZXJzIGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-0395-che others duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDM5NS1jaGUgb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0395-che others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDM5NS1jaGUgcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0395-che pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDM5NS1jaGUgcGV0aXRpb24ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-0395-che petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="224314-free-carboline-drug-in-particulate-form-and-method-of-preparation-thereof.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="224316-surfactant-system-containing-phosphate-ester-and-alkoxylated-ligno-sulfonate.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>224315</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2002/395/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>47/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Nov-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>10-Oct-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>14-Mar-2002</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SANOFI-AVENTIS DEUTSCHLAND GmbH</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>65929 FRANKFURT.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>HABERMANN, PAUL</td>
											<td>ROSSERTSTRASSE 35 65817 EPPSTEIN.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>BENDER RUDOLF</td>
											<td>KELKHEIMERSTRASSE 37, 65812 BAD SODEN,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>CO7K14/815</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP00/08537</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-09-01</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>199 44 870.1</td>
									<td>1999-09-18</td>
								    <td>Germany</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/224315-a-hirudin-precursor-comprising-a-signal-sequence-for-the-production-of-leu-hirudin by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:49:22 GMT -->
</html>
