<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/224344-a-method-for-identifying-drug-non-target-biomolecules by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:42:35 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 224344:A METHOD FOR IDENTIFYING DRUG NON-TARGET BIOMOLECULES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR IDENTIFYING DRUG NON-TARGET BIOMOLECULES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method, comprising: i)contacting a capture compound with a sample containing biomolecules to effect capture of biomolecules in the sample, where the capture compound has formula:</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>W.T.O. APPLICATION<br>
FORM 2<br>
THE PATENTS ACT, 1970 (39 of 1970)<br>
&amp; <br>
THE PATENS RULES, 2003<br>
COMPLETE SPECIFICATION<br>
[See section 10, Rule 13]<br>
CAPTURE COMPOUNDS, COLLECTIONS THEREOF AND METHODS FOR ANALYZING THE PROTEOME AND COMPLEX COMPOSITIONS;<br>
HK PHARMACEUTICALS, INC., A<br>
CORPORATION	ORGANIZED	AND<br>
EXISTING UNDER THE LAWS OF UNITED<br>
STATES OF AMERICA, WHOSE ADDRESS<br>
IS 840 MEMORIAL DRIVE, CAMBRIDGE, MA<br>
02139, U.S.A and KOSTER, HUBERT A<br>
GERMAN	NATIONAL	OF	VILLA<br>
WELLINGTONS, VIA CANTONALE 84, CH-6918 FIGINO, U.S.A.<br>
THE	FOLLOWING	SPECIFICATION<br>
PARTICULARLY	DESCRIBES	THE<br>
INVENTION AND THE MANNER IN WHICH IT IS TO BE PERFORMED. 1<br><br>
-2-<br>
RELATED APPLICATIONS<br>
Priority is claimed herein to U.S. provisional patent application no. 60'441,398, filed January 16, 2003, to Koster et al., entitled "CAPTURE COMPOUNDS, COLLECTIONS THEREOF AND METHODS FOR ANALYZING THE PROTEOME AND COMPLEX COMPOSITIONS." For U.S.<br>
national stage purposes and where appropriate, the above-referenced application is incorporated herein by reference in its entirety. <br>
FIELD<br>
Provided herein are compounds and methods using the compounds to specifically and selectively analyze biomolecules. In particular, the compounds and methods are useful for analyzing the proteome. BACKGROUND<br>
Understanding the basis of disease and the development of therapeutic and preventative treatments has evolved over the last century from empirical observation and experimentation to genome wide mutation scanning. The revolution in genomics has provided researchers with the tools to look for a genomic basis for disease. The Human Genome effort has generated a raw sequence of the 3 billion base pairs of the human genome and revealed about 35,000 genes. Genetic variations amongst different individuals and in and in between populations are being studied in order to determine the association with the predisposition to disease or the correlation to drug efficacy and/or side effects. The promise of personalized medicine based on a panel of genetic markers has tantalized the healthcare community and provides an important goal for those focused on providing diagnostic and treatment options for healthcare providers and patients.<br>
With the development of a variety of tools in molecular biology, such as nucleic amplification methods, cloning and expression systems and methods, disease analysis has been based on a genomics, or bottom up, approach. This approach presumes that a genetic change or set of changes<br><br>
-3-<br>
will have a long reaching effect on protein function by affecting mRNA<br>
transcription or protein structure and function.<br>
Technologies have been developed to analyze single nucleotide polymorphisms (SNPs) in an industrial scale (e.g., MassARRAYTM and the MassARRAYÂ® system, Sequenom, Inc., San Diego, CA) and in pooled samples to study the frequency of SNPs in populations of various gender, ethnicity, age and health condition. The ultimate goal of these efforts is to understand the etiology of disease on the molecular level (e.g., based on genetic variances (pharmacogenomics)), to develop diagnostic assays and effective drugs with few or no side-effects.<br>
Genomics has fallen short of the original expectation that this strategy could be used to stratify a population relative to a defined phenotype, including differences between normal and disease patient population or populations. Although single genetic markers have been found to be associated with or cause or predict a specific disease state, genomic information may not be sufficiient to stratify individual populations by of the association of an SNP (or SNPs) with a given disease, drug side-effect or other target phenotype. Because of the large number of potential targets and regulatory signals that affect protein translation, it is not sufficient to establish the differential expression profiles of messenger RNA in comparing phenotypes or populations, such as healthy and disease states, or such as the analyses using expression DNA chips (e.g., GeneChip'" technology, Affymetrix, Inc., Santa Clara, CA; LifeArray'" technology, Incyte Genomics, Inc., Palo Alto, CA). The metabolic activities in a cell are not performed by mRNA but rather by the translated proteins and subsequently posttranslationally modified products, such as the alkylated, glycosylated and phosphorylated products.<br>
The study of proteomics encompasses the study of individual proteins and how these proteins function within a biochemical pathway. Proteomics<br>
also includes the study of protein interactions, including how they form the<br>
architecture that constitutes living cells. In many human diseases such as<br><br>
-4-<br>
cancer, Alzheimer's disease, diabetes as well as host responses to infectious<br>
diseases, the elucidation of the complex interactions between regulatory<br>
proteins, which can cause diseases, is a critical step to finding effective treatment. Often, SNPs and other nucleic acid mutations occur in genes whose products are such proteins as (1) growth related hormones, (2) membrane receptors for growth hormones, (3) components of the transmembrane signal pathway and (4) DNA binding proteins that act on transcription and the inactivation of suppressor genes (e.g. p53) causing the onset of disease.<br>
Complex protein mixtures are analyzed by two-dimensional (2D) gel electrophoresis and subsequent image processing to identify changes in the pattern (structural changes) or intensity of various protein spots. Two-dimensionsl gel electrophoresis is a laborious, error-prone method with low reproducibility and cannot be effectively automated. This gel technology is unable to effectively analyze membrane proteins.   Further, the resolution of 2D gels is insufficient to analyze the profile of all proteins present in a mixture.<br>
Available protein chips are limited by their ability to specifically capture hydrophobic and membrane proteins, which are frequently targets of drug development. Once bound to the chip, proteins are highly unstable and their structures often do not reflect the true conformation found under physiological conditions.<br>
Proteins form the important structural and functional machinery of the cell, and are the molecular entities with which nearly all of today's marketed drugs interact. Proteins are thus drug targets. Most pharma companies are investing heavily to extract truly promising drug targets from their sea of unvalidated targets derived from gene-based approaches. Typically the mechanism of action defining how drugs act upon their targets is poorly understood; for some marketed drugs the target is not even known. Furthermore, identifying "non-target" proteins with which the drug interacts to<br>
trigger side effects has been especially elusive. It is believed that side effects<br><br>
-5-<br>
of many drugs could be diminished with a greater understanding of the mechanism of action involving their target and the non-target proteins.<br>
Drug programs are discontinued for a variety of reasons {e*g., lack of efficacy compared to placebo), but about half of the terminations relate to clinical safety and toxicity. As a result, the developments of many ill-chosen lead drug compounds are halted late in clinical trials after many years and millions of dollars have been spent. Compounding the financial problems caused by toxicity, the long duration of drug development also substantially reduces the length of patent protection.<br>
Adverse side effects from drugs result in more than two million hospitalizations and more than 100,000 deaths each year. Many major drugs have severe toxic side effects.<br>
â¢	The widely prescribed psoriasis drugs methoxetrate and cyclosporine can cause severe liver and kidney damage and are thus rarely prescribed for more than one year.<br>
â¢	Approximately $ 13 billion has been spent so far in product injury and class action litigation connected with the withdrawal of the fen-phen weight loss drug combination.<br>
â¢	Substantial liabilities were also associated with the hepatotoxicity of the diabetes drug Rezulin (Troglitazone), which was prescribed 2    . million times and resulted in 398 deaths before its withdrawal from the market; 8700 law suits are being filed.<br>
â¢	Baycol, a cholesterol-lowering statin taken by 700,000 Americans, was removed from the market due to reports of a sometimes fatal muscle-related adverse reaction (rhabdomyolysis) and 31 deaths in the USA. Projected annual Baycol revenues prior to the recall were approximately $1 billion.<br>
â¢	Sales growth of Celebrex and Vioxx, blockbusters for the treatment of<br>
arthritis, has also been negatively affected by reports of a potential link<br>
to heart problems.<br>
Thus, there is a need to reduce time and costs of drug development by<br><br>
-6-<br>
(a) accelerating the hit-to-drug selection process by filtering out those hits likely to trigger side effects and (b) re-engineering drug chemical structure based on the knowledge of drug-target and drug-non-target interactions, reducing or eliminating the undesired interactions.<br>
There is also a need to develop technologies for analysis of the proteome that allow scaling up to industrial levels with the features of an industrial process: high accuracy, reproducibility and flexibility in that the process is high-throughput, automatable and cost-effective   There is a need to develop technologies that permit probing and identification of proteins and other biomolecules in their native conformation using automated protocols and systems therefor. In particular, there is a need to develop strategies and technologies for identification and characterization of hydrophobic proteins under physiological conditions. <br>
SUMMARY<br>
Provided herein are methods, capture compounds (also referred to herein as capture agents) and collections thereof for analysis of the proteome on an industrial level in a high-throughput format. The methods, capture compounds and collections permit sorting of complex mixtures of biomolecules. In addition, they permit identification of protein structures predicative or indicative of specific of phenotypes, such as disease states thereby eliminating the need for random SNP analysis, expression profiling and protein analytical methods. The capture compounds, collections and methods sort complex mixtures by providing a variety of different capture agents. In addition, they can be used to identify structural "epitopes" that serve as markers for specific disease states, stratify individual populations<br>
relative to specific phenotypes, permit a detailed understanding'of the proteins underlying molecular function, and provide targets for drug development. The increased understanding of target proteins permit the design of higher efficiency therapeutics.<br>
The capture compounds, collections and methods provided herein also permit screening of biomolecules, including but not limited to receptor<br><br>
-7-<br>
proteins and enzymes, which are drug targets and non-targets, as defined herein, that interact with pharmaceutical drugs under physiological conditions. The screening of biomolecules provides increased understanding of the mechanism of action of the pharmaceutical drugs or drug fragments, metabolites or synthetic intermediates in the drug syntheses, thereby helping the design of more target specific drugs. The methods also provide for identification of non-target biomolecules, such as proteins including but not limited to receptors and enzymes, that interact with pharmaceutical drugs, thereby causing side effects and other undesired therapeutic effects. In one embodiment, various attachments of the drugs or drug fragments, metabolites or synthetic intermediates in the drug syntheses to the capture compounds are used to determine which functionalities of the drugs or drug fragments, metabolites or synthetic intermediates in the drug syntheses interact with the target and non-target biomolecules. In one embodiment, the non-target functionalities are then eliminated from the drug, resulting in an improved drug that exhibits fewer side effects. In another embodiment, a drug is included in the capture compound, proteins that interact with the drug are isolated and identified, the proteins are related to function, and the drug is re-engineered to eliminate or reduce interactions with non-target proteins. The method may be repeated on the re-engineered drug, as desired.<br>
Capture compounds, collections of the compounds and methods that use the compounds, singly or in collections thereof, provided herein are<br>
designed to capture, separate and analyze biomolecules, including, but not limited to, mixtures of biomolecules, including biopolymers and macromolecules, individual biomolecules, such as proteins, including individual or membrane proteins. The capture and separation of biomolecules in the methods provided herein, is based on the unique surface features of the biomolecules or mixtures thereof, including but not limited to chemically rective amino acid residues on the surface of a protein or a mixture of proteins. Thus, the capture compounds provided herein are<br>
designed not to target any specific biomolecule, but to capture the<br><br>
-8-<br>
biomolecules based on the reactive groups present on the surface of the<br>
biomolecules or mixtures thereof<br>
The collections of the compounds provided herein contain a plurality, generally at least two, three, typically at least 10, 50, 100, 1000 or more different capture compounds. The compounds and collections are designed to permit probing of a mixture of biomolecules by virtue of interaction of the capture compounds in the collection with the components of the a mixture under conditions that preserve their three-dimensional configuration. Each member of the collection is designed 1) to bind, either covalently or via some other chemical interaction with high binding affinity (ka) such that the binding is irreversible or stable under conditions of mass spectrometric analysis to fewer than all, typically about 5 to 20 or more component biomolecules in a mixture, depending upon complexity and diversity of the mixture, under physiological conditions, including hydrophobic conditions, and 2) distinguish among biomolecules based upon topological features. In addition, the capture compounds generally include a group, such as a single-stranded oligonucleotide or partially single-stranded oligonucleotide, that permits separation of each set of capture compounds.<br>
The capture compounds and collections are used in a variety of methods, but are particularly designed for assessing biomolecules, such ,as biopolymers or components in mixtures from biological samples. The collections are used in top-down unbiased methods that assess structural changes, including post-translational structural changes and, for example, are used to compare patterns, particularly post-translational protein patterns, in diseased versus healthy cells from primary cells generally from the same<br>
individual   The cells that serve as the sources of biomolecules can be frozen into a selected metabolic state or synchronized to permit direct comparison<br>
and identification of phenotype-specific, such as disease-specific biomolecules, generally proteins.<br>
A capture compound includes at a chemical reactivity group X (also<br>
referred to herein as a function or a functionality), which effects the covalent<br><br>
-9-<br>
or a high binding affinity (high ka) binding, and least one of three other groups (also referred to herein as functions or funtionalities). The other groups are selected from among a selectivity function Y that modulates the interaction of a biomolecule with the reactivity function, a sorting function Q for addressing the components of the collection, and a solubility function W that alters solubility of the capture compound, such as by increasing the solubility of the capture compound under selected conditions, such as various physiological conditions, including hydrophobic conditions of cell membranes. Hence, for example, if membrane proteins are targeted, then the capture compounds in the collection are designed with solubility functions that increase or provide for solubility in such environment.<br>
For example, the reactivity group (reactivity function) includes groups that specifically react or interact with functionalities on the surface of a protein such as hydroxyl, amine, amide, sulfide and carboxylic acid groups, or that recognize specific surface areas, such as an antibody, a lectin or a receptor-specific ligand, or interacts with the active site of enzymes. Those skilled in the art can select from a library of functionalities to accomplish this interaction.   While this interaction can be highly reaction-specific, these compounds can react multiple times within the same protein molecule depending on the number of surface-accessible functional groups. Modification of the reaction conditions allows the identification of surface accessible functional groups with differing reactivity, thereby permitting identification of one or more highly reactive sites used to separate an individual protein from a mixture. Available technologies do not separate species in the resulting reaction mixture. The collections and compounds provided herein solve that problem through a second functionality, the selectivity group, which alters binding of the reactivity groups to the biomolecule.<br>
Selectivity functions include a variety of groups, as well as the geometric spacing of the second functionality, a single stranded unprotected or suitably protected oligonucleotide or oligonucleotide analog. The selective<br><br>
-10-<br>
functionality can be separate from the compound and include the solid or semi-solid support. The selective functionality in this embodiment can be<br>
porosity, hydrophobicity, charge and other chemical properties of the material. For example, selectivity functions interactioncovalently with target proteins to alter the specificity or binding of the reactivity function. Such functions include chemical groups and biomolecules that can sterically hinder proteins of specific size, hydrophilic compounds or proteins (e.g., PEG and trityls), hydrophobic compounds or proteins (e.g., polar aromatic, lipids, glycolipids, phosphotriester, oligosaccharides), positive or negatively charged groups, groups or biomolecules which create defined secondary or tertiary structure.<br>
The capture compounds can also include a sorting function for separation or addressing of each capture compound according to its structure. The sorting function, for example, can be a single-stranded (or partially single-stranded) unprotected or suitably protected oligonucleotide or oligonucleotide analog, typically containing between at least about 5 and up to 25, 35, 50, 100 or any desired number of nucleotides (or analogs thereof) containing a sequence-permuted region and optionally flanking regions. Each such block has a multitude of sequence permutations with or without flanking conserved regions, which is capable of hybridizing with a base-complementary single stranded nucleic acid molecule or a nucleic acid    . analog.  The sorting function can also be a label, such as a symbology, including a bar code, particularly a machine-readable bar code, a color coded-label, such as small colored bead that can be sorted by virtue of its color, a radio-frequency tag or other electronic label or a chemical label. Any functionality that permits sorting of each set of capture compounds to permit separate analysis of bound biomolecules is contemplated.<br>
In certain embodiments, each biomolecule to be captured is derivatized with more than one capture compound provided herein, where each tagged compound provides an additional level of sorting capability. In other embodiments, each of the plurality of compounds that derivatize a single biomolecule is different, allowing for specific and efficient sorting of the<br><br>
-11-<br>
biomolecule mixture (see, e.g., Figure 3). The capture compound also can be multifunctional containing other functionalities that can be used to reduce the complexity of biomolecule mixtures.<br>
Some of the capture compounds include at least a reactivity function and a selectivity function. These capture compounds optionally include sorting functionalities, which are one or more additional moieties that bind either covalently or noncovalently to a specific molecule to permit addressing of the compounds, such as by separation at discrete loci on a solid support, separation of the compounds on discrete loci. These capture compounds also optionally include one or more solubility functions, which are moieties that influence the solubility of the resulting compound, to attenuate or alter the hydrophobicity/hydrophilicity of the compounds (solubility function).<br>
Others of the capture compounds (or capture agents) include at least two functional portions: a reactivity function and a sorting function. The reactive group that specifically interacts with proteins or other biomolecules (reactivity function); and the other is an entity (sorting functions) that binds either covalently or noncovalently to a specific molecule(s). This entity can be a nucleic acid portion or nucleic acid analog portion that includes a single-stranded region that can specifically hybridize to a complementary single-stranded oligonucleotide or analog thereof.<br>
The capture compounds are provided as collections, generally as<br>
collections of sets of different compounds that differ in all functionalities.   For<br>
sorting of complex mixtures of biopolymers the collection includes diverse capture compound members so that, for example, when they are arrayed, each locus of the array contains 0 to 100, generally, 5 to 50 and desirably 1 to 20. typically 5 to 20, different biomolecules at each locus in the array.<br>
In practice in one embodiment, a collection of capture compounds is contacted with a biomolecule mixture and the bound molecules are assessed using, for example, mass spectrometry, followed by optional application of tagging, such as fluorescence tagging, after arraying to identify low abundance proteins. In other embodiments, a single capture compound is<br><br>
-12-<br>
contacted with one or plurality of biomolecules, and the bound molecules are assessed.<br>
Also provided herein are methods for the discovery and identification of proteins, which are selected based on a defined phenotype. The methods allow proteins to bind to the target molecules under physiological conditions while maintaining the correct secondary and tertiary conformation of the target. The methods can be performed under physiological and other conditions that permit discovery of bioglogically important proteins, including membrane proteins, that are selected based upon a defined phenotype.<br>
Before, during or after exposure of one or a plurality of capture compounds to a mixture of biomolecules, including, but not limited to, a mixture of proteins, the oligonucleotide portion, or analog thereof, of these compounds is allowed to hybridize to a complementary strand of<br>
immobilized oligonucleotide(s), or analog(s) thereof, to allow separation, isolation and subsequent analysis of bound biomolecules, such as proteins, by. for example, mass spectrometry, such as matrix assisted laser desorption ionization-time of flight (MALDI-TOF) mass spectrometry, colorimetric, fluorescent or chemiluminescent tagging, or to allow for increased resolution<br>
by mass spectrometry, including MALDI-TOF mass spectrometry.<br>
The collections of capture compounds can be used to generate compound arrays to capture target proteins or groups of related proteins that can mimic biological structures such as nuclear and mitochondrial transmembrane structures, artificial membranes or intact cell walls. Thus, the compounds and compound arrays provided herein are capable of mimicking biological entities and biological surfaces, thereby allowing for capture of biomolecules, including but not limited to proteins, which would otherwise be difficult or impossible to capture, such as those found in transmembrane regions of a cell.<br>
Samples for analysis include any biomolecules, particularly protein-containing samples, such as protein mixtures, including, but not limited to, natural and synthetic sources.   Proteins can be prepared by translation from<br><br>
-13-<br>
isolated chromosomes, genes. cDNA and genomic libraries. Proteins can be isolated from cells, and other sources. In certain embodiments, the capture<br>
f compounds provided herein are designed to selectively capture different post-translational modifications of the same protein (i.e., phosphorylation patterns<br>
(e.g., oncogenes), glycosylation and other post-translational modifications).<br>
Other methods that employ the collections are also provided. In one method, the collections of one or more member capture compounds are used to distinguish between or among different conformations of a protein and, for example, can be used for phenotypic identification, such as for diagnosis. For example, for diseases of protein aggregation, which are diseases involving a conformationally altered protein, such as amyloid diseases, the collections can distinguish between the disease-involved form of the protein from the normal protein and thereby diagnose the disease in a sample. <br>
BRIEF DESCRIPTION OF THE FIGURES<br>
Figure 1 shows the hybridization, separation and mass spectral analysis of a mixture of proteins.<br>
Figure 2 provides a schematic depiction of one embodiment of the apparatus provided herein<br>
Figure 3 illustrates a protein tagged with four compounds provided herein, thereby allowing for specific sorting of the protein.<br>
Figure 4 shows the increased and specific hybridization resulting from use of two or more oligonucleotide tags.<br>
Figure 5 shows tagging of a single protein with two different oligonucleotides in one reaction.<br>
Figure 6 is a flow diagram of recombinant protein production.<br>
Figure 7 illustrates production of an adapted oligonucleotide dT primed cDNA library.<br>
Figure 8 shows production of an adapted sequence motif specific cDNA library.<br>
Figure 9 shows production of an adapted gene specific cDNA.<br>
Figure 10 illustrates purification of amplification products from a<br><br>
-14-<br>
template library.<br>
Figure 11 shows an adapted oligonucleotide dT primed cDNA library<br>
as a universal template for the amplification of gene subpopulations.<br>
Figure 12 illustrates decrease of complexity during PCR amplification.<br>
Figure 13 shows the attachment of a bifunctional molecule to a solid surface.<br>
Figure 14 shows analysis of purified proteins from compound screening and antibody production.<br>
Figure 15 provides synthetic schemes for synthesis of exemplary capture reagents provided herein (see, e.g., Example 4).<br>
Figure 16 provides exemplary reactivity functions for use in the capture reagents provided herein.<br>
Figure 17 provides exemplary selectivity functions for use in the capture reagents provided herein.<br>
Figure 18 depicts exemplary points for regulation of metabolic control mechanisms for cell synchronization.<br>
Figures 19 depict cell separation and synchronization methods; Figure 19a depicts methods for separation of cells from blood from a single patient to separate them by phenotype; Figure 19b shows the results of flow cytometry separation of blood cells without labeling; Figure 19c shows an( example in which synchronized cells in culture are sorted according to DNA content as a way to separate cells by phase of the cell cycle.<br>
Figure 20 shows a schematic of a biomolecule capture assay and results using exemplary capture compounds and proteins.<br>
Figure 21 shows exemplary selectivity functions for use in the capture compounds provided herein.<br>
Figure 22 shows mass spectrometric results of the reaction of hemoglobin with two of the capture compounds provided herein. As shown in the Figure, the more hydrophobic capture compound, i.e., the capture compound with a more hydrophobic selectivity function, reacts with a-<br>
hemoglobin stoichiometrically and with p-hemoglobin, while the less<br><br>
-15-<br>
hydrophobic capture compound reacts incompletely with a-hemoglobin and does not react with (3-hemoglobin.<br>
Figure 23 shows exemplary capture compounds provided herein.<br>
Figure 24 shows mass spectrometric results of +he reaction of a capture compound provided herein with a protein mixture obtained from U937 lymphoma blood cells. The Figure shows selective capture of the indicated protein by the capture compound.<br>
Figure 25 shows mass spectrometric results of the reaction of a capture compound provided herein with Burkitt's lymphoma cytosol. As shown in the Figure, the proteins labeled A-E are captured by the indicated capture compound.<br>
Figure 26 shows mass spectrometric results of the reaction of a capture compound provided herein with total cytosol from Burkitt's lymphoma lymphoblast as compared to healthy age and gender matched lymphoblast. Proteins A, B, C and E are found in both samples. Protein D is expressed only in the Burkitt's lymphoma sample. Proteins labeled (H) are expressed only in the healthy sample. As shown in the Figure, reaction of the Burkitt's lymphoma sample with a capture compound provided herein results in complete capture of protein D allowing for analysis and identification of the protein.<br>
Figure 27 shows exemplary features of the biased and unbiased selectivity groups in the selectivity function of the capture compounds.<br>
Figure 28 illustrates an exemplary protocol for protein identification using capture compounds.<br>
Figure 29 shows mass spectrometric results of the reaction of an capture compound with a trityl scaffold, biotin, NHS reactivity function, OH selectivity function with the cytosolic fraction of cell lines from a 5 year old male acute lymphocytic leukemia (sup B A .L) and an age/gender matched control (wil2). The Figure shows that capture compound covalently captures many proteins which are similar in abundance. However a major protein is detected at ~22kDa in the diseased cell Iin3 that is absent in the control. The<br><br>
-16-<br>
protein is identified by tryptic digest and peptide database matching as HSP-27 (heat shock protein), which is implicated in other cancers in the literature.<br>
Figure 30 illustrates a schematic diagram of the steps involved in protein capture and identification using a capture compound. The figure shows that a capture compound is mixed with a sample containing a mixture of proteins. Proteins with an affinity for the selectivity function (e.g. drug) are allowed to come to equilibrium with the selectivity function. The capture compound is then activated (for example, with hu) forming a radical which is shortlived and covalently captures the proteins for which there was an affinity. Other proteins are not captured if the capture compound was not in very close proximity due to the equilibrium between selectivity function and protein.<br>
The captured protein is isolated with biotin and identified using mass spectrometry.<br>
Figure 31 shows selective protein capture using capture compounds. Capture compounds A and B containing sulfonamide interact with Carbonic Anhydrase. (According to literature, its Kd for CA II isoform is ~10nM, and for CA I is ~1uM (both values independently confirmed using activity assay). Using purified proteins, affinity and capture efficiency is highest for Carbonic II, 'ower for CA I, and negligible for other purified proteins tested.<br>
Figure 32 shows relative binding strengths of protein isoforms to a t known ligand for capture compound B.<br>
Figure 33 shows isolation of Carbonic Anhydrase from complex protein mixtures using capture compound A. CA II was doped into a FPLC purified protein mixture from the human kidney cell line HEK293,. The doped CAII was pulled out from all other proteins using avidin-coated (SoftLink) resin. Other proteins were discarded, yielding purified protein ready for further analysis.<br>
Figure 34 shows isolation of Carbonic Anhydrase from highly complex protein mixtures using capture compound A. CA II was doped into the whole cy'osolic extract from the human kidney cell line HEK293,. The doped CAII was pulled out from all other proteins using avidin-coated (SoftLink) resin.<br><br>
-17-<br>
Other proteins were discarded, yielding purified protein ready for further analysis.<br>
Figure 35 shows capture and isolation of Carbonic Anhydrase from lysed red blood cells. The top spectrum in the figure shows direct MALDI of lysed red blood cells (no purification) wherein signal for Hemoglobin, which is in huge excess over all other proteins, can be seen. Signals are seen for the alpha and beta chains, and also for non-specific dimers (-30 kiloDaltons). Bottom spectrum in the figure is taken after capture compound A, containing a sulfonamide drug with an affinity for Carbonic Anhydrase, is mixed with the lysed red blood cells. The capture compound covalently captures the Carbonic Anhydrase isoforms I and II. All other proteins that are not covalently captured, including nearly all of the Hemoglobin which is in 2-3 log excess, are washed away prior to MALDI analysis. No gel or chromatographic cleanup is required to obtain this spectrum. The intensity of the CA II peak is higher than CAI (which is more ~100x more abundant in RBCs) because the sulfonamide drug has a higher affinity for CAN.<br>
Figure 36 shows direct capture of Carbonic Anhydrase from red blood cells, without pre-lysis of the cells.<br>
Figure 37 shows capture of Carbonic Anhydrase from red blood cell lysate when unbiotinylated proteins including Carbonic Anhydrase are in huge excess.<br>
Figure 38 shows capture of proteins with lower affinities using very<br>
high concentrations of capture compound A. DETAILED DESCRIPTION <br>
A.   Definitions<br>
Unless defined otherwise, all technical and scientific terms used herein have the same meaning as is commonly understood by one of skill in the art to which the invention(s) belong. All patents, patent applications, published applications and publications, Genbank sequences, websites and other published materials referred to throughout the entire disclosure herein, unless noted otherwise, are incorporated by reference in their entirety. In the event<br><br>
-18-<br>
that there are a plurality of definitions for terms herein, those in this section prevail.   Where reference is made to an URL or other such indentifier or<br>
address, it is understood that such identifiers can change and particular information on the internet can come and go, but equivalent information can be found by searching the internet. Reference thereto evidences the availability and public dissemination of such information.<br>
As used herein, an oligonucleotide means a linear sequence of up to about 20, about 50, or about 100, nucleotides joined by phosphodiester bonds. Above this length the term polynucleotide begins to be used.<br>
As used herein, an oligonucleotide analog means a linear sequence of up to about 20, about 50, or about 100, nucleotide analogs, or linear sequence of up to about 20, about 50, or about 100 nucleotides linked by a "backbone" bond other than a phosphodiester bond, for example, a phosphotriester bond, a phosphoramidate bond, a phophorothioate bond, a<br>
methylphosphonate diester bond, a thioester bond, or a peptide bond (peptide nucleic acid).<br>
As used herein, peptide nucleic acid (PNA) refers to nucleic acid analogs in that the ribose-phosphate backbone is replaced by a backbone<br>
held together by amide bonds.<br>
As used herein, proteome means all the proteins present within a cell.<br>
As used herein, a biomolecule is any compound found in nature, or derivatives thereof. Biomolecules include, but are not limited to oligonucleotides, oligonucleosides, proteins, peptides, amino acids, lipids, steroids, peptide nucleic acids (PNAs), oligosaccharides and monosaccharides.<br>
As used herein, MALDI-TOF refers to matrix assisted laser desorption ionization-time of flight mass spectrometry.<br>
As used herein, the term "conditioned" or "conditioning," when used in reference to a protein thereof, means that the polypeptide is modified to decrease the laser energy required to volatilize the protein, to minimize the likelihood of fragmentation of the protein, or to increase the resolution of a<br><br>
-19-<br>
mass spectrum of the protein or of the component amino acids. Resolution of a mass spectrum of a protein can be increased by conditioning the protein<br>
prior to performing mass spectrometry.   Conditioning can be performed at any stage prior to mass spectrometry and, in one embodiment, is performed while the protein is immobilized. A protein can be conditioned, for example, by treating it with a cation exchange material or an anion exchange material,<br>
which can reduce the charge heterogeneity of the protein, thereby for eliminating peak broadening due to heterogeneity in the number of cations (or anions) bound to the various proteins in a population. In one embodiment, removal of all cations by ion exchange, except for H+ and ammonium ions, is<br>
performed. By contacting a polypeptide with an alkylating agent such as alkyliodide, iodoacetamide, iodoethanol, or 2,3epoxy-1-propanol, the formation of disulfide bonds, for example, in a protein can be prevented. Likewise, charged amino acid side chains can be converted to uncharged derivatives employing trialkylsilyl chlorides.<br>
Since the capture compounds contain protein and nucleic acid portions, conditioning suitable for one or both portions is also contemplated. Hence, a prepurification to enrich the biomolecules to be analyzed and the removal of all cations, such as by ion exchange, except for H+ and ammonium, or other conditioning treatment to improve resolution is advantageous for analysis of the nucleic acid portion as well as the protein portion.<br>
Conditioning of proteins is generally unnecessary because proteins are relatively stable under acidic, high energy conditions so that proteins do not require conditioning for mass spectrometry analyses. There are means of improving resolution, however, in one embodiment for shorter peptides, such as by incorporating modified amino acids that are more basic than the corresponding unmodified residues. Such modification in general increases the stability of the polypeptide during mass spectrometric analysis. Also, cation exchange chromatography, as well as general washing and purification procedures that remove proteins and other reaction mixture components<br><br>
-20-<br>
away from the protein can be used to increase the resolution of the spectrum resulting from mass spectrometric analysis of the protein.<br>
As used herein, capture efficiency is the peak area of the captured biomolecule/(peak area captured biomolecule + peak area uncaptured biomolecule) as measured by HPLC analysis.<br>
As used herein, "matrix" refers to the material with which the capture compound biomolecule conjugates are combined for MALDI mass spectrometric analysis. Any matrix material, such as solid acids, including 3"-~ hydroxypicoiinic acid, liquid matrices, such as glycerol, known to those of skill in the art for nucleic acid and/or protein analyses is contemplated. Since the compound biomolecule conjugates contain nucleic acid and protein a mixture (optimal for nucleic acids and proteins) of matrix molecules can be used.<br>
As used herein, macromolecule refers to any molecule having a molecular weight from the hundreds up to the millions. Macromolecules include, but are not limited to. peptides, proteins, nucleotides, nucleic acids, carbohydrates, and other such molecules that are generally synthesized by biological organisms but can be prepared synthetically or using recombinant molecular biology methods.<br>
As used herein, the term "biopolymer" is refers to a biological molecule, including macromolecules, composed of two or more monomeric subunits, or derivatives thereof, which are linked by a bond or a macromolecule. A biopolymer can be, for example, a polynucleotide, a polypeptide, a carbohydrate, or a lipid, or derivatives or combinations thereof, for example, a nucleic acid molecule containing a peptide nucleic acid portion or a glycoprotein. The methods and collections herein, though described with reference to biopolymers, can be adapted for use with other synthetic schemes and assays, such as organic syntheses of pharmaceuticals, or inorganics and any other reaction or assay performed on a solid support or in a well in nanoliter or smaller volumes.<br>
As used herein, biomolecule includes biopolymers and macromolecules and all molecules that can be isolated from living organisms<br><br>
-21-<br>
and viruses, including, but are not limited to, cells, tissues, prions, animals, plants, viruses, bacteria and other organsims.<br>
As used herein, a biological particle refers to a virus, such as a viral vector or viral capsid with or without packaged nucleic add, phage, including a phage vector or phage capsid, with or without encapsulated nucleotide acid, a single cell, including eukaryotic and prokaryotic cells or fragments thereof, a liposome or micellar agent or other packaging particle, and other such biological materials. For purposes herein, biological particles include molecules that are not typically considered macromolecules because they are not generally synthesized, but are derived from cells and viruses.<br>
As used herein, a drug refers to any compound that is a candidate for use as a therapeutic or as a lead compound for designing a therapeutic or that is a known pharmaceutical. Such compounds can be small molecules, including small organic molecules, peptides, peptide mimetics,  antisense molecules, antibodies, fragments of antibodies or recombinant antibodies. Of particular interest are "drugs" that have specific binding properties so that they can be used as selectivity groups or can be used as for sorting of the capture compounds, either a sorting functionality that binds to a target on a support, or linked to a solid support, where the sorting functionality is the drug target.<br>
As used herein, a drug metabolite refers to any compound that is formed after transformation of a drug following its metabolism in the body that results in a different molecule that may be more or less active than the parent drug.<br>
As used herein, a drug fragment refers to a molecule that is a portion or moiety of a drug.<br>
As used herein, a drug synthetic intermediate is a compound that is used as an intermediate in the chemical synthesis of a drug.<br>
As used herein, the term "a" is singular or plural.<br>
As used herein, a "drug target" is a biomolecule, such as a protein including but not limited to receptors and enzymes, that the drug is intended<br><br>
-22-<br>
to interact with in vivo, thereby exerting the desired therapeutic effects.<br>
As used herein, a "drug non-target" is a biomolecule, such as a protein including but not limited to receptors and enzymes, that the drug is not intended to interact with in vivo. The interaction of a drug with drug non-targets may result in undesired therapeutic effects such as side effects.<br>
As used herein, the term "nucleic acid" refers to single-stranded and/or double-stranded polynucleotides such as deoxyribonucleic acid (DNA), and ribonucleic acid (RNA) as well as analogs or derivatives of either RNA or DNA. Nucleic acid molecules are linear polymers of nucleotides, linked by 3',5' phosphodiester linkages. In DNA, deoxyribonucleic acid, the sugar group is deoxyribose and the bases of the nucleotides are adenine, guanine, thymine and cytosine. RNA, ribonucleic acid, has ribose as the sugar and uracil replaces thymine. Also included in the term "nucleic acid" are analogs of nucleic acids such as peptide nucleic acid (PNA), phosphorothioate DNA, and other such analogs and derivatives or combinations thereof.<br>
As used herein, the term "polynucleotide" refers to an oligomer or polymer containing at least two linked nucleotides or nucleotide derivatives,<br>
including a deoxyribonucleic acid (DNA), a ribonucleic acid (RNA), and a DNA or RNA derivative containing, for example, a nucleotide analog or a "backbone" bond other than a phosphodiester bond, for example, a phosphotriester bond, a phosphoramidate bond, a methylphosphonate diester bond, a phophorothioate bond, a thioester bond, or a peptide bond (peptide nucleic acid). The term "oligonucleotide" also is used herein essentially synonymously with "polynucleotide," although those in the art recognize that oligonucleotides, for example, PCR primers, generally are less than about fifty to one hundred nucleotides in length.<br>
Nucleotide analogs contained in a polynucleotide can be, for example, mass modified nucleotides, which allows for mass differentiation of polynucleotides; nucleotides containing a detectable label such as a fluorescent, radioactive, colorometric, luminescent or chemiluminescent label, which allows for detection of a polynucleotide; or nucleotides containing a<br><br>
-23-<br>
reactive group such as biotin or a thiol group, which facilitates immobilization of a polynucleotide to a solid support. A polynucleotide also can contain one<br>
or more backbone bonds that are selectively cleavable, for example, chemically, enzymatically or photolytically. For example, a polynucleotide can include one or more deoxyribonucleotides, followed by one or more ribonucleotides, which can be followed by one or more deoxyribonucleotides, such a sequence being cleavable at the ribonucleotide sequence by base hydrolysis. A polynucleotide also can contain one or more bonds that are relatively resistant to cleavage, for example, a chimeric oligonucleotide primer, which can include nucleotides linked by peptide nucleic acid bonds and at least one nucleotide at the 3' end, which is linked by a phosphodiester bond, or the like, and is capable of being extended by a polymerase. Peptide nucleic acid sequences can be prepared using well known methods (see, for example Weiler et al. (1997) Nucleic acids Res. 25:2792-2799).<br>
A polynucleotide can be a portion of a larger nucleic acid molecule, for example, a portion of a gene, which can contain a polymorphic region, or a portion of an extragenic region of a chromosome, for example, a portion of a region of nucleotide repeats such as a short tandem repeat (STR) locus, a variable number of tandem repeats (VNTR) locus, a microsatellite locus or a minisatellite locus. A polynucleotide also can be single stranded or double stranded, including, for example, a DNA-RNA hybrid, or can be triple stranded or four stranded. Where the polynucleotide is double stranded DNA, it can be in an A, B. L or Z configuration, and a single polynucleotide can contain combinations of such configurations.<br>
As used herein, a "mass modification," with respect to a biomolecule to be analyzed for mass spectrometry, refers to the inclusion of changes in consituent atoms or groups that change the molecular weight of the resulting molecule in defined increments detectable by mass spectrometric analysis. Mass modifications do not include radiolabels, such as isotope labels or or fluroescent gropus or other such tags normally used for detection by means other than mass spectrometry.<br><br>
-24-<br>
As used herein, the term "polypeptide" means at least two amino acids, or amino acid derivatives, including mass modified amino acids and<br>
amino acid analogs, which are linked by^ peptide bond and which can be a modified peptide bond. A polypeptide can be translated from a polynucleotide, which can include at least a portion of a coding sequence or a portion of a nucleotide sequence that is not naturally translated due, for example, to it being located in a reading frame other than a coding frame, or it being an intron sequence, a 3' or 5' untranslated sequence, a regulatory sequence such as a promoter. A polypeptide also can be chemically synthesized and can be modified by chemical or enzymatic methods following translation or chemical synthesis. The terms "polypeptide," "peptide" and "protein" are used essentially synonymously herein, although the skilled artisan recognizes that peptides generally contain fewer than about fifty to one hundred amino acid residues, and that proteins often are obtained from a natural source and can contain, for example, post-translational modifications. A polypeptide can be posttranslationally modified by, for example, phosphorylation (phosphoproteins) or glycosylation (glycoproteins, proteoglycans), which can be performed in a cell or in a reaction in vitro.<br>
As used herein, the term "conjugated" refers to stable attachment, typically by virtue of a chemical interaction, including ionic and/or covalent attachment. Among the conjugation means are streptavidin- or avidin- to biotin interaction; hydrophobic interaction; magnetic interaction (e.g., using functionalized magnetic beads, such as DYNABEADS, which are streptavidin-coated magnetic beads sold by Dynal, Inc. Great Neck, NY and Oslo Norway); polar interactions, such as "wetting" associations between two polar surfaces or between oligo/polyethylene glycol; formation of a covalent bond, such as an amide bond, disulfide bond, thioether bond, or via crosslinking agents; and via an acid-labile or photocleavable linker.<br>
As used herein, "sample" refers to a composition containing a material to be detected. For the purposes herein, sample refers to anything which can contain an biomolecule. The sample can be a biological sample, such as a<br><br>
-25-<br>
biological fluid or a biological tissue obtained from any organism or a cell of or from an organism or a viral particle or portions thereof. Examples of<br>
biological fluids include urine, blood, plasma, serum, saliva, semen, stool,<br>
sputum, cerebral spinal fluid, tears, mucus, sperm, amniotic fluid or the like. Biological tissues are aggregates of cells, usually of a particular kind together with their intercellular substance that form one of the structural materials of a<br>
human, animal, plant, bacterial, fungal or viral structure, including connective,<br>
epithelium, muscle and nerve tissues.  Examples of biological tissues also include organs, tumors, lymph nodes, arteries and individual cell(s).<br>
Thus, samples include biological samples (e.g., any material obtained from a source originating from a living being (e.g., human, animal, plant, bacteria, fungi protist, virus). The biological sample can be in any form, including solid materials (e.g., tissue, cell pellets and biopsies, tissues from cadavers) and biological fluids (e.g., urine, blood, saliva, amniotic fluid and mouth wash (containing buccal cells)).  In certain embodiments, solid materials are mixed with a fluid. In embodiments herein, the a sample for mass spectrometric analysis includes samples that contain a mixture of matrix used for mass spectrometric analyses and the capture compound/biomolecule complexes.<br>
As used herein, the term "solid support" means a non-gaseous, non-liquid material having a surface. Thus, a solid support can be a flat surface constructed, for example, of glass, silicon, metal, plastic or a composite; or can be in the form of a bead such as a silica gel, a controlled pore glass, a magnetic or cellulose bead; or can be a pin, including an array of pins suitable for combinatorial synthesis or analysis.<br>
, As used herein, a collection refers to combination of two or more members, generally 3, 5, 10, 50, 100, 500, 1000 or more members. In particular a collection refers to such combination of the capture compounds as provided herein.<br>
As used herein, an array refers to a collection of elements, such as the capture compounds, containing three or more members. An addressable<br><br>
-26-<br>
array is one in that the members of the array are identifiable, typically by position on a solid phase support but also by virtue of an identifier or detectable label. Hence, in general the members of an array are be immobilized to discrete identifiable loci on the surface of a solid phase. A plurality of  the compounds are attached to a support, such as an array {i.e., a pattern of two or more) on the surface of a support, such as a silicon chip or other surface, generally through binding of the sorting functionality with a group or compound on the surface of the support. Addressing can be achieved by labeling each  member electronically, such as with an radio-frequency (RF) tag, through the use of color coded beads or other such identifiable and color coded labels and through molecular weight. These labels for addressing serve as sorting functions "Q." Hence, in general the members of the array are immobilized to discrete identifiable loci on the surface of a solid phase or directly or indirectly linked to or otherwise associated with the identifiable label, such as affixed to a microsphere or other particulate support (herein referred to as beads) and suspended in solution or spread out on a surface.<br>
As used herein, "substrate" refers to an insoluble support onto which a sample and/or matrix is deposited.   Support can be fabricated from virtually any insoluble or solid material. For example, silica gel, glass (e.g., controlled-pore glass (CPG)), nylon, Wang resin, Merrifield resin, dextran crossâlinked with epichlorohydrin (e.g., SephadexR), agarose (e.g., SepharoseR), cellulose, magnetic beads, Dynabeads, a metal surface (e.g., steel, gold, silver, aluminum, silicon and copper), a plastic material (e.g., polyethylene, polypropylene, polyamide, polyester, polyvinylidenedifluoride (PVDF)) Exemplary substrate include, but are not limited to, beads (e.g., silica gel, controlled pore glass, magnetic, dextran crossâlinked with epichlorohydrin (e.g., SephadexR), agarose (e.g., SepharoseR), cellulose, capillaries, flat supports such as glass fiber filters, glass surfaces, metal surfaces (steel, gold, silver, aluminum, copper and silicon), plastic materials including multiwell plates or membranes (e.g., of polyethylene, polypropylene,<br><br>
-27-<br>
polyamide, polyvinylidenedifluoride), pins (e.g., arrays of pins suitable for combinatorial synthesis or analysis or beads in pits of flat surfaces such as wafers (e.g., silicon wafers) with or without filter plates. The solid support is in any desired form, including, but not limited to, a bead, capillary, plate, membrane, wafer, comb, pin, a wafer with pits, an array of pits or nanoliter wells and other geometries and forms known to those of skill in the art. Supports include flat surfaces designed to receive or link samples at discrete loci. In one embodiment, flat surfaces include those with hydrophobic regions surrounding hydrophilic loci for receiving, containing or binding a sample.<br>
The supports can be particulate or can be in the form of a continuous surface, such as a microtiter dish or well, a glass slide, a silicon chip, a nitrocellulose sheet, nylon mesh, or other such materials. When particulate, typically the particles have at least one dimension in the 510 mm range or smaller   Such particles, referred collectively herein as "beads", are often, but not necessarily, spherical. Reference to "bead," however, does not constrain the geometry of the matrix, which can be any shape, including random shapes, needles, fibers, and elongated. "Beads", particularly microspheres that are sufficiently small to be used in the liquid phase, are also contemplated. The "beads" can include additional components, such as magnetic or paramagnetic particles (see, e.g.,, Dyna beads (Dynal, Oslo, Norway)) for separation using magnets, as long as the additional components do not interfere with the methods and analyses herein.<br>
As used herein, "polymorphism" refers to the coexistence of more than one form of a gene or portion theieof. A portion of a gene of which there are at least two different forms, e.g., two different nucleotide sequences, is referred to as a "polymorphic region of a gene'. A polymorphic region can be a single nucleotide, e.g., a single nucleotide polymorphism (SNP), the identity of which differs in different alleles. A polymorphic region also can be several nucleotides in length.<br>
As used herein, "polymorphic gene" refers to a gene having at least one polymorphic region.<br><br>
-28-<br>
As used herein, "allele", which is used interchangeably herein with "allelic variant" refers to alternative forms of a gene or portions thereof. Alleles<br>
occupy the same locus or position on homologous chromosomes. When a subject has two identical alleles of a gene, the subject is said to be homozygous for the gene or allele. When a subject has two different alleles of a gene, the subject is said to be heterozygous for the gene. Alleles of a specific gene can differ from each other in a single nucleotide, or several nucleotides, and can include substitutions, deletions, and insertions of nucleotides An allele of a gene also can be a form of a gene containing a mutation.<br>
As used herein, "predominant allele" refers to an allele that is represented in the greatest frequency for a given population. The allele or alleles that are present in lesser frequency are referred to as allelic variants.<br>
As used herein, "associated" refers to coincidence with the development or manifestation of a disease, condition or phenotype. Association can be due to, but is not limited to, genes responsible for housekeeping functions whose alteration can provide the foundation for a variety of diseases and conditions, those that are part of a pathway that is involved in a specific disease, condition or phenotype and those that indirectly contribute to the manifestation of a disease, condition or phenotype.<br>
As used herein, the term "subject" refers to a living organism, such as a mammal, a plant, a fungi, an invertebrate, a fish, an insect, a pathogenic oryanlsm, such as a virus or a bacterium, and, includes humans and other mammals.<br>
As used herein, the term "gene" or "recombinant gene" refers to a nucleic acid molecule containing an open reading frame and including at least one exon and (optionally) an intron sequence. A gene can be either RNA or DNA. Genes can include regions preceding and following the coding region.<br>
As used herein, "intron" refers to a DNA fragment present in a given gene that is spliced out during mRNA maturation.<br>
As used herein, "nucleotide sequence complementary to the<br><br>
-29-<br>
nucleotide sequence set forth in SEQ ID NO: x" refers to the nucleotide sequence of the complementary strand of a nucleic acid strand having SEQ<br>
ID NO: x. The term "complementary strand" is used herein interchangeably with the term "complement". The complement of a nucleic acid strand can be the complement of a coding strand or the complement of a noncoding strand. When referring to double stranded nucleic acids, the complement of a<br>
nucleic acid having SEQ ID NO: x refers to the complementary strand of the strand having SEQ ID NO: x or to any nucleic acid having the nucleotide sequence of the complementary strand of SEQ ID NO: x. When referring to a single stranded nucleic acid having the nucleotide sequence SEQ ID NO: x, the complement of this nucleic acid is a nucleic acid having a nucleotide sequence that is complementary to that of SEQ ID NO: x.<br>
As used herein, the term "coding sequence" refers to that portion of a gene that encodes a amino acids that constitute a polypeptide or protein.<br>
As used herein, the term "sense strand" refers to that strand of a double-stranded nucleic acid molecule that has the sequence of the mRNA that encodes the amino acid sequence encoded by the double-stranded nucleic acid molecule.<br>
As used herein, the term "antisense strand" refers to that strand of a double-stranded nucleic acid molecule that is the complement of the sequence of the mRNA that encodes the amino acid sequence encoded by the double-stranded nucleic acid molecule.<br>
As used herein, the amino acids, which occur in the various amino acid sequences appearing herein, are identified according to their well-known, three-letter or one-letter abbreviations. The nucleotides, which occur in the various DNA fragments, are designated with the standard single-letter designations used routinely in the art (see, Table 1).<br>
As used herein, amino acid residue refers to an amino acid formed upon chemical digestion (hydrolysis) of a polypeptide at its peptide linkages. The amino acid residues described herein are, in certain embodiments, in the<br>
"L" isomeric form. Residues in the "D" isomeric form can be substituted for<br><br>
-30-<br>
any Lamino acid residue, as long as the a desired functional property is retained by the polypeptide. NH2 refers to the free amino group present at<br>
the amino terminus of a polypeptide. COOH refers to the free carboxy group present at the carboxyl terminus of a polypeptide. In keeping with standard polypeptide nomenclature described in J. Biol. Chem., 243:355259 (1969) and adopted at 37 C.F.R. Â§ Â§ 1.821 -1.822, abbreviations for amino acid residues are shown in the following Table:<br><br>
-31-<br>
Table 1<br>
Table of Correspondence<br><br>
SYMBOL	<br>
1 Letter	3 Letter	AMINO ACID<br>
Y	Tyr	tyrosine<br>
G	Gly	glycine<br>
F	Phe	phenylalanine<br>
M	Met	methionine<br>
A	Ala	alanine<br>
S	Ser	serine<br>
i	lie	isoleucine<br>
L	Leu	leucine<br>
T	Thr	threonine<br>
V	Val	valine<br>
p	Pro	proline<br>
K	Lys	lysine<br>
H	His	histidine<br>
Q	Gin	glutamine<br>
E	Glu	glutamic acid<br>
z	Glx	Glu and/or Gin<br>
W	Trp	tryptophan<br>
R	Arg	arginine<br>
D	Asp	aspartic acid<br>
N	Asn	asparagine<br>
B	Asx	Asn and/or Asp<br>
C	Cys	cysteine<br>
X	Xaa	Unknown or other<br>
It should be noted that all amino acid residue sequences represented<br><br>
-32-<br>
herein by formulae have a left to right orientation in the conventional direction of aminoterminus to carboxylterminus. In addition, the phrase "amino acid residue" is broadly defined to include the amino acids listed in the Table of Correspondence and modified and unusual amino acids, such as those referred to in 37 C.F.R. Â§ Â§ 1.821-1.822, and incorporated herein by reference. Furthermore, it should be noted that a dash at the beginning or e*nd of an amino acid residue sequence indicates a peptide bond to a further sequence of one or more amino acid residues or to an aminoterminal group such as NH2 or to a carboxylterminal group such as COOH.<br>
In a peptide or protein, suitable conservative substitutions of amino acids are known to those of skill in this art and can be made generally without altering the biological activity of the resulting molecule. Those of skill in this art recognize that, in general, single amino acid substitutions in non-essential regions of a polypeptide do not substantially alter biological activity (see, e.g., Watson et al. Molecular Biology of the Gene, 4th Edition. 1987, The Benjamin/Cummings Pub. co.. p.224).<br>
Such substitutions can be made in accordance with those set forth in TABLE 2 as follows:<br>
TABLE 2<br><br>
Original residue	Conservative substitution<br>
Ala (A)	Gly; Ser<br>
Arg (R)	Lys<br>
Asn (N)	Gin; His<br>
Asp (D)	Glu<br>
Cys (C)	Ser<br>
Gin (Q)	Asn<br>
Glu (E)	Asp<br>
Gly (G)	Ala; Pro<br>
His (H)	Asn; Gin<br>
lie (I)	Leu; Val<br><br>
-33-<br><br>
Original residue	Conservative substitution<br>
Leu (L)	He; Val<br>
Lys (K)	Arg; Gin<br>
Met (M)	Leu; Tyr; He<br>
Phe (F)	Met; Leu; Tyr<br>
Ser(S)	Thr<br>
Thr(T)	Ser<br>
Trp (W)	Tyr<br>
Tyr (Y)	Trp; Phe<br>
Val (V)	He; Leu<br>
Otner substitutions are also permissible and can be determined empirically or in accord with known conservative substitutions.<br>
As used herein, a DNA or nucleic acid homolog refers to a nucleic acid that includes a preselected conserved nucleotide sequence, such as a sequence encoding a therapeutic polypeptide. By the term "substantially homologous" is meant having at least 80%, at least 90% or at least 95% homology therewith or a less percentage of homology or identity and conserved biological activity or function.<br>
The terms "homology" and "identity" are often used interchangeably. In this regard, percent homology or identity can be determined, for example, by comparing sequence information using a GAP computer program. The GAP program uses the alignment method of Needleman and Wunsch (J. Mol. Biol. 48:443 (1970), as revised by Smith and Waterman {Adv. Appl. Math. 2:482 (1981). Briefly, the GAP program defines similarity as the number of aligned symbols {e.g., nucleotides or amino acids) that are similar, divided by the total number of symbols in the shorter of the two sequences. The default parameters for the GAP program can include: (1) a unary comparison matrix (containing a value of 1 for identities and 0 for nonidertities) and the weighted comparison matrix of Gribskov and Burgess, Nucl. Acids Res. 14:6745<br><br>
-34-<br>
(1986), as described by Schwartz and Dayhoff, eds., /ATLAS OF PROTEIN SEQUENCEAND STRUCTURE, National Biomedical Research Foundation, pp. 353358 (1979); (2) a penalty of 3.0 for each gap and an additional 0.10 penalty for each symbol in each gap; and (3) no penalty for end gaps.<br>
Whether any two nucleic acid molecules have nucleotide sequences that are at least 80%, 85%, 90%, 95%, 96%, 97%, 98% or 99% "identical" cap be determined using known computer algorithms such as the "FASTA"<br>
program, using for example, the default parameters as in Pearson and Lipman, Proc. Natl. Acad. Sci. USA 85:2444 (1988). Alternatively the BLAST function of the National Center for Biotechnology Information database can be used to determine identity.<br>
In general, sequences are aligned so that the highest order match is obtained   "Identity" perse has an art-recognized meaning and can be calculated using published techniques. (See, e.g.: Computational Molecular Biology, Lesk. AM   ed., Oxford University Press, New York, 1988: Biocomputing: Informatics and Genome Projects, Smith, D.W., ed., Academic Press, New York, 1993; Computer Analysis of Sequence Data, Parti, Griffin, AM   and Griffin. H.G., eds., Humana Press, New Jersey, 1994; Sequence Analysis in Molecular Biology, von Heinje, G., Academic Press, 1987; and Sequence Analysis Primer, Gribskov, M. and Devereux, J., eds., M Stockton Press, New York. 1991). While there exist a number of methods to measure identity between two polynucleotide or polypeptide sequences, the term "identity" is well known to skilled artisans (Carillo, H. &amp; Lipton, D., SIAM J Applied Math 48:1073 (1988)). Methods commonly employed to determine identity or similarity between two sequences include, but are not limited to, those disclosed in Guide to Huge Computers, Martin J. Bishop, ed., Academic Press, San Diego, 1994, and Carillo, H. &amp; Lipton, D., SIAM J Applied Math 48:1073 (1988). Methods to determine identity a id similarity are codified in computer programs. Computer program method", to determine identity and similarity between two sequences include, but are not limited to, GCG program package (Devereux, J., era/., Nucleic Acids Research 12(l):387<br><br>
-35-<br>
(1984)), BLASTP, BLASTN, FASTA (Atschul, S.F., et al. J Molec Biol 275:403 (1990)).<br>
Therefore, as used herein, the term "identity" represents a comparison between a test and a reference polypeptide or polynucleotide. For example, a test polypeptide can be defined as any polypeptide that is 90% or more identical to a reference polypeptide.<br>
As used herein, the term at least "90% identical to" refers to percent identities from 90 to 99.99 relative to the reference polypeptides. Identity at a level of 90% or more is indicative of the fact that, assuming for exemplification purposes a test and reference polypeptide length of 100 amino acids are compared. No more than 10% (e.g., 10 out of 100) amino acids in the test polypeptide differs from that of the reference polypeptides. Similar comparisons can be made between a test and reference polynucleotides. Such differences can be represented as point mutations randomly distributed over the entire length of an amino acid sequence or they can be clustered in one or more locations of varying length up to the maximum allowable, e.g., 10/100 amino acid difference (approximately 90% identity).  Differences are<br>
defined as nucleic acid or amino acid substitutions, or deletions.<br>
As used herein: stringency of hybridization in determining percentage mismatch is as follows:<br>
1)	high stringency:   0.1 x SSPE, 0.1% SDS, 65Â°C<br>
2)	medium stringency: 0.2 x SSPE. 0.1% SDS, 50Â°C<br>
3)	low stringency:    1.0 x SSPE, 0.1 % SDS, 50Â°C<br>
Those of skill in this art know that the washing step selects for stable hybrids and also know the ingredients of SSPE (see, e.g., Sambrook, E.F. Fritsch. T Maniatis, in: Molecular Cloning. A Laboratory Manual, Cold Spring, Harbor Laboratory Press (1989), vol. 3, p. B.13, see also numerous catalogs that describe commonly used laboratory solutions). SSPE 's pH 7.4 phosphate- buffered, 0.18M NaCI. Further, those of skill in the art recognize that the stability of hybrids is determined by Tm, which is a function of the sodium ion concentration and temperature (Tm = 81.5Â° C-16.6(logio[Na+]) +<br><br>
-36-<br>
0.41(%G+C)-600/l)), so that the only parameters in the wash conditions critical to hybrid stability are sodium ion concentration in the SSPE (or SSC) and temperature.<br>
It is understood that equivalent stringencies can be achieved using alternative buffers, salts and temperatures. By way of example and not limitation, procedures using conditions of low stringency are as follows (see also Shilo and Weinberg, Proc. Natl. Acad. Sci. USA 78:67896792 (1981)): Filters containing DNA are pretreated for 6 hours at 40Â°C in a solution containing 35% formamide. 5X SSC, 50 mM TrisHCI (pH 7.5), 5 mM EDTA, 0.1% PVP, 0.1% Ficoll, 1% BSA, and 500 ig/ml denatured salmon sperm DNA (10X SSC is 1.5 M sodium chloride, and 0.15 M sodium citrate, adjusted to a pH of 7).<br>
Hybridizations are carried out in the same solution with the following modifications: 0.02% PVP, 0.02% Ficoll, 0.2% BSA, 100 ig/ml salmon sperm DNA 10% (wt/vol) dextran sulfate, and 520 X 106 cpm 32Plabeled probe is used. Filters are incubated in hybridization mixture for 1820 hours at 40Â°C, and then washed for 1.5 hours at 55Â°C in a solution containing 2X SSC, 25 mM TrisHCI (pH 7.4), 5 mM EDTA, and 0.1% SDS. The wash solution is replaced with fresh solution and incubated an additional 1.5 hours at 60Â°C. Filters are blotted dry and exposed for autoradiography. If necessary, filters are washed for a third time at 6568Â°C and reexposed to film. Other conditions of low stringency which can be used are well known in the art (e.g., as employed for cross-species hybridizations).<br>
By way of example and not way of limitation, procedures using conditions of moderate stringency include, for example, but are not limited to,<br>
procedures using such conditions of moderate stringency are as follows: filters containing DNA are pretreated for 6 hours at 55Â°C in a solution containing 6X SSC, 5X Denhart's solution, 0.5% SDS and 100 ig/ml denatured salmon sperm DNA. Hybridizations are carried out in the same solution and 520 X 106 cpm 32Plabeled probe is used. Filters are incubated in hybridization mixture for 18-20 hours at 55Â°C, and then washed twice for 30<br><br>
-37-<br>
minutes at 60Â°C in a solution containing 1X SSC and 0.1% SDS. Filters are blotted dry and exposed for autoradiography. Other conditions of moderate stringency which can be used are well-known in the art. Washing of filters is done at 37 C for 1 hour in a solution containing 2X SSC, 0.1% SDS.<br>
By way of example and not way of limitation, procedures using conditions of high stringency are as follows: Prehybridization of filters containing DNA is carried out for 8 hours to overnight at 65Â°C in buffer composed of 6X SSC, 50mM TnsHCI (pH 7.5), 1 mM EDTA, 0.02% PVP, 0.02% Ficoll. 0.02% BSA, and 500 ig/ml denatured salmon sperm DNA. Filters are hybridized for 48 hours at 65;C in prehybridization mixture containing 100 ig/ml denatured salmon sperm DNA and 520 X 106 cpm of 32P!abeled probe. Washing of filters is done at 37Â°C for 1 hour in a solution containing 2X SSC, 0.01% PVP, 0.01% Ficoll, and 0.01% BSA. This is followed by a wash in 0.1 X SSC at 50CC for 45 minutes before autoradiography   Other conditions of high stringency which can be used are well known in the art.<br>
The term substantially identical or substantially homologous or similar varies with the context as understood by those skilled in the relevant art and generally means at least 60% or 70%, preferably means at least 80%, 85% or more preferably at least 90%, and most preferably at least 95% identity.   ,<br>
It is to be understood that the compounds provided herein can contain chiral centers. Such chiral centers can be of either the (R) or (S) configuration, or can be a mixture thereof. Thus, the compounds provided herein can be enantiomerically pure, or be stereoisomeric or diastereomeric mixtures. In the case of amino acid residues, such residues can be of either the L- or D-form   In one embodiment, the configuration for naturally occurring amino acid residues is L.<br>
As used herein, substantially pure means sufficiently homogeneous to appear free of readily detectable impurities as determined by standard methods of analysis, such as thin layer chromatography (TLC), gel electrophoresis, high performance liquid chromatography (HPLC) and mass<br><br>
-38-<br>
spectrometry (MS), used by those of skill in the art to assess such purity, or sufficiently pure such that further purification would not detectably alter the physical and chemical properties, such as enzymatic and biological activities, of the substance. Methods for purification of the compounds to produce substantially chemically pure compounds are known to those of skill in the art. A substantially chemically pure compound can, however, be a mixture of stereoisomers. In such instances, further purification might increase the specific activity of the compound.<br>
As used herein, a cleavable bond or moiety refers to a bond or moiety that is cleaved or cleavable under the specific conditions, such as chemically, enzymatically or photolytically. Where not specified herein, such bond is cleavable under conditions of MALDI-MS analysis, such as by a UV or IR laser.<br>
As used herein, a "selectively cleavable" moiety is a moiety that can be selectively cleaved without affecting or altering the composition of the other portions of the compound of interest. For example, a cleavable moiety L of the compounds provided herein is one that can be cleaved by chemical, enzymatic, photolytic, or other means without affecting or altering composition (e.g.. the chemical composition) of the conjugated biomolecule, including a protein. "Non-cleavable" moieties are those that cannot be selectively cleaved without affecting or altering the composition of the other portions of the compound of interest.<br>
As used herein, binding with high affinity refers to a binding that has an association constant ka of at least 109 and generally 1010 1011 liters/mole or greater) or a Keq of 109,1010, 1011, 1012 or greater. For purposes herein, high affinity bonds formed by the reactivity groups are those that are stable to the laser (UV and IR) used in MALDI-MS analyses.<br>
As used herein, "alkyl", "alkenyl" and "alkynyl", if not specified, contain from 1 to 20 carbons, or 1 to 16 carbons, and are straight or branched carbon chains. Alkenyl carbon chains are from 2 to 20 carbons, and, in certain embodiments, contain 1 to 8 double bonds. Alkenyl carbon chains of 1 to 16<br><br>
-39<br>
carbons, in certain embodiments, contain 1 to 5 double bonds. Alkynyl carbon chains are from 2 to 20 carbons, and, in one embodiment, contain 1<br>
to 8 triple bonds. Alkynyl carbon chains of 2 to 16 carbons, in certain embodiments, contain 1 to 5 triple bonds. Exemplary alkyl, alkenyl and alkynyl groups include, but are not limited to, methyl, ethyl, propyl, isopropyl, isobutyl, n-butyl, sec-butyl, tert-butyl, isopentyl, neopentyl, tert-penytyl and isohexyl. The alkyl, alkenyl and alkynyl groups, unless otherwise specified, can be optionally substituted, with one or more groups, including alkyl group substituents that can be the same or different.<br>
As used herein, "lower alkyl", "lower alkenyl", and "lower alkynyl" refer to carbon chains having less than about 6 carbons.<br>
As used herein, "alk(en)(yn)yl" refers to an alkyl group containing at least one double bond and at least one triple bond.<br>
As used herein, an "alkyl group substituent" includes, but is not limited to, halo, haloalkyi, including halo lower alkyl, aryl, hydroxy, alkoxy, aryloxy, alkyloxy, alkylthio. arylthio, aralkyloxy, aralkylthio, carboxy alkoxycarbonyl, oxo and cycloalkyl.<br>
As used herein, "aryl" refers to aromatic groups containing from 5 to 20 carbon atoms and can be a mono-, multicyclic or fused ring system. Aryl groups include, but are not limited to, phenyl, naphthyla, biphenyl, fluorenyl. and others that can be unsubstituted or are substituted with one or more substituents.<br>
As used herein, "aryl" also refers to aryl-containing groups, including, but not limited to, aryloxy, arylthio, arylcarbonyl and arylamino groups.<br>
As used herein, an "aryl group substituent" includes, but is not limited to, alkyl, alkenyl, alkynyl, cycloalkyi, cycloalkylalkyi, aryl, heteroaryl optionally substituted with 1 or more, including 1 to 3, substituents selected from halo, halo alkyl and alkyl, aralkyl, heteroaralkyl, alkenyl containing 1 to 2 double bonds, alkynyl containing 1 to 2 triple bonds, alk(en)(yn)yl groups, halo, pseudohalo. cyano, hydroxy, haloalkyi and polyhaloalkyl, including halo lower alkyl. especially trifluoromethyl, formyl, alkylcarbonyl, arylcarbonyl that is<br><br>
-40-<br>
optionally substituted with 1 or more, including 1 to 3, substituents selected from halo, halo alkyl and alkyl, heteroarylcarbonyl, carboxy, alkoxycartionyl, aryloxycarbonyl, aminocarbonyl, alkylaminocarbonyl, dialkylaminocarbonyl, arylaminocarbonyl, diarylaminocarbonyl, aralkylaminocarbortyi, alkoxy, aryloxy, perfluoroalkoxy, alkenyloxy, alkynyloxy, arylalkoxy, aminoalkyl, alkylaminoalkyl, dialkylaminoalkyl, arylaminoalkyl, amino, alkylamino, dialkylamino, arylamino, alkylarylamino, alkylcarbonylamino, arylcarbonylamino, azido, nitro, mercapto, alkylthio, arylthio, perfluoroalkylthio, thiocyano, isothiocyanoa, alkylsulfinyl, alkylsulfonyl,<br>
arylsulfinyl, arylsulfonyl, aminosulfonyl, alkylaminosulfonyl, dialkylaminosulfonyl and arylaminosulfonyl.<br>
As used herein, "aralkyl" refers to an alkyl group in that one of the hydrogen atoms of the alkyl is replaced by an aryl group.<br>
As used herein, "heteroaralkyl" refers to an alkyl group in that one of the hydrogen atoms of the alkyl is replaced by a heteroaryl group.<br>
As used herein, "cycloalkyi" refers to a saturated mono- or multicyclic ring system, in one embodiment, of 3 to 10 carbon atoms, or 3 to 6 carbon atoms; cycloalkenyl and cycloalkynyl refer to mono- or multicyclic ring systems that respectively include at least one double bond and at least one triple bond. Cycloalkenyl and cycloalkynyl groups can contain, in one embodiment, 3 to 10 carbon atoms, with cycloalkenyl groups, in other embodiments, containing 4 to 7 carbon atoms and cycloalkynyl groups, in other embodiments, containing 8 to 10 carbon atoms. The ring systems of the cycloalkyi, cycloalkenyl and cycloalkynyl groups can be composed of one ring or two or more rings that can be joined together in a fused, bridged or spiro-connected fashion, and can be optionally substituted with one or more alkyl group substituents. "Cycloalk(en)(yn)yl" refers to a cycloalkyi group containing at least one double bond and at least one triple bond.<br>
As used herein, "heteroaryl" refers to a monocyclic or multicyclic ring system, in one embodiment of about 5 to about 15 members where one or more, or 1 to 3, of the atoms in the ring system is a heteroatom, which is, an<br><br>
-41-<br>
element other than carbon, for example, nitrogen, oxygen and sulfur atoms. The heteroaryl can be optionally substituted with one or more, including 1 to 3, aryl group substituents. The heteroaryl group can be optionally fused to a benzene ring. Exemplary heteroaryl groups include, but are not limited to, pyrroles, porphyrines, furans, thiophenes, selenophenes, pyrazoles, imidazoles, triazoles, tetrazoles, oxazoles, oxadiazoles, thiazoles, thiadiazoles, indoles, carbazoles, benzofurans, benzothiophenes, indazoles, benzimidazoles, benzotriazoles, benzoxatriazoles, benzothiazoles, benzoselenozoles, benzothiadiazoles, benzoselenadiazolss, purines, pyridines, pyridazines, pyrimidines, pyrazines, pyrazines, triazines, quinolines, acridines, isoquinolines, cinnolines, phthalazines, quinazolines, quinoxalines, phenazines, phenanthrolines, imidazinyl, pyrrolidinyl, pyrimidinyl, tetrazolyl, thienyl, pyridyl, pyrrolyl, N-methylpyrrolyl, quinolinyl and isoquinolinyl.<br>
As used herein, "heteroaryl" also refers to heteroaryl-containing groups, including, but not limited to, heteroaryloxy, heteroarylthio, heteroarylcarbonyl and heteroarylamino.<br>
As used herein, "heterocyclic" refers to a monocyclic or multicyclic ring<br>
system, in one embodiment of 3 to 10 members, in another embodiment 4 to 7 members, including 5 to 6 members, where one or more, including 1 to 3 of the atoms in the ring system is a heteroatom, which is, an element other than carbon, for example, nitrogen, oxygen and sulfur atoms. The heterocycle can be optionally substituted with one or more, or 1 to 3 aryl group substituents. In certain embodiments, substituents of the heterocyclic group include hydroxy, amino, alkoxy containing 1 to 4 carbon atoms, halo lower alkyl, including trihalomethyl, such as trifluoromethyl, and halogen. As used herein, the term heterocycle can include reference to heteroaryl.<br>
As used herein, the nomenclature alkyl, alkoxy, carbonyl, etc., are used as is generally understood by those of skill in this art. For example, as used herein alkyl refers to saturated carbon chains that contain one or more carbons; the chains can be straight or branched or include cyclic portions or be cyclic.<br><br>
-42-<br>
Where the number of any given substituent is not specified (e.g.,<br>
"haloalkyl"), there can be one or more substituents present. For example, "haloalkyl" can include one or more of the same or different halogens. As another example, "C1-3alkoxyphenyl" can include one or more of the same or different alkoxy groups containing one, two or three carbons.<br>
Where named substituents such as carboxy or substituents represented by variables such as W are separately enclosed in parentheses, yet possess no subscript outside the parentheses indicating numerical value and that follow substituents not in parentheses, e.g., "C1-4alkyl(W)(carboxy), "W" and "carboxy" are each directly attached to C1-4alkyl.<br>
As used herein, "halogen" or "halide" refers to F, CI, Br or I.<br>
As used herein, pseudohalides are compounds that behave substantially similar to halides. Such compounds can be used in the same manner and treated in the same manner as halides (X, in that X is a halogen, such as CI or Br). Pseudohalides include, but are not limited to, cyanide, cyanate, isocyanate, thiocyanate, isothiocyanate, selenocyanate, trifiuoromethoxy, and azide.<br>
As used herein, "haloalkyl" refers to a lower alkyl radical in that one or more of the hydrogen atoms are replaced by halogen including, but not limited to, chloromethyl, trifluoromethyl, 1chloro2fluoroethyl and the like.   ,<br>
As used herein, "haloalkoxy" refers to RO in that R is a haloalkyl group.<br>
As used herein, "sulfinyl" or "thionyl" refers to S(O). As used herein, "sulfonyl" or "sulfuryl" refers to S(0)2. As used herein, "sulfo" refers to<br>
S(0)20.<br>
As used herein, "carboxy" refers to a divalent radical, C(0)0.<br>
As used herein, "aminocarbonyl" refers to C(0)NH2.<br>
As used herein, "alkylaminocarbonyl" refers to C(0)NHR in that R is hydrogen or alkyl, including lower alkyl.<br>
As used herein "dialkylaminocarbonyl" as used herein refers to C(0)NR R in that R and R are independently selected from hydrogen or alkyl, including lower alkyl.<br><br>
-43-<br>
As used herein, "carboxamide" refers to groups of formula NRCOR. As used herein, "diarylaminocarbonyl" refers to C(0)NRR' in that R<br>
and R' are independently selected from aryl, including lower aryl, such as phenyl.<br>
As used herein, "aralkylaminocarbonyl" refers to C(0)NRR' in that one of Rand R' is aryl. including lower aryl. such as phenyl, and the other of R and R' is alkyl, including lower alkyl.<br>
As used herein, "arylaminocarbonyl" refers to C(0)NHR in that R is aryl, including lower aryl, such as phenyl.<br>
As used herein, "alkoxycarbonyl" refers to C(0)OR in that R is alkyl, including lower alkyl.<br>
As used herein, "aryloxycarbonyl" refers to C(0)OR in that R is aryl, including lower aryl. such as phenyl.<br>
As used herein, "alkoxy" and "alkylthio" refer to RO and RS, in that R is alkyl, including lower alkyl.<br>
As used herein, "aryloxy" and "arylthio" refer to RO and RS, in that R is aryl, including lower aryl. such as phenyl.<br>
As used herein, "alkylene" refers to a straight, branched or cyclic, in one embodiment straight or branched, divalent aliphatic hydrocarbon group, in certain embodiments having from 1 to about 20 carbon atoms, in other embodiments 1 to 12 carbons, including lower alkylene. The alkylene group is optionally substituted with one or more "alkyl group substituents." There can be optionally inserted along the alkylene group one or more oxygen, sulphur or substituted or unsubstituted nitrogen atoms, where the nitrogen substituent is alkyl as previously described. Exemplary alkylene groups include methylene (CH2), ethylene (CH2CH2), propylene (â(CHzb), cyclohexylene (C6Hi0), methylenedioxy (OCH2O) and ethylenedioxy (0(CH2)20). The term "lower alkylene" refers to alkylene groups having 1 to 6 carbons. In certain embodiments, alkylene groups are lower alkylene, including alkylene of 1 to 3 carbon atoms.<br>
As used herein, "alkenylene" refers to a straight, branched or cyclic, in<br><br>
-44-<br>
one embodiment straight or branched, aliphatic hydrocarbon group, in certain embodiments having from 2 to about 20 carbon atoms and at least one<br>
double bond, in other embodiments 1 to 12 carbons, including lower alkenylene. The alkenylene group is optionally substituted with one or more "alkyl group substituents." There can be optionally inserted along the alkenylene group one or more oxygen, sulphur or substituted or unsubstituted<br>
nitrogen atoms, where the nitrogen substituent is alkyl as previously described. Exemplary alkenylene groups include âCH=CHâCH=CHâ and CH=CHCH2. The term "lower alkenylene" refers to alkenylene groups having 2 to 6 carbons.  !n certain embodiments, alkenylene groups are lower alkenylene, including alkenylene of 3 to 4 carbon atoms.<br>
As used herein, "alkynylene" refers to a straight, branched or cyclic, in one embodiment straight or branched, divalent aliphatic hydrocarbon group, in certain embodiments having from 2 to about 20 carbon atoms and at least one triple bond, in other embodiments 1 to 12 carbons, including lower alkynylene. The alkynylene group is optionally substituted with one or more "alkyl group substituents." There can be optionally inserted along the alkynylene group one or more oxygen, sulphur or substituted or unsubstituted nitrogen atoms, where the nitrogen substituent is alkyl as previously described. Exemplary alkynylene groups include âC=CâC=Câ, C=C and C=CCH2. The term "lower alkynylene" refers to alkynylene groups having 2 to. 6 carbons. In certain embodiments, alkynylene groups are lower alkynylene, including alkynylene of 3 to 4 carbon atoms.<br>
As used herein, "alk(en)(yn)ylene" refers to a straight, branched or cyclic, in one embodiment straight or branched, divalent aliphatic hydrocarbon group, in certain embodiments having from 2 to about 20 carbon atoms and at least one triple bond, and at least one double bond; in other embodiments 1 to 12 carbons, including lower alk(en)(yn)ylene. The alk(en)(yn)ylene group is optionally substituted with one or more "alkyl group substituents." There can be optionally inserted along the alkynylene group one or more oxygen, sulphur or substituted or unsubstituted nitrogen atoms, where the nitrogen<br><br>
â¢45-<br>
substituent is alkyl as previously described. Exemplary alk(en)(yn)ylene groups include âC=Câ(CH2)n C=Câ, where n is 1 or 2. The term "lower<br>
alk(en)(yn)ylene" refers to alk(en)(yn)ylene groups having up to 6 carbons. In certain embodiments, alk(en)(yn)ylene groups are lower alk(en)(yn)yl3ne, including alk(en)(yn)ylene of 4 carbon atoms.<br>
As used herein, "arylene" refers to a monocyclic or polycyclic, in one embodiment monocyclic, divalent aromatic group, in certain embodiments having from 5 to about 20 carbon atoms and at least one aromatic ring, in other embodiments 5 to 12 carbons, including lower arylene. The arylene group is optionally substituted with one or more "alkyl group substituents." There can be optionally inserted around the arylene group one or more oxygen, sulphur or substituted or jnsubstituted nitrogen atoms, where the nitrogen substituent is alkyl as previously described. Exemplary arylene groups include 1,2, 1,3- and 1,4-phenylene. The term "lower arylene" refers to arylene groups having 5 or 6 carbons. In certain embodiments, arylene groups are lower arylene.<br>
As used herein, "heteroarylene" refers to a divalent monocyclic or multicyclic ring system, in one embodiment of about 5 to about 15 members where one or more, or 1 to 3 of the atoms in the ring system is a heteroatom, which is, an element other than carbon, for example, nitrogen, oxygen and sulfur atoms. The heteroarylene group can be optionally substituted with one or more, or 1 to 3, aryl group substituents.<br>
As used herein, "alkylidenc" refers to a divalent group, such as =CR'R", which is attached to one atom of another group, forming a double bond. Exemplary alkylidene groups are methylidene (=CH2) and ethylidene (=CHCH3). As used herein, "aralkylidene" refers to an alkyiidene group in that either R' or R" is and aryl group.<br>
As used herein, "amido" refers to the divalent group C(0)NH. "Thioamido" refers to the divalent group C(S)NH. "Oxyarrido" refers to the divalent group OC(0)NH. "Thiaamido" refers to the divalent group SC(0)NH.<br>
"Dithiaamido" refers to the divalent group SC(S)NH. "Ureido" refers to the<br><br>
46-<br>
divalent group HNC(0)NH. "Thioureido" refers to the divalent group HNC(S)NH.<br>
As used herein, "semicarbazide" refers to NHC(0)NHNH. "Carbazate" refers to the divalent group OC(0)NHNH. "Isothiocarbazate" refers to the divalent group SC(0)NHNH. "Thiocarbazate" refers to the divalent group OC(S)NHNH. "Sulfonylhydrazide" refers to the group S02NHNH. "Hydrazide" refers to the divalent group C(0)NHNH. "Azo" refers to the divalent group N=N. "Hydrazinyl" refers to the divalent group NHNH.<br>
As used herein, the term "amino acid" refers to a-amino acids that are racemic, or of either the D- or L-configuration. The designation "d" preceding an amino acid designation (e.g., dAla, dSer, dVal, etc.) refers to the D-isomer of the amino acid   The designation "dl" preceding an amino acid designation (e.g., dIAIa) refers to a mixture of the L- and D-isomers of the amino acid.<br>
As used herein, when any particular group, such as phenyl or pyridyl, is specified, this means that the group is unsubstituted or is substituted. Substituents where not specified ire halo, halo lower alkyl, and lower alkyl.<br>
As used herein, conformationally altered protein disease (or a disease of protein aggregation) refers to diseases associated with a protein or polypeptide that has a disease-associated conformation. The methods and collections provided herein permit detection of a conformer associated with a disease to be detected.   Diseases and associated proteins that exhibit two or more different conformations in which at least one conformation is a conformationally altered protein include, but are not limited to, amyloid diseases and other neurodegenerative diseases known to those of skill in the art and set forth below.<br>
As used herein, cell sorting refers to an assay in which cells are separated and recovered from suspension based upon properties measured in flow cytometry analysis. Most assays used for analysis can serve as the basis for sorting experiments, as long as gates and regions defining the subpopulation(s) to be sorted do not logically overlap. Maximum throughput rates are typically 5000 cells/second (18 x 106 cells/hour). The rate of<br><br>
-47-<br>
collection of the separated population(s) depends primarily upon the condition of the cells and the percentage of reactivity.<br>
As used herein, the abbreviations for any protective groups, amino acids and other compounds, are, unless indicated otherwise, in accord with their common usage, recognized abbreviations, or the IUPAC-IUB Commission on Biochemical Nomenclature (see, Biochem. 1972, 11:942). FOR example, DMF = A/,A/-dimethylformamide, DMAc = N,N-dimethylacetamide; THF = tetrahydrofuran; TRIS =<br>
tris(hydroxymethyl)aminomethane; SSPE = saline-sodium phosphate-EDTA buffer; EDTA = ethylenediaminetetraacetic acid; SDS = sodium dodecyl sulfate. B.       Collections of capture compounds<br>
Collections of capture compounds that selectively bind to biomolecules in samples, such as biomoelcules, particularly, although not exclusively, a cell lysate or in vitro translated polypeptides from a cell lysate are provided. Each capture compound in the collection can bind to specific groups or classes of biopolymers. and is designed to covalently or tightly (sufficient to sustain mass spectrometric analysis, for example) to a subset of all of the biomolecules in the sample.   For example, a sample can contain 1000's of members, for example a cell lysate. The collections of compounds permit sufficient selectivity so that, for example, about 10-20 of the components of the sample bind to each member of the collection. The exact number is a small enough number for routine analyses to identify them, generally in one step, such as by mass spectrometry.<br>
As described in greater detail below, the compounds provided herein are multifunctional synthetic small molecules that can select, covalently bind ("capture") and isolate proteins based on their unique surface features. The solubility of the compound may be modulated in the chemical synthesis process such that water soluble (cytosolic) or insoluble (membrane) protein mixtures may be analyzed. In one embodiment, the compound employs three critical functionalities: (1) a reactivity function; (2) a selectivity function; and<br><br>
-48<br>
(3) a sorting function.<br>
As shown in Figure 27, the selectivity function interacts via non-covalent interactions with a protein e.g. in the active site of enzymes or ligand binding site of receptors ("Biased approach" for e.g. non-target identification), or at a surface affinity motif (SAM) outside of the binding site ("Unbiased approach" for e.g. target discovery). A biased selectivity group enables isolation of specific proteins from complex mixtures. In one embodiment, the selectivity function is a drug (or metabolite thereof) known to cause side effects, attached in several different orientations to make different parts of the molecule accessible to proteins. An unbiased selectivity function utilizes chemical features underlying affinity interactions with the protein surface. The unbiased selectivity function tends to be less specific than the biased, since it is designed to interact with a a broader set of proteins. Use of the unbiased capture compounds to screen for global protein profile differences between healthy and disease cells would require the development of a library of capture compounds which as a set interact with the majority of the proteins in the proteome. This approach enables monitoring of protein profile differences induced by the influence of a drug molecule, or discovering new potential drug targets or biomarkers based on the differences between healthy with disease cells.<br>
In one embodiment, the reactivity function covalently "captures" or<br>
binds to the selected protein. While the selectivity function serves as the bait,<br>
the reactivity function serves as the hook. A protein thus captured will be able to survive downstream purification and analytical processes. Reactivity functions employed are chemically reactive with certain protein side chains (e.g. NHS forms bond with lysine amino function), or require an activation<br>
step (i.e.light) prior to forming covalent bond (e.g. photoactivated moiety such as azide which forms a nitrene radical).<br>
In another embodiment, the sorting (pull-out) function isolates the specific protein from its complex cellular environment using a solid support (e.g. magnetic bead, DNA chip), enabling subsequent structural and<br><br>
-49-<br>
functional characterization.<br>
In another embodiment, the analytical process (Figure 30) is simple<br>
and highly amenable to automation. First, a protein mixture from the cells of interest is incubated with a capture compound in buffer conditions which retain the native structural features of the proteins. The selectivity function reversibly interacts and comes to equilibrium with those proteins for which it has an affinity. The reactivity function then forms a covalent bond irreversibly linking the compound to those proteins for which there was an affinity. Our data indicates that the higher the affinity between the protein and the capture compound, the higher is the percentage covalently captured. Next, the covalently captured proteins are isolated onto a solid support and the uncaptured cellular components and proteins washed away. If the sorting function chosen is a biotin, then avidin or streptavidin beads are used as the<br>
solid support. Mass spectrometry (MS) is used to detect the captured<br>
proteins.<br>
In certain embodiments, with its speed and precision (Mr measured to 0 01% - 0.10%), separating capabilities (even small structural variation lead to mass shift) and ability to multiplex (many proteins scanned simultaneously), MS is used for protein identification. This initial mass spectrum provides the molecular weights of all proteins captured. The identity of each can then be determined by conventional means (e.g. digestion and analysis or peptide fragments and genome/proteome database searches). Use of the capture compounds allows the researcher to further analyze and characterize the protein, since it is physically isolated from all others (e.g. mass spectrum identification, or x-ray crystallography after removal from beads). To do so, the protein is washed from the solid support (e.g., if using avidin / streptavidin beads, treat the beads with biotin to displace captured proteins) or make use of an incorporated photocleavable linker, or enzymatically or chemically cleavable linker, thereby releasing the captured purified protein from the solid support.<br>
The collections permit a top down holistic approach to analysis of the<br><br>
-50-<br>
proteome, including post-translationally modified proteins, and other biomolecules. Protein and other biomolecule patterns are the starting point<br>
for analyses that use these collections; rather than nucleic acids and the genome (bottom up). The collections can be used to assess the biomolecule components of a sample, such as a biological sample, to identify components<br>
specific to a particular phenotype, such as a disease state, to identify<br>
structural function, biochemical pathways and mechanisms of action. The collections and methods of use permit an unbiased analysis of biomolecules, since the methods do not necessarily assess specific classes of targets, --instead, changes in samples are detected or identified. The collections permit the components of a complex mixture of biomolecules (i.e., a mixture of 50, 100, 500, 1000, 2000 and more) to be sorted into discrete loci containing reduced numbers, typically by 10%, 50% or greater reduction in complexity, or to about 1 to 50 different biomolecules per locus in an array, so that the components at each spot can be analyzed, such as by mass spectrometric analysis alone or in combination with other analyses. In some embodiments, such as for phenotypic analyses, homogeneity of the starting sample, such as cells, can be important. To provide homogeneity, cells, with different phenotypes, such as diseased versus healthy, from the same individual are compared.   Methods for doing so are provided herein.<br>
By virtue of the structure of compounds in the collections, the collections can be used to detect structural changes, such as those from the post-translational processing of proteins, and can be used to detect changes in membrane proteins, which are involved in the most fundamental processes, such as signal transduction, ion channels, receptors for ligand interaction and cell-to-cell interactions. When cells become diseased, changes associated with disease, such as transformation, often occur in membrane proteins.<br>
The collections contain sets of member capture compounds. In general, members of each set differ in at least one functional group, and generally in two or three, from members of the other sets. Thus, for example,<br><br>
-51-<br>
if the compounds include a reactivity function, a selectivity function and a sorting function, each set differs in at least the sorting function, typically in at least in the sorting and selectivity function, and generally in all three functions. The solubility functions, if present, which are selected to permit assaying in a selected environment, can differ among the compounds, or can be the same among all sets.<br>
In practicing methods, the collections are contacted with a sample or partially purified or purified components thereof to effect binding of biomolecules to capture compounds in the collection. The capture compounds can be in an addressable array, such as bound to a solid support prior to contacting, or can be arrayed after contacting with the sample. The resulting array is optionally treated with a reagent that specifically cleaves the bound polymers, such as a protease, and is subjected to analysis, particularly mass spectrometric analysis to identify components of the bound biomolecules at each locus. Once a molecular weight of a biomolecule, such as a protein or portion thereof of interest is determined, the biomolecule can be identified. Methods for identification include comparison of the molecular weights with databases, for example protein databases that include protease fragments and their molecular weights.<br>
The capture compounds that include functional groups that confer reactivity, selective and separative properties, depending on the specificity of separation and analysis required (which depends on the complexity of the mixture to be analyzed). As more functional groups are added to the compounds, the compounds can exhibit increased selectivity and develop a signature for target molecules similar to an antigen (Ag) binding site on an antibody. In general, the compounds provided herein include at least two functional groups (functions) selected from four types of functions: a reactivit function, which binds to biopolymers either covalently or with a high ka (generally greater than about 109, 1010, 1012 liters/mole and/or such fruit the binding is substantially irreversible or stable under conditions of mass spectrometric analyses, such as MALDI-MS conditions); a selectivity function<br><br>
-52-<br>
which by virtue of non-covalent interactions alters, generally increases, the specificity of the reactivity function; a sorting function, which permits the<br>
compounds to be addressed (arrayed or otherwise separated based according to the structure of the capture compound; and a solubility function, which when selected alters the solubility of the compounds depending upon the environment in which reactions are performed, permitting the conditions to simulate physiological conditions. In general, the reactivity function is the reactive group that specifically interacts, typically covalently or with high binding affinity (ka), with particular biomolecules, such as proteins, or portions thereof; and the other functionality, the selectivity functions, alters, typically increasing, the specificity of the reactivity function. In general, the reactive function covalently interacts with groups on a particular biomolecule, such as amine groups on the surface of a protein.   The reactivity function interacts with biomolecules to form a covalent bond or a non-covalent bond that is stable under conditions of analysis, generally with a ka of greater than 109 liters/mole or greater than 1010 liters/mole. Conditions of analysis include, but are not limited to, mass spectrophotometric analysis, such as matrix assisted laser desorption ionization-time of flight (MALDI-TOF) mass spectrometry. The selectivity function influences the types of biomolecules that can interact with the reactivity function through a non-covalent interaction. The selectivity function alters the specificity for the particular groups, generally reducing the number of such groups with which the reactivity functions react. A goal is to reduce the  number of proteins or biomolecules bound at a locus, so that the proteins can then be separated, such as by mass spectrometry.<br>
Included among the capture compounds provided herein are those that can . the compounds for use in the methods herein can be classified in at least two sets: one for reactions in aqueous solution (e.g., for reaction with hydrophilic biomolecules), and the other for reaction in organic solvents (e.g., chloroform)(e.g., for reaction with hydrophobic biomolecules). Thus, in certain embodiments, the compounds provided herein discriminate between hydrophilic and hydrophobic biomolecules, including, but not limited to,<br><br>
-53-<br>
proteins, and allow for analysis of both classes of biomolecules. C.      Capture Compounds<br>
Capture compounds (also referred to as capture agents) are provided. The capture con-pounds include a core "Z" that presents one or more reactivity functions "X" and optionally at least a selectivity function "Y" and/or a sorting function "Q", and also optionally one or more solubility functions "W." Additionally, cleavable linkers and other functions are included in the molecules. The particular manner in which the functions are presented on the core or scaffold is a matter of design choice, but are selected such that the resulting molecule has the property that it captures biomolecules, particularly proteins, with sufficient specificity and either covalently or with bonds of sufficient stability or affinity to permit analysis, such as by mass spectrometry, including MALDI mass spectrometric analysis, so that at least a portion of bound biomolecules remain bound (generally a binding affinity of 109, 1010. 1011 liters/mole or greater, or a Keq of 109. 1010, 1Q11, 1012or greater).<br>
X. the reactivity functionality, is selected to be anything that forms such a covalent bond or a bond of high affinity that is stable under conditions of mass spectrometric analysis, particularly MALDI analysis. The selectivity functionality Y, is a group that "looks" at the topology of the protein around reactivity binding sites and functions to select particular groups on biolmolecules from among those with which a reactivity group can form a<br>
covalent bond (or high affinity bond). For example, a selectivity group can cause steric hindrance, or permit specific binding to an epitope, or anything in between   It can be a substrate for a drug, lipid, peptide. It selects the environment of the groups with which the reactivity function interacts.   The selectivity functionality Y, can be one whereby a capture compound forms a cova ent bond with a biomolecule in a mixture or interacts with high stability such that the affinity of binding of the capture compound to the biomolecule through the reactive functionality in the presence of the selectivity functionality is at east ten-fold or 100-fold greater than in the absence of the selectivity<br><br>
-54-<br>
functionality.<br>
Q is a sorting function that can be anything that provides a means for separating each set of capture compounds from the others, such as by arraying, and  includes, groups such as biotin, generally a spacer, binding to an avidin on a surface (or vice versa) array, oligonucleotides for binding oligonucleotide arrays or any molecule that has a cognate binding partner to which it binds with sufficient affinitity to survive mass spectrometric analysis, such as MALDI-MS analysis, can be selected. For any collection a variety of different sorting groups can be used; each set of capture compounds should have unique Q compared to the other sets.   In addition, labeling means that can be sorted by virtue of the label, such as RF tags, fluroescent tags, color-coded tags or beads, bar-coded or other symbology labeled tags and other such labels can be used. For example, the capture compounds or the X, Y. Z, W functionalities can be on a surface that is attached to an RF tag or a colored tag. These can be readily sorted after reaction so that each set can be ceparately analyzed to identify bound biomolecules. Thus, the collections can include capture compounds that have a variety of sorting groups.<br>
The solubility function. W, permits alteration in properties of the capture compound components of the collection.  For example, W can be selected so that the capture compounds are soluble or not in a particular reaction medium or environment, such as a hydrophobic environment, thereby permitting reactions with membrane components. The collections include sets of capture compounds, each of which set differs in Q and at least one or both X and Y.<br>
As noted, among the capture compounds provided are those with at<br>
least hree functionalities: reactivity, sorting and solubility. The sorting function can be selectively cleavable to permit its removal. These compounds also can include a selectivity function to alter the range of binding of the reactivity function, which binds either covalently or with high affinity (ka greater than 109 to biomolecules, and optionally one or both of a sorting and<br><br>
-55-<br>
solubility function.<br>
More detailed description and discussion of each functionality and non-limiting exemplary embodiments follow.<br>
1.       Z, the Core<br>
Generally all compounds include a function, even if it is one atom, such as carbon, for presenting the functional groups. In certain embodiments herein, in the capture compounds for use in the methods provided herein, Z is a moiety that is cleavable prior to or during analysis of the biomolecule, including mass spectral analysis, without altering the chemical structure of the biomolecule. including, but not limited to, a protein.<br>
In certain embodiments, Z is a trifunctional moities containing three functionalities that are each capable of being derivatized selectively in the presence of the other two functionalities. Non-limiting examples of such trifunctional moieties include but are not limited to trifunctionalized trityl groups and amino acids that possess a functionality on the side chain (e.g.,<br>
tyrosine, cysteine, aspartic acid, glutamic acid, lysine, threonine, serine, etc.).<br>
Such amino acids include natural and non-natural amino acids.<br>
For example, in some embodiments, the methods provided herein include a step of mass spectral analysis of biomolecules, including proteins, which are displayed in an addressable format. In certain embodiments, the  -compounds are then bound to an array of single oligonucleotides that include single-stranded portions (or portions that can be made single-stranded) that are complementary to the oligonucleotide portions, or oligonucleotide analog portions, (Q. the sorting function) of the capture compounds. In these embodiments, Z can be selected to be a group that is (i) stable to the reaction conditions required for reaction of the compounds provided herein with the biomolecule, such as a protein, (ii) stable to the conditions required for hybridization of the Q moiety with the single stranded oligonucleotides, and (iii) cleavable prior to or during analysis of the biomolecule.<br>
In another embodiment, Z with the linked functional groups can be designed so that with the Q, X, W and/or Y it dissolved into lipid bilayers of a<br><br>
-56-<br>
cell membrane, thereby contacting internal portions of cell membrane proteins through the X and Y functions. In this embodiment, the support<br>
captures proteins, such as membrane proteins and organelle proteins, including proteins within cell membranes. The capture compounds and<br>
functional group can be selected so that the resulting capture compounds function under selected physiological conditions. Thus, the choice of Z, Q, X,<br>
W .and/or Y allows for design of surfaces and supports that mimic cell<br>
membranes and other biological membranes.<br>
In some embodiments, a lipid bilayer, such as as those used for forming liposomes and other micelles, can be provided on the surface of a support as a way of maintaining the structures of membrane proteins to make a lipid bilayer on the surface.   This can be employed where the support is the "Z" function and the other functions are linked thereto, or where the compounds are linked to a support through a Q group, such as by double-stranded oligonucleotides. The resulting immobilized capture compounds can be coated with or dissolved in a lipid coating. As a result, the compounds and collections provided herein can act as an artificial membrane, dendrimer polymer chemistry can be employed for controlled synthesis of membranes having consistent pore dimensions and membrane thicknesses, through synthesis of amphiphilic dendrimeric or hyperbranched block copolymers that can be self-assembled to form ultrathin organic film membranes on porous supports. In one embodiment, an organic film membrane is composed of a linear-dendritic diblock copolymer composed of polyamidoamine (PAMAM) dendrimer attached to one end of a linear polyethylene oxide (PEO) block.<br>
Z is cleavable under the conditions of mass<br>
spectrometric analysis<br>
In one such embodiment, Z is a photocleavable group that is cleaved by a laser used in MALDI-TOF mass spectrometry. In another embodiment, Z is an acid labile group that is cleaved upon application of a matrix for mass spectrometric analysis to arrayed, such as hybridized compound-biomolecule conjugates, or by exposure to acids {e.g., trifluoroacetic or hydrochloric acids)<br><br>
-57-<br>
in a vapor or liquid form, prior to analysis. In this embodiment, the matrix maintains the special integrity of the array, allowing for addressable analysis<br>
of the array.<br>
Z is not cleavable under the conditions of mass<br>
spectrometric analysis<br>
In certain embodiments, the capture compounds for use in the methods provided herein have a Z moiety that is not cleavable under conditions used for analysis of biomolecules, including, but not limited to, mass spectrometry, such as matrix assisted laser desorption ionization-time of flight (MALDI-TOF) mass spectrometry. Capture compounds of these embodiments can be used, for example, in methods provided herein for identifying biomolecules in mixtures thereof, for determining biomolecule-biomolecule. including protein-protein, interactions, and for determining biomolecule-small molecule, including protein-drug or protein-drug candidate, interactions   In these embodiments, it is not necessary for the Z group to be cleaved for the analysis.<br>
Thus, as noted, Z can be virtually any moiety that serves as a core to present the binding (the selectivity and reactivity functions) and the solubility and sorting functions.  A variety are exemplified herein, but others may be substituted.   The precise nature can be a matter of design choice in view of the disclosure herein and the skill of the skilled artisan <br>
a.       Multivalent or Divalent Z moieties<br>
In one embodiment, Z is a cleavable or non-cleavable multivalent or divalent group that contains, generally 50 or fewer, or less than 20 members, and is selected from straight or branched chain alkylene, straight or branched chain alkenylene, straight or branched chain alkynylene, straight or branched chain alkylenoxy, straight or branched chain alkylenthio, straight or branched chain alkylencarbonyl, straight or branched chain alkylenamino, cycloalkylene, cycloalkenylene, cycloalkynylene, cycloalkylenoxy, cycloalkylenthio. cycloalkylencarbonyl, cycloalkylenamino, heterocyclylene, arylene, arylenoxy, arylenthio, arylencarbonyl, arylenamino, heteroarylene,<br><br>
-58<br>
heteroarylenoxy, heteroarylenthio, heteroarylencarbonyl, heteroarylenamino, oxy, thio. carbonyl, carbonyloxy, ester, amino, amido, phosphino,<br>
phosphineoxido, phosphoramidato, phosphinamidato, sulfonamido, sulfonyl, sulfoxido, carbamate ureido, and combinations thereof, and is optionally substituted with one or more, including one, two, three or four, substituents each independently selected from Y, as described elsewhere herein.<br>
In other embodiments, Z is a multivalent or divalent cleavable or non-cleavable group selected from straight or branched chain alkyl, straight or branched chain alkenyl, straight or branched chain alkynyl, (C(R15)2)d, O, S, (CH2)d, (CH2)dO, (CH2)dS, &gt;N(R15), (S(0)u). (S(0)2)w, &gt;C(0), (C(0))W, (C(S(0)u))w. (C(0)0)W. (C(R15)2)dO, (C(R15)2)dS(0)u, 0(C(R15)2)d, S(0)u(C(R15)2)d, (C(R15)2)dO(C(R15)2)d,(C(R15)2)dS(0)u(C(R15)2)d, N(R15)(C(R15)2)d, (C(Rl5)2)dNR15, (C(R15)2)dN(R15)(C(R15)2)d. -(CH2)dC(0)N(CH2)d-.-(CH2)dC(0)N(CH2)dC(0)N(CH2)d-, (S(R15)(Ou)w, (C(R15)2)d, (C(R15)2)dO(C(R15)2)d, (C(R15)2)d(C(0)0)w(C(R15)2)d,<br>
(C(0)0)â(C(R,5)2)d, (C(R15)2)d(C(0)0)w, (C(S)(R15)w, (C(0))w(CR152)d, (CRl5)d(C(0))w(CR15)d, (C(R15)2)d(C(0))w, N(R15)(C(R15)2)w, OC(R15)2C(0), 0((R15)2C(0)N(R15), (C(R15)2)wN(R15)(C(R15)2)w, (C(R15)2)wN(R15), &gt;P(0)v(R15)x, &gt;P(0)u(R15)3, &gt;P(0)u(C(R15)2)d, &gt;Si(R15)2 and combinations of any of these groups:<br>
where u, v and x are each independently 0 to 5;<br>
each d is independently an integer from 1 to 20, or 1 to 12, or 1-6, or 1 to 3;<br>
each w is independently an integer selected from 1 to 6, or 1 to 3, or 1 to 2; and<br>
each R15 is independently a monovalent group selected from straight or branched chain alkyl, straight or branched chain alkenyl, straight or branched chain alkynyl, cycloalkyl, cycloalkenyl, cycloalkynyl, heterocyclyl, straight or branched chain heterocyclylalkyl, straight or branched chain heterocyclylalkenyl, straight or branched chain heterocyclylalkynyl, aryl, straight or branched chain arylalkyl, straight or branched chain arylalkenyl,<br><br>
-59-<br>
straight or branched chain arylalkynyl, heteroaryl, straight or branched chain heteroarylalkyl, straight or branched chain heteroarylalkenyl, straight or branched chain heteroaryialkynyl, halo, straight or branched chain haloalkyi, pseudohalo, azido, cyano, nitro, OR60, NR60R61, COOR60, C(0)R60, C(O)NR60R61, S(0)qR60 S(0)qOR60, S(O)qNR60R61, NR60C(O)R61, NR60C(O)NR60R61, NR60S(O)qR60, SiR60R61R62, P(R60)2l P(O)(R60)2, P(OR60)2, P(O)(OR60)2, P(O)(OR60)(R61) and P(O)NR60R61, where q is an integer from 0<br>
to 2;<br>
each R60, R61, and R62 is independently hydrogen, straight or branched<br>
chain alkyl, straight or branched chain alkenyl, straight or branched chain alkynyl, aryl, straight or branched chain aralkyl, straight or branched chain aralkenyl, straight or branched chain aralkynyl. heteroaryl, straight or branched chain heteroaralkyl, straight or branched chain heteroaralkenyl, straight or branched chain heteroaralkynyl, heterocyclyl, straight or branched chain heterocyclylaIkyI, straight or branched chain heterocyclylalkenyl or straight or branched chain heteorcyclylalkynyl.<br>
In other embodiments, Z is a cleavable or non-cleavable multivalent divalent group having any combination of the following groups: arylene, heteroarylene, cycloalkylene, &gt;C(R15)2. C(R15)=C(R15), &gt;C=C(R23)(R24), &gt;C(R23)(R24), C=C, O, &gt;S(A)U, &gt;P(D)V(R15), &gt;P(D)V(ER15), &gt;N(R15), &gt;N+(R23)(R24), &gt;Si(R15)2 or &gt;C(E); where u is 0, 1 or 2; v is 0, 1, 2 or 3; A is O or NR15; D is S or O; and E is S, O or NR15; that groups can be combined in any order;<br>
each R15 is a monovalent group independently selected from the group consisting of hydrogen and VR18;<br>
each V is a divalent group independently having any combination of the following groups: a direct link, arylene, heteroarylene, cycloalkylene, &gt;C(R17)2, C(R'7)=C;R17), &gt;C=C(R23)(R24), &gt;C(R23)(R24), C=C, O, &gt;S(A)U, &gt;P(D)V(R17), &gt;P(D) (ER17), &gt;N(R17), &gt;N(COR17), &gt;N+(R23)(R'4), &gt;Si(R17)2 and &gt;C(E); where u is 0, 1 or 2; v is 0, 1, 2 or 3; A is O or NR17; D is S or O; and E is S, O or NR17; thct groups can be combined in any order;<br><br>
-60-<br>
R17 and R18 are each independently selected from the group consisting of hydrogen, halo, pseudohalo, cyano, azido, nitro, SiR27R28R25, alkyl, alkenyl<br>
alkynyl, haloalkyl, haloalkoxy, aryl, aralkyl, aralkenyl, aralkynyl, heteroaryl, heteroaralkyl, heteroaraalkenyl, heteroaralkynyl, heterocyclyl, heterocyclylalkyi,<br>
heterocyclylalkenyl, heterocyclylalkynyl, hydroxy, alkoxy, aryloxy, aralkoxy, heteroaralkoxy and NR19R20;<br><br>
R19   and R20   are each independently selected from hydrogen, alkyl, alkenyl, alkynyl, cycloalkyl, aryl, aralkyl, heteroaryl, heteroaralkyl and heterocyclyl;<br>
R23 and R24 are selected from (i) or (ii) as follows:<br>
(i) R23 and R24 are independently selected from the group consisting of hydrogen, alkyl, alkenyl, alkynyl, cycloalkyl, aryl and heteroaryl; or<br>
(ii) R23 and R24 together form alkylene, alkenylene or cycloalkylene;<br><br>
R25   , R27   and R28   are each independently a monovalent group selected from hydrogen, alkyl, alkenyl, alkynyl, haloalkyl, haloalkoxy, aryl, aralkyl, aralkenyl, aralkynyl, heteroaryl, heteroaralkyl, heteroaralkenyl, heteroaralkynyl, heterocyclyl, heterocyclylalkyi, heterocyclylalkenyl, heterocyclylalkynyl, hydroxy, alkoxy, aryloxy, aralkoxy, heteroaralkoxy and NR19R20;<br>
R15, R17, R18, R19, R20, R23, R24, R25, R27 and R28 can be substituted with one or more substituents each independently selected from Z2, in that Z2 is selected from alkyl, alkenyl, alkynyl, aryl, cycloalkyl, cycloalkenyl, hydroxy, S(0)hR35 in that h is 0, 1 or 2, NR35R36, COOR35, COR35, CONR35R36, OC(0)NR35R36, N(R35)C(0)R36, alkoxy, aryloxy, heteroaryl, heterocyclyl, heteroaryloxy, heterocyclyloxy, aralkyl, aralkenyl, aralkynyl, heteroaralkyl,<br>
heteroaralkenyl, heteroaralkynyl, aralkoxy, heteroaralkoxy, alkoxycarbonyl, carbamoyl, thiocarbamoyl, alkoxycarbonyl, carboxyaryl, halo, pseudohalo, haloalkyl and carboxamido:<br>
R35 an d R36 are each independently selected from among hydrogen, halo, pseudohalo, cyano, azido, nitro, trialkylsilyl, dialkylarylsilyl, alkyldiarylsily!, triarylsilyl, alkyl, alkenyl, alkynyl, haloalkyl, haloalkoxy, aryl,<br><br>
-61-<br>
aralkyl, aralkenyl, aralkynyl, heteroaryl, heteroaralkyl, heteroaralkenyl. heteroaralkynyl, heterocyclyl, heterocyclylalkyl, heterocyclylalkenyl,<br>
heterocyclylalkynyl. hydroxy, alkoxy, aryloxy, aralkoxy, heteroaralkoxy. amino, amido, alkylamino, dialkylamino, alkylarylamino, diarylamino and arylamino.<br>
In certain embodiments herein, the compounds are selected with the proviso that Z is cleavable prior to or during analysis, including mass spectral analysis, such as matrix assisted laser desorption ionization-time of flight (MALDI-TOF) mass spectrometry, of the biomolecule.<br>
In certain embodiments, Z is at least a trivalent moiety selected from the divalent moieties disclosed herein absent at least one hydrogen. The capture compounds in the collections provided herein include a core Z that has a variety of valencies. Among the capture compounds are those in which Z is at least trivalent. Also among the compounds in the collections are those where Z is divalent and linked to either a Q and an X, or a Q and a Y or an X and a Y. or other combination of the functionalities provided herein.<br>
(i)       Cleavable multivalent or divalent Z moieties<br>
In one embodiment, Z is a cleavable multivalent or divalent moiety and<br>
has the formula: (S1)tM(R15)a(S2)bL, where S1 and S2 are spacer moieties; t<br>
and b are each independently 0 or 1; M is a central moiety possessing two or more points of attachment {i.e., divalent or higher valency); in certain embodiments, two to six points of attachment {i.e., divalent to hexavalent), in other embodiments, 2. 3, 4 or 5 points of attachment {i.e., divalent, trivalent,<br>
tetravalent or pentavalent); R15 is as described above; a is 0 to 4, in certain embodiments. 0, 1 or 2; and L is a bond that is cleavable prior to or during analysis, including mass spectral analysis, of a biomolecule without altering the chemical structure of the biomolecule, such as a protein.<br>
(a)      M<br>
In certain embodiments, M is alkylene, phenylene, biphenylene or a multivalent or divalent heterobifunctional trityl derivative. M is unsubstituted or is substituted with 1 to 4 groups, each independently selected from R15.<br>
In other embodiments, M is selected from (CH2)r, (CH20)r,<br><br>
-62-<br>
,<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
-63-<br>
from 1 to 10; R52 is the side chain of a natural or unnatural a-amino acid; and z is an integer from 1 to 4. In one embodiment n1, n2, n3 are each<br>
independently integers from 0 to 4.  In another embodiment, n1, n2 and n3, are r elected with the proviso that n1 + n2 + n310. In another embodiment n1, n2 and n3 are 1 to 3. In another embodiment n1 and n2 are 0. In another embodiment n3 is 2. In one embodiment, z is 1.<br>
Â»    In another embodiment M is<br><br>
-HN	CH-	-C-	-0	  â 		HN â	âCHâC â	-0			HN		CHâ	câ	â0<br>
CH2				I CHâ0â	â		CH, I	<br>
0	C			CH30			ri		<br>
HN	CH â	-câ	-0	   .		HN-	IIâCHâC-	âo			T	<br>
CH,				CH2			i0		<br>
I				s						<br><br><br>
ii<br>
i!<br>
-HN	CH â C	O-<br>
;CH2)3<br>
NH<br>
I C=NH<br><br>
-HN	CHâC	0-<br>
CH2<br>
C=0<br>
O<br><br>
-HN	CHâC	0-<br>
I (CH2)4<br>
NH<br><br>
(CH2&gt;2 0=0<br>
NH	<br>
straight or brached chain alkyl, straight or branched chain alkenyl, straight or branched chain alkynyl, aryl, heteroaryl, cycloalkyl, heterocyclyl, straight or branched chain aralkyl, straight oi branched chain aralkenyl, straight or<br><br>
-64-<br>
branched chain aralkynyl, straight or branched chain heteroaralkyl, straight or branched chain heteroaralkenyl, straight or branched chain heteroaralkynyl,<br>
straight or branched chain cycloalkylalkyl, straight or branched chain cycloalkylalkenyl, straight or branched chain cycloalkylalkynyl, straight or branched chain heterocyclylalkyl, straight or branched chain heteYocyclylalkenyl or straight or branched chain heterocyclylalkynyl.<br>
(b)      S1andS2<br>
Optionally, a spacer region S1 and/or S2 can be present on either or<br>
both sides of the central moiety M (linked to Z) of the compounds, for example, to reduce steric hindrance in reactions with the surface of large biomolecules and/or for facilitating sorting. These can be any groups that provide for spacing, typically without altering desired functional properties of the capture compounds and/or capture compound/biomolecule complexes. Those of skill in the art in light of the disclosure herein, can readily select suitable spacers. Exemplary spacers are set forth below.<br>
For embodiments, for example, where the biomolecule and the sorting function possess low steric hinderance, a spacer is optional. In certain embodiments, steric hindrance also can enhance selectivity in conjunction with Y (or in the absence of a Y). This enhanced selectivity can be achieved either by the presence of a selectivity function, Y, that is attached to M or by the selection of the appropriate spacer molecules for S1 and/or S2. In other embodiments, the spacer group is selected such that the selectivity fuction (e.g. a drug) reaches the binding pocket of a target or non-target protein. Spacer groups may be hydrophobic (e.g. PEGs or phosphodiesters) or hydrophilic; their length may be varied to achieve efficient sorting or selectivity or capture; they may be rigid (e.g. trans olefins). The spacer groups may be selected based on the properties (hydrophobic/hydrophilic, size, etc.) of the biomolecuiar mixture to be analyzed.<br>
If S2 is not required, the reactivity of the cleavable bond L can be influenced by one or more substituted functionalities, for example, R15 on M. Electronic (e.g., mesomeric, inductive) and/or steric effects can be used to<br><br>
-65-<br>
modulate the stability of the cleavable bond L. For example, if M is a trityl derivative, the linkage to the biomolecule, including, but not limited to, a protein, is in one embodiment a trityl ether bond. The sensitivity of this bond to mild acids, such as acetic acid or the vapor of trifluoroacetic acid, can be significantly enhanced by having as R15 one or two electron donating groups, including, but not limited to, alkoxy groups, such as methoxy groups, in the para positions of the aryl rings. Alternatively, the trityl ether bond can be stabilized by the introduction of electron withdrawing groups, including, but not limited to. either halogen, including bromo and chloro, groups, nitro groups or ester moieties, in the para and/or ortho positions of the aromatic rings.<br>
In certain embodiments, S1 and S2 are each independently selected from (CH2)r. (CH20), (CH2CH20)r,(NH(CH2)rC(=0))s, (NHCH(R52)C(=0))Sl<br>
(O(CH)rC(=0))s.<br><br>
(R,5&gt;,<br>
V<br>
(R'5)/<br><br>
-r&gt;ân-<br>
R1S/A=/	\=/<br>
V(R5)y    /=y^<br>
(R15),<br>
where R15 is selected as above; r and s are each independently an integer from 1 to 10; R52 is the side chain of a natural a-amino acid; and y is an integer from 0 to 4. In one embodiment, y is 0 or 1.<br>
In certain embodiments, R15 is H, OH, OR51, SH, SR51, NH2, NHR51,<br><br>
-66-<br>
NR512, F. CI, Br, I. S03H, P024, CH3, CH2CH3, CH(CH3)2 or C(CH3)3; where R51 is straight or branched chain alkyl, straight or branched chain alkenyl, straight or branched chain alkynyl, aryl, heteroaryl, cycloalkyl, heterocyclyl, straight or branched chain aralkyl, straight or branched chain aralkenyl, straight or branched chain aralkynyl, straight or branched chain heteroaralkyl, straight or branched chain heteroaralkenyl, straight or branched chain heteroaralkynyl, straight or branched chain cycloalkylalkyl, straight or   -branched chain cycloalkyialkenyl, straight or branched chain cycloalkylalkynyl, straight or branched chain heterocyclylalkyl, straight or branched chain heterccyclylalkenyl or straight or branched chain heterocyclylalkynyl.<br>
(c)      L <br>
In certain embodiments, the cleavable group L is cleaved either prior to or during analysis of the biomolecule, such as a protein . The analysis can include mass spectral analysis, for example MALDI-TOF mass spectral analysis   The cleavable group L is selected so that the group is stable during conjugation to a biomolecule, and sorting, such as hybridization of a single stranded oligonucleotide Q moiety to a complementary sequence, and washing of the hybrid; but is susceptable to cleavage under conditions of analysis of the biomolecule, including, but not limited to, mass spectral analysis, for example MALDI-TOF analysis. In certain embodiments, the. cleavable group L can be a disulfide moiety, created by reaction of the<br>
compounds where X = SH, with the thiol side chain of cysteine residues on the surface of biomolecules, including, but not limited to, proteins. The resulting disulfide bond can be cleaved under various reducing conditions including, but not limited to, treatment with dithiothreitol and 2-mercaptoethanol.<br>
In another embodiment, L is a photocleavable group, which can be cleaved by a short treatment with UV light of the appropriate wave length either prior to or during mass spectrometry. Photocleavable groups, including those bonds that can be cleaved during MALDI-TOF mass spectrometry by<br>
the action of a laser beam, can be used. For example, a trityl ether or an<br><br>
-67-<br>
ortho nitro substituted aralkyl, including benzyl, group are susceptible to laser induced bond cleavage during MALDI-TOF mass spectrometry. Other useful photocleavable groups include, but are not limited to, o-nitrobenzyl, phenacyl, and nitrophenylsulfenyl groups.<br>
Other photocleavable groups for use herein include those disclosed in International Patent Application Publication No. WO 98/20166. In one embodiment, the photocleavable groups have formula I:<br><br>
where R   is oOalkylene; R   is selected from hydrogen, alkyl, aryl, alkoxycarbonyl, aryloxycarbonyl and carboxy; t is 0-3; and R50 is alkyl, alkoxy,<br>
20<br>
aryl or aryloxy. In one embodiment, Q is attached to R   through (S1)tM(R15)a(S2)b; and the biomolecule of interest is captured onto the R2 CHO moiety via a reactive derivative of the oxygen (e.g., X).<br><br>
where R20 is (0Oalkylene or alkylene; R21 is selected from hydrogen, alkyl,<br>
In another embodiment, the photocleavable groups have formula II:<br><br>
-68-<br>
,20<br>
aryl, alkoxycarbonyl, aryloxycarbonyl and carboxy; and X   is hydrogen, alkyl or OR21. In one embodiment, Q is attached to R20 through (S1)tM(R15)a(S2)b; and the biomolecule of interest is captured onto the R21CHO moiety via a reactive derivative of the oxygen (e.g., X).<br>
In further embodiments, R20 is 0(CH2)3 or methylene; R21 is selected<br>
from hydrogen, methyl and carboxy; and X20 is hydrogen, methyl or OR21. In<br>
21	20<br>
another embodiment, R   is methyl; and X   is hydrogen. In certain embodiments, R20 is methylene; R21 is methyl; and X20 is 3-(4,4'-dimethoxytrityloxy)propoxy.<br>
In another embodiment, the photocleavable groups have formula III:<br><br><br>
(R70<br>
(HI)<br><br>
where R is selected from (1)OalkyleneO and (1)Oalkylene, and is unsubstituted or substituted on the alkylene chain with one or more alkyl groups; c and e are each independently 0-4; and R70 and R71 are each independently alkyl, alkoxy, aryl or aryloxy. In certain embodiments, R2 is (1)Oalkylene, and is substituted on the alkylene chain with a methyl group! In one embodiment, Q is attached to R2 through (S1),M(R15)a(S2)b; and the biomolecule of interest is captured onto the A^CHO moiety via a reacti/e derivative of the oxygen (e.g., X).<br>
In further embodiments, R2 is selected from 30(CH2)30, 40(CH2)4, 30(CH2)3, 2OCH2CH2, OCH2,<br><br>
-69-<br><br><br>
In other embodiments, c and e are 0.<br>
Other cleavable groups L include acid sensitive groups, where bond cleavage is promoted by formation of a cation upon exposure to mild to strong acids. For these acid-labile groups, cleavage of the group L can be effected either prior to or during analysis, including mass spectrometric analysis, by the acidity of the matrix molecules, or by applying a short treatment of the array with an acid, such as the vapor of trifluoroacetic acid. Exposure of a trityl group to acetic or trifluoroacetic acid produces cleavage of the ether bond either before or during MALDI-TOF mass spectrometry.<br>
The capture compound-biomolecule array can be treated by either chemical, including, but not limited to, cyanogen bromide, or enzymatic, including, but not limited to. in embodiments where the biomolecule is a protein, trypsin, chymotrypsin, an exopeptidase (e.g., aminopeptidase and carboxypeptidase) reagents to effect cleavage. For the latter, all but one peptide fragment will remain hybridized when digestion is quantitative. Partial digestion also can be of advantage to identify and characterize proteins following desorption from the array. The cleaved protein/peptide fragments are desorbed, analyzed, and characterized by their respective molecular weights.<br>
In certain embodiments herein. L is selected from SS, OP(=0)(OR51)NH, OC(=o),<br><br><br><br>
-70-<br><br><br><br><br><br><br><br><br><br><br><br><br><br>
where R1 , R5 and y are as defined above. In certain embodiments, R15 is H, OH, OR51, SH. SR51. NH2, NHR51, N(R51)2, F, CI, Br, I. SO3H, P024, CH3, CH2CH3, CH(CH3)2 or C(CH3)3; where R51 is straight or branched chain alkyl, straight or branched chain alkenyl, straight or branched chain alkynyl, aryl, heteroaryl, cycloalkyi, heterocyclyl, straight or branched chain aralkyl, straight or branched chain aralkenyl, straight or branched chain aralkynyl, straight or branched chain heteroaralkyl, straight or branched chain heteroaralkenyl, straight or branched chain heteroaralkynyl, straight or branched chain cycloalkylalkyl. straight or branched chain cycloalkylalkenyl, straight or branched chain cycloalkylalkynyl, straight or branched chain heterocyclylalkyi, straight or branched chain heterocyclylalkenyl or straight or branched chain<br><br>
-71-<br>
heterocyclylalkynyl.<br>
(ii)     Non-cleavable divalent Z moieties<br>
In another embodiment, Z is a non-cleavable divalent moiety and has the formula: (S1),M(R15)a(S2)b, where S1, M, R15. S2. t, a and b are as defined above, b.       Z has a dendrimeric structure<br>
where M is as defined above, and is linked to a plurality of Q, Y, W and X<br>
moieties.<br>
In another embodiment, Z has a dendritic structure {i.e., Z is a multivalent dendrimer) that is linked to a plurality of Q and X moieties. Z, in certain embodiments, has about 4 up to about 6, about 8, about 10, about 20, about 40. about GO or more points of attachment {i.e., Z is tetravalent up to hexavalent. octavalent. decavalent. didecavalent, tetradecavalent, hexadecavalent etc.). In these embodiments, the dendritic moiety Z is based on a multivalent core M. as defined above. The number of points of attachment on M may vary from about 2 up to about 4, about 6, about 8, or more. Thus, in one embodiment, Z has the structure:<br><br><br>
-72-<br>
ln another embodiment, Z has the structure:<br><br>
where M is as defined above, and is linked to a plurality of Q, Y, W and X<br>
moieties.<br>
In other embodiments, the dendritic Z moieties may optionally possess a plurality of spacer groups S1 and/or S2, or for embodiments where Z is a cleavable linkage, a plurality of L groups. The S1, S2 and/or L moieties are attached to the end of the dendritic chain(s).<br>
In these embodiments, the density of the biopolymer to be analyzed, and thus signal intensity of the subsequent analysis, is increased relative to embodiments where Z is a divalent group.<br>
c.       Z is an insoluble support or a substrate<br>
In other embodiments, Z can be an insoluble support or a substrate, such as a particulate solid support, such as a silicon or other "bead" or microsphere, or solid surface so that the surface presents the functional groups (X, Y, Q and, as needed W). In these embodiments, Z has bound to<br><br>
-73-<br>
it one or a plurality of X moieties (typically, 1 to 100, generally 1 to 10) and optionally to at least one Q and/or Y moiety, and also optionally to one or more W moieties. Z, in these embodiments, can have tens up to hundreds, thousands, millions, or more functional moieties (groups) on its surface. FO. example, the capture compound can be a silicon particule or a agarose or other paricle with groups presented on it. As discussed below, it further can be coated with a hydrophobic material, such as lipid bilayers or other lipids that are used, for example to produce liposomes. In such embodiments, the resulting particles with a hydrophobic surface and optional hydrophobic W groups are used in methods for probing cell membrane environments and other intracellular environments.   Gentle lysis of cells, can expose the intracellular compartments and organelles, and hydrophobic capture compounds such as these, can be reacted with them, and the bound<br>
biomolecules assessed by, for example, mass spectrometry or further treated to release the contents of the compartments and organelles and reacted with the capture compounds or other capture compounds.<br>
In embodiments in which Z is an insoluble support, the insoluble support or substrate moiety Z can be based on a flat surface constructed, for example, of glass, silicon, metal, plastic or a composite or other suitable surface: or can be in the form of a "bead" or particle, such as a silica gel, a controlled pore glass, a magnetic or cellulose bead; or can be a pin, including an array of pins suitable for combinatorial synthesis or analysis. Substrates can be fabricated from virtually any insoluble or solid material. For example, silica gel, glass (e.g., controlled-pore glass (CPG)), nylon, Wang resin, Merrifield resin, dextran crossâlinked with epichlorohydrin (e.g., Sephadex ), agarose (e.g., SepharoseÂ®), cellulose, magnetic beads, Dynabeads, a metal surface (e.g., steel, gold, silver aluminum, silicon and copper), a plastic material (e.g., polyethylene, polypropylene, polyamide, polyester, polyvinylidenedifluoride (PVDF)) Exemplary substrate include, but are not limited to, beads (e.g., silica gel, controlled pore glass, magnetic dextran crossâlinked with epichlorohydrin (e.g., SephadexÂ®), agarose (e.g.,<br><br>
-74-<br>
SepharoseÂ®), cellulose, capillaries, flat supports such as glass fiber filters,<br>
glass surfaces, metal surfaces (steel, gold, silver, aluminum, copper and silicon), plastic materials including multiwell plates or membranes (e.g., of pc!^ ethylene, polypropylene, polyamide, polyvinylidenedifluoride), pins (e.g., arrays of pins suitable for combinatorial synthesis or analysis or beads in pits of flat" surfaces such as wafers (e.g., silicon wafers) with or without plates. Thesolid support is in any desired form, including, but not limited to, a bead, capillary, plate, membrane, wafer, comb, pin, a wafer with pits, an array of pits or nanoliter wells and other geometries and forms known to those of skill in the art. Supports include flat surfaces designed to receive or link samples at discrete loci.<br>
In one embodiment, the solid supports or substrates Z are "beads" {i.e., particles, typically in the range of less than 200 urn or less than 50 urn in their largest dimension) including, but not limited to, polymeric, magnetic, colored, Rf-tagged, and other such beads. The beads can be made from hydrophobic materials, including, but not limited to, polystyrene, polyethylene, polypropylene or teflon, or hydrophilic materials, including, but not limited to, cellulose, dextran crossâlinked with epichlorohydrin (e.g., SephadexÂ®), agarose (e.g., SepharoseÂ®), polyacrylamide, silica gel and controlled pore glass beads or particles. These types of capture compounds can be reacted in liquid phase in suspension, and the spun down or other removed from the reaction medium, and the resulting complexes analyzed, such as by mass spectrometry. They can be sorted using the Q function to bind to distinct loci on a solid support, or they can include a label to permit addressing, such as an radio frequency tag or a colored label or bar code or other symbology imprinted thereon.   These can be sorted according to the label, which serves as "Q" function, and then analyzed by mass spectrometry.<br>
In further embodiments, the insoluble support or substrate Z moieties optionally can possess spacer groups S1 and/or S2, or for embodiments where Z is a cleavable linkage, L. The S1, S2 and/or L moieties are attached to the surface of the insoluble support or substrate.<br><br>
-75-<br>
ln these embodiments, the density of the biomolecule to be analyzed, and thus signal intensity of the subsequent analysis, is increased relative to embodiments where Z is a divalent group. In certain embodiments, an appropriate array of single stranded oligonucleotides or oligonucleotide analogs that are complementary to the single stranded oligonucleotide or oligonucleotide analog sorting functions Q will be employed in the methods provided herein.<br>
d.       Mass modified Z moieties<br>
In other embodiments, including embodiments where Z is a cleavable moiety, Z includes a mass modifying tag. In certain embodiments, the mass modifying tag is attached to the cleavable linker L. In one embodiment, the mass modified Z moiety has the formula.<br>
(S1),M(R15)a(S2)bLT, where S1, t, M, R15, a, S2, b and L are selected as above; and T is a mass modifying tag. Mass modifying tags for use herein include, but are not limited to, groups of formula X1R10, where X1 is a divalent group such as O, OC(0)(CH2)yC(0)0, NHC(O), C(0)NH, NHC(0)(CH2)yC(0)0, NHC(S)NH, OP(0-alkyl)0, OS020, OC(0)CH2S, S, NH and<br><br><br><br><br>
and R10 is a divalent group including (CH2CH20)ZCH2CH20, (CH2CH20)2CH2CH20alkylene, alkylene, alkenylene, alkynylene, arylene, heteroarylene, (CH2)zCH20, (CH2)zCH2Oalkylene, (CH2CH2NH)ZCH2CH2NH, CH2CH(OH)CH20, Si(R12)(R13), CHF and CF2; where y is an integer from 1 to 20; z is an integer from 0 to 200; R11 is the side chain of an a-amino acid; and R12 and R13 are each independently selected from alkyl, aryl and aralkyl.<br><br>
-76-<br>
ln other embodiments, X1R10 is selected from SS, S, (NH(CH2)yNHC(0)(CH2)yC(0))2NH(CH2)yNHC(0)(CH2)yC(0)0, (NH(CH2)yC(0))zNH(CH2)yC(0)0, (NHCH(R11)C(0))zNHCH(R11)C(0)0. and (0(CH2)yC(0))zNH(CH2)yC(0)0.<br>
In the above embodiments, where R10 is an oligo-/polyethylene glycol derivative, the mass-modifying increment is 44, i.e., five different mass-modified species can be generated by changing z from 0 to 4, thus adding mass units of 45 (z = 0), 89 (z = 1), 133 (z = 2), 177 (z = 3) and 221 (z = 4) to"' the compounds. The oligo/polyethylene glycols also can be monoalkylated by a lower alkyl such as methyl, ethyl, propyl, isopropyl, t-butyl and the like.<br>
Other mass modifying tags include, but are not limited to CHF, CF2,<br>
Si(CH3)2, Si(CH3)(C2H5) and Si(C2H5)2. In other embodiments, the mass<br>
modifying tags include homo- or heteropeptides. A non-limiting example that<br>
generates mass-modified species with a mass increment of 57 is an<br>
oligoglycine. which produce mass modifications of, e.g., 74 (y = 1, z = 0), 131<br>
(y = 1. z = 2), 188 (y = 1, z = 3) or 245 (y = 1, z = 4). Oligoamides also can be<br>
used, e.g., mass-modifications of 74 (y = 1, z = 0), 88 (y = 2, z = 0), 102 (y =<br>
3. z = 0), 116 (y = 4, z = 0), etc., are obtainable. Those skilled in the art will<br>
appreciate that there are numerous possibilities in addition to those<br>
exemplefied herein for introducing, in a predetermined manner, many<br>
different mass modifying tags to the compounds provided herein.-  	<br>
In other embodiments, R15 and/or S2 can be functionalized with X1R10H or X1R10alkyl, where X1 and R10 are defined as above, to serve as mass modifying tags.<br>
2.       Reactivity Functions "X"<br>
Reactivity functions ("X") confer the ability on the compounds the ability to bind either covalently or with a high affinity (greater than 109, generally greater than 1010 or 1011 liters/mole, typically greater than a monoclonal antibody, and typically stable to mass spectrometric analysis, such as MALDI-MS) to a biomolecule, particularly proteins, including functional groups thereon, which include post-translationally added groups.<br><br>
-77<br>
Generally the binding is covalent or is of such affinity that it is stable under conditions of analysis, such as mass spectral, including MALDI-TOF, analysis. Exemplary groups are set forth herein (see, e.g., Figure 16, and the discussion below). Further groups include groups that are inert toward reaction with a biomolecule, such as a protein, until activated. Such groups include photoactivatable groups, including but not limited to, azide and diazirine groups. In another embodiment, an active ester (e.g. NHS) is used as the reactivity group under acidic conditions. The active ester is inert toward reaction with amine groups under these conditions, but will react upon raising the pH.<br>
In the compounds provided herein, X is a moiety that binds to or interacts with the surface of a biomolecule, including, but not limited to, the surface of a protein; an amino acid side chain of a protein; or an active site of an enzyme (protein) or to functional groups of other biomolecule, including lipids and polysaccharides.<br>
Thus, for example, X is a group that reacts or interacts with functionalities on the surface of a protein to form covalent or non-covalent bonds with high affinity. A wide selection of different functional groups are available for X to interact with a protein. For example, X can act either as a nucleophile or an electrophile to form covalent bonds upon reaction with the amino acid residues on the surface of a protein. Exemplary reagents that bind covalently to amino acid side chains include, but are not limited to, protecting groups for hydroxyl, ca-boxyl, amino, amide, and thiol moieties, including, for example, those disclosed in T.W. Greene and P.G.M. Wuts, "Protective Groups in Organic Synthesis," 3rd ed. (1999, Wiley Interscience);<br>
photoreactive groups, Diels Alder couples (i.e., a diene on one side and a sngle double bond on the other side).<br>
Hydroxyl protecting groups for use as X groups herein include, but are not limited to:<br>
(i) ethers such as methyl, substituted methyl (methoxymethyl, methylthiomethyl, (phenyldimethylsilyl)methoxymethyl, benzyloxymethyl, p-<br><br>
				78<br>
methoxybenzyloxymethyl, p-nitrobenzyloxymethyl, o-nitrobenzyloxymethyl, (4-methoxyphenoxy)methyl, guaiacolmethyl, f-butoxymethyl, 4-pentenyloxymethyl, siloxymethyl, 2-methoxyethoxymethyl, 2,2,2,-trichloroethoxymethyl, bis(2-chloroethoxymethyl), 2-(trimethylsilyl)ethoxymethyl, menthoxymethyl, tetrahydropyranyl, 3-bromotetrahydropyranyl, tetrahydrothiopyranyl, 1-methoxycyclohexyl, 4-methoxytetrahydropyranyl, 4-methoxytetrahydrothiopyranyl, 4-methoxytetrahydrothiopyranyl S,S-dioxide, 1-[(2-chloro-4-methyl)phenyl]-4-methoxypiperidin-4-yl, 1 -(2-fluorophenyl)-4-methoxypiperidin-4-yl, 1,4-dioxan-2-yl, tetrahydrofuranyl, tetrahydrothiofuranyl, 2,3,3a,4,5,6,7,7a-octahydro-7,8,8-trimethyl-4,7-methanobenzofuran-2-yl), substituted ethyl (1-ethoxyethyl, 1 -(2-chloroethoxy)ethyl, 1 -[2-(trimethylsilyl)ethoxy]ethyl, 1 -methyl-1 -methoxyethyl, 1 -methyl-1 -benzyloxyethyl, 1 -methyl-1 -benzyloxy-2-fluoroethyl, 1 -methyl-1 -phenoxyethyl, 2,2,2-trichloroethyl, 1,1-dianisyl-2,2,2-trichloroethyl. 1,1,1,3,3,3-hexafluoro-2-phenylisopropyl, 2-trimethylsilylethyl, 2-(benzylthio)ethyl, 2-(phenylselenyi)ethyl), f-butyl, allyl, propargyl, p-chlorophenyl, p-methoxyphenyl, p-nitrophenyl, 2,4-dinitrophenyl, 2,3,5,6-tetrafluoro-4-(trifluoromethyl)phenyl, benzyl, substituted benzyl (p-methoxybenzyl, 3,4,-dimethoxybenzyl, o-nitrobenzyl, p-nitrcbenzyl, p-halobenzyl, 2,6-dichlorobenzyl, p-phenylbenzyl, p-phenylenzyl, 2,6-difluorobenzyl, p-acylaminobenzyl, p-azidobenzyl, 4-azido-3-chlorobenzyl, 2-trifluoromethylbenzyl, p-(methylsulfinyl)benzyl), 2- and 4-picolyl, 3-methyl-2-picolyl A/-oxido, 2-quinolinylmethyl, 1-pyrenylmethyl, diphenylmethyl, p,p'-dinitrobenzhydryl, 5-dibenzosuberyl, triphenylmethyl, ci-naphthyldiphenylmethyl, p-methoxyphenyldiphenylmethyl, di(p-methoxyphenyl)phenylmethyl, tri(p-methoxyphenyl)methyl, 4-(4-'-bromophenacyloxy)phenyldiphenylmethyl, 4,4',4"-tris(4,5-dichlorophthalimidophenyl)methyl, 4,4',4"-tris(levulinoyloxyphenyl)methyl, 4,4',4"-tris(benzoyloxyphenyl)methyl, 4,4'-dimethoxy-3"-[A/-(imidazolylmethyl)]trityl, 4,4,-dimethoxy-3"-[A/-(imidazolylethyl)carbamoyl]trityl,<br>
1,1-bis(4-methoxyphenyl-1'-pyrenylmethyl, 4-(17-<br><br>
-79-<br>
tetrabenzo[a,c,g./]fluorenylmethyl)-4,4"-dimethoxytrityl, 9-Anthryl, 9-(9-<br>
phenyi)xanthenyl, 9-(9-phenyl-l0-oxo)anthryl, 1,3-benzodithiolan-2-yl,	<br>
benzisothiazolyl s,s-dioxido, silyl ethers (trimethylsilyl, triethylsilyl, triisopropylsilyl, dimethylisopropylsilyl, diethylisopropylsily!, dimethylthexylsilyl, f-butyldimethylsilyl, f-butyldiphenylsilyl, tribenzylsilyl, tri-p-xylylsilyl, triphenylsilyl, diphenylmethylsilyl, dk-butylmethylsilyl, tris(trimethylsilyl)silyl<br>
(sfsyl), (2-hydroxystyryl)dimethylsilyl, (2-hydroxystyryl)diisopropylsilyl, t-<br>
butylmethoxyphenylsilyl, f-butoxydiphenylsilyl):<br>
(ii) esters such as formate, benzoylformate, acetate, substituted acetate (chloroacetate, dichloroacetate, trichloroacetate, trifluoroacetate, methoxyacetate, triphenylmethoxyacetate, phenoxyacetate, p-chiorophenoxyacetate, phenylacetate, p-P-phenylacetate, diphenylacetate), nicotinate, 3-phenylpropionate, 4-pentenoate, 4-oxopentanoate (levulinate), 4,4-(ethylenedithio)pentanoate, 5-[3-bis(4-<br>
methoxyphenyl)hydroxymethylphenoxy]levulinate, pivaloate, 1 -adamantoate, crotonate, 4-methoxycrotonate, benzoate, p-phenylbenzoate, 2,4,6-trimethylbenzoate (mesitoate), carbonates (methyl, methoxymethyl, 9-fluorenylmethyl, ethyl, 2,2,2-trichloroethyl, 1,1,-dimethyl-2,2,2-trichloroethyl, 2-(trimethylsilyl)ethyl, 2-(phenylsulfonyl)ethyl, 2-(triphenylphosphonio)ethyl, isobutyl, vinyl, allyl, p-nitrophenyl, benzyl, p-methoxybenzyl, 3,4,-dimethoxybenzyl, o-nitrobenzyl, p-nitrobenzyl, 2-dansylethyl, 2-(4-nitrophenyl)ethyl, 2-(2,4-dinitrophenyl)ethyl, 2-cyano-1-phenylethyl, S-benzyl thiocarbonate, 4-ethoxy-1-naphthyl, methyl dithiocarbonate), 2-iodobenzoate, 4-azidobutyrate, 4-nitro-4-methylpentanoate, o-(dibromomethyl)benzoate, 2-formylbenzenesulfonate, 2-(methylthiomethoxy)ethyl carbonate, 4-(nethylthiomethoxy)butyrate, 2-(methylthiomethoxymethyl)benzoate, 2-(chloroacetoxymethyl)benzoate, 2-[(2-chloroacetoxy)ethyl]benzoate, 2-[2-(tenzyloxy)ethyl]benzoate, 2-[2-(4-methoxybenzyloxy)ethyl]benzoate, 2,6-dchloro-4-methylphenoxyacetate, 2,6-dichloro-4-(1,1,3,3-tetramethylbutyl)phenoxyacetate, 2,4-bis(1,1-dimethylpropyl)phenoxyacetate, chlorodiphenylacetate, isobutyrate, monosuccionoate, (E)-2-methyl-2-<br><br>
-80-<br>
butenoate (tigloate), o-(methoxycarbonyI)benzoate, p-P-benzoate, a-naphthoate, nitrate, a!kyl N1N1N1N1'-tetramethylphosphorodiamidate, 2-<br>
chlorobenzoate, 4-bromobenzoate, 4-nitrobenzoate, 3'5'-dimethoxybenzoin, a<br>
wild and woo'ly photolabile fluorescent ester, /V-phenylcarbamate, borate, dimethylphosphinothioyl, 2,4-dinitrophenylsulfenate; and<br>
-    (iii) sulfonates (sulfate, allylsulfonate, methanesulfonate (mesylate), bqnzylsulfonate, tosylate, 2-[(4-nitrophenyl)ethyl]suifonate).<br>
Carboxyl protecting groups for use as X groups herein include, but are not limited to:<br>
(i) esters such as enzymatically cleavable esters (heptyl, 2-N-(morpholino)ethyl, choline, (methoxyethoxy)ethyl, methoxyethyl), methyl, substituted methyl (9-fluorenylmethyl, methoxymethyl, methylthiomethyl, tetrahydropyranyl, tetrahydrofuranyl, methoxyethoxymethyl. 2-(trimethylsilyl)ethoxymethyl, benzyloxymethyl, pivaloyloxymethyl, phenylacetoxymethyl, triisopropylsilylmethyl, cyanomethylAA, acetol, phenacyl, p-bromophenacyl, a-methylphenacylA, p-methoxyphenacyl, desyl, carboxamidomethyl, p-azobenzenecarboxamidomethyl, /V-phthalimidomethyl), 2-substituted ethyl (2,2,2-trichloroethyl, 2-haloethyl, w-chloroalkyl, 2-(trimethylsilyl)ethyl, 2-methylthioethyl, 1,3-dithianyl-2-methyl, 2-(p-nitrophenylsulfenyl)ethyl, 2-(p-toluenesulfonyl)ethyl, 2-(2'-pyridyl)ethyl, 2-(p-methoxyphenyl)ethyl, 2-(diphenylphosphino)ethyl, 1-methyl-1-phenylethyl, 2-(4-acetyl-2-nitrophenyl)ethyl, 2-cyanoethyl), f-butyl, 3-methyl-3-pentyl, dicyclopropylmethyl, 2,4-dimethyl-3-pentyl, dicyclopropylmethyl, cyclopentyl, cyclohexyl, allyl, methallyl, 2-methylbut-3-en-2-yl, 3-methylbut-2-(prenyl), 3-buten-1-yl, 4-(trimethylsilyl)-2-buten-1-yl, cinnamyl, a-methylcinnamyl, prop-2-ynyl (propargyl), phenyl, 2,6-dialkylphenyl (2,6,-dimethylphenyl, 2,6,diisopropylphenyl, 2,6-di-f-butyl-4-methylphenyl, 2,6-dk-butyl-4-methoxyphenyl, p-(methylthio)phenyl, pentafluorophenyl, benzyl, substituted benzyl (triphenylmethyl, diphenylmethyl, bis(o-nitrophenyl)methyl, 9-anthrylmethyl, 2-(9,10-dioxo)anthrylmethyl, 5-dibenzosuberyl, 1-pyrsnylmethyl, 2-(trifluoromethyl)-6-chromonylmethyl, 2,4,6-trimethylbenzyl, p-<br><br>
-81<br>
bromobenzyl, o-nitrobenzyl, p-nitrobenzyl, p-methoxybenzyl, 2,6-dimethoxybenzyl, 4-(methylsulfinyl)benzyl, 4-sulfobenzyl, 4-azidomethoxybenzyl, 4-{A/-[1 -(4,4 -dimethyl-2,6-dioxocychlohexylidene)-3-methylbutyl]amino}benzyl, piperonyl, 4-picolyl, p-P-benzyl), silyl (trimethylsilyl, triethylsilyl, f-butyldimethylsilyl, /'-propyldimethylsilyl, phenyldimethylsilyl, di-f-butylmethylsilyl, triisopropylsilyl), activated (thiol), oxazoles, 2-alkyl-1,3-oxazoline: 4-alkyl-5-oxo-1,3-oxazolidine, 2,2,-bistrifluoromethyl-4-alkyl-5-oxo-1-,3-oxazolidine, 5-alkyl-4-oxo-1,3-dioxolane, dioxanones, ortho esters, Braun ortho ester, pentaaminocobalt(iii) complex, stannyl (triethylstannyl, tri-A/-butylstannyl);<br>
(ii) amides (A/,A/-dimethyl, pyrrolidinyl, piperidinyl, 5,6-dihydrophenanthridinyl, o-nitroanilide, A/-7-nitroindolyl, A/-8-nitro-1,2,3,4-tetrahydroquinolyl, 2-(2-aminophenyl)acetaldehyde dimethyl acetal amide, p-P-benzenesulfonamide;<br>
(iii) hydrazides (W-phenyl, A/./V-diisopropyl); and<br>
(iv) tetraalkylammonium salts.<br>
Thiol protecting groups for use as X groups herein include^ but are not limited to:<br>
(i) thioethers (S-alkyl, S-benzyl, S-p-methoxybenzyl, S-o- orp-hydroxy-or acetoxybenzyl, S-p-nitrobenzyl, S-2,4,6-trimethylbenzyl, S-2,4,6-trimethoxybenzyl, S-4-picolyl, S-2-quinolinylmethyl, S-2-picolyl A/-oxido, S-9-anthrylmethyl, S-9-fluorenylmethyl, S-xanthenyl, S-ferrocenylmethyl); S-diphenylmethyl, substituted S-diphenylmethyl and S-triphenylmethyl (S-diphenylmethyl, S-bis(4-methoxyphenyl)methyl, S-5-dibenzosuberyl, S-triphenylmethyl, S-diphenyl-4-pyridylmethyl), S-phenyl, S-2,4-dinitrophenyl, S-f-butyl, S-1-adamantyl, substituted S-methyl including monothio, dithio and aminothioacetals (S-methoxymethyl, S-isobutoxymethyl, S-benzyloxymethyl, S-2-tetrahydropyranyl, S-benzylthiomethyl, S-phenylthiomefhyl, thiazolidine, S-acetamidomethyl, S-trimethylacetomidomethyl, S-benzamidomethyl, S-allyloxycarbonylaminomethyl, S-phenylacetamidomethyl, S-<br>
phthalimidomethyl, 5-acetyl-Â» 5-carboxyl-, and S-cyanomethyl), substituted s-<br><br>
-82-<br>
ethyl (S-(2-nitro-1-phenyl)ethyl, S-2-(2,4-dintrophenyl)ethyl, S-2-(4'-pyridyl)ethyl, S-2-cyanoethyl, S-2-(trimethylsilyl)ethyl, S-(1-/n-nitrophenyl-2-<br>
benzoyl)ethyl, S-2-phenyJsulfonylethyl, S-1-(4-methylphenylsulfonyl)-2-methylprop-2-yl, silyl;<br>
(ii) thioesters (S-acetyl, S-benzoyl, S-trifluoroacetyl, S-N-[[(p-bipKenylyl)isopropoxy]carbonyl]-A/-methyl-a-aminothiobutyrate, S-N-{t-butoxycarbonyl-/V-methyl-a-aminothiobutyrate), thiocarbonates (S-2,2,2-trichloroethoxycarbonyl, S-f-butoxycarbonyl, S-benzyloxycarbonyl, S-p-methoxybenzyloxycarbonyl), thiocarbamates (S-(/V-ethyl), S-{N-methoxymethyl));<br>
(iii) unsymmetrical disulfides (S-ethyl, S-f-butyl, substituted S-phenyl disulfides);<br>
(iv) sulfenyl derivatives (S-sulfonate, S-sulfenylthiocarbonate, S-3-nitro-2-pyridinesulfenyl sulfide, S-[tricarbonyl[1,2,3,4,5-]-2-,4-cyclohexadien-1-yl]-iron(1+), oxathiolone); and<br>
(v) S-methylsulfonium salt, S-benzyl- and S-4-methoxybenzylsulfonium salt, S-1-(4-phthalimidobutyl)sulfonium salt, S-(dimethylphosphinol)thioyl, S-(diphenylphosphino)thioyl.<br>
Amino protecting groups for use as X groups herein include, but are not limited to:<br>
(i)       carbamates (methyl, ethyl, 9-fluorenylmethyl, 9-(2-sulfo)fluorenylmethyl, 9-(2,7-dibromo)fluorenylmethyl, 17-tetrabenzo[a,c,g./]fluorenylmethyl, 2-Chloro-3-indenylmethyl, benz[r]inden-3-ylmethyl, 2,7-di-f-butyl-[9-(10,10-dioxo-10,10,10,10-tetrahydrothiox, 1,1-dioxobenzo[to]thiophene-2-ylmethyl, substituted ethyl (2,2,2-trichloroethyl, 2-trimethylsilylethyl, 2-phenylethyl, 1-('1-adamantyl)-1-methylethyl, 2-chloroethyl, 1,1-dimethyl-2-haloethyl, 1,1-dimethyl-2,2-dibromoethyl, 1,1-dimethyl-2,2,2-trichloroethyl, 1 -methyl-1 -(4-biphenylyl)ethyl, 1 -(3,5-di-f-butylphenyl)-1 -methylethyl, 2-(2'- and 4'-pyridyl)ethyl, 2,2-bis(4'-nitrophenyl)ethyl, N-(2-pivaloylamino)-1,1-dimethylethyl, 2-[(2-nitrophenyl)dithio]-1-phenylethyl, 2-<br>
(/V,/V-dicyclohexylcarboxam.ido)ethyl), /-butyl, 1-adamantyl, 2-adamantyl, vinyl,<br><br>
-83-<br>
allyl. 1-isopropylallyl, cinnamyl, 4-nitrocinnamyl, 3-(3'pyridyl)prop-2-enyl, 8-quinolyi. A/-hydroxypiperidinyl, alkyldithio, benzyl, p-methoxybenzyl, p-<br>
nitrobenzyl, p-bromobenzyl, p-chlorobenzyl, 2,4-dichlorobenzyl, 4-methylsulfinylbenzyl, 9-anthrylmethyl, diphenylmethyl, 2-methylthioethyl, 2-methylsulfonylethyl, 2-(p-toluenesulfonyl)ethyl, [2-(1,3-dithianyl)methyl, 4-methylthiophenyl, 2,4-dimethylthiophenyl, 2-phosphonioethyl, 1-methyl-1 -(triphenylphsophonio)ethyl, 1,1-dimethyl-2-cyanoethyl, 2-dansylethyl, 2-(4-nitrophenyl)ethyl, 4-phenylacetoxybenzyl, 4-azidobenzyl, 4-azidomethoxybenzyl, m-chloro-p-acyloxybenzyl, p-(dihydroxyboryl)benzyl, 5-benzisoxazolylmethyl, 2-(trifluoromethyl)-6-chromonylmethyl, m-nitrophenyl, 3.5-dimethoxybenzyl, 1-methyl-1-(3,5-dimethoxyphenyl)ethyl, a-methylnitropiperonyl, o-nitrobenzyl, 3,4-dimethoxy-6-nitrobenzyl, phenyl(o-nitrophenyl)methyl, 2-(2-nitrophenyl)ethyl, 6-nitroveratryl, 4-methoxyphenacyl, 3.5'-dimethoxybenzoin, ureas (phenothiazinyl-(IO)-carbonyl derivative, N'-p-toluenesulfonylaminocarbonyl, A/'-phenylaminothiocarbonyl), f-amyl, S-benzyl thiocarbamate, butynyl, p-cyanobenzyl, cyclobutyl, cyclohexyl, cyclopentyl, cyclopropylmethyl, p-decyloxybenzyl, diisopropylmethyl, 2,2-dimethoxycarbonylvinyl, (^(A/'-A/'-dimethylcarboxamidoJbenzyl, 1,1-dimethyl-3-(A/'A/'-dimethylcarboxamido)propyl, 1,1-dimethylpropynyl, di(2-pyridyl)methyl), 2-furanylmethyl, 2-lodoethyl, isobornyl, isobutyl, isonicotinyl, p-(p'-methoxyphenylazo)benzyl, 1-methylcyclobutyl, 1-methylcyclohexyl, 1-methyl-1 -cyclopropylmethyl, 1 -methyl-1 -(p-phenylazophenyl)ethyl, 1 -methyl-1 -phenylethyl, 1-methyl-1-(4'-pyridyl)ethyl, phenyl, p-(phenylazo)benzyl, 2,4,6-tri-f-butylphenyl, 4-(trimethylammonium)benzyl, 2,4,6-trimethylbenzyl);<br>
(ii)       amides (A/-formyl, /V-acetyl, A/-chloroacetyl, A/-tricholoroacetyl, A/-trifluoroacetyl, /V-phenylacetyl, /V-3-phenylpropionyl, A/-4-pentenoyl, N-picolinoyl, n-3-pyridylcarboxamido, A/-benzoylphenylalanyl derivative, N-benzoyl. A/-p-phenylbenzoyl, A/-o-nitrophenylacetyl, A/-o-nitrophenoxyacetyl, A/-3-(o-nitrophenyl)propionyl, A/-2-methyl-2-(o-nitrophenoxy)propionyl, N-3-methyl-3-nitrobutyryl, A/-o-nitrocinnamoyl, A/-o-nitrobenzoyl, A/-3-(4-f-butyl-2,6-dinitrophenyl-2,2-dimethylpropionyl, /V-o-(benzoyloxymethyl)benzoyl, N-(2-<br><br>
-84-<br>
acetoxymethyl)benzoyl, /v-2-[(f-butyldiphenylsiloxy)methyl]benzoyl, /V-3-(3',6'-dioxo^'^'.S'-trimethylcyclohexa-l'^'-dieneJ-S.S-dimethylpropionyl, N-o-hydroxy-frans-cinnamoyl, A/-2-methyl-2-(o-phenylazophenoxy)propionyl, A/-4-chlorobutyryl, A/-acetoacetyl, /\/-3-(p-hydroxyphenyl)propionyl, {N-dithiobenzyloxycarbonylamino)acetyl, A/-acetylmethionine derivative, 4,5-diphenyl-3-oxazolin-2-one), cyclic imides (A/-phthaloyl, /V-tetrachlorophthaloyl, /V-4-nitrophthaloyl, /V-dithiasuccinoyl, A/-2,3-diphenylmaleoyl, N-2,5-dimethylpyrrolyl, A/-2,5-bis(triisopropylsiloxy)pyrrolyl, AM,1,4,4-tetramethyldisilylazacyclopentane adduct, AM,1,3,3-tetramethyM ,3-disilaisoindolyl, 5-substituted 1,3-dimethyl-1,3,5-triazacyclohexan-2-one, 5-suostituted 1,3-dibenzyl-1,3,5-triazacyclohexan-2-one, 1-substituted 3,5-dinitro-4-pyridonyl, 1,3,5-dioxazinyl);<br>
(iii) /V-alkyl and A/-aryl amines (W-methyl, /V-f-butyl, A/-allyl, A/-[2-(trimethylsilyl)ethoxy]methyl, /V-3-acetoxypropyl, /V-cyanomethyl, A/-(1-isopropyl-4-nitro-2-oxo-3-pyrrolin-3-yl), /v-2,4-dimethoxybenzyl, A/-2-azanorbornenyl, A/-2,4-dinitrophenyl, quaternary ammonium salts, /V-benzyl, /V-4-methoxybenzyl, A/-2,4-dimethoxybenzyl, A/-2-hydroxybenzyl, N-diphenylmethyl, A/-bis(4-methoxyphenyl)methyl, A/-5-dibenzosuberyl, N-triphenylmethyl, A/-(4-methoxyphenyl)diphenylmethyl, A/-9-phenylfluorenyl, N-ferrocenylmethyl, A/-2-picolylamine Af-oxide);<br>
(iv) imines (A/-1,1-dimethylthiomethylene, A/-benzylidine, N-p-methoxybenzylidene, /v-diphenylmethylene, W-[(2-pyridyl)mesityl]methylene, A/-(A/',A/-dimethylaminomethylene), A/-(A/',A/'-dibenzylaminomethylene), N-(N'-f-butylaminomethylene), A/,A/'-isopropylidene, W-p-nitrobenzylidene, N-salicylidene, /v-5-chlorosalicylidene, A/-(5-chloro-2-hydroxyphenyl)phenylmethylene, A/-cyclohexylidene, AM-butylidene);<br>
(v) enamines (A/-(5,5-dimethyl-3-oxo-1-cyclohexenyl, A/-2,7-dichloro-9-fluorenylmethylene, n-2-(4,4-dimethyl-2,6-dioxocyclohexytidene)ethyl, N-4,4,4-trifluoro-3-oxo-1-buteryl, A/-1-isopropyl-4-nitro-2-oxo-3-pyrrolin-3-yl);<br>
(vi) /V-heteroatom derivatives (A/-borane derivatives, /V-diphenylborinic acid derivative, A/-diethylborinic acid derivative, W-difluoroborinic acid<br><br>
-85-<br>
derivative, M/V'-3,5-bis(trifluoromethyl)phenylboronic acid derivative, N-[phenyl(pentacarbonylchromium- or -tuhgsten)]carbenyl, A/-copper or /V-zinc chelate, 18-crown-6 derivative, A/-nitro, /V-nitroso, /V-oxide, triazene derivative, /V-Jiphenylphosphinyl, /V-dimethyl-and diphenylthiophosphinyl, A/-dialkyl phosphoryl, A/-dibenzyl and diphenyl phosphoryl, iminotriphenylphosphorane derivative, A/-benzenesulfenyl, A/-o-nitrobenzenesulfenyl, A/-2.4-dinitrobenzenesulfenyl, /V-pentachlorobenzenesulfenyl, A/-2-nitro-4-methoxybenzensulfenyl, A/-triphenylmethylsulfenyl, A/-1-(2,2,2-trifluoro-1,1-diphenyl)ethylsulfenyl, /V-3-nitro-2-pyridinesulfenyl,A/-p-toluenesulfonyl, N-benzenesulfonyl, A/-2,3-6-trimethyl-4-methoxybenzenesulfonyl, AZ-2,4,6-trimethoxybenzesulfonyl, /V-2,6-dimethyl-4-methoxybenzenesulfonyl, N-pentamethylbenzenelsulfonyl, A/-2,3,5,6-tetramethyl-4-methoxybenzenesulfonyl, Ay-4-methoxybenzenesulfonyl, /V-2,4,6-trimethylbenzenesulfonyl, A/-2,6-dimethoxy-4-methylbenzenesulfonyl, /V-3-methoxy-4-f-butylbenzenesulfonyl, /V-2,2,5,7,8-pentamethylchroman-6-sulfonyl, N-2- and 4-nitrobenzenesulfonyl, A/-2,4-dinitrobenzenesulfonyl, N-benzothiazole-2-sulfonyl, A/-pyridine-2-sulfonyl, A/-methanesulfonyl, A/-2-(trimethylsilyl)ethanesulfonyl, A/-9-anthracenesulfonyl, /V-4-(4',8'-dimethoxynaphthylmethyl)benzenesulfonyl, A/-benzylsulfonyl, N-trifluoromethylsulfonyl, A/-phenacylsulfonyl, A/-t-butylsulfonyl);<br>
(vii) imidazole protecting groups including A/-sulfonyl derivatives (A/,/V-dimethylsulfonyl, /V-mesitylenesulfonyl, /V-p-methoxyphenylsulfonyl, N-benzenesulfonyl, /V-p-toluenesulfonyl); carbamates (2,2,2-trichloroethyl, 2-(trimethylsilyl)ethyl, f-butyl, 2,4-dimethylpent-3-yl, cyclohexyl, 1,1-dimethyl-2,2,2-trichioroethyl, 1-adamantyl, 2-adamantyl); /V-alkyl and A/-aryl derivatives (/V-vinyl, /V-2-chloroethyl, /V-(1-ethoxy)ethyl, A/-2-(2'-pyridyl)ethyl, A/-2-(4'-pyridyl)ethyl, A7-2-(4'-nitrophenyl)ethyl), AMrialkyl silyl derivatives (N-t-butyldimethylsilyl, A'-triisopropylsilyl), A/-allyl, /V-benzyl, A/-p-methoxy benzyl, N-<br>
3,4-dimethoxybenz/l, /V-3-methoxybenzyl, A/-3,5-dimethoxybenzyl, A/-2-<br>
nitrobenzyl, /V-4-nitrobenzyl, A/-2,4-dinitrophenyl, A/-pyhenacyl, N-<br>
triphenylmethyl, /V-diphenylmethyl, A/-(diphenyl-4-pyridylmethyl), N-{n',n'-<br><br>
-86-<br>
dimethylamino)), amino acetal derivatives (A/-hydroxymethyl, A7-methoxymethyl, W-diethoxymethyi, AV-ethoxymethyl, A/-(2-chloroethoxy)methyl,<br>
A/-[2-(trimethylsilyl)ethoxy]methyl, /V-f-butoxymethyl, N-t-butyldimethylsiloxymethyl, V pivaloyloxymethyl, /V-benzyloxymethyl, N-dimethylaminomethyl, /V-2-tetrahydropyranyl), amides (carbon dioxide adduct, /V-formyl, A/-(n',n'-diethylureidyl), A/-dichloroacetyl, A/-pivaloyl, N-diphenylthiophosphinyl); and<br>
(viii) amide NH protecting groups including amides (A/-allyl, /V-f-butyl, /V-dicyclopropylmethyl, /V-methoxymethyl, A/-methylthiomethyl, N-benzyloxymethyl, /V-2,2,2-trichloroethoxymethyl, N-t-butyldimethylsiloxymethyl, A/-pivaloyloxymethyl, A/-cyanomethyl, N-pyrrolidinomethyl, A7-methoxy, AZ-benzyloxy, A/-methylthio, N-triphenylmethylthio, A/-^-butyldimethylsilyl, A/-triisopropylsilyl, N-4-methoxyphenyl, A/-3,4-dimethoxyphenyl, Ay-4-(methoxymethoxy)phenyl, W-2-methoxy-1-naphthyl, N-benzy\, A/-4-methoxybenzyl, Ay-2,4-dimethoxybenzyl, /V-3,4-dimethoxybenzyl, /V-o-nitrobenzyl, /V-bis(4-methoxyphenyl)methyl, N-bis(4-methoxyphenyl)phenylmethyl, A7-bis(4-methylsulfinylphenyl)methyl, A7-triphenylmethyl, /V-9-phenylfluorenyl, A/-bis(trimethylsilyl)methyl, N-t-butoxycarbonyl, A/-benzyloxycarbonyl, A/-methoxycarbonyl, N-ethoxycarbonyl,/V-p-toluenesulfonyl, A7,0-isopropylidene ketal, N,0-<br>
benzylidene acetal, /V.O-formylidene acetal, A/-butenyl, A/-ethenyl, W-[(e)-(2-methoxycarbonyl)vinyl], A/-diethoxymethyl, A/-(1 -methoxy-2.2-dimethylpropyl), A/-2-(4-methylphenylsulfonyl)ethyl).<br>
These protecting groups react with amino acid side chains such as hydroxyl (serine, threonine, tyrosine); amino (lysine, arginine, histadine, proline); amide (glutamine, asparagine); carboxylic acid (aspartic acid, glutamic acid); a id sulfur derivatives (cysteine, methionine), and are readily adaptable for use in the capture compounds as the reactive moiety X.<br>
It is in adcition to the wide range of group-specific reagents that are known to persons of skill in the art, reagents that are known in natural product chemistry also can serve as a basis for X in forming covalent linkages.<br><br>
-87-<br>
Other choices for, X include protein purification dyes, such as acridine or methylene blue, which have a strong affinity for certain proteins.<br>
Alternatively, X can act as an electron donor or an electron acceptor to form non-covalent bonds or a complex, such as a charge-transfer complex, with a biomolecule, including, but not limited to, a protein, such that the resulting bond has a high stability (i.e., stable under conditions of mass spectrometric analysis, such as MALDI-TOF, as defined above). These reagents include those that interact strongly and with high specificity with biomolecules, including, but not limited to, proteins, without forming covalent bonds through the interaction of complementary affinity surfaces. For example, well known binding pairs, such as biotin and streptavidin, antibody and antigen, receptor and ligand, lectin and carbohydrate or other similar<br>
types of reagents are readily adaptable for use in these compounds as the<br>
reactive moiety X that will react with high affinity to biomolecules with surfaces similar to or identical to the other member of the binding pair. These moieties are selected so that the resulting conjugates (also referred to herein as complexes) have strong interactions that are sufficiently stable enough for suitable washing of the unbound biomolecules, including, but not limited to, proteins, out of the complexed biological mixtures.<br>
The reactivity of X can be influenced by one or more selectivity    , functions Y on the core, i.e., M in the formula above, particularly where S2 is not present.<br>
The Y function, discussed below is employed for electronic (e.g., mesomeric, inductive) and/or steric effects to modulate the reactivity of X and the stability of the resulting X-biomolecule linkage. In these embodiments, biomolecule mixtures, including, but not limited to, protein mixtures, can react and be analyzed due to the modulation by Y, which changes the electronic or steric properties of X and, therefore, increases the selectivity of the reaction of X with the biomolecule.<br>
In certain embodiments, X is an active ester, such as C(=o)OPhpNo2, C(=0)OC6F5 or C(=0)0(Nsuccinimidyl), an active halo moiety, such as an <br><br>
-88-<br>
halo ether or an a-halo carbonyl group, including, but not limited to, OCH2I, OCH2Br, OCH2CI, C(O)CH2l, C(O)CH2Brand C(O)CH2CI; amino acid side<br>
chain-specific functional groups, such as maleimido (for cysteine), a metal complex, including gold or mercury complexes (for cysteine or methionine),<br>
an expoxide or isothiocyanate (for arginine or lysine); reagents that bind to<br>
active sites of enzymes, including, but not limited to, transition state analogs; antibodies, e.g., against phosphorylated peptides; antigens, such as a phage display library; haptens; biotin; avidin; or streptavidin.<br>
In certain embodiments X is an N-hydroxysuccinimidyl ester, or is<br><br><br><br><br><br><br><br><br><br>
or<br>
In another embodiment, X is a photoactivatable group. In these embodiment, the capture compound contains a selectivity function and is allowed to interact with a biomolecular mixture until, for example, equilibrium is reached. The X group is then activated by exposure to the appropriate wavelength of light, whereby the X group then reacts with a surface group of the biomolecule to capture it. In one embodiment, the photoactivatable group is an arylazide, such as a phenylazide. Following exposure to light, the<br><br>
-89-<br>
resulting nitrene will react with, e.g., the side chain of tyrosine to capture the protein. In another embodiment, the photoactivatable group is a diazirine<br>
group, such as 3-trifluoromethyldiazirine.<br>
In other embodiment, the reactivity functionality X, is linked to the central core Z, via a spacer S. A spacer can be any group that provides for spacing, typically without altering desired functional properties of the capture compounds and /or capture compound/biomolecule complexes. The reactive functionality X linked with the spacer can be extended from the central core Z, to reach to the active sites on the surface of the biomolecule, such as proteins. Those of skill in the art in the light of the disclosure herein, can readily select suitable spacers.<br>
In certain embodiments, S is selected from (CH2)r. (CH2O), (CH2CH2O),.(NH(CH2)rC(=0))s, (0(CH)rC(=0))S)-((CH2)ri-C(0)NH-(CH2)r2)s-and-(C(0;NH-(CH2)r)s-, where r, r1, r2 and s are each independently and integer from 1 to 10.<br>
3.       Selectivity Functions "Y"<br>
The selectivity functions ("Y") serves to modulate the reactivity function by reducing the number of groups to which the reactivity functions bind, such as by steric hindrance and other interactions. It is a group that modifies the steric and/or electronic (e.g., mesomeric, inductive effects) properties as well as the resulting affinities of the capture compound. Selectivity functions include any functional groups that increase the selectivity of the reactivity group so that it binds to fewer different biomolecules than in the absence of the selectivity function or binds with greater affinity to biolmolecules than in its absence. In the capture compounds provided herein, Y is allowed to be extensively varied depending on the goal to be achieved regarding steric hindrance and electronic factors as they relate to modulating the reactivity of the cleavable bond L, if present, and the reactive functionality X. For example, a reactivity function X can be selected to bind to amine groups on proteins; the selectivity function can be selected to ensure that only groups exposed on the surface can be accessed. The selectivity function is such<br><br>
-90-<br>
that the compounds bind to or react with (via the reactivity function) fewer different biomolecules when it is part of the molecule than when it is absent and/or the compounds bind with greater specificity and higher affinity The selectivity function can be attached directly to a compound or can be attached via a linker, such as CH2CO2 or CH2-0-(CH2)n-0, where n is an integer from 1 to 12, or 1 to 6, or 2 to 4. See, e.g., Figure 17 and Figure 21 and the discussion below for exemplary selectivity functions. In certain embodiments, the linker is chosen such that the selectivity function can reach the binding pocket of a target or non-target protein. In another embodiment, the selectivity function is a chiral group, which allows for stereoselective capture of biomolecules.<br>
In certain embodiments, each Y is independently a group that modifies the affinity properties and/or steric and/or electronic (e.g., mesomeric, inductive effects) properties of the resulting capture compound. For example, Y, in certain embodiments, is selected from ATP analogs and inhibitors; peptides and peptide analogs; polyethyleneglycol (PEG); activated esters of amino acids, isolated or within a peptide; cytochrome C; and hydrophilic trityl groups.<br>
In another embodiment, Y is a small molecule moiety, a natural product, a protein agonist or antagonist, a peptide or an antibody (see, e.gf, Figure 17). In another embodiment, Y is a hydrophilic compound or protein (e.g.. PEG or trityl ether), a hydrophobic compound or protein (e.g., polar aromatics, lipids, glycolipids, phosphotriesters, oligosaccharides), a positive or negatively charged group, a small molecule, a pharmaceutical compound or a biomolecule that creates defined secondary or tertiary structures.<br>
In certain embodiments, Y is an enzyme inhibitor, an enzyme agonist or antagonist, a pharmaceutical drug or drug fragment, a prodrug or drug metabolite that modifies the selectivity of the capture compounds or collections thereof, to interact with the biomolecules or mixtures thereof, including, but not limited to specific receptors, to form covalent or non-covalent bonds with high affinity. In one embodiment, the capture<br><br>
-91-<br>
compounds/ collections thereof have a selectivity function, which is a cox-2 inhibitor, and a mixture of biomolecules contains cox receptors among other<br>
biomolecules.<br>
In certain embodiments the selectivity function is selected from pharmaceutical drugs or drug fragments set forth below, where attachment of exemplary pharmaceutical drugs to a cental core is shown below. In other embodiments, the selectivity function is a drug, drug fragment, drug metabolite, or a drug synthetic intermediate.<br>
The pharmaceutical drugs or drug fragments can be attached to the central core Z, in different orientations via different points of attachment, thereby modulating the selectivity of the capture compound. The attachment of a drug/drug fragment to the central core can be carried out by methods known to a person with skill in the art. Attachment of some exemplary pharmaceutical drugs at various points, to the central core Z is set forth below.<br>
In another embodiment, the capture compounds provided herein include those where the selectivity function is a drug, drug fragment, drug metabolite or a prodrug. In these embodiments, the capture compounds also contain a reactivity function, as defined elsewhere herein. In further embodiments, the capture compounds also contain a sorting function, as. defined elsewhere herein.<br>
In certain embodiments, the capture compounds that contain drug, drug fragment, drug metabolite or prodrug selectivity functions contain an amino acid core. In one embodiment, the amino acid core may be an amino acid that does not have a functionality on the side chain for attachment of a third function. Such amino acid cores include, but are not limited to, glycine, alanine, phenylalanine and leucine. In these embodiments, the capture compound contains a reactivity function and a selectivity function, which are attached to the amino and carboxy groups of the amino acid.<br>
In another embodiment, the amino acid core may be an amino acid<br>
that possesses a functionality on the side chain for attachment of a third<br><br>
-92-<br>
function. Such amino acid cores include, but are not limited to, serine,<br>
threonine, lysine, tyrosine and cysteine. In these embodiments, the capture<br>
compound contains a reactivity function, a sorting function and a selectivity function, which are attached to the amino, carboxy and side chain functional groups of the amino acid.<br>
In one embodiment, the core is tyrosine and the capture compounds<br>
have the formula:<br><br><br><br><br><br>
where "drug" refers to a drug, drug fragment, drug metabolite or prodrug.<br>
In one embodiment, the drug is LIPITORÂ® (atorvastatin calcium) and the capture compounds have the formulae:<br><br>
-93-<br><br><br>
.0â(CH2)n	sorting function<br>
III      NH ^^ ^"^   C02Reactivity funtion<br><br><br>
(CH2)n-<br>
Ph.NH  Ph<br>
(CH2)n<br>
H02C<br>
Iâ{~^~0â(CH2)n'C<br>
C02Reactivity function<br><br>
o"  \A<br>
P.h-NH   Pt?<br><br><br>
-0.<br>
/Sorting function<br>
NH<br>
C02Reactivity function<br>
^Sorting function<br>
"Â°TXX<br>
C02Reactivity function<br>
,Sorting function<br>
NH<br>
^<br><br>
HO?C<br>
Sorting function C02Reactivity function<br><br>
In other embodiments, the drug is CELEBREXÂ® (celecoxib) and the capture compounds have the formulae:<br><br>
-94-<br><br><br>
(CH2)n<br>
^      NH<br>
^0<br><br>
Sorting function<br>
C02Reactivity function<br><br><br><br>
wsA	n      ^	Sorting function<br>
H2N' *-<br><br>
i-V&gt;<br>
.. i<br><br><br>
,-o<br>
Oâ(CH2)n-u\^       NH<br>
^A<br>
^Sorting function<br>
C02Reactivity function<br>
C02Reactivity function<br>
U^A,<br>
\AYVCF3<br><x></x><br>
&gt;^<br>
A<br>
&amp;*<br><br>
CF,<br><br>
NH,<br><br><br><br><br><br><br>
P^<br>
H.N-V<br><br>
H<br><br>
-95-<br><br><br>
H0?VA	N3      H0'    '0     &gt;   "-""    O V<br>
	) Â°   NV  H<br>
	o H   :<br><br>
CF3<br>
V CF3	(H./S    â    6   S   Â°   ,     -_o<br>
BpT      ^rt06     Â°v<br><br>
HN	N]-^<br><br>
H	H<br>
HN    -    s	N3	HN  
r, â¢	n â¢	â  -	V	Â°.~NH2<br>
0 &gt;-
YÂ»x   -NH -:	H^   ,NH  : Hp(&gt; ^<br>
0	0	Â°	'      "<br>
â¢ 0	0	u     /  N<br>
HN    ',	N	
0	v     0 .
HN         . /	nw                                                          HN<br>
OH N -  J<br>
F3C -*N<br>
QSNH2<br>
In another embodiment, the drug is VIOXXÂ® (rofecoxib) and the capture compounds have the formulae:<br><br>
-96-<br><br><br><br><br><br><br>
(CH2)n<br>
NH<br>
-0<br><br>
^Sorting function<br>
^Reactivity function<br><br><br>
(CH2)n'Â°<br><br>
NH<br>
^Sorting function<br>
C02 Reactivity function<br><br><br>
,rH .n,CL /^	^Sorting function<br>
(OH2)n     Nf^&gt;,       NH<br>
l^sA<br>
C02Reactivity function<br><br>
^Sorting function<br>
(CH2)n-Â°Y^       NH<br>
0       TAA<br>
C02Reactivity function<br>
v^i, &gt;    O     ^	^Sorting function<br>
(CH2)"'Â°Y%       NH'<br>
"C02Reactivity function<br>
In another embodiment, the drug is BAYCOLÂ® (cerivastatin sodium) and the capture compounds have the formula:<br><br>
-97-<br><br><br><br><br>
OH 0<br>
.^(CH2)n	_ ^.     .    _<br>
O	â3     		Sorting function<br>
"T^I     NH<br>
CC^Reactivity funtion<br>
Sorting function<br>
C02Reactivity function<br>
Sorting function<br>
C02Reactivity function<br>
Sorting function<br>
C02Reactivity function<br>
Sorting function<br>
C02Reactivity function<br>
HOOC<br>
^Sorting function<br>
C02Reactivity function<br><br><br><br><br>
-98-<br>
ln another embodiment, the drug is methotrexate and the capture compounds have the formulae:<br>
NH2<br>
H2N-A;      V-N<br>
N=
 <br>
P	H3-VCH3<br>
HM	s	H      .	,-NH   0-
Â°     N H W       o     /	-NH      0      -
H<br>
0"  N&gt;-	.._rO	u	N3       Hq    O<br>
0H<br>
"N	O^-N'tiV<br>
HO	H	^	&gt;-,     NH<br>
HN â¢   s   _..   H       -i     o   w<br>
1     â    V	-N	r J   O     /-^      -O<br>
NH*       O	I	"NH   ""NH<br>
M	O     NÂ»V<br>
/    OH    / .J    r^ N-' H<br>
OH 0=-,<br>
â      O<br>
CH3	HN"     M_<br>
-N	^-0     OH<br>
H2N     NW     &gt;â,	,-,.<br>
N     &gt;-N     W	&gt;=-<br>
N3	/-N	&gt;-NH   OH	H3C-N       =N<br>
H     Â«.	^	HN	0'     M.	^-       &gt;=N<br><br>
Â°'!JÂ»V-  o"   T VNH	â¢-b<br>
H<br>
N3	HN<br>
0^ "ff*"'	H	C	)<br>
0H     /	HN-     S	H    5   *j<br>
N-	Â°     N HV       o     /       i	"NH<br>
H	H	O     N*--<br>
OH    ,<br>
--'   r<br>
H<br>
In other embodiments, Y is a group that is a component of a luminescent, including fluorescent, phosphorescent, chemiluminescent and bioluminescent system, or is a group that can be detected in a colorimetric assay; in certain embodiments, Y is a monovalent group selected from<br>
straight or branched chain alkyl, straight or branched chain alkenyl, straight or branched chain alkynyl, cycloalkyl, cycloalkenyl, cycloalkynyl, heterocyclyl, straight or branched chain heterocyclylalkyl, straight or branched chain heterocyclylalkenyl, straight or branched chain heterocyclylalkynyl, aryl, straight or branched chain arylalkyl, straight or branched chain arylalkenyl, straight or branched chain arylalkynyl, heteroaryl, straight or branched chain heteroarylalkyl, straight or branched chain heteroarylalkenyl, straight or branched chain heteroarylalkynyl, halo, straight or branched chain haloalkyi, pseudohalo, azido, cyano, nitro, OR60, NR60R61, COOR60, CfOJR60,<br><br>
-99-<br>
C(O)NR60R61, S(O)qR60, S(O)qOR5Â°, S(O)qNR60R61, NR60C(O)R61, <br>
NR60C(O)NR60R61, NR60S(O)qR60, SiR60R61R62, P(R60)2, P(O)(R60)2, P(OR60)2, P(O)(OR60)2, P(O)(OR60)(R61) and P(O)NR60R61, where q is an integer from 0 to 2;<br>
each R  , R61, and R62 is independently hydrogen, straight or branched chain alkyl, straight or branched chain alkenyl, straight or branched chain alkynyl, aryl, straight or branched chain aralkyl, straight or branched chain aralkenyl, straight or branched chain aralkynyl, heteroaryl, straight or branched chain heteroaralkyl, straight or branched chain heteroaralkenyl, straight or branched chain heteroaralkynyl, heterocyclyl, straight or branched chain heterocyclylalkyl, straight or branched chain heterocyclylalkenyl or straight or branched chain heteorcyclylalkynyl.<br>
Fluorescent, colorimetric and phosphorescent groups are known to those of skill in the art (see, e.g., U.S. Patent No. 6,274,337; Sapan et al.<br>
(1999) Biotechnol. Appl. Biochem. 29 (Pt. 2J:99-108; Sittampalam et al.<br>
(1997) Curr. Opin. Chem. Biol. 1 (3):384-91; Lakowicz, J. R., Principles of Fluorescence Spectroscopy, New York: Plenum Press (1983); Herman, B., Resonance Energy Transfer Microscopy, in: Fluorescence Microscopy of Living Cells in Culture, Part B, Methods in Cell Biology, vol. 30, ed. Taylor, D. L. &amp; Wang, Y. -L. San Diego: Academic Press (1989), pp. 219-243; Turro, N. J., Modern Molecular Photochemistry, Menlo Park: Benjamin/Cummings Publishing Col, Inc. (1978), pp. 296-361 and the Molecular Probes Catalog (1997), OR, USA). Fluorescent moieties include, but are not limited to, 1- and 2-aminonaphthalene, p,p'-diaminostilbenes, pyrenes, quaternary phenanthridine salts, 9-aminoacrtdines, p,p'-diaminobenzophenone imines, anthracenes, oxacarbocyanine, merocyanine, 3-aminoequilenin, perylene, bis-benzoxazolea, bis-p-oxazolyl benzene, 1,2-benzophenazin, retinol, bis-3-aminopyridinium salts, hellebrigenin, tetracycline, sterophenol, benzimidazolylphenylamine, 2-oxo-3-chromen, indole, xanthen, 7-hydroxycoumarin, phenoxazine, calicylate, strophanthidin, porphyrins, triarylmethanes and flavin. Fluorescent compounds that have functionalities<br><br>
-100-<br>
for linking to a compound provided herein, or that can be modified to incorporate such functionalities include, e.g., dansyl chloride; fluoresceins such as 3,6-dihydroxy-9-phenylxanthhydrol; rhodamineisothiocyanate; N-phenyl 1-an:no-8-sulfonatonaphthalene; N-phenyl 2-amino-6-sulfonatonaphthalene; 4-acetamido-4-isothiocyanato-stilbene-2,2'-disulfonic acid; pyrene-3-sulfonic acid; 2-toluidinonaphthalene-6-sulfonate; N-phenyl-N-<br>
methyl-2-aminoaphthalene-6-sulfonate; ethidium bromide; stebrine;<br>
auromine-0,2-(9'-anthroyl)palmitate; dansyl phosphatidylethanolamine; N,N'-dioctadecyl oxacarbocyanine; N,N'-dihexyl oxacarbocyanine; merocyanine, 4-(3'pyrenyl)stearate; d-3-aminodesoxy-equilenin; 12-(9'-anthroyl)stearate; 2-methylanthracene; 9-vinylanthracene; 2,2'(vinylene-p-<br>
phenylene)bisbenzoxazole; p-bis(2-(4-methyl-5-phenyl-oxazolyl))benzene;6-dimethylamino-1,2-benzophenazin; retinol; bis(3'-aminopyridinium) 1,10-decandiyl diiodide; sulfonaphthylhydrazone of hellibrienin; chlorotetracycline; N-(7-dimethylamino4-methyl-2-oxo-3-chromenyi)maleimide; N-(p-(2-benzimidazolyl)-phenyl)maleimide; N-(4-fluoranthyl)maleimide; bis(homovanillic acid); resazarin; 4-chloro-7-nitro-2,1,3-benzooxadiazole; merocyanine 540; resorufin; rose bengal; and 2,4-diphenyl-3(2H)-furanone. Many fluorescent tags are commercially available from SIGMA chemical company (Saint Louis, Mo.), Molecular Probes, R&amp;D systems (Minneapolis, Minn.), Pharmacia LKB Biotechnology. (Piscataway, N.J.), CLONTECH Laboratories, Inc. (Palo Alto, Calif), Chem Genes Corp., Aldrich Chemical Company (Milwaukee, Wis.), Glen Research, Inc., GIBCO BRL Life Technologies, Inc. (Gaithersberg, Md), Fluka Chemica-Biochemika Analytika (Fiuka Chemie AG, Buchs, Switzerland), and Applied Biosystems (Foster City, Calif.) as well as other commercial sources known to one of skill in the art.<br>
Chemiluminescent groups intended for use herein include any components of light generating systems that are catalyzed by a peroxidase and require superoxide anion (O) (and/or hydrogen peroxide (H202))(see, e.,7., Musiani etal. (1998) Histol. Histopathol. 13(1 ):243-6). Lightgenerating<br><br>
-101-<br>
systems include, but are not limited to, luminol, isoluminol, oeroxyoxalate-fluorophore, acridinium ester, lucigenin, dioxetanes, oxalate esters, acridan, hemin, indoxyl esters including 3-O-indoxyl esters, naphthalene derivatives, such as 7-dimethylamino-naphthalene-1,2-dicarbonic acid hydrazide and cypridina luciferin analogs, including 2-methyl-6-[p-methoxyphenyl]-3,7-dihyroimidazo[1,2-a]pyrazin-3-one, 2methyl-6-phenyl-3,7-dihyroimidazo[1,2-a]pyrazin-3-one and 2-methyl-6-[p-[2-[sodium 3-carboxylato-4-(6-hydroxy-3-xanthenon-9-yl]phenylthioureylene]ethyleneoxy]phenyl]-3,7-dihyroimidazo[1,2-ajpyrazin-3-one. In other embodiments, the chemiluminescent moieties intended for use herein include, but are not limited to, luminol, isoluminol, N-(4-aminobutyl)-N-ethyl isoluminol (ABEI), N-(4-aminobutyl)-N-methyl isoluminol (ABMI), which have the following structures and participate in the following reactions:<br>
R	9.	R	9<br><br><br>
NH ,NH<br><br>
02" (H202)<br>
PEROXIDASE<br><br>
+     N2   +   light<br><br>
6	o<br>
where luminol is represented, when R is NH2 and R1 is H; isoluminol, when R is H and R1 is NH2; for ABEI ((6-[N-(4-aminobutyl)-N-ethylamino]-2,3-dihyrophthalazine-1-4-dione), when R is H and R1 is C2H5-N-(CH2)4NH2; and for ABMI ((6-[N-(4-aminobutyl)-N-methylamino]-2,3-dihyrophthalazine-1-4-dione), when R is H and R1 is CH3-N-(CH2)4NH2.<br>
Bioluminescent groups for use herein include luciferase/luciferin couples, including firefly [Photinus pyralis] luciferase, the Aequorin system (i.e., the purified jellyfish photoprotein, aequorin). Many luciferases and substrates have been studied and well-characterized and are commercially available (e.g., firefly luciferase is available from Sigma, St. Louis, MO, and Boehringer Mannheim Biochemicals,Indianapolis, IN; recombinantly produced<br><br>
-102-<br>
firefly luciferase and other reagents based on this gene or for use with this protein are available from Promega Corporation, Madison, Wl; the aequorin photoprotein luciferase from jellyfish and luciferase from Ren/7/a are commercially available from Sealite Sciences, Bogart, GA; coelenterazine, the naturally-occurring substrate for these luciferases, is available from Molecular Probes, Eugene, OR]. Other bioluminescent systems include crustacean, such as Cyrpidina (Vargula), systems; insect bioluminescence generating systems including fireflies, click beetles, and other insect systems; bacterial systems; dinoflagellate bioluminescence generating systems; systems from molluscs, such as Latia and Pholas; earthworms and other annelids; glow worms; marine polycheate worm systems; South American railway beetle; fish (i.e., those found in species of Aristostomias, such as A. scintillans (see, e.g.. O'Day etal. (1974) Vision Res. 74:545-550), Pachystomias. and Malacosteus, such as M.niger, blue/green emmitters include cyclthone, myctophids, hatchet fish (agyropelecus), vinciguerria, howella, florenciella. and Chauliodus); and fluorescent proteins, including green (i.e., GFPs, including those from Renilla and from Ptilosarcus), red and blue (i.e., BFPs, including those from Vibrio fischeri, Vibrio harveyi or Photobacterium phosphoreum) fluorescent proteins (including Renilla mulleri luciferase, Gaussia species luciferase and Pleuromamma species luciferase) and phycobiliproteins.<br>
Examplary selectivity functions include, but are not limited to, ligands that bind to receptors such as insulin and other receptors (see, e.g., the Table of ligands below); cyclodextrins; enzyme substrates; lipid structures; prostaglandins; antibiotics; steroids; therapeutic drugs; enzyme inhibitors; transition state analogs; specific peptides that bind to biomolecule surfaces, including glue peptides; lectins (e.g., mannose type, lactose type); peptide mimetics; statins; functionalities, such as dyes and other compounds and moieties employed for protein purification and affinity chromatraphy. See e.g., Figure 17, and the following table of peptide ligands:<br><br>
-103-<br><br>
Exemplary peptide ligands                                               II<br>
resignation	Sequence	SEQID<br>
Adrenocorticotropichormone	SYSMEHFRWG KPVGKKRRPVKVYPNGAEDE SAEAFPLEF	1<br>
Adrenomedullin	YRQSMNNFQG LRSFGCRFGT CTVQKLAHQI YQFTDKDKDN VAPRSKISPQ GY	2<br>
Allatostatin l-IV	APSGAQRLYGFGL	3<br>
alpha MSH	WGKPV(ac)SYSMEHFR	4<br>
alpha-Bag Cell Peptide	APRERFYSE	5<br>
alpha-Neo-endorphin	YGGFLRKYPK	6<br>
Alytesin	E'GRLGTQWAV GHLM-NH2	7<br>
Amylin	KCNTATCATN RLANFLVHSS NNFGAILSST NVGSNTY	8<br>
Angiotensin-1	DRVYIHPFHL	9<br>
Angiotensin-2	DRVYIHPF	10<br>
Angiotensin-3	RVYIHPF	11<br>
Apelin-13	NRPRLSHLGPMPF	12<br>
Astressin	*FHLLREVLE*IARAEQLAQEAHKNRL*IEII	13<br>
Atrial Natriuretic Peptide	SLRRSSCFGG RMDRIGAQSG LGCNSFRY	14<br>
Autocamtide 2	KKALRRQETV DAL	15<br>
BAM 12	YGGFMRRVGR PE	16<br>
BAM 18	YGGFMRRVGR PEWW	17<br>
BAM22	YGGFMRRVGR PE	18<br>
Beta Endorphins ("44")	YGGFMTSEKS QTPLVTLFKN AIIKNAYKKGE	19<br>
beta MSH	AEKKDEGPYR MEHFRWGSPP KD	20<br>
beta- Neo-endorphin	YGGFLRKYP	21<br>
BetaAmyloid	DAEFRHASGYE VHHQKLVFFAE DVGSNLGAIIG LMVGGWIAT	22<br>
Beta-Bag Cell Peptide	RLRFH	23<br>
BNP	SPKMVOGSGC FGRKMDRISS SSGLGCKVLR RH	24<br>
I Bradykinin                           I	RPPGFSPFR	25     |<br><br>
-104-<br><br>
Exemplary peptide ligands<br>
Buccalin	GMDSLAFSGG L-NH2	26<br>
Bursin	KHG-NH2	27<br>
C3 (undeca peptide)	ASKKPKRNIKA	28<br>
Caerulein	*EQDY(S03H)TGWMDF	29<br>
Calcineurin	AIP ITSFEEAKGL DRINERMPPR RDAMP	30<br>
Calcitonin	CGNLSTCMLG TYTQDFNKFH TFPQTAIGVG AP	31<br>
Calpain Inhibitor ("42")	DPMSSTYIEE LGKREVTIPP KYRELLA	32<br>
CAP-37	NQGRHFCGGA EIHARFVMTA ASCFN	33<br>
Cardiodilatin	* NPMYNAVSNA DLMDFKNLLD HLEEKN;PLED	34<br>
CD36peptideP (139-155)	CNLAVAAASH IYQNQFVQ	35<br>
Cecropin B	KWKVFKKIEK MGRNIRNGIV KAGPAIAVLG EAKAL	36<br>
Cerebellin	SGSAKVAFSA IRSTNH	37<br>
CGRP-1	ACDTATCVTH RLAGLLSRSG GWKNNFVPT NVGSKAF	38<br>
C3RP-2	ACNTATCVTH RLAGLLSRSG GMVKSNFVPT NVGSKAF	39<br>
CKS17	LQNRRGLDLL FLKEGGL	40<br>
Cortistatins	QEGAPPQQSA RRDRMPCRNF FWKTFSSCK	41f<br>
Crystalline	WG	42<br>
Defensin 1 HNP1	ACYCRIPACI AGERRYGTCI YQGRLWAFCC	43 -<br>
Defensin HNP2	CYCRIPACIA GERRYGTCIY QGRLWAFCC	44<br>
Dermaseptin	ALWKTMLKKL GTMALHAGKA ALGAAADTIS QTQ	45<br>
Dynorphin-A	YGGFLRRIRP KLKWDNQ	46<br>
Dynorphin-B	YGGFLRRQFK WT	47<br>
Eledoisin	E*PSKDAFIGLM-NH2	48<br>
Endomorphin-1	YPWF	49<br>
Endomorphin-2	YPFF	50<br>
Endothelin-1	CSCSSLMDKE CVYFCHLDII W	51<br><br>
-105-<br><br>
Exemplary peptide ligands<br>
Exendin-4	HSDGTFTSDL SKQMEEEAVRLFIEWLKNGG PSSGAPPPS(NH2)	52<br>
Fibrinopeptide	AADSGCGDFLA EGGGVR	53<br>
Fibrinopeptide	BQGVNDNEEGF FSAR	54<br>
Fibrpnectin CS1	EILDVPST	55<br>
FMRF	FMRF	56<br>
Galanin	GWTLNSAGYL LGPHAVGNHR SFSDKNGLTS	57<br>
Galantide	GWTLNSAGYL LGPQQFFGLM(NH2)	58<br>
gamma-Bag Cell Peptide	RLRFD	59<br>
Gastrin	EGPWLEEEEE AYGWMDF	60<br>
Gastrin Releasing	VPLPAGGGTV LTKMYPRGNH WAVGHLM	61<br>
Ghrelin	GSSFLSPEHQ RVQQRKESKK PPAKLQPR	62<br>
GIP	YAEGTFISDY SIAMDKIHQQ DFVNWLLAQK GKKNDWKHNI TQ	63<br>
Glucagon	HSQGTFTSDY SKYLDSRRAQ DFVDWLMNT	64<br>
Grb-7 SH2 domain-1	RRFA C DPDG YDN YFH C VPGG	65<br>
Grb-7SH2domain-10	TGSW C GLMH YDN AWL C NTQG	66<br>
Grb-7 SH2 domain-11	RSKW C RDGY YAN YPQ C WTQG	67<br>
Grb-7 SH2 domain-18	RSTL C WFEG YDN TFP C KYFR	68<br>
Grb-7 SH2 domain-2	RVQE C KYLY YDN DYL C KDDG	69<br>
Grb-7 SH2 domain-23	GLRR C LYGP YDN AWV C NIHE	70<br>
Grb-7 SH2 domain-3	KLFW C TYED YAN EWP C PGYS	71<br>
Grb-7 SH2 domain-34	FCAV C NEEL YEN CGG C SCGK	72<br>
Grb-7 SH2 domain-46	RTSP C GYIG YDN IFE C TYLG	73<br>
Grb-7 SH2 domain-5	TGEW C AQSV YAN YDN C KSAW	74<br>
Grb-7 SH2 domain-6	NVSR C TYIH YDN WSL C GVEV	75<br>
Grb-7 SH2 domain-8	GVSN C VFWG YAN DWL C SDYS	76<br>
Growth hormone releasing factor	YADAIFTNSY RKVLGQLSAR KLLQDIMSRQ QGESNQERGA RARL	77<br>
Guanylin	PGTCEICAYA ACTGC	78<br>
Helodermin	HSDAIFTEEY SKLLAKLALQ KYLASILGSR TSPPP-NH2	79<br><br>
-106-<br><br>
Exemplary peptide ligands<br>
Helospectin-1	HSDATFTAEY SkLLAKLALQ KYLESILGSS TSPRPPSS	80<br>
Helospectin-2	HSDATFTAEY SKLLAKLALQ KYLESILGSS TSPRPPS	81<br>
Histatin 5	DSHAKRHHGY KRKFHEKHHS HRGY	82<br>
ICE inhibitor(lll)	ac-YVAD-fluroacyloxymethylketone	83<br>
Immunostimulating Peptide	VEPIPY	84<br>
Insulin (A-chain)	GIVEQCCTSI CSLYQLENYC N	85<br>
Insulin (B-chain)	FVNQHLCGSH LVEALYLVCG ERGFFYTPKT	86<br>
Insulin (whole molecule)	see above	87<br>
Kinetensin	IARRHPYFL	88<br>
Leu-Enkephalin	YGGFL	89<br>
Litorin	E*QWAVGHFM-NH2	90<br>
Malantide	RTKRSGSVYE PLKI	91<br>
Met-Enkephalin	YGGFM	92<br>
Metorphamide	YGGGFMRRV-NH2	93<br>
Motilin	FVPIFTYGEL QRMQEKERNK GQ	94<br>
Myomodulin	PMSMLRL-NH2	95<br>
Myosin Kinase	IPKKRAARATS-NH2	96<br>
Necrofibrin	GAVSTA	97<br>
Neurokinin A	HKTDSFVGLM-NHz	98<br>
Neurokinin B	DMHDFFVGLM-NH2	99<br>
Neuromedin B	GNLWATGHFM-NH2	100<br>
Neuropeptide Y	YPSKPDNPGE DAPAEDMARY YSAKRHYINL ITRQRY-NH2	101<br>
Neurotensin	E'LYENKPRRPUIL	102<br>
Nociceptin	FGGFTGARKS ARKLANQ	103<br>
Nociceptin/Orphanin FQ	FAEPLPSEEE GESYSKEVPE MEKRYGGFMR F	104<br>
Nocistatin	EQKQLQ	105<br>
Orexin A	E'PLPDCCRQKTCSCRLYELLHGAGNHAAGI LTL-NH2	106<br><br>
-107-<br><br>
Exemplary peptide ligands<br>
Orexin B	RSGPPGLQGR LQRLLQASGN HAAGILTM-NH2                                ,	107<br>
Osteocalcin	YLYQWLGAPV PYPDPLEPRR EVCELNPDCD ELADHIGFQE AYRRFYGPV	108<br>
Oxytocin	CYIQNCPLG-NH2	109<br>
PACAP	HSDGIFTDSY SRYRKQMAVK KYLAAVL	110<br>
P4CAP-RP	DVAHGILNEA YRKVLDQLSA GKHLGSLVA	111<br>
Pancreatic Polypeptide	APLEPVYPGD NATPEQVIAQY AADLRRYINM LTRPRY-NH2	112<br>
Papain Inhibitor	GGYR	113<br>
Peptide E	YGGFMRRVGR PE	114<br>
Peptide YY	YPIKPEAPGE DASPEELNRY YASLRHYLNL VTRQRY-NH2	115<br>
Phosphate acceptor	RRKASGPPV	116<br>
Physalaemin	E*ADPNKFYGLM-NH2	117<br>
Ranatensin	E*VPQWAVGHFM-NH2	118<br>
RGD peptides	X-RGD-X	119<br>
Rigin	GQPR	120<br>
RR-SRC	RRLIEDAEYA ARG	121<br>
Schizophrenia	RPTVL	122<br>
Secretin	HSDGTFTSEL SRLREGARLQ RLLQGLV	123<br>
Serum Thymic Factor	E*AKSQGGSN	124<br>
structural-site zinc ligands-alpha	PQCGKCRICK NPESNYCLK	125<br>
structural-site zinc ligands-beta	PQCGKCRVCK NPESNYCLK	126<br>
structural-site zinc ligands-gamma	PQCGKCRICK NPESNYCLK	127<br>
structural-site-zinc ligands-Pi	PLCRKCKFCLSPLTNLCGK	128<br>
structural-site-zinc ligands-X	PQGECKFCLNPKTNLCQK	129<br>
Substance P	RPKPQQFFGL M-NH2	130<br><br>
-108-<br><br>
Exemplary peptide ligands<br>
Syntide 2	PLARTLSVAG LPGKK	131<br>
Systemin	AVQSKPPSKR DPPKMQTD	132<br>
Thrombin-light chain	TFGSGEADCG LRPLFEKKSL EDKTERELLE SYIDGR	133<br>
Thymopentin	RKDVY	134<br>
Thymus Factor	QAKSQGGSN	135<br>
TRH	E*HP	136<br>
Tuftsin	TKPR	137<br>
Uperolein	E*PDPNAFYGLM-NH2	138<br>
Uremic Pentapeptide	DLWQK	139<br>
Urocortin	DNPSLSIDLT FHLLRTLLEL ARTQSQRERA EQNRIIFDSV	140<br>
Uroguanylin	NDDCELCVNV ACTGCL	141<br>
Vasonatrin	GLSKGCFGLK LDRIGSMSGL GCNSFRY	142<br>
Vasopressin	CYFQNCPRG	143<br>
Vasotocin	CYIQNCPRG	144<br>
VIP	HSDAVITDNY TRLRKQMAVK KYLNSILN	145<br>
Xenin	MLTKFETKSA RVKGLSFHPK RPWIL	146<br>
YXN motif	Tyr-X-Asn	147<br>
Zinc ligand of carbonic anhydrase I	FQFHFHWGS	148<br>
Zinc ligand of carbonic anhydrase	IIIQFHFHWGS	149<br>
Other selections for Y are can be identified by those of skill in the art and include, for example, those disclosed in Techniques in Protein Chemistry, Vol. 1 (1989) T. Hugli ed. (Academic Press); Techniques in Protein Chemistry, Vol. 5(1994) J.W. Crebb ed. (Academic Press); Lundblad Techniques in Protein Modification (1995) (CRC Press, Boca Raton, FL); Glazer et al. (1976) Chemical Modification of Proteins (North Holland (Amsterdam))(American Elsevier, New York); and Hermanson (1996) Bioconjugate Techniques (Academic Press, San Diego, CA).<br><br>
-109-<br>
4.       Sorting Functions "Q"<br>
The compounds provided herein can include a sorting function ("Q"), which permits the compounds to be addressed, such as by capture in a 2-D array. In certain embodiments, the sorting function is selected to not interact with the biomolecules (e.g. target and non-target proteins) in the sample. The sorting functions are "tags", such as oligonucleotide tags, such that when the compounds are bathed over an array of complementary oligonucleotides linked to solid supports, such as beads, chips, under suitable binding     '"-conditions, the oligonucleotides hybridize. The identity of the capture compound can be known by virtue of its position in the array. Other sorting functions can be optically coded, including as color coded or bar coded beads that can be separated, or an electronically-tagged, such as by providing microreactor supports with electronic tags or bar coded supports (see, e.g., U.S. Patent No. 6.025.129: U.S. Patent No. 6,017,496; U.S. Patent No. 5 972.639: U.S. Patent No. 5,961,923; U.S. Patent No. 5,925,562; U.S. Patent No 5.874,214; U.S. Patent No. 5,751,629; U.S. Patent No. 5.741,462). or chemical tags (see, e.g., U.S. Patent No. 5,432,018; U.S. Patent No 5,547,839) or colored tags or other such addressing methods that can be used in place of physically addressable arrays. The sorting function is selected to permit physical arraying or other addressable separation method suitable for analysis, particularly mass spectrometric, including MALDI, analysis.<br>
Other sorting fuctions for use in the compounds provided herein include biotin, (His)6, BODIPY (4,4-difluoro-4-bora-3a,4a-diaza-s-indacene), oligonucleotides, nucleosides, nucleotides, antibodies, immunotoxin conjugates, adhesive peptides, lectins, liposomes, PNA (peptide nucleic acid), activated dextrans and peptides. In one embodiment, the sorting function is an oligonucleotide, particularly, either a single-stranded or partially single-stranted oligonucleotide to permit hybridization to single-stranded regions on complementary oligonucleotides on solid supports.<br>
In one embodiment of the capture compounds provided herein, Q is a<br><br>
-110-<br>
single stranded unprotected or suitably protected oligonucleotide or oligonucleotide analog (e.g., PNA) of up to 50 building blocks, which is<br>
capable of hybridizing with a base-complementary single stranded nucleic acid molecule. In certain embodiments, Q contains from about 5 up to about 10, 15, 25, 30, 35, 40, 45 or 50 building blocks.<br>
Biomolecule mixtures, including, but not limited to, protein mixtures, can have different hydrophobicities (solubility) than the compounds provided herein. In certain embodiments, in order to achieve high reaction yields between the functionality X on the compounds provided herein and the protein surface, the reaction is performed in solution. In other embodiments, the reaction is performed at a solid/liquid or liquid/liquid interface. In certain embodiments, the solubility properties of the compounds provided herein are dominated by the Q moiety. A change in the structure of Q can, in these embodiments, accommodate different solubilities. For example, if the protein mixture is very water soluble, Q can have natural phosphodiester linkages; if the bimolecular mixture is very hydrophobic (lipids, glycolipids, membrane proteins, lipoproteins). Q can have it's phosphodiester bonds protected as phosphotriesters, or alternatively, these bonds can be methylphosphonatediesters or peptide nucleic acids (PNAs). If the biomolecule mixture is of an intermediate hydrophobicity, solubility is achieved, e.g.. with phosphothioate diester bonds. Intermediate solubility also can be attained by mixing phosphodiester with phosphotriester linkages. Those skilled in the art can easily conceive of other means to achieve this goal, including, but not limited to, addition of substituents on Z, as described elsewhere herein, or use of beads for Z that are hydrophobic, including, but not limited to, polystyrene, polyethylene, polypropylene or teflon, or<br>
hydrophilic, including, but not limited to, cellulose, dextran crossâlinked with epichlorohydrin (e.g.. SephadexÂ®), agarose (e.g., SepharoseÂ®), lectins, adhesive polypeptides, and polyacrylamides.<br>
The flexibility of being able to change the solubility of the compounds is<br>
a significant advantage over current methods. In contrast, 2D gef<br><br>
-111-<br>
electrophoresis is useful only for analysis of water soluble proteins with the result that about 30 to 35% of all cellular proteins, such as those residing in the cell membrane, cannot be analyzed by this method. This is a severe limitation of 2D gel electrophoresis since many proteins, including, but not limited to, those involved in tissue specific cell-cell contacts, signal transduction, ion channels and receptors, are localized in the cell membrane.<br>
In one embodiment, after reaction or complexation of the compounds provided herein with a biomolecule, including, but not limited to, a protein, the compounds are brought into contact with a set of spatially resolveo complementary sequences on a flat support, beads or microtiter plates under hybridization conditions.<br>
In certain embodiments, Q is a monovalent oligonucleotide or oligonucleotide analog group that is at least partially single stranded or includes a region that can be single-stranded for hybridization to complementary oligonucleotides on a a support.   Q can have the formula: N1mBiN2n<br>
where N1 and N2 are regions of conserved sequences; B is a region of sequence permutations; m, i and n are the number of building blocks in N1, B<br>
and N2, respectively; and the sum of m, n and i is a number of units able to<br>
hybridize with a complementary nucleic acid sequence to form a stable hybrid. Thus, in embodiments where B is a single stranded DNA or RNA, the number of sequence permutations is equal to 4'.  In one embodiment, the sum of m, n and i is about 5 up to about 10,15, 25, 30, 35, 40, 45 or 50. In certain embodiments m and n are each independently 0 to about 48, or are each independently about 1 to about 25, or about 1 to about 10 or 15, or about 1 to about 5. In other embodiments, i is about 2 to about 25, or is about 3 to about 12, or is about 3 to about 5, 6, 7 or 8.<br>
The oligonucleotide portion, or oligonucleotide analog portion, of the compounds (N1mBiN2n), can be varied to allow optimal size for binding and sequence recognition. The diversity of the sequence permutation region B can be relatively low if the biomolecule mixture, including, but not limited to,<br><br>
-112-<br>
protein mixtures, is of low complexity. If the mixture is of high complexity, the sequence region B has to be of high diversity to afford sufficient resolving<br>
power to separate all the species. The flanking conserved regions N m and N n, need only be long enough to provide for efficient and stable hybrid formation. There is, however, flexibility in designing these regions: N1m and N n can be of the same length and same sequence, of the same length and different sequence or of different length and different sequence. In certain embodiments, including those where B is of sufficient length to provide stable hybrid formation, N1 and/or N2 are absent. In these embodiments, the oligonucleotide portion of the compounds, or oligonucleotide analog portion of<br>
the compounds, has the formula N1mBi, or BN2n, or Bi.<br>
In an exemplary embodiment (see, e.g., EXAMPLE 1.a.), B has a trinucleotide sequence embedded within a 11-mer oligonucleotide sequence, where the N1m and N2n tetranucleotide sequences provide tianking identical (conserved) regions. This arrangement for N1mBiN2n affords 64 different compounds where each compound carries the same reactive functionality X. In another exemplary embodiment (see, e.g., EXAMPLE 1.b.), B has a tetranucleotide sequence embedded within a 12-mer oligonucleotide sequence, where the N1m and N2n oligonucleotide sequences provide flanking but not identical octanucleotide sequences. This arrangement for N1mBjN n affords 256 different compounds where each carry the same reactive functionality X. In a further exemplary embodiment (see, e.g., EXAMPLE 1 c), B has an octanucleotide sequence embedded within a 23-mer oligonucleotide sequence, where the N1m and N2n oligonucleotide sequences provide flanking but not identical octanucleotide sequences. This arrangement for N1mBjN2n affords 65,536 different compounds where each carries the same reactive functionality X, and exceeds the estimated complexity of the human proteome (e.g., 30,000-35,000 different proteins). In certain embodiments, use of a B with excess permutations for the complexity of the protein mixture, as the oligonucleotides with the best hybridization<br>
properties can be used for analysis ;o reduce mismatching.<br><br>
-113-<br>
5.	Solubility Functions "W"<br>
The compounds provided herein can incude a solubility function, W, to<br>
confer desired solubility properties, such as solubility in hydrophobic<br>
environments or hydrophilic environments to permit probing of biomolecules in physiological environments, such as in membranes. Exemplary solubility functions for use in the compounds provided herein include polyethylene glycols, sulfates, polysulfates, phosphates, sulfonates, polysulfbnates, carbohydrates, dextrin, polyphosphates, poly-carboxylic acids, triethanolamine, alcohols, water soluble polymers, salts of alkyl and aryl carboxylic acids and glycols.<br>
Amphiphilic compounds, such as quaternary ammonium salts (i.e., betain, choline, sphingomyelin, tetramethyl (or tetrabutyl) alkyl ammonium salts, cationic. ionic and neutral tensides may also be used as the solubility function W.<br>
In other embodiments, W also can be used to modulate the solubility of the compounds to achieve homogeneous solutions, if desired, when reacting with biomolecule mixtures, including, but not limited to, protein mixtures. In certain embodiments, W is a sulfonate, a polar functionality that can be used to make the compounds more water-soluble. In other embodiments. W is a hydrophobic group, including lower alkyl, such as tert-butyl, tert-amyl, isoamyl, isopropyl, n-hexyl, sec-hexyl, isohexyl, n-butyl, sec-butyl, iso-butyl and n-amyl, or an aryl group, including phenyl or naphthyl.<br>
6.	Exemplary Embodiments<br>
The following provides exemplary capture compounds that exhibit the above-described properties. It is understood that these are exemplary only and that any compounds that can react covalently with a biomolecule or by other highly stable interaction that is stable to analytic conditions, such as those of mass spectrometric analysis, and that can sorted or otherwise identified are contemplated for use in the collections. <br>
a.       Exemplary embodiment 1<br>
In one embodiment, the compounds for use in the methods provided<br><br>
-114-<br>
herein have formulae: OZX or Q-Z-Y,<br>
where Q is a sorting function that contains a single stranded unprotected or suitably protected oligonucleotide or oligonucleotide analog (e.g., peptide nucleic acid (PNA)) of up to 50 building blocks, which is capable of hybridizing with a base-complementary single-stranded nucleic acid molecule;<br>
Z is a moiety that is cleavable prior to or during analysis of a biomolecule, including mass spectral analysis, without altering the structure of the biomolecule, including, but not limited to, a protein;<br>
X is a reactivity functional group that interacts with and/or reacts with functionalities on the surface of a biomolecule, including, but not limited to. a protein, to form covalent bonds or bonds that are stable under conditions of mass spectrometric analysis, particularly MALDI analysis; and<br>
Y is a selectivity functional group that interacts with and/or reacts by imposing unique selectivity by introducing functionalities that interact noncovalently with target proteins.<br>
b.       Exemplary embodiment 2<br>
In another embodiment, the compounds for use in the methods provided herein have formula:<br>
Qâzâx â¢<br>
I<br>
Y<br>
where Q is a single-stranded unprotected or suitably protected oligonucleotide or oligonucleotide analog (e.g., peptide nucleic acid (PNA)) of up to 50 building blocks, which is capable of hybridizing with a base-complementary single stranded nucleic acid molecule;<br>
Z is a moiety that is cleavable prior to or during analysis of a biomolecule, including mass spectral analysis, without altering the structure of the biomolecule, including, but not limited to, a protein;<br>
X is a functional group that interacts with and/or reacts with functionalities on the surface of a biomolecule, including, but not limited to, a<br><br>
-115-<br>
protein, to form covalent bonds or bonds that are stable under conditions of mass spectrometric analysis, particularly MALDI analysis; and<br>
, Y is a functional group that interacts with and/or reacts by imposing unique selectivity by introducing functionalities that interact noncovalently with target proteins.<br>
c.	Exemplary embodiment 3<br>
In another embodiment, the compounds for use in the methods provided herein have formula:<br><br>
where Q is a sorting function that is a compound, or one or more biomolecules (e.g., a pharmaceutical drug preparation, a biomolecule, drug or other compound that immobilizes to the substrate and captures target biomolecules), which is(are) capable of specific noncovalent binding to a known compound to produce a tighly bound capture compound;<br>
Z is a moiety that is cleavable prior to or during analysis of a biomolecule, including mass spectral analysis, without altering the structure of<br>
the biomolecule, including, but not limited to. a protein;<br>
X is a functional group that interacts with and/or reacts with functionalities on the surface of a biomolecule, including, but not limited to, a protein, to form covalent bonds or bonds that are stable under conditions of mass spectrometric analysis, particularly MALDI analysis; and<br>
Y is a functional group that interacts with and/or reacts by imposing unique selectivity by introducing functionalities that interact noncovalently with target proteins.<br>
d.	Exemplary embodiment 4<br>
In another embodiment, the compounds for use in the methods<br>
provided herein have the formulae:<br><br>
-116-<br>
Q	z	(X)m<br>
(Y)n<br>
A<br>
or Q-Z-(X)m or Q-Z-(Y)â,<br>
where Q, Z, X and Y are as defined above; m is an integer from 1 to 100, in one embodiment 1 to 10, in another embodiment 1 to 3, 4 or 5; and n in an integer from i to 100, in one embodiment 1 to 10, in another embodiment 1 to<br>
1    ".   - - r.. . u,   -r   ui   J.<br>
e.	Exemplary embodiment 5<br>
In another embodiment, X is a pharmaceutical drug. The compounds of these embodiments can be used in drug screening by capturing biomolecules, including but not limited to proteins, which bind to the pharmaceutical drug. Mutations in the biomolecules interfering with binding t the pharmaceutical drug are identified, thereby determining possible mechanisms of drug resistance. See, e.g., Hessler et al. (November 9-11, 2001) Ninth Foresight Conference on Molecular Nanotechnology (Abstract)(http://www.foresight.org/Conferences/MNT9/Abstracts/Hessler/).<br>
f.	Other embodiments<br>
In certain embodiments, the compounds provided herein have the<br>
*<br>
formula:<br>
N1mBiN2n(S1),M(R15)a(S2)bLX<br>
where N1, B, N2, S1, M, S2, L, X, m, i, n, t, a and b are as defined above. In<br>
further embodiments, the compounds for use in the methods provided herein<br>
include a mass modifying tag and have the formula:<br>
N1mBiN2n(S1),M(R15)a(S2)bLTX, where<br>
N1, B, N2, S1, M, S2, L, T, X, m, i, n, t, a and b are as defined above.<br>
In other embodiments, including those where Z is not a cleavable linker, the compounds provided herein have the formula: N1mBiN2n(S1)tM(R15)a(S2)bX, where N1, B, N2, S1, M, S2, X, m, i, n, t, a and b are as defined above.<br>
In another embodiment, the compounds for use in the methods<br><br>
-117-<br>
provided herein include those of formulae:<br><br><br><br>
t'<br>
\\      a<br>
\^<br>
V^<br>
/)	C	LY      â¢       ^I<br>
QM'	//<br><br><br>
XM YM<br><br><br>
'   1<br>
-C	LQ    <br>
where L and M are each independently O, S or NR3; X is a reactivity function, as described above; Y is a selectivity function, as described above; Q is a sorting function, as described above; and each R3 is independently hydrogen, substituted or unsubstituted alkyl, substituted or unsubstituted alkenyl, substituted or unsubstituted alkynyl, substituted or unsubstituted cycloalkyl, substituted or unsubstituted heterocyclyl, substituted or unsubstituted aryl, substituted or unsubstituted heteroaryl, substituted or unsubstituted aralkyl, or substituted or unsubstituted heteroaralkyl.In another embodiment, the capture compounds provided herein have<br><br>
-118-<br>
the formula:<br>
Â°*^x<br>
-LY<br>
QM<br>
where L, M, X. Y and Q are as defined above.<br>
In another embodiment, the capture compounds provided herein have the formula:<br><br>
-119-<br><br><br>
^<br>
^N<br>
^<br><br><br>
^	ffl<br><br><br><br>
(R')y'<br><br>
(R : y<br><br>
(R'V<br><br><br><br>
(R' = )y<br>
(R -,y<br><br>
(R,5)y<br><br>
where L, M, X, Y and Q are as defined above. n1, n2 and n3 are 0 to 5. In another embodiment, n1, n2 and n3 are selected with the proviso that n1, n2 and n3 are not all 0.<br>
In another embodiment, the capture compounds provided herein have the formula.<br><br>
-120-<br><br>
Y-HN   or    O<br>
6     or NH-S'-Q<br><br>
orNH-X<br><br>
where X, Y, Q and S1 are as defined above.<br>
In another embodiment, the capture compounds provided herein have the formula:<br>
O<br><br>
Y-HN   or   0<br><br>
-0   orNH-X<br><br>
0'<br><br><br>
MeCT<br><br>
vOMe<br><br>
0' or NH-S'-Q<br>
w! iere Q, Y, X and S1 are as defined above.<br>
In another embodiment, the capture compounds provided herein have the formula:<br><br>
-121-<br><br><br>
O-HN   or   o<br>
,0 (Y or W or H)<br>
(Y or W or H)0<br>
O (Y or W or H)<br><br>
where X. Y Q and W are as defined above.<br>
In another embodiment, the capture compounds provided herein have<br>
the formula:<br><br><br>
(YorWorH)O<br>
O(HorWorY)<br><br>
where X. Y. Q and W are as defined above.<br>
In another embodiment, the capture compounds for use in the methods provided herein have the formulae:<br><br>
-122-<br><br><br><br>
s2-x<br>
&gt;c<br>
\^<br><br><br><br><br><br><br>
^y<br><br>
5<x></x><br>
s2-x<br><br>
^<br><br>
W-R-<br><br>
or<br><br>
W-R-<br><br>
yy<br>
Q<br>
Q<br>
where X. Y, Q and W are selected as above; and R is substituted or unsubstituted alkyl, substituted or unsubstituted cycloalkyi, substituted or unsubstituted cycloalkylalkyl, or substituted or unsubstituted aralkyl. In another embodiment, R is selected,from cyclohexyl, cyclohexyl-(CH2)n, isopropyl, and phenyl-(CH2)n, where n is 1, 2 or 3. As shown in the formulae above, R is optionally substituted with W.<br>
In other embodiments, the compounds for use in the methods provided herein include:<br><br>
-123-<br><br>
MeO-<br><br>
-OH<br><br><br><br>
O^O-N<br><br><br><br>
0<br>
o<br><br>
OMe<br><br>
MeO	<br><br><br>
-OH<br><br><br><br>
O^-o-N<br><br><br><br>
O<br>
0<br><br>
-124-<br>
OMe<br><br><br>
CT	O	N<br><br>
-125-<br><br>
Specific compounds within these embodiments are those resulting from all combinations of the groups listed above for the variables contained in this formula and all can include Q groups. It is intended herein that each of these specific compounds is within the scope of the disclosure herein.<br>
D.       Preparation of the Capture Compounds<br>
The capture compounds are designed by assessing the target biomolecules and reaction conditions. For example, if the target biomolecules are proteins, X functions suitable to effect covalent or binding to proteins with high affinity are selected. Y is selected according to the<br><br>
-126-<br>
complexity of the target mixture and the desired specificity of binding by X. Q is selected according the number of divisions of the mixture that are desired; and W is selected based upon the environment of the biolmolecules that is probed. A variety of capture co, .'.pounds are designed according to such<br>
criteria.<br>
The capture compounds once designed can be synthesized by methods available to those of skill in the art. Preparation of exemplary capture compounds is described below. Any capture compound or similar capture compound can be synthesized according to a method discussed in general below or by minor modification of the methods by selecting appropriate starting materials or by methods known to those of skill in the art<br>
In general, the capture compounds can prepared starting with the central moiety Z.  In certain embodiments, Z is (S1)tM(R15)a(S2)bL.  In these embodiments, the capture compounds can be prepared starting with an appropriately substituted (e.g., with one or more R15 groups) M group. M(R15)a is optionally linked with S1 and/or S2, followed by linkage to the cleavable linker L. Alternatively, the L group is optionally linked to S , followed by reaction with M(R15)a, and optionally S1. This Z group is then derivatized on its S1 (or M(R15)a) terminus to have a functionality for coupling with an oligonucleotide or oligonucleotide analog Q (e.g., a phosphoramidite, H-phosphonate, or phosphoric triester group). The Q group will generally be N-protected on the bases to avoid competing reactions upon introduction of the X moiety. In one embodiment, the Z group is reacted with a mixture of ail possible permutations of an oligonucleotide or oligonucleotide Q (e.g., 4' permutations where i is the number of nucleotides or nucleotide analogs in B). The resulting QZ capture compound or capture compounds is(are) then derivatized through the L terminus to possess an X group for reaction with a biomolecule, such as a protein.  If desired, the N-protecting groups on the Q moiety are then removed. Alternatively, the N-protecting groups can be removed following reaction of the capture compound with a biomolecule, including a protein. In other embodiments, Q can be synthesized on Z,<br><br>
-127-<br>
including embodiments where Z is an insoluble support or substrate, such as a bead. In a further embodiment. Q is presynthesized by standard solid state<br>
techniques, then linked to M. Alternatively, Q can be synthesized stepwise on the M moiety.<br>
Provided below are examples of syntheses of the capture compounds provided herein containing alkaline-labile and photocleavable linkers.  One of skill in the art can prepare other capture compounds disclosure by routine modification of the methods presented herein, or by other methods known to those of skill in the art.<br>
For synthesis of a compound provided herein containing an alkaline-labile linker. 1.4-di(hydroxymethyl)benzene (i.e.. M) is mono-protected, e.g., as the corresponding mono-tert-butyldimethylsilyl ether   The remaining free alcohol is denvatized as the corresponding 2-cyano-ethyl-A/,/V-dii.sopropylphosphoramidite by reaction with 2-cyanoethyl-A/,A/-diisopropylchlorophosphoramidite. Reaction of this amidite with an oligonucleotide (i.e., Q), is followed by removal of the protecting group to provide the corresponding alcohol. Reaction with, e.g., trichloromethyl chloroformate affords the illustrated chloroformate (i.e., X).<br><br>
o conucleotide	!<br>
aoligonucleotide<br>
For the synthesis of a compound provided herein containing a photocleavable linker, 2-nitro-5-hydroxybenzaldehyde (i.e., a precursor of L) is reacted with, e.g., 3-bromo-1-propanol to give the corresponding ether-alcohol. The alcohol is then protected, e.g., as the corresponding tert-<br><br>
-128-<br>
butyldimethylsilyl ether. Reaction of this compound with trimethylaluminum gives the corresponding benzyl alcohol, which is derivatized as its phosphoramidite using the procedure described above. The amidite is reacted with an oligonucleotide (i.e., Q), followed by removal of the protecting group and derivatization of the resulting alcohol as the corresponding chloroformate (i.e., X).<br><br>
CHO	CHO<br>
/^        ^CL	^v.	^OTBDMS<br><br>
For the synthesis of the compounds provided herein containing an acid labile linker, e.g., a heterobifunctional trityl ether, the requisite phosphoramidite trityl ether is reacted with the oligonucleotide or oligonucleotide analog Q, followed by deprotection of the trityl ether and capture of a biomolecule, e.g., a protein, on the alcohol via a reactive derivative of the alcohol (X), as described above.<br><br>
-129-<br>
(R15)a	(R15)a<br><br>
(R,5)a<br>
In another embodiment, the capture compounds provided herein are prepared by the method illustrated below. Briefly, reaction of cystine with a biotin-linker moiety results in derivatization of the amino functionality. Reaction of the resulting compound with N-hydroxysuccinimide and, e.g., dicyclohexylcarbodiimide (DCC) forms the corresponding di-NHS ester. Reduction of the disulfide bond followed by reaction with a drug-linker moiety forms 2 equivalents of the desired capture compound.<br><br>
-130-<br>
NH2<br>
'biotin<br>
HOOC.<br>
vC00H<br>
NH,<br><br>
HNN<br><br>
^biotin<br><br><br><br>
HOOC.<br><br>
NHS<br><br>
COOH<br>
.jiotm Â»^^-^^~NH<br><br><br>
biotin<br>
biotin<br>
a)	redn   .<br>
b)	v^v^^drug<br><br>
H N v^/V/WV ^\/V\^^ Q j Q( J n<br><br><br>
C-jg^v^S<br><br>
An exemplary photoactivatable capture compound may be prepared, by<br>
the following method:<br>
J.    !L	OH<br>
N<br>
C02H<br>
Â°-   ,0<br>
,OH<br><br>
HN      0<br>
H?N'<br>
S02NH2<br>
S02NH2<br>
S02NH2 1<br>
DIC/NHS/DMF<br><br>
-131-<br><br><br>
NOH<br>
C02Me	C02Me<br>
CF3SiMe      r^^i	Dess-Martin<br>
	Â».<br>
CHOH<br>
periodinane<br>
F,C<br>
F,C<br><br><br>
C02Me<br>
F3C<br><br>
1)LiOH/ H20/MeOH 2) TsCI<br><br><br>
anhyd. NH3<br>
l2 orAg20<br>
C02H<br>
F3C    N<br><br>
C02H<br><br>
F3C<br><br><br>
NOTs<br>
C02H<br><br><br><br><br>
HN<br>
Biotin<br><br>
F3C   N<br>
NHS. DIC<br><br>
O^N    H<br><br>
O O-N       /<br>
o    K<br>
0<br><br>
Lysine/Et3N/ 50 mM NaHC03<br><br>
-132-<br><br><br>
1/BOBt DCC/DMF<br>
S02NH2<br>
Other photoactivatable capture compounds may be prepared as<br><br>
follows:<br>
H NH<br>
V niotin<br>
VOH DIC/I-T3N.M1S     N^j/~'S<br>
CTNH^:    o<br>
)MI. R]. o\L-rnighl stirr<br><br>
0<br>
-0 N'<br>
0<br><br>
NH
o* NH'H<br><br>
X<br><br>
NH<br><br>
0 -,      O<br>
NH,       0<br><br>
NH<br>
O "â¢'"NHH<br><br>
S<br>
V<br><br>
O     ~^NH      "TFA^^C02NHS        HVlCS N-N<br><br>
BiocnjugiUc (hemislr\ Vol?! 689 (1993)<br>
t-BcK.-Asp-4-OB/l-l-<br>
NIIS<br>
DMI<br>
O       NH<br><br>
0-<br>
H NH<br>
"V<br>
O^NHH	O       NH<br>
,-     .    ,0        O' NH^  O<br>
,	P   â ' 0<br>
OBzl<br>
NH<br>
^NH^   ^CT"'<br>
0<br>
OBzl/â"NH^ V<br>
NH t-Boc<br>
0<br>
CF,      ^    Â°<br>
N-'N<br>
H NH<br>
H2N02S<br>
=0<br>
O^NS^     O^NH    1/BOBt/T, /       DCC/DMF<br>
0<br>
/~" "" b    NH<br>
CF3  r^^o<br>
&gt;/~^<br>
N-N<br>
o<br>
\	FN<br>
OH/<br>
-/^NH^V<br>
NH<br>
CF3      r^^O<br>
N:N<br><br>
-133-<br>
The above syntheses are exemplary only. One of skill in the art will be able to modify the above syntheses in a routine manner to synthesize other compounds within the scope of the instant disclosure. Syntheses of capture compounds as provided herein are within the skill of the skilled artisan.<br>
 E.       Methods of Use of the Compounds<br>
The capture compounds provided herein can be used for the analysis, quantification, purification and/or identification of the components of biomolecule mixtures, including, but not limited to, protein mixtures   They can be used to screen libraries of small molecules to identify drug candidates, and they can be used to assess biomolecule-biomolecule interactions and to identify biomolecule complexes and intermediates, such as those in biochemical pathways and other biological intermediates.<br>
To initiate an analytical process, mixtures of biomolecules are obtained or prepared   They can then be pre-purified or partially purified as needed, according to standard procedures. Biomolecules are isolated from samples using standard methods. Figure 20a depicts an exemplary capture assay in which capture compounds are bound to biomolecules and analyzed by MALDI-TOF MS. Example 9 and Figures 20b-f show results of exemplary assays using a variety of capture compounds and known proteins. <br>
1.       General methods<br>
The collections provided herein have a wide variety of applications, including reducing the complexity of mixtures of molecules, particularly biomolecules. by contacting the collection with the mixtures to permit covalent binding of molecules in the mixtures. The capture compounds can be arrayed by virtue of the sorting function either before, during or after the contacting. Following contacting and arraying the loci of the array each contain a subset of the molecules in the mixture. The array can then be analyzed, such as by using mass spectrometry.<br>
For example, proteins are isolated from biological fluids and/or tissues by cell lysis followed, for example, by either precipitation methods (e.g., ammonium sulfate) or enzymatic degradation of the nucleic acids and<br><br>
-139-<br>
effectively interacted with the protein or small molecule mixtures of interest. Analysis using the well known 2-hybrid methodology is also possible<br>
and can be detected via mass spectrometry.  See, e.g., U.S. Patent Nos. 5,512,473, 5,580,721, C,580,736, 5,955,280, 5,695,941. See also, Brent et al. (1996) Nucleic Acids Res. 24(17J:3341 -3347.<br>
In the above embodiments, including those where Z contains a<br>
cleavable linkage, the compounds can contain a mass modifying tag.  In these embodiments, the mass modifying tag is used to analyze the differences in structure (e.g., side chain modification such as phosphoylation or dephosphorylation) and/or expression levels of biomolecules, including<br>
proteins. In one embodiment, two compounds (or two sets of compounds having identical permuted B moieties) are used that only differ in the presence or absence of a mass modifying tag (or have two mass tags with appropriate mass differences). One compound (or one set of compounds) is (are) reacted with "healthy" tissue and the mass modified compound(s) are reacted with the "disease" tissue under otherwise identical conditions. The two reactions are pooled and analyzed in a duplex mode. The mass differences will elucidate those proteins that are altered structurally or expressed in different quantity in the disease tissue. Three or more mass modifying tags can be used in separate reactions and pooled for multiplex, analysis to follow the differences during different stages of disease development (i.e.. mass modifying tag 1 at time point 1. mass modifying tag 2 at time point 2 etc.), or, alternatively, to analyze different tissue sections of a disease tissue such as a tumor sample.<br>
Selectivity in the reaction of the compounds provided herein with a biomolecule, such as a protein mixture also can be achieved by performing the reactions under kinetic control and by withdrawing aliquots at different time intervals. Alternatively, different parallel reactions can be performed (for example, all differing in the B moiety of the Q group) and either performed with different stochiometric ratios or stopped at different time intervals and analyzed separately.<br><br>
-140-<br>
ln embodiments where the capture compounds provided herein possess a luminescent or colorimetric group, the immobilized compound-<br>
biomolecule conjugate can be viewed on the insoluble support prior to analysis. Viewing the conjugate provides information about where the conjugate has hybridized (such as for subsequent MALDI-TOF mass spectrometry analysis). In certain embodiments, with selected reagents the quantity of a given protein from separate experiments (e.g.. healthy vs. disease, time point 1 vs. time point 2, etc.) can be determined by using dyes that can be spectrophotometrically differentiated.<br>
A biomolecule modified with this reagent is then labelled with an<br>
oligonucleotide possessing a diene residue. It is appreciated by those of skill<br>
In other embodiments, the methods are performed by tagging the biomolecules to be analyzed, including but not limited to proteins, with more than one, in one embodiment three to five, of the compounds provided herein. Such compounds possess functionality designed to target smaller chemical features of the biomolecules rather than a macromolecular feature. See, e.g., Figure 3. Such smaller chemical features include, but are not limted to, NH2, SH, SS (after capping SH, SS can be targeted by, e.g., gold), and OH. In one non-limiting example, the phenolic OH of tyrosine is selectively captured using a diazo compound, such as an aryldiazonium salt. In this embodiment, the reaction can be performed in water. For example, a functionalized diazonium salt could be used where the functionality allows for subsequent capture of a compound provided herein, thereby providing a oligonucleotide-labelled biomolecule. One such functionalized diazonium salt is:<br><br><br>
-141-<br>
in the art that many reagent couples other that dienophile/diene can be used in these embodiments. In the case of dienophile/diene, the reaction of the dienophile with the diene can be performed in the presence of many other functional groups, including N-hydroxysuccinimido-activated oligonucleotides reacting with an NH2 group. Thus, these two labelling specific reactions can be performed in one reaction. See, e.g., Figure 5.<br>
Subsequently, the multiply-tagged biomolecules are hybridized on an array of antisense oligonucleotides, in one embodiment a chip containing an array of antisense oligonucleotides. Such multiply-tagged biomolecules can be sorted with greater selectivity than singly tagged biomolecules. See, e.g., Figure 4.<br>
In embodiments where the compounds for use in the methods provided herein are insoluble or poorly soluble in water or aqueous buffers, organic solvents are added to the buffers to improve solubility. In one embodiment, the ratio of bufferorganic solvent is such that denaturation of the biomolecule does not occur. In another embodiment, the organic solvents used include, but are not limited to, acetonitrile, formamide and pyridine. In another embodiment, the ratio of bufferorganic solvent is about 4:1. To determine if an organic co-solvent is needed, the rate of reaction of the compounds provided herein with a water-souble amine, such as 5'-aminothymidine, is measured. For example, the following reaction is performed is a variety of solvent mixtures well known to those of skill in the art to determine optimal conditions for subsequent biomolecule tagging and analysis:<br><br>
-142-<br><br><br>
H?N<br><br>
OH<br>
2.       Phenotype analyses<br>
The collections of capture permit a top down holistic approach to analysis of the proteome and other biomolecules. As noted, the collections and methods of use provide an unbiased way to analyze biomolecules, since the methods do not necessarily assess specific classes of targets, but rather detect or identify changes in the samples.' The changes identified include structural changes that are related to the primary sequences and modifications, including post-translational modifications. In addition, since the capture compounds can include a solubility function they can be designed for reaction in hydrophobic conditions, thereby permitting analysis of membrane-bound and membrane-associated molecules, particularly proteins.<br><br>
-143-<br>
Problems with proteome analysis arise from genetic variation that is not related to a target phenotype, proteome variation due to differences, such as gender, age, metabolic state, the complex mixtures of cells in target tissues and variations from cell cycle stage. Thus, to identify or detect changes, such as disease-related changes, among the biomolecule components of tissues and cells, homogeneity of the sample can be important. To provide homogeneity, cells, with different phenotypes, such as diseased versus healthy, from the same individual are compared.   As a result, differences in patterns of biomolecules can be attributed to the differences in the phenotype rather than from differences among individuals. Hence, samples can be obtained from a single individual and cells with<br>
dif erent phenotypes, such as healthy versus diseased and responders<br>
versus non-responders, are separated. In addition, the cells can be synchronized or frozen into a metabolic state to further reduce background differences.<br>
Thus, the collections of capture compounds can be used to identify<br>
phenotype-specific proteins or modifications thereof or other phenotype-specific biomolecules and patterns thereof. This can be achieved by comparing biomolecule samples from cells or tissues with one phenotype to the equivalent cells to biomolecule samples from cells or tissues with another phenotype. Phenotypes in cells from the same individual and cell type are compared. In particular, primary cells, primary cell culture and/or<br>
synchronized cells are compared.   The patterns of binding of biomolecules from the cells to capture compound members of the collection can be identified and used as a signature or profile of a disease or healthy state or other phenotypes. The particular bound biomolecule, such as a protein, also can be identified and new disease-associated markers, such as particular proteins or structures thereof, can be identified.   Example 6 provides an exemplary embodiment in which cells are separated. See also Figure 19. Phenotypes for comparison include, but are not limited to: 1) samples from diseased versus healthy cells or tissues to identify<br><br>
-144-<br>
proteins or other biomolecules associated with disease or that are markers for disease;<br>
2)	samples from drug responders and non responders (i.e. on 20-30%<br>
of malignant melanoma patients respond to alpha interferon and others to do<br>
not) to identify biomolecules indicative of response;<br>
3)	samples from cells or tissues with a toxicity profile to drugs or environmental conditions to identify biomolecules associated with the response or a marker of the response; and<br>
4)	samples from cells or tissues exposed to any condition or exhibiting any phenotype in order to identify biomolecules, such as proteins, associated with the response or phenotype or that are a marker therefor.<br>
Generally the samples for each phenotype are obtained from the same organism, such as from the same mammal so that the cells are essentially matched and any variation should reflect variation due to the phenotype and not the source of the cells. Samples can be obtained from primary cells (or tissues).   In all instances, the samples can be obtained from the same individual either before exposure or treatment or from healthy non-diseased tissue in order to permit identification of phenotype-associated biomolecules.<br>
Cells can be separated by any suitable method that permits identification of a particular phenotype and then separation of the cells based thereon. Any separation method, such as, for example, panning or negative panning (where unwanted cells are captured and the wanted cells remain in the supernatant) where the live cells are recovered can be used. These methods include, but are not limited to:<br>
1)	flow cytometry;<br>
2)	specific capture;<br>
3)	negative panning in which unwanted cells are captured and the targeted cells remain in the supernatant and live cells are recovered for analysis; and<br>
4)	Laser Capture Microdissection (LCM) (Arcturus, Inc Mountain View, CA).<br><br>
-145-<br>
Thus sorting criteria include, but are not limited to, membrane potential, ion flux, enzymatic activity, cell surface markers, disease markers, and other such criteria that permit separation of cells from an individual based on phenotype.<br>
a)        Exemplary separation methods<br>
1)	Laser Capture Microdissection<br>
Laser Capture Microdissection (LCM) (Arcturus, Inc Mountain View, CA) uses a microscope platform combined with a low-energy IR laser to activate a plastic capture film onto selected cells of interest. The cells are then gently lifted from the surrounding tissue. This approach precludes any absorption of laser radiation by microdissected cells or surrounding tissue, thus ensuring the integrity of RNA, DNA, and protein prepared from the microdissected samples for downstream analysis.<br>
2)	Flow cytometry for separation<br>
Flow cytometry is a method, somewhat analogous to fluorescent microscopy, in which measurements are performed on particles (cells) in liquid suspension, which flow one at a time through a focused laser beam at rates up to several thousand particles per second. Light scattered and fluorescence emitted by the particles (cells) is collected, filtered, digitized and sent to a computer for analysis. Typically flow cytometry measures the    , binding of a fluorochrome-labeled probe to cells and the comparison of the resultant fluorescence to the background fluorescence of unstained cells. Cells can be separated using a version of flow cytometry, flow sorting, in which the particles (cells) are separated and recovered from suspension based upon properties measured in flow. Cells that are recovered via flow sorting are viable and can be collected under sterile conditions. Typically, recovered subpopulation; that are in excess of 99.5% pure (see Figures 19a and 19b).<br>
Flow cytometry allows cells to be distinguished using various parameters, including physical and/or chemical characteristics associated<br>
with cells or properties of cell-associated reagents or probes, any of which<br><br>
-146-<br>
are measured by instrument sensors. Separation: Live v. Dead Forward and side scatter are used for preliminary identification and gating of cell populations. Scatter parameters are used to exclude debris, dead cells, and unwanted aggregates. In a peripheral blood or bone marrow sample, lymphocyte, monocyte and granulocyte populations can be defined, and separately gated and analyzed, on the basis of forward and side scatter.<br>
Cells that are recovered via flow sorting are viable and can be collected under sterile conditions. Typically recovered subpopulations are in excess of -99.5% pure.<br>
Common cell sorting experiments usually involve immunofluorescence assays, i.e., staining of cells with antibodies conjugated to fluorescent dyes in order to detect antigens. In addition, sorting can be performed using GFP-reporter constructs in order to isolate pure populations of cells expressing a given gene/construct.<br>
a.	Fluorescence<br>
Fluorescent parameter measurement permits investigation of cell structures and functions based upon direct staining, reactions with fluorochrome labeled probes (eg , antibodies), or expression of fluorescent proteins. Fluorescence signals can be measured as single or multiple parameters corresponding to different laser excitation and fluorescence emission wavelengths. When different fluorochromes are used simultaneously, signal spillover can occur between fluorescence channels. This is corrected through compensation. Certain combinations of fluorochromes cannot be used simultaneously; those of skill in the art can identify such combinations.<br>
b.	Immunofluorescence<br>
Immunofluorescence involves the staining of cells with antibodies<br>
conjugated to fluorescent dyes such as FITC (fluorescein), PE (phycoerythrin), APC (;allophycocyanin), and PE-based tandem   conjugates (R670, CyChrome and others). Cell surface antigens are the usual targets of this assay, but antibodes can be directed at antigens or cytokines in the<br><br>
â¢147-<br>
cytoplasm as well.<br>
DNA staining is used primarily for cell cycle profiling, or as one method<br>
for measuring   apoptosis. Propidium iodide (PJ), the most commonly used DNA stain, cannot enter live cells and can therefore be used for viability assays. For cell cycle or apoptosis assays using PI, cells must first be fixed in order for staining to take place (see protocol). The relative quantity of PI-DNA staining corresponds to the proportion of cells in G0/G1, S, and G2/M phases, with lesser amounts of staining indicating apoptotic/necrotic cells. PI staining can be performed simultaneously with certain fluorochromes, such as FITC and GFP. in assays to further characterize apoptosis or gene expression.<br>
Gene Expression and Transfection can be measured indirectly by using a reporter gene in the construct. Green Fluorescent Protein-type constructs (EGFP, red and blue fluorescent proteins) and ft-galactosidase, for example, can be used to quantify populations of those cells expressing the gene/construct. Mutants of GFP are now available that can be excited at common frequencies, but emit fluorescence at different wavelengths   This allows for measurement of co-transfection, as well as simultaneous detection of gene and antibody expression. Appropriate negative (background) controls for experiments involving GFP-type constructs should be included. Controls include, for example, the same cell type, using the gene insert minus the . GFP-type construct.<br>
3)       Metabolic Studies and other studies<br>
Annexin-V can be labeled with various fluorochromes in order to identify cells in early stages of apoptosis. CFSE binds to cell membranes and is equally distributed when cells divide. The number of divisions cells undergo in a period of time can then be counted. CFSE can be used in conjunction with certain fluorochromes for immunofluorescence. Calcium flux can be measured using lndo-1 markers. This can be combined with immunofluorescent staining.   Intercellular conjugation assays can be performed using combinations of dyes such as calcein or hydroethidine.<br><br>
-148-<br>
b)      Synchronizing cell cycles<br>
Once sorted or separated cells are obtained they can be cultured, and,<br>
can be synchronized or frozen into a particular metabolic state. This enhances the ability to identify phenotype-specific biomolecules. Such cells can be separated by the above methods, including by flow cytometry. Further, cells in the same cell cycle, same metabolic state or other synchronized state can be separated into groups using flow cytometry (see, Figure 19c).<br>
Cell cycles can be synchronized or frozen by a variety of methods, including but are not limited to, cell chelation of critical ions, such as by removal of magnesium, zinc, manganese, cobal and/or other ions that perform specific functions by EDTA or otherchelators (see, e.g., EXAMPLES). Other methods include controlling various metabolic or biochemical pathways   Figure 18 depicts exemplary points of regulation of metabolic control mechanisms for cell synchronization. Examples of synchronizing or "freezing" Metabolic Control for synchronizing cells, include, but are not limited to. the following:<br>
1)	control of gene expression;<br>
2)	regulation of enzyme reactions;<br>
3)	negative control: Feedback inhibition or End product repression and enzyme induction are mechanisms of negative control that lead to a decrease in die transcription of proteins;<br>
4)	positive control: catabolite repression is considered a form of positive control because it affects an increase in transcription of proteins.<br>
5)	Control of individual proteins translation:<br>
a)	oligonucleotides that hybridize to the 5' cap site have<br>
inhibit protein synthesis by inhibiting the initial interaction between the mRNA<br>
and the ribosome 40S sub-unit;<br>
b)	oligonucleotides that hybridize to the 5' UTR up to, and<br>
including, the translation initiation codon inhibit the scanning of the 40S (or<br>
30S) subunit or assembly of the full ribosome (80S for eukaryotes or 70S for<br><br>
-149-<br>
bacterial systems):<br>
5)	control of post translational modification:<br>
6)	control of allosteric enzymes, where the active site binds to the substrate of the enzyme and converts it to a product. The allosteric site is occupied by some small molecule that is not a substrate. If the protein is an enzyme, when the allosteric site is occupied, the enzyme is inactive, i.e., the effector molecule decreases the activity of the enzyme. Some<br>
multicomponent allosteric enzymes have several sites occupied by various effector molecules that modulate enzyme activity over a range of conditions. <br>
3.        Analysis of low abundancy proteins Important disease-associated markers and targets could be low abundancy proteins, that might not be detected by mass spectrometry. To ensure detection, a first capture compound display experiment can be performed   The resulting array of captured proteins is reacted with a non-selective dye such as a fluorescent dye, that will light up or render visible more proteins on the array. The dye can provide a semi-quantitative estimate of the amount of a protein. The number of different proteins detected by the dye can be determined and then compared the number detected by mass spectrometric analysis. If there are more proteins detected using the dye, the experiments can be repeated using a higher starting number of cells so that low abundance proteins can be detected and identified by the mass spectrometric analysis.<br>
For example, housekeeping proteins, such as actin and other such proteins, are present in high abundance and can mask low abundancy proteins. Capture compounds or other purification compound selected or designed to capture or remove the high abundancy proteins or biomolecules from a mixture before using a collection to assess the components of the mixture. Once the high abundancy proteins are removed, low abundancy proteins have an effectively higher concentration and can be detected. These methods, thus, have two steps: a first step to capture high abundancy components of biomolecule mixtures, such as the actins. For example, a cell<br><br>
-150-<br>
lysate can be contacted with capture molecules that include a reactivity group such as biotin or other general reactivity function linked to a sorting group to remove such high abundancy proteins, and then use a suitable collection of capture compounds to identify lower abundancy compounds remaining in the<br>
lysate.<br>
Also, as discussed above, capture compounds can be designed, such as' by appropriate selection of W, to interact with intact organelles before disrupting them in cells that have been gently lysed or otherwise treated to . permit access to organelles and internal membranes. Then the captured organelles can be disrupted, such as one which can inlcude an artificial membrane, such as a lipid bilayer or micelle coating, to capture the organelle proteins and other biomolecules in an environment that retains their three-dimensional structure.   These captured proteins can be analyzed.   This permits the capture compounds to interact with the captured proteins and other biomolecules in their native tertiary structure.<br>
4.	Monitoring protein conformation as an indicator of disease<br>
 The collections and/or members thereof can be used to detect or distinguish specific conformers of proteins. Hence, for example, if a particular conformation of a protein is associated with a disease (or healthy state) the collections or members thereof can detect one conformer or distinguish  . conformers based upon a pattern of binding to the capture compounds in a collection. Thus, the collections and/or members thereof can be used to detect conformationally altered protein diseases (or diseases of protein aggregation), where a disease-associated protein or polypeptide has a<br>
disease-associated conformation. The methods and collections provided<br>
herein permit detection of a conformer associated with a disease to be detected. These diseases include, but are not limited to, amyloid diseases and neurodegenerative diseases. Other diseases and associated proteins that exhibit two or more different conformations in which at least one conformation is associated with disease include those set forth in the following Table:<br><br>
-151-<br><br>
Disease	Insoluble protein               J<br>
Alzheimer's Disease (AD)	APP, Aa, a1-antichymotrypsin, tau, non-Ag component, presenellin 1, presenellin 2, apoE<br>
Prion diseases, including but are not limited to, Creutzfeldt-Jakob disease, scrapie, bovine spongiform encephalopathy	prpSc<br>
amyotrophic lateral sclerosis (ALS)	superoxide dismutase (SOD) and neurofilament<br>
Pick's Disease	Pick body<br>
Parkinson's disease	a-synuclein in Lewy bodies<br>
Frontotemporal dementia	tau in fibrils<br>
Diabetes Type II	amylin<br>
Multiple myeloma	IgGL-chain<br>
Plasma cell dyscrasias	<br>
Familial amyloidotic polynueuropathy	Transthyretin<br>
Medullary carcinoma of thyroid	Procalcitonin<br>
Chronic renal failure	a2-microgobulin<br>
Congestive heart failure	Atrial natriuretic factor<br>
Senile Cardiac and systemic amyloidosis	transthyretin<br>
Chronic inflammation	Serum Amyloid A<br>
Atherosclerosis	ApoAl<br>
Familial amyloidosis	Gelsolin<br>
Huntington's disease	Huntington<br>
The collections can be contacted with a mixture of the conformers and the members that bind or retain each form can be identified, and a pattern thus associated with each conformer. Alternatively, those that bind to only one conformer, such as the conformer associated with disease can be identified, and sub-collections of ore or more of such capture compounds can be used as a diagnostic reagent for the disease.<br><br>
-152-<br>
5.	Small molecule identification and biomolecule-biomolecule<br>
interaction investigation<br>
Biomolecules, such as proteins, are sorted using a covalent or noncovalent interaction with immobilized capture compounds. Collections, such as arrays of capture compounds bound to biomolecules, such as from cellysates, then can be used to screen libraries or other mixtures of drug<br>
candidates or to further screen mixtures of biomolecules to see what binds to the bound biomolecules. The capture biomolecule-biomolecule complexes or biomolecule-drug candidate complexes can be analyzed to identify biochemical pathways and also to identify targets with the candidate drug. For example, protein-protein or protein-biomolecule interactions are exposed to test compounds, typically small molecules, including small organic molecules, peptides, peptide mimetics, antisense molecules or dsRNA, antibodies, fragments of antibodies, recombinant and sythetic antibodies and fragments thereof and other such compounds that can serve as drug candidates or lead compounds. Bound small molecules are identified by mass spectrometry or other analytical methods.<br>
6.	identification of non-target biomolecules<br>
Many pharmaceutical drugs have side effects that may arise from the interaction of the drugs, drug fragments, drug metabolites or prodrugs with, drug non-target biomolecules under physiological conditions.<br>
For example, aspirin reacts with the non-target Cox-1 receptor resulting in side effects such as gastrointestinal toxicity, ulceration, bleeding, perforation of the stomach, liver necrosis, hepatic failure, renal necrosis and possibly stroke and heart attack. Selective Cox-2 inhibitors such as Cox-2 inhibitors such as 4-[5-(4-methylphenyl)-3- (trifluoromethyl)-1H-pyrazol-1-yl] benzenesulfonamide, (Celebrex^) or 4-(4-(methylsulfonyl)phenyl)-3-phenyl-2(tiH)-furanone (VIOXXÂ®) have side effects that may be the result of interaction of the drug with non-target biomolecules. As another example, the thaizolidinedione (TDZ) class of antidiabetic drugs are PPAR-y activators. The<br>
PPAR-y protein is a receptor important in the regulation of genes involved in<br><br>
-153-<br>
the metabolism of glucose and lipids. TDZs are prescribed to diabetic patients in whom blood sugar (glucose) is not properly metabolized. However, TDZ's<br>
are known to also interact with PPAR-a, a protein with a similar structure involved in the synthetic pathway of triglycerides, known to be associated with cardiovascular disease. The TDZ Rezulin was withdrawn from the market due to liver toxicity, and Actos and Avandia were recently reported in a Mayo Clinic study to have cardiovascular side effects.<br>
Drug metabolites can also cause toxicity. There are several enzymatic systems responsible for drug metabolism. One such important system is the Cytochrome P450 family, primarily located in the liver. These proteins work by attaching functional groups to the (usually lipophilic) drug molecules. These functional groups subsequently allow other enzymes to conjugate moieties (glucuronidation, sulfation, etc.) to the metabolites rendering them water-soluble and thus facilitating excretion. Toxicity can occur if a polymorphic form of an enzyme involved in the metabolism malfunction, or a metabolite irreversibly inactivates a cytochrome p450 (suicide inhibition), compromising its excretion potentially leading to a toxic accumulation in the liver. Depending on the presence of these metabolizing enzyme systems in a..g. kidneys, lung, or heart, similar drug toxicities can be observed in those organs.<br>
The capture compounds/collections thereof provided herein, can be used to identify the drug non-target biomolecules that interact with the pharmaceutical drugs/drug fragments, drug metabolites or prodrugs including but not limited to, receptors and enzymes. The identification and characterization of the drug interacting proteins can also lead to unexpected alternative pharmacological benefits. It is not unlikely that drug targets in other unexpected biological pathways would be found, which allow the application of the drug to treat other diseases. A failed drug that might not be efficacious (or too toxic) for one disease could be turned into a blockbuster for<br>
another disease.<br>
In one embodiment, the capture compounds/collections thereof are designed to contain pharmaceutical drugs/drug fragments, drug metabolites<br><br>
-154-<br>
or prodrugs as the selectivity function and suitable reactivity and sorting functionality. In the methods provided herein, the capture<br>
compound/collections thereof are allowed to interact with a mixture of drug target and non-target biomolecules, including but not limited to, receptor proteins. The captured biomolecules are then analyzed to identify drug target and non-target biomolecules. Screening and identification of drug non-target biomolecules can help in understanding side effects of the pharmaceutical drugs and permit modification of the drug structure to eliminate or minimize the side effects while maintaining the efficacy. Exemplary drug molecules that can be used in the methods and collections provided herein are set forth elsewhere herein, and include, but are not limited to,.LIPITORÂ® (atorvastatin calcium). CELEBREXÂ® (celecoxib). VIOXXÂ® (refecoxib) and BAYCOLÂ® (cerivastatin sodium).<br>
Once a protein is identified to interact with the drug, public databases annotating the function of many proteins are queried to determine if that structure is likely related to the observed side effect or therapeutic response. For cases where the function of a protein is unknown, bioinformatics and functional genomic tools are available. These include in silico approaches (bioinformatics) including sequence alignment, pharmacophores, homology models and protein motif correlation; in vitro approaches including liver midrosomes metabolic pathways (e.g. P450), cDNA-expressed enzymes, signal pathways and back-mapping to yeast pathways, simulations and protein/protein interaction of pull-out proteins, in vivo approaches including native polymorphisms, knock-out/knock-in, flow cytometry, therapeutic activity of the drug (i.e. therapeutic profile and experimental toxicity, and prospective genotyping and prospective phenotyping. Using these in conjunction with cell-based assays and ribozyme-based knock-in / knock-out technology, which of the proteins identified above are associated with the therapeutic or toxic effect can be determined.<br>
7.       Drug Re-engineering<br>
An important goal of most drug development projects is to maximize<br><br>
-155-<br>
the interaction between a drug and its target leading to positive therapeutic results, while minimizing interactions with other proteins. Interactions with<br>
proteins other than the intended target can trigger a cascade of cellular events leading to side effects. Provided herein are methods that enable design of drugs which interact with their intended target while minimizing other interactions. Here, the selectivity function of the capture compound is a drug molecule or one of its metabolites, attached in different chemically relevant orientations. Following the procedures described above, the proteins (target and non-targets) that interact with the drug and their respective putative function are identified, screening against all cell types potentially involved in the therapeutic or side-effect-related pathways. Knowledge of the therapeutic effect of the drug, as well as its side effects as previously observed in patients, facilitate the formation of a hypothesis as to which of the captured proteins lead to the desired therapeutic effect, and which are involved in its side effects.<br>
Using these methods, one can iteratively optimize, or re-engineer, the chemical structure of the drug, maintaining or enhancing the desired target protein interactions and eliminating structural features leading to the non-target interactions. Since this process can take place even before preclinical trials, significant cost and time savings can be achieved. The result is a different and patentable new chemical entity (NICE), which can be reintroduced into clinical trials. A reduction of clinical trial time can be envisaged since efficacy data from the related parent drug molecule is already available, and the NCE has been structurally optimized for reduced side effects prior to entering the clinical trail process. An increased success rate of clinical trials would have a tremendous effect on reducing the time and especially the cost of drug development.<br>
Using these methods, analysis is performed to identify the sets of all proteins interacting with the drug, and downstream cellular (functional) assays are used to validate which protein interactions are most likely responsible for the side effects. The drug compounds are redesigned considering data from<br><br>
-156-<br>
all the drugs tested in the disease area to maintain the interaction with the protein leading to the positive therapeutic effect while minimizing other protein<br>
interactions<br>
Exemplary diseases that may be studies using these methods include:<br>
(1)	Diabetes. Diabetes and its major risk factor obesity will be a growing health crisis facing the western population in the coming decade. Rezulin (Troglitazone) has been withdrawn from the market, MK-767 was recently withdrawn from Phase III trials, and sales of other drugs (e.g "Actos, Avandia) have been hampered, all due to side effects.<br>
(2)	Cardiovascular. Nearly one million Americans die each year from cardiovascular diseases, many from heart attacks and strokes due to blocked arteries caused by elevated levels of cholesterol in the bloodstream. However the prescription rate of the statins, including Lipitor, is affected by side effects: patients taking these drugs must be monitored by their physician frequently to ascertain that toxic effects such as liver damage are not taking place.<br>
(3)	Arthritis / Pain / Inflammation. Reports of gastrointestinal and in some cases coronary side effects have limited sales of the anti-inflammatory COX-2 inhibitors Vioxx and Celebrex, as many doctors recommend that their patients take safer but far less effective drugs such as ibuprofen to ease inflammation symptoms.<br>
F.       Systems<br>
In further embodiments, the compounds and the methods described herein are designed to be placed into an integrated system that standardizes and automates the following process steps:<br>
Isolation of biomolecules from a biological source, including<br>
isolation of the proteins from cell lysates (lysis, enzymatic<br>
digestion, precipitation, washing)<br>
Optionally, removal of low molecular weight materials<br>
Optionally, aliquoting the biomolecule mixture, such as a protein<br>
mixture<br><br>
-157-<br>
Reaction of the biomolecule mixture, such as a protein mixture, with compounds of different chemical reactivity (X) and<br>
sequence diversity (B) provided herein; this step can be<br>
performed in parallel using aliquots of the biomolecule mixture<br>
Optionally, removal of excess compound<br>
Hybridization of the compound-biomolecule conjugate, such as<br>
a compound-protein conjugate, to single stranded<br>
oligonucleotides or oligonucleotide analogs that are<br>
complementary to the Q moiety of the compound; the single<br>
stranded oligonucleotides or oligonucleotide analogs are<br>
optionally presented in an array format and are optionally<br>
immobilized on an insoluble support<br>
Optionally, subsequent chemical or enzymatic treatment of the<br>
protein array<br>
Analysis of the biomolecule array, including, but not limited to,<br>
the steps of (i) deposition of matrix, and (ii) spot-by-spot MALDI-<br>
TOF mass spectrometry using an array mass spectrometer (with<br>
or without internal, e.g., on-chip molecular weight standard for<br>
calibration and quantitation).<br>
In another embodiment, the compounds and the methods described herein are designed to be placed into an integrated system that standardizes and automates the following process steps:<br>
Isolation of biomolecules from a biological source, including<br>
isolation of the proteins from cell lysates (lysis, enzymatic<br>
digestion, precipitation, washing)<br>
Optionally, removal of low molecular weight materials<br>
Optionally, aliquoting the biomolecule mixture, such as a protein<br>
mixture<br>
Reaction of the biomolecule mixture, such as a protein mixture,<br>
with compounds of different chemical reactivity (X) and<br>
sequence diversity (B) provided herein; this step can be<br><br>
-158-<br>
performed in parallel using aliquots of the biomolecule mixture Optionally, removal of excess compound<br>
Chemical or enzymatic treatment of the protein array Subsequent hybridization of the compound-biomolecule<br>
conjugate, such as a compound-protein conjugate, to single stranded oligonucleotides or oligonucleotide analogs that are complementary to the Q moiety of the compound; the single stranded oligonucleotides or oligonucleotide analogs are optionally presented in an array format and are optionally immobilized on an insoluble support<br>
Analysis of the biomolecule array, including, but not limited to,<br>
the steps of (i) deposition of matrix, and (ii) spot-by-spot MALDI-TOF mass spectrometry using an array mass spectrometer (with<br>
or without internal, e.g., on-chip molecular weight standard for calibration and quantitation).<br>
The systems include the collections provided herein, optionally arrays of such collections, software for control of the processes of sample preparation and instrumental analyis and for analysis of the resulting data, and instrumentation, such as a mass spectrometer, for analysis of the biolmolecules. The systems include other devices, such as a liquid chromatographic devices so that a protein mixture is at least partially separated. The eluate is collected in a continuous series of aliquots into, e.g., microtiter plates, and each aliquot reacted with a capture compound provided.<br>
In multiplex reactions, aliquots in each well can simultaneously react with one or more of the capture compounds provided herein that, for example each differ in X (i.e., amino, thiol, lectin specific functionality) with each having a specific and differentiating selectivity moiety Y and in the Q group. Chromatography can be done in aqueous or in c rganic medium. The resulting reaction mixtures are pooled and analyzed directly. Alternatively, subsequent secondary reactions or molecular interaction studies are performed prior to analysis, including mass spectrometric analysis.<br><br>
-159-<br>
The systems provided herein can contain an assembly line, such as pipetting robots on xy stages and reagent supply/washing modules that are<br>
linKed with a central separation device and a terminal mass spectrometer for<br>
analysis and d0 .a interpretation. The systems can be programmed to perform process steps including (see, e.g., FIG. 2), for example:<br>
1)	Cell cultures (or tissue samples) are provided in microtiter plates (MTPs) with 1, 2...i wells. To each well, solutions are added for lysis of cells, thereby liberating the proteins. In some embodiments, appropriate washing steps are included, as well as addition of enzymes to digest nucleic acids and other nonprotein components. In further embodiments, instead of regular MTPs, MTPs with filter plates in the bottom of wells are used. Cell debris is removed either by filtration or centrifugation. A conditioning solution for the appropriate separation process is added and the material from each well separately loaded onto the separation device.<br>
2)	Separation utilizes different separation principles such as charge, molecular sizing, adsorption, ion-exchange, and molecular exclusion principles. Depending on the sample size, suitable appropriate dimensions are utilized, such as microbore high performance liquid chromatography (HPLC). In certain embodiments, a continuous flow process is used and the effluent is continuously aliquotted into MTP 1,2...n.<br>
3)	Reaction with Proteome Reagents. Each MTP in turn is transferred to a Proteome Reagent Station harboring 1, 2... m reagents differing only in the oligonucleotide sequence part {i.e., Q) or/and in the chemical nature of the functionality reacting with the proteins {i.e., X). If there are more than one MTP coming from one tissue sample then reagent 1 is added to the same well of the respective MTPs 1. 2...n, i.e., in well A1, reagent 2 in well A2, etc. In embodiments where the MTPs have<br><br>
-160-<br>
96 wells (i = 1-96), S6 different Proteome Reagents (i.e., 96 different compounds provided herien, m = 1-96) are supplied through 96 different nozzles from the Proteome Reagent Station to prevent cross-contamination.<br>
Pooling: Excess Proteome Reagent is deactivated, aliquots from each well belonging to one and the same tissue samples are pooled, and the remaining material is stored at conditions that preserve the structure (and if necessary conformation) of the proteins intact, thereby serving as master MTPs for subsequent experiments.<br>
Excess Proteome Reagent is removed in the pooled sample using, e.g., the biotin/streptavidin system with magnetic beads, then the supernatant is concentrated and conditioned for hybridization.<br>
Transfer to an Oligonucleotide Chip. After a washing step to remove non-hybridized and other low molecular weight material, a matrix is added. Alternatively, before matrix addition, a digestion with, e.g., trypsin or/and chymotrypsin is performed. After washing out the enzyme and the digestion products, the matrix is added.<br>
Transfer of chip to mass spectrometer. In one embodiment, MALDI-TOF mass spectrometry is performed. Other mass spectrometric configurations suitable for protein analysis also can be applied. The mass spectrometer has an xy stage and thereby rasters over each position on the spot for analysis. The Proteome Reagent can be designed so that most of the reagent part (including the part hybridizing with the oligonucleotide chip array) is cleaved either before or during mass spectrometry and therefore will be detected in the low molecular weight area of the spectrum and will be well separated from the peptide (in<br>
case of enzymatic digestion) or protein molecular weight signals<br><br>
-161-<br>
in the mass spectrum. 8)       Finally, the molecular weight signals can be processed for noise reduction, background subtraction and other such processing steps. The data obtained can be archived and interpreted. The molecular weight values of the proteins (or the peptides obtained after enzymatic digestion) are associated with the human DNA sequence information and the derived protein sequence information from the protein coding regions. An interaction with available databases will reveal whether the proteins and their functions are already known.  If the function is unknown, the protein can be expressed from the known DNA sequence in sufficient scale using standard methods to elucidate its function and subsequent location in a biochemical pathway, where it plays its metabolic role in a healthy individual or in the disease pathway for an individual with disease.<br>
Since the master plates containing aliquots from the different proteins<br>
within a given tissue sample have been stored and are available, subsequent experiments then can be performed in a now-preselected way, e.g., the proteins are displayed on the chip surface for protein-protein (biomolecule) interaction studies for target validation or/and to study the interaction with , combinatorial libraries of small molecules for drug candidate selection. G.       Bioinformatics<br>
The raw data generated from the analysis, such as mass spectrometry analysis, of the compound-protein species is processed by background subtraction, noise reduction, molecular weight calibration and peak refinement (e.g., peak integration). The molecular weight values of the cleaved proteins or the digestion products are interpreted and compared with existing protein databases to determine whether the protein in question is known, and if so, what modifications are present (glycosylated or not glycosylated, phosphorylated or not phosphorylated, etc.). The different sets<br>
of experiments belonging to one set of compounds are composed, compared<br><br>
-162-<br>
and interpreted. For example, one set of experiments uses a set of compounds with one X moiety and different Q moieties. This set of<br>
experiments provides data for a portion of the proteome, since not all proteins<br>
in the proteome will react with a given X moiety. Superposition of the data from this set of experiments with data from other sets of experiments with different X moieties provides data for the complete proteome.<br>
Sets of experiments comparing tissues of healthy and disease individuals or from different physiological or developmental stages (e.g.,<br>
tumor progression, dependence of drug treatments to monitor results of<br>
therapy, immune response to virus or bacteria infection) or different tissue areas (e.g., of a tumor) are investigated, and the final data archived.<br>
The following examples are included for illustrative purposes only and are not intended to limit the scope of the invention.<br>
Commercial grade solvents and reagents were used without purification unless otherwise specified, and were purchased from the following vendors: Anhydrous THF (Aldrich), CH2CI2 (Aldrich, Acros, EM Science).  CHCI3 (Aldrich, Mallinckrodt),  Hexanes (Acros, EM science), Ethyl acetate (Alrich, Acros), Acetone (Aldrich. EM science), Methyl alcohol (Aldrich). Diethyl ether (Fisher scientific). 4-Bromobenzoic acid (Aldrich), 2-ammo-2-methyl-1-propanol (Acros), 1,3-dicyclocarbodiimide (Aldrich), N-hydroxysuccinimide (Aldrich), Maleimide (Aldrich). 1-(3-dimethylaminopropyl)-3-ethylcarbcdiimide hydrochloride (Acros). Thionyl chloride (Aldrich), Pyridine (Aldrich). Magnesium turnings (Acros), 4-(Diphenylhydroxymethyl)benzoic acid (Fluka), Sodium ethoxide (Acros), Potassium carbonate, Sodium iodide, Carbon tetrachloride, methyl iodide, RED-AI (Aldrich), anhydrous Na2S04 (Acros), Acetic acid (EM science), Sodium hydroxide (Acros), Molecular sieves AÂ°4 (Aldrich), and Acetyl chloride (Aldrich). 1H NMR spectral data were obtained from a 500 MHz NMR spectrophotometer using CDCI3 as a solvent. Mass spectral data were analyzed using the electrospray method.<br><br>
-163-<br>
EXAMPLE1<br>
Examples for N1mBjN2n<br>
a.        N1 and N2 as identical tetramers, B as a trimer<br>
N1 = N2. m = n = 4, i = 3, B = 64 sequence permutations<br>
GTGC ATG GTGC AAG ACG AGG TTG CTG GTG<br>
GGG b.       N1 and N2 as non-identical tetramers, B as a tetramer<br>
N' + NT. m = n = 4, i = 4, B = 256 sequence permutations<br>
GTCC ATCG CTAC<br>
AACG<br>
ACCG<br>
AGCG<br>
GGGG c.       N1 as a heptamer, N2 as an octamer, B as an octamer<br>
N1 + N2. m = 7, n = 8, i = 8, B = 65,536 sequence permutations.<br>
GCTGCCC      ATTCGTAC      GCCTGCCC<br>
N1	B	N2<br><br>
-164-<br>
EXAMPLE 2<br>
Separation of proteins on a DNA array<br>
N1mBiN2n(S1),M(R15)a(S2)bLXProtein where B is a trimer;<br>
m = n = 4, i = 3, t = b = 1; underlined sequences are N1 and N2<br>
/	CTGC ATG GTGC - S1 - M(R15)a - S2 - L - X- Protein 1<br>
/      " âCACG TAC CACG<br>
/<br>
1	CTGC AAG GTGC - ST - M(R15)a - S2 - L - X- Protein 2<br>
't	âCACG TTC CACG<br>
/<br>
/	CTGC ACG GTGC - S-, - M(R15)a - S2 - L - X- Protein 3<br>
/        âCACG TGC CACG<br>
1	CTGC GGG GTGC - S-, - M(R15)a - S2 - L - X- Protein 64<br>
[	âCACG CCC CACG<br>
EXAMPLE 3<br>
I.	Preparation of protein mixtures from cells or via protein<br>
translation of a cDNA library prepared from cells or tissues<br>
The protein mixtures can be selectively divided on the physical or   . biochemical separation techniques<br>
1.        Preparation of limited complexity protein pools using cell culture or tissue<br>
Proteins can be isolated from cell culture or tissues according to methods well known to those of skill in the art. The isolated proteins are purified using methods well known to those of skill in the art (e.g., TPAE, differential protein precipitation (precipitation by salts, pH, and ionic polymers), differential protein crystallization bulk fractionation, electrophoresis (PAGE, isoelectric focusing, capillary), and chromatography (immunoaffinit J. HPLC, LC)). Individual column fractions containing protein mixtures of limited<br>
complexity are collected for use as antigen.<br><br>
-165-<br>
2.       Preparation of limited complexity protein pools using cDNA expression libraries (Figure 6)<br>
a.	RNA Isolation<br>
i.        Isolation of Total RNA<br>
Cultured celis or tissues are homogenized in a denaturing solution containing 4 M guanidine thiocyanate. The homogenate is mixed sequentially with 2 M sodium acetate (pH 4), phenol, and finally chloroform/isoamyl alcohol or bromochloropropane. The resulting mixture is centrifuged, yielding an upper aqueous phase containing total RNA. Following isopropanol precipitation, the RNA pellet is dissolved in denaturing solution (containing 4 M guanidine thiocyanate), precipitated with isopropanol, and washed with 75% ethanol.<br>
ii.       Isolation of Cytoplasmic RNA<br>
Cells are washed with ice-cold phosphate-buffered saline and kept on ice for all subsequent manipulations. The pellet of harvested cells is resuspended in a lysis buffer containing the nonionic detergent Nonidet P-40. Lysis of the plasma membranes occurs almost immediately. The intact nuclei are removed by a brief micro centrifuge spin, and sodium dodecyl sulfate is added to the cytoplasmic supernatant to denature protein. Protein is digested with protease and removed by extractions with phenol/chlorofocm and chloroform. The cytoplasmic RNA is recovered by ethanol precipitation.<br>
b.	mRNA purification<br>
Messenger RNA is purified from total or cytoplasmic RNA preparation using standard procedures. Poly(A)+ RNA can be separated from total RNA by oligo (dT) binding to the Poly(A) tail of the mRNA. Total RNA is denatured to expose the Poly(A) (polyadenylated) tails. Poly(A)-containing RNA is then bound to magnetic beads coated with oligo(dT) and spirited from the total or cytoplasmic RNA through magnetic forces. The mRNA population can be further enriched for the presence of full-length molecules through the selection of a 5'-cap containing mRNA species.<br><br>
-166-<br>
c.      cDNA synthesis<br>
Different types of primers can be used to synthesis full length or 5'-end containing cDNA libraries from the isolated mRNA.<br>
i.        Oligo (dT) primer, which will generate cDNAs for all mRNA species (Figure 7)<br>
An example of the production of an adapted oligo dT primed cDNA library is provided in Figure 7.<br>
ii.       Functional protein motif specific degenerate oligonucleotide primers will generate a limited number of genes belonging to the same protein family or of functionally related proteins (Figure 8)<br>
An example of the production of an adapted sequence motif specific cDNA library is provided in Figure 8.<br>
iii.      Gene specific oligonucleotide will produce cDNA for only one mRNA species (Figure 9)<br>
The oligonucleotides used for the cDNA production can contain additional sequences, 1) protein tag specific sequences for easier purification of the recombinant proteins (6 x His), 2) restriction enzyme sites, 3) modified 5'-end for cDNA purification or DNA construction purposes (Figure 10).<br>
The conversion of mRNA into double-stranded cDNA for insertion into a vector is carried out in two parts. First, intact mRNA hybridized to an oligonucleotide primer, is copied by reverse transcriptase and the products isolated by phenol extraction and ethanol precipitation. The RNA in the RNA-DNA hybrid is removed with RNase H as E. coli DNA polymerase I fills in the<br>
gaps. The second-strand fragments thus produced are ligated by E. coli DNA<br>
ligase. Second-strand synthesis is completed, residual RNA degraded, and cDNA made blunt with RNase H, RNase A, T4 DNA polymerase, and E. coli DNA ligase.<br><br>
-167-<br>
d.	Adapter ligation<br>
Adapter molecules can be ligated to both ends of the blunt ended double stranded cDNA or to only one end of the cDNA. Site directed adapter ligation could be achieved through the use of 5' modified oligonucleotides (for example biotinylated, aminated) during cDNA synthesis that prevents adapter ligation to the 3' end of the cDNA. The resulting cDNA molecules contain a 5':end cDNA library comprised of the 5' non-translated region, the translational start codon AUG coding for a methionine, followed by the coding region of the gene or genes. The cDNA molecules are flanked by known DNA sequence on their 5'- and 3'-ends (Figures 14, 15 and 16).<br>
e.	cDNA amplification<br>
PCR Primers to the known 5'- and 3'-end sequences or known internal sequences can be synthesized and used for the amplification of either the complete iibrary or specific subpopulations of cDNA using an extended 5'- or 3'- amplification primer in combination with the primer located on the opposite site of the cDNA molecules (Figure 11).<br>
f.	Primer design for the amplification of gene sub-<br>
populations<br>
The sub-population primers contain two portions (Figure 12). The 5'-part of the primer is complementary to the sequence of a known sequence, extending with its 3'-end into the unknown cDNA sequence. Since each nucleotide in the cDNA part of the library can have an adenosine, cytidine, guanosine or thymidine residue, 4 different nucleotides possibilities exist for each nucleotide position. Four different amplification primers can be synthesized, each containing the same known sequence and extending by one nucleotide into the cDNA area of the library. The 4 primers only differ at their most 3'-nucleotide, being either A, C, G or T. If we suppose that each nucleotide (A. C, G, T) is equally represented in a stretch of DNA, each one of the 4 amplification primers will amplify one quarter of the total genes represented in the cDNA library.  Extending the amplification primer sequence further and increasing the number of amplification primers, the complexity of<br><br>
-168-<br>
the amplification products can be further reduced. Extending the sequence by 2 nucleotides requires the synthesis of 16 different primers decreasing the<br>
complexity by 16 fold, 3 nucleotides require 64 different primers and<br>
nucleotide extension requires n4 different primers. g.       PCR amplification<br>
PCR amplification entails mixing template DNA, two appropriate oligonucleotide primers (5'- and 3'-end primers located in the known added sequences directed in complementary orientation), Taq or other thermostable<br>
DNA polymerases, deoxyribonucleoside triphosphates (dNTPs), and a buffer.	<br>
The PCR products are analyzed after cycling on DNA gels or through analysis on an ABI 377 using the genescan analysis software. These analysis methods allow the determination of the complexity of the amplified cDNA pool.<br>
h.	Production of a protein expression library Each amplified cDNA library sub-population is cloned 5' to 3' in a bacterial (E. coli, etc.) or eukaryotic (Baculovirus, yeast, mammalian) protein expression system. The gene s introduced with its own translational initiation signal and a 6xHis tag in all 3 frames. For example: the cDNA is restricted with two different, rare-cutting restriction enzymes (5'-end Bglll and 3'-end Not I) and cloned in the 5' to 3' orientation in the Baculovirus transfer vector pVL1393 under the direct control of the polyhedra promoter. <br>
i.	i.        Protein expression Lineanzed Baculovirus DNA and recombinant transfer-vector DNA are cotransfected into susceptible Sf9 insect cells with calcium phosphate. For cotransfection. 10 ug of purified plasmid DNA is prepared. An initial recombinant Baculovirus stock is prepared and Sf9 cells ars' infected for recombinant protein production.<br>
j.        Protein purification The expressed recombinant proteins contain an affinity tag (an example is a 6xHis tag). They are purified on Ni-NTA agarose. Approximately 1 to 2 mg of 6xHis recombinant fusion protein is routinely<br><br>
-169-<br>
obtained per liter of insect cell culture.<br>
k.        Purification Tag removal<br>
If the expression vector or the amplification primer was constructed with a proteolytic cleavage site for thrombin, the purification tag can be removed from the recombinant proteins after the protein affinity purification<br>
step.<br>
II.	Antibody generation by immunization of different animals with<br>
individual protein mixtures<br>
3.      Preparation of Antibody protein capture reagents<br>
A purified protein preparation translated from a pool of cDNAs is injected intramuscularly, intradermally, or subcutaneously in the presence of adjuvant into an animal of the chosen species (rabbit).  Booster immunizations are started 4 to 8 weeks after the priming immunization and continued a: 2- to 3-week intervals. The polyclonal antiserum is purified using standards known to those skilled in the art.<br>
The purified antibody batches can be used directly as protein capture reagents without modification.  In this case the antibody batches from different animals have to be kept separate (each batch is one capture reagent)<br>
III.	Antibody proteins are isolated and conjugated with nucleic acid -<br>
sequences that correspond to the original antigen preparation<br>
resulting in the antibody capture reagents<br>
Generation of bi-functional capture/sorting molecules for sorting of the complex protein mixture on a solid phase.<br>
,   The glycosylated CH2 domain of the polyclonal antibodies are conjugation to 5' modified oligonucleotides using standard conjugation methods. The resulting molecule has one protein capture moiety (antibody) and one nucleic acid moiety (oligonucleotide) (Figure 13).<br>
The antibody batches after immunization of an animal with a reduced complexity protein pool are conjugated with the one oligonucleotide<br><br>
-170-<br>
sequence. Antibodies produced from multiple immunization events with different protein pools are conjugated to an oligonucleotide with a different f  sequence (Figure 13).<br>
4.       Capture of target proteins using reactivity, functionality and sorting by oligonucleotide hybridization<br>
Two different methods have been developed for making oligorucleotides bound to a solid support: they can be synthesised in situ, or Dresvnthesised and attached to the support. In either case, it is possible to   . use the support-bound oligonucleotides in a hybridization reaction with oligonucleotides in the liquid phase to form duplexes; the excess of oligonucleotide in solution can then be washed away.<br>
The support can take the form of particles, for example, glass spheres, or magnetic beads. In this case the reactions could be carried out in tubes, or in the wells of a microtitre plate. Methods for synthesising oligonucleotides and for attaching presynthesised oligonucleotides to these materials are known (see. e.g., Stahl era/. (1988) Nucleic Acids Research 16(7)3025-3039).<br>
a.       Preparation of amine-functionalized solid support<br>
Oligonucleotides of a defined sequence are synthesized on an amine-functionalized glass support. An amine function was attached at discrete . locations on the glass slide using a solution of 700 il of H2(CH2)3Si(OCH2)3 in 10 ml of 95% ethanol at room temperature for 3 hours. The treated support is washed once with methanol and then once with ethyl ether. The support was dried at room temperature and then baked at 110 Â°C for 15 hours. It was then washed with water, methanol and water, and then dried.<br>
The glass slide was reacted for 30 minutes at room temperature with 250 mg (1 millimole) of phthallic anhydride in the presence of 2 ml of anhydrous pyridine and 61 mg of 4-dimethylaminopyridine.<br>
The product was rinsed with methylene dichloride, ethyl alcohol and ether, and then dried. The products on the slide were reacted with 330 mg of dicvclohexvlcarbodiimide (DCC) for 30 minutes at room temperature. The<br><br>
-171-<br>
solution was decanted and replaced with a solution of 117 mg of 6-amino-1-hexanol in 2 ml of methylene dichloride and then left at room temperature for approximately 8 hours.<br>
b.       Oligonucleotide synthesis on a solid support<br>
The amine-functionalized solid support was prepared for oligonucleotide synthesis by treatment with 400 mg of succinic anhydride and 244 mg of 4-dimethylaminopyridine in 3 ml of anhydrous pyridine for 18 hours at room temperature. The solid support was treated with 2 ml of DMF containing 3 millimoles (330 mg) of DCC and 3 millimoles (420 mg) of p-nitrophenol at room temperature overnight. The slide was washed with DMF, CH3CN, CH2CI2 and ethyl ether. A solution of 2 millimoles (234 mg) of H2N(CH2)60H in 2 ml of DMF was reacted with the slide overnight. The product of this reaction was a support.<br>
0(CH2)3NHCO(CH2)2CONH(CH2)5CH2OH. The slide was washed with DMF, CH3CN, methanol and ethyl ether.<br>
The functionalized ester resulting from the preparation of the glass support was used for the synthesis of a oligonucleotide sequence. Each nucleoside residue was added as a phosphoramidite according to known procedures (see, e.g., U.S. Patent Nos. 4,725,677 and 5,198,540, and RF.34,069, see, also Caruthers et al. U.S. Patent No. 4,415,732).<br>
5.       Protein analysis of the captured proteins and complex protein sample comparison<br>
The purified antibody batches can be either 1) directly attached to a solid surface, and incubated with protein samples, 2) incubated with the samples and subsequently bound to a solid support without using the capture compound or 3) the capture compound can be used to caoture its corresponding protein in a sample and subsequently sort ;he captured proteins through specific nucleotide hybridization (Figure 14).<br><br>
-172-<br>
IV.	Antisense oligonucleotide capture reagents are immobilized in<br>
discrete and known locations on a solid surface to create an<br>
antibody capture array<br>
6.	Preparation of capture array surface<br>
5'-aminated oligonucleotides are synthesized using phosphoramidate chemistry and attached to N-oxysussinimide esters. The attached oligonucleotide sequences are complementary to the sorting oligonucleotides of the bi-functional antibody molecules (Figure 13). Proteins are captured through nucieic acid hybridization of their sorting oligonucleotide to the complementary sequence attached to the solid surface oligonucleotide.<br>
V.	The antibody capture reagents are added to the total protein<br>
mixture (reactivity step). The reaction mixture is then added to<br>
the solid surface array under conditions that allow<br>
oligonucleotide hybridization (sorting step).<br>
7.	Capture compound/protein capture and sorting<br>
The bi-functional antibodies are incubated with the protein sample under conditions that allow the antibodies to bind to their corresponding antigen. The bi-functional antibody molecule with the captured protein is added to the oligonucleotide prepared capture array. Under standard DNA annealing conditions that do not denature the antigen-antibody, binding the bi-functional antibody will hybridize with its nucleic acid moiety to the complementary oligonucleotide.<br>
VI.	The captured protein is identified using MALDI mass<br>
spectrometry<br>
8.	Analysis of the capture proteins<br>
The attached proteins are analyzed using standard protein analysis<br>
methods, such as mass spectrometry.<br><br>
-173-<br>
EXAMPLE 4 Synthesis of Trityl based Protein capture compounds (see Figure 15)<br>
A.	Synthesis of 2-(4-bromophenyl)-4,4-dimethyl-1,3-oxazoline,<br>
1<br>
To 4-Bromobenzoic acid (50 g, 0.25M) placed in a 500 mL round bottom flask fitted with a reflux condenser was added 150 mL of thionyl chl6ride and refluxed for 8 h. The excess thionyl chloride was removed under vacuum and the white solid obtained was dissolved in 100 ml of dry CH2CI2 and kept in an ice bath. To this ice cooled solution of bromo benzoylchloride was added drop wise 45 g of 2-amino-2-methylpropan-1-ol dissolved in another 100 mL of dry CH2CI2 with stirring for the period of 1 h. The ice bath was removed and the reaction mixture was stirred at room temperature for over night. The precipitated white solid was filtered and washed several times with CH2CI2 (4x100 mL). The combined CH2CI2 was removed under rotaevoporator and the solid obtained was slowly dissolved in 150 mL of thionyl chloride and refluxed for 3 h. The excess of SOCI2 was evaporated to one-sixth the volume and poured in to 500 mL of dry ether cooled in ice bath and kept in the refrigerator overnight. The ether was removed and the<br>
precipitated hydrochloride was dissolved in 500 mL of cold water. The<br>
aqueous solution was carefully neutralized using 20% KOH solution on cold condition (ice bath) and the brown oily residue separated was extracted with CH2CI2 (3x200 mL) and dried over anhydrous Na2S04. Removal of the solvent gave 42 g (67%) of 2-(4-bromophenyl)-4,4-dimethyl-1,3-oxazoline as a yellow oil.  1H-NMR (500 MHz, CDCI3) a ppm: 1.36 (s, 6H), 4.08 (s, 2H), 7.52 (d, 2H), 7.79 (d, 2H). Mass: 254.3 (M+).<br>
B.	Synthesis of pheny3-{3-[2-(tetrahydropyran-2-yloxy)-ethoxy]-<br>
phenylj-methanone, 2<br>
1. Method A: In a 100 mL two neck round bottom flask placed with 550 mg (8 mM) of NaOEt in 20 mL of dry DMF was added 3-hydroxy benzophenone (1 g, 5 mM) under argon atmosphere. The reaction<br><br>
-174-<br>
was stirred at room temperature for 10 min and added 2-bromoethoxy tetrahydropyran (1 g, 5 mM) dissolved in 5 m'L of dry DMF by drop wise. The<br>
reaction mixture was heated at 60 Â°C for overnight, cooled and poured into ice water and extracted with CH2CI2 (2x50 mL). The combined solvent was dried over anhydrous Na2S04 and evaporated. The crude residue obtained was purified by silica gel column chromatography using hexane/EtOAc (9:1) mixture as an eluent. Yield : 680mg (42%).<br>
2.	Method B: <br>
To the stirred mixture of 3-hydroxy benzophenone (1 g. 5 mM), anhydrous K2CO3 (3g, 23 mM) and Nal (500 mg) in dry acetone (40ml_) was added 2-bromoethoxytetrahydropyran (1g, 5 mM) dissolved in 10 mL of dry acetone and refluxed for 20 h. The precipitate was<br>
filtered and washed with acetone (3x20 mL). The combined filtrate was evaporated and the yellowish residue obtained was purified by silica gel column chromatography using hexane/EtOAc (9:1) mixture as an eluent. Yield: 55-60%.  1H-NMR (500 MHz, CDCI3) a ppm: 1.5-1.63 (m, 4H), 1.72 (m, 1H). 1.82 (m, 1H). 3.52 (m, 1H), 3.8-3.9 (m. 2H), 4.07 (m, 1H), 4.21 (m, 2H), 4.70(t. 1H). 7.15 (d. 1H). 7.37(m. 3H), (7.47 (t. 2H), 7.58(t,1H), 7.80(d,1H). Mass: 327.2(M"). 349.3<br>
(M+Na*).<br>
C.       Grignard reaction: Synthesis of 2-{4'-(3-(2-tetrahydropyran-2-yloxy)ethoxy)phenyl-4"-phenyl)}-4,4-dimethyl-1,3-oxazoline, 3<br>
To a 100 mL two necked round-bottomed flask fitted with reflux condenser was placed activated Mg turnings (720 mg, 30 mM), a few crystals of l2 and molecular sileves (A4) under argon. To this mixture 10 ml of THF was added. The mixture was heated to 50 Â°C and 2-(4-bromophenyl)-4,4-dimethyl-1,3-oxazoline (6.5g, 26 mM) dissolved in 15 mL of dry THF, a catalytic amount of CH3I, RED-AI and CCI4 were added with stirring and refluxed for 3h. After that the reaction mixture was cooled to room temperature and added phenyl-{3-[2-(tetrahydropyran-2-yloxy)-ethoxy]-phenyl}-methanone (5.1 g, 15.6 mM) dissolved in 15 mL of dry THF and again<br><br>
-175-<br>
refluxed for 3 h, cooled and 3mL of water added. The solvent was removed under roraevaporator and extracted with CHCI3 (3x100 mL) and dried over<br>
anhydrous Na2S04. The residue obtained on removal of the solvent was separated by silica gel column chromatography using hexane/EtOAc (7:3) as an eluent. Evaporation of the column fraction yielded 2-{4'-(3-(2-tetrahydropyran-2-yloxy)ethoxy)phenyl-4"-phenyl)}-4,4-dimethyl-1.3-oxazoline (3) as a yellow crystalline solid (1.4g, 18%). 1H-NMR(500 MHz, CDCI?1 a ppm 1.37 (s. 6H), 1.5-1.63 (m, 4H), 1.68 (m, 1H), 1.80(m, 1H), 2.85 (s, 1H, -OH). 3.49 (m. 1H). 3.75(m, 1H), 3.85(m, 1H), 3.97 (m, 1H), 4.09(m, 4H), 4.66 (t. 1H). 6 80(d. 1H), 6.84(d, 1H), 6.88(s,1H), 7.18-7.31(m. 6H), 7.34 (d, 2H), 7 87(d. 2H). Mass: 502.6 (M+1), 524.5 (M+Na+)<br>
D.	4,4-Dimethyl-2-[4-(phenyl-[2-(tetrahydro-pyran-2-yloxy)-<br>
ethoxy]-{3-[2-(tetrahydro-pyran-2-yloxy)-ethoxy]-phenyl}-<br>
methyl)-phenyl]-4,5-dihydrooxazole, 4<br>
To the stirred mixture of 2-{4'-(3-(2-tetrahydropyran-2-yloxy)ethoxy)phenyl-4'-phenyl)}-4,4-dimethyl-1,3-oxazoline (3, 200 mg, 0.4 mM) and NaH (100 mg. 4 mM) in 3 mL of dry DMF at r.t. was added 2-(2-bromoethoxy)tetrahydro-2H-pyran (500 mg, 2.4 mM) and the reaction was allowed to stir at r.t. for 2h. Then the reaction mixture was poured in to ice water and extracted with CH2CI2 (3x20 mL) and dried over anhydrous Na2S04. Evaporation of the solvent gave 4 as a yellow oily residue in quantitative yield.<br>
E.	4-{(2-Hydroxy-ethoxy)-[3-(2-hydroxy-ethoxy)-phenyl]-phenyl-<br>
methyl}-benzoic acid, 5<br>
A solution of 4 (360 mg) in 3 mL of 80% aqueous acetic acid was heated at 75 :C for 12h. Then the solution was evaporated and the residue obtained was refluxed with 20% NaOH/EtOH (1:1, v/v, 3 mL) for 2 h. The solvent was removed and 10 mL of ice cooled water was added to the residue and the aqueous solution was acidified with 1N HCI. The precipitated yellow solid was filtered and washed several times with water and dried under high<br><br>
-176-<br>
vacuum. Yield: 270 mg (100%, quantitative).<br>
F.	4-{(2-Hydroxy-ethoxy)-[3-(2-hydroxy-ethoxy)-phenyl]-phenyl-<br>
f     methyl}-benzoic acid 2,5-dioxo-pyrrolidin-1-yl ester, 6<br>
1.	Method A: To a stirred solution of trityl acid 5(110 mg, 0.26 mM) and N-hydroxy succinimide (80 mg, 0.7 mM) in dry 1,4-dioxane (2 mL) was added 1-Ethyl-3-(3-dimethylaminopropyl)carbodiimide hydrochloride (EDC, 105 mg, .5 TIM) dissolved in 2 mL of water. The reaction mixture was stirred for 12 h at r.t. and the extracted with CHCb (3x10 mL) and dried over anhydrous Na2SC&gt;4. The solid obtained on evaporation of the solvent was purified by preparative TLC plate. Yield: 5 mg.<br>
2.	Method B. To a stirred solution of trityl acid 5 (12 mg, 0.03 mM) in dry THF (4 mL) was added dicyclohexyl carbodiimide (DDC, 10 mg. 0.05 mM). The reaction mixture was stirred for 30 min at r.t., N-hydroxysuccinimide (11.5 mg, 0.1 mM) and a catalytic amount of<br>
DMAP was added and allowed to stir for overnight. The solvent was removed under rotaevaporator and the solid obtained was dissolved in dry ether. The precipitated DCU was filtered and the solvent ether was evaporated. The crude solid obtained was purified by preparative TLC plate. Yield 7 mg (50%). 'h-NMR (500 MHz. CDCI3) a ppm : 2.90 (s, 4H).  3.92(t, 4H), 4.02 (t. 4H), 6 83( m. 2H), 7.25 (m, 3H), 7.34 (m. 4H). 7.50(d. 2H), 8.0(d, 2H).<br>
G.	4,4-Dimethyl-2-[4-(phenyl-(3-phenyl-propoxy)-{3-[2-<br>
(tetrahydro-pyran-2-yloxy)-ethoxy]-phenyl}-methyl)-phenyl]-4,5-dihydro-oxazole, 7<br>
To the stirred mixture of 2-{4'-(3-(2-tetrahydropyran-2-yloxy)ethoxy)phenyl-4"-phenyl)}-4,4-dimethyl-1.3-oxazoline (3, 300 mg, 0.6 mM) and NaH (100 mg, 4 mM) in 3 mL of dry DMF at r.t. was added 3-bromo-1-phenyl propane (250mg, 1.2 mM) and the reaction was allowed to stir at r.t.<br>
for 2h. Then the reaction mixture was poured into ice water and extracted with CH2CI2 (3x20 mL) and dried over anhydrous Na2S04. Evaporation of the solvent gave 7 as a yellow color residue in quantitative yield.<br><br>
-177-<br>
H.      4-[[3-(2-Hydroxy-ethoxy)-phenyl]-phenyl-(3-phenyl-propoxy)-methylj-benzoic acid, 8<br>
A solution of 7 (550 mg) in 3 ml_ of 807: aqueous acetic acid was heated at 75 Â°C for overnight. Then the solution was evaporated and the residue obtained was refluxed with 20% NaOH/EtOH (1:1, v/v, 3 mL) for 2 h. The solvent was removed, 10 mL of ice cooled water was added to the residue and the aqueous solution acidified win 1N HCI, extracted with CH2CI2 (60 mL) and dried over anhydrous Na2S04. Evaporation of the solvent gave yellow solid Yield: 485 mg (quantitative).<br>
I.        4-[[3-(2-Hydroxy-ethoxy)-phenyl]-phenyl-(3-phenyl-propoxy)-methylj-benzoic acid 2,5-dioxo-pyrrolidin-1-yl ester, 9<br>
To a stirred solution of trityl acid 8 (20C mg. 0.42 mM) in dry THF (6 mL) was added dicyclohexyl carbodiimide (DCC. 206mg, 1 mM). The reaction<br>
mixture was stirred for 30 min at r.t., and N-hydroxysuccinimide (70 mg, 0.6 mM) and a catalytic amount of DMAP added and was allowed to stir for overnight. The solvent was removed under rotaevaporator and the solid obtained was dissolved in dry ether. The precoitated DCU was filtered and<br>
the solvent ether was evaporated. The crude solid obtained was separated by silica column chromatography using CH2CI2 Yield: about 120 mg.   H-NMR (500 MHz, CDCI3) a ppm : 1.70 (m, 2H). 1.9 c 2H). 2.9 (s, 4H), 3.5(m, 2H). 3.9 (t. 2H), 4.0(t, 2H). 6.85( m, 4H), 7.25 (m. 4H). 7.32 (m, 5H), 7.51(m, 3H), 8 09(d, 2H).<br>
J.       1-{4-[[3-(2-Hydroxy-ethoxy)-phenyl]-phenyl-(3-phenyl-propoxy)-methyl]-benzoyl}-pyrrole-2,5-dione, 10<br>
To a stirred solution, of trityl acid 8 (280 mg. 0.42 mM) in dry THF (6 mL) was added dicyclohexyl carbodiimide (DDC. 400mg, 1.95mM). The reaction mixture was stirred for 30 min at r.t. and maleimide (100 mg, 1.1 mM) and a catalytic amount of DMAP was added and allowed to stir for overnight. The solvent was removed under rotaevaporator and the solid obtained was dissolved in ary ether. The precipitated DCU was filtered and the solvent<br><br>
-178-<br>
ether was evaporated. Part of the product was purified by preparative TLC. Yield: 12 mg. 1 H-NMR (500 MHz, CDCI3) a ppm : 1.78 (m 2H), 1.95 (m 2H),<br>
2.9 (s, 4H), 3.51 (m, 2H), 3.93(t, 2H), 4.02(t, 2H), 6.8( m, 5H), 7.25 (m, 5H), 7.29 (m, 5H). 7.37(m. 3H), 7.48(d, 2H), Mass: 561.3 (M+).<br>
EXAMPLE 5<br>
 This Example shows addition of a selectivity function onto a capture compound possessing a N-hydroxy succinimidyl ester reactivity function. Compounds with sorting can be prepared by using an appropriate analog of compound 11 below. Procedure for Mitsunobu Reaction of Trityl Capture Reagents<br><br><br><br><br>
MO-R'<br>
DIM) I'll,I'<br>
Till<br>
Mol Sieves<br><br>
Â°<v></v>
^y<br>
^N<br><br><br><br>
n<br><br>
12<br><br>
1.1 equivalents of triphenylphosphine are added to a reaction vial and dissolved in 1.0 ml THF. 1.1 equivalents of diisopropyl azidodicarboxylate are added to this solution and mixed for 5 minutes. Add 1 equivalent of 11 and stir for 5 minutes. Add nucleophile (R1âOH) and stir overnight at 50 Â°C. Preparative TLC purified the products.<br><br>
-179-<br>
EXAMPLE 6 Cell synchronization<br>
H460 lung cancer and SW480 colon cancer cells were synchronized in Go/G1 with simvastatin and lovastatin (HMG-CoA reductase inhibitors), which can enrich a cancer cell population in Go/G1. Cells arrested in G2/M phase were obtained by treatment with nocodazole.<br>
Cell Culture and Reagents<br>
The SW480 cell line was cultured in Dulbecco's modified Eagle medium (DMEM), the H460 ceil line (ATCC Manassas, VA) was cultured in RPMI 1640, whereas the FK101 was cultured in serum-free medium (SFM) with 5% C02 at 37= C. The cell culture media were supplemented with 10% fetal bovine serum (FBS). 2mM L-glutamine, penicillin(100U/ml)and streptomycm(100U/ml).<br>
Synchronization of Cells<br>
H460 and SW480 cells enriched in G1 phase were obtained after incubation with serum-free medium for 48 hours, or treatment with U026, lovastatin or simvastatin. Cells in S phase were synchronized by incubating cells with medium containing no serum for 24 hours, followed by aphidicolin treatment (2ug/ml) for 20 hours and release of cells from aphidicolin for 3 hours. Cells arrested in G2/M phase were obtained by treatment with nocodazole (0.4-0.8 mg/ml) for 16-20 hours.<br>
EXAMPLE 7<br>
Synthesis of (4,4'-bisphenyl-hydroxymethyl)benzoyl maleimide derivatives<br><br>
-180-<br><br><br><v></v>
^<br><br>
i. SOC'k reflux. Ih<br><br><br><br><br><br>
-OH<br><br>
ii. malcimidc. dr&gt; Till". 2h. r.t. iii.  R-Oll. dr\ p\ridinc. r.t. overnight<br><br>
-OR<br><br><br>
-181-<br><br>
General Procedure: A solution of 4-(diphenylhydroxymethyl)benzoic acid (0.04 mM) in 1 mL of SOCI2 was refluxed for 1 h and the excess SOCI2 was removed under high vacuum To this yellow solid residue obtained was added maleimide (0.045 mM) dissolved in dry freshly distilled THF (1 mL) and stirred at room temperature for 2h. The solvent was removed and added the corresponding alcohol (ROH, 2-5 fold excess) dissolved in dry pyridine (1mL) with stirring. After the reaction mixture stirred at room temperature for overnight the solution was extracted with CH2CI2 (5x3mL) and dried over<br><br>
-182-<br>
anhydrous Na2S04. The residue obtained on evaporation of the solvent was separated by preparative TLC (Silica Gel, 500 im plate) and gave the product<br>
1 in 50-60% yield. The trityl derivatives 1 were fully characterized by 1H NMR<br>
and mass spectral data.<br>
EXAMPLE 8 Succinimidyl Ester Trityl Capture Compound Synthesis Procedure 1<br><br>
4-(Diphenylhydroxymethyl) benzoic acid was reacted with 2 equivalents of N-hydroxysuccinimide using 1.2 equivalents of Diisopropyl carbodiimide   The desired product was purified by Flash Silica chromatography and characterized by ESI mass spectrometry.<br><br>
-183-<br><br><br><br>
V/Â°~N'<br>
k^1<br>
-OH<br>
/S<br><br>
Act!<br><br><br><br>
0	O-N<br>
o<br><br><br><br>
0<br><br>
The 125 jimoles of product from above was added to 1.0 ml Acetyl Chloride. This reaction mixture was stirred at room temperature for 1 hour and evaporated three times with toluene to remove excess acetyl chloride. Equal volumes of the reaction mixture were added to nucleophiles (see below) dissolved in 1.0 M Pyridine/THF. These reaction mixtures were mixed at 60Â°C for 2 hours. The resulting products were extracted from CHCI3 and 10% HOAc. Products were purified by Preparative TLC (Ether). MS and NMR characterize purified products.<br><br>
-184-<br><br>
ROH =<br><br>
HO'<br><br>
xx<br><br>
,oâ<br><br>
"OMe<br><br><br><br>
HO'<br><br>
,0.<br><br>
-o'<br><br>
vO-n-heptyl<br><br><br><br>
HO<br><br>
.0.<br><br>
*o'<br><br>
,0.<br><br>
'OH<br><br><br><br><br><br>
O<br><br><br>
0"	"n-pentadecyl<br>
O^	^n-pentadecyl<br><br>
Procedure 2<br><br><br><br><br>
âOH<br>
Â°^0H<br>
\^<br>
/     <br>
SOCI,<br><br>
HO	N<br><br>
ROH<br><br>
CV/0~N<br>
-OR<br>
-       /        <br>
pyridine<br>
1.64 mmoles of 4-(Diphenylhydroxymethyl) benzoic acid was dissolved into 5 ml Thionyl Chloride. This reaction mixture is heated to 79Â°C and stirred<br><br>
-185-<br>
for 75 minutes. The Thipnyl Chloride is removed under N2 (g) stream. 1.3 equivalents of N-hydroxysuccinimide dissolved in dry THF is added to this<br>
dried reaction mixture and stirred for 1 hour. The THF solvent is removed<br>
under N2 (g) stream. The product is dissolved into dry Pyridine. Equal volumes of this solution are added to nucleophiles dissolved in Pyridine. (See below). The resulting products are extracted from CHCI3 and 10% HOAc. Products are purified by Preparative TLC (Ether). MS and NMR characterize purified products.<br><br>
o<br><br>
-186-<br>
EXAMPLE 9<br>
This example shows exemplary capture binding assays and the effects of selectivity functions on binding. This example shows that changing selectivity can alter reactivity of the capture compound thereby providing a means to probe biomolecule structures and to permit sorting or diversity reduction using the collections.   In this example, the core group of the capture compounds is a trityl group and the reactive group is succinimide, which interacts with a primary amine.   Compound 1341 is a non-selective compound that has a reactivity group, but no selectivity group. Compound 1343 (see Figure 20) is exemplary of such compound where the selectivity group is -OH. As the selectivity group changes there is a difference in reactivity on the target proteins (lysozyme, cytochrome C and ubiquitin). Lysozyme<br>
Three different capture compounds (designated HKC 1343, 1349, 1365; chemical structure of each compound is listed below the Compound name) were reacted individually with Lysozyme (Accession number P00698; Figure 20b)   The capture experiments were analyzed using MALDI-TOF Mass Spectrometry   Binding was performed in 20 uL sample volumes with a 5 uM Lysozyme concentrations in 25 mM HEPES buffer solution, pH 7.0. The tntyl-based capture compounds were added to the protein solution at a 10 uM concentration The binding reaction was incubated at room temperature for 30 minutes The reaction was quenched using 1 uL of a 100 mM TRIZMA base solution.<br>
The capture compound-protein binding mixture was prepared for mass spectrometry by mixing a 1 uL aliquot of a binding reaction with 1 uL of a 10mg/mL sinapinic acid in 30% aqueous acetonitrile. The sample was deposited as a 500 nL spot on the surface of the mass target plates and ai"-dried before mass spectrometric analysis. The results of the mass spectrometry analysis, which are shown in Figure 20b, demonstrate that addition of selectivity groups to compounds permits alterations in the binding<br><br>
-187-<br>
specificity of capture compounds. Cytochrome C<br>
Four different capture compounds (designated HKC 1341, 1343, 1349, 1365; chemical structure of each compound is listed below the Compound name) were reacted individually with Cytochrome C (accession number: P00006, Figure 20c). The capture experiments were analyzed using MALDI-TOF Mass Spectrometry. Binding was performed in 20 uL sample volumes with a 5 uM Cytochrome C concentrations in 25 mM HEPES buffer solution,<br>
pH 7.0. The trityl-based capture compounds were added to the protein solution at a 10 uM concentration. The binding reaction was incubated at room temperature for 30 minutes. The reaction was quenched using 1 uL of a 100 mM TRIZMA base solution. The capture compound-protein binding mixture was prepared for mass spectrometry analysis by mixing a 1 uL aliquot of the binding reaction with 1 uL of a 10mg/ml_ sinapinic acid in 30% aqueous acetonitrile   The sample was deposited as a 500 nL spot on the surface of mass target plates and subsequently air-dried before mass spectrometric analyses. The results of the mass spectrometry analysis, which shown in<br>
Figure 20c, demonstrate that addition of selectivity groups to compounds permits alterations in the binding specificity of capture compounds.<br>
HKC 1343<br>
One of the exemplary capture compounds (HKC 1343) was incubated<br>
with a mixture of three different proteins (Ubiquitin, [P02248], Cytochrome C -[P00006] and Lysozyme [P00698]) (see, Figure 20d). The capture experiment was analyzed using MALDI-TOF Mass Spectrometry. The binding reactions were performed in a 20 uL sample volume with all three proteir.s at 5 uM concentrations in 25 mM HEPES buffer solution pH 7.0. The trityl-based capture compound was added to the protein solution at a 25 uM concentration. The binding reaction was incubated at room temperature 'or 30 minutes and the reaction quenched using 1 uL of a 100 mM TRIZMA base solution. The capture compound-protein binding mixture was prepared for mass spectrometry by mixing a 1 uL aliquot of the binding reaction with * ul_<br><br>
-188-<br>
of10mg/mLsinapinic acid in 30% aqueous acetonitrile. The sample was deposited as a 500 nl_ spot on the surface of mass target plates and air-dried<br>
before mass spectral analysis. The results of the mass spectrometry analysis, which are shown in Figure 2Cd: demonstrate that a plurality of compounds bound to a single capture agent that is selective can be identified by mass spectrometric analysis. HKC 1365<br>
Another of the exemplary capture compounds (HKC 1365) was<br>
incubated with a mixture of three different proteins (Ubiquitin [P02248],<br>
Cytochrome C [P00006] and Lysozyme [P00698]; see Figure 20d). The capture experiment was analyzed using MALDI-TOF Mass Spectrometry. The binding reactions were performed in a 20 uL sample volume with all three proteins at 5 uM concentrations in 25 mM HEPES buffer<br>
solution pH 7.0. The trityl-based capture compound was added to the protein solution at a 15 uM concentration. The binding reaction was incubated at room temperature for 30 minutes, and quenched using 1 uL of a 100 mM TRIZMA base solution. The capture compound-protein binding mixture was prepared for mass spectrometry by mixing a 1 uL aliquot of the binding reaction with 1 uL of a 10mg/ml_ sinapinic acid in 30% aqueous acetonitrile. The sample was deposited as a 500 nl_ spot on the surface of the mass target plates and air-dried before mass spectral analyses. The results of the mass spectrometry analysis, which are shown in Figure 20e, demonstrate that a plurality of compounds bound to a single capture agent that is selective can be identified by mass spectrometric analysis. Reaction of cytochrome C with a non-specific compound<br>
Figure 20f shows mass spectra for a time course reaction of cytochrome C with a non-specific compound (HKC 1341). The succinamide reactive group shows specificity and reactivity with the lysines of cytochrome c. The top spectrum shows no modification at time 0, the middle spectrum shows 1-9 modifications resulting from binding of HKC1341 after 30 minutes, and the bottom spectrum shows, after 24 hours, 17 and 18 modifications,<br><br>
-189-<br>
which correspond to the number of lysines (18) in cytochrome c.<br>
EXAMPLE 10<br>
This example shows the selectivity of the capture compound reacting a<br>
mixture of capture compounds &amp;r J a mixture of proteins Materials:<br>
Reaction buffer:       25 mM HEPES, pH 7.0<br>
Proteins:       mixture of ubiquitin, cytochrome c and lysozyme (molar ratio is 1/5/6), the protein stock is made as 5 mg/ml (total proteins) in reaction buffer.<br>
Capture compounds: HKC 1343 and HKC 1365, stock solution is 1 mM in acetonitrile. Capturing reaction<br>
A protein dilution (mixture) is prepared in the reaction buffer at the concentration of 0.5, 2.5 and 3 iM, for ubiquitin, cytochrome c and lysozyme, respectively. 19.5 il is used for one capturing reaction. Each reaction is started by adding 0.5 il of 1 mM compound stock solution (final 25 iM). The reaction mixture is incubated at room temperature for 30 min before the reaction is stopped by the addition of 5 mM TRIZMA.<br>
Three different reactions are run. The first two tubes contain HKC<br>
1343 and HKC 1365 individually, and a third one is started by adding<br>
compounds HKC 1343 and 1365 (final concentration 25 iM for each<br>
compound). After the reaction, 1 il of each sample is mixed with equal<br>
volume of matrix and subjected to MALDI analysis. Statistic significance of<br>
the results is ensured by triplicate each reaction sample.<br>
EXAMPLE 11<br>
Synthesis of 4-{Hydroxy-[3-(3-{6-[5-(2-oxo-hexahydro-thieno[3)4-d]imidazol-4-yl)-pentanoylamino]-hexanoylamino}-propoxy)-phenyl]-phenyl-methyl}-benzoic acid succinimidyl ester (6)<br><br>
-190-<br><br><br><br>
Â»<br><br>
n-Buli.dtvTIIF<br>
-78 "C. 85%<br><br>
BHCH;VNHB0C<br>
	i<br>
PowCred KOH. DMSO. r.t quantitative \ icld<br><br><br>
O.V.OH<br>
anhy. DMI-. r t     q-j x"â<br>
\:
 <br><br>
s^-<br>
V^<br><br><br><br>
Q<br>
HOâ:<br>
l)l(<br>
anh\   !)\ir. rt<br>
41",,<br><br><br>
Synthesis of 3-{[4-(4,4-Dimethyl-4,5-dihydro-oxazol-2-yl)-phenyl]-hydroxy-phenyl-methyl}-phenol (2)<br>
2-(4-B^mophenyl)-4,4-dimethyl-1,3-oxazoline 1 was prepared as described in Example 4. To a stirred solution of 2-(4-bromophenyl)-4.4-dimethyl-1.3-:xazoline (1.5 g, 6 mM) in anhydrous THF (10 ml) at-78 Â°C was added sickly n-BuLi (384 mg, 6 mM) in hexane over the period of 20 min. After that the reaction mixture was stirred at -78Â°C for another 30 min. To this stirreca solution was added 3-hydroxybenzophenone (534 mg, 2.7 mM) dissolved in aihy. TF (10 mL) by dropwise at-78 Â°C and allowed to stir at<br>
room temperature overnight. To this reaction mixture was added 20 mL of<br><br>
-191-<br>
water to quench the reaction and extracted with CH2CI2 (3x50 mL) and the<br>
combined extract was dried over anhydrous Mg2S04. The oily residue<br>
obtained on vaporation of the solvent was purified by silica gel column<br>
chromatography using hexane/EtOAc (1:1) mixture gave 3-{[4-(4,4-Dimethyl-<br>
4,5-dihydro-oxazol-2-yl)-phenyl]-hydroxy-phenyl-methyl}-phenol (2) as<br>
colorless crystalline solid. Yield: C.855 g (85%). Mass: 374 (MH+), 372 (M-H).<br>
Synthesis of [3-(3-{[4-(4,4-Dimethyl-4,5-dihydro-oxazol-2-yl)-phenyl]-hydroxy-phenyl-methyl}-phenoxy)-propyl]-carbamic acid tert-butyl ester<br>
(3)<br>
To a solution of powdered KOH (45 mg, 0.8 mM) in anhydrous DMSO<br>
(2.5 mL) at room temperature was added 3-{[4-(4,4-Dimethyl-4,5-dihydro-<br>
oxazol-2-yl)-phenyl]-hydroxy-phenyl-methyl}-phenol (2, 150 mg, 0.4 mM) and<br>
(3-Bromo-propyl)-carbamic acid tert-butyl ester (96 mg, 0.4 mM). The reaction<br>
mixture was stirred at room temperature for 3h. Then the reaction mixture<br>
was extracted with ethyl acetate (3x25 mL) and the combin<br>
ed extract was dried over anhydrous Mg2S04. The residue obtained on<br>
evaporation of the solvent was purified by silica gel chromatography using<br>
hexane/EtOAc (1:1) as an eluent. Evapoartion of the solvent gave 3. Yield:<br>
&gt;220 mg (quantitative yield). Mass: 531 (MH+).<br>
Synthesis of 4-{[3-(3-Amino-propoxy)-phenyl]-hydroxy-phenyl-methyl}-benzoic acid (4)<br>
In a 50 mL round bottomed flask placed with [3-(3-{[4-(4,4-Dimethyl-<br>
4,5-dihydro-oxazol-2-yl)-phenyl]-hydroxy-phenyl-methyl}-phenoxy)-propyl]-carbamic acid tert-butyl ester (3. 220 mg) was added 3 mL of 80% aqueous AcOH and the reaction mixture was heated 75 Â°C for overnight. Then the reaction mixture was concentrated and dried and added 3 mL of 20% NaOH/EtOH (1:1, v/v)and refluxed for 3 h. Residue obtained on evaporation of the solvent was dissolved in CH3OH/CHCI3 mixture and adsorbed with silica gel and dried. The dried silica gel with compound was purified by silica gel column already flashed with 1% NH4OH in Et20 solution. Elution of the column at 50% CH3OH/CH2CI2 gave 4-{[3-(3-Amino-propoxy)-phenyl]-hydroxy-phenyl-methyl}-benzoic acid, 4 as a colorless gelly solid. Yield: 96%.<br><br>
-192-<br>
Mass: 378 (MH+), 376 (M-H), 360(M-OH).<br>
Synthesis of 4-{Hydroxy-[3-(3-{6-[5-(2-oxo-hexahydro-thieno[3,4-<br>
d]imidazol-4-yl)-pentanoylamino]-hexanoylamino}-propoxy)-phenyl]-<br>
phenyl-methyl}-benzoic acid (5)<br>
A mixture of Trityl amino acid (4, 100 mg, 0.26 mM) and Biotin-X-NHS (113mg, 0.25 mM) was stirred at room temperature in 3 ml_ of anhydrous<br>
DMF for overnight. After that DMF was removed under high vacuum and the residue obtained was passed through silica gel column using 50%<br>
CH3OH/CHCI3 as a solvent. Evaporation of the solvent yieded biotinlated trityl<br>
acid 5. (97.8%). Mass: 739(M Na+), 715 (M-H).<br>
Synthesis of 4-{Hydroxy-[3-(3-{6-[5-(2-oxo-hexahydro-thieno[3,4-d]imidazol-4-yl)-pentanoylamino]-hexanoylamino}-propoxy)-phenyl]-phenyl-methyl}-benzoic acid succinimidyl ester (6)<br>
To a solution of biotinylated trityl acid (5, 175 mg, 0.244 mM) in anhydrous DMF (3 mL) was added 1.3-diisopropyl carbodiimide (4 mg, 0.35 mM) and stirred the reaction mixture for 5 min. To this reaction mixture was added N-hydroxy succinimide (40mg, 0.32 mM) and stirred for over night at room temperature. The solvent was removed under high vacuum and the residue obtained was purified by silca gel column chromatography using CH3OH/CH2CI2 3:7) mixture as a solvent system. Evaporation of the solvent gave 6 as a white crystalline solid. Yield: 80 mg (41%). 1H-NMR (CD3OD) 8 ppm<br>
1.29-1.71 (m. 12H). 1.90-193 (m, 2H), 2.15 (q, 4H), 2.49 (t, 1H). 2.8-2.91 (m 2H.). 2.90 (s. 4H), 3.17(m, 4H), 3.94 (q, 3H), 4.27 (dd, 1H), 4.46 (d of d , 2H), 4.59 (br. S. 4H), 6.77(s, 1H), 6.86 (m, 2H), 7.18-7.39 (m, 5H), 7.51 (d, 2H), 8.05 (m, 2H). Mass: 836.6(Mna+),   812.4 (M-H).<br>
EXAMPLE 12<br>
Synthesis of 4-[Butoxy-(3-hydroxy-phenyl)-phenyl-methyl]-benzoic acid 2,5-dioxo-pyrrolidin-1-yl ester<br><br>
-193-<br><br>
51%<br>
100 mg of 4-[Hydroxy-(3-hydroxy-phenyl)-phenyl-methyl]-benzoic acid (0.31 mM) placed in a 25 mL round bottomed flask was added thionyl chloride (1 mL) and refluxed at 80 Â°C for an hour. Then the excess of SOCI2 was removed under high vacuum and dried. To this dried solid residue was added freshly distilled anhydrous THF (4 mL) under argon atmosphere followed by N-hydroxy succinimide (38 mg, 0.33 mM) and stirred at room temperature for an hour The solvent was removed under high vacuum and dried. Then the residue obtained was dissolved in dry pyridine (1.5 mL) and added 0.2 mL of n-butanol and the reaction mixture was stirred for 3h. The pyridine was removed under high vacuum and solid obtained was purified by silica gel column using hexane/EtOAc (7:3) as a eluent. Evaporation of the solvent afforded 4-[Butoxy-(3-hydroxy-phenyl)-phenyl-methyl]-benzoic acid 2,5-dioxo-<br>
pyrrolidin-1-yl ester (6). Yield: 50 -52%). 1H-NMR, CDCI3 (5 ppm): 0.88 (t, 3h). 1.38 (m. 2H), 1.61 (m, 2H), 2.87 (br. S, 4H), 3.05 (t, 2H), 6.7 (dd, 1H). 6.9 (dd, 2H). 7.16 (t, 1H), 7.3 (m, 5H), 7.64 (d, 2H), 8.04 (d, 2H). Mass: 496 (Mna+), 472(M-H) 400.3.<br>
EXAMPLE 13 <br>
This example shows addition of of a biotin as a sorting function onto a capture compound.<br><br>
-194-<br><br>
C2H50^0<br><br>
HCLJO<br><br><br><br>
HON<br><br>
DCCDMAP<br><br><br><br>
-OH<br><br>
OH<br><br><br><br>
R-,0<br><br>
RTO<br><br>
CH-.C1-,. r.t<br><br><br><br>
o<br>
1<br>
R,<br><br>
24<br><br>
HN    NH<br><br><br>
25<br><br><br><br>
Ov^O-N<br>
0<br>
OH<br><br>
AcC'l. I olucnc .reflux<br><br>
vÂ°-N&gt;<br>
0<br>
Pi'<br><br>
1    HO^-^^OTHP Pyridine<br>
ii. AcOH.<br><br>
R^<br><br>
26<br><br>
27<br><br><br><br><br>
sv ..*-<br>
H4-4H<br>
HN..NH<br>
I O<br><br>
H      6<br><br>
01 /0-Ns<br>
28<br>
d<br><br>
.OH<br><br>
EXAMPLE 14<br>
Capture and pull-down for target protein from HEK293 cellular fractions with doped carbonic anhydrase II <br>
Materials needed: 20 mM Hepes buffer, pH 7.2.<br><br>
-195-<br>
Add 200 jj 20 mM Hepes, pH 7.2, to reconstitute lyophilized Carbonic anhydrase II (Sigma). Transfer to ah eppedorf tube. Calculate<br>
concentration of working stocks (see later in the protocol) and make the<br>
stocks using the same buffo: and the master stock. Freeze master stock for<br>
long term storage.<br>
HEK293 cellular fractions are FPLC fractionated and multiple fractions collected along the salt gradient.<br>
Dissolve capture compound in 10 mM DMSO stock Make working stock of capture compound A in methanol. Make new stock every week and keep it on ice with aluminum foil to protect it from light.<br>
Pierce spin columns (about 500 ul bed volume). It handles as little as 20 ^l and up to 100 jj sample.<br>
Soft-Link (avidin) resin:. Wash the resin 3x1 ml (for a 100 p.1 resin aliquot) in 20 mM Hepes, pH 7.2. Care should be taken to maintain the right solid/liquid ratio at the end of washing in order to be consistent in the amount of resin used in pull-down experiments.<br>
Washing buffer for pull-down: Hepes/NaCI/TX100/EDTA/DTT. Make the buffer stock with the first 4 components at the right concentration and pH first, then separately make 1M DTT stock and freeze it down in small aliquots until use. Right before the washing procedures in the pull-down experiment (step H), thaw a DTT stock tube and add DTT stock at the required final. Each pull-down tube requires ~1ml washing buffer).<br>
Sigma mass quality water.<br>
Experiment protocol:<br>
A.	In a well on a reaction plate, pipett 25 ul FT293, x ^l of Carbonic<br>
Anhydrase II stock, y jj of compound stock solution, and 25-x-y ^l of 20 mM<br>
Hepes buffer, pK 7.2. Keep the y value at 2.5 ul or less for a 50 ul reaction.<br>
The FT fraction h the mixture is diluted 2 fold in the final mixture. For S100,<br>
more than 3-fold dilution is required.   In certain embodiments, use 15 ul for<br>
S100 in a 50 ul reaction and change the buffer volume accordingly.<br>
B.	Mi), the three thoroughly by pipetting up and down 3x.<br><br>
-196-<br>
C.	Incubate the reaction mixture in the dark at room temperature<br>
for 30 min.<br>
D.	Carry out photoreaction after the incubation. Care must be<br>
taken so as to not to excessively heat the microtiter plate upon flashes from<br>
high intensity broadband photography flash lamp (B1600 from Alien Bees).<br>
Use a total of -20 - 40 shots.<br>
E.	Spin column processing of sample after photoreaction is not<br>
necessary for mixture that has the capture compound around 1 uM. For<br>
reactions using more than 10 uM compound, spin-column processing before<br>
binding can improve the target signal in pull-down.<br>
F.	Isolate captured protein using biotin/avidin.. Wash Soft-Link<br>
resin as above; do not pre-treat with biotin. For each binding and pull-down,<br>
into one PCR tube on a strip, add 5 ul slurry of resin after mixing the resin and<br>
the liquid on top thoroughly, then add 20 ul reaction mixture after<br>
photoreaction or after spin-column. Care should be taken to make sure that<br>
the tip is at the bottom of the tube before releasing the contents, and the<br>
pipettman tips should not touch the inside wall of the tube, especially the top<br>
part. Rotate the binding tube for 30 min at room temperature.<br>
G.	Spin tubes 2 min in the centrifuge. Carefully take the<br>
supernatant out. Try to take as much liquid out as possible without losing any<br>
resin.<br>
H.       Add 200 ul washing buffer into each tube, rotate for 4 min on the same setting. Make sure the resins and liquid are well mixed during the process.<br>
I.	Spin and remove supernatant as described in step G.<br>
J.        Following 4x washes by the washing buffer, switch to water, carry out another 4x washes. After the last wash in water, completely take out the supernatant, add 2 ul water on top.<br>
K.       Mix the resin and water well, take 1 ul onto a mass plate spot, give 1 or 2 minute' to air dry the spot a bit (not completely dry), add 1 ul of matrix, pipett up i nd down 4 times.<br><br>
-197-<br>
L.       If SDS-PAGE is required for the sample, silver staining (Invitrogen's Silver Quest Kit) may be used to detect proteins in the pull-down. Usually half of the pull-down resin is eluted with .SDS-PAGE sample buffer for this purpose<br>
EXAMPLE 15 DETERMINATION OF BINDING STRENGTHS (Dissociation Constants).<br>
this approach is based on the observation that photolysis acts on a very fast time scale, from activation to covalent cross linking (ns to ms, depending on the photoactive moiety ). One can thus envision using photolysis to take a snap shot of a enzyme-substrate complex mixture in equilibrium   The amount of covalently crosslinked enzyme-substrate is directly proportional to that of the enzyme-bound substrate (capture compound) in equilibrium. Most importantly, this amout as a fraction of that of the starting enzyme can be very easily and reliably measured by using an off-the-shelf Maldi Machine following a pulldown step.<br>
Equilibrium Analysis.<br>
The starting point of the analysis is the definition of the dissociation constant.<br>
Kd= [S][E]/[SE]<br>
where [S] , [E] and [SE] are the concentrations of the free substrate, free enzyme and substrate-enzyme complex respectively. To make this equation more useful, one can rewrite the equation using variables that are more immediately measurable, such as:<br>
[S c] = beginning concentration of substrate.<br>
[E o] =beginning concentration of enzyme.<br>
Thus we have<br>
Kd= ([So]-[SE])([E0]-[SE])/[SE].<br>
This is a simple quadratic equation which yields the concentration of the complex as a simple function of K d, S o and E o.<br>
[SE] = 1/2 (S o + E o + K d - Sqrt ( (S o + E o + K d)**2 - 4 S o E o ))<br>
One can further simplify the equation with the assumption that the<br><br>
-198-<br>
substrate concentration is much higher than the complex concentration, i.e. ( [S o ] Â» [SE]). In this case, we simply have<br>
[SE] = E0/(1+Kd/[S0]).<br>
After Photolysis.<br>
The central assumption is that the photolysis process is a very rapid process so that the amount of the covalently crosslinked substrate enzyme complex is directly proportional to the amount of the complex in equilibrium, i.e we are indeed taking just a snap shot of the equilibrium concentrations.<br>
Let u be the conversion efficiency of bound complex to covalently crosslinked complex, The concentration of the covalently crosslinked complex is thus a[SE].<br>
After Pulldown.<br>
If the substrate is a biotinylated compound, then a pull down experiment will isolate the covalently captured complex. Let the pulldown efficiency be p. Then the peak area, A of this complex in a Maldi gives a direct measurement concentration of the pulldown complex<br>
A=   p*a*E0/(1 +Kd/[S0]).<br>
Absolute K a Measurement<br>
From the above equation, one can now obtain a very simple relationship between A and the initial concentration of the substrate:<br>
ln(A) = ln(p) +ln(a) +ln(E o) - ln(1 +K d / [S 0]).<br>
Further assuming that K d Â«[S o], we finally have<br>
ln(A) = ln(p) +ln(a) +ln(E o) - K â / [S 0].<br>
Thus by plotting ln(A) vs 1/[S o], we can obtain K dfrom the slope of the linear fit.<br>
N.B. External Standard might be ,needed to normalize the spectra taken from samples with different values of [S o].<br>
K d Difference Measurement<br>
In the case where the use of external standard is unavailable or undesirable, one can still obtain a measurement of the difference in Kd's.<br>
Suppose that there are 2 species of enzymes that are being captured, pulled-<br><br>
-199-<br>
down and mass-spected. For a very selective compound, it is reasonable to<br>
assume that their photolytic and pulldown efficiencies are also very similar.<br>
1	2<br>
Let their dissociation constants be Kd and Kd , their initial enzyme concentrations E 01 and E 02 . their Maldi peak areas A1 and A2 respectively. We have<br>
ln(A1/ A2) = ln(E 01/ E 02) -( K d1 - K d2)/ [S 0].<br>
Thus by plotting the natural log of the relative areas against 1/[S o], the difference in dissociation contants. (Kd1 - Kd2) can be determined directly from the slope of the linear fit. The appealing feature of this analysis is that since we are dealing with relative areas, there is no need to normalize the areas from different spectra.<br>
EXAMPLE 16 (PROPHETIC) ORAL HYPOGLYCEMICS/ANTIDIABETICS:<br>
Thiazolidinediones       (Glitazones):       Troglitazone       (Rezulinâ¢) Rosiglitazone (Avandiaâ¢) and Pioglitazone (Actosâ¢)<br>
I. Development and Pharmacology<br>
â¢        Troglitazone 'Rezulinâ¢) was the first thiazolidinedione<br>
marketed and was indicated for insulin-resistant patients who are<br>
receiving insulin and also as monotherapy. Troglitazone has since<br>
been removed from the market due to concerns of hepatic toxicity.<br>
However two new "glitazones" have been approved in recent years<br>
and these drugs specifically targets insulin resistance. Each of these<br>
new glitazone also have side effects.<br><br>
-200-<br><br><br><br>
cn<br><br><br>
TlOglitilZODfr<br><br>
Rosislitazone<br><br><br><br><br>
/    ^<br>
CKjCK-<br><br>
"X<br><br>
&gt;=o<br><br>
Pioglitnzone<br>
â¢	The thiazolidinediones are dependent on the presence of insulin for activity, however, they do NOT affect insulin secretion. The thiazolidinediones are highly selective and potent agonists for the peroxisome proliferator activated receptor (PPAR) gamma that regulates the transcription of a number of insulin responsive genes. PPAR receptors can be found in key target tissues for insulin action such as adipose tissue, skeletal muscle, and liver Activation of PPAR-gamma receptors regulates the transcription of insulin-responsive genes involved in the control of glucose production, transport, and utilization. For example, stimulation of these receptors may result in increased production of GLUT1 and GLUT 4 receptors. Additionally, PPAR-gamma responsive genes also play a role in the regulation of fatty acid metabolism Unlike oral sulfonylureas, rosiglitazone enhances tissue sensitivity to insulin rather than stimulates insulin secretion. Also, based on this mechanism, it may take several weeks for these drugs to fully express their activity (and thus to assess their potential).<br>
â¢	Preclinical studies indicate that these drugs decrease hepatic glucose output and increase insulin-dependent glucose disposal in skeletal muscle. In animal models of diabetes, these drugs reduce the hyperglycemia, hyperinsulinemia and hypertriglyceridemia characteristic of insulin resistant states such as NIDDM.<br>
II. Adverse Reactions:<br>
â¢	Minimal hypoglycemia: Hypoglycemia was observed in relatively<br><br>
-201-<br>
few glitazone-treated patients to date. Aggressive insulin dosing in combination with glitazone is associated with further reductions in HbA1c but with an increased risk of hypoglycemia.<br>
â¢	In contrast to troglitazone no svidence of drug-induced hepatotoxicity was noted in clinical studies of pioglitazone or rosiglitazone. However the FDA recommends monitoring hepatic function at the start of glitazones therapy and every two months during the first year of treatment. Patients should also be advised to monitor for signs and symptoms suggestive of hepatic dysfunction such as nausea, vomiting, abdominal pain, fatigue, anorexia, dark urine, or jaundice.<br>
â¢	Edema, hypoglycemia, paresthesias, and elevations of creatinine phosphokinase (CPK) have occurred in some pioglitazone-treated patients. Reductions in hemoglobin and hematocrit have also been observed. Glitazone therapy is not recommended for Class III and IV CHF patients and close monitoring of the fluid status of Class I and II patients is necessary.<br>
â¢	Glitazone-treat patients may experience weight gains in the range of 1 to 4 kg may occur perhaps improved due to glucose control. The glitazones are reported to produce increases in low-density lipoprotein-cholestercl (LDL-C), high-density lipoprotein-cholesterol (HDL-C), and total cholesterol LDL-C is increased the least with pioglitazone. The LDL/HDL ratio is preserved, although with rosiglitazone. there is a lag time of several months before HDL-C rises relative to LDL-C. Triglycerides decrease with troglitazone and pioglitazone, whereas the effect with rosiglitazone is variable.<br>
â¢	AvandiaÂ® and ActosÂ® used to treat Type-ll diabetes can cause fluid buildup and heart failure in some patients. U.S. doctors said on September 9. 2003 (Reuters)<br>
â¢	AvandiaÂ® and ActcsÂ® caused heart failure in six male patients with poor heart and kidney function.<br>
â¢	studies indicate the t the incidence of hypoglycemia may be increased when glitazones are used with a sulfonylurea. Currently there are no controlled published studies c n the hypoglycemic effects of troglitazone<br><br>
-202-<br>
with the biguanides or alpha-glucosidase inhibitors.<br>
â¢	Oral contraceptives: Pioglitazone may induce the metabolism<br>
and reduce efficacy of OCs (some controversy over this interaction). Use additional protection or switch to rosiglitazone which does not alter OC<br>
clearance.<br>
ToxPro Objectives<br>
â¢	Structural classification (i.e. Thiazolidinediones) and sub-classification (i.e. generation)<br>
â¢	Identify key structural features that contribute to pharmacologic/therapeutic profile and differences in activity within a structural subclass (i.e. Thiazolidinediones)<br>
â¢	Detailed understanding of the mechanism of action for each drug/drug class.<br>
Pancreatic and/or extra-pancreatic mechanism(s)? Insulin dependent or independent action Compare drugs from different structural classes in terms of mechanism<br>
â¢	Relative efficacy within a structural series (i.e. Thiazolidinediones) and across series.<br>
â¢	Key disposition factors (protein binding)<br>
â¢	Relative onset of action and relationship to mechanism or other factors<br>
â¢	Metabolic processes and activity of metabolites (contribution to therapeutic activity)<br>
â¢	Elimination profile: Renal and/or non-renal as parent drug and/or metabolites?<br>
â¢	Use/cautions in renally or hepatically impaired patients due to non-target protein binding<br>
â¢	Adverse reactions:<br>
Relative incidence of hypoglycemia and relationship to mechanism of action, duration of actior, etc.<br><br>
-203-<br>
Weight gain <br>
Gl effects<br>
Effects on renal physiology<br>
Other key agents: i.e. lactic acidosis<br>
Similarities and differences within a series (i.e.<br>
Thiazolidinediones) and between structural series in key adverse reactions<br>
â¢	Significant drug interactions that may compromise efficacy:<br>
Pharmacokinetic-based interactions: Interference with Absorption,<br>
Metabolism/Cytochromebased interactions, Competition for elimination, etc.<br>
Pharmacologic: Use with other drugs with hypoglycemic or hyperglycemic actions.<br>
Similarities and differences within a series (i.e. Thiazolidinediones) and between structural series for key drug interactions. <br>
ToxPro Application<br>
The peroxisome proliferator-activated receptor-y (PPAR- y): potential role for insulin resistance and p-ce// function. Thiazolidinediones are pharmacological compounds that reduce insulin resistance both in prediabetic as well as diabetic individuals.<br>
Thiazolidinediones are ligands of the PPAR-y 2. PPAR-y 2 is predominantly<br>
expressed in adipocytes, intestine, and macrophages. There is some evidence that a low level expression might also occur in muscle cells. The PPAR-y receptor is a transcription factor that controls the expression of numerous genes. It is assumed that the effect of thiazolidinediones on insulin sensitivity is mediated through altered expression of PPAR-y 2- dependent genes.<br>
As discussed above, thiadolidinediones, as antidiabetic drugs, clearly show toxicity and undesirable side effects. Thiazolidinediones (Glitazones): Troglitazone (Rezulinâ¢) Rosiglitazone (Avandiaâ¢) and Pioglitazone (Actosâ¢) will be attached to the "Capture Compound (CC)."<br><br>
-204-<br>
The CC-Thiazolidinediones will be incubated with kidney, liver, pancreatic,<br>
colonic epithelium and muscle cells, Rezulin, Avandia and Actos should<br>
capture PPAR-y, PPAR-a as well as non-target proteins. These three drags have different metabolism and pharmacokinetics, therefore it is expected that they should capture different non-target proteins. As discussed above, antidiabetic activity of thiazolidinediones is caused by binding to PPAR-y protein. Structure Activity Relationship (SAR) of thiazolidinediones and crystal structures of and PPAR-a co-crystallized with thiazoiidinediones is known in the literature.<br>
The undesired and toxic side effects of thiazolidinediones could be due to ;ts interaction with PPAR-a and non-target proteins. The ToxPro application of CCMS will be used to identify all proteins which bind to each drug and their respective binding constants. After identifying non-target proteins with CCMS technology, the thiazolidinediones will be chemically re-engineered, through an iterative process, to prevent their binding to PPAR-a and non-target proteins while maintaining the interaction with the target protein PPAR-y.<br>
Rezulin:<br>
Rezulin is attached to the Capture Compound as depicted below:<br><br>
-205-<br><br>
HO<br><br>
O<br><br><br>
CF3	0-'" NH			0<br>
NN	oÂ°	V	^(X/\	H 0      ^<br>
o	N, 6			<br>
0	sStructure 1	1		<br>
0				o<br>
CF,             NH'    NH      .0      n N N        .     00    Ou	0	. NH. 0	NH^H^ s<br>
0				<br>
0	Structure	ii		<br><br>
NH<br>
- H<br><br>
Rezulin is metabolized in the liver to its p-Hydroxy glucose and sulfate complexes. Therefore Structure II is considered.<br>
Rezulin Capture Compound Structures I and II are incubated with kidney, liver pancreatic, colon epithelium, and muscle cells. The target protein PPAR-y as well as non-target protein PPAR-a and protein A, B and C are captured.<br>
Avandia and Its Metabolite:<br>
Avandia is attached to the capture compound as depicted below:<br>
O	O<br>
CF3 -^^NH	&gt;&gt;K|H<br><br>
*0<br>
*^0	O     *^-s<br><br>
Â°-rN^o<br>
Avandia metabolizes to aromatic hydroxy metabolites. Therefore two possible metabolites are attached to the capture compound as depicted below:<br><br>
-206-<br><br><br>
CF<br>
N-N<br><br>
O<br>
NH<br>
CTt)0<br><br>
r<br>
{<br>
P<br>
3<br>
VNH o<br><br>
CF-  ^<br><br>
0<br>
^NH<br><br>
0<br><br>
NH<br><br><br><br>
N~N       ^   0<br><br><br>
o<br><br><br><br><br><br>
Metabolite<br><br>
('   <br>
O<br><br>
-207-<br><br>
H-T<br>
^Â»jfj&gt;uu.    n   .    .   n   -    MU.     ^<br><br>
NN   /.0o	6   ^s-<br><br><br><br>
^	"^	Metabolite<br>
HO<br><br><br>
Xf<br>
z^N fi -OX/^n^O<br>
N-N<br>
0 CF3 ^^NH<br>
-0Â°<br>
0YNv0<br><br>
HO    ^<br><br>
/<br><br>
Metabolite<br><br>
Avandia and its metabolites attached to the Capture Compound are incloated with kidney, liver, pancreatic, colon epithelium, and muscle cells. The target protein PPAR-y as well as non-target protein PPAR-a and protein A. B and C are captured.<br>
Actos and Its Metabolites:<br>
Actos is attached to the Capture Compound as depicted below:<br><br>
-208-<br><br><br>
CP3<br>
HN^S<br><nn></nn><br>
N    &gt;-0-l     '<br><br>
N<br><br>
t&gt;<br><br><br><br>
O-<br><br>
N-N<br>
3<br>
D^^JD<br><br>
"O<br><br><br><br>
P<br>
o   o<br><br>
O^NH<br>
X)<br><br><br>
' H, Nhf<br><br>
CX<br><br>
a<br><br>
NH<br><br>
H,Nlfu<br>
S-AH<br>
Actos' possible metabolite is attached to the capture compound as depicted below:<br><br>
-209-<br><br><br><br>
0    Â°O^NH<br>
CX<br>
^NH<br>
CT&gt;<br>
\ NlV3<br><br>
Actos and its metabolites attached to the Capture Compound are incubated with kidney, liver, pancreatic, colon epithelium, and muscle cells. The target protein PPAR-y as well as non-target protein PPAR-a and protein A, B and C are captured.<br>
Since modifications will be apparent to those of skill in this art, it is intended that this invention be limited only by the scope of the appended claims.<br><br>
2.10<br>
WE CLAIM<br>
1.	A method, comprising:<br>
i) contacting a capture compound with a sample containing biomolecules to effect capture of biomolecules in the sample, where the capture compound<br>
has formula:<br>
Q	z	(X)m<br>
(Y)n<br>
X is selected to covalently bind to biomolecules or to bind with sufficiently high affinity so that the resulting complexes of biomolecule/capture compounds are stable under conditions of mass spectrometric analysis;<br>
Y is a pharmaceutical drug, drug fragment, drug intermediate, drug metabolite or prodrug selected to increase the selectivity of the binding by X such that the capture compound binds to fewer biomolecules when the selectivity moiety Y is present than in its absence;<br>
Q is a sorting function;<br>
m is an integer that is 1 to 100;<br>
n is an integer from 1 to 100; and<br>
ii) isolating and identifying the captured biomolecules.<br>
2.	The method of claim 1, wherein X is a latent reactivity group requiring activation following contacting with the biomolecules to allow for reaction with the biomolecules.<br>
3.	The method of any of claims 1-2, wherein the moiety X of the capture compound comprises an azide, diazirine, or a group which, following activation, reacts with the biomolecule.<br>
4.	The method of any of claims 1-2, wherein each X is selected from the group consisting of an active ester, an active halo moiety, an amino acid side<br>
chain-specific functional group, a specific peptide that binds to a biomolecule surfaces, an a-halo ether, an a-halo carbonyl group, maleimido, a metal<br><br>
211<br>
complex, an expoxide and an isothiocyanate.<br>
5.       The method of any of claims 1-4, wherein the biolomolecules comprise proteins.<br>
5        The method of claim 5, wherein the contacting step is performed under conditions whereby the interactions of the moiety Y of the capture compound with proteins in the sample reach equilibrium.<br>
7.	The method of claim 6, wherein after equilibrium the mixture is treated to form a covalent bond between the capture compound and the proteins<br>
8.	The method of claim 7, wherein the treatment comprises activation with light.<br>
9.	The method of claim 7, wherein the treatment comprises change in PH.<br>
10.	The method of any of claims 1 -9, wherein a concentration of the capture compound is varied in a plurality of different reactions.<br>
11.	The method of any of claims 1-10, wherein a Kd value is determined.<br>
12.	The method of claim 5. wherein the contacting step is performed under conditions, whereby the interactions of the moiety Y of the capture compound with proteins in the sample are kinetically controlled.<br>
13.	The method of any of claims 1-12, wherein the drug is linked to the moiety Z in different orientations via different points of attachments on the drug molecule.<br>
14.	The method of any of claims 1-13, further comprising re-designing the drug to eliminate or alter its binding interactions with a captured biomolecule.<br>
15.	The method of claim 14. wherein the method is repeated with a redesigned drug linked to the capture compound to effect further modification thereof.<br>
16.	The method of any of claims 1 -15, wherein the sample is contacted with a collection of capture compounds.<br><br>
212<br>
17.	The method of any of claims 1-16, further comprising, detecting or identifying a captured biomolecule by mass spectrometric analysis.<br>
18.	The method of claim 17, wherein the mass spectrometry format is selected from among matrix assisted laser desorption ionization (MALDI), continuous or pulsed electrospray (ES) ionization, ionspray, thermospray, and massive duster impact mass spectrometry.<br>
19.	The method of claim 18, wherein the detection format is linear time-of-flight (TOF), reflectron time-of-flight, single quadruple, multiple quadruple, single magnetic sector, multiple magnetic sector, Fourier transform, ion cyclotron resonance (ICR), or ion trap.<br>
20.	The method of claims 1 -19. further comprising identifying a function of the captured biomolecule.<br>
21.	The method of claim 20, wherein the function of a biomolecule is determined by sequence alignment, pharmacophores, homology models and protein motif correlation, liver midrosomes metabolic pathways, cDNA-expressed enzymes, signal pathways and back-mapping to yeast pathways, simulations and protein/protein interaction of pull-out proteins, native polymorphisms, knockout/knock-in, flow cytometry, therapeutic activity of the drug, or prospective genotyping and prospective phenotyping<br>
22.	The method of any of claims 1 -21, wherein the sample comprises a biological sample, body tissue or fluid or cell lysate.<br>
23.	The method of claim 14, wherein the captured biomolecule for which binding is altered is a drug target protein.<br>
24      The method of claim 14, wherein the captured biomolecule for which binding is altered is a drug non-target protein.<br>
25.	The method of any of claims 14-24, wherein redesigning the drug<br>
results in a second drug with fewer side-effects or an increased therapeutic index<br>
as compared to the first drug.<br>
26.	The method of any of claims 1-25, wherein X is a photoactivatable<br>
group.<br><br>
213<br>
27.     The method of any of claims 1-26, wherein X is a group selected from the list of figure 16 or X is<br><br>
\3"     -o<br><br>
2C.      The method of any of claims 1-27, wherein Y is selected from a drug or drug intermediate/fragment set forth in figure 17.<br>
29.	The method of any of claims 1 -27, wherein the moiety Y is selected from among troglitazone, rosiglitazone, pioglitazone, methotrexate, atorvastatin, celecoxib. refecoxib and cerivastatin.<br>
30.	The method of any of claims 1-29, wherein Q is an<br>
oligonucleotide or oligonucleotide analog that includes a single-stranded portion of sufficient length "j" to form a stable hybrid with a base-complementary single stranded nucleic acid molecule or analog.<br>
31.	The method of any of claims 1 -30, wherein Q has formula N1sBiN2u,<br>
wherein:<br>
N1. B and N2 are oligonucleotides or oligonucleotide analogs comprising s, i and u members, respectively;<br>
B is a region of sequence permutations that contains at least two bases;<br>
and<br><br>
214<br>
sum of s, i and u is at least 5.<br>
32.	The method of claim 31, where B is a single stranded DNA or RNA and the number of sequence permutations is equal to 4', and i is about 2 to about 25.<br>
33.	The method of claim 31 or 32, where i is about 3 to about 5, 6, 7 or 8.<br>
34.	The method of any of the claims 1-33, wherein Q is biotin.<br>
35.	The method of any of claims 1-34, wherein Z is a moiety that is cleavable prior to or during mass spectrometric analysis of biomolecules bound to the capture compound.<br>
36.	The method of any of claims 1-34, wherein Z is a moiety that is not cleavable prior to or during mass spectrometric analysis of biomolecules bound to the capture compound.<br>
37.	The method of any of claims 1-35, wherein Z is a photocleavable, acid cleavable, alkaline cleavable, oxidatively cleavable, or reductively cleavable group.<br>
38.	The method of any of claims 1-34 and 36, wherein Z has the formula:<br>
(S1)tM(R15)a(S2)b, wherein:<br>
S1 and S2 are spacer moieties;<br>
t and b are each independently 0 or 1;<br>
a is an integer from 0 to 4;<br>
M is a central moiety possessing three or more points of attachment;<br>
each R15 is a monovalent group independently selected from Y2R18;<br>
each Y2 is a divalent group independently having any combination of the following groups: a direct link, arylene, heteroarylene, cycloalkylene, &gt;C(R17)2,<br>
C(R17)=C(R17), &gt;C=C(R23)(R24), &gt;C(R23)(R24), C=C1, 0, &gt;S(A)Ul &gt;P(D)V(R17), &gt;P(D)V(ER17), &gt;N(R17), &gt;N(COR17), &gt;N+(R23)(R24), &gt;Si(R17)2 and &gt;C(E); where u is 0,1 or 2; v is 0,1.2 or 3; A is O or NR17; D is S or O; and E is S, 0 or NR17;<br><br>
215<br>
that groups can be combined in any order;<br>
R17 and R18 are each independently selected from the group consisting of<br>
hydrogen, halo, pseudohalo, cyano, azido, nitro,<br>
SiR27R28R25, alkyl, alkenyl, alkynyl, haloalkyi, haloalkoxy, aryl, aralkyl, aralkenyl, aralkynyl, heteroaryl, heteroaralkyl, heteroaralkenyl, heteroaralkynyl, heterocyclyl, heterocyclyialkyl, heterocyclylalkenyl. heterocyclylalkynyl, hydroxy, alkoxy, aryloxy, aralkoxy. heteroaralkoxy and NR19R20;<br>
R1S and R2D are each independently selected from hydrogen, alkyl, alkenyl, alkynyl, cycloalkyi, aryl, aralkyl, heteroaryl. heteroaralkyl and heterocyclyl;<br>
R23 and R24 are selected from (i) or (ii) as follows:<br>
(i) R23 and R"4 are independently selected from the group consisting of hydrogen, alkyl, alkenyl. alkynyl, cycloalkyi, aryl and heteroaryl; or<br>
(ii) R23 and R24 together form alkylene, alkenylene or cycloalkylene;<br>
R25. R27 and R28 are each independently a monovalent group selected from hydrogen, alkyl, alkenyl, alkynyl, haloalkyi, haloalkoxy, aryl, aralkyl, aralkenyl, aralkynyl, heteroaryl, heteroaralkyl, heteroaralkenyl, heteroaralkynyl, heterocyclyl, heterocyclyialkyl, heterocyclylalkenyl. heterocyclylalkynyl, hydroxy, alkoxy, aryloxy, aralkoxy. heteroaralkoxy and NR1BR20;<br>
R15, R17, R18, R19. R20, R23, R24, R25, R27 and R28 can be substituted with one or more substituents each independently selected from Z2; Z2 is selected from alkyl, alkenyl, alkynyl, aryl, cycloalkyi, cycloalkenyl, hydroxy, S(0)hR35; h is 0, 1 or 2, NR35R36, COOR35, COR35, CONR35R36, OC(0)NR35R36, N(R35)C(0)R36, alkoxy, aryloxy, heteroaryl, heterocyclyl, heteroaryloxy, heterocyclyloxy, aralkyl, aralkenyl, aralkynyl, heteroaralkyl, heteroaralkenyl, heteroaralkynyl, aralkoxy, heteroaralkoxy, alkoxycarbonyl, carbamoyl, thiocarbamoyl, alkoxycarbonyl, carboxyaryl, halo, pseudohalo, haloalkyi and carboxamido; and<br>
R35 and R36 are each independently selected from among hydrogen, halo,<br>
pseudohalo, cyano, azido, nitro, trialkylsilyl, dialkylarylsilyl, alkyldiarylsilyl, triarylsilyl, alkyl, alkenyl, alkynyl, haloalkyi, haloalkoxy, aryl, aralkyl, aralkenyl,<br><br>
216<br>
aralkynyl, heteroaryl, heteroaralkyl, heteroaralkenyl, heteroaralkynyl, heterocyclyl, heterocyclylalkyi, heterocyclylalkenyl, heterocyclylalkynyl, hydroxy,<br>
alkoxy, aryloxy, aralkoxy, heteroaralkoxy, amino, amido, alkylamino,<br>
dialkylamino, alkylarylamino, diarylamino and arylamino.<br>
39.      The method of any of claims 1-35 and 37, wherein Z has the<br>
formula:<br>
Z has the formula: (S')tM(R15)a(S2)bL, wherein:<br>
S1 and S2 are spacer moieties;<br>
t and b are each independently 0 or 1;<br>
a is an integer from 0 to 4;<br>
M is a centrai moiety possessing three or more points of attachment;<br>
each R15 is a monovalent group independently selected from Y2R18;<br>
each Y2 is a divalent group independently having any combination of the following groups: a direct link, arylene, heteroarylene, cycloalkylene, &gt;C(R17)2, C(R"&gt;C(R17), &gt;C=C(R23)(R?4), &gt;C(R23)(R24), CHC, O, &gt;S(A)U. &gt;P(D)V(R17), &gt;P(D)V(ER17), &gt;N(R17), &gt;N(COR17), &gt;N+(R23)(R24), &gt;Si(R17)2 and &gt;C(E); where u is 0. 1 or 2: v is 0, 1, 2 or 3; A is O or NR17; D is S or O; and E is S, O or NR17'; that groups can be combined in any order;<br>
R1' and R18 are each independently selected from the group consisting of hydrogen, halo, pseudohalo, cyano, azido, nitro,<br>
SiR2 R28R25, alkyl, alkenyl, alkynyl, haloalkyi, haloalkoxy, aryl, aralkyl, aralkenyl, aralkynyl, heteroaryl, heteroaralkyl, heteroaralkenyl, heteroaralkynyl, heterocyclyl, heterocyclylalkyi, heterocyclylalkenyl, heterocyclylalkynyl, hydroxy, alkoxy, aryloxy, aralkoxy, heteroaralkoxy and NR19R20;<br>
R1S and R20 are each independently selected from hydrogen, alkyl, alkenyl, alkynyl, cycloalkyl, aryl, aralkyl, heteroaryl, heteroaralkyl and heterocyclyl;<br>
R23 and R24 are selected from (i) or (ii) as follows:<br><br>
217<br>
(i) R23 and R24 are independently selected from the group consisting of hydrogen, alkyl, alkenyl, alkynyl, cycloalkyi, aryl and heteroaryl; or<br>
(ii) R   and R   together form alkylene, alkenylene or cycloalkylene;<br>
R25, R27 and R28 are each independently a monovalent group selected from hydrogen, alkyl, alkenyl, alkynyl, haloalkyl, haloalkoxy, aryl, aralkyl, aralkenyl, aralkynyl, heteroaryl, heteroaralkyl, heteroaralkenyl, heteroaralkynyl, heterocyclyl, heterocyclylalkyi, heterocyclylaikenyl, heterocyclylalkynyl, hydroxy, alkoxy, aryloxy, aralkoxy, heteroaralkoxy and NR19R20;<br>
R15, R17, R18, R19, R20, R23, R24, R25, R27 and R28 can be substituted with one or more substituents each independently selected from Z2; Z2 is selected from alkyl, alkenyl, alkynyl, aryl, cycloalkyi, cycloalkenyl, hydroxy, S(0)hR35; h is 0, 1 or 2, NR35R36, COOR35, COR35, CONR35R36, OC(0)NR35R36, N(R35)C(0)R36, alkoxy, aryloxy. heteroaryl, heterocyclyl, heteroaryloxy, heterocyclyloxy, aralkyl, aralkenyl, aralkynyl, heteroaralkyl, heteroaralkenyl, heteroaralkynyl, aralkoxy, heteroaralkoxy, alkoxycarbonyl, carbamoyl, thiocarbamoyl, alkoxycarbonyl, carboxyaryl, halo, pseudohalo, haloalkyl and carboxamido;<br>
R35 and R36 are each independently selected from among hydrogen, halo, pseudohalo, cyano, azido, nitro, trialkylsilyl, dialkylarylsilyl, alkyidiarylsilyl, triarylsilyl, alkyl, alkenyl, alkynyl, haloalkyl, haloalkoxy, aryl, aralkyl, aralkenyl,, aralkynyl, heteroaryl, heteroaralkyl, heteroaralkenyl, heteroaralkynyl, heterocyclyl, heterocyclylalkyi, heterocyclylaikenyl, heterocyclylalkynyl, hydroxy, alkoxy, aryloxy, aralkoxy, heteroaralkoxy, amino, amido, alkylamino, dialkylamino, alkylarylamino, diarylamino and arylamino; and<br>
L is a group that is cleavable prior to or during mass spectrometric analysis of the compound.<br>
40.	The method of any of claims 1-39, wherein the capture compounds further comprise a solubility function W that influences the solubility properties of the capture compounds.<br>
41.	The method of any of claims 1-40, wherein capture compounds comprise Z, which contains a functionality conferring luminescence, fluorescence, chemoluminescence or colorimetric properties.<br><br>
218<br>
42.     A collection of capture compounds, comprising a plurality of capture compounds, comprising sets of capture compounds, wherein each set of capture<br>
compounds includes a moiety X that is selected to covalently bind to<br>
biomolecules or to bind with sufficiently high affinity so that the resulting complexes of biomolecule/capture compounds are stable under conditions of mass spectrometric analysis; a moiety Y that increases the selectivity of the binding by X such that the capture compound binds to fewer biomlecules when the selectivity moiety is present than in its absence; and a moiety Z for presenting X and Y, wherein the moiety Z is<br><br>
wherein R15 is H, OH, OR51, SH, SR51, NH2, NHR51, N(R51)2, F, CI, Br, I, S03H, P024, CH3. CH2CH3, CH(CH3)2 or C(CH3)3; where R51 is straight or branched chain alkyl, straight or branched chain alkenyl, straight or branched chain alkynyl, aryl, heteroaryl, cycloalkyl, heterocyclyl, straight or branched chain aralkyl, straight or branched chain aralkenyl, straight or branched chain aralkynyl, straight or branched chain heteroaralkyl, straight or branched chain heteroaralkenyl, straight or branched chain heteroaralkynyl, straight or branched chain cycloalkylalkyl, straight or branched chain cycloalkylalkenyl, straight or branched chain cycloalkylalkynyl, streight or branched chain heterocyclylalkyl, straight or branched chain heterocyclylalkenyl or straight or branched chain heterocyclylalkynyl;<br>
z is an integer from 1 to 4;<br><br>
219<br>
n1, n2, n3 are 0 to 4 with the proviso that all n1, n2 and n3 are not equal to 0 at the same time; and<br>
each set of capture compounds further includes a moiety Q on the moiety Z, such that each set contains a different Q, wherein Q permits separation of each set.<br>
43.     A method for separating protein conformers, comprising:<br>
contacting a composition comprising a biomolecule with a collection of capture compounds of claim 42;<br>
separating the members of the collection; and<br>
identifying the bound proteins from the mixture, whereby each conformer has different binding specificity for members of the collection.<br>
44       A method for identification of phenotype-specific biomolecules, comprising:<br>
sorting cells from a single subject according to a predetermined phenotype to produce at least two separated sets of cells;<br>
contacting mixtures of biomolecules from each set of sorted cells with a collection of capture compounds of claim 42;<br>
comparing the patterns of biomolecules binding from each set to identify biomolecules that differ for each set. thereby identifying phenotype-specific biomolecules.<br>
45.      A method for analyzing biomolecule interactions, comprising:<br>
a)	contacting a mixture of biomolecules with a collection of capture<br>
compounds of claim 42 to form a compound-biomolecule complexes, wherein:<br>
the central core is not cleavable prior to or during mass spectrometric analysis of biomolecules bound to the capture compound; and<br>
the complexes are stable to matrix assisted laser desorption ionization-time of flight (MALDI-TOF) mass spectrometry conditions;<br>
b)	contacting the capture compound-biomolecule complexes with a<br>
mixture containing compounds selected from the group consisting of mixtures of<br><br>
220<br>
biomolecuies and/or small molecules, wherein compounds in the mixture bind to biomolecuies in the complexes based on affinity;<br>
c)	before or after step b) immobilizing the capture compounds on a<br>
solid support via the sorting group of each set of capture compounds;<br>
d)	analyzing the bound compounds by mass spectrometry.<br>
46.	A method of analysis of biomolecuies, comprising:<br>
a)	contacting a composition comprising a biomolecule with a collection<br>
plurality of capture compounds, comprising sets of capture compounds, wherein<br>
each set comprises a capture compound of formula:<br>
Qâz	(X)m<br>
(Y)n<br>
wherein:<br>
X is selected to covalently bind to biomolecuies or to bind with sufficiently high affinity so that the resulting complexes of biomolecule/capture compounds are stable under conditions of mass spectrometric analysis;<br>
Y is a moiety that increases the selectivity of the binding by X such that the capture compound binds to fewer biomlecules when the selectivity moiety is present than in its absence;<br>
Q is such that each set contains a different Q, wherein Q permits separation of each set;<br>
m is an integer that is 1 to 100; and<br>
n is an integer from 1 to 100;<br>
b)	digesting the captured biomolecuies by chemical or enzymatic treatment;<br>
c)	separating each set of captured compounds based on the moiety Q; and<br>
d) analyzing each set of capture compounds to identify the biomolecuies.<br>
47.	A method of analysis of biomolecuies, comprising;<br><br>
221<br>
a)	contacting a composition comprising a biomolecule with a collection of<br>
capture compounds, comprising sets of capture compounds, wherein each set<br>
comprises a capture compound of formula.<br>
Q	Z	(X)"m<br>
(Y)n<br>
wherein:<br>
X is selected to covalently bi;&gt;d to biomolecules or to bind with sufficiently high affinity so that the resulting complexes of biomolecule/capture compounds are stable under conditions of mass spectrometric analysis;<br>
Y is a moiety that increases the selectivity of the binding by X such that the capture compound binds to fewer biomlecules when the selectivity moiety is present than in its absence and where Y is selected from a pharmaceutical drug, drug fragment, drug intermediate, drug metabolite or prodrug;<br>
Q is such that each set contains a different Q, wherein Q permits separation of each set;<br>
m is an integer that is 1 to 100; and<br>
n is an integer from 1 to 100;<br>
b)	separating each set of captured compounds based on the sorting    . moiety Q;<br>
c)	digesting the captured biomolecules by chemical or enzymatic treatment; and<br>
d)	analyzing each set of capture compounds to identify the biomolecules.<br><br>
48.	The method of any of claims 43-47, wherein identification or detection is effected by mass spectrometric analysis.<br>
49.	A method of identifying drug target and drug non-target biomolecules in a mixture of biomolecules, comprising:<br>
i) contacting a capture compound with a sample containing biomolecules<br>
to effect capture of biomolecules in the sample, where the capture compound has formula:<br><br>
222<br>
Qâz	(x&gt;m<br>
(Y)â<br>
X is selected to covalently bind to biomolecules or to bind with sufficiently high affinity so that the resulting complexes of biomolecule/capture compounds are stable under conditions of mass spectrometric analysis;<br>
a moiety Y that increase the selectivity of the binding by X such that the capture compound binds to fewer biomolecules when the selectivity moiety Y is present than in its absence;<br>
Q is a sorting function;<br>
m is an integer that is 1 to 100;<br>
n is an integer from 1 to 100; and<br>
ii) analyzing the captured biomolecules to identify drug targets and drug non-targets.<br>
50.	The method of claim 49, wherein the moiety Y is an enzyme substrate or inhibitor, a co-factor a receptor ligand, a transition state analog, a peptide selected to increase the selectivity of the binding by X such that the capture compound binds to fewer biomolecules when the selectivity moiety Y is present than in its absence.<br>
51.	The method of claims 43-50, wherein Y is selected from among the group listed in figures 17 and 21.<br>
Dated this   12th day of August, 2005.<br><br><br>
-245-<br>
ABSTRACT<br>
Capture compounds and collections thereof and methods using the<br>
compounds for the analysis of biomolecules are provided. In particular, collections, compounds and methods are provided for analyzing complex protein mixtures, such as the proteome. The compounds are multifunctional reagents that provide for the separation and isolation of complex protein mixtures. Automated systems for performing the methods also are provided<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtQUJTVFJBQ1QoQU1FTkRFRCktKDctNS0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-ABSTRACT(AMENDED)-(7-5-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtQUJTVFJBQ1QoR1JBTlRFRCktKDEwLTEwLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-ABSTRACT(GRANTED)-(10-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtYWJzdHJhY3QuZG9j" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-abstract.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtQVNTSUdOTUVOVCAxMS02LTIwMDgucGRm" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-ASSIGNMENT 11-6-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtQVNTSUdOTUVOVCgyMi02LTIwMDYpLnBkZg==" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-ASSIGNMENT(22-6-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtQ0FOQ0VMTEVEIFBBR0VTKDExLTYtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-CANCELLED PAGES(11-6-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtQ0xBSU1TKDE2LTgtMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-CLAIMS(16-8-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtQ0xBSU1TKEdSQU5URUQpLSgxMC0xMC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-CLAIMS(GRANTED)-(10-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtY2xhaW1zLmRvYw==" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-claims.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtQ09SUkVTUE9OREVOQ0UgMTEtNi0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-CORRESPONDENCE 11-6-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtQ09SUkVTUE9OREVOQ0UoMTEtMS0yMDEwKS5wZGY=" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-CORRESPONDENCE(11-1-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtQ09SUkVTUE9OREVOQ0UoMTEtOS0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-CORRESPONDENCE(11-9-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtQ09SUkVTUE9OREVOQ0UoMTUtMTAtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-CORRESPONDENCE(15-10-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtQ09SUkVTUE9OREVOQ0UoNi02LTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-CORRESPONDENCE(6-6-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtQ09SUkVTUE9OREVOQ0UoSVBPKS0oMTktMS0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-CORRESPONDENCE(IPO)-(19-1-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtY29ycmVzcG9uZGVuY2UtcmVjZWl2ZWQtdmVyLTA2MDkwNS5wZGY=" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-correspondence-received-ver-060905.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtY29ycmVzcG9uZGVuY2UtcmVjZWl2ZWQtdmVyLTExMTAwNS5wZGY=" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-correspondence-received-ver-111005.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtY29ycmVzcG9uZGVuY2UtcmVjZWl2ZWQtdmVyLTE2MDgwNS5wZGY=" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-correspondence-received-ver-160805.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtY29ycmVzcG9uZGVuY2UtcmVjZWl2ZWQtdmVyLTE5MDgwNS5wZGY=" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-correspondence-received-ver-190805.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtY29ycmVzcG9uZGVuY2UtcmVjZWl2ZWQtdmVyLTIwMDkwNS5wZGY=" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-correspondence-received-ver-200905.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtY29ycmVzcG9uZGVuY2UtcmVjZWl2ZWQtdmVyLTIyMDYwNS5wZGY=" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-correspondence-received-ver-220605.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtY29ycmVzcG9uZGVuY2UtcmVjZWl2ZWQtdmVyLTIyMDYwNi5wZGY=" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-correspondence-received-ver-220606.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtY29ycmVzcG9uZGVuY2UtcmVjZWl2ZWQtdmVyLTIyMDkwNS5wZGY=" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-correspondence-received-ver-220905.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtY29ycmVzcG9uZGVuY2UtcmVjZWl2ZWQtdmVyLTI0MTAwNS5wZGY=" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-correspondence-received-ver-241005.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtY29ycmVzcG9uZGVuY2UtcmVjZWl2ZWQucGRm" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-correspondence-received.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtREVTQ1JJUFRJT04oQ09NUExFVEUpLSgxNi04LTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-DESCRIPTION(COMPLETE)-(16-8-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtREVTQ1JJUFRJT04oR1JBTlRFRCktKDEwLTEwLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-DESCRIPTION(GRANTED)-(10-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtRFJBV0lORyAxMS02LTIwMDgucGRm" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-DRAWING 11-6-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtRFJBV0lORygxMS02LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-DRAWING(11-6-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtRFJBV0lORygxMS05LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-DRAWING(11-9-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtRFJBV0lORygxNi04LTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-DRAWING(16-8-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtRFJBV0lORyhBTUVOREVEKS0oNy01LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-DRAWING(AMENDED)-(7-5-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtRFJBV0lORyhHUkFOVEVEKS0oMTAtMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-DRAWING(GRANTED)-(10-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtRk9STSAxIDExLTYtMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-FORM 1 11-6-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtRk9STSAxKDExLTYtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-FORM 1(11-6-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtRk9STSAxKDExLTktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-FORM 1(11-9-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtRk9STSAxKDE2LTgtMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-FORM 1(16-8-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtRk9STSAxKDIyLTktMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-FORM 1(22-9-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtRk9STSAxKDctNS0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-FORM 1(7-5-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtRk9STSAxMygxMS0xMC0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-FORM 13(11-10-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtRk9STSAxMygxMS0xMC0yMDA2KS5wZGY=" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-FORM 13(11-10-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtZm9ybSAxMygxMS05LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-form 13(11-9-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtRk9STSAxMyg1LTYtMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-FORM 13(5-6-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtRk9STSAyKENPTVBMRVRFKS0oMTYtOC0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-FORM 2(COMPLETE)-(16-8-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtRk9STSAyKEdSQU5URUQpLSgxMC0xMC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-FORM 2(GRANTED)-(10-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtRk9STSAyKFRJVExFIFBBR0UpIDExLTYtMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-FORM 2(TITLE PAGE) 11-6-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtRk9STSAyKFRJVExFIFBBR0UpLSgxMS05LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-FORM 2(TITLE PAGE)-(11-9-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtRk9STSAyKFRJVExFIFBBR0UpLShDT01QTEVURSktKDExLTYtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-FORM 2(TITLE PAGE)-(COMPLETE)-(11-6-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtRk9STSAyKFRJVExFIFBBR0UpLShDT01QTEVURSktKDE2LTgtMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-FORM 2(TITLE PAGE)-(COMPLETE)-(16-8-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtRk9STSAyKFRJVExFIFBBR0UpLShHUkFOVEVEKS0oMTAtMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-FORM 2(TITLE PAGE)-(GRANTED)-(10-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtRk9STSAzKDEwLTEyLTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-FORM 3(10-12-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtRk9STSAzKDMxLTEtMjAwNikucGRm" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-FORM 3(31-1-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtRk9STSA1KDExLTktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-FORM 5(11-9-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtRk9STSA1KDE2LTgtMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-FORM 5(16-8-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtRk9STSA2KDIyLTYtMjAwNikucGRm" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-FORM 6(22-6-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtZm9ybS0xMy5wZGY=" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtZm9ybS0xOC5wZGY=" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtZm9ybS0yLnBkZg==" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtZm9ybS0yNi5wZGY=" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtZm9ybS02LnBkZg==" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-form-6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtZm9ybS1wY3QtaWItMzA2LnBkZg==" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-form-pct-ib-306.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtZm9ybS1wY3QtaWItMzczLnBkZg==" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-form-pct-ib-373.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtZm9ybS1wY3QtaXBlYS0yMzcucGRm" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-form-pct-ipea-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtZm9ybS1wY3QtaXBlYS1zZXBlcmF0ZSBzaGVldC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-form-pct-ipea-seperate sheet-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtZm9ybS1wY3QtaXNhLTIwMi5wZGY=" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-form-pct-isa-202.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtZm9ybS1wY3QtaXNhLTIwNi5wZGY=" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-form-pct-isa-206.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtZm9ybS1wY3QtaXNhLTIxNy5wZGY=" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-form-pct-isa-217.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtR0VORVJBTCBQT1dFUiBPRiBBVFRPUk5FWSgxMC0xMC0yMDA2KS5wZGY=" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-GENERAL POWER OF ATTORNEY(10-10-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtR0VORVJBTCBQT1dFUiBPRiBBVFRPUk5FWSgyNi0xMC0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-GENERAL POWER OF ATTORNEY(26-10-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLW11bW5wLTIwMDUtcGN0LXNlYXJjaCByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">902-mumnp-2005-pct-search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtUE9XRVIgT0YgQVRUT1JORVkgMTEtNi0yMDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-POWER OF ATTORNEY 11-6-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtU0VRVUVOQ0UgTElTVElORygxNi04LTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-SEQUENCE LISTING(16-8-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtU0VRVUVOQ0UgTElTVElORyhHUkFOVEVEKS0oMTAtMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-SEQUENCE LISTING(GRANTED)-(10-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtU1BFQ0lGSUNBVElPTihBTUVOREVEKS0oNy01LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-SPECIFICATION(AMENDED)-(7-5-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTAyLU1VTU5QLTIwMDUtV08gSU5URVJOQVRJT05BTCBQVUJMSUNBVElPTiBSRVBPUlQoMTYtOC0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">902-MUMNP-2005-WO INTERNATIONAL PUBLICATION REPORT(16-8-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QxLmpwZw==" target="_blank" style="word-wrap:break-word;">abstract1.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="224343-method-of-removing-at-least-partially-a-layer-or-coating-of-material-from-a-substrate.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="224345-self-crimping-fully-drawn-high-bulk-yarns-and-method-of-producing-thereof.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>224344</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>902/MUMNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>06/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>06-Feb-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>10-Oct-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>16-Aug-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CAPROTEC BIOANALYTICS GMBH</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>VILLA WELLINGTONIA, VIA CANTONALE 84, CH-6918,FIGINO</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>KOSTER HUBERT</td>
											<td>VILLA WELLINGTONIA, VIA CANTONALE 84, CH-6918,FIGINO</td>
										</tr>
										<tr>
											<td>2</td>
											<td>LITTLE DANIEL PAUL</td>
											<td>62 CHURCH STREET, WINCHESTER, MA 01980</td>
										</tr>
										<tr>
											<td>3</td>
											<td>SIDDIQI SUHAIB MAHMOOD</td>
											<td>37 UNIVERSITY DRIVE, BURLINGTON, MA 01803</td>
										</tr>
										<tr>
											<td>4</td>
											<td>GREALISH MATTHEW PETER</td>
											<td>7694 SYCAMORE DRIVE, CITRUS HEIGHTS, CA 92129</td>
										</tr>
										<tr>
											<td>5</td>
											<td>MARAPPAN SUBRAMANIAN</td>
											<td>9924 KIKA COURT #2414, SAN DIEGO, CA 92129</td>
										</tr>
										<tr>
											<td>6</td>
											<td>HASSMAN III CHESTER</td>
											<td>5322 W. ESCHE DRIVE, NEWBURGH,IN 47630-2918</td>
										</tr>
										<tr>
											<td>7</td>
											<td>YIP PING</td>
											<td>3641 COPELY AVENUE, SAN DIEGO, CA 92116</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>G01N 33/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2004/001037</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-01-16</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/441,398</td>
									<td>2003-01-16</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/224344-a-method-for-identifying-drug-non-target-biomolecules by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:42:36 GMT -->
</html>
