<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/224300-an-enzyme-exhibiting-aminopeptidase-activity-and-a-dna-construct-comprising-a-dna-sequence-encoding-such-enzyme by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:52:51 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 224300:AN ENZYME EXHIBITING AMINOPEPTIDASE ACTIVITY AND A DNA CONSTRUCT COMPRISING A DNA SEQUENCE ENCODING SUCH ENZYME</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">AN ENZYME EXHIBITING AMINOPEPTIDASE ACTIVITY AND A DNA CONSTRUCT COMPRISING A DNA SEQUENCE ENCODING SUCH ENZYME</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>ABSTRACT 597/MAS/96 An enzyme exhibiting aminopeptidase activity and a DNA construct comprising a DNA sequence encoding such enzyme The present invention relates to an enzyme exhibiting aminopeptidase activity derived from a fungal microorganism, which enzyme comprises one or more of the partial amino acid sequences shown in SEQ ID No. 6, SEQ ID No. 7, SEQ ID No. 8, SEQ ID No. 9, SEQ ID No. 10, or SEQ ID No. 11.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
The present invention relates to an enzyme exhibiting aminopeptidase activity, a method for producing said enzyme, an enzyme preparation containing said enzyme exhibiting aminopeptidase activity, and use of said enzyme for various industrial purposes.<br>
Protein hydrolysates are being used in numerous food prod¬ucts. Traditionally protein hydrolysates were produced by acid hydrolysis, but today enzymatic hydrolysis is regarded as an attractive alternative.<br>
One of the main problems of protein hydrolysates is that they often taste bitter. When using e.g. soy protein or casein, which are rich in hydrophobic L-amino acids, as the protein source, the protein hydrolysate tends to have bitter taste. In general it is believed that whether the taste of proteins is bitter or not depends on the average hydrophobicity of the L-araino acid residues, such as valine, leucine, isoleucine, phenylalanine, tyrosine and tryptophan.<br>
A vast number of enzymes exhibiting peptidase activity are capable of performing enzymatic hydrolysis on vegetable, yeast and/or animal proteins, resulting in highly nutritious protein hydrolysates useful as food additives in products such as soups, sauces, gravies, paste, tofu, bouillon, seasonings, baby formulas, snacks, ready-to-eat meals etc.<br><br>
Peptidases<br>
All peptidases or proteases are hydrolases which act on<br>
proteins or its partial hydrolysate to decompose the peptide<br>
bond.<br>
EP 427,385 (The Japanese R&amp;D Association) discloses a genomic gene encoding an alkaline protease derived from yellow molds such as Aspergillus oryzae.<br>
JP-0-2002374 and JP-0-2002375 (Shokuhin), describes an alkaline protease derived from Aspergillus oryzae for use in medicine, food, and detergents.<br>
SU-891777 (Khark) concerns a microbial protease from Asper¬gillus  oryzae,   which can be used in food, medicine etc.<br>
JP-5-4035283 (Ajinomoto KK) discloses preparation of enzymes from e.g. Aspergillus oryzae exhibiting endopeptidase activ¬ity, which can hydrolyse proteins almost completely.<br>
WO 94/25580 (Novo Nordisk A/S) describes a method for hydrolysing vegetable or animal protein by incubating with a proteolytic enzyme preparation derived from a strain of Aspergillus oryzae.<br>
Aminopeptidases<br>
A subgroup of peptidases (proteases) are called aminopeptidases and are classified under the Enzyme Classifi¬cation number E.G. 3.4.11 (aminopeptidases) in accordance with the Recommendations (1992) of the International Union of Biochemistry and Molecular Biology (lUBMB)).<br>
Aminopeptidases are capable of removing one or more amino terminal residues from polypeptides.<br><br>
JP-7-5034631 (Noda) discloses a leucine aminopeptidase derived from yellow koji mold, which includes Aspergillus oryzae.<br>
JP-7-4021798 (Zaidan Hojin Noda Sangyo) describes the produc¬tion of raiso by adding of a leucine aminopeptidase II pre¬pared by cultivating a number of molds, including Aspergillus oryzae  strain 460 and strain JAM 2616.<br>
Van Heeke et al., Bioch. Biophys. Acta, (1992), 1131, 337-340, have disclosed the cloning of a 30 kDa aminopeptidase from the bacteria Vibrio proteolyticus deposited at the American Type Culture Collection under the ATCC No. 15338.<br>
Aspergillus oryzae 460 is known to produce a number of leucine aminopeptidases. The molecular weight of three of these was calculated to 26,500, 56,000 and 61,000, respectively determined by gel filtration (Nakada et al., Agr. Biol. Chem, (1972), 37(4), 757-765; Nakada et al., Agr. Biol. Chem, (1972), 37(4), 767-774; Nakada et al., Agr. Biol. Chem, (1972), 37(4), 775-782). The Aspergillus oryzae 460 strain is deposited at the American Type Culture Collection as A.   oryzae   (ATTC no. 20386).<br>
Reduction of bitter taste of protein hydrolysates EP 65,663 and EP 325,986  (Miles Inc.)  concerns enzymatic hydrolysis of proteins using a mixture of enzymes containing Aspergillus oryzae   derived proteases. The obtained protein hydrolysate has a bland, non-bitter taste.<br>
JP-4-7029577 (Asahi Electro-chemical Co.) concerns a protease, derived from Aspergillus oryzae, which does not produce any bitter component when decomposing protein.<br>
Prior art discloses a plethora of enzymes exhibiting peptidase, aminopeptidase and other enzyme activities. Said enzymes may be derived from a number of microorganisms, including the fungus species Aspergillus  oryzae.<br><br>
In general products, useful for producing protein hydrolysates without a bitter taste, comprise a mixture of peptidase and aminopeptidase activities.<br>
It would therefore be desirable to be able to provide a single-component enzyme (I.e. substantially without any side activity) exhibiting only an activity useful for reducing the bitterness of protein hydrolysates used in food products.<br>
BRIEF DESCRIPTION OF THE DRAWING<br>
Figure 1 shows the result of SDS-PAGE analysis of supernatant from the AspergrllJus oryzae A01568 35 kOa aminopeptidase producing transformant.<br>
SUMMARY OF THE INVENTION<br>
The object of the present invention is to provide a single-component enzyme exhibiting an activity, which is particular¬ly useful for preparing improving bread products and for producing proteins and/or protein hydrolysates without bitter taste for foodstuff.<br>
The present inventors have surprisingly succeeded in isolat¬ing a DNA sequence encoding an enzyme exhibiting aminopepti¬dase activity, which advantageously may be used for improving the flavour, crust colour, crumb structure and dough sticki¬ness of baked products. Further, said novel enzyme is useful for producing protein or protein hydrolysates without bitter taste.<br>
The complete DNA sequence encoding said aminopeptidase makes it possible to prepare single-component aminopeptidases.<br>
The complete DNA sequence, shown in SEQ ID no. 1, encoding the aminopeptidase of the invention has,  comprised in a<br><br>
plasmid, been transformed into the bacteria strain Escherichia coli DSM no. 9965. This will be described further below.<br>
By a database alignment search it was found that the DNA sequence shown in SEQ ID No. 1 is novel. The highest degree of similarity and identity was found to be 53 % and 32 %, respectively, to the above mentioned 30 kDa aminopeptidase from the bacteria Vibrio proteolyticus   (ATCC No. 15338) .<br>
The inventors have characterized the precursor-form of the aminopeptidase consisting of a secretion signal and the 35 kDa aminopeptidase. The molecular weight (M„) of the precur¬sor-form was calculated to 41 kDa, and the isoelectric point (pi) was estimated to be approximately 4.9. Further, the amino acid composition of the aminopeptidase was estimated as shown in Table 1.<br>
Table 1<br><br>
The deduced  complete  precursor-form  of  the  amino  acid sequence of the 35 kDa enzyme is shown in SEQ ID No. 2.<br><br>
Accordingly, the first aspect of the invention relates to an enzyme exhibiting aminopeptidase activity having an apparent molecular weight (M„) of about 35 kDa determined by SDS-PAGE.<br>
  Mass spectrometry showed that the average mass of the recom¬binant aminopeptidase is in the range from 33 kDa to 35 kDa. The isoelectric point (p1) of the enzyme was determined to be about 4.9.<br>
The isoelectric point, pi, is defined as the pH value where the enzyme molecule complex (with optionally attached metal or other ions) is neutral, i.e. the sum of electrostatic charges (net electrostatic charge, NEC) on the complex is equal to zero. In this sum of course consideration of the positive or negative nature of the electrostatic charge must be taken into account.<br>
In the following the	terms "35 kDa aminopeptidase" and	"the<br>
enzyme  exhibiting	aminopeptidase  activity"   are	used<br>
interchangeably for	the single-component enzyme of	the<br>
present invention.<br>
The enzyme exhibiting aminopeptidase activity of the inven¬tion may be derived from a number of microorganisms. The present inventors have isolated the aminopeptidase of the invention from the filamentous fungus Aspergillus oryzs A01568, which is a strain deposited at the American Type Culture Collection as Aspergillus oryzae 460 (FERM-P no. X149, ATCC no. 20386, and further described in US patent no. 3,914,436.<br>
The enzyme exhibiting aminopeptidase activity of the inven¬tion comprises at least one of the partial amino acid sequences shown in SEQ ID Nos. 6, 7, 8, 9, and 10, respectively. SEQ ID No 11 is a peptide (5) which overlaps and extends the N-terminal sequence.<br><br>
In the second aspect, the invention relates to a DNA con¬struct comprising a DNA sequence encoding said aminopeptidase, which DNA sequence comprises<br>
a)	the aminopeptidase encoding part of the DNA sequence shown in SEQ ID No. 1, and/or the DNA sequence obtainable from E. coli   DSM 9965, or<br>
b)	an analogue of the DNA sequence shown defined in a), which<br>
i) is homologous with the DNA sequence shown in SEQ ID NO. 1 and/or the DNA sequence obtainable from E. coli DSM 9965, or<br>
ii) hybridizes with the same oligonucleotide probe as the DNA sequence shown in SEQ ID No. 1 and/or the DNA sequence obtainable from E.   coli   DSM 9965, or<br>
iii) encodes a polypeptide which is homologous with the polypeptide encoded by a DNA sequence comprising the DNA sequence shown in SEQ ID No. 1 and/or the DNA sequence obtainable from E,   coli  DSM 9965, or<br>
iv) encodes a polypeptide which is immunologically reac¬tive with an antibody raised against the purified aminopeptidase encoded by the DNA sequence shown in SEQ ID No 1 derived from Aspergillus oryzae A01568 or obtainable from E.   coli,   DSM 9965.<br>
In the present context, the "analogue" of the DNA sequence shown in SEQ ID No. 1 and/or the DNA sequence obtainable from E, coli DSM 9965, is intended to indicate any DNA sequence encoding an enzyme exhibiting aminopeptidase activity, which has at least one of the properties i)-iv).<br><br>
The analogous DNA sequence<br>
-	may be isolated from another or related (e.g. the same)<br>
organism producing the enzyme exhibiting aminopeptidase<br>
activity on the basis of any of the DNA sequences shown in<br>
SEQ ID Nos. 3-5, e.g. using the procedures described herein,<br>
and thus, e.g. be an allelic or species variant of the DNA<br>
sequence comprising the DNA sequences shown herein,<br>
-	may be constructed on the basis of any of the DNA sequences<br>
shown in SEQ ID Nos. 3-5, e.g. by introduction of nucleotide<br>
substitutions which do not give rise to another amino acid<br>
sequence of the aminopeptidase encoded by the DNA sequence,<br>
but which correspond to the codon usage of the host organism<br>
intended for production of the enzyme, or by Introduction of<br>
nucleotide substitutions which may give rise to a different<br>
amino acid sequence. However, in the latter case amino acid<br>
changes are preferably of a minor nature, that is conserva¬<br>
tive amino acid substitutions that do not significantly<br>
affect the folding or activity of the polypeptide, small<br>
deletions, typically of one to about 30 amino acids; small amino-<br>
or carboxyl-terminal extensions, such as an amino-terminal methionine residue, a small linker peptide of up to about 20-25 residues, or a small extension that facilitates purifica¬tion, such as a poly-histidine tract, an antigenic epitope or a binding domain. See in general Ford et al., (1991), Protein Expression and Purification 2, 95-107. Examples of conserva¬tive substitutions are within the group of basic amino acids (such as arginine, lysine, histidine), acidic amino acids (such as glutamic acid and aspartic acid), polar amino acids (such as glutamine and asparagine) , hydrophobic amino acids (such as leucine, isoleucine, valine), aromatic amino acids (such as phenylalanine, tryptophan, tyrosine) and small amino acids (such as glycine, alanine, serine, threonine, methionine).<br>
It will be apparent to persons skilled in the art that such substitutions can be made outside the regions critical to the<br><br>
function of the molecule and still result in an active enzy¬me, Amino acids essential to the activity of the polypeptide encoded by the DNA construct of the invention, and therefore preferably not subject to substitution, may be identified according to procedures known in the art, such as site-directed mutagenesis or alanine-scanning mutagenesis (Cun¬ningham and Wells, (1989), Science 244, 1081-1085). In the latter technique mutations are introduced at every residue in the molecule, and the resulting mutant molecules are tested for biological (i.e. aminopeptidolytic) activity to identify amino acid residues that are critical to the activity of the molecule. Sites of substrate-enzyme interaction can also be determined by analysis of crystal structure as determined by such techniques as nuclear magnetic resonance, crystallo¬graphy or photoaffinity labelling. See, for example, de Vos et al. (1992), Science 255, 306-312; Smith et al., (1992), J. Mol. Biol., 224, 899-904; Wlodaver et al., (1992) , FEES Lett., 309, 59-64.<br>
It will be understood that the DKA sequences shown in SEQ ID Nos. 3-5 are sequences which may be used for isolating the entire DNA sequence encoding the aminopeptidase, e.g. the DNA sequence shown in SEQ ID No. 1 and/or the DNA sequence transformed into the deposited strain E. coli DSM 9965. The term "analogue" is intended to include said entire DNA sequence, which comprises one or more of the partial sequences shown in SEQ ID Nos. 3-5 or parts thereof. The amino acid sequence (as deduced from the DNA sequence shown in SEQ ID No. 1) is shown in SEQ ID No. 2.<br>
The homology referred to in i) above is determined as the degree of identity between the two sequences indicating a derivation of the first sequence from the second. The homology may suitably be determined by means of computer programs known in the art such as GAP provided in the GCG program package (Needleman, S.B. and Wunsch, CD., (1970), Journal of Molecular Biology, 48, p. 443-453). Using GAP with<br><br>
the following settings for DNA sequence comparison: GAP creation penalty of 5.0 and GAP extension penalty of 0.3, the coding region of the DNA sequence exhibits a degree of identity preferably of at least at least 70%, preferably at least 80%, especially at least 90%, with the coding region of the DNA sequence shown in SEQ ID No. 1 or the DNA sequence obtainable from the plasmid in E.   coll  DSM 9965.<br>
The hybridization referred to in ii) above is intended to indicate that the analogous DNA sequence hybridizes to the same probe as the DNA sequence encoding the aminopeptidase under certain specified conditions which are described in detail in the Materials and Methods section hereinafter.<br>
Normally, the analogous DNA sequence is highly homologous to the DNA sequence such as at least 60% homologous to the DNA sequence shown in SEQ ID No. 1 or the DNA sequence obtainable from the plasmid in E. coli DSM 9965 encoding an aminopeptidase of the invention, such as at least 65%, at least 70%, at least 75%, at least 80%, at least 85%, at least 90% or even at least 95% homologous to said DNA sequence.<br>
The homology referred to in iii) above is determined as the degree of identity between the two sequences indicating a derivation of the first sequence from the second. The homology may suitably be determined by means of computer programs known in the art such as GAP provided in the GCG program package (Needleman, S.B. and Wunsch, CD., (1970), Journal of Molecular Biology, 48, p. 443-453). Using GAP with the following settings for DNA sequence comparison: GAP creation penalty of 5.0 and GAP extension penalty of 0.3, the coding region of the DNA sequence exhibits a degree of identity preferably of at least at least 70%, preferably at least 80%, especially at least 90%, with the coding region of the DNA sequence shown in SEQ ID No. 1 or the DNA sequence obtainable from the plasmid in E.   coli   DSM 9965.<br><br>
The term "derived from" in connection with property iv) above is intended not only to indicate an aminopeptidase produced by strain A01568, but also an aminopeptidase encoded by a DNA sequence isolated from strain A01568 and produced in a host organism transformed with said DNA sequence. The immunologi¬cal reactivity may be determined by the method described in the "Materials and Methods" section below.<br>
In further aspects the invention relates to an expression vector harbouring a DNA construct of the invention, a cell comprising the DNA construct or expression vector and a method of producing an enzyme exhibiting aminopeptidase activity which method comprises culturing said cell under conditions permitting the production of the enzyme, and recovering the enzyme from the culture.<br>
It is also an object of the invention to provide an enzyme preparation enriched with the 35 kDa aminopeptidase of the invention.<br>
Further, the invention provides a bread-improving or a dough-improving composition comprising an enzyme exhibiting aminopeptidase activity of the invention. Said composition may be combined with other enzymes, such as amylolytic enzymes, and conventional bread improving agents.<br>
In a still further aspect the invention relates to a method for preparing a baked product and frozen dough comprising the 35 kDa aminopeptidase of the invention.<br>
Finally the invention relates to the use of the 35 kDa aminopeptidase of the invention. The enzyme of the invention or a composition of the invention comprising such an enzyme may be used for improving the flavour, crust colour and crumb structure of baked products and to improve the stickiness of frozen dough. The aminopeptidase of the invention may fur¬thermore be used advantageously in connection with producing proteins and protein hydrolysates without bitter taste and<br><br>
may be used for a number of purposes including, degradation or modification of protein containing substances; cleaning of contact lenses, preparation of food and animal feed etc.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The DMA sequence of the invention encoding an enzyme exhibit¬ing aminopeptidase activity may be isolated by a general method involving<br>
-	cloning, in suitable vectors, a DNA library from Aspergillus oryzae,<br>
-	transforming suitable yeast host cells with said vectors,<br>
-	culturing the host cells under suitable conditions to express any enzyme of interest encoded by a clone in the DNA library,<br>
-	screening for positive clones by determining any aminopeptidase activity of the enzyme produced tY such clones, and<br>
-	isolating the DNA coding an enzyme from such clones.<br>
The general method is further disclosed in WO 93/11249 the contents of which are hereby incorporated by reference. A more detailed description of the screening method is given in Example 3 below.<br>
Microbial Sources<br>
The DNA sequence coding for the aminopeptidase of the inven¬tion may for instance be isolated by screening a cDNA library of the donor organism, and selecting for clones expressing the appropriate enzyme activity (i.e. aminopeptidase activity as defined by the ability of the enzyme to hydrolyse Leucine-7 amido-4-methylcoumarin). The appropriate DNA sequence may then be isolated from the clone by standard procedures, e.g. as described in Example 1.<br>
The donor organism may be a fungus of the Aspergillus oryzae (ATCC no. 20386) described in US patent no. 3,914,436<br><br>
The complete full length DNA sequence encoding the aminopeptidase of the invention has been transformed into a strain of the bacteria E, coli, comprised in the expression plasmid pYES 2.0 (Invitrogen). Said bacteria has been deposited by the inventors according to the Budapest Treaty on the International Recognition of the Deposit of Microor¬ganisms for the Purposes of Patent Procedure at the Deutshe Sammlung von Mikroorganismen und Zellkulturen GmbH., Masche-roder Weg lb, D-38124 Braunschweig Federal Republic of Germany, (DSM).<br>
Deposit date        : 11.05.95<br>
Depositor's ref.    : NN49001<br>
DSM designation     : E.   coli   DSM No. 9965<br>
Being an International Depository Authority under the Buda¬pest Treaty, Deutshe Sammlung von Mikroorganismen und Zell¬kulturen GmbH., affords permanence of the deposit in accord¬ance with the rules and regulations of said treaty, vide in particular Rule 9. Access to the two deposits will be avail¬able during the pendency of this patent application to one determined by the Commisioner of the United States Patent and Trademark Office to be entitled thereto under 37 C.F.R. Par. 1.14 and 35 U.S.C. Par. 122. Also, the above mentioned deposits fulfil the requirements of European patent applica¬tions relating to micro-organisms according to Rule 28 EPC.<br>
The above mentioned deposit represents a substantially pure culture of the isolated bacteria. The deposit is available as required by foreign patent laws in countries wherein counter¬parts of the subject application, or its progeny are filed. However, it should be understood that the availability of the deposited strain does not constitute a license to practice the subject invention in derogation of patent rights granted by governmental action.<br><br>
The DNA sequence encoding the enzyme exhibiting aminopeptida-se activity can for instance be isolated from the above mentioned deposited strain by standard methods.<br>
It is expected that a DNA sequence coding for a homologous enzyme, i.e. an analogous DNA sequence, is obtainable from other microorganisms. For instance, the DNA sequence may be derived by similarly screening a cDNA library of another microorganism, in particular a fungus, such as a strain of an Aspergillus sp., in particular a strain of A. aculeatus or A. niger, a strain of another Trichoderma sp., in particular a strain of T. reesei, T. viride, T. longibrachiatum or T. koningii or a strain of a Fusarium sp., in particular a strain of F.   oxysporum,   or a strain of a Humicola sp.<br>
Alternatively, the DNA sequence coding for an enzyme exhibit¬ing aminopeptidase activity of the invention may, in accord¬ance with well-known procedures, conveniently be isolated from DNA from a suitable source, such as any of the above mentioned organisms, by use of synthetic oligonucleotide probes prepared on the basis of a DNA sequence disclosed herein. For instance, a suitable oligonucleotide probe may be prepared on the basis of any of the nucleotide sequences shown in SEQ ID Nos. 3-5 or the amino acid sequence shown in SEQ ID No. 2 or any suitable subsequence thereof.<br>
The DNA sequence may subsequently be inserted into a recombi¬nant expression vector. This may be any vector which may conveniently be subjected to recombinant DNA procedures, and the choice of vector will often depend on the host cell into which it is to be introduced. Thus, the vector may be an autonomously replicating vector, i.e. a vector which exists as an extrachromosomal entity, the replication of which is independent of chromosomal replication, e.g. a plasmid. Alternatively, the vector may be one which, when introduced into a host cell, is integrated into the host cell genome and<br><br>
replicated together with the chromosome(s) into which it has been integrated.<br>
In the vector, the DNA sequence encoding the aminopeptidase should be operably connected to a suitable promoter and terminator sequence. The promoter may be any DNA sequence which shows transcriptional activity in the host cell of choice and may be derived from genes encoding proteins either homologous or heterologous to the host cell. The procedures used to ligate the DNA sequences coding for the aminopeptidase, the promoter and the terminator, respective¬ly, and to insert them into suitable vectors are well known to persons skilled in the art (cf., for instance, Sambrook et al., (1989), Molecular Cloning. A Laboratory Manual, Cold Spring Harbor, NY).<br>
The host cell which is transformed with the DNA sequence encoding the enzyme of the invention is preferably an eukaryotic cell, in particular a fungal cell such as a yeast or filamentous fungal cell. In particular, the cell may belong to a species of Aspergillus, most preferably Aspergil¬lus oryzae or Aspergillus niger. Fungal cells may be trans¬formed by a process involving protoplast formation and transformation of the protoplasts followed by regeneration of the cell wall in a manner known per se. The use of Asper¬gillus as a host microorganism is described in EP 238 023 (Novo Nordisk A/S), the contents of which are hereby incor¬porated by reference. The host cell may also be a yeast cell, e.g. a strain of Saccharomyces, in particular Saccharomyces cerevisiae, Saccharomyces kluyveri or Saccharomyces uvarum, a strain of Schizosaccharomyces sp., such as Schizosaccharomy-ces pombe, a strain of Hansenula sp. Pichia sp., Yarrowia sp. such as Yarrowia lipolytica, or JCluyveromyces sp. such as Kluyveromyces lactis.<br>
A  method of producing an enzyme of the invention<br><br>
In a still further aspect, the present invention relates to a method of producing an enzyme according to the invention, wherein a suitable host cell transformed with a DNA sequence encoding the enzyme is cultured under conditions permitting the production of the enzyme, and the resulting enzyme is recovered from the culture.<br>
The medium used to culture the transformed host cells may be any conventional medium suitable for growing the host cells in question. The expressed aminopeptidase may conveniently be secreted into the culture medium and may be recovered there from by well-known procedures including separating the cells from the medium by centrifugation or filtration, precipitat¬ing proteinaceous components of the medium by means of a salt such as ammonium sulphate, followed by chromatographic procedures such as ion exchange chromatography, affinity chromatography, or the like.<br>
Enzyme preparation<br>
In a still further aspect, the present invention relates to an enzyme preparation useful for reducing the bitterness of proteins and/or protein hydrolysates for foodstuff.<br>
The enzyme preparation, having been enriched with an enzyme Df the invention, may e.g. be an enzyme preparation compris¬ing multiple enzymatic activities, such as an enzyme pre¬paration comprising multiple enzymes for producing protein hydrolysates. The preparation to be enriched can be Flavourzyme® (available from Novo Nordisk A/S). Flavourzyme® is a protease/peptidase complex derived from Aspergillus oryzae  developed for hydrolysis of proteins.<br>
Dependent on the use for which the enzyme preparation is to be used the aminopeptidase of the invention may be combined with other enzyme as mentioned below.<br><br>
In the present context, the term "enriched" is intended to indicate that the aminopeptidase activity of the enzyme preparation has been increased, e.g. with an enrichment factor of at least 1.1, preferable between 1.1 and 10, more preferred between 2 and 8, especially between 4 and 6, conveniently due to addition of an enzyme of the invention prepared by the method described above.<br>
Alternatively, the enzyme preparation enriched with an enzyme exhibiting aminopeptidase activity may be one which comprises an enzyme of the invention as the major enzymatic component, e.g.   a single-component enzyme preparation.<br>
The enzyme preparation may be prepared in accordance with methods known in the art and may be in the form of a liquid or a dry preparation. For instance, the enzyme preparation may be in the form of a granulate or a microgranulate. The enzyme to be included in the preparation may be stabilized in accordance with methods known in the art.<br>
In another aspect the invention relates to a bread-improving or a dough-improving composition comprising an aminopeptidase of the invention. Said composition may further comprise enzymes selected from the group including amylolytic enzyme, such as a-araylase, /5-amylase, maltogenic a-amylase, amyloglu-cosidase, acid stable amylase, and 1,6-pullulanase.<br>
Such enzymes are available from Novo Nordisk A/S as AMG™ (amyloglucosidase) obtained from a strain of Aspergillus niger, Fungamyl™ (fungal amylase) obtained from a strain of Aspergillus oryzae, Novamyl™ (maltogenic amylase) obtained from a strain of Bacillus stearothermophilus.<br>
The composition of the invention may also comprise one or more additional enzymes. Examples of such enzymes include a cellulase a hemicellulase, a pentosanase (useful for the partial hydrolysis of pentosans which increases the extensi-<br><br>
bility of the dough), a lipase (useful for modification of lipids present in the dough or dough constituents so as to soften the dough), a peroxidase (useful for improving the dough consistency), an oxidase, e.g. a glucose oxidase, a laccase, a xylanase, a protease (useful for gluten weakening, in particular when using hard wheat flour).<br>
The other enzyme components are preferably of microbial origin and may be obtained by conventional techniques used in the art as mentioned above.<br>
The enzyme(s) to be used in the present invention may be in any form suited for the use in question, e.g. in the form of a dry powder or granulate, in particular a non-dusting granu¬late, a liquid, in particular a stabilized liquid, or a protected enzyme. Granulates may be produced, e.gr. as dis¬closed in US 4,106,991 and US 4,661,452 (both to Novo Indus-tri A/S), and may optionally be coated by methods known in the art. Liquid enzyme preparations may, for instance, be stabilized by adding nutritionally acceptable stabilizers such as a sugar, a sugar alcohol or another polyol, lactic acid or another organic acid according to established methods. Protected enzymes may be prepared according to the method disclosed in EP 238,216.<br>
Normally, for inclusion in pre-mixes or flour it is advan¬tageous that the enzyme (s) is/are in the form of a dry product, e.g. a non-dusting granulate, whereas for inclusion together with a liquid it is advantageously in a liquid form.<br>
In addition or in an alternative to other enzyme components, the dough-improving and/or bread-improving composition may comprise a conventionally used baking agent, e.g. one or more of the following constituents:<br>
A milk powder (providing crust colour), gluten (to improve the gas retention power of weak flours), an emulsifier (to improve dough extensibility and to some extent the consist-<br><br>
ency of the resulting bread) , granulated fat (for dough softening and consistency of bread), an oxidant (added to strengthen the gluten structure; e.g. ascorbic acid, potas¬sium brornate, potassium iodate or ammonium persulfate), an amino acid (e.g. cysteine), a sugar, and salt (e.g. sodium chloride, calcium acetate, sodium sulfate or calcium sulphate serving to make the dough firmer), flour or starch. Such components may also be added directly to the dough in accord¬ance with a method of the invention.<br>
Examples of suitable emulsifiers are mono- or diglycerides, diacetyl tartaric acid esters of mono- or diglycerides, sugar esters of fatty acids, polyglycerol esters of fatty acids, lactic acid esters of monoglycerides, acetic acid esters of monoglycerides, polyoxyethylene stearates, phospholipids and lecithin.<br>
The bread-improving and/or dough improving composition of the invention is typically included in the dough in an amount corresponding to 0.01-5%, in particular 0.1-3%.<br>
In accordance with the method of the invention, in which an enzyme with aminopeptidase activity of the invention, op¬tionally in combination with other enzymes as described above, is used for the preparation of dough and/or baked products, the enzyme(s) may be added as such to the mixture from which the dough is made or to any ingredient, e.g. flour, from which the dough is to be made. Alternatively, the enzyme(s) may be added as a constituent of a dough-improving and/or a bread-improving composition as described above, either to flour or other dough ingredients or directly to the mixture from which the dough is to be made.<br>
The dosage of the enzyme(s) to be used in the method of the present invention should be adapted to the nature and compo¬sition of the dough in question as well as to the nature of the enzyme(s) to be used. Normally, the enzyme preparation is<br><br>
component enzyme with exopeptidase activity, optionally as constituent(s) of the bread-improving and/or dough-improving composition of the invention. The other enzyme activities may be any of the above described enzymes and may be dosed in accordance with established baking practice.<br>
As mentioned above the enzyme exhibiting aminopeptidase activity, optionally in combination with other enzyme(s) as described above, is added to any mixture of dough ingredi¬ents, to the dough, or to any of the ingredients to be in¬cluded in the dough, in other words the enzyme(s) may be added in any step of the dough preparation and may be added in one, two or more steps, where appropriate.<br>
The handling of the dough and/or baking is performed in any suitable manner for the dough and/or baked product in ques¬tion, typically including the steps of kneading the dough, subjecting the dough to one or more proofing treatments, and baking the product under suitable conditions, i.e. at a suitable temperature and for a sufficient period of time. For instance, the dough may be prepared by using a normal straight dough process, a sour dough process, an overnight dough method, a low-temperature and long-time fermentation method, a frozen dough method, the Chorleywood Bread process, or the Sponge and Dough process.<br>
The dough and/or baked product prepared by the method of the invention are normally based on wheat meal or flour, optio¬nally in combination with other types of meal or flour such as corn flour, rye meal, rye flour, oat flour or meal, soy flour, sorghum meal or flour, or potato meal or flour.<br>
In the present context the term "baked product" is intended to include any product prepared from dough, either of a soft or a crisp character. Examples of baked products, whether of a white, 1ight or dark type, which may advantageously be produced by the present invention are bread (in particular<br><br>
white, whole-meal or rye bread), typically in the form of loaves or rolls, French baguette-type bread, pita bread, tacos, cakes, pan-caKes, biscuits, crisp bread and the like.<br>
The dough of the invention may be of any of the types dis¬cussed above, and may be fresh or frozen.<br>
The preparation of frozen dough is described by K. Kulp and K. Lorenz in "Frozen and Refrigerated Doughs and Batters". When using the aminopeptidase of the invention for frozen bread the flavour, the crust colour and the crispiness are improved.<br>
From the above disclosure it will be apparent that the dough of the invention is normally a leavened dough or a dough to be subj ected to leavening. The dough may be leavened in various ways such as by adding sodium bicarbonate or the like or by adding a leaven (fermenting dough), but it is preferred to leaven the dough by adding a suitable yeast culture such as a culture of Saccharomyces cerevisiae (baker's yeast). Any of the commercially available S. cerevisiae strains may be employed.<br>
As mentioned above, the present invention further relates to a pre-mix, e.g., in the form of a flour composition, for dough and or baked products made from dough, which pre-mix comprises an enzyme exhibiting aminopeptidase activity of the invention and optionally other enzymes as specified above. The pre-mix may be prepared by mixing the relevant enzyme(s) or a bread-improving and/or dough-improving composition of the invention comprising the enzyme(s) with a suitable carrier such as flour/ starch, a sugar or a salt. The pre-mix may contain other dough-improving and/or bread-improving additives, e.g., any of the additives, including enzymes, mentioned above.<br><br>
Uga of the 35 XDa Aminopeptidase of the invention<br>
nse—of., the	Enzyme of the invention for prenarina Baked<br>
Products<br>
The enzyme or an enzyme preparation of the invention may be used in baking, e.g. in order to weaken the gluten components of flour so as to obtain a softening of so-called hard flour.<br>
However, it has surprisingly been found that the aminopep¬tidase of the invention does not degrade the network of the gluten which is normally observed when proteases are used for preparing baked products. Consequently, the dough character¬istics and crumb structure are not affected.<br>
Further, when adding the 35 kDa aminopeptidase of the inven¬tion to the dough or dough ingredients, when preparing baked products, the flavour crust colour and/or the crumb structure and/or the crust colour of a baked product will be substan¬tially improved, as shown in Example 13.<br>
Further, the enzyme of the invention improves the dough stickiness.<br>
The addition of the enzyme of the invention results in baked products having a flavour of yeasty type, providing a "fresh¬ly baked" bread smell. This make the invention of particular interest for the sponge-douqh-system, in which an addition of the enzyme can reduce the sponge fermentation time without a concomitant loss of yeasty flavour, and in the no-time-dough process (most European processes), in which the enzyme can provide a yeasty flavour which is otherwise normally lacking products prepared from such processes.<br>
without being limited to any theory it is presently believed that further improved flavour and/or crust colour of a baked product may be obtained when a single component enzyme with exopeptidase activity is used in combination with an amylo-lytic enzyme, in particular an amylolytic enzyme which is<br><br>
capable of liberating reducing sugar molecules from flour or other constituents of the dough. The increased amount of reducing sugars in the dough provides an increase in Maillard reactions taking place during baking thereby further improv¬ing the flavour and crust colour of the baked product.<br>
Use of the Enzvme of the invention for Reducing Bitter Taste The enzyme or enzyme preparation of the invention may be used for  reducing  the  bitterness  of  proteins  and/or  protein hydrolysate for foodstuff.<br>
Also contemplated according to the invention is the pro¬duction of free amino acids from proteins and/or protein hydrolysates. In the case of the free amino acid are glutamine acid it enhances the flavour of food products.<br>
Said protein or protein hydrolysate may be of animal or vegetable origin.<br>
In an embodiment of the invention the protein to be hydrolysed is casein or soy protein.<br>
The protein may be use for producing foodstuff such as cheese and foodstuff containing cocoa.<br>
Even though the aminopeptidase and enzyme preparations enriched with an enzyme of the invention may be used espe¬cially advantageously in connection with producing proteins or protein hydrolysates without bitter taste, the aminopepti¬dase of the invention can be used for a number of industrial applications, including degradation or modification of protein containing substances, such cell walls. Some pro¬teins, like extensins, are components of plant cell walls. Aminopeptidases will therefore facilitate the degradation or modification of plant cell walls.<br><br>
The dosage of the enzyme preparation of the invention and other conditions under which the preparation is used may be determined on the basis of methods known in the art.<br>
Extraction of Oil from Plants<br>
The enzyme preparation according to the invention may be useful for extraction of oil from plant sources like olives and rape or for production of juice from different fruits like apples, pears and citrus. It may also be useful in the wine industry, especially in the white wine industry, to prevent haze formation. Furthermore, it may be used to modify and degrade proteins, e.g. in order to reduce the viscosity caused or partially caused by proteins, or to facilitate fermentative processes where proteins are involved, or it may be used to improve the digestibility of proteins and other nutrients.<br>
The Use for preparing Food and Feed<br>
The aminopeptidase preparation may also be used in the food and feed industry to improve the digestibility of proteins. For instance, the enzyme or enzyme preparation may be added to animal feed or may be used to process animal feed, in particular feed for piglets or poultry.<br>
Further the enzyme or enzyme preparation of the invention may be useful to make protein hydrolysates from, e.g., vegetable proteins like soy, pea, lupin or rape seed protein, milk like casein, meat proteins, or fish proteins. The aminopeptidase may be used for protein hydrolysates to improve the solubil¬ity, consistency or fermentability, to reduce antigenicity or for other purposes to make food, feed or medical products. The aminopeptidase may be used alone or together with other aminopeptidases or together with other enzymes like exopeptidases. The use of the aminopeptidase of the invention together with exopeptidase rich enzyme preparations will improve the taste of the protein hydrolysates.<br><br>
Furthermore, the enzyme or enzyme preparation may be used in the processing of fish or meat, e.g. to change texture and/or viscosity.<br>
Use in Brewing processes<br>
The enzyme preparation may also be used to facilitate fermentative processes, like yeast fermentation of barley, malt and other raw materials for the production of e.g.   beer.<br>
Use for making Protoplast<br>
The enzyme preparation may be useful for making protoplasts<br>
from fungi.<br>
Use for the production of Peptides<br>
The enzyme preparation may be useful for production of pep¬tides from proteins, where it is advantageous to use a cloned enzyme essentially free from other proteolytic activities.<br>
Use for degradation of Proteins<br>
Further, the aminopeptidase preparation can be used to degrade protein in order to facilitate purification of or to upgrade different products, like in purification or upgrading of gums, like guar gum, xanthan gum, degumming of silk, or improvement of the quality of wool.<br>
Use for cleaning contact Lenses<br>
Further the enzyme or enzyme preparation may be used for<br>
cleaning of contact lenses.<br>
The invention is described in further detail in the following examples which are not in any way intended to limit the scope of the invention as claimed.<br><br>
METHODS AND HATERIALS<br>
Materials<br>
Donor organism: Aspergillus oryzae   A01568 (described in US<br>
patent no, 3,914,436)<br>
Host organism:<br>
Escherichia  coli  MC1061 (Meissner et al., (1987), Proc. Natl. Acad. Sci. U.S.A., 84, 4171-4176: cDNA library strain<br>
Saccharomyces cerevisiae W3124 (van den Hazel et al., (1992), Eur. J. Biochem., 207, 277-283): Activity screening strain.<br>
Schizosaccharomyces pombe: Broker et al., (1989), FEES Letters, 248, 105-110.<br>
Other organisms:<br>
Aspergillus oryzae A1560 (Christensen et al. (1988), Bio/technology 6, 1419-1422).<br>
Plasmids:<br>
pYES 2.0: Transformation vector (Invitrogen).<br>
pHD414: Aspergillus expression vector is a derivative of the plasmid p775 (described in EP 238.023). The construction of the pHD414 is further described in WO 93/11249. pHD414 contains the A. nigrer glucoamylase terminator and the A. oryzae  TAKA amylase promoter.<br>
pHD423 is a derivative of pHD414 (described in WO 94/20611) with a new polylinker.<br><br>
pUClS: Expression vector (Sambrook et al,, (1989), Molecular Cloning: A Laboratory Manual, 2. ed.,- Cold Spring Harbor Laboratory, Cold Spring Harbor, New York)<br>
pClEXP3: plasmid comprising the 1.4 kb cDNA insert encoding the 35 KDa aminopeptidase of the invention.(See Example 3 and SEQ ID NO 1)<br>
pClEXP4: plasmid comprising a cDNA sequence, which is 120 bp shorter that the pClEXP3 1.4 kb cDNA insert, (see Example 10 and SEQ ID NO 12)<br>
p3SR2: A. nidulans amdS+ gene carrying plasmid (Christensen et al., (1988), Bio/Technology 6, 1419-1422)<br>
pPl: yeast expression vector, which is an E. coli/s. pombe shuttle vector containing the ADH promoter and URA 3 as selective marker (Broker et al., (1989), FEBS Letters, 248, 105-110).<br>
primers:<br>
universal pUC primers (Sambrook et al., (1989), supra)<br><br><br><br>
Forward and Reverse pYES primers (Invitrogen)<br>
Enzymes:<br>
Lysine-specific protease Novozym® 234 (Novo Nordisk A/S) Alcalase® (Novo Nordisk A/S) Neutrase® (Novo Nordisk A/S)<br>
Peptides of the 35 kDa aminopeptidase: (see SEQ ID No. 6-11)<br>
N-terminal:<br>
Direct N-terminal sequencing of authentic and recombinant aminopeptidase revealed the same N-terminal amino acid sequence for the two enzymes showing that the recombinant enzyme is proteolytically processed identical to the authen¬tic enzyme. The N-terminal sequence found was<br><br>
(Xaa is a glycosylated Asn-residue)<br>
The following peptide sequence was obtained from peptides derived from a S-carboxymethylated sample of the aminopep¬tidase by cleavage with a lysyl-specific protease.<br>
Peptide 1:<br>
Tyr-Pro-Asp-Ser-Val-Gln-Hls-Xaa-Glu-Thr-Val-Gln-Asn-Leu-Ile-Lys<br>
Peptide 2: Gly-Val-Thr-Val-Glu-Pro-Phe-Lys<br><br><br>
Media and other materials:<br>
STC: 1.2 M sorbitol, 10 mM Tris-HCl, pH = 7.5., 10 mM CaCl2<br>
BSA (Sigma, type H25)<br>
Leucine-7 ainido-4-methylcouinarin (Sigma)<br>
PEG 4000  (polyethylene glycol, molecular weight = 4,000)<br>
(BDH, England)<br>
Sequenase®Kit (United Stated Biochemical, USA)<br>
Hybond-N nylon membrane (Amersham, USA)<br>
Q-sepharose (Pharmacia Tm)<br>
Superdex 200 Tm<br>
Amicon membrane<br>
VG Analytical TofSpec<br>
a-Cyano-4-hydroxycinnamic acid (Aldrich, Stelnheim, Germany).<br>
YPD: 10 g yeast extract, 20 g peptone, HjO to 810 ml. Auto-claved, 90 ml 20% glucose (sterile filtered) added.<br>
, 10 X Basal salt: 66.8 g yeast nitrogen base, 100 g succinic acid, 60 g NaOH, HjO ad 1000 ml, sterile filtered.<br><br>
SC-URA; 90 ml 10 x Basal salt, 22.5 ml 20% casamino acids, 9 ml 1% tryptophan, HO ad 806 ml, autoclaved, 3.6 ml 5% threonine and 90 ml 20% glucose or 20% galactose added.<br>
SC-H broth: 7.5 g/1 yeast nitrogen base without amino acids, 11.3 g/1 succinic acid, 6.8 g/1 NaOH, 5.6 g/1 casamino acids without vitamins, O.i g/1 tryptophan. Autoclaved for 20 min. at 121'*C. After autoclaving, 10 ml of a 30% galactose sol¬ution, 5 ml of a 30% glucose solution and 0.4 ml of a 5% threonine solution were added per 100 ml medium.<br>
SC-H agar: 7.5 g/1 yeast nitrogen base without amino acids, 11.3 g/1 succinic acid, 6.8 g/1 NaOH, 5.6 g/1 casamino acids without vitamins, 0.1 g/1 tryptophan, and 20 g/1 agar (Bacto) . Autoclaved for 20 min. at 121'C. After autoclaving, 55 ml of a 22% galactose solution and 1.8 ml of a 5% threonine solution were added per 450 ml agar.<br>
YNB-1 agar: 3.3 g/1 KH2PO4, 16.7 g/1 agar, pH adjusted to 7. Autoclaved for 20 min. at 121'C. After autoclaving, 25 ml of a 13.6% yeast nitrogen base without amino acids, 25 ml of a 40% glucose solution, 1.5 ml of a 1% L-leucine solution and 1.5 ml of a 1% histidine solution were added per 450 ml agar.<br>
YNB-l broth: Composition as YNB-1 agar, but without the agar.<br>
Minimal plates: (Cove Biochem.Biophys.Acta 113 (1966) 51-56) containing 1.0 M sucrose, pH 7.0, 10 mM acetamide as nitrogen source and 20 mM CsCl<br>
Whey protein hydrolysate: Whey hydrolysed with Alcalase® and Neutrase® was diluted with water until the protein content was 8% (w/w) of the total solution.<br>
Methods<br>
RKA isolation: The total RNA was prepared, from frozen, powdered mycelium of A.    oryzae   A01568, by extraction with<br><br>
guanidinium thiocyanate followed by ultracentrifugation through a 5.7 M csCl cushion (Chirgwin et al., (1979), Biochemistry 18, 5294-5299). The poly(A)RNA was performed by oligo(dT)-cellulose affinity chromatography (Aviv, H, and Leder, P. , (1972), Proc. Natl. Acad. Sci. U.S.A. 69, 1408-1412).<br>
cDNA synthesis: Double-stranded cDNA was synthesized from 5 fiq of Aspergillus oryzae poly (A) " RNA as described by Kofod et al., J. of Biol. Chem., (1994), 269, 29182-29189, except that 2 5 ng of random hexanucleotide primers (Pharmacia, Sweden) were included in the first strand synthesis.<br>
Construction of cDHA library:<br>
The cDNA library was constructed as described by Kofod et<br>
al., J. of Biol. Chera., (1994), 269, 29182-29189.<br>
Transformation of Saccharomyces cerovisiaei<br>
To ensure that all the bacterial clones were tested in yeast,<br>
a number of yeast transformants 5 times larger than the<br>
number of bacterial clones in the original pools was set as<br>
the limit.<br>
One 1 aliquots of purified plasmid DNA (100 ng/tl) from individual pools were electroporated (200 n, 1.5 kV, 24 F) into 40 1 of competent S. cerevisiae cells (OD600 = 1.5 in 500 ml YPD, washed twice in cold distilled water, once in cold 1 M sorbitol, resuspended in 0.5 ml 1 M sorbitol, Becker &amp; Guarante, 1991). After addition of 1 ml 1 M cold sorbitol, 80 ul aliquots were plated on SC + glucose - uracil to give 250-400 c.f.u./plate and incubated at SO'C for 3-5 days.<br>
Transformation of Schizosaccbaromyces pombe<br>
Schizosaccharomyces pombe was transformed as described by Broker, (1987) BioTechnigues, 5, 51-517.<br><br>
Purification of 35 IcDa aminopeptidase from Aspergillus oryzaez<br>
One gram freeze dried powder of the fermentation supernatant of Aspergillus oryzae A01568 was dissolved in 100 ml Tris-acetate buffer (25 mM pH 8 ). Ionic strength was 2 mSi. The suspension was filtered through 45 fi millipore filter. The filtered solution was applied on a 200 ml anion exchange chromatography column packed with Q-sepharose, which was equilibrated with the Tris-acetate buffer. Alkaline protease which is a major endoprotease with isoelectric point of 8 was collected in effluent. The column was washed with the Tris-acetate buffer until no more UV absorbing material was present in effluent.<br>
The bound proteins were eluted with linear salt gradient using 0 to 0.5 M NaCl in the Tris-acetate buffer (pH 8) using 10 column volume with a flow rate of 4 ml/minutes. Fractions containing aminopeptidase activity (see below) were pooled and dialyzed against Tris-acetate buffer (25 mM , pH 6).<br>
The dialyzed pool containing activity was adjusted to pH 6 and ionic strength to 2 mSi and applied on 50 ml High per¬formance Q-sepharose column, equilibrated with 25 mM Tris-acetate buffer pH 6, for anion exchange chromatography. The column was then washed until the UV absorbing material in effluent was under 0.05 at 280 nm. Bound activity was then eluted with 20 column volume linear salt gradient, from 0 to 0.5 M NaCl at a flow rate of 2 ml/minutes. Fractions contain¬ing aminopeptidase activity were pooled and concentrated by ultrafiltration, using 50 mM sodium-acetate buffer (pH 6).<br>
Two ml of the concentrated pool containing aminopeptidase activity was applied on Superdex 200 Tm column equilibrated with 50 mM sodium acetate buffer (pH 6) containing 0.1 M NaCl. The gel filtration was carried out using a flow rate of 0.5 ml/minutes. Samples containing aminopeptidase activity<br><br>
were pooled and concentrated by ultrafiltration using Amicon membrane with a cut-off-value of 10 kDa.<br>
Amino acid sequence determination of N-terminal and internal peptides of the .Aspergillus oryzae aminopeptidase: S-carboxymethylated samples of the purified native aminopeptidase is digested with lysyl-specific protease, and the resulting peptides are separated by reverse phase high pressure liquid chromatography (HPLC) and sequenced as described by Matsudaira, A Practical Guide to Protein and peptide Purification for Microsequencing, 3-88, Academic Press Inc, San Diego, CA, in an Applied Biosystems 473A sequencer according to the manufacturer's instructions (Applied Biosystems).<br>
Reagents and solvents for amino acid sequencing are from Applied Biosystems (Foster City, CA).<br>
Designing of PCR primers:<br>
The PCR primers were designed as described by Kofod et al.,<br>
J. of Biol. Chem., (1994), 269, 29182-29189.<br>
Generation of a cDNA probe for an aminopeptidase using PCR!<br>
One tig of double stranded plasmid DNA from a cDNA library pool was PCR amplified using 500 pmol of each of the designed primers in combinations with 500 pmol of pYES 2.0 polylinker primer (forward and reverse), a DNA thermal cycler (Landgraf, Germany) and 2.5 units of Taq  polymerase (Perkin-Elmer)•<br>
Thirty cycles of PCR were performed using a cycle profile of denaturation at 94°C for 1 minute, annealing at 55"C for 2 minutes, and extension at 72'C for 3 minutes.<br><br>
Dideoxy chain-termination method:<br>
The method was carried out as described by Sanger et al., (1977), Proc. Natl. Acad. Sci. U.S.A. 74, 5463-5467, using the Sequenase®Kit and universal pUC primers.<br>
Characterization of positive cDNA clones by Southern blot analysis:<br>
The positive clones were characterized by the use of Southern blot hybridization using the 0.5 kb random-primed P-labelled PCR-product or 35 kDa aminopeptidase as probe. The hybridiza¬tions were carried out in 2 x ssc, 5 x Denhardt's solution, 0.5% (w/v) SDS, 100 nq/ml denatured salmon sperm DNA for 48 hours at 65 "C followed by washing at high stringency in 2 x SSC (2 X 15 minutes), 2 x SSC, 0.5% SDS (30 minutes), 0.2 x SSC, 0.5% SDS (30 minutes) and finally in 2 x SSC (15 min¬utes), at 65'C (SambrooJc et al. (1989), supra .<br>
Electrophoresis<br>
Electrophoresis was performed on a 0.7% agarose gel from<br>
SeaKem, FMC.<br>
Capillary blotting<br>
Capillary blotting method is described by Sambrook et al.,<br>
(1989), supra) using lo x SSC as transfer buffer<br>
High stringency washes of hybridized clones;<br>
Washing was carried out in 2 x 15 minutes in 2 x SSC, 2 x 30<br>
minutes in 0.1 x SSC, 0.5% SDS and 15 minutes in 2 x SSC, at<br>
eS'-C.<br>
Transformation of Aspergillus oryzaex<br>
Transformation of Aspergillus oryzae was carried out as<br>
described by Christensen et al. ,  (1988), Biotechnology 6,<br>
1419-1422.<br><br>
construction of the aminopeptidase expression cassette for Aspergillus<br>
Plasinid DNA was isolated from the positive E. coll clones using standard procedures and analyzed by restriction enzyme analysis. The cDNA insert was excised using appropriate re¬striction enzymes and ligated into an Aspergillus expression vector.<br>
Transformation of Aspergillus oryzae  or Aspergillus niger (general procedure)<br>
100 ml of YPD (Sherman et al. , Methods in Yeast Genetics, Cold Spring Harbor Laboratory, 1981) is inoculated with spores of A. oryzae or A. niger and incubated with shaking at 37 "C for about 2 days. The mycelium is harvested by filtra¬tion through miracloth and washed with 200 ml of 0.6 M MgSO*. The mycelium is suspended in 15 ml of 1.2 M MgSO. 10 mM NaH2P04, pH = 5.8. The suspension is cooled on ice and 1 ml of buffer containing 120 mg of Novozym® 234 is added. After 5 minutes 1 ml of 12 mg/ml BSA is added and incubation with gentle agitation continued for 1.5-2.5 hours at SVC until a large number of protoplasts is visible in a sample inspected under the microscope.<br>
The suspension is filtered through miracloth, the filtrate transferred to a sterile tube and overlayered with 5 ml of 0.6 M sorbitol, 100 mM Tris-HCl, pH *= 7.0. Centrifugation is performed for 15 minutes at 100 g and the protoplasts are collected from the top of the MgS04 cushion. 2 volumes of STC are added to the protoplast suspension and the mixture is centrifugated for 5 minutes at 1000 g. The protoplast pellet is resuspended in 3 ml of STC and repelleted. This is repeat¬ed. Finally the protoplasts are resuspended in 0.2-1 ml of STC.<br>
100 1 of protoplast suspension is mixed with 5-25 g of the appropriate DNA in 10 1 of STC. Protoplasts are mixed with p3SR2  (an A. nidulans    arads gene carrying plasmid) .  The<br><br>
mixture is left at room temperature for 25 minutes. 0.2 ml of 60% PEG 4000. 10 mM CaCl; and lO mM Tris-HCl, pH 7.5 is added and carefully mixed (twice) and finally 0.85 ml of the same solution is added and carefully mixed. The mixture is left at room temperature for 25 minutes, spun at 2500 g for 15 mi¬nutes and the pellet is resuspended in 2 ml of 1.2 M sorbitol. After one more sedimentation the protoplasts are spread on the appropriate plates. Protoplasts are spread on minimal plates to inhibit background growth. After incubation for 4-7 days at 37°C spores are picked and spread for single colonies. This procedure is repeated and spores of a single colony after the second re-isolation is stored as a defined transformant.<br>
Purification  of  the Aspergillus      oryzae      transformants:<br>
Aspergrliius oryzae colonies are purified through conidial spores on AmdS"-plates (+ 0,01% Triton X-IOO) and growth in YPM for 3 days at 30° C.<br>
Identification of aminopeptldase positive Asper'gillus oryzae transformants:<br>
The supernatants from the Aspergillus oryzae transformants were assayed for aminopeptldase on agar plates overlayered with 60 /jg/ml of Leucine-7 amido-4-methylcoumarin. Positive transformants were identified by analyzing the plates by fluorescence under UV-light after 5 mihutes to 2 hours incubation at 30°C.<br>
SDS-PAGE analysis:<br>
SDS-PAGE analysis of supernatant from an Aspergillus   oryzae<br>
aminopeptldase producing transformant. The transformant was<br>
grown in 5 ml YPM for three days. 10 ill   of supernatant was<br>
applied to 12% SDS-polyacrylamide gel which was subsequently<br>
stained with Coomassie Brilliant Blue.<br>
Mass spectrometry<br>
Mass  spectrometry  is  done  using  matrix  assisted  laser<br>
desorption ionisation time-of flight mass spectrometry in a<br><br>
VG Analytical TofSpec. For mass spectrometry 2 /ul of sample is mixed with 2 fil saturated matrix solution (Q!-cyano-4-hydroxycinnamic acid in 0.1% TFA:acetonitrile (70:30)) and 2 ll of the mixture spotted onto the target plate. Before introduction into the mass spectrometer the solvent was removed by evaporation. Samples are desorbed and ionised by 4 ns laser pulses f337 nm) at threshold laser power and accel¬erated into the field-free flight tube by an accelerating voltage of 25 kV. Ions were detected by a micro channel plate set at 1850 V. The spectra is calibrated externally with proteins of known mass.<br>
Immunological cross-reactivity:<br>
Antibodies to be used in determining immunological cross-reactivity may be prepared by use of a purified aminopept-idase. More specifically, antiserum against the aminopept-idase of the invention may be raised by immunizing rabbits (or other rodents) according to the procedure described by N. Axelsen et al. in: A Manual of Quantitative Immunoelectrophoresis, Blackwell Scientific Publications, 1973, Chapter 23, or A. Johnstone and R. Thorpe, Immunochemistry in Practice, Blackwell Scientific Publica¬tions, 1982 (more specifically pp. 27-31). Purified immun¬oglobulins may be obtained from the antisera, for example by salt precipitation ((NH4)2 SO4), followed by dialysis and ion exchange chromatography, e.g. on DEAE-Sephadex. Immunochemi¬cal characterization of proteins may be done either by Outcherlony double-diffusion analysis (O. Ouchterlony in: Handbook of Experimental Immunology (D.M. Weir, Ed.), Black-well Scientific Publications, 1967, pp. 655-706), by crossed Immunoelectrophoresis (N. Axelsen et al. , supra. Chapters 3 and 4) , or by rocket immunoelectrophoresis (N. Axelsen et al., Chapter 2).<br>
soutbern blot analysis:<br>
Genomic DNA from A.   Qryzae   is isolated according to Yelton et<br>
al., (1984), Proc. Natl. Acad. Sci. U.S.A. 81., p. 1470-1474,<br><br>
and digested to completion with BamHI, Bglll, EcoRI and Hindlll (10 fig/sample), fractionated on a 0.7 % agarose gel, denatured and blotted to a nylon filter (Hybond-N) using 10 x SSC as transfer buffer (Southern, E. M, , (1975), J. Mol. Biol. 98, p. 503-517). The aminopeptidase cDNA is "p-labeled (&gt; 1 X 10* cpm/g) by random-priming and used as a probe in Southern analysis. The hybridization and washing conditions are as described in RNA gel blot analysis. The filter is autoradiographed at -SOC for 12 hours.<br>
RNA gel blot analysis:<br>
Poly (A)* RNA (1 g) from A. oryzae is electrophoresed in 1.2 % agarose-2.2 M formaldehyde gels (Thomas, P. S., (1983) Methods Enzymol. 100, pp. 255-2663) and blotted to a nylon membrane (Hybond-N) with 10 x SSC as transfer buffer. The aminopeptidase cDNA is P-labeled (&gt; 1 x lO' cpm//tg ) by random priming and hybridized to the membrane for 18-20 hours at 65'*C in 5 X SSC, 5 x Denhardt' s solution, 0.5% SDS (w/v) and 100 fig/ml denatured salmon sperm DNA. The filter is washed in 5 x SSC at eS'C (2 x 15 minutes), 2 x SSC, 0.5 % SDS (1 X 30 minutes), 0.2 x SSC, 0.5 % SDS (1 x 30 minutes), and 5 x SSC (2 x 15 minutes) . The filter is autoradiographed at - SOC for 12 hours.<br>
Determination of aminopeptidase activity (LAPU)<br>
One LAPU is defined as the amount of enzyme which hydrolyzes 1 Mmole of L-leucine-p-nitroanilide per minute using the method described in AF 298/1-GB (available on request from Novo Nordisk A/S).<br>
% DH determination based on THBS analysis<br>
The extent of protein hydrolysis may be determined by the degree of hydrolysis achieved. In the context of this inven¬tion, the degree of hydrolysis (DH) is defined by the follow¬ing formula:<br>
h<br>
DH =  	  X 100 %<br>
h„<br><br>
h is the number of peptide bonds hydrolysed and h« is the total number of peptide bonds in the protein, h is dependent on the type of raw material, whereas h can be expressed as a function of meqv leucine NHj, measured by for instance TNBS-analyses<br>
Determination of %DH is described in EF-9415317 (available on request from Novo Nordisk A/S)<br>
Testing of Doughs and Breads<br>
According to the present invention the effect of adding a single component enzyme with aminopeptidase activity may be tested in doughs and breads by using the following method:<br>
Preparation of Breads<br>
Procedure:<br>
1.	Dough mixing (Spiral mixer)<br>
3 min. at 700 RPM<br>
5 min. at 1400 RPM the mixing time is predetermined and adjusted by a skilled baker based on the flour used so as to obtain an optimum dough consistence under the testing conditions used.<br>
2.	1st proof: sec - 80% RH, 15 min.<br>
3.	Scaling and shaping;<br>
4.	Resting for 5 minutes at ambiant temperature;<br><br>
5.	Final proof: 22 "C - 80% RH, 45 minutes for rolls, 55 minutes for bread;<br>
6.	Baking: 225"C, 22 minutes for rolls and 30 minutes for loaf.<br>
Evaluation of Dough and Baked Products<br>
Dough and baked products may be evaluated as follows:<br>
Loaf specific volume: the mean value of 4 loaves volume are measured using the traditional rape seed method. The specific<br><br>
volume is calculated as volume ml per g bread. The specific volume of the control (without enzyme) is defined as 100. The<br><br>
Shock test: After the second proof a pan containing the dough is dropped from a height of 20 cm. The dough is baked and the volume of the resulting bread is determined.<br><br><br>
EXAMPLES<br>
Example 1<br>
Construction of cDNA library<br>
Total RNA was extracted from Aspergillus oryzae A01568. Poly(A)+RNA was isolated by oligo(dT)-cellulose affinity chromatography and double stranded cDNA (ds cDNA) was syn¬thesized.<br>
A cDNA library from Aspergillus oryzae A01568 consisting of 3.5 X 10* clones was constructed into the yeast expression vector pYES 2.0.<br>
Example 2<br>
Amplification and characterization of cDNA clones.<br>
The  aminopeptidase was  purified  from Aspergillus    oryzae<br>
A01568 as described above in the section "Materials and<br>
Methods".<br>
A long NHj-terminal sequence and four internal sequences (including Peptide X to 5) of the aminopeptidase were obtained by digestion of S-carboxymethylated purified protein with a lysyl-specific protease.<br>
Based on these sequences three primers were synthesized (asl, s2 and S3, respectively). Double stranded cDNA (ds cDNA) was used as template in the PCR amplification experiment as described above in the section "Materials and Methods".<br>
Analysis of the resulting PCR-products revealed a 0.5 kb fragment with one primer pair (primer s3 and primer asl).<br>
The PCR-fragment was sub-cloned into Smal-cut dephosphory-lated pUClS vector and sequenced from both ends using the Dideoxy chain-termination method as described above in the section "Material and Methods".<br><br>
In addition to the primer encoded residues, the sequence of Peptide 2 obtained from the purified aminopeptidase, aligned with the deduced amino acid sequence, confirmed that the desired cDNA species had been specifically PCR amplified.<br>
Example 3<br>
screening of the cDNA library for clones encoding the aminopeptidase from Aspergillus  oryzae<br>
Approximately 10,000 colonies from_ the cDNA library from Aspergillus oryzae A01568 were screened by colony hybridiza¬tion as described above. This yielded positive clones with inserts ranging from 650 bp to 1.5 kb.<br>
The positive clones were analyzed by Southern blot analysis as described above in the section "Materials and Methods".<br>
purified plasmid DNA (about 1 nq) from the aminopeptidase cDKA clones was digested to completion by Hindlll and Xbal to release the cDNA inserts from the pYES vector. The samples were electrophoresed and transferred to a Hybond-H nylon membrane by the capillary blotting method. The filters were washed at high stringency resulting in positive clones with inserts ranging from 900 bp to 1700 bp.<br>
Strongly hybridizing clones were analyzed by sequencing the ends of the cDNAs with forward and reverse pYES primers.<br>
Analysis of the sequence data showed that some of these clones were truncated cDNAs whereas others appeared to be full-length clones. The nucleotide sequence and "deduced amino acid sequence of one of the full-length clones (pClEXP3) obtained (shown in figure 1) contains a cDNA insert of 1.4 kb.<br><br>
Example 4<br>
Expression of the aminopeptidase in Aspergillus  oryzae TO obtain high level production of aminopeptidase in Asper¬gillus oryzae,    the cDNA insert from the pClEXP3 clone was sub-cloned into pHD4 23 and co-transformed with the AmdS* plasmid into Aspergillus  oryzae  as described above.<br>
The 1.4 kb cDNA insert from pClEXP3 was isolated from pYES 2.0 by Hind III and Not I digestion, ligated to a Not I/Hind III cleaved pHD 423 vector, and transformed into S. coli.<br>
The resulting transformants were purified twice (see above) and assayed for aminopeptidase activity as described above. Transformant  pA3EXP3/l  showed  detectable  aminopeptidase activity.<br>
Example 5<br>
Expression level of aminopeptidase producing transformant lpA3EXP3/11<br>
The amount and purity (level of expression) of secreted aminopeptidase from the transformant {pA3EXP3/l) was deter¬mined semi-quantitatively by SDS-PAGE using the non-trans¬formed A. oryzae strain A01560 as a negative control and the A.   oryzae  A01568 strain as positive control (see figure 1).<br>
A 36-37 kDa polypeptide could be seen in pA3EXP3/l, not present in the negative control. The size of the recombinant aminopeptidase is approximately 2 kDa higher than that of the native aminopeptidase (a double band of about 35 kDa), possibly due to additional glycosylation or other types of post-translational modifications.<br>
Example 6<br>
Mass spectrometry showed that the recombinant aminopeptidase is glycosylated as the mass determined exceed the mass of the polypeptide calculated from the cDNA sequence to be 32.4 kDa.<br><br>
The average mass of the recombinant aminopeptidase is 34.1 kDa with the masses ranging from 33 kDa to 35 kDa.<br>
The apparent molecular weight (M„) determined by SDS-PAGE (as described in "Materials and Methods" was found to be about 35 kDa.<br>
Example 7<br>
Fermentation of 35 kPa aminopeptidase producing transformant The Aspergillus   oryzae   transformant pA3EXP3/l was grown in one liter shake flasks containing 150 ml DAP 2C (pH = 5.9) for three days at 3 0'*C.<br>
The amount of secreted aminopeptidase was estimated by SDS-PAGE analysis to approximately 0.5 g/liter supernatant. Example 8<br>
Expression of 35 kDa aminopeptidase clones in S. pombe The full-length 35 kDa aminopeptidase cDNA clone pClEXP3 was re-transformed into Schxzosaccharomyces pombe by electropora-tion. The 1.4 kb cDNA insert was released from the pYES 2.0 vector by Spel and NotI digestion and subcloned into the Spel/NotI cleaved yeast expression vector pPl, which is an E. coli/S. pombe shuttle vector containing the ADH promoter, and assayed for aminopeptidase activity.<br>
It was shown that one S. pombe transformant had strong aminopeptidase activity, indicating that S. pombe is able to synthesize and secrete a functionally active 35 kDa aminopeptidase from Aspergillus  oryzae  A0156a.<br>
Example 9<br>
Organization and Expression of the Aminopeptidase gene<br>
The copy number of the aminopeptidase gene in the A. oryzae<br>
A01568 genome was determined by Southern blot hybridization<br>
described in the "Materials and Methods" section. Total DNA<br>
isolated from A.oryzae  was digested to completion with BamHI,<br>
Bglll, EcoRI or Hindlll and hybridized with the aminopeptida-<br><br>
se cDNA. The aminopeptidase probe detects only single strong¬ly hybridizing fragments in each case except in BamHI diges¬tion, which gives two hybridizing fragments due to a BamHI site at nucleotide position 640. This indicates that the aminopeptidase gene is present as a single copy in the A. oryzae  A01568 genome.<br>
Example 10<br>
To study the expression of the 35 kDa aminopeptidase gene, poly (A)* RNA extracted from A, oryzae A01568 mycelium was subjected to Northern blot analysis. Probing of the blotted RHA with the aminopeptidase cDNA revealed two species of mRNA of approximately 1.45 and 1.55 kilobases. These two mRNAs do not appear to represent transcripts of two distinct genes since the same pattern of hybridization was observed when the filters were washed at high stringency. The difference in size of the two mRNAs could be due to different lengths of 3' untranslated region, because of two polyadenylation sites, in accordance with the two cDNA species isolated from the Aspergillus oryzae cDNA library, one corresponding to clones pClEXP3 and another one corresponding to pClEXP4, which is 120 bp shorter. Both mRNAs encode the same aminopeptidase.<br>
Example 11<br>
Removing bitter taste from whey protein hydrolysate<br>
The fermentation broth of Aspergillus    oryzae    transformant<br>
pA3EXP3/l was tested for the ability to debitter a solution<br>
containing 8% (w/w) protease hydrolysed whey protein.<br>
The aminopeptidase activity of fermentation broth was 5.58 LAPU/g.<br>
The substrate protein solution was tested in flasks con¬taining 100 g of the 8% protein hydrolysate. The fermentation broth exhibiting aminopeptidase activity was added until the relationship between enzyme and substrate (E/S) was 0.25%,<br><br>
calculated on the basis of a product with 5000 LAPU/g (equiv¬alent to 12.5 LAPU/g protein), and was then re-hydrolysed for 6 hours at 50'C, pH 7.0.<br>
The pH and the osmolarity was determined (using standard methods) after 1 minute and 6 hours, respectively. % DH was determined using the TNSB-method (described above) and % FAA (% free amino acids) was determined using standard methods.<br><br>
A sample of debittered hydrolysate containing 13 % FFA was analyzed for the content of leucine. It was found that leucine constituted about 2.7%. while leucine constituted about 0.3% of the blind.<br>
Example 12<br>
Taste test<br>
The taste of debittered protein hydrolysate was assessed by a<br>
tasting panel of 5 persons using a Bitterness Index (BI)<br>
between 0 (not bitter) and 10 (blind).<br>
The bitterness, of a sample of bitter tasting 3.5 % protein hydrolysate (blind) and a similar sample subjected to the transformant, was assessed.<br>
The Bitterness Index (BI) for the protein hydrolysate sample subjected to the transformant was found to be in the range of about 6.2.<br><br>
This result shows that the 35 kDa aminopeptidase had debit-tered the protein hydrolysate samples.<br>
Example 13<br>
Bread flavour, dough stickiness and crumb structure The flavour, dough stickiness and crumb structure of bread prepared using from 0 to 300 LAPU per kg flour were compared with bread prepared using the commercial product Flavourzyme®. The bread were prepared and assessed as described above in the "Material and Methods" section.<br><br>
 As can be seen from the above table the 35 kDa aminopeptidase of the invention gives a significant flavour enhancement in the form of a "fresh baked" bread smell when added in amounts from 30 to 300 LAPU per kg flour. The crumb structure is not affected by the aminopeptidase of the invention.  The dough<br><br>
stickiness is improved in comparison to the commercial protease/peptidase complex Flavourzyme®.<br>
As will be apparent to those skilled in the art in the light of the foregoing disclosure, many alterations and modifica¬tions are possible in the practice of this invention without departing from the spirit or scope thereof. Accordingly, the scope of the invention is to be construed in accordance with the substance defined by the claims below.<br><br><br><br>
WE CLAIM:<br>
1.	An enzyme exhibiting aminopeptidase activity derived from a fungal<br>
microorganism, which enzyme comprises one or more of the partial amino acid<br>
sequences shown in SEQ ID No. 6, SEQ ID No. 7, SEQ ID No. 8, SEQ ID No. 9,<br>
SEQ ID No. 10, or SEQ ID No, 11.<br>
2.	The enzyme according to claim 1 which enzyme has an apparent molecular weight (Mw) of 35 kDa determined by SDS-page.<br>
3.	The enzyme according to claim 1, having an estimated isoelectric point (pi) in the<br>
range of about 4.9.<br>
I<br>
4.	The enzyme according to any of claims 1 to 3, derivable from a filamentous fungus<br>
or a yeast.<br>
5.	The enzyme according to claim 4, derivable from a filamentous fungus, such as Aspergilius, in particular A, oryzae. Especially A, oryzae A01568, or Trichoderma, Pencillium, Fusarium or Humicola.<br>
6.	The enzyme according to any of claims 1 to 5, which enzyme comprises or is comprised in the amino acid sequence shown in SEQ ID No. 2.<br>
7.	A DNA construct comprising a DNA sequence encoding an enzyme exhibiting aminopeptidase activity, which DNA sequence comprises<br><br>
a)	the aminopeptidase encoding part of the DNA sequence shown in SEQ ID No, 1, or the DNA sequence obtainable from E, coil DSM 9965 encoding an aminopeptidase, or<br>
b)	an analogue of the DNA sequence defined in a), which<br><br>
i) is 70% identical over the entire length to the DNA sequence shown in SEQ ID No. 1 or the DNA sequence obtainable from E. coli DSM 9965 encoding an aminopeptidase, or<br>
ii) encodes a polypeptide which is 70% identical to the polypeptide encoded by a DNA sequence comprising the DNA sequence shown in SEQ ID No. 1 or the DNA sequence obtainable from E. coli DSM 9965 encoding an aminopeptidase.<br>
8.	The DNA construct according to claim 7, comprising the partial DNA sequences shown in SEQ ID No. 3, and/or the partial DNA sequences shown in SEQ ID No. 4, and/or the partial DNA sequences shown in SEQ ID No. 5.<br>
9.	The DNA construct according to any of claims 7 or 8, in which the DNA sequence is obtainable from a fungal microorganism, such as a filamentous fungus or a yeast.<br>
10.	The DNA construct according to claim 9, in which the DNA sequence is<br>
obtainable from a strain of Aspergillus, such as Aspergillus oryzae, in particular from<br>
the deposited Aspergillus oryzae, ATCC 20386, or Trichoderma, Penicillium,<br>
Eusatium, Humicola or E. coli DSM 9965.<br>
11.	The DNA construct according to claim 10, in which the DNA sequence is isolated from or produced on the basis of a nucleic acid library of Aspergillus oryzae, A01 568.<br>
12.	A recombinant expression vector comprising the DNA construct according to any<br>
of claims 7 to II.<br>
13.	A fungal host cell comprising a DNA construct according to claims 7 to 11 or a<br>
recombinant expression vector according to claim 12.<br><br>
14.	The cell according to claim 13, which is a yeast cell or a filamentous fungal cell.<br>
15.	The cell according to claim 14, wherein the cell belongs to a strain of Aspergillus, in particular a strain of Aspergillus niger or Aspergillus oryzae, a strain of Saccharomyces, in particular Saccharomyces cerevisiae, Saccharomyces Iduyveri or Saccharomyces uvarum, a strain of Schizosaccharomyces sp., such as Schizosaccharomyces pombe, a strain of Hansenula sp. Pichia sp„ Yarrowia sp. such as Yarrowia lipolytica, or Kluyveromyces sp. such as Kluyveromyces lactis-<br>
16.	The cell according to claim 15, wherein the cell belongs to a strain of Aspergillus, in particular a strain of Aspergillus niger ox Aspergillus oryzae.<br>
17.	A method for producing an enzyme exhibiting aminopeptidase activity, which method comprises cultivating a host cell according to any of claims 13 to 16 in suitable culture medium under conditions permitting the expression of the DNA construct according to any of claims 7 to 11 or expression vector according to claim 12, and recovering the enzyme from the culture.<br>
18.	An enzyme preparation useful for reducing bitter taste of proteins or protein hydrolysates for foodstuff, which preparation is enriched with an enzyme exhibiting aminopeptidase activity according to any of claims 1 to 6.<br>
19.	The enzyme preparation according to claim 18, which additionally comprises at least one other enzyme activity, including a cellulase, a hemicellulase, a pentosanase, a lipase, a peroxidase, an oxidase, a laccase, a xylanase, a peptidase, and an endo-protease.<br><br>
20. A composition comprising the enzyme as claimed in claim 1 and another component selected from bread or dough improving agents, an amylolytic enzyme and an enzyme selected from a cellulase, a hemicellulase, a pentosanase, a lipase, a peroxidase, an oxidase, a laccase, and a xylanase.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU5Ny1tYXMtOTYgYWJzdHJhY3QtZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">0597-mas-96 abstract-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU5Ny1tYXMtOTYgYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">0597-mas-96 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU5Ny1tYXMtOTYgYXNzaWdubWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">0597-mas-96 assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU5Ny1tYXMtOTYgY2xhaW1zLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">0597-mas-96 claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU5Ny1tYXMtOTYgY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">0597-mas-96 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU5Ny1tYXMtOTYgY29ycmVzcG9uZGVuY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">0597-mas-96 correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU5Ny1tYXMtOTYgY29ycmVzcG9uZGVuY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">0597-mas-96 correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU5Ny1tYXMtOTYgZGVzY3JpcHRpb24gKGNvbXBsZXRlKS1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">0597-mas-96 description (complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU5Ny1tYXMtOTYgZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">0597-mas-96 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU5Ny1tYXMtOTYgZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">0597-mas-96 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU5Ny1tYXMtOTYgZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">0597-mas-96 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU5Ny1tYXMtOTYgZm9ybS0xOS5wZGY=" target="_blank" style="word-wrap:break-word;">0597-mas-96 form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU5Ny1tYXMtOTYgZm9ybS0yNi5wZGY=" target="_blank" style="word-wrap:break-word;">0597-mas-96 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU5Ny1tYXMtOTYgZm9ybS00LnBkZg==" target="_blank" style="word-wrap:break-word;">0597-mas-96 form-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU5Ny1tYXMtOTYgZm9ybS02LnBkZg==" target="_blank" style="word-wrap:break-word;">0597-mas-96 form-6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU5Ny1tYXMtOTYgcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">0597-mas-96 pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDU5Ny1tYXMtOTYgcGV0aXRpb24ucGRm" target="_blank" style="word-wrap:break-word;">0597-mas-96 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="224299-an-aspergillus-host-cell-for-the-expression-of-a-heterologous-protein-product-and-a-method-for-producing-the-protein.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="224301-a-nail-varnish-comprising-glyceryl-triacetate.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>224300</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>597/MAS/1996</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>47/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Nov-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>10-Oct-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>09-Apr-1996</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVOZYMES A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>KROGSHOJVEJ 36 DK-2880 BAGSVAERD,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SPENDLER, TINA</td>
											<td>C/O. NOVO NORDISK A/S., NOVO ALLE, DDK-2880 BAGSVAERD,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>KAUPPINEN SAKARI</td>
											<td>C/O. NOVO NORDISK A/S., NOVO ALLE, DDK-2880 BAGSVAERD,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>SI JOAN Q1</td>
											<td>C/O. NOVO NORDISK A/S., NOVO ALLE, DDK-2880 BAGSVAERD,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>DAMBMANN, CLAUS</td>
											<td>C/O. NOVO NORDISK A/S., NOVO ALLE, DDK-2880 BAGSVAERD,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>HALKIER, TORBEN</td>
											<td>C/O. NOVO NORDISK A/S., NOVO ALLE, DDK-2880 BAGSVAERD,</td>
										</tr>
										<tr>
											<td>6</td>
											<td>OSTERGAARD, PETER RAHBEK</td>
											<td>C/O. NOVO NORDISK A/S., NOVO ALLE, DDK-2880 BAGSVAERD,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N09/48</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/224300-an-enzyme-exhibiting-aminopeptidase-activity-and-a-dna-construct-comprising-a-dna-sequence-encoding-such-enzyme by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:52:52 GMT -->
</html>
