<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/224401-a-universal-primer-pair-for-amplifying-a-fragment-of-cytochrome-b-gene-of-an-animal-species by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:29:21 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 224401:&quot;A UNIVERSAL PRIMER PAIR FOR AMPLIFYING A FRAGMENT OF CYTOCHROME b GENE OF AN ANIMAL SPECIES&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A UNIVERSAL PRIMER PAIR FOR AMPLIFYING A FRAGMENT OF CYTOCHROME b GENE OF AN ANIMAL SPECIES&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention provides novel universal primers that can amplify the fragment of cytochrome b gene of any animal species in polymerase chain reaction (PCR) and reveal the identity of the biological material of any unknown animal origin and a method for identification of the specific animal from a given biological sample.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>UNIVERSAL PRIMERS FOR WILDLIFE IDENTIFICATION<br>
TECHNICAL FIELD<br>
The invention relates to the identification of novel universal primers that can amplify the<br>
fragment of cytochrome b gene of any animal species in polymerase chain reaction (PCR)<br>
and reveal the identity of the biological material of any unknown animal origin at species<br>
and sub-species sources. The invention also provides a method for the identification of<br>
fragments on mitochondria! cytochrome b gene in biological material of unknown origin.<br>
BACKGROUND ART<br>
A large number of studies in evolutionary biology utilize phylogenetic information<br>
obtained from mitochondria! cytochrome b gene. It has been identified a potent molecule<br>
to distinguish the phylogenetic depth of different lineages to family, genus and species in<br>
molecular taxonomy1'46. A vast database of the sequences of cytochrome b gene of<br>
different animal species has accumulated in public databases such as GenBank, NCBI<br>
(http://www.ncbi.nlm.nih.gov) etc. We have utilized this capacity of cytochrome b gene in<br>
establishing the identity of the origin of animal parts and product to its family, genus and<br>
• species sources. The technique developed is based on a pair of universal primer that can<br>
amplify a small fragment of cytochrome b gene from a vast range of animal species.<br>
Establishing identity of confiscated animal parts and products is a great challenge to law<br>
enforcement agencies because none of the methods available till date is too efficient to<br>
reveal the identity of animal remains beyond a reasonable doubt. Morphological markers,<br>
described for certain species allow the identification of complete specimen of animals<br>
However, a complete specimen is confiscated very rarely by the investigation agencies;<br>
therefore, these marker are not practical in wildlife forensics. The biochemical traits such<br>
as the bile characteristics68 blood heam analysis69'70 etc. have also been employed in<br>
wildlife forensic for identification of individual species. The difficulty of these markers<br>
are that these markers are limited in number and are rarely found in their natural forms in<br>
which these were originally described as the characteristic of a particular species.<br>
The molecular approaches such as micro-satellite based identification , Restriction<br>
fragment length polymorphism analysis of mitochondrial genes or PCR based species<br>
specific STS markers require the prior information of the species to establish the<br>
identity72'73. These methods also need a significant amount of DNA material to be<br>
analysed. We may not have the prior information about the species origin of confiscated<br>
animal parts and product in forensics. therefore, these methods arc not really useful and<br>
practical in wildlife identification. The technique invented fay us is universal, therefore<br>
does not require any background information to establish the identity of any unknown<br>
confiscated remains at family, genus and species sources. Being a PCR based procedure it<br>
can be applied with trace amount of any biological material. Because the amplicon length<br>
is small (472 bp); therefore, it can work perfectly with the mutilated remains, which are<br>
commonly seized by the crime investigation agencies. It does not require the large amount<br>
of genetic material i.e. DNA to be analyzed to establish the identity, hence, can detect a<br>
minute amount of adulteration in food products. The procedure described is simple and<br>
very fast. Due to the said advantages, the procedure invented by us is most suited for<br>
forensic wildlife identification.<br>
OBJECTS OF THE INVENTION<br>
The main object of the invention is to identify a fragment on mitochondrial cytochrome b<br>
gene capable of significantly discriminating among various evolutionary lineages of<br>
different animal species.<br>
Another object is to identify a fragment on mitochondrial cytochrome b gene which is<br>
flanked by the highly conserved sequences at a vast range of animal species.<br>
Yet another object is to detect a fragment on mitochondrial cytochrome b gene which is<br>
polymorphic inter-specifically, but monomorphic at intra species sources.<br>
Still another object is to develop the universal primers to amplify the fragment on<br>
mitochondrial cytochrome b gene using polymerase chain reaction.<br>
Another object is to develop a PCR protocol that works universally with DNA template of<br>
any unknown origin (i.e. ail the animal species).<br>
Yet another object is to provide a universal method for identification of species of analyzed<br>
material (i.e. the DNA isolated from confiscated animal remain of unknown origin) using<br>
the public databases such as GenBank, NCBl etc.<br>
Still another object is to provide a universal method of animal identification to establish<br>
the crime with the criminal beyond a reasonably doubt. /<br>
Another object is provide a universal method to establish the identity of biological<br>
materials such as skin, horns etc confiscated from animal poachers, if it is that of an<br>
endangered species.<br>
Yet another object is to provide a universal method for establishment of the identity of<br>
confiscated animal parts and products of endangered animal species for the purpose ol<br>
production of molecular evidence of animal hunting and related crime in the co"i of law,<br>
so that the human violation to the wildlife resources could be controlled.<br>
Still another object is to provide a universal technique to have an idea of the geographical<br>
location of the commitment of wildlife crime based on the haplotype of poached animal -<br>
identified by the universal primer invented.<br>
Another object is to provide a universal technique of animal identification to detect the<br>
adulteration of animal meat/products in vegetarian food product for the purpose of food<br>
fortification, by the food fortification agencies.<br>
Yet another object is to provide a universal technique for detection of the origin of blood<br>
or blood stains etc collected from the scene of crime related to offences such as murder,<br>
rape etc, in order to establish the origin of blood found at scene of crime when it sounds as<br>
if criminals have wontedly spread the blood of an animal at the scene of crime, to confuse<br>
the crime investigation agencies and forensic scientists with human blood.<br>
Another object is to invent and authenticate a universal technique that can be converted to<br>
a (a) 'MOLECUALR KIT' and (b) 'DNA CHIPS' based application to meet the<br>
requirements of above objectives.<br>
SUMMARY OF THE INVENTION<br>
Accordingly, the invention provides novel universal primers that can amplify the fragment<br>
of cytochrome b gene of any animal species in polymerase chain reaction (PCR) and reveal<br>
the identity of the biological material of any unknown animal origin<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
Keeping in view the above objectives, the cytochrome b gene sequences (1140 bp) of 221<br>
distantly related animal species (listed in Table 1) representing various families were<br>
obtained from public database NCBI (http://www.ncbi.nlm.nih.goy). These sequences<br>
were aligned using the software Clustal X(l,8)(NCBl, USA) and a fragment (of 472 bp,<br>
alignment shown in Table 2) of gene was identified which had all the features mentioned<br>
above under column I, 2 and 3 of sub-heading 'Objectives of invention'. 
identity of this fragment we would like to mention that it includes the nucleotides between<br>
398 to 869 in Amilope cervicapra and Felis catus; however, 399 to 870 in Homo sapiens<br>
sapiens species. Except at few positions (marked as star (*) in Table 2, the nucleotide<br>
sequences of this fragment are highly variable amongst the animal species, giving rise to<br>
their unique molecular signature. These molecular signatures are characteristic of its<br>
species and form the basis of revealing the identity of the biological material of an<br>
unknown animal origin by the procedure invented by us. Considering Antilope cervicapra<br>
as a representative species, the sequence of this fragment is mentioned herewith.<br>
Muochondrial cytochrome b gene sequence (398-809 bp) o( Amilope ccr\-icupru:<br>
"taccatgaggacaaatatctttttgaggagcaacagtcatcaccaatctcctttcagcaatcccatacatcggtacaaacctagtaga<br>
atgaatctgagga^ggttctcagtagataaagcaacccttacccgatttttcgccttccactttatcctcccatttatcattgcagccctt<br>
accatagtacacctactgtttctccacgaaacaggatccaacaaccccacaggaatctcatcagacgcagacaaaattccattccac<br>
ccctactacactatcaaagatatcctaggagctctactanaattttaaccctcatgcttctagtcctattctcaccggacctgcitggag<br>
acccagacaactatacaccagcaaacccacttaatacacccccacatatcaagcccgaatgatacncctantgcatacgcaatcct<br>
ccgatcaattcctaacaaactaggagg".<br>
A pair of universal primer was designed to amplify this fragment in polymerase chain<br>
reaction (PCR). These primers were named as 'mcb398' and 'mcb869' because of its<br>
property to amplify a region of rmtochondrial cytochrome b gene between nucleotides 398<br>
to 8£9 ofAntilope cervicapra, a representative animal species for this invention. We took<br>
this animal species as representative species because the idea of developing such a novel<br>
primers came in the mind of inventors while they were working on the genome of this<br>
animal in Centre for Cellular and Molecular Biology, Hyderabad, India. These primers<br>
work universally because its 3' end are highly conserved amongst a vast range of animal<br>
species (shown in Table 2). As mentioned above, the DNA fragment (sequence of which is<br>
shown above) targeted by these primers is highly polymorphic inter-specifically; however,<br>
iris monomorphic among the individual of same species (Tables 6, 7a, 7b, 7c, -d and 8,<br>
respectively). These unique features of the targeted region enable these primers to generate<br>
the molecular signatures of an individual species; thereby, enabling them to differentiate<br>
amongst the animals of different species (see in Figure Ic). The variation within the<br>
fragment amplified by these pnmers increase with increasing distances of evolutionary<br>
lineages of two animals (Table 8). These unique features of the fragment amplified by the<br>
universal primers 'mcb398' and 'mcb869' invented by the applicants fulfill the objectives<br>
of invention.<br>
Thus, the primers invented by us can generate the molecular signature from any biological<br>
material of unknown animal origin, which actually is the characteristic of its family, genus<br>
and more precisely, the species. When these signatures are compared in-silico with the<br>
signatures already available in public databases (viz., GenBank, NCBl database etc) using<br>
'BLAST software71, it indicates identity of the family, genus or species of the analyzed<br>
material, which in turn is confirmed practically by comparing with the reference animals of<br>
the revealed family, genus or species, by including them in the further analysts by the<br>
primers 'mcb398' and 'mcb869'. The complete procedure involved in the analyses (the<br>
word, 'analyses' should be understood with the stepwise procedure to establish the identity<br>
of the biological remain of any unknown animal origin for the aims mentioned in columns<br>
The animals belonging to similar species cluster together; however, the animals of<br>
different species group in different clusters. The confiscated material under investigation<br>
(i.e. 'adil.flesh') clusters with 'gzlL' (i.e. the known normal leopard 'Panthera pardus')<br>
indicating the identity of the species of 'adil.flesh' as that of a Panthera pardus source.<br>
Figure 2 shows the Agarose gel electrophorogram showing the PCR amplicons (472 bp)<br>
obtained from the reference animals of family felidae listed in Table 5, using universal<br>
primers 'mcb398 and 4mcb869*. Description of different lanes is as follows:<br>
Lanes 1-21: The PCR profiles of the animals 1-21, respectively, listed in Table 5.<br>
Lane 22: The PCR profiles of DNA isolated from confiscated skin of unknown animal<br>
i<br>
origin 'i.e. adil.flesh'<br>
Lane 23: Negative control (no DNA)<br>
Lane M: Molecular weight marker (marker Xffi, Boehringer mannheim)<br>
Figure 3. Shows PCR amplicons obtained from animals listed in Table 9. TV primers<br>
used in PCR are 'AFF' and 'AFR'. The description of different lanes shown is as follows:<br>
Lane 1-4: The PCR profiles of animals 1-4, respectively, listed in Table 9, showing<br>
amplicons of 354 bp.<br>
Lane M: Molecular weight marker (marker XIII, Boehringer mannheim)<br>
Figure 4. Shows PCR amplicons obtained from animals listed in Table 12. This<br>
experiment demonstrates the universal nature of our primers among a vast range of animal<br>
species. Description of different lanes shown is as follows:<br>
Lanes 1-23: The PCR profiles of the animals 1-23, respectively, listed in Table 12. The<br>
PCR product of 472 bp is amplified universally from all the animal species analyzed.<br>
Lane 24: Negative control (no DNA)<br>
Lane M: Molecular weight marker (marker XIII, Boehringer mannheim)<br>
Table 1. List of 221 animal species used for fn-silico analysis to design the universal<br>
primers 'mcb398' and 'mcb869'. Table also demonstrate the 'P.S scores' of 'moh398' and<br>
'mcb869' for different templates. The descriptions of various symbols used in this table are<br>
as follows:<br>
Symbol (#) refers to Number<br>
Symbol (*) refers to the animal species which is either protected species (listed in Wildlife<br>
(Protection) Act, 1972 (Central Act NO 53 of 1972), or an endangered/rare animal species<br>
Symbol (*P,S/F) refers to Probability of match and Stability of mutch of primer 'mcb398'<br>
with different templates (i.e. the cytochromc b gene from different species origin). A<br>
higher P.S score refers to the higher probabilities of significant L\r*r^aficat;o ' of specific<br>
template by the primer. It is calculated by Amplify (1.2) software.<br>
Symbol C*P,S/R) refers to Probability of match and Stability of match of primer 'mcb869'<br>
with different templates. A higher P,S score refers to the higher probabilities of significant<br>
amplification of specific template by the primer. It is calculated by Amplify (1.2) software.<br>
Table 2. Multiple sequence alignment of 472 bp fragment of mitochondnal cytochrome b<br>
gene (identified by inventors to fulfill the requirements of column 1, 2 and 3 mention<br>
under sub-heading 'Objectives of invention1) of 221 animal species listed in Table 1.<br>
Alignments also show the binding sites for universal primers 'mcb398' and 'mcb869'. The<br>
symbol (*) refers to the nucleotide bases which are conserved amongst 221 animal species<br>
listed in Table 1). The alignments have been done using software CLUSTAL X (1.8). The<br>
nucleitide positions that are unmarked are variable amongst 221 animal species analyzed.<br>
These variable sites together constitute the molecular signature of an individual species,<br>
giving rise to molecular basis of species identification by our primers.<br>
Table 3. Results of the blast analysis of the sequence revealed from 'adil.flesh1 in 'mito'<br>
database of NCBI. It shows the most significant alignment of cytochrome b sequence (328<br>
bp) revealed from confiscated skin piece 'adil.flesh' with_/e/w cams cytochrome b gene<br>
sequence (genbank registration number NC_001700.1, bits score 365, E value, e-101)<br>
registered in NCBI database (bits score 365 and E value e-101). It gives an indication that<br>
the species of analyzed material belongs to family felidae. It also fulfills the requirements<br>
of column 6 mention above under sub-heading 'Objectives of invention'.<br>
Table 4. Results of the blast analysis of the sequence revealed from 'adil.flesh' in 'nr'<br>
database of NCBI. It shows the most significant alignment of cytochrome b sequence (328<br>
bp) revealed from confiscated skin piece 'adil.flesh' with Panthera pardus cytochrome b<br>
gene sequence (genbank registration number AY005809, bits score 603, E value, e-170)<br>
registered in NCBI database. It gives an indication that the species of analyzed material<br>
belongs to Panthera paurdus origin. It also fulfills the requirements of column 6 mention<br>
above under sub-heading 'Objectives of invention1.<br>
Table 5. Reference animal belonging to family felidae selected for comparison with<br>
'adil.flesh' to confirm the findings of BLAST analysis results of which are mentioned in<br>
Table 3 and 4, respectively. The animals listed in SN. 1-21 represent different species of<br>
family felidae. SN. 22 and 23 are primate species taken for out-group comparisons.<br>
Table 6 Multiple sequence alignments of cytochrome b sequences (328 bp) revealed from<br>
'adil.flesh' and reference animals listed in Table 5. The positions that have a common<br>
nuclcotide in all the animal species under investigation are shown with a star (*) mark:<br>
however, the positions that are variable in any of the animals under investigation are<br>
unmarked. The nucleotides at these positions constitute the molecular signature of an<br>
individual species, which are unique and highly specific for its species. These signatures<br>
are the molecular basis of identification of individual animal species using our primers<br>
Itncb398' and 'mcb869'.<br>
Table 7 (Tables 7a, 7b, 7c and 7d). The comparison of the molecular signatures of<br>
different animal species investigated along with 'adil.flesh', the confiscated skin of<br>
unknown animal origin. This table demonstrates the variable positions (i.e. the positions<br>
which are not marked with star (*) symbol in Table 6), amongst the 328 bp fragment<br>
revealed from the animals listed in Table 5. The dot (.) mark represents the presence of the<br>
similar nucleotide as listed in lane 1 i.e. the sequence from "adil.flesh' at that position. It<br>
demonstrates that the signatures of each species are unique and specific to its species. The<br>
molecular signatures of 'adilflesh' are comparable (except for position 37 which has a<br>
transition from *T* to 'C') to the molecular signature of 'gzlL' i.e. the known leopard<br>
'Panthera pardus' source, indicating the identity of the source of confiscated skin<br>
'adil.flesh' as that of a leopard 'Panthera pardus' source. The nucleotide variations (at the<br>
positions 153, 198, 223, 264, among the known leopards, (i.e. gzlL, gz2L, and gz3L,<br>
respectively)), give an idea about the geographical habitat of each animals. Various studies<br>
referring to mokvular evolution of different animal species support this hypothesis75;<br>
however, it could further be confirmed by taking the reference animals from different<br>
geographical areas and analyzing by our primers 'mcb 398* and 'mcb869'. If we could<br>
generate the database of different haplotypes (i.e. habitat specific molecular signatures) of<br>
the animal species, it would also enable our primers to reveal the geographical location of<br>
the commitment of wildlife crime.<br>
Table 8. Percent similarity matrix calculated by pair-vise comparisons of nucleotide<br>
sequences aligned (illustrated in Table 6). The cytochrome b gene sequence of DNA<br>
isolated from confiscated material had maximum similarity (99.7% and 98.2%, with the<br>
lineages of animals 'gz2L' and 'gz3L\ respectively) with the sequences obtained from<br>
known normal leopard source, indicating its identity as that of a leopard origin. The<br>
similarity matrix has been calculated using the software PHYLIP (3.5).<br>
Table 9. Animals selected for validation of minimum P.S score for efficient amplification<br>
of cytochrome b gene of different origin by the primers 'mcb398' and 'mcb869'. P,S score<br>
of primers 'AFF' and 'AFR' for these animals arc shown.<br>
Table 10. BLAST analysis of primers 'mcb3')S' in nr database of NCBI It demonstrates<br>
that the 3* end of this primer is highly conserved among a vast range of animal .species. It<br>
also shows the sipificant homology among the primer and templates (i.e. the cytochrome<br>
b gene fragment of different animal species), confirming the universal nature of our primer<br>
Table 11. BLAST analysis of primers 'mcb869' in nr database of NCBI. It demonstrates<br>
that the 3* end of this primer is highly conserved among a vast range of animal species. It<br>
also shows the significant homology among the primer and templates (i.e. the cytochrome<br>
b gene fragment of different animal species), confirming the universal nature of our<br>
primer.<br>
Table 12. Other animal belonging to distantly related animal species, investigated to<br>
confirm the universal nature of primers 'mcb398' and 'rncb869'. Gel photograph showing<br>
the PCR amplicons from these animals are shown in Figure 4.<br>
The mitochondria! cytochrome b gene has very widely been used in molecular taxonomic<br>
studies. It has immense capabilities to reveal different evolutionary lineages of animals in<br>
family, genus and species specific manner. It has also been used to classify the population<br>
of a particular species according to its demographic distributions75. The vast database of<br>
cytochrome b sequences of different animal species has accumulated in public databases<br>
such as Genbank and NCBI1"*5. We have explored these unique characteristics of<br>
cytochrome b gene to establish the identity of confiscated remains of any unknown animal<br>
by inventing a pair of novel primers, 'mdxVJi}' and 'rncb869', that can amplify a small<br>
fragment (472 bp) of cytochrome b gene of wide range of animal species in universal<br>
manner. These primers work universally because its 3' ends target within a highly<br>
conserved region.<br>
The fragment of cytochrome b gene identified had all the features mentioned in columns 1,<br>
2 and 3 listed under sub-heading 'Objective of invention'. We identified this fragment by<br>
aligning the cytochrome b gene sequences (1140 bp) of 221 different animal species listed<br>
in Table I. These sequences are publicly available in NCBI DNA databases. These<br>
sequences were aligned using the software CLUSTAL X (1.8). As mentioned before, the<br>
472 bp fragment of cytochrome b gene identified by us to have the features mentioned in<br>
columns 1, 2 and 3 listed under sub-heading 'Objective of invention' includes the<br>
nucleotides between 398 to 869 in Anlilope cervicupra and Felts cattts; however, 399 to<br>
870 in Homo sapiens sapiens species. Except at few positions (marked as star (*) in Table<br>
2. the nucleotide sequences of this fragment are highly variable amongst the animal<br>
species, revealing the identity of the biological material belonging to that of an unknown<br>
animal origin by the procedure invented by us. As for identity of this fragment we arc<br>
considering Antilope cervicapra as a representative species, and the sequence the above<br>
fragment of cytochrome b gene of Antilope cervicapra is mentioned herewith:<br>
Mitochondrial cytochrome b gene sequence (398-869 bp) of Antilope cervicapra<br>
"taccatgaggacaaatatctttttgaggagcaacagtcatcaccaatctcctttcagcaatcccatacatcggtacaaacctagtaga<br>
atgaatctgaggagggttctcagtagataaagcaacccttacccgatttttcgccttccactttatcctcccatttatcattgcagccctt<br>
accatagtacacctactgtttctccacgaaacaggatccaacaaccccacaggaatctcatcagacgcagacaaaa::.,c.Tticcac<br>
ccctactacactatcaaagatatcctaggagctctactattaattttaaccctcatgcttctagtcctattctcaccggacctgcctggag<br>
acccagacaactatacaccagcaaacccacttaatacacccccacatatcaagcccgaatgatacttcctatttgcatacgcaatcct<br>
ccgatcaattcctaacaaactaggagg<br>
Table 2 presents the alignment of the above fragment of cytochrome b gene of 221 animal<br>
species. Each species in table 2 has been represented by a unique code, which is decoded in<br>
Table 1. We selected these species to represent the vast range of animal families of distant<br>
orders. Of 221 species, about 65 were the protected/endangered or rare species listed in<br>
Wildlife (Protection) Act, 1972 (Central Act NO 53 of 1972). These species are marked<br>
with symbol (*) in Table 1. The NCBI accession number refers to its registration number<br>
in NCBI database and the number in superscript represent the reference cited. Based on the<br>
aligned cytochrome b sequences of different 221 animal species the primers designed were<br>
as follow:<br>
Primers name Sequence (i -y)<br>
'mcb398' "TACCATGAGGACAAATATCATTCTG"<br><mcfa869> "CCTCCTAGnTGTTAGGGATTGATCG"<br>
Tables 2, 10 and 11, respectively, demonstrates that the 3' ends of the primers are highly<br>
conserved amongst ail the animal species analyzed in-silico (In total 221 animal species<br>
listed in Table 1 and about 500 species listed in Tables 10 and 11, respectively) Also, the<br>
5' end of the primers were selected within the conserved region of cytochrome b gene to<br>
improve the probability and stability of match of the primers to their target sequences (i.e.<br>
the above mentioned 472 bp fragment of cytochrome b gene). The primers were<br>
thoroughly checked for internal stabilities, loop or dimmer formation using different<br>
software viz.. 'Amplify (1.2)', 'Primer3' (http://w\vw.uenome.wi.mit.alu/cgibin/<br>
primer/primerJ.cgi) as well as manually.. We assigned the P,S score (Probability of<br>
match. Stability of match) to the primers for each template using the software Amplify<br>
(1.2). The higher scores of P and S ensure a good amplification if all other conditions<br>
standard (which arc mentioned under 'Example 3') are optimum. The Highest score tor<br>
'mcb3y8' was 98.63 (i.e. the situation where the primer has perfect match with template):<br>
however, the highest P, S for 'mcb869' was recorded as 98, 68 for a complete match<br>
between the primer and template. The lowest P,S score observed for 'mcb398' was 81,50<br>
for species Talpa europaea whereas 'mcb869' had a high P, S score for this species (92,<br>
57). The another species which have lowest P, S score for one of the two primers were<br>
Eumeces egregious and Equus ainus. Eumeces egregious had P, S score 86, 55 and 73,51<br>
for 'mcb398' and 'mcb869', respectively, however, the P, S score of Equus ainus was<br>
calculated as 91,61 and 73, 51 for Imcb398' and 'mcb869', respectively. All other animals<br>
had higher P, S scores then the above mentioned species. To ensure that these primers<br>
would work efficiently with the DNA template from the animals having the lowest P, S<br>
score tor one of the primers, we designed an another experiment to validate the lower<br>
limits of one of the two primers sufficient for efficient amplification in PCR. We designed<br>
an another primer pair (AFF= 5'tagtagaatgaatctgaggagg3' and<br>
AFR=5'atgcaaataggaagtatcattc3'.) having more mis-pairing at their annealing sites (but not<br>
at ends), therefore have less internal stability and lower P, S scores for its templates (listed<br>
in Table 9). The P,S scores of 'AFF' and 'APR' were as calculated as low as 41 and 49 for<br>
Platanista gangetica and Sus scrofa These species were amplified efficiently using the<br>
primers 'AFF' and 'APR' (results shown in Figure 3) (keeping all other conditions<br>
standard . the conditions mentioned in 'Example 3'). The lowest P,S scores (86, 55 and<br>
73,51 for species Eumeces egregious) for our primers 'mcb398' and 'mcb869',<br>
respectively, were higher then the above range of combined P, S scores of 'AFF' and<br>
'AFR' for species Sus scrofa (87, 52 and 87, 41), which was efficiently amplified by the<br>
primers 'AFF' and 'AFR'. It gives an indication that the primers 'mcb 398' and 'mcb 869'<br>
would work with all the species including Eumeces egregious efficiently to give rise to the<br>
expected product in PCR. This experiment confirmed that the primers 'mcb398* and 'mcb<br>
869* are capable of amplifying the cytochrome b fragment of most of the animal species in<br>
a universal manner.<br>
For further confirmation of universal nature of our primers, we blasted the sequence of our<br>
primers against the mito and nr databases of NCBI using BLAST software. The results of<br>
these analyses are shown in Tables 10, and 11. respectively.<br>
Finally, the universal nature of the primers was tested in our laboratory with some more<br>
animal species listed in Table 12. These primers amplified all the animal species<br>
efficiently, giving rise to the band of expected size (472 bp). The results are shown in<br>
Figure 4. This experiments substantiated the results of P.S analysis and other in-silico<br>
analyses to show that the primers 'mcb398'and 'mcb 869' arc universal primers.<br>
The flow chart of establishing identity of the species of biological material of unknown<br>
animal origin using primers 'mcb398' and 'mcb869'<br>
Biological material of unknown animal origin<br>
DNA isolation<br>
PCR amplification of DNA isolated using primers 'mcb39S' and 'mcb869'<br>
Sequencing at both the strands in triplicate (using any standard procedure of sequencing such as using ABI<br>
Prism 3700, PE-Applied Bio-systems)<br>
in Mrtjfr database BLAST of revealed sequence in mi^ database aaffN 1CBI fhnp://www. ncbLnlm. nih. eov/B L^ST<br>
(it gives idea about the family of the analyzed material by producing the most significant alignment of the<br>
query sequence with the sequences registered in database)<br>
BLAST of revealed sequennccnnnn nr database of NCBI<br>
(http://www.ncbi.nlni. nfli.gov/BLAST<br>
(it gives idea about the Genus or more precisely, species of the analyzed material by producing the most<br>
significant alignment of the query sequence with the sequences registered in database)<br>
Selection of reference animals belonging to the family/Genus/and species revealed by mito and nr BLAST<br>
searches<br>
Isolation of DNA from the blood of known reference animals; PCR amplification<br>
using primers 'mcb398' and 'mcb869'; sequencing of the PCR products in triplicate using the same primers<br>
Multiple sequence alignments of the revealed sequences of mitochondria) cytochrome b gene of known<br>
reference animals and the biological material of unknown animal origin using software such as<br>
Autoossembler (/CLUST^L X (i.S)<br>
Identification of sequence from the aligned sequences<br>
that is homologous (or significantly similar) to the cytochrome b yene sequence of the DNA obtained from<br>
biological material of unknown animal origin.<br>
The species of homologous sequence would be the species of the biological material under mvcsiigation<br>
Examples<br>
Example 1<br>
Example for identification of a fragment of cytochrome b gene fulfilling the requirements<br>
of columns 1, 2 and 3 mentioned under sub-heading 'Objectives of invention' of heading<br>
'Brief summary of invention'<br>
The cytochrome b molecule has very vastly been used in molecular taxonomic studies.<br>
Being a slow evolving gene, It has a tremendous infonnation in its nucleotide sequences to<br>
distinguish the animals to their family, genus and species sources1'65. A vast database of<br>
the sequences of cytochrome b gene of different animal species has accumulated in the nr<br>
and mito databases of NCBI. We have explored these qualities of cytochrome b gene to<br>
establish the id?- •/ of confiscated remains of unknown animal origin to its family, genus<br>
and species :ources. For this purpose, we have identified a fragment of cytochrome b gene<br>
which is highly p•., .orphic inter-specifically, however, it is monomorphic among the<br>
individual of same species, therefore it can group the individual of an unknown species<br>
with the known individuals of reference species to which it belongs. In order to amplify<br>
this fragment from DNA isolated form any unknown origin, it was necessary that it remain<br>
flanked with the highly conserved sequences amongst a vast range of animal families. To<br>
identify such a unique ment within the cytochrome b gene, we aligned the sequences of<br>
221 distantly related animal species (listed in Table 1) representing various families using<br>
software CLUSTAL X (1.8). These sequences were obtained from public database NCBI<br>
 (httD.://www,ncbi.nlm.nih.gov). The aligned data was examined carefully for the conserved<br>
sites amongst all the species included in in-silico analysis. We identified a fragment (472<br>
bp) of cytochrome b gene that was fulfilling all the requirements mentioned above and also<br>
under column 1,2 and 3 of sub-heading 'Objectives of invention'.<br>
As for the identity of this fragment we would like to mention that it includes the<br>
nucleotides between 398 to 869 in Antilope cervicapra and Felis catus; however, 399 to<br>
870 in Homo sapiens sapiens species. Except at few positions marked as star (*) in Table<br>
2, the nucleotide sequences of this fragment are highly variable amongst the animal<br>
species, giving rise to their unique molecular signature. These molecular signatures are<br>
characteristic of its species and form the basis of revealing the identity of the biological<br>
material of an unknown animal origin by the procedure invented by us. Considering<br>
Antilope cervicapra as a representative species, the sequence of this fragment is mentioned<br>
herewith:<br>
Mitochondrial cytochrome b gene sequence (398-869 bp) of Antilope cer\-icapni<br>
"taccatgaggacaaatatctttttgaggagcaacagtcatcaccaatctcctttcagcaatcccatacatcggtacaaacctagUga<br>
atgaatctgaggagggttctcagtagataaagcaacccttacccgatttttcgccttccactttatcctcccatttatcattgcagccctt<br>
accatagtacacctactgtnctccacgaaacaggatccaacaaccccacaggaatctcatcagacgcagacaaaattccattccac<br>
ccctactacactatcaaagatatcctaggagctctactattaattttaaccctcatgcttctagtcctanctcaccggacctgcttggag<br>
acccagacaactatacaccagcaaacccacttaatacacccccacatatcaagcccgaatgatacttcctatttgcatacgcaatcct<br>
ccgatcaattcctaacaaactaggagg"<br>
Example 2:<br>
Example for development of universal primers to amplify the fragment identified<br>
mentioned under'Example 1'.<br>
A pair of universal primer was designed which has the following features:<br>
1. It targets the fragment identified (mentioned under 'Example I') to amplify it in<br>
polymerase chain reaction (PCR).<br>
2. Its 3' and 5' ends that are highly conserved (marked as star (*) in Table 2),<br>
amongst a vast range of animal species ensuring the amplification of the fragment<br>
mentioned above in a universal manner. The sequencing of the fragment amplified<br>
by these primes reveals the molecular signature of the species of analyzed material,<br>
which on comparison with the sequences of the known reference animals reveals<br>
the identity ur ii" species of unknown biological material under investigation.<br>
3. The tm (melting temperature) of both primers was almost similar (about 58 degree<br>
centigrade) ensuring the significant annealing of both the primers to its template,<br>
therefore significant amplification of targeted region in PCR.<br>
4. The internal stability and P, S, score of the primers were ensured higher while<br>
designing it. The possibilities of internal loop formation, dimmer fonration etc<br>
were also excluded by selecting its sequence uniquely. This ensured that the primer<br>
would be a good primer to be used in PCR for amplification of DNA from<br>
unknown animal origin.<br>
5. The 3' end of the primers were ensured to have either 'G' or *C% to increase the<br>
probability of strong bonding at its 3'ends, which is necessary for efficient<br>
amplification of DNA template in PCR. It also strengthens the universal nature of<br>
the primer.<br>
6. The sequences of trie primers were ensured to be unique so that it does not give rise<br>
to non-specific and spurious products in PCR leading to confusion. U improved the<br>
efficiency and quality of the technique invented by us.<br>
7. These primers were named as 'mcb398' and 'mcb869' because of its property to<br>
amplify a region of mitochondrial cytochrome b gene between nucleotides 3.98 to<br>
869 ofAntilope cervicapra, a representative animal species for this invention. We<br>
took this animal species as representative species because the idea of developing<br>
such a novel primers came in the mind of inventors while they were working on the<br>
genome of this animal in Centre for Cellular and Molecular Biology, Hyderabad,<br>
India.<br>
8. The sequences of the universal primers invented are as follows:<br>
Primers name Sequence (5 '-3')<br>
'mcb398' "TACCATGAGGACAAATATCATTCTG"<br>
'mcb869* "CCTCCTAGTTTGTTAGGGATTGATCG"<br>
Example 3:<br>
Example for development of universal PCR conditions to ensure the amplification of a<br>
template of any unknown origin in PCR, hence strengthening the universal nature of the<br>
technique invented by us<br>
The PCR conditions developed had the following unique features:<br>
1 These were capable of amplifying the DNA template of any animal origin in an<br>
universal manner using the universal primers mentioned under 'Example 2'.<br>
2. The conditions were selected to ensure the; comparable annealing temperature for<br>
both the primers i.e. Vmcb398' and 'mcb869'.<br>
3. The PCR conditions standardized herewith are universal; therefore, the<br>
possibility of PCR failure with a template of unknown origin due to non-standard<br>
conditions is excluded. It ensures the universal nature of our technique to be used in<br>
wildlife forensics.<br>
4. The universal conditions mentioned above are.'<br>
Amplification reactions should be carried out in 20 \i\ reaction volume containing<br>
approximately 20 n.g of template DNA, lOGym each of dNTPs, 1.25 pmole of each primer,<br>
1.5mM MgCIj, 0.5 unit of Amplify Gold (Perkin-Elmer-Cetus, USA) DNA polymerase<br>
and IX PCR buffer (lOmM Tris-HCI, pH 8.3. and 50mM KG). The ampWfication profiles<br>
followed should be. an initial denaturation at 95nC for 10 min. followed by 35 cycles each<br>
of denaturation at 95°C for 45 s. annealing at 51°C for 1 min, and extension at 72"C for 2<br>
min. The extension step at 35th cycles should be held for 10 min.<br>
Example 4:<br>
Establishing the universal nature of our primer and experimental evidences to demonstrate<br>
the universal nature of primers:<br>
The universal nature of the primers 'mcb398' and 'mcb 869' was ensured by the following<br>
measures:<br>
(a) Selecting the primers from the aligned cytochrome b gene sequences of 221 animal of<br>
distantly related species:<br>
The cytochrome b gene sequences (1140 bp) were aligned using software CLUSTAL X<br>
(1.8). The region of cytochrome b gene that was most conserved amongst 221 animal<br>
species was selected to design the primers.<br>
(b) Selecting the 3' and 5* ends of the primers at the highly conserved positions of<br>
cytochrome b gene:<br>
The 3' and 5' ends of the primers were ensured to anneal to a highly conserved position<br>
amongst 221 animal species representing a vast range of animal families. It was done to<br>
ensure an efficient amplification of all the species in PCR. These positions are shown with<br>
star (*) mark in Table 2.<br>
(c) Ensuring either 'G' or 'C' at the 3" end of the primers:<br>
It was ensured the primers to have either 'G' or 'C' at its 3' ends as these are the<br>
nuclt..uiucs liiLit ensure the strong bonding at the 3' ends of the primers due to three<br>
hydrogen bonds while pairing with each other. The strong bonding at 3' ends fteips the<br>
primers to anneal properly with its template resulting in significant amplification in PCR.<br>
(d) Selecting the sequences of the primers to ensure a higher internal stability, higher P, S<br>
score, and no primer dimmer and loop formation:<br>
The sequences of the primers were selected to have a high P, S score for a vast range of<br>
animal species (Shown in Table 1). The care was taken to exclude the possibilities of loop<br>
or primer dimmer formation thai could reduce the efficiency of the pnmers in PCR.<br>
(e) Selecting the sequence of the primers with a comparable melting temperature:<br>
The sequences of the primers were selected to have a comparable melting temperature so<br>
that these could work together to amplify a DNA template in PCR at a similar annealing<br>
temperature. The melting temperature of both the primers was about 58 degree centigrade<br>
and the annealing temperature used in PCR is 51 degree centigrade.<br>
Experimental evidences to demonstrate the universal nature of primers:<br>
( I ) Evidence from In-silico analysis :<br>
 (a) Selecting the primers within the most conserved region of mitochondria! cytochrome b<br>
gene<br>
As mentioned above, the primers were designed to anneal within a highly conserved region<br>
of mitochondrial cytochrome b gene fragment of 472 bp. Table 2 presents the alignment of<br>
the above fragment of cytochrome b gene of 221 animal species representing a vast range<br>
of animal families. The conserved positions of nucleotide sequences are shown with star<br>
(*) mark in Table 2<br>
Table 2 also demonstrates that the 3' ends of the primers are highly conserved amongst all<br>
the animal species analyzed in-silico. In the aligned sequences, the conserved nucleotides<br>
are marked with symbol •'*). Also, the 5' end of the primers were selected within the<br>
conserved region of cytochrome b gene to improve the probability and stability of match of<br>
the primers to their targe1 .equences (i.e. the above mentioned 472 bp fragment of<br>
cytochrome b gene). The primers were thoroughly checked for internal stabilities, loop or<br>
dimmer formation using different software viz., 'Amplify (1.2)', 'PrimerS'<br>
(http://www.Eenome.wi.mit.edu/cgi-bin/primer/prirner3.cgi) as well as manually.<br>
(b) P, S, score ar alys?<br>
We assigned the P,S score /'^Probability of match, S=Stability of match) to the primers<br>
for each template using the software Amplify (1.2). The higher scores of P and S ensure a<br>
good ampl.M
3') are optimum. The Highest score for 'mcb398' was 98,63 (i.e. the situation where the<br>
primer has perfect match with template); however, the highest P, S for 'mcb869' was<br>
recorded as 98, 68 for a complete match between the primer and template. The lowest P,S<br>
score observed for 'mcb398' was 81,50 for species Talpa europaea whereas 'mcb869' had<br>
a high P, S score for this species (92, 57). The another species which have lowest P, S<br>
score for one of the two primers were Eumeces egregious and Equus ainus. Eumeces<br>
egregious had P, S score 86, 55 and 73,51 for 'mcb398' and 'mcbS69', respectively;<br>
however, the P, S score of Equus aima was calculated as 91,61 and 73, 51 for 'mcb398'<br>
and 'mcb869', respectively. All other animals had higher P, S scores then the above<br>
mentioned species. To ensure that these primers,would work efficiently with the DNA<br>
template from the animals having the lowest P, S score for one of the primers, we designed<br>
an another experiment to validate the lower limits.of one of the two primers sufficient for<br>
efficient amplification in PCR. We designed an another primer pair<br>
(AFF= 5 ctagtagaatgaatctyaggugyr and AFR= 5 tatgcaaataggaagtatcattc3.) that have more<br>
mis-pairing at their annealing sites (but not at ends), therefore have less internal stability<br>
13<br>
and lower P, S scores for its templates (listed in Table 9). The P,S scores of 'Af F' and<br>
'APR' were as calculated as low as 41 and 49 for Platanista gangetica and Sus scrofa<br>
These species were amplified efficiently using the primers 'AFF' and 'AFR' (results<br>
shown in Figure 3) (keeping all other conditions standard i.e. the conditions mentioned in<br>
'Example 3')- The lowest P,S scores (86, 55 and 73,51 for species Eumeces egregious) for<br>
our primers 'mcb398' and *mcb869', respectively, were higher then the above range of<br>
combined P, S scores of 'AFF' and 'APR' for species Sus scrofa (87, 52 and 87, 41),<br>
which was efficiently amplified by the primers 'AFF' and 'AFR'. It gives an indication<br>
that the primers 'mcb 398' and 'mcb 869' would work with all the species deluding<br>
Eumeces egregious efficiently to give rise to the expected product in PCR. This<br>
experiment confirmed that the primers 'mcb398' and 'mcb 869* are capable of amplifying<br>
the cytochrome b fragment of most of the animal species in a universal manner.<br>
© BLAST analysis:<br>
The sequences of primers 'mcb398' and 'mcb869' were blasted against mito and nr<br>
databases of NCBI to see its significant alignments with the sequences registered in<br>
GenBank. As expected, the most significant alignments of the sequences were found with<br>
the cytochrome b gene regions (within the 472 bp fragment mentioned in 'Example 1') of<br>
different animal species. This analysis also showed that the 3' as well as 5' ends of the<br>
primers were highly conserved amongst a vast range of animal species, confirming the<br>
universal nature of the primers (Tables 10 and 11, respectively)<br>
(2) Evidence from bench work/experiments done in laboratory conditions:<br>
The DNA from different animals belonging to distantly related species (mentioned in<br>
Table 12) was isolated and subjected to PCR amplification using the primers invented by<br>
us i.e. the primers 4mcb398' and 'mcb869' The PCR products amplified were resolved in<br>
agarose gel by electrophoresis and visualized under UV light. The PCR products of<br>
expected size (472bp) were obtained from all the animals confirming the universal nature<br>
of our primers. These results are shown in Figure 4,<br>
Example 5:<br>
Example to establish the identity of confiscated remains from unknown animal origifl<br>
using the universal primers 'mcb398' and 'mcb869The step-vise procedure to establish the identity of iho biological material from M<br>
unknown animal source is mentioned below:<br>
Biological material of unknown animal origin<br>
DNA isolation<br>
PCR amplification of DNA isolated using primers 'mcb398' and 'mcb869'<br>
Sequencing at both the strands in triplicate (using any standard procedure of sequencing<br>
such as using ABI Prism 3700, PJE-Appiied Bio-systems)<br>
BLAST of revealed sequence in mito database of NCBI<br>
fhttP://www.ncbi.nlrn.nih.eov/BLAST<br>
(it gives idea about the family of the analyzed material by producing the most significant<br>
alignment of the query sequence with the sequences registered in database)<br>
BLAST of revealed sequence in nr database of NCBI<br>
(http://www.ncbi.nlm.nih.gov/BLAST<br>
(it gives idea about the Genus or more precisely, species of the analyzed material by<br>
producing the most significant alignment of the query sequence with the sequences<br>
registered in database)<br>
Selection of reference animals belonging to the family/Genus/and species revealed by mito<br>
and nr BLAST searches<br>
Isolation of DNA from the blood of known reference animals;<br>
PCR amplification using primers 'mcb398' and 'mcb869'; sequencing of the PCR<br>
products in triplicate using the same primers<br>
Multiple sequence alignments of the revealed sequences of mitochondrial cytochrome b<br>
gene of known reference animals and the biological material of unknown animal origin<br>
using software such as Autoasscmbler/CLUSTAL X(l.S)<br>
20<br>
Identification of sequence from the aligned sequences<br>
that is homologous (or significantly similar) to the cytochrome b gene sequence of the<br>
DNA obtained from biological material of<br>
unknown animal origin.<br>
The species of homologous sequence would be the species of the biological material under<br>
investigation<br>
Application of the above information for the objectives mentioned in columns 7-13 under<br>
sub-heading 'Objective of invention' of heading 'Summary of invention'<br>
Example 6:<br>
The actual execution of the technique invented<br>
As a first application and to demonstrate the ease and utility of this method, we<br>
investigated a case of forensic identification submitted at our laboratory to seek scientific<br>
opinion on animal hunting evidence. In this case, we received the half h;;rned re" &gt; a ins of<br>
an unknown animal, confiscated by the crime investigation agencies. The DNA was<br>
isolated from the above material following standard methods74 and subjected to PCR<br>
amplification using the primers mentioned above (viz., 'mcb398' and 'mcb869').<br>
Amplification reactions were carried out in 20 nl reaction volume containing 20 rjg of<br>
template DNA, lOOfim each of dNTPs, 1.25 pmole of each primer, 1.5mM MgCl2.0.5 unit<br>
of Amplify Gold (Perkin-Elmer-Cetus, USA) DNA polymerase and IX PCR buffer<br>
(lOrnM Tris-HG, pH 8.3, and 50mM KC1). The amplification profiles followed were: an<br>
initial denaturation at 95°C for 10 min, followed by 35 cycles each of denaturation at 95°C<br>
for 45 s, annealing at 51°C for I min, and extension at 72°C for 2 min. The extension step<br>
at 35"" cycles was held for 10 min.<br>
The PCR products obtained were sequenced in automated work station (ABI Prism 3700,<br>
PE-Biosystems) on both strands in triplicate and the sequence resolved (328 bp, shown in<br>
Figure la) was blasted against mito databases of NCBI using BLAST program73. The most<br>
significant alignment (bits Value 365, E value e"101) of this sequence was produced with<br>
the cytochrome b gene sequence of Felis cutus, (Table 3) indicating that species of<br>
analyzed material belongs to family felidae. Further, the above sequence revealed from the<br>
confiscated remain was blasted against nr databases of NCBI using BLAST program. The<br>
most significant alignment (bits Value 603, E value e"170) of this sequence was produced<br>
with the cytochrome b gene sequence of Panthera pardus (Table 4), indicating the identity<br>
of the analyzed material as that of a Panthera pardus source. Based on this information,<br>
we selected the reference animals listed in Table 5 representing different species and<br>
subspecies of felidae. The DNA isolated from reference animals was amplified and<br>
sequenced on both strands in triplicate using the primer pair mentioned above. Consensus<br>
sequences obtained were aligned using program CLUSTAL X (1.8) (Table 6). Sequence<br>
comparisons identified 113 variable sites in total amongst all animals analyzed 'Table 7).<br>
Pair-vise comparisons of sequences were performed to find out the variation among<br>
different animals investigated. All the species investigated were differentiated by a their<br>
unique nucleotides sequences. The molecular signatures of different reference animals<br>
were compared with the molecular signature of the confiscated skin 'adil.flesh'. Table 7<br>
demonstrate that the maximum similarity of the adil.flesh with 'gzll' i.e. known Leopard<br>
(Panthera pardus) species, indicating the identity of the adil.flesh, the confiscated skin, as<br>
that of a Panthera pardus origin. We also calculated the similarity matrix showing the pairvise<br>
similarity amongst the animal species under investigation using PHYLIP software<br>
Thi.s matrix is -':.' .vn in Table 8. It demonstrates that the animals belonging to different<br>
species had more variation; however, the animals of same species had maximum similarity<br>
among their cytochrome b sequences. The cytochrome b gene sequence of DNA isolated<br>
from confiscated material had maximum similarity with the sequences obtained from<br>
known Leopard source(99.7%, and 98.2 with 'gzll' and 'gz21f, respectively); establishing<br>
the identity of the source of confiscated material as that of a Normal leopard (Penthera<br>
pardus) species. The step-vise procedure involved in above analysis is illustrated in Figure<br>
la, Iband ic, respectively.<br>
Thus, the primers invented by us can generate the molecular signature from any biological<br>
material of unknown animal origin, which actually is the characteristic of its family, genus<br>
and more precisely, the species. When these signatures are compared in-silico with the<br>
signatures already available in public databases (viz.. GenBank. NCBI database etc) using<br>
BLAST software73, it indicates identity of the family, genus or species of the analyzed<br>
material, which in turn is confirmed practically by comparing with the reference animals of<br>
the revealed family, genus or species, by including them in the further analysis by the<br>
primers 'mcb398' and 'mcb8G9'. Application Of the information revealed could be in<br>
fulfilling the requirements of objectives mentioned in columns 7-13 under sub-heading<br>
'Objective of invention' of heading 'Summary of invention'<br>
The method of the invention can be used to establish the identity of confiscated animal<br>
parts and products is one of the key requirements of wildlife identification in forensics. It<br>
is needed to establish the crime with the criminal beyond a reasonable doubt to avoid the<br>
human violation of wildlife resources. Various morphological biochemical and molecular<br>
approaches have been given for this purpose; however, none of the current methods is<br>
universally applicable to detect the mutilated animal remains of unknown origin. We have<br>
identified a fragment on the mitochondria! cytochrome b gene, which has enormous<br>
information to differentiate among various animal species back to the family, genus and<br>
species sources. We have also found that this fragment is flanked by the highly conserved<br>
sequences amongst a vast range of animal species. We invented a pair of universal primer<br>
that can amplify this fragment of DNA isolated from the biological material of an<br>
unknown animal origin in polymerase chain reaction (PCR) to reveal its identity at species<br>
and sub-species sources. This novel invention has great potential to revolutionize the<br>
whole scenario of wildlife forensic identification and crime investigation.<br><br><br><br><br><br>
We claim:<br>
1.	A universal primer pair for amplifying a fragment of cytochrome b gene of an animal<br>
species in a polymerase chain reaction (PCR) or determining the identity of the<br>
biological material of an animal of unknown origin at species and sub-species level, said<br>
primer pair having the sequence:<br>
primer name	sequence [5' - 3']<br>
mcb398:	"TACCATGAGGACAAATATCATTCTG"<br>
mcb 869:	"CCTCCTAGTTTGTTAGGGATTGATCG"<br>
wherein the said primers being capable of selectively amplifying a fragment of about<br>
472 base pairs of a mitochondrial cytochrome b gene of any of the 221 animal species,<br>
wherein the fragment being selectively amplified has a sequence that varies among each<br>
of the said 221 animal species.<br>
2.	The primers as claimed in claim 1, wherein the said primers are capable of amplifying a fragment of mitochondrial cytochrome b gene that is flanked by highly conserved sequences amongst a vast range of animal species.<br>
3.	The primers as claimed in claim 1, wherein the said primers are capable of amplifying a fragment of mitochondrial cytochrome b gene which is polymorphic inter-specifically, but monomorphic at intra species sources.<br>
4.	The primers as claimed in claim 1, wherein when used to amplify a fragment of cytochrome b gene of Antilope cervicapra species, the sequences of the fragment is as follows:<br>
Mitochondrial cytochrome b gene sequence (398-869 bp) of Antilope cervicapra:<br>
"taccatgaggacaaatatctttttgaggagcaacagtcatcaccaatctcctttcagcaatcccatacatcggtacaaacctagt<br>
agaatgaatctgaggagggttctcagtagataaagcaacccttacccgatttttcgccttccactttatcctcccatttatcattgca<br>
gcccttaccatagtacacctactgtttctccacgaaacaggatccaacaaccccacaggaatctcatcagacgcagacaaaatt<br>
ccattccacccctactacactatcaaagatatcctaggagctctactattaattttaaccctcatgcttctagtcctattctcaccgga<br>
cctgcttggagacccagacaactatacaccagcaaacccacttaatacacccccacatatcaagcccgaatgatacttcctattt<br>
gcatacgcaatcctccgatcaattcctaacaaactaggagg".<br>
5.	A process for determining the identity of an unknown animal from a biological sample,<br>
said process comprising the steps of:<br>
[a]     isolating and amplifying the DNA from the said biological sample to be tested using the primer pair as claimed in claim 1 to form amplified DNA;<br><br>
[b]	sequencing the amplified DNA as obtained in step [a];<br>
[c]	blasting the sequence resolved in step (b) against a mitochondrial database such as that of National Centre for Biotechnology Information (NCBI) using BLAST program and determining the most likely family of the animal source of the biological sample;<br>
[d]	blasting the sequence resolved in step (b) against a non-redundant (nr) database such as that of National Centre for Biotechnology Information (NCBI) using BLAST program and determining the most likely genus, species or more precisely the sub-species of the animal source of the biological sample;<br>
[e]	identifying the most significant alignment of the sequence resolved with cytochrome b gene sequence of the animal identified in steps (c) and (d) respectively and selection of these animals as "reference animals' for further studies;<br>
[fj isolating and amplifying and sequencing the DNA sequences from the reference animal on both strands in triplicate using the primer pair as claimed in claim 1;<br>
[g] aligning the sequences obtained using CLUSTAL program and identifying the variable sites amongst the animals analyzed;<br>
[h] comparing the nucleotide sequences pair-wise to determine the variation among the animals resolved and identifying the nucleotide sequence to which the DNA sequence of the biological sample bears maximum similarity as the source animal of the biological sample.<br>
6. A process as claimed in claim 5, wherein the amplification reactions are carried out in 20micro litres reaction volume containing approximately 20 narro gram of template DNA, 100 micro mole each of dNTPs, 1.25 pico mole of each primer, 1.5 mM MgCl2, 0.5 unit of AmpliTaq Gold (Perkin-Elmer-Cetus, USA) DNA polymerase and IX PCR buffer (10 mM Tris-HCl, pH 8.3, and 50 mM KC1), and the amplification profiles followed are: an initial denaturation at 95 degree C for 10 minutes, followed by 35 cycles each of denaturation at 95 degree C for 45 seconds, annealing at 51 degree C for 1 minute and extension at 72 degree C for 2 minutes, wherein the extension step at 35th cycle is held for 10 minutes.<br><br>
7.	A process as claimed in claim 5, wherein it enables identification of species of analyzed material (i.e. the DNA isolated from confiscated animal remain of unknown origin) using a public databases such as GenBank, NCBI and the like.<br>
8.	A process as claimed in claim 5, wherein the sample of the biological materials is preferably skin, blood, horns, etc.<br>
9.	A universal primer pair and the process for the identification of an unknown animal from a biological sample substantially as herein described with reference to the foregoing examples.<br></mcfa869></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE1MzQtZGVsbnAtMjAwMy1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">01534-delnp-2003-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE1MzQtZGVsbnAtMjAwMy1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">01534-delnp-2003-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE1MzQtZGVsbnAtMjAwMy1jb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">01534-delnp-2003-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE1MzQtZGVsbnAtMjAwMy1kZXNjcmlwdGlvbiAoY29tcGxldGUpLTI1LTA4LTIwMDgucGRm" target="_blank" style="word-wrap:break-word;">01534-delnp-2003-description (complete)-25-08-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE1MzQtZGVsbnAtMjAwMy1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">01534-delnp-2003-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE1MzQtZGVsbnAtMjAwMy1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">01534-delnp-2003-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE1MzQtZGVsbnAtMjAwMy1mb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">01534-delnp-2003-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE1MzQtZGVsbnAtMjAwMy1mb3JtLTE4LnBkZg==" target="_blank" style="word-wrap:break-word;">01534-delnp-2003-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE1MzQtZGVsbnAtMjAwMy1mb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">01534-delnp-2003-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE1MzQtZGVsbnAtMjAwMy1mb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">01534-delnp-2003-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE1MzQtZGVsbnAtMjAwMy1wY3QtMjEwLnBkZg==" target="_blank" style="word-wrap:break-word;">01534-delnp-2003-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUzNC1ERUxOUC0yMDAzLUFic3RyYWN0LSgwMy0xMC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1534-DELNP-2003-Abstract-(03-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUzNC1ERUxOUC0yMDAzLUNsYWltcy0oMjUtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1534-DELNP-2003-Claims-(25-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUzNC1ERUxOUC0yMDAzLUNsYWltcy0oMjYtMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1534-DELNP-2003-Claims-(26-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUzNC1kZWxucC0yMDAzLWNvbXBsZXRlIHNwZWNpZmljYXRpb24gKGdyYW5kZWQpLnBkZg==" target="_blank" style="word-wrap:break-word;">1534-delnp-2003-complete specification (granded).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUzNC1ERUxOUC0yMDAzLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDMtMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1534-DELNP-2003-Correspondence-Others-(03-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUzNC1ERUxOUC0yMDAzLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjUtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1534-DELNP-2003-Correspondence-Others-(25-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUzNC1ERUxOUC0yMDAzLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjYtMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1534-DELNP-2003-Correspondence-Others-(26-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUzNC1ERUxOUC0yMDAzLURyYXdpbmdzLSgyNS0wOC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1534-DELNP-2003-Drawings-(25-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUzNC1ERUxOUC0yMDAzLUZvcm0tMS0oMDMtMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1534-DELNP-2003-Form-1-(03-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUzNC1ERUxOUC0yMDAzLUZvcm0tMS0oMjUtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1534-DELNP-2003-Form-1-(25-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUzNC1ERUxOUC0yMDAzLUZvcm0tMi0oMDMtMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1534-DELNP-2003-Form-2-(03-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUzNC1ERUxOUC0yMDAzLUZvcm0tMi0oMjUtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1534-DELNP-2003-Form-2-(25-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUzNC1ERUxOUC0yMDAzLUZvcm0tMi0oMjYtMDktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1534-DELNP-2003-Form-2-(26-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUzNC1ERUxOUC0yMDAzLUZvcm0tMy0oMjUtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1534-DELNP-2003-Form-3-(25-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUzNC1ERUxOUC0yMDAzLVBldGl0aW9uLTEzNy0oMjUtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1534-DELNP-2003-Petition-137-(25-08-2008).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="224400-isolated-cdna-molecule-with-a-nucleotide-sequences.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="224402-a-n-aroyl-cyclic-amines-compound-of-formula-i.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>224401</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>01534/DELNP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>44/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>31-Oct-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>14-Oct-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>24-Sep-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>COUNCIL OF SCIENTIFIC AND INDUSTRIAL RESEARCH</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>RAFI MARG, NEW DELHI-110 001, INDIA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SUNIL KUMAR VERMA</td>
											<td>CENTRE FOR CELLULAR AND MOLECULAR BIOLOGY, HYDERABAD</td>
										</tr>
										<tr>
											<td>2</td>
											<td>LALJI SINGH</td>
											<td>CENTRE FOR CELLULAR AND MOLECULAR BIOLOGY, HYDERABAD</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IN01/00055</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-03-28</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/224401-a-universal-primer-pair-for-amplifying-a-fragment-of-cytochrome-b-gene-of-an-animal-species by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:29:22 GMT -->
</html>
