<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/224716-degradation-of-hydrophobic-ester-pesticides-and-toxins by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:08:35 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 224716:DEGRADATION OF HYDROPHOBIC ESTER PESTICIDES AND TOXINS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">DEGRADATION OF HYDROPHOBIC ESTER PESTICIDES AND TOXINS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>An invention discloses a method of eliminating or reducing the concentration of a hydrophobic ester pesticide or toxin in a sample, the method comprising contacting the sample with an Dipteran esterase, or a mutant thereof, wherein the Dipteran esterase, or a mutant thereof, comprises a sequence selected from the group consisting of: i) a sequence as shown in SEQ ID NO:1, ii) a sequence as shown in SEQ ID NO:2, and iii) a sequence which is at least 40% identical to i) or ii) which is capable of hydrolysing a hydrophobic ester pesticide or toxin.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>DEGRADATION OF HYDROPHOBIC ESTER PESTICIDES AND TOXINS<br>
Field of the Invention:<br>
This invention relates to enzymes and methods for degrading<br>
hydrophobic ester pesticides and toxins. In particular, the present invention<br>
relates to the use of insect esterases, such as ?-carboxylesterases, and mutants<br>
thereof, in the bioremediation of pyrethroid residues contaminating the<br>
environment and horticultural commodities.<br>
Background of the Invention:<br>
Pyrethroids constitute a major class of chemical pesticides. They are<br>
synthetic analogues of the natural pyrethrins, which are produced in the<br>
flowers of the pyTethrum plant [Tanacetum cinerariifolium). Modification of<br>
their structure has yielded compounds that retain the intrinsically modest<br>
vertebrate toxicity of the natural products but are both more stable and more<br>
potent as pesticides. In the thirty years since their introduction they have<br>
risen to about 10-20% of insecticide sales worldwide and they are projected to<br>
retain substantial market share into the forseeable future. They are now<br>
widely used across agricultural production and processing systems in many<br>
countries and have caused residue incidents in diverse commodities ranging<br>
from cotton and horticulture through to wool.<br>
Residues of pyrethroid pesticides are undesirable contaminants of the<br>
environment and a range of commodities. They are undesirable because of<br>
the broad target range of the pesticide across invertebrates and their<br>
significant toxicity to vertebrates, although they are generally considered to<br>
be amongst the safest pesticides to mammals. Areas of particular sensitivity<br>
include contamination of soil, irrigation tailwater that is re-cycled, used by<br>
irrigators downstream or simply allowed to run off-farm, and residues above<br>
permissible levels in horticultural exports. Animal industries also have<br>
problems with pesticide-contaminated commodities arising through either<br>
their own pesticide use or their reliance on crop products and by-products as<br>
fodder. Processing wastes from food processing plants, carpet dye baths and<br>
animal dips are also contaminated, sometimes quite heavily, with pesticide<br>
residues. Bioremediation strategies are therefore required for eliminating or<br>
reducing these pesticide residues.<br><br>
One proposed bioremediation strategy involves the use of enzymes<br>
capable of immobilising or degrading the pesticide residues. Such enzymes<br>
may be employed, for example, in bioreactors through which contaminated<br>
water could be passed, or in washing solutions after post-harvest<br>
disinfestation of fruit, vegetables or animal products to reduce residue levels<br>
and withholding times. Suitable enzymes for degrading pesticide residues<br>
include OP hydrolases from bacteria, vertebrates and organophosphate (OP)<br>
resistant insects. It is desirable that the hydrolytic enzymes degrade the<br>
pesticide residues at a rapid rate.<br>
Organophosphate resistance in the sheep blowfly, Lucilia cuprina, is<br>
conferred by two different mutations in the gene encoding carboxylesterase<br>
E3. The two mutant enzymes differ in their substrate specificities but<br>
between them can detoxify two major subtypes of OPs. The E3 gene from L<br>
cuprina was cloned by Newcomb et al. (1997) and, using a combination of<br>
DNA sequencing, baculovirus expression and in vitro mutagenesis, these<br>
workers identified the two resistance mutations. One is an Asp for Gly<br>
substitution at residue 137 in the oxyanion hole region of the active site<br>
(Newcomb et al., 1997). The other is a Leu for Trp substitution at residue 251<br>
in the substrate-binding region (Campbell et al., 1998), which results in an<br>
increase in malathion carboxylesterase activity as well as the acquisition of<br>
OP hydrolase activity.<br>
There is a need for methods and enzymes which can be used for the<br>
bioremediation of, for example, soils, foodstuff and water samples<br>
contaminated with hydrophobic ester pesticides and toxins.<br>
Summary of the Invention;<br>
The present inventors have now found that insect esterases, and<br>
mutants thereof, are able to hydrolyse hydrophobic ester pesticides and<br>
toxins such as pyrethroids. The activity of the insect esterases, and mutants<br>
thereof, show a degree of chiral specificity, which differed between mutants.<br>
It is therefore possible to provide a suite of insect esterases, or mutants<br>
thereof, that are able to degrade hydrophobic ester pesticides and toxins that<br>
can act, alone or together, as effective bioremediation agents for hydrophobic<br>
ester pesticides and toxins such as pyrethroids.<br>
Accordingly, in a first aspect, the present invention provides a method<br>
of eliminating or reducing the concentration of a hydrophobic ester pesticide<br><br>
or toxin in a sample, the method comprising contacting the sample with an<br>
insect esterase, or a mutant thereof.<br>
In a preferred embodiment of the first aspect, the insect esterase is a<br>
member of the carboxyl/cholinesterase multi-gene family of enzymes. More<br>
preferably, the insect esterase is an ?-carboxylesterase. Even more preferably,<br>
the insect esterase is a member of the ?-carboxylesterase cluster which forms<br>
a sub-clade within this multi-gene family (Oakeshott et al., 1999). Esterases<br>
which form this sub-clade include at least a-carboxylesterases which can be<br>
isolated from species of Diptera, Hemiptera and Hymenoptera. Specific<br>
enzymes which are found in this sub-clade include, but are not limited to, the<br>
E3 or EST23 esterases. However, orthologous esterases of E3 and EST23 from<br>
other insect species can also be used in the methods of the present invention.<br>
Preferably, the a-carboxylesterases can be isolated from a species of<br>
Diptera. Accordingly, examples of preferred a-carboxylesterases for use in<br>
the present invention are the E3 esterase (SEQ ID NO:1) which is derived<br>
from Lucilia cuprina, or the EST23 esterase (SEQ ID NO:2) which is derived<br>
from Drosophila melanogaster.<br>
In a further preferred embodiment, the mutant insect esterase has a<br>
mutation(s) in the oxyanion hole, acyl binding pocket or anionic site regions<br>
of the active site, or any combination thereof.<br>
In a further preferred embodiment, the mutant a-carboxylesterase is<br>
selected from the group consisting of: E3G137R, E3G137H, E3W251L,<br>
E3W251S, E3W251G, E3W251T, E3W251A, E3W251L/F309L,<br>
E3W251L/G137D, E3W251L/P250S, E3F309L, E3Y148F, E3E217M, E3F354W,<br>
E3F354L, and EST23W251L. Preferably, the mutant a-carboxylesterase is<br>
E3W251L, E3F309L, E3W251L/F309L or EST23W251L.<br>
In another preferred embodiment of the first aspect, the a-<br>
carboxylesterase, or mutant thereof, has a sequence selected from the group<br>
consisting of:<br>
i) a sequence as shown in SEQ ID NO:1,<br>
ii) a sequence as shown in SEQ ID NO:2, and<br>
iii) a sequence which is at least 40% identical to i) or ii) which is<br>
capable of hydrolysing a hydrophobic ester pesticide or toxin. More<br>
preferably, the polypeptide is at least 50% identical, more preferably at least<br>
60% identical, more preferably at least 70% identical, more preferably at least<br>
80% identical, and more preferably at least 90% identical, more preferably at<br><br>
least 95% identical, and even more preferably at least 97% identical to i) or<br>
ii).<br>
As the skilled addressee would be aware, the method of the first aspect<br>
can be performed using more than one insect esterase, or mutants thereof.<br>
This is particularly the case where different insect esterases, or mutants<br>
thereof, have different hydrolytic activity for different stereo-isomers of the<br>
hydrophobic ester pesticide or toxin.<br>
The hydrophobic ester pesticide or toxin can be any molecule which is<br>
hydrophobic in nature, contains an ester group and has some level of toxicity<br>
towards living organisms. A particularly preferred hydrophobic ester<br>
pesticide or toxin is a pyrethroid. The pyrethroid can be a Type I or Type H<br>
pyrethroid. Preferably, the Type I pyrethroid is selected from the group<br>
consisting of: 1S/1R trans permethrin, 1S/1R cis permethrin, NRDC157 IS cis,<br>
and NRDC157 1R cis. Preferably, the Type II pyrethroid is deltamethrin.<br>
Preferably, the sample is a soil sample, a water sample or a biological<br>
sample. Preferred biological samples include matter derived from plants such<br>
as seeds, vegetables or fruits, as well as matter derived from animals such as<br>
meat<br>
Preferably, the method is performed in a liquid containing<br>
environment.<br>
The sample can be exposed to the-insect esterase, or mutant thereof, by<br>
any appropriate means. This includes providing the insect esterase, or<br>
mutant thereof, directly to the sample, with or without carriers or excipients<br>
etc. The insect esterase, or mutant thereof, can also be provided in the form<br>
of a host cell, typically a microorganism such as a bacterium or a fungus,<br>
which expresses a polynucleotide encoding the insect esterase, or mutant<br>
thereof.<br>
The insect esterase, or mutant thereof, can also be as provided a<br>
polymeric sponge or foam, the foam or sponge comprising the insect esterase,<br>
or mutant thereof, immobilized on a polymeric porous support.<br>
Preferably, the porous support comprises polyurethane.<br>
In a preferred embodiment, the sponge or foam further comprises<br>
carbon embedded or integrated on or in the porous support<br>
It is envisaged that the use of a surfactant in the method of the present<br>
invention may liberate hydrophobic ester pesticides and/or toxins from any,<br>
for example, sediment in the sample. Thus increasing efficiency of the<br><br>
method of the present invention. Accordingly, in another preferred<br>
embodiment, the method comprises the presence of a surfactant when the<br>
hydrophobic ester pesticide or toxin is contacted with the insect esterase, or<br>
mutant thereof. More preferably, the surfactant is a biosurfactant<br>
Further, hydrophobic ester pesticide or toxin in a sample can also be<br>
degraded by exposing the sample to a transgenic plant which produces the<br>
insect esterase, or mutant thereof.<br>
In a second aspect the present invention provides a substantially<br>
purified polypeptide which is a mutant of an insect esterase, wherein one or<br>
more mutations are within a region of the esterase selected from the group<br>
consisting of: oxyanion hole, acyl binding pocket and anionic site, wherein<br>
the mutant insect esterase is capable of hydrolysing a hydrophobic ester<br>
pesticide or toxin, with the proviso that the mutant insect esterase is not<br>
E3W251L, E3W251S, E3W251G or E3G137D.<br>
Preferably, the insect esterase is an ?-carboxylesterase.<br>
Preferably, the polypeptide is selected from the group consisting of:<br>
i) a mutant of a sequence as shown in SEQ ID NO:1, and<br>
ii) a mutant of sequence as shown in SEQ ID NO:2,<br>
wherein the mutant is at least 40% identical to at least one of SEQ ID<br>
NO's:l or 2. More preferably, the mutant is at least 80% identical to at least<br>
one of SEQ ID NO's:l or 2. Even more preferably, the mutant is at least 90%<br>
identical to at least one of SEQ ID NO's:l or 2.<br>
Preferably, the mutation is a point mutation.<br>
Preferably, the polypeptide selected from the group consisting of:<br>
E3G137R, E3G137H, E3W251T, E3W251A, E3W251L/F309L,<br>
E3W251L/G137D, E3W251L/P250S, E3F309L, E3Y148F, E3E217M, E3F354W,<br>
E3F354L, EST23W251L.<br>
In a third aspect, the present invention provides a fusion polypeptide<br>
comprising a polypeptide according to the second aspect fused to at least one<br>
other polypeptide sequence.<br>
In a fourth aspect the present invention provides an isolated<br>
polynucleotide encoding a polypeptide according to the second or third<br>
aspects.<br>
In a fifth aspect the present invention provides a vector for replication<br>
and/or expression of a polynucleotide according to the fourth aspect<br><br>
In a sixth aspect the present invention provides a host cell transformed<br>
or transfected with the vector of the fifth aspect<br>
In a seventh aspect the present invention provides a composition for<br>
hydrolysing a hydrophobic ester pesticide or toxin, the composition<br>
comprising a polypeptide according to the second or third aspects, and one or<br>
more acceptable carriers.<br>
In an eighth aspect the present invention provides a method for<br>
generating and selecting an enzyme that hydrolyses a hydrophobic ester<br>
pesticide or toxin, the method comprising<br>
(i) introducing one or more mutations into an insect esterase, or an<br>
insect esterase that has already been mutated, and<br>
(ii) determining the ability of the mutant insect esterase to hydrolyse a<br>
hydrophobic ester pesticide or toxin.<br>
Preferably, the one or more mutations enhances hydrolytic activity<br>
and/or alters the stereospecificty of the esterase.<br>
Such one or more mutations can be introduced by a variety of<br>
techniques known to the skilled addressee. These techniques include, but<br>
are not limited to, site directed mutagenesis, random mutagenesis, or the use<br>
of DNA shuffling in in vitro evolution techniques, each of which are<br>
performed on a polynucleotide encoding the insect esterase or insect esterase<br>
that has already been mutated.<br>
In a preferred embodiment of the eighth aspect, the insect esterase is an<br>
?-carboxylesterase. More preferably, the ?-carboxylesterase is an E3 or<br>
EST23 esterase. More preferably, the a-carboxylesterase has a sequence<br>
selected from the group consisting of:<br>
i) a sequence as shown in SEQ ID NO:1,<br>
ii) a sequence as shown in SEQ ID NO:2, and<br>
iii) a sequence which is at least 40% identical to i) or ii). More<br>
preferably, the polypeptide is at least 50% identical, more preferably at least<br>
60% identical, more preferably at least 70% identical, more preferably at least<br>
80% identical, and more preferably at least 90% identical, more preferably at<br>
least 95% identical, and even more preferably at least 97% identical to i) or<br>
ii).<br>
Preferably, the one or more mutations are within a region of the<br>
esterase selected from the group consisting of: oxyanion hole, acyl binding<br>
pocket and anionic site.<br><br>
In a further preferred embodiment, the insect esterase that has already<br>
been mutated is selected from the group consisting of; E3G137R, E3G137H,<br>
E3W251L, E3W251S, E3W251G, E3W251T, E3W251A, E3W251L/F309L,<br>
E3W251L/G137D, E3W251L/P250S, E3F309L, E3Y148F, E3E217M, E3F354W,<br>
E3F354L, and EST23W25lL.<br>
In a further preferred embodiment of the eighth aspect, the mutation is<br>
a point mutation.<br>
In a ninth aspect the present invention provides an enzyme obtained<br>
by a method according to the eighth aspect.<br>
Throughout this specification the word "comprise", or variations such<br>
as "comprises" or "comprising", will be understood to imply the inclusion of a<br>
stated element, integer or step, or group of elements, integers or steps, but not<br>
the exclusion of any other element, integer or step, or group of elements,<br>
integers or steps.<br>
The invention is hereinafter described by way of the following non-<br>
limiting example and with reference to the accompanying figures.<br>
Brief Description of the Accompanying Drawings:<br>
Figure 1: Amino acid sequence alignment of the E3 (SEQID NO:1) and<br>
Torpedo californica acetylcholinesterase (SEQID NO:3) enzymes. The<br>
sequence around the active site serine and residues Glyl37, Trp251 and<br>
Phe309 are shown in bold and underlined.<br>
Figure 2: Proposed configuration of active site of LcE3 carboxylesterase in an<br>
acylation reaction.<br>
Figure 3: Results of representative titration experiments performed on cell<br>
extracts containing baculovirus expressed esterases.<br>
Figure 4: Molecular structures for 1R/S cis and trans pennethrin, 1R/S cis<br>
and trans NRDC157 and the four stereoisomers of cis deltamethrin.<br>
Figure 5: Hydrolysis of cis and trans pennethrin (0.5?M) by E3W251L.<br>
Key to Sequence Listing;<br>
SEO ID NO:1 - Amino acid sequence of Lucilia cuprina E3 ?-carboxylesterase.<br><br>
SEQ ID NQ:2 - Amino acid sequence of Drosophila melanogaster EST23 ?-<br>
carboxylesterase.<br>
SEQ ID NQ:3 - Partial amino acid sequence of Torpedo californica<br>
acetylcholinesterase.<br>
Detailed Description of the Invention;<br>
General Techniques<br>
Unless otherwise indicated, the recombinant DNA techniques utilized<br>
in the present invention are standard procedures, well known to those skilled<br>
in the art. Such techniques are described and explained throughout the<br>
literature in sources such as, J. Perbal, A Practical Guide to Molecular<br>
Cloning, John Wiley and Sons (1984), J. Sambrook et al., Molecular Cloning:<br>
A Laboratory Manual, Cold Spring Harbour Laboratory Press (1989), T.A.<br>
Brown (editor), Essential Molecular Biology: A Practical Approach, Volumes 1<br>
and 2, IRL Press (1991), D.M. Glover and B.D. Hames (editors), DNA Cloning:<br>
A Practical Approach, Volumes 1-4, IRL Press (1995 and 1996), and F.M.<br>
Ausubel et al. (Editors), Current Protocols in Molecular Biology, Greene Pub.<br>
Associates and Wiley-Interscience (1988, including all updates until present)<br>
and are incorporated herein by reference.<br>
Pyrethroids<br>
Pyrethroids are synthetic analogs of pyrethrum pesticides. For<br>
example, pyrethroids include (in each case common name in accordance with<br>
The Pesticide Manual, 12th Edition): permethrin, fenvalerate, esfenvalerate,<br>
cypermethrin, alpha-cypermethrin, deltamethrin, fenpropathrin, fluvalinate,<br>
flucythrinate, cyfluthrin, acrinathrin, tralomethrin, cycloprothrin, lambda-<br>
cyhalothrin, tefluthrin, bifenthrin, transfluthrin, zeta-cypermethrin, and<br>
halfenprox.<br>
Type I pyrethroid compounds (e.g., permethrin) differ from type II<br>
pyrethroid compounds in that type II compounds possess a cyano group on<br>
the ?-carbon atom of the phenoxybenzyl moiety. Some examples of type II<br>
pyrethroids are cypermethrin, deltamethrin, and fenvalerate.<br>
Examples of pyrethroid pesticides which can be hydrolysed using the<br>
methods of the present invention include, but are not restricted to these<br>
compounds; 3-phenoxybenzyl(1RS)-cis, trans-3-(2,2-dichlorovinyl)-2,2-<br>
dimethylcyclopropane carboxylate [permethrinl, ?-cyano-3-phenoxybenzyl-l-<br><br>
(4-ethoxyphenyl)-2,2-dichlorocyclopropane carboxylate [cyloprothrin], (RS)-<br>
?-cyano-3-phenoxybenzyl(RS)-2-(4-chlorophenyl)-3-isovalerate[fenvalerate],<br>
(S)-?-cyano-3-phenoxybenzyl(S)-2-(4-chlorophenyl)isovalerate<br>
[esfenvalerate], ?-cyano-3-phenoxybenzyl(S)-2-(4-difluoromethoxyphenyl)<br>
isovalerate [flucythrinate], ?-cyano-3-phenoxybenzyl 2-(2-chloro-4-<br>
trifluoromethylaniline)isovalerate [fluvalinate], (RS)-?-cyano-3-<br>
phenoxybenzyl 2,2,3,3-tetramethylcyclopropane carboxylate [fenpropathrin],<br>
3-phenoxybenzyl(lR)-cis,trans-chiysanthemate [d-fenothrin], (RS)-?-cyano-3-<br>
phenoxybenzyl(lR)-cis,trans-chrysanthemate [cyfenothrin], (RS)3-allyl-2-<br>
methyl-4-oxocyclopento-2-enyl(lRS)-cis,trans-chrysanthemate [allethrin], ?-<br>
cyano-3-phenoxybenzyl(lR)-cis,trans-3-phenoxybenzyl(lR)-cis,trans-3-(2,2-<br>
dichlorovinyl)-2,2-dimethylcyclopropane carboxylate [cypennethrin], (S)-?-<br>
cyano-3-phenoxybenzyl(lR)-cis-3-(2,2-dibromovinyl)-2,2-dimethy<br>
lcyclopropane carboxylate [deltamethrin], (S)-?-cyano-3-phenoxybenzyl(lR)-<br>
cis-2,2-dimethyl-3-(l,2,2,2-tetrabro moethyl)cyclopropane carboxylate<br>
[tralomethrin], 3,4,5,6-tetrahydro imidomethyl(lRS) -cis,trans-<br>
chrysanthemate [tetramethrin], 5-benzyl-3-furylmethyl(lRS)-cis,trans-<br>
chrysanthemate [resmethrin], ?-cyano-4-fluoro-3-phenoxybenzyl(lR,trans)-<br>
2,2-dimethyl-3-(2,2-dichl orovinyl)cyclopropane carboxylate [cyfluthrin].<br>
Polypeptides<br>
By "substantially purified" we mean a polypeptide that has been<br>
separated from most of the lipids, nucleic acids, other polypeptides, and other<br>
contaminating molecules with which it is associated in its native state.<br>
The % identity of a polypeptide is determined by GAP (Needleman and<br>
Wunsch, 1970) analysis (GCG program) with a gap creation penalty=5, and a<br>
gap extension penalty =0.3. The query sequence is at least 15 amino acids in<br>
length, and the GAP analysis aligns the two sequences over a region of at least<br>
15 amino acids. More preferably, the query sequence is at least 50 amino<br>
acids in length, and the GAP analysis aligns the two sequences over a region<br>
of at least 50 amino acids. More preferably, the query sequence is at least 100<br>
amino acids in length and the GAP analysis aligns the two sequences over a<br>
region of at least 100 amino acids. More preferably, the query sequence is at<br>
least 250 amino acids in length and the GAP analysis aligns the two<br>
sequences over a region of at least 250 amino acids. Even more preferably, the<br><br>
query sequence is at least 500 amino acids in length and the GAP analysis<br>
aligns the two sequences over a region of at least 500 amino acids.<br>
As used herein, the term "mutant thereof refers to mutants of a<br>
naturally occurring insect esterase which maintains at least some hydrolytic<br>
activity towards a hydrophobic ester pesticide or toxin when compared to the<br>
naturally occurring insect esterase from which they are derived. Preferably,<br>
the mutant has enhanced activity and/or altered stereospecificity when<br>
compared to the naturally occurring insect esterase from which they are<br>
derived.<br>
Amino acid sequence mutants of naturally occurring insect esterases<br>
can be prepared by introducing appropriate nucleotide changes into a nucleic<br>
acid of the present invention, or by in vitro synthesis of the desired<br>
polypeptide. Such mutants include, for example, deletions, insertions or<br>
substitutions of residues within the amino acid sequence. A combination of<br>
deletion, insertion and substitution can be made to arrive at the final<br>
construct, provided that the final protein product possesses the desired<br>
characteristics.<br>
In designing amino acid sequence mutants, the location of the mutation<br>
site and the nature of the mutation will depend on characteristic(s) to be<br>
modified. In a particularly preferred embodiment, naturally occurring insect<br>
esterases are mutated to increase their ability to hydrolyse a hydrophobic<br>
ester pesticide or toxin, particularly a pyrethroid. The sites for mutation can<br>
be modified individually or in series, e.g., by (1) substituting first with<br>
conservative amino acid choices and then with more radical selections<br>
depending upon the results achieved, (2) deleting the target residue, or (3)<br>
inserting other residues adjacent to the located site. Examples of such<br><br>
which can hydrolyse a hydrophobic ester pesticide or toxin. The<br>
stereospecificity of the selected enzymes can also be screened.<br>
Amino acid sequence deletions generally range from about 1 to 30<br>
residues, more preferably about 1 to 10 residues and typically about 1 to 5<br>
contiguous residues.<br>
Substitution mutants have at least one amino acid residue in the<br>
polypeptide molecule removed and a different residue inserted in its place.<br>
The sites of greatest interest for substitutional mutagenesis include sites<br>
identified as the active or binding site(s). Other sites of interest are those in<br>
which particular residues obtained from various strains or species are<br>
identical. These positions may be important for biological activity. These<br>
sites, especially those falling within a sequence of at least three other<br>
identically conserved sites, can be substituted in a relatively conservative<br>
manner. Such conservative substitutions are shown in Table 1 under the<br>
heading of "exemplary substitutions".<br>
Furthermore, if desired, unnatural amino acids or chemical amino acid<br>
analogues can be introduced as a substitution or addition into the insect<br>
esterase, or mutants thereof. Such amino acids include, but are not limited<br>
to, the D-isomers of the common amino acids, 2,4-diaminobutyric acid, ?-<br>
amino isobutyric acid, 4-aminobutyric acid, 2-aminobutyric acid, 6-amino<br>
hexanoic acid, 2-amino isobutyric acid, 3-amino propionic acid, ornithine,<br>
norleucine, norvaline, hydroxyproline, sarcosine, citrulline, homocitrulline,<br>
cysteic acid, t-butylglycine, t-butylalanine, phenylglycine, cyclohexylalanine,<br>
?-alanine, fluoro-amino acids, designer amino acids such as ?-methyl amino<br>
acids, C?-methyl amino acids, Not-methyl amino acids, and amino acid<br>
analogues in general.<br>
Also included within the scope of the invention are insect esterases, or<br>
mutants thereof, which are differentially modified during or after synthesis,<br>
e.g., by biotinylation, benzylation, glycosylation, acetylation,<br>
phosphorylation, derivatization by known protecting/blocking groups,<br>
proteolytic cleavage, linkage to an antibody molecule or other cellular ligand,<br>
etc. These modifications may serve to increase the stability and/or bioactivity<br>
of the polypeptide of the invention.<br><br><br>
Insect esterases, and mutants thereof, can be produced in a variety of<br>
ways, including production and recovery of natural proteins, production and<br>
recovery of recombinant proteins, and chemical synthesis of the proteins. In<br>
one embodiment, an isolated polypeptide encoding the insect esterase, or<br>
mutant thereof, is produced by culturing a cell capable of expressing the<br>
polypeptide under conditions effective to produce the polypeptide, and<br>
recovering the polypeptide. A preferred cell to culture is a recombinant cell<br>
of the present invention. Effective culture conditions include, but are not<br><br>
limited to, effective media, bioreactor, temperature, pH and oxygen<br>
conditions that permit protein production. An effective medium refers to any<br>
medium in which a cell is cultured to produce a polypeptide of the present<br>
invention. Such medium typically comprises an aqueous medium having<br>
assimilable carbon, nitrogen and phosphate sources, and appropriate salts,<br>
minerals, metals and other nutrients, such as vitamins. Cells producing the<br>
insect esterase, or mutant thereof, can be cultured in conventional<br>
fermentation bioreactors, shake flasks, test tubes, microtiter dishes, and petri<br>
plates. Culturing can be carried out at a temperature, pH and oxygen content<br>
appropriate for a recombinant cell. Such culturing conditions are within the<br>
expertise of one of ordinary skill in the art.<br>
Polynucleotides<br>
By "isolated polynucleotide", we mean a polynucleotide separated from<br>
the polynucleotide sequences with which it is associated or linked in its<br>
native state. Furthermore, the term "polynucleotide" is used interchangeably<br>
herein with the term "nucleic acid molecule".<br>
The % identity of a polynucleotide is determined by GAP (Needleman<br>
and Wunsch, 1970) analysis (GGG program) with a gap creation penalty=5,<br>
and a gap extension penalty=0.3. The query sequence is at least 45<br>
nucleotides in length, and the GAP analysis aligns the two sequences over a<br>
region of at least 45 nucleotides. Preferably, the query sequence is at least<br>
150 nucleotides in length, and the GAP analysis aligns the two sequences<br>
over a region of at least 150 nucleotides. More preferably, the query sequence<br>
is at least 300 nucleotides in length and the GAP analysis aligns the two<br>
sequences over a region of at least 300 nucleotides.<br>
Recombinant Vectors<br>
Recombinant vectors can be used to express an insect esterase, or<br>
mutant thereof, for use in the methods of the present invention. In addition,<br>
in another embodiment of the present invention includes a recombinant<br>
vector, which includes at least one isolated polynucleotide molecule of the<br>
present invention, inserted into any vector capable of delivering the<br>
polynucleotide molecule into a host cell. Such vectors contain heterologous<br>
polynucleotide sequences, that is polynucleotide sequences that are not<br>
naturally found adjacent to polynucleotide encoding the insect esterase, or<br><br>
mutant thereof, and that preferably are derived from a species other than the<br>
species from which the esterase is derived. The vector can be either RNA or<br>
DNA, either prokaryotic or eukaryotic, and typically is a virus or a plasmid.<br>
One type of recombinant vector comprises a polynucleotide encoding<br>
an insect esterase, or mutant thereof, operatively linked to an expression<br>
vector. The phrase operatively linked refers to insertion of a polynucleotide<br>
molecule into an expression vector in a manner such that the molecule is able<br>
to be expressed when transformed into a host cell. As used herein, an<br>
expression vector is a DNA or RNA vector that is capable of transforming a<br>
host cell and of effecting expression of a specified polynucleotide molecule.<br>
Preferably, the expression vector is also capable of replicating within the host<br>
cell. Expression vectors can be either prokaryotic or eukaryotic, and are<br>
typically viruses or plasmids. Expression vectors of the present invention<br>
include any vectors that function (i.e., direct gene expression) in recombinant<br>
cells of the present invention, including in bacterial, fungal, endoparasite,<br>
arthropod, other animal, and plant cells. Preferred expression vectors of the<br>
present invention can direct gene expression in bacterial, yeast, arthropod<br>
and mammalian cells and more preferably in the cell types disclosed herein.<br>
Expression vectors of the present invention contain regulatory<br>
sequences such as transcription control sequences, translation control<br>
sequences, origins of replication, and other regulatory sequences that are<br>
compatible with the recombinant cell and that control the expression of<br>
polynucleotide molecules of the present invention. In particular, expression<br>
vectors which comprise a polynucleotide encoding an insect esterase, or<br>
mutant thereof, include transcription control sequences. Transcription<br>
control sequences are sequences which control the initiation, elongation, and<br>
termination of transcription. Particularly important transcription control<br>
sequences are those which control transcription initiation, such as promoter,<br>
enhancer, operator and repressor sequences. Suitable transcription control<br>
sequences include any transcription control sequence that can function in at<br>
least one of the recombinant cells of the present invention. A variety of such<br>
transcription control sequences are known to those skilled in the art.<br>
Preferred transcription control sequences include those which function in<br>
bacterial, yeast, arthropod and mammalian cells, such as, but not limited to,<br>
tac, lac, trp, trc, oxy-pro, omp/lpp, rrnB, bacteriophage lambda, bacteriophage<br>
T7, T7lac, bacteriophage T3, bacteriophage SP6, bacteriophage SP01,<br><br>
metallothionein, alpha-mating factor, Pichia alcohol oxidase, alphavirus<br>
subgenomic promoters (such as Sindbis virus subgenomic promoters),<br>
antibiotic resistance gene, baculovirus, Heliothis zea insect virus, vaccinia<br>
virus, herpesvirus, raccoon poxvirus, other poxvirus, adenovirus,<br>
cytomegalovirus (such as intermediate early promoters), simian virus 40,<br>
retrovirus, actin, retroviral long terminal repeat, Rous sarcoma virus, heat<br>
shock, phosphate and nitrate transcription control sequences as well as other<br>
sequences capable of controlling gene expression in prokaryotic or eukaryotic<br>
cells. Additional suitable transcription control sequences include tissue-<br>
specific promoters and enhancers.<br>
Polynucleotide encoding an insect esterase, or mutant thereof, may also<br>
(a) contain secretory signals (i.e., signal segment nucleic acid sequences) to<br>
enable an expressed insect esterase, or mutant thereof, to be secreted from the<br>
cell that produces the polypeptide and/or (b) contain fusion sequences.<br>
Examples of suitable signal segments include any signal segment capable of<br>
directing the secretion of an insect esterase, or mutant thereof. Preferred<br>
signal segments include, but are not limited to, tissue plasminogen activator<br>
(t-PA), interferon, interleukin, growth hormone, histocompatibility and viral<br>
envelope glycoprotein signal segments, as well as natural signal sequences.<br>
In addition, polynucleotides encoding an insect esterase, or mutant thereof,<br>
can be joined to a fusion segment that directs the encoded protein to the<br>
proteosome, such as a ubiquitin fusion segment.<br>
Host Cells<br>
Another embodiment of the present invention includes a recombinant<br>
cell comprising a host cell transformed with one or more polynucleotides<br>
encoding an insect esterase, or mutant thereof. Transformation of a<br>
polynucleotide molecule into a cell can be accomplished by any method by<br>
which a polynucleotide molecule can be inserted into the cell.<br>
Transformation techniques include, but are not limited to, transfection,<br>
electroporation, microinjection, lipofection, adsorption, and protoplast<br>
fusion. A recombinant cell may remain unicellular or may grow into a tissue,<br>
organ or a multicellular organism. A transformed polynucleotide encoding an<br>
insect esterase, or mutant thereof, can remain extrachromosomal or can<br>
integrate into one or more sites within a chromosome of the transformed (i.e.,<br><br>
recombinant) cell in such a manner that their ability to be expressed is<br>
retained.<br>
Suitable host cells to transform include any cell that can be<br>
transformed with a polynucleotide encoding an insect esterase, or mutant<br>
thereof. Host cells of the present invention either can be endogenously (i.e.,<br>
naturally) capable of producing an insect esterase or mutant thereof, or can be<br>
capable of producing such proteins after being transformed with at least one<br>
polynucleotide encoding an insect esterase, or mutant thereof. Host cells of<br>
the present invention can be any cell capable of producing at least one insect<br>
esterase, or mutant thereof, and include bacterial, fungal (including yeast),<br>
parasite, arthropod, animal and plant cells. Preferred host cells include<br>
bacterial, mycobacterial, yeast arthropod and mammalian cells. More<br>
preferred host cells include Salmonella, Escherichia, Bacillus, Listeria,<br>
Saccharomyces, Spodoptera, Mycobacteria, Trichoplusia, BHK (baby hamster<br>
kidney) cells, MDCK cells (normal dog kidney cell line for canine herpes virus<br>
cultivation), CRFK cells (normal cat kidney cell line for feline herpesvirus<br>
cultivation), CV-1 cells (African monkey kidney cell line used, for example, to<br>
culture raccoon poxvirus), COS (e.g., COS-7) cells, and Vero cells.<br>
Particularly preferred host cells are E. coli, including E. coli K-12 derivatives;<br>
Salmonella typhi; Salmonella typhimurium, including attenuated strains;<br>
Spodoptera frugiperda; Trichoplusia ni; BHK cells; MDCK cells; CRFK cells;<br>
CV-1 cells; COS cells; Vero cells; and non-tumorigenic mouse myoblast G8<br>
cells (e.g., ATCC CRL1246). Additional appropriate mammalian cell hosts<br>
include other kidney cell lines, other fibroblast cell lines (e.g., human,<br>
murine or chicken embryo fibroblast cell lines), myeloma cell lines, Chinese<br>
hamster ovary cells, mouse NIH/3T3 cells, LMTK cells and/or HeLa cells.<br>
Recombinant DNA technologies can be used to improve expression of a<br>
transformed polynucleotide molecule by manipulating, for example, the<br>
number of copies of the polynucleotide molecule within a host cell, the<br>
efficiency with which those polynucleotide molecules are transcribed, the<br>
efficiency with which the resultant transcripts are translated, and the<br>
efficiency of post-translational modifications. Recombinant techniques<br>
useful for increasing the expression of a polynucleotide encoding an insect<br>
esterase, or mutant thereof, include, but are not limited to, operatively linking<br>
polynucleotide molecules to high-copy number plasmids, integration of the<br>
polynucleotide molecule into one or more host cell chromosomes, addition of<br><br>
vector stability sequences to plasmids, substitutions or modifications of<br>
transcription control signals (e.g., promoters, operators, enhancers),<br>
substitutions or modifications of translational control signals (e.g., ribosome<br>
binding sites, Shine-Dalgamo sequences), modification of polynucleotide<br>
molecules of the present invention to correspond to the codon usage of the<br>
host cell, and the deletion of sequences that destabilize transcripts.<br>
Compositions<br>
Compositions useful for the methods of the present invention, or which<br>
comprise a polypeptide of the present invention, include excipients, also<br>
referred to herein as "acceptable carriers". An excipient can be any material<br>
that the animal, plant, plant or animal material, or environment (including<br>
soil and water samples) to be treated can tolerate. Examples of such<br>
excipients include water, saline, Ringer's solution, dextrose solution, Hank's<br>
solution, and other aqueous physiologically balanced salt solutions.<br>
Nonaqueous vehicles, such as fixed oils, sesame oil, ethyl oleate, or<br>
triglycerides may also be used. Other useful formulations include<br>
suspensions containing viscosity enhancing agents, such as sodium<br>
carboxymethylcellulose, sorbitol, or dextran. Excipients can also contain<br>
minor amounts of additives, such as substances that enhance isotonicity and<br>
chemical stability. Examples of buffers include phosphate buffer, bicarbonate<br>
buffer and Tris buffer, while examples of preservatives include thimerosal or<br>
o-cresol, formalin and benzyl alcohol. Excipients can also be used to increase<br>
the half-life of a composition, for example, but are not limited to, polymeric<br>
controlled release vehicles, biodegradable implants, liposomes, bacteria,<br>
viruses, other cells, oils, esters, and glycols.<br>
Furthermore, the insect esterase, or mutant thereof, can be provided in<br>
a composition which enhances the rate and/or degree of degradation of<br>
hydrophobic ester pesticides or toxins, or increases the stability of the<br>
polypeptide. For example, the insect esterase, or mutant thereof, can be<br>
immobilized on a polyurethane matrix (Cordon et al., 1999), or encapsulated<br>
in appropriate liposomes (Petrikovics et al. 2000a and b). The insect esterase,<br>
or mutant thereof, can also be incorporated into a composition comprising a<br>
foam such as those used routinely in fire-fighting (LeJeune et al., 1998).<br><br>
As would be appreciated by the skilled addressee, the insect esterase,<br>
or mutant thereof, could readily be used in a sponge or foam as disclosed in<br>
WO 00/64539, the contents of which are incorporated herein in their entirety.<br>
One embodiment of the present invention is a controlled release<br>
formulation that is capable of slowly releasing a composition comprising an<br>
insect esterase, or mutant thereof, into an animal, plant, animal or plant<br>
material, or the environment (including soil and water samples). As used<br>
herein, a controlled release formulation comprises an insect esterase, or<br>
mutant thereof, in a controlled release vehicle. Suitable controlled release<br>
vehicles include, but are not limited to, biocompatible polymers, other<br>
polymeric matrices, capsules, microcapsules, microparticles, bolus<br>
preparations, osmotic pumps, diffusion devices, liposomes, lipospheres, and<br>
transdermal delivery systems. Preferred controlled release formulations are<br>
biodegradable (i.e., bioerodible).<br>
A preferred controlled release formulation of the present invention is<br>
capable of releasing an insect esterase, or mutant thereof, into soil or water<br>
which is in an area sprayed with a hydrophobic ester pesticide or toxin. The<br>
formulation is preferably released over a period of time ranging from about 1<br>
to about 12 months. A preferred controlled release formulation of the present<br>
invention is capable of effecting a treatment preferably for at least about 1<br>
month, more preferably for at least about 3 months, even more preferably for<br>
at least about 6 months, even more preferably for at least about 9 months, and<br>
even more preferably for at least about 12 months.<br>
The concentration of the insect esterase, or mutant thereof, (or host cell<br>
expressing the insect esterase, or mutant thereof) that will be required to<br>
produce effective compositions for degrading a hydrophobic ester pesticide or<br>
toxin will depend on the nature of the sample to be decontaminated, the<br>
concentration of the hydrophobic ester pesticide or toxin in the sample, and<br>
the formulation of the composition. The effective concentration of the insect<br>
esterase, or mutant thereof, (or host cell expressing the insect esterase, or<br>
mutant thereof) within a composition can readily be determined<br>
experimentally, as will be understood by the skilled artisan.<br>
Surfactants<br>
It is envisaged that the use of a surfactant in the method of the present<br>
invention may liberate hydrophobic ester pesticides and/or toxins, from any,<br><br>
for example, sediment in the sample. Thus increasing efficiency of the<br>
method of the present invention.<br>
Surfactants are amphipathic molecules with both hydrophilic and<br>
hydrophobic (generally hydrocarbon) moieties that partition preferentially at<br>
the interface between fluid phases and different degrees of polarity and<br>
hydrogen bonding such as oil/water or air/water interfaces. These properties<br>
render surfactants capable of reducing surface and interfacial tension and<br>
forming microemulsion where hydrocarbons can solubilize in water or where<br>
water can solubilize in hydrocarbons. Surfactants have a number of useful<br>
properties, including dispersing traits.<br>
Biosurfactants are a structurally diverse group of surface-active<br>
molecules synthesized by microorganisms. These molecules reduce surface<br>
and interfacial tensions in both aqueous solutions and hydrocarbon mixtures.<br>
Biosurfactants have several advantages over chemical surfactants, such as<br>
lower toxicity, higher biodegradability, better environmental compatability,<br>
higher foaming, high selectivity and specificity at extreme temperatures, pH<br>
and salinity, and the ability to be synthesized from a renewable source.<br>
Biosurfactants useful in the bioremediation methods of the present<br>
invention include, but are not limited to; glycolipids such as rhamnolipids<br>
(from, for example, Pseudomonas aeruginosa), trehalolipids (from, for<br>
example, Ehodococcus erythiopolis), sophorolipids (from, for example,<br>
Torulopsis bombicola), and cellobiolipids (from, for example, Ustilago zeae);<br>
lipopeptides and lipoproteins such as serrawettin (from, for example, Serratia<br>
marcescens), surfactin (from, for example, Bacillus subtilis); subtilisin (from,<br>
for example, Bacillus subtilis), gramicidins (from, for example, Bacillus<br>
brevis), and polymyxins (from, for example, Bacillus polymyxa); fatty acids,<br>
neutral lipids, and phospholipids; polymeric surfactants such as emulsan<br>
(from, for example, Acinetobacter calcoaceticus), biodispersan (from, for<br>
example, Acinetobacter calcoaceticus}, mannan-lipid-protein (from, for<br>
example, Candida tropicalis), liposan (from, for example, Candida lypolytica),<br>
protein PA (from, for example, Pseudomonas. aeruginosa); and particulate<br>
biosurfactants such as vesicles and fimbriae from, for example, A<br>
calcoaceticus.<br><br>
Transgenic Plants<br>
The term "plant" refers to whole plants, plant organs (e.g. leaves, stems<br>
roots, etc), seeds, plant cells and the like. Plants contemplated for use in the<br>
practice of the present invention include both monocotyledons and<br>
dicotyledons. Exemplary monocotyledons include wheat, barley, rye,<br>
triticale, oats, rice, and the like.<br>
Transgenic plants, as defined in the context of the present invention<br>
include plants (as well as parts and cells of said plants) and their progeny<br>
which have been genetically modified using recombinant DNA techniques to<br>
either i) cause the production of an the insect esterase, or mutant thereof, in<br>
the desired plant or plant organ.<br>
Several techniques exist for introducing foreign genetic material into a<br>
plant cell. Such techniques include acceleration of genetic material coated<br>
onto microparticles directly into cells (see, for example, US 4,945,050 and US<br>
5,141,131). Plants may be transformed using Agrobacterium technology (see,<br>
for example, US 5,177,010, US 5,104,310, US 5,004,863, US 5,159,135).<br>
Electroporation technology has also been used to transform plants (see, for<br>
example, WO 87/06614, US 5,472,869, 5,384,253, WO 92/09696 and WO<br>
93/21335). In addition to numerous technologies for transforming plants, the<br>
type of tissue which is contacted with the foreign genes may vary as well.<br>
Such tissue would include but would not be limited to embryogenic tissue,<br>
callus tissue type I and n, hypocotyl, meristem, and the like. Almost all plant<br>
tissues may be transformed during development and/or differentiation using<br>
appropriate techniques described herein.<br>
A number of vectors suitable for stable transfection of plant cells or for<br>
the establishment of transgenic plants have been described in, e.g., Pouwels<br>
et al., Cloning Vectors: A Laboratory Manual, 1985, supp. 1987; Weissbach<br>
and Weissbach, Methods for Plant Molecular Biology, Academic Press, 1989;<br>
and Gelvin et al., Plant Molecular Biology Manual, Kluwer Academic<br>
Publishers, 1990. Typically, plant expression vectors include, for example,<br>
one or more cloned plant genes under the transcriptional control of 5' and 3'<br>
regulatory sequences and a dominant selectable marker. Such plant<br>
expression vectors also can contain a promoter regulatory region (e.g., a<br>
regulatory region controlling inducible or constitutive, environmentally- or<br>
developmentally-regulated, or cell- or tissue-specific expression), a<br>
transcription initiation start site, a ribosome binding site, an<br><br>
RNA processing signal, a transcription termination site, and/or a<br>
polyadenylation signal.<br>
Examples of plant promoters include, but are not limited to ribulose-<br>
1,6-bisphosphate carboxylase small subunit, beta-conglycinin<br>
promoter, phaseolin promoter, ADH promoter, heat-shock promoters and<br>
tissue specific promoters. Promoters may also contain certain enhancer<br>
sequence elements that may improve the transcription efficiency. Typical<br>
enhancers include but are not limited to Adh-intron 1 and Adh-intron 6.<br>
Constitutive promoters direct continuous gene expression in all cells<br>
types and at all times (e.g., actin, ubiquitin, CaMV 35S). Tissue specific<br>
promoters are responsible for gene expression in specific cell or tissue types,<br>
such as the leaves or seeds (e.g., zein, oleosin, napin, ACP, globulin and the<br>
like) and these promoters may also be used. Promoters may also be active<br>
during a certain stage of the plants' development as well as active in plant<br>
tissues and organs. Examples of such promoters include but are not limited<br>
to pollen-specific, embryo specific, corn silk specific, cotton fiber specific,<br>
root specific, seed endosperm specific promoters and the like.<br>
Under certain circumstances it may be desirable to use an inducible<br>
promoter. An inducible promoter is responsible for expression of genes in<br>
response to a specific signal, such as: physical stimulus (heat shock genes);<br>
light (RUBP carboxylase); hormone (Em); metabolites; and stress. Other<br>
desirable transcription and translation elements that function in plants may<br>
be used.<br>
In addition to plant promoters, promoters from a variety of sources can<br>
be used efficiently in plant cells to express foreign genes. For example,<br>
promoters of bacterial origin, such as the octopine synthase promoter, the<br>
nopaline synthase promoter, the mannopine synthase promoter; promoters of<br>
viral origin, such as the cauliflower mosaic virus (35S and 19S) and the like<br>
may be used.<br>
The following examples are offered for illustration purposes, and are<br>
not intended to limit or define the invention in any manner.<br><br>
EXAMPLES<br>
Example 1; Construction of Mutants<br>
An alignment of the amino acid sequence of the E3 enzyme with that of<br>
a vertebrate acetylcholinesterase (TcAChE, for which the three dimensional<br>
structure is known; Sussman et al., 1991) is given in Figure 1. Mutants of E3<br>
and EST23 were constructed using the QuickChange Site-Directed<br>
Mutagenesis Kit of Stratagene and are named according to the number of the<br>
residue that has been changed, and the nature of that change. For example,<br>
mutant E3W251L is an E3 mutant in which the Trp residue at position 251 in<br>
the wild-type enzyme (i.e. E3WT) has been mutated to Leu.<br>
E3 and EST23 enzymes were expressed using the baculovirus<br>
expression system as described by Newcomb et al. (1997), but using the HyQ<br>
SFX-insect serum-free medium (HyClone) for increased expression. Cell<br>
extracts were prepared by lysing the cells at a concentration of 108 cells ml'1<br>
in 0.1M phosphate buffer pH 7.0 containing 0.05% Triton X-100. Extracts<br>
were then titrated for the number of esterase molecules using a fluorometric<br>
assay based on the initial release of coumarin (a fluorescent compound) upon<br>
phosphorylation of the enzyme by diethylcoumaryl phosphate (dECP).<br>
Figure 2 illustrates the proposed configuration of the active site of E3<br>
(based on the three dimensional structure of vertebrate AChE) in an acylation<br>
reaction. We have examined mutations in seven E3 residues in regions<br>
corresponding to three distinct subsites of the known AChE active site.<br>
These are the oxyanion hole (E3 residue 137), the anionic site (E3 residues<br>
148, 217 and 354) and acyl binding pocket (E3 residues 250, 251 and 309).<br>
The anionic site and acyl binding pocket correspond to the p1 and p2<br>
subsites in the nomenclature of Jarv (1984).<br>
Mutations in the Oxyanion Hole<br>
In TcAChE the oxyanion hole comprises Glyll8, Glyll9 and Ala201,<br>
which corresponds to Glyl36, Glyl37 and Ala219 in E3. These residues are<br>
highly conserved throughout the carboxyl/cholinesterase multigene family<br>
(Oakeshott et al., 1999) and there is empirical evidence for the conservation<br>
of the oxyanion hole structure from X-ray crystallographic studies of several<br>
cholinesterases and Upases (Cygler and Schrag, 1997), albeit the structure<br>
does change during interfacial activation in some lipases (Derewenda et al.,<br>
1992). There is also empirical structural evidence for their function in<br><br>
stabilising the oxyanion formed by the carbonyl oxygen of the carboxylester<br>
substrate as the first transition state during catalysis (Grochulski et al., 1993;<br>
Martinez et al., 1994). This stabilisation is achieved by a network of<br>
hydrogen bonds to the amide groups of the three key residues in the peptide<br>
chain (Ordentlich et al., 1998). Recently Koellner et al. (2000) have also<br>
shown that both Gly residues in the AChE oxyanion hole make hydrogen<br>
bonds with buried "structural" water molecules, which are retained during<br>
catalysis and thought to act as lubricants to facilitate traffic of substrates and<br>
products within the active site.<br>
Three further mutations were made to the Glyl37 of E3 in addition to<br>
the G137D found naturally in OP resistant L. cuprina. First, Glu was<br>
substituted as the other acidic amino acid, in G137E. The mutant G137H was<br>
also constructed, because His is also non-protonated at neutral pH (pKa about<br>
6.5 cf 4.4 for Asp and Glu) and it was found to confer some OP hydrolysis on<br>
human butyrylcholinesterase when substituted for either Gly in its oxyanion<br>
hole (Broomfield et al., 1999). Finally, Arg (pKa around 12) was substituted at<br>
position 137, to examine the effects of the most strongly basic substitution<br>
possible.<br>
Mutations in the Acyl Binding Pocket<br>
The acyl binding pockets of structurally characterised cholinesterases<br>
are formed principally from four non-polar residues, three of which are<br>
generally also aromatic. Together they create a strongly hydrophobic pocket<br>
to accommodate the acyl moiety of bound substrate. The four residues in<br>
TcAChE are Trp233, Phe288, Phe290 and Val400 corresponding to Trp251,<br>
Val307, Phe309 and Phe422 in E3. Similar arrays of hydrophobic residues<br>
appear to be conserved at the corresponding sites of most<br>
carboxyl/cholinesterases (Oakeshott et al., 1993; Robin et al., 1996; Yao et al.,<br>
1997; Harel et al., 2000). In particular Trp is strongly conserved at residue<br>
233/251 and 290/309 is Phe in cholinesterases and most carboxylesterases,<br>
albeit a Leu or Ile in several lipases and a few carboxylesterases. The residue<br>
corresponding to TcAChE Phe288 is typically a branched chain aliphatic<br>
amino acid in cholinesterases that show a preference for longer chain esters<br>
such as butyrylcholine. This includes mammalian butyrylcholinesterase and<br>
some insect acetylcholinesterases, which have a butyrylcholinesterase-like<br>
substrate specificity. The branched chain aliphatic amino acid appears to<br><br>
provide a greater space in the acyl-binding pocket to accommodate the larger<br>
acyl group.<br>
Mutational studies of 288/307 and 290/309 in several cholinesterases<br>
confirm their key role in determining aspects of substrate specificities related<br>
to acyl group identity. In human AChE replacement of the Phe at either<br>
position with a smaller residue like Ala improves the kinetics of the enzyme<br>
for substrates like propyl- or butyl- (thio)choline with larger acyl groups than<br>
the natural acetyl (thio)choline substrate (Ordentlich et al., 1993). In AChE<br>
from D. melanogaster and the housefly, Musca domestica, natural mutations<br>
of their 290/309 equivalent to the bulkier, polar Tyr that contributes to target<br>
site OP resistance have lower reactivity to both acetylcholine and OPs<br>
(Fournier et al., 1992; Walsh et al., 2001). For D. melanogaster AChE,<br>
substitution of this Phe residue with the smaller Leu gave the predicted<br>
increase in OP sensitivity, although surprisingly replacement with other<br>
small residues like Gly, Ser or Val did not (Villatte et al., 2000).<br>
Trp 233/251 has received much less attention in mutational studies of<br>
cholinesterases but our prior work on E3 shows its replacement with a<br>
smaller Leu residue again increases reactivity for carboxylester substrates<br>
with bulky acyl moieties as in malathion, or for OPs (Campbell et al., 1998a,<br>
b; Devonshire et al., 2002). A mutation to Gly has also been found in a<br>
homologue from the wasp, Anisopteromalus calandrae, that shows enhanced<br>
malathion carboxylesterase (MCE) kinetics (Zhu et al., 1999) while a Ser has<br>
been found in a homologue from M. domestica that may be associated with<br>
malathion resistance (Claudianos et al., 2002). In respect of OP hydrolase<br>
activity Devonshire et al. (2002) proposed that the particular benefit of such<br>
mutations is to accommodate the inversion about the phosphorus that must<br>
occur for the second hydrolysis stage of the reaction to proceed. Notably<br>
Devonshire et al. (2002) found that the kcat for OP hydrolase activity of<br>
E3W251L is an order of magnitude higher for dMUP, with its smaller<br>
dimethyl phosphate group than for dECP, which has a diethyl phosphate<br>
group. This suggests that there remain tight steric constraints on the<br>
inversion even in a mutant with a larger acyl pocket.<br>
We have mutated both the W251 and F309 residues of E3 as well as the<br>
P250 immediately adjacent to W251. In addition to the previously<br>
characterised natural W251L mutation we have now analysed substitutions<br>
with four other small amino acids in W251S, W251G, W251T and W251A. A<br><br>
double mutant of W251L and P250S was also analysed, because a natural<br>
variant of the ortholog of E3 in M. domestica with high MCE activity has Ser<br>
and Leu at positions 250 and 251, respectively. Only one F309 substitution<br>
was examined, F309L, which the AChE results suggest should enhance MCE<br>
and OP hydrolyse activities. F309L was analysed alone and as a double<br>
mutant with W251L.<br>
Mutations in the Anionic Site<br>
The anionic site of cholinesterases is sometimes called the quaternary<br>
binding site (for the quaternary ammonium in acetylcholine), or the p1<br>
subsite in the original nomenclature of Jarv (1984). It principally involves<br>
Trp 84, Glu 199 and Phe 330, with Phe 331 and Tyr 130 (TcAChE<br>
nomenclature) also involved. Except for Glu 199 it is thus a highly<br>
hydrophobic site. Glu 199 is immediately adjacent to the catalytic Ser 200.<br>
The key residues are highly conserved across cholinesterases and to a lesser<br>
extent, many carboxylesterases (Oakeshott et al., 1993; Ordentlich et al.,<br>
1995; Robin et al., 1996; Claudianos et al., 2002). Except for Trp 84 (the<br>
sequence alignment in Figure 1 shows that E3 is missing residues<br>
corresponding to AChE residues 74-85), E3 has identical residues to TcAChE<br>
at the corresponding positions (217, 354 and 148, respectively). Interestingly<br>
the equivalent of Glu 199 is Gln and the equivalent of the Phe 330 is Leu in<br>
some lipases and certain carboxylesterases, whose substrates are known to<br>
have small leaving groups (Thomas et al., 1999; Campbell et al., 2001;<br>
Claudianos et al., 2002).<br>
Structural and mutational studies have provided a detailed picture of<br>
the role of the anionic site in cholinesterase catalysis. The key residues form<br>
part of a hydrogen bonded network at the bottom of the active site, with Tyr<br>
130 and Glu 199 also sharing contact with a structural water molecule<br>
(Ordentlich et al., 1995; Koellner et al., 2000). The anionic site undergoes a<br>
confonnational change when substrate binds a peripheral binding site at the<br>
lip of the active site gorge, the new conformation accommodating the choline<br>
(leaving) group of the substrate and facilitating the interaction of its carbonyl<br>
carbon with the catalytic Ser 200 (Shafferman et al., 1992; Ordentlich et al.,<br>
1995; 1996). Consequently the site functions mainly in the first, enzyme<br>
acylation, stage of the reaction and, in particular, in the formation of the non-<br>
covalent transition state (Nair et al., 1994). Therefore mutations of the key<br><br>
residues mainly affect Km rather than kcat. The interactions with the choline<br>
leaving group are mainly mediated through non-polar and ?-electron<br>
interactions, principally involving Trp 84 and Phe 330 (Ordentlich et al.,<br>
1995).<br>
Studies with OP inhibitors suggest that the anionic site of<br>
cholinesterases also accommodates their leaving group but there is some<br>
evidence that part of the site (mainly Glu 199 and Tyr 130; also possibly Ser<br>
226) may also then affect the reactivity of the phosphorylated enzyme (Qian<br>
and Kovach, 1993; and see also Ordentlich et al., 1996; Thomas et al., 1999).<br>
There has been little mutational analysis of carboxylesterase sites<br>
corresponding to the AChE anionic site but one interesting exception<br>
involves the EST6 carboxylesterase of D. melanogaster, which has a His at the<br>
equivalent of Glu 199. A mutant in which this His is replaced by Glu shows<br>
reduced activity against various carboxylester substrates but has acquired<br>
some acetylthiocholine hydrolytic activity (Myers et al., 1993). The E4<br>
carboxylesterase of the aphid, Myzus persicae, has a Met at this position and<br>
this enzyme is unusually reactive to OPs (Devonshire and Moores, 1982).<br>
However, it is not known whether the Met contributes to the OP hydrolase<br>
activity. Similarly, a Y148F substitution is one of several recorded in the E3<br>
ortholog in an OP resistant strain (ie also G137D) of M. domestica but it is not<br>
known whether this change directly contributes to OP hydrolase activity<br>
(Claudianos et al., 1999).<br>
The Y148, E217 and F354 residues in E3 have now been mutated.<br>
E217M and Y148F mutations were made to test whether the corresponding<br>
mutations in the M. persicae and M. domestica enzymes above contribute<br>
directly to their OP reactivity. Y148F is also tested in a G137D double mutant<br>
since this is the combination found in the resistant M. domestica. F354 was<br>
mutated both to a smaller Leu residue and a larger Trp, Leu commonly being<br>
found at this position in lipases (see above).<br>
Example 2: Enzyme Titrations<br>
Four 100?l reactions were set up for each expressed esterase in<br>
microplate columns 1-4:<br>
plate well blank containing 0.025% Triton X-100, 0.1M phosphate buffer pH<br>
7.0;<br><br>
substrate blank containing 100?M dECP in 0.025% Triton X-100, 0.1M<br>
phosphate buffer pH 7.0;<br>
cell blank containing 50?l cell extract mixed 1:1 with 0.1M phosphate buffer<br>
pH 7.0;<br>
titration reaction containing 50?l cell extract mixed 1:1 with 0.lM phosphate<br>
buffer pH 7.0 containing 200?M dECP.<br>
All components except dECP (freshly prepared at a concentration of<br>
200?M in buffer) were placed in the wells. Several enzymes were assayed<br>
simultaneously in a plate, and the reactions were started by adding dECP<br>
simultaneously to the 2nd and 4th wells down a column. The interval to the<br>
first reading (typically 1 minute) was noted for the subsequent calculations.<br>
The mean value for the plate well blank (A) was subtracted from all<br>
readings before further calculations. Preliminary experiments with various<br>
cell extracts showed that they gave some fluorescence at 460nm and that<br>
their addition to solutions of the assay product, 7-hydroxycoumarin,<br>
quenched fluorescence by 39(+,7)%. Fluorescence values in the titration<br>
reactions (D) were therefore corrected for this quenching effect after<br>
subtraction of the intrinsic fluorescence of the cell extracts (C). Finally, the<br>
substrate blank (B), taken as the mean from all the simultaneous assays in a<br>
plate, was subtracted to give the corrected fluorescence caused by the<br>
esterase-released coumarin. These corrections were most important for cell<br>
lines expressing esterase at very low level (
The fully corrected data were plotted as a progress curve, and the<br>
equilibrium slope extrapolated back to zero time to determine the amount of<br>
esterase, based on its stoichiometric interaction with the inhibitor (the 100<br>
?M concentration of dECP gave full saturation of the esterase catalytic sites of<br>
all these enzymes in 10-20 minutes). A calibration curve for 7-<br>
hydroxycoumarin was prepared alongside the reactions in all plates, and used<br>
to calculate molar concentration of enzyme and product formation.<br>
Figure 3 shows the results of representative titration experiments<br>
performed on cell extracts containing baculovirus expressed esterases.<br>
Example 3: Permethrin Hydrolysis Assays<br>
Expressed enzymes were tested for permethrin hydrolytic activity using<br>
a radiometric partition assay for acid-labelled compounds, or a TLC based<br><br>
assay for those labelled in the alcohol moiety (Devonshire and Moores, 1982).<br>
Features of the assays include keeping the concentration of permethrin below<br>
its published solubility in aqueous solution (0.5 ?M), the concentration of<br>
detergent (used to extract the enzyme from the insect cells in which it is<br>
expressed) below the critical micelle concentration (0.02% for Triton X100),<br>
and performing the assays quickly (ie within 10-30 minutes) to minimise the<br>
substrate sticking to the walls of the assay tubes (glass tubes were used to<br>
minimise stickiness). At these permethrin concentrations the enzyme is not<br>
saturated by the substrate, so Km values could not be determined. However,<br>
specificity constants [kcat/Km could be calculated accurately for each of the<br>
enzymes with permethrin activity, which allows direct comparison of their<br>
efficiency at low substrate concentrations. The power of the analyses was<br>
increased by separating permethrin into its cis and trans isomers.<br>
(a) Separation of cis and trans Isomers of Permethrin<br>
Commercial preparations of permethrin contain four stereoisomers: 1S<br>
cis, 1R cis, 1S trans, 1R trans (Figure 4). Preparative thin layer<br>
chromatography (TLC) on silica was used to separate the isomers into two<br>
enantiomer pairs: 1S/1R cis and 1S/1R trans. The enantiomers could not be<br>
separated further. Enzyme preparations could then be assayed for the<br>
hydrolysis of each enantiomer pair.<br>
(b) Assay Protocol<br>
Pyrethioids radiolabelled in the acid moiety<br>
This assay (Devonshire and Moores, 1982) is used for permethrin<br>
isomers. It relies on incubating the expressed esterase with radiolabelled<br>
substrate and then measuring the radioactive cyclopropanecarboxylate anion<br>
in the aqueous phase after extracting the unchanged substrate into organic<br>
solvent. Based on previous experience, the best extraction protocol utilises a<br>
2:1 (by volume) mixture of methanol and chloroform. When mixed in the<br>
appropriate proportion with aliquots of the assay incubation, the consequent<br>
mixture of buffer, methanol and chloroform is monophasic, which serves the<br>
purpose of stopping the enzyme reaction and ensuring the complete<br>
solubilization of the pyrethroid. Subsequent addition of an excess of<br>
chloroform and buffer exceeds the capacity of the methanol to hold the<br><br>
phases together, so that the organic phase can be removed and the product<br>
measured in the aqueous phase. In detail, the protocol is as follows.<br>
Phosphate buffer (0.1M, pH 7.0) was added to radiolabelled permethrin<br>
(50?M in acetone) to give a l?M solution and the assay then started by-<br>
adding an equal volume of expressed esterase appropriately diluted in the<br>
same buffer. Preliminary work had established that the concentration of<br>
detergent (Triton X-100 used to extract esterase from the harvested cells) in<br>
the incubation had to be below its CMC (critical micelle concentration of<br>
0.02%) to avoid the very lipophilic pyrethroid partitioning into the micelles<br>
and becoming unavailable to the enzyme. Typically, the final volume of the<br>
assay was 500-1000?l, with substrate and acetone concentrations 0.5?M and<br>
1%, respectively. At intervals ranging from 30 seconds to 10 minutes at a<br>
temperature of 30, 100?l aliquots of the incubation were removed, added to<br>
tubes containing 300?l of the 2:1 methanol chloroform mixture and vortex-<br>
mixed. The tubes were then held at room temperature until a batch could be<br>
further processed together, either at the end of the incubation or during an<br>
extended sampling interval. After adding 50?l buffer and 100?l chloroform,<br>
the mixture was vortex-mixed, centrifuged and the lower organic phase<br>
removed with a 500?l Hamilton syringe and discarded. The extraction was<br>
repeated after adding a further 100?l chloroform, and then 200?l of the upper<br>
aqueous phase was removed (using a pipettor with a fine tip) for scintillation<br>
counting. It is critical to avoid taking any of the organic phase. Since the<br>
final volume of the aqueous phase was 260?l (including some methanol), the<br>
total counts produced in the initial 100?l aliquot were corrected accordingly.<br>
Pyrethroids radiolabelled in the alcohol moiety<br>
i) Type I pyrethroids - dibromo analogues (NRDC157) of permethrin:<br>
The 3-phenoxbenzyl alcohol formed on hydrolysis of these esters does<br>
not partition into the aqueous phase in the chloroform methanol extraction<br>
procedure. It was therefore necessary to separate this product from the<br>
substrate by TLC on silica (Devonshire and Mooers, 1982). In detail, the<br>
protocol is as follows.<br>
Incubations were set up as for the acid-labelled substrates. The<br>
reactions were stopped at intervals in 100?l aliquots taken from the<br>
incubation by immediately mixing with 200?l acetone at -79 (solid CO2).<br>
Then 100?l of the mixture was transferred, together with 3?l non-radioactive<br><br>
3-phenoxbenzyl alcohol (2% in acetone), on to the loading zone of LinearQ<br>
channelled silica F254 plates (Whatman). After developing in a 10:3 mixture<br>
of toluene (saturated with formic acid) with diethyl ether, the substrate and<br>
product were located by radioautography for 6-7 days (confirming an<br>
identical mobility of the product to the cold standard 3-phenoxbenzyl alcohol<br>
revealed under UV light). These areas of the TLC plate were then<br>
impregnated with Neatan (Merck) and dried, after which they were peeled<br>
from the glass support and transferred to vials for scintillation counting. The<br>
counts were corrected for the 3-fold dilution of the initial 100?l by acetone<br>
before spotting on the silica.<br>
ii) Type II pyrethroids - deltamethrin isomers:<br>
Preliminary experiments, in which incubations were analysed by TLC<br>
as above, showed primarily the formation of 3-phenoxbenzoic acid, in line<br>
with literature reports that the initial cyanohydrin hydrolyis product is<br>
rapidly converted non-enzymically to the acid. Since the TLC assay is more<br>
protracted than the chloroform-methanol extraction procedure, the latter (as<br>
described above for acid-labelled pyrethroids) was adopted to measure the 3-<br>
phenoxbenzoate anion produced from these substrates.<br>
For all assays the molar amount of product formed was calculated from<br>
the known specific activity of the radiolabelled substrate. Early experiments<br>
on the expressed E3WT esterase showed that the rate of hydrolysis was<br>
directly proportional to the concentration of IRS cis or IRS trans permethrin<br>
in the assay up to 0.5?M, i.e. there was no accumulation of Michaelis<br>
complex. Assays at concentrations greater than 0.5 ?M, which approximates<br>
the published aqueous solubility of permethrin, gave erratic results so<br>
precluding the measurement of Km and kcat. Furthermore, with the racemic<br>
substrates, the rate of hydrolysis slowed dramatically once approximately<br>
50% of the substrate had been hydrolysed, indicating that only one of the two<br>
enantiomers (1R or 1S present in equal amounts in a racemic mixture) was<br>
readily hydrolysed, in line with previously published data for an esterase<br>
from aphids (Devonshire and Moores, 1982). Assay conditions were therefore<br>
adjusted to measure the hydrolysis of the more-readily hydrolysed<br>
enantiomer in each pair. Sequential incubation of trans permethrin with<br>
E3WT homogenates confirmed that both showed preference for the 1S trans<br><br>
enantiomer. In all cases, the rate of hydrolysis at 0.5?M (or 0.25?M for the<br>
one enantiomer in racemic substrates), together with the molar amount of<br>
esterase determined by titration with dECP, were used to calculate the<br>
specificity constant (Kcat/ Km since it was not possible to separate these kinetic<br>
parameters. The same considerations about substrate solubility and<br>
proportionality of response to its concentration were assumed for all enzymes<br>
and substrates.<br>
(c) Calculation of Specificity Constants<br>
Figure 5 presents the results of an experiment in which the trans- and<br>
cis- isomers of permethrin were hydrolysed by the E3W251L enzyme.<br>
Since the rate of hydrolysis of permethrin isomers was directly<br>
proportional to the concentration of substrate used up to 0.5?M (i.e. there was<br>
no significant formation of Michaelis complex), it was not possible to<br>
measure Km and kcat as independent paramters. At concentrations well below<br>
the Km, the Michaelis-Menten equation simplifies to:<br><br>
The specificity constant (ie Kcat/Km) can therefore be calculated from the<br>
above equation using the initial hydrolysis rate (pmol/min, calculated from<br>
the known specific activity of the radiolabelled substrate) and the<br>
concentrations of substrate and enzyme in the assay. The diffusion-limited<br>
maximum value for a specificity constant is 108-109 M-1sec-1 (Stryer, 1981).<br>
Example 4: Malathion Hydrolysis Assays<br>
MCE activity was assayed as described by Campbell et al. (1998), but<br>
without diluting the specific activity of the 14C malathion (25mCi mmol-1) for<br>
enzymes that appeared to have a low Km. This was an end-point assay in<br>
which malathion was extracted into an organic phase while radiolabelled<br>
malathion carboxylic acids, the hydrolysis products remained, in the aqueous<br>
phase. Activity was measured over the range 50nM to 1?M to determine the<br>
Km and Kcat, and analysed by non-linear regression using the Enzfitter 1.05<br>
software (Elsevier-Biosoft), with graphical output to reveal any deviation from<br><br>
Michaelis-Menten kinetics. Specificity constants were calculated directly<br>
from the Km and kcat values.<br>
Example 5: Pennethrin Hydrolytic Activity of E3 and EST23 Variants<br>
Table 2 summarises the kinetic data obtained for eighteen E3 and three<br>
EST23 variants using cis- and trans- pennethrin as substrates. The malathion<br>
hydrolytic activity of the enzymes is also given for comparison. In each case<br>
the data represent the hydrolysis of the enantiomer that is hydrolysed the<br>
fastest out of each of the 1S/1R cis and 1S/1R trans isomer pairs (see above).<br>
The E3WT enzyme found in OP susceptible blowflies, and its EST23 D.<br>
melanogaster orthologue, showed significant levels of pennethrin hydrolytic<br>
activity, which was specific for the trans isomers. The wild-type enzymes<br>
showed at least an order of magnitude higher activity for malathion (although<br>
this high MCE activity does not confer malathion resistance on the blowfly<br>
because the enzyme is readily inhibited by the malaoxon produced in vivo by<br>
the fly; Campbell et al., 1998). Mutations in either the acyl binding pocket or<br>
anionic site regions of the active site of the E3 enzyme resulted in significant<br>
increases in activity for both the irons and cis isomers of pennethrin. These<br>
increases in pennethrin hydrolysis were not in the main correlated with<br>
increases in malathion hydrolytic activity.<br>
a) Oxyanion hole mutations<br>
The E3G137D mutation is responsible for diazinon resistance in the<br>
sheep blowfly. In this mutant the very small, aliphatic, neutral Gly residue in<br>
the oxyanion hole region of the active site of the enzyme is replaced by an<br>
acidic Asp, allowing hydrolysis of a bound oxon OP molecule. However, this<br>
mutant (and its D. melanogaster orthologue) had reduced activity for trans-<br>
permethrin in particular, compared to that of the wild-type enzyme. This<br>
activity was not increased by substitution of Gly-137 with either His or Glu.<br>
However, substitution of Gly-137 with Arg did not affect the activity for<br>
either cis- or frans-pennethrin appreciably. The linear nature of Arg might<br>
mean that it can fold easily and not interfere with binding of pennethrin to<br>
the active site. The MCE activity of this group of mutants correlated broadly<br>
with their activity for trans pennethrin in particular, indicating effects of<br>
G137 substitutions on the accommodation and stabilisation of the substrate<br><br>
acyl group. Effects are generally smaller for permethrin than malathion but<br>
this is consistent with the somewhat smaller acyl group for permethrin.<br>
b) Acyl binding pocket mutations<br>
The E3W251L mutation, which replaces the large aromatic Trp reside<br>
with the smaller aliphatic Leu in the acyl pocket of the active site, resulted in<br>
a 7-fold increase in trans-permethrin hydrolysis and the acquisition of<br>
substantial cis-pennethrin hydrolysis. This is the mutation responsible for<br>
the acquisition of malathion resistance in the sheep blowfly. The MCE<br>
activity of this mutant was 2-fold higher than that of the wild-type enzyme.<br>
The effect of W251L in EST23 was essentially the same as for E3.<br>
Replacement of Trp-251 with even smaller residues in E3 (Thr, Ser, Ala and<br>
Gly in decreasing order of size) also resulted in an increase in permethrin<br>
hydrolytic activity, although the activity of these mutants was not as high as<br>
that of E3W251L. Clearly, steric factors are not the only consideration in the<br>
activity of the mutants. For example, Thr and Ser both contain hydroxyl<br>
groups and are hydrophilic. Furthermore, Ala is both aliphatic and<br>
hydrophobic (like Leu) and even smaller than Leu, yet this mutant was as<br>
active for permethrin as the W251L mutant. Opening up the oxyanion hole of<br>
the W251L mutant (ie E3P250S/W251L) also decreased its activity for both<br>
cis- and trans-permethrin, although the activity was still higher than that of<br>
the wild type. It is interesting to note that increases in specificity constants<br>
for permethrin for all W251 mutants in E3 as well as W251L in EST23<br>
compared to those of the wild types were uniformly more pronounced for the<br>
cis isomers. Whereas the wild type enzymes yielded trans:cis ratios of at least<br>
20:1, these ratios were only 2-6:1 for the W251 mutants. The extra space in<br>
the acyl pocket provided by these mutants was apparently of greatest benefit<br>
for the hydrolysis of the otherwise more problematic cis isomers.<br>
The MCE activity of the E3-251 mutants was not correlated with<br>
permethrin hydrolytic activity. Of this group of mutants, E3W251G had<br>
approximately 10-fold higher MCE activity than the remainder of the group,<br>
and yet its permethrin hydrolytic activity was among the lowest.<br>
Combination of both the W251L and G137D mutations on to the same<br>
E3 molecule increased the activity of the enzyme for cis permethrin over<br>
wild-type levels, but decreased the activity for trans-permethrin and also<br>
malathion. However, the activity of the double mutant was not as great as<br><br>
that of the mutant containing the E3W251L mutation alone (i.e. the mutations<br>
did not act additively).<br>
Some lipases are known to have a Leu residue at the position<br>
corresponding to Phe 309 in L cuprina E3. The E3F309L mutant was<br>
therefore constructed with the aim of conferring activity for lipophilic<br>
substrates like pyrethroids. As can be seen from Table 2, the E3F309L mutant<br>
was much better than E3WT for both isomers. It was even more active for<br>
trans-permethrm than E3W251L, though not as active for the cis isomers.<br>
However, the MCE activity of this mutant was less than half that of the wild-<br>
type enzyme. Combination of both the F309L and W251L mutations on the<br>
same E3 molecule increased the activity for cis-permethrin and decreased the<br>
activity for trans-permethrin to E3W251L levels. In other words, the F309L<br>
mutation had very little effect on the activity of the W251L mutant for<br>
pennethrin, but decreased its activity for malathion.<br>
c) Anionic site mutations<br>
Some lipases are known to have a Leu residue at the position<br>
corresponding to Phe 354 in L cuprina E3. However, substitution of Phe 354<br>
for Leu in E3 did not increase its activity for permethrin appreciably, but<br>
greatly reduced its activity for malathion. Substitution of Phe 354 for the<br>
bulkier aromatic residue, Trp, on the other hand, increased activity for both<br>
cis- and trans-permethrin 3-4-fold, but decreased MCE activity slightly. It is<br>
perhaps surprising that F354W, not F354L, should show increases in activity<br>
against the very lipohilic permethrin, given that it is a Leu that replaces Phe<br>
in some naturally occurring Upases.<br>
Although Y148F is of little consequence for MCE activity it has large<br>
effects on permethrin kinetics and the effects are opposite in direction<br>
depending on genetic background. As a single mutant compared to wild type<br>
it shows 5-6 fold enhancement of activity for both cis and trans permethrin.<br>
As a double mutant with G137D (which as a single mutant gives values much<br>
lower than wild type), it shows a further two fold reduction for trans<br>
permethrin and and almost obliterates activity for cis pennethrin. These<br>
latter results clearly imply a strong interaction of Y148 with the oxyanion<br>
hole in respect of permethrin hydrolysis.<br>
Glu-217, the residue immediately adjacent to the catalytic serine, is<br>
thought to be important in stabilising the transition state intermediate in<br><br>
hydrolysis reactions. However, mutating this residue to Met (E3E217M), as<br>
found naturally in the esterase E4 of the aphid M. persicae, had little effect on<br>
pennethrin activity but greatly reduced its MCE activity.<br>
Example 6: Hydrolysis of Bromo-Permethrin Analogue<br>
Table 2 also summarises the kinetic data obtained for the E3 and EST23<br>
variants using the two cis -dibromovinyl analogues of permethrin (NRDC157).<br>
The 1S cis isomer of this dibromo analogue of permethrin was hydrolysed<br>
with similar efficiency to the 1R/1S cis permethrin by all enzymes except<br>
E3F309L and F309L/W251L. This indicates that the larger bromine atoms did<br>
not substantially obstruct access of this substrate to the catalytic centre.<br>
Although the activities with the E3WT and EST23WT enzymes were too low<br>
for significant comparison between isomers, all other enzymes except<br>
E3F309L and F309L/W251L showed 10 to 100-fold faster hydrolysis of the 1S<br>
isomer. This is the same preference for this configuration at C1 of the<br>
cyclopropane ring as found previously for 1S trans pennethrin in M. persicae<br>
(Devonshire and Moores, 1982).<br>
F309L showed a dramatic effect on NRDC157 kinetics. The single<br>
mutant showed little difference from wild type for 1S cis and the double with<br>
W251L showed less activity than W251L alone for this isomer. However, the<br>
1S/1R preference was reversed, with values of 0.7:1 in the single mutant and<br>
0.4:1 in the double. The result is the two highest values for 1R cis activities<br>
in all the data set The value for the double mutant is in fact about 10 fold<br>
higher than those for either mutant alone.<br>
Example 7: Hydrolysis of Type D Pvrethroids bv Expressed Enzymes<br>
Table 3 summarises the kinetic data obtained for a sub-set of the E3<br>
and EST23 variants using the four deltamethrin cis isomers. With the<br>
exception of E3W251L and E3F309L, the 1R cis isomers of deltamethrin<br>
(whether ?S or ?R) were hydrolysed with similar efficiency to the 1R cis<br>
NRDG157 (which can be considered intermediate in character between<br>
permethrin and deltamethrin in that it has dibromovinyl substituent but lacks<br>
the ? cyano group). Activity against 1R cis isomers was always greater with<br>
the ?R than the ?S conformation. E3W251L and E3F309L were markedly less<br>
efficient with the 1R cis isomers of deltamathrin than with the corresponding<br>
isomers of NRDC157.<br><br>
TABLE 2: Specificity constants of natural and synthetic variants of L<br>
cuprina esterase E3 and D. melanogaster EST23 for the cis- and trans-isomers<br>
of permethrin, malathion and the two cis -dibromovinyl analogues of<br>
permethrin (NRDC157). Ratios of the specificity constants for trans and cis<br>
permethrin, and for IS cis and 1R cis NRDC157 are also indicated.<br><br><br><br>
1 Not determined<br>
2 Not substantially different from values obtained using control cell extracts<br>
Significantly, the 251 mutant with the highest deltamethrin activities<br>
was W251S, while W251L (highest for the other two pyrethroids), and<br>
W251G (highest for malathion) gave the lowest deltamethrin activities of the<br>
five 251 mutants. This suggests that accommodation of the acyano moiety of<br>
the leaving group may be the major impediment to efficient deltamethrin<br>
hydrolysis, sufficient to prevent any significant hydrolysis by wild type E3.<br>
Accommodation of substrate requires significantly different utilisation of<br>
space across the active site compared to other substrates, such that<br>
substitution of W251 in the acyl pocket with a smaller residue allows useful<br>
accommodation, particularly for ?R isomers. Importantly, however, the<br>
details of the spatial requirements, and therefore the most efficacious<br>
mutants, differ from those for the other pyrethroids.<br>
The activity of all enzymes with the 1S cis isomers of deltamethrin was<br>
dramatically less than with the corresponding isomer of NRCC157 lacking the<br>
a cyano group. This dramatic influence of the a cyano group appears to be<br>
expressed with this 1S conformation at C1 of the cyclopropane group. With<br>
the exception of some of the least active mutants, activity against 1S cis<br>
isomers was again always greater with the ?R than the ?S conformation.<br>
Example 8 - General Discussion<br>
Together, the permethrin and NRDC157 results for the 251 series<br>
mutants generate some quite strong and simple inferences about acyl binding<br>
constraints in E3/EST23. Overall, as with malathion, 251 replacements that<br>
should generate a more spacious acyl pocket do facilitate the<br>
accommodation/stabilisation of the bulky acyl groups of these substrates.<br>
These replacements are beneficial to the hydrolysis of all the isomers<br><br>
TABLE 3: Specificity constants for the four deltamethrin cis isomers<br><br>
generated by the two stereocentres across the cyclopropane ring. While the<br>
trans isomers are strongly preferred by wild type enzyme, the mutants can<br>
also hydrolyse at least part of the cis isomer mix relatively well. However,<br>
within the cis isomers the improvements in the mutants is much more<br>
marked for the 1S cis isomers. The 1R cis isomers, which are the most<br>
problematic of all configurations for wild type enzyme, remain the most<br>
problematic for the mutants. Within the mutant series, the improved kinetics<br><br>
are not simply explained by the reduction in side chain size; the smallest<br>
substitution does not give the highest activities, as it does for malathion.<br>
Indeed the best kinetics are obtained with W251L, although Leu has the<br>
greatest side chain size of all the replacements tested.<br>
In contrast to the relatively simple and consistent patterns seen for<br>
permethrin and NRDC157, the deltamethrin results for the 251 series mutants<br>
are quite complex and difficult to interpret As might be expected from their<br>
enhanced kinetics for the other substrates, they do show overall better<br>
activities than wild type for the four cis deltamethrin isomers, albeit as with<br>
wild type they are much lower in absolute terms than for the other substrates.<br>
However, the preference for 1S over 1R isomers, which is so strong in respect<br>
of NRDC157, is weak at best in the deltamethrin data. On the other hand<br>
there is a clear trend across all the mutants for a preference for the ?R over<br>
?S isomers. It is generally only of the order of 2:1, but notably it is opposite<br>
to the trend shown by wild type EST23. It is at first sight unexpected that<br>
these presumptive acyl binding pocket replacements should affect ?R/?S<br>
stereopreferences because the latter apply to the ?-cynano moiety in the<br>
(alcohol) leaving group of the substrate.<br>
Overall the F309L data clearly show a major effect of this residue on<br>
the kinetics of pyrethroid hydrolysis. At one level there are parallels with the<br>
results for the W251 series mutants, both data sets showing enhanced<br>
kinetics consistent with expectations based on the provision of greater space<br>
in the acyl binding pocket. However, there are also important differences,<br>
with the W251 series disproportionately active for the cis vs trans isomers of<br>
permethrin and F309L disproportionately active with 1R vs 1S isomers of cis<br>
NRDC157. The replacements at the two sites also show strong interactions,<br>
consistent with them contributing to a shared structure and function in the<br>
acyl binding pocket For example, both the disproportionate enhancement of<br>
the W251 mutants for cis permethrin and the disproportionate enhancement<br>
of F309L for 1R cis NRDC157 behave as dominant characters in the double<br>
mutant. The 251 and 309 mutants have quantitatively similar enhancing<br>
effects on activities and the same stereospecificities in respect of deltamethrin<br>
hydrolysis and the stereospecific differences seen with the smaller<br>
pyrethroids are not seen. However, we argue that the additional bulk of the<br>
acyano moiety in its leaving group requires such a radical reallocation of<br><br>
space across the active site that the stereospecificities evident with the<br>
smaller pyrethroids are overridden.<br>
It will be appreciated by persons skilled in the art that numerous<br>
variations and/or modifications may be made to the invention as shown in the<br>
specific embodiments without departing from the spirit or scope of the<br>
invention as broadly described. The present embodiments are, therefore, to<br>
be considered in all respects as illustrative and not restrictive.<br>
All publications discussed above are incorporated herein in their<br>
entirety.<br>
Any discussion of documents, acts, materials, devices, articles or the<br>
like which has been included in the present specification is solely for the<br>
purpose of providing a context for the present invention. It is not to be taken<br>
as an admission that any or all of these matters form part of the prior art base<br>
or were common general knowledge in the field relevant to the present<br>
invention as it existed before the priority date of each claim of this<br>
application.<br><br>
References:<br>
Broomfield, G.A. (1999). Chemico-Biological Interactions 120: 251-256.<br>
Campbell, P.M., Harcourt, R.L., Crone, E.J., Claudianos, C., Hammock, B.D.,<br>
Russell, R.J. and Oakeshott, J.G. (2001) Insect Biochem Molec Biol 31:513-520.<br>
Campbell, P.M., Newcomb, R.D., Russell, R.J. and Oakeshott, J.G. (1998a)<br>
Insect Biochem Molec Biol 28, 139-150.<br>
Campbell, P.M., Yen, J.L., Masoumi, A., Russell, R.J., Batterham, P.,<br>
McKenzie, J.A., and Oakeshott, J.G. (1998b) /. Econ. Entomol. 91:367-375.<br>
Claudianos, C., Crone, E., Coppin, C, Russell, R. and Oakeshott, J. (2002) In:<br>
Marshall Clark, J. and Yamagushi, I., (Eds.) Agrochemical Resistance: Extent,<br>
Mechanism and Detection, (in press)<br>
Claudianos, C., Russell, R.J. and Oakeshott, J.G. (1999) insect Biochem Molec<br>
Biol 29, 675-86.<br>
Cygler, M. and Schrag, J.D. (1997) Methods Enzymol 284, 3-27.<br>
Derewenda, U., Brzozwski, A.M., Lawson, D.M., Derewenda, Z.S. (1992)<br>
Biochemistry 31:1532-1541.<br>
Devonshire, A.L., Heidari, R., Bell, K.L., Campbell, P.M., Campbell, B.E.,<br>
Odgers, W.A., Oakeshott, J.G. and Russell, R.J. Kinetic Efficiency of Mutant<br>
Carboxylesterases Implicated in Organophosphate Insecticide Resistance, (in<br>
preparation)<br>
Devonshire, A.L. and Moores, G.D. (1982) Pestic. Biochem. Physiol 18, 235-<br>
246.<br>
Fournier, D., Bride, J.-M., Hoffmann, F. and Karch, F. (1992) J. Biol. Chem.<br>
267, 14270-14274.<br><br>
Gordon, R.K., Feaster, S.R., Russell, A.J., LeJeune, K.E., Maxwell, M.D., Lenz,<br>
D.E., Ross, M. and Doctor, B.P. (1999) Chem Biol Interact 14, 463-70.<br>
Grochulski, P., Li, Y., Schrag, J. D., Bouthillier, F., Smith, P., Harrison, D.,<br>
Rubin, B., Cygler, M. (1993). J Biol Chem 268, 12843-12847.<br>
Harel, M., Kryger, G., Rosenberry, T.L., Mallender, W.D., Lewis, T., Fletcher,<br>
R.J., Guss, M., Silman, I. and Sussman, J.L. (2000) Protein Science 9, 1063-<br>
1072.<br>
Jarv, J. (1984) Bioorganic Chemistry 12, 259-278.<br>
Koellner, G., Kryger, G., Millard, C.B., Silman, I., Sussman, J.L. and Steiner,<br>
T. (2000) The Journal of Molecular Biology 296, 713-735.<br>
LeJuene, K.E., Wild, J.R and RusseU, A.J. (1998) Nature 395, 27-8.<br>
Martinez, C., Nicolas, A., van Tilbeurgh, H., Egloff, M.P., Gudrey, C., Verger,<br>
R. and Cambillau, C. (1994) Biochemistry 33, 83-9.<br>
Myers, M.A., Healy, M.J. and Oakeshott, J.G. (1993) Biochem Genet 31, 259-<br>
78.<br>
Nair, H.K., Seravalli, J., Arbuckle, T. and Quinn, D.M. (1994) Biochemistry<br>
33, 8566-76.<br>
Needleman, S.B. and Wunsch, CD. (1970) JMolBiol 48, 443-53.<br>
Newcomb, R.D., Campbell, P.M., Ollis, D.L., Cheah, E., Russell, R.J. and<br>
Oakeshott, J.G. (1997) Proc Natl Acad Sci USA 94, 7464-8.<br>
Oakeshott, J.G., Claudianos, C., RusseU, RJ. and Robin, G.C. (1999) BioEssays<br>
21, 1031-42.<br>
Oakeshott, J.G., van Papenrecht, E.A., Boyce, T.M., Healy, M.J. and Russell,<br>
R.J. (1993) Genetica 90, 239-268.<br><br>
Ordentlich, A, Barak, D., Kronman, C., Ariel, N., Segal, Y., Velan, B, and<br>
Shaferman, A. (1998) The Journal of Biological Chemistry 273, 19509-19517.<br>
Ordentlich, A, Barak, D., Kronman, C, Ariel, N., Segal, Y., Velan, B. and<br>
Shaferman, A. (1996) The Journal of Biological Chemistry 271, 11953-11962.<br>
Ordentlich, A., Barak, D., Kronman, C., Ariel, N., Segal, Y., Velan, B. and<br>
Shaferman, A. (1995) The Journal of Biological Chemistry 270, 2082-2091.<br>
Ordentlich, A, Barak, D., Kronman, G, Flashner, Y., Leitner, M., Segal, Y.,<br>
Ariel, N., Cohen, S., Velan, B. and Shafferman, A. (1993) The Journal of<br>
Biological Chemistry 268, 17083-17095.<br>
Patten, P.A., Howard, R.J. and Stemmer, W.P. (1997) Curr Opin Biotechnol 8,<br>
724-33.<br>
Petrikovics, L, Cheng, T.C., Papahadjopoulos, D., Hong, K., Yin, R., DeFrank,<br>
J.J., Jaing, J., Zong, Z.H., McGuinn, W.D., Sylvester, D., Pei, L., Madec, J.,<br>
Tamulinas, C, Jaszberenyi, J.C., Barcza, T. and Way, J.L (2000a) Toxicol Sci<br>
57, 16-21.<br>
Petrikovics, L, McGuinn, W.D., Sylvester, D., Yuzapavik, P., Jaing, J., Way,<br>
J.L., Papahadjopoulos, D., Hong, K., Yin, R., Cheng, T.C., and DeFrank, J.J.<br>
(2000b) Drug Delivery 7: 83-89.<br>
Qian, N. and Kovach, I.M. (1993) FEBS Lett 336, 263-6.<br>
Robin, C, R. J. Russell, K. M. Medveczky, and J. G. Oakeshott. (1996) J Mol<br>
Evol 43:241-52.<br>
Shafferman, A, Velan, B., Orentlich, A, Kronman, C., Grosfeld, H., Leitner,<br>
M., Flachner, Y., Cohen, S., Barak, D. and Ariel, N. (1992) EMBOJ. 11, 3561-<br>
3568.<br>
Stryer, L. (1981) Biochemistry, p 115, W.H. Freeman, San Francisco.<br><br>
Sussman, J.S., Harel, M., Frolov, F., Oefher, C., Goldman, A., Toker, L. and<br>
Silman, I. (1991) Science 253, 872-879.<br>
Thomas, B.A., Church, W.B., Lane, T.R. and Hammock, B.D. (1999) Proteins<br>
34, 184-96.<br>
Villatte, F., Ziliani, P., Marcel, V., Menozzi, P. and Fouraier, D. (2000)<br>
Pesticide Biochemistry and Physiology 95-102.<br>
Walsh, S.B., Dolden, T.A., Moores, G.D., Kristensen, M., Lewis, T.,<br>
Devonshire, A.L. and Williamson, M.S. (2001) Biochem J 359, 175-81.<br>
Yao, H., Chunling, Q., Williamson, M.S. and Devonshire, A.L. (1997) Clin. J.<br>
Biotechnol 13:177-183.<br>
Zhu, Y.-C., Dowdy, A.K. and Baker, J.E. (1999) Insect Biochem Molec. Biol.<br>
29:417-425.<br><br>
We claim:<br>
1. A method of eliminating or reducing the concentration of a hydrophobic<br>
ester pesticide or toxin in a sample, the method comprising contacting the<br>
sample with an Dipteran esterase, or a mutant thereof, wherein the Dipteran<br>
esterase, or a mutant thereof, comprises a sequence selected from the group<br>
consisting of:<br>
i) a sequence as shown in SEQ ID NO:1,<br>
ii) a sequence as shown in SEQ ID NO:2, and<br>
iii) a sequence which is at least 40% identical to i) or ii) which is capable<br>
of hydrolysing a hydrophobic ester pesticide or toxin.<br>
2. The method as claimed in claim 1, wherein the sequence is at least 80%<br>
identical to i) or ii).<br>
3. The method as claimed in claim 1, wherein the sequence is at least 90%<br>
identical to i) or ii).<br>
4. The method as claimed in any one of claims 1 to 3, wherein the mutant<br>
Dipteran esterase has a mutation(s) in an oxyanion hole, acyl binding pocket<br>
or anionic site regions of an active site of the esterase, or any combination<br>
thereof.<br>
5. The method as claimed in claim 4, wherein the mutant Dipteran<br>
esterase is selected from the group consisting of: E3G137R, E3G137H,<br>
E3W251L, E3W251S, E3W251G, E3W251T, E3W251A, E3W251L/F309L,<br>
E3W251L/G137D, E3W251L/P250S, E3F309L, E3Y148F, E3E217M,<br>
E3F354W, E3F354L, and EST23W251L.<br>
6. The method as claimed in any one of claims 1 to 5, wherein the method<br>
is performed using two or more Dipteran esterases, or mutants thereof.<br>
7. The method as claimed in any one of claims 1 to 6, wherein the<br>
hydrophobic ester pesticide or toxin is a pyrethroid.<br>
8. The method as claimed in claim 7, wherein the pyrethroid is a Type I or<br>
Type II pyrethroid.<br><br>
9. The method as claimed in claim 8, wherein the Type I pyrethroid is<br>
selected from the group consisting of: 1S/1R trans permethrin, 1S/1R cis<br>
permethrin, NRDC157 1S cis, and NRDC157 1R cis.<br>
10. The method as claimed in claim 8, wherein the Type II pyrethroid is<br>
deltamethrin.<br>
11. The method as claimed in any one of claims 1 to 10, wherein the method<br>
is performed in a liquid containing environment.<br>
12. The method as claimed in any one of claims 1 to 11, wherein the<br>
Dipteran esterase, or mutant thereof, is provided directly to the sample.<br>
13. The method as claimed in any one of claims 1 to 11, wherein the<br>
Dipteran esterase, or mutant thereof, is provided to the sample by expression<br>
of a polynucleotide encoding the Dipteran esterase, or mutant thereof, from a<br>
host cell comprising the polynucleotide.<br>
14. The method as claimed in any one of claims 1 to 11, wherein the<br>
Dipteran esterase, or mutant thereof, is provided as a polymeric sponge or<br>
foam, the foam or sponge comprising the Dipteran esterase, or mutant<br>
thereof, immobilized on a polymeric porous support.<br>
15. The method as claimed in any one of claims 1 to 14, wherein the method<br>
further comprises the presence of a surfactant when the hydrophobic ester<br>
pesticide or toxin is contacted with the Dipteran esterase, or mutant thereof.<br>
16. The method as claimed in claim 15, wherein the surfactant is a<br>
biosurfactant.<br>
17. A substantially purified polypeptide which is a mutant of an Dipteran<br>
esterase, wherein one or more mutations are within a region of the esterase<br>
selected from the group consisting of: oxyanion hole, acyl binding pocket and<br>
anionic site, wherein the mutant Dipteran esterase is capable of hydrolysing a<br>
hydrophobic ester pesticide or toxin, and wherein the mutant is at least 40%<br>
identical to at least one of SEQ ID NO's:1 or 2, with the proviso that the<br>
mutant Dipteran esterase is not E3W251L, E3W251S, E3W251G or E3G137D.<br><br>
18. The polypeptide as claimed in claim 17, wherein the mutant is at least<br>
80% identical to at least one of SEQ ID NO's:1 or 2.<br>
19. The polypeptide as claimed in claim 17, wherein the mutant is at least<br>
90% identical to at least one of SEQ ID NO's:1 or 2.<br>
20. The polypeptide as claimed in any one of claims 17 to 19, wherein the<br>
mutation is a point mutation.<br>
21. The polypeptide as claimed in claim 17 comprising a sequence selected<br>
from the group consisting of: E3G137R, E3G137H, E3W251T, E3W251A,<br>
E3W251L/F309L, E3W251L/G137D, E3W251L/P250S, E3F309L, E3Y148F,<br>
E3E217M, E3F354W, E3F354L, and EST23W251L.<br>
22. A fusion polypeptide comprising a polypeptide as claimed in any one of<br>
claims 17 to 21 fused to at least one other polypeptide sequence.<br>
23. An isolated polynucleotide encoding a polypeptide as claimed in any<br>
one of claims 17 to 22.<br>
24. A vector for replication and/or expression of a polynucleotide as<br>
claimed in claim 23.<br>
25. A host cell transformed or transfected with the vector as claimed in<br>
claim 24.<br>
26. A composition for hydrolysing a hydrophobic ester pesticide or toxin,<br>
the composition comprising a polypeptide as claimed in any one of claims 17<br>
to 22, and one or more carriers.<br>
27. A method for generating and selecting an enzyme that hydrolyses a<br>
hydrophobic ester pesticide or toxin, the method comprising<br>
(a) introducing one or more mutations into an Dipteran esterase, or an<br>
Dipteran esterase that has already been mutated, and<br>
(b) determining the ability of the mutant Dipteran esterase to hydrolyse<br>
a hydrophobic ester pesticide or toxin,<br>
wherein the Dipteran esterase, or a mutant thereof, comprises a sequence<br>
selected from the group consisting of:<br><br>
i) a sequence as shown in SEQ ID NO:1,<br>
ii) a sequence as shown in SEQ ID NO: 2, and<br>
iii) a sequence which is at least 40% identical to i) or ii).<br>
28. The method of claim 27, wherein the sequence is at least 80% identical<br>
to i) or ii).<br>
29. The method of claim 27, wherein the sequence is at least 90% identical<br>
to i) or ii).<br>
30. The method as claimed in any one of claims 27 to 29, wherein the one or<br>
more mutations enhances hydrolytic activity and/or alters the stereospecificty<br>
of the esterase.<br>
31. The method as claimed in any one of claims 27 to 30, wherein the one or<br>
more mutations are within a region of the esterase selected from the group<br>
consisting of: oxyanion hole, acyl binding pocket and anionic site.<br>
32. The method as claimed in any one of claims 27 to 31, wherein the<br>
mutation is a point mutation.<br>
33. The method as claimed in claim 27, wherein the Dipteran esterase that<br>
has already been mutated is selected from the group consisting of: E3G137R,<br>
E3G137H, E3W251L, E3W251S, E3W251G, E3W251T, E3W251A,<br>
E3W251L/F309L, E3W251L/G137D, E3W251L/P250S, E3F309L, E3Y148F,<br>
E3E217M, E3F354W, E3F354L, and EST23W251L.<br>
An invention discloses a method of eliminating or reducing the concentration of a<br>
hydrophobic ester pesticide or toxin in a sample, the method comprising contacting<br>
the sample with an Dipteran esterase, or a mutant thereof, wherein the Dipteran<br>
esterase, or a mutant thereof, comprises a sequence selected from the group<br>
consisting of:<br>
i) a sequence as shown in SEQ ID NO:1,<br>
ii) a sequence as shown in SEQ ID NO:2, and<br>
iii) a sequence which is at least 40% identical to i) or ii) which is capable of<br>
hydrolysing a hydrophobic ester pesticide or toxin.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
		<br>
		<div class="pull-left">
			<a href="224715-process-for-multiple-access-and-multiple-transmission-of-data-in-a-multi-user-system-the-point-to-multipoint-digital-transmission-of-data-over-the-electricity-network.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="224717-photoinitiator-novel-compound-and-photocurable-composition.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>224716</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1135/KOLNP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>43/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>24-Oct-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>22-Oct-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>09-Aug-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>COMMONWEALTH SCIENTIFIC AND INDUSTRIAL RESEARCH ORGANISATION</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>LIMESTONE AVENUE, CAMPBELL, AUSTRALIAN CAPITAL TERRITORY</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>HEIDARI RAMA</td>
											<td>5 SARRE STREET, GUNGHALIN AUSTRALIAN CAPITAL TERRITORY 2912</td>
										</tr>
										<tr>
											<td>2</td>
											<td>DEVONSHIRE ALAN</td>
											<td>1A GROVE AVENUE, HARPENDEN, HERTS AL5 1EU</td>
										</tr>
										<tr>
											<td>3</td>
											<td>DORRIAN SUSAN JANE</td>
											<td>10 FOSKETT STREET, FRASER, AUSTRALIAN CAPITAL TERRITORY 2615</td>
										</tr>
										<tr>
											<td>4</td>
											<td>RUSSELL ROBYN JOYCE</td>
											<td>28 SULLIVAN CRESENT, WANNIASSA, AUSTRALIAN CAPITAL TERRITORY 2903</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12P 7/62</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/AU02/00114</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-02-06</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/224716-degradation-of-hydrophobic-ester-pesticides-and-toxins by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:08:36 GMT -->
</html>
