<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/224897-methods-for-the-recombinant-production-of-antifusogenic-peptides by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:14:24 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 224897:METHODS FOR THE RECOMBINANT PRODUCTION OF ANTIFUSOGENIC PEPTIDES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHODS FOR THE RECOMBINANT PRODUCTION OF ANTIFUSOGENIC PEPTIDES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A process for the reombinant production of an antifusogenic peptide by expression of a nucleic acid encoding said antifusogenic peptide as repaeat peptide in a microbial host cell, isolation of inclusion bodies containing said repeat peptide, solubilization of the inclusion bodies, and isolation of the antifusogenic peptide after cleavage, characterized bywhashing the inclusion bodies with a denaturing agent at a pH value of or below pH 6.5, solubilizing said inclusion bodies containing the repeat peptide at a pH value of at least pH 9 and cleaving said repeat peptide to obtain said antifusogenic peptide.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The invention relates to methods for the recombinant production of peptides which inhibit the fusion of viruses with membranes of target cells. In particular, this invention relates to the recombinant production of peptidic inhibitors of lentivirus such as human immunodeficiency virus (HIV), Simian immunodeficiency virus (SIV), measles virus (MEV), influenza viruses such as respiratory syncytical virus (RSV) or human parainfluenza virus (HPV).<br>
Background of the Invention<br>
Fusion of viruses with cellular membranes is an essential step for the entry of enveloped viruses, such as HIV-I, HIV-II, RSV, measles virus, influenza virus, parainfluenza virus, Epstein-Barr virus and hepatitis virus, into cells. After having entered the cell the cascade of viral replication may be initiated resulting in viral infection.<br>
HIV is a member of the lentivirus genus, which includes retroviruses that possess complex genomes and exhibit cone-shaped capsid core particles. Other examples of lentiviruses include the simian immunodeficiency virus (SIV), visna virus, and equine infectious anemia virus (EIAV). Like all retroviruses, HIV's genome is encoded by RNA, which is reverse-transcribed to viral DNA by the viral reverse transcriptase (RT) upon entering a new host cell. Influenza viruses and their cell entry mechanisms are described by Bullough, P.A., et al., Nature 371 (1994) 37-43; Carr, CM., and Kim, P.S., Cell 73 (1993) 823-832; and Wilson, I.A., et al., Nature 289(1981)366-373.<br>
All lentiviruses are enveloped by a lipid bilayer that is derived from the membrane of the host cell. Exposed surface glycoproteins (SU, gpl20) are anchored to the virus via interactions with the transmembrane protein (TM, gp41). The lipid bilayer also contains several cellular membrane proteins derived from the host cell, including major histocompatibility antigens, actin and ubiquitin (Arthur, L.O., et al., Science 258 (1992) 1935-1938). A matrix shell comprising approximately 2000 copies of the matrix protein (MA, pi7) lines the inner surface of the viral membrane, and a conical capsid core particle comprising ca. 2000 copies of the capsid protein (CA,<br><br>
p24) is located in the center of the virus. The capsid particle encapsidates two copies of the unspliced viral genome, which is stabilized as a ribonucleoprotein complex with ca. 2000 copies of the nucleocapsid protein (NC, p7), and also contains three essential virally encoded enzymes: protease (PR), reverse transcriptase (RT) and integrase (IN). Virus particles also package the accessory proteins, Nef, Vif and Vpr. Three additional accessory proteins that function in the host cell, Rev, Tat and Vpu, do not appear to be packaged.<br>
In the case of HIV, viral entry is associated with the HIV envelope surface glycoproteins (Lawless, M.K., et al, Biochemistry 35 (1996) 13697-13708; and Turner, B.G., and Summers, M.F., J. Mol. Biol. 285 (1999) 1-32). In the case of HIV-I, this surface protein is synthesized as a single 160 kD precursor protein, which is cleaved by a cellular protease into two glycoproteins gp-41 and gp-120. gp-41 is a transmembrane protein, and gp-120 is an extracellular protein_which remains non-covalently associated with gp-41 in a trimeric or multimeric form (Hammarskjold, M.-L., et al., Biochim. Biophys. Acta 989 (1989) 269-280). HIV is targeted to CD4^ lymphocytes because the CD4 surface protein acts as the cellular receptor for the HIV-I virus. Viral entry into cells is dependent upon gp-120 binding to the cellular CD41 receptor molecules while gp-41 anchors the envelope glycoprotein complex in the viral membrane and mediates membrane fusion (McDougal, J.S., et al., Science 231 (1986) 382-385; and Maddon, P.J., et al, Cell 47 (1986)333-348).<br>
gp41 is the transmembrane subunit that mediates fusion of viral and cellular membranes. The gp41 ectodomain core is a six-helix bundle composed of three helical hairpins, each consisting of an N helix paired with an antiparallel C helix (Chan, D.C., et al., Cell 89 (1997) 263-273; Weissenhorn, W., et al., Nature 387 (1997) 426-430; Tan, K., et al., Proc. Natl. Acad. Sci. USA 94 (1997) 12303-12308). The N helices form an interior, trimeric coiled coil with three conserved, hydrophobic grooves; a C helix packs into each of these grooves. This structure likely corresponds to the core of the fusion-active state of gp41. According to Chan, D.C., et al., Proc. Natl. Acad. Sci. USA 95 (1998) 15613-15617, there is evidence that a prominent cavity in the coiled coil of the HIV type 1 gp41 is an attractive drug target.<br><br>
It is assumed that the mechanism by which gp-41 mediates membrane fusion may involve the formation of a coiled-coil trimer, which is thought to drive the transition from resting to fusogenic states, as is described, for example, for influenza hemagglutinin (Wilson, I.A., et al., Nature 289 (1981) 366-373; Carr, CM., and Kim, P.S., Cell 73 (1993) 823-832; Bullough, P.A., et al., Nature 371 (1994)37-43).<br>
C peptides (peptides corresponding to the C helix) of enveloped viruses, such as DP 178 and C34, potently inhibit membrane fusion by both laboratory-adapted strains and primary isolates of HIV-1 (Malashkevich, V.N., et al., Proc. Natl. Acad. Sci. USA 95 (1998) 9134-9139; Wild, C.T., et al., Proc. Natl. Acad. Sci. USA 91 (1994) 9770-9774). A Phase 1 clinical trial with the C peptide DP178 suggests that it has antiviral activity in vivo, resulting in reduced viral loads (Kilby, J.M., et al., Nature Medicine 4 (1998) 1302-1307). The structural features of the gp41 core suggest that these peptides act through a dominant-negative mechanism, in which C peptides bind to the central coiled coil of gp41 and lead to its inactivation (Chan, D.C., et al., Cell 93 (1998) 681-684).<br>
Within each coiled-coil interface is a deep cavity, formed by a cluster of residues in the N helix coiled coil, that has been proposed to be an attractive target for the development of antiviral compounds. Three residues from the C helix (Trp-628, Trp-631, and Ile-635) insert into this cavity and make extensive hydrophobic contacts. Mutational analysis indicates that two of the N-helix residues (Leu-568 and Trp-571) comprising this cavity are critical for membrane fusion activity (Cao, J., et al., J. Virol. 67 (1993) 2747-2755). Therefore, compounds that bind with high affinity to this cavity and prevent normal N and C helix pairing may be effective HIV-1 inhibitors. The residues in the cavity are highly conserved among diverse HIV-1 isolates. Moreover, a C peptide containing the cavity-binding region is much less susceptible to the evolution of resistant virus than DP 178, which lacks this region (Rimsky, L.T., et al., J. Virol. 72 (1998) 986-993). These observations suggest that high-affinity ligands targeting the highly conserved coiled-coil surface, particularly its cavity, will have broad activity against diverse HIV isolates and are less likely to be bypassed by drug-escape mutants.<br>
Fusogenic structures of envelope fusion proteins was shown from influenza, Moloney murine leukemia virus, and simian immunodeficiency virus (cit. in Chan,<br><br>
D.C., Proc. Natl. Acad. Sci. USA 95 (1998) 15613-15617), human respiratory syncytial virus, Ebola, human T cell leukemia virus, simian parainfluenza. It indicates a close relationship between the families of orthomyxoviridae, paramyxoviridae, retroviridae, and others like filoviridae, in which viral entry into target cells is enabled by like transmembrane glycoproteins such as gp41 of HIV-1, hemagglutinin of influenza, pP2 of Ebola and others (Zhao, X., et al., Proc. Natl. Acad. Sci. USA 97 (2000) 14172-14177).<br>
In the state of the art, methods are described for the preparation of peptidic inhibitors (C-peptides) (see, e.g., Root, M.J., et al., Science 291 (2001) 884-888; Root et al. describe peptide C37-H6 which is derived from HIV-1. HXB2 and contains residues 625-661. It was recombinantly expressed as N40-segement with a GGR-linker and a histidine tag, expressed in E.coli and purified from the soluble fraction of bacterial lysates. Zhao, X., et al. describe in Proc. Natl. Acad. Sci. USA 97 (2000) 14172-14177 a synthetic gene of recRSV-1 (human respiratory syncytial virus) which encodes Residues 153-209, a G-rich linker, residues 476-524, Factor Xa cleavage site and a his-tag. Chen, C.H., et al., describe in J. Virol. 69 (1995) 3771-3777 the recombinant expression of the extracellular domain of gp41 synthesized as fusion protein, residues 540- 686, fusion to MBP.<br>
A number of peptidic inhibitors, also designated as antifusogenic peptides, of such membrane fusion-associated events are known, including, for example, inhibiting retroviral transmission to uninfected cells. Such peptides are described, for example, by Lambert, D.M., et al., Proc. Natl. Acad. Sci. USA 93 (1996) 2186-2191 in U.S. Patent Nos. 6,013,263; 6,017,536; and 6,020,459; and in WO 00/69902, WO 99/59615 and WO 96/40191. Further peptides inhibiting fusing associated events are described, for example, in U.S. Patent Nos. 6,093,794; 6,060,065; 6,020,459; 6,017,536; 6,013,263; 5,464,933; 5,656,480; and in WO 96/19495.<br>
Examples of linear peptides derived from the HIV-I gp-41 ectodomain which inhibit viral fusion are DP-107 and DP-178. DP-107 is a portion of gp-41 near the N-terminal fusion peptide and has been shown to be helical, and it strongly oligomerizes in a manner consistent with coiled-coil formation (Gallaher, W.R., et al., Aids Res. Hum. Retrovirus 5 (1989) 431-440, Weissenhorn, W., et al., Nature 387 (1997) 426-430). DP-178 is derived from the C-terminal region of the gp-41 ecto-domain. (Weissenhorn, W., et al., Nature 387 (1997) 426-430). Although<br><br>
without discernible structure in solution this peptide and constrained analogs therefrom adopt a helical structure, bind to a groove of the N-terminal coiled-coil trimer of gp41 and thus prevent the gp41 to transform into the fusogenic state (Judice, J. K., et al., Proc. Natl. Acad. Sci. USA 94 (1997) 13426-13430).<br>
Such short-chain peptides usually are prepared by chemical synthesis. Chemical synthesis is described, for example, by Mergler, M., et al., Tetrahedron Letters 29 (1988) 4005-4008 and 4009-4012; Andersson, L., et al., Biopolymers 55 (2000) 227-250; and by Jones, J.H., J. Pept. Sci. 6 (2000) 201-207. Further methods are described in WO 99/48513.<br>
However, chemical peptide synthesis suffers from several drawbacks. Most important is racemization, which results in insufficient optical purity. In peptide chemistry, racemization also means epimerization at one of several chirality centers. If only 1% racemization occurs for a single coupling step, then at 100 coupling steps only 61% of the target peptide would be received (Jakubke, H.D., Peptide, Spektrum Akad. Verlag, Heidelberg (1996), p. 198). It is obvious that the number of impurities increases with growing chain length and their removal is more and more difficult and costly.<br>
Chemical synthesis on large scale is limited by high costs and lack of availability of protected amino acid derivatives as starting materials. On the one hand, these starting materials should be used in excess to enable complete reactions, on the other hand, their use should be balanced for cost reasons, safety and environmental aspects (Andersson et al., Biopolymers 55 (2000) 227-250).<br>
Lepage, P., et al., in Analytical Biochemistry 213 (1993) 40-48, describe recombinant methods for the production of HIV-1 Rev peptides. The peptides are expressed as fusion proteins with the synthetic immunoglobulin type G (IgG) binding domains of Staphylococcus aureus protein A. The peptides have a length of about 20 amino acids, whereas the IgG-binding part has a length of about 170 amino acids, so that the expressed fusion protein has an overall length of about 190 amino acids. This fusion protein is expressed, secreted in soluble form in the " medium, and purified by affinity chromatography. The authors reported that with this method it might be possible to produce recombinant protein in an amount of hundreds of milligrams per liter of culture. However, this methodology is limited<br><br>
due to alternative processing within the signal peptide sequence and several post-translational modifications of the fused proteins as well as of the cleaved peptides. Assuming an average molecular weight of an amino acid of 110 Daltons, the desired peptides have a molecular weight of about 2,000 to 5,000 Daltons, whereas the fusion tail has a length of at least 170 amino acids (about 19,000 D), if the IgG binding domains of Staphylococcus aureus protein A is used as such a fusion tail. Therefore, only 10 to 25% of the recombinantly produced protein is the desired peptide.<br>
Further examples and methods for the recombinant production of small peptides via fusion proteins in E.coli are described by Uhlen, M., and Moks, T., Methods Enzymol. 185 (1990) 129-143, T.J.R.: Expression of eukaryotic genes in E. coli. In: Genetic Engineering (Williamson, R. ed.), Academic Press, London, vol. 4, 127-185 (1983); and Kopetzki, E., et al., Clin. Chem. 40 (1994) 688-704. Ningyi, L., et al., Gaojishu Tongxun 10 (2000) 28-31 describe the recombinant expression of p24 gag gene in E.coli.<br>
International Application WO 02/103026 describes a process for the production of an antifusogenic peptide as a fusion peptide of a length of about 14 to 70 amino acids in a prokaryotic host cell, characterized in that, under such conditions that inclusion bodies of said fusion peptide are formed, in said host cell there is expressed a nucleic acid encoding said fusion peptide consisting of said antifusogenic peptide of a length of about 10 to 50 amino acids N-terminally linked to a further peptide of a length of about 4 to 30 amino acids; said host cell is cultivated; said inclusion bodies are formed, recovered and solubilized; said fusion peptide is isolated.<br>
The object of the present invention was to provide a method which enables the recombinant production of a high yield of antifusogenic peptides via the inclusion body route and which is suitable for the large-scale industrial production of such peptides.<br>
The invention therefore provides a process for the recombinant production of an antifusogenic peptide by expression of a nucleic acid encoding said antifusogenic peptide as repeat peptide in a microbial host cell, preferably in a prokaryotic host cell, isolating of inclusion bodies containing said repeat peptide, solubilization of<br><br>
the inclusion bodies, and isolation of the antifusogenic peptide after cleavage, characterized by washing the inclusion bodies with a denaturing agent at a pH value of or below pH 6.5, solubilizing said inclusion bodies containing the repeat peptide at a pH value of at least pH 9 and cleaving said repeat peptide to obtain said antifusogenic peptide.<br>
It was surprisingly found that inclusion bodies of repeats of polypeptides having antifusogenic activity (hereinafter also referred to as fusion polypeptides) are resistant to denaturing conditions at pH values of 6.5 and below. Based on this, it is possible to purify inclusion bodies containing such repeat peptides in a highly effective manner from host cell polypeptides and other host cell-derived impurities by the conditions according to the invention, as most of all substances contained in the inclusion bodies are easily soluble under such denaturing conditions and therefore can be removed by simple washing of the inclusion bodies under such denaturing conditions.<br>
In a subsequent step, the inclusion bodies are solubilized at a pH value of 9 and higher for recovery of the soluble repeat peptide, whereby it is not necessary to add detergents or denaturing agents. After solubilisation, the repeat is cleaved chemically or enzymatically to obtain the desired antifusogenic peptide.<br>
In a preferred embodiment of the invention, a cleavage sequence is located between the antifusogenic peptides of the repeat(s) and the repeat peptide is included in a fusion polypeptide characterized in that the repeats are linked together with said cleavage sequence(s) which are preferably of a length of about 1 to 10 amino acids. The antifusogenic peptide itself has a length of 10 to 100 amino acids, whereby (in regard to stable expression and formation of inclusion bodies) the length of the fusion polypeptide preferably is at least about 50 amino acids.<br>
Therefore the invention provides an extremely simple method for recombinant production of antifusogenic peptides via the inclusion bodies route by simply washing the inclusion bodies and subsequently solubilizing them at different pH values.<br><br>
Detailed Description of the Invention<br>
"Antifusogenic" and "anti-membrane fusion" as used herein refer to a peptide's ability to inhibit or reduce the level of fusion events between two or more structures, e.g., cell membranes or viral envelopes or pili relative to the level of membrane fusion which occurs between the structures in the absence of the peptide. Examples hereof are peptidic inhibitors of lentiviruses such as human immunodeficiency virus (HIV), respiratory syncytical virus (RSV), human parainfluenza virus (HPV), measles virus (MEV), and Simian immunodeficiency virus (SIV). Such antifusogenic peptides are derived from C helix of a transmembrane subunit of an envelope fusion protein from a virus of the lentivirus genus and bind to the central coiled coil of the transmembrane subunit of the respective virus.<br>
Especially preferred are HIV-1 antifusogenic peptides, preferably fragments of the C-peptide of gp41. Table 1 describes examples of HIV-1 antifusogenic peptides derived from the C-peptide of gp41. These antifusogenic peptides and fragments thereof are particularly useful in the invention.<br><br>
*   for N-terminally acetylated and/or C-terminally amidated peptide; T-20 (synonymous with DP178) and T-1249 are from human immunodeficiency virus type 1 (HIV-1), T-l 18 is from respiratory syncytial virus (RSV) andT-257 is from measles virus (MV)<br>
" WO 99/59615<br>
21  Rimsky, L.T., et al., J. Virol. 72 (1998) 986-993<br>
3)  Lambert, D.M., Proc. Natl. Acad. Sci. USA 93 (1996) 2186-2191<br><br>
The length of the antifusogenic peptide is not critical. It is however preferred to use lengths of about 10 to 100 amino acids. The length must be sufficient to provide stability against denaturing agents at acidic pH values. The maximum length depends primarily on appropriate handling of the fusion polypeptides during solubilization, cleavage and purification.<br>
In a preferred embodiment of the invention, the repeat peptide according to the invention is linked at its N-terminus with a further peptide of about 4 to 30 amino acids. The purpose of the further peptide is to impart additional properties to the antifusogenic peptide, for example, to improve expression (e.g. an N-terminal fragment of a polypeptide with a high expression rate such as interferon-a-2a, purification (e.g. a His-tag; see, e.g., Zhang, J.-H., et al., Nanjing Daxue Xuebao, Ziran Kexue 36(4) (2000) 515-517) or to allow subsequent N-terminal modification like acetylation or FEGylation.<br>
The further peptide is a short peptidic stretch consisting of at least four amino acids (methionine and three further amino acids for cleavage and/or expression purposes) to about 30 amino acids, preferably from 10 to about 20 amino acids (with regard to the nucleic acid codons level). The length of the further peptide is not critical for the invention. However, it is preferred that the further peptide is very short for improving the yield of the antifusogenic peptide. It is especially preferred that the further peptide consists of an appropriate cleavage site of some amino acids compatible with high level expression or to improve access of cleavage proteases (avoidance of steric hindrance) and/or of some amino acids such as purification or immobilization tags such as a His-tag (for purification means; Hengen, P., Trends Biochem. Sci. 20 (1995) 285-286)) and methionine necessary and encoded by the start codon.<br>
The further peptide according to the invention is used in the fusion peptide also preferably to protect the N-terminus of the antifusogenic peptide during expression, solubilization, purification and peptide modification. Such fusion peptides are especially valuable in a process for the production of N-terminally modified antifusogenic peptides. Such a method involves forming the recombinant polypeptide as a fusion peptide, which fusion part protects the N-terminus. The recombinant fusion peptide can then be reacted with chemical protecting agents to selectively protect reactive side-chain groups. Then the fusion peptide is cleaved<br><br>
with at least one cleavage reagent between the peptide and the fusion part to form an unprotected terminal amino acid reactive group. Thereafter, the unprotected terminal amino acid reactive group can be modified with a chemical modifying agent such as acetic anhydride or acetic N-hydroxysuccinimide ester for N-terminal acetylation. Then the side-chains are selectively deprotected to form a N-terminally modified peptide. Such methods are described, for example, in WO 94/01451; U.S. Patent No. 5,635,371; and U.S. Patent No. 5,656,456.<br>
The further peptide part preferably has such a structure that it facilitates the purification and/or immobilization of the fusion peptide. The further peptide preferably contains for this purpose an "affinity tag" (cf. Pandjaitan, B., et al, Gene 237 (1999) 333-342), such as polyhistidine (about 6 His residues) or the like. In addition, the further peptide contains preferably an N-terminal methionine (encoded by ATG) and C-terminally one or more amino acids encoding for a cleavage site.<br>
Especially preferred further peptides according to the invention are N-terminal fragments of human interferon-a-2a, preferably with an included His-tag (amino acids in standard one-letter code):<br>
MCDLPQTHSLGSR (SEQ ID NO:5) MSDLPQTHSLGSR (SEQ ID NO:6) MSDLPQTHHHHHHSLGSR (SEQ ID NO:7)<br>
According to the invention, the fusion polypeptides contain one or more cleavage sites which can be cleaved enzymatically after solubilization of the inclusion bodies with a specifically cleaving proteinase (restriction proteinase) or by chemical means. The proteinase is selected, taking into consideration the amino acid sequence of the antifusogenic peptide to be produced. Care must be taken that, if possible, the recognition/cleavage sequence of the restriction proteinase does not occur in the antifusogenic peptide, and preferably also not in the further peptide, i.e., it should only occur in the cleavage region (linker region). Suitable specifically cleaving endoproteinases are, for example, Factor Xa, thrombin, subtilisin, BTN variant of subtilisin, ubiquitin protease, rennin, enterokinase, collagenase, precision protease, trypsin, chymotrypsin, endoproteinase Lys-C, kallikrein (Carter, P.: In: Ladisch, M.R.; Willson, R.C.; Painton, C.C.; Builder, S.E., eds., Protein Purification:<br><br><br>
Trypsin is preferably used, which specifically cleaves proteins and peptides at the C-terminal end of arginine and lysine. Such enzymes are known, for example, from porcine, bovine or human pancreas or recombinant yeasts or E.coli (WO 99/10503). Trypsin is particularly suitable for producing the desired polypeptides, if the lysine residues are protected.<br>
The peptide sequence which can be cleaved by an endoproteinase is understood within the sense of the present invention as a short-chain peptide sequence which is preferably composed of 1 to 20 amino acids, preferably 1 to 3 amino acids, and<br><br>
contains a C-terminal cleavage site for the desired endoproteinase. This further peptide preferably additionally contains a combination of several amino acids (first part) between the N-terminal end and the desired endoproteinase recognition sequence, preferably selected from hydrophilic amino acids such as Gly, Thr, Ser, Ala, Pro, Asp, Glu, Arg and Lys. An amino acid stretch in which two to eight of these additional amino acids are the negatively charged amino acids Asp and/or Glu is preferably used as the first part.<br>
Cleavage is also possible using BrCN (chemical cleavage) as long as the antifusogenic peptide does not contain methionine.<br>
In a further preferred embodiment of the invention, the antifusogenic peptide contains a glycine at its C-terminus. This glycine is useful for the purpose of subsequent enzymatic C-terminal amidation (Bradbury, A.F., and Smyth, D.G., Trends Biochem. Sci. 16(1991) 112-115).<br>
Therefore, the elements of the fusion polypeptides preferably are<br>
-	antifusogenic peptide,<br>
-	cleavage site,<br>
-	further peptide for facilitating expression and/or purification,<br>
-	glycine at the C-terminus.<br>
The cleavage sites are necessary for the release of the antifusogenic peptide from the other elements of the fusion polypeptide. Therefore the cleavage sites are located between the antifusogenic peptides in the fusion polypeptide, preferably as an N-terminal protecting group.<br>
According to the invention, an antifusogenic peptide repeat is overexpressed in microorganisms, such as prokaryotes, under conditions whereby insoluble protein inclusion bodies containing said peptide and other polypeptides are formed. The properties according to the invention are already found with a single repeat (two identical antifusogenic peptide sequences) of the antifusogenic peptide. It is, however, also possible to use fusion polypeptides having more than one repeat, i.e., five, ten or up to twenty identical antifusogenic peptide sequences. The fusion peptide to be produced must have a sufficient length for stable expression and for<br><br>
the formation of inclusion bodies in the host cell. Usually, genes encoding at least about 80 amino acids, preferably &gt;100 amino acids, are expressed in a stable manner, are not degraded and are precipitated as inclusion bodies. There is no real upper limit of the repeats. However, to express genes encoding more than 3,000 amino acids is difficult and uncommon in nature.<br>
Escherichia, Salmonella, Streptomyces or Bacillus are for example suitable as prokaryotic host organisms. Yeasts (e.g., Saccharomyces, Pichia, Hansenula, Kluyveromyces, Schizosaccharomyces) are preferred eukaryotic host organisms. For the production of the fusion peptides according to the invention the microorganisms are transformed in the usual manner with the vector which contains the DNA coding for the peptide and subsequently fermented in the usual manner.<br>
Inclusion bodies are found in the cytoplasm and contain the repeat peptide in an aggregated form insoluble in water. Usually, such proteins of inclusion bodies are in a denatured form (e.g., randomly linked disulfide bridges). These inclusion bodies are separated from other cell components, for example by centrifugation after cell lysis.<br>
According to the invention, the inclusion bodies are washed under denaturing conditions at a pH value below 6.5, preferably at about pH 3 to 5. Such denaturing conditions surprisingly do not solubilize the fusion polypeptide to a considerable extent but are able to solubilize a lot of other host cell-derived impurities including polypeptides. Such denaturing agents are well known in the state of the art and are, for example, highly concentrated solutions of guanidinium hydrochloride (e.g. about 6 mol/1) or urea (e.g. about 8 mol/1). The denaturing agent is preferably used as a buffered solution.<br>
After washing, the inclusion bodies are solubilized at alkaline pH values of about pH 9 or higher, preferably without adding of detergents or denaturing agents. It was, in addition, surprisingly found that such alkaline conditions are able to solubilize the fusion polypeptides in a sufficient manner. After solubilization, the fusion polypeptide can be cleaved to recover the antifusogenic peptide.<br><br>
After solubilization, the fusion peptide or antifusogenic peptide can be purified in a simple fashion, for example by size exclusion chromatography, ion exchange chromatrography or reversed phase chromatography.<br>
All further steps in the process for the construction of suitable expression vectors and for the gene expression are state of the art and familiar to a person skilled in the art. Such methods are described for example in Sambrook et al., Molecular Cloning: A Laboratory Manual (1989), Cold Spring Harbor Laboratory Press, New York, USA.<br>
There exist a large number of publications which describe the recombinant production of proteins in microorganisms/prokaryotes via the inclusion bodies route. Examples of such reviews are Misawa, S., et al., Biopolymers 51 (1999) 297-307; Lilie, H., Curr. Opin. Biotechnol. 9 (1998) 497-501; Hockney, R.C., Trends Biotechnol. 12 (1994) 456-463.<br>
The peptides according to the invention are overexpressed in microorganisms/-prokaryotes. Overexpression leads to the formation of inclusion bodies. Methionine encoded by the start codon and mentioned in the examples above is mainly removed during the expression/translation in the host cell. General methods for overexpression of proteins in microorganisms/prokaryotes have been well-known in the state of the art for a long time. Examples of publications in the field are Skelly, J.V., et al., Methods Mol. Biol. 56 (1996) 23-53; Das, A., Methods Enzymol. 182 (1990) 93-112; and Kopetzki, E., et al., Clin. Chem. 40 (1994) 688-704.<br>
Overexpression in prokaryotes means expression using optimized expression cassettes (U.S. Patent No. 6,291,245) with promoters such as the tac or lac promoter (EP-B 0 067 540). Usually, this can be performed by the use of vectors containing chemical inducible promoters or promoters inducible via shift of temperature. One of the useful promoters for E.coli is the temperature-sensitive A.PL promoter (cf. EP-B 0 041 767). A further efficient promoter is the tac promoter (cf. U.S. Patent No. 4,551,433). Such strong regulation signals for prokaryotes such as E.coli usually originate from bacteria-challenging bacteriophages (see Lanzer, M., et al., Proc. Natl. Acad. Sci. USA 85 (1988) 8973-8977; Knaus, R., and Bujard, H., EMBO Journal 7 (1988) 2919-2923; for the A.T7 promoter: Studier, F.W., et al., Methods Enzymol. 185 (1990) 60-89); for the T5 promoter: EP-A 0 186 069; Stuber,<br><br>
D., et al., System for high-level production in Escherichia coli and rapid application to epitope mapping, preparation of antibodies, and structure-function analysis; In: Immunological Methods IV (1990) 121-152).<br>
By the use of such overproducing prokaryotic cell expression systems the peptides according to the invention are produced at levels at least comprising 10% of the total expressed protein of the cell, and typically 30-40%, and occasionally as high as 50%.<br>
"Inclusion bodies" (IBs) as used herein refer to an insoluble form of polypeptides recombinantly produced after overexpression of the encoding nucleic acid in microorganisms/prokaryotes. This phenomenon is widely known in the state of the art and is reviewed, for example, by Misawa S., and Kumagai, I., Biopolymers 51 (1999) 297-307); Guise, A.D., et al., Mol. Biotechnol. 6 (1996) 53-64; and Hockney et al., Trends Biotechnol. 12 (1994) 456-463.<br>
Solubilization of the inclusion bodies is preferably performed by the use of aqueous solutions with pH values of about 9 or higher. Most preferred is a pH value of 10.0 or higher. It is not necessary to add detergents or denaturing agents for solubilization. The optimized pH value can be easily determined according to Example 7. It is obvious that there exists an optimized pH range as strong alkaline conditions might denature the polypeptides. This optimized range is found between pH9andpH 12.<br>
Nucleic acids (DNA) encoding the fusion peptide can be produced according to the methods known in the state of the art. It is further preferred to extend the nucleic acid sequence with additional regulation and transcription elements, in order to optimize the expression in the host cell. A nucleic acid (DNA) that is suitable for the expression can preferably be produced by chemical synthesis. Such processes are,. familiar to a person skilled in the art and are described for example in Beattie, K^L., and Fowler, R. F., Nature 352 (1991) 548-549; EP-B 0 424 990; Itakura, K., et al., Science 198 (1977) 1056-1063. It may also be expedient to modify the nucleic acid sequence of the peptides according to the invention.<br><br>
Such modifications are, for example:<br>
-	modification of the nucleic acid sequence in order to introduce various recognition sequences of restriction enzymes to facilitate the steps of ligation, cloning and mutagenesis;<br>
-	modification of the nucleic acid sequence to incorporate preferred codons for the host cell;<br>
-	extension of the nucleic acid sequence with additional regulation and transcription elements in order to optimize gene expression in the host cell.<br>
The following examples, references, Fig. 1 and the sequence listing are provided to aid the understanding of the present invention, the true scope of which is set forth in the appended claims. It is understood that modifications can be made in the procedures set forth without departing from the spirit of the invention.<br>
Description of the Figure:<br>
Figure 1	shows the expression vector pBRori-URA3-LACI-RFN-Edel.<br>
Starting material<br>
The E.coli host/vector system (E.coli host strain and basic vector) employed for the expression of the multimeric peptide precursor proteins according to the invention is described in US Patent No. 6,291,245.<br>
General Methods<br>
Recombinant DNA technique<br>
Standard methods were used to manipulate DNA as described in Sambrook, J., et al., Molecular cloning: A laboratory manual; Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York, 1989. The molecular biological reagents were used according to the manufacturer's instructions.<br><br>
Protein determination<br>
The protein concentration for the multimeric T-repeat fusion protein was determined by determining the optical density (OD) at 280 nm, using the molar extinction coefficient calculated on the basis of the amino acid sequence [T-repeat: £ = 148680 cmVmol; IFN-a-2a: e = 18020 cm:/mol; F9a = 44650 cm2/mol].<br>
Example 1<br>
Synthesis of the T-repeat fusion gene<br>
1.1	Gene design of the T-repeat fusion gene<br>
The artificial T-repeat fusion gene encodes a fusion protein of 217 amino acids having a molecular weight of 27,242 D. The fusion protein is composed of the N-terminal 13 amino acids of human interferon-a-2a (MCDLPQTHSLGSR (SEQ ID NO:5); carrier peptide) and 5 copies of the inhibitory HIV peptide T-1357 (WQEWEQKITALLEQAQIQQEKNEYELQKLDKWASLWEWF (SEQ ID NO:l), target peptide), which are connected via trypsin-cleavable peptide linkers (GR).<br>
In order to insert the T-repeat structural gene into the E.coli expression plasmid pBRori-URA3-LacI-RFN-Edel (production and description: see Example 2), a synthetic ribosomal RBSII binding site and a singular EcoRI cleavage site were inserted upstream at the 5' end and a singular Celll restriction endonuclease cleavage site upstream at the 3' end.<br>
1.2	Gene synthesis of the T-repeat fusion gene<br>
The RBSII T-repeat gene which is about 690 bp long and flanked by a singular EcoRI and Celll restriction endonuclease cleavage site was prepared from oligonucleotides by chemical synthesis. The double-stranded RBSII T-repeat gene was assembled by annealing and ligation of the oligonucleotides and subsequently cloned as an EcoRI/Celll fragment of a length of 691 bp into an E.coli plasmid. The desired plasmid was designated pT-repeat. The predetermined DNA sequence of the cloned RBSII T-repeat gene was confirmed by DNA sequencing.<br><br>
Example 2<br>
Construction of the E.coli expression plasmids<br>
2.1       Construction of the starting plasmid pBRori-URA3-LacI-RFN-Edel<br>
The plasmid pBRori-URA3-Lad-RFN-Edel is a vector for the expression of interferon-a-2a (IFN-a-2a) in E.coli. It is based on the IFN-a-2b expression plasmid OripBR-URA3-EK-IFN (U.S. Patent No. 6,291,245). pBRori-URA3-Lad-RFN-Edel differs from OripBR-URA3-EK-IFN by an additionally present lad repressor gene and an !FN-a-2a gene instead of an IFN-a-2b gene. IFN-oc-2a and IFN-a-2b differ only by one amino acid at position 21 (Lys21Arg exchange). The lad repressor gene comes from plasmid pUHAl (Stuber, D., et al., System for high-level production in Escherichia coli and rapid application to epitope mapping, preparation of antibodies, and structure-function analysis; In: Immunological Methods IV (1990) 121-152). It was amplified by polymerase chain reaction (PCR) according to the method described by Mullis, K.B., and Faloona, F.A., in Methods Enzymol. 155 (1987) 335-350, using the primers Nl (SEQ ID NO:12) and N2 (SEQ IDNO:13)<br>
Not I<br>
Nl:       5'-AAAAAAGCGGCCGCGACAATTCGCGCGCGAAGGCG-3'<br>
Not I<br>
N2:       5'-AAAAAAGCGGCCGCTCACTGCCCGCTTTCCAGTCGG-3'<br>
and subsequently ligated as a NotI fragment of a length of about 1210 bp into the singular NotI cleavage site of OripBR-URA3-EK-IFN.<br>
2.2       Construction of the expression vector pBRori-URA3-LacI-T-repeat<br>
The T-repeat gene was isolated as an EcoRI/Celll fragment of a length of about 690 bp from the plasmid pT-repeat (see Example 1.2) and subsequently ligated into the ca. 3.1 kbp long pBRori-URA3-LacI-RFN vector fragment digested with EcoRI and Celll. The desired plasmid pBRori-URA3-LacI-T-repeat was identified by restriction mapping and subcloned T-repeat gene was verified again by DNA sequencing.<br><br>
Example 3<br>
Expression of the T-repeat gene in E.coli<br>
For the expression of the fusion gene according to the invention there was employed an E.coli host/vector system which enables an antibiotic-free plasmid selection by complementation of an E.coli auxotrophy (PyrF) (U.S. Patent No. 6,291,245).<br>
3.1	Transformation and cell culturing by complementation of a pyrF<br>
auxotrophy in selective medium<br>
In order to express the T-repeat gene, an E.coli K12 strain [designated UT5600 (ApyrF)] was transformed with the expression plasmid pBRori-URA3-Lad-T-repeat described in Example 2.2. The transformed UT5600(ApyrF)/pBRori-URA3-LacI-T-repeat cells were first grown at 37°C on agar plates and subsequently in a shaking culture in M9 minimal medium containing 0.5% casamino acids (l)ifco) up to an optical density at 550 nm (OD550) of 0.6-0.9 and subsequently induced with IPTG (l-5mmol/l final concentration). After an induction phase of 4-16 hours at 37°C, the cells were harvested by centrifugation, washed with 50 mmol/l potassium phosphate buffer, pH 6.5, and stored at -20°C until further processing.<br>
3.2	Expression analysis<br>
For expression analysis cell pellets from 3 OD550nm units (1 OD55(,nm = 1 ml cell suspension with an OD at 550 nm of 1) of centrifuged culture medium were resuspended in 0.25 ml 10 mmol/l potassium phosphate buffer, pH 6.5, and the cells were lysed by ultrasonic treatment (two pulses of 30 s at 50% intensity). The insoluble cell components were sedimented (14,000 rpm, 5 min) and the supernatant was admixed with 1/5 volumes (vol) 5xSDS sample buffer (lxSDS sample buffer: 50 mmol/l Tris-HCl, pH 6.8, 1% SDS, 50 mmol/l DTT, 10% glycerol, 0.001% bromophenol blue). The insoluble cell debris fraction (pellet) was resuspended in 0.3 ml lxSDS sample buffer, the samples were incubated for 5 min at 95°C and again centrifuged. Subsequently, the proteins were separated by SDS polyacrylamide gel electrophoresis (PAGE) (Laemmli, U.K., Nature 227 (1970) 680-685) and stained with Coomassie Brilliant Blue R dye.<br><br>
The synthesized T-repeat fusion protein was homogeneous and was found exclusively in the insoluble cell debris fraction in the form of insoluble protein aggregates, the so-called inclusion bodies (IBs). The expression yield was comparable within the scope of the measurement accuracy in all clones and was between 30-60% relative to the total E.coli protein.<br>
Example 4<br>
10 1 high cell density fermentations of E.coli for the recombinant production of<br>
T-repeat<br>
Preculture<br>
In order to prepare the preculture, 300 ml M9plus medium (M9 medium supplemented with 0.5% casamino acids and 0.9 g/1 Trp, Fro and Leu each) was inocculated with 1 ml of a glycerol stock of E.coli UT5600ApyrF pBRori-URA3-lacI-T-repeat in a 1000 ml Erlenmeyer flask. The culture was incubated for about 6 hours at 37°C on an excenter shaker with 150 rpm until an ODs78nm of 3.0 was reached.<br>
101 Fed-batch main fermentation<br>
At the beginning of fermentation, the preculture was transferred into the 10 liter fermenter. The main culture was grown in defined M9 salt medium containing 1.4% glycerol instead of glucose, 2% casamino acids and 0.1% of the amino acids Trp, Leu and Pro each, up to an OD578nmof 20. Subsequently, feeding of the culture with a glycerol yeast dosage (stock solution: 30% yeast extract and 33% glycerol) was started, the flow rate of which was varied between 0.8 and 3.5 ml/min depending on the development of the pH value of the culture, thereby avoiding any further addition of correction aids (H3PO4, KOH). The pH was maintained at pH 7.0, the p02 value was held at 50% by controlling the rpm.<br>
At an OD578nm of 70 1.5 mmol/1 IPTG was added and the gene/protein expression was induced. The total fermentation took about 36 hours and was terminated at an<br>
OD578nm Of 160-180.<br><br>
Harvesting the biomass<br>
The content of the fermenter was centrifuged with a flowthrough centrifuge (13,000 rpm, 13 1/h) and the harvested biomass was stored at -20°C until further processing.<br>
Example 5<br>
5.1	Standard methods: cell lysis and preparation of IBs<br>
200 g E.coli cells (wet weight) were suspended in 11 0.1 mol/1 Tris-HCl, pH 7.0, at 0°C, 300 mg lysozyme were added and incubated for 20 minutes at 0°C. Subsequently, the cells were completely lysed mechanically by means of high pressure dispersion and the DNA was digested for 30 minutes at 25°C by adding 2 ml 1 mol/1 MgCb and 10 mg DNAse. Thereafter, 500 ml 60 mmol/I EDTA, 6% Triton X-100 and 1.5 mol/1 NaCl, pH 7.0 were admixed with the lysis solution and incubated for another 30 minutes at 0°C. Subsequently, the insoluble components (cell debris and IBs) were sedimented by centrifugation.<br>
The pellet was suspended in 11 0.1 mol/1 Tris-HCl, 20 mmol/1 EDTA, pH 6.5, incubated for 30 minutes at 25° and the IB preparation was isolated by centrifugation.<br>
5.2	Methods according to the invention: cell lysis and preparation of IBs<br>
200 g E.coli cells (wet weight) were suspended in 1 1 0.1 mol/1 potassium phosphate buffer, pH 5.0, 300 mg lysozyme were added and incubated for 20 minutes at 0°C. Subsequently, the cells were completely lysed mechanically by means of high pressure dispersion, the DNA was digested for 30 minutes at 25°C by adding 2 ml 1 mol/1 MgCl: and 10 mg DNAse and the insoluble components (cell debris and IBs) were sedimented by centrifugation.<br>
Thereafter, the pellet was washed twice with 0.1 mol/1 potassium phosphate buffer, pH 5.0. To this end, the pellet was suspended twice in 1 1 0.1 mol/1 potassium buffer, pH 5.5, incubated at 25°C for 30 minutes while stirring and isolated by centrifugation.<br><br>
5.3       Methods according to the invention: cell lysis and preparation of high purity IBs<br>
200 g E.coli cells (wet weight) were suspended in 1 1 0.1 mol/1 potassium phosphate buffer, pH 5.0, at 0°C, 300 mg lysozyme were added and incubated for 20 minutes at 0°C. Subsequently, the cells were completely lysed mechanically by means of high pressure dispersion, the DNA was digested for 30 minutes at 25°C by adding 2 ml 1 mol/1 MgCl2 and 10 mg DNAse and the insoluble components (cell debris and IBs) were sedimented by centrifugation.<br>
Thereafter, the pellet was washed twice with 5.5 mol/1 guanidinium hydrochloride (and 8 mol/1 urea, respectively) and 10 mmol/l EDTA, pH 3.0). To this end, the pellet was suspended twice in 5.5 mol/1 guanidinium hydrochloride (and 8 mol/1 urea, respectively) and 10 mmol/l EDTA, pH 3.0, incubated at 25°C for 10-30 minutes while stirring and isolated by centrifugation.<br>
Thereafter, the pellet was washed two to three times with 0.1 mol/1 potassium phosphate buffer, pH 5.0. To this end, the pellet was suspended two to three times in 11 0.1 mol/1 potassium phosphate buffer, pH 5.5, incubated at 25°C for 30 minutes while stirring and isolated by centrifugation.<br>
Example 6<br>
Solubility of T-repeat IBs in comparison with "standard" IBs (IFN-ot-2a and F9a)<br>
in the presence of denaturing agents in the acidic range<br>
The IBs to be tested [IFN-oc-2a: prepared as described in Examples 2.1, 3.1, 4 and 5.1, whereby however UT5600(ApyrF)/pBRori-URA3-LacI-RFN-Edel cells were used instead of UT5600(ApyrF)/pBRori-URA3-LacI-T-repeat cells; F9a: prepared as described in WO 97/47737; T-repeat: prepared as described in Examples 2.2, 3.1, 4 and 5.2] were resuspended at a concentration of 1 g/20 ml each in<br>
a)	5.5 mol/1 guanidinium hydrochloride and 10 mmol/l EDTA, pH 3.0 and<br>
b)	8 mol/1 urea and 10 mmol/l EDTA, pH 3.0<br>
at room temperature while stirring.<br><br>
After 12-24 hours, a 200 ul sample was taken in each case, the insoluble components were sedimented by centrifugation (Eppendorf 5415 centrifuge, 14.000 rpm, 10 min) and 80 ul supernatant were mixed with 20 ul 5xSDS sample buffer (lxSDS sample buffer: 50 mmol/1 Tris-HCl, pH 6.8, 1% SDS, 50 mmol/1 DTT, 10% glycerol, 0.001% bromophenol blue). The supernatants containing 5.5 mol/l guanidinium hydrochloride were dialyzed against 8 mol/l urea prior to SDS PAGE. The insoluble pellet was washed once with 200 ul potassium phosphate buffer, pH 5.0 and subsequently resuspended in 250 ul lxSDS sample buffer. All samples were incubated at 95°C for 5 minutes before the proteins were separated by means of SDS polyacrylamide gel electrophoresis (PAGE) and stained with Coomassie Brilliant Blue R dye. Thereafter, the relevant gel bands were analyzed densitometrically for the determination of the polypeptide content (Table 3).<br><br><br><br><br><br><br>
8.2       Methods according to the invention: solubilization of IBs in aqueous slightly alkaline buffers<br>
10 g IB pellet (wet weight) were suspended in 200 ml 50 mmol/1 Tris-HCl buffer, pH 9 (and 50 mmol/1 NaHCO.^ buffer, pH 9-10, respectively) by being stirred for 4-16 hours at 25°C. The insoluble components were separated by centrifugation and the clear supernatant was further processed [optionally after derivatization of the primary amino groups of the T-repeat fusion protein (e-amino group of lysine) with citraconic acid anhydride] by enzymatic cleavage with trypsin.<br><br>
List of References<br>
Andersson, L., et al., Biopolymers 55 (2000) 227-250<br>
Arthur, L.O.,etal., Science 258 (1992) 1935-1938<br>
Beattie, K. L., and Fowler, R. F., Nature 352 (1991) 548-549<br>
Bradbury, A.F., and Smyth, D.G., Trends Biochem. Sci. 16 (1991) 112-115<br>
Bullough, P.A., et al., Nature 371 (1994) 37-43<br>
Cao, J., et al., J. Virol. 67 (1993) 2747-2755<br>
Carr, CM., and Kim, P.S., Cell 73 (1993) 823-832<br>
Carter, P.: In: Ladisch, M.R.; Willson, R.C.; Painton, C.C.; Builder, S.E., eds., Protein Purification: From Molecular Mechanisms to Large-Scale Processes; ACS Symposium Series No. 427, American Chemical Society, pp. 181-193(1990)<br>
Chan, D.C., et al., Proc. Natl. Acad. Sci. USA 95 (1998) 15613-15617<br>
Chan, D.C., et al., Cell 89 (1997) 263-273<br>
Chan, D.C., et al., Cell 93 (1998) 681-684<br>
Chen, C.H., et al., J. Virol. 69 (1995) 3771-3777<br>
Das, A., Methods Enzymol. 182 (1990) 93-112<br>
EP-A 0 186 069<br>
EP-A 0 467 839<br>
EP-B 0 424 990<br>
EP-B 0 041 767<br>
EP-B 0 067 540<br>
Gallaher, W.R., et al., Aids Res. Hum. Retrovirus 5 (1989) 431-440<br>
Guise, A.D., et al., Mol. Biotechnol. 6 (1996) 53-64<br>
Hammarskjold, M.-L., et al., Biochim. Biophys. Acta 989 (1989) 269-280<br>
Hengen, P., Trends Biochem. Sci. 20 (1995) 285-286<br>
Hockney, R.C., Trends Biotechnol. 12 (1994) 456-463<br>
Itakura, K., et al., Science 198 (1977) 1056-1063<br>
Jakubke, H.D., Peptide, Spektrum Akad. Verlag, Heidelberg (1996), p. 198<br>
Jones, J.H., J. Pept. Sci. 6 (2000) 201-207<br>
Judice, J. K., et al., Proc. Natl. Acad. Sci. USA 94 (1997) 13426-13430<br>
Kilby, J.M., et al., Nature Medicine 4 (1998) 1302-1307<br>
Knaus, R., and Bujard, H., EMBO Journal 7 (1988) 2919-2923<br>
Kopetzki, E., et al., Clin. Chem. 40 (1994) 688-704<br>
Laemmli, U.K., Nature 227 (1970) 680-685<br><br>
Lambert, D.M., et al., Proc. Natl. Acad. Sci. USA 93 (1996) 2186-2191<br>
Lanzer, M., et al., Proc. Natl. Acad. Sci. USA 85 (1988) 8973-8977<br>
Lawless, M.K., etal., Biochemistry 35 (1996) 13697-13708<br>
Lepage, P., et al., Analytical Biochemistry 213 (1993) 40-48<br>
Lilie, H., Curr. Opin. Biotechnoi. 9 (1998) 497-501<br>
Maddon, P.J., et al., Cell 47 (1986) 333-348<br>
Malashkevich, V.N., et al., Proc. Natl. Acad. Sci. USA 95 (1998) 9134-9139<br>
McDougal, J.S., et al., Science 231 (1986) 382-385<br>
Mergler, M., et al., Tetrahedron Letters 29 (1988) 4005-4008 and 4009-4012<br>
Misawa, S., et al., Biopolymers 51 (1999) 297-307<br>
Mullis, K.B. und Faloona, F.A., Methods Enzymol. 155 (1987) 335-350<br>
Ningyi, L., et al., Gaojishu Tongxun 10 (2000) 28-31<br>
Pandjaitan, B., et al., Gene 237 (1999) 333-342<br>
Parks, T.D., et al., Anal. Biochem. 216 (1994) 413-417<br>
Pohlner, J., et al., Nature 325 (1987) 458-462<br>
Rimsky, L.T., et al., J. Virol. 72 (1998) 986-993<br>
Root, M.J., et al., Science 291 (2001) 884-888<br>
Sambrook et al., Molecular Cloning: A Laboratory Manual (1989), Cold Spring<br>
Harbor Laboratory Press, New York, USA Skelly, J.V., et al., Methods Mol. Biol. 56 (1996) 23-53 Stuber, D., et al., System for high-level production in Escherichia coli and rapid<br>
application to epitope mapping, preparation of antibodies, and<br>
structure-function analysis; In: Immunological Methods IV (1990)<br>
121-152 Studier, F.W., et al., Methods Enzymol. 185 (1990) 60-89 Tan, K., et al., Proc. Natl. Acad. Sci. USA 94 (1997) 12303-12308 Turner, B.G., and Summers, M.F., J. Mol. Biol. 285 (1999) 1-32 U.S. Patent No. 4,551,433 U.S. Patent No. 5,464,933 U.S. Patent No. 5,635,371 U.S. Patent No. 5,656,456 U.S. Patent No. 5,656,480 U.S. Patent No. 6,013,263 U.S. Patent No. 6,017,536 U.S. Patent No. 6,020,459 U.S. Patent No. 6,060,065<br><br>
U.S. Patent No. 6,093,794<br>
U.S. Patent No. 6,291,245<br>
Uhlen, M, and Moks, T., Methods Enzymol. 185 (1990) 129-143<br>
Weissenhorn, W., et al., Nature 387 (1997) 426-430<br>
Wild, C.T., et al., Proc. Natl. Acad. Sci. USA 91 (1994) 9770-9774<br>
Wilson, I.A., et al., Nature 289 (1981) 366-373<br>
WO 00/69902<br>
WO 02/103026<br>
WO 92/01707<br>
WO 94/01451<br>
WO 95/03405<br>
WO 96/19495<br>
WO 96/40191<br>
WO 99/10503<br>
WO 99/48513<br>
WO 99/59615<br>
Zhang, J.-H., et al., Nanjing Daxue Xuebao, Ziran Kexue 36(4) (2000) 515-517<br>
Zhao, X., et al., Proc. Natl. Acad. Sci. USA 97 (2000) 14172-14177<br><br><br>
WE CLAIM :<br>
1.	Process for the recombinant production of an antifusogenic peptide by expression of a nucleic acid encoding said antifusogenic peptide as repeat peptide in a microbial host cell, isolation of inclusion bodies containing said repeat peptide, solubilization of the inclusion bodies, and isolation of the antifusogenic peptide after cleavage, characterized by washing the inclusion bodies with a denaturing agent at a pH value of or below pH 6.5, solubilizing said inclusion bodies containing the repeat peptide at a pH value of at least pH 9 and cleaving said repeat peptide to obtain said antifusogenic peptide.<br>
2.	Process according to claim 1, wherein washing is performed between pH 3 and 5.<br>
3.	Process according to claim 1 or 2, wherein said repeat peptide is cleaved during or after solubilization of said inclusion bodies.<br>
4.	A nucleic acid encoding a fusion polypeptide consisting of (in N-terminal to C-terminal direction).<br><br>
a)	an antifusogenic peptide as repeat peptide of at least two identical antifusogenic peptide sequences;<br>
b)	a cleavage site located between the antifusogenic peptides.<br><br>
5.	A nucleic acid according to claim 4, wherein the antifusogenic peptide consists of 10 to 100 amino acids.<br>
6.	A nucleic acid according to claim 4 or 5, wherein the repeat consists of 2 to 20 identical antifusogenic peptide sequences.<br><br>
7. A preparation of inclusion bodies containing a fusion polypeptide consisting of (in N-terminal to C-terminal direction)<br>
an antifusogenic peptide as repeat peptide of a length of about 10 to 100 amino acids;<br>
a cleavage site located between the antifusogenic peptides.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ0LWNoZS0yMDAzIGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">944-che-2003 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ0LWNoZS0yMDAzIGFic3RyY2F0IGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">944-che-2003 abstrcat duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ0LWNoZS0yMDAzIGNsYWltcyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">944-che-2003 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ0LWNoZS0yMDAzIGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">944-che-2003 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ0LWNoZS0yMDAzIGNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">944-che-2003 correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ0LWNoZS0yMDAzIGNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">944-che-2003 correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ0LWNoZS0yMDAzIGRlc2NyaXB0aW9uIChjb21wbGV0ZSkgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">944-che-2003 description (complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ0LWNoZS0yMDAzIGRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">944-che-2003 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ0LWNoZS0yMDAzIGRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">944-che-2003 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ0LWNoZS0yMDAzIGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">944-che-2003 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ0LWNoZS0yMDAzIGZvcm0tMTkucGRm" target="_blank" style="word-wrap:break-word;">944-che-2003 form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ0LWNoZS0yMDAzIGZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">944-che-2003 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ0LWNoZS0yMDAzIGZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">944-che-2003 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ0LWNoZS0yMDAzIGZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">944-che-2003 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ0LWNoZS0yMDAzIG90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">944-che-2003 others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ0LWNoZS0yMDAzIHBldGl0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">944-che-2003 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="224896-buble-unit.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="224898-single-chamber-compact-fuel-processor.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>224897</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>944/CHE/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>49/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>05-Dec-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>24-Oct-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>18-Nov-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>F. HOFFMANN - LA ROCHE AG</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>124 GRENZACHERSTRASSE, CH-4070 BASLE</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ALEXANDRA KACZMAREK</td>
											<td>HERZOGSTANDSTRASSE 3, D-82377 PENZBERB</td>
										</tr>
										<tr>
											<td>2</td>
											<td>ERHARD KOPETZKI</td>
											<td>KASTNERHOFSTRASE 21, D-82377 PENZBERG,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>CHRISTIAN SCHANTZ</td>
											<td>WIPPENHAUSER STRASSE 34, D-85402 THALHAUSEN,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>STEFAN SEEBER</td>
											<td>SAALANGER 34, D-82377 PENZBERG,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A61K038/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>02025618.6</td>
									<td>2002-11-19</td>
								    <td>EUROPEAN UNION</td>
								</tr>
								<tr>
									<td>2</td>
									<td>03000988.0</td>
									<td>2003-01-17</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/224897-methods-for-the-recombinant-production-of-antifusogenic-peptides by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:14:25 GMT -->
</html>
