<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/224830-an-aspergillus-which-do-not-produce-proteases by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:34:25 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 224830:AN ASPERGILLUS WHICH DO NOT PRODUCE PROTEASES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">AN ASPERGILLUS WHICH DO NOT PRODUCE PROTEASES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>ABSTRACT &quot; AN ASPERGILLUS WHICH DO NOT PRODUCE PROTEASES&quot; 2287/MAS/96 An Aspergillus, wherein the areA gene by recombinant DNA technology has been modified in a way by which it cannot be expressed in a way providing for a functional AreA activator, and wherein the genes encoding for the extracellular proteases PepC and/or PepE have been inactivated in a manner whereby they are not expressed to produce functional proteases.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Title:	Novel Microorganisms<br>
The present invention relates to fungi, which do not produce proteases. The fungi of the invention are useful as hosts for the production of proteins susceptible to proteolytic degradation by the proteases usually produced, and the invention consequently encompasses processes for the production of proteins of interest in high yields by using the fungi of the invention. The invention also comprises methods for producing such fungi and DNA Constructs to be used in these methods.<br><br>
Fungi, and especially filamentous fungi, are widely used commercially because of their ability to secrete remarkably high levels of proteins<br>
Among the filamentous fungi species belonging to the genus Aspergillus have a long history of commercial use for the production of endogenous and lately also heterologous proteins.<br>
One disadvantage with most microorganisms used for the production of proteins is the inherent production of proteases which may subject a protein product of interest to degradation due to proteolysis.<br>
Various ways of avoiding this have been envisaged. Among other solutions it has been suggested to delete or disrupt the genes encoding the various proteases. Unfortunately, the fungi produce a high number of proteases making such a solution more or less unrealistic.<br>
A need is therefore persisting for strains of filamentous fungi exhibiting no or very low levels of protease production.<br><br>
For a number of years it has been known that the regulatory gene areA which mediates nitrogen metabolite repression in A. nidulans influences the production of extracellular proteases (Arst &amp; Cove, molec. gen. Genet. 126,(1973) 111-141).<br>
The areA gene from A. nidulans has been cloned (Caddick et al., EMBO Journal 5, (1986) 1087-1090) and various modifications made to it to evaluate functions of different regions in the activator protein encoded by this gene (Stankovitch et al. Mol. Microbiol. 7, (1993) 81-87). Furthermore the gene coding the corresponding function in A. Jumigatus apparently has been cloned recently (Hensel et al. 2nd European Conference on Fungal Genetics, April 28 to May 1,1994, Book of Abstracts, Ell).<br>
From the literature a single use is also known of a strain of A. nidulans of genotype argB areA 1 as a host for the production of t-PA (Upshall et al. Biotechnology 5, (1987) 1301-1304). In this example only the argB genotype is used as a selection marker through its arginine prototrophy, while the areA genotype is simply a coincidence.<br>
International Patent Publication No. WO 95/35385 discloses the deletion of the areA gene as a means for reducing the protease level in filamentous fungi.<br>
Apart from the extracellular proteases, fungi also produce a number of intracellular proteases (also called endoplasmic).<br>
Among these a serine protease of the subtilisin type produced by A. niger and designated PepC has been described, the gene expressing it cloned, and a deletion mutant described in EP 574 347 and in Frederick et al, Gene, 125 57-64 (1993)<br>
A further such protease of the aspartic type designated PepE has been disÂ¬closed in Jarai et al, Gene, 145 171-178 (1994). the article discloses the cloning and characterisation of the pepE gene and speculates about the regulation of the pepE and pepC genes.<br>
The present invention has as an object the alleviation of the need for protease free filamentous fungi.<br><br><br>
The present invention consequently relates to fungi, wherein the areA,pepC, and/or pepE genes by recombinant DNA technology have been modified such that they cannot be expressed in a way providing for a functional AreA activator and functional PepC and/or PepE proteases.<br>
The invention furthermore relates to methods for producing such fungi, obtained by deletion of the areA, pepC, and/or pepE genes.<br>
This may be obtained through a method comprising<br>
i)	cloning of the areA,pepC, and/or pepE genes from a fungus of interest,<br>
ii)	producing DNA constructs each comprising one among the areA gene, the<br>
pepC gene, and the pepE gene, wherein an internal part has been substituted,<br>
deleted, or extra DNA has been inserted,<br>
iii)	transforming said fungus with the constructs, and<br>
iv)	isolating transformants which are areA',pepC, and/orpepK.<br>
The information obtained from the above mentioned cloning of the areA, pepC, and/or pepE genes may also be used in connection with the well-known anti-sense technology, to construct an expression plasmid giving rise to synthesis of a RNA molecules complementary to the mRNA transcribed from the areA, pepC, and/or pepE genes, and to transform the fungus of interest therewith.<br>
The invention furthermore relates to DNA constructs intended for use in the above mentioned methods.<br>
Furthermore the invention relates to methods of producing a desired protein or gene product, especially secreted proteins, whereby a fungal host modified and optionally transformed with a DNA construct comprising at least a DNA sequence coding for the protein or gene product of interest, is cultivated in a suitable growth medium at appropriate conditions and the desired gene product is recovered and purified.<br>
When working with the invention it was surprisingly found that the fungi of the invention produces such secreted proteins in a much improved yield.<br><br>
It was also surprisingly found that the only nitrogen source capable of providing good growth of the A. oryzae areA' strains was glutamine.<br>
The invention furthermore relates to protein products produced by the above methods.<br>
Also the invention relates to a DNA sequence coding for the pepC gene from A. oryzae (SEQ ID No. 1) or functional alleles thereof.<br>
The invention also covers a PepC protease from A. oryzae (SEQ ID No. 2), and processes for the production of the PepC protease comprising transforming a suitable host with a DNA construct comprising a DNA sequence coding for the PepC protease, selecting a transformant capable of producing said PepC protease, cultivating said transformant in an appropriate growth medium and recovering said PepC protease from said culture.<br>
Furthermore the invention relates to a DNA sequence coding for the pepE gene from A. oryzae (SEQ ID No. 3) or functional alleles thereof.<br>
Also, the invention relates to a PepE protease from A. oryzae (SEQ ID No. 4), and processes for the production of the PepE protease comprising transforming a suitable host with a DNA construct comprising a DNA sequence coding therefore, selecting a transformant capable of producing said PepE protease, cultivating said transformant in an appropriate growth medium and recovering said PepE protease from said culture.<br>
According to these aspects said host is preferably a fungus, according to the invention, especially A. oryzae, and wherein said DNA construct provides for an extra copy of the gene encoding either said PepC or PepE protease.<br><br>
The invention is described in further detail in the following parts of the specification with reference to the Examples and the drawing, wherein<br><br>
Fig. 1 shows the steps involved in the construction of HowBlOl,<br>
Fig. 2 shows the construction of pToC345,<br>
Fig. 3 shows the steps involved in the construction of pToC315.<br>
Fig. 4 diagrammatically shows a two step gene deletion of thepyrG gene.<br>
Fig. 5 shows the construction of pJaL235,<br>
Fig. 6 shows the steps involved in the construction of pJaL335,<br>
Fig. 7 shows the steps involved in the construction of pJaL363,<br>
Fig. 8 shows the steps involved in the construction of pJaLz,<br>
Fig. 9 shows the construction of pSK5 and pSK9,<br>
Figs. 10a and 10b show the steps involved in the construction of pToC243 and pToC266,<br>
Fig. 11 shows the steps involved in the construction of pMT1606,<br>
Fig. 12 shows the construction of pToC56,<br>
Figs. 13a and 13b show the steps involved in the construction of pJaL368, and<br>
Figs. 14a and 14b show the construction of pToC338.<br>
In the present specification the following definitions are used:<br>
The expression areAD means a strain in which the areA gene is deleted. Similar notations are used for strains, wherein \hepepC, and/or pepE genes are deleted.<br>
The expression areA' means a strain which does not produce a functional AreA activator. The term "loss of function" is also often used for this. Similar notations used for strains, which do not produce functional PepC, and/or PepE protease(s).<br>
The expression "anti-sense technology" describes methods such as disclosed in US Patent No. 5,190,931.<br><br>
As indicated the present invention relates in its first aspect to fungi, wherein the areA gene by recombinant DNA technology has been modified in a way by which it cannot be expressed in a way providing for a functional AreA activator, and wherein the genes encoding for the extracellular proteases PepC and/or PepE has been inactivated in a manner whereby they are not expressed to produce functional proteases.<br>
This object may specifically be obtained by deletion or disruption of the areA, pepC, and/or pepE genes.<br>
The cloning of the areA, pepC, and/or pepE genes are described in the Examples.<br>
AreA homologs from other fungi could be cloned either by cross hybridization with one of the already known genes or by complementation of areA mutants; e.g. A. nidulans areA-18 or the A. oryzae areA deleted strain described in this application.<br>
Methods for deleting or disrupting a gene are specifically described in WO 90/00192 (Genencor).<br>
Methods for substituting DNA in a gene are also generally known, and can be accomplished by substituting one or more continuous parts of the gene, but it may also be obtained by site directed mutagenesis generating a DNA sequence encoding a AreA activator variant that is not functional.<br>
Another method by which such an object may be obtained is by using anti-sense technology.<br>
The anti-sense technology and how to employ it is described in detail in the aforementioned US Patent No. 5,190,931 (University of New York).<br>
A further method of obtaining said inactivation is by inserting extra DNA internally in the areA gene, thereby giving rise to the expression of a dysfunctional activator protein.<br>
In connection with this method information provided by the cloning can be used to make DNA constructs that can be integrated into the areA gene, and even replace it with another gene, such as the pyrG gene.<br>
7<br><br>
A further method of avoiding the presence of the areA activator is by interfering with the regulation of the expression signals regulating the expression of the areA gene itself.<br>
The principles described above apply equally to the/jepC, and/or pepE genes.<br>
According to the invention the fungus preferably belongs to a genus selected from the group comprising Aspergillus, Trichoderma, Humicola, Candida, Acremonium, Fusarium, and Penicillium.<br>
Among these genera species selected from the group comprising A. otyzae, A. niger, A. awamori, A. phoenicis, A. japonicus, A, foetidus, A. nidulans, T. reesei, T. hanianum, H. insolens, H. lanuginosa, F. graminearum, F. solani, P. chrysogenum, and others are preferred.<br>
As indicated the invention also is meant to encompass the method for<br>
producing the fungi of the first aspect of the invention, and wherein said inactivation has<br>
been obtained by deletion of the areA,pepC, and/or pepE genes, which method comprises<br>
i)	cloning of the areA, pepC, and/or pepE genes from a fungus of interest,<br>
ii)	producing DNA constructs each comprising one among the areA gene, the<br>
pepC gene, and/or the pepE gene, wherein an internal part has been<br>
substituted, deleted, or extra DNA has been inserted,<br>
iii)	transforming said fungus with the constructs, and<br>
iv)	isolating transformants which are areA',pepC, and/or pepE.<br>
Since it is believed that the maturation of the PepC protease is controlled by the PepE protease the invention also comprises a method for producing a fungus of the invention, wherein said inactivation has been obtained by deletion of the areA and pepE genes, which method comprises<br>
i)	cloning of the areA and pepE genes from a fungus of interest,<br>
ii)	producing DNA constructs each comprising one among the areA gene and the<br>
pepE gene, wherein an internal part has been substituted, deleted, or extra<br>
DNA has been inserted,<br>
iii)	transforming said fungus with the constructs, and<br>
8<br><br>
iv)	isolating transformants which are areA\ andpepE.<br>
Also included is the method for producing the fungi, wherein the inactivation<br>
has been obtained by using anti-sense technology. Such a method comprising<br>
i)	construction of expression plasmids, each of which give rise to synthesis of an<br>
RNA molecule complementary to the mRNA transcribed from the areA gene,<br>
the pepC gene, and/or the pepE gene,<br>
ii)	transformation of die host fungus with said expression plasmids and a suitable<br>
marker, either on separate plasmids or on the same plasmid,<br>
iii)	selection of transformants using said marker, and<br>
iv)	screening selected transformants for strains exhibiting a reduction in the<br>
synthesis of the AreA, PepC, and/or PepE products.<br>
A further aspect of the invention is meant to comprise DNA constructs for use in the above mentioned methods.<br>
In respect of the former method said DNA constructs may comprise the areA, pepC, and/or pepE genes, wherein an interna) part has been substituted, deleted, or extra DNA has been inserted.<br>
At least one of the DNA constructs may furthermore also comprise DNA sequences encoding a protein product of interest, such as those mentioned later.<br>
In respect of the latter anti-sense method the DNA constructs may comprise inverted DNA sequence of the areA,pepC, and/or pepE genes connected to a functional promoter, whereby the mRNAs are at least partially complementary to mRNAs produced from the areA,pepC, and/or pepE genes.<br>
A further aspect of the invention relates to a process for the production of a desired gene product, preferably a secreted gene product, whereby a iungus according to the invention is cultivated in a suitable growth medium at appropriate conditions and the desired gene product is recovered and purified.<br><br>
In the case of a gene product expressed by a heterologous gene the DNA sequence coding for the desired gene product may be a part of the DNA construct used for producing said fungus.<br>
Normally, however, a separate transformation of the fungus of the invention is performed in order to make the fungus capable of producing the desired product.<br>
Methods for transforming fungi are well known in the art, cf. e.g. EP 0 184 438 A2 (Gist-Brocades N.V.) and EP publication No. 0 98 993 (Novo Nordisk A/S).<br>
For indigenous products this is of course not necessary, but in order to increase the production it may be an advantage to provide for multiple copies of the gene encoding the protein of interest to be incorporated into the host.<br>
The desired gene product is generally a peptide or protein, preferably an enzyme.<br>
Among enzymes it is preferably selected from the group comprising proteases, such as trypsin and chymosin; lipases, cutinases, cellulases, xylanases, laccases, pectinases, etc.<br>
Another type of desired gene product is generally a therapeutically active peptide or protein.<br>
Among the therapeutically active peptide or protein the protein preferably is selected from the group comprising insulin, growth hormone, glucagon, somatostatin, interferons, PDGF, factor VII, factor Vm, urokinase, t-PA, CSF, lactoferrin, TPO etc.<br>
A further aspect of the invention relates to the DNA sequences coding for the pepC gene from A. oryzae (SEQ ID No. 1), the pepE gene from A oryzae (SEQ ID No. 3)or functional alleles thereof. Also encompassed by the invention are the corresponding PepC and PepE proteases and their production, preferably by recombinant means.<br>
In this aspect the invention relates to processes for the production of the PepC protease or PepE protease from A. oryzae comprising transforming a suitable host with a DNA construct comprising a DNA sequence encoding the protease of interest, selecting a transformant capable of producing the protease, cultivating the transformant<br><br>
in an appropriate growth medium and recovering the PepC or PepE protease from the culture.<br>
The host used in such a process is preferably a host according to the above mentioned aspects of the invention<br>
In certain embodiments of the process for producing the PepC or PepE protease the host is A. oryzae. In that case it is preferred that the DNA construct comprising a DNA sequence coding for the protease, provides for an extra copy of the gene already present in the host.<br>
The DNA construct comprising the DNA sequence encoding the protease will normally also comprise regulatory elements in order to provide for proper expression and processing of the protease in the host.<br>
The invention is explained in further detail in the Examples given below. These should, however, not in any way be construed as limiting the scope of the invention as defined in the appended claims.<br>
F.XA1VTPT.FS<br>
Materials and Methods Strains<br>
A. oryzae, IF04177:available  from  Institute   for  Fermentation,   Osaka;   17-25  Juso<br>
Hammachi 2-Chome Yodogawa-Ku, Osaka, Japan.<br>
ToC9I3:	The construction of this strain is described in the Examples.<br>
areA:	This gene codes for a regulatory protein controlling nitrogen catabolism.<br>
pepC:        This gene codes for a serine protease of the subtilisin type<br>
pepE:	This gene codes for an aspartic protease.<br><br>
pyrG:	This   gene  codes   for  orotidine-S'-phosphate   decarboxylase,   an  enzyme<br>
involved in the biosynthesis of uridine.<br>
bar.	This gene was originally isolated from Streptomyces hygroscopicus and codes<br>
for phosphinothricin acetyltransferase. The enzyme modifies phosphinothricin<br>
(=glufosinate) and thereby inactivates this compound which is toxic to<br>
bacteria, fungi and plants.<br>
Plasmids<br>
pUC118:	pS02:	The construction of this plasmid is described in the Examples.<br>
pJers4:	A 2.0 kb subclone of pS02 in pUC118. pJers4 contains a functional A.<br>
oryzaepyrG gene.<br>
pS05:	The construction of this plasmid from pS02 is described in the Examples.<br>
pToC56:        The construction of this plasmid is described in EP publication No. 0 98<br>
993.<br>
pToC68:	The construction of this plasmid is described in WO 91/17243.<br>
pToC90:        A subclone of p3SR2, harboring the amdS gene from Aspergillus nidulans<br>
as a 2.7 kb Xbal fragment [Corrick et al., GENE 1987 53 63-71], on a<br>
pUC19 vector[Yannisch-Perron et al., GENE 1985 33 103-119], prepared<br>
as described in WO 91/17243. pToC266:       The construction of this plasmid is described in the Examples. pToC299:       The construction of this plasmid is described in the Examples. pToC338:       The construction of this plasmid is described in the Examples. pMT1606:      The construction of this plasmid from pBPIT (B. Straubinger et al. Fungal<br>
Genetics Newsletter 39(1992): 82-83) and p775 (EP publication No. 0 98<br>
993) is described in the Examples.<br>
p775:	The construction of this plasmid is described in EP publication No. 0 98<br>
993.<br><br>
p777:	The construction of this plasmid is described in EP publication No. 0 98<br>
993. pHW470:       The construction of this plasmid is described in the Examples.<br>
Example 1<br><br>
The A. oryzae pepE gene was cloned by cross-hybridization with the A. niger gene. A partial A. niger gene was obtained as a 700 bp PCR fragment from a PCR reaction with A. niger chromosomal DNA and pepE specific primers made according to thcpepE sequence published by G. Jarai et al, Gene 145 (1994) 171-178. The fragment was shown to contain pepE sequences by DNA sequencing. It hybridizes to A, oryzae chromosomal DNA under stringent conditions and Southern analysis showed that A. oryzae contains a single pepE like gene.<br>
The pepE gene was deleted both by the gene replacement method and the two step gene replacement method (G. May in "Applied Molecular Genetics of Filamentous Fungi" (1992) pp. 1-25. Eds. J. R. Kinghorn and G. Turner; Blackie Academic and Professional). As marker was used the A. oryzaepyrG gene, the A. oryzae strain was apyrG- strain made by deletion of Xh&amp;pyrG gene.<br><br>
A cosmid library of Aspergillus oryzae was constructed essentially according to the instruction from the supplier (Stratagene) of the "SuperCosl cosmid vector kit".<br>
Genomic DNA of A. oryzae IF04177 was prepared from protoplasts made by standard procedures (Christensen, T., et. al., Biotechnology 6 (1988) 1419-1422). After isolation of the protoplasts they were pelleted by centrifugation at 2500 rpm for 5 minutes in a Labofuge T (Heto), the pellet was suspended in 10 mM NaCl, 20 mM Tris-HCl (pH 8.0), 1 mM EDTA, 100 ug/ml proteinase K and 0.5% SDS as stated in the manual from the Supercos 1 cosmid vector kit and the rest of the DNA preparation was done according to the kit's instructions. The size of the genomic DNA was analysed by electrophoresis<br><br>
using the CHEF-gel apparatus from Biorad. A 1% agarose gel was run for 20 hours at 200 volt with a 10-50 second pulse. The gel was stained by etidium bromide and photographed. The DNA was 50-&gt;100 kb in size. The DNA was partially restricted by Sau3A. The size of the restricted DNA was 20-50 kb determined by the same type of CHEF-gel analysis as above. The CsCl gradient banded SuperCosl vector was prepared according to the manual. Ligation and packaging was likewise done as described. After titration of the library all of the packaging mix from one ligation and packaging was transfected into the host cells XL 1-Blue MR and plated on 50 jig/ml ampicillin LB plates. Approximately 3800 colonies were obtained. Cosmid preparation from 10 colonies showed that they all had inserts of the expected size. The colonies were picked individually and inoculated in microtiter plate wells with 100 ul LB (100 ug/ml ampicillin) and incubated at 37Â°C over night. 100 ul of 50% glycerol was added to each well and the whole library was frozen at -80Â°C. A total of 3822 colonies were stored. This represents the A. oryzae genome apr. 4.4 times.<br>
The individually frozen colonies in the library were inoculated onto LB-plates (100ng/ml ampiciUin) by using a multipin device with 6 times 8 pins fitting into half a microtiter dish. Plates were made containing colonies from all clones in the library. The plates were incubated at 37 C over night. Sterilized Whatman 540 filters cut to the size of a petri dish were placed upon the colonies which were incubated for two more hours at 37 C. The filters were transferred to LB plates containing 200|ag/ml of chloramphenicol and the plates were incubated over night at 37 C.The next day the filters were washed twice in 0.5 M NaOH for 5 minutes, then twice in 0.5 M Tris-HCl (pH=7.4) for 5 minutes and then twice in 2 x SSC for 5 minutes. The filters were wet with ethanol and air-dried.<br>
The filters were hybridized with a 0.7 kb 32P labelled PCR fragment containing part of the pepE gene from A. niger. The PCR fragment was obtained by running PCR on chromosomal DNA from A. niger with two primer 700 bp apart in the DNA sequence. The hybridization was carried out for 16 hours at 65 C in 10 Denhart, 5 x SSC, 0.02 M EDTA, 1% SDS, 0.15 mg/ml polyA and 0.05 mg/ml yeast tRNA. After hybridization the filters were washed in 2 x SSC, 0.1% SDS at 45Â°C twice and placed on X-ray films. 5 colonies hybridized with the probe, 4 of them were subsequently shown to contain the A. oryzae pepE gene by Southern analysis of the isolated cosmid DNA, using<br><br>
the same probe. Three of the cosmids were identical, thus two different cosmid clones containing pepE were isolated, they were called 7C7 and 33C1, names referring to their position in the stored library. Two overlapping fragments, a 4.3 kb EcoRI fragment (pToC299) and a 2.4 kb Hindlll (pToC301) fragment, were subcloned and partially sequenced. SEQ. ID No. 1 shows the DNA sequence and the deduced aa sequence for the protease. The gene shows strong homology to the A. niger gene.<br>
The A. oryzae pyrG gene was cloned by cross hybridization with the A. niger pyrG gene (W. van Hartingsveldt et al., Mol. Gen. Genet 206:71-75 (1987)). A lambda library of partial SauIIlA digested A, oryzae IF04177 DNA was probed at low stringency with a 1 kb DNA fragment from the A. niger pyrG gene. A 3.8 kb Hindlll fragment from a positive clone was subcloned into a pUC118 vector. The resultant plasmid, pS02, was shown to contain ihepyrG gene by complementation of an A. niger pyrG' mutant.<br><br>
A pyrG deletion plasmid, pS05, containing about 1 kb of pyrG flanking sequences on each end was constructed from the plasmid pS02. A. oryzae IF04177 was transformed with this construct and transformants were selected by resistance to 5-fluoro-orotic acid (FOA), a phenotype characteristic of pyrG mutants. One transformant, HowBlOl, was shown by Southern analysis to have the expected deletion at the pyrG locus. Being a pyrG mutant HowBlOl requires uridine for growth. HowBlOl can be transformed with the wtpyrG gene by selection for ability to grow without uridine.<br>
The steps involved in the construction of HowBlOl are illustrated in Fig. 1.<br>
A plasmid, pToC345, designed to replace the pepE gene with the pyrG gene, was constructed.<br><br>
Two PCR reactions were run with pToC299 as template; the first primer set was :<br>
19819	GAAGATCTGCGCGGATGTACATTGTAG<br>
19821	TTAGTCAGAAATTCGTCCCG<br>
The second was:<br>
19820	CCCAAGCTTCATGCTCGACCAGGGCCTCCT<br>
19818	GGTCTGTGTTAACCAAAGAAC<br>
The appr. 800 bp fragment obtained with 19819/19821 was cut with BgllllHindlH and cloned together with the 1.1 kb fragment obtained with 19820/19818 and cut with HindlHIPstI into BglWPstl cut pIC19R (J. L. Marsh et al, Gene 32 (1984) 481-484). The resulting plasmid was cut at the unique Hindlll site, dephosphorylated and the 3.5 kb pyrG containing fragment from pJaL335 (described in Example 2) was inserted. The construction of pToC345 is illustrated in Figure 2.<br>
HowBlOl was transformed with EcoRI cut pToC345 using standard procedures and transformants were selected by their ability to grow without the addition of uridine. 100 transformants were reisolated once through conidiospores. Spores were picked from single colonies on the reisolation plates and suspended in 100 ml of water with 0.01% Trition X-100. 1 ml spore suspension from each transformant and from IF04177, which was included as a control strain, were spotted on two Whatmann 540 filters placed on top of each their YPD plate. The plates were incubated at 30Â°C for 18 hours. The filters were removed from the plates and placed in 20% SDS for two hours at room temperature. They were then bakes for 3 minutes in a 600W micro-wawe oven. The filters were then washed for 5 minutes in 10%SDS, 2 times 5 minutes in 0.5M NaOH, 1.5M NaCl, one time 5 minutes in 0.5MTris-HCl pH=7.5, 1.5M NaCl and one time 5 minutes in 20xSSC and air dried. The two sets of filters were hybridized by standard procedures with each their 32p_iaDeUed probe. One set was hybridized with a 600bp BbuIIHindlll fragment from pToC299 containing the part of the pepE gene that was attempted to be deleted. The other set of filters was hybridized with a DNA<br><br>
fragment from the A. oryzae tpi gene. Any gene present in one copy, but pepE could be used since this is a control of the amount of DNA bound to the filters.<br>
After hybridization the filters were washed with O.lxSSC, 0.1% SDS at 65Â°C and the radioactivity bound to the filters were visualized by a Phospolmager. 13 of the transformants were picked for further analysis because they showed little hybridization to the pepE probe compared to the hybridization to the control probe. Chromosomal DNA was prepared by standard procedures and a Southern blot of the<br>
EcoRI restricted DNA was hybridized with a 32p-iabelled l.lkb Bbul fragment from pToC299 containing the 3' part of XhepepE gene which was not to be deleted. In the wt strain a 4.3 kb fragment should hybridize to the probe, in a correct replacement strain the 4.3 kb fragment should be replaced by a 7.2kb fragment. Two of the transformants looked correct, one had no hybridizing bands at all and most had the wt band plus maybe one other band, indicating integration of the transforming DNA at a nonÂ¬homologous locus.<br>
In order to isolate a pyrG~ derivative of the pepE deleted strain 10? conidiospores were spread on FOA containing plates and resistant colonies were selected. The FOA resistant colonies were reisolated, DNA was prepared and Southern analysis was performed to identify the strains in which the pyrG gene was lost via recombination between the repeat sequences flanking the gene in pToC345.<br><br>
A plasmid, pToC315, designed for a two step gene deletion of \hepepE gene was constructed. A 1.6 kb EcoRI/Hindlll (the Hindlll site was blunt ended by treatment with the Klenow fragment of DNA polymerase) from pToC299 containing sequences upstream from the pepE gene was cloned together with a 1.4 kb Sall/Bbul (the Bbul fragment was blunt ended) containing the 3 'end of the pepE gene into the EcoRI/Sall cut vector pUC19. The resulting plasmid was cut at the unique Hindlll site in the pUC19 linker, dephosphorylated and the 1.8 kb pyrG containing fragment from pJets4 was inserted. The construction of pToC315 is illustrated in Fig. 3.<br><br>
HowBlOl was transformed with pToC315 using standard procedures and transformants were selected by their ability to grow without the addition of uridine. After reisolation chromosomal DNA was prepared from 12 transformants, the DNA was cut with Asp718 and analysed by Southern analyses with a Bbul fragment from pToC301 containing part of ihepepE gene as a radioactive labelled probe. One transformant had the plasmid integrated in the endogenous pepE gene as revealed by the disappearance of the pepE specific Asp718 fragment, which had been replaced by two new bands as predicted if pToC315 had integrated as a single copy by homologous recombination at the pepE locus. The transformant was named ToC1089. 5x10 conidiae spores of ToC1089 were spread on plates containing 5-fluoro-orotic acid selecting for loss of the pyrG gene. This is the second step in a two step gene deletion, the pyrG gene can be lost by recombination with either of two pairs of identical sequences, one of which will result in the deletion of the pepE gene as well. The procedure is depicted in Fig. 4. The frequency of 5-fluoro-orotic acid resistance was approximately 10". The 5-fluoro-orotic acid resistant colonies were reisolated and a strain deleted for the pepE gene was identified by Southern analysis.<br>
EXAMPLE 2<br>
The A. oryzae pepC gene was cloned by cross-hybridization with the A. niger gene. The A. niger gene was obtained as a 1.1 kb PCR fragment from a PCR reaction with A. niger chromosomal DNA and pepC specific primers made according to the pepC sequence published by Frederick. G.D et al. Gene 125 (1993) 57-64. The fragment was shown to contain pepC sequences by DNA sequencing. It hybridizes to A. oryzae chromosomal DNA under stringent conditions and Southern analysis showed that A. oryzae contains a single pepC like gene.<br>
The pepC gene was deleted by a two step gene replacement method (G. May in "Applied Molecular Genetics of Filamentous Fungi" (1992) pp. 1-25. Eds. J. R. Kinghorn and G. Turner; Blackie Academic and Professional). As marker was used the A.<br><br>
otyzae pyrG gene, the A. oryzae strain was a pyrG strain made by deletion of the pyrG<br>
gene.<br>
From the published cDNA nucleotide sequence encoding A. niger pepC (Frederick G.D et al. Gene 125 (1993) 57-64) two oligonucleotides were designed so that the encoding part of the pepC gene where amplified in a PCR reaction. The primer #5258  where made so that the 3' end of the nucleotide sequence corresponds to the N-terminal part of the pepC gene (underline) and the 5'-end is for facilitating cloning (contains a BamHI restriction endonuclease site). The primer #5259<br>
where made so that the 3' end of the nucleotide sequence corresponds to the C-terminal part of thepepC gene and the 5'-end is for facilitating cloning (contains a Xhol restriction endonuclease site).<br>
Genomic DNA from A. niger was used as template in the PCR reaction. Amplification reaction were performed in 100 ul volumes containing 2.5 units Taq-polymerase, 100 ng of A, niger genomic DNA, 50 mM KC1, 10 raM Tris-HCl pH 8.0,1.5 mM MgCl2, 250 nM of each dNTP, and 100 pM of each of the two primers described above.<br>
Amplification was carried out in a Perkin-Elmer Cetus DNA Termal 480, and consisted of one cycle of 3 minutes at 94Â°C, followed by 25 cycles of 1 minutes at 94Â°C, 30 seconds at 55Â°C, and 1 minutes at 72Â°C. The PCR reaction produces one DNA fragment of ca. 1.1 kb in length. This fragment were isolated by gel electrophoresis, purified, cloned into the vector pCR*^ (Invitrogen Corporation), and sequenced using standard methods known in the art of molecular biology. The resulting plasmid were called pJaL197.<br><br>
Southern blot of genomic DNA from A. oryzae IF04177 where hybridized with the 1.1 kb J2P labelled EcoRI DNA fragment from pJaL197 clone containing the A, niger pepC gene. Genomic DNA was cut with the following restriction enzymes: EcoRI, BamHI, Xhol, and Hindlll. Hybridization was carried out for 16 hours at 65Â°C in 10 * Denhart, 5 * SSC, 0.02 M EDTA, 1% SDS, 0.15 mg/ml polyA and 0.05 mg/ml yeast tRNA. After hybridization the filters were washed in 2 x SSC, 0.1% SDS at 65Â°C twice and placed on X-ray films. The probe hybridized to a single size of fragment in each of the four digest, indicating that the pepC gene is present in a single copy in A. oryzae IF04177.<br>
A partially library of A. oryzae genomic DNA was constructed containing BamHI fragments with a size of 4.5-5.5 kband ligated into the vector pIC19H. The above A. niger pepC gene clone was radiolabeled and used to probe the partial A. oryzae BamHI genomic library. Hybridization was carried out as described above. About 4000 E. coli colonies were screened and four positive colonies was obtained. The 4 clones was shown to be identical by restriction enzyme digestion. One of these clones called pJaL235 (Fig. 5), with an insert of 4.6 kb, was analyses further by restriction mapping and Southern blotting. This shows that the pepC gene is located in a 2.9 kb BamHIISall fragment. Sequencing of this 2.9 kb BamHIISall fragment revealed the presence of a long open reading frame of 495 amino acids interrupted by two introns with consensus sequences indicative of intron splicing. The sequence of the A. oryzae pepC gene is shown in SEQ ID No. 3.<br>
By PCR, with the primer #7659<br>
where the 3' end of the nucleotide sequence corresponds to position 7-26 in pS02 (underline) and the 5'-end is for facilitating cloning (contains a BgUI restriction endonuclease site), and the primer #7656<br><br><br>
where the 3' end of the nucleotide sequence corresponds to position 385-407 in pS02 (underline) and the 5'-end is for facilitating cloning (contains a EcoRI and HindlH restriction endonuclease site), on the plasmid pS02 a 432 bp fragment was amplified. The fragment was digested with BgllJ and EcoRI and isolated by gel electrophoresis, purified, and cloned into the corresponding site in pS02, resulting in plasmid pJaL335 (The construction is outlined in Fig. 6.<br>
Plasmid pJaL235 was digested with Pvul and treated with Klenow polymerase to make the ends blunt and then digested with Hindlll. The 2.6 kb fragment were isolated by gel electrophoresis, and purified. The 2.6 kb fragment was cloned into pUC12 digested with Sma/and Hindlll giving plasmid pJaL308.<br>
Plasmid pJaL308 was digested with Smal and treated with bacterial alkaline phosphatase to remove the 5* phosphate groups according to the manufacturers instructions and phenol extracted and precipitated.<br>
Plasmid pS02 was digested with Hindlll, and treated with Klenow polymerase to make the ends blunt. The 3.8 kb fragment encoding the A. oryzae pyrG gene were isolated by gel electrophoresis, and purified.<br>
The two fragments are mixed together and ligated. After transformations of E. coli, the colonies carrying the correct plasmids are identified by restriction enzyme digestion of mini-plasmid preparations. The construction of pJaL363 is illustrated in Fig. 7.<br>
Plasmid pJaL363 consist of pUC12 vector containing a fragment which carries the pepC gene flanked by an EcoRI site and an Hindlll and where the pepC is interrupted by an 3.8 kb DNA fragment encoding the A. oryzae pyrG gene.<br><br>
Plasmid pJaL335 is digested with HindHI, and treated with Klenow polymerase to make the ends blunt. The 3.5 kb fragment encoding the A. otyzaepyrG gene is isolated by gel electrophoresis, and purified. The fragment is cloned into pJaL308 Smal restriction site. The construction of pJaLz is outlined in Fig. 8. The plasmid consist of pUC 12 vector containing a fragment which carries the pepC gene flanked by an EcoRl site and an HindHI and where the pepC is interrupted by an 3.5 kb DNA fragment encoding the A. oryzaepyrG gene.<br><br>
15 ug of either one of the disruption plasmids is digested to completion by HindHI and EcoRl. The completeness of the digest is checked by running an aliquot on a gel and the remainder of the DNA is phenol extracted, precipitated and resuspended in 10 ufof sterile water.<br>
The transformation of A. oryzae HowBlOI host strain is preformed by the protoplast method (Christensen et al. Biotechnology (1988) 6:1419-1422). Typically, A. oryzae mycelia is grown in a rich nutrient broth. The mycelia is separated from the broth by filtration. The enzyme preparation NovozymeÂ® (Novo Nordisk) is added to the mycelia in osmotically stabilizing buffer such as 1.2 M MgSCU buffered to pH 5.0 with sodium phosphate. The suspension was incubated for 60 minutes at 37Â°C with agitation. The protoplast is filtered through mira-cloth to remove mycelial debris. The protoplast is harvested and washed twice with STC (1.2 M sorbitol, 10 mM CaCl2, 10 mM Tris-HCl pH 7.5). The protoplast is finally resuspended in 200-1000 jul STC.<br>
For transformation 5 ug DNA is added to 100 u.1 protoplast suspension and then 200 ul PEG solution (60% PEG 4000, 10 mM CaCl2, 10 mM Tris-HCl pH 7.5) is added and the mixture is incubated for 20 minutes at room temperature. The protoplast is harvested and washed twice with 1 -2 M sorbitol. The protoplast is finally resuspended 200 ml 1.2 M sorbitol, plated on selective plates (minimal medium + 10 g/1 Bacto-Agar (Difco), and incubated at 37Â°C. After 3-4 days of growth at 37Â°C, stable transformants appear as vigorously growing and sporulating colonies.<br><br><br>
' From the stable colonies individual spores is streaked on fresh minimal plates. Single colonies is selected and restreaked to give pure cultures. These are used to inoculate 10 ml of liquid YPM medium (1% yeast extract, 1% peptone, 2% maltose). After 18 hours at 30Â°C shaking at 180 rpm, the mycelia are harvested on filter paper. Mycelia is then transfer to an 2 ml eppendorf tube and freeze dried. After freeze drying DNA is prepared from the individual mycelia by grinding the mycelia to a fine powder with a pestle in the tube. This powder are resuspended in 0.5 ml of 50 mM EDTA pH 8.0, 0.2% SDS, 1 ul DEP by vortexing. These are incubate at 65Â°C for 20 minutes. After this is added 0.1 ml 5 M KAc pH 6.5, the solution is mixed and incubated on ice for 5 minutes. The cell debris is separated from the DNA solution by centrifugation at 20.000 rpm for 5 minutes. 0.4 ml supernatant are precipitated with 0.3 ml isopropanol and centrifiigated at 20.000 rpm for 10 minutes. The DNA pellet is redissolved in 100 ul of sterile TE buffer containing 0.1 <br>
3 |ig of each DNA is digested with EcoRI, fractionated by agarose gel electrophoresis, transferred to Immobilan-N membrane filters, and probe with the 1.5 kb 32P labelled Ncol DNA fragment from pJaL335 containing part of the pepC protease gene. Strains which carry a disruption of the pepC are easily recognized by that the wild type band on 3.6 kb is shifted to a 7.4 kb band in the transformant.<br>
The disrupted A. oryzae pepC strain is made pyrG minus by selecting spontaneous mutant resistance to 5-fluoro-orotic acid, a phenotype characteristic of pyrG mutants. Being a pyrG mutant the strain requires uridine for growth. The strain can be transformed with the wtpyrG gene by selection for the ability to grow without uridine.<br>
EXAMPLE 3<br><br>
The areAA strain was constructed as follows. The areA gene from A. oryzae was cloned. A pyrG strain also deficient for either pepC or pepE or pepC plus pepE was<br><br>
transformed with a plasmid carrying the pyrG gene inserted between DNA fragments upstream and downstream from the areA gene. The coding region for areA was not present on the plasmid. Transformants were selected for their ability to grow in the absence of uridine and in the presence of chlorate. This double selection selects both for a functional pyrG gene and for areA minus. Strains obtained by this selection procedure were finally screened by Southern analysis to identify those in which the chromosomal areA gene was substituted by the pyrG gene.<br><br>
The A. oryzae areA gene was cloned by cross hybridization to the A. nidulans areA gene (B. Kudla et al., EMBO J. 9:1355-1364 (1990)). A genomic library of A. oryzae IF04177 was prepared by partial digestion of chromosomal DNA with SauIIIA and cloning of the obtained DNA fragments into the vector 1GEM-II (obtained from Promega). Cross hybridization of the library with the A. nidulans areA gene was performed in 40% formamide at 37Â°C. Hybridizing 1 clones were isolated and from these fragments were sub-cloned into the vector pBluescript SK+ (obtained from Stratagene) giving rise to the plasmids pSK5 and pSK9 illustrated in Fig. 9. The cloned gene was able to complement an A. nidulans areA mutant, proving that it is indeed the A oryzae areA homolog. 5643 bp of the clone was sequenced, and comparison of the sequences of the A. oryzae and the A. nidulans areA genes shows that they are highly homologous. The sequence of the A. oryzae areA gene is shown in SEQ ID No. 5.<br><br>
In order to delete the areA gene from the A. oryzae chromosome the plasmid pToC266 was constructed. pToC266 contains a 2.1 kb DNA fragment originating upstream of the areA gene (isolated from pSK5) and a 1.4 kb DNA fragment originating downstream from the areA gene (isolated from pSK9). The two fragments are separated by approximately 3.2 kb in the genome, the coding region is situated in this part of the gene. The A. oryzae pyrG gene from pjers4 was inserted between the areA upstream and downstream DNA fragments. The construction of pToC266 is illustrated in Figs. 10a and 10b. pToC266 has a unique EcoRI site and was linearized by cutting with this restriction enzyme before used in transformations.<br>
A pyrG strain also deficient for either pepC or pepE or pepC plus pepE is transformed with linearized pToC266. Transformants are selected on minimal plates (Cove Biochem. biophy. Acta (1966) 113 : 51-56) containing glutamine as the nitrogen source and glucose as the carbon source. Transformants are reisolated twice on the same type of plates, and then subjected to growth test on different nitrogen sources. Transformants growing well on glutamine but not on nitrate, ammonium or urea are expected to be deleted for areA. The deletion is confirmed by Southern analysis.<br>
EXAMPLE 4<br><br>
A plasmid containing the bar gene from Streptomyces hygroscopius (C. J. Thompson et. al, EMBO J. 6 : 2519-2523 (1987)) inserted after the A. oryzae TAKA-amylase promoter and followed by a fragment containing the transcriptional terminator and poEyadenylation signal from the A. niger gla gene was constructed.<br>
The plasmid, pMT1606, can be used for selection of glufosinate resistant transformants of A. oryzae. pMT1606 was constructed by isolating the bar gene from the plasmid pBPIT (B. Straubinger et. al, Fungal Genetics Newsletter 39 : 82-83 (1992)) and<br><br>
cloning it into the fungal expression plasmid p775 described in EP publication No. 0 098 993 Al. Fig. 11 illustrates the construction of pMT1606.<br>
EXAMPLES<br><br>
An A. oryzae areAkgepE&amp;pepC strain is transformed with the plasmid pToC56 (Fig. 12), which is a fungal expression plasmid for the mammalian enzyme chymosin, by co-transformation with pMT1606. Construction of the plasmid pToC56 is described in EP publication No. 0 98 993.<br>
Transformants are selected for growth on minimal medium containing 10 mM ammonium and 1 mg/ml glufosinate and screened for the presence of pToC56 by the ability to produce chymosin. The transformants are grown in shake flasks in minimal medium containing maltodextrin and glutamine for 4 days at 30Â°C.The content of chymosin in the supernatants were analysed by SDS-Page and Western blotting.<br>
Example 6<br><br>
Construction of an expression plasmid for pepC. Plasmid pJaL235 was digested with AatH and Nsil and treated with Klenow polymerase to make the ends blunt. The 1.7 kb fragment was isolated by gel electrophoresis, and purified. The 1.7 kb fragment was cloned into pIC19H digested with Smal giving pJaL365.<br>
Plasmid pJaL365 was digested with BamHI and Xhol and the 1.7 kb fragment was isolated by gel electrophoresis, and purified. The 1.7 kb fragment was cloned into pToC68 digested with BamHI and Xhol giving pJaL368 (Figs. 13a and 13 b).<br><br>
An A. oryzae strain is transformed with the plasmid pJaL368, which is a fungal expression plasmid for the protease PepC, by cotransformation with pToC90.<br>
Transformants are selected for growth on minimal medium containing 10 mM acetamide and screened for the presence of pJaL368 by the ability to produce the protease PepC.<br>
Example 7<br>
Overevpression nf pc.pK<br>
A plasmid called pToC338 carrying the pepE gene fused to the TAKA-amylase promoter from A. oryzae was constructed. Figs. 14a and 14b depicts the construction.<br>
An EcoRllSall fragment from pToC299 containing most of the coding region and appr. 430 bp of the 3' untranslated region of pepE was cloned into EcoRllBamHl cut pUC19 together with a synthetic DNA fragment of the following sequence:<br>
8681    GATCCACCATGAAG 8747    GTGGTACTTCAGCT<br>
The resulting plasmid called pToC334 was cut with BamHIIEcoRI and a fragment containing the entire structural gene of pepE with a BamHI site fused immediately upstream of the start codon was isolated, approximately 430 bp of untranslated 3' sequence was also present in the fragment. The fragment was cloned into EcoRIISai7 cut pUC19 together with an approximately 1.1 kb SalllBamHI fragment from the plasmid p775 containing the TAKA-amylase promoter from A. oryzae. The resulting plasmid was named pToC338.<br>
pToC338 was co-transformed into A, oryzae JaL125 (an A. oryzae alp minus strain described in Danish Patent Application No. 0354/96) with pToC90, which contains the A. nidulans acetamidase (amdS) gene, using standard procedures {e.g. as described in<br><br>
EP 0 098 993 Al). Transformants were selected by their ability to use acetamide as the sole nitrogen source. 11 transformants were reisolated twice through conidiospores. The transformants were fermented for three days at 30Â°C inlO ml YPM (YP with 2% maltose) and the fermentation broth was analysed by SDS-page. One of the transformants produced a protein of the same size as the protein encoded by the pepE gene, protease activity measurements confirmed that the broth from that transformant contained a higher activity toward casein at pH=5.5 compared to the host strain JaL125. The protein was purified and N-terminal sequencing showed that it is indeed the protein encoded by the pepE gene. The N-terminal of the secreted protein was:<br>
gly*-arg-his-asp-val-leu-val-asp-asn-phe-leu-asn-ala-gln-tyr-phe-ser-glu-ile-glu-ile-gly-<br>
thr-pro-pro-gln-lys-phe-lys<br>
*this residue could also be a lysine.<br>
confirming the expression of the PepE protease.<br><br>
SEQUENCE LISTING<br>
(1)	GENERAL INFORMATION:<br>
(i) APPLICANT:<br>
(A)	NAME: Novo Nordisk A/S<br>
(B)	STREET: Novo AUe<br>
(C)	CITY: Bagsvaerd<br>
(E) COUNTRY: DENMARK <f postal code dk-2880 telephone: telefax:></f>
(ii) TITLE OF INVENTION: Novel Microorganisms<br>
(Hi) NUMBER OF SEQUENCES: 12<br>
(iv) COMPUTER READABLE FORM:<br>
(A)	MEDIUM TYPE: Floppy disk<br>
(B)	COMPUTER: IBM PC compatible<br>
(C)	OPERATING SYSTEM: PC-DOS/MS-DOS<br>
(D)	SOFTWARE: Patentln Release #1.0, Version #1.30 (EPO)<br>
(2)	INFORMATION FOR SEQ ID NO: 1:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 2454 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br><br>
(ii) MOLECULE TYPE: DNA (genomic) (iii) HYPOTHETICAL: NO (iv) ANTI-SENSE: NO (vi) ORIGINAL SOURCE:<br>
(A) ORGANISM: Aspergillus oryzae (ix) FEATURE:<br>
(A) NAME/KEY: exon<br>
(B)LOCATION:603..70l (ix) FEATURE:<br>
(A)	NAME/KEY: intron<br>
(B)	LOCATION:702..791 (ix) FEATURE:<br><br>
(A)	NAME/KEY: exon<br>
(B)	LOCATION:792..942 (ix) FEATURE:<br>
(A) NAME/KEY: intron<br>
(B)LOCATION:943..10GI (ix) FEATURE:<br>
(A) NAME/KEY: exon<br>
(B)LOCATION:1002..1656 (ix) FEATURE:<br>
(A)	NAME/KEY: intron<br>
(B)	LOCATION:1657..1?13 (ix) FEATURE:<br>
(A) NAME/KEY: exon<br>
(B)LOCATION:1714..2G01 (ix) FEATURE:<br>
(A) NAME/KEY: CDS<br>
(B)LOCATION:join(603-701, 792-942, 1002-1656, 1714-2001) (ix) FEATURE:<br>
30<br><br>
(A)	NAME/KEY: mat_peptide<br>
(B)	LOCATION:603..2001<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 1:<br>
GCGCGGATGT ACATTGTAGG TACATGGATA GTATGTACTG TATATACCGT CATTTAGTAA      60<br>
GGCAACTAAC TAACTTATCA GCCTAGCTCC CGAGACGGCC TTACATCATC CGCAGCGGCA     120<br>
ATCAGCTCCA CACCCTTGGA TAGTGATAAG AGACACAAGG AGTTCTGAGT ATGGTATTAT     180<br>
AAGTGCAGTG AGTTGGGATG AAACAGAGAG ATAGAGGGAA TACTCCTATT TATCAATGAA    240<br>
CGTATACAGA CATACCCCAG CAGCGTTCCT GGCGGTATTG TAAAAGGGCC GTACCTTGGA    300<br>
GATCAAGTGA TGAGACACCC GTGATGCAGG AACTCCACTT CAATCCAATG ACGCATCGAG     360<br>
TTGCTCCCTG ATTGGTTGAT ACGCAGGTCG CTCCGCAACC GGTCCGCATC ACCTCACTTC     420<br>
CCTCCCCCAG ACCTGGAGGT ACCTCTCCCG TCCTTCTCTC CCTCTCCATC CCATCATCTA    480<br>
TCCCTCTCCA GACCCTGATT GTATTTCATC ATTCCTATCG TCCCATATTA ATAGAGTATT    540<br>
GCTAGTTTTC TTTTGATTTC GTCTGTTGAG GTGCTGCTTT TTTGTCGCCG TTGTCGCCCA     600<br>
CC ATG AAG TCG ACC TTG GTT ACG GCC TCT GTG CTG TTG GGC TGT GCT<br>
647<br>
Met Lys Ser Thr Leu Val Thr Ala Ser Val Leu Leu Gly Cys Ala<br>
15	10	15<br>
TCC GCC GAG GTT CAC AAG CTG AAG CTC AAC AAG GTG CCC GTG TCC GAG      695<br><br>
Ser Ala Glu Val His Lys Leu Lys Leu Asn Lys Val Pro Val Ser Glu<br>
20	25	30<br>
CAA TTT GTGAGTAGAC CTTACTATTC CGGCCATGAA AATATTCATC TACC-<br>
CATCTG       751<br>
GlnPhe<br>
AAAGCTTGTC GGGACGAATT TCTGACTAAA TCGTATCCAG AAC TTG CAC AAC ATC       806<br>
Asn Leu His Asn He<br>
35<br>
GAC ACC CAT GTG CAG GCT CTC GGC CAG AAG TAC ATG GGA ATC CGT CCC  854<br>
Asp Thr His Val Gin Ala Leu Gly Gin Lys Tyr Met Gly He Arg Pro<br>
40	45	50<br>
AAC ATC AAG CAA GAT CTT CTC AAT GAG AAC CCG ATT AAC GAT ATG GGA      902<br>
Asn He Lys Gin Asp Leu Leu Asn Glu Asn Pro He Asn Asp Met Gly<br>
55	60	65	70<br>
CGT CAT GAT GTC CTT GTT GAC AAC TTC CTG AAT GCA CAA T 942<br>
Arg His Asp Val Leu Val Asp Asn Phe Leu Asn Ala Gin<br>
75	80<br>
GTACGAAACC CTAGTAATAC TTGAAGGGGG GCTCCAACTT ACGCGTAGAT TCTCTAAAG AC 1003<br>
Tyr<br>
TTC TCC GAA ATC GAG ATC GGT ACT CCT CCA CAG AAG TTC AAG GTG<br>
GTC     1051<br>
Phe Ser Glu He Glu lie Gly Thr Pro Pro Gin Lys Phe Lys Val Val<br>
85	90	95	100<br>
CTT GAC ACT GGC AGC TCA AAC CTA TGG GTG CCC TCT TCG GAG TGT GGT     1099<br>
Leu Asp Thr Gly Ser Ser Asn Leu Trp Val Pro Ser Ser Glu Cys Gly<br>
105	110	115<br>
TCT ATC GCC TGC TAT TTG CAT AAC AAG TAC GAC TCA TCC TCG TCC<br>
TCC      1147<br>
Ser He Ala Cys Tyr Leu His Asn Lys Tyr Asp Ser Ser Ser Ser Ser<br><br>
120	125	130<br>
ACG TAC CAG AAG AAT GGC AGC GAA TTT GCC ATC AAG TAC GGC TCT<br>
GGT      1195<br>
Thr Tyr Gin Lys Asn Gly Ser Glu Phe Ala He Lys Tyr Gly Ser Gly<br>
135	140	145<br>
AGC CTG AGT GGT TTT GTT TCT CAG GAT ACT CTC AAG ATC GGT GAC CTG      1243<br>
Ser Leu Ser Gly Phe Val Ser Gin Asp Thr Leu Lys He Gly Asp Leu<br>
150	155	160<br>
AAG GTG AAG GAT CAG CTG TTC GCC GAG GCT ACT AGT GAG CCC GGC<br>
CTT      1291<br>
Lys Val Lys Asp Gin Leu Phe Ala Glu Ala Thr Ser Glu Pro Gly Leu<br>
165	170	175	180<br>
GCT TTT GCC TTT GGC CGC TTT GAT GGT ATC CTT GGG TTG GGA TTT GAC     1339<br>
Ala Phe Ala Phe Gly Arg Phe Asp Gly He Leu Gly Leu Gly Phe Asp<br>
185	190	195<br>
ACA ATT TCC GTC AAC AAG ATT CCT CCA CCC TTC TAT AGC ATG CTC GAC      1387<br>
Thr He Ser Val Asn Lys He Pro Pro Pro Phe Tyr Ser Met Leu Asp<br>
200	205	210<br>
CAG GGC CTC CTC GAC GAG CCA GTC TTT GCT TTC TAC CTT GGA GAC<br>
ACT      1435<br>
Gin Gly Leu Leu Asp Glu Pro Val Phe Ala Phe Tyr Leu Gly Asp Thr<br>
215	220	225<br>
AAC AAG GAA GGT GAT GAC TCC GTA GCG ACA TTC GGC GGT GTT GAC AAG      1483<br>
Asn Lys Glu Gly Asp Asp Ser Val Ala Thr Phe Gly Gly Val Asp Lys<br>
230	235	240<br>
GAT CAC TAC ACC GGC GAG TTG GTC AAG ATT CCC CTT CGC CGC AAG<br>
GCC     1531<br>
Asp His Tyr Thr Gly Glu Leu Val Lys He Pro Leu Arg Arg Lys Ala<br>
245	250	255	260<br>
TAC TGG GAG GTT GAC CTT GAT GCT ATC GCC CTT GGC GAT AGC GTT GCT      1579<br><br>
Tyr Tip Glu Val Asp Leu Asp Ala He Ala Leu Gly Asp Ser Val Ala<br>
265	270	275<br>
GAA CTC GAT AAC ACC GGT GTC ATT CTG GAT ACC GGC ACT TCC CTT ATC  1627<br>
Glu Leu Asp Asn Thr Gly Val He Leu Asp Thr Gly Thr Ser Leu He<br>
280	285	290<br>
GCC TTG GCC ACC ACC CTT GCC GAG CTT AT GTAAGTCAAG<br>
CCAGTGTACT       1676<br>
Ala Leu Ala Thr Thr Leu Ala Glu Leu He<br>
295	300<br>
GTGCATGTCT GTCATACTCT TACTAACTAT TCTGAAG T AAC AAG GAA ATC GGT       1729<br>
Asn Lys Glu lie Gly 305<br>
GCC AAG AAG GGC TTC ACC GGC CAA TAC TCG GTT GAC TGT GAC AAG CGC  1777<br>
Ala Lys Lys Gly Phe Thr Gly Gin Tyr Ser Val Asp Cys Asp Lys Arg<br>
310	315	320<br>
GAT TCC TTG CCT GAC CTC ACC TTC ACC CTG AGC GGA TAC AAC TTC ACC  1825<br>
Asp Ser Leu Pro Asp Leu Thr Phe Thr Leu Ser Gly Tyr Asn Phe Thr<br>
325	330	335<br>
ATT GGT CCC TAC GAC TAC ACT CTT GAA GTC CAG GGA TCT TGC ATC<br>
AGC  1873<br>
He Gly Pro Tyr Asp Tyr Thr Leu Glu Val Gin Gly Ser Cys He Ser<br>
340	345	350	355<br>
GCC TTC ATG GGC ATG GAC TTC CCT GAA CCC GTT GGC CCC TTG GCC ATC      1921<br>
Ala Phe Met Gly Met Asp Phe Pro Glu Pro Val Gly Pro Leu Ala He<br>
360	365	370<br>
CTG GGT GAC GCG TTC CTC AGG AAG TGG TAC AGT GTG TAC GAC CTC GCC  1969<br>
Leu Gly Asp Ala Phe Leu Arg Lys Trp Tyr Ser Val Tyr Asp Leu Ala<br>
375	380	385<br><br>
AAC GGT GCT GTT GGC CTG GCC AAG GCT AAG TAACCAAGTA<br>
ATCTACCATG       2019<br>
Asn Gly Ala Val Gly Leu Ala Lys Ala Lys<br>
390	395<br>
CTATGTTCTT ATTGGTTGCT TGTGTATGTG AGACAATGGT ACATGATAGC CTGCCTCGGT   2079<br>
AGTTGGTTGG CCTTTTTCTG TTACGGGAAA TCGGCAAAGC CTTGTTTTCG CTTATGACCT    2139<br>
CTATCCTGTT TGTTATTGAT ATTTTGTGTG ACTCAGTGAG CCACTGGCTA TGCTCTAATG    2199<br>
ACATTCATTG GATGCCGATA GTTCTATATA CATTGCGATT TTAACGCGTA TCTTTGATCT   2259<br>
ATCGGTACAA TGATTCCCTA CTAAAGGTAG CCCAACTAGA CAACTATGCC TACGACCTCT   2319<br>
CTACATTCTT CATAGCTCCG TGTGGAGTCC GTCTCATACA ACCTCGAGCA ACCTGCAGTT   2379<br>
CTTTGGTTAA CACAGACCAC ACCTTAAAAC GGCACGATCC ATTCGAATAG ACAAGCCCTC   2439<br>
TTAATATTTG AATTC	2454<br>
(2) INFORMATION FOR SEQ ID NO: 2:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 397 amino acids<br>
(B)	TYPE: amino acid (D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 2:<br><br>
Met Lys Ser Thr Leu Val Thr Ala Ser Val Leu Leu Gly Cys Ala Ser<br>
15	10	15<br>
Ala Glu Val His Lys Leu Lys Leu Asn Lys Val Pro Val Ser Glu Gin<br>
20	25	30<br>
Phe Asn Leu His Asn He Asp Thr His Val Gin Ala Leu Gly Gin Lys<br>
35	40	45<br>
Tyr Met Gly He Arg Pro Asn He Lys Gin Asp Leu Leu Asn Glu Asn<br>
50	55	60<br>
Pro He Asn Asp Met Gly Arg His Asp Val Leu Val Asp Asn Phe Leu<br>
65	70	75	80<br>
Asn Ala Gin Tyr Phe Ser Glu He Glu He Gly Thr Pro Pro Gin Lys<br>
85	90	95<br>
Phe Lys Val Val Leu Asp Thr Gly Ser Ser Asn Leu Trp Val Pro Ser<br>
100	105	110<br>
Ser Glu Cys Gly Ser He Ala Cys Tyr Leu His Asn Lys Tyr Asp Ser<br>
115	120	125<br>
Ser Ser Ser Ser Thr Tyr Gin Lys Asn Gly Ser Glu Phe Ala He Lys<br>
130	135	140<br>
Tyr Gly Ser Gly Ser Leu Ser Gly Phe Val Ser Gin Asp Thr Leu Lys<br>
145	150	155	160<br>
He Gly Asp Leu Lys Val Lys Asp Gin Leu Phe Ala Glu Ala Thr Ser<br>
165	170	175<br>
Glu Pro Gly Leu Ala Phe Ala Phe Gly Arg Phe Asp Gly He Leu Gly<br>
180	185	190<br>
Leu Gly Phe Asp Thr He Ser Val Asn Lys He Pro Pro Pro Phe Tyr<br>
195	200	205<br>
Ser Met Leu Asp Gin Gly Leu Leu Asp Glu Pro Val Phe Ala Phe Tyr<br>
210	215	220<br>
Leu Gly Asp Thr Asn Lys Glu Gly Asp Asp Ser Val Ala Thr Phe Gly<br>
225	230	235	240<br><br>
Gly Val Asp Lys Asp His Tyr Thr Gly Glu Leu Val Lys He Pro Leu<br>
245	250	255<br>
Arg Arg Lys Ala Tyr Trp Glu Val Asp Leu Asp Ala He Ala Leu Gly<br>
260	265	270<br>
Asp Ser Val Ala Glu Leu Asp Asn Thr Gly Val He Leu Asp Thr Gly<br>
275	280	285<br>
Thr Ser Leu He Ala Leu Ala Thr Thr Leu Ala Glu Leu He Asn Lys<br>
290	295	300<br>
Glu He Gly Ala Lys Lys Gly Phe Thr Gly Gin Tyr Ser Val Asp Cys<br>
305	310	315	320<br>
Asp Lys Arg Asp Ser Leu Pro Asp Leu Thr Phe Thr Leu Ser Gly Tyr<br>
325	330	335<br>
Asn Phe Thr He Gly Pro Tyr Asp Tyr Thr Leu Glu Val Gin Gly Ser<br>
340	345	350<br>
Cys He Ser Ala Phe Met Gly Met Asp Phe Pro Glu Pro Val Gly Pro<br>
355	360	365<br>
Leu Ala He Leu Gly Asp Ala Phe Leu Arg Lys Trp Tyr Ser Val Tyr<br>
370	375	380<br>
Asp Leu Ala Asn Gly Ala Val Gly Leu Ala Lys Ala Lys<br>
385	390	395<br>
(2) INFORMATION FOR SEQ ID NO: 3:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 3224 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: DNA (genomic)<br><br>
(iii) HYPOTHETICAL: NO (iv) ANTI-SENSE: NO (vi) ORIGINAL SOURCE:<br>
(A) ORGANISM: Aspergillus oryzae (ix) FEATURE:<br>
(A) NAME/KEY: exon<br>
(B)LOCATION:388..756 (ix) FEATURE:<br>
(A) NAME/KEY: intron<br>
(B)LOCATION:757..817 (ix) FEATURE:<br>
(A) NAME/KEY: exon<br>
(B)LOCATION:818..I753 (ix) FEATURE:<br>
(A)	NAME/KEY: intron<br>
(B)	LOCATION:1754..1814 (ix) FEATURE:<br><br>
(A)	NAME/KEY: exon<br>
(B)	LOCATION-.1815..1997 (ix) FEATURE:<br>
(A) NAME/KEY: matjeptide (B)LOCATION:388..1997 (ix) FEATURE:<br>
(A) NAME/KEY: CDS (B)LOCATION:join(388..756,818..1753,1815..1997)<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 3:<br>
GGATCCATTA CCCTCTTACC GCCATACCCC AGGTCTTGCG ACCGCGCTAA TCGGGAGCGA     60<br><br>
TCGACGCGCG GCACCTCCTC AGTAAAGCTG TGTCATCATT GTAAATTACC GTATCCCGGT     120<br>
TGCATCATCC TCCGCTGCCC TTGCCTGCTT GGGGGATCGA CCTATTAAGC CCAGCTATCT     180<br>
TACACCTGCT CCCATCCTCC TCTTCTCCAA CTCCTCATCC ATCCTTCCTC CTCCTTCTTC     240<br>
CTTTTAACCC CCCCAACTCA GCATCGTTCC ATCCTCCCAT CTTTCCTTTC TTTTTACCTC    300<br>
AAATCTCCAT CTGTATTCTT TCCTCTTAGA ACTCTTCCTT TCCCCCCTTC TGTACCTTGT    360<br>
GTTTAGACGT CACTCTTGTT GCCCATC ATG AGA GGC ATC CTC GGC CTT<br>
TCC        411<br>
Met Arg Gly He Leu Gly Leu Ser<br>
1	5<br>
CTG CTG CCA CTA CTA GCA GCG GCC TCC CCC GTT GCT GTT GAC TCC ATC      459<br>
Leu Leu Pro Leu Leu Ala Ala Ala Ser Pro Val Ala Val Asp Ser He<br>
10	15	20<br>
CAC AAC GGA GCG GCT CCC ATT CTT TCG GCC TCA AAT GCC AAA GAG GTT      507<br>
His Asn Gly Ala Ala Pro He Leu Ser Ala Ser Asn Ala Lys GIu Val<br>
25	30	35	40<br>
CCA GAC TCT TAC ATT GTC GTC TTC AAG AAG CAT GTT TCC GCT GAA<br>
ACG      555<br>
Pro Asp Ser Tyr He Val Val Phe Lys Lys His Val Ser Ala Glu Thr<br>
45	50	55<br>
GCT GCT GCT CAT CAC ACC TGG GTG CAG GAC ATC CAC GAT TCG ATG ACT  603<br>
Ala Ala Ala His His Thr Trp Val Gin Asp He His Asp Ser Met Thr<br>
60	65	70<br>
GGA CGC ATC GAC CTG AAG AAG CGC TCT CTT TTT GGT TTC AGT GAT GAC      651<br>
Gly Arg He Asp Leu Lys Lys Arg Ser Leu Phe Gly Phe Ser Asp Asp<br>
75	80	85<br><br>
CTT TAC CTC GGT CTC AAG AAC ACC TTC GAT ATC GCC GGG TCC CTA GCG  699<br>
Leu Tyr Leu Gly Leu Lys Asn Thr Phe Asp He Ala Gly Ser Leu Ala<br>
90	95	100<br>
GGC TAC TCC GGA CAT TTC CAT GAG GAT GTG ATC GAG CAG GTC CGG<br>
AGA  747<br>
Gly Tyr Ser Gly His Phe His Glu Asp Val He Glu Gin Val Arg Arg<br>
105	110	115	120<br>
CAT CCT GAT GTAGGTTCCC CCCTCGGCCC ACCCGTTTTT GTAGAGCCCT<br>
796 His Pro Asp<br>
TGGTCTAACT TGATTTTCAA G GTT GAA TAC ATC GAG AAA GAC ACC GAA GTC  847<br>
Val Glu Tyr He Glu Lys Asp Thr Glu Val<br>
125	130<br>
CAC ACC ATG GAG GAG ACA ACC GAG AAG AAT GCT CCC TGG GGC TTG GCT  895<br>
His Thr Met Glu Glu Thr Thr Glu Lys Asn Ala Pro Trp Gly Leu Ala<br>
135	140	145<br>
CGT ATC TCT CAC CGT GAC AGC CTC TCG TTC GGT ACC TTT AAC AAG<br>
TAC      943<br>
Arg He Ser His Arg Asp Ser Leu Ser Phe Gly Thr Phe Asn Lys Tyr<br>
150	155	160	165<br>
CTG TAT GCT TCG GAA GGC GGT GAG GGT GTC GAT GCT TAT ACT ATT GAC  991<br>
Leu Tyr Ala Ser Glu Gly Gly Glu Gly Val Asp Ala Tyr Thr He Asp<br>
170	175	180<br>
ACT GGT ATC AAC ATT GAG CAT GTC GAT TTC GAG GAT CGA GCA CAC TGG  1039<br>
Thr Gly He Asn He Glu His Val Asp Phe Glu Asp Arg Ala His Trp<br>
185	190	195<br>
GGA AAG ACC ATC CCT AGC AAT GAT GAG GAT GCG GAT GGC AAC GGA CAC  1087<br><br>
Gly Lys Thr He Pro Ser Asn Asp Glu Asp Ala Asp Gly Asn Gly His<br>
200	205	210<br>
GGA ACT CAC TGC TCC GGA ACC ATT GCT GGT AAG AAG TAC GGT GTT GCC  1135<br>
Gly Thr His Cys Ser Gly Thr He Ala Gly Lys Lys Tyr Gly Val Ala<br>
215	220	225<br>
AAG AAG GCC AAC ATC TAT GCC GTC AAG GTC TTG AGG TCC AGC GGT<br>
TCT      1183<br>
Lys Lys Ala Asn He Tyr Ala Val Lys Val Leu Arg Ser Ser Gly Ser<br>
230	235	240	245<br>
GGC ACT ATG TCC GAT GTC GTT CTG GGT GTC GAG TGG GCC GTC CAG TCC  1231<br>
Gly Thr Met Ser Asp Val Val Leu Gly Val Glu Trp Ala Val Gin Ser<br>
250	255	260<br>
CAC CTC AAG AAG GCT AAG GAC GCC AAA GAT GCC AAG GTC AAG GGT TTC     1279<br>
His Leu Lys Lys Ala Lys Asp Ala Lys Asp Ala Lys Val Lys Gly Phe<br>
265	270	275<br>
AAG GGC AGC GTT GCC AAC ATG AGT CTT GGT GGT GCC AAG TCC AGG ACC      1327<br>
Lys Gly Ser Val Ala Asn Met Ser Leu Gly Gly Ala Lys Ser Arg Thr<br>
280	285	290<br>
CTT GAG GCT GCT GTC AAT GCT GGT GTT GAG GCT GGT CTT CAC TTC GCC     1375<br>
Leu Glu Ala Ala Val Asn Ala Gly Val Glu Ala Gly Leu His Phe Ala<br>
295	300	305<br>
GTT GCT GCT GGT AAC GAC AAT GCC GAT GCC TGC AAC TAC TCC CCT<br>
GCT  1423<br>
Val Ala Ala Gly Asn Asp Asn Ala Asp Ala Cys Asn Tyr Ser Pro Ala<br>
310	315	320	325<br>
GCC GCT GAG AAT GCC ATC ACT GTC GGT GCC TCG ACC CTT CAG GAT GAG  1471<br>
Ala Ala Glu Asn Ala He Thr Val Gly Ala Ser Thr Leu Gin Asp Glu<br>
330	335	340<br><br>
CGT GCT TAC TTC TCC AAC TAC GGA AAG TGC ACT GAC ATC TTT GCC CCG      1519<br>
Arg Ala Tyr Phe Ser Asn Tyr Gly Lys Cys Thr Asp He Phe Ala Pro<br>
345	350	355<br>
GGT CCC AAC ATT CTT TCC ACC TGG ACT GGC AGC AAG CAC GCT GTC AAC      1567<br>
Gly Pro Asn He Leu Ser Thr Tip Thr Gly Ser Lys His Ala Val Asn<br>
360	365	370<br>
ACC ATC TCT GGA ACC TCT ATG GCT TCT CCT CAC ATT GCT GGT CTG CTG      1615<br>
Thr He Ser Gly Thr Ser Met Ala Ser Pro His He Ala Gly Leu Leu<br>
375	380	385<br>
GCC TAC TTC GTT TCT CTG CAG CCT GCT CAG GAC TCT GCT TTC GCT GTC<br>
1663<br>
Ala Tyr Phe Val Ser Leu Gin Pro Ala Gin Asp Ser Ala Phe Ala Val<br>
390	395	400	405<br>
GAT GAG CTT ACT CCT GCC AAG CTC AAG AAG GAT ATC ATC TCC ATC GCC     1711<br>
Asp Glu Leu Thr Pro Ala Lys Leu Lys Lys Asp He He Ser He Ala<br>
410	415	420<br>
ACC CAG GGT GCC CTT ACT GAT ATC CCA TCT GAC ACC CCC AAC<br>
1753<br>
Thr Gin Gly Ala Leu Thr Asp He Pro Ser Asp Thr Pro Asn<br>
425	430	435<br>
GTAAGTTATA TTATCCATTT TGGTATAATG AAACAGAAAG TGGCTAACTG TTTTATTCTA    1813<br>
G CTT CTC GCC TGG AAC GGC GGT GGT GCC GAC AAC TAC ACC CAG ATT<br>
1859<br>
Leu Leu Ala Trp Asn Gly Gly Gly Ala Asp Asn Tyr Thr Gin He<br>
440	445	450<br>
GTC GCC AAG GGT GGA TAC AAG GCC GGC AGT GAC AAC CTT AAG GAC CGC     1907<br>
Val Ala Lys Gly Gly Tyr Lys Ala Gly Ser Asp Asn Leu Lys Asp Arg<br>
455	460	465<br><br>
TTT GAC GGA CTA GTC AAC AAG GCC GAG AAG TTG CTC GCT GAG GAG CTT      1955<br>
Phe Asp Gly Leu Val Asn Lys Ala Glu Lys Leu Leu Ala Glu Glu Leu<br>
470	475	480<br>
GGA GCT ATT TAC AGT GAG ATC CAG GGT GCT GTT GTT GCA TAG 1997<br>
Gly Ala He Tyr Ser Glu He Gin Gly Ala Val Val Ala *<br>
485	490	495<br>
ATGCAAGACA AGACTTGATT TAGAGTGACG TAACTAGTTT CGTTTATGGC AGGGTATGGG    2057<br>
AATTGGCTAA CCGAACACTG GCGCTGGTAT TTGTTTTTGC TGCTGCTTTT TGGTAACACG   2117<br>
GAGAAGCCGA TGCATTGACT GCATTGGGTA CATTATCCTG ACATGGTTTA CCTGGTCTTT   2177<br>
CATTATTATT ATAGCATACA TGTCCACAAC AATCTTTGAC ATCCTATCTA GAGATACATG   2237<br>
TGCTTGCTTT TAACAGACTG CCAAATCAAT TATGCGACTG TTCTGCACAG ATAATCGTGG    2297<br>
CTTGGTTTGA AGGCTGCCAT AAAGTCTAAC GCTGGCTACC AATTAGGTAG GAGTGTCCCC   2357<br>
TTCCTGCCAG GTTGCTCCAG TCGTAGAAGT AGACTGATAT ATTGAAGATT GCCCATATAC    2417<br>
CATGGACGCT CGTCTTATTC TACATCATAT ATGTCACTCC TAGTGACCAT ATAGACATGC    2477<br>
TAACCATTGC ACAACCCCCC ACAGGTTCAA TCCAACCCAT GACCCCCTCT CATCTTCTGT   2537<br>
TGTATTTTCA GGTTCTAGAT TTGCATACAT ACTACCCATC ATCGGAAGAC GGGTGAGGAG   2597<br>
GCAGATGACC CGACATTATA TTTATTAATT GCTTAGGATG TTTCAACAAC ATTAAAAGTA   2657<br><br>
TATCAATAAG CTTTTCCAGT TTATATTTAC TACCTAAGAT TACGGCATAT AGTGTATTCT   2717<br>
GTGTGCGTAA GAGGTCGCCC TTAAATGGAA ACAGTTCGCG GTTGGAGATA TATATTTGTA   2777<br>
GTGTTCAGGC GGAACGAGTA AAAAAAAAAA AAAATGAGAA GCTGGTGATA TTAACTCCGA   2837<br>
TGTTTATCTT ACATATACCA ATGGATGTAG TCTCATTATA ACGCTTTCTC TGTAGTTTGG   2897<br>
TTGTCATAGA ACTGAATGAC AGGTAAGTGT GTATGTATGT ACAGTACGCA CGGGGGGCCA   2957<br>
TGTGGTCAAC CACACCCAAT GGGCGGTCTT GTCACTTTCC GGACTGGAAA TGAAACGTTC   3017<br>
CATGGAAGAA ATCTGGATGA TTACCTTGAG TACGAGAGAA CTATGGTTGC CGGTAATGGG   3077<br>
TGATTGCCAC AATCATCAGT TCGGTTGAGG CGTTCAACAT CTACGGTACG TTCAGTCACA   3137<br>
TGAATCTGGG AATTCGGGCC TGGTATGCTG GTTTTCGCAA GAGATCCACC CGGCGTGTGC   3197<br>
CAGGTATGCT ACATTTTCTC AGTCGAC	3224<br>
(2) INFORMATION FOR SEQ ID NO: 4:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 496 amino acids<br>
(B)	TYPE: amino acid (D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein<br><br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: A:<br>
Met Arg Gly He Leu Gly Leu Ser Leu Leu Pro Leu Leu Ala Ala Ala<br>
15	10	15<br>
Ser Pro Val Ala Val Asp Ser He His Asn Gly Ala Ala Pro He Leu<br>
20	25	30<br>
Ser Ala Ser Asn Ala Lys Glu Val Pro Asp Ser Tyr He Val Val Phe<br>
35	40	45<br>
Lys Lys His Val Ser Ala Glu Thr Ala Ala Ala His His Thr Trp Val<br>
50	55	60<br>
Gin Asp He His Asp Ser Met Thr Gly Arg He Asp Leu Lys Lys Arg<br>
65	70	75	80<br>
Ser Leu Phe Gly Phe Ser Asp Asp Leu Tyr Leu Gly Leu Lys Asn Thr<br>
85	90	95<br>
Phe Asp He Ala Gly Ser Leu Ala Gly Tyr Ser Gly His Phe His Glu<br>
100	105	110<br>
Asp Val He Glu Gin Val Arg Arg His Pro Asp Val Glu Tyr He Glu<br>
115	120	125<br>
Lys Asp Thr Glu Val His Thr Met Glu Glu Thr Thr Glu Lys Asn Ala<br>
130	135	140<br>
Pro Trp Gly Leu Ala Arg He Ser His Arg Asp Ser Leu Ser Phe Gly<br>
145	150	155	160<br>
Thr Phe Asn Lys Tyr Leu Tyr Ala Ser Glu Gly Gly Glu Gly Val Asp<br>
165	170	175<br>
Ala Tyr Thr He Asp Thr Gly lie Asn He Glu His Val Asp Phe Glu<br>
180	185	190<br>
Asp Arg Ala His Trp Gly Lys Thr He Pro Ser Asn Asp Glu Asp Ala<br>
195	200	205<br>
Asp Gly Asn Gly His Gly Thr His Cys Ser Gly Thr He Ala Gly Lys<br>
210	215	220<br><br>
Lys Tyr Gly Val Ala Lys Lys Ala Asn He Tyr Ala Val Lys Val Leu<br>
225	230	235	240<br>
Arg Ser Ser Gly Ser Gly Thr Met Ser Asp Val Val Leu Gly Val Glu<br>
245	250	255<br>
Trp Ala Val Gin Ser His Leu Lys Lys Ala Lys Asp Ala Lys Asp Ala<br>
260	265	270<br>
Lys Val Lys Gly Phe Lys Gly Ser Val Ala Asn Met Ser Leu Gly Gly<br>
275	280	285<br>
Ala Lys Ser Arg Thr Leu Glu Ala Ala Val Asn Ala Gly Val Glu Ala<br>
290	295	300<br>
Gly Leu His Phe Ala Val Ala Ala Gly Asn Asp Asn Ala Asp Ala Cys<br>
305	310	315	320<br>
Asn Tyr Ser Pro Ala Ala Ala Glu Asn Ala He Thr Val Gly Ala Ser<br>
325	330	335<br>
Thr Leu Gin Asp Glu Arg Ala Tyr Phe Ser Asn Tyr Gly Lys Cys Thr<br>
340	345	350<br>
Asp He Phe Ala Pro Gly Pro Asn He Leu Ser Thr Trp Thr Gly Ser<br>
355	360	365<br>
Lys His Ala Val Asn Thr He Ser Gly Thr Ser Met Ala Ser Pro His<br>
370	375	380<br>
He Ala Gly Leu Leu Ala Tyr Phe Val Ser Leu Gin Pro Ala Gin Asp<br>
385	390	395	400<br>
Ser Ala Phe Ala Val Asp Glu Leu Thr Pro Ala Lys Leu Lys Lys Asp<br>
405	410	415<br>
He He Ser He Ala Thr Gin Gly Ala Leu Thr Asp He Pro Ser Asp<br>
420	425	430<br>
Thr Pro Asn Leu Leu Ala Trp Asn Gly Gly Gly Ala Asp Asn Tyr Thr<br>
435	440	445<br>
Gin He Val Ala Lys Gly Gly Tyr Lys Ala Gly Ser Asp Asn Leu Lys<br><br>
450	455	460<br>
Asp Arg Phe Asp Gly Leu Val Asn Lys Ala Glu Lys Leu Leu Ala Glu<br>
465	470	475	480<br>
Glu Leu Gly Ala He Tyr Ser Glu He Gin Gly Ala Val Val Ala *<br>
485	490	495<br>
(2)	INFORMATION FOR SEQ ID NO: 5;<br>
(i)	SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 5643 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii)	MOLECULE TYPE: DNA (genomic)<br>
(iii)	HYPOTHETICAL: NO<br>
(iii)	ANTI-SENSE: NO<br>
(vi)	ORIGINAL SOURCE:<br>
(A)	ORGANISM; Aspergillus oryzae<br>
(B)	STRAIN: IF04177<br>
(ix)	FEATURE:<br>
(A)	NAME/KEY: intron<br>
(B)	LOCATION: 2701..2769 (ix)      FEATURE:<br><br>
(A)	NAME/KEY; CDS<br>
(B)	LOCATION: join(2282..2700, 2770..4949)<br>
(xi)       SEQUENCE DESCRIPTION: SEQ ID NO: 5:<br>
AAGCTTCGTC CTCGCATCTC GGCCGGGTGA GTAAGGTATG GTATTATTCA TGAAGGGATC      60<br><br>
TCGTTGGTTA CCGTTGTCTA TCCCTAAACA AAGGATTCAA GAGAACAACT CGGAATGCTC     120<br>
CCTCCGCTTA AACCCCTTGA CTCACTGATG GTGTATGTAC TATGGGTACG ACGTTCGGGA     180<br>
TGTGGACTAC CAACCAGAGA GTGATTAGAG AGTCCGGGTT CTCAGTCCAT GATTTTTGCA    240<br>
TCTTTGAAAC AGACGATGCG GAGCGGTCAT TGGCGGAGTT TACTCCCAAA TACGGCCGAA     300<br>
CGGGGTACTT TAAGTGGAAT CTCCGATTTT GGATCTAAGC TCATGAAGGA AAAGTACTAC     360<br>
TAATGCGTAC CTGTGCCTAA TGTTAGTGCT AGTTCGTCTG TTGCATTTTA CCCGTCGGTT    420<br>
AAGACGAATG GATCCGTTCA GGTTTTAAAA TAACTATCTA TGAAATATTT TAGATTTCCC    480<br>
GACATAGTGG TTGGGATGTC TCGATTAACA CTAGGTACAT CAGGCTCAAT TGATTTTGGT     540<br>
TTTAACGAAA CATGATATAG GTCAGGGTCG TGGACCACCC TCCGCCAGGG ATCAGGGGAC     600<br>
GGTTACATGC GAAGGATTCT GATTATATTC ATGATTATGT CAAGCCTTTT CTCTCGTGTG    660<br>
AAGAGGAGCA GAGAATCCGT ACGGGTTTAA TTTAATTTAG CGCCCTGCAG CTTCGAGAAC    720<br>
ATCCCCAGCA ACGTTAAAAA CCACGAGCTA AAATGGGTCG CCACCGGAAG CACTCGAGTC    780<br>
GAGAGATCGG TCGGCTCAGT ATTCGTAATA CCTGCGTTCC AGACGGTTTT GGTCGTTGGT    840<br>
TTCACTCAGG GAACTTAATT CCAGCGGGAC CCAATATAAT TTGAATGATT CATGATACAT     900<br>
CCATTCGTTT GAACCGATCC TGCAAGAGTT CTGTCTGATT TGGTCAACAT AGTTTTCCTC     960<br><br>
TGGGGGAGAC TGGGGAAGAG TCAACACAAT GGTCAGGGAG AGAAGAATGA AAGCTCTCGC    1020<br>
AAGTGGATGA TCATGCTACG TACTGTAGGA ATAAAATTAA TTAATGCGAG GCTGCAAGTA    1080<br>
TCCCTGCGCC GATTTTCTCT TCTTACGGCG GGAACCAAAA AATGTGACGC TGTGATTTTC    1140<br>
TGGAAAAGGT AAGGATGTTT AGTTTCCCAG GATTATTACT GGTTCCGTAT GTGTATGTGT    1200<br>
ATGGATATCA TTCCGTATGG ATACGCCCGT TTCCTCCGCC CAGAACCAGT CCGTCATCCA    1260<br>
TCCTCCACTC TTTCTTCTCT TAGAGCCTTT CCACCTCTCT TCACTTTCTT TTTCTTTCCC    1320<br>
CCCTCCCTCT TTGCTTTCCC TCTCCCAGTA TTATTCTTAT ATTATCGGTT TGACCGTCGC    1380<br>
CTCAGTATCG GCCCCCCGTG AATCACTTTT CGTTTCTCTT GTATTTTACT TTCCTATCTG    1440<br>
GGATTGCTCC TCGATTAGCA GCTCTACTTC ATTCGGCCAT GTGCGTCTAG AGGGTCTAGC    1500<br>
CCCTCTCTCT CTTTGCACTG ACTGTCAGCC ATACCATAGT ATCATCCCGG AATTAAGAAA    1560<br>
AAAAAAGAAA TTATTCTACC TCCGATCTGG ACAAATTATA ACCAGGAGAA AATCAAGCGA    1620<br>
AAGAGGGGCA AAGGAGGAGA CACCATTAAA ACTGGGTCTG GTTTGATTCA TGACATACAT   1680<br>
TCGTCGTCTT GAATTTCAAT AGGTACGGAC TGATGCATTC CACTCGAGCC TTTTTAGCTG    1740<br>
CGTGTCCGTC TCCAATCGCA CTTCTTTTCT TATTTCCTTG TGGGATAAAT TGATTATTTA    1800<br><br>
CCGTTTCGTT TTCTCTATAT TGCGGTGGTG GTGCGACCCA TCCAACTATT ATTATTATAA    1860<br>
TTGGAATTTG ATTTGGATTT TGATTCCTGT GACGGATCTC AGACCAAGTG CCTAAACTAT    1920<br>
AACTGACTTG GACCCCCTTC AGATCCTAGC TTCCCGATTC TTTTCCACCA CTGCTGCATC    1980<br>
CTCTTCCTGC ACGCAGCGTT CGTTTAGGGC GGGTAGACTG GAATTTATTC CTTGCGCCAC    2040<br>
GGACCAATCG CTCCCTCGAC GCTCTCATTC CTGCGTCGAG CTCTTTTTCC CTCGACTCTC    2100<br>
ATTGCTTGCT GGGCTGGTTC TTGAACCTCT TCAATCGTCC TTATCTCTTT CCCCCCATCC   2160<br>
GGCCTGTGAT TCCTATCTTT CCTTTTTTTC TTCCCTTTCT TGTTTGATCC CCCCTCCTCC   2220<br>
CCGTCTTATC GCCTACTATC GTGATCCCCG CCCTTCCCAA TAAAGAGTAG GGCGTGTGAA   2280<br>
C ATG TCC GGG TTA ACC CTC GGG CGA GGC CCT GGG GGC GTG CGA CCG<br>
2326<br>
Met Ser Gly Leu Thr Leu Gly Arg Gly Pro Gly Gly Val Arg Pro<br>
15	10	15<br>
ACT CAA ACC GCA ACT TTT ACC ACC CAC CAC CCG TCC GCC GAT GCT GAC     2374<br>
Thr Gin Thr Ala Thr Phe Thr Thr His His Pro Ser Ala Asp Ala Asp<br>
20	25	30<br>
CGC TCC TCC AAC AAC CTC CCC CCT ACC TCC TCG CAG CTG TCC GAT GAC     2422<br>
Arg Ser Ser Asn Asn Leu Pro Pro Thr Ser Ser Gin Leu Ser Asp Asp<br>
35	40	45<br>
TTT TCT TTC GGT TCC CCT CTG AGC CCC GCC GAC TCA CAG GCC CAT<br>
GAC     2470<br>
Phe Ser Phe Gly Ser Fro Leu Ser Pro Ala Asp Ser Gin Ala His ASD<br><br>
50	55	60<br>
GGC CTA CTT CAG GAC TCC CTC TTC CCT GAA TGG GGG TCT GGT GCG CCT     2518<br>
Gly Leu Leu Gin Asp Ser Leu Phe Pro Glu Tip Gly Ser Gly Ala Pro<br>
65	70	75<br>
CGA CCC GGC ATT GAC AGT CCG GAT GAG ATG CAG AGG CAA GAT CCG CTA  2566<br>
Arg Pro Gly He Asp Ser Pro Asp Glu Met Gin Arg Gin Asp Pro Leu<br>
80	85	90	95<br>
GCG ACT CAA ATA TGG AAG CTC TAT TCT AGG ACC AAG GCC CAG TTG CCC     2614<br>
Ala Thr Gin He Trp Lys Leu Tyr Ser Arg Thr Lys Ala Gin Leu Pro<br>
100	105	110<br>
AAC CAG GAG CGT ATG GAA AAC CTG ACC TGG CGG ATG ATG GCG ATG AGT  2662<br>
Asn Gin Glu Arg Met Glu Asn Leu Thr Trp Arg Met Met Ala Met Ser<br>
115	120	125<br>
TTG AAA CGT AAG GAG CGG GAA CGT GCT CAA CAG TCC AT GTAGGTGTTC  2710<br>
Leu Lys Arg Lys Glu Arg Glu Arg Ala Gin Gin Ser Met<br>
130	135	140<br>
TCCCTCTGTA GAGGAACGGC TGGACCCGCT CATCATTAAT TrrrTTTTTG TCTGTGAAGG  2770<br>
TTT CCT GCG AGA CGC GGT AGC GCT GGC CCC AGT GGT ATC GCT CAA CTG     2818<br>
Phe Pro Ala Arg Arg Gly Ser Ala Gly Pro Ser Gly He Ala Gin Leu<br>
145	150	155<br>
CGC ATT TCC GAC CCG CCC GTT GCC ACC GGT AAC CCT CAG TCA ACC GAC  2866<br>
Arg He Ser Asp Pro Pro Val Ala Thr Gly Asn Pro Gin Ser Thr Asp<br>
160	165	170<br>
CTG ACC GCC GAC CCT ATG AAC CTC GAC GAT TTC ATC GTG CCC TTC GAA  2914<br>
Leu Thr Ala Asp Pro Met Asn Leu Asp Asp Phe He Val Pro Phe Glu<br>
175	180	185<br><br>
TCT CCT TCG GAC CAC CCC TCG CCC AGT GCC GTC AAG ATT TCC GAC TCC     2962<br>
Ser Pro Ser Asp His Pro Ser Pro Ser Ala Val Lys He Ser Asp Ser<br>
190	195	200<br>
ACG GCG TCC GCG GCC ATT CCC ATC AAG TCC CGG AAA GAC CAG CTG<br>
AGA     3010<br>
Thr Ala Ser Ala Ala He Pro He Lys Ser Arg Lys Asp Gin Leu Arg<br>
205	210	215	220<br>
GAT TCT ACC CCG GTG CCG GCC TCG TTC CAC CAT CCG GCT CAG GAT<br>
CAA     3058<br>
Asp Ser Thr Pro Val Pro Ala Ser Phe His His Pro Ala Gin Asp Gin<br>
225	230	235<br>
CGG AAG AAC AGT GAA TTT GGC TAG GTC CCC CGT CGC GTG CGC AAG ACG     3106<br>
Arg Lys Asn Ser Glu Phe Gly Tyr Val Pro Arg Arg Val Arg Lys Thr<br>
240	245	250<br>
AGT ATC GAC GAG CGT CAA TTT TTC TCA CTG CAG GTG CCG ACC CGA AAG  3154<br>
Ser He Asp Glu Arg Gin Phe Phe Ser Leu Gin Val Pro Thr Arg Lys<br>
255	260	265<br>
CGA CCG GCC GAA TCC TCG CCC CAG GTA CCC CCC GTT TCC AAC TCG ATG     3202<br>
Arg Pro Ala Glu Ser Ser Pro Gin Val Pro Pro Val Ser Asn Ser Met<br>
270	275	280<br>
TTG GCC CAC GAT CCG GAC CTC GCT TCC GGC GTG CCC GAT TAT GCC<br>
TTG     3250<br>
Leu Ala His Asp Pro Asp Leu Ala Ser Gly Val Pro Asp Tyr Ala Leu<br>
285	290	295	300<br>
GAC GCC CCG TCC TCG GCC TTT GGC TTC CAT CAG GGT AAC CAC CAT CCG  3298<br>
Asp Ala Pro Ser Ser Ala Phe Gly Phe His Gin Gly Asn His His Pro<br>
305	310	315<br>
GTC AAT CAT CAC AAC CAC ACC TCC CCC GGG GCA CCG TTT GGC TTG GAT     3346<br><br>
Val Asn His His Asn His Thr Ser Pro Gly Ala Pro Phe Gly Leu Asp<br>
320	325	330<br>
ACG TTC GGC CTG GGA GAT GAT CCA ATC TTG CCC TCC GCG GGC CCC TAC  3394<br>
Thr Phe Gly Leu Gly Asp Asp Pro He Leu Pro Ser Ala Gly Pro Tyr<br>
335	340	345<br>
CAG TCG CAA TTC ACC TTC TCA CCC AGC GAG TCT CCG ATG GCC TCC GGT     3442<br>
Gin Ser Gin Phe Thr Phe Ser Pro Ser Glu Ser Pro Met Ala Ser Gly<br>
350	355	360<br>
CAT CCG TTT GCG AAC CTC TAT TCG CAT ACC CCG GTG GCT TCG TCC<br>
CTC  3490<br>
His Pro Phe Ala Asn Leu Tyr Ser His Thr Pro Val Ala Ser Ser Leu<br>
365	370	375	380<br>
AAC TCG ACG GAT TTC TTC TCT CCA CCG CCA TCA GGC TAC CAG TCC ACG     3538<br>
Asn Ser Thr Asp Phe Phe Ser Pro Pro Pro Ser Gly Tyr Gin Ser Thr<br>
385	390	395<br>
GCA TCC ACG CCG CAG CCC ACC TAC GAC GGG GAC CAT TCC GTT TAT TTC      3586<br>
Ala Ser Thr Pro Gin Pro Thr Tyr Asp Gly Asp His Ser Val Tyr Phe<br>
400	405	410<br>
GAT ATG CCG TCG GGC GAC GCG CGC ACC CAG CGC CGC ATT CCG AAC<br>
TAT  3634<br>
Asp Met Pro Ser Gly Asp Ala Arg Thr Gin Arg Arg He Pro Asn Tyr<br>
415	420	425<br>
ATT TCG CAT CGG TCC AAC TTG TCT GCT TCG CTG CAG CCT CGG TAT ATG      3682<br>
He Ser His Arg Ser Asn Leu Ser Ala Ser Leu Gin Pro Arg Tyr Met<br>
430	435	440<br>
TTC AAC CAG AAC AAC CAT GAA CAG GCC AGT TCG TCG ACG GTG CAT<br>
TCG  3730<br>
Phe Asn Gin Asn Asn His Glu Gin Ala Ser Ser Ser Thr Val His Ser<br>
445	450	455	460<br><br>
CCG AGC TAC CCC ATT CCC CAG CCG CAA CAT GTG GAC CCC ACT CAG GTG  3778<br>
Pro Ser Tyr Pro He Pro Gin Pro Gin His Val Asp Pro Thr Gin Val<br>
465	470	475<br>
TTG AAC GCC ACC AAT TAC TCG ACC GGC AAC TCC CAC CAT ACC GGC GCC  3826<br>
Leu Asn Ala Thr Asn Tyr Ser Thr Gly Asn Ser His His Thr Gly Ala<br>
4S0	485	490<br>
ATG TTT TCA TTT GGA GCC GAT TCA GAT AAC GAG GAT GAC GAT GGT CAT     3874<br>
Met Phe Ser Phe Gly Ala Asp Ser Asp Asn GIu Asp Asp Asp Gly His<br>
495	500	505<br>
CAG CTG TCC GAG CGG GCT GGT CTG GCG ATG CCG ACT GAA TAT GGG GAC  3922<br>
Gin Leu Ser Glu Arg Ala Gly Leu Ala Met Pro Thr GIu Tyr Gly Asp<br>
510	515	520<br>
GAG GAC GGG TTC TCG TCG GGC ATG CAG TGG GAT GGG CAG TTC CCG<br>
GGC     3970<br>
Glu Asp Gly Phe Ser Ser Gly Met Gin Trp Asp Gly Gin Phe Pro Gly<br>
525	530	535	540<br>
TCC TTC CAT TCG CTG CCG GGC TTT GGC CCT CAA CAT CGC AAG CAT GTT  4018<br>
Ser Phe His Ser Leu Pro Gly Phe Gly Pro Gin His Arg Lys His Val<br>
545	550	555<br>
ACC ATC GGG TCC ACG GAC ATG ATG GAC ACC CCC GAG GAG TGG AAT CAC  4066<br>
Thr lie Gly Ser Thr Asp Met Met Asp Thr Pro GIu Glu Trp Asn His<br>
560	565	570<br>
GGT GGC AGT TTG GGT CGG ACT CAT GGG TCG GTG GCT TCG GTC AGT<br>
GAG     4114<br>
Gly Gly Ser Leu Gly Arg Thr His Gly Ser Val Ala Ser Val Ser Glu<br>
575	580	585<br>
GTG CGC AAC CGA GAG CAG GAC CCT CGC CGG CAG AAG ATT GCC CGC<br>
ACC  4162<br>
Val Arg Asn Arg Glu Gin Asp Pro Arg Arg Gin Lys He Ala Arg Thr<br><br>
590	595	600<br>
ACG TCC ACC CCC AAT ACG GCC CAG CTG TTG CGC CAA AGC ATG CAC<br>
TCT     4210<br>
Thr Ser Thr Pro Asn Thr Ala Gin Leu Leu Arg Gin Ser Met His Ser<br>
605	610	615	620<br>
AAT AAC AAT ACG TCT CAT ACC TCC CCT AAT ACG CCG CCC GAG TCC GCC  4258<br>
Asn Asn Asn Thr Ser His Thr Ser Pro Asn Thr Pro Pro Glu Ser Ala<br>
625	630	635<br>
CTG AGC AGC GCA GTT CCG TCC CGC CCG GCC AGT CCC GGG GGC AGC AAG  4306<br>
Leu Ser Ser Ala Val Pro Ser Arg Pro Ala Ser Pro Gly Gly Ser Lys<br>
640	645	650<br>
AAC GGC GAC CAA GGC AGC AAC GGA CCG ACC ACC TGC ACG AAC TGC TTC  4354<br>
Asn Gly Asp Gin Gly Ser Asn Gly Pro Thr Thr Cys Thr Asn Cys Phe<br>
655	660	665<br>
ACT CAA ACC ACT CCG CTG TGG CGT CGG AAC CCA GAG GGC CAG CCA CTG     4402<br>
Thr Gin Thr Thr Pro Leu Trp Arg Arg Asn Pro Glu Gly Gin Pro Leu<br>
670	675	680<br>
TGC AAT GCC TGC GGG TTG TTT TTG AAA TTG CAC GGT GTC GTG CGC<br>
CCT     4450<br>
Cys Asn Ala Cys Gly Leu Phe Leu Lys Leu His Gly Val Val Arg Pro<br>
685	690	695	700<br>
CTG TCC CTG AAA ACG GAC GTT ATC AAA AAG CGC AAC CGT AGC AGT GCC     4498<br>
Leu Ser Leu Lys Thr Asp Val He Lys Lys Arg Asn Arg Ser Ser Ala<br>
705	710	715<br>
AAC AGC TTG GCG GTT GGG ACC TCC CGT GCG TCG AAG AAG ACA GCC CGC  4546<br>
Asn Ser Leu Ala Val Gly Thr Ser Arg Ala Ser Lys Lys Thr Ala Arg<br>
720	725	730<br><br>
AAG AAC TCG GTG CAG CAA GCA TCC GTC ACG ACT CCG ACA TCA AGC<br>
CGC  4594<br>
Lys Asn Ser Val Gin Gin Ala Ser Val Thr Thr Pro Thr Ser Ser Arg<br>
735	740	745<br>
GCT CAG AAT GGG ACT TCC TTC GAA TCC CCG CCC GCC GGC TTT AGT GCT     4642<br>
Ala Gin Asn Gly Thr Ser Phe Glu Ser Pro Pro Ala Gly Phe Ser Ala<br>
750	755	760<br>
GCC GCG GGA CGG TCG AAT GGG GTG GTA CCC ATT GCC GCC GCT CCT<br>
CCG  4690<br>
Ala Ala Gly Arg Ser Asn Gly Val Val Pro He Ala Ala Ala Pro Pro<br>
765	770	775	780<br>
AAG GCA GCT CCC TCC GCA GCC GCC TCC CCT AGC ACG GGC CAG ACC CGC  4738<br>
Lys Ala Ala Pro Ser Ala Ala Ala Ser Pro Ser Thr Gly Gin Thr Arg<br>
785	790	795<br>
AAC CCG ATC CAG GCT GCC CCG AAA CGT CAA CGA CGG CTG GAA AAG GCC  4786<br>
Asn Pro He Gin Ala Ala Pro Lys Arg Gin Arg Arg Leu Glu Lys Ala<br>
800	805	810<br>
ACG GAG ATG GAA ACG GAC GAG GCT AAC AAG TCC GCG GGA GGC CGA TCC  4834<br>
Thr Glu Met Glu Thr Asp Glu Ala Asn Lys Ser Ala Gly Gly Arg Ser<br>
815	820	825<br>
AAG GTG GTG CCT CTG GCA CCC GCC ATG CCA CCG GCA GCA GCC AAT CCG  4882<br>
Lys Val Val Pro Leu Ala Pro Ala Met Pro Pro Ala Ala Ala Asn Pro<br>
830	835	840<br>
GCG AAC CAT AGT ATT GCC GGA GGC CAA GGG GCT AGT CAG GAA TGG<br>
GAG  4930<br>
Ala Asn His Ser He Ala Gly Gly Gin Gly Ala Ser Gin Glu Trp Glu<br>
845	850	855	860<br>
TGG TTG ACG ATG AGT CTGTAATGGC CGCGCTTACC TCTCTACTTC TCTACACTCG     4985 Trp Leu Thr Met Ser Leu<br><br>
865<br>
TTTCTTAATA TCTTTCTTGA ACCCCCCCTT ATATTTTCCC ACCGTTGATG CTACGCCATG    5045<br>
ACCGATAGAG ATGATGAATA CTGCAACCAA TGGAATCTCG CTAGACGAGA GGTGTTAGAT    5105<br>
GACGTGGCCC GCGATGCACT TAATGAGATA CGAGGAGGTG CAATGCGTTG GTTACGCTAG    5165<br>
TTTAATGGTA ACATGACGAG GGATATTCGC TCTGTTATTT CGGGCTTTGA TCTGTTTCAG    5225<br>
TCTGCGATTT AACAGCGACT GATCCTCTGC TGTGACAATA CACAGCTTGT CTTGTGGTTC   5285<br>
TGTTGTGGCT TTCTGTTTGT TTGGCTGATT TGATTTATGC TTGATACAAT CGCGTCTGTC   5345<br>
CGGACCCCGG CCTTTGTTTT GTTTTCAGTT CTGATTCTTC ACTGTTTCTG ATTCTCTTGT    5405<br>
TCATGTTTTT GATTTGTTCA AGGCTTGGGG CCGGGCAGAA GTGCGCATCT CTGCTTTGTG   5465<br>
TTTTCCGTCA CCGTGCATAG ACGCTGTATG TATATGCTAC AGCAAGATTC TACTTATCCA    5525<br>
GTCTGAGCCT GTATTCATTG AAGTGTAGCC AGCTGTCGAA TGAGCTTTTT AACGATATTG    5585<br>
TTTTGTTGAG TAGTCAACAA GTAGTATCTG TATATTCCGG AGTCTAAGTA AGACACTT     5643<br>
(2)	INFORMATION FOR SEQ ID NO: 6:<br>
(i)        SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 866 amino acids<br>
(B)	TYPE: amino acid (D)   TOPOLOGY: linear<br><br>
(ii)        MOLECULE TYPE; protein<br>
(xi)       SEQUENCE DESCRIPTION: SEQ ID NO: 6:<br>
Met Ser Gly Leu Thr Leu Gly Arg Gly Pro Gly Gly Val Arg Pro Thr<br>
15	10	15<br>
Gin Thr Ala Thr Phe Thr Thr His His Pro Ser Ala Asp Ala Asp Arg<br>
20	25	30<br>
Ser Ser Asn Asn Leu Pro Pro Thr Ser Ser Gin Leu Ser Asp Asp Phe<br>
35	40	45<br>
Ser Phe Gly Ser Pro Leu Ser Pro Ala Asp Ser Gin Ala His Asp Gly<br>
50	55	60<br>
Leu Leu Gin Asp Ser Leu Phe Pro Glu Trp Gly Ser Gly Ala Pro Arg<br>
65	70	75	80<br>
Pro Gly He Asp Ser Pro Asp Glu Met Gin Arg Gin Asp Pro Leu Ala<br>
85	90	95<br>
Thr Gin He Trp Lys Leu Tyr Ser Arg Thr Lys Ala Gin Leu Pro Asn<br>
100	105	110<br>
Gin Glu Arg Met Glu Asn Leu Thr Trp Arg Met Met Ala Met Ser Leu<br>
115	120	125<br>
Lys Arg Lys Glu Arg Glu Arg Ala Gin Gin Ser Met Phe Pro Ala Arg<br>
130	135	140<br>
Arg Gly Ser Ala Gly Pro Ser Gly He Ala Gin Leu Arg He Ser Asp<br>
145	150	155	160<br>
Pro Pro Val Ala Thr Gly Asn Pro Gin Ser Thr Asp Leu Thr Ala Asp<br>
165	170	175<br>
Pro Met Asn Leu Asp Asp Phe He Val Pro Phe Glu Ser Pro Ser Asp<br>
180	185	190<br>
His Pro Ser Pro Ser Ala Val Lys He Ser Asp Ser Thr Ala Ser Ala<br>
195	200	205<br><br>
Ala Ite Pro He Lys Ser Arg Lys Asp Gin Leu Arg Asp Ser Thr Pro<br>
210	215	220<br>
Val Pro Ala Ser Phe His His Pro Ala Gin Asp Gin Arg Lys Asn Ser<br>
225	230	235	240<br>
Glu Phe Gly Tyr Val Pro Arg Arg Val Arg Lys Thr Ser He Asp Glu<br>
245	250	255<br>
Arg Gin Phe Phe Ser Leu Gin Val Pro Thr Arg Lys Arg Pro Ala Glu<br>
260	265	270<br>
Ser Ser Pro Gin Val Pro Pro Val Ser Asn Ser Met Leu Ala His Asp<br>
275	280	285<br>
Pro Asp Leu Ala Ser Gly Val Pro Asp Tyr Ala Leu Asp Ala Pro Ser<br>
290	295	300<br>
Ser Ala Phe Gly Phe His Gin Gly Asn His His Pro Val Asn His His<br>
305	310	315	320<br>
Asn His Thr Ser Pro Gly Ala Pro Phe Gly Leu Asp Thr Phe Gly Leu<br>
325	330	335<br>
Gly Asp Asp Pro He Leu Pro Ser Ala Gly Pro Tyr Gin Ser Gin Phe<br>
340	345	350<br>
Thr Phe Ser Pro Ser Glu Ser Pro Met Ala Ser Gly His Pro Phe Ala<br>
355	360	365<br>
Asn Leu Tyr Ser His Thr Pro Val Ala Ser Ser Leu Asn Ser Thr Asp<br>
370	375	380<br>
Phe Phe Ser Pro Pro Pro Ser Gly Tyr Gin Ser Thr Ala Ser Thr Pro<br>
385	390	395	400<br>
Gin Pro Thr Tyr Asp Gly Asp His Ser Val Tyr Phe Asp Met Pro Ser<br>
405	410	415<br>
Gly Asp Ala Arg Thr Gin Arg Arg He Pro Asn Tyr He Ser His Arg<br>
420	425	430<br><br>
Ser Asn Leu Ser Ala Ser Leu Gin Pro Arg Tyr Met Phe Asn Gin Asn<br>
435	440	445<br>
Asn His Glu Gin Ala Ser Ser Ser Thr Val His Ser Pro Ser Tyr Pro<br>
450	455	460<br>
He Pro Gin Pro Gin His Val Asp Pro Thr Gin Val Leu Asn Ala Thr<br>
465	470	475	480<br>
Asn Tyr Ser Thr Gly Asn Ser His His Thr Gly Ala Met Phe Ser Phe<br>
485	490	495<br>
Gly Ala Asp Ser Asp Asn Glu Asp Asp Asp Gly His Gin Leu Ser Glu<br>
500	505	510<br>
Arg Ala Gly Leu Ala Met Pro Thr Glu Tyr Gly Asp Glu Asp Gly Phe<br>
515	520	525<br>
Ser Ser Gly Met Gin Trp Asp Gly Gin Phe Pro Gly Ser Phe His Ser<br>
530	535	540<br>
Leu Pro Gly Phe Gly Pro Gin His Arg Lys His Val Thr lie Gly Ser<br>
545	550	555	560<br>
Thr Asp Met Met Asp Thr Pro Glu Glu Trp Asn His Gly Gly Ser Leu<br>
565	570	575<br>
Gly Arg Thr His Gly Ser Val Ala Ser Val Ser Glu Val Arg Asn Arg<br>
580	585	590<br>
Glu Gin Asp Pro Arg Arg Gin Lys He Ala Arg Thr Thr Ser Thr Pro<br>
595	600	605<br>
Asn Thr Ala Gin Leu Leu Arg Gin Ser Met His Ser Asn Asn Asn Thr<br>
610	615	620<br>
Ser His Thr Ser Pro Asn Thr Pro Pro Glu Ser Ala Leu Ser Ser Ala<br>
625	630	635	640<br>
Val Pro Ser Arg Pro Ala Ser Pro Gly Gly Ser Lys Asn Gly Asp Gin<br>
645	650	655<br>
Gly Ser Asn Gly Pro Thr Thr Cys Thr Asn Cys Phe Thr Gin Thr Thr<br>
660	665	670<br><br>
Pro Leu Tip Arg Arg Asn Pro Glu Gly Gin Pro Leu Cys Asn Ala Cys<br>
675	680	685<br>
Gly Leu Phe Leu Lys Leu His Gly Val Val Arg Pro Leu Ser Leu Lys<br>
690	695	700<br>
Thr Asp Val He Lys Lys Arg Asn Arg Ser Ser Ala Asn Ser Leu Ala<br>
705	710	715	720<br>
Val Gly Thr Ser Arg Ala Ser Lys Lys Thr Ala Arg Lys Asn Ser Val<br>
725	730	735<br>
Gin Gin Ala Ser Val Thr Thr Pro Thr Ser Ser Arg Ala Gin Asn Gly<br>
740	745	750<br>
Thr Ser Phe Glu Ser Pro Pro Ala Gly Phe Ser Ala Ala Ala Gly Arg<br>
755	760	765<br>
Ser Asn Gly Val Val Pro He Ala Ala Ala Pro Pro Lys Ala Ala Pro<br>
770	775	780<br>
Ser Ala Ala Ala Ser Pro Ser Thr Gly Gin Thr Arg Asn Pro He Gin<br>
785	790	795	800<br>
Ala Ala Pro Lys Arg Gin Arg Arg Leu Glu Lys Ala Thr Glu Met Glu<br>
805	810	815<br>
Thr Asp Glu Ala Asn Lys Ser Ala Gly Gly Arg Ser Lys Val Val Pro<br>
820	825	830<br>
Leu Ala Pro Ala Met Pro Pro Ala Ala Ala Asn Pro Ala Asn His Ser<br>
835	840	845<br>
He Ala Gly Gly Gin Gly Ala Ser Gin Glu Trp Glu Trp Leu Thr Met<br>
850	855	860<br>
Ser Leu 865<br>
(2)	INFORMATION FOR SEQ ID NO: 7:<br>
(i)        SEQUENCE CHARACTERISTICS:<br><br>
(A)	LENGTH: 27 nucleotides<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii)	MOLECULE TYPE: DNA Primer 19819<br>
(iii)	HYPOTHETICAL; YES<br>
(iii)	ANTI-SENSE: NO<br>
(xi)	SEQUENCE DESCRIPTION: SEQ ID NO: 7<br>
GAAGATCTGC GCGGATGTAC ATTGTAG	27<br>
(2)	INFORMATION FOR SEQ ID NO: 8:<br>
(i)        SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 20 nucleotides<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii)       MOLECULE TYPE: DNA Primer 19821<br>
(iii)       HYPOTHETICAL: YES<br>
(iii)       ANTI-SENSE: NO<br>
ixi) SEQUENCE DESCRIPTION: SEQ ID NO: 8<br>
TTAGTCAGAA ATTCGTCCCG	20<br>
(2)	INFORMATION FOR SEQ ID NO: 9:<br>
(i)        SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 30 nucleotides<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS; single<br>
(D)	TOPOLOGY: linear<br>
62<br><br>
(ii)        MOLECULE TYPE: DNA Primer 19820<br>
(iii)       HYPOTHETICAL: YES<br>
(iii)       ANTI-SENSE: NO<br>
(xi)	SEQUENCE DESCRIPTION: SEQ ID NO: 9<br>
CCCAAGCTTC ATGCTCGACC AGGGCCTCCT	30<br>
(2)	INFORMATION FOR SEQ ID NO: 10:<br>
(i)	SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 21 nucleotides<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii)       MOLECULE TYPE: DNA Primer 19818<br>
(iii)      HYPOTHETICAL: YES<br>
(iii)       ANTI-SENSE: NO<br>
[xi) SEQUENCE DESCRIPTION: SEQ ID NO: 10<br>
GGTCTGTGTT AACCAAAGAA C	21<br>
(2)	INFORMATION FOR SEQ ID NO: 11:<br>
(i)        SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 14 nucleotides<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii)       MOLECULE TYPE: DNA Primer 8681<br>
(iii)       HYPOTHETICAL: YES<br>
(iii)      ANTI-SENSE: NO<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 11<br>
63<br><br>
GATCCACCAT GAAG<br>
(2)	INFORMATION FOR SEQ ID NO: 12:<br>
(i)	SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 14 nucleotides<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii)        MOLECULE TYPE: DNA Primer 8747<br>
(Hi)      HYPOTHETICAL: YES<br>
(iii)       ANTI-SENSE: NO<br>
(xi)	SEQUENCE DESCRIPTION: SEQ ID NO: 12<br>
GTGGTACTTC AGCT<br><br><br>
We claim:<br>
1.	An Aspergillus, wherein the areA gene by recombinant DNA technology has been modified in a way by which it cannot be expressed in a way providing for a functional AreA activator, and wherein the genes encoding for the extracellular proteases PepC and/or PepE have been inactivated in a manner whereby they are not expressed to produce functional proteases.<br>
2.	The Aspergillus as claimed in claim 1, wherein said inactivation has been obtained by deletion of all or parts of the areA, pepC, and/or pepE genes.<br>
3.	The Aspergillus as claimed in claim 1, wherein said inactivation has been obtained by deletion of all or parts of the areA, andpepE genes.<br>
4.	The Aspergillus as claimed in claim 1, wherein said inactivation has been obtained by interfering with the regulation of the expression signals regulating the expression of the areA,pepC, and/or pepE genes themselves.<br>
5.	The Aspergillus as claimed in claim 1, wherein said inactivation has been obtained by interfering with the regulation of the expression signals regulating the expression of the areA, andpepE genes themselves.<br>
6.	The Aspergillus as claimed in claim 1, wherein said inactivation has been obtained by using anti-sense technology.<br>
7.	The Aspergillus as claimed in claim 1, wherein said inactivation has been obtained by inserting extra DNA internally in the areA, pepC, and/or pepE genes.<br><br>
8.	The Aspergillus as claimed in claim 1, which belongs to a species selected from the group comprising A. oryzae, A. niger, A. awamori, A. phoenicis, A. japonicus, A. foetidus, A. nidulans, T. reesei, T. harzianum, H. insolens, H. lanuginosa, F. graminectrum, F. solani, P. chrysogenum, and others.<br>
9.	A method for producing the Aspergillus as claimed in claim 1, wherein said inactivation has been obtained by deletion of the areA, pepC, and/or pepE genes, which method comprises<br>
i)	cloning of the areA,pepC, and/or pepE genes from a Aspergillus of interest,<br>
ii)	producing DNA constructs each comprising one among the areA gene, the<br>
pepC gene, and/or the pepE gene, wherein an internal part has been<br>
substituted, deleted, or extra DNA has been inserted,<br>
iii)	transforming said Aspergillus with the constructs, and<br>
iv)	isolating transformants which are areA',pepC, and/or pepE.<br>
10.	A method for producing the Aspergillus as claimed in claim 1, wherein said<br>
inactivation has been obtained by deletion of the areA and pepE genes, which method<br>
comprises<br>
i)	cloning of the areA zndpepE genes from an Aspergillus of interest,<br>
ii)	producing DNA constructs each comprising one among the areA gene and<br>
the pepE gene, wherein an internal part has been substituted, deleted, or<br>
extra DNA has been inserted,<br>
iii)	transforming said Aspergillus with the constructs, and<br>
iv)	isolating transformants which are areA', andpepE.<br><br>
11.	A method for producing the Aspergillus as claimed in claim 1, wherein<br>
said inactivation has been obtained by deletion of the areA and pepC genes, which<br>
method comprises<br>
i)	cloning of the areA andpepC genes from an Aspergillus of interest,<br>
ii)	producing DNA constructs each comprising one among the areA gene and<br>
the pepC gene, wherein an internal part has been substituted, deleted, or<br>
extra DNA has been inserted,<br>
iii)	transforming said Aspergillus with the constructs, and<br>
iv)	isolating transformants which are areA', mdpepC.<br>
12.	A method for producing the Aspergillus as claimed in claim 1, wherein<br>
said inactivation has been obtained by deletion of the pepC and pepE genes, which<br>
method comprises<br>
i)	cloning of the pepC and pepE genes from an Aspergillus of interest,<br>
ii)	producing DNA constructs each comprising one among the pepC gene and<br>
the pepE gene, wherein an internal part has been substituted, deleted, or<br>
extra DNA has been inserted,<br>
iii)	transforming said Aspergillus with the constructs, and<br>
iv)	isolating transformants which arepepC, andpepE'.<br>
13.	A method for producing the Aspergillus as claimed in claim 1, wherein said<br>
inactivation has been obtained by using anti-sense technology, which method comprises<br>
i)	construction of expression plasmids, each of which give rise to synthesis of<br>
an RNA molecule complementary to the mRNA transcribed from the areA gene, the pepC gene, and/or the pepE gene,<br>
ii)	transformation of the host Aspergillus with said expression plasmids and a<br>
suitable marker, either on separate plasmids or on the same plasmid,<br><br>
iii)	selection of transformants using said marker, and<br>
iv)	screening selected transformants for strains exhibiting a reduction in the<br>
synthesis of the AreA, PepC, and/or PepE products.<br>
14.	A method for producing the Aspergillus as claimed in claim 1, wherein said<br>
inactivation has been obtained by using anti-sense technology, which method comprises<br>
i)	construction of expression plasmids, each of which give rise to synthesis of<br>
an RNA molecule complementary to the mRNA transcribed from the areA<br>
gene and the pepE gene,<br>
ii)	transformation of the host Aspergillus with said expression plasmids and a<br>
suitable marker, either on separate plasmids or on the same plasmid,<br>
iii)	selection of transformants using said marker, and<br>
iv)	screening selected transformants for strains exhibiting a reduction in the<br>
synthesis of the AreA, PepC, and PepE products.<br>
15.	A process for the production of a desired gene product, whereby an Aspergillus as claimed in any of the claims 1 to 8 is cultivated in a suitable growth medium at appropriate conditions and the desired gene product is recovered and purified.<br>
16.	A process for the production of a desired gene product, whereby an the Aspergillus as claimed in any of the claims 1 to 8, which has been transformed to integrate a DNA sequence coding for the desired gene product into the genome of the Aspergillus in a functional manner, is cultivated in a suitable growth medium at appropriate conditions and the desired gene product is recovered and purified.<br>
17.	A process for producing a desired polypeptide comprising cultivating an Aspergillus in an appropriate growth medium and recovering said polypeptide from said<br><br>
culture, said Aspergillus carrying a recombinant DNA construct capable of causing expression of said polypeptide or a precursor thereof in said Aspergillus, said Aspergillus further being characterized by producing lower amounts of functional AreA, PepC, and/or PepE than the wild-type of said Aspergillus.<br>
18.	A method as claimed in claim 17, wherein said Aspergillus has been modified to produce lower than wild-type amounts of AreA, PepC, and/or PepE by a process comprising transforming a parent of said Aspergillus with DNA constructs capable of causing reduced production of functional AreA, PepC, and/or PepE when integrated in the genome of said Aspergillus.<br>
19.	A method as claimed in claim 17, wherein said polypeptide is secreted to the extracellular medium by said fungus.<br>
20.	A method according to claim 17, wherein said Aspergillus produces higher amounts of said polypeptide than a similar Aspergillus where said similar Aspergillus produces AreA, PepC, and/or PepE in amounts similar to those produced by the wild-type of said Aspergillus, said similar Aspergillus being identical to said Aspergillus in all other respects.<br>
21.	The process as claimed in claim 13 or 14 to 20, wherein said gene product is a secreted protein.<br>
22.	The process as claimed in any of the claims 13 to 19, wherein said desired gene product is an industrial peptide or protein, preferably an enzyme.<br><br>
23.	The process as claimed in claim 22, wherein said enzyme is selected from the group comprising a protease, lipase, cutinase, cellulase, chymosin.<br>
24.	The process as claimed in any of the claims 14 to 21, wherein said desired gene product is a therapeutically active peptide or protein.<br>
25.	The process as claimed in claim 24, wherein said therapeutically active peptide or protein is selected from the group comprising insulin, growth hormone, glucagon, somatostatin, interferon, PDGF, factor VII, factor VIII, urokinase, tPA, EPO, orTPO.<br>
26.	A gene product produced in accordance with any of the processes 14 to 25.<br>
27.	A DNA sequence coding for the pepC gene from A. oryzae (SEQ ID No. 1) or functional alleles thereof.<br>
28.	A PepC protease from A. oryzae (SEQ ID No. 2).<br>
29.	A process for the production of the PepC protease as claimed in claim 28 comprising transforming a suitable host with a DNA construct comprising a DNA sequence as claimed in claim 27, selecting a transformant capable of producing said PepC protease, cultivating said transformant in an appropriate growth medium and recovering said PepC protease from said culture.<br>
30.	A DNA sequence coding for thepepE gene from A. oryzae (SEQ ID No. 3) or functional alleles thereof.<br><br>
31.	A PepE protease from A. oryzae (SEQ ID No. 4).<br>
32.	A process for the production of the PepE protease as claimed in claim 31 comprising transforming a suitable host with a DNA construct comprising a DNA sequence as claimed in claim 30, selecting a transformant capable of producing said PepE protease, cultivating said transformant in an appropriate growth medium and recovering said PepE protease from said culture.<br>
33.	The process as claimed in claim 29 or 32, wherein said host is aa Aspergillus as claimed in any of the claims 1 to8.<br>
34.	The process as claimed in claim 29, 32 or 33, wherein said host is A. oryzae, and wherein said DNA construct comprising a DNA sequence as claimed in claim 27 or 30, provides for an extra copy of the gene encoding said PepC or PepE protease.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI4Ny1tYXMtMTk5NiBhYnN0cmFjdC1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">2287-mas-1996 abstract-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI4Ny1tYXMtMTk5NiBhYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">2287-mas-1996 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI4Ny1tYXMtMTk5NiBhc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">2287-mas-1996 assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI4Ny1tYXMtMTk5NiBjbGFpbXMtZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">2287-mas-1996 claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI4Ny1tYXMtMTk5NiBjbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">2287-mas-1996 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI4Ny1tYXMtMTk5NiBjb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">2287-mas-1996 correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI4Ny1tYXMtMTk5NiBjb3JyZXNwb25kZW5jZS1wby5wZGY=" target="_blank" style="word-wrap:break-word;">2287-mas-1996 correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI4Ny1tYXMtMTk5NiBkZXNjcmlwdGlvbiAoY29tcGxldGUpLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">2287-mas-1996 description (complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI4Ny1tYXMtMTk5NiBkZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">2287-mas-1996 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI4Ny1tYXMtMTk5NiBkcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">2287-mas-1996 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI4Ny1tYXMtMTk5NiBmb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">2287-mas-1996 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI4Ny1tYXMtMTk5NiBmb3JtLTE5LnBkZg==" target="_blank" style="word-wrap:break-word;">2287-mas-1996 form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI4Ny1tYXMtMTk5NiBmb3JtLTI2LnBkZg==" target="_blank" style="word-wrap:break-word;">2287-mas-1996 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI4Ny1tYXMtMTk5NiBmb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">2287-mas-1996 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI4Ny1tYXMtMTk5NiBmb3JtLTQucGRm" target="_blank" style="word-wrap:break-word;">2287-mas-1996 form-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI4Ny1tYXMtMTk5NiBmb3JtLTYucGRm" target="_blank" style="word-wrap:break-word;">2287-mas-1996 form-6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI4Ny1tYXMtMTk5NiBwZXRpdGlvbi5wZGY=" target="_blank" style="word-wrap:break-word;">2287-mas-1996 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="224829-purified-polypeptide-having-a-receptor-activity-specific-for-il-13.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="224831-3-amidochromanysulfonyl-thio-ureas-of-formula-i.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>224830</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2287/MAS/1996</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>49/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>05-Dec-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>23-Oct-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>17-Dec-1996</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVOZYMES A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>KROGSHOEJVEJ 36, DK-2880 BAGSVAERD,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>TOVE CHRISTENSEN</td>
											<td>NOVO ALLE, DK-2880 BAGSVAERD,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>JAN LEHMBECK</td>
											<td>NOVO ALLE, DK-2880 BAGSVAERD,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/64</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/224830-an-aspergillus-which-do-not-produce-proteases by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:34:26 GMT -->
</html>
