<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/224903-a-small-synthetic-rna-a-method-of-preparing-the-same-and-uses-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:12:36 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 224903:A SMALL SYNTHETIC RNA, A METHOD OF PREPARING THE SAME AND USES THEREOF</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A SMALL SYNTHETIC RNA, A METHOD OF PREPARING THE SAME AND USES THEREOF</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Translation of the hepatitis C virus (HCV) RNA is mediated by the interaction of ribosomes and cellular proteins with an internal ribosome entry site (IRES) located within the 5&#x27;untranslated region (5&#x27;UTR). We have investigated whether small RNA molecules corresponding to the different stem-loop (SL) domains of the HCV IRES, when introduced in tram, can bind to the cellular proteins and antagonize their binding to the viral IRES, thereby inhibiting HCV IRES-mediated translation. We have found that an RNA molecule corresponding to SL III of the HCV IRES could efficiently inhibit HCV IRES-mediated translation in a dose-dependent manner without affecting cap-dependent translation. The SL III RNA was also found to bind efficiently to most of the cellular proteins which interacted with the HCV 5&#x27;UTR. A smaller RNA corresponding to SL e+f of domain III also strongly and selectively inhibited HCV IRES-mediated translation. This RNA molecule showed strong interaction with the ribosomal S5 protein and prevented the recruitment of the 40S ribosomal subunit by the HCV IRES. In conclusion our results demonstrate a novel approach to selectively block HCV RNA translation using a small RNA molecules mimicking the structure of the stem-loop IIIe+f subdomain of the HCV-IRES. The discovery provides a basis for developing a potent antiviral therapy targeting the interaction between the ribosome and the HCV-IRES RNA.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
This invention relates to a small synthetic ribonucleic acid (RNA) sequence and to method of preparing the same and use of the said sequence as a therapeutic molecule against hepatitis C induced liver cirrhosis and hepato cellular carcinoma.<br>
Prior art:<br>
HCV (Hepatitis C virus) is a major human pathogen with an estimated 170 million chronic carriers throughout the world, many of whom are at a significant risk for developing liver cirrhosis and hepatocellular carcinoma (1). Current therapeutic strategies using Interferon-a, either alone or in combination with ribavirin, have poor efficacy (2). Moreover, an important lesson obtained from the therapy of other viral infections is that multiple drug targets are required to prevent the emergence of drug-resistant varieties of the virus (3).<br>
HCV is a positive stranded RNA virus and the translation of the viral genomic RNA is an early obligatory step of the infection process. The translation initiation of the uncapped viral RNA takes place through the IRES (internal ribosome entry site) located in the 5'UTR (5'-untranslated region) (4). Translation initiation from the IRES is mediated by a number of cellular trans-acting factors like the La autoantigen (5) and polypyrimidine tract-binding protein (PTB) (6) together with some canonical elFs (eukaryotic Initiation Factors) like eIF3 (7), eIF2 and eIF2B (8). It has been suggested that the IRES acts as a structural scaffold with specifically placed recognition sites for recruiting the translation machinery (9). As this mechanism of translation initiation is distinct from the cellular cap-dependent mechanism, it is an attractive target for antiviral therapeutics with high target specificity and low host cytotoxicity (10).<br>
The concept of using RNA molecules as therapeutic agents has aroused increasing interest in the past decade. Antisense RNAs and trans-cleaving ribozymes have been studied as potential inhibitors of HCV translation (11, 12). Recently, RNA interference using small interfering RNAs has proved to be highly efficient in inhibiting the replication of a number of viruses including HCV (3).<br><br>
Another strategy of blocking the replication of RNA viruses has been to express small and structured viral RNA elements in target cells. These RNAs bind to viral regulatory proteins and prevent their binding to corresponding sequences in the viral RNA thus inhibiting viral gene expression. Overexpression of Human Immunodeficiency Virus (HIV) trans-activation response region and Rev response element RNAs in CD4+ T cells prevented the binding of viral Tat and Rev proteins to the viral RNA and made the cells resistant to HIV replication (13, 14). Also, a 60 nt RNA termed IRNA, isolated from Saccharomyces cerevisae, has been shown to block IRES-mediated translation of poliovirus and HCV by sequestering cellular trans-acting factors which interact with the viral IRESs (15, 16). La and PTB-specific SELEX RNA have also been shown to inhibit HCV IRES-mediated translation (17,18).<br>
The limitations of the above -referred known procedures to inhibit the HCV translation/replication is the non-specificity of the procedures and also the possibility of the generation of escape mutants of the virus. Any sequence based approach like RNA interference or antisense has the possibility of generating resistant varieties of the virus because of increased selection pressure. Also, there is a possibility of the function of certain cellular genes being affected.<br>
Herein, a novel approach to inhibit HCV IRES-mediated translation using small RNA molecules rnimicking the structure of the defined SL (stem-loop) domains of the HCV IRES is being demonstrated.. It has been shown that a small RNA molecule corresponding to domain HI of the HCV IRES and a smaller molecule corresponding to the SL HI e+f subdomain binds to specific cellular proteins interacting with the HCV IRES and strongly inhibit HCV IRES-mediated translation without inhibiting cap-dependent translation.<br>
Therefore, the specific inhibition of HCV IRES-mediated mechanism by the SL HI e+f RNA of the HCV 5*UTR has a potential to be used as a therapeutic strategy with a number of associated advantages. Firstly, as the RNA-protein interactions between host cellular proteins and a highly conserved region of the viral RNA are targeted, the chance of generation of viral escape mutants is very low. Any sequence-specific antiviral<br><br>
molecule would exert a selection pressure for the generation of escape variants. On the other hand, a strategy targeting the interaction between host proteins and the viral RNA would not exert a strong selection pressure on the viral genome as the rate of mutation of cellular proteins is entirely independent of viral replication. Moreover, the activity of the HCV IRES being highly structure dependent, only mutational events which can alter the structure in such a way as to enable it to interact with a different protein with the same function or which increases the affinity to an existing interacting partner would allow the virus to circumvent inhibition mediated by this approach. Secondly, the RNA molecule being a part of the viral genome, if administered prophylactically to patients already harbouring the viral RNA, it is not expected to give rise to non-specific immune responses as seen in the case of antisense RNAs (Dove, 2002). Thirdly, as the inhibitory activity is dependent on the RNA structure, stabler derivatives using different nucleotide (eg. phosphorothioate) or backbone (eg. PNA) chemistries and small molecule structural analogs of the RNA can be utilized to increase the deliverability and stability of the therapeutic molecule.<br>
Accordingly, it is the primary object of the present invention to investigate whether small RNA molecules corresponding to the different stem-loop (SL) domains of the HCV IRES, when introduced in trans, can bind to the cellular proteins and antagonize their binding to the viral IRES, thereby inhibiting HCV IRES-mediated translation.<br>
It is another object of the present invention to identify the RNA molecule corresponding to a sub-domain of SL m region of the HCV IRES, which could efficiently inhibit HCV IRES-mediated translation in a dose-dependent manner without affecting cap-dependent translation.<br>
A further object of the present invention is to provide a small synthetic ribonucleic acid sequence GGGA GGGC CCTCTCG GTAGA ACACCA TGACGGA CTATCCCACGAACGCTCACGGGGCCCTCC, its structural analog or mimic thereof.<br>
A further object of the present invention is to provide a therapeutic molecule/composition, which will be useful in the treatment of hepatitis C induced liver cirrhosis and hepato cellular carcinoma and a method of preparing the said composition.<br><br>
Further objects of the present invention are providing a polynucleotide, a recombinant vector, a method of preparing the same using the synthetic HCV IRES ribonucleic acid and a method for inhibiting HCV IRES mediated translation.<br>
The term "structural analogs" can be described as RNAs that are structurally and functionally similar but the nucleotide composition and origin (derived from) could be different.<br>
"Macromolecular mimicry" can be described as a phenomenon, that large parts of proteins (domain or sub domains) resemble in shape (and possibly also in function) a large part of a nucleic acid. The structural mimic can also be extended to materials other then proteins (e.g. compounds). Accordingly, the term "structural mimic" in this specification can be described as substances/compounds that resemble in shape (and possibly also in function) a large part of the RNA structure.<br>
Summary of invention:<br>
HCV is a positive stranded RNA virus and the translation of the viral genomic RNA to produce the proteins required for replication is an early obligatory step of the infection process. The translation initiation of the uncapped viral RNA takes place through the highly structured Internal Ribosome Entry Site (IRES) located in the 5'UTR of the viral RNA. The process of IRES-mediated translation is an attractive target for designing antiviral therapeutics with high target specificity and low host cytotoxicity because of its fundamental difference from the cap-dependent translation of cellular proteins and its importance in the viral life cycle.<br>
The cap independent internal initiation of translation is mediated by the interaction between the cellular trans acting factors with the cis acting elements within the HCV 5'UTRRNA.<br>
The present invention proposes a molecular approach to inhibit HCV IRES-mediated translation by using small RNA molecules analogous to the structure of the defined stem-loop (SL) domains of the HCV 5'UTR (IRES element). These RNA molecules, when introduced in trans, were expected to specifically bind to the cellular proteins that interact<br><br>
with the viral IRES and are required for internal initiation of translation. As a result of this sequestration, the cellular proteins would be prevented from binding to die IRES of the viral RNA and this may lead to the inhibition of viral protein synthesis.<br>
When the above proposal was tested experimentally, it has been observed that a small RNA molecule corresponding to domain HI of the HCV IRES and a smaller molecule corresponding to the SL HI e+f subdoraain binds to specific cellular proteins interacting with the HCV IRES and strongly inhibit HCV IRES-mediated translation without inhibiting cap-dependent translation in vitro and in vivo in cultured cells. Further, the SLIII e+f RNA was found to interact with the 40 S ribosomal subunit protein S5. As the S5 protein of the 40S subunit is crucial for ribosome binding to the HCV IRES, binding of me SL m e+f RNA to me S5 protein blocked me ribosome interaction with the HCV IRES and resulted in the inhibition of translation initiation {Ray and Das, 2004, Nucleic Acids Res).<br>
These observations helped to conclude that a small RNA molecule analogous to a specific stem-loop structure of the HCV IRES can be used to selectively inhibit HCV IRES-mediated translation. Small molecule structural analogs of the SL III e+f RNA, that prevent the binding of ribosomes to the HCV IRES by interacting with the S5 protein, may be developed, which might act as potent inhibitors of HCV translation and may act as novel antivirals to combat HCV infection.<br>
BRIEF DESCRIPTION OF THE ACCOMPANYING DRAWINGS:<br>
Fig. 1: Dose-dependent inhibition of HCV IRES-mediated translation in vitro by SL III RNA.<br>
(A) Proposed secondary structure of the HCV Ires RNA spanning nucleotides 40-372 of the 5'UTR of the viral RNA. The domains that were PCR amplified and cloned to generate small RNAs are delineated. (B) 100-fold and 200-fold molar excess of in vitro transcribed SL II EI and IV RNAs were added to in vitro translation reactions of HCV vicistronic RNA. 5ul of the translation reactions was resolved on SDS-12.5% PAGE and exposed for phosphorimaging. The flue and Rluc protein products are indicated by<br><br>
arrows. (C) The percent Flue activity representing the efficiency of HCV IRES-mediated translation from a HCV bicistronic template in presence of six increasing concentrations of SL n, IH and IV RNAs. The Flue activity at each concentration is represented as a percentage of the control reaction (expressed as 100%). (D) The percent Rluc activity representing the efficiency of cap-dependent translation from the same set of experiments were plotted. The Rluc acitivity at each concentration is represented as a percentage of the control reaction. The translation efficiency was not reduced to below 50% by either SL m or SL H.<br>
Fig. 2: Effect of SL III RNA on HCV IRES-mediated translation in vivo.<br>
(A) Three-way co-transfections were performed in HeLa cells using pRL-CMV, pCD-HCV5'UTR-Fluc and two concentrations of pCD-SL m and pCD-SL H DNAs. DNA quantity per dish was normalized by transfecting pGEM-3Z DNA. The black bars represent Flue activity (HCV IRES-mediated translation) whereas the gray bars represent Rluc activity (cap-dependent translation). (B) The same experiment was repeated in the Huh7 cell line. Combined data from three independent experiments in each cell line are shown. Luciferase activity in control reactions is expressed as 100%. Values which significantly differ from controls (P
Figure 3: Constitutive expression of SL III RNA does not cause general inhibition of cellular transcription and translation.<br>
(A) Huh7 cells were transfected with an eukiaryotic expression vector encoding the SL III RNA, upstream of a ribozyme sequence (schematically represented), which generated the SL IH RNA in vivo. (B) The cells were selected by G418 selection to generate a cell line constitutively expressing the SL m RNA. RT-PCR of total RNA isolated from untransfected Huh7 cells and cells stably transfected with the pCD-SL m-Ribo DNA using SL El-specific or actin-specific primers. The amplified products are indicated. (C) in vivo metabolic labelling of Huh7 and Huh7-SLffl cells using 35S-MET/35S-Cys. The cells were lysed after labelling and the cell lysates resolved by SDS-10%PAGE.<br><br>
Figure 4: Specific inhibition of HCV IRES-mediated translation in vitro by SL III<br>
e+fRNA.<br>
(A) Proposed secondary structure of HCV IRES (internal ribosome entry site) domain in (121-315nt), delineating the SL structures, which were generated by oligonucleotide-driven transcription. (B) Schematic representation of the process of oligonucleoude-driven transcription of the HCV IRES SL RNAs using synthetic oligonucleotide templates (C) The percent Flue activity representing the efficiency of HCV IRES-mediated translation from a HCV bicistronic template in presence of five increasing concentrations of SL III a+c, b, d and e+f RNAs was plotted. Luciferase activity in control reactions is expressed asl00%. (D) The percent Rluc activity representing the efficiency of cap-dependent translation from the same set of experiments was plotted. The reporter gene activity at each concentration is represented as a percentage of the control reaction.<br>
Figure 5: Effect of SL III e+f RNA on HCV IRES-mediated translation and replication in vivo.<br>
Huh 7 cells were co-transfected with 6 ug of in vitro transcribed capped HCV bicistronic RNA and two concentrations (6 and 12 ug) of either SL HI e+f RNA or SL HI d RNA. The RNA quantities in each dish were normalized by adding appropriate amounts of an in vitro transcribed RNA corresponding to the polylinker sequence of the pGEM 3Z plasmid. The black bars represent Flue activity whereas the gray bars represent Rluc activity. Luciferase activity in control reactions is expressed as 100%. Values which significantly differ from controls (P
Figure 6: SL III e+f (A297GF) RNA fails to bind to S5 ribosomal protein and does not inhibit HCV IRES-mediated translation.<br>
In the results Fig 5A and B seem to indicate binding of HeLa lysate and purified S5 protein with all the small RNAs. Please check.<br>
(A) Representation of the SL III e+fRNA showing the muation of A297 to G. (B) 32P-labeled RNAs corresponding to SL III e+f and SL m e+f (A297G) were UV-crosslinked<br><br>
to HeLa S10 extract and digested with RNase A. The nucleoprotein complexes were resolved by SDS-15% PAGE and the position of p25 is indicated. (C) The same RNAs were UV-crosslinked to purified S5 ribosomal protein and the nucleoprotein complexes were resolved by SDS-15% PAGE. (D) 100-fold and 200-fold molar excess of in vitro transcribed SL m e+f (A297G) RNA was added to in vitro translation reactions of HCV bicistronic RNA and luciferase activity was assayed. The black bars represent Flue activity (HCV IRES-mediated translation) whereas the gray bars represent Rluc activity (Cap-dependent translation). Luciferase activity in control reactions is expressed as 100%. Combined data from three independent experiments is represented.<br>
Figure 7: SL ID e+f prevents 40S ribosomal subunit recruitment by the HCV IRES.<br>
(A) Sucrose gradient sedimentation profiles of 32P-UTP-labeled HCV 5'UTR RNA incubated in RRL in absence and presence of 200 fold excess of unlabeled SL III e+f RNA. (B) Sedimentation profile of radiolabeled HCV 5'UTR RNA in presence of 2mM GMP-PNP and in the presence or absence of 200 fold excess of unlabeled SL HI e+f RNA. The filled circles represents the control reaction profile and the open circles show the profile in the presence of SL III e+f. Both profiles show the counts per minute as a percentage of the total counts added to the reaction (~105 cpm) against the fraction number of the gradient. The fractions were collected from the bottom up wards. The SOS and 48S ribosomal peaks are indicated.<br>
Figure 8: SL III e+f does not prevent ribosome recruitment by a capped RNA and binds directly to the 40S subunit.<br>
(A) Sucrose density gradient sedimentation profiles of 32P-UTP-labeled capped-GFP RNA inclubated in RRL in absence and presence of 200 fold excess of unlabeled SL EI e+f RNA. (B) Sedimentation profile of radiolabeled HCV 5'UTR-GEP RNA in presence or absence of 200 fold excess of unlabeled SL III e+f RNA. The filled circles represents the control reaction profile and the open circles show the profile in the presence of SL m e+f. (C) Sucrose density gradient sedimentation profile of 32P-UTP-labeled SL m e+f RNA incubated in RRL. All the profiles show the counts per minute as a percentage of the total counts added to the reaction (~105 cpm) against the fraction number of the<br><br>
gradient. The fractions were collected from the bottom upwards. The 48S ribosomal peak and the polysomal peaks are indicated.<br>
Figure 9: Proposed model of inhibition of HCV IRES-mediatcd translation by SL IHe+fRNA.<br>
The HCV IRES binds to the 40S ribosomal subunit as demonstrated by Spahn et al., 2001, making contact with the S5 protein (shaded in red) via SL m e and f and SL II. When SL HI e+f RNA is added in trans, it binds to the S5 protein on the 40S subunit (shown in blue) and blocks this crucial binding site for the HCV IRES RNA. This prevents ribosome recruitment by the HCV IRES and thereby inhibits HCV IRES-mediated translation. The 40S subunit, HCV IRES and SL HI e+f RNA representations are not to scale.<br>
Preparation of the inhibitor RNA:<br>
OUgonucleotide-driven transcription ofSUIIe+fRNA<br>
The inhibitor RNA was synthesized by in vitro transcription assay using standard protocol described earlier (15) and also in the Promega Protocol. Briefly, synthetic DNA oligonucleotides corresponding to domain HI stem-loops e+f structures with T7 promoter sequences at the 5' end (GGGAGGGCCCTCTCGGTAGAACACCATGACGGACTATCCCACGAACGCTCA GGGGCCCTCC) was obtained from Sigma Aldrich (St. Louis, MO). The oligonucleotide was annealed to T7 RNA polymerase promoter primers and transcribed in vitro using T7 RNA polymerase as described earlier. The transcription reaction was extracted with phenol and chloroform. The synthesized RNA was purified and concentrated by alcohol precipitation. The RNA pellet was dried in vaccum centrifuge and dissolved in nuclease free water.<br>
The SL Ille+f (A297G) oligo had the same sequence as the SL III e+f oligo except for a T residue replaced by C at the 54th position.  Radioactively labeled RNAs were<br>
"if<br>
synthesized similarly using the same templates and a-   P-UTP.<br><br>
In order to express the inhibitor RNA in vivo from eukaryotic expression vector, the oligonucleotide DNA sequences corresponding to the inhibitor RNA (SLIII e+f) was annealed with the antisense oligonucleotide to make double stranded DNA and then cloned into pCDNA3 vector (Invitrogen). The RNA corresponding to SLUI e+f sequences were generated in vitro by Runoff transcription using T7 RNA polymerase and the RNA was tested for its inhibitory effect in the in vitro translation assays.<br>
Examples of assay:<br>
In vitro translation: In vitro translation was carried out using 1 ug of template RNA in 17 ul of micrococcal nuclease-treated Rabbit Reticulocyte Lysate (RRL) medium (Promega) and either 0.5 ul each of amino acid mixtures minus methionine and minus cysteine or 20 uCi of S methionine (Perkin Elmer). The reaction mixtures were preincubated with in vitro transcribed small RNAs as indicated in the results. After adding template RNA, the reaction mixtures were incubated at 30 °C for 1 h 30 min, and the products were analyzed either by Dual Luciferase assay system (Promega) in a TD 20/20 Luminometer (Turner Designs, Sunnyvale, CA) or resolved on SDS-12.5 % polyacrylamide gel followed by phosphorimaging (Fuji Imaging, Japan).<br>
DNA and RNA transfection: 60%- 70% confluent monolayers of HeLa and Huh 7 cells in 35-mm dishes were co-transfected with plasmid DNAs using Tfx 20 reagent (Promega) as indicated in results. The cells were harvested 48 h after transfection and luciferase activity was assayed. Huh 7 cells were co-transfected with in vitro transcribed RNAs using Tfx 20 reagent as indicated in results. The cells were harvested 16 h after transfection and luciferase activity was assayed. DNA and RNA quantities were normalized using pGEM 3Z DNA (Promega) or an in vitro transcribed RNA corresponding to its polylinker sequence. Huh 7 cells were transfected with the BB7 HCV subgenomic replicon RNA followed by retransfection with SL IJJ e+f RNA after 16<br><br>
h. 24 h after transfection with SL III e+f, the cells were harvested and the total RNA was isolated using Tri Reagent (Sigma Aldrich).<br>
Ribosomal Assembly assay: 32P-labeled HCV 5'UTR RNA (~105 cpm) was added to 25 ul of translation reaction containing 17.5 ul RRL, in presence or absence of 200 fold excess of SL III e+f RNA. 2 mM 5'-Guanylyl-imidophosphate (GMP-PNP, Sigma Aldrich) was added to the reaction as indicated in the results. The reactions were incubated at 30 °C for 15 min, diluted to 150 ul with gradient buffer (20 mM Tris-Cl, pH 7.5, 100 mM KC1, 3 mM MgCl2, ImM DTT) and overlaid on a 5%- 30% linear sucrose gradient. The ribosomal complexes were sedimented by ultracentrifiigation for 3 h at 4 °C and 30000 rpm. 500 ul fractions were collected from the bottom of the column and the radioactivity was measured in a liquid scintillation counter.<br>
Methodology for other experiments and data analysis:<br>
in vitro transcription: The plasmid pRL-HCVlb was linearized downstream of Flue and transcribed using T7 RNA polymerase in the presence of RNA Cap Analog (Invitrogen) to generate the bicistronic capped RNA. The plasmids pCD-SL II, SLIII and SL IV were linearized with Eco RI and transcribed by in vitro run-off transcription reactions under standard conditions using reagents from Promega. 32P-labeled HCV 5' UTR RNA and the SL RNAs were transcribed from respective plasmids using T7 RNA polymerase and a-32P Uridine tri-phosphate (Perkin Elmer Life Sciences, Boston, MA).<br>
Ultraviolet light- induced crosslinking of proteins with RNA: The in vitro transcribed 32P-labeled RNAs were incubated widi HeLa S10 extract or purified protein in 2X RNA binding buffer and UV-crosslinked as described earlier (19). Unbound RNAs were digested by treatment with 30 jig of RNase A at 37 °C for 30 min. The protein-nucleotidyl complexes were electrophoresed on SDS-10 % polyacrylamide gels followed by autoradiography.<br>
Statistical analysis: The mean ± SD of five independent in vitro translation experiments is represented. Data from the co-transfection experiments is expressed as mean + SD of three independent replicates. The significance of differences between means was tested<br><br>
by Student's t test. ICSo values for inhibitory RNAs were determined by non-linear regression analysis of the inhibition curves using SigmaPlot (SPSS Inc., Chicago, IL).<br>
Experimental evidences:<br>
1.	Specific inhibition of HCV IRES-mediated translation by RNA corresponding to<br>
SL HI of the HCV 5'UTR.<br>
PvNAs corresponding to the three major domains of the HCV 5'UTR: SL II, SL HI (excluding SL m e+f) and SL IV (including SL m e+f) (Fig. 1A) were transcribed in vitro. 100-fold and 200-fold excess of these RNAs was exogenously added to in vitro translation reactions of the Rluc-HCV-Fluc bicistronic RNA and the reporter gene products were radiolabeled (Fig IB). The three RNA molecules differentially inhibited HCV IRES-mediated translation of Flue and cap-dependent translation of Rluc in a dose-dependent manner. SL II did not significantly inhibit either IRES-mediated or cap-dependent translation whereas SL IV was found to inhibit both. SL in RNA was found to strongly inhibit HCV IRES-mediated translation with no significant effect on cap-dependent translation (Fig. IB). Similar results were obtained when the luciferase activity from non-radiolabeled reporter gene products was assayed (Fig. 1C). SL HI RNA caused a maximum 80% reduction of Flue activity whereas there was 73% and 57% inhibition at corresponding doses of SL IV and SL n respectively (Fig. 1C). IC50 values for SL HI, IV and II were determined to be respectively 35-fold, 60- fold and 152-fold excess of the template RNA. There was no significant inhibition of Rluc activity by either SL ID or SL n, whereas SL IV caused 68% inhibition at 200-fold excess concentration (Fig. ID). These observations indicate both an efficient and selective inhibition of HCV IRES-mediated translation by SL III which was significantly higher than that caused by SL II or SL IV RNA (P
2.	Specific inhibition of HCV IRES-mediated translation in vivo.<br><br>
As SL III was found to strongly inhibit HCV IRES-mediated translation in vitro, we next tested its translation inhibitory activity in vivo. For this purpose, three-way co-transfections were performed in HeLa and Huh7 cells using pRL-CMV, pCD-HCV-Fluc and two different concentrations of pCD-SL III and pCD-SL II DNAs. The pCD-HCV-Fluc contained the entire 341 nt long HCV 5'UTR together with 42 nt from the coding region of the core to maintain the pseudoknot structure around SL IV and enhance the efficiency of translation. pRL-CMV was used to generate a capped Rluc transcript in vivo. Approximately 100 ng of both the plasmid DNAs were co-transfected along with two concentrations of either SL II or SL HI encoding DNA. SL II, which had not shown a significant effect on either IRES-mediated or cap-dependent translation in vitro, was utilized as the negative control hi these experiments. The combined data obtained from three co-transfection experiments in HeLa cells showed that SL HI significantly (P
3. Specific inhibition of HCV IRES-mediated translation in vivo in hepatocellular carcinoma cells constitutively expressing the inhibitor SLIII RNA.<br>
In order to investigate whether the constitutive expression of the SL III RNA in cells had any cytotoxic effect, a Huh7 cell line, stably transfected with the pCD-SL m-Ribo plasmid (Fig. 3A), was generated.<br><br>
This cell line, Huh-HCVSLHI did not show any cytotoxic effect for the period of four months during which it was monitored. Other parameters like cellular morphology and doubling time also remained unchanged. The constitutive expression of the SL in RNA was investigated by RT-PCR analysis of total RNA extracted from these cells using SL HI specific primers. Semi-quantitative RT-PCR showed that the SL in RNA was expressed at a high level in the Huh-HCVSLm cell line, but was absent in untransfected Huh7 cells (Fig.3 B). A control RT-PCR using p-actin specific primers demonstrated that the level of transcription of actin in the Huh-HCVSLIU cells was similar to that in untransfected Huh7 cells, suggesting that the transcription of a housekeeping gene like actin was not compromised in these cells (Fig.3 B).<br>
In order to investigate the effect of SL III expression on cellular protein synthesis, Huh-HCVSLIII cells and untransfected Huh7 cells were metaboHcally labeled using 35S-Mef35S-Cys and the proteins were resolved by SDS-PAGE. No difference in the cellular protein synthesis between the stably transfected and untransfected cells was observed <fig.3c></fig.3c>
When the Huh-HCVSLm cells were transfected with a HCV bicistronic plasmid, HCV IRES-mediated translation was found to be significantly reduced. However, cap-dependent translation was also reduced, specially when low amounts of the bicistronic DNA were transfected, suggesting that constitutive over-expression of the SL III RNA could inhibit cap-dependent translation to some extent, possibly by sequestering some of the protein factors required for cap-dependent translation initiation.<br>
4. Inhibition of HCV IRES-mediated translation by an RNA corresponding to SL Hle+f.<br>
As RNA molecules corresponding to domain in of the HCV IRES showed the maximum translation inhibitory effect, we tried to generate smaller RNAs corresponding to its specific subdomains that could inhibit HCV IRES-mediated translation. Increasing concentrations of small RNAs corresponding to the SL m a+c, b, d and e+f subdomains<br><br>
(Fig. 4A) were exogenously added to in vitro translation reactions of the HCV bicistronic RNA. The RNAs were synthesized by oHgonucleotide-driven transcription, as described schematically in Fig 4B. Interestingly, SL III e+f RNA caused a very strong inhibition (89% reduction from control) of HCV IRES mediated translation (Fig. 4C) with no significant effect on cap-dependent translation (Fig. 4D). SL III b caused 57% inhibition of HCV IRES-mediated translation but this was accompanied by a 53% decrease in cap-dependent translation (Fig 4C and D). SL III a+c and III d RNAs showed minor reductions in the efficiency of HCV IRES-mediated translation. The IC5o of SL III e+f RNA was calculated to be 15-fold excess of template RNA. These observations demonstrated that the small RNA corresponding to SL HI e+f subdomain could strongly inhibit HCV IRES-mediated translation in vitro without significantly inhibiting cap-dependent translation.<br>
5. Effect of SL III e+f RNA on HCV IRES-mediated translation in vivo.<br>
As SL III e+f RNA showed a strong inhibition of HCV IRES-mediated translation in vitro, we next investigated its effect on HCV translation in vivo. Huh 7 cells were co-transfected with in vitro transcribed HCV bicistronic RNA together with two concentrations of SL HI e+f RNA. SL III d, which had not shown a significant effect, was used as the negative control. SL m e+f RNA significantly inhibited (P
As SL HI e+f RNA was found to strongly inhibit HCV IRES-mediated translation in vitro and in vivo, we investigated the protein binding profile of the RNA to correlate it with the translation inhibitory activity. The SL in subdomain RNAs demonstrated differential binding to specific proteins that interacted with the HCV domain III RNA (Fig 6A). A 25 kDa protein was found to interact strongly with SL HI e+f RNA but did interact with the other RNAs. This protein may correspond to the S5 ribosomal protein as a single mutation in this region of the HCV 5'UTR has been shown to abrogate the binding of this<br><br>
protein (19). In order to investigate this possibility, bacterially expressed S5 ribosomal protein was used for UV-crosslinking assay. SL III e+f RNA strongly interacted with the purified protein whereas SL HI d, which had not shown any interaction with p25 from HeLa cells, failed to do so (Fig 6B). The full length HCV 5'UTR also interacted with the S5 protein, but to a lesser extent than SL in e+f. Therefore, the small RNAs corresponding to the subdomains of domain III could bind to a number of cellular proteins which interacted with the HCV IRES and SL III e+f uniquely interacted with the S5 ribosomal protein.<br>
6.	Effect of point mutation within SLIII e+f sub-domain.<br>
As SL III e+f RNA was found to strongly inhibit HCV IRES-mediated translation in vitro and in vivo, and was also found to interact with the S5 ribosomal protein, we investigated whether the binding to the S5 protein was specific and was responsible for its inhibitory activity. A single mutation in SL EI e (A297G) of the HCV 5'UTR has been shown to abrogate the binding of the S5 protein to the full-length HCV IRES RNA. An in vitro transcribed mutant SL HI e+f RNA, harbouring the A297G mutation in SL m e (Fig 6A), showed a drastic reduction in binding to the p25 protein from HeLa cytoplasmic extract and also the purified S5 protein (Fig 6B and 6C). 100 and 200 fold molar excess of this mutant SL III e+f RNA also failed to inhibit HCV IRES-mediated translation in vitro (Fig 6D). These observations suggested that the SL m e+f RNA inhibited HCV IRES-mediated translation specifically by binding to the S5 ribosomal protein.<br>
7.	SL m e+f RNA prevented the assembly of ribosomal complexes on the HCV<br>
IRES.<br>
The binding of the 25 kDa S5 ribosomal protein has been suggested to be crucial for efficient translation mediated by the HCV IRES (20). As SL III e+f RNA was found to interact strongly with the S5 protein, we investigated its effect on ribosome recruitment by the HCV IRES. For this purpose, ribosomal assembly reactions containing radiolabeled HCV 5'UTR were incubated with 200 fold excess of SL m e+f RNA and analyzed by sucrose density gradient ultracentrifugation. In absence of SL III e+f RNA, HCV IRES showed the formation of both 48S and 80S ribosomal complexes (Fig 7 A,<br>
t *i<br><br>
solid line). However, in the presence of SL in e+f RNA, formation of both these complexes was significantly reduced (Fig 7A, dotted line), suggesting that SL m e+f prevented the assembly of ribosomal complexes on the HCV IRES. For further elucidating the role of SL HI e+f RNA, the ribosomal assembly reactions were incubated with GMP-PNP which inhibits translation initiation at the 48S stage by preventing the release of eIF2. Addition of GMP-PNP abolished the 80S peak in the control reaction demonstrating that only the 48S complex was being assembled (Fig 7B, solid line). In presence of SL III e+f the 48S complex formation was also abrogated suggesting that SL HI e+f prevented the binding of the 40S ribosomal subunit to the HCV IRES (Fig 7B, dotted line). This supported the earlier observation that SL HI e+f interacted with the S5 ribosomal protein, a component of the 40S subunit. Taken together, these observations suggest that the SL in e+f RNA inhibited HCV IRES-mediated translation by interacting with a component of the 40S subunit and thereby prevented the ribosomal assembly on the HCV IRES.<br>
To investigate the effect of SL III e+f RNA on ribosome recruitment by a capped transcript, the ribosome assembly experiment was performed using a radiolabeled capped GFP RNA and 200 fold molar excess of the SL III e+f RNA. As the capped transcript incorporated the protein-coding region of GFP, on sucrose density gradient ultracentrifagation a single 48S peak and a number of smaller peaks corresponding to polysomes was observed (Fig. 8A). 200-fold excess of SL HI e+f RNA did not inhibit the formation of either the 48S complex or the polysomes (Fig. 8A). On the other hand, when a radiolabeled transcript containing the GFP gene downstream of the HCV IRES was used for the ribosome assembly reactions 200 fold excess of SL in e+f RNA inhibited the formation of both the 48S complex and the polysome peaks (Fig. 8B). This suggested that SL HI e+f specifically abrogated ribosome binding to the HCV IRES and not to capped mRNAs and supported the earlier observation that SL HI e+f RNA did not inhibit cap-dependent translation. Finally, as SL HI e+f was found to interact with the S5 protein, which is a component of the 40S ribosomal subunit and inhibit ribosome recruitment by the HCV IRES, it was hypothesized that the SL HI e+f RNA directly interacted with the 40S subunit. In order to investigate this, ribosome assembly experiments were performed with a radiolabeled SL III e+f RNA. On sucrose density<br><br>
gradient resolution, a single peak corresponding to the 40S subunit bound to the radiolabeled RNA was obtained (Fig. 8Q. This suggested that the SL in e+f RNA was able to directly interact with the ribosomal 40S subunit probably via the S5 protein, which is a component of the 40S subunit.<br>
Taken together, these observations indicate that the SL IH e+f RNA inhibited HCV IRES-mediated translation by interacting with a component of the 40S ribosomal subunit and thereby preventing the ribosome recruitment by the HCV IRES (Fig. 9, HCV inhibition- model of the present invention).<br><br>
References:<br>
1.	Jenny-Avital ER. Hepatitis C. Curr Opin Infect Dis 1998;11:293-299.<br>
2.	Lindsay KL. Therapy of hepatitis C: overview. Hepatology 1997;26:715-755.<br>
3.	Randall G, Grakoui A, Rice CM. Clearance of replicating hepatitis C virus replicon RNAs in cell culture hy small interfering RNAs. Proc Natl Acad Sci USA 2003;100: 235-240.<br>
4.	Tsukiyama-Kohara K, Iizuka N, Kohara M, Nomoto A. Internal ribosome entry site within hepatitis C virus RNA. J Virol. 1992;66: 1476-1483.<br>
5.	Pudi R, Abhiman S, Srinivasan N, Das S. Hepatitis C virus internal ribosome entry site-mediated translation is stimulated by specific interaction of independent regions of human La autoantigen. J Biol Chem 2003;278:12231-12240.<br>
6.	Ali N, Siddiqui A. Interaction of polypyrimidine tract-binding protein with the 5' noncoding region of the hepatitis C virus RNA genome and its functional requirement in internal initiation of translation. J Virol 1995;69: 6367-6375.<br>
7.	Sizova DV, Kolupaeva VG, Pestova TV, Shatsky IN, Hellen CUT. Specific interaction of eukaryotic translation initiation factor 3 with the 5' nonrranslated regions of hepatitis C virus and classical swine fever virus RNAs. J Virol 1998;72: 4775-4782.<br>
8.	Kruger M, Beger C, Li QX, Welch PJ, Tritz R, Leavitt M, Barber JR, et al. Identification of eIF2B*y and eIF2Y as cofactors of hepatitis C virus internal ribosome entry site-mediated translation using a functional genomics approach. Proc Natl Acad Sci USA 2000;97: 8566-8571.<br>
9.	Kieft JS, Zhou K, Jubin R, Murray MG, Lau JYN, Doudna JA. The hepatitis C virus internal ribosome entry site adopts an ion-dependent tertiary fold. J Mol Biol 1999;292:513-529.<br><br>
10.	Lott WB, Takyar SS, Tuppen J, Crawford DHG, Harrison M, Sloots TP, Gowans EJ. Vitamin B12 and hepatitis C: Molecular biology and human pathology. Proc Natl Acad Sci USA 2001;98:4916-4921.<br>
11.	Wu CH, Wu GY. Targeted inhibition of hepatitis C virus-directed gene expression in human hepatoma cell lines. Gastroenterology 1998;114:1304-1312.<br>
12.	Welch PI, Tritz R, Yei S, Leavitt M, Yu M, Barber J. A potential therapeutic application of hairpin ribozymes: in vitro and in vivo studies of gene therapy for hepatitis C virus infection. Gene Ther 1996;3:994-1001.<br>
13.	Lee, TC, Sullenger BA, Gallardo HF, Ungers GE, Gilboa E. Overexpression of RRE-derived sequences inhibits HIV-1 replication in CEM cells. New Biol 1992;4:66-74.<br>
14.	Sullenger BA, Gallardo HF, Ungers GE, Gilboa E. Overexpression of TAR sequences renders cells resistant to Human Immunodeficiency Virus replication. Cell 1990;63: 601-608.<br>
15.	Das S, Kenan DJ, Bocskai D, Keene JD, Dasgupta A. Sequences within a small yeast RNA required for inhibition of internal initiation of translation: interaction with La and other cellular proteins influences its inhibitory activity. J Virol 1996; 70:1624-1632.<br>
16.	Das S, Ott M, Yamane A, Tsai W, Gromeier M, Lahser F, Gupta S, et al. A small yeast RNA blocks hepatitis C virus internal ribosome entry site (HCV IRES)-mediated translation and inhibits replication of a chimeric poliovirus under translational control of the HCV IRES element. J Virol. 1998;72:5638-5647.<br>
17.	All N, Pruijn GJM, Kenan DJ, Keene JD, Siddiqui A. Human La antigen is required for the hepatitis C virus internal ribosome entry site-mediated translation. J Biol Chem 2000;275:27531-27540.<br>
18.	Anwar A, All N, Tanveer R, Siddiqui A. Demonstration of functional requirement of polypyrimidine tract-binding protein by SELEX RNA during hepatitis C virus<br><br>
internal   ribosome   entry   site-mediated   translation   initiation.   J   Biol   Chem 2000;275:34231-34235.<br>
19.	Odreman-Macchioli, F., Baralle, F.E. and Buratti, E. (2001) Mutational analysis of the different bulge regions of hepatitis C virus Domain II and their influence on internal ribosome entry site translational ability. J. Biol. Chem. 276,41648-41655.<br>
20.	Fukushi, S., Kurihara, C, Ishiyama, N., Hoshino, F.B., Oya, A. and Katayama K. (1997) The sequence element of the internal ribosome entry site and a 25-kilodalton cellular protein contribute to efficient internal initiation of translation of hepatitis C virus RNA. J. Virol., 71,1662-1666.<br>
21.	Gosert, R. and Moradpour, D. (2002) A new twist to translation initiation of the hepatitis C virus. Hepatology, 35, 724-726.<br>
22.	Gitlin, L., Karelsky, S and Andino, R. (2002) Short interfering RNA confers intracellular antiviral immunity in human cells. Nature, 418, 430-434.<br>
23.	Dove A. (2002) Antisense and sensibility. Nat. Biotechnol., 20,121-124.<br><br><br>
WE CLAIM:<br>
1. A small synthetic HCV IRES ribonucleic acid of sequence GGGA GGGC CCTCTCG GTAGA ACACCA TGACGGA CTATCCCACGAACGCTCACGGGGCCCTCC.<br>
2A structural analog or mimic of small synthetic HCV IRES rihonucleic acid of sequence GGGA GGGC CCTCTCG GTAGA ACACCA TGACGGA CTATCCCACGAACGCTCACGGGGCCCTCC.<br>
3.	An inhibitor of HCV IRES- mediated translation mechanism by the SL HI e+F RNA<br>
ofthe HCV 5'UTR comprising small synthetic HCV IRES ribonucleic acid of sequence<br>
GGGA	GGGC	CCTCTCG	GTAGA	ACACCA	TGACGGA<br>
CTATCCCACGAACGCTCACGGGGCCCTCC or the structural analog or mimic<br>
thereof<br>
4.	An antivnal agent to combat HCV infection comprising small synthetic HCV IRES<br>
ribonucleic acid of small synthetic HCV IRES ribonucleic acid of sequence<br>
GGGA	GGGC	CCTCTCG	GTAGA	ACACCA	TGACGGA<br>
CTATCCCACGAACGCTCACGGGGCCCTCC or the structural analog or mimic thereof<br>
5.	A polynucleotide comprising the HCV IRES nucleic acid sequence<br>
GGGA	GGGC	CCTCTCG	GTAGA	ACACCA	TGACGGA<br>
CTATCCCACGAACGCTCACGGGGCCCTCC or the structural analog or numic thereof<br>
6.	A recombinant vector comprising the polynucleotide of claim 5.<br>
7.	A method of synthesizing the HCV IRES nucleic acid sequence<br>
GGGA	GGGC	CCTCTCG	GTAGA	ACACCA	TGACGGA<br>
CTATCCCACGAACGCTCACGGGGCCCTCC or the structural analog or mimic<br>
thereof by in vitro transcription assay.<br><br>
8.	A method as claimed in claim 7, wherein synthetic DNA oligonucleotide corresponding to domain III stem-loops e+f structures with T7 promotor sequences at the 5'end was annealed to T7 RNA polymerase promoter primers and transcribed in vitro using T7 RNA polymerase, extracting the transcription reaction with phenol and chloroform, purifying and concentrating the RNA formed by alcohol precipitation, drying the RNA pellet in vacuum centrifuge and dissolving in nuclease free water.<br>
9.	A method for making a recombinant vector comprising the step of inserting the Polynucleotide or the structural analog or mimic of claim 5 into a vector.<br>
10.	A method for inhibiting HCVIRES mediated translation comprising the intro¬<br>
duction of the secondary structure of the 100-fold and 200-fold molar excess of in<br>
vitro transcribed SL U, HI and IV RNAs to in vitro translation reactions of HCV<br>
bicistronic RNA.<br>
11.	An antiviral composition containing the nucleic acid sequence<br>
GGGA	GGGC	CCTCTCG	GTAGA	ACACCA	TGACGGA<br>
CTATCCCACGAACGCTCACGGGGCCCTCC or a stiuctural analog or mimic optionally admixed with a pharmaceutically acceptable carrier, diluent, excipient or adjuvant.<br>
12.	A method of manufacturing an antiviral composition for treating liver cirrhosis and<br>
hepatocellular carcinoma caused by hepatitis C virus comprising admixing the nucleotide<br>
sequence or a structural analog or mimic according to claim 1 or 2 with a<br>
pharmaceutically acceptable carrier, diluent, excipient or adjuvant.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI0LWNoZS0yMDA0IGFic3RyYWN0IChwcm92aXNpb25hbCkucGRm" target="_blank" style="word-wrap:break-word;">224-che-2004 abstract (provisional).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI0LWNoZS0yMDA0IGFic3RyYWN0IGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">224-che-2004 abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI0LWNoZS0yMDA0IGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">224-che-2004 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI0LWNoZS0yMDA0IGNsYWltcyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">224-che-2004 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI0LWNoZS0yMDA0IGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">224-che-2004 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI0LWNoZS0yMDA0IGNvcnJlc3BvbmRlbmNlIG90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">224-che-2004 correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI0LWNoZS0yMDA0IGNvcnJlc3BvbmRlbmNlIHBvLnBkZg==" target="_blank" style="word-wrap:break-word;">224-che-2004 correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI0LWNoZS0yMDA0IGRlc2NyaXB0aW9uIChjb21wbGV0ZSkgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">224-che-2004 description (complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI0LWNoZS0yMDA0IGRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">224-che-2004 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI0LWNoZS0yMDA0IGRlc2NyaXB0aW9uIChwcm92aXNpb25hbCkucGRm" target="_blank" style="word-wrap:break-word;">224-che-2004 description (provisional).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI0LWNoZS0yMDA0IGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">224-che-2004 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI0LWNoZS0yMDA0IGZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">224-che-2004 form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI0LWNoZS0yMDA0IGZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">224-che-2004 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI0LWNoZS0yMDA0IGZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">224-che-2004 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI0LWNoZS0yMDA0IGZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">224-che-2004 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI0LWNoZS0yMDA0IHBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">224-che-2004 pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="224902-a-method-for-producing-neuronal-precursor-cells.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="224904-vial-system-and-method-for-processing-liquid-based-specimens.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>224903</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>224/CHE/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>49/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>05-Dec-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>24-Oct-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>12-Mar-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>INDIAN INSTITUTE OF SCIENCE</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>BANGALORE - 560 012,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>PARTHO SAROTHI RAY</td>
											<td>DEPT OF MICROBIOLOGY AND CELL BIOLOGY, SIR C.V. RAMAN AVENUE, INDIAN INSTITUTE OF SCIENCE, BANGALORE 560 012,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>DR. SAUMITRA DAS</td>
											<td>DEPT OF MICROBIOLOGY AND CELL BIOLOGY, SIR C.V. RAMAN AVENUE, INDIAN INSTITUTE OF SCIENCE, BANGALORE 560 012,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/09</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/224903-a-small-synthetic-rna-a-method-of-preparing-the-same-and-uses-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:12:37 GMT -->
</html>
