<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/225008-lim-mineralization-protein-splice-variants-to-induce-bone-formation by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:40:19 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 225008:LIM MINERALIZATION PROTEIN SPLICE VARIANTS TO INDUCE BONE FORMATION</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">LIM MINERALIZATION PROTEIN SPLICE VARIANTS TO INDUCE BONE FORMATION</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention is directed to isolated nucleic acid molecules that encode LIM mineralization protein, or LMP. The invention further provides vectors comprising splice variants of nucleotide sequences that encode LMP, as well as host cells comprising those vectors. Moreover, the present invention relates to methods of inducing bone formation by transfecting osteogenic precursor cells with an isolated nucleic acid molecule comprising a nucleotide sequence encoding splice variants of LIM mineralization protein. The transfection may occur ex vivo or in vivo by direct injection of virus or naked plasmid DNA. In a particular embodiment, the invention provides a method of fusing a spine by transfecting osteogenic precursor cells with an isolated nucleic acid molecule having a nucleotide sequence encoding LIM mineralization protein, admixing the transfected osteogenic precursor cells with a matrix and contacting the matrix with the spine. Finally, the invention relates to methods for inducing systemic bone formation by stable transfection of host cells with the vectors of the invention.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>LIM MINERALIZATION PROTEIN SPLICE VARIANTS TO<br>
INDUCE BONE FORMATION<br>
BACKGROUND OF THE INVENTION<br>
1. Field of the Invention<br>
The field of the invention relates generally to osteogenic cells and the<br>
formation of bone and boney tissue in mammalian species. Specifically, the<br>
invention concerns a novel family of proteins, and nucleic acids encoding those<br>
proteins, that enhances the efficacy of bone mineralization in vitro and in vivo.<br>
The invention provides methods for treating a variety of pathological conditions<br>
associated with bone and boney tissue, such as, for example, spine fusion,<br>
fracture repair and osteoporosis.<br>
2. Description of the Related Art<br>
Osteoblasts are thought to differentiate from pluripotent mesenchymal<br>
stem cells. The maturation of an osteoblast results in the secretion of an<br>
extracellular matrix which can mineralize and form bone. The regulation of this<br>
complex process is not well understood but is thought to involve a group of<br>
signaling glycoproteins known as bone morphogenetic proteins (BMPs). These<br>
proteins have been shown to be involved with embryonic dorsal-ventral<br>
patterning, limb bud development, and fracture repair in adult animals. B. L.<br>
Hogan, Genes &amp; Develop., 10:1580 (1996). This group of transforming growth<br>
factor-beta superfamily secreted proteins has a spectrum of activities in a<br>
variety of cell types at different stages of differentiation; differences in<br>
physiological activity between these closely related molecules have not been<br>
clarified. D. M. Kingsley, Trends Genet., 10:16 (199.4).<br>
To better discern the unique physiological role of different BMP signaling<br>
proteins, we recently compared the potency of BMP-6 with that of BMP-2 and<br>
BMP-4, for inducing rat calvarial osteoblast differentiation. Boden et al.,<br>
Endocrinology. 137:3401 (1996). We studied this process in first passage<br>
(secondary) cultures of fetal rat calvaria that require BMP or glucocorticoid for<br>
initiation of differentiation. In this model of membranous bone formation,<br>
glucocorticoid (GC) or a BMP will initiate differentiation to mineralized bone<br>
nodules capable of secreting osteocalcin, the osteoblast-specific protein. This<br>
secondary culture system is distinct from primary rat osteoblast cultures which<br>
undergo spontaneous differentiation. In this secondary system, glucocorticoid<br>
resulted in a ten-fold induction of BMP-6 mRNA and protein expression which<br>
was responsible for the enhancement of osteoblast differentiation. Boden et<br>
al., Endocrinology. 138:2920 (1997).<br>
In addition to extracellular signals, such as the BMPs, intracellular<br>
signals or regulatory molecules may also play a role in the cascade of events<br>
leading to formation of new bone. One broad class of intracellular regulatory<br>
molecules are the LIM proteins, which are so named because they possess a<br>
characteristic structural motif known as the LIM domain. The LIM domain is a<br>
cysteine-rich structural motif composed of two special zinc fingers that are<br>
joined by a 2-amino acid spacer. Some proteins have only LIM domains, while<br>
others contain a variety of additional functional domains. LIM proteins form a<br>
diverse group, which includes transcription factors and cytoskeletal proteins.<br>
The primary role of LIM domains appears to be in mediating protein-protein<br>
interactions, through the formation of dimers with identical or different LIM<br>
domains, or by binding distinct proteins.<br>
In LIM homeodomain proteins, that is, proteins having both LIM domains<br>
and a homeodomain sequence, the LIM domains function as negative<br>
regulatory elements. LIM homeodomain proteins are involved in the control of<br>
cell lineage determination and the regulation of differentiation, although LIM-<br>
only proteins may have similar roles. LIM-only proteins are also implicated in<br>
the control of cell proliferation since several genes encoding such proteins are<br>
associated with oncogenic chromosome translocations.<br>
Humans and other mammalian species are prone to diseases or injuries<br>
that require the processes of bone repair and/or regeneration. For example,<br>
treatment of fractures would be improved by new treatment regimens that<br>
could stimulate the natural bone repair mechanisms, thereby reducing the time<br>
required for the fractured bone to heal. In another example, individuals<br>
afflicted with systemic bone disorders, such as osteoporosis, would benefit<br>
from treatment regimens that would results in systemic formation of new bone.<br>
Such treatment regimens would reduce the incidence of fractures arising from<br>
the loss of bone mass that is a characteristic of this disease.<br>
For at least these reasons, extracellular factors, such as the BMPs,<br>
have been investigated for the purpose of using them to stimulate formation of<br>
new bone in vivo. Despite the early successes achieved with BMPs and other<br>
extracellular signalling molecules, their use entails a number of disadvantages.<br>
For example, relatively large doses of purified BMPs are required to enhance<br>
the production of new bone, thereby increasing the expense of such treatment<br>
methods. Furthermore, extracellular proteins are susceptible to degradation<br>
following their introduction into a host animal. In addition, because they are<br>
typically immunogenic, the possibility of stimulating an immune response to the<br>
administered proteins is ever present.<br>
Due to such concerns, it would be desirable to have available treatment<br>
regimens that use an intracellular signalling molecule to induce new bone<br>
formation. Advances in the field of gene therapy now make it possible to<br>
introduce into osteogenic precursor cells, that is, cells involved in bone<br>
formation, or peripheral blood leukocytes, nucleotide fragments encoding<br>
intracellular signals that form part of the bone formation process. Gene<br>
therapy for bone formation offers a number of potential advantages: (1) lower<br>
production costs; (2) greater efficacy, compared to extracellular treatment<br>
regiments, due to the ability to achieve prolonged expression of the<br>
intracellular signal; (3) it would by-pass the possibility that treatment with<br>
extracellular signals might be hampered due to the presence of limiting<br>
numbers of receptors for those signals; (4) it permits the delivery of transfected<br>
potential osteoprogenitor cells directly to the site where localized bone<br>
formation is required; and (5) it would permit systemic bone formation, thereby<br>
providing a treatment regimen for osteoporosis and other metabolic bone<br>
diseases.<br>
SUMMARY OF THE INVENTION<br>
The present invention seeks to overcome the drawbacks in the prior art<br>
by providing novels compositions and methods for inducing bone formation<br>
using an intracellular signalling molecule that participates early in the cascade<br>
of events that leads to bone formation. Applicants have discovered 10-<br>
4/RLMP (SEQ ID NO: 1, SEQ ID NO: 2), a novel LIM gene with a sequence<br>
originally isolated from stimulated rat calvarial osteoblast cultures. The gene<br>
has been cloned, sequenced and assayed for its ability to enhance the efficacy<br>
of bone mineralization in vitro. The protein RLMP affects mineralization of<br>
bone matrix as well as differentiation of cells into the osteoblast lineage. Unlike<br>
other known cytokines, for example, BMPs, RLMP is not a secreted protein,<br>
but is instead an intracellular signaling molecule. This feature has the<br>
advantage of providing intracellular signaling amplification as well as easier<br>
assessment of transfected cells. It is also suitable for more efficient and<br>
specific in vivo applications. Suitable clinical applications include enhancement<br>
of bone repair in fractures, bone defects, bone grafting, and normal<br>
homeostasis in patients presenting with osteoporosis.<br>
Applicants have also cloned, sequenced and deduced the amino acid<br>
sequence of a corresponding human protein, named human LMP-1. The<br>
human protein demonstrates enhanced efficacy of bone mineralization in vitro<br>
and in vivo.<br>
In addition, the applicants have characterized a truncated (short) version<br>
of LMP-1, termed HLMP-1s. This short version resulted from a point mutation<br>
in one source of a cDNA clone, providing a stop codon which truncated the<br>
protein. The short version (LMP-1 s) is fully functional when expressed in cell<br>
culture and in vivo.<br>
Using PCR analysis of human heart cDNA library, Applicants have<br>
identified two alternative splice variants (referred to as HLMP-2 and HLMP-3)<br>
that differ from HLMP-1 in a region between base pairs 325 and 444 in the<br>
nucleotide sequence encoding HLMP-1. The HLMP-2 sequence has a 119<br>
base pair deletion and an insertion of 17 base pairs in this region. Compared<br>
to HLMP-1, the nucleotide sequence encoding HLMP-3 has no deletions, but it<br>
does have the same 17 base pairs as HLMP-2, which are inserted at position<br>
444 in the HLMP-1 sequence.<br>
Additional features and advantages of the invention will be set forth in<br>
the description which follows, and in part will be apparent from the description,<br>
or may be learned by practice of the invention. The objectives and other<br>
advantages of the invention will be realized and attained by the methods and<br>
compositions of matter particularly pointed out in the written description and<br>
claims hereof.<br>
In one broad aspect, the invention relates to an isolated nucleic acid<br>
molecule comprising a nucleic acid sequence encoding any LIM mineralization<br>
protein, wherein the nucleic acid molecule hybridizes under standard<br>
conditions to a nucleic acid molecule complementary to the full length of SEQ.<br>
ID NO: 25, and wherein the molecule hybridizes under highly stringent<br>
conditions to a nucleic acid molecule complementary to the full length of SEQ.<br>
ID NO: 26. In a specific aspect, the isolated nucleic acid molecule encodes<br>
HLMP-1, HLMP-1s, RLMP, HLMP-2, or HLMP-3. In addition, the invention is<br>
directed to vectors comprising these nucleic acid molecules, as well as host<br>
cells comprising the vectors. In another specific aspect, the invention relates to<br>
the proteins themselves.<br>
In a second broad aspect, the invention relates to antibody that is<br>
specific for LIM mineralization protein, including HLMP-1, HLMP-1s, RLMP,<br>
HLMP-2, and HLMP-3. In one specific aspect, the antibody is a polyclonal<br>
antibody. In another specific aspect, the antibody is a monoclonal antibody.<br>
In a third broad aspect, the invention relates to method of inducing bone<br>
formation wherein osteogenic precursor cells are transfected with an isolated<br>
nucleic acid molecule comprising a nucleotide sequence encoding LIM<br>
mineralization protein. In one specific aspect, the isolated nucleic acid<br>
molecule is in a vector, which may be a plasmid or a virus, such as adenovirus<br>
or retrovirus. The transfection may occur ex vivo or in vivo by direct injection of<br>
the isolated nucleic acid molecule. The transfected isolated nucleic acid<br>
molecule may encode HLMP-1, HLMP-1s, RLMP, HLMP-2, or HLMP-3.<br>
In a further aspect, the invention relates to methods of fusing a spine by<br>
transfecting osteogenic precursor cells with an isolated nucleic acid molecule<br>
having a nucleotide sequence encoding LIM mineralization protein, admixing<br>
the transfected osteogenic precursor cells with a matrix and contacting the<br>
matrix with the spine.<br>
In yet another aspect, the invention relates to methods for inducing<br>
systemic bone formation by stable transfection of host cells with the vectors of<br>
the invention.<br>
It is to be understood that both the foregoing general description and the<br>
following detailed description are exemplary and explanatory and are intended<br>
to provide further explanation of the invention as claimed.<br>
ABBREVIATIONS AND DEFINITIONS<br>
BMP Bone Morphogenetic Protein<br>
HLMP-1 Human LMP-1, also<br>
designated as Human LIM<br>
Protein or HLMP<br>
HLMP-1 s Human LMP-1 Short<br>
(truncated) protein<br>
HLMPU Human LIM Protein Unique<br>
Region<br>
LMP LIM mineralization protein<br>
MEM Minimal essential medium<br>
Trm Triamcinolone<br>
-GlyP Beta-glycerolphosphate<br>
RACE Rapid Amplification of cDNA<br>
Ends<br>
RLMP Rat LIM mineralization protein,<br>
also designated as RLMP-1<br>
RLMPU Rat LIM Protein Unique<br>
Region<br>
RNAsin RNase inhibitor<br>
ROB Rat Osteoblast<br>
10-4 Clone containing cDNA<br>
sequence for RLMP (SEQ ID<br>
NO: 2)<br>
UTR Untranslated Region<br>
HLMP-2 Human LMP Splice Variant 2<br>
HLMP-3 Human LMP Splice Variant 3<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The present invention relates to novel mammalian LIM proteins, herein<br>
designated LIM mineralization proteins, or LMP. The invention relates more<br>
particularly to human LMP, known as HLMP or HLMP-1, or alternative splice<br>
variants of human LMP, which are known as HLMP-2 or HLMP-3. The<br>
applicants have discovered that these proteins enhance bone mineralization in<br>
mammalian cells grown in vitro. When produced in mammals, LMP also<br>
induces bone formation in vivo.<br>
Ex vivo transfection of bone marrow cells, osteogenic precursor cells, <br>
peripheral blood leukocytes, or mesenchymal stem cells with nucleic acid that<br>
encodes LMP or HLMP, followed by reimplantation of the transfected cells in<br>
the donor, is suitable for treating a variety of bone-related disorders or injuries.<br>
For example, one can use this method to: augment long bone fracture repair;<br>
generate bone in segmental defects; provide a bone graft substitute for<br>
fractures; facilitate tumor reconstruction or spine fusion; and provide a local<br>
treatment (by injection) for weak or osteoporotic bone, such as in osteoporosis<br>
of the hip, vertebrae, or wrist. Transfection with LMP or HLMP-encoding<br>
nucleic acid is also useful in: the percutaneous injection of transfected marrow<br>
cells to accelerate the repair of fractured long bones; treatment of delayed<br>
union or non-unions of long bone fractures or pseudoarthrosis of spine fusions;<br>
and for inducing new bone formation in avascular necrosis of the hip or knee.<br>
In addition to ex vivo-based methods of gene therapy, transfection of a<br>
recombinant DNA vector comprising a nucleic acid sequence that encodes<br>
LMP or HLMP can be accomplished in vivo. When a DNA fragment that<br>
encodes LMP or HLMP is inserted into an appropriate viral vector, for example,<br>
an adenovirus vector, the viral construct can be injected directly into a body<br>
site were endochondral bone formation is desired. By using a direct,<br>
percutaneous injection to introduce the LMP or HLMP sequence stimulation of<br>
bone formation can be accomplished without the need for surgical intervention<br>
either to obtain bone marrow cells (to transfect ex vivo) or to reimplant them<br>
into the patient at the site where new bone is required. Alden et al.,<br>
Neurosurgical Focus (1998), have demonstrated the utility of a direct injection<br>
method of gene therapy using a cDNA that encodes BMP-2, which was cloned<br>
into an adenovirus vector.<br>
It is also possible to carry out in vivo gene therapy by directly injecting<br>
into an appropriate body site, a naked, that is, unencapsulated, recombinant<br>
plasmid comprising a nucleic acid sequence that encodes HLMP. In this<br>
embodiment of the invention, transfection occurs when the naked plasmid DNA<br>
is taken up, or internalized, by the appropriate target cells, which have been<br>
described. As in the case of in vivo gene therapy using a viral construct, direct<br>
injection of naked plasmid DNA offers the advantage that little or no surgical<br>
intervention is required. Direct gene therapy, using naked plasmid DNA that<br>
encodes the endothelial cell mitogen VEGF (vascular endothelial growth<br>
factor), has been successfully demonstrated in human patients. Baumgartner<br>
et al., Circulation. 97(12):1114-23 (1998).<br>
By using an adenovirus vector to deliver LMP into osteogenic cells,<br>
transient expression of LMP is achieved. This occurs because adenovirus<br>
does not incorporate into the genome of target cells that are transfected.<br>
Transient expression of LMP, that is, expression that occurs during the lifetime<br>
of the transfected target cells, is sufficient to achieve the objects of the<br>
invention. Stable expression of LMP, however, can occur when a vector that<br>
incorporates into the genome of the target cell is used as a delivery vehicle.<br>
Retrovirus-based vectors, for example, are suitable for this purpose.<br>
Stable expression of LMP is particularly useful for treating various<br>
systemic bone-related disorders, such as osteoporosis and osteogenesis<br>
imperfecta. For this embodiment of the invention, in addition to using a vector<br>
that integrates into the genome of the target cell to deliver an LMP-encoding<br>
nucleotide sequence into target cells, LMP expression is placed under the<br>
control of a regulatable promoter. For example, a promoter that is turned on by<br>
exposure to an exogenous inducing agent, such as tetracycline, is suitable.<br>
Using this approach, one can stimulate formation of new bone on a systemic<br>
basis by administering an effective amount of the exogenous inducing agent.<br>
Once a sufficient quantity of bone mass is achieved, administration of the<br>
exogenous inducing agent is discontinued. This process may be repeated as<br>
needed to replace bone mass lost, for example, as a consequence of<br>
osteoporosis.<br>
Antibodies specific for HLMP are particularly suitable for use in methods<br>
for assaying the osteoinductive, that is, bone-forming, potential of patient cells.<br>
In this way one can identify patients at risk for slow or poor healing of bone<br>
repair. Also, HLMP-specific antibodies are suitable for use in marker assays to<br>
identify risk factors in bone degenerative diseases, such as, for example,<br>
osteoporosis.<br>
Following well known and conventional methods, the genes of the<br>
present invention are prepared by ligation of nucleic acid segments that<br>
encode LMP to other nucleic acid sequences, such as cloning and/or<br>
expression vectors. Methods needed to construct and analyze these<br>
recombinant vectors, for example, restriction endonuclease digests, cloning<br>
protocols, mutagenesis, organic synthesis of oligonucleotides and DNA<br>
sequencing, have been described. For DNA sequencing DNA, the<br>
dieoxyterminator method is the preferred.<br>
Many treatises on recombinant DNA methods have been published,<br>
including Sambrook et al., Molecular Cloning: A Laboratory Manual. Cold<br>
Spring Harbor Press, 2nd edition (1988), Davis et al., Basic Methods in<br>
Molecular Biology. Elsevier (1986), and Ausubel et al., Current Protocols in<br>
Molecular Biology. Wiley Interscience (1988). These reference manuals are<br>
specifically incorporated by reference herein.<br>
Primer-directed amplification of DNA or cDNA is a common step in the<br>
expression of the genes of this invention. It is typically performed by the<br>
polymerase chain reaction (PCR). PCR is described in U.S. Patent No.<br>
4,800,159 to Mullis et al. and other published sources. The basic principle of<br>
PCR is the exponential replication of a DNA sequence by successive cycles of<br>
primer extension. The extension products of one primer, when hybridized to<br>
another primer, becomes a template for the synthesis of another nucleic acid<br>
molecule. The primer-template complexes act as substrate for DNA<br>
polymerase, which in performing its replication function, extends the primers.<br>
The conventional enzyme for PCR applications is the thermostable DNA<br>
polymerase isolated from Thermus aquaticus, or Taq DNA polymerase.<br>
Numerous variations of the basic PCR method exist, and a particular<br>
procedure of choice in any given step needed to construct the recombinant<br>
vectors of this invention is readily performed by a skilled artisan. For example,<br>
to measure cellular expression of 10-4/RLMP, RNA is extracted and reverse<br>
transcribed under standard and well known procedures. The resulting cDNA is<br>
then analyzed for the appropriate mRNA sequence by PCR.<br>
The gene encoding the LIM mineralization protein is expressed in an<br>
expression vector in a recombinant expression system. Of course, the<br>
constructed sequence need not be the same as the original, or its<br>
complimentary sequence, but instead may be any sequence determined by the<br>
degeneracy of the DNA code that nonetheless expresses an LMP having bone<br>
forming activity. Conservative amino acid substitutions, or other modifications,<br>
such as the occurrence of an amino-terminal methionine residue, may also be<br>
employed.<br>
A ribosome binding site active in the host expression system of choice is<br>
ligated to the 5' end of the chimeric LMP coding sequence, forming a synthetic<br>
gene. The synthetic gene can be inserted into any one of a large variety of<br>
vectors for expression by ligating to an appropriately linearized plasmid. A<br>
regulatable promoter, for example, the E. coli lac promoter, is also suitable for<br>
the expression of the chimeric coding sequences. Other suitable regulatable<br>
promoters include trp, tac, recA, T7 and lambda promoters.<br>
DNA encoding LMP is transfected into recipient cells by one of several<br>
standard published procedures, for example, calcium phosphate precipitation,<br>
DEAE-Dextran, electroporation or protoplast fusion, to form stable<br>
transformants. Calcium phosphate precipitation is preferred, particularly when<br>
performed as follows.<br>
DNAs are coprecipitated with calcium phosphate according to the<br>
method of Graham and Van Der, Virology. 52:456 (1973), before transfer into<br>
cells. An aliquot of 40-50 g of DNA, with salmon sperm or calf thymus DNA as<br>
a carrier, is used for 0.5x106 cells plated on a 100 mm dish. The DNA is mixed<br>
with 0.5 ml of 2X Hepes solution (280 mM NaCI, 50 mM Hepes and 1.5 mM<br>
Na2HPO4. pH 7.0), to which an equal volume of 2x CaCI2 (250 mM CaCI2 and<br>
10 mM Hepes, pH 7.0) is added. A white granular precipitate, appearing after<br>
30-40 minutes, is evenly distributed dropwise on the cells, which are allowed to<br>
incubate for 4-16 hours at 37°C. The medium is removed and the cells<br>
shocked with 15% glycerol in PBS for 3 minutes. After removing the glycerol,<br>
the cells are fed with Dulbecco's Minimal Essential Medium (DMEM) containing<br>
10% fetal bovine serum.<br>
DNA can also be transfected using: the DEAE-Dextran methods of<br>
Kimura et al., Virology. 49:394 (1972) and Sompayrac et al., Proc. Natl. Acad.<br>
Sci. USA. 78:7575 (1981); the electroporation method of Potter, Proc. Natl.<br>
Acad. Sci. USA. 81:7161 (1984); and the protoplast fusion method of Sandri-<br>
Goddin et al., Molec. Cell. BioL., 1:743 (1981).<br>
Phosphoramidite chemistry in solid phase is the preferred method for<br>
the organic synthesis of oligodeoxynucleotides and polydeoxynucleotides. In<br>
addition, many other organic synthesis methods are available. Those methods<br>
are readily adapted by those skilled in the art to the particular sequences of the<br>
invention.<br>
The present invention also includes nucleic acid molecules that<br>
hybridize under standard conditions to any of the nucleic acid sequences<br>
encoding the LIM mineralization proteins of the invention. "Standard<br>
hybridization conditions" will vary with the size of the probe, the background<br>
and the concentration of the nucleic acid reagents, as well as the type of<br>
hybridization, for example, in situ, Southern blot, or hybrization of DNA-RNA<br>
hybrids (Northern blot). The determination of "standard hybridization<br>
conditions" is within the level of skill in the art. For example, see U.S. Patent<br>
5,580,775 to Fremeau et al., herein incorporated by reference for this purpose.<br>
See also, Southern, E. M., J. Mol. BioL 98:503 (1975), Alwine et al., Meth.<br>
Enzymol., 68:220 (1979), and Sambrook et al., Molecular Cloning: A<br>
laboratory Manual. 2nd edition, pp. 7.19-7.50, Cold Spring Harbor Press<br>
(1989).<br>
One preferred set of standard hybrization conditions involves a blot that<br>
is prehybridized at 42°C for 2 hours in 50% formamide, 5X SSPE (150 nM<br>
NaCI, 10 mM Na H2PO4 [pH 7.4], 1 mM EDTA [pH 8.0]), 5X Denhardt's solution<br>
(20 mg Ficoll, 20 mg polyvinylpyrrolidone and 20 mg BSA per 100 ml water),<br>
10% dextran sulphate, 1% SDS and 100 g/ml salmon sperm DNA. A 32P-<br>
labelled cDNA probe is added, and hybridization is continued for 14 hours.<br>
Afterward, the blot is washed twice with 2X SSPE, 0.1% SDS for 20 minutes at<br>
22°C, followed by a 1 hour wash at 65°C in 0.1X SSPE, 0.1 %SDS. The blot is<br>
then dried and exposed to x-ray film for 5 days in the presence of an<br>
intensifying screen.<br>
Under "highly stringent conditions," a probe will hybridize to its target<br>
sequence if those two sequences are substantially identical. As in the case of<br>
standard hybridization conditions, one of skill in the art can, given the level of<br>
skill in the art and the nature of the particular experiment, determine the<br>
conditions under which only susbstantially identical sequences will hybridize.<br>
Another aspect of the invention includes the proteins encoded by the<br>
nucleic acid sequences. In still another embodiment, the inventon relates to<br>
the identification of such proteins based on anti-LMP antibodies. In this<br>
embodiment, protein samples are prepared for Western blot analysis by lysing<br>
cells and separating the proteins by SDS-PAGE. The proteins are transferred<br>
to nitrocellulose by electroblotting as described by Ausubel et al., Current<br>
Protocols in Molecular Biology. John Wiley and Sons (1987). After blocking<br>
the filter with instant nonfat dry milk (1 gm in 100 ml PBS), anti-LMP antibody is<br>
added to the filter and incubated for 1 hour at room temperature. The filter is<br>
washed thoroughly with phosphate buffered saline (PBS) and incubated with<br>
horseradish peroxidase (HRPO)-antibody conjugate for 1 hour at room<br>
temperature. The filter is again washed thoroughly with PBS and the antigen<br>
bands are identified by adding diaminobenzidine (DAB).<br>
Monospecific antibodies are the reagent of choice in the present<br>
invention, and are specifically used to analyze patient cells for specific<br>
characteristics associated with the expression of LMP. "Monospecific<br>
antibody" as used herein is defined as a single antibody species or multiple<br>
antibody species with homogenous binding characteristics for LMP.<br>
"Homogeneous binding" as used herein refers to the ability of the antibody<br>
species to bind to a specific antigen or epitope, such as those associated with<br>
LMP, as described above. Monospecific antibodies to LMP are purified from<br>
mammalian antisera containing antibodies reactive against LMP or are<br>
prepared as monoclonal antibodies reactive with LMP using the technique of<br>
Kohler and Milstein, Nature. 256:495-97 (1975). The LMP specific antibodies<br>
are raised by immunizing animals such as, for example, mice, rats, guinea<br>
pigs, rabbits, goats or horses, with an appropriate concentration of LMP either<br>
with or without an immune adjuvant.<br>
In this process, preimmune serum is collected prior to the first<br>
immunization. Each animal receives between about 0.1 mg and about 1000<br>
mg of LMP associated with an acceptable immune adjuvant, if desired. Such<br>
acceptable adjuvants include, but are not limited to, Freund's complete,<br>
Freund's incomplete, alum-precipitate, water in oil emulsion containing<br>
Corynebacterium parvum and tRNA adjuvants. The initial immunization<br>
consists of LMP in, preferably, Freund's complete adjuvant injected at multiple<br>
sites either subcutaneously (SC), intraperitoneally (IP) or both. Each animal is<br>
bled at regular intervals, preferably weekly, to determine antibody titer. The<br>
animals may or may not receive booster injections following the initial<br>
immunization. Those animals receiving booster injections are generally given<br>
an equal amount of the antigen in Freund's incomplete adjuvant by the same<br>
route. Booster injections are given at about three week intervals until maximal<br>
titers are obtained. At about 7 days after each booster immunization or about<br>
weekly after a single immunization, the animals are bled, the serum collected,<br>
and aliquots are stored at about -20° C.<br>
Monoclonal antibodies (mAb) reactive with LMP are prepared by<br>
immunizing inbred mice, preferably Balb/c mice, with LMP. The mice are<br>
immunized by the IP or SC route with about 0.1 mg to about 10 mg, preferably<br>
about 1 mg, of LMP in about 0.5 ml buffer or saline incorporated in an equal<br>
volume of an acceptable adjuvant, as discussed above. Freund's complete<br>
adjuvant is preferred. The mice receive an initial immunization on day 0 and<br>
are rested for about 3-30 weeks. Immunized mice are given one or more<br>
booster immunizations of about 0.1 to about 10 mg of LMP in a buffer solution<br>
such as phosphate buffered saline by the intravenous (IV) route. Lymphocytes<br>
from antibody-positive mice, preferably splenic lymphocytes, are obtained by<br>
removing the spleens from immunized mice by standard procedures known in<br>
the art. Hybridoma cells are produced by mixing the splenic lymphocytes with<br>
an appropriate fusion partner, preferably myeloma cells, under conditions<br>
which will allow the formation of stable hybridomas. Fusion partners may<br>
include, but are not limited to: mouse myelomas P3/NS1/Ag 4-1; MPC-11; S-<br>
194 and Sp 2/0, with Sp 2/0 being preferred. The antibody producing cells and<br>
myeloma cells are fused in polyethylene glycol, about 1000 mol. wt., at<br>
concentrations from about 30% to about 50%. Fused hybridoma cells are<br>
selected by growth in hypoxanthine, thymidine and aminopterin in<br>
supplemented Dulbecco's Modified Eagles Medium (DMEM) by procedures<br>
known in the art. Supernatant fluids are collected from growth positive wells on<br>
about days 14,18, and 21, and are screened for antibody production by an<br>
immunoassay such as solid phase immunoradioassay (SPIRA) using LMP as<br>
the antigen. The culture fluids are also tested in the Ouchterlony precipitation<br>
assay to determine the isotype of the mAb. Hybridoma cells from antibody<br>
positive wells are cloned by a technique such as the soft agar technique of<br>
MacPherson, "Soft Agar Techniques", in Tissue Culture Methods and<br>
Applications. Kruse and Paterson (eds.), Academic Press (1973). See, also,<br>
Harlow et a/., Antibodies: A Laboratory Manual. Cold Spring Laboratory<br>
(1988).<br>
Monoclonal antibodies may also be produced in vivo by injection of<br>
pristane- primed Balb/c mice, approximately 0.5 ml per mouse, with about<br>
2x106 to about 6x106 hybridoma cells about 4 days after priming. Ascites fluid<br>
is collected at approximately 8-12 days after cell transfer and the monoclonal<br>
antibodies are purified by techniques known in the art.<br>
In vitro production in anti-LMP mAb is carried out by growing the<br>
hydridoma cell line in DMEM containing about 2% fetal calf serum to obtain<br>
sufficient quantities of the specific mAb. The mAb are purified by techniques<br>
known in the art.<br>
Antibody titers of ascites or hybridoma culture fluids are determined by<br>
various serological or immunological assays, which include, but are not limited<br>
to, precipitation, passive agglutination, enzyme-linked immunosorbent antibody<br>
(ELISA) technique and radioimmunoassay (RIA) techniques. Similar assays<br>
are used to detect the presence of the LMP in body fluids or tissue and cell<br>
extracts.<br>
It is readily apparent to those skilled in the art that the above described<br>
methods for producing monospecific antibodies may be utilized to produce<br>
antibodies specific for polypeptide fragments of LMP, full-length nascent LMP<br>
polypeptide, or variants or alleles thereof.<br>
In another embodiment, the invention is directed to alternative splice<br>
variants of HLMP-1. PCR analysis of human heart cDNA revealed mRNA for<br>
two HLMP alternative splice variants, named HLMP-2 and HLMP-3, that differ<br>
from HLMP-1 in a region between base pairs 325 and 444 in the hLMP-1<br>
sequence. The HLMP-2 sequence has a 119 base pair deletion and an<br>
insertion of 17 base pairs in this region. These changes preserve the reading<br>
frame, resulting in a 423 amino acid protein, which compared to HLMP-1, has a<br>
net loss of 34 amino acids (40 amino acids deleted plus 6 inserted amino<br>
acids). HLMP-2 contains the c-terminal LIM domains that are present in<br>
HLMP-1.<br>
Compared to HLMP-1, HLMP-3 has no deletions, but it does have the<br>
same 17 base pair insertion at position 444. This insertion shifts the reading<br>
frame, causing a stop codon at base pairs 459-461. As a result, HLMP-3<br>
encodes a protein of 153 amino acids. This protein lacks the c-terminal LIM<br>
domains that are present in HLMP-1 and HLMP-2. The predicted size of the<br>
proteins encoded by HLMP-2 and HLMP-3 was confirmed by western blot<br>
analysis.<br>
PCR analysis of the tissue distribution of the three splice variants<br>
revealed that they are differentially expressed, with specific isoforms<br>
predominating in different tissues. HLMP-1 is apparently the predominant form<br>
expressed in leukocytes, spleen, lung, placenta, and fetal liver. HLMP-2<br>
appears to be the predominant isoform in skeletal muscle, bone marrow, and<br>
heart tissue. HLMP-3, however, was not the predominant isoform in any tissue<br>
examined.<br>
Overexpression of HLMP-3 in secondary rat osteoblast cultures induced<br>
bone nodule formation (287±56) similar to the effect seen for glucicorticoid<br>
(272±7) and HLMP-1 (232±200). Since HLMP-3 lacks the C-terminal LIM<br>
domains, there regions are not required for osteoinductive activity.<br>
Overexpression of HLMP-2, however, did not induce nodule formation (11±3).<br>
These data suggest that the amino acids encoded by the deleted 119 base<br>
pairs are necessary for osteoinduction. The data also suggest that the<br>
distribution of HLMP splice variants may be important for tissue-specific<br>
function. Surprisingly, we have shown that HLMP-2 inhibits steroid-induced<br>
osteoblast formation in secondary rat osteoblast cultures. Therefore, HLMP-2<br>
will have therapeutic utility in clinical situations where bone formation is not<br>
desirable.<br>
On July 22, 1997, a sample of 10-4/RLMP in a vector designated<br>
pCMV2/RLMP (which is vector pRc/CMV2 with insert 10-4 clone/RLMP) was<br>
deposited with the American Type Culture Collection (ATCC), 12301 Parklawn<br>
Drive, Rockville, MD 20852. The culture accession number for that deposit is<br>
209153. On March 19, 1998, a sample of the vector pHis-A with insert HLPM-<br>
1s was deposited at the American Type Culture Collection ("ATCC"). The<br>
culture accession number for that deposit is 209698. On April 14, 2000,<br>
samples of plasmids pHAhLMP-2 (vector pHisA with cDNA insert derived from<br>
human heart muscle cDNA with HLMP-2) and pHAhLMP-3 (vector pHisA with<br>
cDNA insert derived from human heart muscle cDNA with HLMP-3) were<br>
deposited with the ATCC, 10801 University Blvd., Manassas, VA, 20110-2209,<br>
USA, under the conditions of the Budapest treaty. The accession numbers for<br>
these deposits are PTA-1698 and PTA-1699, respectively. These deposits, as<br>
required by the Budapest Treaty, will be maintained in the ATCC for at least<br>
30 years and will be made available to the public upon the grant of a patent<br>
disclosing them. It should be understood that the availability of a deposit does<br>
not constitute a license to practice the subject invention in derogation of patent<br>
rights granted by government action.<br>
In assessing the nucleic acids, proteins, or antibodies of the invention,<br>
enzyme assays, protein purification, and other conventional biochemical<br>
methods are employed. DNA and RNA are analyzed by Southern blotting and<br>
Northern blotting techniques, respectively. Typically, the samples analyzed are<br>
size fractionated by gel electrophoresis. The DNA or RNA in the gels are then<br>
transferred to nitrocellulose or nylon membranes. The blots, which are replicas<br>
of sample patterns in the gels, were then hybridized with probes. Typically, the<br>
probes are radiolabelled, preferably with 32P, although one could label the<br>
probes with other signal-generating molecules known to those in the art.<br>
Specific bands of interest can then be visualized by detection systems, such as<br>
autoradiography.<br>
For purposes of illustrating preferred embodiments of the present<br>
invention, the following, non-limiting examples are included. These results<br>
demonstrate the feasibiliity of inducing or enhancing the formation of bone<br>
using the LIM mineralization proteins of the invention, and the isolated nucleic<br>
acid molecules encoding those proteins.<br>
Example 1: Calvarial Cell Culture<br>
Rat calvarial cells, also known as rat osteoblasts ("ROB"), were obtained<br>
from<br>
20-day pre-parturition rats as previously described. Boden et al.,<br>
Endocrinology. 137(8):3401-07 (1996). Primary cultures were grown to<br>
confluence (7 days), trypsinized, and passed into 6-well plates (1 x 105 cells/35<br>
mm well) as first subculture cells. The subculture cells, which were confluent<br>
at day 0, were grown for an additional 7 days. Beginning on day 0, media were<br>
changed and treatments (Trm and/or BMPs) were applied, under a laminar flow<br>
hood, every 3 or 4 days. The standard culture protocol was as follows: days 1-<br>
7, MEM, 10% FBS, 50 g/ml ascorbic acid, ± stimulus; days 8-14, BGJb<br>
medium, 10% FBS, 5mM -GlyP (as a source of inorganic phosphate to permit<br>
mineralization). Endpoint analysis of bone nodule formation and osteocalcin<br>
secretion was performed at day 14. The dose of BMP was chosen as 50 ng/ml<br>
based on pilot experiments in this system that demonstrated a mid-range effect<br>
on the dose-response curve for all BMPs studied.<br>
EXAMPLE 2: Antisense Treatment and Cell Culture<br>
To explore the potential functional role of LMP-1 during membranous<br>
bone formation, we synthesized an antisense oligonucleotide to block LMP-1<br>
mRNA translation and treated secondary osteoblast cultures that were<br>
undergoing differentiation initiated by glucocorticoid. Inhibition of RLMP<br>
expression was accomplished with a highly specific antisense oligonucleotide<br>
(having no significant homologies to known rat sequences) corresponding to a<br>
25 bp sequence spanning the putative translational start site (SEQ ID NO: 42).<br>
Control cultures either did not receive oligonucleotide or they received sense<br>
oligonucleotide. Experiments were performed in the presence (preincubation)<br>
and absence of lipofectamine. Briefly, 22 g of sense or antisense RLMP<br>
oligonucleotide was incubated in MEM for 45 minutes at room temperature.<br>
Following that incubation, either more MEM or pre-incubated<br>
lipofectamine/MEM (7% v/v; incubated 45 minutes at room temperature) was<br>
added to achieve an oligonucleotide concentration of 0.2 M. The resulting<br>
mixture was incubated for 15 minutes at room temperature. Oligonucleotide<br>
mixtures were then mixed with the appropriate medium, that is,<br>
MEM/Ascorbate/±Trm, to achieve a final oligonucleotide concentration of 0.1<br>
M.<br>
Cells were incubated with the appropriate medium (±stimulus) in the<br>
presence or absence of the appropriate oligonucleotides. Cultures originally<br>
incubated with lipofectamine were re-fed after 4 hours of incubation (37°C; 5%<br>
CO2) with media containing neither lipofectamine nor oligonucleotide. All<br>
cultures, especially cultures receiving oligonucleotide, were re-fed every 24<br>
hours to maintain oligonucleotide levels.<br>
LMP-1 antisense oligonucleotide inhibited mineralized nodule formation<br>
and osteocalcin secretion in a dose-dependent manner, similar to the effect of<br>
BMP-6 oligonucleotide. The LMP-1 antisense block in osteoblast<br>
differentiation could not be rescued by addition of exogenous BMP-6, while the<br>
BMP-6 antisense oligonucleotide inhibition was reversed with addition of BMP-<br>
6. This experiment further confirmed the upstream position of LMP-1 relative<br>
to BMP-6 in the osteoblast differentiation pathway. LMP-1 antisense<br>
oligonucleotide also inhibited spontaneous osteoblast differentiation in primary<br>
rat osteoblast cultures.<br>
EXAMPLE 3: Quantitation of Mineralized Bone Nodule Formation<br>
Cultures of ROBs prepared according to Examples 1 and 2 were fixed<br>
overnight in 70% ethanol and stained with von Kossa silver stain. A semi-<br>
automated computerized video image analysis system was used to quantitate<br>
nodule count and nodule area in each well. Boden et al., Endocrinology.<br>
137(8):3401-07 (1996). These values were then divided to calculate the area<br>
per nodule values. This automated process was validated against a manual<br>
counting technique and demonstrated a correlation coefficient of 0.92<br>
(p 
mean (S.E.M.) calculated from 5 or 6 wells at each condition. Each experiment<br>
was confirmed at least twice using cells from different calvarial preparations.<br>
EXAMPLE 4: Quantitation of Osteocalcin Secretion<br>
Osteocalcin levels in the culture media were measured using a<br>
competitive radioimmunoassay with a monospecific polyclonal antibody (Pab)<br>
raised in our laboratory against the C-terminal nonapeptide of rat osteocalcin<br>
as described in Nanes et al., Endocrinology. 127:588 (1990). Briefly, 1 g of<br>
nonapeptide was iodinated with 1 mCi 125l-Na by the lactoperoxidase method.<br>
Tubes containing 200 I of assay buffer (0.02 M sodium phosphate, 1 mM<br>
EDTA, 0.001% thimerosal, 0.025% BSA) received media taken from cell<br>
cultures or osteocalcin standards (0 -12,000 fmole) at 100 I/tube in assay<br>
buffer. The Pab (1:40,000; 100 I) was then added, followed by the iodinated<br>
peptide (12,000 cpm; 100 I). Samples tested for non-specific binding were<br>
prepared similarly but contained no antibody.<br>
Bound and free PAbs were separated by the addition of 700 I goat anti-<br>
rabbit IgG, followed by incubation for 18 hours at 4°C. After samples were<br>
centrifuged at 1200 rpm for 45 minutes, the supernatants were decanted and<br>
the precipitates counted in a gamma counter. Osteocalcin values were<br>
reported in fmole/1001, which was then converted to pmole/ml medium (3-day<br>
production) by dividing those values by 100. Values were expressed as the<br>
mean ± S.E.M. of triplicate determinations for 5-6 wells for each condition.<br>
Each experiment was confirmed at least two times using cells from different<br>
calvarial preparations.<br>
EXAMPLE 5: Effect of Trm and RLMP on Mineralization In Vitro<br>
There was little apparent effect of either the sense or antisense<br>
oligonucleotides on the overall production of bone nodules in the non-<br>
stimulated cell culture system. When ROBs were stimulated with Trm,<br>
however, the antisense oligonucleotide to RLMP inhibited mineralization of<br>
nodules by &gt; 95%. The addition of exogenous BMP-6 to the oligonucleotide-<br>
treated cultures did not rescue the mineralization of RLMP-antisense-treated<br>
nodules.<br>
Osteocalcin has long been synonymous with bone mineralization, and<br>
osteocalcin levels have been correlated with nodule production and<br>
mineralization. The RLMP-antisense oligonucleotide significantly decreases<br>
osteocalcin production, but the nodule count in antisense-treated cultures does<br>
not change significantly. In this case, the addition of exogenous BMP-6 only<br>
rescued the production of osteocalcin in RLMP-antisense-treated cultures by<br>
10-15%. This suggests that the action of RLMP is downstream of, and more<br>
specific than, BMP-6.<br>
EXAMPLE 6: Harvest and Purification of RNA<br>
Cellular RNA from duplicate wells of ROBs (prepared according to<br>
Examples 1 and 2 in 6-well culture dishes) was harvested using 4M guanidine<br>
isothiocyanate (GIT) solution to yield statistical triplicates. Briefly, culture<br>
supernatant was aspirated from the wells, which were then overiayed with 0.6<br>
ml of GIT solution per duplicate well harvest. After adding the GIT solution, the<br>
plates were swirled for 5-10 seconds (being as consistent as possible).<br>
Samples were saved at -70°C for up to 7 days before further processing.<br>
RNA was purified by a slight modification of standard methods<br>
according to Sambrook et al., Molecular Cloning: a Laboratory Manual. 2nd<br>
Ed., chapter 7.19, Cold Spring Harbor Press (1989). Briefly, thawed samples<br>
received 60 I 2.0 M sodium acetate (pH 4.0), 550 I phenol (water saturated)<br>
and 150 I chloroform:isoamyl alcohol (49:1). After vortexing, the samples were<br>
centrifuged (10000 x g; 20 minutes; 4°C), the aqueous phase transferred to a<br>
fresh tube, 600 I isopropanol was added and the RNA precipitated overnight at<br>
-20°C.<br>
Following the overnight incubation, the samples were centrifuged<br>
(10000 x g; 20 minutes) and the supernatant was aspirated gently. The pellets<br>
were resuspended in 400 I DEPC-treated water, extracted once with<br>
phenol:chloroform (1:1), extracted with chlorofornrisoamyl alcohol (24:1) and<br>
precipitated overnight at -20°C after addition of 40 I sodium acetate (3.0 M; pH<br>
5.2) and 1.0 ml absolute ethanol. To recover the cellular RNA, the samples<br>
were centrifuged (10000 x g; 20 min), washed once with 70% ethanol, air dried<br>
for 5-10 minutes and resuspended in 20 I of DEPC-treated water. RNA<br>
concentrations were calculated from optical densities that were determined<br>
with a spectrophotometer.<br>
EXAMPLE 7: Reverse Transcription-Polymerase Chain Reaction<br>
Heated total RNA (5 g in 10.5 I total volume DEPC-H2O at 65°C for 5<br>
minutes) was added to tubes containing 4 I 5X MMLV-RT buffer, 2 I dNTPs, 2 I<br>
dT17 primer (10 pmol/ml), 0.5 I RNAsin (40U/ml) and 1 I MMLV-RT (200<br>
units/1). The samples were incubated at 37°C for 1 hour, then at 95°C for 5<br>
minutes to inactivate the MMLV-RT. The samples were diluted by addition of<br>
80 I of water.<br>
Reverse-transcribed samples (5 I) were subjected to polymerase-chain<br>
reaction using standard methodologies (50 I total volume). Briefly, samples<br>
were added to tubes containing water and appropriate amounts of PCR buffer,<br>
25 mM MgCI2, dNTPs, forward and reverse primers for glyceraldehyde 3-<br>
phosphate dehydrogenase (GAP, a housekeeping gene) and/or BMP-6), 32P-<br>
dCTP, and Taq polymerase. Unless otherwise noted, primers were<br>
standardized to run consistently at 22 cycles (94°C, 30"; 58°C, 30"; 72°C, 20").<br>
EXAMPLE 8: Quantitation of RT-PCR Products by Polyacrylamide Gel<br>
Electrophoresis (PAGE) and Phosphorlmager Analysis<br>
RT-PCR products received 5 I/tube loading dye, were mixed, heated at<br>
65°C for 10 min and centrifuged. Ten I of each reaction was subjected to<br>
PAGE (12% polyacrylamide:bis; 15 V/well; constant current) under standard<br>
conditions. Gels were then incubated in gel preserving buffer (10% v/v<br>
glycerol, 7% v/v acetic acid, 40% v/v methanol, 43% deionized water) for 30<br>
minutes, dried (80°C) in vacuo for 1-2 hours and developed with an<br>
electronically-enhanced phosphoresence imaging system for<br>
6-24 hours. Visualized bands were analyzed. Counts per band were plotted<br>
graphically.<br>
EXAMPLE 9: Differential Display PCR<br>
RNA was extracted from cells stimulated with glucocorticoid (Trm, 1<br>
nM). Heated, DNase-treated total RNA (5 g in 10.5 I total volume in DEPC-<br>
H2O at 65°C for 5 minutes) was reverse transcribed as described in Example 7,<br>
but H-T11M (SEQ ID. NO: 4) was used as the MMLV-RT primer. The resulting<br>
cDNAs were PCR-amplified as described above, but with various commercial<br>
primer sets (for example,<br>
H-T11G (SEQ ID NO: 4) and H-AP-10 (SEQ ID. NO: 5); GenHunter Corp,<br>
Nashville, TN). Radiolabelled PCR products were fractionated by gel<br>
electrophoresis on a DNA sequencing gel. After electrophoresis, the resulting<br>
gels were dried in vacua and autoradiographs were exposed overnight. Bands<br>
representing differentially-expressed cDNAs were excised from the gel and<br>
reamplified by PCR using the method of Conner et al., Proc. Natl. Acad. Sci.<br>
USA. 88:278 (1983). The products of PCR reamplification were cloned into<br>
the vector PCR-II (TA cloning kit; InVitrogen, Carlsbad, CA).<br>
EXAMPLE 10: Screening of a UMR 106 Rat Osteosarcoma Cell cDNA Library<br>
A UMR 106 library (2.5 x 1010 pfu/ml) was plated at 5 x 104 pfu/ml onto<br>
agar plates (LB bottom agar) and the plates were incubated overnight at 37°C.<br>
Filter membranes were overlaid onto plates for two minutes. Once removed,<br>
the filters were denatured, rinsed, dried and UV cross-linked. The filters were<br>
then incubated in pre-hyridization buffer (2X PIPES [pH 6.5], 5% formamide,<br>
1 % SDS and 100 g/ml denatured salmon sperm DNA) for 2 h at 42°C. A 260<br>
base-pair radiolabelled probe (SEQ ID NO: 3; 32P labelled by random priming)<br>
was added to the entire hybridization mix/filters, followed by hybridization for 18<br>
hours at 42°C. The membranes were washed once at room temperature (10<br>
min, 1 x SSC, 0.1% SDS) and three times at 55°C (15 min, 0.1 x SSC, 0.1%<br>
SDS).<br>
After they were washed, the membranes were analyzed by<br>
autoradiography as described above. Positive clones were plaque purified.<br>
The procedure was repeated with a second filter for four minutes to minimize<br>
spurious positives. Plaque-purified clones were rescued as lambda SK(-)<br>
phagemids. Cloned cDNAs were sequenced as described below.<br>
EXAMPLE 11: Sequencing of Clones<br>
Cloned cDNA inserts were sequenced by standard methods. Ausubel et<br>
a/., Current Protocols in Molecular Biology. Wiley Interscience (1988). Briefly,<br>
appropriate concentrations of termination mixture, template and reaction<br>
mixture were subjected to an appropriate cycling protocol (95°C,30s; 68°C,30s;<br>
72°C,60s; x 25). Stop mixture was added to terminate the sequencing<br>
reactions. After heating at 92°C for 3 minutes, the samples were loaded onto a<br>
denaturing 6% polyacrylamide sequencing gel (29:1 acrylamide:bis-<br>
acrylamide). Samples were electrophoresed for about 4 hours at 60 volts,<br>
constant current. After electrophoresis, the gels were dried in vacuo and<br>
autoradiographed.<br>
The autoradiographs were analyzed manually. The resulting sequences<br>
were screened against the databases maintained by the National Center for<br>
Biotechnology Information (NIH, Bethesda, MD; http://www.ncbi.nlm.nih.gov/)<br>
using the BLASTn program set with default parameters. Based on the<br>
sequence data, new sequencing primers were prepared and the process was<br>
repeated until the entire gene had been sequenced. All sequences were<br>
confirmed a minimum of three times in both orientations.<br>
Nucleotide and amino acid sequences were also analyzed using the<br>
PCGENE software package (version 16.0). Per cent homology values for<br>
nucleotide sequences were calculated by the program NALIGN, using the<br>
following parameters: weight of non-matching nucleotides, 10; weight of non-<br>
matching gaps, 10; maximum number of nucleotides considered, 50; and<br>
minimum number of nucleotides considered, 50.<br>
For amino acid sequences, per cent homology values were calculated<br>
using PALIGN. A value of 10 was selected for both the open gap cost and the<br>
unit gap cost.<br>
FXAMPLE 12: Cloning of RLMP cDNA<br>
The differential display PCR amplification products described in<br>
Example 9 contained a major band of approximately 260 base pairs. This<br>
sequence was used to screen a rat osteosarcoma (UMR 106) cDNA library.<br>
Positive clones were subjected to nested primer analysis to obtain the primer<br>
sequences necessary for amplifying the full length cDNA. (SEQ. ID NOs: 11,<br>
12, 29, 30 and 31) One of those positive clones selected for further study was<br>
designated clone 10-4.<br>
Sequence analysis of the full-length cDNA in clone 10-4, determined by<br>
nested primer analysis, showed that clone 10-4 contained the original 260<br>
base-pair fragment identified by differential display PCR. Clone 10-4 (1696<br>
base pairs; SEQ ID NO: 2) contains an open reading frame of 1371 base pairs<br>
encoding a protein having 457 amino acids (SEQ ID NO: 1). The termination<br>
codon, TGA, occurs at nucleotides 1444-1446. The polyadenylation signal at<br>
nucleotides 1675-1680, and adjacent poly(A)+ tail, was present in the 3'<br>
noncoding region. There were two potential N-glycosylation sites, Asn-Lys-Thr<br>
and Asn-Arg-Thr, at amino acid positions 113-116 and 257-259 in SEQ ID NO:<br>
1, respectively. Two potential cAMP- and cGMP-dependent protein kinase<br>
phosphorylation sites, Ser and Thr, were found at amino acid positions 191<br>
and 349, respectively. There were five potential protein kinase C<br>
phosphorylation sites, Ser or Thr, at amino acid positions 3, 115, 166, 219,<br>
442. One potential ATP/GTP binding site motif A (P-loop), Gly-Gly-Ser-Asn-<br>
Asn-Gly-Lys-Thr, was determined at amino acid positions 272-279.<br>
In addition, two highly conserved putative LIM domains were found at<br>
amino acid positions 341-391 and 400-451. The putative LIM domains in this<br>
newly identified rat cDNA clone showed considerable homology with the LIM<br>
domains of other known LIM proteins. However, the overall homology with<br>
other rat LIM proteins was less than 25%. RLMP (also designated 10-4) has<br>
78.5% amino acid homology to the human enigma protein (see U.S. Patent No.<br>
5,504,192), but only 24.5% and 22.7% amino acid homology to its closest rat<br>
homologs, CLP-36 and RIT-18, respectively.<br>
FXAMPLE 13: Northern Blot Analysis of RLMP Expression<br>
Thirty g of total RNA from ROBs, prepared according to Examples 1 and<br>
2, was size fractionated by formaldehyde gel electrophoresis in 1% agarose<br>
flatbed gels and osmotically transblotted to nylon membranes. The blot was<br>
probed with a 600 base pair EcoR1 fragment of full-length 10-4 cDNA labeled<br>
with 32P-dCTP by random priming.<br>
Northern blot analysis showed a 1.7 kb mRNA species that hybridized<br>
with the RLMP probe. RLMP mRNA was up-regulated approximately 3.7-fold<br>
in ROBs after 24 hours exposure to BMP-6. No up-regulation of RMLP<br>
expression was seen in BMP-2 or BMP-4-stimulated ROBs at 24 hours.<br>
FXAMPLE 14: Statistical Methods<br>
For each reported nodule/osteocalcin result, data from 5-6 wells from a<br>
representative experiment were used to calculate the mean ± S.E.M. Graphs<br>
may be shown with data normalized to the maximum value for each parameter<br>
to allow simultaneous graphing of nodule counts, mineralized areas and<br>
osteocalcin.<br>
For each reported RT-PCR, RNase protection assay or Western blot<br>
analysis, data from triplicate samples of representative experiments, were used<br>
to determine the mean ± S.E.M. Graphs may be shown normalized to either<br>
day 0 or negative controls and expressed as fold-increase above control<br>
values.<br>
Statistical significance was evaluated using a one-way analysis of<br>
variance with post-hoc multiple comparison corrections of Bonferroni as<br>
appropriate. D. V. Huntsberger, The Analysis of Variance," in Elements of<br>
Statistical Variance. P. Billingsley (ed.), pp. 298-330, Allyn &amp; Bacon Inc.,<br>
Boston, MA (1977) and Sigmastat, Jandel Scientific, Corte Madera, CA. Alpha<br>
levels for significance were defined as p 
EXAMPLE 15: Detection of Rat LIM Mineralization Protein by Western Blot<br>
Analysis<br>
Polyclonal antibodies were prepared according to the methods of<br>
England et ai, Biochim.Biophys. Acta. 623:171 (1980) and Timmer et a/., J<br>
Biol. Chem.. 268:24863 (1993).<br>
HeLa cells were transfected with pCMV2/RLMP. Protein was harvested<br>
from the transfected cells according to the method of Hair et al, Leukemia<br>
Research. 20:1 (1996). Western Blot Analysis of native RLMP was performed<br>
as described by Towbin et al., Proc. Natl. Acad. Sci. USA. 76:4350 (1979).<br>
EXAMPLE 16: Synthesis of the Rat LMP-Unique (RLMPU) derived Human<br>
PCR product<br>
Based on the sequence of the rat LMP-1 cDNA, forward and reverse<br>
PCR primers (SEQ ID NOs: 15 and 16) were synthesized and a unique 223<br>
base-pair sequence was PCR amplified from the rat LMP-1 cDNA. A similar<br>
PCR product was isolated from human MG63 osteosarcoma cell cDNA with the<br>
same PCR primers.<br>
RNA was harvested from MG63 osteosarcoma cells grown in T-75<br>
flasks. Culture supernatant was removed by aspiration and the flasks were<br>
overlayed with 3.0 ml of GIT solution per duplicate, swirled for 5-10 seconds,<br>
and the resulting solution was transferred to 1.5 ml eppendorf tubes (5 tubes<br>
with 0.6 ml/tube). RNA was purified by a slight modification of standard<br>
methods, for example, see Sambrook et al., Molecular Cloning: A Laboratory<br>
Manual, chapter 7, page 19, Cold Spring Harbor Laboratory Press (1989) and<br>
Boden et al., Endocrinology. 138:2820-28 (1997). Briefly, the 0.6 ml samples<br>
received 60 I 2.0 M sodium acetate (pH 4.0), 550 I water saturated phenol and<br>
150 I chloroform:isoamyl alcohol (49:1). After addiiton of those reagents, the<br>
samples were vortexed, centrifuged (10000 x. g; 20 min; 4C) and the aqueous<br>
phase transferred to a fresh tube. Isopropanol (600 I) was added and the RNA<br>
was precipitated overnight at -20°C. The samples were centrifuged (10000 x<br>
g; 20 minutes) and the supernatant was aspirated gently. The pellets were<br>
resuspended in 400 I of DEPC-treated water, extracted once with<br>
phenokchloroform (1:1), extracted with chloroform;isoamyl alcohol (24:1) and<br>
precipitated overnight at -20°C in 40 I sodium acetate (3.0 M; pH 5.2) and 1.0<br>
ml absolute ethanol. After precipitation, the samples were centrifuged (10000<br>
x g; 20 min), washed once with 70% ethanol, air dried for 5-10 minutes and<br>
resuspended in 20 I of DEPC-treated water. RNA concentrations were derived<br>
from optical densities.<br>
Total RNA (5 g in 10.5 L total volume in DEPC-H2O) was heated at 65°C<br>
for 5 minutes, and then added to tubes containing 4 I 5X MMLV-RT buffer, 2 I<br>
dNTPs, 2 I dT17 primer (10 pmol/ml), 0.5 I RNAsin (40 U/ml) and 1 I MMLV-RT<br>
(200 units/1). The reactions were incubated at 37°C for 1 hour. Afterward, the<br>
MMLV-RT was inactivated by heating at 95°C for 5 minutes. The samples<br>
were diluted by addition of 80 L water.<br>
Transcribed samples (5 I) were subjected to polymerase-chain reaction<br>
using standard methodologies (50 I total volume). Boden et al., Endocrinology.<br>
138:2820-28 (1997); Ausubel et al., "Quantitation of rare DNAs by the<br>
polymerase chain reaction", in Current Protocols in Molecular Biology, chapter<br>
15.31-1, Wiley &amp; Sons, Trenton, NJ (1990). Briefly, samples were added to<br>
tubes containing water and appropriate amounts of PCR buffer (25 mM MgC12,<br>
dNTPs, forward and reverse primers (for RLMPU; SEQ ID NOs: 15 and 16),<br>
32P-dCTP, and DNA polymerase. Primers were designed to run consistently at<br>
22 cycles for radioactive band detection and 33 cycles for amplification of PCR<br>
product for use as a screening probe (94°C, 30 sec, 58°C, 30 sec; 72°C, 20<br>
sec).<br>
Sequencing of the agarose gel-purified MG63 osteosarcoma-derived<br>
PCR product gave a sequence more than 95% homologous to the RLMPU<br>
PCR product. That sequence is designated HLMP unique region (HLMPU;<br>
SEQ ID NO: 6).<br>
EXAMPLE 17: Screening of reverse-transcriptase-derived MG63 cDNA<br>
Screening was performed with PCR using specific primers (SEQ ID<br>
NOs: 16 and 17) as described in Example 7. A 717 base-pair MG63 PCR<br>
product was agarose gel purified and sequenced with the given primers (SEQ.<br>
ID NOs: 12,15,16, 17, 18, 27 and 28). Sequences were confirmed a<br>
minimum of two times in both directions. The MG63 sequences were aligned<br>
against each other and then against the full-length rat LMP cDNA sequence to<br>
obtain a partial human LMP cDNA sequence (SEQ ID NO: 7).<br>
EXAMPLE 18: Screening of a Human Heart cDNA Library<br>
Based on Northern blot experiments, it was determined that LMP-1 is<br>
expressed at different levels by several different tissues, including human heart<br>
muscle. A human heart cDNA library was therefore examined. The library was<br>
plated at 5 x 104 pfu/ml onto agar plates (LB bottom agar) and plates were<br>
grown overnight at 37° C. Filter membranes were overlaid onto the plates for<br>
two minutes. Afterward, the filters denatured, rinsed, dried, UV cross-linked<br>
and incubated in pre-hyridization buffer (2X PIPES [pH 6.5]; 5% formamide,<br>
1% SDS, 100 g/ml denatured salmon sperm DNA) for 2 h at 42°C. A<br>
radiolabelled, LMP-unique, 223 base-pair probe (32P, random primer labelling;<br>
SEQ ID NO: 6) was added and hybridized for 18 h at 42°C. Following<br>
hybridization, the membranes were washed once at room temperature (10 min,<br>
1 x SSC, 0.1% SDS) and three times at 55°C (15 min, 0.1 x SSC, 0.1% SDS).<br>
Double-positive plaque-purified heart library clones, identified by<br>
autoradiography, were rescued as lambda phagemids according to the<br>
manufacturers' protocols (Stratagene, La Jolla, CA).<br>
Restriction digests of positive clones yielded cDNA inserts of varying<br>
sizes. Inserts greater than 600 base-pairs in length were selected for initial<br>
screening by sequencing. Those inserts were sequenced by standard<br>
methods as described in Example 11.<br>
One clone, number 7, was also subjected to automated sequence<br>
analysis using primers corresponding to SEQ ID NOs: 11-14,16 and 27. The<br>
sequences obtained by these methods were routinely 97-100% homologous.<br>
Clone 7 (Partial Human LMP-1 cDNA from a heart library; SEQ. ID NO: 8)<br>
contained sequence that was more than 87% homologous to the rat LMP<br>
cDNA sequence in the translated region.<br>
EXAMPLE 19: Determination of Full-Lenath Human LMP-1 cDNA<br>
Overlapping regions of the MG63 human osteosarcoma cell cDNA<br>
sequence and the human heart cDNA clone 7 sequence were used to align<br>
those two sequences and derive a complete human cDNA sequence of 1644<br>
base-pairs. NALIGN, a program in the PCGENE software package, was used<br>
to align the two sequences. The overlapping regions of the two sequences<br>
constituted approximately 360 base-pairs having complete homology except for<br>
a single nucleotide substitution at nucleotide 672 in the MG63 cDNA (SEQ ID<br>
NO: 7) with clone 7 having an "A" instead of a "G" at the corresponding<br>
nucleotide 516 (SEQ ID NO: 8).<br>
The two aligned sequences were joined using SEQIN, another<br>
subprogram of PCGENE, using the "G" substitution of the MG63 osteosarcoma<br>
cDNA clone. The resulting sequence is shown in SEQ ID NO: 9. Alignment of<br>
the novel human-derived sequence with the rat LMP-1 cDNA was<br>
accomplished with NALIGN. The full-length human LMP-1 cDNA sequence<br>
(SEQ. ID NO: 9) is 87.3% homologous to the translated portion of rat LMP-1<br>
cDNA sequence.<br>
EXAMPLE 20: Determination of Amino Acid Sequence of Human I MP-1<br>
The putative amino acid sequence of human LMP-1 was determined<br>
with the PCGENE subprogram TRANSL. The open reading frame in SEQ ID<br>
NO: 9 encodes a protein comprising 457 amino acids (SEQ. ID NO: 10). Using<br>
the PCGENE subprogram Palign, the human LMP-1 amino acid sequence was<br>
found to be 94.1% homologous to the rat LMP-1 amino acid sequence.<br>
EXAMPLE 21: Determination of the 5 Prime Untranslated Region of the<br>
Human LMP cDNA<br>
MG63 5' cDNA was amplified by nested RT-PCR of MG63 total RNA<br>
using a 51 rapid amplification of cDNA ends (51 RACE) protocol. This method<br>
included first strand cDNA synthesis using a lock-docking oligo (dT) primer with<br>
two degenerate nucleotide positions at the 3' end (Chenchik et a/.,<br>
CLONTECHniaues. X:5 (1995); Borson et al, PC Methods Applic. 2:144<br>
(1993)). Second-strand synthesis is performed according to the method of<br>
Gubler et ai, Gene. 25:263 (1983), with a cocktail of Escherichia coli DNA<br>
polymerase I, RNase H, and E. coli DNA ligase. After creation of blunt ends<br>
with T4 DNA polymerase, double-stranded cDNA was ligated to the fragment<br>
(5' -CTAATACGACTCACTATAGGGCTCGAGCGGCCGCCCGGGCAGGT- 3')<br>
(SEQ.ID NO: 19). Prior to RACE, the adaptor-ligated cDNA was diluted to a<br>
concentration suitable for Marathon RACE reactions (1:50). Adaptor-ligated<br>
double-stranded cDNA was then ready to be specifically cloned.<br>
First-round PCR was performed with the adaptor-specific<br>
oligonucleotide, 5'-CCATCCTAATACGACTCACTATAGGGC- 3' (AP1) (SEQ.ID<br>
NO: 20) as sense primer and a Gene Specific Primer (GSP) from the unique<br>
region described in Example 16 (HLMPU). The second round of PCR was<br>
performed using a nested primers GSP1-HLMPU (antisense/reverse primer)<br>
(SEQ. ID NO: 23) and GSP2-HLMPUF (SEQ. ID NO: 24) (see Example 16;<br>
sense/forward primer). PCR was performed using a commercial kit (Advantage<br>
cDNA PCR core kit; CloneTech Laboratories Inc., Palo Alto, CA) that utilizes<br>
an antibody-mediated, but otherwise standard, hot-start protocol. PCR<br>
conditions for MG63 cDNA included an initial hot-start denaturation (94°C, 60<br>
sec) followed by: 94°C, 30 sec; 60°C, 30 sec; 68°C, 4 min; 30 cycles. The first-<br>
round PCR product was approximately 750 base-pairs in length whereas the<br>
nested PCR product was approximately 230 base-pairs. The first-round PCR<br>
product was cloned into linearized pCR 2.1 vector (3.9 Kb). The inserts were<br>
sequenced in both directions using M13 Forward and Reverse primers (SEQ.<br>
ID NO: 11; SEQ. ID NO: 12)<br>
FXAMPLE 22: Determination of Full-length Human LMP-1 cDNA with 5 Prime<br>
UTR<br>
Overlapping MG63 human osteosarcoma cell cDNA 5'-UTR sequence<br>
(SEQ ID NO: 21), MG63 717 base-pair sequence (Example 17; SEQ ID NO: 8)<br>
and human heart cDNA clone 7 sequence (Example 18) were aligned to derive<br>
a novel human cDNA sequence of 1704 base-pairs (SEQ.ID NO: 22). The<br>
alignment was accomplished with NALIGN, (both PCGENE and Omiga 1.0;<br>
Intelligenetics). Over-lapping sequences constituted nearly the entire 717<br>
base-pair region (Example 17) with 100% homology. Joining of the aligned<br>
sequences was accomplished with SEQIN.<br>
EXAMPLE 23: Construction of LIM Protein Expression Vector<br>
The construction of pHIS-5ATG LMP-1s expression vector was carried<br>
out with the sequences described in Examples 17 and 18. The 717 base-pair<br>
clone (Example 17; SEQ ID NO: 7) was digested with Clal and EcoRV. A<br>
small fragment (~250 base-pairs) was gel purified. Clone 7 (Example 18; SEQ<br>
ID NO: 8) was digested with Clal and Xbal and a 1400 base-pair fragment was<br>
gel purified. The isolated 250 base-pair and 1400 base-pair restriction<br>
fragments were ligated to form a fragment of-1650 base-pairs.<br>
Due to the single nucleotide substitution in Clone 7 (relative to the 717<br>
base-pair PCR sequence and the original rat sequence) a stop codon at<br>
translated base-pair 672 resulted. Because of this stop codon, a truncated<br>
(short) protein was encoded, hence the name LMP-1s. This was the construct<br>
used in the expression vector (SEQ ID NO: 32). The full length cDNA<br>
sequence with 5' UTR (SEQ ID NO: 33) was created by alignment of SEQ ID<br>
NO: 32 with the 5' RACE sequence (SEQ ID NO: 21). The amino acid<br>
sequence of LMP-1s (SEQ ID NO: 34) was then deduced as a 223 amino acid<br>
protein and confirmed by Western blot (as in Example 15) to run at the<br>
predicted molecular weight of ~ 23.7 kD.<br>
The pHis-ATG vector (InVitrogen, Carlsbad, CA) was digested with<br>
EcoRV and Xbal. The vector was recovered and the1650 base-pair restriction<br>
fragment was then ligated into the linearized pHis-ATG. The ligated product<br>
was cloned and amplified. The pHis-ATG-LMP-1s Expression vector, also<br>
designated pHIS-A with insert HLMP-1s, was purified by standard methods.<br>
EXAMPLE 24: Induction of Bone Nodule Formation and Mineralization In vitro<br>
with LMP Expression Vector<br>
Rat Calvarial cells were isolated and grown in secondary culture<br>
according to Example 1. Cultures were either unstimulated or stimulated with<br>
glucocorticoid (GC) as described in Example 1. A modification of the Superfect<br>
Reagent (Qiagen, Valencia, CA) transfection protocol was used to transfect 3<br>
g/well of each vector into secondary rat calvarial osteoblast cultures according<br>
to Example 25.<br>
Mineralized nodules were visualized by Von Kossa staining, as described in<br>
Example 3. Human LMP-1s gene product overexpression alone induced<br>
bone nodule formation (~203 nodules/well) in vitro. Levels of nodules were<br>
approximately 50% of those induced by the GC positive control (~412<br>
nodules/well). Other positive controls included the pHisA-LMP-Rat expression<br>
vector (~152 nodules/well) and the pCMV2/LMP-Rat-Fwd Expression vector<br>
(-206 nodules/well), whereas the negative controls included the pCMV2/LMP-<br>
Rat-Rev. Expression vector (-2 nodules/well) and<br>
untreated (NT) plates (~4 nodules/well). These data demonstrate that the<br>
human cDNA was at least as osteoinductive as the rat cDNA. The effect was<br>
less than that observed with GC stimulation, most likely due to suboptimal<br>
doses of Expression vector.<br>
EXAMPLE 25: LMP-lnduced Cell Differentiation In Vitro and In Vivo<br>
The rat LMP cDNA in clone 10-4 (see Example 12) was excised from<br>
the vector by double-digesting the clone with Notl and Apal overnight at 37°C.<br>
Vector pCMV2 MCS (InVitrogen, Carlsbad, CA) was digested with the same<br>
restriction enzymes. Both the linear cDNA fragment from clone 10-4 and<br>
pCMV2 were gel purified, extracted and ligated with T4 ligase. The ligated<br>
DNA was gel purified, extracted and used to transform E. coli JM109 cells for<br>
amplification. Positive agar colonies were picked, digested with Notl and Apal<br>
and the restriction digests were examined by gel electrophoresis. Stock<br>
cultures were prepared of positive clones.<br>
A reverse vector was prepared in analogous fashion except that the<br>
restriction enzymes used were Xbal and Hindlll. Because these restriction<br>
enzymes were used, the LMP cDNA fragment from clone 10-4 was inserted<br>
into pRc/CMV2 in the reverse (that is, non-translatable) orientation. The<br>
recombinant vector produced is designated pCMV2/RLMP.<br>
An appropriate volume of pCMV10-4 (60 nM final concentration is<br>
optimal [3 g]; for this experiment a range of 0-600 nM/well [0-30 g/well] final<br>
concentration is preferred) was resuspended in Minimal Eagle Media (MEM) to<br>
450 I final volume and vortexed for 10 seconds. Superfect was added (7.5 I/ml<br>
final solution), the solution was vortexed for 10 seconds and then incubated at<br>
room termperature for 10 minutes. Following this incubation, MEM<br>
supplemented with 10% FBS (1 ml/well; 6 ml/plate) was added and mixed by<br>
pipetting.<br>
The resulting solution was then promptly pipetted (1 ml/well) onto<br>
washed ROB cultures. The cultures were incubated for 2 hours at 37°C in a<br>
humidified atmosphere containing 5% CO2. Afterward, the cells were gently<br>
washed once with sterile PBS and the appropriate normal incubation medium<br>
was added.<br>
Results demonstrated significant bone nodule formation in all rat cell<br>
cultures which were induced with pCMV10-4. For example, pCMV10-4<br>
transfected cells produced 429 nodules/well. Positive control cultures, which<br>
were exposed to Trm, produced 460 nodules/well. In contrast, negative<br>
controls, which received no treatment, produced 1 nodule/well. Similarly, when<br>
cultures were transfected with pCMV10-4 (reverse), no nodules were<br>
observed.<br>
For demonstrating de novo bone formation in vivo, marrow was<br>
aspirated from the hindlimbs of 4-5 week old normal rats (mu/+; heterozygous<br>
for recessive athymic condition). The aspirated marrow cells were washed in<br>
alpha MEM, centrifuged, and RBCs were lysed by resuspending the pellet in<br>
0.83% NH4CI in 10 mM Tris (pH 7.4), The remaining marrow cells were<br>
washed 3x with MEM and transfected for 2 hours with 9 g of pCMV-LMP-1s<br>
(forward or reverse orientation) per 3 x 106 cells. The transfected cells were<br>
then washed 2X with MEM and resuspended at a concentration of 3 x 107<br>
cells/ml.<br>
The cell suspension (100 I) was applied via sterile pipette to a sterile 2 x<br>
5 mm type I bovine collagen disc (Sulzer Orthopaedics, Wheat Ridge, CO).<br>
The discs were surgically implanted subcutaneously on the skull, chest,<br>
abdomen or dorsal spine of 4-5 week old athymic rats (rnu/rnu). The animals<br>
were scarified at 3-4 weeks, at which time the discs or surgical areas were<br>
excised and fixed in 70% ethanol. The fixed specimens were analyzed by<br>
radiography and undecalcified histologic examination was performed on 5 m<br>
thick sections stained with Goldner Trichrome. Experiments were also<br>
performed using devitalized (guanidine extracted) demineralized bone matrix<br>
(Osteotech, Shrewsbury, NJ) in place of collagen discs.<br>
Radiography revealed a high level of mineralized bone formation that<br>
conformed to the form of the original collagen disc containing LMP-1s<br>
transfected marrow cells. No mineralized bone formation was observed in the<br>
negative control (cells transfected with a reverse-oriented version of the LMP-<br>
1s cDNA that did not code for a translated protein), and absorption of the<br>
carrier appeared to be well underway.<br>
Histology revealed new bone trabeculae lined with osteroblasts in the<br>
LMP-1s transfected implants. No bone was seen along with partial resorption<br>
of the carrier in the negative controls.<br>
Radiography of a further experiment in which 18 sets (9 negative control<br>
pCMV-LMP-REV &amp; 9 experimental pCMV-LMP-1s) of implants were added to<br>
sites alternating between lumbar and thoracic spine in athymic rats<br>
demonstrated 0/9 negative control implants exhibiting bone formation (spine<br>
fusion) between vertebrae. All nine of the pCMV-LMP-1s treated implants<br>
exhibited solid bone fusions between vertebrae.<br>
EXAMPLE 26: The Synthesis of pHIS-5' ATG LMP-1s Expression Vector from<br>
the sequences Demonstrated in Examples 2 and 3,<br>
The 717 base-pair clone (Example 17) was digested with Clal and<br>
EcoRV (New England Biologicals, city, MA). A small fragment (~250 base-<br>
pairs) was gel purified. Clone No. 7 (Example 18) was digested with Clal and<br>
Xbal. A 1400 base-pair fragment was gel purified from that digest. The<br>
isolated 250 base-pair and 1400 base-pair cDNA fragments were ligated by<br>
standard methods to form a fragment of ~1650 bp. The pHis-A vector<br>
(InVitrogen) was digested with EcoRV and Xbal. The linearized vector was<br>
recovered and ligated to the chimeric 1650 base-pair cDNA fragment. The<br>
ligated product was cloned and amplified by standard methods, and the pHis-<br>
A-5' ATG LMP-1s expression vector, also denominated as the vector pHis-A<br>
with insert HLMP-1s, was deposited at the ATCC as previously described.<br>
EXAMPLE 27: The Induction of Bone Nodule Formation and Mineralization In<br>
Vitro With pHis-5' ATG LMP-1s Expression Vector<br>
Rat calvarial cells were isolated and grown in secondary culture<br>
according to Example 1. Cultures were either unstimulated or stimulated with<br>
glucocorticoid (GC) according to Example 1. The cultures were transfected<br>
with 3 g of recombinant<br>
pHis-A vector DNA/well as described in Example 25. Mineralized nodules<br>
were visualized by Von Kossa staining according to Example 3.<br>
Human LMP-1s gene product overexpression alone (i.e., without GC<br>
stimulation) induced significant bone nodule formation (~203 nodules/well) in<br>
vitro. This is approximately 50% of the amount of nodules produced by cells<br>
exposed to the GC positive control (~412 nodules/well). Similar results were<br>
obtained with cultures transfected with pHisA-LMP-Rat Expression vector<br>
(~152 nodules/well) and pCMV2/LMP-Rat-Fwd (~206 nodules/well). In<br>
contrast, the negative control pCMV2/LMP-Rat-Rev yielded (~2 nodules/well),<br>
while approximately 4 nodules/well were seen in the untreated plates. These<br>
data demonstrate that the human LMP-1 cDNA was at least as osteoinductive<br>
as the rat LMP-1 cDNA in this model system. The effect in this experiment was<br>
less than that observed with GC stimulation; but in some the effect was<br>
comparable.<br>
EXAMPLE 28: LMP Induces Secretion of a Soluble Osteoinductive Factor<br>
Overexpression of RLMP-1 or HLMP-1s in rat calvarial osteoblast<br>
cultures as described in Example 24 resulted in significantly greater nodule<br>
formation than was observed in the negative control. To study the mechanism<br>
of action of LIM mineralization protein conditioned medium was harvested at<br>
different time points, concentrated to 10 X, sterile filtered, diluted to its original<br>
concentration in medium containing fresh serum, and applied for four days to<br>
untransfected cells.<br>
Conditioned media harvested from cells transfected with RLMP-1 or<br>
HLMP-1s at day 4 was approximately as effective in inducing nodule formation<br>
as direct overexpression of RLMP-1 in transfected cells. Conditioned media<br>
from cells transfected with RLMP-1 or HLMP-1 in the reverse orientation had<br>
no apparent effect on nodule formation. Nor did conditioned media harvested<br>
from LMP-1 transfected cultures before day 4 induce nodule formation. These<br>
data suggest that expression of LMP-1 caused the synthesis and/or secretion<br>
of a soluble factor, which did not appear in culture medium in effectie amounts<br>
until 4 days post transfection.<br>
Since overexpression of rLMP-1 resulted in the secretion of an<br>
osteoinductive factor into the medium, Western blot analysis was used to<br>
determine if LMP-1 protein was present in the medium. The presence of rLMP-<br>
1 protein was assessed using antibody specific for LMP-1 (QDPDEE) and<br>
detected by conventional means. LMP-1 protein was found only in the cell<br>
layer of the culture and not detected in the medium.<br>
Partial purification of the osteoinductive soluble factor was accomplished<br>
by standard 25% and 100% ammonium sulfate cuts followed by DE-52 anion<br>
exchange batch chromatography (100 mM or 500 mM NaCI). All activity was<br>
observed in the high ammonium sulfate, high NaCI fractions. Such localization<br>
is consistent with the possibility of a single factor being responsible for<br>
conditioning the medium.<br>
EXAMPLE 29: Gene Therapy In Lumbar Spine Fusion Mediated by Low Dose<br>
Adenovirus<br>
This study determined the optimal dose of adenoviral delivery of the<br>
LMP-1 cDNA (SEQ ID NO: 2) to promote spine fusion in normal, that is,<br>
immune competent, rabbits.<br>
A replication-deficient human recombinant adenovirus was constructed<br>
with the LMP-1 cDNA (SEQ ID NO: 2) driven by a CMV promoter using the<br>
Adeno-Quest™ Kit (Quantum Biotechnologies, Inc., Montreal). A commercially<br>
available (Quantum Biotechnologies, Inc., Montreal) recombinant adenovirus<br>
containing the beta-galactosidase gene was used as a control.<br>
Initially, an in vitro dose response experiment was performed to<br>
determine the optimal concentration of adenovirus-delivered LMP-1 ("AdV-<br>
LMP-1") to induce bone differentiation in rat calvarial osteoblast cultures using<br>
a 60-minute transduction with a multiplicity of infection ("MOI") of 0.025, 0.25,<br>
2.5, or 25 plaque-forming units (pfu) of virus per cell. Positive control cultures<br>
were differentiated by a 7-day exposure to 109 M glucocorticoid ("GC").<br>
Negative control cultures were left untreated. On day 14, the number of<br>
mineralized bone nodules was counted after von Kossa staining of the<br>
cultures, and the level of osteocalcin secreted into the medium (pmol/mL) was<br>
measured by radioimmunoassay (mean ± SEM).<br>
The results of this experiment are shown in Table I. Essentially no<br>
spontaneous nodules formed in the untreated negative control cultures. The<br>
data show that a MOI equal to 0.25 pfu/cell is most effective for osteoinducing<br>
bone nodules, achieving a level comparable to the positive control (GC).<br>
Lower and higher doses of adenovirus were less effective.<br>
In vivo experiments were then performed to determine If the optimal in<br>
vitro dose was capable of promoting intertransverse process spine fusions in<br>
skeletally mature New Zealand white rabbits. Nine rabbits were anesthetized<br>
and 3 cc of bone marrow was aspirated from the distal femur through the<br>
intercondylar notch using an 18 gauge needle. The buffy coat was then<br>
isolated, a 10-minute transduction with AdV-LMP-1 was performed, and the<br>
cells were returned to the operating room for implantation. Single level<br>
posterolateral lumbar spine arthrodesis was performed with decortication of<br>
transverse processes and insertion of carrier (either rabbit devitalized bone<br>
matrix or a collagen sponge) containing 8-15 million autologous nucleated<br>
buffy coat cells transduced with either AdV-LMP-1 (MOI = 0.4) or AdV-BGal<br>
(MOI = 0.4). Rabbits were euthanized after 5 weeks and spine fusions were<br>
assessed by manual palpation, plain x-rays, CT scans, and undecalcified<br>
histology.<br>
The spine fusion sites that received AdV-LMP-1 induced solid,<br>
continuous spine fusion masses in all nine rabbits. In contrast, the sites<br>
receiving AdV-BGal, or a lower dose of AdV-LMP-1 (MOI = 0.04) made little or<br>
no bone and resulted in spine fusion at a rate comparable to the carrier alone<br>
(
scan, and histology. Plain radiographs, however, sometimes overestimated<br>
the amount of bone that was present, especially in the control sites. LMP-1<br>
cDNA delivery and bone induction was successful with both of the carrier<br>
materials tested. There was no evidence of systemic or local immune<br>
response to the adenovirus vector.<br>
These data demonstrate consistent bone induction in a previously<br>
validated rabbit spine fusion model which is quite challenging. Furthermore,<br>
the protocol of using autogenous bone marrow cells with intraoperative ex vivo<br>
gene transduction (10 minutes) is a more clinically feasible procedure than<br>
other methods that call for overnight transduction or cell expansion for weeks in<br>
culture. In addition, the most effective dose of recombinant adenovirus<br>
(MOI=0.25) was substantially lower than doses reported in other gene therapy<br>
applications (MOI-40-500). We believe this is due to the fact that LMP-1 is an<br>
intracellular signaling molecule and may have powerful signal amplification<br>
cascades. Moreover, the observation that the same concentration of AdV-<br>
LMP-1 that induced bone in cell culture was effective in vivo was also<br>
surprising given the usual required increase in dose of other growth factors<br>
when translating from cell culture to animal experiments. Taken together,<br>
these observations indicate that local gene therapy using adenovirus to deliver<br>
the LMP-1 cDNA is possible and the low dose required will likely minimize the<br>
negative effects of immune response to the adenovirus vector.<br>
EXAMPLE 30: Use of Peripheral Venous Blood Nucleated Cells (Buffy Coat)<br>
for Gene Therapy With LMP-1 cDNA To Make Bone<br>
In four rabbits we performed spine fusion surgery as above (Example<br>
29) except the transduced cells were the buffy coat from venous blood rather<br>
than bone marrow. These cells were transfected with Adeno-LMP or pHIS-<br>
LMP plasmid and had equivalent successful results as when bone marrow cells<br>
were used. This discovery of using ordinary venous blood cells for gene<br>
delivery makes gene therapy more feasible clinically since it avoids painful<br>
marrow harvest under general anesthesia and yields two times more cells per<br>
mL of starting material.<br>
EXAMPLE 31: Isolation of Human LMP-1 Splice Variants<br>
Intron/Exon mRNA transcript splice variants are a relatively common<br>
regulatory mechanism in signal transduction and cellular/tissue development.<br>
Splice variants of various genes have been shown to alter protein-protein,<br>
protein-DNA, protein-RNA, and protein-substrate interactions. Splice variants<br>
may also control tissue specificity for gene expression allowing different forms<br>
(and therefore functions) to be expressed in various tissues. Splice variants<br>
are a common regulatory phenomenon in cells. It is possible that the LMP<br>
splice variants may result in effects in other tissues such as nerve<br>
regeneration, muscle regeneration, or development of other tissues.<br>
To screen a human heart cDNA library for splice variants of the HLMP-1<br>
sequence, a pair of PCR primer corresponding to sections of SEQ ID NO: 22<br>
was prepared. The forward PCR primer, which was synthesized using<br>
standard techniques, corresponds to nucleotides 35-54 of SEQ ID NO: 22. It<br>
has the following sequence:<br>
5' GAGCCGGCATCATGGATTCC 3' (SEQ ID NO: 35)<br>
The reverse PCR primer, which is the reverse complement of<br>
nucleotides 820-839 in SEQ ID NO: 22, has the following sequence:<br>
5' GCTGCCTGCACAATGGAGGT 3' (SEQ ID NO: 36)<br>
The forward and reverse PCR primers were used to screen human heart<br>
cDNA (ClonTech, Cat No. 7404-1) for sequences similar to HLMP-1 by<br>
standard techniques, using a cycling protocol of 94°C for 30 seconds, 64°C for<br>
30 seconds, and 72°C for 1 minute, repeated 30 times and followed by a 10<br>
minute incubation at 72°C. The amplification cDNA sequences were gel-<br>
purified and submitted to the Emory DNA Sequence Core Facility for<br>
sequencing. The clones were sequenced using standard techniques and the<br>
sequences were examined with PCGENE (Intelligenetics; Programs SEQUIN<br>
and NALIGN) to determine homology to SEQ ID NO: 22. Two homologous<br>
nucleotide sequences with putative alternative splice sites compared to SEQ ID<br>
NO: 22 were then translated to their respective protein products with<br>
Intelligenetic's program TRANSL.<br>
One of these two novel human cDNA sequences (SEQ ID NO: 37)<br>
comprises 1456 bp:<br>
Reading frame shifts caused by the deletion of a 119 bp fragment<br>
(between *) and the addition of a 17 bp fragment (underlined) results in a<br>
truncated gene product having the following derived amino acid sequence<br>
(SEQ ID NO: 38):<br>
This 423 amino acid protein demonstrates 100% homology to the<br>
protein shown in Sequence ID No. 10, except for the sequence in the<br>
highlighted area (amino acids 94-99), which are due to the nucleotide changes<br>
depicted above.<br>
The second novel human heart cDNA sequence (SEQ ID NO: 39)<br>
comprises 1575 bp:<br>
Reading frame shifts caused by the addition of a 17 bp fragment<br>
(bolded, italicized and underlined) results in an early translation stop codon at<br>
position 565-567 (underlined).<br>
The derived amino acid sequence (SEQ ID NO: 40) consists of 153<br>
amino acids:<br>
This protein demonstrates 100% homology to SEQ ID NO: 10 until<br>
amino acid 94, where the addition of the 17 bp fragment depicted in the<br>
nucleotide sequence results in a frame shift. Over amino acids 94-153, the<br>
protein is not homologous to SEQ ID NO: 10. Amino acids 154-457 in SEQ ID<br>
NO: 10 are not present due to the early stop codon depicted in nucleotide<br>
sequence.<br>
EXAMPLE 32; Genomic HLMP-1 Nucleotide Sequence<br>
Applicants have identified the genomic DNA sequence encoding HLMP-<br>
1, including putative regulatory elements associated with HLMP-1 expression.<br>
The entire genomic sequence is shown in SEQ ID. NO: 41. This sequence<br>
was derived from AC023788 (clone RP11-564G9). Genome Sequencing<br>
Center, Washington University School of Medicine, St. Louis, MO.<br>
The putative promoter region for HLMP-1 spans nucleotides 2,660-<br>
8,733 in SEQ ID NO: 41. This region comprises, among other things, at least<br>
ten potential glucocorticoid response elements ("GREs") (nucleotides 6148-<br>
6153, 6226-6231, 6247-6252, 6336-6341, 6510-6515, 6552-6557, 6727-6732,<br>
6752-6757, 7738-7743, and 8255-8260), twelve potential Sma-2 homologues<br>
to Mothers against Drosophilla decapentaplegic ("SMAD") binding element<br>
sites (nucleotides 3569-3575,4552-4558,4582-4588, 5226-5232,6228-6234,<br>
6649-6655, 6725-6731, 6930-6936, 7379-7384, 7738-7742, 8073-8079, and<br>
8378-8384), and three TATA boxes (nucleotides 5910-5913, 6932-6935, and<br>
7380-7383). The three TATA boxes, all of the GREs, and eight of the SMAD<br>
binding elements ("SBEs") are grouped in the region spanning nucleotides<br>
5,841-8,733 in SEQ ID NO: 41. These regulatory elements can be used, for<br>
example, to regulate expression of exogenous nucleotide sequences encoding<br>
proteins involved in the process of bone formation. This would permit systemic<br>
administration of therapeutic factors or genes relating to bone formation and<br>
repair, as well as factors or genes associated with tissue differentiation and<br>
development.<br>
In addition to the putative regulatory elements, 13 exons corresponding<br>
to the nucleotide sequence encoding HLMP-1 have been identified. These<br>
exons span the following nucleotides in SEQ ID NO: 4V.<br>
Exon 1 8733-8767<br>
Exon 2 9790-9895<br>
Exon 3 13635-13787<br>
Exon4 13877-13907<br>
Exon 5 14387-14502<br>
Exon 6 15161-15297<br>
Exon 7 15401-15437<br>
Exon 8 16483-16545<br>
Exon 9 16689-16923<br>
Exon 10 18068-18248<br>
Exon 11 22117-22240<br>
Exon 12 22323-22440<br>
Exon 13 22575-22911<br>
in HLMP-2 there is another exon (Exon 5A), which spans nucleotides<br>
14887-14904.<br>
All cited publications and patents are hereby incorporated by reference<br>
in their entirety.<br>
While the foregoing specification teaches the principles of the present<br>
invention, with examples provided for the purpose of illustration, it will be<br>
appreciated by one skilled in the art from reading this disclosure that various<br>
changes in form and detail can be made without departing from the true scope<br>
of the invention.<br>
WE CLAIM:<br>
1. An isolated nucleic acid molecule comprising a SEQ ID NO: 37 or SEQ ID<br>
NO:39.<br>
2. An isolated human LMP protein encoded by SEQ ID NO: 37 or SEQ ID NO: 39.<br>
3. A vector comprising the isolated nucleic acid molecule as claimed in claim 1.<br>
4. A recombinant cell comprising the vector of claim 3.<br>
5. The recombinant cell as claimed in claim 4, wherein the said cell is selected from<br>
the group consisting of prokaryotic cells, yeast cells and mammalian cells.<br>
6. The isolated nucleic acid molecule as claimed in claim 1, further comprising a<br>
label such as described herein.<br>
7. A human LIM mineralization protein comprising an amino acid sequence selected<br>
from the group consisting of SEQ ID NO:38 and SEQ ID NO:40.<br>
8. A monoclonal antibody specific for a HLMP-2 (SEQ ID NO:38) or HLMP-3<br>
(SEQ ID NO:40).<br>
9. A pharmaceutical composition for delivery into the site for inducing bone<br>
formation and fusing spine in mammal, wherein the composition comprises a genetically<br>
modified osteogenic precursor cell or peripheral blood leukocytes transfected ex-vivo<br>
with an isolated nucleic acid molecule with SEQ ID No:39 or SEQ ID No :37 optionally<br>
in association with acceptable vector.<br>
10. A composition as claimed in claim 9, wherein the vector is an expression vector.<br>
11. A composition as claimed in any one of claims 9 and 10, wherein the vector is<br>
plasmid.<br>
12. A composition^ claimed in any one of claims 9 and 10, wherein the vector is<br>
virus.<br>
13. A composition as claimed in any one of claims 9, 10 and 12, wherein virus is<br>
adenovirus.<br>
14. A composition as claimed in any one of claims 9, 10 and 12, wherein the virus is<br>
retrovirus.<br>
15. A composition as claimed in any one of claims 9 to 14, wherein the composition<br>
comprises of genetically modified osteogenic precursor cells or peripheral blood<br>
leukocytes transfected with vector comprising SEQ ID No. 37 or SEQ. ID No. 39<br>
operably linked to a regulatable promoter, the vector is stably maintained in genetically<br>
modified osteogenic precursor cells or peripheral blood leukocytes and regulatable<br>
promoter responding to an exogenous control compound.<br>
16. A pharmaceutical composition in the form of an injectable comprising an isolated<br>
nucleic acid molecule with SEQ. ID No. 39 or SEQ. ID No. 37 in association with an<br>
acceptable vector.<br>
17. A composition as claimed in claim 16, wherein said vector is selected from the<br>
group consisting of a plasmid or virus.<br>
18. A genetically modified osteogenic precursor cells or peripheral blood leukocytes<br>
transfected ex-vivo with an isolated nucleic acid molecule comprising SEQ ID No. 37 or<br>
SEQ ID No. 39 capable of stimulating the production of an osteogenic soluble factor by<br>
osteogenic cell.<br>
19. The osteogenic soluble factor as claimed in claim 18, wherein the osteogenic factor is a<br>
protein.<br>
20. A composition for treating long bone fractures, bone defects, osteoporosis of hip,<br>
vertebrae, or wrist containing (a) nucleic acid molecule comprising SEQ ID No: 39,<br>
SEQ ID No: 37 or (b) a genetically modified osteogenic precursor cell or peripheral blood<br>
leukocytes transfected ex-vivo with an isolated nucleic acid molecule comprising SEQ ID<br>
No: 39 or SEQ ID No: 37 in association with an acceptable vector.<br>
21. A composition containing a nucleic acid molecule comprising SEQ ID No: 39,<br>
SEQ ID No: 37, or an osteogenic precursor cell or a peripheral blood leukocyte<br>
containing said nucleic acid molecule for facilitating tumor reconstruction or spinal<br>
fusion.<br>
22. A composition for inhibiting the expression of HLMP-2 or HLMP-3 comprising<br>
an antisense oligonucleotide directed against HLMP-2, or HLMP-3.<br>
23. A composition for treating long bone fractures, bone defects, osteoporosis of hip,<br>
vertebrae, or wrist comprising an effective amounts of (a) an isolated nucleic acid<br>
comprising SEQ ID No: 37 or (b) an osteogenic precursor cell or peripheral blood<br>
lymphocyte transfected with SEQ ID No: 37.<br>
The present invention is directed to isolated nucleic acid molecules that encode LIM mineralization protein, or LMP.<br>
The invention further provides vectors comprising splice variants of nucleotide sequences that encode LMP, as well as host cells<br>
comprising those vectors. Moreover, the present invention relates to methods of inducing bone formation by transfecting osteogenic<br>
precursor cells with an isolated nucleic acid molecule comprising a nucleotide sequence encoding splice variants of LIM mineralization<br>
protein. The transfection may occur ex vivo or in vivo by direct injection of virus or naked plasmid DNA. In a particular<br>
embodiment, the invention provides a method of fusing a spine by transfecting osteogenic precursor cells with an isolated nucleic<br>
acid molecule having a nucleotide sequence encoding LIM mineralization protein, admixing the transfected osteogenic precursor<br>
cells with a matrix and contacting the matrix with the spine. Finally, the invention relates to methods for inducing systemic bone<br>
formation by stable transfection of host cells with the vectors of the invention.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
		<br>
		<div class="pull-left">
			<a href="225007-a-composition-for-a-human-or-animal-diet.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="225009-caspase-inhibitors.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>225008</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/01208/KOL</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>44/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>31-Oct-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>29-Oct-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>19-Nov-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>EMORY UNIVERSITY</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1380 SOUTH OXFORD ROAD, ATLANTA, GA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>BODEN SCOTT D</td>
											<td>2842 CRAVEY DRIVE N.E. ATLANTA, GA 30345</td>
										</tr>
										<tr>
											<td>2</td>
											<td>HAIR GREGORY A</td>
											<td>1887 CHRYSLER DRIVE N.E. ATLANTA, GA 30345</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A61K 48/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US00/11664</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-04-28</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/132,021</td>
									<td>1999-04-30</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/225008-lim-mineralization-protein-splice-variants-to-induce-bone-formation by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:40:20 GMT -->
</html>
