<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/225218-glf-1-fc-fusion-proteins by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 22:34:52 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 225218:GLF-1/FC FUSION PROTEINS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">GLF-1/FC FUSION PROTEINS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to glucagon-like-1 compounds fused to proteins that have the effect of extending the in vivo half-life of the peptides. These fusion proteins can be used to treat non-insulin dependent diabetes mellitus as well as a variety of other conditions.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present invention relates to glucagon-like peptides<br>
including analogs and derivatives thereof fused to proteins<br>
that have the effect of extending the in vivo half-life of<br>
the peptides. These fusion proteins can be used to treat<br>
non-insulin dependent diabetes mellitus as well as a variety<br>
of other conditions.<br>
Glucagon-Like Peptide 1 (GLP-1) is a 37 amino acid<br>
peptide that is secreted by the L-cells of the intestine in<br>
response to food ingestion. It has been found to stimulate<br>
insulin secretion (insulinotropic action), thereby causing<br>
glucose uptake by cells and decreased serum glucose levels<br>
[see, e.g., Mojsov, S., (1992) Int. J. Peptide Protein<br>
Research, 40:333-343]. However, GLP-1 is poorly active. A<br>
subsequent endogenous cleavage between the 6th and 7th<br>
position produces a more potent biologically active GLP-1(7-<br>
37)OH peptide. Numerous GLP-1 analogs and derivatives are<br>
known and are referred to herein as "GLP-1 compounds."<br>
These GLP-1 analogs include the Exendins which are peptides<br>
found in the venom of the GILA-monster. The Exendins have<br>
sequence homology to native GLP-1 and can bind the GLP-1<br>
receptor and initiate the signal transduction cascade<br>
responsible for the numerous activities that have been<br>
attributed to GLP-1(7-37)OH.<br>
GLP-1 compounds have a variety of physiologically<br>
significant activities. For example, GLP-1 has been shown<br>
to stimulate insulin release, lower glucagon secretion,<br>
inhibit gastric emptying, and enhance glucose utilization.<br>
[Nauck, M.A.. et al. (1993) Diahetologia 36:741-744;<br>
Gutniak, M. , et al. (1992) New England J. of Med. 326:1316-<br>
1322; Nauck, M.A., et al., (1993) J. Clin. Invest. 91:301-<br>
307] .<br>
GLP-1 shows the greatest promise as a treatment for<br>
non-insulin dependent diabetes mellitus (NIDDM). There are<br>
numerous oral drugs on the market to treat the insulin<br>
resistance associated with NIDDM. As the disease<br>
progresses, however, patients must move to treatments that<br>
stimulate the release of insulin and eventually to<br>
treatments that involve injections of insulin. Current<br>
drugs which stimulate the release of insulin, however, can<br>
also cause hypoglycemia as can the actual administration of<br>
insulin. GLP-1 activity, however, is controlled by blood<br>
glucose levels. When levels drop to a certain threshold<br>
level, GLP-1 is not active. Thus, there is no risk of<br>
hypoglycemia associated with treatment involving GLP-1.<br>
However, the usefulness of therapy involving GLP-1<br>
peptides has been limited by their fast clearance and short<br>
half-lives. For example, GLP-1(7-37) has a serum half-life<br>
of only 3 to 5 minutes. GLP-1(7-36) amide has a time action<br>
of about 50 minutes when administered subcutaneously. Even<br>
analogs and derivatives that are resistant to endogenous<br>
protease cleavage, do not have half-lives long enough to<br>
avoid repeated administrations over a 24 hour period. Fast<br>
clearance of a therapeutic agent is inconvenient in cases<br>
where it is desired to maintain a high blood level of the<br>
agent over a prolonged period of time since repeated<br>
administrations will then be necessary. Furthermore, a<br>
long-acting compound is particularly important for diabetic<br>
patients whose past treatment regimen has involved taking<br>
only oral medication. These patients often have an<br>
extremely difficult time transitioning to a regimen that<br>
involves multiple injections of medication.<br>
The present invention overcomes the problems associated<br>
with delivering a compound that has a short plasma half-<br>
life. The compounds of the present invention encompass GLP-<br>
1 compounds fused to another protein with a long circulating<br>
half-life such as the Fc portion of an immunoglobulin or<br>
albumin.<br>
Generally, small therapeutic peptides are difficult to<br>
manipulate because even slight changes in their structure<br>
can affect stability and/or biological activity. This has<br>
been especially true for GLP-1 compounds currently in<br>
development. For example, GLP-1 (7-37)OH has a tendency to<br>
undergo a conformational change from a primarily alpha helix<br>
structure to a primarily beta sheet structure. This beta<br>
sheet form results in aggregated material that is thought to<br>
be inactive. It was, therefore, surprising that<br>
biologically active GLP-1 fusion proteins with increased<br>
half-lives could be developed. This was especially<br>
unexpected given the difficulty of working with GLP-1(7-<br>
37)OH alone and the large size of the fusion partner<br>
relative to the small GLP-1 peptide attached.<br>
Compounds of the present invention include heterologous<br>
fusion proteins comprising a first polypeptide with a N-<br>
terminus and a C-terminus fused to a second polypeptide with<br>
a N-terminus and a C-terminus wherein the first polypeptide<br>
is a GLP-1 compound and the second polypeptide is selected<br>
from the group consisting of<br>
a) human albumin;<br>
b) human albumin analogs; and<br>
c) fragments of human albumin,<br>
and wherein the C-terminus of the first polypeptide is fused<br>
to the N-terminus of the second polypeptide.<br>
Compounds of the present invention also include a<br>
heterologous fusion protein comprising a first<br>
polypeptide with a N-terminus and a C-terminus fused to a<br>
second polypeptide with a N-terminus and a C-terminus<br>
wherein the first polypeptide is a GLP-1 compound and the<br>
second polypeptide is selected from the group consisting of<br>
a) human albumin;<br>
b) human albumin analogs; and<br>
c) fragments of human albumin,<br>
and wherein the C-terminus of the first polypeptide is fused<br>
to the N-terminus of the second polypeptide via a peptide<br>
linker. It is preferred that the peptide linker is selected<br>
from the group consisting of:<br>
a) a glycine rich peptide;<br>
b) a peptide having the sequence [Gly-Gly-Gly-Gly-Ser]n<br>
where n is 1, 2, 3, 4, 5 or 6; and<br>
c) a peptide having the sequence [Gly-Gly-Gly-Gly-<br>
Ser]3.<br>
Additional compounds of the present invention include a<br>
heterologous fusion protein comprising a first polypeptide<br>
with a N-terminus and a C-terminus fused to a second<br>
polypeptide with a N-terminus and a C-terminus wherein the<br>
first polypeptide is a GLP-1 compound and the second<br>
polypeptide is selected from the group consisting of<br>
a) the Fc portion of an immunoglobulin;<br>
b) an analog of the Fc portion of an immunoglobulin;<br>
and<br>
c) fragments of the Fc portion of an immunoglobulin,<br>
and wherein the C-terminus of the first polypeptide is fused<br>
to the N-terminus of the second polypeptide. The GLP-1<br>
compound may be fused to the second polypeptide via a<br>
peptide linker. It is preferable that the peptide linker is<br>
selected from the group consisting of:<br>
a) a glycine rich peptide;<br>
b) a peptide having the sequence [Gly-Gly-Gly-Gly-Ser]n<br>
where n is 1, 2, 3, 4, 5 or 6; and<br>
c) a peptide having the sequence [Gly-Gly-Gly-Gly-Ser]n-<br>
It is generally preferred that the GLP-1 compound that<br>
is part of the heterologous fusion protein have no more than<br>
6 amino acids that are different from the corresponding<br>
amino acid in GLP-1 (7-37)OH, GLP-1(7-36)OH, or Exendin-4.<br>
It is even more preferred that the GLP-1 compound have no<br>
more than 5 amino acids that differ from the corresponding<br>
amino acid in GLP-1(7-37 )OH, GLP-1(7-36)OH, or Exendin-4.<br>
It is most preferred that the GLP-1 compound have no more<br>
than 4, 3, or 2 amino acids that differ from the<br>
corresponding amino acid in GLP-1 (7-37)OH, GLP-1 (7-36)OH, or<br>
Exendin-4. Preferably, a GLP-1 compound that is part of the<br>
heterologous fusion protein has glycine or valine at<br>
position 8.<br>
The present invention also includes polynucleotides<br>
encoding the heterologous fusion protein described herein,<br>
vectors comprising these polynucleotides and host cells<br>
transfected or transformed with the vectors described<br>
herein. Also included is a process for producing a<br>
heterologous fusion protein comprising the steps of<br>
transcribing and translating a polynucleotide described<br>
herein under conditions wherein the heterolgous fusion<br>
protein is expressed in detectable amounts.<br>
The present invention also encompasses a method for<br>
normalizing blood glucose levels in a mammal in need thereof<br>
comprising the administration of a therapeutically effective<br>
amount of a heterologous fusion protein described herein.<br>
The invention is further illustrated with reference to<br>
the following drawings:<br>
Figure 1: IgGl Fc amino acid sequence encompassing the<br>
hinge region, CH2 and CH3 domains.<br>
Figure 2: Human serum albumin amino acid sequence.<br>
Figure 3: A.SDS-PAGE gel and immunoblot of same gel<br>
illustrating the molecular weight of IgGl-Fc and GLP-1-Fc<br>
fusion proteins (Lane 1, MW standards; Lane 2, Purified Fc;<br>
lane 3, Mock transfected media; Lane 4, Val8-GLP-1-Fc; Lane<br>
5, Exendin-4-Fc) B. SDS-PAGE gel and immunoblot of same gel<br>
illustrating the molecular weight of human HSA and GLP-1-HSA<br>
fusion proteins (Lane 1, MW standards; Lane 2, Purified HSA;<br>
lane 3, Mock transfected media; Lane 4, Val8-GLP-1-HSA; Lane<br>
5, Val8-GLP-1-[Gly-Gly-Gly-Gly-Ser]3-HSA; Lane 6, Exendin-4-<br>
HSA; Lane 7, Exendin-4- [Gly-Gly-Gly-Gly-Ser]3-HSA) .<br>
Figure 4: SDS-PAGE gel of purified Fc, albumin, and<br>
GLP-1 fusion proteins (Lane 1, MW standards; Lane 2,<br>
purified Fc; Lane 3, Val8-GLP-1-Fc; Lane 4, Exendin-4-Fc;<br>
Lane 5, MW standard; Lane 6, Val8-GLP-1-HSA; Lane 7,<br>
Exendin-4-HSA; Lane 8, Exendin-4-[Gly-Gly-Gly-Gly-Ser]3-<br>
HSA) .<br>
Figure 5: Expression cloning vector containing the Fc<br>
regions illustrated in figure 1.<br>
Figure 6: Expression cloning vector containing the<br>
albumin sequence illustrated in figure 2.<br>
Figure 7: Expression cloning vector containing DNA<br>
encoding a 15 amino acid linker fused in frame and 5' of the<br>
albumin sequence illustrated in figure 2.<br>
Figure 8: In vitro dose response activity of GLP-1<br>
fusion proteins.<br>
Figure 9: Pharmacokinetics of GLP-1 Fc and HSA fusion<br>
proteins.<br>
Figure 10: Glucodynamic response to Exendin-Fc in two<br>
normal fasted dogs.<br>
Figure 11: Insulinotropic response to Exendin-Fc in two<br>
normal fasted dogs.<br>
Figure 12: DNA sequence encoding a human IgG1 Fc<br>
region.<br>
Figure 13: DNA sequence encoding a human albumin<br>
protein.<br>
The heterologous fusion proteins of the present<br>
invention comprise a GLP-1 compound fused to human albumin,<br>
a human albumin analog, a human albumin fragment, the Fc<br>
portion of an immunoglobulin, an analog of the Fc portion of<br>
an immunoglobulin, or a fragment of the Fc portion of an<br>
immunoglobulin. The C-terminus of the GLP-1 compound may be<br>
fused directly, or fused via a peptide linker, to the N-<br>
terminus of an albumin or Fc protein. These heterologous<br>
fusion proteins are biologically active and have an<br>
increased half-life compared to native GLP-1.<br>
It is preferred that the GLP-1 compounds that make up<br>
part of the heterologous fusion protein encompass<br>
polypeptides having from about twenty-five to about thirty-<br>
nine naturally occurring or non-naturally occurring amino<br>
acids that have sufficient homology to native GLP-1 (7-37)OH<br>
such that they exhibit insulinotropic activity by binding to<br>
the GLP-1 receptor on ß-cells in the pancreas. A GLP-1<br>
compound typically comprises a polypeptide having the amino<br>
acid sequence of GLP-1(7-37)OH, an analog of GLP-1 (7-37)OH,<br>
a fragment of GLP-1(7-37)OH or a fragment of a GLP-1(7-37)OH<br>
analog. GLP-1 (7-37)OH has the amino acid sequence of SEQ ID<br>
NO: 1:<br><br>
By custom in the art, the amino terminus of GLP-1 (7-<br>
37) OH has been assigned number residue 7 and the carboxy-<br>
terminus, number 37. The other amino acids in the<br>
polypeptide are numbered consecutively, as shown in SEQ<br>
ID NO: 1. For example, position 12 is phenylalanine and<br>
position 22 is glycine.<br>
GLP-1 compounds also encompass "GLP-1 fragments." A<br>
GLP-1 fragment is a polypeptide obtained after truncation of<br>
one or more amino acids from the N-terminus and/or C-<br>
terminus of GLP-1(7-37)OH or an analog or derivative<br>
thereof. The nomenclature used to describe GLP-1(7-37)OH is<br>
also applicable to GLP-1 fragments. For example, GLP-1(9-<br>
36)OH denotes a GLP-1 fragment obtained by truncating two<br>
amino acids from the N-terminus and one amino acid from the<br>
C-terminus. The amino acids in the fragment are denoted by<br>
the same number as the corresponding amino acid in GLP-1(7-<br>
37)OH. For example, the N-terminal glutamic acid in GLP-<br>
1(9-36)OH is at position 9; position 12 is occupied by<br>
phenylalanine; and position 22 is occupied by glycine, as in<br>
GLP-1(7-37)OH. For GLP-1(7-36)OH, the glycine at position 37<br>
of GLP-1(7-37)OH is deleted.<br>
GLP-1 compounds also include polypeptides in which one<br>
or more amino acids have been added to the N-terminus and/or<br>
C-terminus of GLP-1(7-37)OH, or fragments or analogs<br>
thereof. It is preferred that GLP-1 compounds of this type<br>
have up to about thirty-nine amino acids. The amino acids<br>
in the "extended" GLP-1 compound are denoted by the same<br>
number as the corresponding amino acid in GLP-1(7-37)OH.<br>
For example, the N-terminus amino acid of a GLP-1 compound<br>
obtained by adding two amino acids to the N-terminal of GLP-<br>
1(7-37)OH is at position 5; and the C-terminus amino acid of<br>
a GLP-1 compound obtained by adding one amino acid to the C-<br>
terminus of GLP-1(7-37)OH is at position 38. Thus, position<br>
12 is occupied by phenylalanine and position 22 is occupied<br>
by glycine in both of these "extended" GLP-1 compounds, as<br>
in GLP-1(7-37)OH. Amino acids 1-6 of an extended GLP-1<br>
compound are preferably the same as or a conservative<br>
substitution of the amino acid at the corresponding position<br>
of GLP-1(1-37)OH. Amino acids 38-45 of an extended GLP-1<br>
compound are preferably the same as or a conservative<br>
substitution of the amino acid at the corresponding position<br>
of glucagon or Exendin-4.<br>
GLP-1 compounds of the present invention encompass<br>
"GLP-1 analogs." A GLP-1 analog has sufficient homology to<br>
GLP-1(7-37)OH or a fragment of GLP-1(7-37)OH such that the<br>
analog has insulinotropic activity. Preferably, a GLP-1<br>
analog has the amino acid sequence of GLP-1(7-37)OH or a<br>
fragment thereof, modified so that from one, two, three,<br>
four or five amino acids differ from the amino acid in the<br>
corresponding position of GLP-1(7-37)OH or a fragment of<br>
GLP-1(7-37)OH. In the nonmenclature used herein to<br>
designate GLP-1 compounds, the substituting amino acid and<br>
its position is indicated prior to the parent structure.<br>
For example, Glu22-GLP-1(7-37)OH designates a GLP-1 compound<br>
in which the glycine normally found at position 22 of GLP-<br>
1(7-37)OH has been replaced with glutamic acid; Val8-Glu22-<br>
GLP-1(7-37)OH designates a GLP-1 compound in which alanine<br>
normally found at position 8 and glycine normally found at<br>
position 22 of GLP-1 (7-37) OH have been replaced with valine<br>
and glutamic acid, respectively.<br>
GLP-1 compounds of the present invention also include<br>
"GLP-1 derivatives." A GLP-1 derivative is defined as a<br>
molecule having the amino acid sequence of GLP-1 or of a<br>
GLP-1 analog, but additionally having chemical modification<br>
of one or more of its amino acid side groups, a-carbon<br>
atoms, terminal amino group, or terminal carboxylic acid<br>
group. A chemical modification includes, but is not limited<br>
to, adding chemical moieties, creating new bonds, and<br>
removing chemical moieties. Modifications at amino acid<br>
side groups include, without limitation, acylation of lysine<br>
e-amino groups, N-alkylation of arginine, histidine, or<br>
lysine, alkylation of glutamic or aspartic carboxylic acid<br>
groups, and deamidation of glutamine or asparagine.<br>
Modifications of the terminal, amino group include, without<br>
limitation, the des-amino, N-lower alkyl, N-di-lower alkyl,<br>
and N-acyl modifications. Modifications of the terminal<br>
carboxy group include, without limitation, the amide, lower<br>
alkyl amide, dialkyl amide, and lower alkyl ester<br>
modifications. Lower alkyl is C1-C4 alkyl. Furthermore,<br>
one or more side groups, or terminal groups, may be<br>
protected by protective groups known to the ordinarily-<br>
skilled protein chemist. The a-carbon of an amino acid may<br>
be mono- or dimethylated.<br>
Any GLP-1 compound can be part of the heterologous<br>
fusion proteins of the present invention as long as the GLP-<br>
1 compound itself is able to bind and induce signaling<br>
through the GLP-1 receptor. GLP-1 receptor binding and<br>
signal transduction can be assessed using in vitro assays<br>
such as those described in EP 619,322 and U.S. Patent No.<br>
5,120,712, respectively.<br>
Numerous active GLP-1 fragments, analogs and<br>
derivatives are known in the art and any of these analogs<br>
and derivatives can also be part of the heterologous fusion<br>
proteins of the present invention. Some examples of novel<br>
GLP-1 analogs as well as GLP-1 analogs and derivatives known<br>
in the art are provided herein.<br>
Some GLP-1 analogs and GLP-1 fragments known in the art<br>
include, for example, GLP-K7-34) and GLP-1 (7-35) , GLP-M7-<br>
36), Gln9-GLP-1(7-37), D-Gln9-GLP-1(7-37), Thr16-Lys18-GLP-<br>
1(7-37), and Lys18-GLP-1(7-37). GLP-1 analogs such as GLP-<br>
1(7-34) and GLP-1 (7-35) are disclosed in U.S. Patent No.<br>
5,118,666. Biologically processed forms of GLP-1 which have<br>
insulinotropic properties, such as GLP-1(7-36) are also<br>
known. Other known biologically active GLP-1 compounds are<br>
disclosed in U.S. Patent No 5,977,071 to Hoffmann, et al.,<br>
U.S. Patent No. 5,545,618 to Buckley, et al., and Adelhorst,<br>
et al., J. Biol. Chem. 269:6215 (1994).<br>
A preferred group of GLP-1 analogs is composed of<br>
GLP-1 analogs of formula I (SEQ ID NO: 2)<br>
7 8 9 10 11 12 13 14 15 16 17<br>
His-Xaa-Xaa-Gly-Xaa-Phe-Thr-Xaa-Asp-Xaa-Xaa-<br>
18 19 20 21 22 23 24 25 26 27 28<br>
Xaa-Xaa-Xaa-Xaa-Xaa-Xaa-Xaa-Xaa-Xaa-Xaa-Phe-<br>
29 30 31 32 33 34 35 36 37 38 39<br>
Ile-Xaa-Xaa-Xaa-Xaa-Xaa-Xaa-Xaa-Xaa-Xaa-Xaa-<br>
40 41 42 43 44 45<br>
Xaa-Xaa-Xaa-Xaa-Xaa-Xaa<br>
Formula I (SEQ ID NO: 2)<br>
wherein:<br>
Xaa at position 8 is Ala, Gly, Ser, Thr, Leu, Ile, Val,<br>
Glu, Asp, or Lys;<br>
Xaa at position 9 is Glu, Asp, or Lys;<br>
Xaa at position 11 is Thr, Ala, Gly, Ser, Leu, Ile, Val,<br>
Glu, Asp, or Lys;<br>
Xaa at position 14 is Ser, Ala, Gly, Thr, Leu, Ile, Val,<br>
Glu, Asp, or Lys;<br>
Xaa at position 16 is Val, Ala, Gly, Ser, Thr, Leu, Ile,<br>
Tyr, Glu, Asp, Trp, or Lys;<br>
Xaa at position 17 is Ser, Ala, Gly, Thr, Leu, Ile, Val,<br>
Glu, Asp, or Lys;<br>
Xaa at position 18 is Ser, Ala, Gly, Thr, Leu, Ile, Val,<br>
Glu, Asp, Trp, Tyr, or Lys;<br>
Xaa at position 19 is Tyr, Phe, Trp, Glu, Asp, Gln, or Lys;<br>
Xaa at position 20 is Leu, Ala, Gly, Ser, Thr, Ile, Val,<br>
Glu, Asp, Met, Trp, Tyr, or Lys;<br>
Xaa at position 21 is Glu, Asp, or Lys;<br>
Xaa at position 22 is Gly, Ala, Ser, Thr, Leu, Ile, Val,<br>
Glu, Asp, or Lys;<br>
Xaa at position 23 is Gln, Asn, Arg, Glu, Asp, or Lys;<br>
Xaa at position 24 is Ala, Gly, Ser, Thr, Leu, Ile, Val,<br>
Arg, Glu, Asp, or Lys;<br>
Xaa at position 25 is Ala, Gly, Ser, Thr, Leu, Ile, Val,<br>
Glu, Asp, or Lys;<br>
Xaa at position 26 is Lys, Arg, Gln, Glu, Asp, or His;<br>
Xaa at position 27 is Leu, Glu, Asp, or Lys;<br>
Xaa at position 30 is Ala, Gly, Ser, Thr, Leu, Ile, Val,<br>
Glu, Asp, or Lys;<br>
Xaa at position 31 is Trp, Phe, Tyr, Glu, Asp, or Lys;<br>
Xaa at position 32 is Leu, Gly, Ala, Ser, Thr, Ile, Val,<br>
Glu, Asp, or Lys;<br>
Xaa at position 33 is Val, Gly, Ala, Ser, Thr, Leu, Ile,<br>
Glu, Asp, or Lys;<br>
Xaa at position 34 is Asn, Lys, Arg, Glu, Asp, or His;<br>
Xaa at position 35 is Gly, Ala, Ser, Thr, Leu, Ile, Val,<br>
Glu, Asp, or Lys;<br>
Xaa at position 36 is Gly, Arg, Lys, Glu, Asp, or His;<br>
Xaa at position 37 is Pro, Gly, Ala, Ser, Thr, Leu, Ile,<br>
Val, Glu, Asp, or Lys, or is deleted;<br>
Xaa at position 38 is Ser, Arg, Lys, Glu, Asp, or His, or is<br>
deleted;<br>
Xaa at position 39 is Ser, Arg, Lys, Glu, Asp, or His, or is<br>
deleted;<br>
Xaa at position 40 is Gly, Asp, Glu, or Lys, or is deleted;<br>
Xaa at position 41 is Ala, Phe, Trp, Tyr, Glu, Asp, or Lys,<br>
or is deleted;<br>
Xaa at position 42 is Ser, Pro, Lys, Glu, or Asp, or is<br>
deleted;<br>
Xaa at position 43 is Ser, Pro, Glu, Asp, or Lys, or is<br>
deleted;<br>
Xaa at position 44 is Gly, Pro, Glu, Asp, or Lys, or is<br>
deleted;<br>
and<br>
Xaa at position 45 is Ala, Ser, Val, Glu, Asp, or Lys, or is<br>
deleted;<br>
provided that when the amino acid at position 37, 38,<br>
39, 40, 41, 42, 43, or 44 is deleted, then each amino acid<br>
downstream of that amino acid is also deleted.<br>
It is preferred that the GLP-1 compound of formula I<br>
contain less than six amino acids that differ from the<br>
corresponding amino acid in GLP-1(7-37)OH or Exendin-4. It<br>
is more preferred that less than five amino acids differ<br>
from the corresponding amino acid in GLP-1(7-37)OH or<br>
Exendin-4. It is even more preferred that less than four<br>
amino acids differ from the corresponding amino acid in GLP-<br>
1(7-37)OH or Exendin-4.<br>
GLP-1 compounds of the present invention include<br>
derivatives of formula I such as a C-1-6-ester, or amide, or<br>
C-1-6-alkylamide, or C-1-6-dialkylamide thereof. WO99/43706<br>
describes derivatives of GLP-1 compounds of formula I and is<br>
incorporated by reference herein in its entirety. The<br>
compounds of formula I derivatized as described in<br>
WO99/43706 and underivatized are encompassed by the present<br>
invention.<br>
Another preferred group of GLP-1 compounds is composed<br>
of GLP-1 analogs of formula II (SEQ ID NO: 3):<br>
7 8 9 10 11 12 13 14 15 16 17<br>
Xaa-Xaa-Xaa-Gly-Xaa-Xaa-Thr-Ser-Asp-Xaa-Ser-<br>
18 19 20 21 22 23 24 25 26 27 28<br>
Xaa-Tyr-Leu-Glu-Xaa-Xaa-Xaa-Ala-Xaa-Xaa-Phe-<br>
29 30 31 32 33 34 35 36 37<br>
Ile-Xaa-Xaa-Leu-Xaa-Xaa-Xaa-Xaa-R<br>
Formula II (SEQ ID NO: 3)<br>
wherein:<br>
Xaa at position 7 is: L-histidine, D-histidine, desamino-<br>
histidine, 2-amino-histidine, ß-hydroxy-histidine,<br>
homohistidine, a-fluoromethyl-histidine or a-methyl-<br>
histidine;<br>
Xaa at position 8 is: Gly, Ala, Val, Leu, Ile, Ser, or Thr;<br>
Xaa at position 9 is: Thr, Ser, Arg, Lys, Trp, Phe, Tyr,<br>
Glu, or His;<br>
Xaa at position 11 is: Asp, Glu, Arg, Thr, Ala, Lys, or His;<br>
Xaa at position 12 is: His, Trp, Phe, or Tyr;<br>
Xaa at position 16 is: Leu, Ser, Thr, Trp, His, Phe, Asp,<br>
Val, Tyr, Glu, or Ala;<br>
Xaa at position 18 is: His, Pro, Asp, Glu, Arg, Ser, Ala, or<br>
Lys;<br>
Xaa at position 22 is: Gly, Asp, Glu, Gin, Asn, Lys, Arg, or<br>
Cys;<br>
Xaa at position 23 is: His, Asp, Lys, Glu, Gln, or Arg;<br>
Xaa at position 24 is: Glu, Arg, Ala, or Lys;<br>
Xaa at position 26 is: Trp, Tyr, Phe, Asp, Lys, Glu, or His;<br>
Xaa at position 27 is: Ala, Glu, His, Phe, Tyr, Trp, Arg, or<br>
Lys;<br>
Xaa at position 30 is: Ala, Glu, Asp, Ser, or His;<br>
Xaa at position 31 is: Asp, Glu, Ser, Thr, Arg, Trp, or Lys;<br>
Xaa at position 33 is: Asp, Arg, Val, Lys, Ala, Gly, or Glu;<br>
Xaa at position 34 is: Glu, Lys, or Asp;<br>
Xaa at position 35 is: Thr, Ser, Lys, Arg, Trp, Tyr, Phe,<br>
Asp, Gly, Pro, His, or Glu;<br>
Xaa at position 36 is: Thr, Ser, Asp, Trp, Tyr, Phe, Arg,<br>
Glu, or His;<br>
R at position 37 is: Lys, Arg, Thr, Ser, Glu, Asp, Trp, Tyr,<br>
Phe, His, Gly, Gly-Pro, or is deleted.<br>
Another preferred group of GLP-1 compounds is composed<br>
of GLP-1 analogs of formula III (SEQ ID NO: 4):<br>
7 8 9 10 11 12 13 14 15 16 17<br>
Xaa-Xaa-Glu-Gly-Xaa-Xaa-Thr-Ser-Asp-Xaa-Ser-<br>
18 19 20 21 22 23 24 25 26 27 28<br>
Ser-Tyr-Leu-Glu-Xaa-Xaa-Xaa-Xaa-Lys-Xaa-Phe-<br>
29 30 31 32 33 34 35 36 37<br>
Ile-Xaa-Trp-Leu-Xaa-Xaa-Xaa-Xaa-R<br>
formula III (SEQ ID NO: 4)<br>
wherein:<br>
Xaa at position 7 is: L-histidine, D-histidine, desamino-<br>
histidine, 2-amino-histidine, ß-hydroxy-histidine,<br>
homohistidine, a-fluoromethyl-histidine or a-methyl-<br>
histidine;<br>
Xaa at position 8 is: Gly, Ala, Val, Leu, Ile, Ser, or Thr;<br>
Xaa at position 11 is: Asp, Glu, Arg, Thr, Ala, Lys, or His;<br>
Xaa at position 12 is: His, Trp, Phe, or Tyr;<br>
Xaa at position 16 is: Leu, Ser, Thr, Trp, His, Phe, Asp,<br>
Val, Glu, or Ala;<br>
Xaa at position 22: Gly, Asp, Glu, Gln, Asn, Lys, Arg, or<br>
Cys;<br>
Xaa at position 23 is: His, Asp, Lys, Glu, or Gln;<br>
Xaa at position 24 is: Glu, His, Ala, or Lys;<br>
Xaa at position 25 is: Asp, Lys, Glu, or His;<br>
Xaa at position 27 is: Ala, Glu, His, Phe, Tyr, Trp, Arg, or<br>
Lys;<br>
Xaa at position 30 is: Ala, Glu, Asp, Ser, or His;<br>
Xaa at position 33 is: Asp, Arg, Val, Lys, Ala, Gly, or Glu;<br>
Xaa at position 34 is: Glu, Lys, or Asp;<br>
Xaa at position 35 is: Thr, Ser, Lys, Arg, Trp, Tyr, Phe,<br>
Asp, Gly, Pro, His, or Glu;<br>
Xaa at position 36 is: Arg, Glu, or His;<br>
R at position 37 is: Lys, Arg, Thr, Ser, Glu, Asp, Trp, Tyr,<br>
Phe, His, Gly, Gly-Pro, or is deleted.<br>
Another preferred group of GLP-1 compounds is composed<br>
of GLP-1 analogs of formula IV (SEQ ID NO: 5) :<br>
7 8 9 10 11 12 13 14 15 16 17<br>
Xaa-Xaa-Glu-Gly-Thr-Xaa-Thr-Ser-Asp-Xaa-Ser-<br>
18 19 20 21 22 23 24 25 26 27 28<br>
Ser-Tyr-Leu-Glu-Xaa-Xaa-Ala-Ala-Xaa-Glu-Phe-<br>
29 30 31 32 33 34 35 36 37<br>
Ile-Xaa-Trp-Leu-Val-Lys-Xaa-Arg-R<br>
formula IV (SEQ ID NO: 5)<br>
wherein:<br>
Xaa at position 7 is: L-histidine, D-histidine, desamino-<br>
histidine, 2-amino-histidine, ß-hydroxy-histidine,<br>
homohistidine, a-fluoromethyl-histidine or a-methyl-<br>
histidine;<br>
Xaa at position 8 is: Gly, Ala, Val, Leu, Ile, Ser, Met, or<br>
Thr;<br>
Xaa at position 12 is: His, Trp, Phe, or Tyr;<br>
Xaa at position 16 is: Leu, Ser, Thr, Trp, His, Phe, Asp,<br>
Val, Glu, or Ala;<br>
Xaa at position 22 is: Gly, Asp, Glu, Gln, Asn, Lys, Arg, or<br>
Cys;<br>
Xaa at position 23 is: His, Asp, Lys, Glu, or Gln;<br>
Xaa at position 26 is: Asp, Lys, Glu, or His;<br>
Xaa at position 30 is: Ala, Glu, Asp, Ser, or His;<br>
Xaa at position 35 is: Thr, Ser, Lys, Arg, Trp, Tyr, Phe,<br>
Asp, Gly, Pro, His, or Glu;<br>
R at position 37 is: Lys, Arg, Thr, Ser, Glu, Asp, Trp, Tyr,<br>
Phe, His, Gly, Gly-Pro, or is deleted.<br>
Another preferred group of GLP-1 compounds is composed<br>
of GLP-1 analogs of formula V (SEQ ID NO: 6) :<br>
7 8 9 10 11 12 13 14 15 16 17<br>
Xaa-Xaa-Glu-Gly-Thr-Phe-Thr-Ser-Asp-Val-Ser-<br>
18 19 20 21 22 23 24 25 26 27 28<br>
Ser-Tyr-Leu-Glu-Xaa-Xaa-Xaa-Ala-Lys-Glu-Phe-<br>
29 30 31 32 33 34 35 36 37<br>
Ile-Xaa-Trp-Leu-Val-Lys-Gly-Arg-R<br>
formula V (SEQ ID NO: 6)<br>
wherein:<br>
Xaa at position 7 is: L-histidine, D-histidine, desamino-<br>
histidine, 2-amino-histidine, ß-hydroxy-histidine,<br>
homohistidine, a-fluoromethyl-histidine or a-methyl-<br>
histidine;<br>
Xaa at position 8 is: Gly, Ala, Val, Leu, Ile, Ser, or Thr;<br>
Xaa at position 22 is: Gly, Asp, Glu, Gln, Asn, Lys, Arg, or<br>
Cys;<br>
Xaa at position 23 is: His, Asp, Lys, Glu, or Gln;<br>
Xaa at position 24 is: Ala, Glu, His, Phe, Tyr, Trp, Arg, or<br>
Lys ;<br>
Xaa at position 30 is: Ala, Glu, Asp, Ser, or His;<br>
R at position 37 is: Lys, Arg, Thr, Ser, Glu, Asp, Trp, Tyr,<br>
Phe, His, Gly, Gly-Pro, or is deleted.<br>
Preferred GLP-1 compounds of formula I, II, III, IV,<br>
and V comprise GLP-1 analogs or fragments of GLP-1 analogs<br>
wherein the analogs or fragments contain an amino acid other<br>
than alanine at position 8 (position 8 analogs). It is<br>
preferable that these position 8 analogs contain one or more<br>
additional changes at positions 9, 11/ 12, 16, 18, 22, 23,<br>
24, 26, 27, 30, 31, 33, 34, 35, 36, and 37 compared to the<br>
corresponding amino acid of native GLP-1(7-37)OH. It is<br>
also preferable that these analogs have 6 or fewer changes<br>
compared to the corresponding amino acids in native GLP-1(7-<br>
37)OH or GLP-1(7-36)OH. More preferred analogs have 5 or<br>
fewer changes compared to the corresponding amino acids in<br>
native GLP-1(7-37)OH or GLP-1(7-36)OH or have 4 or fewer<br>
changes compared to the corresponding amino acids in native<br>
GLP-1(7-37)OH or GLP-1(7-36)OH. It is even more preferable<br>
that these analogs have 3 or fewer changes compared to the<br>
corresponding amino acids in native GLP-1 (7-37)OH or GLP-<br>
1(7-36)OH. It is most preferable that these analogs have 2<br>
or fewer changes compared to the corresponding amino acids<br>
in native GLP-1(7-37)OH.<br>
It has been found that the compounds of formula II,<br>
III, IV, and V have a reduced propensity to aggregate and<br>
generate insoluble forms. This is also important in the<br>
context of a fusion protein wherein the relatively small<br>
GLP-1 peptide must maintain an active conformation despite<br>
being fused to a much larger protein. Preferred GLP-1<br>
compounds of formula II, III, IV, and V encompassed by the<br>
fusion proteins of the present invention comprise GLP-1<br>
analogs or fragments of GLP-1 analogs in which glycine at<br>
position 22 and preferably alanine at position 8 have been<br>
replaced with another amino acid.<br>
When position 22 is aspartic acid, glutamic acid,<br>
arginine or lysine, position 8 is preferably glycine,<br>
valine, leucine, isolecine, serine, threonine or methionine<br>
and more preferably valine or glycine. When position 22 is<br>
a sulfonic acid such as cysteic acid, position 8 is<br>
preferably glycine, valine, leucine, isolecine, serine,<br>
threonine or methionine and more preferably valine or<br>
glycine.<br>
Other preferred GLP-1 compounds include GLP-1 analogs<br>
of formula IV (SEQ ID NO:5) wherein the analogs have the<br>
sequence of GLP-1 (7-37)OH except that the amino acid at<br>
position 8 is preferably glycine, valine, leucine,<br>
isoleucine, serine, threonine, or methionine and more<br>
preferably valine or glycine and position 30 is glutamic<br>
acid, aspartic acid, serine, or histidine and more<br>
preferably glutamatic acid.<br>
Other preferred GLP-1 compounds include GLP-1 analogs<br>
of formula TV (SEQ ID NO:5) wherein the analogs have the<br>
sequence of GLP-1 (7-37)OH except that the amino acid at<br>
position 8 is preferably glycine, valine, leucine,<br>
isoleucine, serine, threonine, or methionine and more<br>
preferably valine or glycine and position 37 is histidine,<br>
lysine, arginine, threonine, serine, glutamic acid, aspartic<br>
acid, tryptophan, tyrosine, phenylalanine and more<br>
preferably histidine.<br>
Other preferred GLP-1 compounds include GLP-1 analogs<br>
of formula IV (SEQ ID NO:5) wherein the analogs have the<br>
sequence of GLP-1(7-37)OH except that the amino acid at<br>
position 8 is preferably glycine, valine, leucine,<br>
isoleucine, serine, threonine, or methionine and more<br>
preferably valine or glycine and position 22 is glutamic<br>
acid, lysine, aspartic acid, or arginine and more preferably<br>
glutamic acid or lysine and position 23 is lysine, arginine,<br>
glutamic acid, aspartic acid, and histidine and more<br>
preferably lysine or glutamic acid.<br>
Other preferred GLP-1 compounds include GLP-1 analogs<br>
of formula V (SEQ ID NO:6) wherein the analogs have the<br>
sequence of GLP-1 (7-37)OH except that the amino acid at<br>
position 8 is preferably glycine, valine, leucine,<br>
isoleucine, serine, threonine, or methionine and more<br>
preferably valine or glycine and position 22 is glutamic<br>
acid, lysine, aspartic acid, or arginine and more preferably<br>
glutamine acid or lysine and position 27 is alanine, lysine,<br>
arginine, tryptophan, tyrosine, phenylalanine, or histidine<br>
and more preferably alanine.<br>
Other preferred GLP-1 compounds include GLP-1 analogs<br>
of formula II wherein the analogs have the sequence of GLP-<br>
1(7-37)OH except that the amino acid at position 8 and one,<br>
two, or three amino acids selected from the group consisting<br>
of position 9, position 11, position 12, position 16,<br>
position 18, position 22, position 23, position 24, position<br>
26, position 27, position 30, position 31, position 33,<br>
position 34, position 35, position 36, and position 37,<br>
differ from the amino acid at the corresponding position of<br>
native GLP-1(7-37)OH.<br>
Other preferred GLP-1 compounds of formula II include:<br>
Val8-GLP-1(7-37)OH, Gly8-GLP-1(7-37)OH, Glu22-GLP-1(7-37)OH,<br>
Asp22-GLP-1(7-37)OH, Arg22-GLP-1(7-37)OH, Lys22-GLP-1(7-<br>
37)OH, Cys22-GLP-1(7-37)OH, Val8-Glu22-GLP-1(7-37)0H, Val8-<br>
Asp22-GLP-1(7-37)OH, Val8-Arg22-GLP-1(7-37)OH, Val8-Lys22-<br>
GLP-1(7-37)OH, Val8-Cys22-GLP-1(7-37)OH, Gly8-Glu22-GLP-1(7-<br>
37)OH, Gly8-Asp22-GLP-1(7-37)OH, Gly8-Arg22-GLP-1(7-37)OH,<br>
Gly8-Lys22-GLP-1(7-37)OH, Gly8-Cys22-GLP-1(7-37)OH, Glu22-<br>
GLP-1(7-36)OH, Asp22-GLP-1(7-36)OH, Arg22-GLP-1(7-36)OH,<br>
Lys22-GLP-1(7-36)OH, Cys22-GLP-1(7-36)OH, Val8-Glu22-GLP-<br>
1(7-36)OH, Val8-Asp22-GLP-1(7-36)OH, Val8-Arg22-GLP-1(7-<br>
36)OH, Val8-Lys22-GLP-1(7-36)OH, Val8-Cys22-GLP-1(7-36)OH,<br>
Gly8-Glu22-GLP-1(7-36)OH, Gly8-Asp22-GLP-1(7-36)OH, Gly8-<br>
Arg22-GLP-1(7-36)OH, Gly8-Lys22-GLP-1(7-36)OH, Gly8-Cys22-<br>
GLP-1(7-36)OH, Lys23-GLP-1(7-37)OH, Val8-Lys23-GLP-1(7-<br>
37)OH, Gly8-Lys23-GLP-1(7-37)OH, His24-GLP-1(7-37)OH, Val8-<br>
His24-GLP-1(7-37)OH, Gly8-His24-GLP-1(7-37)OH, Lys24-GLP-<br>
1(7-37)OH, Val8-Lys24-GLP-1(7-37)OH, Gly8-Lys23-GLP-1(7-<br>
37)OH, Glu30-GLP-1(7-37)OH, Val8-Glu30-GLP-1(7-37)OH, Gly8-<br>
Glu30-GLP-1(7-37)OH, Asp30-GLP-1(7-37)OH, Val8-Asp30-GLP-<br>
1(7-37)OH, Gly8-Asp30_GLP-1(7-37)OH, Gln30_GLP-1(7-37)OH,<br>
Val8-Gln30-GLP-1(7-37)OH, Gly8-Gln30-GLP-1(7-37)OH, Tyr30-<br>
GLP-1(7-37)OH, Val8-Tyr30-GLP-1(7-37)OH, Gly8-Tyr30-GLP-1(7-<br>
37)OH, Ser30-GLP-1(7-37)OH, Val8-Ser30-GLP-1(7-37)OH, Gly8-<br>
Ser30-GLP-1(7-37)OH, His30-GLP-1(7-37)0H, Val8-His30-GLP-<br>
1(7-37)OH, Gly8-His30-GLP-1(7-37)OH, Glu34-GLP-1(7-37)OH,<br>
Val8-Glu34-GLP-1(7-37)OH, Gly8-Glu34-GLP-1(7-37)OH, Ala34-<br>
GLP-1(7-37)OH, Val8-Ala34-GLP-1(7-37)OH, Gly8-Ala34-GLP-1(7-<br>
37)OH, Gly34-GLP-1(7-37)OH, Val8-Gly34-GLP-1(7-37)OH, Gly8-<br>
Gly34-GLP-1(7-37)OH, Ala35-GLP-1(7-37)OH, Val8-Ala35-GLP-<br>
1(7-37)OH, Gly8-Ala35-GLP-1(7-37)OH, Lys35-GLP-1(7-37)OH,<br>
Val8-Lys35-GLP-1(7-37)OH, Gly8-Lys35-GLP-1(7-37)OH, His35-<br>
GLP-1(7-37)OH Val8-His35-GLP-1(7-37)OH, Gly8-His35-GLP-1(7-<br>
37)OH, Pro35-GLP-1(7-37)OH, Val8-Pro35-GLP-1(7-37)OH, Gly8-<br>
Pro35-GLP-1(7-37)OH, Glu35-GLP-1(7-37)OH Val8-Glu35-GLP-1(7-<br>
37)OH, Gly8-Glu35-GLP-1(7-37)OH, Val8-Ala27-GLP-1(7-37)OH,<br>
Val8-His37-GLP-1(7-37)OH, Val8-Glu22-Lys23-GLP-1(7-37)OH,<br>
Val8-Glu22-Glu23-GLP-1(7-37)OH, Val8-Glu22-Ala27-GLP-1(7-<br>
37)OH, Val8-Gly34-Lys35-GLP-1(7-37)OH, Val8-His37-GLP-1(7-<br>
37)OH, Gly8-His37-GLP-1(7-37)OH, Val8-Glu22-Ala27-GLP-1(7-<br>
37)OH, Gly8-Glu22-Ala27_GLP-1(7-37)OH, Val8-Lys22-Glu23-GLP-<br>
1(7-37)OH, and Gly8-Lys22-Glu23-GLP-1(7-37)OH.<br>
Another preferred group of GLP-1 analogs and<br>
derivatives for use in the present invention is composed of<br>
molecules of formula VI (SEQ ID NO: 7)<br>
R1-X-Glu-Gly10-Thr-Phe-Thr-Ser-Asp15-Val-Ser-<br>
Ser-Tyr-Leu20-Y-Gly-Gln-Ala-Ala25-Lys- Z -<br>
Phe-Ile-Ala30-Trp-Leu-Val-Lys-Gly35-Arg-R2<br>
formula VI (SEQ ID NO:7)<br>
wherein: R1 is selected from the group consisting of L-<br>
histidine, D-histidine, desamino-histidine, 2-amino-<br>
histidine, ß-hydroxy-histidine, homohistidine, alpha-<br>
fluoromethyl-histidine, and alpha-methyl-histidine; X is<br>
selected from the group consisting of Ala, Gly, Val, Thr,<br>
Ile, and alpha-methyl-Ala; Y is selected from the group<br>
consisting of Glu, Gln, Ala, Thr, Ser, and Gly; Z is<br>
selected from the group consisting of Glu, Gln, Ala, Thr,<br>
Ser, and Gly; and R2 is Gly-OH.<br>
Another preferred group of GLP-1 compounds for use<br>
in the present invention is disclosed in WO 91/11457, and<br>
consists essentially of GLP-1(7-34), GLP-1(7-35), GLP-1(7-<br>
36), or GLP-1 (7-37), or the amide form thereof, and<br>
pharmaceutically-acceptable salts thereof, having at least<br>
one modification selected from the group consisting of:<br>
(a) substitution of glycine, serine, cysteine,<br>
threonine, asparagine, glutamine, tyrosine, alanine, valine,<br>
isoleucine, leucine, methionine, phenylalanine, arginine, or<br>
D-lysine for lysine at position 26 and/or position 34; or<br>
substitution of glycine, serine, cysteine, threonine,<br>
asparagine, glutamine, tyrosine, alanine, valine,<br>
isoleucine, leucine, methionine, phenylalanine, lysine, or a<br>
D-arginine for arginine at position 36;<br>
(b) substitution of an oxidation-resistant amino<br>
acid for tryptophan at position 31;<br>
(c) substitution of at least one of: tyrosine for<br>
valine at position 16; lysine for serine at position 18;<br>
aspartic acid for glutamic acid at position 21; serine for<br>
glycine at position 22; arginine for glutamine at position<br>
23; arginine for alanine at position 24; and glutamine for<br>
lysine at position 26; and<br>
(d) substitution of at least one of: glycine,<br>
serine, or cysteine for alanine at position 8; aspartic<br>
acid, glycine, serine, cysteine, threonine, asparagine,<br>
glutamine, tyrosine, alanine, valine, isoleucine, leucine,<br>
methionine, or phenylalanin.e for glutamic acid at position<br>
9; serine, cysteine, threonine, asparagine, glutamine,<br>
tyrosine, alanine, valine, isoleucine, leucine, methionine,<br>
or phenylalanine for glycine at position 10; and glutamic<br>
acid for aspartic acid at position 15; and<br>
(e) substitution of glycine, serine, cysteine,<br>
threonine, asparagine, glutamine, tyrosine, alanine, valine,<br>
isoleucine; leucine, methionine. or phenylalanine, or the D-<br>
or N-acylated or alkylated form of histidine for histidine<br>
at position 7; wherein, in the substitutions is (a), (b) ,<br>
(d), and (e), the substituted amino acids can optionally be<br>
in the D-form and the amino acids substituted at position 7<br>
can optionally be in the N-acylated or N-alkylated form.<br>
Because the enzyme, dipeptidyl-peptidase IV (DPP IV),<br>
may be responsible for the observed rapid in vivo<br>
inactivation of administered GLP-1, [see, e.g., Mentlein,<br>
R., et al., Eur. J. Biochem., 214:829-835 (1993)], GLP-1<br>
analogs and derivatives that are protected from the activity<br>
of DPP IV in the context of a fusion protein are preferred,<br>
and fusion proteins wherein the GLP-1 compound is Gly8-GLP-<br>
1(7-37)OH, Val8-GLP-1(7-37)OH, a-methyl-Ala8-GLP-1(7-37)OH,<br>
or Gly8-Gln21-GLP-1(7-37)OH are more preferred.<br>
Another preferred group of GLP-1 compounds for use in<br>
the present invention consists of the compounds of formula<br>
VII (SEQ ID NO: 8) claimed in U.S. Patent No. 5,512,549,<br>
which is expressly incorporated herein by reference.<br><br>
wherein R1 is selected from the group consisting of 4-<br>
imidazopropionyl, 4-imidazoacetyl, or 4-imidazo-<br>
a, a dimethyl-ace tyl; R2 is selected from the group<br>
consisting of C6-C10 unbranched acyl, or is absent; R3 is<br>
selected from the group consisting of Gly-OH or NH2; and,<br>
Xaa is Lys or Arg.<br>
More preferred compounds of formula IV for use in the<br>
present invention are those in which Xaa is Arg and R2 is<br>
C6-C10 unbranched acyl. Even more preferred compounds of<br>
formula IV for use in the present invention are those in<br>
which Xaa is Arg, R2 is C6-C10 unbranched acyl, and R3 is<br>
Gly-OH. Other highly-preferred compounds of formula IV for<br>
use in the present invention are those in which Xaa is Arg,<br>
R2 is C6-C10 unbranched acyl, R3 is Gly-OH, and R1 is 4-<br>
imidazopropionyl. An especially preferred compound of<br>
formula TV for use in the present invention is that in which<br>
Xaa is Arg, R2 is C8 unbranched acyl, R3 is Gly-OH, and R1<br>
is 4-imidazopropionyl.<br>
Preferably, the GLP-1 compounds comprise GLP-1 analogs<br>
wherein the backbone for such analogs or fragments contains<br>
an amino acid other than alanine at position 8 (position 8<br>
analogs). The backbone may also include L-histidine, D-<br>
histidine, or modified forms of histidine such as desamino-<br>
histidine, 2-amino-histidine, ß-hydroxy-histidine,<br>
homohistidine, a-fluoromethyl-histidine, or a-methyl-<br>
histidine at position 7. It is preferable that these<br>
position 8 analogs contain one or more additional changes at<br>
positions 12, 16, 18, 19, 20, 22, 25, 27, 30, 33, and 37<br>
compared to the corresponding amino acid of native GLP-1(7-<br>
37)OH. It is more preferable that these position 8 analogs<br>
contain one or more additional changes at positions 16, 18,<br>
22, 25 and 33 compared to the corresponding amino acid of<br>
native GLP-1(7-37)OH.<br>
In a preferred embodiment, the GLP-1 analog is GLP-1(7-<br>
37)OH wherein the amino acid at position 12 is selected from<br>
the group consisting of tryptophan or tyrosine. It is more<br>
preferred that in addition to the substitution at position<br>
12, the amino acid at position 8 is substituted with<br>
glycine, valine, leucine, isoleucine, serine, threonine, or<br>
methionine and more preferably valine or glycine. It is<br>
even more preferred that in addition to the substitutions at<br>
position 12 and 8, the amino acid at position 22 is<br>
substituted with glutamic acid.<br>
In another preferred embodiment, the GLP-1 analog is<br>
GLP-1(7-37)OH wherein the amino acid at position 16 is<br>
selected from the group consisting of tryptophan,<br>
isoleucine, leucine, phenylalanine, or tyrosine. It is more<br>
preferred that in addition to the substitution at position<br>
16, the amino acid at position 8 is substituted with<br>
glycine, valine, leucine, isoleucine, serine, threonine, or<br>
methionine and more preferably valine or glycine. It is<br>
even more preferred that in addition to the substitutions at<br>
position 16 and 8, the amino acid at position 22 is<br>
substituted with glutamic acid. It is also preferred that<br>
in addition to the substitutions at positions 16 and 8, the<br>
amino acid at position 30 is substituted with glutamic acid.<br>
It is also preferred that in addition to the substitutions<br>
at positions 16 and 8, the amino acid at position 37 is<br>
substituted with histidine.<br>
In another preferred embodiment, the GLP-1 analog is<br>
GLP-1(7-37)OH wherein the amino acid at position 18 is<br>
selected from the group consisting of tryptophan, tyrosine,<br>
phenylalanine, lysine, leucine, or isoleucine, preferably<br>
tryptophan, tyrosine, and isoleucine. It is more preferred<br>
that in addition to the substitution at position 18, the<br>
amino acid at position 8 is substituted with glycine,<br>
valine, leucine, isoleucine, serine, threonine, or<br>
methionine and more preferably valine or glycine. It is<br>
even more preferred that in addition to the substitutions at<br>
position 18 and 8, the amino acid at position 22 is<br>
substituted with glutamic acid. It is also preferred that<br>
in addition to the substitutions at positions 18 and 8, the<br>
amino acid at position 30 is substituted with glutamic acid.<br>
It is also preferred that in addition to the substitutions<br>
at positions 18 and 8, the amino acid at position 37 is<br>
substituted with histidine<br>
In another preferred embodiment, the GLP-1 analog is<br>
GLP-1(7-37)OH wherein the amino acid at position 19 is<br>
selected from the group consisting of tryptophan or<br>
phenylalanine, preferably tryptophan. It is more preferred<br>
that in addition to the substitution at position 19, the<br>
amino acid at position 8 is substituted with glycine,<br>
valine, leucine, isoleucine, serine, threonine, or<br>
methionine and more preferably valine or glycine. It is<br>
even more preferred that in addition to the substitutions at<br>
position 19 and 8, the amino acid at position 22 is<br>
substituted with glutamic acid. It is also preferred that<br>
in addition to the substitutions at positions 19 and 8, the<br>
amino acid at position 30 is substituted with glutamic acid.<br>
It is also preferred that in addition to the substitutions<br>
at positions 19 and 8, the amino acid at position 37 is<br>
substituted with histidine<br>
In another preferred embodiment, the GLP-1 analog is<br>
GLP-1(7-37)OH wherein the amino acid at position 20 is<br>
phenylalanine, tyrosine, or tryptophan. It is more<br>
preferred that in addition to the substitution at position<br>
20, the amino acid at position 8 is substituted with<br>
glycine, valine, leucine, isoleucine, serine, threonine, or<br>
methionine and more preferably valine or glycine. It is<br>
even more preferred that in addition to the substitutions at<br>
position 20 and 8, the amino acid at position 22 is<br>
substituted with glutamic acid. It is also preferred that<br>
in addition to the substitutions at positions 20 and 8, the<br>
amino acid at position 30 is substituted with glutamic acid.<br>
It is also preferred that in addition to the substitutions<br>
at positions 20 and 8, the amino acid at position 37 is<br>
substituted with histidine<br>
In another preferred embodiment, the GLP-1 analog is<br>
GLP-1(7-37)OH wherein the amino acid at position 25 is<br>
selected from the group consisting of valine, isoleucine,<br>
and leucine, preferably valine. It is more preferred that<br>
in addition to the substitution at position 25, the amino<br>
acid at position 8 is substituted with glycine, valine,<br>
leucine, isoleucine, serine, threonine, or methionine and<br>
more preferably valine or glycine. It is even more<br>
preferred that in addition to the substitutions at position<br>
25 and 8, the amino acid at position 22 is substituted with<br>
glutamic acid. It is also preferred that in addition to the<br>
substitutions at positions 25 and 8, the amino acid at<br>
position 30 is substituted with glutamic acid. It is also<br>
preferred that in addition to the substitutions at positions<br>
25 and 8, the amino acid at position 37 is substituted with<br>
histidine.<br>
In another preferred embodiment, the GLP-1 analog is<br>
GLP-1(7-37)OH wherein the amino acid at position 27 is<br>
selected from the group consisting of isoleucine or alanine.<br>
It is more preferred that in addition to the substitution at<br>
position 27, the amino acid at position 8 is substituted<br>
with glycine, valine, leucine, isoleucine, serine,<br>
threonine, or methionine and more preferably valine or<br>
glycine. It is even more preferred that in addition to the<br>
substitutions at position 27 and 8, the amino acid at<br>
position 22 is substituted with glutamic acid. It is also<br>
preferred that in addition to the substitutions at positions<br>
27 and 8, the amino acid at position 30 is substituted with<br>
glutamic acid. It is also preferred that in addition to the<br>
substitutions at positions 27 and 8, the amino acid at<br>
position 37 is substituted with histidine<br>
In another preferred embodiment, the GLP-1 analog is<br>
GLP-1(7-37)OH wherein the amino acid at position 33 is<br>
isoleucine. It is more preferred that in addition to the<br>
substitution at position 33, the amino acid at position 8 is<br>
substituted with glycine, valine, leucine, isoleucine,<br>
serine, threonine, or methionine and more preferably valine<br>
or glycine. It is even more preferred that in addition to<br>
the substitutions at position 33 and 8, the amino acid at<br>
position 22 is substituted with glutamic acid. It is also<br>
preferred that in addition to the substitutions at positions<br>
33 and 8, the amino acid at position 30 is substituted with<br>
glutamic acid. It is also preferred that in addition to the<br>
substitutions at positions 33 and 8, the amino acid at<br>
position 37 is substituted with histidine<br>
The GLP-1 compounds have modifications at one or more<br>
of the following positions: 8, 12, 16, 18, 19, 20, 22, 25,<br>
27, 30, 33, and 37. These GLP-1 compounds show increased<br>
potency compared with GLP-1(7-37)OH and comprise the amino<br>
acid sequence of formula IX (SEQ ID NO:12)<br>
Xaa7-Xaa8-Glu-Gly-Thr-Xaa12-Thr-Ser-Asp-Xaa16-Ser-<br>
Xaa18-Xaa19-Xaa20-Glu-Xaa22-Gln-Ala-Xaa25-Lys-Xaa27-<br>
Phe-Ile-Xaa30-Trp-Leu-Xaa33-Lys-Gly-Arg-Xaa37<br>
Formula IX (SEQ ID NO: 12)<br>
wherein:<br>
Xaa7 is: L-histidine, D-histidine, desamino-histidine, 2-<br>
amino-histidine, ß-hydroxy-histidine,<br>
homohistidine, a-fluoromethyl-histidine, or a-<br>
methyl-histidine;<br>
Xaa8 is: Ala, Gly, Val, Leu, Ile, Ser, or Thr;<br>
Xaa12 is: Phe, Trp, or Tyr;<br>
Xaa16 is: Val, Trp, Ile, Leu, Phe, or Tyr;<br>
Xaa18 is: Ser, Trp, Tyr, Phe, Lys, Ile, Leu, Val;<br>
Xaa19 is: Tyr, Trp, or Phe;<br>
Xaa20 is: Leu, Phe, Tyr, or Trp;<br>
Xaa22 is: Gly, Glu, Asp, or Lys;<br>
Xaa25 is: Ala, Val, Ile, or Leu;<br>
Xaa27 is: Glu, Ile, or Ala;<br>
Xaa30 is: Ala or Glu<br>
Xaa33 is: Val, or He; and<br>
Xaa37 is: Gly, His, NH2, or is absent.<br>
Some preferred GLP-1 compounds of formula IX include<br>
GLP-1(7-37)OH, GLP-1(7-36)-NH2, Gly8-GLP-1(7-37)OH, Gly8-GLP-<br>
1(7-36)NH2, Val8-GLP-1(7-37)OH, Val8-GLP-1 (7-36)NH2, Leu8-<br>
GLP-1(7-37)OH, Leu8-GLP-1(7-36)NH2, Ile8-GLP-1(7-37)OH, Ile8-<br>
GLP-1(7-36)NH2, Ser8-GLP-1(7-37)OH, Ser8-GLP-1(7-36)NH2,<br>
Thr8-GLP-1(7-37)OH, Thr8-GLP-1(7-36)NH2, Val8-Tyr12-GLP-1(7-<br>
37)OH, Val8-Tyr12-GLP-1(7-36)NH2, Val8-Tyr16-GLP-1(7-37)OH,<br>
Val8-Tyr16-GLP-1(7-36)NH2, Val8-Glu22-GLP-1(7-37)OH, Val8-<br>
Glu22-GLP-l(7-36)NH2, Gly8-Glu22-GLP-1(7-37)OH, Gly8-Glu22-GLP-<br>
1(7-36)NH2, Val8-Asp22-GLP-1(7-37)OH, Val8-Asp22-GLP-1(7-<br>
36)NH2, Gly8-Asp22-GLP-1(7-37)OH, Gly8-Asp22-GLP-1(7-36)NH2,<br>
Val8-Lys22-GLP-1(7-37)OH, Val8-Lys22-GLP-1(7-36)NH2, Gly8-<br>
Lys22-GLP-1(7-37)OH, Gly8-Lys22-GLP-1(7-36)NH2, Leu8-Glu22-GLP-<br>
1(7-37)OH, Leu8-Glu22-GLP-1(7-36)NH2, Ile8-Glu22-GLP-1(7-<br>
37) OH, Ile8-Glu22-GLP-1(7-36)NH2, Leu8-Asp22-GLP-1(7-37) OH,<br>
Leu8-Asp22-GLP-1(7-36)NH2, Ile8-Asp22-GLP-1(7-37)OH, Ile8-<br>
Asp22-GLP-1(7-36)NH2, Leu8-Lys22-GLP-1(7-37)OH, LeuB-Lys22-GLP-<br>
1(7-36)NH2, Ile8-Lys22-GLP-1(7-37)OH, Ile8-Lys22-GLP-1(7-<br>
36)NH2, Ser8-Glu22-GLP-1(7-37)OH, Ser8-Glu22-GLP-1(7-36)NH2,<br>
Thr8-Glu22-GLP-1(7-37)OH, Thr8-Glu22-GLP-1(7-36)NH2, Ser8-<br>
Asp22-GLP-1(7-37)OH, Ser8-Asp22-GLP-1(7-36)NH2, Thr8-Asp22-GLP-<br>
1(7-37)OH, Thr8-Asp22-GLP-1(7-36)NH2, Ser8-Lys22-GLP-1(7-<br>
37)OH, Ser8-Lys22-GLP-1(7-36)NH2, Thr8-Lys22-GLP-1(7-37)OH,<br>
Thr8-Lys22-GLP-1(7-36)NH2, Glu22-GLP-1(7-37)OH, Glu22-GLP-1(7-<br>
36)NH2, Asp22-GLP-1(7-37)OH, Asp22-GLP-1 (7-36)NH2, Lys22-GLP-<br>
1(7-37)OH, Lys22-GLP-1(7-36)NH2, Val8-Ala27-GLP-1(7-37)OH,<br>
Val8-Glu22-Ala27-GLP-1(7-37)OH, Val8-Glu30-GLP-1(7-37)OH, Val8-<br>
Glu30-GLP-1(7-36)NH2, Gly8-Glu30-GLP-1(7-37)OH, Gly8-Glu30-GLP-<br>
1(7-36)NH2, Leu8-Glu30-GLP-1(7-37)OH, Leu8-Glu30-GLP-1(7-<br>
36)NH2, Ile8-Glu30-GLP-1(7-37)OH, Ile8-Glu30-GLP-1(7-36)NH2,<br>
Ser8-Glu30-GLP-1(7-37)OH, Ser8-Glu30-GLP-1(7-36)NH2, Thr8-<br>
Glu30-GLP-1(7-37)OH, Thr8-Glu30-GLP-1(7-36)NH2, Val8-His37-GLP-<br>
1(7-37)OH, Val8-His37-GLP-1(7-36)NH2, Gly8-His37-GLP-1(7-37)OH,<br>
Gly8-His37-GLP-1(7-36)NH2, Leu8-His37-GLP-1(7-37)OH, Leu8-His37-<br>
GLP-1(7-36)NH2, Ile8-His37-GLP-1(7-37)OH, Ile8-His37-GLP-1(7-<br>
36)NH2, Ser8-His37-GLP-1(7-37)OH, Ser8-His37-GLP-1(7-36)NH2,<br>
Thr8-His37-GLP-1(7-37)OH, Thr8-His37-GLP-1 (7-36)NH2.<br>
Some preferred GLP-1 compounds of formula IX having<br>
multiple substitutions include GLP-1(7-37)OH wherein<br>
position 8 is valine or glycine, position 22 is glutamic<br>
acid, position 16 is tyrosine, leucine or tryptophan,<br>
position 18 is tyrosine, tryptophan, or isoleucine, position<br>
25 is valine and position 33 is isoleucine. Other preferred<br>
GLP-1 compounds include the following: Val8-Tyr16-GLP-1,(7-<br>
37)OH, Val8-Tyr12-Glu22-GLP-1(7-37)OH, Val8-Tyr16-Phe19-GLP-<br>
1(7-37) OH, Val8-Tyr16-Glu22-GLP-1(7-37)OH, Val8-Trp16-Glu22-<br>
GLP-1(7-37)OH, Val8-Leu16-Glu22-GLP-1(7-37)OH, Val8-Ile16-<br>
Glu22-GLP-1(7-37)OH, Val8-Phe16-Glu22-GLP-1(7-37)OH; Val8-<br>
Trp18-Glu22-GLP-1(7-37)OH, Val8-Tyr18-Glu22-GLP-1(7-37)OH,<br>
Val8-Phe18-Glu22-GLP-1(7-37)OH, and Val8-Ile18-Glu22-GLP-1(7-<br>
37)OH.<br>
The GLP-1 compounds of the present invention also<br>
encompass Exendin compounds. Exendin-3 and Exendin-4 are<br>
biologically active peptides first isolated from<br>
Helodermatidae lizard venoms and have been shown to bind the<br>
GLP-1 receptor and stimulate cAMP-dependent H+ production in<br>
mammalian parietal cells. Exendin-3 and Exendin-4 are both<br>
39 amino acid peptides which are approximately 53%<br>
homologous to GLP-1. They act as potent agonists of GLP-1<br>
activity. Notably, an N-terminally truncated derivative of<br>
Exendin, known as Exendin(9-39 amino acids), is an inhibitor<br>
of Exendin-3, Exendin-4 and GLP-1.<br>
An Exendin compound typically comprises a polypeptide<br>
having the amino acid sequence of Exendin-3, Exendin-4, or<br>
an analog or fragment thereof. Exendin-3 and Exendin-4 are<br>
disclosed in U.S. Patent No. 5,424,286.<br>
Exendin-3 has the amino acid sequence of SEQ ID NO: 9:<br>
7 8 9 10 11 12 13 14 15 16 17<br>
His-Ser-Asp-Gly-Thr-Phe-Thr-Ser-Asp-Leu-Ser-<br>
18 19 20 21 22 23 24 25 26 27 28<br>
Lys-Gln-Met-Glu-Glu-Glu-Ala-Val-Arg-Leu-Phe-<br>
29 30 31 32 33 34 35 36 37 38 39<br>
Ile-Glu-Trp-Leu-Lys-Asn-Gly-Gly-Pro-Ser-Ser-<br>
40 41 42 43 44 45<br>
Gly-Ala-Pro-Pro-Pro-Ser<br>
(SEQ ID NO: 9)<br>
Exendin-4 has the amino acid sequence of SEQ ID NO: 10:<br>
7 8 9 10 11 12 13 14 15 16 17<br>
His-Gly-Glu-Gly-Thr-Phe-Thr-Ser-Asp-Leu-Ser-<br>
18 19 20 21 22 23 24 25 26 27 28<br>
Lys-Gln-Met-Glu-Glu-Glu-Ala-Val-Arg-Leu-Phe-<br>
29 30 31 32 33 34 35 36 37 38 39<br>
Ile-Glu-Trp-Leu-Lys-Asn-Gly-Gly-Pro-Ser-Ser-<br>
40 41 42 43 44 45<br>
Gly-Ala-Pro-Pro-Pro-Ser<br>
(SEQ ID NO: 10)<br>
GLP-1 compounds also include Exendin fragments which<br>
are polypeptides obtained after truncation of one or more<br>
amino acids from the N-terminus and/or C-terminus of Exendin<br>
or an Exendin analog. Furthermore, GLP-1 compounds include<br>
Exendin polypeptides in which one or more amino acids have<br>
been added to the N- terminus and/or C-terminus of Exendin or<br>
fragments thereof. Exendin compounds of this type have up to<br>
about forty-five amino acids.<br>
GLP-1 compounds also include "Exendin analogs." An<br>
Exendin analog has sufficient homology to Exendin-4,<br>
Exendin-3, or a fragment thereof such that the analog has<br>
insulinotropic activity. The activity of Exendin fragments<br>
and/or analogs can be assessed using in vitro assays such as<br>
those described in EP 619,322 and U.S. Patent No. 5,120,712.<br>
Preferably, an Exendin analog has the amino acid<br>
sequence of Exendin-4 or a fragment thereof, modified so<br>
that from one, two, three, four or five amino acids differ<br>
from the amino acid in corresponding position of Exendin-4<br>
or the fragment of Exendin-4. In the nonmenclature used<br>
herein to designate Exendin compounds, the substituting<br>
amino acid and its position is indicated prior to the parent<br>
structure. For example, Val8-Exendin-4 designates an<br>
Exendin compound in which the glycine normally found at<br>
position 8 of Exendin-4 has been replaced with valine.<br>
Another preferred group of GLP-1 compounds is composed<br>
of GLP-1/Exendin-4 analogs of formula VIII (SEQ ID NO:11).<br>
7 8 9 10 11 12 13 14 15 16 17<br>
Xaa-Xaa-Glu-Gly-Thr-Phe-Thr-Ser-Asp-Xaa-Ser-<br>
18 19 20 21 22 23 24 25 26 27 28<br>
Xaa-Xaa-Xaa-Glu-Xaa-Xaa-Ala-Xaa-Xaa-Xaa-Phe-<br>
29 30 31 32 33 34 35 36 37<br>
Ile-Xaa-Trp-Leu-Xaa-Xaa-Gly-Xaa-R<br>
formula VIII (SEQ ID NO: 11)<br>
wherein:<br>
Xaa at position 7 is: L-histidine, D-histidine, desamino-<br>
histidine, 2-amino-histidine, ß-hydroxy-histidine,<br>
homohistidine, a-fluoromethyl-histidine or a-methyl-<br>
histidine;<br>
Xaa at position 8 is: Gly, Ala, or Val;<br>
Xaa at position 16 is: Leu or Val;<br>
Xaa at position 18 is Lys or Ser;<br>
Xaa at position 19 is: Gln or Tyr;<br>
Xaa at position 20 is: Met or Leu;<br>
Xaa at position 22 is: Glu or Gln;<br>
Xaa at position 23 is: Glu, or Gln;<br>
Xaa at position 25 is: Val or Ala;<br>
Xaa at position 26 is: Arg or Lys;<br>
Xaa at position 27 is Leu or Glu;<br>
Xaa at position 30 is: Glu or Ala;<br>
Xaa at position 33 is: Val or Lys;<br>
Xaa at position 34 is: Asn or Lys;<br>
Xaa at position 36 is: Gly or Arg; and<br>
R at position 37 is: Gly, Pro, Pro-Ser-Ser-Gly-Ala-Pro-Pro-<br>
Pro-Ser, or is absent. The activity of 18 different species<br>
that fall within this genus is provided in Table 6.<br>
Further Exendin-analogs that are useful for the present<br>
invention are described in PCT patent publications WO<br>
99/25728 (Beeley et al.), WO 99/25727 Beeley et al.), WO<br>
98/05351 (Young et al.), WO 99/40788 (Young et al.), WO<br>
99/07404 (Beeley et al), and WO 99/43708 (Knudsen et al) .<br>
The GLP-1 fusion proteins of the present invention can<br>
comprise glycosylation sites. Glycosylation is a chemical<br>
modification wherein sugar moieties are added to the protein<br>
at specific sites. Glycosylation of proteins play a role in<br>
ensuring the correct charge, confirmation, and stability of<br>
maturing protein and can target the protein to the cell<br>
surface and eventual secretion of the protein. Most<br>
importantly, glycosylation effects the in vivo clearance<br>
rate for many proteins. Sugars can be O-linked or N-linked.<br>
Generally, O-linked sugars are added to the hydroxyl-group<br>
oxygen of serine and threonine, while N-linked sugars are<br>
added to the amide nitrogen of asparagine. The consensus<br>
site for N-glycosylation is Asn X1 X2 wherein X1 is any<br>
amino acid except Pro and X2 is Ser or Thr.<br>
GLP-1 compounds are generally not glycosylated in vivo;<br>
however, interestingly the GLP-1 fusion proteins of the<br>
present invention that comprise a GLP-1 compound with a C<br>
terminal extension fused to an Fc sequence is glycosylated<br>
at the last serine in the C terminal extension (SSGAPPPS*)<br>
and at threonine at position 11 in the N terminal region of<br>
Fc (AEPKSCDKTHT*CPPC<br>
Heterologous Fc fusion proteins:<br>
The GLP-1 compounds described above can be fused<br>
directly or via a peptide linker to the Fc portion of an<br>
immunoglobulin.<br>
Immunoglobulins are molecules containing polypeptide<br>
chains held together by disulfide bonds, typically having<br>
two light chains and two heavy chains. In each chain, one<br>
domain (V) has a variable amino acid sequence depending on<br>
the antibody specificity of the molecule. The other domains<br>
(C) have a rather constant sequence common to molecules of<br>
the same class.<br>
As used herein, the Fc portion of an immunoglobulin has<br>
the meaning commonly given to the term in the field of<br>
immunology. Specifically, this term refers to an antibody<br>
fragment which is obtained by removing the two antigen<br>
binding regions (the Fab fragments) from the antibody. One<br>
way to remove the Fab fragments is to digest the<br>
immunoglobulin with papain protease. Thus, the Fc portion<br>
is formed from approximately equal sized fragments of the<br>
constant region from both heavy chains, which associate<br>
through non-covalent interactions and disulfide bonds. The<br>
Fc portion can include the hinge regions and extend through<br>
the CH2 and CH3 domains to the C-terminus of the antibody.<br>
Representative hinge regions for human and mouse<br>
immunoglobulins can be found in Antibody Engineering, A<br>
Practical Guide, Borrebaeck, C.A.K., ed., W.H. Freeman and<br>
Co., 1992, the teachings of which are herein incorporated by<br>
reference. The Fc portion can further include one or more<br>
glycosylation sites. The amino acid sequence of a<br>
representative Fc protein containing a hinge region, CH2 and<br>
CH3 domains, and one N-glycosylation site at position 82 is<br>
shown in Figure 1.<br>
There are five types of human inununoglobulin Fc regions<br>
with different effect or and pharmacokinetic properties:<br>
IgG, IgA, IgM, IgD, and IgE. IgG is the most abundant<br>
immunoglobulin in serum. IgG also has the longest half-life<br>
in serum of any immunoglobulin (23 days) . Unlike other<br>
immunoglobulins, IgG is efficiently recirculated following<br>
binding to an Fc receptor. There are four IgG subclasses<br>
G1, G2, G3, and G4, each of which have different effect or<br>
functions. G1, G2, and G3 can bind C1q and fix complement<br>
while G4 cannot. Even though G3 is able to bind C1q more<br>
efficiently than G1, G1 is more effective at mediating<br>
complement-directed cell lysis. G2 fixes complement very<br>
inefficiently. The Clq binding site in IgG is located at<br>
the carboxy terminal region of the CH2 domain.<br>
All IgG subclasses are capable of binding to Fc<br>
receptors (CD16, CD32, CD64) with Gl and G3 being more<br>
effective than G2 and G4. The Fc receptor binding region of<br>
IgG is formed by residues located in both the hinge and the<br>
carboxy terminal regions of the CH2 domain.<br>
IgA can exist both in a monomeric and dimeric form held<br>
together by a J-chain. IgA is the second most abundant Ig<br>
in serum, but it has a half-life of only 6 days. IgA has<br>
three effect or functions. It binds to an IgA specific<br>
receptor on macrophages and eosinophils, which drives<br>
phagocytosis and degranulation, respectively. It can also<br>
fix complement via an unknown alternative pathway.<br>
IgM is expressed as either a pentamer or a hexamer,<br>
both of which are held together by a J-chain. IgM has a<br>
serum half-life of 5 days. It binds weakly to Clq via a<br>
binding site located in its CH3 domain. IgD has a half-life<br>
of 3 days in serum. It is unclear what effect or functions<br>
are attributable to this Ig. IgE is a monomeric Ig and has<br>
a serum half-life of 2.5 days. IgE binds to two Fc<br>
receptors which drives degranulation and results in the<br>
release of proinflammatory agents.<br>
Depending on the desired in vivo effect, the<br>
heterologous fusion proteins of the present invention may<br>
contain any of the isotypes described above or may contain<br>
mutated Fc regions wherein the complement and/or Fc receptor<br>
binding functions have been altered. Thus, the heterologous<br>
fusion proteins of the present invention may contain the<br>
entire Fc portion of an immunoglobulin, fragments of the Fc<br>
portion of an immunoglobulin, or analogs thereof fused to a<br>
GLP-1 compound.<br>
The fusion proteins of the present invention can<br>
consist of single chain proteins or as multi-chain<br>
polypeptides. Two or more Fc fusion proteins can be<br>
produced such that they interact through disulfide bonds<br>
that naturally form between Fc regions. These multimers can<br>
be homogeneous with respect to the GLP-1 compound or they<br>
may contain different GLP-1 compounds fused at the N-<br>
terminus of the Fc portion of the fusion protein.<br>
Regardless of the final structure of the fusion<br>
protein, the Fc or Fc-like region must serve to prolong the<br>
in vivo plasma half-life of the GLP-1 compound fused at the<br>
N-terminus. Furthermore, the fused,GLP-1 compound must<br>
retain some biological activity. An increase in half-life<br>
can be demonstrated using the method described in Example 7<br>
wherein the half-life of the fusion protein is compared to<br>
the half-life of the GLP-1 compound alone. Biological<br>
activity can be determined by in vitro and in vivo methods<br>
known in the art. Representative biological assays are<br>
described in Examples 6, 8, and 9.<br>
Since the Fc region of IgG produced by proteolysis has<br>
the same in vivo half-life as the intact IgG molecule and<br>
Fab fragments are rapidly degraded, it is believed that the<br>
relevant sequence for prolonging half-life reside in the CH2<br>
and/or CH3 domains. Further, it has been shown in the<br>
literature that the catabolic rates of IgG variants that do<br>
not bind the high-affinity Fc receptor or C1q are<br>
indistinguishable from the rate of clearance of the parent<br>
wild-type antibody, indicating that the catabolic site is<br>
distinct from the sites involved in Fc receptor or C1q<br>
binding. [Wawrzynczak et al., (1992) Molecular Immunology<br>
29:221]. Site-directed mutagenesis studies using a xnurine<br>
IgG1 Fc region suggested that the site of the IgG1 Fc region<br>
that controls the catabolic rate is located at the CH2-CH3<br>
domain interface.<br>
Based on these studies, Fc regions can be modified at<br>
the catabolic site to optimize the half-life of the fusion<br>
proteins. It is preferable that the Fc region used for the<br>
heterologous fusion proteins of the present invention be<br>
derived from an IgGl or an IgG4 Fc region. It is even more<br>
preferable that the Fc region be IgG4 or derived from IgG4.<br>
Preferably the IgG Fc region contains both the CH2 and CH3<br>
regions including the hinge region.<br>
Heterologous albumin fusion proteins:<br>
The GLP-1 compounds described above can be fused<br>
directly or via a peptide linker to albumin or an analog,<br>
fragment, or derivative thereof.<br>
Generally the albumin proteins making up part of the<br>
fusion proteins of the present invention can be derived from<br>
albumin cloned from any species. However, human albumin and<br>
fragments and analogs thereof are preferred to reduce the<br>
risk of the fusion protein being immunogenic in humans.<br>
. Human serum albumin (HSA) consists of a single non-<br>
glycosylated polypeptide chain of 585 amino acids with a<br>
formula molecular weight of 66,500. The amino acid sequence<br>
of human HSA is shown in figure 2. [See Meloun, et al.<br>
(1975) FEBS Letters 58:136; Behrens, et al. (1975) Fed.<br>
Proc. 34:591; Lawn, et al. (1981) Nucleic Acids Research<br>
9:6102-6114; Minghetti, et al. (1986) J. Biol. Chem.<br>
261:6747]. A variety of polymorphic variants as well as<br>
analogs and fragments of albumin have been described. [See<br>
Weitkamp, et al., (1973) Ann. Hum. Genet. 37:219]. For<br>
example, in EP 322,094, the inventors disclose various<br>
shorter forms of HSA. Some of these fragments include<br>
HSA(1-373), HSA(l-388), HSAU-389), HSA(l-369) , and HSA(1-<br>
419) and fragments between 1-369 and 1-419. EP 399,666<br>
discloses albumin fragments that include HSA(1-177) and<br>
HSA(l-200) and fragments between HSA(1-177) and HSA(1-200).<br>
It is understood that the heterologous fusion proteins<br>
of the present invention include GLP-1 compounds that are<br>
coupled to any albumin protein including fragments, analogs,<br>
and derivatives wherein such fusion protein is biologically-<br>
active and has a longer plasma half-life than the GLP-1<br>
compound alone. Thus, the albumin portion of the fusion<br>
protein need not necessarily have a plasma half-life equal<br>
to that of native human albumin. Fragments, analogs, and<br>
derivatives are known or can be generated that have longer<br>
half-lives or have half-lives intermediate to that of native<br>
human albumin and the GLP-1 compound of interest.<br>
The heterologous fusion proteins of the present<br>
invention encompass proteins having conservative amino acid<br>
substitutions in the GLP-1 compound and/or the Fc or albumin<br>
portion of the fusion protein. A "conservative<br>
substitution" is the replacement of an amino acid with<br>
another amino acid that has the same net electronic charge<br>
and approximately the same size and shape. Amino acids with<br>
aliphatic or substituted aliphatic amino acid side chains<br>
have approximately the same size when the total number<br>
carbon and heteroatoms in their side chains differs by no<br>
more than about four. They have approximately the same<br>
shape when the number of branches in their side chains<br>
differs by no more than one. Amino acids with phenyl or<br>
substituted phenyl groups in their side chains are<br>
considered to have about the same size and shape. Except as<br>
otherwise specifically provided herein, conservative<br>
substitutions are preferably made with naturally occurring<br>
amino acids.<br>
However, the term "amino acid" is used herein in its<br>
broadest sense, and includes naturally occurring amino acids<br>
as well as non-naturally occurring amino acids, including<br>
amino acid analogs and derivatives. The latter includes<br>
molecules containing an amino acid moiety. One skilled in<br>
the art will recognize, in view of this broad definition,<br>
that reference herein to an amino acid includes, for<br>
example, naturally occurring proteogenic L-amino acids; D-<br>
amino acids; chemically modified amino acids such as amino<br>
acid analogs and derivatives; naturally occurring non-<br>
proteogenic amino acids such as norleucine, ß-alanine,<br>
ornithine, GABA, etc.; and chemically synthesized compounds<br>
having properties known in the art to be characteristic of<br>
amino acids. As used herein, the term "proteogenic"<br>
indicates that the amino acid can be incorporated into a<br>
peptide, polypeptide, or protein in a cell through a<br>
metabolic pathway.<br>
The incorporation of non-natural amino acids, including<br>
synthetic non-native amino acids, substituted amino acids,<br>
or one or more D-amino acids into the heterologous fusion<br>
proteins of the present invention can be advantageous in a<br>
number of different ways. D-amino acid-containing peptides,<br>
etc., exhibit increased stability in vitro or in vivo<br>
compared to L-amino acid-containing counterparts. Thus, the<br>
construction of peptides, etc., incorporating D-amino acids<br>
can be particularly useful when greater intracellular<br>
stability is desired or required. More specifically, D-<br>
peptides, etc., are resistant to endogenous peptidases and<br>
proteases, thereby providing improved bioavailability of the<br>
molecule, and prolonged lifetimes in vivo when such<br>
properties are desirable. Additionally, D-peptides, etc.,<br>
cannot be processed efficiently for major histocompatibility<br>
complex class II-restricted presentation to T helper cells,<br>
and are therefore, less likely to induce humoral immune<br>
responses in the whole organism.<br>
In addition to structure/function analyses of the<br>
various polypeptides encompassed by the present invention,<br>
there are numerous factors that can be considered when<br>
selecting amino acids for substitution. One factor that can<br>
be considered in making such changes is the hydropathic<br>
index of amino acids. The importance of the hydropathic<br>
amino acid index in conferring interactive biological<br>
function on a protein has been discussed by Kyte and<br>
Doolittle (1982, J. Mol. Biol., 157: 105-132). It is<br>
accepted that the relative hydropathic character of amino<br>
acids contributes to the secondary structure of the<br>
resultant protein. This, in turn, affects the interaction<br>
of the protein with molecules such as enzymes, substrates,<br>
receptors, ligands, DNA, antibodies, antigens, etc. Based<br>
on its hydrophobicity and charge characteristics, each amino<br>
acid has been assigned a hydropathic index as follows:<br>
isoleucine (+4.5); valine (+4.2); leucine (+3.8);<br>
phenylalanine (+2.8); cysteine/cystine (+2.5); methionine<br>
(+1.9); alanine (+1.8); glycine (-0.4); threonine (-0.7);<br>
serine (-0.8); tryptophan (-0.9); tyrosine (-1.3); proline<br>
(-1.6); histidine (-3.2);<br>
glutamate/glutamine/aspartate/asparagine (-3.5); lysine<br>
(-3.9); and arginine (-4.5).<br>
As is known in the art, certain amino acids in a<br>
peptide, polypeptide, or protein can be substituted for<br>
other amino acids having a similar hydropathic index or<br>
score and produce a resultant peptide, etc., having similar<br>
or even improved biological activity. In making such<br>
changes, it is preferable that amino acids having<br>
hydropathic indices within ±2 are substituted for one<br>
another. More preferred substitutions are those wherein the<br>
amino acids have hydropathic indices within ±1. Most<br>
preferred substitutions are those wherein the amino acids<br>
have hydropathic indices within ±0.5.<br>
Like amino acids can also be substituted on the basis<br>
of hydrophilicity. U.S. Patent No. 4,554,101 discloses that<br>
the greatest local average hydrophilicity of a protein, as<br>
governed by the hydrophilicity of its adjacent amino acids,<br>
correlates with a biological property of the protein. The<br>
following hydrophilicity values have been assigned to amino<br>
acids: arginine/lysine (+3.0); aspartate/glutamate (+3.0±1);<br>
serine (+0.3); asparagine/glutamine (+0.2); glycine (0);<br>
threonine (-0.4); proline (-0.5±1); alanine/histidine<br>
(-0.5); cysteine (-1.0); methionine (-1.3); valine (-1.5);<br>
leucine/isoleucine (-1.8); tyrosine (-2.3); phenylalanine<br>
(-2.5); and tryptophan (-3.4). Thus, one amino acid in a<br>
peptide, polypeptide, or protein can be substituted by<br>
another amino acid having a similar hydrophilicity score and<br>
still produce a resultant peptide, etc., having similar<br>
biological activity, i.e., still retaining correct<br>
biological function. In making such changes, amino acids<br>
having hydropathic indices within ±2 are preferably<br>
substituted for one another, those within ±1 are more<br>
preferred, and those within ±0.5 are most preferred.<br>
As outlined above, amino acid substitutions in the<br>
fusion proteins of the present invention can be based on the<br>
relative similarity of the amino acid side-chain<br>
substituents, for example, their hydrophobicity,<br>
hydrophilicity, charge, size, etc. Furthermore,<br>
substitutions can be made based on secondary structure<br>
propensity. For example, a helical amino acid can be<br>
replaced with an amino acid that would preserve the helical<br>
structure. Exemplary substitutions that take various of the<br>
foregoing characteristics into consideration in order to<br>
produce conservative amino acid changes resulting in silent<br>
changes within the present peptides, etc., can be selected<br>
from other members of the class to which the naturally<br>
occurring amino acid belongs. Amino acids can be divided<br>
into the following four groups: (1) acidic amino acids; (2)<br>
basic amino acids; (3) neutral polar amino acids; and (4)<br>
neutral non-polar amino acids.<br>
General methods for making the heterologous fusion proteins<br>
of the present invention.<br>
Although the heterologous fusion proteins of the<br>
present invention can be made by a variety of different<br>
methods, recombinant methods are preferred. For purposes of<br>
the present invention, as disclosed and claimed herein, the<br>
following general molecular biology terms and abbreviations<br>
are defined below. The terms and abbreviations used in this<br>
document have their normal meanings unless otherwise<br>
designated. For example, "°C" refers to degrees Celsius;<br>
"mmol" refers to millimole or millimoles; "mg" refers to<br>
milligrams; "µg" refers to micrograms; "ml or mL" refers to<br>
milliliters; and "µl or µL" refers to microliters. Amino<br>
acids abbreviations are as set forth in 37 C.F.R. § 1.822<br>
(b)(2&gt; (1994).<br>
"Base pair" or "bp" as used herein refers to DNA or<br>
RNA. The abbreviations A,C,G, and T correspond to the 5'-<br>
monophosphate forms of the deoxyribonucleosides<br>
(deoxy)adenosine, (deoxy)cytidine, (deoxy)guanosine, and<br>
thymidine, respectively, when they occur in DNA molecules.<br>
The abbreviations U,C,G, and A correspond to the 5'-<br>
monophosphate forms of the ribonucleosides uridine,<br>
cytidine, guanosine, and adenosine, respectively when they<br>
occur in RNA molecules. In double stranded DNA, base pair<br>
may refer to a partnership of A with T or C with G. In a<br>
DNA/RNA, heteroduplex base pair may refer to a partnership<br>
of A with U or C with G. (See the definition of<br>
"complementary", infra.)<br>
"Digestion" or "Restriction" of DNA refers to the<br>
catalytic cleavage of the DNA with a restriction enzyme that<br>
acts only at certain sequences in the DNA ("sequence-<br>
specific endonucleases"). The various restriction enzymes<br>
used herein are commercially available and their reaction<br>
conditions, cofactors, and other requirements were used as<br>
would be known to one of ordinary skill in the art.<br>
Appropriate buffers and substrate amounts for particular<br>
restriction enzymes are specified by the manufacturer or can<br>
be readily found in the literature.<br>
"Ligation" refers to the process of forming<br>
phosphodiester bonds between two double stranded nucleic<br>
acid fragments. Unless otherwise provided, ligation may be<br>
accomplished using known buffers and conditions with a DNA<br>
ligase, such as T4 DNA ligase.<br>
"Plasmid" refers to an extrachromosomal (usually) self-<br>
replicating genetic element. Plasmids are generally<br>
designated by a lower case "p" followed by letters and/or<br>
numbers. The starting plasmids herein are either<br>
commercially available, publicly available on an<br>
unrestricted basis, or can be constructed from available<br>
plasmids in accordance with published procedures. In<br>
addition, equivalent plasmids to those described are known<br>
in the art and will be apparent to the ordinarily skilled<br>
artisan.<br>
"Recombinant DNA cloning vector" as used herein refers<br>
to any autonomously replicating agent, including, but not<br>
limited to, plasmids and phages, comprising a DNA molecule<br>
to which one or more additional DNA segments can or have<br>
been added.<br>
"Recombinant DNA expression vector" as used herein<br>
refers to any recombinant DNA cloning vector in which a<br>
promoter to control transcription of the inserted DNA has<br>
been incorporated.<br>
"Transcription" refers to the process whereby<br>
information contained in a nucleotide sequence of DNA is<br>
transferred to a complementary RNA sequence.<br>
"Transfeetion" refers to the uptake of an expression<br>
vector by a host cell whether or not any coding sequences<br>
are, in fact, expressed. Numerous methods of transfection<br>
are known to the ordinarily skilled artisan, for example,<br>
calcium phosphate co-precipitation, liposome transfection,<br>
and electroporation. Successful transfection is generally<br>
recognized when any indication of the operation of this<br>
vector occurs within the host cell.<br>
"Transformation" refers to the introduction of DNA into<br>
an organism so that the DNA is replicable, either as an<br>
extrachromosomal element or by chromosomal integration.<br>
Methods of transforming bacterial and eukaryotic hosts are<br>
well known in the art, many of which methods, such as<br>
nuclear injection, protoplast fusion or by calcium treatment<br>
using calcium chloride are summarized in J. Sambrook, et<br>
al., Molecular Cloning: A Laboratory Manual, (1989).<br>
Generally, when introducing DNA into Yeast the term<br>
transformation is used as opposed to the term transfection.<br>
"Translation" as used herein refers to the process<br>
whereby the genetic information of messenger RNA (mRNA) is<br>
used to specify and direct the synthesis of a polypeptide<br>
chain.<br>
"Vector" refers to a nucleic acid compound used for the<br>
transfection and/or transformation of cells in gene<br>
manipulation bearing polynucleotide sequences corresponding<br>
to appropriate protein molecules which, when combined with<br>
appropriate control sequences, confers specific properties<br>
on the host cell to be transfected and/or transformed.<br>
Plasmids, viruses, and bacteriophage are suitable vectors.<br>
Artificial vectors are constructed by cutting and joining<br>
DNA molecules from different sources using restriction<br>
enzymes and ligases. The term "vector" as used herein<br>
includes Recombinant DNA cloning vectors and Recombinant DNA<br>
expression vectors.<br>
"Complementary" or "Complementarity", as used herein,<br>
refers to pairs of bases {purines and pyrimiddines) that<br>
associate through hydrogen bonding in a double stranded<br>
nucleic acid. The following base pairs are complementary:<br>
guanine and cytosine; adenine and thymine; and adenine and<br>
uracil.<br>
"Hybridization" as used herein refers to a process in<br>
which a strand of nucleic acid joins with a complementary<br>
strand through base pairing. The conditions employed in the<br>
hybridization of two non-identical, but very similar,<br>
complementary nucleic acids varies with the degree of<br>
complementarity of the two strands and the length of the<br>
strands. Such techniques and conditions are well known to<br>
practitioners in this field.<br>
"Isolated amino acid sequence" refers to any amino acid<br>
sequence, however, constructed or synthesized, which is<br>
locationally distinct from the naturally occurring sequence.<br>
"Isolated DNA compound" refers to any DNA sequence,<br>
however constructed or synthesized, which is locationally<br>
distinct from its natural location in genomic DNA.<br>
"Isolated nucleic acid compound" refers to any RNA or<br>
DNA sequence, however constructed or synthesized, which is<br>
locationally distinct from its natural location.<br>
"Primer" refers to a nucleic acid fragment which<br>
functions as an initiating substrate for enzymatic or<br>
synthetic elongation.<br>
"Promoter" refers to a DNA sequence which directs<br>
transcription of DNA to RNA.<br>
"Probe" refers to a nucleic acid compound or a<br>
fragment, thereof, which hybridizes with another nucleic<br>
acid compound.<br>
"Stringency" of hybridization reactions is readily<br>
determinable by one of ordinary skill in the art, and<br>
generally is an empirical calculation dependent upon probe<br>
length, washing temperature, and salt concentration. In<br>
general, longer probes require higher temperatures for proper<br>
annealing, while short probes need lower temperatures.<br>
Hybridization generally depends on the ability of denatured<br>
DNA to re-anneal when complementary strands are present in an<br>
environment below their melting temperature. The higher the<br>
degree of desired homology between the probe and hybridizable<br>
sequence, the higher the relative temperature that can be<br>
used. As a result, it follows that higher relative<br>
temperatures would tend to make the reactions more stringent,<br>
while lower temperatures less so. For additional details and<br>
explanation of stringency of hybridization reactions, see<br>
Ausubel et al., Current Protocols in Molecular Biology, Wiley<br>
Interscience Publishers, 1995.<br>
"Stringent conditions" or "high stringency conditions",<br>
as defined herein, may be identified by those that (1) employ<br>
low ionic strength and high temperature for washing, for<br>
example, 15 mM sodium chloride/1.5 mM sodium citrate/0.1%<br>
sodium dodecyl sulfate at 50°C; (2) employ during<br>
hybridization a denaturing agent, such as formamide, for<br>
example, 50% (v/v) formamide with 0.1% bovine serum<br>
albumin/0.1% ficoll/0.1% polyvinylpyrrolidone/50 mM sodium<br>
phosphate buffer at pH 6.5 with 750 mM sodium chloride/75 mM<br>
sodium citrate at 42°C; or (3) employ 50% formamide, 5X SSC<br>
(750 mM sodium chloride, 75 mM sodium citrate), 50 mM sodium<br>
phosphate (pH 6.8), 0.1% sodium pyrophosphate, 5X Denhardt's<br>
solution, sonicated salmon sperm DNA (50 µg/ml) , 0.1% SDS, and<br>
10% dextran sulfate at 42°C with washes at 42°C in 0.2X SSC<br>
(30 mM sodium chloride/3 mM sodium citrate) and 50% formamide<br>
at 55°C, followed by a high-stringency wash consisting of 0.1X<br>
SSC containing EDTA at 55°C.<br>
"Moderately stringent conditions" may be identified as<br>
described by Sambrook et al. [Molecular Cloning: A Laboratory<br>
Manual, New York: Cold Spring Harbor Press, (1989)1, and<br>
include the use of washing solution and hybridization<br>
conditions (e.g., temperature, ionic strength, and %SDS) less<br>
stringent than those described above. An example of<br>
moderately stringent conditions is overnight incubation at<br>
37°C in a solution comprising: 20% formamide, 5X SSC (750 mM<br>
sodium chloride, 75 mM sodium citrate) , 50 mM sodium phosphate<br>
at pH 7.6, 5X Denhardt's solution, 10% dextran sulfate, and 20<br>
mg/mL denatured sheared salmon sperm DNA, followed by washing<br>
the filters in 1X SSC at about 37-50°C The skilled artisan<br>
will recognize how to adjust the temperature, ionic strength,<br>
etc., as necessary to accommodate factors such as probe length<br>
and the like.<br>
"PCR" refers to the widely-known polymerase chain<br>
reaction employing a thermally-stable DNA polymerase.<br>
"Leader sequence" refers to a sequence of amino acids<br>
which can be enzymatically or chemically removed to produce<br>
the desired polypeptide of interest.<br>
"Secretion signal sequence" refers to a sequence of<br>
amino acids generally present at the N-terminal region of a<br>
larger polypeptide functioning to initiate association of<br>
that polypeptide with the cell membrane compartments like<br>
endoplasmic reticulum and secretion of that polypeptide<br>
through the plasma membrane.<br>
Construction of DNA encoding the heterologous fusion<br>
proteins of the present invention:<br>
Wild-type albumin and Immunoglobulin proteins can be<br>
obtained from a variety of sources. For example, these<br>
proteins can be obtained from a cDNA library prepared from<br>
tissue or cells which express the mRNA of interest at a<br>
detectable level. Libraries can be screened with probes<br>
designed using the published DNA or protein sequence for the<br>
particular protein of interest. For example, immunoglobulin<br>
light or heavy chain constant regions are described in<br>
Adams, et al. (1980) Biochemistry 19:2711-2719; Goughet, et<br>
al. (1980) Biochemistry 19:2702-2710; Dolby, et al. (1980)<br>
Proc. Natl. Acad. Sci. USA 77:6027-6031; Rice et al. (1982)<br>
Proc. Natl. Acad. Sci. USA 79:7862-7862; Falkner, et al.<br>
(1982) Nature 298:286-288; and Morrison, et al. (1984) Ann.<br>
Rev. Immunol. 2:239-256. Some references disclosing albumin<br>
protein and DNA sequences include Meloun, et al. (1975) FEBS<br>
Letters 58:136; Behrens, et al. (1975) Fed. Proc. 34:591;<br>
Lawn, et al. (1981) Nucleic Acids Research 9:6102-6114; and<br>
Minghetti, et al. (1986) J. Biol. Chem. 261:6747<br>
Screening a cDNA or genomic library with the selected<br>
probe may be conducted using standard procedures, such as<br>
described in Sambrook et al., Molecular Cloning: A<br>
Laboratory Manual, Cold Spring Harbor Laboratory Press, NY<br>
(1989) . An alternative means to isolate a gene encoding an<br>
albumin or immunoglobulin protein is to use PCR methodology<br>
[Sambrook et al., supra; Dieffenbach et al., PCR Primer: A<br>
Laboratory Manual, Cold Spring Harbor Laboratory Press, NY<br>
(1995)] . PCR primers can be designed based on published<br>
sequences.<br>
Generally the full-length wild-type sequences cloned<br>
from a particular species can serve as a template to create<br>
analogs, fragments, and derivatives that retain the ability<br>
to confer a longer plasma half-life on the GLP-1 compound<br>
that is part of the fusion protein. It is preferred that<br>
the Fc and albumin portions of the heterologous fusion<br>
proteins of the present invention be derived from the native<br>
human sequence in order to reduce the risk of potential<br>
imntunogenicity of the fusion protein in humans.<br>
In particular, it is preferred that the immunoglobulin<br>
portion of a fusion protein encompassed by the present<br>
invention contain only an Fc fragment of the immunoglobulin.<br>
Depending on whether particular effect or functions are<br>
desired and the structural characteristics of the fusion<br>
protein, an Fc fragment may contain the hinge region along<br>
with the CH2 and CH3 domains or some other combination<br>
thereof. These Fc fragments can be generated using PCR<br>
techniques with primers designed to hybridize to sequences<br>
corresponding to the desired ends of the fragment.<br>
Similarly, if fragments of albumin are desired, PCR primers<br>
can be designed which are complementary to internal albumin<br>
sequences. PCR primers can also be designed to create<br>
restriction enzyme sites to facilitate cloning into<br>
expression vectors.<br>
DNA encoding the GLP-1 compounds of the present<br>
invention can be made by a variety of different methods<br>
including cloning methods like those described above as well<br>
as chemically synthesized DNA. Chemical synthesis may be<br>
attractive given the short length of the encoded peptide.<br>
The amino acid sequence for GLP-1 has been published as well<br>
as the sequence of the preproglucagon gene. [Lopez, et al.<br>
(1983) Proc. Natl. Acad. Sci., USA 80:5485-5489; Bell, et<br>
al. (1983) Nature, 302:716-718; Heinrich, G., et al. (1984)<br>
Endocrinol, 115:2176-2181; Ghiglione, M., et al. 91984)<br>
Diabetologia 27:599-600]. Thus, primers can be designed to<br>
PCR native GLP-1 compounds and fragments thereof.<br>
The gene encoding a fusion protein can then be<br>
constructed by ligating DNA encoding a GLP-1 compound in-<br>
frame to DNA encoding an albumin or Fc protein. The gene<br>
encoding the GLP-1 compound and the gene encoding the<br>
albumin or Fc protein can also be joined in-frame via DNA<br>
encoding a linker peptide.<br>
The in vivo function and stability of the heterologous<br>
fusion proteins of the present invention can be optimized by<br>
adding small peptide linkers to prevent potentially unwanted<br>
domain interactions. Although these linkers can potentially<br>
be any length and consist of any combination of amino acids,<br>
it is preferred that the length be no longer than necessary<br>
to prevent unwanted domain interactions and/or optimize<br>
biological activity and/or stability. Generally, the<br>
linkers should not contain amino acids with extremely bulky<br>
side chains or amino acids likely to introduce significant<br>
secondary structure. It is preferred that the linker be<br>
serine-glycine rich and be less than 30 amino acids in<br>
length. It is more preferred that the linker be no more<br>
than 20 amino acids in length. It is even more preferred<br>
that the linker be no more than 15 amino acids in length. A<br>
preferred linker contains repeats of the sequence Gly-Gly-<br>
Gly-Gly-Ser. It is preferred that there be between 2 and 6<br>
repeats of this sequence. It is even more preferred that<br>
there be between 3 and 4 repeats of this sequence.<br>
The DNA encoding wild-type GLP-1, albumin, and Fc<br>
polypeptides and fragments thereof can be mutated either<br>
before ligation or in the context of a cDNA encoding an<br>
entire fusion protein. A variety of mutagenesis techniques<br>
are well known in the art. For example, a mutagenic PCR<br>
method utilizes strand overlap extension to create specific<br>
base mutations for the purposes of changing a specific amino<br>
acid sequence in the corresponding protein. This PCR<br>
mutagenesis requires the use of four primers, two in the<br>
forward orientation (primers A and C) and two in the reverse<br>
orientation (primers B and D) . A mutated gene is amplified<br>
from the wild-type template in two different stages. The<br>
first reaction amplifies the gene in halves by performing an<br>
A to B reaction and a separate C to D reaction wherein the B<br>
and C primers target the area of the gene to be mutated.<br>
When aligning these primers With the target area, they<br>
contain mismatches for the bases that are targeted to be<br>
changed. Once the A to B and C to D reactions are complete,<br>
the reaction products are isolated and mixed for use as the<br>
template for the A to D reaction. This reaction then yields<br>
the full, mutated product.<br>
Once a gene encoding an entire fusion protein is<br>
produced it can be cloned into an appropriate expression<br>
vector. Specific strategies that can be employed to make<br>
the GLP-1 fusion proteins of the present invention are<br>
described in example 1.<br>
General methods to recombinantly express the heterologous<br>
fusion proteins of the present invention:<br>
Host cells are transfected or transformed with expression<br>
or cloning vectors described herein for heterologous fusion<br>
protein production and cultured in conventional nutrient media<br>
modified as appropriate for inducing promoters, selecting<br>
transformants, or amplifying the genes encoding the desired<br>
sequences. The culture conditions, such as media, temperature,<br>
pH and the like, can be selected by the skilled artisan without<br>
undue experimentation. In general, principles, protocols, and<br>
practical techniques for maximizing the productivity of cell<br>
cultures can be found in Mammalian Cell Biotechnology: A<br>
Practical Approach, M. Butler, ed. (IRL Press, 1991) and<br>
Sambrook, et al., supra. Methods of transfection are known to<br>
the ordinarily skilled artisan, for example, CaPO4 and<br>
electroporation. General aspects of mammalian cell host system<br>
transformations have been described in U.S. Patent Mo.<br>
4,399,216. Transformations into yeast are typically carried<br>
out according to the method of van Solingen et al., J Bact.<br>
130(2): 946-7 (1977) and Hsiao et al., Proc. Natl. Acad. Sci.<br>
USA 76(8): 3829-33 (1979). However, other methods for<br>
introducing DNA into cells, such as by nuclear microinjection,<br>
electroporation, bacterial protoplast fusion with intact cells,<br>
or polycations, e.g., polybrene or polyomithine, may also be<br>
used. For various techniques for transforming mammalian cells,<br>
see Keown, et al., Methods in Enzymology 185: 527-37 (1990) and<br>
Mansour, et al., Nature 336(6197): 348-52 (1988).<br>
Suitable host cells for cloning or expressing the nucleic<br>
acid (e.g., DNA) in the vectors herein include prokaryote,<br>
yeast, or higher eukaryote cells. Suitable prokaryotes<br>
include but are not limited to eubacteria, such as Gram-<br>
negative or Gram-positive organisms, for example,<br>
Enterobacteriacea such as E. coli. Various E. coli strains<br>
are publicly available, such as E. coli K12 strain MM294 (ATCC<br>
3 1.446); E. coli X1 776 (ATCC 3 1.537); E. coli strain W3 110<br>
(ATCC 27.325) and K5 772 (ATCC 53.635). Other suitable<br>
prokaryotic host cells include Enterobacteriaceae such as<br>
Escherichia, e.g.. E. cola, Enterobacter, Erwinia, Klebisella,<br>
Proteus, Salmonella, e.g., Salmonella typhimurium, Serratia,<br>
e.g., Serratia marcescans, and Shigeila, as well as Bacilli<br>
such as B. subtilis and B. lichentformis (e.g., B.<br>
licheniformis 4 1 P disclosed in DD266,7 10, published 12<br>
April 1989), Pseudomonas such as p. aeruginosa, and<br>
Streptomyces. These examples are illustrative rather than<br>
limiting. Strain W3110 is one particularly preferred host or<br>
parent host because it is a common host strain for recombinant<br>
DNA product fermentations. Preferably, the host cell secretes<br>
minimal amounts of proteolytic enzymes. For example, strain<br>
W3 110 may be modified to effect a genetic mutation in the<br>
genes encoding proteins endogenous to the host, with examples<br>
of such hosts including E. coli W3110 strain 1A2, which has<br>
the complete genotype ronA; E. coli W3 110 strain 9E4, which<br>
has the complete genotype ton4 ptr3; E. coli W3110 strain 27C7<br>
(ATCC 55,244), which has the complete genotype tonA, ptr3 phoA<br>
E15 (argF-lac) 169 degP ompT /can'; E. coli W3110 strain 40B4,<br>
which is strain 37D6 with a non-kanamycin resistant degP<br>
deletion mutation; and an E. coli strain having mutant<br>
periplasmic protease disclosed in U.S. Patent No. 4,946,783<br>
issued 7 August 1990. Alternatively, in vivo methods of<br>
cloning, e.g., PCR or other nucleic acid polymerase reactions,<br>
are suitable.<br>
In addition to prokaryotes, eukaryotic microbes such as<br>
filamentous fungi or yeast are suitable cloning or expression<br>
hosts for fusion protein vectors. Saccharomyces cerevisiae is<br>
a commonly used lower eukaryotic host microorganism. Others<br>
include Schizosaccharomyces pombe [Beach and Nurse, Nature<br>
290: 140-3 (1981); EP 139,383 published 2 May 1995];<br>
Muyveromyces hosts [U.S. Patent No. 4,943,529; Fleer, et al.,<br>
Bio/Technology 9(10): 968-75 (1991)] such as, e.g., K lactis<br>
(MW98-8C, CBS683, CBS4574) [de Louvencourt et al., J.<br>
Bacteriol. 154(2): 737-42 (1983)]; K. fiagilis (ATCC 12,424),<br>
K. bulgaricus (ATCC 16,045), K wickeramii (ATCC 24,178), K<br>
waltii (ATCC 56,500), K. drosophilarum (ATCC 36.906) [Van den<br>
Berg et al., Bio/Technology 8(2): 135-9 (1990)]; K.<br>
thermotoierans, and K. marxianus; yarrowia (EP 402,226);<br>
Pichia pastoris (EP 183,070) [Sreekrishna et al., J. Basic<br>
Microbiol. 28(4): 265-78 (1988)]; Candid; Trichoderma reesia<br>
(EP 244,234); Neurospora crassa [Case, et al., Proc. Natl.<br>
Acad Sex. USA 76(10): 5259-63 (1979)]; Schwanniomyces such as<br>
Schwanniomyces occidentulis (EP 394,538 published 31 October<br>
1990); and filamentous fungi such as, e.g., Neurospora,<br>
Penicillium, Tolypocladium (WO 91/00357 published 10 January<br>
1991), and Aspergillus hosts such as A. nidulans [Ballance et<br>
al., Biochem. Biophys. Res. Comm. 112(1): 284-9 (1983)]; .<br>
Tilburn, et al., Gene 26(2-3): 205-21 (1983); Yelton, et al.<br>
Proc. Natl. Acad. Sci. USA 81(5): 1470-4 (1984)] and A. niger<br>
[Kelly and Hynes, EMBO J. 4(2): 475-9 (1985)]. Methylotropic<br>
yeasts are selected from the genera consisting of Hansenula,<br>
Candida, Kloeckera, Pichia, Saccharomyces, Torulopsis, and<br>
Rhodotoruia. A list of specific species that are exemplary of<br>
this class of yeast may be found in C. Antony, The<br>
Biochemistry of Methylotrophs 269 (1982) .<br>
Suitable host cells for the expression of the fusion<br>
proteins of the present invention are derived from<br>
multicellular organisms. Examples of invertebrate cells<br>
include insect cells such as Drosophila S2 and Spodoptera Sp,<br>
Spodoptera high5 as well as plant cells. Examples of useful<br>
mammalian host cell lines include Chinese hamster ovary (CHO)<br>
and COS cells. More specific examples include monkey kidney<br>
CV1 line transformed by SV40 (COS-7, ATCC CRL 1651); human<br>
embryonic kidney line [293 or 293 cells subcloned for growth<br>
in suspension culture, Graham, et al., J. Gen Virol., 36(1):<br>
59-74 (1977)]; Chinese hamster ovary cells/-DHFR [CHO, Urlaub<br>
and Chasin, Proc. Natl. Acad. Sci. USA, 77(7): 4216-20<br>
(1980)]; mouse sertoli cells [TM4, Mather, Biol. Reprod.<br>
23(l):243-52 (1980)]; human lung cells (W138. ATCC CCL 75);<br>
human liver cells (Hep G2, HB 8065); and mouse mammary tumor<br>
(MMT 060562, ATCC CCL51) . The selection of the appropriate<br>
host cell is deemed to be within the skill in the art.<br>
The fusion proteins of the present invention may be<br>
recombinantly produced directly, or as a protein having a<br>
signal sequence or other additional sequences which create a<br>
specific cleavage site at the N-terminus of the mature fusion<br>
protein. In general, the signal sequence may be a component<br>
of the vector, or it may be a part of the fusion protein-<br>
encoding DNA that is inserted into the vector. The signal<br>
sequence may be a prokaryotic signal sequence selected, for<br>
example, from the group of the alkaline phosphatase,<br>
penicillinase, 1pp, or heat-stable enterotoxin II leaders.<br>
For yeast secretion the signal sequence may be, e.g., the<br>
yeast invertase leader, alpha factor leader (including<br>
Saccharomyces and Kluyveromyces cc-factor leaders, the latter<br>
described in U.S. Patent No. 5,010,182), or acid phosphatase<br>
leader, the C. albicans glucoamylase leader (EP 362,179), or<br>
the signal described in WO 90/13646. In mammalian cell<br>
expression, mammalian signal sequences may be used to direct<br>
secretion of the protein, such as signal sequences from<br>
secreted polypeptides of the same or related species as well<br>
as viral secretory leaders.<br>
Both expression and cloning vectors contain a nucleic<br>
acid sequence that enables the vector to replicate in one or<br>
more selected host cells. Such sequences are well known for a<br>
variety of bacteria, yeast, and viruses. The origin of<br>
replication from the plasmid pBR322 is suitable for most Gram-<br>
negative bacteria, the 2u plasmid origin is suitable for<br>
yeast, and various viral origins (SV40, polyoma, adenovirus,<br>
VSV or BPV) are useful for cloning vectors in mammalian cells.<br>
Expression and cloning vectors will typically contain a<br>
selection gene, also termed a selectable marker. Typical<br>
selection genes encode proteins that (a) confer resistance to<br>
antibiotics or other toxins, e.g., ampicillin, neomycin,<br>
methotrexate, or tetracycline, (b) complement autotrophic<br>
deficiencies, or (c) supply critical nutrients not available<br>
from complex media, e.g., the gene encoding D-alanine racemase<br>
for Bacilli.<br>
An example of suitable selectable markers for mammalian<br>
cells are those that enable the identification of cells<br>
competent to take up the fusion protein-encoding nucleic acid,<br>
such as DHFR or thymidine kinase. An appropriate host cell<br>
when wild-type DHFR is employed is the CHO cell line deficient<br>
in DHFR activity, prepared and propagated as described [Urlaub<br>
and Chasin, Proc. Natl. Acad. Sci. USA, 77(7): 4216-20<br>
(1980)]. A suitable selection gene for use in yeast is the<br>
trpl gene present in the yeast plasmid YRp7 [Stinchcomb, et<br>
al., Nature 282(5734): 39-43 (1979); Kingsman, et al., Gene<br>
7(2): 141-52 (1979); Tschumper, et al., Gene 10(2): 157-66<br>
(1980)]. The trpl gene provides a selection marker for a<br>
mutant strain of yeast lacking the ability to grow in<br>
tryptophan, for example, ATCC No. 44076 or PEPC1 [Jones,<br>
Genetics 85: 23-33 (1977)].<br>
Expression and cloning vectors usually contain a promoter<br>
operably linked to the fusion protein-encoding nucleic acid<br>
sequence to direct mRNA synthesis. Promoters recognized by a<br>
variety of potential host cells are well known. Promoters<br>
suitable for use with prokaryotic hosts include the ß-<br>
lactamase and lactose promoter systems [Chang, et al., Nature<br>
275(5681): 617-24 (1978); Goeddel, et al., Nature 281(5732):<br>
544-8 (1979)], alkaline phosphatase, a tryptophan (up)<br>
promoter system [Goeddel, Nucleic Acids Res. 8(18): 4057-74<br>
(1980); EP 36,776 published 30 September 1981], and hybrid<br>
promoters such as the tat promoter [deBoer, et al., Proc.<br>
Natl. Acad. Sci. USA 80(1): 21-5 (1983)]. Promoters for use<br>
in bacterial systems also will contain a Shine-Dalgarno (S.D.)<br>
sequence operably linked to the DNA encoding the fusion<br>
protein.<br>
Examples of suitable promoting sequences for use with<br>
yeast hosts include the promoters for 3-phosphoglycerate<br>
kinase [Hitzeman, et al., J. Biol. Chem. 255(24): 12073-80<br>
(1980)] or other glycolytic enzymes [Hess et al., J. Adv.<br>
Enzyme Reg. 7: 149 (1968); Holland, Biochemistry 17(23) : 4900-<br>
7 (1978)], such as enolase, glyceraldehyde-3-phosphate<br>
dehydrogenase, hexokinase, pyruvate decarboxylase,<br>
phosphofructokinase, glucose-6-phosphate isomerase, 3-<br>
phosphoglycerate mutase, pyruvate kinase, triosephosphate<br>
isomerase, phosphoglucose isomerase, and glucokinase.<br>
Other yeast promoters, which are inducible promoters<br>
having the additional advantage of transcription controlled by<br>
growth conditions, are the promoter regions for alcohol<br>
dehydrogenase 2, isocytochrome C, acid phosphatase,<br>
degradative enzymes associated with nitrogen metabolism,<br>
metallothionein, glyceraldehyde-3-phosphate dehydrogenase, and<br>
enzymes responsible for maltose and galactose utilization.<br>
Suitable vectors and promoters for use in yeast expression are<br>
further described in EP 73,657. Transcription of fusion<br>
protein-encoding mRNA from vectors in mammalian host cells may<br>
be controlled, for example, by promoters obtained from the<br>
genomes of viruses such as polyoma virus, fowlpox virus,<br>
adenovirus (such as Adenovirus 2) , bovine papilloma virus,<br>
avian sarcoma virus, cytomegalovirus, a retrovirus, hepatitis-<br>
B virus and Simian Virus 40 (SV40) , from heterologous<br>
mammalian promoters, e.g., the actin promoter or an<br>
immunoglobulin promoter, and from heat-shock promoters,<br>
provided such promoters are compatible with the host cell<br>
systems.<br>
Transcription of a polynucleotide encoding a fusion<br>
protein by higher eukaryotes may be increased by inserting an<br>
enhancer sequence into the vector. Enhancers are cis-acting<br>
elements of DNA, usually about from 10 to 300 bp, that act on<br>
a promoter to increase its transcription. Many enhancer<br>
sequences are now known from mammalian genes (globin,<br>
elastase, albumin, a-ketoprotein, and insulin). Typically,<br>
however, one will use an enhancer from a eukaryotic cell<br>
virus. Examples include the SV40 enhancer on the late side of<br>
the replication origin (bp 100-270), the cytomegalovirus early<br>
promoter enhancer, the polyoma enhancer on the late side of<br>
the replication origin, and adenovirus enhancers. The<br>
enhancer may be spliced into the vector at a position 5' or 3'<br>
to the fusion protein coding sequence but is preferably<br>
located at a site 5' from the promoter.<br>
Expression vectors used in eukaryotic host cells (yeast,<br>
fungi, insect, plant, animal, human, or nucleated cells from<br>
other multicellular organisms) will also contain sequences<br>
necessary for the termination of transcription and for<br>
stabilizing the mRNA. Such sequences are commonly available<br>
from the 5' and occasionally 3' untranslated regions of<br>
eukaryotic or viral DNAs or cDNAs. These regions contain<br>
nucleotide segments transcribed as polyadenylated fragments in<br>
the untranslated portion of the mRNA encoding the fusion<br>
protein.<br>
Various forms of a fusion protein may be recovered from<br>
culture medium or from host cell lysates. If membrane-bound,<br>
it can be released from the membrane using a suitable<br>
detergent solution (e.g., Triton-X 100) or by enzymatic<br>
cleavage. Cells employed in expression of a fusion protein<br>
can be disrupted by various physical or chemical means, such<br>
as freeze-thaw cycling, sonication, mechanical disruption, or<br>
cell lysing agents.<br>
Purification of the heterologous fusion proteins of the<br>
present invention:<br>
Once the heterologous fusion proteins of the present<br>
invention are expressed in the appropriate host cell, the<br>
analogs can be isolated and purified. The following<br>
procedures are exemplary of suitable purification procedures:<br>
fractionation on carboxymethyl cellulose; gel filtration such<br>
as Sephadex G-75; anion exchange resin such as DEAE or Mono-Q;<br>
cation exchange such as CM or Mono-S; protein A sepharose to<br>
remove contaminants such as IgG; metal chelating columns to<br>
bind epitope-tagged forms of the polypeptide; reversed-phase<br>
HPLC; chromatofocusing; silica gel; ethanol precipitation; and<br>
ammonium sulfate precipitation.<br>
Various methods of protein purification may be employed<br>
and such methods are known in the art and described, for<br>
example, in Deutscher, Methods in Enzymology 182: 83-9 (1990)<br>
and Scopes, Protein Purification: Principles and Practice,<br>
Springer-Verlag, NY (1982). The purification step(s) selected<br>
will depend on the nature of the production process used and<br>
the particular fusion protein produced. For example, fusion<br>
proteins comprising an Fc fragment can be effectively purified<br>
using a Protein A or Protein G affinity matix. Low or high pH<br>
buffers can be used to elute the fusion protein from the<br>
affinity matrix. Mild elution conditions will aid in<br>
preventing irreversible denaturation of the fusion protein.<br>
Imidazole-containing buffers can also be used. Example 3<br>
describes some successful purification protocols for the<br>
fusion proteins of the present invention.<br>
Characterization of the heterologous fusion proteins of the<br>
present invention:<br>
Numerous methods exist to characterize the fusion<br>
proteins of the present invention. Some of these methods<br>
include: SDS-PAGE coupled with protein staining methods or<br>
immunoblotting using anti-IgG or anti-HSA antibodies. Other<br>
methods include matrix assisted laser desporption/ionization-<br>
mass spectrometry (MALDI-MS), liquid chromatography/mass<br>
spectrometry, isoelectric focusing, analytical anion exchange,<br>
chromatofocussing, and circular dichroism to name a few. A<br>
representative number of heterologous fusion proteins were<br>
characterized using SDS-PAGE coupled with immunoblotting as<br>
well as mass spectrometry (See examples 4 and 5 and figures 3<br>
and 4).<br>
For example table 3 (see example 5) illustrates the<br>
calculated molecular mass for a representative number of<br>
fusion proteins as well as the mass as determined by mass<br>
spectrometry. In addition, Figures 3 and 4 illustrate<br>
molecular weights of a representative number of fusion<br>
proteins as determined by SDS PAGE- All heterologous fusion<br>
proteins tested were expressed and secreted transiently. In<br>
addition, the IgK signal sequence was cleaved to yield<br>
proteins with the correct N-terminus.<br>
Further, table 3 illustrates that in some instances the<br>
mass determined by mass spectrometry is greater than expected.<br>
This is the result of glycosylation of the Fc portion and the<br>
C terminal extension. Enzymatic digestion of the fusion<br>
proteins followed by reversed-phase HPLC and mass spectrometry<br>
can identify peptide fractions that contain sugar moieties.<br>
These fractions can then be N-terminal amino acid sequenced to<br>
identify the potential glycosylation site. For example,<br>
characterization of Exendin-4-Fc (SEQ ID NO: 29) shows that<br>
the serine at position 39 and threonine at position 50 are O-<br>
linked glycosylated and the asaparagine at position 122 is N-<br>
linked glycosylated.<br>
A representative number of GLP-1 fusion proteins were<br>
also tested for activity. Numerous methods exist to detect<br>
GLP-1 activity in vitro and in vivo (see examples 6, 7, 8,and<br>
9). Table 4 (example 6) illustrates GLP-1 receptor activity<br>
associated with several GLP-1 fusions. The numbers are<br>
relative to the activity associated with Val8 -GLP-1(7-37)OH.<br>
All fusion proteins tested had GLP-1 receptor activity. A low<br>
level of in vitro activity is not necessarily indicative of a<br>
weak effect in vivo. Because of the substantial increase in<br>
the half-life of these fusion proteins, weak in vitro activity<br>
is not generally a predictor of weak in vivo activity. Figure<br>
7 and example 7 illustrate the prolonged half-life associated<br>
with the fusion proteins of the present invention. For<br>
example, Val8-GLP-1-Fc had a half-life of approximately 45<br>
hours in monkeys, Val8-GLP-1-HSA had a half-life of about 87<br>
hours in monkeys, Gly8-Glu22-GLP-1-CEx-Linker-IgG1 had a half-<br>
life after IV administration of approximately 55 hours in<br>
dogs, and Gly8-Glu32-GLP-1-CEx-Linker-IgGl had a half-life<br>
after SC administration of approximately 38 hours in dogs.<br>
Compositions of the invention:<br>
Physical stability is also an essential feature for<br>
therapeutic protein formulations. GLP-1 compounds have been<br>
especially difficult to manufacture and formulate due to<br>
structural changes that occur during processing. For example,<br>
some GLP-1 compounds have a general tendency to aggregate. In<br>
addition, it has been shown that some GLP-1 compounds convert<br>
from a soluble and active a-helix form to an insoluble and<br>
potentially inactive ß-sheet form. The fusion of GLP-1<br>
compounds to large proteins such as the Fc region of an IgG or<br>
albumin not only acts to increase the half-life of the GLP-1<br>
compound, but also contributes to the physical and<br>
conformational stability of the GLP-1 compound. For example,<br>
Val8-GLP-1-Linker-HSA in PBS is stable at 37°C out to about 30<br>
days.<br>
The heterologous fusion proteins of the present invention<br>
may be formulated with one or more excipients. The active<br>
fusion proteins of the present invention may be combined with<br>
a pharmaceutically acceptable buffer, and the pH adjusted to<br>
provide acceptable stability, and a pH acceptable for<br>
administration such as parenteral administration.<br>
Optionally, one or more pharmaceutically-acceptable anti-<br>
microbial agents may be added. Meta-cresol and phenol are<br>
preferred pharmaceutically-acceptable microbial agents. One<br>
or more pharmaceutically-acceptable salts may be added to<br>
adjust the ionic strength or tonicity. One or more excipients<br>
may be added to further adjust the isotonicity of the<br>
formulation. Glycerin is an example of an isotonicity-<br>
adjusting excipient. Pharmaceutically acceptable means<br>
suitable for administration to a human or other animal and<br>
thus, does not contain toxic elements or undesirable<br>
contaminants and does not interfere with the activity of the<br>
active compounds therein.<br>
A pharmaceutically-acceptable salt form of the<br>
heterologous fusion proteins of the present invention may be<br>
used in the present invention. Acids commonly employed to<br>
form acid addition salts are inorganic acids such as<br>
hydrochloric acid, hydrobromic acid, hydriodic acid, sulfuric<br>
acid, phosphoric acid, and the like, and organic acids such as<br>
p-toluenesulfonic acid, methanesulfonic acid, oxalic acid, p-<br>
bromophenyl-sulfonic acid, carbonic acid, succinic acid,<br>
citric acid, benzoic acid, acetic acid, and the like.<br>
Preferred acid addition salts are those formed with mineral<br>
acids such as hydrochloric acid and hydrobromic acid.<br>
Base addition salts include those derived from inorganic<br>
bases, such as ammonium or alkali or alkaline earth metal<br>
hydroxides, carbonates, bicarbonates, and the like. Such<br>
bases useful in preparing the salts of this invention thus<br>
include sodium hydroxide, potassium hydroxide, ammonium<br>
hydroxide, potassium carbonate, and the like.<br>
Administration of Compositions:<br>
Administration may be via any route known to be effective<br>
by the physician of ordinary skill. Peripheral, parenteral is<br>
one such method. Parenteral administration is commonly<br>
understood in the medical literature as the injection of a<br>
dosage form into the body by a sterile syringe or some other<br>
mechanical device such as an infusion pump. Peripheral<br>
parenteral routes can include intravenous, intramuscular,<br>
subcutaneous, and intraperitoneal routes of administration.<br>
The heterologous fusion proteins of the present invention<br>
may also be amenable to administration by oral, rectal, nasal,<br>
or lower respiratory routes, which are non-parenteral routes.<br>
Of these non-parenteral routes, the lower respiratory route<br>
and the oral route are preferred.<br>
The fusion proteins of the present invention can be used<br>
to treat a wide variety of diseases and conditions. The<br>
fusion proteins of the present invention primarily exert their<br>
biological effects by acting at a receptor referred to as the<br>
"GLP-1 receptor." Subjects with diseases and/or conditions<br>
that respond favorably to GLP-1 receptor stimulation or to the<br>
administration of GLP-1 compounds can therefore be treated<br>
with the GLP-1 fusion proteins of the present invention.<br>
These subjects are said to "be in need of treatment with GLP-1<br>
compounds" or "in need of GLP-1 receptor stimulation".<br>
Included are subjects with non-insulin dependent diabetes,<br>
insulin dependent diabetes, stroke (see WO 00/16797),<br>
myocardial infarction (see WO 98/08531), obesity (see WO<br>
98/19698), catabolic changes after surgery (see U.S. Patent<br>
No. 6,006,753), functional dyspepsia and irritable bowel<br>
syndrome (see WO 99/64060) . Also included are subjects<br>
requiring prophylactic treatment with a GLP-1 compound, e.g.,<br>
subjects at risk for developing non-insulin dependent diabetes<br>
(see WO 00/07617). Subjects with impaired glucose tolerance<br>
or impaired fasting glucose, subjects whose body weight is<br>
about 25% above normal body weight for the subject's height<br>
and body build, subjects with a partial pancreatectomy,<br>
subjects having one or more parents with non-insulin dependent<br>
diabetes, subjects who have had gestational diabetes and<br>
subjects who have had acute or chronic pancreatitis are at<br>
risk for developing non-insulin dependent diabetes.<br>
An "effective amount" of a GLP-1 compound is the<br>
quantity which results in a desired therapeutic and/or<br>
prophylactic effect without causing unacceptable side-<br>
effects when administered to a subject in need of GLP-1<br>
receptor stimulation. A "desired therapeutic effect"<br>
includes one or more of the following: 1) an amelioration of<br>
the symptom(s) associated with the disease or condition; 2)<br>
a delay in the onset of symptoms associated with the disease<br>
or condition; 3) increased longevity compared with the<br>
absence of the treatment; and 4) greater quality of life<br>
compared with the absence of the treatment. For example, an<br>
"effective amount" of a GLP-1 compound for the treatment of<br>
diabetes is the quantity that would result in greater<br>
control of blood glucose concentration than in the absence<br>
of treatment, thereby resulting in a delay in the onset of<br>
diabetic complications such as retinopathy, neuropathy or<br>
kidney disease. An "effective amount" of a GLP-1 compound<br>
for the prevention of diabetes is the quantity that would<br>
delay, compared with the absence of treatment, the onset of<br>
elevated blood glucose levels that require treatment with<br>
anti-hypoglycaemic drugs such as sulfonyl ureas,<br>
thiazolidinediones, insulin and/or bisguanidines.<br>
The dose of fusion protein effective to normalize a<br>
patient's blood glucose will depend on a number of factors,<br>
among which are included, without limitation, the subject's<br>
sex, weight and age, the severity of inability to regulate<br>
blood glucose, the route of administration and<br>
bioavailability, the pharmacokinetic profile of the fusion<br>
protein, the potency, and the formulation.<br>
The present invention comprises GLP-1 compounds that<br>
have improved biochemical and biophysical properties by<br>
virtue of being fused to an albumin protein, an albumin<br>
fragment, an albumin analog, a Fc protein, a Fc fragment, or<br>
a Fc analog. These heterologous proteins can be<br>
successfully expressed in host cells, retain signaling<br>
activities associated with activation of the GLP-1 receptor,<br>
and have prolonged half-lives.<br>
The following examples are presented to further<br>
describe the present invention. The scope of the present<br>
invention is not to be construed as merely consisting of the<br>
following examples. Those skilled in the art will recognize<br>
that the particular reagents, equipment, and procedures<br>
described are merely illustrative and are not intended to<br>
limit the present invention in any manner.<br>
Example 1; Construction of DNA<br>
encoding heterologous fusion proteins<br>
Example la Construction of DNA encoding Val8-GLP-K7-37)-<br>
Fc: A Fc portion of human IgG1 was isolated from a<br>
cDNA library and contains the full hinge region and the CH2<br>
and CH3 domains. A fragment: containing 696 base pairs of<br>
this Fc portion of human IgGl was subcloned into the NheI<br>
and Eco47III sites of mammalian expression vector pJBO2 to<br>
create pJB02/Fc (see Figure 5). DNA encoding the Ig?<br>
secretion signal sequence fused to Val8-GLP-1(7-37) was<br>
generated by in vitro hybridization of four overlapping and<br>
complementary oligonucleotides:<br>
5' - CTAGCCACCATGGAGACAGACACACTCCTGCTATGGGTACTGCTGCTCTGGGTT<br>
CCAGGTTCCACTGGTGACCAGTG - 3' [SEQ ID NO: 12]<br>
5' - GAGGGCACCTTCACCTCCGACGTGTCCTCCTATCTGGAGGGCCAGGCCGCCAAGGA<br>
GTTCATCGCCTGGCTGGTGAAGGGAAGAGGC - 3' [SEQ ID NO: 13]<br>
5' - TGAAGGTGCCCTCCACGTGGTCACCAGTGGAACCTGGAACCCAGAGCAGCAGTA<br>
CCCATAGCAGGAGTGTGTCTGTCTCCATGGTGG - 3' [SEQ ID<br>
NO:14]<br>
5' - GCCTCTTTCCCTTCACCAGCCAGGCGATGAACTCCrTGGCGGCCTGGCCCTCCAGA<br>
TAGGAGGACACGTCGGAGG - 3' [SEQ ID NO:15]<br>
The hybridization reaction was carried out using<br>
equivalent amounts of each oligonucleotide (1 pm/µl final<br>
concentration for each oligo) . The mixture of<br>
oligonucleotides was heated for 5 min at 100°C in ligation<br>
buffer (50 mM Tris-HCl, pH 7.5, 10 mM MgCl2, 10 mM DTT, 1mM<br>
ATP, 25 µg/ml bovine serum albumin) and then cooled over at<br>
least 2 hours to 30°C.<br>
The resulting hybridization product was ligated for 2<br>
hours at room temperature or overnight at 16°C to the<br>
pJB02/Fc vector backbone which had been digested with NheI<br>
and Eco47III. The ligation products were used to transform<br>
competent XL-1 Blue cells (Stratagene). Recombinant<br>
plasmids were screened for the presence of peptide coding<br>
inserts by digesting clones with NcoI (encoding the Kozak<br>
sequence and first Met of the signal peptide) and sequenced.<br>
The resulting expression plasmid used for transfection<br>
assays was denoted pJB02-V8-GLP-1-Fc (Figure 5).<br>
Example 1b Construction of DNA encoding Val8-GLP-K7-37)-<br>
HSA: The plasmid HSA/pcDNA3. 1GS was purchased from<br>
Invitrogen (Catalog # H-M12523M-pcDNA3.1/GS) and used as a<br>
template to isolate the cDNA encoding human serum albumin<br>
(HSA) . The HSA cDNA was prepared using PCR wherein the DNA<br>
encoding the leader sequence as well as the six amino acid<br>
pro-peptide was removed from the 5' end. In addition, stop<br>
codons were added directly at the 3' end of the HSA coding<br>
sequence. Finally, restriction enzyme sites were engineered<br>
at the 5' and 3' end to facilitate cloning. The HSA DNA<br>
sequence present in the original vector purchased from<br>
Invitrogen contained a single base change in the 3' region<br>
of the gene (position 667) compared to the native human<br>
sequence. This change would result in a codon for Asn<br>
instead of Asp. Thus, using the strand overlapping PCR<br>
mutagenesis method discussed above, the codon was changed to<br>
code for Asp at this position. The resulting HSA encoding<br>
DNA was cloned into the NheI and HindIII sites of pJB02 to<br>
create pJB02-HSA (Figure 6).<br>
The IgK leader sequence fused to the Val8 -GLP-(7-37)<br>
sequence was generated as discussed in Example 1a. This DNA<br>
was ligated into the NheI and FspI sites of pJB02-HSA to<br>
create pJB02- Val8-GLP-1-HSA.<br>
Example lc Construction of DNA encoding Val8-GLP-K7-37)-<br>
linker-HSA:<br>
The vector pJB02-HSA was prepared as discussed in<br>
Example lb. DNA encoding the linker sequence [GGGGS]3 was<br>
ligated in frame to the 5' end of the HSA encoding DNA to<br>
create pJB02-linker-HSA (Figure 7). DNA encoding the Ig?<br>
leader sequence and fused to the Val8-GLP-1(7-37) sequence<br>
and the 5' part of the linker sequence was generated as<br>
discussed in Example la. This DNA was ligated into the NheI<br>
and BspEI sites of pJB02 to create pJB02- Val8-GLP-1-linker-<br>
HSA.<br>
Example 1d Construction of DNA encoding Exendin-4-Fc:<br>
The plasmid pJB02/Fc was prepared as described in<br>
Example 1a. DNA encoding the Ig? signal sequence fused to<br>
Exending-4 was generated by in vitro hybridization of the<br>
- following overlapping and complementary oligonucleotides:<br>
5' - CTAGCCACCATGGAGACAGACACACTCCTGCTATGGGTACTGCTGCTCTG<br>
GGTTCCAGGTTCCACCGGTCAC - 3' [SEQ ID NO: 16]<br>
- 5' - GGAGAGGGAACCTTCACCAGCGACCTGAGCAAGCAGATGGAGGAGGAGGCCGT<br>
GAGACTG - 3' [SEQ ID NO: 17]<br>
5' - TTCATCGAGTGGCTGAAGAACGGAGGACCAAGCAGCGGAGCCCCTCCTCCT<br>
AGC - 3' [SEQ ID NO: 18]<br>
5' - GAACCTGGAACCCAGAGCAGCAGTACCCATAGCAGGAGTGTGTCTGTCTCCA<br>
TGGTGG - 3' [SEQ ID NO:19]<br>
5' - CTCCTCCTCCATCTGCTTGCTCAGGTCGCTGGTGAAGGTTCCCTCTCCGTGA<br>
CCGGTG - 3' [SEQ ID NO:20]<br>
5' - GCTAGGAGGAGGGGCTCCGCTGCTTGGTCCTCCGTTCTTCAGCCACTCGAT<br>
GAACAGTCTCACGGC - 3' [SEQ ID NO:21]<br>
The hybridization reaction was carried out as described<br>
in Example la. . The hybridized product was ligated to the<br>
pJB02 vector which had been digested with NheI and Eco47III<br>
as described in Example la to create pJB02-Exendin-4-Fc.<br>
Example 1e Construction of DNA encoding Exendin-4-HSA:<br>
The plasmid pJB02-HSA was prepared as described in<br>
Example 1b. DNA encoding the Ig? signal sequence fused to<br>
Exending-4 was generated by in vitro hybridization of the<br>
same overlapping and complementary oligonucleotides<br>
described in Example 1d. Hybridization reactions were also<br>
carried out as described above. DNA was cloned into unique<br>
NheI and FspI sites in pJB02-HSA to create pJB02-Exendin-4-<br>
HSA.<br>
Example 1f Construction of DNA encoding Exendin-4-linker-<br>
HSA:<br>
The plasmid pJB02-linker-HSA was constructed as<br>
described in Example 1c. DNA encoding the IgK signal<br>
sequence fused to Exendin-4 and the 5' part of the linker<br>
sequence was generated as in Example 1d. This DNA was<br>
cloned into unique NheI and BspEI sites in pJB02-linker-HSA<br>
to create pJB02-Exendin-4-linker-HSA.<br>
Example 1g Construction of DNA encoding Val8-GLP-1/C-Ex-Fc:<br>
The plasmid pJB02-Exendin-4-Fc was prepared as<br>
described in Example 1d. The Exendin-4 encoding DNA was<br>
excised from the vector with AgeI and Eco47III. The Val8-<br>
GLP-1/C-Ex encoding DNA was generated by in vitro<br>
hybridization of the following overlapping and complementary<br>
oligonucleotides:<br>
5' -CCGGTCACGTGGAGGGCACCTTCACCTCCGACGTGTCCTCCTATCTGGA<br>
GGGCCAGGCCGCCA - 3' [SEQ ID NO:22]<br>
5' - AGGAATTCATCGCCTGGCTGGTGAAGGGCCGGGGCAGCAGCGG<br>
AGCCCCTCCTCCTAGC - 3' [SEQ ID NO:23]<br>
5' - CTCCAGATAGGAGGACACGTCGGAGGTGAAGGTGCCCTCCAC<br>
GTGA - 3' [SEQ ID NO:24]<br>
5' - GCTAGGAGGAGGGGCTCCGCTGCTGCCCCGGCCCTTCACCAGCCAGGCGA<br>
TGAATTCCTTGGCGGCCTGGCC - 3' [SEQ ID NO:25]<br>
The hybridization reaction was carried out as described<br>
in Example la. The hybridized product was ligated in place<br>
of Exendin-4 in the pJB02-Exendin-4-Fc expression vector to<br>
create pJB02-Val8-GLP-1/C-Ex-Fc.<br>
Example 1h Construction of DNA encoding Val8-Glu22-GLP-1-Fc:<br>
The plasmid pJB02-Exendin-4-Fc was prepared as<br>
described in Example 1d. The Exendin-4 encoding DNA was<br>
excised from the vector with AgeI and Eco47III. The Val8-<br>
Glu22-GLP-1 encoding DNA was generated by in vitro<br>
hybridization of the following overlapping and complementary<br>
oligonucleotides:<br>
5' -CCGGTCACGTGGAGGGCACCTTCACCTCCGACGTGTCCTCCTATCTCGA<br>
GGAGCAGGCCGCCA - 3' [SEQ ID NO:26]<br>
5' - AGGAGTTCATCGCCTGGCTGGTGAAGGGCCGGGGC - 3' [SEQ ID NO:27]<br>
5' - GCCCCGGCCCTTCACCAGCCAGGCGATGAACTCCTTGGCGGCC<br>
TGCTC - 3' [SEQ ID NO:28]<br>
5' - CTCGAGATAGGAGGACACGTCGGAGGTGAAGGTGCCCT<br>
CCACGTGA - 3' [SEQ ID NO:29]<br>
The hybridization reaction was carried out as described<br>
in Example 1a. The hybridized product was ligated in place<br>
of Exendin-4 in the pJB02-Exendin-4-Fc expression vector to<br>
create pJB02-Val8 -Glu22 -GLP-1-Fc.<br>
Example 1i Construction of DNA encoding Val8 -Glu22GLP-1/C-<br>
Ex-Fc:<br>
The plasmid pJB02-Exendin-4-Fc was prepared as<br>
described in Example 1d. The Exendin-4 encoding DNA was<br>
excised from the vector with AgeI and Eco47III. The Val8-<br>
Glu22 GLP-1/C-Ex encoding DNA was generated by in vitro<br>
hybridization of the following overlapping and complementary<br>
oligonucleotides:<br>
5' - CCGGTCACGTGGAGGGCACCTTCACCTCCGACGTGTCCTCCTATCTCGA<br>
GGAGCAGGCCGCCA - 3' [SEQ ID NO:30]<br>
5' - AGGAATTCATCGCCTGGCTGGTGAAGGGCCGGGGCAGCAGCGGA<br>
GCCCCTCCTCCTAGC - 3' [SEQ ID NO:31]<br>
5' - CTCGAGATAGGAGGACACGTCGGAGGTGAAGGTGCCC<br>
TCCACGTGA - 3'[SEQ ID NO:32]<br>
5' - GCTAGGAGGAGGGGCTCCGCTGCTGCCCCGGCCCTTCACCAGCCAGGCGA<br>
TGAATTCCTTGGCGGCCTGCTC - 3' [SEQ ID NO:33]<br>
The hybridization reaction was carried out as described<br>
in Example 1a. The hybridized product was ligated in place<br>
of Exendin-4 in the pJB02-Exendin-4-Fc expression vector to<br>
create pJB02-Val8-Glu-GLP-1/C-Ex-Fc.<br>
Example 1j Construction of DNA encoding Gly8-GLP-1-Fc:<br>
The plasmid pJB02-Exendin-4-Fc was prepared as<br>
described in Example 1d. The Exendin-4 encoding DNA was<br>
excised from the vector with AgeI and Eco47III. The Gly8-<br>
GLP-1 encoding DNA was generated by in vitro hybridization<br>
of the following overlapping and complementary<br>
oligonucleotides:<br>
5' - CCGGTCACGGCGAGGGCACCTTCACTAGTGACGTGTCCTCCTATCTGGA<br>
GGGCCAGGCCGCCA - 3' [SEQ ID NO:34]<br>
5' - AGGAGTTCATCGCCTGGCTGGTGAAGGGCCGGGGC - 3' [SEQ ID NO:35]<br>
5' - CTCCAGATAGGAGGACACGTCACTAGTGAAGGTGCCCTC<br>
GCCGTGA - 3' [SEQ ID NO:36]<br>
5' - GCCCCGGCCCTTCACCAGCCAGGCGATGAACTCCTTGGCGGC<br>
CTGGCC - 3' [SEQ ID NO:37]<br>
The hybridization reaction was carried out as described<br>
in Example 1a. The hybridized product was ligated in place<br>
of Exendin-4 in the pJB02-Exendin-4-Fc expression vector to<br>
create pJB02-Gly8-GLP-1-Fc.<br>
Example 2: Expression of heterologous fusion proteins<br>
Expression of the fusion proteins encoded by the DNA<br>
constructs of Example 1 was carried out by transiently<br>
transfecting HEK 293EBNA cells (both adherent and<br>
suspension). Cells were counted and seeded 24 hours prior<br>
to transfection. The transfection cocktail was prepared by<br>
mixing FuGene™6 transfection reagent (Roche Molecular<br>
Biochemicals, catalog # 1814443) with OptiMEM (Gibco/BRL)<br>
and incubating at room temperature for 5 min at which point<br>
DNA was added and the cocktail was incubated for an<br>
additional 15 min. Immediately before transfection, fresh<br>
growth media was added to the plate. Tables 1 and 2 provide<br>
further transfection details.<br>
Table 1: Reagents used in transient transfections of 293EBNA<br>
cells.<br>
Tissue culture vessel	Number of cells seeded	DNA (µg)	FuGene (µl)	OptiMEM media (ml)	Vol. of growth medium (ml)<br>
35 mm	5X10 5	1.5	9	0.102	2<br>
100 mm	2 X 10 6	12	73	0.820	10<br>
700 cm2 (RB)	2 X 10 7	65	400	4.0	100<br>
Table 2: Media composition<br>
Growth and transfaction medium	Harvesting medium<br>
DMEM F12 3:1	Hybritech base<br>
5 % FBS	1 mK Ca 2+<br>
20 mM HEPES	20 mM HEPES<br>
2 mM L-glutamine	1 µg/ml Nuselin (human insulin)<br>
50 µg/ml geneticin (G418 NEO)	1 µg/ml human transferrin<br>
50 µg/ml tobronycin	50 µg/ml tobromycin<br>
For small-scale transfections (35mm - 10mm vessels),<br>
cells were rinsed with PBS and switched to harvesting media<br>
24 hours post- transfection and media was collected and<br>
replaced every 24 hours for several days. In the case of<br>
large-scale transfections (700 cm2 roller bottles), the<br>
roller bottles were rinsed with PBS 48 hours post-<br>
transfection and changed to harvesting media. Media was<br>
collected and changed every 24 hours for at least 10<br>
consecutive days. Routinely, only 10 harvests were used for<br>
subsequent protein purification.<br>
Example 3: Purification of heterologous fusion proteins<br>
Example 3a purification of Val-GLP-1-Fc<br>
Approximately 4.5 liters of conditioned medium (fusion<br>
protein expression level approximately 20 µg/ml) from large-<br>
scale transfections was filtered using a CUNO filter system<br>
and concentrated to 250 ml using a ProFlux tangential flow<br>
filtration system with a 10 K filter membrane. Val8-GLP-1-Fc<br>
was captured with a 5 ml HiTrap protein A column in 1x PBS,<br>
pH 7.4 at a flow rate of 2 ml/min and eluted with 50 mM<br>
citric acid pH 3.3. Fractions (1 ml) were collected in tubes<br>
containing 4 ml lx PBS and 100µl 1M Tris pH 8.<br>
Fractions containing the fusion protein, as determined<br>
by SDS-PAGE and reverse phase-HPLC on Zorbax C8, were pooled<br>
and applied to a Superdex 75 60/60 column in 1x PBS pH 7.4<br>
at a flow rate of 10 ml/min. Positive fractions (20<br>
mis/tube) were collected and pooled. Pooled fractions were<br>
then subjected to C4 Reverse Phase Chromatography in 0.1<br>
%TFA water at a flow rate of 3 ml/min. Val8-GLP-1-Fc was<br>
eluted using a gradient from 5% B (0.1% TFA in acetonitrile)<br>
to 100% B in 70 min. Eluant fractions (3 mis/tube) were<br>
collected. Acetonitrile was removed by vacuum drying and 1<br>
ml of H2O was added. The purified sample (approximately 32<br>
mls) was dialyzed twice against 4 liters of lx PBS pH7.4.<br>
The dialyzed sample was then filtered using a MILLEX-GV<br>
0.22 urn Filter Unit and concentration was determined using<br>
absorption at 280 nm.<br>
Example 3b purification of Val8-GLP-1-HSA or Val8-GLP-1-<br>
Linker-HSA<br>
Approximately 6.5 liters of conditioned medium (fusion<br>
protein expression level approximately 10µg/ml) was filtered<br>
using a CUNO filter system and concentrated to 380 mls using<br>
a ProFlux tangential flow filtration system with a 10 K<br>
filter membrane.<br>
The fusion protein was captured using a 50 ml Past Flow<br>
Q column (Pharmacia) in 20 mM Tris pH 7.4 at a flow rate of<br>
5ml/min. Protein was eluted using a gradient: from 0% to 50%<br>
20mM Tris pH 7.4, 1M NaCl in 10 CV, then to 100%B in 2 CV.<br>
Fractions containing the fusion protein were pooled and<br>
subjected to C4 Reverse Phase Chromatography in 0.1% TFA<br>
water at a flow rate of 5 ml/min. The fusion protein was<br>
eluted using a gradient from 20% B (0.1% TFA in<br>
acetonitrile) to 90% B in 120min. Fractions (3.5 ml/tube)<br>
were collected. Acetonitrile was removed by vacuum drying.<br>
Approximately 9 mls of pooled sample was diluted with<br>
1x PBS pH 7.4 to 40ml and dialyzed against 4 liters of 1x<br>
PBS pH 7.4 overnight. The sample was filtered and<br>
concentration was determined by absorption at 280nm.<br>
Example 3c purification of Exendin-4-Fc:<br>
Approximately 4 liters of conditioned medium (fusion<br>
protein expression level approximately 8 µg/ml) was filtered<br>
using a CUNO filter system and concentrated to 250 mls using<br>
a ProFlux tangential flow filtration system with a 3OK<br>
filter membrane.<br>
Exendin-4-Fc was captured with a 5 ml HiTrap protein A<br>
column in lx PBS, pH 7.4 at a flow rate of 2 ml/min and<br>
eluted with 50 mM citric acid pH 3.3. Fractions containing<br>
the fusion protein were pooled, filtered, and dialyzed<br>
against 4 liters of 1 x PBS over night. The dialyzed sample<br>
was then applied to a Superdex 75 60/60 column in lx PBS<br>
PH7.4, 0.5M NaCl at a flow rate of 10 ml/min. Fractions (20<br>
ml/tube) containing the fusion protein were collected,<br>
pooled, and concentrated to about 1 mg/ml. Concentrated<br>
samples were then filtered using a MILLEX-GV 0.22 urn Filter<br>
Unit.<br>
Example 3d purification of Exendin-4-HSA and Exendin-4-<br>
linker-HSA:<br>
Approximately 1.1 liters of conditioned medium (fusion<br>
protein expression level approximately 6µg/ml) was filtered<br>
using a CUNO filter system and concentrated to 175 mls using<br>
a ProFlux tangential flow filtration system with a 30k<br>
filter membrane.<br>
The fusion protein was captured using a 5 ml HiTrap Q-<br>
sepharose column (Pharmacia) in 20 mM Tris pH 7.4 at a flow<br>
rate of 2 ml/min. Protein was eluted using a gradient from<br>
0% to 50% 20rnM Tris pH 7.4, 1M NaCl in 12 CV and then to<br>
100%B in 4 CV.<br>
Fractions containing the fusion protein were pooled and<br>
subjected to C4 Reverse Phase Chromatography in 0.1% TFA<br>
water at a flow rate of 5 ml/min. The fusion protein was<br>
eluted using a gradient from 10% B (0.1% TFA in<br>
acetonitrile) to 100% B in 70 min. Fractions (10 ml/tube)<br>
containing the fusion protein were collected. Acetonitrile<br>
was removed using a vacuum dryer.<br>
Approximately 8 mls of pooled sample was dialyzed<br>
against 4 liters of 1x PBS pH 7.4 overnight. The sample was<br>
filtered and concentration was determined by absorption at<br>
280nm. The dialyzed sample was then applied to a Superdex<br>
200 26/60 column in 1x PBS pH 7.4, 0.5 M NaCl at a flow rate<br>
of 2 ml/min. Fractions (3 ml/tube) containing the fusion<br>
protein were collected, pooled, concentrated, and filtered.<br>
Example 4: Characterization of fusion proteins by SDS PAGE:<br>
SDS-PAGE followed by immunoblotting was used to analyze<br>
both purified fusion protein as well as conditioned medium<br>
from cells transfected with various fusion protein<br>
expression vectors. SDS-PAGE was performed on a Novex<br>
Powerease 500 system using Novex 16% Tris-Glycine Precast<br>
gels (EC6498), running buffer (10x, LC2675) and sample<br>
buffer (L2676) . Samples were reduced with 50 mM DTT and<br>
heated 3-5 min at 95°C prior to loading.<br>
After running the SDS-PAGE gel, water and transfer<br>
buffer (1X Tris-Glycine Seprabuff (Owl Scientific Cat. No.<br>
ER26-S) with 20% methanol) were used to rinse SDS from the<br>
gels. A Novex transfer apparatus was used with PVDF (BioRad,<br>
Cat. No. 162-0174) and nitrocellulose membranes (BioRad,<br>
Cat. No. 1703965 or 1703932). Transfer was carried out at<br>
room temperature for 90 min at 30-35 V. Membranes were<br>
blocked in 1X PBS with 0.1% Tween-20 (Sigma, Cat. No. P-<br>
7949) and 5% Milk (BioRad, Cat. No. 170-6404) for 1-12 hours<br>
at 4°C. Antibodies are diluted into 1x PBS +5% Milk and the<br>
blots are incubated in these solutions for 1-2 h at 4°C.<br>
Between incubations, the blots are washed four times for 5<br>
min each with 1X PBS and 0.2% Tween-20 at room temperature.<br>
PBS was made from either GIBCO 10X PBS (Cat No. 70011), to<br>
give a final composition of 1 mM monobasic potassium<br>
phosphate, 3 mM dibasic sodium phosphate, 153 mM sodium<br>
chloride, pH 7.4, or PBS pouches from Sigma (Cat. No. 1000-<br>
3), to give 120 mM NaCl, 2.7 mM KC1 and 10 mM phosphate, pH<br>
7.4 at 25°C.<br>
The primary antibodies were either a polycolonal goat<br>
anti-IgG1 or rabbit anti-HSA. The secondary antibody was<br>
either an anti-goat IgG HRP or an anti-rabbit IgG HRP. The<br>
secondary antibody was diluted 1:5000. An BCL system<br>
(Amersham Pharmacia Biotech, Cat. No. RN2108 and Cat. No.<br>
RPN1674H) was used for developing blots.<br>
Figure 3A compares purified Pc protein to conditioned<br>
media from pJB02-Val8-GLP-1-Fc and pJB02-Exendin-4-Fc<br>
transfected cells. The decrease in mobility is consistent<br>
with the increased size due to the GLP-1 portion of the<br>
fusion protein. Figure 3B similarly compares purified HSA<br>
with conditioned media from cells transfected with pJB02-<br>
Val8-GLP-2-HSA, pJB02- Val8-GLP-1-Linker-HSA, pJB02-Exendin-<br>
4-HSA, or pJB02-Exendin-4-Linker-HSA. Figure 4 identifies<br>
purified fusion protein preparations.<br>
Example 5; Characterization of fusion<br>
proteins using mass spectrometry:<br>
All experiments were performed on a Micromass TofSpec-<br>
2E mass spectrometer equipped with Time Lag Focusing<br>
electronics, a Reflectron (used in analysis of the 0-8000 Da<br>
peptide range), a Linear detector (used during high mass /<br>
good signal analysis), and Post Acceleration Detector (or<br>
P.A.D., used for high mass / extremely low signal analysis)<br>
The effective path length of the instrument in Linear mode<br>
is 1.2 meters, in Reflectron mode it is 2.3 meters. Two<br>
dual micro-channel plate detectors are fitted for linear and<br>
reflectron mode detection. The laser used is a Laser<br>
Science Inc. VSL-337i nitrogen laser operating at 337 nm at<br>
5 laser shots per second. All data were acquired using a 2<br>
GHz, 8 bit internal digitizer and up to 50 laser shots were<br>
averaged per spectrum.<br>
The instrument was operated in linear mode for the<br>
analysis of the GLP-1 fusion proteins in question. The<br>
linear detector is a device that detects ions that travel<br>
down the flight tube of the MALDI-ToF-MS instrument. It<br>
measures the ion abundance over time and sends a signal to<br>
the digitizer for conversion. The digitizer is an analog-<br>
to-digital converter that allows the signal from the mass<br>
spectrometer to be transferred to the computer, where it is<br>
reconstructed into a usable m/z spectrum.<br>
A recrystallized saturated sinapinic acid solution<br>
(diluted in 50/50 Acn / H2O and 0.1% TFA) was utilized as<br>
the ionization matrix. Sinapinic acid is a proper matrix for<br>
proteins above 10 kDa. Mass appropriate reference proteins<br>
were used for internal and external calibration files in<br>
order to obtain accurate mass determinations for the samples<br>
analyzed. Samples were all analyzed using a 1:2 sample to<br>
matrix dilution. The instrument was initially set up under<br>
the following linear detector conditions:<br>
Source Voltage: 20.0 keV Pulse Voltage: 3.0 keV<br>
Extraction Voltage: 20.0 keV Laser Coarse: 50<br>
Focus Voltage: 16.0 keV Laser Fine: 50<br>
Linear detector: 3.7 keV<br>
P.A.D.: (off line)<br>
These settings were modified (as needed) to give the<br>
best signal/noise ratio and highest resolution. Table 3<br>
provides a characterization of different GLP-1 fusion<br>
proteins.<br>
CEx refers to a C-terminal extension and comprises the<br>
sequence of Ser-Ser-Gly-Ala-Pro-Pro-Pro-Ser.<br>
Linker is Gly-Gly-Gly-Gly-Ser-Gly-Gly-Gly-Gly-Ser-Gly- Gly-<br>
Gly-Gly-Ser.<br>
Example 6: Activity of heterologous fusion proteins:<br>
The ability of the fusion proteins of the present<br>
invention to activate the GLP-1 receptor was assessed using<br>
in vitro assays such as those described in EP 619,322 to<br>
Gelfand, et al., and D.S. Patent No. 5,120,712,<br>
respectively. The activity of these compounds relative to<br>
the activity of Val8-GLP-1(7-37)OH is reported in Table 4.<br>
Figure 8 represents in vitro dose response curves for Val8-<br>
GLP-1 and Exendin-4 fusion proteins. In addition, Table 5a<br>
and 5b provide the in vitro activity of a large group of<br>
GLP-1 analogs that can be fused to an Fc or an albumin<br>
protein to make biologically active fusion proteins. These<br>
activities are compared to GLP-1(7-37)OH.<br>
Table 4: In vitro activity of GLP-1 fusion proteins<br>
Fusion Protein	In Vitro Activity (% of Val8-GLP-1)<br>
Val8-GLP-1-IgG1	1<br>
Exendin-4-IgG1	240<br>
Val8-GLP-1-Linker-HSA	0.2<br>
Exendin-4-HSA	20<br>
Exendin-4-Linker-HSA	90<br>
Exendin-4	500<br>
Val8-Glu22-GLP-1-IgG1	3.7<br>
Gly8-GLP-1-IgG1	3.3<br>
Val8-GLP-1-CEx-IgG1	3.3<br>
Val8-Glu-GLP-1-CEx-IgG1	29<br>
Gly8-Glu22-GLP-1-C2-IgG1	75<br>
Gly8-Glu22-GLP-1-CEx-Linker-IgG1	150<br>
Exendin-4-C2-IgG1	250<br>
Exendin-4-Linker-IgG1	330<br>
Gly8-Glu22-GLP-1-CEx-Linker-HSA	4<br>
Gly8-Glu22-GLP-1-CEx-Linker-IgG4	80<br>
CEx refers to a C-terminal extension and comprises the<br>
sequence of Ser-Ser-Gly-Ala-Pro-Pro-Pro-Ser.<br>
Linker is Gly-Gly-Gly-Gly-Ser-Gly-Gly-Gly-Gly-Ser-Gly- Gly-<br>
Gly-Gly-Ser<br>
C2 is Ser-Ser-Gly-Ala-Ser-Ser-Gly-Ala.<br>
The amino acid sequences of the fusion proteins<br>
described in Tables 3 and 4 are represented in SED ID NO: 13<br>
to SEQ ID NO: 31.<br>
Val8-GLP-1-Human serum albumin amino acid sequence is<br><br>
Val8-Ala16-GLP-1(7-37)OH 0.112<br>
Val8-Tyr16-GLP-1(7-37)OH 1.175<br>
Val8-Lys20-GLP-1(7-37)OH 0.33<br>
Gln22-GLP-1(7-37)OH 0.42<br>
Val8-Ala22-GLP-1(7-37)OH 0.56<br>
Val8-Ser22-GLP-1(7-37)OH 0.50<br>
Val8-Asp22-GLP-1(7-37)OH 0.40<br>
Val8-Glu22-GLP-1(7-37)OH 1.29<br>
Val8-Lys22-GLP-1(7-37)OH 0.58<br>
Val8-Pro22-GLP-1(7-37)OH 0.01<br>
Val8-His22-GLP-1(7-37)OH 0.14<br>
Val8-Lys22-GLP-1(7-36)NH2 0.53<br>
Val8-Glu22-GLP-1(7-36)NH2 1.0<br>
Gly8-Glu22-GLP-1(7-37)OH 1.07<br>
Val8-Lys23-GLP-1(7-37)OH 0.18<br>
Val8-His24-GLP-1(7-37)OH 0.007<br>
Val8-Lys24-GLP-1(7-37)OH 0.02<br>
Val8-His26-GLP-1(7-37)OH 1.6<br>
Val8-Glu26-GLP-1(7-37)OH 1.5<br>
Val8-His27-GLP-1(7-37)OH 0.37<br>
Val8-Ala27-GLP-1(7-37)OH 0.47<br>
Gly8-Glu30-GLP-1(7-37)OH 0.29<br>
Val8-Glu30-GLP-1(7-37)OH 0.29<br>
Val8-Asp30-GLP-1(7-37)OH 0.15<br>
Val8-Ser30-GLP-1(7-37)OH 0.19<br>
Val8-His30-GLP-1(7-37)OH 0.19<br>
Val8-Glu33-GLP-1(7-37)OH 0.039<br>
Val8-Ala33-GLP-1(7-37)OH 0.1<br>
Val8-Gly33-GLP-1(7-37)OH 0.01<br>
Val8-Glu34-GLP-1(7-37)OH 0.17<br>
Val8-Pro35-GLP-1(7-37)OH 0.094<br>
Val8-His35-GLP-1(7-37)OH 0.41<br>
Val8-Glu35-GLP-l(7-37)OH 0.15<br>
Val8-Glu36-GLP-1(7-37)OH 0.11<br>
Val8-His36-GLP-1(7-37)OH 0.22<br>
Val8-His37-GLP-1(7-37)OH 0.33<br>
Val8-Leu16-Glu26-GLP-1(7-37)OH 0.23<br>
Val8-Lys22-Glu30-GLP-1(7-37)OH 0.37<br>
Val8-Lys22-Glu23-GLP-1(7-37)OH 0.35<br>
Val8-Glu22-Ala27-GLP-1(7-37)OH 1.02<br>
Val8-Glu22-Lys23-GLP-1(7-37)OH 1.43<br>
Val8-Lys33-Val34-GLP-1(7-37)OH 0.08<br>
ValB-Lys33-Asn34-GLP-1(7-37)OH 0. 09<br>
Val8-Gly34-Lys35-GLP-1(7-37)OH 0.34<br>
Val8-Gly36-Pro37-GLP-1(7-37)NH2 0.53<br>
8<br>
Example 7: In vivo pharmacokinetics of Val -GLP-1-IgGl<br>
and Val -GLP-1-HSA:<br>
--------------8 8<br>
A pharmacokinetic study of Val -GLP-1-lgGl and Val -<br>
GLP-1-HSA was performed in cynomologus monkeys. Monkeys<br>
8<br>
were dosed at 5.6 nmoles/kg with either purified Val -GLP-1-<br>
g<br>
IgGl- or Val -GLP-1-HSA . The compounds were administered as<br>
an intravenous bolus administration. Blood was collected<br>
pre-dose and at 0.083, 0.25, 0.5, 1, 4, 8, 12, 24, 48, 72,<br>
96, 120, 144, 168, and 216 hours post-dose into tubes<br>
containing EDTA. Plasma concentrations of immunoreactive<br>
8<br>
Val -GLP-1 were determined using a radioimmunoassay that<br>
utilizes a polyclonal antiserum whose primary specificity is<br>
for the N-terminal (7-16) region of Val -GLP-K7-37) .<br>
8<br>
Figure 9 depicts the plasma concentration of Val -GLP-1-Fc<br>
and Val -GLP-1-Linker-HSA following a single intravenous<br>
dose to two cynomologus monkeys. The Fc fusion protein had<br>
a half-life of approximately 45 hours and the albumin fusion<br>
had a half-life of approximately 87 hours.<br>
Example 8; In vivo pharmacodynamics of Exendin-4-IgG1:<br>
Two chronically cannulated normal male beagle dogs were<br>
studied after an overnight fast. Arterial and venous<br>
vascular access ports were accessed, and a catheter was<br>
inserted percutaneously into a cephalic vein and secured.<br>
Animals were placed in cages, and their catheters were<br>
attached to a swivel/tether system. A solution containing<br>
the fusion protein Exendin-4-IgG1 (11.8 µm) was injected<br>
intravenously (1.0 nmol/kg) through the cephalic vein<br>
catheter. The catheter was then cleared with 10 ml of<br>
saline. Two hours later, a hyperglycemic (150 mg/d1) clamp<br>
was initiated and continued for three hours. Arterial blood<br>
samples were drawn throughout this 5-hour period for<br>
determination of plasma concentrations of the fusion<br>
protein, glucose, and insulin.<br>
The results of this study were compared to those from a<br>
similar, previous study in which both of the animals had<br>
received a bolus of saline, s.c, and three hours later were<br>
studied using a 3-hour hyperglycemic (150 mg/dl) clamp.<br>
In both sets of studies, plasma glucose concentrations<br>
were determined using a Beckman glucose analyzer. Plasma<br>
insulin concentrations were determined by employees of Linco<br>
Research, Inc. using an RIA kit developed in their<br>
laboratories. The data is illustrated in Figures 10 and 11.<br>
Example 9: In vivo pharmacokinetics of<br>
Gly8-Glu22-GLP-1-CEx-Linker-IgG1:<br>
Two groups of three normal male beagle dogs received<br>
0.1 mg/kg of Gly8-Glu22-GLP-1-CEx-Linker-IgGl by subcutaneous<br>
(SC) or intravenous (IV) administration. Plasma<br>
concentrations of Gly8-Glu22-GLP-1-CEx-Linker-IgG1<br>
immunoreactivity were determined by radioimmunoassay in<br>
samples collected from 30 minutes predose to 216 hours<br>
postdose for both the IV and SC groups. These<br>
concentrations were subsequently used to determine the<br>
reported pharmacokinetic parameters. The mean elimination<br>
half-life of IV administered Gly8-Glu22-GLP-1-CEx-Linker-IgG1<br>
was approximately 55 hours and the total body clearance was<br>
1.5 mL/h/kg. The mean elimination half-life of SC<br>
administered Gly8-Glu22-GLP-1-CEx-Linker-IgGl was<br>
approximately 38 hours.<br>
We claim<br>
1. A heterologous fusion protein comprising a first polypeptide with a N-terminus and<br>
a C-terminus fused to a second polypeptide with a N-terminus and a C-terminus<br>
wherein the first polypeptide is a GLP-1 compound and the second polypeptide is<br>
selected from<br>
a) the Fc portion of an immunoglobulin;<br>
b) an analog of the Fc portion of an immunoglobulin; and<br>
c) fragments of the Fc portion of an immunoglobulin<br>
and wherein the C-terminus of the first polypeptide is fused to the N-terminus of the<br>
second polypeptide.<br>
2. The heterologous fusion protein of claim 1 wherein the C-terminus of the first<br>
polypeptide is fused to the N-terminus of the second polypeptide via a peptide<br>
linker.<br>
3. The heterologous fusion protein of claim 2 wherein the peptide linker is selected<br>
from<br>
a) a glycine rich peptide;<br>
b) a peptide having the sequence [Gly-Gly-Gly-Gly-Ser]n where n is 1, 2,<br>
3, 4, 5 or 6; and<br>
c) a peptide having the sequence [Gly-Gly-Gly-Gly-Ser]3<br>
4. The heterologous fusion protein of any one of the preceding claims wherein the<br>
GLP-1 compound comprises the sequence of formula 1 [SEQ ID NO: 2]<br>
7 8 9 10 11 12 13 14 15 16 17<br>
His-Xaa-Xaa-Gly-Xaa-Phe-Thr-Xaa-Asp-Xaa-Xaa-<br>
18 19 20 21 22 23 24 25 26 27 28<br>
Xaa-Xaa-Xaa-Xaa-Xaa-Xaa-Xaa-Xaa-Xaa-Xaa-Phe-<br>
29 30 31 32 33 34 35 36 37 38 39<br>
Ile-Xaa-Xaa-Xaa-Xaa-Xaa-Xaa-Xaa-Xaa-Xaa-Xaa-<br>
40 41 42 43 44 45<br>
Xaa-Xaa-Xaa-Xaa-Xaa-Xaa<br>
Formula I (SEQ ID NO: 2)<br>
wherein:<br>
Xaa at position 8 is Ala, Gly, Ser, Thr, Leu, IIe, Val, Glu, Asp, or Lys;<br>
Xaa at position 9 is Glu, Asp, or Lys;<br>
Xaa at position 11 is Thr, Ala, Gly, Ser, Leu, IIe, Val, Glu, Asp, or Lys;<br>
Xaa at position 14 is Ser. Ala, Gly, Thr, Leu, IIe. Val, Glu, Asp, or Lys;<br>
Xaa at position 16 is Val, Ala, Gly, Ser, Thr, Leu, Ile, Tyr, Glu, Asp, Tip, or Lys;<br>
Xaa at position 17 is Ser, Ala, Gly, Thr, Leu, IIe, Val, Glu, Asp, or Lys;<br>
Xaa at position 18 is Ser, Ala, Gly, Thr, Leu, IIe, Val, Glu, Asp, Trp, Tyr, or Lys;<br>
Xaa at position 19 is Tyr, Phe, Trp, Glu, Asp, Gln, or Lys;<br>
Xaa at position 20 is Leu, Ala, Gly, Ser, Thr, Ile, Val, Glu, Asp, Met, Trp, Tyr, or<br>
Lys;<br>
Xaa at position 21 is Glu, Asp, or Lys;<br>
Xaa at position 22 is Gly, Ala, Ser, Thr, Leu, He, Val, Glu, Asp, or Lys;<br>
Xaa at position 23 is Gln, Asn, Arg, Glu, Asp, or Lys;<br>
Xaa at position 24 is Ala, Gly, Ser, Thr, Leu, IIe, Val, Arg, Glu, Asp, or Lys;<br>
Xaa at position 25 is Ala. Gly, Ser, Thr, Leu, IIe, Val. Glu, Asp, or Lys;<br>
Xaa at position 26 is Lys, Arg. Gln, Glu, Asp, or His;<br>
Xaa at position 27 is Leu. Glu, Asp, or Lys;<br>
Xaa at position 30 is Ala. Gly, Ser, Thr, Leu, IIe. Val. Glu, Asp, or Lys;<br>
Xaa at position 31 is Trp. Phe, Tyr, Glu, Asp, or Lys;<br>
Xaa at position 32 is Leu, Gly, Ala, Ser, Thr, IIe, Val, Glu, Asp, or Lys;<br>
Xaa at position 33 is Val, Gly, Ala, Ser, Thr, Leu, He, Glu, Asp, or Lys;<br>
Xaa at position 34 is Asn, Lys, Arg, Glu, Asp. or His;<br>
Xaa at position 35 is Gly, Ala, Ser, Thr, Leu, IIe, Val, Glu, Asp, or Lys;<br>
Xaa at position 36 is Gly, Arg, Lys, Glu, Asp, or His;<br>
Xaa at position 37 is Pro, Gly, Ala, Ser, Thr, Leu, He, Val, Glu, Asp, or Lys, or is<br>
deleted;<br>
Xaa at position 38 is Ser, Arg, Lys, Glu, Asp, or His, or is deleted;<br>
Xaa at position 39 is Ser, Arg, Lys, Glu, Asp, or His, or is deleted;<br>
Xaa at position 40 is Gly, Asp, Glu, or Lys, or is deleted;<br>
Xaa at position 41 is Ala. Phe, Trp, Tyr, Glu, Asp, or Lys, or is deleted;<br>
Xaa at position 42 is Ser, Pro, Lys. Glu, or Asp, or is deleted;<br>
Xaa at position 43 is Ser, Pro, Glu. Asp, or Lys, or is deleted;<br>
Xaa at position 44 is Gly, Pro, Glu, Asp, or Lys, or is deleted; and<br>
Xaa at position 45 is Ala, Ser, Val, Glu, Asp, or Lys, or is deleted;<br>
provided that when the amino acid at position 37, 38, 39, 40. 41, 42, 43, or 44 is<br>
deleted, then each amino acid downstream of that amino acid is also deleted.<br>
5. The heterologous fusion protein of any one of the preceding claims wherein the<br>
GLP-1 compound has no more than 6 amino acids that are different from the<br>
corresponding amino acid in GLP-1 (7-37)OH, GLP-1 (7-36)OH, or Exendin-4.<br>
6. The heterologous fusion protein of claim 5 wherein the GLP-1 compound has no<br>
more than 5 amino acids that differ from the corresponding amino acid in GLP-1(7-<br>
37)OH, GLP-1(7-36)OH, or Exendin-4.<br>
7. The heterologous fusion protein of claim 6 wherein the GLP-1 compound has no<br>
more than 4 amino acids that differ from the corresponding amino acid in GLP-1(7-<br>
37)OH, GLP-1(7-36)OH, or Exendin-4.<br>
8. The heterologous fusion protein of claim 7 wherein the GLP-1 compound has no<br>
more than 3 amino acids that differ from the corresponding amino acid in GLP-1(7-<br>
37)0H, GLP-1(7-36)OH, or Exendin-4.<br>
9. The heterologous fusion protein of claim 8 wherein the GLP-1 compound has no<br>
more than 2 amino acids that differ from the corresponding amino acid in GLP-1 (7-<br>
37)OH, GLP-1(7-36)OH, or Exendin-4.<br>
10. The heterologous fusion protein of any one of claims 5 to 9 wherein Xaa at position<br>
8 is glycine or valine.<br>
11. The heterologous fusion protein of any one of claims 1 to 10 wherein the second<br>
polypeptide is the Fc portion of an Ig selected from: IgG1, IgG2, IgG3, IgG4, IgE,<br>
IgA, IgD, or IgM.<br>
12. The heterologous fusion protein of claim 11 wherein the second polypeptide is the<br>
Fc portion of an Ig selected from: IgG1, IgG2, IgG3, and IgG4.<br>
13. The heterologous fusion protein of claim 12 wherein the second polypeptide is the<br>
Fc portion of an IgGl immunoglobulin.<br>
14. The heterologous fusion protein of claim 12 wherein the second polypeptide is the<br>
Fc portion of an IgG4 immunoglobulin.<br>
15. The heterologous fusion protein of 12 wherein the IgG is human.<br>
16. The heterologous fusion protein of claim 11 wherein the Fc portion comprises the<br>
hinge, CH2, and CH3 domains.<br>
17. The heterologous fusion protein of claim 1 wherein the second polypeptide has the<br>
sequence of SEQ ID NO: 32.<br>
18. A pharmaceutical formulation adapted for the treatment of patients with non-insulin<br>
dependent diabetes or obesity comprising a heterologous fusion protein of any one of<br>
claims 1 to 17 with excipients, diluents, carriers, pharmaceutically acceptable agents.<br>
The present invention relates to glucagon-like-1 compounds fused to proteins that have<br>
the effect of extending the in vivo half-life of the peptides. These fusion proteins can be<br>
used to treat non-insulin dependent diabetes mellitus as well as a variety of other<br>
conditions.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTQ2LUtPTE5QLTIwMDMtQ09SUkVTUE9OREVOQ0UucGRm" target="_blank" style="word-wrap:break-word;">546-KOLNP-2003-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTQ2LUtPTE5QLTIwMDMtRk9STSAyNy5wZGY=" target="_blank" style="word-wrap:break-word;">546-KOLNP-2003-FORM 27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTQ2LWtvbG5wLTIwMDMtZ3JhbnRlZC1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">546-kolnp-2003-granted-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTQ2LWtvbG5wLTIwMDMtZ3JhbnRlZC1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">546-kolnp-2003-granted-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTQ2LWtvbG5wLTIwMDMtZ3JhbnRlZC1jb3JyZXNwb25kZW5jZS5wZGY=" target="_blank" style="word-wrap:break-word;">546-kolnp-2003-granted-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTQ2LWtvbG5wLTIwMDMtZ3JhbnRlZC1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">546-kolnp-2003-granted-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTQ2LWtvbG5wLTIwMDMtZ3JhbnRlZC1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">546-kolnp-2003-granted-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTQ2LWtvbG5wLTIwMDMtZ3JhbnRlZC1leGFtaW5hdGlvbiByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">546-kolnp-2003-granted-examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTQ2LWtvbG5wLTIwMDMtZ3JhbnRlZC1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">546-kolnp-2003-granted-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTQ2LWtvbG5wLTIwMDMtZ3JhbnRlZC1mb3JtIDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">546-kolnp-2003-granted-form 13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTQ2LWtvbG5wLTIwMDMtZ3JhbnRlZC1mb3JtIDE4LnBkZg==" target="_blank" style="word-wrap:break-word;">546-kolnp-2003-granted-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTQ2LWtvbG5wLTIwMDMtZ3JhbnRlZC1mb3JtIDIucGRm" target="_blank" style="word-wrap:break-word;">546-kolnp-2003-granted-form 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTQ2LWtvbG5wLTIwMDMtZ3JhbnRlZC1mb3JtIDI2LnBkZg==" target="_blank" style="word-wrap:break-word;">546-kolnp-2003-granted-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTQ2LWtvbG5wLTIwMDMtZ3JhbnRlZC1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">546-kolnp-2003-granted-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTQ2LWtvbG5wLTIwMDMtZ3JhbnRlZC1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">546-kolnp-2003-granted-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTQ2LWtvbG5wLTIwMDMtZ3JhbnRlZC1yZXBseSB0byBleGFtaW5hdGlvbiByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">546-kolnp-2003-granted-reply to examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTQ2LWtvbG5wLTIwMDMtZ3JhbnRlZC1zZXF1ZW5jZSBsaXN0aW5nLnBkZg==" target="_blank" style="word-wrap:break-word;">546-kolnp-2003-granted-sequence listing.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTQ2LWtvbG5wLTIwMDMtZ3JhbnRlZC1zcGVjaWZpY2F0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">546-kolnp-2003-granted-specification.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTQ2LWtvbG5wLTIwMDMtZ3JhbnRlZC10cmFuc2xhdGVkIGNvcHkgb2YgcHJpb3JpdHkgZG9jdW1lbnQucGRm" target="_blank" style="word-wrap:break-word;">546-kolnp-2003-granted-translated copy of priority document.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="225217-a-method-and-machine-for-producing-bent-glass-sheets.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="225219-a-biocompatible-drug-release-device-and-a-method-of-preparing-a-biocompatible-implantable-medical-device-to-inhibit-the-proliferation-of-smooth-muscle-cells-after-a-stent-implantation.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>225218</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>546/KOLNP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>45/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>07-Nov-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>05-Nov-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>30-Apr-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>ELI LILLY AND COMPANY</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>A CORPORATION OF THE STATE OF INDIANA, U.S.A.,HAVING A PRINCIPAL PLACE OF BUSINESS AT LILLY CORPORATE CENTER, CITY OF INDIANAPOLIS, STATE OF INDIANA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>GLAESNER, WOLFGANG</td>
											<td>7512 FIELDSTONE COURT, INDIANAPOLIS, IN 46254</td>
										</tr>
										<tr>
											<td>2</td>
											<td>MICANOVIC, RADMILLA</td>
											<td>7126 FIELDSTONE COURT, INDIANAPOLIS, IN 76236</td>
										</tr>
										<tr>
											<td>3</td>
											<td>TSCHANG, SHENG-HUNG, RAINBOW</td>
											<td>4963 RILEY MEWS, CARMEL, IN 46033</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 19/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US01/43165</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-11-29</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/251,954</td>
									<td>2000-12-07</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/225218-glf-1-fc-fusion-proteins by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 22:34:53 GMT -->
</html>
