<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/225386-a-protein-having-luciferase-activity by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:28:20 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 225386:&quot;A PROTEIN HAVING LUCIFERASE ACTIVITY&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A PROTEIN HAVING LUCIFERASE ACTIVITY&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A protein having luciferase activity and having over 60% homology of amino acid sequence to luciferase from Photinus pyralis, Luciola mingrelica, Luciola cruciata or Luciola lateralis, characterized in that the amino acid residue corresponding to residue 270 of Photinus pyralis luciferase, or residue 272 of Luciola mingrelica, Luciola cruciata or Luciola lateralis luciferase is an amino acid other than glutamate, such that the Michaelis-Menten constant (Km) of the protein is decreased as compared to the corresponding wild-type sequence.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present invention relates to a protein having luciferase activity. The present invention relates to novel proteins having luciferase activity and to DNA and vectors encoding for their expression. Particularly the present invention provides luciferases having lower Km for the substrate ATP than existing native and recombinant luciferases of wild and altered wild type-Firefly luciferase catalyses the oxidation of luciferin in the presence of ATP, Mg2+ and molecular oxygen with the resultant production of light. This reaction has a quantum yield of about 0.88 (see DeLuca &amp; McElroy (1978) and Seliger &amp; McElroy (I960)) and this light emitting property has led to its use in luminometric assays where ATP levels are being measured.<br>
Luciferase is obtainable directly from bodies of fireflies or by expression from microorganisms including recombinant DNA constructs encoding for the enzyme. Significant species from which the enzyme may be obtained, or DNA encoding for it derived, are the Japanese GENJI and HEIKE fireflies Luciola cruciata and Luciola lateralis, the East European Firefly Luciola mingrelica. the North American firefly (Photinus pyralis) and the glow-worm and the European glow-worm Lampyris noctiluca.<br>
The heat stability of wild and recombinant type luciferases is such that they lose activity quite rapidly when exposed to temperatures in excess of about 30°C, particularly over 35°C, and this renders the enzyme deficient when used at high ambient temperatures. It is known that Japanese firefly luciferase can be heat stabilised by mutating it at its position 217 to replace a threonine residue by an isoleucine residue (Kajiyama and Nakano (1993) Biochemistry 32 page 13795 to 13799); pH stability and specific activity also being increased.<br>
Copending patent application GB 9405750.2 discloses an amino acid substitution that is capable of increasing the thermostability of inter alia, Photinus pyralis which can be used with the change at 217 to provide luciferase that is relatively heat stable at 50°C or more.<br><br>
The present invention relates to a further enhancement of the properties of luciferase enzymes, making them suitable for use in assays based upon the detection of adenosine triphosphate at relatively low levels. This enhancement is provided by changing the amino acid at the position corresponding to position 270 in the Photinus pyralis luciferase amino acid sequence whereby the Michaelis-Menten constant (Km) of the enzyme is decreased as compared to a corresponding luciferase having wild-type sequence. This corresponds to amino acid 272 in Luciola mingrelica. Luciola cruciata and Luciola lateralis. It also corresponds to amino acid 270 in Lampris Noctiluca.<br>
The present enhancement further provides luciferases that are characterised by the ability oxidise D-luciferin with light emission of a different wavelength to that of wild-type luciferase, thus allowing them to be used as specific labels in binding assays wherein the wavelength of light emitted is characteristic of a particular labelled material being present, or allows DNA encoding for the luciferases to be used as a reporter DNA for genetically engineered cells and cells derived therefrom.<br>
Thus in the first aspect of the present invention there is provided a protein having luciferase activity and having over 60% homology of amino acid sequence with that of Photinus pyralis. Luciola mingrelica. Luciola cruciata or Luciola lateralis characterised in that the amino acid residue corresponding to residue 270 of Photinus pyralis luciferase and residue 272 of Luciola mingrelica. Luciola cruciata and Luciola lateralis luciferase is an amino acid other than glutamate. Preferably is characterised in that it comprises a conserved amino acid sequence F(1)XE(2)FL wherein (1) is D or E, (2) is T or L and X is the amino acid other than glutamate; F, E, L, D and T each relating to the corresponding amino acid as provided for by the single letter amino acid code.<br>
The preferred amino acid X so far determined is lysine, or an analogue or modification thereof. Other preferred amino acids include arginine, glutamine and alanine.<br>
In still more preferred forms of the present invention the protein of the invention also has the amino acid at the position corresponding to amino acid 217 of the Luciola firefly luciferases or 215 of Photinus pyralis changed to a hydrophobic amino acid, preferably to isoleucine, leucine or valine or analogue or these and/or has the amino acid at the position corresponding to amino acid 356 of the Luciola firefly luciferase or 354 of Photinus pyralis changed to an amino acid other than glutamate, particularly to lysine, arginine, leucine, isoleucine or histidine or analogues or modifications of these.<br>
In a second aspect of the invention there is provided DNA encoding for the protein of the invention and in a third aspect there is provided a vector, particularly a plasmid, comprising a luc gene (the gene encoding for luciferase) in such a form as to be capable of expressing the protein of the invention. Such forms are those where the vector includes DNA sequences capable of controlling the expression of the protein of the invention such that when incorporated into a microorganism host cell the protein may readily be expressed as required, if necessary by addition of suitable inducers.<br>
The luc genes for Photinus pyralis. Luciola mingrelica. Luciola cruciata and Luciola lateralis are all known and isolatable by standard molecular biology techniques. This is also the case for Lampris noctiluca. Photinus pyralis luc gene is commercially available from Promega as the plasmid pGEM-luc. Thus convenient methods and sources for deriving starting material for production of DNA of the invention are (i) use of naturally occurring firefly genomic DNA and amplifying the luc gene from it using eg, PCR, (ii) pGEM and (iii) pGLfi? plasmid of Kajiyama and Nakano. Further genes encoding for proteins having luciferase activity, ie the activity of oxidising luciferin with the emission of light, will also be suitable sources for starting material for obtaining a DNA, and ultimately through gene expression, a protein of the invention.<br>
Suitable vectors for use in manipulating wild type or other luc gene DNA in order to produce the DNA of the invention will be any vector in which the DNA can be contained within while alteration of the naturally occurring glutamate to an alternative amino acid is carried out. For chemically induced mutagenesis, eg using agents such as hydroxylamine, this is not particularly critical and many suitable vectors will occur to those skilled in the art that will allow easy manipulation of the gene before and after the mutagenic process. It may be preferred to specifically mutate the luc gene at the glutamate and thus a site directed mutagenesis operation will be required. Such operations may be most easily carried out in vectors and these will be well known to those skilled in the art.<br>
For expression of luc genes of wild and known type, and those of the present invention suitable vectors include pKK223-3, pDR540 (available from Boehringer Mannheim) and pT7-7; the first two having the tac promoter under control of the lactose represser allowing expression to be induced by presence of isopropyl-thiogalactoside (IPTG). pT7-7 allows control by the T7-RNA polymerase promoter and thus provides the basis for a very high level of gene expression in E. coli cells containing T7 RNA polymerase. Of these vectors expression is found to be highest when the luc genes are inserted into the pT7-7 vector.<br>
Expression of luciferase from a luc gene inserted into pKK223-3 and pDR540 results in the expression of wild-type N-terminal sequence luciferase whilst expression from a luc gene inserted into pT7-7 results in synthesis of a fusion protein with extra N-terminal amino acids A-R-I-Q. The ribosome binding site and start codon of the luc gene in each of the respective vectors with the luc gene present (named constructs pPW204, pPW116 and pPW304) are shown in Table 1 of the Examples. pPW60la referred to below is derived by removing the unique Xho I site pPWl 16.<br>
A third aspect of the present invention provides cells capable of expressing the proteins of the invention; methods for producing such proteins using these cells and test kits and reagents comprising the proteins of the invention. Also provided are assay methods wherein ATP is measured using luciferin/luciferase reagents, as is well known in the art, characterised in that the luciferase is a protein of the invention. Luciferase preparations of the invention are relatively low in Km with respect to the corresponding wild type and recombinant luciferases, and preferred double and triple change luciferases (ie 215; 270; 354 changed Photinus or 217; 272; 356 changed Luciola. or 215; 270; 354 changed L. noctiluca also have the property of relative thermostability at 30-70°C, particularly 37-60°C, and especially 40-50°C. Thus the present invention has been established as not preventing the thermostability enhancements of other contemporaneous and previous work by the present inventors and others from being used.<br>
Any cell capable of expressing heterologous protein using DNA sequences in its DNA, or in vectors such as plasmids contained in the cell, may be used to express the proteins of the invention. Typical of such cells will be yeast and bacterial cells such as Saccharomyces cerevisiae and Escherichia coli cells, but many other host organisms suitable for the purpose of protein expression will occur to those skilled in the art.<br>
The protein may be expressed as a protein of similar structure to native and known recombinant luciferases, or may be expressed as a fusion or conjugate of such proteins with other amino acids, peptides, proteins or other chemical entities, eg the A-R-I-Q sequence above.<br>
It will be realised by those skilled in the art that certain hosts may have particular codon preferences, eg bacteria in some cases use different codons to yeast, and thus the DNA incorporated into such a host may advantageously be altered to provide a degenerate codon for a given amino acid that will give more favourable expression in that host. Such degenerate DNAs are of course included in the scope of the DNA of the invention.<br>
E. coli BL21 (DE3) is one suitable host and has the T7 RNA polymerase integrated stably into its chromosome under control of the inducible lacUV5 promoter and is thus compatible with pT7-7 derived constructs.<br>
E. coli B strains like BL21 lack the lon protease and the ompT outer membrane protease. These deficiencies can help to stabilise the expression and accumulation of foreign proteins in E. coli. Assays of crude extracts of E. coli BL21 (DE3) containing each of the three expression constructs described above indicated that the highest levels of expression of luciferase were obtained from cells containing the construct pPW304 (see Table 2). Other suitable cell lines, such as that of the e. coli JM109 cells used in the Examples below will occur to those skilled in the art.<br>
The proteins, DNA, vectors and cells of the invention will now be described by way of illustration only by reference to the following non-limiting Examples, Figures, Tables and Sequence listing. Further proteins,  conjugates of proteins,  DNA, vectors  and cells, and assays  and test kits incorporating any of the above will occur to those skilled in the art in the light of these.<br>
A protein having luciferase activity and having over 60% homology of amino acid sequence to luciferase from Photinus pyralis, Luciola mingrelica, Luciola cruciata or Luciola lateralis, characterized in that the amino acid residue corresponding to residue 270 of Photinus pyralis luciferase, or residue 272 of Luciola mingrelica, Luciola cruciata or Luciola lateralis luciferase is an amino acid other than glutamate, such that the Michaelis-Menten constant (Km) of the protein is decreased as compared to the corresponding wild-type sequence.<br><br>
FIGURES<br>
Figure 1: shows a restriction map of plasmid pPW204 derived from pKK223-3 by insertion of a luc gene as described in the Examples below.<br>
Figure 2: shows a restriction map of plasmid pPWl 16 derived from pDR540 by insertion of a luc gene as described in the Examples below.<br>
Figure 3: shows a restriction map of plasmid pPW304 derived from pT7-7 by insertion of a luc gene as described in the Examples below.<br>
Figure 4: shows a restriction map of plasmid pPW601a derived from pDR540 and BamHl/Sstl fragment from pGEM-luc with the Xho site removed.<br>
Figure 5: shows a graph of heat inactivation of recombinant wild-type Photinus luciferases (Sigma), K™ changed luciferase of the invention, the thermostable 354 lysine mutant provided by copending GB 9405750.2 and Km/354 lysine double mutant of the present invention incubated at a given temperature for 16 minutes as described in the Examples below.<br>
Figure 6:         shows a restriction map of pT7-7 after Tabor.<br>
SEQUENCE LISTING:<br>
The sequence listing provided at the end of this specification describes DNA and amino acid sequences as follows:<br>
SEQ ID No 1: shows the DNA sequence of a DNA encoding for luciferase of the invention wherein the Photinus pyralis wild-type codon at 811 to 813 is mutated; for lysine only the base at 811 is mutated to an A. It also shows the position for introducing thermostability at 1063-65.<br>
SEQ ID No 2: shows the amino acid sequence of a protein of the invention wherein the Photinus pyralis wild-type amino acid 270 glutamate has been changed to a residue Xaa other than glutamate.<br>
SEQ ID No 3: shows the sequence of the oligonucleotide used for the SDM mutation of pPW601a to give a lysine instead of glutamate at position 270.<br>
SEQ ID No 4: shows the amino acid sequence of a protein of the invention wherein the Photinus pyralis wild-type amino acid 270 glutamate has been changed to lysine and the 354 amino acid changed to lysine.<br>
SEQ ID No 5: shows the sequence of the oligonucleotide used for the SDM mutation of pPW601a to give a lysine instead of glutamate at position 354.<br>
EXAMPLES<br>
Example 1:      Production of plasmids containing DNA of the invention<br>
Plasmids pKK223-3 and pDR540 were obtained form Boehringer Mannheim; pDR540 is also available from Pharmacia Biotech St Albans UK. Phagemid pBluescript II SK(+) was obtained from Stratagene La Jolla, USA. E. coli strain BL21 (DE3) was used for the expression of luciferase from pT7-7 derived plasmids and E. coli strain JM109 (4) was used in all cloning experiments and for the expression of luciferase from pDR540 derived plasmids.<br>
Plasmid pT7-7 (see Current protocols in Molecular Biology Vol II Section 16.2.1) was obtained from Stan Tabor, Dept of Biol Chem, Harvard Medical School, Boston, Mass 02115 and (as shown in Figure 6) contains T7 RNA polymerase promoter 010 and the translation start site for the T7 gene 10 protein (T7 bp 22857 to 22972) inserted between the PvuII and Clal sites of pT7-5. Unique restriction sites for creation of fusion proteins (after filling in 5' ends) are Frame 0: EcoRl; Frame 1: Ndcl, Smal, Clal; Frame 2: BamHI, Sail, Hindlll. SacI site of the original polylinker is removed by deletion and an additional Xbal site is provided upstream of the start codon.<br>
As stated in the preamble to the Figures, pPW204 was derived from pKK223-3; pPW116 was derived from pDR540; pPW304 derived from pT7-7; each by insertion of a luc gene derived from Promega pGEM-/wc using standard restriction endonuclease and ligation techniques while pPW601 was created by cloning the luc gene and BamHI/Sstl fragment from pGEM-/wc into pDR540 and pPW601a as derived by removing the unique Xho I site in the polylinker of the plasmid. pPW601a contains a unique recognition site for Ava I which simplifies the SDM procedure for luciferase amino acid 354 changes.<br>
For production of pPW304, pT7-7 is digested with EcoRI, the ends filled using Klenow fragment, the product digested with Sail and the DNA gel purified; pGEM-/wc is digested with Bam HI, the overhangs produced digested with MBN, the product digested with Sail and the l.Kb fragment produced purified and ligated to the purified pT7-7 DNA.<br>
Transformation of plasmids into BMH 71-18 mut S cells was carried out using a Bio-Rad Gene Pulser version 2-89. For production of pPW601 clones harvested cells and purified mixed plasmid pool containing mutated and parental plasmids were provided and secondary restriction digest with Aval was carried out before transformation into E. coli JM109 cells. These cells were plated on selective media (LB agar + 50 ug/ml ampicillin) and clones screened by purifying their plasmid DNA and analysing for the desired change. Plasmid DNA was purified using alkaline lysis (Birnboim &amp; Doly (1979) Nucleic Acids Research 7, p!513).<br>
Relative levels of expression of luciferase from each of the constructs pPW204, pPW116 and pPW304 are 0.1: 0.5: 1.0 from E. coli BL21 (DE3). Cells were grown in LB at 37°C to an OD 600 of 0.3 then induced with IPTG and growth allowed to continue for 4 hours after which crude extract was prepared and luciferase activity measured.<br>
ABLE 1: Ribosome binding sites (underlined) and start codons in the expression constructs used in Example 1.<br>
pPW304   AAGGAGATATACAT ATG* CGT AGA ATT CAA ATG pPW116   AGGAAACAGGATCCA ATG* pPW204   AGGAAACAGCAA ATG*<br>
Partial purification of luciferases was carried out on E. coli JM109 cells harvested in the early stationary phase then resuspended in 50mM Tris HC1 pH 8.0 containing 50mM KC1, O.SmM dithiothreitol and 1 mM EDTA (Buffer A). Cells were broken up by disruption in a MSB soniprep (amplitude 14u) and the lysate centrifuged at 30000 x g for 1 hour. The supernatant of the crude extract was then subjected to fractionation with ammonium sulphate and the fraction precipitated between 35% and 55% saturation contained luciferase activity and was dissolved in Buffer A and dialysed overnight against 500ml of 50mM Tris-HCl buffer pHS.O containing 0.4mM DTT (Buffer B).<br>
Full purification of luciferases was carried out by applying the precipitated and dialysed enzyme to a Mono Q (HR10/10) anion exchange column and eluting that with a linear gradient of 0-200mM NaCl in Buffer B (flow-rate 4ml/minute; 2ml fractions). Peak fractions containing luciferase activity were made to 50% glycerol (v/v) and stored at -20°C.<br>
Firefly luciferase (prepared from a crystalline suspension, Cat No L9009) and coenzyme A and ATP were obtained from Sigma Chemical Co. Beetle luciferin potassium salt was obtained from Promega Corporation, Madison Wisconsin, USA. Cell extracts were prepared as described in the Promega technical bulletin No 101. Aliquots of E. coli cultures were lysed in cell culture lysis reagent (25mM Tris-phosphate, pH7.8, 2mM DTT, 2mM EDTA, 10% glycerol, 1% Triton X-100, 2.5 mg/ml BSA, 1.25 mg/ml lysozyme) for 10 minutes at room temperature, centrifuged at 16000g for 2 minutes and then stored on ice prior to assay.<br>
Luciferase activity of cell lines was assayed by monitoring bioluminescence emitted by colonies by transferring these to nylon filters (Hybond N, Amersham) and then soaking the filters with O.SmM luciferin in lOOmM sodium citrate buffer pH5.0 (Wood &amp; DeLuca, (1987) Anal Biochem 161 p501-507) at room temperature. Luciferase assays in vitro were performed at 21°C using lOOul of assay buffer (20mM Tricine pH7.8 containing ImM MgSO4, O.lmM EDTA, 33.3mM DTT, 0.27mM conenzyme A, 0.47mM D-luciferin, 0.53mM ATP and 1 to 5ul of sample). The final pH of the assay cocktail was 7.8 and light measurements were made with a BioOrbit 1250 luminometer or in microtitre plates using a labsystems luminoskan RS plate luminometer.<br>
Protein was determined by the method of Bradford (1976) Anal. Biochem. 72 p248-254 using BSA as standard. For production of non-specific chemical mutations of DNA, plasmids containing luc genes were treated according to the method of Kironde et al (1989) Biochem. J. 259, p421-426 using 0.8M hydroxylamine, ImM EDTA in O.lmM sodium phosphate pH6.0 for 2 hours at 65°C.<br>
The Km mutant was initially generated by hydroxylamine induced mutagenesis of the luc gene within pPW304 to provide plasmid 304 Gl bearing a single base change in the DNA sequence at 811 of SEQ ID No 1 resulting in an amino acid glutamate change to lysine at position 270. A l.lkb DNA fragment (BstE II/Stu I) was cloned from pPW304 and used to replace the corresponding fragment in pP601a to form pPW601Gl, thus providing a luc gene encoding for luciferase without the four extra amino acids encoded by pPW304 (M not included from M-A-R-I-Q).<br>
This mutagenised plasmid was desalted on a G60 DNA grade Nick column (Pharmacia) followed by transformation into E. coli BL21 (DE3). Luciferase expressed from this showed an identical low Km phenotype to that of the original mutant.<br>
Double stranded DNA sequencing was performed by the dideoxy chain termination method of Sanger et al (1977) Proc. Nat. Acad. Sci. (USA) 74, 5463-5467 using [alpha-32P]dATP and electrophoresis in 8M urea (6% wt/vol) polyacrylamide gels. Automatic sequencing was also undertaken using a DNA model 373A automated sequencer (Applied Biosystems).<br>
Assay for determining the Km value of this luciferase with respect to ATP was carried out at 21°C with lOOul of assay buffer (20mM tricine pH7.8 containing l.OmM MgSO4. 0.1 mM EDTA, 33mM dithiothrietol, 270uM coenzyme A, 470uM D-luciferin and 6.25 to 400uM ATP) using a luminometer to measure cpm.<br>
The Km value for 601a-recombinant wild-type was determined to be 66.1uM (s.e. 4.1); for 601aK (thermostable mutant 354 lysine) was 61.3 (s.e. 4.7) and for 601aGl (270 lysine Km change) was 28.7 (s.e. 0.9) thus illustrating that the 270 change more than halves the ATP concentration for which the enzyme is optimised.<br>
The effect of the 270 change on the thermostability of luciferase is negative, with tm activity being reached after only 2 minutes as compared to wild-type at 7 minutes, both at 37°C; however at 30°C the specific activity of 270 is greater than wild-type.<br>
Example 2:      Preparation of 'double mutant' 270K: 354K Photinus pyralis luciferase In order to offset the reduced thermostability of the 270 change luciferase, a double change luciferase was provided by using site directed mutagenesis to engineer the lysine change at 354 into the 270-lysine luciferases encoding DNA and plasmid described in Example 1.   This involved mutation using specifically designed oligonucleotides to convert pPW601aGl to pPW601a to G1K.<br>
The oligonucleotide used to generate the 354 lysine change by SDM was CATCCCCCTIGGGTGTAATCAG (SEQ ID No 5) with the underlined T being the mismatch.<br>
The site directed mutagenesis required to convert the glutamate 354 of pPW601aE270K, and where required for direct synthesis of 270 mutant from pPW601a, to desired amino acids is carried out using the following protocol with oligonucleotides designed as required.<br>
Site Directed Mutagenesis Protocol: Plasmid selected is denatured and annealed with selection and mutagenic oligonucleotides for the desired change. The mutant DNA strand is synthesised and ligated and the whole primary restriction digested with a restriction endonuclease. Oligonucleotide primers for sequencing and SDM were synthesised using an Applied Biosystems model 380A DNA synthesiser. DNA oligonucleotide primers were designed to destroy either a unique Ava I site within the luc gene or the unique Seal site within the gene for p-lactamase; the presence of these sites being used to select against plasmids that had not undergone mutagenesis. Precise protocols were as described in the Transformer11™ Site -Directed Mutagenesis Kit (Version 2.0) sold by Clontech Laboratories Inc (US) catalog No K1600-1.<br>
The restriction map for pPW601 derived from pDR540 and cloned luc gene is shown as Figure 4. Site directed mutagenesis was carried out as described above and in the Clontech instructions such as to convert the wild-type Photinus luc gene inserted therein into a sequence as shown in SEQ ID No 1 with expressed protein of amino acid sequence modified at position 270 as shown as Xaa in SEQ ID No 2 to Lysine.<br>
Km studies were carried out as described in Example 1 while heat inactivation studies were carried out using crude extracts at 37°C in lysis buffer (25mM Tris phosphate pH7.8, 2mM DTT, 2mM EDTA, 10% glycerol and 1% Triton X-100) at various time points aliquots of enzyme were removed and assayed as described above (with 530uM ATP). The remaining activity was plotted against time.<br>
The Km value for 601aGlK, the double change of this example, was found to be 25.2uM (s.e. 1.5) being again less than half that of the corresponding 354 lysine mutant and the wild-type luciferase.<br>
The t,/2 value, the time after which the activity of the luciferase is reduced on continuous heating to 50% of its initial value, was found to be as follows:<br>
601 a	(recombinant wild-type)	t [ /2 reached after 7.0 minutes<br>
601aGl	(270 Km change)	t1/2 reached after 1.75 minutes<br>
601aK	(354 thermostable change)	t1/2 reached after &gt;35 minutes<br>
601aGlK	(270 + 354 change)	t1/2 reached after 10.5 minutes<br>
The above data is included below (plus other data) along with Km values.<br>
Km ATP	t,/2 37°C (min)<br>
recombinant wild type	66.1	7.0<br>
E270K	28.7	1.75<br>
E354K	61.3	&gt;35<br>
E270K + E354K	25.2	10.5<br>
E270R	32.0	1.75<br>
E270Q	44.0	1.75<br>
E270A	37.0	1.75<br>
SEQUENCE LISTING<br>
(1) GENERAL INFORMATION:<br>
(i) APPLICANT: SECRETARY OF STATE FOR DEFENCE,. (ii) TITLE OF INVENTION: luciferases (iii) NUMBER OF SEQUENCES: 5<br>
(iv) CORRESPONDENCE ADDRESS:<br>
(A)	ADDRESSEE: THE SECRETARY OF STATE FOR DEFENCE<br>
(B)	STREET: WHITEHALL<br>
(C)	CITY: LONDON<br>
(D)	STATE: LONDON<br>
(E)	COUNTRY: UNITED KINGDOM (GB)<br>
(F)ZIP:SW1A2HB<br>
(v) COMPUTER READABLE FORM:<br>
(A)	MEDIUM TYPE: Floppy disk<br>
(B)	COMPUTER: IBM PC compatible<br>
(C)	OPERATING SYSTEM: PC-DOS/MS-DOS<br>
(D)	SOFTWARE: Patentln Release #1.0, Version #1.30<br>
(vi) CURRENT APPLICATION DATA:<br>
(A)	APPLICATION NUMBER: WO<br>
(B)	FILING DATE:<br>
(C)	CLASSIFICATION:<br>
(2) INFORMATION FOR SEQ ID NO:1:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 1722 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: double<br>
(D)	TOPOLOGY: unknown<br>
(ii) MOLECULE TYPE: cDNA to mRNA (iii) HYPOTHETICAL: NO (iv) ANTI-SENSE: NO<br>
(vi) ORIGINAL SOURCE:<br>
(A) ORGANISM: Photinus pyralis<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: CDS<br>
(B)	LOCATION: 4..1653<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: misc_difference<br>
(B)	LOCATION: replace(811..813,"")<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:1:<br>
CAAATGGAAG ACGCCAAAAA CATAAAGAAA GGCCCGGCGC CATTCTATCC TCTAGAGGAT  60<br>
GGAACCGCTG GAGAGCAACT GCATAAGGCT ATGAAGAGAT ACGCCCTGGT TCCTGGAACA   120<br>
ATTGCTTTTA CAGATGCACA TATCGAGGTG AACATCACGT ACGCGGAATA CTTCGAAATG   180<br>
TCCGTTCGGT TGGCAGAAGC TATGAAACGA TATGGGCTGA ATACAAATCA CAGAATCGTC   240<br>
GTATGCAGTG AAAACTCTCT TCAATTCTTT ATGCCGGTGT TGGGCGCGTT ATTTATCGGA   300<br>
GTTGCAGTTG CGCCCGCGAA CGACATTTAT AATGAACGTG AATTGCTCAA CAGTATGAAC   360<br>
ATTTCGCAGC CTACCGTAGT GTTTGTTTCC AAAAAGGGGT TGCAAAAAAT TTTGAACGTG   420<br>
CAAAAAAAAT TACCAATAAT CCAGAAAATT ATTATCATGG ATTCTAAAAC GGATTACCAG   480<br>
GGATTTCAGT CGATGTACAC GTTCGTCACA TCTCATCTAC CTCCCGGTTT TAATGAATAC   540<br>
GATTTTGTAC CAGAGTCCTT TGATCGTGAC AAAACAATTG CACTGATAAT GAATTCCTCT   600<br>
GGATCTACTG GGTTACCTAA GGGTGTGGCC CTTCCGCATA GAACTGCCTG CGTCAGATTC   660<br>
TCGCATGCCA GAGATCCTAT TTTTGGCAAT CAAATCATTC CGGATACTGC GATTTTAAGT   720<br>
GTTGTTCCAT TCCATCACGG TTTTGGAATG TTTACTACAC TCGGATATTT GATATGTGGA   780<br>
TTTCGAGTCG TCTTAATGTA TAGATTTGAA NNNGAGCTGT TTTTACGATC CCTTCAGGAT   840<br>
TACAAAATTC AAAGTGCGTT GCTAGTACCA ACCCTATTTT CATTCTTCGC CAAAAGCACT   900<br>
CTGATTGACA AATACGATTT ATCTAATTTA CACGAAATTG CTTCTGGGGG CGCACCTCTT   960<br>
TCGAAAGAAG TCGGGGAAGC GGTTGCAAAA CGCTTCCATC TTCCAGGGAT ACGACAAGGA  1020<br>
TATGGGCTCA CTGAGACTAC ATCAGCTATT CTGATTACAC CCNNNGGGGA TGATAAACCG  1080<br>
GGCGCGGTCG GTAAAGTTGT TCCATTTTTT GAAGCGAAGG TTGTGGATCT GGATACCGGG  1140<br>
AAAACGCTGG GCGTTAATCA GAGAGGCGAA TTATGTGTCA GAGGACCTAT GATTATGTCC  1200<br>
GGTTATGTAA ACAATCCGGA AGCGACCAAC GCCTTGATTG ACAAGGATGG ATGGCTACAT  1260<br>
TCTGGAGACA TAGCTTACTG GGACGAAGAC GAACACTTCT TCATAGTTGA CCGCTTGAAG  1320<br>
TCTTTAATTA AATACAAAGG ATATCAGGTG GCCCCCGCTG AATTGGAATC GATATTGTTA  1380<br>
CAACACCCCA ACATCTTCGA CGCGGGCGTG GCAGGTCTTC CCGACGATGA CGCCGGTGAA  1440<br>
CTTCCCGCCG CCGTTGTTGT TTTGGAGCAC GGAAAGACGA TGACGGAAAA AGAGATCGTG  1500<br>
GATTACGTCG CCAGTCAAGT AACAACCGCG AAAAAGTTGC GCGGAGGAGT TGTGTTTGTG  1560<br>
GACGAAGTAC CGAAAGGTCT TACCGGAAAA CTCGACGCAA GAAAAATCAG AGAGATCCTC  1620<br>
ATAAAGGCCA AGAAGGGCGG AAAGTCCAAA TTGTAAAATG TAACTGTATT CAGCGATGAC  1680<br>
GAAATTCTTA GCTATTGTAA TCCTCCGAGG CCTCGAGGTC GA	1722<br>
(2) INFORMATION FOR SEQ ID NO:2:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 550 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: unknown<br>
(ii) MOLECULE TYPE: protein (iii) HYPOTHETICAL: NO<br>
(vi) ORIGINAL SOURCE:<br>
(A) ORGANISM: Photinus pyralis<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: Modified-site<br>
(B)	LOCATION: 270<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:2:<br>
Met Glu Asp Ala Lys Asn lie Lys Lys Gly Pro Ala Pro Phe Tyr Pro<br>
15	10	15<br>
Leu Glu Asp Gly Thr Ala Gly Glu Gin Leu His Lys Ala Met Lys Arg<br>
20	25	30<br>
Tyr Ala Leu Val Pro Gly Thr He Ala Phe Thr Asp Ala His He Glu<br>
35	40	45<br>
Val Asn He Thr Tyr Ala Glu Tyr Phe Glu Met Ser Val Arg Leu Ala<br>
50	55	60<br>
Glu Ala Met Lys Arg Tyr Gly Leu Asn Thr Asn His Arg lie Val Val<br>
65	70	75	80<br>
Cys Ser Glu Asn Ser Leu Gin Phe Phe Met Pro Val Leu Gly Ala Leu<br>
85	90	95<br>
Phe lie Gly Val Ala Val Ala Pro Ala Asn Asp lie Tyr Asn Glu Arg<br>
100	105	110<br>
Glu Leu Leu Asn Ser Met Asn lie Ser Gin Pro Thr Val Val Phe Val<br>
115	120	125<br>
Ser Lys Lys Gly Leu Gin Lys lie Leu Asn Val Gin Lys Lys Leu Pro<br>
130	135	140<br>
lie lie Gin Lys lie lie lie Met Asp Ser Lys Thr Asp Tyr Gin Gly<br>
145	150	155	160<br>
Phe Gin Ser Met Tyr Thr Phe Val Thr Ser His Leu Pro Pro Gly Phe<br>
165	170	175<br>
Asn Glu Tyr Asp Phe Val Pro Glu Ser Phe Asp Arg Asp Lys Thr lie<br>
180	185	190<br>
Ala Leu lie Met Asn Ser Ser Gly Ser Thr Gly Leu Pro Lys Gly Val<br>
195	200	205<br>
Ala Leu Pro His Arg Thr Ala Cys Val Arg Phe Ser His Ala Arg Asp<br>
210	215	220<br>
Pro lie Phe Gly Asn Gin lie lie Pro Asp Thr Ala lie Leu Ser Val<br>
225	230	235	240<br><br>
Val Pro Phe His His Gly Phe Gly Met Phe Thr Thr Leu Gly Tyr Leu<br>
245	250	255<br>
lie Cys Gly Phe Arg Val Val Leu Met Tyr Arg Phe Glu Xaa Glu Leu<br>
260	265	270<br>
Phe Leu Arg Ser Leu Gin Asp Tyr Lys lie Gin Ser Ala Leu Leu Val<br>
275	280	285<br>
Pro Thr Leu Phe Ser Phe Phe Ala Lys Ser Thr Leu lie Asp Lys Tyr<br>
290	295	300<br>
Asp Leu Ser Asn Leu His Glu lie Ala Ser Gly Gly Ala Pro Leu Ser<br>
305	310	315	320<br>
Lys Glu Val Gly Glu Ala Val Ala Lys Arg Phe His Leu Pro Gly lie<br>
325	330	335<br>
Arg Gin Gly Tyr Gly Leu Thr Glu Thr Thr Ser Ala lie Leu lie Thr<br>
340	345	350<br>
Pro Glu Gly Asp Asp Lys Pro Gly Ala Val Gly Lys Val Val Pro Phe<br>
355	360	365<br>
Phe Glu Ala Lys Val Val Asp Leu Asp Thr Gly Lys Thr Leu Gly Val<br>
370	375	380<br>
Asn Gin Arg Gly Glu Leu Cys Val Arg Gly Pro Met lie Met Ser Gly<br>
385	390	395	400<br>
Tyr Val Asn Asn Pro Glu Ala Thr Asn Ala Leu lie Asp Lys Asp Gly<br>
405	410	415<br>
Trp Leu His Ser Gly Asp lie Ala Tyr Trp Asp Glu Asp Glu His Phe<br>
420	425	430<br>
Phe lie Val Asp Arg Leu Lys Ser Leu lie Lys Tyr Lys Gly Tyr Gin<br>
435	440	445<br>
Val Ala Pro Ala Glu Leu Glu Ser lie Leu Leu Gin His Pro Asn lie<br>
450	455	460<br>
Phe Asp Ala Gly Val Ala Gly Leu Pro Asp Asp Asp Ala Gly Glu Leu<br>
465	470	475	480<br>
Pro Ala Ala Val Val Val Leu Glu His Gly Lys Thr Met Thr Glu Lys<br>
485	490	495<br>
Glu lie Val Asp Tyr Val Ala Ser Gin Val Thr Thr Ala Lys Lys Leu<br>
500	505	510<br>
Arg Gly Gly Val Val Phe Val Asp Glu Val Pro Lys Gly Leu Thr Gly<br>
515	520	525<br>
Lys Leu Asp Ala Arg Lys lie Arg Glu lie Leu lie Lys Ala Lys Lys<br>
530	535	540<br>
Gly Gly Lys Ser Lys Leu<br>
545	550<br><br>
(2) INFORMATION FOR SEQ ID NO:3:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 30 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: unknown<br>
(ii) MOLECULE TYPE: DNA (genomic) (iii) HYPOTHETICAL: NO<br>
(vi) ORIGINAL SOURCE:<br>
(A) ORGANISM: Photinus pyralis<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: misc_difference<br>
(B)	LOCATION: replace(10,"")<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:3:<br>
GTATAGATTT GAAAAAGAGC TGTTTTTACG<br>
(2) INFORMATION FOR SEQ ID NO:4:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 550 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: unknown<br>
(ii) MOLECULE TYPE: protein<br><br>
(iii) HYPOTHETICAL: NO<br>
(vi) ORIGINAL SOURCE:<br>
(A) ORGANISM: Photinus pyralis<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: Modified-site<br>
(B)	LOCATION: 354<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: Modified-site<br>
(B)	LOCATION: 270<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:4:<br>
Met Glu Asp Ala Lys Asn lie Lys Lys Gly Pro Ala Pro Phe Tyr Pro<br>
15	10	15<br>
Leu Glu Asp Gly Thr Ala Gly Glu Gin Leu His Lys Ala Met Lys Arg<br>
20	25	30<br>
Tyr Ala Leu Val Pro Gly Thr lie Ala Phe Thr Asp Ala His lie Glu<br>
35	40	45<br>
Val Asn lie Thr Tyr Ala Glu Tyr Phe Glu Met Ser Val Arg Leu Ala<br>
50	55	60<br>
Glu Ala Met Lys Arg Tyr Gly Leu Asn Thr Asn His Arg lie Val Val<br>
65	70	75	80<br>
Cys Ser Glu Asn Ser Leu Gin Phe Phe Met Pro Val Leu Gly Ala Leu<br>
85	90	95<br>
Phe lie Gly Val Ala Val Ala Pro Ala Asn Asp He Tyr Asn Glu Arg<br>
100	105	110<br>
Glu Leu Leu Asn Ser Met Asn He Ser Gin Pro Thr Val Val Phe Val<br>
115	120	125<br>
Ser Lys Lys Gly Leu Gin Lys He Leu Asn Val Gin Lys Lys Leu Pro<br>
130	135	140<br>
He He Gin Lys He He He Met Asp Ser Lys Thr Asp Tyr Gin Gly<br>
145	150	155	160<br>
Phe Gin Ser Met Tyr Thr Phe Val Thr Ser His Leu Pro Pro Gly Phe<br>
165	170	175<br>
Asn Glu Tyr Asp Phe Val Pro Glu Ser Phe Asp Arg Asp Lys Thr He<br>
180	185	190<br>
Ala Leu He Met Asn Ser Ser Gly Ser Thr Gly Leu Pro Lys Gly Val<br>
195	200	205<br>
Ala Leu Pro His Arg Thr Leu Cys Val Arg Phe Ser His Ala Arg Asp<br>
210	215	220<br>
Pro He Phe Gly Asn Gin He He Pro Asp Thr Ala He Leu Ser Val<br>
225	230	235	240<br>
Val Pro Phe His His Gly Phe Gly Met Phe Thr Thr Leu Gly Tyr Leu<br>
245	250	255<br>
He Cys Gly Phe Arg Val Val Leu Met Tyr Arg Phe Glu Lys Glu Leu<br>
260	265	270<br><br>
Phe Leu Arg Ser Leu Gin Asp Tyr Lys lie Gin Ser Ala Leu Leu Val<br>
275	280	285<br>
Pro Thr Leu Phe Ser Phe Phe Ala Lys Ser Thr Leu lie Asp Lys Tyr<br>
290	295	300<br>
Asp Leu Ser Asn Leu His Glu lie Ala Ser Gly Gly Ala Pro Leu Ser<br>
305	310	315	320<br>
Lys Glu Val Gly Glu Ala Val Ala Lys Arg Phe His Leu Pro Gly lie<br>
325	330	335<br>
Arg Gin Gly Tyr Gly Leu Thr Glu Thr Thr Ser Ala lie Leu lie Thr<br>
340	345	350<br>
Pro Lys Gly Asp Asp Lys Pro Gly Ala Val Gly Lys Val Val Pro Phe<br>
355	360	365<br>
Phe Glu Ala Lys Val Val Asp Leu Asp Thr Gly Lys Thr Leu Gly Val<br>
370	375	380<br>
Asn Gin Arg Gly Glu Leu Cys Val Arg Gly Pro Met lie Met Ser Gly<br>
385	390	395	400<br>
Tyr Val Asn Asn Pro Glu Ala Thr Asn Ala Leu lie Asp Lys Asp Gly<br>
405	410	415<br>
Trp Leu His Ser Gly Asp lie Ala Tyr Trp Asp Glu Asp Glu His Phe<br>
420	425	430<br>
Phe lie Val Asp Arg Leu Lys Ser Leu lie Lys Tyr Lys Gly Tyr Gin<br>
435	440	445<br><br>
Val Ala Pro Ala Glu Leu Glu Ser lie Leu Leu Gin His Pro Asn lie<br>
450	455	460<br>
Phe Asp Ala Gly Val Ala Gly Leu Pro Asp Asp Asp Ala Gly Glu Leu<br>
465	470	475	480<br>
Pro Ala Ala Val Val Val Leu Glu His Gly Lys Thr Met Thr Glu Lys<br>
485	490	495<br>
Glu lie Val Asp Tyr Val Ala Ser Gin Val Thr Thr Ala Lys Lys Leu<br>
500	505	510<br>
Arg Gly Gly Val Val Phe Val Asp Glu Val Pro Lys Gly Leu Thr Gly<br>
515	520	525<br>
Lys Leu Asp Ala Arg Lys lie Arg Glu lie Leu lie Lys Ala Lys Lys<br>
530	535	540<br>
Gly Gly Lys Ser Lys Leu<br>
545	550<br><br>
(2) INFORMATION FOR SEQ ID NO:5:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 22 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: unknown<br>
(ii) MOLECULE TYPE: cDNA to mRNA (iii) HYPOTHETICAL: NO<br>
(vi) ORIGINAL SOURCE:<br>
(A) ORGANISM: Photinus pyralis<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: misc_difference<br>
(B)	LOCATION: replace(10,"")<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:5:<br>
CATCCCCCTT GGGTGTAATC AG<br><br><br><br>
WE CLAIM:<br>
1.	A protein having luciferase activity and having over 60% homology of amino acid sequence to luciferase from Photinus pyralis, Luciola rningrelica, Luciola cruciata or Luciola lateralis, characterized in that the amino acid residue corresponding to residue 270 of Photinus pyralis luciferase, or residue 272 of Luciola rningrelica, Luciola cruciata or Luciola lateralis luciferase is an amino acid other than glutamate, such that the Michaelis-Menten constant (Km) of the protein is decreased as compared to the corresponding wild-type sequence.<br>
2.	A protein as claimed in claim 1 wherein it comprises an amino acid sequence F(1)XE(2)FL wherein (1) is D or E, (2) is T or L and X is the amino acid residue other than glutamate, where X corresponds to residue 270 of Photinus pyralis luciferase or residue 272 of Luciola rningrelica, Luciola cruciata or Luciola lateralis luciferase.<br>
3.	A protein as claimed in claim 2 wherein the amino acid residue X is lysine.<br>
4.	A protein as claimed in any one of claims 1 to 3 wherein the amino acid residue corresponding to that of residue 215 of Photinus pyralis luciferase or 217 of Luciola rningrelica, Luciola cruciata or Luciola lateralis luciferase is a hydrophobic amino acid.<br>
5.	A protein as claimed in claim 4 wherein the amino acid residue corresponding to that of residue 215 of Photinus pyralis luciferase or residue 217 of Luciola rningrelica, Luciola cruciata or Luciola lateralis luciferase is one of isoleucine, leucine, valine or an analog of any of these.<br>
6.	A protein as claimed in any one of claims 1 to 5 wherein the amino acid residue corresponding to that of residue 354 of Photinus pyralis<br><br>
luciferase or 356  of Luciola mingrelica,  Luciola cruciata or Luciola lateralis luciferase is an amino acid other than glutamate.<br>
7.	A protein as claimed in claim 6 wherein the amino acid residue corresponding to that of residue 354 of Photinus pyralis luciferase or 356 of Luciola mingrelica, Luciola cruciata or Luciola lateralis luciferase is one of lysine, argiriine, leucine, isoleucine or histidine or an anolog of any of these.<br>
8.	A protein as claimed in claim 6 or claim 7 wherein it comprises an amino acid sequence TPXGDDKPGA wherein X is the amino acid other than glutamate which corresponds to residue 354 of Photinus pyralis luciferase or residue 356 of Luciola mingrelica, Luciola cruciata or Luciola lateralis luciferase.<br>
9.	A protein as claimed in claim 8 wherein the amino acid residue X is lysine.<br>
10.	A protein comprising an amino acid sequence as described in SEQ ID No 2 wherein Xaa is lysine.<br>
11.	A recombinant DNA encoding for a protein as claimed in any one of claims 1 to 10.<br>
12.	A recombinant DNA as claimed in claim 11 comprising a nucleotide sequence as described in SEQ ID No 1 wherein the three bases N at 811 to 813 form a codon encoding for an amino acid other than glutamate.<br>
13.	A recombinant DNA as claimed in claim 12 wherein the codon encodes for lysine.<br>
14.	A vector comprising a luc gene encoding for a protein as claimed in any one of claims 1 to 10.<br><br>
15.	A vector as claimed in claim 14 obtainable by treating a vector containing a wild-type or recombinant luc gene by site directed mutagenesis to change the codon responsible for encoding for the glutamate at position 270 of Photalis pyralis luciferase or the glutamate at position 272 of Luciola mingrelica, Luciola cruciata or Luciola lateralis luciferase to an alternative amino acid.<br>
16.	A vector as claimed in claim 15 wherein the alternative amino acid is lysine.<br>
17.	A vector as claimed in any one of claims 14 to 16 selected from pKK223-3, pDR540 and pT7-7 into which a luc gene has been ligated.<br>
18.	A cell capable of expressing a heterologous protein as claimed in any one of claims 1 to 10 comprising DNA or a vector as claimed in any one of claims to 11 to 17.<br>
19.	A cell as claimed in claim 18 being an E. coli or a S. cerevisiae.<br>
20.	A test kit for performance of an assay through measurement of ATP wherein the kit comprises a protein as claimed in any one of claims 1 to 10 contained within a luminescent reagent.<br>
21.	An assay method wherein ATP is measured using luciferin and luciferase to generate light the quantity of which is related to the amount of ATP wherein the luciferase is a protein as claimed in any one of claims 1 to 10.<br>
22.	An assay method as claimed in claim 21 wherein the assay is carried out at a temperature of from 30°C to 70°C.<br>
23.	An assay method as claimed in claim 21 wherein the assay is carried out at a temperature of from 37°C to 60°C.<br><br><br>
24.	An assay method as claimed in claim 21 wherein the assay is carried out at a temperature of from 40°C to 50°C.<br>
25.	A protein comprising an amino acid sequence as described in SEQ ID No 2 wherein Xaa is chosen from arginine, glutamine and alanine.<br>
26.	A protein having luciferase activity substantially as herein described with reference to and as illustrated in the accompanying drawings.<br><br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLURFTC0yMDAwLUFic3RyYWN0LSgxOS0wOC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">183-DEL-2000-Abstract-(19-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLURFTC0yMDAwLUFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">183-DEL-2000-Abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLURFTC0yMDAwLUNsYWltcy0oMTktMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">183-DEL-2000-Claims-(19-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLURFTC0yMDAwLUNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">183-DEL-2000-Claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLURFTC0yMDAwLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTktMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">183-DEL-2000-Correspondence-Others-(19-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLURFTC0yMDAwLUNvcnJlc3BvbmRlbmNlLU90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">183-DEL-2000-Correspondence-Others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLWRlbC0yMDAwLWNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">183-del-2000-correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLURFTC0yMDAwLURlc2NyaXB0aW9uIChDb21wbGV0ZSktMTktMDgtMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">183-DEL-2000-Description (Complete)-19-08-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLURFTC0yMDAwLURlc2NyaXB0aW9uIChDb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">183-DEL-2000-Description (Complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLURFTC0yMDAwLURyYXdpbmdzLSgxOS0wOC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">183-DEL-2000-Drawings-(19-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLURFTC0yMDAwLURyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">183-DEL-2000-Drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLURFTC0yMDAwLUZvcm0tMS0oMTktMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">183-DEL-2000-Form-1-(19-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLURFTC0yMDAwLUZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">183-DEL-2000-Form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLURFTC0yMDAwLUZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">183-DEL-2000-Form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLURFTC0yMDAwLUZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">183-DEL-2000-Form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLURFTC0yMDAwLUZvcm0tMi0oMTktMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">183-DEL-2000-Form-2-(19-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLURFTC0yMDAwLUZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">183-DEL-2000-Form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLURFTC0yMDAwLUZvcm0tMy0oMTktMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">183-DEL-2000-Form-3-(19-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLURFTC0yMDAwLUZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">183-DEL-2000-Form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLURFTC0yMDAwLUZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">183-DEL-2000-Form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLWRlbC0yMDAwLWZvcm0tNi0oMTYtMDEtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">183-del-2000-form-6-(16-01-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLURFTC0yMDAwLUdQQS5wZGY=" target="_blank" style="word-wrap:break-word;">183-DEL-2000-GPA.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLURFTC0yMDAwLVBldGl0aW9uLTEzNy0oMTktMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">183-DEL-2000-Petition-137-(19-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLURFTC0yMDAwLVBldGl0aW9uLTEzOC0oMTktMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">183-DEL-2000-Petition-138-(19-08-2008).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="225385-cosmetic-compositions-containing-nigrospora-sphaerica-extract.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="225387-a-process-for-the-isolation-and-purification-of-sesquiterpene-compounds-from-a-curcuma-plant.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>225386</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>183/DEL/2000</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>48/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>28-Nov-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>11-Nov-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>03-Mar-2000</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>3M INNOVATIVE PROPERTIES COMPANY</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>3M CENTER, POST OFFICE BOX 33427,SAINT PAUL, MINNESOTA 55133-3427,USA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>DAVID JAMES SQUIRRELL</td>
											<td>C/O CBDE OF PORTON DOWN, SALISBURY, WILTSHIRE SP4 0JQ, ENGLAND</td>
										</tr>
										<tr>
											<td>2</td>
											<td>CHRISTOPHER ROBIN LOWE</td>
											<td>C/O UNIVERSITY OF CAMBRIDGE, TENNIS COURT ROAD, CAMBRIDGE , CAMBRIDGESHIRE CB2 1QT U.K.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>PETER JOHN WHITE</td>
											<td>C/O UNIVERSITY OF CAMBRIDGE, TENNIS COURT ROAD, CAMBRIDGE , CAMBRIDGESHIRE CB2 1QT UK</td>
										</tr>
										<tr>
											<td>4</td>
											<td>JAMES AUGUSTUS HENRY MURRAY</td>
											<td>C/O UNIVERSITY OF CAMBRIDGE, TENNIS COURT ROAD, CAMBRIDGE , CAMBRIDGESHIRE CB2 1QT UK</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 9/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>9508301.0</td>
									<td>1995-04-24</td>
								    <td>U.K.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>9501172.2</td>
									<td>1995-01-20</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/225386-a-protein-having-luciferase-activity by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:28:21 GMT -->
</html>
