<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/225528-electrochemiluminescence-helicase-assay by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:12:19 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 225528:ELECTROCHEMILUMINESCENCE HELICASE ASSAY</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">ELECTROCHEMILUMINESCENCE HELICASE ASSAY</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>ABSTRACT lN/PCT/2002/0488/CHE The present invention provides a helicase assay that is rapid, sensitive and suitable for high throughput screening for helicase activity in general and especially for potential helicase inhibitors.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
ELECTROCHEMILUMINTSCENCE HELICASE ASSAY<br>
FIELD OF THE INVE&gt;mQN<br>
The present invention relates to assays which use electrochemiluminescence for the detection of helicase activity. Therefore, the invention provides a helicase assay that is rapid, sensitive and suitable for high throughput screening for helicase activity in general, and especially for potential helicase inhibitors. Helicase inhibitors represent a class of pharmacological agents useful for the treatment of disease.<br>
B.A.CKGROUND OF THE INVENTION<br>
DNA Helicases<br>
DNA helicases are enzymes that are involved in all aspects of nucleic acid metabolism (Lohman and Bjomson, Annu. Rev. Biochem., 1996, 65, 169-214; Matson, S- W., Progress in Nuclear Acid Research and Molecular Biology, 1991, 40, 289-326). The stable form of most DNA in vivo is double stranded helical DNA. Single stranded DNAs are required for DNA replication, repair, recombination and conjugation. In order to form single stranded DNA from a double strand, the DNA duplex must be at least partially unwound and separated. Unwinding of stable duplex DNAs is catalyzed by DNA helicases using the energy derived from the hydrolysis of nucleoside 5'-tiiphosphates (NTP). Helicase enzymes translocate along DNA to unwind (unhybridize) the duplex stands at a rate that can be as fast as 500-1000 bp/s.<br>
DN.\ helicases have been identified in various prokaryoles, eukaryotes, bacteriophages and viruses. Most organisms encode muhiple helicases. For example, E.coli encodes at least 12 different helicases (Matson, et al., BioEssays, 1994, 16, 13-22) and S. cerevisiae encodes at least six helicase forms (Li, et al., Chromosoma. 1992, }Q2, S93-S99). Although these DNA helicases play many different fimctions in DNA metabolism, it is now clear that helicase activity is the function ascribed to unwinding the DNA duplex. Due to the essentia! function of helicases for cellular<br><br>
metabohsm, these enzymes represent important targets for developing therapeutic agents.<br>
Hehcase Assays<br>
Several types of assays have been developed to measure the unwinding of duplex nucleic acids by helicases. One such assay measures the sensitization of labeled duplex DNA to single-strand specific nucleases, such as SI or exonuclease I, resulting in the production of ssDNA during unwinding (Palas et al., J. Biol. Chem., 1990 265:3447). Electron microscopy has also been employed to visualize directly regions of DNA unwound by proteins such as recBCD enzyme, rep protein, E. coli helicases I and II, and SV40 T antigen (Runyon et al., Proc. Natl. Acad. Set.. 1990, 87:6383).<br>
The most common assay for determining helicase activity in vitro utilises electrophoresis (V'enkatesanet al, J. Biol. Chem.. 1982, 257, 12426; Matson, et al., / Biol. Chem.. 1983,258,14017). Helicase enzyme unwinds duplex DNA yielding single stranded DNAs. The products of this reaction are resolved on a polyactylamide or agarose gel. A radioactive label within the DNA pemiits direct visuaUzation and quantitation of the results.<br>
A continuous spectrophotometric assay developed for studying tecBCD enzyme helicase activity utilizes a ssDNA binding protein, either E. coli SSB protein (single-strand DNA binding protein) or phage T4 gene 32 protein, as the reporter molecule (Roman et al.. Biochemistry, 1989, 28:2863). As dsDNA is unwound, the SSB protein binds to the ssDNA fomied, resulting in quenching of its intrinsic fluorescence. Additional helicase assays are described in US Pats. 4,568,649, 5,705,344 and 5,747,247.<br>
Each of these assays described above have certain limitations relating to sensitivity, number of steps required to perform the assay, and/or the limited number of samples that can be tested. Obviously, such methods are not readily adaptable for highly sensitive, high-throughpat drug screening formats. Therefore, there is a need in the art for a simple, yet sensitive helicase assay that can be performed in high throughput. The present invention overcomes these obstacles, and provides a helicase<br><br>
assay that is rapid, sensitive and suitable for a high throughput screening for potential helicase inhibitors.<br>
Elecn-Qchemiluminescence Assays<br>
Electrochemiluminecence is hght generated when a low voltage is applied to an electrode, thereby triggering a cychc oxidation and reduction reaction of ruthenium metal ion bound in a chelate of tris-(bipyridine). This technology has broad applicability to detection of microbiological, biochemical and chemical analytes (Bruno et al., 1997, Rec. Res. Devel. In Micro. 1:25).<br>
Electrochemiluminescent labels for DNA have been developed (Kenten et al., 1991, Clin. Chem. 37:1626; Kenten et al., 1992, Clin. Chem. 3S:873; Origen® label, Origen® Phosphoramidite and TAG Phosphoramidite available from IGEN Inc., Rockville, MD), and used for detection of polymerase chain reaction products (Schutzbank et al., 1995,7. Clin. Microbiol.. 33:2023). These assays have been shown to be rapid and effective for the detection of amplified PCR products.<br>
The citation of any reference herein should not be construed as an admission that such reference is available as "Prior Art" to the instant application.<br>
SUMMARY OF THE liWENTlON<br>
The present invention provides a helicase assay that is rapid, sensitive and suitable for high throughput screening for potential helicase inhibitors. Hehcase inhibitors represent a class of pharmacological agents useful for the treatment of disease.<br>
Therefore, in a first embodiment, the present invention provides an elecU"ochemiluminescent assay method for detecting helicase activity in a sample, the raelhod comprising<br>
(a)       combining the sample with a first nucleic acid or nucleic acid molecule comprising an electrochemiluminescent label and hybridized to a complementary second nucleic acid or nucleic acid molecule;<br><br>
(b)	incubating the sample and hybridized nucleic acids or nucleic acid molecules from step (a) under conditions where helicase present in the sample can unhybridize the first and second nucleic acids or nucleic acid molecules;<br>
(c)	capturing unhybridized first nucleic acid or nucleic acid molecule; and<br>
(d)	measuring electrochemiluminescence from the captured first nucleic acid or nucleic acid molecule.<br>
The helicase may be a human or pathogenic helicase. Preferably, the helicase is a lungal, viral, bacterial or parasitic helicase.<br>
Any method for capturing unhybridized first nucleic acid or nucleic acid molecule maybe employed. In a preferred series of steps, between steps (b) and (c) above, the unhybridized nucleic acids or nucleic acid molecules are incubated with a third nucleic acid or nucleic acid molecule compiemeniary to the first nucleic acid or nucleic acid molecule and comprising a capturing ligand, under conditions where the first and third nucleic acids or nucleic acid molecules can hybridize. Thereafter, the h&gt;bridized first and third nucleic acids or nucleic acid molecules may be captured with a capturing receptor which binds the capturing ligand. The capturing receptor may be on the surface of magnetic beads. In a prefened embodiment, the capturing iigand is biotin and the capturing receptor is avidin. The preferred electrochemiluminescent label is a ruthenium chelate.<br>
In another embodiment, the present invention relates to a method for detecting in a sample an agent that modulates helicase activit&gt;' by<br>
(a)	combining the sample, a helicase and a first nucleic acid or nucleic acid molecule comprising an elecirochemiluminescenl label and hybridized to a complementary second nucleic acid or nucleic acid molecule, thereby forming a mixture;<br>
(b)	incubating the mixture from step (a) under conditions where the helicase can unhybridize the first and second nucleic acids or nucleic acid molecules in the absence of the sample;<br>
(c)	capturing unhybridized first nucleic acid or nucleic acid molecule; and<br><br>
(d)       measuring electrochemiluminescence fi-om the captured first nucleic acid or nucleic acid molecule. In a preferred embodiment, the agent is an inhibitor of tielicase activity. The helicase may be a human or pathogenic helicase. Preferably, the helicase is a fijngal, viral, bacterial or parasitic helicase.<br>
Any method for capturing unhybndized first nucleic acid or nucleic acid molecule may be employed. In a preferred series of steps, between steps (b) and (c) above, the unhybndized nucleic acids or nucleic acid molecules are incubated with a third nucleic acid or nucleic acid molecule complementary to the first nucleic acid or nucleic acid molecule and comprising a capturing ligand, under conditions where the first and third nucleic acids or nucleic acid molecules can hybridize. Thereafter, the hybridized first and third nucleic acids or nucleic acid molecules may be captured with a capturing receptor which binds the capturing ligand. The capturing receptor may be on the surface of magnetic beads. In a preferred embodiment, the capturing ligand is biotin and the capturing receptor is avidin. The preferred eiectrochemiluminescent label is a ruthenium chelate.<br>
In still another embodiment, the invention provides kits for eiectrochemiluminescent detection of helicase activity. Such kits comprise<br>
(a)	a first nucleic acid or nucleic acid molecule comprising an eiectrochemiluminescent label;<br>
(b)	a second nucleic acid or nucleic acid molecule complementary to the first nucleic acid or nucleic acid molecule;<br>
(c)	a hehcase and'or magnetic beads suitable for facilitating electrochemiluminescence detection.<br>
In one aspect, the first nucleic acid or nucleic acid molecule is already hybridized to the second nucleic acid or nucleic acid molecule.      The kits according to the invention may lunher comprise a third nucleic acid or nucleic acid molecule complementary to the first nucleic acid or nucleic acid molecule and comprising a capturing ligand. Under such circumstances, the kit will, in general, include a capturing receptor capable of binding the capturing ligand. In a prefen^ed<br><br>
embodiment, the capturing receptor is on the surface of magnetic beads. In a most preferred embodiment, the capturing ligand is biotin and the capturing receptor is avidin.<br>
BRIEF DESCRIPTION OF THE FIGURES<br>
Figure I:        Radiometric quantitation of £". coli. DnaB helicase activity.<br>
A 60 bp oligonucleotide was labeled with 'P at 3' end. Then a complementary 60 bp oligonucleotide was h&gt;'bridized with the labeled nucleotide. Various amounts of DnaB helicase as indicated were added, and the reaction mixtures were incubated at 37°C for 30 min. EDTA was added to stop the reaction, and the samples were separated on a PAGE gel and detected by autoradiography. Lane 8 contains a boiled sample to show separation of ssDNA and ds DNA. Lane 9 contains only labeled ssDNA.<br>
Figure 2:        Schematic of the ScintillationProximity Assay (SPA) for quantitation of helicase activity using'H-radiolabled oligonucleotides and M13 ssDNA.<br>
Figure 3:        Unwinding of ■^H - substrates using varying amount of DnaB helicase. The coimts per rainute(cpm) is plotted as function of the amount of DnaB hehcase added in each reaction.<br>
Figure 4:        Bar graph of DnaB helicase activity using SPA assay.<br>
Figure 5:        Schematic of Time Resolved Flourecence Quenching Assay for quantitation of helicase acvivity using Eu-radiolabeled oligonucleotides and a Rhodamine labeled complementary oligonucleotides.<br>
Figure 6: Unv-inding of hybridized substrates using varying amounts of DnaB helicase. The amount of Fluorescence released from unwinding dsDNA is plotted zs function of the amount of DnaB helicase added in each reaction.<br><br>
Figure 7:        Schematic of electrochemiluminescence (ECL) assay for quantitation of helicase activity using Ruthenium labeled oligonucleotides and M13 ssDNA.<br>
Figure 8:        Electrochemiluminesent signal generated by unwinding double stranded substrates using varying amounts of DnaB helicase. The ECL signal is plotted a function of the amount of helicase added to the reaction.<br>
DETAILED DESCRIPTION OF THE INVTINTION<br>
The present invention advantageously provides an electrochemiluminescent assay method for detecting helicase activity, and a method for identifying modulators of helicase activity. Therefore, the present invention provides a rapid and sensitive assay for potential helicase inhibitors that is suitable for high throughput screening. The invention also provides kits for use in identifying helicase activity, and screening for potential modulators of helicase activity. Such kits include premeasured amounts of the components used in the disclosed methods.<br>
Since helicases are necessary for a wide variety of cellular fiinctions including growth, target diseases are limited only in that disease or disease progression be subject to inhibition by modulation ofthe activity of one or more specific helicases. As such, target diseases include viral, bacterial and fungal infections, metabolic disease, genetic disease, cell growih and regulatory disfunction, such as neoplasia, inflammation, h&gt;'persensitivity, etc. Tne target diseases may be afflictions of plants, especially agriculmral crops, or animals, especially livestock, domestic animals and humans.<br>
The various aspects of the invention will be sst forth in greater detail in the following sections. This organization into various sections is intended to facilitate understanding ofthe invention, and is in no way intended to be limiting thereof.<br>
Definitions<br>
The following defined temis are used throughout the present specification, and<br><br>
should be helpful in understanding the scope and practice of the present invention.<br>
In a specific embodiment, the term "about" or "approximately" means wirhin 20%, preferably within 10%, and more preferably within 5% of a given value or range.<br>
As used herein, the articles "a" and "an" mean one or more; hence, "a helicase" is meant to include assays involving one of more heUcases, such as simultaneous screens for inhibitors of several helicases.<br>
A "nucleic acid" is a polymeric compound comprised of covalently linked subunits called nucleotides. Nucleic acid includes polyribonucleic acid (RNA) and polydeoxyribonucleic acid (DNA), both of which may be single-stranded or double-stranded. DNA includes cDNA, genomic DNA, synthetic DNA, and semi-synthetic DNA.<br>
A "nucleic acid molecule" refers to the phosphate ester polymeric fomi of ribonucleosides (adenosine, guanosine, uridine or cyiidine; "RNA molecules") or deoxyribonucleosides (deoxyadenosine, deoxyguanosine, deoxythymidine, or deoxycytidine; "DNA molecules"), or any phosphoester anologs thereof, such as phosphorothioates and thioesiers, in either single stranded fomi, or a double-stranded helix. Double stranded DNA-DN.\, DNA-RNA and RNA-RNA helices are possible. The term nucleic acid molecule, and in particular DNA or RNA molecule, refers only to the primary and secondary structure of the molecule, and does not limit it to any particular tertiary forms. Thus, this term includes double-stranded DNA found, inier alia, in linear or circular DNA molecules (e.g., restriction fragments), plasmids, and chromosomes. In discussing the sirucmre of particular double-stranded DNA molecules, sequences may be described herein according to the normal convention of eiving only the sequence in the 5' to 3' direction along the nontranscribed strand of DNA (i.e., the strand having a sequence homologous to the mRNA). A "recombinant DNA molecule" is a DNA molecule that has undergone a molecular biological manipulation.<br>
A nucleic acid molecule is "hybridizable" to another nucleic acid molecule, such as an oligonucleotide, a cDNA, genomic DNA, or RNA, when a single stranded fomi of the nucleic acid molecule can anneal to the other nucleic acid molecule under<br><br>
the appropriate conditions of temperature and solution ionic strength. The conditions of temperature and ionic strength determine the "stringency" of the hybridization. Low stringency hybridization conditions correspond to aim of 55", e.g., 5x SSC, 0.1% SDS, 0.25% milk, and no formamide; or 30% formamide, 5x SSC, 0.5% SDS). Moderate stringency hybridization conditions correspond to a higher Xm, e.g., 40% formamide, with 5x or 6x SCC. High stringency hybridization conditions correspond to the highest Tm, e.g., 50% formamide, 5x or 6x SCC. Hybridization requires that the two nucleic acids contain complementary sequences, although depending on the stringency of the hybridization, mismatches between bases are possible. The appropriate stringency for hybridizing nucleic acids depends on the length of the nucleic acids and the degree of complementation, variables well known in the art. The greater the degree of similarity or homology between two nucleotide sequences, the greater the value of Tn, for hybrids of nucleic acids having those sequences. The relative stability (corresponding to higher Tm) of nucleic acid hybridizations decreases in the following order: RNA:RNA, DNA:RNA, DNA;DNA. For hybrids of greater than 100 nucleotides in length, equations for calculating Tm have been derived (see Sambrook., Fritsch &amp; Maniatis, Molecular Cloning: A Laboratory Manual, Second Edition (1989) Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York, 9.50-0.51). For hybridization with shorter nucleic acids, i.e., oligonucleotides, the position of mismatches becomes more important, and the length of the oligonucleotide determines its specificity (see Sambrook et al., 11.7-1 l.S). Preferably a minimum length for a hybridizabie nucleic acid is at least about 10 nucleotides; preferably at least about 15 nucleotides; and more preferably the length is at least about 20 nucleotides.<br>
As used herein, the term "oligonucleotide" refers to a nucleic acid, generally of at least ! 8 nucleotides, that is hybridizabie to a complementary nucleic acid. Ohgonucleotides can be labeled, e.g., with """P-nucieotides or nucleotides to which a label, such as biotin or ruthenium chelate, has been covalently conjugated. Generally, oligonucleotides are prepared synthetically, preferably on a nucleic acid synthesize:. Accordingly, oligonucleotides can be prepared with non-naturally occurring phosphoester analog bonds, such as thioester bonds, etc.<br><br>
"Avidin" is a meant to include any protein or polypeptide capable of high affinity binding to biolin. Streptavidin is one such example of avidin.<br>
A "helicase" is any protein or polypeptide having the ability to unvi'ind (unhybridize) a duplex of complementary nucleic acids or nucleic acid molecules.<br>
Helicase Assays<br>
The methods defined by the present invention involve forming a mixture of a first nucleic acid or nucleic acid molecule comprising an e!ecD"ochemiluminescent label and hybridized to a complementary second nucleic acid or nucleic acid molecule. The hybridized first and second nucleic acids or nucleic acid molecules may be RNA or DMA, linear or circular, depending on the specificity of the targeted helicase. In addition, other nucleic acids or nucleic acid molecules or structural analogs may be substituted so long as they provide an active substrate for the targeted helicase activity. The nucleic acids or nucleic acid molecules may be of any length amenable to the assay conditions and requirements. For example, ensuring helicase substrate specificity and minimizing non-specific renaniration requires a minimal region of complementarity between the first and second nucleic acids or nucleic acid molecules, typically at least about 12, more typically at least about 18 and preferably at least about 24 continuous base pairs- There may be a size differential between the strands. The sizes are selected to facilitate the assay and to provide statistically significant assay results; frequently the sizes will differ by at least one or two orders of magnitude. In general, optimal lengths are readily determined empirically.<br>
The nucleic acids or nucleic acid molecules may be of any sequence which provides a convenient substrate for the targeted helica5s(3). The nucleic acids or nucleic acid molecules may be complementary over the entire length of at least one of the nucleic acids or nucleic acid molecules or there may be regions of noncomplementary 5' and-'or 3' of the complementary region. Introducing these 5' and'or 3' noncomplementary regions provides molecular forks that yield better substrates for some helicases. Generally, conveniently replicated vectors e.g. phage, or restriction fragments thereof, provide an inexpensive source of the nucleic acids or<br><br>
nucleic acid molecules. The assays are generally compatible with the presence of DNA binding proteins, such as histones. It is often advantageous to include a variety of potential substrates, e.g. double-stranded nucleic acids of varied size, sequence, protein complexing, etc. to improve the likelihood of detecting substrate-sensitive helicsses.<br>
The first nucleic acid or nucleic acid molecule comprises a detectable electrochemiluminescent label. Electrochemiluminecence is light generated when a low voltage is applied to an electrode, thereby triggering a cyclic oxidation and reduction reacrion of the label. Any electrochemiluminescent label capable of such cyclic oxidaiion/reduction and light generation maybe used according to the present invention. The preferred label is ruthenium metal ion bound in a chelate of tris-(bipyridine). Specific eSectrochemiluminescent labels for DNA have been developed (Kenten et al., 1991, Clin. Chem. 37:1626; Kenten el al., 1992, Clin. Chem. 38:873; Origen® label, Origen® phosphoramidite and TAG phosphoramidite, IGEN Inc., Rockviile, MD) and are most preferred for the present invention.<br>
Any pathogenic heiicase (i.e. any helicase capable of acting harmfully to a host ceil or organism) may be analyzed according to the present invention. Rapidly growing cells (e.g. in neoplasia) may be targeted by inhibitors of human helicases, especially replicative helicases. In addition, pathogen-selective or -specific hehcases are used to identify pharmacological therapeutics for the treatment of infectious disease. Fungal, viral, bacterial and parasitic helicases, in particular, provide medically urgent targets for identifying inhibitors by the present methods. Alternatively, a plurality of helicases or panel comprising a preselected range of different helicases can be used to maximize the scope of the assay.<br>
Preferred pathogenic helicases are derived from medically significant infectious fungi, such as Aspergillus, Candida species; bacteria such as Staphylococci (e.g aureus). Streptococci {e.g. pneumoniae), Clostridia (e.g. perfHngens), Neisseria (e.g gonorrhoeae), Enterobacteriaceae (e.g. coli), Helicobacter (e.g pylori). Vibrio (e.g. cholerae), Capylobacter (e.g. jejuni), Pseudomonas (e.g aeruginosa), Haemophilus (e.g. influenzae), Bordetella (e.g. pertussis). Mycoplasma (e.g. pneumoniae), Ureaplasma (e.g. uiealyticum), Legionella (e.g. pneumophila).<br><br>
Spirochetes (e.g. Treponema, Leptospira and Borrelia), Mycobacteria (e.g. tuberculosis, smegmatis), Actinomycies {e.g. (israelii), Nocardia (e.g. asteroides). Chlamydia (e.g. trachomatis), Rickensia, Coxiella, Ehriiichia, Rochalimaea- Bnicella, Yersinia, Fracisella, and Pasteurella; protozoa such as sporozoa (e.g. Plasmodia), rhizopods (e.g. Entamoeba) and flagellates (Tr\panosoma, Leishmania, Trichomonas Giardia, etc.); and viruses such as (+) RNA viruses (examples include Picomavinises, e.g. polio; Togaviruses, e.g. rubella; Flavivimses, e.g. HCV; and Coronavimses), (-) RN'A viruses (examples include Rhabdoviruses, e.g. VSV; Paramyxovimses, e.g. RSV; Orthomyxovimses, e.g. influenza; Bunyaviruses and Arenaviruses), dsDNA viruses (Reoviruses, for example), RNA to DNA viruses, i.e. Retroviruses, e.g. HI\', and certain DNA to RNA viruses such as Hepatitis B virus.<br>
The helicase may be purified from a naniral source or may be recombinant. The helicase is usually provided in at least a partially-purified form. Only a portion of the helicase sufficient for helicase activity may be used in the present assay. Portions capable of imparting the requisite binding specificity and affmiiy are readily identified and produced by those skilled in the art using general molecular and biochemical methods, see e.g. Molecular Cloning, A Laboratory Manual (2nd Ed., Sambrook, Fritsch and Maniatis, Cold Spring Harbor), Current Protocols in Molecular Biology (Eds. Aufubel, Brent, Kingston, More, Feidman, Smith and Smhl, Greene Publ. Assoc, Wiley-Interscience, NY, N.Y., 1992) or that are othenvise known in the art.<br>
The methods of the present invention require capturing unhybridized first nucleic acid or nucleic acid molecule produced by helicase activity present in the assay. Any method for capturing unhybridized first nucleic acid or nucleic acid molecuie may be employed. In a preferred series of steps unhybridized first and second nucleic acids or nucleic acid molecules are incubated with a third nucleic acid or nucleic acid molecule complementary to the first nucleic acid or nucleic acid molecule and comprising a capturing ligand, under conditions where the first and third nucleic acids or nucleic acid molecules can hybridize. Thereafter, the hybridized first and third nucleic acids or nucleic acid molecules may be captured with a capturing receptor which binds the capturing ligand. The capturing receptor may be on the surface of maanetic beads in order to directlv facilitate electrochemiluminescence<br><br>
detection.<br>
The assay process of the present invention maybe utilized in conjunction with any capturing ligand-receptor combination or system that specifically captitres the first nucleic acid or nucleic acid molecule without affecting electrochemiluminescence detection- Examples of suitable Hgand-receptor combinations include antibodies and their corresponding antigens. .Another ligand-receptor system may be composed of protein A and inimunogiobulins, or an Fc portion thereof. Additional capturing systems with which the present invention may be used include: (1) lectins-oligosaccharides or glycoproteins; (2) biotin-avidin; (3) hormone receptor-hormone; (4) enzyme-substrate or cofactor; (5) KNA-DNA; and (6) DNA-DNA. It is to be understood that in the present invention either element may serve as the Iigand or receptor- In a preferred embodiment, the capturing iigand is biotin and the capturing receptor is avidin. The preparation of biotinylated oligonucleotides and of avidin-coated magnetic beads is described in Kenten et al., 1991 {Clin, Chem. 37;I626) and Kenten et al., 1992 iClin. Chem. 38:873).<br>
The present invention is particularly suited to screening candidate agents for the ability to modulate heiicase activity, especially inhibitory activity. The invention is suitable for high-throughput drug screening of a library of candidate agents. Library-derived candidate agents encompass numerous chemical classes, though typically they are organic compounds; preferably small organic compounds. The libraries may comprise synthetic and/or naturally derived compounds. For example, numerous means are available for random anddirected synthesis of a wide variety of organic compounds and biomolecules, including expression of randomized oligonucleotides. Alternatively, libraries of natural compounds in the form of bacterial, fungal, plant and animal extracts are available or readily produced. Additionally, natural and svnthetically produced libraries and compounds are readily modified through conventional chemical, physical, and biochemical means. In addition, kno^vn pharmacological agents may be subject to directed or random chemical modifications, such as acyiation, alkylation, esierification. amidification, etc., to produce structural analogs. The agent is provided in standard serial dilutions or in an amount determined by analogy to known modulators.<br><br>
Typically, the present assays usually include additional reagents, such as salts, buffers, etc. to facilitate or maximize helicase activity. Also, reagents that reduce non¬specific or background denaturation or otherwise improve the efficiency of the assay, such as protease inhibitors, nuclease inhibitors, antimicrobial agents, single-stranded DNA binding protein, etc. may be used.<br>
The present invention may be better understood by reference to the following non-limiting Examples, which are provided as exemplar^' of the invention.<br>
EX.\MPLES<br>
General molecular biology techniques<br>
The methods traditionally used in molecular biology, such as preparative extractions of plasmid DNA, centrifugation of plasmid DNA in a caesium chloride gradient, agarose or acrylamide gel electrophoresis, purification of DNA fragments by electroelution, protein extraction with phenol or phenol/chloroform, ethanol or isopropanol precipitation of DNA in a saline medium, transformation in Escherichia coll, and the like, are well known to a person skilled in the art and are amply described in the literature [Manialis T. et al., "Molecular Cloning, a Laboratory Manual", Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y., 19S2; (2"'^ Ed. 1989); Ausubel F.M. et a], (eds), "Current Protocols in Molecular Biology", John Wiley &amp; Sons, New York, 19S7].<br>
Conventional cloning vehicles include pBR322 and pUC type plasmids and phages of the M13 series. These may be obtained commercially (Bethesda Research Laboratories or New England Biolabs).<br>
For ligation. DNA fragments may be separated according to their size by agarose or acrylamide gel electrophoresis, extracted with phenol or with a phenoLchloroform mixture, precipitated with ethanoi and then incubated in the presence of phage T4 DN.\ ligase (Biolabs) according to the supplier's recommendations.<br>
The filling in of 5' protruding ends may be performed with the Klenow<br><br>
fragment of E. coli DNA polymerase 1 (Biolabs) according to the supplier's specifications. The destruction of 3' protruding ends is performed in the presence of phage T4 DNA polymerase (Biolabs) used according to the manufactiu-er's lecotmnendations. The destruction of 5' protruding ends is performed by a controlled treatment with S1 nuclease.<br>
Mutagenesis directed in vitro by synthetic oiigodeoxynucleotides may be performed according to the method developed by Taylor et al. [Nucleic Acids Res. Jj (1985) 8749-S764] using the kit distributed by Amersham.<br>
The enzymatic amplification of DNA fragments by PCR [Polymerase-catalyzed Chain Reaction, Saiki R.K. et al.. Science 230 (1985} 1350-1354; Mullis K.B. and Faloona F.A., Meth. Enzym. 155 (19S7) 335-350] technique may be perforaied using a "DNA thermal cycler" (Perkin Elmer Cetus) according to the manufacturer's specifications.<br>
Verification of nucleotide sequences may be performed by the method developed by Sanger et al. [Proc. Natl. Acad. Sci. USA, 74 (1977) 5463-5467] using the kit distributed by Amersham.<br>
Plasraid DNAs may be purified by the Qiagen Plasmid Purification System according to the manufacture's instruction.<br>
Example 1: Traditional helicase assav<br>
A 60 bp oligonucleotide (5'-TTT TTT TTT TTT TTT TTT TTT TTT TTT TTT GTT A.'^A GGC CGC TTT TGC GOG ATC GTC ACC-3'; SEQ ID NO: l) is labeled with ^^P using T4-polynucleotide kinase and then hybridized to a complementary unlabeled 60 bp oligonucleotide. The hybridized substrate and various amounts of DnaB helicase [Reha-Krantz et al. J. Biol. Chem. 253 (1978) 4043-4050; LeBowitz et aJ. y. 5io/. Chem. 26! (1986)4738-4748] were mixed in assay buffer (400 mM Hepes pH 7.4, 5 miM DTT, 2 ma/ml BSA,12.5 miM MgCU, and 6.25 mM ATP) and incubated at 37^^ C for 30 rain. The reaction was stopped by addition of 0.5M EDTA, then 2.5 mg of Proteinase K was added to each reaction to release any remaining oligonucleotide bound to DnaB Helicase. Samples were then run on a 15%<br><br>
TBE gel. The single and double strands of DNA were distinguished by autoradiography.<br>
Using the traditional helicase assay, E.coU DnaB was able to unhybridize duplex substrates to yield single strands of DNA (Figure 1). However, to observe single-stranded DNA released from the duplex DNAs by this method, at least 0.75 ug ofDnaB is required to completely imwind 175 finol of duplex DNA substrates. Therefore, this approach is time consuming and the sensitivity is low.<br>
Example 2: Helicase ['"'HI Scintillation Proximity Assav fSPA) enzvme assav<br>
The Scintillation Proximity Assay (SPA) is described in Figure 2. This assay makes use of a ■^H-oligonucieotide that is annealed to a single stranded M13 DNA (M13ssDNA). As a result of helicase activity, the "'H-oligonucleotide is unhybridized from the M13 ssDNA. The released oligonucleotide may then anneal to a biotinylated second oligonucleotide. This complex is captured by streptavidin coated SPA beads. Because of its close proximity to the SPA beads, the complex is able to stimulate scintillant within the beads, resulting in a signal increase. The labeled oligonucleotides that remain bound to Ml 3 ssDN'A iree solution do not stimulate the scintillant.<br>
A SPA helicase assay kit was purchased fi-om Amershara Life Science. The assay was performed according to the provided instructions with minor modification as described below. A ["'H] labeled oligonucleotide is annealed with a M13 single stranded DNA (M13ssDNA) template to form partial duplex substrates. The DnaB helicase and the duplex substrates were mixed in the assay buffer and incubated at ifC for 45 min. The reaction is stopped with the addition of EDTA. A biotinylated complementary oligonucleotide is then annealed to the radiolabeled oligonucleotide released from MIJSSDNA. The complex is captured by streptavidin coated SPA beads by incubation at room temperature for 15 min., and the radioactive signal counted.<br>
Titration of DnaB helicase (Figure 3) indicates that optimal activity can be obtained using 0.16 )j.g of DnaB helicase. Helicase activity decreases when less than 0.16 |j.g of DnaB is used, suggesting that there is not enough helicase to unwind the<br><br>
duplex DNAs. However, when more than 0.16 pg of DnaB helicase is used in each reaction, the helicase activity also decreases. This may result from non-specific binding of excess helicase to single-stranded DNAs released from the duplex DNA substrates, thereby preventing the complementary, biotinylated oligonucleotide from binding these single stranded DNA products. Under the optimal assay condition, the best signal to noise of this assay is about 10-foId (see Figure 4). The DnaB helicase activity is 95% of the maximum possible activity, as determined by the signal generated from products released from boiled, denature duplex DNA substrates. While this method is rapid and offers the possibility of measuring a large number of samples, it is disadvantageous because of high costs and generation of significant radioactive waste.<br>
Example 3: Fluorescence quenching helicase assay<br>
A fluorescence quenching assay has been studied (Figure 5). In this assay Europium- and Rhodamine-labeled probes are present as a fluorescence quenched duplex. Upon addition of a helicase, the duplex DNA substrates unhybridize to yield two single strands of DNA. The oUgonucleotides labeled with Lance Europium released from the duplex substrates give rise to time resolved fluorescence signals.<br>
A 60 bp oligonucleotide (5'-GT GAC GAT CCC OCA AAA GCG GCC TTT AAC TCC CTG CAA GCC TCA GCG ACC GAA TAT ATC G-3'; SEQ E) N0:2) was labeled with Lance-Europium at its 5' end. An oligonucleotide (5'-T TTT TTT TTT TTT TTT TTT TTT TTT TTT TTT GTT AAA GGC CGC TTT TGC GGG ATC GTC CA-3'; SEQ ID NO: 3) having a region complementary with SEQ E) N0:2 was labeled with Rhodamine at it5 5' end. These Eu-labeled and Rh-labeled probes were mixed at a ratio of 1:2 and hybridized to form Y-shape substrates. Various amount of DnaB helicase and SO Snols of the fluorescent labeled substrates were mixed in assay buffer and incubated at 37°C for 45 min. The reaction is stopped by addition of EDTA. The fluorescence signals generated by Eu-oligonucleotides released from the duplex DNA substrates were then recorded.<br>
Titration of DnaB helicase (Figure 6) indicates that maximal activity requires at least 1.25 (ig of DnaB helicase. and the signal to noise is about 4-fold. Together,<br><br>
these results show that the sensitivity of this approach is low.<br>
Example 4: Electrochemiluminescence helicase assay<br>
An electrochemiluminance (ECL) helicase assay is shown in Figure 7. Shon oligonucleotides are labeled with a Ruthenium metal chelate and annealed with MI3 single strand DNAs. Upon addition of helicase, the labeled oligonucleotides are unwound from the duplex DNAs, and available to anneal lo a second complementary oHgonucieotide labeled with biotin. The resulting complexes are captured by streptavidin coated magnitude beads. The bead/complex is channeled through a flow cell and captured at an electrode by a magnetic field. Voltage is applied to the electrode and ECL signals derived from Ruthenium labels are measured using an electrochemiluminescence detector, such as an Origen Anal&gt;"zer (IGEN Inc., Rockville, MD).<br>
A 60 bp oligonucleotide (5'-TTT TTT TTT TTT TTT TTT TTT TTT TTT TTT GTT AAA GGC CGC TTT TGC GGG ATC GTC ACC-3'; SEQ ID NO: 4) or a 30 bp oligonucleotide (5'- GTT AAA GGC CGC TTT TGC GGG ATC GTC ACC-3'; SEQ ID N0:5) are labeled with ruihemum at the 5' end {Kenten et al., Clin. Chem. 3S (1992) S73-879), and resuspended in TE buffer (10 mM Tris HCi, 1 mM EDTA, pH=7.4) to a final stock concentration of 25 pmoles.VL- M13 mplSssDNA is dissolved in TE at a concentration of 0,1S3 pmoles/mL. Equal concentrations of M13mpl8 ssDNAand oligonucleotides are mixed in hybridization buffer (10 mM Tris HCI, 0.3 M NaCl, 30m\t Na Citrate, pH 8.5). The solution is then incubated at 1 OO^C for 2 minutes, and cooled to room temperature. Twenty-five frnoles of duplex substrates (M13 ssDNA annealed with 30 or 60 oligonucleotide) is mixed with 25ng of £. coU DnaB helicase in assay buffer (40 mM Hepes pH 7.4, 0.5 mM DTT, 0.2 mg'ml BSA, 8 miM MgCU, 5 mM ATP). Tne reaction mixture is incubated at 37'C for 45 minutes. To this reaction is added 125 fmoles of a biotinylated oligonucleotide (5'-GGT GAC GAT CCC GCA .\A,i. GCG GCC TTT A-^C AGC-3'; SEQ ID NO: 6) complementary' to SEQ ID NOS: 4 and 5 in 0.5M EDTA, and the reaction is incubated at 37'^C for 15 minutes. Biotinylation is perfonned using the Biotin-ON^" Phosphoramidate Kit from Clontech (PT1402-1) using the procedure provided with<br><br>
the kit. The adenine at position 31 within SEQ ID N0:6 received the biotin label. Finally, 20 \ig of streptavidin magnetic beads in a Iota! volume of 0.25 ml TE buffer is added, and the solution is incubated for 15 minutes while vortexing. The ECL signal is recorded using an Origen Analyzer (IGEN Inc., RockviUc, MD) using conditions sugg'^sted be the manufacmrer.<br>
The data from titration of DnaB helicase indicates that only 25ng of enzyme is required for unhybridizing 25 &amp;nols of duplex substrates, and that the signal to noise is about 30-fold (see Figure 8). Additional experiments demonstrate that, in genera!, the signal to noise is at least 30-fold, and as high as 40-fo!d, This method of detection shows significant sensitivity and reproducibility as compared to all previously described methods (see Table 1).<br><br>
The sensitivity of the electrochemiluminescence assay is at least 7-foId greater than the fluorescence quenching assay and 3-fold over the SPA assay. Overall, the sensitivity is about 18-foId better than the SPA assay, and even greater when<br><br>
compared to the fluorescence assay. The amount of enzyme needed for electrochemiluminescence detection decreases 6-fold over the SPA assay and 50-fold over the quenching assay. The substrates used in the electrocherailuminescence assay are reduced 2-fold compared with both the SPA and quenching assays. The electrochemiluminescence assay enables rapid and sensitive detection of heUcase activity, and can be easily formatted for high throughput screening. The approach is generic, and is suitable for any enzyme having heUcase activity.<br>
The present invention is not to be limited in scope by the specific embodiments described herein. Indeed, various modifications of the invention in addition to those described herein will become apparent to those skilled in the art from the foregoing description and the accompanying figures. Such modifications are intended to fall within the scope of the appended claims.<br>
Various publications are cited herein, the disclosures of which are incorporated by reference in their entireties.<br><br><br>
WHAT IS CLAIMED IS:<br>
1.	An elecirochemiluminescent assay method for detecting heiicase activity in a<br>
sample, the method comprising<br>
(a)	combining the sample with a first nucleic acid or nucleic acid molecule comprising an electrochemiluminescent label and hvtridized to a complementary second nucleic acid or nucleic acid molecule;<br>
(b)	incubating the sample and hybridized nucleic acids or nucleic acid molecules from step (a) under conditions where heiicase present in the sample can unhybridize the first and second nucleic acids or nucleic acid molecules;<br>
(c)	capturing unhybridized first nucleic acid or nucleic acid molecule; and<br>
(d)	measuring electrochemi luminescence from the capmred first nucleic acid or nucleic acid molecule,<br><br>
2.	The electrochemi luminescent assay according to claim I, wherein the heiicase is a human or pathogenic heiicase.<br>
3.	The elecirochemiluminescent assay according to claim 2, wherein the pathogenic heiicase is a fungal, viral, bacterial or parasitic heiicase.<br>
4.	The electrochemiluminescent assay according to claim 1, comprising between steps (b) and (c),<br>
(/)        incubating the unhybridized nucleic acids or nucleic acid molecules<br>
with a third nucleic acid or nucleic acid molecule complementary to the first nucleic acid or nucleic acid molecule and comprising a capturing ligand, under conditions where the first and third nucleic acids or nucleic acid molecules can hybridize; and<br>
(ii)       incubating the hybridized first and third nucleic acids or nucleic acid molecules from step ({) with a capturing receptor which binds the capturing ligand.<br>
5.	The electrochemiluminescent assay according lo claim 4, wherein the<br>
capturing ligand is biotin and the capturing receptor is avidin.<br><br>
6.	The electrochemiluminesceut assay according to claim 4, wherein the capturing receptor is on the surface of magnetic beads.<br>
7.	The electrochemilumiaescent assay according to claim 6, wherein the capturing ligand is biotin and the capturing receptor is avidin.<br>
8.	The eiectrochemiluminescent assay according to claim 1, wherein the electrochemi luminescent label is a ruthenium chelate.<br>
9.	A method for detecting in a sample an agent that modulates helicase activity, the method comprising<br>
(a)       combining the sample, a helicase and a first nucleic acid or nucleic acid molecule comprising an eiectrochemiluminescent label and hybridized to a complemeniary second nucleic acid or nucleic acid molecule, thereby forming a mixture;<br>
fb)       incubating the mixuire from step (a) under conditions where the<br>
helicase can unhybridize the first and second nucleic acids or nucleic acid molecules in the absence of the sample;<br>
(c)	capturing unhybridized first nucleic acid or nucleic acid molecule; and<br>
(d)	measuring electrochemiluminescence from the captured first nucleic acid or nucleic acid molecule.<br><br>
10.	The method according to claim 9, wherein the agent is an inhibitor of helicase activity.<br>
11.	The method according to claim 9, wherein the helicase is a human or pathogenic helicase.<br>
12.	The method according to claim 11, wherein the pathogenic helicase is a fiingal, viral, bacterial or parasitic helicase.<br>
13.	The method according to claim 9, comprising berween steps (b) and (c),<br>
(i)        incubating the unhybridized nucleic acids or nucleic acid molecules<br>
with a third nucleic acid or nucleic acid molecule complementary to the first nucleic acid or nucleic acid moiecuie and comprising a capturing ligand, under conditions where the first and third nucleic acids or nucleic acid molecules can hybridize; and<br><br>
(//)       incubating the hybridized first and third nucleic acids or nucleic acid molecules from step (/) with a capturing receptor which binds the capturing ligand.<br>
14.	The method according to claim 13, wherein the capturing ligand is biotin and the capturing receptor is avidin.<br>
15.	The method according to claim 13, wherein the capturing receptor is on the surface of magnetic beads.<br>
16.	The method according to claim 15, wherein the capturing ligand is biotin and the capturing receptor is avidin.<br>
17.	The method according to claim 9, wherein the elecirochemiluminescent label is a ruthenium chelate.<br>
18.	A kit for electrochemilumincscent detection of helicase activity comprising<br><br>
(a)	a first nucleic acid or nucleic acid molecule comprising an elecirochemiluminescent label;<br>
(b)	a second nucleic acid or nucleic acid molecule complementary to the first nucleic acid or nucleic acid molecule;<br>
(c)	a helicase.<br><br>
19.	The kit according to claim 18, wherein the fost nucleic acid or nucleic acid molecule is hybridized to the second nucleic acid or nucleic acid molecule.<br>
20.	The kit according to claim 18, further comprising a third nucleic acid or nucleic acid molecule complementary to the first nucleic acid or nucleic acid molecule and comprising a capturing ligand.<br>
21.	The kit according to claim 20, fiirther comprising a capturing receptor.<br>
22.	The kit according to claim 21, wherein the capturing receptor is on the surface of magnetic beads.<br>
23.	The kit according to claim 22, wherein the capturing ligand is biotin and the<br><br>
capturing receptor is avidin.<br><br>
24,	An electrochemiluminesceDt assay method for detecting helicase<br>
activity in a sample substantially as herein described with reference to<br>
the accompanying drawings.<br>
25.	A kit for electrochemiluminescent detection of helicase activity<br>
substantially as herein described with reference to the accompanying<br>
drawings.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQ4OC1jaGUgYWJzdHJhY3QtZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0488-che abstract-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQ4OC1jaGUgY2xhaW1zLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-0488-che claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQ4OC1jaGUgY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0488-che claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQ4OC1jaGUgY29ycmVzcG9uZGVuY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0488-che correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQ4OC1jaGUgY29ycmVzcG9uZGVuY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-0488-che correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQ4OC1jaGUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKS1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-0488-che description (complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQ4OC1jaGUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-0488-che description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQ4OC1jaGUgZHJhd2luZ3MtZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0488-che drawings-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQ4OC1jaGUgZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-0488-che drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQ4OC1jaGUgZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0488-che form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQ4OC1jaGUgZm9ybS0xMy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-0488-che form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQ4OC1jaGUgZm9ybS0xOS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-0488-che form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQ4OC1jaGUgZm9ybS0yNi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-0488-che form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQ4OC1jaGUgZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0488-che form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQ4OC1jaGUgZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0488-che form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQ4OC1jaGUgcGN0IHNlYXJjaCByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-0488-che pct search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQ4OC1jaGUgcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0488-che pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQ4OC1jaGUgcGV0aXRpb24ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-0488-che petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="225527-a-method-of-producing-a-continuous-integrated-circuit-panel.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="225529-2-phenyl-substituted-imidazotriazinones-compound-and-process-thereof.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>225528</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2002/488/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>52/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>26-Dec-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>18-Nov-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>04-Apr-2002</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>AVENTIS PHARMACEUTICALS, INC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>55 CORPORATE DRIVE, BRIDGEWATER, NEW JERSEY 08807,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ZHANG, Litao</td>
											<td>456 Shakespear Drive, Collegeville, PA 19426,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>HARRISON, Richard, K</td>
											<td>5 Ashlea Drive, Glenmoore, PA 19343,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2000/027453</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-10-05</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>US 09/410,960</td>
									<td>1999-10-05</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/225528-electrochemiluminescence-helicase-assay by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:12:20 GMT -->
</html>
