<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/225580-a-method-of-producing-a-drought-tolerant-plant by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:27:53 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 225580:A METHOD OF PRODUCING A DROUGHT TOLERANT PLANT</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD OF PRODUCING A DROUGHT TOLERANT PLANT</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a method of producing a drought tolerant plant comprising: a) providing a nucleic acid construct comprising a promoter operably linked to a nucleic acid that inhibits farnesyl transferase beta activity; b) inserting said nucleic construct into a vector; c) transforming a plant, tissue culture, or a plant cell with the vector to obtain a plant, tissue culture or a plant cell with decreased farnesyl transferase beta activity; and d) growing said plant or regenerating a plant from said tissue culture or plant cell, wherein a drought tolerant plant is produced.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
The present invention relates to a method of producing a drought tolerant plant.<br>
Most higher plants encounter at least transient decreases in relative water content at some stage of their life cycle and, as a result, have evolved a number of desiccation protection mechanisms.  If however, the change in water deficit is prolonged the effects on the plant's growth and development can be profound.  Decreased water content due to drought, cold or salt stresses can irreparably damage plant cells which in turn limits plant growth and crop productivity in agriculture.<br>
Plants respond to adverse conditions of drought, salinity and cold with a variety of morphological and physiological changes. Although our understanding of plant tolerance mechanisms to these stresses is fragmentary, the plant hormone abscisic acid (ABA) has been proposed to be an essential mediator between environmental stimulus and plant responses.  ABA levels increase in response to water deficits and exogenously applied ABA mimics many of the responses normally induced by water stress.  Once ABA is synthesized it causes the closure of the leaf stomata thereby decreasing water loss through transpiration.<br>
The identification of genes that transduce ABA into a cellular response opens the possibility of exploiting these regulators to enhance desiccation tolerance in crop species. In principle, these ABA signalling genes can be<br><br>
coupled with the appropriate controlling elements to allow optimal plant growth and development. Thus, not only would these genes allow the genetic tailoring of crops to withstand transitory environmental insults, they should also broaden the environments where traditional crops can be grown.<br>
In addition, little is lenown of the genetic mechanisms which control plant growth and development.  Genes which further affect other metabolic processes such as senescence and growth habits of plants can be useful in a wide variety of crop and horticultural plants.<br>
SUMMARY OF THE INVENTION<br>
This invention relates to isolated nucleic acids which encode a farnesyl transferase comprising SEQ ID N0:1. Nucleic acids also encompassed by this invention are such hybridizing sequences which encode the functional equivalent of SEQ ID N0:l.  The present invention also relates to a method for enhancing the drought tolerance of plants using inhibitors of the products encoded by these nucleic acids.  Further, this invention relates to the control of regulatory functions in photosynthetic organisms; for example, in the control of growth habit, flowering, seed production, seed germination, and senescence in such organisms.<br>
This invention also relates to a method for enhancing the drought tolerance of plants by means of alterations in isolated or recombinant nucleic acids encoding a farnesyl transferase (Ftase) protein or its functional equivalent. Nucleic acids which hybridize to the Ftase-encoding gene i  (ERA1) are also encompassed by this invention when such hybridizing sequences encode the functional equivalent of the Ftase protein.  The present invention also relates to a<br><br>
method for enhancing the drought tolerance of plants through the genetic manipulation of ERAl gene and its functional equivalents to improve stress tolerance in crop<br>
plants.  Loss of ERAl gene function confers enhanced<br>
tolerance to drought at the level of the mature plant.  The nature of an eral mutant with loss of Ftase activity, for<br>
example, demonstrates that inhibition of farnesylation enhances ABA responses in a plant.<br>
Further, this invention relates to inhibition of senescence in photosynthetic organisms through inhibition of farnesyl transferase activity.  The resulting photosynthetic organisms stay green and tissue viability is maintained for a longer period of time.  Thus, methods to provide greener plants and a reduction in senescence are part of this invention.<br>
In yet another embodiment, methods are provided to modify the growth habit and flower induction of plants. Loss of ERAl gene function under particular environmental conditions results in a reduction in the number of lateral branches produced on a plant and an increase in the number of flowers per inflorescence.<br>
This invention also relates to a regulatory sequence useful for genetic engineering of plant cells to provide a method of controlling the tissue pattern of expression of DNA sequences linked to this novel regulatory sequence.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figures 1A-1C show the nucleic acid sequence of the<br>
ERAl gene (SEQ ID N0:1) in which the introns are underlined<br>
and  the start codon (ATG) is at nucleotide positions 1-3.<br>
Figure 2 is the amino acid sequence of the ERAl<br>
protein (SEQ ID N0:2).<br><br>
Figures 3A-3C show the nucleic acid sequence of the ERA1 promoter (SEQ ID NO;3).<br>
Figure 4 is the amino acid sequence of the [3 subunit farnesylation domain from Arabidopsis   (Arab.) (SEQ ID NO:2)<br>
aligned with the (5 subunit farnesylation domains from pea (SEQ ID N0:4), yeast (SEQ ID N0:5) and rat (SEQ ID NO:6). Residues that are identical to the Arabidopsis sequence are<br>
indicated with a dot.  A dash indicates a blank.  The amino acid positions of the Arabidopsis gene are indicated on the<br>
right-hand side.<br>
Figure 5 is a photograph of an eral-transformed<br>
Arabidopsis plant (right) compared to the wild-type<br>
(control; i.e., naturally-occurring) plant (left) under<br>
extremely dry conditions.<br>
Figure 6 is a graph comparing the water content of Arabidopsis plants with inactivated or mutant Ftase<br>
activity {M.   Columbia,   era  1-2) and controls {M.C. control, era 1-2 control).<br>
Figure 7 is a graph comparing the rate of water loss for the Arabidopsis  plants with inactivated or mutant Ftase<br>
activity (M. Columbia,   era 1-2) and controls (M.C. control, era  1-2 control).<br>
Figure 8 is a comparison of aging leaves from control (wild-type) and era-2 mutant plants.<br>
Figure 9 is a comparison of transcript levels in aging leaves from control (wild-type) and era-2  mutant plants.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
This invention relates to isolated nucleic acids  and proteins encoded by these nucleic acids which modify the<br><br>
growth, reproduction and senescence of plants. In particular, the constructs of this invention include an isolated nucleic acid encoding a farnesyl transferase (Ftase) polypeptide comprising SEQ ID N0:1 or its functional equivalent, and the Ftase polypeptides or proteins encoded by these nucleic acids.  In particular, this invention relates to a protein wherein the sequence is SEQ ID NO:2.<br>
Further included in this invention are nucleic acid constructs which comprise a promoter (ERA1 promoter) operably-linked to isolated nucleic acid comprising SEQ ID N0:1 or its functional equivalent or a complement of either.  When incorporated into a plant, the ERA1 promoter is regulated in the guard cells of the plant and can affect water loss through the stomates.  This promoter consists of a nucleic acid comprising SEQ ID N0:3 (Figure 3).<br>
Transgenic plants, seeds, plant cell and tissues incorporating these constructs are also part of this invention.  Accordingly, in one aspect of this invention, a method is provided for producing a gene product under the control of a promoter which operates primarily in guard cells through expression of a gene encoding the gene product in the cell of a plant comprising the steps of: transforming a plant cell with a DNA construct comprising a) a regulatory region comprising SEQ ID NO:3 or a functional portion thereof, DNA comprising a structural gene encoding a gene product, and a 3' untranslated region containing a polyadenylated region; regenerating a plant, photosynthetic organism or tissue culture from the cell; and placing the plant, photosynthetic organisms or tissue culture under conditions so that the promoter induces transcription of the structural gene and the gene product is expressed.<br><br>
In the context of this disclosure, the terms "regulatory region" or "promoter" refer to a sequence of DNA, usually upstream (5') to the coding sequence of a structural gene, which controls the expression of the coding region by providing recognition and binding sites for RNA polymerase and/or other factors required for transcription to start at the correct site.   The term "functional portion" or "functional fragment" refers to a truncated sequence of a promoter of this invention which maintains the capability of inducing transcription of an ERA structural gene under the conditions described for activity of an Ftase protein.<br>
The constructs and methods described herein can be applied to all types of plants and other photosynthetic organisms, including, but not limited to: angiosperms (monocots and dicots), gymnosperms, spore-bearing or vegetatively-reproducing plants and the algae, including the cyanophyta (blue-green algae).  Particularly preferred plants are those plants which provide commercially-valuable crops, such as corn, wheat, cotton, rice, canola, sugar cane, sugar beet, sunflowers, potatoes, tomatoes, broccoli, carrots, lettuce, apple, plum, orange, lemon, rose, and the like.<br>
Further, the constructs and methods of this invention can be adapted to any plant part, protoplast, or tissue culture wherein the tissue is derived from a photosynthetic organism.  The term "plant part" is meant to include a portion of a plant capable of producing a regenerated plant.  Preferable plant parts include roots and shoots and meristematic portions thereof.  Other plant parts encompassed by this invention are: leaves, flowers, seeds, epicotyls, hypocotyls, cotyledons, cotyledonary nodes, explants, pollen, ovules, meristematic or embryonic tissue,<br><br>
protoplasts, and the like.  Transgenic plants can be regenerated from any of these plant parts, including tissue culture or protoplasts, and also from explants. Methods will vary according to the species of plant.<br>
This invention relates to compositions and constructs comprising isolated nucleic acids (both DNA and RNA) encoding an Ftase and portions thereof of photosynthetic organisms. This invention further relates to compositions and constructs comprising isolated nucleic acids encoding an Ftase promoter.  In particular, the ERA1 gene encoding the 3 subunit of Ftase from Arabidopsis  and a regulatory sequence which regulates the transcription of the ERA1 gene have been isolated and sequenced.  Nucleic acids which encode Ftases from photosynthetic organisms, and homologues or analogs of these nucleic acids, are encompassed by this invention.<br>
The invention further relates to methods using isolated and/or recombinant nucleic acids (DNA or RNA) that are characterized by their ability to hybridize to (a) a nucleic acid encoding an Ftase protein or polypeptide, such as a nucleic acid having the sequences of SEQ ID NO:l or (b) a portion of the foregoing (e.g.,   a portion comprising<br>
the minimum nucleotides required to encode a functional Ftase protein; or by the ability to encode a polypeptide having the amino acid sequence of an Ftase(e.g. , SEQ ID<br>
N0:2, or to encode functional equivalents thereof; e.g., a<br>
polypeptide having at least 80% sequence similarity to SEQ ID NO:2, which when incorporated into a plant cell, facilitates the growth habit, seed germination, and metabolism in a photosynthetic organism in the same manner as SEQ ID N0:1).  A functional equivalent of an Ftase<br><br>
therefore, would have at least an 80% similar amino acid sequence and similar characteristics to, or perform in substantially the same way as, the polypeptide encoded by SEQ ID NO:2.  A nucleic acid which hybridizes to a nucleic acid encoding an Ftase polypeptide such as SEQ ID NO:2 can be double- or single-stranded.  Hybridization to DNA such as DNA having the sequence SEQ ID NO:l, includes hybridization to the strand shown or its complementary strand.<br>
In one embodiment, the percent amino acid sequence similarity between an Ftase polypeptide such as SEQ ID NO:2, and functional equivalents thereof is at least about 60% U 60%).  In a preferred embodiment, the percent amino acid sequence similarity between an Ftase polypeptide and its functional equivalents is at least about 75% (^75%). More preferably, the percent amino acid sequence similarity between an Ftase polypeptide and its functional equivalents is at least about 80%, and still more preferably, at least about 90%, when consecutive amino acids are compared.<br>
Isolated and/or recombinant nucleic acids meeting these criteria comprise nucleic acids having sequences identical to sequences of naturally occurring ERA1 genes and portions thereof, or variants of the naturally occurring genes.  Such variants include mutants differing by the addition, deletion or substitution of one or more nucleotides, modified nucleic acids in which one or more nucleotides are modified (e.g., DNA or RNA analogs), and mutants comprising one or more modified nucleotides.<br>
Such nucleic acids, including DNA or RNA, can be detected and isolated by  hybridization under high stringency conditions or moderate stringency conditions, for example, which are chosen so as to not permit the hybridization of nucleic acids having non-complementary<br><br>
sequences.  "Stringency conditions" for hybridizations is a term of art which refers to the conditions of temperature and buffer concentration which permit hybridization of a particular nucleic acid to another nucleic acid in which the first nucleic acid may be perfectly complementary to the second, or the first and second may share some degree of complementarity which is less than perfect.  For example, certain high stringency conditions can be used which distinguish perfectly complementary nucleic acids from those of less complementarity.  "High stringency conditions" and "moderate stringency conditions" for nucleic acid hybridizations are explained on pages 2.10.1-2.10.16 (see particularly 2.10.8-11) and pages 6.3.1-6 in Current Protocols  in Molecular Biology   (Ausubel, F.M. et<br>
al.,   eds., Vol. 1, containing supplements up through<br>
Supplement 29, 1995).  The exact conditions which determine the stringency of hybridization depend not only on ionic strength, temperature and the concentration of destabilizing agents such as formamide, but also on factors such as the length of the nucleic acid sequence, base composition, percent mismatch between hybridizing sequences and the frequency of occurrence of subsets of that sequence within other non-identical sequences.  Thus, high or moderate stringency conditions can be determined empirically.<br>
High stringency hybridization procedures can (1) employ low ionic strength and high temperature for washing, such as 0.015 M NaCl/ 0.0015 M sodium citrate, pH 7.0 (O.lx SSC) with 0.1% sodium dodecyl sulfate (SDS) at 50°C; (2) employ during hybridization 50% (vol/vol) formamide with 5x Denhardt's solution (0.1% weight/volume highly purified bovine serum albumin/ 0.1% wt/vol Ficoll/ 0.1% wt/vol<br><br>
polyvinylpyrrolidone), 50 mM sodium phosphate buffer at pH 6.5 and 5x SSC at 42°C; or (3) employ hybridization with 50% formamide, 5x SSC, 50 mM sodium phosphate {pH 6.8), 0.1% sodium pyrophosphate, 5x Denhardt's solution, sonicated salmon sperm DNA (50 pg/ml), 0.1% SDS, and 10% dextran sulfate at 42°C, with washes at 42°C in 0.2x SSC and 0.1% SDS.  Moderate stringency conditions would be similar except that hybridization would employ 25% formamide in place of 50% formamide.<br>
By varying hybridization conditions from a level of stringency at which no hybridization occurs to a level at which hybridization is first observed, conditions which will allow a given sequence to hybridize with the most similar sequences in the sample can be determined.<br>
Exemplary conditions are described in Krause, M.H. and S.A. Aaronson (1991) Methods in Enzymology,   200:546-556.<br>
Also, see especially page 2.10.11 in Current Protocols in<br>
Molecular Biology   (supra) , which describes how to determine<br>
washing conditions for moderate or low stringency conditions.  Washing is the step in which conditions are usually set so as to determine a minimum level of complementarity of the hybrids.  Generally, from the lowest temperature at which only homologous hybridization occurs, a 1% mismatch between hybridizing nucleic acids results in a 1°C decrease in the melting temperature Tm, for any chosen SSC concentration.  Generally, doubling the concentration of SSC results in an increase in Tm of ~17°C. Using these guidelines, the washing temperature can be determined empirically for moderate or low stringency, depending on the level of mismatch sought.<br>
Isolated and/or recombinant nucleic acids that are characterized by their ability to hybridize to (a) a<br><br>
nucleic acid encoding an Ftase polypeptide, such as the nucleic acids depicted as SEQ ID N0:1, (b) the complement of SEQ ID N0:1, (c) or a portion of (a) or (b) (e.g.   under<br>
high or moderate stringency conditions), may further encode a protein or polypeptide having at least one functional characteristic of an Ftase polypeptide, such as  regulation of lateral branching under diurnal light cycles, or regulation of the response to ABA, or regulation of senescence.<br>
Enzymatic assays, complementation tests, or other suitable methods can also be used in procedures for the identification and/or isolation of nucleic acids which encode a polypeptide such as a polypeptide of the amino acid sequence SEQ ID NO:2 or a functional equivalent of this polypeptide.  The antigenic properties of proteins or polypeptides encoded by hybridizing nucleic acids can be determined by immunological methods employing antibodies that bind to an Ftase polypeptide such as immunoblot, immunoprecipitation and radioimmunoassay.  PCR methodology, including RAGE (Rapid Amplification of Genomic DNA Ends), can also be used to screen for and detect the presence of nucleic acids which encode Ftase-like proteins and polypeptides, and to assist in cloning such nucleic acids from genomic DNA.  PCR methods for these purposes can be found in Innis, M.A., et al.    (1990) PCR Protocols:   A  Guide<br>
to Methods and Applications,   Academic Press, Inc., San<br>
Diego, CA.<br>
The nucleic acids described herein are used in the methods of the present invention for production of proteins or polypeptides which are incorporated into cells, tissues, plant parts, plants and other photosynthetic organisms. In one embodiment, DNA containing all or part of the coding<br><br>
sequence for an Ftase polypeptide, or DNA which hybridizes to DNA having the sequence SEQ ID NO:2 is incorporated into a vector for expression of the encoded polypeptide in suitable host cells.  The encoded polypeptide consisting of an Ftase subunit or its functional equivalent is capable of farnesyl transferase activity. The term "vector" as used herein refers to a nucleic acid molecule capable of transporting another nucleic acid to which it has been linked.<br>
Primers and probes consisting of 20 or more contiguous nucleotides of the above-described nucleic acids are also included as part of this invention.  Thus, one nucleic acid of this invention comprises a specific sequence of about 20 to about 200 or more nucleotides which are identical or complementary to a specific sequence of nucleotides of the Ftase protein-encoding DNA or transcribed mRNA.  These probes and primers can be used to identify and isolate Ftase-encoding nucleic acid from other photosynthetic organisms.<br>
Nucleic acids referred to herein as "isolated" are nucleic acids separated away from the nucleic acids of the genomic DNA or cellular RNA of their source of origin (e.g., as it exists in cells or in a mixture of nucleic acids such as a library), and may have undergone further processing.  "Isolated" nucleic acids include nucleic acids obtained by methods described herein, similar methods or other suitable methods, including essentially pure nucleic acids, nucleic acids produced by chemical synthesis, by combinations of biological and chemical methods, and recombinant nucleic acids which are isolated.  Nucleic acids referred to herein as "recombinant" are nucleic acids which have been produced by recombinant DNA methodology,<br><br>
including those nucleic acids that are generated by-procedures which rely upon a method of artificial recombination, such as the polymerase chain reaction {PCR) and/or cloning into a vector using restriction enzymes. "Recombinant" nucleic acids are also those that result from recombination events that occur through the natural mechanisms of cells, but are selected for after the introduction to the cells of nucleic acids designed to allow or make probable a desired recombination event. Portions of the isolated nucleic acids which code for polypeptides having a certain function can be identified and isolated by, for example, the method of Jasin, M., et<br>
al.,   U.S. Patent No. 4,952,501.<br>
A further embodiment of the invention is antisense nucleic acids or oligonucleotides which are complementary, in whole or in part, to a target molecule comprising a sense strand, and can hybridize with the target molecule. The target can be DNA, or its RNA counterpart (i.e.,<br>
wherein T residues of the DNA are U residues in the RNA counterpart).  When introduced into a cell, antisense nucleic acids or oligonucleotides can inhibit the expression of the gene encoded by the sense strand or the mRNA transcribed from the sense strand.  Antisense nucleic acids can be produced by standard techniques.  See, for example, Shewmaker, et al., U.S. Patent No. 5,107,065.<br>
In a particular embodiment, an antisense nucleic acid or oligonucleotide is wholly or partially complementary to and can hybridize with a target nucleic acid (either DNA or RNA), wherein the target nucleic acid can hybridize to a nucleic acid having the sequence of the complement of the strand in SEQ ID NO:1.  For example, an antisense nucleic acid or oligonucleotide can be complementary to a target<br><br>
nucleic acid having the sequence shown as the strand of the open reading frame of SEQ ID H0:1, or nucleic acid encoding a functional equivalent of Ftase, or to a portion of these nucleic acids sufficient to allow hybridization.  A portion, for example, a sequence of 16 nucleotides could be sufficient to inhibit expression of the protein.  Fragments comprising 25 or more consecutive nucleotides complementary to SEQ ID N0:1 could also be used.  Or, an  antisense nucleic acid or oligonucleotide complementary to 5' or 3' untranslated regions, or overlapping the translation initiation codon (5' untranslated and translated regions), of the ERA1 gene, or a gene encoding a functional equivalent can also be effective.  In another embodiment, the antisense nucleic acid is wholly or partially complementary to and can hybridize with a target nucleic acid which encodes an  Ftase polypeptide.<br>
In addition to the antisense nucleic acids of the invention, oligonucleotides can be constructed which will bind to duplex nucleic acid either in the gene or the DNA:RNA complex of transcription, to form a stable triple helix-containing or triplex nucleic acid to inhibit transcription and/or expression of a gene encoding an Ftase polypeptide or its functional equivalent.  Frank-Kamenetskii, M.D. and Mirkin, S.M. (1995) Ann.   Rev.<br>
Biochem.   64:65-95.  Such oligonucleotides of the invention<br>
are constructed using the base-pairing rules of triple helix formation and the nucleotide sequence of the gene or mRNA for Ftase.  These oligonucleotides can block Ftase-type activity in a number of ways, including prevention of transcription of the ERA1 gene or by binding to mRNA as it is transcribed by the gene.<br><br>
The invention also relates to proteins or polypeptides encoded by the novel nucleic acids described herein.  The proteins and polypeptides of this invention can be isolated and/or recombinant.  Proteins or polypeptides referred to herein as "isolated" are proteins or polypeptides purified to a state beyond that in which they exist in cells.  In a preferred embodiment, they are at least 10% pure; i.e.,<br>
substantially purified.  "Isolated" proteins or polypeptides include proteins or polypeptides obtained by methods described infra, similar methods or other suitable<br>
methods, and include essentially pure proteins or polypeptides, proteins or polypeptides produced by chemical synthesis or by combinations of biological and chemical methods, and recombinant proteins or polypeptides which are isolated.  Proteins or polypeptides referred to herein as "recombinant" are proteins or polypeptides produced by the expression of recombinant nucleic acids.<br>
In a preferred embodiment, the protein or portion thereof has at least one function characteristic of an Ftase; for example, catalytic activity affecting, e.g.,<br>
normal lateral branching, florets/inflorescence, seed germination, or stamatal opening, and binding function, and/or antigenic function (e.g., binding of antibodies that<br>
also bind to naturally occurring Ftase).  As such, these . proteins are referred to as Ftases of plant origin, and include, for example, naturally occurring Ftase, variants (e.g. mutants) of those proteins and/or portions thereof.<br>
Such variants include mutants differing by the addition, deletion or substitution of one or more amino acid ) residues, or modified polypeptides in which one or more residues are modified, and mutants comprising one or more modified residues.<br><br>
The invention also relates to isolated and/or recombinant portions of an Ftase as described above, especially the E&gt; subunit of an Ftase protein.  Portions of the enzyme can be made which have full or partial function on their own, or which when mixed together (though fully, partially, or nonfunctional alone), spontaneously assemble with one or more other polypeptides to reconstitute a functional protein having at least one functional characteristic of an Ftase of this invention.<br>
A number of genes have been identified that are induced by ABA. This suggests that ABA-induced tolerance to adverse environmental conditions is a complex multigenic event. Thus, identification and transfer of single genes into crop plants which improves the viability of the plant under different environmental conditions due to increased responsiveness to ABA is novel and extremely useful.<br>
To identify genes that could be more global controllers of ABA-regulated plant processes, genetic screens were applied in a number of plant species to isolate mutations that alter the response of the plant to the hormone.<br>
Mutations that confer enhanced  response to ABA (era)<br>
in Arabidopsis  seeds were identified by their ability to prevent seed germination with low concentrations of ABA that normally permit wild-type (controls, i.e., naturally-occurring) seed germination.  Of these, the eral mutant<br>
class, which includes one transferred DNA (T-DNA) line (eral-1, ecotype Wassilewskija) and two neutron-generated<br>
mutants (eral-2 and eral-3, ecotype Columbia), was of added<br>
interest because this class showed decreased germination efficiency under normal postimbibition.  Mutations that<br><br>
enhance ABA responsiveness should, in principle, be more dormant.  Dormancy in eral alleles was alleviated by a 4-<br>
day chilling period; the efficiency of eral germination<br>
increased with the length of time the seeds are chilled. In many plant species, breaking dormancy to allow germination requires vernalization and exposure to moist, low-temperature environments for an extended period (Baskin and Baskin, 1971).  The germination profile of era mutants<br>
could reflect an increased state of ABA-induced dormancy; consequently, these seeds require longer vernalization to germinate.  Support for this contention came from construction of double mutants of eral with both ABA<br>
biosynthetic (aJbal-1) and insensitive mutants {abil-i  and<br>
ahi3-6) .     In all cases, the double mutants had reduced<br>
dormancy as compared with eral, indicating that the<br>
increased dormancy observed in eral seed was dependent on<br>
ABA synthesis or sensitivity.<br>
Aside from broadening the spectrum of new ABA response mutants, supersensitivity screens were also used to identify negative regulators of ABA sensitivity.  That is, inhibition of these gene functions enhances the ABA response.  One of these genes (ERA1) has been cloned and demonstrated to encode the p-subunit of a heterodimeric protein farnesyl transferase (Ftase) (Cutler et al., 1996).<br>
The eral-1 mutation, which is due to a T-DNA insertion,<br>
allowed the isolation of plant genomic regions flanking the insertions.  Using the flanking regions as probes, the wild-type cDNA and genomic clones were isolated.  Sequence analysis of these described a gene encompassing 3.5 kb of genomic DNA.  The gene contains 13 introns which are underlined in Figures 1A-1C and the T-DNA insertion site in<br><br>
era.1-1  is in intron 8.  Southern (DNA) analysis of wild-type DNA, eral-2,   and eral-3  probed with EralcDNA revealed<br>
that both fast-neutron alleles contain deletions spanning the ERAl  locus.  Fast-neutron mutagenesis induced small<br>
deletions in Arabidopsis (Shirley et al., 1992), and<br>
subsequent genomic analysis with a 14-kb probe that spans the ERAl   locus determined the size of the eral-2 deletion<br>
to be about 7.5 kb and the eral-3 deletion to be slightly<br>
larger.  Thus all three eral alleles contained DNA<br>
disruptions at the same locus, confirming the identity of the ERA locus.<br>
Conceptual translation of the longest open reading frame (404 amino acids) in the ERAl gene produced a protein (Figures 2 and 4) with a high sequence similarity to yeast, pea, and mammalian protein farnesyl transferase £&gt; subunit genes (Goodman et al.,   1988; Chen et al.,   1991; Yang et<br>
al.,   1993).  Farnesyl transferases consist of a and p<br>
subunits that dimerize, forming an enzyme that catalyzes the attachment of farnesyl pyrophosphate (15 carbons) to proteins containing a COOH-terminal CaaX motif (Schafer and Rine, 1992), where C designates cysteine residue, aa is usually aliphatic amino acids, and X may designate a cysteine, serine, methionine, or glutamine residue. Both plant p subunit genes contain a region of about 50 amino acids near their COOH-terminus that is  absent in yeast and animal p subunit genes.<br>
In yeast and mammalian systems, Ftases modify several signal transduction proteins for membrane localization. This is achieved by the attachment of the lipophilic farnesyl sidechain to the protein target via the Ftase. The attachment of the farnesyl group causes a change in the<br><br>
overall hydrophobicity of the target allowing the protein to anchor itself into the membrane where it usually interacts with other signal transduction molecules.  That the loss of farnesylation activity in the eral mutant leads<br>
to an enhanced response of the seed to ABA suggests a target protein in Arabidopsis  must be localized to the<br>
membrane to attenuate the ABA signal.  Thus farnesylation in Arabidopsis,   appears to be required for the normal<br>
function of a negative regulator of ABA sensitivity.<br>
Subsequent work has shown that loss of ERAl gene function in Arabidopsis confers an enhanced tolerance to<br>
environmental stresses at the level of the mature plant. For example, a comparison of wild-type plants and eral<br>
mutant plants grown in soil under standard laboratory conditions (24 hr light, 150 fiE  m"2sec"1, 30% humidity) showed that the mutants did not require water as frequently as the wild-type plants in order to maintain viability (Figure 5).  When mutant and wild-type plants were grown until flowering occurred, watering was stopped and the plants were observed each subsequent day for signs of stress.  Water loss was significantly reduced in the mutant plants compared to the wild-type plants (Figures 6 and 7).<br>
To determine if the observed increased drought tolerance of era  mutants was related to ERAl gene function,<br>
transgenic plants containing a ERAl promoter fusion to a reporter GUS gene (made by inserting a 5 Kb fragment of the ERAl promoter into a promoterless GUS T-DNA plasmid), were constructed.  Analysis of the transgenic plants showed that ERAl is transcriptionally expressed in the epidermal tissue of Arabidopsis  and that this expression is guard-cell<br>
specific.  Expression of ERAl was also noted in the<br><br>
meristematic tissue of the plants and in root hairs.  The guard cell expression of ERA1 is consistent with the drought tolerance of the mutant as these cells are the major regulators of water transpiration through the plant. It would be expected that ERAl-regulated stomatal conductance would require expression of. the ERA1 gene in the guard cells.  Hence loss of ERA1 gene function results in guard cells which are more responsive to ABA which, in turn, leads to more drought responsive guard cell regulation.  Therefore, modification of Ftase expression or activity in higher plants, especially crop plants, will have profound effects on stomatal conductance and transpiration rates in the plants.<br>
The nature of the eral mutation in Arabidopsis demonstrates that inhibition of farnesylation will enhance ABA responses in a plant and alteration of this enzyme activity in crop species.  Inhibition of Ftase activity in crop plants can be achieved via a number of methods.  For example, antisense technology of cognate ERA1 genes in a variety of crop species can be used to reduce Ftase activity, thus increasing drought tolerance.  By specifically producing ERA1 antisense RNA in guard cells, the amount of Ftase synthesized can be reduced to a level which would mimic era mutant phenotypes.  The ERA1 promoter is regulated in a number of different tissues ranging from shoot meristems to root hairs.  By determining the elements of the ERA1 promoter which allow expression in specific tissues, it is possible to tailor the expression of antisense ERA1 to only one tissue or cell type, such as guard cells.<br>
Another method to inhibit Ftase activity in plants is the production of specific peptide inhibitors of farnesylation in transgenic plants.  In mammalian and yeast<br><br>
systems, the carboxyl terminal target sequence (CaaX, where C=cysteine, x=aliphatic, X=any amino acid) which allows the attachment of the farnesyl group to specific proteins has been clearly defined.  Peptides which mimic these target sequences have been made and shown to inhibit farnesylation of the endogenous target proteins in these systems. Moreover, CAIM is farnesylated in  vivo in AraJbidopsis.<br>
Thus, similar inhibitors can be applied to higher plants to competitively inhibit Ftase in  vivo.     Again, this can be<br>
done through expression of inhibitor peptides in transgenic plants by synthesizing the DNA sequence for a CaaX peptide and fusing it to a guard cell-specific promoter.  In both methods, using the appropriate promoters, antisense Ftase or peptide inhibitors can be specifically targeted and controlled.<br>
Thus, this invention provides a method of producing drought-tolerant plants comprising: preparing a nucleic acid construct which comprises a promoter operably-linked to a nucleic acid comprising or encoding antisense to SEQ ID N0:1, or nucleic acid comprising a functional equivalent of the antisense; inserting the nucleic acid construct into a vector; transforming a plant, tissue culture, or plant cells with the vector; and growing the plant or regenerating a plant from the tissue culture or plant cells,- wherein drought-tolerant plants are produced.  This method can be used wherein the nucleic acid is selected from the group consisting of 25-200 or more consecutive nucleotides complementary to SEQ ID N0:1, oligonucleotides consisting of 25 or more consecutive nucleotides of SEQ ID NO:l or its complement, or nucleic acid encoding a peptide inhibitor of farnesyl transferase<br><br>
In addition to stomatal regulation which is extremely sensitive to ABA, era plants also demonstrate delayed<br>
senescence under drought conditions, indicating that farnesylation negatively regulates a number of drought-induced responses in A.ra\bidopsis.  The era  plants grown<br>
under normal laboratory conditions take longer to turn yellow.  The mutant plants remained green and viable long after the wild-type had senesced and died.  Detached leaves of an era  mutant plant do not yellow as quickly as detached<br>
leaves of wild-type plants (Figure 8).  Similar-si2ed leaves which were developmentally identical were taken from wild-type and era plants and placed on agar-containing<br>
petri plates {See Example 7).  Normally, a wild-type leaf begins to lose chlorophyll about five days later and eventually bleachs.  The leaves of the mutant plants remained green for twice as long.  Because the leaves were in constant contact with the agar they were not drought stressed, indicating the reduced senescence of the eral<br>
mutant is not a drought-induced phenomenon.<br>
Moreover, under a 10 hr day/16 hr night cycle, the plant life cycle can be doubled versus the wild-type plants (3 months).  It appears therefore, that chlorophyll turnover and senescence signals are altered in the eral<br>
mutant.  For example, wild-type and mutant plants were grown in pots under well-watered conditions to stages of development where the wild-type plant leaves would begin to senesce (about the time of flower development).  At this time, developmentally-similar leaves were assayed for senescence-induced marker genes by northern blot analysis (Example 8).  Two genes, SAG12 and SAG13, in which transcription is normally induced during senescence in wild-type plants, were not induced in the eral mutant<br><br>
(Figure 9).  Further, CAB transcription is maintained (Figure 9).  Taken together, these results indicate the senescence induction program in eral  mutants is delayed<br>
compared to wild-type plants, showing that loss of farnesylation activity causes a retardation of the induction of senescence in the plant even under conditions wherein water stress is not an environmental factor.<br>
In addition to effects on senescence and water loss, the eral mutants show a difference in branching and<br>
flowering habit when grown under diurnal light cycles. Under continuous (24 hours light/day) light, the branching pattern of mutants does not differ from that of wild-type plants.  However, when given a dark period, the mutants do not produce as many lateral branches as wild-type plants. When measured, plants with loss of farnesylation activity produced only 2.4 branches per plant compared to 3.6 lateral branches per wild-type plant.  This represents a 3 0% decrease in lateral branches per plant.<br>
Flowering is affected by loss of Ftase activity as well.  Plants lacking Ftase activity produce more flowers per plant (25-30 buds/inflorescence) than wild-type plants (10-15 buds/inflorescence).  Thus, on average there are twice as  many flower buds are present on the mutants than on the wild-type plants.<br>
These pleiotrophic effects of the eral loss of<br>
function mutants on whole plant development indicate that the ERA1 gene can be a coordinate regulator of a collection of plant developmental functions.<br>
Until now, there was no known function for farnesylation in higher plants, including a role in ABA signal transduction.  Ftases have been found in a number of higher plants such as tomato and pea, so it is clear that<br><br>
this enzyme has functions across species boundaries. Furthermore, overproduction of farnesyl transferase target peptides or the use of farnesylation inhibitors completely inactivates Ftase in mammalian and yeast systems.  Thus, similar inhibitors can be applied to higher plants to inactivate Ftase in  vivo.     In both cases with the<br>
appropriate promoters, antisense Ftase or peptide inhibitors can be specifically targeted and controlled.<br>
The farnesylation deficient mutants are also supersensitive to exogenous auxin.  That these mutants show reduced branching and minor alterations in meristem organisation, can be explained by altered auxin regulation. Thus other hormone functions are affected in this mutant, which indicates that, in addition to ABA pathways, other hormone regulated pathways are controlled by Ftase activity.  These results demonstrate that the ERA1 gene provides a molecular mechanism to coordinate regulation of different hormone signaling molecules.<br>
In accordance with the present invention, the plants included within the scope of this invention are higher and lower plants of the plant kingdom.  Mature plants, seedlings and seeds are included in the scope of the invention.  A mature plant includes a plant at any stage in development beyond the seedling.  A seedling is a very young, immature plant in the early stages of development. Plant parts, protoplasts and tissue culture are also provided by this invention.<br>
Transgenic plants are included within the scope of the present invention which have the phenotype characterized by the erai mutation.  Seed of transgenic plants are provided by this invention and can be used to propagate more plants containing the constructs of this invention.<br><br>
ERA1 function in a number of crop plants can be inhibited to enhance the plant's response to adverse environmental conditions that require ABA-mediated signaling.  Control of farnesylation in higher plants regulates both embryonic and vegetative tissue response to this hormone (Cutler, et al.,   1996). The increased<br>
sensitivity translates into a faster response of the tissue to stress conditions which in turn confers increased protection of the plant to the environmental stress. Because this only requires the control of a single gene, ERA1, it should be possible to control farnesylation in a variety of plants by controlling the synthesis or activity of this enzyme.  Furthermore, the work described herein clearly indicates that altering the ABA signal transduction pathway by manipulating the genes that control the ABA response makes it possible to improve the plant's response to adverse water stress conditions.<br>
To produce transgenic plants of this invention, a construct comprising the gene encoding Ftase, or nucleic acid encoding its functional equivalent, and a promoter are incorporated into a vector through methods known and used by those of skill in the art.  The promoter can comprise all or part of SEQ ID NO:3.  The construct can also include any other necessary regulators such as terminators or the like, operably linked to the coding sequence.  It can also be beneficial to include a 5' leader sequence, such as the untranslated leader from the coat protein mRNA of alfalfa mosaic virus (Jobling, S.A. and Gehrke, L. (1987) Nature<br>
325:622-625) or the maize chlorotic mottle virus (MCMV)<br>
leader (Lommel, S.A., et al.    (1991) Virology  82:382-385).<br>
Those of skill in the art will recognize the applicability of other leader sequences for various purposes.<br><br>
Targeting sequences are also useful and can be incorporated into the constructs of this invention.  A targeting sequence is usually translated into a peptide which directs the polypeptide product of the coding nucleic acid sequence to a desired location within the cell, such as to the plastid, and becomes separated from the peptide after transit of the peptide is complete or concurrently with transit.  Examples of targeting sequences useful in this invention include, but are not limited to, the yeast mitochondrial presequence (Schmitz, et al.    (1989) Plant<br>
Cell   1:783-791), the targeting sequence from the pathogenesis-related gene (PR-1) of tobacco (Cornellisen, et al.    (1986) EMBO J.   5:37-40), vacuole targeting signals<br>
(Chrispeels, M.J. and Raikhel, N.V. (1992) Cell   68:613-<br>
616), secretory pathway sequences such as those of the ER or Golgi (Chrispeels, M.J. (1991) Ann. Rev.   Plant Physiol.<br>
Plant Mol.   Biol.   42:21-53).  Intraorganellar sequences may<br>
also be useful for internal sites, e.g.,   thylakoids in<br>
chloroplasts.  Theg, S.M. and Scott, S.V. (1993) Trends in<br>
Cell   Biol.   3:186-190.<br>
In addition to 51 leader sequences, terminator sequences are usually incorporated into the construct.  In plant constructs, a 31 untranslated region (31 UTR) is generally part of the expression plasmid and contains a polyA termination sequence.  The termination region which is employed will generally be one of convenience, since termination regions appear to be relatively interchangeable.  The octopine synthase and nopaline synthase termination regions, derived from the Ti-plasmid of A.   tumefaciens,   are suitable for such use in the<br>
constructs of this invention.<br><br>
Any suitable technique can be used to introduce the nucleic acids and constructs of this invention to produce transgenic plants with an altered genome.  For grasses such as maize, microprojectile bombardment (see for example, Sanford, J.C., et al. ,   U.S. Patent No. 5,100,792 (1992) can<br>
be used.  In this embodiment, a nucleotide construct or a vector containing the construct is coated onto small particles which are then introduced into the targeted tissue (cells) via high velocity ballistic penetration. The vector can be any vector which permits the expression of the exogenous DNA in plant cells into which the vector is introduced.  The transformed cells are then cultivated under conditions appropriate for the regeneration of plants, resulting in production of transgenic plants.<br>
Transgenic plants carrying the construct are examined for the desired phenotype using a variety of methods including but not limited to an appropriate phenotypic marker, such as antibiotic resistance or herbicide resistance, or visual observation of the time of floral induction compared to naturally-occurring plants.<br>
Other known methods of inserting nucleic acid constructs into plants include Agrobacterium-mediated transformation (see for example Smith, R.H., et al.,   U.S.<br>
Patent No. 5,164,310 (1992)), electroporation (see for example, Calvin, N-, U.S. Patent No. 5,098,843 (1992)), introduction using laser beams (see for example, Kasuya, T., et al., U.S. Patent No. 5,013,660 (1991)) or introduction using agents such as polyethylene glycol (see for example Golds, T. et al.    (1993) Biotechnology,   11:95-<br>
97), and the like.  In general, plant cells may be transformed with a variety of vectors, such as viral, episomal vectors, Ti plasmid vectors and the like, in<br><br>
accordance with well known procedures.  The method of introduction of the nucleic acid into the plant cell is not critical to this invention.<br>
The methods of this invention can be used with in<br>
planta  or seed transformation techniques which do not<br>
require culture or regeneration.  Examples of these techniques are described in Bechtold, N". , et al,    (1993) CR<br>
Acad.   Sci.   Paris/Life Sciences  316:118-93;   Chang, S.S., et<br>
al.    (1990) Abstracts or" the Fourth International   Conference<br>
on Arabidopsis Research,   Vienna, p. 28; Feldmann, K.A. and<br>
Marks, D.M (1987) Mol.   Gen.   Genet.   208:1-9;   Ledoux, L. , et<br>
al.    (1985) Arabidopsis  Inf.   Serv.   22:1-11;   Feldmann, K.A.<br>
(1992) In:   Methods in Arabidopsis  Research (Eds. Koncz, C,<br>
Chua, N-H, Schell, J.) pp. 274-289; Chee, et al.,   U.S.<br>
patent, Serial No. 5,376,543.<br>
The transcriptional initiation region may provide for constitutive expression or regulated expression.  In addition to the ERAI promoter, many promoters are available which are functional in plants.<br>
Constitutive promoters for plant gene expression include, but are not limited to, the octopine synthase, nopaline synthase, or mannopine synthase promoters from Agrobacterium,   the cauliflower mosaic virus (35S) promoter,<br>
the figwort mosaic virus (FMV) promoter, and the tobacco mosaic virus (TMV) promoter.  Constitutive gene expression in plants can also be provided by the glutamine synthase promoter (Edwards, et al.    (1990) PNAS  87:3459-3463), the<br>
maize sucrose synthetase 1 promoter (Yang, et al.    (1990)<br>
PNAS  87:4144-4148), the promoter from the Rol-C gene of the<br>
TLDNA of Ri plasmid (Sagaya, et al.    (1989) Plant Cell<br><br>
Physiol.   30:649-654), and the phloem-specific region of the pRVC-S-3A promoter (Aoyagi, et al.    (1988) Mo1.   Gen.   Genet. 223:179-185) .<br>
Heat-shock promoters, the ribulose-1,6-bisphosphate (RUBP) carboxylase small subunit (ssu) promoter, tissue specific promoters, and the like can be used for regulated expression of plant genes.  Developmentally-regulated, stress-induced, wound-induced or pathogen-induced promoters are also useful.<br>
The regulatory region may be responsive to a physical stimulus, such as light, as with the RUBP carboxylase ssu promoter, differentiation signals, or metabolites.  The time and level of expression of the sense or antisense orientation can have a definite effect on the phenotype produced.  Therefore, the promoters chosen, coupled with the orientation of the exogenous DNA, and site of integration of a vector in the genome, will determine the effect of the introduced gene.<br>
Specific examples of regulated promoters also include, but are not limited to, the low temperature Kinl and cor6.6 promoters (Wang, et al.    (1995) Plant Mol.   Biol.   28:605;<br>
Wang, et al.    (1995) Plant Mol.   Biol.   28:619-634), the ABA<br>
inducible promoter (Marcotte Jr., et al.    (1989) Plant Cell<br>
1:969-976), heat shock promoters, such as the inducible<br>
hsp70 heat shock promoter of Drosphilia melanogaster<br>
(Freeling, M., et al.    (1985) Ann. Rev. of Genetics  19:   297-<br>
323), the cold inducible promoter from B.   napus   (White,<br>
T.C., et al.    (1994) Plant Physiol.   106:917), the alcohol<br>
dehydrogenase promoter which is induced by ethanol (Nagao, R.T., et al.,   Miflin, B.J., Ed. Oxford  Surveys  of Plant<br><br>
Molecular  and  Cell   Biology,   Vol. 3, p 384-438, Oxford<br>
University Press, Oxford 1986) , the phloem-specific sucrose synthase ASUS1 promoter from Arahidopsis (Martin, et al.<br>
(1993) Plant  J. 4:367-377), the ACS1 promoter (Rodrigues-Pousada, et al. (1993) Plant Cell   5:897-911), the 22 kDa zein protein promoter from maize (Unger, et al.    (1993) Plant Cell   5:831-841), the psl lectin promoter of pea (de Pater, et al.    (1993) Plant  Cell   5:877-886), the phas promoter from Phaseolus  vulgaris   (Frisch, et al.    (1995) Plant J.   7:503-512), the lea promoter (Thomas, T.L. (1993) Plant Cell   5:1401-1410), the E8 gene promoter from tomato (Cordes, et al.    (1989) Plant Cell   1:1025-1034), the PCNA promoter (Kosugi, et al.    (1995) Plant  J.   7:877-886), the NTP303 promoter (Weterings, et al.    (1995) Plant J". 8:55-63), the OSEM promoter (Hattori, et al.    (1995) Plant  J. 7:913-925), the ADP GP promoter from potato (Muller-Rober, et al.    (1994) Plant  Cell   6:601-604), the Myb promoter from barley (Wissenbach, et al.    (1993) Plant  J.   4:411-422), and the plastocyanin promoter from Arabidopsis   (Vorst, et al. (1993) Plant J. 4:933-945).<br>
The vector can be introduced into cells by a method appropriate to the type of host cells (e.g.,<br>
transformation, electroporation, transfection).  For the purposes of this disclosure, the terms "transformed with", "transformant", "transformation", "transfect with", and "transfection" all refer to the introduction of a nucleic acid into a cell by one of the numerous methods known to persons skilled in the art.  Transformation of prokaryotic cells, for example, is commonly achieved by treating the<br><br>
cells with calcium chloride so as to render them "competent" to take up exogenous DNA, and then mixing such DNA with the competent cells.  Prokaryotic cells can also be infected with a recombinant bacteriophage vector.<br>
Nucleic acids can be introduced into cells of higher organisms by viral infection, bacteria-mediated transfer (e.g., Agrobacterium T-DNA delivery system),<br>
electroporation, calcium phosphate co-precipitation, microinjection, lipofection, bombardment with nucleic-acid coated particles or other techniques, depending on the particular cell type.  For grasses such as corn and sorghum, microprojectile bombardment as described, for example, in Sanford, J.C., et al.,   U.S. Patent No.<br>
5,100,792 (1992) can be used.  Other useful protocols for the transformation of plant cells are provided in Gelvin et<br>
al., 1992.  Suitable protocols for transforming and<br>
transfecting cells are also found in Sambrook. et al. , 1989.<br>
The nucleic acid constructs of this invention can also be incorporated into specific plant parts such as those described supra  through the transformation and transfection<br>
techniques described herein.<br>
To aid in identification of transformed plant cells, the constructs of this invention are further manipulated to include genes coding for plant selectable markers.  Useful selectable markers include enzymes which provide for resistance to an antibiotic such as gentamycin, hygromycin, kanamycin, or the like.  Similarly, enzymes providing for production of a compound identifiable by color change such as GUS (^-glucuronidase), or by luminescence, such as luciferase, are useful.<br>
For example, antisense Ftase can be produced by integrating a complement of the ERA1 gene linked to DNA<br><br>
comprising SEQ ID NO:3 into the genome of a virus that enters the host cells.  By infection of the host cells, the components of a system which permits the transcription of the antisense present in the host cells.<br>
When cells or protoplasts containing the antisense gene driven by a promoter of the present invention are obtained, the cells or protoplasts are regenerated into whole plants.  The transformed cells are then cultivated under conditions appropriate for the regeneration of plants, resulting in production of transgenic plants. Choice of methodology for the regeneration step is not critical, with suitable protocols being available for many varieties of plants, tissues and other photosynthetic organisms.  See, e.g.,   Gelvin S.B. and Schilperoort R.A.,<br>
eds. Plant Molecular Biology Manual,   Second Edition,   Suppl.<br>
1 (1995) Kluwer Academic Publishers, Boston MA, U.S.A.<br>
Transgenic plants carrying the construct are examined for the desired phenotype using a variety of methods including but not limited to an appropriate phenotypic marker, such as antibiotic resistance or herbicide resistance as described supra, or visual observation of<br>
their growth compared to the growth of the naturally-occurring plants under the same conditions.<br>
As used herein, the term transgenic plants includes plants that contain either DNA or RNA which does not naturally occur in the wild type (native) plant or known variants, or additional or inverted copies of the naturally-occurring DNA and which is introduced as described herein.  Transgenic plants include those into which isolated nucleic acids have been introduced and their descendants, produced from seed, vegetative propagation,<br><br>
cell, tissue or protoplast culture, or the like wherein such alteration is maintained.<br>
Such transgenic plants include, in one embodiment, transgenic plants which are angiosperms, both monocotyledons and dicotyledons.  Transgenic plants include those into which DNA has been introduced and their progeny, produced from seed, vegetative propagation, cell, tissue or protoplast culture, or the like.<br>
Seed can be obtained from the regenerated plant or from a cross between the regenerated plant and a suitable plant of the same species.  Alternatively, the plant can be vegetatively propagated by culturing plant parts under conditions suitable for the regeneration of such plant parts.<br>
In yet another aspect of this invention are provided plant tissue culture and protoplasts which contain DNA comprising antisense or an altered ERA1 nucleic acid operably linked to an ERAl promoter, which alters the response of the tissue culture or protoplasts to varying environmental conditions.<br>
The methods of this invention can also be used with in<br>
planta  or seed transformation techniques which do not<br>
require culture or regeneration.  Examples of these techniques are described in Bechtold, N., et al.    (1993) CR<br>
Acad.   Sci,   Paris/Life Sciences  316:118-93; Chang, S.S., et<br>
al.    (1990) Abstracts of  the Fourth  International   conference<br>
on Arabidopsis Research,   Vienna, p. 28; Feldmann, K.A. and<br>
Marks, D.M (1987) Mol.   Gen.   Genet.   208:1-9;   Ledoux, L., et<br>
al.    (1985) Arabidopsis  Inf.   Serv.   22:1-11; Feldmann, K.A.<br>
(1992) In:   Methods in Arabidopsis  Research (Eds. Koncz, C,<br><br>
Chua, N-H, Schell, J.) pp. 274-289; Chee, et al.,   U.S. patent, Serial No. 5,376,543.<br>
EMBODIMENTS<br>
The constructs and methods of this invention have numerous applications of commercial value, especially in the prevention of desiccation of plant tissues under periods of water stress.   Genetic manipulation of crop plants incorporating inhibitors of Ftase or inactivation of the gene encoding endogenous plant Ftase would allow such plants to withstand transitory environmental stress and can broaden the environments where these plants can be grown. Thus, improving tolerance of crop plants to cold, salt and drought stress, can improve the yield of the plants under such adverse conditions.<br>
The technology described herein can also be used to alter harvesting time and harvest quality of plants.  For example, overexpression of Ftase could lead to faster drying times of crops, such as corn and other grasses. Drying corn involves the use of large amounts of propane gas.  Drying times of crops such as hay, which dry naturally in the fields, could be shortened, making it less likely that rain would deteriorate the crop.<br>
In addition, inhibition of farnesylation in plants can also be used to control the senescence program of the plants so  that leaves can be maintained in a green state longer and fruits can be kept immature.  For example, if an antisense construct of ERA1 or CaaX box inhibitor protein construct was placed under the control of a senescence-induced promoter, the plant would induce an inhibitor of farnesylation as the senescence program was initiated, which would in turn inhibit senescence.  The result would be a plant which remains green or fruits which remain<br><br>
immature.  Thus, the plant could be kept producing a product, such as a vegetative part, flower or fruit much longer.  Thus, horticulturalists could produce plants which stayed green and continued to grow even though a wild-type plant of the same variety would senesce under the same conditions.  Cut flowers could be maintained longer.  Or a fruit could be kept immature, an important product for the vegetable industry where produce lifetime to market is extremely important.<br>
Further, the inhibition of Ftase in fruits and vegetables can reduce wilting.  Thus, wilting of produce during transport and shipping could be reduced.  Fruits and vegetables on the grocery shelf would also require less misting to keep them fresh and flavorful, and there would be less need to wax produce such as cucumber, apples and oranges to keep them from drying out.<br>
Less watering would also mean that fungal and bacterial attacks on the crops, or fruits and vegetables would be reduced.  For example, plant diseases in the field which result from splashing of plant pathogens from the soil to the plant leaves and fruits could be inhibited.<br>
In the field of horticulture, many drought-resistant varieties could be produced for landscaping and for use as ornamental house plants.  Especially valuable would be varieties of plants which are used for potting, as ornamentals inside or outside homes and offices, and which can survive infrequent water.  This would be a considerable boon for gardeners, especially during the droughty summer months where forgotten plants dry out quickly in the sun. Further, plants grown under trees and in other shady areas often experience drought conditions and limited light.  The technology provided herein can provide plant varieties which can better survive under these conditions.<br><br>
In a further embodiment, horitculturalists could find many uses for plants wherein lateral branching and/or flower numbers can be regulated with light/dark cycles. Examples of plants in which longer, unbranched stems would confer marketable advantage include roses, carnations, lilies, and the like.  The ability to increase the number of flowers or florets on the plant is also a highly valuable asset.  These traits could also be useful for many agricultural crops in that yields can be increased in a manner which also made harvesting of the crop easier.<br>
Another benefit of the constructs and methods provided herein is that the ERA1 promoter is active in the guard cells of leaves.  A portion of the ERA1 gene promoter can be fused to antisense nucleic acid to the ERA1 gene so Ftase activity is diminished only in the guard cells.<br>
A further embodiment is the use of the drought -resistant trait as a selectable marker of transformation in plants, plant cells and plant tissues.  One method of detecting transformation in plants consists of: (a) incorporating a nucleic acid construct comprising a promoter operably-linked to nucleic acid comprising antisense to SEQ ID N0:1 or nucleic acid comprising a functional equivalent of the antisense; (b) inserting the nucleic acid construct into a plant, plant cell or plant tissue;  (c) growing the plant, or regenerating a plant from the plant cell or plant tissue until stomates are formed; and (d) placing the plant or regenerated plant under conditions wherein the plant is drought stressed, wherein survival of the plant under drought conditions compared to untransformed plants is indicative of transformation.  Thus, this technology can be used as a selectable genetic marker, i.e., a visual marker especially<br><br>
when combined with plant selection and transformation schemes.<br>
In addition, without resorting to stressing a transgenic plant, the branching and/or flowering habit of plants with loss of Ftase function differs substantially from that of wild-type plants and can be used as a marker for successful transformation.  This method would be especially useful where in planta  transformation techniques<br>
have been applied.  Under diurnal light conditions, shoots of transgenic plants will demonstrate less lateral branching than that of untransformed shoots, thus indicating effective loss of Ftase activity without the use of selective antibiotic markers.<br>
EXEMPLIFICATION<br>
Example 1: Mutagenesis conditions<br>
Arabidopsis  plants used in this study were grown under<br>
continuous light in soil or agar containing petri plates, as described elsewhere (Haughn and Somerville 1986) Two distinct wild-types of Arabidopsis  were used: Meyerowitz's<br>
Colombia (Col) (Lelhe Seeds, Dripping Springs, TX) and Wassilewskija (Ws) (ABRC, Ohio State University).  Mutants isolated by screening T-DNA mutagenised seeds were in the Wassilewskija background.  These were obtained from the Ohio State Arabidopsis Seed stock collection (ABRC stock numbers CS2606-2654).  The T-DNA seed collection was comprised of 49 pools of 1200 fourth generation (T4) offspring derived from 100 mutagenized parents. A mutagenized parent was obtained by incubating overnight wild-type seeds (Tl) in a saturating Agrobacterium  culture<br><br>
containing a T-DNA plasmid. The seeds were washed in water and planted into pots. T2 generation seed were obtained from each plant and tested for kanamycin resistance (which is carried on the T-DNA).  Kanamycin-resistant plants were advanced to the T3 generation.  T4 generation plants were given to the stock center. Each pool was screened separately.<br>
Mutants isolated by screening fast neutron irradiated seeds were in Meyerowitz's Columbia background. Mutagenized wild-type seeds (Nl) were irradiated with 60 Gy of fast neutrons and grown to the next generation.  The N2 seeds were obtained as pools of approximately 11,000 seed generated from 1387 Nl parents.  Ten of these pools were screened separately for ABA supersensitive mutations.  In the initial screen the seeds had been stored at 4°C and were plated without imbibing. For all subsequent rescreening seeds were imbibed at 4°C for one week on 0.3 [iM  ABA and scored for cotyledon emergence after 5-7 days at 22°C in the light.<br>
Example 2: Genetic Analysis<br>
Mutant lines were backcrossed to wild type three times. T-DNA mutations were backcrossed to Ws and fast neutron mutants to MCol.  Segregation of the era phenotype<br>
was followed by plating P2 seeds on both 0.3 fxM ABA and imbibing four days at 4°C. Following imbibition, plates were transferred to room temperature in the light. Germination was measured as the presence or absence of expanded cotyledons in seedlings one week post-imbibition. Double mutants were constructed by crossing lines homozygous for each mutation following segregation and identifying lines that carried one of the mutant phenotypes.  The abi3 allele used in this study is ahi3-6<br><br>
(Nambara et al.,   1994) and abil is aJbil-l (Koornneef et al.,   1982).  The eral-2 allele was used as the era parent. Segregation analysis suggested eral partially suppressed the insensitivity of abil so F2 plants were first screened for insensitivity to 3 mM ABA and F3 seed from these plants were scored for sensitivity to 0.3 /iM ABA.  Putative double mutants were progeny tested in the F4 generation and verified by DNA polymorphisms for both Abil and Eral.  For eral abi3 double mutants, F2 seeds were screened for<br>
insensitivity (3 fM  ABA) and mature plants were scored for protruding carpels and immature green seeds (Nambara et<br>
al.,   1994). Putative double mutant lines were verified by<br>
DNA polymorphisms for both Abi3 and Eral.<br>
Example 3: DNA and RNA Analysis<br>
The methods for DNA (Dellaporta et al., 1983) and RNA<br>
(Verwoerd et al.,   1989) extractions were as described.<br>
High stringency Southerns were carried out at 65C as described elsewhere (Sambrook et al.,   1989).  All screening<br>
of genomic and cDNA libraries were done on Gelman BioTrace NT membranes according to manufacturers specifications (Gelman Sciences).  To clone insertion junctions between T-DNA and genomic DNA in the T12W mutant (eral-l) a library<br>
of T12W DNA was made in Y-ZAPII (Stratagene).  Genomic Southern blots of T12W DNA digested with EccRl   and probed with right border (RB) T-DNA produced three bands (13 Kb, 7.0 Kb and 8.0 Kb).  Subsequent analysis with other restriction enzymes verified the 7 and 8 Kb bands as containing the insertion points of T-DNA and flanking plant<br><br>
DNA.  These fragments were cloned by digesting genomic DNA with EcoRI  and the DNA was fractionated using a Prep Cell<br>
(Pharmacia).  Fractions containing the 7 and 8 Kb fragments were identified by Southern analysis of pooled fractions using the RB as a probe.  These pools were ligated to the Y-ZAPII arms according to the manufacturer's instructions<br>
(Stratagene).  A library of 40,000 recombinants was screened.  Five positive plaques were identified and excised plasmid forms of the cloned inserts were isolated according to the manufacturer (Stratagene). Two plasmids, designated pL4B and pL7, which hybridized with RB probe were characterized further.  Using pBluescript, a 2.3 Kb EcoRI BamHI fragment from pL4B was subcloned and designated<br>
pSCIO and 1.3 Kb Hindi 11 BairiHI fragment from pL7 was subcloned and designated pSCll.  These two plasmids contain approximately 1.2 Kb of T-DNA attached to the flanking plant genomic DNA.  pSCIO was used as a probe to screen an Arabidopsis cDNA library, PRL2 1-ZipLox (ABRC, Stock CD4 -7).  Five positive cDNA's were identified and the longest cDNA insert, pZL23, was used to screen an additional 200,000 PRL2 phage.  From this screening a longer cDNA insert, pZL51, (1.35 Kb) was isolated.  These clones were<br>
sequenced and used to screen 30,000 y-ZAPI-1 plaques made from partially digested EcoRI wild-type Columbia genomic DNA.  Construction of this library was as described above except without fractionation of the digested DNA.  Four positive clones were identified. The inserts were excised and a 6 Kb region encompassing the pZL51 clone was completely sequenced.  This genomic insert and a!4 Kb genomic insert isolated from a 1-FIX Lansberg erecta genomic library by similar methods {ABRC Stock CD4-8) were<br><br>
used as probes to size deletions in the fast neutron mutants (eral-2, eral-3).<br>
Example 4: Protein Farnesvl transferase Assay<br>
Farnesyl transferase (Ftase) assays were carried out using cell free extracts from wild-type and mutant plants as the Ftase source and synthetic heptapeptides as substrate for the reaction.  Peptides were purchased from Genemed Biotechnologies, Inc.  The appropriate sequences for the peptides were based on the data of Randall et al.<br>
(1993); these were GGCCAIM (-CAIM) and GGCCAIL£-CAIL). Solutions of peptides were prepared in 100% dimethyl sulfoxide (DMSO) containing 10 mM dithiotreitol (DTT) and diluted in 10 mM'DTT  in the absence of DMSO.  The source of Ftase for transferase assays were soluble protein extracts from the buds of three week old plants.  For both wild-type and mutant plants 1 g (fresh weight) of buds were collected and homogenized in a buffer containing 50 mM Hepes (pH 7.5), 1 mM MgCl2, 1 mM EGTA, 5 mM DTT, 2 iiq/ml leupeptin, 2 /ig/ml aprotinin, and ImM PMSF. Cellular debris and membranes were removed by centrifugation at 4°C at 10 000 g  for 10 min and 100 000 g  for 30 min. The supernatant<br>
was saved and quantification of total soluble protein was assessed according to Bradford (1976).  Soluble extracts were incubated at 30°C with a peptide substrate and 3H-farnesyl pyrophosphate (Amersham) for 4 0 min.  Each reaction mixture contained the following components in a final volume of 25 fil:   50 mM Hepes (pH 7.5), 5 mM MgCl2, 5 mM DTT, 50 pM  peptide, 0.5 /iM  [3H] FPP, and 100 M9 °f soluble protein extract. As a control one reaction contained soluble extracts that had been boiled for 5 min. Reactions were terminated by adding EDTA to a final<br><br>
concentration of 50 mM and then spotted onto Silica Gel 60 thin-layer chromatography plates (Millipore).  Plates were developed with n-propanol-water (7:3 v/v) for 4-5 h.  The plates were dried, sprayed with En3Hance (New England Nuclear), and exposed to Kodak X-OMAT AR film at -70°C for 4 days.<br>
Example 5: ERA1-GUS aene constructs and transgenic plants<br>
ERA1-GUS fusion constructs were made by inserting a 5 Kb EcoRl-Hindlll genomic fragment of the ERA1 promoter in to promoterless GUS T-DNA plasmid (pBI121).  This construct was transformed into Agrobacterium strain L.B4404.  The Agrobacterium  was grown to (0.8 O.D. units, 595 nm) and<br>
then washed extensively in water, cells were resuspended in 10% glycerol and then pulsed in an electroporator at 200 Ohms 25 fiF,   2.5 kvolts.  Cells were then plated on LB media plus Ampicillin (100 ^g/ml) and grown for 2 days at 28°C. Ampicillin-resistant transformants were used in subsequent plant transformation experiments.  Transgenic plants were made by vacuum infiltrating plants with a saturating Agrojbacterium culture (0.8 O.D. units, 595 nm) .  Wild-type<br>
plants were grcv;n under standard laboratory conditions (25°C, 150 iiE m'2  sec _1, humidity, constant light) until they produced their first bolts at approximately 5 weeks. Stems were removed and the plants were submerged in a solution of AgrroJbacterium and placed under vacuum (20<br>
mBars) for 5 minutes.  After the vacuum was broken plants were transferred to soil and allowed to recover under standard laboratory conditions.  Plants produced new flowers and seed which was harvested after 2 months and allowed to dry for 2 weeks.  Seed from individual plants were planted onto minimal media MS plates containing 50<br><br>
/ig/ml Kanamycin.  Green kanamycin-resistant plantlets were identified and moved to soil after 2 weeks and allowed to grow for seed.  These seeds were germinated and the seedlings tested for GUS activity using the fluorescent GUS substrate Imagene Green (Molecular Probes, Eugene, Oregon). The assay was done by suspending seedlings in GUS-buffer (50 mM Sodium phosphate pH 7.0, 10 mM EDTA, 0.1% Triton X-100, 0.1% Sodium sarcosyl, 4 mM Imagene Green) for 2-4 hours in the dark at room temperature.  Seedlings were directly viewed under a microscope (25X) using blue excitation light for a positive fluorescent signal which is yellow on a red chlorophyll autofluorescent background.<br>
Example 6: Drought Experiments.<br>
Six single wild-type and six eral-2 seedlings were<br>
grown for four weeks in constant light with constant watering (25°C, 150 fiE  m"2 sec"1 , 70% humidity, constant<br>
light).  Pots were then covered in tinfoil to retard soil evaporation and the plant and pot was weighed.  At this time, plants were no longer watered and every day each pot was weighed.  At the end of the experiment plants were removed and the pots were allowed to dry another two weeks and then weighed to determine the weight of the dry soil and pot.  This weight was subtracted from each sample.<br>
Rumple 7:  Age-related changes in detached leaves.<br>
The chlorophyll content in adult rosette leaves in wild-type Columbia and eral-2 mutants were compared after detachment from these Arabidopsis plants.  The plants were grown under constant light (150 /iEinsteins/m^sec) and temperature (22°C) to a similar developmental age (3 weeks following germination).  At this time, the fifth leaves of<br><br>
several plants which had emerged after germination were removed and placed on 0.8% agar containing minimal salts in petri plates.  The plates were sealed and placed at 22°C under constant light (50 /iEinsteins/m2-sec) for 12 days. Photographs were taken and color comparisons made at 0, 3, 6, 9, and 12 days.<br>
Example 8: Determinat j on of transcript levels for selected Genes in aaina leaves.<br>
Plants were grown under constant light (150 ^Einsteins/m2sec) and temperature (22°C) to a similar developmental age {4 weeks following germination), at which time the fifth rosette leaf which had emerged following germination was removed from mutant (eral-2) and wild-type plants.  These leaves were assayed for the expression of three genes (CAB, SAG12, SAG13) by northern analysis (0, 4, 8 days after the plants bolted).  The CAB gene encodes the Arabidopsis  chlorophyll binding protein which is involved<br>
in capturing light for photosynthesis.  CAB is required for the green color of the leaf and is a good marker of chlorophyll turnover in the plant.  CAB in wild-type plants shows transcript level reduction once senescence is induced.  No transcript level reduction was observed in aging leaves of eral-2 mutants.  SAG12 and SAG13 are<br>
Arabidopsis  genes cloned by differential expression during<br>
senescence (SAG stands for senescence Activated Gene). Transcription of both genes is induced during the onset of senescence in wild-type Arabidopsis plants.  It was<br>
determined that these genes were not induced under the same developmental conditions in the eral-2 mutants.<br><br>
References<br>
Baskin, JW and Baskin,.CC (1971) Can J Bot 50:277.<br>
Chandler, PM and Robertson, M (1994) Gene expression regulated by abscisic acid and its relationship to stress tolerance. Ann Rev Plant Physiol and Plant Mol Biol 45:113-141.<br>
Chen, W-J, Anders, DA, Goldstein, JL, Russell, DW, Brown, MS (1991) Cell 66:327<br>
Cutler, S, Ghassemian, M, Bonetta, D, Cooney, S, McCourt, P (1996) A protein farnesyl transferase involved in abscisic acid signal transduction in Arahidopsis.   Science 273:1239-<br>
1241.<br>
Dellaporta, S. L., Wood, J. and Hicks, J. B. (1983). A plant DNA minipreparation: version II. Plant Mol. Biol. Rep. 1:19-21.<br>
Eisenmann, D. M. and Kim, S. K. (1994). Signal transduction and cell fate specification during Caenorha.bd.itis  elegans<br>
vulval development. Curr. Opin. Genet. Dev. 4:508-516.<br>
Ellington, A. (1987). Preparation and Analysis of DNA. In Current Protocols in Molecular Biology F. Ausubel et aJ. eds. (Boston, Greene), pp 2 . 0.1-2 .12.5 .<br>
Goodman, LE, Perou, CM, Fujiyama, A, Tamanoi, F (1988) Yeast 4:271<br><br>
Haughn, G. and  Somerville C. R. (1986) . Sulfonylurea-resistant mutants of Arabidopsis thaliana.  Mol. Gen. Genet. 204 :430-434 .<br>
Koornneef, M, Reuling, G and Karssen, CM (1984) The isolation and characterization of abscisic acid-insensitive mutants of Arabidopsis thaliana. Physiol. Plant. 61:377-383.<br>
Leung, J, Bouvier-Durand, M, Morris, P-C, Guerrier, D, Chefdor, F, and Giraudat, J (1994) Arabidopsis  ABA-<br>
response gene ABU: features of a calcium-modulated protein phosphatase. Science 264:1448-1452.<br>
Meyer, K, Leube, MP, and Grill, E (1994) A protein<br>
phosphatase 2C involved in ABA signal transduction in Arabidopsis   thaliana.   Science 264:1452-1455.<br>
Randall, SK, Marshall, MS, Crowell, DN (1993) Protein isoprenylation in suspension-cultured tobacco cells. Plant Cell 5:433-442.<br>
Reid, JB, and Howell, SH (1995) The function of hormones in plant growth and development. In Plant Hormones Physiology, Biochemistry and Molecular Biology. 2nd ed. P. Davies ed. (Dortrecht Kluwer) pp. 448-485.<br>
Sambrook, J., E. F. Fritsch and Maniatis, T. (1989). Molecular Cloning: A Laboratory Manual, Second edition (Cold Spring Harbor, New York: Cold Spring Harbor Laboratory Press)<br><br>
Schafer, WR, and Rine, J (1992) Protein Prenylation: Genes, Enzymes, Targets and Functions. Ann Rev Genet 30:209-237.<br>
Shirley, BW, Hanley, S, Goodman, HM (1992) Plant Cell 4: 333<br>
Verwoerd, T. C, Dekker, B. M. M. and Hoekema, A. (1989). A small-scale procedure for the rapid isolation of plant RNA's. Nucleic Acids Research 17:2362.<br>
Yang, Z, Cramer, CL, and Watson, JC (1993) Protein farnesyl transferase in plants. Plant Physioloqv 101:667-674.<br><br><br>
WE CLAIM;<br>
1,	A method of producing a drought tolerant plant comprising:<br>
a)	providing a nucleic aeid construct comprising a promoter operably linked to a nucleic        acid that inhibits farnesyl transferase beta activity;<br>
b)	inserting said nucleic construct into a vector;<br>
c} transforming a plant, tissue culture, or a plant cell with the vector to obtain a plant, tissue culture or a plant cell with decreased farnesyl transferase beta activity; and<br>
d) growing said plant or regenerating a plant from said tissue cuhure or plant cell,    wherein a drought tolerant plant is produced,<br>
2.	The method as claimed in claim 1, wherein said promoter is a constitutive<br>
promoter, an ABA inducible promoter, a tissue specific promoter or a guard cell<br>
specific promoter.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcxNy1tYXMtMTk5OCBhYnN0cmFjdC1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1717-mas-1998 abstract-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcxNy1tYXMtMTk5OCBhYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1717-mas-1998 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcxNy1tYXMtMTk5OCBjbGFpbXMtZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">1717-mas-1998 claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcxNy1tYXMtMTk5OCBjbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">1717-mas-1998 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcxNy1tYXMtMTk5OCBjb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">1717-mas-1998 correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcxNy1tYXMtMTk5OCBjb3JyZXNwb25kZW5jZS1wby5wZGY=" target="_blank" style="word-wrap:break-word;">1717-mas-1998 correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcxNy1tYXMtMTk5OCBkZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">1717-mas-1998 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcxNy1tYXMtMTk5OCBkZXNjcmlwdGlvbiAoY29tcGx0ZXQpLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">1717-mas-1998 description (compltet)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcxNy1tYXMtMTk5OCBkcmF3aW5ncy1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1717-mas-1998 drawings-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcxNy1tYXMtMTk5OCBkcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">1717-mas-1998 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcxNy1tYXMtMTk5OCBmb3JtLTE5LnBkZg==" target="_blank" style="word-wrap:break-word;">1717-mas-1998 form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcxNy1tYXMtMTk5OCBmb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">1717-mas-1998 form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcxNy1tYXMtMTk5OCBmb3JtLTI2LnBkZg==" target="_blank" style="word-wrap:break-word;">1717-mas-1998 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcxNy1tYXMtMTk5OCBmb3JtLTQucGRm" target="_blank" style="word-wrap:break-word;">1717-mas-1998 form-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcxNy1tYXMtMTk5OCBmb3JtLTYucGRm" target="_blank" style="word-wrap:break-word;">1717-mas-1998 form-6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTcxNy1tYXMtMTk5OCBwZXRpdGlvbi5wZGY=" target="_blank" style="word-wrap:break-word;">1717-mas-1998 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="225579-fungicidal-mixture.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="225581-improved-process-for-obtaining-insulin-precursors-having-correctly-bonded-cystine-bridges.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>225580</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1717/MAS/1998</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>52/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>26-Dec-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>19-Nov-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>31-Jul-1998</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>PERFORMANCE PLANTS, INC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>C/O. QUEEN&#x27;S UNIVERSITY, KINGSTON, ONTARIO K7L 3N6,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>PETER McCOURT</td>
											<td>456 COLLEGE STREET, TORONTO, ONTARIO M6G 4A3,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>MAJID GHASSEMIEN</td>
											<td>1220-1230 CHARLES STREET WEST, TORONTO, ONTARIO M4Y 1R5,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>SEAN CUTLER</td>
											<td>460 COLLEGE STREET, NIAGRA FALLS, NEW YORK 14051,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>DARIO BONETTA</td>
											<td>1414-1424 WELLESLEY STREET WEST, TORONT6O, ONTARIO M4Y 1G1,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/82</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/054,474</td>
									<td>1997-08-01</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/225580-a-method-of-producing-a-drought-tolerant-plant by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:27:54 GMT -->
</html>
