<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/225662-a-polynucleotide-encoding-a-modified-hiv-env-env-polypeptide by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:52:25 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 225662:A POLYNUCLEOTIDE ENCODING A MODIFIED HIV ENV ENV POLYPEPTIDE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A POLYNUCLEOTIDE ENCODING A MODIFIED HIV ENV ENV POLYPEPTIDE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A polynucleotide encoding a modified HIV Env polypeptide wherein the polypeptide has at least one ammo acid deleted or replaced as compared to wild-type in the region corresponding to residues 420 to 436 numbered relative to HXB-2 (SEQ ID NO: 1).</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>MODIFIED HIV ENV POLYPEPTIDES<br>
Technical Field<br>
The invention relates generally to modified HIV envelope (Env) polypeptides which<br>
are useful as immunizing agents or for generating an immune response in a subject, for<br>
example a cellular immune response or a protective immune response. More particularly, the<br>
invention relates Env polypeptides such as gp120, gp140 or gp160, wherein at least one of<br>
the native P-sheet configurations has been modified. The invention also pertains to methods<br>
of using these polypeptides to elicit an immune response against a broad range of HIV<br>
subtypes.<br>
Background of the Invention<br>
The human immunodeficiency virus (HIV-1, also referred to as HTLV-III, LAV or<br>
HTLV-III/LAV) is the etiological agent of the acquired immune deficiency syndrome (AIDS)<br>
and related disorders, (see, e.g., Barre-Sinoussi, et al., (1983) Science 220:868-871; Gallo et<br>
al. (1984) Science 224:500-503; Levy et al., (1984) Science 225:840-842; Siegal et al., (1981)<br>
N. Engl. J. Med. 305:1439-1444). AIDS patients usually have a long asymptomatic period<br>
followed by the progressive degeneration of the immune system and the central nervous<br>
system. Replication of the virus is highly regulated, and both latent and lytic infection of the<br>
CD4 positive helper subset of T-lymphocytes occur in tissue culture (Zagury et al., (1986)<br>
Science 231:850-853). Molecular studies of HIV-1 show that it encodes a number of genes<br>
(Ratner et al., (1985) Nature 313:277-284; Sanchez-Pescador et al., (1985) Science 227:484-<br>
492), including three structural genes -- gag, pol and env -- that are common to all<br>
retroviruses. Nucleotide sequences from viral genomes of other retroviruses, particularly<br>
HIV-2 and simian immunodeficiency viruses, SIV (previously referred to as STLV-III), also<br>
contain these structural genes. (Guyader et al., (1987) Nature 326:662-669; Chakrabarti et<br>
al., (1987) Nature<br>
The envelope protein of HIV-1, HIV-2 and SIV is a glycoprotein of about 160 kd<br>
(gp160). During virus infection of the host cell, gp160 is cleaved by host cell proteases to<br>
form gp120 and the integral membrane protein, gp41. The gp41 portion is anchored in the<br>
membrane bilayer of virion, while the gp120 segment protrudes into the surrounding<br>
environment. gpl20 and gp41 are more covalently associated and free gpl20 can be released<br>
from the surface of virions and infected cells.<br>
As depicted in Figure 1, crystallography studies of the gpl20 core polypeptide<br>
indicate that this polypeptide is folded into two major domains having certain emanating<br>
structures. The inner domain (inner with respect to the N and C terminus) features a two-<br>
helix, two-stranded bundle with a small five-stranded p-sandwich at its termini-proximal end<br>
and a projection at the distal end from which the V1 /V2 stem emanates. The outer domain is<br>
a staked double barrel that lies along side the inner domain so that the outer barrel and inner<br>
bundle axes are approximately parallel. Between the distal inner domain and the distal outer<br>
domain is a four-stranded bridging sheet which holds a peculiar minidomain in contact with,<br>
but distinct from, the inner, the outer domain, and the V1 /V2 domain. The bridging sheet is<br>
composed of four ß-strand structures (ß-3, ß-2, ß-21, ß-20, shown in Figure 1). The bridging<br>
region can be seen in Figure 1 packing primarily over the inner domain, although some<br>
surface residues of the outer domain, such as Phe 382, reach into the bridging sheet to form<br>
part of its hydrophobic core.<br>
The basic unit of the ß-sheet conformation of the bridging sheet region is the ß-strand<br>
which exists as a less tightly coiled helix, with 2.0 residues per turn. The ß-strand<br>
conformation is only stable when incorporated into a ß-sheet, where hydrogen bonds with<br>
close to optimal geometry are formed between the peptide groups on adjacent ß-strands; the<br>
dipole moments of the strands are also aligned favorably. Side chains from adjacent residues<br>
of the same strand protrude from opposite sides of the sheet and do not interact with each<br>
other, but have significant interactions with their backbone and with the side chains of<br>
neighboring strands. For a general description of P-sheets, see, e.g., T.E. Creighton, Proteins:<br>
Structures and Molecular Properties (W.H. Freeman and Company, 1993); and A.L.<br>
Lehninger, Biochemistry (Worth Publishers, Inc., 1975).<br>
The gp120 polypeptide is instrumental in mediating entry into the host cell. Recent<br>
studies have indicated that binding of CD4 to gp120 induces a conformational change in Env<br>
that allows for binding to a co-receptor (e.g. a chemokine receptor) and subsequent entry of<br>
the virus into the cell. (Wyatt, R., et at. (1998) Nature 393:705-711; Kwong, P., et al.(1998)<br>
Nature 393:648-659). Referring again to Figure 1, CD4 is bound into a depression formed at<br>
the interface of the outer domain, the inner domain and the bridging sheet of gp120.<br>
Immunogenicity of the gp120 polypcptide has also been studied. For example,<br>
individuals infected by HIV-1 usually develop antibodies that can neutralize the virus in in<br>
vitro assays, and this response is directed primarily against linear neutralizing determinants in<br>
the third variable loop of gpl20 glycoprotein (Javaherian, K., et al. (1989) Proc. Nail Acad.<br>
Sci. 86:6786-6772; Matsushita, M., et al. (1988) J. Virol. 62:2107-2144; Putney, S., et al.<br>
(1986) Science 234:1392-1395; Rushe, J. R., et al. (1988) Proc. Nat. Acad. Sci. USA 85:<br>
3198-3202.). However, these antibodies generally exhibit the ability to neutralize only a<br>
limited number of HIV-1 strains (Matthews, T. (1986) Proc Natl. Acad. Sci. USA. 83:9709-<br>
9713; Nara, P. L., et al. (1988) J: Virol. 62:2622-2628; Palker, T. J., et al. (1988) Proc. Natl.<br>
Acad. Sci. USA. 85:1932-1936). Later in the course of HIV infection in humans, antibodies<br>
capable of neutralizing a wider range of HIV-1 isolates appear (Barre-Sinoussi, F., et al.<br>
(1983) Science 220:868-871; Robert-Guroff, M., el al. (1985) Nature (London) 316:72-74;<br>
Weis, R., et al. (1985) Nature (London) 316:69-72; Weis, R., et al. (1986) Nature (London)<br>
324:572-575).<br>
Recent work done by Stamatatos et al (1998) AIDS Res Hum Retroviruses<br>
14(13):1129-39, shows that a deletion of the variable region 2 from a HIV-1SF162 virus, which<br>
utilizes the CCR-5 co-receptor for virus entry, rendered the virus highly susceptible to serum-<br>
mediated neutralization. This V2 deleted virus was also neutralized by sera obtained from<br>
patients infected not only with clade B HIV-1 isolates but also with clade A, C, D and F HIV-<br>
1 isolates. However, deletion of the variable region 1 had no effect. Deletion of the variable<br>
regions 1 and 2 from a LAI isolate HIV-I111B also increased the susceptibility to neutralization<br>
by monoclonal antibodies whose epitopes are located within the V3 loop, the CD4-binding<br>
site, and conserved gp120 regions (Wyatt, R., et al. (1995) J Virol. 69:5723-5733). Rabbit<br>
immunogenicity studies done with the HIV-1 virus with deletions in the V1/V2 and V3<br>
region from the LAI strain, which uses the CXCR4 co-receptor for virus entry, showed no<br>
improvement in the ability of Env to raise neutralizing antibodies (Leu et al. (1998) AIDS<br>
Res. and Human Retroviruses. 14:151-155).<br>
Further, a subset of the broadly reactive antibodies, found in most infected<br>
individuals, interferes with the binding of gp120 and CD4 (Kang, C.Y., et al. (1991) Proc.<br>
Natl. Acad. Sci. USA. 88:6171-6175; McDougal, J. S., et al. (1986)/ Immunol. 137.2937-<br>
2944). Other antibodies are believed to bind to the chemokine receptor binding region after<br>
CD4 has bound to Env (Thali et al. (1993)/ Virol. 67:3978-3988). The fact that neutralizing<br>
antibodies generated during the course of HIV infection do not provide permanent antiviral<br>
effect may in part be due to the generation of "neutralization escapes" virus mutants and to<br>
the general decline in the host immune system associated with pathogenesis. In contrast, the<br>
presence of pre-existing neutralizing antibodies upon initial HIV-1 exposure will likely have<br>
a protective effect.<br>
It is widely thought that a successful vaccine should be able to induce a strong,<br>
broadly neutralizing antibody response against diverse HIV-1 strains (Montefiori and Evans<br>
(1999) AIDS Res. Hum. Ret. I 5(8):689-698; Bolognesi, D.,P., et al. (1994) Ann. Int. Med.<br>
8:603-611; Haynes, B., F., et al. (1996) Science ;271: 324-328.). Neutralizing antibodies, by<br>
attaching to the incoming virions, can reduce or even prevent their infectivity for target cells<br>
and prevent the cell-to-cell spread of virus in tissue culture (Hu et al. (1992) Science. 255:456-<br>
459; Burton, D.,R. and Montefiori, D. (1997) AIDS ll(suppl. A): 587-598). However as<br>
described above, antibodies directed against gp120 do not generally exhibit broad antibody<br>
responses against different HIV strains.<br>
Currently, the focus of vaccine development, from the perspective of humoral<br>
immunity, is on the neutralization of primary isolates that utilize the CCR5 chemokine co-<br>
receptor believed to be important in virus entry (Zhu, T., et al. (1993) Science 261:1179-<br>
1181; Fiore, J., et al. (1994) Virology; 204:297-303). These viruses are generally much more<br>
resistant to antibody neutralization than T-cell line adapted strains that use the CXCR4 co-<br>
receptor, although both can be neutralized in vitro by certain broadly and potent acting<br>
monoclonal antibodies, such as IgGlbl2, 2G12 and 2F5 (Trkola, A., et al. (1995) J. Virol.<br>
69:6609-6617; D'Sousa PM, et al (1997)7. Infect. Dis. 175:1062-1075). These monoclonal<br>
antibodies are directed to the CD4 binding site, a glycosylation site and to the gp41 fusion<br>
domain, respectively. The problem that remains, however, is that it is not known how to<br>
induce antibodies of the appropriate specificity by vaccination. Antibodies (Abs) elicited by<br>
gpl20 glycoprotein from a given isolate are usually only able to neutralize closely related<br>
viruses generally from similar, usually from the same, HIV-1 subtype.<br>
Despite the above approaches, there remains a need for Env antigens that can elicit an<br>
immunological response (e.g., neutralizing and/or protective antibodies) in a subject against<br>
multiple HIV strains and subtypes, for example when administered as a vaccine. The present<br>
invention solves these and other problems by providing modified Env polypeptides (e.g.,<br>
gp120) to expose epitopes in or near the CD4 binding site.<br>
Summary of the Invention<br>
In accordance with the present invention, modified HIV Env polypeptides are<br>
provided. In particular, deletions and/or mutations are made in one or more of the 4-ß<br>
antiparallel-bridging sheet in the HIV Env polypeptide. In this way, enough structure is left<br>
to allow correct folding of the polypeptide, for example of gp120, yet enough of the bridging<br>
sheet is removed to expose the CD4 groove, allowing an immune response to be generated<br>
against epitopes in or near the CD4 binding site of the Env polypeptide (e.g.. gp120).<br>
In one aspect, the invention includes a polynucleotide encoding a modified HIV Env<br>
polypeptide wherein the polypeptide has at least one modified (e.g., deleted or replaced)<br>
amino acid residue deleted in the region corresponding to residues 421 to 436 relative to<br>
HXB-2, for example the constructs depicted in Figures 6-29 (SEQ ID N0s:3 to 26). In<br>
certain embodiments, the polynucleotide also has the region corresponding to residues 124-<br>
198 of the polypeptide HXB-2 (e.g., V1/V2) deleted and at least one amino acid deleted or<br>
replaced in the regions corresponding to the residues 119 to 123 and 199 to 210, relative to<br>
HXB-2. In other embodiments, these polynucleotides encode Env polypeptides having at<br>
least one amino acid of the small loop of the bridging sheet (e.g., amino acid residues 427 to<br>
429 relative to HXB-2) deleted or replaced. The amino acid sequences of the modified<br>
polypeptides encoded by the polynucleotides of the present invention can be based on any<br>
HIV variant, for example SF162.<br>
In another aspect, the invention includes immunogenic modified HIV Env<br>
polypeptides having at least one modified (e.g., deleted or replaced) amino acid residue<br>
deleted in the region corresponding to residues 421 to 436 relative to HXB-2, for example a<br>
deletion or replacement of one amino acids in the small loop region (e.g., amino acid residues<br>
427 to 429 relative to HXB-2). These polypeptides may have modifications (e.g., a deletion<br>
or a replacement) of at least one amino acid between about amino acid residue 420 and amino<br>
acid residue 436, relative to HXB-2 and, optionally, may have deletions or truncations of the<br>
V1 and/or V2 regions. The immunogenic, modified polypeptides of the present invention can<br>
be based on any HIV variant, for example SF162.<br>
In another aspect, the invention includes a vaccine composition comprising any of the<br>
polynucleotides encoding modified Env polypeptides described above. Vaccine<br>
compositions comprising the modified Env polypeptides and, optionally, an adjuvant are also<br>
included in the invention.<br>
In yet another aspect, the invention includes a method of inducing an immune<br>
response in subject comprising, administering one or more of the polynucleotides or<br>
constructs described above in an amount sufficient to induce an immune response in the<br>
subject. In certain embodiments, the method further comprises administering an adjuvant to<br>
the subject.<br>
In another aspect, the invention includes a method of inducing an immune response in<br>
a subject comprising administering a composition comprising any of the modified Env<br>
polypeptides described above and an adjuvant. The composition is administered in an<br>
amount sufficient to induce an immune response in the subject.<br>
In another aspect, the invention includes a method of inducing an immune response in<br>
a subject comprising<br>
(a) administering a first composition comprising any of the polynucleotides described<br>
above in a priming step and<br>
(b) administering a second composition comprising any of the modified Env<br>
polypeptides described above, as a booster, in an amount sufficient to induce an immune<br>
response in the subject. In certain embodiments, the first composition, the second<br>
composition or both the first and second compositions further comprise an adjuvant.<br>
These and other embodiments of the subject invention will readily occur to those of<br>
skill in the art in light of the disclosure herein.<br>
Brief Description of the Accompanying Drawings<br>
Figure 1 is a schematic depiction of the tertiary structure of the HIV-1HXB-2 Env gp120<br>
polypeptide, as determined by crystallography studies.<br>
Figures 2A-C depict alignment of the amino acid sequence of wild-type HIV-1HXB-2<br>
Env gp160 polypeptide (SEQ ID NO:1) with amino acid sequence of HIV variants SF162<br>
(shown as "162") (SEQ ID NO:2), SF2, CM236 and US4. Arrows indicate the regions that<br>
are deleted or replaced in the modified polypeptides. Black dots indicate conserved cysteinc<br>
residues. The star indicates the position of the last amino acid in gp120.<br>
Figures 3A-J depict alignment of nucleotide sequences of polynucleotides encoding<br>
modified Env polypeptides having V1 /V2 deletions. The unmodified amino acid residues<br>
encoded by these sequences correspond to wildtype SF162 residues but arc numbered relative<br>
to HXB-2.<br>
Figures 4A-M depict alignment of nucleotide sequences of polynucieotides encoding<br>
modified Env polypeptides having deletions or replacements in the small loop. The<br>
unmodified amino acid residues encoded by these sequences correspond to wildtype SF162<br>
residues but are numbered relative to HXB-2.<br>
Figures 5A-N depict alignment of nucleotide sequences of polynucieotides encoding<br>
modified Env polypeptides having both V1/V2 deletions and, in addition, deletions or<br>
replacements in the small loop. The unmodified amino acid residues encoded by these<br>
sequences correspond to wildtype SF162 residues but are numbered relative to HXB-2.<br>
Figure 6 depicts the nucleotide sequence of the construct designated Vall20-Ala204<br>
(SEQ ID NO:3).<br>
Figure 7 depicts the nucleotide sequence of the construct designated Val120-Ile201<br>
(SEQ ID NO:4).<br>
Figure 8 depicts the nucleotide sequence of the construct designated Vall 20-Ile201B<br>
(SEQ ID NO:5).<br>
Figure 9 depicts the nucleotide sequence of the construct designated Lys121 -Val200<br>
(SEQ ID NO:6).<br>
Figure 10 depicts the nucleotide sequence of the construct designated Leu122-Ser199<br>
(SEQ ID NO:7).<br>
Figure 11 depicts the nucleotide sequence of the construct designated Val120-Thr202<br>
(SEQ ID NO:8).<br>
Figure 12 depicts the nucleotide sequence of the construct designated Trp427-Gly431<br>
(SEQ ID NO:9).<br>
Figure 13 depicts the nucleotide sequence of the construct designated Arg426-Gly431<br>
(SEQ ID NO:10).<br>
Figure 14 depicts the nucleotide sequence of the construct designated Arg426-<br>
Giy431B(SEQ ID NO:11).<br>
Figure 15 depicts the nucleotide sequence of the construct designated Arg426-Lys432<br>
(SEQ ID NO:12).<br>
Figure 16 depicts the nucleotide sequence of the construct designated Asn425-Lys432<br>
(SEQ ID NO.13).<br>
Figure 17 depicts the nucleotide sequence of the construct designated IIe424-Ala433<br>
(SEQ ID NO:14).<br>
Figure 18 depicts the nucleotide sequence of the construct designated Ile423-Met434<br>
(SEQ ID NO:15).<br>
Figure 19 depicts the nucleotide sequence of the construct designated Gln422-Tyr435<br>
(SEQ ID NO:16).<br>
Figure 20 depicts the nucleotide sequence of the construct designated Gln422-<br>
Tyr435B(SEQ ID NO:17).<br>
Figure 21 depicts the nucleotide sequence of the construct designated Leu122-<br>
Ser199;Arg426-Gly431 (SEQ IDNO: 18).<br>
Figure 22 depicts the nucleotide sequence of the construct designated Leu122-<br>
Ser199;Arg426-Lys432 (SEQ ID NO:19).<br>
Figure 23 depicts the nucleotide sequence of the construct designated Leu122-Ser199;<br>
Trp427-Gly431 (SEQ ID NO:20).<br>
Figure 24 depicts the nucleotide sequence of the construct designated Lys121-Va1200;<br>
Asn425-Lys432 (SEQ ID NO:21).<br>
Figure 25 depicts the nucleotide sequence of the construct designated Val 120-Ile201;<br>
Ile424-Ala433 (SEQ ID NO:22).<br>
Figure 26 depicts the nucleotide sequence of the construct designated Val120-<br>
Ile201B; Ile424-Ala433 (SEQ ID NO:23).<br>
Figure 27 depicts the nucleotide sequence of the construct designated Val120-Thr202;<br>
Ile424-Ala433 (SEQ ID NO:24).<br>
Figure 28 depicts the nucleotide sequence of the construct designated Val127-Asn 195<br>
(SEQ ID NO:25).<br>
Figure 29 depicts the nucleotide sequence of the construct designated Val127-<br>
Asn195; Arg426-Gly431 (SEQ ID NO:26).<br>
Detailed Description of the Invention<br>
The practice of the present invention will employ, unless otherwise indicated,<br>
conventional methods of protein chemistry, viral immunobiology, molecular biology and<br>
recombinant DNA techniques within the skill of the art. Such techniques are explained fully<br>
in the literature. See, e.g., T.E. Creighton, Proteins: Structures and Molecular Properties<br>
(W.H. Freeman and Company, 1993); Nelson L.M. and Jerome H.K. HIV Protocols in<br>
Methods in Molecular Medicine, vol. 17, 1999; Sambrook, et al., Molecular Cloning: A<br>
Laboratory Manual (Cold Spring Harbor Laboratory, 1989); F.M. Ausubel et al. Current<br>
Protocols in Molecular Biology. Greene Publishing Associates &amp; Wiley Interscience New<br>
York; and Lipkowitz and Bovd. Reviews in Computational Chemistry, volumes 1-present<br>
(Wiley-VCH, New York, New York, 1999).<br>
It must be noted that, as used in this specification and the appended claims, the<br>
singular forms "a", "an" and "the" include plural referents unless the content clearly dictates<br>
otherwise. Thus, for example, reference to "a polypeptide" includes a mixture of two or more<br>
polypeptides, and the like.<br>
Definitions<br>
In describing the present invention, the following terms will be employed, and are<br>
intended to be defined as indicated below.<br>
The terms "polypeptide," and "protein" are used interchangeably herein to denote any<br>
polymer of amino acid residues. The terms encompass peptides, oligopeptides, dimers,<br>
multimers, and the like. Such polypeptides can be derived from natural sources or can be<br>
synthesized or recombinantly produced. The terms also include postexpression modifications<br>
of the polypeptide, for example, glycosylation, acetylation, phosphorylation, etc.<br>
A polypeptide as defined herein is generally made up of the 20 natural amino acids<br>
Ala (A), Arg (R), Asn (N), Asp (D), Cys (C), Gln (Q), Glu (E), Gly (G), His (H), Ile (I), Leu<br>
(L), Lys (K), Met (M), Phe (F), Pro (P), Ser (S), Thr (T), Trp (W), Tyr (Y) and Val (V) and<br>
may also include any of the several known amino acid analogs, both naturally occurring and<br>
synthesized analogs, such as but not limited to homoisoleucine, asaleucine, 2-<br>
(methylenecyclopropyl)glycine, Snmethylcysteine, S-(prop-1-enyl)cysteine, homoserine,<br>
ornithine, norleucine, norvaline, homoarginine, 3-(3-carboxyphenyl)alanine,<br>
cyclohexylalanine, mimosine, pipecolic acid, 4-methylglutamic acid, canavanine, 2,3-<br>
diaminopropionic acid, and the like. Further examples of polypeptide agents which will find<br>
use in the present invention are set forth below.<br>
By "geometry" or "tertiary structure" of a polypeptide or protein is meant the overall<br>
3-D configuration of the protein. As described herein, the geometry can be determined, for<br>
example, by crystallography studies or by using various programs or algorithms which<br>
predict the geometry based on interactions between the amino acids making up the primary<br>
and secondary structures.<br>
By "wild type" polypeptide, polypeptide agent or polypeptide drug, is meant a<br>
naturally occurring polypeptide sequence, and its corresponding secondary structure. An<br>
"isolated" of "purified" protein or polypeptide is a protein which is separate and discrete from<br>
a whole organism with which the protein is normally associated in nature. It is apparent that<br>
the term denotes proteins of various levels of purity. Typically, a composition containing a<br>
purified protein will be one in which at least about 35%, preferably at least about 40-50%,<br>
more preferably, at least about 75-85%, and most preferably at least about 90% or more, of<br>
the total protein in the composition will be the protein in question.<br>
By "Env polypeptide" is meant a molecule derived from an envelope protein,<br>
preferably from HIV Env. The envelope protein of HIV-1 is a glycoprotein of about 160 kd<br>
(gp160). During virus infection of the host cell, gp160 is cleaved by host cell proteases to<br>
form gp120 and the integral membrane protein, gp41. The gp41 portion is anchored in (and<br>
spans) the membrane bilayer of virion, while the gp120 segment protrudes into the<br>
surrounding environment. As there is no covalent attachment between gp120 and gp41, free<br>
gp120 is released from the surface of virions and infected cells. Env polypeptides may also<br>
include gp140 polypeptides. Env polypeptides can exist as monomers, dimers or multimers.<br>
By a "gp120 polypeptide" is meant a molecule derived from a gp120 region of the<br>
Env polypeptide. Preferably, the gp120 polypeptide is derived from HIV Env. The primary<br>
amino acid sequence of gp120 is approximately 511 amino acids, with a polypeptide core of<br>
about 60,000 daltons. The polypeptide is extensively modified by N-linked glycosylation to<br>
increase the apparent molecular weight of the molecule to 120,000 daltons. The amino acid<br>
sequence of gp120 contains five relatively conserved domains interspersed with five<br>
hypervariable domains. The positions of the 18 cysteine residues in the gp120 primary<br>
sequence of the HIV-1HXB-2 (hereinafter "HXB-2") strain, and the positions of 13 of the<br>
approximately 24 N-linked glycosylation sites in the gp120 sequence are common to most, if<br>
not all, gp120 sequences. The hypervariable domains contain extensive amino acid<br>
substitutions, insertions and deletions. Despite this variation, most, if not all, gp120<br>
sequences preserve the virus's ability to bind to the viral receptor CD4. A "gp120<br>
polypeptide" includes both single subunits or multimers.<br>
Env polypeptides (e.g.. gpl20, gpl40 and gpl60) include a "bridging sheet"<br>
comprised of 4 anti-parallel ß-strands (ß-2, ß-3, ß-20 and ß-21) that form a p-sheet.<br>
Extruding from one pair of the ß-strands (ß-2 and ß-3) are two loops, V1 and V2. The ß-2<br>
sheet occurs at approximately amino acid residue 119 (Cys) to atnino acid residue 123 (Thr)<br>
while ß-3 occurs at approximately amino acid residue 199 (Ser) to amino acid residue 201<br>
(Ile), relative to HXB-2. The "V1/V2 region" occurs at approximately amino acid positions<br>
126 (Cys) to residue 196 (Cys), relative to HXB-2. (see, e.g., Wyatt et al. (1995) J. Virol.<br>
69:5723-5733; Stamatatos et al. (1998)7. Virol. 72:7840-7845). Extruding from the second<br>
pair of ß-strands (ß-20 and ß-21) is a "small-loop" structure, also referred to herein as "the<br>
bridging sheet small loop." In HXB-2, P-20 extends from about amino acid residue 422<br>
(Gln) to amino acid residue 426 (Met) while ß-21 extends from about amino acid residue 430<br>
(Val) to amino acid residue 435 (Tyr). In variant SF162, the Met-426 is an Arg (R) residue.<br>
The "small loop" extends from about amino acid residue 427 (Trp) through 429 (Lys),<br>
relative to HXB-2. A representative diagram of gp120 showing the bridging sheet, the small<br>
loop, and V1/V2 is shown in Figure 1. In addition, alignment of the amino acid sequences of<br>
Env polypeptide gp160 of selected variants is shown, relative to HXB-2, in Figures 2A-C.<br>
Furthermore, an "Env polypeptide" or "gp120 polypeptide" as defined herein is not<br>
limited to a polypeptide having the exact sequence described herein. Indeed, the HIV<br>
genome is in a state of constant flux and contains several variable domains which exhibit<br>
relatively high degrees of variability between isolates. It is readily apparent that the terms<br>
encompass Env (e.g., gpl20) polypeptides from any of the identified HIV isolates, as well as<br>
newly identified isolates, and subtypes of these isolates. Descriptions of structural features<br>
are given herein with reference to HXB-2. One of ordinary skill in the art in view of the<br>
teachings of the present disclosure and the art can determine corresponding regions in other<br>
HTV variants (e.g.. isolates HIVIllb, HIVSF2, HIV-1SF162, HIV-1SF170, HIVLAV, HIVLAI, HIVMN,<br>
HIV-1cm235', HIV-1US4, other HIV-1 strains from diverse subtypes(e.g., subtypes, A through<br>
G, and O), HIV-2 strains and diverse subtypes (e.g., HIV-2UC1 and HIV-2UC2), and simian<br>
immunodeficiency virus (SIV). (See, e.g., Virology, 3rd Edition (W.K. Joklik ed. 1988);<br>
Fundamental Virology, 2nd Edition (B.N. Fields and D.M. Knipe, eds. 1991); Virology, 3rd<br>
Edition (Fields, BN, DM Knipe, PM Howley, Editors, 1996, Lippincott-Raven, Philadelphia,<br>
PA; for a description of these and other related viruses), using for example, sequence<br>
comparison programs (e.g., BLAST and others described herein) or identification and<br>
alignment of structural features (e.g., a program such as the "ALB" program described herein<br>
that can identify ß-sheet regions). The actual amino acid sequences of the modified Env<br>
polypeptides can be based on any HIV variant.<br>
Additionally, the term "Env polypcptide" (e.g.. "gp120 polypeptide") encompasses<br>
proteins which include additional modifications to the native sequence, such as additional<br>
internal deletions, additions and substitutions. These modifications may be deliberate, as<br>
through site-directed mutagenesis, or may be accidental, such as through naturally occurring<br>
mutational events. Thus, for example, if the Env polypeptide is to be used in vaccine<br>
compositions, the modifications must be such that immunological activity (i.e., the ability to<br>
elicit an antibody response to the polypeptide) is not lost. Similarly, if the polypeptides are to<br>
be used for diagnostic purposes, such capability must be retained.<br>
Thus, a "modified Env polypeptide" is an Env polypeptide (e.g., gp120 as defined<br>
above), which has been manipulated to delete or replace all or a part of the bridging sheet<br>
portion and, optionally, the variable regions V1 and V2. Generally, modified Env (e.g.,<br>
gpl20) polypeptides have enough of the bridging sheet removed to expose the CD4 binding<br>
site, but leave enough of the structure to allow correct folding (e.g., correct geometry). Thus,<br>
modifications to the ß-20 and ß-21 regions (between about amino acid residues 420 and 435<br>
relative to HXB-2) are preferred. Additionally, modifications to the ß-2 and ß-3 regions<br>
(between about amino acid residues 119 (Cys) and 201 (Ile)) and modifications (e.g.,<br>
truncations) to the V1 and V2 loop regions may also be made. Although not all possible ß-<br>
sheet and V1 /V2 modifications have been exemplified herein, it is to be understood that other<br>
disrupting modifications are also encompassed by the present invention.<br>
Normally, such a modified polypeptide is capable of secretion into growth medium in<br>
which an organism expressing the protein is cultured. However, for purposes of the present<br>
invention, such polypeptides may also be recovered intracellularly. Secretion into growth<br>
media is readily determined using a number of detection techniques, including, e.g.,<br>
polyacrylamide gel electrophoresis and the like, and immunological techniques such as<br>
Western blotting and immunoprecipitation assays as described in, e.g., International<br>
Publication No. WO 96/04301, published February 15, 1996.<br>
A gp120 or other Env polypeptide is produced "intracellularly" when it is found<br>
within the cell, either associated with components of the cell, such as in association with the<br>
endoplasmic reticulum (ER) or the Golgi Apparatus, or when it is present in the soluble<br>
cellular fraction. The gp120 and other Env polypeptides of the present invention may also be<br>
secreted into growth medium so long as sufficient amounts of the polypeptides remain<br>
present within the cell such that they can be purified from cell lysates using techniques<br>
described herein.<br>
An "immunogenic" gp120 or other Env protein is a molecule that includes at least one<br>
epitope such that the molecule is capable of either eliciting an immunological reaction in an<br>
individual to which the protein is administered or, in the diagnostic context, is capable of<br>
reacting with antibodies directed against the HIV in question.<br>
By "epitope" is meant a site on an antigen to which specific B cells and/or T cells<br>
respond, rendering the molecule including such an epitope capable of eliciting an<br>
immunological reaction or capable of reacting with HIV antibodies present in a biological<br>
sample. The term is also used interchangeably with "antigenic determinant" or "antigenic<br>
determinant site." An epitope can comprise 3 or more amino acids in a spatial conformation<br>
unique to the epitope. Generally, an epitope consists of at least 5 such amino acids and, more<br>
usually, consists of at least 8-10 such amino acids. Methods of determining spatial<br>
conformation of amino acids are known in the art and include, for example, x-ray<br>
crystallography and 2-dimensional nuclear magnetic resonance. Furthermore, the<br>
identification of epitopes in a given protein is readily accomplished using techniques well<br>
known in the art, such as by the use of hydrophobicity studies and by site-directed serology.<br>
See, also, Geysen et al., Proc. Natl Acad. Sci. USA (1984) 81:3998-4002 (general method of<br>
rapidly synthesizing peptides to determine the location of immunogenic epitopes in a given<br>
antigen); U.S. Patent No. 4,708,871 (procedures for identifying and chemically synthesizing<br>
epitopes of antigens); and Geysen et al., Molecular Immunology (1986) 21:709-715<br>
(technique for identifying peptides with high affinity for a given antibody). Antibodies that<br>
recognize the same epitope can be identified in a simple immunoassay showing the ability of<br>
one antibody to block the binding of another antibody to a target antigen.<br>
An "immunological response" or "immune response" as used herein is the<br>
development in the subject of a humoral and/or a cellular immune response to the Env (e.g.,<br>
gp120) polypeptide when the polypeptide is present in a vaccine composition. These<br>
antibodies may also neutralize infectivity, and/or mediate antibody-complement or antibody<br>
dependent cell cytotoxicity to provide protection to an immunized host. Immunological<br>
reactivity may be determined in standard immunoassays, such as a competition assays, well<br>
known in the art.<br>
Techniques for determining amino acid sequence "similarity" are well known in the<br>
art. In general, "similarity" means me exact amino acid to amino acid comparison of two or<br>
more poiypeptides at the appropriate place, where amino acids are identical or possess similar<br>
chemical and/or physical properties such as charge or hydrophobicity. A so-termed "percent<br>
similarity" then can be determined between the compared polypeptide sequences.<br>
Techniques for determining nucleic acid and amino acid sequence identity also arc well<br>
known in the art and include determining the nucleotide sequence of the mRNA for that gene<br>
(usually via a cDNA intermediate) and determining the amino acid sequence encoded<br>
thereby, and comparing this to a second amino acid sequence. In general, "identity" refers to<br>
an exact nucleotide to nucleotide or amino acid to amino acid correspondence of two<br>
polynucleotides or polypeptide sequences, respectively.<br>
Two or more polynucleotide sequences can be compared by determining their<br>
"percent identity." Two or more amino acid sequences likewise can be compared by<br>
determining their "percent identity." The percent identity of two sequences, whether nucleic<br>
acid or peptide sequences, is generally described as the number of exact matches between two<br>
aligned sequences divided by the length of the shorter sequence and multiplied by 100. An<br>
approximate alignment for nucleic acid sequences is provided by the local homology<br>
algorithm of Smith and Waterman, Advances in Applied Mathematics 2:482-489 (1981).<br>
This algorithm can be extended to use with peptide sequences using the scoring matrix<br>
developed by Dayhoff, Atlas of Protein Sequences and Structure, MO. Dayhoff ed., 5 suppl.<br>
3:353-358, National Biomedical Research Foundation, Washington, DC, USA, and<br>
normalized by Gribskov, Nucl. Acids Res. 14(6):6745-6763 (1986). An implementation of<br>
this algorithm for nucleic acid and peptide sequences is provided by the Genetics Computer<br>
Group (Madison, WI) in their BestFit utility application. The default parameters for this<br>
method are described in the Wisconsin Sequence Analysis Package Program Manual, Version<br>
8 (1995) (available from Genetics Computer Group, Madison, WI). Other equally suitable<br>
programs for calculating the percent identity or similarity between sequences are generally<br>
known in the art.<br>
For example, percent identity of a particular nucleotide sequence to a reference<br>
sequence can be determined using the homology algorithm of Smith and Waterman with a<br>
default scoring table and a gap penalty of six nucleotide positions. Another method of<br>
establishing percent identity in the context of the present invention is to use the MPSRCH<br>
package of programs copyrighted by the University of Edinburgh, developed by John F.<br>
Collins and Shane S. Sturrok, and distributed by IntelliGenetics, Inc. (Mountain View, CA).<br>
From this suite of packages, the Smith-Waterman algorithm can be employed where default<br>
parameters are used for the scoring table (for example, gap open penalty of 12, gap extension<br>
penalty of one, and a gap of six). From the data generated, the "Match" value reflects<br>
"sequence identity." Other suitable programs for calculating the percent identity or similarity<br>
between sequences are generally known in the art, such as the alignment program BLAST,<br>
which can also be used with default parameters. For example, BLASTN and BLASTP can be<br>
used with the following default parameters: genetic code = standard; filter = none; strand =<br>
both; cutoff = 60; expect - 10; Matrix = BLOSUM62; Descriptions = 50 sequences; sort by =<br>
HIGH SCORE; Databases = non-redundant, GenBank + EMBL + DDBJ + PDB + GenBank<br>
CDS translations + Swiss protein + Spupdate + PHL Details of these programs can be found<br>
at the following internet address: http://www.ncbi.nhn.gov/cgi-bin/BLAST.<br>
One of skill in the art can readily determine the proper search parameters to use for a<br>
given sequence in the above programs. For example, the search parameters may vary based<br>
on the size of the sequence in question. Thus, for example, a representative embodiment of<br>
the present invention would include an isolated polynucleotide having X contiguous<br>
nucleotides, wherein (i) the X contiguous nucleotides have at least about 50% identity to Y<br>
contiguous nucleotides derived from any of the sequences described herein, (ii) X equals Y,<br>
and (iii) X is greater than or equal to 6 nucleotides and up to 5000 nucleotides, preferably<br>
greater than or equal to 8 nucleotides and up to 5000 nucleotides, more preferably 10-12<br>
nucleotides and up to 5000 nucleotides, and even more preferably 15-20 nucleotides, up to<br>
the number of nucleotides present in the full-length sequences described herein (e.g., see the<br>
Sequence Listing and claims), including all integer values falling within the above-described<br>
ranges.<br>
The synthetic expression cassettes (and purified polynucleotides) of the present<br>
invention include related polynucleotide sequences having about 80% to 100%, greater than<br>
80-85%, preferably greater than 90-92%, more preferably greater than 95%, and most<br>
preferably greater than 98% sequence (including all integer values falling within these<br>
described ranges) identity to the synthetic expression cassette sequences disclosed herein (for<br>
example, to the claimed sequences or other sequences of the present invention) when the<br>
sequences of the present invention are used as the query sequence.<br>
Computer programs are also available to determine the likelihood of certain<br>
polypeptides to form structures such as P-sheets. One such program, described herein, is the<br>
"ALB" program for protein and polypeptide secondary structure calculation and predication.<br>
In addition, secondary protein structure can be predicted from the primary amino acid<br>
sequence, for example using protein crystal structure and aligning the protein sequence<br>
related to the crystal structure (e.g., using Molecular Operating Environment (MOE)<br>
programs available from the Chemical Computing Group Inc., Montreal, P.Q., Canada).<br>
Other methods of predicting secondary structures are described, for example, in Garnier et al.<br>
(1996) Methods Enzymol. 266:540-553; Geourjon et al. (1995) Comput. Applic. Biosci.<br>
11:681-684; Levin (1997) Protein Eng. 10:771-776; and Rost et al. (1993) J. Molec. Biol.<br>
232:584-599.<br>
Homology can also be determined by hybridization of polynucleotides under<br>
conditions which form stable duplexes between homologous regions, followed by digestion<br>
with single-stranded-specific nuclease(s), and size determination of the digested fragments.<br>
Two DNA, or two polypeptide sequences are "substantially homologous" to each other when<br>
the sequences exhibit at least about 80%-85%, preferably at least about 90%, and most<br>
preferably at least about 95%-98% sequence identity over a defined length of the molecules,<br>
as determined using the methods above. As used herein, substantially homologous also refers<br>
to sequences showing complete identity to the specified DNA or polypeptide sequence. DNA<br>
sequences that are substantially homologous can be identified in a Southern hybridization<br>
experiment under, for example, stringent conditions, as defined for that particular system.<br>
Defining appropriate hybridization conditions is within the skill of the art. See, e.g.,<br>
Sambrook et al., supra; DNA Cloning, supra; Nucleic Acid Hybridization, supra.<br>
A "coding sequence" or a sequence which "encodes" a selected protein, is a nucleic<br>
acid sequence which is transcribed (in the case of DNA) and translated (in the case of<br>
mRNA) into a polypeptide in vitro or in vivo when placed under the control of appropriate<br>
regulatory sequences. The boundaries of the coding sequence are determined by a start codon<br>
at the 5' (amino) terminus and a translation stop codon at the 3' (carboxy) terminus. A coding<br>
sequence can include, but is not limited to cDNA from viral nucleotide sequences as well as<br>
synthetic and semisynthetic DNA sequences and sequences including base analogs. A<br>
transcription termination sequence may be located 3' to the coding sequence.<br>
"Control elements" refers collectively to promoter sequences, ribosome binding sites,<br>
polyadenylation signals, transcription termination sequences, upstream regulatory domains,<br>
enhancers, and the like, which collectively provide for the transcription and translation of a<br>
coding sequence in a host cell. Not all of these control elements need always be present so<br>
long as the desired gene is capable of being transcribed and translated.<br>
A control element "directs the transcription" of a coding sequence in a cell when RNA<br>
polymerase will bind the promoter sequence and transcribe the coding sequence into mRNA,<br>
which is then translated into the polypeptide encoded by the coding sequence.<br>
"Operably linked" refers to an arrangement of elements wherein the components so<br>
described are configured so as to perform their usual function. Thus, control elements<br>
operably linked to a coding sequence are capable of effecting the expression of the coding<br>
sequence when RNA polymerase is present. The control elements need not be contiguous<br>
with the coding sequence, so long as they function to direct the expression thereof. Thus, for<br>
example, intervening untranslated yet transcribed sequences can be present between, e.g., a<br>
promoter sequence and the coding sequence and the promoter sequence can still be<br>
considered "operably linked" to the coding sequence.<br>
"Recombinant" as used herein to describe a nucleic acid molecule means a<br>
polynucleotide of genomic, cDNA, semisynthetic, or synthetic origin which, by virtue of its<br>
origin or manipulation: (1) is not associated with all or a portion of the polynucleotide with<br>
which it is associated in nature; and/or (2) is linked to a polynucleotide other than that to<br>
which it is linked in nature. The term "recombinant" as used with respect to a protein or<br>
polypeptide means a polypeptide produced by expression of a recombinant polynucleotide.<br>
"Recombinant host cells," "host cells," "cells," "cell lines," "cell cultures," and other such<br>
terms denoting procaryotic microorganisms or eucaryotic cell lines cultured as unicellular<br>
entities, are used interchangeably, and refer to cells which can be, or have been, used as<br>
recipients for recombinant vectors or other transfer DNA, and include the progeny of the<br>
original cell which has been transfected. It is understood that the progeny of a single parental<br>
cell may not necessarily be completely identical in morphology or in genomic or total DNA<br>
complement to the original parent, due to accidental or deliberate mutation. Progeny of the<br>
parental cell which are sufficiently similar to the parent to be characterized by the relevant<br>
property, such as the presence of a nucleotide sequence encoding a desired peptidc, are<br>
included in the progeny intended by this definition, and are covered by the above terms.<br>
By "vertebrate subject" is meant any member of the subphylum chordata, including,<br>
without limitation, humans and other primates, including non-human primates such as<br>
chimpanzees and other apes and monkey species; farm animals such as cattle, sheep, pigs,<br>
goats and horses; domestic mammals such as dogs and cats; laboratory animals including<br>
rodents such as mice, rats and guinea pigs; birds, including domestic, wild and game birds<br>
such as chickens, turkeys and other gallinaceous birds, ducks, geese, and the like. The term<br>
does not denote a particular age. Thus, both adult and newborn individuals are intended to be<br>
covered.<br>
As used herein, a "biological sample" refers to a sample of tissue or fluid isolated<br>
from an individual, including but not limited to, for example, blood, plasma, serum, fecal<br>
matter, urine, bone marrow, bile, spinal fluid, lymph fluid, samples of the skin, external<br>
secretions of the skin, respiratory, intestinal, and genitourinary tracts, samples derived from<br>
the gastric epithelium and gastric mucosa, tears, saliva, milk, blood cells, organs, biopsies<br>
and also samples of in vitro cell culture constituents including but not limited to conditioned<br>
media resulting from the growth of cells and tissues in culture medium, e.g., recombinant<br>
cells, and cell components.<br>
The terms "label" and "detectable label" refer to a molecule capable of detection,<br>
including, but not limited to, radioactive isotopes, fluorescers, chemiluminescers, enzymes,<br>
enzyme substrates, enzyme cofactors, enzyme inhibitors, chromophores, dyes, metal ions,<br>
metal sols, ligands (e.g., biotin or haptens) and the like. The term "fluorescer" refers to a<br>
substance or a portion thereof which is capable of exhibiting fluorescence in the detectable<br>
range. Particular examples of labels which may be used with the invention include, but are<br>
not limited to fluorescein, rhodamine, dansyl, umbelliferone, Texas red, luminol, acradimum<br>
esters, NADPH, a-ß-galactosidase, horseradish peroxidase, glucose oxidase, alkaline<br>
phosphatase and urease.<br>
Overview<br>
The present invention concerns modified Env polypeptide molecules (e.g.,<br>
glycoprotein ("gp") 120). Without being bound by a particular theory, it appears that it has<br>
been difficult to generate immunological responses against Env because the CD4 binding site<br>
is buried between the outer domain, the inner domain and the V1 /V2 domains. Thus,<br>
although deletion of the V1 /V2 domain may render the virus more susceptible to<br>
neutralization by monoclonal antibody directed to the CD4 site, the bridging sheet covering<br>
most of the CD4 binding domain may prevent an antibody response. Thus, the present<br>
invention provides Env polypeptides that maintain their general overall structure yet expose<br>
the CD4 binding domain. This allows the generation of an immune response (e.g., an<br>
antibody response) to epitopes in or near the CD4 binding site.<br>
Various forms of the different embodiments of the invention, described herein, may<br>
be combined.<br>
ß-Sheet Conformations<br>
In the present invention, location of the ß-sheet structures were identified relative to<br>
3-D (crystal) structure of an HXB-2 crystallized Env protein (see, Example 1 A). Based on<br>
this structure, constructs encoding polypeptides having replacements and or excisions which<br>
maintain overall geometry while exposing the CD4 binding site were designed. In particular,<br>
the crystal structure of HXB-2 was downloaded from the Brookhaven Database. Using the<br>
default parameters of the Loop Search feature of the Biopolymer module of the Sybyl<br>
molecular modeling package, homology and fit of amino acids which could replace the native<br>
loops between ß-strands yet maintain overall tertiary structure were determined. Constructs<br>
encoding the modified Env polypeptides were then designed (Example 1.B.).<br>
Thus, the modified Env polypeptides typically have enough of the bridging sheet<br>
removed to expose the CD4 groove, but have enough of the structure to allow correct folding<br>
of the Env glycoprotein. Exemplary constructs are described below.<br>
Polypeptide Production<br>
The polypeptides of the present invention can be produced in any number of ways<br>
which are well known in the art.<br>
In one embodiment, the polypeptides are generated using recombinant techniques,<br>
well "known in the art. In this regard, oligonucleotide probes can be devised based on the<br>
known sequences of the Env (e.g., gp120) polypeptide genome and used to probe genomic or<br>
cDNA libraries for Env genes. The gene can then be further isolated using standard<br>
techniques and, e.g., restriction enzymes employed to truncate the gene at desired portions of<br>
the full-length sequence. Similarly, the Env gene(s) can be isolated directly from cells and<br>
tissues containing the same, using known techniques, such as phenol extraction and the<br>
sequence further manipulated to produce the desired truncations. See. e.g.. Sambrook et al.,<br>
supra, for a description of techniques used to obtain and isolate DNA.<br>
The genes encoding the modified (e.g., truncated and/or substituted) polypeptides can<br>
be produced synthetically, based on the known sequences. The nucleotide sequence can be<br>
designed with the appropriate codons for the particular amino acid sequence desired. The<br>
complete sequence is generally assembled from overlapping oligonucleotides prepared by<br>
standard methods and assembled into a complete coding sequence. See, e.g., Edge (1981)<br>
Nature 292:756; Nambair et al. (1984) Science 223:1299; Jay et al. (1984) J. Biol. Chem.<br>
259:6311; Stemmer et al. (1995) Gene 164:49-53.<br>
Recombinant techniques are readily used to clone a gene encoding an Env<br>
polypeptide gene which can then be mutagenized in vitro by the replacement of the<br>
appropriate base pair(s) to result in the codon for the desired amino acid. Such a change can<br>
include as little as one base pair, effecting a change in a single amino acid, or can encompass<br>
several base pair changes. Alternatively, the mutations can be effected using a mismatched<br>
primer which hybridizes to the parent nucleotide sequence (generally cDNA corresponding to<br>
the RNA sequence), at a temperature below the melting temperature of the mismatched<br>
duplex. The primer can be made specific by keeping primer length and base composition<br>
within relatively narrow limits and by keeping the mutant base centrally located. See, e.g.,<br>
Innis et al, (1990) PCR Applications: Protocols for Functional Genomics; Zoller and Smith,<br>
Methods Enzymol (1983) 100:468. Primer extension is effected using DNA polymerase, the<br>
product cloned and clones containing the mutated DNA, derived by segregation of the primer<br>
extended strand, selected. Selection can be accomplished using the mutant primer as a<br>
hybridization probe. The technique is also applicable for generating multiple point<br>
mutations. See, e.g., Dalbie-McFarland et al. Proc. Natl. Acad. Sci USA (1982) 79:6409.<br>
Once coding sequences for the desired proteins have been isolated or synthesized,<br>
they can be cloned into any suitable vector or replicon for expression. As will be apparent<br>
from the teachings herein, a wide variety of vectors encoding modified polypeptides can be<br>
generated by creating expression constructs which operably link, in various combinations,<br>
polynucleotides encoding Env polypeptides having deletions or mutation therein. Thus,<br>
polynucleotides encoding a particular deleted V1/V2 region can be operably linked with<br>
polynucleotides encoding polypeptides having deletions or replacements in the small loop<br>
region and the construct introduced into a host cell for polypeptide expression. Non-lirniting<br>
examples of such combinations are discussed in the Examples.<br>
Numerous cloning vectors are known to those of skill in the art, and the selection of<br>
an appropriate cloning vector is a matter of choice. Examples of recombinant DNA vectors<br>
for cloning and host cells which they can transform include the bacteriophage X (E. coli),<br>
pBR322 (E. coli), pACYC177 (E. coli), pKT230 (gram-negative bacteria), pGV 1106<br>
(gram-negative bacteria), pLAFR1 (gram-negative bacteria), pME290 (non-E. coli<br>
gram-negative bacteria), pHV14 (E. coli and Bacillus subtilis), pBD9 (Bacillus), pIJ61<br>
(Streptomyces), pUC6 (Streptomyces), YIp5 (Saccharomyces), YCpl9 (Saccharomyces) and<br>
bovine papilloma virus (mammalian cells). See, generally, DNACloning: Vols. I &amp; II, supra;<br>
Sambrook et al., supra; B. Perbal, supra.<br>
Insect cell expression systems, such as baculovirus systems, can also be used and are<br>
known to those of skill in the art and described in, e.g., Summers and Smith, Texas<br>
Agricultural Experiment Station Bulletin No. 1555 (1987). Materials and methods for<br>
baculovirus/insect cell expression systems are commercially available in kit form from, inter<br>
alia, Invitrogen, San Diego CA ("MaxBac" kit).<br>
Plant expression systems can also be used to produce the modified Env proteins.<br>
Generally, such systems use virus-based vectors to transfect plant cells with heterologous<br>
genes. For a description of such systems see, e.g., Porta et al., Mol. Biotech. (1996) 5:209-<br>
221; and Hackland et al., Arch. Virol. (1994) 139:1,22.<br>
Viral systems, such as a vaccinia based infection/transfection system, as described in<br>
Tomei et al., J. Virol. (1993) 67:4017-4026 and Selby et al., J. Gen. Virol. (1993)<br>
74:1103-1113, will also find use with the present invention. In this system, cells are first<br>
transfected in vitro with a vaccinia virus recombinant that encodes the bacteriophage T7 RNA<br>
polymerase. This polymerase displays exquisite specificity in that it only transcribes<br>
templates bearing T7 promoters. Following infection, cells are transfected with the DNA of<br>
interest, driven by a T7 promoter. The polymerase expressed in the cytoplasm from the<br>
vaccinia virus recombinant transcribes the transfected DNA into RNA which is then<br>
translated into protein by the host translational machinery. The method provides for high<br>
level, transient, cytoplasmic production of large quantities of RNA and its translation<br>
product(s).<br>
The gene can be placed under the control of a promoter, ribosome binding site (for<br>
bacterial expression) and, optionally, an operator (collectively referred to herein as "control"<br>
elements), so that the DNA sequence encoding the desired Env polypeptide is transcribed into<br>
RNA in the host cell transformed by a vector containing this expression construction. The<br>
coding sequence may or may not contain a signal peptide or leader sequence. With the<br>
present invention, both the naturally occurring signal peptides or heterologous sequences can<br>
be used. Leader sequences can be removed by the host in post-translational processing. See,<br>
e.g., U.S. Patent Nos. 4,431,739; 4,425,437; 4338,397. Such sequences include, but are not<br>
limited to, the TPA leader, as well as the honey bee meilitin signal sequence.<br>
Other regulatory sequences may also be desirable which allow for regulation of<br>
expression of the protein sequences relative to the growth of the host cell. Such regulatory<br>
sequences are known to those of skill in the art, and examples include those which cause the<br>
expression of a gene to be turned on or off in response to a chemical or physical stimulus,<br>
including the presence of a regulatory compound. Other types of regulatory elements may<br>
also be present in the vector, for example, enhancer sequences.<br>
The control sequences and other regulatory sequences may be ligated to the coding<br>
sequence prior to insertion into a vector. Alternatively, the coding sequence can be cloned<br>
directly into an expression vector which already contains the control sequences and an<br>
appropriate restriction site.<br>
In some cases it may be necessary to modify the coding sequence so that it may be<br>
attached to the control sequences with the appropriate orientation; i.e., to maintain the proper<br>
reading frame. Mutants or analogs may be prepared by the deletion of a portion of the<br>
sequence encoding the protein, by insertion of a sequence, and/or by substitution of one or<br>
more nucleotides within the sequence. Techniques for modifying nucleotide sequences, such<br>
as site-directed mutagenesis, are well known to those skilled in the art. See, e.g., Sambrook<br>
et ai, supra; DNA Cloning, Vols. I and II, supra; Nucleic Acid Hybridization, supra.<br>
The expression vector is then used to transform an appropriate host cell. A number of<br>
mammalian cell lines are known in the art and include immortalized cell lines available from<br>
the American Type Culture Collection (ATCC), such as, but not limited to, Chinese hamster<br>
ovary (CHO) cells, HeLa cells, baby hamster kidney (BHK.) cells, monkey kidney cells<br>
(COS), human hepatocellular carcinoma cells (e.g.. Hep G2), Vero293 cells, as well as others.<br>
Similarly, bacterial hosts such as E. coli. Bacillus subtilis, and Streptococcus spp., will find<br>
use with the present expression constructs. Yeast hosts useful in the present invention<br>
include inter alia, Saccharomyces cerevisiae, Candida albicans, Candida maltosa,<br>
Hansenula polymorpha, Kluyveromyces fragilis, Kluyveromyces lactis, Pichia guillerimondii,<br>
Pichia pastoris, Schizosaccharomyces pombe and Yarrowia lipolytica. Insect cells for use<br>
with baculovirus expression vectors include, inter alia, Aedes aegypti, Autographa<br>
californica, Bombyx mori, Drosophila melanogaster, Spodoptera frugiperda, and<br>
Trichoplusia ni.<br>
Depending on the expression system and host selected, the proteins of the present<br>
invention are produced by growing host cells transformed by an expression vector described<br>
above under conditions whereby the protein of interest is expressed. The selection of the<br>
appropriate growth conditions is within the skill of the art.<br>
In one embodiment, the transformed cells secrete the polypeptide product into the<br>
surrounding media. Certain regulatory sequences can be included in the vector to enhance<br>
secretion of the protein product, for example using a tissue plasminogen activator (TPA)<br>
leader sequence, a ?-interferon signal sequence or other signal peptide sequences from known<br>
secretory proteins. The secreted polypeptide product can then be isolated by various<br>
techniques described herein, for example, using standard purification techniques such as but<br>
not limited to, hydroxyapatite resins, column chromatography, ion-exchange<br>
chromatography, size-exclusion chromatography, electrophoresis, HPLC, immunoadsorbent<br>
techniques, affinity chromatography, immunoprecipitation, and the like.<br>
Alternatively, the transformed cells are disrupted, using chemical, physical or<br>
mechanical means, which lyse the cells yet keep the Env polypeptides substantially intact.<br>
Intracellular proteins can also be obtained by removing components from the cell wall or<br>
membrane, e.g., by the use of detergents or organic solvents, such that leakage of the Env<br>
polypeptides occurs. Such methods are known to those of skill in the art and are described in,<br>
e.g., Protein Purification Applications: A Practical Approach, (E.L.V. Harris and S. Angal,<br>
Eds., 1990)<br>
For example, methods of disrupting cells for use with the present invention include<br>
but are not limited to: sonication or ultrasonication; agitation; liquid or solid extrusion; heat<br>
treatment; freeze-thaw; desiccation; explosive decompression; osmotic shock; treatment with<br>
lytic enzymes including proteases such as trypsin, neuraminidase and lysozyme; alkali<br>
treatment; and the use of detergents and solvents such as bile salts, sodium dodecylsulphate,<br>
Triton, NP40 and CHAPS. The particular technique used to disrupt the cells is largely a<br>
matter of choice and will depend on the cell type in which the polypeptide is expressed,<br>
culture conditions and any pre-treatment used.<br>
Following disruption of the cells, cellular debris is removed, generally by<br>
centrifugation, and the intracellularly produced Env polypeptides are further purified, using<br>
standard purification techniques such as but not limited to, column chromatography, ion-<br>
exchange chromatography, size-exclusion chromatography, electrophoresis, HPLC,<br>
immunoadsorbent techniques, affinity chromatography, immunoprecipitation, and the like.<br>
For example, one method for obtaining the intracellular Env polypeptides of the<br>
present invention involves affinity purification, such as by immunoaffinity chromatography<br>
using anti-Env specific antibodies, or by lectin affinity chromatography. Particularly<br>
preferred lectin resins are those that recognize mannose moieties such as but not limited to<br>
resins derived from Galanthus nivalis agglutinin (GNA), Lens culinaris agglutinin (LCA or<br>
lentil lectin), Pisum sativum agglutinin (PSA or pea lectin), Narcissus pseudonarcissus<br>
agglutinin (NPA) and Allium ursinum agglutinin (AUA). The choice-of a suitable affinity<br>
resin is within the skill in.the art. After affinity purification, the Env polypeptides can be<br>
further purified using conventional techniques well known in the art, such as by any of the<br>
techniques described above.<br>
It may be desirable to produce Env (eg., gpl20) complexes, either with itself or other<br>
proteins. Such complexes are readily produced by e.g., co-transfecting host cells with<br>
constructs encoding for the Env (e.g., gp120) and/or other polypeptides of the desired<br>
complex. Co-transfection can be accomplished either in trans or cis, i.e., by using separate<br>
vectors or by using a single vector which bears both of the Env and other gene. If done using<br>
a single vector, both genes can be driven by a single set of control elements or, alternatively,<br>
the genes can be present on the vector in individual expression cassettes, driven by individual<br>
control elements. Following expression, the proteins will spontaneously associate.<br>
Alternatively, the complexes can be formed by mixing the individual proteins together which<br>
have been produced separately, either in purified or semi-purified form, or even by mixing<br>
culture media in which host cells expressing the proteins, have been cultured. See,<br>
International Publication No. WO 96/04301, published February 15, 1996, for a description<br>
of such complexes.<br>
Relatively small polypeptides, i.e., up to about 50 amino acids in length, can be<br>
conveniently synthesized chemically, for example by any of several techniques that are<br>
known to those skilled in the peptide art. In general, these methods employ the sequential<br>
addition of one or more amino acids to a growing peptide chain. Normally, either the amino<br>
or carboxyl group of the first amino acid is protected by a suitable protecting group. The<br>
protected or derivatized amino acid can then be either attached to an inert solid support or<br>
utilized in solution by adding the next amino acid in the sequence having the complementary<br>
(amino or carboxyl) group suitably protected, under conditions that allow for the formation of<br>
an amide linkage. The protecting group is then removed from the newly added amino acid<br>
residue and the next amino acid (suitably protected) is then added, and so forth. After the<br>
desired amino acids have been linked in the proper sequence, any remaining protecting<br>
groups (and any solid support, if solid phase synthesis techniques are used) are removed<br>
sequentially or concurrently, to render the final polypeptide. By simple modification of this<br>
general procedure, it is possible to add more than one amino acid at a time to a growing<br>
chain, for example, by coupling (under conditions which do not racemize chiral centers) a<br>
protected tripeptide with a properly protected dipeptide to form, after deprotection, a<br>
pentapeptide. See, e.g., J. M. Stewart and J. D. Young, Solid Phase Peptide Synthesis<br>
(Pierce Chemical Co., Rockford, IL 1984) and G. Barany and R. B. Merrifield, The Peptides:<br>
Analysis. Synthesis. Biology, editors E. Gross and J. Meienhofer, Vol. 2, (Academic Press,<br>
New York, 1980), pp. 3-254, for solid phase peptide synthesis techniques; and M. Bodansky,<br>
Principles of Peptide Synthesis. (Springer-Verlag, Berlin 1984) and E. Gross and J.<br>
Meienhofer, Eds., The Peotides: Analysis. Synthesis. Biology. Vol. 1, for classical solution<br>
synthesis.<br>
Typical protecting groups include t-butyloxycarbonyl (Boc), 9-<br>
fluorenylmethoxycarbonyl (Fmoc) benzyloxycarbonyl (Cbz); p-toluenesulfonyl (Tx); 2,4-<br>
dinitrophenyl; benzyl (Bzl); biphenylisopropyloxycarboxy-carbonyl, t-<br>
amyloxycarbonyl, isobomyloxycarbonyl, o-bromobenzyloxycarbonyl, cyclohexyl, isopropyl,<br>
acetyl, o-nitrophenylsulfonyl and the like.<br>
Typical solid supports are cross-linked polymeric supports. These can include<br>
divinylbenzene cross-linked-styrene-based polymers, for example, divinylbenzene-<br>
hydroxymethylstyrene copolymers, divinylbenzene-chloromethylstyrene copolymers and<br>
divinylbenzene-benzhydrylaminopolystyrene copolymers.<br>
The polypeptide analogs of the present invention can also be chemically prepared by<br>
other methods such as by the method of simultaneous multiple peptide synthesis. See, e.g.,<br>
Houghten Proc. Natl. Acad. Set. USA (1985)32:5131-5135; U.S. Patent No. 4,631,211.<br>
Diagnostic aad Vaccine Applications<br>
The intracellularly produced Env polypeptides of the present invention, complexes<br>
thereof, or the polynucleotides coding therefor, can be used for a number of diagnostic and<br>
therapeutic purposes. For example, the proteins and polynucleotides or antibodies generated<br>
against the same, can be used in a variety of assays, to determine the presence of reactive<br>
antibodies/and or Env proteins in a biological sample to aid in the diagnosis of HIV infection<br>
or disease status or as measure of response to immunization.<br>
The presence of antibodies reactive with the Env (e.g., gp120) polypeptides and,<br>
conversely, antigens reactive with antibodies generated thereto, can be detected using<br>
standard electrophoretic and immunodiagnostic techniques, including immunoassays such as<br>
competition, direct reaction, or sandwich type assays. Such assays include, but are not<br>
limited to, western blots; agglutination tests; enzyme-labeled and mediated immunoassays,<br>
such as ELISAs; biotin/avidin type assays; radioimmunoassays; immunoelectrophoresis;<br>
immunoprecipitation, etc. The reactions generally include revealing labels such as<br>
fluorescent, chemiluminescent, radioactive, or enzymatic labels or dye molecules, or other<br>
methods for detecting the formation of a complex between the antigen and the antibody or<br>
antibodies reacted therewith.<br>
Solid supports can be used in the assays such as nitrocellulose, in membrane or<br>
microtiter well form; polyvinylchloride, in sheets or microtiter wells; polystyrene latex, in<br>
beads or microtiter plates; polyvinylidine fluoride; diazotized paper; nylon membranes;<br>
activated beads, and the like.<br>
Typically, the solid support is first reacted with the biological sample (or the gp120<br>
proteins), washed and then the antibodies, (or a sample suspected of containing antibodies),<br>
applied. After washing to remove any non-bound ligand, a secondary binder moiety is added<br>
under suitable binding conditions, such that the secondary binder is capable of associating<br>
selectively with the bound ligand. The presence of the secondary binder can then be detected<br>
using techniques well known in the art. Typically, the secondary binder will comprise an<br>
antibody directed against the antibody ligands. A number of anti-human immunoglobulin<br>
(Ig) molecules are known in the art (e.g., commercially available goat anti-human Ig or rabbit<br>
anti-human Ig). Ig molecules for use herein will preferably be of the IgG or IgA type.<br>
however, IgM may also be appropriate in some instances. The Ig molecules can be readily<br>
conjugated to a detectable enzyme label, such as horseradish peroxidase, glucose oxidase,<br>
Beta-galactosidase, alkaline phosphatase and urease, among others, using methods known to<br>
those of skill in the art. An appropriate enzyme substrate is then used to generate a detectable<br>
signal.<br>
Alternatively, a "two antibody sandwich" assay can be used to detect the proteins of<br>
the present invention. In this technique, the solid support is reacted first with one or more of<br>
the antibodies directed against Env (e.g., gp120), washed and then exposed to the test sample.<br>
Antibodies are again added and the reaction visualized using either a direct color reaction or<br>
using a labeled second antibody, such as an anti-immunoglobulin labeled with horseradish<br>
peroxidase, alkaline phosphatase or urease.<br>
Assays can also be conducted in solution, such that the viral proteins and antibodies<br>
thereto form complexes under precipitating conditions. The precipitated complexes can then<br>
be separated from the test sample, for example, by centrifugation. The reaction mixture can<br>
be analyzed to determine the presence or absence of antibody-antigen complexes using any of<br>
a number of standard methods, such as those immunodiagnostic methods described above.<br>
The modified Env proteins, produced as described above, or antibodies to the<br>
proteins, can be provided in kits, with suitable instructions and other necessary reagents, in<br>
order to conduct immunoassays as described above. The kit can also contain, depending on<br>
the particular immunoassay used, suitable labels and other packaged reagents and materials<br>
(i.e. wash buffers and the like). Standard immunoassays, such as those described above, can<br>
be conducted using these kits.<br>
The Env polypeptides and polynucleotides encoding the polypeptides can also be used<br>
in vaccine compositions, individually or in combination, in e.g., prophylactic (i.e., to prevent<br>
infection) or therapeutic (to treat HIV following infection) vaccines. The vaccines can<br>
comprise mixtures of one or more of the modified Env proteins (or nucleotide sequences<br>
encoding the proteins), such as Env (e.g., gp120) proteins derived from more than one viral<br>
isolate. The vaccine may also be administered in conjunction with other antigens and<br>
immunoregulatory agents, for example, immunoglobulins, cytokines, iymphokines, and<br>
chemokines, including but not limited to IL-2, modified IL-2 (cys125 -ser125), GM-CSF, IL-<br>
12, ?-interferon, IP-10, MIP1ß and RANTES. The vaccines may be administered as<br>
polypeptides or, alternatively, as naked nucleic acid vaccines (e.g., DNA), using viral vectors<br>
(e.g., retroviral vectors, adenoviral vectors, adeno-associated viral vectors) or non-viral<br>
vectors (e.g.. liposomes, particles coated with nucleic acid or protein). The vaccines may also<br>
comprise a mixture of protein and nucleic acid, which in turn may be delivered using the<br>
same or different vehicles. The vaccine may be given more than once (e.g., a "prime"<br>
administration followed by one or more "boosts") to achieve the desired effects. The same<br>
composition can be administered as the prime and as the one or more boosts. Alternatively,<br>
different compositions can be used for priming and boosting.<br>
The vaccines will generally include one or more "pharmaceutically acceptable<br>
excipients or vehicles" such as water, saline, glycerol, ethanol, etc. Additionally, auxiliary<br>
substances, such as wetting or emulsifying agents, pH buffering substances, and the like, may<br>
be present in such vehicles.<br>
A carrier is optionally present which is a molecule that does not itself induce the<br>
production of antibodies harmful to the individual receiving the composition. Suitable<br>
carriers are typically large, slowly metabolized macromolecules such as proteins,<br>
polysaccharides, polylactic acids, polyglycollic acids, polymeric amino acids, amino acid<br>
copolymers, lipid aggregates (such as oil droplets or liposomes), and inactive virus particles.<br>
Such carriers are well known to those of ordinary skill in the art. Furthermore, the Env<br>
polypeptide may be conjugated to a bacterial toxoid, such as toxoid from diphtheria, tetanus,<br>
cholera, etc.<br>
Adjuvants may also be used to enhance the effectiveness of the vaccines. Such<br>
adjuvants include, but are not limited to: (1) aluminum salts (alum), such as aluminum<br>
hydroxide, aluminum phosphate, aluminum sulfate, etc.; (2) oil-in-water emulsion<br>
formulations (with or without other specific immunostimulating agents such as muramyl<br>
peptides (see below) or bacterial cell wall components), such as for example (a) MF59<br>
(International Publication No. WO 90/14837), containing 5% Squalene, 0.5% Tween 80, and<br>
0.5% Span 85 (optionally containing various amounts of MTP-PE (see below), although not<br>
required) formulated into submicron particles using a microfluidizer such as Model 110Y<br>
microfluidizer (Microfluidics, Newton, MA), (b) SAF, containing 10% Squalane, 0.4%<br>
Tween 80, 5% pluronic-blocked polymer L121, and thr-MDP (see below) either<br>
microfluidized into a submicron emulsion or vortexed to generate a larger particle size<br>
emulsion, and (c) Ribi™ adjuvant system (RAS), (Ribi Immunochem, Hamilton, MT)<br>
containing 2% Squalene, 0.2% Tween 80, and one or more bacterial cell wall components<br>
from the group consisting of monophosphorylipid A (MPL), trehalose dimycolate (TDM),<br>
and cell wall skeleton (CWS), preferably MPL + CWS (Detox™); (3) saponin adjuvants,<br>
such as Stimulon™ (Cambridge Bioscience, Worcester, MA) may be used or particle<br>
generated therefrom such as ISCOMs (immunostimulating complexes); (4) Complete<br>
Freunds Adjuvant (CFA) and Incomplete Freunds Adjuvant (IFA); (5) cytokines, such as<br>
interleukins (IL-1, IL-2, etc.), macrophage colony stimulating factor (M-CSF), tumor necrosis<br>
factor (TNF), etc.; (6) detoxified mutants of a bacterial ADP-ribosylating toxin such as a<br>
cholera toxin (CT), a pertussis toxin (PT), or an E. coli heat-labile toxin (LT), particularly<br>
LT-K63 (where lysine is substituted for the wild-type amino acid at position 63) LT-R72<br>
(where arginine is substituted for the wild-type amino acid at position 72), CT-S109 (where<br>
serine is substituted for the wild-type amino acid at position 109), and PT-K9/G129 (where<br>
lysine is substituted for the wild-type amino acid at position 9 and glycine substituted at<br>
position 129) (see, e.g., International Publication Nos. W093/13202 and W092/19265); and<br>
(7) other substances that act as immunostimulating agents to enhance the effectiveness of the<br>
composition.<br>
Muramyl peptides include, but are not limited to, N-acetyl-muramyl-L-threonyl-D-<br>
isoglutamine (thr-MDP), N-acteyl-normuramyl-L-alanyl-D-isogluatme (nor-MDP), N-<br>
acetylmuramyl-L-alanyl-D-isogluatminyl-L-alanine-2-(1'-2'-dipalmitoyl-sn-glycero-3-<br>
huydroxyphosphoryloxy)-ethylamine (MTP-PE), etc.<br>
Typically, the vaccine compositions are prepared as injectables, either as liquid<br>
solutions or suspensions; solid forms suitable for solution in, or suspension in, liquid vehicles<br>
prior to injection may also be prepared. The preparation also may be emulsified or<br>
encapsulated in liposomes for enhanced adjuvant effect, as discussed above.<br>
The vaccines will comprise a therapeutically effective amount of the modified Env<br>
proteins, or complexes of the proteins, or nucleotide sequences encoding the same, and any<br>
other of the above-mentioned components, as needed. By "therapeutically effective amount"<br>
is meant an amount of a modified Env (e.g., gp120) protein which will induce a protective<br>
immunological response in the uninfected, infected or unexposed individual to which it is<br>
administered. Such a response will generally result in the development in the subject of a<br>
secretory, cellular and/or antibody-mediated immune response to the vaccine. Usually, such<br>
a response includes but is not limited to one or more of the following effects; the production<br>
of antibodies from any of the immunological classes, such as immunoglobulins A, D, E, G or<br>
M; the proliferation of B and T lymphocytes; the provision of activation, growth and<br>
differentiation signals to immunological cells; expansion of helper T cell, suppressor T cell,<br>
and/or cytotoxic T cell.<br>
Preferably, the effective amount is sufficient to bring about treatment or prevention of<br>
disease symptoms. The exact amount necessary will vary depending on the subject being<br>
treated; the age and general condition of the individual to be treated; the capacity of the<br>
individual's immune system to synthesize antibodies; the degree of protection desired; the<br>
severity of the condition being treated; the particular Env polypeptide selected and its mode<br>
of administration, among other factors. An appropriate effective amount can be readily<br>
determined by one of skill in the art. A "therapeutically effective amount" will fall in a<br>
relatively broad range that can be determined through routine trials.<br>
Once formulated, the nucleic acid vaccines may be accomplished with or without viral<br>
vectors, as described above, by injection using either a conventional syringe or a gene gun,<br>
such as the Accell® gene delivery system (PowderJect Technologies, Inc., Oxford, England).<br>
Delivery of DNA into cells of the epidermis is particularly preferred as this mode of<br>
administration provides access to skin-associated lymphoid cells and provides for a transient<br>
presence of DNA in the recipient Both nucleic acids and/or peptides can be injected either<br>
subcutaneously, epidermally, intradermally, intramucosally such as nasally, rectally and<br>
vaginally, intraperitoneally, intravenously, orally or intramuscularly. Other modes of<br>
administration include oral and pulmonary administration, suppositories, needle-less<br>
injection, transcutaneous and transdermal applications. Dosage treatment may be a single<br>
dose schedule or a multiple dose schedule. Administration of nucleic acids may also be<br>
combined with administration of peptides or other substances.<br>
While the invention has been described in conjunction with the preferred specific<br>
embodiments thereof, it is to be understood that the foregoing description as well as the<br>
examples which follow are intended to illustrate and not limit the scope of the invention.<br>
Other aspects, advantages and modifications within the scope of the invention will be<br>
apparent to those skilled in the art to which the invention pertains.<br>
Experimental<br>
Below are examples of specific embodiments for carrying out the present invention.<br>
The examples are offered for illustrative purposes only, and are not intended to limit the<br>
scope of the present invention in any way.<br>
Efforts have been made to ensure accuracy with respect to numbers used (e.g.,<br>
amounts, temperatures, etc.), but some experimental error and deviation should, of course, be<br>
allowed for.<br>
Example 1<br>
A.1. Best-Fit and Homology Searches<br>
The crystal structure of HXB-2 gp 120 was downloaded from the Brookhaven<br>
database (COMPLEX (HIV ENVELOPE PROTEIN/CD4/FAB) 15-JUN-98 1GC1<br>
TITLE: HIV-1 GP120 CORE COMPLEXED WITH CD4 AND A NEUTRALIZING<br>
HUMAN ANTIBODY). Beta strands 3, 2, 21, and 20 of gp 120 form a sheet near the CD4<br>
binding site. Strands ß-3 and ß-2 are connected by the V1/V2 loop. Strands ß-21 and ß-20<br>
are connected by another small loop. The H-bonds at the interface between strands ß-2 and<br>
ß-21 are the only connection between domains of the "lower" half of the protein (joining<br>
helix alpha 1 to the CD4 binding site). This beta sheet and these loops mask some antigens<br>
(e.g., antigens which may generate neutralizing antibodies) that are only exposed during the<br>
CD4 binding.<br>
Constructs that remove enough of the beta sheet to expose the antigens in the CD4<br>
binding site, but leave enough of the protein to allow correct folding were designed.<br>
Specifically targeted were modifications to the small loop and, optional deletion of the V1 /V2<br>
loops. Three different types of constructs were designed: (1) constructs encoding<br>
polypeptides mat leave the number of residues making up the entire 4-strand beta sheet intact,<br>
but replace one or more residues; (2) constructs that encode polypeptide having at least one<br>
residue of at least one beta strand excised or (3) constructs encoding polypeptides having at<br>
least two residues of at least one beta strand excised. Thus, a total of 6 different turns were<br>
needed to rejoin the ends of the strands.<br>
Initially, residues in the small loop (residues 427-430, relative to HXB-2) and<br>
connected beta strands (ß-20 and ß-21) were modified to contain Gly and Pro (common in<br>
beta turns). These sequences were then used as the target to match in each search. The<br>
geometry of the target was matched to known proteins in the Brookhaven Protein Data Bank.<br>
In particular, 5-residue turns (including an overlapping single residue at the N-terminal, the 2<br>
residue target turn and 2 overlapping residues at the C-terminal) were searched in the<br>
databases. In other words, these modified loops add a 2 residue turn that should be able to<br>
support a geometry that will maintain the beta-sheet structure of the wild type protein. The<br>
calculations were performed using the default parameters in the Loop Search feature of the<br>
Biopolymer module of the Sybyl molecular modeling package. In each case, the 25 best fits<br>
based on geometry alone were reviewed and, of those, several selected for homology and fit.<br>
In addition, it was also determined what modifications could be made to remove most<br>
of the V1/V2 loop (residues 124-198, relative to HXB-2) yet leave the geometry of the<br>
protein intact. As with the small loop, constructs were also designed which excised one or<br>
more residues from the ß-2 strand (residues 119-123 of HXB-2), the ß-3 strand (residues 199-<br>
201 of HXB-2) or both ß-2 and ß-3. For these constructs, known loops were searched to<br>
match the geometry of a pentamer (including two remaining residues from the N-terminal<br>
side, a 2 residue turn and 1 C-terminal residue). For these searches, Gly-Gly was preferred as<br>
the insert along with at least one C-terminal substitution.<br>
A.2. Small Loop Replacements<br>
In one aspect, the native sequence was replaced with residues that expose the CD4<br>
binding site, but leave the overall geometry of the protein relatively unchanged. For the<br>
small loop replacements, the target to match was: ASN425-MET426-GLY427-GLY428-<br>
GLY431. Results of the search are summarized in Table 1.<br>
Table 1: Search of Small Loop (Asn425 through Gly431)<br>
Rank	Sequence	RMSD	% Homology	Seq Id No.<br>
Best fit	LYS-ASP-SER-ASN-ASN	0.16689	62.5	27<br>
3	TYR-GLY-LEU-GLY-LEU	0.220308	62.5	28<br>
4	GLU-ARG-GLU-ASP-GLY	0.241754	62.5	29<br>
7	ARG-LYS-GLY-GLY-ASN	0.24881	100	30<br>
12	TRP-THR-GLY-SER-TYR	0.26417	83.33	31<br>
Based on these results, constructs encoding Gly-Gly (#7), Gly-Ser (#12) or Gly-Gly-<br>
Asn (#7) were recommended.<br>
As V1/V2 and one or more residues of ß-2 and ß-3 are also optionally deleted in the<br>
modified polypeptides of the invention, known loops to match the geometry of the V1/V2<br>
loop were also searched. The V1/V2 loop the target to match was: Lys121-Leu-122-Gly123-<br>
Gly124-Ser199. Some notable matches are shown in Table 2:<br>
Table 2; Search of V1/V2 loop (Lys121 through Ser199)<br>
Rank	Sequence	RMSD	% Homology	Seq Id. No.<br>
Best fit	GLN-VAL-H1S-ASP-GLU	0.154764	68.75	32<br>
2	LYS-GLU-GLY-ASP-LYS	0.15718	81.25	33<br>
9	ARG-SER-GLY-ARG-SER	0.173731	68.75	34<br>
11	THR-LEU-GLY-ASN-SER	0.175554	81.25	35<br>
16	HIS-PHE-GLY-ALA-GLY	0.178772	93.75	36<br>
Based on these searches, constructs encoding Gly-Asn in place of V1 /V2 were<br>
recommended.<br>
A.3. One Additional Residue Excisions<br>
For a slightly truncated small loop, one more residue was trimmed from each beta<br>
strand to slightly shorten the beta sheet. The target to match was: ILE424-ASN425-<br>
GLY426-GLY427-LYS432. Results are shown in Table 3:<br>
Table 3: Search of Beta sheet shortened by One residue (Ile424 through Lys432)<br>
Rank	Sequence	RMSD	% Homology	Seq Id No.<br>
Best fit:	ARG-MET-ALA-PRO-VAL	0.316805	58.33	37<br>
Best horn:	ASP-SER-ASP-GLY-PRO	0.440896	83.33	38<br>
Although these searches showed more variation and worse fits than the previous<br>
truncation, the Pro-Val or Pro-Leu encoding constructs were very similar. Accordingly, Ala-<br>
Pro encoding constructs were recommended.<br>
Sequences encoding gp120 polypeptides having V1 /V2 deleted and an additional<br>
residue from ß-2 or ß-3 excised were also searched. The V1/V2 loop the target to match was:<br>
VAL120-LYS121-GLY122-GLY123-VAL200. Some notable matches are shown in Table 4.<br>
Table 4: Search of V1/V2 loop (Val120 through Val200)<br>
Rank	Sequence	RMSD	% Homology	Seq Id No<br>
Best fit:	THR-VAL-ASP-PRO-TYR	0.400892	58.33333	39<br>
2	SER-THR-ASN-PRO-LEU	0.402575	54.16667	40<br>
3	THR-ARG-SER-PRO-LEU	0.403965	58.33333	41<br>
7	ARG-MET-ALA-PRO-VAL	0.440118	58.33333	42<br>
The construct encoding Ala-Pro (e.g., #7) was recommended.<br>
A.4. Further Excisions<br>
In yet another truncation, an additional residue was trimmed from the ß-20 and ß-21<br>
strands to further shorten the beta sheet. The target to match was ILE423-ILE424-GLY425-<br>
GLY426-ALA433. Notable matches are shown in Table 5.<br>
Table 5: Search of Beta sheet shortened by Two Residues (Ile423 through Ala433)<br>
Rank	Sequence	RMSD	% Homology	Seq Id No<br>
Best fit:	THR-TYR-GLU-GLY-VAL	0.130107	79.16666	43<br>
2	GLN-VAL-GLY-ASN-THR	0.138245	79.16666	44<br>
3:	THR-VAL-GLY-GLY-ILE	0.153362	100	45<br>
A construct encoding Gly-Gly (e.g., #3), which has 100% homology, was<br>
recommended.<br>
Also searched were sequences encoding a deleted V1/V2 region and at least two<br>
residues excised from ß-2, ß-3 or at least one residue excised from ß-2 and ß-3. The target to<br>
match was: CYS119-VAL120-GLY121 -GLY122-ILE201. Notable matches are shown in<br>
Table 6.<br>
Table 6: Search of V1/V2 loop (Cys119 through Ile201)<br>
Rank	Sequence	RMSD	% Homology		Seq Id No	<br>
Best fit:	ASP-LEU-PRO-GLY-CYS	0.250501	75		46	<br>
4	ASP-VAL-GLY-GLY-LEU	0.290383	100		47	<br>
It was determined that both constructs would be used.<br>
B. 1. Constructs encoding modified Env polvpeptides<br>
As described above, the native loops extruding from the 4-ß antiparallel-stands were<br>
excised and replaced with 1 to 3 residue turns. The loops were replaced so as to leave the<br>
entire ß-strands or excised by trimming one or more amino acid from each side of the<br>
connected strands. The ends of the strands were rejoined with<br>
turns that preserve the same backbone geometry (e.g., tertiary structure of ß-20 and ß-21), as<br>
determined by searching the Brookhaven Protein Data Bank.<br>
Table 7A is a summary of the truncations of the variable regions 1 and 2<br>
recommended for this study, as determined in Example 1.A. above.<br>
Table 7A<br>
V1/V2 Modifications	SEQ ID NO	Figure<br>
-LEU122-GLY-ASN-SER199	7	10<br>
-L YS121 - ALA-PRO- VAL200-	6	9<br>
-VAL120-GLY-GLY-ILE201-	4	7<br>
-VAL120-PRO-GLY-ILE201B-	5	8<br>
-VAL120-GLY-ALA-GLY-ALA204-	3	6<br>
- VAL120-GLY-GLY-ALA-THR202-	8	11<br>
-VAL127-GLY-ALA-GLY-ASN195-	25	28<br>
As previously noted, the polypeptides encoded by the constructs of the present<br>
invention are numbered relative to HXB-2, but the particular amino acid residue of the<br>
polypeptides encoded by these exemplary constructs is based on SF-162. Thus, for example,<br>
although amino acid residue 195 in HXB-2 is a serine (S), constructs encoding polypeptides<br>
having then wild type SF162 sequence will have an asparagine (N) at this position. Table 7B<br>
shows just three of the variations in amino acid sequence between strains HXB-2 and SF162.<br>
The entire sequences, including differences in residue and amino acid number, of HXB-2 and<br>
SF162 are shown in the alignment of Figure 2 (SEQ ID NOs: 1 and 2).<br>
Table 7B<br>
HXB-2 amino acid number	HXB-2 Residue	SF162 Residue/amino acid number<br>
128	Serine (S)	Thr(T)/114<br>
195	Serine (S)	Asn(N)/188<br>
426	Met(M)	Arg(R)/411<br>
Constructs containing deletions in the ß-20 strand, ß-21 stand and small loop were<br>
also constructed. Shown in Table 8 are constructs encoding truncations in these regions. The<br>
constructs in Table 8 are numbered relative to HXB-2 but the unmodified amino acid<br>
sequence is based on SF162: Thus, the construct encodes an arginine (Arg) as is found in<br>
SF162 in the amino acid position numbered 426 relative to HXB-2 (See, also, Table 7B).<br>
Changes from wildtype (SF162) are shown in bold in Table 8B.<br>
Table 8<br>
Small Loop/ß-20 and ß-21 (Modified)	SEQ ID NO	Figure<br>
-TRP427-GLY-GLY431-	9	12<br>
-ARG426-GLY-GLY-GLY431 -	10	13<br>
-ARG426-GLY-SER-GLY431B-	11	14<br>
-ARG426-GLY-GLY-ASN-LYS432-	12	15<br>
-ASN425-ALA-PRO-LYS432-	13	16<br>
-ILE424-GLY-GLY-ALA433-	14	17<br>
-ILE423-GLY-GLY-MET434-	15	18<br>
GLN422-GLY-GLY-TYR435-	16	19<br>
-GLN422-ALA-PRO-TYR435B-	17	20<br>
The deletion constructs shown in Tables 7 and 8 for each one of the ß-strands and<br>
combinations of them are constructed. These deletions will be tested in the Env forms gp120,<br>
gpl40 and gp160 from different HIV strains like subtype B strains (e.g., SF162, US4, SF2),<br>
subtype E strains (e.g., CM235) and subtype C strains (e.g., AF110968 or AF110975).<br>
Exemplary constructs for SF162 are shown in the<br>
Figures and are summarized in Table 9. As noted above in Figure 2 and Table 7B, in the<br>
bridging sheet region, the amino acid sequence of SF162 differs from HXB-2 in that the<br>
Met426 of HXB-2 is an Arg in SF162. In Table 9, V1/V2 refers to deletions in the V1/V2<br>
region; # bsm refers to a modification in the bridging sheet small loop.<br>
Table 9			<br>
Construct	Seq.Id.	Fig.	Modification/Amino acid sequence<br>
Val120-Ala204	3	6	V1/V2: Val120-Gly-Ala-Gly-Ala204<br>
Vall20-Ilc201	4	7	V1/V2: Val120-Gly-Gly-Ile201<br>
Vall20-Ilc201B	5	8	V1/V2: Val120-Pro-Gly-Ile201<br>
Lys121-Va1200	6	9	V1/V2: Lys121-Ala-Pro-Val200<br>
Combinations of V1/V2 deletions and bridging sheet small loop modifications in<br>
addition to those specifically shown in Table 9 are also within the scope of the present<br>
invention. Various forms of the different embodiments of the invention, described herein,<br>
may be combined.<br>
The first screening will be done after transient expression in COS-7, RD and/or 293<br>
cells. The proteins that are expressed will be analyzed by immunoblot, ELISA, and for<br>
binding to mAbs directed to the CD4 binding site and other important epitopes on gp120 to<br>
determine integrity of structure. They will also be tested in a CD4 binding assay and, in<br>
addition, the binding of neutralizing antibodies, for example using patient sera or mAb 448D<br>
(directed to Glu370 and Tyr384, a region of the CD4 binding groove that is not altered by the<br>
deletions).<br>
The immunogenicity of these novel Env glycoproteins will be tested in rodents and<br>
primates. The structures will be administered as DNA vaccines or adjuvanted protein<br>
vaccines or in combined modalities. The goal of these vaccinations will be to archive broadly<br>
reactive neutralizing antibody responses.<br>
WE CLAIM:<br>
---------<br>
1. A polynucleotide encoding a modified HIV Env polypeptide<br>
wherein the polypeptide has at least one amino acid deleted or<br>
replaced as compared to wild-type in the region corresponding to<br>
residues 420 to 436 numbered relative to HXB-2 (SEQ ID NOs 1).<br>
2. The polynucleotide as claimed in claim 1, wherein the<br>
region corresponding to residues 124-198 numbered relative to<br>
HXB-2 is deleted as compared to wild type and at least one amino<br>
acid is deleted or replaced as compared to wild type in the<br>
regions corresponding to the residues 119 to 123 and 199 to 210<br>
relative to HXB-2 (SEQ ID NO:1).<br>
3. The polynucleotide as claimed in either one of claims 1 or 2,<br>
wherein at least one amino acid in the region corresponding to<br>
residues 427 through 429 relative to HXB-2(SEQ ID NOs 1) is<br>
deleted or replaced as compared to wild type.<br>
4. The polynucleotide as claimed in either one of claims 1 or<br>
2, wherein the amino acid sequence of the modified HIV Env<br>
polypeptide is based on strain SF 162.<br>
5. An immunogenic modified HIV Env polypeptide having at<br>
least one amino acid deleted or replaced as compared to wild type<br>
in the region corresponding to residues 420 through 436, relative<br>
to HXB2 (SEQ ID No:1).<br>
6. ThE polypeptide as claimed in claim 5, wherein one amino<br>
acid is deleted in the region corresponding to residues 420<br>
through 436, relative to HXB-2 (SEQ ID No: 1).<br>
7. The polypeptide as claimed in claim 5 wherein more than<br>
one amino acid is deleted in the region corresponding to residues<br>
420 through 436, relative to HXB-2 (SEQ ID No:1).<br>
8. The polypeptide as claimed in claim 5, wherein at least<br>
one amino acid is replaced in the region corresponding to<br>
residues 420 through 436, relative to HXB-2(SEQ ID NO: 1).<br>
9. The polypeptide as claimed in claim S, wherein at least<br>
one amino acid residue is deleted or replaced in the region<br>
corresponding to amino acid residue 427 through 429, relative to<br>
HXB-2 (SEQ ID NO: 1).<br>
10. The polypeptide as claimed in any one of claims 5-9<br>
wherein the V1 and V2 regions the polypeptide are truncated.<br>
11. The polypeptide as claimed in claims 5-10 wherein the<br>
amino acid sequence of the modified HIV Env polypeptide is based<br>
on strain SF 162.<br>
12. A construct useful for stimulating an immune response,<br>
the construct comprising: control sequences which regulate trans-<br>
cription and translation, a coding sequence regulated by the<br>
control sequences, enhancers, polyadenylation sequences, wherein<br>
the coding sequence comprises a nucleotide sequence selected from<br>
the group consisting of: the nucleotide sequence depicted in<br>
Figure 6 (SEQ ID NO: 3); the nucleotide sequence depicted in<br>
Figure 7 (SEQ ID NO: 4), the nucleotide sequence depicted in<br>
Figure 8 (SEQ ID NO:5), the nucleotide sequence depicted in<br>
Figure 9 (SEQ ID NO: 6), the nucleotide sequence depicted in<br>
Figure 10 (SEQ ID NO: 7), the nucleotide sequence depicted in<br>
Figure 11 (SEQ ID NO: 3), the nucleotide sequence depicted in<br>
Figure 12 (SEQ ID NO: 9), the nucleotide sequence depicted in<br>
Figure 13 (SEQ ID NO: 10), the necleotide sequence depicted in<br>
Figure 14 (SEQ ID NO: 11), the nucleotide sequence depicted in<br>
Figure 15 ( SEQ ID NO: 12), the nucleotide sequence depicted in<br>
Figure 16 (SEQ ID NO: 13), the nucleotide sequence depicted in<br>
Figure 17 ( SEQ ID NO: 14), the nucleotide sequence depicted in<br>
Figure 18 ( SEQ ID NO: 15), the nucleotide sequence depicted in<br>
Figure 19 (SEQ ID NO: 16), the nucleotide sequence depicted in<br>
Figure 20 (SEQ ID NO: 17), the nucleotide sequence depicted in<br>
Figure 21 (SEQ ID NO: 18), the nucleotide sequence depicted in<br>
Figure 22 (SEQ ID NO: 19), the nucleotide sequence depicted in<br>
Figure 23 (SEQ ID Nus 20), the nucleotide sequence depicted in<br>
Figure 24 (SEQ ID NOs 21), the nucleotide sequence depicted in<br>
Figure 25 (SEQ ID NO: 22), the nucleotide sequence depicted in<br>
Figure 26 (SEQ ID NOs 23), the nucleotide sequence depicted in<br>
Figure 27 (SEQ ID NOs 24), the nucleotide sequence depicted in<br>
Figure 28 (SEQ ID NOs 25), and the nucleotide sequence depicted<br>
in Figure 29 (SEQ ID NOs 26).<br>
13. A vaccine composition comprising a polynucleotide<br>
encoding a modified Env polypeptide as claimed in any one of<br>
claims 1-5.<br>
14. A vaccine composition comprising a polynucleotide<br>
construct encoding a modified Env polypeptide as claimed in<br>
claim 12.<br>
15. A vaccine composition comprising a modified Env<br>
polypeptide as claimed in any of claims 5-11.<br>
16. The vaccine composition as claimed in any of claims<br>
13—15 further comprising an adjuvant.<br>
17. A medicament comprising a polynucleotide which encodes<br>
a modified HIV Env polypeptide as claimed in any one of claims<br>
1-5, said medicament suitable for stimulating an immune response.<br>
18. A medicament comprising a polynucieotide construct as<br>
claimed in claim 12, said medicament suitable for stimulating an<br>
immune response.<br>
19. A medicament comprising a polypeptide which comprises a<br>
modified HIV Env polypeptide as claimed in any one of claims 5 to<br>
11, said medicament suitable for stimulating an immune response.<br>
20. he medicament as claimed in any one of claims 17 to 19<br>
further comprising an adjuvant.<br>
21. A medicament constituted of a first composition<br>
comprising a modified env polynucleotide as claimed in any one of<br>
claims 1-5 or 13 for use in a priming step, in combination with a<br>
second composition comprising a modified Env polynucleotide as<br>
claimed in any one of claims 1-5 or 13 in a boosting step, said<br>
medicament being suitable for stimulating an immune response in a<br>
subject.<br>
22. A medicament constituted of a second composition<br>
comprising a modified Env polynucleotide as claimed in any one of<br>
claims 1-5 or 13 for use in a boosting step, in combination with<br>
a first composition comprising a modified env polynucleotide as<br>
claimed in any one of claims 1-5 or 13 in a priming step, said<br>
medicament being suitable for stimulating an immune response in a<br>
subject.<br>
23. A medicament constituted of a first composition<br>
comprising a modified Env polypeptide as claimed in any one of<br>
claims 5-12 for use in a priming step, in combination with a<br>
second composition comprising a modified Env polypeptide as<br>
claimed in any one of claims 5-12 in a boosting step, said<br>
medicament being suitable for stimulating an immune response in a<br>
subject.<br>
24. A medicament constituted of a second composition<br>
comprising a modified Env polypeptide as claimed in any one of<br>
claims 5-12 for use in a boosting step, combination with a first<br>
compostion comprising a modified Env polypeptide as claimed in<br>
any one of claims 5-12 in a priming step, said medicament being<br>
suitable for stimulating an immune response in a subject.<br>
25. A medicament constituted of a first composition<br>
comprising a modified Env polynucleotide as claimed in any one of<br>
claims 1-5 or 13 for use in a priming step, in combination with<br>
a second composition comprising a modified Env polypeptide as<br>
claimed in any one of claims 5-12 in a boosting step, said<br>
medicament being suitable for stimulating an immune response in a<br>
subject.<br>
26. A medicament constituted of a second composition<br>
comprising a modified Env polypeptide as claimed in any one of<br>
claims 5-12 for use in a boosting step, in combination with a<br>
first composition comprising a modified Env polynucleotide as<br>
claimed in any one of claims 1—5 or 13 in a priming step, said<br>
medicament being suitable for stimulating an immune response in a<br>
subject.<br>
27. A medicament constituted of a first composition<br>
compirisng modified Env polypeptide as claimed in any one of<br>
claims 5—12 for use in a priming step, in combination with a<br>
second composition comprising a modified Env polynucleotide as<br>
claimed in any one of claims 1—5 or 13 in a boosting step, said<br>
medicament being suitable for stimulating an immune response in a<br>
subject.<br>
23. A medicament constituted of a second composition<br>
comprising a modified Env polynucleotide as claimed in any one<br>
of claims 1-5 or 13 for use in a boosting step, in combination<br>
with a first composition comprising modified Env polypeptide as<br>
claimed in any one of claims 5-12 in a priming step, said<br>
medicament being suitable for stimulating an immune response in a<br>
subject.<br>
29. The medicament as claimed in any one of claims 21-28<br>
wherein the first composition op second composition further<br>
comprises an adjuvant.<br>
30. The medicament as claimed in any one of claims 21-23<br>
wherein the first and second compositions further comprise an<br>
adjuvant.<br>
A polynucleotide encoding a modified HIV Env polypeptide<br>
wherein the polypeptide has at least one ammo acid deleted or<br>
replaced as compared to wild-type in the region corresponding to<br>
residues 420 to 436 numbered relative to HXB-2 (SEQ ID NO: 1).</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDEtNzc0LUtPTC1DT1JSRVNQT05ERU5DRS5wZGY=" target="_blank" style="word-wrap:break-word;">IN-PCT-2001-774-KOL-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDEtNzc0LUtPTC1GT1JNIDE1LnBkZg==" target="_blank" style="word-wrap:break-word;">IN-PCT-2001-774-KOL-FORM 15.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDEtNzc0LUtPTC1GT1JNLTI3LnBkZg==" target="_blank" style="word-wrap:break-word;">IN-PCT-2001-774-KOL-FORM-27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzc0LWtvbC1ncmFudGVkLWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-774-kol-granted-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzc0LWtvbC1ncmFudGVkLWFzc2lnbm1lbnQucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-774-kol-granted-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzc0LWtvbC1ncmFudGVkLWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-774-kol-granted-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzc0LWtvbC1ncmFudGVkLWNvcnJlc3BvbmRlbmNlLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-774-kol-granted-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzc0LWtvbC1ncmFudGVkLWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-774-kol-granted-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzc0LWtvbC1ncmFudGVkLWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-774-kol-granted-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzc0LWtvbC1ncmFudGVkLWV4YW1pbmF0aW9uIHJlcG9ydC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-774-kol-granted-examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzc0LWtvbC1ncmFudGVkLWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-774-kol-granted-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzc0LWtvbC1ncmFudGVkLWZvcm0gMTgucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-774-kol-granted-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzc0LWtvbC1ncmFudGVkLWZvcm0gMi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-774-kol-granted-form 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzc0LWtvbC1ncmFudGVkLWZvcm0gMjYucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-774-kol-granted-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzc0LWtvbC1ncmFudGVkLWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-774-kol-granted-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzc0LWtvbC1ncmFudGVkLWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-774-kol-granted-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzc0LWtvbC1ncmFudGVkLXJlcGx5IHRvIGV4YW1pbmF0aW9uIHJlcG9ydC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-774-kol-granted-reply to examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzc0LWtvbC1ncmFudGVkLXNwZWNpZmljYXRpb24ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-774-kol-granted-specification.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzc0LWtvbC1ncmFudGVkLXRyYW5zbGF0ZWQgY29weSBvZiBwcmlvcml0eSBkb2N1bWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-774-kol-granted-translated copy of priority document.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDEtNzc0LUtPTC1QQS5wZGY=" target="_blank" style="word-wrap:break-word;">IN-PCT-2001-774-KOL-PA.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="225661-apparatus-for-inputting-text.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="225663-apparatus-and-method-for-interfering-with-pathological-cells-survival-processes.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>225662</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/774/KOL</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>47/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Nov-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>19-Nov-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>27-Jul-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CHIRON CORPORATION</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>4560 HORTON STREET, EMERYVILLE, CA 94608</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>BARNETT SUSAN</td>
											<td>CHIRON CORPORATION, 4560 HORTON STREET-R-440, EMERYVILLE, CA 94608</td>
										</tr>
										<tr>
											<td>2</td>
											<td>HARTOG KARIN</td>
											<td>CHIRON CORPORATION, 4560 HORTON STREET-R 440, EMERYVILLE, CA 94608</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MARTIN ERIC</td>
											<td>CHIRON CORPORATION, 4560 HORTON STREET-R440, EMERYVILLE, CA 94608</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12 15/49</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US99/31272</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>1999-12-30</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/114,495</td>
									<td>1998-12-31</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/156,670</td>
									<td>1999-09-29</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/225662-a-polynucleotide-encoding-a-modified-hiv-env-env-polypeptide by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:52:26 GMT -->
</html>
