<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/225709-a-prokaryotic-recombinant-host-cell-comprising-a-heterologous-replication-initiation-protein by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:06:30 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 225709:&quot;A PROKARYOTIC RECOMBINANT HOST CELL COMPRISING A HETEROLOGOUS REPLICATION INITIATION PROTEIN&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A PROKARYOTIC RECOMBINANT HOST CELL COMPRISING A HETEROLOGOUS REPLICATION INITIATION PROTEIN&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A prokaryotic recombinant host cell comprising a heterologous replication initiation protein that activates a conditional origin of replication and an extrachromosomal DNA molecule comprising a heterologous therapeutic gene and a conditional origin of replication whose functionality in the prokaryotic recombinant host cell requires a replication initiating protein which is foreign to the host cell.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> The present invention relates to a novel conditional replication DNA molecule which can be used in gene therapy or for the production of recombinant proteins. The novel DNA molecules according to the present invention are designated pCOR™ herein after.<br>
Gene therapy consists in correcting a deficiency or an anomaly by introducing genetic information into the affected organ or cell. This information may be introduced either in vitro into a cell extracted from the organ and then reinjected into the body, or in vivo, directly into the target tissue. As a molecule of high molecular weight and of negative charge, DNA has difficulty in spontaneously crossing phospholipid cell membranes. Various vectors are thus. used in order to enable gene transfer to take place: viral vectors, on the one hand, and natural or synthetic chemical and/or biochemical vectors, on the other hand.<br>
Viral vectors (retroviruses, adenoviruses, adeno-associated viruses, etc.) are very effective, in particular for crossing membranes, but present a certain number of risks such as pathogenicity, recombination, replication, and immunogenicity.<br>
Chemical and/or biochemical vectors allow these risks to be avoided (for reviews, see Behr, 1993, Gotten and Wagner 1993). These are, for example, cations (calcium phosphate, DEAE-dextran, etc.) which act by forming precipitates with DNA, which may be "phagocytosed" by the cells. They may also be liposomes in which the DNA is incorporated and which fuse with the plasma membrane. Synthetic gene-transfer vectors are generally lipids or cationic polymers which complex the DNA and form with it a particle bearing positive surface charges. As illustrations of vectors of this type, mention may be made in particular of dioctadecylamidoglycylspermine (DOGS, Transfectam™) or N-[l-(2,3-dioleyloxy)propyl]-N^^-teimemylammonium (DOTMA, Lipofectin™).<br>
However, the use of chemical and/or biochemical vectors or naked DNA implies the possibility of producing large amounts of DNA of pharmacological purity. The reason for this is that in gene therapy techniques, the medicinal product consists of the DNA itself and it is essential to be able to manufacture, in suitable amounts, DNAs having properties which are appropriate for therapeutic use in man.<br><br>
In the case of non-viral vectorology, the vectors used are plasmids of bacterial origin.<br>
The plasmids generally used in gene therapy carry (i) an origin of replication, (ii) a marker<br>
gene such as a gene for resistance to an antibiotic (kanamycin, ampicillin, etc.) and (iii) one or<br>
more transgenes with sequences necessary for their expression (enhancer(s), promoters),<br>
1 polyadenylation sequences, etc.). However, the technology currently available is not entirely<br>
satisfactory.<br>
On the one hand, the risk remains of dissemination in the body. Thus, a bacterium which is present in the body can, at low frequency, receive this plasmid. There is a greater likelihood of this taking place if it involves an in vivo gene therapy treatment in which the i DNA may be disseminated in the body of the patient and may come into contact with bacteria which infect this patient or bacteria of the commensal flora. If the bacterium receiving the plasmid is an enterobacterium, such as E. coli, this plasmid can be replicated. Such an event then leads to dissemination of the therapeutic gene. Insofar as the therapeutic genes used in gene therapy treatments can code, for example, for a lymphokine, a growth factor, an anti-oncogene or a protein whose function is defective in the host and which thus makes it possible to correct a genetic defect, the dissemination of some of these genes could have unforeseeable and worrying effects (for example if a pathogenic bacterium acquired a human growth factor<br>
gene).<br>
On the other hand, the plasmids generally used in non-viral gene therapy also possess a marker for resistance to an antibiotic (ampicillin, kanamycin, etc.). The bacterium acquiring such a plasmid thus has an undeniable selective advantage since any antibiotic therapy, using an antibiotic from the same family as that which selects the plasmid resistance gene, will lead to selection of the plasmid in question. In this respect, ampiciHin. belongs to the a-lactams, which is the family of antibiotics which is most frequently used worldwide. The use in bacteria of selection markers which are not antibiotic-resistance genes would thus be particularly advantageous. This would avoid the selection of bacteria which may have received a plasmid carrying such a marker.<br>
It is thus particularly important to seek to limit the dissemination of therapeutic genes and resistance genes as much as possible.<br>
The subject of the present invention is specifically to propose novel DNA molecules which can be used in gene therapy or for the production of recombinant proteins in vitro and<br>
which replicate only in cells which can complement certain functions of these non-viral<br>
vectors.<br>
The invention also relates to a particularly effective method for preparing these DNA<br>
molecules.<br>
The DNA molecules claimed have the advantage of removing the risks associated with dissemination of the plasmid, such as (1) replication and dissemination, which can lead to uncontrolled overexpression of the therapeutic gene, (2) dissemination and expression of resistance genes. The genetic information contained in the DNA molecules according to the invention effectively comprises the therapeutic gene(s) and the signals for regulating its (their) expression, a functional conditional origin of replication which greatly limits the host cell spectrum of this plasmid, a selection marker of reduced size which is preferably different from a gene which imparts resistance to an antibiotic and, where appropriate, a DNA fragment which allows the resolution of plasmid multimers. The probability of these molecules (and thus the genetic information which they contain) being transferred to a microorganism, and i maintained stably, is very limited.<br>
Lastly, the vectors according to the invention, also referred to as miniplasmids on account of their circular structure, their reduced size and their supercoiled form, have the following additional advantages: on account of their size which is reduced in comparison with the ColEl-derived plasmids used conventionally, the DNA molecules according to the 1 invention potentially have better in vivo bioavailability, and the DNA molecules or pCOR stay in a stable extrachromosomal form in the host prokaryotic or eukaryotic cells that do not contain the initiating protein. In particular, they have improved capacities of cell penetration and distribution. Thus, it is acknowledged that the diffusion coefficient in tissues is inversely proportional to the molecular weight (Jain, 1987). Similarly, in the cell, high molecular weight molecules have poorer permeability across the plasma membrane. In addition, in order for the plasmid to pass into the nucleus, which is essential for its expression, the high molecular weight is also a disadvantage, the nuclear pores imposing a size limit for diffusion into the nucleus (Landford et al., 1986). The reduction in size of the non-therapeutic parts of the DNA molecule (origin of replication and selection gene in particular) according to the invention also makes it possible to decrease the size of the DNA molecules. The part which allows the replication and selection of this plasmid in the bacterium (1 kb) is decreased by a factor of 3, counting, for example, 3 kb for the origin of replication and the resistance marker vector part.<br>
This decrease (i) in molecular weight and (ii) in negative charge imparts improved tissue, cellular and nuclear bioavailability and diffusion to the molecules of the invention.<br>
More precisely, the present invention relates to a circular DNA molecule, which is useful in gene therapy, this molecule comprising at least one nucleic acid sequence of interest, characterized in that the region which allows its replication comprises an origin of replication whose functionality in a host cell requires the presence of at least one specific protein which is foreign to the said host cell.<br>
This DNA molecule may he in single- or double-stranded form and advantageously possesses a supercoiled form.<br>
For the purposes of the present invention, the host cells used can be of various origins. They can be eukaryotic or prokaryotic cells. According to a preferred embodiment of the invention, they are prokaryotic cells.<br>
The replication of bacterial plasmids conventionally requires the presence of at least one protein, which is coded for by the host cell, of the RNA polymerase, Rnase, DNA polymerase, etc. type. For the reasons already explained above, it is not possible to overcome entirely, with this type of replication, any possible risks of dissemination in the treated organism. Advantageously, the functionality of the origin of replication of the DNA molecule according to the invention requires the presence of a specific protein which is foreign to the host cell. The significance of this characteristic is to reduce the host spectrum of the claimed plasmid to specific strains that express this initiator protein. The DNA molecule developed within • the context of -the present invention thus advantageously possesses a so-called conditional origin of replication.<br>
The conditional origin of replication used according to the present invention may originate from plasmids or bacteriophages which share the following characteristics: they contain in their origin- of replication repeat sequences, or iterons, and they code for at least one replication-initiating protein (Rep) which is specific to them. By way of example, mention may be made of the conditional replication systems of the following plasmids and bacteriophages:<br>
According to a preferred embodiment of the invention, the origin of replication used in the DNA molecules claimed is derived from a natural E. coli plasmid referred to as R6K.<br>
The replication functions of R6K are grouped together in a 5.5 kbp DNA fragment i (Figure 1) comprising 3 origins of replication a, (3, and y (Y and a. providing 90 % of the replication) and an operon coding for the n replication-initiator protein and the protein Bis. The •minimum amount of genetic information required to maintain this plasmid at its characteristic number of copies (15 copies per genome) is contained in two elements: the 400 bp of on" y and the gene^zr, whose product is the n initiator protein.<br>
Ori y raay De divided into two functional parts: the core part and the activator element (Figure 1). The core part, which is essential for replication, contains the iterons (7 direct repeats of 22 bp) to which the E protein represented in SEQ E) No. 1 becomes bound, and flanking segments, which are targets of the host proteins (THF, DnaA).<br>
According to a preferred mode of the invention, the origin of replication of the vector claimed consists entirely or partially of this y origin of replication of the plasmid R6K and more preferably, entirely.or partially of SEQ ID No. 1 or one of its derivatives.<br>
The origin of replication described above, which has the advantage of being of very limited size, is functional solely in the presence of a specific initiator protein, protein Pi, produced by the gene pir (SEQ ID No. 2). Since this protein can act in trans, it is possible to physically dissociate the on gamma from the pir gene, which may be introduced into the genome of the cell which is chosen as the specific host for these plasmids. Mutations in II may alter its inhibitory functions (Inuzuka and Wada, 1985) and lead to an increase in the number of copies of the R6K derivatives, up to more than 10 times the initial number of copies. These substitutions may be within a domain of 40 amino acids, which therefore appears to be responsible for the control by n of the number of plasmid copies (Figure 2), or in other regions of the n protein.<br>
According to an advantageous embodiment of the present invention, the II protein, expressed in the host cell, results from the expression of the gene represented in SEQ ID No. 2 or one of its derivatives as defined above and more particularly of the gene pir 116 which comprises a mutation when compared with the pir gene. This mutation corresponds to the replacement of a proline by a leucine at position 106 from the start codon. In this context, the number of copies of the R6K derivatives is about 250 copies per genome.<br>
For the purposes of the present invention, the term derivative denotes any sequence which differs from the sequence considered, obtained by one or more modifications of genetic and/or chemical nature, as well as any sequence which hybridizes with these sequences or fragments thereof and whose product possesses the activity indicated with regard to the replication-initiator protein n. The term modification of the genetic and/or chemical nature may be understood to refer to any mutation, substitution, deletion, addition and/or modification of one or more residues. The term derivative also comprises the sequences homologous with the sequence considered, derived from other cellular sources and in particular cells of human origin, or from other organisms, and possessing an activity of the same type. Such homologous sequences may be obtained by hybridization experiments. The hybridizations may be performed starting with nucleic acid libraries, using the native sequence or a fragment thereof as probe, under conventional conditions of stringency (Maniatis et al., cf. General techniques of molecular biology), or, preferably, under conditions of high stringency.<br>
Besides a conditional origin of replication as defined above, the DNA molecules claimed contain a region comprising one (or more) gene(s) which make it possible to ensure selection of the DNA molecule in the chosen host.<br>
This may be a conventional marker of gene type which imparts resistance to an antibiotic, such as kanamycin, ampicillin, chloramphenicol, streptomycin, spectinomycin, h'vidomycin or the like.<br>
However, according to a preferred embodiment of the invention, this region is different from a gene which imparts resistance to an antibiotic. It may thus be a gene whose product is essential for the viability of the host envisaged, under defined culturing conditions. It may be, for example:<br>
a gene coding for a suppressor tRNA, of natural or synthetic origin. This is, more preferably,<br>
an amber codon tRNA (TAG)<br>
a gene whose product is necessary for metabolism of the cell, under certain culturing<br>
conditions, namely a gene involved in the biosynthesis of a metabolite (arnino acid, vitamin,<br>
: etc.), or a catabolism gene which makes it possible to assimilate a substance present in the culture medium (specific nitrogen or carbon source), etc.<br>
According to a preferred mode of the invention, this region contains an expression cassette of a gene coding for a suppressor tRNA for specific codons. This latter may be chosen, in particular, from those coding for phenylalanine, cysteine, proline, alanine and histidine ami-no acids. It is more particularly a suppressor tRNA for amber codons (TAG).<br>
In this particular case, the system used to select, in the host cells, the DNA molecules which are the subject of the present invention includes two elements: 1) on the DNA molecule, a gene coding for a suppressor transfer RNA for the amber codon (TAG) which constitutes the selection marker, known as (sup) gene and 2) a specific host, one of whose genes, which is essential under certain culture conditions, contains an amber TAG codon. This cell may grow, under the culture conditions for which the product of the gene containing the TAG codon is essential, only if the plasmid allowing the expression of sup is present in the cell. The culture conditions thus constitute the pressure for selection of the DNA molecule. The sup genes used may be of natural origin (Glass et al., 1982) or may originate from a synthetic construction (Nonnanly et al., 1986, Kleina et a/., 1990).<br>
Such a system offers great flexibility insofar as, depending on the gene containing an amber mutation, it is possible to determine various selective media. In the bacterium<br>
Lactococcus lactis for example, the amber codon is located in a purine biosynthesis gene. This allows the selection of the plasmid carrying the gene coding for the suppressor tRNA when the bacteria multiply in milk. Such a marker has the advantage of being very small and of containing no "foreign" sequences, originating from phages or transposons.<br>
According to a particular embodiment of the invention, the DNA molecule also comprises a DNA fragment, the target for site-specific recombinases, which allows the resolution of plasmid multimers.<br>
Thus, such a fragment, introduced on to a DNA molecule which is circular and whose origin of replication is, for example, ori gamma, allows the resolution of multimers of such a plasmid. Such multimers are observed, in particular, when the DNA molecule is prepared in a strain carrying a mutated allele ofpz'r, such &amp;spirlJ6, which makes it possible to increase the number of copies of the R6K derivatives.<br>
This recombination may be achieved by means of various systems which entail site-specific recombination between sequences. More preferably, the site-specific recombination of the invention is obtained by means of specific intramolecular recombination sequences which are capable of recombining with each other in the presence of specific proteins, generally referred to as recombinases. In this specific case, these are the recombinases XerC and XerD. For this reason, the DNA molecules according to the invention generally also comprise   a   sequence   which   allows   this   site-specific   recombination.   The   specific recombination system present in the genetic constructions according to the invention (recombinases and specific recognition site) may be of different origins. In particular, the specific sequences and the recombinases used may belong to different structural, classes, and in particular to the transposon Tn5 resolvase family or to the bacteriophage lambda integrase family. Among the recombinases belonging to the transposon Tn5 family, mention may be made in particular of the resolvase of transposon TnJ or of transposons 1n2J and TnJ22 (Stark et al., 1992); the Gin invertase of bacteriophage mu or alternatively plasmid resolvases, such as that of thepar fragment of RP4 (Abert et al., Mol. Mcrobiol. 12 (1994) 131). Among the recombinases belonging to the bacteriophage X integrase family, mention may be made in particular of the integrase of the phages lambda (Landy et al., Science 197 (1977) 1147), P22 and O80 (Leong et al., J. Biol. Chem. 260 (1985) 4468), HP1 of Haemophilus influenzae (Hauser et al,, J. Biol. Chem. 267 (1992) 6859), the Cre integrase of phage PI, the integrase of<br>
plasmid pSAM2 (EP 350 341) or alternatively the FLP recombinase of the 2p. plasmid and the XerC and XerD recombinases from E. coli.<br>
Preferably, the DNA molecules which form the subject of the present invention contain the fragment cer from the natural E. coli plasmid ColEl. The cer fragment used is a 382 bp HpaTL fragment from ColEl which has been shown to bring about, in cis, the resolution of plasmid multimers (Summers et al., 1984; Leung et al., 1985). It is also possible to use a HpaE-Taql fragment of smaller size (280 bp) or a smaller fragment (about 220 bp), contained in the HpalL fragment, which fragments possess the same properties (Summers and Sherratt, 1988). This resolution takes place by way of a specific intramolecular recombination, which involves four proteins encoded by the genome ofE. coli: ArgR, PepA, XerC and XerD (Stirling et al., 1988, 1989; Colloms et al., 1990, Blakely et al., 1993). It was found that insertion of the fragment cer from the natural E.coli plasmid ColEl allows to obtain a high resolution of plasmids multimers, thereby resulting in high proportion of monomers in a reproducible manner. This is particularly unexpected as it has been shown that the insertion of the cer site into a minicircle which contains the ColEl origin of replication from pBluescript SK+ did not result in efficient multimer resolution (Kreiss et al., AppLMicrobioLBiotechnol, 49:560-567 (1998)), and thus effective resolution in cis of plasmids is unpredictable and seems to depend on the plasmid conformation. In the case of the pCOR plasmid, an effective cis resolution is reached when cer is present on the pCOR, thereby resulting in a unexpectedly high monomers of pCOR in a reproducible manner.<br>
In this respect, it is particularly advantageous to use all or part of the cer fragment of ColEl or one of its derivatives as defined above.<br>
According to an implementation variant, the DNA molecules of the invention may also comprise a sequence capable of interacting specifically with a ligand. Preferably, this is a sequence capable of forming, by hybridization, a triple helix with a specific oligonucleotide. This sequence thus makes it possible to purify the molecules of the invention by selective hybridization with a complementary oligonucleotide immobilized on a support (see application WO 96/18744 and WO 02/07727). The sequence may be naturally present in the origin of replication of the plasmid as described in US publication application 2003/186268 of the Applicant, or naturally present in the transgene as described ha WO 02/07727, and alternatively can be positioned at any site in the DNA molecule of the invention, provided that it does not affect the functionality of the gene of interest and of the origin of replication.<br>
Formation of a triple helix by hybridization thus occurs between the oligonucleotide and the specific complementary sequence present in the DNA. In this connection, to obtain the best yields and the best selectivity, an oligonucleotide and a specific sequence which are fully complementary are used in the method of the invention. These can be, in particular, an oligonucleotide poly(CTT) and a specific sequence poly(GAA). For example, oligonucleotides containing repeated motifs such as CTT are capable of forming a triple helix with a specific sequence containing complementary units (GAA). The sequence in question can, in particular, be a region containing 7, 14 or 17 GAA units, and in the oligonucleotides a corresponding numbers of repeat CTT. In this case, the oligonucleotide i binds in an antiparallel orientation to the polypurine strand. These triple helices are stable only in the presence of Mg2+ (Vasquez et al., Biochemistry, 34: 7243-7251 (1995); Beal andDervan, Science, 251:1360-1363 (1991)).<br>
As stated above, the specific sequence can be a sequence naturally present in the pCOR, or may be a synthetic sequence introduced artificially in the latter.  It is especially advantageous to use an oligonucleotide capable of forming a triple helix with a sequence naturally present in the pCOR, for example in the origin of replication of a plasmid or in a marker gene. The synthesis of oligonucleotides capable of forming triple helices with these natural homopurine-homopyrimidine regions is particularly advantageous, as it may be applied to unmodified plasmids. Particularly preferred target sequences which can form triplex structures with particular oligonucleotides have been identified in ColEl and in origins of replication.   ColEl-derived plasmids contain a 12-mer homopurine sequence  mapped upstream of the RNA-H transcript involved in plasmid replication (Lacatena et al., Nature,. This sequence    forms    a    stable   triplex    structure   with   the    12-mer   complementary oligonucleotide. The pCOR backbone contains a homopurine stretch of 14 non repetitive bases located in the A+T-rich segment of the y origin replicon of pCOR (Levchenko et al., Nucleic Acids Res., 24:1936 (1996)).   This sequence forms a stable triplex structure with the 14-mer complementary oligonucleotide The corresponding oligonucleotides and efficiently . and specifically target their respective complementary sequences located within the origin of replication of either<br>
ColEl ori or pCOR (oriy). Also, use of an oligonucleotide capable of forming a triple helix with a sequence present in an origin of replication or a marker gene is especially advantageous, since it makes it possible, with the same oligonucleotide, to purify any DNA containing the said origin of replication or said marker gene. Hence it is not necessary to modify the plasmid or the double-stranded DNA in order to incorporate an artificial specific sequence in it.<br>
Although fully complementary sequences are preferred, it is understood, however, that some mismatches may be tolerated between the sequence of the oligonucleotide and the sequence present in the DNA, provided they do not lead to too great a loss of affinity. The sequence  present in the E. coli p-lactamase gene may be mentioned. In this case, the thymine interrupting the polypurine sequence may be recognized by a guanine of the third strand, thereby forming a G*TA triplet which it is stable when flanked by two T*AT triplets (Kiessling et al., Biochemistry, 31:2829-2834(1992).<br>
According to a particular embodiment, the oligonucleotides used may comprise the sequence (CCT)n, the sequence (CT)n or the sequence (CTT)n, in which n is an integer between 1 and 15 inclusive. It is especially advantageous to use sequences of the type (CT)n or (CTT)n. Oligonucleotides may also combine (CCT), (CT) or (CTT) units.<br>
The oligonucleotides used may be natural (composed of unmodified natural bases) or chemically modified. Li particular, the oligonucleotide may advantageously possess certain .chemical modifications enabling its resistance to or its protection against nucleases, or its affinity for the specific sequence, to be increased.<br>
As a DNA molecule representative of the present invention, the plasmid pXL2774 and its derivatives may be claimed most particularly. For the purposes of the invention, the term derivative is understood to refer to any construction derived from pXL2774 and containing one or more genes of interest other than the luciferase gene. Mention may also be made of the plasmids pXL3029, pXL3030, and plasmid pXL3179 or NV1FGF containing an expression cassette of a therapeutic gene. In a most preferred embodiment, the invention relates to a pCOR comprising the FGFa or FGF-1 gene as described in US patent 4,686,113 of the Applicant, which is designated pXL 3179 or NV1FGF.<br>
The present invention also relates to the development of a process for the construction of specific host cells, which are particularly effective for the production of these therapeutic<br>
DNA molecules.<br>
Another subject of the present invention relates to a process for the production of a circular DNA molecule, characterized in that a host cell is cultured containing at least one DNA molecule as defined above and a protein, which may or may not be expressed in situ, which conditions the functionality of the origin of replication of the said DNA molecule, which is specific and which is foreign to the said host cell, under conditions which allow the selection of host cells transformed by the said DNA molecules.<br>
More preferably, the protein which conditions the functionality of the origin of replication of the DNA molecule is expressed in situ from a corresponding gene. The gene coding for the repHcation-initiating protein may be carried by a subsidiary replicon, which is compatible with the derivatives of the conditional origin of replication used or which may be introduced into the genome of the host cell by recombination, by means of a transposon, a bacteriophage or any other vector. la the particular case in which the gene expressing the protein is placed on a subsidiary replicon, the latter also contains a promoter region for functional transcription in the cell, as well as a region which is located at the 3' end and which specifies a transcription termination signal As regards the promoter region, this may be a promoter region which is naturally responsible for expressing the gene under consideration when the latter is capable of functioning in the cell. It may also be a case of regions of different origin (responsible for expressing other proteins), or even of synthetic origin. In particular, it may be a case of promoter sequences for prokaryotic or bacteriophage genes. For example, it may be a case of promoter sequences obtained from the cell genome.<br>
As genes coding for the reph'cation-initiating protein, use may be made either of wild-type genes or of mutated alleles which make it possible to obtain an increased number of copies of the plasmids (or derivatives) specific for the initiator protein which conditions the functionality of the origin of replication used in the DNA molecule.<br>
Such mutants have been described in particular for the plasmids R6K (Inuzuka and Wada, 1985; Greener et al, (1990), Rtsl (TerawaM and Itoh, 19S5, Terawald et al, 1990; Zeng et al, 1990), F (Seelke et al, 1982; Helsberg et al, 1985; Kawasaki et al, 1991), RK2 (Durland et al, 1990; Haugan etal, 1992, 1995), pSClOl (Xia et al, 1991; Goebel et al, 1991; Fang etal, 1993).<br>
In the particular case in which the DNA molecule used possesses an origin of replication derived from the plasmid R6K, the initiator protein is a derivative of the n protein of this same plasmid. It is particularly advantageous to express a mutated form of this protein which is capable of increasing the number of initial copies appreciably. To do this, the gene incorporated into the host cell is preferably represented by all or part of the sequence represented in SEQ ID No. 2 or one of its derivatives and more preferably by thepirU6 gene. The associated mutation corresponds to the replacement of a proline by a leucine. According to a particular embodiment of the invention, this.pzW.75 gene is directly incorporated into the<br>
host cell genome.<br>
Advantageously, one of the genes of the specific host cell, which is essential under the culture conditions chosen, contains a specific codon which is recognizable by the selected suppressor tRNA in the DNA molecule. According to a preferred mode of the invention, this is an amber TAG codon. In this particular case, the cell may grow, under culture conditions for which the product of the gene containing the TAG codon is essential, only if the plasmid allowing the expression of sup is present in the host cell. The culture conditions thus constitute the pressure for selection of the DNA molecule.<br>
Preferably, the gene containing the amber codon is a gene involved in the biosynthesis<br>
of an amino acid, arginine. This gene, argE, codes for an N-acetylornithinase (Meinnel et al.,<br>
1992) and in this case .contains a TAG codon corresponding to a point mutation Gln-53<br>
(CAG)-&gt; TAG; the pressure for selection of the plasmid carrying the sup gene is then<br>
provided by culturing in minima.] M9 medium (Maniatis et al, 1989). However, this could<br>
also be, for example, a gene for biosynthesis of a vitamin or a nucleic acid base, or<br>
alternatively a gene which allows a specific nitrogen or carbon source to be used or any other<br>
gene whose functionality is essential for cellular viability under the chosen culture conditions.<br>
The host cell is preferably chosen from E. coli strains and is more preferably<br>
represented by the strain .£ coliXAC-1.<br>
According to a specific embodiment of the invention, the host cell used in the claimed process is a cell of the E. coli strain XAC-1, containing the pirl!6 gene in its genome and transformed by the plasmid pXL2774 or one of its derivatives.<br>
According to an advantageous variant of the invention, the host cell used in the process claimed is a prokaryotic cell in which the endAl gene or a homologous gene is inactivated. The endA gene codes for endonuclease I ofE. coli. This periplasmic enzyme has a<br>
non-specific activity of cleaving double-stranded DNA (Lehman, L R., G. G. Roussos and E. A. Pratt (1962) J. Biol. Chem. 237: 819-828; Wright M. (1971) J. Bacteriol. 107: 87-94). A study carried out on various strains of Escherichia coli (wild-type or endA) showed that the degradation of plasmid DNA incubated in extracts of these bacterial strains existed in the endA+ strains but not in the endA mutants. (Wnendt S. (1994) BioTechniques 17: 270-272). The quality of the plasmid DNA isolated from endA+ strains or from endA mutants was studied by the company Promega using their purification system (Shoenfeld, T., J. Mendez, D. Starts, E. Portman, B.fPatterson, J. Frederiksen and C. Smith. 1995. Effects of bacterial strains carrying the endAl genotype on DNA quality isolated with Wizard plasmid purification systems. Promega notes 53). Their conclusion is as follows: the quality of the DNA prepared from endA mutants is, overall, better than that of DNA prepared in the endA+ strains tested.<br>
The quality of the plasmid DNA preparations is thus affected by any contamination with this endonuclease (relatively long-term degradation of the DNA).<br>
The deletion or mutation of the endA gene can be envisaged without difficulty insofar as the mutants no longer having this endonuclease activity behave on the whole like wild-type bacteria (Diirwald, H. and H. Hoffrnann-Berling (1968) J. Mol. Biol. 34: 331-346).<br>
The endAl gene can be inactivated by mutation, total or partial deletion, disruption, etc. Inactivation of the endA gene of the E. coli strain chosen to produce the pCOR plasmids can be achieved more particularly by transferring, by means of the PI bacteriophage, the AendA:-:TcR deletion described by Cherepanov and Wackemagel (Cherepanov, P. P. and W. Wackemagel. 1995. Gene disruption in Escherichia coli: TcR and KmR cassettes with the option of Flp-catalyzed excision of the antibiotic-resistance determinant. Gene158:9-14) or by exchanging the wild-type allele present in the genome of the bacterium of interest with a mutated or deleted allele of endA, by homologous recombination. The use of this type of strain in the context of the present invention makes it possible advantageously to improve the quality of the DNA produced.<br>
The invention also relates to any recombinant cell containing a DNA molecule as defined above. This maybe a cell of various origins, of eukaryotic, prokaryotic, etc. type.<br>
According to another embodiment of the invention, the E. coli XAC-1 host cell used in the process claimed is designated TEX1, and comprises a traD gene, or a homologous gene thereof, inactivated to abolish F transfer. The traD is at the 5' end of one of the tra operon and<br>
encodes a 81.7 kDa membrane protein that is directly involved in DNA transfer and DNA metabolism (Frost et al., Microbiology Reviews, 1994, 58: 162-210). traD mutants do not transfer DNA (Tanicker et al., J.BacterioL, 1985, 162:584-590). The episomal traD gene may be inactivated by mutation, total or partial deletion, or disruption using methods well known to those of skill in the art (See Example 9). One method of inactivating this gene is described in Example 1, and the resulting E. coli XAC-1 pirl!6 endA' traD' strain so obtained is designated TEX1 (Soubrier et al., Gene Therapy, 1999, 6:1482-1488).<br>
According to one embodiment of the invention, the host cell used in the claimed process is a cell of the E. coli strain XAC-1, containing 1hepirlJ6 mutation combined with thepir42 mutation. Thepirll6 saA.pir42 mutations affect different domains of the pi protein. The pirl!6 mutation affects the copy number control region, whereas the pir42 mutation affects the putative leucine zipper motif, as displayed in Figure 11. The nucleotide and amino acid sequences of the pir gene containing the pirll 6 and pir42 mutations are set forth in Figure 12 and SEQ ID NOs: 21 and 22, respectively. Thepir42 mutation comprises a C to T transition at position 124 from the methionine initiator codon, and thus results in substitution of the pro line at position 42 by a leucine. Thepir42 mutation was described by Miron et al. (Proc Natl Acad Sci U S A, 1994. 91(14): p. 6438-42; EMBO J, 1992. 11(3): p.<br>
•p<br>
1205-16), and was reported to increase the copy number of an "ori gamma R6K-Km -pir42" plasmid by 2.5 fold as compared to the same plasmid harboring the wild-type pir gene. However the pir42 mutation was never used or described in combination with the _pz&gt;776'-mutation and while other copy-up mutations such as cop21 in the pir gene combined with ihepirll6 do not exhibit an increase of the plasmid copy number, combination of the pirl!6 wA.pir42 mutations in a £. coli XAC-1 endA traD' strain surprisingly showed a significant increase of the plasmid copy number. Applicants have thus shown unexpected results of this combination in terms of copy number of the plasmids produced in E. coli host strains comprising the mutated pir 116 and pir42 gene as compared with strains harboring pir 116 alone, or in a host cell comprising the pir 116 mutation combined with another mutation of the pir gene, such as the mutation cop21 (Inuzuka et al., FEES Lett, 1988. 228(1): p. 7-11). For example, E. coli TEXIpir42 (=XAC-1 endA' traD' pir J16 pir42\ exhibited a 2-5 fold increase in the number of plasmids, as compared to &amp; pir 116 strain, or strains comprising combined pirl 16 and cop21 mutations (See Example 11). In other embodiments, the pir gene comprises at least one mutation, which, for example, may<br>
occur in the copy number control region, in the leucine zipper-like motif, in the DNA binding region, or in one or more, of these regions or another region of the protein pi coded bythepzrgene.<br>
The prokaryotic host cell according to the present invention also comprises one or more mutations in the same or a different domain of the protein pi, coded by the pir gene copy, such as the DNA binding domain, and/or the copy number control region and/or the leucine-zipper motif. The prokaryotic recombinant host cell may comprise the heterologous pir gene is in a plasmid or in the genome of the host cell.<br>
Such mutations may be screened by using the fluorescence-based screening method I according to one aspect of the present invention as described thereafter. As shown in the Example 13, host cells comprising at least one mutation in the pir gene, the mutation pirl 16 and a mutation in the DNA binding domain were screened using the fluorescence-based screening method according to the present invention. Host cells comprising mutations present in the DNA binding domain in addition to ^!aspirll6, Le., as for example in the construct 100B, wherein the tyrosine (K) at position 292 is replaced by a methionine (M), in the construct 114C, wherein a glutamic acid (E) at position 130 is replaced by a valine (V), or in the construct 201C, wherein an aspartic acid (D) at position 117 is replaced by a glycine (G) (Fig. 26) are tested for their capacity to produce high copy number of plasmid using the fluorescence-based screening method.<br>
According to another embodiment of the present invention, the host cell used in the process claimed is a prokaryotic host cell in which the recA gene or a homologous gene has been inactivated. Preferably, the host cell according to the present invention is E. coli strain XAC-1 comprising mutations pir 116, pir42, endA', traD', recA'. Such a strain is designated TEX2pir42. recA may be inactivated by methods well known to those in the art. recA encodes a major recombination protein and mutations in this gene reduce the frequency of recombination-mediated alteration in plasmids and intramolecular recombination that could lead to the multimerization of plasmids. As described in Example 12, a deleted recA gene containing 3 translation stop codons (one in each frame) at its 5' end may be obtained by PCR. The resulting inactivated gene was then introduced by gene replacement into TEX1 genome (Example 12.1).<br>
These cells are obtained by any technique known to those skilled in the art which allows the introduction of the said plasmid into a given cell. Such a technique may be, in<br>
particular, transformation, electroporation, conjugation, fusion of protoplasts or any other<br>
technique known to those skilled in the art.<br>
Strain XAC-lpirlJ6 was deposited under the terms of the Budapest Treaty with the<br>
Collection Nationale De Cultures de Micro-organismes (CNCM), Institut Pasteur, 28, rue Dr.<br>
Roux, 75724 Paris Cedex 15, France, on October 10th, 2003 under accession no. 1-3108.<br>
Strain TEX2pir42 was deposited under the terms of the Budapest Treaty with the<br>
Collection Nationale De Cultures de Micro-organismes (CNCM), Institut Pasteur, 28, rue Dr.<br>
Roux, 75724 Paris Cedex 15, France, on October 10th, 2003, under accession no. 1-3109.<br>
The DNA molecules according to the invention may be used in any application of<br>
vaccination or of gene and cell therapy for transferring a gene to a given cell, tissue or<br>
organism, or for the production of recombinant proteins in vitro.<br>
In particular, they may be used for direct in vivo administration or for the modification<br>
of cells in vitro or ex vivo, for the purpose of implanting them into a patient.<br>
In this respect, another subject of the present invention relates to any pharmaceutical<br>
composition comprising at least one DNA molecule as defined above. This molecule may or<br>
may not be associated therein with a chemical and/or biochemical transfection vector. This<br>
may in particular involve cations (calcium phosphate, DEAE-dextran, etc.) or liposomes. The<br>
associated synthetic vectors may be cationic polymers or lipids. Examples of such vectors which may be mentioned are DOGS (Transfectam™) ,or DOTMA (lipofectin™).<br>
The pharmaceutical compositions according to the invention may .be formulated for the purpose of topical, oral, parenteral, intranasal, intravenous, intramuscular, subcutaneous, intraocular, or transdermal administrations. The claimed plasmid is preferably used in an injectable form or in application. It may be mixed with any vehicle which is pharmaceutically acceptable for an injectable formulation, in particular for a direct injection to the site to be treated. This may involve, in particular, sterile, isotonic solutions or dry compositions, in particular freeze-dried compositions, which, by addition, depending on the case, of sterilized water or of physiological saline, allow injectable solutions to be made up. This may in particular involve Tris or PBS buffers diluted in glucose or in sodium chloride. A direct injection into the affected region of the patient is advantageous since it allows the therapeutic effect to be concentrated at the level of the affected tissues. The doses used may be adapted as a function of various parameters, and in particular as a function of the gene, the vector, the mode of administration used, the pathology concerned or the desired duration of the treatment.<br>
The DNA molecules of the invention may contain one or more genes of interest, that is to say one or more nucleic acids (synthetic or semi-synthetic DNA, gDNA, cDNA, etc.) whose transcription and, possibly, whose translation in the target cell generate products of therapeutic, vaccinal, agronomic or veterinary interest<br>
Among the genes of therapeutic interest which may be mentioned more particularly<br>
are genes coding for enzymes, blood derivatives, hormones and lymphokines: interleukins,<br>
interferons, TNF, etc. (FR 92/03120), growth factors, neurotransmitters or their precursors or<br>
synthetic enzymes, and trophic factors (BDNF, CNTF, NGF, IGF, GMF, aFGF, bFGF, NTS,<br>
NTS, VEGF-B, VEGF-C etc.; apolipoproteins: ApoAI, ApoATV, ApoE, etc. (FR 93/05125),<br>
dystrophin or a minidyslrophin (FR 91/11947), tumour-suppressing genes: p53, Rb, RaplA,<br>
DCC, k-rev, etc. (FR 93/04745), genes coding for factors involved in coagulation: factors "VTI,<br>
VUI, DC, etc., suicide genes: thymidine kinase, cytosine deaminase, etc.; or alternatively all or<br>
part  of a  natural  or  artificial  nnmunoglobuUn  (Fab,   ScFv,   etc.),   an  RNA  ligand<br>
(WO 91/19813), etc. The therapeutic gene may also be an antisense sequence or gene, whose<br>
expression in the target cell makes it possible to control the expression of genes or the<br>
transcription of cellular mRNAs. Such sequences may, for example, be transcribed, in the<br>
target cell, into RNAs which are complementary to cellular mRNAs and thus block their<br>
translation into protein, according to the technique described in patent EP 140,308. A insert of<br>
interest that may be carried by the pCOR of the invention is a RNAi, whose is capable of<br>
interfering with the translation of a target gene (Wilson et al., Curr Opin Mol Ther. 2003<br>
Aug;5(4):389-96) and thereby regulating the expression of such gene.<br>
The gene of interest may also be a vaccinating gene, that is to say a gene coding for an antigenic peptide, capable of generating an immune response in man or animals, for the purpose of producing vaccines. These antigenic peptides may in particular be specific antigenic peptides of Epstein-Barr virus, HIV virus, hepatitis B virus (EP 185,573), or pseudorabies virus, or alternatively specific antigenic peptides of rumours (EP 259,212).<br>
Generally, in the DNA molecules of the invention, the gene of therapeutic, vaccinal, agronomic or veterinary interest also contains a promoter region for functional transcription in the target organism or cell, as well as a region located at the 3' end which specifies a transcription termination signal and a polyadenylation site. As regards the promoter region, it may be a promoter region naturally responsible for expression of the gene under consideration when this region is capable of functioning in the cell or the organism concerned. The promoter<br>
regions may also be regions of different origin (responsible for the expression of other proteins) or even of synthetic origin. In particular, they may be promoter sequences from eukaryotic or viral genes. For example, they may be promoter sequences obtained from the genome of the target cell. Among the eukaryotic promoters which may be used are any promoters or derived sequence which stimulates or suppresses the transcription of a gene in a specific or non-specific, inducible or non-inducible, strong or weak manner. The eukaryotic promoters may in particular be ubiquitous promoters (promoters of the genes for HPRT, PGK, a-actin, tubulin, etc.), intermediate filament promoters (promoters of the genes for GFAP, desmin, vimentin, neurofilaments, keratin, etc.), therapeutic gene promoters (for example the promoters of the genes for MDR, CFTR, factor VIH, ApoAI, etc.) tissue-specific promoters (promoters of the genes for pyruvate kinase, villin,, intestinal fatty acid-binding protein,   a-actin of smooth muscle, etc.) or alternatively promoters which respond to a stimulus (steroid hormone receptor, retinoic acid receptor, etc.). Similarly, they may be promoter sequences obtained from the genome of a virus, such as, for example, the promoters of the adenovirus El A and MLP genes, the CMV early promoter or alternatively the LTR promoter of RSV, etc. In addition, these promoter regions may be modified by addition of activating or regulatory sequences or sequences which allow tissue-specific expression or expression which is predominantly tissue-specific.<br>
Moreover, the gene of interest may also contain a signal sequence which directs the synthesized product into the secretory pathways of the target cell. This signal sequence may be the natural signal sequence of the synthesized product, but it may also be any other functional signal sequence or an artificial signal sequence. Preferred signal sequence used according to the present invention is the secretion signal peptide of human interferon as described Taniguchi et al. (Gene, 1980,233 (4763):541-5)<br>
Depending on the gene of interest, the DNA molecules of the invention may be used for the treatment or prevention of several pathologies, including genetic diseases (dystrophy, cystic fibrosis, etc.), neurodegenerative diseases (Alzheimer's disease, Parkinson's disease, ALS, etc.), cancers, pathologies associated with coagulation disorders or with dyslipoproteinaemias, pathologies associated with viral infections (hepatitis, AIDS, etc.), or in the agronomic and veterinary fields, etc.<br>
According a preferred embodiment, the DNA molecules of the present invention are used for treating critical limb ischemia pathologies such as for example peripheral arterial occlusive disease and intermittent claudication.<br>
Moreover, the present invention also relates to the use of conditional replication DNA molecules for the production of recombinant proteins. Bacteria can be used to produce proteins of various origins, eukaryotic or prokaryotic. Among the bacteria, E. coli constitutes the organism of choice for expressing heterologous genes on account of its ease of manipulation, the large number of expression systems available and the large amounts of proteins which can be obtained. It is understood that the system of the invention can be used in other organisms, the tropism being determined by the nature of the origin of replication, as indicated above. For this use, the nucleic acid sequence of interest comprises a coding region under the control of expression signals that are appropriate for the host chosen, in particular a prokaryotic host. These may be, for example, Plac, Ptrp, PT7, Ptrc, Ptac, PL, PBAD or PR promoters, the Shine-Dalgamo sequence, etc. (this set constitutes the expression cassette). The nucleic acid sequence of interest can. be any sequence coding for a protein which is of value in the fields of pharmacy, agri-foods, chemistry or agrochernistry. This may be a structural gene, a complementary DNA sequence, a synthetic or semi-synthetic sequence, etc.<br>
The expression cassette can be introduced onto the conditional replication vector which is the subject of the invention, thus constituting a conditional replication vector which allows.the expression of proteins of interest in E. coli. This vector has several advantages: no use of antibiotic to select it in the bacterium (reduced cost, no need for a study regarding the presence of antibiotic or of potentially toxic derived products in the finished product), virtually no probability of dissemination of the plasmid in nature (conditional origin of replication), possible fermentation hi entirely defined medium. The examples given show the advantageous properties of these conditional vectors for the production of recombinant proteins.<br>
As described above, the DNA molecule according to the present invention comprises an origin of replication ORIy derived from R6K wherein the pir gene is removed and is introduced, into the genome of a specific host cell that is used for the production of the DNA molecules at large scale. There is always a need to produce increasing quantities of plasmid for clinical trials and/or for use in DNA- based gene therapy. Production host cells have been engineered to carry the pir gene containing at least one mutation, such as the mutation pirl 16 and/or pir42. Use of such mutated host strain results in an increase of the plasmids copy<br>
number and thus significantly raises the yield of production. Also, conformation of the plasmids so produced is very satisfying.<br>
According to a particular aspect, a novel fluorescence-based method of screening for copy-up mutant is provided This fluorescence-based screening method is far superior to the classical method of screening based on the level of resistance to antibiotic in the bacteria, which may not be used when the basal copy number of plasmid is already very high such as the one obtained using the mutant pirl 16, e.g., around 400 copies of plasmid per cell. The fluorescence-based method of screening according to the present invention preferably uses the cobA gene as red fluorescence reporter gene of copy-up number. The cobA gene which is a gene from Pseudomonas denitiificans (Crouzet et al, J. Bacteriol. 1999, 172: 5968-79) encodes uro HI methyltransferase, an enzyme of the vitamin B12 pathway, which adds two methyl groups to urogen m molecule. Wildt et al. (Nature Biotechnology, voL 17, 1999, ppl!75) has described the use of cobA as a fluorescent transcriptional reporter gene for E. coli, yeast and mammalian cells. For example, such fluorescent reporter gene was used for the selection of recombinant plasmids containing E. coli strains which accumulate fluorescent porhyrinoid compounds due to overexpression of the cobA gene encoding the uroIH methyltransferase. When illuminated with UV light, the cells fluoresced with a bright red color (Biotechniques, 1995, vol 19, no. 7, p. 760).<br>
The Applicant has surprisingly found a close correlation between the copy number of plasmid carrying the cobA gene and the level of fluorescence from pink to red. The fluorescence-based method of screening of copy-up mutants according to the present invention is thus useful for screening various mutants which can then be evaluated in the genome of the production host cell, such as E. coli, or mutants of any genes such as in the pir gene, which are inserted in the genome of the production host cell or carried in a plasmid.<br>
In addition to the correlation with the copy number of plasmids, the fluorescence-based method of screening of the present invention is easily and rapidly conducted as it is only requires plating and culturing the transformed host cells overnight and exposing to UV lights, to reveal intensity of the fluorescence produced, thereby deducing directly the number of copy of plasmids in the host cell.<br>
Thus, the present invention provides for a method for detecting a plasmid copy-up mutation comprising:<br>
(a)       introducing at least one mutation into a target sequence;<br>
(b)	transforming the mutated target sequence into a host cell<br>
comprising a plasmid, wherein the plasmid comprises a<br>
nucleotide sequence encoding uroEI methyltransferase and the<br>
copy number of the plasmid is effected by the target sequence;<br>
growing the host cell under conditions wherein the nucleotide<br>
sequence is expressed to produce a culture of host cells;<br>
exposing the culture of host cells to UV light; and<br>
(e)	detecting fluorescence produced by the culture of host cells.<br>
According to the present invention, the method further comprises comparing the<br>
fluorescence detected in (e) with fluorescence produced by a culture of host cells comprising an non-mutated target sequence.<br>
Preferably, the uroin methyltransferase gene is coded by the cobA gene from Pseudomonasdenitrificans.<br>
The mutation may be present in a plasmid comprising a heterologous pir gene comprising at least one mutation. The plasmid may comprise at least one mutation in the pir other regions such as in the copy control region and/or in the DNA binding domain, and/or in the leucine-zipper motif and/or in another region of the pir gene. Also, the plasmid may comprise at least one mutation in the heterologous pir gene copy number control region and the leucine zipper-like motif. The plasmid may further comprise a mutation in the pir gene DNA binding region. Furthermore, the plasmid may comprise one or more mutations in the same or a different region of the./&gt;zr gene coding for the copy control region and/or the DNA binding region, and/or the leucine zipper-like motif, or other region of the protein E.<br>
Within limitation, the prokaryotic recombinant host cell according to the present invention comprises the pirl 16 mutation and a second mutation in the DNA "binding region such aspir292,pir!30, or pirl 17 (Fig. 26).<br>
Such mutated production host strain may be advantageously produced using an universal plasmid tool such as the minicircle. The minicircle technology is described inter alia in US patents 6,143,530 and 6,492,164 of the Applicant or in PCT application WO 96/26270.<br>
Mrnicircles are recombinant DNA molecules that do not contain any origin of replication, and thus represent excellent suicide vector for gene replacement of the genome<br>
of any microorganisms. In particular, the gene or genes of interest are flanked by the two sequences permitting site-specific recombination, positioned in the direct orientation in the minicircle. The position in the direct orientation indicates that the two sequences follow the same 5'-3' polarity in the recombinant DNA minicircle. The minicircle genetic constructions are generally circular double-stranded DNA molecules devoid of origin of replication, but may also be in linear form and contain the gene or genes of interest flanked by the two sequences permitting site-specific recombination, positioned in the direct orientation. According to this particular embodiment of the invention, the minicircle may be used to transform any competent microorganisms for the purpose of the gene replacement within the genome thereof (Figure 31).<br>
The minicircle for gene replacement  is  generated  from a parent plasmid comprising at least:<br>
an origin of replication and, a selection marker gene,<br>
two sequences permitting site-specific recombination, positioned in the<br>
direct orientation, and,<br>
c)	placed between said sequences b), one or more genes of interest.<br>
The specific recombination system present in the genetic constructions can be of different origins. In particular, the specific sequences and the recombinases used can belong to different structural classes, and in particular to the integrase family of bacteriophage X or to the resolvase family of the transposon Tn5. Among recombinases belonging to the integrase family of bacteriophage X, there may be mentioned, in particular, the integrase of the phages lambda (Landy et al, Science 197: 1147, 1977), P22 and O80 (Leong et al., J. Biol. Chem. 260: 4468, 1985), HP1 of Haemophilus influenza (Hauser et al, J. Biol. Chem. 267 6859, 1992), the Cre integrase of phage PI, the integrase of the plasmid pSAM2 (EP 350,341) or alternatively the FLP recombinase of the 2p, plasmid. The minicircles are thus prepared by recombination by means of a site-specific system of the integrase family of bacteriophage "k, the DNA molecules according to the invention generally comprise, in addition, a sequence resulting from the recombination between two att attachment sequences of the corresponding bacteriophage or plasmid.<br>
Among recombinases belonging to the family of the transposon Tnf, there may be mentioned, in particular, the resolvase of the transposon Tni or of the transposons Tn2J and TnJ22 (Stark et al, Trends Genet, 8, 432-439, 1992); the Gin invertase of<br>
bacteriophage mu, or, alternatively, the resolvase of plasmids, such as that of the par fragment of BP4 (Albert et al, Mol. Microbiol. 12: 131, 1994). When the minicircles are prepared by recombination by means of a site-specific system of the family of the transposon Tn3, they generally comprise, in addition to the gene of interest that is aimed to be inserted in a microorganism genome, a sequence resulting from the recombination between two recognition sequences of the resolvase of the transposon in question. Sequences permitting site-specific recombination may also be derived from the loxP region of phage PI, which is composed essentially of two repeat sequences capable of recombining specifically with one another in the presence of a protein, designated Cre (Stemberg et al, J. Mol. Biol. 150: 467,1971). The plasmid used to produce the minicircle thus comprises (a) a bacterial origin of replication and, a selection marker gene; (b) the repeat sequences of bacteriophage PI QoxP region); and (c), placed between said sequences (b), one or more genes of interest that one's wish to insert in a microorganism genome.<br>
Minicircles may comprise sequences permitting site-specific recombination are<br>
derived from a bacteriophage, such as attachment sequences (attP and attB sequences) of a<br>
bacteriophage or sequences derived from such attachment sequences. These sequences are<br>
capable of recombining specifically with one another in the presence of a recombinase<br>
referred to as an integrase with or without an excisionase. The term "sequences derived<br>
from such attachment sequences" includes the sequences obtained by modification(s) of the<br>
attachment  sequences  of the bacteriophages  that retain  the  capacity to recombine<br>
specifically in the presence of the appropriate recombinase. Thus, such sequences can be<br>
reduced fragments of these sequences or, alternatively, fragments extended by the addition<br>
of other sequences (restriction sites, and the like).  They can also be variants obtained by<br>
mutation(s), in particular by point mutation(s).   The terms attP and attB sequences of a<br>
bacteriophage or of a plasmid denote, according to the invention, the sequences of the<br>
recombination system specific to said bacteriophage or plasmid, that is to say the attP<br>
sequence present in said phage or plasmid and the corresponding chromosomal attB<br>
sequence. Attachment sequences are well known in the art, and include inter alia the<br>
attachment sequences of the phages \ P22, D80, PI, and HP1 of Haemophilus influeivzae<br>
or, alternatively, of plasmid pSAM2 or the 2|i plasmid.<br>
The minicircles are easly produced from the parent plasmid described above. The nethod for the production of the minicircle consists in bringing into contact culture of cells hat are transformed with the parent plasmid with the integrase with or without the jxcisionase, so as to induce the site-specific recombination. The culture and the integrase tvith or without the excisionase are brought into contact either by transfection or infection mth a plasmid or a phage containing the gene for said integrase and when applicable the gene for the excisionase. Alternatively, for example, the expression of genes coding for 3aid integrase and when applicable the excisionase, present in the host cell, are induced. As mentioned below, these genes may be present in the host cell in integrated form in the genome, on a replicative plasmid, or, alternatively, on the plasmid of the invention, in the<br>
non-therapeutic portion.<br>
To permit the production of the minicircles according to the invention by site-specific recombination in vivo, the integrase with/without the excisionase used are introduced into, or induced in, cells or the culture medium at a particular instant. For this purpose, different methods may be used. According to a first method, a host ceU is used containing, for example, the recombinase gene, Le., the integrase gene with or without the excisionase gene, in a form permitting its regulated expression. The integrase gene with or without the excisionase gene may, for example, be introduced under the control of a promoter, or of a system of inducible promoters, or, alternatively, in a temperature-sensitive system.<br>
• In particular, the integrase gene may be present in a temperature-sensitive phage, latent during the growth phase, and induced at a suitable temperature (for example,<br>
lysogenic phage X Xis" c/857).<br>
Alternatively, the gene may be under the control of a regulated promoter, for example, the placUVS promoter, the host cell is designated E.coH G6191.<br>
Preferably, the integrase with or without the excisionase gene may be under the control of a regulated promoter, for example the pbad promoter of the araBAD (arabinose) operon, which is regulated by arabinose (Guzman et al., J. Bacteriol, 1995, 4121-4130; US5,028,530). Particularly, use of pbad promoter allows sufficient expression of excisionase and integrase in presence of arabinose, as the inducing agent, and thus more than 90% of recombination of the plasmids which are present in high copies number in the bacteria, whereas in absence of arabinose, the promoter is tightly inhibited. The cassette for<br>
expression of the integrase with/without excisionase may be carried by a plasmid, a phage, or even by the plasmid of the invention in the non-therapeutic region. It may be integrated in the genome of the host cell or maintained in replicative form. Such host cells are in particular E.coli G6264 and E.coli G6289. According to another method, the cassette for expression of the gene(s) is carried by a plasmid or a phage used to transfect or infect the cell culture after the growth phase. In this case, it is not necessary for the gene to be in a form permitting its regulated expression. In particular, any constitutive promoter may be used. The DNA may also be brought into contact with the integrase and when applicable the excisionase in vitro, on a plasmid preparation, by direct incubation with the protein.<br>
The minicircle so produced thus comprises an expression cassette containing one or more genes of interest to be inserted in the targeted microorganism, lacks an origin of replication and comprises a sequence attR resulting from site-specific recombination between an attB and an attP sequence, or a sequence attL resulting from site-specific recombination between an attB and an attP sequence. The minicircle may thus be used as universal suicide vector for gene replacement in any microorganisms, hi effect, the minicircle carrying a gene for replacement flanked by homologous sequences and a antibiotic resistance gene will easily integrate in a targeted site of the genome of any microorganism by homologous recombination as represented in Figure 31. A second event .of excision which may be triggered by a second selection pressure may then efficiently select the microorganisms only carrying the new inserted gene within their genome.<br>
The present invention thus also relates to a method of gene engineering of a microorganism. This novel method may used to engineer any microorganism regardless of their origin. In effect, the minicircle does not contain any origin of replication, and thus can be used universally for gene replacement in any types of microorganisms. This method represents an advantageous alternative to the use of the bacteriophage M13 for gene replacement by double homologous recombination in a micro-organism.<br>
According to a particular embodiment of the present invention, the minicircle comprises a first selectable marker such as an antibiotic resistance gene, allowing selecting for the first recombination event. Preferred second selectable marker is the gene HI or the functional deleted gene IH'. The gene HI or its functional variant is capable of conferring sensibility to deoxycholate as described in Boecke et al. (Mol. Gen. Genet., 186, 185-92, 1982) and thus allows for counter-selecting the second event of recombination (Fig. 31).<br>
The method thus consists in introducing the minicircle into the microorganism by any transformation method well known in the art, and preferably by electroporation, selecting the event of integration of the minicircle in a culture supplemented with an antibiotic or under another pression of selection, and selecting a second event of excision by treating with deoxycholate or another appropriate pression of selection.<br>
The present invention will be described more fully with the aid of the examples which follow, which should be considered as non-limiting illustrations.<br>
Brief description of the drawings:<br>
Figure 1: Functional organization of the region of R6K. involved in replication.<br>
Figure 2: Organization of the functional domains of the II protein of the plasmid R6k.<br>
Figure 3: Representation of the protocol for introducing thepz'r gene into the genome<br>
ofE. co/zXACl.<br>
Figure 4: Construction scheme for vectors pXL2666, 2730 and 2754.<br>
Figure 5: Construction of pXL2774.<br>
Figure 6: Growth and production kinetics in a 2L fermenter.<br>
Figure 7: Growth and production kinetics in an 800L fermenter.<br>
Figure 8: Construction of pXL3056.<br>
Figure 9: Visualization of the aFGF protein produced by E. coli XAC-lpirll6 (pXL3056+PT7po!23) after induction. The denatured total cell extracts are deposited on 12.5%-SDS polyacrylamide gel. M: molecular mass marker (Biorad, Low range). Each band is identified by an arrow and a figure which indicates its mass in kDaltons. 1: XAC-lpirJ16 (pXL3056+pUC4K) not induced; 2: XAC-lpirll6 (pXL3056+pUC4K) induced at 42°C; 3: XAC-lpirll6 (pXL3056+PT7po!23) clone 1, not induced; 4: XAC-lpirll6 (pXL3056+PT7po!23) clone 1, induced at 42°C; 5: XAC-lpirll6 (pXL3056+PT7po!23) clone 2, not induced; 6: ~XAC-\pirll6 (pXL3056+PT7po!23) clone 2, induced at 42°C; tl: 1 jig of purified aFGF; t4:4 u-g of purified aFGF.<br>
Figure 10: Schematic representations for vectors pXL3029, pXL3030, and pXL3179<br>
orNVlFGF.<br>
Figure 11: Schematic representation of the functional domains of R6K ti initiator<br>
proteins.<br>
Figure 12: Nucleotide and amino acid sequences of the_pzr gene comprising ih&amp;pirl26<br>
sndpir42 mutations.<br>
Figure 13: Construction ofpir216pir42 suicide vector for homologous<br>
recombination.<br>
Figure 14: Schematic representation of the PCR products obtained when amplifying the region uidA::pirl!6 +/-pir42.<br>
Figure 15: Agarose gel electrophoresis showing the topology of pCOR plasmid pXL3179 produced in TEXl or TEXlpW2.<br>
Figure  16:    Schematic representation of the pXL3749 suicide plasmid carrying pirl!6cop21 gene.<br>
Figure 17: Agarose gel electrophoresis showing the plasmid copy number of pXL2979 when produced in E. coli host cell TEXlcop21 (lines 1-4), in E. coli host cell XAClpir (Lines 5-8), in E. coli TEXl (lines 9-12).<br>
Figure 18: Representation of the cloning strategy for the construction of the recA-suicide vector.<br>
Figure 19: Schematic representation of the PCR products obtained when amplifying regions of £ coli TEX2 strain.<br>
Figure 20: Agarose gel electrophoresis showing the topology of pCOR pXL3179 produced in E. coli TEX2pir42 (line B), in £ coli TEXlpir42 (line C), in K coli TEXl (lineD).<br>
.Figure 21: Analysis of plasmid pXL3179 produced by fermentation in E. coli TEX2pir42.<br>
Figure 22: Fluorescence-based assay showing that fluorescence increases with plasmid copy number.<br>
Figure 23: Diagram of plasmids screened in the fluorescence-based assay. Figure 24: Diagram of plasmid pXL3 830.<br>
Figure 25: Agar plate demonstrating fluorescence-based screening for copy-up mutants generated by random mutagenesis.<br>
Figure 26: Evaluation of copy-up mutants identified by the fluorescence-based screening method.<br>
Figure 27: Diagram of the strategy for evaluating pirll 6 mutants inserted into the bacterial genome.<br>
Figure 28: Evaluation of pXL3179 copy number in differentpirl 16* mutant E. coli<br>
strains.<br>
Figure 29: Construction of a plasmid used to generate mim'circle vectors for homologous recombination in E. coli.<br>
Figure 30: Construction of aminicircle vector used to generate pirl 16* mutant E.<br>
coli strains.<br>
Figure 31: Diagram of gene replacement by homologous recombination using a<br>
mioicircle vector.<br>
Figure 32: Demonstration of double recombinant clones grown on medium containing sodium deoxycholate.<br>
Figures 33A and B: The results of control PCR on double recombinants.<br>
I - MATERIALS AND METHODS<br>
A) Materials<br>
1^ Culture media<br>
Complete LB, 2XTY and SOC media and minimal M9 medium (Maniatis et al, 1989) were used. Agar media were obtained by addition of 15 g of Difco agar. Furthermore, if necessary, these media were supplemented with antibiotics (ampicilKn or kanamycin) at respective concentrations of 100mg/l and 50mg/l. The chromogenic substrates X-Gal and X-Gluc were used at a concentration of 40 mg/1.<br>
2) E. coli strains, plasmids and bacteriophages<br>
The E. coli strains, plasmids and bacteriophages used are respectively identified in the examples below. BY Methods<br>
I) Manipulation of the DNA<br>
The isolation of bacterial DNA (plasmid and genomic) and phage DNA (replicative form of M13), digestion with restriction endonucleases, ligation of the DNA fragments, agarose gel electrophoresis (in TBE buffer) and other standard techniques were carried out according to the manufacturers' recommendations, for the use of enzymes, or in accordance with the procedures described in "Molecular Cloning: a Laboratory Manual" (Maniatis et ai, 1989).<br>
The DNA size markers used during the electrophoreses are as follows: 1 kb ladder (BRL) for the linear fragments and the supercoiled DNA marker (Stratagene) for the undigested plasmids.<br>
Sequencing was carried out according to the Sanger technique (Sanger et al, 1977) adapted to the automated method using fluorescent dideoxynucleotides and Taq DNA polymerase (PRJSM Ready Reaction DyeDideoxy Terminator Cycle Sequencing Kit, Applied<br>
Biosystems).<br>
The oligodeoxynucleotides used (designated by "seq+no.", see below) were synthesized on the "Applied Biosystems 394 DNA/RNA Synthesizer" by the phosphoramidite method, using a-cyanoethyl protecting groups (Sinha et al., 1984). After synthesis, the protecting groups are removed by treatment with ammonia. Two precipitations with butanol allow the oligonucleotide to be purified and concentrated (Sawadogo et al., 1991).<br>
Sequences of the oligpmicleotides used for the PCR amplification:<br>
The PCR reactions (Saiki et al., 1985) were performed under the following conditions, in a total volume of 100 jil. The reaction mixture comprises 150 ng of genomic DNA from the strain to be studied, 1 |J.g of each of the two oligonucleotide primers (24-mer), 10 nl of 10XPCR buffer, the composition of which is as follows "500 mM KC1, 0.1 % gelatin, 20 mM MgCl2j lOOmM Tris-HCl pH 7.5", and 2.5 units of Taq DNA polymerase (Amplitaq Perkin-Elmer). The PCR conditions, on the Perkin-Elmer Cetus DNA Thermal Cycler machine are as follows: 7- mm at 91°C, 30 successive cycles of denaturation (1 min at 91°C), hybridization (2 min at 42°C) and elongation (3 min at 72QC), and finaUy 5 min at 72°C. The products thus obtained, which are or are not digested with a restriction enzyme, are analysed by agarose gel electrophoresis.<br>
Analysis of the various plasmid species by DNA topoisomerases was performed according to the following procedure: the enzymes, purified in the laboratory, are incubated for Ihour at 37°C. The reaction mixtures (total volume: 40 jil) have the following<br>
composition: 150 ng of plasmid, 300 ng of DNA topoisomerase I or 150 ng of E. coli DNA<br>
gyrase, or 160 ng of S. aureus DNA topoisomerase IV and 20 pi of buffer specific for each<br>
enzyme. The composition of these buffers is indicated below:<br>
for DNA topoisomerase I:<br>
50 mM Tris-HCl pH 7.7, 40 mM Kd, 1 mM DTT, 100 p.g/ml BSA, 3 mM MgCl2, 1 mM<br>
EDTA;<br>
for DNA topoisomerase IV:<br>
60 mM Tris-HCl pH 7.7, 6mM MgCl2, 10 mM DTT, 100 ug/ml BSA, 1.5 mM ATP, 350 mM potassium glutamate; for DNA gyrase: 50 mM Tris-HCl pH 7.7, 5 mM MgCl2, 1.5 mM ATP, 5 mM DTT, 100 ug/ml BSA, 20 mM<br>
K Transformation of E. coli<br>
This was performed routinely according to the TSB (Transformation and Storage Buffer) method described by Chung and Miller (1988). For a strain such as TGI (Gibson et al, 1984), the transformation efficiency obtained is about 105-106 transformants per p.g of pUC4K (Vieira and Messing; 1982). When a higher transformation efficiency was necessary, the bacteria were transformed by electroporation according to the procedure recommended by the electroporator manufacturer (Biorad). This method makes it possible to achieve efficiencies of from 108 to 1010 transformants per p,g of pUC4EL<br>
3) Cellular transfection mediated by a cationic lipofectant<br>
The cells used are NIH 3T3 mouse fibroblasts seeded the day before into 24-well plates, at a density of 50,000 cells per well. The culture medium used is DMEM medium, containing 4.5 g/1 of glucose and supplemented with 10 % fetal calf serum and 1 % of solutions of 200 mM glutamine and antibiotics (5.103p/ml streptomycin, 5.103 jig/ml penicillin) (Gibco). The plasmid DNA (1 ug in 25 jjl of 9 %o NaCl) is mixed, on a volume-for-volume basis, with a suspension of lipofectant. Four "lipofectant charges/DNA charges" ratios are tested: 0, 3, 6 and 9. These ratios are calculated by considering that 1 jig of plasmid DNA carries 3.1 nmol of negative charges and that the lipofectant contains 3 positive charges per<br>
molecule. After a contact time of 10 minutes to allow formation of the DNA/lipid complex, 50 ul of DNA-lipofectant mixture are introduced onto the cells in serum-free culture medium (500 ul). The cells were prerinsed twice with this same medium. Inhibition of transfection by the serum is thus avoided. After incubation (2 hours at 37°C in the C02 incubator), 10 % fetal ' calf serum is added to the medium. The cells are then reincubated for 24 hours.<br>
4) Measurement of the luciferase activity of eukarvotic cells<br>
This is carried out 24 hours after the transfection. Luciferase catalyses the oxidation of luciferin in the presence of ATP, Mg2+ and (}&gt;, with concomitant production of a photon. The total amount of light emitted, measured by a luminometer, is proportional to the luciferase activity of the sample. The reagents used are supplied by Promega (luciferase assay system) and used according to the recommended procedure. After lysis of the cells, the insoluble fraction from each extract is eliminated by centrifugation. The assay is carried out on 5 ul of supernatant, which may or may not be diluted in the cell lysis buffer.<br>
51 Measurement of the protein concentration in the cell extracts This is carried out according to the BCA method (Pierce) using bicinchoninic acid (Wiechelman etal, 1988). The standard BSA range is prepared in the lysis buffer (c£ IH-B-4). The samples to be assayed and those of the range are pretreated, on a volume-for-volume basis, with 0.1 M iodoacetamide/0.1 M Tris buffer, pH 8.2, for 1 hour at 37°C. This treatment makes it possible to prevent interference, during the assay, of the reducing agent (DTT) present in the lysis buffer. The assay result is read at 562 nm.<br>
EXAMPLE 1: Construction of XAC-1 pir and ptr!16 host strains by homologous recombination<br>
. The strain used was the E. coli strain XAC-1 (Normanly et al, 1980). The argE gene of this strain advantageously includes a mutation of glutamine-53 (CAG) into the amber codon (TAG) (Meinnel et al, 1992). The argE gene belongs to the argECBH divergent operon and codes for an arginine biosynthesis enzyme, N-acetylornithinase. XAC-1 cannot therefore synthesize arginine and, consequently, grow in minimal medium. This auxotrophy will be relieved if the strain harbors a plasmid which allows the expression of a suppressor<br>
tRNA. It will thus be possible, by culturing in minimal medium, to select bacteria that cany such a plasmid. In order to allow the replication therein of plasmids derived from R6K, it was necessary to introduce, by homologous recombination, the pir gene into the genome of XAC-1. The pir gene (wild-type or mutated) is introduced at the uidA locus by exchange between the wild-type uidA gene and a copy interrupted by the/Kr (arpirllty gene. The uidA gene codes for fj-glucuronidase, the enzyme for hydrolysis of (3-glucuronides. This gene may be inactivated without any problem since it is not essential for growth in standard synthetic media, in which j3-glucuronides are not used. Furthermore, the (3-glucuronidase activity can be monitored by means of a chromogenic substrate, X-Gluc, whose hydrolysis releases a blue pigment.<br>
1) Construction of a suicide vector carrying the cassette "~Kx^-uidA::pir (oryirllG) We used a strategy involving a single bacterial host and minimizing the modifications to the genome of the strain of interest. The phage MlSmplO (Messing et Vieira; 1982) was used as a suicide vector (Blum et a/., 1989). An amber mutation in the gene H, which is essential for replication, reduces the host spectrum of this M13 to the strains, such as TGI (supE), which produce an amber suppressor tRNA; it will therefore not be able to replicate in E. coli sup+ strains, such as XAC-1.<br>
The 3.8 kb BamTH cassettes, containing the kanarnycin-resistance gene of Tn5 and _uidA::pir or pirl!6, were respectively purified from M13wm34 and 33 (Metcalf et al, 1994). They were cloned into M13mplO linearized with BamHL The recombinant clones were selected by plating on LB agar medium supplemented with kanamycin, after electroporating the ligation mixtures into TGI. The conformity of the clones obtained was shown by analysing the restriction profile and by sequencing the region corresponding to the pirl 16 mutation.<br>
2) Introduction of the pir or r&gt;irl!6 genes into the genome of E. coli XAC-1 by<br>
homologous recombination<br>
The strategy <adopted and the various events involved are presented in figure a first recombination event></adopted>
The XAC-1 strain was transformed by elect-operation with 10, 100 or 2000 ng of each KF (mplO- uidA::pir or pir 116}. One-third of each expression mixture was plated out on LB<br>
plates containing kanamycin and incubated overnight at 37°C. The mpW-_uidA::pir or pirl!6 phages cannot replicate in the strain XAC-1 (sup+). The kanamycin resistance ("Km11") marker can therefore only be maintained by integration into the genome of the bacterium via a homologous recombination with the wild-type copy of the gene uidA. The results of the electroporations of XAC-1 are presented in Table 1. The transformation efficiency obtained was 4.109 transformants per jig of pUC4K.<br><br>
(Table Remove)<br>
Under the test conditions, the number of integrants increased in a non-linear manner with the amount of DNA. Given the transformation efficiency and the size of the KFs 1 (11.7kbp), it was possible to have an approximate idea of the level of recombination. By considering the point at 100 ng, a recombination frequency of about 10"6 was obtained, b) Second recombination event<br>
The second recombination event will then be selected by the resistance of the strains to deoxycholate ("Doc*")-<br>
To do this, five integrants of each construct were cultured in 2XTY medium supplemented with 0.2 % sodium deoxycholate. Two distinct populations appeared. Certain clones gave quite visible cloudiness after about 8 hours at 37°C (two clones for the pir construction and three for the pir 116 construction). The other clones gave a dense culture only after one night at 37°C. They were virtually all sensitive to kanamycin ("Kms"), as expected. For each of the electroporants studied, 50 Kms descendants were streaked onto LB medium supplemented with X-Gluc. After 48 hours at 37°C, the UidA"1" clones were pale blue whereas those which had undergone an allele replacement (case No. 1, Figure 3) remained white on this medium (UidA"). Table 2 summarizes the phenotypic analysis of the double recombinants obtained. From 18 to 30% of the double recombinants underwent an allele replacement.<br>
(Table Remove)<br>
3") Checking the Pir+ character nature of the strains obtained by recombination To ensure the PiH- character of the strains obtained by double recombination, we transformed three clones of each construct with pBWSO (Metcalf et al, 1994). The fact that transformants were obtained for all the test strains made it possible to show the functionality of the^zr an.d.pirl!6 genes, which were integrated into the genome of XAC-l. Under the same conditions, no transformant was obtained with the parental strain XAC-l. We continued to study two XAC-lpir clones (B and C) and two XAC- \pirll6 clones (E and D). 4) Checking. bvPCR amplification, of the strains obtained by recombination To confirm the allele replacement, we checked the genomic regions on either side of the uidA locus by PCR amplification. Each pair of oligonucleotides consisted of an oligonueleotide corresponding to an internal region of pir and a second oligonucleotide corresponding to a region, close to chromosomal uidA, but not within the fragment which served for the recombination. The sequence of the latter oh'gonucleotide was determined by means of the ECOUIDAA sequence jfrom Genbank (access number: M14641). We were thus able to verify the exact location of the pir gene in the bacterial genome. The nature of the amplified fragments, whose size is in accordance with that which might be expected, was confirmed by digestion with Mlul.<br>
EXAMPLE 2: Construction of plasmid vectors derived from R6K carrying the selection marker suy Phe<br>
Vectors were constructed containing ori  y from R6K and the kanamycin-resistance gene (pXL2666). The observation of pXL2666 multimers in the strain BW19610 (pirl!6) 5<br>
(Metcalf et al, 1993) led us to study the effect of the cer fragment from ColEl on this phenomenon. We then introduced the expression cassette of the phenylalanine suppressor tRNA (sup Phe) onto the vector ori y -KmR-cer (pXL2730). This vector, pXL2760, serves as a basis for the construction of vectors which can be used in gene therapy.<br>
1) Construction and analysis of vectors containing ori y from R6K and the kanamycin resistance gene<br>
a) Constructs<br>
In the first plasmid constructed, pXL2666, the kanamycin resistance gene originated<br>
from pUC4K (Vieira and Messing; 1982) and the origin of replication, contained hi a 417 bp<br>
EcdRI-BamHl fragment, originated from the suicide vector pUT-T7pol (Herrero et al., 1990)<br>
(Figure 4). The transfer of pXL2666 into the strains BW19094 and 19610 (Metcalf etal.,<br>
1994) made it possible to show that the amount of plasmid is indeed increased in &amp;pirJJ6<br>
strain, when compared with the same plasmid in a pir strain.   However, electrophoretic<br>
analysis of the undigested plasmids showed that this increase goes hand in hand with the<br>
appearance of a few multimeric forms. This phenomenon is quite probably associated with<br>
intermolecular recombination between the multiple copies of the plasmid.    Thus, we<br>
constructed pXL2730 by cloning the cer fragment of the natural E. coli plasmid, ColEl, which<br>
had been shown to permit, in cis, the resolution of plasmid dimers (Summers and Sherrat,<br>
1984), into pXL2666. The fragment used corresponds to a 382 bp HpaR fragment from ColEl<br>
(Leung'et al., 1985).   It contains a specific intermolecular recombination site; in order to<br>
function, it involves only host proteins including the recombinases XerC and XerD and the<br>
accessory factors ArgR and PepA (Stirling et al., 1988, 1989; Colloms et al., 1990).   To<br>
ensure that the effects observed are indeed due to the cer fragment, we also constructed the<br>
control plasmid pXL2754, in which the cer fragment has a 165 bp deletion. This deletion was<br>
shown to abolish the action of cer on the resolution of the multimers (Leung et al., 1985). The<br>
various cloning steps leading to the construction of these plasmids are presented in Figure 4.<br>
b) Quantitative and qualitative analysis of the plasmid species<br>
(i) analysis by agarose gel electrophoresis<br>
Electrophoretic   analysis   of  the   different   plasmids   constructed   allowed   the demonstration of various plasmid species, which are variable according to the strains used.<br>
The size of the undigested plasmids was evaluated relative to a supercoiled DNA marker. In the pir strain (BW19094), the plasmids pXL2666, 2754 and 2730 were almost entirely in monomeric form. The bands above each main band correspond to various slightly less supercoiled topoisomers, as confirmed by the profile observed after the action of DNA gyrase<br>
onpXL2730.<br>
In the case of the pirll6 strain (BW19610), the profiles were different: with the plasmids pXL2666 and 2754 different species were observed ranging from the monomer to multimers (2, 3 or 4 units), the major form being the dimer. After digestion with EcoKL, only the linear plasmid DNA was found; these plasmid species correspond either to plasmid multimers or to various topoisomers. However, since the size of the forms determined according to the supercoiled DNA marker was a whole product of that of the monomer plasmid, it is highly probable that they are multimers. The formation of multimers was most probably attributable to the pirl!6 mutation, although the two strains BW19094 and BW19610 are not strictly isogenic (BW19610 is recA). The profile obtained with pXL2730 was different: although multimeric forms were still visible, the major form is the monomeric form. The cer fragment can thus facilitate resolution of the plasmid multimers which we have constructed, independently of recA, in BW19610.<br>
(ii) analysis after treatment with DNA topoisomerases<br>
To disprove the theory that the forms observed in the-strains carrying ihepirlJ6 allele are specific topoisomers, each plasmid preparation was subjected to the action of DNA topoisomerases. The activities of the various enzymes under the experimental conditions were as follows: relaxing of DNA for E. coli DNA topoisomerase I, negative supercoiling of relaxed DNA for E. coli DNA gyrase, and disentanglement of interlaced DNAs and relaxation of supercoiled DNA by S. aureus DNA topoisomerase IV, The action of DNA topoisomerase IV made it possible to show that the high-molecular-weight plasmid forms did not result from the entanglement of several plasmid molecules; in this case, they would then have been converted into the monomeric species. The functionality of the enzyme was, of course, checked on a preparation of kinetoplast DNA, composed of entangled DNA molecules (not shown). The relaxation activity was also visible since species are obtained which migrate less than in the untreated controls. The action of DNA gyrase made it possible to convert the slightly relaxed topoisomers into the more supercoiled species extracted from the bacterium (monomer or dimer mainly). Furthermore, it made it possible to verify that the DNAs<br>
prepared were mainly in supercoiled form. The samples thus treated allowed the above results to be confirmed as regards the major species for each construct. DNA topoisomerase I did indeed relax DNA, but only partially. This could be due to the fact that the plasmids studied contain only a few single-stranded regions, to which this enzyme preferably binds (Roca, 1995).<br>
2) Introduction of the selection marker sup Phe into pXL2730<br>
We used the expression cassette of the synthetic suppressor tRNA gene (Phe) (Kleina et al, 1990). This introduced a phenylalanine into the growing polypeptide chain in response to a TAG codon. Furthermore, it allowed the production in XAC-1 of an ArgE protein that was sufficiently active to allow good growth in arginine-deficient medium, sup Phe was expressed constitutively on the plasmid pCT-2-F (Normanly et al., 1986) from a synthetic promoter derived from the promoter sequence, Vlpp, of the E. coli Ipp gene. Downstream of this gene, transcription was stopped by the synthetic terminator, T^c, of the E. coli operon rrnC (Normanly et al., 1986). The various cloning steps are indicated in Figure 5.<br>
The various subclonings were performed in XAC-1. The functionality of the suppressor tRNA expression cassette was thus checked by means of the a-galactosidase activity of this strain, which only exists if there is suppression of the amber codon of the gene lacZuiigain. The final step consists of the introduction of the sup Phe expression cassette into pXL2730. The results obtained with the cer fragment (B-l-6) led us to select this plasmid rather than pXL2666. We retained the kanamycin resistance gene for ease of subsequent cloning, in particular in order to have available additional screening during the final cloning (lossofKmR).<br>
EXAMPLE 3: Validation of the plasmid vector for applications in gene therapy by transfection of mouse fibroblasts<br>
1) Construction of the reporter vector pXL2774<br>
To test the validity for gene therapy of the system for producing plasmid DNA, we introduced a reporter gene, which can be used in eukaryotic cells, into pXL2760. We used the gene luc, which codes for Photinus pyralis luciferase, since the bioluminescence measurement test is very sensitive and is linear over a large range, and the background noise due to the endogenous activity of eukaryotic cells is very low. The luc gene was controlled by promoter-<br>
enhancer sequences of a human cytomegalovims early gene (CMV promoter), which allowed a high level of expression. There was an untranslated region at the 3' end of luc, originating from the virus SV40, which contained the polyadenylation signal (poly(A)+). After intermediate cloning, which allowed the number of available restriction sites to be increased,<br>
--•%.<br>
the "CMV promoter-/uc-poly(A)+" cassette was introduced into the minimal vector ori y • cer-sup Phe (pXL2760) in place of the KmR marker. The resulting plasmid has been named pXL2774. Figure 6 shows the various cloning steps. The ligation mixtures were transformed into XAC-lpirl]6'by electroporation. Incubation allowing the bacteria to express selection markers was carried out in rich medium (SOC medium); it was thus necessary to wash the cells twice with M9 medium before plating out. This made it possible to remove the residual medium, which would have resulted in culture background noise on minimal medium.<br>
The medium chosen to plate out the electroporated cells was M9 minimal medium, which makes it possible to select bacteria expressing a suppressor tRNA and thus the presence of our plasmids. The addition of X-Gal made it possible, by means of the blue colouration, to visualize the expression of the suppressor tRNA. The dishes were analysed after about 20 hours at 37°C. The absence of colonies on the DNA-free control assures us that the selection was correct, even with dense seedings. All the clones examined by restriction (8) do indeed carry a plasmid, corresponding to the expected profile. The plasmid thus constructed, pXL2774, was prepared from a clone cultured in one liter of liquid M9 medium (about 18 hours at 37°C), by a technique involving, inter alia, an ion-exchange step (Promega kit, MegaPreps). The amount of DNA collected was 2 mg.<br>
2) Analysis of the reporter vector pXL2774 transfected into mammalian cells.<br>
The capacity of pXL2774 to transfect eulcaryotic cells and to allow the' expression of luciferase was evaluated by transfection into NIH 3T3 mouse fibroblasts. The vector chosen as reference was the plasmid pXL2622 (this is the plasmid pGL2 from Promega whose SV40 promoter has been replaced by the CMV promoter), which carries the same luciferase expression cassette as pXL2774, but on a different replicon. This is a 6.2 kb ColEl derivative which carries the ampicillin-resistance gene. This plasmid serveed as a control. The luciferase activities (expressed as RLU, or relative luminescence units) are indicated in Table 3.<br>
The best results were obtained with a "lipofectant charges/DNA charges" ratio of 6; under these conditions, pXL2622 and 2774 appear to be equivalent<br>
(Table Remove)<br>
EXAMPLE 4: Verification of the suicide vector nature in E. coli of the pCOR plasmids<br>
The non-replicative nature of the pCOR-type plasmids derived from R6K. was verified by an electroporation experiment in JM109 E. coli (Yanisch-Perron et al., 1985) of plasmids pUC4K (ori ColEI-KmR, (Vieira and Messing, 1982)) and pXL2730 (ori gamma from RoTC-KrnR, see Example 2). The electroporator used was the Biorad Gene Pulser and the electrocompetent JM109 cells were prepared and used according to the manufacturer's procedure (Bacterial electro-transformation and pulse controller instruction manual, catalog number 165-2098).<br>
The electrotransformed cells were plated out on LB medium supplemented with kanamycin (50 mg/1) and incubated overnight at 37°C. The results obtained are presented below.<br>
(Table Remove)<br><br>
These results show that there was a minimum of 5 logs of difference between the efficacy of transformation of a ColEI derivative (pUC4K) and that of an R6K derivative (pXL2730) in a strain which does not express the pir gene. In a pir+ strain such as XAC-lpirll6, the electrotransformation efficacy of R6K-derived plasmids conventionally reaches or exceeds the 108 transformants/ug of plasmid.<br>
•F.YAMPT/E 5: Production of plasmid DNA by high-density culturing of the E. coli strain XAC-lDirll6 fpyi.2774^: fermentation process<br>
5.1.	Strains:<br>
Production in XAC-lpirll6E. coli (Example 1) of a mmimal plasmid, pXL2774; this plasmid comprises the following elements: on R6K-cer-tRNAamsupPhe and an expression cassette of the luc reporter gene under the control of the CMV promoter (Example 3). A high-productivity process for the production of plasmids of this type was developed.<br>
5.2.	rhilturing media and conditions:<br>
a) Growth medium:<br>
Composition of the medium denned used for the inoculum cultures (g/1): Na2HPC&gt;4 6, KH2P04 3, NaCl 0.5, MiUCl 1, NEtffcPCU 3, glucose 5, MgS04-7H20 0.24, CaCl2-2H20<br>
0.015, thiamine HC10.010<br>
Composition of the complex medium used for the fed-batch cultures (g/1): KH2PO4 8, K2HP04 6.3, Na2HP04 1.7, (NH^SO-, 0.74, NBL,C1 0.12, yeast extract 3, glucose 2,<br>
MgSO4-7H20 2.4 g/l, CaCl2-2H20 0.015, thiamine 0.010, solution of salts (Fe, Mn, Co, Zn,<br>
Mo, Cu, B, Al).<br>
Composition of the medium defined for the cultures in fed-batch medium identical to the complex medium, but the yeast extract is replaced by 2.5 g/l of NELjCl.<br>
b^l Conditions of fed-batch culturing:<br>
Studies in 2-liter fermenters (Setric France) containing 11 of medium were carried out in order to define the optimum conditions for growing and producing plasmid DNA The fermenter was inoculated with 80 ml of an inoculum culture arrived at the start of the stationary phase of growth.<br>
During the fermentation, the pH was controlled and adjusted automatically between 6.9 and 7.0 with 10% (w/v) aqueous ammonia; the temperature was maintained at 37°C; the aeration was set at 75 1/h ((1.1 vvm) at a pressure of 0.2 bar and the dissolved oxygen was adjusted to (40% of air saturation by retroaction on the stirring rate and, if necessary, by enrichment with pure oxygen.<br>
All the parameters (pH, temperature, stirring, OD, O2 and C02 in the effluent gases) were collected and calculated in line via an HP3852 interface connected to a Hewlett-Packard 9000.<br>
The base composition of the supply medium was as follows: 50% carbon source, 0.7% magnesium sulphate, 0.02% thiamine; for the complex medium, yeast extract was added to a concentration preferably of between 5 and 10%.<br>
To  adapt the  culture conditions to  800-liter fermenters, production sequences composed of two successive inoculum cultures were carried out, on a laboratory scale: inoculum I in an agitated conical flask and inoculum n in a 2-liter fermenter (batch culturing), followed by fed-batch production culturing, in a 7-liter fermenter. 5.3. Results<br>
Various culture conditions were studied in complex medium, in defined medium, and at various growth rates. In all cases, after initial batch culturing of the bacterial strain and consumption of the carbon source, the supply medium was added to the fermenter by means of a peristaltic pump coupled to a pre-programmed addition profile. This profile was deduced from previous experiments in which the supply .rate had been controlled either by the level of dissolved oxygen or by a constant growth rate.<br>
Furthermore, in order to extrapolate without difficulty the 2-liter fermentation condition to an 8001 fermenter without overoxygenation of the medium, the maximum oxygen demand at the end of the culturing was set at 2.5-3 mM/mia. For this, the growth rate of the microorganism was reduced, if necessary, by varying the supply rate of the complementary charge.<br>
As seen in Table 4, very good results were obtained both in complex medium and in<br>
defined medium, both on the laboratory scale and on the 800-liter fermenter scale;<br>
furthermore, the plasmid DNA growth and production kinetics are entirely comparable (cf.<br>
Figures 6 and 7).	<br><br>
(Table Remove)<br>
X = Biomass (weight of dry cells)<br>
From the overall results it emerges that:<br>
changing the scale of the fermenter from 2 liters to 800 liters can be carried<br>
out without any problem,<br>
the oxygen consumed is strongly correlated to the biomass produced (1.08 g<br>
CVg of biomass produced),<br>
the plasmid was stable for at least 50 generations without selection pressure,<br>
-	a high biomass, greater than 40 g of dry cells/liter, can be obtained in<br>
complex medium,<br>
-	the plasmid DNA production reaches 100'ing of supercoiled DNA/1 of<br>
medium,<br>
-	there was very good correlation between the DNA production and the<br>
biomass: the production can be estimated to (1 mg of plasmid DNA/OD unit, or alternatively<br>
(2.7 mg of plasmid DNA/g of biomass, irrespective of the duration of fermentation,<br>
-	the use of a defined medium also makes it possible to achieve a high biomass<br>
(30 g of dry cells/1) and high plasmid DNA production (100mg/l), without any loss of<br>
productivity.<br>
EXAMPLE 6; Transfer of pXL2774 into animal cells, in vitro and in vivo<br>
f. i   Tn vrfrvo transfer of pXL2774 into animal cells<br>
The capacity of the minimal plasmid pXL2774 to transfect various cell lines was tested in vitro, on cells of both human origin and murine origin. The pXL2784 plasmid was used as control. It contains the same eukaryotic expression cassette (CMV promoter-luciferase-polyA) as pXL2774, but this is a 6.4 kb ColEl derivative which comprises the gene for imparting kanamycin resistance in E. coli.<br>
The cells tested are the following:<br><br>
(Table Remove)<br>
The transfection conditions were as follows: D-l:   Inoculation of the cells at a density of 100,000 cells per 2 cm2 well (24-well plate) in DMEM medium Pulbecco's modified Eagle Medium) supplemented with 10% fetal calf serum (PCS).<br>
:	D-3: Transfection of the cells, by 10 ul of a transfection solution containing: 0.5 ug of<br>
DNA, 150 mM NaCl, 5% glucose and 3 nmol of RPR120 535 lipofectant per ug of DNA, in 250 jil of culture medium, which was or was not supplemented with 10% PCS. After incubation for 2 hours, the medium was replaced by 500 ul of DMEM medium supplemented with 10% PCS.<br>
D-4: Renewal of the culture medium<br>
D-5: Washing of the cells with PBS, followed by lysis with 100 ul of Promega lysis buffer (Promega Cell Lysis Buffer E153 A). Assay of the luciferase activity was carried out in a Lumat LB 9501 luminometer (Berthold) on 10 ul of lysate, with a 10-second duration of integration. The reactant used was that from Promega (Promega Luciferase Assay Substrate). The results, collated in the following tables 5-8, are expressed in RLU (Relative Lights Units) for 10 (j.1 of lysate (average of measurement on 4 wells). The coefficients of variation (CV) are also given.<br>
The results of transfections in the absence of serum are presented below.<br><br>
(Table Remove)<br><br>
The results of transfections in the presence of serum (10%) are presented below:<br><br><br>
These results reveal the capacity of pXL2774 to transfect effectively, in vitro, various cell types of both murine and human origin. The expression of the luc reporter gene made it<br>
possible to show that its transfection efficacy is at least as good as that of a "standard" plasmid, derived from ColEl, which carries the same expression cassette of luciferase.<br>
6.2. In vivo transfer, in animals CmiceX of pXL2774<br>
a) Model 1: Naked DNA in mouse cranial tibial muscle<br>
Naked plasmid DNA, dissolved in "5% glucose, 150 mM NaCl" was injected into the cranial tibial muscle of OF1 mice. The muscles were removed 7 days after injection, chopped up, homogenized in 750 nl of lysis buffer (Promega Cell Lysis Buffer E153A) and then centrifuged at 20,000 x g for 10 minutes.<br>
Assay of the luciferase activity was carried out on 10 pi of supernatant after addition of 50 ul of reagent (Promega Luciferase Assay Substrate). The reading was carried out on a LumatLB9501 luminometer (Berthold) with a 10-second duration of integration.<br>
The results are presented in the table below.<br><br>
(Table Remove)<br>
These results show that a conditional replication plasmid such as pXL2774 was indeed capable of transfecting mouse muscle cells in vivo and of doing so with comparable, or even superior, efficacy to that of a "standard" plasmid derived from ColEl, which carries the same expression cassette of the luciferase gene.<br>
b) Model 2: 3T3 HER2 tumour model<br>
The model is as follows:<br>
Swiss/nude adult female type mice<br>
Experimental tumours induced after injection of 107 3T3 HER2 cells<br>
subcutaneously into the flank.<br>
-	The transfection mixture was injected 7 days after injection of the cells.<br>
Solutions injected: The DNA was first dissolved in the buffer. After addition<br>
of all the products, the mixture contained, besides the DNA, NaCl (150mM) and 5% D-glucose in water or 5 mM HEPES buffer.<br>
-	Two days after the injection, the tumour tissue was removed, weighed and<br>
then chopped up and homogenized in 750 il of lysis buffer (Promega Cell Lysis Buffer<br>
E153 A). After centrifugation (20,000 x g for 10 minutes), 10 pi of supernatant was removed<br>
and luciferase activity was evaluated. This activity was determined by measuring the total<br>
light emission obtained after mixing with 50 ul of reagent (Promega Luciferase Assay<br>
Substrate) in a Lumat LB 9501 luminometer (Berthold) with a 10-second duration of<br>
integration.<br>
The resulting activity was expressed in RLU (Relative Light Units) estimated in the entire tumour lysis supernatant.<br><br>
(Table Remove)<br>
These results show that a conditional replication plasmid, such as pXL2774, was indeed capable of transfecting mouse tumour cells in vivo and of doing so with an efficacy at<br>
least comparable to that of a "standard" plasmid, derived from ColEl, which carries the same expression cassette of the luciferase gene.<br>
These various experiments demonstrate that the conditional replication plasmids, and more particularly pXL2774, did indeed have animal cell transfection characteristics that are essential for use in gene therapy. More precisely, the following were shown:<br>
1)	the capacity of pXL2774 to transfect efficiently, in vitro, various cell types<br>
of human or murine origin;<br>
the capacity of pXL2774 to transfect, in vivo, mouse muscle;<br>
the capacity of pXL2774 to transfect, in vivo, tumour cells implanted into<br>
mice.<br>
The electrotransformation, fermentation and transfection experiments thus made it possible to validate conditional replication plasmids as vectors which can be used in gene<br>
therapy by showing:<br>
i) that they did not replicate detectably in an E. coli strain that does not express the pir gene (conditional origin of replication)<br>
ii) mat they could be produced on a scale compatible with industrial production, in a defined medium that does not contain antibiotics;<br>
iii) that these plasmids could transfect, in vitro and especially in vivo, mammalian cells.<br>
EXAMPLE 7: In vitro production of recombiaant proteins<br>
7 1". Construction of the expression vector<br>
To show the feasibility of such an approach, we constructed an expression vector according to the criteria described above (Examples 2 and 3). This vector, pXL3056, contains:<br>
the bacterial part which comprises the conditional origin of replication (ori gamma),<br>
the cer fragment of ColEl, and the gene which ensures selection in bacteria (sup)<br>
the expression cassette, based on the system described by Studier (Studier et al.,<br>
1990), comprising the promoter of gene 10 of bacteriophage-TV, the lacO operator, the gene<br>
:oding for aFGF 154 (acidic Fibroblast Growth factor, form containing 154 ammo acids)<br>
(Jaye et al., 1986), and the TF terminator of bacteriophage T7.   This expression cassette is<br>
identical to the one present on the pXL2434 plasmid, which is described in application<br>
WO 96/08572.<br>
The construction of pXL3056 is presented in Figure 8. The EcdRI-BgHL fragment of pXL2434 (1.1 kb) containing the aFGF expression cassette was cloned in the pXL2979 conditional replication vector (1.1 kb purified fragment) at the BgHL and EcoKL sites to generate pXL305 6.<br>
pXL2979 results from the ligation of 3 fragments: i) Accl-Xbal fragment of pXL2730 (0.8 kb, which provides ori gamma and cer), ii)NarI-SalI fragment of pXL2755 (0.18 kb, which provides the sup Phe gene), iii) SalL-Spel fragment of pXL2660 (1.5 kb, which provides the kanamycin resistance gene).<br>
pXL2660 results from the cloning of the 1.2 kb Psti fragment of pUC4K (Vieira and Messing, 19S2) inpMTL22 (Chambers et aL, 1988) linearized with-PM.<br>
7.2.	Production of the expression strain<br>
Plasmid pXL3056 was introduced by transformation into the XA.C-lpirll6 strain. The resulting strain was then transformed by the plasmid PT7pol23 (Mertens et aL, 1995), at 30°C. In order to express the gene of interest under control of the T7 promoter, the bacterium must contain in its genome, on a plasmid, or abacteriophage, a cassette allowing expression of the RNA polymerase of bacteriophage T7. In the example described, we used the plasmid pT7po!23, which is compatible with R6K derivatives such as pXL3056, and which allows the temperature-inducible expression of bacteriophage T7 RNA polymerase. However, it can also be envisaged to lysogenize the XAC-lpirll6 strain with lambda DE3 (Studier et aL, 1990) to conserve only one plasmid and to induce the production of T7 RNA polymerase by IPTG rather than by temperature.<br>
7.3.	Expression of aFGF<br>
The XAC-lpirll6 strain (pXL3056+PT7po!23) was cultured at 30°C in M9 minimum medium supplemented with 02% of casamino acids (DIFCO) and kanamycin (25 |ag/ml), up to an optical density at 600 nm of 0.6-1.0. Half of the culture was then placed at 42°C (induction of the T7 RNA polymerase), while the other half remained at 30°C (negative control). The same experiment was carried out with the XAC-lpirll6 (pXL3056+pUC4K) strain which constitutes a control for the expression of aFGF in the absence of T7 RNA polymerase.<br>
The results obtained are presented in Figure 9. They show that the production of aFGF was comparable or superior to that observed with BL21(DE3)(pXL2434) (WO 96/08572), which clearly shows the potential of conditional replication plasmids for the expression of recombinant proteins in vitro, especially in E. coli.<br>
EXAMPLE 8: Construction of a pCOR vector which expresses a wild-type or hybrid p53 protein or the FGF1 human protein<br>
This example describes the construction of conditional replication vectors according to<br>
the invention containing a nucleic acid coding for a p53 protein. These vectors can be used to<br>
i restore a p53-type activity in deficient (mutated, deleted) cells such as, in particular, tumour<br>
cells.<br>
The eukaryotic expression cassette contains the following elements:<br>
1) CMV "immediate early" promoter (positions -522 to +72) followed by the leader<br>
sequence of the thymidine kinase gene of type I herpes simplex virus (position -60 to +1 of the<br>
gene, with reference to the sequence in the article by McKnight, S.fL. (1980) Nucleic Acids<br>
Res. 8:5949-5964);<br>
2a) a nucleic acid which codes for wild-type p53 protein or for a p53 variant, as<br>
described in application PCT/FR96/01 111 (V325K variant = V325 with a Kozak sequence<br>
with ATG);<br>
2b) a nucleic acid which codes for the human FGFa or FGF-1 as described in Jaye M. (Sciences 1986; 233(4763) :451,US patent No: 4,686,113, and European Patent No: 259 475;<br>
2c) a nucleic acid which codes for a fusion gene between human fibroblast interferon secretion signal (Taniguchi et al.) and the naturally occurring truncated form of human FGF-1 from amino acid 21 to 154 as described by Jaye et al., and US 5,849,538.<br>
3) the polyA polyadenylation sequence of SV40.<br>
These elements were placed in the form of a fragment Ascl-Xbal on the pCOR vector pXL2988 between the sites BssHH. and SpeL. pXL2988 is identical to pXL2979 (Example 7.1.) apart from the presence of an additional element, a sequence capable of forming a DNA triple helix composed of 17 copies of the trinucleotide GAA, placed alongside the gamma origin of replication.<br>
The resulting plasmids were named pXL3029, pXL3030, pXL3179 or NV1FGF (Figure 10).<br>
The functionality of these constructions was verified in vitro on p53-SAOS2 cells in culture by measuring the transcriptional-activator activity of p53 or pSSsuperWT, or by measuring the secretion of FGFlfor example by ELISA experiments which is well known in<br>
the art.<br>
EXAMPLE 9: Construction of TEX 1 (XAC1 pir!16. endA', traP")<br>
The E. coli XAC-lpirll6 contains an F episome, a circular DNA molecule of approximately 100kb, that carries proB+ lacLnslacZumam- Many male E. coli laboratory strains carry a traD36 mutation on their episome, but no mutation affecting F' transfer ability has been described for XAC-1. The gene traD is at the 5' end of one of the tra (transfer) operons and encodes a membrane protein directly involved in DNA transfer and DNA metabolism (Frost et al., BBRC, 1994, 58:162-210). A 2kb central fragment from traD, comprising 92% of the gene, was replaced with the 2kb omega element (Genbank accession number M60473) from pEP45Cl (Prentki and Krisch, 1984, Gene, 29:303-313) by homologous recombination in XA.C-lpirll6 endA'. The omega element contains the aadA antibiotic resistance gene flanked by short inverted repeats. The gene aadA encodes aminoglycoside-3 adenyltransferase and confers resistance to streptomycin and spectinomycin ("Sp11"). The omega fragment was used because it prematurely terminates RNA and protein synthesis leading to the inactivation of the whole traD operon. This new pCOR strain XAC-Ipirll6 endA- rroZ&gt;::SpR was designated TEX1. Transfer of any resident plasmids, either pCOR or pUC, was undetectable when the donor was TEX1.<br>
•The new pCOR host strain TEX1 was assessed in fermentation experiments. Complex media containing yeast extract were used for fed-batch fermentation with XA.C-lpirll6. pCOR stability (more than 50 generations) makes it possible to use a non-selective media. Under these conditions, XAC-lpirll6 produced more than 40 g/1 dry cell weight and 100 mg/1 of pCOR pXL2774 were obtained from 2-liter fermenters. pCOR copy number was estimated at 400-500 copies per cell and the rate of plasmid DNA synthesis was constant throughout fermentation. These results were extrapolated to an 800-liter fermenter suitable for manufacturing. The fermentation was also performed in the absence of yeast extract or any raw material from animal origin. Similar results (30g/l dry cell weight and 100 mg/1 of plasmid DNA) were obtained using a defined medium in 2-liter cultures with no loss of productivity.<br>
EXAMPLE  10:   Construction  of XAC-lpirll6pir42  host  strains   by homologous<br>
recombination<br>
1) Construction of a suicide vector carrying the cassette "Krr\R-uidA:virn6:pir42" The RaF-uidA::pirll6 cassette from M13wm33 as described in Example 1 (Metcalf W. et al. Gene, 1994, 138(1-2): p. 1-7), was modified by site-directed mutagenesis using PCR (QuickChange site-directed mutagenesis kit, Stratagene, La Jolla, CA) to introduce faepir42 mutation into Haspirll6 gene. The different cloning/mutagenesis steps are described in Figure 13<br>
The oligonucleotides used for mutagenesis contained the pir42 mutation along with a silent mutation that created a Clal site to easily indicate the processing of pir42 by restriction analysis when needed.<br>
The sense and antisense oligonucleotides used are as follows: Sense oligonucleotide number 1<br>
pir42 Clal<br>
Antisense oligonucleotide number <br>
Clal pir42<br>
The technique used to replace pirl!6 by pirl!6pir42 in the genome of E. coli pCOR host TEX1 was based on that of Blum et al. (IBacteriol. 1989, 171, pp 538-46). The recombinant bacteriophage pXL3723 shown in Figure 13 is a suicide vector in all non-.suppressor E. coli strains, because it has a non-sense mutation in gene n encoding M13 nickase that prevents viral genome replication.<br>
Double recombination was performed as described for the construction of XAC-\pirll6 (Example 1, point 2). Clones that had undergone double homologous recombination events were screened by PCR to test for the presence of the pir42 mutation in the genome of TEX1. Genomic DNA isolated from double recombination candidates was used as a template for PCR. Secondly, sequencing was done on each unique amplified fragment, all of which were of the expected size. The PCR fragments are shown in Figure 14.<br>
The PCR primers were the following:<br>
This analysis showed that one out of the six double recombinants tested had i undergone the allele exchange. This new strain, named TEXlpir42, was further evaluated for its ability to replicate pCOR plasmids compared to the parental strain TEX1.<br>
2} Evaluation nfTEXlmr42<br>
pCOR plasmids were transformed in parallel into TEX1 and TEXlpir42 and grown overnight in 2ml of selective M9 medium. Then, the plasmid DNA was extracted with the Wizard SV plus minipreps kit (Promega) to evaluate the relative plasmid copy number and topology of the pCOR plasmids in both strains.<br>
A 2-fold increase in copy number was obtained reproducibly in TEXlpir42 transformed with the pCOR plasmid pXL3516 (2.56 kb). To farther characterize TEXlpir42, the copy number and topology of pCOR plasmids such as pXL3179 and pXL2774 were evaluated by agarose gel electrophoresis analysis after small scale purification of plasmid DNA (4 to 6 clones/strain). Copy number was evaluated on plasmids linearized with EcoBI restriction enzyme. A topology test was run on non-digested plasmids, in the absence of ethidium bromide. The resulting agarose gel is displayed in Figure 15, and clearly shows a higher plasmid copy number when the plasmid pXL3179 was produced in TEXlpir42, than when produced in TEX1 strain. Figure 15 also displays the topology of the plasmid pXL3179, and shows that an increase in plasmid copy number, which were essentially in the form of monomers, with few plasmids in the form of multimers. The results obtained with these pCOR plasmids are also summarized in Table 5. Relative copy number was calculated in comparison with the same plasmid in TEX1. A 2-3 fold increase in plasmid copy number was observed with plasmids pXL3179 and 2774 produced hi TEXlpir42.<br>
Replication and copy number of pCOR plasmids produced in TEXlpir42<br><br>
(Table Remove)<br>
copy number was compared to the same plasmid in <br>
EXAMPLE 11: Comparative experiments: construction of TEXlcop21 (XAC-lendA-traD- virll6coy2T)<br>
11 Construction of TEXlcov21<br>
The TEXlcop21 strain was constructed similarly as that described in Example 10 for TEXlpir42. The following oligonucleotides used to introduced cop21 into faspirll6 gene by directed mutagenesis were as follows: Sense oligonucleotides: 1 1 153 (SEQ ID NO: 1 1)<br>
The cop21 mutation was introduced as a TCC serine codon instead of the TCA serine codon to eliminate a HindlH restriction site close to the mutation.<br>
The template used for directed mutagenesis was pXL3395 (see Figure 13). The resultant plasmid named pXL3432 was used to construct the suicide Ml 3 vector in a similar way as to what is shown for pir42 in Figure 13. The suicide vector pXL3749 is shown in Figure 16.<br>
The E. coli clones obtained after homologous recombination with pXL3749 were screened by PCR and subsequent restriction with HindUL and sequencing to monitor the cop21 andpirll6 mutations.   One clone out of the six double recombinants tested had undergone the gene replacement. The resulting strain was named TEXlcop21. 2} Evaluation of TEX.lcov21<br>
TEXlcop21 was transformed by various pCOR plasmids, including pXL2979, a 2.5 kb KmR pCOR vector (See Example 7.1), and assayed for increased copy number by gel electrophoresis. Such an experiment with pXL2979 is shown in Figure 17. Plasmid DNA from four independent clones for each strain prepared with Promega miniprep kit was linearized with EcoRI, electrophoresed on agarose gel and then stained with ethidium bromide. Each sample represented a similar amount of bacteria, as measured by optical density at 600 nm. The agarose gel electrophoresis obtained for the pCOR plasmid pXL2979 produced in E. coli TEXlcop21, XAClpir, and TEXl is displayed in Figure 17. It clearly shows there was no increase in plasmid copy number when the plasmids are produced hi the TEXIcop21 strain, as compared with TEXl. EXAMPLE 12: Construction ofTEX2pir42 (XAC-lpirll6 pir42 recAl<br>
Firstly, a recA- derivative of TEXl was constructed. Thepir42 mutation was then introduced into the resulting strain named TEX2 to generate 1EX2pir42. 1) Construction of £ coli TEX2, a recA- derivative of TEXl A deleted recA gene containing 3 translation stop codons (one in each frame) at its 5' end was obtained by PCR. This deleted recA gene was introduced by gene replacement (Blum et al., J. Bacteriol, 1989, 171, pp. 538-46) into the TEXl genome. The construction of the suicide vector for homologous recombination is shown in Figure 18. PCR primers used for the amplification of recA fragments are shown in the following <br><br>
(Table Remove)<br>
Restriction sites added to the recA sequence are underlined.<br>
To maintain the RecA+ phenotype necessary for homologous recombination to occur, the recA function was provided to E. coli TEX1 with a plasmid containing a heterologous recA gene that can complement E. coli recA mutants, such as for example the recA gene of the bacterium Agrobacterium radiobacter, and an antibiotic gene resistance, such as the ampicillin resistance gene. After gene replacement, the plasmid was eliminated from the recombinant strain by culture-dilution in non-selective medium (LB). The absence of the plasmid was screened for the loss of antibiotic resistance.<br>
The resulting strain was named TEX2. Gene replacement was monitored by PCR in Figure 19. PCR Primers are described in the following .<br>
(Table Remove)<br>
The first primer was based on the sequence of the recA gene. The second one was based on a sequence close to but outside the homology region present in the suicide vector pXL3457 (immediately 5' or 3' of recA) to ensure that amplification can only occur on a genomic fragment. The sequence of both oligonucleotides was chosen according to the sequence ofE. coli, which comprises the recA locus (GenbankECAE000354).<br>
The PCR fragments obtained from a rec/4-deleted strain were shorter as compared to those obtained with a wild-type strain, as presented in the following .<br><br>
(Table Remove)<br>
The PCR profile obtained was as expected and demonstrated the presence of a truncated recA gene in the genome of TEX2. The recA- phenotype (sensitivity to UV light), as well as phenotypic characteristics of TEX2, were checked. Phenotypic characteristics of TEX2 were the same as those of TEX1 strain, Le.., ara-, Rif*, NalR, SpR, UidA-, Arg-, Kms Amps), as expected.<br>
.   B&gt; Construction of E. coli TEX2vir42<br>
A TEX2pir42 strain was constructed by double homologous recombination, according to the strategy described in Example 10, with the exception that recombination in TEX2 was carried out hi presence of a plasmid carrying a heterologous recA gene capable of complementing the E. coli recA mutants, in order to maintain a recA+ phenotype required for homologous recombination.<br>
Gene replacement was monitored by restriction analysis of the PCR product digested with Clal (see Figure 14). Gene replacement had occurred in two out of the four-studied double recombinant clones.<br>
G) Evaluation of E. coli TEX2pir42<br>
1) Evaluation at lab scale plasmid production:<br>
1EX2pir42 was transformed by the pCOR plasmid pXL3179 (2.4 kb). Production of pXL3179 in TEX2pir42 was intensively studied at the lab scale, in terms of reproducibility of the improvement of plasmid copy number, conditions of culture, as well is stability (number of generations). All the studies consistently showed a 2 to 5-fold ncrease of plasmid copy number as compared to production of pXL3179 in TEX1 under lie same conditions. Plasmid copy number was assessed further to the production of pXL3179 in TEX2pir42, and TEXlpir42 and TEX1 as comparative experiments. In this<br>
experiment, plasmids were extracted from identical bacterial biomass, based on the OD at 600 nm, and analyzed by agarose gel electrophoresis. The gel was stained with ethidium bromide after electrophoresis. The agarose gel electrophoresis, which is displayed in Figure 20, clearly shows that plasmids are produced in TEX2pir42 at high copy number, and advantageously shows that plasmid multimers are reduced when produced in TEX2pir42 instead of TEKlpir42.<br>
2) Evaluation in fermentersj<br>
These results were confirmed at a larger scale in 7-liter fermenters, as described<br>
below.<br>
a) Composition of fermentation media<br>
The composition of the medium used for inoculum cultures was: Na2HP04 6 g/1, KH2P04 3 g/1, NaCl 0.5 g/1, NH4C1 1 g/1, NH^PC^ 3 g/1, glucose 5 g/1, MgS04,7H20 0.24 g/1, CaCl2,2H20 0.015 g/1, thiamine HC1 0.010 g/1.<br>
The composition of the medium used for fed-batch culture was as follows: KH2P04 8 g/1, K2HP04 6.3 g/1, Na2HP04 1.7 g/1, NH4C1 2.5 g/1, glucose 10 g/1, MgS04,7H20 2.6g/l, thiamine 0.011 g/1, Biospumex36 antifoam O.lnil/1, salt mix (see table 9) 2.5 ml/1.<br>
(Table Remove)<br>
Composition of salt mix<br><br>
The composition of the supply medium was as follows: 50% glucose, 0.7%, magnesium 0.02% thiamine-HCl, 1% Biospumex36 antifoam.<br>
bt Fermentation parameters<br>
A 7-liter fermenter containing 3 liters of the fed-batch medium was inoculated with 1.2% of the inoculum culture. Inoculum was prepared as follows: 250ml of the inoculum<br>
•medium in a 2-liter flask was inoculated with 0.25ml of a frozen cell suspension of the E. coli sHain TEX2pir42 (pXL3179).<br>
-Flasks were incubated for 24 hours at 37°C at 220 rpm. After 24 hours, different parameters were measured: residual glucose: Og/1, OD6oonm was 2.7 and pH 6.24. During fermentation, the pH was controlled and adjusted automatically between 6.9 and 7 with NHs. The temperature was maintained at 37°C and the dissolved oxygen adjusted to a 45% p02 by retroaction on the stirring rate.<br>
After initial batch culturing of the bacterial strain for about 17 hours and consumption of the carbon source (glucose), the supply medium was added. Glucose and acids, such lactate and acetate, were maintained at a concentration close to 0.<br>
c) Results<br>
Final results are presented in Table 10, as compared to production in a 100-liter fermenter with E. coli TEX1 (pXL3179) in optimized conditions.<br>
As for XA.C-lpirll6, there was no difference between 7-liter and 800-liter fermenters in terms of plasmid copy number of pXL3179 produced in TEX1.<br>
Plasmids pXL3179 produced in a 7-liter fermenter using a E. coli TEX2pir42 was compared to the production of pXL3179 in a 100-liter fermenter with E. coli TEX1, in optimized conditions. It was demonstrated that as for ~KAC-lpirll6 (See Example 5.3), there is a stable plasmid production rate in a 7-liter, 100-liter, or 800- liter fermenter in<br>
TEX1.<br>
Table 10: Characteristics of the fermentation of TEX1 and TEX2pir42 strains containing<br>
pXL3179.<br><br>
(Table Remove)<br>
There were 3-fold more copies of plasmid pXL3179 per bacterium in TEX2pir42 as compared to TEX1.<br>
Plasmids corresponding to different fermentation time points were extracted from identical bacterial biomass, based on the OD at 600 nm, and analyzed by agarose gel electrophoresis. Figure 21 clearly shows an increase of the plasmid copy number with the duration of the fermentation. Also, Figure 21 shows the topology of the pXL3179 plasmid produced in Opl328S5 TEX2pir42, which was nearly exclusively in a monomeric form.<br>
In conclusion, the E. coli host strain TEX2pir42 according to the present invention provided an unexpectedly high plasmid copy number improvement of pCOR plasmids, such as pXL3179, of 2 to 5-fold in TEX2pir42 as compared to TEX1, at a lab scale and in fermenters. Furthermore, while the plasmid copy number was greatly improved, plasmids so produced exhibited a monomeric topology, not only at lab-scale but also at a larger scale (7-liter fermenter) compatible with industrial production.<br>
5 EXAMPLE 13: New Copy-Up Mutants of pirl 16 Identified by a Novel Fluorescence-Based Screening Method<br>
To  increase pCOR plasmid  copy number in bacterial host  cells,  we have mutagenized ^hepirll6 gene, which encodes a copy-up mutated version of the pir gene. To date all of the mutations increasing the copy number of R6K-derived plasmids, such as &gt; embodiments of pCOR, have been found within the_pz&gt; gene.<br>
After random mutagenesis by PCR, mutated pirl 16 genes were introduced into a pCOR vector containing the cobA reporter gene, which is described below. After fluorescence based screening, the copy number and topology of the selected mutant plasmid were evaluated. We obtained three different mutants of pirl 16 gene that increase plasmid copy number. These novel mutations have not been described previously.<br>
A classical screening method for copy-up mutants is based on antibiotic resistance. In this method, the level of resistance of a host bacterium to an antibiotic is a function of the copy number of an antibiotic resistance gene located on a plasmid within the cell. As the copy number of the plasmid, and therefore the antibiotic resistance gene, increases, the level of antibiotic resistance also increases. This method, however, was not applicable for R6K-derived plasmids in host cells containing the pirl 16 mutation due to a too high baseline copy number of the plasmid (about 400 copies/cell) in these cells. Accordingly a<br>
new screening method based on fluorescence to identify copy-up mutations of ihepirll6 gene was developed.<br>
For this new method, the cobA gene was introduced into a pCOR vector to provide a simple means of monitoring improvement in plasmid copy number. The cobA gene was obtained from Pseudomonas denitrificans ( Crouzet et aL, J. Bacteriol, 172:5968-79 (1990)). It encodes uroin methyltransferase, an enzyme of the vitamin B12 pathway that adds two methyl groups to the urogen lH molecule. "When overexpressed in E. coli, cobA leads to the accumulation of red products that are fluorescent under near UV light. When exposed to UV, bacterial colonies overexpressing this gene appear pink to red. We tested this gene to determine if it could serve as a reporter gene for plasmid copy number in the pCOR system.<br>
To evaluate the relationship between plasmid copy number and level of fluorescence of transformed bacteria exposed to UV light a control plasmid (pXL3767) was constructed comprising cobA deleted of its own promoter (Fig. 22). This plasmid was transformed into three different host strains (XAClpir, XAClpirJ16 and TEXlpir42). These strains were selected based on previous experiments showing that the average copy number of a pCOR plasmid in XAClpir is 1, it is approximately five to ten fold higher in TEX1, and 15 to 30 fold higher in TEXlpir42.<br>
Recombinant colonies were streaked on M9 minimal medium and exposed to UV light on a transiUuminator as shown in Figure 22. We observed that the fluorescence intensity of the colonies was positively correlated with the plasmid copy number, with XAClpirll6 exhibiting more fluorescence than XACpir, and TEXlpir42 exhibiting more fluorescence than XAClpir 116.<br>
The results shown in Fig. 22 demonstrate that this fluorescence-based assay method easily discriminates between the tested plasmid copy numbers, especially between the plasmid copy number found in strains TEX1 and TEXlpir42. That is, the intensity of red fluorescence observed in this assay increases with the plasmid pCOR-co&amp;A copy-number.<br>
Having demonstrated a positive correlation between fluorescence and cobA copy lumber, we constructed a plasmid into which mutagenized pirl!6 genes were introduced for screening. Four plasmids with different combinations of constitutive modules, as shown in Fig. 23, were constructed and tested. One of these plasmids demonstrated a<br>
' significantly different level of fluorescence when transformed mtopirll6 an&amp;pirll6pir42 isogenic strains. This plasmid, pXL3830, is shown in pertinent part in Fig. 24.<br>
Control plasmids were used during the screening and evaluation experiments. First, a baseline level control plasmid, pXL3830, containing "wild type" pirl 16 was used to set a baseline fluorescence level. Second, pXL3795, that contains the double mutation pirl 16-pir42 which increases the copy number of the plasmid by 4 to 6 as compared to pXL3830, was used as a positive control.<br>
Random mutagenesis was performed on pirl 16 gene using the Diversify PCR random mutagenesis kit (BD Biosciences Clontech, Palo Alto, CA, USA). Condition 1, which introduced an average of 2 mutations per 1000 base pairs, was used. A preliminary experiment run using "condition 1" has shown by sequencing of 12 mutants that the mutation rate was actually about 2 mutations in the pirl 16 gene. The pirl 16 gene was amplified as an EcoRI-Sstl fragment with oligonucleotides C8832 (5- and C8833 (5'- After digestion with £coRI and Sstl, the mutagenized fragment containing pirl 16 was cloned into pXL3830 in place of the "wild-type" pirll 6 gene.<br>
Plasmids carrying mutagenized pirll 6 ("pirll 6*") were transformed into K coli strain XAC-1, the parent of pCOR host XA.Clpirll6. Transformants were screened for increased fluorescence under UV light and compared to XACl(pXL3830) and XACl(pXL3795) controls. A duplicate plate was not exposed to UV to minimize secondary mutations. A representative screening plate under UV light is shown hi Fig. 25.<br>
The following flow chart summarizes the results of the screening experiment<br>
16 confirmed and evaluated for plasmid copy number/topology/sequencing ofpir* (gel)<br>
The evaluation of the three selected mutants is summarized in Fig. 26. Each mutant showed an increase in copy number as compared to the pirl!6 plasmid. In the case of mutants 114C and 100B, the plasmid was essentially in monomeric form. This could be an advantage as compared to apirll6pir42 plasmid, which has an increased copy number and a high multimer content, like mutant 201C.<br>
The pirll 6* gene of each mutant was sequenced. Each clone contains a single non-isocodant mutation in. pirll 6 ORF. All three of the mutations affect the C terminus of the pi protein, which is involved in DNA binding. None of these mutations have been described before.<br>
Once detected by screening in a plasmid system, these mutations were evaluated in a production system, that is, where the pirll 6* gene is introduced into the genome of an E. coli pCOR host strain. The strategy for this evaluation is summarized in Fig. 27.<br>
For this evaluation, plasmid pXL3179 plasmid was transformed into each of the three E. coli strains bearing the mutations identified in Fig. 26, and assessed for plasmid copy number and topology. The results of these experiments are presented in Fig. 28. It was observed that plasmid copy number was significantly increased relative to XACIpirll6 only for 201C mutant<br>
EXAMPLE 14: Minicircle with M13 gene IH as a tool for integration by homologous recombination in E. coli<br>
1. Suicide vectors<br>
Gene replacement by double homologous recombination in E. coli requires the use of a suicide vector. These vectors are constructed and produced in a host capable of replicating them and used subsequently for recombination into the chromosome of a host unable to replicate them.<br>
Bacteriophage Ml 3 is a very useful genetic tool that can be used in rep mutants (Metcalf, W., W. Jiang, et al, Gene 138:1-7 (1994)) or in non-suppressor strains ofE. coli<br>
when M13 mp8 through 11 are used (Bhun, P. et al, J BacterioL. 171:538-46 (1989)). Certain limitations in terras of construction, insert size, and instability are frequently encountered. Plasmids carrying the R6K gamma DNA replication origin are well known suicide vectors (Miller, V. and J. Mekalanos, J BacterioL. 170:2575-83 (1988)), but they are not useful for modifying E. coli strains that express the pi protein, which permits such plasmids to replicate.<br>
A universal suicide vector was engineered with a novel counter selectable marker and used to construct E. coli strains wherein mutants of^epirU6 gene (pirl!6*) are inserted into a bacterial genome by homologous recombination. The strategy presented here is demonstrated for mutant 114C, but has also been used to produce strains bearing<br>
other pirl 16* mutants.<br>
2. Counter-selectable marker<br>
Different markers can be used to select for bacteria having undergone a second recombination event. This event leads to the loss of this marker and in some cases to gene replacement after recombination between the chromosome and a suicide vector. For instance, the SacB gene from Bacillus is lethal when bacteria expressing the gene are plated on a medium containing sucrose (Ried, J. L. and A. Collmer, Gene 57:239-46 (1987)). As .another example, the tetracycline resistance gene confers sensitivity to fusaric acid (Bochner, B. R., et al., J BacterioL 143(2):926-3 (1980)). The infection by the bacteriophage M13 confers the sensitivity to the detergent deoxycholate (Blum, P., et al., J BacterioL 171:538-46 (1989)).<br>
Due to a lack of efficiency in some E. coli strains, a positive selection method for double recombinants was developed. Gene in from bacteriophage Ml 3 was evaluated as a counter-selectable marker. This gene encodes a minor virion component responsible for the infectivity of the particles. When overexpressed from the multicopy plasmid pBR322, gene in confers deoxycholate sensitivity on the cells due to insertion of the gene m protein into the membrane of the bacteria (Boeke, J. D. et al., Mol Gen Genet 186:185-92 (1982)). No report indicated if this gene could be used as an efficient counter-selectable marker when present as a single copy in the genome of the E. coli. Therefore, we tested this hypothesis with a minicircle suicide vector.<br>
3. Amplification by PCR of the deleted version of gene IH from Ml 3 To reduce the size of the minicircle vector to be constructed, a deleted version of gene m (gene DP) that is still able to confer sensitivity to deoxycholate (Boeke, J. D., P. Model, et al.,  Mnl Gen Genet 186:185-92 (1982)) was chosen. It was amplified by PCR along with its own promoter from M13mpl8 (Yanisch-Perron, C., J. Vieira, et al., Gene 33:103-19 (1985)) as a Belli- Xbol fragment (see Figure 29). The oligonucleotides were as follows:<br>
The amplified fragment was cloned by T-A cloning into pGEMT-easy (Promega Corporation, Madison,WI, USA) to generate pXL4230 (Fig. 29). The nucleotide sequence of the insert was found to agree with that described in GenBank under accession no. VB0018. pXL4230 confers sensitivity to deoxycholate when transformed in E. coli strain DH10B (Invitrogen), indicating that it functions as expected. 4. Mim'circle-based suicide vector<br>
As it does not contain any origin of replication, a minicircle plasmid may be used as a universal suicide vector. For this purpose, a selectable marker such as the kanamycin resistance gene must be added to select for a first homologous recombination event To counter-select for bacteria that have not undergone a second event of recombination, the gene ET was added to the minicircle vector. The construction of the plasmid used to produce minicircle for recombination is shown in Fig. 29.<br>
The minicircle is generated from a plasmid, such as pXL4235, after induction of the bacteriophage lambda integrase, which recombines between attP and attB on the plasmid (Darquet, A. M et al., Gene Ther 4(12): 1341-9 (1997)). This recombinase is expressed under the control of pbad in a arabinose-dependent manner in E. coli strain G6264, which is described in U.S. application no. 09/981,803. The resulting minicircle contains attL, a TH (triple-helix) forming sequence for purification, the selectable marker Tn903 kanamycin resistance gene, the counter-selectable marker gene HT and the fragment<br>
of interest for homologous recombination, cloned in the multi-cloning site of pXL4235 (Fig. 29).<br>
As an example, the constructs used to generate E. coli strains expressing a copy-up mutation ofpirJ16 are described in Fig. 30. These strains can be used to produce pCOR plasmids (Soubrier, F. et al, Gene Ther 6:1482-1488 (1999)). Since there is no homology between pir and the bacterial genome, Hh.epirJ16* sequences were inserted into the E. coli chromosomal uidA gene, which encodes p-D-glucuronidase. This gene provides sufficient sequence similarity with the E. coli genome for homologous recombination to occur.<br>
The protocol for the purification of minicircle and recombination occurred as follows. Plasmid pXL4256 (Fig. 30) was transformed in E. coli strain G6264 to generate G6656. Fifty ml of LB medium supplemented with ampicillin (100 mg/1) were inoculated with 0.5 ml of an overnight culture of G6656 and incubated at 37°C, with shaking at 200 rpm until the optical density at 600 nm reached 0.7. Minicircle production was induced by the addition of 250ul of a sterile solution of 10% arabinose to the medium. After 30 minutes at 37°C, 200 rpm, total plasmid DNA was extracted using the Wizard Plus Midipreps DNA Purification system (Promega Corporation, Madison WI, USA). Six ug of the plasmid DNA preparation were loaded onto a 0.8% agarose preparative gel. A supercoiled DNA ladder (Promega Corporation, Madison WI, USA) was used as a molecular weight standard. After electrophoresis overnight at 50V, the supercoiled minicircle construct (5.1 kb) was extracted and purified from the gel using an SV gel purification kit (Promega Corporation, Madison WI, USA).<br>
5. Double homologous recombination with Tninicircle 4256 (uidA::pirll6* minicircle suicide vector)<br>
The.recombination steps for constructing the strains and the corresponding phenorypes are described in Fig. 31.<br>
For the first recombination event (integration), 0.2, 1 and 5 ul of purified minicircle 4256 were electroporated in E. coli strain XAC1 (Normanly, J et al., Proc Natl Acad Sci USA 83:6548-52 (1986)), which is the parental strain for pCOR hosts. Kanamycin resistant colonies were obtained on LB Agar supplemented with kanamycin (50 mg/1) after overnight incubation at 37°C.<br>
To evaluate the number of colonies potentially containing contaminant non-recombined pXL4256,   50  KmR  colonies  were  streaked  in  parallel  on LB .Agar<br>
supplemented with kanamycin or ampicillin. Only 4 colonies out of 50 were resistant to kanamycin and ampicillin and were shown by plasmid restriction analysis to contam non-recombined PXL4256. This indicated that 46 colonies out of 50 obtained by electroporaton were .actually minicircle 4256 integrants.<br>
For the second recombination event .(excision), all of the 46 Km integrants were isolated on freshly prepared LB Agar plates containing 1.5 % sodium deoxycholate ("Doc"; Sigma) and incubated at 37°C overnight. Only a few deoxycholate-resistant (Doc ) colonies (1 to 15) were obtained for each integrant, as shown in Fig. 32. This result was consistent with the selection of a relatively rare event, such as the second recombination event 100DocR colonies obtained from 15 integrants were patched in parallel on LB Agar with 1 5% Doc and LB Agar plus kanamycin to screen for DocR and Kms double recombinants. 86 % of the screened colonies were Kms, indicating that they had lost the<br>
suicide vector.<br>
To screen for allele replacement, the chromosomal uidA locus was amplified by<br>
PCR. If allele replacement has occurred, the expected PCR fragment size is 1.3 kb. The fragment size corresponding to wild-type uidA locus, that is, without an integrated pirll6* mutation, is 0.85 kb. The results presented in Fig. 33-panel A indicate mat allele replacement has occurred in 30% of the double recombinants. This was confirmed by phenotypic analysis because these clones are also UidA- (beta glucuronidase -) and give white colonies on LB agar supplemented with Xgluc.<br>
The integrity of the bacterial genome in the region close to the site of homologous recombination was checked by PCR on two independent recombinants. The first primer (seq6113 or seq6115) was based on the sequence of the^zr gene and the second (seq6112 or seq61l6) had a sequence based on a sequence close to, but outside of, the homology region (immediately 5' or 3' of uidA}. XAC1 DNA was used as a negative control, whereas XAClpirll6 or TEX1 (Soubrier, F. et al, Gene_Ther 6:1482-1488 (1999)) were used as positive controls.<br>
The oligonucleotides used as PCR primers were the following: Seqll088 :    <br>
The expected size for the PCR product is 0.83 kb using primers seq6112 and seq6113, and 0.88 kb when using primers seq6114 and seq6115. Results are presented in Figure 33-panel B.- The two double recombinants obtained with the minicircle suicide vector showed the expected PCR profile. This demonstrates that double homologous recombination can be easily achieved in E. coli using minicircle plasmids as suicide vector and the Ml 3 gene HI' as a counterselectable marker. This gene replacement technique can be directly universally carried out in any micro-organisms genetic background.<br>
BIBLIOGRAPHY<br>
Alting-Mees, M.A., J.A. Sorge, and J.M. Short. 1992. Methods Enzymol. 216:483-495<br>
Blum, P., D. Holzschu, H.S. Kwan, D. Riggs, and S. Artz. 1989. J. Bacteriol. 171:538-546.<br>
Brosius, J. 1989. Methods Enzymol. 216:469-483.<br>
Chambers, S.P., S.E. Prior, D.A. Barstow, andN.P. Minton. 1988. Gene 68:139-149.<br>
Chung, C.T., and R.H. Miller. 1988. Nucleic Acids Res. 16:3580.<br>
Colloms, S.D., P. Sykora, G. Szatmari, and D J. Sherrat. 1990 J. Bacteriol. 172:6973-6980.<br>
Datta, N., and P. Kontomichalou. 1965. Nature 208:239-241.<br>
Dickely, F., D. Nilsson, E.B. Hansen, andE. Johansen. 1995. Mol. Microbiol. 15:839-847.<br>
Filutowicz, M., S. Dellis, I. Levchenko, M. Urh, F. Wu, and D. York. 1994. Prog, in Nucleic Acid Res. and Mol. Biol. 48:239-273.<br>
Gibson, TJ. 1984. Ph.D Thesis. University of Cambridge.<br>
Greener, A., M. Filutowicz, M. McEachern, and D. Helsinki. 1990. Mol. Gen. Genet 224:24-32.<br>
Herrero, M., V. de Lorenzo, and K.N. Timmis. 1990. J. Bacteriol. 172:6557-6567.<br>
Hodgson, C.P. 1995. Bio/Technology 13:222-225.<br>
Inuzuka, M., and Y. Wada. 1985. EMBO J. 4:2301-2307.<br>
Jaye, M. et al., (1986) Science 233:541-5<br>
Kleina, L.G., J.M. Masson, J. Normanly, J. Abelson, and LH. Miller. 1990. J. Mol. Biol.<br>
213:705-717.<br>
Kowalczykowski, S.C., and A.K. Eggleston. 1994. Annu. Rev. Biochem. 63:9991-10043. Leung, D.W., E. Chen, G. Cachianes, andD.V. Goeddel. 1985. DNA 4:351-355. Maniatis, T., E.F. Fritsch, and J. SambrooL 1989. Cold Spring Harbor Laboratory, Cold<br>
Spring Harbor, N.Y.<br>
MeinneL, T., E. Schmitt, Y. Mechulam, and S. Blanquet. 1992. J. Bacteriol. 174:2323-2331. Mertens, N., E. Remant and W. Fiers. (1995) Bio/Technology 13:175-179 Messing, J., and J. Vieira. 1982. Gene 19: 269-276. Metcalf, W.W., W. Jiang, andBJL Wanner. 1994. Gene 138:1-7. Miller, V.L., and JJ. Mekalanos. 1988. J. Bacteriol. 170:2575-2583.<br>
Normanly, J., J.M. Masson, L.G. Kleina, J. Abelson, and J.H. Miller. 1986. Proc. Natl. Acad. Sci. US A 83:6548-6552.<br>
NJonnanly, J., L.G. Kleina, JM. Masson, J. Abelson, and J.H. Miller. 1990. J. Mol. Biol.<br>
213:719-726.<br>
Eloca, J. 1995. TIBS 20:156-160. Safld, R.K., S. Scharf, F. Faloona, K.B. Muffis, G.T. Horn, H.A. Erlich, and N. Amheim.<br>
1985. Science 230:1350-1354.<br>
Sanger, F., S. Nicklen, and A.R Coulson. 1977. Proc. Natl. Acad. Sci. USA 74:5463-5467. Sawadogo, M., and M.W. Van Dyke. 1991. Nucleic Acids Res. 19:674. Scott, J.R 1984. Microbiol. Rev. 48:1-23. Simoes, D.A., M. Dal Jensen, E. Dreveton, M.O. Loret, S. Blanchin-Roland, J.L. Uribelarrea,<br>
and J.M. Masson. 1991. Ann. N.Y. Acad. Sci. 646:254-258. Simon, R., U. Priefer, and A Punier. 1983. Bio/Tecnnology 1:784-79.1. Sinha, N.D., J. Biernat, J. McManus, andH. Koster. 1984. Nucleic Acids Res. 12:4539-4557. Stirling, CJ. G. Stewart, andDJ. Sherrat. 1988. Mol. Gen. Genet 214:80-84. Stirling, CJ., S.D. Colloms, J.F. Collins, G. Szatmari, and DJ. Sherrat. 1989. EMBO J.<br>
8:1623-1627. studier, F.W., AH. Rosenberg., J.J. Dunn and J.W. Dubendorff (1990). Methods Enzymol<br>
185:60-89.<br>
Summers,D.K., andDJ. Sherrat. 1984. Cell36:1097-1103. rakahashi, K., Y. Sawasaki, J. Hata, K. Mulcai and T. Goto. (1990) In Vitro Cell Dev. Biol.<br>
26:265-74.<br>
Vieira, J., and J. Messing. 1982. Gene 19:259-268.<br>
Wiechelman, K., R Braun, and J. Fitzpatrick. 1988. Anal. Biochem. 175:231-237. Yanisch-Perron, C. Vieira and J. Messing (1985) Gene 33:103-119 13.<br><br><br><br><br><br>
WE CLAIM;<br>
1.	A    prokaryotic    recombinant    host   cell    comprising    a   heterologous<br>
replication initiation protein that activates a conditional origin of replication<br>
and    an    extrachromosomal    DNA    molecule    comprising    a    heterologous<br>
therapeutic gene and a conditional origin of replication whose functionality in<br>
the prokaryotic recombinant host cell requires a replication initiating protein<br>
which is foreign to the host cell.<br>
2.	The prokaryotic recombinant host cell as claimed in claim 1, wherein the<br>
conditional origin replication is from a bacterial plasmid or a bacteriophage.<br>
3.	The prokaryotic recombinant host cell as claimed in claim 2, wherein the<br>
conditional  origin  of replication  is  from  R2K,   R6K,   Rl,   pSClOl,   Rtsl,  F,<br>
RSF1010, PI, P4, lambda, Phi82 or PhiSO.<br>
4.	The prokaryotic recombinant host cell as claimed in claim 3, wherein the<br>
conditional origin of replication is from bacterial plasmid R6K.<br>
5.	The prokaryotic recombinant host cell as claimed in claim 4, wherein the<br>
host cell further comprises a gene comprising an amber mutation.<br>
6.	The prokaryotic recombinant host cell as claimed in claim 1, wherein<br>
said heterologous replication initiation protein is encoded by gene pir 116.<br>
7.	The prokaryotic recombinant host cell as claimed in claim 6, wherein<br>
said host cell carries a mutation in gene endA.<br>
8.	The prokaryotic recombinant host cell as claimed in claim 7, wherein<br>
said host cell carries a mutation in gene traD.<br>
9.	The prokaryotic recombinant host cell as claimed in claim 8, wherein<br>
said host cell is TEX1.<br>
10.	The prokaryotic recombinant host cell as claimed in claim 9, wherein<br>
said host cell comprises a mutated rec4 gene.<br>
11.	The prokaryotic recombinant host cell as claimed in claim 10, wherein<br>
said heterologous replication initiation protein is encoded by pirl 16pir 42.<br>
12.	The prokaryotic recombinant host cell as claimed in claim 11, wherein<br>
the replication initiation protein is encoded by the sequence set forth in SEQ ID<br>
NO: 21.<br>
13.	The prokaryotic recombinant host cell as claimed in claim 11, wherein<br>
said host cell is TEX2pir42.<br>
14.	A method for producing a plasmid using a host cell as claimed in claim<br>
1, wherein said plasmid comprises a heterologous therapeutic gene and a<br>
conditional origin of replication whose functionality in a prokaryotic host cell<br>
requires a replication initiating protein foreign to the host cell, comprising:<br><br>
a)	culturing TEX2pir42 comprising the plasmid to be produced; and<br>
b)	isolating the plasmid.<br>
15.	The method as claimed in claim  14, wherein the origin of replication<br>
comprises SEQ ID NO: 1.<br>
16.	The   method   as   claimed   in   claim   15,   wherein   the   plasmid   further<br>
comprises a selection gene that does not impart resistance to an antibiotic.<br>
17.	The method as claimed in claim 16, wherein the selection gene encodes a<br>
suppressor transfer RNA for specific codons.<br>
18.	The method as claimed in claim  17, wherein the selection gene is a<br>
suppressor of an amber transfer RNA, and the host cell further comprises a<br>
gene comprising an amber mutation.<br>
19.	A prokaryotic recombinant host cell as claimed in claim 1 comprising a<br>
heterologous pir gene encoding a pi protein and an extrachromosomal DNA<br>
molecule comprising a heterologous therapeutic gene and a conditional origin<br>
of replication whose functionality in the prokaryotic recombinant host cell<br>
requires the pi protein.<br>
20.	The prokaryotic recombinant host cell as claimed in claim 19, wherein<br>
the heterologous pir gene comprises at least one mutation.<br>
21.	The prokaryotic recombinant host cell as claimed in claim 20, wherein<br>
the one or more mutations are present in the same or a different region<br>
encoding of the pir gene coding for the copy control region and/or the DNA<br>
binding region, and/or the leucine zipper-like motif or other regions of the<br>
protein II.<br>
22.	The prokaryotic recombinant host cell as claimed in claim 20, wherein<br>
the heterologous pir gene comprises mutations in the copy number control<br>
region and the leucine zipper-like motif.<br>
23.	The prokaryotic recombinant host cell as claimed in claim 22, further<br>
comprising a mutation in the pir gene DNA binding region.<br>
24. The prokaryotic recombinant host cell as claimed in claim 20, wherein the heterologous pir gene comprises mutations in the copy number control region and the DNA binding region.<br>
25. The prokaryotic recombinant host cell as claimed in claim 20, wherein the heterologous pir gene comprises mutations in the DNA binding region and the leucine zipper-like motif.<br>
26.	The prokaryotic recombinant host cell as claimed in claim 19, wherein<br>
the heterologous pir gene is in a plasmid.<br>
27.	The prokaryotic recombinant host cell as claimed in claim 19, wherein the<br>
heterologous pir gene is in the genome of the host cell .<br>
28.	The prokaryotic recombinant host cell as claimed in claim 20, wherein said<br>
host cell is deficient for the gene endA.<br>
29.	The prokaryotic recombinant host cell as claimed in claim 20, wherein said<br>
host cell is deficient for the gene traD.<br>
30.	The prokaryotic recombinant host cell as claimed in claim 20, wherein said<br>
host cell further comprises a mutated recA gene.<br>
31.	A method for producing a plasmid comprising a heterologous therapeutic<br>
gene and a conditional origin of replication whose functionality in a prokaryotic<br>
host cell requires a replication initiating protein foreign to the host cell,<br>
comprising:<br>
(a) culturing the prokaryotic recombinant host cell as claimed in claim 20 under conditions wherein the plasmid is produced; and<br>
(b) isolating the plasmid.<br>
32.	The method as claimed in claim 31, wherein the plasmid further comprises<br>
a selection gene that does not impart resistance to an antibiotic.<br>
33.	The method according to claim 32, wherein the selection gene encodes a<br>
suppressor transfer RNA for specific codons.<br>
34.	The method as  claimed  in claim 33,  wherein  the  selection gene  is a<br>
suppressor of an amber transfer RNA, and the host cell further comprises a<br>
gene comprising amber mutation.<br>
35.	A method for detecting a plasmid copy-up mutation comprising:<br>
(a)	introducing at least one mutation into a target sequence;<br>
(b)	transforming the mutated target sequence into a host cell comprising a<br>
plasmid, wherein the plasmid comprises a nucleotide sequence encoding uroIII<br>
methyltransferase and the copy number of the plasmid is effected by the target<br>
sequence;<br>
(c)	growing the host cell under conditions wherein the nucleotide sequence z's<br>
expressed to produce culture of host cells;<br>
(d)	exposing the culture of host cells to UV light; and<br>
(e)	detecting fluorescence produced by the culture of host cells.<br>
36.	The method as claimed in claim 35, wherein the nucleotide sequence is the<br>
cobA gene from Pseudomonas denitrificans.<br>
37.	The method as claimed in claim 35, further comprising comparing the<br>
fluorescence detected in (e) with fluorescence produced by a culture of host<br>
cells comprising an non-mutated target sequence.<br>
38.	A plasmid comprising a heterologous pir gene comprising at least one<br>
mutation.<br>
39.	The plasmid as claimed in claim 38, wherein the one or more mutations is<br>
present in the same or a different region encoding of the pir gene coding for the<br>
copy control region  and/or the DNA binding region, and/or the leucine zipper-<br>
like motif and/or other regions of the protein II.<br>
40.	The plasmid as claimed in claim 38, wherein the heterologous pir   gene<br>
comprises DNA mutations in the copy number control region and the leucine<br>
zipper-like motif.<br>
41.	The plasmid as claimed in claim 40, further comprising a mutation in the<br>
pir gene DNA binding region.<br>
42.	The plasmid as claimed in claim 38, wherein the heterologous pir gene<br>
comprises mutations in the copy number control region and the DNA binding<br>
region.<br>
43.	The plasmid as claimed in claim 38, wherein the heterologous pir gene<br>
comprises mutations in the DNA binding region and the leucine zipper-like<br>
motif.<br>
44.	The prokaryotic  recombinant host cell as claimed in claim 24, wherein the<br>
mutation in the DNA binding region is the 114C mutation.<br>
45.	The prokaryotic recombinant host cell as claimed in claim 24, wherein the<br>
mutation in the DNA binding region is the 100B mutation.<br>
46.	The prokaryotic recombinant host cell as claimed in claim 24, wherein the<br>
mutation in the DNA binding region is the 201C mutation.<br>
47.	The plasmid as claimed in claim 42, wherein the mutation in the DNA<br>
binding region is the 114C mutation.<br>
48.	The plasmid as claimed in claim 42, wherein the mutation in the the DNA<br>
binding region is the 100B mutation.<br>
49.	The plasmid as claimed in claim 42, wherein the mutation in the DNA<br>
binding region is the 201C mutation.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQwMS1ERUxOUC0yMDA1LUFic3RyYWN0LSgyMi0xMC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1401-DELNP-2005-Abstract-(22-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQwMS1kZWxucC0yMDA1LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1401-delnp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQwMS1ERUxOUC0yMDA1LUNsYWltcy0oMjItMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1401-DELNP-2005-Claims-(22-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQwMS1kZWxucC0yMDA1LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1401-delnp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQwMS1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjItMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1401-DELNP-2005-Correspondence-Others-(22-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQwMS1kZWxucC0yMDA1LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1401-delnp-2005-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQwMS1ERUxOUC0yMDA1LURlc2NyaXB0aW9uIChDb21wbGV0ZSktKDIyLTEwLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">1401-DELNP-2005-Description (Complete)-(22-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQwMS1kZWxucC0yMDA1LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">1401-delnp-2005-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQwMS1ERUxOUC0yMDA1LURyYXdpbmdzLSgyMi0xMC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1401-DELNP-2005-Drawings-(22-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQwMS1kZWxucC0yMDA1LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1401-delnp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQwMS1ERUxOUC0yMDA1LUZvcm0tMS0oMjItMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1401-DELNP-2005-Form-1-(22-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQwMS1kZWxucC0yMDA1LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">1401-delnp-2005-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQwMS1kZWxucC0yMDA1LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">1401-delnp-2005-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQwMS1ERUxOUC0yMDA1LUZvcm0tMi0oMjItMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1401-DELNP-2005-Form-2-(22-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQwMS1kZWxucC0yMDA1LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">1401-delnp-2005-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQwMS1kZWxucC0yMDA1LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">1401-delnp-2005-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQwMS1kZWxucC0yMDA1LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">1401-delnp-2005-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQwMS1ERUxOUC0yMDA1LUdQQS0oMjItMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1401-DELNP-2005-GPA-(22-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQwMS1kZWxucC0yMDA1LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">1401-delnp-2005-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQwMS1kZWxucC0yMDA1LXBjdC0zMDYucGRm" target="_blank" style="word-wrap:break-word;">1401-delnp-2005-pct-306.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQwMS1kZWxucC0yMDA1LXBjdC00MDkucGRm" target="_blank" style="word-wrap:break-word;">1401-delnp-2005-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQwMS1kZWxucC0yMDA1LXBjdC00MTYucGRm" target="_blank" style="word-wrap:break-word;">1401-delnp-2005-pct-416.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQwMS1ERUxOUC0yMDA1LVBldGl0aW9uLTEzNy0oMjItMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1401-DELNP-2005-Petition-137-(22-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQwMS1ERUxOUC0yMDA1LVBldGl0aW9uLTEzOC0oMjItMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1401-DELNP-2005-Petition-138-(22-10-2008).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="225708-a-process-for-the-preparation-of-3-4-substituted-1-cyano-2-alkylthiobenzene-compound.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="225710-an-in-vitro-cell-culture-process-for-the-production-of-a-soluble-ctla4-molecule.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>225709</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1401/DELNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>21-Nov-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>07-Apr-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CENTELION</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>72-82 RUE LEON GEFFROY, F-94400 VITRY-SUR-SEINE, FRANCE.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>FABIENNE SOUBRIER</td>
											<td>20 RUE DES PINS SYLVESTRES, F-94320 THIAIS, FRANCE.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2003/032512</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-10-14</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>10/268,948</td>
									<td>2002-10-11</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/225709-a-prokaryotic-recombinant-host-cell-comprising-a-heterologous-replication-initiation-protein by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:06:31 GMT -->
</html>
