<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/225716-a-process-and-the-kit-of-for-the-detection-of-mycobacterium-tuberculosis-bacilli-present-in-the-sample-by-nucleic-acid-amplification-test by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:08:36 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 225716:A PROCESS AND THE KIT OF FOR THE DETECTION OF MYCOBACTERIUM TUBERCULOSIS BACILLI PRESENT IN THE SAMPLE BY NUCLEIC ACID AMPLIFICATION TEST</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A PROCESS AND THE KIT OF FOR THE DETECTION OF MYCOBACTERIUM TUBERCULOSIS BACILLI PRESENT IN THE SAMPLE BY NUCLEIC ACID AMPLIFICATION TEST</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention TB NAT - Nucleic acid amplification test- is a novel process and kit for the detection of the Mycobacterium tuberculosis bacilli in human pulmonary and extra pulmonary specimens such as sputum, CSF, bronchial secretions pleural biopsy specimens, gastric aspirates, pleura fluids etc by amplification of the nucleic acid present in the above samples. The invention provides a kit for the amplification and detection of DNA present in the sample by amplification. The method of detection of TB bacilli comprises of treating the nucleic acid with the primers, extending the primers to form primer extension products that acts as templates for synthesizing the desired nucleic acid sequence and detecting the sequence so amplified.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Field of Invention<br>
The present invention TB NAT - Nucleic acid amplification test- relates to a diagnostic kit for the detection of TB Bacilli by amplification of existing DNA if they are present in the pulmonary and extra pulmonary samples including sputum, pleural specimens, CSF, bronchial secretions etc. More specifically it is a method for analyzing the presence of TB Bacilli DNA in the sample.<br>
NAT techniques for the diagnosis of tuberculosis have attracted considerable interest with the hope of shortening the time required to detect and identify mycobacterium tuberculosis in respiratory specimens such as sputum, or BAL and non respiratory specimens including CSF, PF, gastric aspirates etc.<br>
Background of the Invention<br>
Tuberculosis is a persistent problem in the developing world and biggest cause of mortality. Mycobacterium tuberculosis is the causative agent of tuberculosis. Mycobacterium tuberculosis infections often results in the pulmonary disease but can cause skin infections lymphadentis, meningitis, and other manifestations beside advent of AIDS has made the disease a major public health problem which has recently been exacerbated by increasing numbers of high- risk patients.<br>
In some countries including US, the tuberculosis is numerically one of the major notifiable infectious disease. It is well recognized that one of the major problems in tackling tuberculosis disease is the lack of simple, reliable and robust serodiagnositc or gene probe assay. These are necessary because current diagnostic tests, even those available in typically advanced rich nations are poorly specific and insensitive being based on a combination of relatively crude symptomology and radiography, staining for acid fast bacilli and bacterial culture. The first two are widely variable features and the second two are reliable, with particularly available tests several weeks may be required to obtain a definite result. The detection of the small numbers of M. tuberculosis bacteria in heavily contaminated samples is often difficult. Many attempts have been made to develop new laboratory tests for tuberculosis but all have suffered from poor specificity or sensitivity. DNA isolation and amplification can detect small amounts of Mycobacterial tuberculosis bacteria present in the sample . This analysis of isolation and amplification offers a highly sensitive method for the rapid detection of small numbers of specific bacteria in the presence of other organisms. Detection of Mycobacterium tuberculosis species in clinical samples is important as a chemical diagnostic tool.<br>
Many chemical methods for detecting and identifying Mycobacterium tuberculosis species in samples require analysis of the bacteria's physical characterization or biochemical characteristics or physiological characteristics. These methods require relatively high concentrations of bacteria in the sample to be detected may be subjective depending on the<br>
chemicals, technicians experience and are time consuming. Because Mycobacterium species are often very difficult to grow in vitro and may take several weeks to reach a useful density in culture, these methods can also result in to delayed patient treatment and costs associated with isolating an infected individual until the diagnosis is completed.<br>
So there is a need for an improved method for the detection of Mycobacterium tuberculosis bacilli in a patient sample so that the present invention came. The present invention provides composition and in-vitro nucleic acid amplification method that produce relatively long amplified nucleic acid sequence to allow detection of the mycobacterium tuberculosis species present in respiratory and non-respiratory sample within a day and has high sensitivity and specificity for the same.<br>
Prior Arts<br>
US Patent No: 5908744 describes a method, composition and kit for synthesizing multiple copies of a target nucleic acid sequence autocatalytically under conditions of substantially constant temperature, ionic strength, and pH are provided in which multiple RNA copies of the target sequence autocatalytically generate additional copies using a mixture of blocked and unblocked primers and/or promoter-primers to initiate DNA and RNA synthesis, preferably with reduced non-specific product formation. The invention is useful for generating copies of a nucleic acid target sequence for purposes that include assays to quantitate specific<br>
nucleic acid sequences in clinical, environmental, forensic and similar samples, cloning and generating probes.<br>
US Patent No: 6638722 relates to methods of DNA amplification with a first primer that has a random sequence of nucleotides at its 3' end and a generic sequence 5' of the random nucleotides, as well as a second primer with the generic sequence of the first primer. The disclosure further relates to a method of precipitating DNA on a solid medium to improve DNA amplification. In a preferred embodiment, the presently disclosed methods are used for high-throughput genotyping of DNA samples, such as bloodstains or trace amounts of DNA.<br>
US Patent No: 6747141: Methods of detecting Mycobacterium avium complex (MAC) organisms using in vitro nucleic acid amplification with amplification oligonucleotides specific for 16S rRNA or DNA sequences encoding 16S rRNA from MAC species are disclosed. Compositions and kits containing oligonucleotides for amplifying and detecting 16S rRNA or DNA sequences encoding 16S rRNA from MAC species are disclosed<br>
US Patent No 6664081: Methods of detecting Mycobacterium species using oligonucleotides to amplify in vitro 16S rRNA sequences or DNA encoding 16S rRNA sequences for many species in the genus Mycobacterium are disclosed. Amplification oligonucleotides for in vitro amplification of 16S rRNA sequences or DNA encoding 16S rRNA sequences from many Mycobacterium species are disclosed. Kits<br>
containing oligonucleotides useful for in vitro amplification of 16S rRNA sequences or DNA encoding 16S rRNA sequences from many Mycobacterium species are disclosed.<br>
US Paent No: 6852494: Disclosed is an insoluble support that can be used, for example, for producing replicates of sample nucleic acids. The support includes a plurality of attached oligonucleotides that include a prokaryotic promoter sequence and a target annealing sequence 3' of the promote. The proximal end of the promoter sequence is spaced from the insoluble support by a distance greater than 10 nm.<br>
Summary of the invention<br>
The present invention TB NAT relates to a qualitative in-vitro diagnostic kit for the detection of Mycobacterium tuberculosis in human pulmonary and extra pulmonary specimens as sputum, bronchial secretions, plural biopsy specimens, gastric aspirates, CSF etc. The test utilizes the Polymerized Chain Reaction of the target DNA extracted from the samples to be amplified followed by agarose gel electrophoresis of this amplified DNA to visualize the same and detects the positivity or negativity of the sample.<br>
The present invention resides in a process for amplifying specific TB DNA sequence using primers and detecting the amplified DNA. The extension product of one primer when hybridized to another becomes a template for the production of the specific DNA and vice versa, and the process is<br>
repeated to produce a desired amount of DNA for visual verification by gel electrophoresis.<br>
"PCR is the most new scientific technology to come along in the last hundred years" Says Mark R. Hughes, Deputy Director of the National Center for Human Genome Research at the National Institutes of Health. (Perhaps better known as the Human Genome Project). It is hard to exaggerate the impact of the polymerize chain reaction. PCR, the quick, easy method for generating unlimited copies of any fragment of DNA, is one of those scientific developments that actually deserves time worn superlatives like "revolutionary" and "breakthrough". The PCR process was originally developed to amplify short segments of a longer DNA molecule (Saiki et al. 1985). A typical amplification reaction includes the target DNA, a Taq polymerize, two oligonucleotide primers, deoxynucleotide triphosphates (dNTPs), reaction buffer and magnesium chloride. The key to the process automation is Taq polymerase. This is an enzyme extracted from Taq and is stable at high temperature. Once assembled, the reaction is placed in a thermocycler, an instrument that subjects the reaction to a series of different temperatures for varying time. This series of temperature and time adjustments is referred to as one cycle of amplification. Each PCR cycle theoretically doubles the amount of targeted sequence (amplicon) in the reaction. Ten cycles theoretically multiply the amplicon by a factor of about one thousand; 20 cycles, by a factor of more than a million in a matter of hours. Thus PCR reaction<br>
consists of three steps which are repeated multiple times denaturation, annealing, and synthesis. These steps are incorporated into a PCR reaction.<br>
The amplification techniques for the diagnosis of tuberculosis have attracted considerable interest, particularly PCR for the detection of Mycobacterium tuberculosis in clinical specimens requires few hours and has a high (100%) specificity and sensitivity comparable to the culture methods. It requires Oligonucleiotide primers capable of amplifying DNA targets, which is specific for M. tuberculosis.<br>
Principle of the test<br>
PCR based detection of the M. tuberculosis is based on three major processes: specimen preparation, DNA isolation, and PCR amplification of target DNA using sets of MTB specific complimentary primer pairs; thereby detecting the amplified product by agarose gel electophorosis by ethidium bromide staining.<br>
a. Specimen preparation<br>
For determination of M. tuberculosis both respiratory and non respiratory specimens come for PCR analysis. Upon receipt, the specimens should be stored at 2 to 8°C prior to being processed. The sample has to be decontaminated, so samples were first centrifuged at 3500g for 20min by adding equal amount of decontamination solution. For tissue specimens, it is cut and homogenized in a motor under sterile conditions before being centrifuged. This decontamination solution is a mucolytic agent, which<br>
decontaminates the samples for further processing. Supernatant is thrown and sediment reconstituted in 1 ml PBS sol (pH 7.4) which is to be stored at -20°C if not to be processed immediately.<br>
All specimens are collected in gamma irradiated disposible plastic containers. Upon receipt all samples are stored at 2-8°C prior to being processed.<br>
b.	Target selection and Target amplification<br>
PCR can be great value detecting very few bacilli when a rapid diagnosis is imperative. MPB-64 insertion element has been widely demonstrated to be highly specific for the M. tuberculosis complex. Selection of a target DNA sequence depends on identification of regions within entire genome that show maximum sequence conversion at the DNA level among all serotypes. Primers used in the kit amplify a 240 base genomic DNA complex of M. tuberculosis, a fragment called MPB-64. The gene analysis revealed that the structural gene for MPB-64 consisted of 618 b.p. and its deduced molecular weight in 22,400. Processed specimens are added to the amplification mixture for single step PCR.<br>
c.	Detection<br>
The amplification product is analyzed over an agarose gel containing ethidium bromide by visualizing it on UV transilluminator or gel doc<br>
system.<br>
Description of the invention Specimen preparation &amp; DNA isolation<br>
The first part of the present invention relates to the method and the kit for the isolation of DNA from the DNA originating materials. It is a method and kit for the isolation of high quality DNA typically bacterial DNA from the sample. The DNA is extracted by mixing the starting material with a decontaminating solution for the release of nucleic acid from the starting material.<br>
The decontaminated sample then lysed by chaotropic salt GuHCl and proteinase K enzyme to release total nucleic acids. In the presence of a chaotropic salt, nucleic acid binds selectively to glass fiber fleece in a special spin column fitted in centrifuge tube. The nucleic acid remains bound while a series of rapid wash and spin steps remove contaminating cellular components. Finally low salt elution removes the nucleic acid from the glass fiber fleece. The process does not require nucleic acid precipitation, organic solvent extractions or extensive handling of the nucleic acid which causes false positivity in down stream process.<br>
The method includes mixing the starting material with a lysing and denaturing substance to release DNA. For this the sample is mixed with the lysis Buffer ( Buffer 1), proteinase K solution. The sample is vortexed and incubated periodically for 40 minutes at 60° C and 70°C on heat block. To the incubated sample we add chilled ethanol and then the whole material is transferred to the spin column device which is set in the eppendoff. The<br>
spin device is containing trapping membrane of the present invention and centrifuge the same for two minutes at 8000rpm. The trapping membrane is incorporated in to a housing that fits within a micro centrifuge tube. Wash the column with wash buffer twice and centrifuge the same at 8000rpm for 2 minutes. Centrifuge the column again for 10000 rpm for 2 minutes to remove ethanol residue. Now transfer the column in a new eppendof. Add 80 µl to l00µl prewarmed double distilled water. Stand the column for 5 minutes at room temperature and centrifuge for 2 minutes at l0000rpm. Collect the filtrate, which is the pure bacterial DNA.<br>
Procedure for the preparation of the Bacterial Genomic DNA<br>
1.	Take 200 ul specimen test decontaminated sample into a 1.5 ml<br>
sterile microcetrifuge tube.<br>
2.	Add 20µl Proteinase K and 200µl Buffer -1 in to the sample, vortex<br>
it.<br>
3.	Incubate   at 60°C for 20 minutes and vortex the sample every 3-5<br>
minutes during incubation. Adjust the dry bath temperature to 70°C<br>
and incubate the sample tube for 20 minutes.<br>
4.	Add 210 µl of prechilled absolute ethanol to the sample tube and mix<br>
by vortexing.<br>
5.	Place the spin column on to the collection tube. Transfer the<br>
incubated solution in to the spin column and centrifuge it for 2<br>
minutes at 8000rpm.<br>
6.	Discard the collection tube with the filtrate and place the column on<br>
to a new collection tube.<br>
7.	Wash the column twice with 0.5 ml buffer-2 solution by centrifuging<br>
at  8000rpm  for  2  minutes.   Discard  the   flow-  through  after<br>
centrifugation.<br>
8.	Centrifuge the column at 14000 rpm for another 2 minutes to remove<br>
the ethanol residue.<br>
9.	Place the column on to a new 1.5 ml sterile tube. Add 70µl of<br>
preheated double distilled water.<br>
10.Stand the column for 1-5 minutes at room temperature and centrifuge<br>
for 2 minutes at 10000 to elute DNA. 11. Store the DNA at 2-8 °C if it is to be used within 24 hrs. or at -20 °C<br>
for long term storage.<br>
In an embodiment of the present invention, the lysing and denaturing substance is a buffer-1 solution which consists 45- 50mM TRIS HC1, 6-7(M) GU HC1( Guanidinium hydrochloride), 0.5 - Img/ ml lysozyme, and 0.025 - 0.05 mM EDTA (the pH of which is adjusted to 7-7.5) and a proteinase K. Proteinase K is an enzyme to break down the proteinacious material in the specimen sample.<br>
Amplification<br>
The amplification involves the process of treating the sample with the oligonucleotide primers for each strand of each different specific sequence being amplified under conditions such that or each strand of each different sequence being amplified. An extension product of each primer is<br>
synthesized which is complementary to each nucleic acid strand wherein said primer is selected so as to be substantially complementary to each strand of each sequence.<br>
It consists of treating the sample under denaturing conditions to separate the primer extension products from their templates if the sequences to be detected are present.<br>
It involves the process of treating the sample with oligonucleotide primers such that a primer extension product is synthesized using single stranded DNA resulting in the amplification of DNA if present. Detection of the amplified DNA is another process it involves. Agarose gel electrophoresis of the amplified DNA is to done for the detection of the TB bacilli present in the sample.<br>
PCR analysis or Nucleic acid amplification process consists of the following steps such as Denaturation, Annealing and Elongation which have been carried out in PCR machine.<br>
Denaturation<br>
The double stranded DNA has to be heated to 94- 96°C in order to separate the strands. This is called denaturing. In this step, the DNA breaks apart the hydrogen bonds that connects the two DNA strands. DNA is denatured for an extended time to ensure that both the template DNA have completely separated and are single stranded only.<br><br>
Annealing<br>
After separating the strands of DNA, the temperature is lowered so the primers present in the master mix can attach themselves to the single DNA strands. This step is called annealing. The temperature of this stage depends on the primers and is usually 5°C below their melting temperature. Usually this temperature is 45°C to 65°C. Oligonucleotide primers can form stable associations with denatured target DNA and serve as primers for the DNA polymerize.<br>
Elongation<br>
The DNA polymerase has to copy the DNA strands. It starts at the annealed primer and works its way along the DNA strand. This step is called Elongation. The synthesis of new DNA begins as the reaction temperature is raised to the optimum for the DNA polymerase. For the most thermo stable DNA polymerase, this temperature is in the range of 72 to 74°C.<br>
Each step of cycle should be optimized for each template and primer pair combinations. If the temperature during the annealing and extension steps are similar, these two steps can be combined in to a single step in which both primer annealing and extension takes place, After 20- 40 cycles, the amplified products may then be analyzed for size, quality and sequence.<br>
The long products thus formed will act as templates for one or another of the oligonucleotide primers during subsequent cycles and will produce molecules of the desired sequence. These molecules will also function as<br>
templates for one or the other of the oligonucleotide primers, producing further and thus a chain reaction can be sustained which will result in the accumulation of strands at an exponential rate relative to the number of cycles.<br>
Amplification kit:<br>
Amplification kit consists of Master mix (MMX), ready to use loading dye<br>
and positive control.<br>
The amplification is carried out in a thermal cycler. This is a machine that heats and cools the reaction tubes within it to the precise temperature required for each step of the reaction. This series of temperature and time adjustments is referred to as one cycle of amplification.<br>
25-30 µl of Master mix, 10-20 µl of sterile water and 15-25 µl of DNA:<br>
total   75 µl are taken in a 0.2ml PCR tube   and gently mix the sample<br>
together. Cap the tubes and place it on a tray map in a thermo cycler for<br>
amplification. Program the thermal cycler for the test as follows.<br>
Hold program: 5 minute for 90 - 95°C<br>
Cycle program (35 cycle) 1 minute 10 sec at 94°C; 1 minute- 1.55 minutes<br>
at 58°C -65 °C.<br>
1.50 minutes- 2-10 minute 72°C<br>
Hold program: 5-10 minutes 73°C- 75 °C<br>
Hold program: 4°C forever.<br>
In an embodiment of the present invention Master mix is prepared as<br>
follows.<br>
Single step PCR is being carried out in TB NAT process where in the<br>
master mix we include the following components.<br>
1.	Tris Buffer of concentration 5X to 20X<br>
2.	15mM to 50mM Magnesium chloride<br>
3.	1 mM to 3 mM d NTPS. ( Commercial Invitrogen)<br>
4.	10 µM to 230µM Primer (Oligonucleotide)( Commercial Invitrogen)<br>
5.	2 units to  5  units per µl Taq polymer enzyme( Commercial<br>
Invitrogen)<br>
6.	10-15 µl water per reaction as per the DNA concentration.<br>
These components are mixed together and vortex it to get a clear<br>
solution.<br>
Keep it at-20 °C.<br>
Buffer-2 solution is prepared as follows.<br>
Buffer-2 solution comprises of 150-200mM TRIS HC1 of pH 8.3 to 9.00, MgCl2 of 5-25 mM and 1-5 % Triton X. Mix these components together and sterile filtration is done by 0.2 µ filter paper. For long term storage, keep it at -20 °C.<br>
Detection<br>
The next process is the detection of the amplification product performed in post amplification area. All the genetic diseases may be detected by amplifying the DNA and analyzing the same.<br>
The amplification product is analyzed over agarose gel containing ethidium bromide by visualizing it on UV transiluminator or gel doc system.<br>
The diagnostic kit for the detection of the Mycobacterium tuberculosis DNA consists of the following reagents.<br>
1.	Preparation of 5x TBE Buffer<br>
Dissolve 4 g of TRIS base and 27.5g boric acid in 900ml deionized water. Add 20ml 0.5 M EDTA of pH 8.0 and make up the final volume to 1 liter to get 5X TBE Buffer. Store at room temperature.<br>
2.	Preparation of the agarose Gel<br>
1.	Seal the gel casting tray with adhesive tape. Place the well forming<br>
comb properly.<br>
2.	Boil 2.5 g of agarose    in 100ml of IX TBE buffer to form a<br>
homogeneous solution.<br>
3.	Add 0.5   µg of ethidium bromide per ml of agarose gel when the<br>
temperature of the solution is around 60°C. Pour the solution in the<br>
gel casting tray.<br>
4.	Gel  formation needs 30 min incubation at room temperature.<br>
Remove the tape carefully from the gel, casting tray and place the<br>
tray along with the gel in the tank of the electrophoretic assembly<br>
containing 1XTBE buffer containing ethidium bromide.<br>
5.	Mix 5µl of loading dye to 20 µ1 of the amplified product and load<br>
this mixture in the gel.<br>
6.	Load 25 µ1 of DNA ladder as DNA base pair maker.<br>
7.	Carry out the electrophoresis at 100V for 1 hr.<br>
8.	Visualize the PCR amplified product at size 240 base pair for MTB<br>
on an UV trans-illuminator using UV protective goggles and UV<br>
shield.<br>
Interpretation of Result<br>
1.	The sample is positive for MTB DNA if a band of 240 bp ( specific<br>
for MTB DNA) appears on the gel as shown in figure 1.<br>
2.	The sample is negative for MTB if there is no bands in this region as<br>
shown in figure 2.<br>
3.	If no bands appears at 240 bp region for the positive control, the<br>
complete run is invalid.<br>
4.	If band at 240 bp region appears in the negative control, the<br>
complete run is invalid.<br>
Primer<br>
Primer is an oligonucleotide which is complimentary to a template that hybridizes with the template to give a primer - template complex for the initiation of synthesis by DNA polymerase and which is extended by the addition of covalently bonded bases linked to its 3' end that are complimentary to the template. The result is a primer extension product.<br>
Example 1 Master Mix Take :<br>
1.	Tris Buffer of concentration 5X to 20X<br>
2.	15mM to 50mM Magnesium chloride<br>
3.	1 mM to 3 mM d NTPS. (Commercial Invitrogen)<br>
4.	10µM to 230µM Primer (Oligonucleotide)( Commercial Invitrogen)<br>
5.	2  units to  5  units per  µl Taq polymer enzyme( Commercial<br>
Invitrogen)<br>
6.	10-15µl water per reaction as per the DNA concentration.<br>
These components are mixed together and vortex it to get a clear solution. Keep it at -20 °C.<br>
Example 2<br>
Lysis Buffer solution:<br>
Lysing and denaturing substance is a buffer-1    solution which consists  of 45-   50mM  TRIS  HC1,  6-7(M)  GU  HC1(  Guanidinium<br>
hydrochloride), 0.5 - 1mg/ ml lysozyme, and 0.025 - 0.05 mM EDTA (the pH of which is adjusted to 7.5) and a proteinase K. Proteinase K is an enzyme to break down the proteinacious material in the specimen sample.<br><br><br><br>
I Claim<br>
1.	A process of amplifying a nucleic acid having Mycobactrium<br>
Tuberculosis base sequence in a sample comprising the steps of<br>
contacting the said nucleic acid under nucleic acid amplification<br>
conditions by:<br>
taking 25µ1-30µl of Master mix, 10µl -20µ1 of sterile water and<br>
15 µl-25 µl of DNA total 75 µl in a 0.2ml PCR tube and<br>
gently mixing the sample together;<br>
cap the tubes and place it on a tray map in a thermocycler for<br>
amplification;<br>
program the thermocycler for the test as follows;<br>
hold program: 5 minute for 90 - 95°C;<br>
cycle program (35 cycle) 1 minute 10 sec at 94°C; Iminute-<br>
1.55 minutes at 58°C -65 °C;<br>
1.50 minutes- 2-10 minute 72°C<br>
hold program: 5-10 minutes 73°C- 75 °C<br>
hold program: 4°C forever.<br>
2.	A process as claimed in claim 1 wherein the master mix is<br>
prepared as follows:<br>
a.	TRIS buffer of concentration 5X to 20X;<br>
b.	15mM to 50mM Magnesium chloride ;<br>
c.	ImMtolSmMdNTP;<br>
d.	10 µM to 230µM Primer;<br>
e.	2 units to 5 units per microlitre Taq polymer enzyme;<br><br>
f. 10-15µ1 water per reaction as per the DNA concentration where in the components are mixed together and vortexed to get a clear solution and keeping it at -20 °C.<br>
3.	A test kit for analyzing the presence of TB bacilli in a sample<br>
comprising decontamination solution, buffer 1( Lysis buffer)<br>
Buffer-2  (  Wash  Buffer),     Spin  Columns,  Proteinase  K,<br>
Collection tubes, Master Mix , centrifuging tris buffer, MgCl2,<br>
dNTPS, primer, Taq polymer, enzyme, MQ water and loading<br>
dye.<br>
4.	A test kit for analyzing the presence of tuberculosis bacilli in a<br>
sample as claimed in claim 3 where in the buffer 1 (lysis buffer)<br>
solution is made by mixing, 45- 50mM TRIS HC1, 6-7(M) GU<br>
HC1( Guanidinium hydrochloride), 0.5 - Img/ ml   lysozyme,<br>
and 0.025 - 0.05 mM EDTA (the pH of which is adjusted to<br>
7.5) and a proteinase K.<br>
5.	A test kit for analyzing the presence of tuberculosis bacilli as<br>
substantially as herein before described.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk3Mi1kZWwtMjAwNy1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1972-del-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk3Mi1ERUwtMjAwNy1DbGFpbXMtKDE3LTExLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">1972-DEL-2007-Claims-(17-11-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk3Mi1kZWwtMjAwNy1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">1972-del-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk3Mi1kZWwtMjAwNy1jb3JyZXNwb25kZW5jZS1vdGhlcnMgMS5wZGY=" target="_blank" style="word-wrap:break-word;">1972-del-2007-correspondence-others 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk3Mi1ERUwtMjAwNy1Db3JyZXNwb25kZW5jZS1PdGhlcnMtKDE3LTExLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">1972-DEL-2007-Correspondence-Others-(17-11-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk3Mi1kZWwtMjAwNy1jb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">1972-del-2007-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk3Mi1kZWwtMjAwNy1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">1972-del-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk3Mi1kZWwtMjAwNy1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">1972-del-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk3Mi1ERUwtMjAwNy1Gb3JtLTEtKDE3LTExLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">1972-DEL-2007-Form-1-(17-11-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk3Mi1kZWwtMjAwNy1mb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">1972-del-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk3Mi1kZWwtMjAwNy1mb3JtLTE4LnBkZg==" target="_blank" style="word-wrap:break-word;">1972-del-2007-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk3Mi1kZWwtMjAwNy1mb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">1972-del-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk3Mi1ERUwtMjAwNy1Gb3JtLTI2LSgxNy0xMS0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1972-DEL-2007-Form-26-(17-11-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk3Mi1kZWwtMjAwNy1mb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">1972-del-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk3Mi1kZWwtMjAwNy1mb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">1972-del-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk3Mi1kZWwtMjAwNy1mb3JtLTkucGRm" target="_blank" style="word-wrap:break-word;">1972-del-2007-form-9.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=UGF0ZW50LTIyNTcxNi1Qb3N0LUdyYW50LU9wcG9zaXRpb24tKDIyLTA3LTIwMTApLnBkZg==" target="_blank" style="word-wrap:break-word;">Patent-225716-Post-Grant-Opposition-(22-07-2010).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="225715-a-kit-for-analyzing-the-presence-of-igm-antibodies-of-mycobacterium-tuberculosis.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="225717-a-device-kit-for-analyzing-the-presence-of-tuberculosis-bacilli-in-a-sample.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>225716</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1972/DEL/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>48/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>28-Nov-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>21-Nov-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>17-Sep-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MAHAJAN; LALIT</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>N-118, GREATER KAILASH, PART-1, NEW DELHI, INDIA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>MAHAJAN; LALIT</td>
											<td>N-118, GREATER KAILASH, PART-1, NEW DELHI, INDIA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>G01N33/569</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/225716-a-process-and-the-kit-of-for-the-detection-of-mycobacterium-tuberculosis-bacilli-present-in-the-sample-by-nucleic-acid-amplification-test by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:08:37 GMT -->
</html>
