<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/226135-a-pathogen-detection-system-and-method-for-pathogen-analysis by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:53:06 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 226135:A PATHOGEN DETECTION SYSTEM AND METHOD FOR PATHOGEN ANALYSIS .</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A PATHOGEN DETECTION SYSTEM AND METHOD FOR PATHOGEN ANALYSIS .</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention discloses methods and apparatus for implementing microfluidic analysis devices are provided. A monolithic elastomer membrane associated with an integrated pneumatic manifold allows the placement and actuation of dense arrays of a variety of fluid control structures, such as structures for isolating, routing, merging, splitting, and storing volumes of fluid. The fluid control structures can be used to implement a pathogen detection analysis system including integrated immunoaffinity capture and analysis, such as polymerase chain reaction (PCR) and capillary electrophoresis (CE) analysis. An analyte solution can be input into the device and pumped through a series of immunoaffinity capture matrices in microfabricated chambers having antibodies targeted to the various classes of microbiological organisms such as bacteria, viruses and bacterial spores. The immunoaffinity chambers can capture, purify, and concentrate the target for further analysis steps.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>A PATHOGEN DETECTION SYSTEM AND METHOD FOR<br>
PATHOGEN ANALYSIS<br>
The invention relates to a pathogen detection system and method<br>
for pathogen analysis.<br>
BACKGROUND OF THE INVENTION<br>
The present invention relates to pathogen detection and analysis. In one<br>
example, the present invention provides sample preparation, processing, detection,<br>
and analysis systems implemented using microfluidic control structures. In another<br>
example, the present invention provides robuts techniques for making dense arrays<br>
of fluidic control elements for high throughput analysis applications.<br>
Conventional mechanisms for microfluidic analysis are limited. Some<br>
available mechanisms include single channel separation devices and multiple<br>
channel separation devices. Others include analyzers that integrate some sample<br>
preparation and analysis steps. However, many microfluidic analysis devices that<br>
include fluidic control capabilities are chemically or physically incompatible with<br>
many chemical or biochemical assays. In addition, many microfluidic control<br>
elements are difficult to fabricate in dense arrays because of limitations in the<br>
fabrication process, robustness, and/or design. Many conventional devices require<br>
constant actuation to maintain fluidic control. A microfluidic device utilizing such<br>
valves can not be removed from its control system without losing control of the<br>
fluidic contents of the device. In addition, many techniques and mechanisms for<br>
microfluidic analysis furthermore lack sensitivitity, specificity, or quantitative<br>
analysis capabilities. In particular, conventional microfluidic analysis mechanisms<br>
lack the functionality and capabilities to efficiently implement sample preparation<br>
for systems such as pathogen detectors and analyzers.<br>
It is therefore desirable to provide improved methods and apparatus for<br>
implementing microfluidic control mechanisms such as valves, pumps, routers,<br>
reactors, etc. to allow effective integration of sample introduction, preparation<br>
processing, and analysis capabilities in a microfluidic device. In one example, it is<br><br>
desirable to provide microfluidic devices having microfabrication effiencies that can be used to<br>
implement both single channel and array based systems that can be used as pathogen detectors and<br>
analyzers that provide few false positives, high throughput and inexpensive continuous monitoring.<br>
U.S. Patent No. 6,379,929 (Burns et al.) discloses a method for isothermal amplification of a<br>
selected nucleic acid in a microfabricated substrate. The method includes providing a sample<br>
comprising a selected nucleic acid and conducting an isothermal amplification reaction, to amply the<br>
selected nucleic acid.<br>
Wooley et al. Analytical Chemistry. 23 December 1996, Vol.68, pages 4081-4086 discloses<br>
an integrated PCR amplification and capillary electrophoresis microfabricated device for high speed<br>
DNA analysis.<br>
U.S. Patent No. 6,605,454 (Barenburg et al.) discloses a method for lysing cells in a cavity of<br>
a microfludic device. The method includes providing microwave radiation at a predetermined<br>
frequency, predetermine wavelength or predetermined intensity.<br>
U.S. Patent Publication No.2004/0086872 (Childers et al.) discloses an apparatus for the<br>
analysis of a nucleic acid in a sample and method of making a catridge for microfludic analysis of a<br>
nucleic acid in a sample. The microfludic apparatus includes a fluid-handling portion and an assay<br>
portion. The method includes forming a fluid-handling portion, fabricating an assay portion and<br>
attaching the assay portion to the fluid-handling portion.<br><br>
Summary of the Invention<br>
Methods and apparatus for implementing microfluidic analysis devices are<br>
provided. A monolithic elastomer membrane associated with an integrated<br>
pneumatic manifold allows the placement and actuation of dense arrays of a variety<br>
of fluid control structures, such as structures for isolating, routing, merging,<br>
splitting, and storing volumes of fluid. The fluid control structures can be used to<br>
implement a pathogen detection and analysis system including integrated<br>
immunoaffinity capture and analysis, such as polymerase chain reaction (PCR) and<br>
capillary electrophoresis (CE) analysis. An analyte solution can be input into the<br>
device and pumped through a series of immunoaffinity capture matrices in<br>
microfabricated chambers having antibodies targeted to the various classes of<br>
microbiological organisms such as bacteria, viruses and bacterial spores. The<br>
immunoaffinity chambers can capture, purify, and concentrate the target for further<br>
analysis steps.<br>
In one embodiment, a a pathogen detection system is provided. The system<br>
includes an immunocapture chamber integrated on a microfluidic device. The<br>
immunocapture chamber is operable to capture a target provided to the<br>
immunocapture chamber through a microfluidic channel. The system also includes<br>
a DNA analysis mechanism associated with the immunocapture chamber. The<br>
DNA analysis mechanism is integrated on the microfluidic device. The DNA<br>
analysis mechanism is operable to perform DNA analysis on the target.<br>
In another embodiment, a pathogen detection system on a monolithic device<br>
is provided. The system includes a plurality of immunocapture chambers integrated<br>
on the monolithic device. The immunocapture chambers are operable to capture a<br>
target provided to the immunocapture chambers through microfluidic channels. The<br>
system also includes a plurality of DNA analysis mechanisms associated with the<br>
immunocapture chambers. The plurality of DNA analysis mechanisms are<br>
integrated on the monolithic device. The plurality of DNA analysis mechanisms are<br>
operable to perform DNA analysis on the target.<br><br>
In another embodiment, a method for pathogen analysis is provided. A fluid<br>
analyte is provided to a plurality of immunocapture chambers through microfluidic<br>
channels integrated on a monolithic device. A target associated with the fluid<br>
analyte is captured at the immunocapture chambers. DNA analysis is performed on<br>
the target using a plurality of DNA analysis mechanisms associated with the<br>
plurality of immunocapture chambers. The plurality of DNA analysis mechanisms<br>
are integrated on the monolithic device.<br>
These and other features and advantages of the present invention will be<br>
presented in more detail in the following specification of the invention and the<br>
accompanying figures, which illustrate by way of example the principles of the<br>
invention.<br>
Brief Description of the Drawings<br>
The invention may best be understood by reference to the following<br>
description taken in conjunction with the accompanying drawings, that illustrate<br>
specific embodiments of the present invention.<br>
Figures 1A-1E are diagrammatic representations showing mechanisms on a<br>
microfluidic device suitable for implementing the techniques of the present<br>
invention.<br>
Figures 2A and 2B are diagrammatic representation depicting a diaphragm pump.<br>
Figure 3 is a plan view of a diagrammatic representation showing a fiuidic<br>
router.<br>
Figure 4 is a plan view of a diagrammatic representation depicting a mixing<br>
loop.<br>
Figures 5A-5D are diagrammatic representations showing a fluid reservoir.<br>
Figure 6 is a diagrammatic representation showing bus valves.<br>
Figure 7 is a diagrammatic representation of a pathogen detection system.<br>
Figures 8 is a diagrammatic representation depicting immunoaffinity capture<br>
valve mechanisms.<br>
Figures 9 is a diagrammatic representation showing immunoaffinity capture<br>
valve mechanisms.<br>
Figure 10A and 10B are diagrammatic representations showing capture and<br>
routing of analytes for immunoaffinity capture.<br><br>
Figure 11 is a diagrammatic representation showing PCR and CE that can be<br>
integrated with immunoaffinity capture.<br>
Figure 12 is a diagrammatic representation of a combined immunocapture<br>
and PCR chamber.<br>
Figure 13A is a diagrammatic representation of a pathogen detection system.<br>
Figure I3B is a diagrammatic representation showing microfabrication<br>
stages.<br>
Figure 14 is a diagrammatic representation of a radial array of pathogen<br>
detection systems.<br>
Detailed Description of Specific Embodiments<br>
Reference will now be made in detail to some specific embodiments of the<br>
invention including the best modes contemplated by the inventors for carrying out<br>
the invention. Examples of these specific embodiments are illustrated in the<br>
accompanying drawings. While the invention is described in conjunction with these<br>
specific embodiments, it will be understood that it is not intended to limit the<br>
invention to the described embodiments. On the contrary, it is intended to cover<br>
alternatives, modifications, and equivalents as may be included within the spirit and<br>
scope of the invention as defined by the appended claims. For example, the<br>
techniques of the present invention will be described in the context of glass<br>
microfluidic devices, although other devices such as plastic devices could also be<br>
used.<br>
It should be noted that the fluid control structures suitable for use in glass<br>
microfluidic devices can be applied to a variety of microfluidic devices. A pathogen<br>
detection system is a good example of one possible application that can benefit from<br>
the use of fluid control structures. In the following description, numerous specific<br>
details are set forth in order to provide a thorough understanding of the present<br>
invention. The present invention may be practiced without some or all of these<br>
specific details. In other instances, well known process operations have not been<br>
described in detail in order not to unnecessarily obscure the present invention.<br>
The field of microfluidic analysis technology has evolved rapidly from the<br>
earliest single channel separation devices. Some devices include multichannel<br>
separation devices for high throughput analysis and analyzers that integrate sine<br>
sample preparation and analysis on a single chip. Devices that combine both<br><br>
multichannel analysis and integrated sample preparation are capable of reducing the<br>
amount of resources and cost needed to perform a variety of assays. An illustrative<br>
example may be found in the field of genomics: integration of sequencing sample<br>
preparation, purification, and electrophoretic analysis in a single device translates<br>
into decreases in assay time and cost and increased assay throughput efficiency and<br>
robustness. In all cases, a high level of integration in a microfluidic device requires<br>
a robust on chip mechanism for isolating, routing, merging, splitting, and storing<br>
volumes of fluid.<br>
Some valve technologies for use in silicon, glass silicon, polymer, and<br>
elastomer microfluidic devices have addressed these requirements in a limited<br>
manner. However, many of these technologies are chemically or physically<br>
incompatible with many chemical or biochemical assays. Furthermore, many<br>
technologies lack the variety of robust surface modification chemistries available for<br>
glass microfluidic devices. In addition, individual microfluidic valves are typically<br>
fabricated with separate membranes normally held open. Having valves normally<br>
open requires constant actuation to maintain fluidic control. A microfluidic device<br>
utilizing such valves cannot be removed from a control system without losing<br>
control of the fluidic contents of the device. Furthermore, some typical devices use<br>
individually placed latex membranes. Individually placed pneumatically actuated<br>
latex membranes haven been developed but this fabrication method prevents large<br>
scale integration into multichannel, high throughput analysis devices.<br>
Other microfluidic devices are fabricated using anodically bonded silicon<br>
and glass wafers and actuated piezoelectrically. However, the electrical<br>
conductivity and chemical compatibility of silicon complicates use in analytical<br>
devices. Thin films bonded to or deposited on silicon can only partially mitigate the<br>
electrical conductivity and chemical compatibility.<br>
Elastomer devices have also been demonstrated. However, the<br>
hydrophobicity and porosity of elastomeric materials render elastomeric devices<br>
incompatible with many chemical and biochemical assays. It is thus desirable to<br>
minimize the fluidic contact with elastomer surfaces. Complex fabrication,<br>
chemical compatibility, unreliable fluid manipulation and other problems have made<br>
existing fluidic manipulation technologies inadequate for integration into large-<br>
scale, high-throughput lab-on-a-chip devices.<br><br>
Consequently, the techniques and mechanisms of the present invention<br>
provide a monolithic membrane valve structure suitable for high density integration<br>
into microfluidic devices. A variety of fluid control structures based on the<br>
monolithic membrane valve structures are also provided.<br>
A microfluidic device having a monolithic membrane is one example of a<br>
particularly suitable device for implementing a pathogen detection system on a chip.<br>
According to various embodiments, the pathogen detection system includes<br>
immunocapture and DNA analysis mechanisms such as polymerase chain reaction<br>
(PCR), and capillary electrophoresis (CE) mechanisms. In one example, the<br>
pathogen detection system can be implemented on a glass microfluidic device<br>
having a variety of fluidic control structures.<br>
Figures 1A-1E are diagrammatic representations of monolithic membrane<br>
valves that can be implemented on a glass microfluidic device. Figure 1A is a top<br>
view of a diagrammatic representation of a monolithic membrane valve. Figure 1B<br>
is a side view of a diagrammatic representation of a three-layer device having the<br>
valve. Figure 1C is a side view of a diagrammatic representation of a four-layer<br>
device having the valve. Figure 1D is a side view of a diagrammatic representation<br>
of an open valve of a three-layer device. Figure 1E is a side view of aidagrammatic<br>
representation of an open valve four-layer device. According to various<br>
embodiments shown in Figure 1A and 1B, a three-layer glass microfluidic device<br>
includes an elastomer membrane 111 sandwiched between two glass wafers 101 and<br>
105. In one example, the elastomer membrane is a polydimethysiloxane (PDMS)<br>
membrane available as 254 um thick HT-6135 and HT-6240 membranes from Bisco<br>
Silicons of Elk Grove, IL. Other flexible membranes can also be used. The<br>
elastomer membrane 111 makes a reversible but strong bond between the wafers.<br>
A fluidic channel 103 is etched in the wafers prior to bonding and is used to<br>
carry fluids. A manifold channel 107 and a valve area 109 are similarly etched to<br>
carry air or other working fluid under pressure or vacuum to actuate the valves.<br>
Typically, the pneumatic channels 107 and 109 are located on one wafer 105, herein<br>
referred to as the pneumatic wafer, and the fluidic channels are etched on the second<br>
wafer 101, herein referred to as the fluidic wafer. These etched channel features can<br>
directly contact the membrane and form a hybrid glass/elastomer channel as shown<br>
in Figure 1B.<br><br>
Alternatively, the membrane can be between a thermally bonded all-glass<br>
fluidic wafer sandwich (XY) and the pneumatic wafer 159 as shown in the four-<br>
layer device 150 of Figure 1C. Having an all glass channel allows a device to<br>
benefit from the favorable physical and chemical properties of glass. Any layer<br>
having favorable physical and chemical properties is referred to as a chemically<br>
inactive layer. The chemically inactive layer can be used to fabricated XY. In one<br>
example, the sandwich of 151 and 155 that constitutes XY is made of glass.<br>
An example of a four layer devices includes a fluidic wafer 151 thermally<br>
bonded to a via wafer 155. Via holes with small diameters are placed at the<br>
discontinuity in the fluidic channel 153. The elastomer membrane 157 is affixed to<br>
the via wafer 155 side of the fluidic/via wafer sandwich XZ. Valve deflection<br>
chambers 161 are etched in the manifold wafer 159 and bonded to the membrane<br>
157, completing the 4-layer device 150. In this way, fluidic channel 153 retain san<br>
all-glass chemically favorable configuration while allowing implementation of the<br>
large-scale integrated fluidic control structures. In some embodiments, the four<br>
layer device shown in Figure 1C provides substantial benefits over a three layer<br>
device as it minimizes contact between a sample and an elastomer membrane.<br>
According to various embodiments, the various fluid control components<br>
within the monolithic membrane device are actuated by applying pressure or<br>
vacuum to holes on the pneumatic wafer. Any single membrane is referred to<br>
herein as a monolithic membrane. Any single device with a monolithic membrane<br>
is referred to herein as a monolithic device. Mechanisms for supplying pressure or<br>
vacuum to etched channels associated with a pneumatic wafer are herein referred to<br>
as ports or pneumatic ports. In a three layer device, etched channels in the<br>
pneumatic wafer distribute the actuation vacuum to valve region 109 of the<br>
elastomer membrane 111. Vacuum applied via the manifold channel at valve area<br>
region 109 pulls the membrane away from the channel discontinuity, providing a<br>
path for fluid flow across the discontinuity and thus opening the valve as shown in<br>
Figure 1D. Valves that can be opened or closed using pneumatic pressure are herein<br>
referred to as switchable valves or pneumatically switchable valves.<br>
Applying pneumatic pressure includes either applying pressure or applying a<br>
vacuum. The membrane 157 consequently can modulate the flow of fluid in the<br>
adjacent fluid channel as shown in Figure 1D. In Figure 1D, a vacuum is applied to<br><br>
valve area 109 through etched channels associated with pneumatic wafer 105 to<br>
open fluidic channel 103. When vacuum pressure or suction is no longer applied to<br>
valve area 109, the membrane 111 closes the fluidic channel 103 as shown in Figure<br>
1B. Figure IE shows a four layer device. The four layer device includes a channel<br>
layer 151, channel 153, via layer 155, membrane layer 157, and pneumatic layer<br>
159. As noted above, the four layer device provides substantial benefits over a three<br>
layer device as it minimizes contact between a sample and an elastomer membrane<br>
in some cases to only a valve area 161.<br>
It should be noted that the structures shown can be oriented in any direction.<br>
In some examples, valves can be inverted on a device. A pneumatic layer can be<br>
above or below a fluidic layer. The techniques of the present invention allow a<br>
variety of orientations, as gravity does not adversely affect the membrane valves.<br>
The fluidic control structures provide a variety of benefits. For example, the<br>
monolithic membrane valves are normally closed valves, meaning that the valves<br>
remain closed even when the device is disconnected from the actuation pressure<br>
source. Existing normally open microfluidic valves require constant actuation to<br>
maintain control of the fluidic contents of the device. Furthermore, unlike shape<br>
memory alloy structures, both the closed and open temperatures of the valve<br>
structures are at ambient temperature, facilitating work with aqueous biological<br>
fluids.<br>
In many typical implementations, a number of interfaces between the<br>
microfluidic device are needed in order to manipulate various fluidic control<br>
mechanisms. However, according to various embodiments of the present invention,<br>
multiple regions of a membrane can be actuated in parallel by connecting their<br>
pneumatic control channels. In one example, a series of valves can be controlled<br>
using a single pneumatic port. Consequently, a significant number of valves can be<br>
controlled using only a limited number of external interfaces or pneumatic ports.<br>
This simplifies implementation and minimizes problems interfacing with the device.<br>
According to various embodiments, controlling valves in this manner allows<br>
massively parallel pneumatic actuation of a monolithic membrane for operating<br>
valves, pumps, reservoirs, routers, and other fluid control structures within the<br>
device.<br><br>
The membrane valves can be used to form a variety of fluidic control<br>
mechanisms. Figures 2A and 2B are diagrammatic representations of a pump<br>
formed using membrane valves. According to various embodiments shown in<br>
Figures 2A and 2B, three valves placed in series form a diaphragm pump 210.<br>
Pumping is achieved by actuating the valves according to a five step cycle. Figure<br>
2A shows a top view of a three-layer monolithic membrane diaphragm pump.<br>
Figure 2B shows a side view of the three-layer monolithic membrane diaphragm<br>
pump. The diaphragm pump includes an input valve 201, a diaphragm valve 203,<br>
and an output valve 205. It should be noted that the diaphragm pump can operate in<br>
either direction and the designations of the input valve and output valve are<br>
arbitrary. The pump includes a fluidic layer 209 having etched fluidic channels 211,<br>
a monolithic membrane 207, and a manifold layer 213. The air tight nature of the<br>
valves makes the pumps self-priming and capable of pumping air in addition to<br>
other gases and fluids.<br>
According to various embodiments, pumping can be performed in a series of<br>
stages. In a first stage, output valve 205 is closed and an input valve 201 is opened.<br>
In a second stage, a diaphragm valve 203 is opened. In a third stage, the input valve<br>
201 is closed. In a fourth stage, the output valve 205 is opened. In a fifth stage, the<br>
diaphragm valve 203 is closed, pumping analyte fluid through the open output valve<br>
205.<br>
The volume pumped per cycle is determined by the volume contained within<br>
the open diaphragm valve, a volume that, in turn, determined by the size of the<br>
pneumatic chamber in the diaphragm valve. Therefore, pumps designed for<br>
metering known nanoliter to microliter scale volumes of fluid can be fabricated by<br>
modulating the size of the diaphragm valve pneumatic chamber. The diaphragm<br>
pumps are self-priming and can pump fluids forward or backward by reversing the<br>
actuation cycle. It should also be noted that the valve seat where the membrane<br>
contacts the glass sealing surface may be etched to have ridges or other surface<br>
modifications to control the adhesion of the membrane to the glass surface.<br>
Monolithic valves can also be used to form routers, mergers, and splitters. It<br>
should be noted that although the following structures will be described in the<br>
context of three layer structures, the structures can also be implemented using four<br>
or more layers. Figure 3 is a diagrammatic representation of a router 300. The<br><br>
router includes valves 301, 303, 305, and 317; pneumatic channels 331, 333, 335,<br>
337, and 339; fluidic channels 321, 323, 325, and 327; and a diaphragm valve 309.<br>
The router pumps fluid from any input to any output depending upon which of the<br>
input/output valves are actuated at what point during the pumping cycle. Actuating<br>
two or more input valves simultaneously merges several different fluid streams into<br>
one stream at the output valve. Conversely, actuating two or more output valves<br>
simultaneously splits a single fluid stream into several different streams at the<br>
output valves.<br>
For example, to route fluid from channel 327 to channel 321, valves 301 and<br>
305 are held closed. Valves 317, 309, and 303 can then be used as a pump as noted<br>
above. The router includes functionality to merge and split fluid channels. To<br>
merge fluid from channels 325 and 327 into channel 323, valve 303 is held closed.<br>
To split fluid from channel 321 to channels 323 and 327, valve 301 is held closed.<br>
In yet another example, to route a fluid introduced through channel 327 to channel<br>
325, valves 303 and 305 are held closed. Valves 317 and 301 can be opened to<br>
allow flow of a fluid through channel 327 to channel 325. A variety of<br>
arrangements are possible.<br>
A mixing loop can also be formed using monolithic valves. In one example,<br>
mixing can be performed by moving a fluid between two areas of a device. Mixing<br>
can be used for performing all types of on-chip operations. Figure 4 is a<br>
diagrammatic representation of a mixing loop 400. The mixing loop or mixer<br>
includes valves 401, 403, and 405; fluidic channels 411, 413, and 415; and<br>
pneumatic channels 421, 423, and 425. Additional valved channels connect to the<br>
loop and provide fluidic access to or from the mixer. Two or more volumes of<br>
fluids can be admitted into the mixer loop through channels 413 and 415 and<br>
pumped, as noted above, in a circle until the fluids are mixed by diffusion. The<br>
mixture can then be pumped out of the mixer loop. Mixing can also be<br>
accomplished by moving a fluid back and forth between two reservoirs.<br>
Figures 5A-5C are diagrammatic representations of a reservoir 500. Figure<br>
5A is a top view of a reservoir with an etched displacement chamber. Figure 5B is a<br>
side view of the reservoir. Figure 5C is a side view showing a filled reservoir.<br>
Figure 5D is a side view of a large-volume reservoir with drilled displacement<br>
chamber and pump for autonomous filling/dispensing. The reservoir is included on<br><br>
a pneumatic wafer 513 sandwiching a membrane 505 with a fluidic wafer 511. The<br>
reservoir can be filled or emptied through channel 501. According to various<br>
embodiments, an open monolithic membrane valve in valve area 503 functions as a<br>
reservoir for on chip fluid storage. The size of the chamber in the pneumatic wafer<br>
513 determines the volume of fluid stored inside the reservoir; applying vacuum<br>
fills the reservoir and applying pressure empties it.<br>
According to various embodiments, reservoirs for storing large volumes of<br>
fluid can be fabricated by replacing the etched pneumatic chamber with a drilled<br>
hole and applying actuation pressure or vacuum directly to the hole. Alternately, a<br>
reservoir without a direct pneumatic connection can be fabricated by connecting the<br>
reservoir to a diaphragm pump. Figure 5D shows a reservoir 503 connected to a<br>
pump. The reservoir is filled or emptied depending upon the direction of pumping<br>
and has the advantage of variable volume. In one example, pumps such as valves<br>
531, 533, and 535 and be used to fill or dispense fluid for reservoir 503.<br>
A monolithic membrane reservoir with one or more fluidic inputs functions<br>
as an on-chip reactor. Like the reservoir, the reactor can draw in reactants and expel<br>
products directly by using direct pressure or vacuum applied through the pneumatic<br>
manifold wafer. Alternatively, the reactor can draw in reactants and expel products<br>
indirectly using an integrated pump, mixer, and/or router structures. According to<br>
various embodiments, since the volume of the reactor is defined by the size of the<br>
chamber 503 in the pneumatic wafer, reactors with arbitrary volumes can be<br>
included at any point on a device without drastically changing the layout of<br>
structures in the fluidic wafer. Also, the reactor can be partially filled as necessary<br>
for on chip reactions that involve a variable volume of reactants.<br>
Most elastomer membranes are gas permeable, and this property has thus far<br>
been used to simplify fluidic filling of all elastomer devices.<br>
According to various embodiments, the gas permeability of the membrane<br>
can eliminate bubbles and air pockets. When applying an actuating vacuum to a<br>
monolithic membrane reactor, or other fluidic structure, bubbles can be eliminated<br>
from reactions that produce gas. For example, the gas permeable membrane can<br>
reduce bubbles that can form during on chip thermal cycling of PCR reactants that<br>
could result in loss of containment of the reaction mixture.<br><br>
A complex microfluidic device may include several independent modules<br>
connected to a fluidic bus. In one example, it may be useful to provide an analyte<br>
fluid to multiple different fluidic channels. In another example, a variety of<br>
reagents can be available for introduction into a microfluidic device. Figure 6 is a<br>
diagrammatic representation of a bus valve 600 that can be used to distribute an<br>
analyte fluid. The bus valve valve 600 includes valves 601, 603, 605, and 607 that<br>
are designed to route fluids from a fluidic bus channel 611 to fluidic channels 621,<br>
623, 625, and 627. Pneumatic channels 631, 633, 635, and 637 manage the valves<br>
for controlling distribution of the fluid. Typical bus valve implementations have<br>
dead volume on the bus side. Dead volume makes it difficult to rinse the bus<br>
completely between fluid routing operations. According to various embodiments,<br>
the techniques of the present invention provide bus valves with little or no dead<br>
volume on the bus side. This allows the bus to be rinsed completely between fluid<br>
routing operations and prevents mixing or cross contamination between different<br>
fluids during device operation.<br>
The microfluidic device mechanisms can be fabricated using a variety of<br>
technologies. According to various embodiments, channel features are etched into<br>
glass wafers, for example, using standard wet chemical etching. Glass wafers<br>
(1.1mm thick, 100mm diameter) are piranha cleaned (20:1) and coated with a<br>
sacrificial (200nm) polysilicon etch mask layer using an LPCVD furnace or<br>
sputtering system. Borofloat glass wafers or Schott D263 borosilicate glass wafers<br>
are used for devices with the three-layer or four-layer design. After polysilicon<br>
deposition, the wafers are spin-coated with positive photoresist, soft-baked, and<br>
patterned using a contact aligner. UV-exposed regions of photoresist are removed<br>
in Microposit developer. The exposed regions of polysilicon are removed by<br>
etching in SF6 plasma. The wafers are etched isotropically at 7 um/min in HF<br>
solution (49% HF for the Borofloat wafers and 1:1:2 HF:HCI:H2O for the D263<br>
wafers) until the desired etch depth is reached.<br>
According to various embodiments, the fluidic channel wafers are etched<br>
20um deep for the three-layer devices and 40um deep for the four-layer devices.<br>
The manifold wafers are etched 70um deep for the three-layer devices and drilled at<br>
valve locations for the four-layer devices. The remaining photoresist and<br>
polysilicon is then stripped from the wafers using PRS-3000 and SF plasma,<br><br>
respectively. Access holes through the fluidic and manifold wafers are drilled and<br>
the wafers are again piranha cleaned.<br>
In some examples, devices utilizing the three-layer design are assembled by<br>
applying a PDMS membrane (254 um thick HT-6135 and HT-6240, Bisco<br>
Silicones, Elk Grove, EL) over the etched features in the fluidic channel wafer and<br>
pressing the manifold hybrid glass-PDMS fluidic channels with valves located<br>
wherever a drilled or etched displacement chamber on the manifold was oriented<br>
directly across the PDMS membrane from a valve seat. Devices utilizing the four-<br>
layer design are assembled by first thermally bonding the fluidic channel wafer to a<br>
210 um thick D263 via wafer with pairs of 254 um diameter drilled via holes<br>
positioned to correspond to the locations of channel gaps. The fluidic channel and<br>
via wafers are bonded by heating at 570 C for 3.5 h in a vacuum furnace (J.M. Ney,<br>
Yucaipa, CA). The resulting two-layer structure containing all-glass channels is<br>
then bonded to the PDMS membrane and the manifold wafer. The glass-PDMS<br>
bonds formed in this manner are reversible but still strong enough to survive the<br>
range of vacuum and pressures exerted on the device. Optionally, an irreversible<br>
glass-PDMS bond are obtained by cleaning the manifold wafer and PDMS<br>
membrane in a UV ozone cleaner (Jelight Company Inc., Irvine, CA) prior to<br>
assembly.<br>
The microfluidic device mechanisms described above can be used to<br>
implement a variety of devices. The features including valves and pumps can be<br>
flexibly arranged to provide multi-channel lab-on-a-chip instruments that are able to<br>
integrate sample preparation and analysis steps into a single device. The<br>
microfluidic platform is particularly well-suited as one device capable of<br>
implementing an integrated pathogen detection system.<br>
Conventional rapid pathogen detection systems use detection employing<br>
either Enzyme Linked Immunosorbent Assays (ELISA) or Fluorescence<br>
Immunoassays (FIA). Typically, detection involves the immobilization of an<br>
analyte specific antibody, incubation with the sample solution, and recognition with<br>
a sandwich antibody linked to an enzyme or fluorophore followed by development<br>
and detection. Immunofluorescence detection assays have also been used.<br>
However, detection limits associated with each of these assays are relatively<br>
restrictive.<br><br>
The use of various formats of PCR based genetic detection and typing is also<br>
popular because of its high specificity and gain. However, even though DNA based<br>
PCR approaches are powerful, they will respond positively to both viable and<br>
nonviable pathogens, potentially producing false positives. Detection of RNA<br>
targets may be thus preferred because its rapid degradation means that live targets<br>
are required for detection.<br>
A variety of alternative detection methods have also been proposed. Mass<br>
spectrometry methods have been developed to detect pathogens, spores, and other<br>
bioagents, by detection of neutral lipids, polar lipids and spore specific biomarkers.<br>
However, though the speed, throughput and portability of the mass spectrometry<br>
approach is not obvious and the specificity is unproven.<br>
The detection of spores, for example anthrax, from soil, air, etc. is<br>
challenging because it is highly infective (an inhaled dose of 10,000 spores can be<br>
achieved in 10 minutes at 10 spores/L). The most advanced detection concept uses<br>
real time detection of PCR products performed in a silicon microreactor with thin<br>
film heaters and integral fluorescence excitation and detection. This system has<br>
subsequently been extended to a ten channel Advanced Nucleic Acid Analyzer<br>
(ANAA) as well as a portable version. Versions of this system are also being<br>
developed for the military and for the Post Office. A GeneXpert sample preparation<br>
system with integrated (multimicroliter) sample processing for real time PCR<br>
analysis is also being developed.<br>
The development of portable analyzers that can rapidly perform automated<br>
and complex up front chemistries and quantitate pathogen concentrations and<br>
antibiotic resistance would be a major step forward. Similarly the ability to detect<br>
and type large numbers of samples rapidly and with very low false positives in a<br>
high throughput, multisample screening application would also be useful when large<br>
numbers of samples or potentially infected individuals need to be screened, Steps<br>
toward such automated clinical analyzers have been made. In one example,<br>
complex microfluidic circuit systems for blood clinical analyses that are essentially<br>
micro versions of the common autoanalyzer have been developed. A fully<br>
integrated analyzer (microliter volume scale) that was used for preparation of<br>
samples from blood for HIV analysis on microarrays has also been developed. This<br>
system performed a complex assay including a large number of nucleic acid steps<br><br>
and exploited the &gt;100 nL dead volume pneumatic membrane valves which will be<br>
discussed in more detail below.<br>
A lucite microfluidics cube has been developed for controlling the flow of<br>
solutions over six different immunoarray sensors that provide fluid control with a<br>
simple pressure relief system to facilitate the performance of their immunoassay<br>
with small portable systems. This format has been developed as the Raptor portable<br>
analyzer that uses integrated flow systems and fiber optic biosensor capillaries to<br>
analyze four different agents in a ten minute operation. The unique characteristics<br>
of addressable arrays have been recognized to develop an integrated stacked<br>
microlaboratory that performs automated electric field driven immunocapture and<br>
DNA hybridization array analysis. For example, following immunocapture the<br>
bacteria were released for strand displacement amplification (SDA) followed by<br>
hybridization analysis of the amplified Shiga like toxin gene. However, the<br>
multiplex sample analysis was not performed and the limits of detection was not<br>
studied.<br>
While conventional microfabrication is done in silicon, it has been<br>
determined that for chemical and biochemical analyses, glass microfludic structures<br>
exhibit preferable chemical and electrophoretic properties and the extension to<br>
plastic structures is in progress. In the high throughput applications, the techniques<br>
of the present invention provide radial channel layouts that permit the rapid parallel<br>
analysis of 96 to 384 fragment sizing or sequencing separations in parallel. The<br>
integration of PCR directly with CE analysis on a chip is provided with enzymatic<br>
DNA digestion and affinity capture.<br>
According to various embodiments, the microfluidic device mechanisms of<br>
the present invention allow the creation of intricate channel structures that permit<br>
the formation of complex arrays of chambers, valves and CE analysis channels. The<br>
small size of these CE channels together with the use of cross injectors facilitates<br>
the performance of very rapid, high resolution electrophoretic separations.<br>
Substantially all operations that have been performed in chromatographic columns<br>
or capillaries have also been reduced to a chip format with decreases in required<br>
sample volume and improved analysis time and sensitivity.<br>
According to various embodiments, the pathogen detection system of the<br>
present invention has the attributes of sensitivity combined with specificity and<br><br>
quantitation to provide a particularly useful assay. Many pathogens are infective at<br>
&gt;103 ingested bacteria, but V, cholera will not cause symptoms if less than 105<br>
organisms are orally ingested and for B. anthracis much lower levels are considered<br>
significant. Identifying the strain so that the pathogenic can be distinguished from<br>
the nonpathogenic, and identifying the presence of specific toxins or antibiotic<br>
resistance genes can also be critical for identifying the threat and determining the<br>
treatment. Furthermore, the ability to determine the concentration or dose of<br>
bacteria and to report this quantitatively along with the identity will distinguish<br>
important challeneges from background challenges.<br>
Figure 7 is a diagrammatic representation of one example of a pathogen<br>
detection system 700. An analyte is introduced through a channel 701 into<br>
immunoaffinity capture chambers 703, 713, and 723 with waste collected at a<br>
channel 731. According to various embodiments, immunoaffinity reagents are used<br>
to capture, concentrate and stratify input bacterial mixtures into the series of<br>
separate immunological chambers 703, 713, and 723. This facile process addresses<br>
the important macro to micro interface that has previously been a barrier for the<br>
application of microfluidic systems to trace pathogen detection. The first stage of<br>
immunocapture also plays a significant role in enhancing the specificity of the<br>
assay. To achieve the enhanced sensitivity, a user of the pathogen detection system<br>
can then perform PCR based redundant confirmation of the presence of the agent<br>
and also develop methods based on specific primers or more general genotyping<br>
methods such as PCR to identify the specific strain, the presence of toxin genes and<br>
the presence of antibiotic resistance markers using DNA analysis mechanisms 705,<br>
715, and 725. In one example, DNA analysis mechanisms 705, 715, and 725<br>
include PCR and CE.<br>
According to various embodiments, the immunoaffinity capture chambers<br>
703, 713, and 723 are integrated with PCR chambers but CE mechanisms remain<br>
separate. The combination of immunocapture and nucleic acid analysis<br>
dramatically enhances the sensitivity and specificity of the individual assays.<br>
The ability to genetically differentiate pathogenic from nonpathogenic<br>
strains is critical in many applications. The combination of immunocapture as the<br>
front end to PCR analysis provides an important purification of the input bacterial<br>
population to address concerns about the presence of PCR inhibitors often found in<br><br>
impure, complex "real world" samples. According to various embodiments, the<br>
pathogen detection system will be set up to perform PCR in the low cycle number<br>
(not asymptotic) regime so that quantitation of the input target population is<br>
maintained and reported. In many examples, the processed samples can then be<br>
provided for CE analysis. The use of modem microfluidic technologies will result<br>
in the production of inexpensive, rapid and robust assay systems that are small,<br>
portable, and require minimal power, resources and skill for operation.<br>
Integrated immunpaffinity capture chambers are included in a pathogen<br>
analyzer. A variety of capture mechanisms can be used, such as frits, beads, gels,<br>
monoliths, and polymers. Figures 8 and 9 are diagrammatic representations<br>
showing immunocapture chambers implemented using silica frits or beads.<br>
According to various embodiments, immunocapture chambers includes a series of<br>
silica frits fabricated by filling wafer holes with a mixture of silica power and<br>
sodium silicate binder. Upon dehydration and rinsing, the silicate condenses to<br>
silica gel and an insoluable silica frit is formed at 801, 803, 805, and 807.<br>
According to various embodiments, each silica frit formed in a 1.1 mm thick<br>
glass wafer is 1 mm in diameter. The immunocapture chambers are associated with<br>
a channel 821 for introducing and evacuating an analyte. The in-wafer frits can<br>
easily be integrated into devices containing membranes 811 and 813 and valve and<br>
pump structures. In Figure 8, the four silica frits 801, 803, 805, and 807 are sealed<br>
shut by membranes 811 and 813. The large silica surface of each frit is suitable for<br>
chemical derivatization by a wide variety of organosilane reagents. To further<br>
simplify device fabrication, the monolith wafer can be chemically derivatized prior<br>
to non-thermal PDMS bonding to the rest of the device.<br>
In one example, mechanisms such as frits or beads 1.5um are provided into a<br>
capture chamber to allow for capture of many macro species such as spores and<br>
bacteria. Solid-phase capture of many macro-species is known to those of skill in<br>
the art and is well characterized in Weimer, B.C. , M.K. Walsh, C. Beer, R. Koka,<br>
and X. Wang, 2001 Solid Phase Capture Of Proteins, Spores, and Bacteria. Appl<br>
Environ. Microbiology, 67:1300-1307. In some examples, to utilize bead reagents<br>
for capture, the chamber is modified with a weir structure to provide a bead stop, as<br>
well as a bead introduction channel. Electrokinetic bead bed packing and weir bead<br><br>
trapping is known to one of skill in the art. Alternatively, immunofunctionalized<br>
magnetic beads may be introduced into a chamber without a weir.<br>
Figure 9 is a diagrammatic representation showing open valves with the<br>
monoliths no longer sealed. According to various embodiments, pneumatic vacuum<br>
pressure is applied at regions 901, 903, 905, 907, and 909 to allow flow of an<br>
analyte along channel 921 through the frits 931, 933, 935, and 937. Any number of<br>
frits may be included in a fabricated device.<br>
Figure 10A is a diagrammatic representation showing capture of an analyte.<br>
According to various embodiments, a pump 1000 including three membrane valves<br>
1001, 1003, and 1005 is used to pump an analyte solution containing<br>
oligonucleotides, proteins, cells, etc., through the series of immunocapture<br>
chambers.<br>
Chambers can use a variety of mechanisms for capturing a target of interest.<br>
Anything of interest configured for capture in an immunocapture chamber is<br>
referred to herein as a target. The fluid or substance carrying the target is referred to<br>
herein as an analyte. In one example, the target is Salmonella or Listeria carried in a<br>
fluid analyte.<br>
In other examples, each capture chamber is filled with a viscous polymeric<br>
matrix containing oligonucleotide probes to selectively bind the target molecules.<br>
In the case of DNA analysis, Sanger DNA sequencing extension products, including<br>
primers and polymerase reagents in a high salt concentration, are electrophoresed<br>
through an immunocapture chamber containing the immobilized acrylamide matrix<br>
containing the covalent oligonucleotide probe. T'he capture sequence is chosen so<br>
that only DNA amplification products are captured by the probe, but the primers and<br>
polymerase reagents, along with salts, pass through the device. This is not unlike<br>
the need to purify target molecules from complex, dirty mixtures that will be<br>
encountered in point of care analyses.<br>
An alternative approach for the preparation of microcapture chambers with<br>
functionalized polymeric capture matrices includes the preparation of monoliths<br>
with pores in the range of 10-20 urn, and the preparation of chambers with large<br>
microfabricated elements (ca. 100 um) surface modified by a thin crosslinked layer<br>
of functional polymer. This approach is useful as beads are sometimes found<br>
difficult to pack in capture chambers and bead beds are often not sufficiently<br><br>
mechanically stable for routine operation. According to various embodiments,<br>
molded blocks of porous (10-20 um) surface functionalized polymer monoliths are<br>
formed directly within the capture chambers by photo polymerization of a precursor<br>
mixture including monomers and porogenic solvents.<br>
Since the polymerization process is accomplished using UV light, the porous<br>
polymer can be formed in any desired area of the microfluidic device using<br>
photolithography. The kinetics of such a "microlithographic" polymerization<br>
process using glass chips filled with a precursor mixture has been characterized and<br>
is known to one of skill in the art as shown in Yu, C, F. Svec, and J. M. J. Frechet<br>
2000. Towards stationary phases for chromatography on a microchip: Molded<br>
porous polymer monoliths prepared in capillaries by photoinitiated in situ<br>
polymerization as separation media for electrochromatography. Etectrophoresis,<br>
21:120-127 and Yu, C, M. Xu, F. Svec, and J. M. J. Frechet 2002. Preparation of<br>
monolithic polymers with controlled porous properties for microfluidic chip<br>
applications using photoinitated free radial polymerization. J. Polymer Sci., 40:755.<br>
Similarly the precise location of the monolithic material on the device as well as its<br>
surface chemistry can be controlled as is known to one of skill in the art as shown<br>
in, Rohr, T.C, C. Yu, H.M. Davey, F. Svec, and J.M. J. Frechet 2001. Simple and<br>
efficient mixers prepared by direct polymerization in the channels of microfluidic<br>
chips. Electrophoresis, 22:3959. Control over porous properties of the monolithic<br>
polymers can be achieved by adjusting the composition of the porogenic solvents.<br>
Whether a monolith or a surface with microfabricated elements is used, the<br>
same grafting approach can be used to introduce the desired binding elements. Since<br>
the goal is to immobilize antibodies on the pore surface of these monoliths, the<br>
grafted chemistries are specified to readily react with biopolymers. In one example,<br>
units of 2-vinyl-4,4-dimethylazlactone incorporated into a surface graft can react<br>
rapidly with proteins. Such mechanisms are known to one of skill in the art as<br>
shown in Peterson, D.S., T. Rohr, F. Svec, and J.M. J. Frechet. 2002. Enzymatic<br>
microreactor-on-a-chip: protein Mapping using trypsin immobilized on porous<br>
polymer monoliths molded in channels of microfluidic devices. Anal. Chem.,<br>
74:4081:4088. The surface to be modified (porous monolith, or microfabricated<br>
elements) can be immersed in a monomer solution and the device can be irradiated<br>
by UV light to achieve grafting in preselected areas. The extent of surface<br><br>
functionalization is controlled by the concentration of the monomer in the reaction<br>
solution, the irradiation time, and the intensity of the UV light.<br>
In other embodiments, trypsin is immobilized on porous polymer monoliths<br>
consisting of 2-vinyl-4, 4-dimethylazlactone, ethylene dimethacrylate, and<br>
acrylamide or 2-hydroxyethyl methacrylate. The azlactone functionalities react<br>
readily with amine and thiol groups of the enzyme to form stable covalent bonds. In<br>
some examples, the optimized porous properties of the monoliths lead to very low<br>
back pressures enabling the use of simple mechanical pumping to carry out both the<br>
immobilization of the enzyme from its solution and the subsequent analyses of<br>
substrate solutions. The Michealls-Menten kinetic characteristics of the reactors can<br>
be probed using a low molecular weight substrate such as N-a-benzoyl-L-arginine<br>
ethyl ester.<br>
The effects of immobilization variables such as the concentration of trypsin<br>
in solution and percentage of azlactone functionalitiestles in the in the monolith, as<br>
well as the effect of reaction time on the enzymatic activity, and of process variables<br>
such as substrate flow velocity and residence time in the reactor, were studied in<br>
detail. The proteolytic activity of the enzymatic microreactor on chip was<br>
demonstrated at different flow rates with the cleavage of fluorescently labeled<br>
casein used as a substrate. The excellent performance of the monolithic<br>
microreactor was also demonstrated with the digestion of myoglobin as the fast flow<br>
rate of 0.5 uL/min, which affords a residence time of only 11.7s. The digest was<br>
then characterized using MALDI-TOFMS, and 102 out of 153 possible peptide<br>
fragments were identified giving a sequence coverage of 67%.<br>
An enormous effort has been directed toward the development of new<br>
micorfabricated analytical devices and their integration to create micro total<br>
analytical systems (Ptas). These systems offer the promise of increased throughput,<br>
lower sample and reagent consumption, smaller size, and lower operating costs than<br>
full size instrumentation, Among the various applications of microfluidic devices,<br>
analytical techniques such as electrophoresis, electrochromatography, assays<br>
involving enzymes, and imrnuno-assays have already been demonstrated in this<br>
format. Despite the undeniable success of microfiuidic chip technologies, some<br>
problems persist. For example, the vast majority of microfiuidic chips feature open<br>
channel architecture. Consequently, these channels exhibit rather small surface-to-<br><br>
volume ratios. They may be a serious problem in applications such as<br>
chromatographic separations, solid-phase extraction, and heterogeneous catalysis<br>
that rely on interactions with a solid surface. Since only channel walls can be used<br>
to provide the desired interactions, the microdevice can only handle minute amounts<br>
of compounds.<br>
Figure 10B is a diagrammatic representation showing use of the two-<br>
dimensional analysis system. After the monoliths 1027 capture targets provided by<br>
the pump with valves 1001, 1003, and 1005, the monoliths 1027 are sealed. In one<br>
example, each chamber is then heated to melt the double stranded DNA and drive<br>
off the single stranded DNA product. According to various embodiments, the<br>
purification takes place in 120 seconds, and a 200 fold concentration to only 20nL<br>
from an initial volume of 3uL can be achieved. Each line 1011, 1013, 1015, 1017,<br>
1019, and 1021 includes valves for controlling or pumping captured targets for<br>
additional analysis steps.<br>
In one example, captured targets are provided for PCR and CE analysis on<br>
the test device. Captured targets can be released for DNA analysis using<br>
mechanisms such as heat or a change in pH. The basic features of such an<br>
integrated test device include: 1) an immunocapture chamber etched into a glass<br>
substrate with a microfabricated heater and temperature sensor; 2) a polymerase<br>
chain reaction chamber of 100 300 nL for amplification of DNA obtained from<br>
lysing the cells of interest; and 3) a capillary electrophoresis microchannel etched<br>
into the glass substrate for separation and detection of the PCR amplicons.<br>
An optional fourth item, an integrated DNA preconcentration/clean-up<br>
chamber, can also be added to the device for purification of the released pathogen<br>
genomic DNA or for desalting and preconcentration of the amplified DNA before<br>
injection onto the CE microchannel if needed. Although previous studies have<br>
shown that PCR amplicons can be directly injected onto CE microchannels for<br>
successful analysis, potentially obviating the necessity of such additional<br>
complexity, such purification may be necessary to obtain high quality<br>
electropherograms. This amplicon purification could be enabled by using<br>
oligonucleotide capture matrix chemistries. If it is necessary to purify the genomic<br>
DNA, a clean-up chamber could be filled with carboxylated silica beads and used as<br>
a general capture matrix for bacterial DNA before PCR.<br><br>
One approach to integration is simply to fabricate immunocapture, template<br>
purification, PCR, amplicon clean-up, and CE as separate modules on a glass chip.<br>
The modules would then be interfaced with each other using microchannels and<br>
various PDMS valve structures. A schematic of a pathogen analysis chip configured<br>
with separate immunocapture and PCR reactors is presented in Figure 11. The<br>
integrated pathogen detection system includes an immunoafrlnity capture chamber<br>
1101. An analyte is introduced into the pathogen detection system through the<br>
irnmunoaffinity capture chamber 1101. A PCR chamber 1103 is coupled to the<br>
immunoaffuiity capture chamber 1101 and receives targets captured by the<br>
immunoafrlnity capture chamber 1101. A CE channel 1105 is coupled to the PCR<br>
chamber 1103 for further analysis. Microfabricated electrodes 1113 are operable to<br>
provide voltage differentials. A heater (not shown) coupled to the immunocapture<br>
chamber and/or the PCR chamber is also provided. A variety of valves control the<br>
flow an analyte through the integrated system. According to various embodiments,<br>
the valves are monolithic valves.<br>
Although providing immunocapture, PCR, CE and clean-up as separate<br>
modules on a device is a reasonable strategy, the capture efficiency, PCR efficiency<br>
and high sensitivity separation and detection of DNA fragments that are facilitated<br>
according to various embodiments suggest that a less complex device can be used.<br>
While immunocapture and PCR could be performed in separate chambers, in one<br>
example, immunocapture and PCR can be combined to simplify the device and the<br>
process. In this example, PCR can be successfully conducted from solid substrates<br>
and from solid phase immunocapture reagents. In one example, PCR can be<br>
performed using immuno-labeled beads.<br>
Figure 12 is a diagrammatic representation showing a combined<br>
immunocapture and PCR chamber 1201. According to various embodiments, the<br>
combined chamber has integrated resistance heating mechanism (not shown) and a<br>
resistance temperature detector (RTD) 1205 fabricated within the nanoliter chamber.<br>
In some examples, an analyte is introduced through an input 1211 through a<br>
membrane valve 1221. Pathogens of interest are immobilized within the chamber<br>
1201 using pressure driven flow and waste is collected through a valve 1223 at an<br>
output 1213. After pathogens are immobilized, the chamber 1201 is flushed with<br>
buffer to remove loosely adhered cells or non specifically bound agents.<br><br>
PCR buffer is introduced either through the original sample inlet 1211 or<br>
through a separate dedicated inlet Depending on the pathogen of interest in the<br>
chamber 1201, a chemical lysis agent can be included directly into the PCR buffer.<br>
After introduction of the lysis reagent and/or PCR buffer, the integrated heater 1203<br>
in the capture/PCR chamber is used to raise the temperature of the sample to a<br>
temperature at which the pathogens are simultaneously released from the capture<br>
matrix and, depending on the class of agent, lysed.<br>
The simplest and often most effective lysis method is simply performing<br>
heating/cooling cycles. Gram negative bacteria and some eukaryotic cells, with<br>
their thinner outer membranes, are more susceptible to lysis using either heat alone<br>
or heat with a small concentration of chemical lysis solutions. In some cases, such<br>
as for spores or gram positive bacteria, use of a more aggressive lysis agent that<br>
would interfere with the PCR may be necessary. For example, lysozyme, proteinase<br>
K, lysostaphin, and mutanolysin are commonly required separately or in tandem to<br>
lyse some recalcitrant gram positive Staphylococcal and &amp; Streptococcal strains. In<br>
these cases, the use of a separate immunocapture chamber and the addition of a<br>
clean-up/preconcentration chamber allows for intermediate capture of DNA after<br>
cell lysis but before PCR amplification.<br>
In this scenario, following capture and lysis, the extracted DNA can be<br>
electrophoretically driven into the cleanup chamber for storage by adsorption to<br>
carboxyl beads. The purified DNA can be released from the clean-up chamber<br>
using heat or variations in ionic strength and electrophoretically transported into the<br>
PCR chamber for amplification. Once the DNA from the lysed cells is presented to<br>
the chamber with PCR buffer, PCR can be performed directly on the released<br>
genetic material using the microfabricated heater and temperature sensor.<br>
It should be noted that in some instances, the combined use of a single<br>
chamber for both capture and PCR is problematic because of complexity or PCR<br>
inhibition. In these particular instances, the two stages can simply be separated. In<br>
some examples, this may be done if the presence of the capture matrix or beads<br>
inhibits the PCR reaction or if the input sample brings in PCR inhibitors that can not<br>
be washed out or neutralized. In this case, the released DNA could be pumped or<br>
electrophoresed from the lysed bacteria in the capture chamber to a separate PCR<br>
reactor for analysis.<br><br>
Upon completion of PCR, the amplicons can be directly injected onto a CE<br>
micTOchannel for separation and detection, either using intercalating dye in the<br>
separation matrix or fluorescently labeled primers and a denaturing separation<br>
matrix depending on the desired resolution. In some instances, a DNA clean-up<br>
chamber is introduced to desalt and concentrate the amplified DNA prior to<br>
injection onto the CE microchannel. Clean-up is accomplished by electrophoresing<br>
the amplified DNA into the clean-up chamber where it is bound to carboxylated<br>
beads or to an oligonucleotide capture matrix (capture oligos complementary to the<br>
desired targets). Binding followed by washing and temperature dependent release<br>
using a micro heater is followed by electrophorcsis of the concentrated and desalted<br>
PCR amplicons through the injection cross of the CE microchannel for separation<br>
and detection.<br>
The device configuration for using monolithic membrane valves to build a<br>
pathogen detection and analysis system can be varied substantially. Figure 13A is a<br>
diagrammatic representation showing one example of a design for the pathogen<br>
detection and analysis system. The design includes three glass layers, including a<br>
channel layer 1303, a via layer 1305, and a manifold layer 1309. A PDMS<br>
membrane layer 1307 is provided between a via layer 1305 and a manifold layer 1309.<br>
The manifold layer 1309 includes mechanisms allowing vacuum pressure to be<br>
applied to the membrane 1307 to allow control of valve mechanisms.<br>
Electrical connections are provided on a layer 1301 and a manifold chuck<br>
layer is included at layer 1311. The channel layer 1303 includes the<br>
immunocapture/PCR/clean-up chambers and CE microchannels; as well as the<br>
heaters on the top surface of the wafer. According to various embodiments, the<br>
channel layer 1303 is thermally bonded to a thin glass wafer 1305 containing drilled<br>
glass holes that act as valve vias. A PDMS valve/pump membrane 1307 is either<br>
reversibly or irreversibly bonded to this multiple layer stack. The bottom etched<br>
manifold layer 1309 conveys vacuum or pressure to the valves and pumps on the<br>
device.<br>
The use of existing thin film technology to create the temperature control<br>
elements presents a viable first approach to construction of test devices. However,<br>
the fabrication complexity of the device can be reduced through the use of indium<br>
tin oxide (ITO) heaters. ITO heaters are noted, for their low resistivity, optical<br><br>
transparency, and compatibility with glass substrates. These heaters can be<br>
deposited on the same wafer as the temperature sensors, obviating the need for<br>
backside fabrication and electroplating to form the heaters. The heaters can be<br>
placed directly within the chambers for optimal thermal transfer or they can be<br>
placed against the chambers to conduct thermal energy through a glass wafer. The<br>
optical transparency of ITO also allows routing of electrical heater leads over fluid<br>
microchannels without interfering with visualization or detection of sample or PCR<br>
amplicons.<br>
Figure 13B is a diagrammatic representation showing a microfabrication<br>
process according to various embodiments. Microfabrication processes are shown<br>
at 1381 and 1383. In some examples, glass wafers (550 µm thick D263 available<br>
from Schott of Yonkers, NY) are cleaned before sputter deposition of a 2000 A<br>
layer of amorphous silicon on one side by DC magnetron sputtering available from<br>
UHV Sputtering of San Jose, CA). Photoresist available from Shipley 1818 of<br>
Marlborough, MA is spun on and photolithographically patterned using a contact<br>
aligner available from Karl Suss of Waterbury Center, VT and the underlying<br>
silicon etch mask can be selectively removed using SF6 in a parallel-plate reactive<br>
ion etching (RIE) system available from Plasma Therm of St. Petersburg, FL.<br>
In some examples, the fluidic channels, electrophoresis channels, and PCR<br>
chambers are etched to a depth of 36 µm in 49% hydrofluoric acid. Reservoir<br>
access holes (1.5 mm diameter) and fluidic via holes (0.020" diameter) for the<br>
PDMS valves are drilled using a CNC mill available from Flashcut CNC of Menlo<br>
Park, CA with diamond-tipped drill bits. The wafer is then diced using a wafer<br>
dicing saw to form two 20 mm x 75 mm slides.<br>
To form the RTDs and electrodes, a 550 µm-thick D263 wafer can first be<br>
sputtercoated with 200 A of Ti and 2000 A of Pt (UHV). Thick photoresist<br>
available from Shipley (SJR 5740) of Marlborough, MA is spun on and patterned<br>
using a contact aligner available from Suss Microtec of Waterbury Center, VT.<br>
According to various embodiments, the photoresist is hard baked at 70 °C for 2<br>
hours. The metal can be etched using hot aqua regia (3:1 HC1.HNO3, 90 °C) to<br>
form the RTD elements. The integrated heaters are formed by first depositing a<br>
multi-layer thin film of 200 A of Ti and 2000 A of Pt on the backside of the RTD<br>
wafer using RF sputtering available from Perkin Elmer of Wellesley, MA. Thick<br><br>
photoresist is spun on the side, the wafer is patterned using a backside contact<br>
aligner (Suss), and hard baked. Gold is electrodeposited onto the Ti/Pt seed layer at<br>
4.3 mA/cm2 for 23 minutes to a 5 µm thickness using a gold sulfite plating solution<br>
available from Technic (TG 25 E) of Anaheim, CA to form the heater leads.<br>
According to various embodiments, the photoresist is removed and the<br>
backside is re-patterned using thick photoresist. The heating elements are etched<br>
into the Ti/Pt seed layer using an ion beam etching system available form Veeco<br>
Instruments of Plainview, NY. The RTD/heater wafer is diced into two 25 mm x 75<br>
mm slides (Disco). In some examples, the drilled channel wafer is thermally bonded<br>
to the RTD/heater wafer using a programmable vacuum furnace available from<br>
Centurion VPM, J.M. Ney, of Yucaipa, CA.<br>
Although, a single immunocapture, PCR, and CE system can be included on<br>
a substrate, the techniques of the present invention recognize that it may be efficient<br>
to develop a parallel immunocapture, PCR, and CE system for use in clinical<br>
diagnostics. In one example, a portable pathogen analyzer includes three serial<br>
immunocapture/PCR systems targeted towards the detection of three different<br>
pathogens in a sample. The parallelization of the fluidics control systems, electrical<br>
circuitry for heaters, temperature sensors and electrophoresis for three systems is<br>
straightforward and a single microscope slide has sufficient surface area to fabricate<br>
three fully parallel systems.<br>
In another example, a massively parallel immunocapture/PCR system for use<br>
in clinical diagnostics is provided. The ability to analyze multiple distinct agents<br>
across multiple individuals or groups of individuals provides a powerful method for<br>
identifying and epidemiologically tracking infectious agents. Figure 14 is a<br>
diagrammatic representation of a portion of a radially parallel immunocapture/PCR<br>
device 1400. Any system or device having multiple immunocapture and DNA<br>
analysis mechanisms arranged about a circular axis is referred to herein as a radially<br>
parallel device.<br>
According to various embodiments, the design includes an array of pairs of<br>
analyzers each of which includes a unique immunocapture/PCR chamber 1423<br>
integrated with a CE analyzer. The sample travels serially through all chambers<br>
within a given subset of the device, allowing for serial capture of multiple agents.<br>
Separate subsets 1401, 1403, 1405, 1407, 1409, 1411 of the device are capable of<br><br>
analyzing different substances in parallel. Reservoirs 1447 and 1445 provide bead<br>
input and bead waste. Reservoirs 1443 and 1441 are the common capillary<br>
electrophoresis cathode reservoir and waste reservoir, respectively.<br>
The chambers are interconnected for cascaded immunoaffinity capture.<br>
Valves 1431 and 1433 seal the chamber on the cascade loop. Valves 1435 and 1437<br>
seal the chamber from bead introduction and waste channels. CE microchannels are<br>
connected to a common central anode for detection using a proven rotary confocal<br>
fluorescence scanner (not shown). A parallel array of combined capture chambers<br>
1423 and heaters with leads 1451 and the development of robust arrays of valves<br>
and pumps are provided. Since the heaters and temperature sensors associated with<br>
chambers 1423 are operating in parallel on the analysis channels, the use of simple<br>
ring heaters are more than adequate. Thus the individual heaters and temperature<br>
sensors are no longer necessary for providing an efficient and effective parallel<br>
pathogen detection system<br>
Although many of the components and processes are described above in the<br>
singular for convenience, it will be appreciated by one of skill in the art that<br>
multiple components and repeated processes can also be used to practice the<br>
techniques of the present invention.<br>
While the invention has been particularly shown and described with<br>
reference to specific embodiments thereof, it will be understood by those skilled in<br>
the art that changes in the form and details of the disclosed embodiments may be<br>
made without departing from the spirit or scope of the invention. For example, the<br>
embodiments described above may be implemented using a variety of materials.<br>
Therefore, the scope of the invention should be determined with reference to the<br>
appended claims.<br><br>
We Claim :<br>
1.	A pathogen detection system (700; 1400), the system comprising:<br>
an immunocapture chamber (703; 1101) integrated on a microfluidic device, the<br>
immunocapture chamber operable to capture a target, such as herein described, provided<br>
to the immunocapture chamber through a microfluidic channel (701; 1211);<br>
a DNA analysis chamber (1103; 1201; 1423) comprising a DNA analysis<br>
mechanism (705; 1103) in fluid communication with the immunocapture chamber, the<br>
DNA analysis chamber integrated on the microfluidic device, the DNA analysis<br>
mechanism operable to perform DNA analysis on the target; and<br>
at least one pneumatically actuated diaphragm pump (210; 1000; 1307) integrated<br>
on the microfluidic device and configured to pump a determined volume of fluid through<br>
the microfluidic channel into the immunocapture chamber and pump a determined<br>
volume of fluid from the immunocapture chamber into the DNA analysis chamber.<br>
2.	The pathogen detection system as claimed in claim 1, wherein the DNA<br>
analysis mechanism comprises polymerase chain reaction (PCR) and capillary<br>
electrophoresis (CE).<br>
3.	The pathogen detection system as claimed in claim 2, wherein the chamber for<br>
PCR is used for amplification of DNA obtained from lysing the target of interest.<br>
4.	The pathogen detection system as claimed in claim 1, optionally comprising an<br>
etched capillary electrophoresis microchannel (1105) connected to the DNA analysis<br>
chamber for separation and detection of PCR amplicons.<br>
5.	The pathogen detection system as claimed in claim 4, optionally comprising<br>
DNA preconcentration and clean-up chambers for purification of released pathogen<br>
genomic DNA or for desalting and preconcentration of amplified DNA before injection<br>
onto the CE microchannel.<br>
6.	A pathogen detection system (700; 1400), the system comprising:<br>
immunocapture means (703; 1101) integrated on a microfluidic device, the<br><br>
immunocapture means operable to capture a target, such as herein described, provided<br>
through a microfludic channel (701; 1211); DNA analysis means (705; 1103; 1423) in<br>
fluid communication with the immunocapture means, the DNA analysis means integrated<br>
on the microfluidic device, the DNA analysis means operable to perform DNA analysis<br>
on the target; and<br>
at least one pneumatically actuated diaphragm pump (210; 1000; 1307) integrated<br>
on the microfluidic device and configured to pump a determined volume of fluid through<br>
the microfluidic channel into the immunocapture means and pump a determined volume<br>
of fluid from the immunocapture means into the DNA analysis means.<br>
7.	The pathogen detection system as claimed in claim 6, wherein the DNA<br>
analysis means comprises a PCR chamber (705; 1103 connected through a second<br>
microfluidic channel to an immunocapture chamber comprising the immunocapture<br>
means.<br>
8.	The pathogen detection system as claimed in claim 7, wherein the PCR<br>
chamber is used for amplification of DNA obtained from lysing the target of interest.<br>
9.	A pathogen detection system as claimed in claim 1 optionally comprsing:<br>
a plurality of immunocapture chambers (703; 713; 1423) integrated on the<br>
microfludic device, each immunocapture chamber operable to capture a target provided<br>
to the immunocapture chamber through one of a plurality of microfluidic channels; and<br>
a plurality of DNA analysis chambers, comprising DNA analysis mechanisms<br>
(705; 715; 1423) in fluid communication with the immunocapture chambers, the plurality<br>
of DNA analysis mechanisms integrated on the microfluidic device, the plurality of DNA<br>
analysis mechanisms operable to perform DNA analysis on the target[[.]]; and<br>
a plurality of pneumatically actuated diaphragm pumps (210; 1000; 1307)<br>
integrated on the microfluidic device and configured to pump a determined volume of<br>
fluid through the microfluidic channels into at least one of the immunocapture chambers<br>
and pump a determined volume of fluid from at least one of the immunocapture chambers<br>
into at least one of the DNA analysis chambers.<br><br>
10.	The pathogen detection system as claimed in claim 9, wherein the plurality of<br>
DNA analysis mechanisms comprise PCR and CE<br>
11.	The pathogen detection system as claimed in claim 9, optionally comprising a<br>
plurality of etched capillary electrophoresis microchannels for separation and detection of<br>
PCR amplicons.<br>
12.	The pathogen detection system as claimed in claim 11, optionally comprising<br>
a plurality of integrated DNA preconcentration and clean-up chambers for purification of<br>
released pathogen genomic DNA or for desalting and preconcentration of amplified DNA<br>
before injection onto the CE microchannel.<br>
13.	The pathogen detection system as claimed in claim 9, wherein the<br>
immunocapture chambers are further operable to purify and concentrate a target.<br>
14.	The pathogen detection system as claimed in claim 9, wherein the plurality of<br>
microfabricated immunocapture chambers are configured to hold selected antibodies.<br>
15.	The pathogen detection system as claimed in claim 14, wherein the selected<br>
antibodies are held with beads, frits (801, 803, 805, 807), sol-gels, gels, or polymer<br>
monoliths.<br>
16.	The pathogen detection system as claimed in claim 14, wherein the selected<br>
antibodies are held with molded blocks of porous, surface functionalized polymer formed<br>
directly within the capture chambers.<br>
17.	The pathogen detection system as claimed in claim 16, wherein the molded<br>
blocks are formed by photo polymerization of a precursor mixture including monomers<br>
and porogenic solvents.<br>
18.	The pathogen detection system as claimed in claim 14, wherein the plurality<br>
of immunocapture chambers (1423) are configured in a radially parallel manner.<br><br>
19.	The pathogen detection system as claimed in claim 18, optionally comprising<br>
ring heaters coupled to the plurality of immunocapture chambers, the ring heaters<br>
operable to heat the plurality of immunocapture chambers to release the captured target.<br>
20.	The pathogen detection system as claimed in claim 14, wherein the plurality<br>
of immunocapture chambers are configured on a fluidics layer(13O3).<br>
21.	The pathogen detection system as claimed in claim 20, wherein the fluidics<br>
layer is coupled to a monolithic membrane layer (1307).<br>
22.	The pathogen detection system as claimed in claim 20, wherein the fluidics<br>
layer includes a plurality of etched channels, the etched channels operable to provide<br>
paths for fluid flow.<br>
23.	The pathogen detection system as claimed in claim 22, wherein the fluidics<br>
layer and a pneumatic layer (1309) sandwich the membrane layer, wherein the pneumatic<br>
layer comprises pneumatic channels that actuate the diaphragm pumps using vacuum or<br>
pressure.<br>
24.	An apparatus (700; 1400) for detecting pathogens, the apparatus comprising:<br>
means for providing a fluid analyte to a plurality of channels integrated on a<br>
monolithic device wherein each channel comprises:<br>
an immunocapture chamber (703, 713; 1423) comprising<br>
means for capturing a target, such as herein described, associated with the fluid<br>
analyte wherein a plurality of immunocapture chambers capture different targets; a DNA<br>
analysis mechanism (705, 715; 1425) associated with the plurality of immunocapture<br>
chambers comprising means for performing DNA analysis on the target and<br>
at least one pneumatically actuated diaphragm pump (210; 1000; 1307) configured to<br>
pump a determined volume of fluid through at least one of the plurality of<br>
immunocapture chambers and pump a determined volume of fluid from at least one of<br>
the plurality of immunocapture chambers into the DNA analysis mechanism.<br>
25.	A method for pathogen analysis by using the apparatus as claimed in claim<br><br>
24, the method comprising:<br>
providing a fluid analyte to a plurality of immunocapture chambers (703; 713;<br>
1423) through microfluidic channels integrated with a monolithic device (700; 1400);<br>
capturing a target, such as herein described, associated with the fluid analyte at<br>
the immunocapture chambers; and<br>
performing DNA analysis on the target using a plurality of DNA analysis<br>
mechanisms (705; 715; 1423), the plurality of DNA analysis mechanisms integrated with<br>
the microfluidic device.<br>
26.	The method as claimed in claim 25. wherein the plurality of DNA analysis<br>
mechanisms comprise PCR and CE.<br>
27.	The method as claimed in claim 25, wherein the plurality of DNA analysis<br>
mechanisms include PCR chambers for amplification of DNA obtained from lysing the<br>
target of interest.<br>
28.	The method as claimed in claim 25, optionally comprising a plurality of<br>
etched capillary electrophoresis microchannels for separation and detection of PCR<br>
amplicons.<br>
29.	The method as claimed in claim 28, optionallycomprising a plurality of<br>
integrated DNA preconcentration and clean-up chambers for purification of released<br>
pathogen genomic DNA or for desalting and preconcentration of amplified DNA before<br>
injection onto the CE microchannels.<br>
30.	The method as claimed in claim 25, wherein the immunocapture chambers<br>
are further operable to purify and concentrate a target.<br>
31.	The method as claimed in claim 25, wherein the plurality of microfabricated<br>
immunocapture chambers are configured to hold selected antibodies.<br>
32.	The method as claimed in claim 31, wherein the selected antibodies are held<br>
with beads, sol-gels, gels, or polymer monoliths.<br><br>
33. The method as claimed in claim 31, wherein the selected antibodies are held<br>
with molded blocks of porous, surface functionalized polymer formed directly within the<br>
capture chambers.<br><br>
The invention discloses methods and apparatus for implementing microfluidic analysis devices are<br>
provided. A monolithic elastomer membrane associated with an integrated pneumatic manifold<br>
allows the placement and actuation of dense arrays of a variety of fluid control structures, such as<br>
structures for isolating, routing, merging, splitting, and storing volumes of fluid. The fluid control<br>
structures can be used to implement a pathogen detection analysis system including integrated<br>
immunoaffinity capture and analysis, such as polymerase chain reaction (PCR) and capillary<br>
electrophoresis (CE) analysis. An analyte solution can be input into the device and pumped through a<br>
series of immunoaffinity capture matrices in microfabricated chambers having antibodies targeted to<br>
the various classes of microbiological organisms such as bacteria, viruses and bacterial spores. The<br>
immunoaffinity chambers can capture, purify, and concentrate the target for further analysis steps.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI2NS1LT0xOUC0yMDA1LSgxMy0wMS0yMDEyKS1GT1JNLTI3LnBkZg==" target="_blank" style="word-wrap:break-word;">1265-KOLNP-2005-(13-01-2012)-FORM-27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI2NS1LT0xOUC0yMDA1LUNPUlJFU1BPTkRFTkNFLnBkZg==" target="_blank" style="word-wrap:break-word;">1265-KOLNP-2005-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI2NS1LT0xOUC0yMDA1LUZPUk0gMjcucGRm" target="_blank" style="word-wrap:break-word;">1265-KOLNP-2005-FORM 27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI2NS1LT0xOUC0yMDA1LUZPUk0tMjcucGRm" target="_blank" style="word-wrap:break-word;">1265-KOLNP-2005-FORM-27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI2NS1rb2xucC0yMDA1LWdyYW50ZWQtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">1265-kolnp-2005-granted-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI2NS1rb2xucC0yMDA1LWdyYW50ZWQtYXNzaWdubWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">1265-kolnp-2005-granted-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI2NS1rb2xucC0yMDA1LWdyYW50ZWQtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">1265-kolnp-2005-granted-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI2NS1rb2xucC0yMDA1LWdyYW50ZWQtY29ycmVzcG9uZGVuY2UucGRm" target="_blank" style="word-wrap:break-word;">1265-kolnp-2005-granted-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI2NS1rb2xucC0yMDA1LWdyYW50ZWQtZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">1265-kolnp-2005-granted-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI2NS1rb2xucC0yMDA1LWdyYW50ZWQtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">1265-kolnp-2005-granted-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI2NS1rb2xucC0yMDA1LWdyYW50ZWQtZXhhbWluYXRpb24gcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">1265-kolnp-2005-granted-examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI2NS1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">1265-kolnp-2005-granted-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI2NS1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">1265-kolnp-2005-granted-form 13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI2NS1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSAxOC5wZGY=" target="_blank" style="word-wrap:break-word;">1265-kolnp-2005-granted-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI2NS1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">1265-kolnp-2005-granted-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI2NS1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">1265-kolnp-2005-granted-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI2NS1rb2xucC0yMDA1LWdyYW50ZWQtZ3BhLnBkZg==" target="_blank" style="word-wrap:break-word;">1265-kolnp-2005-granted-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI2NS1rb2xucC0yMDA1LWdyYW50ZWQtcmVwbHkgdG8gZXhhbWluYXRpb24gcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">1265-kolnp-2005-granted-reply to examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTI2NS1rb2xucC0yMDA1LWdyYW50ZWQtc3BlY2lmaWNhdGlvbi5wZGY=" target="_blank" style="word-wrap:break-word;">1265-kolnp-2005-granted-specification.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="226134-genetically-engineered-l-sorbose-reductase-deficient-mutants.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="226136-recombinant-microorganisms-capable-of-fermenting-cellobiose.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>226135</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1265/KOLNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>50/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>12-Dec-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>08-Dec-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>29-Jun-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>THE REGENTS OF THE UNIVERSITY OF CALIFORNIA</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>OFFICE OF THE PRESIDENT, 12TH FLOOR, 1111 FRANKLIN STREET, OAKLAND, CA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>MATHIES, RICHARD, A.</td>
											<td>93 DANEFIELD PLACE, MORAGA, CA 94556</td>
										</tr>
										<tr>
											<td>2</td>
											<td>GROVER, WILLIAM, H.</td>
											<td>2214 CHANNING WAY, #1 BERKELEY, CA 94704</td>
										</tr>
										<tr>
											<td>3</td>
											<td>PAEGEL, BRIAN</td>
											<td>2100 CHANNING WAY, #348, BERKELEY, CA 94704</td>
										</tr>
										<tr>
											<td>4</td>
											<td>SKELLEY, ALISON</td>
											<td>1771 HIGHLAND PLACE, #204, BERKELEY, CA 94709</td>
										</tr>
										<tr>
											<td>5</td>
											<td>LIU, CHUNG, N.</td>
											<td>1070 JACKSON STREET, #710, ALBANY, CA 94706</td>
										</tr>
										<tr>
											<td>6</td>
											<td>LAGALLY, ERIC</td>
											<td>483 FAIRBANKS AVENUE, OAKLAND, CA 94610</td>
										</tr>
										<tr>
											<td>7</td>
											<td>BLAZEJ, ROBERT</td>
											<td>2940 CLAREMONT AVENUE, #114, BERKELEY, CA 94705</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>B32B 27/04</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2003/041466</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-12-29</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/437,262</td>
									<td>2002-12-30</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/475,013</td>
									<td>2003-05-30</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/226135-a-pathogen-detection-system-and-method-for-pathogen-analysis by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:53:07 GMT -->
</html>
