<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/226121-a-method-for-screening-of-compounds by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:49:50 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 226121:A METHOD FOR SCREENING OF COMPOUNDS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR SCREENING OF COMPOUNDS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present inven,tion relates to methods of screening of compounds. binding a bacterial RNA polymerase sigma 70 subunit, said compounds being potentially useful as inhibitors of the interaction of the core RNA polymerase and the said sigma subunit, and thereby potentially useful in the treatment of bacterial infections, such as Mycobacterial tuberculosis infections.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
TECHNICAL FIELD<br>
The present invention relates to methods of screening for compounds binding a bacterial RNA polymerase sigma subunit, said compounds being useful as inhibitors of the interaction of the core RNA polymerase and the said sigma subunit, and thereby potentially useful in the treatment of bacterial infections.<br>
BACKGROUND ART<br>
Sigma subunits of RNA polymerase<br>
Transcription of genes to the corresponding RNA molecules is a complex process which is catalyzed by DNA dependent RNA polymerase, and involves many different protein factors. In eubacteria, the core RNA polymerase is composed of a, p and P' subunits in the ratio 2:1:1. To direct RNA polymerase to promoters of specific genes to be transcribed, bacteria produce a variety of proteins, known as sigma (a) factors, which interact with RNA polymerase to form an active holoenzyme. The resulting complexes are able to recognize and attach to selected nucleotide sequences in promoters.<br>
Physical measurements have shown that the sigma subunit induces conformational transition upon binding to core RNA polymerase. Binding of sigma subunit to the core enzyme increases the binding constant of the core enzyme for DNA by several orders of magnitude (Chamberlin, MJ. (1974) Ann. Rev. Biochem. 43,721).<br>
Bacterial sigma factors do not have any homology with eukaryotic transcription factors, and are consequently a potential target for antibacterial compounds. Mutations in the sigma subunit, effecting its association and ability to confer DNA sequence specificity to the enzyme, are known to be lethal to the cell.<br>
Characterisation of sigma subunits, identified and sequenced from various organisms, allows them to be classified into three groups. The Group I sigma has also been referred to the sigma70 class, or the "house keeping" sigma group (For a review see Lonetto et al.<br><br>
(1992) J. Bacterid. 174, 3843-3849). Sigma subunits belonging to this group recognise similar promoter sequences in the cell. These properties are reflected in certain regions of the proteins which are highly conserved between species.<br>
Another important feature of sigma-dependent transcription is that the sigma subunit dissociates from the core enzyme during elongation of mRNA. Consequently, molecules which stabilize the interaction of the sigma subunit with the core enzyme would also be capable of inactivating transcription.<br>
Mycobacterium tuberculosis<br>
Mycobacterium tuberculosis is a major pulmonary pathogen which is characterized by its very slow growth rate. As a pathogen it gains access to the alveolar macrophages where it multiplies within the phagosome, finally lysing the cells and being disseminated through the blood stream, not only to other areas of the lung, but also to extrapulmonary tissues.<br>
The pathogen thus multiplies in at least two entirely different environments, involving the utilisation of different nutrients and a variety of possible host factors. A successful infection would thus involve the coordinated expression of new sets of genes, transcribed by RNA polymerases associating with different sigma factors. This opens the possibility of targeting not only the sigma70 subunits of M. tuberculosis, but also other sigma subunits specific for the different stages of infection and dissemination.<br>
Antisigma factors<br>
A proposed mode of action of a sigma inhibitor, "antisigma", is shown in Fig. 1. Antisigma (Asi) proteins are known in the art. Lysates of bacteriophage T2 (Khesin et al. (1972) Mol. Gen. Genet 119,299) or phage T4 (Bogdanova et al. (1970) Mol. Biol. 4,435; Stevens, A. (1972) Proc. Natl. Acad. Sci. U.S.A. 69, 603) have been reported to inhibit transcription of bacterial genes.<br><br>
It has been established that the T4-dependent antisigma70 activity is borne by a 10 kDa protein (Stevens, A. In: RNA Polymerase p. 617-627 (Eds. R. Losick &amp; M. Chamberlin) Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y., 1976). A 10 kDa protein was shown to co-purify with RNA polymerase from T4 infected E. coli cells and is detached together with sigma70 from the core enzyme on phosphocellulose columns.<br>
A gene called asiA, coding for the 10 kDa anti-sigma70 factor of bacteriophage T4, has been identified by Orsini et al. (1993) J. Bacteriol. 175, 85 - 93. The open reading frame encoded a 90-amino-acid protein with the deduced sequence MNKNIDTVRE IITVASILIK FSREDIVENR ANFIAFLNEI GVTHEGRKLN QNSFRKIVSE LTQEDKKTLI DEFNEGFEGV YRYLEMYTNK (SEQ ID NO: 5). The asiA-encoded protein was overproduced in a phage T7 expression system and partially purified. It showed a strong inhibitory activity towards sigma70 -directed transcription by E. coli RNA polymerase holoenzyme. The nucleotide sequence of gene asiA has been deposited in the GenBank data base under accession no. M99441.<br>
Examples of proteins regulating the sigma subunit of RNA polymerase are known also from other systems. An example is the S. typhimurium flagellar regulation system, which is a complex system controlled by a set of over 50 genes grouped into 13 flagellar operons. Late operon expression is positively regulated by the fliA gene coding for the sigma factor FliA (Suzuki et al. (1978) J. Bacteriol. 133, 904; Suzuki et al. (1981) J. Bacteriol. 145,1036). On the other hand, the late operons are negatively regulated by the flgM gene. A 7.8 kDa protein has been identified as the flgM gene product and purified (Ohnishi et al. (1992) Mol. Microbiol. 6, 3149-3157). This FlgM protein was identified as an antisigma factor since it was capable to bind the FliA protein and disturbed its ability to form a complex with the RNA polymerase core enzyme.<br>
Similarly, in B. subtilis gene expression, the sigma F factor has been shown to be regulated by a 14 kDa antisigma factor encoded by the spoIIAB gene (Duncan &amp; Losick (1993) Proc. Natl. Acad. Sci. U.S.A. 90, 2325-2329), while the sigmaB factor is regulated<br><br>
by a 16 kDa anti-sigma factor encoded by the rsbW gene (Benson &amp; Haldenwang (1993) Proc. Natl. Acad. Sri. U.S.A. 90,2330-2334).<br>
The nucleotide sequences of the genes flgM, spoHAB and rsbW are available in the GenBank data base (Accession Nos.: FLGMST.PRO for flgM, SPOIIAB.PRO for spoIIAB, M34995.PRO for rsbW). The sequences do not show any gross similarity with the asiA sequence disclosed by Orisini et al. Therefore, although the different antisigma factors are functionally similar, it is not possible to anticipate that an antisigma factor from E. coli will neutralise a RNA polymerase sigma subunit from another bacterial species.<br>
BRIEF DESCRIPTION OF THE DRAWINGS Fig. 1: Proposed mode of action of an antisigma factor. Fig. 2: Map of plasmid vector pARC 8112. Fig. 3: Map of plasmid vector pARC 8175. Fig. 4: Map of plasmid vector pARC 8176. Fig. 5: Map of plasmid vector pARC 8171.<br>
Fig. 6: Reconstitution experiments of E. coli RNA polymerase with SigA and SigB proteins from Mycobacterium tuberculosis. Assay: 2.0 mg E. coli core RNA polymerase; 1.0 mg T4 DNA; 3H-UTP (specific activity 4200 cpm/nmol). Fig. 7: Map of plasmid vector pARC 8100. Fig. 8: Map of plasmid vector pARC 8115. Fig. 9: Map of plasmid vector pARC 8101. Fig. 10: Map of plasmid vector pARC 8114. Fig. 11: Map of plasmid vector pARC 8105. Fig. 12: Map of plasmid vector pARC 8180.<br>
Fig. 13: Activity of GST-Asi protein on sigma70 - dependent transcription. Assay: 0.5 mg E. coli core polymerase, 2.5 mg sigma70 protein, 1.0 mg T4 DNA and NTPs. Varying concentrations of GST-Asi was preincubated with sigma protein before addition to the reaction mixture 3H-UTP specific activity was 2800 - 3000 cpm/nmole.<br><br>
Fig. 14: Activity of purified Asi protein on sigma'70-dependent transcription. S70: E, coli<br>
sigma7 factor,  sB; M. tuberculosis SigB factor.Assay: 0.5 mg E. coli core polymerase,<br>
2.S mg sigma70 protein, 1.0 mg T4 DNA and NTPs. Varying concentrations of Asi was<br>
preincubated with sigma protein before addition to the reaction mixture. 3H-UTP specific<br>
activity was 2800 - 3000 cpm/nmole. Asi protein was obtained from S. cerevisiae I<br>
pARC 8180 expression.<br>
Fig.  15: Activity of GST-Asi protein on sigma70 - dependent transcription. sB; M.<br>
tuberculosis SigB factor. Assay: 0.5 mg E. coli core polymerase, 2.5 mg sigma ™ protein,<br>
1.0 mg T4 DNA and NTPs. Varying concentrations of GST-Asi was preincubated with<br>
sigma protein before addition to the reaction mixture. 3H-UTP specific activity was 2800<br>
- 3000 cpm/nmole. GST-Asi protein was obtained from S. cerevisiae I pARC 8180<br>
expression.<br>
Fig. 16: Map of plasmid vector pARC 8209.<br>
Fig. 17:Map of plasmid vector pARC 8198.<br>
Fig. 18: Map of plasmid vector pARC 8199.<br>
Fig. 19: Map of plasmid vector pARC 8205.<br>
Fig. 20: Map of plasmid vector pARC 8216.<br>
Fig. 21: Delineation of Asi residues essential for inhibitory activity.<br>
DISCLOSURE OF THE INVENTION<br>
It has surprisingly been shown that an antisigma factor known from E. coli, or more specifically the anti-sigma ™ protein of bacteriophage T*, can neutralise a RNA polymerase sigma70 subunit also from other bacterial species and that antisigma factors known from one bacterial species can thus be used for treatment of bacterial infections in a general sense.<br>
Consequently, in a first aspect this invention provides a compound capable of competitively binding, to a sigma70 subunit of a bacterial RNA polymerase, with the anti-sigma protein of bacteriophage T4 (SEQ ID NO: 5 or 6), and which compound id thereby inhibits the function of said sigma subunit, for use in the treatment of a bacterial<br><br>
infection. The said sigma70 subunit can e.g. be a sigma70 subunit from Escherichia coli, Mycobacterium tuberculosis or Salmonella typhimurium, for use in the treatment of a bacterial infection.<br>
In another aspect, the invention provides methods of screening peptide libraries or chemical libraries for compounds, which mimic the antisigma protein and which thereby have the capability of inhibiting the interaction of core RNA polymerase and sigma subunits.<br>
More specifically, the invention thus provides a method for screening of compounds capable of inhibiting a sigma ° subunit (i.e. a Group I sigma subunit) of a bacterium which causes infection, said method comprising determining whether a test compound will competitively bind with the anti-sigma70 protein of bacteriophage T4 (SEQ ID NO: 5 or 6), or a functionally modified form thereof, to a sigma70 subunit.<br>
The capability of the identified compounds to inhibit the interaction between a bacterial core RNA polymerase and a sigma subunit can be tested by known methods, in particular by a RNA polymerase assay, such as that disclosed in Orsini et al. (supra), wherein transcription of a DNA template, such as calf thymus DNA, T4 DNA or poly (dA-dT), is measured by the incorporation of a labelled RNA precursor such as e.g. [5-3H]UTP.<br>
The invention thus provides a method for screening of compounds capable of inhibiting a sigma70 subunit of a bacterium which causes infection, said method comprising determining whether a test compound will inhibit sigma70 dependent transcription in vitro.<br>
The invention also provides a method for screening of compounds capable of inhibiting a sigma70 subunit of a bacterium which causes infection, said method comprising the steps (i) transforming a host cell with a plasmid coding for a test compound, under<br><br>
conditions which will allow the test compound to be expressed; and (ii) determine whether the host cell will grow in presence of the expressed test compound.<br>
As another example of a screening method according to the invention, a GST-AsiA fusion protein can be bound to Glutadiione-Sepharose columns and E. coli sigma70 is bound to the GST-AsiA fusion protein immobilised on the column. Compounds to be tested are passed tijrough the column to displace the bound sigma70 unit. Peptides displacing the bound sigma70 subunit will represent the muiimum essential peptide structure needed to bring about the inhibition of the sigma70 subunit in a manner similar to the AsiA protein. Similarly, mutant AsiA proteins and possible inhibitors of sigma70 subunit can also be screened by the same principles.<br>
Consequently, in another aspect the invention provides a method of screening for compounds binding a RNA polymerase sigma subunit, said method comprising the steps (i) immobilising an antisigma compound to a matrix or a solid support; (ii) binding a RNA polymerase sigma subunit to the said compound; (in) contacting a compound to be tested with the said sigma subunit; and (iv) detecting the amount of sigma subunit released from the immobilised antisigma compound. The said antisigma compound can e.g. be a fusion protein comprising an antisigma domain and a glutathione-S-tranferase domain, and the said matrix can in such a case preferably be a glutathione-binding matrix such a Glutathione-Sepharose.<br>
In yet another aspect the invention provides a method of screening for compounds binding a RNA polymerase sigma subunit, said method comprising the steps (i) immobilising a sigma subunit on a matrix or a solid support; (ii) binding a labelled antisigma compound to the said sigma subunit; (iii) contacting a compound to be tested with the said sigma subunit; and (iv) detecting the amount of labelled antisigma compound released from die sigma subunit.<br>
In another aspect die invention provides a method of screening for compounds binding a RNA polymerase sigma subunit, said method comprising the steps (i) immobilising a<br><br>
compound to be tested on a matrix or a solid support; (ii) contacting a sigma subunit, and an antibody to the said sigma subunit, to the said compound; (iii) adding a labelled second antibody binding the said first antibody; and (iv) determining the amount of secondary antibody bound to the (antibody-sigma subunit—test compound) complex formed on the solid support.<br>
In the above methods, the RNA polymerase sigma subunit can preferably be a sigma ™ subunit from Escherichia coli, Mycobacterium tuberculosis or Salmonella typhimurium, more preferably a SigA or a SigB protein from Mycobacterium tuberculosis. Such a SigA or SigB protein can preferably have the amino acid sequence shown as SEQ ID NO: 2 and SEQ ID NO: 4, respectively, in the Sequence Listing, or can be obtainable by a process comprising (i) introducing the plasmid vector pARC 8171 or pARC 8193, respectively, into a host cell; and (ii) growing the resulting cell in or on a culture medium for expression of the protein.<br>
In the further aspect, the invention provides a process for producing a polypeptide which is a variant of an antisigma protein, said process comprising (i) producing, by methods such as site-directed mutagenesis, hydroxylamine mutagenesis or PCR mutagenesis, at least one deletion, substitution, inversion or insertion in DNA molecule coding for an antisigma protein having a sequence shown as SEQ ID NO: 5 or 6 in the Sequence Listing; (ii) introducing the resulting DNA molecule into a host cell; (iii) growing the resulting cell in or on a culture medium for expression of the polypeptide; and <iv recovering the polypeptide.></iv>
In another important aspect, the invention provides a recombinant polypeptide, preferably produced by the above described process, which polypeptide is a variant of an antisigma protein, said antisigma protein having a sequence shown as SEQ ID NO: 5 or 6 in the Sequence Listing, said variant comprising at least amino acids 1-10 and 60-70 of SEQ ID NO: 5 or 6 and in addition comprising at least one deletion, substitution, inversion or insertion, while retaining the biological activities of an antisigma protein. Such a variant can e.g. be a variant comprising at least amino acids 1-70 of SEQ ID NO:<br><br><br>
ability of the peptide to bind specifically to the sigma subunit. In addition, the peptide may or may not inhibit sigma dependent transcription.<br>
There is also provided a method of identifying the structure of a compound inhibiting the interaction between a RNA polymerase sigma subunit and a RNA polymerase core enzyme, said method comprising the steps (i) producing a (further) variant of a polypeptide which itself is an antisigma variant as defined above; and (ii) determining the capability of the produced further antisigma variant to inhibit the interaction between a RNA polymerase sigma subunit and a RNA polymerase core enzyme. The term "structure of a compound" is to be understood as the minimum essential amino acid sequence necessary for inhibition of sigma10 binding to core RNA polymerase and/or sigma  -dependent transcription.<br>
The said further antisigma valiant can also be used in known biophysical methods, such as e.g. NMR {Nuclear Magnetic Resonance Spectroscopy), X-ray crystallography or computer-assisted molecular graphics, for determination of the essential structure of a compound inhibiting the interaction between a RNA polymerase sigma subunit and a RNA polymerase core enzyme.<br>
EXAMPLES<br>
Throughout this description and in particular in the following examples, the terms "standard protocols" and "standard procedures", when used in the context of molecular cloning techniques, are to be understood as protocols and procedures found in an ordinary laboratory manual such as: Sambrook, J., Fritsch, E.F. and Maniatis, T. (1989) Molecular Cloning: A laboratory manual, 2nd Ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY.<br><br>
EXAMPLE 1:<br>
Cloning, expression and purification of E. coli sigma70 PCR primers corresponding to the 5'- and 3'-ends of the coding sequences of E. coli sigma were designed with the 5-sequence including a site for the restriction enzyme EcoRI and the 3'-sequence including a site for the restriction enzyme Sail The PCR amplified DNA fragment was restricted with the above mentioned restriction enzymes and cloned into the EcoRI- Sail sites of the expression vector pTrc99a (Amann et al. (1988) Gene 69, 301), whereafter the ligation mis was transformed into E. coli DH5a. Tiansformants harbouring the recombinant plasmid with the expected restriction profile were identified.<br>
E. coli DH5a cells harbouring the recombinant plasmid, labelled pARC 8112 (Fig. 2), were grown at +37°C in LB till an OD (600 nm) of 0.4 and induced with 1 mM 1PTG. More than 50% of the 90 kDa sigma70 protein was found as inclusion bodies as had been reported earlier (Borukhov &amp; Goldfarb (1993) Protein expression and purification 4, 503). The overexpressed sigma was purified and renatured following standard protocols.<br>
The activity of the purified sigma70 protein was confirmed by its ability to support transcription mediated by E. coli RNA polymerase core enzyme. E. coli RNA polymerase core and holoenzyme forms were purified following the protocol of Burgess and Jendriask (1975) Biochemistry 14, 4634-4638. Polymerase activity using T4 DNA as template was assayed as described by Orsrm et al. (1993) J. Bacterid. 175,85-93.<br>
EXAMPLE 2:<br>
Cloning, expression and purification of Mycobacterium tuberculosis sigma<br>
2.1. Identification of M. ftAercwiosisDNAsequerw^hcmologc^tomesigma70 gene<br>
2.1.1. PCR amplification of putative sigma70 homologues<br>
The following PCR primers were designed, based on the conserved amino acid<br>
sequences of sigma45 (a sigma70 homologue) of Bacillus subtilis and sigma ° of E. coli<br>
(Gitt, M.A. et al. (1985) J. Biol. Chem. 260,7178-7185):<br>
Forward primer:<br><br>
5'-AAG TTC AGC ACG TAC GCC ACG TGG TGG ATC-3'<br>
C	G     C<br>
Reverse primer:<br>
5"-Crr GGC CTC GAT CTG GCG GAT GCG CTC-3'.<br>
c	c	c<br>
The alternative nucleotides indicated at certain positions indicate that the primers are degenerate primers suitable for amplification of the unidentified gene.<br>
Chromosomal DNA from M. tuberculosis H37RV (ATCC 27294) was prepared following<br>
standard protocols. PCR amplification of a DNA fragment of approximately 500 bp was<br>
carried out using the following conditions:<br>
Annealing:	+55°C     1 min<br>
Denaturation:    +93°C      1 min<br>
Extension:	+73°C      2 min<br>
2.1.2. Southern hybridisation ofM. tuberculosis DNA<br>
Chromosomal DNA from M. tuberculosis H37RV (ATCC 27294), M. tuberculosis H37RA and Mycobacterium smegmatis was prepared following standard protocols and restricted with the restriction enzyme Sail. The DNA fragments were resolved on a 1% agarose gel by electrophoresis and transferred onto nylon membranes which were subjected to "Southern blotting" analysis following standard procedures. To detect homologous fragments, the membranes were probed with a radioactively labelled -500 bp DNA fragment, generated by PCR as described above.<br>
Analysis of the Southern hybridisation experiment revealed the presence of at least three hybridising fragments of approximately 4.2, 2.2 and 0.9 kb, respectively, in the Sall-digested DNA of bom of the M. tuberculosis strains. In M. smegmatis, two hybridising fragments of 4.2 and 2.2 kb, respectively, were detected. It could be concluded that there were multiple DNA fragments with homology to the known sigma70 genes.<br><br>
Similar Southern hybridisation experiments, performed with four different clinical isolates of M. tuberculosis, revealed identical patterns, indicating the presence of similar genes also in other virulent isolates of M. tuberculosis.<br>
2.2. Cloning of putative sigma70 homologues 2.2.1. Cloning of M. tuberculosis sigA<br>
A lambda gtll library (obtained from WHO) of the chromosomal DNA of M, tuberculosis Erdman strain was screened, using the 500 bp PCR probe as described above, following standard procedures. One lambda gtll phage with a 4.7 kb EcoRI insert was identified and confirmed to hybridise with the PCR probe. Restriction analysis of this 4.7 kb insert revealed it to have an internal 2.2 kb Sail fragment which hybridised with the PCR probe.<br>
The 4.7 kb fragment was excised from the lambda gt 11 DNA by EcoRI restriction, and subcloned into the cloning vector pBR322, to obtain the recombinant plasrmd pARC 8175 (Fig. 3) (Deposited under accession no. NCIMB 40738).<br>
The putative sigma70 homologue on the 2.2 kb Sail fragment was designated M. tuberculosis sigA. The coding sequence of the sigA gene was found to have an internal Sail site, which could explain the hybridization of die 0.9 kb fragment in the Southern experiments.<br>
2. 2. 2. Cloning of M. tuberculosis sigB<br>
M. tuberculosis H37Rv DNA was restricted with Sail and the DNA fragments were resolved by preparative agarose gel electrophoresis. The agarose gel piece corresponding to the 4.0 to 5.0 kb size region was cut out, and the DNA from this gel piece was extracted following standard protocols. This DNA was ligated to the cloning vector pBR329 at its Sail site, and the ligated DNA was transformed into E. coli DH5a to obtain a sub-library. Transformants of this sub-library were identified by colony blotting,<br><br>
using the PCR-derived 500 bp probe, following standard protocols. Individual transformant colonies were analyzed for their plasmid profile One of the recombinant plasmids retaining the expected plasmid size, was analyzed in detail by restriction mapping and was found to harbour the expected 4.2 kb Sail DNA fragment. This plasmid with the sigB gene on the 4.2 kb insert was designated pARC 8176 (Fig. 4) (Deposited under accession no. NCIMB 40739).<br>
2. 3. Nucleotide sequence ofM. tuberculosis sigA and sigB genes 2.3.1. Nucleotide sequence of sigA<br>
The EcoRV - EcoRI DNA fragment expected to encompass the entire sigA gene was subcloned into appropriate Ml 3 vectors and both strands of the gene sequenced by the dideoxy method. The sequence obtained is shown as SEQ ID NO: 1 in the Sequence Listing. An open reading frame (ORF) of 1580 nucleotides (positions 70 to 1650 in SEQ ID NO: 1) coding for a protein of 526 amino acids was predicted from the DNA sequence. The N-terminal amino acid has been assigned tentatively based on the first GTG (initiation codon) of the ORF.<br>
The derived amino acid sequence of the gene product SigA (SEQ ID NO: 2) showed 60% identity with the E, coli sigma70 and 70% identity with the HrdB sequence of Streptomyces coelicolor. The overall anatomy of the SigA sequence is compatible with that seen among sigma70 proteins of various organisms. This anatomy comprises a highly conserved C-terminal half, while the N-terminal half generally shows lesser homology. The two regions are linked by a stretch of amino acids which varies in length and is found to be generally unique for the protein. The SigA sequence has a similar structure, where me unconserved central stretch correspond to amino acids 270 to 306 in SEQ ID NO: 2.<br>
The N-terminal half has limited homology to E. coli sigma70, but shows resemblance to that of the sigma sigma70 homologue HrdB of S. coelicolor. The highly conserved motifs of regions 3.1, 3.2, 4.1 and 4.2 of S. coelicolor which were proposed to be involved in<br><br>
DNA binding (Lonetto, M. etal. (1992) J. Bacteriol. 174, 3843-3849) are found to be nearly identical also in the M. tuberculosis SigA sequence. The N-terminal start of the protein has been tentatively assigned, based on homologous motifs of the S. coelicolor HrdB sequence.<br>
The overall sequence similarity of the SigA and SigB amino acid sequences to known sigma70 sequences suggests assignment of the M. tuberculosis SigA to the Group I sigma70 proteins. However, SigA also shows distinct differences with known sigma70 proteins, in particular a unique and lengthy N-terminal stretch of amino acids (positions 24 to 263 in SEQ ID NO: 2), which may be essential for the recognition and initiation of transcription from promoter sequences of M. tuberculosis.<br>
2.3.2. Nucleotide sequence of sigB<br>
The nucleotide sequence of the sigB gene (SEQ ID NO: 3) encodes a protein of 323 amino acids (SEQ ID NO: 4). The N-terminal start of the protein has been tentatively identified based on the presence of the first methionine of the ORF. The ORF is thus estimated to start at position 325 and to end at 1293 in SEQ ID NO: 3. Alignment of the amino acid sequence of the SigB gene with other sigma70 proteins places the SigB gene into the Group I family of sigma70 proteins. The overall structure of the gene product SigB follows the same pattern as described for SigA. However, the SigB sequence has only 60% homology with the SigA sequence, as there are considerable differences not only within the unconsented regions of the protein, but also within the putative DNA binding regions of the SigB protein. These characteristics suggest that the SigB protein may play a distinct function in the physiology of the organism.<br>
2.4. Expression ofsigA and sigB<br>
2. 4.1. Expression of M. tuberculosis sigA gene in E. coli<br>
The N-terminal portion of the sigA gene was amplified by PCR using the following primers:<br><br><br>
Reverse Primer:<br>
5'-GTA CAG GCC AGC CTC GAT CCG CTT GGC-3'<br>
(a). A fragment of approximately 750 bp was amplified from the sigA gene construct pARC 8175. The amplified product was restricted with Ncol and BamHI to obtain a 163 bp fragment.<br>
(b). A 1400 bp DNA fragment was obtained by digestion of pARC 8175 with BamHI and EcoRV.<br>
(c). The expression plasmid pET 8ck, which is a derivative of pET 8c (Studier et al. (1990) Methods Enzyniol. 185, 60-89) in which the p- lactamase gene has been replaced by the gene conferring kanamycin resistance, was digested with Ncol and EcoRV and a fragment of approximately 4.2 kb was purified.<br>
These three fragments (a), (b) and (c) were ligated by standard methods and the product was transformed into E. coti DH5a. Individual transformants were screened for the plasmid profile following standard protocols. The transformant was identified based on the expected plasmid size (approximately 6.35 kb) and restriction mapping of the plasmid. The recombinant plasmid harbouring the coding fragment of sigA was designated pARC 8171 (Fig. 5).<br>
The plasmid pARC 8171 was transformed into the T7 expression host E. coli BL2 (DE3). Individual transformants were screened for the presence of the 6.35 kb plasmid and confirmed by restriction analysis. One of the transformants was grown at +37°C and induced with  1  mM  isopropyl-p-D-thiogalactopyranoside  (IFTG) using standard<br><br>
protocols. A specific 90 kDa protein was induced on expression. Cells were harvested by low speed centrifUgation and lysed by sonication in phosphate buffered saline, pH 7.4. The lysate was centrifugated at 100, 000 x g to fractionate into supernatant and pellet. The majority of the 70 kDa product obtained after induction with IPTG was present in the pellet fraction, indicating that the protein formed inclusion bodies.<br>
For purifying the induced sigA gene product, the cell lysate as obtained above was clarified by centrifugation at 1000 rpm in Beckman JA 21 rotor for 15 min. The clarified supernatant was layered on a 15-60% sucrose gradient and centrifugated at 100,000 x g for 60 min. The inclusion bodies sedimented as a pellet through the 60% sucrose cushion. This pellet was solubilised in 6 M guanidine hydrochloride which was removed by sequential dialysis against buffer containing decreasing concentration of guanidine hydrochloride. The dialysate was 75% enriched for the SigA protein which was purified essentially following the protocol for purification E, coli sigma as described by Brokhov, S. and Goldfarb, A. (1993) Protein expression and purification, vol. 4, 503-511.<br>
2.4.2. Expression of M. tuberculosis sigB gene in E. coli<br>
The sigB gene product was expressed and purified from inclusion bodies. The coding<br>
sequence of the sigB gene was amplified by PCR using the following primers:<br>
Forward primer comprising an Ncol restriction site:<br>
5'-  TTTCATGGCCGATGCACCCACAAGGGCC-3' MA       DAPT       RA<br>
Reverse primer comprising an EcoRI restriction site: 5'-CTT GAA TTC AGC TGG CGTACG ACC GCA-3'<br>
The amplified 920 bp fragment was digested with EcoRI and Ncol and ligated to the EcoRI- and Ncol-digested pRSET B (Kroll et al. (1993) DNA and Cell Biology 12, 441). The ligation mix was transformed into E, coli DH5a. Individual transfonnants were screened for plasmid profile and restriction analysis. The recombinant plasmid having the expected plasmid profile was designated pARC 8193.<br><br>
E. coli DH5a harbouring pARC 8193 was cultured in LB containing in 50 mg/ml ampicillin till an OD of 0.5, and induced with 1 mM IPTG at 37°C, following standard protocols. The induced SigB protein was obtained as inclusion bodies which were denatured and renatured following the same protocol as described for the SigA protein. The purified SigB protein was &gt;90% homogenous and suitable for transcription assays.<br>
2. 5. Reconstitution of E. coli core enzyme transcription activity with sigma10 homologues of M. tuberculosis<br>
E. coli RNA polymerase core enzyme was purified following the protocol of Burgess and Jendriask (1975) Biochemistry 14, 4634-4638. Increasing concentrations of purified M. tuberculosis SigA or SigB were added to the reaction mixture and the ability of the reconstitued holoenzyme to initiate transcription on the T4 template was investigated. As seen in Fig. 6, M. tuberculosis SigA and SigB were able to confer on the E. coli core enzyme the ability to initiate transcription, thus indicating the ability of SigA and SigB from M. tuberculosis to associate with uie a, P and p' subunits of E. coli RNA polymerase.<br>
EXAMPLE 3: Cloning, expression and purification of Salmonella typhimurium sigma70<br>
The coding sequence of the sigma70 of S. typhimurium was amplified using the following<br>
primers:<br>
Forward primer 5'- ATG GAA TTC AAC CCG CAG TCA CAG CTG -3'<br>
Reverse primer: 5'- TGA GTC GAC TTA ATC GTC GAG GAA GCT -3'<br>
The forward primer includes the site for the restriction enzyme EcoRI, while the reverse primer includes the sequence encoding the stop codon and a site for the restriction enzyme SalL S. typhimurium DNA was used as template and the amplified fragment digested with appropriate enzymes. This fragment was then ligated to EcoRI - Sail digested pTrc 99B (Amann et al. (1988) Gene 69, 301) and transformed into E, coli<br><br>
DH5a to obtain the recombinant plasmid pARC 8118. Plasmid DNA of pARC 8118 was digested with EcoRI - Hindm and the released 2.3 kb DNA fragment encompassing the entire coding sequence of S. typhimurium sigma70 was cloned into the EcoRI - HindHI digested pRSET B (Kroll et al. (1993) DNA and Cell Biol. 12, 441) vector which has not only the T7 promoter but also sequences encoding a his tag which is now fused at the C-terminus of S. typhimurium sigma70. The recombinant plasmid encoding the S. typhimurium sigma70 fused to a his tag was labelled pARC 8133.<br>
Plasmid DNA of pARC 8133 was then used to transform the expression host E. coli BL21 (DE3) and transformants obtained at +37°C. Cells harbouring pARC 8133 were grown in LB at +37"C till an OD of 0.4 at 600nm and induced with 1 mM IPTG for 4 hours. The cells were pelleted, suspended in buffer and sonicated, according to standard methods. The lysed cells were clarified by centrifugation at 45,000 rpm for 60 min at +4°C and the clarified supernatant was loaded directly on Ni-agarose (Pharmacia) and washed with 50 mM imidazole. The bound protein was eluted with 200 mM imidazole and dialysed overnight at +4°C against 50 mM Tris-Cl, pH 7.5. The eluate was concentrated using Amicon cone filters to 1/10 volume. The concentrated protein was cleaved with enterokinase, used at a ratio of 1:50 of the enzyme, at+25°C for 14 hr. The cleaved mixture was passed through a Ni2+ agarose column and the unbound material collected. The cleaved protein was analyzed for homogeneity by SDS-PAGE.<br>
EXAMPLE 4: Toxicity of antisigma protein 4.1. Cloning of the asiA gene<br>
The coding sequence for the asiA gene (Orsini et al., supra) was amplified from the<br>
genomic DNA of bacteriophage T+ by PCR using the forward primer designated TG<br>
137'<br>
(5'-GGC CAT GGG CAA TAA AAA CAT TGA T-3*)<br>
and a reverse primer designated TG 138"<br>
(5'-GG GGA TCC TTA TTT GTT CGT ATA CAT-31).<br><br>
PCR was performed under the following conditions:<br>
Annealing	+55°C       1 min<br>
Denaturation      +92°C       1 min<br>
Extension	+72°C       1 min<br>
The 5'-primer included the sequence of the restriction enzyme Ncol while the 3'-primer included the sequence for the restriction enzyme BamHI. The coding sequence was amplified by PCR following standard protocols and the amplified fragment ligated to Ncol - BamHI restricted pBR329 and transformed into competent cells E. coli DH5a. Recombinants were selected at +37°C as chloramphenicol sensitive, ampicillin resistant transformants. One of the transformants having the desired restriction pattern was labelled pARC 8100 (Fig. 7).<br>
The inclusion of die sequence encoding Ncol resulted in the change of me second amino acid from asparagine to glycine (SEQ ID NO: 6). The nucleotide sequence of the asiA gene cloned in pARC 8100 was verified by double stranded sequencing and found to be identical to the asiA sequence as disclosed by Orsini et al., except for the expected change of codon for the second amino acid as a result of me PCR cloning protocol used.<br>
4.2. In vivo toxicity of asiA product in E. coli<br>
(A)<br>
The Ncol - BamHI DNA fragment from pARC 8100 was ligated to pET 8c(Srudier et al. (1990) Methods Enzymol. 185, 60-89) and kanamycin resistant transformants with E. coli DH5a were selected at +37°C. One of the transformants harbouring a plasmid with the expected restriction enzyme profile was labelled pARC 8115 (Fig. 8). pARC 8115 plasmid DNA was tiien used to transform the expression host E. coli BL21 (DE3) (Srudier et al., supra) and transformants selected both at +37°C and at +30°C. However, no viable transformants could be obtained at either of the temperatures. The leaky expression from the T7 promoter in pET 8c being much higher than from pET 11 d(Km) (Srudier et al., supra), die toxicity of die asiA product could explain the non-transformability.<br><br>
iranstormants could however be obtained using E. coli BL21 (DE3)/pLysS as host, in which the leaky expression is additionally repressed by the T7 lysozyme expressed from pLysS.<br>
(B)<br>
The Ncol - BamHI 366 bp DNA fragment was obtained from pARC 8100 and ligated to<br>
the Ncol - BamHI sites of pETl Id and transformed into E. coli DH5a.<br>
Transformants were selected for kanamycin resistance. Plasmid preparations from individual transformants were digested with restriction enzymes and the correct transformant that released the 366 bp fragment after Ncol - BamHI digest was labelled pARC 8101 (Fig. 9). The transformants were selected at +37°C and appeared normal.<br>
pARC 8101 DNA was then used to transform the expression host E. coli BL26(DE3). BL26 is a lacZ" isogenic strain of BL21 (Studier et al., supra). Transformants selected for kanamycin both at +37°C and at +30°C. The transformants obtained at +37°C were morphologically sick and non-viable indicating the acute toxicity of the leaky expression of the asiA gene. In contrast, healthy colonies could be obtained when transformants were selected at +30°C where the leaky expression from the T7 promoter can be expected to be negligible.<br>
BL26 (DE3)/pLysS colonies were healthy both at +37°C and +30°C, indicating that the tight regulation of expression in this strain made the asiA gene non-toxic to the host. Transformants obtained from DH5a were also healthy both at +37°C and +30°C, indicating that the gene when transformed into a non-expression host was non-toxic.<br>
(C)<br>
The asiA gene was excised from pARC 8101 as a Xbal - BamHI DNA fragment to include the sequence for the ribosome binding site and ligated to the low copy vector, Xbal - BamHI cleaved pWKS129 (Wang &amp; Kushner (1991) Gene 100,195) and the<br>
11<br><br>
ligation mix transformed to E. coli DH5a. Recombinant plasmid harbouring the asiA gene was identified by restriction profile and labelled pARC 8114 (Fig. 10). This plasmid DNA when transformed into E. coli BL21 (DE 3) gave viable transformants both at +37°C and +30°C, since the low copy number of the plasmid reduced the level of the AsiA protein expressed through leaky expression.<br>
EXAMPLE 5: Antisigma—glutathione-S-transferase fusion protein 5.1. Cloning of fusion protein in E.coli<br>
The coding sequence for the asiA gene was excised as an Ncol - BamHI fragment from pARC 8100 and ligated to Ncol - BamHI cleaved pARC 0499. The plasmid pARC 0499 (Deposited under the Budapest Treaty with accession no. NCIMB 40664) has a. Ncol site in frame with the glutathione-S-transferase encoding sequence, enabling fusion of the N-terminus of asiA sequences. The ligation mix was transformed into E. coli DH5a and transformants selected at +37°C and +30°C. Viable colonies were obtained at both temperatures indicating the N-terminal fusion reduces the toxicity of the protein. The recombinant plasmid obtained with the sequences encoding GST-AsiA was labelled pARC 8105 (Fig. 11).<br>
The GST-AsiA fusion protein was purified as follows: E. coli DH5a harbouring pARC 8105 was grown in LB till an OD at 600 run of 0.6 at +37°C followed by the addition of 1 mM IPTG and allowing growth for a further period of 2 hours. The cells were men centrifuged at 5000 rpm for 10 min. The pelleted cells were suspended in Buffer A (PBS pH 7.5, 5 mg/ml aprotinin, 5 mg/ml leupeptin) and sonicated. The sonicate was clarified at 45,000 rpm for 10 min at +4°C and the supernatant passed through a Glutathione-Sepharose 4B column previously equilibrated with PBS. The column was washed with 5 bed volumes of 1 M urea once followed by washing with PBS (10 bed volumes). The bound protein was eluted with 10 mM glutathione and the eluate dialysed overnight against buffer containing 50 mM Tris-Cl, pH 7.5. The dialysed protein was then concentrated to 1/10 vol using Amicon cone filters. The concentrated protein was men treated with 1:50 ratio of Factor Xa protease in a buffer containing 50 mM Tris-Cl, pH<br><br>
7.5, 100 mM NaCl, 1 mM CaCfe at +25°C for 4 hours. The digested eluate was then passed through a Glutathione-Sepharose column and the unbound fraction collected and analyzed for the purity of the AsiA protein by SDS-PAGE.<br>
5.	2. Cloning of fusion protein in yeast<br>
The expression and purification of the GST-AsiA fusion protein in E. coli allowed the production of AsiA protein at a scale of 40 mg/ml culture. Larger amounts could not be obtained because of the inherent toxicity of the fusion protein. To obtain large quantities of the AsiA protein, the possibility of expressing the GST-AsiA fusion protein in the Saccharomyces host was investigated. As the Saccharomyces species do not have a sigma homologue, the AsiA protein was expected not to have toxicity effects on the transcription apparatus of the host.<br>
The gene encoding the GST-AsiA fusion protein on the plasmid pARC 8105 was<br>
amplified using the following primers:<br>
Forward primer:<br>
5'- GAA AGA TCT CAT ATG TCC CCT ATA CTA GGT-3'<br>
Reverse primer:<br>
5'-CIA AGC TIT TAT TIG TIC GTA TAC ATC TC-3'<br>
The amplified 1.0 kb fragment was acted with Bgin and Hindm which are the sites<br>
introduced by the primer sequence at the 5'- and 3'-ends of the amplified PCR fragment.<br>
The BgUI — HindlE restricted fragment was ligated into the Bgin — HindlD restricted<br>
Saccharomyces cerevisiae cloning vectors pSW6 (Pascall et al. (1991) J. Mol. Endocrinol.<br>
6,	63-70) or pYES2.0 (Invitrogen Co.). The ligation mixture was transformed into the<br>
Saccharomyces cerevisiae host CGY 1585 and transformants selected on leucine (3 mg/ml<br>
for the pSW6 vector ligation mix) or on uracil (1 mg/ml for the pYES2.0 vector ligation<br>
mix).<br>
Individual transformants from both reaction mixtures were screened for the presence of plasmid with the expected restriction profile. The recombinant plasmid obtained with the<br><br>
vector PSW6 was labelled pARC 8180 (Fig. 12) and that obtained with the pYES 2.0 vector labelled pARC 8195. S. cerevisiae CGY 1585 strain harbouring either pARC 8180 or pARC 8195 were grown using standard procedures and lysed by passing the concentrated cell suspension through a French Press. The lysate was then centrifuged at 10,000 rpm and the supernatant applied to a Glutathione- Sepharose column (Pharmacia) and the fusion protein purified following standard protocols.<br>
The purified GST-AsiA fusion protein was cleaved with Factor Xa as recommended and the AsiA protein separated. The yield of the purified AsiA protein was 500 mg/1. The activity of the GST-AsiA protein, expressed and purified from Saccharomyces, was compared to that obtained from E. coli and found to be identical in its ability to inhibit sigma70 mediated transcription of T4 template (Fig. 13). AsiA protein purified from GST-AsiA after Factor Xa cleavage also inhibited E coli sigma70 dependent transcription.<br>
The ability to overexpress and purify large quantities of the AsiA protein in Saccharomyces without toxicity problems for the host also indicates that the AsiA protein does not have a corresponding sigma70 homologue in this species. As the Saccharomyces RNA polymerase is similar to that of higher eukaryotes it also substantiates the fact that the AsiA protein is toxic only to the prokaryotic transcription apparatus.<br>
EXAMPLE 6: In vitro AsiA-sigma70 interaction assays 6.1. RNA polymerase assay<br>
The E. coli RNA polymerase assay was standardised following the protocol of Orsini et al. (J. Bacteriol. 175, 85-93, 1993) using T4 phage DNA as template to quantify sigma70 dependent transcription. E. coli RNA polymerase core enzyme was purified following me protocol of Burgess and Jendriask. (1975) Biochemistry 14, 4634-4638. 6. 2. Inhibition of E, coli transcription by AsiA protein<br><br>
Increasing concentrations of AsiA protein expressed in S. cerevisiae was added to the reaction mixture and the transcription mediated by the E. coli RNA polymerase quantified. As seen in Fig. 14, increasing concentrations of AsiA protein completely inhibited transcription mediated by E. coli sigma70.<br>
6. 3. Inhibition of £ coli transcription by GST-AsiA fusion protein<br>
The GST-AsiA fusion protein was purified from S. cerevisiae as described in Section 5.2 and used at different concentrations in the E. coli sigma70 mediated RNA polymerase transcription assay, using phage T4 DNA as template. The GST-AsiA fusion protein inhibited &gt;80% the sigma70 mediated transcription when the core enzyme was reconstituted with 2.5mg of sigma70 protein.<br>
6. 4. Reactivation of E. coli sigma70 mediated transcription<br>
Increasing concentrations of E. coli sigma70 protein were added to the reaction mixture containing 0.1 mg of AsiA. As shown in Table 1, addition of sigma70 protein could reactivate transcription mediated by E. coli RNA polymerase. This reversal of inhibition demonstrates the specific interaction of the AsiA protein with sigma70.<br><br><br>
6.5.	Effect of Salmonella typhimwium sigma70 on E.coli transcription<br>
As described in Section 6.1, the E. coli RNA polymerase activity assay was standardised with purified enzyme and phage T4 DNA as template. The RNA polymerase could be &gt;80% inhibited by 0.07 mg of purified AsiA protein.<br>
To a reaction mixture containing the E.coli RNA polymerase, T4 DNA and 0.1 mg of AsiA, increasing concentrations of purified Salmonella typhimurium sigma70 was added and the RNA polymerase activity quantified. As shown in Table 1, addition of 4 mg of S.typhimurium sigma70 could restore the activity of the RNA polymerase.<br>
6.6.	Preincubation of sigma70 with AsiA.<br>
To a reaction mixture containing E.coli RNA polymerase and T4 DNA template there was added 0.1 mg AsiA and 4mg E.coli or S.typhimurium sigma70, preincubated at +37 °C for 10 min with O.lmg of AsiA. As shown in Table 1, preincubation of the sigma70 with AsiA abolished the ability of the sigma70, from both E. coli and S. typhimurium, to activate E. coli RNA polymerase.<br>
6.7.	Effect of AsiA protein ORM. tuberculosis sigma70 mediated transcription<br>
The ability of the AsiA protein to interact with M. tuberculosis sigma70dependent transcription of the T4 DNA template was investigated, using the same experimental protocol as for the studies on E. coli sigma70<br>
To a reaction mixture containing E. coli core RNA polymerase reconstituted wim 2.5 mg ofM tuberculosis SigB protein, increasing concentrations of GST-AsiA fusion protein, or AsiA protein purified from Saccharomyces, were added and the transcription activity monitored. As seen in Figs. 14 and 15, both the GST-AsiA fusion protein and the purified AsiA protein inhibited the reconstituted E. coli RNA polymerase containing the M. tuberculosis SigB enzyme activity. This experiment clearly indicates the ability of the AsiA protein to inhibit transcription of M. tuberculosis SigB dependent transcription.<br><br>
Furthermore, the specificity of the inhibition could also be verified by the reversal of the inhibition by the addition of increasing concentrations of either E. coli sigma70 or M. tuberculosis SigB to the inhibited reaction mixture.<br>
EXAMPLE 7: Coelution of GST-AsiA and sigma70 proteins<br>
7.1.	Binding assay<br>
GST-AsiA fusion protein was bound to a Glutatbione-Sepharose matrix as earlier described. Purified E. coli sigma subunit was passed through the column. The complex was eluted with glutathione. Eluted protein was digested with Factor Xa and the digest analyzed on SDS-PAGE. Results demonstrate that E. coli sigma70 subunit specifically bound to the E. coli antiagma protein. Similar results were obtained with S. typhimurium sigma70 subunit and M. tuberculosis SigB and SigA proteins.<br>
7.2.	Modified binding assay based on truncated AsiA<br>
Studies on the deletion analysis of the AsiA protein indicated mat the C-terminal 24 amino acids are dispensable for me in vivo toxicity of the protein. The amino acid sequence of me AsiA protein has three tyrosines at positions 81, 83 and 87. A mutant gene was constructed expressing a truncated form of the AsiA, containing only the tyrosine corresponding to position 81. This mutant AsiA protein was found to be identical in its inhibitory activity to the E. coli RNA polymerase sigma dependent transcription.<br>
The presence of only one tyrosine allowed the specific I25I-labelling of the AsiA protein. This labelled protein, which was found to be identical in activity to that of the non-iodinated protein, was then used in competition experiments in which the sigma70 protein was immobilised as a GST-Sigma fusion on a Glutathione-Sepharose column, or by coating on microtitre plates, and the binding of different AsiA mutants or organic molecules to the sigma protein was monitored by competing with 125I- labelled AsiA.<br><br>
EXAMPLE 8: Competitive ELISA for quantification of AsiA-sigma70 interaction Determination of AsiA-sigma    interaction can be assessed according to the following principles:<br>
(a)	AsiA protein is immobilised on a solid surface;<br>
(b)	a sigma70 protein and an antibody to the said sigma70 protein are added and incubated at optimum conditions, optionally together with putative competitive inhibitors of the AsiA-sigma   interaction;<br>
(c)	a secondary antibody coupled to a reporter molecule is added; and<br>
(d)	the AsiA-Sigma70 subunit interaction, and the efficacy of the putative inhibitor of the said interaction, can be quantified based on the amount of reporter molecule bound to the sigma70 -antibody complex.<br>
EXAMPLE 9: Detection of protein-protein interactions 9. 1. General principles<br>
The Matchmaker Two Hybrid System ® (Clontech labs, U.S.A.) is a yeast-based genetic system for detecting protein-protein interactions in vivo. The two interacting proteins are expressed on two different plasmids, whereby one plasmid encodes a first heterologous protein, in fiision with a Gal4 protein binding domain, while the other plasmid encodes a protein binding to the said first heterologous protein, in fusion with a Gal4 protein activating domain.<br>
When both proteins are expressed in vivo, as a result of the interaction of the heterologous proteins, the Gal4 activator and binding domains interact, which result in the transcription of a downstream reporter gene, namely a p-Gal gene. Thus, heterologous protein interaction can be studied by monitoring p-galactosidase activity in vivo i.e. by the appearance of die blue colonies in the presence of X-gal.<br>
9. 2. Preparation of recombinant plasmids<br>
The gene encoding AsiA was amplified by PCR using T4 DNA as template and the<br>
forward primer "asi 22"<br><br>
(5'-A TTC TGC AGG GCT AAA TAA TTA TTT GTICGT-31)<br>
which encoded a BamHI site at the 5'-end, while the reverse primer "asi 18"<br>
(5'-AG TGG ATC CAT AAA AAC ATT GAT ACA G-31)<br>
encoded a PstI site the 3' end. The BamHI—PstI restricted product was cloned into the<br>
corresponding sites of pGAD 424, containing the Gal4 activating domain, to obtain the<br>
plasmid pARC 8209 (Fig. 16).<br>
For cloning E. coli sigma70 and S. typhimwium Sigma70 as translational fusion with the Gal4 binding domain, the 1.8 kb EcoRI—Sail fragment from pARC 8112 and pARC 8119, respectively, were cloned into EcoRI—Sail sites of pGB79, containing the Gal4 binding domain, to obtain the recombinant plasmids pARC8198 (Fig. 17) and pARC8199 (Fig. 18), respectively.<br>
The gene encoding SigB of M- tuberculosis was amplified using PCR from the construct<br>
pARC8176 using the forward primer "asi20"<br>
(5'-GCT GAA TTC GAT GCA CCC ACA AGG GCC-3')<br>
and the reverse primer "asi21"<br>
(5'-GA TGT CGA TCA GCT GGC GTA CGA C-3').<br>
The product thus obtained encodes a EcoRI site at me 5'-end and a Sail site at the 3'-end. EcoRI— Sail digested product was cloned into EcoRI— Sail site of pGB79 to obtain pARC8205(Fig.l9).<br>
Similarly, the gene encoding SigA of M tuberculosis was amplified by PCT using pARC<br>
8175 as template. The forward primer<br>
(5'-AGG GAA TTC GTG GCA GCG ACC AAA GCA-31)<br>
encoded a EcoRI site, while the reverse primer<br>
(5'-GTA CAG GCC AGC CTC GAT CCG CTT GGC-3')<br>
encoded a BamHI site. The product obtained was digested with EcoRI and BamHI and<br>
the 330 bp fragment was cloned into the EcoRI—BamHI site of pBR329 to obtain pARC<br>
8204. The 330 bp EcoRI-BamHI fragment from pARC 8204, the 1.4 kb BamHI-EcoRV<br><br>
fragment obtained from pARC 8171 and the EcoRI—EcoRV digest of pBR329 were ligated in a three way ligation reaction to obtain pARC 8214. Subsequently, the 1.7 kb EcoRI-EcoRV fragment from pARC 8214 was cloned into the EcoRI-Smal site of pGBT9 to obtain pARC 8216 (Fig. 20).<br>
In order to delineate the regions of sigma interacting with Asi, convenient restrictions sites in E. coii sigma70 coding gene were used for making translation fusion with binding domain-encoding fragment of pGBT9. The following constructs were made and assayed for beta-galactosidase activity in the presence of Asi.<br><br>
From the above experiment it is clear that the C-terminal 177 amino acids are sufficient to bind Asi. Truncated sigma70 shows higher beta-galactosidase activity than the wild type.<br>
Similar experiments with S. typhimuriutn, M. tuberculosis sigma A and M. tuberculosis sigma B gave the following results:<br><br><br>
The results show that Asi binding to the common structural features of E. coli sigma7, S. typkimurium sigma70, M. tuberculosis sigma A and M. tuberculosis sigma B enables development of sigma specific inhibitors of RNA polymerase that would be effective against a wide spectrum of bacterial strains as potential therapeutic molecules.<br>
9.3. Protein interaction studies<br>
Plasmids pARC 8209, pARC 8198, pARC 8199 pARC 8216 and pARC 8205 were individually transformed into the b-Gal host strain S. cerevisiae SFY 4526 and pairs of pARC 8209 with either pARC 8198, pARC 8199, pARC 8205 or pARC 8216 were separately cotransformed into SFY 4526. The resulting transformants were selected for p-galactosidase. All protocols were as prescribed in the Clontech manual.<br>
The cotransformants pARC 8209/pARC 8198 as well as pARC 8209/pARC 8199 expressed high levels of p-galactosidase, indicating that there was a high affinity binding between E. coli IS. typkimurium sigma70 subunits and the AsiA protein.<br>
On the other hand, p-galactosidase expression in pARC 8209/pARC 8205 and pARC 8209/pARC 8216 cotransformants was poor. This could indicate that M. tuberculosis SigA and SigB binding to AsiA is of a low affinity type. However, a lower level of expression of M. tuberculosis SigA and SigB in the cotransformants,<br><br>
compared to the expression of E. coii and S. typhimurium sigma70 in the corresponding experiments, could also account for these results.<br>
9. 4. Screening assays<br>
The above principles provide a basis for a method of identification of oligonucleotide sequences encoding peptides, which may be either related or unrelated to the AsiA peptide, and which could efficiently bind to sigma™ subunits from E. coii, S. typhimurium, M. tuberculosis and/or other housekeeping or virulence-associated sigma subunits of bacterial pathogens.<br>
The identification of such peptide structures would enable the further identification, by known methods, of peptoids, peptidomimetics and organic molecules which can be tested in transcription assays for inhibition of sigma70 dependent transcription.<br>
EXAMPLE 10: Delineation of amino acid residues involved in sigma70-AsiA<br>
interaction<br>
10.1. Strategy for selection of AsiA mutants<br>
The expression of AsiA protein is toxic to E. coii cells. Only sick colonies could be obtained on the transformation of E. coii BL26 (DE3) with pARC 8101 at +37°C (cf. Section 4.2). However at +42°C, in the presence of 5 mM IPTG, no colonies were obtained indicating the lethality of the AsiA protein even under low levels of expression. In addition mild induction (20 mM IPTG) of the T7 promoter at +30°C also resulted in sick cells.<br>
Thus following mutagenesis and transformation, healthy colonies should represent transformants in which the toxicity of the AsiA protein is lost due to a mutation in the gene encoding the AsiA protein. Identification of the sequence changes in the mutant asiA gene would identify amino acids critical for the toxicity / activity of the AsiA protein.<br><br>
10.2. Mutagenesis of the asiA gene 10 2.1.   Deletion mutants<br>
The coding sequence of the asiA gene includes an EcoRI site corresponding to the 72nd amino acid of the gene. The asiA gene was restricted with Ncol and EcoRI and the coding sequence corresponding to a protein of 72 amino acids was cloned into the T7 expression vector pARC 0400 and transformed into the expression host E. coli BL21 (DE3), Healthy colonies could be obtained at +37°C indicating the non¬toxic nature of the truncated mutant AsiA protein.<br>
10. 2. 3. Random hydroxylarninemutagenesis<br>
DNA from the plasmid pARC 8101 {Fig. 9) was mutagenised with hydroxylamine following the method of Wilke et al. (EMBO Journal 7, 2601,1988). The mutagenised DNA was transformed into E. coli BL26 (DE3) and transformants selected at +42°C in the presence of 5 mM IPTG. Transformants expressing mutant AsiA protein would grow as healthy colonies. Individual plasmid DNA was made from these colonies and the asiA gene was sequenced to identify the mutation.<br>
10.2.3. PCR mutagenesis<br>
Low fidelity PCR amplification of the asiA gene was carried out as described by Kassenbruck et al. (EMBO Journal 12,3023,1993). The mutagenised asiA gene was then cloned into the expression vector pET lld(km) and transformed into E. coli BL26 (DE3). Transformants were again selected at +42°C in the presence of 5 mM IPTG. Healthy colonies were selected as putative mutants and the plasmid DNA isolated from individual clones. The mutation on the asiA gene was identified by DNA sequencing.<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
WE CLAIM:<br>
1. A method for screening of compounds capable of inhibiting a sigma70 subunit of a bacterium which causes infection, said method comprising determining whether a srit test compound will competitively bind with the anti-sigma70 protein of bacteriophage T4 (SEQ ID NO: 5 or 6), or a functionally modified form thereof, to the sigma70 subunit.<br>
2.The method according to claim 1 comprising the steps (i) immobilising the antisigma protein to a matrix or a solid support; (ii) binding a RNA polymerase sigma subunit to the said compound; (iii) contacting a compound to be tested with the said sigma subunit; and (iv) detecting the amount of sigma subunit released from the immobilised antisigma compound.<br>
3 The method according to claim 2 wherein the said antisigma70 protein is a fusion protein comprising an antisigma domain and a glutathione-S-tranferase domain, and wherein the said matrix is a glutathione-binding matrix.<br>
4.	The  method according to claim 1 comprising the steps (i) immobilising a sigma70<br>
70<br>
subunit on a matrix or a solid support; (ii) binding a labelled antisigma protein to the said sigma subunit; (iii) contacting a compound to be tested with the said sigma70 subunit; and (iv) detecting the amount of labelled antisigma70 protein released from the sigma70 subunit.<br>
5.The  method according to claim  1  comprising the steps (i) immobilizing a<br>
•	•	70<br>
compound to be tested on a matrix or a solid support; (ii) contacting a sigma subunit, and an antibody to the said sigma70 subunit, to the said compound; (iii) adding a labelled second antibody binding the said first antibody;  and (iv)<br>
7fl<br>
determining the amount of secondary antibody bound to the (antibody sigma subunit test compound) complex formed on the solid support.<br><br>
6.	The method according to any one of claims 1 to 5 wherein the sigma70 subunit is<br>
obtained from Escherichia coli, Mycobacterium tuberculosis or Salmonella<br>
typhimurium.<br>
7.	The method according to claim 6 wherein the sigma70 subunit is a SigA or a SigB<br>
protein from Mycobacterium tuberculosis.<br>
8.	 The method according to claim 7 wherein the said SigA or SigB protein has the<br>
amino acid sequence shown as SEQ ID NO: 2 or SEQ ID NO: 4, respectively, in<br>
the Sequence Listing.<br>
9.	The method according to claim 7 wherein said SigA and Sig B proteins are<br>
obtainable by a process comprising (i) introducing the plasmid vector pARC 8171<br>
or pARC 8193, respectively, into a host cell; and (ii) growing the resulting cell in or<br>
on a culture medium for expression of the protein.<br>
10.	The method for screening of compounds capable of inhibiting a sigma subunit of<br>
a bacterium which causes infection, said method comprising determining whether a<br>
test compound will inhibit sigma70 dependent transcription in vitro.<br>
11.	The method for screening of compounds capable of inhibiting a sigma subunit<br>
of a bacterium which causes infection, said method comprising the steps (i)<br>
transforming a host cell with a plasmid coding for a test compound, under<br>
conditions which will allow the test compound to be expressed; and (ii) determine<br>
whether the host cell will grow in presence of the expressed test compound.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODczLW1hcy05NSBhYnN0cmFjdCBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">873-mas-95 abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODczLW1hcy05NSBhYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">873-mas-95 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODczLW1hcy05NSBjbGFpbXMgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">873-mas-95 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODczLW1hcy05NSBjbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">873-mas-95 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODczLW1hcy05NSBjb3JyZXNwb25kZW5jZSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">873-mas-95 correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODczLW1hcy05NSBjb3JyZXNwb25kZW5jZSBwby5wZGY=" target="_blank" style="word-wrap:break-word;">873-mas-95 correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODczLW1hcy05NSBkZXNjcmlwdGlvbiAoY29tcGxldGUpIGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">873-mas-95 description (complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODczLW1hcy05NSBkZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">873-mas-95 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODczLW1hcy05NSBkcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">873-mas-95 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODczLW1hcy05NSBmb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">873-mas-95 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODczLW1hcy05NSBmb3JtLTE5LnBkZg==" target="_blank" style="word-wrap:break-word;">873-mas-95 form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODczLW1hcy05NSBmb3JtLTI2LnBkZg==" target="_blank" style="word-wrap:break-word;">873-mas-95 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODczLW1hcy05NSBmb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">873-mas-95 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODczLW1hcy05NSBmb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">873-mas-95 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODczLW1hcy05NSBmb3JtLTYucGRm" target="_blank" style="word-wrap:break-word;">873-mas-95 form-6.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="226120-a-system-and-method-for-controllably-routing-communications-and-electrical-power-with-time-varying-current-through-a-piping-structure.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="226122-compound-of-general-formula-ii.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>226121</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>873/MAS/1995</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>02/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>09-Jan-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>10-Dec-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>12-Jul-1995</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>ASTRA RESEARCH CENTRE INDIA</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>18TH CROSS, MALLESWARAM, BANGALORE 560 003,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>DR. MEENAKSHI BALGANESH</td>
											<td>47, 5TH MAIN, POSTAL COLONY 2ND STAGE, SANJAYNAGAR, BANGALORE 560 094,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>DR. TANJORE SOUNDARARAJAN BALGANESH</td>
											<td>47, 5TH MAIN, POSTAL COLONY 2ND STAGE, SANJAYNAGAR, BANGALORE 560 094,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>UMENDER, K. SHARMA</td>
											<td>HOUSE NO. 385, 12TH C&#x27; CROSS, VYALIKAVAL, BANGALORE 560 003,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K123/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/226121-a-method-for-screening-of-compounds by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:49:51 GMT -->
</html>
