<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/226459-a-protein-from-streptococcus-groups-a-and-b by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 04:06:54 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 226459:A PROTEIN FROM STREPTOCOCCUS GROUPS A AND B</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A PROTEIN FROM STREPTOCOCCUS GROUPS A AND B</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>1. A protein comprising: (a) an amino acid sequence selected from the group consisting of SEQ IDs 3922 and 8780; (b) an amino acid sequence having 50% or greater sequence identity to SEQ IDs 3922 or 8780; or (c) a fragment of n or more consecutive amino acids from amino acid sequence SEQ ID 3922 or SEQ ID 8780 wherein n is 7.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>All documents cited herein are incorporated by reference in their entirety.<br>
TECHNICAL FIELD<br>
This invention relates to nucleic acid and proteins from the bacteria Streptococcus agalactiae (GBS) and<br>
Streptococcus pyogenes (GAS).<br>
BACKGROUND ART<br>
Once thought to infect only cows, the Gram-positive bacterium Streptococcus agalactiae (or "group B<br>
streptococcus", abbreviated to "GBS") is now known to cause serious disease, bacteremia and<br>
meningitis, in immunocompromised individuals and in neonates. There are two types of neonatal<br>
infection. The first (early onset, usually within 5 days of birth) is manifested by bacteremia and<br>
pneumonia. It is contracted vertically as a baby passes through the bird) canal. GBS colonises the vagina<br>
of about 25% of young women, and approximately 1% of infants born via a vaginal birth to colonised<br>
mothers will become infected Mortality is between 50-70%. The second is a meningitis that occurs 10 to<br>
60 days after birth. If pregnant women are vaccinated with type HI capsule so that the infants are<br>
passively immunised, the incidence of the late onset meningitis is reduced but is not entirety eliminated.<br>
The "B" in "GBS" refers to the Lancefield classification, which is based on the antigenicity of a<br>
carbohydrate which is soluble in dilute acid and called the C carbohydrate. Lancefield identified 13 types<br>
of C carbohydrate, designated A to O, that could be serologically differentiated. The organisms that<br>
most commonly infect humans are found in groups A, B, D, and G. Within group B, strains can be<br>
divided into 8 serotypes (Ia, Ib, Ia/c, II, III, IV, V, and VI) based on the structure of their<br>
polysaccharide capsule.<br>
Group A streptococcus ("GAS", S.pyogenes) is a frequent human pathogen, estimated to be present in<br>
between 5-15% of normal individuals without signs of disease. When host defences are compromised,<br>
or when the organism is able to exert its virulence, or when it is introduced to vulnerable tissues or hosts,<br>
however, an acute infection occurs. Diseases include puerperal fever, scarlet fever, erysipelas,<br>
pharyngitis, impetigo, necrotising fasciitis, myositis and streptococcal toxic shock syndrome.<br>
S.pyogenes is typically treated using antibiotics. Although S.agatactiae is inhibited by antibiotics,<br>
however, it is not killed by penicillin as easily as GAS. Prophylactic vaccination is thus preferable.<br>
Current GBS vaccines are based on polysaccharide antigens, although these suffer from poor<br>
immunogenicity. Anti-idiotypic approaches have also been used (e.g. WO99/54457). There remains a<br>
need, however, for effective adult vaccines against S.agalactiae infection. There also remains a need for<br>
vaccines against S.pyogenes infection.<br>
It is an object of the invention to provide proteins which can be used in the development of such<br>
vaccines. The proteins may also be useful for diagnostic purposes, and as targets for antibiotics.<br>
DISCLOSURE OF THE INVENTION<br>
The invention provides proteins comprising the S.agalactiae amino acid sequences disclosed in the<br>
examples, and proteins comprising the S.pyogenes amino acid sequences disclosed in the examples.<br>
These amino acid sequences are the even SEQ IDs between 1 and 10960.<br>
It also provides proteins comprising amino acid sequences having sequence identity to the S.agalactiae<br>
amino acid sequences disclosed in the examples, and proteins comprising amino acid sequences having<br>
sequence identity to the S.pyogenes amino acid sequences disclosed in the examples. Depending on the<br>
particular sequence, the degree of sequence identity is preferably greater than 50% (e.g. 60%, 70%,<br>
80%, 90%, 95%, 99% or more). These proteins include homologs, ormologs, allehc variants and<br>
functional mutants. Typically, 50% identity or more between two proteins is considered to be an<br>
indication of functional equivalence. Identity between proteins is preferably determined by the<br>
Smith-Waterman homology search algorithm as implemented in the MPSRCH program (Oxford<br>
Molecular), using an affine gap search with parameters gap open penalty 12 and gap extension<br>
penalty=1.<br>
Preferred proteins of the invention are GBS1 to GBS689 (see Table IV).<br>
The invention further provides proteins comprising fragments of the S.agalactiae amino acid sequences<br>
disclosed in the examples, and proteins comprising fragments of the S.pyogenes amino acid sequences<br>
disclosed in the examples. The fragments should comprise at least n consecutive amino acids from the<br>
sequences and, depending on the particular sequence, n is 7 or more (e.g. 8, 10, 12, 14, 16, 18, 20, 30,<br>
40, 50, 60, 70, 80, 90, 100 or more). Preferably the iragments comprise one or more epitopes from the<br>
sequence. Other preferred fragments are (a) the N-terminal signal pepudes of the proteins disclosed in<br>
the examples, (b) the proteins disclosed in the examples, but without their N-terminal signal peptides, (c)<br>
fragments common to the related GAS and GBS proteins disclosed in the examples, and (d) the proteins<br>
disclosed in the examples, but without their N-terminal amino acid residue.<br>
The proteins of the invention can, of course, be prepared by various means (e.g. recombinant<br>
expression, purification from GAS or GBS, chemical synthesis etc.) and in various forms (e.g. native,<br>
fusions, glycosylated, non-glycosylated etc). They are preferably prepared in substantially pure form<br>
(i.e. substantially free from other streptococcal or host cell proteins) or substantially isolated form.<br>
Proteins of the invention are preferably streptococcal proteins.<br>
According to a further aspect, the invention provides antibodies which bind to these proteins. These<br>
may be polyclonal or monoclonal and may be produced by any suitable means (e.g. by recombinant<br>
expression). To increase compatibility with the human immune system, the antibodies may be chimeric<br>
or humanised (e.g. Breedveld (2000) Lancet 355(9205):735-740:. Gorman &amp; Clark (1990) Semin.<br>
Immunol. 2:457-466), or fully human antibodies may be used. The antibodies may include a detectable<br>
label (e.g. for diagnostic assays).<br>
According to a further aspect, the invention provides nucleic acid composing the S.agalactiae<br>
nucleotide sequences disclosed in the examples, and nucleic acid comprising the S.pyogenes nucleotide<br>
sequences disclosed in the examples. These nucleic acid sequences are the odd SEQ IDs between 1 and<br>
10966.<br>
In addition, the invention provides nucleic acid comprising nucleotide sequences having sequence<br>
identity to the S.agalactiae nucleotide sequences disclosed in the examples, and nucleic acid comprising<br>
nucleotide sequences having sequence identity to the S.pyogenes nucleotide sequences disclosed in the<br>
examples. Identity between sequences is preferably determined by the Smith-Waterman homology<br>
search algorithm as described above.<br>
Furthermore, the invention provides nucleic acid which can hybridise to the S.agalactiae nucleic acid<br>
disclosed in the examples, and nucleic acid which can hybridise to the S.pyogenes nucleic acid disclosed<br>
in the examples preferably under 'high stringency' conditions {e.g. 65°C in 0.1xSSC, 0.5% SDS<br>
solution).<br>
Nucleic acid comprising fragments of these sequences are also provided. These should comprise at least<br>
n consecutive nucleotides from the S.agalactiae or S.pyogenes sequences and, depending on the<br>
particular sequence, n is 10 or more (e.g. 12, 14, 15, 18, 20, 25, 30, 35, 40, 50, 60, 70, 80,90, 100, 150,<br>
200 or more). The fragments may comprise sequences which are common to the related GAS and GBS<br>
sequences disclosed in the examples.<br>
According to a further aspect, the invention provides nucleic acid encoding the proteins and protein<br>
fragments of the invention.<br>
The invention also provides: nucleic acid comprising nucleotide sequence SEQ ID 10967; nucleic acid<br>
comprising nucleotide sequences having sequence identity to SEQ ID 10967; nucleic acid which can<br>
hybridise to SEQ ID 10967 (preferably under 'high stringency' conditions); nucleic acid comprising a<br>
fragment of at least n consecutive nucleotides from SEQ ID 10967, wherein n is 10 or more e.g. 12, 14,<br>
15, 18, 20,25, 30,35,40, 50, 60, 70, 80, 90, 100, 150, 200, 250, 300, 350,400, 450,500,600, 700, 800,<br>
900,1000,1500,2000,3000,4000, 5000,10000,100000,1000000 or more<br>
Nucleic acids of the invention can be used in hybridisation reactions (e.g. Northern or Southern blots, or<br>
in nucleic acid microarrays or 'gene chips') and amplification reactions (e.g. PCR, SDA, SSSR, LCR,<br>
TMA, NASBA etc.) and other nucleic acid techniques.<br>
It should also be appreciated that the invention provides nucleic acid comprising sequences<br>
complementary to those described above (e.g. for antisense or probing, or for use as primers).<br>
Nucleic acid according to the invention can, of course, be prepared in many ways (e.g. by chemical<br>
synthesis, from genomic or cDNA libraries, from the organism itself etc.) and can take various forms<br>
(e.g. single stranded, double stranded, vectors, primers, probes, labelled etc.). The nucleic acid is<br>
preferably in substantially isolated form.<br>
Nucleic acid according to the invention may be labelled e.g. with a radioactive or fluorescent label. This<br>
is particularly useful where the nucleic acid is to be used in nucleic acid detection techniques e.g. where<br>
the nucleic acid is a primer or as a probe for use in techniques such as PCR, LCR, TMA, NASBA etc.<br>
In addition, the term "nucleic acid" includes DNA and RNA, and also their analogues, such as those<br>
containing modified backbones, and also peptide nucleic acids (PNA) etc.<br>
According to a further aspect, the invention provides vectors comprising nucleotide sequences of the<br>
invention (e.g. cloning or expression vectors) and host cells transformed with such vectors.<br>
According to a further aspect, the invention provides compositions comprising protein, antibody, and/or<br>
nucleic acid according to the invention. These compositions may be suitable as immunogenic<br>
compositions, for instance, or as diagnostic reagents, or as vaccines.<br>
The invention also provides nucleic acid, protein, or antibody according to the invention for use as<br>
medicaments (e.g. as immunogenic compositions or as vaccines) or as diagnostic reagents. It also<br>
provides the use of nucleic acid, protein, or antibody according to the invention in the manufacture of: (i)<br>
a medicament for treating or preventing disease and/or infection caused by streptococcus; (ii) a<br>
diagnostic reagent for detecting the presence of streptococcus or of antibothes raised against<br>
streptococcus; and/or (iii) a reagent which can raise antibothes against streptococcus. Said<br>
streptococcus may be any species, group or strain, but is preferably S.agalactiae, especially serotype<br>
III or V, or S.pyogenes. Said disease may be bacteremia, meningitis, puerperal fever, scarlet fever,<br>
erysipelas, pharyngitis, impetigo, necrotising fasciitis, myositis or toxic shock syndrome.<br>
The invention also provides a method of treating a patient, comprising administering to the patient a<br>
therapeutically effective amount of nucleic acid, protein, and/or antibody of the invention. The patient<br>
may either be at risk from the disease themselves or may be a pregnant woman ('maternal immunisation'<br>
e.g. Glezen &amp; Alpers (1999) Clin. Infect. Dis. 28:219-224).<br>
Administration of protein antigens is a preferred method of treatment for inducing immunity.<br>
Administration of antibothes of the invention is another preferred method of treatment This method of<br>
passive immunisation is particularly useful for newborn children or for pregnant women. This method<br>
will typically use monoclonal antibothes, which will be humanised or fully human.<br>
The invention also provides a kit comprising primers (e.g. PCR primers) for amplifying a template<br>
sequence contained within a Streptococcus {e.g. S.pyogenes or S.agalactiae) nucleic acid sequence, the<br>
kit comprising a first primer and a second primer, wherein the first primer is substantially complementary<br>
to said template sequence and the second primer is substantially complementary to a complement of said<br>
template sequence, wherein the parts of said primers which have substantial complementarity define the<br>
termini of the template sequence to be amplified. The first primer and/or the second primer may include<br>
a detectable label (e.g. a fluorescent label).<br>
The invention also provides a kit comprising first and second single-stranded ohgonucleotides which<br>
allow amplification of a Streptococcus template nucleic acid sequence contained in. a single- or double-<br>
stranded nucleic acid (or mixture thereof), wherein: (a) the first oligonucleotide comprises a primer<br>
sequence which is substantially complementary to said template nucleic acid sequence; (b) the second<br>
oligonucleotide comprises a primer sequence which is substantially complementary to the complement<br>
of said template nucleic acid sequence; (c) the first oligonucleotide and/or the second oligonucleotide<br>
comprises) sequence which is not compementary to said template nucleic acid; and (d) said primer<br>
sequences define the termini of the template sequence to be amplified. The non-complementary<br>
sequence(s) of feature (c) are preferably upstream of (i.e. 5' to) the primer sequences. One or both of<br>
these (c) sequences may comprise a restriction site (e.g. EP-B-0509612) or a promoter sequence (e.g.<br>
EP-B-0505012). The first oligonucleotide and/or the second oligonucleotide may include a detectable<br>
label (e.g. a fluorescent label).<br>
The template sequence may be any part of a genome sequence (e.g. SEQ ED 10967). For example, it<br>
could be a rRNA gene (e.g. Turenne et al. (2000) J. Clin. Microbiol. 38:513-520; SEQ IDs 12018-12024<br>
herein) or a protein-coding gene. The template sequence is preferably specific to GBS.<br>
The invention also provides a computer-readable medium (e.g. a floppy disk, a hard: disk, a CD-ROM, a<br>
DVD etc.) and/or a computer database containing one or more of the sequences in the sequence listing.<br>
The medium preferably contains SEQ ID 10967.<br>
The invention also provides a hybrid protein represented by the formula NH2-A-[-X-L-]-B-COOH,<br>
wherein X is a protein of the invention, L is an optional linker ammo acid sequence, A is an optional<br>
N-terminal amino acid sequence, B is an optional C-terminal amino acid sequence, and n is an integer<br>
greater than 1. The value of n is between 2 and x, and the value of x is typically 3, 4, 5, 6, 7, 8, 9 or 10.<br>
Preferably n is 2, 3 or 4; it is more preferabry 2 or 3; most preferably, n - 2. For each n instances, -X-<br>
may be the same or different For each n instances of [-X-L-], linker amino acid sequence -L- may be<br>
present or absent. For instance, when n=2 the hybrid may be NH2-X1-L1-X2-L2-COOH, NH2-X1-X2-<br>
COOH, NH2-X1L1-X2-COOH, NH2-X1-X1-L2-COOH, etc. Linker amino acid sequence(s) -L- will<br>
typically be short (eg. 20 or fewer amino acids i.e. 19, 18, 17, 16, 15,14, 13, 12, 11, 10, 9, 8, 7, 6, 5, 4,<br>
3,2,1). Examples include short peptide sequences which facilitate cloning, poly-glycine linkers (i.e. Glyn<br>
where n = 2,3, 4, 5, 6, 7, 8, 9, 10 or more), and histidine tags (i.e. Hisn where n = 3, 4, 5, 6, 7, 8, 9, 10<br>
or more). Other suitable linker amino acid sequences will be apparent to those skilled in the art -A- and -<br>
B- are optional sequences which will typically be short (e.g. 40 or fewer ammo acids i.e. 39, 38, 37, 36,<br>
35, 34, 33, 32, 31, 30, 29, 28, 27, 26, 25, 24, 23, 22, 21, 20, 19, 18, 17, 16, 15, 14, 13, 12, 11, 10, 9, 8,<br>
7, 6, 5, 4, 3, 2, 1). Examples include leader sequences to direct protein trafficking, or short peptide<br>
sequences which facilitate cloning or purification (e.g. histidine tags i.e. His, where n = 3, 4, 5, 6, 7, 8, 9,<br>
10 or more). Other suitable N-terminal and Gterminal amino acid sequences will be apparent to those<br>
skilled in the art in some embodiments, each X will be a GBS sequence; in others, mixtures of GAS and<br>
GBS will be used.<br>
According to further aspects, the invention provides various processes.<br>
A process for producing proteins of the invention is provided, comprising the step of culturing a host<br>
cell of to fee invention under conditions which induce protein expression.<br>
A process for producing protein or nucleic acid of the invention is provided, wherein the protein or<br>
nucleic acid is synthesised in part or in whole using chemical means.<br>
A process for detecting polynucleotides of the invention is provided, comprising the steps of: (a)<br>
contacting a nucleic probe according to the invention with a biological sample under hybridising<br>
conditions to form duplexes; and (b) detecting said duplexes.<br>
A process for detecting Streptococcus in a biological sample (e.g. blood) is also provided, comprising<br>
the step of contacting nucleic acid according to the invention with the biological sample under<br>
hybridising conditions. The process may involve nucleic acid amplification ie.g. PCR, SDA, SSSR,<br>
LCR, TMA, NASBA etc.) or hybridisation (e.g. microarrays, blots, hybridisation with a probe in<br>
solution etc.). PCR detection of Streptococcus in clinical samples, in particular S.pyogenes, has been<br>
reported [see e.g. Louie et al. (2000) CMAJ 163:301-309; Louie et al. (1998) J. Clin. Microbiol.<br>
36:1769-1771]. Clinical assays based on nucleic acid are described in general in Tang et al. (1997) Clin.<br>
Chem. 432021-2038.<br>
A process for detecting proteins of the invention is provided, comprising the steps of: (a) contacting an<br>
antibody of the invention with a biological sample under conditions suitable for the formation of an<br>
antibody-antigen complexes; and (b) detecting said complexes.<br>
A process for identifying an amino acid sequence is provided, comprising the step of searching for<br>
putative open reading frames or protein-coding regions within a genome sequence of S.agalactiae. This<br>
will typically involve in silico searching the sequence for an initiation codon and for an in-frame<br>
termination codon in the downstream sequence. The region between these initiation and termination<br>
codons is a putative protein-coding sequence. Typically, all six possible reading frames will be searched.<br>
Suitable software for such analysis includes ORFFINDER (NCBI), GENEMARK [Borodovsky &amp;<br>
McIninch (1993) Computers Chem. 17:122-133), GLIMMER [Salzberg et al. (1998) Nucleic Acids Res.<br>
26:544-548; Salzberg et al. (1999) Genomics 59-24-31; Delcher et al. (1999) Nucleic Acids Res. 27:4636-<br>
4641], or other software which uses Markov models [e.g. Shmarkov et al. (1999) Bioinformatics<br>
15:874-876]. The invention also provides a protein comprising the identified amino acid sequence. These<br>
proteins can then expressed using conventional techniques.<br>
The invention also provides a process for determining whether a test compound binds to a protein bf the<br>
invention. If a test compound binds to a protein of the invention and this binding inhibits the life cycle of<br>
the GBS bacterium, men the test compound can be used as an antibiotic or as a lead compound for the<br>
design of antibiotics. The process will typically comprise the steps of contacting a test compound with a<br>
protein of the invention, and determining whether the test compound binds to said protein. Preferred<br>
proteins of the invention for use in these processes are enzymes(e.g. tRNA synthetases), membrane<br>
transporters and ribosomal proteins. Suitable test compounds include proteins, polypeptides,<br>
carbohydrates, lipids, nucleic acids (e.g. DNA, RNA, and modified forms thereof), as well as small<br>
organic compounds (e.g. MW between 200 and 2000 Da). The test compounds may be provided<br>
indrvidually, but will typically be port of a library (e.g. a combinatorial library). Methods for detecting a<br>
binding interaction include NMR, filter-binding assays, gel-retardation assays, displacement assays,<br>
surface plasmon resonance, reverse two-hybrid etc. A compound which binds to a protein of the<br>
invention can be tested for antibiotic activity by contacting the compound with GBS bacteria and then<br>
monitoring for inhibition of growth. The invention also provides a compound identified using these<br>
methods.<br>
The invention also provides a composition comprising a protein or the invention and one or more of the<br>
following antigens:<br>
- a protein antigen from Helicobacter pylori such as VacA, CagA, NAP, HopX, HopY [e.g.<br>
WO98/04702] and/or urease.<br>
- a protein antigen from N.meningitidis serogroup B, such as those in WO99/24578, WO99/36544,<br>
WO99/57280, WOOO/22430, Tettelin et al. (2000) Science 287:1809-1815, Pizza et al. (2000)<br>
Science 287:1816-1820 and WO96/29412, with protein '287' and derivatives being particularly<br>
preferred.<br>
- an outer-membrane vesicle (OMV) preparation from N.meningitidis serogroup B, such as those<br>
disclosed in WO01/52885; Bjune et al. (1991) Lancet 338(8775):1093-1096; Fukasawa et al. (1999)<br>
Vaccine 17:2951-2958; Rosenqvist et al. (1998) Dev. Biol. Stand. 92:323-333 etc<br>
- a saccharide antigen from N.meningitidis serogroup A, C, W135 and/or Y, such as the<br>
oligosaccharide disclosed in Costantino et al. (1992) Vaccine 10:691-698from serogroup C [see<br>
also Costantino et al. (1999) Vaccine 17:1251-1263].<br>
- a saccharide antigen from Streptococcus pneumoniae [e.g. Watson (2000) Pediatr Infect Dis J<br>
19:331-332; Rubin (2000) Pediatr Clin North Am 47:269-285, v; Jedrzejas (2001) Microbiol Mol<br>
Biol Rev 65:187-207].<br>
- an antigen from hepatitis A virus, such as inactivated virus (e.g. Bell (2000) Pediatr Infect Dis J<br>
19:1187-1188; Iwarson (1995) APM1S 103:321-326].<br>
- an antigen from hepatitis B virus, such as the surface and/or core antigens [e.g. Gerlich et al. (1990)<br>
Vaccine 8 Suppl:S63-68 &amp; 79-80].<br>
- an antigen from hepatitis C virus [e.g. Hsu et al. (1999) Clin Liver Dis 3:901-915].<br>
- an antigen from Bordetella pertussis, such as pertussis holotoxin (PT) and filamentous<br>
haemagglutinin (FHA) from B.pertussis, optionally also in combination with pertactin and/or<br>
agglutinogens 2 and 3 [e.g Gustafsson et al. (1996) N. Engl. J. Med. 334:349-355; Rappuoli et al.<br>
(1991) TIBTECH 9:232-238].<br>
- a diphtheria antigen, such as a diphtheria toxoid [e.g chapter 3 of Vaccines (1988) eds. Plotkin &amp;<br>
Mortimer. ISBN 0-7216-1946-0] e.g. the CRM197 mutant [e.g Del Guidice et al. (1998) Molecular<br>
Aspects of Medicine 19:1-70].<br>
- a tetanus antigen, such as a tetanus toxoid [e.g chapter 4 of Plotkin &amp; Mortimer],<br>
- a saccharide antigen from Haemophilus influenzae B.<br>
- an antigen from N.gonorrhoeae [e.g WO99/24578, WO99/36544, WO99/57280].<br>
- an antigen from Chlamydia pneumoniae [e.g PCT/IB01/01445; Kabnan et al. (1999) Nature<br>
Genetics 21:385-389; Read et al. (2000) Nucleic Acids Res 28:1397-406; Shirai et al. (2000) J.<br>
Infect Dis. 181(Suppl 3):S524-S527; WO99/27105; WO00/27994; WOOO/37494]<br>
- an antigen from Chlamydia trachomatis [e.g WO99/28475].<br>
- an antigen from Porphyromonas gingivalis [e.g Ross et al. (2001) Vaccine 19:4135-4142].<br>
- polio antigen(s) [e.g Sutter et al. (2000) Pediatr Clin North Am 47:287-308; Zimmerman &amp; Spann<br>
(1999) Am Font Physician 59:113-118, 125-126] such as IPV or OPV.<br>
- rabies antigen(s) [e.g Dreesen (1997) Vaccine 15 Suppl:S2-6] such as lyophilised inactivated virus<br>
[e.g MMWR Morb Mortal Wkly Rep 1998 Jan 16;47(1):12,19; RabAvert™].<br>
- measles, mumps and/or rubella antigens [e.g chapters 9,10 &amp; 11 of Plotkin &amp; Mortimer].<br>
- influenza antigens) [e.g chapter 19 of Plotkin &amp; Mortimer], such as the haemagglutinin and/or<br>
neuraminidase surface proteins.<br>
- an antigen from Moraxella catarrhalis [e.g McMichael (2000) Vaccine 19 Suppl l:S101-107].<br>
- an antigen from Staphylococcus aureus [e.g Kuroda et al. (2001) Lancet 357(9264): 1225-1240;<br>
see also pages 1218-1219].<br>
Where a saccharide or carbohydrate antigen is included, it is preferably conjugated to a carrier protein in<br>
order to enhance immunogencity [e.g Ramsay et al. (2001) Lancet 357(9251):195-196; Lindberg (1999)<br>
Vaccine 17 Suppl 2:S28-36; Conjugate Vaccines (eds. Cruse et al.) ISBN 3805549326, particularly vol.<br>
10:48-114 etc.]. Preferred carrier proteins are bacterial toxins or toxoids, such as diphtheria or tetanus<br>
toxoids. The CRM197 diphtheria toxoid is particularly preferred Other suitable carrier proteins include<br>
the N.meningitidis outer membrane protein [e.g EP-0372501], synthetic peptides [e.g EP-0378881, EP-<br>
0427347], heat shock proteins [e.g WO93/17712], pertussis proteins [e.g. WO98/58668; EP-0471177],<br>
protein D from H.influenzae [e.g WO00/56360], toxin A or B from C.difficile [e.g WO00/61761], etc<br>
Any suitable conjugation reaction can be used, with any suitable linker where necessary.<br>
Toxic protein antigens may be detoxified where necessary (e.g. detoxification of pertussis toxin by<br>
chemical and/or genetic means).<br>
Where a diphtheria antigen is included in the composition it is preferred also to include tetanus antigen<br>
and pertussis antigens. Similarly, where a tetanus antigen is included it is preferred also to include<br>
diphtheria and pertussis antigens. Similarly, where a pertussis antigen is included it is preferred also to<br>
include diphtheria and tetanus antigens.<br>
Antigens are preferably adsorbed to an aluminium salt.<br>
Antigens in the composition will typically be present at a concentration of at least 1µg/ml each. In<br>
general, the concentration of any given antigen will be sufficient to elicit an immune response against that<br>
antigen.<br>
The invention also provides compositions comprising two or more proteins of the present invention.<br>
The two or more proteins may comprise GBS sequences or may comprise GAS and GBS sequences.<br>
A summary of standard techniques and procedures which may be employed to perform the invention<br>
(e.g. to utilise the disclosed sequences for vaccination or diagnostic purposes) follows. This summary is<br>
not a limitation on the invention but, rather, gives examples that may be used, but are not required.<br>
General<br>
The practice of the present invention will employ, unless otherwise indicated, conventional techniques of<br>
molecular biology, microbiology, recombinant DNA, and immunology, which are within the skill of the<br>
art Such techniques are explained fully in the literature eg. Sambrook Molecular Cloning; A Laboratory<br>
Manual, Second Edition (1989); DNA Cloning, Volumes I and II (D.N Glover ed. 1985);<br>
Oligonucleotide Synthesis (M.J. Gait ed, 1984); Nucleic Acid Hybridization (B.D. Hames &amp; S.J.<br>
Higgins eds. 1984); Transcription and Translation (B.D. Hames &amp; S.J. Higgins eds. 1984); Animal<br>
Cell Culture (R.I. Freshney ed 1986); Immobilized Cells and Enzymes (IRL Press, 1986); B. Perbal, A<br>
Practical Guide to Molecular Cloning (1984); the Methods in Enzymology series (Academic Press,<br>
Inc.), especially volumes 154&amp; 155; Gene Transfer Vectors for Mammalian Cells (J.H Miller and M.P.<br>
Calos eds. 1987, Cold Spring Harbor Laboratory); Mayer and Walker, eds. (1987), Immunochemical<br>
Methods in Cell and Molecular Biology (Academic Press, London); Scopes, (1987) Protein<br>
Purification: Principles and Practice, Second Edition (Springer-Verlag, N.Y.), and Handbook of<br>
Experimental Immunology, Volumes I-IV (D.M. Weir and C. C. Blackwell eds 1986).<br>
Standard abbreviations for nucleotides and amino acids are used in this specification.<br>
Definitions<br>
A composition containing X is "substantially free of Y when at least 85% by weight of the total X+Y in the composition is X.<br>
Preferably, X comprises at least about 90% by weight of the total of X+Y in the composition, more preferable at least about 95%<br>
or even 99% by weight<br>
The tarn "comprising" means "including" as well as "consisting" e.g. a composition "comprising" X may consist exclusively of<br>
X or may include something additional e.g. X+Y.<br>
The term "heterologous" refers to two biological components that are not found together in nature. The components may be host<br>
cells, genes, or regulatory regions, such as promoters. Although the beterologous components are not found together in nature,<br>
they can function together, as when a promoter heterologous to a gene is operably linked to the gene. Another example is where a<br>
streptococcus sequence is heterologous to a mouse host cell. A further examples would be two epitopes from the same or<br>
differ proteins which have been assembled in a single protein in an arrangement not found in nature.<br>
An "origin of replication" is a polynucleotide sequence that initiates and regulates replication of polynucleotides, such as an<br>
expression vector. The origin of replication behaves as an autonomous unit of polynucleotide replication within a cell, capable of<br>
replication under its own control. An origin of replication may be needed for a vector to replicate in a particular host cell. With<br>
certain origins of replication, an expression vector can be reproduced at a high copy number in the presence of the qppropriate<br>
protein within the cell. Examples of origins are the autonomously replicating sequences, which are effective in yeast; and the viral<br>
T-antigen, effective in OOS-7 cells.<br>
A "mutant" sequence is defined as DNA, RNA or amino acid sequence differing from but having sequence identify with the<br>
native or disclosed sequence. Depending on the particular sequence, the degree of sequence identity between the native or<br>
disclosed sequence and the mutant sequence is preferably greater than 50% (e.g. 60%, 70%, 80%, 90%, or more,<br>
calculated using the Smith-Watarman algorithm as described above). As used herein, an "allelic variant" of a nucleic acid<br>
molecule, or region, for which nucleic acid sequence is provided herein is a nucleic acid molecule, or region, that occurs essentially<br>
at the same locus in the genome of another or second isolate, and that, due to natural variation caused by, for example, mutation<br>
or recombination, has a similar but not identical nucleic acid sequence. A coding region allelic variant typically ensodes a protein<br>
having similar activity to that of the protein encoded by the gene to which it is being compared. An allelic variant can also<br>
comprise an alteration in the 5' or 3' untranslated region of the gene, such as in regulatory control regions (e.g. see US patent<br>
5,753,235).<br>
Expression system<br>
The streptococcus nucleotide sequences can be expressed in a variety of different expression systems; for example those used<br>
with mammalian cells, beculoviruses, plants, bacteria, and yeast.<br>
Mammalian expression systems are known in the art. A mammalian promoter is any DNA sequence capable of binding<br>
mammalian RNA polymerase and initiating the downstream (3') transcription of a coding sequence (e.g. structural gene) into<br>
mRNA. A promoter will have a transcription initiating region, which is usually placed proximal to the 5' end of the coding<br>
sequence, and a TATA box, usually located 25-30 base pairs (bp) upstream of the transcription initiation site. The TATA box is<br>
thought to direct RNA polymerase II to begin RNA synthesis at the correcsst site. A mammalian promoter will also contain an<br>
upstream promoter element, usually located within 100 to 200 bp upstream of the TATA box. An upstream promoter element<br>
determines the rate at which transcription is initiated and can act in either orientation [Sambrook et al. (1989) "Expression of<br>
Cloned Genes in Mammalian Cells." In Molecular Cloning: A Laboratory Manual, 2nd ed.]<br>
Mammalian viral genes ate often highly expressed and have a broad host range; therefore sequences encoding mammalian viral<br>
genes provide particularly useful promoter sequences. Examples include the SV40 early promoter, mouse mammary tumor virus<br>
LTR promoter, adenovirus major late promoter (Ad MLP), and herpes simplex virus promoter. In addition, sequences derived<br>
from non-viral genes, such as the munite metallotheionein gene, also provide useful promoter sequences. Expression may be either<br>
constitutive or regulated (inducible), depending on the promoter can be induced with glucocorticoid in hormone-responsive cells.<br>
The presence of an enbancer element (enhancer), combined with the promoter elements described above, will usually increase<br>
expression levels. An enhancer is a regulatory DNA sequence that can stimulate transcription up to 1000-fold when linked to<br>
homologous or heterotogous promoters, with synthesis beginning at the normal RNA start site. Enhancers are also active when<br>
they are placed upstream or downstream from the transcription initiation site, in either normal or flipped orientation or at a<br>
distance of more than 1000 nucleotides from the promoter [Maniatis et al. (1987) Science 236:1237; Alberts et al. (1989)<br>
Molecular Biology of the Cell, 2nd ed]. Enhancer elements derived fiom viruses may be particularly useful, because they<br>
usually have a broader host range. Examples include the SV40 early gene enhancer [Dijkema et al (1985) EMBO J. 4:761] and<br>
the enhancer/promoters derived from the long terminal repeat (LTR) of the Rous Sarcoma Vine [Gorman et al. (1982b) Proa<br>
Natl. Acad. Sci. 79:6777] and fiom human cytomegalovirus (Boshart et al. (1985) Cell 41:521]. Additioally, some enhancers<br>
are regulatable and become active only in the presence of an inducer, such as a hormaone or metal ion [Sassone-Corsi an Borelli<br>
(1986) Trends Genet. 2:215; Maniatis etal. (1987) Science 236:1237].<br>
A DNA molecule may be expressed intracelllularly in mammalian cells. A promoter sequence may be ditectly linked with the<br>
DNA molecule, in which case the first amino acid at the N-terminus of the recombinant protein will always be a methionine, which<br>
is encoded by the ATG start codon. If desired, the N-tenninus may be cleaved from the protein by in vitro incubation with<br>
cyanogen bromide.<br>
Alternatively, foreign proteins can also be secreted from the cell into the growth media by creating chimeric DNA molecules that<br>
encode a fusion protein comprised of a leader sequence fiagment that provides for secretion of the foreign protein in mammalian<br>
cells. Preferably, there are processing sites encoded between the leader fragment and the foreign gene that can be cleaved either<br>
in vivo or in vitro. The leader sequence fragment usually encodes a signal peptide comprised of hydrophobic amino acids which<br>
direct the secretion of the protein from the cell. The adenovirus triparite leader is an example of a leader sequence that provides<br>
for secretion of a foreign protein in mammalian cells.<br>
Usually, translation termination and polyadenylation sequences recongnized by mammalian cells are regulatory regions located 3'<br>
to the translation stop codon and thus, together with the promoter elements, flank the coding sequence. The 3' terminus of the<br>
mature mRNA is formed by site specific post transcriptional cleavage and polydenylation [Binstiel et al. (1985) cekk 41:349;<br>
Proudfoot and Whitelaw (1988) "Termination and 3' end processing of eukaryotic RNA. In Transcription and splicing (ed.<br>
B.D. Hames and D.M. Glover); Proudfoot (1989) Trends Biochem. Sci. 14:105]. These sequences direct the transcription of an<br>
mRNA which can be translated into the polypeptide encoded by the DNA. Examples of transcription terminater/polyadenylation<br>
signals include those derived from SV40 [Sambrook et al (1989) "Exprission of cloned genes in cultured mammalian cells." In<br>
Molecular Cloning: A Laboratory Manual].<br>
Usually, the above described components, comprising a promoter, polyadenylation signal, and transcription termination sequence<br>
are put together into expression constructs. Enhancers, introns with funcional splice donor and acceptor sites, and leader<br>
sequences may also be included in an expression construct, if desired. Expression constructs are often maintained in a replicon,<br>
such as an extrachromosomal element (e.g. plasmids) capable of stable maintenance in a host, such as mammalian cells or<br>
bacteria. Mammalian replication systems incllude those derived from animal viruses, which require trans acting factors to replicate.<br>
For example, plasmids containing the replication systems of papovaviruses, such as SV40 [Gluzman (1981) Cell 25:175] or<br>
polyomavirus, replication systems, thus allowing it to be maintained, for example, in mammalian cells for expression and in a<br>
mammalian replicas include those derived from bovine papillomavirus and Epstsin-Barr virus. Additionally, the replicon may<br>
have two replicaton systems, thus allowing it to be mainted, for example, in mammalian cells for expression and in a<br>
prokaryotic host far cloning and amplification. Examples of such mammalian-bacteria shuttle vectors include pMT2 [Kaufman et<br>
al. (1989) Mol. Cell. Biol. 9.946] and pHEBO [Shimizu et al. (1986) Mol. Cell Biol. 6:1074].<br>
The transformation procedure used depends upon the host to be transformed Methods for introduction of heterologous<br>
polynucleotides into mammalian cells are known in the art and include dextran-mediated transfection, calcium phosphate<br>
precipitation, polybrene mediated transtection, protoplast fusion, electroporation, encapsulation of the polynucleotide(s) in<br>
liposomes, and direct microinjection of the DNA into nuclei.<br>
Mammalian cell lines available as hosts for expression are known in the art include many immortallized cell lines available from<br>
the American Type Culture Collection (ATCC), including but not limited to, Chinese hermater ovary (CHO) cells, Hela cells,<br>
baby hamster kidney (BHK) cells, monkey kidney cells (COS), human hepatocellular carcinoma cell (e.g. Hep G2), and a<br>
number of other cell lines.<br>
ii Baculovirus Systems<br>
The polynucleotide encoding the protein can also be inserted into a suitable insect expression vector, and is operably linked to the<br>
control elements within that vector. Vector construction employs techniques which are known in fee art. Generally, the<br>
components of the expression system include a transfer vector, usually a bacterial plasmid, which contains both a fragment of the<br>
baculovirus genome, and a convenient restriction site for insertion of the heterologous gene or genes to be expressed; a wild type<br>
baculovirus with a sequence homologous to the baculovirus specific fragment in the transfer vector (this allows for the<br>
homologous recombination of the heterologous gene in to the baculovirus genome); and appropriate insect host cells and growth<br>
media.<br>
After inserting the DNA sequence encoding the protein into the transfer vector, the vector and the wild type viral genome are<br>
transfected into an insect host cell where the vector and viral genome are allowed to recombine. The packaged recombinent virus<br>
is expressed and recombinant plaques are identified and purified. Materials and methods for baculovirus/insect cell expression<br>
systems are commercially available in kit form from, inter alia, Invitrogen, San Diego CA("MaxBac" kit). These techniques are<br>
generally known to those skilled in the art and fully described in Summers and Smith, Texas Agricultural Experiment Station<br>
Bulletin No. 1555 (1987) (hereinafter "Summers and Smith").<br>
Prior to inserting the DMA sequence encoding the protein into the baculovirus genome, the above described components,<br>
comprising a promoter, leader (if desired), coding sequence, and transcription termination sequence, are usually assembled into an<br>
intermediate transplacement construct (transfer vector). This may contain a single gene and operably linked regulatory elements;<br>
multiple genes, each with its owned set of operably linked regulatory elements; or multiple genes, regulated by the same set of<br>
regulatory elements. Intermediate transplacement constructs are often maintained in a replicon, such as an extra-chromosomal<br>
element (e.g. plasmids) capable of stable maintenance in a host, such as a bacterium. The replicon will have a replication system,<br>
thus allowing it to be maintained in a suitable host for cloning and amplification.<br>
Currently, the most commonly used transfer vector for introducing foreign genes into AcNPV is pAc373. Many other vectors,<br>
known to those of skill in the art, have also been designed. These include, for example, pVL985 (which alters the polyhedrin start<br>
codon form ATG to ATT, and which introduces a BamHI cloning site 32 basepairs downstream from the ATT; see Luckow and<br>
summers, Virology (1989) 17:31.<br>
The plasmid usually also contains the polyhedrin polyadenylation signal (Miller et al. (1988) Ann. Rev. Microbiol., 42:177) and a<br>
prokaryotic ampicillin-resistance (amp) gene and origin of replication for selection and propagation in E. coli.<br>
Baculovirus transfer vectors usually contain a baculovirus promoter. A baculovirus promoter is any DNA sequence capable of<br>
binding a baculovirus RNA polymerase and initiating the downstream (5' to 3) transcription of a coding sequence (e.g. structural<br>
gene) into mRNA. A promoter will have a transcription initiation region which is usually placed proximal to fee 5' end of fee<br>
coding sequence. This transcription initiation region usually include an RNA polymerase binding site and a transcription initiation<br>
site. A baculovirus transfer vector may also have a second domain called an enhancer, which, if present, is usually disital to the<br>
structural gene. Expression may be either regulated or constitutive.<br>
Structural genes, abundantly transcribed at late times in a viral infection cycle, provide particularly useful promoter sequences.<br>
Examples include sequences derived from the gene encoding the viral polyhedron protein, Friesen et al. (1986) The Regulation<br>
of Baculovirus Gene Expression," in: The Molecular Biology of Baculoviruses (ed. Walter Doerfler); EPO Publ. Nos. 127 839<br>
and 155 476; and the gene encoding the p10 protein, Vlak et al., (1988),J. Gen. Virol. 69:165.<br>
DNA encoding suitable signal sequences can be derived from genes for secreted insect or baculovinjs proteins, such as the<br>
baculovirus polyhedrin gene (Carbonell et al. (1988) Gene, 75:409). Alternatively, since the signals for mammalian cell<br>
posttranslational modifications (such as signal peptide cleavage, proteolytic cleavage, and phospborylation) appear to be<br>
recognized by insect cells, and the signals required for secretion and nuclear accumulation also appear to be conserved between<br>
the invertebrate cells and vertebate cells, leaders of non-insect origin, such as those derived from genes encoding human a-<br>
interferon, Maeda et al., (1985), Nature 375:592; human gastrin-releasing peptide, Lebacq-Verheyden et al., (1988), Molec.<br>
Cell. Biol. 8:3129; human IL-2, Smith et al., (1985) Proc. Nat'l Acad. Sci. USA, 82:8404; mouse IL-3, (miyajima et al.,<br>
(1987) Gene 58:273; and human glucocerebrosidase, Martin et aL (1988) DNA, 7:99, can also be used to provide for secretion<br>
in insects.<br>
A recombinant polypeptide or polyprotein may be expressed intracellularly or, if it is expressed with the proper regulatory<br>
sequences, it can be secreted. Good intracellular expression of nonfused foreign protein usually requires heterologous genes that<br>
ideally have a short leader sequence containing suitable translation initiation signals preceding an ATG start signal. If desired,<br>
methionine at the N-terminus may be cleaved from the mature protein by in vitro incubation with cyanogen bromide.<br>
Alternatively, recombinant polyproteins or proteins which are not naturally secreted can be secreted from the insect cell by<br>
creating chimeric DNA molecules that encode a fusion protein comprised of a leader sequence fragment that provides for<br>
secretion of the foreign protein in insects. The leader sequence fragment usually encodes a signal peptide comprised of<br>
hydrophobic amino acids which direct the translocation of the protein into the endoplasmic reticulum.<br>
After insertion of the DNA sequence and/or the gene encoding the expression product precursor of the protein, an insect cell host<br>
is co-transformed with the heterologous DNA of the transfer vector and the genomic DNA of wild type baculovirus-usually by<br>
co-transfection. The promoter and transcription termination sequence of the construct will usually comprise a 2-5kb section of the<br>
baculovirus genome. Methods for introducing heterologous DNA into the desired site in the baculovirus virus are known in the art.<br>
(See Summers and Smith supra: Ju et al. (1987); Smith et al., Mol. Cell Biol. (1983) 3:2156; and Luckow and Summers<br>
(1989)). For example, the insertion can be into a gene such as the polyhedrin gene, by homologous double crossover<br>
recombination; insertion can also be into a restriction enzyme site engineered into the desired baculovirus gene. Miller et al.,<br>
(1989), Bioessays 4:91. The DNA sequence, when cloned in place of the polyhedrin gene in the expression vector, is flanked<br>
both 5' and 3' by polyhedrin-specific sequences and is positioned downstream of the polyhedrin promoter.<br>
The newly formed baculovirus expression vector is subsequently packaged into an infectious recombinant baculovirus.<br>
Homologous recombination occurs at low frequency (between about 1% and abort 5%); thus, the majority of the virus produced<br>
after cotransfection is still wild-type virus Therefore, a method is necessary to identify recombinant viruses. An advantage of the<br>
expression system is a visual screen allowing recombinant viruses to be distinguised. The polyhedrin protein, which is produced<br>
by the native virus, is produced at very high levels in the nuclei of infected cells at late times after viral infection, Accumulated<br>
polyhedrin protein forms occlusion bodies that also contain embedded particles. These occlusion bodies, up to 15µm in size, are<br>
highly retractile, giving them a bright shiny appearance that is readily visualized under the light microscope. Cells infected with<br>
recombinant viruses lack occlusion bodies. To distinguish recombinant virus from wild-type virus, the transfection supernatant is<br>
plaqued onto a monolayer of insect cells by techniques known to those skilled in the art. Namely, the plaques are screened under<br>
the light miaoscope for the presence (indicative of wild-type virus) or absence (indicative of recombinant virus) of occlusion<br>
bodies. "Current Protocols in Microbiology" Vol. 2 (Ausubel et al. eds) at 16.8 (Supp. 10, 1990); Summers and Smith, supra;<br>
Miller et al. (1989).<br>
Recombinant baculovirus expression vectors have been developed for infection into several insect cells. For example,<br>
recombinant baculoviruses have been developed for, inter alia: Aedes aegypti, Autographa californica, Bombyx mori,<br>
Drosophila melanogaster, spodoptera frugiperda, and Trichoplusia ni (WO 89/046699; Carbonell et al., (1985) J. Virol<br>
56:153; Wright (1986) Nature 527:718; Smith et al, (1983) Mol. Cell Biol 3:2156; and see generally, Fraser, et al. (1989) In<br>
Vitro Cett.Dev.Biol 25:225).<br>
Cells and cell culture media are commercially available for both direct and fusion expression of heterologous polypeptides in a<br>
baculovirus/expression system; cell culture technology is generally known to those skilled in the art. See, e.g. Summers and Smith<br>
supra.<br>
The modified insect cells may then be grown in an appropriate nutrient medium, which allows for stable maintenance of the<br>
plasmid(s) present in the modified insect host Where the expression product gene is tinder inducible control, the host may be<br>
grown to high density, and expression induced. Alternatively, wine expression is constitutive, the product will be continuously<br>
expressed into the medium and the nutrient medium must be continuously circulated, while removing the product of interest and<br>
augmenting depleted nutrients. The product may be purified by such techniques as chromatography, eg. HPLC, affinity<br>
chromatography, ion exchange chromatography, etc.; electrophoresis; density gradient centrifugation; solvent extraction, etc. As<br>
appropriate, the product may be further purified, as required, so as to remove substantially any insect proteins which are also<br>
present in the medium, so as to provide a product which is at least substantially free of host debris, eg. proteins, lipib and<br>
polysaccharides.<br>
In order to obtain protein expression, recombinant host cells derived from the transformants are incubated under conditions which<br>
allow expression of the recombinant protein encoding sequence. These conditions will vary, dependent upon the host cell<br>
selected However, the conditions are readily ascertainable to those of ordinary skill in the art, based upon what is known in the<br>
art.<br>
iii. Plant Systems<br>
There are many plant cell culture and whole plant genetic expression systems known in the art. Exemplary plant cellular genetic<br>
expression systems include those described in patents, such as: US 5,693,506; US 5,659,122; and US 5,608,143. Additional<br>
examples of genetic expression in plant cell culture has been described by Zenk, Phytochemistry 30:3861-3863 (1991).<br>
Descriptions of plant protein signal peptides may be found in addition to the references described above in Vaulcombe et al., Mol.<br>
Gen. Genet. 2O9:33-40 (1987); Chandler et al., Plant Molecular Biology 3:407-418 (1984); Rogers, J. Biol Chem.<br>
260:3731-3738 (1985); Rothstein et al., Gene 55:353-356 (1987); Whittier et al., Nucleic Acids Research 15:2515-2535<br>
(1987); Wirsel et al., Molecular Microbiology 33-14 (1989); Yu et al. Gene 122:247-253 (1992). A description of the<br>
regulation of plant gene expression by the phytohormone, gibberellic acid and secreted enzymes induced by gibberellic acid can<br>
be found in R.L. Jones and J. MacMillin, Gibberellins: in Advanced Plant Physiology, Malcolm B. Wilkms, ed, 1984 Pitman<br>
Publishing limited, London, pp. 21-51 References that describe other metabolically-regulated genes: Sheen, Plant Cell,<br>
2:1027-1038(1990); Maas et al., EMBO J. 9-3447-3452 (1990); Benkel and Hickey, Proc Natl. Acad. Sci. 84:1337-1339<br>
(1987).<br>
Typically, using techniques known in the art, a desired polynucleotide sequence is inserted into an expression cassette comprising<br>
genetic regulatory elements designed for operation in plants. The expression cassette is inserted into a desired expression vector<br>
with companion sequences upstream and downstream from the expression cassette suitable for expression in a plant host. The<br>
companion sequences will be of plasmid or viral origin and provide necessary characteristics to the vector to permit the vectors to<br>
move DNA from an original cloning host, such as bacteria, to the desired plant chromosomes. Where the heterologous gene is<br>
preferably provide a broad host range prokaryote replication origin; a prokaryote selectable marker; and, for Agrobacterium<br>
transformations, T DNA sequences for Agrobacterium-mediated transfer to plant chromosomes. Where the heterologous gene is<br>
not readily amenable to detection, the construct will preferably also have a selectable marker gene suitable for determining if a<br>
plant cell has been transformed. A general review of suitable makers, for example for the members of the grass family, is found in<br>
Wilmink and Dons, 1993, Plant Mol Biol. Reptr, 11(2):165-185.<br>
Sequences suitable for permitting integration of the heterologous sequence into the plant genome are also recommended. These<br>
might include transposon sequences and the like for homologous recombination as well as Ti sequences which permit random<br>
insertion of a heterologous expression cassette into a plant genome. Suitable prokaryote selectable markers include resistance<br>
toward antibiotics such as ampicillin or tetracycline. Other DNA sequences encoding additional functions may also be present in<br>
the vector, as is known in the art.<br>
The nucleic acid molecules of the subject invention may be included into an expression cassette for expression of the protein(s) of<br>
interest. Usually, line will be only one expression cassette, although two or more are feasible. The reoombinant expression<br>
cassatte will contain in addition to the heterologous protein encoding sequence the following elements, a promoter region, plant 5'<br>
untranslated sequences, initiation codon depending upon whether or not the structural gene comes equipped with one, and a<br>
transcription and translation termination sequence. Unique restriction enzyme sites at the 5' and 3' ends of the cassette allow for<br>
easy insertion into a pre-existing vector.<br>
A heterologous coding sequence may be for any protein relating to the present invention. The sequence encoding the protein of<br>
interest will encode a signal peptide which allows processing and translocation of the protein, as appropriate, and will usually lack<br>
any sequence which might result in the binding of the desired protein of the invention to a membrane. Since, for the most part, the<br>
transcriptional initiation region will be for a gene which is expressed and translocated during germination, by employing the signal<br>
peptide which provides for translocation, one may also provide for translocation of the protein of interest. In this way, the<br>
protein(s) of interest will be translocated from fee cells in which they are expressed and may be efficiency harvested. Typically<br>
secretion in seeds are across the aleurone or scutellar epithelium layer into the endosperm of the seed. While it is not required that<br>
the protein be secreted from the cells in which the protein is produced, this facilitates the isolation and purification of the<br>
recombinant protein.<br>
Since the ultimate expression of the desired gene product will be in a eucaryotic cell it is desirable to determine whether any<br>
portion of the cloned gene contains sequences which wffl be processed out as introns by the hosts splicosome machinery. If so,<br>
site-directed mutagenesis of the "intron" region may be conducted to prevent losing a portion of the genetic message as a false<br>
intran code, Reed and Maniatis,Cell 41:95-105, 1985.<br>
The vector can be microinjected directly into plant cells by use of micropipetes to mechanically transfer the recombinant DNA<br>
Crossway, Mol. Gen. Genet, 202:179-185, 1985. The genetic material may also be transferred into the plant cell by using<br>
polyethylene glycol, Krens, et al., Nature, 296, 72-74, 1982. Another method of introduction of nucleic acid segments is high<br>
velocity balistic penetration by small particles with the nucleic acid either within the matrix of small beads or particles, or on the<br>
surface, Klein, et al., Nature, 327, 70-73, 1987 and Knudsen and Muller, 1991, Planta, 185-330-336 teaching paricle<br>
bombaardment of barley endosperm to create transgenic barley. Yet another method of introduction would be fusion of protoplasts<br>
with other entities, either minicells, cells, lysosomes or other fusible lipid-surfaced bodies, Fraley, et al., Proc. Natl. Acad. Sci.<br>
USA, 79,1859-1863, 1982.<br>
The vector may also be introduced into the plant cells by electroporation. (From et al., Proc. Natl. Acad. Sci. USA 82:5824,<br>
1985). In this technique, plant protoplasts re electroporated in the presence of plasmids containing the gene construct. Electrical<br>
impulses of high field strength reversibly permeabilize biomembranes allowing the introduction of the plasmids. Electropoated<br>
plant protoplasts feform the cell wall, divide, and form plant cellus.<br>
All plants from which protoplasts can be isolated and cultured to give whole regenerated plants can be transformed by the present<br>
invention so flat whole plants are recovered which contain the transferred gene. It is known that practically all phots can be<br>
regenerated from cultured cells or tissues, including but not limited to all major species of sugarcane, sugar beet, cotton, fruit and<br>
other trees, legumes and vegetables. Some suitable plants include, for example, species from the genera Frugaria, Lotus,<br>
Medicago, Onobrychis, Trifotium, Trigonella, Vigna, Citrus, Linum, Geranium, Mamhot, Dauaa, Arabidopsis, Brassica,<br>
Raphanus, Sinapis, Atropa, Capsicum, Datura, Hyoseyamus, Lycopersion, Nicotiana, Solanum, Petunia, Digitalis,<br>
Majorana, Cichorium, Helianthus, Lactuca, Bromus, Asparagus, Antirrhinum, Hererocallis, Nemesia, Pelargonium,<br>
Panicum, Pennisetum, Ranunculus, Senecio, Salpiglossis, Cucumis, Browaalia, Glycine, Lolium, Zea, Triticum, Sorghum,<br>
and Datura.<br>
Means for regeneration vary from species to species of plants, but generally a suspension of transformed protoplasts containing<br>
copies of the heterologous gene is first provided Callus tissue is formed and shoots may be induced from callus and subsequently<br>
rooted. Alternatively, embryo formation can be induced from the protoplast suspension. These embryos gemmate as natural<br>
embryos to form plants. The culture media will generally contain various amino acids and hormones, such as sixom and cytokinins<br>
It is also advantageous to add glutamic acid and proline to the medium, especially for such species as corn and alfalfa. Shoots and<br>
roots normally develop simultaneously. Efficient regeneration will depend on the medium, on the genotype, and on the history of<br>
the culture. If these three variables are controlled, then regeneration is fully reproducible and repeatable.<br>
In some plant cell culture systems, the desired protein of the invention may be excreted or alternatively, the protein may be<br>
extracted from the whole plant. Where the desired protein of the invention is secreted into the medium, it may be collected.<br>
Alternatively, the embryos and embryoless-half seeds or other plant tissue may be mechanically distrupted to release any secreted<br>
protein between cells and tissues. The mixture may be suspended in a buffer solution to retrieve soluble proteins. Conventional<br>
protein isolation and purification methods will be then used to purify the recombinant protein. Paarameters of time, temperature pH<br>
oxygen, and volumes will be adjusted through routine methods to optimize expression and recovery of heterologous protein.<br>
iv. Bacterial System<br>
Bacterial expression techniques are known in the art. A bacterial promoter is any DNA sequence capable of binding bacterial<br>
RNA polymerase and initiating the downstream (3') transcription of a coding sequence (e.g. structural gene) into mRNA. A<br>
promoter will have a transcription initiation region which is usually placed proximal to the 5' end of the coding sequence. This<br>
transcription initiation region usually includes an RNA polymerase binding site and a transcription initiation site. A bacterial<br>
promoter may also have a second domain called an operator, that may overlap an adjacent RNA polymerase binding site at<br>
which RNA synthesis begins. The operator permits negative regulated (inclucible) transcription, as a gene represser protein may<br>
bind the operator and thereby inhibit transcription of a specific gene. Constitutive expression may occur in the absence of negative<br>
regulatory elements, such as the operator. In addition, positive regulation may be achieved by a gene activator protein binding<br>
sequence, which, if present is usually proximal (5') to the RNA polymerase binding sequence. An example of a gene activator<br>
protein is the catabolite activator protein (CAP), which helps initiate transcription of the lac operon in Escherichia coli (E.coli)<br>
[Raibaud et al. (1984) Annu. Rev. Genet. 18:173]. Regulated expression may therefore be either positive or negative, thereby<br>
either enhancing or reducing transcription.<br>
Sequences encoding metabolic pathway enzymes provide particularly useful promoter sequences. Examples include promoter<br>
sequences derived from sugar metabolizing enzymes, such as galaciose, lactose (lac) [Chang et al. (1977) Nature 198:1056],<br>
and maltose. Additional examples include promoter sequences derived from biosynthetic enzymes such as tryptophan (trp)<br>
[Goeddel et al. (1980) Nuc Acids Res. 8:4057; Yelvorton et al. (1981) Nucl. Acids Res. 9:731; US patent 4,738,921; EP-A-<br>
0036776 and EP-A-0121775]. The g-lactamase (bla) promoter system [Weissmann (1981) "The ctoning of interferon and other<br>
mistakes." In Interferon 3 (ed. I. Gresser)], bacteriophage lambda PL [Shimatake et al. (1981) Nature 292:128] and T5 [US<br>
patent 4,689,406] promoter systems also provide useful promoter sequences.<br>
In addition, synthetic promoters which do not occur in nature also function as bacterial promoters. For example, transcription<br>
activation sequences of one bacterial or bacteriophage promoter may be joined with the operon sequnces of another bacterial or<br>
bacteriophage promoter, creating a synthetic hybrid promotor [US patent 4,551,433]. Furthermore, a bacterial promoter can include<br>
trp-lac promoter comprised of both trp promoter and lac operon sequence that is regulated by the lac repressor [Amam et al.<br>
(1983) Gene 25:167; de Boer et al. (1983) Proc Natl. Acad. Sci. 8021]. Furthermore, a bacterial promoter can include<br>
naturally occurring promoters of non-bacterial origin that have the ability to bind bacterial RNA polymerase and initiate<br>
transcription. A naturally occurring promoter of non-bacterial origin can also be coupied with a compatible RNA polymenanse to<br>
produce high levels of expression of some genes in prokaryotes. The bacteriophage T7 RNA polymerase/promoter system is an<br>
example of a coupled promoter system [Studier et al. (1986) J. Mol. Biol. 189:113; Tabor et al. (1985) Proc Natl. Acad. Sci.<br>
82:1074]. In addition, a hybrid promoter can also be comprised of a bacteriophage promoter and an E.coli operator region<br>
(EPO-A-0267851).<br>
In addition to a functioning promoter sequence, an efficient ribosame binding site is also useful for the expression of foreign genes<br>
in prokaryotes. In E.coli, the ribosome binding site is called the Shine-Dalgamo (SD) sequence and includes an initiation codon<br>
(ATG) and a sequence 3-9 nucleotides in length located 3-11 nucleotides upstream of the initiation codon [Shine et al. (1975)<br>
Nature 254:34]. The SD sequence is thought to promote binding of mRNA to the ribosome by the pairing of bases between the<br>
SD sequence and the 3' and of E.coli 16S rRNA [Strifz et al. (1979) "Genetic signals and nucleotide sequences in messenger<br>
RNA." In Biological Regulation and Development: Gene Expression (ed. R.F. Goldberger)]. To express eukaryotic genes<br>
and prokaryotic genes with weak ribosome-binding site [Sambrook et al. (1989) "Expression of cloned genes in Escherichia<br>
coli." In Molecular Cloning: A Laboratory Manual].<br>
A DNA molecule may be expressed intracellularly. A promoter sequence may be directly linked with the DNA molecule, in which<br>
case the first ammo acid at the N-teminns will always be a methionine, which is encoded by the ATG start codon. If desired,<br>
methionine at the N-terminus may be cleaved from the protein by in vitro incubation with cyanogen bromide or by either in vivo<br>
on in vitro incubation with a bacterial methionine N-terminal peptidase (EP-A-0 219 237).<br>
Fusion proteins provide an alternative to direct expression. Usually, a DNA sequence encoding the N-terminal potion of an<br>
endogenous bacterial protein, or other stable protein, is fused to the 5' end of heterologous coding sequences. Upon expression,<br>
das construct will provide a fusion of the two ammo acid sequences. For example, the bacteriophage lambda cell gene can be<br>
linked at the 5' terminus of a foreign gene and expressed in bacteria The resulting fusion protein preferably retains a site for a<br>
processing enzyme (factor Xa) to cleave the bacteriophage protein from the foreign gene [Nagai et al. (1984) Nature 309:810].<br>
Fusion proteins can also be made with sequences from the lacZ [lia et al. (1987) Gene 60:197], trpE [Allen et aL (1987) J.<br>
Biotechnol. 5:93; Makoff et al. (1989) J. Gen. Microbiol. 135:11], and Chey [EP-A-0 324 647] genes. The DNA sequence at<br>
the junction of the two amino acid sequences may or may not encode a cleavable site. Another example is a ubiquitin fusion<br>
protein. Such a fusion protein is made with the ubiquitin region that preferably retains a site for a processing enzyme (e.g. ubiquin<br>
specific processing-protease) to cleave the ubiquitin from the foreign protein. Through this method, native foreign protein can be<br>
isolated [Miller et al. (1989) Bio/Technology 7:698].<br>
Alternatively, foreign proteins can also be secreted from the cell by creating chimeric DNA molecules that encode a fusion protein<br>
comprised of a signal peptide sequence fragment that provides for secretion of the foreign protein in bacteria [US patent<br>
4336336]. The signal sequence fragment usually encodes a signal peptide comprised of hydrophobic amino acids which direct<br>
the secretion of the protein from the cell. The protein is either secreted into the growth media (gram-positive bacteria) or into the<br>
periplasmic space, located between the inner and outer membrane of the cell (gram-negative bacteria). Preferably there are<br>
processing sites, which can be cleaved either in vivo or in vitro encoded between the signal peptide fragment and the foreign<br>
gene.<br>
DNA encoding suitable signal sequences can be derived from genes for secreted bacterial proteins, such as the E.coli outer<br>
membrane protein gene (ompA) [Masui et al. (1983), in: Experimental Manipulation of Gene Expression; Ghrayeb et al.<br>
(1984) EMBO J. 3:2437] and the E.coli alkatine phosphatase signal sequence (phoA) [Oka ei al. (1985) Proc. Natl. Acad.<br>
Sci. 82:7212]. As an additional example, the signal sequence of the alpha-amylase gene from various Bacillus strains can be used<br>
to secrete heterologous proteins from B. subtilis [Palva et al. (1982) Proc. Natl. Acad. Sci. USA 79:5582; EP-A-0 244 042].<br>
Usually, transcription termination sequences recognized by bacteria are regulatory regions located 3' to the translation stop codon,<br>
and thus together with the promoter flank the coding sequence. These sequences direct the transcription of an mRNA which can<br>
be translated into the polypeptide encoded by the DNA. Transcription termination sequences frequently include DNA sequences<br>
of about 50 nucleotides capable of forming stem loop structures that aid in terminating transcription. Examples include<br>
transcription termination sequences derived from genes with strong promoters, such as the trp gene in E.coli as well as other<br>
biosynthetic genes.<br>
Usually, fee above described components, comprising a promoter, signal sequence (if desired), coding sequence of interest, and<br>
transcription termination sequence, are put together into expression constructs. Expression constructs arc often maintained in a<br>
replicon, such as an extrachromosomal element (e.g. plasmids) capable of stable maintenance in a host, such as bacteria. The<br>
replicon will have a replication system, thus allowing it to be maintained in a prokaryotic host either for expression or for cloning<br>
and amplification. In addition, a replicon may be cither a high or low copy number plasmid. A high copy number plasmid will<br>
generally have a copy number ranging from about 5 to about 200, and usually about 10 to about 150. A host contianing a high<br>
copy number plasmid will preferably contain at least about 10, and more preferably at least about 20 plasmids. Either a high or<br>
low copy member vector may be selected, depending upon the effect of the vector and the foreign protein on the host.<br>
Alternatively, the expression constructs can be integrated into the bacterial genome with an integrating vector. Integrating vectors<br>
usually contain at least one sequence homologous to the bacterial chromosome that allows the vector to integrate, Integrations<br>
appear to result from recombinations between homologous DNA m in the vector and the bacterial chromosome. For example, inte-<br>
grating vectors constructed with DNA from various Bacillus strains integrate into the Bacillus chromosome (EP-A- 0 127 328).<br>
Integrating vectors may also be comprised of bacteriophage or transposon sequences.<br>
Usually, extrachromosomal and integrating expression constructs may contain selectable markers to allow for the selection of<br>
bacterial strains that have been transformed Selectable markers can be expressed in the bacterial host and may include genes<br>
which render bacteria resistant to drags such as ampicillin, chloramphenicol, erythromycin, kanarnycin (neomycin), and<br>
tetracycline [Davies et al. (1978) Annu. Rev. Microbiol. 32:469]. Selectable markers may also include biosynthetic genes, such<br>
as those in the bistidine, tryptophan, and leucine biosynthetic pathways.<br>
Alternatively, some of the above described components can be put together in transtomation vectors. Transformation vectors are<br>
usually comprised of a selectable market that is either maintained in a replicon or developed into an integrating vector, as<br>
described above.<br>
Expression and transformation vectors, either extra-chromosomal replicons or integrating vectors, have been developed for<br>
transformation into many bacteria. For example, expression vectors have been developed for, inter alia, the following bacteria:<br>
Bacillus subtilis [Palva et al. (1982) Proc Nad. Acad. Sci. USA 79:5582; EP-A-0 036 259 and EP-A-0 063 953; WO<br>
84/04541], Escherichia coli [Shimatake et al. (1981) Nature 292:128, Amaun et al. (1985) Gene 40:183; Studier et al. (1986)<br>
J. Mol. Biol. 189:113; EP-A-0 036 776.EP-A-0 136 829 and EP-A-0136 907], Streptococcus cremoris (Powell et al. (1988)<br>
Appl. Environ. Microbiol. 54:655]; Streptococcus lividans [Powell et al. (1988) Appl. Environ. Microbiol. 54:655],<br>
Streptomyces lividans [US patent 4,745,056].<br>
Methods of introducing exogenous DNA into bacterial hosts are well-known in the art, and usually include either the<br>
transformation of bacteria treated with CaCl2 or other agents, such as divalent cations and DMSO. DNA can also be introduced<br>
into bacterial cells by electroporation. Transformation procedures usually vary with the bacterial species to be transformed. See<br>
eg. [Masson et al. (1989) FEMS Microbiol. l.ett. (50:273; Palva et al. (1982) Proc Nad. Acad. Sci. USA 79:5582; EP-A-0<br>
036 259 and EP-A-0 063 953; WO 84/04541, Bacillus], [Miller et al. (1988) Proc. Nail. Acad. Sci. 55:856; Wang et al.<br>
(1990)/ Bacterial. 172549, Campylobacter], [Cohen et al. (1973) Proc. Natl. Acad. Sci. 69:2110; Dower et al. (1988)<br>
Nucleic Acids Res. 76:6127; Kushner (1978) "An improved method for transformation of Escherichia coli with ColEl-derived<br>
plasmids. In Genetic Engineering: Proceedings of the International Symposium on Genetic Engineering (eds. H.W. Boyer<br>
and S. Nicosia); Mandel et al. (1970) J. Mol. Biol. 53:159; Taketo (1988) Biochim. Biophys. Acta 949:318; Escherichia],<br>
[Chassy et al. (1987) FEMS Microbiol. l.ett. 44:173 l.actobacillus]; [Fiedler et al. (1988) Anal. Biochem 770:38,<br>
Pseudomonas]; [Auguslin et al. (1990) FEMS Microbiol. l.ett. 56:203, Staphylococcus], [Barany et al. (1980) J. Bacteriol.<br>
744:698; Harlander (1987) Transformation of Streptococcus lactis by electroporation, in: Streptococcal Genetics (ed. J.<br>
Ferretti and R. Curtiss III); Perry et al. (1981) Infect Immun. 52:1295; Powell et al. (1988) Appl. Environ. Microbiol<br>
54:655; Somkuti et al. (1987) Proc 4th Evr. Cong. Biotechnology 7:412, Streptococcus].<br>
v. Yeast Expression<br>
Yeast expression systems are also known to one of ordinary skill in the art A yeast promoter is any DNA sequence capable of<br>
binding yeast RNA polymerase and initiating the downstream (3) transcription of a coding sequence (eg. structural gene) into<br>
mRNA A promoter will have a transcription initiation region which is usually placed proximal to the 5' end of the coding<br>
sequence. This transcription initiation region usually includes an RNA polymerase binding site (the "TATA Box") and a<br>
transcription initiation site. A yeast promoter may also have a second domain called an upstream activator sequence (UAS),<br>
which, if present, is usually distal to the structural gene. The UAS permits regulated (inducible) expression. Constitutive expression<br>
occurs in the absence of a UAS. Regulated expression may be either positive or negative, thereby either enhancing or reducing<br>
transcription.<br>
Yeast is a fermenting organism with an active metabolic pathway, therefore sequences encoding enzymes in the metabolic<br>
pathway provide particularly useful promoter sequences. Examples include alcohol dehydrogenase (ADH) (EP-A-0 284 044),<br>
enolase, glucokinaae, glucose-6-phosphate isomerase, glyceraldehyde-3-phosphate-dehydrogenase (GAP or GAPDH),<br>
hexokinase, phosphofructokinase, 3-phosphoglycerate mutase, and pyruvate kinase (PyK) (EPO-A-0 329 203). The yeast<br>
PHO5 gene, encoding acid phosphatase, also provides useful promoter sequences [Myanobara et al. (1983) Proc. Nad. Acad<br>
Sci USA 80:1].<br>
In addition, synthetic promoters which do not occur in nature also function as yeast promoters. For example, USA sequences of<br>
one yeast promoter may be joined with the transcription activation region of another yeast promoter, creating a synthetic hybrid<br>
promoter. Examples of such hybrid promoters include the ADH regulatory sequence linked to the GAP transcription activation<br>
region (US Patent Nos. 4,876,197 and 4,880,734). Other examples of hybrid promoters include promoters which consist of the<br>
regulatory sequences of either the ADH2, GAL4, GAL10, OR PH05 genes, combined with the transcriptional activation region<br>
of a glycorytic enzyme gene such as GAP or PyK (EP-A-0 164 556). Furthermore, a yeast promoter can include naturally<br>
occurring promotes of non-yeast origin that have the ability to bind yeast RNA polymerase and initiate transcription. Examples of<br>
such, promoters include, inter alia, [Cohen et al. (1980) Proc. Nail. Acad. Sci. USA 77:1078; Henikoff et al. (1981) Nature<br>
283:835;Hollenberg et al. (1981) Curr. Topics Microbiol. Immunol. 96:119; Hollenberg et al. (1979) "The Expression of<br>
Bacterial Antibiotic Resistance Genes in the Yeast Saccharomyces cerevisiae," in: Plasmids of Medical, Environmental and<br>
Commercial Importance (eds. K.N. Timmis and A. Publer); Mercerau-Puigalon et al. (1980) Gene 77:163; Panther et al.<br>
(1980) Curr. Genet 2:109;].<br>
A DNA molecule may be expressed intracellularly in yeast. A promoter sequence may be directly linked with the DNA molecule,<br>
in which case the first amino acid at the N-teminus of the recombinant protein will always be a methionine, which is encoded by<br>
the ATG start codon. If desired, methionine at the N-terminus may be cleaved from the protein by in vitro incubation with<br>
cyanogen bromide.<br>
Fusion proteins provide an alternative far yeast expression systems, as well as in mammalian, baculovirus, and bacterial<br>
expression systems. Usually, a DMA sequence encoding the N-terminal portion of an endogenous; yeast protein, or other stable<br>
protein, is fused to the 5' end of heterologous coding sequences. Upon expression, this construct will provide a fussion of the two<br>
amino acid sequences. For example, the yeast or human superoxide dismutase (SOD) gene, can be linked at the 5' terminus of a<br>
foreign gene and expressed in yeast The DNA sequence at the junction of the two amino acid sequences may or may not encode<br>
a cleavable site. See eg. EP-A-0 196 056. Another example is a ubiquitin fusion protein Such a fusion protein is made with the<br>
ubiquitin region bat preferably retains a site for a processing enzyme (e.g. ubiquitin-specific processing protease) to cleave the<br>
ubiquitin from the foreign protein. Through this method, Therefore, native foreign protein can be isolated (eg. WO88/024066).<br>
Alternatively, foreign proteins can also be secreted from the cell into the growth media by creating chimeric DNA molecules that<br>
encode a fusion protein comprised of a leader sequence fragment that provide for secretion in yeast of the foreign protein.<br>
Preferably, there are processing sites encoded between the leader fragment and the foreign gene that can be cleaved either in vivo<br>
or in vitro. The leader sequence fragment usually encodes a signal peptide comprised of hydrophobic atnino acids which direct<br>
the secretion of the protein from the cell.<br>
DNA encoding suitable signal sequences can be derived from genes for secreted yeast proteins, such as the yeast invertase gene<br>
(EP-A-0 012 873; JPO. 62,096,086) and the A-foctor gene (US patent 4,588,684). Alternatively, leaders of non-yeast origin,<br>
such as an interferon leader, exist that also provide for secretion in yeast (EP-A-0 060 057).<br>
A preferred class of secretion leaders are those that employ a fragment of the yeast alpha-factor gene, which contains both a "pie"<br>
signal sequence, and a "pro" region. The types of alpha-factor fragments that can be employed include the full-length pre-pro<br>
alpha factor leader (about 83 ammo acid residues) as well as truncated alpha-factor leaders (usually about 25 to about 50 amino<br>
acid residues) (US Patents 4,546,083 and 4,870,008; EP-A-0 324 274). Additional leaders employing an alpha-factor leader<br>
fragment that provides for secretion include hybrid alpha-factor leadets made with a presequence of a first yeast, but a pro-region<br>
from a second yeast alphafactor. (eg. see WO 89/02463.)<br>
Usually, transcription termination sequences recognized by yeast arc regulatory regions located 3' to the translation stop codon,<br>
and thus together with the promoter flank the coding sequence. These sequences direct the transcription of an mRNA which can<br>
be translated into the polypeptide encoded by the DNA. Examples of transcription terminator sequence and other yeast-<br>
recognized termination sequences, such as those coding for glycolytic enzymes.<br>
Usually, the above described components, comprising a promoter, leader (if desired), coding sequence of interest, and<br>
transcription termination sequence, are put together into expression constructs. Expression constructs are often maintained in a<br>
replicon, such as an extrachromosomal element (eg. plasmids) capable of stable maintenance in a host, such as yeast or bacteria<br>
The replicon may have two replication systems, thus allowing it to be maintained, for example, in yeast for expression and in a<br>
prokaryotic host for cloning and amplification Examples of such yeast-bacteria shuttle vectors include YEp24 [Botstein et al.<br>
(1979) Gene 5:17-24], pCl/1 [Brake et al. (1984) Proc Natl. Acad. Sci USA 81:4642-4646], and YRpl7 [Stinchcomb et al.<br>
(1982) J. Mol. Biol. 158:157]. In addition, a replicon may be either a high or low copy number plasmid. A high copy number<br>
plasmid will generally have a copy number ranging from about 5 to about 200, and usually about 10 to about 150. A host<br>
containing a high copy number plasmid will preferably have at least about 10, and more preferably at least abort 20. Enter a high<br>
or low copy number vector may be selected, depending upon the effect of the vector and the foreign protein on the host. See eg.<br>
Brake et al, supra.<br>
Alternatively, the expression constructs can be integrated into the yeast genome with an integrating vector. Integrating vectors<br>
usually contain at least one sequence homologous to a yeast chromosome that allows the vector to integrate, and preferably<br>
contain two homologous sequences flanking the expression construct Integrations appear to result from recombinations between<br>
homologous DNA in the vector and the yeast chromosome [Orr-Weaver et al.(1983) Methods in Enzymol. 101:228-245]. An<br>
integrating vector may be directed to a specific locus in yeast by selecting the appropriate homologous sequence for inclusion in<br>
the vector. See Orr-Weavcr et al., supra. One or more expression constmct may integrate, possibly affecting levels of<br>
recombinant protein produced [Rine et al. (1983) Proc. Natl. Acad. Sci. USA 80:6750]. The chromosomal sequences included<br>
in the vector can occur either as a single segment in the vector, which results in the integration of the entire vector, or two<br>
segments homologous to adjacent segments in. the chromosome and flanking the expression construct in the vector, which can<br>
result in the stable integration of only the expression construct.<br>
Usually, extrachromosomal and integrating expression constructs may contain selectable markers to allow for the selection of<br>
yeast strains that have been transformed Selectable markers may include biosynthetic genes that can be expressed in the yeast<br>
host, such as ADE2, HIS4, LEU2, TRPl, and ALG7, and the G418 resistance gene, which confer resistance in yeast cells to<br>
tunicamycin and G418, respectively. In addition, a suitable selectable marker may also provide yeast with the ability to grow in the<br>
presence of toxic compounds, such as metal. For example, the presence of CUP1 allows yeast to grow in the presence of copper<br>
ions [Butt et. al. (1987) Microbiol. Rev. 51:351]:<br>
Alternatively, some of the above described components can be put together into transformation vectors. Transformation vectors<br>
are usually comprised of a selectable marker that is either maintained in a replicon or developed into an integrating vector, as<br>
described above.<br>
Expression and transformation vectors, either extrachromosomal replicons or integrating vectors, have been developed for<br>
transformation into many yeasts. For example, expression vectors have been developed for, inter alia, the following<br>
yeasts:Candida albkans [Kurtz, et al. (1986) Mol. Cell. Biol. 6:142], Candida maltosa [Kunze, et al. (1985) J. Basic<br>
Microbiol. 25:141]. Hansenula polymorpha [Geeson, et al. (1986) J. Gen. Microbiol. 7323459; Roggenkamp et al. (1986)<br>
Mol. Gen. Genet 202-302], Kluyveromyces fragilis [Das, et al. (1984) J. Bacteriol. 755:1165], Kluyveromyces lactis [De<br>
Louvencourt et al. (1983) J. Bacteriol. 754:737; Van den Berg et al. (1990) Bio/Technology 8:135], Pichia guillerimondii<br>
[Kunze et al. (1985) J. Basic Microbiol. 25:141], Pichia pastoris [Cregg, et al. (1985) Mol. Cell. Biol. 5:3376; US Patent<br>
Nos. 4,837,148 and 4,929,555], Saccharomyces cerevisiae et al. (1978) Proc Natl. Acad. Sci. USA 75:1929; Bo et<br>
al. (1983) J. Bacteriol. 755:163], Schizosacharomyces pombe [Beach and Nurse (1981) Nature 500:706], and Yarrowia<br>
lipolytica [Davidow, et al. (1985) Curr. Genet 7ft380471 Gaillardin, et al. (1985) Curr. Genet. 10:49].<br>
Methods of introducing exogenous DNA into yeast hosts are well-known in the ait, and usually include either the transformation<br>
of spheroplasts or of intact yeast cells treated with alkali cations. Transformation procedures usually vary with the yeast species to<br>
be transformed See eg. {Kurtz et al. (1986) Mol. Cell. Biol. 6:142; Kunze et al. (1985) J. Basic Microbiol. 25:141; Candida];<br>
[Geeson et al. (1986) J. Gen. Microbiol. 132:3459; Roggenkamp et al. (1986) Mol. Gen. Genet. 202302; Hansenula]; [Das<br>
et al. (1984) J. Bacteriol. 158:1165; De l.ouvencourt et al. (1983) J. Bacteriol. 154:1165; Van den Berg et al. (1990)<br>
Bio/Technology 8:135; Kluyveromyces]; [Cregg et al. (1985) Mol. Cell. Biol. 5:3376; Kunze et al. (1985) J. Basic Microbiol.<br>
25:141; US Patent Nos. 4,837,148 and 4,929,555; Pichia]; [Hinnen et al. (1978) Proa Natl. Acad. Sci. USA 75;1929; Ito et<br>
al. (1983) J. Bacteriol. 755:163 Saccharomyces]; [Beach and Nurse (1981) Nature 300:106; Schizosaccharomyces];<br>
[Davidow et al. (1985) Curr. Genet 1059; Gaillardin et al. (1985) Curr. Genet 70:49; Yarrowia|.<br>
Antibothes<br>
As used herein, the term "antibody" refers to a polypeptide or group of polypeptides composed of at least one antibody<br>
combining site. An "antibody combining site" is the three-dimensional binding space with an internal surface shape and charge<br>
distribution complementary to the features of an epitope of an antigen, which allows a binding of the antibody with the antigen.<br>
"Antibody" includes, for example, vertebrate antibothes, hybrid antibothes, chimeric antibothes, humanised antibothes, altered<br>
antibodies, univalent antibothes, Fab proteins, and single domain antibothes.<br>
Antibothes against the proteins of the invention are useful for affinity chrornatography, immunoassays, and distinguishing/identigying<br>
streptococcus proteins.<br>
Antibothes to the proteins of the invention, both polyclonal and monoclonal, may be prepared by conventional methods. In<br>
general, the protein is first used to immunize a suitable animal, preferably a mouse, rat, rabbit or goat Rabbits and goats are<br>
preferred for the preparation of palyclonal sera due to the volume of serum obtainable, and availability of labeled antirabbit<br>
and anti-goat antibothes. Immumzation is generally performed by mixing or emulsifying the protein in saline, preferably in an<br>
adjuvant such as Flood's complete adjuvant, and injecting the mixture or emulsion parenterally (generally subcutaneously or<br>
intramuscularly). A dose of 50-200 µg/injection is typically sufficient Immunization is generally boosted 26 weeks later with one<br>
or more injections of the protein in saline, preferably using Freund's incomplete adjuvant One may alternatively generate<br>
antibothes by in vitro immunization using methods known in the art, which for the purposes of tins invention is considered<br>
equivalent to in vivo immunization. Poryclonal antisera is obtained by bleeding the immunized animal into a glass or plastic<br>
container, incubating the blood at 25°C for one hour, followed by incubating at 4°C for 2-18 hours. The serum is recovered by<br>
centriiugation(eg. 1,000g for 10 minutes). About 20-50 ml per bleed may be obtained from rabbits.<br>
Monoclonal antibothes are prepared using the standard method of Kohler &amp; Milstein [Nature (1975) 256:495-96], or a<br>
modification thereof. Typically, a mouse or rat is immunized as described above. However, rather than bleeding the animal to<br>
extract serum, the spleen (and optionally several large lymph nodes) is removed and dissociated into single cells. If desired, the<br>
spleen cells may be screened (after removal of nonspecifically adherent cells) by applying a cell suspension to a plate or well<br>
coated with the protein antigen. B-cells expressing membrane-bound immunoglobulin specific for the antigen bind to the plate, and<br>
are not rinsed away with the rest of the suspension. Resulting B-cells, or all dissociated spleen cells, are then induced to fuse with<br>
myeloma cells to form hybridomas, and ate cultured in a selective medium (eg. hypoxanthine, aminopterin, thymidine medium,<br>
"HAT"). The resulting hybridomas ate plated by limiting dilution, and are assayed for production of antibothes which bind<br>
specifically to the immunizing antigen (and which do not bind to unrelated antigens). The selected MAb-secreting hybridomas are<br>
then cultured either in vitro (eg. in tissue culture bottles or hollow fibre reactors), or in vivo (as ascites in mice).<br>
If desired, the antibothes (whether polyclonal or monoclonal) may be labeled using conventional techniques. Suitable labels<br>
include fluorophores, chromophores, radioactive atoms (particularly 32P and 125I), electron-dense reagents, enzymes, and ligands<br>
having specific binding partners. Enzymes ate typically detected by their activity. For example, horseradish peroxidase is usually<br>
detected by its ability to convert 3,3',5,5'-tetramethylbenzidine (TMB) to a blue pigment, quantifiable with a spectrophotometer.<br>
"Specific binding partner" refers to a protein capable of binding a ligand molecule with high specificity, as for example in 'the case<br>
of an antigen and a monoclonal antibody specific therefor. Other specific binding partners include biotin and avidin or streptavidin,<br>
IgG and protein A, and the numerous receptor-ligand couples known in the art ft should be understood that the above<br>
description is not meant to categorize the various labels into distinct classes, as the same label may serve in several different<br>
modes. For example, 125I may serve as a radioactive label or as an electron-dense reagent HRP may serve as enzyme or as<br>
antigen for a MAb. Further, one may combine various labels for desired effect For example;, MAbs and avidin also require labels<br>
in the practice of this invention: thus, one might label a MAb with biotin, and detect its presence with avidin labeled with 125I, or<br>
with an anti-biotin MAb labeled with HRP. Other permutations and possibilities will be readily apparent to those of ordinary skill<br>
in the art, and are considered as equivalents within the scope of the instant invention.<br>
Pharmaceutical Compositions<br>
Pharmaceutical compositions can comprise either polypeptides, antibothes, or nucleic acid of the invention. The pharmaceutical<br>
compositions will comprise a therapeutically effective amount of either polypeptides, antibothes, or polynucleotides of the claimed<br>
invention.<br>
The term "therapeutically effective amount" as used herein refers to an amount of a therapeutic agent to treat, ameliorate, or<br>
prevent a desired disease or condition, or to exhibit a detectable therapeutic or preventative effect The effect can be detected by,<br>
for example, chemical markers or antigen levels. Therapeutic effects also include reduction in physical symptoms, such as<br>
decreased body temperature. The precise effective amount for a subject will depend upon the subjects size and health, the nature<br>
and extent of the condition, and the therapeutics or combination of therapeutics selected for administration. Thus, it is not useful to<br>
specify an exact effective amount in advance. However, the effective amount for a given situation am be determined by routine<br>
experimentation and is within the judgement of the clinician.<br>
For purposes of the present invention, an effective dose will be from about 0.01 mg/ kg to 50 mg/kg or 0.05 mg/kg to about 10<br>
mg/kg of the molecule of the invention in the individual to which it is administered.<br>
A pharmaceutical composition can also contain a pharmaceutically acceptable carrier. The term "pharmaceutically acceptable<br>
carrier" refers to a carrier for administration of a therapeutic agent, such as antibothes or a polypeptide, genes, and other<br>
therapeutic agents. The term refers to any pharmaceutical carrier that does not itself induce the production of antibothes harmful to<br>
the individual receiving the composition, and which may be administered without undue toxicity. Suitable carriers may be large,<br>
slowly metabolized macromolecules such as proteins, polysaccharides, polyiactic acids, polyglycolic acids, polymeric ammo acids,<br>
amino acid copolymers, and inactive virus particles. Such carriers are well known to those of ordinary skill in the art<br>
Pharmaceutically acceptable salts can be used therein, for example, mineral acid salts such as hydrochlorides, hydrobromides,<br>
phosphates, sulfates, and the like; and the salts of organic acids such as acetates, propionates, malonates, benzoates, and the like.<br>
A thorough discussion of pharmaceutically acceptable excipients is available in Remington's Pharmaceutical Sciences (Mack Pub.<br>
Ca.NJ.1991).<br>
Pharmaceutically acceptable carriers in therapeutic compositions may contain liquids such as water, saline, glycerol and ethanol.<br>
Additionally, auxiliary substances, such as wetting or emulsifying agents, pH buffering substances, and the like, may be present in<br>
such vehicles. Typically, the therapeutic compositions are prepared as injectables, either as liquid solutions or suspensions; solid<br>
forms suitable for solution in, or suspension in, liquid vehicles prior to injection may also be prepared. l.iposomes are included<br>
within the definition of a pharmaceutically acceptable carrier.<br>
Delivery Methods<br>
Once formulated, the compositions of the invention can be administered directly to the subject The subjects to be treated can be<br>
animals; in particular, human subjects can be treated.<br>
Direct delivery of the compositions will generally be accomplished by injection, either subcutaneously, intraperitoneally,<br>
intravenously or intramuscularly or delivered to the interstitial space of a tissue. The compositions can also be administered into a<br>
lesion. Other modes of administration include oral and pulmonary administration, suppositories, and transdermal or transcutaneous<br>
applications (eg. see WO98/20734), needles, and gene guns or hyposprays. Dosage treatment may be a single dose schedule or<br>
a multiple dose schedule.<br>
Vaccines<br>
Vaccines according to the invention may either be prophylactic (ie. to prevent infection) or therapeutic (ie. to treat disease after<br>
infection).<br>
Such vaccmes comprise immunising antigen(s), immunogen(s), polypeptide(s), protein(s) or nucleic acid, usually in combination<br>
with "pharmaceutically acceptable carriers," which include any carrier that does not itself induce the production of antibothes<br>
harmful to the individual receiving the composition. Suitable carriers are typically large, slowly metabolized macromolecules such<br>
as proteins, polysaccharides, polylactic acids, polyglycolic acids, polymeric amino acids, amino acid copolymers, lipid aggregates<br>
(such as oil droplets or liposomes), and inactive virus particles. Such carriers are well known to those of ordinary skill in the art<br>
Additionally, these carriers may function as lmmunostimulatmg agents ("adjuvants") Furthermore, the antigen or immunogen may<br>
be conjugated to a bacterial toxoid, such as a toxoid from diphtheria, tetanus, cholera,.H. pylori, etc. pathogens.<br>
Preferred adjuvants to enhance effectiveness of the composition include, but are not limited to: (1) oil-in-water emulsion<br>
formulations (with or without other specific immunostimulating agents such as muramyl peptides (see below) or bacterial cell wall<br>
components), such as for example (a) MF59™ (WO90/14837; Chapter 10 in Vaccine Design - the subunit and adjuvant<br>
approach (1995) ed. Powell &amp; Newman), containing 5% Squalene, 0.5% Tween 80, and 05% Span 85 (optionally containing<br>
MTP-PE) formulated into submicron particles using a rnicrofluidizer, (b) SAF, containing 10% Squalane, 0.4% Tween 80, 5%<br>
pluronic-blocked polymer l.121, and thr-MDP either microfluidized into a submicron emulsion or vortexed to generate a larger<br>
particle size emulsion, and (c) Ribi™ adjuvant system (RAS), (Ribi Immunocnem, Hamilton, MT) containing 2% Squalene, 02%<br>
Tween 80, and one or more bacterial cell wall components from the group consisting of monopbosphorylipid A (MPL), trehalose<br>
dimycolate (TDM), and cell wall skeleton (CWS), preferably MPl. + CWS (Detox"'); (2) saponin adjuvants, such as QS21 or<br>
Stimulon™ (Cambridge Bioscience, Worcester, MA) may be used or paracles generated therefrom such as ISCOMs<br>
(immunostirrulating complexes), which ISCOMS may be devoid of additional detergent e.g. WO00/07621; (3) Complete<br>
Freund's Adjuvant (CFA) and Incomplete Freund's Adjuvant (IFA); (4) cytokines, such as interleukins (e.g. IL-1,IL-2, IL-4,<br>
IL-5, IL-6, IL-7, IL-12 (WO99/44636), etc.), interferons (e.g. gamma interferon), macrophage colony stimulating iactor (M-<br>
CSF), tumor necrosis factor (TNF), etc.; (5) monophosphoryl lipid A (MPl.) or 3-O-deacylated MPl. (3dMPl.) eg. GB-<br>
2220221, EP-A-0689454; (6) combinations of 3dMPl. with, for example, QS21 and/or oil-in-water emulsions e.g. EP-A-<br>
0835318, EP-A-0735898, EP-A-0761231; (7) oligonucleotides comprising CpG motifs [Krieg Vaccine 2000, 19, 618-622;<br>
Krieg Curr opin Mol Ther 2001 3:15-24; Roman et al, Nat. Med, 1997, 3, 849-854; Weiner et al, PNAS USA, 1997, 94,<br>
10833-10837; Davis et al, J. Immunol, 1998, 160, 870-876; Chu et al., J. Exp. Med., 1997, 186,1623-1631; Lipford et<br>
al., Eur.J. Immunol, 1997,27,2340-2344; Moldoveanu et al., Vaccine, 1988, 16,1216-1224, Krieg et al., Nature, 1995,<br>
374, 546-549; Klinman et al, PNAS USA, 1996, 93, 2879-2883; Ballas et al, J. Immunol, 1996, 157, 1840-1845;<br>
Cowdery et al, J. Immunol, 1996, 156,45704575; Halpern et al, Cell. Immunol, 19%, 167, 72-78; Yamamoto et al,<br>
Jpn. J. Cancer Res., 1988, 79, 866-873; Stacey et al., J. Immunol., 1996, 157, 2116-2122, Messina et al, J. Immunol.,<br>
1991,147,1759-1764; Yi et al., J. Immunol, 1996,157,4918-4925; Yi et al, J. Immunol, 1996,157, 5394-5402; Yi et<br>
al, J. Immunol, 1998, 160,4755-4761; and Yi et al., J. Immunol, 1998. 160, 5898-5906; International patent applications<br>
WO96/02555, WO98/16247, WO98/18810, WO98/40100, WO98/55495, WO98/37919 and WO98/52581] l.e. containing<br>
at least one CG dinucleotide, with 5-methylcytosine optionally being used in place of cytosine; (8) a polyoxyethylene ether or a<br>
polyoxyethylene ester e.g. WO99/52549; (9) a polyoxyethylene sorbitan ester surfactant in commbination with an octoxynol (e.g.<br>
WO01/21207) or a polyoxyethylene alkyl ether or ester surfactant in combination with at least one additional non-ionic surfactant<br>
such as an octoxynol (e.g. WO01/21152); (10) an irnrnunostimulatory oligonudeotide (e.g. a CpG oliganucleotide) and a saponin<br>
e.g. WO00/62800; (11) an immunostimulant and a particle of metal salt e.g. WO00/23105; (12) a saponin and an oil-in-water<br>
emulsion eg. WO99/11241; (13) a saponin (e.g. QS21) + 3dMPl. + IL-12 (optionally - a sterol) e.g. WO98/57659; (14)<br>
aluminium salts, preferably hydroxide or phosphate, but any other suitable salt may also be used (eg. hydroxyphosphate,<br>
oxyhydroxide, orthophosphate, sulphate etc. [e.g see chapters 8 &amp; 9 of Powell &amp; Newman]). Mixtures of different aluminium<br>
salts may also be used. The salt may take any suitable form (e.g. gel, crystalline, amorphous etc.); (15) other substances that act<br>
as immunostimulating agents to enhance the efficacy of the composition. Aluminium salts and/or MF59TM are preferred.<br>
As mentioned above, muramyl peptides include, but are not limited to N-acetyl-muramyl-L-threonyl-D-isoglutamine (thr-MDP),<br>
N-acetyl-muramyl-L-threonyl-D-isoglutamine (nor-MDP), N-acetyl-muramyl-L-threonyl-D-isoglutaminyl-L-alanine-2-(1'-2'- <br>
dipalmitoyl-sn-glycero-3-hydroxyphosphoryloxy)-ethylamine (MTP-PE), etc.<br>
The immunogeric compositions (eg. the immunising antigen/immunogen/polypeptide/protein/nucleic acid, pharmaceutically<br>
acceptable carrier, and adjuvant) typically will contain diluents, such as water, saline, glycerol, ethanol, etc. Additionally, auxiliary<br>
substances, such as wetting or emulsifying agents, pH buffering substances, and the like, may be present in such vehicles..<br>
Typically, the irnmunogenic compositions are prepared as injectables, either as liquid solutions or suspensions; solid forms suitable<br>
for solution in, or suspension in, liquid vehicles prior to injection may also be prepared. The preparation also may be emulsified or<br>
encapsulated in liposomes for enhanced adjuvant effect, as discussed above under pharmaceuotically acceptable carriers.<br>
Immunogenic compositions used as vaccines comprise an immunologically effective amount of the antigenic or immunogenic<br>
polypeptides, as well as any other of the above-mentioned components, as needed By "immumologically effective amount", it is<br>
meant that the administration of that amount to an individual, either in a single dose or as part of a series, is effective for treatment<br>
or prevention. This amount varies depending upon the health and physical condition of the indrvidual to be treated, the taxonomic<br>
group of individual to be treated (eg. nonhuman primate, primate, etc.), the capacity of the individual's immune system to<br>
synthesize antibodies, the degree of protection desired, the formulation of the vaccine, the treating doctor's assessment of the<br>
medical situation, and other relevant factors. It is expected that the amount will fell in a relatively broad range that can be<br>
determined through routine trials.<br>
The immunogenic compositions are conventionally administered parenterally, eg. by injection, either subcutaneously,<br>
intramuscularly, or transdennally/transcutaneously (eg. WO98/20734). Additional formulations; suitable for other modes of<br>
administration include oral and pulmonary formulations, suppositories, and transdermal applications. Dosage treatment may be a<br>
single dose schedule or a multiple dose schedule. The vaccine may be administered in conjunction wife other immunoregulatory<br>
agents.<br>
As an alternative to protein-based vaccines, DMA vaccination may be used (eg. Robinson &amp; Tones (1997) Seminars in<br>
Immunol 9:271-283; Donnelly et al (1997) Annu Rev Immunol 15:617-648; later herein].<br>
Gene Delivery Vehicles<br>
Gene therapy vehicles for delivery of constructs including a coding sequence of a therapeutic of the invention, to be delivered to<br>
the mammal for expression in the mammal, can be administered either locally or systemically. Those constructs can utilize viral or<br>
non-viral vector approaches in in vivo or ex vivo modality. Expression of such coding sequence can be induced using endogenous<br>
mammalian or heterotogous promoters. Expression of the coding sequence in vivo can be either constitutive or regulated.<br>
The invention includes gene delivery vehicles capable of expressing the contemplated nucleic add sequences. The gene delivery<br>
vehicle is preferably a viral vector and, more preferably, a retroviral, adenovinal, adeno-associated viral (AAV), herpes viral, or<br>
alphavirus vector. The viral vector can also be an astrovirus, coranavirus, orthomyxovirus, papovavirus, paramyxovirus,<br>
parvovirus, picomavirus, poxvirus, or togavirus viral vector. See generally, Jolly (1994) Cancer Gene Therapy 1:51-64; Kimuna<br>
(1994) Human Gene Therapy 5:845-852; Connelly (1995) Human Gene Therapy 6:185-193; and Kaplitt (1994) Nature<br>
Genetics 6:148-153.<br>
Retroviral vectors are well known in the art and we contemplate that any retroviral gene therapy vector is employable in the<br>
invention, including B, C and D type retroviruses, xenotropic rettoviruses (for example, NZB-X1, NZB-X2 and NZB9-1 (see<br>
O'Neill (1985) J. Virol 53:160) polytropic retroviruses eg. MCF and MCF-MLV (see Kelly (1983) J. Virol. 45:291),<br>
spumaviruses and lentrviruses. See RNA Tumor Viruses, Second Edition, Cold Spring Harbor Laboratory, 1985.<br>
Portions of the retroviral gene therapy vector may be derived from different letrovinises. For example, retrovector l.TRs may be<br>
derived from a Murine Sarcoma Virus, a tRNA binding site from a Rous Sarcoma Virus, a packaging signal item a Murine<br>
l.eukemia Virus, and an origin of second strand synthesis from an Avian l.eukosis Virus.<br>
These recombinant retroviral vectors may be used to generate transduction competent retioviral vector particles by introducing<br>
them into appropriate packaging cell lines (see US patent 5,591,624). Retrovirus vectors can be constructed for site-specific<br>
integration into host cell DNA by incorporation of a chimeric integrate enzyme into the retroviral particle (see WO96/37626). It is<br>
preferable that the recombinant viral vector is a replication defective recombinant virus.<br>
Packaging cell lines suitable for use with the above-described retrovirus vectors are well known in the art, are readily prepared<br>
(see WO95/30763 and WO92/05266), and can be used to create producer cell lines (also tamed vector cell lines or "VCLs")<br>
for the production of recombinant vector particles. Preferably, the packaging cell lines are made from human parent cells (eg.<br>
HT1080 cells) or mink patent cell lines, which eliminates inactivation in human serum.<br>
Preferred retroviruses for the construction of retroviral gene therapy vectors include Avian Leukosis Virus, Bovine Leukemia,<br>
Virus, Murine Leukemia Virus, Mink-Cell Focus-Inducing Vims, Murine Sarcoma Virus, Reticuloendotheliosis Virus and Rous<br>
Sarcoma Virus. Particularly preferred Munine Leukemia Viruses include 4070A and 1504A (Hartley and Rowe (1976) J Virol<br>
19:19-25), Abelson (ATCC No. VR499), Friend (ATCC No. VR-245), Graffi, Gross (ATCC No. VR-590), Kirsten, Harvey<br>
Sarcoma Virus and Rauscher (ATCC No. VR-998) and Moloney Murine leukemia Virus (ATCC No. VR-190). Such<br>
retroviruses may be obtained from depositories or collections such as the American Type Culture Collection ("ATCC") in<br>
Rockville, Maryland or isolated from known sources using commonly available techniques.<br>
Exemplary known retroviral gene therapy vectors employable in this invention, include those described in patent applications<br>
GB2200651, EP0415731, EP0345242, EP0334301, WO89/02468; WO89W5349, WO89/09271, WO90/02806,<br>
WO90/07936, WO94/03622, WO93/25698, WO93/25234, WO93/11230, WO93/10218, WO91/02805, WO91/02825,<br>
WO95/07994, US 5,219,740, US 4,405,712, US 4,861,719, US 4,980,289, US 4,777,127, US 5,591,624. See also Vile<br>
(1993) Cancer Res 53:3860-3864; Vile (1993) Cancer Res 53:962-967; Ram (1993) Cancer Res 53 (1993) 83-88;<br>
Takamiva (1992)J Neurosci Res 33:493-503; Baba (1993) J Neurosurg 79:729-735; Mann (1983) Cell 33:153; Cane (1984)<br>
Proc Nail Acad Sci 81:6349; and Miller (1990) Human Gene Therapy 1.<br>
Human adenoviral gene therapy vectors are also known in the art and employable in this invention. See, for example, Berkner<br>
(1988) Biotechniques 6:616 and Rosenfeld (1991) Science 252:431, and WO93/07283, WO93/06223, and WO93/07282.<br>
Exemplary known adenoviral gene therapy vectors employable in this invention include those described in the above referenced<br>
documents and in WO94/12649, WO93/03769, WO93/19191, WO94/28938, WO95/11984, WO95/00655, WO95/27071,<br>
WO95/29993, WO95/34671, WO967/055320, WO94/08026, WO94/11506, WO93/06223, WO94/24299, WO95/14102,<br>
WO95/24297, WO95/02697, WO94/28152, WO94/24299, WO95/09241, WO95/25807, WO95/05835, WO94/18922 and<br>
WO95/09654. Alternatively, administration of DNA linked to killed adenovirus as described in Curiel (1992) Hum. Gene Ther.<br>
3:147-154 may be employed. The gene delivery vehicles of the invention also include adenovirus associated virus (AAV) vectors.<br>
l.eading and preferred examples of such vectors for use in this invention are the AAV-2 based vectors disclosed in Srivastava,<br>
WO93/09239. Most preferred AAV vectors comprise the two AAV inverted terminal repeats in which the native Dsequences<br>
are modified by substitution of nuckotides, such that at least 5 native nudeotides and up to 18 native nudeotides, preferably at<br>
least 10 native nudeotides up to 18 native nudeotides, most preferably 10 native nucleotides are retained and the remaining<br>
nucleotides of the D-sequence are deleted or replaced with non-native nucleotides. The native D-sequences of the AAV inverted<br>
terminal repeats are sequences of 20 consecutive nudeotides in each AAV inverted terminal repeat (ie there is one sequence at<br>
each end) which are not involved in HP formation. The non-native replacement nucleotide may be any nucleotide other than the<br>
nucleotide found in the native Dsequence in the same position. Other employable exemplary AAV vectors are pWP-19,<br>
pWN-1, both of which are disclosed in Nahreini (1993) Gene 124:257-262. Another example of such an AAV vector is<br>
psub201 (see Samulski (1987) J. Virol. 61:3096). Another exemplary AAV vector is the Doubk-D ITR vector. Construction of<br>
the Doubk-D TTR vector is disclosed in US Patent 5,478,745. Still other vectors are those disdosed in Carter US Patent<br>
4,797368 and Muzyczka. US Patent 5,139,941, Chartejee US Patent 5,474,935, and Kotin WO94/288157. Yet a further<br>
example of an AAV vector employable in this invention is SSV9AFABTKneo, which contains the AFP enhancer and albumin<br>
promoter and directs expression predominantly in the liver. Its structure and construction are disdosed in Su (1996) Human<br>
Gene Therapy 7:463470. Additional AAV gene therapy vectors are described in US 5,354,678, US 5,173,414, US<br>
5,139,941, and US 5,252,479.<br>
The gene therapy vectors of the invention also include herpes vectors. Leading and preferred examples are herpes simplex virus<br>
vectors containing a sequence encoding a thymidine kinase polypeptide such as those disclosed in US 5,288,641 and<br>
EP0176170 (Roizman). Additional exemplary herpes simplex virus vectors include HFEM/ICP6-LacZ disdosed in<br>
WO95/04139 (Wistar Institute), pHSVlac described in Geller (1988) Science 241:1667-1669 and in WO90/09441 and<br>
WO92/07945, HSV Us3::pgC-lacZ described in Fink (1992) Human Gene Therapy 3:11-19 and HSV 7134,2 RH 105 and<br>
GAM described in EP 0453242 (Breakefield), and those deposited with (he ATCC with accession numbers VR-977 and<br>
VR-260.<br>
Also contemplated are alpha virus gene therapy vectors that can be employed in mis invention. Preferred alpha virus vectors are<br>
Sindbis viruses vectors. Togaviruses, Semliki Forest virus (ATCC VR-67; ATCC VR-1247), Middkberg virus (ATCC<br>
VR-370), Roes River virus (ATCC VR-373; ATCC VR-1246), Venezndan equine encephalitis virus (ATCC VR923; ATCC<br>
VR-1250; ATCC VR-1249; ATCC VR-532), and those described in US patents 5,091,309, 5,217,879, and WO92/10578.<br>
More particularly, those alpha vinis vectors described in US Serial No. 08/405,627, filed March 15, 1995,WO94/21792,<br>
WO92/10578, WO95/07994, US 5,091,309 and US 5,217,879 are employable. Such alpha viruses may be obtained from<br>
depositories or collections such as the ATCC in Rockville, Maryland or isolated from known sources using commonly available<br>
techniques. Preferably, alphavirus vectors with reduced cytotoxicity are used (see USSN 08/679640).<br>
DNA vector systems such as eukaiyotic layered expression systems are also useful for expressing the nucleic acids of the<br>
invention. See WO95/07994 for a detailed description of eukaryotic layered expression systems. Preferably, the eukaiyotic<br>
layered expression systems of the invention are derived from alphavirus vectore and most preferably from Sindbis viral vectors.<br>
Other viral vectors suitable for use in the present invention include those derived from polioviius, for example ATCC VR-58 and<br>
those described in Evans, Nature 339 (1989) 385 and Sabin (1973) J. Biol. Standardization 1:115; rhimovirus, for example<br>
ATCC VR-1110 and those described in Arnold (1990)J CellBiochem L401; pox viruses such as canary pox virus or vaccinia<br>
virus, for example ATCC VR-111 and ATCC VR-2010 and those described m Fisher-Hoch (1989) Proc Nati Acad Sci<br>
86:317; Flexner (1989) Ann NY Acad Sci 569:86, Eexner (1990) Vaccine 8:17; in US 4,603,112 and US 4,769330 and<br>
WO89/01973; SV40 virus, for example ATCC VR-305 and those described in Mulligan (1979) Nature 277:108 and Madzak<br>
(1992) J Gen Virol 73:1533; influenza virus, for example ATCC VR-797 and recombinant influenza viruses made employing<br>
reverse genetics techniques as described in US 5,166,057 and in Enami (1990) Proc Nad Acad Sci 87:3802-3805; Enami &amp;<br>
Palese (1991) J Virol 65:2711-2713 and Luy^es (1989) Cell 59:110, (see also McMichael (1983) NEJ Med 309:13, and Yap<br>
(1978) Nature 273:238 and Nature (1979) 277:108); human immunodeficiency virus as described in EP-0386882 and in<br>
Buchschacher (1992) J. Virol. 662731; measles virus, for example ATCC VR-67 and VR-1247 and those described in EP-<br>
0440219; Aura virus, for example ATCC VR-368; Bebaru virus, for example ATCC VR-600 and ATCC VR-1240; Cabassou<br>
virus, for example ATCC VR-922; Chiknngunya virus, for example ATCC VR-64 and ATCC VR-1241; Fort Morgan Virus,<br>
for example ATCC VR-924; Getah virus, for example ATCC VR-369 and ATCC VR-1243; Kyzyiagacfa virus, for example<br>
ATCC VR-927; Mayaro virus, for example ATCC VR-66; Mucambo virus, for example ATCC vk-580 and ATCC VR-1244;<br>
Ndumu virus, for example ATCC VR-371; Pixuna virus, for example ATCC VR-372 and ATCC VR-1245; Tonate virus, for<br>
example ATCC VR-925; Triniti virus, for example ATCC VR-469; Una virus, for example ATCC VR-374; Whataroa virus, for<br>
example ATCC VR-926; Y-62-33 virus, for example ATCC VR-375; CTNyang virus, Eastern encephalitis virus, far example<br>
ATCC VR-65 and ATCC VR-1242; Western encephalitis virus, for example ATCC VR-70, ATCC VR-1251, ATCC VR-622<br>
and ATCC VR-1252; and coronavirus, for example ATCC VR-740 and those described in Hamre (1966) Proc Soc Exp Biol<br>
Med 121:190.<br>
Delivery of the compositions of this invention into cells is not limited to the abwe mentioned viral vectors. Other delivery methods<br>
and media may be employed such as, for example, nucleic acid expression vectors, polycatknic condensed DNA linked or<br>
unlinked to lolled adenovirus alone, for example see US Serial No. 08/366,787, filed December 30,1994 and Curiel (1992)<br>
Hum Gene Ther 3:147-154 ligand linked DNA, for example see Wu (1989) J Biol Chem 264:16985-16987, eucaryotic cell<br>
delivery vehicles cells, for example see US Serial No.08/240,030, filed May 9,1994, and US Serial No. 08/404,796, (fcpoatkm<br>
of photoporymerized hydrogel materials, hand-held gene transfer partide gun, as described in US Patent 5,149,655, ionizing<br>
radiation as described in US5,206,152 and in WO92/11033, nucleic charge neutralization or fusion with cell membranes.<br>
Additional approaches are described in Philip (1994) Mol Ceil Biol 14:2411-2418 and in Woffendin (1994) Proc Natl Acad<br>
Sci 91:1581-1585.<br>
Particle mediated gene transfer may be employed, for example see US Serial No. 60/023,867. Briefly, me sequence can be<br>
insetted into conventional vectors that contain conventional control sequences lor high level expression, and then incubated with<br>
synthetic gene transfer molecules such as polymeric DNA-binding cations like rjolylysine, protamine, and albumin, linked to cell<br>
targeting ligands such as asialoorosomucoid, as described in Wu &amp; Wu (1987) J. Biol. Chem. 262:4429-4432, insulin as<br>
described in Hucked (1990) Biochem Pharmacol 40:253-263, galactose as described in Plank (1992) Bioconjugate Chem<br>
3:533-539, lactose or transfetrin.<br>
Naked DNA may also be employed. Exemplary naked DNA introduction methods arc described in WO 90/11092 and US<br>
5,580,859. Uptake efficiency may be improved using biodegradable latex beads. DNA crated latex beads are efficiently<br>
transported into cells after endocytosis initiation by the beads. Hie method ma}' be improved further by treatment of the beads to<br>
increase hydrophobicity and thereby facilitate disruption of the endosome and release of the DNA into the cytoplasm.<br>
Liposomes that can act as gene delivery vehicles are described in US 5,422,120, WO95/13796, WO94/23697, WO91/14445<br>
and EP-524,968. As described in USSN. 60/023,867, on non-viral delivery, the nucleic acid sequences encoding a polypepnde<br>
can be inserted into conventional vectors that contain conventional control sequences for high level expression, and then be<br>
incubated with synthetic gene transfer molecules such as polymeric DNA-binding cations like polylysine, protamine, and albumin<br>
linked to cell targeting ligands such as asialoorosomucoid, insulin, galactose, lactose, or transferrin. Other delivery systems include<br>
the use of liposomes to encapsulate DNA composing the gene under the control of a variety of tissue-specific or<br>
ubiquitously-active promotes. Further non-viral deliveiy suitable for use includes mechanical delivery systems such as the<br>
approach described in Woffendin et al (1994) Proc. Nad. Acad. Sci. USA 91(24):11581-11585. Moreover, the coding<br>
sequence and the product of expression of such can be delivered through deposition of photopolymerized hydrogel materials.<br>
Other conventional methods for gene delivery that can be used for delivery of the coding sequence include, for example, use of<br>
hand-held gene transfer particle gun, as described in US 5,149,655; use of ionizing radiation for activating tansferred gene, as<br>
described in US 5,206,152 and WO92/11033<br>
Exemplary liposome and polycationic gene delivery vehicles are those described in US 5,422,120 and 4,762,915; in WO<br>
95/13796; WO94/23697; and WO91/14445; in EP-0524968; and in Stryer, Biochemistry, pages 236-240 (1975) W.H<br>
Freeman, San Francisco; Szoka (1960) Biochem Biophys Acta 600:1; Bayer (1979) Biochem Biophys Acta 550:464; Rivnay<br>
(1987) Meth Enzymol 149:119; Wang (1987) Proc Natl Acad Sci 84:7851:. Plant (1989) Anal Biochem 176:420.<br>
A Dolynudeotide composition can comprises therapeutically effective amount of a gene therapy vehicle, as the term is defined<br>
above. For purposes of the present invention, an effective dose will be from about 0.01 mg/ kg to 50 mg/kg or 0.05 mg/kg to<br>
about 10 mg/kg of the DNA constructs in the individual to which it is administered.<br>
Delivery Methods<br>
Once formulated, the polynucleotide compositions of the invention can be administered (1) directly to the subject; (2) delivered ex<br>
vivo, to cells derived from the subject; or (3) in vitro for expression of recombinant proteins. The subjects to be treated can be<br>
mammals or birds. Also, human subjects can be treated.<br>
Direct delivery of the compositions will generally be accomplished by injection, either subcutaneousty, intraperitoneally,<br>
intravenously or intramuscularly or delivered to the interstitial space of a tissue. The compositions can also be administered into a<br>
lesion. Other modes of adminstation include oral and pulmonary administration, suppositories, and transdermal or transcutaneous<br>
applications (eg. see WO98/2Q734), needles, and gene guns or hyposprays. Dosage treatment may be a single dose schedule or<br>
a multiple dose schedule.<br>
Methods for the ex vivo delivery and reimpiantation of transformed cells into a subject are known in the art and described in eg.<br>
WO93/14778. Examples of cells useful in ex vivo applications include, for example, stem cells, particularly hematopoetic, lymph<br>
cells, macrophages, dendritic cells, or tumor cells.<br>
Generally, delivery of nucleic acids far both ex vivo and in vitro applications can be accomplished by the following procedures,<br>
for example, dextran-mediated transfecticn, calcium phosphate precipitation, polybrene mediated tansfection, protoplast fusion,<br>
electroporation, encapsulation of the polynucleotide(s) in liposomes, and direct microinjection of the DNA into nuclei, all well<br>
known in the art.<br>
Polynucleotide and polypeptide pharmaceutical compositions<br>
In addition to the pharmaceutically acceptable carriers and salts described above; the following additional agents can be used with<br>
polynuckotide and/or polypeptide compositions.<br>
A Polypeptides<br>
One example are polypeptides which include, without limitation: asioloorosomucoid (ASOR); transferrin; asialoglycoproteins;<br>
antibothes; antibody fragments; fernitin; interkukins; interferons, granulocyte, macrophage cokxry stimulating factor (GM-CSF),<br>
gtanulocyte colony stimulating factor (G-CSF), macrophage colony stimulating factor (M-CSF), stem cell factor and<br>
erythropoietia Viral antigens, such as envelope proteins, can also be used. Also, proteins from other invasive organisms, such as<br>
the 17 amino acid peptide from the encumsporozoite protein of plasmodium falciparum known as R1l.<br>
B.Honnones. Vitangns. etc.<br>
Other groups that can be included are, for example: hormones, steroids, androgeiis, estrogens, thyroid hormone, or vitamins, folk<br>
acid<br>
C-Polyalkylenes, Polysaccharides. etc.<br>
Also, polyalkykne glycol can be included with the desiied polynucdeotides/polypeptides. In a preferred embodiment, tbe<br>
poryalkylene glycol is polyethylene glycol. In addition, mono-, di-, or polysaccharides can be included. In a preferred<br>
embodiment of this aspect, the polysaccharide is dextran or DEAE4eKlran. Also, chitosan and poly(lactide-co-glycolide)<br>
D.Lipids. and Liposomes<br>
The desiied polynucleotide/polypeptide can also be encapsulated in lipids or packaged in liposomes prior to delivery to the<br>
subject or to cells derived therefrom.<br>
Lipid encapsulation is generally accomplished using liposomes which are able to stably bind or entrap and retain nucleic acid. The<br>
ratio of condensed polynudeotide to lipid preparation can vary but will generally be around 1:1 (rng DNAmicromoles lipid), or<br>
more of lipid. For a review of tbe use of liposomes as carriers for delivery of nucleic acids, see, Hug and Sleight (1991 )Biochim.<br>
Biophys.Acta. 1097:1-17; Straubinger (1983) Meth.Enzymol 101:512-527.<br>
Liposomal preparations far use in tbe present invention include cationic (positively charged), anionic (negatively charged) and<br>
neutral preparations. Cationic liposomes have been shown to mediate intracellular delivery of plasmid DNA (Feigner 0987)<br>
Proc. Nad. Acad. Sci. USA 84:7413-7416); mRNA (Malone (1989) Proc Natl. Acad Sci. USA 86:6077-6081); and purified<br>
transcription factors (Debs (1990) J. Biol Chein. 265:10189-10192), in functional form.<br>
Cationic Iiposomes are readily available. For example, N[l-2,3-dioleyloxy)propyl]-N,N,N-triethylamonium (DOTMA)<br>
liposomes are available under the trademark Lipofectin, from GIBCO BRI, Grand Island, NY. (See, also Frlgner supra). Other<br>
commercially available lipesomes indude transfectace (DDAB/DOPE) and DOTAP/DOPE (Boerhinger). Other cationic<br>
liposomes can be prepared from readily available meterials using techiques well known in the art See, eg. Szoka (1978) Proc.<br>
Natl. Acad, Sci. USA 75:4194-4198; WO90/11092 for a description of the synthesis of DOTAP<br>
(l,2-bis(oleoyloxy)-3-(trimethylammonio)propane) liposomes.<br>
Similarly, anionic and neutral Iiposomes are readily available, such as from Avanti Polar Lipids (Birmingham, AL), or can be easily<br>
prepared using readily available materials. Such materials include phospharkryi cholrne, cholesterol, phosphatidyl ethanolamme,<br>
dioleoylphospbatidyl cboline (DOPQ, dioleoylphospharklyl gtycerol (DOPG), dioleoyiphoshatdyi ethanolamine (DOPE), among<br>
others. These materials can also be mixed with the DOTMA and DOTAP starting materials in appropriate ratios. Methods for<br>
making liposomes using these materials are well known in the art.<br>
The liposomes can comprise multilammeiar vesicles (Ml.Vs), small unilamellar vesicles (SUVs), or large unflamrilar vesicles<br>
(LUVs). The various liposoms-nucleic acid complexes are prepared using methods known in the sat See eg. Straubinger (1983)<br>
Meth. Immunol. 101^12-527; Szoka (1978) Proc. Nad. Acad. Sci. USA 75:4194-4198; Papahadjopoulos (1975) Biochim.<br>
Biophys. Acta 394:483; Wilson (1979) Cell 17:77); Deamer &amp; Bangbam (1976) Biochim. Biopkys. Ada 443:629; Ostro<br>
(1977) Biochem. Biopkys. Res. Commun. 76:836; Fraley (1979) Proc. Nad. Acad. Scl. USA 763348); Enoch &amp; Stritrmatter<br>
(1979) Proc. Natl. Acad. Sci. USA 76:145; Fraley (1980) /. Biol Chan. (1980) 255:10431; Szoka &amp; Papahadjopoulos<br>
(1978) Proc. Natl. Acad. Sci. USA 75:145; and Schaefer-Ridder (1982) Science 215:166.<br>
ELipoproteins<br>
In addition, lipoproteins can be included with the polvnucfeodde/polypeptide to be delivered. Examples of Kpoproteins to be<br>
utilized include: chyiomicrons, HDL, IDL, LDL, and VLDL Mutants, fragments, or fusions of these proteins can also be used<br>
Abo, modifications of naturally occurring lipoprotons can be used, such as acetybted LDL. These lipoproteins can target the<br>
delivery of pdynucleotides to cells expressing bpoprotein receptors. Preferably, if lipoproteins are including with the<br>
polyriudeotidetobeMvCTedr»otirctarge^<br>
Naturally occurring lipoproteins comprise a lrpid and a protein portion. The protein portion are known as apoproteins. At the<br>
present, apoproteins A, B, C, D, and E have been isolated and identified. At least two of these contain several proteins,<br>
designated by Roman numerals, AI, AII, AIV; CI, CII, CIII.<br>
A Hpoprotein can comprise more than one apoprotein. For example, naturally occurring drykxrncrons comprises of A, B, C &amp; E,<br>
over time these lipoproteins lose A and acquire C &amp; F.. VLDL comprises A, B, C &amp; E apoproteins, LDL comprises apoprotein<br>
B; and HDL comprises apoproteins A, C, &amp; E<br>
The amino acid of these apoproteins are known and are described in, for example, Breslow (1985) Annu Rev. Biochem 54:699;<br>
Law (1986) Adv. Exp Med. Bid 151:162; Chen (1986) J Biol Chem 261:12918; Kane (1980) Proc Natl Acad Soi USA<br>
77:2465; and Utermann (1984) Hum Genet 65:232<br>
Lipoproteins contain a variety of lipids induing, trigrycerides, cholesterol (free and esters), and phosphohpids. The composition<br>
of the lipids varies in naturally occurring lipoproteins. For example, chylomicrons comprise mainly triglycerides. A more detailed<br>
description of the lipid content of naturally occuring lipoproteins can be found, for example, in Meth. Enzymol. 128 (1986). The<br>
composition of the lipids ate chosen to aid in conformation of the apoprotan for receptor binding activity. The composition of<br>
lipids can also be chosen to facititate hydrophobic interaction and association with the polynucleotide binding molecule.<br>
Naturally occuning lipoproteins can be isolated from serum by ultiacentrifugation, for instance. Such methods are described in<br>
Metk Enzymol (supra); Pitas (1980) J. Biochem. 255:5454-5460 and Mahey (1979) J Clin. Invest 64:743-750.<br>
l.ipoproteins can also be produced by in vitro or lecombinant methods by expression of the apopotem genes in a desired host<br>
cell. See for example, Atkinson (1996) Annu Rev Biophys Chem 15:403 and Radding (195$) Biochim Biophys Acta 30.443.<br>
l.ipoproteins can also be purchased from commercial suppliers, such as Biomedical Techniologies, Inc., Stoughton, MA, USA.<br>
Further description of lipoproteins can be found in WO98/06437..<br>
E.Polycationic Agents<br>
Polycationic agents can be included, with or without Hpopnrtein, in a composition with the desired polynucleotide/polypeptide to<br>
be delivered.<br>
Pdycatkric agents, typically, exhibit a net positive charge at physiological relevant pH and are capable of neutralizing the<br>
electrical charge of nucleic acids to facilitate delivery to a desired location. These agents have both, in vitro, ex vivo, and in vivo<br>
applications. Polycatknic agents can be used to deliver nucleic adds to a living subject either intramuscularly, subcutaneousry,efc.<br>
The Mowing are examples of useful polypeptides as polycationic agents: polyrysine, polyarginine, polyomitrrine, and protamine.<br>
Other examples include bistones, protamines, human serum albumin, DNA binding protons, ncoiKtooe chromosomal proteins,<br>
coat proteins from DNA viruses, such as (X174, transcriptional factors also contam domains that bind DNA and therefore may<br>
be useful as nucleic aid condensing agents. Briefly, transcriptional factors such as C/CEBP, ejun, c-fos, AP-1, AP-2, AP-3,<br>
CPF, Prot-1, Sp-1, Oct-1, Oct-2, CREP, andTFUD roiitain bask domaiiis that bind DNA secfiaices.<br>
Organic polycationic agents include: spermine, spemidine, and purtrescine.<br>
The dimensions and of the physical properties of a polycationic agent can be extrapolated from the list above, to construct other<br>
porypeptide polycationic agents or to produce synthetic polycatknic agents.<br>
Synthetic porycationic agents winch are useful include, for example, DEA&amp;dextran, polybreoe. l.ipofectin™, and<br>
lipofedAMINE™ are inononro that fempotycat^^<br>
Immunodiagnostic Assays<br>
Streptococcus antigens of the invention can be used in immunoassays to detect antibody levels (or conversely anti-streptococcus<br>
antibothes can be used to detect antigen levels). Immunoassays based an well defined, recombmant antigens can be developed to<br>
or serum samples, can be detected Design of the immunoassays is subject to a great deal of variation, and a variety of these are<br>
known in the art Protocols for (he rnimunnassgy may be based, for example, upon competition, <r direct reaction or sandwich></r>
type assays. Protocols may also, for example, use solid supports, or may be by unmumprecipitation. Most assays involve the use<br>
Assays which amplify the signals from the probe are also known; examples ofwbicfa are assays wrjkhutifeebictin and avidin, and<br>
enzyme-labeled and mediated immuooassays, such as El.BA assays.<br>
Kits suitable for immunodiagnosis and containing the appropriate labeled reagents are constructed by packaging the appropriate<br>
materials, including the compositions of the invention, in suitable containers, along with the renaming reagents and materials (for<br>
example, suitable buffers, safe solutions, etc.) required for the conduct of the assay, as well as suitable set of assay instructions.<br>
Nucleic Acid Hybridisation<br>
"Hybridization" refers to the association of two nucleic acid sequences to one another by hydrogen bonding. Typically, one<br>
sequence will be fixed to a solid support and the other will be free in solution. Then the two sequeoxs will be placed in contact<br>
with one another under conditions that favor hydrogen bonding. Factors that atfect ths hooding include: the type and volume of<br>
solvent; reaction temperature; time of hybridization; agitation; agents to block the non-specific attachment of the liquid phase<br>
sequence to the solid support (Denhardt's reagent or BLOTTO); concentration of the sequences; use of compounds to increase<br>
the rate of association of sequences (dextan sulfate or polyethylene glycol); and the stringency of tie washing conditions following<br>
hybridization. See Sambrooket al. [supra] Volume 2, chapter 9, pages 9.47 to 9..T.<br>
"Stringency" refers to conditions m a hybridization reaction that favor association of very similar sequences over sequences that<br>
differ. For example, !he combination ofteiupeiuture and salt concentration should be chosen that is approximately 120 to 20CC<br>
below the calculated Tm of the hybrid under study. The temperature and sat conditions can often be determined empirically id<br>
preliminary experiments in which samples of genomic DNA immobilized on fikets are hybridized to the sequence of interest and<br>
then washed under conditions of different stringencies. See Sarnbrook et al., at page 9.50.<br>
Variables to consider when performing, for example, a Southern blot are (1) the complexity of the DNA being blotted and (2) the<br>
homology between the probe and the sequences being detected The total amount of the fragments) to be stuthed can vary a<br>
magnitude of 10, from 0.1 to lug for a plasnrid or phage digest to 10-9 to 10-8 g tor a single copy gene in a highly complex<br>
eukaryotic genome. For lower complexity porynucleotides, substantially shorter blotting, hybridization, and exposure times, a<br>
smaller amount of starting polynucleotides, and lower specific activity of probes can be used. Kor example, a single-copy yeast<br>
gene can be detected with an exposure time of only 1 hour starting with 1 µg of yeast DNA, blotting for two hours, and<br>
hybridizing for 4-8 hours with a probe of 105 cpnVug. For a single-copy mammalian gene a conservative approach would start<br>
with 10 µg of DNA, blot overnight, and hybridize overnight in the presence of 10% dextran sulfaie using a probe of greater than<br>
108 cpm/µg, resulting in an exposure time of ~24 hours.<br>
Several factors can affect the melting temperature (Tin) of a DNA-DNA hybrid between the probe and the fragment of interest,<br>
and consequently, the appropriate conditions for hybridization and washing. In many cases the probe is not 100% homologous to<br>
the fragment Other commonly encountered variables include the, length and total G+C content of the hybridizing sequences and<br>
the ionic strength and formamide content of Ihe hybridization buffer. The effects of all of these factors can be approximated by a<br>
single equation:<br>
Tm= 81 + 16.6(log0Ci) + 0.4[%(G + C)]-0.6(%formamide) - 600/n-1.5(%mismatch).<br>
where Q is the salt concentration (monovalent ions) and n is the length of the hybrid in base pairs (slightly modified from<br>
Meinkoth&amp;Wabl (1984) And. Biochem. 138: 267-284).<br>
In designing a hybridization experiment, some factors affecting nucleic acid hybridization can be conveniently altered. The<br>
temperature of the hybridization and washes and the salt concentration during the washes are the simplest to adjust As the<br>
temperature of the hybridization increases (ie. stringency), it becomes less likely for hybridization to occur between strands that<br>
are nonbomologous, and as a result, background decreases. If the radiolabeled probe is not completely homologous with the<br>
immobilized fragment (as is frequently the case in gene family and interspecies hybridization experiments), the hybridization<br>
temperature must be reduced, and background will increase. The temperature of the washes affects the intensity of the hybridizing<br>
band and the degree of background in a similar manner. The stringency of the washes is also increased with decreasing salt<br>
concentrations.<br>
In general, convenient hybridization temperatures in the presence of 50% formamide are 42°C tor a probe with is 95% to 100%<br>
homologous to the target fragment, 37°C for 90% to 95% homology, and 32°C for 85% to 90% homology. For lower<br>
homologies, fonnamide content should be lowered and temperature adjusted accordingly, usdng the equation above. If the<br>
homology between the probe and the target fragment are not known, the simplest approach is to start with both hybridization and<br>
wash conditions which are nonstringent. If non-specific bands or high background are observed after autoradiography, the filter<br>
can be washed at high stringency and reexposed. If the time required for exposure makes this approach impractical, several<br>
hybridization and/or washing stringencies should be tested in parallel.<br>
Nucleic Acid Probe Assays<br>
Methods such as PCR, branched DNA probe assays, or blotting techniques utilizing nucleic acid probes according to the<br>
invention can determine the presence of cDNA or mRNA. A probe is said to "hybridize' with a sequence of the invention if it can<br>
form a duplex or double stranded complex, which is siable enough to be detected.<br>
The nucleic acid probes will hybridize to the streptococcus nucleotide sequences of the invention (including both sense and<br>
ann'sense strands). Though many different nucleotide sequences will encode the amino acid sequence, the native streptococcus<br>
sequence is preferred because it is the actual sequence present in cells. mRNA represents a coding sequence and so a probe<br>
should be complementary to the coding sequence; single-stranded cDNA is compleraentary tn mRNA, and so a cDNA probe<br>
should be complementary to the non-coding sequence.<br>
The probe sequence need not be identical to the streptococcus sequence (or ils axnpiement) — some variation in the sequence<br>
and length can lead to increased assay sensitivity if the nucleic acid probe can form a duplex with target nucleotides, which can be<br>
detected. Also, the nucleic acid probe can include additional nucleotides to stabilize the formed duplex Additional streptococcus<br>
sequence may also be helpful as a label to detect the formed duplex. For example, a nortcomplernentary nucleotide sequence<br>
may be attached to the 5' end of the probe, with the remainder uf the probe sequence being complementary to a streptococcus<br>
sequence. Alternatively, non-complementary bases or longer sequences can be interspersed into the probe, provided that the<br>
probe sequence has sufficient complementarity with the a streptococcus sequence in order to hybridfe therewith and (hereby<br>
form a duplex which can be detected.<br>
The exact length and sequence of the probe wfll depend on the hybridizaticn conditions (eg. temperature, salt condition etc.). For<br>
example, for diagnostic applications, depending on the complexity of the analyte sequence, the nucleic acid probe typically<br>
contains at least 10-20 nudeotides, preferably 15-25, and more preferably at least 30 nucleotides, although it may be shelter than<br>
this. Short primers generally require cooler temperatures to fismisuflBdenth/stabk hybrid axnplexeswifti the teinplate.<br>
103:3185], or according to Urdea et al [Proc Natl. Acad. Sci. USA (1983) 80: 7461], or using commercially available<br>
automated otigonudeotide synthesizers.<br>
The chemical nature of the probe can be selected according to preference. For certain applications, DNA or RNA are<br>
appropriate. For other applications, modifications may be incorporated eg. backbone modifications, such as phosphorcfthioates<br>
or methylphosphonates, can be used to increase in vivo half-life, alter RNA affinity, increase nuclease resistance etc. [eg. see<br>
Agrawal &amp; Iyer (1995) Curr Opin Biotechnol 6:12-19; Agrawal (1996) JWTECH 14:376-387]; analogues such sis peptide<br>
nucleic acids may also be used [eg. see Corey (1997) TWTECH 15:224-229; Buchardtef ai (1993) TWTECHl 1:384-386].<br>
Alternatively, the poiymerase chain reaction (PCR) is another well-known means for detecting small amounts of target nucleic<br>
acid. The assay is described in NMlis et al [Metk Enzymol. (1987) 155335-350] &amp; US patents 4,683,195 &amp; 4,683,202. Two<br>
"primer" nucleorides hybridize with the target nucleic acids and are used to prime the reaction. The primers can comprise<br>
sequence that does not hybridize to the sequence of the amplification target (or its complement) to Eiid with duplex stability or, for<br>
example, to incorporate a convenient restriction site. Typically, such sequence will flank the desired streptococcus sequence.<br>
A thermostable polymerase creates copies of target nucleic acids from the primers using the original target nucleic acids as a<br>
template. After a threshold amount of target nucleic acids are generated by the porymerase, they can be delected by more<br>
traditional methods, such as Southern blots. When using the Southern Wot method, the labelled probe will hybridize to the<br>
streptococcus sequence (or its complement).<br>
Also, mRNA or cDNA can be detected by traditional blotting techniques described in Sambrook et al [supra]. mRNA, or<br>
cDNA generated from mRNA using a poiymerase enzyme, can be purified and separated using gd dectrcphcresis. The nucleic<br>
acids on the gel are then blotted onto a solid support, such as nitrocellulose. The solid support is exposed to a labelled probe and<br>
then washed to remove any unhybridized probe. Next, the duplexes containing the labeled probe are delected Typically, the<br>
probe is labelled with a radioactive moiety.<br>
BRIEF DESCRIPTION OF DRAWINGS<br>
Figures 1 to 85, 119 to 188, 238 and 239 show SDS-PAGE analysis of total cell extracts from<br>
cultures of recombinant E.coli expressing GBS proteins of the invention. Lane 1 in each gel (except for<br>
Figure 185) contains molecular weight markers. These are 94, 67, 43, 30, 20.1 &amp; 14.4 kDa (except for<br>
Figures 7, 8, 10, 11,13,14,15 and 119-170, which use 250, 150, 100, 75, 50, 37, 25, 15 &amp; 10 kDa).<br>
Figure 86A shows the pDESTIS vector and Figure 86B shows the pDEST17~1 vector.<br>
Figures 88 to 118 and 247 to 319 show protein characterisation data for various proteins of the<br>
invention.<br>
Figures 189 to 237 and 240 to 246 show SDS-PAGE analysis of purified GBS proteins of the<br>
invention. The left-hand lane contains molecular weight markers. These are 94,, 67, 43, 30. 20.1 &amp; 14.4<br>
kDa.<br>
MODES FOR CARRYING OUT THE INVENTION<br>
The following examples describe nucleic acid sequences which have been identified in Streptococcus,<br>
along with their inferred translation products. The examples are generally in the following format.<br>
• a nucleotide sequence which has been identified in Streptococcus<br>
• the inferred translation product of this sequence<br>
• a computer analysis (e.g. PSORT output) of the translation product, indicating antigenicity<br>
Most examples describe nucleotide sequences from S.agalactiae. The specific strain which was<br>
sequenced was from serotype V, and is a clinical strain isolated in Italy which expresses the R antigen<br>
(ISS/Rome/Italy collection, strain2603 V/R). For several of these examples, the corresponding<br>
sequences from S.pyogenes are also given. Where GBS and GAS show homology in this way, there is<br>
conservation between species which suggests an essential function and also gives good cross-species<br>
reactivity.<br>
In contrast, several examples describe nucleotide sequences from GAS for which no homolog in GBS<br>
has been identified.. This lack of homology gives molecules which are useful for distinguishing GAS<br>
from GBS and for making GAS-specific products. The same is true for GBS sequences which lack<br>
GAS homologs e.g. these are useful for making GBS-specific products.<br>
The examples typically include details of homology to sequences in the public databases. Proteins that<br>
are similar in sequence are generally similar in both structure and function, and the homology often<br>
indicates a common evolutionary origin. Comparison with sequences of proteins of known function is<br>
widely used as a guide for the assignment of putative protein function to a new sequence and has proved<br>
particularly useful in whole-genome analyses.<br>
Various tests can be used to assess the in vivo immunogenicity of the proteins identified in the examples.<br>
For example, the proteins can be expressed recombinantly and used to screen patient sera by<br>
immunoblot. A positive reaction between the protein and patient serum indicates that the patient has<br>
previously mounted an immune response to the protein in question i.e. the protein is an immunogen. This<br>
method can also be used to identify immunodominant proteins. The mouse model used in the examples<br>
can also be used.<br>
The recombinant protein can also be conveniently used to prepare antibodies e.g. in a mouse. These can<br>
be used for direct confirmation mat a protein is located on the cell-surface. l.abelled antibody (e.g.<br>
fluorescent labelling for FACS) can be incubated with intact bacteria and the presence of label on the<br>
bacterial surface confirms the location of the protein.<br>
For many GBS proteins, the following data are given:<br>
- SDS-PAGE analysis of total recombinant E.coli cell extracts for GBS protein expression<br>
- SDS-PAGE analysis after the protein purification<br>
- Western-blot analysis of GBS total cell extract using antiseta raised against recombinant proteins<br>
- FACS and ELISA analysis against GBS vising antisera raise against recombinant proteins<br>
- Results of the in vivo passive protection assay<br>
Details of experimental techniques used are presented below:<br>
Sequence analysis<br>
Open reading frames (ORFs) within nucleotide sequences were predicted using the GLIMMER program<br>
[Salzberg et al. (1998) Nucleic Acids Res 26:544-8]. Where necessary, start codons were modified and<br>
corrected manually on the basis of the presence of ribosome-binding sites and promoter regions on the<br>
upstream DNA sequence.<br>
ORFs were then screened against the non-redundant protein databases using the programs BLASTp<br>
[Altschul et al. (1990) J. Mol. Biol. 215:403-410] and PRAZE, a modification of the Smith-Waterman<br>
algorithm [Smith &amp; Waterman (1981) J Mol Biol 147:195-7; see Fleischmann et al (1995) Science<br>
269:496-512].<br>
Leader peptides within the ORFs were located using three different approaches: (i) PSORT [Nakai<br>
(1991) Bull. Inst. Chem. Res., Kyoto Univ. 69:269-291; Horton &amp; Nakai (1996) Intellig. Syst. Mol. Biol.<br>
4:109-115; Horton &amp; Nakai (1997) Intellig. Syst. Mol. Biol. 5:147-152]; (ii) SignalP [Nielsen &amp; Krogh<br>
(1998) in Proceedings of the Sixth International Conference on Intelligent Systems for Molecular<br>
Biology (ISMB 6), AAAI Press, Menlo Park, California, pp. 122-130; Nielsen et al. (1999) Protein<br>
Engineering 12:3-9; Nielsen et al. (1997). Int. J. Neural Sys. 8:581-599]; and (iii) visual inspection of the<br>
ORF sequences. Where a signal sequences is given a "possible site" value, the value represents the<br>
C-terminus residue of the signal peptide e.g. a "possible site" of 26 means that the signal sequence<br>
consists of ammo acids 1-26.<br>
l.ipoprotein-specific signal peptides were located using three different approaches: (i) PSORT [see<br>
above]; (ii) the "prokaryotic membrane lipoprotein lipid attachment site" PROSITE motif [Hefmann et<br>
al. (1999) Nucleic Acids Res. 27:215-219; Bucher &amp; Bairoch (1994) in Proceedings 2nd International<br>
Conference on Intelligent Systems for Molecular Biology (ISMB-94), AAAI Press, pages 53-61]; and<br>
(iii) the FINDPATTERNS program available in the GCG Wisconsin Package, using the pattern<br>
(M,L,V)x(9,35}LxxCx.<br>
Transmembrane domains were located using two approaches: (i) PSORT [see above]; (ii) TopPred [von<br>
Heijne (1992) J. Mol. Biol. 225:487-494]<br>
l.PXTG motifs, characteristic of cell-wall attached proteins in Gram -positive bacteria [Fischetti et al.<br>
(1990) Mol Microbiol 4:1603-5] were located with FIKDPATTERNS using the pattern<br>
(l,i,v,m,y,f)px(t,a,s,g) (g,n,s,t,a,l) .<br>
RGD motife, characteristic of cell-adhesion molecules [D'Souza et al. (1991) Trends Biochem Sci<br>
16:246-50] were located using FINDPATTERNS.<br>
Enzymes belonging to the glycolytic pathway were also selected as antigens, because these have been<br>
found experimentally expressed on the surface of Streptococci [e.g Pancholi &amp; Fischetti (1992) ./ Exp<br>
Med 176:415-26; Pancholi &amp; Fischetti (1998) J Biol Chem 273:14503-15].<br>
Cloning, expression and purification of proteins<br>
GBS genes were cloned to facilitate expression in E.coli as two different types of fusion proteins:<br>
a) proteins having a hexa-histidine tag at the amino-terminus (His-gbs)<br>
b) proteins having a GST fusion partner at the amino-terrmnus (Gst-gbs)<br>
Cloning was performed using the Gateway™ technology (Life Technologies), which is based on the site-<br>
specific recombination reactions that mediate integration and excision of phage lambda into and from the<br>
E.coli genome. A single cloning experiment included the following steps:<br>
1- Amplification of GBS chromosomal DNA to obtain a PCR product coding for a single ORF<br>
flanked by attB recombination sites.<br>
2- Insertion of the PCR product into a pDONR vector (containing attP sites) through a BP reaction<br>
(attB x attP sites). This reaction gives a so called 'pEntry' vector, which now contains atth sites<br>
flanking the insert.<br>
3- Insertion of the GBS gene into E.coli expression vectors (pDestination vectors, containing attR<br>
sites) through a l.R reaction between pEntry and pDestination plasmids (attl. x attR sites).<br>
A) Chromosomal DNA preparation<br>
For chromosomal DNA preparation, GBS strain 2603 V/R (Istituto Superiore Sanita, Rome) was grown<br>
to exponential phase in 2 litres TH Broth (Difco) at 37°C, harvested by centrifugation, and dissolved in<br>
40 ml TES (50 mM Tris pH 8, 5 mM EDTA pH 8, 20% sucrose). After addition of 2.5 ml lysozyme<br>
solution (25 mg/ml in TES) and 0.5 ml mutanolysm (Sigma M-9901, 25000U/ml in I^O), the suspension<br>
was incubated at 37°C for 1 hour. 1 ml RNase (20 mg/ml) and 0.1 ml proteinase K (20 mg/ml) were<br>
added and incubation was continued for 30 min. at 37°C<br>
Cell lysis was obtained by adding 5 ml sarkosyl solution (10% N-laurylsarcosine in 250 mM EDTA pH<br>
8.0), and incubating 1 hour at 37°C with frequent inversion. After sequential extraction with phenol,<br>
phenol-chloroform and chloroform, DNA was precipitated with 0.3M sodium apetate pH 5 1 and 2<br>
volumes of absolute ethanol. The DNA pellet was nnsed with 70% ethanol and dissolved m Tl butler<br>
(10 mM Tris-HCl, 1 mM EDTA, pH 8). DNA concentration was evaluated by OD260.<br>
B) Oligonucleotide design<br>
Synthetic oligonucleotide primers were designed on the basis of the coding sequence of each ORF. The<br>
aim was to express the protein's extracellular region. Accordingly, predicted signal peptides were<br>
omitted (by deducing the 5' end amplification primer sequence immediately downstream from the<br>
predicted leader sequence) and C-terminal cell-wall ancoring regions were removed (e.g. l.PXTG motifs<br>
and downstream amino acids). Where additional nucleotides have been deleted, this is indicated by the<br>
suffix 'd' (e.g. GBS352d' - see Table V). Conversely, a suffix 'l.' refers to expression without these<br>
deletions. Deletions of C- orN-terminal residues were also sometimes made, as indicated by a 'C or 'N'<br>
suffix.<br>
The amino acid sequences of the expressed GBS proteins (including 'd' and 'l.' forms etc.) are<br>
definitively defined by the sequences of the oligonuclotide primers given in Table JH.<br>
5' tails of forward primers and 3' tails of reverse primers included attB 1 and attB2 sites respectively:<br>
Forward primers: 5'-GGGGACAAGTTTGTACAAAAAAGCAGGCT(?r-ORF in frame-31 (the TCT<br>
sequence preceding the ORF was omitted when the ORF's first coding triplet began with T).<br>
Reverse primers: 5'-GGGGACCACTTTGTACAAGAAAGCTGGGTT-ORF reverse complement-3'.<br>
The number of nucleotides which hybridized to the sequence to be amplified depended on the melting<br>
temperature of the primers, which was determined as described by Breslauer et al. [PNAS USA (1986)<br>
83:3746-50]. The average melting temperature of the selected oligos was 50-55 °C for the hybridizing<br>
region and 80-85°C for the whole oligos.<br>
C) Amplification<br>
The standard PCR protocol was as follows: 50 ng genomic DNA were used as template m the presence<br>
of 0.5 µM each primer, 200 µM each dNTP, 1.5 mM MgCfc, Jx bu0er minus Mg (Gibco-BRl.) and 2<br>
units of Taq DNA polymerase (Platinum Taq, Gibco-BRl.) in a final volume of 100 ul. Each sample<br>
underwent a double-step of amplification: 5 cycles performed using as the hybridizing temperature 50°C,<br>
followed by 25 cycles at 68°C.<br>
The standard cycles were as follows:<br>
Denaturation: 94°C, 2 min<br>
5 cycles: Denaturation: 94°C, 30 seconds<br>
Hybridization: 50°C, 50 seconds<br>
Elongation: 72°C, 1 min. or 2 min. and 40 sec.<br>
25 cycles : Denaturation: 94°C, 30 seconds<br>
Hybridization: 68°C, 50 seconds<br>
Elongation: 72°C, 1 min. or 2 min. and 40 sec.<br>
Elongation time was 1 minute for ORFs shorter than 2000bp and 2:40 minutes for ORFs longer than<br>
2000bp. Amplifications were performed using a Gene Amp PCR system 9600 (Perkin Elmer).<br>
To check amplification results, 2µl of each PCR product were loaded onto 1-1.5 agarose gel and the<br>
size of amplified fragments was compared with DNA molecular weight standards (DNA marker DC<br>
Roche, 1kb DNA ladder Biolabs).<br>
Single band PCR products were purified by PEG precipitation: 300 ul of TE buffer and 200 ul of 30%<br>
PEG 8000/30 mM MgCl2 were added to 100 µl PCR reaction. After vortexing, the DNA was centrifuged<br>
for 20 min at 10000g, washed with 1 vol. 70% ethanol and the pellet dissolved in 30 µl TE. PCR<br>
products smaller than 350 bp were purified using a PCR purification Kit (Qiagen) and eluted with 30 p.1<br>
of the provided elution buffer.<br>
In order to evaluate the yield, 2µl of the purified DNA were subjected to agarose gel electrophoresis and<br>
compared to titrated molecular weight standards.<br>
D) Clonine of PCR products into expression vectors<br>
Cloning was performed following the Gateway™ technology's "one-tube protocol", which consists of a<br>
two step reaction (BP and l.R) for direct insertion of PCR products into expression vectors.<br>
BP reaction (attB x attP sites): The reaction allowed insertion of the PCR product into a pDONR<br>
vector. The pDONR™ 201 vector we used contains the killer toxin gene ccdB between attP1 and attP2<br>
sites to minimize background colonies lacking the PCR insert, and a selectable marker gene for<br>
kanamycin resitance. The reaction resulted in a so called pEntry vector, in which the GBS gene was<br>
located between attL1 and attL2 sites.<br>
60 finol of PCR product and 100 ng of pDONR™ 201 vector were incubated with 2.5 ul of BP<br>
clonase™ in a final volume of 12.5 µl for 4 hours at 25°C.<br>
l.R reaction (attLx atiR. sites): The reaction allowed the insertion of the GBS gene, now present in the<br>
pEntry vector, into E.coli expression vectors (pDestination vectors, containing attR sites). Two<br>
pDestination vectors were used (pDEST15 for N- terminal GST fusions - Figure 86; and pDEST17-l<br>
for N-terminal His-tagged fusions — Figure 87). Both allow transcription of the ORF fusion coding<br>
mRNA under T7 RNA polymerase promoter [Stuther et al (1990) Meth. Enzymol 185: 60ff].<br>
To 5 µl of BP reaction were added 0.25 µl of 0.75 M NaCl. 100 ng of destination vector and 1.5 µl of<br>
l.R clonase™ . The reaction was incubated at 25°C for 2 hours and stopped with 1 µl of 1 mg/ml<br>
proteinase K solution at 37°C for 15 min.<br>
1 µl of the completed reaction was used to transform 50 µl electrocompetent Bl.21-SI™ cells (0.1 cm,<br>
200 ohms, 25 µF). BL21-SI cells contain an integrated T7 RNA polymerase gene under the control of<br>
the salt-inducible prU promoter [Gowrishankar (1985) J. Bacterial. 164:434ff]. After electroporation<br>
cells were diluted in lml SOC medium (20 g/1 bacto-tryptone, 5 g/1 yeast extract, 0.58 g/1 NaCl, 0.186 g/1<br>
KC1, 20 mM glucose, 10 mM MgCt) and incubated at 37°C for I hour. 200 ul cells were plated onto<br>
l.BON plates (Luria Broth medium without NaCl) containing 100 µg/ ml ampicillin. Plates were then<br>
incubated for 16 hours at 37°C.<br>
Entry clones: In order to allow the future preparation of Gateway compatible pEntry plasmids<br>
containing genes which might turn out of interest after immunological assays, 2.5 µl of BP reaction were<br>
incubated for 15 min in the presence of 3 ul 0.15 mg/ml proteinase K solution and then kept at -20°C.<br>
The reaction was in this way available to transform E.coli competent cells so as to produce Entry clones<br>
for future introduction of the genes in other Destination vectors.<br>
E) Protein expression<br>
Single colonies derived from the transformation of LR reactions were inoculated as small-scale cultures<br>
in 3 ml l.BON 100 ug/ml ampicillin for overnight growth at 25°C. 50-200 ul of the culture was inoculated<br>
in 3 ml l.BON/Amp to an initial OD600 of 0.1. The cultures were grown at 37°C until OD600 0.4-0.6<br>
and recombinant protein expression was induced by adding NaCl to a final concentration of 0.3 M. After<br>
2 hour incubation the final OD was checked and the cultures were cooled on ice. 0.5 OD^o of cells were<br>
harvested by centrifugation. The cell pellet was suspended in 50 ul of protein l.oading Sample Buffer (50<br>
mM TRIS-HC1 pH 6.8, 0.5% w/v SDS, 2.5% v/'v glycerin, 0.05% w/v Bromophenol Blue, 100 mM<br>
DTT) and incubated at 100 °C for 5 min. 10 µl of sample was analyzed by SDS-PAGE and Coomassie<br>
Blue staining to verify the presence of induced protein band.<br>
F) Purification of the recombinant proteins<br>
Single colonies were inoculated in 25 ml l.BON 100 µg/ml ampicillin and grown at 25°C overnight. The<br>
overnight culture was inoculated in 500 ml l.BON/amp and grown under shaking at 25 °C until OD600<br>
values of 0.4-0.6. Protein expression was then induced by adding NaCl to a final concentration of 0.3 M.<br>
After 3 hours incubation at 25 °C the final OD600 was checked and the cultures were cooled on ice. After<br>
centrifugation at 6000 rpm (JA10 rotor, Beckman) for 20 min., the cell pellet was processed for<br>
purification or frozen at -20 °C.<br>
Proteins were purified in 1 of 3 ways depending on the fusion partner and the protein's solubility<br>
Purification of soluble His-tagged proteins from E.coli<br>
1. Transfer pellets from -20°C to ice bath and reconstitute each pellet with 10 ml B-PERTM solution<br>
(Bacterial-Protein Extraction Reagent, Pierce cat. 78266). 10 µl of a 100 mM MgCl2 solution, 50<br>
µl of DNAse I (Sigma D-4263, 100 Kunits in PBS) and 100 ul of 100 mg/ml lysozyme in PBS<br>
(Sigma l.-7651, final concentration 1 mg/ml).<br>
2. Transfer resuspended pellets in 50 ml centrifuge tubes and leave at room temperature for 30-40<br>
minutes, vortexing 3-4 times.<br>
3. Centrifuge 15-20 minutes at about 30-40000 xg.<br>
4. Prepare Poly-Prep (Bio-Rad) columns containing 1 ml of Fast Flow Ni-activated Chelating<br>
Sepharose (Pharmacia). Equilibrate with 50 mM phosphate buffer, 300 mM MaCl, pH 8.0.<br>
5. Store the pellet at -20°C, and load the supernatant on to the columns.<br>
6. Discard the flow through.<br>
7. Wash with 10 ml 20 mM imidazole buffer, 50 mM phosphate, 300 mM NaCl, pH 8.0.<br>
8. Elute the proteins bound to the columns with 4.5 ml (1.5 ml + 1.5 ml + 1.5 ml) 250 mM imidazole<br>
buffer, 50 mM phosphate, 300 mM NaCl, pH 8.0 and collect three fractions of ~ 1.5 ml each. Add<br>
to each tube 15 ul DTT 200 mM (final concentration 2 mM).<br>
9. Measure the protein concentration of the collected fractions with the Bradford method and analyse<br>
the proteins by SDS-PAGE.<br>
10. Store the collected fractions at +4°C while waiting for the results of the SDS-PAGE analysis.<br>
11. For immunisation prepare 4-5 aliquots of 20-100 µg each in 0.5 ml in 40% glycerol. The dilution<br>
buffer is the above elution buffer, plus 2 mM DTT. Store the aliquots at -20°C until immunisation.<br>
Purification of His-tagged proteins from inclusion bothes<br>
1. Bacteria are collected from 500 ml cultures by centrifugation. If required store bacterial pellets at<br>
-20°C. Transfer the pellets from -20°C to room temperature and reconstitute each pellet with 10<br>
ml B-PER™ solution, 10 ul of a 100 mM MgCU solution (final 1 mM), 50 ul of DNAse I<br>
equivalent to 100 Kimits units in PBS and 100 ul of a 100 mg/ml h/sozime (Sigma l.-7651) solution<br>
in PBS (equivalent to 10 mg, final concentration I mg/ml).<br>
2. Transfer the resuspended pellets in 50 ml centrifuge tubes and let at room temperature for 30-40<br>
minutes, vortexing 3-4 times.<br>
3. Centrifuge 15 minutes at 30-4000 x g and collect the pellets.<br>
4. Dissolve the pellets with 50 mM TRIS-HC1. 1 mM [CEP {Tris(2-carboxyethyl)-phosphine<br>
hydrochloride, Pierce} , 6M guanidine hydrochloride, pH S.5. Stir for ~ l.0 rain, with a magnetic<br>
bar.<br>
5. Centrifuge as described above, and collect the supernatant.<br>
6. Prepare Poly-Prep (Bio-Rad) columns containing 1 ml of Fast Flow Ni-activated Chelating<br>
Sepharose (Pharmacia). Wash the columns twice with 5 ml of H2O and equilibrate with 50 mM<br>
TRIS-HC1, 1 mM TCEP, 6M guanidine hydrochloride, pH 8.5<br>
7. l.oad the supematants from step 5 onto the columns, and wash with 5 ml of 50 mM TRIS-HC1<br>
buffer, 1 mM TCEP, 6M urea, pH 8.5<br>
8. Wash the columns with 10 ml of 20 mM imidazole, 50 mM TRIS-HC1 , 6M urea, 1 mM TCEP,<br>
pH 8.5. Collect and set aside the first 5 ml for possible further controls.<br>
9. Elute proteins bound to columns with 4.5ml buffer containing 250 mM imidazole, 50 mM TRIS-<br>
HC1, 6M urea, 1 mM TCEP, pH 8.5. Add the elution buffer in three 1.5 ml aliquots. and collect<br>
the corresponding three fractions. Add to each fraction 15 µl DTT (final concentration 2 mM).<br>
10. Measure eluted protein concentration with Bradford method and analyse proteins by SDS-PAGE.<br>
11. Dialyse overnight the selected fraction against 50 mM Na phosphate buffer, pH 8.8, containing<br>
10% glycerol, 0.5 M arginine, 5 mM reduced glutathione, 0.5 mM oxidized ghitathione, 2 M urea.<br>
12. Dialyse against 50 mM Na phosphate buffer, pH 8.8, containing 10% glycerol, 0.5 M arginine, 5<br>
mM reduced glutathione, 0.5 mM oxidized glutathione.<br>
13. Clarify the dialysed protein preparation by centrifugation and discard the non-soluble material and<br>
measure the protein concentration with the Bradford method.<br>
14. For each protein destined to the immunization prepare 4-5 aliquot of 20-100 µg each in 0.5 ml<br>
after having adjusted the glycerol content up to 40%. Store the prepared aliquots at -20° C until<br>
immunization.<br>
Purification of GST-fusion proteins from E.coli<br>
1. Bacteria are collected from 500 ml cultures by centrifugation. If required store bacterial pellets at<br>
-20°C. Transfer the pellets from -20°C to room temperature and reconstitute each pellet with 10<br>
ml B-PER™ solution, 10 ul of a 100 mM MgCl2 solution (final 1 mM), 50 ul of DNAse I<br>
equivalent to 100 Kunits units in PBS and 100 ul of a 100 mg/ml lysozime (Sigma l.-7651) solution<br>
in PBS (equivalent to 10 mg, final concentration 1 mg/ml)<br>
2. Transfer the resuspended pellets in 50 ml centrifuge tubes and let at room temperature for 30-40<br>
minutes, vortexing 3-4 times.<br>
3. Centrifuge 15-20 minutes at about 30-40000 x g.<br>
4. Discard centrifugation pellets and load supematants onto the chromatography columns, as<br>
follows.<br>
5. Prepare Poly-Prep (Bio-Rad) columns containing 0.5 ml of Glutaduone-Sepharose 4B resin. Wash<br>
the columns twice with 1 ml of H2O and equilibrate with l.0 ml PBS, pH 7.4.<br>
6. Load supematants on to the columns and discard the flow through.<br>
7. Wash the columns with 10 ml PBS, pH 7.4.<br>
8. Elute proteins bound to columns with 4.5 ml of 50 mM TR1S buffer, 10 mM reduced glutathione,<br>
pH 8.0, adding 1.5 ml + 1.5 ml + 1 5 ml and collecting the respective 3 fractions of ~1.5 ml each.<br>
9. Measure protein concentration of the fractions with the Bradford method and analyse the proteins<br>
by SDS-PAGE.<br>
10. Store the collected fractions at +4°C while waiting for the results of the SDS-PAGE analysis.<br>
11. For each protein destined for immunisation prepare 4-5 aliquots of 20-100 µg each in 0.5 ml of<br>
40% glycerol. The dilution buffer is 50 mM TRIS-HC1, 2 mM DTT, pH 8.0. Store the aliquots at<br>
-20°C until immunisation.<br>
Figures 167 to 170 and 238 to 239<br>
For the experiments shown in Figures 167 to 170, Figure 238 and lanes 2-6 of Figure 239, the GBS<br>
proteins were fused at the N-terminus to thioredoxin and at C-terminus to a poly-His tail. The plasmid<br>
used for cloning is pBAD-DEST49 (Invitrogen Gateway™ technology) and expression is under the<br>
control of an l.(+)-Arabinose dependent promoter. For the production of these GBS antigens, bacteria<br>
are grown on RM medium (6gA Na2HPO4, 3g/l KH2PO4, 0.5 g/1 NaCl, 1 g/1 NR.C1, pH7,4, 2%<br>
casaminoacids, 0.2 % glucose, 1 mM MgCl2) containing 100 µg/ml ampicillin. After incubation at 37°C<br>
until cells reach OD600=0.5, protein expression is induced by adding 0.2% (v/v) l.(+)Axabinose for 3<br>
hours.<br>
Immunisations with GBS proteins<br>
The purified proteins were used to immunise groups of four CD-I mice intraperitoneally. 20 ug of each<br>
purified protein was injected in Freund's adjuvant at days 1, 21 &amp; 35. Immune responses were<br>
monitored by using samples taken on day 0 &amp; 49. Sera were analysed as pools of sera from each group<br>
of mice.<br>
FACScan bacteria Binding Assay procedure.<br>
GBS serotype V 2603 V/R strain was plated on TSA blood agar plates and incubated overnight at 37°C.<br>
Bacterial colonies were collected from the plates using a sterile dracon swab and inoculated into 100ml<br>
Todd Hewitt Broth. Bacterial growth was monitored every 30 minutes by following OD600. Bacteria were<br>
grown until OD600 = 0.7-0.8. The culture was centrifuged for 20 minutes at 5000rpm. The supernatant<br>
was discarded and bacteria were washed once with PBS, resuspended in ½ culture volume of PBS<br>
containing 0.05% paraformaldehyde, and incubated for 1 hour at 37°C and then overnight at 4°C.<br>
50µl bacterial cells (OD600 0.1) were washed once with PBS and resuspended in 20µl blocking serum<br>
(Newborn Calf Serum, Sigma) and incubated for 20 minutes at room temperature. The cells were then<br>
incubated with 100µl diluted sera (1:200) in dilution buffer (20% Newborn Calf Serum 0.1% BSA in<br>
PBS) for 1 hour at 4°C. Cells were centrifuged at 5000rpm, the supernatant aspirated and cells washed<br>
by adding 200µl washing buffer (0.1% BSA in PBS). 50µl R-Phicoerytnn conjugated F(ab)2 goat anti-<br>
mouse, diluted 1:100 in dilution buffer, was added to each sample and incubated for 1 hour at 4°C. Cells<br>
were spun down by centrifugation at 5000rpm and washed by adding 200µl of washing buffer. The<br>
supernatant was aspirated and cells resuspended in 200µl PBS. Samples were transferred to FACScan<br>
tubes and read. The condition for FACScan setting were: Fl.2 on; FSC-H threshold:54; FSC PMT<br>
Voltage: E 02; SSC PMT: 516; Amp. Gains 2.63; Fl.-2 PMT: 728. Compensation values: 0.<br>
Samples were considered as positive if they had a A mean values &gt; 50 channel values.<br>
Whole Extracts preparation<br>
GBS serotype III COH1 strain and serotype V 2603 V/R strain cells were grown overnight in Todd<br>
Hewitt Broth, lml of the culture was inoculated into 100ml Todd Hewitt Broth. Bacterial growth was<br>
monitored every 30 minutes by following OD600. The bacteria were grown until the OD reached 0.7-0.8.<br>
The culture was centrifuged for 20 minutes at 5000 rpm. The supernatant was discarded and bacteria<br>
were washed once with PBS, resuspended in 2ml 50mM Tris-HCl, pH 6.8 adding 400 units of<br>
Mutanolysin (Sigma-Aldrich) and incubated 3 hrs at 37°C. After 3 cycles of freeze/thaw, cellular debris<br>
were removed by centrifugation at 14000g for 15 minutes and the protein concentration of the<br>
supernatant was measured by the Bio-Rad Protein assay, using BSA as a standard.<br>
Western blotting<br>
Purified proteins (50ng) and total cell extracts (25µg) derived from GBS serotype III COH1 strain and<br>
serotype V 2603 V/R strain were loaded on 12% or 15% SDS-PAGE and transferred to a nitrocellulose<br>
membrane. The transfer was performed for 1 hours at 100V at 4°C, in transferring buffer (25mM Tris<br>
base, 192mM glycine, 20% methanol). The membrane was saturated by overnight incubation at 4°C in<br>
saturation buffer (5 % skimmed milk, 0.1% Tween 20 in PBS). The membrane was incubated for 1 hour<br>
at room temperature with 1:1000 mouse sera diluted in saturation buffer. The membrane was washed<br>
twice with washing buffer (3 % skimmed milk, 0.1% Tween 20 in PBS) and incubated for 1 hour with a<br>
1:5000 dilution of horseradish peroxidase labelled anti-mouse Ig (Bio-Rad). The membrane was washed<br>
twice with 0.1% Tween 20 in PBS and. developed with the Opti-4CN Substrate Kit (Bio-Rad). The<br>
reaction was stopped by adding water.<br>
Unless otherwise indicated, lanes 1, 2 and 3 of blots in the drawings are: (1) the purified protein; (2)<br>
GBS-III extracts; and (3) GBS-V extracts. Molecular weight markers are also shown.<br>
In vivo passive protection assay in neonatal sepsis mouse model.<br>
The immune sera collected from the CDI immunized mice were tested in a mouse neonatal sepsis model<br>
to verify their protective efficacy in mice challenged with GBS serotype in. Newborn Balb/C littermates<br>
were randomly divided in two groups within 24 hrs from birth and injected subcutaneously with 25µl of<br>
diluted sera (1:15) from immunized CD1 adult mice. One group received preimmune sera, the other<br>
received immune sera. Four hours later all pups were challenged with a 75% lethal dose of the GBS<br>
serotype III COHl strain. The challenge dose obtained diluting a mid log phase culture was administered<br>
in 25 µl of saline. The number of pups surviving GBS infection was assessed every 12<br>
are in Table III.<br>
SEQ ID 2 (GBS4) was expressed in E.coli as a GST-fasion product. SDS-PAGE analysis of total cell<br>
extract is shown in Figure 9 (lane 3: MW 43.1kD)a) and Figure 63 (lane 4; MW 50kDa). It was also<br>
expressed in E.coli as a His-fusion product. SDS-PAGE analysis of total cell extract is shown in Figure 12<br>
(lane 7; MW 30kDa), Figure 63 (lane 3; MW 30kDa) and in Figure 178 (lane 3; MW 30kDa)<br>
GBS4-GST was purified as shown in Figure 190 (lane 6) and Figure 209 (lane 8).<br>
Purified GBS4-His is shown in Figures 89A, 191 (lane 10), 209 (lane 7) and 228 (lanes 9 &amp; 10).<br>
The purified GBS4-His fusion product was used to immunise mice (lane 2 product; 201g/mouse). The<br>
resulting antiserum was used for Western blot (Figure 89B), FACS, and in the in vivo passive protection<br>
assay (Table III). These tests confirm that the protein is immunoaccessible on GBS ba.cteria and that it is an<br>
effective protective immunogen.<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
SEQ ID 8712 (GBS166) was expressed in E.coli as a His-fusion product. SDS-PAGE analysis of total cell<br>
extract is shown in Figure 30 (lane 2; MW 13.1kDa).<br>
The GBS166-His fusion product was purified (Figure 200, lane 10) and used to immunise mice. The<br>
resulting antiserum was used for FACS (Figure 315), which confirmed that the protein is immunoaccessible<br>
on GBS bacteria.<br>
SEQ ID 4 (GBS 15) was expressed in E.coli as a GST-fusion product. SDS-PAGE analysis of total cell<br>
extract is shown in Figure 9 (lane 5; MW 44.8kDa), Figure 63 (lane 5; MW 44.8kDa) and Figure 66 (lane 7;<br>
MW 45kDa). It was also expressed in E.coli as a His-fusion product. SDS-PAGE analysis of total cell<br>
extract is shown in Figure 10 (lane 4; MW 22.3kDa). It was also expressed as GBS15l., with SDS-PAGE<br>
analysis of total cell extract is shown in Figure 185 (lane 1; MW 50kDa).<br>
Purified GBS15-GST is shown in Figure 91A, Figure 190 (lane 9), Figure 210 (lane 4) and Figure 245<br>
(lanes 4 &amp; 5).<br>
The purified GBS15-GST fusion product was used to immunise mice (lane 1+2 products; 20µg/mouse).<br>
The resulting antiserum was used for Western blot (Figure 91B), FACS (Figure 91C ), and in the in vivo<br>
passive protection assay (Table III). These tests confirm that the protein is immunoaccessible on GBS<br>
bacteria and that it is an effective protective immunogen.<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
SEQ ID 6 (GBS 103) was expressed in E.coli as a His-fusion product. SDS-PAGE analysis of total cell<br>
extract is shown in Figure 36 (lane 4; MW 32kDa).<br>
The GBS103-His fusion product was purified (Figure 107A; see also Figure 201, lane 9) and used to<br>
immunise mice (lane 2+3 product; 18.5µg/mouse). The resulting antiserum was used for Western blot<br>
(Figure 107B), FACS (Figure 107C ) and in the in vivo passive protection assay (Table III). These tests<br>
confirm that the protein is immunoaccessible on GBS bacteria and that it is an effective protective<br>
immunogen.<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
SEQ ID 8 (GBS 195) was expressed in E.coli as a His-fusion product. SDS-PAGb analysis of total cell<br>
extract is shown in Figure 24 (lane 8). It was also expressed in E.coli as a GST-fusion product. SDS-PAGE<br>
analysis of total cell extract is shown in Figure 31 (lane 5).<br>
GBS195C was expressed in E.coli as a GST-fusion product. SDS-PAGE analysis of total cell extract is<br>
shown tn Figure 175 (lane 6 &amp; 7; MW 8lkDa).<br>
GBS195L was expressed in E.coli as a His-fusion product. SDS-PAGE analysis of total cell extract is<br>
shown in Figure 83 (lane 2; MW 123kDa).<br>
GBS195l.N was expressed in E.coli as a His-fusion product SDS-PAGE analysis of total cell extract is<br>
shown in Figure 83 (lane 3; MW 66kDa).<br>
GBS195-GST was purified as shown in Figure 198, lane 5. GBS195-His was purified as shown in Figure<br>
222, lane 4-5. GBS195N-His was purified as shown in Figure 222, lane 6-7.<br>
The GBS195-GST fusion product was purified (Figure 87A) and used to immunise mice (lane 1 product;<br>
13.6µg/mouse). The resulting antiserum was used for Western blot (Figure 87B), FACS, and in the in vivo<br>
passive protection assay (Table III). These tests confirm that the protein is immunoaccessible on GBS<br>
bacteria and that it is an effective protective immunogen.<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 5<br>
A DNA sequence (GBSxG002) was identified in S.agatactiae <seq id> which encodes the amino<br>
acid sequence <seq id>. This protein is predicted to be lipoprotein MtsA. Analysis of this protein<br>
sequence reveals the following:<br>
SEQ ED 9404 (GBS679) was expressed in E.coli as a His-fusion product. SDS-PAGE analysis of total cell<br>
extract is shown in Figure 164 (lane 7-9; MW 36kDa) and in Figure 188 (lane 8; MW 36kDa). Purified<br>
protein is shown in Figure 242, lanes 9 &amp; 10.<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 6<br>
A DNA sequence (GBSx0003) was identified in S.agalactiae <seq id> which encodes the amino<br>
acid sequence <seq id>. This protein is predicted to be ATP-binding protein MtsB. Analysis of this<br>
protein sequence reveals the following:<br>
Example 7<br>
A DNA sequence (GBSx0004) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
SEQ ID 18 (GBS205) was expressed in E.coli as a His-fusion product. SDS-PAGE analysis of total cell<br>
extract is shown in Figure 51 (lane 13; MW 31kDa).<br>
GBS205-His was purified as shown in Figure 206, lane 8.<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 11<br>
A DNA sequence (GBSx0008) was identified in S.agalactiae <seq ii> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be sporulation protein SpoIIIE (ftsK). Analysis of this<br>
protein sequence reveals the following:<br>
A related GBS nucleic acid sequence <seq id> which encodes arnino acid sequence <seq id></seq>
10036&gt; was also identified.<br>
The protein has homology with the following sequences in the GENPEPI database:<br>
SEQ ID 22 (GBS272d) was expressed in E.coli as a GST-fusion product. SDS-PAGE analysis of total cell<br>
extract is shown in Figure 147 (lane 9; MW 55kDa + lane 10; MW 70kDa). It was also expressed in E.coli<br>
as a His-fusion product. SDS-PAGE analysis of total cell extract is shown in Figure 147 (lane 11 &amp; 13; MW<br>
85kDa + lane 12; MW 74kDa).<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 12<br>
A DNA sequence (GBSx0009) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be para-aminobenzoate synthetase (pabB) (pabB).<br>
Analysis of this protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 13<br>
A DNA sequence (GBSx00l0) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
No corresponding DNA sequence was identified in S.pyogenes.<br>
Based on this analysis, it was predicted that this protein and its epitopes, could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 17<br>
A DNA sequence (GBSx0014) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be drug transporter. Analysis of this protein sequence<br>
reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 18<br>
A DNA sequence (GBSx0015) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be transposase. Analysis of this protein sequence<br>
reveals the following:<br>
The protein has no significant homology with any sequences in the GENPEPT database<br>
No corresponding DNA sequence was identified in S.pyogenes.<br>
Based on this analysis, it was predicted that this protein and its epitopes, could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 19<br>
A DNA sequence (GBSx0016) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be L11 protein (rplK). Analysis of this protein<br>
sequence reveals the following:<br>
A related DNA sequence was identified in S.pyogenes <seq> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following.<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 20<br>
A DNA sequence (GBSx0017) was identified in S.agalactiae <seq ed> which encodes the ammo acid<br>
sequence <seq id>. 'This protein is predicted to be ribosomal protein L1 (rp1A). Analysis of this protein<br>
sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their cpitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 21<br>
A DNA sequence (GBSx0018) was identified in S.agalactiae <seq id> which encodes the arnino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
A related DNA sequence was identified in S.pyogenes <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
SEQ ID 8470 (GBS186) was expressed in E.coli as a His-fusion pruduct. SDS-PAGE analysis of total cell<br>
extract is shown in Figure 35 (lane 7; MW 60kDa). It was also expressed in E.coli as a GST-fusion<br>
product. SDS-PAGE analysis of total cell extract is shown in Figure 41 (lane 6; MW 85.7kDa).<br>
GBS186-GST was purified as shown in Figure 202, lane 4.<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 22<br>
A DNA sequence (GBSx0019) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 23<br>
A DNA sequence (GBSx0020) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be nickel transport system (permease). Analysis of this<br>
protein sequence reveals the following:<br>
A related GBS gene <seq id> and protein <seq id> were also identified. Analysis of this<br>
protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 24<br>
A DNA sequence (GBSx0021) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be peptide ABO transporter, ATP-binding protein.<br>
Analysis of this protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 25<br>
A DNA sequence (GBSx0022) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be peptide ABC transporter, ATP-binding protein.<br>
Analysis of this protein sequence reveals the following:<br>
Example 27<br>
A DNA sequence (GBSx0024) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
The protein has no significant homology with any sequences in the GENPEPT database.<br>
No corresponding DNA sequence was identified in S.pyogenes.<br>
Based on this analysis, it was predicted that this protein and its epitopes, could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 28<br>
A DNA sequence (GBSx0025) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be ribosome recycling factor (frr). Analysis of this<br>
protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 29<br>
A DNA sequence (GBSx0026) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 30<br>
A DNA sequence (GBSx0028) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be peptide methionine sulfoxidc reductase (msrA).<br>
Analysis of this protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 31<br>
A DNA sequence (GBSx0029) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 32<br>
A DNA sequence (GBSx0030) was identified in S.agalactiae <shq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be antigen. 67 kDa (myosin-crossreactive). Analysis<br>
of this protein sequence reveals the following:<br>
SEQ ID 8476 (GBS90) was expressed in E.coli as a His-fusion product. SDS-PAGE analysis of total cell<br>
extract is shown in Figure 18 (lane 6; MW 68.5kDa).<br>
The GBS90-His fusion product was purified (Figure 194, Jane 11) and used to immunise mice. The<br>
resulting antiserum was used for Western blot (Figure 256A), FACS (Figure 256B), and in the in vivo<br>
passive protection assay (Table III). These tests confirm that the protein is immunoaccessible on GBS<br>
bacteria and that it is an effective protective immunogen.<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 33<br>
A DNA sequence (GBSx0031) was identified in S.agalactiae <seq id> which encodes the ammo acid<br>
sequence <seq id>. This protein is predicted ro be phoh-like protein (phoH). Analysis of this protein<br>
sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 34<br>
A DNA sequence (GBSx0032) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
The protein has no significant homology with any sequences in the GENPEPT database.<br>
No corresponding DNA sequence was identified in S.pyogenes.<br>
Based on this analysis, it was predicted that this protein and its epitopes, could be useful antigens tor<br>
vaccines or diagnostics.<br>
Example 35<br>
A DNA sequence (GBSx0033) was identified in S.agalactiae <seq id> which encodes the ammo acid<br>
sequence <seq id>. This protein is predicted to be MutT/nudix family protein. Analysis of this protein<br>
sequence reveals the following:<br>
A related DNA sequence was identified in S.pyogenes <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 36<br>
A DNA sequence (GBSx0034) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
The protein has no significant homology with any sequences in the GENPEPT database<br>
No corresponding DNA sequence was identified in S.pyogenes.<br>
Based on this analysis, it was predicted that this protein and its epitopes, could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 37<br>
A DNA sequence (GBSx0035) was identified in S agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
No corresponding DNA sequence was identified in S.pyogenes.<br>
A related GBS gene <seq id> and protein <seq id> were also identified. Analysis of this<br>
protein sequence reveals the following:<br>
SEQ ID 8478 (GBS176) was expressed in E coli as a His-fusion product. SDS-PAGE analysis of total cell<br>
extract is shown in Figure 36 (lane 5 &amp; 6; MW 30kDa). It was also expressed in E.coli as a GST-fusion<br>
product. SDS-PAGE analysis of total cell extract is shown in Figure 41 (lane 7; MW 55.4kDa).<br>
The GBS176-GST fusion product was purified (Figure 117A; see also Figure 202, lane 5) and used to<br>
immunise mice (lane 1+2 product; 13.5µg/mouse). The resulting antiserum was used for Western blot<br>
(Figure 117B), FACS (Figure 117C), and in the in vivo passive protection assay (Table III). These tests<br>
confirm that the protein is immunoaccessible on GBS bacteria and that it is an effective protective<br>
immunogen.<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 38<br>
A DNA sequence (GBSx0036) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
A related DNA sequence was identified in S.pyogenes <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 39<br>
A DNA sequence (GBSx0038) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be phosphoglycerate dehydrogenase (serA) (serA).<br>
Analysis of this protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 40<br>
A DNA sequence (GBSx0039) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be alpha-glycerophosphate oxidase. Analysis of this<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens: for<br>
vaccines or diagnostics.<br>
Example 41<br>
A DNA sequence (GBSx0040) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 42<br>
A DNA sequence (GBSx0041) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be lipopolysaccharide core biosynthesis protein kdtB<br>
(kdtB). Analysis of this protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 43<br>
A DNA sequence (GBSx0042) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
The protein has no significant homology with any sequences in the GENPEPT database.<br>
No corresponding DNA sequence was identified in S.pyogenes.<br>
Based on this analysis, it was predicted that this protein and its epitopes, could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 44<br>
A DNA sequence (GBSx0043) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysts of this protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 45<br>
A DNA sequence (GBSx0044) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be UDP-sugar hydrolase. Analysis of this protein<br>
sequence reveals the following:<br>
No corresponding DNA sequence was identified in S.pyogenes.<br>
Based on this analysis, it was predicted that this protein and its epitopes, could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 46<br>
A DNA sequence (GBSx0045) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be UDP-sugar hydrolase. Analysis of this protein<br>
sequence reveals the following:<br>
No corresponding DNA sequence was identified in S.pyogenes.<br>
Based on this analysis, it was predicted that this protein and its epitopes, could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 47<br>
A DNA sequence (GBSx0046) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be unnamed protein product. Analysis of this protein<br>
sequence reveals the following:<br>
A related DNA sequence was identified in S.pyogenes <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 49<br>
A DNA sequence (GBSx0048) was identified in S.agalactiae <seq id> which encodes the ammo acid<br>
sequence <seq id>. This protein is predicted to be VanZF. Analysis of this protein sequence reveals<br>
the following:<br>
No corresponding DNA sequence was identified in S.pyogenes.<br>
Based on this analysis, it was predicted that this protein and its epitopes, could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 50<br>
A DNA sequence (GBSx0049) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be multidrug resistance-like ATP-binding protein rndl.<br>
Analysis of this protein sequence reveals the following:<br>
There is also homology to SEQ IDs 330, 4634 and 5788.<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 52<br>
A DNA sequence (GBSx0051) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 54<br>
A DNA sequence (GBSx0053) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br><br>
The protein has no significant homology with any sequences in the GENPEPT database.<br>
No corresponding DNA sequence was identified in S.pyogenes.<br>
Based on this analysis, it was predicted that this protein and its epitopes, could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 55<br>
A DNA sequence (GBSx0054) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br><br>
A related GBS nucleic acid sequence <seq id> which encodes amino acid sequence <seq id><br>
was also identified.<br>
The protein has homology with the following sequences in the GENPEPT database:<br><br><br>
No corresponding DNA sequence was identified in S.pyogenes.<br>
Based on this analysis, it was predicted that this protein and its epitopes, could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 56<br>
A DNA sequence (GBSx0055) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br><br>
A related GBS nucleic acid sequence <seq id> which encodes amino acid sequence <seq id><br>
was also identified.<br>
The protein has no significant homology with any sequences in the GENPEPT database.<br>
No corresponding DNA sequence was identified in S.pyogenes.<br>
Based on this analysis, it was predicted that this protein and its epitopes, could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 57<br>
A DNA sequence (GBSx0056) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br><br>
The protein has no significant homology with any sequences in the GENPEPT database.<br>
No corresponding DNA sequence was identified in S.pyogenes.<br>
Based on this analysis, it was predicted that this protein and its. epitopes, could be useful antigens tor<br>
vaccines or diagnostics.<br>
Example 58<br>
A DNA sequence (GBSx0057) was identified in S.agalactiae <seq id> which eacodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 59<br>
A DNA sequence (GBSx0058) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 60<br>
A DNA sequence (GBSx0059) was identified in S. agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be endonuclease III (pdg). Analysis of this protein<br>
sequence reveals the following:<br><br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 61<br>
A DNA sequence (GBSx0060) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br><br>
The protein has homology with the following sequences in the GENPEPT database:<br><br>
A related DNA sequence was identified in S.pyogenes <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br><br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 62<br>
A DNA sequence (GBSx0061) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
A related DNA sequence was identified in S.pyogenes <seq id> which encodes the amino acid<br>
sequence <seq ed>. Analysis of this protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 63<br>
A DNA sequence (GBSx0062) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
SEQ ID 8484 (GBS13) was expressed in E.coli as a His-fusion product. SDS-PAGE analysis of total cell<br>
extract is shown in Figure 3 (lane 4; MW 16kDa). It was also expressed in E.coli as a GST-fusion product.<br>
SDS-PAGE analysis of total cell extract is shown in Figure 9 (lane 2; MW 40.5kDa).<br>
The GST-fusion protein was purified as shown in Figure 190, lane 5.<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 64<br>
A DNA sequence (GBSx0063) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be regulatory protein TypA (typA). Analysis of this<br>
protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 65<br>
A DNA sequence (GBSx0065) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be D-glutarnic acid adding enzyme MurD (murD).<br>
Analysis of this protein sequence reveals the following:<br>
A related GBS nucleic acid sequence <seq id> which encodes amino acid sequence <seq id><br>
was also identified.<br>
The protein has homology with the following sequences in the GENPEPT database:<br>
SEQ ID 208 (GBS305) was expressed in E.coli as a His-fusion product. SDS-PAGE analysis of total cell<br>
extract is shown in Figure 51 (lane 11; MW 53.7kDa). Ft was also expressed in E.coli as a GST-fusion<br>
product. SDS-PAGE analysis of total cell extract is shown in Figure 56 (lane 3: MW 79kDa).<br>
The GBS305-GST fusion product was purified (Figure 207, lane 8) and used to immunise mice. The<br>
resulting antiserum was used for FACS (Figure 270), which confirmed that the protein is immunoaccessible<br>
on GBS bacteria.<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostic;;.<br>
Example 66<br>
A DNA sequence (GBSx0066) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
SEQ ID 212 (GBS306) was expressed in E.coli as a His-fusion produce. SDS-PAGE analysis of total cell<br>
extract is shown in Figure 51 (lane 12; MW 43kDa). It was also expressed in E.coli as a GST-fusion<br>
product SDS-PAGE analysis of total cell extract is shown in Figure 56 (lane 4; MW 68kDa).<br>
GBS306-GST was purified as shown in Figure 207. lane 9.<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 67<br>
A DNA sequence (GBSx0067) was identified in S.agalactiae <seq id> which encodes the ammo acid<br>
sequence <seq id>. This protein is predicted to be cell division protein DivIB. Analysis of this protein<br>
sequence reveals the following:<br>
SEQ ID 216 (GBS85) was expressed in E.coli as a His-fusion product. SDS-PAGE analysis of total cell<br>
extract is shown in Figure 17 (lane 10; MW 45.2kDa).<br>
The GBS85-His fusion product was purified (Figure 105A; see also Figure 193, lane 5) and used to<br>
immunise mice (lane 1 product; 20µg/mouse). The resulting antiserum was used for Western blot (Figure<br>
105B), FACS (Figure 105C), and in the in vivo passive protection assay (Table III). These tests confirm<br>
that the protein is immunoaccessible on GBS bacteria and that it is an effective protective immunogen.<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 68<br>
A DNA sequence (GBSx0068) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be cell division protein FtsA (ftsA). Analysis of this<br>
protein sequence reveals the following:<br>
SEQ ID 220 (GBS73) was expressed in E.coli as a His-fusion product. SDS-PAGE analysis of total cell<br>
extract is shown in Figure 17 (lane 5; MW 47.8kDa). It was also expressed in E.coli as a GST-fusion<br>
product SDS-PAGE analysis of total cell extract is shown in Figure 20 (lane 5; MW 70.1kDa).<br>
GBS73-GST was purified as shown in Figure 197, lane 7.<br>
The GBS73-His fusion product was purified (Figure 103A) and used to immunise mice (lane 1 product;<br>
20µg/mouse). The resulting antiserum was used for Western blot (Figure 103B), FACS (Figure 103C ) and<br>
in the in vivo passive protection assay (Table III). These tests confirm that the protein is immunoaccessible<br>
on GBS bacteria and that it is an effective protective immunogen.<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 69<br>
A DNA sequence (GBSx0069) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq ed>. This protein is predicted to be cell division protein FtsZ (ftsz). Analysis of this<br>
protein sequence reveals the following:<br><br>
SEQ ID 224 (GBS163) was expressed in E.coli as a His-fusion product. SDS-PAGE analysis of total cell<br>
extract is shown in Figure 28 (lane 7; MW 44kDa). It was also expressed in E.coli as a GST-fusion<br>
product. SDS-PAGE analysis of total cell extract is shown in Figure 34 (lane 4; MW 69kDa).<br>
The GBS163-GST fusion product was purified (Figure 114A; see also Figure 198, lane 11) and used to<br>
immunise mice (lane 1 product; 20µg/mouse). The resulting antiserum was used for Western blot (Figure<br>
114B), FACS and in the in vivo passive protection assay (Table III). These tests confirm that the protein is<br>
iminunoaccessible on GBS bacteria and that it is an effective protective immunogen.<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 70<br>
A DNA sequence (GBSx0070) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br><br><br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 71<br>
A DNA sequence (GBSx0071) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be YlmF. Analysis of this protein sequence reveals<br>
the following:<br><br>
Based on this analysis, it was predicted that these proteins and their epitopcs could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 72<br>
A DNA sequence (GBSx0072) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be YlmH. Analysis of this protein sequence reveals<br>
the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 73<br>
A DNA sequence (GBSx0073) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be cell division protein DivIVA (septumplacement).<br>
Analysis of this protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 74<br>
A DNA sequence (GBSx0074) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 75<br>
A DNA sequence (GBSx0075) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 76<br>
A DNA sequence (GBSx0076) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be AP4A hydrolase. Analysis of this protein sequence<br>
reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 77<br>
A DNA sequence (GBSx0077) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be ClpE (clpB-1). Analysis of this protein sequence<br>
reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 78<br>
A DNA sequence (GBSx0078) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be glutamine ABC transporter, permease protein<br>
(glnP). Analysis of this protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopcs could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 79<br>
A DNA sequence (GBSx0079) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq ed>. This protein is predicted to be phosphomannomutase (manB). Analysis of this<br>
protein sequence reveals the following:<br>
A related DNA sequence was identified in S.pyogenes <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 81<br>
A DNA sequence (GBSx0081) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
SEQ ID 272 (GBS413) was expressed in E.coli as a His-fusion product SDS-PAGE analysis of total cell<br>
extract is shown in Figure 79 (lane 2; MW 34.2kDa). It was also expressed in E.coli as a GST-fusion<br>
product SDS-PAGE analysis of total cell extract is shown in Figure 171 (lane 7; MW 59kDa).<br>
GBS413-GST was purified as shown in Figure 218, lane 12.<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 82<br>
A DNA sequence (GBSx0082) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be Exonuclease VII large subunit (xseA). Analysis of<br>
this protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 83<br>
A DNA sequence (GBSx0083) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
SEQ ID 286 (GBS310) was expressed in E.coli as a His-fusion product. SDS-PAGE analysis of total cell<br>
extract is shown in Figure 57 (lane 3; MW 34kDa). It was also expressed in E.coli as a GST-fusion<br>
product. SDS-PAGE analysis of total cell extract is shown in Figure 61 (lane 4; MW S8.8kDa).<br>
The GBS310-GST fusion product was purified (Figure 210, lane 10) and used to immunise mice. The<br>
resulting antiserum was used for FACS (Figure 282), which confirmed that the protein is immunoaccessible<br>
on GBS bacteria.<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 86<br>
A DNA sequence (GBSx0086) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br><br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 87<br>
A DNA sequence (GBSx0088) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br><br>
The protein has no significant homology with any sequences in the GENPEPT database.<br>
No corresponding DNA sequence was identified in S.pyogenes.<br>
Based on this analysis, it was predicted that this protein and its epitopes, could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 88<br>
A DNA sequence (GBSx0089) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be DNA repair protein recn (recN). Analysis of this<br>
protein sequence reveals the following:<br><br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 89<br>
A DNA sequence (GBSx0090) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be degV protein. Analysis of this protein sequence<br>
reveals the following:<br>
A related DNA sequence was identified in S.pyogenes <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
SEQ ID 300 (GBS113) was expressed in E.coli as a His-fusion product. Purified protein is shown in Figure<br>
201, lane 8.<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 90<br>
A DNA sequence (GBSx0092) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
A related GBS gene <seq id> and protein <seq id> were also identified. Analysis of this<br>
protein sequence reveals the following:<br>
SEQ ID 308 (GBS20) was expressed in E.coli as a His-fusion product SDS-PAGE analysis of total cell<br>
extract is shown in Figure 4 (lane 5; MW 25kDa) and in Figure 167 0ane 12-14; MW 37kDa - thioredoxin<br>
fusion). It was also expressed in E.coli as a GST-fusion product. SDS-PAGE analysis of total cell extract is<br>
shown in Figure 9 (lane 7; MW 47.6kDa). Purified Thio-GBS20-His is shown in Figure 244, lane 12.<br>
Example 91<br>
A DNA sequence (GBSx0093) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be histone-like DNA-binding protein. Analysis of this<br>
protein sequence reveals the following:<br>
SEQ ID 9294 (GBS663) was expressed in E.coli as a GST-fusion product. SDS-PAGE analysis of total cell<br>
extract is shown in Figure 137 (lane 3; MW 89.5kDa). It was also expressed in E.coli as a His-fusion<br>
product. SDS-PAGE analysis of total cell extract is shown in Figure 137 (lane 5-7; MW 64.5kDa), in Figure<br>
179 (lane 11; MW 65kDa) and in Figure 65 (lane 2; MW 61kDa) Purified GBS663-His is shown in Figure<br>
231, lane 3-4. Purified GBS324-His is shown in lane 6 of Figure 210.<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 93<br>
A DNA sequence (GBSx0095) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be transmembrane protein OppB (oppB). Analysis of<br>
this protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 95<br>
A DNA sequence (GBSx0097) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be ATPase OppD (oppD). Analysis of this protein<br>
sequence reveals the following:<br>
There is also homology to SEQ ID 72.<br>
SEQ ID 326 (GBS375) was expressed in E.coli as a His-fusion product SDS-PAGE analysis of total cell<br>
extract is shown in Figure 64 (lane 9; MW 42kDa). It was also expressed in E.coli as a GST-fusion<br>
product SDS-PAGE analysis of total cell extract is shown in Figure 71 (lane 3; MW 67kDa).<br>
GBS375-GST was purified as shown in Figure 215, lane 10.<br>
Based on this analysis, it was predicted that this protein and its epitopes, could be useful antigens for<br>
vaccines or diagnostics.<br><br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 97<br>
A repeated DNA sequence (GBSx0099) was identified in S.agalactiae <seq id> which encodes the<br>
amino acid sequence <seq id>. Analysis of this protein sequence reveals the following:<br><br>
The protein has no significant homology with any sequences in the GENPEPT database.<br>
No corresponding DNA sequence was identified in S.pyogenes.<br>
Based on this analysis, it was predicted that this protein and its epitopes, could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 98<br>
A repeated DNA sequence (GBSx0l00) was identified in S.agalactiae <seq id> which encodes the<br>
amino acid sequence <seq id>. Analysis of this protein sequence reveals the following:<br><br>
The protein has no significant homology with any sequences in the GENPEPT database.<br>
No corresponding DNA sequence was identified in S.pyogenes.<br>
Based on this analysis, it was predicted that this protein and its epitopes, could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 99<br>
A repeated DNA sequence (GBSxOlOl) was identified in S.agalactiae <seq id> which encodes the<br>
amino acid sequence <seq id>. Analysis of this protein sequence reveals the following:<br><br>
The protein has no significant homology with any sequences in the GENPEPT database.<br>
No corresponding DNA sequence was identified in S.pyogenes.<br>
Based on this analysis, it was predicted that this protein and its epitopes, could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 100<br>
A repeated DNA sequence (GBSx0103) was identified in S.agalactiae <seq id> which encodes the<br>
amino acid sequence <seq id>. Analysis of this protein sequence reveals the following:<br><br>
The protein has no significant homology with any sequences in the GENPEPT database.<br>
No corresponding DNA sequence was identified in S.pyogenes.<br>
Based on this analysis, it was predicted that this protein and its epitopes, could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 101<br>
A repeated DNA sequence (GBSx0104) was identified in S.agalactiae <seq id> which encodes the<br>
amino acid sequence <seq id>. Analysis of this protein sequence reveals the following:<br><br>
The protein has no significant homology with any sequences in the GENPEPT database.<br>
No corresponding DNA sequence was identified in S.pyogenes.<br>
Based on this analysis, it was predicted that this protein and its epitopes, could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 102<br>
A repeated DNA sequence (GBSx0105) was identified in S.agalactiae <seq id> which encodes the<br>
amino acid sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
The protein has no significant homology with any sequences in the GENPEPT database.<br>
No corresponding DNA sequence was identified in S.pyogenes.<br>
Based on this analysis, it was predicted that this protein and its epitopes, could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 103<br>
A repeated DNA sequence (GBSx0106) was identified in S.agalactiae <seq id> which encodes the<br>
amino acid sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
The protein has do significant homology with any sequences in the GENPEPT database.<br>
No corresponding DNA sequence was identified in S.pyogenes.<br>
Based on this analysis, it was predicted that this protein and its epitopes, could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 104<br>
A repeated DNA sequence (GBSx0107) was identified in S.agalactiae <seq id> which encodes the<br>
amino acid sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
No corresponding DNA sequence was identified in S.pyogenes.<br>
Based on this analysis, it was predicted that this protein and its epitopes, could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 105<br>
A DNA sequence (GBSx0108) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
Based on tins analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 106<br>
A DNA sequence (GBSx0109) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be AdcR protein. Analysis of this protein sequence<br>
reveals the following:<br>
Based cm this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 107<br>
A DNA sequence (GBSx0110) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be AdcC protein. Analysis of this protein sequence<br>
reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 108<br>
A DNA sequence (GBSx0111) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
SEQ ID 374 (GBS64d) was expressed in E.coli as a GST-fusion product. SDS-PAGE analysis of total cell<br>
extract is shown in Figure 120 (lane 2-4; MW 107kDa). It was also expressed in E.coli as a His-fusion<br>
product. SDS-PAGE analysis of total cell extract is shown in Figure 120 (lane 5-7; MW 82kDa) and in<br>
Figure 179 (lane 2; MW 82kDa).<br>
GBS64d-His was purified as shown in Figure 231, lane 7-8.<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 113<br>
A DNA sequence (GBSx0116) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be DNA-dependent RNA polymerase subunit beta<br>
(rpoB). Analysis of this protein sequence reveals the following:<br>
Example 114<br>
A DNA sequence (GBSxO118) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be DNA-directed RNA polymerase, beta subunit<br>
(rpoC). Analysis of this protein sequence reveals the following:<br>
A related DNA sequence was identified in S.pyogenes <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
SEQ ID 390 (GBS63) was expressed in E.coli as a His-fusion product. SDS-PAGE analysis of total cell<br>
extract is shown in Figure 5 (lane 5; MW 39kDa). It was also expressed in E.coli as a GST-fusion product.<br>
SDS-PAGE analysis of total cell extract is shown in Figure 13 (lane 2; MW 64kDa).<br>
The GBS63-GST fusion product was purified (Figure 101A; see also Figure 191, lane 3) and used to<br>
immunise mice (lane 1 product; 20µg/mouse). The resulting antiserum was used for Western blot (Figure<br>
101B), FACS (Figure 101C ), and in the in vivo passive protection assay (Table III). These tests confirm<br>
that the protein is immunoaccessible on GBS bacteria and that it is an effective protective immunogen.<br>
SEQ ID 8494 (GBS49) was expressed in E.coli as a His-fusion product SDS-PAGE analysis of total cell<br>
extract is shown in Figure 11 (lane 3; MW 15kDa). It was also was expressed in E.coli as a GST-fusion<br>
product. SDS-PAGE analysis of total cell extract is shown in Figure 15 (lane 5; MW 60kDa).<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 118<br>
A DNA sequence (GBSx0123) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be Com YD or ComGD. Analysis of this protein<br>
sequence reveals the following:<br>
SEQ ID 398 (GBS6) was expressed in E.coli as a GST-fusion product SDS-PAGE analysis of total cell<br>
extract is shown in Figure 1 (lane 2; MW 40kDa). It was also expressed in E.coli as a His-fusion product.<br>
SDS-PAGE analysis of total cell extract is shown in Figure 2 (lane 2; MW 15kDa). The GBS6-GST fusion<br>
product was purified (Figure 189, lane 2) and used to immunise mice. The resulting antiserum was used for<br>
FACS (Figure 260), which confirmed that the protein is immunoaccessible on GBS bacteria.<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 120<br>
A DNA sequence (GBSx0125) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be acetate kinase (ackA-1). Analysis of this protein<br>
sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 121<br>
A DNA sequence (GBSx0126) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be repressor protein. Analysis of this protein sequence<br>
reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 122<br>
A DNA sequence (GBSx0127) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
Based on tins analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 12S<br>
A DNA sequence (GBSx0130) was identified in S.agalactiae <seq id> which encodes tire amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
The protein has no significant homology with any sequences in the GENPEPT database.<br>
No corresponding DNA sequence was identified in S.pyogenes.<br>
Based on this analysis, it was predicted that this protein and its epitopes, could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 126<br>
A DNA sequence (GBSx0131) was identified in S.agalactiae <seq id> which encodes the ammo acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
SEQ ID 8498 (GBS214) was expressed in E.coli as a His-fusion product. SDS-PAGE analysis of total cell<br>
extract is shown in Figure 40 (lane 3; MW 13.9kDa). It was also expressed in E.coli as a GST-fusion<br>
product. SDS-PAGE analysis of total cell extract is shown in Figure 46 (lane 6; MW 39kDa).<br>
Based on this analysis, it was predicted that these proteins and their epitopcs could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 127<br>
A DNA sequence (GBSx0132) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be thioredoxin H1 (trxA). Analysis of this protein<br>
sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 128<br>
A DNA sequence (GBSxO133) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be phenylalanyl-tRNA synthetase beta subunit, non-<br>
spirochete. Analysis of this protein sequence reveals the following:<br>
No corresponding DNA sequence was identified in S.pyogenes.<br>
Based on this analysis, it was predicted that this protein and its epitopes, could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 130<br>
A DNA sequence (GBSx0136) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 131<br>
A DNA sequence (GBSx0137) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
The protein has no significant homology with any sequences in the GENPEPT database.<br>
No corresponding DNA sequence was identified in S-pyagenes.<br>
Based on this analysis, it was predicted that this protein and its epitopes, could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 134<br>
A DNA sequence (GBSx0140) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
SEQ ID 454 (GBS248d) was expressed in E.coli as a GST-fusion product SDS-PAGE analysis of total cell<br>
extract is shown in Figure 124 (lane 2-4; MW 71kDa). It was also expressed in E.coli as a His-fusion<br>
product. SDS-PAGE analysis of total cell extract is shown in Figure 124 (lane 5-7; MW 46kDa) and in<br>
Figure 180 (lane 2; MW 46kDa).<br>
GBS248d-His was purified as shown in Figure 234, lane 3-4.<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 135<br>
A DNA sequence (GBSx014l) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. This protein is predicted to be two-component response regulator (lytT). Analysis<br>
of this protein sequence reveals the following:<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 136<br>
A DNA sequence (GBSx0142) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
The protein has no significant homology with any sequences in the GENPEPT database.<br>
No corresponding DMA. sequence was identified in S.pyogenes.<br>
Based on this analysis, it was predicted that this protein and its epitopes, could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 137<br>
A DNA sequence (GBSx0143) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
SEQ ID 8502 (GBS106) was expressed in E.coli as a His-fusion product. SDS-PAGE analysis of total cell<br>
extract is shown in Figure 18 (lane 3; MW 
The GBS106-His fusion product was purified (Figure 194, lane 2) and used to immunise mice. The<br>
resulting antiserum was used for Western blot (Figure 255A), FACS (Figure 255B), and in the in vivo<br>
passive protection assay (Table III). These tests confirm that the protein is immunoaccessible on GBS<br>
bacteria and that it is an effective protective immunogen.<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 140<br>
A DNA sequence (GBSx0146) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
The protein has no significant homology with any sequences in the GENPEPT database.<br>
No corresponding DNA sequence was identified in S.pyogenes.<br>
Based on this analysis, it was predicted that this protein and its epitopes, could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 141<br>
A DNA sequence (GBSx0147) was identified in S.agalactiae <seq id> which encodes the amino acid<br>
sequence <seq id>. Analysis of this protein sequence reveals the following:<br>
SEQ ID 8472 (GBS436) was expressed in E.coli as a GST-fusion product SDS-PAGE analysis of total cell<br>
extract is shown in Figure 173 (lane 9; MW 54kDa).<br>
Based on this analysis, it was predicted that these proteins and their epitopes could be useful antigens for<br>
vaccines or diagnostics.<br>
Example 142<br>
A DNA sequence (GBSx0148) was identified in S.agalactiae <seq id> which encodes the amino acid<br></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></shq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></seq></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI2NDU5LUZPUk0gMjcucGRm" target="_blank" style="word-wrap:break-word;">226459-FORM 27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDYxLUtPTE5QLTIwMDMtKDA0LTEwLTIwMTIpLUZPUk0tMjcucGRm" target="_blank" style="word-wrap:break-word;">461-KOLNP-2003-(04-10-2012)-FORM-27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDYxLUtPTE5QLTIwMDMtRk9STS0yNy0xLjEucGRm" target="_blank" style="word-wrap:break-word;">461-KOLNP-2003-FORM-27-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDYxLUtPTE5QLTIwMDMtRk9STS0yNy0xLnBkZg==" target="_blank" style="word-wrap:break-word;">461-KOLNP-2003-FORM-27-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDYxLUtPTE5QLTIwMDMtRk9STS0yNy5wZGY=" target="_blank" style="word-wrap:break-word;">461-KOLNP-2003-FORM-27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDYxLWtvbG5wLTIwMDMtZ3JhbnRlZC1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">461-kolnp-2003-granted-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDYxLWtvbG5wLTIwMDMtZ3JhbnRlZC1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">461-kolnp-2003-granted-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDYxLWtvbG5wLTIwMDMtZ3JhbnRlZC1jb3JyZXNwb25kZW5jZS5wZGY=" target="_blank" style="word-wrap:break-word;">461-kolnp-2003-granted-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDYxLWtvbG5wLTIwMDMtZ3JhbnRlZC1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">461-kolnp-2003-granted-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDYxLWtvbG5wLTIwMDMtZ3JhbnRlZC1leGFtaW5hdGlvbiByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">461-kolnp-2003-granted-examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDYxLWtvbG5wLTIwMDMtZ3JhbnRlZC1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">461-kolnp-2003-granted-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDYxLWtvbG5wLTIwMDMtZ3JhbnRlZC1mb3JtIDE4LnBkZg==" target="_blank" style="word-wrap:break-word;">461-kolnp-2003-granted-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDYxLWtvbG5wLTIwMDMtZ3JhbnRlZC1mb3JtIDIucGRm" target="_blank" style="word-wrap:break-word;">461-kolnp-2003-granted-form 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDYxLWtvbG5wLTIwMDMtZ3JhbnRlZC1mb3JtIDI2LnBkZg==" target="_blank" style="word-wrap:break-word;">461-kolnp-2003-granted-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDYxLWtvbG5wLTIwMDMtZ3JhbnRlZC1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">461-kolnp-2003-granted-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDYxLWtvbG5wLTIwMDMtZ3JhbnRlZC1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">461-kolnp-2003-granted-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDYxLWtvbG5wLTIwMDMtZ3JhbnRlZC1yZXBseSB0byBleGFtaW5hdGlvbiByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">461-kolnp-2003-granted-reply to examination report.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="226458-high-efficiency-switching-power-amplifier.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="226460-a-method-of-continuous-casting-of-molten-steel-for-sheet-metal.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>226459</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>461/KOLNP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>51/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>19-Dec-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>17-Dec-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>16-Apr-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CHIRON S.P.A.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>VIA FIORENTIMA 1,53100 SIENA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>TELFORD JHON</td>
											<td>C/O CHIRON SPA, OF VIA FIORENTIMA 1,53100 SIENA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>MASIGNANI VEGA</td>
											<td>C/O CHIRON SPA, OF VIA FIORENTIMA 1,53100 SIENA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MARGARIT Y ROS IMMACULADA</td>
											<td>C/O CHIRON SPA, OF VIA FIORENTIMA 1,53100 SIENA</td>
										</tr>
										<tr>
											<td>4</td>
											<td>GRANDI GUIDO</td>
											<td>C/O CHIRON SPA, OF VIA FIORENTIMA 1,53100 SIENA</td>
										</tr>
										<tr>
											<td>5</td>
											<td>FRASER CLAIRE</td>
											<td>C/O THE INSTITUTE FOR GENOMIC RESEARCH 9712 MEDICAL CENTER DRIVE, ROCKVILLE, MARYLAND 20850</td>
										</tr>
										<tr>
											<td>6</td>
											<td>TETTELIN HERVE</td>
											<td>C/O THE INSTITUTE FOR GENOMIC RESEARCH 9712 MEDICAL CENTER DRIVE, ROCKVILLE, MARYLAND 20850</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 14/195</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/GB01/04789</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-10-29</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0105640.7</td>
									<td>2001-03-07</td>
								    <td>U.K.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>0026333.5</td>
									<td>2000-10-27</td>
								    <td>U.K.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>0028727.6</td>
									<td>2000-11-24</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/226459-a-protein-from-streptococcus-groups-a-and-b by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 04:06:55 GMT -->
</html>
