<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/226736-a-method-for-reducing-the-expression-of-a-gene-of-interest-in-a-plant-cell by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:01:56 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 226736:A METHOD FOR REDUCING THE EXPRESSION OF A GENE OF INTEREST IN A PLANT CELL</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR REDUCING THE EXPRESSION OF A GENE OF INTEREST IN A PLANT CELL</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention provides method for reducing the expression of a gene of interest in a plant cell. The method comprises providing a chimeric gene to the plant cell in which the chimeric gene comprises a DNA dependant RNA polymerase III of type 3 of a plant cell and a DNA fragment which when transcribed yields a short hairpin RNA.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>A METHOD FOR REDUCING THE EXPRESSION OF A GENE OF<br>
INTEREST IN A PLANT CELL<br>
FIELD OF THE INVENTION<br>
The current invention relates generally to the field of genetic modification of plants, more<br>
particularly to the use of short double stranded (dsRNA) sequences to deliberately<br>
silence the expression of one or more genes in plant cells and plants. Methods and means<br>
are provided to increase the efficiency of gene silencing when using dsRNA sequences<br>
which have a stem length shorther than about 200 basepairs.<br>
BACKGROUND<br>
The mechanism of posttranscriptional silencing of gene expression in plants and animals<br>
triggered by target-gene specific dsRNA, provided either exogenously or endogenously<br>
through transcription of dsRNA encoding chimeric genes, has recently become the subject<br>
of numerous studies. Since the initial description of this phenomenon in animals and<br>
plants (Fire et al., 1998; Hamilton et al., 1998; Waterhouse et al., 1998), it has become dear<br>
that the dsRNA is processed by an RNAse with a preference for dsRNA (such as DICER<br>
in Drosophila) into short, approximately 21 nucleotide long RNA molecules that are used<br>
as guide sequences, providing sequence-specificity to a complex capable of degrading<br>
specific mRNA molecules.<br>
The high specificity and efficiency of gene silencing initiated by dsRNA that is<br>
homologous to the gene to be silenced rapidly turned this methodology into the preferred<br>
tool to generate eukaryotic organisms wherein expression of one or more specific<br>
transcribed nucleotide sequences is reduced or inactivated. Such reduction or inactivation<br>
of the expression of a gene of interest may be achieved with a goal to produce eukaryotic<br>
organisms with a preferred phenotype (see e.g. WO 02/029028, wherein Brassica plants<br>
are generated which develop sepals instead petals using dsRNA technology). Reduction<br>
or inactivation of expression of transcribed sequences also plays an important role in<br>
experimental studies trying to allocate a function to the wealth of nucleotide sequences<br>
which have become available through various genome sequencing programs.<br><br>
Particularly for the latter, it may be advantageous to use short dsRNA sequences, since<br>
such oligonucleotides may conveniently be generated in vitro. In higher animals, the use<br>
of short dsRNA molecules is preferred in view of the fact that larger dsRNA molecules<br>
seem to trigger interferon responses (Elbashir et al. 2001).<br>
Up to now, the production of inhibitory RNA (used herein to describe antisense RNA,<br>
sense RNA and dsRNA) inside the cells of eukaryotic orgiinisms, mostly occurs through<br>
the action of DNA dependent RNA polymerase II (PolII) recognizing the common PolII<br>
type promoters.<br>
Antisense RNA production through the action of RNA polymerase HI in plants has been<br>
documented.<br>
Bourque and Folk (1992) described suppression of the expression of a CAT gene,<br>
transiently delivered to plant cells, by co-electroporation with a DNA comprising inverted<br>
sequences of the chloramphenicol actetyltransferase reporter gene, fused to a soybean<br>
tRNAmet gene lacking a terminator, such that the tRNAmet sequences caused the<br>
transcription of CAT antisense sequences by RNA polymerase III.<br>
US 5,354,854 describes an expression system and method to use the same in plants to<br>
suppress gene expression, the system including a constitutive promoter element from a<br>
tRNA gene and an antisense strand DNA fused to the promoter element for being co-<br>
transcribed with the promoter element by an RNA polymerase HI to suppress expression<br>
of a gene.<br>
Yukawa et al. 2002 described antisense RNA sequences targeted against conserved<br>
structural elements or domains in the RNAs of potato spindle tuber viroid, hop latent<br>
viroid and potato virus S which were embedded in the anticodon region or a Nicotiana<br>
tRNAtyr gene or near the 3' end of an Arabidopsis 7SL RNA gene, and demonstrated in<br>
vitro transcription of such chimeric genes in a homologous plant extract.<br>
EP 0 387 775 describes and claims a DNA molecule, optionally occuring in multiple<br>
copies, containing sections of a gene transcribed by polymerase III and a DNA sequence<br>
encoding for an inhibiting RNA molecule, characterized in that it contains the<br><br>
transcription units of a tRNA gene necessary for transcription by polymerase III,<br>
including the sequence which determine the secondary structure of the tRNA, and that<br>
the DNA sequence coding for the inhibiting RNA molecule is arranged inside the DNA<br>
molecule in such a way that the inhibiting RNA molecule is a part of the transcript.<br>
Expression of small interfering RNAs in mammalian cells has recently been well<br>
documented. Paddison et al. 2002; Sook Lee et al. 2002; Myagishi et al. 2002, Sui et al.<br>
2002; Brummelkamp et al, 2002 and Paul et al. 2002, all describe the expression of small<br>
interfering RNA in human or mammalian cells using RNA polymerase III specific<br>
promoters derived from either H1-RNA or U6 snRNA.<br>
US 6,146,886 describes and claims a transcribed non-naturally occuring RNA molecule<br>
comprising a desired RNA portion, wherein said non-naturally occuring RNA molecule<br>
comprises an intramolecular stem formed by base-pairing interactions between a 3' region<br>
and 5' complementary nucleotides in said RNA, wherein said intramolecular stem<br>
comprises at least 8 basepairs; wherein said desired RNA portion is selected from the<br>
group consisting of antisense RNA, decoy RNA, enzymatic RNA, agonist RNA and<br>
antagonist RNA, wherin said RNA molecule is transcribed by a type 2 RNA polymerase<br>
HI promoter system.<br>
The prior art remains however deficient in providing methods for highly efficient<br>
expression of small interfering dsRNAs in plant cells. This problem has been solved as<br>
hereinafter described.<br>
SUMMARY OF THE INVENTION<br>
The invention provides methods for reducing the expression of a gene of interest in a<br>
plant cell, comprising the following steps:<br>
(a) providing a chimeric gene to the plant cell, the chimeric gene comprising the<br>
following operably linked DNA fragments:<br>
i) a promoter recognized by a DNA dependent RNA polymerase HI of the plant<br>
cell characterized in that the promoter is a promoter of type HI (type 3)<br>
comprising all cis-acting promoter elements which interact with the DNA<br>
dependent RNA polymerase HI, preferably a type 3 POLIII promoter selected<br><br>
from the promoter of a gene encoding U6snRNA , the promoter of a gene<br>
encoding U3snRNA, the promoter of a gene encoding 7SL RNA, more<br>
preferably a promoter comprising the nucleotide sequence of promoter is<br>
selected from the nucleotide sequences of SEQ ID Nr 1, SEQ ID Nr 2, SEQ ID<br>
Nr 3, SEQ ID Nr 4, SEQ ID Nr 5, SEQ ID Nr 6, SEQ ID Nr 7 or SEQ ID Nr 8;<br>
ii) a DNA fragment which, when transcribed, yields an RNA molecule, the RNA<br>
molecule comprising a sense and antisense nucleotide sequence,<br>
(1)	the sense nucleotide sequence comprising about 19 contiguous nucleotides<br>
having about 90 to about 100% sequence identity to a nucleotide sequence<br>
of about 19 contiguous nucleotide sequences from the RNA transcribed<br>
from the gene of interest;<br>
(2)	the antisense nucleotide sequence comprising about 19 contiguous<br>
nudeotides having about 90 to 100% sequence identity to the complement<br>
of a nudeotide sequence of about 19 contiguous nucleotide sequence of the<br>
sense sequence;<br>
wherein the sense and antisense nudeotide sequence are capable of<br>
forming a double stranded RNA of about 19 to about 200 nudeotides in<br>
length; and<br>
iii) an oligo dT stretch comprising at least 4 consecutive T-residues; and<br>
(b) identifying plant cells wherein the expression of the gene of interest is reduced<br>
when compared to the expression of the gene of interest in plant cells which do<br>
not comprise the chimeric gene.<br>
The invention further provides a chimeric gene comprising the following operably<br>
linked DNA fragments:<br>
i) a promoter recognized by a DNA dependent RNA polymerase III of the plant<br>
cell characterized in that the promoter is a promoter of type III comprising all<br>
ds-acting promoter elements which interact with the DNA dependent RNA<br>
polymerase El, preferably a type 3 POLIII promoters selected from the<br>
promoter of a gene encoding U6snRNA , the promoter of a gene encoding<br>
U3snRNA, the promoter of a gene encoding 7SL RNA, more preferably a<br>
promoter comprising the nudeotide sequence of promoter is selected from the<br>
nudeotide sequences of SEQ ID Nr 1, SEQ ID Nr 2, SEQ ID Nr 3, SEQ ID Nr 4,<br>
SEQ ID Nr 5, SEQ ID Nr 6, SEQ ID Nr 7 or SEQ ID Nr 8;<br><br>
ii) a DNA fragment which, when transcribed, yields an RNA molecule, the RNA<br>
molecule comprising a sense and antisense nucleoti.de sequence,<br>
(1)	the sense nucleotide sequence comprising about 19 contiguous nucleotides<br>
having about 90 to about 100% sequence identity to a nucleotide sequence<br>
of about 19 contiguous nucleotide sequences from the RNA transcribed<br>
from a gene of interest in a plant cell;<br>
(2)	the antisense nucleotide sequence comprising about 19 contiguous<br>
nucleotides having about 90 to 100% sequence identity to the complement<br>
of a nucleotide sequence of about 19 contiguous nucleotide sequence of the<br>
sense sequence;<br>
wherein the sense and antisense nucleotide sequence are capable of<br>
forming a double stranded RNA of about 19 to about 200 nucleotides in<br>
length; and<br>
iii) an oligo dT stretch comprising at least 4 consecutive T-residues<br>
The invention further provides plant cell and plants comprising the above mentioned<br>
chimeric genes<br>
BRIEF DESCRIPTION OF THE FIGURE<br>
Figure 1 outlines schematically a convenient cloning strategy for creating and handling a<br>
coding region encoding short dsRNA sequences.<br>
DETAILED DESCRIPTION<br>
The current invention is based on the observation that chimeric genes encoding short<br>
dsRNA molecules, preferably ranging between about 20 basepairs (bp) and about 100 bp<br>
under control of type 3 promoters recognized by RNA polymerase III, resulted in more<br>
efficient gene silencing than similar constructs driven by the strong constitutive RNA<br>
Polymerase II promoter CaMV 35S.<br>
These type 3 promoters have the additional advantage that all required cis-acting<br>
elements of the promoter are located in the region upstream of the transcribed region, in<br><br>
contrast to type 2 promoters recognized by RNA polymerase III, which had been used in<br>
the prior art to direct expression of antisense RNA.<br>
Thus, in a first embodiment, the current invention relates to a method for reducing the<br>
expression of a gene of interest in a plant cell, comprising the following steps:<br>
(a)	providing a chimeric gene to the plant cell, the chimeric gene comprising the<br>
following operably linked DNA fragments :<br>
i) a promoter recognized by a DNA dependent RNA polymerase III of the plant<br>
cell whereby the promoter is a promoter of type 3 comprising all as-acting<br>
promoter elements which interact with DNA dependent RNA polymerase III;<br>
ii) a DNA fragment which, when transcribed, yields an RNA molecule, the RNA<br>
molecule comprising a sense and antisense nucleotide sequence, and wherein<br>
(1)	the sense nucleotide sequence comprises about 19 contiguous nudeotides<br>
having about 90 to about 100% sequence identity to a nucleotide sequence<br>
of about 19 contiguous nucleotide sequences from the RNA transcribed<br>
from the gene of interest;<br>
(2)	the antisense nucleotide sequence comprising about 19 contiguous<br>
nucleotides having about 90 to 100% sequence identity to the complement<br>
of a nucleotide sequence of about 19 contiguous nucleotide sequence of the<br>
sense sequence;<br>
wherein the sense and antisense nucleotide sequence are capable of<br>
forming a double stranded RNA of about 19 to about 200 nucleotides in<br>
length; and<br>
iii) an oligo dT stretch comprising at least 4 consecutive T-residues; and<br>
(b)	identifying plant cells wherein the expression of the gene of interest is reduced<br>
when compared to the expression of the gene of interest in plant cells which do<br>
not comprise the chimeric gene.<br>
As used herein « a promoter recognized by the DNA dependent RNA polymerase III» is<br>
a promoter which directs transcription of the associated DNA region through the<br>
polymerase action of RNA polymerase III. These include genes encoding 5S RNA, tRNA,<br>
7SL RNA, U6 snRNA and a few other small stable RNAs, many involved in RNA<br>
processing. Most of the promoters used by Pol III require sequence elements downstream<br>
of +1, within the transcribed region. A minority of pol III templates however, lack any<br><br>
requirement for intragenic promoter elements. These are referred to as type 3 promoters.<br>
In other words, «type 3 Pol III promoters », are those promoters which are recognized by<br>
RNA polymerase III and contain all as-acting elements, interacting with the RNA<br>
polymerase III upstream of the region normally transcribed by RNA polymerase III. Such<br>
type 3 Pol III promoters can thus easily be combined in a chimeric gene with a<br>
heterologous region, the transcription of which is desired, such as the dsRNA coding<br>
regions of the current invention.<br>
Typically, type 3 Pol III promoters contain a TATA box (located between -25 and -30 in<br>
Human U6 snRNA gene) and a Proximal Sequence element (PSE; located between —47<br>
and -66 in Human U6 snRNA). They may also contain a Distal Sequence Element (DSE;<br>
located between -214 and -244 in Human U6 snRNA).<br>
Type 3 Pol III promoters can be found e.g. associated with the genes encoding 7SL RNA,<br>
U3 snRNA and U6 snRNA. Such sequences have been isolated from Arabidopsis, rice and<br>
tomato and representative sequences of such promoters are represented in the sequence<br>
listing under the entries SEQ ID No 1-8.<br>
Other nucleotide sequences for type 3 Pol HI promoters can be found in nucleotide<br>
sequence databases under the entries for the A. thaliana gene AT7SL-1 for 7SL RNA<br>
(X72228), A. thaliana gene AT7SL-2 for 7SL RNA (X72229), A. thaliana gene AT7SL-3 for<br>
7SL RNA (AJ290403), Humulus luptdus H17SL-1 gene (AJ236706), Humulus lupulus<br>
H17SL-2 gene (AJ236704), Humulus lupulus H17SL-3 gene (AJ236705), Humulus lupulus<br>
H17SL-4 gene (AJ236703), A. thaliana U6-1 snRNA gene (X52527), A. thaliana U6-26<br>
snRNA gene (X52528), A. thaliana U6-29 snRNA gene (X52529), A. thaliana U6-1 snRNA<br>
gene (X52527), Zea mays U3 snRNA gene (Z29641), Solanum tuberosum U6 snRNA gene<br>
(Z17301; X 60506; S83742), Tomato U6 smal nuclear RNA gene (X51447), A. thaliana U3C<br>
snRNA gene (X52630), A. thaliana U3B snRNA gene (X52629), Oxyza sativa U3 snRNA<br>
promoter (X79685), Tomato U3 smal nuclear RNA gene (X14411), Triticum aestivum U3<br>
snRNA gene (X63065), Triticum aestivum U6 snRNA gene (X63066).<br>
It goes without saying that variant type 3 Pol III promoters may be isolated from other<br>
varieties of tomato, rice or Arabidopsis, or from other plant species without little<br>
experimentation. E.g. libraries of genomic clones from such plants may be isolated using<br><br>
U6 snRNA, U3 snRNA or 7SL RNA coding sequences (such as the coding sequences of<br>
any of the above mentioned sequences identified by their accession number and<br>
additionally the Vicia alba U6snRNA coding sequence (X04788), the maize DMA for U6<br>
snRNA (X52315) or the maize DNA for 7SL RNA (X14661)) as a probe, and the upstream<br>
sequences, preferably the about 300 to 400 bp upstream of the transcribed regions may be<br>
isolated and used as type 3 Pol III promoters. Alternatively, PCR based techniques such as<br>
inverse-PCR or TAIL®-PCR may be used to isolate the genomic sequences including the<br>
promoter sequences adjacent to known transcribed regions;. Moreover, any of the type 3<br>
Po1III promoter sequences attached or of the above mentioned promoter sequences,<br>
identified by their accession numbers, may be used as probes under stringent<br>
hybridization conditions or as source of information to generate PCR primers to isolate<br>
the corresponding promoter sequences from other varieties or plant species.<br>
"Stringent hybridization conditions" as used herein mean that hybridization will<br>
generally occur if there is at least 95% and preferably at least 97% sequence identity<br>
between the probe and the target sequence. Examples of stringent hybridization<br>
conditions are overnight incubation in a solution comprising 50% formamide, 5 x SSC (150<br>
mM NaCl, 15 mM trisodium citrate), 50 mM sodium phosphate (pH 7.6), 5x Denhardfs<br>
solution, 10% dextran sulfate, and 20 µg/ml denatured, sheared carrier DNA such as<br>
salmon sperm DNA, followed by washing the hybridization support in 0.1 x SSC at<br>
approximately 65 °C. Other hybridization and wash conditions are well known and are<br>
exemplified in Sambrook et al. Molecular Cloning: A Laboratory Manual, Second Edition,<br>
Cold Spring Harbor, NY (1989), particularly chapter 11.<br>
Although the type 3 Pol HI promoters have no requirement for cis-acting elements located<br>
with the transcribed region, it is clear that sequences normally located downstream of the<br>
transcription initiation site may nevertheless be included in the chimeric constructs of the<br>
invention.<br>
It has also been observed that type 3 Pol III promoters originally isolated from<br>
monocotyeldonous plants can be used to good effect in both dicotyledonous and<br>
monocotyleodous plant cells and plants, whereas type 3 Pol III promoters originally<br>
isolated from dicotyledonous plants can only be efficiently used in dicotyledonous plant<br><br>
cells and plants. Moreover, the most efficient gene silencing has been obtained when<br>
chimeric genes were used comprising a type 3 Pol III promoter derived from the same or<br>
closely related species.<br>
As used herein, a «gene of interest» may be any nucleic acid of interest, which is<br>
transcribed (or replicated) into an RNA molecule, and which is prone to post<br>
transcriptional RNA degradation. These include but are not limited to transgenes,<br>
endogenous genes and transcribed viral sequences. It will also be immediately apparent<br>
that for the methods of the invention, it is not required to have knowledge of the<br>
nucleotide sequence: of the gene of interest. Indeed, it may be possible to directly derive<br>
small fragments and operably link them in inverted repeat orientation, under control of a<br>
type 3 Pol III promoter<br>
As indicated above, the transcribed DNA region should be capable of encoding an RNA<br>
molecule comprising a sense and antisense nucleotide region, whereby the sense<br>
nucleotide sequence comprises about 19 contiguous nucleotides having about 90 to about<br>
100% sequence identity to a nucleotide sequence of about 19 contiguous nucleotide<br>
sequences from the RNA transcribed from the gene of interest and whereby the antisense<br>
nucleotide sequence comprising about 19 contiguous nucleotides having about 90 to 100%<br>
sequence identity to the complement of a nucleotide sequence of about 19 contiguous<br>
nucleotide sequence of the sense sequence. The sense and antisense nucleotide sequence<br>
should be capable of forming a double stranded RNA of about 19 to about 200<br>
nucleotides, particularly about 21 to about 90 or 100 nucleotides, more particularly about<br>
40 to about 50 nucleotides in length. However, the length of the dsRNA stem may also be<br>
about 30, about 60, about 70 or about 80 nucleotides in length. It will be clear that where<br>
the dsRNA region is larger than 19 nucleotides, there is only a requirement that there is at<br>
least one double stranded region of about 19 nucleotides (whereby there can be about one<br>
mismatch between the sense and antisense region) the sense strand of which is<br>
«identical» (allowing for one mismatch) with 19 consecutive nucleotides of the target<br>
nucleic acid or gene of interest.<br>
For the purpose of this invention, the "sequence identity" of two related nucleotide<br>
sequences, expressed as a percentage, refers to the number of positions in the two<br>
optimally aligned sequences which have identical residues (x100) divided by the number<br><br>
of positions compared. A gap, i.e. a position in an alignment: where a residue is present in<br>
one sequence but not in the other is regarded as a position with non-identical residues.<br>
The alignment of the two sequences is performed by tine Needleman and Wunsch<br>
algorithm (Needleman and Wunsch 1970) Computer-assisted sequence alignment, can be<br>
conveniently performed using standard software program such as GAP which is part of<br>
the Wisconsin Package Version 10.1 (Genetics Computer Group, Madison, Wisconsin,<br>
USA) using the default scoring matrix with a gap creation penalty of 50 and a gap<br>
extension penalty of 3.<br>
The transcribed DNA region may comprise a stretch of nudeotides ranging from 3 to<br>
about 100 nudeotides or more spedfically from about 6 to about 40 nudeotides, which are<br>
located between the sense and antisense encoding nudeotide region, and which are not<br>
related to the nudeotide sequence of the target gene (a so-called spacer region).<br>
The diimeric genes of the current invention, comprising a transcribed DNA region with<br>
short antisense and sense fragments may conveniently be: constructed using a stuffer<br>
DNA sequence between the short antisense and sense fragments during the cloning<br>
procedures, which may thereafter be removed. To that end, the stuffer segment may be<br>
equipped with restriction enzymes recognitions sites, sudi as rare-cutting restriction<br>
enzymes for the easy removal of the stuffer sequence and re-ligation (self-ligation) of the<br>
doning vector, whereby the short sense and antisense region are now brought in vicinity<br>
of each other. As outlined in Figure 1, a DNA fragment comprising a short sense<br>
sequence, a short, antisense sequence complementary to the sense sequence, and a stuffer<br>
DNA sequence may be conveniently construct by PCR amplification using<br>
oligonudeotide primers comprising the sense or antisense sequence and a sequence<br>
corresponding to part of the stuffer DNA sequence.<br>
The above mentioned « oligo dT stretch » is a stretch of consecutive T-residues which<br>
serve as a terminator for the RNA polymerase III activity. It should comprise at least 4 T-<br>
residues, but obviously may contain more T-residues.<br>
Chimeric genes according to the invention may be provided to plant cells by introduction<br>
into plant cells using any means of DNA transformation available in the art, induding but<br>
not limited to Agrobacterium mediated transformation, microprojectile bombardment,<br><br>
direct DNA uptake into protoplasts or plant tissues (by eleetroporation, PEG-mediated<br>
uptake, etc.) and may result in transiently or stably transformed plant cells. The chimeric<br>
genes may also be provided to the plant cells using viral vectors, capable of replicating in<br>
plant cells. Chimeric genes may also be provided to plant cells by crossing parental plants,<br>
at least one of which comprises a chimeric gene according to the invention.<br>
As used herein, « reducing the expression of a gene of interest » refers to the comparison<br>
of the expression of the gene of interest in the plant cell in the presence of the dsRNA or<br>
chimeric genes of the invention, to the expression of the gene of interest in the absence of<br>
the dsRNA or chimeric genes of the invention. The expression in the presence of the<br>
chimeric RNA of the invention should thus be lower than the expression in absence<br>
thereof, e.g. be only about 75% or 50% or 10% or about 5% of the expression in absence of<br>
the chimeric RNA. The expression may be completely inhibited for all practical purposes<br>
by the presence of the chimeric RNA or the chimeric gene encoding such an RNA.<br>
A reduction of expression of a gene of interest may be measured as a reduction in<br>
transcription of (part of) that gene, a reduction in translation of (part of) that gene or a<br>
reduction in the effect the presence of the transcribed RNA(s) or translated polypeptide(s)<br>
have on the plant cell or the plant, and will ultimately lead to altered phenotypic traits. It<br>
is clear that the reduction in expression of a gene of interest, may be accompanied by or<br>
correlated to an increase in expression of another gene. Although the main effect of<br>
dsRNA is the post-transcriptional degradation of specific RNAs, effects of dsRNA on the<br>
transcription process have been documented. Such additional effects will also contribute<br>
to the reduction of expression of a gene of interest mediated by dsRNA.<br>
Other embodiments of the invention relate to the chimeric genes as herein described, as<br>
well as to plants, plant cells, plant tissues or seeds comprising the chimeric genes of the<br>
invention.<br>
It is also an object of the invention to provide plant cells and plants containing the<br>
chimeric genes according to the invention. Gametes, seeds, embryos, either zygotic or<br>
somatic, progeny or hybrids of plants comprising the chimeric genes of the present<br>
invention, which are produced by traditional breeding methods are also included within<br>
the scope of the present invention.<br><br>
The methods and means described herein are believed to be suitable for all plant cells and<br>
plants, both dicotyledonous and monocotyledonous pliant cells and plants including but<br>
not limited to cotton, Brassica vegetables, oilseed rape, wheat, corn or maize, barley,<br>
sunflowers, rice, oats, sugarcane, soybean, vegetables (including chicory, lettuce, tomato),<br>
tobacco, potato, sugarbeet, papaya, pineapple, mango, Arabidopsis thaliana, but also<br>
plants used in horticulture, floriculture or forestry.<br>
The following non-limiting Examples describe the construction of chimeric genes for the<br>
reduction of the expression of a gene of interest in a plant cell by small dsRNA and the<br>
use of such genes.<br>
Unless stated otherwise in the Examples, all recombinamt DNA techniques are carried out<br>
according to standard protocols as described in Sambrook et al. (1989) Molecular Cloning:<br>
A Laboratory Manual, Second Edition, Cold Spring Harbor Laboratory Press, NY and in<br>
Volumes 1 and 2 of Ausubel et al. (1994) Current Protocols in Molecular Biology, Current<br>
Protocols, USA. Standard materials and methods for plant molecular work are described<br>
in Plant Molecular Biology Labfax (1993) by R.D.D. Croy, jointly published by BIOS<br>
Scientific Publications Ltd (UK) and Blackwell Scientific Publications, UK. Other<br>
references for standard molecular biology techniques include Sambrook and Russell<br>
(2001) Molecular Cloning: A Laboratory Manual, Third Edition, Cold Spring Harbor<br>
Laboratory Press, NY, Volumes I and E of Brown (1998) Molecular Biology LabFax,<br>
Second Edition, Academic Press (UK). Standard materials and methods for polymerase<br>
chain reactions can be found in Dieffenbach and Dveksler (1995) PCR Primer: A<br>
Laboratory Manual, Cold Spring Harbor Laboratory Press, and in McPherson at al. (2000)<br>
PCR - Basics: From Background to Bench, First Edition, Springer Verlag, Germany.<br>
Throughout the description and Examples, reference is made to the following sequences:<br>
SEQ ID No. 1: sequence of the promoter of the 7SL-2 gene of Arabidopsis thaliana var.<br>
Landsberg erecta, followed by a unique restriction site in front of an oligo<br>
dT stretch.<br>
SEQ ID No. 2: sequence of the promoter of the 7SL-2 gene of Arabidopsis thaliana var.<br>
Landsberg erecta including 86 bases downstream of the transcription<br><br>
initiation site, followed by a unique restriction site in front of an oligo dT<br>
stretch.<br>
SEQ ID No. 3: sequence of the promoter of the USB snRNA of Arabidopsis thaliana var.<br>
Landsberg erecta, followed by a unique restriction site in front of an oligo<br>
dT stretch.<br>
SEQ ID No. 4: sequence of the promoter of the U3B snRNA gene of Arabidopsis thaliana<br>
var. Landsberg erecta including 136 bases downstream of the<br>
transcription initiation site, followed by a unique restriction site in front<br>
of an oligo dT stretch.<br>
SEQ ID No. 5: sequence of the promoter of the U6-26 snRNA gene of Arabidopsis thaliana<br>
var. Landsberg erecta including 3 bases downstream of the transcription<br>
initiation site, followed by a unique restriction site in front of an oligo dT<br>
stretch.<br>
SEQ ID No. 6: sequence of the promoter of the U6-26 snRNA gene of Arabidopsis thaliana<br>
var. Landsberg erecta including 20 bases downstream of the transcription<br>
initiation site, followed by a unique restriction site in front of an oligo dT<br>
stretch.<br>
SEQ ID No. 7: sequence of the promoter of the U3 snRNA of rice (Oryza sativa Indica<br>
ER36), followed by a unique restriction site in front of an oligo dT stretch.<br>
SEQ ID No. 8: sequence of the promoter of the U3 snRMA of tomato (a garden variety<br>
with small gourd-shaped yellow fruit), followed by a unique restriction<br>
site in front of an oligo dT stretch,<br>
SEQ ID No. 9: sequence of the dsRNA encoding region of 94bp for silencing expression of<br>
the GUS gene (GUShp94).<br>
SEQ ID No. 10: sequence of the dsRNA encoding region of 41 bp for silencing expression<br>
of the GUS gene (GUShp41).<br>
SEQ ID No. 11: sequence of the dsRNA encoding region of 21 bp for silencing expression<br>
of the GUS gene (GUShp21).<br>
SEQ ID No. 12: sequence of the dsRNA encoding region of 42 bp for silencing expression<br>
of the PHYB gene, derived from the 5' end of PHYB (PHYB5hp42)-upper<br>
strand.<br>
SEQ ID No. 13: sequence of the dsRNA encoding region of 42 bp for silencing expression<br>
of the PHYB gene, derived from the 5' end of PHYB (PHYB5hp42)-Iower<br>
strand.<br><br>
SEQ ID No. 14: sequence of the dsRNA encoding region of 21 bp for silencing expression<br>
of the PHYB gene, derived from the 5' end of PHYB (PHYB5hp21)-upper<br>
strand.<br>
SEQ ID No. 15: sequence of the dsRNA encoding region, of 21 bp for silencing expression<br>
of the PHYB gene, derived from the 5' end of PHYB (PHYB5hp21)-lower<br>
strand.<br>
SEQ ID No. 16: sequence of the dsRNA encoding region of 42 bp for silencing expression<br>
of the PHYB gene, derived from the center of PHYB (PHYBChp42)-upper<br>
strand.<br>
SEQ ID No. 17: sequence of the dsRNA encoding region of 42 bp for silencing expression<br>
of the PHYB gene, derived from the carter of PHYB (PHYBChp42)-lower<br>
strand.<br>
SEQ ID No. 18: sequence of the dsRNA encoding region of 21 bp for silencing expression<br>
of the PHYB gene, derived from the center of PHYB (PHYBChp21)-upper<br>
strand.<br>
SEQ ID No. 19: sequence of the dsRNA encoding region of 21 bp for silencing expression<br>
of the PHYB gene, derived from the center of PHYB (PHYBChp21)-lower<br>
strand.<br>
SEQ ID No. 20: sequence of the dsRNA encoding region of 42 bp for silencing expression<br>
of the PHYB gene, derived from the 3' end of PHYB (PHYB3hp42)-upper<br>
strand.<br>
SEQ ID No. 21: sequence of the dsRNA encoding region of 42 bp for silencing expression<br>
of the PHYB gene, derived from the 3' end of PHYB (PHYB3hp42)-lower<br>
strand.<br>
SEQ ID No. 22: sequence of the dsRNA encoding region of 21 bp for silencing expression<br>
of the PHYB gene, derived from the 3' end of PHYB (PHYB3hp21)-upper<br>
strand.<br>
SEQ ID No. 23: sequence of the dsRNA encoding region of 21 bp for silencing expression<br>
of the PHYB gene, derived from the 3' end of PHYB (PHYB3hp21)-lower<br>
strand.<br>
SEQ ID No. 24: sequence of the dsRNA encoding region of 42 bp for silencing expression<br>
of the PDS gene (PDS42)-upper strand .<br>
SEQ ID No. 25: sequence of the dsRNA encoding region of 42 bp for silencing expression<br>
of the PDS gene (PDS42)-lower strand.<br><br>
SEQ ID No. 26: sequence of the dsRNA encoding region of 21 bp for silencing expression<br>
of the PDS gene (PDS21)-upper strand.<br>
SEQ ID No. 27: sequence of the dsRNA encoding region of 21 bp for silencing expression<br>
of the PDS gene (PDS21)-lower strand.<br>
SEQ ID No. 28: sequence of a dsRNA encoding region of 42 bp for silencing expression of<br>
a GUS gene(GUS-A)<br>
SEQ ID No. 29: sequence of a dsRNA encoding region of 42 bp for silencing expression of<br>
a GUS gene(GUS-B).<br>
SEQ ID No. 30: sequence of a dsRNA encoding region of 42 .bp for silencing expression of<br>
a GUS gene(GUS-C).<br>
SEQ ID No. 31: sequence of a dsENA encoding region of 42 bp for silencing expression of<br>
EIN (EIN-A).<br>
SEQ ID No. 32: sequence of a dsRNA encoding region of 42 bp for silencing expression of<br>
EIN (EIN-B).<br>
SEQ ID No. 33: sequence of a dsRNA encoding region of 42 bp for silencing expression of<br>
EIN(EIN-C).<br><br>
EXAMPLES<br>
Example 1. Construction of type 3 Pol III promoter-oligodT stretch cassettes.<br>
Type 3 Pol III promoters were isolated from Arabidopsis, rice or tomato 7SL,<br>
U3snRNA or U6snRNA genes using PCR amplification, designed in such a way<br>
that<br>
a)	the resulting fragments were flanked by restriction enzyme recognition sites<br>
not present within the amplified fragment;<br>
b)	the promoter fragments were followed by a unique restriction site (Sall, Xhol<br>
or Pvul), followed by<br>
c)	a poly(T) sequence (with 7-9 T residues) as Pol. III terminator.<br>
In some of the cloned promoter fragments, additional sequences of the coding region<br>
downstream of the transcription initiation site were included to investigate the possible<br>
effect of conserved motifs in the coding region of the small RNAs on transcription and/or<br>
gene silencing. The resulting fragments (represented in SEQ IDs No 1 to 8) were cloned in<br>
intermediate cloning vectors (see Table 1). Sense, antisense or inverted repeat sequences<br>
can readily be inserted in the unique restriction site between the type 3 Pol HI promoters<br>
and the polyT stretch.<br><br><br>
**This number represents the sequence from the coding region of the small RNA gene.<br>
***The sizes given include the restriction sites and the oligo (dT)s added to the PCR<br>
primers.<br>
Example 2. Testing of the PolIII promoters in gene silencing constructs against a GUS<br>
reporter gene (Nicotians tabacum).<br>
To test these Point promoters for silencing, a GUS inve:rted-repeat sequence (SEQ ID No<br>
9) was synthesized, which consists of 186 bp sense sequence of GUS (nt. 690-875 of GUS<br>
coding sequence) fused at the 3' end with an antisense version of the first 94 bp in the 186<br>
bp fragment (nt. 690-783 of GUS coding sequence). This i/r sequence is flanked by two<br>
Sail sites and two Pvul sites, and can therefore be cloned into the PolIII promoter vectors<br>
as a Sail or Pvul fragment. Constructs were prepared with all the PolIII promoters<br>
described in Table 1 using the i/rGUS sequence (GUShp94) (see Table 2). In addition to<br>
the GUShp94 sequence, constructs were also prepared with the AtU3B+136 promoter<br><br>
(SEQ ID No 4.) and the CaMV35S promoter using smaller i/r GUS sequences such<br>
GUShp41a (41 bp in the stem spaced by a 9 bp non-GUS sequence; SEQ ID No 10) and<br>
GUShp21 (21 bp in the stem spaced by a 6 bp non-GUS sequence, SEQ ID No 11) (Table<br>
2),<br><br>
These constructs were introduced to binary vectors pART27 or pWBVec4a for plant<br>
transformation. Two different transgenic tobacco lines expressing GUS, were transformed<br>
by all these constructs. A control construct in which the GUShp94 sequence was driven by<br>
a 35S promoter (in pART7) was also included.<br>
Leaf tissue from transformed tobacco plantlets on rooting medium was assayed for GUS<br>
activity (fluorometric MUG assay) and the results are summarized in Table 3.<br>
The results show that the GUShp94 constructs with AtU3 (pMBW468), At7SL (pMBW470),<br>
At7SL+86 (pMBW472), AtU3+3 (pMBW476), AtU3+20 (pMBW477) and TomU3<br>
(pLMW64) promoters all activated silencing of the GUS gene in tobacco. The AtU3,<br>
AtU6+20, and TomU3 constructs appeared to perform better than the others. The<br><br>
AtU3+136 construct (pMBW466) did not seem to give significant GUS silencing in<br>
tobacco. Also, the OsU3 construct (pMBW473) appeared to confer only a low level of GUS<br>
silencing. The PoM promoter construct pLMW58 (AtU3+136-GUShp41a) gave significant<br>
levels of GUS silencing in tobacco whereas the 35S consteuct pLMW53 (35S-GUShp41a)<br>
did not, suggesting that the PolDI promoters are more effective than the PolII promoters<br>
in driving the expression of small hairpin RNA.<br><br><br><br><br>
Table 4. Summary of constructs tested in Arabidopsis<br><br>
Leaf tissues from T1 plants that showed high-levels of resistance to the selective agent<br>
PPT were assayed for GUS activity. The MUG assay data are summarized in Table 5.<br>
For the GUShp94 sequence all the U3 and U6 promoter-driven constructs conferred GUS<br>
silencing, although the TomU3 and AtU6+20 gave more consistent and better silencing..<br>
The two At7SL promoter constructs did not appear to confer significant GUS silencing<br>
although a few lines showed moderate silencing, which may be due to T-DNA insertion<br>
next to endogenous promoters.<br>
With the GUShp41 sequence, the AtU3+136 construct: performed better than the 35S<br>
construct in terms of the degree of GUS silencing, again suggesting that PolIII promoters<br>
are more effective than PolII promoters for driving expression of small hairpin RNA<br>
expression in plants.<br><br><br>
Example 4. Testing of the PolIII promoters in gene silencing constructs against a GUS<br>
reporter gene (Ozyza sativa).<br>
The constructs pMBW479, pMBW481, pMBW485, pMBVV486 and pLMW62 (see Table 4)<br>
were super-transformed into rice that expresses a Ubil-GUS-nos gene. GUS staining<br>
showed that only pMBW485 (OsU3-GUShp94) and pMBW479 (35S-GUShp94) conferred<br>
significant silencing to the resident GUS gene. These results indicate that dicotyldonous<br>
type 3 PolIII promoters will not function in monocots.<br><br>
Example 5. Testing of the PolIII promoters in gene silencing constructs against<br>
Arabidopsis endogenous genes.<br>
The Arabidopsis U6-26 construct contains the promoter from -446 to +3 bp (SEQ ID 5)<br>
and additional sequences added by PCR creating Xhol sites at each end of the fragment.<br>
These were used to clone the PCR product into the Sail site of a pGEM derived plasmid.<br>
The insert was excised with NotI and inserted into the pART27 binary vector for plant<br>
transformation. The PCR also incorporated a Sail site between the promoter and<br>
termination sequences (T8) for insertion of oligonucleotide sequences.<br>
Two genes were targeted, phytoene desaturase (PDS - silencing gives a photobleached<br>
phenotype) and phytochrome B (PHYB - silencing gives hypocotyl elongation in white<br>
light). For PDS a single target region was chosen, for PHYB, three target regions were<br>
used, respectively from the 5'UTR, a region of the coding region conserved between<br>
phytochromes and the 3' UTR. For each target region two oligonucleotides were made,<br>
one to make a double stranded section of 21 bp long, the other to make a 42bp double<br>
stranded section. The double stranded oligos were made as two single strands (upper and<br>
lower) and annealed to form a double stranded DNA fragment. Overhang sequences were<br>
included at the 5' and 3' ends to create Sail compatible ends. The oligo sequences are<br>
represented in the sequence listing as SEQ ID 12 to 23 for the PHYB constructs and SEQ<br>
ID 24 to 27 for PDS constructs.<br>
The PDShp42 constructs gave phenotypes in most of the examined plants. The results are<br>
summarized in table 6.<br><br><br>
Insertion of the construct with the dsRNA coding region PDS42 under control of the 35S<br>
promoter resulted in more plants with no silencing phenotype than the construct with the<br>
dsRNA coding region PDS42 under control of the U6 promoter, and plants with a<br>
phenotype only showed the weak bleached cotyledon phenotype and no bleaching of the<br>
leaves.<br>
For the PHYB silencing experiments, most satisfactory silencing results are obtained with<br>
the PHYBC42 dsJRNA coding region, where most of the plants show more elongation than<br>
the controls. Most of the other constructs do not show a phenotype or else only have one<br>
or two plants showing a phenotype suggesting that the choice of target sequence may be<br>
important.<br>
Hypocotyl lengths of white light grown plants were measured and grouped in 5 mm<br>
categories From the summary of the data in Table 7, it appears that the U6 promoter<br>
driven construct is more effective that the CaMV35S promoter driven construct, buth the<br>
results are not as pronounced as for the above mentioned PDS gene silencing<br>
experiments.<br>
Table 7. Silencing of PHYB.<br><br>
Categories<br>
Hypocotyl length<br>
(cm)	WT	35S-PHYbC42	U6-PHYbC42<br>
0.1-0.5			<br>
0.6-1.0	13	1	2<br>
1.1-1.5	10	2	5<br>
1.6-2.0	3	5	10<br>
2.1-2.5		6	6<br>
2.6-3.0	5		9<br>
3.1-3.5	3	3	2<br>
3.6-4.0		1	11<br>
4.1-4.5			2<br>
4.6-5.0		1	1<br><br>
Thus, the following conclusion scan be drawn from the experiments :<br>
1.	Type 3 Pol III promoters can be used to effectively drive the expression of<br>
dsRNA molecules in plant cells.<br>
2.	The At U6 promoter seems to be the most effective promoter tested.<br>
3.	The monocot PolIIII promoter is functional both in monocotyledonous and<br>
dicotyledonous plants, but the dicotyledonous promoters seem not to be<br>
functional in monocotyledonous plants.<br>
4.	The type III Pol III promoters appear to be more effective than CaMV35S<br>
promoter for gene silencing with relatively short hairpin sequences.<br>
Example 6. Additional experiments with small hairpin RNA encoding constructs<br>
By using a the cloning strategy as outlined Figure 1, 20 new small hairpin constructs, as<br>
summarized in Table 7, were prepared . The predicted small hpRNAs from all of these<br>
constructs comprise a 42 bp dsRNA stem (corresponding to the target gene sequences)<br>
and a 9-nt loop (non-target sequence). Three target sequences corresponding to different<br>
regions of EIN2 (represented in SEQ ID 31 to 33) or GUS (represented in SEQ ID 28 to 30)<br>
were selected. These constructs have been used to transform tobacco to assess their<br>
efficacy for inducing the silencing of corresponding endogenous (EIN2) or reporter (GUS)<br>
genes.<br>
Tobacco shoots were assayed for GUS expression, and the result is shown in Table 8.<br>
The result shows that<br>
1)	most of the small GUS hairpin constructs conferred good GUS silencing; and<br>
2)	the Poim promoter driven constructs pLMW164 and pLMW165 result in more<br>
consistent GUS silencing than the 35S promoter driven, construct pLMW155.<br><br>
Table 7. Summary of additional small hairpinRNA encoding constructs<br><br><br><br>
The tobacco tissue assayed was mostly leaf pieces from small regenerating shoots<br>
growing on medium with hygromycin, which usually gives tight selection.<br>
References<br>
Bourque and Folk (1992) Plant Mol Biol. 19:641-647<br>
Brummelkamp et al. (2002) Science296 : 550-553<br>
Elbashir et al. (2001) Nature411:494-498<br>
Fire etal. (1998) Nature391:806-811<br>
Hamilton et al. (1998) Plant J. 15:737-746<br>
Miyagishi et al. (2002) Nature Biotechnology20: 497-499<br>
Needleman and Wunsch (1970) J. Mol. Biol. 48:443-453<br>
Paddison et al. (2002) Genes &amp; Development 16: 948-958<br>
Paul et al. (2002) Nature Biotechnology20. 505-508<br><br>
Sook Lee et al. (2002) Nature Biotechnology 20: 500-505<br>
Sui et al. (2002) Proa Natl. Acad. Sd. USA 99:5515-5520<br>
Waterhouse et al(1998) Proc. Natl. Acad. Sci. USA 95:13959-13964<br>
Yukawa et al. (2002) Plant Mol. Biol 50:713-723<br><br>
WE CLAIM:<br>
1.	A method for reducing the expression of a gene of interest in a plant cell, comprising<br>
the following steps :<br>
(a)	providing a chimeric gene to said plant cell, said chimeric gene comprising the<br>
following operably linked DNA fragments :<br>
i) a promoter recognized by a DNA dependent RNA polymerase III of said plant<br>
cell characterized in that said promoter is a promoter of type 3 comprising all<br>
cis-acting promoter elements which interact with said DNA dependent RNA<br>
polymerase III;<br>
ii) a DNA fragment which, when transcribed, yields an RNA molecule, said RNA<br>
molecule comprising a sense nucleotide sequence and an antisense nucleotide<br>
sequence,<br>
(1)	said sense nucleotide sequence comprising 19 contiguous nucleotides<br>
having 90 to 100% sequence identity to a nucleotide sequence of 19<br>
contiguous nucleotides from the RNA transcribed from said gene of<br>
interest;<br>
(2)	said antisense nucleotide sequence comprising 19 contiguous nucleotides<br>
having 90 to 100% sequence identity to the complement of a nucleotide<br>
sequence of 19 contiguous nucleotides of said sense sequence;<br>
wherein said sense and antisense nucleotide sequences are capable of<br>
forming a double stranded RNA of 19 to 200 nucleotides in length ; and<br>
iii) an oligo dT stretch comprising at least 4 consecutive T-residues; and<br>
(b)	identifying plant cells wherein said expression of said gene of interest is reduced<br>
when compared to the expression of said gene of interest in plant cells which do<br>
not comprise said chimeric gene.<br>
2.	The method as claimed in claim 1, wherein said promoter is a type 3 POLIII promoter<br>
selected from a promoter of a plant gene encoding U6snRNA, a promoter of a plant<br>
gene encoding U3snRNA and a promoter of a plant gene encoding 7SL RNA.<br>
3.	The method as claimed in claim 1 or claim 2, wherein said chimeric gene comprises a<br>
promoter consisting of a nucleotide sequence selected from the nucleotide sequence of<br>
SEQ ID NO:1 from the nucleotide at position 7 to the nucleotide at position 322, SEQ<br><br>
ID NO:2 from the nucleotide at position 7 to the nucleotide at position 408, SEQ ID<br>
NO:3 from the nucleotide at position 7 to the nucleotide at position 313, SEQ ID<br>
NO:4 from the nucleotide at position 7 to the nucleotide at position 446, SEQ ID<br>
NO:5 from the nucleotide at position 7 to the nucleotide at position 436, SEQ ID<br>
NO:6 from the nucleotide at position 7 to the nucleotide at position 468, SEQ ID<br>
NO:7 from the nucleotide at position 7 to the nucleotide at position 384 or SEQ ID<br>
NO:8 from the nucleotide at position 7 to the nucleotide at position 421.<br>
4.	The method as claimed in claim 1 or claim 2, wherein said plant cell is a<br>
dicotyledonous plant cell and said promoter is derived from a dicotyledonous or<br>
monocotyledonous plant or plant cell.<br>
5.	The method as claimed in claim 1 or claim 2, wherein said plant cell is a<br>
monocotyledonous plant cell, and said promoter is derived from a monocotyledonous<br>
plant or plant cell.<br>
6.	The method as claimed in claim 1 or claim 2 wherein said promoter is endogenous to<br>
said plant cell.<br>
7.	The method as claimed in claim 1 or claim 2, wherein said gene of interest is a<br>
transgene.<br>
8.	The method as claimed in claim 1 or claim 2, wherein said gene of interest is an<br>
endogenous gene.<br>
9.	The method as claimed in claim 1 or claim 2, wherein said plant cell is comprised<br>
within a plant.<br>
10.	The method as claimed in claim 1 or claim 2, wherein said DNA fragment ii)<br>
comprises a spacer region of 3 to 100 nucleotides which is located between the DNA<br>
encoding the sense nucleotide sequence and the DNA encoding the antisense<br>
nucleotide sequence.<br>
11.	The method as claimed in claim 1 or claim 2, wherein the length of said double<br>
stranded RNA is between 20 and 100 basepairs.<br><br>
12.	A chimeric gene for use in a plant cell, comprising the following operably linked<br>
DNA fragments :<br>
i) a promoter recognized by a DNA dependent RNA polymerase III of said plant<br>
cell characterized in that said promoter is a promoter of type 3 comprising all<br>
cis-acting promoter elements which interact with said DNA dependent RNA<br>
polymerase III;<br>
ii) a DNA fragment which, when transcribed, yields an RNA molecule, said RNA<br>
molecule comprising a sense nucleotide sequence and an antisense nucleotide<br>
sequence,<br>
(1)	said sense nucleotide sequence comprising 19 contiguous nucleotides<br>
having 90 to 100% sequence identity to a nucleotide sequence of 19<br>
contiguous nucleotides from the RNA transcribed from a plant gene of<br>
interest;<br>
(2)	said antisense nucleotide sequence comprising 19 contiguous nucleotides<br>
having 90 to 100% sequence identity to the complement of a nucleotide<br>
sequence of 19 contiguous nucleotides of said sense nucleotide sequence;<br>
wherein said sense and antisense nucleotide sequences are capable of<br>
forming a double stranded RNA of 19 to 200 nucleotides in length ; and<br>
iii) an oligo dT stretch comprising at least 4 consecutive T-residues,<br>
wherein said chimeric gene when provided to said plant cell reduces the expression of<br>
the plant gene of interest.<br>
13.	The chimeric gene as claimed in claim 12, wherein said promoter is a type 3 POLIII<br>
promoter selected from a promoter of a plant gene encoding U6snRNA, a promoter of<br>
a plant gene encoding U3snRNA and a promoter of a plant gene encoding 7SL RNA.<br>
14.	The chimeric gene as claimed in claim 12 or claim 13, wherein said chimeric gene<br>
comprises a promoter consisting of a nucleotide sequence selected from the nucleotide<br>
sequence of SEQ ID NO:1 from the nucleotide at position 7 to the nucleotide at<br>
position 322, SEQ ID NO:2 from the nucleotide at position 7 to the nucleotide at<br>
position 408, SEQ ID NO:3 from the nucleotide at position 7 to the nucleotide at<br>
position 313, SEQ ID NO:4 from the nucleotide at position 7 to the nucleotide at<br>
position 446, SEQ ID NO:5 from the nucleotide at position 7 to the nucleotide at<br>
position 436, SEQ ID NO:6 from the nucleotide at position 7 to the nucleotide at<br><br>
position 468, SEQ ID NO:7 from the nucleotide at position 7 to the nucleotide at<br>
position 384 or SEQ ID NO:8 from the nucleotide at position 7 to the nucleotide at<br>
position 421.<br>
15.	The chimeric gene as claimed in claim 12 or claim 13, wherein said plant cell is a<br>
dicotyledonous plant cell and said promoter is derived from a dicotyledonous or<br>
monocotyledonous plant or plant cell.<br>
16.	The chimeric gene as claimed in claim 12 or claim 13, wherein said plant cell is a<br>
monocotyledonous plant cell, and said promoter is derived from a monocotyledonous<br>
plant or plant cell.<br>
17.	The chimeric gene as claimed in claim 12 or claim 13 wherein said promoter is<br>
endogenous to said plant cell.<br>
18.	The chimeric gene as claimed in claim 12 or claim 13, wherein said gene of interest is<br>
a transgene.<br>
19.	The chimeric gene as claimed in claim 12 or claim 13, wherein said gene of interest is<br>
an endogenous gene.<br>
20.	The chimeric gene as claimed in claim 12 or claim 13, wherein said plant cell is<br>
comprised within a plant.<br>
21.	The chimeric gene as claimed in claim 12 or claim 13, wherein said DNA fragment ii)<br>
comprises a spacer region of 3 to 100 nucleotides which is located between the DNA<br>
encoding the sense nucleotide sequence and the DNA encoding the antisense<br>
nucleotide sequence.<br>
22.	The chimeric gene as claimed in claim 12 or claim 13, wherein the length of said<br>
double stranded RNA is between 20 and 100 basepairs.<br><br>
The present invention provides method for reducing the expression of a gene of interest in a<br>
plant cell. The method comprises providing a chimeric gene to the plant cell in which the<br>
chimeric gene comprises a DNA dependant RNA polymerase III of type 3 of a plant cell and<br>
a DNA fragment which when transcribed yields a short hairpin RNA.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU2My1LT0xOUC0yMDA1LUZPUk0tMjcucGRm" target="_blank" style="word-wrap:break-word;">1563-KOLNP-2005-FORM-27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU2My1rb2xucC0yMDA1LWdyYW50ZWQtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">1563-kolnp-2005-granted-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU2My1rb2xucC0yMDA1LWdyYW50ZWQtYXNzaWdubWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">1563-kolnp-2005-granted-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU2My1rb2xucC0yMDA1LWdyYW50ZWQtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">1563-kolnp-2005-granted-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU2My1rb2xucC0yMDA1LWdyYW50ZWQtY29ycmVzcG9uZGVuY2UucGRm" target="_blank" style="word-wrap:break-word;">1563-kolnp-2005-granted-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU2My1rb2xucC0yMDA1LWdyYW50ZWQtZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">1563-kolnp-2005-granted-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU2My1rb2xucC0yMDA1LWdyYW50ZWQtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">1563-kolnp-2005-granted-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU2My1rb2xucC0yMDA1LWdyYW50ZWQtZXhhbWluYXRpb24gcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">1563-kolnp-2005-granted-examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU2My1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">1563-kolnp-2005-granted-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU2My1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">1563-kolnp-2005-granted-form 13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU2My1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSAxOC5wZGY=" target="_blank" style="word-wrap:break-word;">1563-kolnp-2005-granted-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU2My1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">1563-kolnp-2005-granted-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU2My1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">1563-kolnp-2005-granted-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU2My1rb2xucC0yMDA1LWdyYW50ZWQtZ3BhLnBkZg==" target="_blank" style="word-wrap:break-word;">1563-kolnp-2005-granted-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU2My1rb2xucC0yMDA1LWdyYW50ZWQtcmVwbHkgdG8gZXhhbWluYXRpb24gcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">1563-kolnp-2005-granted-reply to examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU2My1rb2xucC0yMDA1LWdyYW50ZWQtc2VxdWVuY2UgbGlzdGluZy5wZGY=" target="_blank" style="word-wrap:break-word;">1563-kolnp-2005-granted-sequence listing.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU2My1rb2xucC0yMDA1LWdyYW50ZWQtc3BlY2lmaWNhdGlvbi5wZGY=" target="_blank" style="word-wrap:break-word;">1563-kolnp-2005-granted-specification.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="226735-a-colour-printing-system.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="226737-a-high-molecular-weight-water-soluble-polymer-comprising-pendant-salicylic-acid-groups.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>226736</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1563/KOLNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>52/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>26-Dec-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>24-Dec-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>08-Aug-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>COMMONWEALTH SCIENTIFIC AND INDUSTRIAL RESEARCH ORGANISATION</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>LIMESTONE AVENUE, CAMPBELL, AUSTRALIAN CAPITAL TERRITORY 2601</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>WANG MING BO</td>
											<td>3 KIEWA STREET, KALEEN, AUSTRALIAN CAPITAL TERRITORY 2617</td>
										</tr>
										<tr>
											<td>2</td>
											<td>HELLIWELL CHRISTOPHER ANDREW</td>
											<td>167 WATTLE STREET, O&#x27;CONNOR, AUSTRALIAN CAPITAL TERRITORY 2602</td>
										</tr>
										<tr>
											<td>3</td>
											<td>WATERHOUSE PETER MICHAEL</td>
											<td>5 BANJINE STREET, O&#x27;CONNOR, AUSTRALIAN CAPITAL TERRITORY 2602</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A01H 1/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/AU2004/000199</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-02-19</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/447,711</td>
									<td>2003-02-19</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/226736-a-method-for-reducing-the-expression-of-a-gene-of-interest-in-a-plant-cell by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:01:57 GMT -->
</html>
