<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/226798-a-recombinant-cancer-associated-epitope-vaccines-comprising-it-antibodies-binding-to-said-epitope-kits-and-compositions-comprising-said-antibodies by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:14:18 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 226798:A RECOMBINANT CANCER-ASSOCIATED EPITOPE, VACCINES COMPRISING IT, ANTIBODIES BINDING TO SAID EPITOPE, KITS AND COMPOSITIONS COMPRISING SAID ANTIBODIES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A RECOMBINANT CANCER-ASSOCIATED EPITOPE, VACCINES COMPRISING IT, ANTIBODIES BINDING TO SAID EPITOPE, KITS AND COMPOSITIONS COMPRISING SAID ANTIBODIES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>This invention discloses an isolated cancer-associated epitope comprising recombinant cytokeratin 8 and cytokeratin 18 polypeptide fragments, wherein the cytokeratin 8 polypeptide is shorter than 482 amino acids and comprises SEQ ID NO:3 or SEQ ID NO:5, and the cytokeratin 18 polypeptide is shorter than 429 amino acids and comprises SEQ ID NO:4 or SEQ ID NO:6. The invention is also for vaccines comprising it, antibodies binding to said epitope, kits and compositions comprising said antibodies.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>A RECOMBINANT CANCER-ASSOCIATED EPITOPE, VACCINES<br>
COMPRISING IT, ANTIBODIES BINDING TO SAID EPITOPE, KITS AND<br>
COMPOSITIONS COMPRISING SAID ANTIBODIES<br>
FIELD OF THE INVENTION<br>
The present invention relates to cancer-associated epitopes, antibodies and<br>
polypeptide binding entities directed against such epitopes. The invention also relates to<br>
diagnostic agents comprising the epitopes, antibodies or binding entities, and to the use of<br>
the epitope, antibodies or binding entities for a variety of diagnostic or therapeutic<br>
purposes. Pharmaceutical compositions are also contemplated by the invention, where the<br>
composition include the epitopes, antibodies or binding entities.<br>
BACKGROUND OF THE INVENTION<br>
Malignant tumors sometimes express characteristic antigens or "markers" that<br>
offers a means for detecting and possibly treating the tumors. For example, antigens that<br>
are characteristics of the tumor may be purified and formulated and used to generate<br>
antibodies. The antibodies raised by these antigens can be used as detection tools to<br>
monitor the level of tumor marker in the host to track the course of the disease or the<br>
effectiveness of treatment. Antibodies have also been coupled to toxins and administered<br>
to treat cancer. In some instances, the antigens can be used as vaccines to stimulate an<br>
antibody response and a celluler immune response within a cancer patient and thereby<br>
discourage the growth and spread of the cancer.<br>
Glandular epithelia cells contain a network of intermediate filaments that<br>
predominantly consists of complexes of cytokeratin 8 (K8) and cytokeratin 18 (K18).<br>
These filaments provide resilience in response to mechanical stress by forming a stable<br>
network attached to specific cell-cell contacts of the desmosome eukaryotes are<br>
expressed in a tissue-specific and cell type-restricted pattern (2). In epithelia cells,<br>
intermediate filaments consist of stoichiometrically equal amounts of type I (smaller,<br>
acidic) and type II (larger, neutral or basic) cytokeratin polypeptides that form strongly<br>
interacting heterodimers (3-5).<br>
Each cytokeratin polypeptide consists of a central 300-350 amino acid α-helical<br>
rod domain that is flanked by non-helical head (N-terminal) and tail (C-<br><br>
terminal) domains of various lengths and compositions. The rod domain can be<br>
further subdivided into four sub-domains (1 A, 1B, 2A and 2B), which are<br>
interspaced by short non-helical linkers (L1, L12, L2). Likewise, the head<br>
domain can be subdivided into three domains: the end domain (El), the variable<br>
domain (V1) and a region of sequence homology (H1) nearest to the rod domain<br>
(6).<br>
The assembly of intermediate filaments appears to involve several<br>
association steps that depend on interactions between different domains. In<br>
general, a type I and a type II cytokeratin polypeptide align in parallel to yield a<br>
coiled coil heterodimer (7,8). Subsequently, a tetramer is formed by anti-<br>
parallel, staggered, side-by-side aggregation of two dimers. The tetramers<br>
polymerize end to end to form a protofilament, and eight protofilaments then<br>
combine to produce the final 10 nm filament (9).<br>
The assembled rods form a protofilament backbone structure from two<br>
coiled coil subunits. However, the head and the tail domain are not thought to be<br>
part of the filamentous backbone. Instead, the head and tail domain appear to<br>
protrude laterally and to contribute to protofilament and intermediate filaments<br>
packing. The head and tail domains may also contribute to intermediate filament<br>
interaction with other cellular components (10-12). Thus, cytokeratins lacking<br>
the head and tail domains are generally capable of coiled coil and higher order<br>
lateral interactions, but are deficient in filament elongation (13).<br>
Cytokeratin 8 (K8) type I and cytokeratin 18 (K18) type II are the major<br>
components of intermediate filaments of simple or single layer epithelia, such as<br>
those of the intestine, liver and breast ducts (4). These two cytokeratins form<br>
heterodimers and filaments. Deletion studies of K8 and K18 cytokeratins have<br>
shown that the head domains play a crucial role in forming heterodimers and<br>
filaments. Co-transfection of head-deleted K8 and head-deleted Kl 8 resulted in<br>
the formation of a dispersed non-fibrillar pattern, while co-transfection of a<br>
combination of one headless plus one intact cytokeratin resulted in the formation<br>
of cytoplasmic granules or fibrils (12). More detailed analysis showed that only<br>
short and irregular intermediate filaments were generated when K8 and K18<br>
were N-terminally truncated by deleting the first 66 amino acid of each of the<br><br>
cytokeratins (13). The whole, or nearly the whole, H1 region of the head domain<br>
was required for generation of these short filaments. Only tetramers were<br>
generated when a major part of the H1 domain was additionally removed to form<br>
a complex between a truncated K8 (amino acids 75-483) and a truncated K18<br>
(amino acids 67-385) (13). The importance of H1 region apparently relates to its<br>
involvement in the alignment of the two heterodimers and to the stabilization of<br>
the formed heterotetramer complexes.<br>
The precise function of K8 and K18 remains largely unclear, although<br>
recent data indicates that both cytokeratins are important in natural development.<br>
A mutant of K18 (arg89 cys), expressed as a dominant trait in transgenic mice,<br>
resulted in marked disruption of the liver and pancreas intermediate filament<br>
network, leading to hepatocyte instability and associated liver inflammation and<br>
necrosis (14,15). The phenotype of K8 or K19 knockout mice included complete<br>
or partial midgestational embryonic lethality depending on the genetic<br>
background, female sterility and adult colorectal hyperplasia in the surviving<br>
animals (16-18). Other data have suggested that K8/K18 filaments play a role in<br>
multiple drug resistance (19-21).<br>
During cell transformation and tumor development, the cell type<br>
specificities of K8 and K18 are conserved, making them useful as clinical<br>
histopathological markers for tissue type identification (22-24). Given that the<br>
cell type specificities of K8 and K18 are conserved during cell transformation<br>
and tumor development, one would not expect that the KS and K18 cytokeratins<br>
would exhibit a new antigenic epitope in cancerous cells.<br>
SUMMARY OF THE INVENTION<br>
The invention provides an isolated cancer-associated epitope comprising<br>
two separate polypeptides. The first polypeptide can have SEQ ID NO:3 of<br>
cytokeratin 8 and the second polypeptide can have SEQ ID NO:4 of cytokeratin<br>
18. Alternatively, the first polypeptide can have SEQ ID NO:5 of cytokeratin 8<br>
and the second polypeptide can have SEQ ID NO:6 of cytokeratin 18. Moreover,<br>
the first polypeptide can have SEQ ID NO:3 of cytokeratin 8 and the second<br>
polypeptide can have SEQ ID NO:6 of cytokeratin 18. The first polypeptide can<br><br>
also have SEQ ID NO:5 of cytokeratin 8 and the second polypeptide can have<br>
SEQ ID NO:4 of cytokeratin 18.<br>
Such isolated epitopes can be detected in filamentous cytoplasmic<br>
structures of adenocarcinoma cells but are not substantially detected in normal<br>
cells. Examples of adenocarcinomas where these epitopes can be detected<br>
include colon adenocarcinoma, ovarian adenocarcinoma, renal adenocarcinoma,<br>
mammary adenocarcinoma, lung adenocarcinoma, pancreatic adenocarcinoma<br>
and non-seminomal testis carcinoma cells. These epitopes are useful for making<br>
cancer-specific antibodies, and for.diagnosing cancer by detecting either the<br>
antigenic epitopes or antibodies directed against these epitopes in the blood,<br>
serum, feces or urine of a cancer patient. Accordingly, in one embodiment, the<br>
epitopes are provided in a kit.<br>
In another embodiment, the invention provides an antibody or other<br>
binding entity that can bind any of the cancer-associated epitopes of the<br>
invention. In one embodiment, the antibody or binding entity can include a<br>
polypeptide comprising any one of SEQ ID NO:7-35. Preferred antibodies or<br>
binding entities include polypeptides comprising any one of SEQ ED NO:21-35,<br>
or a combination thereof. In another embodiment, the invention is directed to a<br>
polypeptide comprising any combination of SEQ ID NO:7-33, wherein the<br>
polypeptide that can bind an epitope of the invention. The antibody or binding<br>
entity can be encoded by a nucleic acid comprising any one of SEQ ID NO.36-<br>
39. Such a binding entity or antibody can detect the cancer-associated epitope in<br>
filamentous cytoplasmic structures of colon adenocarcinoma, ovarian<br>
adenocarcinoma, renal adenocarcinoma, mammary adenocarcinoma, lung<br>
adenocarcinoma, pancreatic adenocarcinoma and non-seminomal testis<br>
carcinoma cells. The antibody or binding entity can have a label or diagnostic<br>
imaging agent attached to it. The invention also provides such compositions and<br>
kits containing the binding entities or antibodies. Preferably, when antibodies<br>
are employed, the antibody is not a COU-1 monoclonal antibody.<br>
The invention further provides a method of detecting adenocarcinoma by<br>
contacting an antibody or binding entity of the invention with a test sample and<br>
detecting whether the antibody or binding entity binds to a cancer-associated<br><br>
epitope. The antibodies and binding entities can have a label or diagnostic<br>
imaging agent attached thereto.<br>
The invention also provides a method of treating cancer in a mammal by<br>
administering a therapeutically effective amount of an antibody or binding entity<br>
of the invention that can bind to the cancer-associated epitope. Such an antibody<br>
or binding entity can have a therapeutically useful agent attached thereto.<br>
The invention further provides a method of treating cancer in a mammal<br>
comprising administering a therapeutically effective amount of a cancer-<br>
associated epitope of the invention.<br>
The invention also provides a method of treating cancer in a mammal<br>
comprising administering a therapeutically effective amount of a protease<br>
inhibitor that can inhibit formation of the cancer-associated epitopes of the<br>
invention by inhibiting the protease(s) that cleave cytokeratin 8 and cytokeratin<br>
18. In one embodiment, the protease is a trypsin-like protease and the inhibitor<br>
is a serine protease inhibitor or a trypsin inhibitor.<br>
The invention also provides a method of identifying a mutant antibody<br>
comprising fusing a nucleic acid encoding a polypeptide having any one of SEQ<br>
ID NO:7-35 to a nucleic acid encoding a phage coat protein to generate a<br>
recombinant nucleic acid encoding a fusion protein, mutating the recombinant<br>
nucleic acid encoding the fusion protein to generate a mutant nucleic acid<br>
encoding a mutant fusion protein, expressing the mutant fusion protein on the<br>
surface of a phage and selecting phage that bind to a cancer-associated epitope of<br>
the invention.<br>
DESCRIPTION OF THE FIGURES<br>
Figure 1 illustrates the purification of cytokeratin from colon cancer<br>
tissue.<br>
Figure 1A provides an elution profile (OD280, solid line) of cytokeratin-<br>
enriched material applied to a QFF anion-exchange column (100-ml bed volume)<br>
in SDS-containing buffer and eluted with a linear gradient to 1 M NaCl (dashed<br>
line). Fractions containing reactivity with COU-1 were observed in the first and<br>
second peaks of the salt gradient.<br><br>
Figure 1B provides an expanded view of a region the elution profile<br>
shown in Figure 1A. This expanded profile provides a comparison of OD280<br>
absorption (solid line), salt gradient (dashed line) and COU-1 reactivity (dotted<br>
line) for fractions (30-53). Proteins from these fractions were coated onto<br>
ELISA wells and reacted with the COU-1 antibody.<br>
Figure 1C provides a Coomassie-stained blot of electrophoretically<br>
separated proteins from fractions 41-50, raw homogenate (H), and material<br>
applied to the QFF anion-exchange column (S). An extract of the colon cancer<br>
cell line Colon 137 (C137) was included as control.<br>
Figure 1D provides a Western blot (stained with COU-1) of<br>
electrophoretically separated proteins from fractions 41-50, raw homogenate (H),<br>
and material applied to the QFF anion-exchange column (S). An extract of the<br>
colon cancer cell line Colon 137 (C137) was included as control. This blot<br>
illustrates the purity of the cytokeratins obtained from the QFF column and the<br>
reactivity of the COU-1 antibodies with 3 bands at molecular weights of 42-46<br>
kDa.<br>
Figure 2 provides a Western blot analysis of cytokeratin preparations<br>
purified from colon cancer tissue or from normal colon epithelia under identical<br>
conditions. Homogenate (homog) and anion-exchange chromatography-purified<br>
material (QFF) was applied to the SDS-PAGE gel at 10-fold dilutions (1:1,1:10,<br>
1:100). The proteins were transferred to PVDF membranes and stained with<br>
COU-1 or murine anti-K18 Mab. While the anti-K18 antibody intensely stained<br>
cytokeratin preparations from normal and malignant colon epithelia, COU-1<br>
intensely stained only cytokeratin proteins from cancer tissue (three bands of<br>
about 42-46 kDa), and not cytokeratin proteins from normal epithelia.<br>
Figure 3 illustrates the SDS-PAGE separation and N-terminal sequencing<br>
of cytokeratins purified from colon cancer tissue and further illustrates the<br>
presence of N-terminally-truncated K8, K18 and K19 cytokeratins in these colon<br>
cancer tissues.<br>
Figure 3 A is a PVDF membrane blot of SDS-PAGE-separated purified<br>
cytokeratin. Region (a) of this membrane was stained with Coomassie. Strips of<br>
the membrane were also stained with the COU-1 antibody (region b), a mouse<br><br>
anti-K8 antibody (region c) and a mouse anti-K18 antibody (region d). These<br>
data illustrate that ten different protein bands (1-10) can be detected. These ten<br>
bands were each N-terminally sequenced.<br>
Figure 3B provides the amino acid sequences of cytokeratin proteins<br>
isolated from colon cancer tissues as determined by N-tenninal sequencing (SEQ<br>
ID NOs:54-6l). In addition, the reactivity of the different isolated cytokeratin<br>
proteins with a panel of K8/K18 antibodies is shown.<br>
Figure 4 provides a map of the structural domains of the K8 and K18<br>
cytokeratins (center). The secondary structural domains of the cytokeratin<br>
polypeptides were identified from the amino acid sequences. A central rod<br>
domain is shown that is flanked by a non-helical N-tenninal head domain and a<br>
non-helical C-terminal tail domain. Domains 1A, 1B, 2A and 2B are α-helical<br>
subdomains of the rod interspaced by linkers L1, L12 and L2. This figure also<br>
provides a schematic representation of the K8 and K18 N-terminal and C-<br>
terminal deletion proteins. Deletion protein names provide the starting and<br>
ending amino acid residue numbers of the deletion protein. All deletion proteins<br>
were expressed as GST fusion proteins. The positive (+) and negative (-)<br>
reactivity of the deletion protein fragments with COU-1 after incubation with the<br>
complementary keratin is shown in parentheses.<br>
Figure 5 provides a Western blot analysis of a panel of C-terminal deleted<br>
K18 fragments. SDS-PAGE gels containing a panel of C-terminal deleted K18<br>
fragments expressed as GST fusion proteins were run in parallel, transferred to<br>
PVDF membranes and stained with either a goat anti-GST antibody (A), COU-1<br>
(B) or a mouse anti-KI8 antibody (CY-90) (D).<br>
Figure 5 A is a Western blot of an electrophoretically separated panel of<br>
C-terminal deleted K18 protein fragments expressed as GST fusion proteins that<br>
was stained with a goat anti-GST antibody. The identity of the different K18<br>
protein fragments is provided at the top, where the numbers indicate which<br>
amino acids are present in the different K18 protein fragments. A GST protein<br>
preparation was used a positive control for GST antibody staining. An MCF7<br>
cancer cell lysate was used as positive control for the cytokeratin epitope (no<br>
staining is visible because the GST protein is not present in the lysate). The GST<br><br>
antibody staining demonstrated that all K18 fragments were expressed well and<br>
at approximately the same levels.<br>
Figure 5B is a replica of the Western blot of an electrophoretically<br>
separated panel of C-terminally deleted K18 fragments provided in Figure 5A<br>
that was stained with the COU-1 antibody. On this blot, COU-1 only reacted<br>
with MCF7 cancer cell lysate used as positive control. No substantial binding of<br>
COU-1 to individual K18 fragments was observed.<br>
Figure 5C is a replica of the Western blot of an electrophoretically<br>
separated panel of C-terminally deleted K18 fragments provided in Figure 5A.<br>
However, this blot was incubated with purified intact K8 prior to staining COU-<br>
1. COU-1 bound strongly to some of the K18 fragments when complexed with<br>
K8.<br>
Figure 5D is a replica of the Western blot of an electrophoretically<br>
separated panel of C-terminally deleted K18 fragments provided in Figure 5A<br>
that was stained with a mouse anti-K18 antibody (CY-90). The CY-90 antibody<br>
reacted with an epitope corresponding to residues 340-390 in the C-terminal part<br>
of non-complexed K18.<br>
Figure 6 provides an SDS-PAGE and Western blot analysis of a panel of<br>
C-terminal deleted K8 fragments. SDS-PAGE gels containing a panel of C-<br>
terminal deleted K8 fragments expressed as GST fusion proteins were run in<br>
parallel and stained with Coomassie Blue (A), or transferred to PVDF<br>
membranes and stained with COU-1 (B) or incubated with purified intact K18<br>
prior to staining HMab COU-1 (C).<br>
Figure 6A is an SDS-PAGE gel of an electrophoretically separated panel<br>
of C-terminal deleted K8 fragments stained with Coomassie Blue. The identity<br>
of the different K8 protein fragments is provided at the top, where the numbers<br>
indicate the range of amino acids present in the different K18 protein fragments.<br>
Coomassie Blue staining demonstrated that all fragments were expressed<br>
approximately equally well.<br>
Figure 6B is a replica of the Western blot of an electrophoretically-<br>
separated panel of C-terminally deleted K8 fragments provided in Figure 6A that<br>
was stained with the COU-1 antibody. No binding of COU-1 to individual K8<br><br>
fragments was observed. On this blot, COU-1 only reacted with the positive<br>
control, a MCF7 cancer cell iysate.<br>
Figure 6C is a replica of the Western blot of an electrophoretically<br>
separated panel of C-terminally deleted K8 fragments provided in Figure 6A that<br>
was incubated with purified intact K18 prior to staining HMab COU-1. COU-1<br>
bound strongly to some of the K8 fragments because when they had formed<br>
complexes with K18.<br>
Figure 7 provides a Western blot analysis of a panel of C-terminally<br>
deleted K8 or K18 fragments that were electrophoretically separated, transferred<br>
to a PVDF membrane and then incubated with different purified C-terminally<br>
deleted K8 or K18 fragments to form K8/K18 complexes prior to staining with<br>
COU-1<br>
Figure 7A provides a Western blot analysis of a panel of C-terminally<br>
deleted Kl8(l-72), K18(l-124), Kl8(1-187) and intact K18 protein fragments<br>
that were electrophoretically separated and transferred to a PVDF membrane.<br>
The membrane was then incubated with purified C-terminally deleted K8(l-129)<br>
fragment and stained with COU-1. The COU-1 antibodies bound strongly to<br>
K18(1-124)/K8(1-129) complexes.<br>
Figure 7B is a replica of the Western blot of an electrophoretically-<br>
separated panel of C-terminally deleted K18 fragments provided in Figure 7A<br>
that was incubated with purified C-terminally deleted K8(l-233) fragment and<br>
stained with COU-1. COU-1 staining was absent or only minimally observed for<br>
the K18(l-187)/K8(l-233) complex.<br>
Figure 7C is a replica of the Western blot of an electrophoretically-<br>
separated panel of C-terminally deleted K18 fragments provided in Figure 7A<br>
that was incubated with purified intact K8 and stained with COU-1. COU-1<br>
staining was absent or only minimally observed for the K18(l-187)/intact K8<br>
complex.<br>
Figure 7D provides a Western blot analysis of a panel of C-terminally<br>
deleted K8(l-85), K8(l-129), K8(l-233) and intact K8 polypeptides that were<br>
electrophoretically separated and transferred to a PVDF membrane. The<br>
membrane was then incubated with purified K18(1-124) and stained with COU-<br><br>
1. The COU-1 antibody recognized K8(l-129) complexed with K18(l-124), and<br>
K8(l-233) complexed with K18(l-124). No COU-1 binding was observed with<br>
complexes containing K8(l-85).<br>
Figure 7E is a replica of the Western blot of an electrophoretically-<br>
separated panel of C-terminally deleted K8 fragments provided in Figure 7D that<br>
was incubated with purified K18(l-187) and stained with COU-1. The COU-1<br>
antibody recognized K8(l-129) complexed with K18(l-187) and K8(l-233)<br>
complexed with K18(l-187). No COU-1 binding was observed with any<br>
complexes containing K8(l-85).<br>
Figure 7F is a replica of the Western blot of an electrophoretically-<br>
separated panel of C-terminally deleted K8 fragments provided in Figure 7D that<br>
was incubated with purified K18(1-213) and stained with COU-1. The COU-1<br>
antibody recognized K18(l-213) complexed with K8(l-129) and K18(l-213)<br>
complexed with K8(l-233) K18(l-187). No COU-1 binding was observed with<br>
any complexes containing K8(l-85).<br>
Figure 8 provides a schematic representation of the N-terminal head<br>
domain and the adjacent rod domain of K8/K18 heterotypic complex.<br>
Figure 8A identifies the sites where K8 (SEQ ID NO:62) and K18 (SEQ<br>
ID NO:63) are proteolytically cleaved (arrows). For cytokeratin K8, cleavage<br>
was after Arg-22, after Arg-39, after Val-65 and after Leu-75. For cytokeratin<br>
K18, cleavage was after Arg-49 and after IIe-67. The positions of residues that<br>
are post-translationally phosphorylated (PO4, P) or glycosylated (GlcNac, G) are<br>
also identified.<br>
Figure 8B is a schematic diagram illustrating how cleavage of the N-<br>
terminal head domain of K8 and K18 cytokeratins can cause a conformational<br>
change allowing the COU-1 antibody to access the epitope. This diagram is<br>
consistent with observations made in vivo in cancer cells and made on the<br>
formation of recombinant K8/K18 complexes. This diagram further illustrates<br>
how in vitro deletion of a major portion of the C-terminal domain of one of the<br>
two cytokeratin proteins may also artificially expose the COU-1 epitope. This<br>
diagram is also consistent with complex formation studies on a panel of<br>
recombinant K8/K18 deletion proteins.<br><br>
Figure 9 illustrates that COU-1 binds preferentially to heterotypic<br>
complexes containing N-terminally deleted K8 fragments. This figure provides a<br>
PVDF membrane blot of SDS-PAGE-separated K8 and K18 proteins. Region A<br>
has intact K8 or K8 (66-483) proteins that were electrophoretically-separated and<br>
then reacted with purified K18 (50-430) prior to staining with COU-1. Region B<br>
has intact K8 or K8 (66-483) proteins incubated with purified intact K18 prior to<br>
staining with COU-1. Region C has K18 (50-430) and intact K18 that were<br>
electrophoretically-separated and then incubated with purified K8 (66-483) prior<br>
to staining with COU-1. Strong staining of a band of about 75 kDa (K8/K18<br>
proteins + the GST fusion protein) was observed in lanes containing K8(66-<br>
483)/K18(50-430) and K8(66-4S3)/intact K18 with COU-1. Region D has<br>
Kl8(50-430) and intact K18 proteins that were electrophoretically-separated and<br>
then incubated with purified intact K8 prior to staining with COU-1. Only weak<br>
staining was observed to intact K8/K18 (50-430) and intact K8/intact K18.<br>
Figure 10 illustrates that COU-1 binds preferentially to truncated forms<br>
of recombinant heterotypic K8/K18 complexes as measured by ELISA.<br>
Heterotypic complexes were generated by combining purified recombinant K8(l-<br>
129) or intact K8 with purified recombinant K18(1-124) or intact K18 in equal<br>
molar ratio. A 5 µg/ml solution of these complexes was used to coat ELISA<br>
plates. The ELISA plates were incubated with COU-1 in serial dilutions. Bound<br>
COU-1 was detected and visualized with an AP-labeled secondary anti-human<br>
kappa antibody and para-nitrophenylphosphate. As shown, the K8(1-129)/intact<br>
K18 complex (diamond symbols) and the K8/K18(l-124) (circles) bound more<br>
COU-1 antibodies than the intact K8/intact K18 complex.<br>
Figure 11 illustrates the distribution of N-terminally-truncated K8/K18<br>
complexes (A and E) and K18 (B and F) in breast cancer cells. Ethanol-fixed<br>
MCF7 breast cancer cells were separately incubated with COU-1 antibodies (A<br>
and E) and CY90 monoclonal antibodies (B and F), which binds to intact K18.<br>
Bound COU-1 was detected with FITC-goat anti-human γ-chain antibody (green)<br>
and bound CY90 detected with a Texas Red-goat-anti-mouse IgG antibody (red).<br>
DIC images (D and H) were included to visualize the composition of the cells.<br>
Partial co-localization, as visualized by yellow in the merged images (C and G),<br><br>
was observed between the two antibodies. However, N-terminally truncated<br>
K8/K18 complexes were predominantly found in newly-formed, proliferating<br>
cancer cells (arrows), whereas K18 structures were equally present in all cells<br>
(arrowheads).<br>
Figure 12 shows the cellular distribution of the N-terminally-truncated<br>
K8/K18 complex recognized by COU-1 (A and E) and K18 recognized by Mab<br>
CY-90 (B and F) in breast cancer cells. MCF7 breast cancer cells were<br>
processed and stained as described in Figure 11. DIC images (D and H) were<br>
included to visualize the composition of the cells. While whole intermediate<br>
filaments were stained with Mab CY-90 (arrowheads), COU-1 (arrows) only<br>
stained short fibrils and globular structures. Some co-localization of the two<br>
antibodies, as visualized by yellow in the merged images (C and G), was<br>
observed.<br>
Figure 13 provides the amino acid sequences for the variable heavy and<br>
light chain of the human monoclonal antibody COU-1 in comparison to closest<br>
known germ-line sequences (SEQ ID NOs:7-20).<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
According to the invention, the separate K8 cytokeratin polypeptide joins<br>
with the K18 cytokeratin polypeptide to form an antigenic epitope that is only<br>
visible in cancerous, and not in normal, cells. Such neoepitopes are generated by<br>
specific proteolytic cleavage of K8/K18 complexes in carcinoma cells. The new<br>
epitopes visible in cancer cells are used to generate antibodies or binding entities<br>
that are diagnostic of cancer and that are useful for treatment of cancer patients.<br>
Definitions<br>
The term "antibody" is used in the broadest sense, and specifically covers<br>
monoclonal antibodies (including full length monoclonal antibodies), polyclonal<br>
antibodies, multispecific antibodies (e.g., bispecific antibodies), and antibody<br>
fragments (e.g., Fab, F(ab')2 and Fv) so long as they exhibit the desired biological<br>
activity.<br><br>
A "binding entity," as used herein, is a polypeptide that can bind to the<br>
epitope identified by the invention. For example, a binding entity of the<br>
invention is a polypeptide that can bind to an epitope comprising two separate<br>
polypeptides, a cytokeratin S polypeptide and a cytokeratin 18 polypeptide,<br>
wherein the cytokeratin 8 polypeptide comprises SEQ ID NO:3 or SEQ ID NO:5,<br>
and the cytokeratin 18 polypeptide comprises SEQ ID NO:4 or SEQ ID NO:6.<br>
The terms "cancer" and "cancerous" refer to or describe the physiological<br>
condition in mammals that is typically characterized by unregulated cell growth.<br>
Examples of cancer include but are not limited to, carcinoma, lymphoma,<br>
blastoma, sarcoma, and leukemia. More particular examples of such cancers<br>
include in colon adenocarcinoma, ovarian adenocarcinoma, renal<br>
adenocarcinoma, mammary adenocarcinoma, lung adenocarcinoma, pancreatic<br>
adenocarcinoma and non-seminomal testis carcinoma tissues.<br>
The COU-1 antibody is a monoclonal antibody produced by the human<br>
hybridoma cell line B9165 (ECACC 87040201). It can bind to a carcinoma-<br>
associated antigen that has an apparent molecular weight of about 43,000 and an<br>
isoelectric point in the range of about 5.4-6.2.<br>
The expression "control sequences" refers to DNA sequences necessary<br>
for the expression of an operably linked coding sequence in a particular host<br>
organism. The control sequences that are suitable for prokaryotes, for example,<br>
include a promoter, optionally an operator sequence, and a ribosome binding site.<br>
Eukaryotic cells are known to utilize promoters, polyadenylation signals, and<br>
enhancers.<br>
The "derivative" of a reference antigenic epitope, antibody, nucleic acid,<br>
protein, polypeptide or peptide, has related but different sequence or chemical<br>
structure than the respective reference antigenic epitope, antibody, nucleic acid,<br>
protein, polypeptide or peptide. Such a derivative antigenic epitope, antibody,<br>
nucleic acid, protein, polypeptide or peptide is generally made purposefully to<br>
enhance or incorporate some chemical, physical or functional property that is<br>
absent or only weakly present in the reference antigenic epitope, antibody,<br>
nucleic acid, protein, polypeptide or peptide. A derivative nucleic acid differs in<br>
nucleotide sequence from a reference nucleic acid whereas a derivative antigenic<br><br>
epitope, antibody, protein, polypeptide or peptide differs in amino acid sequence<br>
from the reference antigenic epitope, antibody, protein, polypeptide or peptide,<br>
respectively. Such sequence differences include one or more substitutions,<br>
insertions, additions, deletions, fusions and truncations, which can be present in<br>
any combination. Differences can be minor (e.g., a difference of one nucleotide<br>
or amino acid), or more substantial, involving several or many nucleotides or<br>
amino acids. However, the sequence of the derivative is not so different from<br>
the reference that one of skill in the art would not recognize that the derivative<br>
and reference are related in structure and/or function. Generally, differences are<br>
limited so that the reference and the derivative are closely similar overall and, in<br>
many regions, identical. A "variant" differs from a "derivative" nucleic acid,<br>
protein, polypeptide or peptide in that the variant can have silent structural<br>
differences that do not significantly change the chemical, physical or functional<br>
properties of the reference nucleic acid, protein, polypeptide or peptide. In<br>
contrast, the differences between the reference and derivative nucleic acid,<br>
protein, polypeptide or peptide are intentional changes made to improve one or<br>
more chemical, physical or functional properties of the reference nucleic acid,<br>
protein, polypeptide or peptide.<br>
The term "identity" or "homology" shall be construed to mean the<br>
percentage of amino acid residues in the candidate sequence that are identical<br>
with the residue of a corresponding sequence to which it is compared, after<br>
aligning the sequences and introducing gaps, if necessary to achieve the<br>
maximum percent identity for the entire sequence, and not considering any<br>
conservative substitutions as part of the sequence identity. Neither N- or C-<br>
terminal extensions nor insertions shall be construed as reducing identity or<br>
homology. Methods and computer programs for the alignment are well known in<br>
the art. Sequence identity may be measured using sequence analysis software<br>
(e.g., Sequence Analysis Software Package, Genetics Computer Group,<br>
University of Wisconsin Biotechnology Center, 1710 University Ave., Madison,<br>
Wis. 53705). This software matches similar sequences by assigning degrees of<br>
homology to various substitutions, deletions, and other modifications.<br><br>
A "liposome" is a small vesicle composed of various types of lipids,<br>
phospholipids and/or surfactant that is useful for delivery of a drug (such as the<br>
antigenic epitopes and antibody mutants disclosed herein and, optionally,<br>
chemotherapeutic agent) to a mammal. The components of the liposome are<br>
commonly arranged in a bilayer formation, similar to the lipid arrangement of<br>
biological membranes.<br>
"Mammal" refers to any animal classified as a mammal, including<br>
human, domestic and farm animals, nonhuman primates, and zoo, sports, or pet<br>
animals, such as dogs, horses, cats, cows, etc.<br>
Nucleic acid is "operably linked" when it is placed into a functional<br>
relationship with another nucleic acid sequence. This can be a gene and a<br>
regulatory sequence(s) that are connected in such a way as to permit gene<br>
expression when the appropriate molecules (e.g., transcriptional activator<br>
proteins) are bound to the regulatory sequences(s). For example, DNA for a<br>
presequence or secretory leader is operably linked to DNA for a polypeptide if it<br>
is expressed as a preprotein that participates in the secretion of the polypeptide; a<br>
promoter or enhancer is operably linked to a coding sequence if it affects the<br>
transcription of the sequence; or a ribosome binding site is operably linked to a<br>
coding sequence if it affects the transcription of the sequence; or a ribosome<br>
binding site is operably linked to a coding sequence if it is positioned so as to<br>
facilitate translation. Generally, "operably linked" means that the DNA<br>
sequences being linked are contiguous, and, in the case of a secretory leader,<br>
contiguous and in reading phase. However, enhancers do not have to be<br>
contiguous. Linking is accomplished by ligation at convenient restriction sites. If<br>
such sites do not exist, the synthetic oligonucleotide adaptors or linkers are used<br>
in accordance with conventional practice.<br>
The terms "protein," "polypeptide" and "peptide" are used<br>
interchangeably. They refer to a chain of two (2) or more amino acids that are<br>
linked together with peptide or amide bonds, regardless of post-translational<br>
modification (e.g., glycosylation or phosphorylation). Antigens, epitopes and<br>
antibodies are specifically intended to be within the scope of this definition. The<br><br>
polypeptides of this invention may comprise more than one subunit, where each<br>
subunit is encoded by a separate DNA sequence.<br>
The phrase "substantially identical" with respect to an antigen, antibody<br>
or binding entity polypeptide sequence shall be construed as a polypeptide<br>
exhibiting at least 70%, preferably 75%, more preferably 80%, more preferably<br>
85%, even more preferably 90%, even more preferably 95% and especially<br>
preferably 97% or 98% sequence identity to the reference polypeptide sequence.<br>
For polypeptides, the length of the comparison sequences will generally be at<br>
least 25 amino acids. For nucleic acids, the length will generally be at least 75<br>
nucleotides.<br>
The "variant" of a reference antigenic epitope, antibody segment, binding<br>
entity, nucleic acid, protein, polypeptide or peptide, is an antigenic epitope,<br>
antibody segment, binding entity, nucleic acid, protein, polypeptide or peptide,<br>
respectively, with a related but different sequence than the respective reference<br>
antigenic epitope, antibody segment, binding entity, nucleic acid, protein,<br>
polypeptide or peptide. The differences between variant and reference antigenic<br>
epitopes, antibody segments, binding entities, nucleic acids, proteins,<br>
polypeptides or peptides are silent or conservative differences. A variant<br>
nucleic acid differs in nucleotide sequence from a reference nucleic acid whereas<br>
a variant antigenic epitope, antibody segment, binding entity, protein,<br>
polypeptide or peptide differs in amino acid sequence from the reference<br>
antigenic epitope, antibody segment, binding entity, protein, polypeptide or<br>
peptide, respectively. A variant and reference antigenic epitope, antibody<br>
segment, binding entity, nucleic acid, protein, polypeptide or peptide may differ<br>
in sequence by one or more substitutions, insertions, additions, deletions, fusions<br>
and truncations, which may be present in any combination. Differences can be<br>
minor (e.g., a difference of one nucleotide or amino acid), or more substantial.<br>
However, the structure and function of the variant is not so different from the<br>
reference that one of skill in the art would not recognize that the variant and<br>
reference are related in structure and/or function. Generally, differences are<br>
limited so that the reference and the variant are closely similar overall and, in<br>
many regions, identical.<br><br>
Epitope<br>
According to the invention, one or more novel neoepitopes that are<br>
immunologically recognizable are generated in a variety of adenocarcinoma- cells<br>
through specific proteolytic cleavage of cytokeratin K8 and cytokeratin K18<br>
proteins. Normal, non-cancerous cells do not display such neoepitopes. The<br>
cytokeratin KS and cytokeratin K18 proteins are separate proteins. However,<br>
they do form a cytokeratin K8/cytokeratin K18 complex. The immunologically<br>
recognizable neoepitope contains amino acids from both the cytokeratin K8 and<br>
cytokeratin K18 proteins.<br>
The epitope of the invention is not substantially present in normal tissues.<br>
However, the epitope becomes exposed in colon adenocarcinoma, ovarian<br>
adenocarcinoma, renal adenocarcinoma, mammary adenocarcinoma, lung<br>
adenocarcinoma, pancreatic adenocarcinoma and non-seminomal testis<br>
carcinoma tissues. The epitope of the invention is predominantly present in<br>
filamentous cytoplasmic structures of these types of cells during proliferation.<br>
Testing indicates that the epitope is not detected in certain sarcomas, malignant<br><br><br><br><br><br>
The epitopes of the invention consist of two polypeptides, a cytokeratin<br>
K8 polypeptide and a cytokeratin K18 polypeptide. However, the cytokeratin K8<br>
polypeptide is shorter than the full-length cytokeratin K.8 polypeptide that has<br>
482 amino acids. Moreover, the cytokeratin K18 polypeptide is shorter than the<br>
full-length cytokeratin K18 polypeptide that has 429 amino acids. In some<br><br>
embodiments, the cytokeratin K8 polypeptide is shorter than about 475 amino<br>
acids, or shorter than about 450 amino acids, or shorter than about 425 amino<br>
acids, or shorter than about 400 amino acids. In some embodiments, the<br>
cytokeratin K18 polypeptide is shorter than about 425 amino acids, or shorter<br>
than about 415 amino acids, or shorter than about 400 amino acids, or shorter<br>
than about 375 amino acids.<br>
One example of an epitope of the invention constitutes two peptidyl<br>
regions of two separate proteins, cytokeratin K.8 (SEQ ID NO:3) and cytokeratin<br>
K18 (SEQ ID NO:3). The epitope involves amino acids 85-129 of cytokeratin 8<br>
sequence, designated SEQ ID NO:3 and provided below.<br>
1 AVRTQEKEQI KTLNNKFASF IDKVRFLEQQ NKMLETKWSL<br>
41 LQQQ<br>
The epitope further involves amino acids 72-124 of cytokeratin 18, designated<br>
SEQ ID NO.4 and provided below.<br>
1 AGMGGIQNEK ETMQSLNDRL ASYLDRVRSL ETENRRLESK<br>
41 IREHLEKKGP QVR<br>
In some instances an appropriate three dimensional structure permitting<br>
interaction between cytokeratin K8 and cytokeratin K18 polypeptides may be<br>
needed to obtain optimal immunoreactivity. Hence, longer cytokeratin<br>
polypeptides can be used as antigens. For example, a cytokeratin K8 polypeptide<br>
having SEQ ID NO:5 can be used with an appropriate cytokeratin K18<br>
polypeptide to generate antibodies. SEQ ID NO:5 is as follows.<br><br><br><br>
Antigenic epitope "fragments" are also contemplated by the invention.<br>
Such fragments do not encompass a full-length cytokeratin but do encode an<br>
antigen that has similar or improved immunological properties relative to an<br>
antigenic epitope having SEQ ID NO:3-6. Thus, fragments of antigenic epitopes<br>
such as SEQ ID NO:3-6 may be as small as about 6 amino acids, about 9 amino<br>
acids, about 12 amino acids, about 15 amino acids, about 17 amino acids, about<br>
18 amino acids, about 20 amino acids, about 25 amino acids, about 30 amino<br>
acids or more. In general, a fragment antigenic epitope of the invention can have<br>
any upper size limit so long as it is has similar or immunological properties<br>
relative to an epitope form by a combination of any one of SEQ ED NO:3-6.<br>
The invention also contemplates a fusion protein comprising a<br>
combination of the SEQ ID NO:3 and the SEQ ID NO:4 peptide. Such a fusion<br>
protein links the two peptides together so that the peptides can more easily form<br>
the cancer associated epitope of the invention.<br>
Fusion polypeptides may generally be prepared using standard<br>
techniques, including chemical conjugation. A fusion polypeptide can also<br>
expressed as a recombinant polypeptide, allowing the production of increased<br>
levels, relative to a non-fused polypeptide, in an expression system. Briefly,<br>
DNA sequences encoding the polypeptide components may be assembled<br>
separately, and ligated into an appropriate expression vector. The 3' end of the<br>
DNA sequence encoding one polypeptide component is ligated, with or without a<br><br>
peptide linker, to the 5' end of a DNA sequence encoding the second polypeptide<br>
component so that the reading frames of the sequences are in phase. This permits<br>
translation into a single fusion polypeptide that retains the biological activity of<br>
both component polypeptides.<br>
A linker sequence may be employed to separate the first and second<br>
polypeptide components by a distance sufficient to ensure that each polypeptide<br>
folds into its secondary and tertiary structures. Such a linker can be a peptide,<br>
polypeptide, alkyl chain or other convenient spacer molecule.<br>
A polypeptide or peptide linker sequence is incorporated into the fusion<br>
polypeptide using standard techniques well known in the art. Suitable peptide<br>
linker sequences may be chosen based on the following factors: (1) their ability<br>
to adopt a flexible extended conformation; (2) their inability to adopt a secondary<br>
structure that could interact with functional epitopes on the first and second<br>
polypeptides; and (3) the lack of hydrophobic or charged residues that might<br>
react with the polypeptide functional epitopes. In some embodiments, peptide<br>
linker sequences contain Gly, Asn and Ser residues. Other near neutral amino<br>
acids, such as Thr and Ala may also be used in the linker sequence. Amino acid<br>
sequences that may be usefully employed as linkers include those disclosed in<br>
Maratea et al., Gene 40:39-46,1985; Murphy et al., Proc. Natl. Acad. Sci. USA<br>
83:8258-8262, 1986; U.S. Pat. No. 4,935,233 and U.S. Pat. No. 4,751,180. The<br>
linker sequence may generally be from 1 to about 50 amino acids in length.<br>
Linker sequences are generally not required when the first and second<br>
polypeptides have non-essential N-terminal amino acid regions that can be used<br>
to separate the functional domains and prevent steric interference.<br>
The fusion polypeptide can comprise the polypeptide epitope (e.g. SEQ<br>
ID NO:3 and SEQ ID NO:4 peptides) as described herein together with an<br>
unrelated immunogenic protein, such as an immunogenic protein capable of<br>
eliciting a recall response. Examples of such proteins include tetanus,<br>
tuberculosis and hepatitis proteins (see, for example, Stoute et al. New Engl. J<br>
Med., 336:86-91,1997).<br>
In one embodiment, a peptide or polypeptide that can facilitate<br>
development of an immune response against the SEQ ID NO:3 and SEQ ID<br><br>
N0:4 peptide epitope is used as the linker. Such an immunological fusion<br>
partner can be derived from a Mycobacterium sp. For example, the<br>
immunological fusion partner can be a Mycobacterium tuberculosis-derived<br>
Ral2 fragment. Ral2 compositions and methods for their use in enhancing the<br>
expression and/or immunogenicity of heterologous polynucleotide/polypeptide<br>
sequences is described in U.S. patent application Ser. No. 60/158,585, the<br>
disclosure of which is incorporated herein by reference in its entirety. Briefly,<br>
Ral2 refers to a polynucleotide region that is a subsequence of a Mycobacterium<br>
tuberculosis MTB32A nucleic acid. MTB32A is a serine protease of 32 KD<br>
molecular weight encoded by a gene in virulent and avirulent strains of M.<br>
tuberculosis. The nucleotide sequence and amino acid sequence of MTB32A<br>
have been described (for example, U.S. patent application Ser. No. 60/158,585;<br>
see also, Skeiky et al., Infection and Immun. (1999) 67:3998-4007, incorporated<br>
herein by reference). C-terminal fragments of the MTB32A coding sequence<br>
express at high levels and remain as a soluble polypeptides throughout the<br>
purification process. Moreover, Ral2 may enhance the immunogenicity of<br>
heterologous immunogenic polypeptides with which it is fused. One useful Ral2<br>
fusion polypeptide comprises a 14 KD C-terminal fragment corresponding to<br>
amino acid residues 192 to 323 of MTB32A. Other useful Ral2 polynucleotides<br>
generally comprise at least about 15 consecutive nucleotides, at least about 30<br>
nucleotides, at least about 60 nucleotides, at least about 100 nucleotides, at least<br>
about 200 nucleotides, or at least about 300 nucleotides that encode a portion of<br>
a Ral2 polypeptide.<br>
Ral2 polynucleotides may comprise a native sequence (i.e., an<br>
endogenous sequence that encodes a Ral2 polypeptide or a portion thereof) or<br>
may comprise a variant of such a sequence. Ral2 polynucleotide variants may<br>
contain one or more substitutions, additions, deletions and/or insertions such that<br>
the biological activity of the encoded fusion polypeptide is not substantially<br>
diminished, relative to a fusion polypeptide comprising a native Ral2<br>
polypeptide. Variants preferably exhibit at least about 70% identity, more<br>
preferably at least about 80% identity and most preferably at least about 90%<br><br>
identity to a polynucleotide sequence that encodes a native Ral2 polypeptide or a<br>
portion thereof.<br>
In another embodiment, an immunological fusion partner is derived from<br>
protein D, a surface protein of the gram-negative bacterium Haemophilus<br>
influenza B (WO 91/18926). Useful portions of protein D comprise<br>
approximately the first third of the protein (e.g., the first N-terminal 100-110<br>
amino acids). Moreover, such a protein D fusion partner may be lipidated.<br>
Within certain preferred embodiments, the first 109 residues of a Lipoprotein D<br>
fusion partner is included on the N-terminus to provide the polypeptide with<br>
additional exogenous T-cell epitopes and to increase the expression level in E.<br>
coli (thus functioning as an expression enhancer). The lipid tail ensures optimal<br>
presentation of the antigen to antigen presenting cells. Other fusion partners<br>
include the non-structural protein from influenzae virus, NS1 (hemaglutinin).<br>
Typically, the N-terminal 81 amino acids are used, although different fragments<br>
that include T-helper epitopes may be used.<br>
In another embodiment, the immunological fusion partner is the protein<br>
known as LYTA, or a portion thereof (preferably a C-terminal portion). LYTA is<br>
derived from Streptococcus pneumoniae, which synthesizes an N-acetyl-L-<br>
alanine amidase known as amidase LYTA (encoded by the LytA gene; Gene<br>
43:265-292,1986). LYTA is an autolysin that specifically degrades certain<br>
bonds in the peptidoglycan backbone. The C-terminal domain of the LYTA<br>
protein is responsible for the affinity to the choline or to some choline analogues<br>
such as DEAE. This property has been exploited for the development of E. coli<br>
C-LYTA expressing plasmids useful for expression of fusion proteins.<br>
Purification of hybrid proteins containing the C-LYTA fragment at the amino<br>
terminus has been described (see Biotechnology 10:795-798,1992). Within a<br>
preferred embodiment, a repeat portion of LYTA may be incorporated into a<br>
fusion polypeptide. A repeat portion is found in the C-terminal region starting at<br>
residue 178. A particularly preferred repeat portion incorporates residues 188-<br>
305.<br>
Another illustrative embodiment involves fusion polypeptides, and the<br>
polynucleotides encoding them, wherein the fusion partner comprises a targeting<br><br>
signal capable of directing a polypeptide to the endosomal/lysosomal<br>
compartment, as described in U.S. Pat. No. 5,633,234. An immunogenic<br>
polypeptide of the invention, when fused with this targeting signal, will associate<br>
more efficiently with MHC class II molecules and thereby provide enhanced in<br>
vivo stimulation of CD4.sup.+ T-cells specific for the polypeptide.<br>
Polypeptides and fusion proteins of the invention are prepared using any<br>
of a variety of well-known synthetic and/or recombinant techniques.<br>
Polypeptides and fusion proteins that are less than about 150 amino acids can be<br>
generated by synthetic means, using techniques well known to those of ordinary<br>
skill in the art. In one illustrative example, such polypeptides are synthesized<br>
using any of the commercially available solid-phase techniques, such as the<br>
Merrifield solid-phase synthesis method, where amino acids are sequentially<br>
added to a growing amino acid chain. See Merrifield, J. Am. Chem. Soc.<br>
85:2149-2146,1963. Equipment for automated synthesis of polypeptides is<br>
commercially available from suppliers such as Perkin Elmer/Applied<br>
BioSystems Division (Foster City, Calif), and may be operated according to the<br>
manufacturer's instructions.<br>
Small and large fusion proteins and polypeptide epitopes of the invention<br>
can be produced by any other method available to one of skill in the art. For<br>
example, the fusion proteins and polypeptide epitopes can be made<br>
recombinantly by inserting a nucleic acid encoding a selected fusion protein or<br>
polypeptide epitope into an expression vector using any of a variety of<br>
procedures. In general, a nucleic acid encoding the desired protein or polypeptide<br>
is inserted into an appropriate restriction endonuclease site(s) using techniques<br>
known in the art. See generally, Sambrook et al., 1989, Molecular Cloning, A<br>
Laboratory Manual, 2d ed., Cold Spring Harbor Laboratory Press, Cold Spring<br>
Harbor, N.Y.; Sambrook and Russell, Molecular Cloning: A Laboratory Manual,<br>
3rd edition (January 15, 2001) Cold Spring Harbor Laboratory Press, ISBN:<br>
0879695765; Ausubel et al., Current Protocols in Molecular Biology, Green<br>
Publishing Associates and Wiley Interscience, NY (1989)). Construction of<br>
suitable expression vectors containing a fusion protein or a polypeptide epitope<br>
employs standard ligation techniques that are known to the skilled artisan.<br><br>
The ligated nucleic acid sequences are operably linked to suitable<br>
transcriptional or translational regulatory elements that facilitate expression of<br>
the fusion proteins and polypeptide epitopes of the invention. The regulatory<br>
elements responsible for expression of proteins are located only 51 to the coding<br>
region for the polypeptide. Similarly, stop codons required to end translation and<br>
transcription termination signals are only present 3' to the nucleic acid sequence<br>
encoding the fusion protein or polypeptide epitope. After construction of a<br>
nucleic acid encoding the polypeptide of interest with the operably linked<br>
regulatory elements, this expression cassette can be introduced into a host cell<br>
and the encoded polypeptide can be expressed.<br>
In general, polypeptide compositions (including fusion polypeptides) of<br>
the invention are isolated. An "isolated" polypeptide is one that is removed from<br>
its original environment. For example, a naturally-occurring protein or<br>
polypeptide is isolated if it is separated from some or all of the coexisting<br>
materials in the natural system. Such polypeptides can also be purified. For<br>
example, the polypeptide epitopes and fusion proteins can be at least about 90%<br>
pure, or at least about 95% pure or at least about 99% pure.<br>
Antibodies and Binding Entities<br>
The cytokeratin epitopes of the invention are displayed in a uniform<br>
punctate pattern on the surface of viable carcinoma and adenocarcinoma cells.<br>
Immunohistological studies have demonstrated that the cancer associated epitope<br>
of the invention, in contrast to normal cytokeratin 8 and 18, can be used to<br>
differentiate between malignant and normal colon epithelia, and between colon<br>
cancer metastasis in the liver and surrounding normal hepatocytes. In addition,<br>
the cancer associated epitope of the invention is associated with the membranes<br>
of proliferating cells within the malignant area of biopsies, while resting cells<br>
had a filamentous pattern when stained for the epitope.<br>
The invention provides antibody preparations and binding entities<br>
directed against the epitopes of the invention, for example, antibodies or binding<br>
entities capable of binding an antigenic mixture of at least one peptide from<br>
cytokeratin K8 and at least one peptide from cytokeratin Kl8. Examples of<br><br>
peptides from cytokeratin K8 include SEQ ID NO:3 and SEQ ID NO:5.<br>
Examples of peptides from cytokeratin K18 include SEQ ED NO:4, and SEQ ID<br>
NO:6.<br>
In one embodiment, the antibody or binding entity can include a<br>
polypeptide comprising any one of SEQ ID NO:7-35,47-49. In some<br>
embodiments, antibodies and binding entities include a polypeptide consisting<br>
essentially of any one of SEQ ID NO:21-35, 47-49. In other embodiments,<br>
antibodies and binding entities include a polypeptide consisting essentially of<br>
any one of SEQ ID NO:8, 10, 12,15, 17, 19, 22, 24, 27, 29 or 32. In another<br>
embodiment, the invention is directed to a binding entity polypeptide comprising<br>
any combination of SEQ ID NO:7-33,47-49, wherein the polypeptide that can<br>
bind an epitope of the invention.<br>
The invention also provides nucleic acids encoding antibody-like<br>
polypeptides of the invention. In one embodiment, the nucleic acid encodes a<br>
polypeptide comprising any one of SEQ ID NO:7-35, 47-49 wherein such a<br>
nucleic acid encodes a polypeptide that can bind an epitope of the invention. In<br>
another embodiment, the nucleic acid encodes a combination of two or more of<br>
SEQ ID NO:7-33, 47-49 wherein such a nucleic acid encodes a binding entity<br>
polypeptide that can bind an epitope of the invention. Preferred nucleic acids<br>
encode a polypeptide consisting essentially of any one of SEQ ID NO:21-33 or<br>
any one of SEQ ID NO:8,10, 12,15, 17, 19, 22, 24,27, 29 or 32. Other nucleic<br>
acids of the invention include nucleotide sequences SEQ ID NO:36-39.<br>
The invention also provides antibodies made by available procedures that<br>
can bind an epitope of the invention.<br>
Antibody molecules belong to a family of plasma proteins called<br>
immunoglobulins, whose basic building block, the immunoglobulin fold or<br>
domain, is used in various forms in many molecules of the immune system and<br>
other biological recognition systems. A standard antibody is a tetrameric<br>
structure consisting of two identical immunoglobulin heavy chains and two<br>
identical light chains and has a molecular weight of about 150,000 daltons.<br>
The heavy and light chains of an antibody consist of different domains.<br>
Each light chain has one variable domain (VL) and one constant domain (CL),<br><br>
while each heavy chain has one variable domain (VH) and three or four constant<br>
domains (CH). See, e.g., Alzari, P. N., Lascombe, M.-B. &amp; Poljak, R. J. (1988)<br>
Three-dimensional structure of antibodies. Annu. Rev. Immunol. 6, 555-580.<br>
Each domain, consisting of about 110 amino acid residues, is folded into a<br>
characteristic β-sandwich structure formed from two β-sheets packed against<br>
each other, the immunoglobulin fold. The VH and VL domains each have three<br>
complementarity determining regions (CDR1-3) that are loops, or turns,<br>
connecting β-strands at one end of the domains. The variable regions of both the<br>
light and heavy chains generally contribute to antigen specificity, although the<br>
contribution of the individual chains to specificity is not always equal. Antibody<br>
molecules have evolved to bind to a large number of molecules by using six<br>
randomized loops (CDRs).<br>
Depending on the amino acid sequences of the constant domain of their<br>
heavy chains, immunoglobulins can be assigned to different classes. There are at<br>
least five (5) major classes of immunoglobulins: IgA, IgD, IgE, IgG and IgM,<br>
and several of these may be further divided into subclasses (isotypes), e.g. IgG-1,<br>
IgG-2, IgG-3 and IgG-4; IgA-1 and IgA-2. The heavy chains constant domains<br>
that correspond to the different classes of immunoglobulins are called alpha (a),<br>
delta (δ), epsilon (ε), gamma (γ) and mu (µ), respectively. The light chains of<br>
antibodies can be assigned to one of two clearly distinct types, called kappa (K)<br>
and lambda (λ), based on the amino sequences of their constant domain. The<br>
subunit structures and three-dimensional configurations of different classes of<br>
immunoglobulins are well known.<br>
The term "variable" in the context of variable domain of antibodies,<br>
refers to the fact that certain portions of the variable domains differ extensively<br>
in sequence among antibodies. The variable domains are for binding and<br>
determine the specificity of each particular antibody for its particular antigen.<br>
However, the variability is not evenly distributed through the variable domains<br>
of antibodies. It is concentrated in three segments called complementarity<br>
determining regions (CDRs) also known as hypervariable regions both in the<br>
light chain and the heavy chain variable domains.<br><br>
The more highly conserved portions of variable domains are called the<br>
framework (FR). The variable domains of native heavy and light chains each<br>
comprise four FR regions, largely a adopting a β-sheet configuration, connected<br>
by three CDRs, which form loops connecting, and in some cases forming part of,<br>
the P-sheet structure. The CDRs in each chain are held together in close<br>
proximity by the FR regions and, with the CDRs from the other chain, contribute<br>
to the formation of the antigen-binding site of antibodies. The constant domains<br>
are not involved directly in binding an antibody to an antigen, but exhibit various<br>
effector functions, such as participation of the antibody in antibody-dependent<br>
cellular toxicity.<br>
An antibody that is contemplated for use in the present invention thus can<br>
be in any of a variety of forms, including a whole immunoglobulin, an antibody<br>
fragment such as Fv, Fab, and similar fragments, a single chain antibody which<br>
includes the variable domain complementarity determining regions (CDR), and<br>
the like forms, all of which fall under the broad term "antibody", as used herein.<br>
The present invention contemplates the use of any specificity of an antibody,<br>
polyclonal or monoclonal, and is not limited to antibodies that recognize and<br>
immunoreact with a specific antigen. In preferred embodiments, in the context<br>
of both the therapeutic and screening methods described below, an antibody or<br>
fragment thereof is used that is immunospecific for an antigen or epitope of the<br>
invention. In some embodiments, the antibody is not the COU-1 antibody.<br>
The term "antibody fragment" refers to a portion of a full-length<br>
antibody, generally the antigen binding or variable region. Examples of antibody<br>
fragments include Fab, Fab', F(ab') 2 and Fv fragments. Papain digestion of<br>
antibodies produces two identical antigen binding fragments, called the Fab<br>
fragment, each with a single antigen binding site, and a residual "Fc" fragment,<br>
so-called for its ability to crystallize readily. Pepsin treatment yields an F(ab')2<br>
fragment that has two antigen binding fragments that are capable of cross-linking<br>
antigen, and a residual other fragment (which is termed pFc'). Additional<br>
fragments can include diabodies, linear antibodies, single-chain antibody<br>
molecules, and multispecific antibodies formed from antibody fragments. As<br><br>
used herein, "functional fragment" with respect to antibodies, refers to Fv, F(ab)<br>
and F(ab')2 fragments.<br>
Antibody fragments contemplated by the invention are therefore not full-<br>
length antibodies but do have similar or improved immunological properties<br>
relative to an antibody such as the COU-1 antibody. Thus, fragments of the<br>
COU-1 antibody and/or fragments of polypeptides having any one of SEQ ID<br>
NO:7-35 antibody are contemplated by the invention. Such antibody fragments<br>
may be as small as about 4 amino acids, 5 amino acids, 6 amino acids, 7 amino<br>
acids, 9 amino acids, about 12 amino acids, about 15 amino acids, about 17<br>
amino acids, about 18 amino acids, about 20 amino acids, about 25 amino acids,<br>
about 30 amino acids or more.<br>
In general, an antibody fragment of the invention can have any upper size<br>
limit so long as it is has similar or immunological properties relative to antibody<br>
that binds with specificity to an epitope formed by a combination of any one of<br>
SEQ ID NO:3-6. Such a reference antibody can be the COU-1 antibody. For<br>
example, binding entities and light chain antibody fragments can have less than<br>
about 200 amino acids, less than about 175 amino acids, less than about 150<br>
amino acids, or less than about 120 amino acids if the antibody fragment is<br>
related to a light chain antibody subunit. Moreover, binding entities and heavy<br>
chain antibody fragments can have less than about 425 amino acids, less than<br>
about 400 amino acids, less than about 375 amino acids, less than about 350<br>
amino acids, less than about 325 amino acids or less than about 300 amino acids<br>
if the antibody fragment is related to a heavy chain antibody subunit.<br>
Antibody fragments retain some ability to selectively bind with its<br>
antigen, epitope or receptor. Some types of antibody fragments are defined as<br>
follows:<br>
(1)	Fab is the fragment that contains a monovalent antigen-binding<br>
fragment of an antibody molecule. A Fab fragment can be produced by digestion<br>
of whole antibody with the enzyme papain to yield an intact light chain and a<br>
portion of one heavy chain.<br>
(2)	Fab' is the fragment of an antibody molecule can be obtained by<br>
treating whole antibody with pepsin, followed by reduction, to yield an intact<br><br>
light chain and a portion of the heavy chain. Two Fab' fragments are obtained<br>
per antibody molecule. Fab' fragments differ from Fab fragments by the addition<br>
of a few residues at the carboxyl terminus of the heavy chain CHI domain<br>
including one or more cysteines from the antibody hinge region.<br>
(3)	(Fab')2 is the fragment of an antibody that can be obtained by<br>
treating whole antibody with the enzyme pepsin without subsequent reduction.<br>
F(ab')2 is a dimer of two Fab' fragments held together by two disulfide bonds.<br>
(4)	Fv is the minimum antibody fragment that contains a complete<br>
antigen recognition and binding site. This region consists of a dimer of one<br>
heavy and one light chain variable domain in a right, non-covalent association<br>
(VH -VL dimer). It is in this configuration that the three CDRs of each variable<br>
domain interact to define an antigen binding site on the surface of the VH -VL<br>
dimer. Collectively, the six CDRs confer antigen binding specificity to the<br>
antibody. However, even a single variable domain (or half of an Fv comprising<br>
only three CDRs specific for an antigen) has the ability to recognize and bind<br>
antigen, although at a lower affinity than the entire binding site.<br>
(5)	Single chain antibody ("SCA"), defined as a genetically<br>
engineered molecule containing the variable region of the light chain, the<br>
variable region of the heavy chain, linked by a suitable polypeptide linker as a<br>
genetically fused single chain molecule. Such single chain antibodies are also<br>
referred to as "single-chain Fv" or "sFv" antibody fragments. Generally, the Fv<br>
polypeptide further comprises a polypeptide linker between the VH and VL<br>
domains that enables the sFv to form the desired structure for antigen binding.<br>
For a review of sFv see Pluckthun in The Pharmacology of Monoclonal<br>
Antibodies, vol. 113. Rosenburg and Moore eds. Springer-Verlag, N.Y., pp. 269-<br>
315 (1994).<br>
The term "diabodies" refers to a small antibody fragments with two<br>
antigen-binding sites, which fragments comprise a heavy chain variable domain<br>
(VH) connected to a light chain variable domain (VL) in the same polypeptide<br>
chain (VH-VL). By using a linker that is too short to allow pairing between the<br>
two domains on the same chain, the domains are forced to pair with the<br>
complementary domains of another chain and create two antigen-binding sites.<br><br>
Diabodies are described more fully in, for example, EP 404,097; WO 93/11161,<br>
and Hollinger et al., Proc. Natl. Acad Sci. USA 90: 6444-6448 (1993).<br>
Methods for the preparation of polyclonal antibodies are availableto<br>
those skilled in the art. See, for example, Green, et al., Production of Polyclonal<br>
Antisera, in: Immunochermcal Protocols (Manson, ed.), pages 1-5 (Humana<br>
Press); Coligan, et al., Production of Polyclonal Antisera in Rabbits, Rats Mice<br>
and Hamsters, in: Current Protocols in Immunology, section 2.4.1 (1992), which<br>
are hereby incorporated by reference.<br>
The preparation of monoclonal antibodies likewise is conventional. See,<br>
for example, Kohler &amp; Milstein, Nature, 256:495 (1975); Coligan, et al., sections<br>
2.5.1-2.6.7; and Harlow, et al., in: Antibodies: A Laboratory Manual, page 726<br>
(Cold Spring Harbor Pub. (1988)), which are hereby incorporated by reference.<br>
Monoclonal antibodies can be isolated and purified from hybridoma cultures by<br>
a variety of well-established techniques. Such isolation techniques include<br>
affinity chromatography with Protein-A Sepharose, size-exclusion<br>
chromatography, and ion-exchange chromatography. See, e.g., Coligan, et al.,<br>
sections 2.7.1-2.7.12 and sections 2.9.1-2.9.3; Barnes, et al., Purification of<br>
Immunoglobulin G (IgG), in: Methods in Molecular Biology. Vol. 10, pages 79-<br>
104 (Humana Press (1992).<br>
Methods of in vitro and in vivo manipulation of monoclonal antibodies<br>
are well known to those skilled in the art. For example, the monoclonal<br>
antibodies to be used in accordance with the present invention may be made by<br>
the hybridoma method first described by Kohler and Milstein, Nature 256, 495<br>
(1975), or may be made by recombinant methods, e.g., as described in U.S. Pat.<br>
No. 4,816,567. The monoclonal antibodies for use with the present invention<br>
may also be isolated from phage antibody libraries using the techniques<br>
described in Clackson et al. Nature 352:624-628 (1991), as well as in Marks et<br>
al., J. Mol Biol. 222: 581-597 (1991). Another method involves humanizing a<br>
monoclonal antibody by recombinant means to generate antibodies containing<br>
human specific and recognizable sequences. See, for review, Holmes, et al., J.<br>
Immunol., 158:2192-2201 (1997) and Vaswani, et al., Annals Allergy, Asthma &amp;<br>
Immunol., 81:105-115 (1998).<br><br>
The term "monoclonal antibody" as used herein refers to an antibody<br>
obtained from a population of substantially homogeneous antibodies, i.e., the<br>
individual antibodies comprising the population are identical except for passible<br>
naturally occurring mutations that may be present in minor amounts. Monoclonal<br>
antibodies are highly specific, being directed against a single antigenic site.<br>
Furthermore, in contrast to conventional polyclonal antibody preparations that<br>
typically include different antibodies directed against different determinants<br>
(epitopes), each monoclonal antibody is directed against a single determinant on<br>
the antigen. In additional to their specificity, the monoclonal antibodies are<br>
advantageous in that they are synthesized by the hybridoma culture,<br>
uncontaminated by other immunoglobulins. The modifier "monoclonal" indicates<br>
the character of the antibody indicates the character of the antibody as being<br>
obtained from a substantially homogeneous population of antibodies, and is not<br>
to be construed as requiring production of the antibody by any particular method.<br>
The monoclonal antibodies herein specifically include "chimeric"<br>
antibodies (immunoglobulins) in which a portion of the heavy and/or light chain<br>
is identical with or homologous to corresponding sequences in antibodies<br>
derived from a particular species or belonging to a particular antibody class or<br>
subclass, while the remainder of the chain(s) is identical with or homologous to<br>
corresponding sequences in antibodies derived from another species or belonging<br>
to another antibody class or subclass, as well as fragments of such antibodies, so<br>
long as they exhibit the desired biological activity (U.S. Pat. No. 4,816,567);<br>
Morrison et al. Proc. Natl. Acad Sci. 81, 6851-6855 (1984).<br>
Methods of making antibody fragments are also known in the art (see for<br>
example, Harlow and Lane, Antibodies: A Laboratory Manual. Cold Spring<br>
Harbor Laboratory, New York, (1988), incorporated herein by reference).<br>
Antibody fragments of the present invention can be prepared by proteolytic<br>
hydrolysis of the antibody or by expression in E. coli of DNA encoding the<br>
fragment. Antibody fragments can be obtained by pepsin or papain digestion of<br>
whole antibodies conventional methods. For example, antibody fragments can<br>
be produced by enzymatic cleavage of antibodies with pepsin to provide a 5S<br>
fragment denoted F(ab')2. This fragment can be further cleaved using a thiol<br><br>
reducing agent, and optionally a blocking group for the sulfhydryl groups<br>
resulting from cleavage of disulfide linkages, to produce 3.5S Fab' monovalent<br>
fragments. Alternatively, an enzymatic cleavage using pepsin produces two<br>
monovalent Fab' fragments and an Fc fragment directly. These methods are<br>
described, for example, in US Patents No. 4,036,945 and No. 4,331,647, and<br>
references contained therein. These patents are hereby incorporated in their<br>
entireties by reference.<br>
Other methods of cleaving antibodies, such as separation of heavy chains<br>
to form monovalent light-heavy chain fragments, further cleavage of fragments,<br>
or other enzymatic, chemical, or genetic techniques may also be used, so long as<br>
the fragments bind to the antigen that is recognized by the intact antibody. For<br>
example, Fv fragments comprise an association of VH and VL chains. This<br>
association may be noncovalent or the variable chains can be linked by an<br>
intermolecular disulfide bond or cross-linked by chemicals such as<br>
glutaraldehyde. Preferably, the Fv fragments comprise VH and VL chains<br>
connected by a peptide linker. These single-chain antigen binding proteins (sFv)<br>
are prepared by constructing a structural gene comprising DNA sequences<br>
encoding the VH and VL domains connected by an oligonucleotide. The<br>
structural gene is inserted into an expression vector, which is subsequently<br>
introduced into a host cell such as E. coli. The recombinant host cells synthesize<br>
a single polypeptide chain with a linker peptide bridging the two V domains.<br>
Methods for producing sFvs are described, for example, by Whitlow, et al.,<br>
Methods: a Companion to Methods in Enzvmologv. Vol. 2, page 97 (1991);<br>
Bird, et al., Science 242:423-426 (1988); Ladner, et al, US Patent No. 4,946,778;<br>
and Pack, et al., Bio/Technology 11:1271-77 (1993).<br>
Another form of an antibody fragment is a peptide coding for a single<br>
complementarity-deteimining region (CDR). CDR peptides ("minimal<br>
recognition units") are often involved in antigen recognition and binding. CDR<br>
peptides can be obtained by cloning or constructing genes encoding the CDR of<br>
an antibody of interest. Such genes are prepared, for example, by using the<br>
polymerase chain reaction to synthesize the variable region from RNA of<br><br>
antibody-producing cells. See, for example, Larrick, et al., Methods: a<br>
Companion to Methods in Enzvmology, Vol. 2, page 106 (1991).<br>
The invention contemplates human and humanized forms of non-human<br>
(e.g. murine) antibodies. Such humanized antibodies are chimeric<br>
immunoglobulins, immunoglobulin chains or fragments thereof (such as Fv, Fab,<br>
Fab', F(ab')2 or other antigen-binding subsequences of antibodies) that contain<br>
minimal sequence derived from non-human immunoglobulin. For the most part,<br>
humanized antibodies are human immunoglobulins (recipient antibody) in which<br>
residues from a complementary determining region (CDR) of the recipient are<br>
replaced by residues from a CDR of a nonhuman species (donor antibody) such<br>
as mouse, rat or rabbit having the desired specificity, affinity and capacity.<br>
In some instances, Fv framework residues of the human immunoglobulin<br>
are replaced by corresponding non-human residues. Furthermore, humanized<br>
antibodies may comprise residues that are found neither in the recipient antibody<br>
nor in the imported CDR or framework sequences. These modifications are made<br>
to further refine and optimize antibody performance. In general, humanized<br>
antibodies will comprise substantially all of at least one, and typically two,<br>
variable domains, in which all or substantially all of the CDR regions correspond<br>
to those of a non-human immunoglobulin and all or substantially all of the FR<br>
regions are those of a human immunoglobulin consensus sequence. The<br>
humanized antibody optimally also will comprise at least a portion of an<br>
immunoglobulin constant region (Fc), typically that of a human<br>
immunoglobulin. For further details, see: Jones et al., Nature 321, 522-525<br>
(1986); Reichmann et al., Nature 332,323-329 (1988); Presta, Curr. Op. Struct.<br>
Biol. 2, 593-596 (1992); Holmes, et al., J. Immunol., 158:2192-2201 (1997) and<br>
Vaswani, et al., Annals Allergy, Asthma &amp; Immunol, 81:105-115 (1998).<br>
While standardized procedures are available to generate antibodies, the<br>
size of antibodies, the multi-stranded structure of antibodies and the complexity<br>
of six binding loops present in antibodies constitute a hurdle to the improvement<br>
and the manufacture of large quantities of antibodies. Hence, the invention<br>
further contemplates using binding entities, which comprise polypeptides that<br>
can recognize and bind to the epitope of the invention.<br><br>
The invention is therefore further directed to antibodies and other binding<br>
entities that can bind the cancer-associated epitope of the invention. In some<br>
embodiments, the antibodies and binding entities have SEQ ID NO.7-33: "The<br>
sequences for SEQ ID NO.7-33 are provided below.<br><br><br><br>
Nucleic acids encoding peptides SEQ ID NO:7-33 were isolated from cells that<br>
secrete the COU-1 antibody. While not all of the polypeptides encoded by the nucleic<br>
acids isolated in this screen could bind the cancer-associated epitope, peptides SEQ ID<br>
NO:7-33 were shown to play a role in binding by phage display and other experiments.<br>
Moreover, several differences were found in similar regions of different antibody<br>
fragment clones. For example, variable light chain CDR1 fragments that were isolated<br>
had RASQSVSSSYLA (SEQ ID NO:15) as well as KSSQSLLYSSNNKNYLA (SEQ<br>
ID NO:27). Similarly, variable light chain CDR2 fragments isolated had DASNRAT<br>
(SEQ ID NO: 17), GASSRAT (SEQ ID NO:22) or WASTRES (SEQ ID NO:29).<br>
Moreover, variable light chain CDR3 fragments isolated had QQYGNSPPYT (SEQ ID<br>
NO:24) or QQYYSTPPM (SEQ ID NO:32). Hence, not all clones were identical.<br>
A number of proteins can serve as protein scaffolds to which binding domains<br>
(e.g. any of the SEQ ID NO:7-33,47-49 peptides or variants thereof) can be attached.<br>
The binding domains bind or interact with the cancer-associated epitope of the<br>
invention while the protein scaffold merely holds and stabilizes the binding domains so<br>
that they can bind. A number of protein scaffolds can be used. For example, phage<br>
capsid proteins can be used. Review in Clackson &amp; Wells, Trends Biotechnol. 12:173-<br>
184 (1994). Indeed, such phage capsid proteins were used as described herein to screen<br>
for the SEQ ID NO:7-33 peptides (see Examples). Phage capsid proteins have also<br>
been used as scaffolds for displaying random peptide sequences, including bovine<br>
pancreatic trypsin inhibitor (Roberts et al., PNAS 89:2429-2433 (1992)), human growth<br>
hormone (Lowman et al., Biochemistry 30:10832-10838 (1991)), Venturini et al.,<br>
Protein Peptide Letters 1:70-75 (1994)), and the IgG binding domain of Streptococcus<br>
(O'Neil et al., Techniques in Protein Chemistry V (Crabb, L,. ed.) pp. 517-524,<br>
Academic Press, San Diego (1994)). These scaffolds have displayed a single<br>
randomized loop or region that can be modified to include the binding domains<br>
provided herein (e.g. SEQ ID NO:7-33, 47-49).<br><br>
Researchers have also used the small 74 amino acid α-amylase inhibitor<br>
Tendamistat as a presentation scaffold on the filamentous phage M13. McConnell, S.<br>
J., &amp; Hoess, R. H., J.Mol. Biol. 250:460-470 (1995). Tendamistat is a fcaheet protein<br>
from Streptomyces tendae. It has a number of features that make it an attractive scaffold<br>
for binding peptides, including its small size, stability, and the availability of high<br>
resolution NMR and X-ray structural data. The overall topology of Tendamistat is<br>
similar to that of an immunoglobulin domain, with two β-sheets connected by a series<br>
of loops. In contrast to immunoglobulin domains, the β-sheets of Tendamistat are held<br>
together with two rather than one disulfide bond, accounting for the considerable<br>
stability of the protein. By analogy with the CDR loops found in immunoglobulins, the<br>
loops of Tendamistat may serve a similar function and can be easily randomized by in<br>
vitro mutagenesis. Tendamistat, however, is derived from Streptomyces tendae and<br>
may be antigenic in humans. Its small size, however, may reduce or inhibit its<br>
antigenicity.<br>
Fibronectin type III domain has also been used as a protein scaffold to which<br>
binding entities can be attached. Sequences, vectors and cloning procedures for using<br>
such a fibronectin type III domain as a protein scaffold for binding entities (e.g. CDR<br>
peptides) are provided, for example, in U.S. Patent Application Publication<br>
20020019517. Fibronectin is a large protein that plays an essential role in the<br>
formation of extracellular matrix and cell-cell interactions. Fibronectin consists of<br>
many repeats of three types (I, II and III) of small domains. Baron, M., Norman, D. G.<br>
&amp; Campbell, I. D. (1991) Protein modules Trends Biochem. Sci. 16,13-17.<br>
Fibronectin type H1 is part of a large subfamily (Fn3 family or s-type Ig family) of the<br>
immunoglobulin superfamily. The Fn3 family includes cell adhesion molecules, cell<br>
surface hormone and cytokine receptors, chaperonins, and carbohydrate-binding<br>
domains. For reviews, see Bork, P. &amp; Doolittle, R. F. (1992) Proposed acquisition of<br>
an animal protein domain by bacteria. Proc. Natl. Acad. Sci. USA 89, 8990-8994;<br>
Jones, E. Y. (1993) The immunoglobulin superfamily Curr. Opinion Struct. Biol. 3,<br>
846-852; Bork, P., Horn, L. &amp; Sander, C. (1994) The immunoglobulin fold. Structural<br>
classification, sequence patterns and common core. J. Mol. Biol. 242, 309-320;<br>
Campbell, LD.&amp; Spitzfaden, C. (1994) Building proteins with fibronectin type H1<br>
modules Structure 2,233-337; Harpez, Y. &amp; Chothia, C. (1994) Many of the<br><br>
immunoglobulin superfamily domains in cell adhesion molecules and surface receptors<br>
belong to a new structural set which is close to that containing variable domains J. Mol.<br>
Biol. 238, 528-539.<br>
In the immune system, specific antibodies are selected and amplified from a<br>
large library (affinity maturation). The combinatorial techniques employed in immune<br>
cells can be mimicked by mutagenesis and generation of combinatorial libraries of<br>
binding entities. Binding entities, antibody fragments and antibodies therefore can be<br>
generated through display-type technologies, including, without limitation, phage<br>
display, retroviral display, ribosomal display, and other techniques, using techniques<br>
well known in the art and the resulting molecules can be subjected to additional<br>
maturation, such as affinity maturation, as such techniques are well known in the art.<br>
Wright and Harris, supra., Hanes and Plucthau PNAS USA 94:4937-4942 (1997)<br>
(ribosomal display), Parmley and Smith Gene 73:305-318 (1988) (phage display), Scott<br>
T1BS 17:241-245 (1992), Cwirla et al. PNAS USA 87:6378-6382 (1990), Russel et al.<br>
Nucl. Acids Research 21:1081-1085 (1993), Hoganboom et al. Immunol. Reviews<br>
130:43-68 (1992), Chiswell and McCafferty T1BTECH 10:80-84 (1992), and U.S. Pat.<br>
No. 5,733,743.<br>
The invention therefore also provides methods of mutating antibodies to<br>
optimize their affinity, selectivity, binding strength and/or other desirable properties. A<br>
mutant antibody refers to an amino acid sequence variant of an antibody. In general,<br>
one or more of the amino acid residues in the mutant antibody is different from what is<br>
present in the reference antibody. Such mutant antibodies necessarily have less than<br>
100% sequence identity or similarity with the reference amino acid sequence. In<br>
general, mutant antibodies have at least 75% amino acid sequence identity or similarity<br>
with the amino acid sequence of either the heavy or light chain variable domain of the<br>
reference antibody. Preferably, mutant antibodies have at least 80%, more preferably at<br>
least 85%, even more preferably at least 90%, and most preferably at least 95% amino<br>
acid sequence identity or similarity with the amino acid sequence of either the heavy or<br>
light chain variable domain of the reference antibody. One method of mutating<br>
antibodies involves affinity maturation using phage display.<br>
For example, affinity maturation using phage display can be utilized as one<br>
method for generating mutant antibodies. Affinity maturation using phage display<br><br>
refers to a process described in Lowman et al., Biochemistry 30(45): 10832-10838<br>
(1991), see also Hawkins et al., J. Mol Biol. 254: 889-896 (1992). While not strictly<br>
limited to the following description, this process can be described briefly as involving<br>
mutation of several antibody hypervariable regions in a number of different sites with<br>
the goal of generating all possible amino acid substitutions at each site. The antibody<br>
mutants thus generated are displayed in a monovalent fashion from filamentous phage<br>
particles as fusion proteins. Fusions are generally made to the gene IE product of M13.<br>
The phage expressing the various mutants can be cycled through several rounds of<br>
selection for the trait of interest, e.g. binding affinity or selectivity. The mutants of<br>
interest are isolated and sequenced. Such methods are described in more detail in U.S.<br>
Patent 5,750,373, U.S. Patent 6,290,957 and Cunningham, B. C. et al., EMBO J.<br>
13(11), 2508-2515 (1994).<br>
In one embodiment, the invention provides methods of manipulating<br>
antibody polypeptides or antibody-encoding nucleic acids to generate antibodies<br>
and antibody fragments with improved binding properties that recognize the<br>
same epitope as COU-1 antibodies.<br>
Such methods of mutating portions of a COU-1 antibody involve fusing a<br>
nucleic acid encoding a polypeptide having any one of SEQ ED NO:7-35 or any one of<br>
SEQ ID NO:8,10, 12, 15, 17,19, 22, 24, 27, 29, 32, 47, 48 or 49 to a nucleic acid<br>
encoding a phage coat protein to generate a recombinant nucleic acid encoding a fusion<br>
protein, mutating the recombinant nucleic acid encoding the fusion protein to generate a<br>
mutant nucleic acid encoding a mutant fusion protein, expressing the mutant fusion<br>
protein on the surface of a phage and selecting phage that bind to an epitope of the<br>
invention.<br>
In one embodiment, the method involves fusing a nucleic acid encoding a<br>
polypeptide having any combination of SEQ ID NO:7-35 or any combination of SEQ<br>
ID NO:8,10,12,15,17,19,22,24,27, 29,32, 47,48 or 49 to a nucleic acid encoding<br>
a phage coat protein to generate a recombinant nucleic acid encoding a fusion protein,<br>
mutating the recombinant nucleic acid encoding the fusion protein to generate a mutant<br>
nucleic acid encoding a mutant fusion protein, expressing the mutant fusion protein on<br>
the surface of a phage and selecting phage that bind to an epitope of the invention.<br><br>
In another embodiment, the method involves fusing a nucleic acid encoding a<br>
polypeptide having each one of SEQ ID NO:26, 15, 21, 22, 23, 24 and 25 to a nucleic<br>
acid encoding a phage coat protein to generate a recombinant nucleic acid encoding a<br>
fusion protein, mutating the recombinant nucleic acid encoding the fusion protein to<br>
generate a mutant nucleic acid encoding a mutant fusion protein, expressing the mutant<br>
fusion protein on the surface of a phage and selecting phage that bind to an epitope of<br>
the invention.<br>
In another embodiment, the method involves fusing a nucleic acid encoding<br>
a polypeptide having each one of SEQ ID NO:26, 27, 28, 29, 30, 31, 32 and 33 to a<br>
nucleic acid encoding a phage coat protein to generate a recombinant nucleic acid<br>
encoding a fusion protein, mutating the recombinant nucleic acid encoding the fusion<br>
protein to generate a mutant nucleic acid encoding a mutant fusion protein, expressing<br>
the mutant fusion protein on the surface of a phage and selecting phage that bind to an<br>
epitope of the invention.<br>
In another embodiment, the method involves fusing a nucleic acid<br>
encoding a polypeptide having SEQ ID NO.34 or SEQ ID NO:35 to a nucleic acid<br>
encoding a phage coat protein to generate a recombinant nucleic acid encoding a<br>
fusion protein, mutating the recombinant nucleic acid encoding the fusion protein<br>
to generate a mutant nucleic acid encoding a mutant fusion protein, expressing the<br>
mutant fusion protein on the surface of a phage and selecting phage that bind to<br>
an epitope of the invention. SEQ ID NO:34 and 35 encode useful variable light<br>
chains that may bind to epitopes of the invention. SEQ ID NO:34 is provided<br>
below.<br><br><br>
The method can also involve fusing a nucleic acid comprising a variable heavy<br>
or light chain relating to COU-1 (e.g. any one of SEQ ID NO:36-39) to a nucleic acid<br>
encoding a phage coat protein to generate a recombinant nucleic acid encoding a fusion<br>
protein, mutating the recombinant nucleic acid encoding the fusion protein to generate a<br>
mutant nucleic acid encoding a mutant fusion protein, expressing the mutant fusion<br>
protein on the surface of a phage and selecting phage that bind to an epitope of the<br>
invention.<br>
Hence, the invention is directed to a nucleic acid encoding a variable heavy<br>
chain relating to COU-1, for example, SEQ ID NO:36 provided below.<br><br><br>
In another embodiment the invention is directed to a nucleic acid<br>
encoding a variable light chain relating to COU-1, for example, SEQ ID NO:38<br>
(also called L8) provided below.<br><br>
In another embodiment the invention is directed to a nucleic acid encoding<br>
a variable light chain related to COU-1, for example, SEQ ID NO:39 (also called<br>
T5).<br><br>
Such methods can further include constructing a replicable expression<br>
vector containing a nucleic acid encoding a polypeptide of the invention, for<br>
example, a polypeptide comprising any one of SEQ ID NO:7-35, or a nucleic<br>
acid comprising any one of SEQ ID NO:36-39. The nucleic acid can also encode<br>
a fusion protein comprising a polypeptide of the invention (e.g. any of SEQ ID<br>
NO:7-35) and at least a portion of a natural or wild-type phage coat protein. The<br>
expression vector can also have a transcription regulatory element operably<br>
linked to the nucleic acids encoding the fusion protein. The vector is mutated at<br><br>
one or more selected positions within the nucleic acid encoding the antibody<br>
polypeptide to form a family or "library" of plasmids containing related nucleic<br>
acids, each encoding a slightly different antibody polypeptide. Suitable hest<br>
cells are transformed with the family of plasmids. The transformed host cells are<br>
infected with a helper phage having a gene encoding the phage coat protein and<br>
the transformed, infected host cells are cultured under conditions suitable for<br>
forming recombinant phagemide particles. Each recombinant phagemid displays<br>
approximately one copy of the fusion protein on the surface of the phagemid<br>
particle. To screen the phagemids, phagemid particles are contacted with an<br>
epitope or antigen of the invention. Phagemid particles that bind are separated<br>
from those that do not bind the epitope or antigen. Preferably, further rounds of<br>
selection are performed by separately cloning phagemids with acceptable binding<br>
properties and re-testing their binding affinity one or more times. The plasmids<br>
from phagemid particles that appropriately bind the epitope or antigen can also<br>
be isolated, cloned and even mutated again to further select for the antibody<br>
properties desired, e.g. with good binding affinity.<br>
The method is applicable to polypeptide complexes that are composed of<br>
more than one subunit polypeptides. In this case, a nucleic acid encoding each<br>
subunit of interest is separately fused to a phage coat protein and separately<br>
analyzed for its binding properties.<br>
Any cloning procedure used by one of skill in the art can be employed to<br>
make the expression vectors used in such affinity maturation/phage display<br>
procedures. For example, one of skill in the art can readily employ known<br>
cloning procedures to fuse a nucleic acid encoding, an antibody hypervariable<br>
region to a nucleic acid encoding a phage coat protein. See, e.g., Sambrook et<br>
al., Molecular Cloning, A Laboratory Manual, Cold Spring Harbor Laboratory,<br>
N.Y., 1989; Sambrook et al., Molecular Cloning, A Laboratory Manual, Cold<br>
Spring Harbor Laboratory, N.Y., 2001.<br>
The invention is therefore directed to a method for selecting antibodies<br>
and/or antibody fragments or polypeptides with desirable properties. Such<br>
desirable properties can include increased binding affinity or selectivity for the<br>
epitopes of the invention<br><br>
The antibodies and antibody fragments of the invention are isolated<br>
antibodies and antibody fragments. An isolated antibody is one that has been<br>
identified and separated and/or recovered from a component of the environment<br>
in which it was produced. Contaminant components of its production<br>
environment are materials that would interfere with diagnostic or therapeutic<br>
uses for the antibody, and may include enzymes, hormones, and other<br>
proteinaceous or nonproteinaceous solutes. The term "isolated antibody" also<br>
includes antibodies within recombinant cells because at least one component of<br>
the antibody's natural environment will not be present. Ordinarily, however,<br>
isolated antibody will be prepared by at least one purification step<br>
If desired, the antibodies of the invention can be purified by any available<br>
procedure. For example, the antibodies can be affinity purified by binding an<br>
antibody preparation to a solid support to which the antigen used to raise the<br>
antibodies is bound. After washing off contaminants, the antibody can be eluted<br>
by known procedures. Those of skill in the art will know of various techniques<br>
common in the immunology arts for purification and/or concentration of<br>
polyclonal antibodies, as well as monoclonal antibodies (see for example,<br>
Coligan, et al., Unit 9, Current Protocols in Immunology. Wiley Interscience,<br>
1991, incorporated by reference).<br>
In preferred embodiments, the antibody will be purified as measurable by<br>
at least three different methods: 1) to greater than 95% by weight of antibody as<br>
determined by the Lowry method, and most preferably more than 99% by<br>
weight; 2) to a degree sufficient to obtain at least 15 residues of N-terminal or<br>
internal amino acid sequence by use of a spinning cup sequentator; or 3) to<br>
homogeneity by SDS-PAGE under reducing or non-reducing conditions using<br>
Coomasie blue or, preferably, silver stain.<br>
Antigen, Binding Entity and Antibody Variants and Derivatives<br>
The invention also provides variants and derivative of the antigenic<br>
epitopes, binding entities and antibody segments identified herein. For example,<br>
any derivative or variant of a SEQ ID NO:3,4, 5 or 6 antigenic epitope is<br>
contemplated as being within the scope of the invention, particularly when the<br><br>
variant or derivative retains, or has improved, specificity as a vaccine for<br>
preventing or treating adenocarcinomas or is an improved marker for detecting<br>
adenocarcinomas. Similarly, any derivative or variant of a SEQ ID NO:735<br>
antibody polypeptide is contemplated by the invention, particularly when the<br>
variant or derivative antibody polypeptide has improved specificity or binding<br>
affinity for an antigenic epitope of the invention, for example, an antigenic<br>
epitope having SEQ ID NO:3, 4, 5 or 6.<br>
Derivative and variant antigenic epitopes and antibody segments of the<br>
invention are derived from the reference antigenic epitopes and antibody<br>
segments by deletion or addition of one or more amino acids to the N-terminal<br>
and/or C-terminal end of the reference antigenic epitopes and antibody segments;<br>
deletion or addition of one or more amino acids at one or more sites within the<br>
reference antigenic epitopes and antibody segments; or substitution of one or<br>
more amino acids at one or more sites within the reference antigenic epitopes<br>
and antibody segments. Thus, the antigenic epitopes and antibody segments of<br>
the invention may be altered in various ways including amino acid substitutions,<br>
deletions, truncations, and insertions.<br>
Such variant and derivative antigenic epitopes and antibody segments<br>
may result, for example, from human manipulation. For example, the affinity<br>
maturation techniques using phage display described above may be used to<br>
generate variants and derivatives of both the antigenic epitopes and antibody<br>
segments of the invention. Other methods for mutating or altering the sequence<br>
of polypeptide are generally available in the art. For example, amino acid<br>
sequence variants of the antigenic epitopes and antibody segments can be<br>
prepared by mutations in the DNA encoding these antigenic epitopes and<br>
antibody segments. Methods for mutagenesis and nucleotide sequence<br>
alterations are also available in the art. See, for example, Kunkel, Proc. Natl.<br>
Acad. Sci. USA, 82,488 (1985); Kunkel et al., Methods in Enzymol., 154,367<br>
(1987); U. S. Patent No. 4,873,192; Walker and Gaastra, eds., Techniques in<br>
Molecular Biology, MacMillan Publishing Company, New York (1983) and the<br>
references cited therein. Guidance as to appropriate amino acid substitutions that<br>
do not adversely affect the structural integrity and/or biological activity of the<br><br>
peptide of interest may be found in the model of Dayhoff et al., Atlas of Protein<br>
Sequence and Structure, Natl. Biomed. Res. Found., Washington, CD. (1978),<br>
herein incorporated by reference.<br>
The derivatives and variants of the antigenic epitopes and antibody<br>
segments of the invention have identity with at least about 90%, 91%, 92%, 93%<br>
or 94% of the amino acid positions of any one of SEQ ID NO.3-35 and generally<br>
have similar or improved immunological properties relative to those of the<br>
antigenic epitopes and antibody segments having any one of SEQ ID NO:3-35.<br>
In a desirable embodiment, the antigenic epitopes and antibody segment<br>
derivatives and variants have identity with at least about 95% or 96% of the<br>
amino acid positions of any one of SEQ ID NO:3-35 and generally have<br>
immunological properties that are similar or better than the antigenic epitopes<br>
and antibody segments having SEQ ID NO: 3-3 5. In a more desirable<br>
embodiment, the antigenic epitopes and antibody segments derivatives and<br>
variants have identity with at least about 97% or 98% of the amino acid positions<br>
of any one of SEQ ID NO:3-35 and generally have similar or improved<br>
immunological properties relative to those of the antigenic epitopes and antibody<br>
segments having SEQ ID NO.3-35.<br>
By "similar or improved immunological properties" is meant that a<br>
derivative or variant of a SEQ ID NO:3,4, 5 or 6 antigenic epitope retains, or has<br>
improved, activity as a vaccine for preventing or treating adenocarcinomas or is<br>
an improved marker for detecting adenocarcinomas. Similarly, derivatives or<br>
variants of a SEQ ID NO:7-35 antibody polypeptide have "similar or improved<br>
immunological properties" when they have improved specificity or binding<br>
affinity for an antigenic epitope of the invention, for example, an antigenic<br>
epitope having SEQ ID NO:3,4, 5 or 6.<br>
Amino acid.residues of the antigenic epitopes, binding entities and<br>
antibody segments and of the derivatives and variants thereof can be genetically<br>
encoded L-amino acids, naturally occurring non-genetically encoded L-amino<br>
acids, synthetic L-amino acids or D-enantiomers of any of the above. The amino<br>
acid notations used herein for the twenty genetically encoded L-amino acids and<br>
common non-encoded amino acids are conventional and are as shown in Table 1.<br><br><br><br><br>
Variants of the present antigenic epitopes and antibody segments that are<br>
encompassed within the scope of the invention can have one or more amino<br>
acids substituted with an amino acid of similar chemical and/or physical<br>
properties, so long as the backbone portions of these variant peptides have<br>
similar or improved immunological properties relative to those of antigenic<br><br>
epitopes and antibody segments having any one of SEQ ID NO.3-35. Derivative<br>
antigenic epitopes and antibody segments can have additional peptide or<br>
chemical moieties as well as one or more amino acids substituted with amino<br>
acids having different chemical and/or physical properties, so long as these<br>
derivative antigenic epitopes and antibody segments have similar or improved<br>
immunological properties relative to those of antigenic epitopes and antibody<br>
segments having any one of SEQ ID NO:3-35.<br>
Amino acids that are substitutable for each other to form a variant<br>
antigenic epitopes and antibody segments of the invention generally reside<br>
within similar classes or subclasses. As known to one of skill in the art, amino<br>
acids can be placed into three main classes: hydrophilic amino acids,<br>
hydrophobic amino acids and cysteine-like amino acids, depending primarily on<br>
the characteristics of the amino acid side chain. These main classes may be<br>
further divided into subclasses. Hydrophilic amino acids include amino acids<br>
having acidic, basic or polar side chains and hydrophobic amino acids include<br>
amino acids having aromatic or apolar side chains. Apolar amino acids may be<br>
further subdivided to include, among others, aliphatic amino acids. The<br>
definitions of the classes of amino acids as used herein are as follows:<br>
"Hydrophobic Amino Acid" refers to an amino acid having a side chain<br>
that is uncharged at physiological pH and that is repelled by aqueous solution.<br>
Examples of genetically encoded hydrophobic amino acids include IIe, Leu and<br>
Val. Examples of non-genetically encoded hydrophobic amino acids include t-<br>
BuA.<br>
"Aromatic Amino Acid" refers to a hydrophobic amino acid having a<br>
side chain containing at least one ring having a conjugated π-electron system<br>
(aromatic group). The aromatic group may be further substituted with substituent<br>
groups such as alkyl, alkenyl, alkynyl, hydroxyl, sulfonyl, nitro and amino<br>
groups, as well as others. Examples of genetically encoded aromatic amino acids<br>
include phenylalanine, tyrosine and tryptophan. Commonly encountered non-<br>
genetically encoded aromatic amino acids include phenylglycine, 2-<br>
naphthylalanine, P-2-tbienyialanine, l,2,3,4-tetrahydroisoquinohne-3-carboxylic<br><br>
acid, 4-chlorophenylalanine, 2-fluorophenylalanine, 3-fluorophenylalanine and<br>
4-fluorophenylalanine.<br>
"Apolar Amino Acid" refers to a hydrophobic amino acid having a side<br>
chain that is generally uncharged at physiological pH and that is not polar.<br>
Examples of genetically encoded apolar amino acids include glycine, proline and<br>
methionine. Examples of non-encoded apolar amino acids include Cha.<br>
"Aliphatic Amino Acid" refers to an apolar amino acid having a saturated<br>
or unsaturated straight chain, branched or cyclic hydrocarbon side chain.<br>
Examples of genetically encoded aliphatic amino acids include Ala, Leu, Val and<br>
IIe. Examples of non-encoded aliphatic amino acids include Nle.<br>
"Hydrophilic Amino Acid" refers to an amino acid having a side chain<br>
that is attracted by aqueous solution. Examples of genetically encoded<br>
hydrophilic amino acids include Ser and Lys. Examples of non-encoded<br>
hydrophilic amino acids include Cit and hCys.<br>
"Acidic Amino Acid" refers to a hydrophilic amino acid having a side<br>
chain pK value of less than 7. Acidic amino acids typically have negatively<br>
charged side chains at physiological pH due to loss of a hydrogen ion. Examples<br>
of genetically encoded acidic amino acids include aspartic acid (aspartate) and<br>
glutamic acid (glutamate).<br>
"Basic Amino Acid" refers to a hydrophilic amino acid having a side<br>
chain pK value of greater than 7. Basic amino acids typically have positively<br>
charged side chains at physiological pH due to association with hydronium ion.<br>
Examples of genetically encoded basic amino acids include arginine, lysine and<br>
histidine. Examples of non-genetically encoded basic amino acids include the<br>
non-cyclic amino acids ornithine, 2,3-diaminopropionic acid, 2,4-diaminobutyric<br>
acid and homoarginine.<br>
"Polar Amino Acid" refers to a hydrophilic amino acid having a side<br>
chain that is uncharged at physiological pH, but which has a bond in which the<br>
pair of electrons shared in common by two atoms is held more closely by one of<br>
the atoms. Examples of genetically encoded polar amino acids include<br>
asparagine and glutamine. Examples of non-genetically encoded polar amino<br>
acids include citrulline, N-acetyl lysine and methionine sulfoxide.<br><br>
"Cysteine-Like Amino Acid" refers to an amino acid having a side chain<br>
capable of forming a covalent linkage with a side chain of another amino acid<br>
residue, such as a disulfide linkage. Typically, cysteine-like amino acids<br>
generally have a side chain containing at least one thiol (SH) group. Examples<br>
of genetically encoded cysteine-like amino acids include cysteine. Examples of<br>
non-genetically encoded cysteine-like amino acids include homocysteine and<br>
penicillamine.<br>
As will be appreciated by those having skill in the art, the above<br>
classifications are not absolute. Several amino acids exhibit more than one<br>
characteristic property, and can therefore be included in more than one category.<br>
For example, tyrosine has both an aromatic ring and a polar hydroxyl group.<br>
Thus, tyrosine has dual properties and can be included in both the aromatic and<br>
polar categories. Similarly, in addition to being able to form disulfide linkages,<br>
cysteine also has apolar character. Thus, while not strictly classified as a<br>
hydrophobic or apolar amino acid, in many instances cysteine can be used to<br>
confer hydrophobicity to a polypeptide.<br>
Certain commonly encountered amino acids that are not genetically<br>
encoded and that can be present, or substituted for an amino acid, in the variant<br>
polypeptides of the invention include, but are not limited to, β-alanine (b-Ala)<br>
and other omega-amino acids such as 3-aminopropionic acid (Dap), 2,3-<br>
diaminopropionic acid (Dpr), 4-aminobutyric acid and so forth; α-<br>
aminoisobutyric acid (Aib); e-aminohexanoic acid (Aha); δ-aminovaleric acid<br>
(Ava); N-methylglycine (MeGly); ornithine (Orn); citrulline (Cit); t-butylalanine<br>
(t-BuA); t-butylglycine (t-BuG); N-methylisoleucine (MeIle); phenylglycine<br>
(Phg); cyclohexylalanine (Cha); norleucine (Nle); 2-naphthylalanine (2-Nal); 4-<br>
chlorophenylalanine (Phe(4-Cl)); 2-fluorophenylalanine (Phe(2-F)); 3-<br>
fluorophenylalanine (Phe(3-F)); 4-fluorophenylalanine (Phe(4-F)); penicillamine<br>
(Pen); l,2,3,4-tetrahydroisoquinoline-3-carboxylic acid (Tic); P-2-thienylalanine<br>
(Thi); methionine sulfoxide (MSO); homoarginine (hArg); N-acetyl lysine<br>
(AcLys); 2,3-diaminobutyric acid (Dab); 2,3-diaminobutyric acid (Dbu); p-<br>
aminophenylalanine (Phe(pNH2)); N-methyl valine (MeVal); homocysteine<br><br>
(hCys) and homoserine (hSer). These amino acids also fall into the categories<br>
defined above.<br>
The classifications of the above-described genetically encoded and fton-<br>
encoded amino acids are summarized in Table 2, below. It is to be understood<br>
that Table 2 is for illustrative purposes only and does not purport to be an<br>
exhaustive list of amino acid residues that may comprise the variant and<br>
derivative antigenic epitopes and antibody segments described herein. Other<br>
amino acid residues that are useful for making the variant and derivative<br>
polypeptides described herein can be found, e.g., in Fasman, 1989, CRC<br>
Practical Handbook of Biochemistry and Molecular Biology, CRC Press, Inc.,<br>
and the references cited therein. Amino acids not specifically mentioned herein<br>
can be conveniently classified into the above-described categories on the basis of<br>
known behavior and/or their characteristic chemical and/or physical properties as<br>
compared with amino acids specifically identified.<br><br>
Antigenic epitopes and antibody segments of the invention can have any amino<br>
acid substituted by any similarly classified amino acid to create a variant<br>
antigenic epitope or a variant antibody segment, so long as the variant has similar<br><br>
or improved immunological properties relative to those of an antigenic epitope or<br>
antibody segment having any one of SEQ ID NO:3-35.<br>
The invention is therefore also directed to binding entities and antibodies with<br>
binding domains related to the variable light or heavy chain CDR fragments isolated<br>
according to the invention. For example, the variable light chain CDR1 fragments can<br>
be aligned as follows:<br><br>
Related variable light chain CDR1 fragments and binding entities are of the following<br>
formula (SEQ ID NO:47).<br><br><br><br><br>
Detecting the Cancer-Associated Epitope<br>
The invention also provides methods of detecting the cancer-associated<br>
epitopes of the invention in biological test samples. Any immunoassay or in<br>
vivo imaging procedure known to one of skill in the art can be used to detect the<br>
cancer-associated epitopes of the invention in a biological test sample. For<br>
example, the cancer-associated epitopes of the invention can be detected by<br>
immunochemical, immunohistological, ELISA, radioimmunoassay, nuclear<br>
magnetic resonance, magnetic resonance imaging, surface plasmon resonance<br>
and related procedures.<br>
Such methods can include the steps of contacting a test sample with an<br>
antibody or binding entity capable of binding to a cancer-associated epitope of<br><br>
the invention, and determining whether the antibody or binding entity binds to a<br>
component of the sample. These methods can also include the steps of obtaining<br>
a biological sample (e.g., cells, blood, plasma, tissue, etc.) from a patient<br>
suspected of having cancer, contacting the sample with a labeled antibody or a<br>
labeled binding entity that is specific for the cancer-associated epitope of the<br>
invention, and detecting the epitope using standard immunoassay and/or<br>
diagnostic imaging techniques. Binding of the antibody or binding entity to the<br>
biological sample indicates that the sample contains the epitope.<br>
In another embodiment, the cancer-associated epitope can be used to<br>
detect antibodies in the blood, serum or tissues of a mammal with cancer. Such<br>
antibodies can arise naturally within the mammal when the cancer-associated<br>
epitope becomes exposed during malignant transformation.<br>
Accordingly, the invention provides a method of detecting cancer in a<br>
mammal by contacting a test sample with a cancer-associated epitope of the<br>
invention and detecting whether an antibody from the test sample has bound to<br>
the cancer-associated epitope.<br>
Antibodies or binding entities that are reactive with cancer-associated<br>
epitope of the invention and/or polypeptides comprising a cancer-associated<br>
epitope of the invention can be labeled or coupled to a diagnostic imaging agent<br>
for convenient detection of cancer.<br>
The words "label" and diagnostic imaging agent refer to a detectable<br>
compound or composition that is conjugated directly or indirectly to an antibody<br>
or antigen or epitope. The label may itself be detectable (e.g., radioisotope labels<br>
or fluorescent labels) or, in the case of an enzymatic label, may catalyze<br>
chemical alteration of a substrate compound or composition that is detectable.<br>
Such labels or diagnostic imaging agents are useful for imaging of cells<br>
and tissues that express the cancer-associated epitope. Such labels can also be<br>
used with a cancer-associated epitope of the invention in standard<br>
immunoassays. Labels and diagnostic imaging agents include, but are not<br>
limited to barium sulfate, iocetamic acid, iopanoic acid, ipodate calcium,<br>
diatrizoate sodium, diatrizoate meglumine, metrizamide, tyropanoate sodium and<br>
radiodiagnostics including positron emitters such as fluorine-18 and carbon-11,<br><br>
gamma emitters such as iodine-123, technitium-99m, iodine-131 and indium-<br>
111, nuclides for nuclear magnetic resonance such as fluorine and gadolinium.<br>
Paramagnetic isotopes for purposes of in vivo diagnosis can be used<br>
according to the methods of this invention. There are numerous examples of<br>
elements that are useful in magnetic resonance imaging. For discussions on in<br>
vivo nuclear magnetic resonance imaging, see, for example, Schaefer et al.,<br>
(1989) JACC 14, 472-480; Shreve et al., (1986) Magn. Reson. Med. 3, 336-340;<br>
Wolf, G. L., (1984) Physiol. Chem. Phys. Med. NMR 16,93-95; Wesbey et al.,<br>
(1984) Physiol. Chem. Phys. Med. NMR 16,145-155; Runge et al., (1984)<br>
Invest. Radiol. 19,408-415. Examples of suitable fluorescent labels include a<br>
fluorescein label, an isothiocyalate label, a rhodamine label, a phycoerythrin<br>
label, a phycocyanin label, an allophycocyanin label, an ophthaldehyde label, a<br>
fluorescamine label, etc. Examples of chemiluminescent labels include a luminal<br>
label, an isoluminal label, an aromatic acridinium ester label, an imidazole label,<br>
an acridinium salt label, an oxalate ester label, a luciferin label, a luciferase label,<br>
an aequorin label, etc. Those of ordinary skill in the art will know of other<br>
suitable labels that may be employed in accordance with the present invention.<br>
The attachment of these labels to antibodies or fragments thereof can be<br>
accomplished using standard techniques commonly known to those of ordinary<br>
skill in the art. Typical techniques are described by Kennedy et al., (1976) Clin.<br>
Chim. Acta 70,1-31; and Schurs et al., (1977) Clin. Chim. Acta 81, 1-40.<br>
Coupling techniques mentioned in the latter are the glutaraldehyde method, the<br>
periodate method, the dimaleimide method, the m-maleimidobenzyl-N-hydroxy-<br>
succinimide ester method. All of these methods are incorporated by reference<br>
herein.<br>
A solid phase or a solid support can be used in conjunction with the<br>
antibodies, binding entities, antigens or epitopes of the invention. Such a solid<br>
phase or solid support refers to a non-aqueous matrix to which the antibody,<br>
binding entity, antigen or epitope can adhere. Examples of solid phases and<br>
supports encompassed herein include those formed partially or entirely of glass<br>
(e.g. controlled pore glass), polysaccharides (e.g., agarose), polyacrylamides,<br>
polystyrene, polyvinyl alcohol and silicones. In certain embodiments, depending<br><br>
on the context, the solid phase or support can comprise the well of an assay<br>
plate; in others it is a purification column (e.g. an affinity chromatography<br>
column). This term also includes a discontinuous solid phase of discrete<br>
particles, such as those described in U.S. Pat. No. 4,275,149.<br>
Therapy<br>
According to the invention, the antigenic epitopes of the invention,<br>
antibodies or binding entities directed against such epitopes and protease<br>
inhibitors that inhibit formation of the epitopes of the invention can be used for<br>
cancer prevention and/or therapy. The antigenic epitopes of the invention can be<br>
used as vaccines to stimulate an immunological response in a mammal that is<br>
directed against cells having the cancer-associated epitope. Antibodies or<br>
binding entities directed against the antigenic epitopes of the invention can<br>
combat or prevent adenocarcinomas. Moreover, the invention contemplates<br>
administering protease inhibitors that inhibit cleavage of cytokeratin 8 and/or<br>
cytokeratin 18 to prevent or treat adenocarcinomas.<br>
In one embodiment, the invention provides a method of preventing or<br>
treating adenocarcinoma in a mammal by administering an antigenic epitope<br>
comprising any one of SEQ ID NO:3-6 to the mammal in an amount sufficient<br>
to stimulate an immunological response against the antigenic epitope. Two or<br>
more polypeptides comprising SEQ ID NO:3-6 can be combined in a therapeutic<br>
composition and administered in several doses over a period of time that<br>
optimizes the immunological response of the mammal. Such an immunological<br>
response can be detected and monitored by observing whether antibodies<br>
directed against the epitopes of the invention are present in the bloodstream of<br>
the mammal.<br>
Antibodies and binding entities generated as provided herein that react<br>
selectively with the cancer-associated epitope of the invention also be used for<br>
cancer therapy. Accordingly, the invention provides methods of preventing or<br>
treating adenocarcinoma in a mammal by administering to the mammal a<br>
therapeutically effective amount of an antibody or binding entity that can bind an<br>
antigenic epitope comprising any one of SEQ ID NO:3-6.<br><br>
Such antibodies or binding entities can be used alone or coupled to, or<br>
combined with, therapeutically useful agents. Antibodies and/or binding entities<br>
can be administered to mammals suffering from any cancer that displays the<br>
cancer-associated epitope of the invention. Such administration can provide<br>
both therapeutic treatment, and prophylactic or preventative measures. For<br>
example, the therapeutic methods of the invention can be used to deter the spread<br>
of a cancer and lead to its remission.<br>
As used herein, "therapeutically useful agents" include any therapeutic<br>
molecule that can beneficially be targeted to a cell expressing the cancer epitope<br>
disclosed herein, including antineoplastic agents, radioiodinated compounds,<br>
toxins, chemotherapeutic agents, cytostatic or cytolytic drugs.<br>
Such therapeutically useful agents include, for example, adrimycin,<br>
aminoglutethimide, aminopterin, azathioprine, bleomycin sulfate, bulsulfan,<br>
carboplatin, carminomycin, carmustine, chlorambucil, cisplatin,<br>
cyclophosphamide, cyclosporine, cytarabidine, cytosine arabinoside, cytoxin<br>
dacarbazine, dactinomycin, daunomycin, daunorubicin, doxorubicin,<br>
esperamicins (see U.S. Pat. No. 4,675,187), etoposide, fluorouracil, ifosfamide,<br>
interferon-a, lomustine, melphalan, mercaptopurine, methotrexate, mitomycin C,<br>
mitotane, mitoxantrone, procarbazine HC1, taxol, taxotere (docetaxel),<br>
teniposide, thioguanine, thiotepa, vinblastine sulfate, vincristine sulfate and<br>
vinorelbine. Additional agents include those disclosed in Chapter 52,<br>
Antineoplastic Agents (Paul Calabresi and Bruce A. Chabner), and the<br>
introduction thereto, pp.1202-1263, of Goodman and Gilman's "The<br>
Pharmacological Basis of Therapeutics", Eighth Edition, 1990, McGraw-Hill,<br>
Inc. (Health Professions Division). Toxins can be proteins such as, for example,<br>
pokeweed anti-viral protein, cholera toxin, pertussis toxin, ricin, gelonin, abrin,<br>
diphtheria exotoxin, or Pseudomonas exotoxin. Toxin moieties can also be high<br>
energy-emitting radionuclides such as cobalt-60,1-131,1-125, Y-90 and Re-186,<br>
and enzymatically active toxins of bacterial, fungal, plant or animal origin, or<br>
fragments thereof.<br>
According to the invention, such chemotherapeutic agents can be used to<br>
reduce the growth or spread of cancer cells and tumors that express the tumor<br><br>
associated epitope of the invention. Animals that can be treated by the<br>
chemotherapeutic agents of the invention include humans, non-human primates,<br>
cows, horses, pigs, sheep, goats, dogs, cats, rodents and the like. In all<br>
embodiments human tumor antigens and human subjects are preferred.<br>
The invention also contemplates using species-dependent antibodies for<br>
use in the present therapeutic methods. Such a species-dependent antibody has<br>
constant regions that are substantially non-immunologically reactive with the<br>
chosen species. Such species-dependent antibody is particularly useful for<br>
therapy because it gives rise to substantially no immunological reactions. The<br>
species-dependent antibody can be of any of the various types of antibodies as<br>
defined above, but preferably is mammalian, and more preferably is a humanized<br>
or human antibody.<br>
Therapeutically useful agents can be formulated into a composition with<br>
the antibodies of the invention and need not be directly attached to the antibodies<br>
of the invention. However, in some embodiments, therapeutically useful agents<br>
are attached to the antibodies of the invention using methods available to one of<br>
skill in the art, for example, standard coupling procedures.<br>
The invention further provides methods of preventing or treating<br>
adenocarcinoma in a mammal by administering to the mammal a therapeutically<br>
effective amount of a protease inhibitor that prevents formation of an antigenic<br>
epitope comprising any one of SEQ ID NO:3-6. According to the invention, the<br>
sites of protease cleavage at amino acids 22 and 40 on cytokeratin K8, and at<br>
amino acid 50 on cytokeratin K18, all contained consensus sequence<br>
Xaa1SR↓Xaa4 (SEQ ID NO.40), where Xaa1 is serine, phenylalanine or valine<br>
and Xaa4 is serine or valine. The structure of these cleavage sites indicates that<br>
the enzyme responsible for these cleavages is a trypsin-like protease. Trypsin<br>
inhibitors are available to one of skill in the art. See, e.g., U.S. Patent 6,239,106;<br>
U.S. Patent 6,159,938; U.S. Patent 5,962,266. Such trypsin inhibitors include<br>
inhibitors available for serine proteases such as kallikrein, chymotrypsins A and<br>
B, trypsin, elastase, subtilisin, coagulants and procoagulants, particularly those in<br>
active form, including coagulation factors such as factors VIIa, IXa, Xa, XIa, and<br><br>
XIIa, plasmin, thrombin; proteinase-3, enterokinase, acrosin, cathepsin,<br>
urokinase, and tissue plasminogen activator.<br>
According to the invention, any inhibitor capable of inhibiting a protease<br>
that can cleave Xaa1SR↓Xaa4 (SEQ ID NO:40) may be used to prevent or treat<br>
adenocarcinomas. For example, peptides with homology to Xaa1SR↓Xaa4 (SEQ<br>
ID NO:40) but that cannot be cleaved may be used as inhibitors in the present<br>
therapeutic methods. Other examples of inhibitors that may be used include, for<br>
example, soybean trypsin inhibitor (or STI, from Sigma Chemical Co.), alpha-2-<br>
macroglobulin, alpha-1-antitrypsin, aprotinin, pancreatic secretory trypsin<br>
inhibitor (PSTI) corn and pumpkin trypsin inhibitors (Wen, et al., Protein Exp. &amp;<br>
Purif. 4:215 (1993); Pedersen, et al., J. Mol. Biol. 236:385 (1994)), and so forth.<br>
One candidate for a useful inhibitor of human origin is found in circulating<br>
isoforms of the human amyloid P-protein precursor (APPI), also known as<br>
protease nexin-2. APPI contains a Kunitz serine protease inhibitor domain<br>
known as KPI (Kunitz Protease Inhibitor). See Ponte et al., Nature, 331:525<br>
(1988); Tanzi et al., Nature 331:528 (1988); Johnstone et al., Biochem. Biophys.<br>
Res. Commun. 163:1248 (2989); Oltersdorf et al., Nature 341.144 (1989).<br>
Human KPI shares about 45% amino acid sequence identity with aprotinin. The<br>
isolated KPI domain has been prepared by recombinant expression in a variety of<br>
systems, and has been shown to be an active serine protease inhibitor. See, for<br>
example, Sinha, et al., J. Biol. Chem. 265:8983 (1990).<br>
Progression of adenocarcinoma cancer and/or the therapeutic efficacy of<br>
chemotherapy may be measured using procedures available in the art. For<br>
example, the efficacy of a particular chemotherapeutic agent can be determined<br>
by measuring the amount of cancer-associated epitope released from<br>
adenocarcinoma cells undergoing cell death. The concentration of antigenic<br>
epitope (e.g. a polypeptide having any one of SEQ ID NO:3-6, or a combination<br>
of such polypeptides) released from cells can be compared to standards from<br>
healthy, untreated patients to assess whether heightened levels of the present<br>
epitopes are present in a patient. Fluid samples can be collected at discrete<br>
intervals during treatment and compared to a standard. It is contemplated that<br>
changes in the level of a cancer-associated antigenic epitope of the invention,<br><br>
will be indicative of the efficacy of treatment (that is, the rate of cancer cell<br>
death). It is contemplated that the release of cancer-associated antigenic epitopes<br>
can be measured in many test samples, including blood, plasma, serum, feces,<br>
urine, sputum, vaginal secretions, seminal fluids, semen and any tissue sample.<br>
Where the assay is used to monitor tissue viability or progression of<br>
adenocarcinoma, the step of detecting the presence and abundance of the<br>
antigenic epitope in samples of interest is repeated at intervals and these values<br>
then are compared, the changes in the detected concentrations reflecting changes<br>
in the status of the tissue. For example, an increase in the level of<br>
adenocarcinoma-associated epitope may correlate with progression of the<br>
adenocarcinoma. Where the assay is used to evaluate the efficacy of a therapy,<br>
the monitoring steps occur following administration of the therapeutic agent or<br>
procedure (e.g., following administration of a chemotherapeutic agent or<br>
following radiation treatment). Similarly, a decrease in the level of<br>
adenocarcinoma cancer-associated epitopes of the invention may correlate a<br>
regression of the adenocarcinoma.<br>
Thus, adenocarcinomas may be identified by the presence of cancer-<br>
associated antigenic epitopes as provided herein. Once identified, the<br>
adenocarcinoma may be treated using antibodies and protease inhibitors that<br>
reduce cleavage of cytokeratins 8 and 18. Moreover, the methods provided<br>
herein can be used to monitor the progression of the disease and/or treatment of<br>
the disease.<br>
Compositions<br>
The invention is further directed to compositions containing the present<br>
antibodies, binding entities, antigenic epitopes or trypsin-like protease inhibitors.<br>
Such compositions are useful for detecting the antigenic epitopes of the<br>
invention and for therapeutic methods involving prevention and treatment of<br>
cancers associated with the presence of the antigenic epitopes of the invention.<br>
The antibodies, binding entities, antigenic epitopes and protease<br>
inhibitors of the invention can be formulated as pharmaceutical compositions<br>
and administered to a mammalian host, such as a human patient in a variety of<br><br>
forms adapted to the chosen route of administration. Routes for administration<br>
include, for example, intravenous, intra-arterial, subcutaneous, intramuscular,<br>
intraperitoneal and other routes selected by one of skill in the art.<br>
Solutions of the antibodies, binding entities, antigenic epitopes and<br>
protease inhibitors of the invention can be prepared in water or saline, and<br>
optionally mixed with a nontoxic surfactant. Formulations for intravenous or<br>
intra-arterial administration may include sterile aqueous solutions that may also<br>
contain buffers, liposomes, diluents and other suitable additives.<br>
The pharmaceutical dosage forms suitable for injection or infusion can<br>
include sterile aqueous solutions or dispersions comprising the active ingredient<br>
that are adapted for administration by encapsulation in liposomes. In all cases,<br>
the ultimate dosage form must be sterile, fluid and stable under the conditions of<br>
manufacture and storage.<br>
Sterile injectable solutions are prepared by incorporating the antibodies,<br>
binding entities, antigenic epitopes and protease inhibitors in the required<br>
amount in the appropriate solvent with various of the other ingredients<br>
enumerated above, as required, followed by filter sterilization.<br>
Useful dosages of the antibodies, binding entities, antigenic epitopes and<br>
protease inhibitors can be determined by observing their in vitro activity, and in<br>
vivo activity in animal models. Methods for the extrapolation of effective<br>
dosages in mice, and other animals, to humans are known to the art; for example,<br>
see U.S. Patent No. 4,938,949.<br>
In general, a suitable dose of the antibodies, binding entities, antigenic<br>
epitopes and protease inhibitors will be in the range of from about 1 to about<br>
2000 µg/kg, for example, from about 2.0 to about 1500 µg/kg of body weight per<br>
treatment. Preferred doses are in the range of about 3 to about 500 µg per<br>
kilogram body weight of the recipient per treatment, more preferably in the range<br>
of about 10 to about 300 μg/kg/treatment, most preferably in the range of about<br>
20 to about 200 μg/kg/treatment.<br>
The antibodies, binding entities, antigenic epitopes and protease<br>
inhibitors are conveniently administered in unit dosage form; for example,<br><br>
containing 5 to 1000 μg, conveniently 10 to 750 μg, most conveniently, 50 to<br>
500 µg of active ingredient per unit dosage form.<br>
Ideally, the antibodies, binding entities, antigenic epitopes and protease<br>
inhibitors should be administered to achieve peak plasma concentrations of from<br>
about 0.1 to about 10 nM, preferably, about 0.2 to 10 nM, most preferably, about<br>
0.5 to about 5 nM. This may be achieved, for example, by the intravenous<br>
injection of a 0.05 to 25% solution of the antibodies, optionally in saline.<br>
Desirable blood levels may be maintained by continuous infusion to provide<br>
about 0.01-10.0 μg/kg/hr or by intermittent infusions containing about 0.4-50<br>
μg/kg of the antibodies.<br>
The desired dose may conveniently be presented in a single dose or as<br>
divided doses administered at appropriate intervals, for example, as two, three,<br>
four or more sub-doses per day. The sub-dose itself may be further divided, for<br>
example, into a number of discrete loosely spaced administrations; such as<br>
multiple intravenous doses. For example, it is desirable to administer the present<br>
compositions intravenously over an extended period, either by continuous<br>
infusion or in separate doses.<br>
Kits<br>
The invention further provides kits for detection of the antigenic epitope<br>
of the invention and for treatment of adenocarcinomas.<br>
A kit for detection of the antigenic epitope of the invention may contain a<br>
container containing an antibody or binding entity capable of binding to an<br>
antigenic epitope of the invention. Such an antibody or binding entity may be<br>
labeled for easy detection. Individual kits may be adapted for performing one or<br>
more of the methods of the invention.<br>
Optionally, the subject kit may further comprise at least one other reagent<br>
required for performing the method that the kit is adapted to perform. Examples<br>
of such additional reagents include: a label, a standard, a control, a buffer, a<br>
solution for diluting the test sample, or a reagent that facilitates detection of the<br>
label. The reagents included in the kits of the invention may be supplied in<br>
premeasured units so as to provide for greater precision and accuracy. Typically,<br><br>
kits reagents and other components are placed and contained in separate vessels.<br>
A reaction vessel, test tube, microwell tray, microtiter dish or other container can<br>
also be included in the kit. Different labels can be used on different reagents so<br>
that each reagent can be distinguished from another.<br>
A further aspect of the invention relates to a kit for treatment of<br>
adenocarcinomas comprising a pharmaceutical composition of the invention and<br>
an instructional material. Such a kit may contain a container having an antigenic<br>
epitope, an antibody, a binding entity or an inhibitor of the invention. The<br>
antigenic epitope may act as a vaccine for preventing formation of metastatic<br>
adenocarcinoma. The antibody or binding entity is directed against an antigenic<br>
epitope of the invention and can be administered to treat or prevent the spread of<br>
adenocarcinomas. An inhibitor of cytokeratin 8 or 18 cleavage can also inhibit<br>
the formation and spread of adenocarcinomas. Any one of these antigenic<br>
epitopes, antibodies, binding entities or inhibitors may be contained within an<br>
appropriate container in the kit. Alternatively, a combination of antigenic<br>
epitopes, antibodies, binding entities or inhibitors may be contained within an<br>
appropriate container in the kit.<br>
As used herein, an "instructional material" includes a publication, a<br>
recording, a diagram, or any other medium of expression that is used to<br>
communicate the usefulness of the pharmaceutical composition of the invention<br>
for inhibiting cleavage of cytokeratin 8 or 18 or for stimulating the immune<br>
system to recognize the epitopes of the invention in a mammal or patient. The<br>
instructional material may also, for example, describe an appropriate dose of the<br>
pharmaceutical composition of the invention. The instructional material of the<br>
kit of the invention may, for example, be affixed to a container that contains a<br>
pharmaceutical composition of the invention or be shipped together with a<br>
container that contains the pharmaceutical composition. Alternatively, the<br>
instructional material may be shipped separately from the container with the<br>
intention that the instructional material and the pharmaceutical composition be<br>
used cooperatively by the recipient.<br>
The invention also includes a kit comprising a pharmaceutical<br>
composition of the invention and a delivery device for delivering the<br><br>
composition to a mammal, for example, a human patient who may have an<br>
adenocarcinoma. By way of example, the delivery device may be a squeezable<br>
spray bottle, a metered-dose spray bottle, an aerosol spray device, an atomizer, a<br>
dry powder delivery device, a self-propelling solvent/powder-dispensing device,<br>
a syringe, a needle, a tampon, or a dosage measuring container.<br>
The invention will be further described by reference to the following<br>
detailed examples, which are given for illustration of the invention, and are not<br>
intended to be limiting thereof.<br>
EXAMPLE 1: Cancer-Associated Epitope Characterization<br>
Isolation of COU-1 Monoclonal Antibodies<br>
The IgM HMab, COU-1, is secreted by the hybridoma cell line, B9165,<br>
derived by fusing the human lymphoblastoid cell line WI-L2-729-HF2 with<br>
lymphocytes obtained from mesenteric lymph nodes from a colon cancer patient<br>
(35). Mesenteric lymph nodes draining the tumor region in patients with<br>
colorectal cancer were minced under sterile conditions. Debris was removed by<br>
filtration through cotton wool and the lymphocytes were purified by<br>
centrifugation on Ficoll-Isopaque (Boehringer-Mannheim, Mannheim, Federal<br>
Republic of Germany).<br>
The lymphocytes were fused with the human fusion cell line W1-L2-729-<br>
HF2 (referred to as HF2) (from Tecniclone Int., Santa Ana, Calif., USA)<br>
according to Kohler, Immunological Methods Vol. II, Academic Press, 1981, pp.<br>
285-298. The ratio between the HF2 and lymphocytes (107) was 1:2.<br>
After washing the HF2 and the lymphocytes together in RPMI-1640<br>
medium and collecting the cells by centrifugation, the cell pellet was<br>
resuspended in 0.5 ml of 50% polyethylene glycol (PEG) 6000 over a period of 1<br>
minute with constant shaking. Before dilution of the PEG with RPMI-1640, the<br>
cells were incubated for another 2 minutes. The resulting fusion product was<br>
washed and resuspended in solution medium [RPMI-1640,10% FCS (fetal calf<br>
serum) supplemented with HAT (2 x 10-4 M hypoxanthine, 4 x 10-7 M<br>
aminopterin, 3.2 x 10-6 M thymidine)]. The cells were plated in 96-well<br><br>
microtiter plates using 200 µl containing 2 x 105 cells per well. The cells were<br>
maintained in selective medium for two weeks. Further culturing was carried out<br>
in RPMI-1640 with 10% FCS supplemented with hypoxanthine and thymidine.<br>
Growing hybrids appeared 10 days to 4 weeks after fusion. Cloning was<br>
performed by limiting dilution without feeder cells.<br>
Supernatants from wells with growing clones were analyzed for<br>
immunoglobulin production by ELISA using microtiter plates coated with rabbit<br>
anti-human Ig (H and L chain) (Dakopatts, Copenhagen, Denmark) diluted<br>
1:10,000 in 0.1M bicarbonate, pH 9.6. Coated wells were washed with PBS-<br>
Tween (phosphate buffered saline—0.05% Tween 20) and incubated for 2 hours<br>
at room temperature with supernatants diluted 1:10 in PBS-Tween. Development<br>
was carried out with alkaline phosphatase (AP)-coupled antibody specific for<br>
IgM, IgA or IgG (Dakopatts, Copenhagen) diluted 1:3000 in PBS-Tween. After<br>
incubation for 1 hour at room temperature, the substrate p-nitrophenylphosphate<br>
(PNPP), 1 mg/ml 10% diethanolamine, 1 mM MgC12, pH 9.6, was added.<br>
Optical density was measured at 405 run after 1 hour of incubation at 37°C.<br>
Standard curves for quantification were constructed with dilution of IgM<br>
(Cappel) or IgG (Kabi AB, Stockholm, Sweden). Hybrids producing<br>
immunoglobulin (Ig) assayed by ELISA were propagated by transfer to 24-well<br>
macroplates (Nunc A/S, Denmark). The hybridoma cell line B9165 (ECACC<br>
87040201) selected by the methods secreted the COU-1 antibodies described<br>
below and was shown by ELISA to produce between 1 and 5 µg of IgM per ml<br>
when allowed to grow for two weeks without change of media.<br>
The hybridoma cell line B9165 was deposited with European Collection<br>
of Cell Cultures (ECACC), CAMR, Salisbury, Wiltshire, SP4 OJG, UK, Deposit<br>
no. ECACC 87040201.<br>
COU-1 hybridoma supernatants were further analyzed by<br>
immunocytochemical analysis for reaction with tumor cells or by<br>
immunohistochemical analysis for reactions with tumor tissues as described<br>
below.<br><br>
Immunocytochemical Analysis of COU-1 Antibodies<br>
Immunocytochemical analysis was performed on cell smears prepared<br>
from different human tumor cell lines and from peripheral human blood<br>
leukocytes. Cells were fixed on slides by treatment with formol-acetone (9.5%<br>
formaldehyde, 43% acetone in 86 mM phosphate buffer, pH 7.2).<br>
Approximately 50 µl of COU-1 supernatant (from the hybridoma B9165;<br>
ECACC 87040201) was placed on the smear of fixed cells and incubated<br>
overnight at 4°C in a humidified chamber before rinsing and incubation for 1<br>
hour at room temperature with horseradish peroxidase (HRP)-labeled rabbit anti-<br>
human IgM (Dakopatts) diluted to 1:80 in PBS-Tween. Finally, peroxidase<br>
substrate (0.01% H2O2 and diaminobenzidine at 0.6 µg/ml in PBS) was added.<br>
The smears were lightly counterstained with hematoxylin and mounted. Table 3<br>
shows the results obtained by analysis of COU-1 on smears of various cells.<br><br>
A selective reactivity with colon and mammary adenocarcinomas was apparent.<br>
Live COLO 201 cells (colonic adenocarcinoma cells) were incubated<br>
with the COU-1 antibody at 4°C, followed by the enzyme-labeled anti-Ig<br>
antibody. The cells were then smeared on slides, fixed with glutaraldehyde<br><br>
(0.17% in PBS) and incubated with substrate. COLO 201 cells stained with<br>
COU-1 while control cells did not (data not shown).<br>
Immunohistochemical analysis.<br>
A preliminary immunohistochemical analysis was performed on frozen<br>
tissue sections fixed in acetone. Endogenous IgM was blocked by incubation<br>
with Fab' fragments of anti-µ-chain antibody (purchased from Dakopatts,<br>
Copenhagen, Denmark) before incubation with the COU-1 antibody (0.5 μg/ml).<br>
The anti-mu-chain antibody Fab' fragment was prepared according to B. Nielsen<br>
et al., Hybridoma 6 (1), 1987, pp. 103-109). While clear-cut specificity for<br>
cancerous tissues was observed using the COU-1 antibody, some non-specific<br>
binding was observed in certain tissue types (for example, mammary tubules).<br>
An improved fixation procedure was used that substantially eliminated<br>
non-specific cross-reactivity with certain tissue types, including mammary<br>
ductuli and tubules. Tissue specimens were obtained from colorectal cancer<br>
patients undergoing surgical resection. Normal colon tissue was taken from the<br>
resectate approximately 15 cm away from the site of the tumor. Tissues were<br>
fixed in 96% alcohol for 6 h at 4°C. Afterwards, tissues were paraffin embedded<br>
and cut into 5 µm sections. Sections were deparaffinized in xylol, rehydrated<br>
through graded alcohol and washed in PBS-Tween. Sections were incubated for<br>
2 h at room temperature in a humidified chamber with 100 µl of murine<br>
monoclonal antibody, human monoclonal IgM antibody or normal polyclonal<br>
human IgM, all at 0.5-10 μg/ml. The slides were washed and incubated with AP-<br>
Iabeled rabbit anti-human IgM (Dako, Glostrup, Denmark), horse-radish<br>
peroxidase (HRP) labeled rabbit anti-human IgM (Dako) or HRP-labeled rabbit<br>
anti-mouse IgG (Dako) diluted in PBS with 10% (w/v) bovine serum albumin for<br>
1 h at room temperature. After washing, the HRP was visualized by development<br>
with chromogenic substrate (0.6 mg diaminobenzidine per ml PBS with 0.01%<br>
H2O2) and AP with 0.2 mg naphthol-AS-Mx phosphate (Sigma), 1 mg Fast Red<br>
TR Salt (Sigma), 20ug dimethylformamide per ml 0.1M Tris/HCl, 1M<br>
levamisole, pH 8.2. The sections were counterstained with Mayer's<br><br>
haematoxylin, dehydrated in xylene and mounted in Aquamount (Gurr, Poole,<br>
England).<br>
Bound antibody was visualized as described above for the<br>
immunocytochemical analysis. Only the tumor cells in sections of colon<br>
adenocarcinomas were stained COU-1. No staining was observed in tonsillar<br>
tissue. Tables 4A and 4B summarize the reactivity of the COU-1 antibody with a<br>
variety of tissues, where the reactivity of malignant tissues is provided in Table<br>
4A and the lack of reactivity of non-malignant tissues is provided in Table 4B.<br><br><br><br>
Normal colon epithelium showed binding of all analyzed human IgM,<br>
monoclonal antibodies, myeloma IgM as well as normal polyclonal human IgM.<br>
This general binding of IgM to normal colon epithelium was thus judged to be<br>
non-specific.<br>
EXAMPLE 2: Cancer-Associated Epitope Mapping<br>
Materials and Methods<br>
Antibodies<br>
The IgM HMab, COU-1, is secreted by the hybridoma cell line, B9165,<br>
derived by fusing the human lymphoblastoid cell line WI-L2-729-HF2 with<br>
lymphocytes obtained from mesenteric lymph nodes from a colon cancer patient,<br>
as described above. The hybridoma cell line B9165 was deposited with European<br><br>
Collection of Cell Cultures 'ECACC), CAMR, Salisbury, Wiltshire, SP4 OJG,<br>
UK, Deposit no. ECACC. 040201. More information about ECACC can be<br>
obtained on the website at ecacc.org.<br>
The human-human hybridoma cell line was grown in protein-free<br>
medium: RPMI1640 medium (Gibco, Grand Island, NY) supplemented with<br>
SSR3 serum replacement (Medicult, Copenhagen, Denmark). HMab COU-1 was<br>
purified from cell culture supernatant by affinity chromatography on Sepharose-<br>
coupled murine anti-human µ-chain monoclonal antibody (Mab)(HB57, ATCC,<br>
Rockville, MD). The antibody was eluted with 0.1 M diethylamine, pH 10.5,<br>
followed by fractionation by FPLC. IgM purified from normal human serum<br>
(Cappel, Cochranville, PA) was used as a control. Murine Mabs, M20 directed<br>
against normal K8 and CY-90 directed against normal K18, were obtained from<br>
Sigma Chemical Co. (St. Louis, MO).<br>
ELISA<br>
ELISA wells (Costar, Cambridge, MA) were coated overnight at 4°C<br>
with fractions from cytokeratin purification procedures or with different<br>
recombinant K8/K18 complexes (5 μg/ml) in PBS, pH 7.4. The wells were<br>
washed twice with PBS, blocked with 3% BSA in PBS for 1 h at 37°C, and<br>
incubated with HMab COU-1 antibody for 2 h at 37°C. Plates were washed 10X<br>
with PBS-0.05% Tween 20 and bound antibody was detected with alkaline<br>
phosphatase (AP)-labeled goat anti-human kappa-chain (Sigma) diluted 1000<br>
fold in PBS. Bound antibody was visualized with para-nitrophenylphosphate<br>
(Sigma)(l mg/ml 1 mM MgCl2,10% (w/v) diethanolamine, pH 9.6) and read at<br>
405 nm.<br>
Cell culture<br>
The human breast adenocarcinoma cell line MCF7 (ATCC) was<br>
maintained in Eagle's MEM (Gibco), supplemented with 10% FCS, non-<br>
essential amino acids, lmM sodium pyruvate, lmM HEPES buffer, 100 U<br>
penicillin/ml, 100 mg streptomycin/ml and 2mM L-glutamine. The human colon<br>
adenocarcinoma cell line Colon 137 (kindly provided by Dr. Ebbesen, Aarhus<br><br>
University, Denmark) was maintained in RPMI1640 (Gibco), supplemented<br>
with FCS, penicillin, streptomycin and L-glutamine as above.<br>
Purification of cytokeratin from normal and malignant tissue<br>
Cytokeratin were prepared from fresh, surgically-removed, colon cancer<br>
tissue or normal colon epithelia. Tissue samples (1-5 g) were minced with a<br>
shears and homogenized in 10-30 ml of Tris-buffered saline (TBS)(10 mM Tris,<br>
0.14 M NaCl, 15 mM NaN3,pH 7.6) containing 1% (v/v) Emulphogene (Sigma)<br>
using a blade rotor (Euro Turrax T20b basic, DCA Labortechnik, Staufen,<br>
Germany) for 3 x 5 sec at 27.000 rpm on ice. Enzyme inhibitors: 5 mM<br>
iodoacetamide, 10 mM PMSF, 5 mM EDTA (all Sigma), 5 mM Cyclocapron<br>
(KABI, Stockholm, Sweden), and 10 U Aprotinin (Bayer, Leverkusen, Germany)<br>
per ml were included in the buffers during the homogenization, sonication and<br>
ion exchange chromatography. The suspension was pelleted by centrifugation at<br>
10.000 g for 10 min at 4°C, washed twice in TBS containing 1% Emulphogene<br>
and resuspended in buffer A (10 mM Tris pH 8.6 containing 0.1% SDS (w/v)<br>
and 0.05% Emulphogene). The suspension was sonicated for 3 x 15 sec on ice<br>
and centrifuged at 12.000 g for 10 min at 4°C. The supernatant was applied to an<br>
anion exchange column (20 ml Q-Sepharose Fast Flow column, QFF,<br>
(Pharmacia Upjohn, Uppsala, Sweden)) pre-equilibrated with buffer A. After<br>
washing the column with 10 column volumes of buffer A, bound proteins were<br>
eluted with a linear gradient to 1M NaCl in buffer A. Fractions of 1ml were<br>
collected and further analyzed by SDS-PAGE/Western blotting and ELISA. For<br>
ELISA, 10µl of each fraction was added to wells containing 10 µl of SM2 beads<br>
(BioRad) in 100 µl TBS, followed by incubation with COU-1 as described<br>
above. The beads bind the detergent and thus allow for the direct coating of the<br>
proteins in the fractions.<br>
SDS-PAGE and Western Blot analysis<br>
Electrophoresis was performed in a discontinuous buffer system on 8 cm<br>
4-20% or 10% (w/v) polyacrylamide gels for analysis and on 15 cm 14%<br>
polyacrylamide gels for N-terminal sequencing (36). Samples were mixed with 2<br><br>
x sample buffer (4% SDS, 0.2% bromophenol blue, 20% glycerol in 100 mM<br>
Tris buffered saline), boiled for 5 rain and resolved under denaturing and<br>
reducing (100 mM DTT) conditions. Protein bands were visualized with<br>
Coomassie Brilliant Blue. Separated proteins were also electroblotted onto<br>
polyvinylidene difluoride membranes (PVDF, Immobilon P, Millipore, Bedford,<br>
MA), at 100 Volts for 1 h in ice, using transfer buffer (10% (v/v) ethanol, 25 mM<br>
Tris, 200 mM glycine). Prior to transfer, the membrane was pre-soaked in<br>
ethanol for 2 min and the membranes and the gel were incubated in transfer<br>
buffer for 10 min. Following transfer, the membrane was blocked for 2 h in<br>
Western blot buffer (50 mM Tris, 350 mM NaCl, 15 mM NaN3, 0.1% Tween-<br>
20) washed 3x with Western blot buffer and incubated with COU-1 antibody (5<br>
μg/ml), mouse anti-K8 antibody (diluted 1/2000), mouse anti K-18 antibody<br>
(diluted 1/2000) or goat-anti-GST antibody (diluted 1/1000, Pharmacia Upjohn)<br>
overnight at room temperature. The membrane was washed in Western blot<br>
buffer and incubated with AP-conjugated rabbit-anti-goat IgG antibody (diluted<br>
1/1000, Sigma), or AP-conjugated rabbit-anti-human IgM antibody (diluted<br>
1/500, DAKO, Glostrup, Denmark) for 2 h at room temperature. Following 3<br>
washes in PBS, the membrane was fixed with 0.2% glutaraldehyde in PBS for 15<br>
min at room temperature and finally washed in PBS. Bound AP conjugate was<br>
visualized by NBT/BCIP (Bio-Rad, Hercules, CA). MCF7 or Colon 137 cells,<br>
resuspended in SDS sample buffer and sonicated, were used as antigen control.<br>
A low range protein marker (Bio-Rad) was used to indicate the molecular weight<br>
of the fragments.<br>
Amino Acid Sequencing and Amino Acid Analysis<br>
Previously described procedures (37) were employed for amino acid<br>
sequencing and amino acid analysis. For N-terminal sequencing, purified<br>
cytokeratin was run on SDS-PAGE and electroblotted onto PVDF membranes<br>
prior to detection with Coomassie. The different bands were excised from the<br>
blot and sequenced in an Applied Biosystems 470 A protein sequencer (ABI,<br>
Forster City, CA). Sequences similar to cytokeratins were searched for in<br>
GenBank/EBI/DDBI/PDB databases using the BLAST program.<br><br>
Expression and purification of recombinant K8 and K18 proteins<br>
E. coli DH5a harboring plasmids encoding a panel of K8 and K18<br>
proteins were analyzed. The panel consisted of the full length and several N-<br>
terminal and C-terminal deleted fragments of K8 and K18, cloned as GST fusion<br>
proteins into a modified pGEX-2T vector (38). The E. coli cultures were grown<br>
in Super Broth medium, supplemented with 20 mM MgCl2 and 50 mg<br>
carbenicillin/ml at 37°C until OD600 reached 0.6. Protein expression was then<br>
induced with 1 mM IPTG (Sigma) and 4 µM cAMP and the culture allowed to<br>
grow for an additional 3 h at 30°C. The bacteria were pelleted at 4.000 g for 15<br>
min at 4°C. For SDS-PAGE, the pellet was resuspended in sample buffer and<br>
sonicated 5 x 10 sec before electrophoresis. For purification of the recombinant<br>
K8 or K18 proteins, the pellet of a 400 ml culture grown and processed as<br>
described above was resuspended in 50 ml lysis buffer (50 mM Tris-HCl, 100<br>
mM NaCl, 1 mM EDTA, 5 mM P-mercaptoethanol, pH 8.0) containing I mg/ml<br>
lysozyme and incubated for 30 min at 4°C. The suspension was sonicated 3 x 20<br>
sec and pelleted at 20.000 g at 4°C. The pellet was washed twice in a high salt<br>
buffer (50 mM Tris-HCl, 2 M NaCl, 10 mM EDTA, 5 mM p-mercaptoethanol,<br>
1% NP40, pH 8.0) and once in lysis buffer. The pellet was subsequently washed<br>
twice in lysis buffer containing 2 M urea and stored at 4°C in lysis buffer<br>
containing 8 M urea.<br>
Heterotypic association assay<br>
Panels of different C- or N-terminal-deleted or intact K8 and K18<br>
proteins were separated by SDS-PAGE and transferred to a PVDF membrane, as<br>
described above. After blocking, the membrane was incubated for 16 h at 4°C<br>
with 100 µg/ml of purified K8 or K18 protein in PBS, 2% BSA and 4 M urea; if<br>
K8 proteins were transferred to the membrane, the membrane was subsequently<br>
incubated with a purified K18 protein, and vice versa (38). The membrane was<br>
then washed with PBS, incubated with COU-1 (5 μg/ml) in Western blot buffer<br>
containing 10% FCS for 2 h at room temperature and binding detected as<br>
described above.<br><br>
Surface plasmon resonance<br>
The kinetics of HMab COU-1 binding to heterotypic complexes of<br>
recombinant intact K8 or K18 (and fragments thereof) was determined by surface<br>
plasmon resonance measurements using the BIAcore instrument (Pharmacia).<br>
The sensor chip was activated for immobilization with N-hydroxysuccinimide<br>
and N-ethyl-NT-(3-diethyl aminopropyl) carbodiimide. The heterotypic<br>
cytokeratin complexes were coupled to the surface by injection of 50 µl of a 50<br>
μg/ml sample. Excess activated esters were quenched with 30 µl 1 M<br>
ethanolamine, pH 8.5. Typically, 3000 resonance units were immobilized.<br>
Binding of COU-1 to immobilized heterotypic cytokeratin complexes was<br>
studied by injecting COU-1 in a range of concentrations (0.5-80 μg/ml) at a flow<br>
rate of 5 ul/min. The association was monitored as the increase in resonance<br>
units per unit time. Dissociation measurements were obtained following the end<br>
of the association phase with a flow rate of 20 ul/min. The binding surface was<br>
regenerated with 10 mM HC1,1M NaCl, pH 2.0, and remained active for 10<br>
measurements. The association and dissociation rate constants, kon and koff,<br>
were determined from a series of measurements, as described previously (39).<br>
Association and dissociation constants were deduced from kinetic data using the<br>
Bioevaluation program version 3.1 (Pharmacia).<br>
Confocal laser scanning microscopy<br>
Cells were seeded into Lab Tek chamber slides (Nalge Nunc, Naperville,<br>
DL) and allowed to grow and adhere to the glass slides for 48 h at 37° C, 5% CO2.<br>
Cells were fixed with ice-cold 96% ethanol for 5 min, washed 3x with PBS and<br>
blocked with 10% normal goat serum in PBS for 1 h at room temperature. COU-<br>
1 (5 μg/ml) together with either mouse anti-K8 antibody (1/1000) or mouse anti<br>
K-18 antibody (1/1000) were incubated overnight at 4 °C. After washing with<br>
PBS, the cells were incubated with FITC-labeled goat-anti-human γ-chain and<br>
Texas Red-labeled goat anti-mouse IgG antibody (diluted 1/200, both from<br>
Jackson ImmunoResearch, West Grove, PA) for 1 h at room temperature in the<br>
dark. The cells were washed with PBS for 3 x 5 min and the slides mounted with<br><br>
anti-fading reagent Slow Fade™ in PBS/glycerol (Molecular Probes, Eugene,<br>
OR). Results were analyzed using a MRC-1024 confocal laser scanning<br>
microscope (Bio-Rad), attached to a Zeiss Anyvert 100TV. As a control, all.<br>
experiments were also performed omitting the primary antibody or including<br>
species and isotype matched control antibody instead of the primary antibody. In<br>
addition, differential interference contrast (DIC) images of analyzed cells were<br>
obtained.<br>
Results<br>
Purification of cytokeratins from colon cancer and normal colon<br>
epithelia<br>
Fresh, surgically-removed colon cancer tissue and normal colon epithelia<br>
was used to separately extract cytokeratin K8 an K18 by taking advantage of the<br>
fact that cytokeratins and other cytoskeletal proteins are present as insoluble<br>
filamentous structures in buffer at physiological salt concentrations. Non-ionic<br>
detergent was added to the buffer to improve homogenization, partially by<br>
disrupting cell membranes. The insoluble intermediate filaments proteins were<br>
precipitated by centrifugation, subsequently solubilized in an SDS -containing<br>
buffer, and separated by QFF anion-exchange chromatography using a linear salt<br>
gradient.<br>
Figure 1A shows the elution profile from the QFF anion-exchange<br>
column. The fractions containing COU-1 reactivity were found in the first and<br>
second peak of the gradient (fractions 41-48). COU-1 reactivity was detected by<br>
coating the proteins in these fractions onto ELISA wells followed by incubation<br>
with COU-1 (Figure 1B). Western blot analysis demonstrated reactivity of COU-<br>
1 with three main bands in the same fractions (Figure 1D). The proteins in these<br>
three bands represented only a portion of the proteins with molecular weight in<br>
the 41-46 kDa range found in these fractions, as revealed by Coomassie staining<br>
of the SDS-separated gels (Figure 1C).<br>
Western blot analysis and Coomassie staining of cytokeratin purified<br>
from colon cancers of four different patients revealed a similar pattern of protein<br>
bands, reactive and non-reactive with HMab COU-1. Cytokeratin was also<br><br>
isolated from normal colon epithelia obtained from three individuals using the<br>
same purification procedure to compare the nature of the KS/K18 in colon cancer<br>
versus normal colon tissues. Tissue homogenate and purified cytokeratin<br>
preparations (QFF eluate) from the two sources were examined by Western<br>
blotting using a panel of anti-K8 and anti-K18 antibodies. When approximately<br>
equal amounts of cytokeratin from cancer and the normal epithelia were<br>
analyzed, protein bands (in the 42-46 kDa range) of equal intensity were<br>
observed following staining with the anti-K18 antibody, CYK-90, which<br>
recognizes a linear epitope in the C-terminal part of K18 (Figure 2). In contrast,<br>
when the same preparations were stained with COU-1, three bands at molecular<br>
weights of 42-46 kDa were stained in the cytokeratin preparations from the colon<br>
cancer tissue but not from the normal colon epithelia (Figure 2).<br>
To determine the nature of the different K8/K18-like proteins found in<br>
the colon cancer tissues, an improved separation of the individual protein bands<br>
of the purified cytokeratin preparations was used. Purified cytokeratin<br>
preparations from different colon cancer tissues were, therefore, individually<br>
separated on large 14% SDS-PAGE gels, the proteins were blotted onto filters<br>
and Coomassie stained. Strips of the blot were incubated with either the anti-K8<br>
antibody M20, the anti-K18 antibody CY-90, or the COU-1. Figure 3 A exhibits a<br>
typical blot of a colon cancer tissue sample, displaying approximately 10<br>
different bands visualized by Coomassie staining. At this increased separation, 5<br>
bands showed clear COU-1 reactivity. Additional bands, not stained with COU-<br>
1, were stained either with the anti-K8 antibody, the anti-K18 antibody or both<br>
(Figure 3A).<br>
All ten bands were N-termmal sequenced. As shown in Figure 3B, the<br>
bands corresponded to different forms of K8, K18 and K19, except for one band<br>
that was identified as migration inhibitory factor-related protein 8 (MRPS, also<br>
known as calretinin), a calcium binding protein that may bind to cytokeratins<br>
(40). Most of the bands were N-tenninally truncated K8 or K18, as<br>
demonstrated by the identified amino acid sequence starting at residue 23 to 76,<br>
instead of at the expected residue 1.<br><br>
The amino terminal truncations of K8 corresponded to residues 23, 40,<br>
66, and 76, while the truncations of K18 corresponded to residues 50 and 68.<br>
Significantly, the K8 and K18 truncations were found at the same residue in<br>
three different colon cancers, indicating that the truncations were caused by<br>
specific proteases.<br>
Analysis of the sequence surrounding the cleavage site suggested that at<br>
least two different proteases were responsible for the cleavage, including one<br>
trypsin-like protease. The bands recognized by COU-1 were N-terminally<br>
truncated K8 and K18. Interestingly, however, not all the N-tenninal truncated<br>
K8 and K18 proteins were recognized by COU-1. For example, no COU-1<br>
binding was observed to an N-terminally truncated K8 protein where the first 22<br>
amino acids were missing, nor did the antibody react with intact K8 or K18. The<br>
latter two observations were made by staining with anti-K8 and anti-K18<br>
antibodies, respectively (bands 1 and 3 in Figure 3A), but not by N-terminal<br>
sequencing, because the proteins were N-terminally blocked (K18 contains an<br>
acetylated serine at its N-terminus).<br>
Mapping the COU-1 epitope using recombinant K8 and K18<br>
fragments<br>
To detail the nature of the epitope recognized by COU-1, this epitope was<br>
mapped using a panel of recombinant N- or C-terminally-deleted K8 and K18<br>
fragments or intact K8 and K18 expressed as GST-fusion proteins. The nature of<br>
these fragments is depicted in Figure 4.<br>
Initially, the panel of K8 and K18 fragments were separated by SDS-<br>
PAGE and blotted onto PVDF membranes. Subsequent analyses of these<br>
Western blots, surprisingly, showed that the COU-1 antibody did not bind to any<br>
of the individual K8 or K18 fragments- Nor did the COU-1 antibody bind to the<br>
intact K8 or K18 molecules (Figures 5B and 6B).<br>
In each experiment, MCF7 cell lysate was included as a positive control,<br>
providing positively reacting bands at molecular weights of 42-46 kDa. To<br>
assure that the K8 and K18 fragment were evenly expressed, gels containing the<br>
fragment panel were run in parallel and the gels for the Western blots were<br><br>
stained with Coomassie blue (Figure 6A). Moreover, blots of SDS-PAGE-<br>
separated K8/GST or K18/GST fusion proteins were stained with an anti-GST<br>
antibody (Figure 5A). The results demonstrated an approximately even<br>
expression of the different fusion proteins, and that the lack of signal with COU1<br>
was not due to a low expression level of the cytokeratin fragments or to<br>
incomplete transfer of proteins.<br>
In addition, Mabs to K8 and K18, respectively, were tested for binding to<br>
the panel of K8 or K18 fragments. As shown in Figure 5D, the anti-K18 Mab<br>
reacted strongly with K18(l-356), Kl8(l-385) and intact K18, but not with<br>
K18(l-312), indicating that its epitope was located in the region 312-356, Next,<br>
cytokeratins K8 and K18 complexes were tested to see whether those complexes<br>
were recognized by COU-1. Western blots of the panel of K18 fragments were<br>
incubated with the intact purified K8 and the unbound K8 was washed away<br>
before staining with COU-1 (Figure 5C). COU-1 bound strongly to complexes<br>
formed between intact K8 and the K18 fragments Kl8(1-213) through K18(l-<br>
385). In contrast, COU-1 bound only weakly to intact K8/K18(1-187) and intact<br>
K8/intact K18, and no binding was observed to intact K8/K18(1-65) and intact<br>
K8/K18(l-124) (Figure 5C).<br>
Likewise, blots containing the panel of K8 fragments were incubated<br>
with intact K18 before staining with COU-1. COU-1 bound strongly to<br>
complexes formed between the intact K18 and the K8 fragments K8(l-213)<br>
through K8(l-385). In contrast, COU-1 bound weakly to intact K8/intact K18<br>
and no binding was observed for K8(l-65)/intact K18 complexes (Figure 6C).<br>
N-terminal sequencing demonstrated that both K8 and K18 proteins from<br>
colon cancer patients were truncated. Experiments were performed to identify<br>
the K8/K18 heterotypic epitope bound by COU-1. In parallel, Western blots<br>
containing the C-terminal-deleted fragments surrounding the COU-1 epitope,<br>
K18(l-72), K18(l-124), K18(l-187) and intact K18 were generated. These blots<br>
were then incubated with one of the K8 fragments surrounding the COU-1<br>
epitope, K8(l-85), K8(l-129) or K8(l-233), or the intact K8 protein. After<br>
permitting K8-K18 complex formation, the blots were incubated with COU-1<br>
antibodies.<br><br>
As shown in Figure 7 (A-C), the epitope recognized by COU-1 is not<br>
exposed, or is only minimally exposed, on K18(l-124)/intact K8 or K18(l-<br>
124)/K8(l-233) complexes. In contrast, strong binding of COU-1 was observed<br>
for K18(l-124)/K8(l-129) complexes. No COU-1 binding was observed for any<br>
of the heterotypic complexes containing K8(l-85) or K18(l-72).<br>
Taken together, these results confirm that the epitope recognized by<br>
COU-1 involves the K8 region 85-129 and the K18 region 72-124. As shown in<br>
Figures 4 and 8, this region involves the C-terminal part of the N-terminal head<br>
domain and the N-terminal part of the first helical domain, 1A, of the alpha-<br>
helical rod domain of both K8 and K18.<br>
The results further demonstrate that this epitope is poorly exposed on<br>
heterotypic complexes of intact K8 and K18, even when intact K8 is cornplexed<br>
with Kl8(1-124). The COU-1 epitope is revealed when the first domain, Al, of<br>
the alpha helical rod is not in its normal coil-coil structure. This can be caused by<br>
truncation that removed essential contact points for the existing association<br>
leaving the COU-1 binding region of the K8/K18 complex in an unfolded state.<br>
The combination described above was reversed such that Western blots<br>
of the C-terminal deleted fragments of K8(l-85), K8(l-129), K8(l-233) and<br>
intact K8 were incubated with the fragments of K18 surrounding the COU-1<br>
epitope, K18(l-72), K18(l-124), K18(l-187) and intact K18. These blots were<br>
then incubated with the COU-1 antibody. As shown in Figure 7 (D-F), the<br>
epitope recognized by COU-1 was equally exposed when K8(l-129) was<br>
cornplexed with K18(l-72), K18(l-124), and K18(l-187) or intact K18. Again,<br>
no COU-1 binding was observed with any heterotypic complexes containing<br>
K8(l-85).<br>
COU-1 binding was tested using a panel of heterotypic complexes<br>
consisting of N-terminal deleted K8 and K18 combined with intact K8 and K18<br>
using the heterotypic Western blot assay. These fragments were missing the first<br>
129 atnino acids or more as detailed in Figure 4. However, no COU-1 binding<br>
was observed to any of these N-terminal-deleted heterotypic K8/K18 complexes,<br>
indicating that the COU-1 epitope was located within the N-terminal 129 amino<br><br>
acids (data not shown). The control showed that the N-terminal-deleted<br>
fragments were well recognized by the murine anti-K8 and anti-K18 Mabs.<br>
The N-terminal sequencing data and the recombinant mapping datas<br>
indicated that the COU-1 epitope were well exposed when the first 6.5 amino<br>
acids of K8 and the first 49 amino acids of K18 were missing.<br>
Two additional N-terminal deleted fragments, K8(66-483) and K18(50-<br>
430) were generated as GST fusion proteins. Figure 9 shows blots of intact K8<br>
and K8(66-483) incubated with K18(50-430) (A) or intact K18 (B). Figure 9<br>
also shows the blots of K18(50-430) and intact K18 incubated with K8(66-483)<br>
(C) or intact K8. Significantly stronger COU-1 binding was observed for K8(66-<br>
483)/K18(50-430) and K8(66-483)/intact K18 complexes than for intact<br>
K8/K18(50-430) or intact K8/intact K18 complexes.<br>
Further investigations were made to determine whether the N-terminal<br>
cleavage of K8 and K18 observed in cancer cells might be caused by adenovirus<br>
infection. The adenovirus L3 23-kDa proteinase promotes specific cleaving of<br>
the N-terminal domain of K18, while leaving K8 intact in adenovirus infection<br>
of HeLa cells (41, 42). This cleavage resulted in removal of region 1-73 of the<br>
head-domain of K18 and the disassembly of the cytokeratin network into<br>
spheroid globules. Tests were performed to examine whether the fragmentation<br>
caused by adenovirus infection would result in the conformation change that<br>
allowed COU-1 binding.<br>
Previous data indicated that COU-1 antibodies do not bind to K8/K18<br>
from HeLa cells. Cytokeratin from HeLa cells infected with adenovirus were<br>
purified and separated by SDS-PAGE, demonstrating a band at a molecular<br>
weight of 41 kDa, in accordance with previous reports. Incubation of COU-1<br>
with Western blot of the adenovirus-infected HeLa cells resulted in no staining<br>
(data not shown), suggesting that the cytokeratin fragments found in the<br>
adenocarcinomas were not a result of adenovirus infection.<br>
It seemed clear that the COU-1 epitope was only present when<br>
heterotypic K8/K18 complexes were formed. The epitope is not present on<br>
individual K8 and K18 molecules. However, the question remained as to why<br>
COU-1 binding to Western blots of SDS-separated cancer cell lysate was<br><br>
observed where the K8/K18 complexes may have dissociated. A possible<br>
explanation was that during the incubation steps, part of the different<br>
cytokeratins dissociate from the membrane and subsequently attach to and form<br>
high affinity heterotypic complexes with its complementary cytokeratin still<br>
bound to the membrane.<br>
To examine this hypothesis, Western blots of lysate of the colon cancer<br>
cell line, colon 137, were separated into halves. One half was fixed with ethanol<br>
before incubation with the antibodies, while the other half was processed as<br>
usual without fixation. Staining was observed with anti-K18 antibody (CY-90)<br>
on both the fixed and the unfixed blots, while staining with COU-1 was only<br>
observed on the unfixed blot.<br>
Earlier immunohistochemical studies had showed that ethanol fixation of<br>
tissue sections had no effect on the COU-1 antigen. Dot blots of the cancer lysate<br>
were tested for detection of the cancer-associated epitope with or without<br>
fixation. Staining with COU-1 was observed both with and without fixation,<br>
confirming that the COU-1 epitope was not affected by ethanol treatment. In<br>
conclusion, it seems the initial hypothesis was correct, i.e. that heterodimer<br>
formation of cytokeratins takes place during the development of Western blots<br>
and that such heterodimer formation by partially truncated cytokeratin is required<br>
for the formation of the COU-1 epitope.<br>
COU-1 binding to the different recombinant heterotypic K8/K18<br>
complexes was also measured by ELISA. Purified recombinant fragments of K8<br>
or intact K.8 were combined with purified recombinant fragments of K18 or<br>
intact K18 in a molar ratio of 1:1 to generate heterotypic complexes in urea. The<br>
samples were then dialyzed against PBS to allow the formation of the heterotypic<br>
complex, and coated at 5 μg/ml on ELISA plates. Intact K8 was combined with<br>
K18(l-124), Kl8(1-187), K18(l-213), and intact K18. In addition, intact K18<br>
was combined with K8(l-65), K8(l-85), K8(l-129), and K8(l-233).<br>
COU-1 bound with various intensity to all the complexes in this ELISA<br>
assay, except to K8(l-65)/intact K18 and intact K8(l-85)/ intact K18 complexes.<br>
These data are in accord with the results from Western blot analysis. Figure 10<br>
shows the titration of COU-1 on three of the heterotypic complexes,<br><br>
demonstrating significantly stronger binding to the fragmented K8/K18 than to<br>
the intact K8/K18 complexes.<br>
The kinetic parameters for the binding of COU-1 to different<br>
recombinant heterotypic K8/K18 complexes were measured by real-time<br>
biospecific interaction analysis (BIAcore). COU-1 exhibited high affinity<br>
binding to the heterotypic complexes of K8(l-124)/intact K18 and K8(l-<br><br>
Cellular distribution of truncated heterotypic K8/K18 complexes<br>
To evaluate the cellular distribution of normal K8 and K18 compared to<br>
truncated K8/K18 heterotypic complexes, breast and colon cancer cell lines<br>
MCF-7 and BrCaOl were co-stained with COU-1 and either Mab M20 (anti-K8)<br>
or Mab CY-90 (anti-K18) and an analyzed by high resolution confocal<br>
microscopy (Figure 11 and 12). Mabs M20 and CY-90 both stained long fibers<br>
of intermediate filaments forming complex interconnecting networks. The fibers<br>
emanate from a perinuclear ring, from which the filaments appear to connect to<br>
the nuclear surface and extend throughout the cytoplasm, terminating at the<br>
plasma membrane. In contrast, COU-1 exhibited a speckled pattern, with<br>
staining of short filament fragments and rod-like particles, indicative of<br>
fragmented intermediate filaments.<br>
Examining the staining pattern of MCF7 cells within cell clusters, only<br>
the peripheral, newly-formed, proliferating cells were strongly positive for COU-<br>
1, while all cells were stained with anti-K18 and anti-K8 Mabs (Figure 11).<br>
Within the proliferating cells of a cluster, COU-1 staining was must prominent at<br>
the outward cell surface, facing away from the cluster. In contrast, Mabs M20<br><br>
and CY-90 stained the intermediate filamentous network throughout the cells.<br>
The speckled COU-1 staining was seen in close association to the intact<br>
intermediate filament network, as determined by overlay of images stained with<br>
COU-1 and Mab M20 or Mab CY-90 (Figure 12).<br>
Accordingly, N-terminally truncated forms of K8/K18 complexes were<br>
identified only in cancerous epithelia, whereas intact K8/K18 complexes were<br>
observed in both normal and cancerous simple glandular epithelia. The cleavage<br>
of both cytokeratin K8 and cytokeratin K18 at identical sites in different cancer<br>
patients indicates that specific proteases are involved. The cleavage sites at<br>
amino acids 22 and 40 on K8, and at amino acid 50 on K18, all contained the<br>
(S/F/V)XSR↓X(S/V) (SEQ ID NO:50) consensus sequence, suggesting that the<br>
enzyme responsible for these cleavages is a trypsin-like protease (Figure 8).<br>
Analysis of the amino acid sequences in the vicinity of the cleavage sites<br>
revealed one other site on K8 that had the same general sequence (amino acid 32,<br>
GSR↓I (SEQ ID NO:64), but was not cleaved. This suggests that the amino acids<br>
at P3 or P1' positions of the substrate are also influencing the recognition by this<br>
protease.<br>
A consensus sequence was not apparent at the three remaining cleavage<br>
sites on K8 and K18 (TAV↓T (SEQ ID NO:51), SPL↓V (SEQ ID NO:52),<br>
TGI↓A (SEQ ID NO:53)). A protease that requires less stringent recognition<br>
conditions or several different proteases may be responsible for these cleavages.<br>
One such protease may be elastase-type protease that accepts valine, leucine and<br>
isoleucine in the P1 position.<br>
It is unlikely that cleavage of cytokeratins K8 and K18 fragments<br>
occurred during the purification of cytokeratin from the tissue samples for<br>
several reasons. First, a cocktail of five enzyme-inhibitors was present at all<br>
times. Second, cytokeratin fragments were not observed following purification of<br>
cytokeratin from normal colon epithelia using identical purification conditions.<br>
Third, the HMab COU-1, which only recognizes the truncated form of K8/K18,<br>
can detect its epitope in cancerous, but not in normal, epithelia when tissue<br>
samples were minimally handled and immediately fixed.<br><br>
In contrast to the earlier views, the maintenance of the cytokeratin<br>
network in epithelial cells is a dynamic process involving constant restructuring<br>
by assembly and disassembly of intermediate bundles (45). Microinjection of<br>
biotin-labeled cytokeratin or transfection with fluorescence-labeled cytokeratin<br>
has demonstrated an inward-directed flow of diffuse material at the cell<br>
periphery moving in the form of dots and thin filaments towards the deeper<br>
cytoplasm, where it coalesces with other filaments and filament bundles (46).<br>
While this process occurs in both normal and malignant epithelia cells, the<br>
results provided by the invention indicate the presence of a second degradation<br>
pathway specifically within cancer cells.<br>
Also according to the invention, the human antibody, COU-1, cloned<br>
from a tumor-draining lymph node of a colon cancer patient, specifically<br>
recognizes the N-terminal truncated form of K8 and K18 when the two<br>
cytokeratins formed a heterotypic complex. Previous analysis of COU-1<br>
indicated selective reaction of COU-1 with K18 (35,48), or a modifiedKlS (31,<br>
32,49). Proteolytic cleavage of K18 in association with apoptosis has been<br>
reported (56). However, the cleavage sites for the apoptotic proteases, caspase-<br>
3, -6 and -7, are located in the conserved L1-2 linker and in C-terminal tail<br>
domain, and quite distant to the N-tenninal cleavage sites, as we have studied in<br>
vital tumor tissue (56). Recently, an antibody (M30) was reported to recognize a<br>
neoepitope only exposed in apoptotic cancer cells and not vital or necrotic cells<br>
(57). The neoepitope become exposed when the C-terminus tail domain was<br>
liberated after cleavage by caspase-3, -6 or -7 into 26,22 and 19 kDa fragments.<br>
The cleavage sites observed in colon cancer cells were also different from the<br>
one reported for adenovirus infected HeLa cells, where the N-terminal 73 amino<br>
acids of K18 were removed (41,42). Surprisingly, no COU-1 binding to cleaved<br>
K8/K18 heterotypic complexes from infected HeLa cells was observed, while<br>
COU-1 bound K8/K18 complexes where the 67 most N-terminal amino acids of<br>
K18 were removed. This suggests, although the cleavage sites seem close,<br>
additional removal of 6 amino acids may cause conformational changes that<br>
prevent COU-1 from binding.<br><br>
Some evidence indicates that K8/K18 is intimately associated with cell<br>
migration and invasiveness. N-terminal cleavage of K8/K18 may influence these<br>
processes. Moreover, the missing N-terminal head domain of K8/K18 contains<br>
several important phosphorylation sites, including ser52 on K.18, which has been<br>
associated with filament reorganization and compartment localization and a<br>
second phosphorylation site important for binding to the 14-3-3 protein (58, 59).<br>
In K8 the phosphorylation site ser23 has been associated with mitogen activation<br>
(60).<br>
The abbreviations used herein are: K8, cytokeratin 8; K18, cytokeratin<br>
18; IF, intermediate filaments; HMab, human monoclonal antibody; FCS, fetal<br>
calf serum; AP, alkaline phosphatase; QFF, Q-Sepharose fast flow; ELISA,<br>
enzyme-linked immunosorbent assay; PBS, phosphate-buffered saline; TBS,<br>
Tris-buffered saline; PVDF, polyvinylidene difluoride membranes; FITC,<br>
fluorescein isothiocyanate.<br>
EXAMPLE 3: Cloning Antibody Fragments that Bind<br>
the Cancer-Associated Epitope<br>
To further develop antibodies useful for detection of cancer, nucleic acids<br>
encoding portions of antibodies were cloned and screened by phage display<br>
selection for binding to the cancer-associated epitope of the invention. These<br>
nucleic acids encode human Fab and other fragments.<br>
Materials and Methods<br>
Antibodies. The human monoclonal IgM antibody, COU-1, is secreted<br>
by the hybridoma cell line, B9165, derived by fusing the human lymphoblastoid<br>
cell line WI-L2-729-HF2 and lymphocytes obtained from mesenteric lymph<br>
nodes from a patient with colon cancer as described above. See also Borup-<br>
Christensen, P., Erb, K., Jensenius, J. C, Nielsen, B. &amp; Svehag, S. -E. (1986)<br>
Int. J. Cancer 37, 683-688. The human-human hybridoma cell line was grown in<br>
protein free medium: RPMI1640 medium (G1BCO, Grand Island, N. Y.)<br>
supplemented with SSR3 serum replacement (Medicult, Copenhagen, Denmark).<br>
The COU-1 antibody was purified from cell culture supernatant by affinity<br>
chromatography on Sepharose-coupled murine monoclonal anti-human u chain<br><br>
antibody (HB57, American Type Culture Collections, Rockville, MD). The<br>
antibody was eluted with 0.1 M diethylamine, pH 10.5, followed by fractionation<br>
by FPLC. IgM purified from nonnal human serum (Cappel, Cochranville, PA.)<br>
was used as a control.<br>
The human monoclonal IgM antibody, 16.88 was obtained from Dr. R.<br>
McCabe. See Haspel, et al., (1985) Cancer Res. 45, 3951-3961. This antibody<br>
has been used successfully for tumor imaging in humans. See Steis et al. (1990)<br>
J. Clin. Oncol. 8,476-490; Boven et al. (1991) Eur. J. Cancer 27,1430-1436;<br>
Rosenblum et al. (1994) Cancer Immunol. Immunother. 39,397-400). Two<br>
murine monoclonal antibodies, M20 directed against normal cytokeratin 8 and<br>
CY-90 directed against normal cytokeratin 18, were obtained from Sigma<br>
Chemical Co. (St. Louis, MO).<br>
PCR amplification and cloning of the variable heavy and light chain<br>
genes. Total RNA was prepared from the B9165 hybridoma cell line by the<br>
guanidinium method. After reverse transcription, the µ (Fd region) and K chains<br>
were amplified by the polymerase chain reaction (PCR) using a set of family-<br>
specific primers using methods described in Persson et al., (1991) Proc. Natl.<br>
Acad. Sci. USA 88,2432-2436. The amplified light chain DNA was cut with the<br>
restriction enzymes Sac I and Xba I and ligated with Sac I/Xba 1-lmea.rized<br>
pComb3 vector for 3 h as described in Burton et al., (1991) Proc. Nail. Acad. Sci.<br>
USA 88,10134-10137, and Barbas et al. (1991) Proc. Natl. Acad. Sci. USA 88,<br>
7978-7982. The ligated material was purified and transformed by electroporation<br>
into 200 µl Escherichia coli XL1-Blue cells. After transformation, the cells were<br>
grown overnight and phagemid DNA was prepared.<br>
Subsequently, the PCR amplified heavy chain and isolated phagemid<br>
DNA containing the light chain were digested with the restriction enzymes Spe I<br>
and Xho I. The heavy chain phagemid fragments were ligated and used to<br>
transform XLl-Blue. The Fab library was grown in SOC medium for 1 h at 37°C<br>
following addition of SB medium containing carbenicillin (50 μg/ml) and<br>
tetracycline (10 μg/ml). After 3 h, helper phage VCS-M13 (1012 plaque-forming<br>
units) was added and the culture was shaken for an additional 2 h. Kanamycin (70<br><br>
µg/ml) was added and the culture was incubated at 30°C overnight. The<br>
supernatant was cleared by centrifugation (4000 x g for 20 min) at 4°C. Phage<br>
were precipitated by a second round of centrifugation after the addition of 5%<br>
polyethylene glycol and 0.15 M NaCl and incubation on ice for 30 min. Phage<br>
pellets were resuspended in phosphate-buffered saline, pH 7.4 (PBS) containing<br>
1% (w/v) bovine serum albumin (BSA) and centrifuged for 3 min at 10,000 x g to<br>
pellet debris.<br>
Enrichment of antigen-binding phage through panning. Panning of<br>
the B9165 antibody library was carried out using methods described in Burton et<br>
al. (1991) Proc. Natl. Acad. Sci. USA 88,10134-10137. In brief, microtiter wells<br>
were coated overnight with ultrasonicated lysate of a colon cancer cell line<br>
(colol37) in 0.1 M bicarbonate buffer, pH 8.6 at 4°C. See Ditzel et al. (1992)<br>
Eur. J. Nucl. Med. 19,409-417. Following blocking with PBS containing 3%<br>
BSA for 1 h at 37°C, 50 ul phage suspension in PBS was added to each well and<br>
incubated for 2 h. Unbound phage were removed by vigorous washing 10 times<br>
with PBS containing 0.05% (w/v) Tween 20 (PBS-Tween)(Merck, Darmstadt,<br>
FRG). Bound phage, enriched for those bearing antigen-binding Fabs, were eluted<br>
with 0.2 M glycine/HCl, pH 2.2. The eluted phage were amplified by infection of<br>
E. coli and recovered by superinfection with VCS-M13 helper phage. The<br>
panning procedure was carried out twice. Phagemid DNA was isolated from the<br>
last round of panning, cut with NheI and SpeI and religated. This step excised the<br>
cpIII gene, resulting in a vector producing soluble Fab fragments.<br>
ELISA analysis of B9165 Fab and intact antibodies. Fabs were<br>
prepared as bacterial supematants through a freeze-thawing procedure, using<br>
methods reported by Burton et al. (1991) Proc. Natl. Acad. Sci. USA 88,10134-<br>
10137, and Barbas et al. (1991) Proc. Natl. Acad. Sci. USA 88, 7978-7982.<br>
To assess specificity, supematants and purified Fabs were screened in an<br>
ELISA system for binding to ultrasonicates of colon cancer cells (Colon 137),<br>
BSA (Sigma), ovalbumin (Sigma), recombinant HIV-1 gpl20 (IIIB) (Intracel,<br>
Issaquah, WA) and human placental DNA (Sigma). ELISA wells (Costar) were<br><br>
coated overnight at 4 °C with 50 µl of antigen (1-10 μg/ml) in 0.1 M bicarbonate<br>
buffer, pH 8.6. DNA in PBS was dried on the ELISA wells at 37°C. The wells<br>
were washed twice with PBS, blocked by filling the wells with 3% BSA in-PBS<br>
for 1 h at 37°C, and incubated with human Fab samples or intact human IgM<br>
antibody for 2 h at 37°C. Plates were washed 10 times with PBS-Tween and<br>
bound Fab was detected with alkaline phosphatase (AP) labeled goat anti-human<br>
IgG F(ab')2 (Pierce Chemical Co, Rockford, EL) diluted 500 fold in PBS or<br>
alkaline phosphatase-labeled rabbit anti-human K-chain (Sigma) diluted 1000 fold<br>
in PBS. Bound antibody was visualized with para-nitrophenylphosphate<br>
(Sigma)(l mg/ml, 1 mM MgCl2,10% (w/v) diethanolamine, pH 9.6) and read at<br>
405 nm.<br>
Purification of Fab. Recombinant B9165 Fab was purified losing methods<br>
described in Ditzel et al. (1995) J. Immunol. 154, 895-908 with some<br>
modifications. In brief, E. coli containing the appropriate clone was inoculated<br>
into one liter cultures of superbroth containing carbenicillin (50 μg/ml),<br>
tetracycline (10µg/ml) and MgCl2 (20 mM), and grown at 37°C, with shaking, for<br>
6 h. Protein expression was then induced with 2 mM isopropyl p-D-<br>
thiogalactopyranoside and growth continued at 30°C overnight. Soluble Fab was<br>
purified from bacterial supernatants by affinity chromatography using a goat<br>
antibody against human IgG F(ab')2 (Pierce) cross-linked to protein G<br>
gammabind matrix (Pharmacia). The column was washed with PBS and bound<br>
Fab eluted with 0.2 M glycine/HCl, pH 2.2, and immediately neutralized with 1<br>
M Tris/HCl, pH 9.0.<br>
Nucleotide sequencing. Sequencing was carried out on a 373A automated<br>
DNA sequencer (ABI, Foster City, Ca) using a Taq fluorescent dideoxy<br>
terminator cycle sequencing kit (ABI). Primers for the elucidation of light chain<br>
sequence were the SEQKb primer (5'-ATAGAAGTTGTTCAGCAGGCA-3',<br>
SEQ ID NO.41), hybridizing to the (+) strand and the KEF primer (5'-<br>
GAATTCTAAACTAGCTAGTTCG-3', SEQ ID NO:42) hybridizing to the (-)<br>
strand. For the heavy chain, the CMHD primer (5'-<br><br><br>
Analysis by confocal laser scanning microscopy. Human colon cancer<br>
cell-lines (H3619 and colol37), and breast cancer cell-lines (MCF-7 and H3396)<br>
were grown in Iscove's modified Dulbecco's medium containing 10% FBS and<br>
allowed to adhere to chambered coverslips (Nunc, Kamstrup, Denmark) for 48 h<br>
at 37°C, 5% CO2, in order to form monolayers. Experiments were performed<br>
using the primary COU-1 antibodies, B9165 Fab, murine anti-cytokeratin S,<br>
murine anti-cytokeratin 18, and HuMab 16.88 as indicated below. All antibodies<br>
were tested at 10 µg/ml except B9165 Fab (30 μg/ml).<br>
1)	Intracellular staining. H3619 and colo 137 cells were permeabilized<br>
with methanol at -20°C for 5 min, blocked with normal goat serum followed by<br>
incubation with primary antibodies at room temperature for 1 h. The cells were<br>
then washed 3 times with culture medium and incubated with FITC-labeled goat<br>
anti-human K-chain antibody (Southern biotech) or FITC-labeled goat anti-mouse<br>
IgG (BioSource) diluted 1:100 and 1:50 respectively in PBS for 1 h at room<br>
temperature.<br>
2)	Surface staining. Live H3619 cells were incubated with COU-1<br>
antibodies at 4°C for 2 h, washed 3 times with cold culture medium and incubated<br>
with secondary FITC-labeled antibody at 4°C for 1 h.<br>
3)	Internalization. Live H3619 and colol37 cells were incubated with<br>
COU-1 antibodies or B9165 Fab at 37°C for 6 h, followed by washing 3 times<br>
and permeabilization with methanol at -20°C for 5 min. Cells were blocked with<br>
normal goat serum and incubated with secondary FITC-labeled antibody at RT for<br>
1 h. For all experiments, following primary and secondary antibody incubations,<br>
the cells were washed, fixed with 2% paraformaldehyde in PBS for 15 min at<br>
room temperature, washed twice and mounted in anti-fading reagent (30mM<br>
dithioerythritol:PBS:glycerol, 2:9:1). Staining of cells was evaluated by confocal<br>
laser scanning microscopy. As control all experiments were carried out omitting<br>
the primary antibody.<br><br>
Immunohistochemical analysis. Tissue specimens were obtained from<br>
colorectal cancer patients undergoing surgical resection. Normal colon tissue was<br>
taken from the resectate approximately 15 cm away from the site of the tumor.<br>
Tissues were fixed in 96% alcohol for 6 h at 4°C. Afterwards, tissues were<br>
paraffin embedded and cut into 5 um sections. Sections were deparaffinized in<br>
xylol, rehydrated through graded alcohol and washed in PBS-Tween. Sections<br>
were incubated for 2 h at room temperature in a humidified chamber with 100 µl<br>
of murine monoclonal antibody, human monoclonal IgM antibody or normal<br>
polyclonal human IgM, all at 0.5-10 μg/ml. The slides were washed and<br>
incubated with AP-labeled rabbit anti-human IgM (Dako, Glostrup, Denmark),<br>
horse-radish peroxidase (HRP) labeled rabbit anti-human IgM (Dako) or HRP-<br>
labeled rabbit anti-mouse IgG (Dako) diluted in PBS with 10% (w/v) bovine<br>
serum albumin for 1 h at room temperature. After washing, the HRP was<br>
visualized by development with chromogenic substrate (0.6 mg<br>
diaminobenzidine per ml PBS with 0.01% H2O2) and AP with 0.2 mg naphthol-<br>
AS-Mx phosphate (Sigma), 1 mg Fast Red TR Salt (Sigma), 20µg<br>
dimethylformamide per ml 0.1M Tris/HCl, 1M levamisole, pH 8.2. The sections<br>
were counterstained with Mayer's haematoxylin, dehydrated in xylene and<br>
mounted in Aquamount (Gurr, Poole, England). The staining intensity was<br>
graded as follows: (-) no staining, (+) weak staining, (++) moderate staining,<br>
(+++) strong staining.<br>
Results<br>
Phage display expression and sequencing of HuMab that can bind the<br>
Cancer-Associated Epitope. RNA was extracted from the B9165 cell line and<br>
the heavy (µ, Fd region) and light (k)-chain genes from the corresponding cDNA<br>
were amplified by PCR using 3' family specific primers and a 5' constant primer.<br>
The light and heavy chain products were then sequentially cloned into the M13<br>
phage surface expression vector pComb3 to generate a library of 2x106 members.<br>
The phage library was selected twice on an ultrasonicate of the COU-1 antigen<br>
positive colon cancer cell line (colon 137). Eluted phage from the last round of<br><br>
selection were used to infect E. coli XLI-blue cells. DNA was prepared from<br>
these cells and gene H1 fragment removed by NheI/SpeI digestion and ligation.<br>
The reconstructed phagemids were used to transform XLI-Blue to produce clones<br>
secreting soluble Fab fragments. Supernatants of three of the 80 single Fab<br>
expression clones tested, exhibited binding to colon 137 lysate and no binding to<br>
ovalbumin in ELISA.<br>
The sequences of these three clones were identical. Sequence analysis<br>
showed that the B9165 hybridoma cell light chain belongs to the VKIII family<br>
and that it exhibits 97% (269/276) nucleotide homology to L6 as closest germ-<br>
line (Figure 13). The B9165 light chain contained an extra serine inserted<br>
corresponding to codon 30. The light chain J segment showed 95% (36/38)<br>
nucleotide homology to the germ-line JK5 segment. Further, sequence analysis<br>
showed that the heavy chain belongs to the VHI family, exhibiting 98%<br>
nucleotide homology to the heavy chain germ-line DP-7. The heavy chain J<br>
segment showed 96% (53/55) nucleotide homology to the germ-line JH6b<br>
segment. The D segment of COU-1 showed closest homology to the D2 germ-line<br>
D segment with a 16 nucleotide stretch of complete homology.<br>
Purified recombinant B9165 Fab was tested in parallel with the intact<br>
COU-1 antibodies and normal polyclonal IgM for binding to lysate of colon<br>
cancer cells (colol37) and irrelevant antigens in ELISA. The B9165 Fab and<br>
COU-1 exhibited strong binding to colon 137 lysate, but not to a panel of other<br>
antigens including BSA, ovalbumin, human DNA and HIV-1 gpl20 (data not<br>
shown). In contrast, normal human IgM did not bind to any of the antigens. The<br>
concentration needed for saturation was significantly higher for the B9165 Fab<br>
(20 μg/ml) than for the intact antibody (1 μg/ml) and was similar to that<br>
previously measured for chemical derived half-monomeric fragments, exhibiting<br>
a Ka of 2x106 M-1 (Ditzel, H., Erb, K., Leslie, G. &amp; Jensenius, J. C. (1993) Hum.<br>
Antibod. Hybridomas 4, 86-93).<br>
COU-1 binds preferentially to malignant carcinoma cells. The<br>
subcellular localization of the antigen recognized by COU-1 in tissue biopsies of<br>
colon and rectal adenocarcinomas was studied using an indirect<br><br>
The antibodies were compared for staining of colon metastases in liver<br>
versus surrounding normal liver tissue. COU-1 gave intense staining of the<br>
immunoperoxidase and alkaline-phosphatase techniques. At high magnification,<br>
distinct fibrillar staining of intermediated filaments by COU-1 was observed. In<br>
small cell clusters or individual cells, intense staining was seen at the periphery,<br>
possibly associated with the cell surface. In addition, enhanced staining associated<br>
with the junctional zone between adjacent cells was seen. No staining was<br>
observed in adjacent normal colon crypt epithelial cells in five of eight colon or<br>
rectal cancer. In the other three cancers, weak staining of a few individual cell<br>
surrounded by negative cells was observed in adjacent morphologically normal<br>
colon tissues in addition to strong staining of the cancer tissue. Although these<br>
colon epithelia looked morphologically normal, this may not be the case. Murine<br>
anti-cytokeratin 8 antibodies and anti-cytokeratin 18 antibodies (not shown) gave<br>
intense staining of the adjacent normal colon epithelia as well as of the colon<br>
cancer tissue. COU-1, however, reacted only with the malignant cells and not<br>
with the normal epithelia. A comparison of the staining levels for COU-1, murine<br>
anti-cytokeratin 8 and 18, and 16.88 is given in Table 3. The 16.88 antibody<br>
showed strong staining of the colon cancer cells, weak staining only in some areas<br>
of the normal colon epithelia, but in addition stained the smooth muscle fibers<br>
and myoepithelia derived connective tissue was observed (Table 5).<br><br><br>
metastasis whereas no staining of the majority of hepatocytes was observed. A<br>
few hepatocytes in the periportal zones were weakly positive. Similarly, the 16.88<br>
antibody did not stain the majority of the hepatocytes. However, the myoepithelia<br>
connective tissue was stained by 16.88, but not with COU-1. Both human<br>
antibodies stained the biliary ducts. The murine anti-cytokeratin 8 and 18 (not<br>
shown) antibodies stained the metastases as well as the normal hepatocytes<br>
strongly and with equal intensity. The staining decreased towards the<br>
centrilobular area. Particular strong staining was seen associated with the cell<br>
membrane of the hepatocytes with the murine Mabs.<br>
Phage display and bacterial expression was therefore used to clone and<br>
further characterize Fab and other antibody fragments from a hybridoma cell line<br>
expressing the human monoclonal antibody COU-1. The binding characteristics<br>
of the cloned B9165 Fab were very similar to previous reports for the half-<br>
monomeric fragments generated by chemical reduction and alkylation (Ditzel, H.,<br>
Erb, K., Leslie, G. &amp; Jensenius, J. C. (1993) Hum. Antibod. Hybridomas 4, 86-<br>
93). Sequence analysis showed that the variable region of the heavy and light<br>
chain had minimal somatic mutations with 98% and 97% nucleotide homology to<br>
the closest germ-line V genes, respectively. This is in accordance with COU-1<br>
being an IgM antibody, and indicates that substantial affinity maturation through<br>
site directed mutagenesis is possible.<br>
The foregoing specification, including the specific embodiments and<br>
examples, is intended to be illustrative of the present invention and is not to be<br>
taken as limiting. Numerous other variations and modifications can be effected<br>
without departing from the true spirit and scope of the present invention.<br>
References<br>
The following references, to the extent that they provide exemplary<br>
procedural or other details supplementary to those set forth herein, are<br>
specifically incorporated herein by reference.<br>
1. Schmidt, A., Heid, H. W., Schafer, S., Nuber, U. A., and Zimbelmann, R.<br>
Desmosomes and cytoskeletal architecture in epithelial differentiation:<br>
cell type-specific plaque components and intermediate filament<br>
anchorage. European Journal of Cell Biology, 65:229-245,1994.<br><br>
2.	Fuchs, E. and Cleveland, D. W. A structural scaffolding of intermediate<br>
filaments in health and disease. Science, 279: 514-519,1998.<br>
3.	Hatzfeld, M. and Franke, W. W. Pair formation and promiscuity of<br>
cytokeratins: formation in vitro of heterotypic complexes and<br>
intermediate-sized filaments by homologous and heterologous<br>
recombinations of purified polypeptides. J. Cell Biol., 101: 1826-1841,<br>
1985.<br>
4.	Fuchs, E. and Weber, K. Intermediate filaments: structure, dynamics,<br>
function, and disease. Annu. Rev. Biochem., 63: 345-382,1994.<br>
5.	Moll, R. Cytokeratins as markers of differentiation in the diagnosis of<br>
epithelial tumors. Subcell. Biochem., 31:205-262,1998.<br>
6.	Steinert, P. M. and Roop, D. R. Molecular and cellular biology of<br>
intermediate filaments. Ann. Rev. Biochem., 57: 593-625,1988.<br>
7.	Hatzfeld, M. and Weber, K. The coiled coil of in vitro assembled keratin<br>
filaments is a heterodimer of type I &amp; II keratins; use of site-specific<br>
mutagenesis and recombinant protein expression. J. Cell Biol., 110:<br>
1199-1210,1990.<br><br>
8.	Steinert, P. M. The two-chain coiled-coil molecule of native epidermal<br>
keratin intermediate filaments is a type I-type II heterodimer. J. Biol.<br>
Chem., 265: 8766-8774,1990.<br>
9.	Traub, P. Intermediate filaments: a review, vol. ed. Heidelberg,<br>
Germany: Springer-Verlag, 1985.<br><br>
10.	Hatzfeld, M. and Weber, K. Tailless keratins assemble into regular<br>
intermediate filaments in vitro. J. Cell Sci., 97: 317-324,1990.<br>
11.	Shoeman, R. L., Mothes, E., Kesselmeier, C, and Traub, P. Intermediate<br>
filament assembly and stability in vitro: effect and implications of the<br>
removal of head and tail domains of vimentin by the human<br><br>
immunodeficiency virus type I protease. Cell Biol. Int. Rep., 14: 583-594,<br>
1990.<br>
12.	Bader, B. L., Magin, T. M, Freudenmann, M., Stumpp, S., and Franke,<br>
W. W. Intermediate filaments formed de novo from tail-less cytokeratins<br>
in the cytoplasm and in the nucleus. J. Cell Biol., 115:1293-1307, 1991.<br>
13.	Hatzfeld, M. and Burba, M. Function of type I and type II keratin head<br>
domains: their role in dimer, tetramer and filament formation. J. Cell<br>
Science, 107: 1959-1972,1994.<br>
14.	Ku, N. O., Michie, S., Oshima, R. G., and Omary, M. B. Chronic<br>
hepatitis, hepatocyte fragility, and increased soluble<br>
phosphoglycokeratins in trans genie mice expressing a keratin 18<br>
conserved arginine mutant. J. Cell Biology, 131: 1303-1314, 1995.<br>
15.	Ku, N. O., Gish, R., Wright, T. L., and Omary, M. B. Keratin 8 mutations<br>
in patients with cryptogenic liver disease. N. Engl. J. Med., 344: 1580-<br>
1587,2001.<br>
16.	Baribault, H., Price, J., and Oshima, R. G. Mid-gestational lethality in<br>
mice lacking keratin 8. Genes and Development, 7: 1191-1202,1993.<br>
17.	Baribault, H., Penner, J., Iozzo, R. V., and Wilson-Heiner, M. Colorectal<br>
hyperplasia and inflammation in keratin 8-deficient FVB/N mice. Genes<br>
and Development, 8: 2964-2973,1994.<br>
18.	Hesse, M, Franz, T., Tamai, Y., Taketo, M. M., and Magin, T. M.<br>
Targeted deletion of keratins 18 and 19 leads to trophoblast fragility and<br>
early embryonic lethality. EMBO J., 19: 5060-5070, 2000.<br>
19.	Bauman, P. A., Dalton, W. S., Anderson, J. M., and Cress, A. E.<br>
Expression of cytokeratin confers multiple drug resistance. PNAS, 91:<br>
5311-5314,1994.<br><br>
20.	Parekh, H. K. and Simpkins, H. The differential expression of cytokeratin<br>
18 in cisplatin-sensitive and -resistant human ovarian adenocarcinoma<br>
cells and its association with drug sensitivity. Cancer Research, 55: 5203-<br>
5206,1995.<br>
21.	Anderson, J. M., Heindl, L. M., Bauman, P. A., Ludi, C. W., Dalton, W.<br>
S., and Cress, A. E. Cytokeratin expression results in a drug-resistant<br>
phenotype to six different chemotherapeutic agents. Clinical Cancer<br>
Research, 2: 97-105, 1996.<br>
22.	Schaafsma, H. E., Ramaekers, F. C. S., van Muijen, G. N. P., Lane, E. B.,<br>
Leigh, I. M, Robben, H., Huijsmans, A., Ooms, E. C. M, and Ruiter, D.<br>
J. Distribution of cytokeratin polypeptides in human transitional cell<br>
carcinomas, with special emphasis on changing expression patterns<br>
during tumor progression. Am. J. Pathology, 136: 329-343,1990.<br>
23.	Moll, R. Cytokeratins in the histological diagnosis of malignant tumors.<br>
Int. J. Biol. Markers, 9: 63-69, 1994.<br>
24.	Moll, R., Franke, W. W., Schiller, D. L., Geiger, B., and Krepler, R. The<br>
catalog of human cytokeratins: Patterns of expression in normal epithelia,<br>
tumors and cultured cells. Cell, 31: 11-24,1982.<br>
25.	Pankov, R., Umezawa, A., Maki, R., Der, C. J., Hauser, C. A., and<br>
Oshima, R. G. Keratin 18 activation by Ha-ras is mediated through ETS<br>
and Jun binding sites. PNAS, 91: 873-877,1994.<br><br>
26.	Oshima, R. G., Abrams, L., and Kulesh, D. Activation of an intron<br>
enhancer within the keratin 18 gene by expression of c-fos and c-jun in<br>
undifferentiated F9 embryonal carcinoma cells. Genes. Dev., 4: 835-848,<br>
1990.<br>
27.	Oshima, R. G., Baribault, H., and Caulin, C. Oncogenic regulation and<br>
function of keratins 8 and 18. Cancer and Metastasis Reviews, 15: 445-<br>
471,1996.<br><br>
28.	Chu, Y. W., Runyan, R. B., Oshima, R. G., and Hendrix, M. J. C.<br>
Expression of complete keratin filaments in mouse L cells augments cell<br>
migration and invasion. PNAS, 90: 4261-4265,1993.<br>
29.	Pankov, R., Simcha, I., Zoller, M., Oshima, R. G., and Ben-Ze'ev, A.<br>
Contrasting effects of k8 and k18 on stabilizing k19 expression, cell<br>
motility and tumorigenicity in the BSp73 adenocarcinoma. J. Cell<br>
Science, 110: 965-974,1997.<br>
30.	Hendrix, M. J. C, Seftor, E. A., Chu, Y. W., Seftor, R. E. B., Nagle, R.<br>
B., McDaniel, K. M., Leong, S. P. L., and Yohem, K. H. Co-expression<br>
of vimentin and keratins by human melanoma tumor cells: Correlation<br>
with invasive and metastatic potential. J. Natl. Cancer Inst., 84:165-174,<br>
1992.<br>
31.	Ditzel, H. J., Garrigues, U., Andersen, C. B., Larsen, M. K., Garrigues,<br>
H. J., Svejgaard, A., Hellstrom, L, Hellstrom, K. E., and Jensenius, J. C.<br>
Modified cytokeratins expressed on the surface of carcinoma cells<br>
undergo endocytosis upon binding of human monoclonal antibody and its<br>
recombinant Fab fragment. PNAS USA, 94: 8110-8115,1997a.<br>
32.	Ditzel, H. J., Rasmussen, J. W., Erb, K., Borup-Christensen, P., Titlestad,<br>
I., Lassen, E., Fenger, C, Kronborg, O., and Jensenius, J. C. Tumor<br>
detection with 1311-labeled human monoclonal antibody COU-1 in<br>
patients with suspected colorectal carcinoma. Cancer Research, 53: 5920-<br>
5928,1993.<br>
33.	Chan, R., Rossitto, P. V., Edwards, B. F., and Cardiff, R. I). Presence of<br>
proteolytically processed keratins in the culture medium of MCF-7 cells.<br>
Cancer Research, 46:6353-6359, 1986.<br>
34.	Hembrough, T. A., Vasudevan, J., Allietta, M. M., Glass, W. F.H, and<br>
Gonias, S. L. A cytokeratin 8-like protein with plasminogen-binding<br>
activity is present on the external surfaces of hepatocytes, HepG2 cells<br>
and breast carcinoma cell lines. J. Cell Science, 108: 1071-1082,1995.<br><br>
35.	Borup-Christensen, P., Erb, K., Ditzel, H., Nielsen, B., Larsen, J. K.,<br>
Svehag, S. E., and Jensenius, J. C. Human hybridoma producing<br>
monoclonal antibodies against colorectal cancer-associated antigens.<br>
APMIS, 98: 674-684,1990.<br>
36.	Towbin, H., Staehelin, T., and Gordon, J. Electrophoretic transfer of<br>
proteins from polyacrylamide gels to nitrocellulose sheets: procedure and<br>
some applications. PNAS, 76: 4350-4354,1979.<br>
37.	Crawford, C, Brown, N. R., and Willis, A. C. Investigation of the<br>
structural basis of the interaction of calpain II with phospholipid and with<br>
carbohydrate. Biochem. J., 265: 575-579,1990.<br>
38.	Waseem, A., Lane, E. B., Harrison, D., and Waseem, N. A keratin<br>
antibody recognizing a heterotypic complex: Epitope mapping to<br>
complementary locations on both components of the complex.<br>
Experimental Cell Research, 223: 203-214,1996.<br>
39.	Karlsson, R. A., Michaelsson, A., and Mattsson, L. Kinetic analysis of<br>
monoclonal antibody-antigen interactions with a new biosensor based<br>
analytical system. J. Immunol. Methods, 145: 229-240,1991.<br>
40.	Marilley, D. and Schwaller, B. Association between the calcium-binding<br>
protein calretinin and cytoskeletal components in the human colon<br>
adenocarcinoma cell line WiDr. Experimental Cell Research, 259: 12-22,<br>
2000.<br>
41.	Chen, P. H., Ornelles, D. A., and Shenk, T. The adenovirus L3 23-<br>
kilodalton proteinase cleaves the amino-terminal head domain from<br>
cytokeratin 18 and disrupts the cytokeratin network of HeLa cells. J.<br>
Virology, 67: 3507-3514, 1993.<br>
42.	Zhang, Y. and Schneider, R. J. Adenovirus inhibition of cell translation<br>
facilitates release of virus particles and enhances degradation of the<br>
cytokeratin network. J. Virology, 68: 2544-2555,1994.<br><br>
43.	Broers, J. L. V., Ramaekers, F. C. S., Rot, M. K., Oostendorp, T.,<br>
Huysmans, A., van Muijen, G. N. P., Wagenaar, S. S., and Vooijs, G. P.<br>
Cytokeratins in different types of human lung cancer as monitored by<br>
chain-specific monoclonal antibodies. Cancer Research, 48: 3221-3229,<br>
1988.<br>
44.	Shitara, K., Fujiwara, K., Kusano, A., Yamaguchi, K., Yoshida, H., Sato,<br>
S., and Hanai, N. Application of anti lung adenocarcinoma monoclonal<br>
antibody recognizing cytokeratin-like cytoplasmic antigen for tumor<br>
diagnosis. Anticancer Res, 12: 1121-1129, 1992.<br>
45.	Lane, E. B., Goodman, S. L., and Trejdosiewicz, L. K. Disruption of the<br>
keratin filament network during epithelial cell division . EMBO Journal,<br>
1: 1365-1372,1982.<br>
46.	Miller, R. K., Khuon, S., and Goldman, R. D. Dynamics of keratin<br>
assembly: Exogenous type I keratin rapidly associates with type II keratin<br>
in vivo. J. Cell Biology, 122: 123-135,1993.<br>
47.	Nishibori, H., Matsuno, Y., Iwaya, M., Osada, T., Kubomura, N.,<br>
Iwamatsu, A., Kohno, H., Sato, S., Kitajima, M., and Hirohashi, S.<br>
Human colorectal carcinomas specifically accumulate Mr 42,000<br>
ubiquitin-conjugated cytokeratin 8 fragments. Cancer Res., 56: 2752-<br>
2757,1996.<br><br>
48.	Erb, K., Borup-Christensen, P., Ditzel, H., Chemnitz, J., Haas, H., and<br>
Jensenius, J. C. Characterization of a human-human hybridoma antibody,<br>
C-OU1, directed against a colon tumor-associated antigen. Hybridoma,<br>
11: 121-134, 1992.<br>
49.	Ditzel, H. J. Human monoclonal antibodies: a took for cancer detection<br>
in vivo. APMIS Suppl., 94: 5-42,1999.<br>
50.	Hembrough, T. A., Kralovich, K. R., Li, L., and Gonias, S. L.<br>
Cytokeratin 8 released by breast carcinoma cells in vitro binds<br><br>
plasminogen and tissue-type plasminogen activator and promotes<br>
plasminogen activation. Biochem. J., 317: 763-769,1996.<br>
51.	Kralovich, K. R., Li, L., Hembrough, T. A., Webb, D. J., Karns, L. R.,<br>
and Gonias, S. L. Characterization of the binding sites for plasminogen<br>
and tissue-type plasminogen activator in cytokeratin 8 and cytokeratin 18.<br>
J. Protein Chem., 17: 845-854,1998.<br>
52.	Wells, M. J., Hatton, M. W. C, Hewlett, B., Podor, T. J., Sheffield, W.<br>
P., and Blajchman, M. A. Cytokeratin 18 is expressed on the hepatocyte<br>
plasma membrane surface and interacts with thrombin-antithrombin<br>
complexes. J. Biological Chemistry, 272: 28574-28581,1997.<br>
53.	Rydlander, L., Ziegler, E., Bergman, T., Schoberl, E., Steiner, G.,<br>
Bergman, A. C, Zetterberg, A., Marberger, M., Bjorklund, P., Skern, T.,<br>
Einarsson, R., and Jornvall, H. Molecular characterization of a tissue-<br>
polypeptide-specific-antigen epitope and its relationship to human<br>
cytokeratin 18. Eur. J. Biochem., 241: 309-314,1996.<br>
54.	Silen, A., Wiklund, B., Norlen, B. J., and Nilsson, S. Evaluation of a new<br>
tumor marker for cytokeratin 8 and 18 fragments in healthy individuals<br>
and prostate cancer patients. Prostate, 24: 326-332,1994.<br>
55.	Godfroid, E., Geuskens, M., Dupressoir, T., Parent, L, and Szpirer, C.<br>
Cytokeratins are exposed on the outer surface of established human<br>
mammary carcinoma cells. J. Cell Science, 99: 595-607,1991.<br>
56.	Caulin, C, Salvesen, G. S., and Oshima, G. Caspase cleavage of keratin<br>
18 and reorganization of intermediate filaments during epithelial cell<br>
apoptosis. J. Cell Biol., 138:1379-1394,1997.<br>
57.	Leers, M. P., Kolgen, W., Bjorklund, V., Bergman, T., Tribbick, G.,<br>
Persson, B., Bjorklund, P., Ramaekers, F. C, Bjorklund, B., Nap, M.,<br>
Jomvall, H., and Schutte, B. Immunocytochemical detection and<br><br>
mapping of a cytokeratin 18 neo-epitope exposed during early apoptosis.<br>
J. Pathol., 187: 567-572,1999.<br>
58.	Ku, N. O. and Omary, M. B. Identification of the major physiologic<br>
phosphorylation site of human keratin 18: potential kinases and a role in<br>
filament reorganization. J. CellBiol., 727: 161-171, 1994.<br>
59.	Liao, J., Lowthert, L. A., Ku, N. O., Fernandez, R., and Omary, M. B.<br>
Dynamics of human keratin 18 phosphorylation: Polarized distribution of<br>
phosphorylated keratins in simple epithelial tissues. J. Cell Biology, 131:<br>
1291-1301,1995.<br>
60. Ku, N. O., Liao, J., Chou, C. F., and Omary, M. B. Implications of<br>
intermediate filament protein phosphorylation. Cancer and Metastasis<br>
Reviews, 15: 429-444,1996.<br><br>
We claim:<br>
1.	An isolated cancer-associated epitope comprising recombinant cytokeratin 8 and<br>
cytokeratin 18 polypeptide fragments, wherein the cytokeratin 8 polypeptide is shorter than 482<br>
amino acids and comprises SEQ ID NO:3 or SEQ ID NO:5, and the cytokeratin 18 polypeptide<br>
is shorter than 429 amino acids and comprises SEQ ID NO:4 or SEQ ID NO:6.<br>
2.	The isolated epitope as claimed in claim 1, wherein the cytokeratin 8 polypeptide is<br>
shorter than about 475 amino acids and the cytokeratin 18 polypeptide is shorter than about 425<br>
amino acids.<br>
3.	The isolated epitope as claimed in claim 1, wherein the cancer-associated epitope is<br>
detected in filamentous cytoplasmic structures of adenocarcinoma cells but is substantially<br>
undetected in normal cells.<br>
4.	The isolated epitope as claimed in claim 1, wherein the cancer-associated epitope is<br>
detected in filamentous cytoplasmic structures of colon adenocarcinoma, ovarian<br>
adenocarcinoma, renal adenocarcinoma, mammary adenocarcinoma, lung adenocarcinoma,<br>
pancreatic adenocarcinoma and non-seminomal testis carcinoma cells.<br>
5.	A vaccine composition for preventing or treating adenocarcinoma comprising<br>
recombinant cytokeratin 8 and cytokeratin 18 polypeptide fragments, wherein the cytokeratin 8<br>
polypeptide is shorter than about 482 amino acids and comprises SEQ ID NO:3 or SEQ ID<br>
NO:5, and the cytokeratin 18 polypeptide is shorter than about 429 amino acids and comprises<br>
SEQ ID NO:4 or SEQ ID NO:6.<br>
6.	The vaccine composition as claimed in claim 5, wherein the cytokeratin 8 polypeptide is<br>
shorter than about 475 amino acids and the cytokeratin 18 polypeptide is shorter than about 425<br>
amino acids.<br>
7.	The vaccine composition as claimed in claim 5, wherein the adenocarcinoma is colon<br>
adenocarcinoma, ovarian adenocarcinoma, renal adenocarcinoma, mammary adenocarcinoma,<br>
lung adenocarcinoma, pancreatic adenocarcinoma or non-seminomal testis carcinoma.<br><br>
8.	An isolated antibody that can bind to a cancer-associated epitope consisting of a complex<br>
between two separate polypeptides, a cytokeratin 8 polypeptide missing the first 65 amino acids<br>
and a cytokeratin 18 polypeptide missing the first 49 amino acids, wherein the antibody<br>
specifically binds cytokeratin 8 SEQ ID NO:3 or SEQ ID NO:5, and cytokeratin 18 SEQ ID<br>
NO:4 or SEQ ID NO:6, wherein the antibody is not a COU-I IgM antibody secreted by a B9165<br>
cell line and the antibody specifically binds to adenocarcinoma tissue but has substantially<br>
weaker binding to normal epithelial cells and smooth muscle fibers.<br>
9.	The isolated antibody as claimed in claim 8, wherein the is shorter than about 425 amino<br>
acids.<br>
10.	The isolated antibody as claimed in claim 8, wherein the antibody is shorter than about<br>
200 amino acids.<br>
11.	The isolated as claimed in claim 8, wherein the antibody is an IgG, IgA, IgD, IgE or IgM<br>
antibody.<br>
12.	The isolated antibody as claimed in claim 8, wherein the antibody can detect the cancer-<br>
associated epitope in filamentous cytoplasmic structures of adenocarcinoma cells but in<br>
substantially no filamentous structures of normal cells.<br>
13.	The binding entity as claimed in claim 8, wherein the antibody can detect the cancer-<br>
associated in filamentous cytoplasmic structures of colon adenocarcinoma, ovarian<br>
adenocarcinoma, renal adenocarcinoma, mammary adenocarcinoma, lung adenocarcinoma,<br>
pancreatic adenocarcinoma and non-seminomal testis carcinoma cells.<br>
14.	The antibody as claimed in claim 8, wherein the antibody comprises a polypeptide having<br>
an amino acid sequence with at least 98% homology to any one of SEQ ID NO:7-35.<br>
15.	The antibody as claimed in claim 8, wherein the antibody comprises a polypeptide having<br>
any one of SEQ ID NO:7-35 or SEQ ID NO:47-49.<br>
16.	The antibody as claimed in claim 8, wherein the antibody is encoded by a nucleic acid<br>
comprising any one of SEQ ID NO:36-39.<br><br>
17.	A kit for detecting cancer comprising a container containing an antibody that can bind to<br>
a cancer-associated epitope, wherein the cancer-associated epitope consists of a complex<br>
between two separate polypeptides, a cytokeratin 8 polypeptide missing the first 65 amino acids<br>
and a cytokeratin 18 polypeptide missing the first 49 amino acids, wherein the antibody<br>
specifically binds cytokeratin 8 polypeptide SEQ ID NO:3 or SEQ ID NO:5, and cytokeratin 18<br>
polypeptide SEQ ID NO.4 or SEQ ID NO.6, wherein the antibody is not a COU-1 IgM antibody<br>
secreted by a B9165 cell line and the antibody specifically binds to adenocarcinoma tissue but<br>
has substantially weaker binding to normal epithelial cells and smooth muscle fibers<br>
18.	The kit as claimed in claim 17, wherein the antibody is shorter than about 425 amino<br>
acids.<br>
19.	The kit as claimed in claim 17, wherein the antibody is shorter than about 200 amino<br>
acids.<br>
20.	The kit as claimed in claim 18, wherein the antibody is an IgG antibody.<br>
21.	The kit as claimed in claim 17, wherein the cancer is an adenocarcinoma.<br>
22.	The kit as claimed in claim 17, wherein the cancer is colon adenocarcinoma, ovarian<br>
adenocarcinoma, renal adenocarcinoma, mammary adenocarcinoma, lung adenocarcinoma or<br>
non-seminomal testis carcinoma.<br>
23.	The kit as claimed in claim 17, wherein the antibody further comprises a label or<br>
diagnostic imaging agent.<br>
24.	The kit as claimed in claim 17, wherein the antibody further comprises barium sulfate,<br>
iocetamic acid, iopanoic acid, ipodate calcium, diatrizoate sodium, diatrizoate meglumine,<br>
metrizamide, tyropanoate sodium, fluorine-18, carbon-11, iodine-123, technitium-99m, iodine-<br>
131, indium-111, fluorine, gadolinium, fluorescein, isothiocyalate, rhodamine, phycoerythrin,<br>
phycocyanin, allophycocyanin, ophthaldehyde, fluorescamine, luminal, isoluminal, luciferin,<br>
luciferase or aequorin.<br>
25.	The kit as claimed in claim 17, wherein the antibody comprises a polypeptide having an<br>
amino acid sequence with at least 98% homology to any one of SEQ ID NO:7-35.<br><br>
26.	The kit as claimed in claim 18, wherein the antibody comprises a polypeptide having any<br>
one of SEQ ID NO:7-35 or SEQ ID NO:47-49.<br>
27.	The kit as claimed in claim 17, wherein the antibody is encoded by a nucleic acid<br>
comprising any one of SEQ ID NO:36-39.<br>
28.	A therapeutic composition comprising an antibody and a pharmaceutically acceptable<br>
carrier, wherein the antibody can bind to a cancer-associated epitope consisting of a complex<br>
between two separate polypeptides, a cytokeratin 8 polypeptide missing the first 65 amino acids<br>
and a cytokeratin 18 polypeptide missing the first 49 amino acids, wherein the cytokeratin 8<br>
polypeptide comprises SEQ ID NO:3 or SEQ ID NO:5, and the cytokeratin 18 polypeptide<br>
comprises SEQ ID NO:4 or SEQ ID NO:6, wherein the antibody is not a COU-I IgM antibody<br>
secreted by a B9165 cell line, and the antibody specifically binds to adenocarcinoma tissue but<br>
has substantially weaker binding to normal epithelial cells and smooth muscle fibers<br>
29.	The therapeutic composition as claimed in claims 28, wherein the antibody is shorter than<br>
about 425 amino acids.<br>
30.	The therapeutic composition as claimed in claims 28, wherein the antibody is shorter than<br>
about 200 amino acids.<br>
31.	The therapeutic composition as claimed in claims 28, wherein the antibody is an IgG<br>
antibody.<br>
32.	The therapeutic composition as claimed in claims 28, wherein the antibody can bind to<br>
the cancer-associated epitope in filamentous cytoplasmic structures of adenocarcinoma cells but<br>
in substantially no filamentous structures of normal cells.<br>
33.	The therapeutic composition as claimed in claims 28, wherein the antibody can bind to<br>
the cancer-associated epitope in filamentous cytoplasmic structures of colon adenocarcinoma,<br>
ovarian adenocarcinoma, renal adenocarcinoma, mammary adenocarcinoma, lung<br>
adenocarcinoma, pancreatic adenocarcinoma and non-seminomal testis carcinoma cells.<br><br>
34.	The therapeutic composition as claimed in claim 28, wherein the antibody comprises a<br>
polypeptide having an amino acid sequence with at least 98% homology to any one of SEQ ID<br>
NO:7-35.<br>
35.	The therapeutic composition as claimed in claim 28, wherein the antibody comprises a<br>
polypeptide having any one of SEQ ID NO.7-35 or SEQ ID NO.47-49.<br>
36.	The therapeutic composition as claimed in claim 28, wherein the antibody is encoded by<br>
a nucleic acid comprising any one of SEQ ID NO.36-39.<br>
37.	A therapeutic composition for treating adenocarcinoma comprising an inhibitor of a<br>
protease that cleaves Xaa1SR↓Xaa4 (SEQ ID NO:40) and a pharmaceutically acceptable carrier,<br>
wherein Xaa1 is serine, phenylalanine or valine and Xaa4 is serine or valine.<br><br>
38.	The therapeutic composition as claimed in claim 37, wherein the protease is a trypsin-like<br>
protease.<br>
39.	The therapeutic composition as claimed in claim 37, wherein the inhibitor is soybean<br>
trypsin inhibitor, alpha-2-macroglobulin, alpha-1-antitrypsin, aprotinin, pancreatic secretory<br>
trypsin inhibitor, corn trypsin inhibitor, pumpkin trypsin inhibitor or human amyloid β-protein<br>
precursor inhibitor.<br>
40.	The therapeutic composition as claimed in claim 37, wherein the adenocarcinoma is<br>
colon adenocarcinoma, ovarian adenocarcinoma, renal adenocarcinoma, mammary<br>
adenocarcinoma, lung adenocarcinoma, pancreatic adenocarcinoma or non-seminomal testis<br>
carcinoma.<br>
41.	An in vitro method of identifying a mutant binding entity comprising:<br>
fusing a nucleic acid encoding a polypeptide having any one of SEQ ID NO:7-35 to a<br>
nucleic acid encoding a display protein to generate a recombinant nucleic acid encoding a fusion<br>
protein;<br>
mutating the recombinant nucleic acid encoding the fusion protein to generate a mutant<br>
nucleic acid encoding a mutant fusion protein;<br>
expressing the mutant fusion protein; and<br><br>
selecting a mutant fusion protein that can bind to a cancer-associated epitope comprising<br>
two separate polypeptides, the first polypeptide comprising SEQ ID NO:3 of cytokeratin 8 and<br>
the second polypeptide comprising SEQ ID NO:4 of cytokeratin 18.<br>
42.	The method as claimed in claim 41, wherein the binding entity is a CDR or Fab fragment.<br>
43.	The method as claimed in claim 41, wherein the display protein is a phage display<br>
protein, retroviral display protein, or a ribosomal display protein.<br>
44.	A composition comprising an antibody and a pharmaceutically acceptable carrier,<br>
wherein the antibody can bind with specificity to a cancer-associated epitope consisting of a<br>
fusion protein consisting of two separate polypeptides, a cytokeratin 8 polypeptide missing the<br>
first 65 amino acids and a cytokeratin 18 polypeptide missing the first 49 amino acids, wherein<br>
the antibody specifically binds cytokeratin 8 SEQ ID NO:3 and cytokeratin 18 SEQ ID NO:4 and<br>
wherein the antibody is not a COU-I IgM antibody secreted by a B9165 cell line.<br>
45.	The composition of claim 30, wherein the antibody is a Fab, Fab', F(ab')2, Fv, single<br>
chain antibody (scFv) or diabody.<br>
46.	The antibody of claim 8, wherein the antibody is a Fab, Fab', F(ab')2, Fv, single<br>
chain antibody (scFv) or diabody.<br><br>
This invention discloses an isolated cancer-associated epitope comprising recombinant<br>
cytokeratin 8 and cytokeratin 18 polypeptide fragments, wherein the cytokeratin 8<br>
polypeptide is shorter than 482 amino acids and comprises SEQ ID NO:3 or SEQ ID<br>
NO:5, and the cytokeratin 18 polypeptide is shorter than 429 amino acids and comprises<br>
SEQ ID NO:4 or SEQ ID NO:6.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI0LWtvbG5wLTIwMDQtZ3JhbnRlZC1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">924-kolnp-2004-granted-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI0LWtvbG5wLTIwMDQtZ3JhbnRlZC1hc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">924-kolnp-2004-granted-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI0LWtvbG5wLTIwMDQtZ3JhbnRlZC1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">924-kolnp-2004-granted-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI0LWtvbG5wLTIwMDQtZ3JhbnRlZC1jb3JyZXNwb25kZW5jZS5wZGY=" target="_blank" style="word-wrap:break-word;">924-kolnp-2004-granted-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI0LWtvbG5wLTIwMDQtZ3JhbnRlZC1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">924-kolnp-2004-granted-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI0LWtvbG5wLTIwMDQtZ3JhbnRlZC1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">924-kolnp-2004-granted-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI0LWtvbG5wLTIwMDQtZ3JhbnRlZC1leGFtaW5hdGlvbiByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">924-kolnp-2004-granted-examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI0LWtvbG5wLTIwMDQtZ3JhbnRlZC1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">924-kolnp-2004-granted-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI0LWtvbG5wLTIwMDQtZ3JhbnRlZC1mb3JtIDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">924-kolnp-2004-granted-form 13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI0LWtvbG5wLTIwMDQtZ3JhbnRlZC1mb3JtIDE4LnBkZg==" target="_blank" style="word-wrap:break-word;">924-kolnp-2004-granted-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI0LWtvbG5wLTIwMDQtZ3JhbnRlZC1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">924-kolnp-2004-granted-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI0LWtvbG5wLTIwMDQtZ3JhbnRlZC1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">924-kolnp-2004-granted-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI0LWtvbG5wLTIwMDQtZ3JhbnRlZC1ncGEucGRm" target="_blank" style="word-wrap:break-word;">924-kolnp-2004-granted-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI0LWtvbG5wLTIwMDQtZ3JhbnRlZC1yZXBseSB0byBleGFtaW5hdGlvbiByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">924-kolnp-2004-granted-reply to examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI0LWtvbG5wLTIwMDQtZ3JhbnRlZC1zZXF1ZW5jZSBsaXN0aW5nLnBkZg==" target="_blank" style="word-wrap:break-word;">924-kolnp-2004-granted-sequence listing.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTI0LWtvbG5wLTIwMDQtZ3JhbnRlZC1zcGVjaWZpY2F0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">924-kolnp-2004-granted-specification.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="226797-a-genetic-vector-for-stable-transfection-and-expression-of-a-desired-protein.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="226799-an-antenna-system-for-a-wireless-communications-device-and-a-method-for-providing-enhanced-reception-in-a-diversity-antenna-system.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>226798</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>924/KOLNP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>52/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>26-Dec-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>24-Dec-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>02-Jul-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>THE SCRIPPS RESEARCH INSTITUTE</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>10555 NORTH TORREY PINES ROAD, LA JOLLA, CA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>DITZEL HENRIK</td>
											<td>4411 CAMINITO SANA, #4, SAN DIEGO, CA 92122</td>
										</tr>
										<tr>
											<td>2</td>
											<td>JENSENIUS JENS C</td>
											<td>FINSENS ALLE 28, 5320 ODENSE M</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 5/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US03/00297</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-01-03</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/345,208</td>
									<td>2002-01-03</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/226798-a-recombinant-cancer-associated-epitope-vaccines-comprising-it-antibodies-binding-to-said-epitope-kits-and-compositions-comprising-said-antibodies by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:14:19 GMT -->
</html>
