<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/226998-a-baking-composition-comprising-a-polypeptide-exhibiting-phospholipase-a-activity by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:54:14 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 226998:A BAKING COMPOSITION COMPRISING A POLYPEPTIDE EXHIBITING PHOSPHOLIPASE A ACTIVITY</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A BAKING COMPOSITION COMPRISING A POLYPEPTIDE EXHIBITING PHOSPHOLIPASE A ACTIVITY</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>ABSTRACT 2824/MAS/97 &quot;A BAKING COMPOSITION COMPRISING A POLYPEPTIDE EXHIBITING PHOSPHOLIPASE A ACTIVITY&quot; The present invention relates to a baking composition comprising a polypeptide exhibiting phospholipase A activity selected from the group consisting of: a) a polypeptide encoded by the phospholipase A-encoding part of the DNA sequence cloned into plasmid pYES 2.0 present in Escherichia coli DSM 11299; b) a polypeptide having an amino acid sequence as shown in positions 31-346 ofSEQIDN0 2; c) a polypeptide having an amino acid sequence as shown in position 31-303 of SEQ ID NO 2; and d) a polypeptide which is at least 70% homologous with said polypeptide defined in (b) or (c), and an emulsifier or flour.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
FIELD OP INVENTION<br>
The present invention relates to a method for reducing the content of phosphorous containing components in an edible oil comprising a high amount of non-hydratable phosphorus, by the use of a phospholipase.<br>
Further, the present invention relates to an enzyme with phospholipase activity, a cloned DNA sequence encoding the enzyme with phospholipase activity, a method of producing the enzyme, and the use of said enzyme for a number of industrial applications.<br>
BACKGROUND OF THE INVENTION<br>
Enzymatic dequmminq of edible oils comprising a relatively high amount of non-hydratable phosphorus content:<br>
It is known to use phospholipase for enzymatic degumming of a water deguimned edibl6 oil (US 5,264,367,<br>
Metallgesellschaft, R&amp;hm), to reduce the phosphorous content of said water degummed edible oil.<br>
However, this process may still be improved, especially to perform enzymatic degumming of edible oils comprising a high amount of non-hydratable phosphorus (NHP) and/or relatively high amounts of mucilage.<br>
Consequently, an object of the invention is to provide a method for reducing the content of phosphorus containing components of such oils, wherein said method comprises use of a phospholipase.<br>
A phospholipase of the invention<br>
Phospholipids, such as lecithin or phosphatidylcholine, consist of glycerol esterified with two fatty acids in an outer (sn-1) and the middle (sn-2) positions and esterified with phosphoric acid in the third position; the phosphoric acid, in turn, may be<br><br>
2451-58, 1988; FEMS, Microbiol. Letters, 124, 29-34), Schizosaccharomyces pombe (H. Oishi et al., Biosci. Biotech. Biochem., 60 (7), 1087-92, 1996), Aspergillus niger   (Technical Bulletin, G-zyme™ G999, Enzyme Bio-Systems Ltd.; Process Biochemistry 30(5): 393-401 (1995)) and Corticium centrifugum (S. Uehara et al., Agric. Biol. Chem., 43 (3), 517-525, 1979).<br>
Section two:<br>
EP 575133 A2 describes isolation and characterization of a fungal phoshoslipase Al obtained from Aspergillus  and the use thereof for industrial applications.<br>
No sequence information (neither DNA nor amino acid) is disclosed in the application nor is any strategy or suggestion for cloning any of the Aspergillus  phospholipase discussed or indicated in the application.<br>
Tsung-Che et al. (Phytopathological notes 58:1437T38 (1968)) briefly describes characterization of a phospholipase from Fusarium  solani.<br>
EP 130,064 describes an isolated fraction of a fermentation broth exhibiting lipase activity obtained from the strain Fusarium oxysporum, DSM 2672. Furthermore, the use thereof in detergent compositions is disclosed. However, EP 130,064 does not describe this fraction as exhibiting any phospholipase activity.<br>
WO 96/13579 describes a lipase obtained from the strain Fusarium culmorum,   CBS 513.94 including its N-terminal sequence.<br>
However, WO 96/13579 does not describe any enzyme exhibit¬ing phospholipase activity.<br>
A cDNA sequence encoding a lipase from Fusarium heterosporum  is described (Cloning and nucleotide sequence of cDKA encoding a lipase from Fusarium heterosporum,   J. Biochem. 116, 536-540, 1994.). This sequence is presently believed to be the most related DNA sequence compared to a cloned DNA sequence of the invention (See section "Comparison with prior art" {vide infra). However, this reference does not describe any enzyme exhibiting phospholipase activity.<br>
A cDNA sequence encoding a phospholipase B from Penicillum notatum  is described (Eur. J. Biochem 202:783-787, 1991). However<br><br>
this cloned DNA sequence has very limited homology to a DNA sequence of the invention (See section "Comparison with prior art" (vide infra).<br>
Industrial application of phospholipases:<br>
A number of uses of ptiospholipases are known in the art, such as to use phospholipase in, e.g.   enzymatic degumming of a watsr degummed oil (US 5,264,367, Metallgesellschaft, RBhm); treatment of starch hydrolysate (particularly from wheat starch) to improve the filterability (EP 219,269, CPC International); as an additive to bread dough to improve the elasticity of the bread (US 4,567,046, Kyowa Hakko); and for preparation ^f lyso-lecithin with special emulsifying properties.<br>
Presently, the phospholipase Lecitase® (Novo Nordisk A/S)<br>
is used commercially such as for degumming of oils. Lecitase® is a mammalian enzyme obtained from porcine pancreas.<br>
It is well known that it is possible to produce fungal enzymes recombinantly at industrially economically acceptable yields, especially from filamentous fungi.<br>
Consequently, it is an object of this invention to provide an improved phospholipase e.g. for use in the processes de¬scribed above.<br>
Further, it is an object of the present invention to describe processes and methods for recombinant production at industrially acceptable yields of a phospholipase obtained from a filamentous fungus.<br>
SUMMARY OF THE INVENTION<br>
Water degumming of edible oils is carried out by extraction with water. In that treatment, a part of the phosphatides is left in the oil. That part is described by the generic term "non-hydratable phosphatides" (NHP). In the production of oils, it is essential to remove the NHP content (US 5264367).<br>
The present invention provides a method to remove the NHP content in an oil comprising a relatively high amount of NHP.<br>
Accordingly, in a first aspect the invention relates to a method for reducing the content of phosphorus containing<br><br>
components in an edible oil having a non-hydratable phosphorus content of at least 50 ppm measured by,<br>
i)    pre-treating the edible oil, at 60°C, by addition of a solution comprising citric acid monohydrate in water (added water vs. oil equals 4.8% w/w; [citric acid] in water phase = 106 mM, in water/oil emulsion *= 4.6 niM) for 3 0 minutes; ii)   transferring 10 ml of the pre-treated water in oil<br>
emulsion to a tube; iii)  heating the emulsion in a boiling water bath for 30<br>
minutes; iv}   centrifuging at 5000 rpm for 10 minutes-, v)    transferring about 8 ml of the upper (oil) phase to a<br>
new tube and leaving it to settle for 24 hours; and vi)   thereafter drawing 2 g from the upper clear phase for measurement of the non-hydratable phosphorus content (ppm) in the edible oil; and wherein said method comprises,<br>
contacting said oil at a pH from 1.5-8 with an aqueous solution of a phospholipase Ai, a phospholipase A2, or a phospholipase B, which solution is emulsified in the oil until the phosphorus content of the oil is reduced to less than 11 ppm, and then separating the aqueous phase from the treated oil.<br>
In another aspect the invention relates to a new cloned phospholipase.<br>
Further investigations into the nature of the lipase activity found in Fusarium oxysporum,   DSM 2672 (and described in EP 130,064) revealed that the isolated fraction comprises several components having lipase activity of which one exhibited phospholipase activity.<br>
Despite a number of technical difficulties (vide Infra), the present inventors have been able to clone an enzyme exhibiting phospholipase A activity from a strain of the genus Fusarium,   more specifically Fusarium oxysporum.<br>
This is the first time a filamentous fungal phospholipase A has been cloned and consequently, the present invention provides a cloned DNA sequence encoding a filamentous fungal<br><br>
phospholipase A enzyme.<br>
Accordingly, an aspect of the invention relates to a cloned DNA sequence encoding a polypeptide having phospholipase A activity wherein the DNA sequence is obtained from a filamentous fungus.<br>
A cDNA sequence encoding a phospholipase B from Penicillum notatum  is described in Eur. J. Biochem 202:783-787, 1991.<br>
However, this DNA sequence shows only a very limited DNA identity of 39% to the DNA sequence of the present invention (SEQ ID NO 1 23-1060), and further, a physiological characteristic such as molecular mass varies considerably between said PLB from P.   notatum   (66 kDa) and a phospholipase of the invention (29 ± 10 kDa (vide  infra)).<br>
Further, a comparison with prior art nucleotide and amino acid sequences has shown that the DNA sequence and/or the corresponding encoded amino acid sequence of the invention has only little homology to any prior art DNA and/or amino acid sequences (vide infra).<br>
Consequently, it is presently believed that the DNA sequence information provided in the present application will be highly valuable in order to e.g. clone another<br>
related/homologous phospholipase encoding DNA sequence, since a specific hybridization probe and/or PCR primers can now easily be constructed on the basis of said DNA sequence of the invention.<br>
Further, it is presently believed that it is possible to clone both a related/homologous phospholipase A and/or phospholipase B encoding DNA sequence based on the sequence information provided in the present application.<br>
Accordingly, in a further aspect the invention relates to a cloned DNA sequence encoding an enzyme exhibiting phospholipase A and/or phospholipase B activity, which DNA sequence is selected from the group comprising:<br>
(a)	the phospholipase A encoding part of the DNA sequence cloned<br>
into plasmid pYES 2.0 present in Escherichia  coli  DSM 11299;<br>
(b)	the DNA sequence shown in positions 23-1063 in SEQ ID NO 1,<br><br>
more preferably positions H3-1063 in SEQ ID NO 1, or even more preferably positions 113-929 in SEQ ID NO i, or its complementary strand;<br>
(c)	a DNA sequence which is at least 70% homologous with said DNA sequences defined in (a) or (b);<br>
(d)	a DNA sequence defined in (a) or (b), which encodes a polypeptide exhibiting phospholipase activity and is at least 70% homologous with the polypeptide sequence shown in positions 31-346 of SEQ ID No 2, or more preferably at least 70% homologous with the polypeptide sequence shown in positions 31-3 03 of SEQ ID No 2;<br><br>
(e)	a DNA sequence which hybridizes with a doubl,e-stranded DNA probe comprising the DNA sequence shown in positions 2 3-1063 in SEQ ID NO 1 at low stringency;<br>
(f)	a DNA sequence which encodes for a polypeptide having the same amino acid sequences at position residues 1 to 346, 31 to 303 or 31 to 303 of SEQ ID No 2, or the amino acid sequences encosed by any of the DNA sequences of (e); and<br>
(g)	a DNA sequence which is a fragment of the DNA sequences specified in (a), (b), (c), (d), (e), or (f).<br>
Further, a phospholipase of the invention has been intensively characterized and it has been found that it has phospholipase activity at low pH; this property makes it very suitable for use in oil degumming. The phospholipase is not membrane bound, which makes it suitable for commercial production and purification.<br>
Accordingly, in a further aspect the invention relates to an isolated polypeptide having phospholipase A activity, which polypeptide is obtained from a strain of the genus Fusarium  and has i)   PLA activity in the pH range 3-10, measured at 40°C;<br>
ii)  a molecular mass of 29 ± 10 kDa, as determined by SDS-PAGE;<br>
iii) an isoelectric point (pi) in the range 4.5-8;<br>
iv)  a temperature optimum for phospholipase activity in the range<br>
between 25-55°C, measured with lecithin as substrate at pH 5;<br>
and/or v)  a pH optimum for phospholipase activity in the pH range<br><br>
between 6-12, measured with lecithin as substrate at 37°C .<br>
A deduced amino acid sequence of an isolated phospholipase of the invention is shown in SEQ ID No 2.<br>
The N-terminal amino acid sequence of a mature secreted isolated phospholipase has been determined. Said N-terminal sequence showed that the mature part of a phospholipase ot  the invention, having the amino acid sequence shown in SEQ IP No 2, starts at amino acid No 31 in SEQ ID No 2. See working example herein for further details (vide infra).<br>
Further, the C-terminal sequence of an active secreted phospholipase of the invention, having the amino acid sequence shown in SEQ ID No 2, has been determined. Said C-terminal determined phospholipase was recombinantly expressed in the filamentous fungal strain Aspergillus oryzae.   See working example herein for further reference.<br>
These results showed that the enzyme was C-terminal processed during expression from A.   oryzae,   and the results indicate .that the Ser303 in SEQ ID NO 2 is the most likely C-terminal residue in the expressed mature active enzyme. However, it is foreseen that even further C-terminal processing may occur (I.e. giving a fragment of said sequences) , and still having an expressed mature active enzyme.<br>
Accordingly, in a further aspect the invention relates to an isolated enzyme exhibiting phospholipase A and/or B activity and being selected from the group comprising:<br>
(a)	a polypeptide encoded by the phospholipase A and/or B enzyme<br>
encoding part of the DNA sequence cloned into plasmid pYES 2.0 present in Escherichia  coli  DSM 11299;<br>
(b)	a polypeptide having an amino acid sequence as shown in positions 31-346 of SEQ ID NO 2;<br>
(c)	a polypeptide having an amino acid sequence as shown in position 31-303 of SEQ ID NO 2;<br>
(d)	an analogue of the polypeptide defined in (a), (b) or (c) which analogue is at least 70% homologous with said polypeptide; and<br>
(e)	a fragment of (a), (b) (c) or (d).<br><br>
In a still fxirther aspect the invention provides a recombinant expression vector, which enables heterologous recombinant production of an enzyme of the invention. Thereby it is possible to make a highly pxirified phospholipase composition, characterized in being free from homologous impurities. This is highly advantageous for a number of industrial applications.<br>
The present invention demonstrates experimentally {vide infra) that a phospholipase obtained from both a strain of Fusarium culmorum  and Fusarium oxyspozrum  has improved properties for use in industrial relevant applications. It is foreseen that phospholipases obtained from a strain of the genus Fusarium  will have improved properties relevant for use in industrial relevant applications.<br>
Accordingly, in an even further aspect the invention relates to the use of a phospholipase obtained from a strain of the genus Fusarium,   such as a strain of F.   culmorum,   F. heter-osporujD, F.   solani,   or in particular a strain of Fusarium oxysporum,   in a process comprising treatment of a phospholipid or lysophospholipid with the phospholipase so as to hydrolyze fatty acyl groups.<br>
Finally, the invention relates to an isolated, substan¬tially pure biological culture of the Escherichia  coli   strain DSM No. 11299 harbouring a phospholipase encoding DNA sequence (the phospholipase encoding part of the DNA sequence cloned into plasmid pYES 2.0 present in Escherichia  coll  DSM 11299) obtained from a strain of the filamentous fungus Fusarium  oxysporum,   or any mutant of said E.coli  strain having retained the phospho¬lipase encoding capability.<br>
Sequence homology comparison with prior art<br>
A homology search with the phospholipase of the invention against nucleotide and protein databases was performed. The homology search showed that the closest related known sequence was a lipase from Fusarium heterosporum   (an amino acid alignment is illustrated in Figure 1).<br>
The DNA sequence of the invention (SEQ ID NO 1 23-1060) encoding the phospholipase shows only 62% DNA homology to the known lipase sequence from Fusarium heterosporum   (Genbank database reference S77816), and the corresponding amino acid<br><br>
sequence of the phospholipase of the invention (SEQ ID NO 2) shows only 60% homology to a deduced amino acid sequence based on the known DNA sequence above (see figure 1).<br>
This shows that the DNA and/or the amino acid sequence of a phospholipase of the invention indeed is different from any known DNA and/or amino acid sequence(s).<br>
A cDNA sequence encoding a phospholipase B from Penicillujn notatum  is described (Eur. J. Biochem 202:783-787, 1991). However, this DNA sequence (Genbank database reference X6034S) shows only very limited DNA identity 39%, to the DNA sequence of the present invention (SEQ ID NO l, 23-1060), and the corresponding amino acid sequence of the phospholipase of the ijivention (SEQ ID NO 2) shows only 20% identity to a deduced amino acid sequence based on the known PLB DNA sequence above.<br>
The calculation of homology was done as described later in this specification.<br>
DRAWINGS:<br>
Figure 1: Alignment of the amino acid sequence shown in SEQ ID No 2 with a prior art lipase sequence from Fusarium heterosporum.<br>
Figure 2:   Comparison of enzymatic degumming capacity of<br>
Lecitase™ and a phospholipase from Fusarium oxysporum  according to the invention.<br>
DEFINITIONS<br>
Prior to discussing this invention in further detail, the following terms will be defined.<br>
"A cloned DNA sequence"; The term "A cloned DNA sequence" refers to a DNA sequence cloned in accordance with standard cloning procedures used in genetic engineering to relocate a segment of DNA from its natural location to a different site where it will be reproduced. The cloning process involves excision and isolation of the desired DNA segment, insertion of the piece of DNA into the vector molecule and incorporation of the recombinant vector into a cell where multiple copies or clones of the DNA segment will be replicated.<br><br>
The "cloned DNA sequence" of the invention may alternatively be termed "DNA construct" or "a cloned polynucleotide having a DNA sequence" "isolated DNA sequence".<br>
"Obtained from": For the purpose of the present invention the term "obtained from", as used herein in connection with a specific microbial source, means that the enzyme and consequently the DNA sequence encoding said enzyme is produced by the specific source or by a cell in which a gene from the source has been inserted.<br>
"An isolated polypeptide"; As defined herein the term, "an isolated polypeptide" or "isolated phospholipase", as used about the phospholipase of the invention, is a phospholipase or phospholipase part which is essentially free from other non-phospholipase polypeptides, e.g., at least 20% pure, preferably at least 40% pxure, more preferably 60% pure, even more preferably 80% pure, most preferably 90% pure, and even most preferably 95% pure, as determined by SDS-PAGE.<br>
When the isolated polypeptide is at least 60% pure the term "A highly isolated polypeptide" may be used. The "isolated polypeptide" may alternatively be termed "purified polypeptide".<br>
"Homologous impurities"; As used herein the term "homologous impurities" means any impurity (e.g. another polypeptide than the enzyme of the invention) which originates from the homologous cell from which the enzyme of the invention is originally obtained. In the present invention the homologous cell may e.g. be a strain of Fusarium oxysporuia.<br>
"Phospholipase encoding part": As used herein the term "phospholipase encoding part" used in connection with a DNA sequence means the region of the DNA sequence which corresponds to the region which is translated into a polypeptide sequence.<br>
In the DNA sequence shown in SEQ ID NO 1 it is the region between the first "ATG" start codon ("AUG" codon in mENA) and the following stop codon ("TAA", "TAG" or "TGA").<br>
The translated polypeptide may further, in addition to the mature sequence exhibiting phospholipase activity, comprise an N-terminal signal and/or a pro-peptide sequence. The signal sequence generally guides the secretion of the polypeptide  and the pro-peptide generally guides the folding of the polypeptide.<br><br>
For further information see Egnell, P. et al. Molecular Microbiol. 6(9):1115-19 (1992) or Stryer, L., "Biochemistry" W.H., Freeman and Company/New York, ISBN 0-7167-1920-7.<br>
"Modification's) of a DNA and/or amino acid sequence" The term "modification(s)" used in connection with modifica-tion(s) of a DNA and/or amino acid sequence as discussed herein is defined to include chemical modification as well as genetic manipulation(s). The modificationCs) may be substitution, deletion and/or insertion in or at the amino acid(s) of inter¬est.<br>
"Phospholipase A": The term "Phospholipase A" used herein in connection with an enzyme of the invention is intended to cover an enzyme with Phospholipase Al and/or Phospholipase A2 activity.<br>
Phospholipase Al is defined according to standard enzyme EC-classification as EC 3.1.1.32.<br>
Official Name: Phospholipase Al (PLAi).<br>
Reaction catalyzed:<br>
phosphatidylcholine + H(2)0 <br>
2-acylglycerophosphocholine +  a fatty acid anion<br>
Comment(s)<br>
has a much broader specificity than ec 3.1.1.4.<br>
Phospholipase A2 is defined according to standard enzyme<br>
EC-classification as EC 3.1.1.4<br>
Official Name: phospholipase A2 CPLA2).<br>
Alternative Name(s)iphosphatidylcholine 2-acylhydrolase.<br>
lecithinase a; phosphatidase; or phosphatidolipase.<br>
Reaction catalysed:<br>
phosphatidylcholine + h(2)o <br>
1-acylglycerophosphocholine + a fatty acid anion<br>
comment(s): also acts on phosphatidylethanolamine,<br>
choline plasmalogen and phosphatides, removing the fatty<br>
acid attached to the 2-position.<br>
"Phospholipase B": Phospholipase B is defined according<br>
to standard enzyme EC-classification as EC 3.1.1.5.<br>
Official Name:lysophospholipase.<br>
Alternative Name(s):lecithinase b; lysolecithinase;<br>
phospholipase b; or plb.<br><br>
Reaction catalysed:<br>
2-lysophosphatidylcholine + h(2)o  glycerophosphocholine + a fatty acid anion "Phospholipase activity"; The term "phospholipase activity" or "having/exhibiting phospholipase activity" as used herein in connection with an enzyme of the invention is intended to specify an enzyme having at least the amount of phospholipase activity (be it PLA or PLB) experimentally defined below.<br>
Accordingly, an enzyme which exhibits phospholipase activity is herein defined as an enzyme which in the "monolayer phospholipase assay" shown in Example 6 herein {vide  infra}   has a phospholipase activity of at least 0.25 nmol/min, enzyme dose:<br>
60 fig, at 25°C; more preferably at least 0.40 nmol/min, enzyme<br>
dose: 60 |i,g, at 25°C; more preferably at least 0.75 nmol/min,<br>
enzyme dose: 60 [ig,   at 25°C; more preferably at least i.o<br>
nmol/min, enzyme dose: 60 \Juq,   at 25'C; more preferably at least<br>
1.25 nmol/min, enzyme dose: 60 |j.g, at 25°C; and even more<br>
preferably at least 1.5 nmol/min, enzyme dose: 60 fig, at 25°C.<br>
It is presently believed that only an enzyme having such a significant phospholipase activity is of industrial importance, for instance for use in degumming (US 5,264,367).<br>
"A lipase with phospholipase side activity": The term "lipase with phospholipase side activity" is accordingly defined as a lipase with a phospholipase side activity wherein the phospholipase side activity in the "monolayer phospholipase assay" shown in Example 6 is less than the above mentioned figures.<br>
A number of lipases have such phospholipase side activity. In working example 6 herein (vide  infra)   some of the lipases having phospholipase side activity are shown.<br>
"A crude oil"; A crude oil (also called a non-degummed oil) may be a pressed or extracted oil or a mixture thereof from e.g. rapeseed, soybean, or sunflower. The phosphatide content in a crude oil may vary from 0.5-3% w/w corresponding to a phosphorus content in the range of 200-1200 ppm, more preferably in the range of 250-1200 ppm. Apart from the phosphatides the crude oil also contains small concentrations of carbohydrates, sugar compounds and metal/phosphatide acid complexes of Ca, Mg and Fe.<br><br>
"A semicrude oil": Any oil which is not a crude oil, but which has a phosphatide content above 250 ppm, more preferably above 500 ppm. Such an oil could e.g. be achieved by subjecting a crude oil to a process similar to the "water degummed oil" process described below.<br>
"ft. water-dequamed oil": A water-degummed oil is typically obtained by mixing 1-3% w/w of hot water with warm (60-90°C) crude oil. Usual treatment periods are 30-60 minutes. The water-degumming step removes the phosphatides and mucilaginous gums which become insoluble in the oil when hydrated. The hydrated phosphatides and gums can be separated from the oil by settling, filtering or centrifuging - centrifuging being the more preva¬lent practice.<br>
The essential object in said water-degumming process is to separate the hydrated phosphatides from the oil. The mixing of hot water into the oil, described above, should herein be undetstood broadly as mixing of an aqueous solution into the oil according to standard water-degumming procedures in the art.<br>
Alternatively, the process here termed "A water-degummed oil" may be called "wet refining to remove mucilage" (see US 5,264,367).<br>
DETAILED DESCRIPTION OP THE INVENTION<br>
A metliod for enzymatic deguauaing of an edible oil comprising a high amount of non-hydratable phosphatides/phos¬pholipids<br>
For the present invention the amount of non-hydratable phosphorus in an edible oil is measured by,<br>
i}   pre-treating the edible oil, at 60°c,   by addition of a solution comprising citric acid monohydrate in water (added water vs. oil equals 4.8% w/w; [citric acid] in water phase = 106 mM, in water/oil emulsion = 4.6 mM) for 3 0 minutes;<br>
ii)   transferring 10 ml of the pre-treated water in oil emulsion to a tube;<br>
iii)  heating the emulsion in a boiling water bath for 30<br><br>
minutes;<br>
iv)   centrifuging at 5000 rpm for 10 ininutes,<br>
v)    transferring about 8 ml of upper (oil) phase to a new<br>
tube and leaving it (to settle) for 24 hours; vi)   after settling drawing 2 g from upper clear phase for<br>
measurement of the non-hydratable phosphorus content<br>
(ppm) in the edible oil.<br>
For further details reference is made to working examples herein.<br>
As illustrated in working examples herein the phospholipid composition (hydratable vs. non-hydratable phosphplipid) of different edible oils varies considerably. Consequently, the level of remaining phospholipid in different water-degummed oils will vary over a wide range (e.g.   from around 30 ppm to 200 ppm) .<br>
For enzymatic degumming, the optimal enzyme dosage is de¬pendent on the amount of non-hydratable phosphatides present after water degumming or citric acid/water pre-treatment as defined above.<br>
Further, the higher the amount of non-hydratable phosphatides present in the oil is, the more efficient is the enzymatic degumming method.<br>
The present invention provides a method to remove the NHP content in oil comprising a relatively high amount of NHP.<br>
Preferably, the edible oil comprises a non-hydratable phosphorus content of at least 60 ppm, more preferably at least 100 ppm, and even more preferably at least 200 ppm.<br>
More preferably, the edible oil comprises a non-hydratable phosphorous content in the range of 60-500 ppm, more preferably in the range of 100-500 ppm, and even more preferably in the range of 200-500 ppm.<br>
An edible oil defined as having a relatively high amount of a non-hydratable phosphorus, according to the description herein, may be a water degummed oil, or more preferably a crude-oil or a semi-crude oil.<br>
Accordingly, an embodiment of the invention relates to a method according to the first aspect of the invention, wherein said edible oil is a crude oil, characterized in that said crude<br><br>
edible oil, prior to carrying out the method of the invention, is an oil with a phosphorus content above 250 parts per million (ppm), which oil has not been water-degummed (water degumming comprises mixing hot water into a warm crude oil followed by removing phosphatides which become insoluble in the oil when hydrated) prior to carrying out the method of the invention.<br>
Preferably, such a crude edible oil has, prior to carrying out said method of the invention, a phosphorus content above 350 ppm, more preferably above 400 ppm, even more preferably above 500 ppm, and most preferably above 600 ppm.<br>
Further, said crude edible oil preferably has, prior to carrying out said method of the invention, a phosphorus content in the range of 250-1500 ppm, more preferably in the range of 350-1500 ppm, even more preferably in the range of 500-1500 ppm, and most preferably in the range of 500-1500 ppm.<br>
The enzymatic degumming method of a crude edible oil according to the invention is advantageous over the prior art method for enzymatic degumming of a water degummed edible oils (US 5,264,367), since a direct enzymatic degumming method for treatment of a crude oil, according to the invention, will save the prior step of water degumming of the oil.<br>
This saves both time and money. A water-degummed oil is typically obtained by mixing hot water into warm (60-90°C) crude oil for usually 30-60 minutes. In contrast, the complete process of enzymatic degumming of crude oils according to the invention may be performed in less than 1 hour, with actual enzymatic treatment for around 25 minutes. See working example herein for further details.<br>
Further, an edible oil defined as having a relatively high amount of a non-hydratable phosphorus, according to the description herein, may be a semi-crude oil.<br>
Accordingly, an embodiment of the invention relates to a method according to the first aspect of the invention, wherein said edible oil is a semi-crude edible oil, characterized in that said semi-crude edible oil, prior to carrying out the method of the invention, has a phosphorus content above 500 parts per million (ppm), and wherein said oil has been water-degummed prior to carrying out the method of the invention.<br><br>
Preferably, said semi-crude edible oil is an oil which, prior to carrying out said method, has a phosphorus content above 600 parts per million (ppm), more preferably above 750 parts per million (ppm).<br>
In general, water degiunming of an edible oil will reduce the phosphorous contents in the oil to a level below 500 ppm.<br>
Accordingly, a semi-crude oil as described herein may e.gr. only have been partly water-degximmed prior to carrying out a method for reducing the level of phosphorus containing components in an edible oil according to the invention.<br>
The term "partly water degummed" denotes that the water degumming procedure of the oil has only been a partial/short process as compared to a standard water-degiomming procedure.<br>
A "partly water degumming" process may be performed by only mixing only 0.5% of hot water into the oil (standard is 1-3% hot water. See section "Definitions" herein), or by reducing the treatment period to 10 minutes (standard is 30-60 minutes).<br>
Alternatively, a semi-crude oil as defined herein, may be a mixture of a crude oil and a semi-crude oil.<br>
An embodiment of the invention relates to a method according to any of the first aspect of the invention, which comprises the following steps:<br>
i)    adjusting the temperature of the edible oil to a temperature between 25°C-70°C;<br>
ii)   pre-treating the edible oil to the above adjusted<br>
temperature by adding 0.5-6% (by weight relative to the oil) of an aqueous solution comprising at least 85% water for 5-120 minutes, wherein said pre-treatment is not followed by removal of hydrated mucilage and phosphorous content in the oil;<br>
iii)  adjusting the pH of the water/oil emulsion to a pH between 1.5-8 (e.gr. by addition of a suitable amount of a NaOH solution);<br>
iv)   contacting the water/oil emulsion with an aqueous solution of a phospholipase (at a temperature (+/- 5°C) adjusted according to step i), which phospholipase is emulsified in the oil until the phosphorus content of the oil is reduced<br><br>
to less than 11 ppm; v)    separating the aqueous phase from the treated oil.<br>
The temperature of the edible oil in step i) immediately above is preferably adjusted to a temperature which is the optimal phospholipase activity temperature of the enzyme used in the method.<br>
For the commercially available phospholipase Lecitase™ (Novo<br>
Nordisk A/S) this is around 60°C, and for a phospholipase of the invention obtained from the filamentous fungus genus Fusarium, it is around 45°C. See working examples herein for further details relating to this issue.<br>
It is foreseen that a majority of filamentous fungal phospholipases will have a temperature optimum around 3 5-50°C.<br>
Accordingly, an embodiment of the invention relates to the method described immediately above, wherein the temperature of the edible oil in step i) is adjusted to a temperature between 35°C-50°C, and the phospholipase used in step iv) is obtained from a filamentous fungus strain.<br>
In step ii) of the method above, the edible oil is pre-treated at the adjusted temperature (step i)) by addition of 0.5-6% (by weight relative to the oil) of an aqueous solution comprising at least 85% water for 5-120 minutes, and wherein said pre-treatment is not followed by removal of hydrated mucilage and phosphorous content in the oil.<br>
This step is a standard pre-treatment step in enzymatic degumming of edible oils (US 5,264,367; US 5,558,781). The purpose of step ii) is to hydrate the hydratable/hydrophilic components (such as the hydratable phosphorous content) in the edible oil, which when hydrated becomes insoluble in the oil.<br>
However, this step is different from what is termed "water-degumming of an edible oil" in the present context. One major difference is that said pre-treatment step does not remove the hydrated phosphatides and mucilage from the oil. Removal of said hydrated content from the oil is the main purpose of water-degumming of edible oils.<br>
Accordingly, when the phospholipase is contacted with the oil in step iv) above, the oil still comprises said hydrated<br><br>
phosphatides and mucilage.<br>
In other words, if the edible oil is a non-water degummed edible oil, the method above describes a simplified enzymatic degumming method, which does not remove the hydrated phosphatides and mucilage from the oil prior to contacting said oil with the phospholipase.<br>
Preferably, the aqueous solution comprising at least 85% water (step ii above) further comprises citric acid. Preferably, there is between 1-15% (w/w) citric acid in said aqueous solution, more preferably there is between 3-11% (w/w) citric acid in said aqueous solution.<br>
Preferably, the time period in step ii) is 15^-50 minutes, and more preferably 15-30 minutes.<br>
For further details relating to said pre-treatment in step ii) above reference is made to working examples herein.<br>
In step iii) above the pH of the water/oil emulsion is adjusted to pH 1.5-8 (e.g. by addition of a suitable amount of a NaOH solution). This is done in order to adjust the pH value of the oil before the phospholipase is contacted with oil in step iv). In general, the actual optimal pH value will depend on the actual enzyme used to contact the oil in step iv). For further details relating to this issue reference is made to working examples herein.<br>
In general it is preferred, according to the first aspect and their embodiments of the invention, that the contacting of said oil with an aqueous solution comprising a phospholipase is performed at pH 1.5-6, more preferably at pH 3-6.<br>
The pH value in the water in the oil emulsion is measured by taking 2 ml of water from the oil emulsion and mix them with 2 ml of water. After phase separation, the resulting top oil layer is to be pipetted off, and the pH is to be measured in aqueous phase. Measurements are transformed to "real" pH values by the following formula pHreai = PHmeasured ~ 0.38. For further details reference is made to working examples herein.<br>
Preferably, in a method for reducing the amount of phosphorus containing components in an edible oil according to the invention, the amount of a phospholipase which is emulsified in the oil is in the range of 0.1-15 mg enzyme (dry matter)/kg oil, more preferably 0.25-5 mg enzyme (dry matter)/kg oil, and<br><br>
even more preferably 0.25-2.5 mg enzyme (dry matter)/kg oil.<br>
In general, is it advantageous to optimize both the amount of phospholipase used, and the time used for enzymatic degumming of an edible oil to obtain a phosphorus content below 11 ppm. The actual optimal enzyme dose and time period will, i.a., depend on the actual phospholipase used. For further details relating to optimization of enzyme dosage and time period of the method, reference is made to working examples herein.<br>
Preferably, in a method for reducing the amount of phosphorus containing components in an edible oil according to the invention, the phosphorus content of the oil is reduced to less than 11 ppm after contacting said oil with 0^5-6 mg phospholipase (dry matter)/kg oil, and wherein the phospholipase is in contact with said oil for a time period of 1-6 hours, more preferably the phosphorus content of the oil is reduced to less than 11 ppm after contacting said oil with between 0.25-2.5 mg phospholipase (dry matter)/kg oil, and wherein the phospholipase is in contact with said oil for a time period of 15 minutes to 2 hours.<br>
See working examples herein for further details relating to identification of optimal temperatures for individual phospholipases.<br>
Preferably, in all aspects and embodiments of a method for reducing the amount of phosphorus containing components in an edible oil according to the invention, the phosphorus content of the oil is reduced to less than 5 ppm.<br>
The phosphorus content in the oil is measured as parts per million (ppm) in the oil phase of the water present in the oil emulsion. The phosphorus content analysis is carried out accord¬ing to procedure 2.421 in "Standard Methods for the Analysis of Oils, Fats, and Derivatives, 7^^ ed. (1987)". For further de¬tails reference is made to working examples herein.<br>
An embodiment of the invention relates to a method for reducing the amount of phosphorus containing components in an edible oil according to the invention, wherein the phospholipase is obtained from a mammalian species, in particular wherein the phospholipase is obtained from a pancreas in said mammalian species, and most preferably wherein the phospholipase is obtained from a porcine pancreaf<br><br>
Preferably, in a method for reducing the amount of phosphorus containing components in an edible oil according to the invention, the phospholipase is obtained from a microorganism, preferably a filamentous fungus, a yeast, or a bacterium.<br>
Preferably, when said filamentous fungus mentioned immediately above, is a species within the genus Fusarium, preferred strains are strains such as a strain of F.   culmorum, F. heterospoiram,  F.   solan!,   or in particular a strain of F. oxysporum.<br>
Further, when said filamentous fungus above is a species within the genus Aspergillus,  preferred strains are strains such as a strain of Aspergillus awamori,  Aspergillus foetidus, Aspergillus japonicus,  Aspergillus niger  or in particular Aspergillus oryzae.<br>
Further, in a method for reducing the amount of phosphorus containing components in an edible oil according to the invention, the edible oil is preferably a soybean oil, sunflower seed oil, or more preferably a rapeseed oil.<br>
Characterization of Phospholipase obtained from Fusarium oxysporum.<br>
A phospholipase of the invention obtained from Fusarium oxysporum  has been intensively characterized.<br>
Accordingly, an aspect of the invention is preferably an isolated Phosholipase A which is obtained from a strain of the genus Fusarium  and has phospholipase A activity in the pH range 3-10 measured at 40°C, more preferably having phospholipase A activity in the pH range 3-7 measured at 40°C, more preferably having phospholipase A activity in the pH range 3.5-6 measured at 40°C, and even more preferably having phospholipase A activity in the pH range 4.5-5.5 measured at 40°C.<br>
The phospholipase A activity was determined with Soybean Lecithin as substrate NEFA test bases assay), or in a buffer comprising 2% Lecithin, 2% Triton X-100, 20 mM Britton-Robinson (BR). See working examples herein for further details.<br>
In a further embodiment of the invention, an isolated Phosholipase A, which is obtained from a strain of the genus<br><br>
Fusarium,   is preferably one which has a molecular mass of 29 ± 10 kDa, more preferably a molecular mass of 29 ±5 kDa, even more preferably a molecular mass of 29 ± 3 kDa, and most preferably a molecular mass of 29 ± 2 kDa.<br>
The molecular mass is measured by SDS-PAGE electrophoresis as further described in the "Materials and Methods" section (vide infra).<br>
In a fiorther embodiment of the invention, an isolated Phosholipase A which is obtained from a strain of the genus Fusarium  is preferably one which has an isoelectric point (pi) in the range 4.5-8, more preferably an isoelectric point (pi) in the range 5-7.5, and even more preferably an isoelectric point (pi) in the range 5,5-7.5.<br>
The Isoelectric point (pi) was determined by using Am-pholine PAGE plates from Pharmacia. See working example herein for further details {vide infra).<br>
In a further embodiment of the invention, an isolated Phosholipase A which is obtained from a strain of the genus Fusarium  is preferably one which has a temperature optimxim for phospholipase activity in the range between 25-55°C, measured with lecithin as substrate at pH 5; more preferably in the range of 30-50°C, measured with lecithin as substrate at pH 5, and even more preferably in the range of 40-50°C, measured with lecithin as<br>
substrate at pH 5.<br>
The temperature optimum for phospholipase activity was measvired in a buffer comprising 2% lecitihn, 2% Triton X-100, 20 mM Britton Robinson buffer, at pH 5. See working example herein for further details {vide infra).<br>
In a still further embodiment of the invention, an isolated Phospholipase A which is obtained from a strain of the genus Fusarium is preferably one which has phospholipase activity pH optimiom in the pH range of 6-12, at 37°C; more preferably in the pH range of 7-11.5, at 37°C; more preferably in the pH range of 8-11, at 37°C; and even more preferably in the pH range of 8.5-11,<br>
at 37°C.<br>
The phospholipase activity pH optimum was determined in a buffer comprising 2% Lecithin, 2% Triton X-100, 20 mM Britton-<br><br>
Robinson (BR), at 37°C. See working examples herein for further details.<br>
Preferably, a phospholipase of the invention comprises at least two out of the five (numbered i) to v)) above mentioned physical characteristics of the enzyme, more preferably a phospholipase of the invention comprises at least three out of the five (numbered i) to v)) above mentioned physical characteristics of the enzyme, even more preferably a phospholipase of the invention comprises at least four out of the five (numbered i) to v)) above mentioned physical characteristics of the enzyme, and most preferably a phospholipase of the invention comprises all of the five (numbered i) to v)) above mentioned physical characteristics of the enzyme.<br>
As described above, a phospholipase of the invention has been cloned, recombinantly expressed, purified, and the N-terminal and C-terminal sequences of the active secreted enzyme have been determined.<br>
Accordingly, a further embodiment of the invention relates to an isolated polypeptide having phospholipase A activity which polypeptide is obtained from a strain of the genus Fusarium  and has<br>
i)   PLA activity in the pH range 3-10, measured at 40°C; ii)  a molecular mass of 29 ± 10 kDa, as determined by SDS-PAGE; iii) an isoelectric point (pi) in the range 4.5-8; iv)  a temperature optimum for phospholipase activity in the range<br>
of 25-55°C, measured with lecithin as substrate at pH 5;<br>
and/or V)   a phospholipase activity pH optimiom in the pH range of 6-12,<br>
measured with lecithin as substrate at 37°C;<br>
and further comprises an amino acid sequence selected from the group comprising:<br>
(a)	a polypeptide encoded by the phospholipase A enzyme encoding<br>
part of the DNA sequence cloned into plasmid pYES 2.0 present in Escherichia coli  DSM 11299;<br>
(b)	a polypeptide having an aihino acid sequence as shown in<br>
positions 31-3 46 of SEQ ID NO 2;<br><br>
(c)	a polypeptide having:an amino acid sequence as shown in position 31-3 03 of SEQ ID NO 2;<br>
(d)	an analogue of the polypeptide defined in (a), (h)   or (c) which is at least 70 % homologous with said polypeptide; and<br>
(e)	a fragment of (a), (b) (c) or (d).<br>
In an embodiment of the invention, the isolated polypeptide having phospholipase activity according to the invention is phospholipase having phospholipase Al activity.<br>
In a further embodiment, the isolated polypeptide having phospholipase activity according to the invention is phospholipase having a phospholipase A2 activity, and in an even further embodiment the isolated polypeptide having phospholipase activity according to the invention is a phospholipase having phospholipase B activity.<br>
Preferably, said isolated polypeptide having phospholipase activity according to the invention, is phospholipase having phospholipase Al activity.<br>
For specific examples of standard techniques to measure individual PLAl, PLA2 and/or PLB activity reference is made to working examples herein.<br>
In a fxirther embodiment the invention relates to an isolated polypeptide having phospholipase activity according to the invention, wherein the phospholipase is a phospholipase which is substantially independent of Ca^'*' concentration measured as, relative phospholipase activity at 5 mM EDTA and 5inM Ca^"*" in a phospholipase activity assay measuring release of free fatty acids from lecithin in a buffer comprising 2% lecithin, 2% Triton X-100, 20 mM citrate, pH 5; incubated for 10 min. at 37°C followed by stop of reaction at 95°C for 5 min.; wherein the ratio of relative phospholipase activity at 5mM EDTA/5 mM Ca^"^ is greater than 0.25, more preferably greater than 0.5, and most preferably greater than 0.80.<br>
For further details relating to measurement of the dependence of the enzyme activity on the Ca2+ concentration, reference is made to working examples herein.<br>
Some lipases may have limited phospholipase activity. In the present context such limited phospholipase activity of said<br><br>
lipases are defined as "a lipase with phospholipase side activity" (see section "Definitions" herein). The present invention relates to an isolated polypeptide having phospholipase activity, wherein in phospholipase activity of said isolated polypeptide is as high as it is of industrial relevance.<br>
Accordingly, the invention relates to an isolated polypeptide having phospholipase activity according to the invention, wherein the phospholipase is a phospholipase having a phospholipase activity which is at least 0.25 nmol/min, enzyme dose: 60 fig, at 25°C; more preferably at least 0.40 nmol/min,<br>
enzyme dose: 60 |ig, at 25°C; measured in a monolayer<br>
phospholipase assay as follows:	r*<br>
a.	in a monolayer equipment (zero-order through) on a<br>
thoroughly purified surface of a buffer solution (10 mM TRIS, pH<br>
8.0, 25°C) a monolayer of the phospholipid DDPC (Di Dicanoyl<br>
(CIO) Phosphatidyl Choline) is spread from a chloroform<br>
solution;<br>
b.	after relaxation of the monolayer (evaporation of<br>
chloroform), the surface pressure is adjusted to 15 mN/m,<br>
corresponding to a mean molecular area of DDPC of approx. 63<br>
A^/molecule;<br>
c.	a buffer solution (as above) containing 60 /xg enzyme is<br>
injected through the monolayer into the subphase of the reaction<br>
compartment (cylinder with area 1520 mm^ and volume 3 0400 mm^)<br>
in the "zero-order trough";<br>
d.	enzymatic activity is determined through the speed of a<br>
mobile barrier compressing the monolayer in order to maintain<br>
constant surface pressure as insoluble substrate molecules are<br>
hydrolysed into more water soluble reaction products, where the<br>
number of DDPC-molecules hydrolyzed per minute by the enzyme is<br>
estimated from the mean molecular area (MMA) of DDPC.<br>
See section "Definitions" and working examples herein for further descriptions of preferred amounts of phospholipase activities for an isolated polypeptide having phospholipase activity according to the invention.<br>
Further, the specific phospholipase activity of a phospholipase according to th^ invention may be measured by known standard phospholipase activity assays.<br><br>
Accordingly, in a further embodiment the present invention relates to an isolated polypeptide having phospholipase activity according to the invention, wherein the phospholipase is a phospholipase which has a phospholipase activity capable of releasing at least 7 |4.mol of free fatty acid/min./mg enzyme; more preferably at least 15 |imol of free fatty acid/min./mg enzyme; even more preferably at least 30 |imol of free fatty acid/min./mg enzyme, and most preferably at least 50 jimol of free fatty acid/min./mg enzyme measured as follows:<br>
phospholipase activity is measured in an assay measuring release of free fatty acids from lecithin in a buffer comprising 2% lecithin, 2% Triton X-100, 20 mM citrate, pH 5; incubated for 10 min. at 37°C followed by stop of reaction at 95°C for 5 min.<br>
For further details relating to this embodiment of the invention reference is made to working examples herein.<br>
An isolated polypeptide having phospholipase activity according to the invention is very suitable for performing enzymatic degiomming of an edible oil.<br>
Accordingly, the invention relates to:<br>
1.	an isolated polypeptide having phospholipase activity<br>
according to the invention, wherein the phospholipase is capable<br>
of performing enzymatic degumming of an edible oil, according to<br>
a method of the invention in order to reduce the amount of<br>
phosphorus containing components in an edible oil comprising a<br>
&gt;  non-hydratable phosphorus content of at least 50 ppm; and<br>
2.	an isolated polypeptide having phospholipase activity<br>
according to the invention, wherein the phospholipase is capable<br>
of performing enzymatic degumming of a water-degiommed edible oil<br>
(having a phosphorus content of 50-250 ppm) thereby reducing the<br>
» phosphorus content of the oil to less than 11 ppm, wherein the enzymatic degumming process comprises contacting said oil at a pH from 1.5-8 with an aqueous solution of the phospholipase which is emulsified in the oil until the phosphorus content of the oil is reduced to less than 11 ppm, and then separating the<br>
i aqueous phase from the treated oil.<br>
Preferably, the isolated polypeptide having phospholipase activity according to the invention is capable of performing said enzymatic degximming process of the water-degummed edible<br><br>
oil (defined immediately above) in less than 1.5 hours, and uses less than 2 mg phospholipase (dry matter)/kg oil.<br>
Preferably, an isolated polypeptide exhibiting phospholipase activity and having the cheuracteristic shown above of the invention is obtained from a filamentous fvingal strain within the genus of Fusarium.<br>
However, without being limited to any theory it is at present contemplated that a phospholipase of the invention can also be obtained from another microorganism, preferably another filamentous fungal strain. Examples thereof are given in the section "Microbial sources" (vide infra).<br>
Cloned DNA sequence<br>
Despite a number of technical difficulties (See section "Protocol for cloning a filamentous fungal phospholipase", vide infra)   the present inventors have been able to clone a phospholipase exhibiting PLA activity from a strain of the genus Fusarium,  more specifically Fusarium oxysporum.<br>
Further, it is presently believed that it is possible to clone both a related phospholipase A and/or phospholipase B encoding DNA sequence based on the sequence infomaation provided in the present application.<br>
Accordingly, an aspect of the invention relates to a cloned DNA sequence encoding an enzyme exhibiting phospholipase A and/or phospholipase B activity, which DNA sequence is selected from the group comprising:<br>
(a)	the phospholipase A encoding part of the polynucleotide cloned into plasmid pYES 2.0 present in Escherichia coli  DSM 11299;<br>
(b)	the DNA sequence shown in positions 23-1063 in SEQ ID NO 1, more preferably positions 113-1063 in SEQ ID NO 1, or even more preferably positions 113-929 in SEQ ID NO 1, or its complementary strand;<br>
(c)	a DNA sequence which is at least 70% homologous with said DNA sequence defined in (a) or (b);<br>
(d)	a DNA sequence defined in (a) or (b), which encodes a polypeptide exhibiting phospholipase activity and is at least 70% homologous with the polypeptide sequence shown in positions 31-346 of SEQ ID No 2, or more preferably at least<br><br>
70% homologous with the polypeptide sequence shown in positions 31-3 03 of SEQ ID No 2;<br>
(e)	a DNA sequence which hybridizes with a double-stranded DNA probe comprising the DNA sequence shown in positions 23-1063 in SEQ ID NO 1 at low stringency;<br>
(f)	a DNA sequence which encodes for a polypeptide having the amino acid sequences of residues 1 to 346, 31 to 346 or 31 to 3 03 of SEQ ID No 2 or the amino acid sequences encoded by any of the DNA sequences of (e); and<br>
(g)	a DNA sequence which is a fragment of the DNA sequences specified in (a), (b), (c), (d), (e), or (f).<br>
In this specification, whenever reference is made to the phospholipase encoding part of the DNA sequence cloned into plasmid pYES 2.0 present in DSM 11299 such reference is also intended to include the phospholipase encoding part of the DNA sequence presented in SEQ ID NO 1.<br>
Accordingly, the terms "the phospholipase encoding part of the DNA sequence cloned into plasmid pYES 2.0 present in DSM 11299" and "the phospholipase encoding part of the DNA sequence presented in SEQ ID NO 1" may be used interchangeably.<br>
The DNA sequence may be of genomic, cDNA, or synthetic origin or any combination thereof.<br>
The present invention also encompasses a cloned DNA sequence which encodes an enzyme exhibiting phospholipase A and/or phospholipase B activity having the amino acid sequence set forth as the mature part of SEQ ID NO 2, which differs from SEQ ID NO 1 by virtue of the degeneracy of the genetic code.<br>
The DNA sequence shown in SEQ ID NO 1 and/or an analogue DNA sequence of the invention may be cloned from a strain of the filamentous fungus Fusarium oxysporum  producing the enzyme with phospholipase activity, or another or related organism as further described below (See section "Microbial sources").<br>
Alternatively, the analogous sequence may be constructed on the basis of the DNA sequence presented as the phospholipase encoding part of SEQ ID No. 1, e.g.   be a sub-sequence thereof, and/or be constructed by introduction of nucleotide substitutions which do not give rise to another amino acid sequence of the phospholipase encoded by the DNA sequence, but<br><br>
corresponds to the codon usage of the host organism intended for production of the enzyme, or by introduction of nucleotide substitutions which may give rise to a different amino acid sequence (i.e. a variant of the phospholipase of the invention).<br>
When carrying out nucleotide substitutions, amino acid changes are preferably of a minor nature, i.e. conservative amino acid substitutions which do not significantly affect the folding or activity of the protein; small deletions, typically of one to about 3 0 amino acids; small amino- or carboxyl-terminal extensions, such as an amino-terminal methionine residue; a small linker peptide of up to about 20-25 residues; or a small extension that facilitates purification, such as a poly-histidine tract; an antigenic epitope or a binding domain.<br>
Examples of conservative substitutions are within the group of basic amino acids, such as arginine, lysine, histidine; acidic amino acids, such as glutamic acid and aspartic acid; polar amino acids, such as glutamine and asparagine; hydrophobic amino acids, such as leucine, isoleucine, valine; aromatic amino acids, such as phenylalanine, tryptophan, tyrosine; and small<br>
I amino acids, such as glycine, alanine, serine, threonine, methionine. For a general description of nucleotide substitution, see e.g.   Ford et al., (1991), Protein Expression and Purification 2, 95-107.<br>
It will be apparent to persons skilled in the art that<br>
I such substitutions can be made outside the regions critical to the function of the molecule and still result in an active poly¬peptide. Amino acids essential to the activity of the poly¬peptide encoded by the cloned DNA sequence of the invention and therefore preferably not subject to substitution may be<br>
) identified according to procedures known in the art, such as site-directed mutagenesis or alanine-scanning mutagenesis (cf. e.g.   Cunningham and Wells, (1989), Science 244, 1081-1085). In the latter technique mutations are introduced at every residue in the molecule, and the resulting mutant molecules are tested<br>
5 for biological (i.e. phospholipase) activity to identify amino acid residues that are critical to the activity of the molecule. Sites of substrate-enzyme interaction can also be determined by analysis of crystal structure as determined by such techniques as nuclear magnetic resonance analysis, crystallography or photo<br><br>
affinity labelling (cf. e.g. de Vos et al., (1992), Science 255, 306-312; Smith et al., (1992), J. Mol. Biol. 224, 899-904; Wlodaver et al., (1992), FEES Lett. 309, 59-64).<br>
Polypeptides of the present invention also include fused polypeptides or cleavable fusion polypeptides, into which another polypeptide is fused at the N-terminal or the C-terminal of the polypeptide or fragment thereof.  A fused polypeptide is produced by fusing a nucleic acid sequence (or a portion thereof) encoding another polypeptide to a nucleic acid sequence (or a portion thereof) of the present invention.  Techniques for producing fused polypeptides are known in the art and include ligating the coding sequences encoding the polypeptides so that they are in frame and so that expression of the fused polypeptide is under control of the same promoter(s) and terminator.<br>
The DNA sequence of the invention can be cloned from the strain Escherichia coli  DSM No. 11299 using standard cloning techniques e.g. as described by Sambrook et al., (1989), Molecular Cloning: A Laboratory Manual. Cold Spring Harbor Lab.; Cold Spring Harbor, NY.<br>
Since the present inventors have solved the problem of developing a suitable screening assay for use in an expression cloning technique to clone a phospholipase of the invention, see section with the heading "Protocol for cloning a filamentous fungal phospholipase", the DNA sequence of the invention can now be cloned by any general method involving<br>
•	cloning, in suitable vectors, a cDNA library from any<br>
organism expected to produce the phospholipase of<br>
interest,<br>
I   •   transforming suitable yeast host cells with said vectors,<br>
•	culturing the host cells under suitable conditions to<br>
express any enzyme of interest encoded by a clone in the<br>
cDNA library,<br>
;   •   screening for positive clones by determining any<br>
phospholipase activity of the enzyme produced by such clones, and<br>
•	isolating the enzyme encoding DNA from such clones.<br><br>
Alternatively, since the present invention for the first time provides a cloned DNA sequence encoding a filamentous fungal PLA enzyme, the DNA encoding a phospholipase of the invention may, in accordance with well-known procedures, conveniently be cloned from a suitable source, such as any of the organisms mentioned in the section "Microbial Sources", by use of synthetic oligonucleotide probes prepared on the basis of a DNA sequence disclosed herein. For instance, a suitable oligonucleotide probe may be prepared on the basis of the phospholipase encoding part of the nucleotide sequences presented as SEQ ID No. 1 or any suitable subsequence thereof, or the basis of the amino acid sequence SEQ ID NO 2.<br>
Further, since a cloned DNA sequence of the invention encodes a polypeptide having phospholipase activity according to the invention, a number of the specific embodiments relating to an isolated polypeptide having phospholipase activity of the invention are also embodiments of the invention for a cloned DNA sequence of the invention encoding a polypeptide having phospholipase activity. Consequently, references and preferred and most preferred embodiments of said isolated polypeptide having phospholipase activity also relate to a cloned DNA sequence of the invention.<br>
Accordingly, an embodiment of the invention relates to a cloned DNA sequence according to the invention, wherein the phospholipase encoded by said DNA sequence is a phospholipase Al.<br>
In a further embodiment, a cloned sequence according to the invention is a cloned DNA sequence, wherein the phospholipase encoded by said DNA sequence is a phospholipase A2, and in an even I further embodiment a cloned sequence according to the invention is a cloned DNA sequence, wherein the phospholipase encoded by said DNA sequence is a phospholipase B.<br>
Preferably, said cloned DNA sequence according to the invention encodes a polypeptide having phospholipase Al activity. 1    Further, the invention relates to a cloned DNA sequence according to the invention, wherein the phospholipase encoded by said DNA sequence is a phospholipase which is substantially independent of Ca^"' concentration measured as:<br>
relative phospholipase activity at 5 mM EDTA and 5mM Ca^"*" in<br><br>
a phospholipase activity assay measuring release of free fatty acids from lecithin in a buffer comprising 2% lecithin, 2% Triton X-100, 20 mM citrate, pH 5; incubated for 10 min. at 37°C followed by stop of reaction at 95°C for 5 min.; wherein the ratio of relative phospholipase activity at 5mM EDTA/5 mM Ca^"*" is a ratio which is greater than 0.25, more preferably a ratio which is greater than 0.5.<br>
Even further, the invention relates to a cloned DNA sequence according to the invention, wherein the phospholipase encoded by said DNA sequence is a phospholipase having a phospholipase activity which is at least 0.25 nmol/min, enzyme dose: 60 pig, at 25°C; more preferably at least 0.40 nmol/min, enzyme dose: 60 ug, at 25°C; measured in a monolayer phospholipase assay as follows:<br>
a.  in a monolayer equipment (zero-order through) on a thoroughly purified surface of a buffer solution (10 mM TRIS, pH 8.0, 25°C) a monolayer of the phospholipid DDPC (Di Dicanoyl (CIO) Phosphatidyl Choline) is spread from a chloroform solution; I b.  after relaxation of the monolayer (evaporation of chloroform), the surface pressure is adjusted to 15 mN/m, corresponding to a mean molecular area of DDPC of approx. 63 A^/molecule;<br>
c.	an buffer solution (as above) containing 60 /xg enzyme is<br>
i injected through the monolayer into the subphase of the reaction compartment (cylinder with area 1520 mm^ and volume 30400 mm^) in the "zero-order trough";<br>
d.	enzymatic activity is determined through the speed of a<br>
mobile barrier compressing the monolayer in order to maintain<br>
) constant surface pressure as insoluble substrate molecules are hydrolysed into more water soluble reaction products, where the number of DDPC-molecules hydrolyzed per minute by the enzyme is estimated from the mean molecular area (MMA) of DDPC.<br>
In a further embodiment the invention relates to a cloned<br>
5 DNA sequence according to the invention, wherein the phospholipase encoded by said DNA sequence is a phospholipase having a phospholipase activity which is capable of releasing at<br><br>
least 7 M,mol of free fatty acid/min./mg enzyme; more preferably<br>
at least 15 nmol of free fatty acid/min./mg enzyme; measured as follows:<br>
phospholipase activity is measured in an assay measuring release of free fatty acids from lecithin in a buffer comprising 2% lecithin, 2% Triton X-100, 20 mM citrate, pH 5; incubated for<br>
10 min. at 37°C followed by stop of reaction at 95°C for 5 min. In further embodiments the invention relates to: a cloned DNA sequence according to the invention, wherein the phospholipase encoded by said DNA sequence is capable of performing enzymatic degumming of an edible oil according to a method of the invention in order to reduce the amount of phosphorus containing components in an edible oil comprising a non-hydratable phosphorus content of at least 50 ppm; and<br>
a cloned DNA sequence according to the invention, wherein the phospholipase encoded by said DNA sequence is capable of performing an enzymatic degumming of a water-degummed edible oil (having a phosphorus content of 50-250 ppm) and thereby reducing the phosphorus content of the oil to less than 11 ppm, wherein<br>
I the enzymatic degumming process comprises contacting said oil at a pH from 1.5-8 with an aqueous solution of the phospholipase which is emulsified in the oil until the phosphorus content of the oil is reduced to less than 11 ppm, and then separating the aqueous phase from the treated oil.<br>
i    Preferably, a cloned DNA sequence according to the invention is a cloned DNA sequence, wherein the phospholipase encoded by said DNA sequence is capable of performing said enzymatic degumming process of the water-degummed edible oil by using less than 2 mg phospholipase (dry matter)/kg oil, and wherein the<br>
) phospholipase is in contact with said oil for a time period of 15 minutes to 2 hours.<br>
Protocol for cloning a filamentous fungal phospholipase<br>
A number of technical difficulties were encountered when ) attemting to isolate a phospholipase of the invention or clone a polynucleotide coding for it. It seemed impossible to isolate the enzyme, and the problem of cloning the polynucleotide was pursued.<br><br>
As described herein no prior DNA sequence encoding a filamentous fungal phospholipase A was available. Consequently, the present inventors developed a cloning strategy based on the expression cloning in yeast technique (H. Dalboege et al. Mol. Gen. Genet (1994) 243:253-260.; WO 93/11249; and WO 94/14953).<br>
One of the major problems encountered with this technique was that yeast produces an internal activity giving rise to a phospholipase background on plate assays. This background was found to be highly dependent on the amount of substrate in the assay plates, and the amount of substrate thus had to be carefully titrated to a level where the background was low enough for the assay to be reliable during the expression cloning screening procedure, but high enough for the reaction to take place. In order to remove this background, it was ???<br>
Furthermore, filamentous fungal strains in general comprise a number of different lipases, some of which even exhibit limited phospholipase activity. Such lipases are herein defined as "A lipase with phospholipase side activity" (see section "Definitions" herein).<br>
In the plate assay, the background of such lipases with phospholipase side activity was found also to be highly dependent on the amount of substrate in the assay plates, and the amount of substrate thus had to be even more carefully titrated in order to eliminate the background activity from both the yeast cells and the filamentous fungal lipases with phospholipase side activity.<br>
On top of this it was found that a carefull selection of the substrate had to be performed, since many did not provide any functional solution to this problem, since a number of the I phospholipase substrates tested gave a background activity because lipases, without phospholipase activity, were able to react on the substrates. Accordingly, a high number of substrates had to be tested and titrated in order to identify a suitable substrate, i        The solution found to make it possible to perform the expression cloning of a phospholipase coding polynucleotide was to use Lipoid E80 (from Lipoid GmbH) in carefully monitored concentrations. In the Materials and Method section herein a detailed description of the complete expression cloning in yeast<br><br>
protocol is disclosed, including a plate assay solving the above described problems.<br>
Homology/identity of DNA sequences<br>
The DNA sequence homology/identity referred to above is determined as the degree of identity between two sequences indicating a deviation of the first sequence from the second. The homology may suitably be determined by means of computer programs known in the art, such as GAP provided in the GCG program package (Program Manual for the Wisconsin Package, Version 8, August 1994, Genetics Computer Group, 575 Science Drive, Madison, Wisconsin, USA 53711)(Needleman, S.B. and Wunsch, CD., (1970), Journal of Molecular Biology, 48, 443-453). Using GAP with the following settings for DNA sequence comparison: GAP creation penalty of 5.0 and GAP extension penalty of 0.3, the coding region of the DNA sequence exhibits a degree of identity of preferably at least 70%, more preferably at least 80%, more preferably at least 90%, more preferably at least 95%, more preferably at least 97% with the phospholipase encoding part of the DNA sequence shown in SEQ ID No. 1 (i.e. position 23-1063 in SEQ ID NO 1); or more preferred with the DNA sequence shown in position 113-1063 in SEQ ID NO 1 (pos. 113 corresponds to the N-terminal residue of the mature enzyme); or even more preferred with the DNA sequence shown in position 23-929 in SEQ ID NO 1 (pos. 929 corresponds to the C-terminal residue in C-terminal processed secreted active enzyme).<br>
Hybridization<br>
The hybridization referred to above is intended to I comprise an analogous DNA sequence which hybridizes to a double-stranded DNA probe corresponding to the phospholipase encoding part of the DNA sequence shown in SEQ ID NO 1, i.e. nucleotides 23-1063, or more preferred with a double-stranded DNA probe corresponding to the DNA sequence shown in position 113-1063 in i SEQ ID NO 1 (pos. 113 corresponds to the N-terminal residue of the mature enzyme); or even more preferred with a double-stranded DNA probe corresponding to the DNA sequence shown in position 23-929 in SEQ ID NO 1 (pos. 929 corresponds to the C-terminal residue in the C-terminal processed secreted active<br><br>
enzyme), under at least low stringency conditions as described in detail below.<br>
Suitable experimental conditions for determining hybridization at low, medium, or high stringency between a nucleotide probe and a homologous DNA or RNA sequence involve presoaking of the filter containing the DNA fragments or RNA to hybridize in 5 x SSC (Sodium chloride/Sodium citrate, Sambrook et al. 1989) for 10 min, and prehybridization of the filter in a solution of 5 X SSC, 5 x Denhardt's solution (Sambrook et al. 1989), 0.5 % SDS and 100 /xg/ml of denatured sonicated salmon sperm DNA (Sambrook et al. 1989), followed by hybridization in the same solution containing 10 ng/ml of a random-primed (Feinberg, A. P. and Vogelstein, B. (1983) Anal.  Biochem.   132:6-13) , -^^P-dCTP-labeled (specific activity &gt; 1 x 10^ cpm/^g ) probe for 12 hours at ca. 45°C. The filter is then washed twice for 30 minutes in 2 x SSC, 0.5 % SDS at a temperature of at least 55«*C (low stringency) , more preferably at least 60°C (medium stringency), still more preferably at least 65°C (medium/high stringency), even more preferably at least 70°C<br>
I (high stringency), even more preferably at least 75°C (very high stringency).<br>
Molecules to which the oligonucleotide probe hybridizes under these conditions are detected using an X-ray film.<br>
It has been found that it is possible to theoretically<br>
I predict whether or not two given DNA sequences will hybridize under certain specified conditions.<br>
Accordingly, as an alternative to the above described experimental method the determination of whether or not an analogous DNA sequence will hybridize to the nucleotide probe<br>
) described above can be based on a theoretical calculation of the Tm (melting temperature) at which two heterologous DNA sequences with known sequences will hybridize under specified conditions (e.g. with respect to cation concentration and temperature). In order to determine the melting temperature for<br>
■) heterologous DNA sequences (Tm(hetero)) it is necessary to initially determine the melting temperature (Tm(homo)) for homologous DNA sequences.<br>
The melting temperature (Tm(homo) between two fully complementary DNA strands (homoduplex formation) may be determined<br><br>
by use of the following formula:<br>
Tm(homo) = 81.5°C + 16.6(log M) + 0.41(%GC) - 0.61 (% form) -500/L ("Current protocols in Molecular Biology". John Wiley and Sons, 1995), wherein<br>
"M" denotes the molar cation concentration in wash<br>
buffer,<br>
"%GC" % Guanine (G) and Cytosine (C) of the total number<br>
of bases in the DNA sequence,<br>
"% form" % formamid in the wash buffer, and<br>
"L" the length of the DNA sequence.<br>
Using this formula and the experimental wash conditions given above, Tm(homo) for the homoduplex formation of the nucleotide probe corresponding to the DNA sequence shown in SEQ ID NO 1, i.e. nucleotides 23-1060 is:<br>
Tm(homo) = 81.5 + 16.6 (log 0.30) + 0.41(56) - 0.61(0) -(500/1038) Tm(homo) = 103.5° C.<br>
"M": 2 X SSC corresponds to a cation cone, of 0.3M.<br>
"%GC" The %GC in SEQ ID No 1 pos. 23-1060 is 56% I      "% form": There is no formamid in the wash buffer.<br>
"L": The length of SEQ ID No 1 SEQ ID No 1 pos. 23-1063<br>
1038 bp.<br>
The Tm determined by the above formula is the Tm of a homoduplex formation (Tm(homo)) between two fully complementary ; DNA sequences. In order to adapt the Tm value to that of two heterologous DNA sequences, it is assumed that a 1% difference in nucleotide sequence between the two heterologous sequences equals a 1°C decrease in Tm ("Current protocols in Molecular Biology". John Wiley and Sons, 1995). Therefore, the Tm(hetero) for the ) heteroduplex formation is found by subtracting the homology % difference between the analogous sequence in question and the nucleotide probe described above from the Tm(homo). The DNA homology percentage to be subtracted is calculated as described herein (vide supra).<br>
3<br>
Homology to luaino acid sequences<br>
The polypeptide homology referred to above is determined as the degree of identity between two sequences indicating a<br><br>
deviation of the first sequence from the second. The homology may suitably be determined by means of computer programs known in the art such as GAP provided in the GCG program package (Program Manual for the Wisconsin Package,  Version 8, August 1994, Genetics Computer Group, 575 Science Drive, Madison, Wisconsin, USA 53711) (Needleman, S.B. and Wunsch, CD., (1970), Journal of Molecular Biology, 48, 443-453. Using GAP with the following settings for polypeptide sequence comparison: GAP creation penalty of 3.0 and GAP extension penalty of 0.1, the mature part of a polypeptide encoded by an analogous DNA sequence exhibits a degree of identity preferably of at least 70%, more preferably at least 80%, more preferably at least 90%, more preferably at least 95%, especially at least 97% with the mature part of the amino acid sequence shown in SEQ ID NO 2, i.e. position 31-346 in SEQ ID NO 2, or more preferred with the amino acid sequence shown in position 31-303 of SEQ ID NO 2 (pos. 3 03 is the C-terminal residue in C-terminal processed secreted active enzyme).<br>
The present invention is also directed to phospholipase<br>
I variants with an amino acid sequence which differs by not more than three amino acids, preferably by not more than two amino acids, and more preferably by not more than one amino acid from the mature part of the amino acid sequence set forth in SEQ ID NO 2.<br>
)     Further, the above mentioned preferred amino acid identities also relate to an analogue of a cloned DNA sequence of the invention, which sequence encodes a polypeptide exhibiting phospholipase activity, and which is at least 70% homologous with the polypeptide sequence shown in positions 31-<br>
) 346 of SEQ ID No 2, or more preferably at least 70% homologous with the polypeptide sequence comprising positions 31-303 of SEQ ID No 2.<br>
Immunological cross-reactivity<br>
5        Antibodies to be used in determining immunological cross-reactivity may be prepared by using a purified phospholipase. More specifically, antiserum against the phospholipase of the invention may be raised by immunizing rabbits (or other rodents) according to the procedure described<br><br>
by N. Axelsen et al. in A Manual of Quantitative Immunoelectrophoresis, Blackwell Scientific Publications, 1973, Chapter 23, or A. Johnstone and R. Thorpe, Immunochemistry in Practice, Blackwell Scientific Publications, 1982 (more speci¬fically p. 27-31). Purified immunoglobulins may be obtained from the antiserum obtained, for example by salt precipitation ((NH4)2 SO4), followed by dialysis and ion exchange chromatography, e.g.   on DEAE-Sephadex. Immunochemical characterization of proteins may be performed either by Outcherlony double-diffusion analysis (O. Ouchterlony in: Handbook of Experimental Immunology (D.M. Weir, Ed.)/ Blackwell Scientific Publications, 1967, pp. 655-706), by crossed Immunoelectrophoresis (N. Axelsen et al., supra. Chapters 3 and 4), or by rocket Immunoelectrophoresis (N. Axelsen et al.. Chapter 2).<br>
Microbial Sources<br>
At the priority date of the present invention, the taxonomy applied below is in accordance with the World Wide Web (WWW) NCBI taxonomy browser.<br>
An isolated polypeptide having phospholipase activity and the corresponding cloned DNA sequence of the invention may be obtained from any microorganism, preferably a filamentous fungus, a yeast cell, or a bacterium.<br>
Preferably, a phospholipase and the corresponding cloned DNA sequence of the invention may be obtained from a filamentous fungal strain, wherein a preferred phylum is Ascomycota,  wherein a preferred class is Pyrenomycetes  comprising the preferred family Nectriaceae.<br>
More preferably, the phospholipase and the corresponding cloned DNA sequence of the invention may be obtained from a strain of the genus Fusarium,   such as a strain of F.   culmorum, F. heterosporum,  or F.  solan!,   in particular a strain of Fusarium oxysporum.<br>
Further, a phospholipase and the corresponding cloned DNA sequence of the invention may be obtained from a filamentous fungal strain within the genus Aspergillus,  such as a strain of Aspergillus awamori, Aspergillus foetidus,  Aspergillus japonicus, Aspergillus niger  or in particular Aspergillus oryzae.<br><br>
An isolate of a strain of Fusarium oxysporum  from which a phospholipase of the invention can be obtained has been deposited according to the Budapest Treaty on the International Recognition of the Deposit of Microorganisms for the Purposes of Patent Procedure at the Deutche Sammlung von Mikroorganismen und Zellkulturen GmbH., Mascheroder Weg lb, D-38124 Braunschweig, Federal Republic of Germany, (DSM).<br>
Deposit date       : 6 June 1983<br>
Depositor's ref.    : NN041759<br>
DSM No.	: Fusarium oxysporum  DSM No. 2672<br>
Further, the expression plasmid pYES 2.0 comprising the full length cDNA sequence encoding the phospholipase of the invention has been transformed into a strain of the Escherichia coli  which was deposited according to the Budapest Treaty on the International Recognition of the Deposit of Microorganisms for the Purposes of Patent Procedure at the Deutche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg lb, D-38124 Braunschweig, Federal Republic of Germany, (DSM).<br>
Deposit date       : 25 November 1996<br>
Depositor's ref.   : NN049279<br>
DSM No.	: Escherichia coli  DSM No. 11299<br>
Expression vectors<br>
The expression vector of the invention may be any expression vector that is conveniently subjected to recombinant DNA procedures, and the choice of vector will often depend on the host cell into which the vector it is to be introduced. Thus, the vector may be an autonomously replicating vector, i.e. a vector which exists as an extrachromosomal entity, the replication of which is independent of chromosomal replication, e.g. a plasmid. Alternatively, the vector may be one which, when introduced into a host cell, is integrated into the host cell genome and replicated together with the chromosome(s) into which it has been integrated.<br>
In the expression vector, the DNA sequence encoding the phospholipase should be operably connected to a suitable<br><br>
promoter and terminator sequence. The promoter may be any DNA sequence which shows transcriptional activity in the host cell of choice and may be derived from genes encoding proteins which are either homologous or heterologous to the host cell. The procedures used to ligate the DNA sequences coding for the phospholipase, the promoter and the terminator and to insert them into suitable vectors are well known to persons skilled in the art (cf. e.g.   Sambrook et al., (1989), Molecular Cloning. A Laboratory Manual, Cold Spring Harbor, NY).<br>
Examples of suitable promoters for use in filamentous fungus host cells are, e.g. the ADH3 promoter {McKnight et al., The EMBO J. 1 (1985), 2093 - 2099) or the tpiA promoter. Examples of other useful promoters are those derived from the gene encoding Aspergillus oryzae  TAKA amylase, Rhizomucor miehei aspartic proteinase, Aspergillus niger  neutral a-amylase, Aspergillus niger  acid stable a-amylase, Aspergillus niger  or Aspergillus awamori  glucoamylase (gliiA) , Rhizomucor miehei lipase, Aspergillus oryzae  alkaline protease, Aspergillus oryzae triose phosphate isomerase or Aspergillus nidulans  acetamidase.<br>
Host Cells<br>
The present invention also relates to recombinant host cells comprising a nucleic acid sequence of the invention, which cells may advantageously be used in the recombinant production of the polypeptides.  The term "host cell" encompasses any progeny of a parent cell which is not identical to the parent cell due to mutations that occur during replication.<br>
The cell is preferably transformed with a vector comprising a nucleic acid sequence of the invention followed by integration of the vector into the host chromosome.<br>
"Transformation" means introducing a vector comprising a nucleic acid sequence of the present invention into a host cell so that the vector is maintained as a chromosomal integrant or as a self-replicating extra-chromosomal vector.  Integration is generally considered to be an advantage as the nucleic acid sequence is more likely to be stably maintained in the cell. Integration of the vector into the host chromosome may occur by homologous or non-homologous recombination as described above.<br><br>
In a preferred embodiment, the host cell is a fungal cell. "Fungi" as used herein includes the phyla Ascomycota, Basidiomycota,  Chytridiomycota,   and Zygomycota   (as defined by Hawksworth et al., In,  Ainsworth and Bisbys Dictionary of The Fungi,  8th edition, 1995, CAB International, University Press, Cambridge, UK) as well as the Oomycota  (as cited in Hawksworth et al.,  1995, supra,  page 171) and all mitosporic fungi (Hawksworth et al.,   1995, supra).     Representative groups of Ascomycota include, e.g.,  Neurospora,  Eupenicillium  (=Penicillium), Emericella   (=Aspergillus),  Eurotium  i=Aspergillus),   and the true yeasts listed above.  Examples of Basidiomycota  include mushrooms, rusts, and smuts.  Representative groups of Chytridiomycota include, e.g.,  Allomyces,  Blastocladiella,   Coelomomyces,  and aquatic fungi. Representative groups of Oomycota  include, e.g., saprolegniomycetous aquatic fungi (water molds) such as Achlya. Examples of mitosporic fungi include Aspergillus,  Penicillium, Candida,   and Altemaria.     Representative groups of Zygomycota include, e.g., Rhizopus  and Mucor.<br>
In a preferred embodiment, the fungal host cell is a filamentous fungal cell.  "Filamentous fungi" includes all filamentous forms of the subdivision Eumycota  and Oomycota   (as defined by Hawksworth et al.,   1995, supra).  The filamentous fungi are characterized by a vegetative mycelium composed of chitin, cellulose, glucan, chitosan, mannan, and other complex polysaccharides.  Vegetative growth is by hyphal elongation and carbon catabolism is obligately aerobic.  In contrast, vegetative growth by yeasts such as Saccharomyces cerevisiae  is by budding of a unicellular thallus and carbon catabolism may be fermentative. In a more preferred embodiment, the filamentous fungal host cell is a cell of a species of, but not limited to, Acremonium, Aspergillus,  Fusarium,  Humicola,  Mucor,  Myceliophthora, Neurospora,  Penicillium,  Thielavia,  Tolypocladium,   and Trichoderma or a teleomorph or synonym thereof.  In an even more preferred embodiment, the filamentous fungal host cell is an Aspergillus cell.  In another still more preferred embodiment, the filamentous fxingal host cell is an Acremonium  cell.  In another even more preferred embodiment, the filamentous fungal host cell is a FusariuTzi cell.  In another even more preferred embodiment, the filamentous fungal host cell is a Humicola  cell.  In another even<br>
more preferred embodiment, the filamentous fungal host cell is a Mucor  cell.  In another even more preferred embodiment, the filamentous fungal host cell is a Myceliophthora  cell,  in another even more preferred embodiment, the filamentous f\angal host cell is a Neurospora  cell.  In another even more preferred embodiment, the filamentous fungal host cell is a Penicillium  cell.  In another even more preferred embodiment, the filamentous fungal host cell is a rhielavla cell.  In another even more preferred embodiment, the filamentous fiongal host cell is a Tolypocladium cell.  In another even more preferred embodiment, the filamentous fungal host cell is a Trichoderma  cell.  In a most preferred embodiment, the filamentous fungal host cell is an Aspergillus awamori,  Aspergillus foetidus, Aspergillus japonicus,  Aspergillus niger  or Aspergillus oryzae  cell.  In another most preferred embodiment, the filamentous fungal host cell is a f^isarlum cell of the section Discolor (also knovm as the section Fusariiom). In another preferred embodiment, the filamentous fungal parent cell is a Fusarium  strain of the section Elegans, e.g.,  Fusarium oxysporum.     In another most preferred embodiment, the filamentous fungal host cell is a Humicola insolens  or Thexrmomyces lanuginosa cell.  In another most preferred embodiment, the filamentous fungal host cell is a J?hizojnucor mlehei cell.  In another most preferred embodiment, the filamentous fungal host cell is a Myceliophthora thermophilum  cell. In another most preferred embodiment, the filamentous fungal host cell is a Neurospora crassa  cell.  In another most preferred embodiment, the filamentous fungal host cell is a Penicillium purpurogenum  cell. In another most preferred embodiment, the filamentous fungal host cell is a Thielavia terrestris  cell.  In another most preferred embodiment, the Trichoderma  cell is a Trichoderma harzianum, Trichoderma koningii,  Trichoderma longibrachiatum,  Trichoderma reesei ox Trichoderma viride  cell.<br>
Fungal cells may be transformed by a process involving protoplast formation, transformation of the protoplasts, and regeneration of the cell wall in a manner knovm per se.     Suitable procedures for transformation of Aspergillus  host cells are described in EP 238 023 and Yelton et al.,   1984, Proceedings of the National Academy of Sciences USA  81:1470-1474. A suitable method of transforming Fusarium  species is described by Malardier<br><br>
et al.,   1989, Gene  78:147-15€ or in copending US Serial No. 08/269,449.  Yeast may be transformed using the procediires described by Becker and Guarente, In  Abelson, J.N. and Simon, M.I., editors, Guide to Yeast Genetics and Molecular Biology, Methods in Enzymology, Volume 194, pp 182-187, Academic Press, Inc., New York; Ito et al.,   1983, Journal of Bacteriology  153:163; and Hinnen et al.,   1978, Proceedings of the National Academy of Sciences USA  75:1920. Mammalian cells may be transformed by direct uptake using the calcium phosphate precipitation method of Graham and Van der Eb (1978, Virology  52:546).<br>
Method of producing phospholipase<br>
The present invention provides a method of producing an isolated enzyme according to the invention, wherein a suitable host cell, which has been transformed with a DNA sequence encoding the enzyme, is cultured under conditions permitting the production of the enzyme, and the resulting enzyme is recovered from the culture.<br>
When an expression vector comprising a DNA sequence encoding the enzyme is transformed into a heterologous host cell, it is possible to enable heterologous recombinant production of the enzyme of the invention.<br>
Thereby it is possible to obtain a highly purified phospholipase composition, characterized in being free from homologous impurities.<br>
In the present invention the homologous host cell may be a strain of Fusarium oxysporum.<br>
The medium used to culture the transformed host cells may be any conventional medium suitable for growing the host cells in question. The expressed phospholipase may conveniently be secreted into the culture medium and may be recovered therefrom by well-known procedures including separating the cells from the medium by centrifugation or filtration, precipitating proteinaceous components of the medium by means of a salt such as ammonium sulphate, followed by chromatographic procedures such as ion exchange chromatography, affinity chromatography, or the like.<br><br>
Use of phospholipase<br>
Beside the use of a phospholipase in a new method of the invention for enzymatic degumming of an edible oil comprising a high amount of non-hydratable phosphorus, a number of other uses of phospholipases are known in the art.<br>
Such in the art known uses/applications of phospholipases are described below.<br>
The phospholipase of the invention can be used in any application where it is desired to hydrolyze the fatty acyl group(s) of a phospholipid or lyso-phospholipid, such as lecithin or lyso-lecithin. The phospholipase is preferably used at pH 3-10 and at 30-70°C (particularly 40-60°C) . If desired, the phospholipase may be inactivated after the reaction by subjecting it to heat treatment, e.g. at pH 7, 80°C for 1 hour or 90°C for 10 minutes.<br>
As an example, the phospholipase of the invention can be used in the preparation of dough, bread and cakes, e.g. to improve the elasticity of the bread or cake. Thus, the phospholipase can be used in a process for making bread, comprising adding the phospholipase to the ingredients of a dough, kneading the dough and baking the dough to make the bread. This can be done in analogy with US 4,567,046 (Kyowa Hakko), JP-A 60-78529 (QP Corp.), JP-A 62-111629 (QP Corp.), JP-A 63-258528 (QP Corp.) or EP 426211 (Unilever).<br>
The phospholipase of the invention can also be used to improve the filterability of an aqueous solution or slurry of carbohydrate origin by treating it with the phospholipase. This is particularly applicable to a solution or slurry containing a starch hydrolysate, especially a wheat starch hydrolysate, since this tends to be difficult to filter and to give cloudy filtrates. The treatment can be done in analogy with EP 219,269 (CPC International).<br>
Further, a phospholipase of the invention may be used for partial hydrolysis of phospholipids, preferably Lecithin, to obtain improved phospholipid emulsifiers. This application is further described in product sheets for Lecitase™ (Novo Nordisk A/S) relating to this use, and in Ullmann's Encyclopedia of Industrial Chemistry (Publisher: VCH Weinheim (1996)).<br><br>
Further, a phospholipase of the invention may be used in a process for the production of an animal feed which comprises mixing the phospholipase with feed substances and at least one phospholipid. This can be done in analogy with EP 743 017.<br>
Degxuu&amp;ing of vegetable/edible oils according to procedures known in the art<br>
According to procedures known in the art the phospholipase of the invention can be used in a process for reducing the content of phospholipid in an edible oil, comprising treating the oil with the phospholipase so as to hydrolyze a major part of the phospholipid, and separating an aqueous phase containing the hydrolyzed phospholipid from the oil. This process is applicable to the purification of any edible oil which contains phospholipid, e.g. vegetable oil such as soy bean oil, rape seed oil and sunflower oil.<br>
Prior to the enzymatic treatment, the vegetable oil is preferably pretreated to remove slime (mucilage), e.g. by wet refining. Typically, the oil will contain 50-250 ppm of phosphorus as phospholipid at the start of the treatment with phospholipase, and the process of the invention can reduce this value to below 11 ppm, more preferably below 5 ppm.<br>
The enzymatic treatment is conducted by dispersing an aqueous solution of the phospholipase, preferably as droplets with an average diameter below 10 |j,(micro)m. The amount of water is preferably 0.5-5% by weight in relation to the oil. An emulsifier may optionally be added. Mechanical agitation may be applied to maintain the emulsion.<br>
The enzymatic treatment can be conducted at any pH in the range 1.5-8. The pH may be adjusted by adding citric acid, a citrate buffer or HCl.<br>
A suitable temperature is generally 30-70°C (particularly 40-60°C) . The reaction time will typically be 0.5-12 hours (e.g. 2-6 hours), and a suitable enzyme dosage will usually be 100-5000 lU per liter of oil, particularly 200-2000 IU/1.<br>
The enzymatic treatment may be conducted batchwise, e.g. in a tank with stirring, or it may be continuous, e.g. a series of stirred tank reactors.<br><br>
The enzymatic treatment is followed by separation of an aqueous phase and an oil phase. This separation may be performed by conventional means, e.g. centrifugation.<br>
In other respects, the process can be conducted according to principles known in the art, e.g. in analogy with US 5,264,367 (Metallgesellschaft, Rohm); K. Dahlke &amp; H. Buchold, INFORM, 6 (12), 1284-91 (1995); H. Buchold, Fat Sci. Technol., 95 (8), 300-304 (1993); JP-A 2-153997 (Showa Sangyo); or EP 654,527 (Metallgesellschaft, Rohm).<br>
Use of a phospholipase of the invention in baking<br>
The phospholipase of the invention also may be used in bread-improving additives, e.g. dough compositions, dough additives, dough conditioners, pre-mixes, and similar preparations conventionally added to the flour and/or the dough during processes for making bread or other bciked products to provide improved properties of bread or other baked products.<br>
Therefore, an embodiment of the invention relates to a bread-improving and/or a dough-improving composition, and fiirther, to the use of a phospholipase of the invention in such compositions, and to a dough or baked product comprising a bread-improving and/or a dough-improving composition of the invention.<br>
In the present context the terms "bread-improving composition" and "dough-improving composition" are intended to indicate compo-sitions which, in addition to the enzyme component, may comprise other substances conventionally used in baking to improve the properties of dough and/or baked products. Examples of such components are given below.<br>
In the present context the term "improved properties" is I intended to indicate any property which may be improved by the action of a phospholipase enzyme of the invention. In particular, the use of phospholipase results in an increased volxime and an improved criamb structure and anti-staling properties of the baked product, as well as an increased strength, stability and reduced ; stickiness and thereby improved machinability of the dough. The effect on the dough has been found to be particularly good when a poor quality flour has been used. The improved machinability is of particular importance in connection with dough which is to be processed industrially.<br><br>
The improved properties are evaluated by comparison with dough and/or baked products prepared without addition of phospholipase in accordance with the present invention.<br>
The bread- and/or dough-improving composition of the invention may further comprise another enzyme. Examples of other enzymes are a cellulase, a hemicellulase, a pentosanase (useful for the partial hydrolysis of pentosans which increases the extensibility of the dough), a glucose oxidase (useful for strengthening the dough), a lipase (useful for the modification of lipids present in the dough or dough constituents so as to soften the dough), a peroxidase (useful for improving the dough consistency), a prote¬ase (useful for gluten weakening, in particular when using hard<br>
wheat flour), a peptidase and/or an amylase, e.g. a-amylase (useful for providing sugars fermentable by yeast).<br>
In addition or as an alternative to other enzyme components, the dough-improving and/or bread-improving composition may com¬prise a conventionally used baking agent, e.g. one or more of the following constituents:<br>
A milk powder (providing crust colour), gluten (to improve the<br>
gas retention power of weak f lo\irs) , an emulsif ier (to im¬prove dough extensibility and to some extent the consistency of the resulting bread), granulated fat (for dough softening and consistency of bread), an oxidant (added to strengthen the gluten structure; e.g. ascorbic acid, potassium bromate, po-tassiiim iodate or ammonium persulfate) , an amino acid (e.g. cysteine), a sugar, and salt (e.g. sodium chloride, calcium acetate, sodium sulfate or calcium sulfate serving to make the dough firmer), flour or starch.<br>
Examples of suitable emulsifiers are mono- or diglycerides, diacetyl tartaric acid esters of mono- or diglycerides, sugar esters of fatty acids, polyglycerol esters of fatty acids, lactic acid esters of monoglycerides, acetic acid esters of monogly-cerides, polyoxyethylene stearates, phospholipids and lecithin.<br>
In the present context the term "baked product" is intended to include any product prepared from dough, either of a soft or a crisp character. Examples of baked products, whether of a white, light or dark type, which may advantageously be produced by the<br><br>
present invention are bread (in particular white, whole-meal or rye bread), typically in the form of loaves or rolls, French baguette-type bread, pita bread, tacos, cakes, pan-cakes, bis¬cuits, crisp bread and the like.<br>
The dough of the invention may be of any of the types dis¬cussed above, and may be fresh or frozen.<br>
From the above disclosure it will be apparent that the dough of the invention is normally a leavened dough or a dough to be subjected to leavening. The dough may be leavened in various ways such as by adding sodium bicarbonate or the like or by adding a leaven (fermenting dough), but it is preferred to leaven the dough by adding a suitable yeast culture such as a culture of Saccharo-myces cerevisiae   (baker's yeast). Any of the commercially avail¬able S.   cereviciae  strains may be employed.<br>
In a final embodiment the invention relates to the use of a phospholipase of the invention for the preparation of pasta dough, preferably prepared from durum flour or a flour of comparable quality. The dough may be prepared by use of conventional tech¬niques and the phospholipase used in a similar dosage as that described above. The phospholipase is preferably of microbial origin, e.g. as disclosed herein. It is contemplated that when used in the preparation of pasta the phospholipase results in a strengthening of the gluten structure and thus a reduction in the dough stickiness and an increased dough strength.<br>
Use of lipase activity of an enzyme of the invention<br>
As shown in working examples herein a phospholipase of the invention may further exhibit lipase activity.<br>
Accordingly the invention further relates to use of this lipase activity in standard uses of a lipase, in particular for use in cleaning and detergent compositions. Such cleaning and detergent compositions are well described in the art and reference is made to WO 96/34946; WO 97/07202; and WO 95/30011 for further description of suitable cleaning and detergent compositions.<br>
The invention is described in further detail in the following examples which are not in any way intended to limit the scope of the invention as claimed.<br><br>
MATERIALS AMD METHODS Deposited organisms:<br>
Fusarium oxysporum  DSM No. 2672 comprises the phospholipase encoding DNA sequence of the invention.<br>
Escherichia coli  DSM 11299 containing the plasmid comprising the full length cDNA sequence, coding for the phospholipase of the invention, in the shuttle vector pYES 2.0.<br>
Other strains:<br>
Yeast strain: The Saccharomyces cerevisiae  strain used was W3124 (MATa; ura 3-52; leu 2-3, 112; his 3-D200; pep 4-1137; prcl::HIS3; prbl:: LEU2; cir+).<br>
E.   coli   strain: DHIOB (Life Technologies)<br>
Plasmids:<br>
The lispergillus  expression vector pHD414 is a derivative of the plasmid p775 (described in EP 238 023). The construction of pHD414 is further described in WO 93/11249.<br>
pYES 2.0 (Invitrogen)<br>
PA2PH10 (See example 7)<br>
General molecular biology methods<br>
I Unless otherwise mentioned the DNA manipulations and transforma¬tions were performed using standard methods of molecular biology (Sambrook et al. (1989) Molecular cloning: A laboratory manual, Cold Spring Harbor lab., Cold Spring Harbor, NY; Ausubel, F. M. et al. (eds.) "Current protocols in Molecular Biology". John<br>
i Wiley and Sons, 1995; Harwood, C. R., and Cutting, S. M. (eds.) "Molecular Biological Methods for Bacillus". John Wiley and Sons, 1990).<br>
Enzymes for DNA manipulations were used according to the specifications of the suppliers.<br><br>
Enzymes for DNA manipulations<br>
Unless otherwise mentioned all enzymes for DNA manipulations, such as e.g. restriction endonucleases, ligases etc., are obtained from New England Biolabs, Inc.<br>
Phospholipase Activity Assay based on NEFA-C test:<br>
Substrate: L-a-lysophosphatidylcholine (Sigma).<br>
Substrate: Soybean Lecithin (Sigma #P3644). Used to measure<br>
phospholipase A activity.<br>
Nefa-C test kit is from Wako Chemicals Germany.<br>
Buffer: 20 mM NaOAc pH 4.5<br>
Substrate solution: 10 mg substrate in 1 mL milli Q water and 1 mL buffer (make enough substrate solution to all samples)<br>
1.	15 Hi enzyme is added to 150 [xl  substrate solution<br>
2.	Incubation for 10 min. at 40°C<br>
3.	30 |il is transferred to 300 |il reagent 1 (from Nefa kit)<br>
I 4.	Incubation for 10 min. at 37°C<br>
5.	Addition of 600 |il reagent 2 (from Nefa-kit)<br>
6.	Incubation for 10 min. at 37°C<br>
7.	Absorption of final reaction product is measured at 550 nm, according to Nefa-kit instructions.<br>
The enzyme activity required to produce 1 |imol of fatty acid per minute of the enzyme reaction was defined as 1 unit.<br>
Expression cloning in yeast<br>
) Expression cloning in yeast was done as comprehensively<br>
described by H. Dalboege et al. (H. Dalboege et al Mol. Gen.<br>
Genet (1994) 243:253-260.; WO 93/11249; WO 94/14953), which are<br>
hereby incorporated by reference.<br>
All individual steps of Extraction of total RNA, cDNA ; synthesis, mung bean nuclease treatment, blunt-ending with T4<br>
DNA polymerase, and Construction of libraries was done according<br>
to the references mentioned above.<br><br>
Fermentation procedure of Fusarium oxysporum  DSM No. 2672 for mRNA isolation:<br>
Fusarium oxysporum  DSM No. 2672 was cultivated in YPD medium for<br>
4 days at 30°C. 10 jal Supernatant was tested for phospholipase activity in the plate assay described below.<br>
mRNA was isolated from mycelium from this culture as described in H. Dalboege et al Mol. Gen. Genet (1994) 243:253-260.; WO 93/11249; and WO 94/14953.<br>
Identification of positive yeast clones (plate assay):<br>
Identification of positive yeast clones (i.e. clones which comprise a gene encoding for phospholipase activity) was done as described below.<br>
The yeast transformants are plated on SC agar containing 2% glucose and incubated for 3 days at 30°C. A cellulose acetate filter (OE67, Schleicher &amp; Schuell) is placed on top of the cells and then transferred to the plates containing SC agar and 2% galactose with the cells on top of the filter. After 3 days<br>
I of incubation at 30°C the filter with cells is transferred to substrate plates. Positive clones are identified as colonies giving rise to a blue-green zone in the substrate plate under the colony.<br>
The substrate plates are made the following way: 2.5 g<br>
i agar (BA-30 INA Agar*, Funakoshi Co. Ltd.) is added to 137.5 ml of H2O, heated to boiling in a microwave oven. After cooling to about 60°C, 30 ml of the following mixture is added: 62.5 ml 0.4 M Tris-HCl buffer (pH 7.5) and 50 ml 3% Lipoid E80 (Lipoid GmbH, D-67065 Ludwigshafen, Germany) dissolved in 2% Triton X-100<br>
) (v/v) and 0.5 ml 2% Brilliant Green solution in H2O. The concentration of the substrate is important. If the concentration is to high it can give rise to background activity from the yeast cells and/or from filamentous fungal lipases with phospholipase side activity.<br>
s<br>
Isolation of a cDMA gene for expression in Aspergillust<br>
A phospholipase-producing yeast colony is inoculated into 20 ml YPD broth in a 50 ml glass test tube. The tube is<br><br>
shaken for 2 days at 3 0°C. The cells are harvested by centri-fugation for 10 min. at 3000 rpm.<br>
DNA is isolated according to WO 94/14953 and dissolved in 50 ml water. The DNA is transformed into E.   coll  by standard procedures. Plasmid DNA is isolated from E.   coli  using standard procedures, and analyzed by restriction enzyme analysis. The cDNA insert is excised using appropriate restriction enzymes and ligated into an Aspergillus  expression vector.<br>
Transformation of Aspergillus oryzae  or Aspergillus niger<br>
Protoplasts may be prepared as described in WO 95/02043,<br>
p. 16, line 21 - page 17, line 12, which is hereby incorporated<br>
by reference.<br>
100 jul of protoplast suspension is mixed with 5-25 jug of<br>
the appropriate DNA in 10 /il of STC (1.2 M sorbitol, 10 mM Tris-<br>
HCl, pH = 7.5, 10 mM CaCl ) . Protoplasts are mixed with p3SR2<br>
(an A. nidulans  amdS gene carrying plasmid). The mixture is left at room temperature for 25 minutes. 0.2 ml of 60% PEG 4000 (BDH 29576), 10 mM CaCl and 10 mM Tris-HCl, pH 7.5 is added and<br>
carefully mixed (twice) and finally 0.85 ml of the same solution is added and carefully mixed. The mixture is left at room temperature for 25 minutes, spun at 2500 g for 15 minutes and the pellet is resuspended in 2 ml of 1.2 M sorbitol. After one more sedimentation the protoplasts are spread on minimal plates (Cove, Biochem. Biophys. Acta 113 (1966) 51-56) containing 1.0 M sucrose, pH 7.0, 10 mM acetamide as nitrogen source and 20 mM CsCl to inhibit background growth. After incubation for 4-7 days at 37°C spores are picked and spread for single colonies. This procedure is repeated and spores of a single colony after the second reisolation are stored as a defined transformant.<br>
Test of A.  oryzae  or Aspergillus niger  transformants<br>
Each of the A.   oryzae  transformants are inoculated in 10 ml of YPM (cf. below) and propagated. After 2-5 days of incuba¬tion at 30'C, the supernatant is removed. 20 |J.l of supernatant is loaded into holes punched in a substrate plate {vide supra). After 1-24 hours, phospholipase activity appears as a blue-green zone around the hole.<br><br>
Fed batch fermentation<br>
Fed batch fermentation was performed in a medivim comprising maltodextrin as a carbon source, urea as a nitrogen source and yeast extract. The fed batch fermentation was performed by inoculating a shake flask culture of A. oryzae  host cells in question into a mediiom comprising 3.5% of the carbon source and 0.5% of the nitrogen source. After 24 hours of cultivation at pH 7.0 and 34°C the continuous supply of additional carbon and nitrogen sources was initiated. The carbon source was kept as the limiting factor and it was ensured that oxygen was present in excessive amounts. The fed batch cultivation was continued for 4 days.<br>
Isolation of the DNA sequence shown in 8EQ ID No. l<br>
The phospholipase encoding part of the DNA sequence shown in SEQ ID No. 1 coding for the phospholipase of the invention can be obtained from the deposited organism Escherichia coli  DSM 11299 by extraction of plasmid DNA by methods known in the art (Sambrook et al. (1989) Molecular cloning: A laboratory manual, Cold Spring Harbor lab., Cold Spring Harbor, NY).<br>
Media<br>
yPD: 10 g yeast extract, 20 g peptone, H2O to 900 ml. Autoclaved, 100 ml 20% glucose (sterile filtered) added.<br>
YPM: 10 g yeast extract, 20 g peptone, H2O to 900 ml. Autoclaved, 100 ml 20% maltodextrin (sterile filtered) added.<br>
10 X Basal salt: 75 g yeast nitrogen base, 113 g succinic acid, 68 g NaOH, H2O ad 1000 ml, sterile filtered.<br>
SC-URA: 100 ml 10 x Basal salt, 28 ml 20% casamino acids without vitamins, 10 ml 1% tryptophan, H2O ad 900 ml, autoclaved, 3.6 ml 5% threonine and 100 ml 20% glucose or 20% galactose added.<br><br>
SC-agar: SC-URA, 20 g/1 agar added.<br>
SC-variant agar: 20 g agar, 20 ml 10 x Basal salt, H2O ad 900 ml, aiitofclaved<br>
PEG 4000 (polyethylene glycol, molecular weight = 4,000) (BDH, England)<br>
EXAMPLES<br>
EXAMPLE 1<br>
Fermentation of Fusariuin oxysporum  phosptaolipase<br>
A culture of Fusarium oxysporum,  DSM 2672, on an agar<br>
slant was transferred to five 500 ml shaking flasks, each with 100<br>
ml of Bouillon-3 medium, and shaken at 30"*C for 1 day (200 rpm,<br>
amplitude 2.5 cm).<br>
The composition of Bouillon-3 medium was as follows:<br>
Peptone	6  g/1<br>
Trypsin digested casein	4  g/1<br>
Yeast extract	3  g/1<br>
Meat extract	1.5 g/1<br>
Glucose	1  g/1<br>
The medium was autoclaved at 121°C for 40 minutes.<br>
The culture broth of these Bullion-3 shake flasks was used<br>
as a seed culture for inoculating twenty 500 ml shake flasks, each<br>
with 200 ml PL-1 medium.<br>
The composition of the PL-1 medium was as follows:<br>
Peptone	10  g/1<br>
I        Tween®-80	12  g/1<br>
MgS04;7H20	2   g/1<br>
CaCl2;2H20	0.1 g/1<br>
pH before autoclaving	6.0<br>
i        The mediiom was autoclaved at 121 °C for 40 minutes.<br>
Each PL-1 shedce flask was inoculated with 0.5-2 ml of Bouillon-3 culture broth, and shaken at 200 rpm (amplitude 2.5 cm) at 30°C for 5 days. The culture broth from the shake flasks was pooled at harvest, totaling 3.9 1 with an enzyme yield of 53<br><br>
LU/ml. EXAMPLE 2<br>
Purification of Phospholipase<br>
Step 1) One liter fermentation supernatant was centri-fuged, and the resulting precipitate was discarded. The super¬natant was then adjusted to 0.8 M ammonium acetate by adding solid ammonium acetate.<br>
Step 2) - Hydrophobic chromatography - Toyopearl butyl 650 C matrix was purchased from Toso Hass (Rohm and Haas com¬pany, Germany). A fifty ml column was packed with the matrix. The column was washed with 50 % ethanol and subsequently with water. The column was then equilibrated with 0.8 M ammonium acetate. The fermentation supernatant adjusted with 0.8 M ammonium acetate was then applied on the column. Unbound mate¬rial was then washed with 0.8 M ammonium acetate until all the UV absorbing material (280 nm) was removed.<br>
The column was then eluted with water and subsetjuently with 50% ethanol.<br>
Phospholipase activity was determined at pH 4.5 and 40°C using NEFA kit as described in above. Fractions containing activity in water and alcohol eluate were pooled. Activity was assayed at pH 4.5 using NEFA kit assay.<br>
Fractions containing phospholipase activity were then pooled and dialyzed and concentrated using an Amicon ultrafil¬tration membrane with a cut of 10 kDa.<br>
Step 3 -Negative absorption on DEAE fast flow chromatog¬raphy .<br>
DEAE FF was bought from Pharmacia and a 50 ml column was packed with the matrix.<br>
The column was then washed as described by the manufac¬turer and equilibrated with 25 mM Tris acetate buffer pH 7.<br>
The dialysed and concentrated sample was then adjusted to pH 7 and conductance to 2 mSi. and applied on anionic ex¬changer DEAE FF column.<br>
Activity was collected as effluent. The activity does not bind to anion exchanger at pH 7.<br><br>
The effluent from the DEAE FF containing activity was concentrated and dialyzed using an Amicon membrane with acut of 10 kDa. and buffer 25 mM Sodium acetate buffer pH 6.<br>
Gelfiltration on Superdex 75.<br>
Superdex 75 prepacked column Hiload Tm 16/60 from Pharmacia was washed and equilibrated with 25 mM sodium acetate pH 6 containing 150 mM NaCl.<br>
Two ml of the concentrated effluent from anion exchanger exhibiting phospholipase activity at pH 4.5 and 40 degrees was applied on the superdex column.<br>
The activity was separated by gel filtration with a flow rate of 1 ml /minute.<br>
EXAMPLE 3<br>
Characterization of purified phospholipase obtained from<br>
Fusarium oxysporum<br>
A characterization as described below was performed on a Fusarium oxysporum  phospholipase fermented as described in example 1 and purified as described in example 2.<br>
The molecular weight of the phospholipase enzyme was determined by using 4 to 20 % SDS-PAGE precast plates from Novex Tm. The molecular weight of the protein was determined under reducing conditions as described before.<br>
For the F.  oxyspojrum  phospholipase the molecular weight was found to be 29-30 kDa under reducing conditions.<br>
The isoelectric point was determined by using Ampholine PAGE plates from Pharmacia.<br>
For the F.  oxysporum  pi of the protein was found to be around neutral pH, preferably in the range 5.8 to 6.8.<br>
Thermostability of phospholipase<br>
The thermostability of phospholipase from Fusarium oxysporum  was tested by means of DSC (Differential Scanning Calorimetry). The thermal denaturation temperature, Td, was taken as the top of the denaturation peak in thermograms (Cp vs. T) obtained after heating of enzyme solutions at a constant, programmed heating rate.<br><br>
Experimental;<br>
A DSC II from Hart Scientific (Utah, US, 1993) was used for the experiments.<br>
50 mM buffered solutions were used as solvent for the enzyme (approx. 2 mg/ml) at either pH 10 (50 mM Glycine buffer), pH 7 (50 mM HEPES buffer+10 mM EDTA) or pH 4 (50 mM Citrate buffer). Enzyme was purified according to example 2 above.<br>
750 /ul enzyme solution was transferred into standard 1 ml sealable hastelloy ampoules from Hart Scientific. Ampoules were loaded into the calorimeter and cooled to 5°C for 15 min. Thermal equilibration was carried out prior to the DSC scan. The DSC scan was performed from 5°C to 95°C at a scan rate of ap¬prox. 90 K/hr. Denaturation temperatures were determined at an accuracy of approx. +/- 2°C.<br>
Results:<br>
Table No 1; Top to denaturation peak as a function of pH<br>
pH Td(°C)<br>
4 57°C 7 62°C 10  55°C<br>
I     It should be noted that these experiments were performed in the absence of an oil matrix which may influence enzyme stabil¬ity significantly. The DSC results indicate a maximum stability near neutral pH.<br>
Assuming irreversible thermal denaturation, a relevant per-<br>
i formance temperature in an industrial application such as deg\imming of oils (US 5,264,367) is at least approx. 10 degrees lower than the Td-temperatures listed in table No 1 above.<br><br>
Aminoterminal sequence<br>
Aminoteritiinal analysis was determined by using Edman degradation<br>
with Applied Biosystem equipment (ABI 473A protein sequencer,<br>
Applied Biosytem, USA) carried out as described by the<br>
manufacturer.<br>
N-terminal sequence(s):<br>
For the F.  oxysporum  phospholipase the N-terminal sequence is:<br>
N-terminal A-V-G-V-T-T-T-D-F-S-N-F-K-F-Y-I<br>
The N-terminal amino acid "A" (Ala) is position 31 in SEQ ID NO 2. This indicates that the mature phospholipase enzyme of the invention starts at position 31 in SEQ ID No 2.<br>
Consequently, the mature sequence is from 31-346 in SEQ ID No 2.<br>
EXAMPLE 4<br>
Phospholipase A activity<br>
The phospholipase A activity was determined with Soybean Lecithin as substrate as described above (NEFA test bases assay)<br>
at pH 4.5 at 40°C.<br>
The F.   oxysporum  phospholipase showed significant phospholipase A activity at the conditions described above.<br>
EXAMPLE 5<br>
Activity toward L-a-lysophosphatidylcholine<br>
The phospholipase activity was determined with L-a-lysophosphatidylcholine as substrate as described above (NEFA test bases assay) at pH 4.5 at 40°C.<br>
The F.  oxysporum  phospholipase showed significant activity against L-a-lysophosphatidylcholine at the conditions described above.<br><br>
EXl^MPLE 6<br>
Fhospholipase activity in monolayer setup<br>
A monolayer equipment (zero-order trough, KSV5000, KSV Instruments, Finland) has been used to evaluate the activity of various enzymes toward the phospholipid DDPC (Di Dicanoyl (CIO) Phosphatidyl Choline).<br>
Experiments<br>
On a thoroughly purified surface of a buffer solution (10 mM TRIS, pH 8.0, 25°C) a monolayer of DDPC was spread from a chloroform solution. After relaxation of the monolayer (evaporation of chlorofom) the surface pressure is adjusted to 15 mN/m, corresponding to a mean molecular area of DDPC of approx. 63 A^/molec. An buffer solution (see above) containing approximately 60 /^g (micro gram) enzyme is injected through the monolayer into the subphase of the reaction compartment (cylinder with area 1520 mm^ and volume 30400 mm^) in the "zero-order trough". Enzymatic activity is manifested through the speed of a mobile barrier compressing the monolayer in order to maintain constant surface pressure as insoluble substrate molecules are hydrolysed into more water soluble reaction products. Having verified that the aqueous solubility of the reaction products (capric acid and DDPC) are considerably higher than for DDPC the number of DDPC-molecules hydrolyzed per minute by the enzyme is estimated from the mean molecular area (MMA) of DDPC.<br>
Results<br>
Table 2. Activity of enzymes towards DDPC in a monolayer setup.<br>
Enzyme	Activity (nmol/min) *)<br>
Sigma P9279 (PLA2 from bee venom, 850 U/mg)	1.9<br>
Enzyme from Fusarium oxysporum	2.7<br>
Candida antarctica  B component lipase	0<br><br>
Candida antarctica  A component lipase	0<br>
Recombinant Guinea Pig pancreatic Lipase (rGPL)       0.2<br>
Lipolase®(Novo Nordisk A/S)	
*) Calculated from reduction in monolayer area per unit time induced by the presence of enzyme.<br>
"Enzyme from F.  Oxysporum"  in table 2 is a phospholipase of the invention, purified as described in Example 2.<br>
Conclusion<br>
No phospholipase activity was detected for most of the enzymes except for lipases obtained from guinea pig lipase, which showed minor phospholipase activity.<br>
The phospholipase of the invention obtained from Fusarium oxysporum  showed surprisingly high significant phospholipase activity.<br>
Consequently, in the present invention the term "phospholipase activity", used herein in connection with a phospholipase of the invention, is defined as an activity which in the "monolayer phospholipase assay" shown above is at least 0.25 nmol/min, enzyme dose: 60 |j.g; more preferably at least 0.40 nmol/min, enzyme dose: 60 [iq;  more preferably at least 0.75 nmol/min, enzyme dose: 60 iig;  more preferably at least 1.0 nmol/min, enzyme dose: 60 (xg; more preferably at least 1.25 nmol/min, enzyme dose: 60 |ig; and even more preferably at least 1.5 nmol/min, enzyme dose: 60 |xg.<br>
The term "lipase with phospholipase side activity" is accordingly defined as a lipase with a phospholipase side activity wherein the phospholipase side activity in the "monolayer phospholipase assay" shown in Example 6 is less than the above mentioned figures specifying phospholipase activity.<br>
An example of a lipase with phospholipase side activity according to the definitions herein is the guinea pig lipase shown in table 2 above. Said guinea pig lipase has a phospholipase side activity in the "monolayer phospholipase<br>
assay" which is less than 0.25 nmol/min, enzyme dose: 60 |ig.<br><br>
EXAMPLE 7<br>
Cloning and expression of a phospholipase from Fusariuin oxysporum  DSM No. 2672<br>
Cloning and expression were performed by using the expression cloning in yeast technique as described above.<br>
itiRNA was isolated from Fusarium oxysporum,   DSM No. 2672, grown as described above including agitation to ensure sufficient aeration. Mycelia were harvested after 3-5 days' growth, immediately frozen in liquid nitrogen and stored at -80°c. A library from Fusarium oxysporum,   DSM No. 2672, consisting of approx. 9x10^ individual clones was constructed in E.   coli  as described with a vector background of 1%. Plasmid DNA from some of the pools was transformed into yeast, and 50-100 plates containing 250-400 yeast colonies were obtained from each pool.<br>
Phospholipase-positive colonies were identified and isolated on substrate plates {vide supra).   cDNA inserts were amplified directly from the yeast colonies and characterized as described in the Materials and Methods section above. The DNA sequence of the cDNA encoding the phospholipase is shown in SEQ ID No. 1 and the corresponding amino acid sequence is shown in SEQ ID No. 2. In SEQ ID No. 1 DNA nucleotides from No 23 to No. 1060 define the phospholipase encoding region. The part of the DNA sequence in SEQ ID NO 1 which is encoding the mature part of the phospholipase comprises positions 113 to 1060, which correspond to amino acid positions 31-346 in SEQ ID NO 2.<br>
The cDNA is obtainable from the plasmid in DSM 11299.<br>
Total DNA was isolated from a yeast colony and plasmid DNA was rescued by transformation of E.   coli  as described above. In order to express the phospholipase in Aspergillus,   the DNA was digested with appropriate restriction enzymes, size fractionated on gel, and a fragment corresponding to the phospholipase gene was purified. The gene was subsequently ligated to pHD414, digested with appropriate restriction enzymes, resulting in the plasmid pA2PH10.<br>
After amplification of the DNA in E.   coli  the plasmid was transformed into Aspergillus oryzae  as described above.<br><br>
Test of A.  oryzae  transformants<br>
Each of the transformants were tested for enzyme activity as described above. Some of the transformants had phospholipase activity which was significantly higher than the Aspergillus oryzae  background. This demonstrates efficient expression of the phospholipase in Aspergillus oryzae.<br>
EXAMPLE 8<br>
Recombinant expression of an F.oxysporum  phospholipase<br>
An A.oryzae  transformant comprising the Aspergillus expression vector pA2PH10 (See example 7) was fed batch fermented as described above. Purification of the recombinantly produced F. oxysporum  phospholipase was performed as described in example 2.<br>
EXAMPLE 9<br>
Characterization of a recombinantly expressed and purified phospholipase obtained from Fusarium oxysporum I     The characterization was performed on a recombinantly expressed and subsequently purified Fusarium oxysporum phospholipase (see example 8).<br>
These characterization results in respect of the recombinant F.  oxysporum  phospholipase of the invention correlated perfectly with the characterization results shown in example 3, where it was demonstrated that the recombinantly expressed and purified enzyme was the same as the non-recombinantly expressed and purified phospholipase characterized in example 3.<br>
I General assays used to characterize a recombinantly produced phospholipase obtained from F. oxysporum<br>
Phospholipase assays:<br>
Phospholipase activity (PHLU) was measured as the release of 1 free fatty acids from lecithin. 50 fil 4% L-alpha-phosphatidylcholine (plant lecithin from Avanti, USA), 4% Triton X-100, 5 mM CaCl2 in 50 mM HEPES, pH 7 was added, 50 |xl enzyme solution diluted to an appropriate concentration in 50 mM HEPES, pH 7. The samples were incubated for 10 min at 30°C and the<br><br>
reaction stopped at 95°C for 5 min prior to centrifugation (5 min at 7000 rpm). Free fatty acids were determined using the NEFA C kit from Wako Chemicals GmbH; 25 /zl reaction mixture was added to 250 /il reagent A and incubated for 10 min at 37'C. Then 500 Ml Reagent B was added and the sample was incubated again, 10 min at 37°C. The absorption at 550 nm was measured using an HP 8452A diode array spectrophotometer. Samples were run at least in duplicates. Substrate and enzyme blinds (preheated enzyme samples (10 min at 95"C) + substrate) were included. Oleic acid was used as a fatty acid standard. 1 PHLU equals the amount of enzyme capable of releasing 1 jXTdol of free fatty acid/min under these conditions.<br>
Alternatively, the assay was run at 37'»C in 20 mM citrate buffer, pH 5 (Ca^''--dependence) or 2 0 mM Britton-Robinson buffer (pH-profile/temperature-profile/stability).<br>
Phospholipase Al activity (PLAl) was measured using l-(S-<br>
decanoyl)-2-decanoyl-l-thio-sn-glycero-3-phosphocholine   (D3761<br>
Molecular Probes) as a substrate. 190 /xl substrate (100 jul D3761<br>
(2 mg/ml in ethanol) + 50 /xl 1 % Triton X-100 + 1.85 ml 50 mM<br>
I HEPES, 0.3 mM DTNB, 2 mM CaCl2, pH 7) in a 200 /il cuvette were added to 10 /xl enzyme, and the absorption at 410 nm was measured as a function of time on the HP 8452A diode array spectropho¬tometer at room temperature. Activity was calculated as the slope of the curve in the linear range. PLAl equals the amount<br>
; of enzyme capable of releasing 1 /xmol of free fatty acid (thiol)/min at these conditions.<br>
Phospholipase A2 activity (PIiA2) was measured at 40°C using 1-hexadecanoyl-2-(1-pyrenedecanoyl)-sn-glycero-3-phosphocholine (H361 Molecular Probes) . 2 ml substrate (50 jul 1% Triton X-100 +<br>
) 25 /xl 0.1% H3 61 in methanol + 10 ml 50mM HEPES, pH 7) in a 2 ml cuvette with stirring was added to 10 /xl enzyme, and the pyrene fluorescence emission was measured at 376 nm (excitation at 340 nm) as a function of time (1 sec. intervals) using the Perkin Elmer LS50 apparatus. In the Triton X-100/phospholipid micelles<br>
i the concentration of phospholipid was adjusted to have excimer formation (emits at 480 nm) . Upon cleavage the fatty acid in the 2-position containing the pyrene group is released into the aqueous phase resulting in an increase in the monomer emission. PLA2 was taken as the slope of the curve in the linear range at<br><br>
equal conditions.<br>
Lipase assays:<br>
Lipase activity (LU) was measured according to Novo Nordisk publication AF 95. The hydrolysis of tributyrin at SCC at pH 7 was followed in a pH-stat titration experiment. 1 LU equals the amount of enzyme capable of releasing 1 jumol butyric acid/min under standard conditions.<br>
Activity on olive oil (SLU) was measured as follows: 12 ml 5 mM Tris-HCl, 40 mM NaCl, 5 mM CaCl2, pH 9 was to added 2.5 ml Sigma Lipase Substrate. The pH was adjusted to pH 9 or just below prior to adding 0.5 ml lipase solution (diluted in buffer) and running a pH-stat titration assay at 30°C using the Titralab commercially available from Radiometer A/S, Copenhagen, Denmark. 1 SLU equals the amount of enzyme capable of releasing 1 /imol of free fatty acid/min at pH 9, 30 "C.<br>
Characterization of a recombinantly produced F.   oxysporum phospholipase of the invention I The assays used to characterize the enzymes mentioned below were the assays described iitanediately above.<br>
Enzymes: 1 PL from Fusarium oxysporum  having the amino acid sequence shown<br>
in SEQ NO 2.<br>
Batch F-9700989, OD280 0*83 (0.69 mg/ml) , purity &gt; 95 % (SDS-<br>
PAGE).<br>
The enzyme was recombinantly expressed and purified as described ) above.<br>
Lecitase™ Batch L546-F06 (10368 lU/ml, approx. 20 mg/ml)<br>
Lipolase® (Novo Nordisk A/S)<br>
The influence of Ca^"*" on the phospholipase activity of the ) F.   oxysporum  lipase/phospholipase was investigated. No major difference was observed no matter whether EDTA or Ca^"*" was included in the assay or not (see Table 3 below), and thus the enzyme appears to be relatively independent of Ca^"*".<br><br>
Table 3<br>
F.  oxysporum  phospholipase activity (PHLU) dependence on EDTA<br>
and CaCl2 - 2% Lecithin, 2% Triton X-100, 20 mM citrate, pH 5 at<br>
37°C.<br><br>
Relative activity^<br><br>
5 mM EDTA<br>
1.05<br><br>
1 mM EDTA<br>
1.10<br><br>
1 mM<br>
Cacl2<br><br>
2 mM CaCl2<br>
0.90<br><br>
5 mM CaCl2<br>
0.90<br><br>
10 mM CaCl2<br>
0.89<br><br>
^Relative activity is relative to the activity at 1 mM CaCl2, which is normalized to 1.<br>
The pH-profile was studied in Britton-Robinson buffer using plant lecithin as a substrate (Table 4). Although the enzyme shows an alkaline pH-profile on phospholipid with an optimum at pH 9 or higher, the activity is still sufficiently high to provide degumming of oils at low pH and performance in baking (see below for a comparison of specific activities).<br>
Table 4<br>
Fusarium oxysporum  phospholipase pH-profile 2% Lecithin, 2%<br>
Triton X-100, 20 mM BR, 37 °C.<br><br>
pH 3<br><br>
pH 4<br><br>
pH 5<br><br>
pH 6<br><br>
pH 7<br><br>
pH 8<br><br>
pH 9<br><br><br><br>
Relative activity-'<br><br>
0.08<br><br>
0.12<br><br>
0.16<br><br>
0.28<br><br>
0.52<br><br>
0.76<br><br>
1.00<br><br>
) ^  Relative activity is relative to the activity at pH 9^ which<br>
is normalized to 1.<br>
Temperature profiles for the phospholipase were obtained at pH 5; activity starts to decline at temperatures above 40°C (Table 5). This is in reasonable agreement with the temperature stability measured by pre-incubating the enzyme and then measur¬ing the residual activity (Table 6), where the enzyme is stable at temperatures up to 45°C at pH 5.<br><br>
Table 5<br>
F,  oxyspojTum  phospholipase temperature profile 2% Lecithin, 2'<br>
Triton X-100, 20 mM BR.<br><br>
	3 0  °C	40   °C	45   °C	50   °C	55   °C<br>
pH  5	0.85	1.00	0.67	0.38	0.13<br>
All data are shown as relative activity data relative to the activity at pH 5, 40 °C, which is normalized to 1.<br>
Table 6<br>
Temperature stability of F.  oxysporum  phospholipase; pre¬incubation 30 min in 20 mM BR<br><br>
	5   °C	30   °C	40   °C	45   °C	50  °C	55   °C<br>
pH  5	1.00	0.91	1.03	1.07	0.65	0.00<br>
All data are shown as residual activity data, where the activity after pre-incubation at 5°C is normalized to 1.<br>
The low stability of the enzyme is may be advantageously to register an eventual product as a process aid as the active enzyme should not be expected in the final product in either degumming of edible oils or baked products.<br>
The phospholipase obtained from Fusarium oxysporum  of the invention has both phospholipase and lipase activity.<br>
Accordingly, the activity of the enzyme on various differ¬ent lipase and phospholipase substrates was investigated and compared to the activity of the commercially available phos¬pholipase Lecitase™, and the commercially available lipase Lipolase® (Novo Nordisk A/S) .<br>
The F.   oxyspoirum  phospholipase/lipase has high activity on both tributyrin and olive oil at pH 7 and 9 (Table 7). For comparative reasons the specific activity of Lipolase® is<br>
around 5000 LU/mg. However, contrary to Lipolase®, the F. oxysporum lipase exhibits a much broader specificity with considerable phospholipase activity and also thioesterase<br>
activity (see monolayer example 6 above, showing that Lipolase® does not have a measurable phospholipase activity).<br><br>
The F. oxysporum  phospholipase/lipase of the invention has a specific activity on lecithin considerably higher than that of the phospholipase Lecitase™ (pig pancreatic PIiA2) at pH 7 (Table 7).<br>
Compared to Lecitase™ the F.  oxysporum  enzyme has a 100 fold higher activity at pH 7. The phospholipase:lipase ratio for the F.   oxysporum  enzyme is around 0.225 (1000 LU/mg / 225 PHLU/mg) under similar conditions (pH 7 and 30 °C).<br>
Table 7<br>
Activity of the F.   oxysporum   lipase/phospholipase - comparison<br>
to Lecitase™.<br><br>
Enzyme	LU/mg	PLU-'/mg	PHLU/mg	SLU/m<br>
g	PLAl/m<br>
g<br>
F.oxysporum	1000	73	225	3090	2.04<br>
Lecitase	
i ^PLU was measured like PHLU but in 20 mM citrate, pH 5 and at 3700 instead of SOmM HEPES, pH 7 at 30 °C<br>
The specificity of the F.    oxysporum   lipase/phospholipase was investigated using substrates specific for phospholipase Al; ) measuring the cleavage of the thio-ester linkage in the 1-position   of   l-(S-decanoyl)-2-decanoyl-l-thio-sn-glycero-3-phosphocholine .<br>
The enzyme clearly hydrolyses the 1-position in phosphol¬ipid (Table 7) , while Lecitase™ (pig pancreatic PLA2) as ex-i pected showed no activity on this substrate.<br>
C-terminal amino acid sequence of the Fusarium oxysporum phospholipase of the invention<br>
The N-terminal amino acid sequence of the recombinantly ) expressed mature phospholipase protein was determined as de¬scribed in example 3, and this N-terminal sequence was confirmed to be the same as determined for the non-recombinantly produced and purified enzyme (see example 3).<br><br>
MALDI-TOF mass spectrometry was carried out using a VG TofSpec mass spectrometer (Micromass, Manchester, UK) as de¬scribed in Christgau et al. Biochem. J. 319, 705-712, 1996.<br>
Bac/cgrrouncJ<br>
The N-terminal amino acid sequence of the Fusarium ox¬ysporum phospholipase, as deduced from the DNA sec[uence, fore¬sees in combination with the known N-terminal amino acid se¬quence of the mature phospholipase a protein of 315 amino acid residues (Amino acids 31-346 in SEQ ID NO 2) . The theoretical mass of this predicted protein is 33,256.8 Da.<br>
Using MALDI-TOF mass spectrometry we have previously determined the mass of the authentic lipase/phospholipase from F. oxysporum to be 28.2 kDa (data not shown), and on SDS-PAGE it was shown that the molecular weight is 29-30 kDa {vide supra).<br>
As the N-terminal amino acid sequences of the authentic and the recombinant F. oxysporum lipases are identical it is likely that the mass difference seen between the predicted mass and the experimental mass is caused by C-terminal processing. I To investigate this we have isolated the C-terminal peptide from the recombinant F. oxysporum lipase expressed in A. oryzae  and sequenced it through its C-terminus.<br>
Strategy<br>
i The average mass of the authentic lipase/phospholipase from F. oxysporum of 28.2 kDa can be used to predict the most likely C-terminal residue which turns out to be Ser303 (SEQ ID No 2) .<br>
This deduction is based on the assumption that the enzyme<br>
) is non-glycosylated. The single potential N-glycosylation site found in the sequence at Asnl63 is probably not used since a Pro-residue is found at position 164. The presence of a Pro-residue as the second residue in the consensus sequence for N-glycosylation (Asn-Xaa-Ser/Thr)  has never been reported.  In<br>
j addition, the shape of the peak in the mass spectrum does not indicate glycosylation. However, the peak is broader than usually encountered for homogeneous proteins, which indicates the possibility of size heterogeneity. As the N-terminus of the enzyme is well-defined, the size heterogeneity most likely has<br><br>
its basis in heterogeneous C-terminal processing.<br>
Inspection of SEQ ID NO 2 (see below) reveals that the predicted C-terminus is located closely to the last of the 8 Cys-residues in the sequence. Introduction of a radioactive tag on the Cys-residues makes the peptides containing Cys-residues easy to trace through peptide purification. Combining the radioactive tagging with proteolytic degradation using the Asp-N protease that cleaves in front of Asp-residues would result in a tagged C-terminal peptide. In addition, three internal peptides would be tagged. Sequencing all tagged peptides should reveal the C-terminus of the enzyme.<br>
31 AVGVTTTDFS NFKFYIQHGA AAYCNSEAAA GSKITCSNNG CPTVQGNGAT  80<br>
81 IVTSFVGSKT GIGGYVATDS ARKEIWSFR GSINIRNWLT NLDFGQEDCS 130<br>
U	(0)<br>
131 LVSGCGVHSG FQRAWNEISS QATAAVASAR KANPSFNVIS TGHSLGGAVA 180<br>
181 VLAAANLRVG GTPVDIYTYG SPRVGNAQLS AFVSNQAGGE YRVTHADDPV 230 231 PRLPPLIFGY RHTTPEFWLS GGGGDKVDYT ISDVKVCEGA ANLGCNGGTL 280<br>
281 GLDIAAHLHY FQATDACNAG GFSWRRYRSA ESVDKRATMT DAELEKKLNS 330<br>
t<br>
331 YVQMDKEYVK NNQARS	346<br>
SEP ID NO 2; Predicted amino acid sequence of the F.   oxysporum lipase/phospholipase<br>
The sequence is deduced from the DNA sequence and starts<br><br>
at the N-terminus determined experimentally for both the authen¬tic and the recombinant enzyme. The 8 Cys-residues are indicated<br>
by U while the C-terminal Ser-residue predicted from MALDI-TOF mass spectrometry of the authentic enzyme is indicated by t. The Asn-residue found in the consensus sequence for N-glycosylation (NXS/T) is shown by (0) but is in all likelihood not used as X is a Pro-residue.<br>
Experimental Results<br>
Enzyme was PL from Fusarium oxysporum  having the amino acid sequence shown in SEQ NO 2.<br>
Batch F-9700989, OD28O 0.83 (0.69 mg/ml), purity &gt; 95 % (SDS-PAGE).<br>
The enzyme was recombinantly expressed and purified as described above.<br>
The enzyme was denatured and the disulfide-bonds were reduced before the thiol-groups were reacted with I[l-^*C]CH2CONH2.<br>
Following the radioactive tagging of the Cys-residues the I lipase was degraded using the Asp-N protease.<br>
The generated peptides were fractionated using reversed phase HPLC. The collected fractions were subjected to MALDI-TOF mass spectrometry and scintillation counting. Fractions contain¬ing significant amounts of radioactivity were selected for re-I purification using reversed phase HPLC.<br>
The re-purified fractions were subjected to scintillation counting and the fractions containing radioactivity were subse¬quently sequenced.<br>
Below is given a summary of the results. This scheme may I look chaotic due to the many sequences given. However, the scheme contains all the sequence data obtained from the radioac¬tive fractions, and thus it represents the basis for the conclu¬sion drawn. It should be noted that all Cys-residues have been covered through the sequencing; most of them more than once. ) Another thing to note is the aberrant cleavages seen resulting in a large number of small radioactively tagged peptides.<br><br>
NG CPT NNG CPTVQ CSN<br>
CSNNG CP CSNNG CPTV CNSEAAA GSKI 31 AVGVTTTDFS NFKFYIQHGA AAYCNSEAAA GSKITCSNNG CPTVQGNGAT  80<br>
u<br>
DCS DCS 81 IVTSFVGSKT GIGGYVATDS ARKEIWSFR GSINIRNWLT NLDFGQEDCS 130<br>
li LVSGC<br>
LVSGCGVHSG FQRAW 131 LVSGCGVHSG- FQRAWNEISS QATAAVASAR KANPSFNVIS TGHSLGGAVA 180<br>
181 VLAAANLRVG GTPVDIYTYG SPRVGNAQLS AFVSNQAGGE YRVTHADDPV 230<br>
u	u<br>
DVKVCEG<br>
DVKVCEGA ANLGCNGGTL DVKVCEGA ANLGCNGGTL 231 PRLPPLIFGY RHTTPEFWLS GGGGDKVDYT ISDVKVCEGA ANLGCNGGTL 280<br>
I<br>
Ji<br>
DACNAG GFS<br>
GL	TDACNAG GF<br>
281 GLDIAAHLHY FQATDACNAG GFSWRRYRSA ESVDKRATMT DAELEKKLNS 330<br>
I	t<br><br>
331 YVQMDKEYVK NNQARS	346<br>
The amino acid sequences obtained by sequencing of the radioactively tagged peptides derived from recombinant F. oxysporum enzyme. The sequences are aligned to the predicted amino acid sequence as deduced from the DNA sequence. The 8 Cys-<br>
residues are indicated by U while the C-terminal Ser-residue predicted from MALDI-TOF mass spectrometry of the authentic enzyme is indicated by t.<br>
Experimental  conclusion<br>
From the sequencing of all the radioactively tagged peptides it is clear that the C-terminal part of the amino acid sequence encoded in the DNA is processed during expression of the lipase from F. oxysporum. The peptide sequences point to Ser303 as the most likely C-terminal residue in the mature enzyme in accordance with the result from MALDI-TOF mass spec¬trometry .<br>
Based on the data it cannot, however, be ruled out that I differential C-terminal processing occurs leading to heterogene¬ous C-termini; e.g. one peptide indicates that Phe272 could also be found as a C-terminal residue.<br>
EXAMPLE 10 
Equipment for carrying out enzymatic decrumminq<br>
The equipment consists of a 1 1 jacketed steel reactor fitted<br>
with a steel lid, a propeller (600 rpm), baffles, a temperature<br>
) sensor, an inlet tube at the top, a reflux condenser (4°C) at the top, and an outlet tube at the bottom. The reactor jacket is connected to a thermostat bath. The outlet tube is connected via silicone tubing to a silverson in-line mixer head equipped with a "square hole high shear screen", driven by a Silverson L4RT<br>
&gt; high shear lab mixer (8500 rpm, flow ca. 1.1 1/minute). The mixer head is fitted with a cooling coil (5-10 °C) and an outlet tube, which is connected to the inlet tube of the reactor via silicone tubing. A temperature sensor is inserted in the sili-<br><br>
cone tubing just after the mixer head. The only connection from the reactor/mixer head system to the atmosphere is through the reflux condenser.<br>
General procedure for carrying out enzymatic decnimminq All cooling and thermostat equipment is turned on. Then 0.6 1 (ca. 560 g) of oil is loaded in the reactor, which is kept at about the temperature needed for the specific experiment. The lab mixer is turned on, whereby the oil starts to circulate from the reactor to the mixer head and back to the reactor. The system is allowed to equilibrate for about 10 minutes, during which period the temperature is fine tuned. The pre-treatment period starts with addition of 0.6 g (2.86 mmol) citric acid monohydrate in 27 g MilliQ water (added water vs. oil equals 4.8% w/w; [citric acid] in water phase = 106 mM, in water/oil emulsion =4.6 mM), which sets t = 0. At t = 30 minutes a suitable amount of 4 M NaOH solution is added.<br>
0.0	equiv.	4 M NaOH	-&gt;pH 3.7<br>
1.0	equiv.	4 M NaOH	(0.714 ml)-&gt;pH 4.5<br>
1.5	equiv.	4 M NaOH	(1.07 ml) -^pH 5.0<br>
2.0	equiv.	4 M NaOH	(1.43 ml) -&gt;pH 5.5<br>
2.5	equiv.	4 M NaOH	(1.79 ml) -&gt;pH 6.2<br>
3.0	equiv.	4 M NaOH	(2.14 ml) -&gt;pH 8.0<br>
At t = 35 minutes samples are drawn for P-analysis and pH determination. Just after this the required amount of enzyme solution is added (end of pre-treatment period). Samples for P-analysis and pH determination are drawn att=l, 2, 3.5, 5, 6 hours, and then the reaction is stopped.<br>
The reactor/mixer system is emptied and rinsed with 2x500<br>
ml 10% Deconex/DI water solution followed by minimxim 3x500 ml of DI water. Table 8 is a presentation of the various additions and samplings during the reaction.<br>
Table 8. Schedule for enzymatic degumming<br>
Sampling	|<br><br>
Time	Addition of		<br><br><br>
	P-analysis X	pH determina- 1 tion     1<br><br><br>
0	Citric acid		<br>
5 min.			X<br>
30 min.		X	X<br>
30+5 min.	NaOH		<br>
35 min.		X	X<br>
35+6 min.	Enzyme		<br>
1 hour		X	X<br>
2 hours		X	X<br>
3.5 hours		X	X<br>
5 hours		X	X<br>
6 hours		X	X<br>
Phosphorus analysis: Sampling for P-analysis:<br>
Take 10 ml of water in oil emulsion in a glass centrifuge tube. Heat the emulsion in a boiling water bath for 30 minutes. Centrifuge at 5000 rpm for 10 minutes. Transfer about 8 ml of upper (oil) phase to a 12 ml polystyrene tube and leave it (to settle) for 12-24 hours. After settling draw about 1-2 g from<br>
I the upper clear phase for P-analysis.<br>
P-analysis was carried out according to procedure 2.421 in "Standard Methods for the Analysis of Oils, Fats, and Deriva¬tives, 7^^ ed. (1987)":<br>
Weigh 100 mg of MgO (leicht, Merck #5862) in a porcelain<br>
i dish and heat with a gas burner. Add 1-2 g of oil and ignite with a gas burner to give a black, hard mass. Heat in a Vecstar furnace at 850°C for 2 hours to give white ashes. Dissolve the ashes in 5 ml of 6 M HNO3 and add 20 ml of reagent mix. Leave for 20 minutes. Measure absorbance at 460 nm (use a blank (5 ml<br>
) HNO3 + 20 ml reagent mix) for zero adjustment). Calculate by using calibration curve.<br><br>
pH determination<br>
Take 2 ml of water in oil emulsion and mix with 2 ml of MilliQ water. After phase separation, pipette off top oil layer. Measure pH in aqueous phase with pH electrode Orion. Measure¬ments are transformed to "real" pH values by the formula<br>
P^real ~  P^measured ~ 0.38.<br>
A calibration curve was obtained by dissolving 0.6 g of citric acid monohydrate in 27 g of DI water; pH of this solution was measured by pH electrode Orion (pHj-eai) • 100 jul were mixed with 2 ml MilliQ water, and pH of this solution was measured by pH electrode Orion (pHmeasured) • PH of the citric acid solution was changed gradually by adding NaOH solution, and for each adjustment dilution and pH measurements were carried out as described above.)<br>
EXAMPLE 11<br>
Optimal degumming conditions for Lecitase™<br>
All experiments concerning degumming of edible oil were performed as described in example 10.<br>
Oil;<br>
Water degummed rape seed oil (Colzro) from Aarhus Oliefabrik, Denmark.<br>
Batch C00730/B01200, 9 kg, P-content 186 ppm (0.47% phosphat¬ide) .<br>
The oil is not a commercially available product, but taken directly from the production line at the mill.<br>
Enzyme:<br>
Lecitase™ lOL<br>
Batch L646-F02 (10190 U/ml), estimated cone. 20 mg/ml.<br>
The specific conditions for a series of parameter optimisa¬tion experiments with Lecitase™ are given in Table 9. Standard conditions are: enzyme dosage 535 U/kg oil (1.1 mg/kg oil), 60<br><br>
"C, 2.0 eq. NaOH (pH 5.5). The enzyme dosage has been varied from 2 68-1070 U/kg oil, the temperature has been varied from 40-70°C, and NaOH addition has been varied from 1.0-3.0 eq. corre¬sponding to the various pH levels as shown in Table 9.<br>
Table 9. Specific conditions for Lecitase'™ optimisation<br><br>
Experi-	Rape seed	Temp.	Eq. NaOH	pH level'	Enzyme<br>
ment #	oil	(»C)			dosage (U/kg oil)<br>
10	Colzro 1200	60»C	2.0	5.5	0 (blank)<br>
21	Colzro 1208	60°C	0.0	3.7	0 (blank)<br>
8	Colzro 1200	60°C	2.0	5.5	535<br>
9	Colzro 1200	60°C	2.0	5.5	535<br>
11	Colzro 1200	60°C	2.0	5.5	268<br>
12	Colzro 1200	eo^c	2.0	5.5	1070<br>
15	Colzro 1200	70»C	2.0	5.5	535<br>
17	Colzro 1200	50»C	2.0	5.5	535<br>
18	Colzro 1200	40»C	2.0	5.5	535<br>
19	Colzro 1200	60°C	1.0	4.5	535<br>
40	Colzro 1209	60°C	1.5	5.0	535<br>
44	Colzro 1429	60°C	2.5	7.0	535<br>
20	Colzro 1200	60°C	3.0	8.0	535<br>
pH from t = 35 min. - 6 hours. Within this time period all pH<br><br>
determinations were within a narrow window. This is further illustrated in example 13 below.<br>
Presentations of the separate optimisation studies are<br>
given in table 10.<br>
The results in table 10 show that,<br>
i)   from the dose/response investigation it is seen that<br>
optimal enzyme dose (at 60°C and 2.0 eq. NaOH) is about 535 U/kg oil. Half dosage increases degvimming time from about 3.5 to 6 hours, and double dosage does not bring about any change in degumming performance. The enzyme blank results have been inserted for comparison;<br>
ii)  that optimal NaOH addition is about 2.0 eq. (pH at about<br>
5.5), with bad performance at 1.0 eq. (pH at about 4.5) and 3.0 eq. (pH at about 8);<br>
iii) that optimum temperature is about ecc, since 70°C does not bring the P-level totally down, 50"C increases degumming time from about 3.5 to 6 hours, and 40°C gives bad perform¬ance.<br>
Table 10: Results of optimisation of  Lecitase™ degumming condi¬tions<br><br>
Ex #	Time"^ 0	Time"^ 0.50	Time-^ 0.58	Time"^ 1.0	Time"^ 2.0	Time-^ 3.5	Time"^ 5.0	Time-^ 6.0<br>
10	160	140	116	118	108	109	105	109<br>
21	178	149	—	143	142	143	147	154<br>
8	164	139	117	85	30	-	2	3<br>
9	164	136	109	79	14	4	3	4<br>
11	183	149	123	104	78	35	10	7<br>
12	165	131	117	71	13	3	4	3<br>
15	170	139	127	83	23	10	11	9<br>
17	162	134	127	95	56	15	11	5<br>
18	176	151	136	100	66	28	24	28<br>
19	171	139	147	142	142	118	91	80<br>
40	184	149	157	126	109	73	40	30<br><br>
44 "20"<br><br>
226 165<br><br>
202 136<br><br>
197 111<br><br>
148 102<br><br>
99<br>
"gcT<br><br>
66<br><br>
40 "73~<br><br>
34<br>
72<br><br>
•^Phosphorus content (ppm) in oil phase at indicated times in hours.<br>
E3CAMPLE 12<br>
Optimal degumming conditions for a Fusarium oxysporum  phospholi-<br>
pase according to the invention<br>
All enzymatic degumming of edible oil experiments was per¬formed as described in example 10.<br>
Oil:<br>
Water degummed rape seed oil (Colzro) from Aarhus Oliefabrik,<br>
Denmark.<br>
Batch C00730/B01208, P-content about 200 ppm<br>
Batch C00730/B01209, P-content about 200 ppm<br>
Batch C00730/B01429, P-content 227 ppm<br>
Batch C00730/B01430, P-content 252 ppm<br>
The oils are not commercially available, but taken directly from<br>
the production line at the mill. I<br>
Enzyme;<br>
PL from Fusarium oxysporum having the amino acid sequence shown<br>
in SEQ NO 2.<br>
Batch F-9700123, OD280 1'48, purity ca. 58%, estimated cone. I 0.9 mg/ml.<br>
The enzyme was recombinantly expressed and purified as described<br>
above.<br>
The specific conditions for a series of parameter optimisa¬tion experiments with PL from Fusarium oxysporum  are given in ) Table 11. Standard conditions are: enzyme dosage 1.6 mg/kg oil,<br>
40°C, 1.5 eq. NaOH (pH about 5.0). The enzyme dosage has been<br>
varied from 0.2-1.6 mg/kg oil, the temperature has been varied<br>
from 30-50°c, and the NaOH addition has been varied from 1.0-2.5<br>
eq. corresponding to the various pH levels as shown in Table 11.<br>
Table 11. Specific conditions for optimisation of PL from<br><br>
Fusarium oxysporum.<br><br>
Experiment #	Rape	Temp.	Eq.	pH	Enzyme |<br>
	seed oil	(»C)	NaOH	level	dosage<br>
(mg/kg<br>
oil)<br>
31	Colzro 1208	40°C	1.5	5.0	1.6<br>
53	Colzro 1429	40'C	1.5	5.3	1.6<br>
33	Colzro 1209	40'C	1.5	5.0	0.8<br>
35	Colzro 1209	40°C	1.5	5.0	0.4<br>
36	Colzro 1209	40»C	1.5	5.0	0.2<br>
38	Colzro 1209	50"C	1.5	5.0	1.6<br>
64	Colzro 1430	45»C	1.5	5.0	1.6<br>
39	Colzro 1209	ao'c	1.5	5.0	1.6<br>
32	Colzro 1209	40°C	1.0	3.5	1.6<br>
13	Colzro 1200	40»C	1.0	4.5	1.6<br>
45	Colzro 1429	40°C	1.25	5.0	1.6<br>
46	Colzro 1429	40°C	1.75	5.5	1.6<br>
34	Colzro 1209	40°C	2.0	5.5	1.6<br>
37	Colzro	40"&gt;C	2.5	6.2	1.6<br>
	1209				<br>
The experimental results are presented in table 12 below. The pH deviations in the time window 35 min. - 6 hours all fell<br><br>
within the expected intervals with only minor irregularities.<br>
In summary, the results in table 12 below show that, i)   from the dose/response tests it is seen that the optimal<br>
enzyme dose (at 40°C and 1.5 eq. NaOH) is about 0.8 mg/kg<br>
oil; ii)  optimal NaOH addition is about 1.5 eq. (pH about 5.0), with<br>
no performance at 1.0 eq. (pH about 4.5), and with limited<br>
performance at 2.0 eq. (pH about 5.5) and 2.5 eq. (pH about<br>
6.2); and iii) the optimum temperature is around 45*'C, and 50°C gives<br>
limited performance.<br>
Table 12: Results of Fusarium oxysporum  deg\imming conditions optimisation<br><br>
EX #	0	Titt6* 0,50	Tim*<br>
0«58	Tiifte*<br>
1*0	Tiaie*	3*S		Tiflte*<br>
«,0<br>
31	169	130	136	15	8	7	8	7<br>
53	232	203	208	32	10	7	7	4<br>
33	188	156	160	27	7	6	6	8<br>
35	181	153	153	78	5	5	4	6<br>
36	187	162	157	117	61	32	20	15<br>
38	187	149	146	84	83	68	58	55<br>
64	252	192	201	10	4	4	4	4<br>
39	184	163	158	36	7	7	9	9<br>
32	167	137	165	152	146	151	148	146<br>
13	170	140	141	140	133	126	130	131<br>
45	221	189	195	161	118	99	92	95<br>
46	225	187	163	93	4	7	6	15<br>
34	189	174	165	61	27	25	26	19<br>
37	205	168	157	88	22	23	20	21<br>
^Phosphorus content (ppm) in oil phase at indicated times in hours.<br>
) EXAMPLE 13 Illustration of standard pH deviations during an enzymatic<br><br>
degumming process<br>
The table 13 below shows an average example of pH devia¬tions during the enzymatic degumming process carried out as described in Example 10.<br>
The experiments are carried out with Lecitase™. See exam¬ple 11 for further details.<br>
Table 13: pH values from t = 35 min. - 6 hours.<br><br>
Time (hours)	pH<br>
Ex. #8<br>
(2.0 q.)<br>
4.97	PH<br>
Ex. #15<br>
(2.0 q.)<br>
5.80	PH Ex. #19 (1.0 q.)	"^         1 Ex. #20 1<br>
(3.0 q.) 1<br>
0.58	<br><br>
	4.45	7.38   1<br>
1.0	5.82	5.75	4.46	7.63   1<br>
2.0	5.50	5.44	4.57	8.13   1<br>
3.5	5.35	5.34	—	8.37   1<br>
5.0	5.25	5.47	4.47	8.21  1<br>
6.0	5.01	5.26	4.43	8.05   1<br>
If no further mentioned in examples of enzymatic degumming experiments disclosed herein, the standard pH deviations in said experiments were as shown in table 13 above.<br>
EXAMPLE 14<br>
Comparison of enzymatic degunming capacity of Lecitase™ and a pbospholipase from Fusarium oxyspoznm  according to the inven¬tion.<br>
In Figure 2 the results from the PL's under their respec¬tive optimal conditions are shown, as determined in examples 11 and 12 above.<br>
Experimental conditions shown in figure 2:<br>
Lecitase™: 60 °C, pH 5.5 (2.0 eq. NaOH), and 1 mg enzyme/kg oil (about 535 U) (exp. # 9).<br>
Fusariujn oxysporum PL: 40 °C,   pH 5.0 (1.5 eq. NaOH), and 0.8 mg enzyme/kg oil (exp. # 33).<br><br>
Fusarium oxysporum  PL: 45 °C, pH 5.0 (1.5 eq. NaOH), and 1.6 mg enzyme/kg oil (exp. # 64).<br>
Apparently, the PL from Fusarium oxysporum gives a very fast degumming effect as compared to Lecitase"™.<br>
The PL from Fusarium, according to the invention, gives an almost total degumming after about 25 minutes of contacting with the enzyme to the oil.<br>
EXAMPLE 15<br>
Determination of amount of non-hydratable phospholipids present<br>
in different types of edible oils<br>
Oils;<br>
Crude rape seed oil from Arhus Oliefabrik (AOM), Denmark. Batch<br>
C00745/B01146, P-content 609 ppm. This batch contains solid<br>
residues.<br>
Crude rape seed oil from Scanola, (Denmark) Batch C00745/B01593, P-content 315 ppm.<br>
Filtered crude rape seed oil<br>
Batch C00745/B01146 filtered, P-content 231 ppm.<br>
This oil is Batch C00745/B01146 above (609 ppm) filtered through<br>
a lOOum Johnson filter.<br>
Crude rape seed oil from Arhus Oliefabrik (AOM), Denmark. Batch C00745/B01700, P-content 459 ppm.<br>
Rape seed oil from Lurgi, Germany Batch C00932/B01381, P-content 148 ppm.<br>
Crude soybean oil from Arhus Oliefabrik, Denmark. Batch C00744/B01145, P-content 593 ppm<br>
Determination of the amount of non-hydratable phospholipids present in the different types of edible oils shown above was carried out by pre-treatment of the oils by a solution compris-<br><br>
ing citric acid monohydrate in water as described in example 10 above.<br>
Briefly, the pre-treatment process comprises, i)   pre-treating the edible oil, at 60°C, by addition of a solution comprising citric acid monohydrate in water (added water vs. oil equals 4.8% w/w; [citric acid] in water phase = 106 mM, in water/oil emulsion =4.6 mM) for 30 minutes; ii)   transferring 10 ml of the pre-treated water in oil<br>
emulsion to a tube; ill)  heating the emulsion in a boiling water bath for 30<br>
minutes; iv)   centrifuging at 5000 rpm for 10 minute, v)   transferring about 8 ml of upper (oil) phase to a new tube<br>
and leaving it to settle for 24 hours; after settling draw 2 g from upper clear phase for measurement of the non-hydratable phosphorus content (ppm) in the edible oil. The ppm value was determined as described in example 10 above.<br>
Following this process, the amount of non-hydratable phos¬pholipids present in the different types of edible oils shown above was,<br>
the crude rape seed oil #1146 from AOM contains solid particulate matter which is partly responsible for the high P-level (609 ppm) ; filtering through a 100 jum Johnson sieve gave a clear oil with a P-content of 231 ppm.<br>
Pre-treatment of the crude oil and of the filtered oil gave a P-level of 140 ppm, which is a measure of the non-hydratable phospholipids present in the oil;<br>
the phospholipid content of a crude rape seed oil from Scanola was reduced from 315 ppm to about 30 ppm by the pre-treatment ;<br>
the phospholipid content of a rape seed oil obtained from Lurgi (probably arbitrary mix of crude oil and totally refined oil) was reduced to 60 ppm by the pre-treatment process;<br>
pretreatment of crude rape seed oil #1710 from AOM reduced the P-content from 459 to 200-250 ppm;<br>
with crude soybean oil #1145 from AOM pre-treatment reduced<br><br>
the P-level from 593 to 10 ppm. This soybean oil constitutes an example of oils that can be degummed by water-degmnming/citrate treatment alone. Enzyme addition to this crude soybean oil after pre-treatment did not reduce the P-content further.<br>
These data show that the phospholipid composition (hydratable vs. non-hydratable phospholipid) of crude rape seed oil varies greatly from one batch to another, and consequently the level of remaining phospholipid in water-degximmed rape seed oil will vary over a wide range (30 ppm (Scanola) to 200-250 ppm (AOM)) .<br>
For enzymatic degumming, optimal enzyme dosage is dependent on the amount of non-hydratable phospholipid present after degumming or pre-treatment.<br>
Further, as higher the amount of non-hydratable phosphol¬ipid present the oil is, the more useful is the enzymatic degumming method.<br>
This is too illustrated in example 16 below, where the pre¬sent invention show enzymatic degumming of the crude rape seed oil # 1146, having a non-hydratable phospholipids level of around 140 ppm.<br>
EXAMPLE 16<br>
Degumming of crude rape seed edible oil (I)<br>
Experiments A and B were carried out according to the "General procedure for carrying out enzymatic degumming" as described in example 10 above.<br>
Oil;<br>
Crude rape seed oil from Arhus Oliefabrik (AOM), Denmark. Batch<br>
C00745/B01146, P-content 609 ppm. This batch contains solid<br>
residues.<br>
Enzvme:<br>
Lecitase™ lOL<br>
Batch L646-F02 (10190 U/ml), estimated cone. 20 mg/ml<br>
PL from Fusarium oxysporum  having the amino acid sequence shown<br><br>
in ,7EQ NO 2.<br>
Batch F-9700123, OD28O 1*48, purity ca. 58%, estimated cone.<br>
0.9 mg/ml.<br>
The enzyme was recombinantly expressed and purified as described<br>
above.<br>
Experiment A (reference)<br>
0.6 1 (580 g) of crude rape seed oil is loaded in the equipment and heated to 60°C. At t = 30 min, 1.43 ml (5.7 mmoles) of 4 M NaOH solution are added, which yield a pH of about 5.6. At t = 35 min. , 30 fj.1   (300 Unit) of Lecitase lOL (obtained from Novo Nordisk A/S) are added. The measured phosphorus content in the oil phase after centrifuging as well as the pH values in the aqueous phase is shown in Table 14.<br>
Table 14. Results from degumming of crude rape seed oil with Lecitase™.<br><br>
Time   (hours)	Phosphorus content in oil phase	^^^j^rwrwnTTii TnTrii.rii|i<br>
0	609	<br>
0.50	155	4.8<br>
0.58	146	5.6<br>
1.0	127	5.6<br>
2.0	88	5.7<br>
3.5	61	5.7<br>
5.0	44	5.6<br>
6.0	34	5.8<br>
Experiment B<br>
0.6 1 (581 g) of crude rape seed oil is loaded in the equipment and heated to 40°C. At t = 30 min. 1.07 ml (4.3 mmoles) of 4 M NaOH solution were added, which yield a pH of about 5.4. At t = 35 min., 1 ml (0.9 mg) of a purified solution (example 2) of phospholipase from F.   oxysporum  is added. The measured phospho¬rus content in the oil phase after centrifugation as well as the<br><br>
pH values in the aqueous phase is shown in Table 15.<br>
Table 15. Results from deg\imming of crude rape seed oil with phospholipase from F.  oxysporum.<br><br>
Time (hours)	Phosphorus content in oil phase	PH<br>
0	609	<br>
0.50	155	4.9<br>
0.58	149	5.4<br>
1.0	91	5.3<br>
2.0	13	5.4<br>
3.5	11	5.3<br>
5.0	13	5.4      1<br>
6.0	10	5.2<br>
EXAMPLE 17<br>
Degumming of crude rape seed edible oil (II)<br>
Experiments A and B were carried out according to the "General procedure for carrying out enzymatic degumming" as described in example 10 above.<br>
Oil:<br>
Crude rape seed oil from Arhus Oliefabrik (AOM), Denmark.<br>
Batch C00745/B01710, P-content 459 ppm.<br>
Enzyme;<br>
Lecitase™ lOL<br>
Batch L646-F02 (10190 U/ml), estimated cone. 20 mg/ml<br>
PL from Fusarxum  oxysporum having the amino acid sequence shown<br>
in SEQ NO 2.<br>
Batch F-9700476, OD280 0.8, purity ca. 58%, estimated cone.<br>
0.45 mg/ml.<br>
The enzyme was recombinantly expressed and purified as described<br>
above.<br><br>
Experiment A<br>
0.6 1 (580 g) of crude rape seed oil is loaded in the equipment and heated at eCC. At t = 30 min. 1.43 ml (5.7 mmoles) of 4 M NaOH solution are added, which yield a pH of about 5.6. At t = 35 min. an appropriate amount (e.g. 50 /xl (around 500 Unit) for 1 mg enzyme/kg oil) of Lecitase lOL (obtained from Novo Nordisk A/S) is added. The measured phosphorus content in the oil phase after centrifuging is shown in Table 16.<br>
Table 16. Results from degumming of crude rape seed oil with Lecitase.<br><br>
Time (hours)	1 mg Lecitase/ kg oil P(ppm)	2 mg Lecitase/ kg oil P(ppm)	3 mg Lecitase/ kg oil P(ppm)<br>
0	459	459	459<br>
0.50	251	235	248<br>
0.58	202	194	202<br>
1.0	181	186	183<br>
2.0	165	156	107<br>
3.5	111	66	11<br>
5.0	52	12	12<br>
6.0	20	5	9<br>
Experiment B<br>
0.6 1 (581 g) of crude rape seed oil is loaded in the equipment and heated to 40oc. At t = 30 min, 1.07 ml (4.3 mmoles) of 4 M NaOH solution are added, which yield a pH of about 5.0. At t = 35 min, an appropriate amount (i.e. 1.6 mg enzyme/kg oil, and 3.2 mg enzyme/kg oil) of a purified solution of phospholipase from F. oxysporum  is added. The measured phosphorus content in the oil phase after centrifugation is given in Table 17.<br>
Table 17. Results from degumming of crude rape seed oil with phospholipase from F.   oxysporum.<br><br>
Time	1.6 mg	3.2 mg<br>
(hours)	Fusarium/	Fusarium/<br>
	kg oil	kg oil<br>
	P (ppm)	P(ppm)<br>
0	459	459<br>
0.50	236	208<br>
0.58	193	173<br>
1.0	109	96<br>
2.0	9	7<br>
3.5	9	8<br>
5.0	9	9<br>
6.0	9	9<br>
In summary the results show, Lecitase. 60°C. pH 5.5<br>
The enzyme dosage was varied from 1.0 to 3.0 mg/kg oil. The results are given in Table 16 above. At an enzyme dosage of 1.0 mg/kg oil degumming was slow and gave about 20 ppm at 6 hours. With the high enzyme dosages degumming performance was improved to give 10 ppm phosphorus content at about 3.5 hours with 3.0 mg enzyme/kg oil.<br>
It is assumed that performance will improve further if higher enzyme dosages are employed.<br>
F.  oxysporum  PL. 45°C. pH 5.0<br>
Enzyme dosages 1.6 and 3.2 mg/kg oil were tested, and the performance was found to be equally good (Table 17 above). With 1.6 mg enzyme/kg oil - or possibly lower - excellent degumming was observed giving 9 ppm P at about 2 hours. It is foreseen that it is possible to use even lower amounts of the F.   ox¬ysporum  phospholipase (e.g. 0.9 mg/kg oil) and still have good degumming performance.<br>
EXAMPLE 18<br>
Degunming of water degunmed edible oil using an phospholipase<br>
preparation obtained from Fusarium culmorum<br><br>
An experiment was carried out according to the "General procedure for carrying out enzymatic degumming" as described in example 10 above.<br>
Oil;<br>
Water-degummed rape seed oil from Arhus Oliefabrik (AOM),<br>
Denmark.<br>
Batch C00730/B01700, P-content 231 ppm.<br>
Enzyme;<br>
A fermentation broth from Fusarium culmorum.<br>
A Fusarium culmorum  strain was cultivated, centrifuged, and the<br>
supernatant was purified as described below.<br>
Seed cultures of the strain Fusarium culmorum  CBS 513.94<br>
(deposit date 25 October 1994) were produced in 500 ml shakeflasks<br>
containing 100 ml of the following composition ;<br>
Corn steep liquer (dried)	12 g/1<br>
Glucose	24 g/1<br>
To each flask 0.5 g CaCOs and 0.5 ml of oil are added. pH is adjusted to 5.5 before autoclavation.<br>
After 3 days at 26 C and 250 rpm, 5 ml of each of the seed cultvires were inoculated in shakeflasks containing 100ml of the following medium;<br>
Peptone, Difco 0118	6 g/1<br>
Pepticase, Sheffield Products	4 g/1<br>
Yeast extract, Difco 0127	3 g/1<br>
Meat extract, Difco 0126	1.5 g/1<br>
Dextrose, Rocpiette 101-0441	1 g/1<br>
Olive oil, Sigma	10 g/1<br>
pH is adjusted to 7.3-7.4 before autoclavation.<br>
Cultivation took place for 9 days at 26 C and 250 rpm. The broths were centrifuged and filtrated (0.45 \xm) the supernatants collected, and applied for the degiomming experiment shown below.<br><br>
Estimated activity 200 PHLU/ml.<br>
Experiment; Enzymatic deqummina of a water decrummed oil using a phospholipase preparation obtained from Fusarium culmorum<br>
0.6 1 (581 g) of crude rape seed oil is loaded in the equipment and heated to 40«'C. At t = 30 min, 1.43 ml (5.7 mmoles) of 4 M NaOH solution are added, which yield a pH of about 5.5. At t = 35 min., an appropriate amount (i.e. 1070 PHLU/kg oil) of a purified solution of phospholipase from F.   culmorxim  is added. The measured phosphorus content in the oil phase after centrifu-gation is shown in Table 18.<br>
Table 18. Results from degumming of crude rape seed oil with phospholipase from F.   culmorum.<br><br>
Time (hours)	1070 U<br>
F.culmorum/<br>
kg oil<br>
P(ppm)<br>
0	254<br>
0.50	"<br>
0.58	213<br>
1.0	137<br>
2.0	61<br>
3.5	9<br>
5.0	8<br>
6.0	7<br>
EXAMPLE 19 ) Enzymatic degumming of crude oil using Degomma VOD<br>
oil<br>
Crude rape seed oil C00745/B01700, P-content 459 ppm Enzyme 5 A commercial available phospholipase Degomma VOD (Rohm; Ger¬many) , est. cone. 10 mg/ml.<br><br>
0.6 1 (581 g) of crude rape seed oil is loaded in the equipment and heated to 50°C. At t = 30 min, 0.714 ml (2.86 mmoles) of 4 M NaOH solution is added, which yields a pH of about 4.5. At t = 35 min, an appropriate amount (i.e. 3.6 mg/kg oil, or 7.1 mg/kg oil) of a purified solution of Degomma VOD phospholipase is added. The measured phosphorus content in the oil phase after centrifugation is shown in Table 19.<br>
Table 19<br><br>
Time	3.6 mg/kg oil 276	7.1 mg/kg oil<br>
0	<br>
	273<br>
0.50	216	253<br>
0.58	210	246<br>
1.0	127	94<br>
2.0	45	16<br>
3.5	15	7<br>
5.0	15	10<br>
6.0	14	10<br>
This example illustrates that Degomma VOD is able to degum an edible oil. However, in order to obtain a satisfactory degumming of said oil, relatively high doses of Degomma VOD are required as compared to the Fusarium  phospholipase of the invention. See e.g. examples 16 and 17 for a comparison.<br>
EXJ^PLE 20<br>
Use of a phospholipase obtained from F. oxysporum  as a bread I improving agent.<br>
Materials and methods<br>
Preparation of bread<br>
i European straight dough white bread and rolls were prepared from the following basic recipe:<br>
BASIC RECIPE<br><br>
Flour (Meneba BBZ)<br>
Water<br>
Yeast<br>
Salt<br>
Sugar<br>
Ascorbic acid<br><br>
100 % (2000g) 61 % 4 % 1.5 % 1.5 % 40 ppm<br><br>
BAKING PROCEDURE<br>
Mixing (spiral mixer), 625 RPM	3 min<br>
Mixing (spiral mixer), 1250 RPM	3.5 min<br>
Evaluation of dough	7 min<br>
Fermentation (room temperature)	15 min<br>
Sheeting / moulding	3 min<br>
Relaxation at room temperature	5 min<br>
Folding	2 min<br>
Relaxation at room temperature	5 min<br>
Sheeting / moulding / panning	2 min<br>
Proofing (32°C, 82% RH)      Rolls:	45 min<br>
Panned bread:  55 min<br>
Baking (230°C)	Rolls:	22 min<br>
Panned bread:  3 5 min<br>
Evaluation of dough and baked products<br>
Properties of the dough and baked products were deter¬mined as follows:<br>
Specific volume index: the volume of a loaf or roll is measured by means the traditional rape seed displacement method. I The specific volume is calculated as volume ml per g bread. The specific volume of the control (without enzyme) is defined as 100. The relative specific volume index is calculated as:<br><br>
specific volume of loaf<br>
Specific vol. index = 	<br>
specific volume of control loaf<br><br>
* 100<br><br>
The dough stickiness is evaluated manually according to the following scale:<br><br>
Roll Standing<br><br><br>
very flat	1<br>
flat	2<br>
normal	3<br>
good/round	4<br>
very good	5<br>
too round	6<br><br>
RESULTS<br>
Table 20<br><br>
Enzyme/additive								<br>
Lecimulthin 100*<br>
(g/kg flour)					1	1	1	1<br>
F.o. Phospholipase<br>
(LU/kg flour)		500	1500	3000		500	1500	3000<br>
Specific volume index (rolls)	100	110	106	93	99	111	116	108<br>
Specific volume index (panned bread)	100	106	99	94	102	107	109	103<br>
Roll standing (score)	3	4	4	3	3	4	5	4.5<br>
* Commercial lecithin preparation for baking (Superfos, Den¬mark) .<br>
The results show a clear volume increasing effect of the Fusarium oxysporum  phospholipase on both rolls and panned bread, ) in the recipe not containing lecithin. If lecithin is included in the recipe, even better volume effects are obtained, although<br><br>
the lecithin does not contribute to volume itself. A statistical analysis (ANOVA, a=0.05), performed in Statgraphics Plus, release 3.0, shows a significant positive synergy between the phospholipase and the lecithin.<br>
Both with and without lecithin in the recipe, a signifi¬cantly improved shape of rolls (roll standing) is obtained with the F.   oxyspojrum  phospholipase. In this example, the best roll standing was obtained by combining lecithin and phospholipase (1500 LU/kg flour).<br>
EXAMPLE 21<br>
Use Of a phospholipase obtained from F.  oxysporum  as an anti-<br>
staling agent in bread<br>
Materials and methods<br>
Preparation of bread<br>
European straight dough white bread and rolls were prepared from the following basic recipe:<br><br>
BASIC RECIPE<br>
Flour (Meneba BBZ)<br>
Water<br>
Yeast<br>
Salt<br>
Sugar<br>
Ascorbic acid<br><br>
100 % (2000g)<br>
61 %<br>
5 %<br>
1.5 %<br>
1.5 %<br>
40 ppm<br><br><br><br>
BAKING PROCEDURE<br>
Mixing (spiral mixer), 625 RPM Mixing (spiral mixer), 1250 RPM Evaluation of dough Fermentation (room temperature) Sheeting / moulding Relaxation at room temperature Folding Relaxation at room temperature<br><br>
3 min 3.5 min 7 min 15 min 3 min 5 min 2 min 5 min<br><br>
Sheeting / moulding / panning Proofing (32°C, 82% RH) Baking (230°C)<br><br>
2	min<br>
55 min<br>
3	5 min<br><br>
In this example, the loaves were panned in lidded pans, in order to avoid differences in the specific volxomes before texture analysis. After cooling, the loaves were stored at room tempera¬ture, packed in plastic bags.<br>
Evaluation of baked products<br>
Evaluation of staleness and texture may be performed according to AACC method 74-09. Evaluation of softness of bread crumb as indicators of bread staling were conducted 0, 1, 3, and 7 days after baking, according to the following procedure:<br>
A slice of bread was compressed at constant speed in a texture analyzer (TA TX-2), and the force for compression was measured in g. The firmness of the crumb is measured as the force at 25% compression. The firmness of a bread crumb increases as the bread becomes stale.<br>
RESULTS<br>
The results of firmness measurements as a function of storage days are shown in table 2. Lecimulthin was added in a concentra¬tion of 1 g/kg flour, and the Fusarium oxysporum  phospholipase was added in a dosage of 500 U/kg flour. Every figure in the table is the mean value of 6 measurements (2 loaves, 3 measure¬ments on each).<br>
Table 21<br><br>
Enzyme/additive	Firmness Day 0	Firmness Day 1	Firmness Day 3	Firmness Day 7<br>
Control	223	350	631	1061<br>
Lecimulthin 100*	225	261	532	1010<br><br>
Phospholipase	201	303	573	1257<br>
Lecimulthin 100* + Phospholipase	169	304	468	834<br>
* Commercial lecithin preparation for baking (Superfos, Den¬mark) .<br>
As shown in table 21, the breads treated with phospholipase were slightly softer than the control up to 3 days of storage. In combination with lecithin, a significant anti-staling effect could be obtained through the whole storage (not obtainable with lecithin or phospholipase alone).<br><br><br>
WE CLAIM:<br>
1.	A baking composition comprising a polypeptide exhibiting phospholipase A<br>
activity selected from the group consisting of:<br>
a)	a polypeptide encoded by the phospholipase A-encoding part of the DNA sequence cloned into plasmid pYES 2.0 present in Escherichia coli DSM 11299;<br>
b)	a polypeptide having an amino acid sequence as shown in positions 31-346 ofSEQIDN0 2;<br>
c)	a polypeptide having an amino acid sequence as shown in position 31-303 ofSEQIDN0 2;and<br>
d)	a polypeptide which is at least 70% homologous with said polypeptide<br>
defined in (b) or (c),<br>
and an emulsifier or flour.<br>
2.	The  composition  as  claimed  in Claim   1,  wherein the  polypeptide  is  a phospholipase Al.<br>
3.	A polypeptide exhibiting phospholipase A activity selected from the group consisting of:<br><br>
a)	a polypeptide encoded by the phospholipase A-encoding part of the DNA sequence cloned into plasmid pYES 2.0 present in Escherichia coli DSM 11299;<br>
b)	a polypeptide having an amino acid sequence as shown in positions 31-346 ofSEQIDN0 2;<br>
c)	a polypeptide having an amino acid sequence as shown in position 31-303 of SEQ ID NO 2; and<br>
d)	a polypeptide which is at least 70% homologous with said polypeptide<br>
defined in (a), (b) or (c)<br><br>
in a process for making a baked product, comprising adding the polypeptide to a dough, and baking the dough to make the baked product.<br>
4.        The  polypeptide  as  claimed  in  Claim  3,  wherein  the  polypeptide   is  a phospholipase Al.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyNC1tYXMtMTk5NyBhYnN0cmFjdCBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">2824-mas-1997 abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyNC1tYXMtMTk5NyBhYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">2824-mas-1997 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyNC1tYXMtMTk5NyBhc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">2824-mas-1997 assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyNC1tYXMtMTk5NyBjbGFpbXMgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">2824-mas-1997 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyNC1tYXMtMTk5NyBjbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">2824-mas-1997 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyNC1tYXMtMTk5NyBjb3JyZXNwb25kZW5jZSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">2824-mas-1997 correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyNC1tYXMtMTk5NyBjb3JyZXNwb25kZW5jZSBwby5wZGY=" target="_blank" style="word-wrap:break-word;">2824-mas-1997 correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyNC1tYXMtMTk5NyBkZXNjcmlwdGlvbiAoY29tcGxldGUpIGR1cGxpY2F0ZS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">2824-mas-1997 description (complete) duplicate-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyNC1tYXMtMTk5NyBkZXNjcmlwdGlvbiAoY29tcGxldGUpIGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">2824-mas-1997 description (complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyNC1tYXMtMTk5NyBkZXNjcmlwdGlvbiAoY29tcGxldGUpLTEucGRm" target="_blank" style="word-wrap:break-word;">2824-mas-1997 description (complete)-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyNC1tYXMtMTk5NyBkZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">2824-mas-1997 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyNC1tYXMtMTk5NyBkcmF3aW5ncyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">2824-mas-1997 drawings duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyNC1tYXMtMTk5NyBkcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">2824-mas-1997 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyNC1tYXMtMTk5NyBmb3JtLTE5LnBkZg==" target="_blank" style="word-wrap:break-word;">2824-mas-1997 form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyNC1tYXMtMTk5NyBmb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">2824-mas-1997 form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyNC1tYXMtMTk5NyBmb3JtLTI2LnBkZg==" target="_blank" style="word-wrap:break-word;">2824-mas-1997 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyNC1tYXMtMTk5NyBmb3JtLTQucGRm" target="_blank" style="word-wrap:break-word;">2824-mas-1997 form-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyNC1tYXMtMTk5NyBmb3JtLTYucGRm" target="_blank" style="word-wrap:break-word;">2824-mas-1997 form-6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyNC1tYXMtMTk5NyBwY3Qgc2VhcmNoIHJlcG9ydC5wZGY=" target="_blank" style="word-wrap:break-word;">2824-mas-1997 pct search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjgyNC1tYXMtMTk5NyBwZXRpdGlvbi5wZGY=" target="_blank" style="word-wrap:break-word;">2824-mas-1997 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="226997-a-method-for-integrity-checking-of-messages.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="226999-a-herbal-antimicrobial-dermatological-drug.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>226998</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2824/MAS/1997</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>07/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>13-Feb-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>31-Dec-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>09-Dec-1997</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVOZYMES A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>KROGSHOEJVEJ 36, DK-2880 BAGSVAERD,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>CLAUSEN IB GROTH</td>
											<td>NOVO ALLE, DK-2880 BAGSVAERD,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>BORCH KIM</td>
											<td>NOVO ALLE, DK-2880 BAGSVAERD,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>FUNGLSANG CLAUS CRONE</td>
											<td>NOVO ALLE, DK-2880 BAGSVAERD,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>HALKIER TORBEN</td>
											<td>HESTKOBVEJ 11E, DK-3460 BIRKEROD,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>PATKAR SHAMKANT ANANT</td>
											<td>NOVO ALLE, DK-2880 BAGSVAERD,</td>
										</tr>
										<tr>
											<td>6</td>
											<td>BARFOED MARTIN</td>
											<td>NOVO ALLE, DK-2880 BAGSVAERD,</td>
										</tr>
										<tr>
											<td>7</td>
											<td>CLAUSEN KIM</td>
											<td>NOVO ALLE, DK-2880 BAGSVAERD,</td>
										</tr>
										<tr>
											<td>8</td>
											<td>DYBDAL LONE</td>
											<td>NOVO ALLE, DK-2880 BAGSVAERD,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A21D/36</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>1432/96</td>
									<td>1996-12-16</td>
								    <td>Denmark</td>
								</tr>
								<tr>
									<td>2</td>
									<td>1283/97</td>
									<td>1997-11-11</td>
								    <td>Denmark</td>
								</tr>
								<tr>
									<td>3</td>
									<td>1408/96</td>
									<td>1996-12-09</td>
								    <td>Denmark</td>
								</tr>
								<tr>
									<td>4</td>
									<td>0211/97</td>
									<td>1997-02-26</td>
								    <td>Denmark</td>
								</tr>
								<tr>
									<td>5</td>
									<td>0190/97</td>
									<td>1997-02-21</td>
								    <td>Denmark</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/226998-a-baking-composition-comprising-a-polypeptide-exhibiting-phospholipase-a-activity by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:54:15 GMT -->
</html>
