<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/227001-method-for-producing-nucleoside-5-phosphate-ester by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:54:50 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 227001:METHOD FOR PRODUCING NUCLEOSIDE-5&#x27;-PHOSPHATE ESTER</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHOD FOR PRODUCING NUCLEOSIDE-5&#x27;-PHOSPHATE ESTER</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Abstract METHOD FOR PRODUCING NUCLEOSIDE- 5&#x27;- PHCSPHATE ESTER Nucleoside-5&#x27;-phosphate ester is produced inexpensively and efficiently by allowing an acid phosphatase, especially an acid phosphatase having a lowered phosphomonoesterase activity to act under a condition of pH 3.0 to 5.5 on a nucleoside and a phosphate group donor selected from the group consisting of polyphosphoric acid or a salt thereof, phenylphosphoric acid or a salt thereof, and carbamyl phosphate or a salt thereof to produce nucleoside-5&#x27;-phosphate ester, and collecting it.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
METHOD FOR PRODUCING NUCL-EQS-I-OE-5 '-PHOSPHATE" ESTER<br>
The present invention relates to a method for producing nucleoside-5'-phosphate ester.  The present invention also relates to a novel acid phosphatase, a gene coding for the acid phosphatase, a recombinant DNA containing the gene, and a microorganism harboring the recombinant DNA which are useful to produce nucleoside¬s' -phosphate ester.  Nucleoside-5'-phosphate ester is useful as a seasoning, a pharmaceutical, and a row material for producing such substances.<br>
Background Art<br>
Methods for biochemically phosphorylating nucleoside to produce nucleoside-5'-phosphate ester by using the following phosphate group donors are known, including a method which uses p-nitrophenyphosphoric acid (Japanese Patent Publication No. 39-29858), a method which uses inorganic phosphoric acid (Japanese Patent Publication No. 42-1186), a method which uses polyphosphoric acid (Japanese Patent Laid-open No. 53-56390), a method which uses acetylphosphoric acid (Japanese Patent Laid-open No. 56-82098), and a method which uses adenosine triphosphate (ATP) (Japanese Patent<br><br>
Laid-open No. 63-230094).  However, these methods have not been satisfactory to produce nucleoside-5'-phosphate ester efficiently and inexpensively because the substrates to be used are expensive, or because by¬products are produced in the reaction.<br>
Thus the present inventors have developed a method for efficiently producing nucleoside-5'-phosphate ester without by-producing 2'-, 3'-nucleotide isomers by allowing cells of a specified microorganism to act under an acidic condition on a nucleoside and a phosphate group donor selected from the group consisting of polyphosphoric acid or a salt thereof, phenylphosphoric acid or a salt thereof, and carbamyl phosphate or a salt thereof (Japanese Patent Laid-open No. 7-231793).<br>
However, even this method has had the following drawbacks.  Namely, for example, a part of the substrate is degraded during the reaction due to a nucleoside-degrading activity which unfortunately exists in a slight amount in the cells of the microorganism to be used.  Moreover, if the reaction is continued, produced and accumulated nucleoside-5'-phosphate ester is degraded.  Therefore, by-products are produced in a reaction solution, and it has been impossible to obtain a sufficient yield.  In addition, the reaction cannot be performed if the substrate is added at a high concentration because of a low transphosphorylation activity oer microbial cell.<br><br>
Disclosure of the Invention<br>
An object of the present invention is to provide a method for inexpensively and efficiently producing nucleoside-5'-phosphate ester.  Another object of the present invention is to provide an enzyme, a gene coding for the enzyme, a recombinant DNA containing the gene, and a microorganism harboring the recombinant DNA which are useful for the method for producing nucleoside-5'-phosphate ester.<br>
As a result of various investigations made by the present inventors in order to develop a method for producing nucleoside-5'-phosphate ester which is more efficient than the conventional methods, it has .been found out that nucleoside-5'-phosphate ester can be efficiently produced at a high yield by allowing an acid phosphatase purified from a cell-free extract of a microorganism to act under a condition of pH 3.0 to 5.5 on a nucleoside and a phosphate group donor selected from the group consisting of polyphosphoric acid or a salt thereof, phenylphosphoric acid or a salt thereof, and carbamyl phosphate or a salt thereof.  Further, the present inventors have succeeded in obtaining wild type genes coding for acid phosphatases from various bacteria and genes coding for acid phosphatases having lowered phosphomonoesteraseT activities from bacterium belonging to the genus Morqanella and bacterium belonging to the<br><br>
genus Escherichia.  Moreover, the present inventors have found out that productivity of nucleoside-5'-phosphate ester is remarkably improved by expressing the gene in a large amount in accordance with genetic engineering techniques.  Thus the present invention has been completed.<br>
Namely, the present invention provides a method for producing nucleoside-5'-phosphate ester comprising the steps of allowing an acid phosphatase, preferably an acid phosphatase having a lowered phosphomonoesterase activity to act under a condition of pH 3.0 to 5.5 on a nucleoside and a phosphate group donor selected from the group consisting of polyphosphoric acid or a salt thereof, phenylphosphoric acid or a salt thereof, and carbamyl phosphate or a salt thereof to produce nucleoside-5'-phosphate ester, and collecting it.<br>
In another aspect, the present invention provides mutant acid phosphatases having a lowered phosphomonoesterase activity, genes coding for the acid phosphatases, recombinant DNAs containing the genes, and microorganisms harboring the recombinant DNA.<br>
In still another aspect, the present invention provides novel acid phosphatases derived from bacteria belonging to the genus Escherichia, Enterobacter, Klebsiella or Serratia, genes coding for the acid phosphatases, recombinant DNAs containing the genes, and microorganisms harboring the recombinant DNA.<br><br>
The present invention will be explained in detail below.<br>  Preparation of acid phosphatase<br>
The acid phosphatase to be used in the present invention is not specifically limited provided that it catalyzes the reaction to produce nucleoside-5'-phosphate ester by phosphate group transfer to the nucleoside from the phosphate group donor selected from the group consisting of polyphosphoric acid or a salt thereof, phenylphosphoric acid or a salt thereof, and carbamyl phosphate or a salt thereof under the condition of pH 3.0 to 5.5.  Such an acid phosphatase preferably includes those derived from microorganisms.  In an especially preferred embodiment, the present invention uses an enzyme derived from a bacterium belonging to the genus Morqanella, Escherichia, Providencia, Enterobacter, Klebsiella or Serratia.  Representative examples of such a bacterium include the following bacterial strains.<br>
Morqanella morqanii NCIMB 10466<br>
Morqanella morqanii IFO 3168<br>
Morqanella morqanii IFO 3848<br>
Escherichia biattae JCM 1650<br>
Escherichia biattae ATCC 33429<br>
Escherichia biattae ATCC 33430<br>
Providencia stuartii ATCC 29851<br><br>
Providencia stuartii ATCC 33672 Enterobacter aeroqenes IFO 12010 Enterobacter aeroqenes IFO 13534 Klebsiella planticola IFO 14939 Klebsiella planticola IAM 1133 Serratia ficaria IAM 13450 Serratia marcescens IAM 12143<br>
It is noted that acid phosphatase (EC 3.1.3.2) is originally an enzyme which catalyzes a reaction to hydrolyze phosphate ester under an acidic condition, and it has a nucleotidase activity to degrade nucleoside-5'-phosphate ester produced by the transphosphorylation reaction (hereinafter, the nucleotidase activitiy is referred to as "phosphomonoesterase activity").  Even such an acid phosphatase can be used in the method for producing nucleoside-5'-phosphate ester of the present invention.  However, in order to obtain nucleoside-5'-phosphate ester at a high yield, it is desirable to use the mutant acid phosphatase in which the<br>
phosphomonoesterase activity is lowered as compared with the wild type acid phosphatase produced by the bacteria as described above (hereinafter simply referred to as "mutant acid phosphatase", if necessary).<br><br>
The mutant acid phosphatase is obtained by expressing a mutant gene obtained by directly mutating a gene coding for an acid phosphatase as described below. Alternatively, the mutant acid phosphatase may be also obtained by treating a microorganism which produces an acid phosphatase with irradiation of ultraviolet light or a mutating agent usually used for artificial mutation such as  N-methyl-N'-nitro-N-nitrosoguanidine (NTG), and cultivating a microorganism mutated to produce a mutant acid phosphatase having a lowered phosphomonoesterase activity.<br>
A protein having the acid phosphatase activity may be obtained frorr: the microorganisms as described above by cultivating the microbial strain having the activity in an appropriate medium, harvesting proliferated microbial cells, disrupting the microbial cells to prepare a cell-free extract, and adequately purifying the protein therefrom.<br>
The medium for cultivating the microorganism is not specifically limited, for which an ordinary medium may be available, containing an ordinary carbon source, a nitrogen source, inorganic ions, and optionally an organic nutrient source.  The carbon source to be adequately used includes, for example, saccharides such as glucose and sucrose, alcohols such as glycerol, and organic acids.__Ihe nitrogen source to be used includes, for example, ammonia gas, aqueous ammonia, and ammonium.<br><br>
salts.  The inorganic ions to be adequately used if necessary include, for example, magnesium ion, phosphate ion, potassium ion, iron ion, and manganese ion.  The organic nutrient source to be adequately used includes, for example, vitamins and amino acids as well as those containing them such as yeast extract, peptone, meat extract, corn steep liquor, casein hydrolysate, and soybean hydrolysate.<br>
The cultivation condition is also not specifically limited.  The microorganism may be cultivated, for example, under an aerobic condition for about 12 to 48 hours while appropriately controlling pH and temperature within ranges of pH 5 to 8 and temperature of 25 to 40 C.<br>
Proliferated microbial cells may be harvested from a culture liquid, for example, by centrifugation.  The cell-free extract is prepared from the harvested microbial cells by using an ordinary method.  Namely, the cell-free extract is obtained by disrupting the microbial cells by means of a method such as ultrasonic treatment, Dyno-mill, and French Press, and removing cell debris by centrifugation.<br>
The acid phosphatase is purified from the cell-free extract by using an adequate combination of techniques usually used for enzyme purification such as ammonium sulfate fractionation, ion exchange chromatography, hydrophobic chromatography, affinity chromatography, gel<br><br>
filtration chromatography, and isoelectric purification. As for the precipitation, it is not necessarily indispensable to completely purify the acid phosphatase. It is sufficient to achieve removal of contaminants such as an enzyme which participates in degradation of nucleoside as the substrate.<br>  Preparation of acid phosphatase gene<br>
A DNA fragment, which contains a structural gene coding for the protein having the acid phosphatase activity, can be cloned starting from, for example, cells of the microorganism having the enzyme activity. The cloning method includes, for example, a method in which a chromosomal gene expression library is screened by using the enzyme activity as an index, a method in which an antibody against the protein is prepared to screen a chromosomal gene expression library, and a method in which an amino acid sequence such as an N-terminal sequence of the purified protein is analyzed, on the basis of which a probe is prepared to screen a gene library.<br>
Specifically, the gene coding for the acid phosphatase of Morqanella morqanii, Escherichia blattae, Providencia stuartii, Enterobacter aeroqenes, Klebsiella planticola, Serratia ficaria or Serratia marcescens described above can be cloned by preparing a chromosomal gene expression library of each of the microorganisms,<br><br>
and screening the library by using the phosphatase activity as an index.<br>
Namely, a chromosomal gene expression library can be prepared by firstly preparing chromosomal DNA from Morqanella morqanii or Escherichia blattae, partially degrading it with an appropriate restriction enzyme, subsequently ligating it with a vector autonomously replicable in Escherichia coli, and transforming Escherichia coli with the obtained recombinant DNA.  A wide variety of restriction enzymes can be used for digesting chromosomal DNA by adjusting the digestion reaction time to adjust the degree of digestion.  Any vector may be used for cloning the gene provided rhat it is autonomously replicable in Escherichia coli.  It is possible to use, for example, pUC19, pUC118, pHSG298, pBR322, and pBluescript II.<br>
The vector may be ligated with the DNA fragment containing the gene coding for the acid phosphatase to prepare the recombinant DNA by previously digesting the vector with the same restriction enzyme as that used for digesting chromosomal DNA, or with a restriction enzyme which generates a cleaved edge complementary with a cleaved edge of the chromosomal DNA fragment, and ligating it with the DNA fragment by using ligase such as T4 DNA ligase.  Any microbial strain may be used as a recipient for the prepared recombinant DNA provided that it is appropriate for replication of the vector.  It is<br><br>
possible to use, for example, microbial strains of Escherichia coli such as HB101, JM109, and DH5.<br>
Transformants thus obtained are grown on an agar medium to form their colonies.  After that, when a reaction solution containing p-nitrophenylphosphoric acid is poured onto a surface of the medium to perform a reaction, then a strain, which has expressed the phosphatase activity, liberates p-nitrophenol and exhibits a yellow color.  A transformant, which harbors a DNA fragment containing the gene coding for the objective acid phosphatase, can be selected by performing the reaction described above under an acidic condition, and selecting the transformant by using the color development as an index.<br>
After that, a recombinant DNA is recovered from the selected transformant to analyze the structure of the DNA fragment containing the gene coding for the acid phosphatase ligated with the vector.  A nucleotide sequence of the gene coding for the acid phosphatase is shown in SEQ ID NO: 2 in Sequence Listing in the case of a gene derived from Morqanella morqanii NCIMB 10466, SEQ ID NO: 9 in Sequence Listing in the case of a gene derived from Escherichia blattae JCM 1650, SEQ ID NO: 17 in Sequence Listing in the case of a gene derived from Providencia stuartii ATCC 29851, SEQ ID NO: 19 in Sequence Listing in the case of a gene derived from Enterobacter aeroqenes IFO 12010, SEQ ID NO: 21 in<br><br>
Sequence Listing in the case of a gene derived from Klebsiella planticola IF014939, or SEQ ID NO: 23 in Sequence Listing in the case of a gene derived from Serratia ficaria IAM 13540.<br>
The deduced amino acid sequences of the acid phosphatases encoded by the above genes are illustrated in SEQ ID NO:  4, 11, 18, 20, 22 and 24.  The acid phosphatases encoded by the above genes are prefferably used for the present invention.  In addition, the acid phosphatase comprising an amino acid sequence which is substantially identical with an amino acid sequence of any one of the acid phosphatases encoded by the above genes is also prefferably used for the present invention.  The term "substantially identical" means that amino acid sequences of the acid phosphatases may have substitution, deletion, insertion or transition of one or a plurality of amino acid residues without losing an activity to produce nucleoside-5'-phosphate ester (hereinafter referred to as "transphosphorylation activity").<br>  Preparation of gene coding for mutant acid phosphatase<br>
The wild type acid phosphatase obtained as described above has a phosphomonoesterase activity. Therefore, the phosphomonoesterase activity may serve as a factor to cause accompanying degradation of the<br><br>
product as the reaction time passes in the production of nucleoside-5'-phosphate ester, resulting in decrease in reaction yield.  In order to overcome such a circumstance, it is advantageous to cause artificial mutation on the gene coding for the acid phosphatase so that the phosphomonoesterase activity is lowered.<br>
Methods for site-directed mutagenesis for causing objective mutation at an objective site of DNA include, for example, a method to use PCR (Higuchi, R., 61, in PCR technology, Erlich, H. A. Eds., Stockton press (1989); Carter, P., Meth. in Enzymol., 154, 382 (1987)), and a method to use phage (Kramer, W. and Frits, H. J., Meth. in Enzymol., 154, 350 (1987); Kunkel, T. A. et al., Meth. in Enzymol., 154, 367 (1987)).<br>
The mutant acid phosphatase having the lowered phosphomonoesterase activity is exemplified by the acid phosphatase comprising an amino acid sequence which is substantially identical with an amino acid sequence selected from the group consisting of sequences illustrated in SEQ ID NOs: 4, 11, 18, 20, 22 and 24 in Sequence Listing, and has mutation which lowers phosphomonoesterase activity of wild type acid phosphatase.  Concretely, the mutant acid phosphatase having the lowered phosphomonoesterase activity is exemplified, for the enzyme derived from Morqanella morqanii NCIMB 10466, by one in which the 72th glycine residue and/or the 151th isoleucine residue is<br><br>
substituted with another amino acid residue in an amino acid sequence illustrated in SEQ ID NO: 4 in Sequence Listing.  In Examples described below, a gene coding for a mutant acid phosphatase is illustrated as an example in which the 72th glycine residue is substituted with an aspartic acid residue, and the 151th isoleucine residue is substituted with a threonine residue.  On the other hand, the acid phosphatase having the lowered phosphomonoesterase activity is exemplified, for the enzyme derived from Escherichia blattae JCF^ 1650, by one in which the 74th glycine residue and .'or the 153th isoleucine residue is substituted with another amino acid residue in an amino acid sequence illustrated in SEQ ID NO: 11 in Sequence Listing.  In Examples described below, a gene coding for mutant, acid phosphatase is illustrated as an example in which the 74th glycine residue is substituted with an aspartic acid residue, and the 153th isoleucine residue is substituted with a threonine residue.<br>
Therefore, the nucleotide may be substituted at the specified site of the wild type gene in accordance with the site-directed mutagenesis method described above so that these mutant acid phosphatases are encoded.  The mutation to lower the phosphomonoesterase activity is desirably a type of mutation by which the activity to produce nucleoside-5'-phosphate" ester is not substantially lowered in comparison with wild type acid<br><br>
phosphatase.  However, even in the case that the activity to produce nucleoside-5'-phosphate ester is lowrerd, it will be sufficient if degree of decrease of phosphomonoesterase activity is larger than that of the activity to produce nucleoside-5'-phosphate ester, with the result that a ratio of phosphomonoesterase activity to the activity to produce nucleoside-5'-phosphate ester of the mutant acid phosphatase is lowered in comparison with the wild type acid phosphatase.  As for the degree of decrease in the phosphomonoesterase activity , the activity may be decreased to less than about 40 % of that of the wild type enzyme.  As illustrated below in the embodiments, the amino acid sequence of the acid phosphatase of Escherichia blattae JCM 1650 is highly homologous to that of Morqanella morqanii NCIMB 10466, and the 72th glycine residue and the 151th isoleucine residue in an amino acid sequence illustrated in SEQ ID NO: 4 correspond to the 74th glycine residue and the 153th isoleucine residue in an amino acid sequence illustrated in SEQ ID NO: 11 respectively.  Further, in addition to Escherichia blattae JCM 1650, amino acid sequences of acid phosphatases derived from microorganisms such as Providencia stuartii ATCC 29851, Enterobacter aeroqenes IFO 12010, Klebsiella planticola IFO 14939, and Serratia ficaria IAM 13450 have high homology with that of Morqanella morqanii NCIMB 10466, and amino acid sequences of these acid phosphatases<br><br>
include amino acids resudues each of which corresponds to the 72th glycine residue and the 151th isoleucine residue in an amino acid sequence illustrated in SEQ ID NO: 4 respectively.  Therefore, genes coding for mutant acid phosphatases derived from these microorganisms may be obtained as described above.  The 92th glycine residue and the 171th isoleucine residue in the amino acid sequence of the acid phosphatase derived from Providencia stuartii ATCC 29851, Enterobacter aeroqenes IFO 12010 or Klebsiella planticola IFO 14939 illustrated in SEQ ID NO: 18, 20 or 22, and the 88th glycine residue and the 167th isoleucine residue in the amino acid sequence of the acid phosphatase derived from Serratia ficaria IAM 13450 illustrated in SEQ ID NO: 24 respectively correspond to the 72th glycine residue and the 151th isoleucine residue in an amino acid sequence illustrated in SEQ ID NO: 4.<br>  Introduction of acid phosphatase gene into host<br>
A recombinant microorganism for expressing the acid phosphatase activity at a high level can be obtained by introducing the DNA fragment containing the gene coding for the protein having the acid phosphatase activity obtained as described above into cells of a host after recombining the DNA fragment again with an appropriate vector.  In such a procedure, the wild type acid phosphatase is expressed by using the gene coding for<br><br>
the wild type acid phosphatase, while the mutant acid phosphatase is expressed by using the gene coding for the mutant acid phosphatase.<br>
The host includes the microbial strains of Escherichia coli such as HB101, JM109, and DH5 described above.  Other than these strains, all bacteria can be utilized as the host provided that a replication origin of constructed recombinant DNA and the acid phosphatase gene make their functions, the recombinant DNA is replicable, and the acid phosphatase gene is expressible.  One of the most preferred hosts is Escherichia coli JM109.<br>
The vector for incorporating the gene coding for the acid phosphatase thereinto is not specifically "limited provided that it is replicable in the host. When Escherichia coli is used as the host, the vector may be exemplified by plasmids autonomously replicable in this bacterium.  For example, it is possible to use ColEl type plasmids, pl5A type plasmids, R factor type plasmids, and phage type plasmids.  Such plasmids specifically include, for example, pBR322 (Gene, 2, 95 (1977)), pUC19 (Gene, 33, 103 (1985)), pUC119 (Methods in Enzymology, 153, 3 (1987)), pACYC184 (J. Bacterid., 134, 1141 (1978;), and pSClOl (Proc. Natl. Acad. Sci. U.S.A., 70, 3240 (1973)).<br>
When the DNA fragment containing the gene coding for the acid phosphatase contains a promoter which is<br><br>
functional in the host, the DNA fragment may be ligated with the vector as it is.  When the DNA fragment does not contain such a promoter, another promoter which works in the host microorganism such as lac, trp, and PL may be ligated at a position upstream from the gene. Even when the DNA fragment contains the promoter, the promoter may be substituted with another promoter in order to efficiently express the gene coding for the acid phosphatase.<br>
There is no special limitation for a method for introducing, into the host, the recombinant DNA constructed by ligating the vector with the DNA fragment containing the gene coding for the acid phosphatase. The recombinant DNA may be introduced into the host by using an ordinary method.  When Escherichia coli is used as the host, it is possible to use, for example, a calcium chloride method (J. Mol. Biol., 53, 159 (1970)), a method of Hanahan (J. Mol. Biol., 166, 557 (1983)), an SEM method (Gene, 96, 23 (1990)), a method of Chung et al. (Proc. Natl. Acad. Sci. U.S.A., 86, 2172 (1989)), and electroporation (Nucleic Acids Res., 16, 6127 (1988) ).<br>
The acid phosphatase gene may be inserted into the autonomously replicable vector DNA, which may be introduced into the host so that it is harbored by the host as extrachromosomal DNA as described above. Alternatively, the acid phosphatase gene may be<br><br>
incorporated into chromosome of the host microorganism in accordance with a method which uses transduction, transposon (Biotechnol.. i, 417 (1983)), Mu phage (Japanese Patent Laid-open No. 2-109985), or homologous recombination (Experiments in Molecular Genetics, Cold Spring Harbor Lab. (1972)).<br>  Expression of acid phosphatase gene by recombinant microorganism<br>
The transfcrmant obtained as described above, into which the recombinant DNA containing the gene coding for the acid phosphatase has been introduced, is capable of expressing the acid phosphatase activity at a high level in its cells by cultivating it in an'appropriate medium containing a carbon source, a nitrogen source, inorganic ions, and optionally an organic nutrient source.  The carbon source to be adequately used includes, for example, carbohydrates such as glucose, alcohols such as glycerol, and organic acids.  The nitrogen source to be used includes, for example, ammonia gas, aqueous ammonia, and ammonium salts.  The inorganic ions to be adequately used if necessary include, for example, magnesium ion, phosphate ion, potassium ion, iron ion, and manganese ion.  The organic nutrient source to be adequately used includes, for example, vitamins and amino acids as well as those containing them such as yeast extract, peptone, meat extract, corn steep liquor,<br><br>
casein hydrolysate, and soybean hydrolysate.  The amount of expression of the acid phosphatase activity may be increased by adding, to the medium, an expression-inducing agent depending on a promoter such as IPTG (isopropyl-p-D-thiogalactopyranoside).<br>
The cultivation condition is also not specifically limited.  The cultivation may be performed, for example, under an aerobic condition for about 12 to 48 hours while appropriately controlling pH and temperature within ranges of pH 5 to 8 and temperature of 2 5 to 40 ° C.<br>
After that, microbial cells are harvested from a culture, and a cell-free extract is obtained by disruption, from which the acid phosphatase can be purified.  The purification is performed by using an appxopriate combination of techniques usually used for enzyme purification such as those described in the aforementioned item .  As for the purification, it is not necessarily indispensable to completely purify the acid phosphatase.  It is sufficient to achieve removal of contaminants such as an enzyme which participates in degradation of nucleoside as the substrate.<br>  Production of nucleoside-5 '-phosphate, ester<br>
Nucleoside-5'-phosphate ester can be produced in a reaction mixture by allowing the acid phosphatase obtained as described in the item , or the wild type<br><br>
acid phosphatase or the mutant acid phosphatase obtained by expressing the gene in a large amount in accordance with the genetic engineering technique as described in the item  to make contact and cause the reaction of a nucleoside with a phosphate group donor selected from the group consisting of polyphosphoric acid or a salt thereof, phenylphosphoric acid or a salt thereof, and carbamyl phosphate or a salt thereof.  In order to achieve a high productivity in this reaction, it is important to adjust pH of the reaction solution to be weakly acidic in a range of 3.0 to 5.5.<br>
When the gene coding for the acid phosphatase is expressed in a large amount by means of the genetic engineering technique, especially when the gene coding for the mutant acid phosphatase having the lowered phosphomonoesterase activity is expressed in a large amount, then it is also possible to produce nucleoside-5'-phosphate ester inexpensively and efficiently by using a culture containing microbial cells of the transformant, the microbial cells separated and harvested from the culture, or a product obtained from the microbial cells in accordance with, for example, an immobilizing treatment, an acetone treatment, or a lyophilizing treatment, instead of the purified acid phosphatase.<br>
The nucleoside to be used includes, for example, purine nucleosides such as inosine, guanosine,<br><br>
adenosine, xanthosine, purine riboside, 6-methoxypurine riboside, 2,6-diaminopurine riboside, 6-fluoropurine riboside, 6-thiopurine riboside, 2-amino-6-thiopurine riboside, and mercaptoguanosine; and pyrimidine nucleosides such as uridine, cytidine, 5-aminouridine, 5-hydroxyuridine, 5-bromouridine, and 6-azauridine.  As a result of the reaction, these natural type nucleosides and nonnatural type nucleosides are specifically phosphorylated at their 5'-positions, and corresponding nucleoside-5'-phosphate esters are produced respectively.<br>
The nucleoside is desirably added to the reaction solution at a concentration of 1 to 20 g/dl.  In the case of use of a nucleoside which is scarcely soluble in water, the reaction yield may be improved by adding boric acid or a surfactant such as dimethyl sulfoxide.<br>
As for the phosphate group donor to be used, those usable as the polyphosphoric acid or the salt thereof include, for example, pyrophosphoric acid, tripolyphosphoric acid, trimetaphosphoric acid, tetrametaphosphoric acid, hexametaphosphoric acid, mixtures thereof, sodium salts thereof, potassium salts thereof, and mixtures of these salts.  Those usable as the phenylphosphoric acid or the salt thereof include, for example, disodium phenylphosphate, dipotassium phenylphosphate, 0,O-diphenylphosphoric acid anhydride, and mixtures thereof.  Those usable as the carbamyl<br><br>
phosphate or the salt thereof include, for example, disodium carbamyl phosphate, dipotassium carbamyl phosphate, diammonium carbamyl phosphate, dilithium carbamyl phosphate, and mixtures thereof.  The concentration at which the phosphate group donor is used is determined by the concentration of the nucleoside, as the phosphate group acceptor.  The phosphate group donor is usually used in an amount which is 1 to 5 times that of the nucleoside.<br>
A preferred result is obtained in the reaction usually at a temperature of 20 to 60 °C, preferably 30 to 40 :C at a pH on a weakly acidic side of 3.5 to 6.5, preferably 4.0 to 5.0.  The reaction may be performed by adopting any one of a stationary method and an agitating method.  The reaction time defers depending on the condition such as the activity of the enzyme to be used and the substrate concentration, however, it is 1 to 100 hours.<br>
The nucleoside-5'-phosphate ester thus produced may be collected and separated from a .mixture after completion of the reaction by adopting a method to use a synthetic resin for adsorption, a method to use a precipitating agent, and other ordinary methods for collection and separation.<br>
Brief Description of the Drawings<br><br>
Fig. 1 illustrates a relationship between reaction pH and produced amount of 5'-inosinic acid in a reaction performed by using an enzyme derived from Morqanella morqanii.<br>
Fig. 2 illustrates a relationship between reaction pH and produced amount of 5'-inosinic acid in a reaction performed by using an enzyme derived from Escherichia blattae.<br>
Fig. 3 illustrates a restriction enzyme map of a chromosomal DNA fragment of Morqanella morqanii containing a gene coding for an acid phosphatase.<br>
Fig. 4 illustrates produced amount of 5'-inosinic acid in a reaction performed by using a strain harboring phosphatase gene derived from Morqanella morqanii.<br>
Fig. 5 illustrates produced amount of 5'-inosinic acid in reactions performed by using a strain horboring the wild type acid phosphatase gene and a strain horboring the mutant acid phosphatase gene derived from Morqanella morqanii respectively.<br>
Fig. 6 illustrates a restriction enzyme map of a chromosomal DNA fragment of Escherichia blattae containing a gene coding for an acid phosphatase.<br>
Fig. 7 illustrates a diagram showing produced amount of 5'-inosinic acid in a reaction performed by using a strain harboring the acid phosphatase gene derived from Escherichia blattae.<br>
Fig. 8 illustrates produced amount of 5'-inosinic<br><br>
acid in reactions performed by using a strain harboring the wild type acid phosphatase gene and a strain harboring the mutant acid phosphatase gene derived from Escherichia blattae respectively.<br>
Fig. 9 illustrates a restriction enzyme map of a chromosomal DNA fragment derived from Enterobacter aeroqenes which contains the gene coding for acid phosphatase.<br>
Fig. 10 illustrates a restriction enzyme map of a chromosomal DNA fragment derived from Klebsiella planticola which contains the gene coding for acid phosphatase.<br>
Fig. 11 illustrates a restriction enzyme map of a chromosomal DNA fragment derived from .Serratia ficaria which contains the gene coding for acid phosphatase.<br>
Fig.12 illustrates amino acid sequences in one-letter deduced from nucleotide sequences of acid phosphatases derived from Morqanella morqanii, Escherichia blattae, Providencia stuartii, Enterobacter aeroqenes„ Klebsiella planticola and Serratia ficaria.<br>
Description of Preferred Embodiments<br>
The present invention will be specifically explained below with reference to Examples, however, the present invention is not limited to these Examples.<br><br>
The transphosphorylation activity was measured under the following condition using inosine as a substrate.  The reaction was performed at pH 5.0 at 30 ° C for 10 minutes in a reaction solution (1 ml) containing 40  umol/ml of inosine,   100  umol/ml of sodium pyrophosphate, 100 umol/ml of sodium acetate buffer (pH 5.0), and an enzyme.  The reaction was stopped by adding 200 pi of 2 N hydrochloric acid.  After that, precipitates were removed by centrifugation.  Then, 5'-Inosinic acid produced by the transphosphorylation reaction was quantitatively measured.  An amount of enzyme to produce 1 umol of 5'-inosinic acid per 1 minute under this standard reaction condition was defined as 1 unit.<br>
The phosphomonoesterase activity was measured under the following condition using 5'-inosinic acid as a substrate.  The reaction was performed at 30 °C for 10 minutes in a reaction solution (1 ml) containing 10 umol/ml of 5'-inosinic acid, 100 umol/ml of MES/NaOH buffer (pH 6.0), and an enzyme.  The reaction was stopped by adding 200 pi of 2 N hydrochloric acid. After that, precipitates were removed by centrifugation. Then, inosine produced by the hydrolytic reaction was quantitatively measured.  An amount of enzyme to produce 1 pmol of inosine per 1 minute under this standard reaction condition was defined as 1 unit.'<br>
Inosine and 5'-inosinic acid were analyzed under<br><br>
the following condition by means of high-performance liquid chromatography (HPLC).<br>
Column: Cosmosil 5C18-AR (4.6 x 150 mm) [produced<br>
by nacalai tesque]; Mobile phase: 5 mM potassium phosphate buffer<br>
(pH 2.8)/methanol = 95/5; Flow rate: 1.0 ml/min; Temperature: room temperature; Detection: UV 245 nm.<br>
Incidentally, in the reaction to produce nucleoside-5'-phosphate esters using nucleosides other than inosine as raw materials, the nucleosides as raw materials and produced nucleoside-5'-phosphate esters "were analyzed by HPLC as described above.<br>
Example 1: Purification and Caracterozatoion of Acid Phosphatase Derived from Morqanella morqanii<br>
A nutrient medium (pH 7.0, 50 ml) containing 1 g/dl of peptone, 0.5 g/dl of yeast extract, and 1 g/dl of sodium chloride was poured into Sakaguchi flasks (500 ml), which was sterilized at 120 °C for 20 minutes.  A slant culture of Morqanella morqanii NCIMB 10466 was inoculated to each of the flasks once with a platinum loop, which was cultivated at 30 °C for 16 hours with shaking.  Microbial cells (about 3,000 g), which were<br><br>
harvested from a culture by centrifugation, were suspended in 100 mM potassium phosphate buffer (1 L, pH 7.0).  A ultrasonic treatment was performed at 4 °C for 20 minutes to disrupt the microbial cells.  The treated suspension was centrifuged to remove its insoluble fraction.  Thus a cell-free extract was prepared.<br>
Ammonium sulfate was added to the cell-free extract so that 30 % saturation was achieved.  Appeared precipitate was removed by centrifugation, and then ammonium sulfate was further added to supernatant so that 60 % saturation was achieved.  Appeared precipitate was collected by centrifugation, and it was dissolved in 100 mM potassium phosphate buffer.<br>
This crude enzyme solution was dialyzed four times against 5 L of 100 mM potassium phosphate buffer (pH 7.0), and then it was applied to a DEAE-Toyopeal 650M column (4&gt; 4.1 x 22 cm) equilibrated with 20 mM potassium phosphate buffer (pH 7.0), followed by washing with 800 ml of 20 mM potassium phosphate buffer (pH 7.0).  The transphosphorylation activity was found in a fraction which passed through the column, and thus the fraction was recovered.<br>
The fraction was added with ammonium sulfate so that 35 % saturation was achieved, which was adsorbed to a Butyl-Toyopeal column (4&gt; 3.1 x 26 cm) equilibrated with 20 mM potassium phosphate buffer (pH 7.0) containing ammonium sulfate at 35 % saturation.  Elutioh<br><br>
was performed by using a linear concentration gradient from 35 % saturation to 20 % saturation in potassium phosphate buffer (pH 7.0).<br>
Active fractions were collected and dialyzed against 1 L of 50 mM potassium phosphate buffer (pH 7.0), followed by being applied to a hydroxyapatite column ((}) 5 x 6.5 cm) equilibrated with 50 mM potassium phosphate buffer (pH 7.0).  Elution was performed by U S 2-TlG  a linear concentration gradient from 50 mM to 300 mM of potassium phosphate buffer (pH 7,0),<br>
Active fractions were collected and concentrated by ultrafiltration.  This enzyme solution was applied into a HiLoad TM 16/60 Superdex 200 column (produced by Pharmacia).  Elution was performed at a flow rate of 1.0 ml/minute by using 50 mM potassium phosphate Bu-ffer containing 100 mM sodium chloride.<br>
In accordance with the procedure as described above, the enzyme exhibiting the transphosphorylation activity was purified from the cell-free extract consequently about 550-fold at a recovery ratio of about 10 %.  The specific activity and the recovery ratio in this purification process are shown in Table 1.  This enzyme sample was homogeneous on SDS-polyacrylamide gel electrophoresis.<br><br>
Total	Total	Specific	<br>
activity	protein	activity	ratio<br>
(unit)	(mq)	(unit/mq)	(%)<br>
597	127,200	0.005	100<br>
568	122,210	0.005	95<br>
Table 1<br>
Step Recovery<br>
1.	Cell-free extract  597<br>
2.	Ammonium sulfate fractionation (30 to 60 %)<br>
3.	DEAE-Toyopearl     517      36,498   0.014     87<br>
1,121	0.351	66<br>
50	2.244	19<br>
24	2.530	10<br>
4.	Butyl-Tcyopearl    394<br>
5.	Hydroxyapatite     112<br>
6.	Superdex 200        63<br><br>
The purified enzyme had the following properties.<br>
(1)	Action: Phosphate group is transferred from a phosphate group donor such as polyphosphoric acid to nucleoside, and nucleoside-5'-phosphate ester is produced.  Reversely, this enzyme also exhibits an activity to hydrolyze phosphate ester.<br>
(2)	Substrate specificity: Those which serve as the phosphate group donor in the transphosphorylation reaction include, for example, pyrophosphoric acid, tripolyphosphoric acid, trimetaphosphoric acid, tetrametaphosphoric acid, hexametaphosphoric acid,<br>
disodium phenylphosphate, dipotassium phenylphosphate, 0,0-diphenylphosphoric acid anhydride, disodium carbamyl<br><br>
phosphate, dipotassium carbamyl phosphate, diammonium carbamyl phosphate, and dilithium carbamyl phosphate. Those which serve as the phosphate group acceptor include, for example, purine riboside, inosine, guanosine, adenosine, xanthosine, uridine, and cytidine. On the other hand, those which undergo the action in the phosphate ester hydrolytic reaction include, for example, inorganic phosphoric acid such as pyrophosphoric acid, tripolyphosphoric acid, trimetaphosphoric acid, tetrametaphosphcric acid, hexametaphcspiooric acid; phosphate ester such as disodium phenylphosphate, dipotassium phenylphosphate, 0,O-diphenylphosphoric acid anhydride, disodium carbamyl phosphate, dipotassium carbamyl phosphate, diammonium carbamyl phosphate, and dilithium carbamyl phosphate;* and 5'-nucleotide such as 5'-purine ribotide, 5'-inosinic acid, 5'-guanylic acid, 5'-adenylic acid, 5'-xanthylic acid, 5'-uridylic acid, and 5'-cytidylic acid.<br>
(3)	Optimum pH: 5.2 (transphosphorylation reaction), 6.5 (phosphate ester hydrolytic reaction).<br>
(4)	pH stability: pH 3.0 to 12.0 (treatment at 30 °C for 60 minutes).<br>
(5)	Optimum temperature: about 35 °C.<br>
(6)	Temperature stability: stable up to 30 °C (treatment at pH 7.0 for 30 minutes).<br>
(7)	Effect of the addition of metal ion and inhibitor: This enzyme exhibits no activation phenomenon relevant<br><br>
to its activity by addition of any metal ion.  The activity is inhibited by Ag2\ Pb2\ Hg2*, and Cu2\  The activity is also inhibited by iodoacetic acid.<br>
(8)	Molecular weight: A calculated molecular weight is about 190,000 in accordance with high-performance liquid chromatography (TSKgel G-3000SW, produced by Toyo Soda).<br>
(9)	Subunit molecular weight: A calculated subunit molecular weight is about 25,000 in accordance with SDS-polyacrylamide gel electrophoresis.<br>
This enzyme exhibits not only the activity to transfer phosphate group to nucleoside, but also the activity to reversely hydrolyze phosphate ester. Moreover, this enzyme exhibits the phosphate ester hydrolytic activity (phosphomonoestrase activity) which is higher than the transphosphorylation activity by not less than 20 times.  Other properties are well coincident with those of a known acid phosphatase produced by a bacterium belonging to the genus Morqanella (Microbiology, 140, 1341-1350 (1994)). Accordingly, it has been clarified that this enzyme is an acid phosphatase.<br>
Sodium pyrophosphate (10 g/dl) and inosine (2 g/dl) were dissolved in sodium acetate buffers each having pH of 5.5, 5.0, 4.5, 4.0, and 3.5, to which the enzyme sample described above was added so that a concentration of 50 units/dl was obtained.  The reaction mixture was<br><br>
incubated at 30 °C for 6 hours while maintaining each pH, and the amount of produced 5'-inosinic acid was measured along with passage of time.  Produced inosinic acid contained only 5'-inosinic acid.  By-production of 2'-inosinic acid and 3*-inosinic acid was not observed at all.  A result is shown in Fig. 1.  The velocity of 5'-inosinic acid production was maximum at pH 5.0. However, the maximum accumulated amount of 5'-inosinic acid was higher at lower pH.  The reaction condition at pH 4.0 was most efficient for production of 5'-inosinic acid, in which 5'-inosinic acid was produced and accumulated in an amount of 2.60 g/dl by performing the reaction for 3 hours.<br>
Example  2:  Phosphorylation Reaction of  Various Nucleosides by Acid Phosphatase Sample Derived from Morqanella morqanii<br>
Sodium pyrophosphate (10 g/dl) and inosine, guanosine, uridine, or cytidine (2 g/dl) as a phosphate group acceptor were dissolved in sodium acetate buffer (pH 4.0), to which the enzyme sample prepared in Example 1 was added so that its concentration was 50 units/dl. The reaction mixture was incubated at 30 °C for 3 hours while maintaining pH at 4.0.  The amount of nucleoside¬s' -ester produced by the reaction is shown in Table 2.<br>
Produced nucleotide contained only nucleoside-5'-<br><br>
ester.  By-production of nucleoside-2'-ester and nucleoside-3'-ester was not observed at all.<br>
Table 2<br>
Nucleoside  Product	Produced amount<br>
(g/dl) Inosine     5'-inosinic acid       2.60 Guanosine   5'-guanylic acid       1.90 Uridine     5'-uridylic acid       1.30 Cytidine    5'-cytidylic acid      Q.98<br>
Example 3: Production of 5'-Inosinic acid from Various Phosphate Compounds as Phosphate Gx"oup Donors by Acid Phosphatase Sample Derived from Morqanella morqanii<br>
Inosine (2 g/dl) and sodium tripolyphosphate, sodium polyphosphate (trade name: Polygon P, produced by Chiyoda Chemical), disodium phenylphosphate, or disodium carbamyl phosphate (10 g/dl) as a phosphate group donor were dissolved in sodium acetate buffer (pH 4.0), to which the enzyme sample prepared in Example 1 was added so that its concentration was 50 units/dl.  The reaction mixture was incubated at 30 °C for 3 hours while maintaining pH at 4.0.  The amount of 5'-inosinic acid produced by the reaction is shown in Table 3.<br>
5'-Inosinic acid was efficiently produced and<br><br>
accumulated by using any of the phosphate group donors. However, the accumulated amount of 5'-inosinic acid was the highest when sodium polyphosphate was used as the phosphate group donor.<br>
Table 3<br>
Phosphate group donor      Produced 5'-inosinic<br>
acid (q/dl)<br>
Sodium tripolyphosphate	2.10<br>
Sodium polyphosphate	2.72<br>
Disodium phenylphosphate	2.33<br>
DisoGiujn carbamyl phosphate        2.54<br>
Example 4: Purification and Characterization of Acid Phosphatase Derived from Escherichia blattae<br>
"A nutrient medium (pH 7.0, 50 ml) containing 1 g/dl of peptone, 0.5 g/dl of yeast extract, and 1 g/dl of sodium chloride was poured into Sakaguchi flasks (500 ml), which was sterilized at 120 °C for 20 minutes.  A slant culture of Escherichia blattae JCM 1G50 was inoculated to each of the flasks once with a platinum loop, which was cultivated at 30 'C for 16 hours with shaking.  Microbial cells were harvested from a culture by centrif.ugation.  The microbial cells (about 3,-300—g-)— were suspended in 100 mM potassium phosphate buffer (1 L, pH 7.0).  A ultrasonic treatment was performed at 4 °<br><br>
C for 20 minutes to disrupt the microbial cells.  The treated suspension was centrifuged to remove its insoluble fraction.  Thus a cell-free extract was prepared.<br>
Ammonium sulfate was added to the cell-free extract so that 30 % saturation was achieved.  Appeared precipitate was removed by centrifugation, and then ammonium sulfate was further added to supernatant so that 60 % saturation was achieved.  Appeared precipitate was recovered by centrifugation, and it was dissolved in 100 mM potassium phosphate buffer.<br>
This crude enzyme solution was dialyzed four times against 5 L of 100 mM potassium phosphate buffer (pH 7.0), and then it was applied to a DEAE-Toyopeal 650M column ((j) 6.2 x 35 cm) equilibrated with 2C mM potassium phosphate buffer (pH 7.0), followed by washing with 20 mM potassium phosphate buffer (pH 7.0).  The transphosphorylation activity was found in a fraction which passed through the column, and thus the fraction was collected.<br>
The active fraction was added with ammonium sulfate so that 35 % saturation was achieved, which was applied to a Butyl-Toyopeal column (4&gt; 5.0 x 22.5 cm) equilibrated with 20 mM potassium phosphate buffer (pH 7.0) containing ammonium sulfate at 35 % saturation. Elution was performed by using a linear concentration gradient from 35 % saturation to 20 % saturation in<br><br>
potassium phosphate buffer (pH 7.0).<br>
Active fractions were collected and dialyzed against 1 L of 100 mM potassium phosphate buffer (pH 7.0), followed by being applied to a hydroxyapatite column ( 3.0 x 7.0 cm) equilibrated with 100 mM potassium phosphate buffer (pH 7.0).  Elution was performed by using a linear concentration gradient from 50 mM to 100 mM of potassium phosphate buffer (pH 7.0), and active fractions were collected.<br>
This enzyme solution was dialyzed against 1 L of 10 mM potassium phosphate buffer (pH 6.0), followed by being applied to a CM-Toyopeari column (<j> 2.0 x 14.0 cm) equilibrated with 10 mM potassium phosphate buffer (pH 6.0).  Elution was performed by using a linear concentration gradient in potassium phosphate buffer (pH 6.0) containing from 0 mM to 300 mM potassium chloride. Active fractions eluted from the column were collected.<br>
In accordance with the procedure as described above, the enzyme exhibiting the transphosphorylation activity was purified from the cell-free extract consequently about 600-fold at a recovery ratio of about 16 %.  The specific activity and the recovery ratio in this purification process are shown in Table 4.  This enzyme sample was homogeneous on SDS-polyacrylamide gel electrophoresis.<br><br>
Table 4<br>
Step               Total	Total   Specific	Recovery<br>
activity	protein activity	ratio<br>
(unit)	(mq)    (unit/mq)	(%)<br>
1.	Cell-free extract  365	160,650  0.002	100<br>
2.	Ammonium sulfate   340	138,895   0.002	93 fractionation<br>
(30 to 60 %)<br>
3.	DEAE-Toyopearl     318	30,440   0.010	87<br>
4.	Butyl-Toyopearl    232	661   0.347	63<br>
5.	Hydroxyapatite      96	9 5   1.000	26<br>
6.	CM-Toyopearl        59	4 3   1.35 5	16<br>
The purified enzyme had the following properties.<br>
(1)	Action: Phosphate group is transferred from a phosphate group donor such as polyphosphoric acid to nucleoside, and nucleoside-5'-phosphate ester is produced.  Reversely, this enzyme also exhibits an activity to hydrolyze phosphate ester.<br>
(2)	Substrate specificity: Those which serve as the phosphate group donor in the transphosphorylation reaction include, for example, pyrophosphoric acid, tripolyphosphoric acid, trimetaphosphoric acid, tetrametaphosphoric acid, hexametaphosphoric acid, disodium phenylphosphate, dipotassium phenylphosphate, 0,0-diphenylphosphoric acid anhydride, disodium carbamyl<br>
phosphate, dipotassium carbamyl phosphate, diammonium carbamyl phosphate, and dilithium carbamyl phosphate.<br><br>
Those which serve as the phosphate group acceptor include, for example, purine riboside, inosine, guanosine, adenosine, xanthosine, uridine, and cytidine. On the other hand, those which undergo the action in the phosphate ester hydrolytic reaction include, for example, inorganic phosphoric acid such as pyrophosphoric acid, tripolyphosphoric acid, trimetaphosphoric acid, tetrametaphosphoric acid, hexametaphosphoric acid; phosphate ester such as disodium pnenylphosphate, dipotassium phenylphosphate, 0,0-diphenylphosphoric acid anhydride, disodium carbamyl phosphate, dipotassiurn carbamyl phosphate, diammonium carbamyl phosphate, and dilithium carbamyl phosphate; and 5'-nucleotide such as 5'-purine ribotide, 5'-inosinic acid, 5'-guanylic acid, 5'-adenylic acid, 5'-xanthylic acid, 5 '-uridylic acid, and 5 '-cytidylic acid,.<br>
(3)	Optimum pH: 5.2 (transphosphorylation reaction), 6.5 (phosphate ester hydrolytic reaction).<br>
(4)	pH stability: pH 3.5 to 12.0 (treatment at 30 °C for 60 minutes).<br>
(5)	Optimum temperature: about 35 °C.<br>
(6)	Temperature stability: stable up to 40 °C (treatment at pH 7.0 for 30 minutes).<br>
(7)	Effect of the addition of metal ion and inhibitor: This enzyme exhibits no activation phenomenon relevant to its activity by addition of any metal ion The activity is inhibited by Fe2t, Ag2+, Pb2\ Hg2+, and Cu2 <br>
The activity is also inhibited by iodoacetic acid.<br>
(8)	Molecular weight: A calculated molecular weight is about 188,000 in accordance with high-performance liquid chromatography (TSKgel G-3000SW, produced by Toyo Soda).<br>
(9)	Subunit molecular weight: A calculated subunit molecular weight is about 24,500 in accordance with SDS-polyacrylamide gel electrophoresis.<br>
This enzyme also exhibits not only the activity to transfer phosphate group to nucleoside, but also the activity to reversely hydroiyze phosphate ester, in the same manner as the enzyme purified from the cell-free extract of Morqanelia morqanii NCIMB 10466.  Moreover, this enzyme exhibits the phosphate ester hydrolytic activity (phosphomonoesterase activity) which is higher than the transphosphorylation activity by not less than 30 times.  Accordingly, it has been clarified that this enzyme is an acid phosphatase.<br>
Sodium pyrophosphate (15 g/dl) and inosine (3 g/dl) were dissolved in sodium acetate buffers each having pH of 5.5, 5.0, 4.5, 4.0, and 3.5, to which the enzyme sample described above was added so that a concentration of 50 units/dl was obtained.  The reaction mixture was incubated at 30 °C for 6 hours while maintaining each pH, and the amount of produced 5'-inosinic acid was measured along with passage of time.  Produced inosinic acid contained only 5'-inosinic acid.  By-production of<br><br>
2'-inosinic acid and 3'-inosinic acid was not observed at all.  A result is shown in Fig. 2.  The velocity of 5'-inosinic acid production was maximum at pH 5.0. However, the maximum accumulated amount of 5'-inosinic acid was higher at lower pH.  The reaction condition at pH 4.0 was most efficient for production of 5'-inosinic acid.  5'-Inosinic acid was produced and accumulated in an amount of 1.56 g/dl by performing the reaction at 30 °C at pH 4.0 for 3 hours.<br>
Example  5:  Phosphorylation  Reaction  of  Various Nucleosides by Acid Phosphatase Sample Derived from Escherichia blattae<br>
Sodium pyrophosphate (15 g/dl) and inosine, guanosine, uridine, or cytidine (3 g/dl) were dissolved in sodium acetate buffer (pH 4.0), to which the enzyme sample prepared in Example 4 was added so that its concentration was 50 units/dl.  The reaction mixture was incubated at 35 °C for 3 hours while maintaining pH at 4.0.  The amount of produced nucleoside-5'-ester is shown in Table 5.<br>
Produced nucleotide contained only nucleoside-5'-ester.  By-production of nucleoside-2'-ester and nucleoside-3'-ester was not observed at all.<br><br>
Table 5<br>
Nucleoside Product	Produced amount<br>
(q/dl) Inosine    5'-inosinic acid      1.56 Guanosine   5'-guanylic acid       1.05 Uridine    5'-uridylic acid      1.87 Cytidine   5'-cytidylic acid     1.22<br>
Example 6: Production of 5'-Inosinic acid from Various Phosphate Compounds as Phosphate Group Donors by Acid Phosphatase Sample Derived from Escherichia blattae<br>
Inosine (2 g/dl) and sodium tripolyphosphate, sodium polyphosphate (trade name: Polygon P, produced by Chiyoda Chemical), disodium phenylphosphate, or disodium carbamyl phosphate (10 g/dl) as a phosphate group donor were dissolved in sodium acetate buffer (pH 4.0), to which the enzyme sample prepared in Example 4 was added so that its concentration was 50 units/dl.  The reaction miture was incubated at 35 °C for 3 hours while maintaining pH at 4.0.  The amount of produced 5'-inosinic acid is shown in Table 6.<br>
5'-Inosinic acid was efficiently produced and accumulated by using any of the phosphate group donors. However, the accumulated amount of 5'-inosinic acid was.<br><br>
the highest when sodium polyphosphate was used as the phosphate group donor.<br>
Table 6<br>
Phosphate group donor	Produced 5'-inosinic<br>
acid (g/dl)<br>
Sodium tripolyphosphate	1.20<br>
Sodium polyphosphate	1.79<br>
Disodium phenylphosphate	1.50<br>
Disodiunj carbaiayl phosphate	1.53<br>
Example 7: -isolation of Gene Coding for Acid Phosphatase from Chromosome of Morganella morganii<br>
(1)  Determination of N-terminal amino acid sequence<br>
The acid phosphatase purified from the cell-free extract of Morganella morganii NCIMB 10466 in accordance with the method described in Example 1 was adsorbed to DITC membrane (produced by Milligen/Biosearch), and its N-terminal amino acid sequence was determined by using Prosequencer 6625 (produced by Milligen/Biosearch).  An N-terminal amino acid sequence comprising 20 residues shown in SEQ ID NO: 1 in Sequence Listing was determined.<br><br>
(2)  Isolation of DNA fragment containing gene coding for acid phosphatase<br>
Chromosomal DNA was extracted from cultivated microbial cells of Morganella morganii NCIMB 10466 in accordance with a method of Murray and Thomson (Nucl. Acid Res., 4321, 8 (1980)).  The chromosomal DNA was partially degraded with restriction enzyme Sau3AI. After that, DNA fragments of 3 to 6 kbp were fractionated by means of sucrose density gradient centrifugation.  A plasmid vector pUC118 (produced by Takara Shuzo) was digested with restriction enzyme BamHI, which was ligated with the partially degraded chromosomal DNA fragments.  DNA ligation was performed by using DNA ligation kit (produced by Takara Shuzo) in accordance with a designated method.  After that, Escherichia coli JM109 (produced by Takara Shuzo) was transformed with an obtained DNA mixture in accordance with an ordinary method.  Transformants were plated on an L agar medium containing 100 ug/ml of ampicillin, and they were grown to prepare a gene library.<br>
A reaction solution containing 4 mM p-nitrophenylphosphoric acid and 100 mM MES/NaOH buffer (pH 6.5) was poured onto a surface of the agar medium on which the transformants had grown, and the temperature was kept at 30 °C for 15 minutes.  Strains which had expressed the phosphatase activity liberated p-nitrophenol and exhibited a yellow color.  Accordingly,<br><br>
transformants were selected by using this phenomenon as an index.  As a result of screening for a gene expression library comprising about 20,000 strains of transformants, 30 strains of transformants which had expressed the phosphatase activity were obtained.<br>
The transformants (30 strains), which had expressed the phosphatase activity, were subjected to single colony isolation.  Single colonies were inoculated to an L-medium (2.5 ml) containing 100 ug/ml of ampicillin, and they were cultivated at 37  C for 16 hours.  Sodium acetate buffer (100 mM, pK 5.0, 50 ul) containing inosine (2 g/dl i and sodium pyrophosphate (10 g/dl ) was added to microbial ceils harvested from culture, and the reaction mixture was incubated at 30 °C for 16 hours. Production of 5'-inosinic acid was detected by HPLC analysis to select microbial s~trains having the transphosphorylation activity.  As a result, we succeeded in obtaining 5 strains of transformants which exhibited the transphosphorylation activity and which were assumed to harbor a DNA fragment containing the objective acid phosphatase gene.<br>
Example 8: Determination of Nucleotide Sequence of<br>
Acid Phosphatase Gene Derived from<br>
Morqanella morqanii NCIMB 10466<br>
The inserted DNA fragment was analyzed by preparing<br><br>
a plasmid in accordance with an alkaline lysis method (Molecular Cloning 2nd edition (J. Sambrook, E. F. Fritsch and T. Maniatis, Cold Spring Harbour Laboratory Press, pi. 25 (1989)) from one strain of the transformants which were assumed to harbor the DNA fragment containing the acid phosphatase gene derived from Morqanella morqanii NCIMB 10466 obtained in Example 7.  This plasmid was designated as pMPl501.  Fig. 3 shows a determined restriction enzyme map of the inserted DNA fragment.<br>
The region of the acid phosphatase gene was further specified by subcloning.  As a result, it was suggested that this acid phosphatase gene was contained in a fragment having a size of 1.2 Kbp excised by restriction enzymes HindiII and EcoRI.  Thus in order to determine the nucleotide sequence, plasmid DNA was constructed in which the fragment of 1.2 kbp was ligated with pUC118 having been digested with Hindlll and EcoRI. Escherichia coli JM109 (produced by Takara Shuzo) was transformed with this plasmid DNA designated as pMPl505 in accordance with an ordinary method, which was plated on an L agar medium containing 100 ug/ml of ampicillin to obtain a transformant.<br>
The plasmid was extracted in accordance with the alkaline lysis method from the transformant of Escherichia coli JM109 (produced by Takara Shuzo) harboring pMPl505 to determine the nucleotide sequence.<br><br>
The nucleotide sequence was determined in accordance with a method of Sanger (J. Mol. Biol., 143, 161 (1980)) by using Taq DyeDeoxy Terminator Cycle Sequencing Kit (produced by Applied Biochemical).  A nucleotide sequence of a determined open reading frame is shown in SEQ ID NO: 2 in Sequence Listing.  An amino acid sequence of the protein deduced from the nucleotide sequence is shown in SEQ ID NO: 3 in Sequence Listing. A partial sequence, which was completely coincident with the N-terminal amino acid sequence of the purified enzyme, was found in the amino acid sequence.  The N-terminal of the purified enzyme starts from a 21th alanine residue of the sequence shown in SEQ ID NO: 3. Accordingly, it is assumed that the amino acid sequence shown in SEQ ID NO: 3 is that of a precursor protein, and that a peptide comprising a range from the 1st methionine residue to the 20th alanine residue is eliminated after translation.  An amino acid sequence of a mature protein thus deduced is shown in SEQ ID NO: 4 in Sequence Listing.  A molecular weight of the mature protein estimated from the amino acid sequence is calculated to be 24.9 kilodaltons, which is well coincident with the result of SDS-PAGE for the purified enzyme.  According to the results described above, and because of the fact that the transformant harboring the plasmid containing this fragment exhibited the transphosphorylation activity, it was identified that<br><br>
this open reading frame was the region coding for the objective acid phosphatase.<br>
The nucleotide seguence and the amino acid sequence were respectively compared with known sequences for homology.  Data bases of EMBL and SWISS-PROT were used. As a result, it has been revealed that the nucleotide sequence shown in SEQ ID NO: 2 in Sequence Listing is coincident with a nucleotide sequence of a known acid phosphatase gene derived from Morqanella morqanii (Thaller, M. C. et al., Microbiology, 140, 1341 (1994)) except that 54th G is A, 72th G is A, 276th T is G, 378th T is C, 420th G is T, 525th C is G, 529th C is T, and 531th G is A in the latter, and that the amino acid sequence shown in SEQ ID NO: 4 in Sequence Listing is the same as that of the acid phosphatase gene derived from Morqanella morqanii.  Namely, the gene, which codes for the protein comprising the amino acid seguence shown in SEQ ID NO: 4 in Sequence Listing, is the acid phosphatase gene of Morqanella morqanii NCIMB 10466.<br>
A precursor protein comprises 249 amino acids, and a molecular weight of the protein deduced from its sequence is 27.0 kilodaltons.<br>
The strain of Escherichia coli JM109 transformed by a plasmid pMPl505, has been designated as AJ13143, which has been internationally deposited on February 23, 1996 in National Institute of Bioscience and Human Technology of Agency of Industrial Science and Technology (postal<br><br>
code: 305, 1-3, Higashi 1-chome, Tsukuba-shi, Ibaraki-ken, Japan) under the provision of the Budapest Treaty, and awarded a deposition number of FERM BP-5422.<br>
Example 9: Amplification of Activity<br>
by Expressing Gene of Acid Phosphatase<br>
Derived from Morqanella morqanii NCIMB 10466<br>
Escherichia coli JM109/pMPI505 constructed in Example 8 was inoculated to an L-medium (50 ml) containing 100 yg/rnl of ampicillin and 1 inM of IPTG, and it was cultivated at 37 °C for 16 hours.  Microbial cells were harvested from iis culture by centrifugation, and they were washed once with physiological saline. The microbial cells were suspended in 100 mM potassium phosphate buffer (5 ml, pH 7.2), and they were disrupted by means of a ultrasonic treatment performed at 4 °C for 20 minutes.  The treated solution was centrifuged to remove an insoluble fraction, and thus a cell-free extract was prepared.<br>
The transphosphorylation activity of the obtained cell-free extract was measured while using controls of cell-free extracts prepared from the wild type strain of Morqanella morqanii and Escherichia coli JM109 transformed with the plasmid pUC118 in the same manner as described above.  A result is shown in Table 7.  'i'ne transphosphorylation activity was not detected in<br><br>
Escherichia coli JM109/pUC118.  The transphosphorylation activity was also low in the wild type strain of Morganella morganii.  On the other hand, Escherichia coli JM109/pMPI505 exhibited a high transphosphorylation activity which was 150 times as high as that of the wild type strain of Morganella morganii in sepcific activity. According to the result, it has been demonstrated that the introduced DNA fragment allows Escherichia coli to express the acid phosphatase at a high level.<br>
Table 7<br>
Microbial strain	Transphosphrylation<br>
Activity (units/mg) Morganella morganii NCIMB 10466     0.008 Escherichia coli JM109/pUC118     not detected Escherichia coli JM109/pMPI505       1.250<br>
Example 10: Production of 5'-Inosinic Acid from Inosine<br>
by Using Strain Harboring Acid Phosphatase Gene<br>
Derived from Morganella morganii NCIMB 10466<br>
Sodium pyrophosphate (12 g/dl) and inosine (6 g/dl) were dissolved in 100 mM sodium acetate buffer (pH 4.0), to which the microbial cells of Escherichia coli JM109/pMPI505 described above were added to give a cell<br><br>
concentration of 100 mg/dl as converted into a dry weight of the microbial cells.  The reaction mixture was incubated at 30 °C for 6 hours while maintaining pH at 4.0, and the amount of produced 5'-inosinic acid was measured along with passage of time.  Produced inosinic acid contained only 5'-inosinic acid.  By-production of 2'-inosinic acid and 3'-inosinic acid was not observed at all.  A result is shown in Fig. 4.  The stain harboring the acid phosphatase gene expressed a considerable amount of the acid phosphatase., and 5:-inosinic acid was produced and accumulated extremely efficiently in a short period of time in the reaction to produce 5'-inosinic acid from pyrophosphate and inosine by using this microorganism.  However, when the reaction time is prolonged, it was observed that the produced and accumulated 5'-inosinic acid was decreased due to degradation.<br>
Example 11: Preparation of Phosphomonoesterase activity-Lowered Type Acid Phosphatase G&amp;ne<br>
As described in Examples 9 and 10, the strain harboring the acid phosphatase gene expresses a considerable amount of the acid phosphatase, and 5'-inosinic acid is produced and accumulated extremely efficiently in a short period of time in T;he reaction to produce 5'-inosinic acid from pyrophosphate and inosine<br><br>
by using this microorganism.  However, it has been revealed that the accumulated amount of 5'-inosinic acid does not exceed a certain degree because produced 5'-inosinic acid undergoes degradation by the phosphomonoesterase activity possessed by the acid phosphatase itself.  Thus the enzyme was improved by introducing mutation into the acid phosphatase gene derived from Morqanella morqanii NCIMB 10466 cloned in Example 7, in accordance with the site-directed mutagenesis method by using PCR.<br>
Oligonucleotides MUT500, MUT510, and MUT520 having sequences shown in SEQ ID NOs: 5, 6, and 7 in Sequence Listing were synthesized respectively in accordance with the phosphoa-midite method by using a DNA synthesizer (Model 394 produced by Applied Biosystems).<br>
The plasmid pMPI505 (1 ng) as a template prepared in Example 8, M13 primer RV (produced by Takara Shuzo) and MUT510 oligonucleotide (each 2.5 umol) as primers, and Taq DNA polymerase (2.5 units, produced by Takara Shuzo) were added to 100 mM Tris-HCl buffer (pH 8.3, 100 ul) containing dATP, dCTP, dGTP, dTTP (each 200 uM), potassium chloride (50 mM), and magnesium chloride (1.5 mM) to perform a PCR reaction in which a cycle comprising periods of 30 seconds at 94 °C, 2 minutes at 55 °C, and 3 minutes at 72 °C was repeated 25 times. The PCR reaction was performed by using Thermal Cycler PJ2000 type (produced by Takara Shuzo).  Also, a PCR<br><br>
reaction was performed in the same manner as described above by using plasmid DNA pMPI505 (1 ng) as a temperate, and M13 primer M4 (produced by Takara Shuzo) and MUT500 oligonucleotide (each 2.5 umol) as primers. Each of the reaction products was purified by gel filtration to remove the primers by using Microspin column S-400 (produced by Pharmacia).<br>
Each of the PCR reaction products (1 ul) was added to 100 mM Tris-HCl buffer (pH 8.3, 95 pi) containing dATP, dCTP, dGTP, dTTP (each 200 uM), potassium chloride (50 mM), and magnesium chloride (1.5 mM), and it was heated at 94 °C for 10 minutes, followed by cooling to 37 CC over 60 minutes.  After that, the temperature was kept at 37 °C for 15 minutes to form a heteroduplex. Taq DNA polymerase (2.5 units) was added thereto to perform a reaction at 72 °C for 3 minutes so that the heteroduplex was completed.  After that, M13 primer RV and M13 primer M4 (each 2.5 umol) were added to this reaction solution to perform a PCR reaction in which a cycle comprising periods of 30 seconds at 94 °C, 2 minutes at 55 °C, and 3 minutes at 72 °C was repeated 10 times.<br>
A product of the second PCR reaction was digested with Hindlll and EcoRI followed by phenol/chloroform extraction and ethanol precipitation.  This DNA fragment was ligated with pUC118 having been digested with Hindlll and EcoRI.  Escherichia coli JM109 (produced by<br><br>
Takara Shuzo) was transformed with obtained plasmid DNA in accordance with an ordinary method, which was plated on an L agar medium containing 100 ug/ml of ampicillin to obtain a transformant.  The plasmid was extracted from the transformant in accordance with the alkaline lysis method to determine its nucleotide sequence, confirming that the objective nucleotide was substituted.  The nucleotide sequence was determined in accordance with a method of Sanger (J. Mol. Biol., 143, 161 (1980)) by using Taq DyeBeoxy Terminator Cycle Sequencing Kit (produced by Applied Biochemical).  Thus a mutant gene coding for a mutant phosphatase was prepared in which the 72th glycine residue (GGT) of the mature protein was substituted with an aspartic acid residue (G*AT).  The plasmid containing this mutant gene was designated as pMPI510.<br>
Further, a mutant gene coding for a mutant phosphatase was prepared in which the 151th isoleucine residue (ATC) of the mature protein was substituted with a threonine residue (A*CC), in accordance with the same procedure as described above by using pMPI505 as a template, and MUT500 and MUT520 oligonucleotides as primers.  The plasmid containing this mutant gene was designated as pMPl520.  Moreover, a mutant gene coding for a mutant phosphatase was prepared in which the 72th glycine residue (GGT) of the mature protein was substituted with an aspartic acid residue (G*AT), and<br>
the 151th isoleucine residue (ATC) of the mature protein was substituted with a threonine residue (A*CC), in accordance with the same procedure as described above by using pMPI510 as a template, and MUT500 and MUT520 oligonucleotides as primers.  The plasmid containing this mutant gene was designated as pMPI530.<br>
Escherichia coli JM109/pMPl510, Escherichia coli JM109/pMPI520, and Escherichia coli JM109/pMPI530 into which the plasmids containing the respective mutant acid phosphatase genes had been introduced, and Escherichia coli JMI09/pMPl505 into which the plasmid containing the wild type acid phosphatase gene had been introduced were inoculated to an L medium (50 ml) containing 100 ug/ml of ampicillin and 1 mM of, IPTG, and they were cultivated at 37 °C   for 16 hours.  Microbial cells were harvested from their culture, and they were washed once with physiological saline.  The microbial cells were suspended in 100 mM potassium phosphate buffer (5 ml, pH 7.0), and were disrupted by means of a ultrasonic treatment performed at 4 °C for 20 minutes.  The treated solutions were centrifuged to remove insoluble fractions, and thus cell-free extracts were prepared. Phosphomonoesterase activities and transphosphorylation acitivities of the obtained cell-fxee extracts were measured at pH 4.0, and they were compared with an "activity of the wild strain.<br>
Table 8 shows the result of mesurement of<br><br>
phosphomonoesterase activities and transphosphorylation acitivities of wild type acid phosphatase and mutant acid phosphatases.  It shows that both of phosphomonoesterase activities and transphosphorylation acitivities of mutants acid phosphatases are lowered as compared with wild type acid phosphatase, and that degrees of decrease of phosphomonoesterase activities are larger than that of transphosphorylation activity, with the result that a ratio of phosphomonoesterase activity to transphosphorylation activity of the mutant acid phosphatase is lowered in comparison with the wild type acid phosphatase.<br>
Table 8<br>
Plasmid	Phosphomonoesterase	Transphosphorylation	Ratio 1]<br>
	activity	activity	(Relative value)<br>
	(units/me)	(units/ma)	<br>
pMPI5()5	5.91	0.625	9.45 (100)<br>
pMPI510	0.59	0.090	6.55 ( 69)<br>
pMPI520	2.24	0.583	3.84 ( 40)<br>
pMPI530	1.07	0.318	3.36 ( 35)<br>
1): Ratio of phosphomonoesterase activities to the activities to produce nucleoside-5'-phosphate ester<br><br>
Example 12: Production of 5'-Inosinic Acid from Inosine<br>
by Using The Strains Horborinq A Gene Encoding The Acid<br>
Phosphatase with Lowered Phosphomonoesterase Activity<br>
Escherichia coli JM109/pMPI510, Escherichia coli JM109/pMPI520, and Escherichia coli JM109/pMPI530 into which the plasmids containing the mutant acid phosphatase genes had been introduced, and Escherichia coli JM109/pMPI505 into which the plasmid containing the wild type acid phosphatase gene had been introduced were inoculated to an L medium (50 ml) containing 100 ug/ml of ampicillin and 1 mM of IPTG, and they were cultivated at 37 °C for 16 hours.<br>
Sodium pyrophosphate (12 g/dl) and inosine (6 g/dl) were dissolved in 100 mM sodium acetate buffer (pH 4.0), to which microbial cells of each of the strains of Escherichia coli obtained by the cultivation described above were added to give a cell concentration of 100 mg/dl as converted into a dry weight of the microbial cells.  The reaction mixture was incubated at 30 °C for 22 hours while maintaining pH at 4.0, and the amount of produced 5'-inosinic acid was measured along with passage of time.  A result is shown in Fig. 5.<br>
In Fig. 5, the axis of ordinate indicates the concentration of S'-inosinic acid (mg/dl), and the axis of abscissa indicates the reaction time (h).  Progress<br><br>
of the reaction is indicated by solid circles for Escherichia coli JM109/pMPI505, solid triangles for Escherichia coli JM109/pMPI510, blanked circles for Escherichia coli JM109/pMPI520, and blanked squares for Escherichia coli JM109/pMPI530, as measured by using the microbial cells of the respective strains.<br>
The velocity of degradation of produced 5'-inosinic acid was decreased in the reaction to produce 5'-inosinic acid from inosine by using the strains harboring a gene encoding the acid phosphatase with lowered phosphomonoesterase activity-  As a result, the yield and the accumulated amount of 5'-inosinic acid were increased.  The highest accumulation of 5'-inosinic acid was exhibited by Escherichia coli JM109/pMPI530 as the strain harboring the mutant acid phosphatase gene in which the 72th glycine residue and the 151th isoleucine residue were substituted with the aspartic acid residue and the threonine residue respectively.<br>
Example 13:  Production of Various Nucleoside-5'-Phosphate Esters by Using The Strains Horborinq A Gene<br>
Encoding	The  Acid.  Phosphatase  with  Lowered<br>
Phosphomonoesterase	Activity<br>
Escherichia coli JM109/pMPI530 into which the plasmid containing the mutant acid phosphatase gene had been introduced was inoculated to an L medium (50 ml)<br><br>
containing 100 ug/ml of ampicillin and 1 mM of IPTG, and it was cultivated at 37 "C for 16 hours.<br>
Sodium pyrophosphate (12 g/dl), and inosine, guanosine, uridine, or cytidine (6 g/dl) as a phosphate group acceptor were dissolved in 100 mM sodium acetate buffer (pH 4.0), to which the microbial cells described above were added to give a cell concentration of 100 mg/dl as converted into a dry weight of the cells.  The reaction mixture was incubated at 30 °C for 22 hours while maintaining pH at 4.0.     Amounts of produced nucleoside-5'-phosphate esters are shown in Table 9. Produced nucleotide contained only nucleoside-5'-phosphate ester.  By-production of nucleoside-2'-phosphate ester and nucleoside-3'-phosphate ester was not observed at all.<br>
Table 9<br>
Nucleoside   Product	Produced amount<br>
(g/dl) Inosine       5'-inosinic acid       10.01 Guanosine    5'-guanylic acid       6.72 Uridine      5'-uridylic acid      11.90 Cytidine     5'-cytidylic acid      7.82<br><br>
Example 14: Production of 5'-Inosinic Acid from Various<br>
Phosphate Compounds as Phosphate Group Donors by Using The Strains Horborinq A Gene Encoding The Acid Phosphatase with Lowered Phosphomonoesterase Activity<br>
Escherichia coli JMl09/pMPI530 into which the d containing the mutant acid phosphatase gene had bv_ .    troduced was inoculated to an L medium (50 ml) contax Lng 100 ug/ml of ampicillin and 1 mM of IPTG, and it was cultivated at 37 °C for 16 hours.<br>
Inosine (6 g/dl) and sodium tripolyphosphate, sodium polyphosphate (trade name: Polygon P, produced by Chiyoda Chemical), disodium phenylphosphate, or disodium carbamyl phosphate (10 -g/dl) as a phosphate group donor were"dissolved in sodium acetate buffer (pH 4.5), to which the microbial cells described above were added to give a cell concentration of 100 mg/dl as converted into a dry weight of the microbial cells.  The reaction mixture was incubated at 30 °C for 22 hours while maintaining pH at 4.0.  The amount of produced 5'-inosinic acid is shown in Table 10.  5'-Inosinic acid was efficiently produced and accumulated by using any of the phosphate group donors.  However, the accumulated amount of 5'-inosinic acid was the highest when polyphosphoric acid was used as the phosphate group donor.<br><br>
Table 10<br>
Phosphate group donor      Produced 5'-inosinic<br>
acid (q/dl)<br>
Sodium tripolyphosphate	8.93<br>
Sodium polyphosphate	11.45<br>
Disodium phenylphosphate	9.62<br>
Disodium carbamyl phosphate       9.89<br>
Example 15:  Isolation of Gene Coding for Acid Phosphatase from Chromosome of Escherichia blattae<br>
(1)  Determination of N-terminal amino acid sequence<br>
The acid phosphatase purified from the cell-free extract of Escherichia blattae JCM 1650 was adsorbed to " DITC membrane (produced by Milligen/Biosearch), and its N-terminal amino acid sequence was determined by using Prosequencer 6625 (produced by Milligen/Biosearch).  An N-terminal amino acid sequence comprising 15 residues shown in SEQ ID NO: 8 in Sequence Listing was determined.<br>
(2)   Isolation of DN'A fragment containing gene coding for acid phosphatase<br>
Chromosomal DNA was extracted from cultivated cells of Escherichia blattae JCM 1650 in accordance with a method of Murray and Thomson (Nucl. Acid Res., 4321, 8 (1980)).  The chromosomal DNA was partially degraded<br><br>
with Sau3AI.  After that, DNA fragments of 3 to 6 kbp were fractionated by means of sucrose density gradient centrifugation.  A plasmid vector pUC118 (produced by Takara Shuzo) was digested with BamHI, which was ligated with the partially degraded chromosomal DNA fragments. DNA ligation was performed by using DNA ligation kit (produced by Takara Shuzo) in accordance with a designated method.  After that, Escherichia coli JM109 (produced by Takara Shuzo) was transformed with an obtained DNA mixture in accordance with an ordinary method.  Transformants were plated on  an L agar medium containing 100 ug/ml of ampicillin, and they were grown to prepare a gene library.<br>
A reaction solution containing 4 mM p--nitrophenylphosphoric acid and 100 mM MES/NaOH buffer (pH 6.5) was poured onto a surface of the agar medium on which the transformants had grown, and the temperature was kept at 30 °C for 15 minutes.  Strains which had expressed the phosphatase activity liberated p-nitrophenol and exhibited a yellow color.  Accordingly, transformants were selected by using this phenomenon as an index.  As a result of screening for a chromosomal gene expression library comprising about 8,000 strains of transformants, 14 strains of transformants which had expressed the phosphatase activity were obtained.<br>
The transformants (14 strains), which had expressed the phosphatase activity, were subjected to single<br><br>
colony isolation.  Single colonies were inoculated to a L-medium (2.5 ml) containing 100 ug/ml of ampicillin, and they were cultivated at 37 °C for 16 hours.  Sodium acetate buffer (100 mM, pH 5.0, 50 ul) containing inosine (2 g/dl) and sodium pyrophosphate (10 g/dl) was added to microbial cells harvested from culture liquids to perform the reaction at 30 °C for 16 hours. Production of 5'-inosinic acid was detected by HPLC analysis to select strains having the transphosphorylation activity.  As a result, we succeeded in obtaining 3 strains of transformants which exhibited the transphosphorylation activity and which were assumed to harbor a DNA fragment containing the objective acid phosphatase gene.<br>
Example 16: Determination of Nucleotide Sequence of<br>
Acid Phosphatase Gene Derived from<br>
Escherichia blattae JCM 1650<br>
The inserted DNA fragment was analyzed by extracting a plasmid in accordance with the alkaline lysis method from one strain of the transformants which were assumed to harbor the DNA fragment containing the acid phosphatase gene derived from Escherichia blattae JCM 1650 obtained in Example 15.  This plasmid was designated as pEPl301.  Fig. 6 shows a determined restriction enzyme map of the inserted DNA fragment.<br><br>
The region of the acid phosphatase gene was further specified by subcloning.  As a result, it was suggested that this acid phosphatase gene was included in a fragment having a size of 2.4 Kbp excised by restriction enzymes Clal and BamHI.  Thus in order to determine the nucleotide sequence, plasmid DNA was constructed in which the fragment was ligated with pBluescript KS(+) (produced by Stratagene) having been digested with Clal and BamHI.  Escherichia coli JM109 (produced by Takara Shuzo) was transformed with tne plasrnid DNA designated as pEPI305 in accordance with an ordinary method, which was plated on an L agar medium containing 100 ug/ml of ampicillin to obtain a transformant.<br>
The plasmid was extracted in accordance with the alkaline lysis- method from the transformant. of Escherichia coli JM109 (produced by Takara Shuzo) harboring pEPl305 to determine the nucleotide sequence. A nucleotide sequence of a determined open reading frame is shown in SEQ ID NO: 9 in Sequence Listing.  An amino acid sequence of the protein deduced from the nucleotide sequence is shown in SEQ ID NO: 10 in Sequence Listing. A partial sequence, which was completely coincident with the N-terminal amino acid sequence of the purified enzyme, was found in the amino acid sequence.  The N-terminal of the purified enzyme starts from the 19th leucine residue of the sequence shown in SEQ ID NO: 10. Accordingly, it is assumed that the amino acid sequence<br><br>
shown in SEQ ID NO: 10 is that of a precursor protein and that a peptide comprising a range from the 1st methionine residue to the 18th alanine residue is eliminated after translation.  An amino acid sequence of a mature protein thus deduced is shown in SEQ ID NO: 11 in Sequence Listing.  Accordingly, an estimated molecular weight of the mature protein is calculated to be 25.1 kilodaltons, which is well coincident with the result of SDS-PAGE for the purified enzyme.  According to the results described above, and because of the fact that the transformant harboring the plasmid containing this fragment exhibited the transphosphorylatton activity, it was identified that this open reading frame was the region coding for the objective acid phosphatase.<br>
Namely, the gene, which codes for the protein comprising the amino acid sequence shown in SEQ ID NO: 11 in Sequence Listing, is the acid phosphatase gene of Escherichia blattae JCM 1650.<br>
The nucleotide sequence and the amino acid sequence were respectively compared with known sequences for homology.  Data bases of EMBL and SWISS-PROT were used. As a result, it has been revealed that the protein shown in SEQ ID NO: 8 and DNA coding for it are novel.  A precursor protein encoded by this gene comprises 249 amino acids, and a molecular weight of the protein deduced from its sequence is 27.0 kilodaltons.<br><br>
The amino acid sequence was compared with known sequences respectively for homology.  As a result, this protein exhibited a high degree of homology with the acid phosphatase of Providencia stuartii (77.1 %) with the acid phosphatase of Morqanella morqanii in Example 8 (77.1 %), and with acid phosphatase of Salmonella typhimurium (44.3 %).<br>
The strain of Escherichia coli JM109 transformed by a plasmid pEPI305, has been designated as AJ13144, which has been internationally deposited on February 23, 1996 in National Institute cf Bioscience and Human Technology of Agency of Industrial Science and Technology (postal code: 305, 1-3, Higashi 1-chome, Tsukuba-shi, Ibaraki-ken, Japan) under the provision of the Budapest Treaty, and awarded a deposition number of FERM BP-5423.<br>
Example 17: Amplification of Activity by<br>
Expressing Gene of Acid Phosphatase Derived from Escherichia blattae JCM 1650<br>
Escherichia coli JMl09/pEPI305 constructed in Example 16 was inoculated to an L medium (50 ml) containing 100 ug/ml of ampicillin and 1 mM of IPTG, and it was cultivated at 37 °C  for 16 hours.  Microbial cells were harvested from its culture by centrifugation, and they were washed once with physiological saline. The microbial cells were suspended in 100 mM potassium<br><br>
phosphate buffer (5 ml, pH 7.2), and were disrupted by means of a ultrasonic treatment performed at 4 °C for 20 minutes.  The treated solution was centrifuged to remove an insoluble fraction, and thus a cell-free extract was prepared.<br>
The transphosphorylation activity of the obtained cell-free extract was measured while using controls of cell-free extracts prepared from the wild type strain of Escherichia blattae and Escherichia coli JM109 transformed with the plasmid pBluescript KS(+) in the same manner as described above.  A result is shown in Table 11.  The transphosphorylation activity was not detected in Escherichia coli JM109/pBluescript KS(+). The transphosphorylation activity was also low in the wild type strain of Escherichia blattae.  On the other hand, Escherichia coli JM109/pEPI305 exhibited a high transphosphorylation activity which was 120 times as high as that of the wild type strain of Escherichia blattae in sepcific activity.  According to the result, it has been demonstrated that the introduced DNA fragment allows Escherichia coli to express the acid phosphatase at a high level.<br><br>
Table 11<br>
Microbial strain	Transphosphorvlation<br>
Activity<br>
(units/ma)<br>
Escherichia blattae JCM 1650	0.002<br>
Escherichia coli JM109/pBluescript KS(+)  not detected<br>
Escherichia coli JM109/pEPI305	0.264<br>
Example 18: Production of 5'-Inosinic Acid from Inosine<br>
by Using Strain Harboring Acid Phosphatase Gene<br>
Derived from Escherichia blattae JCM  15 50<br>
Sodium pyrophosphate (12 g/dl) and inosine (6 g/dl) were dissolved in 100 mM sodium acetate buffer (pH 4.0), to which the microbial cells of Escherichia coli JM109/pEPI305 described above were added to give a cell concentration of 200 mg/dl as converted into a dry weight of the microbial cells.  The reaction mixture was incubated at 35 °C for 10 hours while maintaining pH at 4.0, and the amount of produced 5'-inosinic acid was measured along with passage of time.  Produced inosinic acid contained only 5'-inosinic acid.  By-production of 2'-inosinic acid and 3'-inosinic acid was not observed at all.  A result is shown in Fig. 7.  5'-Inosinic acid was produced and accumulated extremely efficiently in a short period of time in the reaction to produce 5'-inosinic acid from pyrophosphate and inosine by using<br><br>
this microorganism.<br>
Example 19: Preparation of A Gene Encoding An Acid Phosphatase with lowered Phosphomonoesterase acitivity<br>
As described in Examples 17 and 18, the strain harboring the acid phosphatase gene derived from Escherichia blattae expresses a considerable amount of the acid phosphatase, and 5'-inosinic acid is produced and accumulated extremely efficiently in a short period of time in the reaction to produce 5'-inosinic acid from pyrophosphate and inosine by using this microorganism. However, it has oeen revealed that the accumulated amount of 5'-inosinic acid does not exceed a certain degree because produced 5'-inosinic acid undergoes degradation by the phosphomonoesterase activity possessed by the acid phosphatase itself.  Thus the enzyme was intended to be improved by introducing mutation into the acid phosphatase gene derived from Escherichia blattae cloned in Example 15, in accordance with the site-directed mutagenesis method by using PCR.<br>
Oligonucleotides MUT300, MUT310, and MUT320 shown in SEQ ID NOs: 12, 13, and 14 in Sequence Listing were synthesized respectively in accordance with the phosphoamidite method by using a DNA synthesizer (Model 394 produced by Applied Biosystems).<br>
The plasmid pEP!305 (1 ng) as a template prepared<br><br>
in Example 16, M13 primer RV (produced by Takara Shuzo) and MUT310 oligonucleotide (each 2.5 pmol) as primers, and Taq DNA polymerase (2.5 units, produced by Takara Shuzo) were added to 100 mM Tris-HCl buffer (pH 8.3, 100 pi) containing dATP, dCTP, dGTP, dTTP (each 200 uM), potassium chloride (50 mM), and magnesium chloride (1.5 mM) to perform a PCR reaction in which a cycle comprising periods of 30 seconds at 94 °C, 2 minutes at 55 °C, and 3 minutes at 72 °C was repeated 25 times. The PCR reaction was performed by using Thermal Cycler PJ2000 type (produced by Takara Shuzo).  Also, a PCR reaction was performed in the same manner as described above by using plasmid pEP!305 (1 ng ) as a temperate, and M13 primer.. M3 (produced by Takara Shuzo) and MUT300 oligonucleotide (each 2.5 umol) as primers.  Each of the reaction solutions was purified by gel filtration to remove the primers by using Microspin column S-400 (produced by Pharmacia).<br>
Each of the PCR reaction products (1 pi) was added to 100 mM Tris-HCl buffer (pH 8.3, 9 5 pi) containing dATP, dCTP, dGTP, dTTP (each 200 pM), potassium chloride (50 mM), and magnesium chloride (1.5 mM), and it was heated at 94 °C for 10 minutes, followed by cooling to 37.°C over 60 minutes.  After that, the temperature was kept at 37 °C for 15 minutes to form a heteroduplex. Taq DNA polymerase (2.5 units) was added thereto to perform a reaction at 72 °C for 3 minutes so that the<br><br>
heteroduplex was completed.  After that, M13 primer RV and M13 primer M3 (each 2.5 umol) were added to this reaction solution to perform a PCR reaction in which a cycle comprising periods of 30 seconds at 94 "C, 2 minutes at 55 °C, and 3 minutes at 72 °C was repeated 10 times.<br>
A product of the second PCR reaction was digested with Clal and BamHI followed by phenol/chloroform extraction and ethanol precipitation.  This DNA fragment was ligated with pBluescript KS(+) having beer, digested with Clal and BamHI.  Escherichia coii JMI09 (produced by Takara Shuzo) was transformed with obtained plasmid DNA in accordance with an ordinary method, which was plated on an L agar medium containing 100 ug/ml of ampicillin to obtain a transformant.<br>
The plasmid was extracted from the transformant in accordance with the alkaline lysis method to determine its nucleotide sequence, confirming that the objective nucleotide was substituted.  Thus a mutant gene coding for a mutant phosphatase v/as prepared in which the 74th glycine residue (GGG) of the mature protein was substituted with an aspartic acid residue (G*A*T).  The plasmid containing this mutant gene was designated as PEPI310.<br>
A mutant gene coding for a mutant phosphatase was prepared in which the 153th isoleucine residue (ATC) of the mature protein was substituted with a threonine<br><br>
residue (A*CC), in accordance with the same procedure as described above by using pEPI305 as a template, and MUT300 and MUT320 oligonucleotides as primers.  The plasmid containing this mutant gene was designated as pEPI320.  Further, a mutant gene coding for a mutant phosphatase was prepared in which the 74th glycine residue (GGG) of the mature protein was substituted with an aspartic acid residue (G*A*T), and the 153th isoleucine residue (ATC) of the mature protein was substituted with a threonine residue 'A*CC), in accordance with the same procedure as described above by using pEPl310 as a template, and MUT300 and MUT320 oligonucleotides as primers.  The plasmid containing this mutant gene was designated as pEPl33£K<br>
Escherichia coli JM109/pEPl310,^Escherichia coli JM109/pEPI320, and Escherichia coli JM109/pEPI330 into which the plasmids containing the respective mutant acid phosphatase genes had been introduced, and Escherichia coli JM109/pEPl305 into which the plasmid containing the wild type acid phosphatase gene had been introduced were inoculated to an L medium (50 ml) containing 100 ug/ml of ampicillin and 1 mM of IPTG, and they were cultivated at 37 °C for 16 hours.  Microbial cells were harvested from their culture, and they were washed once with physiological saline.  The microbial cells were<br>
suspended in 100 mM potassium phosphate buffer (5 ml, pH 7.0), and they were disrupted by means of a ultrasonic<br><br>
treatment performed at 4 °C for 20 minutes.  The treated solutions were centrifuged to remove insoluble fractions, and thus cell-free extracts were prepared. Phosphomonoesterase activities and transphosphorylation acitivities of the obtained cell-free extracts were measured at pH 4.0, and they were compared with an activity of the wild strain.<br>
Table 12 shows the result of measurement of phosphomonoesterase activities and transphosphorylation acitivities of wild type acid phosphatase and mutant acid phosphatases.  It shows that both of phosphomonoesterase activities and transphosphorylation acitivities of mutants acid phosphatases are lowered as compared with wild type acid phosphatase, and that degrees of decrease of„ phosphomonoesterase activities are larger than that of transphosphorylation activities, with the result that a ratio of phosphomonoesterase activity to transphosphorylation activity of the mutant acid phosphatase is lowered in comparison with the wild type acid phosphatase.<br><br>
Table 12<br>
Plasmid	Phosphomonoesterase	Transphosphorv lation	Ratio °<br>
	activity	activity	(Relative value)<br>
	(units/me)	(units/me)	<br>
^EPI305	2.38	0.132	18.03 (100)<br>
^pEPI310	0.26	0.019	13.68 ( 76)<br>
pEPI330	S^      0.42	0.070	6.00 ( 33)<br>
i): Ratio ot phosphomonoesterase activities to the activities to produce nucleoside-5'-phosphate ester<br>
Example 20 ^Production of 5'-Inosinic Acid from Inosine<br>
by Using The Strains Horborinq A Gene Encoding The Acid<br>
Phosphatase with Lowered Phosphomonoesterase Activity<br>
Escherichia coli JMlOQ/pEPlSl^ Escherichia coli JM109/pEPl320, and Escherichia coli JM109/pEPl330 into which the plasmids containing the mutant acid phosphatase genes had been introduced, and Escherichia coli JM]09/pEPl305 into which the plasmid containing the wild type acid phosphatase gene had been introduced were inoculated to an L medium (50 ml) containing 100 ug/ml of ampicillin and 1 mM of IPTG, and they were cultivated at 37 °C for 16 houra^<br>
Sodium pyrophosphate (12 g/dl) and inosine (6 g/dl) were dissolved in sodium acetate buffer (pH 4.0), to which microbial cells r' ~-"-»- ~* the strains of<br><br>
Escherichia coli obtained by the cultivation described above were added to give a cell concentration of 200 mg/dl as converted into a dry weight of the microbial cells.  The reaction mixture was incubated at 35 CC for 32 hours while maintaining pH at 4.0, and the amount of produced 5'-inosinic acid was measured along with passage of time.  A result is shown in Fig. 8.<br>
In Fig. 8, the axis of ordinate indicates the concentration of 5'-inosinic acid (mg/dl), and the axis of abscissa indicates the reaction time (h).  Progress of the reaction is indicated by solid circles for Escherichia coli JM109/pEPI305, solid triangles for Escherichia coli JKi09/pEPI310, blanked circles for Escherichia coli JM109/pEPl320, and blanked squares for Escherichia coli JM109/pEPJ!330,. as measured by using the cells of the respective strains.<br>
The velocity of degradation of produced 5'-inosinic acid was decreased in the reaction to produce 5'-inosinic acid from inosine by using the stains harboring the acid phosphatase with lowered phosphomonoesterase activity.  As a result, the yield and the accumulated amount of 5'-inosinic acid were increased.  The highest accumulation of 5'-inosinic acid was exhibited by Escherichia coli JM109/pEPl330 as the strain harboring the mutant acid phosphatase gene in which the 74th glycine residue and the 153th isoleucine residue were substituted with the aspartic acid residue and the<br><br>
threonine residue respectively.<br>
,—r."'~'  	'      '^"^"Example 21: J Production of Various Nucleoside-5'-<br>
Phosphate Esters by Using The Strains Horborinq A Gene<br>
Encoding  The  Acid  Phosphatase  with  Lowered<br>
Phosphomonoesterase Activity<br>
Escherichia coli JM109/pEPl330 into which the plasmid containing the mutant acid phosphatase gene had been introduced was inoculated to an L medium (50 ml) containing 100 ug/ml of ampicillin and 1 mM of IPTG, and it was cultivated at 37 °C for 16 hours.<br>
Sodium pyrophosphate (12 g/dl), and inosine, guanosine, uridine, or cytidine (6 g/dl) as a phosphate group acceptor were dissolved in 100 mM sodium acetate buffer (pH 4.0), to which the microbial ceils described above were added to give a cell concentration of 200 mg/dl as converted into a dry weight of the cells.  The reaction mixture was incubated at 35 °C for 32 hours while maintaining pH at 4.0.  Amounts of produced nucleoside-5'-phosphate esters are shown in Table 13. Produced nucleotide contained only nucleoside-5'-phosphate ester.  By-production of nucleoside-2'-phosphate ester and nucleoside-3'-phosphate ester was not observed at all.<br><br>
Table 13<br>
Nucleoside   Product	Produced amount<br>
(g/dl) Inosine      5'-inosinic acid       7.45 Guanosine    5'-guanylic acid       4.77 Uridine      5'-uridylic acid       8.93 Cytidine     5'-cytidylic acid      6.60<br>
Example 22: Production of 5'-Inosinic Acid from Various Phosphate Compounds as Phosphate Group Donors by Using The Strains Horborinq A Gene Encoding The Acid Phosphatase with Lowered Phosphomonoesterase Activity<br>
Escherichia coli JM109/pEPl330 into which the plasmid containing the mutant acid phosphatase gene had been introduced was inoculated to an L medium (50 ml) containing 100 ug/ml of ampicillin and 1 mM of IPTG, and it was cultivated at 37 °C for 16 hours.<br>
Inosine (6 g/dl) and sodium tripolyphosphate, sodium polyphosphate (trade name: Polygon P, produced by Chiyoda Chemical), disodium phenylphosphate, or disodium carbamyl phosphate (12 g/dl) as a phosphate group donor were dissolved in 100 mM sodium acetate buffer (pH 4.0), to which the microbial cells described above were added to give a cell concentration of 200 mg/dl as converted into a dry weight of the cells.  The reaction mixture was incubated at 35 °C for 32 hours while maintaining pH at 4.0.  The amount of produced 5'-inosinic acid is<br><br>
shown in Table 14.  5'-Inosinic acid was efficiently produced and accumulated by using any of the phosphate group donors.  However, the accumulated amount of 5'-inosinic acid was the highest when polyphosphoric acid was used as the phosphate group donor.<br>
Table 14<br>
Produced 5'-inosinic acid (q/dl) 5.96 8.84 7.60<br>
1      T3 t   • i &gt;~i<br>
Phosphate group donor<br>
Sodium tripoiyphosphate Sodium polyphosphate Disodium phenylphcsphate Disodium carbamyi phosphate<br>
Example 23: Isolation of Acid Phosphatase Gene Derived from  Chromosome  of . Providencia  stuartii  and Determination of Nucleotide Sequence of the Gene<br>
Oligonucleotides, PRP1 and PRP2, having nucleotide sequences illustrated in SEQ ID NO: 15 and 16 in Sequence Listing, respectively, were synthesized.  These oligonucleotides are designed to amplify a gene coding for acid phosphatase of Providencia stuartii on the basis of known nucleotide sequence of the gene coding for acid phosphatase of Providencia stuartii (Database of EMBL Accession number X64820).<br>
Chromosomal DNA was extracted from cultivated<br><br>
microbial cells of Providencia stuartii ATCC 29851 in accordance with a method of Murray and Thomson (Nucl. Acid Res.. 4321, 8 (1980)).  The chromosomal DNA (0.1 ng) as a template, oligonucleotides PRPl and PRP2 (each 2.5 umol) as primers, and Taq DNA polymerase (2.5 units, produced by Takara Shuzo) were added to 100 mM Tris-HCl buffer (pH 8.3, 100 pi) containing dATP, dCTP, dGTP, dTTP (each 200 uM), potassium chloride (50 mM), and magnesium chloride (1.5 mM) to perform a PCR reaction in which a cycle comprising periods of 30 seconds at 94 °C, 2 minutes at 55 "C, and 3 minutes at 72 °C was repeated 30 times.  The reaction solution was subjected to agarose gel electrophoresis, followed by recovering the amplified DNA fragment of about 1 kbp by means of glass powders (made by Takara Shuzo).  The gene fragment was digested with BamHI, which was iigated with pUC118 degiested with BamHI.  The plasmid obtained as described above was designated as pPRPlOO.<br>
Phosphomonoesterase activity  and transphosphorylation activity of Escherichia coli JM109/pPRP100, a transformant to which pPRPlOO was introduced, were mesured.  As a result, the strain showed an activity to transphosphorylate to nucleoside as well as phosphomonoesterase activity.<br>
The plasmid was extracted in accordance with the alkaline lysis method from the transformant of Escherichia coli JM109/pPRPl00 to determine the<br><br>
nucleotide sequence.   A nucleotide sequence of a determined open reading frame and an amino acid sequence of the protein deduced from the nucleotide sequence are shown in SEQ ID NO: 17 and 18 in Sequence Listing.  The nucleotide sequence of the open reading frame is completely coincident with the nucleotide sequence of the known acid phosphatase gene of Providencia stuartii.<br>
Example 24: Isolation of Acid Phosphatase Genes Derived from Chromosomes of Enterobacter aeroqenes, Klebsiella planticola and Serratia ficaria and Determination of Nucleotide Sequences of the Genes<br>
Chromosomal, DNA was extracted from cultivated microbial cells of Enterobacter aeroqenes IFO 12010, Klf.hsi.ella planticola IFO 14939 and Serratia ficaria I AM 13540 in accordance with a method of Murray and Thomson (Nucl. Acid Res., 4321, 8 (1980)).  Then, in accordance with the method described in Example 7, a chromosomal gene expression library comprising about 20,000 transformants of Escherichia coli JM109 was constructed and screened to obtain transformants which showed transphosphorylation activity.  It was considered that each of these transformants harbour the acid phosphatase gene derived from each of the original strains.<br>
Plasmid DNA was extracted from one of the transformants of Escherichia coli which was considered<br><br>
to have the acid phosphatase gene derived from Enterobacter aeroqenes IFO 12010 in accordance with an alkaline lysis method and the inserted DNA of the plasmid was analyzed.  The above plasmid was designated as pENPlOO.   A restriction enzyme map of the inserted DNA derived from  Enterobacter aeroqenes IFO 12010 is shown in Fig. 9.<br>
As a result of specifying the region of acid phosphatase gene by subcloning, it was suggested that the acid phosphatase gene i_s contained in the 1.6 kbp fragment excised by restriction enzymes Sail and Kpnl. Then, the Sail- Kpnl fragment was ligated with pUC118 which was digested with Sail and Kpnl to construct a plasmid.  The resulting plasmid was designated as pENPHO.<br>
According to the procedure as described above, plasmid DNA was extracted from one of the transformants of Escherichia coli which was considered to have the acid phosphatase gene derived from Klebsiella planticola IFO 14939 in accordance with an alkaline lysis method and the insert DNA of the plasmid was analyzed.  The above plasmid was designated as pKLPlOO.   A restriction enzyme map of the inserted DNA derived from Klebsiella planticola IFO 14939 is shown in Fig. 10.<br>
As a result of specifying the region of acid phosphatase gene by subcloning, it was suggested that—, the acid phosphatase gene is contained in the 2.2 kbp<br><br>
fragment excised by restriction enzymes Kpnl andEcoRI. Then, the Kpnl- EcoRI fragment was ligated with pUC118 which was digested with Kpnl and EcoRI to construct a plasmid.  The resulting plasmid was designated as pKLPHO.<br>
Similarly, plasmid DNA was extracted from one of the transformants of Escherichia coli which was considered to have the acid phosphatase gene derived from Serratia ficaria IAM 13540 in accordance with an alkaline lysis method and the inserted DNA of the plasmid was analyzed.  The above plasmid was designated as pSEPlOO.   A restriction enzyme map of the inserted DNA derived from Serratia ficaria IAM 13540 is shown in Fig. 11.<br>
As. a result of specifying the region of acid phosphatase gene by subcloning, it was suggested that the acid phosphatase gene is contained in the 1.4 kbp fragment excised by restriction enzymes Hindlll.  Then, the Hindlll fragment was ligated with pUC118 which was digested with Hindlll to construct a plasmid.  The resulting plasmid was designated as pSEPllO.<br>
Then, the plasmid DNAs were extracted from the transformants, Escherichia coli JM109/pENP110, Escherichia coli JM109/pKLPll0 and Escherichia coli JM109/pSEP110, to which pENPllO pKLPHO and pSEPllO had "Been introduced, respectively, in accordance with an alkaline lysis method.  The nucleotide sequences of<br><br>
inserts of these plasmids were determined in accordance with the method described in Example 8.  The determined nucleotide sequences of open reading frames of the inserts are shown in SEQ ID NO :19 for Enterobacter aeroqenes IFO 12010, in SEQ ID NO: 21 for Klebsiella planticola IFO 14939 and in SEQ ID NO: 23 for Serratia ficaria IAM 13540.  Additionally, the deduced amino acid sequences are shown in SEQ ID NOs: 20, 22 and 24, respectively.  Because of the fact that the transformarrts harboring the plasmids containing these fragments exhibited the transphosphorylation activity, it was identified that these open reading frames were the objective acid phosphatase genes.<br>
The nucleotide sequences and the deduced amino acid sequences were respectively compared with known sequences for homology-  Data bases of EMBL and SWISS-PROT were used.  As a result, it has been revealed that the genes illustrated in SEQ ID NO: 19, 21 and 23 in Sequence Listing are novel genes.  It is assumed that the protein encoded by the gene derived from Enterobacter aeroqenes IFO 12010 comprises 248 amino acid residues, the protein encoded by the gene derived from Klebsiella planticola IFO 14939 comprises 248 amino acid residues and the protein encoded by the gene derived from Serratia ficaria IAM 13540 comprises 244 amino acid residues.  There is a possibility that' these proteins may be precursor proteins like the acid<br><br>
phosphatases derived from Morqanella morqanii and Escherichia blattae.<br>
The amino acid sequences deduced from the nucleotide sequences are shown in Fig. 12 in one-letter together with the deduced amino acid sequence of the acid phosphatase derived from Morqanella morqanii NCIMB 10466 obtained in Example 8, that of Escherichia blattae JCM 1650 obtained in Example 16 and the known amino acid sequence of the acid phosphatase of Providencia stuartii (EMEL Accession number X64820).  Common amino acid residues among all of the amino acids sequences are indicated with asterisks under the sequences in Fig. 12.<br>
As shown in Fig. 12, the amino acid sequences of the acid phosphatases derived from six strains are highly homologous each other and 130 amino acid residues are common among all of the amino acid sequences.  Thus, it is assumed that these acid phosphatases have similar functions.<br>
Example 25: Amplification of Activity by Expressing Gene<br>
of Acid Phosphatase Derived from Enterobacter aeroqenes,<br>
Klebsiella planticola and Serratia ficaria<br>
Escherichia coli JM109/pPRP100 constructed in Example 23, Escherichia coll JM109/pENPllO, Escherichia coli JM109/pKLPllO and Escherichia coli JM109/pSEP110<br><br>
constructed in Example 24 were inoculated to an L-medium (50 ml) containing 100 ug/ml of ampicillin and 1 mM of IPTG, and were cultivated at 37 °C for 16 hours. Microbial cells were harvested from these cultures by centrifugation, and they were washed once with physiological saline.  The microbial cells were suspended in 100 mM potassium phosphate buffer (5 ml, pH 7.0), and they were disrupted by means of a ultrasonic treatment performed at 4 °C for 20 minutes.  The treated solutions were centrifuged to remove an insoluble fraction, and thus cell-free extracts were prepared.<br>
The transphosphorylation activities of the obtained cell-free extracts were measured while using controls of cell-free extracts prepared from Providencia stuartii ATCC 29851, Enterobacter aeroqenes IFO 12010, Klebsiella planticola IFO 14939, Serratia ficaria IAM 13450, and Escherichia coli JM109 transformed with the plasmid pUC118 in the same manner as described above.  Results are shown in Table 15.  The transphosphorylation activities were low in all of the wild type strains. The transphosphorylation activity was not detected in Escherichia coli JM109/pUC118.   On the other hand, the transformants of Escherichia coli JM109 to which the acid phosphatase genes were introduced exhibited high transphosphorylation activities in comparison with wild type strains.  According to the result, it nas ueen demonstrated that each of the introduced DNA fragment<br><br>
allow Escherichia coli to express the acid phosphatase at a high level.<br><br>
Table 15	<br>
Microbial strain	Transphosphrylation<br>
	Activity<br>
	(units/ma)<br>
Providencia stuartii ATCC 29851	0.005<br>
Enterobacter aeroqenes IFO 12010	0.002<br>
Klebsiella Dianticola IFO 14939	0.002<br>
Serratia ficaxia IAM 13450	0.001<br>
Escherichia coli JM109/pUC118	not detected<br>
Escherichia coli JM109/pPRP100	G. 83 3<br>
Escherichia coli JM109/pENFil0	0.301<br>
Escherichia coli JM109/pKLP110	0.253<br>
Escherichia coli JM109/pSEP110	0.123<br>
Industrial Applicability<br>
According to the present invention, nucleoside-5'-phosphate ester can be produced inexpensively and efficiently by allowing the acid phosphatase to act under the condition of pH 3.0 to 5.5 on a nucleoside and a phosphate group donor selected from the group consisting of polyphosphoric acid or a salt thereof, phenylphosphoric acid or a salt thereof, and carbamyl phosphate or a salt thereof.  Especially, nucleoside-5'-phosphate ester can be produced more efficiently by<br><br>
using the acid phosphatase provided by the present invention, the acid phosphatase having the mutation to lower the phosphomonoesterase activity.<br><br>
WE CLAIM :<br>
1.	A method for producing nucleoside-5'—phosphate ester comprising the steps of allowing an acid phosphatase comprising a mutation to lower its phosphomonoesterase activity, as compared to the wild type enzyme, to act under a condition of pH 3.0 to 5.5 on a nucleoside and a phosphate group donor selected from the group consisting of polyphosphoric acid or a salt thereof, phenyiphosphoric acid or a salt thereof, and carbamyl phosphate or salt thereof to produce nucleoside-5'-phosphate ester, and collecting it.<br>
2.	The method for producing nucleoside-5' -phosphate ester as claimed in claim 1, wherein said acid phosphatase comprises an amino acid sequence selected from the group consisting of sequences illustrated in SEQ ID NOs: 4, 11, 18, 20, 22 and 24 in Sequence Listing, and said acid phosphatase has mutation which lowers phosphomonoesterase activity of an acidphosphatase, as compared to the wild type enzyme.<br>
3.	The method for producing nucleoside-5'—phosphate ester as claimed in claim 2, wherein said mutation lowering phosphomonoesterase activity of said acid phosphatase, as compared to the wild type enzyme, is selected from the group consisting of substitutions of amino acid residue corresponding to substitution(s) of the 72nd glycine residue and/or the 151st isoleucine residue with another amino acid in SEQ ID NO: 4 in Sequence Listing.<br>
4.	The method for producing nucleoside—5'-phosphate ester as claimed in claim 3, wherein said mutation lowering phosphomonoesterase activity of said acid phosphatase, as compared to the wild type enzyme, is selected from the group consisting of substitution(s) of the 72nd glycine residue and/or the 151st isoleucine residue with another amino acid in SEQ ID NO: 4 in Sequence Listing, substitution(s) of the 74th glycine residue and/or the 153rd isoleucine residue with another amino acid in SEQ ID NO: 11 in Sequence Listing, substitution(s) of the 92nd<br><br>
glycine residue and/or the 171st isoleucine residue with another amino acid in SEQ ID NO: 18, 20 or 22 in Sequence Listing, and substitution(s) of the 88th glycine residue and/or the 167th isoleucine residue with another amino acid in SEQ ID NO: 24 in Sequence Listing.<br>
5. The method for producing nucleoside-5'-phosphate ester as claimed in claim 1, wherein said acid phosphatase is derived from one of the following bacterial strains:<br>
Morganella morganii NCIMB 10466 Morganella morganii IFO 3168 Morganella morganii IFO 3848 Escherichia blattae JCM1650 Escherichia blattae ATCC 33429 Escherichia blattae ATCC 33430 Providencia stuartii ATCC 29851 Providencia stuartii A TCC 33672 Enterobacter aerogenes IFO 12010 Enterobacter aerogenes IFO 13534 Klebsiella planticola IFO 14939 Klebsiella planticola 1AM1133 Serratia flcaria I AM 13540 Serratia marcescens 1AM12143<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br></j></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4LW1hcy0xOTk2IGFic3RyYWN0LWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">818-mas-1996 abstract-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4LW1hcy0xOTk2IGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">818-mas-1996 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4LW1hcy0xOTk2IGNsYWltcy1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">818-mas-1996 claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4LW1hcy0xOTk2IGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">818-mas-1996 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4LW1hcy0xOTk2IGNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">818-mas-1996 correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4LW1hcy0xOTk2IGNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">818-mas-1996 correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4LW1hcy0xOTk2IGRlc2NyaXB0aW9uIChjb21wbGV0ZSktMS5wZGY=" target="_blank" style="word-wrap:break-word;">818-mas-1996 description (complete)-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4LW1hcy0xOTk2IGRlc2NyaXB0aW9uIChjb21wbGV0ZSktZHVwbGljYXRlLTEucGRm" target="_blank" style="word-wrap:break-word;">818-mas-1996 description (complete)-duplicate-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4LW1hcy0xOTk2IGRlc2NyaXB0aW9uIChjb21wbGV0ZSktZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">818-mas-1996 description (complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4LW1hcy0xOTk2IGRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">818-mas-1996 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4LW1hcy0xOTk2IGRyYXdpbmdzLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">818-mas-1996 drawings-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4LW1hcy0xOTk2IGRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">818-mas-1996 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4LW1hcy0xOTk2IGZvcm0tMTkucGRm" target="_blank" style="word-wrap:break-word;">818-mas-1996 form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4LW1hcy0xOTk2IGZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">818-mas-1996 form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4LW1hcy0xOTk2IGZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">818-mas-1996 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4LW1hcy0xOTk2IGZvcm0tMjkucGRm" target="_blank" style="word-wrap:break-word;">818-mas-1996 form-29.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4LW1hcy0xOTk2IGZvcm0tNC5wZGY=" target="_blank" style="word-wrap:break-word;">818-mas-1996 form-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4LW1hcy0xOTk2IGZvcm0tNi5wZGY=" target="_blank" style="word-wrap:break-word;">818-mas-1996 form-6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4LW1hcy0xOTk2IG90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">818-mas-1996 others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4LW1hcy0xOTk2IHBldGl0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">818-mas-1996 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="227000-a-process-for-the-production-of-high-strength-porous-ceramic-tiles-utilizing-industrial-solid-wastes.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="227002-antithrombotic-compound.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>227001</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>818/MAS/1996</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>07/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>13-Feb-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>31-Dec-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>15-May-1996</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>AJINOMOTO CO., INC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>15-1, KYOBASHI 1-CHOME, CHUO-KU, TOKYO,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>TAKASHI UTAGAWA</td>
											<td>C/O AJINOMOTO CO., INC., CENTRAL RESEARCH LABORATORIES, 1-1, SUZUKI-CHO, KAWASAKI-KU, KAWASKI-SHI, KANAGAWA 210,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>YASUHIRO MIHARA</td>
											<td>C/O AJINOMOTO CO., INC., CENTRAL RESEARCH LABORATORIES, 1-1, SUZUKI-CHO, KAWASAKI-KU, KAWASKI-SHI, KANAGAWA 210,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>HIDEAKI YAMADA</td>
											<td>19-1, MATSUGASAKI KINOMOTO-CHO, SAKYO-KU, KYOTO-SHI, KYOTO,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>YASUHISA ASANO</td>
											<td>3-1-321, TAIKOYAMA 9-CHOME, KOSUGI-CHO, IMIZU-GUN, TOYAMA,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>CO7H19/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>8-94680</td>
									<td>1996-03-26</td>
								    <td>Japan</td>
								</tr>
								<tr>
									<td>2</td>
									<td>7-149781</td>
									<td>1995-05-25</td>
								    <td>Japan</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/227001-method-for-producing-nucleoside-5-phosphate-ester by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:54:51 GMT -->
</html>
