<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/227333-modification-of-a-plant-disease-resistance-gene-specificity-and-method-for-engineering-altered-specificity by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:51:14 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 227333:&quot;MODIFICATION OF A PLANT DISEASE RESISTANCE GENE SPECIFICITY AND METHOD FOR ENGINEERING ALTERED SPECIFICITY</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;MODIFICATION OF A PLANT DISEASE RESISTANCE GENE SPECIFICITY AND METHOD FOR ENGINEERING ALTERED SPECIFICITY</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>. An isolated nucleic acid fragment comprising a nucleic acid sequence encoding an altered Pi-ta resistance polypeptide selected from the group consisting of: (a) a nucleic acid sequence having at least 90% sequence identity when compared to nucleotides 2426-5212 of the nucleic acid sequence of SEQ ID NO:57, wherein the altered Pi-ta resistance polypeptide has an amino acid substitution for alanine at position 918 and confers resistance in a plant transformed therewith against the fungus Magnaporthe comprising in its genome one or more virulent and/or avirulent AVR-Pita alleles, and (b) a nucleic acid sequence encoding an amino acid sequence as set forth in SEQ ID NO:58 wherein the amino acid sequence has a single amino acid substitution at position 918 which confers resistance in a plant transformed therewith against the fungus Magnaporthe comprising in its genome one or more virulent or avirulent AVR-Pita alleles.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM 2<br>
THE PATENTS ACT 1970<br>
[39 OF 1970]<br>
COMPLETE SPECIFICATION<br>
[See Section 10]<br>
"MODIFICATION OF A PLANT DISEASE RESISTANCE GENE SPECIFICITY AND METHOD FOR ENGINEERING ALTERED SPECIFICITY"<br>
E.I. DU PONT DE NEMOURS AND COMPANY, a Corporation organized and existing under the laws of the State of Delaware, U.S.A., of 1007 Market Street, Wilmington, Delaware   19898, United States of America<br>
The following specification particularly describes and ascertain the nature of the invention and the manner in which it is to be performed :-<br><br><br>
This application claims the benefit of U.S. Provisional Application No. 60/248,335, filed November 14, 2000, the entire contents of which are herein -incorporated by reference.<br>
FIELD OF THE INVENTION This invention relates to the preparation and use of an isolated nucleic acid fragment in order to confer a resistance gene mediated defense response in plants against a fungus comprising in its genome virulent and/or avirulent AVR-Pita alleles. Chimeric genes incorporating such fragments or functionally equivalent subfragments thereof and suitable regulatory sequences can be used to create transgenic plants which can produce a resistance gene mediated defense response against a fungus comprising in its genome virulent and/or avirulent AVR-Pita alleles.<br>
BACKGROUND OF THE INVENTION Plants can be damaged by a wide variety of pathogenic organisms including viruses, bacteria, fungi and nematodes. The invasion of a plant by a potential pathogen can result in a range of outcomes: the pathogen can successfully proliferate in the host, causing associated disease symptoms, or its growth can be halted by the host defenses. In some plant-pathogen interactions, the visible hallmark of an active defense response is the so-called hypersensitive response (HR). The HR involves rapid necrosis of cells near the site of the infection and may include the formation of a visible brown fleck or lesion. Pathogens which elicit an HR on a given host are said to be avirulent (AVR) on that host, the host is said to be resistant, and the plant-pathogen interaction is said to be incompatible. Strains " which proliferate and cause disease on a particular host are said to be virulent, in this case the host is said to be susceptible, and the plant-pathogen interaction is said to be compatible.<br>
Genetic analysis has been used to help elucidate the genetic basis-of, plant-pathogen recognition for those cases in which a series of strains (races) of a particular fungal or bacterial pathogen are either virulent or avirulent on a series of cultivars of a particular host species. In many such cases, genetic analysis of both the host and the pathogen revealed that many avirulent fungal and bacterial strains differ from virulent ones by the possession of one or more avirulence ("avr" or "AVR") genes that have corresponding "resistance" (R) genes in the host.<br>
This avirulence gene-resistance gene model is termed the "gene-for-gene\ model (Crute et al. (1985) pp 197-309 in: Mechanisms of Resistance of Plant Disease. R. S. S. Fraser, ed.; Ellingboe, (1981) Annu. Rev. Phytopathol.<br><br>
19:125-143; Flbr, (1971)'/torn/. Rev.Phythopathol. 9:275-296). According to a simple formulation of this model, plant resistance genes encode specific receptors for molecular signals generated by avr genes. Signal transduction pathway(s) then carry the signal to a set of target genes that initiate the host defenses. Despite this simple predictive model, the molecular basis of the avr-resistance gene interaction is still unknown.<br>
The first R-gene cloned was the Hm1 gene from corn (Zea mays), which confers resistance to specific races of the fungal pathogen Cochliobolus carbonum (Johal et al., 1992, Science 258:985-987). Hm1 encodes a reductase that detoxifies a toxin produced by the pathogen. Next to be cloned was the Pto gene from tomato (Lycopersicon pimpinellifollium) (Martin etal., 1993, Science 262:1432-1436; U.S. Patent No. 5,648,599). Pto encodes a serine-threonine protein kinase that confers resistance in tomato to strains of the bacterial pathogen Pseudomonas syringae pv. tomato that express the avrPto avirulence gene. Taking center stage now are R-genes that encode proteins containing leucine-rich-repeats (LRRs) (Jones and Jones, 1997, Adv. Bot Res. Incorp. Adv. Plant Pathol. 24:89-167). Two classes of membrane anchored proteins with extracellular LRRs have been identified. One subclass includes R-gene products that lack a cytoplasmic serine/threonine kinase domain such as the tomato Cf-9 gene for resistance to the fungus Cladosporium fulvum (Jones et al., 1994, Science 266:789-793, WO 95/18230), and the other subclass includes an R-gene product with a cytoplasmic serine/threonine kinase domain, the rice Xa-21 gene for resistance to the bacterial pathogen Xanthomonas oryzae (Song et al., 1995, Science 270:1804-1806; U.S. Patent No. 5,859,339). The largest class of R-genes includes those encoding proteins with cytoplasmic LRRs such as the Arabidopsis R-genes RPS2 (Bent et al., 1994, Science 265:1856-1860; Mindrinosetal., 1994, Cell 78:1089-1099) and RPM1 (Grant etal., 1995, Science 269:843-846). These R-proteins also possess a putative nucleotide binding site (NBS), and either a leucine zipper (LZ) motif or a sequence homologous to the Toll/lnterleukin-1 receptor (TIR). Table 1 has been reproduced in part from Baker et al. (1997, Science 276:726-733) as a concise summary of classes of R-genes cloned to date and examples of cloned genes within each class.<br><br><br>
1This Table has been reproduced in part from Baker et al. (1997, Science 276:726-733). References for each of the listed R-genes can be found in this review article.<br>
Nucleotide binding sites (NBS) are found in many families of proteins that are critical for fundamental eukaryotic cellular functions such as cell growth, differentiation, cytoskeletal organization, vesicle transport, and defense. Key examples include the RAS group, adenosine triphosphatases, elongation factors, and heterotrimeric GTP binding proteins called G-proteins (Saraste et al., 1990, Trends in Biochem, Science 15:430). These proteins have in comnnon the ability to bind ATP or GTP (Traut, 1994, Eur. J. Biochem. 229:9-19).<br>
It has long been hypothesized that the rice blast system represented a classical gene-for-gene system as defined by H. H. Flor (Flor, 1971, Annu. Rev. Phytopathol. 19:125-143). Genetic analyses needed to identify AVR-genes in the rice blast pathogen, Magnaporthe grisea, has been hampered by the low fertility that typifies M. grisea field isolates that infect rice. Genetic crosses between poorly fertile M. grisea rice pathogens and highly fertile M. grisea pathogens of other grasses (such as weeping lovegrass, Eragrostis curvula, and finger millet, Eleusine coracana) have provided laboratory strains of the fungus with the level of sexual fertility required for identifying AVR-genes (Valent et al., 1991, Genetics 127:87-101). Rare fertile rice pathogens have since allowed demonstration of a<br><br>
one-to-one genetic or functional correspondence between blast fungus AVR-genes and particular rice R-genes (Silue et al., 1992, Phytopathology 82:577-580).<br>
Interest in the rice blast pathosystem is keen because rice blast disease, caused world-wide by the fungal pathogen Magnaporthe grisea (Hebert) Barr (anamorph Pyricularia grisea Sacc), continues as the most explosive and potentially damaging disease of the rice crop despite decades of research towards its control. Manipulation of blast resistance genes remains one of the primary targets in all rice breeding programs, as fungal populations evolve to defeat deployed resistance strategies (See The Rice Blast Disease, 1994, ed. Zeigler, Leong and Teng, CAB International, Wallingford).<br>
Commercial fungicide usage to supplement genetic control strategies began around 1915 when rice farmers used inorganic copper-based fungicides (Chapter 29 in The Rice Blast Disease, 1994, ed. Zeigler, Leong and Teng, CAB International, Wallingford). The fungicides used to control blast disease have changed through time, with some compounds, such as the organomercurials used in the 1950s, causing major environmental damage. The control of rice blast with fungicides currently represents a cost of more than $500 million per year to farmers. This expense for blast control is the largest segment of the world rice fungicide market, which totaled $752 million in 1998 (Wood Mackenzie). Expectations are that the disease problems will intensify as the world rice requirements increase by an estimated 1.7% annually between 1990 and 2025 (See The Rice Blast Disease, 1994, ed. Zeigler, Leong and Teng, CAB International, Wallingford). This estimated need for an additional 13 million tons of rough rice per year to feed the growing population must come from intensification of production on decreasing available land. Rice blast disease is favored by agronomic production practices aimed at high yields, and thus the disease will continue, and most likely increase, as a constraint to rice crop yields unless durable genetic resistance against rice blast disease can be engineered into rice. This invention represents an advance towards the long term goal of engineering durable genetic resistance to rice blast by generating novel Pl-ta alleles that have different specificities as regards the spectrum of AVR-Pita gene products they recognize. The fungus M. grisea has a large host range including species of different tribes within the grass family, Triticeae (e.g., wheat), Oryzeae (e.g., rice), Clorideae (e.g., finger millet), Paniceae (e.g., pearl millet), Andropogoneae (e.g., sorghum) and Maydeae (e.g., maize). Molecular analyses have now defined 8 host species-specific subpopulations of M. grisea, each with a restricted set of host species specificities (Reviewed by Vaient, 1997, The Mycota V, Plant Relationships, Carroll/Tudzynski, eds., Springer-Verlag Berlin Heidelberg pp 37-54). Table 2 gives a current view of pathogen subpopulations according to<br><br>
mitochondrial DNA (mtDNA) type. This view is strongly supported by separate analyses of ribosomal DNA (rDNA) polymorphisms (including both Restriction Fragment Length Polymorphism (RFLP) and Internal Transcribed Spacer (ITS) sequences) and of polymorphisms in both repetitive DNAs and single copy sequences.<br>
The pathogens of rice, wheat, finger millet, barley and corn (mtDNA types la-e) appear closely related, while pathogens of Digitaria spp. and Pennisetum spp. (mtDNA types ll-IV) are highly divergent from the previous groups and from each other. However, M. grisea strains throughout this broad host range can cause significant crop damage. This pathogen has been shown to be the main cause of yield loss of finger millet (Eleusine coracana) in Africa, while infections in wheat (Triticum aestivum; Urashima et al., 1993, Plant Disease 77:1211-1216) and pearl millet (Pennisetum glaucum; Hanna et al., 1989, J. Heredity 80:145-147), although less widespread, can be severe under humid weather conditions. The disease has been documented on barley and corn (See refs. In Urashima et al., 1993, Plant Disease 77:1211-1216).<br>
Table 2<br>
	Host Specificities Within Magnaporthe grisea<br>
Subpopulation1	Defining Host Species2	Crops at Risk<br>
la	Oryza sativa	Rice, Barley, Corn<br>
lb	Triticum aestivum	Wheat, Barley<br>
le	Eleusine spp.	Finger Millet<br>
Ic	Eleusine spp.	Finger Millet<br>
lla	Digitaria spp.	<br>
lib	Digitaria spp.	<br>
III	Pennisetum spp.	Pearl Millet<br>
IV	Pennisetum spp.	<br>
1 Designated by mitochondrial-DNA haplotype.<br>
•^Pathogenicity to the "Defining Host Species" appears conserved within the subpopulation. Some hosts, such as barley, are infected by members of two or more subpopulations.<br>
Knowledge of pathogenicity and host specificity for plant pathogenic fungi is not as advanced as for bacterial and viral pathogens, and likewise, less is known about the molecular basis of resistance in cereal crop plants than in dicot crops or in dicot model systems such as Arabidopsis (Baker et al, 1997, Science 276:726-733). Sasaki reported the first results on the inheritance of resistance to<br><br>
rice blast disease from studies begun in Japan in 1917 (Sasaki, 1922, Japanese Genetics, Japan 1, 81-85).<br>
Since this time, over 30 R-genes have been defined through extensive genetic analysis worldwide, and many of these blast resistance genes have been mapped to rice chromosomes (See Refs. In Takahashi,/1965, The Rice Blast Disease, Johns Hopkins Press, Baltimore, 303-329; Causse et al., 1994, Genetics 138:1251-1274). These R-genes include 20 major resistance genes and 10 putative quantitative trait loci (QTLs). Kiyosawa has described 13 major resistance genes with 9 of these genes found as multiple alleles at 3 loci; 5 at the Pi-k locus on chromosome 11, 2 at the Pi-z locus on chromosome 6 and 2 at the Pi-ta locus on chromosome 12 (Kiyosawa, 1984, Rice Genetics Newsletter 1:95-97). Recent studies in Japan (Ise, 1992, International Rice Research Newsletter 17:8-9) and at the International Rice Research Institiute (IRRI) (Mackill et al., 1992, Phytopathology 82:746-749) have produced near isogenic rice lines (NILs) for use as "differential" rice varieties for determining which resistance genes are effective in controlling individual strains of the fungus. The (RRf NfLs, which provide indica differentials for the blast fungus populations in tropical regions, have been analyzed for genetic relationships between their resistance genes and those present in Kiyosawa Differentials (Inukai et al., 1994, Phytopathology 84:1278-1283).<br>
Molecular markers (or "tags") tightly linked to R-genes have utility for efficient introgression and manipulation of those R-genes in breeding programs. By comparing genotypic patterns of near-isogenic lines, their donors, and their recurrent parents, Yu et al. (1987, Phytopathology 77:323-326) were able to identify five restriction fragment length polymorphic (RFLP) markers linked to three blast resistance genes and to map them to rice chromosomes using segregating populations. RFLP markers linked to the R-genes have been reported (Yu et al., 1991, TheorAppI Genet 81:471-476). Molecular cloning of agronomically important R-genes represents a further advance to the ability of researchers to combine R-genes with other input and output traits in key crop varieties.<br>
In the course of the above mentioned investigations on the inheritance of resistance, Sasaki discovered physiological races of the rice blast pathogen by observing that different field isolates of the blast fungus vary in their ability to cause disease on different varieties of rice (Sasaki, 1922, Journal of Plant Protection 9:631-644; Sasaki, 1923, Journal of Plant Protection 10:1-10). Instability, or "breaking down" under field conditions, of major R-gene resistance to the rice blast fungus has resulted in identification of numerous races, or pathotypes, defined according to virulence spectra on differential rice varieties (Chapters 13 and 16 in The Rice Blast Disease, 1994, ed. Zeigler, Leong and Tena. CAB International,<br><br>
Wallingford). Pathogen populations are dynamic in response to deployment of a new resistance gene, sometimes resulting in new races that overcome the resistance gene within one or two years after deployment in the field.<br>
Accordingly, incorporation of diseases resistance (R) genes into crop plants has not achieved durable resistance to highly variable fungal pathogens such as Magnaporthe grisea (Hebert) Barr, the causal agent of the devastating rice blast disease worldwide. (Rossman et al., Commonwealth Mycological Institute, Kew, Surrey, Second Edition, 1985; Rice Blast Disease, Zeigleret al., eds., CAB International, Wallingford, Oxon 0X10 8DE, UK (1994)). In other words, R-gene utility in controlling rice blast disease has been limited by the inherent field variability of the pathogen.<br>
There are a number of virulent AVR-Pita alleles in different strains of M. grisea for which no corresponding R-gene variants have been identified in rice that recognize these alleles. No one heretofore has been able to engineer an R-gene to recognize such alleles. Clearly, an ability to do so would provide a valuable tool to control currently virulent strains of the rice blast fungus and other pathogens. Clearly, researchers have not adequately succeeded in this regard. Applicants' assignee's copending patent application which was filed on June 21,1999 and having Application No. 09/336, 946 (PCT Publication No. WO 00/08162, which was published on February 17, 2000), describes a Pi-ta gene conferring disease resistance. It does not address the need to modify R-genes to increase their utility by altering their specificity with respect to the AVR-Pita alleles which it can recognize in different strains of a fungus.<br>
Wang et al. (1999) Plant J 79:55-64 describe another rice blast resistance gene, Pib, different from Pi-ta,<br>
WO 00/34479, which published on June 15, 2000, describes nucleic acid fragments which encode a different disease resistance protein that confers resistance to M. grisea.<br>
U.S. Patent No. 5,648,599, issued to Tanksley and Martin on July 15,1997, describes an isolated gene fragment from tomato which encodes the Pto serine/threonine kinase, conferring disease resistance to plants by responding to an avirulence gene in a bacterial plant pathogen.<br>
WO 95/28423, which published on October 26, 1995, describes resistance due to the Pseudomonas syringae RPS2 gene family, primers, probes and detection methods. This published international application includes broad claims to genes encoding proteins with particular NH2-terminal motifs, NBS motifs and leucine rich repeats for protecting plants against pathogens. There are some unique features of the Pi-ta protein. The Pi-ta gene product has a unique amino terminus, lacking<br><br>
either the potential leucine zipper motif of the RPS2 gene-product subfamily (Bent et al, 1994, Science 265:1856-1860; Mindrinos et al., 1994, Cell 78:1089-1099) or the Toll/lnterleukin-1 receptor homology encoded by the N gene subfamily (Whitman et al., 1994, Cell 78:1101-1115). Most importantly, the carboxy terminal portion of the Pi~ta gene product is leucine rich, but it does not fit the consensus sequences for ieucine-rich repeats reported for R-gene products (Jones and Jones, 1997, Adv. Bot. Res. Incorp. Adv. Plant Pathol. 24:89-167).<br>
U.S. Patent No. 5,571,706, issued to Baker et al. on November 5, 1996, covers plant virus resistance conferred by the N gene,<br>
U.S. Patent No. 5,859,351, issued to Staskawicz et al. on January 12,1999, describes the PRF protein and nucleic acid sequence, which is involved in disease resistance in tomato.<br>
U.S. Patent No. 5,859,339, issued to Ronald et al. on January 12, 1999, describes the first resistance gene cloned from rice, Xa-21, which encodes an integral membrane protein with both LRR and serine/threonine kinase domains, and confers resistance in rice to bacterial blight.<br>
WO 91/15585 which published on October 17, 1991 and U.S. Patent No. 5,866,776 issued to de Wit et al. on February 2,1999 describe a method for the protection of plants against pathogens using a combination of a pathogen avirulence gene and a corresponding plant resistance gene.<br>
U.S. Patent No. 5,674,993 ('993 patent), issued to Kawasaki et al. on October 7, 1997, describes nucleic acid markers that co-segregate with the rice blast resistance genes Pi-b, Pi-ta and Pi-ta2 and the suggestion that rice blast resistance genes could be isolated and cloned by using these nucleic acid markers. However, no nucleotide sequences are provided for any rice blast resistance genes in the *993 patent. It should be noted that a putative sequence for the Pi-b rice blast resistance gene is now available in Genbank (accession number AB013448).<br>
In addition, Kawasaki et al. have also published two papers. The first paper, Rybka et af., MPMf, 10(4):517-524 (1997), is entitled "High Resolution Mapping of the Indica-Derived Rice Blast Resistance Genes. II. Pi-ta2 and Pi-ta and a Consideration of Their Origin." The sequence for the RAPD primer that is set forth at the top of column 2 on page 519 is not the same as the RAPD primer set forth in SEQ ID NO:2 in the '993 patent. The sequence for the primer in the paper is TCCCCAGCCA. The sequence for the primer in the '993 patent is TCGCCAGCCA. It is not clear which sequence is correct. Notwithstanding this, it is clear that this paper does not set forth any nucleotide sequences for any rice blast genes. The second paper is Nakamura et al., Mol. Gen. Genet. 254:611-62 (1997). This paper describes the construction of an 800-kb contig in the near-centromeric region of the<br><br>
rice blast resistance gene Pi-ta2 using a rice BAC library. Again, no nucleotide sequence for any rice blast genes is disclosed.<br>
Thus, it is believed that no one heretofore has addressed the need to modify R-genes to increase their utility by broadening their specificity with respect to the AVR-Pita alleles. The broadened specificity enables the modified R-gene to recognize different fungal strains.<br>
SUMMARY OF THE INVENTION<br>
This invention relates to an isolated nucleic acid fragment comprising a nucleic acid sequence or subsequence thereof encoding an altered Pi-ta resistance polypeptide wherein the polypeptide has a single amino acid alteration at position 918 which confers a resistance gene mediated defense response against a fungus comprising in its genome virulent and/or avirulent A VR-Pita alleles.<br>
In another aspect, this invention concerns alterations at position 918 which are selected from the group consisting of M, C, I, R, K, N, L and Q.<br>
In still another aspect, this invention concerns chimeric genes comprising the nucleic acid fragment of the invention.<br>
Also of interest are plants comprising in their genome the chimeric genes described herein as well as seeds obtained from such plants.<br>
In an even further aspect, this invention concerns a method of conferring a resistance gene mediated defense response in plants against a fungus comprising in its genome virulent and/or avirulent AVR-Pita alleles in plants which comprises:<br>
(a)	transforming a plant with a chimeric gene of the invention; and<br>
(b)	selecting transformed plants of step (a) which are resistant to a fungus comprising in it genome virulent and/or avirulent A VR-Pita aiieies.<br>
BIOLOGICAL DEPOSIT The fungal strain 0-137 (collected in 1985 at the China National Rice Research Institute in Hangzhou) has been deposited with the American Type Culture Collection (ATCC), 10801 University Boulevard, Manassas, VA 20110-2209, and bears the following designation, accession number and date of deposit. Fungal strain G-213, a pathogen isolated from Digitaria smutsii in Japan, was obtained from the collection of Jean Loup Notteghem, Laboratoire de phytopathologie, Institut de Recherches Agronomiques Tropicales et des Cultures Vivrieres, Centre de Cooperation Internationale en Recherche Agronomique pour le Developpement (CIRAD), BP 5035, 34032 Montpellier Cedex 1, France, and is available Under the name JP34. Rice pathogen strain G-198, also obtained from Jean Loup Notteghem, was originally isolated from barley in Thailand. Rice pathogen strain GUY11, also obtained from Jean Loup Notteghem, is described in Leung et al. (1Q88) Phytopathology 78, 1227-1233. Rice pathogen strain Ina 72 is<br><br>
described in Kiyosawa (1976) SABRAO Journal 8:53-67. All strains have been deposited with the ATCC.<br>
Plasmid pCB2022 which contains sequences of Pi-ta promoter, P/-fa cDNA, linker sequence and In2-1 terminator sequence described in Example 6 has likewise been deposited with the ATCC.<br><br>
Designation	Material M. grisea	Accession Number	Date of Deposit<br>
0-137	<br>
	ATCC 74457	August 3,1998<br>
G-213	M. grisea	PTA-191	June 8,1999<br>
G-198	M. grisea	PTA-190	June 8, 1999<br>
GUY 11	M. grisea	PTA-192	June 8, 1999<br>
Ina 72	M. grisea	PTA-2606	October 18,2000<br>
pCB2022	Plasmid	PTA-2631	October 25, 2000<br>
BRIEF DESCRIPTION OF THE SEQUENCE LISTINGS AND FIGURES<br>
SEQ ID NO:1 sets forth the sequence of plasmid pCB980 which is a 4119 bp plasmid containing sequences of pBluescript SK+ (nucleotides 1-676, 1880-4119), AVR-Pita promoter (nucleotides 677-1157) and AVR-Pita cDNA (nucleotides 1158-1829) from M. grisea strain 0-137.<br>
SEQ ID NO;2 is the 672 nucleotide sequence of an AVR-Pita cDNA from M. grisea strain 0-137.<br>
SEQ ID NO:3 is the predicted amino acid sequence, of the protein encoded by the AVR-Pita cDNA sequence set forth in SEQ ID NO:2.<br>
SEQ ID NO:4 is the 788 nucleotide sequence of an AVR-Pita cDNA from M. grisea strain G-198.<br>
SEQ ID NO:5 is the predicted amino acid sequence of the protein encoded by the AVR-Pita cDNA set forth in SEQ ID NO:4.<br>
SEQ ID N0.6 is a predicted 672 nucleotide sequence of an AVR-Pita cDNA from M. grisea strain GUY 11 based on genomic sequence set forth in SEQ ID NO:59.<br>
SEQ ID NO:7 is the predicted amino acid sequence of the protein encoded by the AVR-Pita cDNA set forth in SEQ ID NO:6.<br>
SEQ ID NO:8 is the 884 nucleotide sequence of an AVR-Pita genomic clone from M. grisea strain Ina 72.<br>
SEQ ID NO:9 is a predicted 675 nucleotide sequence of an AVR-Pita cDNA from M. grisea strain Ina 72 based on genomic sequence set forth in SEQ ID NO:8.<br>
SEQ ID NO:10 is the predicted amino acid sequence of the protein (Ina 72.1) encoded by the predicted AVR-Pita cDNA sequence set forth in SEQ ID NO:9.<br><br>
SEQ ID N0:11 is the 884 nucleotide sequence of a second AVR-Pita genomic clone from M. grisea strain Ina 72.<br>
SEQ ID NO:12 is a predicted 675 nucleotide sequence of an AVR-Pita cDNA from M. grisea strain Ina 72 based on genomic sequence set forth in SEQ ID NO:11. SEQ ID NO:13 is the predicted amino acid sequence of the protein (Ina 72.2) encoded by the predicted AVR-Pita cDNA sequence set forth in SEQ ID NO: 12.<br>
SEQ ID NO: 14 is the 884 nucleotide sequence of an AVR-Pita genomic clone from M. grisea strain G-213.<br>
SEQ ID NO: 15 is a predicted 675 nucleotide sequence of an AVR-Pita cDNA from M. grisea strain G-213 based on genomic sequence set forth in SEQ ID NO:14.<br>
SEQ ID NO:16 is the predicted amino acid sequence of the protein encoded by the predicted AVR-Pita cDNA sequence set forth in SEQ ID NO:15.<br>
SEQ ID NOS:17 and 18 are PCR primers used to amplify AVR-Pita cDNA and genomic DNA from strain 0-137, and an AVR-Pita genomic nucleic acid fragment from strains G-198 and GUY11.<br>
SEQ ID N0S:19 and 20 are PCR primers used to amplify a functional AVR-Pita promoter fragment from strain 0-137.<br>
SEQ ID N0:21 is a PCR primer used along with SEQ ID NO: 18 to amplify A VR-Pita genomic nucleic acid fragments from strain Ina 72.<br>
SEQ ID NOS:22 and 23 are PCR primers used to amplify an AVR-Pita genomic nucleic acid fragment from strain G-213.<br>
SEQ ID NOS:24 and 25 are PCR primers used to amplify an AVR-Pita nucleic acid fragment in the process of constructing pAVR3.<br>
SEQ ID NOS:26 and 27 are PCR primers used to amplify AVR-Pita176, an AVR-Pita nucleic acid fragment that directly encodes the putative mature protease.<br>
SEQ ID NOS:28 and 29 are PCR primers used to amplify AVR-Pita cDNA from strain G-198.<br>
SEQ ID NOS:30 and 31 are PCR primers used to amplify a partial Pi-ta cDNA.<br>
SEQ ID NO:32 is a PCR primer used along with SEQ ID NO:30 to amplify a susceptible Pi-ta nucleic acid fragment from susceptible C101A51 rice.<br>
SEQ ID NOS:33-51 are PCR primers used to modify Pi~ta coding sequence, resulting in an array of plasmids comprising nucleic acid fragments that encode different Pi-ta proteins with all 20 amino acids represented at position 918.<br>
SEQ ID NOS:52 and 53 are PCR primers used to amplify the Pi-ta leucine' rich domain (LRD).<br>
SEQ ID NOS:54 and 55 are PCR primers used to amplify the In2-1 terminator sequence.<br><br>
SEQ ID NO:56 is the 5757 nucleotide sequence of the genomic clone of the Pi-ta gene from Oryza sativa variety Yashiro-mochi.<br>
SEQ ID NO:57 is the 5222 nucleotide sequence of an EcoR\-Hindlll fragment that contains 2425 bp of the native Pi-ta promoter (nucleotides 1 to 2425) and Pi-ta cDNA (nucleotides 2426-5212) from rice variety Yashiro-mochi.<br>
SEQ ID NO:58 is the predicted P/-fa protein sequence encoded by the Pi-ta nucleotide sequences set forth in SEQ ID NOS:56 and 57.<br>
SEQ ID NO:59 is the 881 nucleotide sequence of an AVR-Pita genomic done from M. grisea strain GUY11.<br>
SEQ ID NO.60 sets forth the sequence of the insert in plasmid pCB2022 which contains sequences of Pi-ta promoter (nucleotides 1-242$), Pi-ta cDNA (nucleotides 2426-5212), (inker sequence (nucleotides 5213-5241) and In2-1 terminator sequence (nucleotides 5242-5690).<br>
Figure 1. Comparison of the deduced amino acid sequences encoded by avirulent and virulent AVR-Pita nucleic acid fragments. The AVR-Pita nucleic acid fragments from avirulent strains 0-137 and G-213, and the first sequence from strain Ina 72 (Ina 72.1) have been shown to confer avirulence (Example 5). The nucleic acid fragment encoding the second Ina 72 sequence (Ina 72.2) and the fragments from virulent strains G-198 and Guy11 do not confer avirulence (Example 5). Only differences from the deduced amino acid sequence of AVR-Pita from strain 0-137 are indicated. Identical amino acids are indicated by a dash (-), whereas a gap that is introduced to maximize alignment is indicated by a dot (.). The underlined amino acids indicate the putative protease motif.<br>
Figure 2. Diagram of constructs in plasmids pCB1947, pML63 and pCB1926.<br>
A.	pCB1947 contains a construct of the AVR-Pita isolated nucleic acid<br>
fragment (formerly called AVR2-YAM0) engineered to encode directly the putative<br>
mature protease for the transient expression experiments. This processed form of<br>
the isolated nucleic acid fragment, designated AVR-Pita^, encodes a polypeptide<br>
comprised of amino acids 48 to 223 as set forth in SEQ ID N0:6. An initiation<br>
methionine was added to the construct through the Ncol site used in the cloning<br>
process. The AVR-Pita coding sequence was first amplified by PCR using primers<br>
, YL30 (SEQ ID NO:26) containing an in-frame Afcol site and YL37 (SEQ ID NO:27) containing a Kpn\ site, and cloned into the Nco\-Kpn\ site of pML.142, resulting in vector pCB1947. The maize Adh1-6 intron inserted downstream of the 35S promoter results in enhanced expression in monocots. This intron is described in Mascarenhas etal. (1990) Plant Mol Biol. 15:913-920.<br>
B.	pML63 contains the uidA gene (which encodes the GUS enzyme) operably<br>
linked to the CaMV35S promoter and 3' NOS sequence. DML63 is modified from<br><br>
pMH40 to produce a minimal 3' NOS terminator fragment. pMH 40 is described in WO 98/16650 wifich published on April 23, 1998, the disclosure of whicj^is hereby incorporated by reference. Using standard techniques familiar to those skilled in the art, the 770 base pair terminator sequence contained in pMH40 was replaced with a new 3' NOS terminator sequence comprising nucleotides 1277 to 1556 of the sequence published by Depicker et al. (1982, J. Appl. Genet 1:561-574).<br>
C. pCB1926 contains the Pi-ta cDNA construct that was created by first amplifying a 2.1 kb partial Pi-ta cDNA nucleic acid fragment from first strand cDNA using primers F12-1 (SEQ /D NO:31) and GB67 (SBQ )D NO:30). A synthetic fuli-length cDNA was generated by incorporating a 706 bp Nco\-BamH\ fragment containing the 5' end of the genomic Pi-ta gene from pCB1649, resulting in plasmid pCB1906. A 3.1 kb EcoRI fragment from pCB1649 containing 2425 bp of the native Pi-ta promoter sequences (pPi-ta) and 736 bp of the 5' Pi-ta coding sequence was then inserted into the EcoRI sites of pCB1906 to replace the 736 bp 5' end of the synthetic cDNA nucleic acid fragment, producing pCB1926.<br>
Figure 3. DNA Genomic Blot Analysis of AVR-Pita Copy Number. M. grisea genomic DNA from the indicated strains was isolated as described previously (Sweigard et al., 1995, The Plant Cell 7:1221-1233), digested with Eco Rl, efectrophoretically fractionated through an agarose gel, blotted onto a filter, and probed with an AVR-Pita fragment using standard molecular biology protocols (Sambrook). Since there is no Eco Rl site in the AVR-Pita gene, the number of bands per lane indicates the number of A VR-Pita genes present in that particular M. grisea strain. For example, the leftmost lane shows the result for M. grisea strain G-198 which shows two bands, indicating that G-198 has two AVR-Pita genes. The short horizontal lines along the left border of the figure indicate the location of DNA size markers from a Hindlll digest of X DNA of sizes from top to bottom of 23.1 kb, 9.4 kb, 6.6 kb, 4.4 kb, 2.3, and 2.0 kb.<br>
DESCRIPTION OF THE INVENTION<br>
In the context of this disclosure, a number of terms shall be utilized:<br>
The term "disease resistance gene" means a gene encoding a polypeptide capable of triggering a defense response in a plant cell or plant tissue. The terms "disease resistance gene", "resistance (R) gene" and "R" gene are used interchangeably herein. The resistance that results from a defense response can take several forms. For example, in some genetic backgrounds resistance may take the form of no visible symptoms on inoculated plant tissue, and in others resistance may include small brown spots from which the fungus does not sporulate and does not reinitiate infection. In both cases, the fungal pathogen does not complete its life cycle and disease development is stopped. In some cases disease resistance may<br><br>
take the form of smaller-sized lesions that do not produce the quantity of fungal spores typical of the full disease. •<br>
A "defense response" is a specific defensive reaction produced by a host, e.g., a plant, to combat the presence of an infectious agent or pathogen.<br>
A "Phta resistance gene" is a disease resistance gene encoding a Pi-ta-resistance polypeptide capable of triggering a defense response in a plant cell or plant tissue against a fungal pathogen such as Magnaporthe grisea.<br>
The term "Pi-ta resistance gene mediated defense response" means a defense response due to the production of the polypeptide encoded by the Pi-ta resistance gene and elicited by the presence of a fungal pathogen. The term "resistance gene mediated defense response" means a defense response due to the production of a polypeptide encoded by a resistance gene and elicited by the presence of a fungal pathogen or a fungal pathogen elicitor.<br>
A "fungal pathogen elicitor" is a pathogen signal molecule that is directly or indirectly recognized by a resistance gene product.<br>
A "virulent AVR-Pita allele" is a variant of the AVR-Pita gene whose gene product normally does not elicit a Pi-ta resistance gene-mediated defense response in rice that expresses the functional Pi-ta resistance protein (A918 described herein; SEQ ID NO:58) such as Yashiro-mochi. The avr-pita gene from M. grisea strain G-198 described herein is an example of a virulent AVR-Pita allele.<br>
An "avirulent A VR-Pita allele" is a variant of the AVR -Pita gene whose gene product normally elicits a Pi-ta resistance gene-mediated defense response in rice that expresses the functional Pi-ta resistance protein (A918 described herein; SEQ ID NO:58) such as Yashiro-mochi. The AVR -Pita gene from M. grisea strain 0-137 described herein is an example of an avirulent A VR -Pita allele.<br>
An "AVR-Pita isolated nucleic acid fragment" is a nucleic acid fragment isolated from a pathogen wherein the nucleic acid fragment encodes a polypeptide whose direct or indirect interaction with the Pi-ta resistance protein is responsible for triggering the Pi-ta resistance gene mediated defense response.<br>
An "isolated nucleic acid fragment" is a polymer of RNA or DNA that is single- or double-stranded, optionally containing synthetic, non-natural or altered nucleotide bases. An isolated nucleic acid fragment in the form of a polymer of DNA may be comprised of one or more segments of cDNA, genomic DNA or synthetic DNA.<br>
The terms "subfragment that is functionally equivalent" "functionally equivalent subfragment", "functionally equivalent subsequence" and "subsequence" are used interchangeably herein. These terms refer to a portion or subsequence of an isolated nucleic acid fragment in which the ability to alter qene exoression or<br><br>
produce a certain phenotype is retained whether or not the fragment or subfragment encodes an active enzyme. For example, the fragment or subfragment can be used in the design of chimeric genes to produce the desired phenotype in a transformed plant. Chimeric genes can be designed for use in co-suppression or antisense by linking a nucleic acid fragment or subfragment thereof, whether or not it encodes an active en2yme, in the appropropriate orientation relative to a plant promoter sequence.<br>
The terms "substantially similar" and "corresponding substantially" as used herein refer to nucleic acid fragments wherein changes in one or more nucleotide bases does not affect the ability of the nucleic acid fragment to mediate gene expression or produce a certain phenotype. These terms also refer to modifications of the nucleic acid fragments of the instant invention such as deletion or insertion of one or more nucleotides that do not substantially alter the functional properties of the resulting nucleic acid fragment relative to the initial, unmodified fragment. It is therefore understood, as those skilled in the art will appreciate, that the invention encompasses more than the specific exemplary sequences.<br>
Moreover, the skilled artisan recognizes that substantially similar nucleic acid sequences encompassed by this invention are also defined by their ability to hybridize, under moderately stringent conditions (for example, 0.5 X SSC, 0.1% SDS, 60°C) with the sequences exemplified herein, or to any portion of the nucleotide sequences reported herein and which are functionally equivalent to the isolated nucleic acid fragment of the invention. Preferred substantially similar nucleic acid sequences encompassed by this invention are those sequences that are 80% identical to the nucleic acid fragments reported herein or which are 80% identical to any portion of the nucleotide sequences reported herein. More preferred are nucleic acid fragments which are 90% identical to the nucleic acid sequences reported herein, or which are 90% identical to any portion of the nucleotide sequences reported herein. Most preferred are nucleic acid fragments which are 95% identical to the nucleic acid sequences reported herein, or which are 95% identical to any portion of the nucleotide sequences reported herein. Sequence alignments and percent similarity calculations may be determined using the Megalign program of the LASARGENE bioinformatics computing suite (DNASTAR Inc., Madison, WI). Multiple alignment of the sequences are performed using the Clustal method of alignment (Higgins and Sharp (1989) CABIOS. 5:151-153) with the default parameters (GAP PENALTY=10, GAP LENGTH PENALTY=10). Default parameters for pairwise alignments and calculation of percent identiy of protein sequences using the Clustal method are KTUPLE=1, GAP PENALTY=3, WINDOW=5 and DIAGONALS SAVED=5. For nucleic acids these   parameters are GAP PENALTYIO, GAP LENGTH<br><br>
PENALTY=10J KTUPLE=2, GAP PENALTY=5, WIND0W=4 and DIAGONALS SAVED=4. A "substantial portion" of an amino acid or nucle^e sequence comprises enough of the amino acid sequence of a polypepti*~iJ or the nucleotide sequence of a gene to afford putative identification of that polypeptide or gene, either by manual evaluation of the sequence by one skilled in the art, or by computer-automated sequence comparison and identification using algorithms such as BLAST (Altschul, S. F., et al., (1993; J. Mot. Biol. 275:403-410) and Gapped Blast (Altschul, S. F. et al., (1997) Nucleic Acids Res. 25:3389-3402).<br>
"Gene" refers to a nucleic acid fragment that expresses a functional RNA transcript, such as but not limited to mRNA, rRNA, tRNA, or antisense RNA, including regulatory sequences preceding (5' non-coding sequences) and following (3* non-coding sequences) the coding sequence. "Native gene" refers to a gene as found in nature with its own regulatory sequences. "Chimeric gene" refers to any gene that is not a native gene, comprising at least one regulatory sequence and coding sequence that are not found together in nature. Accordingly, a chimeric gene may comprise regulatory sequences and coding sequences that are derived from different sources, or regulatory sequences and coding sequences derived from the same source, but arranged in a manner different than that found in nature. "Endogenous gene" refers to a native gene in its natural location in the genome of an organism. A "foreign" gene refers to a gene not normally found in the host organism, but that is introduced into the host organism by gene transfer. Foreign genes can comprise native genes inserted into a non-native organism, or chimer genes. A "transgene" is a gene that has been introduced into the genome by a transformation procedure.<br>
"Coding sequence" refers to a DNA sequence that codes for an RNA transcript, and in the case of a gene encoding a polypeptide, a specific amino acid sequence. "Regulatory sequences" refer to nucleotide sequences located upstream (5' non-coding sequences), within, or downstream (3' non-coding sequences) of a coding sequence, and which influence the transcription, RNA processing or stability, or translation of the associated coding sequence. Such sequences can be native or non-native. Regulatory sequences may include, but are not limited to, promoters, translation leader sequences, introns, and polyadenylation recognition sequences.<br>
"Pathogen" refers to an organism or an infectious agent whose infection around or inside the cells of viable plant tissue elicits a disease response.<br>
"Promoter" refers to a DNA sequence capable of controlling the expression of a coding sequence or functional RNA. The promoter sequence consists of proximal and more distal upstream elements, the latter elements often referred to as enhancers. Accordingly, an "enhancer" is a DNA sequence which can stimulate<br><br>
promoter activity and may be an innate element of the promoter or a heterologous element inserted to enhance the level or tissue-specificity of a promoter. Promoters may be derived in their entirety from a native gene, or be composed of different elements derived from different promoters found in nature, or even comprise synthetic DNA segments. It is understood by those skilled in the art that different promoters may direct the expression of a gene in different tissues or cell types, or at different stages of development, or in response to different environmental conditions. Promoters which cause a gene to be expressed in most cell types at most times are commonly referred to as "constitutive promoters". New promoters of various types useful in plant cells are constantly being discovered; numerous examples may be found in the compilation by Okamuro and Goldberg, 1989, Biochemistry of Plants 75:1-82. It is further recognized that since in most cases the exact boundaries of regulatory sequences have not been completely defined, DNA fragments of some variation may have identical promoter activity.<br>
An "intron" is an intervening sequence in a gene that does not encode a portion of the protein sequence. Thus, such sequences are transcribed into RNA but are then excised and are not translated. The term is also used for the excised RNA sequences. An "exon" is a portion of the sequence of a gene that is transcribed and is found in the mature messenger RNA derived from the gene, but is not necessarily a part of the sequence that encodes the final gene product.<br>
The "translation leader sequence" refers to a DNA sequence located between the promoter sequence of a gene and the coding sequence. The translation leader sequence is present in the fully processed mRNA upstream of the translation start sequence. The translation leader sequence may affect processing of the primary transcript to mRNA, mRNA stability or translation efficiency. Examples of translation leader sequences have been described (Turner, R. and Foster, G. D.t 1995, Mol. Biotechnol. 3:225).<br>
The "31 non-coding sequences" refer to DNA sequences located downstream of a coding sequence and include polyadenylation recognition sequences and other sequences encoding regulatory signals capable of affecting mRNA processing or gene expression. The polyadenylation signal is usually characterized by affecting the addition of polyadenylic acid tracts to the 3' end of the mRNA precursor. The use of different 3' non-coding sequences is exemplified by Ingelbrecht et al., (1989) Plant Cell 7:671-680.<br>
"RNA transcript" refers to the product resulting from RNA polymerase-catalyzed transcription of a DNA sequence. When the RNA transcript is a perfect complementary copy of the DNA sequence, it is referred to as the primary transcript or it may be a RNA sequence derived from posttranscriptional processing of the<br><br>
primary transcript and is referred to as the mature RNA. "Messenger RNA (mRNA)" refers to the RNA that is without introns and that can be translated into protein by the cell. "cDNA" refers to a DNA that is complementary to and synthesized from a mRNA template using the enzyme reverse transcriptase. The cDNA can be single-stranded or converted into the double-stranded form using the klenow fragment of DNA polymerase I. "Sense" RNA refers to RNA transcript that includes the mRNA and so can be translated into protein within a cell or in vitro. "Antisense RNA" refers to a RNA transcript that is complementary to all or part of a target primary transcript or mRNA and that blocks the expression of a target gene (U.S. Patent No. 5,107,065). The complementarity of an antisense RNA may be with any part of the specific gene transcript, i.e., at the 5' non-coding sequence, 3' non-coding sequence, introns, or the coding sequence. "Functional RNA" refers to antisense RNA, ribozyme RNA, or other RNA that may not be translated but yet has an effect on cellular processes.<br>
The term "operably linked" refers to the association of nucleic acid sequences on a single nucleic acid fragment so that the function of one is affected by the other. For example, a promoter is operably linked with a coding sequence when it is capable of affecting the expression of that coding sequence (i.e., that the coding sequence is under the transcriptional control of the promoter). Coding sequences can be operably linked to at least one regulatory sequence in sense or antisense orientation.<br>
The term "expression", as used herein, refers to the production of a functional end-product. Expression or overexpression of a gene involves transcription of the gene and translation of the mRNA into a precursor or mature protein. "Antisense inhibition" refers to the production of antisense RNA transcripts capable of suppressing the expression of the target protein. "Overexpression" refers to the production of a gene product in transgenic organisms that exceeds levels of production in normal or non-transformed organisms. "Co-suppression" refers to the production of sense RNA transcripts capable of suppressing the expression of identical or substantially similar foreign or endogenous genes (U.S. Patent No. 5,231,020).<br>
"Altered expression" refers to the production of gene product(s) in transgenic organisms in amounts or proportions that differ significantly from that activity in comparable tissue (organ and of developmental type) from wild-type organisms.<br>
"Mature" protein refers to a post-translationally processed polypeptide; i.e., one from which any pre- or propeptides present in the primary translation product have been removed. "Precursor" protein refers to the primary product of translation<br><br>
of mRNA; i.e., with pre- and propeptides still present. Pre- and propeptides may be but are not limited to intracellular localization signals.<br>
A "chloroplast transit peptide" is an amino acid sequence which is translated in conjunction with a protein and directs the protein to the chloroplast or other plastid types present in the cell in which the protein is made. "Chloroplast transit sequence" refers to a nucleotide sequence that encodes a chloroplast transit peptide. A "signal peptide" is an amino acid sequence which is translated in conjunction with a protein and directs the protein to the secretory system (Chrispeels, J. J., (1991) Ann. Rev. Plant Phys. Plant Mol. Biol. 42:21-53). If the protein is to be directed to a vacuole, a vacuolar targeting signal (supra) can further be added, or if to the endoplasmic reticulum, an endoplasmic reticulum retention signal (supra) may be added. If the protein is to be directed to the nucleus, any signal peptide present should be removed and instead a nuclear localization signal included (Raikhel (1992) Plant Phys. 100:1627-1632).<br>
"Transformation" refers to the transfer of a nucleic acid fragment into the genome of a host organism, resulting in genetically stable inheritance. Host organisms containing the transformed nucleic acid fragments are referred to as "transgenic" organisms. The preferred method of cell transformation of rice, corn and other monocots is the use of particle-accelerated or "gene gun" transformation technology (Klein et al., (1987) Nature (London) 327:70-73; U.S. Patent No. 4,945,050), or an Agrobacterium-mediated method using an appropriate Ti plasmid containing the transgene (Ishida Y. et al., 1996, Nature Biotech. 14:745-750).<br>
Standard recombinant DNA and molecular cloning techniques used herein are well known in the art and are described more fully in Sambrook, J., Fritsch, E.F. and Maniatis, T. Molecular Cloning: A Laboratory Manual; Cold Spring Harbor Laboratory Press: Cold Spring Harbor, 1989 (hereinafter "Sambrook").<br>
"PCR" or "Polymerase Chain Reaction" is a technique for the synthesis of large quantities of specific DNA segments, consists of a series of repetitive cycles (Perkin Elmer Cetus Instruments, Norwalk, CT). Typically, the double stranded DNA is heat denatured, the two primers complementary to the 3' boundaries of the target segment are annealed at low temperature and then extended at an intermediate temperature. One set of these three consecutive steps comprises a cycle.<br>
An "expression construct" as used herein comprises any of the isolated nucleic acid fragments of the invention used either alone or in combination with each other as discussed herein and further may be used in conjunction with a vector or a subfragment thereof. If a vector is used then the choice of vector is dependent<br><br>
upon the method that will be used to transform host plants as is well known to those skilled in the art. For example, a plasmid vector can be used. The skilled artisan is well aware of the genetic elements that must be present on the vector in order to successfully transform, select and propagate host cells comprising any of the isolated nucleic acid fragments of the invention. The skilled artisan will also recognize that different independent transformation events will result in different levels and patterns of expression (Jones et al., (1985) EMBO J. 4:2411-2418; De Almeida et al., (1989) Mol. Gen. Genetics 218:78-86), and thus that multiple events must be screened in order to obtain lines displaying the desired expression level and pattern. Such screening may be accomplished by Southern analysis of DNA, Northern analysis of mRNA expression, Western analysis of protein expression, or phenotypic analysis. The terms "expression construct" and "recombinant expression construct" are used interchangeably herein.<br>
Cloned R-genes can be used to facilitate the construction of crop plants that are resistant to pathogens. In particular, transformation technology can be used to stack multiple single genes into an agronomic germplasm without linked genomic sequences that accompany genes transferred by classical breeding techniques. Cloned R-genes also can be used to overcome the inability to transfer disease resistance genes between plant species by classical breeding.<br>
The present invention concerns an isolated nucleic acid fragment comprising a nucleic acid sequence or subsequence thereof encoding an altered Pi-ta    / resistance polypeptide wherein the polypeptide has a single amino acid alteration at PfjSJt'QD Qlfi^hinh confers a resistance gene mediated defense response against a fungus comprising in its genome virulent and/or avirulent AVR-Pita alleles._jn___^ another aspect, the present invention provides an isolated nucleic acid fragment that has utility in controlling rice blast disease, caused by the fungus Magnaporthe grisea, in rice.<br>
As discussed below, other subpopulations of M. grisea possess AVR-genes that are homologous to that contained in strains that elicit the Pi-ta specific defense response in rice. Demonstrations that AVR-genes which trigger a P/-te-resistance gene-mediated defense response can be present in M. grisea rice pathogens in subpopulation I, in Digitaria pathogens in subpopulation II, and in Pennisetum pathogens in subpopulation III, support broad utility for this gene in controlling M. grisea on the range of graminaceous hosts infected by this fungus.<br>
Thus, it is believed that the isolated nucleic acid fragments of the invention will have utility in controlling diseases caused by a fungus comprising in its genome virulent and/or avirulent AVR-Pita alleles. For example, this invention will have<br><br>
utility in controlling diseases caused by M. grisea on other cereal crops including, but not limited to, wheat, barley, corn, finger millet, sorghum, and pearl miliet.<br>
Alternatively, some virulent avr-pita alleles encode substantially similar proteins (such as SEQ. ID NO:5 for G-198 and SEQ ID NO: 13 for Ina 72.2) to the avirulent AVR-Pita protein and yet fail to trigger a P/-fe-mediated resistance response. In its most extreme form, single amino acid substitutions introduced either through spontaneous mutations, or by in vitro mutagenesis, eliminate the ability of AVR-Pita to trigger a Pz-fe-resistance gene-mediated defense response.<br>
The genetic cross (cross 4360) that identified the PWL2 gene, an AVR-gene controlling host species specificity on weeping lovegrass (Sweigard et a)., 1995, The Plant Cell 7:1221), also segregated for an additional fungal gene that determined the ability of rice pathogens to infect rice variety Yashiro-mochi. This second AVR-gene, AVR-Pita (formerly called AVR2-YAMO for Yashiro-mochi) was inherited from the parental strain, 4224-7-8, and was derived from the Chinese field isolate 0-137 (collected in 1985 at the China National Rice Research Institute in Hangzhou). In each of five complete tetrads derived from cross 4360, four of eight ascospore progeny were able to infect Yashiro-mochi and the others were not. Random spore analysis of cross 4360 and subsequent crosses confirmed segregation of AVR-Pita. Avirulent progeny from cross 4360 frequently produced a few fully pathogenic lesions on Yashiro-mochi. It was speculated that these rare lesions might be due to spontaneous mutations occurring at the AVR-Pita locus. Mutants that had lost function of AVR-Pita were isolated as described in Sweigard et al (1995, The Plant Cell 7:1221). These mutants that were now fully virulent toward Yashiro-mochi retained morphological and fertility characteristics as well as the MGR586 DNA fingerprinting profiles of the presumptive parent. The host specificities of the mutants toward rice varieties with other R-genes were unchanged.<br>
Although dominance is not easy to assess for genes in predominantly haploid fungi like M. grisea, the occurrence of virulent mutants suggested that the expressed form of this AVR gene functions to stop infection of Yashiro-mochi, as predicted by the gene-for-gene hypothesis. The genetic instability of the AVR-Pita gene aided in its cloning. The AVR-gene was found to cosegregate with a cluster of physical markers including the telomeric repeat sequence at the end of a linkage group in the M. grisea RFLP map produced from cross 4360 (Sweigard et al., 1993, Genetic Maps, edited by S.J. O'Brien, Cold Spring Harbor Laboratory, pp 3.112-3.117). Spontaneous mutants that had become virulent on Yashiro-mochi rice showed structural changes in telomeric restriction fragments that mapped with the avirulence gene, suggesting the gene resided within 1 to 2 kb of the tip of the<br><br>
chromosome (Valentand Chumley, 1994, The Rice Blast Disease, edited by Zeigler, Leong and Teng, CAB International, Wallingford). Southern analysis of genomic DNA from wild type avirulent strains and from spontaneous mutants that had acquired deletions at the chromosome end, identified the sizes of the terminal chromosome fragment produced by digestion of genomic DNA with various restriction enzymes. This analysis suggested that the AVR-gene resided within a telomeric 6.5 kb Bglll fragment that corresponded to the chromosome end. Cloning of the corresponding telomeric fragment allowed demonstration that it did indeed contain the AVR-Pita gene, which functioned to transform virulent pathogens of rice cultivar Yashiro-mochi into avirulent strains on Yashiro-mochi.<br>
The AVR-Pita nucleic acid fragment isolated from the Chinese rice pathogen 0-137 encodes a protein with 223 amino acids (SEQ ID N0:3). Amino acids 173-182 form a characteristic motif of a neutral zinc metalloproteinase and natural or in vitro mutation of the motif residues destroys AVR-gene activity, that is, it no longer transforms virulent strains of the pathogen to avirulence on rice variety Yashiro-mochi (Valent and Chumley, 1994, The Rice Blast Disease, edited by Zeigler, Leong and Teng, CAB International, Wallingford). The predicted amino acid sequence has low levels of homology to other metalloproteinases characterized from fungi (Genbank Accession numbers L37524 and S16547). The best characterized secreted fungal metalloprotease, Npll from Aspergillus oryzae, contains a 175 amino acid prepro-region that precedes a 177 amino acid mature region (Tatsumi etal., 1991, Mol. Gen. Genet. 228:97-103). The predicted AVR-Pita amino acid sequence exhibits 35% homology and 29% identity with Npll, with the most significant homology confined to the mature 177 amino acid form of Npll. In addition, alignment of the amino acid sequences of AVR-Pita and Npll showed conservation of the cysteines involved in disulphide bonds in the mature Npll protein. It was anticipated that the AVR-Pita isolated nucleic acid fragment encodes a preproprotein that is processed to a mature metalloprotease containing 176 amino acids. Based on this prediction, an AVR-Pita176 expression construct was engineered to produce directly the putative mature protease for functional analyses.<br>
Functional AVR-Pita nucleic acid fragments have been cloned from M. grisea strains that infect host plants other than rice, and are distantly related to rice pathogens in subpopulation la, including a Digitaria pathogen (JP34, also known as G-213, isolated in Japan) from subpopulation III, and a Pennisetum pathogen (BF17, isolated in Burkina Faso) from subpopulation IV. The AVR-Pita nucleic acid fragment cloned from the Digitaria pathogen (SEQ ID NO:14) corresponds to a translated amino acid sequence (SEQ ID NO:16) with 87.9% similarity and 84.7% identity to the 0-137 AVR-Pita amino acid sequence when compared by the Bestfit<br><br>
algorithm of the University of Wisconsin Computer group package 9.1 (Devereux et al., 1984, Proc Natl Acad Sci USA 12:387-395). The G-213 A VR-Pita (avirulence) nucleic acid fragment has the most divergent sequence identified which retains the ability to transform virulent rice pathogens into avirulent strains that elicit a Pi-ta resistance gene mediated defense response. Conservation of AVR-gene function between distantly related M. grisea strains that infect different grass species suggests that a cloned Pi-ta resistance gene will be effective in controlling the blast fungus on its other host plants, in addition to rice. n"he present invention concerns an isolated nucleic acid fragment comprising a nucleic acid sequence or subsequence thereof encoding an altered Pi-ta resistance polypeptide wherein the polypeptide has a single amino acid alteration at position 918 which confers a     L resistance gene mediated defense response against a fungus comprising in its
More specifically, it has been found that the single amino acid alteration at 'f position 918 can be selected from the group consisting of methionine, cysteine, ^&gt;isoleucine, arginine, lysine, asparagine, leucine, and glutamine (M, C, I, RC'K, N, L, and Q). As is shown in the examples below, this single amino acid alteration in susceptible and resistant forms of the Pi-ta resistance protein correlate with recognition specificity.<br>
Alteration of the amino acid at position 918 makes it possible to generate mutant Pi-ta genes to recognize virulent AVR-Pita alleles for which no R-gene has currently been identified. Clearly, an ability to do so provides a valuable tool to control currently virulent strains of the rice blast fungus and other pathogens.<br>
In another aspect this invention concerns chimeric genes comprising isolated nucleic acid fragments described herein operably linked to at least one regulatory sequence. Also of interest are plants transformed with such chimeric genes and seeds obtained from such plants.<br>
Transgenic plants of the invention can be made using techniques well known to those of ordinary skill in the art, as is dicussed above, which are capable of mounting a resistance gene mediated defense response against a fungus comprising in its genome virulent and/or avirulent AVR-Pita alleles. Introduction of transgenes into plants, i.e., transformation is well known to those skilled in the art. A preferred method of plant cell transformation is the use of particle-accelerated or "gene gun" transformation technology (Klein et al, (1978) Nature (London) 327:70-73; U.S. Patent No. 4,945,050). Examples of plants that can be transformed with such transgenes include, but are not limited to, monocots. Preferably, the monocot is a cereal. Most preferably, the monocot is rice, wheat, barley, corn, finger millet, sorghum, or pearl millet.<br><br>
In still another aspect, this invention concerns a method of conferring a resistance gene mediated defense response in plants against a fungus comprising in its genome virulent and/or avirulent/AVft-P/fa alleles in plants which comprises: (a) transforming a plant with a chimeric gene of the invention; and (b) selecting transformed plants of step (a) which are resistant to a fungus comprising in it genome virulent and/or avirulent AVR-Pita alleles.<br>
EXAMPLES<br>
The present invention is further defined in the following Examples. From the above discussion and these Examples, one skilled in the art can ascertain the essential characteristics of this invention, and without departing from the spirit and scope thereof, can make various changes and modifications of the invention to adapt it to various usages and conditions.<br>
Unless otherwise stated, all parts and percentages are by weight and degrees are Celsius. Techniques in molecular biology were typically performed as described in Sambrook, J., Fritsch, E. F. and Maniatis, T. 1989. Molecular cloning -A Laboratory Manual, 2nd ed. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York.<br>
In some instances, amino acids are indicated by their one-letter designations widely used in the literature. For example A918 indicates that alanine (A) is at position 918 of the amino acid sequence of the protein.<br>
EXAMPLE 1<br>
Pathogen Strain Development for Identification of<br>
the Corresponding R-Gene<br>
An avirulent isolate of the pathogen is necessary to identify a host resistance gene, and the use of new pathogen strains may identify resistance genes that have so far escaped detection. Two fungal strains that show virulence (that is, they fail to elicit an R-gene mediated defense response) toward a rice variety with a particular R-gene are considered to lack the corresponding functional AVR-gene. However, although an avirulent strain of the fungus with a corresponding AVR-gene is necessary for identifying a host resistance gene, when two strains are avirulent toward a particular rice variety, they could have the same AVR-gene, or different AVR-genes triggering different R-gene mediated defense responses. Use of new or uncharacterized fungal strains may identify previously unidentified R-genes. This caution is especially required when working with field isolate rice pathogens such as 0-137, the source of the AVR-Pita gene. In addition, rice blast infection assays are notoriously sensitive to environmental conditions, especially if the rice varieties have some level of general resistance to blast, or if the pathogens used are unstable in<br><br>
pathogenicity and morphology characteristics. Therefore, the use of thoroughly characterized pathogen strains is key to success.<br>
Multiple pathogen resources, including genetic populations segregating for the AVR-Pita gene, facilitated cloning the Pi-ta gene. Progeny strains such as 4360-R-17, 4360-R-27, 4360-R-30, and subsequent generation strain 4375-R-26 (Sweigard et al., 1995, The Plant Cell 7:1221-1233) had improved characteristics for these studies. Because the rice crosses in these experiments had at least two R-genes segregating, Pi-ta and Pi-km, and because pathogen cross 4360 was segregating for AVR-genes corresponding to both known R-genes, there was a need to obtain genetically characterized pathogen strains bearing single AVR-genes. Such strains were needed to identify the effects of individual R-gene/AVR gene interactions on lesion development. There was also the need to maximize phenotypic differences between the resistant and susceptible interactions in order to assure accuracy of scoring the resistance phenotype among the rice progeny in the mapping population. Twenty-eight progeny strains were obtained from cross 4360 that contained AVR-Pita, but which did not contain AVR1-TSUY. It is believed that AVR1-TSUY is the AVR-gene corresponding to the blast resistance gene Pi-km in the japonica variety Tsuyuake (Yamada et al. (1976) Ann. Phytopathol, Soc. Japan 42: 216-219). These 28 progeny were screened for stable ability to produce fully susceptible disease symptoms (Type 4 - Type 5 lesions) on one parental rice variety, and typical resistance responses (Type 0 - Type 1 lesions) on the other rice variety (Valent et al., 1991, Genetics 127:87). M. grisea isolate 4360-R-62 was chosen as a test strain which contained only A VR-Pita. Out of an additional 9 strains screened, 4360-R-67 was chosen as an excellent pathogen that contains only^AVR1-TSUY.<br>
In addition, mutants that have lost AVR-gene function are useful for comparison to the parental strain from which they were derived. For example, strain CP987 (Sweigard et al. (1995) The Plant Cell 7:1221-1233) was derived from strain 4360-R-17 by isolation of a spontaneous virulent mutant. The AVR-Pita gene in CP987 was inactivated by a small deletion. Virulent mutants can easily be obtained from avirulent field isolates or laboratory strains as described in Sweigard et al. (1995) The Plant Cell 7:1221-1233. In the opposite direction of pathogen trait alteration, field isolate Guy11, which is virulent on Yashiro-mochi (Yamada et al. (1976) Ann. Phytopathol. Soc. Japan 42: 216-219), became avirulent on this host when it was transformed with a cloned AVR-Pita gene to produce strain CP3285. Fungal transformation in the rice blast system is routine, as described in Sweigard et al. (1995) Plant Cell 7:1221.<br><br>
EXAMPLE 2 Plant Infection and Evaluation Infection assays were performed as previously described (Valent et a/., 1991, Genetics 127:87). Conidia were collected from cultures grown on oatmeal agar plates by washing with a sterile 0.25% gelatin solution. Five to six individuals from each rice variety were sown in Metro-Mix® potting medium within plastic pots in a growth chamber. Plants were grown on a day cycle of 14 hours of light, 28 degrees and 70 to 85% relative humidity. Night conditions were 22 degrees and 85% relative humidity. Pots with two-week-old plants were placed into plastic bags in order to maintain 95-100% relative humidity required for pathogen penetration, and inoculated with pathogen strains such as but not limited to 4360-R-62 or 4360-R-67 described in Example 1. A four ml aqueous suspension, containing 2.5 X 105 conidia per ml, was sprayed onto the plants using an artist's air brush (Pasha, size 1). The plastic bags containing the inoculated plants were then closed with a       twist-tie, and were incubated in low light conditions at room temperature for<br>
24 hours. After 24 hours, the plants were removed from the bags, and were placed back in the growth chamber. Infection types, i.e., lesion symptoms, were scored 7 days after inoculation. A scale of 0-5 was used to classify infection phenotypes (Valent et al., 1991, Genetics 127:87-101). Lesion types of 0 and 1 were scored as resistant while lesion types 3 to 5 were scored as susceptible.<br>
EXAMPLE 3 Isolation of Nucleic Acid Fragments Encoding Avirulent and Virulent AVR-Pita Alleles Isolation ofAVR-Pita genomic nucleic acid fragments<br>
The AVR-Pita genomic nucleic acid fragments from M. grisea strains 0-137, Ina 72, GUY11, G-198 and G-213 are all believed to contain 3 introns within the AVR-Pita coding sequence. Southern blot analysis determined that strains 0-137 and G-213 contain a single AVR-Pita gene whereas GUY11, G-198 and lira 72 contain two A VR-Pita genes. To efficiently express the A VR-Pita nucleic fragment in rice leaves under the control 0f the CaMV 35S-Adh1 promoter, it was necessary to isolate the cDNA. AVR-Pit® nucleic acid fragments have been cloned and/or sequenced from M. grisea strains 0-137, G-213, GUY11, G-198 and two from Ina 72, and it is possible to use any of them to prepare AVR-Pita176 nucleic acid fragments for the transient expression experiments. Cloning of the AVR-Pita gene from strain Q-137 by cloning a telomere fragment<br>
AVR-Pita was mapped to one end of M. grisea linkage group 2c of the 0-137-derived progeny strain 4224-7^8 (Sweigard et al., 1993. In Genetic Maps, 6th edition, S.J. O'Brien, ed., Cold Spring Harbor Laboratory Press, pp.3.112-3.117). In this<br><br>
analysis, the AVR-Pita gene co-segregated with the telomere repeat sequences at the end of the chromosome. Identification of telomeric DNA fragments was routinely performed. Standard protocols were used for restriction enzyme digestions, RNA and DNA gel blot analysis (Sambrook; Ausubel et al., 1994, Current Protocols in Molecular Biology, Greene Publishing Associates/Wiley Interscience, New York) except as noted below for hybridizations with the oligonucleotide probe. The oligonucleotide [5' - (AACCCT)4 - 3'] was synthesized and radiolabeled by kinase treatment with y-32P-ATP for detection of the hexameric telomere repeat sequence by DNA gel blot analysis. Genomic DNAs were prepared as described (Sweigard et al., 1995, Plant Cell7:1221-1233), digested with restriction enzymes, electrophoresed on 0.7% agarose gels, and blotted to Hybond-N membranes. For hybridization with the telomeric repeat oligonucleotide, membranes were prehybridized in 6xSSPET 5xDenhardts, 0.5% SDS, 100 g/ml ssCT DNA for 2 hrs at 42°C, and hybridized with radiolabeled oligonucleotide overnight at 42°C. Membranes were washed first for 10 minutes in 2xSSPE, 0.1%SDS at room temperature and then for 15 minutes in the same solution at 30-34°C.<br>
Changes in the sizes of telomeric restriction fragments that correlated with a loss of avirulence toward Yashiro-mochi suggested the AVR-Pita gene is located directly adjacent to its linked telomere. Because cosmid libraries were unlikely to contain chromosome ends, we decided to clone this telomere specifically. Genomic DNA from the avirulent parental strain 4224-7-8 (Sweigard et al., 1995, The Plant Cell 7:1221-1233) was first treated with BAL31 nuclease in order to remove any 3' overhang of the G-rich strand and produce a blunt end at the telomere. The genomic DNA was treated with 0.125 units/ml of BAL31 nuclease (New England Biolabs, Inc.) for 50 min at 30°C as described (Richards and Ausubel (1988) Cell 53:127-136). Under these conditions, no visible decrease occurred in the size of the telomeric fragments as determined by DNA gel blot analysis. An enriched fraction of genomic DNA that would contain the telomere fragment was produced based on our deduction that the 6.5 kb-Bglll telomeric fragment that contains AVR-Pita does not contain any Sail sites. The Bal31treated DNA was digested with the restriction enzymes Bglll and Sail and subjected to electrophoresis on a 0.8% low melting agarose gel, and used to create a genomic sub-library of 6- to 7-kb fragments. DNA fragments in the size range of 7- to 8-kb were eluted from the gel and ligated into the BamHI and EcoRV polylinker sites of pBluescript SK(+). The sub-library was screened for telomere-containing clones using the telomeric oligonucleotide, and positive clones were analyzed further. A single clone, designated pCB780, was obtained with the predicted restriction fragments.<br><br>
The AVR-Pita gene was identified in pCB780 by its ability to transform M. grisea virulent strains to avirulence towards rice containing Pi-ta. Fungal transformation for complementation analysis for function in conferring avirulence activity was performed as described (Sweigard et al., 1995, Plant Cell 7:1221-1233). To increase stability of the clone in M. grisea, pCB780 was cut with Kpnl and partially digested with exonuclease 111, resulting in plasmid pCB806 which had 123 bp of the telomeric repeat (almost the entire repeat) deleted. pCB806 was then cut with Ncol and Xbal to delete 4.5 kb of insert DNA, and the remaining insert (1928 bp) and vector was blunted and re-ligated to create pCB813 which was confirmed to contain an intact AVR-Pita gene using the functional assay described above. Nucleotide sequence of the insert in pCB813 (2 kb) was obtained, and used as basis of some of the oligonucleotide primers described herein (e.g., LF2C and LF2D). Isolation of an AVR-Pita cDNA for the 0-137 Allele<br>
AVR-Pita did not appear to be transcribed at detectable levels during axenic growth of the fungus in culture. We therefore subcloned the AVR-Pita genomic coding sequence into a constitutive expression vector, pCB963, under the control of the Aspergillus TrpC promoter and terminator (Staben et al., 1989). Strains transformed with pCB963 and grown in liquid culture expressed A VR-Pita as determined by RNAgel blot analysis using the Dralll - EcoRI fragment from pCB780 as a probe. A cDNA clone obtained from this transformed strain confirmed the positions of the three predicted introns.<br>
Specifically, the constitutive expression vector, pCB963, was produced as follows. The vector pCSN43 (Staben et al. (1989) Fungal Genet Newslett 36:79-81) was first modified to eliminate extra BamHI and Clal sites by deletion of the smaller MIul - Sacl fragment. The AVR-Pita genomic coding sequence was cloned by PCR using oligonucleotides LF2C (SEQ ID NO: 17) and LF2D (SEQ ID NO: 18), designed to place a Clal site (underlined in SEQ ID NO: 17 below) at the start ATG and a BamHI site (underlined in SEQ ID NO:18 below) at the telomere end.<br>
LF2C:   5'-GATCGAATCGATATGC I I I I I lATTCATTATl I I I I ITTC-3'<br>
(SEQIDNO:17) LF2D:   5'- GATCGAGGATCCCCCTCTATTGTTAGATTGACC-3'<br>
(SEQ ID NO: 18)<br>
The coding sequence of HPH in pCSN43 was then removed by digestion with Clal and BamHI and the Clal/BamHI fragment containing the AVR-Pita genomic coding sequence was inserted to produce pCB963.<br><br>
Transgenic fungus containing pCB963 was grown in liquid culture for purification of RNA. The Perkin Elmer Cetus - GeneAmp RNA PCR kit protocol was used to reverse transcribe RNA using random hexamer priming followed by PCR amplification of cDNA using oligonucleotides LF2C (SEQ ID NO: 17) and LF2D (SEQ ID NO:18). The cDNA fragment encoding AVR-Pita was digested with Clal and BamHI and cloned into Clal/BamHI-cut pBluescript SK+ (Stratagene) to produce pCB979.<br>
A 486 bp functional promoter fragment was amplified from the 0-137 genomic DNA done pCB8l3 using primers LF2H (SEQ ID N0:19) and LF12 (SEQ IDNO:20).<br>
LF2H: 5'-MGCATATCGATAAAMTMTGTTAATTGTGCAG-3'      (SEQ ID NO: 19)<br>
LF12: 5'-GCCGAGTCGTTCTGAGGG-3'	(SEQ ID NO:20)<br>
The 672 bp PCR product was end-filled using Klenow polymerase (Sambrook), digested with Clal and cloned into the Clal and Hindi sites of pCB979 to create pCB980. Cloning of AVR-Pita genomic coding sequences from, other fungal strains<br>
Genomic coding sequences were amplified from genomic DNA isolated from strains G-198, Ina 72 and G^213 using primers LF2C (SEQ ID NO: 17), LF2D (SEQ ID NO: 18), LF2C* (SEQ ID N0:21), GB84 (SEQ ID NO:22) and GB85 (SEQ ID NO:23) as shown in Table 3.<br>
LF2C*: S'-GATCGAATCGATATGCTTTTTTATTCATTGTTATTTTTATTTC^'<br>
(SEQIDN0:21)<br>
GB84: 5'-CCCTGGGATCCMCACTMCGTTATTTAACA-3'	(SEQ ID NO:22)<br>
GB85: S'-GCCGCATCGATATGCTTTTTTATTCATTTATATTTTA-S'<br>
(SEQ ID NO:23)<br>
In each case, the 960 bp PCR product obtained was digested with BamHI and Clal. pCB980 was also digested with BamHI and Clal, and the 3394 bp fragment containing both the functional AVR-Pita promoter from strain 0-137 and pBluescript SK+ vector sequences was gel-purified. The digested 945 bp PCR product was then cloned into the 3394 bp vector fragment. This cloning step essentially replaced the 725 bp 0-137 AVR-Pita cDNA fragment with each of the genomic AVR-Pita nucleic acid fragments from G-198, Ina72 and G-213.<br><br>
TABLE 3<br>
Construction of Plasmids Containina AVR-Pita Genomic Fraaments<br>
	From Various M. arisea Strains	<br>
			Resulting<br>
AVR-Pita Allele	Phenotype	Primers used for PCR	Plasmid<br>
G-198	Virulence	LF2C(SEQIDNO:17), LF2D(SEQIDNO:18)	PCB1447<br>
Ina72.l	Avirulehce	LF2D(SEQIDNO:18), LF2C* (SEQ ID NO:21)	PCB2076<br>
Ina72.2	Virulence	LF2D (SEQ ID NO:18), LF2C*(SEQIDNO:21)	pCB2077<br>
G-213	Avirulence	GB84 (SEQ ID NO:22), GB85 (SEQ ID NO:23)	pCB1965<br>
All genomic AVR-Pita nucleic acid fragments isolated thus far are believed to contain the three intron sequences. Predicted splicing of the introns was again confirmed by isolation of a cDNA clone encoding AVR-Pita of G-198 as described below. The cDNA sequence for the G-198 allele is set forth in SEQ ID NO:4. For all other AVR-Pita clones, the three intron sequences may be removed to generate a cDNA clone, using the appropriate oligonucleotides, subcloning and/or PCR techniques well known to those skilled in the art. The AVR-Pita genomic sequences from Ina 72 are set forth in SEQ ID NO;8 (Ina 72.1) and SEQ ID NO:11 (Ina 72.2), and corresponding predicted cDNA sequences derived from these genomic sequences are respectively set forth in SEQ ID NO:9 and SEQ ID NO:12. The genomic sequence of the AVR-Pita nucleic acid fragment from G-213 is set forth in SEQ ID NO:14, and a predicted cDNA sequence of AVR-Pita from G-213 derived from SEQ ID NO: 14 is set forth in SEQ ID NO: 15.<br>
To obtain the sequence of a virulent allele from GUY11, an AVR-Pita nucleic acid fragment was amplified from genomic GUY11 DNA by PCR using primers LF2D (SEQ ID NO:18) and LF2C (SEQ ID NO:17). The PCR product was sequenced directly after purification using a QIAquick™ PCR Purification Kit (Qiagen) The genomic sequence thus obtained is set forth in SEQ ID NO:59.<br>
Figure 1 is an alignment of the AVR-Pita amino acid sequences derived from nucleotide sequences obtained from M. grisea strains 0-137, G-213, Ina 72, G-198, andGUYH.<br><br>
Isolation of AVR-Pita cDNA nucleic acid fragments for expression of the putative mature protease in plants<br>
Infection assays using pathogen strains 0-137 and G-198 were performed as previously described (Valent et a!., 1991, Genetics 127:87) and as recited in Example 2.<br>
pCB980 (SEQ ID N0:1) contains 0-137 AVR-Pita cDNA, which was cloned using the method desribed above.<br>
Plasmid pAVR3 contains nucleotides 139-672 of the AVR-Pita nucleic acid<br>
fragment (SEQ ID N0:2) from M. grisea strain 0-137 encoding the predicted mature<br>
protease plus one additional N-terminal amino acid (AVR-Pita-|77, beginning with<br>
lle-47 of the preproprotein) and a start codon met fused to the 35S/Adh1-6 promoter<br>
in vector pML142. The AVR~Pita nucleic acid fragment was amplified by PCR from<br>
AVR-Pita cDNA using primers AV1 (SEQ ID NO:24) and AV3 (SEQ ID N0.25),<br>
digested with PmJi and Kpnl, blunted with Klenow polymerase and cloned into<br>
pML142 that had also been cut with Pmll and Kpnl and blunted with Klenow<br>
polymerase, resulting in pAVR3.	.<br>
AV1:   S'-GCCGGCACGTGCCATGATTGMCGCTATTCCCAATG-S'<br>
(SEQ ID NO:24)<br>
AV3:   5'-GCCGGGATCCCCCTCTAHGnAGATTGAC-3'	(SEQ ID NO:25)<br>
The coding sequence for the predicted mature protease (beginning with Glu-48 of the preproprotein) was obtained by PCR-amplification from the AVR-Pita nucleic acid fragment (SEQ ID N0:2) in plasmid pAVR3 using oligonucleotides YL30 containing an in-frame Ncol site (SEQ ID N0.26) and YL37 (SEQ ID NO:27).<br>
YL30:   S'-ACAACAAGCCGGCACGTGCCATGGAACGCT-S'	(SEQ ID NO:26)<br>
YL37:   S'-TCCTTCTTTAGGTACCGCTCTCTC-S'	(SEQ ID NO:27)<br>
The PCR fragment was cloned Nco\IKpn\ into pML142 resulting in vector pCB1947. This was done to eliminate the lle-47 codon (att) in pAVR3 and generate AVR-P»ta176. Additional details are described in PCT Publication No. WO 00/08162, the disclosure of which is hereby incorporated by reference.<br>
To isolate AVR-Pita cDNA nucleic acid fragments from strain G-198, plasmid pCB1447 was transformed into M. grisea strain CP987 and transformants containing this plasmid were used to infect susceptible Tsuyuake rice. Nucleic acid fragments comprising the AVR-Pita cDNA coding sequence from strain G-198 was amplified by RT-PCR from mRNA isolated from infected rice leaf tissue 72 hours<br><br>
after inoculation with 2x106 spores ml"1 of M. grisea strain CP3402 (CP987 transformed with pCB1447) using primers GB183 (SEQ ID NO:28) and GB184 (SEQ ID NO:29) containing Ncol and Kpni sites, respectively.<br>
GB183:   5'-GGGCTTCCATGGAACGCTATTCCCAATGTTCAG-3'  (SEQ ID NO:28) GB184:   5'-CACTAAGGTACCTTMCMTATTTATAACGTGCAC-3'<br>
(SEQ ID NO:29)<br>
The PCR product was digested with Ncol and Kpnl and this was cloned into pML142 (described in WO 00/08162). pML142 had also been digested with Ncol and Kpnl and the 5.1 kb fragment isolated and purified from an agarose gel. The digested G-198 PCR nucleic acid fragment was ligated with the 5.1 kb pML142 nucleic acid fragment using techniques familiar to those skilled in the art to generate plasmid pCB2148.<br>
EXAMPLE 4<br>
Mutagenesis of Pi-ta at Position 918 to Create a Library of<br>
Pi-ta Plasmids With All 20 Amino Acids Represented at Position 918<br>
Mutating the Pi-ta gene at codon 918 to create a set of modified Pi-ta genes with each of the 20 amino acids represented at position 918 required a two step process which is described below. Two amino acids were already represented, A918 (alanine at position 918) occurs in resistant forms of Pi-ta while S918 (serine at position 918) occurs in susceptible forms.<br>
A full-length Pi-ta cDNA fragment was cloned using reverse transcriptase (RT) PCR and subcloning. The approach involved isolating mRNA from transgenic Nipponbare line 27-4-8-1 which contained a transgene comprising a genomic Pi-ta nucleic acid fragment from Oryza sativa variety Yashiro-mochi operably linked to the CaMV 35S promoter (described in WO 00/01862, Example 10, expression construct 3) and which was shown on a northern blot to overexpress Pi-ta. First strand cDNA was synthesized using the isolated mRNA fraction as template and the oligonucleotide GB67 (SEQ ID NO:30) as primer.<br>
GB67:5,-CCATTAAGCTTGGTTTCAAACAATC-3'	(SEQ ID NO:30)<br>
A partial Pi-ta cDNA (2.1 kb) was amplified from first strand cDNA using primers F12-1 (SEQ ID NO:31) and GB67 (SEQ ID NO:30).<br>
F12-1:    5'-GTGGCTTCCATTGTTGGATC-3*	(SEQ ID NO:31)<br><br>
It was then cloned into pSL1180 (Pharmacia) using the BamHI (restriction site present in the Pi-ta nucleic acid fragment) and H/ndlll (restriction site present in GB67 sequence) cloning sites. To obtain a full-length synthetic cDNA, a 706 bp Nco\-BamH\ fragment containing the 5' end of the Pi-ta coding sequence was isolated from pCB1649 (described In WO 00/08162) and cloned into the A/col-BamHI site upstream of the 2090 bp BamHI-H/ndlll partial Pi-ta cDNA fragment to create a full-length promoter-less Pi-ta cDNA. DNA sequence analysis determined that there was a 2 bp deletion present at codon 796 (probably a PCR artifact) resulting in a frameshift mutation that would have truncated the predicted Pi-ta protein by 119 amino acids. This was corrected by replacing a 1400 bp $ph\-BgI\\ fragment with the corresponding fragment from pCB1649 which contained the correct sequence, to create pCB1906. DNA sequence analysis also determined that the predicted intron was precisely spliced in this synthetic cDNA; A native Pi-ta promoter fragment (2425 bp) was added by cloning a 3173 bp EcoRI fragment from pCB1649 into the EcoRI sites of pCB1906, resulting in plasmid pCB1926 which contained the final Pi-ta cDNA construct (Figure 2C) comprising 2425 bp of the native Pi-ta promoter (nucleotides 1 to 2425 in SEQ ID NO:57) and P/-fa cDNA (nucleotides 2426-5212 in SEQ ID NO:57) from rice variety Yashiro-mochi. The deduced amino acid sequence of the protein encoded by this particular Pi-ta cDNA is set forth in SEQ ID NO:58.<br>
Plasmid pCB2020 was produced by amplifying a susceptible Pi-ta nucleic acid fragment from susceptible C101A51 rice (Mackill and Bonman (1992) Phytopathology 82:746-749) genomic DNA in a PCR reaction using primers GB61 (SEQ ID NO:32) and GB67 (SEQ ID NO:30).<br>
GB61:     5'-CAATGCCGAGTGTGCAAAGA-3'	(SEQIDNO:32)<br>
The resulting 440 bp PCR fragment was digested with Bglll and Hindlll and cloned into plasmid pCB1926 that had been digested with the same enzymes (and the corresponding 4900 bp nucleic acid fragment isolated and purified from an agarose gel). This cloning step produced plasmid pCB2020 which is identical to pCB1926 except that codon 918 encodes S and not A.<br>
The remaining 18 amino acid modifications were introduced into Pi-ta by first amplifying the fragment from plasmid pCB1926 using PCR primer GB60 (SEQ ID NO:33) and a set of 18 primers GB164-GB179, GB181 and GB182 (SEQ ID NOS:34-51) in 18 separate PCR reactions, one for each primer.<br><br>
GB60:   5'-CAATGCCGAGTGTGCAAAGG-3	(SEQIDNO:33)<br>
GB164:   S'-AGGCGAGTCGACGTTTCAAACAATCATCAAGTCAGGTTGAAGA<br>
TGCATCTCAGGTAAAGATAGAAGC -3'	(SEQ ID NO:34)<br>
GB165:   S'-AGGCGAGTCGACGTTTCAAACMTCATCAAGTCAGGTTGAAGAT<br>
I	GCATATCAGGTAAAGATAGAAGC-3'	(SEQIDNO:35)<br>
GB166:   5'-AGGCGAGTCGACGTTTCAAACMTCATCAAGTCAGGTTGAAGAT<br>
GCATCCTAGGTAAAGATAGAAGC -3'	(SEQ ID NO:36)<br>
GB167:   S'-AGGCGAGTCGACGTTTCAAACAATCATCAAGTCAGGTTGAAGAT<br>
GCATCTTAGGTAAAGATAGAAGC -3'	(SEQ ID NO:37)<br>
i    GB168:   S'-AGGCGAGTCGACGTTTCAAACAATCATCAAGTCAGGTTGAAGAT<br>
GCATATTAGGTAAAGATAGAAGC -3'	(SEQ ID NO:38)<br>
GB169:   S'-AGGCGAGTCGACGTTTCAAACAATCATCAAGTCAGGTTGAAGAT<br>
GCATCATAGGTAAAGATAGAAGC -3*	(SEQ ID NO:39)<br>
GB170:   S'-AGGCGAGTCGACGTTTCAAACMTCATCAAGTCAGGTTGAAGAT<br>
GCATAATAGGTAAAGATAGAAGC -3'	(SEQ ID NO:40)<br>
GB171:   S'-AGGCGAGTCGACGTTTCAAACAATCATCAAGTCAGGTTGAAGAT<br>
GCATTGTAGGTAAAGATAGAAGC -3'	(SEQ ID N0:41)<br>
GB172:   S'-AGGCGAGTCGACGTTTCAAACAATCATCMGTCAGGTTGAAGAT<br>
GCATCCAAGGTAAAGATAGAAGC -3'	(SEQ ID NO:42)<br>
GB173:  5'-AGGCGAGTCGACGTTTCAAACMTCATCMGTCAGGTTGAAGAT<br>
GCATACAAGGTAAAGATAGAAGC -3'	(SEQ ID NO:43)<br>
GB174:   S'-AGGCGAGTCGACGTTTCAAACMTCATCAAGTCAGGTTGAAGAT<br>
GCATGTAAGGTAAAGATAGAAGC -3'	(SEQ ID NO:44)<br>
GB175:  5'-AGGCGAGTCGACGTTTCAAACMTCATCMGTCAGGTTGAAGAT<br>
GCATGAAAGGTAAAGATAGAAGC -3'	(SEQ ID NO:45)<br>
GB176:   5'-AGGCGAGTCGACGTTTCAMCMTCATCMGTCAGGTTGAAGAT<br>
GCATTTGAGGTAAAGATAGAAGC -3'	(SEQ ID NO:46)<br>
GB177:  S'-AGGCGAGTCGACGTTTCAAACAATCATCAAGTCAGGTTGAAGAT<br>
GCATGTGAGGTAAAGATAGAAGC -3'	(SEQ ID NO:47)<br>
GB178:   S'-AGGCGAGTCGACGTTTCAAACAATCATCAAGTCAGGTTGAAGAT<br>
GCATCAGAGGTAAAGATAGAAGC-3'	(SEQIDNO:48)<br>
GB179:   S'-AGGCGAGTCGACGTTTCAAACAATCATCAAGTCAGGTTGAAGAT<br>
GCATCGGAGGTAAAGATAGAAGC -3'	(SEQ ID NO:49)<br>
GB181:   5'-AGGCGAGTCGACGTTTCAAACAATCATCAAGTCAGGTTGAAGAT<br>
GCATGCCAGGTAAAGATAGAAGC-3'	(SEQ ID NO:50)<br>
GB182:  S'-AGGCGAGTCGACGTTTCAAACAATCATCAAGTCAGGTTGMGAT<br>
GCATAACAGGTAAAGATAGAAGC-3'	(SEQ ID N0:51)<br>
The resulting 440 bp PCR nucleic acid fragments were digested with Bglll and Sail and the 322 bp fragment ligated into plasmid pCB2114 (described below) that had been digested with the same enzymes. This replaced the resistant Pi-ta 322 bp Bglll-Sall fragment in plasmid pCB2114 with a nucleic acid fragment containing the altered codon at position 918. This cloning step produced plasmids pCB2153-pCB2170 (Tabled). Plasmid pCB2114 comprises nucleotide sequence encoding<br><br>
GB60:   5'-CAATGCCGAGTGTGCAAAGG-3	(SEQ ID NO:33)<br>
GB164:   S'-AGGCGAGTCGACGTTTCAAACAATCATCAAGTCAGGTTGAAGA<br>
TGCATCTCAGGTAAAGATAGAAGC -3'	(SEQ ID NO:34)<br>
GB165:   S'-AGGCGAGTCGACGTTTCAAACAATCATCAAGTCAGGTTGAAGAT<br>
 	GCATATCAGGTAAAGATAGAAGC -3'	(SEQ ID NO:35)<br>
GB166:   S'-AGGCGAGTCGACGTTTCAAACAATCATCAAGTCAGGTTGAAGAT<br>
GCATCCTAGGTAAAGATAGAAGC -3*	(SEQ ID NO:36)<br>
GB 167:   S'-AGGCGAGTCGACGTTTCAAACAATCATCAAGTCAGGTTGAAGAT<br>
GCATCTTAGGTAAAGATAGAAGC -3'	(SEQ ID NO:37)<br>
     GB168:   S'-AGGCGAGTCGACGTTTCAAACAATCATCAAGTCAGGTTGAAGAT<br>
GCATATTAGGTAAAGATAGAAGC -3'	(SEQ ID NO:38)<br>
GB 169:   S'-AGGCGAGTCGACGTTTCAAACAATCATCAAGTCAGGTTGAAGAT<br>
GCATCATAGGTAAAGATAGAAGC -3'	(SEQ ID NO:39)<br>
GB170:   5'-AGGCGAGTCGACGTTTCAAACAATCATCAAGTCAGGTTGAAGAT<br>
 	GCATAATAGGTAAAGATAGAAGC -3'	(SEQ ID NO:40)<br>
GB171:   5'-AGGCGAGTCGACGTTTCAAACAATCATCAAGTCAGGTTGAAGAT<br>
GCATTGTAGGTAAAGATAGAAGC -3*	(SEQ ID NO:41)<br>
GB172:   5VAGGCGAGTCGACGTTTCAAACAATCATCAAGTCAGGTTGAAGAT<br>
GCATCCAAGGTAAAGATAGAAGC -3'	(SEQ ID NO:42)<br>
     GB173:   S'-AGGCGAGTCGACGTTTCAAACAATCATCAAGTCAGGTTGAAGAT<br>
GCATACAAGGTAAAGATAGAAGC -3'	(SEQ ID NO:43)<br>
GB 174:   S'-AGGCGAGTCGACGTTTCAAACAATCATCAAGTCAGGTTGAAGAT<br>
GCATGTAAGGTAAAGATAGAAGC -3'	(SEQ ID NO:44)<br>
GB 175:   5'-AGGCGAGTCGACGTTTCAAACAATCATCAAGTCAGGTTGAAGAT<br>
 	GCATGAAAGGTAAAGATAGAAGC -3*	(SEQ ID NO:45)<br>
GB176:   5'-AGGCGAGTCGACGTTTCAAACAATCATCMGTCAGGTTGAAGAT<br>
GCATTTGAGGTAAAGATAGAAGC -3'	(SEQ ID NO:46)<br>
GB 177:   5'-AGGCGAGTCGACGTTTCAMCAATCATCMGTCAGGTTGAAGAT<br>
GCATGTGAGGTAAAGATAGAAGC -3'	(SEQ ID NO:47)<br>
GB178:   5*-AGGCGAGTCGACGTTTCAAACAATCATCAAGTCAGGTTGAAGAT<br>
GCATCAGAGGTAAAGATAGAAGC -3'	(SEQ ID NO:48)<br>
GB179:   S'-AGGCGAGTCGACGTrTCAAACAATCATCAAGTCAGGTTGAAGAT<br>
GCATCGGAGGTAAAGATAGAAGC -3'	(SEQ ID NO:49)<br>
GB181:   5*-AGGCGAGTCGACGTTTCAMCAATCATCMGTCAGGTTGAAGAT<br>
GCATGCCAGGTAAAGATAGAAGC-31	(SEQ ID NO:50)<br>
GB 182:   S'-AGGCGAGTCGACGTTTCAAACAATCATCAAGTCAGGTTGAAGAT<br>
GCATAACAGGTAAAGATAGAAGC-3'	(SEQ ID NO:51)<br>
The resulting 440 bp PCR nucleic acid fragments were digested with Bglll and Sail and the .322 bp fragment ligated into plasmid pCB2114 (described below) that had been digested with the same enzymes. This replaced the resistant Pi-ta 322 bp Bglll-Sall fragment in plasmid pCB2114 with a nucleic acid fragment containing the altered codon at position 918. This cloning step produced plasmids pCB2153-pCB2170 (Table X). Plasmid pCB2114 comprises nucleotide sequence encoding<br><br>
the Pi-ta leucine rich domain (LRD) described in WO 00/01862 which was amplified in a PCR reaction with primers GB47 (SEQ ID NO:52) and GB107 (SEQ ID NO:53). The 1050 bp PCR fragment was digested with EcoRI and Sail and cloned into plasmid pGAL4-BD-Cam (Stratagene) that had been digested with the same enzymes.<br>
GB47:   5V\ATGCAGAATTCACAACACCACTAGCAGGTTTG-3'   (SEQ ID NO:52) GB107:   5'-AGGCGAGTCGACGTTTCAMCAATCATCAAGTCAGG-3'<br>
(SEQ ID NO:53)<br>
The second cloning step required the modified Pi-ta fragment in plasmids pCB2153-pCB2170 to be amplified with primers GB60 (SEQ ID NO:33) and GB67 (SEQ ID NO:30) in 18 separate PCR reactions. GB67 (SEQ ID NO:30) adds a Hindlll cloning site after the TGA stop codon. The resulting 18 PCR fragments were digested with Bglll and Hindlll and cloned into plasmid pCB1926 that had been digested with the same enzymes (and the corresponding 4900 bp nucleic acid fragment isolated and purified from an agarose gel). This cloning step generated plasmids pCB2171-pCB2188 (Table 4). Table 4 identifies the plasmids with respect to the amino acid at position 918 of the Pi-ta protein encoded by the nucleic acid fragments contained in the plasmids. For example, the table indicates that pCB2153 (line 3) contains a nucleic acid fragment encoding a Pi-ta protein in which the amino acid at position 918 is a glutamic acid residue (E) instead of an alanine residue (A); this change was brought about by changing the Pi-ta nucleotide sequence via PCR as described above using oligonucleotides SEQ ID NO:33 and SEQ ID NO:34 (column 2). Plasmid pCB2153 was then used to make pCB2171 (column 4) as described above; pCB2171 was the one actually used ("transient assay plasmid") in the transient assay experiments described in Example 5. The sequence of each of these plasmids was verified by sequencing using primers that are enclosed in Example 7 in WO 00/01862. This resulting set of plasmids pCB2171-pCB2188, pCB1926 and pCB2020 represented a Pi-ta nucleic acid fragment with all 20 possible amino acid combinations at position 918 linked to a native 2424 bp Pi-ta promoter nucleic acid fragment.<br><br><br><br><br>
Transient Particle Bombardment Assay for Testing Efficacy of Triggering Pi-ta Mediated Resistance bv Modified Pi-ta Genes Toward the Virulent A\/R-P/Ya Allele from M. grisea Strain G-198 Particle Bombardment Demonstrates that AVR-Pita from avirulent M. grisea Strain 0-137 But Not avr-pita from virulent M. grisea Strain G-198 Elicits a Hypersensitive Response in Resistant Rice containing Pi-ta<br>
As is standard practice to those skilled in the art, high velocity biolistic bombardment of plant tissue with particles coated with recombinant expression constructs of interest results in transient expression of the nucleic acid fragments from the introduced plasmids. Function of disease resistance genes can be demonstrated in transient leaf bombardment experiments using reporter gene expression to assay for triggering of the hypersensitive cell death resistance response. Such an assay has demonstrated utility for analyzing function of the Pi-ta resistance gene and is described in WO 00/08162.<br>
In order to overcome the obstacle to uniform incorporation of the fungal AVR-gene within the plant tissue, introduction of an AVR-Pita expression construct was tested by co-bombardment along with the GUS reporter gene. In particular, constructs were engineered that express the putative mature protease (AVR-Pita^jo) from strains 0-137 (pCB1947) and G-198 (pCB2148) under control of the 35S promoter for constitutive expression in plant cells.<br>
To determine if co-expression of the AVR-Pita-jjQ construct from strains 0-137 and G-198 triggers Pi-ta mediated defense responses when introduced into Phta-conta'ming plant ceils, seedlings from P/-ta-plants (Yashiro-mochi andYT14) and plants that lack Pi-ta (Nipponbare and YT16) were co-bombarded with<br><br>
pCB1947 or pCB2148 containing the 35S/Adh1-6::AVR-Pita176 gene construct and pML63 (which is described in WO 00/08162) containing the 35S::GUS reporter gene. YT14 and YT16 are described in WO 00/08162. Nipponbare is a rice variety that is widely studied (e.g., Mao et al. (2000) Genome Res 10:982-990). Seeds were germinated on leaf assay media: 1/2 strength MS medium (Murashige and Skoog, 1962, Physiol. Plant. 15:473-497) supplemented with 100 mg casein hydrolysate and 0.5% agarose for a week in an incubator at 25°C for 48 hours in 12 hr photoperiod with a 100 uErrr^s"1 of cool, white light. Two-leaf seedlings were excised from the agar medium using a surgical razor and placed in a petri dish containing a prewetted filter paper. Plantlets were labeled at the base with a permanent marker for identification. Biolistic bombardment of the seedlings was performed using Bio-Rad PDS-1000/He apparatus and 1150-psi rupture disks. Gold particles (0.6 urn diameter) were prepared according to the instructions provided by the manufacturer. For each cobombardment, 1 ug of gold particles was coated with 1.5 ug of 35S/GUS and 1 ug of other plasmids. After bombardment, seedlings were maintained at 25°C for 48 hours in Petri dishes containing prewetted filter paper. Leaves were cleared in 70% ethanol and histochemically assayed for ^-Glucuronidase (GUS) activity using 5-bromo-4-chloro-3-indolyl glucuronide (X-gluc) as a substrate (Jefferson, 1987, Plant Mol. Bio. Rep. 5:387-405).<br>
The AVR-PitauQ expression construct pCB1947 mediated a striking decrease in GUS expression (suggesting cell death prior to expression), when co-bombarded with the GUS construct into YT14 leaves containing the endogenous Pi-ta gene. This effect was not observed in susceptible YT16 leaves treated the same way. A virulent form of AVR-Pita176 in construct pCB2148 failed to decrease GUS expression in either resistant TY14 or susceptible YT16 leaves. Thus, only expression of the AVR-Pita coding sequence from 0-137 triggers the rice defense response when introduced directly into rice leaves containing the endogenous Pi-ta gene.<br>
In the second part of this experiment, a third construct was added, pCB1926. Seedlings from P/-te-plants (Yashiro-mochi and YT14) and plants that lack Pi-ta (Nipponbare and YT16) were co-bombarded with the 35S/Adh1-6::AVR-Pita176 gene construct pCB1947 or pCB2148, the Pi-ta cDNA under the control of its native promoter (pCB1926) and the 35S::GUS reporter gene. In this case susceptible YT16 rice leaves responded in the same way as resistant YT14 rice leaves to the avirulent AVR-Pita176 construct from strain 0-137. It is therefore possible to add Pi-ta exogenously to susceptible rice leaves to elicit a HR response to AVR-Pita. No HR response was observed towards the virulent avr-Pita construct from virulent strain G-198.<br><br>
This assay forms the basis for the next experiment described where 18 new Pi-ta variants described in Example 4 above were then tested for their ability to encode a novel Pi-ta protein that responds to virulent avr-Pita proteins to elicit an HR.,<br>
Testing Pi-ta transient expression vectors encoding Pi-ta proteins altered at position 918<br>
The 20 vectors pCB1926, pCB2020 and pCB2171-pCB2188 described in Example 4 above were tested in the transient assay using the conditions described above. All were co-bombarded with pML63 which contains a 35S::GUS reporter gene and either pCB1947 or pCB2148 which contain different 35S/Adh1-6::AVR-Pita176 gene constructs. Results are summarized in Table 5. In the table,"-" indicates lack of GUS staining that is indicative of the hypersensitive response (there is productive recognition between the AVR-Pita and Pi-ta gene products); "+/-" indicates a few rare GUS loci that may indicate productive recognition between the AVR-Pita and Pi-ta gene products;"+" indicates a small number of GUS loci indicative of possible albeit suboptimal recognition between the AVR-Pita and Pi-ta gene products; and "++" and "+++" indicate multiple GUS loci indicative of no productive recognition between the AVR-Pita and Pi-ta gene products. Plates 1 and 2 represent two replicates for each AVR-Pita/Pi-ta combination, with each plate having two test seedlings. "Repeats" indicates the number of times a particular experiment was repeated.<br>
Table 5<br>
Effect of Amino Acid at Position 918 or	i Ability of Pi-ta to Recoqnize<br>
AVR-Pita from M. orisea strain 0-137 and avr-Dita	from M.	arisea strain G-198<br>
		GUS Activity @ 48 hr			<br>
Pi-ta	AVR-Pita<br>
(0-137)<br>
(pCB1947)<br>
Plate    Plate<br>
1           2	avr-pita<br>
(G-198)<br>
(pCB2148)<br>
Plate    Plate<br>
1           2	NoAVR<br>
Plate 1	gene<br>
Plate 2	Repeats	Conclusion<br>
Constructs							<br>
A918 (Resistant)<br>
s918 (Susceptible)<br>
G918	++        ++ ++       +++	+ + ++	++ ++ ++			4 4 1	resistant control - recognizes avirulent<br>
susceptible control - no recognition<br>
not functional - no<br>
v918	++        ++	+	++			1	recognition<br>
not functional - no<br>
E918	+		++			1	recognition<br>
not functional - no<br>
Dgi8	+++      +++	+	+			1	recognition<br>
not functional - no<br>
recognition<br>
/<br><br><br>
Consistent with disease phenotypes observed when Yashiro-mochi is challenged with various M. grisea strains, Table 5 indicates that the Pi-ta protein from Yashiro-mochi (A918) recognizes the AVR-Pita protein from 0-137 (which in plants results in disease resistance phenotype when Yashiro-mochi is challenged with 0-137) but not the AVR-Pita form from G-198 (which in planta results in disease susceptibility phenotype when Yashiro-mochi is challenged with G-198). In addition, the results described in Table 5 identified two Pi-ta constructs encoding Pi-ta proteins with amino acid 918 changed to either M or C that recognized both avimlent (AVR-Pita from 0-137) and virulent (avr-pita from G-198) AVR-Pita gene products. As can be seen from the table, M918 Pi-ta gave rise to multiple GUS loci when there was no AVR-Pita gene co-bombarded with the other constructs, indicating that by itself, M918 is not able to cause the hypersensitive response; this indicated that the absence of GUS loci observed when AVR-Pita gene was present was indeed due to productive recognition between AVR-Pita and Pi-ta gene products and not from other reasons like autoactivation of the Pi-ta protein leading to the hypersensitive response. Another form with an I at position 918 had switched specificity relative to the Pi-ta protein from Yashiro-mochi (A918) since it onJy recognized the virulent G-198 allele but not the 0-137 allele. Other constructs encoding proteins containing R, K, N, L, and Q at position 918 also resulted in recognition of the G-198 allele.<br><br>
I<br>
These data suggest that it is. possible to alter the range of A VR-Pita alleles that Pi-ta can recognize by changing the amino acid at position 918 of the Pi-ta protein. Consequently, transgenic rice plants that are able to recognize M. grisea strain G-198 and/or other strains containing an AVR-Pita gene substantially similar to that found in SEQ ID NO: and undergo an HR or disease resistance response may be obtained by transforming rice with a modified Pi-ta nucleic acid fragment which encodes a Pi-ta protein that has at position 918 an amino acid selected from the group consisting of M, C, I, R, K, N, L and Q.<br>
Additionally, modified Pi-ta nucleic acid fragments which encode a Pi-ta protein that has at position 918 an amino acid selected from the group consisting of M, C..I, R, K, N, L and Q, may be introduced into rice varieties that already have a functional Pi-ta (A918), resulting in lines which can mount a disease resistance response to a broader range of M. grisea strains.<br>
EXAMPLE 6<br>
Construction of Chimeric Genes Encoding Modified Pi-ta Proteins<br>
for Stable Rice Transformation<br>
Plasmids comprising nucleic acid fragments encoding the altered forms of Pi-ta identified in Example 5 were constructed using vector pCB1926 and thus do not have a terminator sequence. For stable rice transformation, the vector pCB2022 is the preferred Pi-ta format for a Pi-ta nucleic acid fragment as in addition to the 2424 bp of Pi-ta native promoter sequence and 2784 bp of Pi-ta coding sequence it contains an In2-1 terminator sequence (SEQ ID NO:60). Plasmid pCB2022 has been deposited with the ATCC. To make the precursor plasmid pCB2021, the entire 5222 bp EcoRI-Hindlll Pi-ta nucleic acid fragment was excised from plasmid pCB1926 using a partial digest (as there is a second EcoRI site within the Pi-ta cDNA nucleic acid fragment) and cloned into the same sites of plasmid Litmus 28a (New England Bioiabs). Plasmid pCB2021 was then digested with Agel and Kpnl and a 501 bp Agel-Kpnl In2-1 terminator nucleic acid fragment was cloned into the same sites to create pCB2022. The In2-1 terminator sequence may be amplified from plasmids containing In2-1 terminator sequence (e.g., pJE514, pJE516, and pTDS136 disclosed in US Patent 5,364,780, the disclosure of which is hereby incorporated by reference) using primers GB188 (SEQ ID NO:54) and GB189 (SEQ ID NO:55), then digested with Agel and Kpnl.<br>
GB188:   5'-GCCGACCGGTAGATCTGACAAAGCAGCATTAG-3'    (SEQ ID NO:54)<br>
GB189:   5*-CGGCGGTACCGCTCTCTCTCTCCCCTTGC-3'	(SEQ ID NO:55)<br><br>
Modified Pi-ta nucleic acid fragments identified in Example 5 which encode Pi-ta protein wherein the amino acid alteration at position 918 is selected from the group consisting of M, C, I, R, K, N, L and Q, and are suitable for stable rice transformation can be made in a two-stage cloning step. A 2.1 kb BamHI-Hindlll       fragment from pCB2176, pCB2180, pCB2177, pCB2173, pCB2174, pCB2175, pCB2185, or pCB2183 can be cloned into vector pCB2021 to replace the corresponding fragment from the wild-type Pi-ta nucleic acid fragment present in pCB2021. Then the 501 bp Agel-Kpnl In2-1 terminator nucleic acid fragment can be excised from vector pCB2022 by digesting with Agel and Kpnl and cloned into the same sites on the new vector that has also been digested with Agel and Kpnl. These new vectors can then be used in stable rice transformation as described in Example 7 below.<br>
EXAMPLE 7 Testing Chimeric Genes Encoding Modified Pi-ta Proteins from Example 6 bv Stable Rice Transformation The bacterial hygromycin B phosphotransferase (Hpt II) gene from Streptomyces hygroscopicus that confers resistance to the antibiotic can be used as the selectable marker for rice transformation. In the vector that could be used, pML18, the Hpt II gene has been engineered with the 35S promoter from Cauliflower Mosaic Virus and the termination and polyadenylation signals from the octopine synthase gene of Agrobacterium tumefaciens. pML18 is described in WO 97/47731, which was published on December 18, 1997, the disclosure of which is hereby incorporated by reference.<br>
Embryogenic callus cultures derived from the scutellum of germinating Nipponbare seeds can serve as source material for transformation experiments. This material is generated by germinating sterile rice seeds on a callus initiation media (MS salts, Nitsch and Nitsch vitamins, 1.0 mg/l 2,4-D and 10 |j.M AgN03) in the dark at 27-28° C. Embryogenic callus proliferating from the scutellum of the embryos are then transferred to CM media (N6 salts, Nitsch and Nitsch vitamins, 1 mg/l 2,4-D, Chu et al., 1985, Sci. Sinica 18: 659-668). Callus cultures can be maintained on CM by routine sub-culture at two week intervals and used for transformation within 10 weeks of initiation.<br>
. Callus can be prepared for transformation by subculturing 0.5-1.0 mm pieces approximately 1 mm apart, arranged in a circular area of about 4 cm in diameter, in the center of a circle of Whatman #541 paper placed on CM media. The plates with callus are then incubated in the dark at 27-28° C for 3-5 days. Prior to bombardment, the filters with callus are transferred to CM supplemented with<br><br>
0.25 M mannitol and 0.25 M sorbitol for 3 hr. in the dark. The petri dish lids are then left ajar for 20-45 minutes in a sterile hood to allow moisture on tissue to dissipate.<br>
Circular plasmid DNAfrom two different plasmids, for example, pML18 containing the selectable marker for rice transformation and a vector containing the modified Pi-ta nucleic acid fragment, can be co-precipitated onto the surface of gold particles. To accomplish this, a total of 10  g of DNA at a 2:1 ratio of traitselectable marker DNAs are added to 50 I aliquot of gold particles that have been resuspended at a concentration of 60 mg ml"'. Calcium chloride (50 I of a 2.5 M solution) and spermidine (20  I of a 0.1 M solution) are then added to the gold-DNA suspension and the tube would be vortexed for 3 min. The gold particles are centrifuged in a microfuge for 1 sec and the supernatant removed. The gold particles are then washed twice with 1 ml of absolute ethanol and then resuspended in 50   I of absolute ethanol and sonicated (bath sonicator) for one second to disperse the gold particles. The gold suspension is then incubated at -70°C for five minutes and sonicated (bath sonicator) if needed to disperse the particles. Six   I of the DNA-coated gold particles can then be loaded onto mylar macrocarrier disks and the ethanol allowed to evaporate.<br>
At the end of the drying period, a petri dish containing the tissue is placed in the chamber of the PDS-1000/He. The air in the chamber is then evacuated to a vacuum of 28-29 inches Hg. The macrocarrier accelerated with a helium shock wave using a rupture membrane that bursts when the He pressure in the shock tube reaches 1080-1100 psi. The tissue is placed approximately 8 cm from the stopping screen and the callus bombarded two times. Five to seven plates of tissue are bombarded in this way with the DNA-coated gold particles. Following bombardment, the callus tissue is transferred to CM media without supplemental sorbitol or mannitol.<br>
Within 3-5 days after bombardment the callus tissue is transferred to SM media (CM medium containing 50 mg/l hygromycin). To accomplish this, callus tissue is transferred from plates to sterile 50 ml conical tubes and weighed. Molten top-agar at 40° C can be added using 2.5 ml of top agar/100 mg of callus. Callus clumps are broken into fragments of less than 2 mm diameter by repeated dispensing through a 10 ml pipet. Three ml aliquots of the callus suspension are plated onto fresh SM media and the plates can be incubated in the dark for 4 weeks at 27-28°C. After 4 weeks, transgenic callus events are identified, transferred to      fresh SM plates and grown for an additional 2 weeks in the dark at 27-28°C.<br>
Growing callus is transferred to RM1 media (MS salts, Nitsch and Nitsch vitamins, 2% sucrose, 3% sorbitol, 0.4% gelrite +50 ppm hyg B) for 2 weeks in the dark at 25°C. After 2 weeks the callus can be transferred to RM2 media (MS salts,<br><br><br><br><br><br>
Nitsch and Nitsch vitamins, 3% sucrose, 0.4% gelrite + 50 ppm hyg B) and placed under cool white light (-40 nErrr2s-1) with a 12 hr photoperiod at 25°C and 30^0% humidity. After 2-4 weeks in the light, callus begins to organize, and form shoots. Shoots are then removed from surrounding callus/media and gently transferred to RM3 media (1/2 x MS salts, Nitsch and Nitsch vitamins, 1% sucrose + 50 ppm hygromycin B) in phytatrays (Sigma Chemical Co., St. Louis, MO) and incubation is continued using the same conditions as described in the previous step.<br>
Plants are transferred from RM3 to 4" pots containing Metro mix 350 after 2-3 weeks, when sufficient root and shoot growth had occurred. Plants can be grown using a 12 hr/12 hr light/dark cycle using -30/18°Cday/night temperature regimen.<br><br>
We Claim:<br>
1.       An isolated nucleic acid fragment comprising a nucleic acid sequence<br>
encoding an altered Pi-ta resistance polypeptide selected from the<br>
group consisting of:<br>
(a)	a nucleic acid sequence having at least 90% sequence identity when compared to nucleotides 2426-5212 of the nucleic acid sequence of SEQ ID NO:57, wherein the altered Pi-ta resistance polypeptide has an amino acid substitution for alanine at position 918 and confers resistance in a plant transformed therewith against the fungus Magnaporthe comprising in its genome one or more virulent and/or avirulent AVR-Pita alleles, and<br>
(b)	a nucleic acid sequence encoding an amino acid sequence as set forth in SEQ ID NO:58 wherein the amino acid sequence has a single amino acid substitution at position 918 which confers resistance in a plant transformed therewith against the fungus Magnaporthe comprising in its genome one or more virulent or avirulent AVR-Pita alleles.<br><br>
2	The isolated nucleic acid fragment as claimed in claim 1 wherein the amino acid at position 918 is selected from the group consisting of M, C, I, R, K, N, L and Q.<br>
3	A chimeric gene comprising the nucleic acid fragment as claimed in claim 1 or 2 operably linked to at least one regulatory sequence.<br><br>
4. A method of conferring resistance in a plant against the fungus Magnaporthe comprising in its genome virulent and/ or avirulent AVR-Pita alleles which comprises transforming a plant with the chimeric gene as claimed in claim 3;<br>
wherein  said  plant  has increased  resistance  against  the fungus Magnaporthe comprising in its genome one or more virulent and/or<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LU1VTU5QLTIwMDMtQUJTVFJBQ1QoOS03LTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">355-MUMNP-2003-ABSTRACT(9-7-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LW11bW5wLTIwMDMtYXNzaWdubWVudCgwOC0wOC0yMDA2KS5wZGY=" target="_blank" style="word-wrap:break-word;">355-mumnp-2003-assignment(08-08-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LW11bW5wLTIwMDMtY2FuY2VsbGVkIHBhZ2VzKDA4LTA2LTIwMDYpLnBkZg==" target="_blank" style="word-wrap:break-word;">355-mumnp-2003-cancelled pages(08-06-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LU1VTU5QLTIwMDMtQ0FOQ0VMTEVEIFBBR0VTKDktNy0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">355-MUMNP-2003-CANCELLED PAGES(9-7-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LU1VTU5QLTIwMDMtQ0xBSU1TKDktNy0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">355-MUMNP-2003-CLAIMS(9-7-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LW11bW5wLTIwMDMtY2xhaW1zKGdyYW50ZWQpLSgwOC0wNi0yMDA2KS5kb2M=" target="_blank" style="word-wrap:break-word;">355-mumnp-2003-claims(granted)-(08-06-2006).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LW11bW5wLTIwMDMtY2xhaW1zKGdyYW50ZWQpLSgwOC0wNi0yMDA2KS5wZGY=" target="_blank" style="word-wrap:break-word;">355-mumnp-2003-claims(granted)-(08-06-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LW11bW5wLTIwMDMtY29ycmVzcG9uZGVuY2UoMTAtMDgtMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">355-mumnp-2003-correspondence(10-08-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LU1VTU5QLTIwMDMtQ09SUkVTUE9OREVOQ0UoOS03LTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">355-MUMNP-2003-CORRESPONDENCE(9-7-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LU1VTU5QLTIwMDMtQ09SUkVTUE9OREVOQ0UoSVBPKS0oMjItMTItMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">355-MUMNP-2003-CORRESPONDENCE(IPO)-(22-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LU1VTU5QLTIwMDMtREVFRCBPRiBBU1NJR05NRU5UKDgtOC0yMDA2KS5wZGY=" target="_blank" style="word-wrap:break-word;">355-MUMNP-2003-DEED OF ASSIGNMENT(8-8-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LU1VTU5QLTIwMDMtREVTQ1JJUFRJT04oQ09NUExFVEUpLSg5LTctMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">355-MUMNP-2003-DESCRIPTION(COMPLETE)-(9-7-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LU1VTU5QLTIwMDMtRFJBV0lORyg5LTctMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">355-MUMNP-2003-DRAWING(9-7-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LW11bW5wLTIwMDMtZm9ybSAxMygwOS0wNy0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">355-mumnp-2003-form 13(09-07-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LW11bW5wLTIwMDMtZm9ybSAxMygxMy0wOC0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">355-mumnp-2003-form 13(13-08-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LW11bW5wLTIwMDMtZm9ybSAxOCgxNC0xMS0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">355-mumnp-2003-form 18(14-11-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LW11bW5wLTIwMDMtZm9ybSAxYSgxMy0wOC0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">355-mumnp-2003-form 1a(13-08-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LU1VTU5QLTIwMDMtRk9STSAxQSgxMy04LTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">355-MUMNP-2003-FORM 1A(13-8-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LW11bW5wLTIwMDMtZm9ybSAxYSgzMS0wMy0yMDAzKS5wZGY=" target="_blank" style="word-wrap:break-word;">355-mumnp-2003-form 1a(31-03-2003).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LU1VTU5QLTIwMDMtRk9STSAxQSgzMS0zLTIwMDMpLnBkZg==" target="_blank" style="word-wrap:break-word;">355-MUMNP-2003-FORM 1A(31-3-2003).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LW11bW5wLTIwMDMtZm9ybSAyKGdyYW50ZWQpLSgwOC0wNi0yMDA2KS5kb2M=" target="_blank" style="word-wrap:break-word;">355-mumnp-2003-form 2(granted)-(08-06-2006).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LW11bW5wLTIwMDMtZm9ybSAyKGdyYW50ZWQpLSgwOC0wNi0yMDA2KS5wZGY=" target="_blank" style="word-wrap:break-word;">355-mumnp-2003-form 2(granted)-(08-06-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LU1VTU5QLTIwMDMtRk9STSAyKFRJVExFIFBBR0UpLSg5LTctMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">355-MUMNP-2003-FORM 2(TITLE PAGE)-(9-7-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LW11bW5wLTIwMDMtZm9ybSAzKDA4LTA4LTIwMDYpLnBkZg==" target="_blank" style="word-wrap:break-word;">355-mumnp-2003-form 3(08-08-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LW11bW5wLTIwMDMtZm9ybSAzKDMxLTAzLTIwMDMpLnBkZg==" target="_blank" style="word-wrap:break-word;">355-mumnp-2003-form 3(31-03-2003).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LU1VTU5QLTIwMDMtRk9STSAzKDgtOC0yMDA2KS5wZGY=" target="_blank" style="word-wrap:break-word;">355-MUMNP-2003-FORM 3(8-8-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LW11bW5wLTIwMDMtZm9ybSA1KDMxLTAzLTIwMDMpLnBkZg==" target="_blank" style="word-wrap:break-word;">355-mumnp-2003-form 5(31-03-2003).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LU1VTU5QLTIwMDMtRk9STSA1KDktNy0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">355-MUMNP-2003-FORM 5(9-7-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LW11bW5wLTIwMDMtZm9ybS1wY3QtaXBlYS00MDkoMzEtMDMtMjAwMykucGRm" target="_blank" style="word-wrap:break-word;">355-mumnp-2003-form-pct-ipea-409(31-03-2003).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LW11bW5wLTIwMDMtZm9ybS1wY3QtaXNhLTIxMCgzMS0wMy0yMDAzKS5wZGY=" target="_blank" style="word-wrap:break-word;">355-mumnp-2003-form-pct-isa-210(31-03-2003).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LW11bW5wLTIwMDMtZm9ybS1wY3QtaXNhLTIyMCgzMS0wMy0yMDAzKS5wZGY=" target="_blank" style="word-wrap:break-word;">355-mumnp-2003-form-pct-isa-220(31-03-2003).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LU1VTU5QLTIwMDMtUENULUlQRUEtNDA5KDktNy0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">355-MUMNP-2003-PCT-IPEA-409(9-7-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LU1VTU5QLTIwMDMtUENULUlTQS0yMTAoOS03LTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">355-MUMNP-2003-PCT-ISA-210(9-7-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LW11bW5wLTIwMDMtcGV0aXRpb24gdW5kZXIgcnVsZSAxMzcoMDgtMDgtMjAwNikucGRm" target="_blank" style="word-wrap:break-word;">355-mumnp-2003-petition under rule 137(08-08-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LU1VTU5QLTIwMDMtUEVUSVRJT04gVU5ERVIgUlVMRSAxMzcoOC04LTIwMDYpLnBkZg==" target="_blank" style="word-wrap:break-word;">355-MUMNP-2003-PETITION UNDER RULE 137(8-8-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LW11bW5wLTIwMDMtcGV0aXRpb24gdW5kZXIgcnVsZSAxMzgoMDgtMDgtMjAwNikucGRm" target="_blank" style="word-wrap:break-word;">355-mumnp-2003-petition under rule 138(08-08-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LU1VTU5QLTIwMDMtUEVUSVRJT04gVU5ERVIgUlVMRSAxMzgoOC04LTIwMDYpLnBkZg==" target="_blank" style="word-wrap:break-word;">355-MUMNP-2003-PETITION UNDER RULE 138(8-8-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LW11bW5wLTIwMDMtcG93ZXIgb2YgYXV0aG9yaXR5KDA4LTA4LTIwMDYpLnBkZg==" target="_blank" style="word-wrap:break-word;">355-mumnp-2003-power of authority(08-08-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LW11bW5wLTIwMDMtcG93ZXIgb2YgYXV0aG9yaXR5KDIwLTA3LTIwMDApLnBkZg==" target="_blank" style="word-wrap:break-word;">355-mumnp-2003-power of authority(20-07-2000).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LU1VTU5QLTIwMDMtUE9XRVIgT0YgQVVUSE9SSVRZKDgtOC0yMDA2KS5wZGY=" target="_blank" style="word-wrap:break-word;">355-MUMNP-2003-POWER OF AUTHORITY(8-8-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LU1VTU5QLTIwMDMtU0VRVUVOQ0UgTElTVElORyg5LTctMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">355-MUMNP-2003-SEQUENCE LISTING(9-7-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU1LU1VTU5QLTIwMDMtV08gSU5URVJOQVRJT05BTCBQVUJMSUNBVElPTiBSRVBPUlQgQTIoOS03LTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">355-MUMNP-2003-WO INTERNATIONAL PUBLICATION REPORT A2(9-7-2007).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="227332-an-improved-boom-for-use-with-a-crane.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="227334-an-insulating-concrete-form.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>227333</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>355/MUMNP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>10/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>06-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>06-Jan-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>31-Mar-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>E.I. DU PONT DE NEMOURS AND COMPANY</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1007 MARKET STREET, WILMINGTON, DELAWARE 19898, UNITED STATES OF AMERICA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>GREGORY BRYAN</td>
											<td>11 KARAMU AVENUE, PALMERSTON NORTH, KS5301, NEW ZEALAND.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>BARBARA VALENT</td>
											<td>4920 LAKEWOOD DRIVE, MANHATTAN, KS 66503, USA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US01/43985</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-11-14</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/248,335</td>
									<td>2000-11-14</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/227333-modification-of-a-plant-disease-resistance-gene-specificity-and-method-for-engineering-altered-specificity by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:51:15 GMT -->
</html>
