<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/227419-method-for-preparing-polypeptide-variants by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:05:27 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 227419:METHOD FOR PREPARING POLYPEPTIDE VARIANTS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHOD FOR PREPARING POLYPEPTIDE VARIANTS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>ABSTRACT 1438/MAS/96 &quot;METHOD FOR PREPARING POLYPEPTIDE VARIANTS&quot; The present invention relates to a method for preparing a shuffled polypeptide variant by in vivo recombination, said method comprising the steps of: (a) linearizing at least one circular plasmid, wherein the plasmid comprises a DNA sequence encoding a polypeptide of interest and the linearization is within the DNA sequence encoding the polypeptide of interest; (b) preparing two or more DNA fragments comprising DNA sequences encoding variants of the polypeptide of interest; (c) introducing the at least one linearized plasmid of step (a) with the at least two DNA fragments of step (b) into a host cell, wherein recombination occurs between the at least one linearized plasmid and the two or more DNA fragments to generate a recombinant circular plasmid comprising a shuffled DNA sequence encoding the shuffled polypeptide variant; (d) cultivating the host cell comprising the recombinant circular plasmid under conditions where the shuffled DNA sequence is expressed, and (e) screening for the shuffled polypeptide variant.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
FIELD OF THE INVENTION<br>
The present invention relates to a method for preparing polypeptide variants by in  vivo  recombination.<br>
BACKGROUND OF THE INVENTION<br>
The advantages of producing biologically active polypeptides by cloning naturally occurring DNA sequences from microorganisms, such as fungal organisms and bacteria using recombinant DNA technology have been known for quite some years.<br>
Preparation of novel polypeptide variants and mutants, such as novel modified enzymes with altered characteristics, e.g. specific activ¬ity, substrate specificity, pH-optimum, pi, K„, v,™,, etc., have especially during the recent years diligently and successfully been used for obtaining polypeptides with improved properties.<br>
For instance, within the technical field of enzymes the washing and/or dishwashing performance of e.g. proteases, lipases, amylases and cellulases have been improved significantly.<br>
In most cases these improvements have been obtained by site-directed mutagenesis resulting in substitution, deletion or insertion of specific amino acid residues which have been chosen either on the basis of their type or on the basis of their location in the second¬ary or tertiary structure of the mature enzyme {see for instance US patent no. 4,518,584).<br>
An alternative general approach for modifying proteins and enzymes have been based on random mutagenesis, for instance, as disclosed in US 4,894,331 and WO 93/01285<br>
As it is a cumbersome and time consuming process to obtain po¬lypeptide variants or mutants with improved functional properties a few alternative methods for rapid preparation of modified polypeptides have been suggested.<br><br>
Weber et al -, (1983), Nucleic Acids Research, vol 11, 5661-5661, describes a method for modifying genes by in vivo recombination between to homologous genes. A linear DNA sequence comprising a piasmid vector flanked to a DNA sequence encoding alpha-1 human interferon in the 5'-end and a DNA sequence encoding alpha-2 human interferon in the 3'-end is constructed and transfected into a rec A positive strain of E. coli. Recombinants were identified and isolated using a resistance marker.<br>
Pompon el al., (1989), Gene 83, p. 15-24, describes a method for shuffling gene domains of mammalian cytochrome P-450 by in vivo recombination of partially homologous sequences in Saccharomyces cerevisiae by transforming Saccharomyces cerevisia with a linearized piasmid with filled-in ends, and a DNA fragment being partially homologous to the ends of said piasmid.<br>
Stemmer, (1994), Proc. Natl. Acad. Sci. USA, Vol. 91, 10747-10751; Stemmer, (1994), Nature, vol. 370, 389- 391, concern methods for shuffling homologous DNA sequences by an in vitro PCR method. One cycle of shuffling consists of digesting a pool of homologous genes with DNase I. The resulting small fragments are reassembled into full-length genes. Positive recombinant genes containing shuffled DNA sequences are selected from a DNA library based on their improved function. Positive recombinants can be used as the starting material for (an)other shuffling round(s).<br>
US patent no. 5,093,257 (Assignee: Genencor Int. Inc.) discloses a method for producing hybrid polypeptides by in vivo recombination. Hybrid DNA sequences are produced by forming a circular vector comprising a replication sequence, a first DNA sequence encoding the amino-terminal portion of the hybrid polypeptide, a second DNA sequence encoding the carboxy-terminal portion of said hybrid polypeptide. The circular vector is transformed into a rec positive microorganism in which the circular vector is amplified. This results in recombination of said circular vector mediated by the naturally occurring recombination mechanism of the rec positive microorganism, which include prokaryotes such as Bacillus and E. coli,   and eukaryotes such as Saccharomyces cerevisiae.<br><br>
A drawback of known prior art methods are that they are either not very effective, none random or difficult to perform.<br>
Consequently, there is a need for an improved iterative recom¬bination method for preparing novel positive polypeptide variants.<br>
SUM4ARY OF THE IHVENTION<br>
The object of the present invention is to provide an improved method for preparing positive polypeptide variants by an in vivo recombination method.<br>
The inventor of the present invention have surprisingly found that such positive polypeptide variants may advantageously be prepared by shuffling different nucleotide sequences of homologous DNA sequences by in vivo  recombination comprising the steps of<br>
a)	forming at least one circular plasmid comprising a DNA sequence<br>
encoding a polypeptide,<br>
b)	opening said circular plasmid(s) within the DNA sequence(s) encoding the polypeptide(s),<br>
c)	preparing at least one DNA fragment comprising a DNA sequence homologous to at least a part of the polypeptide coding region on at least one of the circular plasmid (s) , d) introducing at least one of said opened plasmidfs), together with at least one of said homologous DNA fragment(s) covering full-length DNA sequences encoding said polypeptide(s) or parts thereof, into a recombination host cell,<br><br>
e)	cultivating said recombination host cell, and<br>
f)	screening for positive polypeptide variants.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figure 1 shows the yeast expression plasmid pJS026 comprising DNA<br>
sequence encoding the Humicols  lanuginosa  lipase gene.<br>
Figure 2 shows the yeast expression plasmid pJS037, comprising DNA<br>
sequence encoding the Humicola    lanuginosa    lipase gene containing<br>
twelve additional restriction sites.<br>
Figure 3 shows the plasmid pJS026.<br><br>
Figure 4 shows the plasmid pJS037,<br>
Figure 5 shows the in   vivo   recombination of the 0.9 kb synthetic wild-type Humicola   lanuginosa   Lipase with pJS037 using Saccfearomyces cerevisiae  as the recombination host cell (described in Example 1). Figure 6 shows the in vivo recombination of a DNA fragment prepared from Humicola   lanuginosa  lipase variant (y) with Humicola   lanuginosa lipase variant  (d)  comprised in a plasmid using  Saccharomyces cerevisiae  as the recombination host cell (described in Example 2). Figure 1   shows an overview over the location of the inactivation site of the Humicola  lanuginosa  lipase  gene and the number of the clone (referred to as "blue number" in the tables). Location of restriction enzyme sites and clone numbers are relative to the initiation codon of the lipase gene. In all cases a stop codon was located in the new reading frame 10 to 50 bp from the frameshift. Figure 8 shows an overview of the creation of active humicola lanuginosa lipase genes from the recombinations in table 2A and B by a "mosaic mechanism". Lines indicate the introduction of the fragment sequence into the vector and lines with a x indicate sequences that are not introduced in the active lipase colonies. The primers used for the PCR fragment are shown together with the location of the frameshift mutation (marked by the restriction site used for the construction).<br>
Figure 9 shows an overview of fragments used in the recombination of 2 partial overlapping fragments into a gapped vector. The primers used for the PCR fragments are shown together with the location of the frameshift mutation (if not wild type). Figure 10 shows an overview of fragments used in the recombination of 3 partial overlapping fragments into a gapped vector. The primers used for the PCR fragments are shown. The overlap between PCR353 and 355 is only a 10 bp.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The object of the present invention is to provide an improved method for preparing positive polypeptide variants by an iterative in<br><br>
vivo recombination system using a eukaryotic cell as a recombination host cell.<br>
A "recombination host cell" is in the context of the present invention a cell capable of mediating shuffling of a number of homologous DNA sequences.<br>
The term "shuffling" means recombination of nucleotide sequence (s) between two or more homologous DNA sequences resulting in output DNA sequences (i.e. DNA sequences having been subjected to a shuffling cycle) having a number of nucleotides exchanged, in comparison to the input DNA sequences (i.e. starting point homologous DNA sequences).<br>
An important advantage of the invention is that mosaic DNA Sequences with multiple replacement points or replacements, not related to the opening site, is created, which is not discovered in Pompon's method.<br>
An other important advantage of the present invention is that when using a mixture of fragments and opened vectors (in the screening set up) it gives the possibility of many different clones to recombine pairwise or even triplewise (as can be seen in a couple of examples below),<br>
The in vivo recombination method of the invention simple to	perform<br>
and results in a high level of mixing of homologous genes or<br>
variants. A large number of variants or homologous genes	can be<br>
mixed in one transformation. The mixing of improved variants	or wild<br>
type genes followed by screening increases the number of	further<br>
improved variants manyfold compared to doing only	random<br>
mutagenesis.<br>
Recombination of multiple overlapping fragments is possible with a high efficiency increasing the mixing of variants or homologous genes using the in vivo recombination method. An overlap as small as 10 bp is sufficient for recombination which may be utilized for very easy domain shuffling of even distantly related genes.<br>
The invention relates to a method for preparing polypeptide variants by shuffling different nucleotide sequences of homologous DNA<br><br>
sequences by in vivo  recombination comprising the steps of<br>
a)	forming at least one circular plasmid comprising a DNA sequence<br>
encoding a polypeptide!<br>
b)	opening said circular plasmid(s) within the DNA sequence(s)<br>
encoding the polypeptide(s),<br>
c)	preparing at least one DNA fragment comprising a DNA sequence<br>
homologous to at least a part of the polypeptide coding region on at<br>
least one of the circular plasmid(s), d) introducing at least one<br>
of said opened plasmid (s) , together with at least one of said<br>
homologous DNA fragment(s) covering full-length DNA sequences<br>
encoding said polypeptide(s) or parts thereof, into a recombination<br>
host cell,<br>
e)	cultivating said recombination host cell, and<br>
f)	screening for positive polypeptide variants.<br>
According to the invention more than one cycle of step a) to f) may be performed.<br>
The opening of the plasmid(s) in step b) can be directed toward any site within the polypeptide coding region of the plasmid. The plamid (s) may be opened by any suitable methods known in the art. The opened ends of the plasmid may be filled-in with nucleotides as described in Pompon et al. (1989), supra} . It is preferred not to fill in the opened ends as it might create a frameshift.<br>
It is preferred to open the plasmid (s) around the middle of the polypeptide coding DNA sequence(s), as this is believed to result in a more effective recombination between DNA fragment(s) and opened plasmid(s).<br>
In an embodiment of the invention the DNA fragment (s) is (are) prepared under conditions resulting in a low, medium or high random mutagenesis frequency.<br>
To obtain low mutagenesis frequency the DNA sequence(s) (comprising the DNA fragment(s)) may be prepared by a standard PCR amplification method (US 4,683,202 or Saiki et al., (1988), Science 239, 487 -491) .<br>
A medium or high mutagenesis frequency may be obtained by performing<br>
the PCR amplification under conditions which increase the mis-<br><br>
incorporation of nucleotides, for instance as described by Deshler, (1992), GATA 9{4), 103-106; Leung et al., (1989), Technique, Vol. 1, No. 1, 11-15.<br>
It is also contemplated according to the invention to combine the PCR amplification (i.e. according to this embodiment also DNA fragment mutation) with a mutagenesis step using a suitable physical or chemical mutagenizing agent, e.g., one which induces transitions, transversions, inversions, scrambling, deletions, and/or insertions.<br>
In the context of the present invention the term "positive poly¬peptide variants" means resulting polypeptide variants possessing functional properties which has been improved in comparison to the polypeptides producible from the corresponding input DNA sequences. Examples, of such improved properties can be as different as e.g. biological activity, enzyme washing performance, antibiotic resis¬tance etc.<br>
Consequently, which screening method to be used for identifying positive variants depend on the desired improved property of the polypeptide variant in question.<br>
If, for instance, the polypeptide in question is an enzyme and the desired improved functional property is the wash performance, the screening in step f) may conveniently be performed by use of a filter assay based on the following principle:<br>
The recombination host cell is incubated on a suitable medium and under suitable conditions for the enzyme to be secreted, the medium being provided with a double filter comprising a first protein-binding filter and on top of that a second filter exhibiting a low protein binding capability. The recombination host cell is located on the second filter. Subsequent to the incubation, the first filter comprising the enzyme secreted from the recombination host cell is separated from the second filter comprising said cells. The first filter is subjected to screening for the desired enzymatic activity and the corresponding microbial colonies present on the second filter are identified.<br>
The filter used for binding the enzymatic activity may be any protein binding filter e. g.   nylon or nitrocellulose. The topfilter<br><br>
carrying the colonies of the expression organism may be any filter that has no or low affinity for binding proteins e.g. cellulose acetate or Durapore6. The filter may be pre-treated with any of the conditions to be used for screening or may be treated during the detection of enzymatic activity.<br>
The enzymatic activity may be detected by a dye, fluorescence, precipitation, pH indicator, IR-absorbance or any other known technique for detection of enzymatic activity.<br>
The detecting compound may be immobilized by any immobilizing agent e.g. agarose, agar, gelatine, polyacrylamide, starch, filter paper, cloth; or any combination of immobilizing agents.<br>
If the improved functional property of the polypeptide is not sufficiently good after one cycle of shuffling, the polypeptide may be subjected to another cycle.<br>
In an embodiment of the invention at least one shuffling cycle is a backcrossing cycle with the initially used DNA fragment, which may be the wild-type DNA fragment. This eliminates non-essential muta¬tions. Non-essential mutations may also be eliminated by using wild-type DNA fragments as the initially used input DNA material.<br>
It is to be understood that the method of the invention is suitable for all types of polypeptide, including enzymes such as proteases, amylases, lipases, cutinases, amylases, cellulases, peroxidases and oxidases.<br>
Also contemplated according to the invention is polypeptides having biological activity such as insulin, ACTH, glucagon, somatostatin, somatotropin, thymosin, parathyroid hormone, pigmentary hormones, somatomedin, erythropoietin, luteinizing hormone, chorionic gonadotropin, hypothalamic releasing factors, antidiuretic hormones, thyroid stimulating hormone, relaxin, interferon, thrombopoietin (TPO) and prolactin.<br>
Especially contemplated according to the present invention is initially to use input DNA sequences being either wild-type, variant or modified DNA sequences, such as a DNA sequences coding for wild-type,  variant or modified enzymes,  respectively,  in particular<br><br>
enzymes exhibiting lipolytic activity.<br>
In an embodiment of the invention the lipolytic activity is a lipase activity derived from the filamentous fungi of the Humicola sp., in particular Humicola lanuginosa,   especially Humicola lanuginosa.<br>
In a specific embodiment of the invention the initially used input DNA fragment to be shuffled with a homologous polypeptide is the wild-type DNA sequence encoding the Humicola lanuginosa lipase derived from Humicola lanuginosa DSM 4109 described in EP 305 216 (Novo Nordisk A/S).<br>
Also specifically encompassed by the scope of the invention is input DNA sequences selected from the group of vectors (a) to (f) and/or DNA fragments (g) to (aa) coding for Humicola lanuginosa lipase variants from the list below in the Material and Method section.<br>
Throughout the present application the name Humicola  lanuginosa  has been used to identify one preferred parent enzyme, i.e. the one mentioned immediately above. However, in recent years H.   lanuginosa has also been termed Thermomyces  lanuginosus   (a species introduced the first time by Tsiklinsky in 1989} since the fungus show morphological and physiological similarity to Thermomyces lanuginosus.   Accordingly, it will be understood that whenever reference is made to H.   lanuginosa  this term could be replaced by Thermomyces  lanuginosus.   The DNA encoding part of the 18S ribosomal gene from Thermomyces  lanuginosus   (or H.   lanuginosa)   have been sequenced. The resulting 18S sequence was compared to other 18S sequences in the GenBank database and a phylogenetic analysis using parsimony (PAUP, Version 3.1.1, Smithsonian Institution, 1993) have also been made. This clearly assigns Thermomyces  lanuginosus  to the class of Plectomycetes,   probably to the order of Eurotiales. According to the Entrez Browser at the NCBI (National Center for Biotechnology Information), this relates Thermomyces  lanuginosus  to families like Eremascaceae,   Monoascaceae,   Pseudoeurotiaceae  and Trichocomaceae,   the latter containing genera like Emericeiia, Aspergillus,   Penicillium,   Eupenicillium,   Paecilomyces,   Talaromyces, Thermoascus  and Sclerocleista.<br>
Consequently, such genes encoding lipolytic enzymes of filamentous<br><br>
fungi of the genera Emericella,   Aspergillus,   Penicillium, Eupenicillium,   Paecilomyces,   Talaromyces,   Thermoascus  and Sclerocleista  are also specifically contemplated according to the present invention.<br>
Other examples of relevant filamentous fungi genes encoding lipolytic enzymes include strains of the Absidia sp. e.g. the strains listed in WO 96/13578 (from Novo Nordisk A/S) which are hereby incorporated by reference. Absidia sp. strains listed in WO 96/13578 include Absidia blakesleeana, Absidia corymbifera and Absidia  reflexa.<br>
Strains of Rhizopus sp., in particular Rh- niveus and Rh. oryzea are also contemplated according to the invention.<br>
The lipolytic gene may also be derived from a bacteria, such as a strain of the Pseudomonas sp., in particular Ps. fragi, Ps. stutzeri, Ps. cepacia and Ps. fluorescens (WO 89/04361), or Ps. plantarii or Ps. gladioli (US 4,950,417) or Ps. alcaligenes and Ps. pseudoalcaligenes (EP 218 272, EP 331 376, or WO 94/25578 (disclosing variants of the Ps. pseudoalcaligenes lipolytic enzyme), the Pseudomonas sp. variants disclosed in EP 407 225, or a Pseudomonas sp, lipolytic enzyme, such as the Ps. mendocina (also termed Ps. putida) lipolytic enzyme described in WO 88/09367 and US 5,389,536 or variants thereof as described in US 5,352,594, or Ps. auroginosa or Ps. glumae, or Ps. syringaer or Ps. wisconsinensis (WO 96/12012 from Solvay) or a strain of Bacillus sp., e.g. the B. subtilis described by Dartois et al •, (1993) Biochemica et Biophysica acta 1131, 253-260, or B. stearothermophilus (JP 64/7744992) or B. pumilus (WO 91/16422) or a strain of Streptomyces sp., e.g. S. scabies, or a strain of chroroobacterium sp. e.g C. viscosum.<br>
In connection with the Pseudomonas sp. lipases it has been found that lipases from the following organisms have a high degree of homology, such as at least 60% homology, at least 80% homology or at least 90% homology, and thus are contemplated to belong to the same family of lipases: Ps. ATCC21808, Pseudomonas sp. lipase commercially available as Liposam©, Ps. aeruginosa EF2, Ps. aeruginosa  PAC1R, Ps.   aeruginosa  PA01, ps.   aeruginosa  TE 3285, Ps.<br><br>
sp. 109, Ps. pseudoalcaligenes Ml, Ps. glumae, Ps. cepacia DSM 3959, Ps. cepacia M-12-33, Ps. sp. KHI-56, Ps. putida IFO 3458, Ps. putida IFO 12049 (Gilbert, E. J., (1993), Pseudomonas lipases: Biochemical properties and molecular cloning. Enzyme Microb. Technol., 15, 634-645) . The species Pseudojnonas cepacia has recently been reclassified as Burkholderia cepacia, but is termed Ps. cepacia in the present application.<br>
Also genes encoding lipolytic enzymes from yeasts are relevant, ans include lipolytic genes from Candida sp., in particular Candida rugosa,   or  Geotrichum  sp.,   in particular Geotrichum  candidum.<br>
Specific examples of microorganisms comprising genes encoding lipolytic enzymes used for commercially available products and which may serve as donor of genes to be shuffled according to the invention include Humicola lanuginosa, used in Lipolase®, Lipolase® Ultra, Ps. mendocina used in Lumafast®, Ps. alcaligenes used in Lipomax®, Fusarium solani, Bacillus sp. {US 5427936, EP 528828), Ps.   mendocina,  used in Liposam®.<br>
It is to be emphasized that genes encoding lipolytic enzyme to be shuffled according to the invention may be any of the above mentioned genes of lipolytic enzymes and any variant, modification, or truncation thereof. Examples of such genes which are specifically contemplated include the genes encoding the enzymes described in WO 92/05249, WO 94/01541, WO 94/14951, WO 94/25577, WO 95/22615 and a protein engineered lipase variants as described in EP 407 225; a protein engineered Ps. mendocina lipase as described in US 5,352,594; a cutinase variant as described in WO 94/14964; a variant of an Aspergillus lipolytic enzyme as described in EP patent 167,309;   and  Pseudomonas  sp.   lipase   described  in WO   95/06720.<br>
A request to the DNA sequences, encoding the polypeptide(s), to be shuffled, is that they are at least 60%, preferably at least 70%, better more than 80%, especially more than 90%, and even better up to almost 100% homologous. DNA sequences being less homologous will have less  inclination to interact and recombine.<br>
It is also contemplated according to the invention to shuffle parent (homologous)   wildt  type  organisms of  different  genera.<br>
Further,    the   DNA   fragment (s)    to   be   shuffled  mav   nrefor^hl v   ha^*»   a<br><br>
length of from about 20 bp to 8 kb, preferably about 40 bp to 6 kb, more preferred about 80 bp to 4 kb, especially about 100 bp to 2 kb, to be able to interact optimally with the opened plasmid.<br>
The method of the invention is very efficient for preparing po¬lypeptide variants in comparison to prior art method comprising transforming linear DNA fragments/sequences.<br>
The inventor found that the transformation frequency of a mixture of opened plasmid and a DNA fragment were significantly higher than when transforming a plasmid cut at the same site alone. The trans¬formation frequency of the opened plasmid and DNA fragment were as high as for uncut plasmid.<br>
Without being limited to any theory it is believed that the opening of the plasmid(s) restrict(s) the replication of (opened) plasmid(s) when not interacting with at least one DNA fragment. In accordance with this an increased number of recorabined DNA sequences were found after only one shuffling cycle.<br>
As described in Example 1 50% of the resulting transformants contained recombined DNA sequences of both input DNA sequences. As high as 20% of the total number of recombined DNA sequences were "random" mixtures (i.e. having more than one region of nucleotides exchanged).<br>
The input DNA sequences may be any DNA sequences including wild-type DNA sequences, DNA sequences encoding variants or mutants, or modifications thereof, such as extended or elongated DNA sequences, and may also be the outcome of DNA sequences having been subjected to one or more cycles of shuffling (i.e. output DNA sequences} according to the method of the invention or any other method (e.g. any of the methods described in the prior art section).<br>
When using the method of the invention the output DNA sequences (i.e. shuffled DNA sequences), have had a number of nucleotide(s) exchanged. This results in replacement of at least one amino acid within the polypeptide variant, if comparing it with the parent polypeptide. It is to be understood that also silent mutations is contemplated (i.e. nucleotide exchange which does not result in changes in the amino acid sequence).<br><br>
However, the method of the present invention will in most cases lead to the replacement of a considerable number of amino acid and may in certain cases even alter the structure of one or more polypeptide domains (i.e. a folded unit of polypeptide structure).<br>
According to the present invention more than two DNA sequences are shuffled at the same time. Actually any number of different DNA fragments and homologous polypeptides comprised in suitable plasmids may be shuffles at the same time. This is advantageous as a vast number of quite different variants can be made rapidly without an abundance of iterative procedures.<br>
The inventor have tested the nucleotide shuffling method of the invention using significantly more than two homologous DNA sequences. As described in Example 2 it was surprisingly found that the method of the invention advantageously can be used for recombining more than two DNA sequences.<br>
One cycle of shuffling according to the method of the invention may result in the exchange of from 1 to 1000 nucleotides into the opened plasmid DNA sequence encoding the polypeptide in question. The exchanged nucleotide sequence(s) may be continuous or may be present as a number of sub-sequences within the full-length sequence(s).<br>
To support the present invention the inventor made a number of additional experiments on different aspect on the method of the invention. The experiments are described below and illustrated in the Example 3 to 6 below.<br>
A number of vectors and fragments comprising an inactivated synthetic Humicola   lanuginosa   lipase genes were constructed by introducing frameshift/stop codon mutations in the lipase gene at various positions.  These were used for monitoring the in vivo recombination of different combinations of opened vector(s) and DNA fragments. The number of active lipase colonies were scored as described in Example 3. The number of colonies determines the efficiency of the opened vector(s) and fragment(s) recombination.<br>
One frameshift mutation in said Humicola  lanuginosa  lipase gene in the opened vector and another in the fragment on the opposite side<br><br>
of the opening site gave 3 to 32% of active lipase colonies depending on the location and combination. It was concluded that the closer that the mutation is at the ends of the vector the higher mixing.<br>
One frameshift mutation in the opened vector and two in the fragment on each side of the opening site gave 4 to 42% of active colonies depending on the location and combination. Some of these active colonies can be considered to be mosaics, not only related to the opening site.<br>
Two frameshift mutations in the opened vector on each side of the opening site and one in the fragment gave 0.5 to 3.1% of active colonies depending on the location and combination. Most of these active colonies are mosaics of the "parent" DNA.<br>
Two frameshift mutations in the opened vector on each side of the opening site and a wild type fragment gave 7.7 to 10.7% of active colonies depending on the location.<br>
It was also found that the amount of vectors relative to fragments and the size of the fragments are also influencing the result.<br>
Using of the S.   cerevisiae  rad52 mutants as the recombination host cell showed that the rad52  mutant transformed very well with wild type plasmid(s) and expressed the Humicola  lanuginosa  lipase gene, but gave no transformants at all with the opened vectors and fragments.<br>
The RAD52 function is required for "classical recombination" (but not for unequal sister-strand mitotic recombination) showing that the recombination of opened vector and fragment could involve a classical recombination mechanism.<br>
Classical recombination is the recombination mechanism involved in the recombination between genes located on nonsister chromatids of homologous chromosomes as defined in for example Petes TD, Halone RE and Symington LS (1991) "Recombination in Yeast", page 407-522, in The Molecular and Cellular Biology of the Yeast Saccharomyces, Volume 1 (eds. Broach JR, Pringle JR and Jones EW), Cold Spring Harbor Laboratory Press, New York.<br><br>
Multiple partially overlapping fraqements<br>
The inventor also tested recombination of multiple partial<br>
overlapping fragments using the method of the invention.<br>
The recombination of 2 and 3 partial overlapping fragments into a gapped (i.e. that the opening result in cutting out of a little part of the gene) vector were tested and gave a high recovery of recombined Humicola   lanuginosa  lipase gene. The recovery of active lipase gene from different combinations of inactivated Humicola lanuginosa  genes was tested for the recombination of 2 partial overlapping fragments. The tendency was a higher mixing in the overlapping region between the 2 fragments in the gapped region than in the vector and fragment overlap.<br>
When recombining many fragments from the same region, the multiple overlapping fragment technique will increase the mixing by itself, but it is also important to have a relative high random mixing in overlapping regions in order to mix closely located variants/differences.<br>
An overlap as small as 10 bp between two fragments were found to be sufficient to obtain a very efficient recombination. Therefore, overlapping in the range from 5 to 5000 bp, preferably from 10 bp to 500 bp, especially 10 bp to 100 bp is suitable according to the method of the invention.<br>
According to this embodiment of the present invention 2 or more overlapping fragments, preferable 2 to 6 overlapping fragments, especially 2 to 4 overlapping fragments may advantageously be used as input fragments in a shuffling cycle.<br>
Besides increasing the mixing of genes, this is a very useful method for domain shuffling by creating small overlaps between DNA fragments from different domains and screen for the best combination.<br>
For instance, in the case of three DNR fragments the overlapping regions may be as follows:<br>
- the first end of the first fragment overlaps the first end. of the opened plasmid,<br><br>
-	the first end of the second fragment overlaps the second end of the first fragment, and the second end of the second fragment overlaps the first end of the third fragment,<br>
-	the first end of the third fragment overlaps (as stated above) the second end of the second fragment, and the second end of the third fragment overlaps the second end of the opened plasmid.<br>
It is to be understood that when using two or more DNA fragments as starting material it is preferred to have continuos overlaps between the ends of the plasmid and the DNA fragments.<br>
Even though it is preferred to shuffle homologous DNA sequences in the form of DNA fragment (s) and opened plasmid(s) , it is also contemplated according to the invention to shuffle two or more opened plasmids comprising homologous DNA sequences encoding polypeptides. However, in such case it is compulsory to open the plasmids at different sites.<br>
In an further embodiment of the invention two or more opened plasmids and one or more homologous DNA fragments are used as the starting material to be shuffled. The ratio between the opened plasmid(s) and homologous DNA fragment(s} preferably lie in the range from 20:1 to 1:50, preferable from 2:1 to 1:10 (mol vectonmol fragments) with the specific concentrations being from 1 pH to 10 M of the DNA.<br>
The opened plasmids may advantagously be gapped in such a way that the overlap between the fragments is deleted in the vector in order to select for the recombination).<br>
Preparing the DNA fragment<br>
The DNA fragment to be shuffled with the homologous polypeptide comprised in an opened plasmid may be prepared by any suitable method. For instance, the DNA fragment may be prepared by PCR amplification (polymerase chain reaction), as described above, of a plasmid or vector comprising the gene of the polypeptide, using specific primers, for instance as described in US 4,683,202 or Saiki et al., (1988), Science 239, 487 - 491. The DNA fragment may also be cut out from a vector or plasmid comprising the desired DNA sequence by digestion with restriction enzymes, followed by isolation using e.g.   electrophoresis.<br><br>
The DNA fragment encoding the homologous polypeptide in question may alternatively be prepared synthetically by established standard methods, e.g. the phosphoamidite method described by Beaucage and Caruthers, (1981), Tetrahedron Letters 22, 1859 - 1869, or the method described by Matthes et al., (1984), EMBO Journal 3, 801 -805. According to the phosphoamidite method, oligonucleotides are synthesized, e.g. in an automatic DNA synthesizer, purified, annealed, ligated and cloned in suitable vectors.<br>
Furthermore, the DNA fragment may be of mixed synthetic and genomic, mixed synthetic and cDNA or mixed genomic and cDNA origin prepared by ligating fragments of synthetic, genomic or cDNA origin (as appropriate), the fragments corresponding to various parts of the entire DNA sequence, in accordance with standard techniques.<br>
The plasmid<br>
The plasitiid comprising the DNA sequence encoding the polypeptide in question may be prepared by ligating said DNA sequence into a suitable vector or plasmid, or by any other suitable method.<br>
Said vector may be any vector which may conveniently be subjected to recombinant DNA procedures. The choice of vector will often depend on the recombination host cell into which it is to be introduced.<br>
Thus, the vector may be an autonomously replicating vector, i.e. a vector which exists as an extrachromosomal entity, the replication of which is independent of chromosomal replication, e.g. a plasmid. Alternatively, the vector may be one which, when introduced into the recombination host cell, is integrated into the host cell genome and replicated together with the chromosome (s) into which it has been integrated.<br>
To facilitate the screening process it is preferred that the vector is an expression vector in which the DNA sequence encoding the polypeptide in question is operably linked to additional segments required for transcription of the DNA. In general, the expression vector is derived from a plasmid, a cosmid or a bacteriophage, or may contain elements of any or all of these.<br><br>
The term, "operably linked" indicates that the segments are arranged so that they function in concert for their intended purposes, e.g. transcription initiates in a promoter and proceeds through the DNA sequence coding for the polypeptide in question.<br>
The promoter may be any DNA sequence which shows transcriptional activity in the recombination host cell of choice and may be derived from genes encoding proteins, such as enzymes, either homologous or heterologous to the host cell.<br>
Examples of suitable promoters for use in yeast host cells include promoters from yeast glycolytic genes (Hitzeman et al.,(1980), J. Biol. Chem. 255, 120"J3 - 12080; Alber and Kawasaki, (1982), J. Mol. Appl. Gen. 1, 419 - 434) or alcohol dehydrogenase genes (young et al., in Genetic Engineering of Microorganisms for Chemicals (Hollaender et al, eds.), Plenum Press, New York, 1982), or the TP11 (US 4,599,311) or ADH2-4c (Russell et al., (1983), Nature 304, 652 -654) promoters.<br>
Examples of suitable promoters for use in filamentous fungus host cells are, for instance, the ADH3 promoter (HcKnight et al., (1985), The EMBO J. 4, 2093 - 2099) or the tpiA promoter. Examples of other useful promoters are those derived from the gene encoding A. oryzae TAKA amylase, Rhizomucor miehei aspartic proteinase, A. niger neu¬tral a-amylase, A. niger acid stable a-amylase, A. niger or A. awamori glucoamylase (gluA), Rhizomucor miehei lipase, A, oryzae alkaline protease, A. oryzae triose phosphate isomerase or A. nidulans acetamidase. Preferred are the TAKA-amylase and gluA promoters.<br>
The DNA sequence encoding polypeptide in question invention may also, if necessary, be operably connected to a suitable terminator, such as the human growth hormone terminator (Palmiter et al., op. cit.) or (for fungal hosts) the TPI1 (Alber and Kawasaki, op. cit.) or ADH3 (McKnight et al., op_^_ cit.) terminators. The vector may further comprise elements such as polyadenylation signals (e.g. from SV40 or the adenovirus 5 Elb region), transcriptional enhancer sequences (e.g. the SV40 enhancer) and translational enhancer sequences (e.g. the ones encoding adenovirus VA RNAs).<br><br>
The vector may further comprise a DNA sequence enabling the vector to replicate in the recombination host cell in question. When the host cell is a yeast cell, suitable sequences enabling the vector to replicate are the yeast plasmid 2m replication genes REP 1-3 and origin of replication.<br>
The plasmid pYl can be used for production of useful proteins and peptides, using filamentous fungi, such as Aspergillus sp., and yeasts as recombinant host cells (JP06245777-A).<br>
The vector may also comprise a selectable marker, e.g. a gene the product of which complements a defect in the recombination host cell, such as the gene coding for dihydrofolate reductase (DHFR) or the Schizosaccharomyces pombe TPI gene (described by P.R. Russell, (1985), Gene 40, 125-130).<br>
Another example of such suitable selective markers are the ura3 and leu2 genes which complements the corresponding defect genes of e.g. the yeast strain Sacchazomyces cerevisiae  YNG318.<br>
The vector may also comprise a selectable marker which confers resistance to a drug, e.g. ampicillin, kanamycin, tetracyclin, chloramphenicol, neomycin, hygromycin or methotrexate. For fi¬lamentous fungi, selectable markers include amdS, pyrG, argB, niaD, sC, trpC, pyr4, and DHFR.<br>
To direct the polypeptide in question into the secretory pathway of the recombination host cell, a secretory signal sequence (also known as a leader sequence, prepro sequence or pre sequence) may be provided in the recombinant vector. The secretory signal sequence is joined to the DNA sequence encoding the lipolytic enzyme in the correct reading frame, Secretory s ignal sequences are commonly positioned 5' to the DNA sequence encoding the polypeptide. The secretory signal sequence may be the signal normally associated with the polypeptide in question or may be from a gene encoding another secreted protein.<br>
The signal peptide may be naturally occurring signal peptide, or a functional part thereof, or it may be a synthetic peptide. For secretion from yeast cells, suitable signal peptides have been found to be the a-factor signal peptide (cf. US 4,870,008), the signal<br><br>
peptide of mouse salivary amylase (cf. 0. Hagenbuchle et al,, (1981), Nature 289, 643-646), a modified carboxypeptidase signal peptide (cf, L-A. Vails et al., (1987) , Cell 48, 887-897) , the Humicola lanuginosa lipase signal peptide, the yeast BAR1 signal peptide (cf. WO 87/02670), or the yeast aspartic protease 3 (YAF3) signal peptide (cf. M. Egel-Mitani et al., (1990) , Yeast 6, 127-137).<br>
For efficient secretion in yeast, a sequence encoding a leader peptide may also be inserted downstream of the signal sequence and upstream of the DNA sequence encoding the polypeptide in question. The function of the leader peptide is to allow the expressed polypeptide to be directed from the endoplasmic reticulum to the Golgi apparatus and further to a secretory vesicle for secretion into the culture medium (i.e. exportation of the polypeptide across the cell wall or at least through the cellular membrane into the periplasmic space of the yeast cell). The leader peptide may be the yeast a-factor leader {the use of which is described in e.g. US 4,546,082, EP 16 201, EP 123 294, EP 123 544 and EP 163 529). Alternatively, the leader peptide may be a synthetic leader peptide, which is to say a leader peptide not found in nature. Synthetic leader peptides may, for instance, be constructed as described in WO 89/02463 or WO 92/11378.<br>
For use in filamentous fungi, the signal peptide may conveniently be derived from a gene encoding an Aspergillus sp. amylase or glucoamylase, a gene encoding a Rhizomucor miehei lipase or protease, a Humicola lanuginosa lipase. The signal peptide is preferably derived from a gene encoding A. oryzae TAKA amylase, A. niger neutral ct-amylase, A. niger acid-stable amylase, or A, niger glucoamylase.<br>
The recombination host cell<br>
The recombination host cell, into which the mixture of plas-mid/fragment DNA sequences are to be introduced, may be any eukaryotic cell, including fungal cells and plant cells, capable of recombining the homologous DNA sequences in question.<br>
According to prior art prokaryotic microorganisms, such as bacteria including Bacillus    and E.     coli;    eukaryotic organisms,  such as<br><br>
filamentous fungi, including Aspergillus and yeasts such as Saccharomyces cerevisiae; and tissue culture cells from avian or mammalian origins have been suggested for in vivo recombination. All of said organisms can be used as recombination host cell, but in general prokaryotic cells are not sufficiently effective (i.e. does not result in a sufficient number of variants) to be suitable for recombination methods for industrial use.<br>
Consequently, preferred recombination host cells according to the present invention are fungal cells, such as yeast cells or filament¬ous fungi.<br>
Examples of suitable yeast cells include cells of Saccharomyces sp., in particular strains of Saccharomyces cerevisiae or Saccharomyces kluyveri or Schizosaccharomyces sp., Methods for transforming yeast cells with heterologous DNA and producing heterologous polypeptides therefrom are described, e.g. in US 4,599,311, US 4,931,373, US 4,870,008, 5,037,743, and US 4,845,075, all of which are hereby incorporated by reference. Transformed cells may be selected by, e.g., a phenotype determined by a selectable marker, commonly drug resistance or the ability to grow in the absence of a particular nutrient, e.g. leucine. A preferred vector for use in yeast is the POT1 vector disclosed in US 4,931,373. The DNA sequence encoding the polypeptide may be preceded by a signal sequence and optionally a leader sequence, e.g. as described above. Further examples of suitable yeast cells are strains of Kluyveromyces, such as K. lactis, Hansenula, e.g. M. polymorphs, or Pichia, e.g. P. pastoris (cf. Gleeson et al..(1986J, J- Gen. Microbiol. 132, 3459-3465; US 4,882,279).<br>
Examples of other fungal cells are cells of filamentous fungi, e.g. Aspergillus sp., Weurospora sp., Fusarium sp. or Trichoderma sp., in particular strains of A. oryzae, A. nidulans or A. niger. The use of Aspergillus sp. for the expression of proteins is described in, e.g., EP 272 277, EP 230 023. The transformation of F. oxysporum may, for instance, be carried out as described by Malardier et al., (1989), Gene 78, 147-156.<br>
In a preferred embodiment of the invention the recombination host cell is a cell of the genus Saccharomyces, in particular S. cerevisiae.<br><br>
METHODS  AND MATERIALS<br><br><br>
Strains:<br>
Expression system host;<br>
Saccharomyces   cerevisiae   YNG318: MATa Dpep4[cir*] ura3-52, Ieu2-D2,<br>
his 4-539<br>
Saccharomyces    cerevisiae   Rad52:    Strain    M1533   = MATa    rad52    ura3,<br>
obtained from Torsten Nilsson Tillgren,  Institute of Genetics,<br>
University of Copenhagen.<br>
Plasmids:<br>
pJS026 (see figure 3) pJS037 (see figure 4) pYES 2.0 (Invitrogen)<br>
Transformation selective marker<br>
ura3<br>
leu2<br>
Media<br>
SC-ura": 90 ml 10 x Basal salt, 22.5 ml 20% casamino acids, 9 ml 1%<br>
tryptophan, H20 ad 806 ml, autoclaved, 3.6 ml 5% threonine and 90 ml<br>
20% glucose or 20% galactose added.<br>
LB-medium: 10 g Bacto-tryptone, 5 g Bacto yeast extract, 10 g NaCl<br>
in 1 litre water.<br>
Brilliant Green (BG) (Merck, art. No. 1.01310)<br>
BG-reagent: 4 mg/ml Brilliant Green (BG) dissolved in water<br>
Substrate 1:<br>
10 ml olive oil (Sigma CAT NO. 0-1500)<br>
20 ml 2% polyvinyl alcohol (PVA)<br>
The Substrate is homogenised for 15-20 minutes.<br>
Methods:<br>
Construction of yeast expression vector<br>
The expression plasmids pJS026 and pJS037, are derived from pYES 2.0. The inducible GAL1-promoter of pYES 2.0 was replaced with the constitutively expressed TPI (triose phosphate isomerase)-promoter from Saccharomyces cerevisiae (Albert and Karwasaki, (1982), J. Mol. Appl Genet., 1, 419-434), and the ura3 promoter has been deleted. A restriction map of pJS026 and pJS037 is shown in figure 3 and figure 4, respectively.<br><br>
Preparation of the wild-type DNA fragment<br>
A lipase wild-type DNA fragment can be prepared either by PCR amplification (resulting in low, medium or high mutagenesis), of the pJS026 plasmid or by cutting the DNA fragment out by digesting with a suitable restriction enzyme.<br>
Fermentation of Humicola lanuginosa  lipase variants in yeast<br>
10 ml of SC~ura" medium is inoculated with a S.    cerevisiae   colony<br>
and grown at 30°C for 2 days. The 10 ml is used for inoculating 300<br>
ml SC-ura~ medium which is grown at 30°C for 3 days. The 300 ml is<br>
used for inoculation 5 1 of the following G-substrate:<br>
400 g    Ami case<br>
6.7 g    yeast extract (Difco)<br>
12.5 g   L-Leucin (Fluka)<br>
6.7 g    (NH«)jS04<br>
10  g	MgS04-7H2O<br>
17  g	K2S04<br>
10 ml    Trace compounds<br>
5 ml     Vitamin solution<br>
6.7 ml   H3PO4<br>
25 ml    20% Pluronic {antifcam}<br>
In a total volume of 5000 ml:<br>
The yeast cells are fermented for 5 days at 30°C. They are given a start dosage of 100 ml 70% glucose and added 400 ml 70% glucose/day. A pH=5.0 is kept by addition of a 10% NH3 solution. Agitation is 300 rpm for the first 22 hours followed by 900 rpm for the rest of the fermentation. Air is given with 11 air/1/min for the first 22 hours followed by 1.5 1 air/1/min for the rest of the fermentation.<br>
Trace compounds:<br>
6.8  g	ZnCl2<br>
54.0 g	FeCl2-6H20<br>
19.1 g	MnCl2"4H20<br>
2.2  g	CuS<v5h></v5h>
2.5B   g	CoCl2<br>
0.62 g	H3B03<br>
0.024  g	(NH4)6Ho,034-4H20<br>
0.2  g	KI<br><br>
100 ml   HC1 (concentrated) In a total volume of 1 1.<br>
Vitamin, solution:<br>
250 mg   Biotin<br>
3 g     Thiamin<br>
10 g     D-Calciumpanthetonat<br>
100 g    Myo-Inositol<br>
50 g     Cholinchlorid<br>
1.6 g    Pyridoxin<br>
1.2 g    Niacinamid<br>
0.4 g    folicacid<br>
0.4 g    Riboflavin<br>
In a total volume of 11.<br>
Transformation of yeast<br>
Saccharomyces cerevisiae is transformed by standard methods (cf. Sambrooks et al., (1989), Molecular Cloning: A Laboratory Manual, 2nd Ed., cold Spring Harbor)<br>
Determination of yeast transformation frequency<br>
The transformation frequency is determined by cultivating the transformants on SC-ura"plates for 3 days and counting the number of colonies appearing. The number of transformants per mg opened plasmid is the transformation frequency.<br>
Screening for positive variants with improved wash performance<br>
The following filter assay can be used for screening positive<br>
variants with improved wash performance.<br>
Low calcium filter assay<br>
1)	Provide SC Ura" replica plates (useful for selecting strains carrying the expression vector) with a first protein binding filter (Nylon membrane) and a second low protein binding filter (Cellulose acetate) on the top.<br>
2)	Spread yeast cells containing a parent lipase gene or a mutated lipase gene on the double filter and incubate for 2 or 3 days at 30°C.<br>
3)	Keep the colonies on the top filter by transferring the top-filter to a new plate.<br><br>
4)	Remove the protein binding filter to an empty petri dish.<br>
5)	Pour an agarose solution comprising an olive oil emulsion [2% PVAiolive oil=3:l), Brilliant green (indicator,0.004%), 100 mM tris buffer pH9 and EGTA (final concentration 5mM) on the bottom filter so as to identify colonies expressing lipase activity in the form of blue-green spots.<br>
6)	Identify colonies found in step 5) having a reduced dependency for calcium as compared to the parent lipase.<br>
DNA sequencing was performed by using applied Biosystems ABI DNA sequence model 373A according to the protocol in the ABI Dye Terminator Cycle Sequencing kit.<br>
Assessing the effiency of recombination<br>
The number of colonies determines the efficiency of the opened vector and fragment recombination. The percentage of colonies with active lipase activity gives an estimate of the mixing of the active and inactive genes - theoretically it can be calculated for one frameshift that the closer to 50% the better mixing if equal likelihood of wild type and frameshift, 25% for 2 frameshifts and 12.5% for 3 frameshifts.<br>
Frameshift mutation<br>
The frameshift mutation were created either by filling in a restriction site (in case of 5' overhang) or deleting the "sticky ends" (in case of 3' overhang) by T4 DNA polymerase with or without dNTP (deoxynucleotides = equal amounts of dATP, dTTP, dCTP and dGTP). Methods for filling in of restriction sites (referred to as "F" on Figure 7) and deleting the sticky ends (referred to as "(D)" on Figure 7) are well known in the art.<br>
Method for assessing colonies with lipase activity<br>
The number of colonies and positives (i.e. with lipase activity) are<br>
calculated as the average of 3 plates.<br>
The cultivation condition and screening condition used is the<br>
following:<br>
1)	Provide SC Ura-plates with a protein binding filter (Nylon<br>
filter) onto the plate.<br>
2)	Spread yeast cells containing a parent lipase gene or a mutated<br>
lipase gene on the filter and incubate for 3 or 4 days at 30°C.<br><br>
3) Remove the protein binding filter with the colonies to a petri dish containing: An agarose solution comprising an olive oil emulsion (2% PVfl:01ive oil«2:l). Brilliant green (indicator,0.004%) , 100 mM tris buffer pH 9.<br>
5) Identify colonies expressing lipase activity in the form of blue-green spots.<br>
EXAMPLES<br>
Example 1<br>
Testing in vivo  recombination of two homologous genes<br>
The Saccharomyces      cerevisiae     expression  plasmid  pJS02 6  was<br>
constructed as described above in the "Material and Methods"-<br>
section.<br>
A synthetic Humicola lanuginosa lipase gene (in pJS037) containing 12 additional restriction sites (see figure 4) was cut with Nrul, PstI, and Nrul and PstI, respectively, to open the gene approximately in the middle of the DNA sequence encoding the lipase.<br>
The opened plasmid (pJS037) was transformed into Saccharomyces cerevisiae YNG318 together with an about 0.9 kb wild-type Humicola lanuginosa lipase DNA fragment (see figure 1) prepared from pJS026 by PCR amplification.<br>
Further, the opened plasmid was also transformed into the yeast recombination host cell alone (i.e. without the 0.9 kb Synthetic lipase DNA fragment).<br>
The transformed yeast cells were grown as described in the "Ma¬terials and Method"-section above, and the transformation frequency was determined as described above.<br>
It was found that the transformation frequency of the opened plasmid alone was very low (10 transformants per mg opened plasmid), in comparison to the transformation frequency of said plasmid/fragment {50,000 transformants per mg opened plasmid).<br>
The plasmid/fragment was PCR amplified resulting in 20 transformants containing  fragments  covering  the  lipase  gene  region of the<br><br>
recombined plasmid/fragments. The recombination mixture of the 20 transformants were analyzed by restriction site digestion using standard methods. The result is displayed in Table 1.<br><br>
As can bee seen from Table 1 10 transformants (equivalent to 50%) contained recombined DNA sequences. 4 of these 10 DMA sequences (equivalent to 20%) contained either a region of the wild-type gene recombined into the synthetic gene or a region of the synthetic gene recombined into the wild-type fragment.<br>
Example 2<br>
In vivo  recombination of Humicola lanuginosa  lipase variants<br>
The DNA sequences of 20 variants of the Humicola   lanuginosa   lipase<br>
were in vivo  recombined in the same mixture.<br><br>
Six vectors were prepared from the lipase variants (a) to (f) (see the list above) by ligation into the yeast expression vector pJS037. All vectors were cut open with Nrul.<br>
DNA fragment of all 20 homologous DNA sequences (g) to (aa) (see the list above) were prepared by PCR amplification using standard methods.<br>
The 20 DNA fragments and the 6 opened vectors were mixed and transformed into the yeast Saccharomyces cerevisiae YNG318 by standard methods. The recombination host cell was cultivated as described above and screened as described above. About 20 trans¬formants were isolated and tested for improved wash performance using the filter assay method described in the "Material and Methods"-section.<br>
Two positive transformants (named A and B) were identified using the filter assay.<br>
In comparison to the wild-type amino acid sequence the two re-combined positive transformants had the following mutations.<br>
A:       D57G, N94K, D96L, P256T<br>
A is a recombination of two variants.<br>
	 originates from the vector (d)<br>
===== originates from the DNA fragment prepared from variant (y)<br>
B:  D57G, G59V, N94K, D96L, L97M, S116P, S170P, N249R<br>
	  ????  	  	  	  
B is a recombination of vector (c) , DNA fragments (n) and (u).<br>
	 originates from the vector (c)<br>
«« originates from the DNA fragment prepared from variant (u)<br>
===== originates from the DNA fragment prepared from variant (n)<br>
???? Amino acid mutation which is not a result of recombination.<br>
As can be seen the resulting positive variants have been formed by recombination two or more variants. The amino acid mutations marked "?????" are not a result of in vivo recombination, as none of the shuffled lipase variants (see the list above) comprise any of said mutations. Consequently, these mutations are a result of random<br><br>
mutagenesis  arisen during preparation of the  DNA fragments by standard PCR amplification.<br>
Example 3<br>
Recombination with one frameshift mutantions<br>
Synthetic Humicola   lanuginosa   lipase gene (in vector JS037) was made inactive at various positions by deleting (positions 164/385) or filling-in (position 290/317/518/746) restriction enzyme sites or by site-directed introduction of a stop codon. All inactive synthetic lipase genes of 900 bp can be deduced from Figure 7).<br>
A number of different 900 bp DNA fragments were made from the above vectors using primer 4699 and primer 5164 using standard PCR technique. Smaller PCR fragments were made using primer 8487 and primer 4548 (260bp), primer 2843 and primer 4548 (488bp).<br>
0.5 ml Capp. 0.1 mg) of vectors Blue 425, Blue 426, Blue 428 and Blue 429, opened with Pst I (i.e. position 385), vectors Blue 424 and Blue 425 opened with Nrul (i.e. position 464) were together with 3 ml (app. 0.5 mg) of  fragments 424, 425, 426, 428, 429 in varios combination transformed into 100 ml Sacchromyces  cerevisiae YNG318 competent cells as displayed in Table 1A.<br>
The number of colonies and positives (i.e. with lipase activity) were calculated as the average of 3 plates as described in the Material and Methods section.<br>
The result of the test is shown in Table 1A<br>
Table 1A<br><br>
vector -f Fragment	Number of colonies	* of colonies with active lipase activity<br>
1.   Blue   428  +   429a	774	16%<br>
2.   Blue   429  +   428*	645	3%<br>
3.   Blue   426  + 4254	276	25%<br>
4.   Blue   425   +   426	528	18%<br>
5.   Blue  425/Nru   I + 426	539	28%<br><br>
6.   Blue  425 +  424	139	7%<br>
7. Blue   424/NruI  + 425"	74	32%<br>
8.   Blue   428   +   425	81	12%<br>
9.   Blue  428  +   wt fragment	317	37%<br>
Pairwise recombinations of one frameshift mutation on the vector and another on the fragment on the opposite side of the opening site, a determined by 9 plates; # determined by 6 plates.<br>
The first 2 rows of Table 1A displays vectors and fragments with a frameshift on each side of the PstI site. The "mirror image" experiment in row 2 compared to row 1 gives a reproducible lower number of active colonies. The same is true for row 3 and 4 even though it is not as pronounced. Moving the opening site closer to the frameshift in the vector increases the number of actives as seen in row 5. This can explain the reason for the difference in the "mirror image" experiments. In both cases the higher number of positives has the opening site closer to the frameshift in the vector.<br>
It can therefore be concluded that the closer the mutation is to the end of the vector the higher chance of mixing. This is probably arising from the well known fact that free DNA ends have a high recombinogenic potential. Therefore it is desirable to have as many free DNA ends as possible to increase the mixing of the genes. This is for example obtained in the later example with recombination of multiple overlapping fragments.<br>
Row 6 has a rather low number of actives probably due to the location of the frameshift on the fragment exactly at the PstI opening site of the vector.<br>
Row 7 has the frameshift of the vector close to the opening site and again it gives a high number of actives.<br>
Recombination with one stop codon mutantions<br>
In order to test if there are any difference in the recombination efficiency of stop codon mutations compared to frameshift mutations the following experiments were made..<br>
The same way as described above 0.5 ml (app. 0.1 mg) vectors Blue 624, Blue 625 and Blue 626 (see Table IB) opened with PstI<br><br>
comprising stop codons at specified positions (positions 184, 317 and 746, respectively) (perpared by site-directed mutagenesis) were together with 3 ml (app. 0.5 mg) of  fragments 624, 625 and 626 transformed into 100 ml Sacchromyces  cerevisiae  YNG318 competent cells in varios combination as displayed in Table IB.<br><br>
Pairwise recombinations of one stop codon mutation on the vector and another on the fragment on the opposite side of the opening site. ND = not determined but a high number.<br>
Row 1 and 2 (in Table IB) have the mutations located at the same place as row 1 and 2 in Table 1A. As can be seen the number of colonies with lipase activity is clearly higher for the stop codon mutations compared to the frameshift mutations, but the same relative difference between the "mirror image" experiments.<br>
This might indicate that the stop codon mutations, which is closer to the "application" of the method, gives a better mixing than frameshift mutations. Bow 3 and 4 confirms that the closer the mutation is to the end of the vector the higher chance of mixing.<br>
Recombination with  one or two frameshift mutation in the vector and one or two frameshift mutations in the fragment<br>
Using the same approach as described above the influence of one or two frameshift mutations in the vector and one or two frameshift mutations in the fragment were tested using vectors Blue 425, 426 and 428 (one mutation) and vectors Blue 442, Blue 443 (two mutations)  and fragments 442 and 443 (two frameshift mutations) and fragments 424, 425, 426, 427, 428 (one mutation) and wild-type (no mutation)<br><br>
The vectors Blue 442 and 443 are double frameshift mutations: Blue 442 = 428+429 and blue 443 = 427+429 (see Figure 7).<br>
Recombination was performed by transforming 0.5 ml vector (app. 0.1 mg) opened with PstI and 3 ml PCR-fragment (app. 0.5 mg)  into 100 ml Sacchromyces  cerevlsiae  YNG31S competent cells.<br>
The result of the test is shown in Table 2A and Table 2B<br><br><br>
frameshift on the vector) except for the last row where the frameshift on the vector is located far from the opening site. Lane 4 has fewer actives than lane 3 probably due to that the frameshift on the vector is located further away from the opening site than the frameshift on the fragment making the active genes mosaics that are not related to the opening site (see figure 2A), In Table 2B a very low number of actives are observed when there are 2 frameshifts located on the vector. Most of these active colonies are mosaics of the "parent" DNA meaning that the mixing is not related to the opening site (see figure 2B).<br>
Recombination with two different vectors or fragments<br><br>
A low number of colonies are seen for the control experiment in row 1 of table 3 as expected. The fragment added in the middle row has two frameshifts each corresponding to the frameshift on each vector. Via a tripartite recombination 4.2% actives are created, with two fragments with each one frameshifts and a vector with the same two frameshifts very few actives are found.<br>
Recombination with vectors opened at different sites<br>
Opening the vector in one side instead of approximately in the middle still gives good recombination as shown in Table 4. Two<br><br>
vectors opened at different sites can also recombine to some extent (compare with the vector controls in table 13).<br><br><br>
The size of the fragment also influences the recombination result as seen in Table 6.<br><br>
Example 4<br>
Test of S.   cerevisiae  mutants altered in recombination<br>
Using the same approach as described in Example 3 recombination of opened and unopened vectors and fragments were tested using a Saccharomyces   cerevisiae  rad52 mutant as the recombination host<br><br><br><br><br>
As can be seen in Table 15, the recovery of the Humicola lanuginosa lipase gene is very efficient. The last 5 rows in Table 15 shows that the opened vector alone or with only one fragment not covering the whole gap (see figure 3) gives only very few colonies.<br>
The first row is with wild type fragments gives 100% of active colonies.<br>
The second row is with two fragments each containing a frameshift. The fragment PCR331 fragment has the frameshift located at the Bglll site which, in this recombination, is not covered by a wild type fragment (see figure 3) and therefore gives about 0% of active lipase. The same is the case for row 3 and 6.<br>
In the row 4, fragment PCR386 containing a frameshift at the SphI site which is overlapped by wild type sequences in the gapped vector. The frameshift was recombined into less than 10% of the genes which is lower than the result for one fragment recombination in the last row of Table 1A above.<br>
In row 5 a rather high mixing is observed between the 2 fragments each containing a frameshift and the wild type gapped vector giving<br><br>
25% active and 75% inactive lipase colonies. This is probably due to that the fragment PCR321 has the frameshift in the overlap between the 2 fragments and in the gapped region of the vector, if<br>
fragment PCR386 contributes to 10% inactives like in row 4, fragment PCR321 gives the remaining 65% inactives - therefore PCR386 gives 35% wt in the overlap.<br>
Row 7 is the "mirror image" of row 4 with the frameshift at the SphI site on the vector (see Figure 7) and 2 wild type fragments giving an integration of the wild type fragment into more than 90% of the vectors.<br>
Row 8 shows like in row 5 that the frameshift of PCR321 in the overlap and gap region gives a very high number of inactive.<br>
In row 9, fragment PCR385 with a frameshift in the vector overlap, causes a very high number of inactives.<br>
Row 10 gives a rather high number of inactives compared to row 7 and 4. it is not increased in row 11.<br>
Row 12 shows that two frameshifts on the vector gives a lower number of actives compared to one in row 7.<br>
The recombination of 3 partial overlapping fragments into a gapped vector is also very efficient as seen in Table 16. The last row with the vector alone gives very few colonies. As can be seen in figure 4 all fragments used are wt. In the first row in table 16, there are rather long overlaps between the vector and fragments, but in the middle row the overlap between PCR353 and 355 is only 10 bp long and it is still very efficiently recombined! This surprising result may be utilized for very easy domain shuffling of even distantly related genes. For example can 3 different domains from 10 different genes be made as PCR fragments, designed to have a ID to 20 bp overlap by primer design and recombined together and subseguently screened for the best combination (1000 possible combinations).<br>
Table 16<br><br>
Vector + Fragment	Number of	% of colonies with active<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
WE CLAIM:<br>
1.	A method for preparing a shuffled polypeptide variant by in vivo recombination,<br>
said method comprising the steps of:<br>
(a)	linearizing at least one circular plasmid, wherein the plasmid comprises a DNA sequence encoding a polypeptide of interest and the linearization is within the DNA sequence encoding the polypeptide of interest;<br>
(b)	preparing two or more DNA fragments comprising DNA sequences encoding variants of the polypeptide of interest;<br>
(c)	introducing the at least one linearized plasmid of step (a) with the at least two DNA fragments of step (b) into a host cell, wherein recombination occurs between the at least one linearized plasmid and the two or more DNA fragments to generate a recombinant circular plasmid comprising a shuffled DNA sequence encoding the shuffled polypeptide variant;<br>
(d)	cultivating the host cell comprising the recombinant circular plasmid under conditions where the shuffled DNA sequence is expressed, and<br>
(e)	screening for the shuffled polypeptide variant.<br><br>
2.	The method as claimed in claim 1, wherein each DNA fragment introduced into the host cell encodes a polypeptide variant having at least 70% sequence identity to the polypeptide of interest.<br>
3.	The method as claimed in claim 1, wherein the DNA fragments prepared in step (b) are mutagenized.<br>
4.	The method as claimed in claim 1, wherein more than one cycle of step (a) through (e) are performed.<br><br>
The method as claimed in any of claims 1 to 4, wherein the opened plasmids are gapped.<br>
The method as claimed in any of claims 1 to 5, wherein the two or more DNA fragments have overlapping regions in the range from 5 to 5000 bp.<br>
The method as claimed in any of claims 1 to 6, wherein the polypeptide of interest is an enzyme or a protein with biological activity.<br>
The method as claimed in claim 7, wherein the polypeptide of interest is selected from the group of enzymes consisting of proteases, lipases, cutinases, cellulases, amylases, peroxidases, oxidases and phytases.<br>
The method as claimed in claim 7, wherein the polypeptide of interest is selected from the group of proteins with biological activity consisting of insulin, ACTH, glucagon, somatostatin, somatotropin, thymosin, parathyroid hormone, pigmentary hormones, somatomedin, erythropoietin, luteinizing hormone, chorionic gonadotropin, hypothalamic releasing factors, antidiuretic hormones, thyroid stimulating hormone, relaxin, interferon, thrombopoietin and prolactin.<br>
The method as claimed in any of claims 1 to 9, wherein the host cell is a eukaryotic cell.<br>
The method as claimed in claim 10, wherein the eukaryotic host cell is a fungal cell.<br><br>
12. The method as claimed in claim 11, wherein the fungal cell is a yeast cell from the group of Saccharomyces sp. or Schizosaccharomyces sp. or Kluyveromyces sp. or Pichia sp. or the fungal cell is a filamentous fungal cell from the group of Aspergillus sp. or Neurospora sp. or Fusarium sp. or Trichoderma sp.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQzOC1tYXMtMTk5NiBhYnN0cmFjdC1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1438-mas-1996 abstract-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQzOC1tYXMtMTk5NiBhYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1438-mas-1996 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQzOC1tYXMtMTk5NiBhc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">1438-mas-1996 assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQzOC1tYXMtMTk5NiBjbGFpbXMtZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">1438-mas-1996 claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQzOC1tYXMtMTk5NiBjbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">1438-mas-1996 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQzOC1tYXMtMTk5NiBjb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">1438-mas-1996 correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQzOC1tYXMtMTk5NiBjb3JyZXNwb25kZW5jZS1wby5wZGY=" target="_blank" style="word-wrap:break-word;">1438-mas-1996 correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQzOC1tYXMtMTk5NiBkZXNjcmlwdGlvbiAoY29tcGxldGUpLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">1438-mas-1996 description (complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQzOC1tYXMtMTk5NiBkZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">1438-mas-1996 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQzOC1tYXMtMTk5NiBkcmF3aW5ncy1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1438-mas-1996 drawings-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQzOC1tYXMtMTk5NiBkcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">1438-mas-1996 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQzOC1tYXMtMTk5NiBmb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">1438-mas-1996 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQzOC1tYXMtMTk5NiBmb3JtLTE5LnBkZg==" target="_blank" style="word-wrap:break-word;">1438-mas-1996 form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQzOC1tYXMtMTk5NiBmb3JtLTI2LnBkZg==" target="_blank" style="word-wrap:break-word;">1438-mas-1996 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQzOC1tYXMtMTk5NiBmb3JtLTQucGRm" target="_blank" style="word-wrap:break-word;">1438-mas-1996 form-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQzOC1tYXMtMTk5NiBmb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">1438-mas-1996 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQzOC1tYXMtMTk5NiBmb3JtLTYucGRm" target="_blank" style="word-wrap:break-word;">1438-mas-1996 form-6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTQzOC1tYXMtMTk5NiBwZXRpdGlvbi5wZGY=" target="_blank" style="word-wrap:break-word;">1438-mas-1996 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="227418-camptothecin-derivatives-and-a-process-for-preparing-the-camptothecin-compound.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="227420-antimycotic-agents-having-high-active-compound-release.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>227419</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1438/MAS/1996</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>10/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>06-Mar-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>07-Jan-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>14-Aug-1996</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVOZYMES A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>KROGSHOEJVEJ 36, DK-2880 BAGSVAERD</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>JENS SIGURD OKKELS</td>
											<td>C/O., NOVO NORDISK A/S, NOVO ALLE, DK-2880 BAGSVAERD</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/10</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>1047/95</td>
									<td>1995-09-20</td>
								    <td>Denmark</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/227419-method-for-preparing-polypeptide-variants by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:05:28 GMT -->
</html>
